<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATMI_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natmi---163">NATMI - 163</h2>
<ul>
<li><details>
<summary>
(2022b). Publisher correction: Closed-form continuous-time neural
networks. <em>NATMI</em>, <em>4</em>(12), 1267. (<a
href="https://doi.org/10.1038/s42256-022-00597-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Hasani, Ramin and Lechner, Mathias and Amini, Alexander and Liebenwein, Lucas and Ray, Aaron and Tschaikowski, Max and Teschl, Gerald and Rus, Daniela},
  doi          = {10.1038/s42256-022-00597-y},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1267},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Closed-form continuous-time neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Low-power object-detection challenge on unmanned aerial
vehicles. <em>NATMI</em>, <em>4</em>(12), 1265–1266. (<a
href="https://doi.org/10.1038/s42256-022-00567-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A design contest for object detection with deep learning on embedded small devices leads to winning hardware–software co-design approaches.},
  archive      = {J_NATMI},
  author       = {Jia, Zhenge and Xu, Xiaowei and Hu, Jingtong and Shi, Yiyu},
  doi          = {10.1038/s42256-022-00567-4},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1265-1266},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Low-power object-detection challenge on unmanned aerial vehicles},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Large-scale chemical language representations capture
molecular structure and properties. <em>NATMI</em>, <em>4</em>(12),
1256–1264. (<a
href="https://doi.org/10.1038/s42256-022-00580-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Models based on machine learning can enable accurate and fast molecular property predictions, which is of interest in drug discovery and material design. Various supervised machine learning models have demonstrated promising performance, but the vast chemical space and the limited availability of property labels make supervised learning challenging. Recently, unsupervised transformer-based language models pretrained on a large unlabelled corpus have produced state-of-the-art results in many downstream natural language processing tasks. Inspired by this development, we present molecular embeddings obtained by training an efficient transformer encoder model, MoLFormer, which uses rotary positional embeddings. This model employs a linear attention mechanism, coupled with highly distributed training, on SMILES sequences of 1.1 billion unlabelled molecules from the PubChem and ZINC datasets. We show that the learned molecular representation outperforms existing baselines, including supervised and self-supervised graph neural networks and language models, on several downstream tasks from ten benchmark datasets. They perform competitively on two others. Further analyses, specifically through the lens of attention, demonstrate that MoLFormer trained on chemical SMILES indeed learns the spatial relationships between atoms within a molecule. These results provide encouraging evidence that large-scale molecular language models can capture sufficient chemical and structural information to predict various distinct molecular properties, including quantum-chemical properties. Large language models have recently emerged with extraordinary capabilities, and these methods can be applied to model other kinds of sequence, such as string representations of molecules. Ross and colleagues have created a transformer-based model, trained on a large dataset of molecules, which provides good results on property prediction tasks.},
  archive      = {J_NATMI},
  author       = {Ross, Jerret and Belgodere, Brian and Chenthamarakshan, Vijil and Padhi, Inkit and Mroueh, Youssef and Das, Payel},
  doi          = {10.1038/s42256-022-00580-7},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1256-1264},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Large-scale chemical language representations capture molecular structure and properties},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Simple nearest-neighbour analysis meets the accuracy of
compound potency predictions using complex machine learning models.
<em>NATMI</em>, <em>4</em>(12), 1246–1255. (<a
href="https://doi.org/10.1038/s42256-022-00581-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Compound potency prediction is a popular application of machine learning in drug discovery, for which increasingly complex models are employed. The general aim is the identification of new chemical entities that are highly potent against a given target. The relative performance of potency prediction models and their accuracy limitations continue to be debated in the field, and it remains unclear whether deep learning can further advance potency prediction. We have analysed and compared approaches of varying computational complexity for potency prediction and shown that simple nearest-neighbour analysis consistently meets or exceeds the accuracy of machine learning methods regarded as the state of the art in the field. Moreover, completely random predictions using different models were shown to reproduce experimental values within an order of magnitude, resulting from the potency value distributions in commonly used compound data sets. Taken together, these findings have important implications for typical benchmark calculations to evaluate machine learning performance. Simple controls such as nearest-neighbour analysis should generally be included in model evaluation. Furthermore, the narrow margin separating the best and completely random potency predictions is unrealistic and requires the consideration of alternative benchmark criteria, as discussed herein. There is growing interest in using sophisticated machine learning models for the prediction of molecular properties, such as potency of novel drugs. However, Janela and Bajorath show that simple nearest-neighbour analysis meets or exceeds the accuracy of state-of-the-art complex machine learning methods and that randomized prediction models still reproduce compound potency values within an order of magnitude.},
  archive      = {J_NATMI},
  author       = {Janela, Tiago and Bajorath, Jürgen},
  doi          = {10.1038/s42256-022-00581-6},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1246-1255},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Simple nearest-neighbour analysis meets the accuracy of compound potency predictions using complex machine learning models},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A critical problem in benchmarking and analysis of
evolutionary computation methods. <em>NATMI</em>, <em>4</em>(12),
1238–1245. (<a
href="https://doi.org/10.1038/s42256-022-00579-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Benchmarking is a cornerstone in the analysis and development of computational methods, especially in the field of evolutionary computation, where theoretical analysis of the algorithms is almost impossible. In this Article, we show that some of the frequently used benchmark functions have their respective optima in the centre of the feasible set and that this poses a critical problem for the analysis of evolutionary computation methods. We carry out an analysis of seven recently published methods and find that these contain a centre-bias operator that lets them find optima in the centre of the benchmark set with ease. However, this mechanism makes their comparison with other methods (that do not have a centre-bias) meaningless. We compare the computational performance of these seven new methods to two long-standing ones in evolutionary computation (‘differential evolution’ and ‘particle swarm optimization’) on shifted problems and on more advanced benchmark problems. Only one of the seven methods performed consistently better than the pair of old methods, three performed on par, two performed very badly and the worst one performed barely better than a random search. We provide several suggestions that could help to improve analysis and benchmarking in evolutionary computation. Evolutionary computation is a very active field of research, with an ever-growing number of metaheuristic optimization algorithms being published. A serious problem plaguing the field is the use of inadequate benchmarks. Kudela exposes the issue and provides recommendations that can help to fairly evaluate and compare new methods.},
  archive      = {J_NATMI},
  author       = {Kudela, Jakub},
  doi          = {10.1038/s42256-022-00579-0},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1238-1245},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A critical problem in benchmarking and analysis of evolutionary computation methods},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Joint structural annotation of small molecules using liquid
chromatography retention order and tandem mass spectrometry data.
<em>NATMI</em>, <em>4</em>(12), 1224–1237. (<a
href="https://doi.org/10.1038/s42256-022-00577-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural annotation of small molecules in biological samples remains a key bottleneck in untargeted metabolomics, despite rapid progress in predictive methods and tools during the past decade. Liquid chromatography–tandem mass spectrometry, one of the most widely used analysis platforms, can detect thousands of molecules in a sample, the vast majority of which remain unidentified even with best-of-class methods. Here we present LC-MS2Struct, a machine learning framework for structural annotation of small-molecule data arising from liquid chromatography–tandem mass spectrometry (LC-MS2) measurements. LC-MS2Struct jointly predicts the annotations for a set of mass spectrometry features in a sample, using a novel structured prediction model trained to optimally combine the output of state-of-the-art MS2 scorers and observed retention orders. We evaluate our method on a dataset covering all publicly available reversed-phase LC-MS2 data in the MassBank reference database, including 4,327 molecules measured using 18 different LC conditions from 16 contributors, greatly expanding the chemical analytical space covered in previous multi-MS2 scorer evaluations. LC-MS2Struct obtains significantly higher annotation accuracy than earlier methods and improves the annotation accuracy of state-of-the-art MS2 scorers by up to 106\%. The use of stereochemistry-aware molecular fingerprints improves prediction performance, which highlights limitations in existing approaches and has strong implications for future computational LC-MS2 developments. Liquid chromatography–tandem mass spectrometry (LC-MS2) provides high-throughput screening of molecules with a large number of features. But these features are difficult to associate with specific molecular structures of each sample. To improve structure prediction from these features, Bach et al. propose a machine learning model trained to also take into account stereochemistry to combine the different kinds of features provided by LC-MS2.},
  archive      = {J_NATMI},
  author       = {Bach, Eric and Schymanski, Emma L. and Rousu, Juho},
  doi          = {10.1038/s42256-022-00577-2},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1224-1237},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Joint structural annotation of small molecules using liquid chromatography retention order and tandem mass spectrometry data},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A generalizable deep learning framework for inferring
fine-scale germline mutation rate maps. <em>NATMI</em>, <em>4</em>(12),
1209–1223. (<a
href="https://doi.org/10.1038/s42256-022-00574-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Germline mutation rates are essential for genetic and evolutionary analyses. Yet, estimating accurate fine-scale mutation rates across the genome is a great challenge, due to relatively few observed mutations and intricate relationships between predictors and mutation rates. Here, we present Mutation Rate Learner (MuRaL), a deep learning framework to predict mutation rates at the nucleotide level using only genomic sequences as input. Harnessing human germline variants for comprehensive assessment, we show that MuRaL achieves better predictive performance than current state-of-the-art methods. Moreover, MuRaL can build models with relatively few training mutations and a moderate number of sequenced individuals, and can leverage transfer learning to further reduce data and time demands. We apply MuRaL to produce genome-wide mutation rate maps for four representative species—Homo sapiens, Macaca mulatta, Drosophila melanogaster and Arabidopsis thaliana—demonstrating its high applicability. As an example, we use improved mutation rate estimates to stratify human genes into distinct groups that are enriched for different functions, and highlight that many developmental genes are subject to high mutational burden. The open-source software and generated mutation rate maps can greatly facilitate related research. Mutation rates are crucial for genetic and evolutionary analyses. Fang et al. present a generalizable deep learning method to build fine-scale mutation rate maps with DNA sequences as input, which can benefit analyses reliant on mutation rates.},
  archive      = {J_NATMI},
  author       = {Fang, Yiyuan and Deng, Shuyi and Li, Cai},
  doi          = {10.1038/s42256-022-00574-5},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1209-1223},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A generalizable deep learning framework for inferring fine-scale germline mutation rate maps},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High-speed quadrupedal locomotion by imitation-relaxation
reinforcement learning. <em>NATMI</em>, <em>4</em>(12), 1198–1208. (<a
href="https://doi.org/10.1038/s42256-022-00576-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fast and stable locomotion of legged robots involves demanding and contradictory requirements, in particular rapid control frequency as well as an accurate dynamics model. Benefiting from universal approximation ability and offline optimization of neural networks, reinforcement learning has been used to solve various challenging problems in legged robot locomotion; however, the optimal control of quadruped robot requires optimizing multiple objectives such as keeping balance, improving efficiency, realizing periodic gait and following commands. These objectives cannot always be achieved simultaneously, especially at high speed. Here, we introduce an imitation-relaxation reinforcement learning (IRRL) method to optimize the objectives in stages. To bridge the gap between simulation and reality, we further introduce the concept of stochastic stability into system robustness analysis. The state space entropy decreasing rate is a quantitative metric and can sharply capture the occurrence of period-doubling bifurcation and possible chaos. By employing IRRL in training and the stochastic stability analysis, we are able to demonstrate a stable running speed of 5.0 m s–1 for a MIT-MiniCheetah-like robot. The design of legged robots with agility and speed is challenging. The authors present a method with reinforcement learning-based controllers for locomotion control of quadruped robots. The pipeline achieves improvements in performance, such as running speed.},
  archive      = {J_NATMI},
  author       = {Jin, Yongbin and Liu, Xianwei and Shao, Yecheng and Wang, Hongtao and Yang, Wei},
  doi          = {10.1038/s42256-022-00576-3},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1198-1208},
  shortjournal = {Nat. Mach. Intell.},
  title        = {High-speed quadrupedal locomotion by imitation-relaxation reinforcement learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Three types of incremental learning. <em>NATMI</em>,
<em>4</em>(12), 1185–1197. (<a
href="https://doi.org/10.1038/s42256-022-00568-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Incrementally learning new information from a non-stationary stream of data, referred to as ‘continual learning’, is a key feature of natural intelligence, but a challenging problem for deep neural networks. In recent years, numerous deep learning methods for continual learning have been proposed, but comparing their performances is difficult due to the lack of a common framework. To help address this, we describe three fundamental types, or ‘scenarios’, of continual learning: task-incremental, domain-incremental and class-incremental learning. Each of these scenarios has its own set of challenges. To illustrate this, we provide a comprehensive empirical comparison of currently used continual learning strategies, by performing the Split MNIST and Split CIFAR-100 protocols according to each scenario. We demonstrate substantial differences between the three scenarios in terms of difficulty and in terms of the effectiveness of different strategies. The proposed categorization aims to structure the continual learning field, by forming a key foundation for clearly defining benchmark problems. A challenge for any machine learning system is to continually adapt to new data. While methods to address this issue are developed, their performance is hard to compare. A new framework to facilitate benchmarking divides approaches into three categories, defined by whether models need to adapt to new tasks, domains or classes.},
  archive      = {J_NATMI},
  author       = {van de Ven, Gido M. and Tuytelaars, Tinne and Tolias, Andreas S.},
  doi          = {10.1038/s42256-022-00568-3},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1185-1197},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Three types of incremental learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep learning models for predicting RNA degradation via dual
crowdsourcing. <em>NATMI</em>, <em>4</em>(12), 1174–1184. (<a
href="https://doi.org/10.1038/s42256-022-00571-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Medicines based on messenger RNA (mRNA) hold immense potential, as evidenced by their rapid deployment as COVID-19 vaccines. However, worldwide distribution of mRNA molecules has been limited by their thermostability, which is fundamentally limited by the intrinsic instability of RNA molecules to a chemical degradation reaction called in-line hydrolysis. Predicting the degradation of an RNA molecule is a key task in designing more stable RNA-based therapeutics. Here, we describe a crowdsourced machine learning competition (‘Stanford OpenVaccine’) on Kaggle, involving single-nucleotide resolution measurements on 6,043 diverse 102–130-nucleotide RNA constructs that were themselves solicited through crowdsourcing on the RNA design platform Eterna. The entire experiment was completed in less than 6 months, and 41\% of nucleotide-level predictions from the winning model were within experimental error of the ground truth measurement. Furthermore, these models generalized to blindly predicting orthogonal degradation data on much longer mRNA molecules (504–1,588 nucleotides) with improved accuracy compared with previously published models. These results indicate that such models can represent in-line hydrolysis with excellent accuracy, supporting their use for designing stabilized messenger RNAs. The integration of two crowdsourcing platforms, one for dataset creation and another for machine learning, may be fruitful for other urgent problems that demand scientific discovery on rapid timescales. Predicting RNA degradation is a fundamental task in designing RNA-based therapeutics. Two crowdsourcing platforms, Kaggle and Eterna, united to develop accurate deep learning models for RNA degradation on a timescale of 6 months.},
  archive      = {J_NATMI},
  author       = {Wayment-Steele, Hannah K. and Kladwang, Wipapat and Watkins, Andrew M. and Kim, Do Soon and Tunguz, Bojan and Reade, Walter and Demkin, Maggie and Romano, Jonathan and Wellington-Oguri, Roger and Nicol, John J. and Gao, Jiayang and Onodera, Kazuki and Fujikawa, Kazuki and Mao, Hanfei and Vandewiele, Gilles and Tinti, Michele and Steenwinckel, Bram and Ito, Takuya and Noumi, Taiga and He, Shujun and Ishi, Keiichiro and Lee, Youhan and Öztürk, Fatih and Chiu, King Yuen and Öztürk, Emin and Amer, Karim and Fares, Mohamed and Das, Rhiju},
  doi          = {10.1038/s42256-022-00571-8},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1174-1184},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning models for predicting RNA degradation via dual crowdsourcing},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Super-resolution generative adversarial networks of
randomly-seeded fields. <em>NATMI</em>, <em>4</em>(12), 1165–1173. (<a
href="https://doi.org/10.1038/s42256-022-00572-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reconstruction of field quantities from sparse measurements is a problem arising in a broad spectrum of applications. This task is particularly challenging when the mapping between sparse measurements and field quantities is performed in an unsupervised manner. Further complexity is added for moving sensors and/or random on–off status. Under such conditions, the most straightforward solution is to interpolate the scattered data onto a regular grid. However, the spatial resolution achieved with this approach is ultimately limited by the mean spacing between the sparse measurements. In this work, we propose a super-resolution generative adversarial network framework to estimate field quantities from random sparse sensors. The algorithm exploits random sampling to provide incomplete views of the high-resolution underlying distributions. It is hereby referred to as the randomly seeded super-resolution generative adversarial network (RaSeedGAN). The proposed technique is tested on synthetic databases of fluid flow simulations, ocean surface temperature distribution measurements and particle-image velocimetry data of a zero-pressure-gradient turbulent boundary layer. The results show excellent performance even in cases with high sparsity or noise level. This generative adversarial network algorithm provides full-field high-resolution estimation from randomly seeded fields with no need of full-field high-resolution representations for training. The problem of reconstructing full-field quantities from incomplete observations arises in various real-world applications. Güemes and colleagues propose a super-resolution algorithm based on a generative adversarial network that can achieve reconstruction of the underlying field from random sparse measurements without requiring full-field high-resolution training data.},
  archive      = {J_NATMI},
  author       = {Güemes, Alejandro and Sanmiguel Vila, Carlos and Discetti, Stefano},
  doi          = {10.1038/s42256-022-00572-7},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1165-1173},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Super-resolution generative adversarial networks of randomly-seeded fields},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep transfer operator learning for partial differential
equations under conditional shift. <em>NATMI</em>, <em>4</em>(12),
1155–1164. (<a
href="https://doi.org/10.1038/s42256-022-00569-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transfer learning enables the transfer of knowledge gained while learning to perform one task (source) to a related but different task (target), hence addressing the expense of data acquisition and labelling, potential computational power limitations and dataset distribution mismatches. We propose a new transfer learning framework for task-specific learning (functional regression in partial differential equations) under conditional shift based on the deep operator network (DeepONet). Task-specific operator learning is accomplished by fine-tuning task-specific layers of the target DeepONet using a hybrid loss function that allows for the matching of individual target samples while also preserving the global properties of the conditional distribution of the target data. Inspired by conditional embedding operator theory, we minimize the statistical distance between labelled target data and the surrogate prediction on unlabelled target data by embedding conditional distributions onto a reproducing kernel Hilbert space. We demonstrate the advantages of our approach for various transfer learning scenarios involving nonlinear partial differential equations under diverse conditions due to shifts in the geometric domain and model dynamics. Our transfer learning framework enables fast and efficient learning of heterogeneous tasks despite considerable differences between the source and target domains. A promising area for deep learning is in modelling complex physical processes described by partial differential equations (PDEs), which is computationally expensive for conventional approaches. An operator learning approach called DeepONet was recently introduced to tackle PDE-related problems, and in new work, this approach is extended with transfer learning, which transfers knowledge obtained from learning to perform one task to a related but different task.},
  archive      = {J_NATMI},
  author       = {Goswami, Somdatta and Kontolati, Katiana and Shields, Michael D. and Karniadakis, George Em},
  doi          = {10.1038/s42256-022-00569-2},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1155-1164},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep transfer operator learning for partial differential equations under conditional shift},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Detecting multiple retinal diseases in ultra-widefield
fundus imaging and data-driven identification of informative regions
with deep learning. <em>NATMI</em>, <em>4</em>(12), 1143–1154. (<a
href="https://doi.org/10.1038/s42256-022-00566-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultra-widefield (UWF) imaging is a promising modality that captures a larger retinal field of view compared with traditional fundus photography. Previous studies have shown that deep learning models are effective for detecting retinal disease in UWF images, but primarily considered individual diseases under less-than-realistic conditions (excluding images with other diseases, artefacts, comorbidities or borderline cases; and balancing healthy and diseased images) and did not systematically investigate which regions of the UWF images are relevant for disease detection. Here we first improve on the state of the field by proposing a deep learning model that can recognize multiple retinal diseases under more realistic conditions than what has previously been considered. We then use global explainability methods to identify which regions of the UWF images the model generally attends to. Our model performs very well, separating between healthy and diseased retinas with an area under the receiver operating characteristic curve (AUC) of 0.9196 (±0.0001) on an internal test set, and an AUC of 0.9848 (±0.0004) on a challenging, external test set. When diagnosing specific diseases, the model attends to regions where we would expect those diseases to occur. We further identify the posterior pole as the most important region in a purely data-driven fashion. Surprisingly, 10\% of the image around the posterior pole is sufficient for achieving comparable performance across all labels to having the full images available. Advances in ultra-widefield retinal imaging have created a need for automated disease detection. Engelmann and colleagues develop a deep learning model for the detection of retinal diseases. They evaluate it under more realistic conditions than has been considered previously and investigate what regions of ultra-widefield images are important for the performance of such a model.},
  archive      = {J_NATMI},
  author       = {Engelmann, Justin and McTrusty, Alice D. and MacCormick, Ian J. C. and Pead, Emma and Storkey, Amos and Bernabeu, Miguel O.},
  doi          = {10.1038/s42256-022-00566-5},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1143-1154},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Detecting multiple retinal diseases in ultra-widefield fundus imaging and data-driven identification of informative regions with deep learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A multilevel generative framework with hierarchical
self-contrasting for bias control and transparency in structure-based
ligand design. <em>NATMI</em>, <em>4</em>(12), 1130–1142. (<a
href="https://doi.org/10.1038/s42256-022-00564-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative models for structure-based molecular design hold considerable promise for drug discovery, with the potential to speed up the hit-to-lead development cycle while improving the quality of drug candidates and reducing costs. Data sparsity and bias are, however, the two main roadblocks to the development of three-dimensionally aware models. Here we propose a training protocol based on multilevel self-contrastive learning for improved bias control and data efficiency. The framework leverages the large data resources available for two-dimensional generative modelling with datasets of ligand–protein complexes, resulting in hierarchical generative models that are topologically unbiased, explainable and customizable. We show how, by deconvolving the generative posterior into chemical, topological and structural context factors, we not only avoid common pitfalls in the design and evaluation of generative models, but also gain detailed insight into the generative process itself. This improved transparency considerably aids method development and allows fine-grained control over novelty versus familiarity. Generative AI that transforms candidate molecules into potent drugs could pave the way towards better, faster drug design. Chan and co-workers developed a self-contrastive learning framework that helps AI models overcome data biases and provide explainable insights for augmented interactive design.},
  archive      = {J_NATMI},
  author       = {Chan, Lucian and Kumar, Rajendra and Verdonk, Marcel and Poelking, Carl},
  doi          = {10.1038/s42256-022-00564-7},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1130-1142},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A multilevel generative framework with hierarchical self-contrasting for bias control and transparency in structure-based ligand design},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Generalizability of an acute kidney injury prediction model
across health systems. <em>NATMI</em>, <em>4</em>(12), 1121–1129. (<a
href="https://doi.org/10.1038/s42256-022-00563-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Delays in the identification of acute kidney injury in hospitalized patients are a major barrier to the development of effective interventions for treatment. A recent study described a series of models that outperformed previously published models in predicting acute kidney injury up to 48 h in advance, including a recurrent neural network that achieved state-of-the-art performance (area under the curve 0.92) and a gradient-boosted decision tree model that was close behind (area under the curve 0.89). Because these models were trained in a population of US veterans that was 94\% male, questions have arisen about its generalizability to other health systems where the populations are more sex balanced. In this study, we aimed to evaluate how well an acute kidney injury model trained in a population of US veterans performs in females at the Veterans Affairs and the extent to which its performance generalizes to a large academic hospital setting. We found that the model performed worse in predicting acute kidney injury in females in both populations, with miscalibration in lower stages of acute kidney injury and worse discrimination (a lower area under the curve) in higher stages of acute kidney injury. We demonstrate that, while this discrepancy in performance can be largely corrected in non-veterans by updating the original model using data from a sex-balanced academic hospital cohort, the worse model performance persists in veterans. Our study sheds light on the importance of characterizing the generalizability of artificial intelligence studies, and on the complexity of discrepancies in model performance in subgroups that cannot be explained simply on the basis of sample size. The lack of generalizability and reproducibility of machine learning models in medical applications is increasingly recognized as a substantial barrier to implementing such approaches in real-world clinical settings. Highlighting this issue, Jie Cao et al. aim to reproduce a recent acute kidney injury prediction model, and find persistent discrepancies in model performance in different subgroups.},
  archive      = {J_NATMI},
  author       = {Cao, Jie and Zhang, Xiaosong and Shahinian, Vahakn and Yin, Huiying and Steffick, Diane and Saran, Rajiv and Crowley, Susan and Mathis, Michael and Nadkarni, Girish N. and Heung, Michael and Singh, Karandeep},
  doi          = {10.1038/s42256-022-00563-8},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1121-1129},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Generalizability of an acute kidney injury prediction model across health systems},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Data-driven discovery of intrinsic dynamics. <em>NATMI</em>,
<em>4</em>(12), 1113–1120. (<a
href="https://doi.org/10.1038/s42256-022-00575-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamical models underpin our ability to understand and predict the behaviour of natural systems. Whether dynamical models are developed from first-principles derivations or from observational data, they are predicated on our choice of state variables. The choice of state variables is driven by convenience and intuition, and, in data-driven cases, the observed variables are often chosen to be the state variables. The dimensionality of these variables (and consequently the dynamical models) can be arbitrarily large, obscuring the underlying behaviour of the system. In truth these variables are often highly redundant and the system is driven by a much smaller set of latent intrinsic variables. In this study we combine the mathematical theory of manifolds with the representational capacity of neural networks to develop a method that learns a system’s intrinsic state variables directly from time-series data, as well as predictive models for their dynamics. What distinguishes our method is its ability to reduce data to the intrinsic dimensionality of the nonlinear manifold they live on. This ability is enabled by the concepts of charts and atlases from the theory of manifolds, whereby a manifold is represented by a collection of patches that are sewn together—a necessary representation to attain intrinsic dimensionality. We demonstrate this approach on several high-dimensional systems with low-dimensional behaviour. The resulting framework provides the ability to develop dynamical models of the lowest possible dimension, capturing the essence of a system. Learning minimal representations of dynamical systems is essential for mathematical modelling and prediction in science and engineering. Floryan and Graham propose a deep learning framework able to estimate accurate global dynamical models by sewing together multiple local representations learnt from high-dimensional time-series data.},
  archive      = {J_NATMI},
  author       = {Floryan, Daniel and Graham, Michael D.},
  doi          = {10.1038/s42256-022-00575-4},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1113-1120},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Data-driven discovery of intrinsic dynamics},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). AtomAI framework for deep learning analysis of image and
spectroscopy data in electron and scanning probe microscopy.
<em>NATMI</em>, <em>4</em>(12), 1101–1112. (<a
href="https://doi.org/10.1038/s42256-022-00555-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past several decades, electron and scanning probe microscopes have become critical components of condensed matter physics, materials science and chemistry research. At the same time, the infrastructure for establishing a connection between microscopy observations and materials behaviour over a broader parameter space is lacking. Here we introduce AtomAI, an open-source software package bridging instrument-specific Python libraries, deep learning and simulation tools into a single ecosystem. AtomAI allows direct applications of deep neural networks for atomic and mesoscopic image segmentation converting image and spectroscopy data into class-based local descriptors for downstream tasks such as statistical and graph analysis. For atomically resolved imaging data, the output is types and positions of atomic species, with an option for subsequent refinement. AtomAI further allows the implementation of a broad range of image and spectrum analysis functions, including invariant variational autoencoders for disentangling structural factors of variation and im2spec type of encoder–decoder models for mapping structure–property relationships. Finally, our framework allows seamless connection to the first principles modelling with a Python interface on the inferred atomic positions. In recent years, deep learning techniques have enhanced the possibility to extract useful, high-resolution physical information from electron and scanning probe microscopy images. AtomAI, an open-source software package, can accelerate this process by bringing deep learning and simulation tools into a single framework for a range of instruments.},
  archive      = {J_NATMI},
  author       = {Ziatdinov, Maxim and Ghosh, Ayana and Wong, Chun Yin (Tommy) and Kalinin, Sergei V.},
  doi          = {10.1038/s42256-022-00555-8},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1101-1112},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AtomAI framework for deep learning analysis of image and spectroscopy data in electron and scanning probe microscopy},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evaluating deep learning for predicting epigenomic profiles.
<em>NATMI</em>, <em>4</em>(12), 1088–1100. (<a
href="https://doi.org/10.1038/s42256-022-00570-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning has been successful at predicting epigenomic profiles from DNA sequences. Most approaches frame this task as a binary classification relying on peak callers to define functional activity. Recently, quantitative models have emerged to directly predict the experimental coverage values as a regression. As new models with different architectures and training configurations continue to emerge, a major bottleneck is forming due to the lack of ability to fairly assess the novelty of proposed models and their utility for downstream biological discovery. Here we introduce a unified evaluation framework and use it to compare various binary and quantitative models trained to predict chromatin accessibility data. We highlight various modelling choices that affect generalization performance, including a downstream application of predicting variant effects. In addition, we introduce a robustness metric that can be used to enhance model selection and improve variant effect predictions. Our empirical study largely supports that quantitative modelling of epigenomic profiles leads to better generalizability and interpretability. Recent developments in deep learning have allowed for a leap in computational analysis of epigenomic data, but a fair comparison of different architectures is challenging. Toneyan et al. use GOPHER, their new framework for model evaluation and comparison, to perform a comprehensive analysis, exploring modelling choices of deep learning for epigenomic profiles.},
  archive      = {J_NATMI},
  author       = {Toneyan, Shushan and Tang, Ziqi and Koo, Peter K.},
  doi          = {10.1038/s42256-022-00570-9},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1088-1100},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Evaluating deep learning for predicting epigenomic profiles},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Transferring policy of deep reinforcement learning from
simulation to reality for robotics. <em>NATMI</em>, <em>4</em>(12),
1077–1087. (<a
href="https://doi.org/10.1038/s42256-022-00573-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep reinforcement learning has achieved great success in many fields and has shown promise in learning robust skills for robot control in recent years. However, sampling efficiency and safety problems still limit its application to robot control in the real world. One common solution is to train the robot control policy in a simulation environment and transfer it to the real world. However, policies trained in simulations usually have unsatisfactory performance in the real world because simulators inevitably model reality imperfectly. Inspired by biological transfer learning processes in the brains of humans and other animals, sim-to-real transfer reinforcement learning has been proposed and has become a focus of researchers applying reinforcement learning to robotics. Here, we describe state-of-the-art sim-to-real transfer reinforcement learning methods, which are inspired by insights into transfer learning in nature, such as extracting features in common between tasks, enriching training experience, multitask learning, continual learning and fast learning. Our objective is to present a comprehensive survey of the most recent advances in sim-to-real transfer reinforcement learning. We hope it can facilitate the application of deep reinforcement learning to solve complex robot control problems in the real world. Applying deep reinforcement learning to robot control poses challenges. The authors review methods for transferring deep reinforcement learning policies learned in simulation to the real world.},
  archive      = {J_NATMI},
  author       = {Ju, Hao and Juan, Rongshun and Gomez, Randy and Nakamura, Keisuke and Li, Guangliang},
  doi          = {10.1038/s42256-022-00573-6},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1077-1087},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Transferring policy of deep reinforcement learning from simulation to reality for robotics},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Language and culture internalization for human-like
autotelic AI. <em>NATMI</em>, <em>4</em>(12), 1068–1076. (<a
href="https://doi.org/10.1038/s42256-022-00591-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Building autonomous agents able to grow open-ended repertoires of skills across their lives is a fundamental goal of artificial intelligence (AI). A promising developmental approach recommends the design of intrinsically motivated agents that learn new skills by generating and pursuing their own goals—autotelic agents. But despite recent progress, existing algorithms still show serious limitations in terms of goal diversity, exploration, generalization or skill composition. This Perspective calls for the immersion of autotelic agents into rich socio-cultural worlds, an immensely important attribute of our environment that shapes human cognition but is mostly omitted in modern AI. Inspired by the seminal work of Vygotsky, we propose Vygotskian autotelic agents—agents able to internalize their interactions with others and turn them into cognitive tools. We focus on language and show how its structure and informational content may support the development of new cognitive functions in artificial agents as it does in humans. We justify the approach by uncovering several examples of new artificial cognitive functions emerging from interactions between language and embodiment in recent works at the intersection of deep reinforcement learning and natural language processing. Looking forward, we highlight future opportunities and challenges for Vygotskian autotelic AI research, including the use of language models as cultural models supporting artificial cognitive development. A goal of AI is to develop autonomous artificial agents with a wide set of skills. The authors propose the immersion of intrinsically motivated agents within rich socio-cultural worlds, focusing on language as a way for artificial agents to develop new cognitive functions.},
  archive      = {J_NATMI},
  author       = {Colas, Cédric and Karch, Tristan and Moulin-Frier, Clément and Oudeyer, Pierre-Yves},
  doi          = {10.1038/s42256-022-00591-4},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1068-1076},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Language and culture internalization for human-like autotelic AI},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Interpretability of artificial neural network models in
artificial intelligence versus neuroscience. <em>NATMI</em>,
<em>4</em>(12), 1065–1067. (<a
href="https://doi.org/10.1038/s42256-022-00592-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The notion of ‘interpretability’ of artificial neural networks (ANNs) is of growing importance in neuroscience and artificial intelligence (AI). But interpretability means different things to neuroscientists as opposed to AI researchers. In this article, we discuss the potential synergies and tensions between these two communities in interpreting ANNs.},
  archive      = {J_NATMI},
  author       = {Kar, Kohitij and Kornblith, Simon and Fedorenko, Evelina},
  doi          = {10.1038/s42256-022-00592-3},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1065-1067},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Interpretability of artificial neural network models in artificial intelligence versus neuroscience},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Advancing ethics review practices in AI research.
<em>NATMI</em>, <em>4</em>(12), 1061–1064. (<a
href="https://doi.org/10.1038/s42256-022-00585-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The implementation of ethics review processes is an important first step for anticipating and mitigating the potential harms of AI research. Its long-term success, however, requires a coordinated community effort, to support experimentation with different ethics review processes, to study their effect, and to provide opportunities for diverse voices from the community to share insights and foster norms.},
  archive      = {J_NATMI},
  author       = {Srikumar, Madhulika and Finlay, Rebecca and Abuhamad, Grace and Ashurst, Carolyn and Campbell, Rosie and Campbell-Ratcliffe, Emily and Hongo, Hudson and Jordan, Sara R. and Lindley, Joseph and Ovadya, Aviv and Pineau, Joelle},
  doi          = {10.1038/s42256-022-00585-2},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1061-1064},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Advancing ethics review practices in AI research},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Can AI invent? <em>NATMI</em>, <em>4</em>(12), 1057–1060.
(<a href="https://doi.org/10.1038/s42256-022-00582-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence systems are used for an increasing range of intellectual tasks, but can they invent, or will they be able to do so soon? A recent series of patent applications for two inventions that are claimed to have been made by an artificial intelligence program are bringing these questions to the fore.},
  archive      = {J_NATMI},
  author       = {George, Alexandra and Walsh, Toby},
  doi          = {10.1038/s42256-022-00582-5},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1057-1060},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Can AI invent?},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Much to discuss in AI ethics. <em>NATMI</em>,
<em>4</em>(12), 1055–1056. (<a
href="https://doi.org/10.1038/s42256-022-00598-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {2022 has seen eye-catching developments in AI applications. Work is needed to ensure that ethical reflection and responsible publication practices are keeping pace.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00598-x},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1055-1056},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Much to discuss in AI ethics},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Author correction: A generalized-template-based graph
neural network for accurate organic reactivity prediction.
<em>NATMI</em>, <em>4</em>(11), 1054. (<a
href="https://doi.org/10.1038/s42256-022-00586-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Chen, Shuan and Jung, Yousung},
  doi          = {10.1038/s42256-022-00586-1},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {1054},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: A generalized-template-based graph neural network for accurate organic reactivity prediction},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Gradient-based learning drives robust
representations in recurrent neural networks by balancing compression
and expansion. <em>NATMI</em>, <em>4</em>(11), 1053. (<a
href="https://doi.org/10.1038/s42256-022-00565-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Farrell, Matthew and Recanatesi, Stefano and Moore, Timothy and Lajoie, Guillaume and Shea-Brown, Eric},
  doi          = {10.1038/s42256-022-00565-6},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {1053},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Gradient-based learning drives robust representations in recurrent neural networks by balancing compression and expansion},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Human-behaviour-based social locomotion model improves the
humanization of social robots. <em>NATMI</em>, <em>4</em>(11),
1040–1052. (<a
href="https://doi.org/10.1038/s42256-022-00542-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The widespread use of robots in service fields requires humanoid robots that mimic human social behaviour. Previous quantitative studies exist in human social behaviour, but engineering social robots requires translating these findings into algorithms to enable reliable and safe robot locomotion. To bridge this gap, we first quantitatively investigate the social rules that apply when people pass one another in social settings in laboratory and real-world experiments. We then developed a social locomotion model based on these observations to predict human path selections and walking trajectories in complex dynamic social scenes. The model was implemented into a socially aware navigation algorithm for a service robot. The robot navigating by the social locomotion algorithm behaved more like humans and received higher comfort ratings compared with previous social navigation algorithms tested. The model sheds new light on how to directly translate the findings of human behavioural experiments into robotic engineering. To be useful in human life, robots need to learn the social rules of human society. Zhou et al. investigate the social rules that apply in spaces mutually occupied by humans and robots. The authors develop a social locomotion model for a mobile robot and implement it for socially aware navigation.},
  archive      = {J_NATMI},
  author       = {Zhou, Chen and Miao, Ming-Cheng and Chen, Xin-Ran and Hu, Yi-Fei and Chang, Qi and Yan, Ming-Yuan and Kuai, Shu-Guang},
  doi          = {10.1038/s42256-022-00542-z},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {1040-1052},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Human-behaviour-based social locomotion model improves the humanization of social robots},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Image prediction of disease progression for osteoarthritis
by style-based manifold extrapolation. <em>NATMI</em>, <em>4</em>(11),
1029–1039. (<a
href="https://doi.org/10.1038/s42256-022-00560-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Disease-modifying management aims to prevent deterioration and progression of the disease, and not just to relieve symptoms. We present a solution for the management by a methodology that allows the prediction of progression risk and morphology in individuals using a latent extrapolation approach. To this end, we combined a regularized generative adversarial network and a latent nearest neighbour algorithm for joint optimization to generate plausible images of future time points. We evaluated our method on osteoarthritis data from a multicenter longitudinal study (the Osteoarthritis Initiative). With presymptomatic baseline data, our model is generative and considerably outperforms the end-to-end learning model in discriminating the progressive cohort. Two experiments were performed with seven radiologists. When no synthetic follow-up radiographs were provided, our model performed better than all seven radiologists. In cases in which the synthetic follow-ups generated by our model were made available to the radiologist for diagnosis support, the specificity and sensitivity of all readers in discriminating progressors increased from 72.3\% to 88.6\% and from 42.1\% to 51.6\%, respectively. Our results open up a new possibility of using model-based morphology and risk prediction to make predictions about disease occurrence, as demonstrated by the example of osteoarthritis. Predicting disease progression is an important medical problem, but it can be challenging for end-to-end machine learning approaches. Han and colleagues demonstrate that generative models can work together with medical experts to jointly predict the progression of a disease, osteoarthritis.},
  archive      = {J_NATMI},
  author       = {Han, Tianyu and Kather, Jakob Nikolas and Pedersoli, Federico and Zimmermann, Markus and Keil, Sebastian and Schulze-Hagen, Maximilian and Terwoelbeck, Marc and Isfort, Peter and Haarburger, Christoph and Kiessling, Fabian and Kuhl, Christiane and Schulz, Volkmar and Nebelung, Sven and Truhn, Daniel},
  doi          = {10.1038/s42256-022-00560-x},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {1029-1039},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Image prediction of disease progression for osteoarthritis by style-based manifold extrapolation},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting functional effect of missense variants using
graph attention neural networks. <em>NATMI</em>, <em>4</em>(11),
1017–1028. (<a
href="https://doi.org/10.1038/s42256-022-00561-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate prediction of damaging missense variants is critically important for interpreting a genome sequence. Although many methods have been developed, their performance has been limited. Recent advances in machine learning and the availability of large-scale population genomic sequencing data provide new opportunities to considerably improve computational predictions. Here we describe the graphical missense variant pathogenicity predictor (gMVP), a new method based on graph attention neural networks. Its main component is a graph with nodes that capture predictive features of amino acids and edges weighted by co-evolution strength, enabling effective pooling of information from the local protein context and functionally correlated distal positions. Evaluation of deep mutational scan data shows that gMVP outperforms other published methods in identifying damaging variants in TP53, PTEN, BRCA1 and MSH2. Furthermore, it achieves the best separation of de novo missense variants in neurodevelopmental disorder cases from those in controls. Finally, the model supports transfer learning to optimize gain- and loss-of-function predictions in sodium and calcium channels. In summary, we demonstrate that gMVP can improve interpretation of missense variants in clinical testing and genetic studies. Computational methods are important for interpreting missense variants in genetic studies and clinical testing. Zhang and colleagues develop a method based on graph attention neural networks to predict pathogenic missense variants. The method pools information from functionally correlated positions and can improve the interpretation of missense variants.},
  archive      = {J_NATMI},
  author       = {Zhang, Haicang and Xu, Michelle S. and Fan, Xiao and Chung, Wendy K. and Shen, Yufeng},
  doi          = {10.1038/s42256-022-00561-w},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {1017-1028},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting functional effect of missense variants using graph attention neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accurate prediction of molecular properties and drug targets
using a self-supervised image representation learning framework.
<em>NATMI</em>, <em>4</em>(11), 1004–1016. (<a
href="https://doi.org/10.1038/s42256-022-00557-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The clinical efficacy and safety of a drug is determined by its molecular properties and targets in humans. However, proteome-wide evaluation of all compounds in humans, or even animal models, is challenging. In this study, we present an unsupervised pretraining deep learning framework, named ImageMol, pretrained on 10 million unlabelled drug-like, bioactive molecules, to predict molecular targets of candidate compounds. The ImageMol framework is designed to pretrain chemical representations from unlabelled molecular images on the basis of local and global structural characteristics of molecules from pixels. We demonstrate high performance of ImageMol in evaluation of molecular properties (that is, the drug’s metabolism, brain penetration and toxicity) and molecular target profiles (that is, beta-secretase enzyme and kinases) across 51 benchmark datasets. ImageMol shows high accuracy in identifying anti-SARS-CoV-2 molecules across 13 high-throughput experimental datasets from the National Center for Advancing Translational Sciences. Via ImageMol, we identified candidate clinical 3C-like protease inhibitors for potential treatment of COVID-19. Predicting the properties of a molecule from its structure with high accuracy is a crucial problem in digital drug design. Instead of sequence features, Zeng and colleagues use an image representation of a large collection of bioactive molecules to pretrain a model that can be fine-tuned on specific property prediction tasks.},
  archive      = {J_NATMI},
  author       = {Zeng, Xiangxiang and Xiang, Hongxin and Yu, Linhui and Wang, Jianmin and Li, Kenli and Nussinov, Ruth and Cheng, Feixiong},
  doi          = {10.1038/s42256-022-00557-6},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {1004-1016},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accurate prediction of molecular properties and drug targets using a self-supervised image representation learning framework},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Closed-form continuous-time neural networks.
<em>NATMI</em>, <em>4</em>(11), 992–1003. (<a
href="https://doi.org/10.1038/s42256-022-00556-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Continuous-time neural networks are a class of machine learning systems that can tackle representation learning on spatiotemporal decision-making tasks. These models are typically represented by continuous differential equations. However, their expressive power when they are deployed on computers is bottlenecked by numerical differential equation solvers. This limitation has notably slowed down the scaling and understanding of numerous natural physical phenomena such as the dynamics of nervous systems. Ideally, we would circumvent this bottleneck by solving the given dynamical system in closed form. This is known to be intractable in general. Here, we show that it is possible to closely approximate the interaction between neurons and synapses—the building blocks of natural and artificial neural networks—constructed by liquid time-constant networks efficiently in closed form. To this end, we compute a tightly bounded approximation of the solution of an integral appearing in liquid time-constant dynamics that has had no known closed-form solution so far. This closed-form solution impacts the design of continuous-time and continuous-depth neural models. For instance, since time appears explicitly in closed form, the formulation relaxes the need for complex numerical solvers. Consequently, we obtain models that are between one and five orders of magnitude faster in training and inference compared with differential equation-based counterparts. More importantly, in contrast to ordinary differential equation-based continuous networks, closed-form networks can scale remarkably well compared with other deep learning instances. Lastly, as these models are derived from liquid networks, they show good performance in time-series modelling compared with advanced recurrent neural network models. Physical dynamical processes can be modelled with differential equations that may be solved with numerical approaches, but this is computationally costly as the processes grow in complexity. In a new approach, dynamical processes are modelled with closed-form continuous-depth artificial neural networks. Improved efficiency in training and inference is demonstrated on various sequence modelling tasks including human action recognition and steering in autonomous driving.},
  archive      = {J_NATMI},
  author       = {Hasani, Ramin and Lechner, Mathias and Amini, Alexander and Liebenwein, Lucas and Ray, Aaron and Tschaikowski, Max and Teschl, Gerald and Rus, Daniela},
  doi          = {10.1038/s42256-022-00556-7},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {992-1003},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Closed-form continuous-time neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhanced spatio-temporal electric load forecasts using less
data with active deep learning. <em>NATMI</em>, <em>4</em>(11), 977–991.
(<a href="https://doi.org/10.1038/s42256-022-00552-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An effective way to mitigate climate change is to electrify most of our energy demand and supply the necessary electricity from renewable wind and solar power plants. Spatio-temporal predictions of electric load become increasingly important for planning this transition, while deep learning prediction models provide increasingly accurate predictions for it. The data that are used for training deep learning models, however, are usually collected at random using a passive learning approach. This naturally results in a large demand for data and associated costs for sensors such as smart meters, posing a large barrier for electric utilities when decarbonizing their grids. Here we investigate whether electric utilities can use active learning to collect a more informative subset of data by leveraging additional computation for better distributing smart meters. We predict ground-truth electric load profiles for single buildings using only remotely sensed data from aerial imagery of these buildings and meteorological conditions in the area of these buildings at different times. We find that active learning can enable 26–81\% more accurate predictions using 29–46\% less data at the price of 4–11 times more computation compared with passive learning. To enable electrification of cities, thereby decarbonizing energy grids, it is essential to predict electricity consumption with high spatio-temporal accuracy. To reduce the need for large amounts of training data, an active deep learning approach is developed to make accurate forecasts of electric load profiles at the scale of single buildings.},
  archive      = {J_NATMI},
  author       = {Aryandoust, Arsam and Patt, Anthony and Pfenninger, Stefan},
  doi          = {10.1038/s42256-022-00552-x},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {977-991},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Enhanced spatio-temporal electric load forecasts using less data with active deep learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting unseen antibodies’ neutralizability via adaptive
graph neural networks. <em>NATMI</em>, <em>4</em>(11), 964–976. (<a
href="https://doi.org/10.1038/s42256-022-00553-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most natural and synthetic antibodies are ‘unseen’. That is, the demonstration of their neutralization effects with any antigen requires laborious and costly wet-lab experiments. The existing methods that learn antibody representations from known antibody–antigen interactions are unsuitable for unseen antibodies owing to the absence of interaction instances. The DeepAAI method proposed herein learns unseen antibody representations by constructing two adaptive relation graphs among antibodies and antigens and applying Laplacian smoothing between unseen and seen antibodies’ representations. Rather than using static protein descriptors, DeepAAI learns representations and relation graphs ‘dynamically’, optimized towards the downstream tasks of neutralization prediction and 50\% inhibition concentration estimation. The performance of DeepAAI is demonstrated on human immunodeficiency virus, severe acute respiratory syndrome coronavirus 2, influenza and dengue. Moreover, the relation graphs have rich interpretability. The antibody relation graph implies similarity in antibody neutralization reactions, and the antigen relation graph indicates the relation among a virus’s different variants. We accordingly recommend probable broad-spectrum antibodies against new variants of these viruses. The effects of novel antibodies are hard to predict owing to the complex interactions between antibodies and antigens. Zhang and colleagues use a graph-based method to learn a dynamic representation that allows for predictions of neutralization activity and demonstrate the method by recommending probable antibodies for human immunodeficiency virus, severe acute respiratory syndrome coronavirus 2, influenza and dengue.},
  archive      = {J_NATMI},
  author       = {Zhang, Jie and Du, Yishan and Zhou, Pengfei and Ding, Jinru and Xia, Shuai and Wang, Qian and Chen, Feiyang and Zhou, Mu and Zhang, Xuemei and Wang, Weifeng and Wu, Hongyan and Lu, Lu and Zhang, Shaoting},
  doi          = {10.1038/s42256-022-00553-w},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {964-976},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting unseen antibodies’ neutralizability via adaptive graph neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A fast blind zero-shot denoiser. <em>NATMI</em>,
<em>4</em>(11), 953–963. (<a
href="https://doi.org/10.1038/s42256-022-00547-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Image noise is a common problem in light microscopy. This is particularly true in real-time live-cell imaging applications in which long-term cell viability necessitates low-light conditions. Modern denoisers are typically trained on a representative dataset, sometimes consisting of just unpaired noisy shots. However, when data are acquired in real time to track dynamic cellular processes, it is not always practical nor economical to generate these training sets. Recently, denoisers have emerged that allow us to denoise single images without a training set or knowledge about the underlying noise. But such methods are currently too slow to be integrated into imaging pipelines that require rapid, real-time hardware feedback. Here we present Noise2Fast, which can overcome these limitations. Noise2Fast uses a novel downsampling technique we refer to as ‘chequerboard downsampling’. This allows us to train on a discrete 4-image training set, while convergence can be monitored using the original noisy image. We show that Noise2Fast is faster than all similar methods with only a small drop in accuracy compared to the gold standard. We integrate Noise2Fast into real-time multi-modal imaging applications and demonstrate its broad applicability to diverse imaging and analysis pipelines. Image noise is a common problem in light microscopy, and denoising is a key step in microscopic imaging pipelines. Lequyer et al. propose a self-supervised denoising method and apply it to diverse imaging and analysis pipelines.},
  archive      = {J_NATMI},
  author       = {Lequyer, Jason and Philip, Reuben and Sharma, Amit and Hsu, Wen-Hsin and Pelletier, Laurence},
  doi          = {10.1038/s42256-022-00547-8},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {953-963},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A fast blind zero-shot denoiser},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A multi-use deep learning method for CITE-seq and
single-cell RNA-seq data integration with cell surface protein
prediction and imputation. <em>NATMI</em>, <em>4</em>(11), 940–952. (<a
href="https://doi.org/10.1038/s42256-022-00545-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CITE-seq, a single-cell multi-omics technology that measures RNA and protein expression simultaneously in single cells, has been widely applied in biomedical research, especially in immune-related disorders and other diseases such as influenza and COVID-19. Despite the proliferation of CITE-seq, it is still costly to generate such data. Although data integration can increase information content, this raises computational challenges. First, combining multiple datasets is prone to batch effects that need to be addressed. Second, it is difficult to combine multiple CITE-seq datasets because the protein panels in different datasets may only partially overlap. Integrating multiple CITE-seq and single-cell RNA sequencing (scRNA-seq) datasets is important because this allows the utilization of as many data as possible to uncover cell population heterogeneity. To overcome these challenges, we present sciPENN, a multi-use deep learning approach that supports CITE-seq and scRNA-seq data integration, protein expression prediction for scRNA-seq, protein expression imputation for CITE-seq, quantification of prediction and imputation uncertainty, and cell type label transfer from CITE-seq to scRNA-seq. Comprehensive evaluations spanning multiple datasets demonstrate that sciPENN outperforms other current state-of-the-art methods. Single-cell multi-omics technologies have increased dramatically in biomedical research. Lakkis et al. develop a deep learning method to address computational challenges in CITE-seq and single-cell RNA-seq datasets.},
  archive      = {J_NATMI},
  author       = {Lakkis, Justin and Schroeder, Amelia and Su, Kenong and Lee, Michelle Y. Y. and Bashore, Alexander C. and Reilly, Muredach P. and Li, Mingyao},
  doi          = {10.1038/s42256-022-00545-w},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {940-952},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A multi-use deep learning method for CITE-seq and single-cell RNA-seq data integration with cell surface protein prediction and imputation},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Visual speech recognition for multiple languages in the
wild. <em>NATMI</em>, <em>4</em>(11), 930–939. (<a
href="https://doi.org/10.1038/s42256-022-00550-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Visual speech recognition (VSR) aims to recognize the content of speech based on lip movements, without relying on the audio stream. Advances in deep learning and the availability of large audio-visual datasets have led to the development of much more accurate and robust VSR models than ever before. However, these advances are usually due to the larger training sets rather than the model design. Here we demonstrate that designing better models is equally as important as using larger training sets. We propose the addition of prediction-based auxiliary tasks to a VSR model, and highlight the importance of hyperparameter optimization and appropriate data augmentations. We show that such a model works for different languages and outperforms all previous methods trained on publicly available datasets by a large margin. It even outperforms models that were trained on non-publicly available datasets containing up to to 21 times more data. We show, furthermore, that using additional training data, even in other languages or with automatically generated transcriptions, results in further improvement. Recognition of speech from lip movements is still a challenging problem and much effort is concentrated on the English language. Ma et al. have used auxiliary tasks to train a model such that it works for a range of different languages, including Mandarin, Spanish, Italian, French and Portuguese.},
  archive      = {J_NATMI},
  author       = {Ma, Pingchuan and Petridis, Stavros and Pantic, Maja},
  doi          = {10.1038/s42256-022-00550-z},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {930-939},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Visual speech recognition for multiple languages in the wild},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Development of metaverse for intelligent healthcare.
<em>NATMI</em>, <em>4</em>(11), 922–929. (<a
href="https://doi.org/10.1038/s42256-022-00549-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The metaverse integrates physical and virtual realities, enabling humans and their avatars to interact in an environment supported by technologies such as high-speed internet, virtual reality, augmented reality, mixed and extended reality, blockchain, digital twins and artificial intelligence (AI), all enriched by effectively unlimited data. The metaverse recently emerged as social media and entertainment platforms, but extension to healthcare could have a profound impact on clinical practice and human health. As a group of academic, industrial, clinical and regulatory researchers, we identify unique opportunities for metaverse approaches in the healthcare domain. A metaverse of ‘medical technology and AI’ (MeTAI) can facilitate the development, prototyping, evaluation, regulation, translation and refinement of AI-based medical practice, especially medical imaging-guided diagnosis and therapy. Here, we present metaverse use cases, including virtual comparative scanning, raw data sharing, augmented regulatory science and metaversed medical intervention. We discuss relevant issues on the ecosystem of the MeTAI metaverse including privacy, security and disparity. We also identify specific action items for coordinated efforts to build the MeTAI metaverse for improved healthcare quality, accessibility, cost-effectiveness and patient satisfaction. The metaverse is gaining prominence in industry, academia and social media. Wang and colleagues envision a medical technology and AI ecosystem, and present this perspective on the future of healthcare in the metaverse.},
  archive      = {J_NATMI},
  author       = {Wang, Ge and Badal, Andreu and Jia, Xun and Maltz, Jonathan S. and Mueller, Klaus and Myers, Kyle J. and Niu, Chuang and Vannier, Michael and Yan, Pingkun and Yu, Zhou and Zeng, Rongping},
  doi          = {10.1038/s42256-022-00549-6},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {922-929},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Development of metaverse for intelligent healthcare},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Developing robust benchmarks for driving forward AI
innovation in healthcare. <em>NATMI</em>, <em>4</em>(11), 916–921. (<a
href="https://doi.org/10.1038/s42256-022-00559-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning technologies have seen increased application to the healthcare domain. The main drivers are openly available healthcare datasets, and a general interest from the community to use its powers for knowledge discovery and technological advancements in this more conservative field. However, with this additional volume comes a range of questions and concerns — are the obtained results meaningful and conclusions accurate; how do we know we have improved state of the art; is the clinical problem well defined and does the model address it? We reflect on key aspects in the end-to-end pipeline that we believe suffer the most in this space, and suggest some good practices to avoid reproducing these issues. Finding good benchmarks is an important and pervasive problem in machine learning for healthcare. This Perspective highlights key aspects that require scrutiny in the whole process of benchmark generation and use, including problem formulation, creation of datasets, development of a suite of machine learning models and evaluation of these models.},
  archive      = {J_NATMI},
  author       = {Mincu, Diana and Roy, Subhrajit},
  doi          = {10.1038/s42256-022-00559-4},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {916-921},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Developing robust benchmarks for driving forward AI innovation in healthcare},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Developing moral AI to support decision-making about
antimicrobial use. <em>NATMI</em>, <em>4</em>(11), 912–915. (<a
href="https://doi.org/10.1038/s42256-022-00558-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of decision-support systems based on artificial intelligence approaches in antimicrobial prescribing raises important moral questions. Adopting ethical frameworks alongside such systems can aid the consideration of infection-specific complexities and support moral decision-making to tackle antimicrobial resistance.},
  archive      = {J_NATMI},
  author       = {Bolton, William J. and Badea, Cosmin and Georgiou, Pantelis and Holmes, Alison and Rawson, Timothy M.},
  doi          = {10.1038/s42256-022-00558-5},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {912-915},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Developing moral AI to support decision-making about antimicrobial use},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Federated learning and indigenous genomic data sovereignty.
<em>NATMI</em>, <em>4</em>(11), 909–911. (<a
href="https://doi.org/10.1038/s42256-022-00551-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Indigenous peoples are under-represented in genomic datasets, which can lead to limited accuracy and utility of machine learning models in precision health. While open data sharing undermines rights of Indigenous communities to govern data decisions, federated learning may facilitate secure and community-consented data sharing.},
  archive      = {J_NATMI},
  author       = {Boscarino, Nima and Cartwright, Reed A. and Fox, Keolu and Tsosie, Krystal S.},
  doi          = {10.1038/s42256-022-00551-y},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {909-911},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Federated learning and indigenous genomic data sovereignty},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Modelling human tool use in robots. <em>NATMI</em>,
<em>4</em>(11), 907–908. (<a
href="https://doi.org/10.1038/s42256-022-00562-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tool use is one of the defining traits of human cognition that sets our species apart from other animals. A novel computational framework may enable robots to use tools as intelligently as humans do.},
  archive      = {J_NATMI},
  author       = {Jamone, Lorenzo},
  doi          = {10.1038/s42256-022-00562-9},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {907-908},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Modelling human tool use in robots},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Data sovereignty in genomics and medical research.
<em>NATMI</em>, <em>4</em>(11), 905–906. (<a
href="https://doi.org/10.1038/s42256-022-00578-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AI promises to bring many benefits to healthcare and research, but mistrust has built up owing to many instances of harm to under-represented communities. To amend this, participatory approaches can directly involve communities in AI research that will impact them. An important element of such approaches is ensuring that communities can take control over their own data and how they are shared.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00578-1},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {905-906},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Data sovereignty in genomics and medical research},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Author correction: Advances, challenges and opportunities
in creating data for trustworthy AI. <em>NATMI</em>, <em>4</em>(10),
904. (<a href="https://doi.org/10.1038/s42256-022-00548-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Liang, Weixin and Tadesse, Girmaw Abebe and Ho, Daniel and Fei-Fei, L. and Zaharia, Matei and Zhang, Ce and Zou, James},
  doi          = {10.1038/s42256-022-00548-7},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {904},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Advances, challenges and opportunities in creating data for trustworthy AI},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Encoding of tactile information in hand via skin-integrated
wireless haptic interface. <em>NATMI</em>, <em>4</em>(10), 893–903. (<a
href="https://doi.org/10.1038/s42256-022-00543-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vivid haptic feedback remains a challenge in truly immersive virtual reality and augmented reality. As the tactile sensitivity among different individuals and different parts of the hand within a person varies widely, a universal method to encode tactile information into faithful feedback in hands according to sensitivity features is urgently needed. In addition, existing haptic interfaces worn on the hand are usually bulky, rigid and tethered by cables, which is a hurdle for accurately and naturally providing haptic feedbacks. Here we report a soft, ultrathin, miniaturized and wireless electrotactile system (WeTac) that delivers current through the hand to induce tactile sensations as the skin-integrated haptic interface. With a relatively high pixel density over the whole hand area, the WeTac can provide tactile stimulation and measure the sensation thresholds of users in a flexible way. By mapping the thresholds for different electrical parameters, personalized threshold data can be acquired to reproduce virtual touching sensations on the hand with optimized stimulation intensity and avoid causing pain. With an accurate control of sensation level, temporal and spatial perception, it allows providing personalized feedback when users interact with virtual objects. This technique is promising for a more vivid touching experience in the virtual world and in human–machine interactions. The haptic interface is an essential part of human–machine interfaces where tactile information is delivered between human and machine. Yao et al. develop a soft, ultrathin, miniaturized and wireless electrotactile system that allows virtual tactile information to be reproduced over the hand.},
  archive      = {J_NATMI},
  author       = {Yao, Kuanming and Zhou, Jingkun and Huang, Qingyun and Wu, Mengge and Yiu, Chun Ki and Li, Jian and Huang, Xingcan and Li, Dengfeng and Su, Jingyou and Hou, Senlin and Liu, Yiming and Huang, Ya and Tian, Ziyan and Li, Jiyu and Li, Hu and Shi, Rui and Zhang, Binbin and Zhu, Jingyi and Wong, Tsz Hung and Jia, Huiling and Gao, Zhan and Gao, Yuyu and Zhou, Yu and Park, Wooyoung and Song, Enming and Han, Mengdi and Zhang, Haixia and Yu, Junsheng and Wang, Lidai and Li, Wen Jung and Yu, Xinge},
  doi          = {10.1038/s42256-022-00543-y},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {893-903},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Encoding of tactile information in hand via skin-integrated wireless haptic interface},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A context-aware deconfounding autoencoder for robust
prediction of personalized clinical drug response from cell-line
compound screening. <em>NATMI</em>, <em>4</em>(10), 879–892. (<a
href="https://doi.org/10.1038/s42256-022-00541-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate and robust prediction of patient-specific responses to a new compound is critical to personalized drug discovery and development. However, patient data are often too scarce to train a generalized machine learning model. Although many methods have been developed to utilize cell-line screens for predicting clinical responses, their performances are unreliable owing to data heterogeneity and distribution shift. Here we have developed a novel context-aware deconfounding autoencoder (CODE-AE) that can extract intrinsic biological signals masked by context-specific patterns and confounding factors. Extensive comparative studies demonstrated that CODE-AE effectively alleviated the out-of-distribution problem for the model generalization and significantly improved accuracy and robustness over state-of-the-art methods in predicting patient-specific clinical drug responses purely from cell-line compound screens. Using CODE-AE, we screened 59 drugs for 9,808 patients with cancer. Our results are consistent with existing clinical observations, suggesting the potential of CODE-AE in developing personalized therapies and drug response biomarkers. Predicting patient-specific clinical drug responses from cell-line screens using machine learning is challenging. He and colleagues develop a deep learning method to predict patient-specific clinical responses from cell-line and other disease models for drug discovery and personalized medicine.},
  archive      = {J_NATMI},
  author       = {He, Di and Liu, Qiao and Wu, You and Xie, Lei},
  doi          = {10.1038/s42256-022-00541-0},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {879-892},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A context-aware deconfounding autoencoder for robust prediction of personalized clinical drug response from cell-line compound screening},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Benchmarking saliency methods for chest x-ray
interpretation. <em>NATMI</em>, <em>4</em>(10), 867–878. (<a
href="https://doi.org/10.1038/s42256-022-00536-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Saliency methods, which produce heat maps that highlight the areas of the medical image that influence model prediction, are often presented to clinicians as an aid in diagnostic decision-making. However, rigorous investigation of the accuracy and reliability of these strategies is necessary before they are integrated into the clinical setting. In this work, we quantitatively evaluate seven saliency methods, including Grad-CAM, across multiple neural network architectures using two evaluation metrics. We establish the first human benchmark for chest X-ray segmentation in a multilabel classification set-up, and examine under what clinical conditions saliency maps might be more prone to failure in localizing important pathologies compared with a human expert benchmark. We find that (1) while Grad-CAM generally localized pathologies better than the other evaluated saliency methods, all seven performed significantly worse compared with the human benchmark, (2) the gap in localization performance between Grad-CAM and the human benchmark was largest for pathologies that were smaller in size and had shapes that were more complex, and (3) model confidence was positively correlated with Grad-CAM localization performance. Our work demonstrates that several important limitations of saliency methods must be addressed before we can rely on them for deep learning explainability in medical imaging. Saliency methods are used to localize areas of medical images that influence machine learning model predictions, but their accuracy and reliability require investigation. Saporta and colleagues evaluate seven saliency methods using different model architectures, and find that saliency maps perform worse than a human radiologist benchmark.},
  archive      = {J_NATMI},
  author       = {Saporta, Adriel and Gui, Xiaotong and Agrawal, Ashwin and Pareek, Anuj and Truong, Steven Q. H. and Nguyen, Chanh D. T. and Ngo, Van-Doan and Seekins, Jayne and Blankenberg, Francis G. and Ng, Andrew Y. and Lungren, Matthew P. and Rajpurkar, Pranav},
  doi          = {10.1038/s42256-022-00536-x},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {867-878},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Benchmarking saliency methods for chest X-ray interpretation},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ScBERT as a large-scale pretrained deep language model for
cell type annotation of single-cell RNA-seq data. <em>NATMI</em>,
<em>4</em>(10), 852–866. (<a
href="https://doi.org/10.1038/s42256-022-00534-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Annotating cell types on the basis of single-cell RNA-seq data is a prerequisite for research on disease progress and tumour microenvironments. Here we show that existing annotation methods typically suffer from a lack of curated marker gene lists, improper handling of batch effects and difficulty in leveraging the latent gene–gene interaction information, impairing their generalization and robustness. We developed a pretrained deep neural network-based model, single-cell bidirectional encoder representations from transformers (scBERT), to overcome the challenges. Following BERT’s approach to pretraining and fine-tuning, scBERT attains a general understanding of gene–gene interactions by being pretrained on huge amounts of unlabelled scRNA-seq data; it is then transferred to the cell type annotation task of unseen and user-specific scRNA-seq data for supervised fine-tuning. Extensive and rigorous benchmark studies validated the superior performance of scBERT on cell type annotation, novel cell type discovery, robustness to batch effects and model interpretability. Cell type annotation is a core task for single cell RNA-sequencing, but current bioinformatic tools struggle with some of the underlying challenges, including high dimensionality, data sparsity, batch effects and a lack of labels. In a self-supervised approach, a transformer model called scBERT is pretrained on millions of unlabelled public single cell RNA-seq data and then fine-tuned with a small number of labelled samples for cell annotation tasks.},
  archive      = {J_NATMI},
  author       = {Yang, Fan and Wang, Wenchuan and Wang, Fang and Fang, Yuan and Tang, Duyu and Huang, Junzhou and Lu, Hui and Yao, Jianhua},
  doi          = {10.1038/s42256-022-00534-z},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {852-866},
  shortjournal = {Nat. Mach. Intell.},
  title        = {ScBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Transformer-based protein generation with regularized latent
space optimization. <em>NATMI</em>, <em>4</em>(10), 840–851. (<a
href="https://doi.org/10.1038/s42256-022-00532-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of powerful natural language models has improved the ability to learn meaningful representations of protein sequences. In addition, advances in high-throughput mutagenesis, directed evolution and next-generation sequencing have allowed for the accumulation of large amounts of labelled fitness data. Leveraging these two trends, we introduce Regularized Latent Space Optimization (ReLSO), a deep transformer-based autoencoder, which features a highly structured latent space that is trained to jointly generate sequences as well as predict fitness. Through regularized prediction heads, ReLSO introduces a powerful protein sequence encoder and a novel approach for efficient fitness landscape traversal. Using ReLSO, we explicitly model the sequence–function landscape of large labelled datasets and generate new molecules by optimizing within the latent space using gradient-based methods. We evaluate this approach on several publicly available protein datasets, including variant sets of anti-ranibizumab and green fluorescent protein. We observe a greater sequence optimization efficiency (increase in fitness per optimization step) using ReLSO compared with other approaches, where ReLSO more robustly generates high-fitness sequences. Furthermore, the attention-based relationships learned by the jointly trained ReLSO models provide a potential avenue towards sequence-level fitness attribution information. The space of possible proteins is vast, and optimizing proteins for specific target properties computationally is an ongoing challenge, even with large amounts of data. Castro and colleagues combine a transformer-based model with regularized prediction heads to form a smooth and pseudoconvex latent space that allows for easier navigation and more efficient optimization of proteins.},
  archive      = {J_NATMI},
  author       = {Castro, Egbert and Godavarthi, Abhinav and Rubinfien, Julian and Givechian, Kevin and Bhaskar, Dhananjay and Krishnaswamy, Smita},
  doi          = {10.1038/s42256-022-00532-1},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {840-851},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Transformer-based protein generation with regularized latent space optimization},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Physically constrained generative adversarial networks for
improving precipitation fields from earth system models. <em>NATMI</em>,
<em>4</em>(10), 828–839. (<a
href="https://doi.org/10.1038/s42256-022-00540-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precipitation results from complex processes across many scales, making its accurate simulation in Earth system models (ESMs) challenging. Existing post-processing methods can improve ESM simulations locally but cannot correct errors in modelled spatial patterns. Here we propose a framework based on physically constrained generative adversarial networks to improve local distributions and spatial structure simultaneously. We apply our approach to the computationally efficient CM2Mc–LPJmL ESM. Our method outperforms existing ones in correcting local distributions and leads to strongly improved spatial patterns, especially regarding the intermittency of daily precipitation. Notably, a double-peaked Intertropical Convergence Zone, a common problem in ESMs, is removed. Enforcing a physical constraint to preserve global precipitation sums, the generative adversarial network can generalize to future climate scenarios unseen during training. Feature attribution shows that the generative adversarial network identifies regions where the ESM exhibits strong biases. Our method constitutes a general framework for correcting ESM variables and enables realistic simulations at a fraction of the computational cost. Earth system models (ESMs) are powerful tools for simulating climate fields, but weather forecasting and in particular precipitation prediction with ESMs are challenging. A generative adversarial network, constrained by the sum of global precipitation, is developed that substantially improves ESM predictions of spatial patterns and intermittency of daily precipitation.},
  archive      = {J_NATMI},
  author       = {Hess, Philipp and Drüke, Markus and Petri, Stefan and Strnad, Felix M. and Boers, Niklas},
  doi          = {10.1038/s42256-022-00540-1},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {828-839},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Physically constrained generative adversarial networks for improving precipitation fields from earth system models},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Forecasting SARS-CoV-2 transmission and clinical risk at
small spatial scales by the application of machine learning
architectures to syndromic surveillance data. <em>NATMI</em>,
<em>4</em>(10), 814–827. (<a
href="https://doi.org/10.1038/s42256-022-00538-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Timely and well-informed syndromic surveillance is essential for effective public health policy. The monitoring of traditional epidemiological indicators can be lagged and misleading, which hampers efforts to identify hotspot locations. The increasing predominance of digitalized healthcare-seeking behaviour necessitates that it is fully exploited for the public benefit of effective pandemic management. Using the highest-resolution spatial data for Google Trends relative search volumes, Google mobility, telecoms mobility, National Health Service Pathways calls and website testing journeys, we have developed a machine learning early indicator modelling approach of SARS-CoV-2 transmission and clinical risk at small geographic scales. We trained shallow learning algorithms as the baseline against a geospatial neural network architecture that we termed the spatio-integrated long short-term memory (SI-LSTM) algorithm. The SI-LSTM algorithm was able to—for the assessed temporal periods—accurately identify hotspot locations over time horizons of a month or more with an accuracy in excess of 99\%, and an improved performance of up to 15\% against the shallow learning algorithms. Furthermore, in public health operational use, this model highlighted the localized exponential growth of the Alpha variant in late 2020, the Delta variant in April 2021 and the Omicron variant in November 2021 within the United Kingdom prior to their spatial dispersion and growth being confirmed by clinical data. Identifying epidemic hotspots in a timely way with syndromic surveillance can provide highly valuable information for public health policy. A machine learning early indicator model that uses highly granular data from digitalized healthcare-seeking behaviour, including from Google Trends and National Health Service Pathways calls, can identify SARS-CoV-2 risk at small geographic scales. The model can retrospectively identify hotspots in the United Kingdom for various variants in 2020 and 2021 before the wider spread and growth of these variants being confirmed by clinical data.},
  archive      = {J_NATMI},
  author       = {Ward, Thomas and Johnsen, Alexander and Ng, Stanley and Chollet, François},
  doi          = {10.1038/s42256-022-00538-9},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {814-827},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Forecasting SARS-CoV-2 transmission and clinical risk at small spatial scales by the application of machine learning architectures to syndromic surveillance data},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reusability report: Comparing gradient descent and monte
carlo tree search optimization of quantum annealing schedules.
<em>NATMI</em>, <em>4</em>(10), 810–813. (<a
href="https://doi.org/10.1038/s42256-022-00535-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Wauters, Matteo M. and van Nieuwenburg, Evert},
  doi          = {10.1038/s42256-022-00535-y},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {810-813},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Comparing gradient descent and monte carlo tree search optimization of quantum annealing schedules},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). AI model transferability in healthcare: A sociotechnical
perspective. <em>NATMI</em>, <em>4</em>(10), 807–809. (<a
href="https://doi.org/10.1038/s42256-022-00544-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To deliver value in healthcare, artificial intelligence and machine learning models must be integrated not only into technology platforms but also into local human and organizational ecosystems and workflows. To realize the promised benefits of applying these models at scale, a roadmap of the challenges and potential solutions to sociotechnical transferability is needed.},
  archive      = {J_NATMI},
  author       = {Wiesenfeld, Batia Mishan and Aphinyanaphongs, Yin and Nov, Oded},
  doi          = {10.1038/s42256-022-00544-x},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {807-809},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI model transferability in healthcare: A sociotechnical perspective},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Avoid diluting democracy by algorithms. <em>NATMI</em>,
<em>4</em>(10), 804–806. (<a
href="https://doi.org/10.1038/s42256-022-00537-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is a tendency among AI researchers to use the concepts of democracy and democratization in ways that are only loosely connected to their political and historical meanings. We argue that it is important to take the concept more seriously in AI research by engaging with political philosophy.},
  archive      = {J_NATMI},
  author       = {Sætra, Henrik Skaug and Borgebund, Harald and Coeckelbergh, Mark},
  doi          = {10.1038/s42256-022-00537-w},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {804-806},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Avoid diluting democracy by algorithms},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A new regulatory framework for algorithm-powered
recommendation services in china. <em>NATMI</em>, <em>4</em>(10),
802–803. (<a href="https://doi.org/10.1038/s42256-022-00546-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Yang, Fei and Yao, Yu},
  doi          = {10.1038/s42256-022-00546-9},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {802-803},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A new regulatory framework for algorithm-powered recommendation services in china},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Revisiting code reusability. <em>NATMI</em>, <em>4</em>(10),
801. (<a href="https://doi.org/10.1038/s42256-022-00554-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduced reusability reports, an article type to highlight code reusability, almost two years ago. On the basis of the results and positive feedback from authors and referees, we remain enthusiastic about the format.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00554-9},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {801},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Revisiting code reusability},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Interpretable meta-score for model performance.
<em>NATMI</em>, <em>4</em>(9), 792–800. (<a
href="https://doi.org/10.1038/s42256-022-00531-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Benchmarks are an integral part of machine learning development. However, the most common benchmarks share several limitations. For example, the difference in performance between two models has no probabilistic interpretation, it makes no sense to compare such differences between data sets and there is no reference point that indicates a significant performance improvement. Here we introduce an Elo-based predictive power meta-score that is built on other performance measures and allows for interpretable comparisons of models. Differences between this score have a probabilistic interpretation and can be compared directly between data sets. Furthermore, this meta-score allows for an assessment of ranking fitness. We prove the properties of the Elo-based predictive power meta-score and support them with empirical results on a large-scale benchmark of 30 classification data sets. Additionally, we propose a unified benchmark ontology that provides a uniform description of benchmarks. The performance of machine learning models is usually compared via the mean value of a selected performance measure such as the area under the receiver operating characteristic curve on a specific benchmark data set. However, this measure, its mean value or even relative differences between models do not provide a good prediction of whether the results can translate to other data sets. Gosiewska and colleagues present here a comparison based on Elo ranking, which offers a probabilistic interpretation of how much better one model is than another.},
  archive      = {J_NATMI},
  author       = {Gosiewska, Alicja and Woźnica, Katarzyna and Biecek, Przemysław},
  doi          = {10.1038/s42256-022-00531-2},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {792-800},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Interpretable meta-score for model performance},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Recovery of continuous 3D refractive index maps from
discrete intensity-only measurements using neural fields.
<em>NATMI</em>, <em>4</em>(9), 781–791. (<a
href="https://doi.org/10.1038/s42256-022-00530-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intensity diffraction tomography (IDT) refers to a class of optical microscopy techniques for imaging the three-dimensional refractive index (RI) distribution of a sample from a set of two-dimensional intensity-only measurements. The reconstruction of artefact-free RI maps is a fundamental challenge in IDT due to the loss of phase information and the missing-cone problem. Neural fields has recently emerged as a new deep learning approach for learning continuous representations of physical fields. The technique uses a coordinate-based neural network to represent the field by mapping the spatial coordinates to the corresponding physical quantities, in our case the complex-valued refractive index values. We present Deep Continuous Artefact-free RI Field (DeCAF) as a neural-fields-based IDT method that can learn a high-quality continuous representation of a RI volume from its intensity-only and limited-angle measurements. The representation in DeCAF is learned directly from the measurements of the test sample by using the IDT forward model without any ground-truth RI maps. We qualitatively and quantitatively evaluate DeCAF on the simulated and experimental biological samples. Our results show that DeCAF can generate high-contrast and artefact-free RI maps and lead to an up to 2.1-fold reduction in the mean squared error over existing methods. Producing high-quality 3D refractive index maps from 2D intensity-only measurements is a long-standing objective in computational microscopy, with many applications involving the visualization of cellular and subcellular structures. A new method can reconstruct high-contrast and artefact-free images by employing the neural fields technique, which can learn a continuous 3D representation using a neural network that maps spatial coordinates to the refractive index values.},
  archive      = {J_NATMI},
  author       = {Liu, Renhao and Sun, Yu and Zhu, Jiabei and Tian, Lei and Kamilov, Ulugbek S.},
  doi          = {10.1038/s42256-022-00530-3},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {781-791},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Recovery of continuous 3D refractive index maps from discrete intensity-only measurements using neural fields},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A generalized-template-based graph neural network for
accurate organic reactivity prediction. <em>NATMI</em>, <em>4</em>(9),
772–780. (<a href="https://doi.org/10.1038/s42256-022-00526-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The reliable prediction of chemical reactivity remains in the realm of knowledgeable synthetic chemists. Automating this process by using artificial intelligence could accelerate synthesis design in future digital laboratories. While several machine learning approaches have demonstrated promising results, most current models deviate from how human chemists analyse and predict reactions based on electronic changes. Here, we propose a chemistry-motivated graph neural network called LocalTransform, which learns organic reactivity based on generalized reaction templates to describe the net changes in electron configuration between the reactants and products. The proposed concept dramatically reduces the number of reaction rules and exhibits state-of-the-art product prediction accuracy. In addition to the built-in interpretability of the generalized reaction templates, the high score–accuracy correlation of the model allows users to assess the uncertainty of the machine predictions. To understand reactions in organic chemistry, ideally simple rules would help us predict the outcome of new reactions, but in reality such rules are not easily identified. Chen and Jung extract generalized reaction templates from data and show that they can be used in graph neural networks to predict the outcome of reactions and, despite simplification, still represent a high percentage of existing reactions.},
  archive      = {J_NATMI},
  author       = {Chen, Shuan and Jung, Yousung},
  doi          = {10.1038/s42256-022-00526-z},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {772-780},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A generalized-template-based graph neural network for accurate organic reactivity prediction},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep neural networks with controlled variable selection for
the identification of putative causal genetic variants. <em>NATMI</em>,
<em>4</em>(9), 761–771. (<a
href="https://doi.org/10.1038/s42256-022-00525-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep neural networks (DNNs) have been successfully utilized in many scientific problems for their high prediction accuracy, but their application to genetic studies remains challenging due to their poor interpretability. Here we consider the problem of scalable, robust variable selection in DNNs for the identification of putative causal genetic variants in genome sequencing studies. We identified a pronounced randomness in feature selection in DNNs due to its stochastic nature, which may hinder interpretability and give rise to misleading results. We propose an interpretable neural network model, stabilized using ensembling, with controlled variable selection for genetic studies. The merit of the proposed method includes: flexible modelling of the nonlinear effect of genetic variants to improve statistical power; multiple knockoffs in the input layer to rigorously control the false discovery rate; hierarchical layers to substantially reduce the number of weight parameters and activations, and improve computational efficiency; and stabilized feature selection to reduce the randomness in identified signals. We evaluate the proposed method in extensive simulation studies and apply it to the analysis of Alzheimer’s disease genetics. We show that the proposed method, when compared with conventional linear and nonlinear methods, can lead to substantially more discoveries. Genomic sequencing offers a wealth of information that could be analysed with deep neural networks. But despite good performance, neural networks can choose random features for their prediction. Kassani et al. present a method to stabilize the features selected by a DNN to make it more interpretable.},
  archive      = {J_NATMI},
  author       = {Kassani, Peyman H. and Lu, Fred and Le Guen, Yann and Belloy, Michael E. and He, Zihuai},
  doi          = {10.1038/s42256-022-00525-0},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {761-771},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep neural networks with controlled variable selection for the identification of putative causal genetic variants},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep learning-based robust positioning for all-weather
autonomous driving. <em>NATMI</em>, <em>4</em>(9), 749–760. (<a
href="https://doi.org/10.1038/s42256-022-00520-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interest in autonomous vehicles (AVs) is growing at a rapid pace due to increased convenience, safety benefits and potential environmental gains. Although several leading AV companies predicted that AVs would be on the road by 2020, they are still limited to relatively small-scale trials. The ability to know their precise location on the map is a challenging prerequisite for safe and reliable AVs due to sensor imperfections under adverse environmental and weather conditions, posing a formidable obstacle to their widespread use. Here we propose a deep learning-based self-supervised approach for ego-motion estimation that is a robust and complementary localization solution under inclement weather conditions. The proposed approach is a geometry-aware method that attentively fuses the rich representation capability of visual sensors and the weather-immune features provided by radars using an attention-based learning technique. Our method predicts reliability masks for the sensor measurements, eliminating the deficiencies in the multimodal data. In various experiments we demonstrate the robust all-weather performance and effective cross-domain generalizability under harsh weather conditions such as rain, fog and snow, as well as day and night conditions. Furthermore, we employ a game-theoretic approach to analyse the interpretability of the model predictions, illustrating the independent and uncorrelated failure modes of the multimodal system. We anticipate our work will bring AVs one step closer to safe and reliable all-weather autonomous driving. Changing weather conditions pose a challenge for autonomous vehicles. Almalioglu and colleagues use a geometry-aware learning technique that fuses visual, lidar and radar information, such that the benefits of each can be used under different weather conditions.},
  archive      = {J_NATMI},
  author       = {Almalioglu, Yasin and Turan, Mehmet and Trigoni, Niki and Markham, Andrew},
  doi          = {10.1038/s42256-022-00520-5},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {749-760},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning-based robust positioning for all-weather autonomous driving},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accelerated rational PROTAC design via deep learning and
molecular simulations. <em>NATMI</em>, <em>4</em>(9), 739–748. (<a
href="https://doi.org/10.1038/s42256-022-00527-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteolysis-targeting chimeras (PROTACs) have emerged as effective tools to selectively degrade disease-related proteins by using the ubiquitin-proteasome system. Developing PROTACs involves extensive tests and trials to explore the vast chemical space. To accelerate this process, we propose a novel deep generative model for the rational design of PROTACs in a low-resource setting, which is then guided to sample PROTACs with optimal pharmacokinetics through deep reinforcement learning. Applying this method to the bromodomain-containing protein 4 target protein, we generated 5,000 compounds that were further filtered through machine learning-based classifiers and physics-driven simulations. As a proof of concept, we identified, synthesized and experimentally tested six candidate bromodomain-containing protein 4-degrading PROTACs, of which three were validated by cell-based assays and western blot analysis. One lead candidate was further tested and demonstrated favourable pharmacokinetics in mice. This combination of deep learning and molecular simulations may facilitate rational PROTAC design and optimization. PROTACs can directly and selectively degrade proteins, which opens promising applications in the design of novel drugs, but designing effective PROTACs is a hard challenge due to the complexity of pharmacokinetics. Zheng et al. use a deep generative model to create likely candidates and screen them further to identify a novel BRD4-degrading PROTAC.},
  archive      = {J_NATMI},
  author       = {Zheng, Shuangjia and Tan, Youhai and Wang, Zhenyu and Li, Chengtao and Zhang, Zhiqing and Sang, Xu and Chen, Hongming and Yang, Yuedong},
  doi          = {10.1038/s42256-022-00527-y},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {739-748},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accelerated rational PROTAC design via deep learning and molecular simulations},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bio-robots step towards brain–body co-adaptation.
<em>NATMI</em>, <em>4</em>(9), 737–738. (<a
href="https://doi.org/10.1038/s42256-022-00528-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In animals, both body and neural control have co-evolved to be adaptable to the environment. While a newborn foal learns quickly how to use its legs, traditional robotic approaches require careful engineering and calibration for stable walking robots. Bio-inspired robotics aims to bridge this gap.},
  archive      = {J_NATMI},
  author       = {Valero-Cuevas, Francisco J. and Erwin, Andrew},
  doi          = {10.1038/s42256-022-00528-x},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {737-738},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Bio-robots step towards brain–body co-adaptation},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Distinguishing two features of accountability for AI
technologies. <em>NATMI</em>, <em>4</em>(9), 734–736. (<a
href="https://doi.org/10.1038/s42256-022-00533-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Policymakers and researchers consistently call for greater human accountability for AI technologies. We should be clear about two distinct features of accountability.},
  archive      = {J_NATMI},
  author       = {Porter, Zoe and Zimmermann, Annette and Morgan, Phillip and McDermid, John and Lawton, Tom and Habli, Ibrahim},
  doi          = {10.1038/s42256-022-00533-0},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {734-736},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Distinguishing two features of accountability for AI technologies},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Collaborative creativity in AI. <em>NATMI</em>,
<em>4</em>(9), 733. (<a
href="https://doi.org/10.1038/s42256-022-00539-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The public release of ‘Stable Diffusion’, a high-quality image generation tool, sets new standards in open-source AI development and raises new questions.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00539-8},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {733},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Collaborative creativity in AI},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Improving de novo molecular design with
curriculum learning. <em>NATMI</em>, <em>4</em>(8), 731. (<a
href="https://doi.org/10.1038/s42256-022-00522-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Guo, Jeff and Fialková, Vendy and Arango, Juan Diego and Margreitter, Christian and Janet, Jon Paul and Papadopoulos, Kostas and Engkvist, Ola and Patronov, Atanas},
  doi          = {10.1038/s42256-022-00522-3},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {731},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Improving de novo molecular design with curriculum learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-objective goal-directed optimization of de novo stable
organic radicals for aqueous redox flow batteries. <em>NATMI</em>,
<em>4</em>(8), 720–730. (<a
href="https://doi.org/10.1038/s42256-022-00506-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in the field of goal-directed molecular optimization offer the promise of finding feasible candidates for even the most challenging molecular design applications. One example of a fundamental design challenge is the search for novel stable radical scaffolds for an aqueous redox flow battery that simultaneously satisfy redox requirements at the anode and cathode, as relatively few stable organic radicals are known to exist. To meet this challenge, we develop a new open-source molecular optimization framework based on AlphaZero coupled with a fast, machine-learning-derived surrogate objective trained with nearly 100,000 quantum chemistry simulations. The objective function comprises two graph neural networks: one that predicts adiabatic oxidation and reduction potentials and a second that predicts electron density and local three-dimensional environment, previously shown to be correlated with radical persistence and stability. With no hard-coded knowledge of organic chemistry, the reinforcement learning agent finds molecule candidates that satisfy a precise combination of redox, stability and synthesizability requirements defined at the quantum chemistry level, many of which have reasonable predicted retrosynthetic pathways. The optimized molecules show that alternative stable radical scaffolds may offer a unique profile of stability and redox potentials to enable low-cost symmetric aqueous redox flow batteries. Finding stable radical compounds for redox flow batteries is a challenging molecular design task. Sowndarya et al. combine an AlphaZero-based framework with a surrogate objective function trained on quantum chemistry simulations to generate suitable radical candidates that are stable. The approach promises to contribute to the development of low-cost, reliable energy storage technologies.},
  archive      = {J_NATMI},
  author       = {S. V., Shree Sowndarya and Law, Jeffrey N. and Tripp, Charles E. and Duplyakin, Dmitry and Skordilis, Erotokritos and Biagioni, David and Paton, Robert S. and St. John, Peter C.},
  doi          = {10.1038/s42256-022-00506-3},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {720-730},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Multi-objective goal-directed optimization of de novo stable organic radicals for aqueous redox flow batteries},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reconstructing kinetic models for dynamical studies of
metabolism using generative adversarial networks. <em>NATMI</em>,
<em>4</em>(8), 710–719. (<a
href="https://doi.org/10.1038/s42256-022-00519-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Kinetic models of metabolism relate metabolic fluxes, metabolite concentrations and enzyme levels through mechanistic relations, rendering them essential for understanding, predicting and optimizing the behaviour of living organisms. However, due to the lack of kinetic data, traditional kinetic modelling often yields only a few or no kinetic models with desirable dynamical properties, making the analysis unreliable and computationally inefficient. We present REKINDLE (Reconstruction of Kinetic Models using Deep Learning), a deep-learning-based framework for efficiently generating kinetic models with dynamic properties matching the ones observed in cells. We showcase REKINDLE’s capabilities to navigate through the physiological states of metabolism using small numbers of data with significantly lower computational requirements. The results show that data-driven neural networks assimilate implicit kinetic knowledge and structure of metabolic networks and generate kinetic models with tailored properties and statistical diversity. We anticipate that our framework will advance our understanding of metabolism and accelerate future research in biotechnology and health. Kinetic models of metabolism capture time-dependent behaviour of cellular states and provide valuable insights into cellular physiology, but, due to the lack of experimental data, traditional kinetic modelling can be unreliable and computationally inefficient. A generative framework based on deep learning called REKINDLE can efficiently parameterize large-scale kinetic models, enabling new opportunities to study cellular metabolic behaviour.},
  archive      = {J_NATMI},
  author       = {Choudhury, Subham and Moret, Michael and Salvy, Pierre and Weilandt, Daniel and Hatzimanikatis, Vassily and Miskovic, Ljubisa},
  doi          = {10.1038/s42256-022-00519-y},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {710-719},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reconstructing kinetic models for dynamical studies of metabolism using generative adversarial networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Contrastive learning enables rapid mapping to multimodal
single-cell atlas of multimillion scale. <em>NATMI</em>, <em>4</em>(8),
696–709. (<a href="https://doi.org/10.1038/s42256-022-00518-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell datasets continue to grow in size, posing computational challenges for dealing with expanded scale, extended modality and inevitable batch effects. Deep learning-based approaches have recently emerged to address these points by deriving nonlinear cell embeddings. Here we present contrastive learning of cell representations, Concerto, which leverages a self-supervised distillation framework to model multimodal single-cell atlases. Simply by discriminating each cell from the others, Concerto can be adapted to various downstream tasks such as automatic cell type classification, data integration and especially reference mapping. Unlike current mainstream packages, Concerto’s contrastive setting well supports operating on all genes to preserve biological variations. Concerto can flexibly generalize to multiomics to obtain unified cell representations. Benchmarking on both simulated and real datasets, Concerto substantially outperforms competing methods. By mapping to a comprehensive reference, Concerto recapitulates differential immune responses and discovers disease-specific cell states in patients with COVID-19. Concerto is easily parallelizable and efficiently scalable to build a 10-million-cell reference within 1.5 h and query 10,000 cells within 8 s. Overall, Concerto will facilitate biomedical research by enabling iteratively constructing single-cell reference atlases and rapidly mapping novel dataset against them to transfer relevant cell annotations. Single-cell datasets continue to grow in size and complexity, calling for computational tools to process and analyse data. Yang et al. present a contrastive learning framework to learn cell representations from single-cell multiomics datasets.},
  archive      = {J_NATMI},
  author       = {Yang, Meng and Yang, Yueyuxiao and Xie, Chenxi and Ni, Ming and Liu, Jian and Yang, Huanming and Mu, Feng and Wang, Jian},
  doi          = {10.1038/s42256-022-00518-z},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {696-709},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Contrastive learning enables rapid mapping to multimodal single-cell atlas of multimillion scale},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Federated disentangled representation learning for
unsupervised brain anomaly detection. <em>NATMI</em>, <em>4</em>(8),
685–695. (<a href="https://doi.org/10.1038/s42256-022-00515-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the advent of deep learning and increasing use of brain MRIs, a great amount of interest has arisen in automated anomaly segmentation to improve clinical workflows; however, it is time-consuming and expensive to curate medical imaging. Moreover, data are often scattered across many institutions, with privacy regulations hampering its use. Here we present FedDis to collaboratively train an unsupervised deep convolutional autoencoder on 1,532 healthy magnetic resonance scans from four different institutions, and evaluate its performance in identifying pathologies such as multiple sclerosis, vascular lesions, and low- and high-grade tumours/glioblastoma on a total of 538 volumes from six different institutions. To mitigate the statistical heterogeneity among different institutions, we disentangle the parameter space into global (shape) and local (appearance). Four institutes jointly train shape parameters to model healthy brain anatomical structures. Every institute trains appearance parameters locally to allow for client-specific personalization of the global domain-invariant features. We have shown that our collaborative approach, FedDis, improves anomaly segmentation results by 99.74\% for multiple sclerosis, 83.33\% for vascular lesions and 40.45\% for tumours over locally trained models without the need for annotations or sharing of private local data. We found out that FedDis is especially beneficial for institutes that share both healthy and anomaly data, improving their local model performance by up to 227\% for multiple sclerosis lesions and 77\% for brain tumours. Federated learning and unsupervised anomaly detection are common techniques in machine learning. The authors combine them, using multicentred datasets and various diseases, to automate the segmentation of brain abnormalities without the need for annotations or sharing private local data.},
  archive      = {J_NATMI},
  author       = {Bercea, Cosmin I. and Wiestler, Benedikt and Rueckert, Daniel and Albarqouni, Shadi},
  doi          = {10.1038/s42256-022-00515-2},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {685-695},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Federated disentangled representation learning for unsupervised brain anomaly detection},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Artificial microtubules for rapid and collective transport
of magnetic microcargoes. <em>NATMI</em>, <em>4</em>(8), 678–684. (<a
href="https://doi.org/10.1038/s42256-022-00510-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed transport of microcargoes is essential for living organisms as well as for applications in microrobotics, nanotechnology and biomedicine. Existing delivery technologies often suffer from low speeds, limited navigation control and dispersal by cardiovascular flows. In cell biology, these issues are largely overcome by cytoskeletal motors that carry vesicles along microtubule highways. Thus inspired, here we developed an artificial microtubule (AMT), a structured microfibre with embedded micromagnets that serve as stepping stones to guide particles rapidly through flow networks. Compared with established techniques, the microcargo travels an order of magnitude faster using the same driving frequency, and dispersal is mitigated by a strong dynamic anchoring effect. Even against strong fluid flows, the large local magnetic-field gradients enable both anchoring and guided propulsion. Finally, we show that AMTs can facilitate the self-assembly of microparticles into active-matter clusters, which then enhance their walking speed by bridging over stepping stones collectively. Hence, we demonstrate a unique strategy for robust delivery inside microvascular networks and for minimally invasive interventions, with non-equilibrium effects that could be equally relevant for enhancing biological transport processes. Targeted drug delivery is an exciting application of nanorobotics, but directing particles in the blood stream to the right location and in sufficient number is challenging. Gu and colleagues have developed a microtubule scaffold with embedded micromagnets that allows cargo, such as drug particles, to be transported in microvascular networks with precision and speed.},
  archive      = {J_NATMI},
  author       = {Gu, Hongri and Hanedan, Emre and Boehler, Quentin and Huang, Tian-Yun and Mathijssen, Arnold J. T. M. and Nelson, Bradley J.},
  doi          = {10.1038/s42256-022-00510-7},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {678-684},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Artificial microtubules for rapid and collective transport of magnetic microcargoes},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Advances, challenges and opportunities in creating data for
trustworthy AI. <em>NATMI</em>, <em>4</em>(8), 669–677. (<a
href="https://doi.org/10.1038/s42256-022-00516-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As artificial intelligence (AI) transitions from research to deployment, creating the appropriate datasets and data pipelines to develop and evaluate AI models is increasingly the biggest challenge. Automated AI model builders that are publicly available can now achieve top performance in many applications. In contrast, the design and sculpting of the data used to develop AI often rely on bespoke manual work, and they critically affect the trustworthiness of the model. This Perspective discusses key considerations for each stage of the data-for-AI pipeline—starting from data design to data sculpting (for example, cleaning, valuation and annotation) and data evaluation—to make AI more reliable. We highlight technical advances that help to make the data-for-AI pipeline more scalable and rigorous. Furthermore, we discuss how recent data regulations and policies can impact AI. It has become rapidly clear in the past few years that the creation, use and maintenance of high-quality annotated datasets for robust and reliable AI applications requires careful attention. This Perspective discusses challenges, considerations and best practices for various stages in the data-to-AI pipeline, to encourage a more data-centric approach.},
  archive      = {J_NATMI},
  author       = {Liang, Weixin and Tadesse, Girmaw Abebe and Ho, Daniel and Fei-Fei, L. and Zaharia, Matei and Zhang, Ce and Zou, James},
  doi          = {10.1038/s42256-022-00516-1},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {669-677},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Advances, challenges and opportunities in creating data for trustworthy AI},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reinforcement learning supercharges redox flow batteries.
<em>NATMI</em>, <em>4</em>(8), 667–668. (<a
href="https://doi.org/10.1038/s42256-022-00523-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Designing viable molecular candidates is pivotal to devising low-cost and sustainable storage systems. A reinforcement learning framework has been developed that can identify stable candidates for redox flow batteries in the large search space of organic radicals.},
  archive      = {J_NATMI},
  author       = {Cao, Yang and Ser, Cher Tian and Skreta, Marta and Jorner, Kjell and Kusanda, Nathanael and Aspuru-Guzik, Alán},
  doi          = {10.1038/s42256-022-00523-2},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {667-668},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reinforcement learning supercharges redox flow batteries},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mining for informative signals in biological sequences.
<em>NATMI</em>, <em>4</em>(8), 665–666. (<a
href="https://doi.org/10.1038/s42256-022-00524-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning models for sequential data can be trained to make accurate predictions from large biological datasets. New tools from computer vision and natural language processing can help us make these models interpretable to biologists.},
  archive      = {J_NATMI},
  author       = {Alaa, Ahmed M.},
  doi          = {10.1038/s42256-022-00524-1},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {665-666},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Mining for informative signals in biological sequences},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Delivering microcargo with artificial microtubules.
<em>NATMI</em>, <em>4</em>(8), 663–664. (<a
href="https://doi.org/10.1038/s42256-022-00521-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed, active transport of cargo is essential for life on all length scales. A new system of artificial microtubules — consisting of a fibre with an embedded periodic array of magnetic inclusions — provides controlled active transport of microcargo by a rotating magnetic field, even under adverse flow conditions.},
  archive      = {J_NATMI},
  author       = {Gompper, Gerhard},
  doi          = {10.1038/s42256-022-00521-4},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {663-664},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Delivering microcargo with artificial microtubules},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Achieving net zero emissions with machine learning: The
challenge ahead. <em>NATMI</em>, <em>4</em>(8), 661–662. (<a
href="https://doi.org/10.1038/s42256-022-00529-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is growing interest in using machine learning to mitigate climate change. But as avoiding catastrophic temperature rises becomes more urgent, action is also needed to understand the environmental impact of machine learning research.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00529-w},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {661-662},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Achieving net zero emissions with machine learning: The challenge ahead},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning plastic matching of robot dynamics in closed-loop
central pattern generators. <em>NATMI</em>, <em>4</em>(7), 652–660. (<a
href="https://doi.org/10.1038/s42256-022-00505-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animals achieve agile locomotion performance with reduced control effort and energy efficiency by leveraging compliance in their muscles and tendons. However, it is not known how biological locomotion controllers learn to leverage the intelligence embodied in their leg mechanics. Here we present a framework to match control patterns and mechanics based on the concept of short-term elasticity and long-term plasticity. Inspired by animals, we design a robot, Morti, with passive elastic legs. The quadruped robot Morti is controlled by a bioinspired closed-loop central pattern generator that is designed to elastically mitigate short-term perturbations using sparse contact feedback. By minimizing the amount of corrective feedback on the long term, Morti learns to match the controller to its mechanics and learns to walk within 1 h. By leveraging the advantages of its mechanics, Morti improves its energy efficiency by 42\% without explicit minimization in the cost function. Using the natural dynamics of a legged robot for locomotion is challenging and can be computationally complex. A newly designed quadruped robot called Morti uses a central pattern generator inside two feedback loops as an adaptive method so that it efficiently uses the passive elasticity of its legs and can learn to walk within 1 h.},
  archive      = {J_NATMI},
  author       = {Ruppert, Felix and Badri-Spröwitz, Alexander},
  doi          = {10.1038/s42256-022-00505-4},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {652-660},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning plastic matching of robot dynamics in closed-loop central pattern generators},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An adaptive graph learning method for automated molecular
interactions and properties predictions. <em>NATMI</em>, <em>4</em>(7),
645–651. (<a href="https://doi.org/10.1038/s42256-022-00501-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Improving drug discovery efficiency is a core and long-standing challenge in drug discovery. For this purpose, many graph learning methods have been developed to search potential drug candidates with fast speed and low cost. In fact, the pursuit of high prediction performance on a limited number of datasets has crystallized their architectures and hyperparameters, making them lose advantage in repurposing to new data generated in drug discovery. Here we propose a flexible method that can adapt to any dataset and make accurate predictions. The proposed method employs an adaptive pipeline to learn from a dataset and output a predictor. Without any manual intervention, the method achieves far better prediction performance on all tested datasets than traditional methods, which are based on hand-designed neural architectures and other fixed items. In addition, we found that the proposed method is more robust than traditional methods and can provide meaningful interpretability. Given the above, the proposed method can serve as a reliable method to predict molecular interactions and properties with high adaptability, performance, robustness and interpretability. This work takes a solid step forward to the purpose of aiding researchers to design better drugs with high efficiency. Deep learning methods can provide useful predictions for drug design, but their hyperparameters need to be carefully tweaked to give good performance on a specific problem or dataset. Li et al. present here a method that finds appropriate architectures and hyperparameters for a wide range of drug design tasks and can achieve good performance without human intervention.},
  archive      = {J_NATMI},
  author       = {Li, Yuquan and Hsieh, Chang-Yu and Lu, Ruiqiang and Gong, Xiaoqing and Wang, Xiaorui and Li, Pengyong and Liu, Shuo and Tian, Yanan and Jiang, Dejun and Yan, Jiaxian and Bai, Qifeng and Liu, Huanxiang and Zhang, Shengyu and Yao, Xiaojun},
  doi          = {10.1038/s42256-022-00501-8},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {645-651},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An adaptive graph learning method for automated molecular interactions and properties predictions},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning biophysical determinants of cell fate with deep
neural networks. <em>NATMI</em>, <em>4</em>(7), 636–644. (<a
href="https://doi.org/10.1038/s42256-022-00503-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning is now a powerful tool in microscopy data analysis, and is routinely used for image processing applications such as segmentation and denoising. However, it has rarely been used to directly learn mechanistic models of a biological system, owing to the complexity of the internal representations. Here, we develop an end-to-end machine learning approach capable of learning an explainable model of a complex biological phenomenon, cell competition, directly from a large corpus of time-lapse microscopy data. Cell competition is a quality control mechanism that eliminates unfit cells from a tissue, during which cell fate is thought to be determined by the local cellular neighbourhood over time. To investigate this, we developed a new approach (τ-VAE) by coupling a probabilistic encoder to a temporal convolution network to predict the fate of each cell in an epithelium. Using the τ-VAE’s latent representation of the local tissue organization and the flow of information in the network, we decode the physical parameters responsible for correct prediction of fate in cell competition. Remarkably, the model autonomously learns that cell density is the single most important factor in predicting cell fate—a conclusion that is in agreement with our current understanding from over a decade of scientific research. Finally, to test the learned internal representation, we challenge the network with experiments performed in the presence of drugs that block signalling pathways involved in competition. We present a novel discriminator network, which using the predictions of the τ-VAE can identify conditions that deviate from the normal behaviour, paving the way for automated, mechanism-aware drug screening. An end-to-end machine learning approach that can learn which mechanisms determine cell fate and competition from a large time-lapse microscopy dataset is developed. The approach makes use of a probabilistic autoencoder to learn an interpretable representation of the organization of cells, and provides cell fate predictions that can be tested in drug screening experiments.},
  archive      = {J_NATMI},
  author       = {Soelistyo, Christopher J. and Vallardi, Giulia and Charras, Guillaume and Lowe, Alan R.},
  doi          = {10.1038/s42256-022-00503-6},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {636-644},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning biophysical determinants of cell fate with deep neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Molecular convolutional neural networks with DNA regulatory
circuits. <em>NATMI</em>, <em>4</em>(7), 625–635. (<a
href="https://doi.org/10.1038/s42256-022-00502-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex biomolecular circuits enabled cells with intelligent behaviour to survive before neural brains evolved. Since DNA computing was first demonstrated in the mid-1990s, synthetic DNA circuits in liquid phase have been developed as computational hardware to perform neural network-like computations that harness the collective properties of complex biochemical systems. However, scaling up such DNA-based neural networks to support more powerful computation remains challenging. Here we present a systematic molecular implementation of a convolutional neural network algorithm with synthetic DNA regulatory circuits based on a simple switching gate architecture. Our DNA-based weight-sharing convolutional neural network can simultaneously implement parallel multiply–accumulate operations for 144-bit inputs and recognize patterns in up to eight categories autonomously. Further, this system can be connected with other DNA circuits to construct hierarchical networks to recognize patterns in up to 32 categories with a two-step approach: coarse classification on language (Arabic numerals, Chinese oracles, English alphabets and Greek alphabets) followed by classification into specific handwritten symbols. We also reduced the computation time from hours to minutes by using a simple cyclic freeze–thaw approach. Our DNA-based regulatory circuits are a step towards the realization of a molecular computer with high computing power and the ability to classify complex and noisy information. Artificial DNA circuits that can perform neural network-like computations have been developed, but scaling up these circuits to recognize a large number of patterns is a challenging task. Xiong, Zhu and colleagues experimentally demonstrate a convolutional neural network algorithm using a synthetic DNA-based regulatory circuit in vitro and develop a freeze–thaw approach to reduce the computation time from hours to minutes, paving the way towards more powerful biomolecular classifiers.},
  archive      = {J_NATMI},
  author       = {Xiong, Xiewei and Zhu, Tong and Zhu, Yun and Cao, Mengyao and Xiao, Jin and Li, Li and Wang, Fei and Fan, Chunhai and Pei, Hao},
  doi          = {10.1038/s42256-022-00502-7},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {625-635},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Molecular convolutional neural networks with DNA regulatory circuits},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Neural error mitigation of near-term quantum simulations.
<em>NATMI</em>, <em>4</em>(7), 618–624. (<a
href="https://doi.org/10.1038/s42256-022-00509-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Near-term quantum computers provide a promising platform for finding the ground states of quantum systems, which is an essential task in physics, chemistry and materials science. However, near-term approaches are constrained by the effects of noise, as well as the limited resources of near-term quantum hardware. We introduce neural error mitigation, which uses neural networks to improve estimates of ground states and ground-state observables obtained using near-term quantum simulations. To demonstrate our method’s broad applicability, we employ neural error mitigation to find the ground states of the H2 and LiH molecular Hamiltonians, as well as the lattice Schwinger model, prepared via the variational quantum eigensolver. Our results show that neural error mitigation improves numerical and experimental variational quantum eigensolver computations to yield low energy errors, high fidelities and accurate estimations of more complex observables such as order parameters and entanglement entropy without requiring additional quantum resources. Furthermore, neural error mitigation is agnostic with respect to the quantum state preparation algorithm used, the quantum hardware it is implemented on and the particular noise channel affecting the experiment, contributing to its versatility as a tool for quantum simulation. So-called noisy intermediate-scale quantum devices will be capable of a range of quantum simulation tasks, provided that the effects of noise can be sufficiently reduced. A neural error mitigation approach is developed that uses neural networks to improve the estimates of ground states and ground-state observables of molecules and quantum systems obtained using quantum simulations on near-term devices.},
  archive      = {J_NATMI},
  author       = {Bennewitz, Elizabeth R. and Hopfmueller, Florian and Kulchytskyy, Bohdan and Carrasquilla, Juan and Ronagh, Pooya},
  doi          = {10.1038/s42256-022-00509-0},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {618-624},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Neural error mitigation of near-term quantum simulations},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning the missing channel. <em>NATMI</em>, <em>4</em>(7),
616–617. (<a href="https://doi.org/10.1038/s42256-022-00514-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microscopy-based drug screens with fluorescent markers can shed light on how drugs affect biological processes. Without adding markers and imaging channels, which is cumbersome and costly, a new generative deep-learning method extracts new fluorescence channels from images, potentially improving the drug-discovery pipeline.},
  archive      = {J_NATMI},
  author       = {Heigwer, Florian},
  doi          = {10.1038/s42256-022-00514-3},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {616-617},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning the missing channel},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). In vitro convolutional neural networks. <em>NATMI</em>,
<em>4</em>(7), 614–615. (<a
href="https://doi.org/10.1038/s42256-022-00508-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neural networks can be implemented by using purified DNA molecules that interact in a test tube. Convolutional neural networks to classify high-dimensional data have now been realized in vitro, in one of the most complex demonstrations of molecular programming so far.},
  archive      = {J_NATMI},
  author       = {Poole, William},
  doi          = {10.1038/s42256-022-00508-1},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {614-615},
  shortjournal = {Nat. Mach. Intell.},
  title        = {In vitro convolutional neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bringing artificial intelligence to business management.
<em>NATMI</em>, <em>4</em>(7), 611–613. (<a
href="https://doi.org/10.1038/s42256-022-00512-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence (AI) can support managers by effectively delegating management decisions to AI. There are, however, many organizational and technical hurdles that need to be overcome, and we offer a first step on this journey by unpacking the core factors that may hinder or foster effective decision delegation to AI.},
  archive      = {J_NATMI},
  author       = {Feuerriegel, Stefan and Shrestha, Yash Raj and von Krogh, Georg and Zhang, Ce},
  doi          = {10.1038/s42256-022-00512-5},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {611-613},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Bringing artificial intelligence to business management},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). New deepfake regulations in china are a tool for social
stability, but at what cost? <em>NATMI</em>, <em>4</em>(7), 608–610. (<a
href="https://doi.org/10.1038/s42256-022-00513-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {China is pushing ahead of the European Union and the United States with its new synthetic content regulations. New draft provisions would place more responsibility on platforms to preserve social stability, with potential costs for online freedoms. They show that the Chinese Communist Party is prepared to protect itself against the unique threats of emerging technologies.},
  archive      = {J_NATMI},
  author       = {Hine, Emmie and Floridi, Luciano},
  doi          = {10.1038/s42256-022-00513-4},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {608-610},
  shortjournal = {Nat. Mach. Intell.},
  title        = {New deepfake regulations in china are a tool for social stability, but at what cost?},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A teachable moment for dual-use. <em>NATMI</em>,
<em>4</em>(7), 607. (<a
href="https://doi.org/10.1038/s42256-022-00511-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Urbina, Fabio and Lentzos, Filippa and Invernizzi, Cédric and Ekins, Sean},
  doi          = {10.1038/s42256-022-00511-6},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {607},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A teachable moment for dual-use},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). AI podcasts for the summer. <em>NATMI</em>, <em>4</em>(7),
605–606. (<a href="https://doi.org/10.1038/s42256-022-00517-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As with last summer, COVID-19 is still with us, but there is a semblance of what life was like before the pandemic. Here, we recommend AI podcasts from the past year that may inform, inspire or entertain, as we get an opportunity to travel or take time away from regular activities.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00517-0},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {605-606},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI podcasts for the summer},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Interpreting the b-cell receptor repertoire with single-cell
gene expression using benisse. <em>NATMI</em>, <em>4</em>(6), 596–604.
(<a href="https://doi.org/10.1038/s42256-022-00492-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {B-cell receptors (BCRs) are a crucial player in the development and activation of B cells, and their mature forms are secreted as antibodies, which execute functions such as the neutralization of invading pathogens. All current analytical approaches for BCRs solely investigate the BCR sequences and ignore their correlations with the transcriptomics of the B cells, yielding conclusions of unknown functional relevance regarding the roles of BCRs and B cells, and could generate biased interpretation. Many single-cell RNA-sequencing (scRNA-seq) techniques can now capture both the gene expression and BCR of each B cell, which could potentially address this issue. Here, we investigated 43,938 B cells from 13 scRNA-seq datasets with matched scBCR sequencing, and we observed an association between the BCRs and the B cells’ transcriptomics. Motivated by this, we developed the Benisse model (BCR embedding graphical network informed by scRNA-seq) to provide refined analyses of BCRs guided by single-cell gene expression. Benisse revealed a gradient of B-cell activation along BCR trajectories. We discovered a stronger coupling between BCRs and B-cell gene expression during COVID-19 infections. We found that BCRs form a directed pattern of continuous and linear evolution to achieve the highest antigen targeting efficiency, compared with the convergent evolution pattern of T-cell receptors. Overall, a simultaneous digestion of the BCR and gene expression of B cells, viewed through the lens of Benisse, will lead to a more insightful interpretation of the functional relevance of the BCR repertoire in different biological contexts. B-cell receptors (BCRs) and their impact on B cells play a vital role in our immune system; however, the manner in which B cells are activated by BCRs are still poorly understood. Ze Zhang and colleagues present a graph-based method that connects BCR and single B-cell RNA sequencing data and identifies notable coupling between BCR and B-cell expression in COVID-19.},
  archive      = {J_NATMI},
  author       = {Zhang, Ze and Chang, Woo Yong and Wang, Kaiwen and Yang, Yuqiu and Wang, Xinlei and Yao, Chen and Wu, Tuoqi and Wang, Li and Wang, Tao},
  doi          = {10.1038/s42256-022-00492-6},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {596-604},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Interpreting the B-cell receptor repertoire with single-cell gene expression using benisse},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Trans-channel fluorescence learning improves high-content
screening for alzheimer’s disease therapeutics. <em>NATMI</em>,
<em>4</em>(6), 583–595. (<a
href="https://doi.org/10.1038/s42256-022-00490-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In microscopy-based drug screens, fluorescent markers carry critical information on how compounds affect different biological processes. However, practical considerations, such as the labour and preparation formats needed to produce different image channels, hinder the use of certain fluorescent markers. Consequently, completed screens may lack biologically informative but experimentally impractical markers. Here we present a deep learning method for overcoming these limitations. We accurately generated predicted fluorescent signals from other related markers and validated this new machine learning (ML) method on two biologically distinct datasets. We used the ML method to improve the selection of biologically active compounds for Alzheimer’s disease from a completed high-content high-throughput screen (HCS) that only contained the original markers. The ML method identified novel compounds that effectively blocked tau aggregation, which had been missed by traditional screening approaches unguided by ML. The method improved triaging efficiency of compound rankings over conventional rankings by raw image channels. We reproduced this ML pipeline on a biologically independent cancer-based dataset, demonstrating its generalizability. The approach is disease-agnostic and applicable across diverse fluorescence microscopy datasets. Fluorescent markers in microscopy-based drug screens carry information on how compounds affect biological processes, but practical considerations may hinder their use. Wong et al. develop a deep learning method for generating images in drug discovery, with broad applicability across diverse fluorescence microscopy datasets.},
  archive      = {J_NATMI},
  author       = {Wong, Daniel R. and Conrad, Jay and Johnson, Noah R. and Ayers, Jacob and Laeremans, Annelies and Lee, Joanne C. and Lee, Jisoo and Prusiner, Stanley B. and Bandyopadhyay, Sourav and Butte, Atul J. and Paras, Nick A. and Keiser, Michael J.},
  doi          = {10.1038/s42256-022-00490-8},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {583-595},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Trans-channel fluorescence learning improves high-content screening for alzheimer’s disease therapeutics},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A textile exomuscle that assists the shoulder during
functional movements for everyday life. <em>NATMI</em>, <em>4</em>(6),
574–582. (<a href="https://doi.org/10.1038/s42256-022-00495-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Effortlessly performing activities of daily living constitutes a cornerstone of our personal independence. Naturally, various forms of upper limb impairments can have a substantial impact on quality of life. We developed the Myoshirt, a textile-based soft wearable robot, or exomuscle, that autonomously follows the user’s movements and thereby assists the shoulder against gravity. With the Myoshirt, participants without impairments (n = 10, 5 male) experienced a delayed onset of muscular fatigue by 51.1 s (36.1\%, P &amp;lt; 0.001), while during a functional task their muscular activity decreased by 49.1\% (P &amp;lt; 0.001). Analogously, two participants with upper limb impairments due to a muscular dystrophy and a spinal cord injury experienced a delayed onset of muscular fatigue during unloaded arm lifts by 256.4 s (61.5\%) and by 450.6 s (210.3\%), respectively. Our evidence suggests that the Myoshirt is an effective tool that intuitively assists the shoulder during functional reaching tasks, with the potential of increasing the personal independence of people with upper limb impairments. Exoskeletons can assist movement in upper limb impairments to recover mobility and independence, but rigid or heavy exoskeletons can be impractical. Georgarakis and colleagues have developed a soft, tendon-driven device that assists shoulder movements and counteracts gravity to reduce muscle fatigue.},
  archive      = {J_NATMI},
  author       = {Georgarakis, Anna-Maria and Xiloyannis, Michele and Wolf, Peter and Riener, Robert},
  doi          = {10.1038/s42256-022-00495-3},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {574-582},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A textile exomuscle that assists the shoulder during functional movements for everyday life},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Gradient-based learning drives robust representations in
recurrent neural networks by balancing compression and expansion.
<em>NATMI</em>, <em>4</em>(6), 564–573. (<a
href="https://doi.org/10.1038/s42256-022-00498-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neural networks need the right representations of input data to learn. Here we ask how gradient-based learning shapes a fundamental property of representations in recurrent neural networks (RNNs)—their dimensionality. Through simulations and mathematical analysis, we show how gradient descent can lead RNNs to compress the dimensionality of their representations in a way that matches task demands during training while supporting generalization to unseen examples. This can require an expansion of dimensionality in early timesteps and compression in later ones, and strongly chaotic RNNs appear particularly adept at learning this balance. Beyond helping to elucidate the power of appropriately initialized artificial RNNs, this fact has implications for neurobiology as well. Neural circuits in the brain reveal both high variability associated with chaos and low-dimensional dynamical structures. Taken together, our findings show how simple gradient-based learning rules lead neural networks to solve tasks with robust representations that generalize to new cases. Neural networks in the brain often exhibit chaotic dynamics that can be captured by a small number of dimensions. Farrell et al. find that recurrent neural networks trained with gradient-based learning rules exhibit similar features. This helps form robust but generalizable input representations.},
  archive      = {J_NATMI},
  author       = {Farrell, Matthew and Recanatesi, Stefano and Moore, Timothy and Lajoie, Guillaume and Shea-Brown, Eric},
  doi          = {10.1038/s42256-022-00498-0},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {564-573},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Gradient-based learning drives robust representations in recurrent neural networks by balancing compression and expansion},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Improving de novo molecular design with curriculum
learning. <em>NATMI</em>, <em>4</em>(6), 555–563. (<a
href="https://doi.org/10.1038/s42256-022-00494-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reinforcement learning is a powerful paradigm that has gained popularity across multiple domains. However, applying reinforcement learning may come at the cost of multiple interactions between the agent and the environment. This cost can be especially pronounced when the single feedback from the environment is slow or computationally expensive, causing extensive periods of non-productivity. Curriculum learning provides a suitable alternative by arranging a sequence of tasks of increasing complexity, with the aim of reducing the overall cost of learning. Here we demonstrate the application of curriculum learning for drug discovery. We implement curriculum learning in the de novo design platform REINVENT, and apply it to illustrative molecular design problems of different complexities. The results show both accelerated learning and a positive impact on the quality of the output when compared with standard policy-based reinforcement learning. While reinforcement learning can be a powerful tool for complex design tasks such as molecular design, training can be slow when problems are either too hard or too easy, as little is learned in these cases. Jeff Guo and colleagues provide a curriculum learning extension to the REINVENT de novo molecular design framework that provides problems of increasing difficulty over epochs such that the training process is more efficient.},
  archive      = {J_NATMI},
  author       = {Guo, Jeff and Fialková, Vendy and Arango, Juan Diego and Margreitter, Christian and Janet, Jon Paul and Papadopoulos, Kostas and Engkvist, Ola and Patronov, Atanas},
  doi          = {10.1038/s42256-022-00494-4},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {555-563},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Improving de novo molecular design with curriculum learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning interpretable representations of entanglement in
quantum optics experiments using deep generative models. <em>NATMI</em>,
<em>4</em>(6), 544–554. (<a
href="https://doi.org/10.1038/s42256-022-00493-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum physics experiments produce interesting phenomena such as interference or entanglement, which are the core properties of numerous future quantum technologies. The complex relationship between the setup structure of a quantum experiment and its entanglement properties is essential to fundamental research in quantum optics but is difficult to intuitively understand. We present a deep generative model of quantum optics experiments where a variational autoencoder is trained on a dataset of quantum optics experiment setups. In a series of computational experiments, we investigate the learned representation of our quantum optics variational autoencoder (QOVAE) and its internal understanding of the quantum optics world. We demonstrate that QOVAE learns an interpretable representation of quantum optics experiments and the relationship between the experiment structure and entanglement. We show QOVAE is able to generate novel experiments for highly entangled quantum states with specific distributions that match its training data. QOVAE can learn to generate specific entangled states and efficiently search the space of experiments that produce highly entangled quantum states. Importantly, we are able to interpret how QOVAE structures its latent space, finding curious patterns that we can explain in terms of quantum physics. The results demonstrate how we can use and understand the internal representations of deep generative models in a complex scientific domain. QOVAE and the insights from our investigations can be immediately applied to other physical systems. A variational autoencoder is trained on a dataset of quantum optics experiment configurations and learns an interpretable representation of the relationship between experiment setup and quantum entanglement. The approach can be used to explore new experiment designs with specific, highly entangled states.},
  archive      = {J_NATMI},
  author       = {Flam-Shepherd, Daniel and Wu, Tony C. and Gu, Xuemei and Cervera-Lierta, Alba and Krenn, Mario and Aspuru-Guzik, Alán},
  doi          = {10.1038/s42256-022-00493-5},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {544-554},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning interpretable representations of entanglement in quantum optics experiments using deep generative models},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A framework for tool cognition in robots without prior tool
learning or observation. <em>NATMI</em>, <em>4</em>(6), 533–543. (<a
href="https://doi.org/10.1038/s42256-022-00500-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human tool use prowess distinguishes us from other animals. In many scenarios, a human is able to recognize objects seen for the first time as potential tools for a task and use them without requiring any learning. Here we propose a framework to enable similar abilities in robots. We first characterized human tools to identify a special category of tools that humans are able to use immediately through a process of skill transfer from their limbs, rather than tool learning. Motivated by the tool characterization and our neuroscientific studies on human tool use and embodiment, we then developed a tool cognition framework that enables a robot to recognize a previously unseen object as a tool for a task and plan how to grasp and wield the tool in the face of constraints and obstacles, before finally performing the task with the tool. Furthermore, the framework allows for flexibility in tool use, where the same tool can be adapted for different tasks, and different tools for the same task, all without any prior learning or observation of tool use. We demonstrate the possibilities offered by our tool cognition framework in several robot experiments with both toy and real objects as tools. Robots usually learn to use tools from direct experience or from observing the use of a tool. While knowledge can be transferred between similar tools, novel and creative use of tools is challenging. Tee and colleagues present a method where skill transfer does not come from experience of using other tools but from using the robot’s own limbs.},
  archive      = {J_NATMI},
  author       = {Tee, Keng Peng and Cheong, Samuel and Li, Jun and Ganesh, Gowrishankar},
  doi          = {10.1038/s42256-022-00500-9},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {533-543},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A framework for tool cognition in robots without prior tool learning or observation},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Controllable protein design with language models.
<em>NATMI</em>, <em>4</em>(6), 521–532. (<a
href="https://doi.org/10.1038/s42256-022-00499-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The twenty-first century is presenting humankind with unprecedented environmental and medical challenges. The ability to design novel proteins tailored for specific purposes would potentially transform our ability to respond to these issues in a timely manner. Recent advances in the field of artificial intelligence are now setting the stage to make this goal achievable. Protein sequences are inherently similar to natural languages: amino acids arrange in a multitude of combinations to form structures that carry function, the same way as letters form words and sentences carry meaning. Accordingly, it is not surprising that, throughout the history of natural language processing (NLP), many of its techniques have been applied to protein research problems. In the past few years we have witnessed revolutionary breakthroughs in the field of NLP. The implementation of transformer pre-trained models has enabled text generation with human-like capabilities, including texts with specific properties such as style or subject. Motivated by its considerable success in NLP tasks, we expect dedicated transformers to dominate custom protein sequence generation in the near future. Fine-tuning pre-trained models on protein families will enable the extension of their repertoires with novel sequences that could be highly divergent but still potentially functional. The combination of control tags such as cellular compartment or function will further enable the controllable design of novel protein functions. Moreover, recent model interpretability methods will allow us to open the ‘black box’ and thus enhance our understanding of folding principles. Early initiatives show the enormous potential of generative language models to design functional sequences. We believe that using generative text models to create novel proteins is a promising and largely unexplored field, and we discuss its foreseeable impact on protein design. Both proteins and natural language are essentially based on a sequential code, but feature complex interactions at multiple scales, which can be useful when transferring machine learning models from one domain to another. In this Review, Ferruz and Höcker summarize recent advances in language models, such as transformers, and their application to protein design.},
  archive      = {J_NATMI},
  author       = {Ferruz, Noelia and Höcker, Birte},
  doi          = {10.1038/s42256-022-00499-z},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {521-532},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Controllable protein design with language models},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Lessons from infant learning for unsupervised machine
learning. <em>NATMI</em>, <em>4</em>(6), 510–520. (<a
href="https://doi.org/10.1038/s42256-022-00488-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The desire to reduce the dependence on curated, labeled datasets and to leverage the vast quantities of unlabeled data has triggered renewed interest in unsupervised (or self-supervised) learning algorithms. Despite improved performance due to approaches such as the identification of disentangled latent representations, contrastive learning and clustering optimizations, unsupervised machine learning still falls short of its hypothesized potential as a breakthrough paradigm enabling generally intelligent systems. Inspiration from cognitive (neuro)science has been based mostly on adult learners with access to labels and a vast amount of prior knowledge. To push unsupervised machine learning forward, we argue that developmental science of infant cognition might hold the key to unlocking the next generation of unsupervised learning approaches. We identify three crucial factors enabling infants’ quality and speed of learning: (1) babies’ information processing is guided and constrained; (2) babies are learning from diverse, multimodal inputs; and (3) babies’ input is shaped by development and active learning. We assess the extent to which these insights from infant learning have already been exploited in machine learning, examine how closely these implementations resemble the core insights, and propose how further adoption of these factors can give rise to previously unseen performance levels in unsupervised learning. Unsupervised machine learning algorithms reduce the dependence on curated, labeled datasets that are characteristic of supervised machine learning. The authors argue that the developmental science of infant cognition could inform the design of unsupervised machine learning approaches.},
  archive      = {J_NATMI},
  author       = {Zaadnoordijk, Lorijn and Besold, Tarek R. and Cusack, Rhodri},
  doi          = {10.1038/s42256-022-00488-2},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {510-520},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Lessons from infant learning for unsupervised machine learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). From algorithmic accountability to digital governance.
<em>NATMI</em>, <em>4</em>(6), 508–509. (<a
href="https://doi.org/10.1038/s42256-022-00504-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Mökander, Jakob and Floridi, Luciano},
  doi          = {10.1038/s42256-022-00504-5},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {508-509},
  shortjournal = {Nat. Mach. Intell.},
  title        = {From algorithmic accountability to digital governance},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). No chemical killer AI (yet). <em>NATMI</em>, <em>4</em>(6),
506–507. (<a href="https://doi.org/10.1038/s42256-022-00497-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Blum, Marc-Michael},
  doi          = {10.1038/s42256-022-00497-1},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {506-507},
  shortjournal = {Nat. Mach. Intell.},
  title        = {No chemical killer AI (yet)},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Privacy debate obscures pandemic power shifts.
<em>NATMI</em>, <em>4</em>(6), 505. (<a
href="https://doi.org/10.1038/s42256-022-00507-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soon into the COVID-19 pandemic, civil-rights groups raised the alarm over the increase in digital surveillance infringing on individual rights. But there are other potential harms as tech companies accelerate their expansion into new areas essential to public-service provision.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00507-2},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {505},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Privacy debate obscures pandemic power shifts},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An interpretable deep learning workflow for discovering
subvisual abnormalities in CT scans of COVID-19 inpatients and
survivors. <em>NATMI</em>, <em>4</em>(5), 494–503. (<a
href="https://doi.org/10.1038/s42256-022-00483-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tremendous efforts have been made to improve diagnosis and treatment of COVID-19, but knowledge on long-term complications is limited. In particular, a large portion of survivors has respiratory complications, but currently, experienced radiologists and state-of-the-art artificial intelligence systems are not able to detect many abnormalities from follow-up computerized tomography (CT) scans of COVID-19 survivors. Here we propose Deep-LungParenchyma-Enhancing (DLPE), a computer-aided detection (CAD) method for detecting and quantifying pulmonary parenchyma lesions on chest CT. Through proposing a number of deep-learning-based segmentation models and assembling them in an interpretable manner, DLPE removes irrelevant tissues from the perspective of pulmonary parenchyma, and calculates the scan-level optimal window, which considerably enhances parenchyma lesions relative to the lung window. Aided by DLPE, radiologists discovered novel and interpretable lesions from COVID-19 inpatients and survivors, which were previously invisible under the lung window. Based on DLPE, we removed the scan-level bias of CT scans, and then extracted precise radiomics from such novel lesions. We further demonstrated that these radiomics have strong predictive power for key COVID-19 clinical metrics on an inpatient cohort of 1,193 CT scans and for sequelae on a survivor cohort of 219 CT scans. Our work sheds light on the development of interpretable medical artificial intelligence and showcases how artificial intelligence can discover medical findings that are beyond sight. Respiratory complications after a COVID infection are a growing concern, but follow-up chest CT scans of COVID-19 survivors hardly present any recognizable lesions. A deep learning-based method was developed that calculates a scan-specific optimal window and removes irrelevant tissues such as airways and blood vessels from images with segmentation models, so that subvisual abnormalities in lung scans become visible.},
  archive      = {J_NATMI},
  author       = {Zhou, Longxi and Meng, Xianglin and Huang, Yuxin and Kang, Kai and Zhou, Juexiao and Chu, Yuetan and Li, Haoyang and Xie, Dexuan and Zhang, Jiannan and Yang, Weizhen and Bai, Na and Zhao, Yi and Zhao, Mingyan and Wang, Guohua and Carin, Lawrence and Xiao, Xigang and Yu, Kaijiang and Qiu, Zhaowen and Gao, Xin},
  doi          = {10.1038/s42256-022-00483-7},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {494-503},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An interpretable deep learning workflow for discovering subvisual abnormalities in CT scans of COVID-19 inpatients and survivors},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Autonomous environment-adaptive microrobot swarm navigation
enabled by deep learning-based real-time distribution planning.
<em>NATMI</em>, <em>4</em>(5), 480–493. (<a
href="https://doi.org/10.1038/s42256-022-00482-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Navigating a large swarm of micro-/nanorobots is critical for potential targeted delivery/therapy applications owing to the limited volume/function of a single microrobot, and microrobot swarms with distribution reconfigurability can adapt to environments during navigation. However, current microrobot swarms lack the intelligent behaviour to autonomously adjust their distribution and motion according to environmental change. Such autonomous navigation is challenging, and requires real-time appropriate decision-making capability of the swarm for unknown and unstructured environments. Here, to tackle this issue, we propose a framework that defines different autonomy levels for environment-adaptive microrobot swarm navigation and designs corresponding system components for each level. To realize high autonomy levels, real-time autonomous distribution planning is a key capability for the swarm, regarding which we show that deep learning is an enabling approach that allows the microrobot swarm to learn optimal distributions in extensive unstructured environmental morphologies. For real-world demonstration, we study the reconfigurable magnetic nanoparticle swarm and experimentally demonstrate autonomous swarm navigation for targeted delivery and cargo transport in environments with channels or obstacles. This work could introduce computational intelligence to micro-/nanorobot swarms, enabling them to autonomously make appropriate decisions during navigation in unstructured environments. Swarms of microrobots could eventually be used to deliver drugs to specific targets in the body, but the coordination of these swarms in complex environments is challenging. Yang and colleagues present a real-time autonomous distribution planning method based on deep learning that controls both the shape and position of the swarm, as well as the imaging system used for swarm navigation to cover longer distances.},
  archive      = {J_NATMI},
  author       = {Yang, Lidong and Jiang, Jialin and Gao, Xiaojie and Wang, Qinglong and Dou, Qi and Zhang, Li},
  doi          = {10.1038/s42256-022-00482-8},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {480-493},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Autonomous environment-adaptive microrobot swarm navigation enabled by deep learning-based real-time distribution planning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A long short-term memory for AI applications in spike-based
neuromorphic hardware. <em>NATMI</em>, <em>4</em>(5), 467–479. (<a
href="https://doi.org/10.1038/s42256-022-00480-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spike-based neuromorphic hardware holds promise for more energy-efficient implementations of deep neural networks (DNNs) than standard hardware such as GPUs. But this requires us to understand how DNNs can be emulated in an event-based sparse firing regime, as otherwise the energy advantage is lost. In particular, DNNs that solve sequence processing tasks typically employ long short-term memory units that are hard to emulate with few spikes. We show that a facet of many biological neurons, slow after-hyperpolarizing currents after each spike, provides an efficient solution. After-hyperpolarizing currents can easily be implemented in neuromorphic hardware that supports multi-compartment neuron models, such as Intel’s Loihi chip. Filter approximation theory explains why after-hyperpolarizing neurons can emulate the function of long short-term memory units. This yields a highly energy-efficient approach to time-series classification. Furthermore, it provides the basis for an energy-efficient implementation of an important class of large DNNs that extract relations between words and sentences in order to answer questions about the text. Deep learning could be less energy intensive when implemented on spike-based neuromorphic chips. An approach inspired by a characteristic feature of biological neurons, the presence of slowly changing internal currents, is developed to emulate long short-term memory units in a sparse spiking regime for neuromorphic implementation.},
  archive      = {J_NATMI},
  author       = {Rao, Arjun and Plank, Philipp and Wild, Andreas and Maass, Wolfgang},
  doi          = {10.1038/s42256-022-00480-w},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {467-479},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A long short-term memory for AI applications in spike-based neuromorphic hardware},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Microscopy analysis neural network to solve detection,
enumeration and segmentation from image-level annotations.
<em>NATMI</em>, <em>4</em>(5), 455–466. (<a
href="https://doi.org/10.1038/s42256-022-00472-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of deep learning approaches to detect, segment or classify structures of interest has transformed the field of quantitative microscopy. High-throughput quantitative image analysis presents a challenge due to the complexity of the image content and the difficulty to retrieve precisely annotated datasets. Methods capable of reducing the annotation burden associated with the training of a deep neural network on microscopy images becomes primordial. Here we introduce a weakly supervised MICRoscopy Analysis neural network (MICRA-Net) that can be trained on a simple main classification task using image-level annotations to solve multiple more complex tasks such as semantic segmentation. MICRA-Net relies on the latent information embedded within a trained model to achieve performances similar to established architectures when no precisely annotated dataset is available. This learnt information is extracted from the network using gradient class activation maps, which are combined to generate detailed feature maps of the biological structures of interest. We demonstrate how MICRA-Net substantially alleviates the expert annotation process on various microscopy datasets and can be used for high-throughput quantitative analysis of microscopy images. An important problem in microscopy analysis is to reduce the need for manual annotation of datasets. Bilodeau et al. develop a neural network method that can be trained on a simple main classification task using image-level annotations to solve more complex tasks such as semantic segmentation.},
  archive      = {J_NATMI},
  author       = {Bilodeau, Anthony and Delmas, Constantin V. L. and Parent, Martin and De Koninck, Paul and Durand, Audrey and Lavoie-Cardinal, Flavie},
  doi          = {10.1038/s42256-022-00472-w},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {455-466},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Microscopy analysis neural network to solve detection, enumeration and segmentation from image-level annotations},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prediction of robust scientific facts from literature.
<em>NATMI</em>, <em>4</em>(5), 445–454. (<a
href="https://doi.org/10.1038/s42256-022-00474-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growth of published science in recent years has escalated the difficulty that human and algorithmic agents face in reasoning over prior knowledge to select the next experiment. This challenge is increased by uncertainty about the reproducibility of published findings. The availability of massive digital archives, machine reading, extraction tools and automated high-throughput experiments allows us to evaluate these challenges computationally at scale and identify novel opportunities to craft policies that accelerate scientific progress. Here we demonstrate a Bayesian calculus that enables positive prediction of robust scientific claims with findings extracted from published literature, weighted by scientific, social and institutional factors demonstrated to increase replicability. Illustrated with the case of gene regulatory interactions, our approach automatically estimates and counteracts sources of bias, revealing that scientifically focused but socially and institutionally diverse research activity is most likely to replicate. This results in updated certainty about the literature, which accurately predicts robust scientific facts on which new experiments should build. Our findings allow us to identify and evaluate policy recommendations for scientific institutions that may increase robust scientific knowledge, including sponsorship of increased diversity of and independence between investigations of any particular scientific phenomenon, and diversity of scientific phenomena investigated. With the availability of a vast and growing number of digital publications, machine reading and other knowledge mining tools, computational methods can be applied at scale to extract insights from the scientific literature. Belikov et al. develop a Bayesian method to mine the biomedical literature that identifies robust scientific findings which could improve the planning of further experiments and scientific investigation.},
  archive      = {J_NATMI},
  author       = {Belikov, Alexander V. and Rzhetsky, Andrey and Evans, James},
  doi          = {10.1038/s42256-022-00474-8},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {445-454},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Prediction of robust scientific facts from literature},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automated causal inference in application to randomized
controlled clinical trials. <em>NATMI</em>, <em>4</em>(5), 436–444. (<a
href="https://doi.org/10.1038/s42256-022-00470-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Randomized controlled trials (RCTs) are considered the gold standard for testing causal hypotheses in the clinical domain; however, the investigation of prognostic variables of patient outcome in a hypothesized cause–effect route is not feasible using standard statistical methods. Here we propose a new automated causal inference method (AutoCI) built on the invariant causal prediction (ICP) framework for the causal reinterpretation of clinical trial data. Compared with existing methods, we show that the proposed AutoCI allows one to clearly determine the causal variables of two real-world RCTs of patients with endometrial cancer with mature outcome and extensive clinicopathological and molecular data. This is achieved via suppressing the causal probability of non-causal variables by a wide margin. In ablation studies, we further demonstrate that the assignment of causal probabilities by AutoCI remains consistent in the presence of confounders. In conclusion, these results confirm the robustness and feasibility of AutoCI for future applications in real-world clinical analysis. The invariant causal prediction (ICP) framework tries to determine the causal variables given an outcome variable, but considerable effort is needed to adapt existing ICP methods to the clinical domain. The authors propose an automated causal inference method that could potentially address the challenges of applying the ICP framework to complex clinical datasets.},
  archive      = {J_NATMI},
  author       = {Wu, Ji Q. and Horeweg, Nanda and de Bruyn, Marco and Nout, Remi A. and Jürgenliemk-Schulz, Ina M. and Lutgens, Ludy C. H. W. and Jobsen, Jan J. and van der Steen-Banasik, Elzbieta M. and Nijman, Hans W. and Smit, Vincent T. H. B. M. and Bosse, Tjalling and Creutzberg, Carien L. and Koelzer, Viktor H.},
  doi          = {10.1038/s42256-022-00470-y},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {436-444},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Automated causal inference in application to randomized controlled clinical trials},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Functional mimicry of ruffini receptors with fibre bragg
gratings and deep neural networks enables a bio-inspired large-area
tactile-sensitive skin. <em>NATMI</em>, <em>4</em>(5), 425–435. (<a
href="https://doi.org/10.1038/s42256-022-00487-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Collaborative robots are expected to physically interact with humans in daily living and the workplace, including industrial and healthcare settings. A key related enabling technology is tactile sensing, which currently requires addressing the outstanding scientific challenge to simultaneously detect contact location and intensity by means of soft conformable artificial skins adapting over large areas to the complex curved geometries of robot embodiments. In this work, the development of a large-area sensitive soft skin with a curved geometry is presented, allowing for robot total-body coverage through modular patches. The biomimetic skin consists of a soft polymeric matrix, resembling a human forearm, embedded with photonic fibre Bragg grating transducers, which partially mimics Ruffini mechanoreceptor functionality with diffuse, overlapping receptive fields. A convolutional neural network deep learning algorithm and a multigrid neuron integration process were implemented to decode the fibre Bragg grating sensor outputs for inference of contact force magnitude and localization through the skin surface. Results of 35 mN (interquartile range 56 mN) and 3.2 mm (interquartile range 2.3 mm) median errors were achieved for force and localization predictions, respectively. Demonstrations with an anthropomorphic arm pave the way towards artificial intelligence based integrated skins enabling safe human–robot cooperation via machine intelligence. Tactile sensing is needed for robots to physically interact with humans in daily living and in the workplace. A scientific challenge in robotics is how to simultaneously detect contact location and intensity. The authors describe a large-area sensing skin for robotic system applications, specifically for human–machine interactions.},
  archive      = {J_NATMI},
  author       = {Massari, Luca and Fransvea, Giulia and D’Abbraccio, Jessica and Filosa, Mariangela and Terruso, Giuseppe and Aliperta, Andrea and D’Alesio, Giacomo and Zaltieri, Martina and Schena, Emiliano and Palermo, Eduardo and Sinibaldi, Edoardo and Oddo, Calogero Maria},
  doi          = {10.1038/s42256-022-00487-3},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {425-435},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Functional mimicry of ruffini receptors with fibre bragg gratings and deep neural networks enables a bio-inspired large-area tactile-sensitive skin},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The promise and perils of using artificial intelligence to
fight corruption. <em>NATMI</em>, <em>4</em>(5), 418–424. (<a
href="https://doi.org/10.1038/s42256-022-00489-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Corruption presents one of the biggest challenges of our time, and much hope is placed in artificial intelligence (AI) to combat it. Although the growing number of AI-based anti-corruption tools (AI-ACT) have been summarized, a critical examination of their promises and perils is lacking. Here we argue that the success of AI-ACT strongly depends on whether they are implemented top–down (by governments) or bottom–up (by citizens, non-governmental organizations or journalists). Top–down use of AI-ACT can consolidate power structures and thereby pose new corruption risks. Bottom–up use of AI-ACT has the potential to provide unprecedented means for the citizenry to keep their government and bureaucratic officials in check. We outline the societal and technical challenges that need to be overcome to harness the potential for AI to fight corruption. Despite the growing number of initiatives that employ AI to counter corruption, few studies empirically tackle the political and social consequences of embedding AI in anti-corruption efforts. The authors outline the societal and technical challenges that need to be overcome for AI to fight corruption.},
  archive      = {J_NATMI},
  author       = {Köbis, Nils and Starke, Christopher and Rahwan, Iyad},
  doi          = {10.1038/s42256-022-00489-1},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {418-424},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The promise and perils of using artificial intelligence to fight corruption},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A cautionary tale from the machine scientist.
<em>NATMI</em>, <em>4</em>(5), 416–417. (<a
href="https://doi.org/10.1038/s42256-022-00491-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine reading and knowledge extraction methods can be used to mine the scientific literature and reveal the direction and robustness of discoveries. Such efforts now point to the importance of independent tests of reported claims.},
  archive      = {J_NATMI},
  author       = {Amaral, Luís A. Nunes},
  doi          = {10.1038/s42256-022-00491-7},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {416-417},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A cautionary tale from the machine scientist},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A soft touch for robots. <em>NATMI</em>, <em>4</em>(5), 415.
(<a href="https://doi.org/10.1038/s42256-022-00496-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A biomimetic sense of touch could ground robots better in the physical world and improve their interactions with humans.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00496-2},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {415},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A soft touch for robots},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Autoencoders on field-programmable gate
arrays for real-time, unsupervised new physics detection at 40 MHz at
the large hadron collider. <em>NATMI</em>, <em>4</em>(4), 414. (<a
href="https://doi.org/10.1038/s42256-022-00486-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Govorkova, Ekaterina and Puljak, Ema and Aarrestad, Thea and James, Thomas and Loncar, Vladimir and Pierini, Maurizio and Pol, Adrian Alan and Ghielmetti, Nicolò and Graczyk, Maksymilian and Summers, Sioni and Ngadiuba, Jennifer and Nguyen, Thong Q. and Duarte, Javier and Wu, Zhenbin},
  doi          = {10.1038/s42256-022-00486-4},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {414},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Autoencoders on field-programmable gate arrays for real-time, unsupervised new physics detection at 40 MHz at the large hadron collider},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Advancing COVID-19 diagnosis with
privacy-preserving collaboration in artificial intelligence.
<em>NATMI</em>, <em>4</em>(4), 413. (<a
href="https://doi.org/10.1038/s42256-022-00485-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Bai, Xiang and Wang, Hanchen and Ma, Liya and Xu, Yongchao and Gan, Jiefeng and Fan, Ziwei and Yang, Fan and Ma, Ke and Yang, Jiehua and Bai, Song and Shu, Chang and Zou, Xinyu and Huang, Renhao and Zhang, Changzheng and Liu, Xiaowu and Tu, Dandan and Xu, Chuou and Zhang, Wenqing and Wang, Xi and Chen, Anguo and Zeng, Yu and Yang, Dehua and Wang, Ming-Wei and Holalkere, Nagaraj and Halin, Neil J. and Kamel, Ihab R. and Wu, Jia and Peng, Xuehua and Wang, Xiang and Shao, Jianbo and Mongkolwat, Pattanasak and Zhang, Jianjun and Liu, Weiyang and Roberts, Michael and Teng, Zhongzhao and Beer, Lucian and Sanchez, Lorena E. and Sala, Evis and Rubin, Daniel L. and Weller, Adrian and Lasenby, Joan and Zheng, Chuansheng and Wang, Jianming and Li, Zhen and Schönlieb, Carola and Xia, Tian},
  doi          = {10.1038/s42256-022-00485-5},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {413},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Advancing COVID-19 diagnosis with privacy-preserving collaboration in artificial intelligence},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep learning-inferred multiplex immunofluorescence for
immunohistochemical image quantification. <em>NATMI</em>, <em>4</em>(4),
401–412. (<a href="https://doi.org/10.1038/s42256-022-00471-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reporting biomarkers assessed by routine immunohistochemical (IHC) staining of tissue is broadly used in diagnostic pathology laboratories for patient care. So far, however, clinical reporting is predominantly qualitative or semi-quantitative. By creating a multitask deep learning framework, DeepLIIF, we present a single-step solution to stain deconvolution/separation, cell segmentation and quantitative single-cell IHC scoring. Leveraging a unique de novo dataset of co-registered IHC and multiplex immunofluorescence (mpIF) staining of the same slides, we segment and translate low-cost and prevalent IHC slides to more informative, but also more expensive, mpIF images, while simultaneously providing the essential ground truth for the superimposed brightfield IHC channels. A new nuclear-envelope stain, LAP2beta, with high (&amp;gt;95\%) cell coverage is also introduced to improve cell delineation/segmentation and protein expression quantification on IHC slides. We show that DeepLIIF trained on clean IHC Ki67 data can generalize to noisy images as well as other nuclear and non-nuclear markers. Multiplex immunofluorescence imaging can provide a wealth of data compared to immunohistochemical staining, which is cheaper and more widely available. Ghahremani et al. present DeepLIIF, a GAN-based cell segmentation and classification approach, which is trained on co-registered images of these two modalities to provide the insights from the more data-rich muliplex data from simpler IHC images.},
  archive      = {J_NATMI},
  author       = {Ghahremani, Parmida and Li, Yanyun and Kaufman, Arie and Vanguri, Rami and Greenwald, Noah and Angelo, Michael and Hollmann, Travis J. and Nadeem, Saad},
  doi          = {10.1038/s42256-022-00471-x},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {401-412},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning-inferred multiplex immunofluorescence for immunohistochemical image quantification},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Inferring transcription factor regulatory networks from
single-cell ATAC-seq data based on graph neural networks.
<em>NATMI</em>, <em>4</em>(4), 389–400. (<a
href="https://doi.org/10.1038/s42256-022-00469-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequence-specific transcription factors (TFs) are the key effectors of eukaryotic gene control and they regulate hundreds to thousands of downstream genes. Of particular interest are interactions in which a given TF regulates other TFs; these interactions define the TF regulatory networks (TRNs) that underlie cellular identity and major function. Chromatin accessibility depicts whether or not a DNA sequence is physically accessible and provides a direct measurement of transcriptional regulation. Benefiting from the accumulating chromatin accessibility data and deep learning advances, we developed a new computational method named DeepTFni to infer TRNs from the single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) data. By implementing a graph neural network, which is more suitable for network representation, DeepTFni shows outstanding performance in TRN inference, which it supports with limited numbers of cells. Furthermore, by applying DeepTFni we identified hub TFs in tissue development and tumorigenesis and revealed that many mixed-phenotype acute leukemia associated genes undergo a prominent alteration to the TRN while there is moderate difference in messenger RNA level. The DeepTFni webserver is easy to use and has provided the predicted TRNs for several popular cell lines. Transcription factor regulatory networks underlie major features of cellular identity and complex function such as pluripotency, development and differentiation. Li and colleagues develop a graph neural network to predict transcription factor regulatory networks based on single-cell ATAC-seq data.},
  archive      = {J_NATMI},
  author       = {Li, Hao and Sun, Yu and Hong, Hao and Huang, Xin and Tao, Huan and Huang, Qiya and Wang, Longteng and Xu, Kang and Gan, Jingbo and Chen, Hebing and Bo, Xiaochen},
  doi          = {10.1038/s42256-022-00469-5},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {389-400},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Inferring transcription factor regulatory networks from single-cell ATAC-seq data based on graph neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ad hoc learning of peptide fragmentation from mass spectra
enables an interpretable detection of phosphorylated and cross-linked
peptides. <em>NATMI</em>, <em>4</em>(4), 378–388. (<a
href="https://doi.org/10.1038/s42256-022-00467-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectrometry-based proteomics provides a holistic snapshot of the entire protein set of living cells on a molecular level. Currently, only a few deep learning approaches exist that involve peptide fragmentation spectra, which represent partial sequence information of proteins. Commonly, these approaches lack the ability to characterize less studied or even unknown patterns in spectra because of their use of explicit domain knowledge. Here, to elevate unrestricted learning from spectra, we introduce ‘ad hoc learning of fragmentation’ (AHLF), a deep learning model that is end-to-end trained on 19.2 million spectra from several phosphoproteomic datasets. AHLF is interpretable, and we show that peak-level feature importance values and pairwise interactions between peaks are in line with corresponding peptide fragments. We demonstrate our approach by detecting post-translational modifications, specifically protein phosphorylation based on only the fragmentation spectrum without a database search. AHLF increases the area under the receiver operating characteristic curve (AUC) by an average of 9.4\% on recent phosphoproteomic data compared with the current state of the art on this task. Furthermore, use of AHLF in rescoring search results increases the number of phosphopeptide identifications by a margin of up to 15.1\% at a constant false discovery rate. To show the broad applicability of AHLF, we use transfer learning to also detect cross-linked peptides, as used in protein structure analysis, with an AUC of up to 94\%. Fragmentation of peptides leaves characteristic patterns in mass spectrometry data, which can be used to identify protein sequences, but this method is challenging for mutated or modified sequences for which limited information exist. Altenburg et al. use an ad hoc learning approach to learn relevant patterns directly from unannotated fragmentation spectra.},
  archive      = {J_NATMI},
  author       = {Altenburg, Tom and Giese, Sven H. and Wang, Shengbo and Muth, Thilo and Renard, Bernhard Y.},
  doi          = {10.1038/s42256-022-00467-7},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {378-388},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Ad hoc learning of peptide fragmentation from mass spectra enables an interpretable detection of phosphorylated and cross-linked peptides},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Combinatorial optimization with physics-inspired graph
neural networks. <em>NATMI</em>, <em>4</em>(4), 367–377. (<a
href="https://doi.org/10.1038/s42256-022-00468-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combinatorial optimization problems are pervasive across science and industry. Modern deep learning tools are poised to solve these problems at unprecedented scales, but a unifying framework that incorporates insights from statistical physics is still outstanding. Here we demonstrate how graph neural networks can be used to solve combinatorial optimization problems. Our approach is broadly applicable to canonical NP-hard problems in the form of quadratic unconstrained binary optimization problems, such as maximum cut, minimum vertex cover, maximum independent set, as well as Ising spin glasses and higher-order generalizations thereof in the form of polynomial unconstrained binary optimization problems. We apply a relaxation strategy to the problem Hamiltonian to generate a differentiable loss function with which we train the graph neural network and apply a simple projection to integer variables once the unsupervised training process has completed. We showcase our approach with numerical results for the canonical maximum cut and maximum independent set problems. We find that the graph neural network optimizer performs on par or outperforms existing solvers, with the ability to scale beyond the state of the art to problems with millions of variables. Combinatorial optimization, the search for the minimum of an objective function within a finite but very large set of candidate solutions, finds many important and challenging applications in science and industry. A new graph neural network deep learning approach that incorporates concepts from statistical physics is used to develop a robust solver that can tackle a large class of NP-hard combinatorial optimization problems.},
  archive      = {J_NATMI},
  author       = {Schuetz, Martin J. A. and Brubaker, J. Kyle and Katzgraber, Helmut G.},
  doi          = {10.1038/s42256-022-00468-6},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {367-377},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Combinatorial optimization with physics-inspired graph neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multiscale simulations of complex systems by learning their
effective dynamics. <em>NATMI</em>, <em>4</em>(4), 359–366. (<a
href="https://doi.org/10.1038/s42256-022-00464-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predictive simulations of complex systems are essential for applications ranging from weather forecasting to drug design. The veracity of these predictions hinges on their capacity to capture effective system dynamics. Massively parallel simulations predict the system dynamics by resolving all spatiotemporal scales, often at a cost that prevents experimentation, while their findings may not allow for generalization. On the other hand, reduced-order models are fast but limited by the frequently adopted linearization of the system dynamics and the utilization of heuristic closures. Here we present a novel systematic framework that bridges large-scale simulations and reduced-order models to learn the effective dynamics of diverse, complex systems. The framework forms algorithmic alloys between nonlinear machine learning algorithms and the equation-free approach for modelling complex systems. Learning the effective dynamics deploys autoencoders to formulate a mapping between fine- and coarse-grained representations and evolves the latent space dynamics using recurrent neural networks. The algorithm is validated on benchmark problems, and we find that it outperforms state-of-the-art reduced-order models in terms of predictability, and large-scale simulations in terms of cost. Learning the effective dynamics is applicable to systems ranging from chemistry to fluid mechanics and reduces the computational effort by up to two orders of magnitude while maintaining the prediction accuracy of the full system dynamics. We argue that learning the effective dynamics provides a potent novel modality for accurately predicting complex systems. Accurate prediction of complex systems such as protein folding, weather forecasting and social dynamics is a core challenge in various disciplines. By fusing machine learning algorithms and classic equation-free methodologies, it is possible to reduce the computational effort of large-scale simulations by up to two orders of magnitude while maintaining the prediction accuracy of the full system dynamics.},
  archive      = {J_NATMI},
  author       = {Vlachas, Pantelis R. and Arampatzis, Georgios and Uhler, Caroline and Koumoutsakos, Petros},
  doi          = {10.1038/s42256-022-00464-w},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {359-366},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Multiscale simulations of complex systems by learning their effective dynamics},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Autoregressive neural-network wavefunctions for ab initio
quantum chemistry. <em>NATMI</em>, <em>4</em>(4), 351–358. (<a
href="https://doi.org/10.1038/s42256-022-00461-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, neural-network quantum states have emerged as powerful tools for the study of quantum many-body systems. Electronic structure calculations are one such canonical many-body problem that have attracted sustained research efforts spanning multiple decades, whilst only recently being attempted with neural-network quantum states. However, the complex non-local interactions and high sample complexity are substantial challenges that call for bespoke solutions. Here, we parameterize the electronic wavefunction with an autoregressive neural network that permits highly efficient and scalable sampling, whilst also embedding physical priors reflecting the structure of molecular systems without sacrificing expressibility. This allows us to perform electronic structure calculations on molecules with up to 30 spin orbitals—at least an order of magnitude more Slater determinants than previous applications of conventional neural-network quantum states—and we find that our ansatz can outperform the de facto gold-standard coupled-cluster methods even in the presence of strong quantum correlations. With a highly expressive neural network for which sampling is no longer a computational bottleneck, we conclude that the barriers to further scaling are not associated with the wavefunction ansatz itself, but rather are inherent to any variational Monte Carlo approach. To perform electronic structure calculations in quantum chemistry systems, methods are needed that are both accurate and scalable as the size of the molecule of interest increases. Barrett and colleagues employ an autoregressive neural-network ansatz that allows them to study larger molecules than previously attempted with neural-network quantum state approaches.},
  archive      = {J_NATMI},
  author       = {Barrett, Thomas D. and Malyshev, Aleksei and Lvovsky, A. I.},
  doi          = {10.1038/s42256-022-00461-z},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {351-358},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Autoregressive neural-network wavefunctions for ab initio quantum chemistry},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Experimental discovery of structure–property relationships
in ferroelectric materials via active learning. <em>NATMI</em>,
<em>4</em>(4), 341–350. (<a
href="https://doi.org/10.1038/s42256-022-00460-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emergent functionalities of structural and topological defects in ferroelectric materials underpin an extremely broad spectrum of applications ranging from domain wall electronics to high dielectric and electromechanical responses. Many of these functionalities have been discovered and quantified via local scanning probe microscopy methods. However, the search has until now been based on either trial and error, or using auxiliary information such as the topography or domain wall structure to identify potential objects of interest on the basis of the intuition of operator or pre-existing hypotheses, with subsequent manual exploration. Here we report the development and implementation of a machine learning framework that actively discovers relationships between local domain structure and polarization-switching characteristics in ferroelectric materials encoded in the hysteresis loop. The hysteresis loops and their scalar descriptors such as nucleation bias, coercive bias and the hysteresis loop area (or more complex functionals of hysteresis loop shape) and corresponding uncertainties are used to guide the discovery of these relationships via automated piezoresponse force microscopy and spectroscopy experiments. As such, this approach combines the power of machine learning methods to learn the correlative relationships between high-dimensional data, as well as human-based physics insights encoded into the acquisition function. For ferroelectric materials, this automated workflow demonstrates that the discovery path and sampling points of on- and off-field hysteresis loops are largely different, indicating that on- and off-field hysteresis loops are dominated by different mechanisms. The proposed approach is universal and can be applied to a broad range of modern imaging and spectroscopy methods ranging from other scanning probe microscopy modalities to electron microscopy and chemical imaging. An automated workflow for scanning probe microscopy, steered by an active learning framework, can efficiently explore relationships between local domain structure and physical properties. Such a capability is demonstrated in a piezoresponse force microscopy experiment to guide measurements of ferroelectric materials.},
  archive      = {J_NATMI},
  author       = {Liu, Yongtao and Kelley, Kyle P. and Vasudevan, Rama K. and Funakubo, Hiroshi and Ziatdinov, Maxim A. and Kalinin, Sergei V.},
  doi          = {10.1038/s42256-022-00460-0},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {341-350},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Experimental discovery of structure–property relationships in ferroelectric materials via active learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep-learning-based identification, tracking, pose
estimation and behaviour classification of interacting primates and mice
in complex environments. <em>NATMI</em>, <em>4</em>(4), 331–340. (<a
href="https://doi.org/10.1038/s42256-022-00477-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantification of behaviours of interest from video data is commonly used to study brain function, the effects of pharmacological interventions, and genetic alterations. Existing approaches lack the capability to analyse the behaviour of groups of animals in complex environments. We present a novel deep learning architecture for classifying individual and social animal behaviour—even in complex environments directly from raw video frames—that requires no intervention after initial human supervision. Our behavioural classifier is embedded in a pipeline (SIPEC) that performs segmentation, identification, pose-estimation and classification of complex behaviour, outperforming the state of the art. SIPEC successfully recognizes multiple behaviours of freely moving individual mice as well as socially interacting non-human primates in three dimensions, using data only from simple mono-vision cameras in home-cage set-ups. The use of deep neural networks for the automated analysis of behavioural videos has emerged as a tool in neuroscience, medicine and psychology. Marks and colleagues present a pipeline capable of tracking and identifying animals, as well as classifying individual and interacting animal behaviour in video recordings and even in complex environments.},
  archive      = {J_NATMI},
  author       = {Marks, Markus and Jin, Qiuhan and Sturman, Oliver and von Ziegler, Lukas and Kollmorgen, Sepp and von der Behrens, Wolfger and Mante, Valerio and Bohacek, Johannes and Yanik, Mehmet Fatih},
  doi          = {10.1038/s42256-022-00477-5},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {331-340},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep-learning-based identification, tracking, pose estimation and behaviour classification of interacting primates and mice in complex environments},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Barriers to academic data science research in the new realm
of algorithmic behaviour modification by digital platforms.
<em>NATMI</em>, <em>4</em>(4), 323–330. (<a
href="https://doi.org/10.1038/s42256-022-00475-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The era of behavioural big data has created new avenues for data science research, with many new contributions stemming from academic researchers. Yet data controlled by platforms have become increasingly difficult for academics to access. Platforms now routinely use algorithmic behaviour modification techniques to manipulate users’ behaviour, leaving academic researchers further isolated in conducting important data science and computational social science research. This isolation results from researchers’ lack of access to human behavioural data and, crucially, to both the data on machine behaviour that triggers and learns from the human data and the platform’s behaviour modification mechanisms. Given the impact of behaviour modification on individual and societal well-being, we discuss the consequences for data science knowledge creation, and encourage academic data scientists to take on new roles in producing research to promote (1) platform transparency and (2) informed public debate around the social purpose and function of digital platforms. Behavioural big data and algorithmic behaviour modification technologies controlled by commercial platforms have become difficult for academic researchers to access. Greene et al. describe barriers to academic research on such data and algorithms, and make a case for enhancing platform access and transparency.},
  archive      = {J_NATMI},
  author       = {Greene, Travis and Martens, David and Shmueli, Galit},
  doi          = {10.1038/s42256-022-00475-7},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {323-330},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Barriers to academic data science research in the new realm of algorithmic behaviour modification by digital platforms},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Designing a strong test for measuring true common-sense
reasoning. <em>NATMI</em>, <em>4</em>(4), 318–322. (<a
href="https://doi.org/10.1038/s42256-022-00478-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common-sense reasoning has recently emerged as an important test for artificial general intelligence, especially given the much-publicized successes of language representation models such as T5, BERT and GPT-3. Currently, typical benchmarks involve question answering tasks, but to test the full complexity of common-sense reasoning, more comprehensive evaluation methods that are grounded in theory should be developed.},
  archive      = {J_NATMI},
  author       = {Kejriwal, Mayank and Santos, Henrique and Mulvehill, Alice M. and McGuinness, Deborah L.},
  doi          = {10.1038/s42256-022-00478-4},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {318-322},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Designing a strong test for measuring true common-sense reasoning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ethics methods are required as part of reporting guidelines
for artificial intelligence in healthcare. <em>NATMI</em>,
<em>4</em>(4), 316–317. (<a
href="https://doi.org/10.1038/s42256-022-00479-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Sounderajah, Viknesh and McCradden, Melissa D. and Liu, Xiaoxuan and Rose, Sherri and Ashrafian, Hutan and Collins, Gary S. and Anderson, James and Bossuyt, Patrick M. and Moher, David and Darzi, Ara},
  doi          = {10.1038/s42256-022-00479-3},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {316-317},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Ethics methods are required as part of reporting guidelines for artificial intelligence in healthcare},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The perils of machine learning in designing new chemicals
and materials. <em>NATMI</em>, <em>4</em>(4), 314–315. (<a
href="https://doi.org/10.1038/s42256-022-00481-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Shankar, Sadasivan and Zare, Richard N.},
  doi          = {10.1038/s42256-022-00481-9},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {314-315},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The perils of machine learning in designing new chemicals and materials},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Tackling the perils of dual use in AI. <em>NATMI</em>,
<em>4</em>(4), 313. (<a
href="https://doi.org/10.1038/s42256-022-00484-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Considering the potential for unintended harmful applications of AI tools can lead to deeply concerning findings. An urgent question is how to achieve the right balance between keeping science open and preventing misuse or malicious repurposing.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00484-6},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {313},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Tackling the perils of dual use in AI},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Automatic strain sensor design via
active learning and data augmentation for soft machines. <em>NATMI</em>,
<em>4</em>(3), 312. (<a
href="https://doi.org/10.1038/s42256-022-00473-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Yang, Haitao and Li, Jiali and Lim, Kai Zhuo and Pan, Chuanji and Van Truong, Tien and Wang, Qian and Li, Kerui and Li, Shuo and Xiao, Xiao and Ding, Meng and Chen, Tianle and Liu, Xiaoli and Xie, Qian and Alvarado, Pablo Valdivia y. and Wang, Xiaonan and Chen, Po-Yen},
  doi          = {10.1038/s42256-022-00473-9},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {312},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Automatic strain sensor design via active learning and data augmentation for soft machines},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A transformer-based model to predict peptide–HLA class i
binding and optimize mutated peptides for vaccine design.
<em>NATMI</em>, <em>4</em>(3), 300–311. (<a
href="https://doi.org/10.1038/s42256-022-00459-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human leukocyte antigen (HLA) can recognize and bind foreign peptides to present them to specialized immune cells, then initiate an immune response. Computational prediction of the peptide and HLA (pHLA) binding can speed up immunogenic peptide screening and facilitate vaccine design. However, there is a lack of an automatic program to optimize mutated peptides with higher affinity to the target HLA allele. Here, to fill this gap, we develop the TransMut framework—composed of TransPHLA for pHLA binding prediction and an automatically optimized mutated peptides (AOMP) program—which can be generalized to any binding and mutation task of biomolecules. First, TransPHLA is developed by constructing a transformer-based model to predict pHLA binding, which is superior to 14 previous methods on pHLA binding prediction and neoantigen and human papilloma virus vaccine identification. For vaccine design, the AOMP program is then developed by exploiting the attention scores generated by TransPHLA to automatically optimize mutated peptides with higher affinity to the target HLA allele and with high homology to the source peptide. The proposed framework may automatically generate potential peptide vaccines for experimentalists. The human leukocyte antigen (HLA) complex plays an important role in building an immune response, but it is hard to predict which peptides will bind to it. Chu et al. present a transformer-based approach to identify which peptides have a high binding affinity to HLA, a task that can also be translated to other binding problems.},
  archive      = {J_NATMI},
  author       = {Chu, Yanyi and Zhang, Yan and Wang, Qiankun and Zhang, Lingfeng and Wang, Xuhong and Wang, Yanjing and Salahub, Dennis Russell and Xu, Qin and Wang, Jianmin and Jiang, Xue and Xiong, Yi and Wei, Dong-Qing},
  doi          = {10.1038/s42256-022-00459-7},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {300-311},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A transformer-based model to predict peptide–HLA class i binding and optimize mutated peptides for vaccine design},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Asymmetric predictive relationships across histone
modifications. <em>NATMI</em>, <em>4</em>(3), 288–299. (<a
href="https://doi.org/10.1038/s42256-022-00455-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Decoding the epigenomic landscapes in diverse tissues and cell types is fundamental to understanding molecular mechanisms underlying many essential cellular processes and human diseases. Recent advances in artificial intelligence provide new methods and strategies for imputing unknown epigenomes based on existing data, yet how to reveal the predictive relationships among epigenetic marks remains largely unexplored. Here we present a machine learning approach for epigenomic imputation and interpretation. Through dissection of the spatial contributions from six histone marks, we reveal the prevalent and asymmetric cross-prediction relationships among these marks. Meanwhile, our approach achieved high predictive performance on held-out prospective epigenomes and outperformed the state of the art. To facilitate future research, we further applied this approach to impute a total of 527 and 2,455 unavailable genome-wide histone modification signal tracks for the ENCODE3 and Roadmap datasets, respectively. Knowledge of the wide array of epigenomic signals provides biological insight into the state of a give cell type, but it is infeasible to experimentally characterize all possible types of epigenomic signal in the multitude of cell types in the human body. The authors present Ocelot, a machine learning approach for imputing cell-type-specific epigenomic signals along the genome.},
  archive      = {J_NATMI},
  author       = {Li, Hongyang and Guan, Yuanfang},
  doi          = {10.1038/s42256-022-00455-x},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {288-299},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Asymmetric predictive relationships across histone modifications},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Molecular contrastive learning of representations via graph
neural networks. <em>NATMI</em>, <em>4</em>(3), 279–287. (<a
href="https://doi.org/10.1038/s42256-022-00447-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular machine learning bears promise for efficient molecular property prediction and drug discovery. However, labelled molecule data can be expensive and time consuming to acquire. Due to the limited labelled data, it is a great challenge for supervised-learning machine learning models to generalize to the giant chemical space. Here we present MolCLR (Molecular Contrastive Learning of Representations via Graph Neural Networks), a self-supervised learning framework that leverages large unlabelled data (~10 million unique molecules). In MolCLR pre-training, we build molecule graphs and develop graph-neural-network encoders to learn differentiable representations. Three molecule graph augmentations are proposed: atom masking, bond deletion and subgraph removal. A contrastive estimator maximizes the agreement of augmentations from the same molecule while minimizing the agreement of different molecules. Experiments show that our contrastive learning framework significantly improves the performance of graph-neural-network encoders on various molecular property benchmarks including both classification and regression tasks. Benefiting from pre-training on the large unlabelled database, MolCLR even achieves state of the art on several challenging benchmarks after fine-tuning. In addition, further investigations demonstrate that MolCLR learns to embed molecules into representations that can distinguish chemically reasonable molecular similarities. Molecular representations are hard to design due to the large size of the chemical space, the amount of potentially important information in a molecular structure and the relatively low number of annotated molecules. Still, the quality of these representations is vital for computational models trying to predict molecular properties. Wang et al. present a contrastive learning approach to provide differentiable representations from unlabelled data.},
  archive      = {J_NATMI},
  author       = {Wang, Yuyang and Wang, Jianren and Cao, Zhonglin and Barati Farimani, Amir},
  doi          = {10.1038/s42256-022-00447-x},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {279-287},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Molecular contrastive learning of representations via graph neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Optimizing quantum annealing schedules with monte carlo tree
search enhanced with neural networks. <em>NATMI</em>, <em>4</em>(3),
269–278. (<a href="https://doi.org/10.1038/s42256-022-00446-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum annealing is a practical approach to approximately implement the adiabatic quantum computational model in a real-world setting. The goal of an adiabatic algorithm is to prepare the ground state of a problem-encoded Hamiltonian at the end of an annealing path. This is typically achieved by driving the dynamical evolution of a quantum system slowly to enforce adiabaticity. Properly optimized annealing schedules often considerably accelerate the computational process. Inspired by the recent success of deep reinforcement learning such as DeepMind’s AlphaZero, we propose a Monte Carlo tree search (MCTS) algorithm and its enhanced version boosted by neural networks—which we name QuantumZero (QZero)—to automate the design of annealing schedules in a hybrid quantum–classical framework. Both the MCTS and QZero algorithms perform remarkably well in discovering effective annealing schedules even when the annealing time is short for the 3-SAT examples considered in this study. Furthermore, the flexibility of neural networks allows us to apply transfer-learning techniques to boost QZero’s performance. We demonstrate in benchmark studies that MCTS and QZero perform more efficiently than other reinforcement learning algorithms in designing annealing schedules. Quantum annealers are computational models implemented on quantum hardware that can efficiently solve combinatorial optimization problems. Annealing schedules with enhanced performance can be discovered with a Monte Carlo tree search algorithm and an enhanced version incorporating value and policy neural networks—as inspired by DeepMind’s AlphaZero.},
  archive      = {J_NATMI},
  author       = {Chen, Yu-Qin and Chen, Yu and Lee, Chee-Kong and Zhang, Shengyu and Hsieh, Chang-Yu},
  doi          = {10.1038/s42256-022-00446-y},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {269-278},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Optimizing quantum annealing schedules with monte carlo tree search enhanced with neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Large pre-trained language models contain human-like biases
of what is right and wrong to do. <em>NATMI</em>, <em>4</em>(3),
258–268. (<a href="https://doi.org/10.1038/s42256-022-00458-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial writing is permeating our lives due to recent advances in large-scale, transformer-based language models (LMs) such as BERT, GPT-2 and GPT-3. Using them as pre-trained models and fine-tuning them for specific tasks, researchers have extended the state of the art for many natural language processing tasks and shown that they capture not only linguistic knowledge but also retain general knowledge implicitly present in the data. Unfortunately, LMs trained on unfiltered text corpora suffer from degenerated and biased behaviour. While this is well established, we show here that recent LMs also contain human-like biases of what is right and wrong to do, reflecting existing ethical and moral norms of society. We show that these norms can be captured geometrically by a ‘moral direction’ which can be computed, for example, by a PCA, in the embedding space. The computed ‘moral direction’ can rate the normativity (or non-normativity) of arbitrary phrases without explicitly training the LM for this task, reflecting social norms well. We demonstrate that computing the ’moral direction’ can provide a path for attenuating or even preventing toxic degeneration in LMs, showcasing this capability on the RealToxicityPrompts testbed. Large language models identify patterns in the relations between words and capture their relations in an embedding space. Schramowski and colleagues show that a direction in this space can be identified that separates ‘right’ and ‘wrong’ actions as judged by human survey participants.},
  archive      = {J_NATMI},
  author       = {Schramowski, Patrick and Turan, Cigdem and Andersen, Nico and Rothkopf, Constantin A. and Kersting, Kristian},
  doi          = {10.1038/s42256-022-00458-8},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {258-268},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Large pre-trained language models contain human-like biases of what is right and wrong to do},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Quantifying the spatial homogeneity of urban road networks
via graph neural networks. <em>NATMI</em>, <em>4</em>(3), 246–257. (<a
href="https://doi.org/10.1038/s42256-022-00462-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantifying the topological similarities of different parts of urban road networks enables us to understand urban growth patterns. Although conventional statistics provide useful information about the characteristics of either a single node’s direct neighbours or the entire network, such metrics fail to measure the similarities of subnetworks or capture local, indirect neighbourhood relationships. Here we propose a graph-based machine learning method to quantify the spatial homogeneity of subnetworks. We apply the method to 11,790 urban road networks across 30 cities worldwide to measure the spatial homogeneity of road networks within each city and across different cities. We find that intracity spatial homogeneity is highly associated with socioeconomic status indicators such as gross domestic product and population growth. Moreover, intercity spatial homogeneity values obtained by transferring the model across different cities reveal the intercity similarity of urban network structures originating in Europe, passed on to cities in the United States and Asia. The socioeconomic development and intercity similarity revealed using our method can be leveraged to understand and transfer insights between cities. It also enables us to address urban policy challenges including network planning in rapidly urbanizing areas and regional inequality. The spatial homogeneity of urban road networks can be quantified in a fine-grained manner with graph neural networks. This method is studied across 11,790 inner-city road networks around the world and can be used to study socioeconomic development and help with urban planning.},
  archive      = {J_NATMI},
  author       = {Xue, Jiawei and Jiang, Nan and Liang, Senwei and Pang, Qiyuan and Yabe, Takahiro and Ukkusuri, Satish V. and Ma, Jianzhu},
  doi          = {10.1038/s42256-022-00462-y},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {246-257},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Quantifying the spatial homogeneity of urban road networks via graph neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning functional properties of proteins with language
models. <em>NATMI</em>, <em>4</em>(3), 227–245. (<a
href="https://doi.org/10.1038/s42256-022-00457-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data-centric approaches have been used to develop predictive methods for elucidating uncharacterized properties of proteins; however, studies indicate that these methods should be further improved to effectively solve critical problems in biomedicine and biotechnology, which can be achieved by better representing the data at hand. Novel data representation approaches mostly take inspiration from language models that have yielded ground-breaking improvements in natural language processing. Lately, these approaches have been applied to the field of protein science and have displayed highly promising results in terms of extracting complex sequence–structure–function relationships. In this study we conducted a detailed investigation over protein representation learning by first categorizing/explaining each approach, subsequently benchmarking their performances on predicting: (1) semantic similarities between proteins, (2) ontology-based protein functions, (3) drug target protein families and (4) protein–protein binding affinity changes following mutations. We evaluate and discuss the advantages and disadvantages of each method over the benchmark results, source datasets and algorithms used, in comparison with classical model-driven approaches. Finally, we discuss current challenges and suggest future directions. We believe that the conclusions of this study will help researchers to apply machine/deep learning-based representation techniques to protein data for various predictive tasks, and inspire the development of novel methods. Deep learning methods have in recent years shown promising results in characterizing proteins and extracting complex sequence–structure–function relationships. This Analysis describes a benchmarking study to compare the performances and advantages of recent deep learning approaches in a range of protein prediction tasks.},
  archive      = {J_NATMI},
  author       = {Unsal, Serbulent and Atas, Heval and Albayrak, Muammer and Turhan, Kemal and Acar, Aybar C. and Doğan, Tunca},
  doi          = {10.1038/s42256-022-00457-9},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {227-245},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning functional properties of proteins with language models},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reusability report: Capturing properties of biological
objects and their relationships using graph neural networks.
<em>NATMI</em>, <em>4</em>(3), 222–226. (<a
href="https://doi.org/10.1038/s42256-022-00454-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Hong, Chenyang and Cao, Qin and Zhang, Zhenghao and Tsui, Stephen Kwok-Wing and Yip, Kevin Y.},
  doi          = {10.1038/s42256-022-00454-y},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {222-226},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Capturing properties of biological objects and their relationships using graph neural networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The transformational role of GPU computing and deep learning
in drug discovery. <em>NATMI</em>, <em>4</em>(3), 211–221. (<a
href="https://doi.org/10.1038/s42256-022-00463-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning has disrupted nearly every field of research, including those of direct importance to drug discovery, such as medicinal chemistry and pharmacology. This revolution has largely been attributed to the unprecedented advances in highly parallelizable graphics processing units (GPUs) and the development of GPU-enabled algorithms. In this Review, we present a comprehensive overview of historical trends and recent advances in GPU algorithms and discuss their immediate impact on the discovery of new drugs and drug targets. We also cover the state-of-the-art of deep learning architectures that have found practical applications in both early drug discovery and consequent hit-to-lead optimization stages, including the acceleration of molecular docking, the evaluation of off-target effects and the prediction of pharmacological properties. We conclude by discussing the impacts of GPU acceleration and deep learning models on the global democratization of the field of drug discovery that may lead to efficient exploration of the ever-expanding chemical universe to accelerate the discovery of novel medicines. GPUs, which are highly parallel computer processing units, were originally designed for graphics applications, but they have played an important role in accelerating the development of deep learning methods. In this Review, Pandey and colleagues summarize how GPUs have advanced machine learning in the field of drug discovery.},
  archive      = {J_NATMI},
  author       = {Pandey, Mohit and Fernandez, Michael and Gentile, Francesco and Isayev, Olexandr and Tropsha, Alexander and Stern, Abraham C. and Cherkasov, Artem},
  doi          = {10.1038/s42256-022-00463-x},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {211-221},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The transformational role of GPU computing and deep learning in drug discovery},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Biological underpinnings for lifelong learning machines.
<em>NATMI</em>, <em>4</em>(3), 196–210. (<a
href="https://doi.org/10.1038/s42256-022-00452-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biological organisms learn from interactions with their environment throughout their lifetime. For artificial systems to successfully act and adapt in the real world, it is desirable to similarly be able to learn on a continual basis. This challenge is known as lifelong learning, and remains to a large extent unsolved. In this Perspective article, we identify a set of key capabilities that artificial systems will need to achieve lifelong learning. We describe a number of biological mechanisms, both neuronal and non-neuronal, that help explain how organisms solve these challenges, and present examples of biologically inspired models and biologically plausible mechanisms that have been applied to artificial systems in the quest towards development of lifelong learning machines. We discuss opportunities to further our understanding and advance the state of the art in lifelong learning, aiming to bridge the gap between natural and artificial intelligence. It is an outstanding challenge to develop intelligent machines that can learn continually from interactions with their environment, throughout their lifetime. Kudithipudi et al. review neuronal and non-neuronal processes in organisms that address this challenge and discuss pathways to developing biologically inspired approaches for lifelong learning machines.},
  archive      = {J_NATMI},
  author       = {Kudithipudi, Dhireesha and Aguilar-Simon, Mario and Babb, Jonathan and Bazhenov, Maxim and Blackiston, Douglas and Bongard, Josh and Brna, Andrew P. and Chakravarthi Raja, Suraj and Cheney, Nick and Clune, Jeff and Daram, Anurag and Fusi, Stefano and Helfer, Peter and Kay, Leslie and Ketz, Nicholas and Kira, Zsolt and Kolouri, Soheil and Krichmar, Jeffrey L. and Kriegman, Sam and Levin, Michael and Madireddy, Sandeep and Manicka, Santosh and Marjaninejad, Ali and McNaughton, Bruce and Miikkulainen, Risto and Navratilova, Zaneta and Pandit, Tej and Parker, Alice and Pilly, Praveen K. and Risi, Sebastian and Sejnowski, Terrence J. and Soltoggio, Andrea and Soures, Nicholas and Tolias, Andreas S. and Urbina-Meléndez, Darío and Valero-Cuevas, Francisco J. and van de Ven, Gido M. and Vogelstein, Joshua T. and Wang, Felix and Weiss, Ron and Yanguas-Gil, Angel and Zou, Xinyun and Siegelmann, Hava},
  doi          = {10.1038/s42256-022-00452-0},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {196-210},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Biological underpinnings for lifelong learning machines},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Data-driven design of soft sensors. <em>NATMI</em>,
<em>4</em>(3), 194–195. (<a
href="https://doi.org/10.1038/s42256-022-00453-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the performance of a tactile sensor from its composition and morphology is nearly impossible with traditional computational approaches. Machine learning can not only predict device-level performance, but also recommend new material compositions for soft machine applications.},
  archive      = {J_NATMI},
  author       = {Glazar, James T. and Shenoy, Vivek B.},
  doi          = {10.1038/s42256-022-00453-z},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {194-195},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Data-driven design of soft sensors},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Half a decade of graph convolutional networks.
<em>NATMI</em>, <em>4</em>(3), 192–193. (<a
href="https://doi.org/10.1038/s42256-022-00466-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Graph convolutional networks have become a popular tool for learning with graphs and networks. We reflect on the reasons behind the success story.},
  archive      = {J_NATMI},
  author       = {Haghir Chehreghani, Mostafa},
  doi          = {10.1038/s42256-022-00466-8},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {192-193},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Half a decade of graph convolutional networks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Dual use of artificial-intelligence-powered drug discovery.
<em>NATMI</em>, <em>4</em>(3), 189–191. (<a
href="https://doi.org/10.1038/s42256-022-00465-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An international security conference explored how artificial intelligence (AI) technologies for drug discovery could be misused for de novo design of biochemical weapons. A thought experiment evolved into a computational proof.},
  archive      = {J_NATMI},
  author       = {Urbina, Fabio and Lentzos, Filippa and Invernizzi, Cédric and Ekins, Sean},
  doi          = {10.1038/s42256-022-00465-9},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {189-191},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Dual use of artificial-intelligence-powered drug discovery},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The graph connection. <em>NATMI</em>, <em>4</em>(3),
187–188. (<a href="https://doi.org/10.1038/s42256-022-00476-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The number of graph neural network papers in this journal has grown as the field matures. We take a closer look at some of the scientific applications.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00476-6},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {187-188},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The graph connection},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Design of potent antimalarials with generative chemistry.
<em>NATMI</em>, <em>4</em>(2), 180–186. (<a
href="https://doi.org/10.1038/s42256-022-00448-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in generative modelling allow designing novel compounds through deep neural networks. One such neural network model, JT-VAE (the Junction Tree Variational Auto-Encoder), excels at proposing chemically valid structures. Here, on the basis of JT-VAE, we built a generative modelling approach, JAEGER, for finding novel chemical matter with desired bioactivity. Using JAEGER, we designed compounds to inhibit malaria. To prioritize the compounds for synthesis, we used the in-house pQSAR (Profile-QSAR) program, a massively multitask bioactivity model based on 12,000 Novartis assays. On the basis of pQSAR activity predictions, we selected, synthesized and experimentally profiled two compounds. Both compounds exhibited low nanomolar activity in a malaria proliferation assay as well as a biochemical assay measuring activity against PI(4)K, which is an essential kinase that regulates intracellular development in malaria. The compounds also showed low activity in a cytotoxicity assay. Our findings show that JAEGER is a viable approach for finding novel active compounds for drug discovery. Tropical diseases, such as malaria, can develop resistance to specific drugs. Godinez and colleagues present here a generative design approach to find new anti-malarial drugs to circumvent this resistance.},
  archive      = {J_NATMI},
  author       = {Godinez, William J. and Ma, Eric J. and Chao, Alexander T. and Pei, Luying and Skewes-Cox, Peter and Canham, Stephen M. and Jenkins, Jeremy L. and Young, Joseph M. and Martin, Eric J. and Guiguemde, W. Armand},
  doi          = {10.1038/s42256-022-00448-w},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {180-186},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Design of potent antimalarials with generative chemistry},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Versatile modular neural locomotion control with fast
learning. <em>NATMI</em>, <em>4</em>(2), 169–179. (<a
href="https://doi.org/10.1038/s42256-022-00444-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Legged robots have significant potential to operate in unstructured environments. The design of locomotion control is, however, still challenging. Currently, controllers must be manually designed for specific robots and tasks, or automatically designed via machine learning methods that require long training times and yield large opaque controllers. Drawing inspiration from animal locomotion, we propose a simple yet versatile modular neural control structure with fast learning. The key advantages of our approach are that behaviour-specific control modules can be added incrementally to obtain increasingly complex emergent locomotion behaviours, and that neural connections can be quickly and automatically learned. In a series of experiments, we show how eight modules can be quickly learned and added to a base control module to obtain emergent adaptive behaviours allowing a hexapod robot to navigate in complex environments. We also show that modules can be added and removed during operation without affecting the functionality of the remaining controller. Finally, the controller is successfully demonstrated on a physical robot. Taken together, our study reveals a significant step towards fast automatic design of versatile neural locomotion control. Controllers for robotic locomotion patterns deal with complex interactions and need to be carefully designed or extensively trained. Thor and Manoonpong present a modular approach for neural pattern generators that allows incremental and fast learning.},
  archive      = {J_NATMI},
  author       = {Thor, Mathias and Manoonpong, Poramate},
  doi          = {10.1038/s42256-022-00444-0},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {169-179},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Versatile modular neural locomotion control with fast learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Simultaneous dimensionality reduction and integration for
single-cell ATAC-seq data using deep learning. <em>NATMI</em>,
<em>4</em>(2), 162–168. (<a
href="https://doi.org/10.1038/s42256-022-00443-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in single-cell technologies enable the routine interrogation of chromatin accessibility for tens of thousands of single cells, elucidating gene regulatory processes at an unprecedented resolution. Meanwhile, size, sparsity and high dimensionality of the resulting data continue to pose challenges for its computational analysis, and specifically the integration of data from different sources. We have developed a dedicated computational approach: a variational auto-encoder using a noise model specifically designed for single-cell ATAC-seq (assay for transposase-accessible chromatin with high-throughput sequencing) data, which facilitates simultaneous dimensionality reduction and batch correction via an adversarial learning strategy. We showcase its benefits for detailed cell-type characterization on individual real and simulated datasets as well as for integrating multiple complex datasets. High-throughput single-cell sequencing data can provide valuable biological insights but are computationally challenging to analyse due to the dimensionality of the data and batch-specific biases. Kopp and colleagues have developed a variational auto-encoder-based method using a novel loss function for simultaneous batch correction and dimensionality reduction.},
  archive      = {J_NATMI},
  author       = {Kopp, Wolfgang and Akalin, Altuna and Ohler, Uwe},
  doi          = {10.1038/s42256-022-00443-1},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {162-168},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Simultaneous dimensionality reduction and integration for single-cell ATAC-seq data using deep learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Autoencoders on field-programmable gate arrays for
real-time, unsupervised new physics detection at 40 MHz at the large
hadron collider. <em>NATMI</em>, <em>4</em>(2), 154–161. (<a
href="https://doi.org/10.1038/s42256-022-00441-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To study the physics of fundamental particles and their interactions, the Large Hadron Collider was constructed at CERN, where protons collide to create new particles measured by detectors. Collisions occur at a frequency of 40 MHz, and with an event size of roughly 1 MB it is impossible to read out and store the generated amount of data from the detector and therefore a multi-tiered, real-time filtering system is required. In this paper, we show how to adapt and deploy deep-learning-based autoencoders for the unsupervised detection of new physics signatures in the challenging environment of a real-time event selection system at the Large Hadron Collider. The first-stage filter, implemented on custom electronics, decides within a few microseconds whether an event should be kept or discarded. At this stage, the rate is reduced from 40 MHz to about 100 kHz. We demonstrate the deployment of an unsupervised selection algorithm on this custom electronics, running in as little as 80 ns and enhancing the signal-over-background ratio by three orders of magnitude. This work enables the practical deployment of these networks during the next data-taking campaign of the Large Hadron Collider. The Large Hadron Collider produces 40 million collision events per second, most of which need to be discarded by a real-time filtering system. Unsupervised deep learning algorithms are developed and deployed on custom electronics to search for rare events indicating new physics, rather than for specific events led by theory.},
  archive      = {J_NATMI},
  author       = {Govorkova, Ekaterina and Puljak, Ema and Aarrestad, Thea and James, Thomas and Loncar, Vladimir and Pierini, Maurizio and Pol, Adrian Alan and Ghielmetti, Nicolò and Graczyk, Maksymilian and Summers, Sioni and Ngadiuba, Jennifer and Nguyen, Thong Q. and Duarte, Javier and Wu, Zhenbin},
  doi          = {10.1038/s42256-022-00441-3},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {154-161},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Autoencoders on field-programmable gate arrays for real-time, unsupervised new physics detection at 40 MHz at the large hadron collider},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). When and how convolutional neural networks generalize to
out-of-distribution category–viewpoint combinations. <em>NATMI</em>,
<em>4</em>(2), 146–153. (<a
href="https://doi.org/10.1038/s42256-021-00437-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Object recognition and viewpoint estimation lie at the heart of visual understanding. Recent studies have suggested that convolutional neural networks (CNNs) fail to generalize to out-of-distribution (OOD) category–viewpoint combinations, that is, combinations not seen during training. Here we investigate when and how such OOD generalization may be possible by evaluating CNNs trained to classify both object category and three-dimensional viewpoint on OOD combinations, and identifying the neural mechanisms that facilitate such OOD generalization. We show that increasing the number of in-distribution combinations (data diversity) substantially improves generalization to OOD combinations, even with the same amount of training data. We compare learning category and viewpoint in separate and shared network architectures, and observe starkly different trends on in-distribution and OOD combinations, that is, while shared networks are helpful in distribution, separate networks significantly outperform shared ones at OOD combinations. Finally, we demonstrate that such OOD generalization is facilitated by the neural mechanism of specialization, that is, the emergence of two types of neuron—neurons selective to category and invariant to viewpoint, and vice versa. The combination of object recognition and viewpoint estimation is essential for visual understanding. However, convolutional neural networks often fail to generalize to object category–viewpoint combinations that were not seen during training. The authors investigate the impact of data diversity and architectural choices on the capability of generalizing to out-of-distribution combinations.},
  archive      = {J_NATMI},
  author       = {Madan, Spandan and Henry, Timothy and Dozier, Jamell and Ho, Helen and Bhandari, Nishchal and Sasaki, Tomotake and Durand, Frédo and Pfister, Hanspeter and Boix, Xavier},
  doi          = {10.1038/s42256-021-00437-5},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {146-153},
  shortjournal = {Nat. Mach. Intell.},
  title        = {When and how convolutional neural networks generalize to out-of-distribution category–viewpoint combinations},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A soft thumb-sized vision-based sensor with accurate
all-round force perception. <em>NATMI</em>, <em>4</em>(2), 135–145. (<a
href="https://doi.org/10.1038/s42256-021-00439-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vision-based haptic sensors have emerged as a promising approach to robotic touch due to affordable high-resolution cameras and successful computer vision techniques; however, their physical design and the information they provide do not yet meet the requirements of real applications. We present a robust, soft, low-cost, vision-based, thumb-sized three-dimensional haptic sensor named Insight, which continually provides a directional force-distribution map over its entire conical sensing surface. Constructed around an internal monocular camera, the sensor has only a single layer of elastomer over-moulded on a stiff frame to guarantee sensitivity, robustness and soft contact. Furthermore, Insight uniquely combines photometric stereo and structured light using a collimator to detect the three-dimensional deformation of its easily replaceable flexible outer shell. The force information is inferred by a deep neural network that maps images to the spatial distribution of three-dimensional contact force (normal and shear). Insight has an overall spatial resolution of 0.4 mm, a force magnitude accuracy of around 0.03 N and a force direction accuracy of around five degrees over a range of 0.03–2 N for numerous distinct contacts with varying contact area. The presented hardware and software design concepts can be transferred to a wide variety of robot parts. High-fidelity haptic sensors with three-dimensional sensing surfaces are needed to advance dexterous robotic manipulation. The authors develop a sensor design that offers accurate force sensation across a three-dimensional surface while being robust, low-cost and easy to fabricate.},
  archive      = {J_NATMI},
  author       = {Sun, Huanbo and Kuchenbecker, Katherine J. and Martius, Georg},
  doi          = {10.1038/s42256-021-00439-3},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {135-145},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A soft thumb-sized vision-based sensor with accurate all-round force perception},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Geometry-enhanced molecular representation learning for
property prediction. <em>NATMI</em>, <em>4</em>(2), 127–134. (<a
href="https://doi.org/10.1038/s42256-021-00438-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Effective molecular representation learning is of great importance to facilitate molecular property prediction. Recent advances for molecular representation learning have shown great promise in applying graph neural networks to model molecules. Moreover, a few recent studies design self-supervised learning methods for molecular representation to address insufficient labelled molecules; however, these self-supervised frameworks treat the molecules as topological graphs without fully utilizing the molecular geometry information. The molecular geometry, also known as the three-dimensional spatial structure of a molecule, is critical for determining molecular properties. To this end, we propose a novel geometry-enhanced molecular representation learning method (GEM). The proposed GEM has a specially designed geometry-based graph neural network architecture as well as several dedicated geometry-level self-supervised learning strategies to learn the molecular geometry knowledge. We compare GEM with various state-of-the-art baselines on different benchmarks and show that it can considerably outperform them all, demonstrating the superiority of the proposed method. Molecules are often represented as topological graphs while their true three-dimensional geometry contains a lot of valuable information. Xiaomin Fang and colleagues present a self-supervised molecule representation method that uses this geometric data in graph neural networks to predict a range of molecular properties.},
  archive      = {J_NATMI},
  author       = {Fang, Xiaomin and Liu, Lihang and Lei, Jieqiong and He, Donglong and Zhang, Shanzhuo and Zhou, Jingbo and Wang, Fan and Wu, Hua and Wang, Haifeng},
  doi          = {10.1038/s42256-021-00438-4},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {127-134},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Geometry-enhanced molecular representation learning for property prediction},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cell type annotation of single-cell chromatin accessibility
data via supervised bayesian embedding. <em>NATMI</em>, <em>4</em>(2),
116–126. (<a href="https://doi.org/10.1038/s42256-021-00432-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in single-cell technologies have enabled the characterization of epigenomic heterogeneity at the cellular level. Computational methods for automatic cell type annotation are urgently needed given the exponential growth in the number of cells. In particular, annotation of single-cell chromatin accessibility sequencing (scCAS) data, which can capture the chromatin regulatory landscape that governs transcription in each cell type, has not been fully investigated. Here we propose EpiAnno, a probabilistic generative model integrated with a Bayesian neural network, to annotate scCAS data automatically in a supervised manner. We systematically validate the superior performance of EpiAnno for both intra- and inter-dataset annotation on various datasets. We further demonstrate the advantages of EpiAnno for interpretable embedding and biological implications via expression enrichment analysis, partitioned heritability analysis, enhancer identification, cis-coaccessibility analysis and pathway enrichment analysis. In addition, we show that EpiAnno has the potential to reveal cell type-specific motifs and facilitate scCAS data simulation. The investigation of single-cell epigenomics with technologies such as single-cell chromatin accessibility sequencing (scCAS) presents an opportunity to expand the understanding of gene regulation at the cellular level. The authors develop a probabilistic generative model to better characterize cell heterogeneity and accurately annotate the cell type of scCAS data.},
  archive      = {J_NATMI},
  author       = {Chen, Xiaoyang and Chen, Shengquan and Song, Shuang and Gao, Zijing and Hou, Lin and Zhang, Xuegong and Lv, Hairong and Jiang, Rui},
  doi          = {10.1038/s42256-021-00432-w},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {116-126},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Cell type annotation of single-cell chromatin accessibility data via supervised bayesian embedding},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Stable learning establishes some common ground between
causal inference and machine learning. <em>NATMI</em>, <em>4</em>(2),
110–115. (<a href="https://doi.org/10.1038/s42256-022-00445-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Causal inference has recently attracted substantial attention in the machine learning and artificial intelligence community. It is usually positioned as a distinct strand of research that can broaden the scope of machine learning from predictive modelling to intervention and decision-making. In this Perspective, however, we argue that ideas from causality can also be used to improve the stronghold of machine learning, predictive modelling, if predictive stability, explainability and fairness are important. With the aim of bridging the gap between the tradition of precise modelling in causal inference and black-box approaches from machine learning, stable learning is proposed and developed as a source of common ground. This Perspective clarifies a source of risk for machine learning models and discusses the benefits of bringing causality into learning. We identify the fundamental problems addressed by stable learning, as well as the latest progress from both causal inference and learning perspectives, and we discuss relationships with explainability and fairness problems. Machine learning performs well at predictive modelling based on statistical correlations, but for high-stakes applications, more robust, explainable and fair approaches are required. Cui and Athey discuss the benefits of bringing causal inference into machine learning, presenting a stable learning approach.},
  archive      = {J_NATMI},
  author       = {Cui, Peng and Athey, Susan},
  doi          = {10.1038/s42256-022-00445-z},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {110-115},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Stable learning establishes some common ground between causal inference and machine learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Responsible artificial intelligence in agriculture requires
systemic understanding of risks and externalities. <em>NATMI</em>,
<em>4</em>(2), 104–109. (<a
href="https://doi.org/10.1038/s42256-022-00440-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global agriculture is poised to benefit from the rapid advance and diffusion of artificial intelligence (AI) technologies. AI in agriculture could improve crop management and agricultural productivity through plant phenotyping, rapid diagnosis of plant disease, efficient application of agrochemicals and assistance for growers with location-relevant agronomic advice. However, the ramifications of machine learning (ML) models, expert systems and autonomous machines for farms, farmers and food security are poorly understood and under-appreciated. Here, we consider systemic risk factors of AI in agriculture. Namely, we review risks relating to interoperability, reliability and relevance of agricultural data, unintended socio-ecological consequences resulting from ML models optimized for yields, and safety and security concerns associated with deployment of ML platforms at scale. As a response, we suggest risk-mitigation measures, including inviting rural anthropologists and applied ecologists into the technology design process, applying frameworks for responsible and human-centred innovation, setting data cooperatives for improved data transparency and ownership rights, and initial deployment of agricultural AI in digital sandboxes. Machine learning applications in agriculture can bring many benefits in crop management and productivity. However, to avoid harmful effects of a new round of technological modernization, fuelled by AI, a thorough risk assessment is required, to review and mitigate risks such as unintended socio-ecological consequences and security concerns associated with applying machine learning models at scale.},
  archive      = {J_NATMI},
  author       = {Tzachor, Asaf and Devare, Medha and King, Brian and Avin, Shahar and Ó hÉigeartaigh, Seán},
  doi          = {10.1038/s42256-022-00440-4},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {104-109},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Responsible artificial intelligence in agriculture requires systemic understanding of risks and externalities},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Potent antimalarial drugs with validated activities.
<em>NATMI</em>, <em>4</em>(2), 102–103. (<a
href="https://doi.org/10.1038/s42256-022-00451-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug resistance in tropical diseases such as malaria requires constant improvement and development of new drugs. To find potential candidates, generative machine learning methods that can search for novel bioactive molecules can be employed.},
  archive      = {J_NATMI},
  author       = {Winkler, David A.},
  doi          = {10.1038/s42256-022-00451-1},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {102-103},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Potent antimalarial drugs with validated activities},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Human autonomy in the age of artificial intelligence.
<em>NATMI</em>, <em>4</em>(2), 99–101. (<a
href="https://doi.org/10.1038/s42256-022-00449-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current AI policy recommendations differ on what the risks to human autonomy are. To systematically address risks to autonomy, we need to confront the complexity of the concept itself and adapt governance solutions accordingly.},
  archive      = {J_NATMI},
  author       = {Prunkl, Carina},
  doi          = {10.1038/s42256-022-00449-9},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {99-101},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Human autonomy in the age of artificial intelligence},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). FDA fosters innovative approaches in research, resources and
collaboration. <em>NATMI</em>, <em>4</em>(2), 97–98. (<a
href="https://doi.org/10.1038/s42256-022-00450-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Gallas, Brandon D. and Badano, Aldo and Dudgeon, Sarah and Elfer, Katherine and Garcia, Victor and Lennerz, Jochen K. and Myers, Kyle and Petrick, Nicholas and Margerrison, Ed},
  doi          = {10.1038/s42256-022-00450-2},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {97-98},
  shortjournal = {Nat. Mach. Intell.},
  title        = {FDA fosters innovative approaches in research, resources and collaboration},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Safe driving cars. <em>NATMI</em>, <em>4</em>(2), 95–96. (<a
href="https://doi.org/10.1038/s42256-022-00456-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autonomous vehicle technologies need to be safer than humans by a considerable margin before they can be truly self-driving. But they can provide substantial benefits as assistive driving technology already today — provided their limitations are properly communicated.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00456-w},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {95-96},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Safe driving cars},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Automatic strain sensor design via active learning and data
augmentation for soft machines. <em>NATMI</em>, <em>4</em>(1), 84–94.
(<a href="https://doi.org/10.1038/s42256-021-00434-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emerging soft machines require high-performance strain sensors to achieve closed-loop feedback control. Machine learning is a versatile tool to uncover complex correlations between fabrication recipes and sensor performance at the device level. Here a three-stage machine learning framework was realized for a high-accuracy prediction model capable of automating the design of strain sensors. First, a support-vector machine classifier was trained by using 351 compositions of various nanomaterials. Second, through 12 active learning loops, 125 strain sensors were stagewise fabricated to enrich the multidimensional dataset. Third, to address the challenge of data scarcity, data augmentation was implemented to synthesize &amp;gt;10,000 virtual data points, followed by genetic algorithm-based selection to optimize the model’s prediction accuracy. Several data-driven design rules for piezoresistive nanocomposites were generalized and validated by in situ microscopic studies. As final demonstrations, model-suggested strain sensors can be integrated into/onto various soft machines to endow them with real-time strain-sensing capabilities. Piezoresistors can be used in strain sensors for soft machines, but the traditional design process relies on intuition and human ingenuity alone. Haitao Yang and colleagues present a method built on genetic algorithms and other machine learning methods to design and fabricate strain sensors with improved capabilities.},
  archive      = {J_NATMI},
  author       = {Yang, Haitao and Li, Jiali and Lim, Kai Zhuo and Pan, Chuanji and Van Truong, Tien and Wang, Qian and Li, Kerui and Li, Shuo and Xiao, Xiao and Ding, Meng and Chen, Tianle and Liu, Xiaoli and Xie, Qian and Alvarado, Pablo Valdivia y. and Wang, Xiaonan and Chen, Po-Yen},
  doi          = {10.1038/s42256-021-00434-8},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {84-94},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Automatic strain sensor design via active learning and data augmentation for soft machines},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hierarchical deep reinforcement learning reveals a modular
mechanism of cell movement. <em>NATMI</em>, <em>4</em>(1), 73–83. (<a
href="https://doi.org/10.1038/s42256-021-00431-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Time-lapse images of cells and tissues contain rich information about dynamic cell behaviours, which reflect the underlying processes of proliferation, differentiation and morphogenesis. However, we lack computational tools for effective inference. Here we exploit deep reinforcement learning (DRL) to infer cell–cell interactions and collective cell behaviours in tissue morphogenesis from three-dimensional (3D) time-lapse images. We use hierarchical DRL (HDRL), known for multiscale learning and data efficiency, to examine cell migrations based on images with a ubiquitous nuclear label and simple rules formulated from empirical statistics of the images. When applied to Caenorhabditis elegans embryogenesis, HDRL reveals a multiphase, modular organization of cell movement. Imaging with additional cellular markers confirms the modular organization as a novel migration mechanism, which we term sequential rosettes. Furthermore, HDRL forms a transferable model that successfully differentiates sequential rosettes-based migration from others. Our study demonstrates a powerful approach to infer the underlying biology from time-lapse imaging without prior knowledge. Reinforcement learning has shown remarkable success in areas such as game-playing and protein folding, but it has not been extensively explored in modelling cell behaviour. The authors develop an approach that uses deep reinforcement learning to uncover collective cell behaviours and the underlying mechanism of cell migration from 3D time-lapse images of tissues.},
  archive      = {J_NATMI},
  author       = {Wang, Zi and Xu, Yichi and Wang, Dali and Yang, Jiawei and Bao, Zhirong},
  doi          = {10.1038/s42256-021-00431-x},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {73-83},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Hierarchical deep reinforcement learning reveals a modular mechanism of cell movement},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Neurons learn by predicting future activity. <em>NATMI</em>,
<em>4</em>(1), 62–72. (<a
href="https://doi.org/10.1038/s42256-021-00430-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how the brain learns may lead to machines with human-like intellectual capacities. It was previously proposed that the brain may operate on the principle of predictive coding. However, it is still not well understood how a predictive system could be implemented in the brain. Here we demonstrate that the ability of a single neuron to predict its future activity may provide an effective learning mechanism. Interestingly, this predictive learning rule can be derived from a metabolic principle, whereby neurons need to minimize their own synaptic activity (cost) while maximizing their impact on local blood supply by recruiting other neurons. We show how this mathematically derived learning rule can provide a theoretical connection between diverse types of brain-inspired algorithm, thus offering a step towards the development of a general theory of neuronal learning. We tested this predictive learning rule in neural network simulations and in data recorded from awake animals. Our results also suggest that spontaneous brain activity provides ‘training data’ for neurons to learn to predict cortical dynamics. Thus, the ability of a single neuron to minimize surprise—that is, the difference between actual and expected activity—could be an important missing element to understand computation in the brain. In artificial neural networks, a typical neuron generally performs a simple summation of inputs. Using computational and electrophysiological data, the authors show that a single neuron predicts its future activity. Neurons that predict their own future responses are a potential mechanism for learning in the brain and neural networks.},
  archive      = {J_NATMI},
  author       = {Luczak, Artur and McNaughton, Bruce L. and Kubo, Yoshimasa},
  doi          = {10.1038/s42256-021-00430-y},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {62-72},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Neurons learn by predicting future activity},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting myocardial infarction through retinal scans and
minimal personal information. <em>NATMI</em>, <em>4</em>(1), 55–61. (<a
href="https://doi.org/10.1038/s42256-021-00427-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In ophthalmologic practice, retinal images are routinely obtained to diagnose and monitor primary eye diseases and systemic conditions affecting the eye, such as diabetic retinopathy. Recent studies have shown that biomarkers on retinal images, for example, retinal blood vessel density or tortuosity, are associated with cardiac function and may identify patients at risk of coronary artery disease. In this work we investigate the use of retinal images, alongside relevant patient metadata, to estimate left ventricular mass and left ventricular end-diastolic volume, and subsequently, predict incident myocardial infarction. We trained a multichannel variational autoencoder and a deep regressor model to estimate left ventricular mass (4.4 (–32.30, 41.1) g) and left ventricular end-diastolic volume (3.02 (–53.45, 59.49) ml) and predict risk of myocardial infarction (AUC = 0.80 ± 0.02, sensitivity = 0.74 ± 0.02, specificity = 0.71 ± 0.03) using just the retinal images and demographic data. Our results indicate that one could identify patients at high risk of future myocardial infarction from retinal imaging available in every optician and eye clinic. Routine eye clinic imaging could help screen patients with cardiovascular risk as studies indicate strong associations between biomarkers in the retina and the heart. This potential is supported by a multimodal study, employing a deep learning model, that can infer cardiac functional indices based on retinal images and demographic data.},
  archive      = {J_NATMI},
  author       = {Diaz-Pinto, Andres and Ravikumar, Nishant and Attar, Rahman and Suinesiaputra, Avan and Zhao, Yitian and Levelt, Eylem and Dall’Armellina, Erica and Lorenzi, Marco and Chen, Qingyu and Keenan, Tiarnan D. L. and Agrón, Elvira and Chew, Emily Y. and Lu, Zhiyong and Gale, Chris P. and Gale, Richard P. and Plein, Sven and Frangi, Alejandro F.},
  doi          = {10.1038/s42256-021-00427-7},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {55-61},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting myocardial infarction through retinal scans and minimal personal information},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Interpreting neural networks for biological sequences by
learning stochastic masks. <em>NATMI</em>, <em>4</em>(1), 41–54. (<a
href="https://doi.org/10.1038/s42256-021-00428-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequence-based neural networks can learn to make accurate predictions from large biological datasets, but model interpretation remains challenging. Many existing feature attribution methods are optimized for continuous rather than discrete input patterns and assess individual feature importance in isolation, making them ill-suited for interpreting nonlinear interactions in molecular sequences. Here, building on work in computer vision and natural language processing, we developed an approach based on deep learning—scrambler networks—wherein the most important sequence positions are identified with learned input masks. Scramblers learn to predict position-specific scoring matrices where unimportant nucleotides or residues are scrambled by raising their entropy. We apply scramblers to interpret the effects of genetic variants, uncover nonlinear interactions between cis-regulatory elements, explain binding specificity for protein–protein interactions, and identify structural determinants of de novo-designed proteins. We show that scramblers enable efficient attribution across large datasets and result in high-quality explanations, often outperforming state-of-the-art methods. Neural networks have become a useful approach for predicting biological function from large-scale DNA and protein sequence data; however, researchers are often unable to understand which features in an input sequence are important for a given model, making it difficult to explain predictions in terms of known biology. The authors introduce scrambler networks, a feature attribution method tailor-made for discrete sequence inputs.},
  archive      = {J_NATMI},
  author       = {Linder, Johannes and La Fleur, Alyssa and Chen, Zibo and Ljubetič, Ajasja and Baker, David and Kannan, Sreeram and Seelig, Georg},
  doi          = {10.1038/s42256-021-00428-6},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {41-54},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Interpreting neural networks for biological sequences by learning stochastic masks},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Generalized radiograph representation learning via
cross-supervision between images and free-text radiology reports.
<em>NATMI</em>, <em>4</em>(1), 32–40. (<a
href="https://doi.org/10.1038/s42256-021-00425-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pre-training lays the foundation for recent successes in radiograph analysis supported by deep learning. It learns transferable image representations by conducting large-scale fully- or self-supervised learning on a source domain; however, supervised pre-training requires a complex and labour-intensive two-stage human-assisted annotation process, whereas self-supervised learning cannot compete with the supervised paradigm. To tackle these issues, we propose a cross-supervised methodology called reviewing free-text reports for supervision (REFERS), which acquires free supervision signals from the original radiology reports accompanying the radiographs. The proposed approach employs a vision transformer and is designed to learn joint representations from multiple views within every patient study. REFERS outperforms its transfer learning and self-supervised learning counterparts on four well-known X-ray datasets under extremely limited supervision. Moreover, REFERS even surpasses methods based on a source domain of radiographs with human-assisted structured labels; it therefore has the potential to replace canonical pre-training methodologies. To train machine learning models for medical imaging, large amounts of training data are needed. Zhou and colleagues instead propose a method of weak supervision which uses the information of radiology reports to learn visual features without the need for expert labelling.},
  archive      = {J_NATMI},
  author       = {Zhou, Hong-Yu and Chen, Xiaoyu and Zhang, Yinghao and Luo, Ruibang and Wang, Liansheng and Yu, Yizhou},
  doi          = {10.1038/s42256-021-00425-9},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {32-40},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Generalized radiograph representation learning via cross-supervision between images and free-text radiology reports},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Optimizing molecules using efficient queries from property
evaluations. <em>NATMI</em>, <em>4</em>(1), 21–31. (<a
href="https://doi.org/10.1038/s42256-021-00422-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning-based methods have shown potential for optimizing existing molecules with more desirable properties, a critical step towards accelerating new chemical discovery. Here we propose QMO, a generic query-based molecule optimization framework that exploits latent embeddings from a molecule autoencoder. QMO improves the desired properties of an input molecule based on efficient queries, guided by a set of molecular property predictions and evaluation metrics. We show that QMO outperforms existing methods in the benchmark tasks of optimizing small organic molecules for drug-likeness and solubility under similarity constraints. We also demonstrate substantial property improvement using QMO on two new and challenging tasks that are also important in real-world discovery problems: (1) optimizing existing potential SARS-CoV-2 main protease inhibitors towards higher binding affinity and (2) improving known antimicrobial peptides towards lower toxicity. Results from QMO show high consistency with external validations, suggesting an effective means to facilitate material optimization problems with design constraints. Zeroth-order optimization is used on problems where no explicit gradient function is accessible, but single points can be queried. Hoffman et al. present here a molecular design method that uses zeroth-order optimization to deal with the discreteness of molecule sequences and to incorporate external guidance from property evaluations and design constraints.},
  archive      = {J_NATMI},
  author       = {Hoffman, Samuel C. and Chenthamarakshan, Vijil and Wadhawan, Kahini and Chen, Pin-Yu and Das, Payel},
  doi          = {10.1038/s42256-021-00422-y},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {21-31},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Optimizing molecules using efficient queries from property evaluations},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Intelligent problem-solving as integrated hierarchical
reinforcement learning. <em>NATMI</em>, <em>4</em>(1), 11–20. (<a
href="https://doi.org/10.1038/s42256-021-00433-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {According to cognitive psychology and related disciplines, the development of complex problem-solving behaviour in biological agents depends on hierarchical cognitive mechanisms. Hierarchical reinforcement learning is a promising computational approach that may eventually yield comparable problem-solving behaviour in artificial agents and robots. However, so far, the problem-solving abilities of many human and non-human animals are clearly superior to those of artificial systems. Here we propose steps to integrate biologically inspired hierarchical mechanisms to enable advanced problem-solving skills in artificial agents. We first review the literature in cognitive psychology to highlight the importance of compositional abstraction and predictive processing. Then we relate the gained insights with contemporary hierarchical reinforcement learning methods. Interestingly, our results suggest that all identified cognitive mechanisms have been implemented individually in isolated computational architectures, raising the question of why there exists no single unifying architecture that integrates them. As our final contribution, we address this question by providing an integrative perspective on the computational challenges to develop such a unifying architecture. We expect our results to guide the development of more sophisticated cognitively inspired hierarchical machine learning architectures. Although artificial reinforcement learning agents do well when rules are rigid, such as games, they fare poorly in real-world scenarios where small changes in the environment or the required actions can impair performance. The authors provide an overview of the cognitive foundations of hierarchical problem-solving, and propose steps to integrate biologically inspired hierarchical mechanisms to enable problem-solving skills in artificial agents.},
  archive      = {J_NATMI},
  author       = {Eppe, Manfred and Gumbsch, Christian and Kerzel, Matthias and Nguyen, Phuong D. H. and Butz, Martin V. and Wermter, Stefan},
  doi          = {10.1038/s42256-021-00433-9},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {11-20},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Intelligent problem-solving as integrated hierarchical reinforcement learning},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). AI reflections in 2021. <em>NATMI</em>, <em>4</em>(1), 5–10.
(<a href="https://doi.org/10.1038/s42256-021-00435-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For a third year in a row, we followed up with authors of several recent Comments and Perspectives in Nature Machine Intelligence about what happened after their article was published: how did the topic they wrote about develop, did they gain new insights, and what are their hopes and expectations for AI in 2022?},
  archive      = {J_NATMI},
  author       = {Buckner, Cameron and Miikkulainen, Risto and Forrest, Stephanie and Milano, Silvia and Zou, James and Prunk, Carina and Irrgang, Christopher and Cohen, I. Glenn and Su, Hao and Murphy, Robin R. and Taylor, Russell H. and Krieger, Axel and Kovač, Mirko and Sadowski, Jathan and Marda, Vidushi},
  doi          = {10.1038/s42256-021-00435-7},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {5-10},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI reflections in 2021},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Environmental robotics for a sustainable future in circular
economies. <em>NATMI</em>, <em>4</em>(1), 3–4. (<a
href="https://doi.org/10.1038/s42256-021-00436-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Grau Ruiz, María Amparo and O’Brolchain, Fiachra},
  doi          = {10.1038/s42256-021-00436-6},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {3-4},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Environmental robotics for a sustainable future in circular economies},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The rise and fall (and rise) of datasets. <em>NATMI</em>,
<em>4</em>(1), 1–2. (<a
href="https://doi.org/10.1038/s42256-022-00442-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Growing criticisms of datasets that were built from user-generated data scraped from the web have led to the retirement or redaction of many popular benchmarks. Their afterlife, as copies or subsets that continue to be used, is a cause for concern.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-022-00442-2},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {1-2},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The rise and fall (and rise) of datasets},
  volume       = {4},
  year         = {2022},
}
</textarea>
</details></li>
</ul>

</body>
</html>
