<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth---415">NMETH - 415</h2>
<ul>
<li><details>
<summary>
(2024a). Author correction: scGHOST: Identifying single-cell 3D
genome subcompartments. <em>NMETH</em>, <em>21</em>(12), 2408. (<a
href="https://doi.org/10.1038/s41592-024-02462-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Xiong, Kyle and Zhang, Ruochi and Ma, Jian},
  doi          = {10.1038/s41592-024-02462-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2408},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: scGHOST: identifying single-cell 3D genome subcompartments},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Influence of different sample preparation approaches on
proteoform identification by top-down proteomics. <em>NMETH</em>,
<em>21</em>(12), 2397–2407. (<a
href="https://doi.org/10.1038/s41592-024-02481-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Top-down proteomics using mass spectrometry facilitates the identification of intact proteoforms, that is, all molecular forms of proteins. Multiple past advances have lead to the development of numerous sample preparation workflows. Here we systematically investigated the influence of different sample preparation steps on proteoform and protein identifications, including cell lysis, reduction and alkylation, proteoform enrichment, purification and fractionation. We found that all steps in sample preparation influence the subset of proteoforms identified (for example, their number, confidence, physicochemical properties and artificially generated modifications). The various sample preparation strategies resulted in complementary identifications, substantially increasing the proteome coverage. Overall, we identified 13,975 proteoforms from 2,720 proteins of human Caco-2 cells. The results presented can serve as suggestions for designing and adapting top-down proteomics sample preparation strategies to particular research questions. Moreover, we expect that the sampling bias and modifications identified at the intact protein level will also be useful in improving bottom-up proteomics approaches. A systematic analysis of the influence of different sample preparation steps on proteoform identification by top-down proteomics serves as a useful reference for designing appropriate workflows for specific research questions.},
  archive      = {J_NMETH},
  author       = {Kaulich, Philipp T. and Jeong, Kyowon and Kohlbacher, Oliver and Tholey, Andreas},
  doi          = {10.1038/s41592-024-02481-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2397-2407},
  shortjournal = {Nat. Meth.},
  title        = {Influence of different sample preparation approaches on proteoform identification by top-down proteomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Top-down mass spectrometry of native proteoforms and their
complexes: A community study. <em>NMETH</em>, <em>21</em>(12),
2388–2396. (<a
href="https://doi.org/10.1038/s41592-024-02279-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The combination of native electrospray ionization with top-down fragmentation in mass spectrometry (MS) allows simultaneous determination of the stoichiometry of noncovalent complexes and identification of their component proteoforms and cofactors. Although this approach is powerful, both native MS and top-down MS are not yet well standardized, and only a limited number of laboratories regularly carry out this type of research. To address this challenge, the Consortium for Top-Down Proteomics initiated a study to develop and test protocols for native MS combined with top-down fragmentation of proteins and protein complexes across 11 instruments in nine laboratories. Here we report the summary of the outcomes to provide robust benchmarks and a valuable entry point for the scientific community. The Consortium for Top-Down Proteomics conducted a study to develop and test protocols for native mass spectrometry combined with top-down fragmentation of proteins and protein complexes across eleven instruments in nine laboratories. They report the summary of the outcomes and their recommendations in this Analysis.},
  archive      = {J_NMETH},
  author       = {Habeck, Tanja and Brown, Kyle A. and Des Soye, Benjamin and Lantz, Carter and Zhou, Mowei and Alam, Novera and Hossain, Md Amin and Jung, Wonhyeuk and Keener, James E. and Volny, Michael and Wilson, Jesse W. and Ying, Yujia and Agar, Jeffrey N. and Danis, Paul O. and Ge, Ying and Kelleher, Neil L. and Li, Huilin and Loo, Joseph A. and Marty, Michael T. and Paša-Tolić, Ljiljana and Sandoval, Wendy and Lermyte, Frederik},
  doi          = {10.1038/s41592-024-02279-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2388-2396},
  shortjournal = {Nat. Meth.},
  title        = {Top-down mass spectrometry of native proteoforms and their complexes: A community study},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Analysis of behavioral flow resolves latent phenotypes.
<em>NMETH</em>, <em>21</em>(12), 2376–2387. (<a
href="https://doi.org/10.1038/s41592-024-02500-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accurate detection and quantification of rodent behavior forms a cornerstone of basic biomedical research. Current data-driven approaches, which segment free exploratory behavior into clusters, suffer from low statistical power due to multiple testing, exhibit poor transferability across experiments and fail to exploit the rich behavioral profiles of individual animals. Here we introduce a pipeline to capture each animal’s behavioral flow, yielding a single metric based on all observed transitions between clusters. By stabilizing these clusters through machine learning, we ensure data transferability, while dimensionality reduction techniques facilitate detailed analysis of individual animals. We provide a large dataset of 771 behavior recordings of freely moving mice—including stress exposures, pharmacological and brain circuit interventions—to identify hidden treatment effects, reveal subtle variations on the level of individual animals and detect brain processes underlying specific interventions. Our pipeline, compatible with popular clustering methods, substantially enhances statistical power and enables predictions of an animal’s future behavior. BehaviorFlow is a behavioral analysis package that overcomes challenges with multiple testing when dealing with large numbers of behavioral variables and limited availability of data. BehaviorFlow also allows combining datasets from different experiments.},
  archive      = {J_NMETH},
  author       = {von Ziegler, Lukas M. and Roessler, Fabienne K. and Sturman, Oliver and Waag, Rebecca and Privitera, Mattia and Duss, Sian N. and O’Connor, Eoin C. and Bohacek, Johannes},
  doi          = {10.1038/s41592-024-02500-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2376-2387},
  shortjournal = {Nat. Meth.},
  title        = {Analysis of behavioral flow resolves latent phenotypes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Long-term optical imaging of the spinal cord in awake
behaving mice. <em>NMETH</em>, <em>21</em>(12), 2363–2375. (<a
href="https://doi.org/10.1038/s41592-024-02476-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in optical imaging and fluorescent biosensors enable study of the spatiotemporal and long-term neural dynamics in the brain of awake animals. However, methodological difficulties and fibrosis limit similar advances in the spinal cord. Here, to overcome these obstacles, we combined in vivo application of fluoropolymer membranes that inhibit fibrosis, a redesigned implantable spinal imaging chamber and improved motion correction methods that together permit imaging of the spinal cord in awake behaving mice, for months to over a year. We demonstrated a robust ability to monitor axons, identified a spinal cord somatotopic map, performed months-long imaging in freely moving mice, conducted Ca2+ imaging of neural dynamics in behaving mice responding to pain-provoking stimuli and observed persistent microglial changes after nerve injury. The ability to couple in vivo imaging and behavior at the spinal cord level will drive insights not previously possible at a key location for somatosensory transmission to the brain. Long-term imaging in the spinal cord is achieved by placing a fluoropolymer membrane on the spinal cord, which reduces fibrosis. This approach, combined with deep-learning-based motion correction, enables months-long imaging of the same neurons.},
  archive      = {J_NMETH},
  author       = {Ahanonu, Biafra and Crowther, Andrew and Kania, Artur and Rosa-Casillas, Mariela and Basbaum, Allan I.},
  doi          = {10.1038/s41592-024-02476-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2363-2375},
  shortjournal = {Nat. Meth.},
  title        = {Long-term optical imaging of the spinal cord in awake behaving mice},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). NeuroMechFly v2: Simulating embodied sensorimotor control in
adult drosophila. <em>NMETH</em>, <em>21</em>(12), 2353–2362. (<a
href="https://doi.org/10.1038/s41592-024-02497-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering principles underlying the control of animal behavior requires a tight dialogue between experiments and neuromechanical models. Such models have primarily been used to investigate motor control with less emphasis on how the brain and motor systems work together during hierarchical sensorimotor control. NeuroMechFly v2 expands Drosophila neuromechanical modeling by enabling vision, olfaction, ascending motor feedback and complex terrains that can be navigated using leg adhesion. We illustrate its capabilities by constructing biologically inspired controllers that use ascending feedback to perform path integration and head stabilization. After adding vision and olfaction, we train a controller using reinforcement learning to perform a multimodal navigation task. Finally, we illustrate more bio-realistic modeling involving complex odor plume navigation, and fly–fly following using a connectome-constrained visual network. NeuroMechFly can be used to accelerate the discovery of explanatory models of the nervous system and to develop machine learning-based controllers for autonomous artificial agents and robots. NeuroMechFly v2 extends the capabilities of the original neuromechanical modeling platform for Drosophila, NeuroMechFly, by including sensory input, motor feedback and the ability to simulate complex terrains.},
  archive      = {J_NMETH},
  author       = {Wang-Chen, Sibo and Stimpfling, Victor Alfred and Lam, Thomas Ka Chung and Özdil, Pembe Gizem and Genoud, Louise and Hurtak, Femke and Ramdya, Pavan},
  doi          = {10.1038/s41592-024-02497-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2353-2362},
  shortjournal = {Nat. Meth.},
  title        = {NeuroMechFly v2: Simulating embodied sensorimotor control in adult drosophila},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Image processing tools for petabyte-scale light sheet
microscopy data. <em>NMETH</em>, <em>21</em>(12), 2342–2352. (<a
href="https://doi.org/10.1038/s41592-024-02475-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light sheet microscopy is a powerful technique for high-speed three-dimensional imaging of subcellular dynamics and large biological specimens. However, it often generates datasets ranging from hundreds of gigabytes to petabytes in size for a single experiment. Conventional computational tools process such images far slower than the time to acquire them and often fail outright due to memory limitations. To address these challenges, we present PetaKit5D, a scalable software solution for efficient petabyte-scale light sheet image processing. This software incorporates a suite of commonly used processing tools that are optimized for memory and performance. Notable advancements include rapid image readers and writers, fast and memory-efficient geometric transformations, high-performance Richardson–Lucy deconvolution and scalable Zarr-based stitching. These features outperform state-of-the-art methods by over one order of magnitude, enabling the processing of petabyte-scale image data at the full teravoxel rates of modern imaging cameras. The software opens new avenues for biological discoveries through large-scale imaging experiments. PetaKit5D offers versatile processing workflows for light sheet microscopy data including performant image input/output, geometric transformations, deconvolution and stitching. The software is efficient and scalable to petabyte-size datasets.},
  archive      = {J_NMETH},
  author       = {Ruan, Xiongtao and Mueller, Matthew and Liu, Gaoxiang and Görlitz, Frederik and Fu, Tian-Ming and Milkie, Daniel E. and Lillvis, Joshua L. and Kuhn, Alexander and Gan Chong, Johnny and Hong, Jason Li and Herr, Chu Yi Aaron and Hercule, Wilmene and Nienhaus, Marc and Killilea, Alison N. and Betzig, Eric and Upadhyayula, Srigokul},
  doi          = {10.1038/s41592-024-02475-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2342-2352},
  shortjournal = {Nat. Meth.},
  title        = {Image processing tools for petabyte-scale light sheet microscopy data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Neural space–time model for dynamic multi-shot imaging.
<em>NMETH</em>, <em>21</em>(12), 2336–2341. (<a
href="https://doi.org/10.1038/s41592-024-02417-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational imaging reconstructions from multiple measurements that are captured sequentially often suffer from motion artifacts if the scene is dynamic. We propose a neural space–time model (NSTM) that jointly estimates the scene and its motion dynamics, without data priors or pre-training. Hence, we can both remove motion artifacts and resolve sample dynamics from the same set of raw measurements used for the conventional reconstruction. We demonstrate NSTM in three computational imaging systems: differential phase-contrast microscopy, three-dimensional structured illumination microscopy and rolling-shutter DiffuserCam. We show that NSTM can recover subcellular motion dynamics and thus reduce the misinterpretation of living systems caused by motion artifacts. A neural space–time model can recover a dynamic scene by modeling its spatiotemporal relationship in multi-shot imaging reconstruction for reduced motion artifacts and improved imaging of fast processes in living cells.},
  archive      = {J_NMETH},
  author       = {Cao, Ruiming and Divekar, Nikita S. and Nuñez, James K. and Upadhyayula, Srigokul and Waller, Laura},
  doi          = {10.1038/s41592-024-02417-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2336-2341},
  shortjournal = {Nat. Meth.},
  title        = {Neural space–time model for dynamic multi-shot imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Proteome-scale recombinant standards and a robust high-speed
search engine to advance cross-linking MS-based interactomics.
<em>NMETH</em>, <em>21</em>(12), 2327–2335. (<a
href="https://doi.org/10.1038/s41592-024-02478-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advancing data analysis tools for proteome-wide cross-linking mass spectrometry (XL-MS) requires ground-truth standards that mimic biological complexity. Here we develop well-controlled XL-MS standards comprising hundreds of recombinant proteins that are systematically mixed for cross-linking. We use one standard dataset to guide the development of Scout, a search engine for XL-MS with MS-cleavable cross-linkers. Using other, independent standard datasets and published datasets, we benchmark the performance of Scout and existing XL-MS software. We find that Scout offers an excellent combination of speed, sensitivity and false discovery rate control. The results illustrate how our large recombinant standard can support the development of XL-MS analysis tools and evaluation of XL-MS results. This Article reports cross-linking mass spectrometry (XL-MS) standard datasets with fully controlled protein interactions that are an order of magnitude more complex than existing ones. These datasets are used to benchmark XL-MS software and establish a fast, error-controlled search tool for XL-MS with cleavable reagents.},
  archive      = {J_NMETH},
  author       = {Clasen, Milan Avila and Ruwolt, Max and Wang, Cong and Ruta, Julia and Bogdanow, Boris and Kurt, Louise U. and Zhang, Zehong and Wang, Shuai and Gozzo, Fabio C. and Chen, Tao and Carvalho, Paulo C. and Lima, Diogo Borges and Liu, Fan},
  doi          = {10.1038/s41592-024-02478-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2327-2335},
  shortjournal = {Nat. Meth.},
  title        = {Proteome-scale recombinant standards and a robust high-speed search engine to advance cross-linking MS-based interactomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). CryoSTAR: Leveraging structural priors and constraints for
cryo-EM heterogeneous reconstruction. <em>NMETH</em>, <em>21</em>(12),
2318–2326. (<a
href="https://doi.org/10.1038/s41592-024-02486-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resolving conformational heterogeneity in cryogenic electron microscopy datasets remains an important challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any preexisting structural knowledge as prior or constraints. Here we present cryoSTAR, which harnesses atomic model information as structural regularization to elucidate such heterogeneity. Our method uniquely outputs both coarse-grained models and density maps, showcasing the molecular conformational changes at different levels. Validated against four diverse experimental datasets, spanning large complexes, a membrane protein and a small single-chain protein, our results consistently demonstrate an efficient and effective solution to conformational heterogeneity with minimal human bias. By integrating atomic model insights with cryogenic electron microscopy data, cryoSTAR represents a meaningful step forward, paving the way for a deeper understanding of dynamic biological processes. CryoSTAR is a deep neural network model that resolves continuous conformational heterogeneity from cryo-EM datasets using an initial atomic model as the reference to generate both density maps and reasonable coarse-grained models for different conformations.},
  archive      = {J_NMETH},
  author       = {Li, Yilai and Zhou, Yi and Yuan, Jing and Ye, Fei and Gu, Quanquan},
  doi          = {10.1038/s41592-024-02486-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2318-2326},
  shortjournal = {Nat. Meth.},
  title        = {CryoSTAR: Leveraging structural priors and constraints for cryo-EM heterogeneous reconstruction},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DiffModeler: Large macromolecular structure modeling for
cryo-EM maps using a diffusion model. <em>NMETH</em>, <em>21</em>(12),
2307–2317. (<a
href="https://doi.org/10.1038/s41592-024-02479-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron microscopy (cryo-EM) has now been widely used for determining multichain protein complexes. However, modeling a large complex structure, such as those with more than ten chains, is challenging, particularly when the map resolution decreases. Here we present DiffModeler, a fully automated method for modeling large protein complex structures. DiffModeler employs a diffusion model for backbone tracing and integrates AlphaFold2-predicted single-chain structures for structure fitting. DiffModeler showed an average template modeling score of 0.88 and 0.91 for two datasets of cryo-EM maps of 0–5 Å resolution and 0.92 for intermediate resolution maps (5–10 Å), substantially outperforming existing methodologies. Further benchmarking at low resolutions (10–20 Å) confirms its versatility, demonstrating plausible performance. DiffModeler is a fully automated structure fitting method for modeling large protein complex structures in cryo-EM maps with resolutions up to 15 Å.},
  archive      = {J_NMETH},
  author       = {Wang, Xiao and Zhu, Han and Terashi, Genki and Taluja, Manav and Kihara, Daisuke},
  doi          = {10.1038/s41592-024-02479-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2307-2317},
  shortjournal = {Nat. Meth.},
  title        = {DiffModeler: Large macromolecular structure modeling for cryo-EM maps using a diffusion model},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Elemental mapping in single-particle reconstructions by
reconstructed electron energy-loss analysis. <em>NMETH</em>,
<em>21</em>(12), 2299–2306. (<a
href="https://doi.org/10.1038/s41592-024-02482-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For macromolecular structures determined by cryogenic electron microscopy, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors and lipids that play essential roles in activity and regulation. Elemental mapping in the electron microscope is well established for dose-tolerant samples but is challenging for biological samples, especially in a cryo-preserved state. Here we combine electron energy-loss spectroscopy with single-particle image processing to allow elemental mapping in cryo-preserved macromolecular complexes. Proof-of-principle data show that our method, reconstructed electron energy-loss (REEL) analysis, allows a three-dimensional reconstruction of electron energy-loss spectroscopy data, such that a high total electron dose is accumulated across many copies of a complex. Working with two test samples, we demonstrate that we can reliably localize abundant elements. We discuss the current limitations of the method and potential future developments. An approach combining electron energy-loss spectroscopy with image processing tools from single-particle cryo-electron microscopy enables elemental mapping in macromolecular complexes, paving the way for the accurate assignment of metals, ions, ligands and lipids.},
  archive      = {J_NMETH},
  author       = {Pfeil-Gardiner, Olivia and Rosa, Higor Vinícius Dias and Riedel, Dietmar and Chen, Yu Seby and Lörks, Dominique and Kükelhan, Pirmin and Linck, Martin and Müller, Heiko and Van Petegem, Filip and Murphy, Bonnie J.},
  doi          = {10.1038/s41592-024-02482-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2299-2306},
  shortjournal = {Nat. Meth.},
  title        = {Elemental mapping in single-particle reconstructions by reconstructed electron energy-loss analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accurate RNA 3D structure prediction using a language
model-based deep learning approach. <em>NMETH</em>, <em>21</em>(12),
2287–2298. (<a
href="https://doi.org/10.1038/s41592-024-02487-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate prediction of RNA three-dimensional (3D) structures remains an unsolved challenge. Determining RNA 3D structures is crucial for understanding their functions and informing RNA-targeting drug development and synthetic biology design. The structural flexibility of RNA, which leads to the scarcity of experimentally determined data, complicates computational prediction efforts. Here we present RhoFold+, an RNA language model-based deep learning method that accurately predicts 3D structures of single-chain RNAs from sequences. By integrating an RNA language model pretrained on ~23.7 million RNA sequences and leveraging techniques to address data scarcity, RhoFold+ offers a fully automated end-to-end pipeline for RNA 3D structure prediction. Retrospective evaluations on RNA-Puzzles and CASP15 natural RNA targets demonstrate the superiority of RhoFold+ over existing methods, including human expert groups. Its efficacy and generalizability are further validated through cross-family and cross-type assessments, as well as time-censored benchmarks. Additionally, RhoFold+ predicts RNA secondary structures and interhelical angles, providing empirically verifiable features that broaden its applicability to RNA structure and function studies. RhoFold+ is an end-to-end language model-based deep learning method to predict RNA three-dimensional structures of single-chain RNAs from sequences.},
  archive      = {J_NMETH},
  author       = {Shen, Tao and Hu, Zhihang and Sun, Siqi and Liu, Di and Wong, Felix and Wang, Jiuming and Chen, Jiayang and Wang, Yixuan and Hong, Liang and Xiao, Jin and Zheng, Liangzhen and Krishnamoorthi, Tejas and King, Irwin and Wang, Sheng and Yin, Peng and Collins, James J. and Li, Yu},
  doi          = {10.1038/s41592-024-02487-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2287-2298},
  shortjournal = {Nat. Meth.},
  title        = {Accurate RNA 3D structure prediction using a language model-based deep learning approach},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Statistical inference with a manifold-constrained RNA
velocity model uncovers cell cycle speed modulations. <em>NMETH</em>,
<em>21</em>(12), 2271–2286. (<a
href="https://doi.org/10.1038/s41592-024-02471-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across biological systems, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold. While low-dimensional dynamics can be extracted using RNA velocity, these algorithms can be fragile and rely on heuristics lacking statistical control. Moreover, the estimated vector field is not dynamically consistent with the traversed gene expression manifold. To address these challenges, we introduce a Bayesian model of RNA velocity that couples velocity field and manifold estimation in a reformulated, unified framework, identifying the parameters of an explicit dynamical system. Focusing on the cell cycle, we implement VeloCycle to study gene regulation dynamics on one-dimensional periodic manifolds and validate its ability to infer cell cycle periods using live imaging. We also apply VeloCycle to reveal speed differences in regionally defined progenitors and Perturb-seq gene knockdowns. Overall, VeloCycle expands the single-cell RNA sequencing analysis toolkit with a modular and statistically consistent RNA velocity inference framework. VeloCycle is a manifold-constrained generative framework to estimate RNA velocity during the cell cycle.},
  archive      = {J_NMETH},
  author       = {Lederer, Alex R. and Leonardi, Maxine and Talamanca, Lorenzo and Bobrovskiy, Daniil M. and Herrera, Antonio and Droin, Colas and Khven, Irina and Carvalho, Hugo J. F. and Valente, Alessandro and Dominguez Mantes, Albert and Mulet Arabí, Pau and Pinello, Luca and Naef, Felix and La Manno, Gioele},
  doi          = {10.1038/s41592-024-02471-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2271-2286},
  shortjournal = {Nat. Meth.},
  title        = {Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Probe set selection for targeted spatial transcriptomics.
<em>NMETH</em>, <em>21</em>(12), 2260–2270. (<a
href="https://doi.org/10.1038/s41592-024-02496-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted spatial transcriptomic methods capture the topology of cell types and states in tissues at single-cell and subcellular resolution by measuring the expression of a predefined set of genes. The selection of an optimal set of probed genes is crucial for capturing the spatial signals present in a tissue. This requires selecting the most informative, yet minimal, set of genes to profile (gene set selection) for which it is possible to build probes (probe design). However, current selections often rely on marker genes, precluding them from detecting continuous spatial signals or new states. We present Spapros, an end-to-end probe set selection pipeline that optimizes both gene set specificity for cell type identification and within-cell type expression variation to resolve spatially distinct populations while considering prior knowledge as well as probe design and expression constraints. We evaluated Spapros and show that it outperforms other selection approaches in both cell type recovery and recovering expression variation beyond cell types. Furthermore, we used Spapros to design a single-cell resolution in situ hybridization on tissues (SCRINSHOT) experiment of adult lung tissue to demonstrate how probes selected with Spapros identify cell types of interest and detect spatial variation even within cell types. Spapros is a probe set selection pipeline for targeted spatial transcriptomics that optimizes for both transcriptional and within-cell type variation.},
  archive      = {J_NMETH},
  author       = {Kuemmerle, Louis B. and Luecken, Malte D. and Firsova, Alexandra B. and Barros de Andrade e Sousa, Lisa and Straßer, Lena and Mekki, Ilhem Isra and Campi, Francesco and Heumos, Lukas and Shulman, Maiia and Beliaeva, Valentina and Hediyeh-Zadeh, Soroor and Schaar, Anna C. and Mahbubani, Krishnaa T. and Sountoulidis, Alexandros and Balassa, Tamás and Kovacs, Ferenc and Horvath, Peter and Piraud, Marie and Ertürk, Ali and Samakovlis, Christos and Theis, Fabian J.},
  doi          = {10.1038/s41592-024-02496-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2260-2270},
  shortjournal = {Nat. Meth.},
  title        = {Probe set selection for targeted spatial transcriptomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quality control for single-cell analysis of high-plex tissue
profiles using CyLinter. <em>NMETH</em>, <em>21</em>(12), 2248–2259. (<a
href="https://doi.org/10.1038/s41592-024-02328-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumors are complex assemblies of cellular and acellular structures patterned on spatial scales from microns to centimeters. Study of these assemblies has advanced dramatically with the introduction of high-plex spatial profiling. Image-based profiling methods reveal the intensities and spatial distributions of 20–100 proteins at subcellular resolution in 103–107 cells per specimen. Despite extensive work on methods for extracting single-cell data from these images, all tissue images contain artifacts such as folds, debris, antibody aggregates, optical aberrations and image processing errors that arise from imperfections in specimen preparation, data acquisition, image assembly and feature extraction. Here we show that these artifacts dramatically impact single-cell data analysis, obscuring meaningful biological interpretation. We describe an interactive quality control software tool, CyLinter, that identifies and removes data associated with imaging artifacts. CyLinter greatly improves single-cell analysis, especially for archival specimens sectioned many years before data collection, such as those from clinical trials. Microscopy artifacts and tissue imperfections interfere with single-cell analysis. CyLinter software offers quality control for high-plex tissue profiling by removing artifactual cells, thereby facilitating accuracy of biological interpretation.},
  archive      = {J_NMETH},
  author       = {Baker, Gregory J. and Novikov, Edward and Zhao, Ziyuan and Vallius, Tuulia and Davis, Janae A. and Lin, Jia-Ren and Muhlich, Jeremy L. and Mittendorf, Elizabeth A. and Santagata, Sandro and Guerriero, Jennifer L. and Sorger, Peter K.},
  doi          = {10.1038/s41592-024-02328-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2248-2259},
  shortjournal = {Nat. Meth.},
  title        = {Quality control for single-cell analysis of high-plex tissue profiles using CyLinter},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inferring allele-specific copy number aberrations and tumor
phylogeography from spatially resolved transcriptomics. <em>NMETH</em>,
<em>21</em>(12), 2239–2247. (<a
href="https://doi.org/10.1038/s41592-024-02438-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analyzing somatic evolution within a tumor over time and across space is a key challenge in cancer research. Spatially resolved transcriptomics (SRT) measures gene expression at thousands of spatial locations in a tumor, but does not directly reveal genomic aberrations. We introduce CalicoST, an algorithm to simultaneously infer allele-specific copy number aberrations (CNAs) and reconstruct spatial tumor evolution, or phylogeography, from SRT data. CalicoST identifies important classes of CNAs—including copy-neutral loss of heterozygosity and mirrored subclonal CNAs—that are invisible to total copy number analysis. Using nine patients’ data from the Human Tumor Atlas Network, CalicoST achieves an average accuracy of 86%, approximately 21% higher than existing methods. CalicoST reconstructs a tumor phylogeography in three-dimensional space for two patients with multiple adjacent slices. CalicoST analysis of multiple SRT slices from a cancerous prostate organ reveals mirrored subclonal CNAs on the two sides of the prostate, forming a bifurcating phylogeography in both genetic and physical space. This work presents CalicoST for inferring allele-specific copy numbers and reconstructing spatial tumor evolution by using spatial transcriptomics data.},
  archive      = {J_NMETH},
  author       = {Ma, Cong and Balaban, Metin and Liu, Jingxian and Chen, Siqi and Wilson, Michael J. and Sun, Christopher H. and Ding, Li and Raphael, Benjamin J.},
  doi          = {10.1038/s41592-024-02438-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2239-2247},
  shortjournal = {Nat. Meth.},
  title        = {Inferring allele-specific copy number aberrations and tumor phylogeography from spatially resolved transcriptomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Large language modeling and deep learning shed light on RNA
structure prediction. <em>NMETH</em>, <em>21</em>(12), 2237–2238. (<a
href="https://doi.org/10.1038/s41592-024-02488-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an RNA language model-based deep learning pipeline for accurate and rapid de novo RNA 3D structure prediction, demonstrating strong accuracy in modeling single-stranded RNAs and excellent generalization across RNA families and types while also being capable of capturing local features such as interhelical angles and secondary structures.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02488-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2237-2238},
  shortjournal = {Nat. Meth.},
  title        = {Large language modeling and deep learning shed light on RNA structure prediction},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Content-aware motion correction for multi-shot imaging.
<em>NMETH</em>, <em>21</em>(12), 2235–2236. (<a
href="https://doi.org/10.1038/s41592-024-02520-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new deep learning approach enables motion correction across sequential acquisitions, even if each image in the sequence was acquired differently.},
  archive      = {J_NMETH},
  author       = {Laine, Romain F.},
  doi          = {10.1038/s41592-024-02520-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2235-2236},
  shortjournal = {Nat. Meth.},
  title        = {Content-aware motion correction for multi-shot imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cryo-electron microscopy in color. <em>NMETH</em>,
<em>21</em>(12), 2233–2234. (<a
href="https://doi.org/10.1038/s41592-024-02427-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron microscopy with energy resolution, using the EELS-STEM method, allows researchers to identify the approximate locations of certain heavier atoms within single frozen, hydrated protein particles.},
  archive      = {J_NMETH},
  author       = {Stahlberg, Henning},
  doi          = {10.1038/s41592-024-02427-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2233-2234},
  shortjournal = {Nat. Meth.},
  title        = {Cryo-electron microscopy in color},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Uncovering cell cycle speed modulations with statistical
inference. <em>NMETH</em>, <em>21</em>(12), 2231–2232. (<a
href="https://doi.org/10.1038/s41592-024-02484-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Bayesian inference framework improves RNA velocity estimates to make them more coherent and biologically interpretable.},
  archive      = {J_NMETH},
  author       = {Zhang, Pengzhi and Wang, Guangyu},
  doi          = {10.1038/s41592-024-02484-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2231-2232},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering cell cycle speed modulations with statistical inference},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024l). More dimensions of the 3D genome. <em>NMETH</em>,
<em>21</em>(12), 2229. (<a
href="https://doi.org/10.1038/s41592-024-02550-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances aim to capture comprehensive, dynamic genome structures in living cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02550-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2229},
  shortjournal = {Nat. Meth.},
  title        = {More dimensions of the 3D genome},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Molecular motion in situ. <em>NMETH</em>, <em>21</em>(12),
2229–2230. (<a
href="https://doi.org/10.1038/s41592-024-02551-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Algorithms help to capture macromolecular motion and structural heterogeneity in native cellular environments.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02551-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2229-2230},
  shortjournal = {Nat. Meth.},
  title        = {Molecular motion in situ},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Biomechanical modeling of whole bodies. <em>NMETH</em>,
<em>21</em>(12), 2228. (<a
href="https://doi.org/10.1038/s41592-024-02548-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detailed biomechanical models of animal bodies can help to tackle questions about how the brain controls movement and bodily interactions with the environment.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02548-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2228},
  shortjournal = {Nat. Meth.},
  title        = {Biomechanical modeling of whole bodies},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Organoids get an upgrade. <em>NMETH</em>, <em>21</em>(12),
2228–2229. (<a
href="https://doi.org/10.1038/s41592-024-02549-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immune–organoid systems will be the next generation of in vitro models.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02549-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2228-2229},
  shortjournal = {Nat. Meth.},
  title        = {Organoids get an upgrade},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Subcellular spatial proteomics. <em>NMETH</em>,
<em>21</em>(12), 2227. (<a
href="https://doi.org/10.1038/s41592-024-02546-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in multiplexed super-resolution microscopy will usher in the next era of spatial proteomics.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02546-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2227},
  shortjournal = {Nat. Meth.},
  title        = {Subcellular spatial proteomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024j). Mapping cell history and fates. <em>NMETH</em>,
<em>21</em>(12), 2227–2228. (<a
href="https://doi.org/10.1038/s41592-024-02547-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New experimental and computational methods illuminate the history of cell differentiation and its molecular underpinnings.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-024-02547-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2227-2228},
  shortjournal = {Nat. Meth.},
  title        = {Mapping cell history and fates},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Adaptable CRISPR screening. <em>NMETH</em>,
<em>21</em>(12), 2226. (<a
href="https://doi.org/10.1038/s41592-024-02544-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR screens are untangling molecular mechanisms that drive biological processes with greater precision and detail.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02544-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2226},
  shortjournal = {Nat. Meth.},
  title        = {Adaptable CRISPR screening},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Understanding protein interaction dynamics. <em>NMETH</em>,
<em>21</em>(12), 2226–2227. (<a
href="https://doi.org/10.1038/s41592-024-02545-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimental and computational studies are paving way for a deeper understanding of the dynamic nature of protein–protein interactions.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02545-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2226-2227},
  shortjournal = {Nat. Meth.},
  title        = {Understanding protein interaction dynamics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integrative spatial protein profiling with multi-omics.
<em>NMETH</em>, <em>21</em>(12), 2223–2225. (<a
href="https://doi.org/10.1038/s41592-024-02533-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial proteomics holds the potential to transform the study of proteins in situ in complex tissues, but it needs to be integrated with other layers of omics data to gain a holistic view of cellular function, heterogeneity and interactions, and the underlying mechanisms of these processes. I highlight current challenges and emerging opportunities for multi-omic spatial protein profiling to advance basic research and translational applications.},
  archive      = {J_NMETH},
  author       = {Fan, Rong},
  doi          = {10.1038/s41592-024-02533-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2223-2225},
  shortjournal = {Nat. Meth.},
  title        = {Integrative spatial protein profiling with multi-omics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A new understanding of tissue biology from MS-based
proteomics at single-cell resolution. <em>NMETH</em>, <em>21</em>(12),
2220–2222. (<a
href="https://doi.org/10.1038/s41592-024-02541-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial mass spectrometry (MS)-proteomics is a rapidly evolving technology, particularly in the form of Deep Visual Proteomics (DVP), which allows the study of single cells directly in their native environment. We believe that this approach will reshape our understanding of tissue biology and redefine fundamental concepts in cell biology, tissue physiology and ultimately human health and disease.},
  archive      = {J_NMETH},
  author       = {Nordmann, Thierry M. and Mund, Andreas and Mann, Matthias},
  doi          = {10.1038/s41592-024-02541-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2220-2222},
  shortjournal = {Nat. Meth.},
  title        = {A new understanding of tissue biology from MS-based proteomics at single-cell resolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Revolutionizing cancer research with spatial proteomics and
visual intelligence. <em>NMETH</em>, <em>21</em>(12), 2216–2219. (<a
href="https://doi.org/10.1038/s41592-024-02542-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial proteomics has transformed cancer research by providing unparalleled insights into the microenvironmental landscape of tumors. Here we discuss how these technologies have significantly advanced our understanding of cell–cell interactions, tissue organization and spatially coordinated mechanisms underlying antitumor immune responses, and will pave the way for emerging breakthroughs in cancer research.},
  archive      = {J_NMETH},
  author       = {Quail, Daniela F. and Walsh, Logan A.},
  doi          = {10.1038/s41592-024-02542-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2216-2219},
  shortjournal = {Nat. Meth.},
  title        = {Revolutionizing cancer research with spatial proteomics and visual intelligence},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multiplexed image analysis: What have we achieved and where
are we headed? <em>NMETH</em>, <em>21</em>(12), 2212–2215. (<a
href="https://doi.org/10.1038/s41592-024-02539-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed tissue imaging has transformed tissue biology by revealing cellular diversity and interactions, but the analysis of its massive datasets remains a bottleneck. Here, we provide an overview of computational advancements, discuss current challenges and envision an AI-driven future in which integrated tools streamline analysis and visualization, unlocking the full potential of multiplexed imaging for breakthroughs in spatial biology.},
  archive      = {J_NMETH},
  author       = {Bussi, Yuval and Keren, Leeat},
  doi          = {10.1038/s41592-024-02539-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2212-2215},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed image analysis: What have we achieved and where are we headed?},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Highly multiplexed imaging in the omics era: Understanding
tissue structures in health and disease. <em>NMETH</em>,
<em>21</em>(12), 2209–2211. (<a
href="https://doi.org/10.1038/s41592-024-02538-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial proteomics is advancing rapidly, transforming physiological and biomedical research by enabling the study of how multicellular structures and intercellular communication shape tissue function in health and disease. Through the analysis of large human tissue collections, spatial proteomics will reveal the complexities of human tissues and uncover multicellular modules that can serve as drug targets and diagnostics, paving the way for precision medicine and revolutionizing histopathology.},
  archive      = {J_NMETH},
  author       = {Bodenmiller, Bernd},
  doi          = {10.1038/s41592-024-02538-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2209-2211},
  shortjournal = {Nat. Meth.},
  title        = {Highly multiplexed imaging in the omics era: Understanding tissue structures in health and disease},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Atlases galore: Where to next? <em>NMETH</em>,
<em>21</em>(12), 2203–2208. (<a
href="https://doi.org/10.1038/s41592-024-02536-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hundreds of researchers collaborate on maps of the human body and the subcellular realm. As they scout out their next mapping expeditions, they take stock of atlas-making.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02536-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2203-2208},
  shortjournal = {Nat. Meth.},
  title        = {Atlases galore: Where to next?},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A data portal for providing standardized annotations for
cryo-electron tomography. <em>NMETH</em>, <em>21</em>(12), 2200–2202.
(<a href="https://doi.org/10.1038/s41592-024-02477-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Ermel, Utz and Cheng, Anchi and Ni, Jun Xi and Gadling, Jessica and Venkatakrishnan, Manasa and Evans, Kira and Asuncion, Jeremy and Sweet, Andrew and Pourroy, Janeece and Wang, Zun Shi and Khandwala, Kandarp and Nelson, Benjamin and McCarthy, Dannielle and Wang, Eric M. and Agarwal, Richa and Carragher, Bridget},
  doi          = {10.1038/s41592-024-02477-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2200-2202},
  shortjournal = {Nat. Meth.},
  title        = {A data portal for providing standardized annotations for cryo-electron tomography},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BioNumPy: Array programming for biology. <em>NMETH</em>,
<em>21</em>(12), 2198–2199. (<a
href="https://doi.org/10.1038/s41592-024-02483-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Rand, Knut Dagestad and Grytten, Ivar and Pavlović, Milena and Kanduri, Chakravarthi and Sandve, Geir Kjetil},
  doi          = {10.1038/s41592-024-02483-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2198-2199},
  shortjournal = {Nat. Meth.},
  title        = {BioNumPy: Array programming for biology},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Method of the year 2024: Spatial proteomics. <em>NMETH</em>,
<em>21</em>(12), 2195–2196. (<a
href="https://doi.org/10.1038/s41592-024-02565-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Approaches for profiling the spatial proteome in tissues are the basis of atlas-scale projects that are delivering on their promise for understanding biological complexity in health and disease.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02565-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2195-2196},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year 2024: Spatial proteomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). First-timers at a huge meeting. <em>NMETH</em>,
<em>21</em>(12), 2197. (<a
href="https://doi.org/10.1038/s41592-024-02527-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {At some meetings, one gets to know all attendees. But at large conferences, that’s rather impossible. Some first-time attendees share how they navigated the sizable Society for Neuroscience annual meeting.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02527-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {12},
  pages        = {2197},
  shortjournal = {Nat. Meth.},
  title        = {First-timers at a huge meeting},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Benchmarking algorithms for single-cell multi-omics
prediction and integration. <em>NMETH</em>, <em>21</em>(11), 2182–2194.
(<a href="https://doi.org/10.1038/s41592-024-02429-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of single-cell multi-omics technology has greatly enhanced our understanding of biology, and in parallel, numerous algorithms have been proposed to predict the protein abundance and/or chromatin accessibility of cells from single-cell transcriptomic information and to integrate various types of single-cell multi-omics data. However, few studies have systematically compared and evaluated the performance of these algorithms. Here, we present a benchmark study of 14 protein abundance/chromatin accessibility prediction algorithms and 18 single-cell multi-omics integration algorithms using 47 single-cell multi-omics datasets. Our benchmark study showed overall totalVI and scArches outperformed the other algorithms for predicting protein abundance, and LS_Lab was the top-performing algorithm for the prediction of chromatin accessibility in most cases. Seurat, MOJITOO and scAI emerge as leading algorithms for vertical integration, whereas totalVI and UINMF excel beyond their counterparts in both horizontal and mosaic integration scenarios. Additionally, we provide a pipeline to assist researchers in selecting the optimal multi-omics prediction and integration algorithm. This Analysis study compares computational methods for single-cell multi-omics prediction and integration, generating useful insights for method users and developers working with different analysis purposes and biological problems.},
  archive      = {J_NMETH},
  author       = {Hu, Yinlei and Wan, Siyuan and Luo, Yuanhanyu and Li, Yuanzhe and Wu, Tong and Deng, Wentao and Jiang, Chen and Jiang, Shan and Zhang, Yueping and Liu, Nianping and Yang, Zongcheng and Chen, Falai and Li, Bin and Qu, Kun},
  doi          = {10.1038/s41592-024-02429-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2182-2194},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking algorithms for single-cell multi-omics prediction and integration},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Brain-wide neural recordings in mice navigating physical
spaces enabled by robotic neural recording headstages. <em>NMETH</em>,
<em>21</em>(11), 2171–2181. (<a
href="https://doi.org/10.1038/s41592-024-02434-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technologies that can record neural activity at cellular resolution at multiple spatial and temporal scales are typically much larger than the animals that are being recorded from and are thus limited to recording from head-fixed subjects. Here we have engineered robotic neural recording devices—‘cranial exoskeletons’—that assist mice in maneuvering recording headstages that are orders of magnitude larger and heavier than the mice, while they navigate physical behavioral environments. We discovered optimal controller parameters that enable mice to locomote at physiologically realistic velocities while maintaining natural walking gaits. We show that mice learn to work with the robot to make turns and perform decision-making tasks. Robotic imaging and electrophysiology headstages were used to record recordings of Ca2+ activity of thousands of neurons distributed across the dorsal cortex and spiking activity of hundreds of neurons across multiple brain regions and multiple days, respectively. To avoid head fixation or drawbacks of miniaturized devices for freely moving rodents, a robotic device can move a headstage for microscopy or electrophysiology with the animal, thereby enabling naturalistic behavior.},
  archive      = {J_NMETH},
  author       = {Hope, James and Beckerle, Travis M. and Cheng, Pin-Hao and Viavattine, Zoey and Feldkamp, Michael and Fausner, Skylar M. L. and Saxena, Kapil and Ko, Eunsong and Hryb, Ihor and Carter, Russell E. and Ebner, Timothy J. and Kodandaramaiah, Suhasa B.},
  doi          = {10.1038/s41592-024-02434-z},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2171-2181},
  shortjournal = {Nat. Meth.},
  title        = {Brain-wide neural recordings in mice navigating physical spaces enabled by robotic neural recording headstages},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Volumetric voltage imaging of neuronal populations in the
mouse brain by confocal light-field microscopy. <em>NMETH</em>,
<em>21</em>(11), 2160–2170. (<a
href="https://doi.org/10.1038/s41592-024-02458-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Voltage imaging measures neuronal activity directly and holds promise for understanding information processing within individual neurons and across populations. However, imaging voltage over large neuronal populations has been challenging owing to the simultaneous requirements of high imaging speed and signal-to-noise ratio, large volume coverage and low photobleaching rate. Here, to overcome this challenge, we developed a confocal light-field microscope that surpassed the traditional limits in speed and noise performance by incorporating a speed-enhanced camera, a fast and robust scanning mechanism, laser-speckle-noise elimination and optimized light efficiency. With this method, we achieved simultaneous recording from more than 300 spiking neurons within an 800-µm-diameter and 180-µm-thick volume in the mouse cortex, for more than 20 min. By integrating the spatial and voltage activity profiles, we have mapped three-dimensional neural coordination patterns in awake mouse brains. Our method is robust for routine application in volumetric voltage imaging. Confocal light-field microscopy allows volumetric voltage imaging in the mouse brain at speeds sufficiently high to extract the spiking activity of hundreds of neurons.},
  archive      = {J_NMETH},
  author       = {Bai, Lu and Cong, Lin and Shi, Ziqi and Zhao, Yuchen and Zhang, Yujie and Lu, Bin and Zhang, Jing and Xiong, Zhi-Qi and Xu, Ninglong and Mu, Yu and Wang, Kai},
  doi          = {10.1038/s41592-024-02458-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2160-2170},
  shortjournal = {Nat. Meth.},
  title        = {Volumetric voltage imaging of neuronal populations in the mouse brain by confocal light-field microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A polarized FGF8 source specifies frontotemporal signatures
in spatially oriented cell populations of cortical assembloids.
<em>NMETH</em>, <em>21</em>(11), 2147–2159. (<a
href="https://doi.org/10.1038/s41592-024-02412-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoids generating major cortical cell types in distinct compartments are used to study cortical development, evolution and disorders. However, the lack of morphogen gradients imparting cortical positional information and topography in current systems hinders the investigation of complex phenotypes. Here, we engineer human cortical assembloids by fusing an organizer-like structure expressing fibroblast growth factor 8 (FGF8) with an elongated organoid to enable the controlled modulation of FGF8 signaling along the longitudinal organoid axis. These polarized cortical assembloids mount a position-dependent transcriptional program that in part matches the in vivo rostrocaudal gene expression patterns and that is lost upon mutation in the FGFR3 gene associated with temporal lobe malformations and intellectual disability. By producing spatially oriented cell populations with signatures related to frontal and temporal area identity within individual assembloids, this model recapitulates in part the early transcriptional divergence embedded in the protomap and enables the study of cortical area-relevant alterations underlying human disorders. Cortical development is influenced by morphogen gradients. To mimic patterning events during brain development, polarized cortical assembloids are generated with the help of a localized FGF8 source.},
  archive      = {J_NMETH},
  author       = {Bosone, Camilla and Castaldi, Davide and Burkard, Thomas Rainer and Guzman, Segundo Jose and Wyatt, Tom and Cheroni, Cristina and Caporale, Nicolò and Bajaj, Sunanjay and Bagley, Joshua Adam and Li, Chong and Sorre, Benoit and Villa, Carlo Emanuele and Testa, Giuseppe and Krenn, Veronica and Knoblich, Jürgen Arthur},
  doi          = {10.1038/s41592-024-02412-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2147-2159},
  shortjournal = {Nat. Meth.},
  title        = {A polarized FGF8 source specifies frontotemporal signatures in spatially oriented cell populations of cortical assembloids},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Differentiating visceral sensory ganglion organoids from
induced pluripotent stem cells. <em>NMETH</em>, <em>21</em>(11),
2135–2146. (<a
href="https://doi.org/10.1038/s41592-024-02455-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to generate visceral sensory neurons (VSN) from induced pluripotent stem (iPS) cells may help to gain insights into how the gut–nerve–brain axis is involved in neurological disorders. We established a protocol to differentiate human iPS-cell-derived visceral sensory ganglion organoids (VSGOs). VSGOs exhibit canonical VSN markers, and single-cell RNA sequencing revealed heterogenous molecular signatures and developmental trajectories of VSGOs aligned with native VSN. We integrated VSGOs with human colon organoids on a microfluidic device and applied this axis-on-a-chip model to Alzheimer’s disease. Our results suggest that VSN could be a potential mediator for propagating gut-derived amyloid and tau to the brain in an APOE4- and LRP1-dependent manner. Furthermore, our approach was extended to include patient-derived iPS cells, which demonstrated a strong correlation with clinical data. A protocol for differentiating visceral sensory ganglion organoids from induced pluripotent stem cells allows the establishment of an in vitro model for the gut–visceral nerve–brain axis and study of the propagation of pathogenic proteins involved in Alzheimer’s disease along the vagus nerve.},
  archive      = {J_NMETH},
  author       = {Ahn, Kyusik and Park, Hwee-Seon and Choi, Sieun and Lee, Hojeong and Choi, Hyunjung and Hong, Seok Beom and Han, Jihui and Han, Jong Won and Ahn, Jinchul and Song, Jaehoon and Park, Kyunghyuk and Cha, Bukyung and Kim, Minseop and Liu, Hui-Wen and Song, Hyeonggyu and Kim, Sang Jeong and Chung, Seok and Kim, Jong-Il and Mook-Jung, Inhee},
  doi          = {10.1038/s41592-024-02455-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2135-2146},
  shortjournal = {Nat. Meth.},
  title        = {Differentiating visceral sensory ganglion organoids from induced pluripotent stem cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-shot 20-fold expansion microscopy. <em>NMETH</em>,
<em>21</em>(11), 2128–2134. (<a
href="https://doi.org/10.1038/s41592-024-02454-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expansion microscopy (ExM) is in increasingly widespread use throughout biology because its isotropic physical magnification enables nanoimaging on conventional microscopes. To date, ExM methods either expand specimens to a limited range (~4–10× linearly) or achieve larger expansion factors through iterating the expansion process a second time (~15–20× linearly). Here, we present an ExM protocol that achieves ~20× expansion (yielding &amp;lt;20-nm resolution on a conventional microscope) in a single expansion step, achieving the performance of iterative expansion with the simplicity of a single-shot protocol. This protocol, which we call 20ExM, supports postexpansion staining for brain tissue, which can facilitate biomolecular labeling. 20ExM may find utility in many areas of biological investigation requiring high-resolution imaging. 20ExM achieves isotropic ~20× expansion of cells and tissues in a single shot for super-resolution imaging with &amp;lt;20-nm resolution on a conventional microscope.},
  archive      = {J_NMETH},
  author       = {Wang, Shiwei and Shin, Tay Won and Yoder, Harley B. and McMillan, Ryan B. and Su, Hanquan and Liu, Yixi and Zhang, Chi and Leung, Kylie S. and Yin, Peng and Kiessling, Laura L. and Boyden, Edward S.},
  doi          = {10.1038/s41592-024-02454-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2128-2134},
  shortjournal = {Nat. Meth.},
  title        = {Single-shot 20-fold expansion microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Restoring protein glycosylation with GlycoShape.
<em>NMETH</em>, <em>21</em>(11), 2117–2127. (<a
href="https://doi.org/10.1038/s41592-024-02464-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite ground-breaking innovations in experimental structural biology and protein structure prediction techniques, capturing the structure of the glycans that functionalize proteins remains a challenge. Here we introduce GlycoShape ( https://glycoshape.org ), an open-access glycan structure database and toolbox designed to restore glycoproteins to their native and functional form in seconds. The GlycoShape database counts over 500 unique glycans so far, covering the human glycome and augmented by elements from a wide range of organisms, obtained from 1 ms of cumulative sampling from molecular dynamics simulations. These structures can be linked to proteins with a robust algorithm named Re-Glyco, directly compatible with structural data in open-access repositories, such as the Research Collaboratory for Structural Bioinformatics Protein Data Bank (RCSB PDB) and AlphaFold Protein Structure Database, or own. The quality, performance and broad applicability of GlycoShape is demonstrated by its ability to predict N-glycosylation occupancy, scoring a 93% agreement with experiment, based on screening all proteins in the PDB with a corresponding glycoproteomics profile, for a total of 4,259 N-glycosylation sequons. GlycoShape is an open-access web-based platform designed to supplement three-dimensional glycoprotein structures with missing structural information on glycans. To link them, the Re-Glyco algorithm evaluates the steric complementarity of glycans using their conformational ensemble with the protein surface.},
  archive      = {J_NMETH},
  author       = {Ives, Callum M. and Singh, Ojas and D’Andrea, Silvia and Fogarty, Carl A. and Harbison, Aoife M. and Satheesan, Akash and Tropea, Beatrice and Fadda, Elisa},
  doi          = {10.1038/s41592-024-02464-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2117-2127},
  shortjournal = {Nat. Meth.},
  title        = {Restoring protein glycosylation with GlycoShape},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). De novo protein design with a denoising diffusion network
independent of pretrained structure prediction models. <em>NMETH</em>,
<em>21</em>(11), 2107–2116. (<a
href="https://doi.org/10.1038/s41592-024-02437-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recent success of RFdiffusion, a method for protein structure design with a denoising diffusion probabilistic model, has relied on fine-tuning the RoseTTAFold structure prediction network for protein backbone denoising. Here, we introduce SCUBA-diffusion (SCUBA-D), a protein backbone denoising diffusion probabilistic model freshly trained by considering co-diffusion of sequence representation to enhance model regularization and adversarial losses to minimize data-out-of-distribution errors. While matching the performance of the pretrained RoseTTAFold-based RFdiffusion in generating experimentally realizable protein structures, SCUBA-D readily generates protein structures with not-yet-observed overall folds that are different from those predictable with RoseTTAFold. The accuracy of SCUBA-D was confirmed by the X-ray structures of 16 designed proteins and a protein complex, and by experiments validating designed heme-binding proteins and Ras-binding proteins. Our work shows that deep generative models of images or texts can be fruitfully extended to complex physical objects like protein structures by addressing outstanding issues such as the data-out-of-distribution errors. SCUBA-D is a denoising diffusion probabilistic model that can perform a variety of protein backbone design tasks, including generation of backbones from random noise, from user-sketched designs, and backbones to scaffold predefined motifs that can bind small molecules or other proteins.},
  archive      = {J_NMETH},
  author       = {Liu, Yufeng and Wang, Sheng and Dong, Jixin and Chen, Linghui and Wang, Xinyu and Wang, Lei and Li, Fudong and Wang, Chenchen and Zhang, Jiahai and Wang, Yuzhu and Wei, Si and Chen, Quan and Liu, Haiyan},
  doi          = {10.1038/s41592-024-02437-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2107-2116},
  shortjournal = {Nat. Meth.},
  title        = {De novo protein design with a denoising diffusion network independent of pretrained structure prediction models},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multiplexed profiling of intracellular protein abundance,
activity, interactions and druggability with LABEL-seq. <em>NMETH</em>,
<em>21</em>(11), 2094–2106. (<a
href="https://doi.org/10.1038/s41592-024-02456-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we describe labeling with barcodes and enrichment for biochemical analysis by sequencing (LABEL-seq), an assay for massively parallel profiling of pooled protein variants in human cells. By leveraging the intracellular self-assembly of an RNA-binding domain (RBD) with a stable, variant-encoding RNA barcode, LABEL-seq facilitates the direct measurement of protein properties and functions using simple affinity enrichments of RBD protein fusions, followed by high-throughput sequencing of co-enriched barcodes. Measurement of ~20,000 variant effects for ~1,600 BRaf variants revealed that variation at positions frequently mutated in cancer minimally impacted intracellular abundance but could dramatically alter activity, protein–protein interactions and druggability. Integrative analysis identified networks of positions with similar biochemical roles and enabled modeling of variant effects on cell proliferation and small molecule-promoted degradation. Thus, LABEL-seq enables direct measurement of multiple biochemical properties in a native cellular context, providing insights into protein function, disease mechanisms and druggability. Labeling with barcodes and enrichment for biochemical analysis by sequencing (LABEL-seq) enables massively parallel profiling of thousands of pooled protein variants in cells, yielding insight into protein function, interactions and druggability.},
  archive      = {J_NMETH},
  author       = {Simon, Jessica J. and Fowler, Douglas M. and Maly, Dustin J.},
  doi          = {10.1038/s41592-024-02456-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2094-2106},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed profiling of intracellular protein abundance, activity, interactions and druggability with LABEL-seq},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Effective genome editing with an enhanced ISDra2 TnpB system
and deep learning-predicted ωRNAs. <em>NMETH</em>, <em>21</em>(11),
2084–2093. (<a
href="https://doi.org/10.1038/s41592-024-02418-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposon (IS200/IS605)-encoded TnpB proteins are predecessors of class 2 type V CRISPR effectors and have emerged as one of the most compact genome editors identified thus far. Here, we optimized the design of Deinococcus radiodurans (ISDra2) TnpB for application in mammalian cells (TnpBmax), leading to an average 4.4-fold improvement in editing. In addition, we developed variants mutated at position K76 that recognize alternative target-adjacent motifs (TAMs), expanding the targeting range of ISDra2 TnpB. We further generated an extensive dataset on TnpBmax editing efficiencies at 10,211 target sites. This enabled us to delineate rules for on-target and off-target editing and to devise a deep learning model, termed TnpB editing efficiency predictor (TEEP; https://www.tnpb.app ), capable of predicting ISDra2 TnpB guiding RNA (ωRNA) activity with high performance (r &amp;gt; 0.8). Employing TEEP, we achieved editing efficiencies up to 75.3% in the murine liver and 65.9% in the murine brain after adeno-associated virus (AAV) vector delivery of TnpBmax. Overall, the set of tools presented in this study facilitates the application of TnpB as an ultracompact programmable endonuclease in research and therapeutics. This work introduces engineered TnpBmax proteins with enhanced efficiency and an expanded targeting range. By leveraging an extensive dataset on editing efficiencies, it also integrates a deep learning model to predict guide RNA activity.},
  archive      = {J_NMETH},
  author       = {Marquart, Kim Fabiano and Mathis, Nicolas and Mollaysa, Amina and Müller, Saphira and Kissling, Lucas and Rothgangl, Tanja and Schmidheini, Lukas and Kulcsár, Péter István and Allam, Ahmed and Kaufmann, Masako M. and Matsushita, Mai and Haenggi, Tatjana and Cathomen, Toni and Kopf, Manfred and Krauthammer, Michael and Schwank, Gerald},
  doi          = {10.1038/s41592-024-02418-z},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2084-2093},
  shortjournal = {Nat. Meth.},
  title        = {Effective genome editing with an enhanced ISDra2 TnpB system and deep learning-predicted ωRNAs},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Interpretable spatially aware dimension reduction of spatial
transcriptomics with STAMP. <em>NMETH</em>, <em>21</em>(11), 2072–2083.
(<a href="https://doi.org/10.1038/s41592-024-02463-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics produces high-dimensional gene expression measurements with spatial context. Obtaining a biologically meaningful low-dimensional representation of such data is crucial for effective interpretation and downstream analysis. Here, we present Spatial Transcriptomics Analysis with topic Modeling to uncover spatial Patterns (STAMP), an interpretable spatially aware dimension reduction method built on a deep generative model that returns biologically relevant, low-dimensional spatial topics and associated gene modules. STAMP can analyze data ranging from a single section to multiple sections and from different technologies to time-series data, returning topics matching known biological domains and associated gene modules containing established markers highly ranked within. In a lung cancer sample, STAMP delineated cell states with supporting markers at a higher resolution than the original annotation and uncovered cancer-associated fibroblasts concentrated on the tumor edge’s exterior. In time-series data of mouse embryonic development, STAMP disentangled the erythro-myeloid hematopoiesis and hepatocytes developmental trajectories within the liver. STAMP is highly scalable and can handle more than 500,000 cells. Spatial Transcriptomics Analysis with topic Modeling to uncover spatial Patterns (STAMP) is a versatile and scalable method for dimension reduction in spatially resolved transcriptomics that enables discovery of biologically relevant tissue domains.},
  archive      = {J_NMETH},
  author       = {Zhong, Chengwei and Ang, Kok Siong and Chen, Jinmiao},
  doi          = {10.1038/s41592-024-02463-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2072-2083},
  shortjournal = {Nat. Meth.},
  title        = {Interpretable spatially aware dimension reduction of spatial transcriptomics with STAMP},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multiomic characterization of RNA microenvironments by
oligonucleotide-mediated proximity-interactome mapping. <em>NMETH</em>,
<em>21</em>(11), 2058–2071. (<a
href="https://doi.org/10.1038/s41592-024-02457-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA molecules form complex networks of molecular interactions that are central to their function and to cellular architecture. But these interaction networks are difficult to probe in situ. Here, we introduce Oligonucleotide-mediated proximity-interactome MAPping (O-MAP), a method for elucidating the biomolecules near an RNA of interest, within its native context. O-MAP uses RNA-fluorescence in situ hybridization-like oligonucleotide probes to deliver proximity-biotinylating enzymes to a target RNA in situ, enabling nearby molecules to be enriched by streptavidin pulldown. This induces exceptionally precise biotinylation that can be easily optimized and ported to new targets or sample types. Using the noncoding RNAs 47S, 7SK and Xist as models, we develop O-MAP workflows for discovering RNA-proximal proteins, transcripts and genomic loci, yielding a multiomic characterization of these RNAs’ subcellular compartments and new regulatory interactions. O-MAP requires no genetic manipulation, uses exclusively off-the-shelf parts and requires orders of magnitude fewer cells than established methods, making it accessible to most laboratories. Oligonucleotide-mediated proximity-interactome MAPping (O-MAP) enables precise multiomic characterization of biomolecular interaction networks at target RNAs. Distinct O-MAP workflows reveal RNA-adjacent proteins, transcripts and genomic loci.},
  archive      = {J_NMETH},
  author       = {Tsue, Ashley F. and Kania, Evan E. and Lei, Diana Q. and Fields, Rose and McGann, Christopher D. and Marciniak, Daphnée M. and Hershberg, Elliot A. and Deng, Xinxian and Kihiu, Maryanne and Ong, Shao-En and Disteche, Christine M. and Kugel, Sita and Beliveau, Brian J. and Schweppe, Devin K. and Shechner, David M.},
  doi          = {10.1038/s41592-024-02457-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2058-2071},
  shortjournal = {Nat. Meth.},
  title        = {Multiomic characterization of RNA microenvironments by oligonucleotide-mediated proximity-interactome mapping},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ScNanoSeq-CUT&amp;tag: A single-cell long-read CUT&amp;tag
sequencing method for efficient chromatin modification profiling within
individual cells. <em>NMETH</em>, <em>21</em>(11), 2044–2057. (<a
href="https://doi.org/10.1038/s41592-024-02453-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin modifications are fundamental epigenetic marks that determine genome functions, but it remains challenging to profile those of repetitive elements and complex genomic regions. Here, we develop scNanoSeq-CUT&amp;amp;Tag, a streamlined method, by adapting modified cleavage under targets and tagmentation (CUT&amp;amp;Tag) to the nanopore sequencing platform for genome-wide chromatin modification profiling within individual cells. We show that scNanoSeq-CUT&amp;amp;Tag can accurately profile histone marks and transcription factor occupancy patterns at single-cell resolution as well as distinguish different cell types. scNanoSeq-CUT&amp;amp;Tag efficiently maps the allele-specific chromatin modifications and allows analysis of their neighboring region co-occupancy patterns within individual cells. Moreover, scNanoSeq-CUT&amp;amp;Tag can accurately detect chromatin modifications for individual copies of repetitive elements in both human and mouse genomes. Overall, we prove that scNanoSeq-CUT&amp;amp;Tag is a valuable single-cell tool for efficiently profiling histone marks and transcription factor occupancies, especially for previously poorly studied complex genomic regions and blacklist genomic regions. This work presents scNanoSeq-CUT&amp;amp;Tag, using long-read sequencing to profile histone modifications and DNA-binding proteins at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Li, Qingqing and Guo, Yuqing and Wu, Zixin and Xu, Xueqiang and Jiang, Zhenhuan and Qi, Shuyue and Liu, Zhenyu and Wen, Lu and Tang, Fuchou},
  doi          = {10.1038/s41592-024-02453-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2044-2057},
  shortjournal = {Nat. Meth.},
  title        = {ScNanoSeq-CUT&amp;Tag: A single-cell long-read CUT&amp;Tag sequencing method for efficient chromatin modification profiling within individual cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Strainy: Phasing and assembly of strain haplotypes from
long-read metagenome sequencing. <em>NMETH</em>, <em>21</em>(11),
2034–2043. (<a
href="https://doi.org/10.1038/s41592-024-02424-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacterial species in microbial communities are often represented by mixtures of strains, distinguished by small variations in their genomes. Short-read approaches can be used to detect small-scale variation between strains but fail to phase these variants into contiguous haplotypes. Long-read metagenome assemblers can generate contiguous bacterial chromosomes but often suppress strain-level variation in favor of species-level consensus. Here we present Strainy, an algorithm for strain-level metagenome assembly and phasing from Nanopore and PacBio reads. Strainy takes a de novo metagenomic assembly as input and identifies strain variants, which are then phased and assembled into contiguous haplotypes. Using simulated and mock Nanopore and PacBio metagenome data, we show that Strainy assembles accurate and complete strain haplotypes, outperforming current Nanopore-based methods and comparable with PacBio-based algorithms in completeness and accuracy. We then use Strainy to assemble strain haplotypes of a complex environmental metagenome, revealing distinct strain distribution and mutational patterns in bacterial species. This work presents Strainy, a long-read metagenome assembler that allows the identification of strain distributions and mutational patterns in environmental metagenomes.},
  archive      = {J_NMETH},
  author       = {Kazantseva, Ekaterina and Donmez, Ataberk and Frolova, Maria and Pop, Mihai and Kolmogorov, Mikhail},
  doi          = {10.1038/s41592-024-02424-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2034-2043},
  shortjournal = {Nat. Meth.},
  title        = {Strainy: Phasing and assembly of strain haplotypes from long-read metagenome sequencing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Absolute quantitative and base-resolution sequencing reveals
comprehensive landscape of pseudouridine across the human transcriptome.
<em>NMETH</em>, <em>21</em>(11), 2024–2033. (<a
href="https://doi.org/10.1038/s41592-024-02439-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pseudouridine (Ψ) is one of the most abundant modifications in cellular RNA. However, its function remains elusive, mainly due to the lack of highly sensitive and accurate detection methods. Here, we introduced 2-bromoacrylamide-assisted cyclization sequencing (BACS), which enables Ψ-to-C transitions, for quantitative profiling of Ψ at single-base resolution. BACS allowed the precise identification of Ψ positions, especially in densely modified Ψ regions and consecutive uridine sequences. BACS detected all known Ψ sites in human rRNA and spliceosomal small nuclear RNAs and generated the quantitative Ψ map of human small nucleolar RNA and tRNA. Furthermore, BACS simultaneously detected adenosine-to-inosine editing sites and N1-methyladenosine. Depletion of pseudouridine synthases TRUB1, PUS7 and PUS1 elucidated their targets and sequence motifs. We further identified a highly abundant Ψ114 site in Epstein–Barr virus-encoded small RNA EBER2. Surprisingly, applying BACS to a panel of RNA viruses demonstrated the absence of Ψ in their viral transcripts or genomes, shedding light on differences in pseudouridylation across virus families. This study introduces a chemical method, BACS, that generates Ψ-to-C mutation signatures, allowing for sequencing and quantification of Ψ at single-base resolution.},
  archive      = {J_NMETH},
  author       = {Xu, Haiqi and Kong, Linzhen and Cheng, Jingfei and Al Moussawi, Khatoun and Chen, Xiufei and Iqbal, Aleema and Wing, Peter A. C. and Harris, James M. and Tsukuda, Senko and Embarc-Buh, Azman and Wei, Guifeng and Castello, Alfredo and Kriaucionis, Skirmantas and McKeating, Jane A. and Lu, Xin and Song, Chun-Xiao},
  doi          = {10.1038/s41592-024-02439-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2024-2033},
  shortjournal = {Nat. Meth.},
  title        = {Absolute quantitative and base-resolution sequencing reveals comprehensive landscape of pseudouridine across the human transcriptome},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Personalized pangenome references. <em>NMETH</em>,
<em>21</em>(11), 2017–2023. (<a
href="https://doi.org/10.1038/s41592-024-02407-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pangenomes reduce reference bias by representing genetic diversity better than a single reference sequence. Yet when comparing a sample to a pangenome, variants in the pangenome that are not part of the sample can be misleading, for example, causing false read mappings. These irrelevant variants are generally rarer in terms of allele frequency, and have previously been dealt with by filtering rare variants. However, this blunt heuristic both fails to remove some irrelevant variants and removes many relevant variants. We propose a new approach that imputes a personalized pangenome subgraph by sampling local haplotypes according to k-mer counts in the reads. We implement the approach in the vg toolkit ( https://github.com/vgteam/vg ) for the Giraffe short-read aligner and compare its accuracy to state-of-the-art methods using human pangenome graphs from the Human Pangenome Reference Consortium. This reduces small variant genotyping errors by four times relative to the Genome Analysis Toolkit and makes short-read structural variant genotyping of known variants competitive with long-read variant discovery methods. This work introduces a k-mer-based approach to customizing a pangenome reference, making it more relevant to a new sample of interest. This method enhances the accuracy of genotyping small variants and large structural variants.},
  archive      = {J_NMETH},
  author       = {Sirén, Jouni and Eskandar, Parsa and Ungaro, Matteo Tommaso and Hickey, Glenn and Eizenga, Jordan M. and Novak, Adam M. and Chang, Xian and Chang, Pi-Chuan and Kolmogorov, Mikhail and Carroll, Andrew and Monlong, Jean and Paten, Benedict},
  doi          = {10.1038/s41592-024-02407-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2017-2023},
  shortjournal = {Nat. Meth.},
  title        = {Personalized pangenome references},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). In vivo imaging of the human brain with the iseult 11.7-t
MRI scanner. <em>NMETH</em>, <em>21</em>(11), 2013–2016. (<a
href="https://doi.org/10.1038/s41592-024-02472-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The understanding of the human brain is one of the main scientific challenges of the twenty-first century. In the early 2000s, the French Atomic Energy Commission launched a program to conceive and build a human magnetic resonance imaging scanner operating at 11.7 T. We have now acquired human brain images in vivo at such a magnetic field. We deployed parallel transmission tools to mitigate the radiofrequency field inhomogeneity problem and tame the specific absorption rate. The safety of human imaging at such high field strength was demonstrated using physiological, vestibular, behavioral and genotoxicity measurements on the imaged volunteers. Our technology yields T2 and T2*-weighted images reaching mesoscale resolutions within short acquisition times and with a high signal and contrast-to-noise ratio. In a technological tour de force, a whole-body 11.7-T MRI scanner has been developed. Here images of the human brain are presented while safety for the imaged human volunteers has been ascertained.},
  archive      = {J_NMETH},
  author       = {Boulant, Nicolas and Mauconduit, Franck and Gras, Vincent and Amadon, Alexis and Le Ster, Caroline and Luong, Michel and Massire, Aurélien and Pallier, Christophe and Sabatier, Laure and Bottlaender, Michel and Vignaud, Alexandre and Le Bihan, Denis},
  doi          = {10.1038/s41592-024-02472-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2013-2016},
  shortjournal = {Nat. Meth.},
  title        = {In vivo imaging of the human brain with the iseult 11.7-T MRI scanner},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Building pangenome graphs. <em>NMETH</em>, <em>21</em>(11),
2008–2012. (<a
href="https://doi.org/10.1038/s41592-024-02430-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pangenome graphs can represent all variation between multiple reference genomes, but current approaches to build them exclude complex sequences or are based upon a single reference. In response, we developed the PanGenome Graph Builder, a pipeline for constructing pangenome graphs without bias or exclusion. The PanGenome Graph Builder uses all-to-all alignments to build a variation graph in which we can identify variation, measure conservation, detect recombination events and infer phylogenetic relationships. PGGB is a modular framework for efficiently building unbiased pangenome graphs, supporting diverse downstream analyses.},
  archive      = {J_NMETH},
  author       = {Garrison, Erik and Guarracino, Andrea and Heumos, Simon and Villani, Flavia and Bao, Zhigui and Tattini, Lorenzo and Hagmann, Jörg and Vorbrugg, Sebastian and Marco-Sola, Santiago and Kubica, Christian and Ashbrook, David G. and Thorell, Kaisa and Rusholme-Pilcher, Rachel L. and Liti, Gianni and Rudbeck, Emilio and Golicz, Agnieszka A. and Nahnsen, Sven and Yang, Zuyu and Mwaniki, Moses Njagi and Nobrega, Franklin L. and Wu, Yi and Chen, Hao and de Ligt, Joep and Sudmant, Peter H. and Huang, Sanwen and Weigel, Detlef and Soranzo, Nicole and Colonna, Vincenza and Williams, Robert W. and Prins, Pjotr},
  doi          = {10.1038/s41592-024-02430-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {2008-2012},
  shortjournal = {Nat. Meth.},
  title        = {Building pangenome graphs},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Controlling the sound of light: Photoswitching optoacoustic
imaging. <em>NMETH</em>, <em>21</em>(11), 1996–2007. (<a
href="https://doi.org/10.1038/s41592-024-02396-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optoacoustic (photoacoustic) imaging advances allow high-resolution optical imaging much deeper than optical microscopy. However, while label-free optoacoustics have already entered clinical application, biological imaging is in need of ubiquitous optoacoustic labels for use in ways that are similar to how fluorescent proteins propelled optical microscopy. We review photoswitching advances that shine a new light or, in analogy, ‘bring a new sound’ to biological optoacoustic imaging. Based on engineered labels and novel devices, switching uses light or other energy forms and enables signal modulation and synchronous detection for maximizing contrast and detection sensitivity over other optoacoustic labels. Herein, we explain contrast enhancement in the spectral versus temporal domains and review labels and key concepts of switching and their properties to modulate optoacoustic signals. We further outline systems and applications and discuss how switching can enable optoacoustic imaging of cellular or molecular contrast at depths and resolutions beyond those of other optical methods. This Review describes how photoswitchable probes and associated hardware innovations are poised to transform optoacoustic imaging in life sciences research.},
  archive      = {J_NMETH},
  author       = {Stiel, Andre C. and Ntziachristos, Vasilis},
  doi          = {10.1038/s41592-024-02396-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1996-2007},
  shortjournal = {Nat. Meth.},
  title        = {Controlling the sound of light: Photoswitching optoacoustic imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Exoskeleton empowers large-scale neural recordings in freely
roaming mice. <em>NMETH</em>, <em>21</em>(11), 1994–1995. (<a
href="https://doi.org/10.1038/s41592-024-02435-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A robotic neural recording system can sense the intended motion of the mouse that it is connected to, and it moves with the animal as it explores physical spaces. The robot can support recording interfaces that are much heavier and larger than mice and provide new capabilities for brain-wide recordings in locomoting mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02435-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1994-1995},
  shortjournal = {Nat. Meth.},
  title        = {Exoskeleton empowers large-scale neural recordings in freely roaming mice},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping voltage activity in a live mouse brain in 3D using
confocal light field microscopy. <em>NMETH</em>, <em>21</em>(11),
1992–1993. (<a
href="https://doi.org/10.1038/s41592-024-02459-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Voltage imaging, a promising technique for directly recording neuronal activity, faces barriers to broad application due to current limitations in compatible imaging modalities. Our team introduces an advanced confocal light field microscopy method enabling high-throughput, rapid and low-noise 3D voltage imaging in awake mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02459-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1992-1993},
  shortjournal = {Nat. Meth.},
  title        = {Mapping voltage activity in a live mouse brain in 3D using confocal light field microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SCUBA-d: A freshly trained diffusion model generates
high-quality protein structures. <em>NMETH</em>, <em>21</em>(11),
1990–1991. (<a
href="https://doi.org/10.1038/s41592-024-02465-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accuracy of SCUBA-D, a protein backbone structure diffusion model trained independently and orthogonally to existing protein structure prediction networks, is confirmed by the X-ray structures of 16 designed proteins and a protein complex, and by experimental validation of designed heme-binding proteins and Ras-binding proteins.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02465-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1990-1991},
  shortjournal = {Nat. Meth.},
  title        = {SCUBA-D: A freshly trained diffusion model generates high-quality protein structures},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Measuring multiple intracellular biochemical properties of
proteins with next-generation sequencing. <em>NMETH</em>,
<em>21</em>(11), 1988–1989. (<a
href="https://doi.org/10.1038/s41592-024-02461-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed LABEL-seq, a platform that enables measurement of protein properties and functions at scale by leveraging the intracellular self-assembly of an RNA-binding domain (RBD) and protein-encoding RNA barcode. Enrichment of RBD–protein fusions, followed by high-throughput sequencing of the co-enriched barcodes, enables the profiling of protein abundance, activity, interactions and druggability at scale.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02461-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1988-1989},
  shortjournal = {Nat. Meth.},
  title        = {Measuring multiple intracellular biochemical properties of proteins with next-generation sequencing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Expanding the genome editing toolbox with designer
CRISPR–cas-like transposons. <em>NMETH</em>, <em>21</em>(11), 1986–1987.
(<a href="https://doi.org/10.1038/s41592-024-02460-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Similarly to CRISPR–Cas systems, TnpB proteins from bacterial transposons can be employed as RNA-guided endonucleases for genome editing. By combining rational protein design and machine learning, ISDra2 TnpB variants with enhanced editing efficiency and a broader targeting range were developed, along with a prediction tool to design effective guiding RNAs.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02460-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1986-1987},
  shortjournal = {Nat. Meth.},
  title        = {Expanding the genome editing toolbox with designer CRISPR–Cas-like transposons},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A method for quantitative and base-resolution sequencing of
pseudouridine. <em>NMETH</em>, <em>21</em>(11), 1984–1985. (<a
href="https://doi.org/10.1038/s41592-024-02466-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative and sensitive methods for the detection of pseudouridine (Ψ) have been lacking. Now, a method termed 2-bromoacrylamide-assisted cyclization sequencing (BACS) has been developed that enables the accurate quantification of Ψ stoichiometry, precise identification of Ψ positions and robust detection of densely modified Ψ sites.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02466-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1984-1985},
  shortjournal = {Nat. Meth.},
  title        = {A method for quantitative and base-resolution sequencing of pseudouridine},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ultra-high-field MRI for fast imaging of the human brain at
mesoscale resolution. <em>NMETH</em>, <em>21</em>(11), 1982–1983. (<a
href="https://doi.org/10.1038/s41592-024-02473-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Very high-resolution images of the human brain obtained in vivo in a few minutes with MRI at an ultra-high magnetic field of 11.7 T reveal exquisite details. Biological and behavioral tests confirm the safety of the method, opening the door for human brain exploration at mesoscale resolution.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02473-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1982-1983},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-field MRI for fast imaging of the human brain at mesoscale resolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Constructing and personalizing population pangenome graphs.
<em>NMETH</em>, <em>21</em>(11), 1980–1981. (<a
href="https://doi.org/10.1038/s41592-024-02402-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pangenome graphs signify a new frontier in genome representation. Recent advances in constructing and personalizing them mark progress in this area.},
  archive      = {J_NMETH},
  author       = {Chikhi, Rayan and Dufresne, Yoann and Medvedev, Paul},
  doi          = {10.1038/s41592-024-02402-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1980-1981},
  shortjournal = {Nat. Meth.},
  title        = {Constructing and personalizing population pangenome graphs},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pushing the limits of MRI brain imaging. <em>NMETH</em>,
<em>21</em>(11), 1975–1979. (<a
href="https://doi.org/10.1038/s41592-024-02489-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new generation of increasingly powerful magnets is dramatically extending the resolution, speed and analytical capabilities of magnetic resonance imaging for brain research.},
  archive      = {J_NMETH},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41592-024-02489-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1975-1979},
  shortjournal = {Nat. Meth.},
  title        = {Pushing the limits of MRI brain imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Multi-pass nanopore for single-molecule protein sequencing.
<em>NMETH</em>, <em>21</em>(11), 1974. (<a
href="https://doi.org/10.1038/s41592-024-02509-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02509-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1974},
  shortjournal = {Nat. Meth.},
  title        = {Multi-pass nanopore for single-molecule protein sequencing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Fallopian tube assembloids. <em>NMETH</em>,
<em>21</em>(11), 1974. (<a
href="https://doi.org/10.1038/s41592-024-02510-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02510-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1974},
  shortjournal = {Nat. Meth.},
  title        = {Fallopian tube assembloids},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). A leap for mesoscale imaging. <em>NMETH</em>,
<em>21</em>(11), 1973. (<a
href="https://doi.org/10.1038/s41592-024-02508-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A computational mesoscale microscope offers large-scale volumetric imaging of intravital dynamics with cellular resolution.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02508-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1973},
  shortjournal = {Nat. Meth.},
  title        = {A leap for mesoscale imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Making fluorescence-based integrative structures and
associated kinetic information accessible. <em>NMETH</em>,
<em>21</em>(11), 1970–1972. (<a
href="https://doi.org/10.1038/s41592-024-02428-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hanke, Christian A. and Westbrook, John D. and Webb, Benjamin M. and Peulen, Thomas-O. and Lawson, Catherine L. and Sali, Andrej and Berman, Helen M. and Seidel, Claus A. M. and Vallat, Brinda},
  doi          = {10.1038/s41592-024-02428-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1970-1972},
  shortjournal = {Nat. Meth.},
  title        = {Making fluorescence-based integrative structures and associated kinetic information accessible},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Navigate: An open-source platform for smart light-sheet
microscopy. <em>NMETH</em>, <em>21</em>(11), 1967–1969. (<a
href="https://doi.org/10.1038/s41592-024-02413-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Marin, Zach and Wang, Xiaoding and Collison, Dax W. and McFadden, Conor and Lin, Jinlong and Borges, Hazel M. and Chen, Bingying and Mehra, Dushyant and Shen, Qionghua and Gałecki, Seweryn and Daetwyler, Stephan and Sheppard, Steven J. and Thien, Phu and Porter, Baylee A. and Conzen, Suzanne D. and Shepherd, Douglas P. and Fiolka, Reto and Dean, Kevin M.},
  doi          = {10.1038/s41592-024-02413-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1967-1969},
  shortjournal = {Nat. Meth.},
  title        = {Navigate: An open-source platform for smart light-sheet microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The bearded dragon pogona vitticeps. <em>NMETH</em>,
<em>21</em>(11), 1964–1966. (<a
href="https://doi.org/10.1038/s41592-024-02485-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Australian bearded dragon is so called for its distinctive ‘beard’ of spiky scales that can darken and expand during social and defensive displays. This lizard has become a reptilian model system to study the evolution, function and dynamics of neurons and neural circuits (including during sleep) in the amniote brain.},
  archive      = {J_NMETH},
  author       = {Fenk, Lorenz A. and Baier, Felix and Laurent, Gilles},
  doi          = {10.1038/s41592-024-02485-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1964-1966},
  shortjournal = {Nat. Meth.},
  title        = {The bearded dragon pogona vitticeps},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Microscopic art. <em>NMETH</em>, <em>21</em>(11), 1961. (<a
href="https://doi.org/10.1038/s41592-024-02531-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a pictorial Editorial this month, we celebrate the beauty of microscopy images.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02531-z},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1961},
  shortjournal = {Nat. Meth.},
  title        = {Microscopic art},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024q). Scientists who marry scientists. <em>NMETH</em>,
<em>21</em>(11), 1962–1963. (<a
href="https://doi.org/10.1038/s41592-024-02490-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When spouses are both scientists, they mix the typical research career decisions with some marriage-related ones.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02490-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {11},
  pages        = {1962-1963},
  shortjournal = {Nat. Meth.},
  title        = {Scientists who marry scientists},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: BigNeuron: A resource to benchmark and
predict performance of algorithms for automated tracing of neurons in
light microscopy datasets. <em>NMETH</em>, <em>21</em>(10), 1959. (<a
href="https://doi.org/10.1038/s41592-024-02395-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Manubens-Gil, Linus and Zhou, Zhi and Chen, Hanbo and Ramanathan, Arvind and Liu, Xiaoxiao and Liu, Yufeng and Bria, Alessandro and Gillette, Todd and Ruan, Zongcai and Yang, Jian and Radojević, Miroslav and Zhao, Ting and Cheng, Li and Qu, Lei and Liu, Siqi and Bouchard, Kristofer E. and Gu, Lin and Cai, Weidong and Ji, Shuiwang and Roysam, Badrinath and Wang, Ching-Wei and Yu, Hongchuan and Sironi, Amos and Iascone, Daniel Maxim and Zhou, Jie and Bas, Erhan and Conde-Sousa, Eduardo and Aguiar, Paulo and Li, Xiang and Li, Yujie and Nanda, Sumit and Wang, Yuan and Muresan, Leila and Fua, Pascal and Ye, Bing and He, Hai-yan and Staiger, Jochen F. and Peter, Manuel and Cox, Daniel N. and Simonneau, Michel and Oberlaender, Marcel and Jefferis, Gregory and Ito, Kei and Gonzalez-Bellido, Paloma and Kim, Jinhyun and Rubel, Edwin and Cline, Hollis T. and Zeng, Hongkui and Nern, Aljoscha and Chiang, Ann-Shyn and Yao, Jianhua and Roskams, Jane and Livesey, Rick and Stevens, Janine and Liu, Tianming and Dang, Chinh and Guo, Yike and Zhong, Ning and Tourassi, Georgia and Hill, Sean and Hawrylycz, Michael and Koch, Christof and Meijering, Erik and Ascoli, Giorgio A. and Peng, Hanchuan},
  doi          = {10.1038/s41592-024-02395-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1959},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: BigNeuron: a resource to benchmark and predict performance of algorithms for automated tracing of neurons in light microscopy datasets},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: CRISPR-array-mediated imaging of
non-repetitive and multiplex genomic loci in living cells.
<em>NMETH</em>, <em>21</em>(10), 1958. (<a
href="https://doi.org/10.1038/s41592-024-02394-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Yang, Liang-Zhong and Min, Yi-Hui and Liu, Yu-Xin and Gao, Bao-Qing and Liu, Xiao-Qi and Huang, Youkui and Wang, Haifeng and Yang, Li and Liu, Zhe J. and Chen, Ling-Ling},
  doi          = {10.1038/s41592-024-02394-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1958},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: CRISPR-array-mediated imaging of non-repetitive and multiplex genomic loci in living cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genomics 2 proteins portal: A resource and discovery tool
for linking genetic screening outputs to protein sequences and
structures. <em>NMETH</em>, <em>21</em>(10), 1947–1957. (<a
href="https://doi.org/10.1038/s41592-024-02409-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in AI-based methods have revolutionized the field of structural biology. Concomitantly, high-throughput sequencing and functional genomics have generated genetic variants at an unprecedented scale. However, efficient tools and resources are needed to link disparate data types—to ‘map’ variants onto protein structures, to better understand how the variation causes disease, and thereby design therapeutics. Here we present the Genomics 2 Proteins portal ( https://g2p.broadinstitute.org/ ): a human proteome-wide resource that maps 20,076,998 genetic variants onto 42,413 protein sequences and 77,923 structures, with a comprehensive set of structural and functional features. Additionally, the Genomics 2 Proteins portal allows users to interactively upload protein residue-wise annotations (for example, variants and scores) as well as the protein structure beyond databases to establish the connection between genomics to proteins. The portal serves as an easy-to-use discovery tool for researchers and scientists to hypothesize the structure–function relationship between natural or synthetic variations and their molecular phenotypes. The Genomics 2 Proteins portal is an open-source tool for proteome-wide linking of human genetic variants to protein sequences and structures. The portal serves as a discovery tool to hypothesize the structure–function relationship between natural or synthetic variations and their molecular phenotypes.},
  archive      = {J_NMETH},
  author       = {Kwon, Seulki and Safer, Jordan and Nguyen, Duyen T. and Hoksza, David and May, Patrick and Arbesfeld, Jeremy A. and Rubin, Alan F. and Campbell, Arthur J. and Burgin, Alex and Iqbal, Sumaiya},
  doi          = {10.1038/s41592-024-02409-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1947-1957},
  shortjournal = {Nat. Meth.},
  title        = {Genomics 2 proteins portal: A resource and discovery tool for linking genetic screening outputs to protein sequences and structures},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Collaborative augmented reconstruction of 3D neuron
morphology in mouse and human brains. <em>NMETH</em>, <em>21</em>(10),
1936–1946. (<a
href="https://doi.org/10.1038/s41592-024-02401-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Digital reconstruction of the intricate 3D morphology of individual neurons from microscopic images is a crucial challenge in both individual laboratories and large-scale projects focusing on cell types and brain anatomy. This task often fails in both conventional manual reconstruction and state-of-the-art artificial intelligence (AI)-based automatic reconstruction algorithms. It is also challenging to organize multiple neuroanatomists to generate and cross-validate biologically relevant and mutually agreed upon reconstructions in large-scale data production. Based on collaborative group intelligence augmented by AI, we developed a collaborative augmented reconstruction (CAR) platform for neuron reconstruction at scale. This platform allows for immersive interaction and efficient collaborative editing of neuron anatomy using a variety of devices, such as desktop workstations, virtual reality headsets and mobile phones, enabling users to contribute anytime and anywhere and to take advantage of several AI-based automation tools. We tested CAR’s applicability for challenging mouse and human neurons toward scaled and faithful data production. Collaborative augmented reconstruction (CAR) is a platform for large-scale reconstruction of neurons and other cells from multi-dimensional imaging datasets. It can be accessed from a variety of devices simultaneously for efficient and accurate reconstruction.},
  archive      = {J_NMETH},
  author       = {Zhang, Lingli and Huang, Lei and Yuan, Zexin and Hang, Yuning and Zeng, Ying and Li, Kaixiang and Wang, Lijun and Zeng, Haoyu and Chen, Xin and Zhang, Hairuo and Xi, Jiaqi and Chen, Danni and Gao, Ziqin and Le, Longxin and Chen, Jie and Ye, Wen and Liu, Lijuan and Wang, Yimin and Peng, Hanchuan},
  doi          = {10.1038/s41592-024-02401-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1936-1946},
  shortjournal = {Nat. Meth.},
  title        = {Collaborative augmented reconstruction of 3D neuron morphology in mouse and human brains},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Gapr for large-scale collaborative single-neuron
reconstruction. <em>NMETH</em>, <em>21</em>(10), 1926–1935. (<a
href="https://doi.org/10.1038/s41592-024-02345-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-brain analysis of single-neuron morphology is crucial for unraveling the complex structure of the brain. However, large-scale neuron reconstruction from terabyte and even petabyte data of mammalian brains generated by state-of-the-art light microscopy is a daunting task. Here, we developed ‘Gapr’ (Gapr accelerates projectome reconstruction) that streamlines deep learning-based automatic reconstruction, ‘automatic proofreading’ that reduces human workloads at high-confidence sites, and high-throughput collaborative proofreading by crowd users through the Internet. Furthermore, Gapr offers a seamless user interface that ensures high proofreading speed per annotator, on-demand conversion for handling large datasets, flexible workflows tailored to diverse datasets and rigorous error tracking for quality control. Finally, we demonstrated Gapr’s efficacy by reconstructing over 4,000 neurons in mouse brains, revealing the morphological diversity in cortical interneurons and hypothalamic neurons. Here, we present Gapr as a solution for large-scale single-neuron reconstruction projects. Gapr is an efficient platform for reconstructing neurons in large-scale light microscopy datasets. It enables various proofreading modes as well as collaboration among many annotators.},
  archive      = {J_NMETH},
  author       = {Gou, Lingfeng and Wang, Yanzhi and Gao, Le and Zhong, Yiting and Xie, Lucheng and Wang, Haifang and Zha, Xi and Shao, Yinqi and Xu, Huatai and Xu, Xiaohong and Yan, Jun},
  doi          = {10.1038/s41592-024-02345-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1926-1935},
  shortjournal = {Nat. Meth.},
  title        = {Gapr for large-scale collaborative single-neuron reconstruction},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A modular chemigenetic calcium indicator for multiplexed in
vivo functional imaging. <em>NMETH</em>, <em>21</em>(10), 1916–1925. (<a
href="https://doi.org/10.1038/s41592-024-02411-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded fluorescent calcium indicators allow cellular-resolution recording of physiology. However, bright, genetically targetable indicators that can be multiplexed with existing tools in vivo are needed for simultaneous imaging of multiple signals. Here we describe WHaloCaMP, a modular chemigenetic calcium indicator built from bright dye-ligands and protein sensor domains. Fluorescence change in WHaloCaMP results from reversible quenching of the bound dye via a strategically placed tryptophan. WHaloCaMP is compatible with rhodamine dye-ligands that fluoresce from green to near-infrared, including several that efficiently label the brain in animals. When bound to a near-infrared dye-ligand, WHaloCaMP shows a 7× increase in fluorescence intensity and a 2.1-ns increase in fluorescence lifetime upon calcium binding. We use WHaloCaMP1a to image Ca2+ responses in vivo in flies and mice, to perform three-color multiplexed functional imaging of hundreds of neurons and astrocytes in zebrafish larvae and to quantify Ca2+ concentration using fluorescence lifetime imaging microscopy (FLIM). WHaloCaMP is a chemigenetic calcium indicator that can be combined with different rhodamine dyes for multiplexed or FLIM imaging in vivo, as demonstrated for calcium imaging in neuronal cultures, brain slices, Drosophila, zebrafish larvae and the mouse brain.},
  archive      = {J_NMETH},
  author       = {Farrants, Helen and Shuai, Yichun and Lemon, William C. and Monroy Hernandez, Christian and Zhang, Deng and Yang, Shang and Patel, Ronak and Qiao, Guanda and Frei, Michelle S. and Plutkis, Sarah E. and Grimm, Jonathan B. and Hanson, Timothy L. and Tomaska, Filip and Turner, Glenn C. and Stringer, Carsen and Keller, Philipp J. and Beyene, Abraham G. and Chen, Yao and Liang, Yajie and Lavis, Luke D. and Schreiter, Eric R.},
  doi          = {10.1038/s41592-024-02411-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1916-1925},
  shortjournal = {Nat. Meth.},
  title        = {A modular chemigenetic calcium indicator for multiplexed in vivo functional imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ECLiPSE: A versatile classification technique for structural
and morphological analysis of 2D and 3D single-molecule localization
microscopy data. <em>NMETH</em>, <em>21</em>(10), 1909–1915. (<a
href="https://doi.org/10.1038/s41592-024-02414-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy (SMLM) has gained widespread use for visualizing the morphology of subcellular organelles and structures with nanoscale spatial resolution. However, analysis tools for automatically quantifying and classifying SMLM images have lagged behind. Here we introduce Enhanced Classification of Localized Point clouds by Shape Extraction (ECLiPSE), an automated machine learning analysis pipeline specifically designed to classify cellular structures captured through two-dimensional or three-dimensional SMLM. ECLiPSE leverages a comprehensive set of shape descriptors, the majority of which are directly extracted from the localizations to minimize bias during the characterization of individual structures. ECLiPSE has been validated using both unsupervised and supervised classification on datasets, including various cellular structures, achieving near-perfect accuracy. We apply two-dimensional ECLiPSE to classify morphologically distinct protein aggregates relevant for neurodegenerative diseases. Additionally, we employ three-dimensional ECLiPSE to identify relevant biological differences between healthy and depolarized mitochondria. ECLiPSE will enhance the way we study cellular structures across various biological contexts. Enhanced Classification of Localized Point clouds by Shape Extraction (ECLiPSE) is a robust feature extraction and classification pipeline for diverse and heterogeneous structures in both 2D and 3D single-molecule localization microscopy data.},
  archive      = {J_NMETH},
  author       = {Hugelier, Siewert and Tang, Qing and Kim, Hannah Hyun-Sook and Gyparaki, Melina Theoni and Bond, Charles and Santiago-Ruiz, Adriana Naomi and Porta, Sílvia and Lakadamyali, Melike},
  doi          = {10.1038/s41592-024-02414-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1909-1915},
  shortjournal = {Nat. Meth.},
  title        = {ECLiPSE: A versatile classification technique for structural and morphological analysis of 2D and 3D single-molecule localization microscopy data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Self-inspired learning for denoising live-cell
super-resolution microscopy. <em>NMETH</em>, <em>21</em>(10), 1895–1908.
(<a href="https://doi.org/10.1038/s41592-024-02400-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Every collected photon is precious in live-cell super-resolution (SR) microscopy. Here, we describe a data-efficient, deep learning-based denoising solution to improve diverse SR imaging modalities. The method, SN2N, is a Self-inspired Noise2Noise module with self-supervised data generation and self-constrained learning process. SN2N is fully competitive with supervised learning methods and circumvents the need for large training set and clean ground truth, requiring only a single noisy frame for training. We show that SN2N improves photon efficiency by one-to-two orders of magnitude and is compatible with multiple imaging modalities for volumetric, multicolor, time-lapse SR microscopy. We further integrated SN2N into different SR reconstruction algorithms to effectively mitigate image artifacts. We anticipate SN2N will enable improved live-SR imaging and inspire further advances. SN2N, a Self-inspired Noise2Noise module, offers a versatile solution for volumetric time-lapse super-resolution imaging of live cells. SN2N uses self-supervised data generation and self-constrained learning for training with a single noisy frame.},
  archive      = {J_NMETH},
  author       = {Qu, Liying and Zhao, Shiqun and Huang, Yuanyuan and Ye, Xianxin and Wang, Kunhao and Liu, Yuzhen and Liu, Xianming and Mao, Heng and Hu, Guangwei and Chen, Wei and Guo, Changliang and He, Jiaye and Tan, Jiubin and Li, Haoyu and Chen, Liangyi and Zhao, Weisong},
  doi          = {10.1038/s41592-024-02400-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1895-1908},
  shortjournal = {Nat. Meth.},
  title        = {Self-inspired learning for denoising live-cell super-resolution microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Arkitekt: Streaming analysis and real-time workflows for
microscopy. <em>NMETH</em>, <em>21</em>(10), 1884–1894. (<a
href="https://doi.org/10.1038/s41592-024-02404-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative microscopy workflows have evolved dramatically over the past years, progressively becoming more complex with the emergence of deep learning. Long-standing challenges such as three-dimensional segmentation of complex microscopy data can finally be addressed, and new imaging modalities are breaking records in both resolution and acquisition speed, generating gigabytes if not terabytes of data per day. With this shift in bioimage workflows comes an increasing need for efficient orchestration and data management, necessitating multitool interoperability and the ability to span dedicated computing resources. However, existing solutions are still limited in their flexibility and scalability and are usually restricted to offline analysis. Here we introduce Arkitekt, an open-source middleman between users and bioimage apps that enables complex quantitative microscopy workflows in real time. It allows the orchestration of popular bioimage software locally or remotely in a reliable and efficient manner. It includes visualization and analysis modules, but also mechanisms to execute source code and pilot acquisition software, making ‘smart microscopy’ a reality. Arkitekt is an open-source platform that facilitates the implementation of complex quantitative bioimaging workflows in real time, from acquisition to visualization and analysis.},
  archive      = {J_NMETH},
  author       = {Roos, Johannes and Bancelin, Stéphane and Delaire, Tom and Wilhelmi, Alexander and Levet, Florian and Engelhardt, Maren and Viasnoff, Virgile and Galland, Rémi and Nägerl, U. Valentin and Sibarita, Jean-Baptiste},
  doi          = {10.1038/s41592-024-02404-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1884-1894},
  shortjournal = {Nat. Meth.},
  title        = {Arkitekt: Streaming analysis and real-time workflows for microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). POLCAM: Instant molecular orientation microscopy for the
life sciences. <em>NMETH</em>, <em>21</em>(10), 1873–1883. (<a
href="https://doi.org/10.1038/s41592-024-02382-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for single-molecule orientation localization microscopy (SMOLM) require optical setups and algorithms that can be prohibitively slow and complex, limiting widespread adoption for biological applications. We present POLCAM, a simplified SMOLM method based on polarized detection using a polarization camera, which can be easily implemented on any wide-field fluorescence microscope. To make polarization cameras compatible with single-molecule detection, we developed theory to minimize field-of-view errors, used simulations to optimize experimental design and developed a fast algorithm based on Stokes parameter estimation that can operate over 1,000-fold faster than the state of the art, enabling near-instant determination of molecular anisotropy. To aid in the adoption of POLCAM, we developed open-source image analysis software and a website detailing hardware installation and software use. To illustrate the potential of POLCAM in the life sciences, we applied our method to study α-synuclein fibrils, the actin cytoskeleton of mammalian cells, fibroblast-like cells and the plasma membrane of live human T cells. Combining localization and polarization microscopy can yield detailed insights into subcellular structures. POLCAM uses a polarization camera and wide-field microscopy for rapid measurement of super-resolution orientation imaging in live cells.},
  archive      = {J_NMETH},
  author       = {Bruggeman, Ezra and Zhang, Oumeng and Needham, Lisa-Maria and Körbel, Markus and Daly, Sam and Cheetham, Matthew and Peters, Ruby and Wu, Tingting and Klymchenko, Andrey S. and Davis, Simon J. and Paluch, Ewa K. and Klenerman, David and Lew, Matthew D. and O’Holleran, Kevin and Lee, Steven F.},
  doi          = {10.1038/s41592-024-02382-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1873-1883},
  shortjournal = {Nat. Meth.},
  title        = {POLCAM: Instant molecular orientation microscopy for the life sciences},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MiLoPYP: Self-supervised molecular pattern mining and
particle localization in situ. <em>NMETH</em>, <em>21</em>(10),
1863–1872. (<a
href="https://doi.org/10.1038/s41592-024-02403-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography allows the routine visualization of cellular landscapes in three dimensions at nanometer-range resolutions. When combined with single-particle tomography, it is possible to obtain near-atomic resolution structures of frequently occurring macromolecules within their native environment. Two outstanding challenges associated with cryo-electron tomography/single-particle tomography are the automatic identification and localization of proteins, tasks that are hindered by the molecular crowding inside cells, imaging distortions characteristic of cryo-electron tomography tomograms and the sheer size of tomographic datasets. Current methods suffer from low accuracy, demand extensive and time-consuming manual labeling or are limited to the detection of specific types of proteins. Here, we present MiLoPYP, a two-step dataset-specific contrastive learning-based framework that enables fast molecular pattern mining followed by accurate protein localization. MiLoPYP’s ability to effectively detect and localize a wide range of targets including globular and tubular complexes as well as large membrane proteins, will contribute to streamline and broaden the applicability of high-resolution workflows for in situ structure determination. MiLoPYP is a two-step, dataset-specific contrastive learning-based method for fast and accurate detection and localization of a diverse range of target structures in cryo-electron tomography data, enabling improved in situ structural biology.},
  archive      = {J_NMETH},
  author       = {Huang, Qinwen and Zhou, Ye and Bartesaghi, Alberto},
  doi          = {10.1038/s41592-024-02403-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1863-1872},
  shortjournal = {Nat. Meth.},
  title        = {MiLoPYP: Self-supervised molecular pattern mining and particle localization in situ},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DynaMight: Estimating molecular motions with improved
reconstruction from cryo-EM images. <em>NMETH</em>, <em>21</em>(10),
1855–1862. (<a
href="https://doi.org/10.1038/s41592-024-02377-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How to deal with continuously flexing molecules is one of the biggest outstanding challenges in single-particle analysis of proteins from cryogenic-electron microscopy (cryo-EM) images. Here, we present DynaMight, a software tool that estimates a continuous space of conformations in a cryo-EM dataset by learning three-dimensional deformations of a Gaussian pseudo-atomic model of a consensus structure for every particle image. Inversion of the learned deformations is then used to obtain an improved reconstruction of the consensus structure. We illustrate the performance of DynaMight for several experimental cryo-EM datasets. We also show how error estimates on the deformations may be obtained by independently training two variational autoencoders on half sets of the cryo-EM data, and how regularization of the three-dimensional deformations through the use of atomic models may lead to important artifacts due to model bias. DynaMight is distributed as free, open-source software, as part of RELION-5. DynaMight models continuous structural heterogeneity in cryo-EM datasets, leading to an improved reconstruction of the consensus structure. The study also explores the issue of overfitting when modeling structural flexibility.},
  archive      = {J_NMETH},
  author       = {Schwab, Johannes and Kimanius, Dari and Burt, Alister and Dendooven, Tom and Scheres, Sjors H. W.},
  doi          = {10.1038/s41592-024-02377-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1855-1862},
  shortjournal = {Nat. Meth.},
  title        = {DynaMight: Estimating molecular motions with improved reconstruction from cryo-EM images},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). FICTURE: Scalable segmentation-free analysis of
submicron-resolution spatial transcriptomics. <em>NMETH</em>,
<em>21</em>(10), 1843–1854. (<a
href="https://doi.org/10.1038/s41592-024-02415-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics (ST) technologies have advanced to enable transcriptome-wide gene expression analysis at submicron resolution over large areas. However, analysis of high-resolution ST is often challenged by complex tissue structure, where existing cell segmentation methods struggle due to the irregular cell sizes and shapes, and by the absence of segmentation-free methods scalable to whole-transcriptome analysis. Here we present FICTURE (Factor Inference of Cartographic Transcriptome at Ultra-high REsolution), a segmentation-free spatial factorization method that can handle transcriptome-wide data labeled with billions of submicron-resolution spatial coordinates and is compatible with both sequencing-based and imaging-based ST data. FICTURE uses the multilayered Dirichlet model for stochastic variational inference of pixel-level spatial factors, and is orders of magnitude more efficient than existing methods. FICTURE reveals the microscopic ST architecture for challenging tissues, such as vascular, fibrotic, muscular and lipid-laden areas in real data where previous methods failed. FICTURE’s cross-platform generality, scalability and precision make it a powerful tool for exploring high-resolution ST. FICTURE is a segmentation-free approach for identifying tissue architecture in spatial transcriptomics data. FICTURE is compatible with both imaging-based and sequencing-based methods and is uniquely suited for handling the largest available datasets.},
  archive      = {J_NMETH},
  author       = {Si, Yichen and Lee, ChangHee and Hwang, Yongha and Yun, Jeong H. and Cheng, Weiqiu and Cho, Chun-Seok and Quiros, Miguel and Nusrat, Asma and Zhang, Weizhou and Jun, Goo and Zöllner, Sebastian and Lee, Jun Hee and Kang, Hyun Min},
  doi          = {10.1038/s41592-024-02415-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1843-1854},
  shortjournal = {Nat. Meth.},
  title        = {FICTURE: Scalable segmentation-free analysis of submicron-resolution spatial transcriptomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping cellular interactions from spatially resolved
transcriptomics data. <em>NMETH</em>, <em>21</em>(10), 1830–1842. (<a
href="https://doi.org/10.1038/s41592-024-02408-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell–cell communication (CCC) is essential to how life forms and functions. However, accurate, high-throughput mapping of how expression of all genes in one cell affects expression of all genes in another cell is made possible only recently through the introduction of spatially resolved transcriptomics (SRT) technologies, especially those that achieve single-cell resolution. Nevertheless, substantial challenges remain to analyze such highly complex data properly. Here, we introduce a multiple-instance learning framework, Spacia, to detect CCCs from data generated by SRTs, by uniquely exploiting their spatial modality. We highlight Spacia’s power to overcome fundamental limitations of popular analytical tools for inference of CCCs, including losing single-cell resolution, limited to ligand–receptor relationships and prior interaction databases, high false positive rates and, most importantly, the lack of consideration of the multiple-sender-to-one-receiver paradigm. We evaluated the fitness of Spacia for three commercialized single-cell resolution SRT technologies: MERSCOPE/Vizgen, CosMx/NanoString and Xenium/10x. Overall, Spacia represents a notable step in advancing quantitative theories of cellular communications. Spacia is a multiple-instance learning model for cell–cell communication (CCC) interference in single-cell resolution spatially resolved transcriptomics data. Spacia can map complex CCCs by modeling cell proximity and CCC-driven gene perturbation.},
  archive      = {J_NMETH},
  author       = {Zhu, James and Wang, Yunguan and Chang, Woo Yong and Malewska, Alicia and Napolitano, Fabiana and Gahan, Jeffrey C. and Unni, Nisha and Zhao, Min and Yuan, Rongqing and Wu, Fangjiang and Yue, Lauren and Guo, Lei and Zhao, Zhuo and Chen, Danny Z. and Hannan, Raquibul and Zhang, Siyuan and Xiao, Guanghua and Mu, Ping and Hanker, Ariella B. and Strand, Douglas and Arteaga, Carlos L. and Desai, Neil and Wang, Xinlei and Xie, Yang and Wang, Tao},
  doi          = {10.1038/s41592-024-02408-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1830-1842},
  shortjournal = {Nat. Meth.},
  title        = {Mapping cellular interactions from spatially resolved transcriptomics data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Search and match across spatial omics samples at single-cell
resolution. <em>NMETH</em>, <em>21</em>(10), 1818–1829. (<a
href="https://doi.org/10.1038/s41592-024-02410-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics technologies characterize tissue molecular properties with spatial information, but integrating and comparing spatial data across different technologies and modalities is challenging. A comparative analysis tool that can search, match and visualize both similarities and differences of molecular features in space across multiple samples is lacking. To address this, we introduce CAST (cross-sample alignment of spatial omics), a deep graph neural network-based method enabling spatial-to-spatial searching and matching at the single-cell level. CAST aligns tissues based on intrinsic similarities of spatial molecular features and reconstructs spatially resolved single-cell multi-omic profiles. CAST further allows spatially resolved differential analysis (∆Analysis) to pinpoint and visualize disease-associated molecular pathways and cell–cell interactions and single-cell relative translational efficiency profiling to reveal variations in translational control across cell types and regions. CAST serves as an integrative framework for seamless single-cell spatial data searching and matching across technologies, modalities and sample conditions. CAST is a deep learning-based method that enables across-sample searching and matching based on spatial molecular features and reconstructing spatially resolved single-cell multi-omic profiles, as well as supports downstream differential analysis.},
  archive      = {J_NMETH},
  author       = {Tang, Zefang and Luo, Shuchen and Zeng, Hu and Huang, Jiahao and Sui, Xin and Wu, Morgan and Wang, Xiao},
  doi          = {10.1038/s41592-024-02410-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1818-1829},
  shortjournal = {Nat. Meth.},
  title        = {Search and match across spatial omics samples at single-cell resolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inferring pattern-driving intercellular flows from
single-cell and spatial transcriptomics. <em>NMETH</em>,
<em>21</em>(10), 1806–1817. (<a
href="https://doi.org/10.1038/s41592-024-02380-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {From single-cell RNA-sequencing (scRNA-seq) and spatial transcriptomics (ST), one can extract high-dimensional gene expression patterns that can be described by intercellular communication networks or decoupled gene modules. These two descriptions of information flow are often assumed to occur independently. However, intercellular communication drives directed flows of information that are mediated by intracellular gene modules, in turn triggering outflows of other signals. Methodologies to describe such intercellular flows are lacking. We present FlowSig, a method that infers communication-driven intercellular flows from scRNA-seq or ST data using graphical causal modeling and conditional independence. We benchmark FlowSig using newly generated experimental cortical organoid data and synthetic data generated from mathematical modeling. We demonstrate FlowSig’s utility by applying it to various studies, showing that FlowSig can capture stimulation-induced changes to paracrine signaling in pancreatic islets, demonstrate shifts in intercellular flows due to increasing COVID-19 severity and reconstruct morphogen-driven activator–inhibitor patterns in mouse embryogenesis. Using single-cell and spatial transcriptomics data, FlowSig provides a unified signaling modeling framework by connecting intercellular communication mediated by ligand–receptor interactions and intracellular gene expression modules.},
  archive      = {J_NMETH},
  author       = {Almet, Axel A. and Tsai, Yuan-Chen and Watanabe, Momoko and Nie, Qing},
  doi          = {10.1038/s41592-024-02380-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1806-1817},
  shortjournal = {Nat. Meth.},
  title        = {Inferring pattern-driving intercellular flows from single-cell and spatial transcriptomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Simultaneous multicolor fluorescence imaging using PSF
splitting. <em>NMETH</em>, <em>21</em>(10), 1801–1805. (<a
href="https://doi.org/10.1038/s41592-024-02383-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a way to encode more information in fluorescence imaging by splitting the original point spread function (PSF), which offers broadband operation and compatibility with other PSF engineering modalities and existing analysis tools. We demonstrate the approach using the ‘Circulator’, an add-on that encodes the fluorophore emission band into the PSF, enabling simultaneous multicolor super-resolution and single-molecule microscopy using essentially the full field of view. Point spread function (PSF) splitting with the ‘Circulator’, which encodes the fluorophore emission band into the PSF, improves the information content of fluorescence microscopy and enables improved super-resolution imaging and single-particle tracking.},
  archive      = {J_NMETH},
  author       = {Van den Eynde, Robin and Hertel, Fabian and Abakumov, Sergey and Krajnik, Bartosz and Hugelier, Siewert and Auer, Alexander and Hellmeier, Joschka and Schlichthaerle, Thomas and Grattan, Rachel M. and Lidke, Diane S. and Jungmann, Ralf and Leutenegger, Marcel and Vandenberg, Wim and Dedecker, Peter},
  doi          = {10.1038/s41592-024-02383-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1801-1805},
  shortjournal = {Nat. Meth.},
  title        = {Simultaneous multicolor fluorescence imaging using PSF splitting},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integration of mass cytometry and mass spectrometry imaging
for spatially resolved single-cell metabolic profiling. <em>NMETH</em>,
<em>21</em>(10), 1796–1800. (<a
href="https://doi.org/10.1038/s41592-024-02392-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The integration of spatial omics technologies can provide important insights into the biology of tissues. Here we combined mass spectrometry imaging-based metabolomics and imaging mass cytometry-based immunophenotyping on a single tissue section to reveal metabolic heterogeneity at single-cell resolution within tissues and its association with specific cell populations such as cancer cells or immune cells. This approach has the potential to greatly increase our understanding of tissue-level interplay between metabolic processes and their cellular components. The authors present a workflow integrating imaging mass cytometry and imaging mass spectrometry to deconvolute metabolic heterogeneity at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Nunes, Joana B. and Ijsselsteijn, Marieke E. and Abdelaal, Tamim and Ursem, Rick and van der Ploeg, Manon and Giera, Martin and Everts, Bart and Mahfouz, Ahmed and Heijs, Bram and de Miranda, Noel F. C. C.},
  doi          = {10.1038/s41592-024-02392-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1796-1800},
  shortjournal = {Nat. Meth.},
  title        = {Integration of mass cytometry and mass spectrometry imaging for spatially resolved single-cell metabolic profiling},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Analyzing submicron spatial transcriptomics data at their
original resolution. <em>NMETH</em>, <em>21</em>(10), 1794–1795. (<a
href="https://doi.org/10.1038/s41592-024-02416-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {FICTURE software addresses a critical challenge in spatial omics analysis: making high-resolution inference with only a few molecules per square micron. This tool fully realizes the potential of contemporary spatial platforms by learning latent spatial factors from the whole transcriptome while preserving the resolution of each technology at scale.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02416-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1794-1795},
  shortjournal = {Nat. Meth.},
  title        = {Analyzing submicron spatial transcriptomics data at their original resolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A structural learning method to uncover how information
between single cells flows. <em>NMETH</em>, <em>21</em>(10), 1792–1793.
(<a href="https://doi.org/10.1038/s41592-024-02381-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA-sequencing and spatial transcriptomics data enable the inference of how information is transmitted from one cell to another and how it modulates gene expression within cells. Now, a learning method infers networks describing how the inflow of one signal, mediated by intracellular gene modules, drives the outflow of other signals for intercellular communication.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02381-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1792-1793},
  shortjournal = {Nat. Meth.},
  title        = {A structural learning method to uncover how information between single cells flows},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). 20 years of nature methods: How some papers shaped science
and careers. <em>NMETH</em>, <em>21</em>(10), 1786–1791. (<a
href="https://doi.org/10.1038/s41592-024-02452-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several researchers share tales around the development of their widely used methods.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02452-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1786-1791},
  shortjournal = {Nat. Meth.},
  title        = {20 years of nature methods: How some papers shaped science and careers},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Immune intestine interfaces in vitro. <em>NMETH</em>,
<em>21</em>(10), 1785. (<a
href="https://doi.org/10.1038/s41592-024-02469-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02469-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1785},
  shortjournal = {Nat. Meth.},
  title        = {Immune intestine interfaces in vitro},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Alignment-free homology detection. <em>NMETH</em>,
<em>21</em>(10), 1785. (<a
href="https://doi.org/10.1038/s41592-024-02470-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-024-02470-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1785},
  shortjournal = {Nat. Meth.},
  title        = {Alignment-free homology detection},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024o). Synthetic condensates probe chromatin viscoelasticity.
<em>NMETH</em>, <em>21</em>(10), 1784. (<a
href="https://doi.org/10.1038/s41592-024-02467-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02467-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1784},
  shortjournal = {Nat. Meth.},
  title        = {Synthetic condensates probe chromatin viscoelasticity},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Disentangling parallel processes with BEYOND.
<em>NMETH</em>, <em>21</em>(10), 1784. (<a
href="https://doi.org/10.1038/s41592-024-02468-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Nakhle, Jean},
  doi          = {10.1038/s41592-024-02468-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1784},
  shortjournal = {Nat. Meth.},
  title        = {Disentangling parallel processes with BEYOND},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Adventures in AI. <em>NMETH</em>, <em>21</em>(10), 1783.
(<a href="https://doi.org/10.1038/s41592-024-02450-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-024-02450-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1783},
  shortjournal = {Nat. Meth.},
  title        = {Adventures in AI},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Ringside seats to a pandemic. <em>NMETH</em>,
<em>21</em>(10), 1783. (<a
href="https://doi.org/10.1038/s41592-024-02451-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02451-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1783},
  shortjournal = {Nat. Meth.},
  title        = {Ringside seats to a pandemic},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Holistic views of individual cells. <em>NMETH</em>,
<em>21</em>(10), 1782. (<a
href="https://doi.org/10.1038/s41592-024-02443-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02443-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1782},
  shortjournal = {Nat. Meth.},
  title        = {Holistic views of individual cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Celebrating milestones in structural biology.
<em>NMETH</em>, <em>21</em>(10), 1782–1783. (<a
href="https://doi.org/10.1038/s41592-024-02449-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02449-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1782-1783},
  shortjournal = {Nat. Meth.},
  title        = {Celebrating milestones in structural biology},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Making an impact with fluorescent probes. <em>NMETH</em>,
<em>21</em>(10), 1781–1782. (<a
href="https://doi.org/10.1038/s41592-024-02431-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02431-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1781-1782},
  shortjournal = {Nat. Meth.},
  title        = {Making an impact with fluorescent probes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A decade of neuroscience. <em>NMETH</em>, <em>21</em>(10),
1781. (<a href="https://doi.org/10.1038/s41592-024-02448-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02448-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1781},
  shortjournal = {Nat. Meth.},
  title        = {A decade of neuroscience},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping the brain: An editor’s journey. <em>NMETH</em>,
<em>21</em>(10), 1780. (<a
href="https://doi.org/10.1038/s41592-024-02446-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Pastrana, Erika},
  doi          = {10.1038/s41592-024-02446-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1780},
  shortjournal = {Nat. Meth.},
  title        = {Mapping the brain: An editor’s journey},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). One cell, two cell, dead cell, true cell. <em>NMETH</em>,
<em>21</em>(10), 1780–1781. (<a
href="https://doi.org/10.1038/s41592-024-02447-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Nawy, Tal},
  doi          = {10.1038/s41592-024-02447-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1780-1781},
  shortjournal = {Nat. Meth.},
  title        = {One cell, two cell, dead cell, true cell},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). From blobology to atomic resolution. <em>NMETH</em>,
<em>21</em>(10), 1779. (<a
href="https://doi.org/10.1038/s41592-024-02441-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Doerr, Allison},
  doi          = {10.1038/s41592-024-02441-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1779},
  shortjournal = {Nat. Meth.},
  title        = {From blobology to atomic resolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The dawn of an era. <em>NMETH</em>, <em>21</em>(10),
1779–1780. (<a
href="https://doi.org/10.1038/s41592-024-02445-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {de Souza, Natalie},
  doi          = {10.1038/s41592-024-02445-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1779-1780},
  shortjournal = {Nat. Meth.},
  title        = {The dawn of an era},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Connecting the points. <em>NMETH</em>, <em>21</em>(10),
1778. (<a href="https://doi.org/10.1038/s41592-024-02442-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Evanko, Daniel},
  doi          = {10.1038/s41592-024-02442-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1778},
  shortjournal = {Nat. Meth.},
  title        = {Connecting the points},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Listening to an RNA orchestra and seeing CRISPR in action.
<em>NMETH</em>, <em>21</em>(10), 1778–1779. (<a
href="https://doi.org/10.1038/s41592-024-02444-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For Nature Methods’ 20th anniversary, our current and past editors reminisce about their favorite papers, initiatives and projects at the journal.},
  archive      = {J_NMETH},
  author       = {Rusk, Nicole},
  doi          = {10.1038/s41592-024-02444-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1778-1779},
  shortjournal = {Nat. Meth.},
  title        = {Listening to an RNA orchestra and seeing CRISPR in action},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cell painting gallery: An open resource for image-based
profiling. <em>NMETH</em>, <em>21</em>(10), 1775–1777. (<a
href="https://doi.org/10.1038/s41592-024-02399-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Weisbart, Erin and Kumar, Ankur and Arevalo, John and Carpenter, Anne E. and Cimini, Beth A. and Singh, Shantanu},
  doi          = {10.1038/s41592-024-02399-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1775-1777},
  shortjournal = {Nat. Meth.},
  title        = {Cell painting gallery: An open resource for image-based profiling},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The SplitsTree app: Interactive analysis and visualization
using phylogenetic trees and networks. <em>NMETH</em>, <em>21</em>(10),
1773–1774. (<a
href="https://doi.org/10.1038/s41592-024-02406-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Huson, Daniel H. and Bryant, David},
  doi          = {10.1038/s41592-024-02406-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1773-1774},
  shortjournal = {Nat. Meth.},
  title        = {The SplitsTree app: Interactive analysis and visualization using phylogenetic trees and networks},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Danionella fishes. <em>NMETH</em>, <em>21</em>(10),
1767–1769. (<a
href="https://doi.org/10.1038/s41592-024-02433-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Miniature, transparent Danionella fishes, which are among the smallest living adult vertebrates, allow investigation of general principles of brain-wide neural circuits and evolutionary and developmental mechanisms for neurobehavioral innovations.},
  archive      = {J_NMETH},
  author       = {Bass, Andrew H. and Perelmuter, Jonathan T.},
  doi          = {10.1038/s41592-024-02433-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1767-1769},
  shortjournal = {Nat. Meth.},
  title        = {Danionella fishes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Twenty years of methods. <em>NMETH</em>, <em>21</em>(10),
1765. (<a href="https://doi.org/10.1038/s41592-024-02474-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As Nature Methods celebrates a milestone anniversary, we look back on the past two decades of methods development in basic biological research.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02474-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1765},
  shortjournal = {Nat. Meth.},
  title        = {Twenty years of methods},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Propensity score matching. <em>NMETH</em>, <em>21</em>(10),
1770–1772. (<a
href="https://doi.org/10.1038/s41592-024-02405-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {I don’t have good luck in the match points. —Rafael Nadal, Spanish tennis player.},
  archive      = {J_NMETH},
  author       = {Kurz, Christoph F. and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-024-02405-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {10},
  pages        = {1770-1772},
  shortjournal = {Nat. Meth.},
  title        = {Propensity score matching},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024v). Why they take risks. <em>NMETH</em>, <em>21</em>(10), 1766.
(<a href="https://doi.org/10.1038/s41592-024-02432-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Developing new methods takes passion and a penchant for risk-taking.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02432-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {10},
  pages        = {1766},
  shortjournal = {Nat. Meth.},
  title        = {Why they take risks},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The DNA-PAINT palette: A comprehensive performance analysis
of fluorescent dyes. <em>NMETH</em>, <em>21</em>(9), 1755–1762. (<a
href="https://doi.org/10.1038/s41592-024-02374-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA points accumulation for imaging in nanoscale topography (DNA-PAINT) is a super-resolution fluorescence microscopy technique that achieves single-molecule ‘blinking’ by transient DNA hybridization. Despite blinking kinetics being largely independent of fluorescent dye choice, the dye employed substantially affects measurement quality. Thus far, there has been no systematic overview of dye performance for DNA-PAINT. Here we defined four key parameters characterizing performance: brightness, signal-to-background ratio, DNA-PAINT docking site damage and off-target signal. We then analyzed 18 fluorescent dyes in three spectral regions and examined them both in DNA origami nanostructures, establishing a reference standard, and in a cellular environment, targeting the nuclear pore complex protein Nup96. Finally, having identified several well-performing dyes for each excitation wavelength, we conducted simultaneous three-color DNA-PAINT combined with Exchange-PAINT to image six protein targets in neurons at ~16 nm resolution in less than 2 h. We thus provide guidelines for DNA-PAINT dye selection and evaluation and an overview of performances of commonly used dyes. The dyes chosen for DNA-PAINT microscopy are pivotal for data quality. This Analysis shows a comprehensive comparison of 18 fluorescent dyes in DNA-PAINT and offers guidance for optimum dye selection in single-color and multiplexed imaging.},
  archive      = {J_NMETH},
  author       = {Steen, Philipp R. and Unterauer, Eduard M. and Masullo, Luciano A. and Kwon, Jisoo and Perovic, Ana and Jevdokimenko, Kristina and Opazo, Felipe and Fornasiero, Eugenio F. and Jungmann, Ralf},
  doi          = {10.1038/s41592-024-02374-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1755-1762},
  shortjournal = {Nat. Meth.},
  title        = {The DNA-PAINT palette: A comprehensive performance analysis of fluorescent dyes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic comparison of sequencing-based spatial
transcriptomic methods. <em>NMETH</em>, <em>21</em>(9), 1743–1754. (<a
href="https://doi.org/10.1038/s41592-024-02325-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent developments of sequencing-based spatial transcriptomics (sST) have catalyzed important advancements by facilitating transcriptome-scale spatial gene expression measurement. Despite this progress, efforts to comprehensively benchmark different platforms are currently lacking. The extant variability across technologies and datasets poses challenges in formulating standardized evaluation metrics. In this study, we established a collection of reference tissues and regions characterized by well-defined histological architectures, and used them to generate data to compare 11 sST methods. We highlighted molecular diffusion as a variable parameter across different methods and tissues, significantly affecting the effective resolutions. Furthermore, we observed that spatial transcriptomic data demonstrate unique attributes beyond merely adding a spatial axis to single-cell data, including an enhanced ability to capture patterned rare cell states along with specific markers, albeit being influenced by multiple factors including sequencing depth and resolution. Our study assists biologists in sST platform selection, and helps foster a consensus on evaluation standards and establish a framework for future benchmarking efforts that can be used as a gold standard for the development and benchmarking of computational tools for spatial transcriptomic analysis. This analysis presents a systematic comparison of 11 sequencing-based spatial transcriptomics methods using well-characterized references, which offers insights into performance variations in spatial transcriptomic techniques.},
  archive      = {J_NMETH},
  author       = {You, Yue and Fu, Yuting and Li, Lanxiang and Zhang, Zhongmin and Jia, Shikai and Lu, Shihong and Ren, Wenle and Liu, Yifang and Xu, Yang and Liu, Xiaojing and Jiang, Fuqing and Peng, Guangdun and Sampath Kumar, Abhishek and Ritchie, Matthew E. and Liu, Xiaodong and Tian, Luyi},
  doi          = {10.1038/s41592-024-02325-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1743-1754},
  shortjournal = {Nat. Meth.},
  title        = {Systematic comparison of sequencing-based spatial transcriptomic methods},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A cortical surface template for human neuroscience.
<em>NMETH</em>, <em>21</em>(9), 1736–1742. (<a
href="https://doi.org/10.1038/s41592-024-02346-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuroimaging data analysis relies on normalization to standard anatomical templates to resolve macroanatomical differences across brains. Existing human cortical surface templates sample locations unevenly because of distortions introduced by inflation of the folded cortex into a standard shape. Here we present the onavg template, which affords uniform sampling of the cortex. We created the onavg template based on openly available high-quality structural scans of 1,031 brains—25 times more than existing cortical templates. We optimized the vertex locations based on cortical anatomy, achieving an even distribution. We observed consistently higher multivariate pattern classification accuracies and representational geometry inter-participant correlations based on onavg than on other templates, and onavg only needs three-quarters as much data to achieve the same performance compared with other templates. The optimized sampling also reduces CPU time across algorithms by 1.3–22.4% due to less variation in the number of vertices in each searchlight. Onavg is a surface template of the human cortex. In contrast to existing templates, the cortical surface is uniformly sampled, which has advantages in numerous applications.},
  archive      = {J_NMETH},
  author       = {Feilong, Ma and Jiahui, Guo and Gobbini, Maria Ida and Haxby, James V.},
  doi          = {10.1038/s41592-024-02346-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1736-1742},
  shortjournal = {Nat. Meth.},
  title        = {A cortical surface template for human neuroscience},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Rapid, biochemical tagging of cellular activity history in
vivo. <em>NMETH</em>, <em>21</em>(9), 1725–1735. (<a
href="https://doi.org/10.1038/s41592-024-02375-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intracellular calcium (Ca2+) is ubiquitous to cell signaling across biology. While existing fluorescent sensors and reporters can detect activated cells with elevated Ca2+ levels, these approaches require implants to deliver light to deep tissue, precluding their noninvasive use in freely behaving animals. Here we engineered an enzyme-catalyzed approach that rapidly and biochemically tags cells with elevated Ca2+ in vivo. Ca2+-activated split-TurboID (CaST) labels activated cells within 10 min with an exogenously delivered biotin molecule. The enzymatic signal increases with Ca2+ concentration and biotin labeling time, demonstrating that CaST is a time-gated integrator of total Ca2+ activity. Furthermore, the CaST readout can be performed immediately after activity labeling, in contrast to transcriptional reporters that require hours to produce signal. These capabilities allowed us to apply CaST to tag prefrontal cortex neurons activated by psilocybin, and to correlate the CaST signal with psilocybin-induced head-twitch responses in untethered mice. CaST is a Ca2+-activated version of split-TurboID. The tool allows labeling active neurons quickly, simply by administration of exogenous biotin, thus enabling the study of behaviors that would be impaired by hardware required for the use of other, light-dependent tools.},
  archive      = {J_NMETH},
  author       = {Zhang, Run and Anguiano, Maribel and Aarrestad, Isak K. and Lin, Sophia and Chandra, Joshua and Vadde, Sruti S. and Olson, David E. and Kim, Christina K.},
  doi          = {10.1038/s41592-024-02375-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1725-1735},
  shortjournal = {Nat. Meth.},
  title        = {Rapid, biochemical tagging of cellular activity history in vivo},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BNP-track: A framework for superresolved tracking.
<em>NMETH</em>, <em>21</em>(9), 1716–1724. (<a
href="https://doi.org/10.1038/s41592-024-02349-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Superresolution tools, such as PALM and STORM, provide nanoscale localization accuracy by relying on rare photophysical events, limiting these methods to static samples. By contrast, here, we extend superresolution to dynamics without relying on photodynamics by simultaneously determining emitter numbers and their tracks (localization and linking) with the same localization accuracy per frame as widefield superresolution on immobilized emitters under similar imaging conditions (≈50 nm). We demonstrate our Bayesian nonparametric track (BNP-Track) framework on both in cellulo and synthetic data. BNP-Track develops a joint (posterior) distribution that learns and quantifies uncertainty over emitter numbers and their associated tracks propagated from shot noise, camera artifacts, pixelation, background and out-of-focus motion. In doing so, we integrate spatiotemporal information into our distribution, which is otherwise compromised by modularly determining emitter numbers and localizing and linking emitter positions across frames. For this reason, BNP-Track remains accurate in crowding regimens beyond those accessible to other single-particle tracking tools. Bayesian nonparametric Track (BNP-Track) simultaneously determines emitter numbers and their tracks alongside uncertainty, extending the superresolution paradigm from static samples to single-particle tracking even in dense environments.},
  archive      = {J_NMETH},
  author       = {Sgouralis, Ioannis and Xu, Lance W. Q. and Jalihal, Ameya P. and Kilic, Zeliha and Walter, Nils G. and Pressé, Steve},
  doi          = {10.1038/s41592-024-02349-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1716-1724},
  shortjournal = {Nat. Meth.},
  title        = {BNP-track: A framework for superresolved tracking},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ProDOL: A general method to determine the degree of labeling
for staining optimization and molecular counting. <em>NMETH</em>,
<em>21</em>(9), 1708–1715. (<a
href="https://doi.org/10.1038/s41592-024-02376-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining the label to target ratio, also known as the degree of labeling (DOL), is crucial for quantitative fluorescence microscopy and a high DOL with minimal unspecific labeling is beneficial for fluorescence microscopy in general. Yet robust, versatile and easy-to-use tools for measuring cell-specific labeling efficiencies are not available. Here we present a DOL determination technique named protein-tag DOL (ProDOL), which enables fast quantification and optimization of protein-tag labeling. With ProDOL various factors affecting labeling efficiency, including substrate type, incubation time and concentration, as well as sample fixation and cell type can be easily assessed. We applied ProDOL to investigate how human immunodeficiency virus-1 pathogenesis factor Nef modulates CD4 T cell activation measuring total and activated copy numbers of the adapter protein SLP-76 in signaling microclusters. ProDOL proved to be a versatile and robust tool for labeling calibration, enabling determination of labeling efficiencies, optimization of strategies and quantification of protein stoichiometry. Protein-tag degree of labeling (ProDOL) is a versatile reference-based approach for experimentally determining the degree of target labeling for improved protein counting and quantification and for optimizing labeling protocols in fixed and live cells.},
  archive      = {J_NMETH},
  author       = {Tashev, Stanimir Asenov and Euchner, Jonas and Yserentant, Klaus and Hänselmann, Siegfried and Hild, Felix and Chmielewicz, Wioleta and Hummert, Johan and Schwörer, Florian and Tsopoulidis, Nikolaos and Germer, Stefan and Saßmannshausen, Zoe and Fackler, Oliver T. and Klingmüller, Ursula and Herten, Dirk-Peter},
  doi          = {10.1038/s41592-024-02376-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1708-1715},
  shortjournal = {Nat. Meth.},
  title        = {ProDOL: A general method to determine the degree of labeling for staining optimization and molecular counting},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantification of absolute labeling efficiency at the
single-protein level. <em>NMETH</em>, <em>21</em>(9), 1702–1707. (<a
href="https://doi.org/10.1038/s41592-024-02242-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {State-of-the-art super-resolution microscopy allows researchers to spatially resolve single proteins in dense clusters. However, accurate quantification of protein organization and stoichiometries requires a general method to evaluate absolute binder labeling efficiency, which is currently unavailable. Here we introduce a universally applicable approach that uses a reference tag fused to a target protein of interest. By attaching high-affinity binders, such as antibodies or nanobodies, to both the reference tag and the target protein, and then employing DNA-barcoded sequential super-resolution imaging, we can correlate the location of the reference tag with the target molecule binder. This approach facilitates the precise quantification of labeling efficiency at the single-protein level. Super-resolution imaging of reference and target structures enables precise determination of the labeling efficiency of high-affinity binding proteins in cells for improved quantitative assessment of protein organization at the single-molecule level.},
  archive      = {J_NMETH},
  author       = {Hellmeier, Joschka and Strauss, Sebastian and Xu, Shuhan and Masullo, Luciano A. and Unterauer, Eduard M. and Kowalewski, Rafal and Jungmann, Ralf},
  doi          = {10.1038/s41592-024-02242-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1702-1707},
  shortjournal = {Nat. Meth.},
  title        = {Quantification of absolute labeling efficiency at the single-protein level},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Serialized on-grid lift-in sectioning for tomography
(SOLIST) enables a biopsy at the nanoscale. <em>NMETH</em>,
<em>21</em>(9), 1693–1701. (<a
href="https://doi.org/10.1038/s41592-024-02384-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-focused ion beam milling has substantially advanced our understanding of molecular processes by opening windows into cells. However, applying this technique to complex samples, such as tissues, has presented considerable technical challenges. Here we introduce an innovative adaptation of the cryo-lift-out technique, serialized on-grid lift-in sectioning for tomography (SOLIST), addressing these limitations. SOLIST enhances throughput, minimizes ice contamination and improves sample stability for cryo-electron tomography. It thereby facilitates the high-resolution imaging of a wide range of specimens. We illustrate these advantages on reconstituted liquid–liquid phase-separated droplets, brain organoids and native tissues from the mouse brain, liver and heart. With SOLIST, cellular processes can now be investigated at molecular resolution directly in native tissue. Furthermore, our method has a throughput high enough to render cryo-lift-out a competitive tool for structural biology. This opens new avenues for unprecedented insights into cellular function and structure in health and disease, a ‘biopsy at the nanoscale’. Serialized on-grid lift-in sectioning for tomography (SOLIST) improves the throughput of the serial lift-out technique for creating lamellas, addressing a major bottleneck in the use of cryo-electron tomography for in situ structural biology.},
  archive      = {J_NMETH},
  author       = {Nguyen, Ho Thuy Dung and Perone, Gaia and Klena, Nikolai and Vazzana, Roberta and Kaluthantrige Don, Flaminia and Silva, Malan and Sorrentino, Simona and Swuec, Paolo and Leroux, Frederic and Kalebic, Nereo and Coscia, Francesca and Erdmann, Philipp S.},
  doi          = {10.1038/s41592-024-02384-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1693-1701},
  shortjournal = {Nat. Meth.},
  title        = {Serialized on-grid lift-in sectioning for tomography (SOLIST) enables a biopsy at the nanoscale},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Serial lift-out: Sampling the molecular anatomy of whole
organisms. <em>NMETH</em>, <em>21</em>(9), 1684–1692. (<a
href="https://doi.org/10.1038/s41592-023-02113-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-focused ion beam milling of frozen-hydrated cells and subsequent cryo-electron tomography (cryo-ET) has enabled the structural elucidation of macromolecular complexes directly inside cells. Application of the technique to multicellular organisms and tissues, however, is still limited by sample preparation. While high-pressure freezing enables the vitrification of thicker samples, it prolongs subsequent preparation due to increased thinning times and the need for extraction procedures. Additionally, thinning removes large portions of the specimen, restricting the imageable volume to the thickness of the final lamella, typically &amp;lt;300 nm. Here we introduce Serial Lift-Out, an enhanced lift-out technique that increases throughput and obtainable contextual information by preparing multiple sections from single transfers. We apply Serial Lift-Out to Caenorhabditis elegans L1 larvae, yielding a cryo-ET dataset sampling the worm’s anterior–posterior axis, and resolve its ribosome structure to 7 Å and a subregion of the 11-protofilament microtubule to 13 Å, illustrating how Serial Lift-Out enables the study of multicellular molecular anatomy. Serial Lift-Out creates a series of lamellae from one lift-out volume for cryo-ET, increasing the ease and throughput of cryo-lift-out and enabling the study of molecular anatomy in multicellular systems including C. elegans larvae.},
  archive      = {J_NMETH},
  author       = {Schiøtz, Oda Helene and Kaiser, Christoph J. O. and Klumpe, Sven and Morado, Dustin R. and Poege, Matthias and Schneider, Jonathan and Beck, Florian and Klebl, David P. and Thompson, Christopher and Plitzko, Jürgen M.},
  doi          = {10.1038/s41592-023-02113-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1684-1692},
  shortjournal = {Nat. Meth.},
  title        = {Serial lift-out: Sampling the molecular anatomy of whole organisms},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Geometric deep learning of protein–DNA binding specificity.
<em>NMETH</em>, <em>21</em>(9), 1674–1683. (<a
href="https://doi.org/10.1038/s41592-024-02372-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting protein–DNA binding specificity is a challenging yet essential task for understanding gene regulation. Protein–DNA complexes usually exhibit binding to a selected DNA target site, whereas a protein binds, with varying degrees of binding specificity, to a wide range of DNA sequences. This information is not directly accessible in a single structure. Here, to access this information, we present Deep Predictor of Binding Specificity (DeepPBS), a geometric deep-learning model designed to predict binding specificity from protein–DNA structure. DeepPBS can be applied to experimental or predicted structures. Interpretable protein heavy atom importance scores for interface residues can be extracted. When aggregated at the protein residue level, these scores are validated through mutagenesis experiments. Applied to designed proteins targeting specific DNA sequences, DeepPBS was demonstrated to predict experimentally measured binding specificity. DeepPBS offers a foundation for machine-aided studies that advance our understanding of molecular interactions and guide experimental designs and synthetic biology. DeepPBS is a deep-learning model designed to predict the binding specificity of protein–DNA interactions using physicochemical and geometric contexts. DeepPBS functions across protein families and on experimentally determined as well as predicted protein–DNA complex structures.},
  archive      = {J_NMETH},
  author       = {Mitra, Raktim and Li, Jinsen and Sagendorf, Jared M. and Jiang, Yibei and Cohen, Ari S. and Chiu, Tsu-Pei and Glasscock, Cameron J. and Rohs, Remo},
  doi          = {10.1038/s41592-024-02372-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1674-1683},
  shortjournal = {Nat. Meth.},
  title        = {Geometric deep learning of protein–DNA binding specificity},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Proteome-scale tagging and functional screening in mammalian
cells by ORFtag. <em>NMETH</em>, <em>21</em>(9), 1668–1673. (<a
href="https://doi.org/10.1038/s41592-024-02339-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The systematic determination of protein function is a key goal of modern biology, but remains challenging with current approaches. Here we present ORFtag, a versatile, cost-effective and highly efficient method for the massively parallel tagging and functional interrogation of proteins at the proteome scale. ORFtag uses retroviral vectors bearing a promoter, peptide tag and splice donor to generate fusions between the tag and endogenous open reading frames (ORFs). We demonstrate the utility of ORFtag through functional screens for transcriptional activators, repressors and posttranscriptional regulators in mouse embryonic stem cells. Each screen recovers known and identifies new regulators, including long ORFs inaccessible by other methods. Among other hits, we find that Zfp574 is a highly selective transcriptional activator and that oncogenic fusions often function as transactivators. This work presents ORFtag that enables proteome-wide functional screens by tagging and overexpression of endogenously encoded proteins via randomly integrated cassettes.},
  archive      = {J_NMETH},
  author       = {Nemčko, Filip and Himmelsbach, Moritz and Loubiere, Vincent and Yelagandula, Ramesh and Pagani, Michaela and Fasching, Nina and Brennecke, Julius and Elling, Ulrich and Stark, Alexander and Ameres, Stefan L.},
  doi          = {10.1038/s41592-024-02339-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1668-1673},
  shortjournal = {Nat. Meth.},
  title        = {Proteome-scale tagging and functional screening in mammalian cells by ORFtag},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deciphering spatial domains from spatial multi-omics with
SpatialGlue. <em>NMETH</em>, <em>21</em>(9), 1658–1667. (<a
href="https://doi.org/10.1038/s41592-024-02316-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in spatial omics technologies now allow multiple types of data to be acquired from the same tissue slice. To realize the full potential of such data, we need spatially informed methods for data integration. Here, we introduce SpatialGlue, a graph neural network model with a dual-attention mechanism that deciphers spatial domains by intra-omics integration of spatial location and omics measurement followed by cross-omics integration. We demonstrated SpatialGlue on data acquired from different tissue types using different technologies, including spatial epigenome–transcriptome and transcriptome–proteome modalities. Compared to other methods, SpatialGlue captured more anatomical details and more accurately resolved spatial domains such as the cortex layers of the brain. Our method also identified cell types like spleen macrophage subsets located at three different zones that were not available in the original data annotations. SpatialGlue scales well with data size and can be used to integrate three modalities. Our spatial multi-omics analysis tool combines the information from complementary omics modalities to obtain a holistic view of cellular and tissue properties. SpatialGlue is a graph neural network-based approach for integrating multimodal spatial omics data. Combining complementary data modalities improves the discovery of spatial domains as well as the identification of cell subpopulations across tissues.},
  archive      = {J_NMETH},
  author       = {Long, Yahui and Ang, Kok Siong and Sethi, Raman and Liao, Sha and Heng, Yang and van Olst, Lynn and Ye, Shuchen and Zhong, Chengwei and Xu, Hang and Zhang, Di and Kwok, Immanuel and Husna, Nazihah and Jian, Min and Ng, Lai Guan and Chen, Ao and Gascoigne, Nicholas R. J. and Gate, David and Fan, Rong and Xu, Xun and Chen, Jinmiao},
  doi          = {10.1038/s41592-024-02316-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1658-1667},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering spatial domains from spatial multi-omics with SpatialGlue},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). CRISPR-array-mediated imaging of non-repetitive and
multiplex genomic loci in living cells. <em>NMETH</em>, <em>21</em>(9),
1646–1657. (<a
href="https://doi.org/10.1038/s41592-024-02333-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamic imaging of genomic loci is key for understanding gene regulation, but methods for imaging genomes, in particular non-repetitive DNAs, are limited. We developed CRISPRdelight, a DNA-labeling system based on endonuclease-deficient CRISPR–Cas12a (dCas12a), with an engineered CRISPR array to track DNA location and motion. CRISPRdelight enables robust imaging of all examined 12 non-repetitive genomic loci in different cell lines. We revealed the confined movement of the CCAT1 locus (chr8q24) at the nuclear periphery for repressed expression and active motion in the interior nucleus for transcription. We uncovered the selective repositioning of HSP gene loci to nuclear speckles, including a remarkable relocation of HSPH1 (chr13q12) for elevated transcription during stresses. Combining CRISPR–dCas12a and RNA aptamers allowed multiplex imaging of four types of satellite DNA loci with a single array, revealing their spatial proximity to the nucleolus-associated domain. CRISPRdelight is a user-friendly and robust system for imaging and tracking genomic dynamics and regulation. This work introduces CRISPRdelight, a dCas12a-based DNA-imaging tool that facilitates the imaging of non-repetitive loci and the tracking of DNA dynamics.},
  archive      = {J_NMETH},
  author       = {Yang, Liang-Zhong and Min, Yi-Hui and Liu, Yu-Xin and Gao, Bao-Qing and Liu, Xiao-Qi and Huang, Youkui and Wang, Haifeng and Yang, Li and Liu, Zhe J. and Chen, Ling-Ling},
  doi          = {10.1038/s41592-024-02333-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1646-1657},
  shortjournal = {Nat. Meth.},
  title        = {CRISPR-array-mediated imaging of non-repetitive and multiplex genomic loci in living cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A systematic search for RNA structural switches across the
human transcriptome. <em>NMETH</em>, <em>21</em>(9), 1634–1645. (<a
href="https://doi.org/10.1038/s41592-024-02335-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA structural switches are key regulators of gene expression in bacteria, but their characterization in Metazoa remains limited. Here, we present SwitchSeeker, a comprehensive computational and experimental approach for systematic identification of functional RNA structural switches. We applied SwitchSeeker to the human transcriptome and identified 245 putative RNA switches. To validate our approach, we characterized a previously unknown RNA switch in the 3ʹ untranslated region of the RORC (RAR-related orphan receptor C) transcript. In vivo dimethyl sulfate (DMS) mutational profiling with sequencing (DMS-MaPseq), coupled with cryogenic electron microscopy, confirmed its existence as two alternative structural conformations. Furthermore, we used genome-scale CRISPR screens to identify trans factors that regulate gene expression through this RNA structural switch. We found that nonsense-mediated messenger RNA decay acts on this element in a conformation-specific manner. SwitchSeeker provides an unbiased, experimentally driven method for discovering RNA structural switches that shape the eukaryotic gene expression landscape. SwitchSeeker combines computational and experimental techniques to identify functional RNA structural switches. Applied to the human transcriptome, it identified a novel RNA switch in the 3ʹUTR of RORC, linked to nonsense-mediated decay.},
  archive      = {J_NMETH},
  author       = {Khoroshkin, Matvei and Asarnow, Daniel and Zhou, Shaopu and Navickas, Albertas and Winters, Aidan and Goudreau, Jackson and Zhou, Simon K. and Yu, Johnny and Palka, Christina and Fish, Lisa and Borah, Ashir and Yousefi, Kian and Carpenter, Christopher and Ansel, K. Mark and Cheng, Yifan and Gilbert, Luke A. and Goodarzi, Hani},
  doi          = {10.1038/s41592-024-02335-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1634-1645},
  shortjournal = {Nat. Meth.},
  title        = {A systematic search for RNA structural switches across the human transcriptome},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell EpiChem jointly measures drug–chromatin binding
and multimodal epigenome. <em>NMETH</em>, <em>21</em>(9), 1624–1633. (<a
href="https://doi.org/10.1038/s41592-024-02360-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies of molecular and cellular functions of small-molecule inhibitors in cancer treatment, eliciting effects by targeting genome and epigenome associated proteins, requires measurement of drug-target engagement in single-cell resolution. Here we present EpiChem for in situ single-cell joint mapping of small molecules and multimodal epigenomic landscape. We demonstrate single-cell co-assays of three small molecules together with histone modifications, chromatin accessibility or target proteins in human colorectal cancer (CRC) organoids. Integrated multimodal analysis reveals diverse drug interactions in the context of chromatin states within heterogeneous CRC organoids. We further reveal drug genomic binding dynamics and adaptive epigenome across cell types after small-molecule drug treatment in CRC organoids. This method provides a unique tool to exploit the mechanisms of cell type-specific drug actions. EpiChem enables the co-profiling of small molecule–DNA interactions and multiple epigenomic features in single cells.},
  archive      = {J_NMETH},
  author       = {Dong, Chao and Meng, Xiaoxuan and Zhang, Tong and Guo, Zhifang and Liu, Yaxi and Wu, Peihuang and Chen, Shiwei and Zhou, Fanqi and Ma, Yanni and Xiong, Haiqing and Shu, Shaokun and He, Aibin},
  doi          = {10.1038/s41592-024-02360-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1624-1633},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell EpiChem jointly measures drug–chromatin binding and multimodal epigenome},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Analyzing single-cell bisulfite sequencing data with
MethSCAn. <em>NMETH</em>, <em>21</em>(9), 1616–1623. (<a
href="https://doi.org/10.1038/s41592-024-02347-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell bisulfite sequencing (scBS) is a technique that enables the assessment of DNA methylation at single-base pair and single-cell resolution. The analysis of large datasets obtained from scBS requires preprocessing to reduce the data size, improve the signal-to-noise ratio and provide interpretability. Typically, this is achieved by dividing the genome into large tiles and averaging the methylation signals within each tile. Here we demonstrate that this coarse-graining approach can lead to signal dilution. We propose improved strategies to identify more informative regions for methylation quantification and a more accurate quantitation method than simple averaging. Our approach enables better discrimination of cell types and other features of interest and reduces the need for large numbers of cells. We also present an approach to detect differentially methylated regions between groups of cells and demonstrate its ability to identify biologically meaningful regions that are associated with genes involved in the core functions of specific cell types. Finally, we present the software tool MethSCAn for scBS data analysis ( https://anders-biostat.github.io/MethSCAn ). This work highlights the technical issues in previous approaches and introduces a preprocessing approach along with a software package, MethSCAn, for single-cell bisulfite sequencing data analysis.},
  archive      = {J_NMETH},
  author       = {Kremer, Lukas P. M. and Braun, Martina M. and Ovchinnikova, Svetlana and Küchenhoff, Leonie and Cerrizuela, Santiago and Martin-Villalba, Ana and Anders, Simon},
  doi          = {10.1038/s41592-024-02347-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1616-1623},
  shortjournal = {Nat. Meth.},
  title        = {Analyzing single-cell bisulfite sequencing data with MethSCAn},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Smart parallel automated cryo-electron tomography.
<em>NMETH</em>, <em>21</em>(9), 1612–1615. (<a
href="https://doi.org/10.1038/s41592-024-02373-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In situ cryo-electron tomography enables investigation of macromolecules in their native cellular environment. Samples have become more readily available owing to recent software and hardware advancements. Data collection, however, still requires an experienced operator and appreciable microscope time to carefully select targets for high-throughput tilt series acquisition. Here, we developed smart parallel automated cryo-electron tomography (SPACEtomo), a workflow using machine learning approaches to fully automate the entire cryo-electron tomography process, including lamella detection, biological feature segmentation, target selection and parallel tilt series acquisition, all without the need for human intervention. This degree of automation will be essential for obtaining statistically relevant datasets and high-resolution structures of macromolecules in their native context. Smart parallel automated cryo-electron tomography (SPACEtomo) uses deep learning to fully automate data collection from lamella detection to tilt series acquisition, driving the future of cryo-ET through improved throughput and statistics.},
  archive      = {J_NMETH},
  author       = {Eisenstein, Fabian and Fukuda, Yoshiyuki and Danev, Radostin},
  doi          = {10.1038/s41592-024-02373-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1612-1615},
  shortjournal = {Nat. Meth.},
  title        = {Smart parallel automated cryo-electron tomography},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Microsecond time-resolved x-ray scattering by utilizing MHz
repetition rate at second-generation XFELs. <em>NMETH</em>,
<em>21</em>(9), 1608–1611. (<a
href="https://doi.org/10.1038/s41592-024-02344-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detecting microsecond structural perturbations in biomolecules has wide relevance in biology, chemistry and medicine. Here we show how MHz repetition rates at X-ray free-electron lasers can be used to produce microsecond time-series of protein scattering with exceptionally low noise levels of 0.001%. We demonstrate the approach by examining Jɑ helix unfolding of a light-oxygen-voltage photosensory domain. This time-resolved acquisition strategy is easy to implement and widely applicable for direct observation of structural dynamics of many biochemical processes. The MHz repetition rates available at second-generation X-ray free-electron lasers enable the collection of microsecond time-resolved X-ray scattering data with exceptionally low noise, providing insights into protein structural dynamics.},
  archive      = {J_NMETH},
  author       = {Konold, Patrick E. and Monrroy, Leonardo and Bellisario, Alfredo and Filipe, Diogo and Adams, Patrick and Alvarez, Roberto and Bean, Richard and Bielecki, Johan and Bódizs, Szabolcs and Ducrocq, Gabriel and Grubmueller, Helmut and Kirian, Richard A. and Kloos, Marco and Koliyadu, Jayanath C. P. and Koua, Faisal H. M. and Larkiala, Taru and Letrun, Romain and Lindsten, Fredrik and Maihöfer, Michael and Martin, Andrew V. and Mészáros, Petra and Mutisya, Jennifer and Nimmrich, Amke and Okamoto, Kenta and Round, Adam and Sato, Tokushi and Valerio, Joana and Westphal, Daniel and Wollter, August and Yenupuri, Tej Varma and You, Tong and Maia, Filipe and Westenhoff, Sebastian},
  doi          = {10.1038/s41592-024-02344-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1608-1611},
  shortjournal = {Nat. Meth.},
  title        = {Microsecond time-resolved X-ray scattering by utilizing MHz repetition rate at second-generation XFELs},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantms: A cloud-based pipeline for quantitative proteomics
enables the reanalysis of public proteomics data. <em>NMETH</em>,
<em>21</em>(9), 1603–1607. (<a
href="https://doi.org/10.1038/s41592-024-02343-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The volume of public proteomics data is rapidly increasing, causing a computational challenge for large-scale reanalysis. Here, we introduce quantms ( https://quant,ms.org/ ), an open-source cloud-based pipeline for massively parallel proteomics data analysis. We used quantms to reanalyze 83 public ProteomeXchange datasets, comprising 29,354 instrument files from 13,132 human samples, to quantify 16,599 proteins based on 1.03 million unique peptides. quantms is based on standard file formats improving the reproducibility, submission and dissemination of the data to ProteomeXchange. Scalable tools are needed for the analysis of increasingly large mass spectrometry-based proteomics datasets. quantms offers an open-source, cloud-based pipeline for massively parallel proteomics data analysis.},
  archive      = {J_NMETH},
  author       = {Dai, Chengxin and Pfeuffer, Julianus and Wang, Hong and Zheng, Ping and Käll, Lukas and Sachsenberg, Timo and Demichev, Vadim and Bai, Mingze and Kohlbacher, Oliver and Perez-Riverol, Yasset},
  doi          = {10.1038/s41592-024-02343-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1603-1607},
  shortjournal = {Nat. Meth.},
  title        = {Quantms: A cloud-based pipeline for quantitative proteomics enables the reanalysis of public proteomics data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Small data methods in omics: The power of one.
<em>NMETH</em>, <em>21</em>(9), 1597–1602. (<a
href="https://doi.org/10.1038/s41592-024-02390-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the last decade, biology has begun utilizing ‘big data’ approaches, resulting in large, comprehensive atlases in modalities ranging from transcriptomics to neural connectomics. However, these approaches must be complemented and integrated with ‘small data’ approaches to efficiently utilize data from individual labs. Integration of smaller datasets with major reference atlases is critical to provide context to individual experiments, and approaches toward integration of large and small data have been a major focus in many fields in recent years. Here we discuss progress in integration of small data with consortium-sized atlases across multiple modalities, and its potential applications. We then examine promising future directions for utilizing the power of small data to maximize the information garnered from small-scale experiments. We envision that, in the near future, international consortia comprising many laboratories will work together to collaboratively build reference atlases and foundation models using small data methods. This Perspective discusses the integration of small-scale datasets with each other or with larger reference atlases, particularly in the context of single-cell approaches.},
  archive      = {J_NMETH},
  author       = {Johnston, Kevin G. and Grieco, Steven F. and Nie, Qing and Theis, Fabian J. and Xu, Xiangmin},
  doi          = {10.1038/s41592-024-02390-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1597-1602},
  shortjournal = {Nat. Meth.},
  title        = {Small data methods in omics: The power of one},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MethSCAn: Accurate exploratory analysis of single-cell
methylomes. <em>NMETH</em>, <em>21</em>(9), 1595–1596. (<a
href="https://doi.org/10.1038/s41592-024-02348-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell bisulfite sequencing enables the genome-wide quantification of DNA methylation at single-cell resolution, but methods to analyze the resulting data are lacking. The MethSCAn software accurately distinguishes cell types and states by scanning the genome for informative regions and providing a robust approach to quantifying methylation within these regions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02348-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1595-1596},
  shortjournal = {Nat. Meth.},
  title        = {MethSCAn: Accurate exploratory analysis of single-cell methylomes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A machine learning tool for spatial multi-omics.
<em>NMETH</em>, <em>21</em>(9), 1593–1594. (<a
href="https://doi.org/10.1038/s41592-024-02358-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SpatialGlue is a tool designed to decipher spatial domains from spatial multi-omics data acquired from a single tissue section. It employes graph neural networks with a dual-attention mechanism to accomplish within-omics integration of measured features and spatial information, followed by cross-omics integration.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02358-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1593-1594},
  shortjournal = {Nat. Meth.},
  title        = {A machine learning tool for spatial multi-omics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). CRISPRdelight illuminates non-repetitive genomic loci in
living cells. <em>NMETH</em>, <em>21</em>(9), 1591–1592. (<a
href="https://doi.org/10.1038/s41592-024-02352-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We demonstrate CRISPRdelight, a robust CRISPR–Cas12a-based method for imaging non-repetitive genomic DNAs in a highly efficient way. This system is a powerful tool for studying functional links between gene dynamics, localization and regulation, and reveals heterogeneity in the expression of differently localized alleles in the same cells.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02352-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1591-1592},
  shortjournal = {Nat. Meth.},
  title        = {CRISPRdelight illuminates non-repetitive genomic loci in living cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Advancing sequencing-based spatial transcriptomics with a
comprehensive benchmarking study. <em>NMETH</em>, <em>21</em>(9),
1589–1590. (<a
href="https://doi.org/10.1038/s41592-024-02326-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A systematic comparison of 11 sequencing-based spatial transcriptomics methods reveals molecular diffusion as a critical variable that influences the effective resolution and data interpretation across platforms. Our benchmarking study should aid biologists in selecting the most appropriate method for their specific tissue.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02326-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1589-1590},
  shortjournal = {Nat. Meth.},
  title        = {Advancing sequencing-based spatial transcriptomics with a comprehensive benchmarking study},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Serial lift-out for in situ structural biology of
multicellular specimens. <em>NMETH</em>, <em>21</em>(9), 1587–1588. (<a
href="https://doi.org/10.1038/s41592-024-02317-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The capability of high-resolution in situ imaging by electron cryo-tomography (cryo-ET) has now been expanded to large multicellular tissues by newly developed workflows involving lift-out and serial sectioning using focused ion beam milling under cryogenic conditions.},
  archive      = {J_NMETH},
  author       = {Wang, Zhexin and Bharat, Tanmay A. M.},
  doi          = {10.1038/s41592-024-02317-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1587-1588},
  shortjournal = {Nat. Meth.},
  title        = {Serial lift-out for in situ structural biology of multicellular specimens},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bioimaging for all. <em>NMETH</em>, <em>21</em>(9),
1581–1586. (<a
href="https://doi.org/10.1038/s41592-024-02397-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The global imaging community is pursuing innovative approaches to achieve more equitable access to instruments and expertise.},
  archive      = {J_NMETH},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41592-024-02397-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1581-1586},
  shortjournal = {Nat. Meth.},
  title        = {Bioimaging for all},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Non-invasive metabolic imaging of brown adipose tissue.
<em>NMETH</em>, <em>21</em>(9), 1580. (<a
href="https://doi.org/10.1038/s41592-024-02422-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Nakhle, Jean},
  doi          = {10.1038/s41592-024-02422-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nat. Meth.},
  title        = {Non-invasive metabolic imaging of brown adipose tissue},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Biocomputation using tristate buffers. <em>NMETH</em>,
<em>21</em>(9), 1580. (<a
href="https://doi.org/10.1038/s41592-024-02423-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-024-02423-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nat. Meth.},
  title        = {Biocomputation using tristate buffers},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024p). Wobble base improves precision in RNA editing.
<em>NMETH</em>, <em>21</em>(9), 1579. (<a
href="https://doi.org/10.1038/s41592-024-02420-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02420-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1579},
  shortjournal = {Nat. Meth.},
  title        = {Wobble base improves precision in RNA editing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). LLMs predict protein phases. <em>NMETH</em>,
<em>21</em>(9), 1579. (<a
href="https://doi.org/10.1038/s41592-024-02421-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02421-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1579},
  shortjournal = {Nat. Meth.},
  title        = {LLMs predict protein phases},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Studying tension with DNA origami. <em>NMETH</em>,
<em>21</em>(9), 1578. (<a
href="https://doi.org/10.1038/s41592-024-02419-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA origami tension sensors can provide insights into mechanotransduction in a physiological environment.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02419-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1578},
  shortjournal = {Nat. Meth.},
  title        = {Studying tension with DNA origami},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unleashing the power of biomaterials to enhance organoid
differentiation and function. <em>NMETH</em>, <em>21</em>(9), 1575–1577.
(<a href="https://doi.org/10.1038/s41592-024-02393-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biomaterials are revolutionizing organoid development by offering tunable platforms that provide instructive cues, which enhance cell fate transitions, tissue-level functions and reproducibility. These advances are crucial for harnessing the translational potential of organoids.},
  archive      = {J_NMETH},
  author       = {Musah, Samira and Arzaghi, Hamidreza},
  doi          = {10.1038/s41592-024-02393-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1575-1577},
  shortjournal = {Nat. Meth.},
  title        = {Unleashing the power of biomaterials to enhance organoid differentiation and function},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Collaborative ethics: Innovating collaboration between
ethicists and life scientists. <em>NMETH</em>, <em>21</em>(9),
1571–1574. (<a
href="https://doi.org/10.1038/s41592-024-02320-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientific breakthroughs can change how we understand and live in the world, disrupting long-held assumptions and concepts and raising new questions for philosophy and science. To address these challenges, we describe a model for collaboration of scientists with philosophers and ethicists, and its benefits to the research process and outcomes.},
  archive      = {J_NMETH},
  author       = {Lunshof, Jeantine E. and Rijssenbeek, Julia},
  doi          = {10.1038/s41592-024-02320-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1571-1574},
  shortjournal = {Nat. Meth.},
  title        = {Collaborative ethics: Innovating collaboration between ethicists and life scientists},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Research ethics matter. <em>NMETH</em>, <em>21</em>(9),
1569. (<a href="https://doi.org/10.1038/s41592-024-02425-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {All life sciences research is potentially subject to ethical considerations. Institutions should support collaborations with professional ethicists and philosophers to help life scientists navigate ethical crossroads.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02425-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1569},
  shortjournal = {Nat. Meth.},
  title        = {Research ethics matter},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024t). Stem cell science starters. <em>NMETH</em>, <em>21</em>(9),
1570. (<a href="https://doi.org/10.1038/s41592-024-02398-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Early-career scientists shared some of their plans, hopes and dreams about being a principal investigator at the 2024 annual meeting of the International Society for Stem Cell Research.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02398-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {9},
  pages        = {1570},
  shortjournal = {Nat. Meth.},
  title        = {Stem cell science starters},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pretraining a foundation model for generalizable
fluorescence microscopy-based image restoration. <em>NMETH</em>,
<em>21</em>(8), 1558–1567. (<a
href="https://doi.org/10.1038/s41592-024-02244-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence microscopy-based image restoration has received widespread attention in the life sciences and has led to significant progress, benefiting from deep learning technology. However, most current task-specific methods have limited generalizability to different fluorescence microscopy-based image restoration problems. Here, we seek to improve generalizability and explore the potential of applying a pretrained foundation model to fluorescence microscopy-based image restoration. We provide a universal fluorescence microscopy-based image restoration (UniFMIR) model to address different restoration problems, and show that UniFMIR offers higher image restoration precision, better generalization and increased versatility. Demonstrations on five tasks and 14 datasets covering a wide range of microscopy imaging modalities and biological samples demonstrate that the pretrained UniFMIR can effectively transfer knowledge to a specific situation via fine-tuning, uncover clear nanoscale biomolecular structures and facilitate high-quality imaging. This work has the potential to inspire and trigger new research highlights for fluorescence microscopy-based image restoration. A pretrained foundation model (UniFMIR) enables versatile and generalizable performance across diverse fluorescence microscopy image reconstruction tasks.},
  archive      = {J_NMETH},
  author       = {Ma, Chenxi and Tan, Weimin and He, Ruian and Yan, Bo},
  doi          = {10.1038/s41592-024-02244-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1558-1567},
  shortjournal = {Nat. Meth.},
  title        = {Pretraining a foundation model for generalizable fluorescence microscopy-based image restoration},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Contextual AI models for single-cell protein biology.
<em>NMETH</em>, <em>21</em>(8), 1546–1557. (<a
href="https://doi.org/10.1038/s41592-024-02341-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding protein function and developing molecular therapies require deciphering the cell types in which proteins act as well as the interactions between proteins. However, modeling protein interactions across biological contexts remains challenging for existing algorithms. Here we introduce PINNACLE, a geometric deep learning approach that generates context-aware protein representations. Leveraging a multiorgan single-cell atlas, PINNACLE learns on contextualized protein interaction networks to produce 394,760 protein representations from 156 cell type contexts across 24 tissues. PINNACLE’s embedding space reflects cellular and tissue organization, enabling zero-shot retrieval of the tissue hierarchy. Pretrained protein representations can be adapted for downstream tasks: enhancing 3D structure-based representations for resolving immuno-oncological protein interactions, and investigating drugs’ effects across cell types. PINNACLE outperforms state-of-the-art models in nominating therapeutic targets for rheumatoid arthritis and inflammatory bowel diseases and pinpoints cell type contexts with higher predictive capability than context-free models. PINNACLE’s ability to adjust its outputs on the basis of the context in which it operates paves the way for large-scale context-specific predictions in biology. PINNACLE is a context-specific geometric deep learning model for generating protein representations. Leveraging single-cell transcriptomics combined with networks of protein–protein interactions, cell type-to-cell type interactions and a tissue hierarchy, PINNACLE generates high-resolution protein representations tailored to each cell type.},
  archive      = {J_NMETH},
  author       = {Li, Michelle M. and Huang, Yepeng and Sumathipala, Marissa and Liang, Man Qing and Valdeolivas, Alberto and Ananthakrishnan, Ashwin N. and Liao, Katherine and Marbach, Daniel and Zitnik, Marinka},
  doi          = {10.1038/s41592-024-02341-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1546-1557},
  shortjournal = {Nat. Meth.},
  title        = {Contextual AI models for single-cell protein biology},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). CryoDRGN-ET: Deep reconstructing generative networks for
visualizing dynamic biomolecules inside cells. <em>NMETH</em>,
<em>21</em>(8), 1537–1545. (<a
href="https://doi.org/10.1038/s41592-024-02340-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecules in native cellular environments. While cryo-ET can reveal structures at molecular resolution, image processing algorithms remain a bottleneck in resolving the heterogeneity of biomolecular structures in situ. Here, we introduce cryoDRGN-ET for heterogeneous reconstruction of cryo-ET subtomograms. CryoDRGN-ET learns a deep generative model of three-dimensional density maps directly from subtomogram tilt-series images and can capture states diverse in both composition and conformation. We validate this approach by recovering the known translational states in Mycoplasma pneumoniae ribosomes in situ. We then perform cryo-ET on cryogenic focused ion beam–milled Saccharomyces cerevisiae cells. CryoDRGN-ET reveals the structural landscape of S. cerevisiae ribosomes during translation and captures continuous motions of fatty acid synthase complexes inside cells. This method is openly available in the cryoDRGN software. cryoDRGN-ET is a generative neural network method for heterogeneous reconstruction of cryo-ET subtomograms. Using subtomogram tilt-series images, it can capture states diverse in both composition and conformation.},
  archive      = {J_NMETH},
  author       = {Rangan, Ramya and Feathers, Ryan and Khavnekar, Sagar and Lerer, Adam and Johnston, Jake D. and Kelley, Ron and Obr, Martin and Kotecha, Abhay and Zhong, Ellen D.},
  doi          = {10.1038/s41592-024-02340-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1537-1545},
  shortjournal = {Nat. Meth.},
  title        = {CryoDRGN-ET: Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Learning structural heterogeneity from cryo-electron
sub-tomograms with tomoDRGN. <em>NMETH</em>, <em>21</em>(8), 1525–1536.
(<a href="https://doi.org/10.1038/s41592-024-02210-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) enables observation of macromolecular complexes in their native, spatially contextualized cellular environment. Cryo-ET processing software to visualize such complexes at nanometer resolution via iterative alignment and averaging are well developed but rely upon assumptions of structural homogeneity among the complexes of interest. Recently developed tools allow for some assessment of structural diversity but have limited capacity to represent highly heterogeneous structures, including those undergoing continuous conformational changes. Here we extend the highly expressive cryoDRGN (Deep Reconstructing Generative Networks) deep learning architecture, originally created for single-particle cryo-electron microscopy analysis, to cryo-ET. Our new tool, tomoDRGN, learns a continuous low-dimensional representation of structural heterogeneity in cryo-ET datasets while also learning to reconstruct heterogeneous structural ensembles supported by the underlying data. Using simulated and experimental data, we describe and benchmark architectural choices within tomoDRGN that are uniquely necessitated and enabled by cryo-ET. We additionally illustrate tomoDRGN’s efficacy in analyzing diverse datasets, using it to reveal high-level organization of human immunodeficiency virus (HIV) capsid complexes assembled in virus-like particles and to resolve extensive structural heterogeneity among ribosomes imaged in situ. TomoDRGN is a deep learning framework designed to model conformational and compositional heterogeneity from cryo-ET datasets on a per-particle basis.},
  archive      = {J_NMETH},
  author       = {Powell, Barrett M. and Davis, Joseph H.},
  doi          = {10.1038/s41592-024-02210-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1525-1536},
  shortjournal = {Nat. Meth.},
  title        = {Learning structural heterogeneity from cryo-electron sub-tomograms with tomoDRGN},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). OpenFold: Retraining AlphaFold2 yields new insights into its
learning mechanisms and capacity for generalization. <em>NMETH</em>,
<em>21</em>(8), 1514–1524. (<a
href="https://doi.org/10.1038/s41592-024-02272-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AlphaFold2 revolutionized structural biology with the ability to predict protein structures with exceptionally high accuracy. Its implementation, however, lacks the code and data required to train new models. These are necessary to (1) tackle new tasks, like protein–ligand complex structure prediction, (2) investigate the process by which the model learns and (3) assess the model’s capacity to generalize to unseen regions of fold space. Here we report OpenFold, a fast, memory efficient and trainable implementation of AlphaFold2. We train OpenFold from scratch, matching the accuracy of AlphaFold2. Having established parity, we find that OpenFold is remarkably robust at generalizing even when the size and diversity of its training set is deliberately limited, including near-complete elisions of classes of secondary structure elements. By analyzing intermediate structures produced during training, we also gain insights into the hierarchical manner in which OpenFold learns to fold. In sum, our studies demonstrate the power and utility of OpenFold, which we believe will prove to be a crucial resource for the protein modeling community. OpenFold is a trainable open-source implementation of AlphaFold2. It is fast and memory efficient, and the code and training data are available under a permissive license.},
  archive      = {J_NMETH},
  author       = {Ahdritz, Gustaf and Bouatta, Nazim and Floristean, Christina and Kadyan, Sachin and Xia, Qinghui and Gerecke, William and O’Donnell, Timothy J. and Berenberg, Daniel and Fisk, Ian and Zanichelli, Niccolò and Zhang, Bo and Nowaczynski, Arkadiusz and Wang, Bei and Stepniewska-Dziubinska, Marta M. and Zhang, Shang and Ojewole, Adegoke and Guney, Murat Efe and Biderman, Stella and Watkins, Andrew M. and Ra, Stephen and Lorenzo, Pablo Ribalta and Nivon, Lucas and Weitzner, Brian and Ban, Yih-En Andrew and Chen, Shiyang and Zhang, Minjia and Li, Conglong and Song, Shuaiwen Leon and He, Yuxiong and Sorger, Peter K. and Mostaque, Emad and Zhang, Zhao and Bonneau, Richard and AlQuraishi, Mohammed},
  doi          = {10.1038/s41592-024-02272-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1514-1524},
  shortjournal = {Nat. Meth.},
  title        = {OpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Dependency-aware deep generative models for multitasking
analysis of spatial omics data. <em>NMETH</em>, <em>21</em>(8),
1501–1513. (<a
href="https://doi.org/10.1038/s41592-024-02257-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved transcriptomics (SRT) technologies have significantly advanced biomedical research, but their data analysis remains challenging due to the discrete nature of the data and the high levels of noise, compounded by complex spatial dependencies. Here, we propose spaVAE, a dependency-aware, deep generative spatial variational autoencoder model that probabilistically characterizes count data while capturing spatial correlations. spaVAE introduces a hybrid embedding combining a Gaussian process prior with a Gaussian prior to explicitly capture spatial correlations among spots. It then optimizes the parameters of deep neural networks to approximate the distributions underlying the SRT data. With the approximated distributions, spaVAE can contribute to several analytical tasks that are essential for SRT data analysis, including dimensionality reduction, visualization, clustering, batch integration, denoising, differential expression, spatial interpolation, resolution enhancement and identification of spatially variable genes. Moreover, we have extended spaVAE to spaPeakVAE and spaMultiVAE to characterize spatial ATAC-seq (assay for transposase-accessible chromatin using sequencing) data and spatial multi-omics data, respectively. Using a dependency-aware deep generative framework, spaVAE efficiently models spatially resolved transcriptomics data and advances diverse analysis tasks. Following similar strategies, spaPeakVAE and spaMultiVAE enable spatial ATAC-seq data and spatial multi-omics data modeling and analysis, respectively.},
  archive      = {J_NMETH},
  author       = {Tian, Tian and Zhang, Jie and Lin, Xiang and Wei, Zhi and Hakonarson, Hakon},
  doi          = {10.1038/s41592-024-02257-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1501-1513},
  shortjournal = {Nat. Meth.},
  title        = {Dependency-aware deep generative models for multitasking analysis of spatial omics data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Toward universal cell embeddings: Integrating single-cell
RNA-seq datasets across species with SATURN. <em>NMETH</em>,
<em>21</em>(8), 1492–1500. (<a
href="https://doi.org/10.1038/s41592-024-02191-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of single-cell datasets generated from diverse organisms offers unprecedented opportunities to unravel fundamental evolutionary processes of conservation and diversification of cell types. However, interspecies genomic differences limit the joint analysis of cross-species datasets to homologous genes. Here we present SATURN, a deep learning method for learning universal cell embeddings that encodes genes’ biological properties using protein language models. By coupling protein embeddings from language models with RNA expression, SATURN integrates datasets profiled from different species regardless of their genomic similarity. SATURN can detect functionally related genes coexpressed across species, redefining differential expression for cross-species analysis. Applying SATURN to three species whole-organism atlases and frog and zebrafish embryogenesis datasets, we show that SATURN can effectively transfer annotations across species, even when they are evolutionarily remote. We also demonstrate that SATURN can be used to find potentially divergent gene functions between glaucoma-associated genes in humans and four other species. SATURN performs cross-species integration and analysis using both single-cell gene expression and protein representations generated by protein language models.},
  archive      = {J_NMETH},
  author       = {Rosen, Yanay and Brbić, Maria and Roohani, Yusuf and Swanson, Kyle and Li, Ziang and Leskovec, Jure},
  doi          = {10.1038/s41592-024-02191-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1492-1500},
  shortjournal = {Nat. Meth.},
  title        = {Toward universal cell embeddings: Integrating single-cell RNA-seq datasets across species with SATURN},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Large-scale foundation model on single-cell transcriptomics.
<em>NMETH</em>, <em>21</em>(8), 1481–1491. (<a
href="https://doi.org/10.1038/s41592-024-02305-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large pretrained models have become foundation models leading to breakthroughs in natural language processing and related fields. Developing foundation models for deciphering the ‘languages’ of cells and facilitating biomedical research is promising yet challenging. Here we developed a large pretrained model scFoundation, also named ‘xTrimoscFoundationα’, with 100 million parameters covering about 20,000 genes, pretrained on over 50 million human single-cell transcriptomic profiles. scFoundation is a large-scale model in terms of the size of trainable parameters, dimensionality of genes and volume of training data. Its asymmetric transformer-like architecture and pretraining task design empower effectively capturing complex context relations among genes in a variety of cell types and states. Experiments showed its merit as a foundation model that achieved state-of-the-art performances in a diverse array of single-cell analysis tasks such as gene expression enhancement, tissue drug response prediction, single-cell drug response classification, single-cell perturbation prediction, cell type annotation and gene module inference. scFoundation, with 100 million parameters covering about 20,000 genes, pretrained on over 50 million single-cell transcriptomics profiles, is a foundation model for diverse tasks of single-cell analysis.},
  archive      = {J_NMETH},
  author       = {Hao, Minsheng and Gong, Jing and Zeng, Xin and Liu, Chiming and Guo, Yucheng and Cheng, Xingyi and Wang, Taifeng and Ma, Jianzhu and Zhang, Xuegong and Song, Le},
  doi          = {10.1038/s41592-024-02305-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1481-1491},
  shortjournal = {Nat. Meth.},
  title        = {Large-scale foundation model on single-cell transcriptomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ScGPT: Toward building a foundation model for single-cell
multi-omics using generative AI. <em>NMETH</em>, <em>21</em>(8),
1470–1480. (<a
href="https://doi.org/10.1038/s41592-024-02201-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative pretrained models have achieved remarkable success in various domains such as language and computer vision. Specifically, the combination of large-scale diverse datasets and pretrained transformers has emerged as a promising approach for developing foundation models. Drawing parallels between language and cellular biology (in which texts comprise words; similarly, cells are defined by genes), our study probes the applicability of foundation models to advance cellular biology and genetic research. Using burgeoning single-cell sequencing data, we have constructed a foundation model for single-cell biology, scGPT, based on a generative pretrained transformer across a repository of over 33 million cells. Our findings illustrate that scGPT effectively distills critical biological insights concerning genes and cells. Through further adaptation of transfer learning, scGPT can be optimized to achieve superior performance across diverse downstream applications. This includes tasks such as cell type annotation, multi-batch integration, multi-omic integration, perturbation response prediction and gene network inference. Pretrained using over 33 million single-cell RNA-sequencing profiles, scGPT is a foundation model facilitating a broad spectrum of downstream single-cell analysis tasks by transfer learning.},
  archive      = {J_NMETH},
  author       = {Cui, Haotian and Wang, Chloe and Maan, Hassaan and Pang, Kuan and Luo, Fengning and Duan, Nan and Wang, Bo},
  doi          = {10.1038/s41592-024-02201-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1470-1480},
  shortjournal = {Nat. Meth.},
  title        = {ScGPT: Toward building a foundation model for single-cell multi-omics using generative AI},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Biophysical modeling with variational autoencoders for
bimodal, single-cell RNA sequencing data. <em>NMETH</em>,
<em>21</em>(8), 1466–1469. (<a
href="https://doi.org/10.1038/s41592-024-02365-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present biVI, which combines the variational autoencoder framework of scVI with biophysical models describing the transcription and splicing kinetics of RNA molecules. We demonstrate on simulated and experimental single-cell RNA sequencing data that biVI retains the variational autoencoder’s ability to capture cell type structure in a low-dimensional space while further enabling genome-wide exploration of the biophysical mechanisms, such as system burst sizes and degradation rates, that underlie observations. biVI models the biophysical processes generating nascent and mature single-cell transcriptomes using variational autoencoders.},
  archive      = {J_NMETH},
  author       = {Carilli, Maria and Gorin, Gennady and Choi, Yongin and Chari, Tara and Pachter, Lior},
  doi          = {10.1038/s41592-024-02365-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1466-1469},
  shortjournal = {Nat. Meth.},
  title        = {Biophysical modeling with variational autoencoders for bimodal, single-cell RNA sequencing data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Assessing GPT-4 for cell type annotation in single-cell
RNA-seq analysis. <em>NMETH</em>, <em>21</em>(8), 1462–1465. (<a
href="https://doi.org/10.1038/s41592-024-02235-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we demonstrate that the large language model GPT-4 can accurately annotate cell types using marker gene information in single-cell RNA sequencing analysis. When evaluated across hundreds of tissue and cell types, GPT-4 generates cell type annotations exhibiting strong concordance with manual annotations. This capability can considerably reduce the effort and expertise required for cell type annotation. Additionally, we have developed an R software package GPTCelltype for GPT-4’s automated cell type annotation. This study evaluates the performance of GPT-4 in single-cell type annotation.},
  archive      = {J_NMETH},
  author       = {Hou, Wenpin and Ji, Zhicheng},
  doi          = {10.1038/s41592-024-02235-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1462-1465},
  shortjournal = {Nat. Meth.},
  title        = {Assessing GPT-4 for cell type annotation in single-cell RNA-seq analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Applying interpretable machine learning in computational
biology—pitfalls, recommendations and opportunities for new
developments. <em>NMETH</em>, <em>21</em>(8), 1454–1461. (<a
href="https://doi.org/10.1038/s41592-024-02359-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in machine learning have enabled the development of next-generation predictive models for complex computational biology problems, thereby spurring the use of interpretable machine learning (IML) to unveil biological insights. However, guidelines for using IML in computational biology are generally underdeveloped. We provide an overview of IML methods and evaluation techniques and discuss common pitfalls encountered when applying IML methods to computational biology problems. We also highlight open questions, especially in the era of large language models, and call for collaboration between IML and computational biology researchers. This Perspective discusses the methodologies, application and evaluation of interpretable machine learning (IML) approaches in computational biology, with particular focus on common pitfalls when using IML and how to avoid them.},
  archive      = {J_NMETH},
  author       = {Chen, Valerie and Yang, Muyu and Cui, Wenbo and Kim, Joon Sik and Talwalkar, Ameet and Ma, Jian},
  doi          = {10.1038/s41592-024-02359-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1454-1461},
  shortjournal = {Nat. Meth.},
  title        = {Applying interpretable machine learning in computational biology—pitfalls, recommendations and opportunities for new developments},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Guiding questions to avoid data leakage in biological
machine learning applications. <em>NMETH</em>, <em>21</em>(8),
1444–1453. (<a
href="https://doi.org/10.1038/s41592-024-02362-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning methods for extracting patterns from high-dimensional data are very important in the biological sciences. However, in certain cases, real-world applications cannot confirm the reported prediction performance. One of the main reasons for this is data leakage, which can be seen as the illicit sharing of information between the training data and the test data, resulting in performance estimates that are far better than the performance observed in the intended application scenario. Data leakage can be difficult to detect in biological datasets due to their complex dependencies. With this in mind, we present seven questions that should be asked to prevent data leakage when constructing machine learning models in biological domains. We illustrate the usefulness of our questions by applying them to nontrivial examples. Our goal is to raise awareness of potential data leakage problems and to promote robust and reproducible machine learning-based research in biology. This Perspective discusses the issue of data leakage in machine learning based models and presents seven questions designed to identify and avoid the problems resulting from data leakage.},
  archive      = {J_NMETH},
  author       = {Bernett, Judith and Blumenthal, David B. and Grimm, Dominik G. and Haselbeck, Florian and Joeres, Roman and Kalinina, Olga V. and List, Markus},
  doi          = {10.1038/s41592-024-02362-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1444-1453},
  shortjournal = {Nat. Meth.},
  title        = {Guiding questions to avoid data leakage in biological machine learning applications},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Transformers in single-cell omics: A review and new
perspectives. <em>NMETH</em>, <em>21</em>(8), 1430–1443. (<a
href="https://doi.org/10.1038/s41592-024-02353-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent efforts to construct reference maps of cellular phenotypes have expanded the volume and diversity of single-cell omics data, providing an unprecedented resource for studying cell properties. Despite the availability of rich datasets and their continued growth, current single-cell models are unable to fully capitalize on the information they contain. Transformers have become the architecture of choice for foundation models in other domains owing to their ability to generalize to heterogeneous, large-scale datasets. Thus, the question arises of whether transformers could set off a similar shift in the field of single-cell modeling. Here we first describe the transformer architecture and its single-cell adaptations and then present a comprehensive review of the existing applications of transformers in single-cell analysis and critically discuss their future potential for single-cell biology. By studying limitations and technical challenges, we aim to provide a structured outlook for future research directions at the intersection of machine learning and single-cell biology. This Perspective presents a comprehensive and in-depth overview of computational models based on the deep learning architecture of transformers for single-cell omics analysis.},
  archive      = {J_NMETH},
  author       = {Szałata, Artur and Hrovatin, Karin and Becker, Sören and Tejada-Lapuerta, Alejandro and Cui, Haotian and Wang, Bo and Theis, Fabian J.},
  doi          = {10.1038/s41592-024-02353-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1430-1443},
  shortjournal = {Nat. Meth.},
  title        = {Transformers in single-cell omics: A review and new perspectives},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Language models for biological research: A primer.
<em>NMETH</em>, <em>21</em>(8), 1422–1429. (<a
href="https://doi.org/10.1038/s41592-024-02354-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Language models are playing an increasingly important role in many areas of artificial intelligence (AI) and computational biology. In this primer, we discuss the ways in which language models, both those based on natural language and those based on biological sequences, can be applied to biological research. This primer is primarily intended for biologists interested in using these cutting-edge AI technologies in their applications. We provide guidance on best practices and key resources for adapting language models for biology. This primer introduces the elements and best practices of adapting and using language models for biology.},
  archive      = {J_NMETH},
  author       = {Simon, Elana and Swanson, Kyle and Zou, James},
  doi          = {10.1038/s41592-024-02354-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1422-1429},
  shortjournal = {Nat. Meth.},
  title        = {Language models for biological research: A primer},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Contextual AI models for context-specific prediction in
biology. <em>NMETH</em>, <em>21</em>(8), 1420–1421. (<a
href="https://doi.org/10.1038/s41592-024-02342-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed PINNACLE, a graph-based AI model for learning protein representations across cell-type contexts. These contextualized protein representations enable the integration of 3D protein structure with single-cell genomic-based representations to enhance protein–protein interaction prediction, analysis of drug effects across cell-type contexts, and prediction of therapeutic targets in a cell type-specific manner.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02342-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1420-1421},
  shortjournal = {Nat. Meth.},
  title        = {Contextual AI models for context-specific prediction in biology},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Studying RNA dynamics from single-cell RNA sequencing
snapshots. <em>NMETH</em>, <em>21</em>(8), 1418–1419. (<a
href="https://doi.org/10.1038/s41592-024-02366-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid advancements in transcriptomics have enabled the quantification of individual transcripts for thousands of genes in millions of single cells. By coupling a machine learning inference framework with biophysical models describing the RNA life cycle, we can explore the dynamics driving RNA production, processing and degradation across cell types.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02366-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1418-1419},
  shortjournal = {Nat. Meth.},
  title        = {Studying RNA dynamics from single-cell RNA sequencing snapshots},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Toward learning a foundational representation of cells and
genes. <em>NMETH</em>, <em>21</em>(8), 1416–1417. (<a
href="https://doi.org/10.1038/s41592-024-02367-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by the success of large-scale machine learning models in natural language, several groups are adapting these models for cellular data using massive single-cell datasets.},
  archive      = {J_NMETH},
  author       = {Lotfollahi, Mohammad},
  doi          = {10.1038/s41592-024-02367-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1416-1417},
  shortjournal = {Nat. Meth.},
  title        = {Toward learning a foundational representation of cells and genes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024o). Quest for AI literacy. <em>NMETH</em>, <em>21</em>(8),
1412–1415. (<a
href="https://doi.org/10.1038/s41592-024-02369-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As scientists avidly use, tinker and build with artificial intelligence tools, best practices begin to emerge.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02369-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1412-1415},
  shortjournal = {Nat. Meth.},
  title        = {Quest for AI literacy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Proximity-triggered protein trans-splicing. <em>NMETH</em>,
<em>21</em>(8), 1411. (<a
href="https://doi.org/10.1038/s41592-024-02385-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02385-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1411},
  shortjournal = {Nat. Meth.},
  title        = {Proximity-triggered protein trans-splicing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Splicing inhibition unlocks human totipotency.
<em>NMETH</em>, <em>21</em>(8), 1411. (<a
href="https://doi.org/10.1038/s41592-024-02386-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Nakhle, Jean},
  doi          = {10.1038/s41592-024-02386-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1411},
  shortjournal = {Nat. Meth.},
  title        = {Splicing inhibition unlocks human totipotency},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Epigenetic editing with CHARM. <em>NMETH</em>,
<em>21</em>(8), 1410. (<a
href="https://doi.org/10.1038/s41592-024-02387-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02387-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1410},
  shortjournal = {Nat. Meth.},
  title        = {Epigenetic editing with CHARM},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). The human brain revealed. <em>NMETH</em>, <em>21</em>(8),
1410. (<a href="https://doi.org/10.1038/s41592-024-02389-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02389-1},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1410},
  shortjournal = {Nat. Meth.},
  title        = {The human brain revealed},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Studying naturalistic behavior virtually. <em>NMETH</em>,
<em>21</em>(8), 1409. (<a
href="https://doi.org/10.1038/s41592-024-02388-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Virtual models of rats can be used to simulate behaviors and gain insights into the underlying neural activity.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02388-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1409},
  shortjournal = {Nat. Meth.},
  title        = {Studying naturalistic behavior virtually},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). AI meets biology: A call for community governance.
<em>NMETH</em>, <em>21</em>(8), 1407–1408. (<a
href="https://doi.org/10.1038/s41592-024-02332-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Risks from AI in basic biology research can be addressed with a dual mitigation strategy that comprises basic education in AI ethics and community governance measures that are tailored to the needs of individual research communities.},
  archive      = {J_NMETH},
  author       = {Prunkl, Carina},
  doi          = {10.1038/s41592-024-02332-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1407-1408},
  shortjournal = {Nat. Meth.},
  title        = {AI meets biology: A call for community governance},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A thousand and one tumors: The promise of AI for cancer
biology. <em>NMETH</em>, <em>21</em>(8), 1403–1406. (<a
href="https://doi.org/10.1038/s41592-024-02364-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breakthroughs in AI and multimodal genomics are unlocking the ability to study the tumor microenvironment. We explore promising machine learning techniques to integrate and interpret high-dimensional data, examine cellular dynamics and unravel gene regulatory mechanisms, ultimately enhancing our understanding of tumor progression and resistance.},
  archive      = {J_NMETH},
  author       = {Fan, Joy Linyue and Nazaret, Achille and Azizi, Elham},
  doi          = {10.1038/s41592-024-02364-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1403-1406},
  shortjournal = {Nat. Meth.},
  title        = {A thousand and one tumors: The promise of AI for cancer biology},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unlocking human immune system complexity through AI.
<em>NMETH</em>, <em>21</em>(8), 1400–1402. (<a
href="https://doi.org/10.1038/s41592-024-02351-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advancements in artificial intelligence (AI) have led to unprecedented success in modeling technically challenging domains including language, audio, image and video understanding. Here we discuss the opportunities represented by recent AI methods to advance immunology research.},
  archive      = {J_NMETH},
  author       = {Berson, Eloise and Chung, Philip and Espinosa, Camilo and Montine, Thomas J. and Aghaeepour, Nima},
  doi          = {10.1038/s41592-024-02351-1},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1400-1402},
  shortjournal = {Nat. Meth.},
  title        = {Unlocking human immune system complexity through AI},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Next-generation AI for connectomics. <em>NMETH</em>,
<em>21</em>(8), 1398–1399. (<a
href="https://doi.org/10.1038/s41592-024-02336-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New approaches in artificial intelligence (AI), such as foundation models and synthetic data, are having a substantial impact on many areas of applied computer science. Here we discuss the potential to apply these developments to the computational challenges associated with producing synapse-resolution maps of nervous systems, an area in which major ambitions are currently bottlenecked by AI performance.},
  archive      = {J_NMETH},
  author       = {Januszewski, Michał and Jain, Viren},
  doi          = {10.1038/s41592-024-02336-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1398-1399},
  shortjournal = {Nat. Meth.},
  title        = {Next-generation AI for connectomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Visual interpretability of bioimaging deep learning models.
<em>NMETH</em>, <em>21</em>(8), 1394–1397. (<a
href="https://doi.org/10.1038/s41592-024-02322-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The success of deep learning in analyzing bioimages comes at the expense of biologically meaningful interpretations. We review the state of the art of explainable artificial intelligence (XAI) in bioimaging and discuss its potential in hypothesis generation and data-driven discovery.},
  archive      = {J_NMETH},
  author       = {Rotem, Oded and Zaritsky, Assaf},
  doi          = {10.1038/s41592-024-02322-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1394-1397},
  shortjournal = {Nat. Meth.},
  title        = {Visual interpretability of bioimaging deep learning models},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multimodal large language models for bioimage analysis.
<em>NMETH</em>, <em>21</em>(8), 1390–1393. (<a
href="https://doi.org/10.1038/s41592-024-02334-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multimodal large language models have been recognized as a historical milestone in the field of artificial intelligence and have demonstrated revolutionary potentials not only in commercial applications, but also for many scientific fields. Here we give a brief overview of multimodal large language models through the lens of bioimage analysis and discuss how we could build these models as a community to facilitate biology research.},
  archive      = {J_NMETH},
  author       = {Zhang, Shanghang and Dai, Gaole and Huang, Tiejun and Chen, Jianxu},
  doi          = {10.1038/s41592-024-02334-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1390-1393},
  shortjournal = {Nat. Meth.},
  title        = {Multimodal large language models for bioimage analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Beyond protein lists: AI-assisted interpretation of
proteomic investigations in the context of evolving scientific
knowledge. <em>NMETH</em>, <em>21</em>(8), 1387–1389. (<a
href="https://doi.org/10.1038/s41592-024-02324-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectrometry-based proteomics provides broad and quantitative detection of the proteome, but its results are mostly presented as protein lists. Artificial intelligence approaches will exploit prior knowledge from literature and harmonize fragmented datasets to enable mechanistic and functional interpretation of proteomics experiments.},
  archive      = {J_NMETH},
  author       = {Gyori, Benjamin M. and Vitek, Olga},
  doi          = {10.1038/s41592-024-02324-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1387-1389},
  shortjournal = {Nat. Meth.},
  title        = {Beyond protein lists: AI-assisted interpretation of proteomic investigations in the context of evolving scientific knowledge},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Programmable biology through artificial intelligence: From
nucleic acids to proteins to cells. <em>NMETH</em>, <em>21</em>(8),
1384–1386. (<a
href="https://doi.org/10.1038/s41592-024-02338-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence-enabled computational tools not only help us to elucidate biological processes but also facilitate the programming of biology through molecular and cellular engineering.},
  archive      = {J_NMETH},
  author       = {Abudayyeh, Omar O. and Gootenberg, Jonathan S.},
  doi          = {10.1038/s41592-024-02338-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1384-1386},
  shortjournal = {Nat. Meth.},
  title        = {Programmable biology through artificial intelligence: From nucleic acids to proteins to cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Towards the prediction of general biomolecular interactions
with AI. <em>NMETH</em>, <em>21</em>(8), 1382–1383. (<a
href="https://doi.org/10.1038/s41592-024-02350-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods for predicting bimolecular interactions are seeing tremendous growth, but challenges remain in capturing the full physical complexity of these interactions.},
  archive      = {J_NMETH},
  author       = {Baek, Minkyung},
  doi          = {10.1038/s41592-024-02350-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1382-1383},
  shortjournal = {Nat. Meth.},
  title        = {Towards the prediction of general biomolecular interactions with AI},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unlocking the power of spatial omics with AI.
<em>NMETH</em>, <em>21</em>(8), 1378–1381. (<a
href="https://doi.org/10.1038/s41592-024-02363-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics technologies have transformed biomedical research by offering detailed, spatially resolved molecular profiles that elucidate tissue structure and function at unprecedented levels. AI can potentially unlock the full power of spatial omics, facilitating the integration of complex datasets and discovery of novel biomedical insights.},
  archive      = {J_NMETH},
  author       = {Coleman, Kyle and Schroeder, Amelia and Li, Mingyao},
  doi          = {10.1038/s41592-024-02363-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1378-1381},
  shortjournal = {Nat. Meth.},
  title        = {Unlocking the power of spatial omics with AI},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unlocking gene regulation with sequence-to-function models.
<em>NMETH</em>, <em>21</em>(8), 1374–1377. (<a
href="https://doi.org/10.1038/s41592-024-02331-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By exploiting recent advances in modern artificial intelligence and large-scale functional genomic datasets, sequence-to-function models learn the relationship between genomic DNA and its multilayer gene regulatory functions. These models are poised to uncover mechanistic relationships across layers of cellular biology, which will transform our understanding of cis gene regulation and open new avenues for discovering disease mechanisms.},
  archive      = {J_NMETH},
  author       = {Sasse, Alexander and Chikina, Maria and Mostafavi, Sara},
  doi          = {10.1038/s41592-024-02331-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1374-1377},
  shortjournal = {Nat. Meth.},
  title        = {Unlocking gene regulation with sequence-to-function models},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Omega — harnessing the power of large language models for
bioimage analysis. <em>NMETH</em>, <em>21</em>(8), 1371–1373. (<a
href="https://doi.org/10.1038/s41592-024-02310-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Royer, Loïc A.},
  doi          = {10.1038/s41592-024-02310-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1371-1373},
  shortjournal = {Nat. Meth.},
  title        = {Omega — Harnessing the power of large language models for bioimage analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BioImage.IO chatbot: A community-driven AI assistant for
integrative computational bioimaging. <em>NMETH</em>, <em>21</em>(8),
1368–1370. (<a
href="https://doi.org/10.1038/s41592-024-02370-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Lei, Wanlu and Fuster-Barceló, Caterina and Reder, Gabriel and Muñoz-Barrutia, Arrate and Ouyang, Wei},
  doi          = {10.1038/s41592-024-02370-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1368-1370},
  shortjournal = {Nat. Meth.},
  title        = {BioImage.IO chatbot: A community-driven AI assistant for integrative computational bioimaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Embedding AI in biology. <em>NMETH</em>, <em>21</em>(8),
1365–1366. (<a
href="https://doi.org/10.1038/s41592-024-02391-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advanced artificial intelligence approaches are rapidly transforming how biological data are acquired and analyzed.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02391-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1365-1366},
  shortjournal = {Nat. Meth.},
  title        = {Embedding AI in biology},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). First-gen scientists leap hurdles and give back.
<em>NMETH</em>, <em>21</em>(8), 1367. (<a
href="https://doi.org/10.1038/s41592-024-02368-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Being first in a family to aim for a science career takes resilience and resourcefulness.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02368-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {8},
  pages        = {1367},
  shortjournal = {Nat. Meth.},
  title        = {First-gen scientists leap hurdles and give back},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic assessment of long-read RNA-seq methods for
transcript identification and quantification. <em>NMETH</em>,
<em>21</em>(7), 1349–1363. (<a
href="https://doi.org/10.1038/s41592-024-02298-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Long-read RNA-Seq Genome Annotation Assessment Project Consortium was formed to evaluate the effectiveness of long-read approaches for transcriptome analysis. Using different protocols and sequencing platforms, the consortium generated over 427 million long-read sequences from complementary DNA and direct RNA datasets, encompassing human, mouse and manatee species. Developers utilized these data to address challenges in transcript isoform detection, quantification and de novo transcript detection. The study revealed that libraries with longer, more accurate sequences produce more accurate transcripts than those with increased read depth, whereas greater read depth improved quantification accuracy. In well-annotated genomes, tools based on reference sequences demonstrated the best performance. Incorporating additional orthogonal data and replicate samples is advised when aiming to detect rare and novel transcripts or using reference-free approaches. This collaborative study offers a benchmark for current practices and provides direction for future method development in transcriptome analysis. This Registered Report presents the results of the Long-read RNA-Seq Genome Annotation Assessment Project, which is a community effort for benchmarking long-read methods for transcriptome analyses, including transcript isoform detection, quantification and de novo transcript detection.},
  archive      = {J_NMETH},
  author       = {Pardo-Palacios, Francisco J. and Wang, Dingjie and Reese, Fairlie and Diekhans, Mark and Carbonell-Sala, Sílvia and Williams, Brian and Loveland, Jane E. and De María, Maite and Adams, Matthew S. and Balderrama-Gutierrez, Gabriela and Behera, Amit K. and Gonzalez Martinez, Jose M. and Hunt, Toby and Lagarde, Julien and Liang, Cindy E. and Li, Haoran and Meade, Marcus Jerryd and Moraga Amador, David A. and Prjibelski, Andrey D. and Birol, Inanc and Bostan, Hamed and Brooks, Ashley M. and Çelik, Muhammed Hasan and Chen, Ying and Du, Mei R. M. and Felton, Colette and Göke, Jonathan and Hafezqorani, Saber and Herwig, Ralf and Kawaji, Hideya and Lee, Joseph and Li, Jian-Liang and Lienhard, Matthias and Mikheenko, Alla and Mulligan, Dennis and Nip, Ka Ming and Pertea, Mihaela and Ritchie, Matthew E. and Sim, Andre D. and Tang, Alison D. and Wan, Yuk Kei and Wang, Changqing and Wong, Brandon Y. and Yang, Chen and Barnes, If and Berry, Andrew E. and Capella-Gutierrez, Salvador and Cousineau, Alyssa and Dhillon, Namrita and Fernandez-Gonzalez, Jose M. and Ferrández-Peral, Luis and Garcia-Reyero, Natàlia and Götz, Stefan and Hernández-Ferrer, Carles and Kondratova, Liudmyla and Liu, Tianyuan and Martinez-Martin, Alessandra and Menor, Carlos and Mestre-Tomás, Jorge and Mudge, Jonathan M. and Panayotova, Nedka G. and Paniagua, Alejandro and Repchevsky, Dmitry and Ren, Xingjie and Rouchka, Eric and Saint-John, Brandon and Sapena, Enrique and Sheynkman, Leon and Smith, Melissa Laird and Suner, Marie-Marthe and Takahashi, Hazuki and Youngworth, Ingrid A. and Carninci, Piero and Denslow, Nancy D. and Guigó, Roderic and Hunter, Margaret E. and Maehr, Rene and Shen, Yin and Tilgner, Hagen U. and Wold, Barbara J. and Vollmers, Christopher and Frankish, Adam and Au, Kin Fai and Sheynkman, Gloria M. and Mortazavi, Ali and Conesa, Ana and Brooks, Angela N.},
  doi          = {10.1038/s41592-024-02298-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1349-1363},
  shortjournal = {Nat. Meth.},
  title        = {Systematic assessment of long-read RNA-seq methods for transcript identification and quantification},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Outcomes of the EMDataResource cryo-EM ligand modeling
challenge. <em>NMETH</em>, <em>21</em>(7), 1340–1348. (<a
href="https://doi.org/10.1038/s41592-024-02321-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The EMDataResource Ligand Model Challenge aimed to assess the reliability and reproducibility of modeling ligands bound to protein and protein–nucleic acid complexes in cryogenic electron microscopy (cryo-EM) maps determined at near-atomic (1.9–2.5 Å) resolution. Three published maps were selected as targets: Escherichia coli beta-galactosidase with inhibitor, SARS-CoV-2 virus RNA-dependent RNA polymerase with covalently bound nucleotide analog and SARS-CoV-2 virus ion channel ORF3a with bound lipid. Sixty-one models were submitted from 17 independent research groups, each with supporting workflow details. The quality of submitted ligand models and surrounding atoms were analyzed by visual inspection and quantification of local map quality, model-to-map fit, geometry, energetics and contact scores. A composite rather than a single score was needed to assess macromolecule+ligand model quality. These observations lead us to recommend best practices for assessing cryo-EM structures of liganded macromolecules reported at near-atomic resolution. The EMDataResource Ligand Model Challenge aimed at assessing the reliability and reproducibility of modeling ligands bound to protein and protein–nucleic acid complexes in cryo-EM maps determined at near-atomic resolution. This analysis presents the results and recommends best practices for assessing cryo-EM structures of liganded macromolecules.},
  archive      = {J_NMETH},
  author       = {Lawson, Catherine L. and Kryshtafovych, Andriy and Pintilie, Grigore D. and Burley, Stephen K. and Černý, Jiří and Chen, Vincent B. and Emsley, Paul and Gobbi, Alberto and Joachimiak, Andrzej and Noreng, Sigrid and Prisant, Michael G. and Read, Randy J. and Richardson, Jane S. and Rohou, Alexis L. and Schneider, Bohdan and Sellers, Benjamin D. and Shao, Chenghua and Sourial, Elizabeth and Williams, Chris I. and Williams, Christopher J. and Yang, Ying and Abbaraju, Venkat and Afonine, Pavel V. and Baker, Matthew L. and Bond, Paul S. and Blundell, Tom L. and Burnley, Tom and Campbell, Arthur and Cao, Renzhi and Cheng, Jianlin and Chojnowski, Grzegorz and Cowtan, K. D. and DiMaio, Frank and Esmaeeli, Reza and Giri, Nabin and Grubmüller, Helmut and Hoh, Soon Wen and Hou, Jie and Hryc, Corey F. and Hunte, Carola and Igaev, Maxim and Joseph, Agnel P. and Kao, Wei-Chun and Kihara, Daisuke and Kumar, Dilip and Lang, Lijun and Lin, Sean and Maddhuri Venkata Subramaniya, Sai R. and Mittal, Sumit and Mondal, Arup and Moriarty, Nigel W. and Muenks, Andrew and Murshudov, Garib N. and Nicholls, Robert A. and Olek, Mateusz and Palmer, Colin M. and Perez, Alberto and Pohjolainen, Emmi and Pothula, Karunakar R. and Rowley, Christopher N. and Sarkar, Daipayan and Schäfer, Luisa U. and Schlicksup, Christopher J. and Schröder, Gunnar F. and Shekhar, Mrinal and Si, Dong and Singharoy, Abhishek and Sobolev, Oleg V. and Terashi, Genki and Vaiana, Andrea C. and Vedithi, Sundeep C. and Verburgt, Jacob and Wang, Xiao and Warshamanage, Rangana and Winn, Martyn D. and Weyand, Simone and Yamashita, Keitaro and Zhao, Minglei and Schmid, Michael F. and Berman, Helen M. and Chiu, Wah},
  doi          = {10.1038/s41592-024-02321-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1340-1348},
  shortjournal = {Nat. Meth.},
  title        = {Outcomes of the EMDataResource cryo-EM ligand modeling challenge},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Keypoint-MoSeq: Parsing behavior by linking point tracking
to pose dynamics. <em>NMETH</em>, <em>21</em>(7), 1329–1339. (<a
href="https://doi.org/10.1038/s41592-024-02318-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Keypoint tracking algorithms can flexibly quantify animal movement from videos obtained in a wide variety of settings. However, it remains unclear how to parse continuous keypoint data into discrete actions. This challenge is particularly acute because keypoint data are susceptible to high-frequency jitter that clustering algorithms can mistake for transitions between actions. Here we present keypoint-MoSeq, a machine learning-based platform for identifying behavioral modules (‘syllables’) from keypoint data without human supervision. Keypoint-MoSeq uses a generative model to distinguish keypoint noise from behavior, enabling it to identify syllables whose boundaries correspond to natural sub-second discontinuities in pose dynamics. Keypoint-MoSeq outperforms commonly used alternative clustering methods at identifying these transitions, at capturing correlations between neural activity and behavior and at classifying either solitary or social behaviors in accordance with human annotations. Keypoint-MoSeq also works in multiple species and generalizes beyond the syllable timescale, identifying fast sniff-aligned movements in mice and a spectrum of oscillatory behaviors in fruit flies. Keypoint-MoSeq, therefore, renders accessible the modular structure of behavior through standard video recordings. Keypoint-MoSeq is an unsupervised behavior segmentation algorithm that extracts behavioral modules from keypoint tracking data acquired with diverse algorithms, as demonstrated in mice, rats and fruit flies. The extracted modules faithfully reflect human-annotated behaviors even though they are obtained in an unsupervised fashion.},
  archive      = {J_NMETH},
  author       = {Weinreb, Caleb and Pearl, Jonah E. and Lin, Sherry and Osman, Mohammed Abdal Monium and Zhang, Libby and Annapragada, Sidharth and Conlin, Eli and Hoffmann, Red and Makowska, Sofia and Gillis, Winthrop F. and Jay, Maya and Ye, Shaokai and Mathis, Alexander and Mathis, Mackenzie W. and Pereira, Talmo and Linderman, Scott W. and Datta, Sandeep Robert},
  doi          = {10.1038/s41592-024-02318-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1329-1339},
  shortjournal = {Nat. Meth.},
  title        = {Keypoint-MoSeq: Parsing behavior by linking point tracking to pose dynamics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Lightning pose: Improved animal pose estimation via
semi-supervised learning, bayesian ensembling and cloud-native
open-source tools. <em>NMETH</em>, <em>21</em>(7), 1316–1328. (<a
href="https://doi.org/10.1038/s41592-024-02319-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Contemporary pose estimation methods enable precise measurements of behavior via supervised deep learning with hand-labeled video frames. Although effective in many cases, the supervised approach requires extensive labeling and often produces outputs that are unreliable for downstream analyses. Here, we introduce ‘Lightning Pose’, an efficient pose estimation package with three algorithmic contributions. First, in addition to training on a few labeled video frames, we use many unlabeled videos and penalize the network whenever its predictions violate motion continuity, multiple-view geometry and posture plausibility (semi-supervised learning). Second, we introduce a network architecture that resolves occlusions by predicting pose on any given frame using surrounding unlabeled frames. Third, we refine the pose predictions post hoc by combining ensembling and Kalman smoothing. Together, these components render pose trajectories more accurate and scientifically usable. We released a cloud application that allows users to label data, train networks and process new videos directly from the browser. Lightning Pose is an efficient pose estimation approach that requires few labeled training data owing to its semi-supervised learning strategy and ensembling.},
  archive      = {J_NMETH},
  author       = {Biderman, Dan and Whiteway, Matthew R. and Hurwitz, Cole and Greenspan, Nicholas and Lee, Robert S. and Vishnubhotla, Ankit and Warren, Richard and Pedraja, Federico and Noone, Dillon and Schartner, Michael M. and Huntenburg, Julia M. and Khanal, Anup and Meijer, Guido T. and Noel, Jean-Paul and Pan-Vazquez, Alejandro and Socha, Karolina Z. and Urai, Anne E. and Cunningham, John P. and Sawtell, Nathaniel B. and Paninski, Liam},
  doi          = {10.1038/s41592-024-02319-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1316-1328},
  shortjournal = {Nat. Meth.},
  title        = {Lightning pose: Improved animal pose estimation via semi-supervised learning, bayesian ensembling and cloud-native open-source tools},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Virtual reality-empowered deep-learning analysis of brain
cells. <em>NMETH</em>, <em>21</em>(7), 1306–1315. (<a
href="https://doi.org/10.1038/s41592-024-02245-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Automated detection of specific cells in three-dimensional datasets such as whole-brain light-sheet image stacks is challenging. Here, we present DELiVR, a virtual reality-trained deep-learning pipeline for detecting c-Fos+ cells as markers for neuronal activity in cleared mouse brains. Virtual reality annotation substantially accelerated training data generation, enabling DELiVR to outperform state-of-the-art cell-segmenting approaches. Our pipeline is available in a user-friendly Docker container that runs with a standalone Fiji plugin. DELiVR features a comprehensive toolkit for data visualization and can be customized to other cell types of interest, as we did here for microglia somata, using Fiji for dataset-specific training. We applied DELiVR to investigate cancer-related brain activity, unveiling an activation pattern that distinguishes weight-stable cancer from cancers associated with weight loss. Overall, DELiVR is a robust deep-learning tool that does not require advanced coding skills to analyze whole-brain imaging data in health and disease. Generating training data for training deep-learning-based tools is time consuming. The DELiVR pipeline facilitates this process as demonstrated in this study on detecting c-Fos+ cells or microglia in the brain, following tissue clearing and imaging with light-sheet microscopy.},
  archive      = {J_NMETH},
  author       = {Kaltenecker, Doris and Al-Maskari, Rami and Negwer, Moritz and Hoeher, Luciano and Kofler, Florian and Zhao, Shan and Todorov, Mihail and Rong, Zhouyi and Paetzold, Johannes Christian and Wiestler, Benedikt and Piraud, Marie and Rueckert, Daniel and Geppert, Julia and Morigny, Pauline and Rohm, Maria and Menze, Bjoern H. and Herzig, Stephan and Berriel Diaz, Mauricio and Ertürk, Ali},
  doi          = {10.1038/s41592-024-02245-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1306-1315},
  shortjournal = {Nat. Meth.},
  title        = {Virtual reality-empowered deep-learning analysis of brain cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-throughput volumetric mapping of synaptic transmission.
<em>NMETH</em>, <em>21</em>(7), 1298–1305. (<a
href="https://doi.org/10.1038/s41592-024-02309-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volumetric imaging of synaptic transmission in vivo requires high spatial and high temporal resolution. Shaping the wavefront of two-photon fluorescence excitation light, we developed Bessel-droplet foci for high-contrast and high-resolution volumetric imaging of synapses. Applying our method to imaging glutamate release, we demonstrated high-throughput mapping of excitatory inputs at &amp;gt;1,000 synapses per volume and &amp;gt;500 dendritic spines per neuron in vivo and unveiled previously unseen features of functional synaptic organization in the mouse primary visual cortex. Bessel-droplet two-photon fluorescence microscopy offers high-contrast and high-resolution volumetric imaging in vivo and can be used for high-throughput mapping of functional synaptic organization in the mouse brain.},
  archive      = {J_NMETH},
  author       = {Chen, Wei and Ge, Xinxin and Zhang, Qinrong and Natan, Ryan G. and Fan, Jiang Lan and Scanziani, Massimo and Ji, Na},
  doi          = {10.1038/s41592-024-02309-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1298-1305},
  shortjournal = {Nat. Meth.},
  title        = {High-throughput volumetric mapping of synaptic transmission},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DART.2: Bidirectional synaptic pharmacology with
thousandfold cellular specificity. <em>NMETH</em>, <em>21</em>(7),
1288–1297. (<a
href="https://doi.org/10.1038/s41592-024-02292-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precision pharmacology aims to manipulate specific cellular interactions within complex tissues. In this pursuit, we introduce DART.2 (drug acutely restricted by tethering), a second-generation cell-specific pharmacology technology. The core advance is optimized cellular specificity—up to 3,000-fold in 15 min—enabling the targeted delivery of even epileptogenic drugs without off-target effects. Additionally, we introduce brain-wide dosing methods as an alternative to local cannulation and tracer reagents for brain-wide dose quantification. We describe four pharmaceuticals—two that antagonize excitatory and inhibitory postsynaptic receptors, and two that allosterically potentiate these receptors. Their versatility is showcased across multiple mouse-brain regions, including cerebellum, striatum, visual cortex and retina. Finally, in the ventral tegmental area, we find that blocking inhibitory inputs to dopamine neurons accelerates locomotion, contrasting with previous optogenetic and pharmacological findings. Beyond enabling the bidirectional perturbation of chemical synapses, these reagents offer intersectional precision—between genetically defined postsynaptic cells and neurotransmitter-defined presynaptic partners. DART (drug acutely restricted by tethering) enables the manipulation of native receptors on genetically defined neurons. This work describes second-generation DART reagents for manipulating GABAA and AMPA receptors with higher cellular specificity than previously achieved.},
  archive      = {J_NMETH},
  author       = {Shields, Brenda C. and Yan, Haidun and Lim, Shaun S. X. and Burwell, Sasha C. V. and Cammarata, Celine M. and Fleming, Elizabeth A. and Yousefzadeh, S. Aryana and Goldenshtein, Victoria Z. and Kahuno, Elizabeth W. and Vagadia, Purav P. and Loughran, Marie H. and Zhiquan, Lei and McDonnell, Mark E. and Scalabrino, Miranda L. and Thapa, Mishek and Hawley, Tammy M. and Field, Greg D. and Hull, Court and Schiltz, Gary E. and Glickfeld, Lindsey L. and Reitz, Allen B. and Tadross, Michael R.},
  doi          = {10.1038/s41592-024-02292-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1288-1297},
  shortjournal = {Nat. Meth.},
  title        = {DART.2: Bidirectional synaptic pharmacology with thousandfold cellular specificity},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A bistable inhibitory optoGPCR for multiplexed optogenetic
control of neural circuits. <em>NMETH</em>, <em>21</em>(7), 1275–1287.
(<a href="https://doi.org/10.1038/s41592-024-02285-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Information is transmitted between brain regions through the release of neurotransmitters from long-range projecting axons. Understanding how the activity of such long-range connections contributes to behavior requires efficient methods for reversibly manipulating their function. Chemogenetic and optogenetic tools, acting through endogenous G-protein-coupled receptor pathways, can be used to modulate synaptic transmission, but existing tools are limited in sensitivity, spatiotemporal precision or spectral multiplexing capabilities. Here we systematically evaluated multiple bistable opsins for optogenetic applications and found that the Platynereis dumerilii ciliary opsin (PdCO) is an efficient, versatile, light-activated bistable G-protein-coupled receptor that can suppress synaptic transmission in mammalian neurons with high temporal precision in vivo. PdCO has useful biophysical properties that enable spectral multiplexing with other optogenetic actuators and reporters. We demonstrate that PdCO can be used to conduct reversible loss-of-function experiments in long-range projections of behaving animals, thereby enabling detailed synapse-specific functional circuit mapping. PdCO is a switchable optogenetic tool for inhibiting synaptic transmission in neuronal terminals in vivo, as demonstrated in a variety of contexts mainly in the mouse.},
  archive      = {J_NMETH},
  author       = {Wietek, Jonas and Nozownik, Adrianna and Pulin, Mauro and Saraf-Sinik, Inbar and Matosevich, Noa and Gowrishankar, Raajaram and Gat, Asaf and Malan, Daniela and Brown, Bobbie J. and Dine, Julien and Imambocus, Bibi Nusreen and Levy, Rivka and Sauter, Kathrin and Litvin, Anna and Regev, Noa and Subramaniam, Suraj and Abrera, Khalid and Summarli, Dustin and Goren, Eva Madeline and Mizrachi, Gili and Bitton, Eyal and Benjamin, Asaf and Copits, Bryan A. and Sasse, Philipp and Rost, Benjamin R. and Schmitz, Dietmar and Bruchas, Michael R. and Soba, Peter and Oren-Suissa, Meital and Nir, Yuval and Wiegert, J. Simon and Yizhar, Ofer},
  doi          = {10.1038/s41592-024-02285-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1275-1287},
  shortjournal = {Nat. Meth.},
  title        = {A bistable inhibitory optoGPCR for multiplexed optogenetic control of neural circuits},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Permittivity tensor imaging: Modular label-free imaging of
3D dry mass and 3D orientation at high resolution. <em>NMETH</em>,
<em>21</em>(7), 1257–1274. (<a
href="https://doi.org/10.1038/s41592-024-02291-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dry mass and the orientation of biomolecules can be imaged without a label by measuring their permittivity tensor (PT), which describes how biomolecules affect the phase and polarization of light. Three-dimensional (3D) imaging of PT has been challenging. We present a label-free computational microscopy technique, PT imaging (PTI), for the 3D measurement of PT. PTI encodes the invisible PT into images using oblique illumination, polarization-sensitive detection and volumetric sampling. PT is decoded from the data with a vectorial imaging model and a multi-channel inverse algorithm, assuming uniaxial symmetry in each voxel. We demonstrate high-resolution imaging of PT of isotropic beads, anisotropic glass targets, mouse brain tissue, infected cells and histology slides. PTI outperforms previous label-free imaging techniques such as vector tomography, ptychography and light-field imaging in resolving the 3D orientation and symmetry of organelles, cells and tissue. We provide open-source software and modular hardware to enable the adoption of the method. Permittivity tensor imaging is a label-free computational microscopy approach that enables the three-dimensional measurement of molecular permittivity tensors, revealing information about a biomolecule’s dry mass and orientation in cells and tissues.},
  archive      = {J_NMETH},
  author       = {Yeh, Li-Hao and Ivanov, Ivan E. and Chandler, Talon and Byrum, Janie R. and Chhun, Bryant B. and Guo, Syuan-Ming and Foltz, Cameron and Hashemi, Ezzat and Perez-Bermejo, Juan A. and Wang, Huijun and Yu, Yanhao and Kazansky, Peter G. and Conklin, Bruce R. and Han, May H. and Mehta, Shalin B.},
  doi          = {10.1038/s41592-024-02291-w},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1257-1274},
  shortjournal = {Nat. Meth.},
  title        = {Permittivity tensor imaging: Modular label-free imaging of 3D dry mass and 3D orientation at high resolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deconwolf enables high-performance deconvolution of
widefield fluorescence microscopy images. <em>NMETH</em>,
<em>21</em>(7), 1245–1256. (<a
href="https://doi.org/10.1038/s41592-024-02294-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microscopy-based spatially resolved omic methods are transforming the life sciences. However, these methods rely on high numerical aperture objectives and cannot resolve crowded molecular targets, limiting the amount of extractable biological information. To overcome these limitations, here we develop Deconwolf, an open-source, user-friendly software for high-performance deconvolution of widefield fluorescence microscopy images, which efficiently runs on laptop computers. Deconwolf enables accurate quantification of crowded diffraction limited fluorescence dots in DNA and RNA fluorescence in situ hybridization images and allows robust detection of individual transcripts in tissue sections imaged with ×20 air objectives. Deconvolution of in situ spatial transcriptomics images with Deconwolf increased the number of transcripts identified more than threefold, while the application of Deconwolf to images obtained by fluorescence in situ sequencing of barcoded Oligopaint probes drastically improved chromosome tracing. Deconwolf greatly facilitates the use of deconvolution in many bioimaging applications. Deconwolf is a computationally efficient and user-friendly software tool for fluorescence microscopy image deconvolution that improves the analysis of diverse fluorescence in situ hybridization methods and can handle large datasets.},
  archive      = {J_NMETH},
  author       = {Wernersson, Erik and Gelali, Eleni and Girelli, Gabriele and Wang, Su and Castillo, David and Mattsson Langseth, Christoffer and Verron, Quentin and Nguyen, Huy Q. and Chattoraj, Shyamtanu and Martinez Casals, Anna and Blom, Hans and Lundberg, Emma and Nilsson, Mats and Marti-Renom, Marc A. and Wu, Chao-ting and Crosetto, Nicola and Bienko, Magda},
  doi          = {10.1038/s41592-024-02294-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1245-1256},
  shortjournal = {Nat. Meth.},
  title        = {Deconwolf enables high-performance deconvolution of widefield fluorescence microscopy images},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accurate and efficient integrative reference-informed
spatial domain detection for spatial transcriptomics. <em>NMETH</em>,
<em>21</em>(7), 1231–1244. (<a
href="https://doi.org/10.1038/s41592-024-02284-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved transcriptomics (SRT) studies are becoming increasingly common and large, offering unprecedented opportunities in mapping complex tissue structures and functions. Here we present integrative and reference-informed tissue segmentation (IRIS), a computational method designed to characterize tissue spatial organization in SRT studies through accurately and efficiently detecting spatial domains. IRIS uniquely leverages single-cell RNA sequencing data for reference-informed detection of biologically interpretable spatial domains, integrating multiple SRT slices while explicitly considering correlations both within and across slices. We demonstrate the advantages of IRIS through in-depth analysis of six SRT datasets encompassing diverse technologies, tissues, species and resolutions. In these applications, IRIS achieves substantial accuracy gains (39–1,083%) and speed improvements (4.6–666.0) in moderate-sized datasets, while representing the only method applicable for large datasets including Stereo-seq and 10x Xenium. As a result, IRIS reveals intricate brain structures, uncovers tumor microenvironment heterogeneity and detects structural changes in diabetes-affected testis, all with exceptional speed and accuracy. Integrative and reference-informed tissue segmentation (IRIS) harnesses single-cell RNA sequencing data for the accurate identification of spatial domains in spatially resolved transcriptomics. IRIS is computationally efficient and uniquely suited for analyzing large datasets.},
  archive      = {J_NMETH},
  author       = {Ma, Ying and Zhou, Xiang},
  doi          = {10.1038/s41592-024-02284-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1231-1244},
  shortjournal = {Nat. Meth.},
  title        = {Accurate and efficient integrative reference-informed spatial domain detection for spatial transcriptomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Recovering true FRET efficiencies from smFRET investigations
requires triplet state mitigation. <em>NMETH</em>, <em>21</em>(7),
1222–1230. (<a
href="https://doi.org/10.1038/s41592-024-02293-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule fluorescence resonance energy transfer (smFRET) methods employed to quantify time-dependent compositional and conformational changes within biomolecules require elevated illumination intensities to recover robust photon emission streams from individual fluorophores. Here we show that outside the weak-excitation limit, and in regimes where fluorophores must undergo many rapid cycles of excitation and relaxation, non-fluorescing, excitation-induced triplet states with lifetimes orders of magnitude longer lived than photon-emitting singlet states degrade photon emission streams from both donor and acceptor fluorophores resulting in illumination-intensity-dependent changes in FRET efficiency. These changes are not commonly taken into consideration; therefore, robust strategies to suppress excited state accumulations are required to recover accurate and precise FRET efficiency, and thus distance, estimates. We propose both robust triplet state suppression and data correction strategies that enable the recovery of FRET efficiencies more closely approximating true values, thereby extending the spatial and temporal resolution of smFRET. Although not widely appreciated, measured smFRET efficiencies can vary depending on illumination intensity. Self-healing fluorophores offer robust triplet state suppression to enable accurate smFRET measurements under broad illumination regimes.},
  archive      = {J_NMETH},
  author       = {Pati, Avik K. and Kilic, Zeliha and Martin, Maxwell I. and Terry, Daniel S. and Borgia, Alessandro and Bar, Sukanta and Jockusch, Steffen and Kiselev, Roman and Altman, Roger B. and Blanchard, Scott C.},
  doi          = {10.1038/s41592-024-02293-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1222-1230},
  shortjournal = {Nat. Meth.},
  title        = {Recovering true FRET efficiencies from smFRET investigations requires triplet state mitigation},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Data-driven regularization lowers the size barrier of
cryo-EM structure determination. <em>NMETH</em>, <em>21</em>(7),
1216–1221. (<a
href="https://doi.org/10.1038/s41592-024-02304-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Macromolecular structure determination by electron cryo-microscopy (cryo-EM) is limited by the alignment of noisy images of individual particles. Because smaller particles have weaker signals, alignment errors impose size limitations on its applicability. Here, we explore how image alignment is improved by the application of deep learning to exploit prior knowledge about biological macromolecular structures that would otherwise be difficult to express mathematically. We train a denoising convolutional neural network on pairs of half-set reconstructions from the electron microscopy data bank (EMDB) and use this denoiser as an alternative to a commonly used smoothness prior. We demonstrate that this approach, which we call Blush regularization, yields better reconstructions than do existing algorithms, in particular for data with low signal-to-noise ratios. The reconstruction of a protein–nucleic acid complex with a molecular weight of 40 kDa, which was previously intractable, illustrates that denoising neural networks will expand the applicability of cryo-EM structure determination for a wide range of biological macromolecules. Blush regularization makes use of a neural network pre-trained on a diverse set of high-resolution cryo-EM half-maps to improve image alignment, effectively lowering the size barrier, during cryo-EM structure determination.},
  archive      = {J_NMETH},
  author       = {Kimanius, Dari and Jamali, Kiarash and Wilkinson, Max E. and Lövestam, Sofia and Velazhahan, Vaithish and Nakane, Takanori and Scheres, Sjors H. W.},
  doi          = {10.1038/s41592-024-02304-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1216-1221},
  shortjournal = {Nat. Meth.},
  title        = {Data-driven regularization lowers the size barrier of cryo-EM structure determination},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Predicting glycan structure from tandem mass spectrometry
via deep learning. <em>NMETH</em>, <em>21</em>(7), 1206–1215. (<a
href="https://doi.org/10.1038/s41592-024-02314-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glycans constitute the most complicated post-translational modification, modulating protein activity in health and disease. However, structural annotation from tandem mass spectrometry (MS/MS) data is a bottleneck in glycomics, preventing high-throughput endeavors and relegating glycomics to a few experts. Trained on a newly curated set of 500,000 annotated MS/MS spectra, here we present CandyCrunch, a dilated residual neural network predicting glycan structure from raw liquid chromatography–MS/MS data in seconds (top-1 accuracy: 90.3%). We developed an open-access Python-based workflow of raw data conversion and prediction, followed by automated curation and fragment annotation, with predictions recapitulating and extending expert annotation. We demonstrate that this can be used for de novo annotation, diagnostic fragment identification and high-throughput glycomics. For maximum impact, this entire pipeline is tightly interlaced with our glycowork platform and can be easily tested at https://colab.research.google.com/github/BojarLab/CandyCrunch/blob/main/CandyCrunch.ipynb . We envision CandyCrunch to democratize structural glycomics and the elucidation of biological roles of glycans. CandyCrunch is a deep learning-based tool for predicting glycan structures from tandem mass spectrometry data. The paper also introduces CandyCrumbs that automatically annotates fragment ions in higher-order tandem mass spectrometry spectra.},
  archive      = {J_NMETH},
  author       = {Urban, James and Jin, Chunsheng and Thomsson, Kristina A. and Karlsson, Niclas G. and Ives, Callum M. and Fadda, Elisa and Bojar, Daniel},
  doi          = {10.1038/s41592-024-02314-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1206-1215},
  shortjournal = {Nat. Meth.},
  title        = {Predicting glycan structure from tandem mass spectrometry via deep learning},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). CellRank 2: Unified fate mapping in multiview single-cell
data. <em>NMETH</em>, <em>21</em>(7), 1196–1205. (<a
href="https://doi.org/10.1038/s41592-024-02303-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing allows us to model cellular state dynamics and fate decisions using expression similarity or RNA velocity to reconstruct state-change trajectories; however, trajectory inference does not incorporate valuable time point information or utilize additional modalities, whereas methods that address these different data views cannot be combined or do not scale. Here we present CellRank 2, a versatile and scalable framework to study cellular fate using multiview single-cell data of up to millions of cells in a unified fashion. CellRank 2 consistently recovers terminal states and fate probabilities across data modalities in human hematopoiesis and endodermal development. Our framework also allows combining transitions within and across experimental time points, a feature we use to recover genes promoting medullary thymic epithelial cell formation during pharyngeal endoderm development. Moreover, we enable estimating cell-specific transcription and degradation rates from metabolic-labeling data, which we apply to an intestinal organoid system to delineate differentiation trajectories and pinpoint regulatory strategies. CellRank 2 is a comprehensive framework of cell fate analysis using large-scale multiview single-cell data.},
  archive      = {J_NMETH},
  author       = {Weiler, Philipp and Lange, Marius and Klein, Michal and Pe’er, Dana and Theis, Fabian},
  doi          = {10.1038/s41592-024-02303-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1196-1205},
  shortjournal = {Nat. Meth.},
  title        = {CellRank 2: Unified fate mapping in multiview single-cell data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantifying cell-state densities in single-cell phenotypic
landscapes using mellon. <em>NMETH</em>, <em>21</em>(7), 1185–1195. (<a
href="https://doi.org/10.1038/s41592-024-02302-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell-state density characterizes the distribution of cells along phenotypic landscapes and is crucial for unraveling the mechanisms that drive diverse biological processes. Here, we present Mellon, an algorithm for estimation of cell-state densities from high-dimensional representations of single-cell data. We demonstrate Mellon’s efficacy by dissecting the density landscape of differentiating systems, revealing a consistent pattern of high-density regions corresponding to major cell types intertwined with low-density, rare transitory states. We present evidence implicating enhancer priming and the activation of master regulators in emergence of these transitory states. Mellon offers the flexibility to perform temporal interpolation of time-series data, providing a detailed view of cell-state dynamics during developmental processes. Mellon facilitates density estimation across various single-cell data modalities, scaling linearly with the number of cells. Our work underscores the importance of cell-state density in understanding the differentiation processes, and the potential of Mellon to provide insights into mechanisms guiding biological trajectories. Mellon accurately estimates cell-state densities using high-dimensional single-cell data, unveiling regulatory dynamics and mechanisms in cell differentiation.},
  archive      = {J_NMETH},
  author       = {Otto, Dominik J. and Jordan, Cailin and Dury, Brennan and Dien, Christine and Setty, Manu},
  doi          = {10.1038/s41592-024-02302-w},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1185-1195},
  shortjournal = {Nat. Meth.},
  title        = {Quantifying cell-state densities in single-cell phenotypic landscapes using mellon},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantifying DNA replication speeds in single cells by
scEdU-seq. <em>NMETH</em>, <em>21</em>(7), 1175–1184. (<a
href="https://doi.org/10.1038/s41592-024-02308-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In a human cell, thousands of replication forks simultaneously coordinate duplication of the entire genome. The rate at which this process occurs might depend on the epigenetic state of the genome and vary between, or even within, cell types. To accurately measure DNA replication speeds, we developed single-cell 5-ethynyl-2′-deoxyuridine sequencing to detect nascent replicated DNA. We observed that the DNA replication speed is not constant but increases during S phase of the cell cycle. Using genetic and pharmacological perturbations we were able to alter this acceleration of replication and conclude that DNA damage inflicted by the process of transcription limits the speed of replication during early S phase. In late S phase, during which less-transcribed regions replicate, replication accelerates and approaches its maximum speed. This work describes scEdU-seq for studying replication fork speed in single cells, which enables researchers to investigate variability in replication speed along S phase and its associations with transcription and DNA damage.},
  archive      = {J_NMETH},
  author       = {van den Berg, Jeroen and van Batenburg, Vincent and Geisenberger, Christoph and Tjeerdsma, Rinskje B. and de Jaime-Soguero, Anchel and Acebrón, Sergio P. and van Vugt, Marcel A. T. M. and van Oudenaarden, Alexander},
  doi          = {10.1038/s41592-024-02308-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1175-1184},
  shortjournal = {Nat. Meth.},
  title        = {Quantifying DNA replication speeds in single cells by scEdU-seq},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Effects and avoidance of photoconversion-induced artifacts
in confocal and STED microscopy. <em>NMETH</em>, <em>21</em>(7),
1171–1174. (<a
href="https://doi.org/10.1038/s41592-024-02297-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence microscopy is limited by photoconversion due to continuous illumination, which results in not only photobleaching but also conversion of fluorescent molecules into species of different spectral properties through photoblueing. Here, we determined different fluorescence parameters of photoconverted products for various fluorophores under standard confocal and stimulated emission depletion (STED) microscopy conditions. We observed changes in both fluorescence spectra and lifetimes that can cause artifacts in quantitative measurements, which can be avoided by using exchangeable dyes. Photoconversion of popular organic dyes results in blue-shifted emission and altered fluorescence lifetimes that can cause artifacts in quantitative microscopy. These can be avoided through proper labeling strategies such as using exchangeable dyes.},
  archive      = {J_NMETH},
  author       = {Dasgupta, Anindita and Koerfer, Agnes and Kokot, Boštjan and Urbančič, Iztok and Eggeling, Christian and Carravilla, Pablo},
  doi          = {10.1038/s41592-024-02297-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1171-1174},
  shortjournal = {Nat. Meth.},
  title        = {Effects and avoidance of photoconversion-induced artifacts in confocal and STED microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The tidyomics ecosystem: Enhancing omic data analyses.
<em>NMETH</em>, <em>21</em>(7), 1166–1170. (<a
href="https://doi.org/10.1038/s41592-024-02299-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growth of omic data presents evolving challenges in data manipulation, analysis and integration. Addressing these challenges, Bioconductor provides an extensive community-driven biological data analysis platform. Meanwhile, tidy R programming offers a revolutionary data organization and manipulation standard. Here we present the tidyomics software ecosystem, bridging Bioconductor to the tidy R paradigm. This ecosystem aims to streamline omic analysis, ease learning and encourage cross-disciplinary collaborations. We demonstrate the effectiveness of tidyomics by analyzing 7.5 million peripheral blood mononuclear cells from the Human Cell Atlas, spanning six data frameworks and ten analysis tools. tidyomics offers a software ecosystem for omic data manipulation and analysis that bridges Bioconductor with the tidyverse framework.},
  archive      = {J_NMETH},
  author       = {Hutchison, William J. and Keyes, Timothy J. and Crowell, Helena L. and Serizay, Jacques and Soneson, Charlotte and Davis, Eric S. and Sato, Noriaki and Moses, Lambda and Tarlinton, Boyd and Nahid, Abdullah A. and Kosmac, Miha and Clayssen, Quentin and Yuan, Victor and Mu, Wancen and Park, Ji-Eun and Mamede, Izabela and Ryu, Min Hyung and Axisa, Pierre-Paul and Paiz, Paulina and Poon, Chi-Lam and Tang, Ming and Gottardo, Raphael and Morgan, Martin and Lee, Stuart and Lawrence, Michael and Hicks, Stephanie C. and Nolan, Garry P. and Davis, Kara L. and Papenfuss, Anthony T. and Love, Michael I. and Mangiola, Stefano},
  doi          = {10.1038/s41592-024-02299-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1166-1170},
  shortjournal = {Nat. Meth.},
  title        = {The tidyomics ecosystem: Enhancing omic data analyses},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep 3D histology powered by tissue clearing, omics and AI.
<em>NMETH</em>, <em>21</em>(7), 1153–1165. (<a
href="https://doi.org/10.1038/s41592-024-02327-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To comprehensively understand tissue and organism physiology and pathophysiology, it is essential to create complete three-dimensional (3D) cellular maps. These maps require structural data, such as the 3D configuration and positioning of tissues and cells, and molecular data on the constitution of each cell, spanning from the DNA sequence to protein expression. While single-cell transcriptomics is illuminating the cellular and molecular diversity across species and tissues, the 3D spatial context of these molecular data is often overlooked. Here, I discuss emerging 3D tissue histology techniques that add the missing third spatial dimension to biomedical research. Through innovations in tissue-clearing chemistry, labeling and volumetric imaging that enhance 3D reconstructions and their synergy with molecular techniques, these technologies will provide detailed blueprints of entire organs or organisms at the cellular level. Machine learning, especially deep learning, will be essential for extracting meaningful insights from the vast data. Further development of integrated structural, molecular and computational methods will unlock the full potential of next-generation 3D histology. This Perspective discusses the methods and tools required for three-dimensional histology in large samples, an approach that promises insights into tissue and organ physiology as well as disease.},
  archive      = {J_NMETH},
  author       = {Ertürk, Ali},
  doi          = {10.1038/s41592-024-02327-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1153-1165},
  shortjournal = {Nat. Meth.},
  title        = {Deep 3D histology powered by tissue clearing, omics and AI},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Simplifying deep learning to enhance accessibility of
large-scale 3D brain imaging analysis. <em>NMETH</em>, <em>21</em>(7),
1151–1152. (<a
href="https://doi.org/10.1038/s41592-024-02246-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created DELiVR, a deep-learning pipeline for 3D brain-cell mapping that is trained with virtual reality-generated reference annotations. It can be deployed via the user-friendly interface of the open-source software Fiji, which makes the analysis of large-scale 3D brain images widely accessible to scientists without computational expertise.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02246-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1151-1152},
  shortjournal = {Nat. Meth.},
  title        = {Simplifying deep learning to enhance accessibility of large-scale 3D brain imaging analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep learning method for the prediction of glycan structures
from mass spectrometry data. <em>NMETH</em>, <em>21</em>(7), 1149–1150.
(<a href="https://doi.org/10.1038/s41592-024-02315-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The annotation of structural glycomics data is a bottleneck in gaining insight into complex carbohydrates. An artificial intelligence model, CandyCrunch, has now been developed that accelerates the annotation process by orders of magnitude and achieves high accuracy in the prediction of glycan structures from tandem mass spectrometry data.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02315-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1149-1150},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning method for the prediction of glycan structures from mass spectrometry data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024k). How’s your health, forests? <em>NMETH</em>, <em>21</em>(7),
1145–1148. (<a
href="https://doi.org/10.1038/s41592-024-02337-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assessing forest health and resilience is an urgent, many-methods task that takes global collaboration, data collected at multiple scales and a willingness to be surprised.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02337-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1145-1148},
  shortjournal = {Nat. Meth.},
  title        = {How’s your health, forests?},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Capturing nascent transcripts in single cells.
<em>NMETH</em>, <em>21</em>(7), 1144. (<a
href="https://doi.org/10.1038/s41592-024-02356-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02356-w},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1144},
  shortjournal = {Nat. Meth.},
  title        = {Capturing nascent transcripts in single cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Multiplexed base editing. <em>NMETH</em>, <em>21</em>(7),
1144. (<a href="https://doi.org/10.1038/s41592-024-02357-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02357-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1144},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed base editing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Neurotransmitters at a glance. <em>NMETH</em>,
<em>21</em>(7), 1143. (<a
href="https://doi.org/10.1038/s41592-024-02355-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning approaches can distinguish six different classes of presynapses from electron micrographs across the Drosophila brain.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02355-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1143},
  shortjournal = {Nat. Meth.},
  title        = {Neurotransmitters at a glance},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Realization of cellomics to dive into the whole-body or
whole-organ cell cloud. <em>NMETH</em>, <em>21</em>(7), 1138–1142. (<a
href="https://doi.org/10.1038/s41592-024-02307-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissues, organs and organ systems are composed of interacting cells (the cellome). We discuss the emergence of an omics approach that we refer to as cellomics. It enables cellome-wide analysis in whole-organ or whole-body specimens, based on advanced three-dimensional imaging and image analysis technology. We think that cellomics will pave the way for the incorporation of cellular, intercellular and spatial information across millions of cells in our body.},
  archive      = {J_NMETH},
  author       = {Mitani, Tomoki T. and Susaki, Etsuo A. and Matsumoto, Katsuhiko and Ueda, Hiroki R.},
  doi          = {10.1038/s41592-024-02307-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1138-1142},
  shortjournal = {Nat. Meth.},
  title        = {Realization of cellomics to dive into the whole-body or whole-organ cell cloud},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bats. <em>NMETH</em>, <em>21</em>(7), 1135–1137. (<a
href="https://doi.org/10.1038/s41592-024-02330-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bats, the only flying mammals, comprise almost 25% of mammalian species. They are excellent navigators, highly social, and extremely long-lived. Their sense of echolocation has been studied for many years — but many species possess also excellent vision and olfaction. In recent years, bats have emerged as new models for neurobiology of navigation, social neuroscience, aging, and immunity.},
  archive      = {J_NMETH},
  author       = {Las, Liora and Ulanovsky, Nachum},
  doi          = {10.1038/s41592-024-02330-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1135-1137},
  shortjournal = {Nat. Meth.},
  title        = {Bats},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Tissue histology in 3D. <em>NMETH</em>, <em>21</em>(7),
1133. (<a href="https://doi.org/10.1038/s41592-024-02361-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissues and organs are inherently three-dimensional. Studies to understand their function and dysfunction should therefore aim to maintain the 3D spatial context.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02361-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1133},
  shortjournal = {Nat. Meth.},
  title        = {Tissue histology in 3D},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024u). Summer school in wartime. <em>NMETH</em>, <em>21</em>(7),
1134. (<a href="https://doi.org/10.1038/s41592-024-02329-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amid the brutality of war around them, some Ukrainian scientists find calm in bioinformatics.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02329-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {7},
  pages        = {1134},
  shortjournal = {Nat. Meth.},
  title        = {Summer school in wartime},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: “Ceci n’est pas un embryon?” The ethics
of human embryo model research. <em>NMETH</em>, <em>21</em>(6), 1132.
(<a href="https://doi.org/10.1038/s41592-024-02306-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {de Graeff, Nienke and De Proost, Lien and Munsie, Megan},
  doi          = {10.1038/s41592-024-02306-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1132},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: ‘Ceci n’est pas un embryon?’ the ethics of human embryo model research},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Brainlife.io: A decentralized and
open-source cloud platform to support neuroscience research.
<em>NMETH</em>, <em>21</em>(6), 1131. (<a
href="https://doi.org/10.1038/s41592-024-02296-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hayashi, Soichi and Caron, Bradley A. and Heinsfeld, Anibal Sólon and Vinci-Booher, Sophia and McPherson, Brent and Bullock, Daniel N. and Bertò, Giulia and Niso, Guiomar and Hanekamp, Sandra and Levitas, Daniel and Ray, Kimberly and MacKenzie, Anne and Avesani, Paolo and Kitchell, Lindsey and Leong, Josiah K. and Nascimento-Silva, Filipi and Koudoro, Serge and Willis, Hanna and Jolly, Jasleen K. and Pisner, Derek and Zuidema, Taylor R. and Kurzawski, Jan W. and Mikellidou, Kyriaki and Bussalb, Aurore and Chaumon, Maximilien and George, Nathalie and Rorden, Christopher and Victory, Conner and Bhatia, Dheeraj and Aydogan, Dogu Baran and Yeh, Fang-Cheng F. and Delogu, Franco and Guaje, Javier and Veraart, Jelle and Fischer, Jeremy and Faskowitz, Joshua and Fabrega, Ricardo and Hunt, David and McKee, Shawn and Brown, Shawn T. and Heyman, Stephanie and Iacovella, Vittorio and Mejia, Amanda F. and Marinazzo, Daniele and Craddock, R. Cameron and Olivetti, Emanuale and Hanson, Jamie L. and Garyfallidis, Eleftherios and Stanzione, Dan and Carson, James and Henschel, Robert and Hancock, David Y. and Stewart, Craig A. and Schnyer, David and Eke, Damian O. and Poldrack, Russell A. and Bollmann, Steffen and Stewart, Ashley and Bridge, Holly and Sani, Ilaria and Freiwald, Winrich A. and Puce, Aina and Port, Nicholas L. and Pestilli, Franco},
  doi          = {10.1038/s41592-024-02296-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1131},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: brainlife.io: a decentralized and open-source cloud platform to support neuroscience research},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Detailed mapping of the complex fiber structure and white
matter pathways of the chimpanzee brain. <em>NMETH</em>, <em>21</em>(6),
1122–1130. (<a
href="https://doi.org/10.1038/s41592-024-02270-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-standing questions about human brain evolution may only be resolved through comparisons with close living evolutionary relatives, such as chimpanzees. This applies in particular to structural white matter (WM) connectivity, which continuously expanded throughout evolution. However, due to legal restrictions on chimpanzee research, neuroscience research currently relies largely on data with limited detail or on comparisons with evolutionarily distant monkeys. Here, we present a detailed magnetic resonance imaging resource to study structural WM connectivity in the chimpanzee. This open-access resource contains (1) WM reconstructions of a postmortem chimpanzee brain, using the highest-quality diffusion magnetic resonance imaging data yet acquired from great apes; (2) an optimized and validated method for high-quality fiber orientation reconstructions; and (3) major fiber tract segmentations for cross-species morphological comparisons. This dataset enabled us to identify phylogenetically relevant details of the chimpanzee connectome, and we anticipate that it will substantially contribute to understanding human brain evolution. This resource presents a high-resolution diffusion MRI dataset of a chimpanzee brain, which should be useful for evolutionary studies of primate brain evolution.},
  archive      = {J_NMETH},
  author       = {Eichner, Cornelius and Paquette, Michael and Müller-Axt, Christa and Bock, Christian and Budinger, Eike and Gräßle, Tobias and Jäger, Carsten and Kirilina, Evgeniya and Lipp, Ilona and Morawski, Markus and Rusch, Henriette and Wenk, Patricia and Weiskopf, Nikolaus and Wittig, Roman M. and Crockford, Catherine and Friederici, Angela D. and Anwander, Alfred},
  doi          = {10.1038/s41592-024-02270-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1122-1130},
  shortjournal = {Nat. Meth.},
  title        = {Detailed mapping of the complex fiber structure and white matter pathways of the chimpanzee brain},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Three million images and morphological profiles of cells
treated with matched chemical and genetic perturbations. <em>NMETH</em>,
<em>21</em>(6), 1114–1121. (<a
href="https://doi.org/10.1038/s41592-024-02241-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of genetic and chemical perturbations with similar impacts on cell morphology can elucidate compounds’ mechanisms of action or novel regulators of genetic pathways. Research on methods for identifying such similarities has lagged due to a lack of carefully designed and well-annotated image sets of cells treated with chemical and genetic perturbations. Here we create such a Resource dataset, CPJUMP1, in which each perturbed gene’s product is a known target of at least two chemical compounds in the dataset. We systematically explore the directionality of correlations among perturbations that target the same protein encoded by a given gene, and we find that identifying matches between chemical and genetic perturbations is a challenging task. Our dataset and baseline analyses provide a benchmark for evaluating methods that measure perturbation similarities and impact, and more generally, learn effective representations of cellular state from microscopy images. Such advancements would accelerate the applications of image-based profiling of cellular states, such as uncovering drug mode of action or probing functional genomics. The CPJUMP1 Resource comprises Cell Painting images and profiles of 75 million cells treated with hundreds of chemical and genetic perturbations. The dataset enables exploration of their relationships and lays the foundation for the development of advanced methods to match perturbations.},
  archive      = {J_NMETH},
  author       = {Chandrasekaran, Srinivas Niranj and Cimini, Beth A. and Goodale, Amy and Miller, Lisa and Kost-Alimova, Maria and Jamali, Nasim and Doench, John G. and Fritchman, Briana and Skepner, Adam and Melanson, Michelle and Kalinin, Alexandr A. and Arevalo, John and Haghighi, Marzieh and Caicedo, Juan C. and Kuhn, Daniel and Hernandez, Desiree and Berstler, James and Shafqat-Abbasi, Hamdah and Root, David E. and Swalley, Susanne E. and Garg, Sakshi and Singh, Shantanu and Carpenter, Anne E.},
  doi          = {10.1038/s41592-024-02241-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1114-1121},
  shortjournal = {Nat. Meth.},
  title        = {Three million images and morphological profiles of cells treated with matched chemical and genetic perturbations},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The multimodality cell segmentation challenge: Toward
universal solutions. <em>NMETH</em>, <em>21</em>(6), 1103–1113. (<a
href="https://doi.org/10.1038/s41592-024-02233-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell segmentation is a critical step for quantitative single-cell analysis in microscopy images. Existing cell segmentation methods are often tailored to specific modalities or require manual interventions to specify hyper-parameters in different experimental settings. Here, we present a multimodality cell segmentation benchmark, comprising more than 1,500 labeled images derived from more than 50 diverse biological experiments. The top participants developed a Transformer-based deep-learning algorithm that not only exceeds existing methods but can also be applied to diverse microscopy images across imaging platforms and tissue types without manual parameter adjustments. This benchmark and the improved algorithm offer promising avenues for more accurate and versatile cell analysis in microscopy imaging. Cell segmentation is crucial in many image analysis pipelines. This analysis compares many tools on a multimodal cell segmentation benchmark. A Transformer-based model performed best in terms of performance and general applicability.},
  archive      = {J_NMETH},
  author       = {Ma, Jun and Xie, Ronald and Ayyadhury, Shamini and Ge, Cheng and Gupta, Anubha and Gupta, Ritu and Gu, Song and Zhang, Yao and Lee, Gihun and Kim, Joonkee and Lou, Wei and Li, Haofeng and Upschulte, Eric and Dickscheid, Timo and de Almeida, José Guilherme and Wang, Yixin and Han, Lin and Yang, Xin and Labagnara, Marco and Gligorovski, Vojislav and Scheder, Maxime and Rahi, Sahand Jamal and Kempster, Carly and Pollitt, Alice and Espinosa, Leon and Mignot, Tâm and Middeke, Jan Moritz and Eckardt, Jan-Niklas and Li, Wangkai and Li, Zhaoyang and Cai, Xiaochen and Bai, Bizhe and Greenwald, Noah F. and Van Valen, David and Weisbart, Erin and Cimini, Beth A. and Cheung, Trevor and Brück, Oscar and Bader, Gary D. and Wang, Bo},
  doi          = {10.1038/s41592-024-02233-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1103-1113},
  shortjournal = {Nat. Meth.},
  title        = {The multimodality cell segmentation challenge: Toward universal solutions},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Large-scale deep tissue voltage imaging with
targeted-illumination confocal microscopy. <em>NMETH</em>,
<em>21</em>(6), 1094–1102. (<a
href="https://doi.org/10.1038/s41592-024-02275-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Voltage imaging with cellular specificity has been made possible by advances in genetically encoded voltage indicators. However, the kilohertz rates required for voltage imaging lead to weak signals. Moreover, out-of-focus fluorescence and tissue scattering produce background that both undermines the signal-to-noise ratio and induces crosstalk between cells, making reliable in vivo imaging in densely labeled tissue highly challenging. We describe a microscope that combines the distinct advantages of targeted illumination and confocal gating while also maximizing signal detection efficiency. The resulting benefits in signal-to-noise ratio and crosstalk reduction are quantified experimentally and theoretically. Our microscope provides a versatile solution for enabling high-fidelity in vivo voltage imaging at large scales and penetration depths, which we demonstrate across a wide range of imaging conditions and different genetically encoded voltage indicator classes. In targeted-illumination confocal microscopy, the combination of targeted illumination with confocal detection allows one-photon voltage imaging across large fields of view with a high signal-to-noise ratio.},
  archive      = {J_NMETH},
  author       = {Xiao, Sheng and Cunningham, William J. and Kondabolu, Krishnakanth and Lowet, Eric and Moya, Maria V. and Mount, Rebecca A. and Ravasio, Cara and Bortz, Emma and Shaw, Dana and Economo, Michael N. and Han, Xue and Mertz, Jerome},
  doi          = {10.1038/s41592-024-02275-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1094-1102},
  shortjournal = {Nat. Meth.},
  title        = {Large-scale deep tissue voltage imaging with targeted-illumination confocal microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Universal inverse modeling of point spread functions for
SMLM localization and microscope characterization. <em>NMETH</em>,
<em>21</em>(6), 1082–1093. (<a
href="https://doi.org/10.1038/s41592-024-02282-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The point spread function (PSF) of a microscope describes the image of a point emitter. Knowing the accurate PSF model is essential for various imaging tasks, including single-molecule localization, aberration correction and deconvolution. Here we present universal inverse modeling of point spread functions (uiPSF), a toolbox to infer accurate PSF models from microscopy data, using either image stacks of fluorescent beads or directly images of blinking fluorophores, the raw data in single-molecule localization microscopy (SMLM). Our modular framework is applicable to a variety of microscope modalities and the PSF model incorporates system- or sample-specific characteristics, for example, the bead size, field- and depth- dependent aberrations, and transformations among channels. We demonstrate its application in single or multiple channels or large field-of-view SMLM systems, 4Pi-SMLM, and lattice light-sheet microscopes using either bead data or single-molecule blinking data. uiPSF (universal inverse modeling of point spread functions) offers a versatile solution for inferring accurate PSF models from images of beads and blinking fluorophores, enabling improved imaging and image processing in localization microscopy.},
  archive      = {J_NMETH},
  author       = {Liu, Sheng and Chen, Jianwei and Hellgoth, Jonas and Müller, Lucas-Raphael and Ferdman, Boris and Karras, Christian and Xiao, Dafei and Lidke, Keith A. and Heintzmann, Rainer and Shechtman, Yoav and Li, Yiming and Ries, Jonas},
  doi          = {10.1038/s41592-024-02282-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1082-1093},
  shortjournal = {Nat. Meth.},
  title        = {Universal inverse modeling of point spread functions for SMLM localization and microscope characterization},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Temporal analysis of relative distances (TARDIS) is a
robust, parameter-free alternative to single-particle tracking.
<em>NMETH</em>, <em>21</em>(6), 1074–1081. (<a
href="https://doi.org/10.1038/s41592-023-02149-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In single-particle tracking, individual particles are localized and tracked over time to probe their diffusion and molecular interactions. Temporal crossing of trajectories, blinking particles, and false-positive localizations present computational challenges that have remained difficult to overcome. Here we introduce a robust, parameter-free alternative to single-particle tracking: temporal analysis of relative distances (TARDIS). In TARDIS, an all-to-all distance analysis between localizations is performed with increasing temporal shifts. These pairwise distances represent either intraparticle distances originating from the same particle, or interparticle distances originating from unrelated particles, and are fitted analytically to obtain quantitative measures on particle dynamics. We showcase that TARDIS outperforms tracking algorithms, benchmarked on simulated and experimental data of varying complexity. We further show that TARDIS performs accurately in complex conditions characterized by high particle density, strong emitter blinking or false-positive localizations, and is in fact limited by the capabilities of localization algorithms. TARDIS’ robustness enables fivefold shorter measurements without loss of information. Temporal analysis of relative distances (TARDIS) is a conceptually new alternative to traditional single-particle tracking methods that overcomes challenges associated with high particle density, emitter blinking and spurious localizations.},
  archive      = {J_NMETH},
  author       = {Martens, Koen J. A. and Turkowyd, Bartosz and Hohlbein, Johannes and Endesfelder, Ulrike},
  doi          = {10.1038/s41592-023-02149-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1074-1081},
  shortjournal = {Nat. Meth.},
  title        = {Temporal analysis of relative distances (TARDIS) is a robust, parameter-free alternative to single-particle tracking},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Dissecting cell membrane tension dynamics and its effect on
piezo1-mediated cellular mechanosensitivity using force-controlled
nanopipettes. <em>NMETH</em>, <em>21</em>(6), 1063–1073. (<a
href="https://doi.org/10.1038/s41592-024-02277-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dynamics of cellular membrane tension and its role in mechanosensing, which is the ability of cells to respond to physical stimuli, remain incompletely understood, mainly due to the lack of appropriate tools. Here, we report a force-controlled nanopipette-based method that combines fluidic force microscopy with fluorescence imaging for precise manipulation of the cellular membrane tension while monitoring the impact on single-cell mechanosensitivity. The force-controlled nanopipette enables control of the indentation force imposed on the cell cortex as well as of the aspiration pressure applied to the plasma membrane. We show that this setup can be used to concurrently monitor the activation of Piezo1 mechanosensitive ion channels via calcium imaging. Moreover, the spatiotemporal behavior of the tension propagation is assessed with the fluorescent membrane tension probe Flipper-TR, and further dissected using molecular dynamics modeling. Finally, we demonstrate that aspiration and indentation act independently on the cellular mechanobiological machinery, that indentation induces a local pre-tension in the membrane, and that membrane tension stays confined by links to the cytoskeleton. FluidFM-based force-controlled nanopipettes enable control of mechanical stimuli for the investigation of Piezo1-induced mechanosensation in cell membranes.},
  archive      = {J_NMETH},
  author       = {Lüchtefeld, Ines and Pivkin, Igor V. and Gardini, Lucia and Zare-Eelanjegh, Elaheh and Gäbelein, Christoph and Ihle, Stephan J. and Reichmuth, Andreas M. and Capitanio, Marco and Martinac, Boris and Zambelli, Tomaso and Vassalli, Massimo},
  doi          = {10.1038/s41592-024-02277-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1063-1073},
  shortjournal = {Nat. Meth.},
  title        = {Dissecting cell membrane tension dynamics and its effect on piezo1-mediated cellular mechanosensitivity using force-controlled nanopipettes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial transition tensor of single cells. <em>NMETH</em>,
<em>21</em>(6), 1053–1062. (<a
href="https://doi.org/10.1038/s41592-024-02266-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics and messenger RNA splicing encode extensive spatiotemporal information for cell states and transitions. The current lineage-inference methods either lack spatial dynamics for state transition or cannot capture different dynamics associated with multiple cell states and transition paths. Here we present spatial transition tensor (STT), a method that uses messenger RNA splicing and spatial transcriptomes through a multiscale dynamical model to characterize multistability in space. By learning a four-dimensional transition tensor and spatial-constrained random walk, STT reconstructs cell-state-specific dynamics and spatial state transitions via both short-time local tensor streamlines between cells and long-time transition paths among attractors. Benchmarking and applications of STT on several transcriptome datasets via multiple technologies on epithelial–mesenchymal transitions, blood development, spatially resolved mouse brain and chicken heart development, indicate STT’s capability in recovering cell-state-specific dynamics and their associated genes not seen using existing methods. Overall, STT provides a consistent multiscale description of single-cell transcriptome data across multiple spatiotemporal scales. STT is a method that connects mRNA splicing and cell-state transitions across spatiotemporal scales and at single-cell resolution.},
  archive      = {J_NMETH},
  author       = {Zhou, Peijie and Bocci, Federico and Li, Tiejun and Nie, Qing},
  doi          = {10.1038/s41592-024-02266-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1053-1062},
  shortjournal = {Nat. Meth.},
  title        = {Spatial transition tensor of single cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Molecular pixelation: Spatial proteomics of single cells by
sequencing. <em>NMETH</em>, <em>21</em>(6), 1044–1052. (<a
href="https://doi.org/10.1038/s41592-024-02268-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The spatial distribution of cell surface proteins governs vital processes of the immune system such as intercellular communication and mobility. However, fluorescence microscopy has limited scalability in the multiplexing and throughput needed to drive spatial proteomics discoveries at subcellular level. We present Molecular Pixelation (MPX), an optics-free, DNA sequence-based method for spatial proteomics of single cells using antibody–oligonucleotide conjugates (AOCs) and DNA-based, nanometer-sized molecular pixels. The relative locations of AOCs are inferred by sequentially associating them into local neighborhoods using the sequence-unique DNA pixels, forming &amp;gt;1,000 spatially connected zones per cell in 3D. For each single cell, DNA-sequencing reads are computationally arranged into spatial proteomics networks for 76 proteins. By studying immune cell dynamics using spatial statistics on graph representations of the data, we identify known and new patterns of spatial organization of proteins on chemokine-stimulated T cells, highlighting the potential of MPX in defining cell states by the spatial arrangement of proteins. Molecular Pixelation is an optics-free method that uses DNA-tagged antibodies to enable identification of the relative location of proteins on single cells.},
  archive      = {J_NMETH},
  author       = {Karlsson, Filip and Kallas, Tomasz and Thiagarajan, Divya and Karlsson, Max and Schweitzer, Maud and Navarro, Jose Fernandez and Leijonancker, Louise and Geny, Sylvain and Pettersson, Erik and Rhomberg-Kauert, Jan and Larsson, Ludvig and van Ooijen, Hanna and Petkov, Stefan and González-Granillo, Marcela and Bunz, Jessica and Dahlberg, Johan and Simonetti, Michele and Sathe, Prajakta and Brodin, Petter and Barrio, Alvaro Martinez and Fredriksson, Simon},
  doi          = {10.1038/s41592-024-02268-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1044-1052},
  shortjournal = {Nat. Meth.},
  title        = {Molecular pixelation: Spatial proteomics of single cells by sequencing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Post-translational modification-centric base editor screens
to assess phosphorylation site functionality in high throughput.
<em>NMETH</em>, <em>21</em>(6), 1033–1043. (<a
href="https://doi.org/10.1038/s41592-024-02256-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Signaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here we describe the high-throughput, functional assessment of phosphorylation sites through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of PHLPP1, which promotes NFAT but inhibits NFκB activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways. Combining genome-wide CRISPR–Cas9-mediated base editors with temporally resolved phosphoproteomics enables the functional screening of thousands of post-translational modification sites involved in T cell activation.},
  archive      = {J_NMETH},
  author       = {Kennedy, Patrick H. and Alborzian Deh Sheikh, Amin and Balakar, Matthew and Jones, Alexander C. and Olive, Meagan E. and Hegde, Mudra and Matias, Maria I. and Pirete, Natan and Burt, Rajan and Levy, Jonathan and Little, Tamia and Hogan, Patrick G. and Liu, David R. and Doench, John G. and Newton, Alexandra C. and Gottschalk, Rachel A. and de Boer, Carl G. and Alarcón, Suzie and Newby, Gregory A. and Myers, Samuel A.},
  doi          = {10.1038/s41592-024-02256-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1033-1043},
  shortjournal = {Nat. Meth.},
  title        = {Post-translational modification-centric base editor screens to assess phosphorylation site functionality in high throughput},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Electrospray-assisted cryo-EM sample preparation to mitigate
interfacial effects. <em>NMETH</em>, <em>21</em>(6), 1023–1032. (<a
href="https://doi.org/10.1038/s41592-024-02247-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Addressing interfacial effects during specimen preparation in cryogenic electron microscopy remains challenging. Here we introduce ESI-cryoPrep, a specimen preparation method based on electrospray ionization in native mass spectrometry, designed to alleviate issues associated with protein denaturation or preferred orientation induced by macromolecule adsorption at interfaces. Through fine-tuning spraying parameters, we optimized protein integrity preservation and achieved the desired ice thickness for analyzing target macromolecules. With ESI-cryoPrep, we prepared high-quality cryo-specimens of five proteins and obtained three-dimensional reconstructions at near-atomic resolution. Our findings demonstrate that ESI-cryoPrep effectively confines macromolecules within the middle of the thin layer of amorphous ice, facilitating the preparation of blotting-free vitreous samples. The protective mechanism, characterized by the uneven distribution of charged biomolecules of varying sizes within charged droplets, prevents the adsorption of target biomolecules at air–water or graphene–water interfaces, thereby avoiding structural damage to the protein particles or the introduction of dominant orientation issues. ESI-cryoPrep is a cryo-EM specimen preparation method that employs electrospray ionization techniques to deposit charged macromolecule-containing droplets on EM grids. Demonstrated across various protein samples, this approach effectively prevents biomolecule adsorption at air–water or graphene–water interfaces, addressing challenges related to protein denaturation and preferred orientation.},
  archive      = {J_NMETH},
  author       = {Yang, Zi and Fan, Jingjin and Wang, Jia and Fan, Xiao and Ouyang, Zheng and Wang, Hong-Wei and Zhou, Xiaoyu},
  doi          = {10.1038/s41592-024-02247-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1023-1032},
  shortjournal = {Nat. Meth.},
  title        = {Electrospray-assisted cryo-EM sample preparation to mitigate interfacial effects},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Scalable and unbiased sequence-informed embedding of
single-cell ATAC-seq data with CellSpace. <em>NMETH</em>,
<em>21</em>(6), 1014–1022. (<a
href="https://doi.org/10.1038/s41592-024-02274-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Standard scATAC sequencing (scATAC-seq) analysis pipelines represent cells as sparse numeric vectors relative to an atlas of peaks or genomic tiles and consequently ignore genomic sequence information at accessible loci. Here we present CellSpace, an efficient and scalable sequence-informed embedding algorithm for scATAC-seq that learns a mapping of DNA k-mers and cells to the same space, to address this limitation. We show that CellSpace captures meaningful latent structure in scATAC-seq datasets, including cell subpopulations and developmental hierarchies, and can score transcription factor activities in single cells based on proximity to binding motifs embedded in the same space. Importantly, CellSpace implicitly mitigates batch effects arising from multiple samples, donors or assays, even when individual datasets are processed relative to different peak atlases. Thus, CellSpace provides a powerful tool for integrating and interpreting large-scale scATAC-seq compendia. By learning to embed DNA k-mers and cells into a joint space, CellSpace improves single-cell ATAC-seq analysis in multiple tasks such as latent structure discovery, transcription factor activity inference and batch effect mitigation.},
  archive      = {J_NMETH},
  author       = {Tayyebi, Zakieh and Pine, Allison R. and Leslie, Christina S.},
  doi          = {10.1038/s41592-024-02274-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1014-1022},
  shortjournal = {Nat. Meth.},
  title        = {Scalable and unbiased sequence-informed embedding of single-cell ATAC-seq data with CellSpace},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SIMBA: Single-cell embedding along with features.
<em>NMETH</em>, <em>21</em>(6), 1003–1013. (<a
href="https://doi.org/10.1038/s41592-023-01899-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most current single-cell analysis pipelines are limited to cell embeddings and rely heavily on clustering, while lacking the ability to explicitly model interactions between different feature types. Furthermore, these methods are tailored to specific tasks, as distinct single-cell problems are formulated differently. To address these shortcomings, here we present SIMBA, a graph embedding method that jointly embeds single cells and their defining features, such as genes, chromatin-accessible regions and DNA sequences, into a common latent space. By leveraging the co-embedding of cells and features, SIMBA allows for the study of cellular heterogeneity, clustering-free marker discovery, gene regulation inference, batch effect removal and omics data integration. We show that SIMBA provides a single framework that allows diverse single-cell problems to be formulated in a unified way and thus simplifies the development of new analyses and extension to new single-cell modalities. SIMBA is implemented as a comprehensive Python library ( https://simba-bio.readthedocs.io ). SIMBA learns a co-embedding space of single cells and multiple features such as genes, chromatin-accessible regions and transcription-factor-binding sequences, boosting the performances of various analyses of cellular diversity and regulation.},
  archive      = {J_NMETH},
  author       = {Chen, Huidong and Ryu, Jayoung and Vinyard, Michael E. and Lerer, Adam and Pinello, Luca},
  doi          = {10.1038/s41592-023-01899-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {1003-1013},
  shortjournal = {Nat. Meth.},
  title        = {SIMBA: Single-cell embedding along with features},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Indexing and searching petabase-scale nucleotide resources.
<em>NMETH</em>, <em>21</em>(6), 994–1002. (<a
href="https://doi.org/10.1038/s41592-024-02280-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Searching vast and rapidly growing nucleotide content in resources, such as runs in the Sequence Read Archive and assemblies for whole-genome shotgun sequencing projects in GenBank, is currently impractical for most researchers. Here we present Pebblescout, a tool that navigates such content by providing indexing and search capabilities. Indexing uses dense sampling of the sequences in the resource. Search finds subjects (runs or assemblies) that have short sequence matches to a user query, with well-defined guarantees and ranks them using informativeness of the matches. We illustrate the functionality of Pebblescout by creating eight databases that index over 3.7 petabases. The web service of Pebblescout can be reached at https://pebblescout.ncbi.nlm.nih.gov . We show that for a wide range of query lengths, Pebblescout provides a data-driven way for finding relevant subsets of large nucleotide resources, reducing the effort for downstream analysis substantially. We also show that Pebblescout results compare favorably to MetaGraph and Sourmash. The Pebblescout tool achieves an efficient search for subjects in a large nucleotide database such as runs in Sequence Read Archive data.},
  archive      = {J_NMETH},
  author       = {Shiryev, Sergey A. and Agarwala, Richa},
  doi          = {10.1038/s41592-024-02280-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {994-1002},
  shortjournal = {Nat. Meth.},
  title        = {Indexing and searching petabase-scale nucleotide resources},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multiplex profiling of developmental cis-regulatory elements
with quantitative single-cell expression reporters. <em>NMETH</em>,
<em>21</em>(6), 983–993. (<a
href="https://doi.org/10.1038/s41592-024-02260-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The inability to scalably and precisely measure the activity of developmental cis-regulatory elements (CREs) in multicellular systems is a bottleneck in genomics. Here we develop a dual RNA cassette that decouples the detection and quantification tasks inherent to multiplex single-cell reporter assays. The resulting measurement of reporter expression is accurate over multiple orders of magnitude, with a precision approaching the limit set by Poisson counting noise. Together with RNA barcode stabilization via circularization, these scalable single-cell quantitative expression reporters provide high-contrast readouts, analogous to classic in situ assays but entirely from sequencing. Screening &amp;gt;200 regions of accessible chromatin in a multicellular in vitro model of early mammalian development, we identify 13 (8 previously uncharacterized) autonomous and cell-type-specific developmental CREs. We further demonstrate that chimeric CRE pairs generate cognate two-cell-type activity profiles and assess gain- and loss-of-function multicellular expression phenotypes from CRE variants with perturbed transcription factor binding sites. Single-cell quantitative expression reporters can be applied in developmental and multicellular systems to quantitatively characterize native, perturbed and synthetic CREs at scale, with high sensitivity and at single-cell resolution. Single-cell quantitative expression reporters enable high-sensitivity quantitative characterization of cis-regulatory elements at the single-cell level in multicellular systems.},
  archive      = {J_NMETH},
  author       = {Lalanne, Jean-Benoît and Regalado, Samuel G. and Domcke, Silvia and Calderon, Diego and Martin, Beth K. and Li, Xiaoyi and Li, Tony and Suiter, Chase C. and Lee, Choli and Trapnell, Cole and Shendure, Jay},
  doi          = {10.1038/s41592-024-02260-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {983-993},
  shortjournal = {Nat. Meth.},
  title        = {Multiplex profiling of developmental cis-regulatory elements with quantitative single-cell expression reporters},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Simultaneous single-cell three-dimensional genome and gene
expression profiling uncovers dynamic enhancer connectivity underlying
olfactory receptor choice. <em>NMETH</em>, <em>21</em>(6), 974–982. (<a
href="https://doi.org/10.1038/s41592-024-02239-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The simultaneous measurement of three-dimensional (3D) genome structure and gene expression of individual cells is critical for understanding a genome’s structure–function relationship, yet this is challenging for existing methods. Here we present ‘Linking mRNA to Chromatin Architecture (LiMCA)’, which jointly profiles the 3D genome and transcriptome with exceptional sensitivity and from low-input materials. Combining LiMCA and our high-resolution scATAC-seq assay, METATAC, we successfully characterized chromatin accessibility, as well as paired 3D genome structures and gene expression information, of individual developing olfactory sensory neurons. We expanded the repertoire of known olfactory receptor (OR) enhancers and discovered unexpected rules of their dynamics: OR genes and their enhancers are most accessible during early differentiation. Furthermore, we revealed the dynamic spatial relationship between ORs and enhancers behind stepwise OR expression. These findings offer valuable insights into how 3D connectivity of ORs and enhancers dynamically orchestrate the ‘one neuron–one receptor’ selection process. LiMCA offers a tool for co-profiling 3D genome structure and gene expression at the single-cell level, enabling researchers to elucidate the olfactory receptor gene selection process.},
  archive      = {J_NMETH},
  author       = {Wu, Honggui and Zhang, Jiankun and Jian, Fanchong and Chen, Jinxin Phaedo and Zheng, Yinghui and Tan, Longzhi and Sunney Xie, X.},
  doi          = {10.1038/s41592-024-02239-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {974-982},
  shortjournal = {Nat. Meth.},
  title        = {Simultaneous single-cell three-dimensional genome and gene expression profiling uncovers dynamic enhancer connectivity underlying olfactory receptor choice},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Metabuli: Sensitive and specific metagenomic classification
via joint analysis of amino acid and DNA. <em>NMETH</em>,
<em>21</em>(6), 971–973. (<a
href="https://doi.org/10.1038/s41592-024-02273-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metagenomic taxonomic classifiers analyze either DNA or amino acid (AA) sequences. Metabuli ( https://metabuli.steineggerlab.com ), however, jointly analyzes both DNA and AA to leverage AA conservation for sensitive homology detection and DNA mutations for specific differentiation of closely related taxa. In the Critical Assessment of Metagenome Interpretation 2 plant-associated dataset, Metabuli covered 99% and 98% of classifications of state-of-the-art DNA- and AA-based classifiers, respectively. Metabuli enhances metagenomic read classification by jointly analyzing DNA and amino acid sequences for specificity and sensitivity, respectively.},
  archive      = {J_NMETH},
  author       = {Kim, Jaebeom and Steinegger, Martin},
  doi          = {10.1038/s41592-024-02273-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {971-973},
  shortjournal = {Nat. Meth.},
  title        = {Metabuli: Sensitive and specific metagenomic classification via joint analysis of amino acid and DNA},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Scalable telomere-to-telomere assembly for diploid and
polyploid genomes with double graph. <em>NMETH</em>, <em>21</em>(6),
967–970. (<a href="https://doi.org/10.1038/s41592-024-02269-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in long-read sequencing technologies, constructing a near telomere-to-telomere assembly is still computationally demanding. Here we present hifiasm (UL), an efficient de novo assembly algorithm combining multiple sequencing technologies to scale up population-wide near telomere-to-telomere assemblies. Applied to 22 human and two plant genomes, our algorithm produces better diploid assemblies at a cost of an order of magnitude lower than existing methods, and it also works with polyploid genomes. By effective and efficient integration of PacBio HiFi, Oxford Nanopore Technologies ultra-long and other sequencing data types, hifiasm (UL) enables telomere-to-telomere diploid and polyploid genome assembly at a population scale.},
  archive      = {J_NMETH},
  author       = {Cheng, Haoyu and Asri, Mobin and Lucas, Julian and Koren, Sergey and Li, Heng},
  doi          = {10.1038/s41592-024-02269-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {967-970},
  shortjournal = {Nat. Meth.},
  title        = {Scalable telomere-to-telomere assembly for diploid and polyploid genomes with double graph},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unveiling microbial diversity: Harnessing long-read
sequencing technology. <em>NMETH</em>, <em>21</em>(6), 954–966. (<a
href="https://doi.org/10.1038/s41592-024-02262-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing has recently transformed metagenomics, enhancing strain-level pathogen characterization, enabling accurate and complete metagenome-assembled genomes, and improving microbiome taxonomic classification and profiling. These advancements are not only due to improvements in sequencing accuracy, but also happening across rapidly changing analysis methods. In this Review, we explore long-read sequencing’s profound impact on metagenomics, focusing on computational pipelines for genome assembly, taxonomic characterization and variant detection, to summarize recent advancements in the field and provide an overview of available analytical methods to fully leverage long reads. We provide insights into the advantages and disadvantages of long reads over short reads and their evolution from the early days of long-read sequencing to their recent impact on metagenomics and clinical diagnostics. We further point out remaining challenges for the field such as the integration of methylation signals in sub-strain analysis and the lack of benchmarks. This Review summarizes the computational strategies for microbiome analysis using long-read sequencing data.},
  archive      = {J_NMETH},
  author       = {Agustinho, Daniel P. and Fu, Yilei and Menon, Vipin K. and Metcalf, Ginger A. and Treangen, Todd J. and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41592-024-02262-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {954-966},
  shortjournal = {Nat. Meth.},
  title        = {Unveiling microbial diversity: Harnessing long-read sequencing technology},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ultra-high-resolution diffusion MRI resource of chimpanzee
white matter connectivity. <em>NMETH</em>, <em>21</em>(6), 952–953. (<a
href="https://doi.org/10.1038/s41592-024-02271-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comparing brain connectivity between chimpanzees and humans is a means of understanding human cognition and evolution. To address the scarcity of chimpanzee neuroimaging data, we introduce a high-quality MRI resource that reveals previously unseen anatomical details, offering valuable insights into human brain evolution.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02271-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {952-953},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-resolution diffusion MRI resource of chimpanzee white matter connectivity},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Creating a universal cell segmentation algorithm.
<em>NMETH</em>, <em>21</em>(6), 950–951. (<a
href="https://doi.org/10.1038/s41592-024-02254-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell segmentation currently involves the use of various bespoke algorithms designed for specific cell types, tissues, staining methods and microscopy technologies. We present a universal algorithm that can segment all kinds of microscopy images and cell types across diverse imaging protocols.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02254-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {950-951},
  shortjournal = {Nat. Meth.},
  title        = {Creating a universal cell segmentation algorithm},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Targeted illumination confocal microscopy enables in vivo
voltage imaging in thick tissue. <em>NMETH</em>, <em>21</em>(6),
948–949. (<a href="https://doi.org/10.1038/s41592-024-02276-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fundamental mechanism for information processing in the brain is electrical activity. However, observing such activity at the single-cell level is challenging. We have developed an optical microscope that combines the advantages of targeted illumination and confocal gating to enable kilohertz-rate voltage imaging across large fields of view in thick tissue.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02276-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {948-949},
  shortjournal = {Nat. Meth.},
  title        = {Targeted illumination confocal microscopy enables in vivo voltage imaging in thick tissue},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Decoding microscopy images by accurate measurement of point
spread functions. <em>NMETH</em>, <em>21</em>(6), 946–947. (<a
href="https://doi.org/10.1038/s41592-024-02283-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {uiPSF is a toolbox to measure point spread functions based on inverse modeling that improves single-molecule localization microscopy (SMLM) localization and microscope characterization, and that works for many microscopy technologies.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02283-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {946-947},
  shortjournal = {Nat. Meth.},
  title        = {Decoding microscopy images by accurate measurement of point spread functions},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Studying cell membrane tension and mechanosensation during
mechanical stimulation. <em>NMETH</em>, <em>21</em>(6), 944–945. (<a
href="https://doi.org/10.1038/s41592-024-02278-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An experimental method to study how cells sense and react to external mechanical forces combines controlled mechanical stimulation using nanopipettes with fluorescence imaging of membrane tension. This approach facilitates the study of mechanosensitive ion channels and the propagation of cell membrane tension.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02278-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {944-945},
  shortjournal = {Nat. Meth.},
  title        = {Studying cell membrane tension and mechanosensation during mechanical stimulation},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A transition tensor for quantifying spatial transcriptome
attractors. <em>NMETH</em>, <em>21</em>(6), 942–943. (<a
href="https://doi.org/10.1038/s41592-024-02267-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics and mRNA splicing measurements encode rich spatiotemporal information for cell states and their transitions. We present a multiscale dynamical system method for reconstructing cell-state-specific dynamics and spatial state transitions. This theory-based approach reconciles short-timescale local tensor streamlines between cells with long-timescale transition paths that connect cell attractors.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02267-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {942-943},
  shortjournal = {Nat. Meth.},
  title        = {A transition tensor for quantifying spatial transcriptome attractors},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A genome-scale approach for determining the function of
phosphorylation sites. <em>NMETH</em>, <em>21</em>(6), 940–941. (<a
href="https://doi.org/10.1038/s41592-024-02264-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combining post-translational modification site-centric base editing with phenotypic screens uncovers the function of phosphorylation sites in high throughput, enabling the study of expansive signaling networks at a speed comparable to that of functional genomics.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02264-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {940-941},
  shortjournal = {Nat. Meth.},
  title        = {A genome-scale approach for determining the function of phosphorylation sites},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pebblescout is an easy-to-use tool for fast sequence search
in petabase-scale nucleotide resources. <em>NMETH</em>, <em>21</em>(6),
938–939. (<a href="https://doi.org/10.1038/s41592-024-02281-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pebblescout navigates vast, rapidly growing nucleotide content in resources by providing indexing and search capabilities. We used Pebblescout to index a metagenomic subset of Sequence Read Archive and seven other resources into databases spanning over 3.7 petabases and searchable interactively at a pilot website using queries as short as 42 bases.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02281-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {938-939},
  shortjournal = {Nat. Meth.},
  title        = {Pebblescout is an easy-to-use tool for fast sequence search in petabase-scale nucleotide resources},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantitative profiling of regulatory DNA activity at
single-cell resolution. <em>NMETH</em>, <em>21</em>(6), 936–937. (<a
href="https://doi.org/10.1038/s41592-024-02261-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a two-pronged strategy to functionally probe the enormous repertoire of noncoding DNA within genomes. Our approach markedly improved signal-to-noise ratio and successfully intersected single-cell genomics with reporter assays. The result delivers a multiplex and highly quantitative readout of regulatory sequences’ activity in dynamic and multicellular systems.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02261-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {936-937},
  shortjournal = {Nat. Meth.},
  title        = {Quantitative profiling of regulatory DNA activity at single-cell resolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). LiMCA: Hi-c gets an RNA twist. <em>NMETH</em>,
<em>21</em>(6), 934–935. (<a
href="https://doi.org/10.1038/s41592-024-02205-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A multiomics method measures both the cellular three-dimensional genome and transcriptome at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Kawaoka, Jane and Lomvardas, Stavros},
  doi          = {10.1038/s41592-024-02205-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {934-935},
  shortjournal = {Nat. Meth.},
  title        = {LiMCA: Hi-C gets an RNA twist},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024s). Seeing data as t-SNE and UMAP do. <em>NMETH</em>,
<em>21</em>(6), 930–933. (<a
href="https://doi.org/10.1038/s41592-024-02301-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dimension reduction helps to visualize high-dimensional datasets. These tools should be used thoughtfully and with tuned parameters. Sometimes, these methods take a second thought.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02301-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {930-933},
  shortjournal = {Nat. Meth.},
  title        = {Seeing data as t-SNE and UMAP do},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Mechanics of morphogenesis. <em>NMETH</em>, <em>21</em>(6),
929. (<a href="https://doi.org/10.1038/s41592-024-02312-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02312-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {929},
  shortjournal = {Nat. Meth.},
  title        = {Mechanics of morphogenesis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Linking enhancers to genes. <em>NMETH</em>, <em>21</em>(6),
929. (<a href="https://doi.org/10.1038/s41592-024-02313-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-024-02313-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {929},
  shortjournal = {Nat. Meth.},
  title        = {Linking enhancers to genes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Profiling single molecules in solution. <em>NMETH</em>,
<em>21</em>(6), 928. (<a
href="https://doi.org/10.1038/s41592-024-02311-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A label-free optical technique enables monitoring of the dynamic properties of individual biomolecules as they move freely in solution.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02311-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {928},
  shortjournal = {Nat. Meth.},
  title        = {Profiling single molecules in solution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DL4MicEverywhere: Deep learning for microscopy made
flexible, shareable and reproducible. <em>NMETH</em>, <em>21</em>(6),
925–927. (<a href="https://doi.org/10.1038/s41592-024-02295-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hidalgo-Cenalmor, Iván and Pylvänäinen, Joanna W. and G. Ferreira, Mariana and Russell, Craig T. and Saguy, Alon and Arganda-Carreras, Ignacio and Shechtman, Yoav and Jacquemet, Guillaume and Henriques, Ricardo and Gómez-de-Mariscal, Estibaliz},
  doi          = {10.1038/s41592-024-02295-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {925-927},
  shortjournal = {Nat. Meth.},
  title        = {DL4MicEverywhere: Deep learning for microscopy made flexible, shareable and reproducible},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). What’s in a method name? <em>NMETH</em>, <em>21</em>(6),
923. (<a href="https://doi.org/10.1038/s41592-024-02323-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide some advice about how to best name methods and software tools.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02323-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {923},
  shortjournal = {Nat. Meth.},
  title        = {What’s in a method name?},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024n). Integrating one’s LGBTQ+ identity into one’s science.
<em>NMETH</em>, <em>21</em>(6), 924. (<a
href="https://doi.org/10.1038/s41592-024-02300-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sometimes their queer identity is one that people set apart from their science identity. Others find unique ways to integrate multiple facets of their identity.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02300-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {6},
  pages        = {924},
  shortjournal = {Nat. Meth.},
  title        = {Integrating one’s LGBTQ+ identity into one’s science},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Pulsed stimulated brillouin microscopy
enables high-sensitivity mechanical imaging of live and fragile
biological specimens. <em>NMETH</em>, <em>21</em>(5), 922. (<a
href="https://doi.org/10.1038/s41592-024-02259-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Yang, Fan and Bevilacqua, Carlo and Hambura, Sebastian and Neves, Ana and Gopalan, Anusha and Watanabe, Koki and Govendir, Matt and Bernabeu, Maria and Ellenberg, Jan and Diz-Muñoz, Alba and Köhler, Simone and Rapti, Georgia and Jechlinger, Martin and Prevedel, Robert},
  doi          = {10.1038/s41592-024-02259-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {922},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Pulsed stimulated brillouin microscopy enables high-sensitivity mechanical imaging of live and fragile biological specimens},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spike sorting with kilosort4. <em>NMETH</em>,
<em>21</em>(5), 914–921. (<a
href="https://doi.org/10.1038/s41592-024-02232-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spike sorting is the computational process of extracting the firing times of single neurons from recordings of local electrical fields. This is an important but hard problem in neuroscience, made complicated by the nonstationarity of the recordings and the dense overlap in electrical fields between nearby neurons. To address the spike-sorting problem, we have been openly developing the Kilosort framework. Here we describe the various algorithmic steps introduced in different versions of Kilosort. We also report the development of Kilosort4, a version with substantially improved performance due to clustering algorithms inspired by graph-based approaches. To test the performance of Kilosort, we developed a realistic simulation framework that uses densely sampled electrical fields from real experiments to generate nonstationary spike waveforms and realistic noise. We found that nearly all versions of Kilosort outperformed other algorithms on a variety of simulated conditions and that Kilosort4 performed best in all cases, correctly identifying even neurons with low amplitudes and small spatial extents in high drift conditions. Kilosort4 is a spike-sorting algorithm with improved performance compared to previous versions, owing to the use of a graph-based clustering approach. The tool extracts the activity of individual neurons from electrophysiological recordings acquired with, for example, Neuropixels electrodes.},
  archive      = {J_NMETH},
  author       = {Pachitariu, Marius and Sridhar, Shashwat and Pennington, Jacob and Stringer, Carsen},
  doi          = {10.1038/s41592-024-02232-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {914-921},
  shortjournal = {Nat. Meth.},
  title        = {Spike sorting with kilosort4},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). RoboEM: Automated 3D flight tracing for synaptic-resolution
connectomics. <em>NMETH</em>, <em>21</em>(5), 908–913. (<a
href="https://doi.org/10.1038/s41592-024-02226-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping neuronal networks from three-dimensional electron microscopy (3D-EM) data still poses substantial reconstruction challenges, in particular for thin axons. Currently available automated image segmentation methods require manual proofreading for many types of connectomic analysis. Here we introduce RoboEM, an artificial intelligence-based self-steering 3D ‘flight’ system trained to navigate along neurites using only 3D-EM data as input. Applied to 3D-EM data from mouse and human cortex, RoboEM substantially improves automated state-of-the-art segmentations and can replace manual proofreading for more complex connectomic analysis problems, yielding computational annotation cost for cortical connectomes about 400-fold lower than the cost of manual error correction. RoboEM enables automated proofreading of electron microscopy datasets using a strategy akin to that of self-steering cars. This decreases the need for manual proofreading of segmented datasets and facilitates connectomic analyses.},
  archive      = {J_NMETH},
  author       = {Schmidt, Martin and Motta, Alessandro and Sievers, Meike and Helmstaedter, Moritz},
  doi          = {10.1038/s41592-024-02226-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {908-913},
  shortjournal = {Nat. Meth.},
  title        = {RoboEM: Automated 3D flight tracing for synaptic-resolution connectomics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A multicolor suite for deciphering population coding of
calcium and cAMP in vivo. <em>NMETH</em>, <em>21</em>(5), 897–907. (<a
href="https://doi.org/10.1038/s41592-024-02222-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {cAMP is a universal second messenger regulated by various upstream pathways including Ca2+ and G-protein-coupled receptors (GPCRs). To decipher in vivo cAMP dynamics, we rationally designed cAMPinG1, a sensitive genetically encoded green cAMP indicator that outperformed its predecessors in both dynamic range and cAMP affinity. Two-photon cAMPinG1 imaging detected cAMP transients in the somata and dendritic spines of neurons in the mouse visual cortex on the order of tens of seconds. In addition, multicolor imaging with a sensitive red Ca2+ indicator RCaMP3 allowed simultaneous measurement of population patterns in Ca2+ and cAMP in hundreds of neurons. We found Ca2+-related cAMP responses that represented specific information, such as direction selectivity in vision and locomotion, as well as GPCR-related cAMP responses. Overall, our multicolor suite will facilitate analysis of the interaction between the Ca2+, GPCR and cAMP signaling at single-cell resolution both in vitro and in vivo. Improved green cAMP and red calcium sensors were developed to facilitate dual-color imaging in vivo. These sensors will allow studying the relationship between calcium and cAMP signaling.},
  archive      = {J_NMETH},
  author       = {Yokoyama, Tatsushi and Manita, Satoshi and Uwamori, Hiroyuki and Tajiri, Mio and Imayoshi, Itaru and Yagishita, Sho and Murayama, Masanori and Kitamura, Kazuo and Sakamoto, Masayuki},
  doi          = {10.1038/s41592-024-02222-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {897-907},
  shortjournal = {Nat. Meth.},
  title        = {A multicolor suite for deciphering population coding of calcium and cAMP in vivo},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Selective-plane-activation structured illumination
microscopy. <em>NMETH</em>, <em>21</em>(5), 889–896. (<a
href="https://doi.org/10.1038/s41592-024-02236-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The background light from out-of-focus planes hinders resolution enhancement in structured illumination microscopy when observing volumetric samples. Here we used selective plane illumination and reversibly photoswitchable fluorescent proteins to realize structured illumination within the focal plane and eliminate the out-of-focus background. Theoretical investigation of the imaging properties and experimental demonstrations show that selective plane activation is beneficial for imaging dense microstructures in cells and cell spheroids. The combination of light sheet illumination and reversibly switchable fluorophores enables improved structured illumination microscopy for fast, low-background super-resolution imaging in cells and spheroids.},
  archive      = {J_NMETH},
  author       = {Temma, Kenta and Oketani, Ryosuke and Kubo, Toshiki and Bando, Kazuki and Maeda, Shunsuke and Sugiura, Kazunori and Matsuda, Tomoki and Heintzmann, Rainer and Kaminishi, Tatsuya and Fukuda, Koki and Hamasaki, Maho and Nagai, Takeharu and Fujita, Katsumasa},
  doi          = {10.1038/s41592-024-02236-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {889-896},
  shortjournal = {Nat. Meth.},
  title        = {Selective-plane-activation structured illumination microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Super-sectioning with multi-sheet reversible saturable
optical fluorescence transitions (RESOLFT) microscopy. <em>NMETH</em>,
<em>21</em>(5), 882–888. (<a
href="https://doi.org/10.1038/s41592-024-02196-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light-sheet fluorescence microscopy is an invaluable tool for four-dimensional biological imaging of multicellular systems due to the rapid volumetric imaging and minimal illumination dosage. However, it is challenging to retrieve fine subcellular information, especially in living cells, due to the width of the sheet of light (&amp;gt;1 μm). Here, using reversibly switchable fluorescent proteins (RSFPs) and a periodic light pattern for photoswitching, we demonstrate a super-resolution imaging method for rapid volumetric imaging of subcellular structures called multi-sheet RESOLFT. Multiple emission-sheets with a width that is far below the diffraction limit are created in parallel increasing recording speed (1–2 Hz) to provide super-sectioning ability (&amp;lt;100 nm). Our technology is compatible with various RSFPs due to its minimal requirement in the number of switching cycles and can be used to study a plethora of cellular structures. We track cellular processes such as cell division, actin motion and the dynamics of virus-like particles in three dimensions. Multi-sheet RESOLFT combines the speed and optical sectioning of light-sheet fluorescence microscopy with reversibly photoswitchable fluorescent proteins to enable fast, volumetric super-resolution imaging in live cells.},
  archive      = {J_NMETH},
  author       = {Bodén, Andreas and Ollech, Dirk and York, Andrew G. and Millett-Sikking, Alfred and Testa, Ilaria},
  doi          = {10.1038/s41592-024-02196-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {882-888},
  shortjournal = {Nat. Meth.},
  title        = {Super-sectioning with multi-sheet reversible saturable optical fluorescence transitions (RESOLFT) microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Generation of complex bone marrow organoids from human
induced pluripotent stem cells. <em>NMETH</em>, <em>21</em>(5), 868–881.
(<a href="https://doi.org/10.1038/s41592-024-02172-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human bone marrow (BM) niche sustains hematopoiesis throughout life. We present a method for generating complex BM-like organoids (BMOs) from human induced pluripotent stem cells (iPSCs). BMOs consist of key cell types that self-organize into spatially defined three-dimensional structures mimicking cellular, structural and molecular characteristics of the hematopoietic microenvironment. Functional properties of BMOs include the presence of an in vivo-like vascular network, the presence of multipotent mesenchymal stem/progenitor cells, the support of neutrophil differentiation and responsiveness to inflammatory stimuli. Single-cell RNA sequencing revealed a heterocellular composition including the presence of a hematopoietic stem/progenitor (HSPC) cluster expressing genes of fetal HSCs. BMO-derived HSPCs also exhibited lymphoid potential and a subset demonstrated transient engraftment potential upon xenotransplantation in mice. We show that the BMOs could enable the modeling of hematopoietic developmental aspects and inborn errors of hematopoiesis, as shown for human VPS45 deficiency. Thus, iPSC-derived BMOs serve as a physiologically relevant in vitro model of the human BM microenvironment to study hematopoietic development and BM diseases. The authors describe stem cell-derived bone marrow organoids that accurately model the structural and functional properties of the human bone marrow niche.},
  archive      = {J_NMETH},
  author       = {Frenz-Wiessner, Stephanie and Fairley, Savannah D. and Buser, Maximilian and Goek, Isabel and Salewskij, Kirill and Jonsson, Gustav and Illig, David and zu Putlitz, Benedicta and Petersheim, Daniel and Li, Yue and Chen, Pin-Hsuan and Kalauz, Martina and Conca, Raffaele and Sterr, Michael and Geuder, Johanna and Mizoguchi, Yoko and Megens, Remco T. A. and Linder, Monika I. and Kotlarz, Daniel and Rudelius, Martina and Penninger, Josef M. and Marr, Carsten and Klein, Christoph},
  doi          = {10.1038/s41592-024-02172-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {868-881},
  shortjournal = {Nat. Meth.},
  title        = {Generation of complex bone marrow organoids from human induced pluripotent stem cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multimodal probing of t-cell recognition with hexapod
heterostructures. <em>NMETH</em>, <em>21</em>(5), 857–867. (<a
href="https://doi.org/10.1038/s41592-023-02165-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies using antigen-presenting systems at the single-cell and ensemble levels can provide complementary insights into T-cell signaling and activation. Although crucial for advancing basic immunology and immunotherapy, there is a notable absence of synthetic material toolkits that examine T cells at both levels, and especially those capable of single-molecule-level manipulation. Here we devise a biomimetic antigen-presenting system (bAPS) for single-cell stimulation and ensemble modulation of T-cell recognition. Our bAPS uses hexapod heterostructures composed of a submicrometer cubic hematite core (α-Fe2O3) and nanostructured silica branches with diverse surface modifications. At single-molecule resolution, we show T-cell activation by a single agonist peptide-loaded major histocompatibility complex; distinct T-cell receptor (TCR) responses to structurally similar peptides that differ by only one amino acid; and the superior antigen recognition sensitivity of TCRs compared with that of chimeric antigen receptors (CARs). We also demonstrate how the magnetic field-induced rotation of hexapods amplifies the immune responses in suspended T and CAR-T cells. In addition, we establish our bAPS as a precise and scalable method for identifying stimulatory antigen-specific TCRs at the single-cell level. Thus, our multimodal bAPS represents a unique biointerface tool for investigating T-cell recognition, signaling and function. A biomimetic antigen-presenting system uses hexapod heterostructures to probe T-cell recognition at single-cell resolution.},
  archive      = {J_NMETH},
  author       = {Huang, Xiaodan and Meng, Lingyuan and Cao, Guoshuai and Prominski, Aleksander and Hu, Yifei and Yang, Chuanwang and Chen, Min and Shi, Jiuyun and Gallagher, Charles and Cao, Thao and Yue, Jiping and Huang, Jun and Tian, Bozhi},
  doi          = {10.1038/s41592-023-02165-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {857-867},
  shortjournal = {Nat. Meth.},
  title        = {Multimodal probing of T-cell recognition with hexapod heterostructures},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). De novo identification of CD4+ t cell epitopes.
<em>NMETH</em>, <em>21</em>(5), 846–856. (<a
href="https://doi.org/10.1038/s41592-024-02255-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CD4+ T cells recognize peptide antigens presented on class II major histocompatibility complex (MHC-II) molecules to carry out their function. The remarkable diversity of T cell receptor sequences and lack of antigen discovery approaches for MHC-II make profiling the specificities of CD4+ T cells challenging. We have expanded our platform of signaling and antigen-presenting bifunctional receptors to encode MHC-II molecules presenting covalently linked peptides (SABR-IIs) for CD4+ T cell antigen discovery. SABR-IIs can present epitopes to CD4+ T cells and induce signaling upon their recognition, allowing a readable output. Furthermore, the SABR-II design is modular in signaling and deployment to T cells and B cells. Here, we demonstrate that SABR-IIs libraries presenting endogenous and non-contiguous epitopes can be used for antigen discovery in the context of type 1 diabetes. SABR-II libraries provide a rapid, flexible, scalable and versatile approach for de novo identification of CD4+ T cell ligands from single-cell RNA sequencing data using experimental and computational approaches. SABR-IIs are engineered bifunctional receptors that enable CD4+ T cell antigen discovery.},
  archive      = {J_NMETH},
  author       = {Zdinak, Paul M. and Trivedi, Nishtha and Grebinoski, Stephanie and Torrey, Jessica and Martinez, Eduardo Zarate and Martinez, Salome and Hicks, Louise and Ranjan, Rashi and Makani, Venkata Krishna Kanth and Roland, Mary Melissa and Kublo, Lyubov and Arshad, Sanya and Anderson, Mark S. and Vignali, Dario A. A. and Joglekar, Alok V.},
  doi          = {10.1038/s41592-024-02255-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {846-856},
  shortjournal = {Nat. Meth.},
  title        = {De novo identification of CD4+ t cell epitopes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SLIDE: Significant latent factor interaction discovery and
exploration across biological domains. <em>NMETH</em>, <em>21</em>(5),
835–845. (<a href="https://doi.org/10.1038/s41592-024-02175-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern multiomic technologies can generate deep multiscale profiles. However, differences in data modalities, multicollinearity of the data, and large numbers of irrelevant features make analyses and integration of high-dimensional omic datasets challenging. Here we present Significant Latent Factor Interaction Discovery and Exploration (SLIDE), a first-in-class interpretable machine learning technique for identifying significant interacting latent factors underlying outcomes of interest from high-dimensional omic datasets. SLIDE makes no assumptions regarding data-generating mechanisms, comes with theoretical guarantees regarding identifiability of the latent factors/corresponding inference, and has rigorous false discovery rate control. Using SLIDE on single-cell and spatial omic datasets, we uncovered significant interacting latent factors underlying a range of molecular, cellular and organismal phenotypes. SLIDE outperforms/performs at least as well as a wide range of state-of-the-art approaches, including other latent factor approaches. More importantly, it provides biological inference beyond prediction that other methods do not afford. Thus, SLIDE is a versatile engine for biological discovery from modern multiomic datasets. SLIDE is a machine learning-based method for identification of significant latent factors across multiomics datasets.},
  archive      = {J_NMETH},
  author       = {Rahimikollu, Javad and Xiao, Hanxi and Rosengart, AnnaElaine and Rosen, Aaron B. I. and Tabib, Tracy and Zdinak, Paul M. and He, Kun and Bing, Xin and Bunea, Florentina and Wegkamp, Marten and Poholek, Amanda C. and Joglekar, Alok V. and Lafyatis, Robert A. and Das, Jishnu},
  doi          = {10.1038/s41592-024-02175-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {835-845},
  shortjournal = {Nat. Meth.},
  title        = {SLIDE: Significant latent factor interaction discovery and exploration across biological domains},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SciCSR infers b cell state transition and predicts
class-switch recombination dynamics using single-cell transcriptomic
data. <em>NMETH</em>, <em>21</em>(5), 823–834. (<a
href="https://doi.org/10.1038/s41592-023-02060-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Class-switch recombination (CSR) is an integral part of B cell maturation. Here we present sciCSR (pronounced ‘scissor’, single-cell inference of class-switch recombination), a computational pipeline that analyzes CSR events and dynamics of B cells from single-cell RNA sequencing (scRNA-seq) experiments. Validated on both simulated and real data, sciCSR re-analyzes scRNA-seq alignments to differentiate productive heavy-chain immunoglobulin transcripts from germline ‘sterile’ transcripts. From a snapshot of B cell scRNA-seq data, a Markov state model is built to infer the dynamics and direction of CSR. Applying sciCSR on severe acute respiratory syndrome coronavirus 2 vaccination time-course scRNA-seq data, we observe that sciCSR predicts, using data from an earlier time point in the collected time-course, the isotype distribution of B cell receptor repertoires of subsequent time points with high accuracy (cosine similarity ~0.9). Using processes specific to B cells, sciCSR identifies transitions that are often missed by conventional RNA velocity analyses and can reveal insights into the dynamics of B cell CSR during immune response. sciCSR is a computational workflow that leverages single-cell RNA sequencing data to predict B cell dynamics and class-switch recombination events.},
  archive      = {J_NMETH},
  author       = {Ng, Joseph C. F. and Montamat Garcia, Guillem and Stewart, Alexander T. and Blair, Paul and Mauri, Claudia and Dunn-Walters, Deborah K. and Fraternali, Franca},
  doi          = {10.1038/s41592-023-02060-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {823-834},
  shortjournal = {Nat. Meth.},
  title        = {SciCSR infers b cell state transition and predicts class-switch recombination dynamics using single-cell transcriptomic data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). ScGHOST: Identifying single-cell 3D genome subcompartments.
<em>NMETH</em>, <em>21</em>(5), 814–822. (<a
href="https://doi.org/10.1038/s41592-024-02230-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell Hi-C (scHi-C) technologies allow for probing of genome-wide cell-to-cell variability in three-dimensional (3D) genome organization from individual cells. Computational methods have been developed to reveal single-cell 3D genome features based on scHi-C, including A/B compartments, topologically associating domains and chromatin loops. However, no method exists for annotating single-cell subcompartments, which is important for understanding chromosome spatial localization in single cells. Here we present scGHOST, a single-cell subcompartment annotation method using graph embedding with constrained random walk sampling. Applications of scGHOST to scHi-C data and contact maps derived from single-cell 3D genome imaging demonstrate reliable identification of single-cell subcompartments, offering insights into cell-to-cell variability of nuclear subcompartments. Using scHi-C data from complex tissues, scGHOST identifies cell-type-specific or allele-specific subcompartments linked to gene transcription across various cell types and developmental stages, suggesting functional implications of single-cell subcompartments. scGHOST is an effective method for annotating single-cell 3D genome subcompartments in a broad range of biological contexts. scGHOST offers a computational tool to annotate single-cell subcompartments from scHi-C or imaging data through graph representation learning with constrained random walk sampling.},
  archive      = {J_NMETH},
  author       = {Xiong, Kyle and Zhang, Ruochi and Ma, Jian},
  doi          = {10.1038/s41592-024-02230-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {814-822},
  shortjournal = {Nat. Meth.},
  title        = {ScGHOST: Identifying single-cell 3D genome subcompartments},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Brainlife.io: A decentralized and open-source cloud
platform to support neuroscience research. <em>NMETH</em>,
<em>21</em>(5), 809–813. (<a
href="https://doi.org/10.1038/s41592-024-02237-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuroscience is advancing standardization and tool development to support rigor and transparency. Consequently, data pipeline complexity has increased, hindering FAIR (findable, accessible, interoperable and reusable) access. brainlife.io was developed to democratize neuroimaging research. The platform provides data standardization, management, visualization and processing and automatically tracks the provenance history of thousands of data objects. Here, brainlife.io is described and evaluated for validity, reliability, reproducibility, replicability and scientific utility using four data modalities and 3,200 participants. brainlife.io is a one-stop cloud platform for data management, visualization and analysis in human neuroscience. It is web-based and provides access to a variety of tools in a reproducible and reliable manner.},
  archive      = {J_NMETH},
  author       = {Hayashi, Soichi and Caron, Bradley A. and Heinsfeld, Anibal Sólon and Vinci-Booher, Sophia and McPherson, Brent and Bullock, Daniel N. and Bertò, Giulia and Niso, Guiomar and Hanekamp, Sandra and Levitas, Daniel and Ray, Kimberly and MacKenzie, Anne and Avesani, Paolo and Kitchell, Lindsey and Leong, Josiah K. and Nascimento-Silva, Filipi and Koudoro, Serge and Willis, Hanna and Jolly, Jasleen K. and Pisner, Derek and Zuidema, Taylor R. and Kurzawski, Jan W. and Mikellidou, Kyriaki and Bussalb, Aurore and Chaumon, Maximilien and George, Nathalie and Rorden, Christopher and Victory, Conner and Bhatia, Dheeraj and Aydogan, Dogu Baran and Yeh, Fang-Cheng F. and Delogu, Franco and Guaje, Javier and Veraart, Jelle and Fischer, Jeremy and Faskowitz, Joshua and Fabrega, Ricardo and Hunt, David and McKee, Shawn and Brown, Shawn T. and Heyman, Stephanie and Iacovella, Vittorio and Mejia, Amanda F. and Marinazzo, Daniele and Craddock, R. Cameron and Olivetti, Emanuale and Hanson, Jamie L. and Garyfallidis, Eleftherios and Stanzione, Dan and Carson, James and Henschel, Robert and Hancock, David Y. and Stewart, Craig A. and Schnyer, David and Eke, Damian O. and Poldrack, Russell A. and Bollmann, Steffen and Stewart, Ashley and Bridge, Holly and Sani, Ilaria and Freiwald, Winrich A. and Puce, Aina and Port, Nicholas L. and Pestilli, Franco},
  doi          = {10.1038/s41592-024-02237-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {809-813},
  shortjournal = {Nat. Meth.},
  title        = {Brainlife.io: A decentralized and open-source cloud platform to support neuroscience research},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Neurodesk: An accessible, flexible and portable data
analysis environment for reproducible neuroimaging. <em>NMETH</em>,
<em>21</em>(5), 804–808. (<a
href="https://doi.org/10.1038/s41592-023-02145-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuroimaging research requires purpose-built analysis software, which is challenging to install and may produce different results across computing environments. The community-oriented, open-source Neurodesk platform ( https://www.neurodesk.org/ ) harnesses a comprehensive and growing suite of neuroimaging software containers. Neurodesk includes a browser-accessible virtual desktop, command-line interface and computational notebook compatibility, allowing for accessible, flexible, portable and fully reproducible neuroimaging analysis on personal workstations, high-performance computers and the cloud. Neurodesk is a platform for analyzing human neuroimaging data, which provides numerous tools in a containerized form, thereby ensuring reproducibility and portability.},
  archive      = {J_NMETH},
  author       = {Renton, Angela I. and Dao, Thuy T. and Johnstone, Tom and Civier, Oren and Sullivan, Ryan P. and White, David J. and Lyons, Paris and Slade, Benjamin M. and Abbott, David F. and Amos, Toluwani J. and Bollmann, Saskia and Botting, Andy and Campbell, Megan E. J. and Chang, Jeryn and Close, Thomas G. and Dörig, Monika and Eckstein, Korbinian and Egan, Gary F. and Evas, Stefanie and Flandin, Guillaume and Garner, Kelly G. and Garrido, Marta I. and Ghosh, Satrajit S. and Grignard, Martin and Halchenko, Yaroslav O. and Hannan, Anthony J. and Heinsfeld, Anibal S. and Huber, Laurentius and Hughes, Matthew E. and Kaczmarzyk, Jakub R. and Kasper, Lars and Kuhlmann, Levin and Lou, Kexin and Mantilla-Ramos, Yorguin-Jose and Mattingley, Jason B. and Meier, Michael L. and Morris, Jo and Narayanan, Akshaiy and Pestilli, Franco and Puce, Aina and Ribeiro, Fernanda L. and Rogasch, Nigel C. and Rorden, Chris and Schira, Mark M. and Shaw, Thomas B. and Sowman, Paul F. and Spitz, Gershon and Stewart, Ashley W. and Ye, Xincheng and Zhu, Judy D. and Narayanan, Aswin and Bollmann, Steffen},
  doi          = {10.1038/s41592-023-02145-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {804-808},
  shortjournal = {Nat. Meth.},
  title        = {Neurodesk: An accessible, flexible and portable data analysis environment for reproducible neuroimaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Open-top multisample dual-view light-sheet microscope for
live imaging of large multicellular systems. <em>NMETH</em>,
<em>21</em>(5), 798–803. (<a
href="https://doi.org/10.1038/s41592-024-02213-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multicellular systems grow over the course of weeks from single cells to tissues or even full organisms, making live imaging challenging. To bridge spatiotemporal scales, we present an open-top dual-view and dual-illumination light-sheet microscope dedicated to live imaging of large specimens at single-cell resolution. The configuration of objectives together with a customizable multiwell mounting system combines dual view with high-throughput multiposition imaging. We use this microscope to image a wide variety of samples and highlight its capabilities to gain quantitative single-cell information in large specimens such as mature intestinal organoids and gastruloids. This work presents a highly versatile open-top, dual-view and dual-illumination light-sheet microscope for live imaging of large specimens.},
  archive      = {J_NMETH},
  author       = {Moos, Franziska and Suppinger, Simon and de Medeiros, Gustavo and Oost, Koen Cornelius and Boni, Andrea and Rémy, Camille and Weevers, Sera Lotte and Tsiairis, Charisios and Strnad, Petr and Liberali, Prisca},
  doi          = {10.1038/s41592-024-02213-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {798-803},
  shortjournal = {Nat. Meth.},
  title        = {Open-top multisample dual-view light-sheet microscope for live imaging of large multicellular systems},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SQANTI3: Curation of long-read transcriptomes for accurate
identification of known and novel isoforms. <em>NMETH</em>,
<em>21</em>(5), 793–797. (<a
href="https://doi.org/10.1038/s41592-024-02229-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SQANTI3 is a tool designed for the quality control, curation and annotation of long-read transcript models obtained with third-generation sequencing technologies. Leveraging its annotation framework, SQANTI3 calculates quality descriptors of transcript models, junctions and transcript ends. With this information, potential artifacts can be identified and replaced with reliable sequences. Furthermore, the integrated functional annotation feature enables subsequent functional iso-transcriptomics analyses. SQANTI3 offers a flexible tool for quality control, curation and annotation of long-read RNA sequencing data.},
  archive      = {J_NMETH},
  author       = {Pardo-Palacios, Francisco J. and Arzalluz-Luque, Angeles and Kondratova, Liudmyla and Salguero, Pedro and Mestre-Tomás, Jorge and Amorín, Rocío and Estevan-Morió, Eva and Liu, Tianyuan and Nanni, Adalena and McIntyre, Lauren and Tseng, Elizabeth and Conesa, Ana},
  doi          = {10.1038/s41592-024-02229-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {793-797},
  shortjournal = {Nat. Meth.},
  title        = {SQANTI3: Curation of long-read transcriptomes for accurate identification of known and novel isoforms},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell immune repertoire analysis. <em>NMETH</em>,
<em>21</em>(5), 777–792. (<a
href="https://doi.org/10.1038/s41592-024-02243-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell T cell and B cell antigen receptor-sequencing data analysis can potentially perform in-depth assessments of adaptive immune cells that inform on understanding immune cell development to tracking clonal expansion in disease and therapy. However, it has been extremely challenging to analyze and interpret T cells and B cells and their adaptive immune receptor repertoires at the single-cell level due to not only the complexity of the data but also the underlying biology. In this Review, we delve into the computational breakthroughs that have transformed the analysis of single-cell T cell and B cell antigen receptor-sequencing data. This Review provides an overview of bioinformatic approaches that enable immune repertoire analyses at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Irac, Sergio E. and Soon, Megan Sioe Fei and Borcherding, Nicholas and Tuong, Zewen Kelvin},
  doi          = {10.1038/s41592-024-02243-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {777-792},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell immune repertoire analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Can AlphaFold’s breakthrough in protein structure help
decode the fundamental principles of adaptive cellular immunity?
<em>NMETH</em>, <em>21</em>(5), 766–776. (<a
href="https://doi.org/10.1038/s41592-024-02240-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells are essential immune cells responsible for identifying and eliminating pathogens. Through interactions between their T-cell antigen receptors (TCRs) and antigens presented by major histocompatibility complex molecules (MHCs) or MHC-like molecules, T cells discriminate foreign and self peptides. Determining the fundamental principles that govern these interactions has important implications in numerous medical contexts. However, reconstructing a map between T cells and their antagonist antigens remains an open challenge for the field of immunology, and success of in silico reconstructions of this relationship has remained incremental. In this Perspective, we discuss the role that new state-of-the-art deep-learning models for predicting protein structure may play in resolving some of the unanswered questions the field faces linking TCR and peptide–MHC properties to T-cell specificity. We provide a comprehensive overview of structural databases and the evolution of predictive models, and highlight the breakthrough AlphaFold provided the field. This Perspective discusses the potential of protein structure-prediction models for exploring the structural landscape and specificity of TCR–pMHC interactions.},
  archive      = {J_NMETH},
  author       = {McMaster, Benjamin and Thorpe, Christopher and Ogg, Graham and Deane, Charlotte M. and Koohy, Hashem},
  doi          = {10.1038/s41592-024-02240-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {766-776},
  shortjournal = {Nat. Meth.},
  title        = {Can AlphaFold’s breakthrough in protein structure help decode the fundamental principles of adaptive cellular immunity?},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Building an automated three-dimensional flight agent for
neural network reconstruction. <em>NMETH</em>, <em>21</em>(5), 764–765.
(<a href="https://doi.org/10.1038/s41592-024-02227-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RoboEM, an artificial intelligence (AI)-based flight agent, automatically steers through three-dimensional electron microscopy (3D-EM) images of brain tissue to follow neurites. RoboEM substantially improves state-of-the-art automated reconstructions, eliminating manual proofreading needs in complex connectomic analysis problems and paving the way for high-throughput, cost-effective, large-scale mapping of neuronal networks — connectomes.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02227-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {764-765},
  shortjournal = {Nat. Meth.},
  title        = {Building an automated three-dimensional flight agent for neural network reconstruction},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Using stem cells to model the human bone marrow in a dish.
<em>NMETH</em>, <em>21</em>(5), 762–763. (<a
href="https://doi.org/10.1038/s41592-024-02173-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We established a method to generate complex self-organizing bone marrow-like organoids (BMOs) via concomitant differentiation of human induced pluripotent stem cells. These BMOs consist of hematopoietic cells, stromal niche cells and de novo vascular networks. In addition, they contain multipotent hematopoietic stem and progenitor cells, as well as mesenchymal stem and progenitor cells; they model aspects of the three-dimensional bone marrow architecture and can be used to study developmental and aberrant hematopoiesis.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02173-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {762-763},
  shortjournal = {Nat. Meth.},
  title        = {Using stem cells to model the human bone marrow in a dish},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A biomimetic robotic antigen-presenting system for sensitive
t cell recognition. <em>NMETH</em>, <em>21</em>(5), 760–761. (<a
href="https://doi.org/10.1038/s41592-023-02164-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce a biomimetic antigen-presenting system that uses hexapod heterostructures for specific T cell recognition at the single-molecule and single-cell levels. The system enables high-resolution T cell activation, uses magnetic forces to increase immune responses, and offers flexible and precise identification of antigen-specific T cell receptors, aiding the study of T cell recognition and immune cell mechanics.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02164-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {760-761},
  shortjournal = {Nat. Meth.},
  title        = {A biomimetic robotic antigen-presenting system for sensitive t cell recognition},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Uncovering hidden states driving biological outcomes using
machine learning. <em>NMETH</em>, <em>21</em>(5), 758–759. (<a
href="https://doi.org/10.1038/s41592-024-02176-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed Significant Latent Factor Interaction Discovery and Exploration (SLIDE), an interpretable machine learning approach that can infer hidden states (latent factors) underlying biological outcomes. These states capture the complex interplay between factors derived from multiscale, multiomic datasets across biological contexts and scales of resolution.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02176-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {758-759},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering hidden states driving biological outcomes using machine learning},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). “Snipping” the transcriptome unravels the dynamics of
antibody response. <em>NMETH</em>, <em>21</em>(5), 756–757. (<a
href="https://doi.org/10.1038/s41592-023-02061-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell inference of class-switch recombination (sciCSR) is a computational method that analyzes single-cell RNA sequencing data to deduce the temporal trajectory of how B cells develop antibody response.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02061-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {756-757},
  shortjournal = {Nat. Meth.},
  title        = {‘Snipping’ the transcriptome unravels the dynamics of antibody response},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accessible computing platforms democratize neuroimaging data
analysis. <em>NMETH</em>, <em>21</em>(5), 754–755. (<a
href="https://doi.org/10.1038/s41592-024-02238-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several research groups are making it easier for other neuroscientists to analyze large datasets by providing tools that can be accessed and used from anywhere in the world.},
  archive      = {J_NMETH},
  author       = {Uddin, Lucina Q.},
  doi          = {10.1038/s41592-024-02238-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {754-755},
  shortjournal = {Nat. Meth.},
  title        = {Accessible computing platforms democratize neuroimaging data analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024r). ScRNA-seq: Oh, the joys. <em>NMETH</em>, <em>21</em>(5),
750–753. (<a href="https://doi.org/10.1038/s41592-024-02263-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To those who seek transcriptomic information at high resolution, scale and throughput, single-cell RNA sequencing brings the data. Scientists share tips and future plans as they reflect on the method’s rise to stardom.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02263-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {750-753},
  shortjournal = {Nat. Meth.},
  title        = {ScRNA-seq: Oh, the joys},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Automated atomic modeling in cryo-EM maps. <em>NMETH</em>,
<em>21</em>(5), 749. (<a
href="https://doi.org/10.1038/s41592-024-02289-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02289-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {749},
  shortjournal = {Nat. Meth.},
  title        = {Automated atomic modeling in cryo-EM maps},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Probing single-molecule loading rates. <em>NMETH</em>,
<em>21</em>(5), 749. (<a
href="https://doi.org/10.1038/s41592-024-02290-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02290-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {749},
  shortjournal = {Nat. Meth.},
  title        = {Probing single-molecule loading rates},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Fast and multiplexed DNA-PAINT. <em>NMETH</em>,
<em>21</em>(5), 748. (<a
href="https://doi.org/10.1038/s41592-024-02287-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02287-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {748},
  shortjournal = {Nat. Meth.},
  title        = {Fast and multiplexed DNA-PAINT},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Augmented translation via multitailed mRNA. <em>NMETH</em>,
<em>21</em>(5), 748. (<a
href="https://doi.org/10.1038/s41592-024-02288-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02288-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {748},
  shortjournal = {Nat. Meth.},
  title        = {Augmented translation via multitailed mRNA},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Decoding immune kinetics: Unveiling secrets using
custom-built mathematical models. <em>NMETH</em>, <em>21</em>(5),
744–747. (<a href="https://doi.org/10.1038/s41592-024-02265-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Custom-built mathematical models make the immune response more predictable and offer mechanistic insights into fundamental immunology.},
  archive      = {J_NMETH},
  author       = {Smith, Amber M.},
  doi          = {10.1038/s41592-024-02265-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {744-747},
  shortjournal = {Nat. Meth.},
  title        = {Decoding immune kinetics: Unveiling secrets using custom-built mathematical models},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A practitioner’s view of spectral flow cytometry.
<em>NMETH</em>, <em>21</em>(5), 740–743. (<a
href="https://doi.org/10.1038/s41592-023-02042-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spectral flow cytometry enables the simultaneous analysis of a large number of cell surface markers at the single-cell level},
  archive      = {J_NMETH},
  author       = {Sharma, Siddhartha and Boyer, Josh and Teyton, Luc},
  doi          = {10.1038/s41592-023-02042-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {740-743},
  shortjournal = {Nat. Meth.},
  title        = {A practitioner’s view of spectral flow cytometry},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Investigating immunity. <em>NMETH</em>, <em>21</em>(5),
737–738. (<a href="https://doi.org/10.1038/s41592-024-02286-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent methods development in immunology has galvanized our understanding of immune responses.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02286-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {737-738},
  shortjournal = {Nat. Meth.},
  title        = {Investigating immunity},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024p). Quest: My postdoc home. <em>NMETH</em>, <em>21</em>(5),
739. (<a href="https://doi.org/10.1038/s41592-024-02253-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For a postdoctoral fellowship, it’s advisable to be selective about lab choice and to be clear about expectations.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02253-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {5},
  pages        = {739},
  shortjournal = {Nat. Meth.},
  title        = {Quest: My postdoc home},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: CheckM2: A rapid, scalable and accurate
tool for assessing microbial genome quality using machine learning.
<em>NMETH</em>, <em>21</em>(4), 735. (<a
href="https://doi.org/10.1038/s41592-024-02248-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Chklovski, Alex and Parks, Donovan H. and Woodcroft, Ben J. and Tyson, Gene W.},
  doi          = {10.1038/s41592-024-02248-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {735},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: CheckM2: a rapid, scalable and accurate tool for assessing microbial genome quality using machine learning},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multicenter integrated analysis of noncoding CRISPRi
screens. <em>NMETH</em>, <em>21</em>(4), 723–734. (<a
href="https://doi.org/10.1038/s41592-024-02216-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ENCODE Consortium’s efforts to annotate noncoding cis-regulatory elements (CREs) have advanced our understanding of gene regulatory landscapes. Pooled, noncoding CRISPR screens offer a systematic approach to investigate cis-regulatory mechanisms. The ENCODE4 Functional Characterization Centers conducted 108 screens in human cell lines, comprising &amp;gt;540,000 perturbations across 24.85 megabases of the genome. Using 332 functionally confirmed CRE–gene links in K562 cells, we established guidelines for screening endogenous noncoding elements with CRISPR interference (CRISPRi), including accurate detection of CREs that exhibit variable, often low, transcriptional effects. Benchmarking five screen analysis tools, we find that CASA produces the most conservative CRE calls and is robust to artifacts of low-specificity single guide RNAs. We uncover a subtle DNA strand bias for CRISPRi in transcribed regions with implications for screen design and analysis. Together, we provide an accessible data resource, predesigned single guide RNAs for targeting 3,275,697 ENCODE SCREEN candidate CREs with CRISPRi and screening guidelines to accelerate functional characterization of the noncoding genome. This analysis provides 108 noncoding CRISPR screens collated by the ENCODE4 consortium and establishes experimental guidelines for future CRISPRi screens characterizing functional cis-regulatory elements.},
  archive      = {J_NMETH},
  author       = {Yao, David and Tycko, Josh and Oh, Jin Woo and Bounds, Lexi R. and Gosai, Sager J. and Lataniotis, Lazaros and Mackay-Smith, Ava and Doughty, Benjamin R. and Gabdank, Idan and Schmidt, Henri and Guerrero-Altamirano, Tania and Siklenka, Keith and Guo, Katherine and White, Alexander D. and Youngworth, Ingrid and Andreeva, Kalina and Ren, Xingjie and Barrera, Alejandro and Luo, Yunhai and Yardımcı, Galip Gürkan and Tewhey, Ryan and Kundaje, Anshul and Greenleaf, William J. and Sabeti, Pardis C. and Leslie, Christina and Pritykin, Yuri and Moore, Jill E. and Beer, Michael A. and Gersbach, Charles A. and Reddy, Timothy E. and Shen, Yin and Engreitz, Jesse M. and Bassik, Michael C. and Reilly, Steven K.},
  doi          = {10.1038/s41592-024-02216-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {723-734},
  shortjournal = {Nat. Meth.},
  title        = {Multicenter integrated analysis of noncoding CRISPRi screens},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Benchmarking spatial clustering methods with spatially
resolved transcriptomics data. <em>NMETH</em>, <em>21</em>(4), 712–722.
(<a href="https://doi.org/10.1038/s41592-024-02215-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial clustering, which shares an analogy with single-cell clustering, has expanded the scope of tissue physiology studies from cell-centroid to structure-centroid with spatially resolved transcriptomics (SRT) data. Computational methods have undergone remarkable development in recent years, but a comprehensive benchmark study is still lacking. Here we present a benchmark study of 13 computational methods on 34 SRT data (7 datasets). The performance was evaluated on the basis of accuracy, spatial continuity, marker genes detection, scalability, and robustness. We found existing methods were complementary in terms of their performance and functionality, and we provide guidance for selecting appropriate methods for given scenarios. On testing additional 22 challenging datasets, we identified challenges in identifying noncontinuous spatial domains and limitations of existing methods, highlighting their inadequacies in handling recent large-scale tasks. Furthermore, with 145 simulated data, we examined the robustness of these methods against four different factors, and assessed the impact of pre- and postprocessing approaches. Our study offers a comprehensive evaluation of existing spatial clustering methods with SRT data, paving the way for future advancements in this rapidly evolving field. A benchmark study compares 13 spatial clustering methods on spatial transcriptomics data.},
  archive      = {J_NMETH},
  author       = {Yuan, Zhiyuan and Zhao, Fangyuan and Lin, Senlin and Zhao, Yu and Yao, Jianhua and Cui, Yan and Zhang, Xiao-Yong and Zhao, Yi},
  doi          = {10.1038/s41592-024-02215-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {712-722},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking spatial clustering methods with spatially resolved transcriptomics data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A-SOiD, an active-learning platform for expert-guided,
data-efficient discovery of behavior. <em>NMETH</em>, <em>21</em>(4),
703–711. (<a href="https://doi.org/10.1038/s41592-024-02200-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To identify and extract naturalistic behavior, two methods have become popular: supervised and unsupervised. Each approach carries its own strengths and weaknesses (for example, user bias, training cost, complexity and action discovery), which the user must consider in their decision. Here, an active-learning platform, A-SOiD, blends these strengths, and in doing so, overcomes several of their inherent drawbacks. A-SOiD iteratively learns user-defined groups with a fraction of the usual training data, while attaining expansive classification through directed unsupervised classification. In socially interacting mice, A-SOiD outperformed standard methods despite requiring 85% less training data. Additionally, it isolated ethologically distinct mouse interactions via unsupervised classification. We observed similar performance and efficiency using nonhuman primate and human three-dimensional pose data. In both cases, the transparency in A-SOiD’s cluster definitions revealed the defining features of the supervised classification through a game-theoretic approach. To facilitate use, A-SOiD comes as an intuitive, open-source interface for efficient segmentation of user-defined behaviors and discovered sub-actions. A-SOiD is a computational platform for behavioral annotation whose training includes elements of supervised and unsupervised learning. The approach is demonstrated on mouse, macaque and human datasets.},
  archive      = {J_NMETH},
  author       = {Tillmann, Jens F. and Hsu, Alexander I. and Schwarz, Martin K. and Yttri, Eric A.},
  doi          = {10.1038/s41592-024-02200-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {703-711},
  shortjournal = {Nat. Meth.},
  title        = {A-SOiD, an active-learning platform for expert-guided, data-efficient discovery of behavior},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improved green and red GRAB sensors for monitoring
spatiotemporal serotonin release in vivo. <em>NMETH</em>,
<em>21</em>(4), 692–702. (<a
href="https://doi.org/10.1038/s41592-024-02188-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The serotonergic system plays important roles in both physiological and pathological processes, and is a therapeutic target for many psychiatric disorders. Although several genetically encoded GFP-based serotonin (5-HT) sensors were recently developed, their sensitivities and spectral profiles are relatively limited. To overcome these limitations, we optimized green fluorescent G-protein-coupled receptor (GPCR)-activation-based 5-HT (GRAB5-HT) sensors and developed a red fluorescent GRAB5-HT sensor. These sensors exhibit excellent cell surface trafficking and high specificity, sensitivity and spatiotemporal resolution, making them suitable for monitoring 5-HT dynamics in vivo. Besides recording subcortical 5-HT release in freely moving mice, we observed both uniform and gradient 5-HT release in the mouse dorsal cortex with mesoscopic imaging. Finally, we performed dual-color imaging and observed seizure-induced waves of 5-HT release throughout the cortex following calcium and endocannabinoid waves. In summary, these 5-HT sensors can offer valuable insights regarding the serotonergic system in both health and disease. Deng et al. expand the toolbox of neurotransmitter sensors with high-sensitivity green and red genetically encoded serotonin sensors. These are suitable for in vivo applications, as demonstrated in a variety of applications in mice.},
  archive      = {J_NMETH},
  author       = {Deng, Fei and Wan, Jinxia and Li, Guochuan and Dong, Hui and Xia, Xiju and Wang, Yipan and Li, Xuelin and Zhuang, Chaowei and Zheng, Yu and Liu, Laixin and Yan, Yuqi and Feng, Jiesi and Zhao, Yulin and Xie, Hao and Li, Yulong},
  doi          = {10.1038/s41592-024-02188-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {692-702},
  shortjournal = {Nat. Meth.},
  title        = {Improved green and red GRAB sensors for monitoring spatiotemporal serotonin release in vivo},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improved green and red GRAB sensors for monitoring
dopaminergic activity in vivo. <em>NMETH</em>, <em>21</em>(4), 680–691.
(<a href="https://doi.org/10.1038/s41592-023-02100-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dopamine (DA) plays multiple roles in a wide range of physiological and pathological processes via a large network of dopaminergic projections. To dissect the spatiotemporal dynamics of DA release in both dense and sparsely innervated brain regions, we developed a series of green and red fluorescent G-protein-coupled receptor activation-based DA (GRABDA) sensors using a variety of DA receptor subtypes. These sensors have high sensitivity, selectivity and signal-to-noise ratio with subsecond response kinetics and the ability to detect a wide range of DA concentrations. We then used these sensors in mice to measure both optogenetically evoked and behaviorally relevant DA release while measuring neurochemical signaling in the nucleus accumbens, amygdala and cortex. Using these sensors, we also detected spatially resolved heterogeneous cortical DA release in mice performing various behaviors. These next-generation GRABDA sensors provide a robust set of tools for imaging dopaminergic activity under a variety of physiological and pathological conditions. Next-generation red and green G-protein-coupled receptor-based dopamine sensors with improved properties have been developed. Their performance is demonstrated in cell culture, in brain slices and in vivo in the mouse.},
  archive      = {J_NMETH},
  author       = {Zhuo, Yizhou and Luo, Bin and Yi, Xinyang and Dong, Hui and Miao, Xiaolei and Wan, Jinxia and Williams, John T. and Campbell, Malcolm G. and Cai, Ruyi and Qian, Tongrui and Li, Fengling and Weber, Sophia J. and Wang, Lei and Li, Bozhi and Wei, Yu and Li, Guochuan and Wang, Huan and Zheng, Yu and Zhao, Yulin and Wolf, Marina E. and Zhu, Yingjie and Watabe-Uchida, Mitsuko and Li, Yulong},
  doi          = {10.1038/s41592-023-02100-w},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {680-691},
  shortjournal = {Nat. Meth.},
  title        = {Improved green and red GRAB sensors for monitoring dopaminergic activity in vivo},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial landmark detection and tissue registration with deep
learning. <em>NMETH</em>, <em>21</em>(4), 673–679. (<a
href="https://doi.org/10.1038/s41592-024-02199-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial landmarks are crucial in describing histological features between samples or sites, tracking regions of interest in microscopy, and registering tissue samples within a common coordinate framework. Although other studies have explored unsupervised landmark detection, existing methods are not well-suited for histological image data as they often require a large number of images to converge, are unable to handle nonlinear deformations between tissue sections and are ineffective for z-stack alignment, other modalities beyond image data or multimodal data. We address these challenges by introducing effortless landmark detection, a new unsupervised landmark detection and registration method using neural-network-guided thin-plate splines. Our proposed method is evaluated on a diverse range of datasets including histology and spatially resolved transcriptomics, demonstrating superior performance in both accuracy and stability compared to existing approaches. Effortless landmark detection is an unsupervised deep learning-based approach that addresses key challenges in landmark detection and image registration for accurate performance across diverse tissue imaging datasets.},
  archive      = {J_NMETH},
  author       = {Ekvall, Markus and Bergenstråhle, Ludvig and Andersson, Alma and Czarnewski, Paulo and Olegård, Johannes and Käll, Lukas and Lundeberg, Joakim},
  doi          = {10.1038/s41592-024-02199-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {673-679},
  shortjournal = {Nat. Meth.},
  title        = {Spatial landmark detection and tissue registration with deep learning},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An optogenetic method for the controlled release of single
molecules. <em>NMETH</em>, <em>21</em>(4), 666–672. (<a
href="https://doi.org/10.1038/s41592-024-02204-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a system for optogenetic release of single molecules in cells. We confined soluble and transmembrane proteins to the Golgi apparatus via a photocleavable protein and released them by short pulses of light. Our method allows for a light dose-dependent delivery of functional proteins to the cytosol and plasma membrane in amounts compatible with single-molecule imaging, greatly simplifying access to single-molecule microscopy of any protein in live cells. We were able to reconstitute ion conductance by delivering BK and LRRC8/volume-regulated anion channels to the plasma membrane. Finally we were able to induce NF-kB signaling in T lymphoblasts stimulated by interleukin-1 by controlled release of a signaling protein that had been knocked out. We observed light-induced formation of functional inflammatory signaling complexes that triggered phosphorylation of the inhibitor of nuclear factor kappa-B kinase only in activated cells. We thus developed an optogenetic method for the reconstitution and investigation of cellular function at the single-molecule level. An optogenetic system enables the controlled release of soluble and transmembrane proteins for precise exploration of cellular protein function at the single-molecule level and streamlined single-molecule imaging.},
  archive      = {J_NMETH},
  author       = {Kashyap, Purba and Bertelli, Sara and Cao, Fakun and Kostritskaia, Yulia and Blank, Fenja and Srikanth, Niranjan A. and Schlack-Leigers, Claire and Saleppico, Roberto and Bierhuizen, Dolf and Lu, Xiaocen and Nickel, Walter and Campbell, Robert E. and Plested, Andrew J. R. and Stauber, Tobias and Taylor, Marcus J. and Ewers, Helge},
  doi          = {10.1038/s41592-024-02204-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {666-672},
  shortjournal = {Nat. Meth.},
  title        = {An optogenetic method for the controlled release of single molecules},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bright and stable monomeric green fluorescent protein
derived from StayGold. <em>NMETH</em>, <em>21</em>(4), 657–665. (<a
href="https://doi.org/10.1038/s41592-024-02203-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The high brightness and photostability of the green fluorescent protein StayGold make it a particularly attractive probe for long-term live-cell imaging; however, its dimeric nature precludes its application as a fluorescent tag for some proteins. Here, we report the development and crystal structures of a monomeric variant of StayGold, named mBaoJin, which preserves the beneficial properties of its precursor, while serving as a tag for structural proteins and membranes. Systematic benchmarking of mBaoJin against popular green fluorescent proteins and other recently introduced monomeric and pseudomonomeric derivatives of StayGold established mBaoJin as a bright and photostable fluorescent protein, exhibiting rapid maturation and high pH/chemical stability. mBaoJin was also demonstrated for super-resolution, long-term live-cell imaging and expansion microscopy. We further showed the applicability of mBaoJin for neuronal labeling in model organisms, including Caenorhabditis elegans and mice. mBaoJin is a monomeric derivative of the bright and photostable green fluorescent protein StayGold. mBaoJin offers favorable photophysical properties for use in diverse protein tagging and subcellular labeling applications.},
  archive      = {J_NMETH},
  author       = {Zhang, Hanbin and Lesnov, Gleb D. and Subach, Oksana M. and Zhang, Wenhao and Kuzmicheva, Tatyana P. and Vlaskina, Anna V. and Samygina, Valeriya R. and Chen, Liangyi and Ye, Xianxin and Nikolaeva, Alena Yu. and Gabdulkhakov, Azat and Papadaki, Stavrini and Qin, Wenming and Borshchevskiy, Valentin and Perfilov, Maxim M. and Gavrikov, Alexey S. and Drobizhev, Mikhail and Mishin, Alexander S. and Piatkevich, Kiryl D. and Subach, Fedor V.},
  doi          = {10.1038/s41592-024-02203-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {657-665},
  shortjournal = {Nat. Meth.},
  title        = {Bright and stable monomeric green fluorescent protein derived from StayGold},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). StayGold variants for molecular fusion and
membrane-targeting applications. <em>NMETH</em>, <em>21</em>(4),
648–656. (<a href="https://doi.org/10.1038/s41592-023-02085-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although StayGold is a bright and highly photostable fluorescent protein, its propensity for obligate dimer formation may hinder applications in molecular fusion and membrane targeting. To attain monovalent as well as bright and photostable labeling, we engineered tandem dimers of StayGold to promote dispersibility. On the basis of the crystal structure of this fluorescent protein, we disrupted the dimerization to generate a monomeric variant that offers improved photostability and brightness compared to StayGold. We applied the new monovalent StayGold tools to live-cell imaging experiments using spinning-disk laser-scanning confocal microscopy or structured illumination microscopy. We achieved cell-wide, high-spatiotemporal resolution and sustained imaging of dynamic subcellular events, including the targeting of endogenous condensin I to mitotic chromosomes, the movement of the Golgi apparatus and its membranous derivatives along microtubule networks, the distribution of cortical filamentous actin and the remolding of cristae membranes within mobile mitochondria. Monomeric and tandem dimer derivatives of the bright and photostable green fluorescent protein StayGold offer versatile tools for tagging target proteins and membranes in extended live-cell imaging.},
  archive      = {J_NMETH},
  author       = {Ando, Ryoko and Shimozono, Satoshi and Ago, Hideo and Takagi, Masatoshi and Sugiyama, Mayu and Kurokawa, Hiroshi and Hirano, Masahiko and Niino, Yusuke and Ueno, Go and Ishidate, Fumiyoshi and Fujiwara, Takahiro and Okada, Yasushi and Yamamoto, Masaki and Miyawaki, Atsushi},
  doi          = {10.1038/s41592-023-02085-6},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {648-656},
  shortjournal = {Nat. Meth.},
  title        = {StayGold variants for molecular fusion and membrane-targeting applications},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DIP-MS: Ultra-deep interaction proteomics for the
deconvolution of protein complexes. <em>NMETH</em>, <em>21</em>(4),
635–647. (<a href="https://doi.org/10.1038/s41592-024-02211-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most proteins are organized in macromolecular assemblies, which represent key functional units regulating and catalyzing most cellular processes. Affinity purification of the protein of interest combined with liquid chromatography coupled to tandem mass spectrometry (AP–MS) represents the method of choice to identify interacting proteins. The composition of complex isoforms concurrently present in the AP sample can, however, not be resolved from a single AP–MS experiment but requires computational inference from multiple time- and resource-intensive reciprocal AP–MS experiments. Here we introduce deep interactome profiling by mass spectrometry (DIP-MS), which combines AP with blue-native-PAGE separation, data-independent acquisition with mass spectrometry and deep-learning-based signal processing to resolve complex isoforms sharing the same bait protein in a single experiment. We applied DIP-MS to probe the organization of the human prefoldin family of complexes, resolving distinct prefoldin holo- and subcomplex variants, complex–complex interactions and complex isoforms with new subunits that were experimentally validated. Our results demonstrate that DIP-MS can reveal proteome modularity at unprecedented depth and resolution. Deep interactome profiling by mass spectrometry (DIP-MS) combines affinity purification with native BN-PAGE fractionation and mass spectrometry to resolve protein complexes sharing the same target protein. The paper also presents PPIprophet, a data-driven neural network-based protein complex deconvolution approach.},
  archive      = {J_NMETH},
  author       = {Frommelt, Fabian and Fossati, Andrea and Uliana, Federico and Wendt, Fabian and Xue, Peng and Heusel, Moritz and Wollscheid, Bernd and Aebersold, Ruedi and Ciuffa, Rodolfo and Gstaiger, Matthias},
  doi          = {10.1038/s41592-024-02211-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {635-647},
  shortjournal = {Nat. Meth.},
  title        = {DIP-MS: Ultra-deep interaction proteomics for the deconvolution of protein complexes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ScPROTEIN: A versatile deep graph contrastive learning
framework for single-cell proteomics embedding. <em>NMETH</em>,
<em>21</em>(4), 623–634. (<a
href="https://doi.org/10.1038/s41592-024-02214-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell proteomics sequencing technology sheds light on protein–protein interactions, posttranslational modifications and proteoform dynamics in the cell. However, the uncertainty estimation for peptide quantification, data missingness, batch effects and high noise hinder the analysis of single-cell proteomic data. It is important to solve this set of tangled problems together, but the existing methods tailored for single-cell transcriptomes cannot fully address this task. Here we propose a versatile framework designed for single-cell proteomics data analysis called scPROTEIN, which consists of peptide uncertainty estimation based on a multitask heteroscedastic regression model and cell embedding generation based on graph contrastive learning. scPROTEIN can estimate the uncertainty of peptide quantification, denoise protein data, remove batch effects and encode single-cell proteomic-specific embeddings in a unified framework. We demonstrate that scPROTEIN is efficient for cell clustering, batch correction, cell type annotation, clinical analysis and spatially resolved proteomic data exploration. scPROTEIN is a deep graph contrastive learning framework that can estimate the uncertainty of peptide quantification, denoise protein data, remove batch effects and encode single-cell proteomic-specific embeddings under a unified framework.},
  archive      = {J_NMETH},
  author       = {Li, Wei and Yang, Fan and Wang, Fang and Rong, Yu and Liu, Linjing and Wu, Bingzhe and Zhang, Han and Yao, Jianhua},
  doi          = {10.1038/s41592-024-02214-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {623-634},
  shortjournal = {Nat. Meth.},
  title        = {ScPROTEIN: A versatile deep graph contrastive learning framework for single-cell proteomics embedding},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ultralong transients enhance sensitivity and resolution in
orbitrap-based single-ion mass spectrometry. <em>NMETH</em>,
<em>21</em>(4), 619–622. (<a
href="https://doi.org/10.1038/s41592-024-02207-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Orbitrap-based charge detection mass spectrometry utilizes single-molecule sensitivity to enable mass analysis of even highly heterogeneous, high-mass macromolecular assemblies. For contemporary Orbitrap instruments, the accessible ion detection (recording) times are maximally ~1–2 s. Here by modifying a data acquisition method on an Orbitrap ultrahigh mass range mass spectrometer, we trapped and monitored individual (single) ions for up to 25 s, resulting in a corresponding and huge improvement in signal-to-noise ratio (×5 compared with 1 s), mass resolution (×25) and accuracy in charge and mass determination of Orbitrap-based charge detection mass spectrometry. Implementation of ultralong transients on an Orbitrap mass spectrometer improves mass resolution, sensitivity and accuracy of charge determination in the analysis of large macromolecular ions.},
  archive      = {J_NMETH},
  author       = {Deslignière, Evolène and Yin, Victor C. and Ebberink, Eduard H. T. M. and Rolland, Amber D. and Barendregt, Arjan and Wörner, Tobias P. and Nagornov, Konstantin O. and Kozhinov, Anton N. and Fort, Kyle L. and Tsybin, Yury O. and Makarov, Alexander A. and Heck, Albert J. R.},
  doi          = {10.1038/s41592-024-02207-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {619-622},
  shortjournal = {Nat. Meth.},
  title        = {Ultralong transients enhance sensitivity and resolution in orbitrap-based single-ion mass spectrometry},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Real-time detection of 20 amino acids and discrimination of
pathologically relevant peptides with functionalized nanopore.
<em>NMETH</em>, <em>21</em>(4), 609–618. (<a
href="https://doi.org/10.1038/s41592-024-02208-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precise identification and quantification of amino acids is crucial for many biological applications. Here we report a copper(II)-functionalized Mycobacterium smegmatis porin A (MspA) nanopore with the N91H substitution, which enables direct identification of all 20 proteinogenic amino acids when combined with a machine-learning algorithm. The validation accuracy reaches 99.1%, with 30.9% signal recovery. The feasibility of ultrasensitive quantification of amino acids was also demonstrated at the nanomolar range. Furthermore, the capability of this system for real-time analyses of two representative post-translational modifications (PTMs), one unnatural amino acid and ten synthetic peptides using exopeptidases, including clinically relevant peptides associated with Alzheimer’s disease and cancer neoantigens, was demonstrated. Notably, our strategy successfully distinguishes peptides with only one amino acid difference from the hydrolysate and provides the possibility to infer the peptide sequence. A copper(II)-functionalized Mycobacterium smegmatis porin A nanopore enables direct identification of all 20 proteinogenic amino acids, one unnatural amino acid and two post-translational modifications, and shows potential for peptide discrimination and sequencing.},
  archive      = {J_NMETH},
  author       = {Zhang, Ming and Tang, Chao and Wang, Zichun and Chen, Shanchuan and Zhang, Dan and Li, Kaiju and Sun, Ke and Zhao, Changjian and Wang, Yu and Xu, Mengying and Dai, Lunzhi and Lu, Guangwen and Shi, Hubing and Ren, Haiyan and Chen, Lu and Geng, Jia},
  doi          = {10.1038/s41592-024-02208-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {609-618},
  shortjournal = {Nat. Meth.},
  title        = {Real-time detection of 20 amino acids and discrimination of pathologically relevant peptides with functionalized nanopore},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A statistical method for quantifying progenitor cells
reveals incipient cell fate commitments. <em>NMETH</em>, <em>21</em>(4),
597–608. (<a href="https://doi.org/10.1038/s41592-024-02189-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantifying the number of progenitor cells that found an organ, tissue or cell population is of fundamental importance for understanding the development and homeostasis of a multicellular organism. Previous efforts rely on marker genes that are specifically expressed in progenitors. This strategy is, however, often hindered by the lack of ideal markers. Here we propose a general statistical method to quantify the progenitors of any tissues or cell populations in an organism, even in the absence of progenitor-specific markers, by exploring the cell phylogenetic tree that records the cell division history during development. The method, termed targeting coalescent analysis (TarCA), computes the probability that two randomly sampled cells of a tissue coalesce within the tissue-specific monophyletic clades. The inverse of this probability then serves as a measure of the progenitor number of the tissue. Both mathematic modeling and computer simulations demonstrated the high accuracy of TarCA, which was then validated using real data from nematode, fruit fly and mouse, all with related cell phylogenetic trees. We further showed that TarCA can be used to identify lineage-specific upregulated genes during embryogenesis, revealing incipient cell fate commitments in mouse embryos. Targeting coalescent analysis (TarCA) is a statistical method that quantifies the number of progenitor cells of a given population using single-cell phylogenetic data.},
  archive      = {J_NMETH},
  author       = {Deng, Shanjun and Gong, Han and Zhang, Di and Zhang, Mengdong and He, Xionglei},
  doi          = {10.1038/s41592-024-02189-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {597-608},
  shortjournal = {Nat. Meth.},
  title        = {A statistical method for quantifying progenitor cells reveals incipient cell fate commitments},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-scale pan-cancer interrogation of lncRNA dependencies
using CasRx. <em>NMETH</em>, <em>21</em>(4), 584–596. (<a
href="https://doi.org/10.1038/s41592-024-02190-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although long noncoding RNAs (lncRNAs) dominate the transcriptome, their functions are largely unexplored. The extensive overlap of lncRNAs with coding and regulatory sequences restricts their systematic interrogation by DNA-directed perturbation. Here we developed genome-scale lncRNA transcriptome screening using Cas13d/CasRx. We show that RNA targeting overcomes limitations inherent to other screening methods, thereby considerably expanding the explorable space of the lncRNAome. By evolving the screening system toward pan-cancer applicability, it supports molecular and phenotypic data integration to contextualize screening hits or infer lncRNA function. We thereby addressed challenges posed by the enormous transcriptome size and tissue specificity through a size-reduced multiplexed gRNA library termed Albarossa, targeting 24,171 lncRNA genes. Its rational design incorporates target prioritization based on expression, evolutionary conservation and tissue specificity, thereby reconciling high discovery power and pan-cancer representation with scalable experimental throughput. Applied across entities, the screening platform identified numerous context-specific and common essential lncRNAs. Our work sets the stage for systematic exploration of lncRNA biology in health and disease. A CasRx-based screening platform for genome-scale long noncoding RNA transcriptome perturbation is described.},
  archive      = {J_NMETH},
  author       = {Montero, Juan J. and Trozzo, Riccardo and Sugden, Maya and Öllinger, Rupert and Belka, Alexander and Zhigalova, Ekaterina and Waetzig, Paul and Engleitner, Thomas and Schmidt-Supprian, Marc and Saur, Dieter and Rad, Roland},
  doi          = {10.1038/s41592-024-02190-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {584-596},
  shortjournal = {Nat. Meth.},
  title        = {Genome-scale pan-cancer interrogation of lncRNA dependencies using CasRx},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Constructing telomere-to-telomere diploid genome by
polishing haploid nanopore-based assembly. <em>NMETH</em>,
<em>21</em>(4), 574–583. (<a
href="https://doi.org/10.1038/s41592-023-02141-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Draft genomes generated from Oxford Nanopore Technologies (ONT) long reads are known to have a higher error rate. Although existing genome polishers can enhance their quality, the error rate (including mismatches, indels and switching errors between paternal and maternal haplotypes) can be significant. Here, we develop two polishers, hypo-short and hypo-hybrid to address this issue. Hypo-short utilizes Illumina short reads to polish an ONT-based draft assembly, resulting in a high-quality assembly with low error rates and switching errors. Expanding on this, hypo-hybrid incorporates ONT long reads to further refine the assembly into a diploid representation. Leveraging on hypo-hybrid, we have created a diploid genome assembly pipeline called hypo-assembler. Hypo-assembler automates the generation of highly accurate, contiguous and nearly complete diploid assemblies using ONT long reads, Illumina short reads and optionally Hi-C reads. Notably, our solution even allows for the production of telomere-to-telomere diploid genomes with additional manual steps. As a proof of concept, we successfully assembled a fully phased telomere-to-telomere diploid genome of HG00733, achieving a quality value exceeding 50. This work introduces two polishers for refining the draft genome generated from nanopore long reads, as well as an assembler pipeline for producing telomere-to-telomere diploid genome with low error rate.},
  archive      = {J_NMETH},
  author       = {Darian, Joshua Casey and Kundu, Ritu and Rajaby, Ramesh and Sung, Wing-Kin},
  doi          = {10.1038/s41592-023-02141-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {574-583},
  shortjournal = {Nat. Meth.},
  title        = {Constructing telomere-to-telomere diploid genome by polishing haploid nanopore-based assembly},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MINSTED tracking of single biomolecules. <em>NMETH</em>,
<em>21</em>(4), 569–573. (<a
href="https://doi.org/10.1038/s41592-024-02209-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we show that MINSTED localization, a method whereby the position of a fluorophore is identified with precisely controlled beams of a STED microscope, tracks fluorophores and hence labeled biomolecules with nanometer/millisecond spatiotemporal precision. By updating the position for each detected photon, MINSTED recognizes fluorophore steps of 16 nm within &amp;lt;250 μs using about 13 photons. The power of MINSTED tracking is demonstrated by resolving the stepping of the motor protein kinesin-1 walking on microtubules and switching protofilaments. MINSTED quantifies tiny movements of individual biomolecules with high spatiotemporal precision to successfully resolve the steps of the molecular motor protein kinesin-1 labeled with a single fluorophore as it switches protofilaments.},
  archive      = {J_NMETH},
  author       = {Scheiderer, Lukas and von der Emde, Henrik and Hesselink, Mira and Weber, Michael and Hell, Stefan W.},
  doi          = {10.1038/s41592-024-02209-6},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {569-573},
  shortjournal = {Nat. Meth.},
  title        = {MINSTED tracking of single biomolecules},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Square condenser apertures for square cameras in low-dose
transmission electron microscopy. <em>NMETH</em>, <em>21</em>(4),
566–568. (<a href="https://doi.org/10.1038/s41592-024-02206-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In transmission electron microscopy (TEM), cameras are square or rectangular but beams are round so the circular lobes irradiate adjacent areas, precluding further neighboring acquisition for beam-sensitive samples. We present condenser aperture plates with square and rectangular shapes that improve the efficiency of area usage by 70% and enhance montage imaging for beam-sensitive specimens. We demonstrate the compatibility of these condenser aperture plates with high-resolution cryogenic TEM by reconstructing a 1.8-Å map of equine apo-ferritin. Square and rectangular beams made with new condenser apertures enable more efficient image capture in low-dose transmission electron microscopy with no compromise to image quality in single-particle cryoelectron microscopy.},
  archive      = {J_NMETH},
  author       = {Brown, Hamish G. and Smith, Dan and Wardle, Benjamin C. and Hanssen, Eric},
  doi          = {10.1038/s41592-024-02206-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {566-568},
  shortjournal = {Nat. Meth.},
  title        = {Square condenser apertures for square cameras in low-dose transmission electron microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Square beams for optimal tiling in transmission electron
microscopy. <em>NMETH</em>, <em>21</em>(4), 562–565. (<a
href="https://doi.org/10.1038/s41592-023-02161-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging large fields of view at a high magnification requires tiling. Transmission electron microscopes typically have round beam profiles; therefore, tiling across a large area is either imperfect or results in uneven exposures, a problem for dose-sensitive samples. Here, we introduce a square electron beam that can easily be retrofitted in existing microscopes, and demonstrate its application, showing that it can tile nearly perfectly and deliver cryo-electron microscopy imaging with a resolution comparable to conventional set-ups. A square electron beam improves imaging of large fields of view in transmission electron microscopes by facilitating montage tomography of vitrified specimens with no loss in data quality relative to conventional round beams.},
  archive      = {J_NMETH},
  author       = {Chua, Eugene Y. D. and Alink, Lambertus M. and Kopylov, Mykhailo and Johnston, Jake D. and Eisenstein, Fabian and de Marco, Alex},
  doi          = {10.1038/s41592-023-02161-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {562-565},
  shortjournal = {Nat. Meth.},
  title        = {Square beams for optimal tiling in transmission electron microscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Creating diploid assemblies from nanopore and illumina reads
with hypo-assembler. <em>NMETH</em>, <em>21</em>(4), 560–561. (<a
href="https://doi.org/10.1038/s41592-023-02142-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diploid assembly is a difficult task that requires several types of genomic sequencing data, including — but not limited to — HiFi reads and parental sequences. Hypo-assembler, an assembly algorithm, uses high quality solid k-mers extracted from Illumina data alongside Nanopore reads to produce a high-quality diploid assembly using only Nanopore and Illumina data.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02142-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {560-561},
  shortjournal = {Nat. Meth.},
  title        = {Creating diploid assemblies from nanopore and illumina reads with hypo-assembler},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Breaking up the StayGold dimer yields three photostable
monomers. <em>NMETH</em>, <em>21</em>(4), 558–559. (<a
href="https://doi.org/10.1038/s41592-024-02223-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The exceptionally photostable green fluorescent protein StayGold has been monomerized in different laboratories, which has generated three unique monomeric variants that will enable new imaging applications.},
  archive      = {J_NMETH},
  author       = {Goedhart, Joachim and Gadella, Theodorus W. J.},
  doi          = {10.1038/s41592-024-02223-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {558-559},
  shortjournal = {Nat. Meth.},
  title        = {Breaking up the StayGold dimer yields three photostable monomers},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unlocking cryo-EM’s multishot potential with square or
rectangular beams. <em>NMETH</em>, <em>21</em>(4), 555–557. (<a
href="https://doi.org/10.1038/s41592-024-02224-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New condenser aperture designs form square or rectangular beams that match the camera dimensions, which efficiently expands the data acquisition area in cryogenic electron microscopy.},
  archive      = {J_NMETH},
  author       = {Zhao, Xiaowei},
  doi          = {10.1038/s41592-024-02224-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {555-557},
  shortjournal = {Nat. Meth.},
  title        = {Unlocking cryo-EM’s multishot potential with square or rectangular beams},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Closing in on cancer heterogeneity with organoids.
<em>NMETH</em>, <em>21</em>(4), 551–554. (<a
href="https://doi.org/10.1038/s41592-024-02231-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoids are an emerging way to model the fraught complexity of tumors.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02231-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {551-554},
  shortjournal = {Nat. Meth.},
  title        = {Closing in on cancer heterogeneity with organoids},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Modeling locomotion from environment to neurons.
<em>NMETH</em>, <em>21</em>(4), 550. (<a
href="https://doi.org/10.1038/s41592-024-02251-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02251-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {550},
  shortjournal = {Nat. Meth.},
  title        = {Modeling locomotion from environment to neurons},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024m). Protein language models using convolutions. <em>NMETH</em>,
<em>21</em>(4), 550. (<a
href="https://doi.org/10.1038/s41592-024-02252-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-024-02252-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {550},
  shortjournal = {Nat. Meth.},
  title        = {Protein language models using convolutions},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Protein–ligand structure prediction. <em>NMETH</em>,
<em>21</em>(4), 549. (<a
href="https://doi.org/10.1038/s41592-024-02249-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02249-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {549},
  shortjournal = {Nat. Meth.},
  title        = {Protein–ligand structure prediction},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A map of mouse embryogenesis. <em>NMETH</em>,
<em>21</em>(4), 549. (<a
href="https://doi.org/10.1038/s41592-024-02250-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02250-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {549},
  shortjournal = {Nat. Meth.},
  title        = {A map of mouse embryogenesis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The placozoan trichoplax. <em>NMETH</em>, <em>21</em>(4),
543–545. (<a href="https://doi.org/10.1038/s41592-024-02228-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After having been largely neglected for more than a century, the tiny marine animal Trichoplax adhaerens is enjoying a revival and is now poised to become a mainstream model organism for the study of epithelial evolution, diversification and specialization.},
  archive      = {J_NMETH},
  author       = {Leria, Marvin and Requin, Magali and Le Bivic, André and Pasini, Andrea},
  doi          = {10.1038/s41592-024-02228-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {543-545},
  shortjournal = {Nat. Meth.},
  title        = {The placozoan trichoplax},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Peer review demystified: Part 2. <em>NMETH</em>,
<em>21</em>(4), 541. (<a
href="https://doi.org/10.1038/s41592-024-02258-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We continue our explanation of the peer review process at Nature Methods.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02258-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {541},
  shortjournal = {Nat. Meth.},
  title        = {Peer review demystified: Part 2},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Comparing classifier performance with baselines.
<em>NMETH</em>, <em>21</em>(4), 546–548. (<a
href="https://doi.org/10.1038/s41592-024-02234-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {All animals are equal, but some animals are more equal than others. —George Orwell},
  archive      = {J_NMETH},
  author       = {Megahed, Fadel M. and Chen, Ying-Ju and Jones-Farmer, L. Allison and Rigdon, Steven E. and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-024-02234-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {4},
  pages        = {546-548},
  shortjournal = {Nat. Meth.},
  title        = {Comparing classifier performance with baselines},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Found: A skilled, courageous postdoctoral fellow.
<em>NMETH</em>, <em>21</em>(4), 542. (<a
href="https://doi.org/10.1038/s41592-024-02217-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To select the best postdoctoral fellows for their labs, principal investigators assess scientific qualifications and more.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02217-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {4},
  pages        = {542},
  shortjournal = {Nat. Meth.},
  title        = {Found: A skilled, courageous postdoctoral fellow},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ScPerturb: Harmonized single-cell perturbation data.
<em>NMETH</em>, <em>21</em>(3), 531–540. (<a
href="https://doi.org/10.1038/s41592-023-02144-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis across a growing number of single-cell perturbation datasets is hampered by poor data interoperability. To facilitate development and benchmarking of computational methods, we collect a set of 44 publicly available single-cell perturbation–response datasets with molecular readouts, including transcriptomics, proteomics and epigenomics. We apply uniform quality control pipelines and harmonize feature annotations. The resulting information resource, scPerturb, enables development and testing of computational methods, and facilitates comparison and integration across datasets. We describe energy statistics (E-statistics) for quantification of perturbation effects and significance testing, and demonstrate E-distance as a general distance measure between sets of single-cell expression profiles. We illustrate the application of E-statistics for quantifying similarity and efficacy of perturbations. The perturbation–response datasets and E-statistics computation software are publicly available at scperturb.org. This work provides an information resource for researchers working with single-cell perturbation data and recommendations for experimental design, including optimal cell counts and read depth. scPerturb is an information resource for single-cell perturbation data analysis and comparison.},
  archive      = {J_NMETH},
  author       = {Peidli, Stefan and Green, Tessa D. and Shen, Ciyue and Gross, Torsten and Min, Joseph and Garda, Samuele and Yuan, Bo and Schumacher, Linus J. and Taylor-King, Jake P. and Marks, Debora S. and Luna, Augustin and Blüthgen, Nils and Sander, Chris},
  doi          = {10.1038/s41592-023-02144-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {531-540},
  shortjournal = {Nat. Meth.},
  title        = {ScPerturb: Harmonized single-cell perturbation data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multiscale biochemical mapping of the brain through
deep-learning-enhanced high-throughput mass spectrometry.
<em>NMETH</em>, <em>21</em>(3), 521–530. (<a
href="https://doi.org/10.1038/s41592-024-02171-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics technologies can reveal the molecular intricacy of the brain. While mass spectrometry imaging (MSI) provides spatial localization of compounds, comprehensive biochemical profiling at a brain-wide scale in three dimensions by MSI with single-cell resolution has not been achieved. We demonstrate complementary brain-wide and single-cell biochemical mapping using MEISTER, an integrative experimental and computational mass spectrometry (MS) framework. Our framework integrates a deep-learning-based reconstruction that accelerates high-mass-resolving MS by 15-fold, multimodal registration creating three-dimensional (3D) molecular distributions and a data integration method fitting cell-specific mass spectra to 3D datasets. We imaged detailed lipid profiles in tissues with millions of pixels and in large single-cell populations acquired from the rat brain. We identified region-specific lipid contents and cell-specific localizations of lipids depending on both cell subpopulations and anatomical origins of the cells. Our workflow establishes a blueprint for future development of multiscale technologies for biochemical characterization of the brain. MEISTER is an integrative experimental and computational framework for mass spectrometry that integrates three-dimensional, organ-wide biomolecular mapping with single-cell analysis for multiscale profiling of spatial–biochemical organization.},
  archive      = {J_NMETH},
  author       = {Xie, Yuxuan Richard and Castro, Daniel C. and Rubakhin, Stanislav S. and Trinklein, Timothy J. and Sweedler, Jonathan V. and Lam, Fan},
  doi          = {10.1038/s41592-024-02171-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {521-530},
  shortjournal = {Nat. Meth.},
  title        = {Multiscale biochemical mapping of the brain through deep-learning-enhanced high-throughput mass spectrometry},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A paintbrush for delivery of nanoparticles and molecules to
live cells with precise spatiotemporal control. <em>NMETH</em>,
<em>21</em>(3), 512–520. (<a
href="https://doi.org/10.1038/s41592-024-02177-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Delivery of very small amounts of reagents to the near-field of cells with micrometer spatial precision and millisecond time resolution is currently out of reach. Here we present μkiss as a micropipette-based scheme for brushing a layer of small molecules and nanoparticles onto the live cell membrane from a subfemtoliter confined volume of a perfusion flow. We characterize our system through both experiments and modeling, and find excellent agreement. We demonstrate several applications that benefit from a controlled brush delivery, such as a direct means to quantify local and long-range membrane mobility and organization as well as dynamical probing of intercellular force signaling. Micro-kiss (μkiss) is a micropipette-based approach for delivering very small amounts of nanoparticles and small molecules to the cell surface with exquisite spatiotemporal control, enabling a wide range of biological investigations.},
  archive      = {J_NMETH},
  author       = {Holler, Cornelia and Taylor, Richard William and Schambony, Alexandra and Möckl, Leonhard and Sandoghdar, Vahid},
  doi          = {10.1038/s41592-024-02177-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {512-520},
  shortjournal = {Nat. Meth.},
  title        = {A paintbrush for delivery of nanoparticles and molecules to live cells with precise spatiotemporal control},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). VIBRANT: Spectral profiling for single-cell drug responses.
<em>NMETH</em>, <em>21</em>(3), 501–511. (<a
href="https://doi.org/10.1038/s41592-024-02185-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-content cell profiling has proven invaluable for single-cell phenotyping in response to chemical perturbations. However, methods with improved throughput, information content and affordability are still needed. We present a new high-content spectral profiling method named vibrational painting (VIBRANT), integrating mid-infrared vibrational imaging, multiplexed vibrational probes and an optimized data analysis pipeline for measuring single-cell drug responses. Three infrared-active vibrational probes were designed to measure distinct essential metabolic activities in human cancer cells. More than 20,000 single-cell drug responses were collected, corresponding to 23 drug treatments. The resulting spectral profile is highly sensitive to phenotypic changes under drug perturbation. Using this property, we built a machine learning classifier to accurately predict drug mechanism of action at single-cell level with minimal batch effects. We further designed an algorithm to discover drug candidates with new mechanisms of action and evaluate drug combinations. Overall, VIBRANT has demonstrated great potential across multiple areas of phenotypic screening. Vibrational painting (VIBRANT) is a high-content single-cell phenotypic profiling method using mid-infrared imaging with vibrational probes for metabolic activity, which offers high accuracy with minimal batch effects to capture cellular responses to perturbation.},
  archive      = {J_NMETH},
  author       = {Liu, Xinwen and Shi, Lixue and Zhao, Zhilun and Shu, Jian and Min, Wei},
  doi          = {10.1038/s41592-024-02185-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {501-511},
  shortjournal = {Nat. Meth.},
  title        = {VIBRANT: Spectral profiling for single-cell drug responses},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Tapioca: A platform for predicting de novo protein–protein
interactions in dynamic contexts. <em>NMETH</em>, <em>21</em>(3),
488–500. (<a href="https://doi.org/10.1038/s41592-024-02179-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein–protein interactions (PPIs) drive cellular processes and responses to environmental cues, reflecting the cellular state. Here we develop Tapioca, an ensemble machine learning framework for studying global PPIs in dynamic contexts. Tapioca predicts de novo interactions by integrating mass spectrometry interactome data from thermal/ion denaturation or cofractionation workflows with protein properties and tissue-specific functional networks. Focusing on the thermal proximity coaggregation method, we improved the experimental workflow. Finely tuned thermal denaturation afforded increased throughput, while cell lysis optimization enhanced protein detection from different subcellular compartments. The Tapioca workflow was next leveraged to investigate viral infection dynamics. Temporal PPIs were characterized during the reactivation from latency of the oncogenic Kaposi’s sarcoma-associated herpesvirus. Together with functional assays, NUCKS was identified as a proviral hub protein, and a broader role was uncovered by integrating PPI networks from alpha- and betaherpesvirus infections. Altogether, Tapioca provides a web-accessible platform for predicting PPIs in dynamic contexts. Tapioca is an ensemble machine learning framework for studying protein–protein interactions (PPIs) that facilitates integration of curve-based dynamic PPI data from thermal proximity coaggregation, ion-based proteome-integrated solubility alteration or cofractionation mass spectrometry data with static interaction data to predict PPIs in dynamic contexts.},
  archive      = {J_NMETH},
  author       = {Reed, Tavis. J. and Tyl, Matthew. D. and Tadych, Alicja and Troyanskaya, Olga. G. and Cristea, Ileana. M.},
  doi          = {10.1038/s41592-024-02179-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {488-500},
  shortjournal = {Nat. Meth.},
  title        = {Tapioca: A platform for predicting de novo protein–protein interactions in dynamic contexts},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). CombFold: Predicting structures of large protein assemblies
using a combinatorial assembly algorithm and AlphaFold2. <em>NMETH</em>,
<em>21</em>(3), 477–487. (<a
href="https://doi.org/10.1038/s41592-024-02174-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning models, such as AlphaFold2 and RosettaFold, enable high-accuracy protein structure prediction. However, large protein complexes are still challenging to predict due to their size and the complexity of interactions between multiple subunits. Here we present CombFold, a combinatorial and hierarchical assembly algorithm for predicting structures of large protein complexes utilizing pairwise interactions between subunits predicted by AlphaFold2. CombFold accurately predicted (TM-score &amp;gt;0.7) 72% of the complexes among the top-10 predictions in two datasets of 60 large, asymmetric assemblies. Moreover, the structural coverage of predicted complexes was 20% higher compared to corresponding Protein Data Bank entries. We applied the method on complexes from Complex Portal with known stoichiometry but without known structure and obtained high-confidence predictions. CombFold supports the integration of distance restraints based on crosslinking mass spectrometry and fast enumeration of possible complex stoichiometries. CombFold’s high accuracy makes it a promising tool for expanding structural coverage beyond monomeric proteins. CombFold is a combinatorial and hierarchical assembly algorithm for predicting structures of large protein complexes utilizing pairwise interactions between subunits predicted by AlphaFold2.},
  archive      = {J_NMETH},
  author       = {Shor, Ben and Schneidman-Duhovny, Dina},
  doi          = {10.1038/s41592-024-02174-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {477-487},
  shortjournal = {Nat. Meth.},
  title        = {CombFold: Predicting structures of large protein assemblies using a combinatorial assembly algorithm and AlphaFold2},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Direct prediction of intrinsically disordered protein
conformational properties from sequence. <em>NMETH</em>, <em>21</em>(3),
465–476. (<a href="https://doi.org/10.1038/s41592-023-02159-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intrinsically disordered regions (IDRs) are ubiquitous across all domains of life and play a range of functional roles. While folded domains are generally well described by a stable three-dimensional structure, IDRs exist in a collection of interconverting states known as an ensemble. This structural heterogeneity means that IDRs are largely absent from the Protein Data Bank, contributing to a lack of computational approaches to predict ensemble conformational properties from sequence. Here we combine rational sequence design, large-scale molecular simulations and deep learning to develop ALBATROSS, a deep-learning model for predicting ensemble dimensions of IDRs, including the radius of gyration, end-to-end distance, polymer-scaling exponent and ensemble asphericity, directly from sequences at a proteome-wide scale. ALBATROSS is lightweight, easy to use and accessible as both a locally installable software package and a point-and-click-style interface via Google Colab notebooks. We first demonstrate the applicability of our predictors by examining the generalizability of sequence–ensemble relationships in IDRs. Then, we leverage the high-throughput nature of ALBATROSS to characterize the sequence-specific biophysical behavior of IDRs within and between proteomes. ALBATROSS is a deep-learning-based model for predicting ensemble properties of intrinsically disordered proteins and protein regions, such as radius of gyration, end-to-end distance, polymer-scaling exponent and ensemble asphericity, directly from sequences.},
  archive      = {J_NMETH},
  author       = {Lotthammer, Jeffrey M. and Ginell, Garrett M. and Griffith, Daniel and Emenecker, Ryan J. and Holehouse, Alex S.},
  doi          = {10.1038/s41592-023-02159-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {465-476},
  shortjournal = {Nat. Meth.},
  title        = {Direct prediction of intrinsically disordered protein conformational properties from sequence},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Exonuclease-enhanced prime editors. <em>NMETH</em>,
<em>21</em>(3), 455–464. (<a
href="https://doi.org/10.1038/s41592-023-02162-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing (PE) is a powerful gene-editing technique based on targeted gRNA-templated reverse transcription and integration of the de novo synthesized single-stranded DNA. To circumvent one of the main bottlenecks of the method, the competition of the reverse-transcribed 3′ flap with the original 5′ flap DNA, we generated an enhanced fluorescence-activated cell sorting reporter cell line to develop an exonuclease-enhanced PE strategy (‘Exo-PE’) composed of an improved PE complex and an aptamer-recruited DNA-exonuclease to remove the 5′ original DNA flap. Exo-PE achieved better overall editing efficacy than the reference PE2 strategy for insertions ≥30 base pairs in several endogenous loci and cell lines while maintaining the high editing precision of PE2. By enabling the precise incorporation of larger insertions, Exo-PE complements the growing palette of different PE tools and spurs additional refinements of the PE machinery. Exo-PE is an approach to improve prime editing efficacy for insertions while maintaining precision.},
  archive      = {J_NMETH},
  author       = {Truong, Dong-Jiunn Jeffery and Geilenkeuser, Julian and Wendel, Stephanie Victoria and Wilming, Julius Clemens Heinrich and Armbrust, Niklas and Binder, Eva Maria Hildegard and Santl, Tobias Heinrich and Siebenhaar, Annika and Gruber, Christoph and Phlairaharn, Teeradon and Živanić, Milica and Westmeyer, Gil Gregor},
  doi          = {10.1038/s41592-023-02162-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {455-464},
  shortjournal = {Nat. Meth.},
  title        = {Exonuclease-enhanced prime editors},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). TISSUE: Uncertainty-calibrated prediction of single-cell
spatial transcriptomics improves downstream analyses. <em>NMETH</em>,
<em>21</em>(3), 444–454. (<a
href="https://doi.org/10.1038/s41592-024-02184-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-transcriptome spatial profiling of genes at single-cell resolution remains a challenge. To address this limitation, spatial gene expression prediction methods have been developed to infer the spatial expression of unmeasured transcripts, but the quality of these predictions can vary greatly. Here we present Transcript Imputation with Spatial Single-cell Uncertainty Estimation (TISSUE) as a general framework for estimating uncertainty for spatial gene expression predictions and providing uncertainty-aware methods for downstream inference. Leveraging conformal inference, TISSUE provides well-calibrated prediction intervals for predicted expression values across 11 benchmark datasets. Moreover, it consistently reduces the false discovery rate for differential gene expression analysis, improves clustering and visualization of predicted spatial transcriptomics and improves the performance of supervised learning models trained on predicted gene expression profiles. Applying TISSUE to a MERFISH spatial transcriptomics dataset of the adult mouse subventricular zone, we identified subtypes within the neural stem cell lineage and developed subtype-specific regional classifiers. Transcript Imputation with Spatial Single-cell Uncertainty Estimation (TISSUE) offers a general framework for estimating uncertainty for spatial gene expression predictions, enabling improved downstream analysis of spatially resolved transcriptomics data.},
  archive      = {J_NMETH},
  author       = {Sun, Eric D. and Ma, Rong and Navarro Negredo, Paloma and Brunet, Anne and Zou, James},
  doi          = {10.1038/s41592-024-02184-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {444-454},
  shortjournal = {Nat. Meth.},
  title        = {TISSUE: Uncertainty-calibrated prediction of single-cell spatial transcriptomics improves downstream analyses},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep generative design of RNA family sequences.
<em>NMETH</em>, <em>21</em>(3), 435–443. (<a
href="https://doi.org/10.1038/s41592-023-02148-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA engineering has immense potential to drive innovation in biotechnology and medicine. Despite its importance, a versatile platform for the automated design of functional RNA is still lacking. Here, we propose RNA family sequence generator (RfamGen), a deep generative model that designs RNA family sequences in a data-efficient manner by explicitly incorporating alignment and consensus secondary structure information. RfamGen can generate novel and functional RNA family sequences by sampling points from a semantically rich and continuous representation. We have experimentally demonstrated the versatility of RfamGen using diverse RNA families. Furthermore, we confirmed the high success rate of RfamGen in designing functional ribozymes through a quantitative massively parallel assay. Notably, RfamGen successfully generates artificial sequences with higher activity than natural sequences. Overall, RfamGen significantly improves our ability to design functional RNA and opens up new potential for generative RNA engineering in synthetic biology. RNA family sequence generator (RfamGen) is a deep generative model for designing novel, functional RNA sequences. RfamGen is applicable to diverse RNA families and can yield ribozymes with higher enzymatic activity.},
  archive      = {J_NMETH},
  author       = {Sumi, Shunsuke and Hamada, Michiaki and Saito, Hirohide},
  doi          = {10.1038/s41592-023-02148-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {435-443},
  shortjournal = {Nat. Meth.},
  title        = {Deep generative design of RNA family sequences},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). TREX reveals proteins that bind to specific RNA regions in
living cells. <em>NMETH</em>, <em>21</em>(3), 423–434. (<a
href="https://doi.org/10.1038/s41592-024-02181-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Different regions of RNA molecules can often engage in specific interactions with distinct RNA-binding proteins (RBPs), giving rise to diverse modalities of RNA regulation and function. However, there are currently no methods for unbiased identification of RBPs that interact with specific RNA regions in living cells and under endogenous settings. Here we introduce TREX (targeted RNase H-mediated extraction of crosslinked RBPs)—a highly sensitive approach for identifying proteins that directly bind to specific RNA regions in living cells. We demonstrate that TREX outperforms existing methods in identifying known interactors of U1 snRNA, and reveals endogenous region-specific interactors of NORAD long noncoding RNA. Using TREX, we generated a comprehensive region-by-region interactome for 45S rRNA, uncovering both established and previously unknown interactions that regulate ribosome biogenesis. With its applicability to different cell types, TREX is an RNA-centric tool for unbiased positional mapping of endogenous RNA–protein interactions in living cells. TREX introduces an RNA-centric tool for identifying proteins binding to specific RNA regions in living cells.},
  archive      = {J_NMETH},
  author       = {Dodel, Martin and Guiducci, Giulia and Dermit, Maria and Krishnamurthy, Sneha and Alard, Emilie L. and Capraro, Federica and Rekad, Zeinab and Stojic, Lovorka and Mardakheh, Faraz K.},
  doi          = {10.1038/s41592-024-02181-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {423-434},
  shortjournal = {Nat. Meth.},
  title        = {TREX reveals proteins that bind to specific RNA regions in living cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). RNA structure profiling at single-cell resolution reveals
new determinants of cell identity. <em>NMETH</em>, <em>21</em>(3),
411–422. (<a href="https://doi.org/10.1038/s41592-023-02128-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA structure is critical for multiple steps in gene regulation. However, how the structures of transcripts differ both within and between individual cells is unknown. Here we develop a SHAPE-inspired method called single-cell structure probing of RNA transcripts that enables simultaneous determination of transcript secondary structure and abundance at single-cell resolution. We apply single-cell structure probing of RNA transcripts to human embryonic stem cells and differentiating neurons. Remarkably, RNA structure is more homogeneous in human embryonic stem cells compared with neurons, with the greatest homogeneity found in coding regions. More extensive heterogeneity is found within 3′ untranslated regions and is determined by specific RNA-binding proteins. Overall RNA structure profiles better discriminate cell type identity and differentiation stage than gene expression profiles alone. We further discover a cell-type variable region of 18S ribosomal RNA that is associated with cell cycle and translation control. Our method opens the door to the systematic characterization of RNA structure–function relationships at single-cell resolution. Single-cell structure probing of RNA transcripts enables simultaneous determination of transcript secondary structure and abundance in single cells, allowing new insights into RNA structural heterogeneity within and among cells.},
  archive      = {J_NMETH},
  author       = {Wang, Jiaxu and Zhang, Yu and Zhang, Tong and Tan, Wen Ting and Lambert, Finnlay and Darmawan, Jefferson and Huber, Roland and Wan, Yue},
  doi          = {10.1038/s41592-023-02128-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {411-422},
  shortjournal = {Nat. Meth.},
  title        = {RNA structure profiling at single-cell resolution reveals new determinants of cell identity},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An improved pathway for autonomous bioluminescence imaging
in eukaryotes. <em>NMETH</em>, <em>21</em>(3), 406–410. (<a
href="https://doi.org/10.1038/s41592-023-02152-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery of the bioluminescence pathway in the fungus Neonothopanus nambi enabled engineering of eukaryotes with self-sustained luminescence. However, the brightness of luminescence in heterologous hosts was limited by performance of the native fungal enzymes. Here we report optimized versions of the pathway that enhance bioluminescence by one to two orders of magnitude in plant, fungal and mammalian hosts, and enable longitudinal video-rate imaging. Improvements to the fully genetically encoded Neonothopanus nambi bioluminescence pathway enhance autobioluminescence by up to two orders of magnitude in plants and other species, enabling novel applications of bioluminescence imaging in biology.},
  archive      = {J_NMETH},
  author       = {Shakhova, Ekaterina S. and Karataeva, Tatiana A. and Markina, Nadezhda M. and Mitiouchkina, Tatiana and Palkina, Kseniia A. and Perfilov, Maxim M. and Wood, Monika G. and Hoang, Trish T. and Hall, Mary P. and Fakhranurova, Liliia I. and Alekberova, Anna E. and Malyshevskaia, Alena K. and Gorbachev, Dmitry A. and Bugaeva, Evgenia N. and Pletneva, Ludmila K. and Babenko, Vladislav V. and Boldyreva, Daria I. and Gorokhovatsky, Andrey Y. and Balakireva, Anastasia V. and Gao, Feng and Choob, Vladimir V. and Encell, Lance P. and Wood, Keith V. and Yampolsky, Ilia V. and Sarkisyan, Karen S. and Mishin, Alexander S.},
  doi          = {10.1038/s41592-023-02152-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {406-410},
  shortjournal = {Nat. Meth.},
  title        = {An improved pathway for autonomous bioluminescence imaging in eukaryotes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Correcting PCR amplification errors in unique molecular
identifiers to generate accurate numbers of sequencing molecules.
<em>NMETH</em>, <em>21</em>(3), 401–405. (<a
href="https://doi.org/10.1038/s41592-024-02168-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unique molecular identifiers are random oligonucleotide sequences that remove PCR amplification biases. However, the impact that PCR associated sequencing errors have on the accuracy of generating absolute counts of RNA molecules is underappreciated. We show that PCR errors are a source of inaccuracy in both bulk and single-cell sequencing data, and synthesizing unique molecular identifiers using homotrimeric nucleotide blocks provides an error-correcting solution that allows absolute counting of sequenced molecules. This study introduces a method utilizing homotrimeric nucleotide blocks to achieve accurate counts of RNA molecules in both bulk and single-cell sequencing data.},
  archive      = {J_NMETH},
  author       = {Sun, Jianfeng and Philpott, Martin and Loi, Danson and Li, Shuang and Monteagudo-Mesas, Pablo and Hoffman, Gabriela and Robson, Jonathan and Mehta, Neelam and Gamble, Vicki and Brown, Tom and Canzar, Stefan and Oppermann, Udo and Cribbs, Adam P.},
  doi          = {10.1038/s41592-024-02168-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {401-405},
  shortjournal = {Nat. Meth.},
  title        = {Correcting PCR amplification errors in unique molecular identifiers to generate accurate numbers of sequencing molecules},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Challenges and perspectives in computational deconvolution
of genomics data. <em>NMETH</em>, <em>21</em>(3), 391–400. (<a
href="https://doi.org/10.1038/s41592-023-02166-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering cell-type heterogeneity is crucial for systematically understanding tissue homeostasis and its dysregulation in diseases. Computational deconvolution is an efficient approach for estimating cell-type abundances from a variety of omics data. Despite substantial methodological progress in computational deconvolution in recent years, challenges are still outstanding. Here we enlist four important challenges related to computational deconvolution: the quality of the reference data, generation of ground truth data, limitations of computational methodologies, and benchmarking design and implementation. Finally, we make recommendations on reference data generation, new directions of computational methodologies, and strategies to promote rigorous benchmarking. This Perspective provides an overview of major challenges and associated recommendations in computational deconvolution for genomics data.},
  archive      = {J_NMETH},
  author       = {Garmire, Lana X. and Li, Yijun and Huang, Qianhui and Xu, Chuan and Teichmann, Sarah A. and Kaminski, Naftali and Pellegrini, Matteo and Nguyen, Quan and Teschendorff, Andrew E.},
  doi          = {10.1038/s41592-023-02166-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {391-400},
  shortjournal = {Nat. Meth.},
  title        = {Challenges and perspectives in computational deconvolution of genomics data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). VIBRANT: A phenotyping method for drug discovery using
vibrational spectroscopy. <em>NMETH</em>, <em>21</em>(3), 389–390. (<a
href="https://doi.org/10.1038/s41592-024-02186-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a high-content profiling method named vibrational painting (VIBRANT) for single-cell drug response measurements, combining vibrational imaging, multiplexed vibrational probes and machine learning. VIBRANT showed high performance in predicting drug mechanisms of action, discovering novel compounds and assessing drug combinations, demonstrating great promise for phenotypic drug discovery.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02186-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {389-390},
  shortjournal = {Nat. Meth.},
  title        = {VIBRANT: A phenotyping method for drug discovery using vibrational spectroscopy},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deciphering protein interaction network dynamics with a
machine learning-based framework. <em>NMETH</em>, <em>21</em>(3),
387–388. (<a href="https://doi.org/10.1038/s41592-024-02180-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed Tapioca, an integrative ensemble machine learning-based framework, to accurately predict global protein–protein interaction network dynamics. Tapioca enabled the characterization of host regulation during reactivation from latency of an oncogenic virus. Introducing an interactome homology analysis method, we identified a proviral host factor with broad relevance for herpesviruses.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02180-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {387-388},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering protein interaction network dynamics with a machine learning-based framework},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An easy-to-use computational tool for predicting 3D
properties of disordered proteins. <em>NMETH</em>, <em>21</em>(3),
385–386. (<a href="https://doi.org/10.1038/s41592-023-02160-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intrinsically disordered regions of proteins are prevalent across the kingdoms of life; however, biophysical characterization is expensive, requiring specialized expertise and equipment and time-consuming sample preparation. By combining simulations and deep learning, we have developed a method to predict their average ensemble properties directly from sequence.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02160-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {385-386},
  shortjournal = {Nat. Meth.},
  title        = {An easy-to-use computational tool for predicting 3D properties of disordered proteins},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A prime editor that makes space for insertions.
<em>NMETH</em>, <em>21</em>(3), 383–384. (<a
href="https://doi.org/10.1038/s41592-023-02163-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a prime editing (PE) strategy by incorporating a 5′–3′ exonuclease activity, which enhanced the efficacy and precision of ≥30-nucleotide DNA insertions without a secondary nick. Our optimization of the PE complex revealed that recruiting the exonuclease via an RNA aptamer outperformed direct protein fusions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02163-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {383-384},
  shortjournal = {Nat. Meth.},
  title        = {A prime editor that makes space for insertions},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). TREX identifies region-specific protein interactors of RNA
molecules. <em>NMETH</em>, <em>21</em>(3), 381–382. (<a
href="https://doi.org/10.1038/s41592-024-02182-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interactions between RNA and RNA-binding proteins (RBPs) define the fate and function of every RNA molecule. We present TREX, or targeted RNase H-mediated extraction of crosslinked RBPs, an efficient and accurate method to unbiasedly reveal the protein interactors of specific regions of RNAs isolated from living cells.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02182-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {381-382},
  shortjournal = {Nat. Meth.},
  title        = {TREX identifies region-specific protein interactors of RNA molecules},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Homotrimer barcodes enable accurate counting of RNA
molecules during high-throughput RNA sequencing. <em>NMETH</em>,
<em>21</em>(3), 379–380. (<a
href="https://doi.org/10.1038/s41592-024-02169-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We pinpoint PCR artifacts as the primary source of inaccurate quantification in both short- and long-read RNA sequencing, a problem that intensifies with an increase in PCR cycles in both bulk and single-cell sequencing contexts. To overcome this challenge, we engineered a novel unique molecular identifier (UMI) barcode composed of homotrimer nucleotide blocks. This design facilitates accurate quantification of RNA molecules, substantially improving molecular counting.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02169-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {379-380},
  shortjournal = {Nat. Meth.},
  title        = {Homotrimer barcodes enable accurate counting of RNA molecules during high-throughput RNA sequencing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell probing of RNA structure. <em>NMETH</em>,
<em>21</em>(3), 377–378. (<a
href="https://doi.org/10.1038/s41592-024-02178-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {sc-SPORT offers a way to probe RNA structure at the single-cell level. It reveals cell-to-cell heterogeneity in RNA folding.},
  archive      = {J_NMETH},
  author       = {Jolley, Elizabeth A. and Bevilacqua, Philip C.},
  doi          = {10.1038/s41592-024-02178-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {377-378},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell probing of RNA structure},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Collaborate, disseminate, accelerate. <em>NMETH</em>,
<em>21</em>(3), 372–376. (<a
href="https://doi.org/10.1038/s41592-024-02202-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Collaborations between researchers and companies can progress swimmingly and teams quickly validate findings and mature methods. All too often, things can’t advance and the ‘Valley of Death’ looms. New ways to collaborate, underpinned by computational muscle, can help.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02202-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {372-376},
  shortjournal = {Nat. Meth.},
  title        = {Collaborate, disseminate, accelerate},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Graphene sandwich for cryo-EM. <em>NMETH</em>,
<em>21</em>(3), 371. (<a
href="https://doi.org/10.1038/s41592-024-02219-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-024-02219-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {371},
  shortjournal = {Nat. Meth.},
  title        = {Graphene sandwich for cryo-EM},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Long-term monosynaptic tracing. <em>NMETH</em>,
<em>21</em>(3), 371. (<a
href="https://doi.org/10.1038/s41592-024-02220-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02220-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {371},
  shortjournal = {Nat. Meth.},
  title        = {Long-term monosynaptic tracing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Neural networks for biomechanics. <em>NMETH</em>,
<em>21</em>(3), 370. (<a
href="https://doi.org/10.1038/s41592-024-02218-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02218-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {370},
  shortjournal = {Nat. Meth.},
  title        = {Neural networks for biomechanics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A leap for directed evolution. <em>NMETH</em>,
<em>21</em>(3), 370. (<a
href="https://doi.org/10.1038/s41592-024-02221-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-024-02221-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {370},
  shortjournal = {Nat. Meth.},
  title        = {A leap for directed evolution},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). EfficientBioAI: Making bioimaging AI models efficient in
energy and latency. <em>NMETH</em>, <em>21</em>(3), 368–369. (<a
href="https://doi.org/10.1038/s41592-024-02167-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zhou, Yu and Cao, Jiajun and Sonneck, Justin and Banerjee, Sweta and Dörr, Stefanie and Grüneboom, Anika and Lorenz, Kristina and Zhang, Shanghang and Chen, Jianxu},
  doi          = {10.1038/s41592-024-02167-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {368-369},
  shortjournal = {Nat. Meth.},
  title        = {EfficientBioAI: Making bioimaging AI models efficient in energy and latency},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). OpenMS 3 enables reproducible analysis of large-scale mass
spectrometry data. <em>NMETH</em>, <em>21</em>(3), 365–367. (<a
href="https://doi.org/10.1038/s41592-024-02197-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Pfeuffer, Julianus and Bielow, Chris and Wein, Samuel and Jeong, Kyowon and Netz, Eugen and Walter, Axel and Alka, Oliver and Nilse, Lars and Colaianni, Pasquale Domenico and McCloskey, Douglas and Kim, Jihyung and Rosenberger, George and Bichmann, Leon and Walzer, Mathias and Veit, Johannes and Boudaud, Bertrand and Bernt, Matthias and Patikas, Nikolaos and Pilz, Matteo and Startek, Michał Piotr and Kutuzova, Svetlana and Heumos, Lukas and Charkow, Joshua and Sing, Justin Cyril and Feroz, Ayesha and Siraj, Arslan and Weisser, Hendrik and Dijkstra, Tjeerd M. H. and Perez-Riverol, Yasset and Röst, Hannes and Kohlbacher, Oliver and Sachsenberg, Timo},
  doi          = {10.1038/s41592-024-02197-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {365-367},
  shortjournal = {Nat. Meth.},
  title        = {OpenMS 3 enables reproducible analysis of large-scale mass spectrometry data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The brown seaweed ectocarpus. <em>NMETH</em>,
<em>21</em>(3), 363–364. (<a
href="https://doi.org/10.1038/s41592-024-02198-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brown seaweeds are multicellular eukaryotes that have been evolving independently of animals and plants for more than a billion years. The filamentous brown alga Ectocarpus has been used as a model to understand the biology of these enigmatic organisms and to shed light on a range of major questions, from the molecular basis of complex developmental patterns to the evolution of sex.},
  archive      = {J_NMETH},
  author       = {Coelho, Susana M.},
  doi          = {10.1038/s41592-024-02198-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {363-364},
  shortjournal = {Nat. Meth.},
  title        = {The brown seaweed ectocarpus},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Peer review demystified: Part 1. <em>NMETH</em>,
<em>21</em>(3), 361. (<a
href="https://doi.org/10.1038/s41592-024-02225-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peer review is at the heart of publishing scientific papers. In this first installment of a two-part Editorial, we explain how we manage the process at Nature Methods.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02225-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {361},
  shortjournal = {Nat. Meth.},
  title        = {Peer review demystified: Part 1},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Early-career navigation tips. <em>NMETH</em>,
<em>21</em>(3), 362. (<a
href="https://doi.org/10.1038/s41592-024-02183-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In a group interview, early-career researchers in proteomics share how they plan for the future and navigate divides over methods.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02183-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {3},
  pages        = {362},
  shortjournal = {Nat. Meth.},
  title        = {Early-career navigation tips},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Real-time visualization of structural dynamics of synapses
in live cells in vivo. <em>NMETH</em>, <em>21</em>(2), 353–360. (<a
href="https://doi.org/10.1038/s41592-023-02122-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The structural plasticity of synapses is crucial for regulating brain functions. However, currently available methods for studying synapse organization based on split fluorescent proteins (FPs) have been limited in assessing synaptic dynamics in vivo due to the irreversible binding of split FPs. Here, we develop ‘SynapShot’, a method for visualizing the structural dynamics of intact synapses by combining dimerization-dependent FPs (ddFPs) with engineered synaptic adhesion molecules. SynapShot allows real-time monitoring of reversible and bidirectional changes of synaptic contacts under physiological stimulation. The application of green and red ddFPs in SynapShot enables simultaneous visualization of two distinct populations of synapses. Notably, the red-shifted SynapShot is highly compatible with blue light-based optogenetic techniques, allowing for visualization of synaptic dynamics while precisely controlling specific signaling pathways. Furthermore, we demonstrate that SynapShot enables real-time monitoring of structural changes in synaptic contacts in the mouse brain during both primitive and higher-order behaviors. SynapShot combines ddFPs with engineered synaptic adhesion molecules for real-time observation of the structural plasticity of synapses in cultured cells and animals.},
  archive      = {J_NMETH},
  author       = {Son, Seungkyu and Nagahama, Kenichiro and Lee, Jinsu and Jung, Kanghoon and Kwak, Chuljung and Kim, Jihoon and Noh, Young Woo and Kim, Eunjoon and Lee, Sangkyu and Kwon, Hyung-Bae and Heo, Won Do},
  doi          = {10.1038/s41592-023-02122-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {353-360},
  shortjournal = {Nat. Meth.},
  title        = {Real-time visualization of structural dynamics of synapses in live cells in vivo},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping enzyme activity in living systems by real-time
mid-infrared photothermal imaging of nitrile chameleons. <em>NMETH</em>,
<em>21</em>(2), 342–352. (<a
href="https://doi.org/10.1038/s41592-023-02137-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Simultaneous spatial mapping of the activity of multiple enzymes in a living system can elucidate their functions in health and disease. However, methods based on monitoring fluorescent substrates are limited. Here, we report the development of nitrile (C≡N)-tagged enzyme activity reporters, named nitrile chameleons, for the peak shift between substrate and product. To image these reporters in real time, we developed a laser-scanning mid-infrared photothermal imaging system capable of imaging the enzymatic substrates and products at a resolution of 300 nm. We show that when combined, these tools can map the activity distribution of different enzymes and measure their relative catalytic efficiency in living systems such as cancer cells, Caenorhabditis elegans, and brain tissues, and can be used to directly visualize caspase–phosphatase interactions during apoptosis. Our method is generally applicable to a broad category of enzymes and will enable new analyses of enzymes in their native context. Real-time mid-infrared photothermal imaging of nitrile chameleons enables simultaneous, multiplexed measurement of enzymatic activity in living systems and is poised to reveal the spatiotemporal regulation of enzymes in health and disease.},
  archive      = {J_NMETH},
  author       = {He, Hongjian and Yin, Jiaze and Li, Mingsheng and Dessai, Chinmayee Vallabh Prabhu and Yi, Meihui and Teng, Xinyan and Zhang, Meng and Li, Yueming and Du, Zhiyi and Xu, Bing and Cheng, Ji-Xin},
  doi          = {10.1038/s41592-023-02137-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {342-352},
  shortjournal = {Nat. Meth.},
  title        = {Mapping enzyme activity in living systems by real-time mid-infrared photothermal imaging of nitrile chameleons},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Thermal-plex: Fluidic-free, rapid sequential multiplexed
imaging with DNA-encoded thermal channels. <em>NMETH</em>,
<em>21</em>(2), 331–341. (<a
href="https://doi.org/10.1038/s41592-023-02115-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed fluorescence imaging is typically limited to three- to five-plex on standard setups. Sequential imaging methods based on iterative labeling and imaging enable practical higher multiplexing, but generally require a complex fluidic setup with several rounds of slow buffer exchange (tens of minutes to an hour for each exchange step). We report the thermal-plex method, which removes complex and slow buffer exchange steps and provides fluidic-free, rapid sequential imaging. Thermal-plex uses simple DNA probes that are engineered to fluoresce sequentially when, and only when, activated with transient exposure to heating spikes at designated temperatures (thermal channels). Channel switching is fast (&amp;lt;30 s) and is achieved with a commercially available and affordable on-scope heating device. We demonstrate 15-plex RNA imaging (five thermal × three fluorescence channels) in fixed cells and retina tissues in less than 4 min, without using buffer exchange or fluidics. Thermal-plex introduces a new labeling method for efficient sequential multiplexed imaging. The thermal-plex method for highly multiplexed imaging uses DNA probes activated when briefly elevated to designated temperatures for rapid, fluidics-free sequential imaging in cells and tissues.},
  archive      = {J_NMETH},
  author       = {Hong, Fan and Kishi, Jocelyn Y. and Delgado, Ryan N. and Jeong, Jiyoun and Saka, Sinem K. and Su, Hanquan and Cepko, Constance L. and Yin, Peng},
  doi          = {10.1038/s41592-023-02115-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {331-341},
  shortjournal = {Nat. Meth.},
  title        = {Thermal-plex: Fluidic-free, rapid sequential multiplexed imaging with DNA-encoded thermal channels},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Content-aware frame interpolation (CAFI): Deep
learning-based temporal super-resolution for fast bioimaging.
<em>NMETH</em>, <em>21</em>(2), 322–330. (<a
href="https://doi.org/10.1038/s41592-023-02138-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of high-resolution microscopes has made it possible to investigate cellular processes in 3D and over time. However, observing fast cellular dynamics remains challenging because of photobleaching and phototoxicity. Here we report the implementation of two content-aware frame interpolation (CAFI) deep learning networks, Zooming SlowMo and Depth-Aware Video Frame Interpolation, that are highly suited for accurately predicting images in between image pairs, therefore improving the temporal resolution of image series post-acquisition. We show that CAFI is capable of understanding the motion context of biological structures and can perform better than standard interpolation methods. We benchmark CAFI’s performance on 12 different datasets, obtained from four different microscopy modalities, and demonstrate its capabilities for single-particle tracking and nuclear segmentation. CAFI potentially allows for reduced light exposure and phototoxicity on the sample for improved long-term live-cell imaging. The models and the training and testing data are available via the ZeroCostDL4Mic platform. Content-aware frame interpolation (CAFI) improves the temporal resolution in time-lapse imaging by accurately predicting images in between image pairs. By allowing fewer frames to be imaged, CAFI also enables gentler live-cell imaging.},
  archive      = {J_NMETH},
  author       = {Priessner, Martin and Gaboriau, David C. A. and Sheridan, Arlo and Lenn, Tchern and Garzon-Coral, Carlos and Dunn, Alexander R. and Chubb, Jonathan R. and Tousley, Aidan M. and Majzner, Robbie G. and Manor, Uri and Vilar, Ramon and Laine, Romain F.},
  doi          = {10.1038/s41592-023-02138-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {322-330},
  shortjournal = {Nat. Meth.},
  title        = {Content-aware frame interpolation (CAFI): Deep learning-based temporal super-resolution for fast bioimaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Image restoration of degraded time-lapse microscopy data
mediated by near-infrared imaging. <em>NMETH</em>, <em>21</em>(2),
311–321. (<a href="https://doi.org/10.1038/s41592-023-02127-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Time-lapse fluorescence microscopy is key to unraveling biological development and function; however, living systems, by their nature, permit only limited interrogation and contain untapped information that can only be captured by more invasive methods. Deep-tissue live imaging presents a particular challenge owing to the spectral range of live-cell imaging probes/fluorescent proteins, which offer only modest optical penetration into scattering tissues. Herein, we employ convolutional neural networks to augment live-imaging data with deep-tissue images taken on fixed samples. We demonstrate that convolutional neural networks may be used to restore deep-tissue contrast in GFP-based time-lapse imaging using paired final-state datasets acquired using near-infrared dyes, an approach termed InfraRed-mediated Image Restoration (IR2). Notably, the networks are remarkably robust over a wide range of developmental times. We employ IR2 to enhance the information content of green fluorescent protein time-lapse images of zebrafish and Drosophila embryo/larval development and demonstrate its quantitative potential in increasing the fidelity of cell tracking/lineaging in developing pescoids. Thus, IR2 is poised to extend live imaging to depths otherwise inaccessible. InfraRed-mediated Image Restoration (IR2) uses deep learning to combine the benefits of deep-tissue imaging with NIR probes and the convenience of imaging with GFP for improved time-lapse imaging of embryogenesis.},
  archive      = {J_NMETH},
  author       = {Gritti, Nicola and Power, Rory M. and Graves, Alyssa and Huisken, Jan},
  doi          = {10.1038/s41592-023-02127-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {311-321},
  shortjournal = {Nat. Meth.},
  title        = {Image restoration of degraded time-lapse microscopy data mediated by near-infrared imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Smart lattice light-sheet microscopy for imaging rare and
complex cellular events. <em>NMETH</em>, <em>21</em>(2), 301–310. (<a
href="https://doi.org/10.1038/s41592-023-02126-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light-sheet microscopes enable rapid high-resolution imaging of biological specimens; however, biological processes span spatiotemporal scales. Moreover, long-term phenotypes are often instigated by rare or fleeting biological events that are difficult to capture with a single imaging modality. Here, to overcome this limitation, we present smartLLSM, a microscope that incorporates artificial intelligence-based instrument control to autonomously switch between epifluorescent inverted imaging and lattice light-sheet microscopy (LLSM). We apply this approach to two unique processes: cell division and immune synapse formation. In each context, smartLLSM provides population-level statistics across thousands of cells and autonomously captures multicolor three-dimensional datasets or four-dimensional time-lapse movies of rare events at rates that dramatically exceed human capabilities. From this, we quantify the effects of Taxol dose on spindle structure and kinetochore dynamics in dividing cells and of antigen strength on cytotoxic T lymphocyte engagement and lytic granule polarization at the immune synapse. Overall, smartLLSM efficiently detects rare events within heterogeneous cell populations and records these processes with high spatiotemporal four-dimensional imaging over statistically significant replicates. smartLLSM uses artificial intelligence-based instrument control to switch between epiflouorescence and lattice light-sheet microscopy to monitor cells at the population level while also capturing multicolor three-dimensional datasets of rare events of interest.},
  archive      = {J_NMETH},
  author       = {Shi, Yu and Tabet, Jimmy S. and Milkie, Daniel E. and Daugird, Timothy A. and Yang, Chelsea Q. and Ritter, Alex T. and Giovannucci, Andrea and Legant, Wesley R.},
  doi          = {10.1038/s41592-023-02126-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {301-310},
  shortjournal = {Nat. Meth.},
  title        = {Smart lattice light-sheet microscopy for imaging rare and complex cellular events},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A data analysis framework for combining multiple batches
increases the power of isobaric proteomics experiments. <em>NMETH</em>,
<em>21</em>(2), 290–300. (<a
href="https://doi.org/10.1038/s41592-023-02120-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a framework for the analysis of multiplexed mass spectrometry proteomics data that reduces estimation error when combining multiple isobaric batches. Variations in the number and quality of observations have long complicated the analysis of isobaric proteomics data. Here we show that the power to detect statistical associations is substantially improved by utilizing models that directly account for known sources of variation in the number and quality of observations that occur across batches. In a multibatch benchmarking experiment, our open-source software (msTrawler) increases the power to detect changes, especially in the range of less than twofold changes, while simultaneously increasing quantitative proteome coverage by utilizing more low-signal observations. Further analyses of previously published multiplexed datasets of 4 and 23 batches highlight both increased power and the ability to navigate complex missing data patterns without relying on unverifiable imputations or discarding reliable measurements. msTrawler improves quantitative analysis of multiplexed mass spectrometry proteomics data when combining isobaric data across multiple batches.},
  archive      = {J_NMETH},
  author       = {O’Brien, Jonathon J. and Raj, Anil and Gaun, Aleksandr and Waite, Adam and Li, Wenzhou and Hendrickson, David G. and Olsson, Niclas and McAllister, Fiona E.},
  doi          = {10.1038/s41592-023-02120-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {290-300},
  shortjournal = {Nat. Meth.},
  title        = {A data analysis framework for combining multiple batches increases the power of isobaric proteomics experiments},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving deep learning protein monomer and complex
structure prediction using DeepMSA2 with huge metagenomics data.
<em>NMETH</em>, <em>21</em>(2), 279–289. (<a
href="https://doi.org/10.1038/s41592-023-02130-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leveraging iterative alignment search through genomic and metagenome sequence databases, we report the DeepMSA2 pipeline for uniform protein single- and multichain multiple-sequence alignment (MSA) construction. Large-scale benchmarks show that DeepMSA2 MSAs can remarkably increase the accuracy of protein tertiary and quaternary structure predictions compared with current state-of-the-art methods. An integrated pipeline with DeepMSA2 participated in the most recent CASP15 experiment and created complex structural models with considerably higher quality than the AlphaFold2-Multimer server (v.2.2.0). Detailed data analyses show that the major advantage of DeepMSA2 lies in its balanced alignment search and effective model selection, and in the power of integrating huge metagenomics databases. These results demonstrate a new avenue to improve deep learning protein structure prediction through advanced MSA construction and provide additional evidence that optimization of input information to deep learning-based structure prediction methods must be considered with as much care as the design of the predictor itself. The DeepMSA2 pipeline employs iterative alignment search against large genomic and metagenomic sequence databases to construct single- and multichain multiple-sequence alignment (MSA) for proteins. Use of these MSAs shows improvement for deep learning-based protein tertiary and quaternary structure predictions.},
  archive      = {J_NMETH},
  author       = {Zheng, Wei and Wuyun, Qiqige and Li, Yang and Zhang, Chengxin and Freddolino, P. Lydia and Zhang, Yang},
  doi          = {10.1038/s41592-023-02130-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {279-289},
  shortjournal = {Nat. Meth.},
  title        = {Improving deep learning protein monomer and complex structure prediction using DeepMSA2 with huge metagenomics data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unsupervised and supervised discovery of tissue cellular
neighborhoods from cell phenotypes. <em>NMETH</em>, <em>21</em>(2),
267–278. (<a href="https://doi.org/10.1038/s41592-023-02124-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is poorly understood how different cells in a tissue organize themselves to support tissue functions. We describe the CytoCommunity algorithm for the identification of tissue cellular neighborhoods (TCNs) based on cell phenotypes and their spatial distributions. CytoCommunity learns a mapping directly from the cell phenotype space to the TCN space using a graph neural network model without intermediate clustering of cell embeddings. By leveraging graph pooling, CytoCommunity enables de novo identification of condition-specific and predictive TCNs under the supervision of sample labels. Using several types of spatial omics data, we demonstrate that CytoCommunity can identify TCNs of variable sizes with substantial improvement over existing methods. By analyzing risk-stratified colorectal and breast cancer data, CytoCommunity revealed new granulocyte-enriched and cancer-associated fibroblast-enriched TCNs specific to high-risk tumors and altered interactions between neoplastic and immune or stromal cells within and between TCNs. CytoCommunity can perform unsupervised and supervised analyses of spatial omics maps and enable the discovery of condition-specific cell–cell communication patterns across spatial scales. CytoCommunity enables both supervised and unsupervised analyses of spatial omics data in order to identify complex tissue cellular neighborhoods based on cell phenotypes and spatial distributions.},
  archive      = {J_NMETH},
  author       = {Hu, Yuxuan and Rong, Jiazhen and Xu, Yafei and Xie, Runzhi and Peng, Jacqueline and Gao, Lin and Tan, Kai},
  doi          = {10.1038/s41592-023-02124-2},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {267-278},
  shortjournal = {Nat. Meth.},
  title        = {Unsupervised and supervised discovery of tissue cellular neighborhoods from cell phenotypes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SEVtras delineates small extracellular vesicles at droplet
resolution from single-cell transcriptomes. <em>NMETH</em>,
<em>21</em>(2), 259–266. (<a
href="https://doi.org/10.1038/s41592-023-02117-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Small extracellular vesicles (sEVs) are emerging as pivotal players in a wide range of physiological and pathological processes. However, a pressing challenge has been the lack of high-throughput techniques capable of unraveling the intricate heterogeneity of sEVs and decoding the underlying cellular behaviors governing sEV secretion. Here we leverage droplet-based single-cell RNA sequencing (scRNA-seq) and introduce an algorithm, SEVtras, to identify sEV-containing droplets and estimate the sEV secretion activity (ESAI) of individual cells. Through extensive validations on both simulated and real datasets, we demonstrate SEVtras’ efficacy in capturing sEV-containing droplets and characterizing the secretion activity of specific cell types. By applying SEVtras to four tumor scRNA-seq datasets, we further illustrate that the ESAI can serve as a potent indicator of tumor progression, particularly in the early stages. With the increasing importance and availability of scRNA-seq datasets, SEVtras holds promise in offering valuable extracellular insights into the cell heterogeneity. SEVtras is an algorithm that uses single-cell RNA sequencing data to assess small extracellular vesicle activity at droplet resolution.},
  archive      = {J_NMETH},
  author       = {He, Ruiqiao and Zhu, Junjie and Ji, Peifeng and Zhao, Fangqing},
  doi          = {10.1038/s41592-023-02117-1},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {259-266},
  shortjournal = {Nat. Meth.},
  title        = {SEVtras delineates small extracellular vesicles at droplet resolution from single-cell transcriptomes},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Profiling of RNA-binding protein binding sites by in situ
reverse transcription-based sequencing. <em>NMETH</em>, <em>21</em>(2),
247–258. (<a href="https://doi.org/10.1038/s41592-023-02146-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) regulate diverse cellular processes by dynamically interacting with RNA targets. However, effective methods to capture both stable and transient interactions between RBPs and their RNA targets are still lacking, especially when the interaction is dynamic or samples are limited. Here we present an assay of reverse transcription-based RBP binding site sequencing (ARTR-seq), which relies on in situ reverse transcription of RBP-bound RNAs guided by antibodies to identify RBP binding sites. ARTR-seq avoids ultraviolet crosslinking and immunoprecipitation, allowing for efficient and specific identification of RBP binding sites from as few as 20 cells or a tissue section. Taking advantage of rapid formaldehyde fixation, ARTR-seq enables capturing the dynamic RNA binding by RBPs over a short period of time, as demonstrated by the profiling of dynamic RNA binding of G3BP1 during stress granule assembly on a timescale as short as 10 minutes. This work introduces ARTR-seq for in situ measurement of RNA-binding protein (RBP) binding sites, which has been demonstrated in a small number of cells and for capturing dynamic RBP binding within short timeframes.},
  archive      = {J_NMETH},
  author       = {Xiao, Yu and Chen, Yan-Ming and Zou, Zhongyu and Ye, Chang and Dou, Xiaoyang and Wu, Jinjun and Liu, Chang and Liu, Shun and Yan, Hao and Wang, Pingluan and Zeng, Tie-Bo and Liu, Qinzhe and Fei, Jingyi and Tang, Weixin and He, Chuan},
  doi          = {10.1038/s41592-023-02146-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {247-258},
  shortjournal = {Nat. Meth.},
  title        = {Profiling of RNA-binding protein binding sites by in situ reverse transcription-based sequencing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MEnrich-seq: Methylation-guided enrichment sequencing of
bacterial taxa of interest from microbiome. <em>NMETH</em>,
<em>21</em>(2), 236–246. (<a
href="https://doi.org/10.1038/s41592-023-02125-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metagenomics has enabled the comprehensive study of microbiomes. However, many applications would benefit from a method that sequences specific bacterial taxa of interest, but not most background taxa. We developed mEnrich-seq (in which ‘m’ stands for methylation and seq for sequencing) for enriching taxa of interest from metagenomic DNA before sequencing. The core idea is to exploit the self versus nonself differentiation by natural bacterial DNA methylation and rationally choose methylation-sensitive restriction enzymes, individually or in combination, to deplete host and background taxa while enriching targeted taxa. This idea is integrated with library preparation procedures and applied in several applications to enrich (up to 117-fold) pathogenic or beneficial bacteria from human urine and fecal samples, including species that are hard to culture or of low abundance. We assessed 4,601 bacterial strains with mapped methylomes so far and showed broad applicability of mEnrich-seq. mEnrich-seq provides microbiome researchers with a versatile and cost-effective approach for selective sequencing of diverse taxa of interest. mEnrich-seq leverages DNA methylation differences between bacteria to enrich taxa of interest from metagenomic samples for selective sequencing.},
  archive      = {J_NMETH},
  author       = {Cao, Lei and Kong, Yimeng and Fan, Yu and Ni, Mi and Tourancheau, Alan and Ksiezarek, Magdalena and Mead, Edward A. and Koo, Tonny and Gitman, Melissa and Zhang, Xue-Song and Fang, Gang},
  doi          = {10.1038/s41592-023-02125-1},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {236-246},
  shortjournal = {Nat. Meth.},
  title        = {MEnrich-seq: Methylation-guided enrichment sequencing of bacterial taxa of interest from microbiome},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Massively parallel single-cell sequencing of diverse
microbial populations. <em>NMETH</em>, <em>21</em>(2), 228–235. (<a
href="https://doi.org/10.1038/s41592-023-02157-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell genetic heterogeneity is ubiquitous in microbial populations and an important aspect of microbial biology; however, we lack a broadly applicable and accessible method to study this heterogeneity in microbial populations. Here, we show a simple, robust and generalizable method for high-throughput single-cell sequencing of target genetic loci in diverse microbes using simple droplet microfluidics devices (droplet targeted amplicon sequencing; DoTA-seq). DoTA-seq serves as a platform to perform diverse assays for single-cell genetic analysis of microbial populations. Using DoTA-seq, we demonstrate the ability to simultaneously track the prevalence and taxonomic associations of &amp;gt;10 antibiotic-resistance genes and plasmids within human and mouse gut microbial communities. This workflow is a powerful and accessible platform for high-throughput single-cell sequencing of diverse microbial populations. DoTA-seq leverages a microfluidic droplet system to isolate and lyse diverse microbes and amplify target genetic loci, enabling high-throughput single-cell sequencing of microbial populations.},
  archive      = {J_NMETH},
  author       = {Lan, Freeman and Saba, Jason and Ross, Tyler D. and Zhou, Zhichao and Krauska, Katie and Anantharaman, Karthik and Landick, Robert and Venturelli, Ophelia S.},
  doi          = {10.1038/s41592-023-02157-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {228-235},
  shortjournal = {Nat. Meth.},
  title        = {Massively parallel single-cell sequencing of diverse microbial populations},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A fast, scalable and versatile tool for analysis of
single-cell omics data. <em>NMETH</em>, <em>21</em>(2), 217–227. (<a
href="https://doi.org/10.1038/s41592-023-02139-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell omics technologies have revolutionized the study of gene regulation in complex tissues. A major computational challenge in analyzing these datasets is to project the large-scale and high-dimensional data into low-dimensional space while retaining the relative relationships between cells. This low dimension embedding is necessary to decompose cellular heterogeneity and reconstruct cell-type-specific gene regulatory programs. Traditional dimensionality reduction techniques, however, face challenges in computational efficiency and in comprehensively addressing cellular diversity across varied molecular modalities. Here we introduce a nonlinear dimensionality reduction algorithm, embodied in the Python package SnapATAC2, which not only achieves a more precise capture of single-cell omics data heterogeneities but also ensures efficient runtime and memory usage, scaling linearly with the number of cells. Our algorithm demonstrates exceptional performance, scalability and versatility across diverse single-cell omics datasets, including single-cell assay for transposase-accessible chromatin using sequencing, single-cell RNA sequencing, single-cell Hi-C and single-cell multi-omics datasets, underscoring its utility in advancing single-cell analysis. SnapATAC2 uses a matrix-free spectral embedding algorithm for nonlinear dimension reduction of single-cell omics data, which shows an improved performance in capturing cellular heterogeneity and scalability for large datasets.},
  archive      = {J_NMETH},
  author       = {Zhang, Kai and Zemke, Nathan R. and Armand, Ethan J. and Ren, Bing},
  doi          = {10.1038/s41592-023-02139-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {217-227},
  shortjournal = {Nat. Meth.},
  title        = {A fast, scalable and versatile tool for analysis of single-cell omics data},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Segmentation metric misinterpretations in bioimage analysis.
<em>NMETH</em>, <em>21</em>(2), 213–216. (<a
href="https://doi.org/10.1038/s41592-023-01942-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative evaluation of image segmentation algorithms is crucial in the field of bioimage analysis. The most common assessment scores, however, are often misinterpreted and multiple definitions coexist with the same name. Here we present the ambiguities of evaluation metrics for segmentation algorithms and show how these misinterpretations can alter leaderboards of influential competitions. We also propose guidelines for how the currently existing problems could be tackled. This study shows the importance of proper metrics for comparing algorithms for bioimage segmentation and object detection by exploring the impact of metrics on the relative performance of algorithms in three image analysis competitions.},
  archive      = {J_NMETH},
  author       = {Hirling, Dominik and Tasnadi, Ervin and Caicedo, Juan and Caroprese, Maria V. and Sjögren, Rickard and Aubreville, Marc and Koos, Krisztian and Horvath, Peter},
  doi          = {10.1038/s41592-023-01942-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {213-216},
  shortjournal = {Nat. Meth.},
  title        = {Segmentation metric misinterpretations in bioimage analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Metrics reloaded: Recommendations for image analysis
validation. <em>NMETH</em>, <em>21</em>(2), 195–212. (<a
href="https://doi.org/10.1038/s41592-023-02151-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increasing evidence shows that flaws in machine learning (ML) algorithm validation are an underestimated global problem. In biomedical image analysis, chosen performance metrics often do not reflect the domain interest, and thus fail to adequately measure scientific progress and hinder translation of ML techniques into practice. To overcome this, we created Metrics Reloaded, a comprehensive framework guiding researchers in the problem-aware selection of metrics. Developed by a large international consortium in a multistage Delphi process, it is based on the novel concept of a problem fingerprint—a structured representation of the given problem that captures all aspects that are relevant for metric selection, from the domain interest to the properties of the target structure(s), dataset and algorithm output. On the basis of the problem fingerprint, users are guided through the process of choosing and applying appropriate validation metrics while being made aware of potential pitfalls. Metrics Reloaded targets image analysis problems that can be interpreted as classification tasks at image, object or pixel level, namely image-level classification, object detection, semantic segmentation and instance segmentation tasks. To improve the user experience, we implemented the framework in the Metrics Reloaded online tool. Following the convergence of ML methodology across application domains, Metrics Reloaded fosters the convergence of validation methodology. Its applicability is demonstrated for various biomedical use cases. Metrics Reloaded is a comprehensive framework for guiding researchers in the problem-aware selection of metrics for common tasks in biomedical image analysis.},
  archive      = {J_NMETH},
  author       = {Maier-Hein, Lena and Reinke, Annika and Godau, Patrick and Tizabi, Minu D. and Buettner, Florian and Christodoulou, Evangelia and Glocker, Ben and Isensee, Fabian and Kleesiek, Jens and Kozubek, Michal and Reyes, Mauricio and Riegler, Michael A. and Wiesenfarth, Manuel and Kavur, A. Emre and Sudre, Carole H. and Baumgartner, Michael and Eisenmann, Matthias and Heckmann-Nötzel, Doreen and Rädsch, Tim and Acion, Laura and Antonelli, Michela and Arbel, Tal and Bakas, Spyridon and Benis, Arriel and Blaschko, Matthew B. and Cardoso, M. Jorge and Cheplygina, Veronika and Cimini, Beth A. and Collins, Gary S. and Farahani, Keyvan and Ferrer, Luciana and Galdran, Adrian and van Ginneken, Bram and Haase, Robert and Hashimoto, Daniel A. and Hoffman, Michael M. and Huisman, Merel and Jannin, Pierre and Kahn, Charles E. and Kainmueller, Dagmar and Kainz, Bernhard and Karargyris, Alexandros and Karthikesalingam, Alan and Kofler, Florian and Kopp-Schneider, Annette and Kreshuk, Anna and Kurc, Tahsin and Landman, Bennett A. and Litjens, Geert and Madani, Amin and Maier-Hein, Klaus and Martel, Anne L. and Mattson, Peter and Meijering, Erik and Menze, Bjoern and Moons, Karel G. M. and Müller, Henning and Nichyporuk, Brennan and Nickel, Felix and Petersen, Jens and Rajpoot, Nasir and Rieke, Nicola and Saez-Rodriguez, Julio and Sánchez, Clara I. and Shetty, Shravya and van Smeden, Maarten and Summers, Ronald M. and Taha, Abdel A. and Tiulpin, Aleksei and Tsaftaris, Sotirios A. and Van Calster, Ben and Varoquaux, Gaël and Jäger, Paul F.},
  doi          = {10.1038/s41592-023-02151-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {195-212},
  shortjournal = {Nat. Meth.},
  title        = {Metrics reloaded: Recommendations for image analysis validation},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Understanding metric-related pitfalls in image analysis
validation. <em>NMETH</em>, <em>21</em>(2), 182–194. (<a
href="https://doi.org/10.1038/s41592-023-02150-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Validation metrics are key for tracking scientific progress and bridging the current chasm between artificial intelligence research and its translation into practice. However, increasing evidence shows that, particularly in image analysis, metrics are often chosen inadequately. Although taking into account the individual strengths, weaknesses and limitations of validation metrics is a critical prerequisite to making educated choices, the relevant knowledge is currently scattered and poorly accessible to individual researchers. Based on a multistage Delphi process conducted by a multidisciplinary expert consortium as well as extensive community feedback, the present work provides a reliable and comprehensive common point of access to information on pitfalls related to validation metrics in image analysis. Although focused on biomedical image analysis, the addressed pitfalls generalize across application domains and are categorized according to a newly created, domain-agnostic taxonomy. The work serves to enhance global comprehension of a key topic in image analysis validation. This Perspective presents a reliable and comprehensive source of information on pitfalls related to validation metrics in image analysis, with an emphasis on biomedical imaging.},
  archive      = {J_NMETH},
  author       = {Reinke, Annika and Tizabi, Minu D. and Baumgartner, Michael and Eisenmann, Matthias and Heckmann-Nötzel, Doreen and Kavur, A. Emre and Rädsch, Tim and Sudre, Carole H. and Acion, Laura and Antonelli, Michela and Arbel, Tal and Bakas, Spyridon and Benis, Arriel and Buettner, Florian and Cardoso, M. Jorge and Cheplygina, Veronika and Chen, Jianxu and Christodoulou, Evangelia and Cimini, Beth A. and Farahani, Keyvan and Ferrer, Luciana and Galdran, Adrian and van Ginneken, Bram and Glocker, Ben and Godau, Patrick and Hashimoto, Daniel A. and Hoffman, Michael M. and Huisman, Merel and Isensee, Fabian and Jannin, Pierre and Kahn, Charles E. and Kainmueller, Dagmar and Kainz, Bernhard and Karargyris, Alexandros and Kleesiek, Jens and Kofler, Florian and Kooi, Thijs and Kopp-Schneider, Annette and Kozubek, Michal and Kreshuk, Anna and Kurc, Tahsin and Landman, Bennett A. and Litjens, Geert and Madani, Amin and Maier-Hein, Klaus and Martel, Anne L. and Meijering, Erik and Menze, Bjoern and Moons, Karel G. M. and Müller, Henning and Nichyporuk, Brennan and Nickel, Felix and Petersen, Jens and Rafelski, Susanne M. and Rajpoot, Nasir and Reyes, Mauricio and Riegler, Michael A. and Rieke, Nicola and Saez-Rodriguez, Julio and Sánchez, Clara I. and Shetty, Shravya and Summers, Ronald M. and Taha, Abdel A. and Tiulpin, Aleksei and Tsaftaris, Sotirios A. and Van Calster, Ben and Varoquaux, Gaël and Yaniv, Ziv R. and Jäger, Paul F. and Maier-Hein, Lena},
  doi          = {10.1038/s41592-023-02150-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {182-194},
  shortjournal = {Nat. Meth.},
  title        = {Understanding metric-related pitfalls in image analysis validation},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Community-developed checklists for publishing images and
image analyses. <em>NMETH</em>, <em>21</em>(2), 170–181. (<a
href="https://doi.org/10.1038/s41592-023-01987-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Images document scientific discoveries and are prevalent in modern biomedical research. Microscopy imaging in particular is currently undergoing rapid technological advancements. However, for scientists wishing to publish obtained images and image-analysis results, there are currently no unified guidelines for best practices. Consequently, microscopy images and image data in publications may be unclear or difficult to interpret. Here, we present community-developed checklists for preparing light microscopy images and describing image analyses for publications. These checklists offer authors, readers and publishers key recommendations for image formatting and annotation, color selection, data availability and reporting image-analysis workflows. The goal of our guidelines is to increase the clarity and reproducibility of image figures and thereby to heighten the quality and explanatory power of microscopy data. Community-developed checklists offer best-practice guidance for biologists preparing light microscopy images and describing image analyses for publications.},
  archive      = {J_NMETH},
  author       = {Schmied, Christopher and Nelson, Michael S. and Avilov, Sergiy and Bakker, Gert-Jan and Bertocchi, Cristina and Bischof, Johanna and Boehm, Ulrike and Brocher, Jan and Carvalho, Mariana T. and Chiritescu, Catalin and Christopher, Jana and Cimini, Beth A. and Conde-Sousa, Eduardo and Ebner, Michael and Ecker, Rupert and Eliceiri, Kevin and Fernandez-Rodriguez, Julia and Gaudreault, Nathalie and Gelman, Laurent and Grunwald, David and Gu, Tingting and Halidi, Nadia and Hammer, Mathias and Hartley, Matthew and Held, Marie and Jug, Florian and Kapoor, Varun and Koksoy, Ayse Aslihan and Lacoste, Judith and Le Dévédec, Sylvia and Le Guyader, Sylvie and Liu, Penghuan and Martins, Gabriel G. and Mathur, Aastha and Miura, Kota and Montero Llopis, Paula and Nitschke, Roland and North, Alison and Parslow, Adam C. and Payne-Dwyer, Alex and Plantard, Laure and Ali, Rizwan and Schroth-Diez, Britta and Schütz, Lucas and Scott, Ryan T. and Seitz, Arne and Selchow, Olaf and Sharma, Ved P. and Spitaler, Martin and Srinivasan, Sathya and Strambio-De-Castillia, Caterina and Taatjes, Douglas and Tischer, Christian and Jambor, Helena Klara},
  doi          = {10.1038/s41592-023-01987-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {170-181},
  shortjournal = {Nat. Meth.},
  title        = {Community-developed checklists for publishing images and image analyses},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Increasing the analytic power for multi-batch proteome
profiling with isobaric mass tags. <em>NMETH</em>, <em>21</em>(2),
168–169. (<a href="https://doi.org/10.1038/s41592-023-02121-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have developed a framework for the analysis of multi-batch proteome profiling data using isobaric mass tags. Our framework improves quantitative accuracy and increases statistical power by accounting for known sources of variation between batches, thus enabling multiplexed proteome profiling analysis to be performed on large numbers of samples and population cohorts.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02121-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {168-169},
  shortjournal = {Nat. Meth.},
  title        = {Increasing the analytic power for multi-batch proteome profiling with isobaric mass tags},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SEVtras characterizes cell-type-specific small extracellular
vesicle secretion. <em>NMETH</em>, <em>21</em>(2), 166–167. (<a
href="https://doi.org/10.1038/s41592-023-02118-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although single-cell RNA-sequencing has revolutionized biomedical research, exploring cell states from an extracellular vesicle viewpoint has remained elusive. We present an algorithm, SEVtras, that accurately captures signals from small extracellular vesicles and determines source cell-type secretion activity. SEVtras unlocks an extracellular dimension for single-cell analysis with diagnostic potential.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02118-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {166-167},
  shortjournal = {Nat. Meth.},
  title        = {SEVtras characterizes cell-type-specific small extracellular vesicle secretion},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ARTR-seq for determining RNA-binding protein target sites
through in situ reverse transcription. <em>NMETH</em>, <em>21</em>(2),
164–165. (<a href="https://doi.org/10.1038/s41592-023-02147-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {ARTR-seq uses antibody-guided in situ reverse transcription to efficiently and accurately identify RNA-binding protein target sites in as few as 20 cells, or in a formaldehyde-fixed tissue section. The high temporal resolution of ARTR-seq opens opportunities for the investigation of dynamic RNA-binding protein–RNA interactions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02147-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {164-165},
  shortjournal = {Nat. Meth.},
  title        = {ARTR-seq for determining RNA-binding protein target sites through in situ reverse transcription},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024m). Inside the chase after those elusive proteoforms.
<em>NMETH</em>, <em>21</em>(2), 158–163. (<a
href="https://doi.org/10.1038/s41592-024-02170-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human cells contain crowds of protein variants, but, especially in a time of funding challenges, chasing these proteoforms takes dogged persistence.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-024-02170-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {158-163},
  shortjournal = {Nat. Meth.},
  title        = {Inside the chase after those elusive proteoforms},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024j). Spotting t and b cell receptors. <em>NMETH</em>,
<em>21</em>(2), 157. (<a
href="https://doi.org/10.1038/s41592-024-02194-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-024-02194-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {157},
  shortjournal = {Nat. Meth.},
  title        = {Spotting t and b cell receptors},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024n). Sharpen SV detection. <em>NMETH</em>, <em>21</em>(2), 157.
(<a href="https://doi.org/10.1038/s41592-024-02195-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-024-02195-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {157},
  shortjournal = {Nat. Meth.},
  title        = {Sharpen SV detection},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Development of the human head. <em>NMETH</em>,
<em>21</em>(2), 156. (<a
href="https://doi.org/10.1038/s41592-024-02192-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-024-02192-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {156},
  shortjournal = {Nat. Meth.},
  title        = {Development of the human head},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024k). More deaminases towards precise editing. <em>NMETH</em>,
<em>21</em>(2), 156. (<a
href="https://doi.org/10.1038/s41592-024-02193-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-024-02193-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {156},
  shortjournal = {Nat. Meth.},
  title        = {More deaminases towards precise editing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). RepoRT: A comprehensive repository for small molecule
retention times. <em>NMETH</em>, <em>21</em>(2), 153–155. (<a
href="https://doi.org/10.1038/s41592-023-02143-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kretschmer, Fleming and Harrieder, Eva-Maria and Hoffmann, Martin A. and Böcker, Sebastian and Witting, Michael},
  doi          = {10.1038/s41592-023-02143-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {153-155},
  shortjournal = {Nat. Meth.},
  title        = {RepoRT: A comprehensive repository for small molecule retention times},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024j). From zero to hero in budget-making. <em>NMETH</em>,
<em>21</em>(2), 152. (<a
href="https://doi.org/10.1038/s41592-023-02153-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The grant deadline approaches but budget details are still missing. Experienced budget-makers share how they manage budgets and the ‘boom–bust’ cycle.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-02153-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {152},
  shortjournal = {Nat. Meth.},
  title        = {From zero to hero in budget-making},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Where imaging and metrics meet. <em>NMETH</em>,
<em>21</em>(2), 151. (<a
href="https://doi.org/10.1038/s41592-024-02187-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When it comes to bioimaging and image analysis, details matter. Papers in this issue offer guidance for improved robustness and reproducibility.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02187-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {151},
  shortjournal = {Nat. Meth.},
  title        = {Where imaging and metrics meet},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Automated neuron tracking inside moving and deforming
c. Elegans using deep learning and targeted augmentation.
<em>NMETH</em>, <em>21</em>(1), 142–149. (<a
href="https://doi.org/10.1038/s41592-023-02096-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reading out neuronal activity from three-dimensional (3D) functional imaging requires segmenting and tracking individual neurons. This is challenging in behaving animals if the brain moves and deforms. The traditional approach is to train a convolutional neural network with ground-truth (GT) annotations of images representing different brain postures. For 3D images, this is very labor intensive. We introduce ‘targeted augmentation’, a method to automatically synthesize artificial annotations from a few manual annotations. Our method (‘Targettrack’) learns the internal deformations of the brain to synthesize annotations for new postures by deforming GT annotations. This reduces the need for manual annotation and proofreading. A graphical user interface allows the application of the method end-to-end. We demonstrate Targettrack on recordings where neurons are labeled as key points or 3D volumes. Analyzing freely moving animals exposed to odor pulses, we uncover rich patterns in interneuron dynamics, including switching neuronal entrainment on and off. Targettrack is a deep-learning-based pipeline for automatic tracking of neurons within freely moving C. elegans. Using targeted augmentation, the pipeline has a reduced need for manually annotated training data.},
  archive      = {J_NMETH},
  author       = {Park, Core Francisco and Barzegar-Keshteli, Mahsa and Korchagina, Kseniia and Delrocq, Ariane and Susoy, Vladislav and Jones, Corinne L. and Samuel, Aravinthan D. T. and Rahi, Sahand Jamal},
  doi          = {10.1038/s41592-023-02096-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {142-149},
  shortjournal = {Nat. Meth.},
  title        = {Automated neuron tracking inside moving and deforming c. elegans using deep learning and targeted augmentation},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The cousa objective: A long-working distance air objective
for multiphoton imaging in vivo. <em>NMETH</em>, <em>21</em>(1),
132–141. (<a href="https://doi.org/10.1038/s41592-023-02098-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiphoton microscopy can resolve fluorescent structures and dynamics deep in scattering tissue and has transformed neural imaging, but applying this technique in vivo can be limited by the mechanical and optical constraints of conventional objectives. Short working distance objectives can collide with compact surgical windows or other instrumentation and preclude imaging. Here we present an ultra-long working distance (20 mm) air objective called the Cousa objective. It is optimized for performance across multiphoton imaging wavelengths, offers a more than 4 mm2 field of view with submicrometer lateral resolution and is compatible with commonly used multiphoton imaging systems. A novel mechanical design, wider than typical microscope objectives, enabled this combination of specifications. We share the full optical prescription, and report performance including in vivo two-photon and three-photon imaging in an array of species and preparations, including nonhuman primates. The Cousa objective can enable a range of experiments in neuroscience and beyond. The Cousa objective is an ultra-long working distance air objective optimized for two- and three-photon imaging. Bypassing challenges caused by water immersion and short working distances, the Cousa enables and improves imaging of diverse specimens.},
  archive      = {J_NMETH},
  author       = {Yu, Che-Hang and Yu, Yiyi and Adsit, Liam M. and Chang, Jeremy T. and Barchini, Jad and Moberly, Andrew H. and Benisty, Hadas and Kim, Jinkyung and Young, Brent K. and Heng, Kathleen and Farinella, Deano M. and Leikvoll, Austin and Pavan, Rishaab and Vistein, Rachel and Nanfito, Brandon R. and Hildebrand, David G. C. and Otero-Coronel, Santiago and Vaziri, Alipasha and Goldberg, Jeffrey L. and Ricci, Anthony J. and Fitzpatrick, David and Cardin, Jessica A. and Higley, Michael J. and Smith, Gordon B. and Kara, Prakash and Nielsen, Kristina J. and Smith, Ikuko T. and Smith, Spencer LaVere},
  doi          = {10.1038/s41592-023-02098-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {132-141},
  shortjournal = {Nat. Meth.},
  title        = {The cousa objective: A long-working distance air objective for multiphoton imaging in vivo},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DeepMainmast: Integrated protocol of protein structure
modeling for cryo-EM with deep learning and structure prediction.
<em>NMETH</em>, <em>21</em>(1), 122–131. (<a
href="https://doi.org/10.1038/s41592-023-02099-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Three-dimensional structure modeling from maps is an indispensable step for studying proteins and their complexes with cryogenic electron microscopy. Although the resolution of determined cryogenic electron microscopy maps has generally improved, there are still many cases where tracing protein main chains is difficult, even in maps determined at a near-atomic resolution. Here we developed a protein structure modeling method, DeepMainmast, which employs deep learning to capture the local map features of amino acids and atoms to assist main-chain tracing. Moreover, we integrated AlphaFold2 with the de novo density tracing protocol to combine their complementary strengths and achieved even higher accuracy than each method alone. Additionally, the protocol is able to accurately assign the chain identity to the structure models of homo-multimers, which is not a trivial task for existing methods. DeepMainmast is a protein structure modeling protocol for cryo-EM that combines the strengths of a deep-learning-based de novo protein main-chain-tracing approach with AlphaFold2-based structure predictions for improved performance.},
  archive      = {J_NMETH},
  author       = {Terashi, Genki and Wang, Xiao and Prasad, Devashish and Nakamura, Tsukasa and Kihara, Daisuke},
  doi          = {10.1038/s41592-023-02099-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {122-131},
  shortjournal = {Nat. Meth.},
  title        = {DeepMainmast: Integrated protocol of protein structure modeling for cryo-EM with deep learning and structure prediction},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accurate prediction of protein–nucleic acid complexes using
RoseTTAFoldNA. <em>NMETH</em>, <em>21</em>(1), 117–121. (<a
href="https://doi.org/10.1038/s41592-023-02086-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein–RNA and protein–DNA complexes play critical roles in biology. Despite considerable recent advances in protein structure prediction, the prediction of the structures of protein–nucleic acid complexes without homology to known complexes is a largely unsolved problem. Here we extend the RoseTTAFold machine learning protein-structure-prediction approach to additionally predict nucleic acid and protein–nucleic acid complexes. We develop a single trained network, RoseTTAFoldNA, that rapidly produces three-dimensional structure models with confidence estimates for protein–DNA and protein–RNA complexes. Here we show that confident predictions have considerably higher accuracy than current state-of-the-art methods. RoseTTAFoldNA should be broadly useful for modeling the structure of naturally occurring protein–nucleic acid complexes, and for designing sequence-specific RNA and DNA-binding proteins. RoseTTAFoldNA extends the RoseTTAFold2 platform to predict the structures of protein–DNA and protein–RNA complexes.},
  archive      = {J_NMETH},
  author       = {Baek, Minkyung and McHugh, Ryan and Anishchenko, Ivan and Jiang, Hanlun and Baker, David and DiMaio, Frank},
  doi          = {10.1038/s41592-023-02086-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {117-121},
  shortjournal = {Nat. Meth.},
  title        = {Accurate prediction of protein–nucleic acid complexes using RoseTTAFoldNA},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). AlphaFold predictions are valuable hypotheses and accelerate
but do not replace experimental structure determination. <em>NMETH</em>,
<em>21</em>(1), 110–116. (<a
href="https://doi.org/10.1038/s41592-023-02087-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence-based protein structure prediction methods such as AlphaFold have revolutionized structural biology. The accuracies of these predictions vary, however, and they do not take into account ligands, covalent modifications or other environmental factors. Here, we evaluate how well AlphaFold predictions can be expected to describe the structure of a protein by comparing predictions directly with experimental crystallographic maps. In many cases, AlphaFold predictions matched experimental maps remarkably closely. In other cases, even very high-confidence predictions differed from experimental maps on a global scale through distortion and domain orientation, and on a local scale in backbone and side-chain conformation. We suggest considering AlphaFold predictions as exceptionally useful hypotheses. We further suggest that it is important to consider the confidence in prediction when interpreting AlphaFold predictions and to carry out experimental structure determination to verify structural details, particularly those that involve interactions not included in the prediction. An analysis of AlphaFold protein structure predictions shows that while in many cases the predictions are highly accurate, there are also many instances where the predicted structures or parts of predicted structures do not agree with experimentally resolved data. Therefore, care must be taken when using these predictions for informing structural hypotheses.},
  archive      = {J_NMETH},
  author       = {Terwilliger, Thomas C. and Liebschner, Dorothee and Croll, Tristan I. and Williams, Christopher J. and McCoy, Airlie J. and Poon, Billy K. and Afonine, Pavel V. and Oeffner, Robert D. and Richardson, Jane S. and Read, Randy J. and Adams, Paul D.},
  doi          = {10.1038/s41592-023-02087-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {110-116},
  shortjournal = {Nat. Meth.},
  title        = {AlphaFold predictions are valuable hypotheses and accelerate but do not replace experimental structure determination},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Peptide sequencing based on host–guest interaction-assisted
nanopore sensing. <em>NMETH</em>, <em>21</em>(1), 102–109. (<a
href="https://doi.org/10.1038/s41592-023-02095-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Direct protein sequencing technologies with improved sensitivity and throughput are still needed. Here, we propose an alternative method for peptide sequencing based on enzymatic cleavage and host–guest interaction-assisted nanopore sensing. We serendipitously discovered that the identity of any proteinogenic amino acid in a particular position of a phenylalanine-containing peptide could be determined via current blockage during translocation of the peptide through α-hemolysin nanopores in the presence of cucurbit[7]uril. Building upon this, we further present a proof-of-concept demonstration of peptide sequencing by sequentially cleaving off amino acids from C terminus of a peptide with carboxypeptidases, and then determining their identities and sequence with a peptide probe in nanopore. With future optimization, our results point to a different way of nanopore-based protein sequencing. A phenylalanine-containing peptide probe can be used for discriminating all 20 amino acids via current blockage during translocation through an α-hemolysin (αHL) nanopore. The paper provides proof-of-concept peptide sequencing demonstrations.},
  archive      = {J_NMETH},
  author       = {Zhang, Yun and Yi, Yakun and Li, Ziyi and Zhou, Ke and Liu, Lei and Wu, Hai-Chen},
  doi          = {10.1038/s41592-023-02095-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {102-109},
  shortjournal = {Nat. Meth.},
  title        = {Peptide sequencing based on host–guest interaction-assisted nanopore sensing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unambiguous discrimination of all 20 proteinogenic amino
acids and their modifications by nanopore. <em>NMETH</em>,
<em>21</em>(1), 92–101. (<a
href="https://doi.org/10.1038/s41592-023-02021-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural proteins are composed of 20 proteinogenic amino acids and their post-translational modifications (PTMs). However, due to the lack of a suitable nanopore sensor that can simultaneously discriminate between all 20 amino acids and their PTMs, direct sequencing of protein with nanopores has not yet been realized. Here, we present an engineered hetero-octameric Mycobacterium smegmatis porin A (MspA) nanopore containing a sole Ni2+ modification. It enables full discrimination of all 20 proteinogenic amino acids and 4 representative modified amino acids, Nω,N’ω-dimethyl-arginine (Me-R), O-acetyl-threonine (Ac-T), N4-(β-N-acetyl-d-glucosaminyl)-asparagine (GlcNAc-N) and O-phosphoserine (P-S). Assisted by machine learning, an accuracy of 98.6% was achieved. Amino acid supplement tablets and peptidase-digested amino acids from peptides were also analyzed using this strategy. This capacity for simultaneous discrimination of all 20 proteinogenic amino acids and their PTMs suggests the potential to achieve protein sequencing using this nanopore-based strategy. A Ni2+-modified MspA nanopore construct can unambiguously discriminate the 20 proteogenic amino acids as well as several post-translational modifications.},
  archive      = {J_NMETH},
  author       = {Wang, Kefan and Zhang, Shanyu and Zhou, Xiao and Yang, Xian and Li, Xinyue and Wang, Yuqin and Fan, Pingping and Xiao, Yunqi and Sun, Wen and Zhang, Panke and Li, Wenfei and Huang, Shuo},
  doi          = {10.1038/s41592-023-02021-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {92-101},
  shortjournal = {Nat. Meth.},
  title        = {Unambiguous discrimination of all 20 proteinogenic amino acids and their modifications by nanopore},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multigroup analysis of compositions of microbiomes with
covariate adjustments and repeated measures. <em>NMETH</em>,
<em>21</em>(1), 83–91. (<a
href="https://doi.org/10.1038/s41592-023-02092-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbiome differential abundance analysis methods for two groups are well-established in the literature. However, many microbiome studies involve more than two groups, sometimes even ordered groups such as stages of a disease, and require different types of comparison. Standard pairwise comparisons are inefficient in terms of power and false discovery rates. In this Article, we propose a general framework, ANCOM-BC2, for performing a wide range of multigroup analyses with covariate adjustments and repeated measures. We illustrate our methodology through two real datasets. The first example explores the effects of aridity on the soil microbiome, and the second example investigates the effects of surgical interventions on the microbiome of patients with inflammatory bowel disease. ANCOM-BC2 is developed to perform multigroup differential abundance analysis and allows modeling of covariates and longitudinal measures while controlling false discovery rate (FDR) or mixed directional FDR.},
  archive      = {J_NMETH},
  author       = {Lin, Huang and Peddada, Shyamal Das},
  doi          = {10.1038/s41592-023-02092-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {83-91},
  shortjournal = {Nat. Meth.},
  title        = {Multigroup analysis of compositions of microbiomes with covariate adjustments and repeated measures},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Combinatorial single-cell profiling of major chromatin types
with MAbID. <em>NMETH</em>, <em>21</em>(1), 72–82. (<a
href="https://doi.org/10.1038/s41592-023-02090-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression programs result from the collective activity of numerous regulatory factors. Studying their cooperative mode of action is imperative to understand gene regulation, but simultaneously measuring these factors within one sample has been challenging. Here we introduce Multiplexing Antibodies by barcode Identification (MAbID), a method for combinatorial genomic profiling of histone modifications and chromatin-binding proteins. MAbID employs antibody–DNA conjugates to integrate barcodes at the genomic location of the epitope, enabling combined incubation of multiple antibodies to reveal the distributions of many epigenetic markers simultaneously. We used MAbID to profile major chromatin types and multiplexed measurements without loss of individual data quality. Moreover, we obtained joint measurements of six epitopes in single cells of mouse bone marrow and during mouse in vitro differentiation, capturing associated changes in multifactorial chromatin states. Thus, MAbID holds the potential to gain unique insights into the interplay between gene regulatory mechanisms, especially for low-input samples and in single cells. MAbID offers a multiplexing approach to uncover the genomic distributions of various epigenetic markers, enabling the study of how these markers jointly direct gene expression.},
  archive      = {J_NMETH},
  author       = {Lochs, Silke J. A. and van der Weide, Robin H. and de Luca, Kim L. and Korthout, Tessy and van Beek, Ramada E. and Kimura, Hiroshi and Kind, Jop},
  doi          = {10.1038/s41592-023-02090-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {72-82},
  shortjournal = {Nat. Meth.},
  title        = {Combinatorial single-cell profiling of major chromatin types with MAbID},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). System-wide analysis of RNA and protein subcellular
localization dynamics. <em>NMETH</em>, <em>21</em>(1), 60–71. (<a
href="https://doi.org/10.1038/s41592-023-02101-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although the subcellular dynamics of RNA and proteins are key determinants of cell homeostasis, their characterization is still challenging. Here we present an integrative framework to simultaneously interrogate the dynamics of the transcriptome and proteome at subcellular resolution by combining two methods: localization of RNA (LoRNA) and a streamlined density-based localization of proteins by isotope tagging (dLOPIT) to map RNA and protein to organelles (nucleus, endoplasmic reticulum and mitochondria) and membraneless compartments (cytosol, nucleolus and cytosolic granules). Interrogating all RNA subcellular locations at once enables system-wide quantification of the proportional distribution of RNA. We obtain a cell-wide overview of localization dynamics for 31,839 transcripts and 5,314 proteins during the unfolded protein response, revealing that endoplasmic reticulum-localized transcripts are more efficiently recruited to cytosolic granules than cytosolic RNAs, and that the translation initiation factor eIF3d is key to sustaining cytoskeletal function. Overall, we provide the most comprehensive overview so far of RNA and protein subcellular localization dynamics. An integrative framework to simultaneously interrogate the dynamics of the transcriptome and proteome at subcellular resolution that combines two methods, localization of RNA (LoRNA) and a streamlined density-based localization of proteins by isotope tagging (dLOPIT).},
  archive      = {J_NMETH},
  author       = {Villanueva, Eneko and Smith, Tom and Pizzinga, Mariavittoria and Elzek, Mohamed and Queiroz, Rayner M. L. and Harvey, Robert F. and Breckels, Lisa M. and Crook, Oliver M. and Monti, Mie and Dezi, Veronica and Willis, Anne E. and Lilley, Kathryn S.},
  doi          = {10.1038/s41592-023-02101-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {60-71},
  shortjournal = {Nat. Meth.},
  title        = {System-wide analysis of RNA and protein subcellular localization dynamics},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep generative modeling of transcriptional dynamics for RNA
velocity analysis in single cells. <em>NMETH</em>, <em>21</em>(1),
50–59. (<a href="https://doi.org/10.1038/s41592-023-01994-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA velocity has been rapidly adopted to guide interpretation of transcriptional dynamics in snapshot single-cell data; however, current approaches for estimating RNA velocity lack effective strategies for quantifying uncertainty and determining the overall applicability to the system of interest. Here, we present veloVI (velocity variational inference), a deep generative modeling framework for estimating RNA velocity. veloVI learns a gene-specific dynamical model of RNA metabolism and provides a transcriptome-wide quantification of velocity uncertainty. We show that veloVI compares favorably to previous approaches with respect to goodness of fit, consistency across transcriptionally similar cells and stability across preprocessing pipelines for quantifying RNA abundance. Further, we demonstrate that veloVI’s posterior velocity uncertainty can be used to assess whether velocity analysis is appropriate for a given dataset. Finally, we highlight veloVI as a flexible framework for modeling transcriptional dynamics by adapting the underlying dynamical model to use time-dependent transcription rates. veloVI enhances RNA velocity analysis with uncertainty quantification and extensibility by deep generative modeling of gene-specific transcriptional dynamics.},
  archive      = {J_NMETH},
  author       = {Gayoso, Adam and Weiler, Philipp and Lotfollahi, Mohammad and Klein, Dominik and Hong, Justin and Streets, Aaron and Theis, Fabian J. and Yosef, Nir},
  doi          = {10.1038/s41592-023-01994-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {50-59},
  shortjournal = {Nat. Meth.},
  title        = {Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improved sequence mapping using a complete reference genome
and lift-over. <em>NMETH</em>, <em>21</em>(1), 41–49. (<a
href="https://doi.org/10.1038/s41592-023-02069-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complete, telomere-to-telomere (T2T) genome assemblies promise improved analyses and the discovery of new variants, but many essential genomic resources remain associated with older reference genomes. Thus, there is a need to translate genomic features and read alignments between references. Here we describe a method called levioSAM2 that performs fast and accurate lift-over between assemblies using a whole-genome map. In addition to enabling the use of several references, we demonstrate that aligning reads to a high-quality reference (for example, T2T-CHM13) and lifting to an older reference (for example, Genome reference Consortium (GRC)h38) improves the accuracy of the resulting variant calls on the old reference. By leveraging the quality improvements of T2T-CHM13, levioSAM2 reduces small and structural variant calling errors compared with GRC-based mapping using real short- and long-read datasets. Performance is especially improved for a set of complex medically relevant genes, where the GRC references are lower quality. By combining fast lift-over and selective re-mapping, levioSAM2 enables efficient and accurate read mapping and variant calling leveraging complete reference genomes.},
  archive      = {J_NMETH},
  author       = {Chen, Nae-Chyun and Paulin, Luis F. and Sedlazeck, Fritz J. and Koren, Sergey and Phillippy, Adam M. and Langmead, Ben},
  doi          = {10.1038/s41592-023-02069-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {41-49},
  shortjournal = {Nat. Meth.},
  title        = {Improved sequence mapping using a complete reference genome and lift-over},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving resolution and resolvability of single-particle
cryoEM structures using gaussian mixture models. <em>NMETH</em>,
<em>21</em>(1), 37–40. (<a
href="https://doi.org/10.1038/s41592-023-02082-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron microscopy is widely used in structural biology, but its resolution is often limited by the dynamics of the macromolecule. Here we developed a refinement protocol based on Gaussian mixture models that integrates particle orientation and conformation estimation and improves the alignment for flexible domains of protein structures. We demonstrated this protocol on multiple datasets, resulting in improved resolution and resolvability, locally and globally, by visual and quantitative measures. This manuscript describes a refinement protocol that extends the e2gmm method to optimize both the orientation and conformation estimation of particles to improve the alignment for flexible domains of proteins.},
  archive      = {J_NMETH},
  author       = {Chen, Muyuan and Schmid, Michael F. and Chiu, Wah},
  doi          = {10.1038/s41592-023-02082-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {37-40},
  shortjournal = {Nat. Meth.},
  title        = {Improving resolution and resolvability of single-particle cryoEM structures using gaussian mixture models},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Uniform quantification of single-nucleus ATAC-seq data with
paired-insertion counting (PIC) and a model-based insertion rate
estimator. <em>NMETH</em>, <em>21</em>(1), 32–36. (<a
href="https://doi.org/10.1038/s41592-023-02103-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing approaches to scoring single-nucleus assay for transposase-accessible chromatin with sequencing (snATAC-seq) feature matrices from sequencing reads are inconsistent, affecting downstream analyses and displaying artifacts. We show that, even with sparse single-cell data, quantitative counts are informative for estimating the regulatory state of a cell, which calls for a consistent treatment. We propose Paired-Insertion Counting as a uniform method for snATAC-seq feature characterization and provide a probability model for inferring latent insertion dynamics from snATAC-seq count matrices. This study demonstrates the need and advantage of uniformly quantifying single-nucleus ATAC-seq data using Paired-Insertion Counting.},
  archive      = {J_NMETH},
  author       = {Miao, Zhen and Kim, Junhyong},
  doi          = {10.1038/s41592-023-02103-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {32-36},
  shortjournal = {Nat. Meth.},
  title        = {Uniform quantification of single-nucleus ATAC-seq data with paired-insertion counting (PIC) and a model-based insertion rate estimator},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Modeling fragment counts improves single-cell ATAC-seq
analysis. <em>NMETH</em>, <em>21</em>(1), 28–31. (<a
href="https://doi.org/10.1038/s41592-023-02112-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell ATAC sequencing coverage in regulatory regions is typically binarized as an indicator of open chromatin. Here we show that binarization is an unnecessary step that neither improves goodness of fit, clustering, cell type identification nor batch integration. Fragment counts, but not read counts, should instead be modeled, which preserves quantitative regulatory information. These results have immediate implications for single-cell ATAC sequencing analysis. This paper proposes quantitative modeling of single-cell ATAC-seq data, which improves various downstream analyses.},
  archive      = {J_NMETH},
  author       = {Martens, Laura D. and Fischer, David S. and Yépez, Vicente A. and Theis, Fabian J. and Gagneur, Julien},
  doi          = {10.1038/s41592-023-02112-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {28-31},
  shortjournal = {Nat. Meth.},
  title        = {Modeling fragment counts improves single-cell ATAC-seq analysis},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inferring how animals deform improves cell tracking.
<em>NMETH</em>, <em>21</em>(1), 26–27. (<a
href="https://doi.org/10.1038/s41592-023-02097-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tracking cells is a time-consuming part of biological image analysis, and traditional manual annotation methods are prohibitively laborious for tracking neurons in the deforming and moving Caenorhabditis elegans brain. By leveraging machine learning to develop a ‘targeted augmentation’ method, we substantially reduced the number of labeled images required for tracking.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02097-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {26-27},
  shortjournal = {Nat. Meth.},
  title        = {Inferring how animals deform improves cell tracking},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ultra-long-working-distance multiphoton objective unlocks
new possibilities for imaging. <em>NMETH</em>, <em>21</em>(1), 24–25.
(<a href="https://doi.org/10.1038/s41592-023-02116-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In 1858, the first standard for microscope objectives was established to encourage interchangeable components. Over the following 150 years, standards have evolved to constrain the size of objectives, which limits the parameters of working distance, field of view and resolution. A new design breaks out of this conventional envelope, offering an ultra-long working distance in air and enabling new neuroscience experiments.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02116-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {24-25},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-long-working-distance multiphoton objective unlocks new possibilities for imaging},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Using machine learning to predict the structure of proteins
that bind to DNA and RNA. <em>NMETH</em>, <em>21</em>(1), 22–23. (<a
href="https://doi.org/10.1038/s41592-023-02088-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a machine learning model, RoseTTAFoldNA, that can predict the structures of protein–DNA and protein–RNA complexes. Our model is capable of predicting accurate structures of protein families for which structural information is unknown.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02088-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {22-23},
  shortjournal = {Nat. Meth.},
  title        = {Using machine learning to predict the structure of proteins that bind to DNA and RNA},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multifactorial epigenomic profiling of six chromatin states
in single cells. <em>NMETH</em>, <em>21</em>(1), 20–21. (<a
href="https://doi.org/10.1038/s41592-023-02091-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed MAbID, a method for combined genomic profiling of histone modifications and chromatin-binding proteins in single cells, enabling researchers to study the interconnectivity between gene-regulatory mechanisms. We demonstrated MAbID’s implementation in profiling multifactorial changes in chromatin signatures during in vitro neural differentiation and in primary mouse bone marrow tissue.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02091-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {20-21},
  shortjournal = {Nat. Meth.},
  title        = {Multifactorial epigenomic profiling of six chromatin states in single cells},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Structural biology in the age of AI. <em>NMETH</em>,
<em>21</em>(1), 18–19. (<a
href="https://doi.org/10.1038/s41592-023-02123-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How accurate is the prediction of protein structure by AlphaFold? Terwilliger et al. address this question with a rigorous assessment of the accuracy of AlphaFold-predicted structures by comparing them with experimentally determined X-ray crystallographic data.},
  archive      = {J_NMETH},
  author       = {Subramaniam, Sriram},
  doi          = {10.1038/s41592-023-02123-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {18-19},
  shortjournal = {Nat. Meth.},
  title        = {Structural biology in the age of AI},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Engineered nanopores for exopeptidase protein sequencing.
<em>NMETH</em>, <em>21</em>(1), 16–17. (<a
href="https://doi.org/10.1038/s41592-023-02136-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two studies show that nanopores can identify the 20 proteinogenic amino acids and some of their post-translational modifications. Coupled with an exopeptidase, a bottom-up approach to protein sequencing using nanopores is on the horizon.},
  archive      = {J_NMETH},
  author       = {Bonini, Andrea and Sauciuc, Adina and Maglia, Giovanni},
  doi          = {10.1038/s41592-023-02136-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {16-17},
  shortjournal = {Nat. Meth.},
  title        = {Engineered nanopores for exopeptidase protein sequencing},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Aging research comes of age. <em>NMETH</em>,
<em>21</em>(1), 11–15. (<a
href="https://doi.org/10.1038/s41592-023-02140-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As money pours into aging research, the field can combine its many methods to home in on what underpins aging. Approaches differ, but researchers share the desire to not overpromise quick-fix anti-aging methods.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-02140-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {11-15},
  shortjournal = {Nat. Meth.},
  title        = {Aging research comes of age},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Chroma is a generative model for protein design.
<em>NMETH</em>, <em>21</em>(1), 10. (<a
href="https://doi.org/10.1038/s41592-023-02155-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-02155-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {10},
  shortjournal = {Nat. Meth.},
  title        = {Chroma is a generative model for protein design},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). CD4 t cell antigen discovery. <em>NMETH</em>,
<em>21</em>(1), 10. (<a
href="https://doi.org/10.1038/s41592-023-02156-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-02156-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {10},
  shortjournal = {Nat. Meth.},
  title        = {CD4 t cell antigen discovery},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Predicting neural activity from facial expressions.
<em>NMETH</em>, <em>21</em>(1), 9. (<a
href="https://doi.org/10.1038/s41592-023-02154-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Facemap tracks keypoints on the mouse face and feeds the information into a deep neural network to predict neural activity.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-02154-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nat. Meth.},
  title        = {Predicting neural activity from facial expressions},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). JDLL: A library to run deep learning models on java bioimage
informatics platforms. <em>NMETH</em>, <em>21</em>(1), 7–8. (<a
href="https://doi.org/10.1038/s41592-023-02129-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {García López de Haro, Carlos and Dallongeville, Stéphane and Musset, Thomas and Gómez-de-Mariscal, Estibaliz and Sage, Daniel and Ouyang, Wei and Muñoz-Barrutia, Arrate and Tinevez, Jean-Yves and Olivo-Marin, Jean-Christophe},
  doi          = {10.1038/s41592-023-02129-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {7-8},
  shortjournal = {Nat. Meth.},
  title        = {JDLL: A library to run deep learning models on java bioimage informatics platforms},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Errors in predictor variables. <em>NMETH</em>,
<em>21</em>(1), 4–6. (<a
href="https://doi.org/10.1038/s41592-023-02119-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is the mark of an educated mind to rest satisfied with the degree of precision that the nature of the subject admits and not to seek exactness where only an approximation is possible. —Aristotle},
  archive      = {J_NMETH},
  author       = {Altman, Naomi and Krzywinski, Martin},
  doi          = {10.1038/s41592-023-02119-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {4-6},
  shortjournal = {Nat. Meth.},
  title        = {Errors in predictor variables},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024l). If at first you don’t succeed. <em>NMETH</em>,
<em>21</em>(1), 3. (<a
href="https://doi.org/10.1038/s41592-023-02135-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists have successes to celebrate but must also cope with the sting of failures. In the way she handles both, Nobel laureate Katalin Karikó inspires others.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-02135-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {3},
  shortjournal = {Nat. Meth.},
  title        = {If at first you don’t succeed},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Year in review 2023. <em>NMETH</em>, <em>21</em>(1), 1–2.
(<a href="https://doi.org/10.1038/s41592-023-02158-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As we begin a new year, we reflect on some of our favorites among the papers published in 2023 in Nature Methods.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02158-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {1-2},
  shortjournal = {Nat. Meth.},
  title        = {Year in review 2023},
  volume       = {21},
  year         = {2024},
}
</textarea>
</details></li>
</ul>

</body>
</html>
