<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATMI_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natmi---164">NATMI - 164</h2>
<ul>
<li><details>
<summary>
(2021). Advancing COVID-19 diagnosis with privacy-preserving
collaboration in artificial intelligence. <em>NATMI</em>,
<em>3</em>(12), 1081–1089. (<a
href="https://doi.org/10.1038/s42256-021-00421-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence provides a promising solution for streamlining COVID-19 diagnoses; however, concerns surrounding security and trustworthiness impede the collection of large-scale representative medical data, posing a considerable challenge for training a well-generalized model in clinical practices. To address this, we launch the Unified CT-COVID AI Diagnostic Initiative (UCADI), where the artificial intelligence (AI) model can be distributedly trained and independently executed at each host institution under a federated learning framework without data sharing. Here we show that our federated learning framework model considerably outperformed all of the local models (with a test sensitivity/specificity of 0.973/0.951 in China and 0.730/0.942 in the United Kingdom), achieving comparable performance with a panel of professional radiologists. We further evaluated the model on the hold-out (collected from another two hospitals without the federated learning framework) and heterogeneous (acquired with contrast materials) data, provided visual explanations for decisions made by the model, and analysed the trade-offs between the model performance and the communication costs in the federated training process. Our study is based on 9,573 chest computed tomography scans from 3,336 patients collected from 23 hospitals located in China and the United Kingdom. Collectively, our work advanced the prospects of utilizing federated learning for privacy-preserving AI in digital health. The COVID-19 pandemic sparked the need for international collaboration in using clinical data for rapid development of diagnosis and treatment methods. But the sensitive nature of medical data requires special care and ideally potentially sensitive data would not leave the organization which collected it. Xiang Bai and colleagues present a privacy-preserving AI framework for CT-based COVID-19 diagnosis and demonstrate it on data from 23 hospitals in China and the United Kingdom.},
  archive      = {J_NATMI},
  author       = {Bai, Xiang and Wang, Hanchen and Ma, Liya and Xu, Yongchao and Gan, Jiefeng and Fan, Ziwei and Yang, Fan and Ma, Ke and Yang, Jiehua and Bai, Song and Shu, Chang and Zou, Xinyu and Huang, Renhao and Zhang, Changzheng and Liu, Xiaowu and Tu, Dandan and Xu, Chuou and Zhang, Wenqing and Wang, Xi and Chen, Anguo and Zeng, Yu and Yang, Dehua and Wang, Ming-Wei and Holalkere, Nagaraj and Halin, Neil J. and Kamel, Ihab R. and Wu, Jia and Peng, Xuehua and Wang, Xiang and Shao, Jianbo and Mongkolwat, Pattanasak and Zhang, Jianjun and Liu, Weiyang and Roberts, Michael and Teng, Zhongzhao and Beer, Lucian and Sanchez, Lorena E. and Sala, Evis and Rubin, Daniel L. and Weller, Adrian and Lasenby, Joan and Zheng, Chuansheng and Wang, Jianming and Li, Zhen and Schönlieb, Carola and Xia, Tian},
  doi          = {10.1038/s42256-021-00421-z},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1081-1089},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Advancing COVID-19 diagnosis with privacy-preserving collaboration in artificial intelligence},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Physics-based machine learning for subcellular segmentation
in living cells. <em>NATMI</em>, <em>3</em>(12), 1071–1080. (<a
href="https://doi.org/10.1038/s42256-021-00420-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Segmenting subcellular structures in living cells from fluorescence microscope images is a ground truth (GT)-deficient problem. The microscopes’ three-dimensional blurring function, finite optical resolution due to light diffraction, finite pixel resolution and the complex morphological manifestations of the structures all contribute to GT-hardness. Unsupervised segmentation approaches are quite inaccurate. Therefore, manual segmentation relying on heuristics and experience remains the preferred approach. However, this process is tedious, given the countless structures present inside a single cell, and generating analytics across a large population of cells or performing advanced artificial intelligence tasks such as tracking are greatly limited. Here we bring modelling and deep learning to a nexus for solving this GT-hard problem, improving both the accuracy and speed of subcellular segmentation. We introduce a simulation-supervision approach empowered by physics-based GT, which presents two advantages. First, the physics-based GT resolves the GT-hardness. Second, computational modelling of all the relevant physical aspects assists the deep learning models in learning to compensate, to a great extent, for the limitations of physics and the instrument. We show extensive results on the segmentation of small vesicles and mitochondria in diverse and independent living- and fixed-cell datasets. We demonstrate the adaptability of the approach across diverse microscopes through transfer learning, and illustrate biologically relevant applications of automated analytics and motion analysis. To train deep learning methods to segment very small subcellular structures, the training data have to be labelled by experts as the optical effects at such a small scale and the narrow depth of focus make it difficult to identify individual structures. Sekh et al. use a physics-based simulation approach to train neural networks to automatically segment subcellular structures despite the optical artefacts.},
  archive      = {J_NATMI},
  author       = {Sekh, Arif Ahmed and Opstad, Ida S. and Godtliebsen, Gustav and Birgisdottir, Åsa Birna and Ahluwalia, Balpreet Singh and Agarwal, Krishna and Prasad, Dilip K.},
  doi          = {10.1038/s42256-021-00420-0},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1071-1080},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Physics-based machine learning for subcellular segmentation in living cells},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A case-based interpretable deep learning model for
classification of mass lesions in digital mammography. <em>NATMI</em>,
<em>3</em>(12), 1061–1070. (<a
href="https://doi.org/10.1038/s42256-021-00423-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interpretability in machine learning models is important in high-stakes decisions such as whether to order a biopsy based on a mammographic exam. Mammography poses important challenges that are not present in other computer vision tasks: datasets are small, confounding information is present and it can be difficult even for a radiologist to decide between watchful waiting and biopsy based on a mammogram alone. In this work we present a framework for interpretable machine learning-based mammography. In addition to predicting whether a lesion is malignant or benign, our work aims to follow the reasoning processes of radiologists in detecting clinically relevant semantic features of each image, such as the characteristics of the mass margins. The framework includes a novel interpretable neural network algorithm that uses case-based reasoning for mammography. Our algorithm can incorporate a combination of data with whole image labelling and data with pixel-wise annotations, leading to better accuracy and interpretability even with a small number of images. Our interpretable models are able to highlight the classification-relevant parts of the image, whereas other methods highlight healthy tissue and confounding information. Our models are decision aids—rather than decision makers—and aim for better overall human–machine collaboration. We do not observe a loss in mass margin classification accuracy over a black box neural network trained on the same data. The black-box nature of neural networks is a concern for high-stakes medical applications in which decisions must be based on medically relevant features. The authors develop an interpretable machine learning-based framework that aims to follow the reasoning processes of radiologists in providing predictions for cancer diagnosis in mammography.},
  archive      = {J_NATMI},
  author       = {Barnett, Alina Jade and Schwartz, Fides Regina and Tao, Chaofan and Chen, Chaofan and Ren, Yinhao and Lo, Joseph Y. and Rudin, Cynthia},
  doi          = {10.1038/s42256-021-00423-x},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1061-1070},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A case-based interpretable deep learning model for classification of mass lesions in digital mammography},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Protein function prediction for newly sequenced organisms.
<em>NATMI</em>, <em>3</em>(12), 1050–1060. (<a
href="https://doi.org/10.1038/s42256-021-00419-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent successes in protein function prediction have shown the superiority of approaches that integrate multiple types of experimental evidence over methods that rely solely on homology. However, newly sequenced organisms continue to represent a difficult challenge, because only their protein sequences are available and they lack data derived from large-scale experiments. Here we introduce S2F (Sequence to Function), a network propagation approach for the functional annotation of newly sequenced organisms. Our main idea is to systematically transfer functionally relevant data from model organisms to newly sequenced ones, thus allowing us to use a label propagation approach. S2F introduces a novel label diffusion algorithm that can account for the presence of overlapping communities of proteins with related functions. As most newly sequenced organisms are bacteria, we tested our approach in the context of bacterial genomes. Our extensive evaluation shows a great improvement over existing sequence-based methods, as well as four state-of-the-art general-purpose protein function prediction methods. Our work demonstrates that employing a diffusion process over networks of transferred functional data is an effective way to improve predictions over simple homology. S2F is applicable to any type of newly sequenced organism as well as to those for which experimental evidence is available. A free, easy to run version of S2F is available at https://www.paccanarolab.org/s2f . Predicting the function of proteins in newly sequenced organisms is a challenging problem. Mateo Torres et al. present here a method to transfer the functional relations from known organisms and improve the prediction using network diffusion.},
  archive      = {J_NATMI},
  author       = {Torres, Mateo and Yang, Haixuan and Romero, Alfonso E. and Paccanaro, Alberto},
  doi          = {10.1038/s42256-021-00419-7},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1050-1060},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Protein function prediction for newly sequenced organisms},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A deep generative model for molecule optimization via one
fragment modification. <em>NATMI</em>, <em>3</em>(12), 1040–1049. (<a
href="https://doi.org/10.1038/s42256-021-00410-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecule optimization is a critical step in drug development to improve the desired properties of drug candidates through chemical modification. We have developed a novel deep generative model, Modof, over molecular graphs for molecule optimization. Modof modifies a given molecule through the prediction of a single site of disconnection at the molecule and the removal and/or addition of fragments at that site. A pipeline of multiple, identical Modof models is implemented into Modof-pipe to modify an input molecule at multiple disconnection sites. Here we show that Modof-pipe is able to retain major molecular scaffolds, allow controls over intermediate optimization steps and better constrain molecule similarities. Modof-pipe outperforms the state-of-the-art methods on benchmark datasets. Without molecular similarity constraints, Modof-pipe achieves 81.2\% improvement in the octanol–water partition coefficient, penalized by synthetic accessibility and ring size, and 51.2\%, 25.6\% and 9.2\% improvement if the optimized molecules are at least 0.2, 0.4 and 0.6 similar to those before optimization, respectively. Modof-pipe is further enhanced into Modof-pipem to allow modification of one molecule to multiple optimized ones. Modof-pipem achieves additional performance improvement, at least 17.8\% better than Modof-pipe. To improve desired properties of drugs or other molecules, deep learning can be used to guide the optimization process. Chen et al. present a method that optimizes molecules one fragment at a time and requires fewer parameters and training data while still improving optimization performance.},
  archive      = {J_NATMI},
  author       = {Chen, Ziqi and Min, Martin Renqiang and Parthasarathy, Srinivasan and Ning, Xia},
  doi          = {10.1038/s42256-021-00410-2},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1040-1049},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A deep generative model for molecule optimization via one fragment modification},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A geometric deep learning approach to predict binding
conformations of bioactive molecules. <em>NATMI</em>, <em>3</em>(12),
1033–1039. (<a
href="https://doi.org/10.1038/s42256-021-00409-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the interactions formed between a ligand and its molecular target is key to guiding the optimization of molecules. Different experimental and computational methods have been applied to better understanding these intermolecular interactions. Here we report a method based on geometric deep learning that is capable of predicting the binding conformations of ligands to protein targets. The model learns a statistical potential based on the distance likelihood, which is tailor-made for each ligand–target pair. This potential can be coupled with global optimization algorithms to reproduce the experimental binding conformations of ligands. We show that the potential based on distance likelihood, described here, performs similarly or better than well-established scoring functions for docking and screening tasks. Overall, this method represents an example of how artificial intelligence can be used to improve structure-based drug design. Predicting binding of ligands to molecular targets is a key task in the development of new drugs. To improve the speed and accuracy of this prediction, Méndez–Lucio and colleagues developed DeepDock, a method that uses geometric deep learning to inform a statistical potential to find conformations of ligand–target pairs.},
  archive      = {J_NATMI},
  author       = {Méndez-Lucio, Oscar and Ahmad, Mazen and del Rio-Chanona, Ehecatl Antonio and Wegner, Jörg Kurt},
  doi          = {10.1038/s42256-021-00409-9},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1033-1039},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A geometric deep learning approach to predict binding conformations of bioactive molecules},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Geometric deep learning on molecular representations.
<em>NATMI</em>, <em>3</em>(12), 1023–1032. (<a
href="https://doi.org/10.1038/s42256-021-00418-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geometric deep learning (GDL) is based on neural network architectures that incorporate and process symmetry information. GDL bears promise for molecular modelling applications that rely on molecular representations with different symmetry properties and levels of abstraction. This Review provides a structured and harmonized overview of molecular GDL, highlighting its applications in drug discovery, chemical synthesis prediction and quantum chemistry. It contains an introduction to the principles of GDL, as well as relevant molecular representations, such as molecular graphs, grids, surfaces and strings, and their respective properties. The current challenges for GDL in the molecular sciences are discussed, and a forecast of future opportunities is attempted. Geometric representations are becoming more important in molecular deep learning as the spatial structure of molecules contains important information about their properties. Kenneth Atz and colleagues review current progress and challenges in this emerging field of geometric deep learning.},
  archive      = {J_NATMI},
  author       = {Atz, Kenneth and Grisoni, Francesca and Schneider, Gisbert},
  doi          = {10.1038/s42256-021-00418-8},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1023-1032},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Geometric deep learning on molecular representations},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AI-generated characters for supporting personalized learning
and well-being. <em>NATMI</em>, <em>3</em>(12), 1013–1022. (<a
href="https://doi.org/10.1038/s42256-021-00417-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advancements in machine learning have recently enabled the hyper-realistic synthesis of prose, images, audio and video data, in what is referred to as artificial intelligence (AI)-generated media. These techniques offer novel opportunities for creating interactions with digital portrayals of individuals that can inspire and intrigue us. AI-generated portrayals of characters can feature synthesized faces, bodies and voices of anyone, from a fictional character to a historical figure, or even a deceased family member. Although negative use cases of this technology have dominated the conversation so far, in this Perspective we highlight emerging positive use cases of AI-generated characters, specifically in supporting learning and well-being. We demonstrate an easy-to-use AI character generation pipeline to enable such outcomes and discuss ethical implications as well as the need for including traceability to help maintain trust in the generated media. As we look towards the future, we foresee generative media as a crucial part of the ever growing landscape of human–AI interaction. Digitally recreating the likeness of a person used to be a costly and complex process. Through the use of generative models, AI-generated characters can now be made with relative ease. Pataranutaporn et al. discuss in this Perspective how this technology can be used for positive applications in education and well-being.},
  archive      = {J_NATMI},
  author       = {Pataranutaporn, Pat and Danry, Valdemar and Leong, Joanne and Punpongsanon, Parinya and Novy, Dan and Maes, Pattie and Sra, Misha},
  doi          = {10.1038/s42256-021-00417-9},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1013-1022},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI-generated characters for supporting personalized learning and well-being},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Combining views for newly sequenced organisms.
<em>NATMI</em>, <em>3</em>(12), 1011–1012. (<a
href="https://doi.org/10.1038/s42256-021-00426-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Newly sequenced organisms present a challenge for protein function prediction, as they lack experimental characterisation. A network-propagation approach that integrates functional network relationships with protein annotations, transferred from well-studied organisms, produces a more complete picture of the possible protein functions.},
  archive      = {J_NATMI},
  author       = {Zhang, Yingying and Wierbowski, Shayne D. and Yu, Haiyuan},
  doi          = {10.1038/s42256-021-00426-8},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1011-1012},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Combining views for newly sequenced organisms},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improving customer decisions in web-based e-commerce through
guerrilla modding. <em>NATMI</em>, <em>3</em>(12), 1008–1010. (<a
href="https://doi.org/10.1038/s42256-021-00424-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The provisioning of information about product attributes in e-commerce environments is today left entirely to owners of online platforms. Product transparency in online stores can be increased by client-side enrichment of retailer Web pages.},
  archive      = {J_NATMI},
  author       = {Mayer, Simon and Fuchs, Klaus and Brügger, Dominic and Lian, Jie and Ciortea, Andrei},
  doi          = {10.1038/s42256-021-00424-w},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1008-1010},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Improving customer decisions in web-based e-commerce through guerrilla modding},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A new age for content filters. <em>NATMI</em>,
<em>3</em>(12), 1007. (<a
href="https://doi.org/10.1038/s42256-021-00429-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A well-known internet truth is that if the product is free, you are the product being sold. But with a growing range of regulations and web content tools, users can gain more control over the data they interact with.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00429-5},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {1007},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A new age for content filters},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A deep learning method for recovering missing signals in
transcriptome-wide RNA structure profiles from probing experiments.
<em>NATMI</em>, <em>3</em>(11), 995–1006. (<a
href="https://doi.org/10.1038/s42256-021-00412-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing-based RNA structure probing can generate transcriptome-wide profiles of RNA secondary structures. Sufficient structural coverage is needed to obtain unbiased insights about RNA structures and functions, yet probing methods often yield uneven coverage, with missing structural scores across many transcripts. To overcome this barrier, we developed StructureImpute, a deep learning framework inspired by depth completion from computer vision that integrates an RNA sequence with available RNA structural information of neighbouring nucleotides to infer missing structure scores. We demonstrate the strong imputation performance of StructureImpute, with accuracy much superior to predictions based on RNA sequence alone. We also show that StructureImpute reliably reconstructs RNA structural patterns at biologically impactful RNA regulation regions, including protein-binding and RNA-modification sites. Strikingly, StructureImpute can use transfer learning to apply a model trained on one dataset to accurately infer missing structural scores in other datasets, even if they were generated with different technologies (for example, icSHAPE and DMS-seq). RNA structure profiling methods suffer from missing values in RNA structurome data. Inspired by a computer vision approach, Gong and colleagues develop a deep learning method that imputes missing RNA structure scores and increases the structural coverage of the transcriptome.},
  archive      = {J_NATMI},
  author       = {Gong, Jing and Xu, Kui and Ma, Ziyuan and Lu, Zhi John and Zhang, Qiangfeng Cliff},
  doi          = {10.1038/s42256-021-00412-0},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {995-1006},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A deep learning method for recovering missing signals in transcriptome-wide RNA structure profiles from probing experiments},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep reinforcement learning-designed radiofrequency waveform
in MRI. <em>NATMI</em>, <em>3</em>(11), 985–994. (<a
href="https://doi.org/10.1038/s42256-021-00411-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Carefully engineered radiofrequency (RF) pulses play a key role in a number of systems such as mobile phone, radar and magnetic resonance imaging. The design of an RF waveform, however, is often posed as an inverse problem with no general solution. As a result, various design methods, each with a specific purpose, have been developed on the basis of the intuition of human experts. In this work we propose an artificial intelligence (AI)-powered RF pulse design framework, DeepRF, which utilizes the self-learning characteristics of deep reinforcement learning to generate a novel RF pulse. The effectiveness of DeepRF is demonstrated using four types of RF pulses that are commonly used. The DeepRF-designed pulses successfully satisfy the design criteria while reporting reduced energy. Analyses demonstrate the pulses utilize new mechanisms of magnetization manipulation, suggesting the potentials of DeepRF in discovering unseen design dimensions beyond human intuition. This work may lay the foundation for an emerging field of AI-driven RF waveform design. Radiofrequency pulses of different shapes can increase the efficiency of applications such as broadcasting or medical imaging, but finding the optimal shape for a specific use can be computationally costly. Shin and colleagues present a new method based on deep reinforcement learning to design radiofrequency pulses for use in MRI, which is demonstrated to cover different types of optimization goals for each application.},
  archive      = {J_NATMI},
  author       = {Shin, Dongmyung and Kim, Younghoon and Oh, Chungseok and An, Hongjun and Park, Juhyung and Kim, Jiye and Lee, Jongho},
  doi          = {10.1038/s42256-021-00411-1},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {985-994},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep reinforcement learning-designed radiofrequency waveform in MRI},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A deep generative model enables automated structure
elucidation of novel psychoactive substances. <em>NATMI</em>,
<em>3</em>(11), 973–984. (<a
href="https://doi.org/10.1038/s42256-021-00407-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past decade, the illicit drug market has been reshaped by the proliferation of clandestinely produced designer drugs. These agents, referred to as new psychoactive substances (NPSs), are designed to mimic the physiological actions of better-known drugs of abuse while skirting drug control laws. The public health burden of NPS abuse obliges toxicological, police and customs laboratories to screen for them in law enforcement seizures and biological samples. However, the identification of emerging NPSs is challenging due to the chemical diversity of these substances and the fleeting nature of their appearance on the illicit market. Here we present DarkNPS, a deep learning-enabled approach to automatically elucidate the structures of unidentified designer drugs using only mass spectrometric data. Our method employs a deep generative model to learn a statistical probability distribution over unobserved structures, which we term the structural prior. We show that the structural prior allows DarkNPS to elucidate the exact chemical structure of an unidentified NPS with an accuracy of 51\% and a top-10 accuracy of 86\%. Our generative approach has the potential to enable de novo structure elucidation for other types of small molecules that are routinely analysed by mass spectrometry. Identifying a chemical substance using mass spectrometry without knowing its structure is challenging. To help detect novel designer drugs from their mass spectra, Skinnider et al. describe a generative model that is biased towards creating potentially psychoactive molecules and thus helps identify potential candidates for a specific sample.},
  archive      = {J_NATMI},
  author       = {Skinnider, Michael A. and Wang, Fei and Pasin, Daniel and Greiner, Russell and Foster, Leonard J. and Dalsgaard, Petur W. and Wishart, David S.},
  doi          = {10.1038/s42256-021-00407-x},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {973-984},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A deep generative model enables automated structure elucidation of novel psychoactive substances},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Out-of-distribution generalization from labelled and
unlabelled gene expression data for drug response prediction.
<em>NATMI</em>, <em>3</em>(11), 962–972. (<a
href="https://doi.org/10.1038/s42256-021-00408-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data discrepancy between preclinical and clinical datasets poses a major challenge for accurate drug response prediction based on gene expression data. Different methods of transfer learning have been proposed to address such data discrepancy in drug response prediction for different cancers. These methods generally use cell lines as source domains, and patients, patient-derived xenografts or other cell lines as target domains; however, it is assumed that the methods have access to the target domain during training or fine-tuning, and they can only take labelled source domains as input. The former is a strong assumption that is not satisfied during deployment of these models in the clinic, whereas the latter means these methods rely on labelled source domains that are of limited size. To avoid these assumptions, we formulate drug response prediction in cancer as an out-of-distribution generalization problem, which does not assume that the target domain is accessible during training. Moreover, to exploit unlabelled source domain data—which tends to be much more plentiful than labelled data—we adopt a semi-supervised approach. We propose Velodrome, a semi-supervised method of out-of-distribution generalization that takes labelled and unlabelled data from different resources as input and makes generalizable predictions. Velodrome achieves this goal by introducing an objective function that combines a supervised loss for accurate prediction, an alignment loss for generalization and a consistency loss to incorporate unlabelled samples. Our experimental results demonstrate that Velodrome outperforms state-of-the-art pharmacogenomics and transfer learning baselines on cell lines, patient-derived xenografts and patients. Finally, we showed that Velodrome models generalize to different tissue types that were well-represented, under-represented or completely absent in the training data. Overall, our results suggest that Velodrome may guide precision oncology more accurately. Providing patient specific predictions for drug responses is challenging as preclinical data across a large population is hard to collect. Sharifi-Noghabi and colleagues present a semi-supervised method to predict drug response from limited data that can generalize successfully to different tissue types.},
  archive      = {J_NATMI},
  author       = {Sharifi-Noghabi, Hossein and Harjandi, Parsa Alamzadeh and Zolotareva, Olga and Collins, Colin C. and Ester, Martin},
  doi          = {10.1038/s42256-021-00408-w},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {962-972},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Out-of-distribution generalization from labelled and unlabelled gene expression data for drug response prediction},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Variational neural annealing. <em>NATMI</em>,
<em>3</em>(11), 952–961. (<a
href="https://doi.org/10.1038/s42256-021-00401-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many important challenges in science and technology can be cast as optimization problems. When viewed in a statistical physics framework, these can be tackled by simulated annealing, where a gradual cooling procedure helps search for ground-state solutions of a target Hamiltonian. Although powerful, simulated annealing is known to have prohibitively slow sampling dynamics when the optimization landscape is rough or glassy. Here we show that, by generalizing the target distribution with a parameterized model, an analogous annealing framework based on the variational principle can be used to search for ground-state solutions. Modern autoregressive models such as recurrent neural networks provide ideal parameterizations because they can be sampled exactly without slow dynamics, even when the model encodes a rough landscape. We implement this procedure in the classical and quantum settings on several prototypical spin glass Hamiltonians and find that, on average, it substantially outperforms traditional simulated annealing in the asymptotic limit, illustrating the potential power of this yet unexplored route to optimization. Optimization problems can be described in terms of a statistical physics framework. This offers the possibility to make use of ‘simulated annealing’, which is a procedure to search for a target solution similar to the gradual cooling of a condensed matter system to its ground state. The approach can now be sped up significantly by implementing a model of recurrent neural networks, in a new strategy called variational neural annealing.},
  archive      = {J_NATMI},
  author       = {Hibat-Allah, Mohamed and Inack, Estelle M. and Wiersema, Roeland and Melko, Roger G. and Carrasquilla, Juan},
  doi          = {10.1038/s42256-021-00401-3},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {952-961},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Variational neural annealing},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Global field reconstruction from sparse sensors with voronoi
tessellation-assisted deep learning. <em>NATMI</em>, <em>3</em>(11),
945–951. (<a href="https://doi.org/10.1038/s42256-021-00402-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Achieving accurate and robust global situational awareness of a complex time-evolving field from a limited number of sensors has been a long-standing challenge. This reconstruction problem is especially difficult when sensors are sparsely positioned in a seemingly random or unorganized manner, which is often encountered in a range of scientific and engineering problems. Moreover, these sensors could be in motion and could become online or offline over time. The key leverage in addressing this scientific issue is the wealth of data accumulated from the sensors. As a solution to this problem, we propose a data-driven spatial field recovery technique founded on a structured grid-based deep-learning approach for arbitrary positioned sensors of any numbers. It should be noted that naive use of machine learning becomes prohibitively expensive for global field reconstruction and is furthermore not adaptable to an arbitrary number of sensors. In this work, we consider the use of Voronoi tessellation to obtain a structured-grid representation from sensor locations, enabling the computationally tractable use of convolutional neural networks. One of the central features of our method is its compatibility with deep learning-based super-resolution reconstruction techniques for structured sensor data that are established for image processing. The proposed reconstruction technique is demonstrated for unsteady wake flow, geophysical data and three-dimensional turbulence. The current framework is able to handle an arbitrary number of moving sensors and thereby overcomes a major limitation with existing reconstruction methods. Our technique opens a new pathway toward the practical use of neural networks for real-time global field estimation. Complex physical processes such as flow fields can be predicted using deep learning methods if good quality sensor data is available, but sparsely placed sensors and sensors that change their position present a problem. A new approach from Kai Fukami and colleagues based on Voronoi tessellation now allows to use data from an arbitrary number of moving sensors to reconstruct a global field.},
  archive      = {J_NATMI},
  author       = {Fukami, Kai and Maulik, Romit and Ramachandra, Nesar and Fukagata, Koji and Taira, Kunihiko},
  doi          = {10.1038/s42256-021-00402-2},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {945-951},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Global field reconstruction from sparse sensors with voronoi tessellation-assisted deep learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The immuneML ecosystem for machine learning analysis of
adaptive immune receptor repertoires. <em>NATMI</em>, <em>3</em>(11),
936–944. (<a href="https://doi.org/10.1038/s42256-021-00413-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adaptive immune receptor repertoires (AIRR) are key targets for biomedical research as they record past and ongoing adaptive immune responses. The capacity of machine learning (ML) to identify complex discriminative sequence patterns renders it an ideal approach for AIRR-based diagnostic and therapeutic discovery. So far, widespread adoption of AIRR ML has been inhibited by a lack of reproducibility, transparency and interoperability. immuneML ( immuneml.uio.no ) addresses these concerns by implementing each step of the AIRR ML process in an extensible, open-source software ecosystem that is based on fully specified and shareable workflows. To facilitate widespread user adoption, immuneML is available as a command-line tool and through an intuitive Galaxy web interface, and extensive documentation of workflows is provided. We demonstrate the broad applicability of immuneML by (1) reproducing a large-scale study on immune state prediction, (2) developing, integrating and applying a novel deep learning method for antigen specificity prediction and (3) showcasing streamlined interpretability-focused benchmarking of AIRR ML. The proliferation of molecular biology and bioinformatics tools necessary to generate huge quantities of immune receptor data has not been matched by frameworks that allow easy data analysis. The authors present immuneML, an open-source collaborative ecosystem for machine learning analysis of adaptive immune receptor repertoires.},
  archive      = {J_NATMI},
  author       = {Pavlović, Milena and Scheffer, Lonneke and Motwani, Keshav and Kanduri, Chakravarthi and Kompova, Radmila and Vazov, Nikolay and Waagan, Knut and Bernal, Fabian L. M. and Costa, Alexandre Almeida and Corrie, Brian and Akbar, Rahmad and Al Hajj, Ghadi S. and Balaban, Gabriel and Brusko, Todd M. and Chernigovskaya, Maria and Christley, Scott and Cowell, Lindsay G. and Frank, Robert and Grytten, Ivar and Gundersen, Sveinung and Haff, Ingrid Hobæk and Hovig, Eivind and Hsieh, Ping-Han and Klambauer, Günter and Kuijjer, Marieke L. and Lund-Andersen, Christin and Martini, Antonio and Minotto, Thomas and Pensar, Johan and Rand, Knut and Riccardi, Enrico and Robert, Philippe A. and Rocha, Artur and Slabodkin, Andrei and Snapkov, Igor and Sollid, Ludvig M. and Titov, Dmytro and Weber, Cédric R. and Widrich, Michael and Yaari, Gur and Greiff, Victor and Sandve, Geir Kjetil},
  doi          = {10.1038/s42256-021-00413-z},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {936-944},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The immuneML ecosystem for machine learning analysis of adaptive immune receptor repertoires},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Designing clinically translatable artificial intelligence
systems for high-dimensional medical imaging. <em>NATMI</em>,
<em>3</em>(11), 929–935. (<a
href="https://doi.org/10.1038/s42256-021-00399-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The National Institutes of Health in 2018 identified key focus areas for the future of artificial intelligence in medical imaging, creating a foundational roadmap for research in image acquisition, algorithms, data standardization and translatable clinical decision support systems. Among the key issues raised in the report, data availability, the need for novel computing architectures and explainable artificial intelligence algorithms are still relevant, despite the tremendous progress made over the past few years alone. Furthermore, translational goals of data sharing, validation of performance for regulatory approval, generalizability and mitigation of unintended bias must be accounted for early in the development process. In this Perspective, we explore challenges unique to high-dimensional clinical imaging data, in addition to highlighting some of the technical and ethical considerations involved in developing machine learning systems that better represent the high-dimensional nature of many imaging modalities. Furthermore, we argue that methods that attempt to address explainability, uncertainty and bias should be treated as core components of any clinical machine learning system. Substantial advances have been made in the past decade in developing high-performance machine learning models for medical applications, but translating them into practical clinical decision-making processes remains challenging. This Perspective provides insights into a range of challenges specific to high-dimensional, multimodal medical imaging.},
  archive      = {J_NATMI},
  author       = {Shad, Rohan and Cunningham, John P. and Ashley, Euan A. and Langlotz, Curtis P. and Hiesinger, William},
  doi          = {10.1038/s42256-021-00399-8},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {929-935},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Designing clinically translatable artificial intelligence systems for high-dimensional medical imaging},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Address behaviour vulnerabilities in the next generation of
autonomous robots. <em>NATMI</em>, <em>3</em>(11), 927–928. (<a
href="https://doi.org/10.1038/s42256-021-00415-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As service and industrial robots enter our lives, new types of cybersecurity issues emerge that involve the manipulation of a robot’s behaviour. Now is the time to develop countermeasures.},
  archive      = {J_NATMI},
  author       = {Colledanchise, Michele},
  doi          = {10.1038/s42256-021-00415-x},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {927-928},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Address behaviour vulnerabilities in the next generation of autonomous robots},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Interpretable deep-learning models to help achieve the
sustainable development goals. <em>NATMI</em>, <em>3</em>(11), 926. (<a
href="https://doi.org/10.1038/s42256-021-00414-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Vinuesa, Ricardo and Sirmacek, Beril},
  doi          = {10.1038/s42256-021-00414-y},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {926},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Interpretable deep-learning models to help achieve the sustainable development goals},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Optimizing the synergy between physics and machine learning.
<em>NATMI</em>, <em>3</em>(11), 925. (<a
href="https://doi.org/10.1038/s42256-021-00416-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although the initial inspiration of neural networks came from biology, insights from physics have helped neural networks to become usable. New connections between physics and machine learning produce powerful computational methods.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00416-w},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {925},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Optimizing the synergy between physics and machine learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Learning fast in autonomous drone racing. <em>NATMI</em>,
<em>3</em>(10), 923. (<a
href="https://doi.org/10.1038/s42256-021-00405-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the AlphaPilot Challenge, teams compete to fly autonomous drones through an obstacle course as fast as possible. The 2019 winning team MAVLab reflects on the challenge of beating human pilots.},
  archive      = {J_NATMI},
  author       = {De Wagter, C. and Paredes-Vallés, F. and Sheth, N. and de Croon, G.},
  doi          = {10.1038/s42256-021-00405-z},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {923},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning fast in autonomous drone racing},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Multi-constraint molecular generation based on conditional
transformer, knowledge distillation and reinforcement learning.
<em>NATMI</em>, <em>3</em>(10), 914–922. (<a
href="https://doi.org/10.1038/s42256-021-00403-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning-based generative models can generate novel molecules with desirable physiochemical and pharmacological properties from scratch. Many excellent generative models have been proposed, but multi-objective optimizations in molecular generative tasks are still quite challenging for most existing models. Here we proposed the multi-constraint molecular generation (MCMG) approach that can satisfy multiple constraints by combining conditional transformer and reinforcement learning algorithms through knowledge distillation. A conditional transformer was used to train a molecular generative model by efficiently learning and incorporating the structure–property relations into a biased generative process. A knowledge distillation model was then employed to reduce the model’s complexity so that it can be efficiently fine-tuned by reinforcement learning and enhance the structural diversity of the generated molecules. As demonstrated by a set of comprehensive benchmarks, MCMG is a highly effective approach to traverse large and complex chemical space in search of novel compounds that satisfy multiple property constraints. Combining generative models and reinforcement learning has become a promising direction for computational drug design, but it is challenging to train an efficient model that produces candidate molecules with high diversity. Jike Wang and colleagues present a method, using knowledge distillation, to condense a conditional transformer model to make it usable in reinforcement learning while still generating diverse molecules that optimize multiple molecular properties.},
  archive      = {J_NATMI},
  author       = {Wang, Jike and Hsieh, Chang-Yu and Wang, Mingyang and Wang, Xiaorui and Wu, Zhenxing and Jiang, Dejun and Liao, Benben and Zhang, Xujun and Yang, Bo and He, Qiaojun and Cao, Dongsheng and Chen, Xi and Hou, Tingjun},
  doi          = {10.1038/s42256-021-00403-1},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {914-922},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Multi-constraint molecular generation based on conditional transformer, knowledge distillation and reinforcement learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Accurate and efficient time-domain classification with
adaptive spiking recurrent neural networks. <em>NATMI</em>,
<em>3</em>(10), 905–913. (<a
href="https://doi.org/10.1038/s42256-021-00397-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by detailed modelling of biological neurons, spiking neural networks (SNNs) are investigated as biologically plausible and high-performance models of neural computation. The sparse and binary communication between spiking neurons potentially enables powerful and energy-efficient neural networks. The performance of SNNs, however, has remained lacking compared with artificial neural networks. Here we demonstrate how an activity-regularizing surrogate gradient combined with recurrent networks of tunable and adaptive spiking neurons yields the state of the art for SNNs on challenging benchmarks in the time domain, such as speech and gesture recognition. This also exceeds the performance of standard classical recurrent neural networks and approaches that of the best modern artificial neural networks. As these SNNs exhibit sparse spiking, we show that they are theoretically one to three orders of magnitude more computationally efficient compared to recurrent neural networks with similar performance. Together, this positions SNNs as an attractive solution for AI hardware implementations. The use of sparse signals in spiking neural networks, modelled on biological neurons, offers in principle a highly efficient approach for artificial neural networks when implemented on neuromorphic hardware, but new training approaches are needed to improve performance. Using a new type of activity-regularizing surrogate gradient for backpropagation combined with recurrent networks of tunable and adaptive spiking neurons, state-of-the-art performance for spiking neural networks is demonstrated on benchmarks in the time domain.},
  archive      = {J_NATMI},
  author       = {Yin, Bojian and Corradi, Federico and Bohté, Sander M.},
  doi          = {10.1038/s42256-021-00397-w},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {905-913},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accurate and efficient time-domain classification with adaptive spiking recurrent neural networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Empirical observation of negligible fairness–accuracy
trade-offs in machine learning for public policy. <em>NATMI</em>,
<em>3</em>(10), 896–904. (<a
href="https://doi.org/10.1038/s42256-021-00396-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing use of machine learning in policy and social impact settings has raised concerns over fairness implications, especially for racial minorities. These concerns have generated considerable interest among machine learning and artificial intelligence researchers, who have developed new methods and established theoretical bounds for improving fairness, focusing on the source data, regularization and model training, or post-hoc adjustments to model scores. However, few studies have examined the practical trade-offs between fairness and accuracy in real-world settings to understand how these bounds and methods translate into policy choices and impact on society. Our empirical study fills this gap by investigating the impact of mitigating disparities on accuracy, focusing on the common context of using machine learning to inform benefit allocation in resource-constrained programmes across education, mental health, criminal justice and housing safety. Here we describe applied work in which we find fairness–accuracy trade-offs to be negligible in practice. In each setting studied, explicitly focusing on achieving equity and using our proposed post-hoc disparity mitigation methods, fairness was substantially improved without sacrificing accuracy. This observation was robust across policy contexts studied, scale of resources available for intervention, time and the relative size of the protected groups. These empirical results challenge a commonly held assumption that reducing disparities requires either accepting an appreciable drop in accuracy or the development of novel, complex methods, making reducing disparities in these applications more practical. A growing number of researchers are developing approaches to improve fairness in machine learning applications in areas such as healthcare, employment and social services, to avoid propagating and amplifying racial and other inequities. An empirical study explores the trade-off between increasing fairness and model accuracy across several social policy areas and finds that this trade-off is negligible in practice.},
  archive      = {J_NATMI},
  author       = {Rodolfa, Kit T. and Lamba, Hemank and Ghani, Rayid},
  doi          = {10.1038/s42256-021-00396-x},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {896-904},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Empirical observation of negligible fairness–accuracy trade-offs in machine learning for public policy},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Iterative human and automated identification of wildlife
images. <em>NATMI</em>, <em>3</em>(10), 885–895. (<a
href="https://doi.org/10.1038/s42256-021-00393-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Camera trapping is increasingly being used to monitor wildlife, but this technology typically requires extensive data annotation. Recently, deep learning has substantially advanced automatic wildlife recognition. However, current methods are hampered by a dependence on large static datasets, whereas wildlife data are intrinsically dynamic and involve long-tailed distributions. These drawbacks can be overcome through a hybrid combination of machine learning and humans in the loop. Our proposed iterative human and automated identification approach is capable of learning from wildlife imagery data with a long-tailed distribution. Additionally, it includes self-updating learning, which facilitates capturing the community dynamics of rapidly changing natural systems. Extensive experiments show that our approach can achieve an ~90\% accuracy employing only ~20\% of the human annotations of existing approaches. Our synergistic collaboration of humans and machines transforms deep learning from a relatively inefficient post-annotation tool to a collaborative ongoing annotation tool that vastly reduces the burden of human annotation and enables efficient and constant model updates. Camera trapping is a widely adopted method for monitoring terrestrial mammals. However, a drawback is the amount of human annotation needed to keep pace with continuous data collection. The authors developed a hybrid system of machine learning and humans in the loop, which minimizes annotation load and improves efficiency.},
  archive      = {J_NATMI},
  author       = {Miao, Zhongqi and Liu, Ziwei and Gaynor, Kaitlyn M. and Palmer, Meredith S. and Yu, Stella X. and Getz, Wayne M.},
  doi          = {10.1038/s42256-021-00393-0},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {885-895},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Iterative human and automated identification of wildlife images},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Neutralizing the impact of atmospheric turbulence on complex
scene imaging via deep learning. <em>NATMI</em>, <em>3</em>(10),
876–884. (<a href="https://doi.org/10.1038/s42256-021-00392-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A turbulent medium with eddies of different scales gives rise to fluctuations in the index of refraction during the process of wave propagation, which interferes with the original spatial relationship, phase relationship and optical path. The outputs of two-dimensional imaging systems suffer from anamorphosis brought about by this effect. Randomness, along with multiple types of degradation, make it a challenging task to analyse the reciprocal physical process. Here, we present a generative adversarial network (TSR-WGAN), which integrates temporal and spatial information embedded in the three-dimensional input to learn the representation of the residual between the observed and latent ideal data. Vision-friendly and credible sequences are produced without extra assumptions on the scale and strength of turbulence. The capability of TSR-WGAN is demonstrated through tests on our dataset, which contains 27,458 sequences with 411,870 frames of algorithm simulated data, physical simulated data and real data. TSR-WGAN exhibits promising visual quality and a deep understanding of the disparity between random perturbations and object movements. These preliminary results also shed light on the potential of deep learning to parse stochastic physical processes from particular perspectives and to solve complicated image reconstruction problems given limited data. Turbulent optical distortions in the atmosphere limit the ability of optical technologies such as laser communication and long-distance environmental monitoring. A new method using adversarial networks learns to counter the physical processes underlying the turbulence so that complex optical scenes can be reconstructed.},
  archive      = {J_NATMI},
  author       = {Jin, Darui and Chen, Ying and Lu, Yi and Chen, Junzhang and Wang, Peng and Liu, Zichao and Guo, Sheng and Bai, Xiangzhi},
  doi          = {10.1038/s42256-021-00392-1},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {876-884},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Neutralizing the impact of atmospheric turbulence on complex scene imaging via deep learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning-based prediction of the t cell
receptor–antigen binding specificity. <em>NATMI</em>, <em>3</em>(10),
864–875. (<a href="https://doi.org/10.1038/s42256-021-00383-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neoantigens play a key role in the recognition of tumour cells by T cells; however, only a small proportion of neoantigens truly elicit T-cell responses, and few clues exist as to which neoantigens are recognized by which T-cell receptors (TCRs). We built a transfer learning-based model named the pMHC–TCR binding prediction network (pMTnet) to predict TCR binding specificities of the neoantigens—and T cell antigens in general—presented by class I major histocompatibility complexes. pMTnet was comprehensively validated by a series of analyses and exhibited great advances over previous works. By applying pMTnet to human tumour genomics data, we discovered that neoantigens were generally more immunogenic than self-antigens, but human endogenous retrovirus E (a special type of self-antigen that is reactivated in kidney cancer) is more immunogenic than neoantigens. We further discovered that patients with more clonally expanded T cells that exhibit better affinity against truncal rather than subclonal neoantigens had more favourable prognosis and treatment response to immunotherapy in melanoma and lung cancer but not in kidney cancer. Predicting TCR–neoantigen/antigen pairing is one of the most daunting challenges in modern immunology; however, we achieved an accurate prediction of the pairing using only the TCR sequence (CDR3β), antigen sequence and class I major histocompatibility complex allele, and our work revealed unique insights into the interactions between TCRs and major histocompatibility complexes in human tumours, using pMTnet as a discovery tool. T-cell immunity is driven by the interaction between peptides presented by major histocompatibility complexes (pMHCs) and T-cell receptors (TCRs). Only a small proportion of neoantigens elicit T-cell responses, and it is not clear which neoantigens are recognized by which TCRs. The authors develop a transfer learning model to predict TCR binding specificity to class-I pMHCs.},
  archive      = {J_NATMI},
  author       = {Lu, Tianshi and Zhang, Ze and Zhu, James and Wang, Yunguan and Jiang, Peixin and Xiao, Xue and Bernatchez, Chantale and Heymach, John V. and Gibbons, Don L. and Wang, Jun and Xu, Lin and Reuben, Alexandre and Wang, Tao},
  doi          = {10.1038/s42256-021-00383-2},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {864-875},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning-based prediction of the t cell receptor–antigen binding specificity},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reusability report: Feature disentanglement in generating a
three-dimensional structure from a two-dimensional slice with sliceGAN.
<em>NATMI</em>, <em>3</em>(10), 861–863. (<a
href="https://doi.org/10.1038/s42256-021-00400-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Chung, Hyungjin and Ye, Jong Chul},
  doi          = {10.1038/s42256-021-00400-4},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {861-863},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Feature disentanglement in generating a three-dimensional structure from a two-dimensional slice with sliceGAN},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The neural resource allocation problem when enhancing human
bodies with extra robotic limbs. <em>NATMI</em>, <em>3</em>(10),
850–860. (<a href="https://doi.org/10.1038/s42256-021-00398-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of robotic body augmentation provides exciting innovations that will revolutionize the fields of robotics, human–machine interaction and wearable electronics. Although augmentative devices such as extra robotic arms and fingers are informed by restorative technologies in many ways, they also introduce unique challenges for bidirectional human–machine collaboration. Can humans adapt and learn to operate a new robotic limb collaboratively with their biological limbs, without restricting other physical abilities? To successfully achieve robotic body augmentation, we need to ensure that, by giving a user an additional (artificial) limb, we are not trading off the functionalities of an existing (biological) one. Here, we introduce the ‘neural resource allocation problem’ and discuss how to allow the effective voluntary control of augmentative devices without compromising control of the biological body. In reviewing the relevant literature on extra robotic fingers and arms, we critically assess the range of potential solutions available for this neural resource allocation problem. For this purpose, we combine multiple perspectives from engineering and neuroscience with considerations including human–machine interaction, sensory–motor integration, ethics and law. In summary, we aim to define common foundations and operating principles for the successful implementation of robotic body augmentation. The development of extra fingers and arms is an exciting research area in robotics, human–machine interaction and wearable electronics. It is unclear, however, whether humans can adapt and learn to control extra limbs and integrate them into a new sensorimotor representation, without sacrificing their natural abilities. The authors review this topic and describe challenges in allocating neural resources for robotic body augmentation.},
  archive      = {J_NATMI},
  author       = {Dominijanni, Giulia and Shokur, Solaiman and Salvietti, Gionata and Buehler, Sarah and Palmerini, Erica and Rossi, Simone and De Vignemont, Frederique and d’Avella, Andrea and Makin, Tamar R. and Prattichizzo, Domenico and Micera, Silvestro},
  doi          = {10.1038/s42256-021-00398-9},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {850-860},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The neural resource allocation problem when enhancing human bodies with extra robotic limbs},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Harnessing the power of artificial intelligence to transform
hearing healthcare and research. <em>NATMI</em>, <em>3</em>(10),
840–849. (<a href="https://doi.org/10.1038/s42256-021-00394-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The advances in artificial intelligence that are transforming many fields have yet to make an impact in hearing. Hearing healthcare continues to rely on a labour-intensive service model that fails to provide access to the majority of those in need, while hearing research suffers from a lack of computational tools with the capacity to match the complexities of auditory processing. This Perspective is a call for the artificial intelligence and hearing communities to come together to bring about a technological revolution in hearing. We describe opportunities for rapid clinical impact through the application of existing technologies and propose directions for the development of new technologies to create true artificial auditory systems. There is an urgent need to push hearing towards a future in which artificial intelligence provides critical support for the testing of hypotheses, the development of therapies and the effective delivery of care worldwide. AI has impacted many fields, but hearing has lagged behind. This Perspective calls for a collaboration between the AI and hearing communities, with the goal of transforming hearing healthcare and research.},
  archive      = {J_NATMI},
  author       = {Lesica, Nicholas A. and Mehta, Nishchay and Manjaly, Joseph G. and Deng, Li and Wilson, Blake S. and Zeng, Fan-Gang},
  doi          = {10.1038/s42256-021-00394-z},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {840-849},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Harnessing the power of artificial intelligence to transform hearing healthcare and research},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Machine intelligence enabled radiomics. <em>NATMI</em>,
<em>3</em>(10), 838–839. (<a
href="https://doi.org/10.1038/s42256-021-00404-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The radiomics features of disease lesions can be learned from medical imaging data, but is it possible to identify interpretable biomarkers that can help make clinical predictions across heterogeneous diseases and data from different modalities?},
  archive      = {J_NATMI},
  author       = {Wang, Yue and Herrington, David M.},
  doi          = {10.1038/s42256-021-00404-0},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {838-839},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine intelligence enabled radiomics},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Robotic body augmentation. <em>NATMI</em>, <em>3</em>(10),
837. (<a href="https://doi.org/10.1038/s42256-021-00406-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Can the human brain cope with controlling an extra robotic arm or digit added to the body?},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00406-y},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {837},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Robotic body augmentation},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fast and energy-efficient neuromorphic deep learning with
first-spike times. <em>NATMI</em>, <em>3</em>(9), 823–835. (<a
href="https://doi.org/10.1038/s42256-021-00388-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For a biological agent operating under environmental pressure, energy consumption and reaction times are of critical importance. Similarly, engineered systems are optimized for short time-to-solution and low energy-to-solution characteristics. At the level of neuronal implementation, this implies achieving the desired results with as few and as early spikes as possible. With time-to-first-spike coding, both of these goals are inherently emerging features of learning. Here, we describe a rigorous derivation of a learning rule for such first-spike times in networks of leaky integrate-and-fire neurons, relying solely on input and output spike times, and show how this mechanism can implement error backpropagation in hierarchical spiking networks. Furthermore, we emulate our framework on the BrainScaleS-2 neuromorphic system and demonstrate its capability of harnessing the system’s speed and energy characteristics. Finally, we examine how our approach generalizes to other neuromorphic platforms by studying how its performance is affected by typical distortive effects induced by neuromorphic substrates. Spiking neural networks promise fast and energy-efficient information processing. The ‘time-to-first-spike’ coding scheme, where the time elapsed before a neuron’s first spike is utilized as the main variable, is a particularly efficient approach and Göltz and Kriener et al. demonstrate that error backpropagation, an essential ingredient for learning in neural networks, can be implemented in this scheme.},
  archive      = {J_NATMI},
  author       = {Göltz, J. and Kriener, L. and Baumbach, A. and Billaudelle, S. and Breitwieser, O. and Cramer, B. and Dold, D. and Kungl, A. F. and Senn, W. and Schemmel, J. and Meier, K. and Petrovici, M. A.},
  doi          = {10.1038/s42256-021-00388-x},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {823-835},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Fast and energy-efficient neuromorphic deep learning with first-spike times},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Automating crystal-structure phase mapping by combining deep
learning with constraint reasoning. <em>NATMI</em>, <em>3</em>(9),
812–822. (<a href="https://doi.org/10.1038/s42256-021-00384-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crystal-structure phase mapping is a core, long-standing challenge in materials science that requires identifying crystal phases, or mixtures thereof, in X-ray diffraction measurements of synthesized materials. Phase mapping algorithms have been developed that excel at solving systems with up to several unique phase mixtures, where each phase has a readily distinguishable diffraction pattern. However, phase mapping is often beyond materials scientists’ capabilities and also poses challenges to state-of-the-art algorithms due to complexities such as the existence of dozens of phase mixtures, alloy-dependent variation in the diffraction patterns and multiple compositional degrees of freedom, creating a major bottleneck in high-throughput materials discovery. Here we show how to automate crystal-structure phase mapping. We formulate phase mapping as an unsupervised pattern demixing problem and describe how to solve it using deep reasoning networks (DRNets). DRNets combine deep learning with constraint reasoning for incorporating prior scientific knowledge and consequently require only a modest amount of (unlabelled) data. DRNets compensate for the limited data by exploiting and magnifying the rich prior knowledge about the thermodynamic rules governing the mixtures of crystals. DRNets are designed with an interpretable latent space for encoding prior-knowledge domain constraints and seamlessly integrate constraint reasoning into neural network optimization. DRNets surpass previous approaches on crystal-structure phase mapping, unravelling the Bi–Cu–V oxide phase diagram and aiding the discovery of solar fuels materials. Incorporating prior knowledge in deep learning models can overcome the difficulties of supervised learning, including the need for large amounts of annotated data. An approach in this area called deep reasoning networks is applied to the complex task of mapping crystal structures from X-ray diffraction data for multi-element oxide structures, and identified 13 phases from 307 X-ray diffraction patterns in the previously unsolved Bi-Cu-V oxide system.},
  archive      = {J_NATMI},
  author       = {Chen, Di and Bai, Yiwei and Ament, Sebastian and Zhao, Wenting and Guevarra, Dan and Zhou, Lan and Selman, Bart and van Dover, R. Bruce and Gregoire, John M. and Gomes, Carla P.},
  doi          = {10.1038/s42256-021-00384-1},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {812-822},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Automating crystal-structure phase mapping by combining deep learning with constraint reasoning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modality attention and sampling enables deep learning with
heterogeneous marker combinations in fluorescence microscopy.
<em>NATMI</em>, <em>3</em>(9), 799–811. (<a
href="https://doi.org/10.1038/s42256-021-00379-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence microscopy allows for a detailed inspection of cells, cellular networks and anatomical landmarks by staining with a variety of carefully selected markers visualized as colour channels. Quantitative characterization of structures in acquired images often relies on automatic image analysis methods. Despite the success of deep learning methods in other vision applications, their potential for fluorescence image analysis remains underexploited. One reason lies in the considerable workload required to train accurate models, which are normally specific for a given combination of markers and therefore applicable to a very restricted number of experimental settings. We herein propose ‘marker sampling and excite’—a neural network approach with a modality sampling strategy and a novel attention module that together enable (1) flexible training with heterogeneous datasets with combinations of markers and (2) successful utility of learned models on arbitrary subsets of markers prospectively. We show that our single neural network solution performs comparably to an upper bound scenario in which an ensemble of many networks is naively trained for each possible marker combination separately. We also demonstrate the feasibility of this framework in high-throughput biological analysis by revising a recent quantitative characterization of bone-marrow vasculature in three-dimensional confocal microscopy datasets and further confirm the validity of our approach on another substantially different dataset of microvessels in foetal liver tissues. Not only can our work substantially ameliorate the use of deep learning in fluorescence microscopy analysis, but it can also be utilized in other fields with incomplete data acquisitions and missing modalities. Deep learning has revolutionized image analysis, but applying it to voluminous and multimodal images can be challenging. The authors propose a neural network approach with a modality sampling strategy and an attention module for segmentation in fluorescence microscopy images.},
  archive      = {J_NATMI},
  author       = {Gomariz, Alvaro and Portenier, Tiziano and Helbling, Patrick M. and Isringhausen, Stephan and Suessbier, Ute and Nombela-Arrieta, César and Goksel, Orcun},
  doi          = {10.1038/s42256-021-00379-y},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {799-811},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Modality attention and sampling enables deep learning with heterogeneous marker combinations in fluorescence microscopy},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Radiological tumour classification across imaging modality
and histology. <em>NATMI</em>, <em>3</em>(9), 787–798. (<a
href="https://doi.org/10.1038/s42256-021-00377-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Radiomics refers to the high-throughput extraction of quantitative features from radiological scans and is widely used to search for imaging biomarkers for the prediction of clinical outcomes. Current radiomic signatures suffer from limited reproducibility and generalizability, because most features are dependent on imaging modality and tumour histology, making them sensitive to variations in scan protocol. Here, we propose novel radiological features that are specially designed to ensure compatibility across diverse tissues and imaging contrast. These features provide systematic characterization of tumour morphology and spatial heterogeneity. In an international multi-institution study of 1,682 patients, we discover and validate four unifying imaging subtypes across three malignancies and two major imaging modalities. These tumour subtypes demonstrate distinct molecular characteristics and prognoses after conventional therapies. In advanced lung cancer treated with immunotherapy, one subtype is associated with improved survival and increased tumour-infiltrating lymphocytes compared with the others. Deep learning enables automatic tumour segmentation and reproducible subtype identification, which can facilitate practical implementation. The unifying radiological tumour classification may inform prognosis and treatment response for precision medicine. Radiomics has been used to discover imaging signatures that predict therapy response and outcomes, but clinical translation has been slow. Using machine learning methods, the authors report tumour subtypes that are applicable across major imaging modalities and three cancer types. The tumour subtypes have distinct radiological and molecular features, as well as survival outcomes after conventional therapies.},
  archive      = {J_NATMI},
  author       = {Wu, Jia and Li, Chao and Gensheimer, Michael and Padda, Sukhmani and Kato, Fumi and Shirato, Hiroki and Wei, Yiran and Schönlieb, Carola-Bibiane and Price, Stephen John and Jaffray, David and Heymach, John and Neal, Joel W. and Loo, Billy W. and Wakelee, Heather and Diehn, Maximilian and Li, Ruijiang},
  doi          = {10.1038/s42256-021-00377-0},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {787-798},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Radiological tumour classification across imaging modality and histology},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Learning function from structure in neuromorphic networks.
<em>NATMI</em>, <em>3</em>(9), 771–786. (<a
href="https://doi.org/10.1038/s42256-021-00376-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The connection patterns of neural circuits in the brain form a complex network. Collective signalling within the network manifests as patterned neural activity and is thought to support human cognition and adaptive behaviour. Recent technological advances permit macroscale reconstructions of biological brain networks. These maps, termed connectomes, display multiple non-random architectural features, including heavy-tailed degree distributions, segregated communities and a densely interconnected core. Yet, how computation and functional specialization emerge from network architecture remains unknown. Here we reconstruct human brain connectomes using in vivo diffusion-weighted imaging and use reservoir computing to implement connectomes as artificial neural networks. We then train these neuromorphic networks to learn a memory-encoding task. We show that biologically realistic neural architectures perform best when they display critical dynamics. We find that performance is driven by network topology and that the modular organization of intrinsic networks is computationally relevant. We observe a prominent interaction between network structure and dynamics throughout, such that the same underlying architecture can support a wide range of memory capacity values as well as different functions (encoding or decoding), depending on the dynamical regime the network is in. This work opens new opportunities to discover how the network organization of the brain optimizes cognitive capacity. The relationship between brain organization, connectivity and computation is not well understood. The authors construct neuromorphic artificial neural networks endowed with biological connection patterns derived from diffusion-weighted imaging. The neuromorphic networks are trained to perform a memory task, revealing an interaction between network structure and dynamics.},
  archive      = {J_NATMI},
  author       = {Suárez, Laura E. and Richards, Blake A. and Lajoie, Guillaume and Misic, Bratislav},
  doi          = {10.1038/s42256-021-00376-1},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {771-786},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning function from structure in neuromorphic networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chemical language models enable navigation in sparsely
populated chemical space. <em>NATMI</em>, <em>3</em>(9), 759–770. (<a
href="https://doi.org/10.1038/s42256-021-00368-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep generative models are powerful tools for the exploration of chemical space, enabling the on-demand generation of molecules with desired physical, chemical or biological properties. However, these models are typically thought to require training datasets comprising hundreds of thousands, or even millions, of molecules. This perception limits the application of deep generative models in regions of chemical space populated by a relatively small number of examples. Here, we systematically evaluate and optimize generative models of molecules based on recurrent neural networks in low-data settings. We find that robust models can be learned from far fewer examples than has been widely assumed. We identify strategies that further reduce the number of molecules required to learn a model of equivalent quality, notably including data augmentation by non-canonical SMILES enumeration, and demonstrate the application of these principles by learning models of bacterial, plant and fungal metabolomes. The structure of our experiments also allows us to benchmark the metrics used to evaluate generative models themselves. We find that many of the most widely used metrics in the field fail to capture model quality, but we identify a subset of well-behaved metrics that provide a sound basis for model development. Collectively, our work provides a foundation for directly learning generative models in sparsely populated regions of chemical space. Deep learning-based methods to generate new molecules can require huge amounts of data to train. Skinnider et al. show that models developed for natural language processing work well for generating molecules from small amounts of training data, and identify robust metrics to evaluate the quality of generated molecules.},
  archive      = {J_NATMI},
  author       = {Skinnider, Michael A. and Stacey, R. Greg and Wishart, David S. and Foster, Leonard J.},
  doi          = {10.1038/s42256-021-00368-1},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {759-770},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Chemical language models enable navigation in sparsely populated chemical space},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Adversarial interference and its mitigations in
privacy-preserving collaborative machine learning. <em>NATMI</em>,
<em>3</em>(9), 749–758. (<a
href="https://doi.org/10.1038/s42256-021-00390-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the rapid increase of data available to train machine-learning algorithms in many domains, several applications suffer from a paucity of representative and diverse data. The medical and financial sectors are, for example, constrained by legal, ethical, regulatory and privacy concerns preventing data sharing between institutions. Collaborative learning systems, such as federated learning, are designed to circumvent such restrictions and provide a privacy-preserving alternative by eschewing data sharing and relying instead on the distributed remote execution of algorithms. However, such systems are susceptible to malicious adversarial interference attempting to undermine their utility or divulge confidential information. Here we present an overview and analysis of current adversarial attacks and their mitigations in the context of collaborative machine learning. We discuss the applicability of attack vectors to specific learning contexts and attempt to formulate a generic foundation for adversarial influence and mitigation mechanisms. We moreover show that a number of context-specific learning conditions are exploited in similar fashion across all settings. Lastly, we provide a focused perspective on open challenges and promising areas of future research in the field. When the training data for machine learning are highly personal or sensitive, collaborative approaches can help a collective of stakeholders to train a model together without having to share any data. But there are still risks to the privacy of the data. This Perspective provides an overview of potential attacks on collaborative machine learning and how these threats could be addressed.},
  archive      = {J_NATMI},
  author       = {Usynin, Dmitrii and Ziller, Alexander and Makowski, Marcus and Braren, Rickmer and Rueckert, Daniel and Glocker, Ben and Kaissis, Georgios and Passerat-Palmbach, Jonathan},
  doi          = {10.1038/s42256-021-00390-3},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {749-758},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Adversarial interference and its mitigations in privacy-preserving collaborative machine learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ethical requirements for responsible research with hacked
data. <em>NATMI</em>, <em>3</em>(9), 744–748. (<a
href="https://doi.org/10.1038/s42256-021-00389-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hacked data—data obtained in an unauthorized manner through illicit access to a computer or computer network—are increasingly being used in scientific research. Examples include conflict modelling studies based on WikiLeaks datasets, and studies on sexual behaviour based on data leaked from Ashley Madison, a dating website hacked in 2015. Studies of this kind are prima facie lawful because the researchers who processed the data had no involvement in the data breach itself, and the hacked datasets were made publicly available. However, research integrity is not simply reducible to lawfulness. We contend that the ethical acceptability of using datasets of illicit origin cannot simply be presumed, but requires ethical justification. Therefore, the scientific community is urged to address a fundamental ethical question: whether, and under which conditions, conducting research using hacked data can be considered morally justifiable behaviour. This contribution will discuss and critically evaluate the ethical boundaries of research involving hacked data. This emerging ethical dilemma will be examined by analogy with previous debates on the ethics of research using data of unethical origin, such as research with data from Nazi medical experiments. Furthermore, we discuss the compatibility of research involving hacked data with current research ethics instruments such as the Belmont Report and the Oviedo Convention. We posit that research using hacked data should be deemed ethically problematic. However, exceptions may be acknowledged, depending on how the data are processed and the overall value of the study to society. To this end, we propose six ethical and procedural requirements for conducting research with hacked data in an ethically responsible manner. The ethical use of publicly available datasets with human data for which consent has not been explicitly given needs urgent attention from researchers, funders, research institutes and publishers. A specific challenging case is research involving hacked data and this Perspective discusses whether and under what conditions it is morally and ethically justified to conduct such research.},
  archive      = {J_NATMI},
  author       = {Ienca, Marcello and Vayena, Effy},
  doi          = {10.1038/s42256-021-00389-w},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {744-748},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Ethical requirements for responsible research with hacked data},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Sparsity provides a competitive advantage. <em>NATMI</em>,
<em>3</em>(9), 742–743. (<a
href="https://doi.org/10.1038/s42256-021-00387-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuromorphic chips that use spikes to encode information could provide fast and energy-efficient computing for ubiquitous embedded systems. A bio-plausible spike-timing solution for training spiking neural networks that makes the most of sparsity is implemented on the BrainScaleS-2 hardware platform.},
  archive      = {J_NATMI},
  author       = {Frenkel, Charlotte},
  doi          = {10.1038/s42256-021-00387-y},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {742-743},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Sparsity provides a competitive advantage},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Wiring up recurrent neural networks. <em>NATMI</em>,
<em>3</em>(9), 740–741. (<a
href="https://doi.org/10.1038/s42256-021-00391-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Functional subsystems of the macroscale human brain connectome are mapped onto a recurrent neural network and found to perform optimally in a critical regime at the edge of chaos.},
  archive      = {J_NATMI},
  author       = {Imam, Nabil},
  doi          = {10.1038/s42256-021-00391-2},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {740-741},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Wiring up recurrent neural networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Regulating AI in medicine in the united states and europe.
<em>NATMI</em>, <em>3</em>(9), 738–739. (<a
href="https://doi.org/10.1038/s42256-021-00386-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The regulatory landscape for artificial intelligence (AI) is shaping up on both sides of the Atlantic, urgently awaited by the scientific and industrial community. Commonalities and differences start to crystallize in the approaches to AI in medicine.},
  archive      = {J_NATMI},
  author       = {Vokinger, Kerstin N. and Gasser, Urs},
  doi          = {10.1038/s42256-021-00386-z},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {738-739},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Regulating AI in medicine in the united states and europe},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The big question. <em>NATMI</em>, <em>3</em>(9), 737. (<a
href="https://doi.org/10.1038/s42256-021-00395-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Very large neural network models such as GPT-3, which have many billions of parameters, are on the rise, but so far only big tech has the resources to train, deploy and study such models. This needs to change, say Stanford AI researchers, who call for an investment in academic collaborations to build and study large neural networks.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00395-y},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {737},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The big question},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Lessons from the first DBTex challenge. <em>NATMI</em>,
<em>3</em>(8), 735–736. (<a
href="https://doi.org/10.1038/s42256-021-00378-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new international competition aims to speed up the development of AI models that can assist radiologists in detecting suspicious lesions from hundreds of millions of pixels in 3D mammograms. The top three winning teams compare notes.},
  archive      = {J_NATMI},
  author       = {Park, Jungkyu and Shoshan, Yoel and Martí, Robert and Gómez del Campo, Pablo and Ratner, Vadim and Khapun, Daniel and Zlotnick, Aviad and Barkan, Ella and Gilboa-Solomon, Flora and Chłędowski, Jakub and Witowski, Jan and Millet, Alexandra and Kim, Eric and Lewin, Alana and Pysarenko, Kristine and Chen, Sardius and Goldberg, Julia and Patel, Shalin and Plaunova, Anastasia and Wegener, Melanie and Wolfson, Stacey and Lee, Jiyon and Hava, Sana and Murthy, Sindhoora and Du, Linda and Gaddam, Sushma and Parikh, Ujas and Heacock, Laura and Moy, Linda and Reig, Beatriu and Rosen-Zvi, Michal and Geras, Krzysztof J.},
  doi          = {10.1038/s42256-021-00378-z},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {735-736},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Lessons from the first DBTex challenge},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Dynamic particle swarm optimization of biomolecular
simulation parameters with flexible objective functions. <em>NATMI</em>,
<em>3</em>(8), 727–734. (<a
href="https://doi.org/10.1038/s42256-021-00366-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular simulations are a powerful tool to complement and interpret ambiguous experimental data on biomolecules to obtain structural models. Such data-assisted simulations often rely on parameters, the choice of which is highly non-trivial and crucial to performance. The key challenge is weighting experimental information with respect to the underlying physical model. We introduce FLAPS, a self-adapting variant of dynamic particle swarm optimization, to overcome this parameter selection problem. FLAPS is suited for the optimization of composite objective functions that depend on both the optimization parameters and additional, a priori unknown weighting parameters, which substantially influence the search-space topology. These weighting parameters are learned at runtime, yielding a dynamically evolving and iteratively refined search-space topology. As a practical example, we show how FLAPS can be used to find functional parameters for small-angle X-ray scattering-guided protein simulations. Molecular simulations informed by experimental data can provide detailed knowledge of complex biomolecular structure. However, it is a challenging task to weight experimental information with respect to the underlying model. A self-adapting type of dynamic particle swarm optimization can tackle the parameter selection problem, which is demonstrated on small-angle X-ray scattering-guided protein simulations.},
  archive      = {J_NATMI},
  author       = {Weiel, Marie and Götz, Markus and Klein, André and Coquelin, Daniel and Floca, Ralf and Schug, Alexander},
  doi          = {10.1038/s42256-021-00366-3},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {727-734},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Dynamic particle swarm optimization of biomolecular simulation parameters with flexible objective functions},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Machine learning to guide the use of adjuvant therapies for
breast cancer. <em>NATMI</em>, <em>3</em>(8), 716–726. (<a
href="https://doi.org/10.1038/s42256-021-00353-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate prediction of the individualized survival benefit of adjuvant therapy is key to making informed therapeutic decisions for patients with early invasive breast cancer. Machine learning technologies can enable accurate prognostication of patient outcomes under different treatment options by modelling complex interactions between risk factors in a data-driven fashion. Here, we use an automated and interpretable machine learning algorithm to develop a breast cancer prognostication and treatment benefit prediction model—Adjutorium—using data from large-scale cohorts of nearly one million women captured in the national cancer registries of the United Kingdom and the United States. We trained and internally validated the Adjutorium model on 395,862 patients from the UK National Cancer Registration and Analysis Service (NCRAS), and then externally validated the model among 571,635 patients from the US Surveillance, Epidemiology, and End Results (SEER) programme. Adjutorium exhibited significantly improved accuracy compared to the major prognostic tool in current clinical use (PREDICT v2.1) in both internal and external validation. Importantly, our model substantially improved accuracy in specific subgroups known to be under-served by existing models. Adjutorium is currently implemented as a web-based decision support tool ( https://vanderschaar-lab.com/adjutorium/ ) to aid decisions on adjuvant therapy in women with early breast cancer, and can be publicly accessed by patients and clinicians worldwide. Methods are available to support clinical decisions regarding adjuvant therapies in breast cancer, but they have limitations in accuracy, generalizability and interpretability. Alaa et al. present an automated machine learning model of breast cancer that predicts patient survival and adjuvant treatment benefit to guide personalized therapeutic decisions.},
  archive      = {J_NATMI},
  author       = {Alaa, Ahmed M. and Gurdasani, Deepti and Harris, Adrian L. and Rashbass, Jem and van der Schaar, Mihaela},
  doi          = {10.1038/s42256-021-00353-8},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {716-726},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine learning to guide the use of adjuvant therapies for breast cancer},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Integration of millions of transcriptomes using batch-aware
triplet neural networks. <em>NATMI</em>, <em>3</em>(8), 705–715. (<a
href="https://doi.org/10.1038/s42256-021-00361-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efficient integration of heterogeneous and increasingly large single-cell RNA sequencing data poses a major challenge for analysis and, in particular, comprehensive atlasing efforts. Here we developed a novel deep learning algorithm called INSCT (Insight) to overcome batch effects using batch-aware triplet neural networks. We use simulated and real data to demonstrate that INSCT generates an embedding space that accurately integrates cells across experiments, platforms and species. Our benchmark comparisons with current state-of-the-art single-cell RNA sequencing integration methods revealed that INSCT outperforms competing methods in scalability while achieving comparable accuracies. Moreover, using INSCT in semisupervised mode enables users to classify unlabelled cells by projecting them into a reference collection of annotated cells. To demonstrate scalability, we applied INSCT to integrate more than 2.6 million transcriptomes from four independent studies of mouse brains in less than 1.5 h using less than 25 GB of memory. This feature empowers researchers to perform atlasing-scale data integration in a typical desktop computer environment. INSCT is freely available at https://github.com/lkmklsmn/insct . Single-cell RNA sequencing efforts have made large amounts of data available for transcriptomics research. Simon and colleagues develop a neural network embedding approach that avoids batch effects, such that it can rapidly and efficiently integrate large datasets from different studies.},
  archive      = {J_NATMI},
  author       = {Simon, Lukas M. and Wang, Yin-Ying and Zhao, Zhongming},
  doi          = {10.1038/s42256-021-00361-8},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {705-715},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Integration of millions of transcriptomes using batch-aware triplet neural networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning prediction of patient response time course
from early data via neural-pharmacokinetic/pharmacodynamic modelling.
<em>NATMI</em>, <em>3</em>(8), 696–704. (<a
href="https://doi.org/10.1038/s42256-021-00357-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Longitudinal analyses of patient response time courses following doses of therapeutics are currently performed using pharmacokinetic/pharmacodynamic (PK/PD) methodologies, which require considerable human experience and expertise in the modelling of dynamical systems. By utilizing recent advancements in deep learning, we show that the governing differential equations can be learned directly from longitudinal patient data. In particular, we propose a novel neural-PK/PD framework that combines key pharmacological principles with neural ordinary differential equations. We applied it to an analysis of drug concentration and platelet response from a clinical dataset consisting of over 600 patients. We show that the neural-PK/PD model improves on a state-of-the-art model with respect to metrics for temporal prediction. Furthermore, by incorporating key PK/PD concepts into its architecture, the model can generalize and enable the simulations of patient responses to untested dosing regimens. These results demonstrate the potential of neural-PK/PD for automated predictive analytics of patient response time course. The response of the body to drugs follows complex dynamical processes that can be difficult to predict. Lu and colleagues combine a neural network approach with pharmacokinetic/pharmacodynamic modelling to learn these complex dynamics.},
  archive      = {J_NATMI},
  author       = {Lu, James and Bender, Brendan and Jin, Jin Y. and Guan, Yuanfang},
  doi          = {10.1038/s42256-021-00357-4},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {696-704},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning prediction of patient response time course from early data via neural-pharmacokinetic/pharmacodynamic modelling},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Learning equilibria in symmetric auction games using
artificial neural networks. <em>NATMI</em>, <em>3</em>(8), 687–695. (<a
href="https://doi.org/10.1038/s42256-021-00365-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Auction theory is of central importance in the study of markets. Unfortunately, we do not know equilibrium bidding strategies for most auction games. For realistic markets with multiple items and value interdependencies, the Bayes Nash equilibria (BNEs) often turn out to be intractable systems of partial differential equations. Previous numerical techniques have relied either on calculating pointwise best responses in strategy space or iteratively solving restricted subgames. We present a learning method that represents strategies as neural networks and applies policy iteration on the basis of gradient dynamics in self-play to provably learn local equilibria. Our empirical results show that these approximated BNEs coincide with the global equilibria whenever available. The method follows the simultaneous gradient of the game and uses a smoothing technique to circumvent discontinuities in the ex post utility functions of auction games. Discontinuities arise at the bid value where an infinite small change would make the difference between winning and not winning. Convergence to local BNEs can be explained by the fact that bidders in most auction models are symmetric, which leads to potential games for which gradient dynamics converge. Auction games present an interesting challenge for multi-agent learning. Finding the Bayes Nash equilibria for optimum bidding strategies is intractable for numerical approaches. In a new, deep learning approach, strategies are represented as neural networks, and policy iteration based on gradient dynamics in self-play enables learning of local equilibria.},
  archive      = {J_NATMI},
  author       = {Bichler, Martin and Fichtl, Maximilian and Heidekrüger, Stefan and Kohring, Nils and Sutterer, Paul},
  doi          = {10.1038/s42256-021-00365-4},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {687-695},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning equilibria in symmetric auction games using artificial neural networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Automatic heterogeneous quantization of deep neural networks
for low-latency inference on the edge for particle detectors.
<em>NATMI</em>, <em>3</em>(8), 675–686. (<a
href="https://doi.org/10.1038/s42256-021-00356-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although the quest for more accurate solutions is pushing deep learning research towards larger and more complex algorithms, edge devices demand efficient inference and therefore reduction in model size, latency and energy consumption. One technique to limit model size is quantization, which implies using fewer bits to represent weights and biases. Such an approach usually results in a decline in performance. Here, we introduce a method for designing optimally heterogeneously quantized versions of deep neural network models for minimum-energy, high-accuracy, nanosecond inference and fully automated deployment on chip. With a per-layer, per-parameter type automatic quantization procedure, sampling from a wide range of quantizers, model energy consumption and size are minimized while high accuracy is maintained. This is crucial for the event selection procedure in proton–proton collisions at the CERN Large Hadron Collider, where resources are strictly limited and a latency of $${\mathcal{O}}(1)\,\upmu{\rm{s}}$$ is required. Nanosecond inference and a resource consumption reduced by a factor of 50 when implemented on field-programmable gate array hardware are achieved. With edge computing on custom hardware, real-time inference with deep neural networks can reach the nanosecond timescale. An important application in this regime is event processing at particle collision detectors like those at the Large Hadron Collider (LHC). To ensure high performance as well as reduced resource consumption, a method is developed, and made available as an extension of the Keras library, to automatically design optimal quantization of the different layers in a deep neural network.},
  archive      = {J_NATMI},
  author       = {Coelho, Claudionor N. and Kuusela, Aki and Li, Shan and Zhuang, Hao and Ngadiuba, Jennifer and Aarrestad, Thea Klaeboe and Loncar, Vladimir and Pierini, Maurizio and Pol, Adrian Alan and Summers, Sioni},
  doi          = {10.1038/s42256-021-00356-5},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {675-686},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Automatic heterogeneous quantization of deep neural networks for low-latency inference on the edge for particle detectors},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards neural earth system modelling by integrating
artificial intelligence in earth system science. <em>NATMI</em>,
<em>3</em>(8), 667–674. (<a
href="https://doi.org/10.1038/s42256-021-00374-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Earth system models (ESMs) are our main tools for quantifying the physical state of the Earth and predicting how it might change in the future under ongoing anthropogenic forcing. In recent years, however, artificial intelligence (AI) methods have been increasingly used to augment or even replace classical ESM tasks, raising hopes that AI could solve some of the grand challenges of climate science. In this Perspective we survey the recent achievements and limitations of both process-based models and AI in Earth system and climate research, and propose a methodological transformation in which deep neural networks and ESMs are dismantled as individual approaches and reassembled as learning, self-validating and interpretable ESM–network hybrids. Following this path, we coin the term neural Earth system modelling. We examine the concurrent potential and pitfalls of neural Earth system modelling and discuss the open question of whether AI can bolster ESMs or even ultimately render them obsolete. In the past few years, AI approaches have been used to enhance Earth and climate modelling. This Perspective examines the opportunity to go further, and build from scratch hybrid systems that integrate AI tools and models based on physical process knowledge to make more efficient use of daily observational data streams.},
  archive      = {J_NATMI},
  author       = {Irrgang, Christopher and Boers, Niklas and Sonnewald, Maike and Barnes, Elizabeth A. and Kadow, Christopher and Staneva, Joanna and Saynisch-Wagner, Jan},
  doi          = {10.1038/s42256-021-00374-3},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {667-674},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards neural earth system modelling by integrating artificial intelligence in earth system science},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Machine learning and algorithmic fairness in public and
population health. <em>NATMI</em>, <em>3</em>(8), 659–666. (<a
href="https://doi.org/10.1038/s42256-021-00373-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Until now, much of the work on machine learning and health has focused on processes inside the hospital or clinic. However, this represents only a narrow set of tasks and challenges related to health; there is greater potential for impact by leveraging machine learning in health tasks more broadly. In this Perspective we aim to highlight potential opportunities and challenges for machine learning within a holistic view of health and its influences. To do so, we build on research in population and public health that focuses on the mechanisms between different cultural, social and environmental factors and their effect on the health of individuals and communities. We present a brief introduction to research in these fields, data sources and types of tasks, and use these to identify settings where machine learning is relevant and can contribute to new knowledge. Given the key foci of health equity and disparities within public and population health, we juxtapose these topics with the machine learning subfield of algorithmic fairness to highlight specific opportunities where machine learning, public and population health may synergize to achieve health equity. Algorithmic solutions to improve treatment are starting to transform health care. Mhasawade and colleagues discuss in this Perspective how machine learning applications in population and public health can extend beyond clinical practice. While working with general health data comes with its own challenges, most notably ensuring algorithmic fairness in the face of existing health disparities, the area provides new kinds of data and questions for the machine learning community.},
  archive      = {J_NATMI},
  author       = {Mhasawade, Vishwali and Zhao, Yuan and Chunara, Rumi},
  doi          = {10.1038/s42256-021-00373-4},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {659-666},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine learning and algorithmic fairness in public and population health},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Playing with symmetry with neural networks. <em>NATMI</em>,
<em>3</em>(8), 658. (<a
href="https://doi.org/10.1038/s42256-021-00380-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finding the optimum design of a complex auction is a challenging and important economic problem. Multi-agent deep learning can help find equilibria by making use of inherent symmetries in bidding strategies.},
  archive      = {J_NATMI},
  author       = {Parkes, David C.},
  doi          = {10.1038/s42256-021-00380-5},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {658},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Playing with symmetry with neural networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Resource-efficient inference for particle physics.
<em>NATMI</em>, <em>3</em>(8), 656–657. (<a
href="https://doi.org/10.1038/s42256-021-00381-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Selecting interesting proton–proton collisions from the millions taking place each second in the Large Hadron Collider is a challenging task. A neural network optimized for a field-programmable gate array hardware enables 60 ns inference and reduces power consumption by a factor of 50.},
  archive      = {J_NATMI},
  author       = {Rousseau, David},
  doi          = {10.1038/s42256-021-00381-4},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {656-657},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Resource-efficient inference for particle physics},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Road safety, health inequity and the imminence of autonomous
vehicles. <em>NATMI</em>, <em>3</em>(8), 654–655. (<a
href="https://doi.org/10.1038/s42256-021-00382-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Nunes, Ashley and Axhausen, Kay W.},
  doi          = {10.1038/s42256-021-00382-3},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {654-655},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Road safety, health inequity and the imminence of autonomous vehicles},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Striving for health equity with machine learning.
<em>NATMI</em>, <em>3</em>(8), 653. (<a
href="https://doi.org/10.1038/s42256-021-00385-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Health disparities need to be addressed so that the benefits of medical progress are not limited to selected groups. Big data and machine learning approaches are transformative tools for public and population health, but need ongoing support from insights in algorithmic fairness.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00385-0},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {653},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Striving for health equity with machine learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Back to the future with machine
learning. <em>NATMI</em>, <em>3</em>(7), 652. (<a
href="https://doi.org/10.1038/s42256-021-00372-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Korsunska, Ania and Fajgenbaum, David C.},
  doi          = {10.1038/s42256-021-00372-5},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {652},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Back to the future with machine learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep recurrent optical flow learning for particle image
velocimetry data. <em>NATMI</em>, <em>3</em>(7), 641–651. (<a
href="https://doi.org/10.1038/s42256-021-00369-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A wide range of problems in applied physics and engineering involve learning physical displacement fields from data. In this paper we propose a deep neural network-based approach for learning displacement fields in an end-to-end manner, focusing on the specific case of particle image velocimetry (PIV), a key approach in experimental fluid dynamics that is of crucial importance in diverse applications such as automotive, aerospace and biomedical engineering. The current state of the art in PIV data processing involves traditional handcrafted models that are subject to limitations including the substantial manual effort required and difficulties in generalizing across conditions. By contrast, the deep learning-based approach introduced in this paper, which is based on a recent optical flow learning architecture known as recurrent all-pairs field transforms, is general, largely automated and provides high spatial resolution. Extensive experiments, including benchmark examples where true gold standards are available for comparison, demonstrate that the proposed approach achieves state-of-the-art accuracy and generalization to new data, relative to both classical approaches and previously proposed optical flow learning schemes. Particle image velocimetry is an imaging technique to determine the velocity components of flow fields, of use in a range of complex engineering problems including in environmental, aerospace and biomedical engineering. A recurrent neural network-based approach for learning displacement fields in an end-to-end manner is applied to this technique and achieves state-of-the-art accuracy and, moreover, allows generalization to new data, eliminating the need for traditional handcrafted models.},
  archive      = {J_NATMI},
  author       = {Lagemann, Christian and Lagemann, Kai and Mukherjee, Sach and Schröder, Wolfgang},
  doi          = {10.1038/s42256-021-00369-0},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {641-651},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep recurrent optical flow learning for particle image velocimetry data},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modular deep learning enables automated identification of
monoclonal cell lines. <em>NATMI</em>, <em>3</em>(7), 632–640. (<a
href="https://doi.org/10.1038/s42256-021-00354-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monoclonalization refers to the isolation and expansion of a single cell derived from a cultured population. This is a valuable step in cell culture that serves to minimize a cell line’s technical variability downstream of cell-altering events, such as reprogramming or gene editing, as well as for processes such as monoclonal antibody development. However, traditional methods for verifying clonality do not scale well, posing a critical obstacle to studies involving large cohorts. Without automated, standardized methods for assessing clonality post hoc, methods involving monoclonalization cannot be reliably upscaled without exacerbating the technical variability of cell lines. Here, we report the design of a deep learning workflow that automatically detects colony presence and identifies clonality from cellular imaging. The workflow, termed Monoqlo, integrates multiple convolutional neural networks and, critically, leverages the chronological directionality of the cell-culturing process. Our algorithm design provides a fully scalable, highly interpretable framework that is capable of analysing industrial data volumes in under an hour using commodity hardware. We focus here on monoclonalization of human induced pluripotent stem cells, but our method is generalizable. Monoqlo standardizes the monoclonalization process, enabling colony selection protocols to be infinitely upscaled while minimizing technical variability. Monoclonalization, the isolation and expansion of a single cell derived from a cultured population, is an essential step in large-scale human cell culture and experiments. A new deep learning-based workflow called Monoqlo automatically detects colony presence and identifies clonality from cellular imaging, enabling single-cell selection protocols to be scalable while minimizing technical variability.},
  archive      = {J_NATMI},
  author       = {Fischbacher, Brodie and Hedaya, Sarita and Hartley, Brigham J. and Wang, Zhongwei and Lallos, Gregory and Hutson, Dillion and Zimmer, Matthew and Brammer, Jacob and Paull, Daniel},
  doi          = {10.1038/s42256-021-00354-7},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {632-640},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Modular deep learning enables automated identification of monoclonal cell lines},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improving performance of deep learning models with axiomatic
attribution priors and expected gradients. <em>NATMI</em>,
<em>3</em>(7), 620–631. (<a
href="https://doi.org/10.1038/s42256-021-00343-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent research has demonstrated that feature attribution methods for deep networks can themselves be incorporated into training; these attribution priors optimize for a model whose attributions have certain desirable properties—most frequently, that particular features are important or unimportant. These attribution priors are often based on attribution methods that are not guaranteed to satisfy desirable interpretability axioms, such as completeness and implementation invariance. Here we introduce attribution priors to optimize for higher-level properties of explanations, such as smoothness and sparsity, enabled by a fast new attribution method formulation called expected gradients that satisfies many important interpretability axioms. This improves model performance on many real-world tasks where previous attribution priors fail. Our experiments show that the gains from combining higher-level attribution priors with expected gradients attributions are consistent across image, gene expression and healthcare datasets. We believe that this work motivates and provides the necessary tools to support the widespread adoption of axiomatic attribution priors in many areas of applied machine learning. The implementations and our results have been made freely available to academic communities. Neural networks are becoming increasingly popular for applications in various domains, but in practice, further methods are necessary to make sure the models are learning patterns that agree with prior knowledge about the domain. A new approach introduces an explanation method, called ‘expected gradients’, that enables training with theoretically motivated feature attribution priors, to improve model performance on real-world tasks.},
  archive      = {J_NATMI},
  author       = {Erion, Gabriel and Janizek, Joseph D. and Sturmfels, Pascal and Lundberg, Scott M. and Lee, Su-In},
  doi          = {10.1038/s42256-021-00343-w},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {620-631},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Improving performance of deep learning models with axiomatic attribution priors and expected gradients},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AI for radiographic COVID-19 detection selects shortcuts
over signal. <em>NATMI</em>, <em>3</em>(7), 610–619. (<a
href="https://doi.org/10.1038/s42256-021-00338-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence (AI) researchers and radiologists have recently reported AI systems that accurately detect COVID-19 in chest radiographs. However, the robustness of these systems remains unclear. Using state-of-the-art techniques in explainable AI, we demonstrate that recent deep learning systems to detect COVID-19 from chest radiographs rely on confounding factors rather than medical pathology, creating an alarming situation in which the systems appear accurate, but fail when tested in new hospitals. We observe that the approach to obtain training data for these AI systems introduces a nearly ideal scenario for AI to learn these spurious ‘shortcuts’. Because this approach to data collection has also been used to obtain training data for the detection of COVID-19 in computed tomography scans and for medical imaging tasks related to other diseases, our study reveals a far-reaching problem in medical-imaging AI. In addition, we show that evaluation of a model on external data is insufficient to ensure AI systems rely on medically relevant pathology, because the undesired ‘shortcuts’ learned by AI systems may not impair performance in new hospitals. These findings demonstrate that explainable AI should be seen as a prerequisite to clinical deployment of machine-learning healthcare models. The urgency of the developing COVID-19 epidemic has led to a large number of novel diagnostic approaches, many of which use machine learning. DeGrave and colleagues use explainable AI techniques to analyse a selection of these approaches and find that the methods frequently learn to identify features unrelated to the actual disease.},
  archive      = {J_NATMI},
  author       = {DeGrave, Alex J. and Janizek, Joseph D. and Lee, Su-In},
  doi          = {10.1038/s42256-021-00338-7},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {610-619},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI for radiographic COVID-19 detection selects shortcuts over signal},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improved protein structure prediction by deep learning
irrespective of co-evolution information. <em>NATMI</em>, <em>3</em>(7),
601–609. (<a href="https://doi.org/10.1038/s42256-021-00348-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the tertiary structure of a protein from its primary sequence has been greatly improved by integrating deep learning and co-evolutionary analysis, as shown in CASP13 and CASP14. We describe our latest study of this idea, analysing the efficacy of network size and co-evolution data and its performance on both natural and designed proteins. We show that a large ResNet (convolutional residual neural networks) can predict structures of correct folds for 26 out of 32 CASP13 free-modelling targets and L/5 long-range contacts with precision over 80\%. When co-evolution is not used, ResNet can still predict structures of correct folds for 18 CASP13 free-modelling targets, greatly exceeding previous methods that do not use co-evolution either. Even with only the primary sequence, ResNet can predict the structures of correct folds for all tested human-designed proteins. In addition, ResNet may fare better for the designed proteins when trained without co-evolution than with co-evolution. These results suggest that ResNet does not simply de-noise co-evolution signals, but instead may learn important protein sequence–structure relationships. This has important implications for protein design and engineering, especially when co-evolutionary data are unavailable. In the last few years, computational protein structure prediction has greatly advanced by combining deep learning including convolutional residual networks (ResNet) with co-evolution data. A new study finds that using deeper and wider ResNets improves predictions in the absence of co-evolution information, suggesting that the ResNets do not not simply de-noise co-evolution signals, but instead may learn important protein sequence–structure relationships.},
  archive      = {J_NATMI},
  author       = {Xu, Jinbo and McPartlon, Matthew and Li, Jin},
  doi          = {10.1038/s42256-021-00348-5},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {601-609},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Improved protein structure prediction by deep learning irrespective of co-evolution information},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Segmentation of neurons from fluorescence calcium recordings
beyond real time. <em>NATMI</em>, <em>3</em>(7), 590–600. (<a
href="https://doi.org/10.1038/s42256-021-00342-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescent genetically encoded calcium indicators and two-photon microscopy help understand brain function by generating large-scale in vivo recordings in multiple animal models. Automatic, fast and accurate active neuron segmentation is critical when processing these videos. Here we developed and characterized a novel method, Shallow U-Net Neuron Segmentation (SUNS), to quickly and accurately segment active neurons from two-photon fluorescence imaging videos. We used temporal filtering and whitening schemes to extract temporal features associated with active neurons, and used a compact shallow U-Net to extract spatial features of neurons. Our method was both more accurate and an order of magnitude faster than state-of-the-art techniques when processing multiple datasets acquired by independent experimental groups; the difference in accuracy was enlarged when processing datasets containing few manually marked ground truths. We also developed an online version, potentially enabling real-time feedback neuroscience experiments. Calcium imaging is a valuable tool for recording in vivo neural activity, but the task of extracting signals of individual neurons is computationally challenging. Bao and colleagues present a U-Net-based method that is both accurate and fast enough to potentially allow real-time processing and closed-loop experiments.},
  archive      = {J_NATMI},
  author       = {Bao, Yijun and Soltanian-Zadeh, Somayyeh and Farsiu, Sina and Gong, Yiyang},
  doi          = {10.1038/s42256-021-00342-x},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {590-600},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Segmentation of neurons from fluorescence calcium recordings beyond real time},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Research community dynamics behind popular AI benchmarks.
<em>NATMI</em>, <em>3</em>(7), 581–589. (<a
href="https://doi.org/10.1038/s42256-021-00339-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The widespread use of experimental benchmarks in AI research has created competition and collaboration dynamics that are still poorly understood. Here we provide an innovative methodology to explore these dynamics and analyse the way different entrants in these challenges, from academia to tech giants, behave and react depending on their own or others’ achievements. We perform an analysis of 25 popular benchmarks in AI from Papers With Code, with around 2,000 result entries overall, connected with their underlying research papers. We identify links between researchers and institutions (that is, communities) beyond the standard co-authorship relations, and we explore a series of hypotheses about their behaviour as well as some aggregated results in terms of activity, performance jumps and efficiency. We characterize the dynamics of research communities at different levels of abstraction, including organization, affiliation, trajectories, results and activity. We find that hybrid, multi-institution and persevering communities are more likely to improve state-of-the-art performance, which becomes a watershed for many community members. Although the results cannot be extrapolated beyond our selection of popular machine learning benchmarks, the methodology can be extended to other areas of artificial intelligence or robotics, and combined with bibliometric studies. Experimental benchmarks such as ImageNet and Atari games play an important part in advancing artificial intelligence research. An analysis of results and papers linked to 25 popular benchmarks shows that research dynamics beyond conventional co-authorship has developed in this area.},
  archive      = {J_NATMI},
  author       = {Martínez-Plumed, Fernando and Barredo, Pablo and hÉigeartaigh, Seán Ó and Hernández-Orallo, José},
  doi          = {10.1038/s42256-021-00339-6},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {581-589},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Research community dynamics behind popular AI benchmarks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reusability report: Compressing regulatory networks to
vectors for interpreting gene expression and genetic variants.
<em>NATMI</em>, <em>3</em>(7), 576–580. (<a
href="https://doi.org/10.1038/s42256-021-00371-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Zeng, Wanwen and Xin, Jingxue and Jiang, Rui and Wang, Yong},
  doi          = {10.1038/s42256-021-00371-6},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {576-580},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Compressing regulatory networks to vectors for interpreting gene expression and genetic variants},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reusability report: Learning the language of synthetic
methods used in medicinal chemistry. <em>NATMI</em>, <em>3</em>(7),
572–575. (<a href="https://doi.org/10.1038/s42256-021-00367-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Janet, Jon Paul and Tomberg, Anna and Boström, Jonas},
  doi          = {10.1038/s42256-021-00367-2},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {572-575},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Learning the language of synthetic methods used in medicinal chemistry},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Governing AI safety through independent audits.
<em>NATMI</em>, <em>3</em>(7), 566–571. (<a
href="https://doi.org/10.1038/s42256-021-00370-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Highly automated systems are becoming omnipresent. They range in function from self-driving vehicles to advanced medical diagnostics and afford many benefits. However, there are assurance challenges that have become increasingly visible in high-profile crashes and incidents. Governance of such systems is critical to garner widespread public trust. Governance principles have been previously proposed offering aspirational guidance to automated system developers; however, their implementation is often impractical given the excessive costs and processes required to enact and then enforce the principles. This Perspective, authored by an international and multidisciplinary team across government organizations, industry and academia, proposes a mechanism to drive widespread assurance of highly automated systems: independent audit. As proposed, independent audit of AI systems would embody three ‘AAA’ governance principles of prospective risk Assessments, operation Audit trails and system Adherence to jurisdictional requirements. Independent audit of AI systems serves as a pragmatic approach to an otherwise burdensome and unenforceable assurance challenge. As highly automated systems become pervasive in society, enforceable governance principles are needed to ensure safe deployment. This Perspective proposes a pragmatic approach where independent audit of AI systems is central. The framework would embody three AAA governance principles: prospective risk Assessments, operation Audit trails and system Adherence to jurisdictional requirements.},
  archive      = {J_NATMI},
  author       = {Falco, Gregory and Shneiderman, Ben and Badger, Julia and Carrier, Ryan and Dahbura, Anton and Danks, David and Eling, Martin and Goodloe, Alwyn and Gupta, Jerry and Hart, Christopher and Jirotka, Marina and Johnson, Henric and LaPointe, Cara and Llorens, Ashley J. and Mackworth, Alan K. and Maple, Carsten and Pálsson, Sigurður Emil and Pasquale, Frank and Winfield, Alan and Yeong, Zee Kin},
  doi          = {10.1038/s42256-021-00370-7},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {566-571},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Governing AI safety through independent audits},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Machine learning and computation-enabled intelligent sensor
design. <em>NATMI</em>, <em>3</em>(7), 556–565. (<a
href="https://doi.org/10.1038/s42256-021-00360-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past several decades the dramatic increase in the availability of computational resources, coupled with the maturation of machine learning, has profoundly impacted sensor technology. In this Perspective, we discuss computational sensing with a focus on intelligent sensor system design. By leveraging inverse design and machine learning techniques, data acquisition hardware can be fundamentally redesigned to ‘lock-in’ to the optimal sensing data with respect to a user-defined cost function or design constraint. We envision a new generation of computational sensing systems that reduce the data burden while also improving sensing capabilities, enabling low-cost and compact sensor implementations engineered through iterative analysis of data-driven sensing outcomes. We believe that the methodologies discussed in this Perspective will permeate the design phase of sensing hardware, and thereby will fundamentally change and challenge traditional, intuition-driven sensor and readout designs in favour of application-targeted and perhaps highly non-intuitive implementations. Such computational sensors enabled by machine learning can therefore foster new and widely distributed applications that will benefit from ‘big data’ analytics and the internet of things to create powerful sensing networks, impacting various fields, including for example, biomedical diagnostics, environmental sensing and global health, among others. Traditional sensing techniques apply computational analysis at the output of the sensor hardware to separate signal from noise. A new, more holistic and potentially more powerful approach proposed in this Perspective is designing intelligent sensor systems that ‘lock-in’ to optimal sensing of data, making use of machine leaning strategies.},
  archive      = {J_NATMI},
  author       = {Ballard, Zachary and Brown, Calvin and Madni, Asad M. and Ozcan, Aydogan},
  doi          = {10.1038/s42256-021-00360-9},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {556-565},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine learning and computation-enabled intelligent sensor design},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AI on the beach. <em>NATMI</em>, <em>3</em>(7), 555. (<a
href="https://doi.org/10.1038/s42256-021-00375-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COVID-19 pandemic is not over and the future is uncertain, but there has lately been a semblance of what life was like before. As thoughts turn to the possibility of a summer holiday, we offer suggestions for books and podcasts on AI to refresh the mind.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00375-2},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {555},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI on the beach},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Predictive control of aerial swarms in cluttered
environments. <em>NATMI</em>, <em>3</em>(6), 545–554. (<a
href="https://doi.org/10.1038/s42256-021-00341-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Classical models of aerial swarms often describe global coordinated motion as the combination of local interactions that happen at the individual level. Mathematically, these interactions are represented with potential fields. Despite their explanatory success, these models fail to guarantee rapid and safe collective motion when applied to aerial robotic swarms flying in cluttered environments of the real world, such as forests and urban areas. Moreover, these models necessitate a tight coupling with the deployment scenarios to induce consistent swarm behaviours. Here, we propose a predictive model that incorporates the local principles of potential field models in an objective function and optimizes those principles under the knowledge of the agents’ dynamics and environment. We show that our approach improves the speed, order and safety of the swarm, it is independent of the environment layout and is scalable in the swarm speed and inter-agent distance. Our model is validated with a swarm of five quadrotors that can successfully navigate in a real-world indoor environment populated with obstacles. The movement of drone swarms can be coordinated using virtual potential fields to reach a global goal and avoid local collisions. Soria et al. propose here to extend potential fields with a predictive model that takes into account the agents’ flight dynamics to improve the speed and safety of the swarm.},
  archive      = {J_NATMI},
  author       = {Soria, Enrica and Schiano, Fabrizio and Floreano, Dario},
  doi          = {10.1038/s42256-021-00341-y},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {545-554},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predictive control of aerial swarms in cluttered environments},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Simultaneous deep generative modelling and clustering of
single-cell genomic data. <em>NATMI</em>, <em>3</em>(6), 536–544. (<a
href="https://doi.org/10.1038/s42256-021-00333-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in single-cell technologies, including single-cell ATAC-seq (scATAC-seq), have enabled large-scale profiling of the chromatin accessibility landscape at the single-cell level. However, the characteristics of scATAC-seq data, including high sparsity and high dimensionality, have greatly complicated the computational analysis. Here, we propose scDEC, a computational tool for scATAC-seq analysis with deep generative neural networks. scDEC is built on a pair of generative adversarial networks, and is capable of simultaneously learning the latent representation and inferring cell labels. In a series of experiments, scDEC demonstrates superior performance over other tools in scATAC-seq analysis across multiple datasets and experimental settings. In downstream applications, we demonstrate that the generative power of scDEC helps to infer the trajectory and intermediate state of cells during differentiation and the latent features learned by scDEC can potentially reveal both biological cell types and within-cell-type variations. We also show that it is possible to extend scDEC for the integrative analysis of multi-modal single cell data. Although technologies enable large-scale profiling of chromatin accessibility at the single-cell level, there are methodological challenges due to high dimensionality and high sparsity of data. Liu and colleagues describe a computational tool for the simultaneous determination of latent representation and clustering of cells from single-cell ATAC-seq data using a pair of generative adversarial networks.},
  archive      = {J_NATMI},
  author       = {Liu, Qiao and Chen, Shengquan and Jiang, Rui and Wong, Wing Hung},
  doi          = {10.1038/s42256-021-00333-y},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {536-544},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Simultaneous deep generative modelling and clustering of single-cell genomic data},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A machine learning platform to estimate anti-SARS-CoV-2
activities. <em>NATMI</em>, <em>3</em>(6), 527–535. (<a
href="https://doi.org/10.1038/s42256-021-00335-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies for drug discovery and repositioning are urgently need with respect to COVID-19. Here we present REDIAL-2020, a suite of computational models for estimating small molecule activities in a range of SARS-CoV-2-related assays. Models were trained using publicly available, high-throughput screening data and by employing different descriptor types and various machine learning strategies. Here we describe the development and use of eleven models that span across the areas of viral entry, viral replication, live virus infectivity, in vitro infectivity and human cell toxicity. REDIAL-2020 is available as a web application through the DrugCentral web portal ( http://drugcentral.org/Redial ). The web application also provides similarity search results that display the most similar molecules to the query, as well as associated experimental data. REDIAL-2020 can serve as a rapid online tool for identifying active molecules for COVID-19 treatment. There is an urgent need to identify drugs that may be effective against SARS-CoV-2. A platform with a range of machine learning models is made available to predict anti-COVID-19 activity in candidate drugs and to help prioritize compounds for virtual screening.},
  archive      = {J_NATMI},
  author       = {KC, Govinda B. and Bocci, Giovanni and Verma, Srijan and Hassan, Md Mahmudulla and Holmes, Jayme and Yang, Jeremy J. and Sirimulla, Suman and Oprea, Tudor I.},
  doi          = {10.1038/s42256-021-00335-w},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {527-535},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A machine learning platform to estimate anti-SARS-CoV-2 activities},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Integration of multiomics data with graph convolutional
networks to identify new cancer genes and their associated molecular
mechanisms. <em>NATMI</em>, <em>3</em>(6), 513–526. (<a
href="https://doi.org/10.1038/s42256-021-00325-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The increase in available high-throughput molecular data creates computational challenges for the identification of cancer genes. Genetic as well as non-genetic causes contribute to tumorigenesis, and this necessitates the development of predictive models to effectively integrate different data modalities while being interpretable. We introduce EMOGI, an explainable machine learning method based on graph convolutional networks to predict cancer genes by combining multiomics pan-cancer data—such as mutations, copy number changes, DNA methylation and gene expression—together with protein–protein interaction (PPI) networks. EMOGI was on average more accurate than other methods across different PPI networks and datasets. We used layer-wise relevance propagation to stratify genes according to whether their classification was driven by the interactome or any of the omics levels, and to identify important modules in the PPI network. We propose 165 novel cancer genes that do not necessarily harbour recurrent alterations but interact with known cancer genes, and we show that they correspond to essential genes from loss-of-function screens. We believe that our method can open new avenues in precision oncology and be applied to predict biomarkers for other complex diseases. Identifying cancer driver genes from high-throughput genomic data is an important task to understand the molecular basis of cancer and to develop new treatments including precision medicine. To tackle this challenge, EMOGI, a new deep learning method based on graph convolutional networks is developed, which combines protein–protein interaction networks with multiomics datasets.},
  archive      = {J_NATMI},
  author       = {Schulte-Sasse, Roman and Budach, Stefan and Hnisz, Denes and Marsico, Annalisa},
  doi          = {10.1038/s42256-021-00325-y},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {513-526},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Integration of multiomics data with graph convolutional networks to identify new cancer genes and their associated molecular mechanisms},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Integration of deep learning and soft robotics for a
biomimetic approach to nonlinear sensing. <em>NATMI</em>, <em>3</em>(6),
507–512. (<a href="https://doi.org/10.1038/s42256-021-00330-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Traditional approaches to sensing have often been aimed at simple sensor characteristics to make interpretation of the sensor outputs easier, but this has also limited the quality of the encoded sensory information. Integrating a complex sensor with deep learning could hence be a strategy for removing current limitations on the information that sensory inputs can carry. Here, we demonstrate this concept with a soft-robotic sensor that mimics fast non-rigid deformation of the ears in certain bat species. We show that a deep convolutional neural network can use the nonlinear Doppler shift signatures generated by these motions to estimate the direction of a sound source with an estimation error of ~0.5°. Previously, determining the direction of a sound source based on pressure receivers required either multiple frequencies or multiple receivers. Our current results demonstrate a third approach that makes do with only a single frequency and a single receiver. Bats with sophisticated biosonar systems move their ears at a high speed to help localize sound sources. Yin and Müller present a system inspired by this strategy, which can localize sounds with high accuracy and with a single detector, using a flexible silicone model of a bat’s ear and a deep convolutional neural network to process the complex Doppler signatures.},
  archive      = {J_NATMI},
  author       = {Yin, Xiaoyan and Müller, Rolf},
  doi          = {10.1038/s42256-021-00330-1},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {507-512},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Integration of deep learning and soft robotics for a biomimetic approach to nonlinear sensing},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Unsupervised behaviour analysis and magnification (uBAM)
using deep learning. <em>NATMI</em>, <em>3</em>(6), 495–506. (<a
href="https://doi.org/10.1038/s42256-021-00326-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motor behaviour analysis is essential to biomedical research and clinical diagnostics as it provides a non-invasive strategy for identifying motor impairment and its change caused by interventions. State-of-the-art instrumented movement analysis is time- and cost-intensive, because it requires the placement of physical or virtual markers. As well as the effort required for marking the keypoints or annotations necessary for training or fine-tuning a detector, users need to know the interesting behaviour beforehand to provide meaningful keypoints. Here, we introduce unsupervised behaviour analysis and magnification (uBAM), an automatic deep learning algorithm for analysing behaviour by discovering and magnifying deviations. A central aspect is unsupervised learning of posture and behaviour representations to enable an objective comparison of movement. Besides discovering and quantifying deviations in behaviour, we also propose a generative model for visually magnifying subtle behaviour differences directly in a video without requiring a detour via keypoints or annotations. Essential for this magnification of deviations, even across different individuals, is a disentangling of appearance and behaviour. Evaluations on rodents and human patients with neurological diseases demonstrate the wide applicability of our approach. Moreover, combining optogenetic stimulation with our unsupervised behaviour analysis shows its suitability as a non-invasive diagnostic tool correlating function to brain plasticity. Being able to precisely analyse behaviour is essential for the study of motor behaviour in health and disease, but is often time- and labour-intensive. Brattoli et al. develop an automatic approach based on deep learning for analysing motor behaviour and evaluate it on different species and diverse motor functions.},
  archive      = {J_NATMI},
  author       = {Brattoli, Biagio and Büchler, Uta and Dorkenwald, Michael and Reiser, Philipp and Filli, Linard and Helmchen, Fritjof and Wahl, Anna-Sophia and Ommer, Björn},
  doi          = {10.1038/s42256-021-00326-x},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {495-506},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Unsupervised behaviour analysis and magnification (uBAM) using deep learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Unassisted noise reduction of chemical reaction datasets.
<em>NATMI</em>, <em>3</em>(6), 485–494. (<a
href="https://doi.org/10.1038/s42256-021-00319-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing deep learning models applied to reaction prediction in organic chemistry can reach high levels of accuracy (&amp;gt;90\% for natural language processing-based ones). With no chemical knowledge embedded other than the information learnt from reaction data, the quality of the datasets plays a crucial role in the performance of the prediction models. Human curation is prohibitively expensive, so unaided approaches to remove chemically incorrect entries from existing datasets are essential to improve the performance of artificial intelligence models in synthetic chemistry tasks. Here, we propose a machine learning-based, unassisted approach to remove chemically wrong entries from chemical reaction collections. We apply this method to the Pistachio collection of chemical reactions and to an open dataset, both extracted from United States Patent and Trademark Office patents. Our results show an improved prediction quality for models trained on the cleaned and balanced datasets. For retrosynthetic models, the roundtrip accuracy metric grows by 13 percentage points and the value of the cumulative Jensen–Shannon divergence decreases by 30\% compared to its original record. The coverage remains high at 97\%, and the value of the class diversity is not affected by the cleaning. The proposed strategy is the first unassisted rule-free technique to address automatic noise reduction in chemical datasets. The recent advances in computational chemistry rely on large data collections of chemical compounds and reactions. However, not all entries in these datasets are correct. Toniato and colleagues present here an automated approach to identify incorrect reactions, using the effect of catastrophic forgetting in neural networks.},
  archive      = {J_NATMI},
  author       = {Toniato, Alessandra and Schwaller, Philippe and Cardinale, Antonio and Geluykens, Joppe and Laino, Teodoro},
  doi          = {10.1038/s42256-021-00319-w},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {485-494},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Unassisted noise reduction of chemical reaction datasets},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). End-to-end privacy preserving deep learning on
multi-institutional medical imaging. <em>NATMI</em>, <em>3</em>(6),
473–484. (<a href="https://doi.org/10.1038/s42256-021-00337-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using large, multi-national datasets for high-performance medical imaging AI systems requires innovation in privacy-preserving machine learning so models can train on sensitive data without requiring data transfer. Here we present PriMIA (Privacy-preserving Medical Image Analysis), a free, open-source software framework for differentially private, securely aggregated federated learning and encrypted inference on medical imaging data. We test PriMIA using a real-life case study in which an expert-level deep convolutional neural network classifies paediatric chest X-rays; the resulting model’s classification performance is on par with locally, non-securely trained models. We theoretically and empirically evaluate our framework’s performance and privacy guarantees, and demonstrate that the protections provided prevent the reconstruction of usable data by a gradient-based model inversion attack. Finally, we successfully employ the trained model in an end-to-end encrypted remote inference scenario using secure multi-party computation to prevent the disclosure of the data and the model. Gaining access to medical data to train AI applications can present problems due to patient privacy or proprietary interests. A way forward can be privacy-preserving federated learning schemes. Kaissis, Ziller and colleagues demonstrate here their open source framework for privacy-preserving medical image analysis in a remote inference scenario.},
  archive      = {J_NATMI},
  author       = {Kaissis, Georgios and Ziller, Alexander and Passerat-Palmbach, Jonathan and Ryffel, Théo and Usynin, Dmitrii and Trask, Andrew and Lima, Ionésio and Mancuso, Jason and Jungmann, Friederike and Steinborn, Marc-Matthias and Saleh, Andreas and Makowski, Marcus and Rueckert, Daniel and Braren, Rickmer},
  doi          = {10.1038/s42256-021-00337-8},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {473-484},
  shortjournal = {Nat. Mach. Intell.},
  title        = {End-to-end privacy preserving deep learning on multi-institutional medical imaging},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Epistemic fragmentation poses a threat to the governance of
online targeting. <em>NATMI</em>, <em>3</em>(6), 466–472. (<a
href="https://doi.org/10.1038/s42256-021-00358-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Online targeting isolates individual consumers, causing what we call epistemic fragmentation. This phenomenon amplifies the harms of advertising and inflicts structural damage to the public forum. The two natural strategies to tackle the problem of regulating online targeted advertising, increasing consumer awareness and extending proactive monitoring, fail because even sophisticated individual consumers are vulnerable in isolation, and the contextual knowledge needed for effective proactive monitoring remains largely inaccessible to platforms and external regulators. The limitations of both consumer awareness and of proactive monitoring strategies can be attributed to their failure to address epistemic fragmentation. We call attention to a third possibility that we call a civic model of governance for online targeted advertising, which overcomes this problem, and describe four possible pathways to implement this model. Online targeted advertising fuelled by machine learning can lead to the isolation of individual consumers. This problem of ‘epistemic fragmentation’ cannot be tackled with current regulation strategies and a new, civic model of governance for advertising is needed.},
  archive      = {J_NATMI},
  author       = {Milano, Silvia and Mittelstadt, Brent and Wachter, Sandra and Russell, Christopher},
  doi          = {10.1038/s42256-021-00358-3},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {466-472},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Epistemic fragmentation poses a threat to the governance of online targeting},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Back to the future with machine learning. <em>NATMI</em>,
<em>3</em>(6), 464–465. (<a
href="https://doi.org/10.1038/s42256-021-00363-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug repurposing provides a way to identify effective treatments more quickly and economically. To speed up the search for antiviral treatment of COVID-19, a new platform provides a range of computational models to identify drugs with potential anti-COVID-19 effects.},
  archive      = {J_NATMI},
  author       = {Korsunska, Ania and Fajgenbaum, David C.},
  doi          = {10.1038/s42256-021-00363-6},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {464-465},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Back to the future with machine learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Large language models associate muslims with violence.
<em>NATMI</em>, <em>3</em>(6), 461–463. (<a
href="https://doi.org/10.1038/s42256-021-00359-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models, which are increasingly used in AI applications, display undesirable stereotypes such as persistent associations between Muslims and violence. New approaches are needed to systematically reduce the harmful bias of language models in deployment.},
  archive      = {J_NATMI},
  author       = {Abid, Abubakar and Farooqi, Maheen and Zou, James},
  doi          = {10.1038/s42256-021-00359-2},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {461-463},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Large language models associate muslims with violence},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reducing racial bias in AI models for clinical use requires
a top-down intervention. <em>NATMI</em>, <em>3</em>(6), 460. (<a
href="https://doi.org/10.1038/s42256-021-00362-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Kapur, Supriya},
  doi          = {10.1038/s42256-021-00362-7},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {460},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reducing racial bias in AI models for clinical use requires a top-down intervention},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Collaborative learning without sharing data. <em>NATMI</em>,
<em>3</em>(6), 459. (<a
href="https://doi.org/10.1038/s42256-021-00364-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate and fair medical machine learning requires large amounts and diverse data to train on. Privacy-preserving methods such as federated learning can help improve machine learning models by making use of datasets in different hospitals and institutes while the data stays where it is collected.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00364-5},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {459},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Collaborative learning without sharing data},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Dimensionality reduction to maximize
prediction generalization capability. <em>NATMI</em>, <em>3</em>(5),
457. (<a href="https://doi.org/10.1038/s42256-021-00352-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Isomura, Takuya and Toyoizumi, Taro},
  doi          = {10.1038/s42256-021-00352-9},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {457},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Dimensionality reduction to maximize prediction generalization capability},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Machine learning pipeline for battery state-of-health
estimation. <em>NATMI</em>, <em>3</em>(5), 447–456. (<a
href="https://doi.org/10.1038/s42256-021-00312-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lithium-ion batteries are ubiquitous in applications ranging from portable electronics to electric vehicles. Irrespective of the application, reliable real-time estimation of battery state of health (SOH) by on-board computers is crucial to the safe operation of the battery, ultimately safeguarding asset integrity. In this Article, we design and evaluate a machine learning pipeline for estimation of battery capacity fade—a metric of battery health—on 179 cells cycled under various conditions. The pipeline estimates battery SOH with an associated confidence interval by using two parametric and two non-parametric algorithms. Using segments of charge voltage and current curves, the pipeline engineers 30 features, performs automatic feature selection and calibrates the algorithms. When deployed on cells operated under the fast-charging protocol, the best model achieves a root-mean-squared error of 0.45\%. This work provides insights into the design of scalable data-driven models for battery SOH estimation, emphasizing the value of confidence bounds around the prediction. The pipeline methodology combines experimental data with machine learning modelling and could be applied to other critical components that require real-time estimation of SOH. Rechargeable lithium-ion batteries play a crucial role in many modern-day applications, including portable electronics and electric vehicles, but they degrade over time. To ensure safe operation, a battery’s ‘state of health’ should be monitored in real time, and this machine learning pipeline, tested on a variety of charging conditions, can provide such an online estimation of battery state of health.},
  archive      = {J_NATMI},
  author       = {Roman, Darius and Saxena, Saurabh and Robu, Valentin and Pecht, Michael and Flynn, David},
  doi          = {10.1038/s42256-021-00312-3},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {447-456},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine learning pipeline for battery state-of-health estimation},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Dimensionality reduction to maximize prediction
generalization capability. <em>NATMI</em>, <em>3</em>(5), 434–446. (<a
href="https://doi.org/10.1038/s42256-021-00306-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generalization of time series prediction remains an important open issue in machine learning; earlier methods have either large generalization errors or local minima. Here, we develop an analytically solvable, unsupervised learning scheme that extracts the most informative components for predicting future inputs, which we call predictive principal component analysis (PredPCA). Our scheme can effectively remove unpredictable noise and minimize test prediction error through convex optimization. Mathematical analyses demonstrate that, provided with sufficient training samples and sufficiently high-dimensional observations, PredPCA can asymptotically identify hidden states, system parameters and dimensionalities of canonical nonlinear generative processes, with a global convergence guarantee. We demonstrate the performance of PredPCA using sequential visual inputs comprising handwritten digits, rotating three-dimensional objects and natural scenes. It reliably estimates distinct hidden states and predicts future outcomes of previously unseen test input data, based exclusively on noisy observations. The simple architecture and low computational cost of PredPCA are highly desirable for neuromorphic hardware. Predicting what comes next in a previously unseen time series of input data is a challenging task for machine learning. A novel unsupervised learning scheme termed predictive principal component analysis can extract the most informative components for predicting future inputs with low computational cost.},
  archive      = {J_NATMI},
  author       = {Isomura, Takuya and Toyoizumi, Taro},
  doi          = {10.1038/s42256-021-00306-1},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {434-446},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Dimensionality reduction to maximize prediction generalization capability},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Variable selection with false discovery rate control in deep
neural networks. <em>NATMI</em>, <em>3</em>(5), 426–433. (<a
href="https://doi.org/10.1038/s42256-021-00308-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep neural networks are famous for their high prediction accuracy, but they are also known for their black-box nature and poor interpretability. We consider the problem of variable selection, that is, selecting the input variables that have significant predictive power on the output, in deep neural networks. Most existing variable selection methods for neural networks are only applicable to shallow networks or are computationally infeasible on large datasets; moreover, they lack a control on the quality of selected variables. Here we propose a backward elimination procedure called SurvNet, which is based on a new measure of variable importance that applies to a wide variety of networks. More importantly, SurvNet is able to estimate and control the false discovery rate of selected variables empirically. Further, SurvNet adaptively determines how many variables to eliminate at each step in order to maximize the selection efficiency. The validity and efficiency of SurvNet are shown on various simulated and real datasets, and its performance is compared with other methods. Especially, a systematic comparison with knockoff-based methods shows that although they have more rigorous false discovery rate control on data with strong variable correlation, SurvNet usually has higher power. Identifying salient input features can be a challenge in neural networks. The authors developed a variable selection procedure with false discovery rate control that works on classification or regression problems, one or multiple output neurons, and deep or shallow neural networks.},
  archive      = {J_NATMI},
  author       = {Song, Zixuan and Li, Jun},
  doi          = {10.1038/s42256-021-00308-z},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {426-433},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Variable selection with false discovery rate control in deep neural networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computationally instrument-resolution-independent de novo
peptide sequencing for high-resolution devices. <em>NATMI</em>,
<em>3</em>(5), 420–425. (<a
href="https://doi.org/10.1038/s42256-021-00304-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo peptide sequencing is the key technology for finding novel peptides from mass spectra. The overall quality of sequencing results depends on the de novo peptide sequencing algorithm as well as the quality of mass spectra. Over the past decade, the resolution and accuracy of mass spectrometers have improved by orders of magnitude and higher-resolution mass spectra have been generated. How to effectively take advantage of those high-resolution data without substantially increasing the computational complexity remains a challenge for de novo peptide sequencing tools. Here we present PointNovo, a neural network-based de novo peptide sequencing model that can robustly handle any resolution levels of mass spectrometry data while keeping the computational complexity unchanged. Our extensive experiment results show PointNovo outperforms existing de novo peptide sequencing tools by capitalizing on the ultra-high resolution of the latest mass spectrometers. Increased resolution of mass spectroscopy can provide better data for sequencing, but also increases the computational complexity of analysing the data. Qiao and colleagues present here a neural network-based method that processes sequencing data of any resolution while improving the accuracy of predicted sequences.},
  archive      = {J_NATMI},
  author       = {Qiao, Rui and Tran, Ngoc Hieu and Xin, Lei and Chen, Xin and Li, Ming and Shan, Baozhen and Ghodsi, Ali},
  doi          = {10.1038/s42256-021-00304-3},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {420-425},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Computationally instrument-resolution-independent de novo peptide sequencing for high-resolution devices},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Real-world embodied AI through a morphologically adaptive
quadruped robot. <em>NATMI</em>, <em>3</em>(5), 410–419. (<a
href="https://doi.org/10.1038/s42256-021-00320-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Robots are traditionally bound by a fixed morphology during their operational lifetime, which is limited to adapting only their control strategies. Here we present the first quadrupedal robot that can morphologically adapt to different environmental conditions in outdoor, unstructured environments. Our solution is rooted in embodied AI and comprises two components: (1) a robot that permits in situ morphological adaptation and (2) an adaptation algorithm that transitions between the most energy-efficient morphologies on the basis of the currently sensed terrain. We first build a model that describes how the robot morphology affects performance on selected terrains. We then test continuous adaptation on realistic outdoor terrain while allowing the robot to constantly update its model. We show that the robot exploits its training to effectively transition between different morphological configurations, exhibiting substantial performance improvements over a non-adaptive approach. The demonstrated benefits of real-world morphological adaptation demonstrate the potential for a new embodied way of incorporating adaptation into future robotic designs. The morphology of a robot determines how efficiently it can traverse different terrain. Nygaard and colleagues present here a robot that can adapt it’s morphology when it is detecting different terrain and learn which configuration is most effective.},
  archive      = {J_NATMI},
  author       = {Nygaard, Tønnes F. and Martin, Charles P. and Torresen, Jim and Glette, Kyrre and Howard, David},
  doi          = {10.1038/s42256-021-00320-3},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {410-419},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Real-world embodied AI through a morphologically adaptive quadruped robot},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Machine-learning-based dynamic-importance sampling for
adaptive multiscale simulations. <em>NATMI</em>, <em>3</em>(5), 401–409.
(<a href="https://doi.org/10.1038/s42256-021-00327-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiscale simulations are a well-accepted way to bridge the length and time scales required for scientific studies with the solution accuracy achievable through available computational resources. Traditional approaches either solve a coarse model with selective refinement or coerce a detailed model into faster sampling, both of which have limitations. Here, we present a paradigm of adaptive, multiscale simulations that couple different scales using a dynamic-importance sampling approach. Our method uses machine learning to dynamically and exhaustively sample the phase space explored by a macro model using microscale simulations and enables an automatic feedback from the micro to the macro scale, leading to a self-healing multiscale simulation. As a result, our approach delivers macro length and time scales, but with the effective precision of the micro scale. Our approach is arbitrarily scalable as well as transferable to many different types of simulations. Our method made possible a multiscale scientific campaign of unprecedented scale to understand the interactions of RAS proteins with a plasma membrane in the context of cancer research running over several days on Sierra, which is currently the second-most-powerful supercomputer in the world. Tackling scientific problems often requires computational models that bridge several spatial and temporal scales. A new simulation framework employing machine learning, which is scalable and can be used on standard laptops as well as supercomputers, promises exhaustive multiscale explorations.},
  archive      = {J_NATMI},
  author       = {Bhatia, Harsh and Carpenter, Timothy S. and Ingólfsson, Helgi I. and Dharuman, Gautham and Karande, Piyush and Liu, Shusen and Oppelstrup, Tomas and Neale, Chris and Lightstone, Felice C. and Van Essen, Brian and Glosli, James N. and Bremer, Peer-Timo},
  doi          = {10.1038/s42256-021-00327-w},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {401-409},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine-learning-based dynamic-importance sampling for adaptive multiscale simulations},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An automated framework for efficiently designing deep
convolutional neural networks in genomics. <em>NATMI</em>,
<em>3</em>(5), 392–400. (<a
href="https://doi.org/10.1038/s42256-021-00316-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Convolutional neural networks (CNNs) have become a standard for analysis of biological sequences. Tuning of network architectures is essential for a CNN’s performance, yet it requires substantial knowledge of machine learning and commitment of time and effort. This process thus imposes a major barrier to broad and effective application of modern deep learning in genomics. Here we present Automated Modelling for Biological Evidence-based Research (AMBER), a fully automated framework to efficiently design and apply CNNs for genomic sequences. AMBER designs optimal models for user-specified biological questions through the state-of-the-art neural architecture search (NAS). We applied AMBER to the task of modelling genomic regulatory features and demonstrated that the predictions of the AMBER-designed model are significantly more accurate than the equivalent baseline non-NAS models and match or even exceed published expert-designed models. Interpretation of AMBER architecture search revealed its design principles of utilizing the full space of computational operations for accurately modelling genomic sequences. Furthermore, we illustrated the use of AMBER to accurately discover functional genomic variants in allele-specific binding and disease heritability enrichment. AMBER provides an efficient automated method for designing accurate deep learning models in genomics. At present, deep learning models in genomics are manually tuned through trial and error, which is time consuming and imposes a barrier for biomedical researchers not trained in machine learning. The authors develop an automated framework to design and apply convolutional neural networks for genomic sequences.},
  archive      = {J_NATMI},
  author       = {Zhang, Zijun and Park, Christopher Y. and Theesfeld, Chandra L. and Troyanskaya, Olga G.},
  doi          = {10.1038/s42256-021-00316-z},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {392-400},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An automated framework for efficiently designing deep convolutional neural networks in genomics},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reusability report: Predicting spatiotemporal nonlinear
dynamics in multimode fibre optics with a recurrent neural network.
<em>NATMI</em>, <em>3</em>(5), 387–391. (<a
href="https://doi.org/10.1038/s42256-021-00347-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Teğin, Uğur and Dinç, Niyazi Ulaş and Moser, Christophe and Psaltis, Demetri},
  doi          = {10.1038/s42256-021-00347-6},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {387-391},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Predicting spatiotemporal nonlinear dynamics in multimode fibre optics with a recurrent neural network},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fitting elephants in modern machine learning by
statistically consistent interpolation. <em>NATMI</em>, <em>3</em>(5),
378–386. (<a href="https://doi.org/10.1038/s42256-021-00345-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Textbook wisdom advocates for smooth function fits and implies that interpolation of noisy data should lead to poor generalization. A related heuristic is that fitting parameters should be fewer than measurements (Occam’s razor). Surprisingly, contemporary machine learning approaches, such as deep nets, generalize well, despite interpolating noisy data. This may be understood via statistically consistent interpolation (SCI), that is, data interpolation techniques that generalize optimally for big data. Here, we elucidate SCI using the weighted interpolating nearest neighbours algorithm, which adds singular weight functions to k nearest neighbours. This shows that data interpolation can be a valid machine learning strategy for big data. SCI clarifies the relation between two ways of modelling natural phenomena: the rationalist approach (strong priors) of theoretical physics with few parameters, and the empiricist (weak priors) approach of modern machine learning with more parameters than data. SCI shows that the purely empirical approach can successfully predict. However, data interpolation does not provide theoretical insights, and the training data requirements may be prohibitive. Complex animal brains are between these extremes, with many parameters, but modest training data, and with prior structure encoded in species-specific mesoscale circuitry. Thus, modern machine learning provides a distinct epistemological approach that is different both from physical theories and animal brains. Modern machine learning approaches, such as deep neural networks, generalize well despite interpolating noisy data, in contrast with textbook wisdom. Mitra describes the phenomenon of statistically consistent interpolation (SCI) to clarify why data interpolation succeeds, and discusses how SCI elucidates the differing approaches to modelling natural phenomena represented in modern machine learning, traditional physical theory and biological brains.},
  archive      = {J_NATMI},
  author       = {Mitra, Partha P.},
  doi          = {10.1038/s42256-021-00345-8},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {378-386},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Fitting elephants in modern machine learning by statistically consistent interpolation},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards universal translation. <em>NATMI</em>,
<em>3</em>(5), 376–377. (<a
href="https://doi.org/10.1038/s42256-021-00346-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {State of the art neural network approaches enable massive multilingual translation. How close are we to universal translation between any spoken, written or signed language?},
  archive      = {J_NATMI},
  author       = {Costa-jussà, Marta R.},
  doi          = {10.1038/s42256-021-00346-7},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {376-377},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards universal translation},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Linking the length scales. <em>NATMI</em>, <em>3</em>(5),
374–375. (<a href="https://doi.org/10.1038/s42256-021-00351-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A challenge for multiscale simulations is how to link the macroscopic and microscopic length scales effectively. A new machine-learning-based sampling approach enables full exploration of macro configurations while retaining the precision of a microscale model.},
  archive      = {J_NATMI},
  author       = {Wang, Shangying and Bianco, Simone},
  doi          = {10.1038/s42256-021-00351-w},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {374-375},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Linking the length scales},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Neural network architecture search with AMBER.
<em>NATMI</em>, <em>3</em>(5), 372–373. (<a
href="https://doi.org/10.1038/s42256-021-00350-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning applied to genomics can learn patterns in biological sequences, but designing such models requires expertise and effort. Recent work demonstrates the efficiency of a neural network architecture search algorithm in optimizing genomic models.},
  archive      = {J_NATMI},
  author       = {Zhang, Yi and Liu, Yang and Liu, X. Shirley},
  doi          = {10.1038/s42256-021-00350-x},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {372-373},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Neural network architecture search with AMBER},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Defining a role for AI ethics in national security.
<em>NATMI</em>, <em>3</em>(5), 370–371. (<a
href="https://doi.org/10.1038/s42256-021-00344-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We spoke with Mariarosaria Taddeo, an associate professor and senior research fellow at the Oxford Internet Institute and Dstl Ethics Fellow at the Alan Turing Institute working on digital and AI ethics about two recent reports from the UK and the US on using AI in national defence and security.},
  archive      = {J_NATMI},
  author       = {Venema, Liesbeth},
  doi          = {10.1038/s42256-021-00344-9},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {370-371},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Defining a role for AI ethics in national security},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A human in the loop in surgery automation. <em>NATMI</em>,
<em>3</em>(5), 368–369. (<a
href="https://doi.org/10.1038/s42256-021-00349-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Fosch-Villaronga, Eduard and Khanna, Pranav and Drukarch, Hadassah and Custers, Bart H. M.},
  doi          = {10.1038/s42256-021-00349-4},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {368-369},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A human in the loop in surgery automation},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). How to be responsible in AI publication. <em>NATMI</em>,
<em>3</em>(5), 367. (<a
href="https://doi.org/10.1038/s42256-021-00355-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A white paper from Partnership on AI provides timely advice on tackling the urgent challenge of navigating risks of AI research and responsible publication.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00355-6},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {367},
  shortjournal = {Nat. Mach. Intell.},
  title        = {How to be responsible in AI publication},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Morphological and molecular breast cancer profiling through
explainable machine learning. <em>NATMI</em>, <em>3</em>(4), 355–366.
(<a href="https://doi.org/10.1038/s42256-021-00303-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in cancer research and diagnostics largely rely on new developments in microscopic or molecular profiling techniques, offering high levels of detail with respect to either spatial or molecular features, but usually not both. Here, we present an explainable machine-learning approach for the integrated profiling of morphological, molecular and clinical features from breast cancer histology. First, our approach allows for the robust detection of cancer cells and tumour-infiltrating lymphocytes in histological images, providing precise heatmap visualizations explaining the classifier decisions. Second, molecular features, including DNA methylation, gene expression, copy number variations, somatic mutations and proteins are predicted from histology. Molecular predictions reach balanced accuracies up to 78\%, whereas accuracies of over 95\% can be achieved for subgroups of patients. Finally, our explainable AI approach allows assessment of the link between morphological and molecular cancer properties. The resulting computational multiplex-histology analysis can help promote basic cancer research and precision medicine through an integrated diagnostic scoring of histological, clinical and molecular features. Cancers are complex diseases that are increasingly studied using a diverse set of omics data. At the same time, histological images show the interaction of cells, which is not visible with bulk omics methods. Binder and colleagues present a method to learn from both kinds of data, such that molecular markers can be associated with visible patterns in the tissue samples and be used for more accurate breast cancer diagnosis.},
  archive      = {J_NATMI},
  author       = {Binder, Alexander and Bockmayr, Michael and Hägele, Miriam and Wienert, Stephan and Heim, Daniel and Hellweg, Katharina and Ishii, Masaru and Stenzinger, Albrecht and Hocke, Andreas and Denkert, Carsten and Müller, Klaus-Robert and Klauschen, Frederick},
  doi          = {10.1038/s42256-021-00303-4},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {355-366},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Morphological and molecular breast cancer profiling through explainable machine learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Predicting ultrafast nonlinear dynamics in fibre optics with
a recurrent neural network. <em>NATMI</em>, <em>3</em>(4), 344–354. (<a
href="https://doi.org/10.1038/s42256-021-00297-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The propagation of ultrashort pulses in optical fibre plays a central role in the development of light sources and photonic technologies, with applications from fundamental studies of light–matter interactions to high-resolution imaging and remote sensing. However, short pulse dynamics are highly nonlinear, and optimizing pulse propagation for application purposes requires extensive and computationally demanding numerical simulations. This creates a severe bottleneck in designing and optimizing experiments in real time. Here, we present a solution to this problem using a recurrent neural network to model and predict complex nonlinear propagation in optical fibre, solely from the input pulse intensity profile. We highlight particular examples in pulse compression and ultra-broadband supercontinuum generation, and compare neural network predictions with experimental data. We also show how the approach can be generalized to model other propagation scenarios for a wider range of input conditions and fibre systems, including multimode propagation. These results open up novel perspectives in the modelling of nonlinear systems, for the development of future photonic technologies and more generally in physics for studies in Bose–Einstein condensates, plasma physics and hydrodynamics. The propagation of ultrashort pulses in optical fibres, of interest in scientific studies of nonlinear systems, depends sensitively on both the input pulse and the fibre characteristics and normally requires extensive numerical simulations. A new approach based on a recurrent neural network can predict complex nonlinear propagation in optical fibre, solely from the input pulse intensity profile, and helps to design experiments in pulse compression and ultra-broadband supercontinuum generation.},
  archive      = {J_NATMI},
  author       = {Salmela, Lauri and Tsipinakis, Nikolaos and Foi, Alessandro and Billet, Cyril and Dudley, John M. and Genty, Goëry},
  doi          = {10.1038/s42256-021-00297-z},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {344-354},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting ultrafast nonlinear dynamics in fibre optics with a recurrent neural network},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Out-of-the-box deep learning prediction of pharmaceutical
properties by broadly learned knowledge-based molecular representations.
<em>NATMI</em>, <em>3</em>(4), 334–343. (<a
href="https://doi.org/10.1038/s42256-021-00301-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Successful deep learning critically depends on the representation of the learned objects. Recent state-of-the-art pharmaceutical deep learning models successfully exploit graph-based de novo learning of molecular representations. Nonetheless, the combined potential of human expert knowledge of molecular representations and convolution neural networks has not been adequately explored for enhanced learning of pharmaceutical properties. Here we show that broader exploration of human-knowledge-based molecular representations enables more enhanced deep learning of pharmaceutical properties. By broad learning of 1,456 molecular descriptors and 16,204 fingerprint features of 8,506,205 molecules, a new feature-generation method MolMap was developed for mapping these molecular descriptors and fingerprint features into robust two-dimensional feature maps. Convolution-neural-network-based MolMapNet models were constructed for out-of-the-box deep learning of pharmaceutical properties, which outperformed the graph-based and other established models on most of the 26 pharmaceutically relevant benchmark datasets and a novel dataset. The MolMapNet learned important features that are consistent with the literature-reported molecular features. While deep learning models have allowed the extraction of fingerprints from the structural description of molecules, they can miss information that is present in the molecular descriptors that chemists use. Shen and colleagues present a method to combine both sources of information into two-dimensional fingerprint maps, which can be used in a wide variety of pharmaceutical tasks to predict the properties of drugs.},
  archive      = {J_NATMI},
  author       = {Shen, Wan Xiang and Zeng, Xian and Zhu, Feng and Wang, Ya li and Qin, Chu and Tan, Ying and Jiang, Yu Yang and Chen, Yu Zong},
  doi          = {10.1038/s42256-021-00301-6},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {334-343},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Out-of-the-box deep learning prediction of pharmaceutical properties by broadly learned knowledge-based molecular representations},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Expanding functional protein sequence spaces using
generative adversarial networks. <em>NATMI</em>, <em>3</em>(4), 324–333.
(<a href="https://doi.org/10.1038/s42256-021-00310-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo protein design for catalysis of any desired chemical reaction is a long-standing goal in protein engineering because of the broad spectrum of technological, scientific and medical applications. However, mapping protein sequence to protein function is currently neither computationally nor experimentally tangible. Here, we develop ProteinGAN, a self-attention-based variant of the generative adversarial network that is able to ‘learn’ natural protein sequence diversity and enables the generation of functional protein sequences. ProteinGAN learns the evolutionary relationships of protein sequences directly from the complex multidimensional amino-acid sequence space and creates new, highly diverse sequence variants with natural-like physical properties. Using malate dehydrogenase (MDH) as a template enzyme, we show that 24\% (13 out of 55 tested) of the ProteinGAN-generated and experimentally tested sequences are soluble and display MDH catalytic activity in the tested conditions in vitro, including a highly mutated variant of 106 amino-acid substitutions. ProteinGAN therefore demonstrates the potential of artificial intelligence to rapidly generate highly diverse functional proteins within the allowed biological constraints of the sequence space. A protein’s three-dimensional structure and properties are defined by its amino-acid sequence, but mapping protein sequence to protein function is a computationally highly intensive task. A new generative adversarial network approach learns from natural protein sequences and generates new, diverse protein sequence variations, which are experimentally tested.},
  archive      = {J_NATMI},
  author       = {Repecka, Donatas and Jauniskis, Vykintas and Karpus, Laurynas and Rembeza, Elzbieta and Rokaitis, Irmantas and Zrimec, Jan and Poviloniene, Simona and Laurynenas, Audrius and Viknander, Sandra and Abuajwa, Wissam and Savolainen, Otto and Meskys, Rolandas and Engqvist, Martin K. M. and Zelezniak, Aleksej},
  doi          = {10.1038/s42256-021-00310-5},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {324-333},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Expanding functional protein sequence spaces using generative adversarial networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Teaching recurrent neural networks to infer global temporal
structure from local examples. <em>NATMI</em>, <em>3</em>(4), 316–323.
(<a href="https://doi.org/10.1038/s42256-021-00321-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to store and manipulate information is a hallmark of computational systems. Whereas computers are carefully engineered to represent and perform mathematical operations on structured data, neurobiological systems adapt to perform analogous functions without needing to be explicitly engineered. Recent efforts have made progress in modelling the representation and recall of information in neural systems. However, precisely how neural systems learn to modify these representations remains far from understood. Here, we demonstrate that a recurrent neural network (RNN) can learn to modify its representation of complex information using only examples, and we explain the associated learning mechanism with new theory. Specifically, we drive an RNN with examples of translated, linearly transformed or pre-bifurcated time series from a chaotic Lorenz system, alongside an additional control signal that changes value for each example. By training the network to replicate the Lorenz inputs, it learns to autonomously evolve about a Lorenz-shaped manifold. Additionally, it learns to continuously interpolate and extrapolate the translation, transformation and bifurcation of this representation far beyond the training data by changing the control signal. Furthermore, we demonstrate that RNNs can infer the bifurcation structure of normal forms and period doubling routes to chaos, and extrapolate non-dynamical, kinematic trajectories. Finally, we provide a mechanism for how these computations are learned, and replicate our main results using a Wilson–Cowan reservoir. Together, our results provide a simple but powerful mechanism by which an RNN can learn to manipulate internal representations of complex information, enabling the principled study and precise design of RNNs. Recurrent neural networks (RNNs) can learn to process temporal information, such as speech or movement. New work makes such approaches more powerful and flexible by describing theory and experiments demonstrating that RNNs can learn from a few examples to generalize and predict complex dynamics including chaotic behaviour.},
  archive      = {J_NATMI},
  author       = {Kim, Jason Z. and Lu, Zhixin and Nozari, Erfan and Pappas, George J. and Bassett, Danielle S.},
  doi          = {10.1038/s42256-021-00321-2},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {316-323},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Teaching recurrent neural networks to infer global temporal structure from local examples},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A versatile deep learning architecture for classification
and label-free prediction of hyperspectral images. <em>NATMI</em>,
<em>3</em>(4), 306–315. (<a
href="https://doi.org/10.1038/s42256-021-00309-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hyperspectral imaging is a technique that provides rich chemical or compositional information not regularly available to traditional imaging modalities such as intensity imaging or colour imaging based on the reflection, transmission or emission of light. Analysis of hyperspectral imaging often relies on machine learning methods to extract information. Here we present a new flexible architecture—the U-within-U-Net—that can perform classification, segmentation and prediction of orthogonal imaging modalities on a variety of hyperspectral imaging techniques. Specifically, we demonstrate feature segmentation and classification on the Indian Pines hyperspectral dataset and simultaneous location prediction of multiple drugs in mass spectrometry imaging of rat liver tissue. We further demonstrate label-free fluorescence image prediction from hyperspectral stimulated Raman scattering microscopy images. The applicability of the U-within-U-Net architecture on diverse datasets with widely varying input and output dimensions and data sources suggest that it has great potential in advancing the use of hyperspectral imaging across many different application areas ranging from remote sensing, to medical imaging, to microscopy. Imaging over a wide spectral profile allows for detailed chemical characterization, but processing this data is not trivial. A new neural network architecture can process hyperspectral information from a variety of imaging techniques effectively by using the popular U-net motive around another U-net processing spatial patterns.},
  archive      = {J_NATMI},
  author       = {Manifold, Bryce and Men, Shuaiqian and Hu, Ruoqian and Fu, Dan},
  doi          = {10.1038/s42256-021-00309-y},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {306-315},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A versatile deep learning architecture for classification and label-free prediction of hyperspectral images},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Generating three-dimensional structures from a
two-dimensional slice with generative adversarial network-based
dimensionality expansion. <em>NATMI</em>, <em>3</em>(4), 299–305. (<a
href="https://doi.org/10.1038/s42256-021-00322-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative adversarial networks (GANs) can be trained to generate three-dimensional (3D) image data, which are useful for design optimization. However, this conventionally requires 3D training data, which are challenging to obtain. Two-dimensional (2D) imaging techniques tend to be faster, higher resolution, better at phase identification and more widely available. Here we introduce a GAN architecture, SliceGAN, that is able to synthesize high-fidelity 3D datasets using a single representative 2D image. This is especially relevant for the task of material microstructure generation, as a cross-sectional micrograph can contain sufficient information to statistically reconstruct 3D samples. Our architecture implements the concept of uniform information density, which ensures both that generated volumes are equally high quality at all points in space and that arbitrarily large volumes can be generated. SliceGAN has been successfully trained on a diverse set of materials, demonstrating the widespread applicability of this tool. The quality of generated micrographs is shown through a statistical comparison of synthetic and real datasets of a battery electrode in terms of key microstructural metrics. Finally, we find that the generation time for a 108 voxel volume is on the order of a few seconds, yielding a path for future studies into high-throughput microstructural optimization. A generative approach called SliceGAN is demonstrated that can construct complex three-dimensional (3D) images from representative two-dimensional (2D) image examples. This is a promising approach in particular for studying microstructured materials where acquiring good-quality 3D data is challenging; 3D datasets can be created with SliceGAN, making use of high-quality 2D imaging techniques that are widely available.},
  archive      = {J_NATMI},
  author       = {Kench, Steve and Cooper, Samuel J.},
  doi          = {10.1038/s42256-021-00322-1},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {299-305},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Generating three-dimensional structures from a two-dimensional slice with generative adversarial network-based dimensionality expansion},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Code-free deep learning for multi-modality medical image
classification. <em>NATMI</em>, <em>3</em>(4), 288–298. (<a
href="https://doi.org/10.1038/s42256-021-00305-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A number of large technology companies have created code-free cloud-based platforms that allow researchers and clinicians without coding experience to create deep learning algorithms. In this study, we comprehensively analyse the performance and featureset of six platforms, using four representative cross-sectional and en-face medical imaging datasets to create image classification models. The mean (s.d.) F1 scores across platforms for all model–dataset pairs were as follows: Amazon, 93.9 (5.4); Apple, 72.0 (13.6); Clarifai, 74.2 (7.1); Google, 92.0 (5.4); MedicMind, 90.7 (9.6); Microsoft, 88.6 (5.3). The platforms demonstrated uniformly higher classification performance with the optical coherence tomography modality. Potential use cases given proper validation include research dataset curation, mobile ‘edge models’ for regions without internet access, and baseline models against which to compare and iterate bespoke deep learning approaches. Several technology companies offer platforms for users without coding experience to develop deep learning algorithms. This Analysis compares the performance of six ‘code-free deep learning’ platforms (from Amazon, Apple, Clarifai, Google, MedicMind and Microsoft) in creating medical image classification models.},
  archive      = {J_NATMI},
  author       = {Korot, Edward and Guan, Zeyu and Ferraz, Daniel and Wagner, Siegfried K. and Zhang, Gongyu and Liu, Xiaoxuan and Faes, Livia and Pontikos, Nikolas and Finlayson, Samuel G. and Khalid, Hagar and Moraes, Gabriella and Balaskas, Konstantinos and Denniston, Alastair K. and Keane, Pearse A.},
  doi          = {10.1038/s42256-021-00305-2},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {288-298},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Code-free deep learning for multi-modality medical image classification},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Direct-to-consumer medical machine learning and artificial
intelligence applications. <em>NATMI</em>, <em>3</em>(4), 283–287. (<a
href="https://doi.org/10.1038/s42256-021-00331-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Direct-to-consumer medical artificial intelligence/machine learning applications are increasingly used for a variety of diagnostic assessments, and the emphasis on telemedicine and home healthcare during the COVID-19 pandemic may further stimulate their adoption. In this Perspective, we argue that the artificial intelligence/machine learning regulatory landscape should operate differently when a system is designed for clinicians/doctors as opposed to when it is designed for personal use. Direct-to-consumer applications raise unique concerns due to the nature of consumer users, who tend to be limited in their statistical and medical literacy and risk averse about their health outcomes. This creates an environment where false alarms can proliferate and burden public healthcare systems and medical insurers. While similar situations exist elsewhere in medicine, the ease and frequency with which artificial intelligence/machine learning apps can be used, and their increasing prevalence in the consumer market, calls for careful reflection on how to effectively regulate them. We suggest regulators should strive to better understand how consumers interact with direct-to-consumer medical artificial intelligence/machine learning apps, particularly diagnostic ones, and this requires more than a focus on the system’s technical specifications. We further argue that the best regulatory review would also consider such technologies’ social costs under widespread use. Medical artificial intelligence and machine learning technologies marketed directly to consumers are on the rise. The authors argue that the regulatory landscape for such technologies should operate differently when a system is designed for personal use than when it is designed for clinicians and doctors.},
  archive      = {J_NATMI},
  author       = {Babic, Boris and Gerke, Sara and Evgeniou, Theodoros and Cohen, I. Glenn},
  doi          = {10.1038/s42256-021-00331-0},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {283-287},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Direct-to-consumer medical machine learning and artificial intelligence applications},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Predicting hidden structure in dynamical systems.
<em>NATMI</em>, <em>3</em>(4), 281–282. (<a
href="https://doi.org/10.1038/s42256-021-00329-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dynamical properties of a nonlinear system can be learned from its time-series data, but is it possible to predict what happens when the system is tuned far away from its training values?},
  archive      = {J_NATMI},
  author       = {Gauthier, Daniel J. and Fischer, Ingo},
  doi          = {10.1038/s42256-021-00329-8},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {281-282},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting hidden structure in dynamical systems},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Enhancing hyperspectral imaging. <em>NATMI</em>,
<em>3</em>(4), 279–280. (<a
href="https://doi.org/10.1038/s42256-021-00336-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hyperspectral imaging can reveal important information without the need for staining. To extract information from this extensive data, however, new methods are needed that can interpret the spatial and spectral patterns present in the images.},
  archive      = {J_NATMI},
  author       = {Bhargava, Rohit and Falahkheirkhah, Kianoush},
  doi          = {10.1038/s42256-021-00336-9},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {279-280},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Enhancing hyperspectral imaging},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Escape from flatland. <em>NATMI</em>, <em>3</em>(4),
277–278. (<a href="https://doi.org/10.1038/s42256-021-00334-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {3D image reconstruction is important for the understanding of materials and their function in devices. A generative adversarial network architecture reconstructs 3D materials microstructures from 2D images.},
  archive      = {J_NATMI},
  author       = {Franco, Alejandro A.},
  doi          = {10.1038/s42256-021-00334-x},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {277-278},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Escape from flatland},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cross-validation is safe to use. <em>NATMI</em>,
<em>3</em>(4), 276. (<a
href="https://doi.org/10.1038/s42256-021-00332-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {King, Ross D. and Orhobor, Oghenejokpeme I. and Taylor, Charles C.},
  doi          = {10.1038/s42256-021-00332-z},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {276},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Cross-validation is safe to use},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). People have the AI power. <em>NATMI</em>, <em>3</em>(4),
275. (<a href="https://doi.org/10.1038/s42256-021-00340-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Citizen scientists are empowered by mobile technology to collect data and crowdsource knowledge. Furthermore, automated machine learning tools allow non-experts in AI to analyse data. Ethical and regulatory questions arise, however, as data collection and AI technologies become enmeshed in people’s lives.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00340-z},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {275},
  shortjournal = {Nat. Mach. Intell.},
  title        = {People have the AI power},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning STEM-EDX tomography of nanocrystals.
<em>NATMI</em>, <em>3</em>(3), 267–274. (<a
href="https://doi.org/10.1038/s42256-020-00289-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Energy-dispersive X-ray spectroscopy (EDX) is often performed simultaneously with high-angle annular dark-field scanning transmission electron microscopy (STEM) for nanoscale physico-chemical analysis. However, high-quality STEM-EDX tomographic imaging is still challenging due to fundamental limitations such as sample degradation with prolonged scan time and the low probability of X-ray generation. To address this, we propose an unsupervised deep learning method for high-quality 3D EDX tomography of core–shell nanocrystals, which can be usually permanently dammaged by prolonged electron beam. The proposed deep learning STEM-EDX tomography method was used to accurately reconstruct Au nanoparticles and InP/ZnSe/ZnS core–shell quantum dots, used in commercial display devices. Furthermore, the shape and thickness uniformity of the reconstructed ZnSe/ZnS shell closely correlates with optical properties of the quantum dots, such as quantum efficiency and chemical stability. Advanced electron microscopy and spectroscopy techniques can reveal useful structural and chemical details at the nanoscale. An unsupervised deep learning approach helps to reconstruct 3D images and observe the relationship between optical and structural properties of semiconductor nanocrystals, of interest in optoelectronic applications.},
  archive      = {J_NATMI},
  author       = {Han, Yoseob and Jang, Jaeduck and Cha, Eunju and Lee, Junho and Chung, Hyungjin and Jeong, Myoungho and Kim, Tae-Gon and Chae, Byeong Gyu and Kim, Hee Goo and Jun, Shinae and Hwang, Sungwoo and Lee, Eunha and Ye, Jong Chul},
  doi          = {10.1038/s42256-020-00289-5},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {267-274},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning STEM-EDX tomography of nanocrystals},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improving representations of genomic sequence motifs in
convolutional networks with exponential activations. <em>NATMI</em>,
<em>3</em>(3), 258–266. (<a
href="https://doi.org/10.1038/s42256-020-00291-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep convolutional neural networks (CNNs) trained on regulatory genomic sequences tend to build representations in a distributed manner, making it a challenge to extract learned features that are biologically meaningful, such as sequence motifs. Here we perform a comprehensive analysis of synthetic sequences to investigate the role that CNN activations have on model interpretability. We show that employing an exponential activation in the first layer filters consistently leads to interpretable and robust representations of motifs compared with other commonly used activations. Strikingly, we demonstrate that CNNs with better test performance do not necessarily imply more interpretable representations with attribution methods. We find that CNNs with exponential activations significantly improve the efficacy of recovering biologically meaningful representations with attribution methods. We demonstrate that these results generalize to real DNA sequences across several in vivo datasets. Together, this work demonstrates how a small modification to existing CNNs (that is, setting exponential activations in the first layer) can substantially improve the robustness and interpretabilty of learned representations directly in convolutional filters and indirectly with attribution methods. Model interpretability is important in genomics. Koo and Ploenzke show that exponential activations in the first layer of convolutional neural networks lead to interpretable and robust representations of genomic sequence motifs.},
  archive      = {J_NATMI},
  author       = {Koo, Peter K. and Ploenzke, Matt},
  doi          = {10.1038/s42256-020-00291-x},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {258-266},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Improving representations of genomic sequence motifs in convolutional networks with exponential activations},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A deep learning framework for high-throughput
mechanism-driven phenotype compound screening and its application to
COVID-19 drug repurposing. <em>NATMI</em>, <em>3</em>(3), 247–257. (<a
href="https://doi.org/10.1038/s42256-020-00285-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotype-based compound screening has advantages over target-based drug discovery, but is unscalable and lacks understanding of mechanism of drug action. A chemical-induced gene expression profile provides a mechanistic signature of phenotypic response; however, the use of such data is limited by their sparseness, unreliability and relatively low throughput. Few methods can perform phenotype-based de novo chemical compound screening. Here we propose a mechanism-driven neural network-based method, DeepCE—which utilizes a graph neural network and multihead attention mechanism to model chemical substructure–gene and gene–gene associations—for predicting the differential gene expression profile perturbed by de novo chemicals. Moreover, we propose a novel data augmentation method that extracts useful information from unreliable experiments in the L1000 dataset. The experimental results show that DeepCE achieves superior performances to state-of-the-art methods. The effectiveness of gene expression profiles generated from DeepCE is further supported by comparing them with observed data for downstream classification tasks. To demonstrate the value of DeepCE, we apply it to drug repurposing of COVID-19 and generate novel lead compounds consistent with clinical evidence. DeepCE thus provides a potentially powerful framework for robust predictive modelling by utilizing noisy omics data and screening novel chemicals for the modulation of a systemic response to disease. In drug discovery and repurposing, systematic analysis of genome-wide gene expression of chemical perturbations on human cell lines is a useful approach, but is limited due to a relatively low experimental throughput. Computational, deep learning methods can help. In this work a graph neural network called Deep Chemical Expression is developed that can predict chemical-induced gene expression profiles. It is applied to identify drug repurposing candidates for COVID-19 treatments.},
  archive      = {J_NATMI},
  author       = {Pham, Thai-Hoang and Qiu, Yue and Zeng, Jucheng and Xie, Lei and Zhang, Ping},
  doi          = {10.1038/s42256-020-00285-9},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {247-257},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A deep learning framework for high-throughput mechanism-driven phenotype compound screening and its application to COVID-19 drug repurposing},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). High-resolution radar road segmentation using weakly
supervised learning. <em>NATMI</em>, <em>3</em>(3), 239–246. (<a
href="https://doi.org/10.1038/s42256-020-00288-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autonomous driving has recently gained lots of attention due to its disruptive potential and impact on the global economy; however, these high expectations are hindered by strict safety requirements for redundant sensing modalities that are each able to independently perform complex tasks to ensure reliable operation. At the core of an autonomous driving algorithmic stack is road segmentation, which is the basis for numerous planning and decision-making algorithms. Radar-based methods fail in many driving scenarios, mainly as various common road delimiters barely reflect radar signals, coupled with a lack of analytical models for road delimiters and the inherit limitations in radar angular resolution. Our approach is based on radar data in the form of a two-dimensional complex range-Doppler array as input into a deep neural network (DNN) that is trained to semantically segment the drivable area using weak supervision from a camera. Furthermore, guided back propagation was utilized to analyse radar data and design a novel perception filter. Our approach creates the ability to perform road segmentation in common driving scenarios based solely on radar data and we propose to utilize this method as an enabler for redundant sensing modalities for autonomous driving. Self-driving vehicles must reliably detect the drivable area in front of them in any weather condition. An actively developed sensor approach is camera-based road segmentation, but it is limited by the visible spectrum. Radar-based approaches are a promising alternative and a new method extracts the drivable area from raw radar data by training a deep neural network using paired camera data, which can be labelled automatically using pretrained computer vision models.},
  archive      = {J_NATMI},
  author       = {Orr, Itai and Cohen, Moshik and Zalevsky, Zeev},
  doi          = {10.1038/s42256-020-00288-6},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {239-246},
  shortjournal = {Nat. Mach. Intell.},
  title        = {High-resolution radar road segmentation using weakly supervised learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Optimized spiking neurons can classify images with high
accuracy through temporal coding with two spikes. <em>NATMI</em>,
<em>3</em>(3), 230–238. (<a
href="https://doi.org/10.1038/s42256-021-00311-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spike-based neuromorphic hardware promises to reduce the energy consumption of image classification and other deep-learning applications, particularly on mobile phones and other edge devices. However, direct training of deep spiking neural networks is difficult, and previous methods for converting trained artificial neural networks to spiking neurons were inefficient because the neurons had to emit too many spikes. We show that a substantially more efficient conversion arises when one optimizes the spiking neuron model for that purpose, so that it not only matters for information transmission how many spikes a neuron emits, but also when it emits those spikes. This advances the accuracy that can be achieved for image classification with spiking neurons, and the resulting networks need on average just two spikes per neuron for classifying an image. In addition, our new conversion method improves latency and throughput of the resulting spiking networks. Spiking neural networks could offer a low-energy consuming solution to deep learning applications on the edge and in mobile devices. Using temporal coding, where the timing of spikes carries extra information, a new method efficiently converts conventional artificial neural networks to spiking networks.},
  archive      = {J_NATMI},
  author       = {Stöckl, Christoph and Maass, Wolfgang},
  doi          = {10.1038/s42256-021-00311-4},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {230-238},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Optimized spiking neurons can classify images with high accuracy through temporal coding with two spikes},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Learning nonlinear operators via DeepONet based on the
universal approximation theorem of operators. <em>NATMI</em>,
<em>3</em>(3), 218–229. (<a
href="https://doi.org/10.1038/s42256-021-00302-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is widely known that neural networks (NNs) are universal approximators of continuous functions. However, a less known but powerful result is that a NN with a single hidden layer can accurately approximate any nonlinear continuous operator. This universal approximation theorem of operators is suggestive of the structure and potential of deep neural networks (DNNs) in learning continuous operators or complex systems from streams of scattered data. Here, we thus extend this theorem to DNNs. We design a new network with small generalization error, the deep operator network (DeepONet), which consists of a DNN for encoding the discrete input function space (branch net) and another DNN for encoding the domain of the output functions (trunk net). We demonstrate that DeepONet can learn various explicit operators, such as integrals and fractional Laplacians, as well as implicit operators that represent deterministic and stochastic differential equations. We study different formulations of the input function space and its effect on the generalization error for 16 different diverse applications. Neural networks are known as universal approximators of continuous functions, but they can also approximate any mathematical operator (mapping a function to another function), which is an important capability for complex systems such as robotics control. A new deep neural network called DeepONet can lean various mathematical operators with small generalization error.},
  archive      = {J_NATMI},
  author       = {Lu, Lu and Jin, Pengzhan and Pang, Guofei and Zhang, Zhongqiang and Karniadakis, George Em},
  doi          = {10.1038/s42256-021-00302-5},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {218-229},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning nonlinear operators via DeepONet based on the universal approximation theorem of operators},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Common pitfalls and recommendations for using machine
learning to detect and prognosticate for COVID-19 using chest
radiographs and CT scans. <em>NATMI</em>, <em>3</em>(3), 199–217. (<a
href="https://doi.org/10.1038/s42256-021-00307-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning methods offer great promise for fast and accurate detection and prognostication of coronavirus disease 2019 (COVID-19) from standard-of-care chest radiographs (CXR) and chest computed tomography (CT) images. Many articles have been published in 2020 describing new machine learning-based models for both of these tasks, but it is unclear which are of potential clinical utility. In this systematic review, we consider all published papers and preprints, for the period from 1 January 2020 to 3 October 2020, which describe new machine learning models for the diagnosis or prognosis of COVID-19 from CXR or CT images. All manuscripts uploaded to bioRxiv, medRxiv and arXiv along with all entries in EMBASE and MEDLINE in this timeframe are considered. Our search identified 2,212 studies, of which 415 were included after initial screening and, after quality screening, 62 studies were included in this systematic review. Our review finds that none of the models identified are of potential clinical use due to methodological flaws and/or underlying biases. This is a major weakness, given the urgency with which validated COVID-19 models are needed. To address this, we give many recommendations which, if followed, will solve these issues and lead to higher-quality model development and well-documented manuscripts. Many machine learning-based approaches have been developed for the prognosis and diagnosis of COVID-19 from medical images and this Analysis identifies over 2,200 relevant published papers and preprints in this area. After initial screening, 62 studies are analysed and the authors find they all have methodological flaws standing in the way of clinical utility. The authors have several recommendations to address these issues.},
  archive      = {J_NATMI},
  author       = {Roberts, Michael and Driggs, Derek and Thorpe, Matthew and Gilbey, Julian and Yeung, Michael and Ursprung, Stephan and Aviles-Rivero, Angelica I. and Etmann, Christian and McCague, Cathal and Beer, Lucian and Weir-McCall, Jonathan R. and Teng, Zhongzhao and Gkrania-Klotsas, Effrossyni and Rudd, James H. F. and Sala, Evis and Schönlieb, Carola-Bibiane},
  doi          = {10.1038/s42256-021-00307-0},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {199-217},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Common pitfalls and recommendations for using machine learning to detect and prognosticate for COVID-19 using chest radiographs and CT scans},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). LncADeep performance on full-length transcripts.
<em>NATMI</em>, <em>3</em>(3), 197–198. (<a
href="https://doi.org/10.1038/s42256-019-0108-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Yang, Cheng and Zhou, Man and Xie, Haoling and Zhu, Huaiqiu},
  doi          = {10.1038/s42256-019-0108-2},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {197-198},
  shortjournal = {Nat. Mach. Intell.},
  title        = {LncADeep performance on full-length transcripts},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: LncADeep performance on full-length transcripts.
<em>NATMI</em>, <em>3</em>(3), 196. (<a
href="https://doi.org/10.1038/s42256-019-0107-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Amin, Noorul and McGrath, Annette and Chen, Yi-Ping Phoebe},
  doi          = {10.1038/s42256-019-0107-3},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {196},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reply to: LncADeep performance on full-length transcripts},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The best of both worlds. <em>NATMI</em>, <em>3</em>(3),
194–195. (<a href="https://doi.org/10.1038/s42256-021-00315-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuromorphic computing could unlock low-power machine learning that can run on edge devices. A new algorithm that implements an artificial neuron emitting a sparse number of spikes could help realize this goal.},
  archive      = {J_NATMI},
  author       = {Hamilton, Tara},
  doi          = {10.1038/s42256-021-00315-0},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {194-195},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The best of both worlds},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Generalizing universal function approximators.
<em>NATMI</em>, <em>3</em>(3), 192–193. (<a
href="https://doi.org/10.1038/s42256-021-00318-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {At the heart of many challenges in scientific research lie complex equations for which no analytical solutions exist. A new neural network model called DeepONet can learn to approximate nonlinear functions as well as operators.},
  archive      = {J_NATMI},
  author       = {Higgins, Irina},
  doi          = {10.1038/s42256-021-00318-x},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {192-193},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Generalizing universal function approximators},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Speeding up machine hearing. <em>NATMI</em>, <em>3</em>(3),
190–191. (<a href="https://doi.org/10.1038/s42256-021-00317-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational models that capture the nonlinear processing of the inner ear have been prohibitively slow to use for most machine-hearing systems. A convolutional neural network model replicates hallmark features of cochlear signal processing, potentially enabling real-time applications.},
  archive      = {J_NATMI},
  author       = {Carney, Laurel H.},
  doi          = {10.1038/s42256-021-00317-y},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {190-191},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Speeding up machine hearing},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). On the importance of ethnographic methods in AI research.
<em>NATMI</em>, <em>3</em>(3), 187–189. (<a
href="https://doi.org/10.1038/s42256-021-00323-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To truly understand the societal impact of AI, we need to look beyond the exclusive focus on quantitative methods, and focus on qualitative methods like ethnography, which shed light on the actors and institutions that wield power through the use of these technologies.},
  archive      = {J_NATMI},
  author       = {Marda, Vidushi and Narayan, Shivangi},
  doi          = {10.1038/s42256-021-00323-0},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {187-189},
  shortjournal = {Nat. Mach. Intell.},
  title        = {On the importance of ethnographic methods in AI research},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Physical human–robot interaction for clinical care in
infectious environments. <em>NATMI</em>, <em>3</em>(3), 184–186. (<a
href="https://doi.org/10.1038/s42256-021-00324-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COVID-19 pandemic has highlighted key challenges for patient care and health provider safety. Adaptable robotic systems, with enhanced sensing, manipulation and autonomy capabilities could help address these challenges in future infectious disease outbreaks.},
  archive      = {J_NATMI},
  author       = {Su, Hao and Di Lallo, Antonio and Murphy, Robin R. and Taylor, Russell H. and Garibaldi, Brian T. and Krieger, Axel},
  doi          = {10.1038/s42256-021-00324-z},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {184-186},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Physical human–robot interaction for clinical care in infectious environments},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AI, COVID-19 and the long haul. <em>NATMI</em>,
<em>3</em>(3), 183. (<a
href="https://doi.org/10.1038/s42256-021-00328-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It has been a little over a year since a worldwide COVID-19 pandemic was declared. Science has moved fast to fight the virus but preparations need to be underway for fighting future outbreaks.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00328-9},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {183},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI, COVID-19 and the long haul},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: High-resolution radar road segmentation
using weakly supervised learning. <em>NATMI</em>, <em>3</em>(2), 182.
(<a href="https://doi.org/10.1038/s42256-021-00314-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s42256-021-00314-1.},
  archive      = {J_NATMI},
  author       = {Orr, Itai and Cohen, Moshik and Zalevsky, Zeev},
  doi          = {10.1038/s42256-021-00314-1},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {182},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: High-resolution radar road segmentation using weakly supervised learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Learning MRI artefact removal with
unpaired data. <em>NATMI</em>, <em>3</em>(2), 181. (<a
href="https://doi.org/10.1038/s42256-021-00300-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s42256-021-00300-7.},
  archive      = {J_NATMI},
  author       = {Liu, Siyuan and Thung, Kim-Han and Qu, Liangqiong and Lin, Weili and Shen, Dinggang and Yap, Pew-Thian},
  doi          = {10.1038/s42256-021-00300-7},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {181},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Learning MRI artefact removal with unpaired data},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep neural networks identify sequence context features
predictive of transcription factor binding. <em>NATMI</em>,
<em>3</em>(2), 172–180. (<a
href="https://doi.org/10.1038/s42256-020-00282-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factors bind DNA by recognizing specific sequence motifs, which are typically 6–12 bp long. A motif can occur many thousands of times in the human genome, but only a subset of those sites are actually bound. Here we present a machine-learning framework leveraging existing convolutional neural network architectures and model interpretation techniques to identify and interpret sequence context features most important for predicting whether a particular motif instance will be bound. We apply our framework to predict binding at motifs for 38 transcription factors in a lymphoblastoid cell line, score the importance of context sequences at base-pair resolution and characterize context features most predictive of binding. We find that the choice of training data heavily influences classification accuracy and the relative importance of features such as open chromatin. Overall, our framework enables novel insights into features predictive of transcription factor binding and is likely to inform future deep learning applications to interpret non-coding genetic variants. The transcription process of DNA is highly complex and while short DNA sequence motifs recognized by transcription factors are well known, less is known about the context in the DNA sequence that determines whether a transcription factor will actually bind its motif. Zheng and colleagues present a method that uses convolutional neural networks to identify sequence features that help predict whether transcribing proteins can bind to their target sequences in DNA.},
  archive      = {J_NATMI},
  author       = {Zheng, An and Lamkin, Michael and Zhao, Hanqing and Wu, Cynthia and Su, Hao and Gymrek, Melissa},
  doi          = {10.1038/s42256-020-00282-y},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {172-180},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep neural networks identify sequence context features predictive of transcription factor binding},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Global voxel transformer networks for augmented microscopy.
<em>NATMI</em>, <em>3</em>(2), 161–171. (<a
href="https://doi.org/10.1038/s42256-020-00283-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in deep learning have led to remarkable success in augmented microscopy, enabling us to obtain high-quality microscope images without using expensive microscopy hardware and sample preparation techniques. Current deep learning models for augmented microscopy are mostly U-Net-based neural networks, thus sharing certain drawbacks that limit the performance. In particular, U-Nets are composed of local operators only and lack dynamic non-local information aggregation. In this work, we introduce global voxel transformer networks (GVTNets), a deep learning tool for augmented microscopy that overcomes intrinsic limitations of the current U-Net-based models and achieves improved performance. GVTNets are built on global voxel transformer operators, which are able to aggregate global information, as opposed to local operators like convolutions. We apply the proposed methods on existing datasets for three different augmented microscopy tasks under various settings. Computational augmentation of microscopic images aims at reducing the need to chemically label or stain cells to extract information. The popular U-Net model often employed for these tasks uses mostly local information. A new method for augmenting microscopic images is presented that allows for global information to be used at each step of the process.},
  archive      = {J_NATMI},
  author       = {Wang, Zhengyang and Xie, Yaochen and Ji, Shuiwang},
  doi          = {10.1038/s42256-020-00283-x},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {161-171},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Global voxel transformer networks for augmented microscopy},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Extraction of protein dynamics information from cryo-EM maps
using deep learning. <em>NATMI</em>, <em>3</em>(2), 153–160. (<a
href="https://doi.org/10.1038/s42256-020-00290-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elucidation of both the three-dimensional structure and the dynamics of a protein is essential to understand its function. Technical breakthroughs in single-particle analysis based on cryo-electron microscopy (cryo-EM) have enabled the three-dimensional structures of numerous proteins to be solved at atomic or near-atomic resolution. However, the analysis of the dynamics of protein targets using cryo-EM is often challenging because of their large sizes and complex structural assemblies. Here, we describe DEFMap, a deep learning-based approach to directly extract the dynamics associated with the atomic fluctuations that are hidden in cryo-EM density maps. Using only cryo-EM density data, DEFMap provides dynamics that correlate well with data obtained from molecular dynamics simulations and experimental approaches. Furthermore, DEFMap successfully detects changes in dynamics that are associated with molecular recognition. This strategy combines deep learning, experimental data and molecular dynamics simulations, and may reveal a new multidisciplinary approach for protein science. Cryo-electron microscopy (cryo-EM) can be used to determine the three-dimensional structure of proteins at atomic-scale resolution. It is challenging to observe the dynamics of proteins using cryo-EM because of their large sizes and complex structural assemblies. A new deep-learning approach called DEFMap extracts the dynamics associated with the atomic fluctuations that are hidden in cryo-EM density maps.},
  archive      = {J_NATMI},
  author       = {Matsumoto, Shigeyuki and Ishida, Shoichi and Araki, Mitsugu and Kato, Takayuki and Terayama, Kei and Okuno, Yasushi},
  doi          = {10.1038/s42256-020-00290-y},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {153-160},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Extraction of protein dynamics information from cryo-EM maps using deep learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mapping the space of chemical reactions using
attention-based neural networks. <em>NATMI</em>, <em>3</em>(2), 144–152.
(<a href="https://doi.org/10.1038/s42256-020-00284-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organic reactions are usually assigned to classes containing reactions with similar reagents and mechanisms. Reaction classes facilitate the communication of complex concepts and efficient navigation through chemical reaction space. However, the classification process is a tedious task. It requires identification of the corresponding reaction class template via annotation of the number of molecules in the reactions, the reaction centre and the distinction between reactants and reagents. Here, we show that transformer-based models can infer reaction classes from non-annotated, simple text-based representations of chemical reactions. Our best model reaches a classification accuracy of 98.2\%. We also show that the learned representations can be used as reaction fingerprints that capture fine-grained differences between reaction classes better than traditional reaction fingerprints. The insights into chemical reaction space enabled by our learned fingerprints are illustrated by an interactive reaction atlas providing visual clustering and similarity searching. Organic chemical reactions can be divided into classes that allow chemists to use the knowledge they have about optimal conditions for specific reactions in the context of other reactions of similar type. Schwaller et al. present here an efficient method based on transformer neural networks that learns a chemical space in which reactions of a similar class are grouped together.},
  archive      = {J_NATMI},
  author       = {Schwaller, Philippe and Probst, Daniel and Vaucher, Alain C. and Nair, Vishnu H. and Kreutter, David and Laino, Teodoro and Reymond, Jean-Louis},
  doi          = {10.1038/s42256-020-00284-w},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {144-152},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Mapping the space of chemical reactions using attention-based neural networks},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A convolutional neural-network model of human cochlear
mechanics and filter tuning for real-time applications. <em>NATMI</em>,
<em>3</em>(2), 134–143. (<a
href="https://doi.org/10.1038/s42256-020-00286-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Auditory models are commonly used as feature extractors for automatic speech-recognition systems or as front-ends for robotics, machine-hearing and hearing-aid applications. Although auditory models can capture the biophysical and nonlinear properties of human hearing in great detail, these biophysical models are computationally expensive and cannot be used in real-time applications. We present a hybrid approach where convolutional neural networks are combined with computational neuroscience to yield a real-time end-to-end model for human cochlear mechanics, including level-dependent filter tuning (CoNNear). The CoNNear model was trained on acoustic speech material and its performance and applicability were evaluated using (unseen) sound stimuli commonly employed in cochlear mechanics research. The CoNNear model accurately simulates human cochlear frequency selectivity and its dependence on sound intensity, an essential quality for robust speech intelligibility at negative speech-to-background-noise ratios. The CoNNear architecture is based on parallel and differentiable computations and has the power to achieve real-time human performance. These unique CoNNear features will enable the next generation of human-like machine-hearing applications. Extensive work has gone into developing peripheral auditory models that capture the nonlinear processing of the ear. But the resulting models are prohibitively slow to use at scale for most machine hearing systems. The authors present a convolutional neural network model that replicates hallmark features of cochlear signal processing, potentially enabling real-time applications.},
  archive      = {J_NATMI},
  author       = {Baby, Deepak and Van Den Broucke, Arthur and Verhulst, Sarah},
  doi          = {10.1038/s42256-020-00286-8},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {134-143},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A convolutional neural-network model of human cochlear mechanics and filter tuning for real-time applications},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An open source machine learning framework for efficient and
transparent systematic reviews. <em>NATMI</em>, <em>3</em>(2), 125–133.
(<a href="https://doi.org/10.1038/s42256-020-00287-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To help researchers conduct a systematic review or meta-analysis as efficiently and transparently as possible, we designed a tool to accelerate the step of screening titles and abstracts. For many tasks—including but not limited to systematic reviews and meta-analyses—the scientific literature needs to be checked systematically. Scholars and practitioners currently screen thousands of studies by hand to determine which studies to include in their review or meta-analysis. This is error prone and inefficient because of extremely imbalanced data: only a fraction of the screened studies is relevant. The future of systematic reviewing will be an interaction with machine learning algorithms to deal with the enormous increase of available text. We therefore developed an open source machine learning-aided pipeline applying active learning: ASReview. We demonstrate by means of simulation studies that active learning can yield far more efficient reviewing than manual reviewing while providing high quality. Furthermore, we describe the options of the free and open source research software and present the results from user experience tests. We invite the community to contribute to open source projects such as our own that provide measurable and reproducible improvements over current practice. It is a challenging task for any research field to screen the literature and determine what needs to be included in a systematic review in a transparent way. A new open source machine learning framework called ASReview, which employs active learning and offers a range of machine learning models, can check the literature efficiently and systemically.},
  archive      = {J_NATMI},
  author       = {van de Schoot, Rens and de Bruin, Jonathan and Schram, Raoul and Zahedi, Parisa and de Boer, Jan and Weijdema, Felix and Kramer, Bianca and Huijts, Martijn and Hoogerwerf, Maarten and Ferdinands, Gerbrich and Harkema, Albert and Willemsen, Joukje and Ma, Yongchao and Fang, Qixiang and Hindriks, Sybren and Tummers, Lars and Oberski, Daniel L.},
  doi          = {10.1038/s42256-020-00287-7},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {125-133},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An open source machine learning framework for efficient and transparent systematic reviews},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Bioinspired acousto-magnetic microswarm robots with upstream
motility. <em>NATMI</em>, <em>3</em>(2), 116–124. (<a
href="https://doi.org/10.1038/s42256-020-00275-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to propel against flows, that is, to perform positive rheotaxis, can provide exciting opportunities for applications in targeted therapeutics and non-invasive surgery. So far no biocompatible technologies exist for navigating microparticles upstream when they are in a background fluid flow. Inspired by many naturally occurring microswimmers—such as bacteria, spermatozoa and plankton—that utilize the no-slip boundary conditions of the wall to exhibit upstream propulsion, here we report on the design and characterization of self-assembled microswarms that can execute upstream motility in a combination of external acoustic and magnetic fields. Both acoustic and magnetic fields are safe to humans, non-invasive, can penetrate deeply into the human body and are well-developed in clinical settings. The combination of both fields can overcome the limitations encountered by single actuation methods. The design criteria of the acoustically induced reaction force of the microswarms, which is needed to perform rolling-type motion, are discussed. We show quantitative agreement between experimental data and our model that captures the rolling behaviour. The upstream capability provides a design strategy for delivering small drug molecules to hard-to-reach sites and represents a fundamental step towards the realization of micro- and nanosystem navigation against the blood flow. Microrobotics offers great potential for precise drug delivery as medication can be released in the bloodstream only where it is needed. But the dynamic environment of the bloodstream is a challenge for navigation. An approach presented by Ahmed and colleagues combines magnetic and acoustic fields to allow swarms of particles to swim against a current.},
  archive      = {J_NATMI},
  author       = {Ahmed, Daniel and Sukhov, Alexander and Hauri, David and Rodrigue, Dubon and Maranta, Gian and Harting, Jens and Nelson, Bradley J.},
  doi          = {10.1038/s42256-020-00275-x},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {116-124},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Bioinspired acousto-magnetic microswarm robots with upstream motility},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A definition, benchmark and database of AI for social good
initiatives. <em>NATMI</em>, <em>3</em>(2), 111–115. (<a
href="https://doi.org/10.1038/s42256-021-00296-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Initiatives relying on artificial intelligence (AI) to deliver socially beneficial outcomes—AI for social good (AI4SG)—are on the rise. However, existing attempts to understand and foster AI4SG initiatives have so far been limited by the lack of normative analyses and a shortage of empirical evidence. In this Perspective, we address these limitations by providing a definition of AI4SG and by advocating the use of the United Nations’ Sustainable Development Goals (SDGs) as a benchmark for tracing the scope and spread of AI4SG. We introduce a database of AI4SG projects gathered using this benchmark, and discuss several key insights, including the extent to which different SDGs are being addressed. This analysis makes possible the identification of pressing problems that, if left unaddressed, risk hampering the effectiveness of AI4SG initiatives. Many researchers have become interested in implementing artificial intelligence methods in applications with socially beneficial outcomes. To provide a way to study and benchmark such ‘AI for social good’ applications, Josh Cowls et al. use the United Nations’ Sustainable Development Goals to systematically analyse AI for social good applications.},
  archive      = {J_NATMI},
  author       = {Cowls, Josh and Tsamados, Andreas and Taddeo, Mariarosaria and Floridi, Luciano},
  doi          = {10.1038/s42256-021-00296-0},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {111-115},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A definition, benchmark and database of AI for social good initiatives},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Institutionalizing ethics in AI through broader impact
requirements. <em>NATMI</em>, <em>3</em>(2), 104–110. (<a
href="https://doi.org/10.1038/s42256-021-00298-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Turning principles into practice is one of the most pressing challenges of artificial intelligence (AI) governance. In this Perspective, we reflect on a governance initiative by one of the world’s largest AI conferences. In 2020, the Conference on Neural Information Processing Systems (NeurIPS) introduced a requirement for submitting authors to include a statement on the broader societal impacts of their research. Drawing insights from similar governance initiatives, including institutional review boards (IRBs) and impact requirements for funding applications, we investigate the risks, challenges and potential benefits of such an initiative. Among the challenges, we list a lack of recognized best practice and procedural transparency, researcher opportunity costs, institutional and social pressures, cognitive biases and the inherently difficult nature of the task. The potential benefits, on the other hand, include improved anticipation and identification of impacts, better communication with policy and governance experts, and a general strengthening of the norms around responsible research. To maximize the chance of success, we recommend measures to increase transparency, improve guidance, create incentives to engage earnestly with the process, and facilitate public deliberation on the requirement’s merits and future. Perhaps the most important contribution from this analysis are the insights we can gain regarding effective community-based governance and the role and responsibility of the AI research community more broadly. The Conference on Neural Information Processing Systems (NeurIPS) introduced a new requirement in 2020 that submitting authors must include a statement on the broader impacts of their research. Prunkl and colleagues discuss challenges and benefits of this requirement and propose suggestions to address the challenges.},
  archive      = {J_NATMI},
  author       = {Prunkl, Carina E. A. and Ashurst, Carolyn and Anderljung, Markus and Webb, Helena and Leike, Jan and Dafoe, Allan},
  doi          = {10.1038/s42256-021-00298-y},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {104-110},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Institutionalizing ethics in AI through broader impact requirements},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Transformers for future medicinal chemists. <em>NATMI</em>,
<em>3</em>(2), 102–103. (<a
href="https://doi.org/10.1038/s42256-021-00299-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical reactions can be grouped into classes, but determining what class a specific reaction belongs to is not trivial on a large-scale. A new study demonstrates data-driven automatic classification of chemical reactions with methods borrowed from natural language processing.},
  archive      = {J_NATMI},
  author       = {Boström, Jonas},
  doi          = {10.1038/s42256-021-00299-x},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {102-103},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Transformers for future medicinal chemists},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Listen to this. <em>NATMI</em>, <em>3</em>(2), 101. (<a
href="https://doi.org/10.1038/s42256-021-00313-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hearing and vision are powerful and important senses for interacting with our surroundings. So far, advances in the area of machine vision have been the most prominent, but machine hearing research that closely mimics the complex sound processing in the human ear has exciting opportunities to offer.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00313-2},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {101},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Listen to this},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Automating turbulence modelling by
multi-agent reinforcement learning. <em>NATMI</em>, <em>3</em>(1), 99.
(<a href="https://doi.org/10.1038/s42256-021-00295-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s42256-021-00295-1 .},
  archive      = {J_NATMI},
  author       = {Novati, Guido and de Laroussilhe, Hugues Lascombes and Koumoutsakos, Petros},
  doi          = {10.1038/s42256-021-00295-1},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {99},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Automating turbulence modelling by multi-agent reinforcement learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Publisher correction: Automating turbulence modelling by
multi-agent reinforcement learning. <em>NATMI</em>, <em>3</em>(1), 98.
(<a href="https://doi.org/10.1038/s42256-021-00293-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s42256-021-00293-3.},
  archive      = {J_NATMI},
  author       = {Novati, Guido and de Laroussilhe, Hugues Lascombes and Koumoutsakos, Petros},
  doi          = {10.1038/s42256-021-00293-3},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {98},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Automating turbulence modelling by multi-agent reinforcement learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Augmenting vascular disease diagnosis by
vasculature-aware unsupervised learning. <em>NATMI</em>, <em>3</em>(1),
97. (<a href="https://doi.org/10.1038/s42256-021-00292-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s42256-020-0188-z},
  archive      = {J_NATMI},
  author       = {Wang, Yong and Ji, Mengqi and Jiang, Shengwei and Wang, Xukang and Wu, Jiamin and Duan, Feng and Fan, Jingtao and Huang, Laiqiang and Ma, Shaohua and Fang, Lu and Dai, Qionghai},
  doi          = {10.1038/s42256-021-00292-4},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {97},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Augmenting vascular disease diagnosis by vasculature-aware unsupervised learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Automating turbulence modelling by multi-agent
reinforcement learning. <em>NATMI</em>, <em>3</em>(1), 87–96. (<a
href="https://doi.org/10.1038/s42256-020-00272-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Turbulent flow models are critical for applications such as aircraft design, weather forecasting and climate prediction. Existing models are largely based on physical insight and engineering intuition. More recently, machine learning has been contributing to this endeavour with promising results. However, all efforts have focused on supervised learning, which is difficult to generalize beyond training data. Here we introduce multi-agent reinforcement learning as an automated discovery tool of turbulence models. We demonstrate the potential of this approach on large-eddy simulations of isotropic turbulence, using the recovery of statistical properties of direct numerical simulations as a reward. The closure model is a control policy enacted by cooperating agents, which detect critical spatio-temporal patterns in the flow field to estimate the unresolved subgrid-scale physics. Results obtained with multi-agent reinforcement learning algorithms based on experience replay compare favourably with established modelling approaches. Moreover, we show that the learned turbulence models generalize across grid sizes and flow conditions. Turbulence modelling is an essential flow simulation tool, but is typically dependent on physical insight and engineering intuition. Novati et al. develop a multi-agent reinforcement learning approach for learning turbulence models that can generalize across grid sizes and flow conditions.},
  archive      = {J_NATMI},
  author       = {Novati, Guido and de Laroussilhe, Hugues Lascombes and Koumoutsakos, Petros},
  doi          = {10.1038/s42256-020-00272-0},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {87-96},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Automating turbulence modelling by multi-agent reinforcement learning},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Inverse design of nanoporous crystalline reticular materials
with deep generative models. <em>NATMI</em>, <em>3</em>(1), 76–86. (<a
href="https://doi.org/10.1038/s42256-020-00271-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reticular frameworks are crystalline porous materials that form via the self-assembly of molecular building blocks in different topologies, with many having desirable properties for gas storage, separation, catalysis, biomedical applications and so on. The notable variety of building blocks makes reticular chemistry both promising and challenging for prospective materials design. Here we propose an automated nanoporous materials discovery platform powered by a supramolecular variational autoencoder for the generative design of reticular materials. We demonstrate the automated design process with a class of metal–organic framework (MOF) structures and the goal of separating carbon dioxide from natural gas or flue gas. Our model shows high fidelity in capturing MOF structural features. We show that the autoencoder has a promising optimization capability when jointly trained with multiple top adsorbent candidates identified for superior gas separation. MOFs discovered here are strongly competitive against some of the best-performing MOFs/zeolites ever reported. Reticular frameworks are crystalline porous materials with desirable properties such as gas separation, but their large design space presents a challenge. An automated nanoporous materials discovery platform powered by a supramolecular variational autoencoder can efficiently explore this space.},
  archive      = {J_NATMI},
  author       = {Yao, Zhenpeng and Sánchez-Lengeling, Benjamín and Bobbitt, N. Scott and Bucior, Benjamin J. and Kumar, Sai Govind Hari and Collins, Sean P. and Burns, Thomas and Woo, Tom K. and Farha, Omar K. and Snurr, Randall Q. and Aspuru-Guzik, Alán},
  doi          = {10.1038/s42256-020-00271-1},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {76-86},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Inverse design of nanoporous crystalline reticular materials with deep generative models},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A deep learning framework for drug repurposing via emulating
clinical trials on real-world patient data. <em>NATMI</em>,
<em>3</em>(1), 68–75. (<a
href="https://doi.org/10.1038/s42256-020-00276-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug repurposing is an effective strategy to identify new uses for existing drugs, providing the quickest possible transition from bench to bedside. Real-world data, such as electronic health records and insurance claims, provide information on large cohorts of users for many drugs. Here we present an efficient and easily customized framework for generating and testing multiple candidates for drug repurposing using a retrospective analysis of real-world data. Building upon well-established causal inference and deep learning methods, our framework emulates randomized clinical trials for drugs present in a large-scale medical claims database. We demonstrate our framework on a coronary artery disease cohort of millions of patients. We successfully identify drugs and drug combinations that substantially improve the coronary artery disease outcomes but haven’t been indicated for treating coronary artery disease, paving the way for drug repurposing. Many approved drugs can be used to treat diseases other than the one they were developed for, which has the added benefit that the safety of the drug has already been tested. To identify possible candidates for re-purposing trials, Liu et al. have developed a method to use existing electronic patient data to simulate clinical trials and identify drugs that influence the progression of diseases with which they were not previously associated.},
  archive      = {J_NATMI},
  author       = {Liu, Ruoqi and Wei, Lai and Zhang, Ping},
  doi          = {10.1038/s42256-020-00276-w},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {68-75},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A deep learning framework for drug repurposing via emulating clinical trials on real-world patient data},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Learning MRI artefact removal with unpaired data.
<em>NATMI</em>, <em>3</em>(1), 60–67. (<a
href="https://doi.org/10.1038/s42256-020-00270-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Retrospective artefact correction (RAC) improves image quality post acquisition and enhances image usability. Recent machine-learning-driven techniques for RAC are predominantly based on supervised learning, so practical utility can be limited as data with paired artefact-free and artefact-corrupted images are typically insufficient or even non-existent. Here we show that unwanted image artefacts can be disentangled and removed from an image via an RAC neural network learned with unpaired data. This implies that our method does not require matching artefact-corrupted data to be either collected via acquisition or generated via simulation. Experimental results demonstrate that our method is remarkably effective in removing artefacts and retaining anatomical details in images with different contrasts. To remove artefacts from medical imaging, machine learning can be a useful tool, but supervised approaches need examples of the same image with and without artefacts. Liu et al. present a method to train an artefact removal network without needing matching images of corrupted and uncorrupted images.},
  archive      = {J_NATMI},
  author       = {Liu, Siyuan and Thung, Kim-Han and Qu, Liangqiong and Lin, Weili and Shen, Dinggang and Yap, Pew-Thian},
  doi          = {10.1038/s42256-020-00270-2},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {60-67},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning MRI artefact removal with unpaired data},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A soft robot that adapts to environments through shape
change. <em>NATMI</em>, <em>3</em>(1), 51–59. (<a
href="https://doi.org/10.1038/s42256-020-00263-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many organisms, including various species of spiders and caterpillars, change their shape to switch gaits and adapt to different environments. Recent technological advances, ranging from stretchable circuits to highly deformable soft robots, have begun to make shape-changing robots a possibility. However, it is currently unclear how and when shape change should occur, and what capabilities could be gained, leading to a wide range of unsolved design and control problems. To begin addressing these questions, here we simulate, design and build a soft robot that utilizes shape change to achieve locomotion over both a flat and inclined surface. Modelling this robot in simulation, we explore its capabilities in two environments and demonstrate the automated discovery of environment-specific shapes and gaits that successfully transfer to the physical hardware. We found that the shape-changing robot traverses these environments better than an equivalent but non-morphing robot, in simulation and reality. Inspired by many examples in nature where organisms change shape to concur environments, there is much interest in designing robots that are capable of shape change. Shah et al. demonstrate a method for automatically discovering shape and gait changes for soft robots that can adapt to different terrains.},
  archive      = {J_NATMI},
  author       = {Shah, Dylan S. and Powers, Joshua P. and Tilton, Liana G. and Kriegman, Sam and Bongard, Josh and Kramer-Bottiglio, Rebecca},
  doi          = {10.1038/s42256-020-00263-1},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {51-59},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A soft robot that adapts to environments through shape change},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Estimation of continuous valence and arousal levels from
faces in naturalistic conditions. <em>NATMI</em>, <em>3</em>(1), 42–50.
(<a href="https://doi.org/10.1038/s42256-020-00280-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Facial affect analysis aims to create new types of human–computer interactions by enabling computers to better understand a person’s emotional state in order to provide ad hoc help and interactions. Since discrete emotional classes (such as anger, happiness, sadness and so on) are not representative of the full spectrum of emotions displayed by humans on a daily basis, psychologists typically rely on dimensional measures, namely valence (how positive the emotional display is) and arousal (how calming or exciting the emotional display looks like). However, while estimating these values from a face is natural for humans, it is extremely difficult for computer-based systems and automatic estimation of valence and arousal in naturalistic conditions is an open problem. Additionally, the subjectivity of these measures makes it hard to obtain good quality data. Here we introduce a novel deep neural network architecture to analyse facial affect in naturalistic conditions with a high level of accuracy. The proposed network integrates face alignment and jointly estimates both categorical and continuous emotions in a single pass, making it suitable for real-time applications. We test our method on three challenging datasets collected in naturalistic conditions and show that our approach outperforms all previous methods. We also discuss caveats regarding the use of this tool, and ethical aspects that must be considered in its application. The annotation of the visual signs of emotions can be important for psychological studies and even human–computer interactions. Instead of only ascribing discrete emotions, Toisoul and colleagues use a single neural network that predicts emotional labels on a spectrum of valence and arousal without separate face-alignment steps.},
  archive      = {J_NATMI},
  author       = {Toisoul, Antoine and Kossaifi, Jean and Bulat, Adrian and Tzimiropoulos, Georgios and Pantic, Maja},
  doi          = {10.1038/s42256-020-00280-0},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {42-50},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Estimation of continuous valence and arousal levels from faces in naturalistic conditions},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Enhancing optical-flow-based control by learning visual
appearance cues for flying robots. <em>NATMI</em>, <em>3</em>(1), 33–41.
(<a href="https://doi.org/10.1038/s42256-020-00279-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Flying insects employ elegant optical-flow-based strategies to solve complex tasks such as landing or obstacle avoidance. Roboticists have mimicked these strategies on flying robots with only limited success, because optical flow (1) cannot disentangle distance from velocity and (2) is less informative in the highly important flight direction. Here, we propose a solution to these fundamental shortcomings by having robots learn to estimate distances to objects by their visual appearance. The learning process obtains supervised targets from a stability-based distance estimation approach. We have successfully implemented the process on a small flying robot. For the task of landing, it results in faster, smooth landings. For the task of obstacle avoidance, it results in higher success rates at higher flight speeds. Our results yield improved robotic visual navigation capabilities and lead to a novel hypothesis on insect intelligence: behaviours that were described as optical-flow-based and hardwired actually benefit from learning processes. Autonomous flight is challenging for small flying robots, given the limited space for sensors and on-board processing capabilities, but a promising approach is to mimic optical-flow-based strategies of flying insects. A new development improves this technique, enabling smoother landings and better obstacle avoidance, by giving robots the ability to learn to estimate distances to objects by their visual appearance.},
  archive      = {J_NATMI},
  author       = {de Croon, G. C. H. E. and De Wagter, C. and Seidl, T.},
  doi          = {10.1038/s42256-020-00279-7},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {33-41},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Enhancing optical-flow-based control by learning visual appearance cues for flying robots},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Li yan et al. reply. <em>NATMI</em>, <em>3</em>(1), 28–32.
(<a href="https://doi.org/10.1038/s42256-020-00251-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Goncalves, Jorge and Yan, Li and Zhang, Hai-Tao and Xiao, Yang and Wang, Maolin and Guo, Yuqi and Sun, Chuan and Tang, Xiuchuan and Cao, Zhiguo and Li, Shusheng and Xu, Hui and Cheng, Cheng and Jin, Junyang and Yuan, Ye},
  doi          = {10.1038/s42256-020-00251-5},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {28-32},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Li yan et al. reply},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). External validation demonstrates limited clinical utility of
the interpretable mortality prediction model for patients with COVID-19.
<em>NATMI</em>, <em>3</em>(1), 25–27. (<a
href="https://doi.org/10.1038/s42256-020-00254-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Barish, Matthew and Bolourani, Siavash and Lau, Lawrence F. and Shah, Sareen and Zanos, Theodoros P.},
  doi          = {10.1038/s42256-020-00254-2},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {25-27},
  shortjournal = {Nat. Mach. Intell.},
  title        = {External validation demonstrates limited clinical utility of the interpretable mortality prediction model for patients with COVID-19},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Replication of a mortality prediction model in dutch
patients with COVID-19. <em>NATMI</em>, <em>3</em>(1), 23–24. (<a
href="https://doi.org/10.1038/s42256-020-00253-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Quanjel, Marian J. R. and van Holten, Thijs C. and Gunst-van der Vliet, Pieternel C. and Wielaard, Jette and Karakaya, Bekir and Söhne, Maaike and Moeniralam, Hazra S. and Grutters, Jan C.},
  doi          = {10.1038/s42256-020-00253-3},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {23-24},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Replication of a mortality prediction model in dutch patients with COVID-19},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Limited applicability of a COVID-19 specific mortality
prediction rule to the intensive care setting. <em>NATMI</em>,
<em>3</em>(1), 20–22. (<a
href="https://doi.org/10.1038/s42256-020-00252-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Dupuis, C. and De Montmollin, E. and Neuville, M. and Mourvillier, B. and Ruckly, S. and Timsit, J. F.},
  doi          = {10.1038/s42256-020-00252-4},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {20-22},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Limited applicability of a COVID-19 specific mortality prediction rule to the intensive care setting},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: Consider the laboratory aspects in developing
patient prediction models. <em>NATMI</em>, <em>3</em>(1), 19. (<a
href="https://doi.org/10.1038/s42256-020-0220-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Yan, Li and Goncalves, Jorge and Zhang, Hai-Tao and Li, Shusheng and Yuan, Ye},
  doi          = {10.1038/s42256-020-0220-3},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {19},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reply to: Consider the laboratory aspects in developing patient prediction models},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Consider laboratory aspects in developing patient prediction
models. <em>NATMI</em>, <em>3</em>(1), 18. (<a
href="https://doi.org/10.1038/s42256-020-0221-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Reeve, Janice L. V. and Twomey, Patrick J.},
  doi          = {10.1038/s42256-020-0221-2},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {18},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Consider laboratory aspects in developing patient prediction models},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: Clinical interpretation of an interpretable
prognostic model for patients with COVID-19. <em>NATMI</em>,
<em>3</em>(1), 17. (<a
href="https://doi.org/10.1038/s42256-020-0206-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Yuan, Ye and Goncalves, Jorge and Xiao, Yan and Zhang, Hai-Tao and Xu, Hui and Cao, Zhiguo},
  doi          = {10.1038/s42256-020-0206-1},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {17},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reply to: Clinical interpretation of an interpretable prognostic model for patients with COVID-19},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Clinical interpretation of an interpretable prognostic model
for patients with COVID-19. <em>NATMI</em>, <em>3</em>(1), 16. (<a
href="https://doi.org/10.1038/s42256-020-0207-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Giacobbe, Daniele Roberto},
  doi          = {10.1038/s42256-020-0207-0},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {16},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Clinical interpretation of an interpretable prognostic model for patients with COVID-19},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A biological perspective on evolutionary computation.
<em>NATMI</em>, <em>3</em>(1), 9–15. (<a
href="https://doi.org/10.1038/s42256-020-00278-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionary computation is inspired by the mechanisms of biological evolution. With algorithmic improvements and increasing computing resources, evolutionary computation has discovered creative and innovative solutions to challenging practical problems. This paper evaluates how today’s evolutionary computation compares to biological evolution and how it may fall short. A small number of well-accepted characteristics of biological evolution are considered: openendedness, major transitions in organizational structure, neutrality and genetic drift, multi-objectivity, complex genotype-to-phenotype mappings and co-evolution. Evolutionary computation exhibits many of these to some extent but more can be achieved by scaling up with available computing and by emulating biology more carefully. In particular, evolutionary computation diverges from biological evolution in three key respects: it is based on small populations and strong selection; it typically uses direct genotype-to-phenotype mappings; and it does not achieve major organizational transitions. These shortcomings suggest a roadmap for future evolutionary computation research, and point to gaps in our understanding of how biology discovers major transitions. Advances in these areas can lead to evolutionary computation that approaches the complexity and flexibility of biology, and can serve as an executable model of biological processes. Evolutionary computation is inspired by biological evolution and exhibits characteristics familiar from biology such as openendedness, multi-objectivity and co-evolution. This Perspective highlights where major differences still exist, and where the field of evolutionary computation could attempt to approach features from biological evolution more closely, namely neutrality and random drift, complex genotype-to-phenotype mappings with rich environmental interactions and major organizational transitions.},
  archive      = {J_NATMI},
  author       = {Miikkulainen, Risto and Forrest, Stephanie},
  doi          = {10.1038/s42256-020-00278-8},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {9-15},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A biological perspective on evolutionary computation},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AI reflections in 2020. <em>NATMI</em>, <em>3</em>(1), 2–8.
(<a href="https://doi.org/10.1038/s42256-020-00281-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We invited authors of selected Comments and Perspectives published in Nature Machine Intelligence in the latter half of 2019 and first half of 2020 to describe how their topic has developed, what their thoughts are about the challenges of 2020, and what they look forward to in 2021.},
  archive      = {J_NATMI},
  author       = {Jobin, Anna and Man, Kingson and Damasio, Antonio and Kaissis, Georgios and Braren, Rickmer and Stoyanovich, Julia and Van Bavel, Jay J. and West, Tessa V. and Mittelstadt, Brent and Eshraghian, Jason and Costa-jussà, Marta R. and Tzachor, Asaf and Jamjoom, Aimun A. B. and Taddeo, Mariarosaria and Sinibaldi, Edoardo and Hu, Yipeng and Luengo-Oroz, Miguel},
  doi          = {10.1038/s42256-020-00281-z},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {2-8},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI reflections in 2020},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Room for improvement. <em>NATMI</em>, <em>3</em>(1), 1. (<a
href="https://doi.org/10.1038/s42256-021-00294-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reflecting on 2020 brings into focus clear challenges for the year ahead, including for AI research that contemplates its broader societal impact.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-021-00294-2},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Room for improvement},
  volume       = {3},
  year         = {2021},
}
</textarea>
</details></li>
</ul>

</body>
</html>
