<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt---434">NBT - 434</h2>
<ul>
<li><details>
<summary>
(2024a). Biopharma deals get smaller and earlier. <em>NBT</em>,
<em>43</em>(1), 11–18. (<a
href="https://doi.org/10.1038/s41587-024-02506-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Merger and acquisition activity is at a seven-year low in 2024 as buyers digest prior deals and US election jitters delayed further spending. Expect a pick-up in 2025.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-024-02506-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {11-18},
  shortjournal = {Nature Biotech.},
  title        = {Biopharma deals get smaller and earlier},
  volume       = {43},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). People. <em>NBT</em>, <em>42</em>(12), 1926. (<a
href="https://doi.org/10.1038/s41587-024-02501-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02501-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1926},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Intellectual property training should be embedded in the
biomedical education process. <em>NBT</em>, <em>42</em>(12), 1924–1925.
(<a href="https://doi.org/10.1038/s41587-024-02499-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Implementation of intellectual property education in academic institutions can result in increased opportunities for protecting intellectual property and limit costs, but to be successful technology transfer offices must also adapt.},
  archive      = {J_NBT},
  author       = {Serafini, Randal A. and Curtis, Micaila D. E. and Alimperti, Stella},
  doi          = {10.1038/s41587-024-02499-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1924-1925},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property training should be embedded in the biomedical education process},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Precision targeting of
autoantigen-specific b cells in muscle-specific tyrosine kinase
myasthenia gravis with chimeric autoantibody receptor t cells.
<em>NBT</em>, <em>42</em>(12), 1923. (<a
href="https://doi.org/10.1038/s41587-024-02502-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Oh, Sangwook and Mao, Xuming and Manfredo-Vieira, Silvio and Lee, Jinmin and Patel, Darshil and Choi, Eun Jung and Alvarado, Andrea and Cottman-Thomas, Ebony and Maseda, Damian and Tsao, Patricia Y. and Ellebrecht, Christoph T. and Khella, Sami L. and Richman, David P. and O’Connor, Kevin C. and Herzberg, Uri and Binder, Gwendolyn K. and Milone, Michael C. and Basu, Samik and Payne, Aimee S.},
  doi          = {10.1038/s41587-024-02502-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1923},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Precision targeting of autoantigen-specific b cells in muscle-specific tyrosine kinase myasthenia gravis with chimeric autoantibody receptor t cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Prime editing using CRISPR-cas12a and
circular RNAs in human cells. <em>NBT</em>, <em>42</em>(12), 1921–1922.
(<a href="https://doi.org/10.1038/s41587-024-02160-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Liang, Ronghong and He, Zixin and Zhao, Kevin Tianmeng and Zhu, Haocheng and Hu, Jiacheng and Liu, Guanwen and Gao, Qiang and Liu, Meiyan and Zhang, Rui and Qiu, Jin-Long and Gao, Caixia},
  doi          = {10.1038/s41587-024-02160-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1921-1922},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Prime editing using CRISPR-cas12a and circular RNAs in human cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). KARR-seq reveals cellular higher-order RNA structures and
RNA–RNA interactions. <em>NBT</em>, <em>42</em>(12), 1909–1920. (<a
href="https://doi.org/10.1038/s41587-023-02109-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA fate and function are affected by their structures and interactomes. However, how RNA and RNA-binding proteins (RBPs) assemble into higher-order structures and how RNA molecules may interact with each other to facilitate functions remain largely unknown. Here we present KARR-seq, which uses N3-kethoxal labeling and multifunctional chemical crosslinkers to covalently trap and determine RNA–RNA interactions and higher-order RNA structures inside cells, independent of local protein binding to RNA. KARR-seq depicts higher-order RNA structure and detects widespread intermolecular RNA–RNA interactions with high sensitivity and accuracy. Using KARR-seq, we show that translation represses mRNA compaction under native and stress conditions. We determined the higher-order RNA structures of respiratory syncytial virus (RSV) and vesicular stomatitis virus (VSV) and identified RNA–RNA interactions between the viruses and the host RNAs that potentially regulate viral replication. RNA interactions and structures are determined with a chemical labeling and crosslinking method.},
  archive      = {J_NBT},
  author       = {Wu, Tong and Cheng, Anthony Youzhi and Zhang, Yuexiu and Xu, Jiayu and Wu, Jinjun and Wen, Li and Li, Xiao and Liu, Bei and Dou, Xiaoyang and Wang, Pingluan and Zhang, Linda and Fei, Jingyi and Li, Jianrong and Ouyang, Zhengqing and He, Chuan},
  doi          = {10.1038/s41587-023-02109-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1909-1920},
  shortjournal = {Nature Biotech.},
  title        = {KARR-seq reveals cellular higher-order RNA structures and RNA–RNA interactions},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Characterizing the impacts of dataset imbalance on
single-cell data integration. <em>NBT</em>, <em>42</em>(12), 1899–1908.
(<a href="https://doi.org/10.1038/s41587-023-02097-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational methods for integrating single-cell transcriptomic data from multiple samples and conditions do not generally account for imbalances in the cell types measured in different datasets. In this study, we examined how differences in the cell types present, the number of cells per cell type and the cell type proportions across samples affect downstream analyses after integration. The Iniquitate pipeline assesses the robustness of integration results after perturbing the degree of imbalance between datasets. Benchmarking of five state-of-the-art single-cell RNA sequencing integration techniques in 2,600 integration experiments indicates that sample imbalance has substantial impacts on downstream analyses and the biological interpretation of integration results. Imbalance perturbation led to statistically significant variation in unsupervised clustering, cell type classification, differential expression and marker gene annotation, query-to-reference mapping and trajectory inference. We quantified the impacts of imbalance through newly introduced properties—aggregate cell type support and minimum cell type center distance. To better characterize and mitigate impacts of imbalance, we introduce balanced clustering metrics and imbalanced integration guidelines for integration method users. Iniquitate quantifies the effects of cell type imbalance on single-cell RNA sequencing data integration.},
  archive      = {J_NBT},
  author       = {Maan, Hassaan and Zhang, Lin and Yu, Chengxin and Geuenich, Michael J. and Campbell, Kieran R. and Wang, Bo},
  doi          = {10.1038/s41587-023-02097-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1899-1908},
  shortjournal = {Nature Biotech.},
  title        = {Characterizing the impacts of dataset imbalance on single-cell data integration},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computationally designed sensors detect endogenous ras
activity and signaling effectors at subcellular resolution.
<em>NBT</em>, <em>42</em>(12), 1888–1898. (<a
href="https://doi.org/10.1038/s41587-023-02107-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The utility of genetically encoded biosensors for sensing the activity of signaling proteins has been hampered by a lack of strategies for matching sensor sensitivity to the physiological concentration range of the target. Here we used computational protein design to generate intracellular sensors of Ras activity (LOCKR-based Sensor for Ras activity (Ras-LOCKR-S)) and proximity labelers of the Ras signaling environment (LOCKR-based, Ras activity-dependent Proximity Labeler (Ras-LOCKR-PL)). These tools allow the detection of endogenous Ras activity and labeling of the surrounding environment at subcellular resolution. Using these sensors in human cancer cell lines, we identified Ras-interacting proteins in oncogenic EML4-Alk granules and found that Src-Associated in Mitosis 68-kDa (SAM68) protein specifically enhances Ras activity in the granules. The ability to subcellularly localize endogenous Ras activity should deepen our understanding of Ras function in health and disease and may suggest potential therapeutic strategies. Computationally designed sensors map the activity and signaling environment of Ras.},
  archive      = {J_NBT},
  author       = {Zhang, Jason Z. and Nguyen, William H. and Greenwood, Nathan and Rose, John C. and Ong, Shao-En and Maly, Dustin J. and Baker, David},
  doi          = {10.1038/s41587-023-02107-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1888-1898},
  shortjournal = {Nature Biotech.},
  title        = {Computationally designed sensors detect endogenous ras activity and signaling effectors at subcellular resolution},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An oncolytic virus–t cell chimera for cancer immunotherapy.
<em>NBT</em>, <em>42</em>(12), 1876–1887. (<a
href="https://doi.org/10.1038/s41587-023-02118-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The efficacy of oncolytic adenoviruses (OAs) for cancer therapy has been limited by insufficient delivery to tumors after systemic injection and the propensity of OAs to induce the expression of immune checkpoints. To address these limitations, we use T cells to deliver OAs into tumors and engineer the OA to express a Cas9 system targeting the PDL1 gene encoding the immune checkpoint protein PD-L1. By cloaking OAs with cell membranes presenting T cell-specific antigens, we physically conjugated OAs onto T cell surfaces by antigen–receptor interaction. We tested the oncolytic virus–T cell chimera (ONCOTECH) via intravenous delivery in mouse cancer models, including models of melanoma, pancreatic adenocarcinoma, lung cancer and glioblastoma. In the melanoma model, the in vivo delivery of ONCOTECH resulted in a strong accumulation of OAs in tumor cells, where PD-L1 expression was reduced by 50% and the single administration of ONCOTECH enabled 80% survival over 70 days. Collectively, ONCOTECH represents a promising translational technology to combine virotherapy and cell therapy. Oncolytic adenovirus is delivered on T cells and engineered to express a checkpoint inhibitor.},
  archive      = {J_NBT},
  author       = {Chen, Yuxuan and Chen, Xiaohong and Bao, Weier and Liu, Gang and Wei, Wei and Ping, Yuan},
  doi          = {10.1038/s41587-023-02118-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1876-1887},
  shortjournal = {Nature Biotech.},
  title        = {An oncolytic virus–T cell chimera for cancer immunotherapy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Prime editing using CRISPR-cas12a and circular RNAs in
human cells. <em>NBT</em>, <em>42</em>(12), 1867–1875. (<a
href="https://doi.org/10.1038/s41587-023-02095-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing with prime editors based on CRISPR-Cas9 is limited by the large size of the system and the requirement for a G/C-rich protospacer-adjacent motif (PAM) sequence. Here, we use the smaller Cas12a protein to develop four circular RNA-mediated prime editor (CPE) systems: nickase-dependent CPE (niCPE), nuclease-dependent CPE (nuCPE), split nickase-dependent CPE (sniCPE) and split nuclease-dependent CPE (snuCPE). CPE systems preferentially recognize T-rich genomic regions and possess a potential multiplexing capacity in comparison to corresponding Cas9-based systems. The efficiencies of the nuclease-based systems are up to 10.42%, whereas niCPE and sniCPE reach editing frequencies of up to 24.89% and 40.75% without positive selection in human cells, respectively. A derivative system, called one-sniCPE, combines all three RNA editing components under a single promoter. By arraying CRISPR RNAs for different targets in one circular RNA, we also demonstrate low-efficiency editing of up to four genes simultaneously with the nickase prime editors niCPE and sniCPE. Prime editing with CRISPR-Cas12a broadens targeting scope and enables multiplexing.},
  archive      = {J_NBT},
  author       = {Liang, Ronghong and He, Zixin and Zhao, Kevin Tianmeng and Zhu, Haocheng and Hu, Jiacheng and Liu, Guanwen and Gao, Qiang and Liu, Meiyan and Zhang, Rui and Qiu, Jin-Long and Gao, Caixia},
  doi          = {10.1038/s41587-023-02095-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1867-1875},
  shortjournal = {Nature Biotech.},
  title        = {Prime editing using CRISPR-cas12a and circular RNAs in human cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ultra-fast label-free quantification and comprehensive
proteome coverage with narrow-window data-independent acquisition.
<em>NBT</em>, <em>42</em>(12), 1855–1866. (<a
href="https://doi.org/10.1038/s41587-023-02099-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectrometry (MS)-based proteomics aims to characterize comprehensive proteomes in a fast and reproducible manner. Here we present the narrow-window data-independent acquisition (nDIA) strategy consisting of high-resolution MS1 scans with parallel tandem MS (MS/MS) scans of ~200 Hz using 2-Th isolation windows, dissolving the differences between data-dependent and -independent methods. This is achieved by pairing a quadrupole Orbitrap mass spectrometer with the asymmetric track lossless (Astral) analyzer which provides &amp;gt;200-Hz MS/MS scanning speed, high resolving power and sensitivity, and low-ppm mass accuracy. The nDIA strategy enables profiling of &amp;gt;100 full yeast proteomes per day, or 48 human proteomes per day at the depth of ~10,000 human protein groups in half-an-hour or ~7,000 proteins in 5 min, representing 3× higher coverage compared with current state-of-the-art MS. Multi-shot acquisition of offline fractionated samples provides comprehensive coverage of human proteomes in ~3 h. High quantitative precision and accuracy are demonstrated in a three-species proteome mixture, quantifying 14,000+ protein groups in a single half-an-hour run. A new mass spectrometry strategy combines the advantages of data-dependent and data-independent acquisition.},
  archive      = {J_NBT},
  author       = {Guzman, Ulises H. and Martinez-Val, Ana and Ye, Zilu and Damoc, Eugen and Arrey, Tabiwang N. and Pashkova, Anna and Renuse, Santosh and Denisov, Eduard and Petzoldt, Johannes and Peterson, Amelia C. and Harking, Florian and Østergaard, Ole and Rydbirk, Rasmus and Aznar, Susana and Stewart, Hamish and Xuan, Yue and Hermanson, Daniel and Horning, Stevan and Hock, Christian and Makarov, Alexander and Zabrouskov, Vlad and Olsen, Jesper V.},
  doi          = {10.1038/s41587-023-02099-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1855-1866},
  shortjournal = {Nature Biotech.},
  title        = {Ultra-fast label-free quantification and comprehensive proteome coverage with narrow-window data-independent acquisition},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Activation of recombinases at specific DNA loci by
zinc-finger domain insertions. <em>NBT</em>, <em>42</em>(12), 1844–1854.
(<a href="https://doi.org/10.1038/s41587-023-02121-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recombinases have several potential advantages as genome editing tools compared to nucleases and other editing enzymes, but the process of engineering them to efficiently recombine predetermined DNA targets demands considerable investment of time and labor. Here we sought to harness zinc-finger DNA-binding domains (ZFDs) to program recombinase binding by developing fusions, in which ZFDs are inserted into recombinase coding sequences. By screening libraries of hybrid proteins, we optimized the insertion site, linker length, spacing and ZFD orientation and generated Cre-type recombinases that remain dormant unless the insertionally fused ZFD binds its target site placed in the vicinity of the recombinase binding site. The developed fusion improved targeted editing efficiencies of recombinases by four-fold and abolished measurable off-target activity in mammalian cells. The ZFD-dependent activity is transferable to a recombinase with relaxed specificity, providing the means for developing fully programmable recombinases. Our engineered recombinases provide improved genome editing tools with increased precision and efficiency. Recombinases fused with zinc-finger domains are active only when bound to their DNA targets.},
  archive      = {J_NBT},
  author       = {Mukhametzyanova, Liliya and Schmitt, Lukas Theo and Torres-Rivera, Julia and Rojo-Romanos, Teresa and Lansing, Felix and Paszkowski-Rogacz, Maciej and Hollak, Heike and Brux, Melanie and Augsburg, Martina and Schneider, Paul Martin and Buchholz, Frank},
  doi          = {10.1038/s41587-023-02121-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1844-1854},
  shortjournal = {Nature Biotech.},
  title        = {Activation of recombinases at specific DNA loci by zinc-finger domain insertions},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Kirigami electronics for long-term electrophysiological
recording of human neural organoids and assembloids. <em>NBT</em>,
<em>42</em>(12), 1836–1843. (<a
href="https://doi.org/10.1038/s41587-023-02081-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Realizing the full potential of organoids and assembloids to model neural development and disease will require improved methods for long-term, minimally invasive recording of electrical activity. Current technologies, such as patch clamp, penetrating microelectrodes, planar electrode arrays and substrate-attached flexible electrodes, do not allow chronic recording of organoids in suspension, which is necessary to preserve architecture. Inspired by kirigami art, we developed flexible electronics that transition from a two-dimensional to a three-dimensional basket-like configuration with either spiral or honeycomb patterns to accommodate the long-term culture of organoids in suspension. Here we show that this platform, named kirigami electronics (KiriE), integrates with and enables chronic recording of cortical organoids for up to 120 days while preserving their morphology, cytoarchitecture and cell composition. We demonstrate integration of KiriE with optogenetic and pharmacological manipulation and modeling phenotypes related to a genetic disease. Moreover, KiriE can capture corticostriatal connectivity in assembloids following optogenetic stimulation. Thus, KiriE will enable investigation of disease and activity patterns underlying nervous system assembly. Electrical activity in neural organoids is captured with a device inspired by a paper-cutting art.},
  archive      = {J_NBT},
  author       = {Yang, Xiao and Forró, Csaba and Li, Thomas L. and Miura, Yuki and Zaluska, Tomasz J. and Tsai, Ching-Ting and Kanton, Sabina and McQueen, James P. and Chen, Xiaoyu and Mollo, Valentina and Santoro, Francesca and Pașca, Sergiu P. and Cui, Bianxiao},
  doi          = {10.1038/s41587-023-02081-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1836-1843},
  shortjournal = {Nature Biotech.},
  title        = {Kirigami electronics for long-term electrophysiological recording of human neural organoids and assembloids},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Simultaneous nanopore profiling of mRNA m6A and
pseudouridine reveals translation coordination. <em>NBT</em>,
<em>42</em>(12), 1831–1835. (<a
href="https://doi.org/10.1038/s41587-024-02135-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {N6-methyladenosine (m6A) and pseudouridine (Ψ) are the two most abundant modifications in mammalian messenger RNA, but the coordination of their biological functions remains poorly understood. We develop a machine learning-based nanopore direct RNA sequencing method (NanoSPA) that simultaneously analyzes m6A and Ψ in the human transcriptome. Applying NanoSPA to polysome profiling, we reveal opposing transcriptomic co-occurrence of m6A and Ψ and synergistic, hierarchical effects of m6A and Ψ on the polysome. A machine learning pipeline for nanopore sequencing analyzes m6A and Ψ simultaneously.},
  archive      = {J_NBT},
  author       = {Huang, Sihao and Wylder, Adam C. and Pan, Tao},
  doi          = {10.1038/s41587-024-02135-0},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1831-1835},
  shortjournal = {Nature Biotech.},
  title        = {Simultaneous nanopore profiling of mRNA m6A and pseudouridine reveals translation coordination},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Efficient prime editing in two-cell mouse embryos using
PEmbryo. <em>NBT</em>, <em>42</em>(12), 1822–1830. (<a
href="https://doi.org/10.1038/s41587-023-02106-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using transient inhibition of DNA mismatch repair during a permissive stage of development, we demonstrate highly efficient prime editing of mouse embryos with few unwanted, local byproducts (average 58% precise edit frequency, 0.5% on-target error frequency across 13 substitution edits at 8 sites), enabling same-generation phenotyping of founders. Whole-genome sequencing reveals that mismatch repair inhibition increases off-target indels at low-complexity regions in the genome without any obvious phenotype in mice. Prime editing is used to efficiently edit early-stage mouse embryos with few on-target byproducts.},
  archive      = {J_NBT},
  author       = {Kim-Yip, Rebecca P. and McNulty, Ryan and Joyce, Bradley and Mollica, Antonio and Chen, Peter J. and Ravisankar, Purnima and Law, Benjamin K. and Liu, David R. and Toettcher, Jared E. and Ivakine, Evgueni A. and Posfai, Eszter and Adamson, Britt},
  doi          = {10.1038/s41587-023-02106-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1822-1830},
  shortjournal = {Nature Biotech.},
  title        = {Efficient prime editing in two-cell mouse embryos using PEmbryo},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Biology and applications of CRISPR–Cas12 and
transposon-associated homologs. <em>NBT</em>, <em>42</em>(12),
1807–1821. (<a
href="https://doi.org/10.1038/s41587-024-02485-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR-associated Cas12 proteins are a highly variable collection of nucleic acid-targeting proteins. All Cas12 variants use RNA guides and a single nuclease domain to target complementary DNA or, in rare cases, RNA. The high variability of Cas12 effectors can be explained by a series of independent evolution events from different transposon-associated TnpB-like ancestors. Despite basic structural and functional similarities, this has resulted in unprecedented variation of the Cas12 effector proteins in terms of size, domain composition, guide structure, target identity and interference strategy. In this Review, we compare the unique molecular features of natural and engineered Cas12 and TnpB variants. Furthermore, we provide an overview of established genome editing and diagnostic applications and discuss potential future directions. Natural and engineered Cas12 and TnpB variants offer diverse applications in genome editing and diagnostics.},
  archive      = {J_NBT},
  author       = {Wu, Wen Y. and Adiego-Pérez, Belén and van der Oost, John},
  doi          = {10.1038/s41587-024-02485-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1807-1821},
  shortjournal = {Nature Biotech.},
  title        = {Biology and applications of CRISPR–Cas12 and transposon-associated homologs},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). KARR-seq maps higher-order RNA structures and RNA–RNA
interactions across the transcriptome. <em>NBT</em>, <em>42</em>(12),
1804–1805. (<a
href="https://doi.org/10.1038/s41587-023-02110-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping higher-order RNA structures and intermolecular RNA–RNA interactions throughout the transcriptome is critical for understanding RNA functions. We developed KARR-seq, a chemical-assisted RNA proximity capture and sequencing technology that enables sensitive and accurate detection of the RNA structurome and functional RNA–RNA interactions.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02110-1},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1804-1805},
  shortjournal = {Nature Biotech.},
  title        = {KARR-seq maps higher-order RNA structures and RNA–RNA interactions across the transcriptome},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Imbalanced single-cell data integration leads to loss of
biological information. <em>NBT</em>, <em>42</em>(12), 1802–1803. (<a
href="https://doi.org/10.1038/s41587-023-02114-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Iniquitate pipeline assessed the impacts of cell-type imbalance on single-cell RNA sequencing integration through perturbations to dataset balance. The results indicated that cell-type imbalance not only leads to loss of biological signal in the integrated space, but also can change the interpretation of downstream analyses after integration.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02114-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1802-1803},
  shortjournal = {Nature Biotech.},
  title        = {Imbalanced single-cell data integration leads to loss of biological information},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computer-based design of sensors that can monitor endogenous
ras activity. <em>NBT</em>, <em>42</em>(12), 1800–1801. (<a
href="https://doi.org/10.1038/s41587-023-02108-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many questions on the activity of the Ras proto-oncogene are unanswered due to the lack of tools for detecting active Ras in living cells. Here, we used protein design and structure prediction algorithms to develop biosensors that detect the activity and environment of endogenous Ras.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02108-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1800-1801},
  shortjournal = {Nature Biotech.},
  title        = {Computer-based design of sensors that can monitor endogenous ras activity},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A conditional, zinc-finger-dependent recombinase system for
DNA editing. <em>NBT</em>, <em>42</em>(12), 1798–1799. (<a
href="https://doi.org/10.1038/s41587-023-02122-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By applying the logic of conditional enzymes, we have developed a zinc-finger-dependent recombinase system, the editing activity of which is induced by zinc finger DNA binding. The system combines the precision of recombinases with the DNA target site programmability of zinc finger domains.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02122-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1798-1799},
  shortjournal = {Nature Biotech.},
  title        = {A conditional, zinc-finger-dependent recombinase system for DNA editing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A new mass analyzer shakes up the proteomics field.
<em>NBT</em>, <em>42</em>(12), 1796–1797. (<a
href="https://doi.org/10.1038/s41587-024-02129-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new mass analyzer enables protein identification at high speed and depth.},
  archive      = {J_NBT},
  author       = {Kuster, Bernhard and Tüshaus, Johanna and Bayer, Florian P.},
  doi          = {10.1038/s41587-024-02129-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1796-1797},
  shortjournal = {Nature Biotech.},
  title        = {A new mass analyzer shakes up the proteomics field},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Eavesdropping on brain organoids. <em>NBT</em>,
<em>42</em>(12), 1794–1795. (<a
href="https://doi.org/10.1038/s41587-024-02128-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A basket-like device records the electrical activity of neural organoids over months of maturation.},
  archive      = {J_NBT},
  author       = {Brandt, James Newton and Rajasethupathy, Priya},
  doi          = {10.1038/s41587-024-02128-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1794-1795},
  shortjournal = {Nature Biotech.},
  title        = {Eavesdropping on brain organoids},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024p). Discovering the crucial function of long noncoding RNAs.
<em>NBT</em>, <em>42</em>(12), 1793. (<a
href="https://doi.org/10.1038/s41587-024-02509-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02509-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1793},
  shortjournal = {Nature Biotech.},
  title        = {Discovering the crucial function of long noncoding RNAs},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024m). CRISPR unlocks a sweet tomato that keeps its size.
<em>NBT</em>, <em>42</em>(12), 1793. (<a
href="https://doi.org/10.1038/s41587-024-02510-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02510-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1793},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR unlocks a sweet tomato that keeps its size},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024u). Evo learns biological complexity from the molecular to
genome scale. <em>NBT</em>, <em>42</em>(12), 1793. (<a
href="https://doi.org/10.1038/s41587-024-02514-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02514-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1793},
  shortjournal = {Nature Biotech.},
  title        = {Evo learns biological complexity from the molecular to genome scale},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Stem cells. <em>NBT</em>, <em>42</em>(12), 1792. (<a
href="https://doi.org/10.1038/s41587-024-02496-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to compositions, methods of preparation and use of stem cells.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02496-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1792},
  shortjournal = {Nature Biotech.},
  title        = {Stem cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Leveraging machine learning and big data techniques to map
the global patent landscape of phage therapy. <em>NBT</em>,
<em>42</em>(12), 1781–1791. (<a
href="https://doi.org/10.1038/s41587-024-02493-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A patent analysis of the phage therapy field underscores a dynamic, evolving landscape marked by regional disparities and a shift in research focus towards broad-spectrum medical applications and treatments for bacterial infections.},
  archive      = {J_NBT},
  author       = {Mbogning Fonkou, Maxime Descartes and Kong, Jude Dzevela},
  doi          = {10.1038/s41587-024-02493-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1781-1791},
  shortjournal = {Nature Biotech.},
  title        = {Leveraging machine learning and big data techniques to map the global patent landscape of phage therapy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A technical approach to global plant genome editing
regulation. <em>NBT</em>, <em>42</em>(12), 1773–1780. (<a
href="https://doi.org/10.1038/s41587-024-02489-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Innovate Genomics Institute brought together regulators from 16 countries to discuss global capacity building for the regulation of genome-edited crops. The workshop provided insights into the suitable use of technical analyses to validate edits and raised future considerations regarding regulation reporting, offering suggestions to help countries meet their objectives in the ever-growing landscape of genome editing techniques.},
  archive      = {J_NBT},
  author       = {Groover, Evan and Njuguna, Elizabeth and Bansal, Kailash Chander and Muia, Anne and Kwehangana, Musa and Simuntala, Christopher and Mills, Richard Lloyd and Kwakye, Emmanuel and Rocha, Pedro and Amedu, Josephine and Morillo, Eduardo and Anthonysamy, Mohana Anita and Khaldun, A. B. M. and Chimpepo, Lilian and Akoudjin, Massouroudini and Senanayake, D. M. J. B. and Wangmo, Dechen and Atnafu, Dessalegn and Eusebio, Geronima P. and Kongsawat, Chalinee and Kliegman, Melinda},
  doi          = {10.1038/s41587-024-02489-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1773-1780},
  shortjournal = {Nature Biotech.},
  title        = {A technical approach to global plant genome editing regulation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Response to “why japan lacks a vibrant biotech industry.”
<em>NBT</em>, <em>42</em>(12), 1771–1772. (<a
href="https://doi.org/10.1038/s41587-024-02492-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ashida, Hiroki and Stanford, John},
  doi          = {10.1038/s41587-024-02492-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1771-1772},
  shortjournal = {Nature Biotech.},
  title        = {Response to ‘Why japan lacks a vibrant biotech industry’},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An update on cuban biotech. <em>NBT</em>, <em>42</em>(12),
1768–1770. (<a
href="https://doi.org/10.1038/s41587-024-02458-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Buxadó, José A. and Limonta Fernández, Miladys and Guillén Nieto, Gerardo E. and Gómez, Raimundo Ubieta and Ayala Ávila, Marta},
  doi          = {10.1038/s41587-024-02458-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1768-1770},
  shortjournal = {Nature Biotech.},
  title        = {An update on cuban biotech},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Challenges for developing broad-based mucosal vaccines for
respiratory viruses. <em>NBT</em>, <em>42</em>(12), 1765–1767. (<a
href="https://doi.org/10.1038/s41587-024-02486-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Seo, Junghwa and Polster, Jordan and Israelow, Benjamin and Corbett-Helaire, Kizzmekia S. and Martinez, David R.},
  doi          = {10.1038/s41587-024-02486-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1765-1767},
  shortjournal = {Nature Biotech.},
  title        = {Challenges for developing broad-based mucosal vaccines for respiratory viruses},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Plant bling: Cellulose that sparkles. <em>NBT</em>,
<em>42</em>(12), 1762. (<a
href="https://doi.org/10.1038/s41587-024-02507-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02507-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1762},
  shortjournal = {Nature Biotech.},
  title        = {Plant bling: Cellulose that sparkles},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BioNTech boosts oncology pipeline with china buy.
<em>NBT</em>, <em>42</em>(12), 1761. (<a
href="https://doi.org/10.1038/s41587-024-02518-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02518-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1761},
  shortjournal = {Nature Biotech.},
  title        = {BioNTech boosts oncology pipeline with china buy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Base editing boosts hemoglobin in sickle cell disease.
<em>NBT</em>, <em>42</em>(12), 1759. (<a
href="https://doi.org/10.1038/s41587-024-02517-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02517-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1759},
  shortjournal = {Nature Biotech.},
  title        = {Base editing boosts hemoglobin in sickle cell disease},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Top ten news stories in 2024. <em>NBT</em>, <em>42</em>(12),
1757–1758. (<a
href="https://doi.org/10.1038/s41587-024-02500-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {From mRNA drugs and the Achilles heel of obesity drugs, to single-cell biology and foundation models for drug discovery, these are some of the big stories most viewed by our readers.},
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-024-02500-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1757-1758},
  shortjournal = {Nature Biotech.},
  title        = {Top ten news stories in 2024},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). 2024: Research in review. <em>NBT</em>, <em>42</em>(12),
1755–1756. (<a
href="https://doi.org/10.1038/s41587-024-02508-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology editors pick their favorite research articles from 2024.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02508-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1755-1756},
  shortjournal = {Nature Biotech.},
  title        = {2024: Research in review},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Editor’s pick: Gate bioscience. <em>NBT</em>,
<em>42</em>(12), 1763–1764. (<a
href="https://doi.org/10.1038/s41587-024-02483-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Gate Bioscience wants to tailor gates that stop problematic proteins at the source.},
  archive      = {J_NBT},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41587-024-02483-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1763-1764},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Gate bioscience},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Lab-grown breast milk. <em>NBT</em>, <em>42</em>(12),
1759–1761. (<a
href="https://doi.org/10.1038/s41587-024-02498-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Meet the biotech startups brewing milk in bioreactors to improve on baby formula.},
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-024-02498-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1759-1761},
  shortjournal = {Nature Biotech.},
  title        = {Lab-grown breast milk},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). People. <em>NBT</em>, <em>42</em>(11), 1754. (<a
href="https://doi.org/10.1038/s41587-024-02464-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02464-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1754},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Behind the graduate mental health crisis in science.
<em>NBT</em>, <em>42</em>(11), 1749–1753. (<a
href="https://doi.org/10.1038/s41587-024-02457-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Survey results identify how scientific research and teaching contribute to the graduate student mental health crisis.},
  archive      = {J_NBT},
  author       = {Busch, Carly A. and Wiesenthal, Nicholas J. and Gin, Logan E. and Cooper, Katelyn M.},
  doi          = {10.1038/s41587-024-02457-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1749-1753},
  shortjournal = {Nature Biotech.},
  title        = {Behind the graduate mental health crisis in science},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: The cost of drug patent expiration date
errors. <em>NBT</em>, <em>42</em>(11), 1747. (<a
href="https://doi.org/10.1038/s41587-024-02459-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Tu, S. Sean and Cheian, Dinis and Gabriele, Sarah and Rome, Benjamin N. and Kesselheim, Aaron S.},
  doi          = {10.1038/s41587-024-02459-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1747},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: The cost of drug patent expiration date errors},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Decoder-seq enhances mRNA capture efficiency in spatial RNA
sequencing. <em>NBT</em>, <em>42</em>(11), 1735–1746. (<a
href="https://doi.org/10.1038/s41587-023-02086-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics technologies with high resolution often lack high sensitivity in mRNA detection. Here we report a dendrimeric DNA coordinate barcoding design for spatial RNA sequencing (Decoder-seq), which offers both high sensitivity and high resolution. Decoder-seq combines dendrimeric nanosubstrates with microfluidic coordinate barcoding to generate spatial arrays with a DNA density approximately ten times higher than previously reported methods while maintaining flexibility in resolution. We show that the high RNA capture efficiency of Decoder-seq improved the detection of lowly expressed olfactory receptor (Olfr) genes in mouse olfactory bulbs and contributed to the discovery of a unique layer enrichment pattern for two Olfr genes. The near-cellular resolution provided by Decoder-seq has enabled the construction of a spatial single-cell atlas of the mouse hippocampus, revealing dendrite-enriched mRNAs in neurons. When applying Decoder-seq to human renal cell carcinomas, we dissected the heterogeneous tumor microenvironment across different cancer subtypes and identified spatial gradient-expressed genes related to epithelial–mesenchymal transition with the potential to predict tumor prognosis and progression. A spatial transcriptomics method achieves high gene detection sensitivity and flexible resolution.},
  archive      = {J_NBT},
  author       = {Cao, Jiao and Zheng, Zhong and Sun, Di and Chen, Xin and Cheng, Rui and Lv, Tianpeng and An, Yu and Zheng, Junhua and Song, Jia and Wu, Lingling and Yang, Chaoyong},
  doi          = {10.1038/s41587-023-02086-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1735-1746},
  shortjournal = {Nature Biotech.},
  title        = {Decoder-seq enhances mRNA capture efficiency in spatial RNA sequencing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Prediction of single-cell RNA expression profiles in live
cells by raman microscopy with Raman2RNA. <em>NBT</em>, <em>42</em>(11),
1726–1734. (<a
href="https://doi.org/10.1038/s41587-023-02082-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing and other profiling assays have helped interrogate cells at unprecedented resolution and scale, but are inherently destructive. Raman microscopy reports on the vibrational energy levels of proteins and metabolites in a label-free and nondestructive manner at subcellular spatial resolution, but it lacks genetic and molecular interpretability. Here we present Raman2RNA (R2R), a method to infer single-cell expression profiles in live cells through label-free hyperspectral Raman microscopy images and domain translation. We predict single-cell RNA sequencing profiles nondestructively from Raman images using either anchor-based integration with single molecule fluorescence in situ hybridization, or anchor-free generation with adversarial autoencoders. R2R outperformed inference from brightfield images (cosine similarities: R2R &amp;gt;0.85 and brightfield &amp;lt;0.15). In reprogramming of mouse fibroblasts into induced pluripotent stem cells, R2R inferred the expression profiles of various cell states. With live-cell tracking of mouse embryonic stem cell differentiation, R2R traced the early emergence of lineage divergence and differentiation trajectories, overcoming discontinuities in expression space. R2R lays a foundation for future exploration of live genomic dynamics. Transcriptional profiles of single living cells are generated from Raman imaging data.},
  archive      = {J_NBT},
  author       = {Kobayashi-Kirschvink, Koseki J. and Comiter, Charles S. and Gaddam, Shreya and Joren, Taylor and Grody, Emanuelle I. and Ounadjela, Johain R. and Zhang, Ke and Ge, Baoliang and Kang, Jeon Woong and Xavier, Ramnik J. and So, Peter T. C. and Biancalani, Tommaso and Shu, Jian and Regev, Aviv},
  doi          = {10.1038/s41587-023-02082-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1726-1734},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of single-cell RNA expression profiles in live cells by raman microscopy with Raman2RNA},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Engineered serum markers for non-invasive monitoring of gene
expression in the brain. <em>NBT</em>, <em>42</em>(11), 1717–1725. (<a
href="https://doi.org/10.1038/s41587-023-02087-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Measurement of gene expression in the brain requires invasive analysis of brain tissue or non-invasive methods that are limited by low sensitivity. Here we introduce a method for non-invasive, multiplexed, site-specific monitoring of endogenous gene or transgene expression in the brain through engineered reporters called released markers of activity (RMAs). RMAs consist of an easily detectable reporter and a receptor-binding domain that enables transcytosis across the brain endothelium. RMAs are expressed in the brain but exit into the blood, where they can be easily measured. We show that expressing RMAs at a single mouse brain site representing approximately 1% of the brain volume provides up to a 100,000-fold signal increase over the baseline. Expression of RMAs in tens to hundreds of neurons is sufficient for their reliable detection. We demonstrate that chemogenetic activation of cells expressing Fos-responsive RMA increases serum RMA levels &amp;gt;6-fold compared to non-activated controls. RMAs provide a non-invasive method for repeatable, multiplexed monitoring of gene expression in the intact animal brain. Gene expression in the mouse brain is measured with a simple blood draw.},
  archive      = {J_NBT},
  author       = {Lee, Sangsin and Nouraein, Shirin and Kwon, James J. and Huang, Zhimin and Wojick, Jessica A. and Xia, Boao and Corder, Gregory and Szablowski, Jerzy O.},
  doi          = {10.1038/s41587-023-02087-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1717-1725},
  shortjournal = {Nature Biotech.},
  title        = {Engineered serum markers for non-invasive monitoring of gene expression in the brain},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Slow release of a synthetic auxin induces formation of
adventitious roots in recalcitrant woody plants. <em>NBT</em>,
<em>42</em>(11), 1705–1716. (<a
href="https://doi.org/10.1038/s41587-023-02065-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clonal propagation of plants by induction of adventitious roots (ARs) from stem cuttings is a requisite step in breeding programs. A major barrier exists for propagating valuable plants that naturally have low capacity to form ARs. Due to the central role of auxin in organogenesis, indole-3-butyric acid is often used as part of commercial rooting mixtures, yet many recalcitrant plants do not form ARs in response to this treatment. Here we describe the synthesis and screening of a focused library of synthetic auxin conjugates in Eucalyptus grandis cuttings and identify 4-chlorophenoxyacetic acid–l-tryptophan-OMe as a competent enhancer of adventitious rooting in a number of recalcitrant woody plants, including apple and argan. Comprehensive metabolic and functional analyses reveal that this activity is engendered by prolonged auxin signaling due to initial fast uptake and slow release and clearance of the free auxin 4-chlorophenoxyacetic acid. This work highlights the utility of a slow-release strategy for bioactive compounds for more effective plant growth regulation. Adventitious roots are induced in various woody plants, enabling clonal propagation.},
  archive      = {J_NBT},
  author       = {Roth, Ohad and Yechezkel, Sela and Serero, Ori and Eliyahu, Avi and Vints, Inna and Tzeela, Pan and Carignano, Alberto and Janacek, Dorina P. and Peters, Verena and Kessel, Amit and Dwivedi, Vikas and Carmeli-Weissberg, Mira and Shaya, Felix and Faigenboim-Doron, Adi and Ung, Kien Lam and Pedersen, Bjørn Panyella and Riov, Joseph and Klavins, Eric and Dawid, Corinna and Hammes, Ulrich Z. and Ben-Tal, Nir and Napier, Richard and Sadot, Einat and Weinstain, Roy},
  doi          = {10.1038/s41587-023-02065-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1705-1716},
  shortjournal = {Nature Biotech.},
  title        = {Slow release of a synthetic auxin induces formation of adventitious roots in recalcitrant woody plants},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). IL-10-expressing CAR t cells resist dysfunction and mediate
durable clearance of solid tumors and metastases. <em>NBT</em>,
<em>42</em>(11), 1693–1704. (<a
href="https://doi.org/10.1038/s41587-023-02060-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The success of chimeric antigen receptor (CAR) T cell therapy in treating several hematopoietic malignancies has been difficult to replicate in solid tumors, in part because of T cell exhaustion and eventually dysfunction. To counter T cell dysfunction in the tumor microenvironment, we metabolically armored CAR T cells by engineering them to secrete interleukin-10 (IL-10). We show that IL-10 CAR T cells preserve intact mitochondrial structure and function in the tumor microenvironment and increase oxidative phosphorylation in a mitochondrial pyruvate carrier-dependent manner. IL-10 secretion promoted proliferation and effector function of CAR T cells, leading to complete regression of established solid tumors and metastatic cancers across several cancer types in syngeneic and xenograft mouse models, including colon cancer, breast cancer, melanoma and pancreatic cancer. IL-10 CAR T cells also induced stem cell-like memory responses in lymphoid organs that imparted durable protection against tumor rechallenge. Our results establish a generalizable approach to counter CAR T cell dysfunction through metabolic armoring, leading to solid tumor eradication and long-lasting immune protection. CAR T cells engineered to express IL-10 eradicate solid tumors and metastases.},
  archive      = {J_NBT},
  author       = {Zhao, Yang and Chen, Jiangqing and Andreatta, Massimo and Feng, Bing and Xie, Yu-Qing and Wenes, Mathias and Wang, Yi and Gao, Min and Hu, Xiaomeng and Romero, Pedro and Carmona, Santiago and Sun, Jie and Guo, Yugang and Tang, Li},
  doi          = {10.1038/s41587-023-02060-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1693-1704},
  shortjournal = {Nature Biotech.},
  title        = {IL-10-expressing CAR t cells resist dysfunction and mediate durable clearance of solid tumors and metastases},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). In vivo human t cell engineering with enveloped delivery
vehicles. <em>NBT</em>, <em>42</em>(11), 1684–1692. (<a
href="https://doi.org/10.1038/s41587-023-02085-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Viruses and virally derived particles have the intrinsic capacity to deliver molecules to cells, but the difficulty of readily altering cell-type selectivity has hindered their use for therapeutic delivery. Here, we show that cell surface marker recognition by antibody fragments displayed on membrane-derived particles encapsulating CRISPR–Cas9 protein and guide RNA can deliver genome editing tools to specific cells. Compared to conventional vectors like adeno-associated virus that rely on evolved capsid tropisms to deliver virally encoded cargo, these Cas9-packaging enveloped delivery vehicles (Cas9-EDVs) leverage predictable antibody–antigen interactions to transiently deliver genome editing machinery selectively to cells of interest. Antibody-targeted Cas9-EDVs preferentially confer genome editing in cognate target cells over bystander cells in mixed populations, both ex vivo and in vivo. By using multiplexed targeting molecules to direct delivery to human T cells, Cas9-EDVs enable the generation of genome-edited chimeric antigen receptor T cells in humanized mice, establishing a programmable delivery modality with the potential for widespread therapeutic utility. Cell-specific molecular delivery with enveloped delivery vehicles enables genome editing ex vivo and in vivo.},
  archive      = {J_NBT},
  author       = {Hamilton, Jennifer R. and Chen, Evelyn and Perez, Barbara S. and Sandoval Espinoza, Cindy R. and Kang, Min Hyung and Trinidad, Marena and Ngo, Wayne and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-023-02085-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1684-1692},
  shortjournal = {Nature Biotech.},
  title        = {In vivo human t cell engineering with enveloped delivery vehicles},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Disentanglement of single-cell data with biolord.
<em>NBT</em>, <em>42</em>(11), 1678–1683. (<a
href="https://doi.org/10.1038/s41587-023-02079-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biolord is a deep generative method for disentangling single-cell multi-omic data to known and unknown attributes, including spatial, temporal and disease states, used to reveal the decoupled biological signatures over diverse single-cell modalities and biological systems. By virtually shifting cells across states, biolord generates experimentally inaccessible samples, outperforming state-of-the-art methods in predictions of cellular response to unseen drugs and genetic perturbations. Biolord is available at https://github.com/nitzanlab/biolord . Biolord models the contributions of diverse biological processes to single-cell expression profiles.},
  archive      = {J_NBT},
  author       = {Piran, Zoe and Cohen, Niv and Hoshen, Yedid and Nitzan, Mor},
  doi          = {10.1038/s41587-023-02079-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1678-1683},
  shortjournal = {Nature Biotech.},
  title        = {Disentanglement of single-cell data with biolord},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Fast mass spectrometry search and clustering of untargeted
metabolomics data. <em>NBT</em>, <em>42</em>(11), 1672–1677. (<a
href="https://doi.org/10.1038/s41587-023-01985-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The throughput of mass spectrometers and the amount of publicly available metabolomics data are growing rapidly, but analysis tools such as molecular networking and Mass Spectrometry Search Tool do not scale to searching and clustering billions of mass spectral data in metabolomics repositories. To address this limitation, we designed MASST+ and Networking+, which can process datasets that are up to three orders of magnitude larger than those processed by state-of-the-art tools. MASST+ speeds up querying of metabolomics mass spectrometry data by two orders of magnitude.},
  archive      = {J_NBT},
  author       = {Mongia, Mihir and Yasaka, Tyler M. and Liu, Yudong and Guler, Mustafa and Lu, Liang and Bhagwat, Aditya and Behsaz, Bahar and Wang, Mingxun and Dorrestein, Pieter C. and Mohimani, Hosein},
  doi          = {10.1038/s41587-023-01985-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1672-1677},
  shortjournal = {Nature Biotech.},
  title        = {Fast mass spectrometry search and clustering of untargeted metabolomics data},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High prevalence of acquired cancer-related mutations in 146
human pluripotent stem cell lines and their differentiated derivatives.
<em>NBT</em>, <em>42</em>(11), 1667–1671. (<a
href="https://doi.org/10.1038/s41587-023-02090-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To survey cancer-related mutations in human pluripotent stem cells and their derivatives, we analyzed &amp;gt;2,200 transcriptomes from 146 independent lines in the NCBI’s Sequence Read Archive. Twenty-two per cent of samples had at least one cancer-related mutation; of these, 64% had TP53 mutations, which conferred a pronounced selective advantage, perturbed target gene expression and altered cellular differentiation. These findings underscore the need for robust surveillance of cancer-related mutations in pluripotent cells, especially in clinical applications. Many human pluripotent stem cell lines have acquired cancer-related mutations.},
  archive      = {J_NBT},
  author       = {Lezmi, Elyad and Jung, Jonathan and Benvenisty, Nissim},
  doi          = {10.1038/s41587-023-02090-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1667-1671},
  shortjournal = {Nature Biotech.},
  title        = {High prevalence of acquired cancer-related mutations in 146 human pluripotent stem cell lines and their differentiated derivatives},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Delivery of a sebum modulator by an engineered skin microbe
in mice. <em>NBT</em>, <em>42</em>(11), 1661–1666. (<a
href="https://doi.org/10.1038/s41587-023-02072-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microorganisms can be equipped with synthetic genetic programs for the production of targeted therapeutic molecules. Cutibacterium acnes is the most abundant commensal of the human skin, making it an attractive chassis to create skin-delivered therapeutics. Here, we report the engineering of this bacterium to produce and secrete the therapeutic molecule neutrophil gelatinase-associated lipocalin, in vivo, for the modulation of cutaneous sebum production. An engineered skin microbe produces a therapeutic molecule that reduces sebum in mice.},
  archive      = {J_NBT},
  author       = {Knödlseder, Nastassia and Fábrega, María-José and Santos-Moreno, Javier and Manils, Joan and Toloza, Lorena and Marín Vilar, Maria and Fernández, Cristina and Broadbent, Katrina and Maruotti, Julien and Lemenager, Hélène and Carolis, Carlo and Zouboulis, Christos C. and Soler, Concepció and Lood, Rolf and Brüggemann, Holger and Güell, Marc},
  doi          = {10.1038/s41587-023-02072-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1661-1666},
  shortjournal = {Nature Biotech.},
  title        = {Delivery of a sebum modulator by an engineered skin microbe in mice},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Generating expression profiles of single living cells from
raman microscopy. <em>NBT</em>, <em>42</em>(11), 1658–1659. (<a
href="https://doi.org/10.1038/s41587-023-02083-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expression profiles of single live cells are generated from Raman microscopy using deep learning, enabling us to track expression dynamics along cell reprogramming or differentiation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02083-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1658-1659},
  shortjournal = {Nature Biotech.},
  title        = {Generating expression profiles of single living cells from raman microscopy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Monitoring gene expression in the brain with a blood test.
<em>NBT</em>, <em>42</em>(11), 1656–1657. (<a
href="https://doi.org/10.1038/s41587-023-02088-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed synthetic serum markers that can be expressed in the brain but are transported into the blood, enabling minimally invasive monitoring of gene expression in the brain with a simple blood test.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02088-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1656-1657},
  shortjournal = {Nature Biotech.},
  title        = {Monitoring gene expression in the brain with a blood test},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Metabolically enhanced CAR t cells efficiently clear solid
tumors in mice. <em>NBT</em>, <em>42</em>(11), 1654–1655. (<a
href="https://doi.org/10.1038/s41587-023-02061-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor (CAR) T cells in the solid tumor microenvironment enter a partially dysfunctional state called T cell exhaustion. Interleukin (IL)-10-producing CAR T cells retain their metabolic fitness, resist T cell exhaustion and display unprecedented antitumor activity indicated by high cure rates and durable responses in mice. Clinical studies of IL-10-producing CAR T cells are underway.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02061-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1654-1655},
  shortjournal = {Nature Biotech.},
  title        = {Metabolically enhanced CAR t cells efficiently clear solid tumors in mice},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). From bench to podcast: Ali shaib. <em>NBT</em>,
<em>42</em>(11), 1653. (<a
href="https://doi.org/10.1038/s41587-024-02438-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02438-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1653},
  shortjournal = {Nature Biotech.},
  title        = {From bench to podcast: Ali shaib},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A synthetic auxin for cloning mature trees. <em>NBT</em>,
<em>42</em>(11), 1651–1652. (<a
href="https://doi.org/10.1038/s41587-024-02132-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A synthetic auxin improves the growth of adventitious roots in various plant species.},
  archive      = {J_NBT},
  author       = {Bellini, Catherine},
  doi          = {10.1038/s41587-024-02132-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1651-1652},
  shortjournal = {Nature Biotech.},
  title        = {A synthetic auxin for cloning mature trees},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024aa). LucaProt reveals the diverse global RNA virome.
<em>NBT</em>, <em>42</em>(11), 1650. (<a
href="https://doi.org/10.1038/s41587-024-02476-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02476-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1650},
  shortjournal = {Nature Biotech.},
  title        = {LucaProt reveals the diverse global RNA virome},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024w). Glucose-responsive insulin for diabetes. <em>NBT</em>,
<em>42</em>(11), 1650. (<a
href="https://doi.org/10.1038/s41587-024-02477-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02477-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1650},
  shortjournal = {Nature Biotech.},
  title        = {Glucose-responsive insulin for diabetes},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A bacterial neoantigen cancer vaccine. <em>NBT</em>,
<em>42</em>(11), 1650. (<a
href="https://doi.org/10.1038/s41587-024-02479-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02479-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1650},
  shortjournal = {Nature Biotech.},
  title        = {A bacterial neoantigen cancer vaccine},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Virus-like elements. <em>NBT</em>, <em>42</em>(11), 1649.
(<a href="https://doi.org/10.1038/s41587-024-02452-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to retroviruses, transposons and other virus-like elements for use in gene transfer and gene therapy.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02452-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1649},
  shortjournal = {Nature Biotech.},
  title        = {Virus-like elements},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Who made the mRNA vaccine? Measuring division of labor in
therapeutic innovation. <em>NBT</em>, <em>42</em>(11), 1643–1648. (<a
href="https://doi.org/10.1038/s41587-024-02441-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To accelerate future vaccine development, understanding the longer-term funding dynamics behind recent successes such as the USA’s Operation Warp Speed and the UK’s Vaccine Taskforce is imperative.},
  archive      = {J_NBT},
  author       = {Ho, Martin and Price, Henry C. W. and Evans, Tim S. and O’Sullivan, Eoin},
  doi          = {10.1038/s41587-024-02441-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1643-1648},
  shortjournal = {Nature Biotech.},
  title        = {Who made the mRNA vaccine? measuring division of labor in therapeutic innovation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). AIntibody: An experimentally validated in silico antibody
discovery design challenge. <em>NBT</em>, <em>42</em>(11), 1637–1642.
(<a href="https://doi.org/10.1038/s41587-024-02469-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Erasmus, M. Frank and Spector, Laura and Ferrara, Fortunato and DiNiro, Roberto and Pohl, Thomas J. and Perea-Schmittle, Katheryn and Wang, Wei and Tessier, Peter M. and Richardson, Crystal and Turner, Laure and Kumar, Sumit and Bedinger, Daniel and Sormanni, Pietro and Fernández-Quintero, Monica L. and Ward, Andrew B. and Loeffler, Johannes R. and Swanson, Olivia M. and Deane, Charlotte M. and Raybould, Matthew I. J. and Evers, Andreas and Sellmann, Carolin and Bachas, Sharrol and Ruffolo, Jeff and Nastri, Horacio G. and Ramesh, Karthik and Sørensen, Jesper and Croasdale-Wood, Rebecca and Hijano, Oliver and Leal-Lopes, Camila and Shahsavarian, Melody and Qiu, Yu and Marcatili, Paolo and Vernet, Erik and Akbar, Rahmad and Friedensohn, Simon and Wagner, Rick and Kurella, Vinodh babu and Malhotra, Shipra and Kumar, Satyendra and Kidger, Patrick and Almagro, Juan C. and Furfine, Eric and Stanton, Marty and Graff, Christilyn P. and Villalba, Santiago David and Tomszak, Florian and Teixeira, Andre A. R. and Hopkins, Elizabeth and Dovner, Molly and D’Angelo, Sara and Bradbury, Andrew R. M.},
  doi          = {10.1038/s41587-024-02469-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1637-1642},
  shortjournal = {Nature Biotech.},
  title        = {AIntibody: An experimentally validated in silico antibody discovery design challenge},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(11), 1632. (<a
href="https://doi.org/10.1038/s41587-024-02474-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02474-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1632},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Wasp gut microbes yield beer with extra tang. <em>NBT</em>,
<em>42</em>(11), 1631. (<a
href="https://doi.org/10.1038/s41587-024-02473-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-024-02473-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1631},
  shortjournal = {Nature Biotech.},
  title        = {Wasp gut microbes yield beer with extra tang},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Italy tests first gene-edited vines for winemaking.
<em>NBT</em>, <em>42</em>(11), 1630. (<a
href="https://doi.org/10.1038/s41587-024-02478-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Europe’s first field trial of gene-edited vines began in northern Italy on 30 September 2024. Developed by EdiVite, a spinoff from the University of Verona, these Chardonnay vines have undergone gene inactivation to enable them to better defend themselves against downy mildew, a major fungal disease. The trial is being conducted on university land, with plans to expand to another site in the Veneto region. Researchers aim to gather initial data by 2025, with the potential for experimental winemaking in 2026.},
  archive      = {J_NBT},
  author       = {Meldolesi, Anna},
  doi          = {10.1038/s41587-024-02478-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1630},
  shortjournal = {Nature Biotech.},
  title        = {Italy tests first gene-edited vines for winemaking},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). CRISPR nobelists surrender their own european patents.
<em>NBT</em>, <em>42</em>(11), 1629. (<a
href="https://doi.org/10.1038/s41587-024-02472-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A strategic move by lawyers acting for Doudna and Charpentier is the latest twist in the battleground for CRISPR–Cas9 technology.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-024-02472-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1629},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR nobelists surrender their own european patents},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Shot in the arm for biotech fuels. <em>NBT</em>,
<em>42</em>(11), 1628. (<a
href="https://doi.org/10.1038/s41587-024-02484-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02484-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1628},
  shortjournal = {Nature Biotech.},
  title        = {Shot in the arm for biotech fuels},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). First proof-of-mechanism for RNA editing in humans.
<em>NBT</em>, <em>42</em>(11), 1627. (<a
href="https://doi.org/10.1038/s41587-024-02481-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02481-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1627},
  shortjournal = {Nature Biotech.},
  title        = {First proof-of-mechanism for RNA editing in humans},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cancer cases in bluebird’s gene therapy trials.
<em>NBT</em>, <em>42</em>(11), 1626. (<a
href="https://doi.org/10.1038/s41587-024-02482-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02482-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1626},
  shortjournal = {Nature Biotech.},
  title        = {Cancer cases in bluebird’s gene therapy trials},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Amyloid is everywhere, but new treatments could stop the
toxic build up. <em>NBT</em>, <em>42</em>(11), 1625–1628. (<a
href="https://doi.org/10.1038/s41587-024-02471-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amyloid deposits are present in the brains of cognitively impaired individuals, but also accumulate throughout the body. Some biotechs aim to keep these misfolded proteins in check to treat a range of diseases, including heart failure, ALS and Alzheimer’s disease.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-024-02471-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1625-1628},
  shortjournal = {Nature Biotech.},
  title        = {Amyloid is everywhere, but new treatments could stop the toxic build up},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). What will it take to get miRNA therapies to market?
<em>NBT</em>, <em>42</em>(11), 1623–1624. (<a
href="https://doi.org/10.1038/s41587-024-02480-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Nobel Prize in medicine was awarded for the discovery of miRNA, but miRNA therapeutics have a long way to go before they outcompete other therapies.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02480-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1623-1624},
  shortjournal = {Nature Biotech.},
  title        = {What will it take to get miRNA therapies to market?},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pest control gets the CRISPR treatment. <em>NBT</em>,
<em>42</em>(11), 1633–1636. (<a
href="https://doi.org/10.1038/s41587-024-02449-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precision genetic methods are enabling more efficient, environmentally friendly pest control methods for agricultural use as well as stopping the spread of disease.},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-024-02449-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {11},
  pages        = {1633-1636},
  shortjournal = {Nature Biotech.},
  title        = {Pest control gets the CRISPR treatment},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). People. <em>NBT</em>, <em>42</em>(10), 1622. (<a
href="https://doi.org/10.1038/s41587-024-02421-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02421-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1622},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Five questions with weixin tang. <em>NBT</em>,
<em>42</em>(10), 1621. (<a
href="https://doi.org/10.1038/s41587-024-02419-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A chemist pioneering the discovery and engineering of CRISPR-derived gene-editing agents for medical adaptation talks about her scientific journey, the challenges of leading a lab and teaching undergraduates.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02419-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1621},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with weixin tang},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accessible training for academic clinicians and researchers
to address the need for academic entrepreneurs. <em>NBT</em>,
<em>42</em>(10), 1618–1620. (<a
href="https://doi.org/10.1038/s41587-024-02417-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A short course in the fundamentals of academic entrepreneurship is a feasible way to train scientists and physicians from varied backgrounds and experience.},
  archive      = {J_NBT},
  author       = {Bartek, Victoria and Ferro, Daria and Winston, Flaura and Kessel, Mark},
  doi          = {10.1038/s41587-024-02417-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1618-1620},
  shortjournal = {Nature Biotech.},
  title        = {Accessible training for academic clinicians and researchers to address the need for academic entrepreneurs},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Publisher correction: Detection of mosaic and
population-level structural variants with sniffles2. <em>NBT</em>,
<em>42</em>(10), 1616. (<a
href="https://doi.org/10.1038/s41587-024-02141-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Smolka, Moritz and Paulin, Luis F. and Grochowski, Christopher M. and Horner, Dominic W. and Mahmoud, Medhat and Behera, Sairam and Kalef-Ezra, Ester and Gandhi, Mira and Hong, Karl and Pehlivan, Davut and Scholz, Sonja W. and Carvalho, Claudia M. B. and Proukakis, Christos and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41587-024-02141-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1616},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Detection of mosaic and population-level structural variants with sniffles2},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Combined small-molecule treatment
accelerates maturation of human pluripotent stem cell-derived neurons.
<em>NBT</em>, <em>42</em>(10), 1615. (<a
href="https://doi.org/10.1038/s41587-024-02233-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hergenreder, Emiliano and Minotti, Andrew P. and Zorina, Yana and Oberst, Polina and Zhao, Zeping and Munguba, Hermany and Calder, Elizabeth L. and Baggiolini, Arianna and Walsh, Ryan M. and Liston, Conor and Levitz, Joshua and Garippa, Ralph and Chen, Shuibing and Ciceri, Gabriele and Studer, Lorenz},
  doi          = {10.1038/s41587-024-02233-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1615},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Combined small-molecule treatment accelerates maturation of human pluripotent stem cell-derived neurons},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Characterization and visualization of tandem repeats at
genome scale. <em>NBT</em>, <em>42</em>(10), 1606–1614. (<a
href="https://doi.org/10.1038/s41587-023-02057-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tandem repeat (TR) variation is associated with gene expression changes and numerous rare monogenic diseases. Although long-read sequencing provides accurate full-length sequences and methylation of TRs, there is still a need for computational methods to profile TRs across the genome. Here we introduce the Tandem Repeat Genotyping Tool (TRGT) and an accompanying TR database. TRGT determines the consensus sequences and methylation levels of specified TRs from PacBio HiFi sequencing data. It also reports reads that support each repeat allele. These reads can be subsequently visualized with a companion TR visualization tool. Assessing 937,122 TRs, TRGT showed a Mendelian concordance of 98.38%, allowing a single repeat unit difference. In six samples with known repeat expansions, TRGT detected all expansions while also identifying methylation signals and mosaicism and providing finer repeat length resolution than existing methods. Additionally, we released a database with allele sequences and methylation levels for 937,122 TRs across 100 genomes. A set of tools maps tandem repeats across complete genomes.},
  archive      = {J_NBT},
  author       = {Dolzhenko, Egor and English, Adam and Dashnow, Harriet and De Sena Brandine, Guilherme and Mokveld, Tom and Rowell, William J. and Karniski, Caitlin and Kronenberg, Zev and Danzi, Matt C. and Cheung, Warren A. and Bi, Chengpeng and Farrow, Emily and Wenger, Aaron and Chua, Khi Pin and Martínez-Cerdeño, Verónica and Bartley, Trevor D. and Jin, Peng and Nelson, David L. and Zuchner, Stephan and Pastinen, Tomi and Quinlan, Aaron R. and Sedlazeck, Fritz J. and Eberle, Michael A.},
  doi          = {10.1038/s41587-023-02057-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1606-1614},
  shortjournal = {Nature Biotech.},
  title        = {Characterization and visualization of tandem repeats at genome scale},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mosaic integration and knowledge transfer of single-cell
multimodal data with MIDAS. <em>NBT</em>, <em>42</em>(10), 1594–1605.
(<a href="https://doi.org/10.1038/s41587-023-02040-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrating single-cell datasets produced by multiple omics technologies is essential for defining cellular heterogeneity. Mosaic integration, in which different datasets share only some of the measured modalities, poses major challenges, particularly regarding modality alignment and batch effect removal. Here, we present a deep probabilistic framework for the mosaic integration and knowledge transfer (MIDAS) of single-cell multimodal data. MIDAS simultaneously achieves dimensionality reduction, imputation and batch correction of mosaic data by using self-supervised modality alignment and information-theoretic latent disentanglement. We demonstrate its superiority to 19 other methods and reliability by evaluating its performance in trimodal and mosaic integration tasks. We also constructed a single-cell trimodal atlas of human peripheral blood mononuclear cells and tailored transfer learning and reciprocal reference mapping schemes to enable flexible and accurate knowledge transfer from the atlas to new data. Applications in mosaic integration, pseudotime analysis and cross-tissue knowledge transfer on bone marrow mosaic datasets demonstrate the versatility and superiority of MIDAS. MIDAS is available at https://github.com/labomics/midas . Single-cell, multiomic datasets are integrated using dimensionality reduction, imputation and batch correction.},
  archive      = {J_NBT},
  author       = {He, Zhen and Hu, Shuofeng and Chen, Yaowen and An, Sijing and Zhou, Jiahao and Liu, Runyan and Shi, Junfeng and Wang, Jing and Dong, Guohua and Shi, Jinhui and Zhao, Jiaxin and Ou-Yang, Le and Zhu, Yuan and Bo, Xiaochen and Ying, Xiaomin},
  doi          = {10.1038/s41587-023-02040-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1594-1605},
  shortjournal = {Nature Biotech.},
  title        = {Mosaic integration and knowledge transfer of single-cell multimodal data with MIDAS},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Discovery of sparse, reliable omic biomarkers with stabl.
<em>NBT</em>, <em>42</em>(10), 1581–1593. (<a
href="https://doi.org/10.1038/s41587-023-02033-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adoption of high-content omic technologies in clinical studies, coupled with computational methods, has yielded an abundance of candidate biomarkers. However, translating such findings into bona fide clinical biomarkers remains challenging. To facilitate this process, we introduce Stabl, a general machine learning method that identifies a sparse, reliable set of biomarkers by integrating noise injection and a data-driven signal-to-noise threshold into multivariable predictive modeling. Evaluation of Stabl on synthetic datasets and five independent clinical studies demonstrates improved biomarker sparsity and reliability compared to commonly used sparsity-promoting regularization methods while maintaining predictive performance; it distills datasets containing 1,400–35,000 features down to 4–34 candidate biomarkers. Stabl extends to multi-omic integration tasks, enabling biological interpretation of complex predictive models, as it hones in on a shortlist of proteomic, metabolomic and cytometric events predicting labor onset, microbial biomarkers of pre-term birth and a pre-operative immune signature of post-surgical infections. Stabl is available at https://github.com/gregbellan/Stabl . Stabl selects sparse and reliable biomarker candidates from predictive models.},
  archive      = {J_NBT},
  author       = {Hédou, Julien and Marić, Ivana and Bellan, Grégoire and Einhaus, Jakob and Gaudillière, Dyani K. and Ladant, Francois-Xavier and Verdonk, Franck and Stelzer, Ina A. and Feyaerts, Dorien and Tsai, Amy S. and Ganio, Edward A. and Sabayev, Maximilian and Gillard, Joshua and Amar, Jonas and Cambriel, Amelie and Oskotsky, Tomiko T. and Roldan, Alennie and Golob, Jonathan L. and Sirota, Marina and Bonham, Thomas A. and Sato, Masaki and Diop, Maïgane and Durand, Xavier and Angst, Martin S. and Stevenson, David K. and Aghaeepour, Nima and Montanari, Andrea and Gaudillière, Brice},
  doi          = {10.1038/s41587-023-02033-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1581-1593},
  shortjournal = {Nature Biotech.},
  title        = {Discovery of sparse, reliable omic biomarkers with stabl},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Detection of mosaic and population-level structural
variants with sniffles2. <em>NBT</em>, <em>42</em>(10), 1571–1580. (<a
href="https://doi.org/10.1038/s41587-023-02024-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Calling structural variations (SVs) is technically challenging, but using long reads remains the most accurate way to identify complex genomic alterations. Here we present Sniffles2, which improves over current methods by implementing a repeat aware clustering coupled with a fast consensus sequence and coverage-adaptive filtering. Sniffles2 is 11.8 times faster and 29% more accurate than state-of-the-art SV callers across different coverages (5–50×), sequencing technologies (ONT and HiFi) and SV types. Furthermore, Sniffles2 solves the problem of family-level to population-level SV calling to produce fully genotyped VCF files. Across 11 probands, we accurately identified causative SVs around MECP2, including highly complex alleles with three overlapping SVs. Sniffles2 also enables the detection of mosaic SVs in bulk long-read data. As a result, we identified multiple mosaic SVs in brain tissue from a patient with multiple system atrophy. The identified SV showed a remarkable diversity within the cingulate cortex, impacting both genes involved in neuron function and repetitive elements. Sniffles2 detects mosaic structural variation from bulk long-read sequencing data.},
  archive      = {J_NBT},
  author       = {Smolka, Moritz and Paulin, Luis F. and Grochowski, Christopher M. and Horner, Dominic W. and Mahmoud, Medhat and Behera, Sairam and Kalef-Ezra, Ester and Gandhi, Mira and Hong, Karl and Pehlivan, Davut and Scholz, Sonja W. and Carvalho, Claudia M. B. and Proukakis, Christos and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41587-023-02024-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1571-1580},
  shortjournal = {Nature Biotech.},
  title        = {Detection of mosaic and population-level structural variants with sniffles2},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ultrafast bisulfite sequencing detection of 5-methylcytosine
in DNA and RNA. <em>NBT</em>, <em>42</em>(10), 1559–1570. (<a
href="https://doi.org/10.1038/s41587-023-02034-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bisulfite sequencing (BS-seq) to detect 5-methylcytosine (5mC) is limited by lengthy reaction times, severe DNA damage, overestimation of 5mC level and incomplete C-to-U conversion of certain DNA sequences. We present ultrafast BS-seq (UBS-seq), which uses highly concentrated bisulfite reagents and high reaction temperatures to accelerate the bisulfite reaction by ~13-fold, resulting in reduced DNA damage and lower background noise. UBS-seq allows library construction from small amounts of purified genomic DNA, such as from cell-free DNA or directly from 1 to 100 mouse embryonic stem cells, with less overestimation of 5mC level and higher genome coverage than conventional BS-seq. Additionally, UBS-seq quantitatively maps RNA 5-methylcytosine (m5C) from low inputs of mRNA and allows the detection of m5C stoichiometry in highly structured RNA sequences. Our UBS-seq results identify NSUN2 as the major ‘writer’ protein responsible for the deposition of ~90% of m5C sites in HeLa mRNA and reveal enriched m5C sites in 5′-regions of mammalian mRNA, which may have functional roles in mRNA translation regulation. Accelerated bisulfite sequencing detects 5-methylcytosine in low-input DNA and RNA samples},
  archive      = {J_NBT},
  author       = {Dai, Qing and Ye, Chang and Irkliyenko, Iryna and Wang, Yiding and Sun, Hui-Lung and Gao, Yun and Liu, Yushuai and Beadell, Alana and Perea, José and Goel, Ajay and He, Chuan},
  doi          = {10.1038/s41587-023-02034-w},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1559-1570},
  shortjournal = {Nature Biotech.},
  title        = {Ultrafast bisulfite sequencing detection of 5-methylcytosine in DNA and RNA},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A palmitoyl transferase chemical–genetic system to map
ZDHHC-specific s-acylation. <em>NBT</em>, <em>42</em>(10), 1548–1558.
(<a href="https://doi.org/10.1038/s41587-023-02030-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The 23 human zinc finger Asp-His-His-Cys motif-containing (ZDHHC) S-acyltransferases catalyze long-chain S-acylation at cysteine residues across an extensive network of hundreds of proteins important for normal physiology or dysregulated in disease. Here we present a technology to directly map the protein substrates of a specific ZDHHC at the whole-proteome level, in intact cells. Structure-guided engineering of paired ZDHHC ‘hole’ mutants and ‘bumped’ chemically tagged fatty acid probes enabled probe transfer to specific protein substrates with excellent selectivity over wild-type ZDHHCs. Chemical–genetic systems were exemplified for five human ZDHHCs (3, 7, 11, 15 and 20) and applied to generate de novo ZDHHC substrate profiles, identifying &amp;gt;300 substrates and S-acylation sites for new functionally diverse proteins across multiple cell lines. We expect that this platform will elucidate S-acylation biology for a wide range of models and organisms. The substrates of specific S-acyltransferases are mapped proteome-wide in live cells.},
  archive      = {J_NBT},
  author       = {Ocasio, Cory A. and Baggelaar, Marc P. and Sipthorp, James and Losada de la Lastra, Ana and Tavares, Manuel and Volarić, Jana and Soudy, Christelle and Storck, Elisabeth M. and Houghton, Jack W. and Palma-Duran, Susana A. and MacRae, James I. and Tomić, Goran and Carr, Lotte and Downward, Julian and Eggert, Ulrike S. and Tate, Edward W.},
  doi          = {10.1038/s41587-023-02030-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1548-1558},
  shortjournal = {Nature Biotech.},
  title        = {A palmitoyl transferase chemical–genetic system to map ZDHHC-specific S-acylation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Glycosylase-based base editors for efficient t-to-g and
c-to-g editing in mammalian cells. <em>NBT</em>, <em>42</em>(10),
1538–1547. (<a
href="https://doi.org/10.1038/s41587-023-02050-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Base editors show promise for treating human genetic diseases, but most current systems use deaminases, which cause off-target effects and are limited in editing type. In this study, we constructed deaminase-free base editors for cytosine (DAF-CBE) and thymine (DAF-TBE), which contain only a cytosine-DNA or a thymine-DNA glycosylase (CDG/TDG) variant, respectively, tethered to a Cas9 nickase. Multiple rounds of mutagenesis by directed evolution in Escherichia coli generated two variants with enhanced base-converting activity—CDG-nCas9 and TDG-nCas9—with efficiencies of up to 58.7% for C-to-A and 54.3% for T-to-A. DAF-BEs achieve C-to-G/T-to-G editing in mammalian cells with minimal Cas9-dependent and Cas9-independent off-target effects as well as minimal RNA off-target effects. Additional engineering resulted in DAF-CBE2/DAF-TBE2, which exhibit altered editing windows from the 5′ end to the middle of the protospacer and increased C-to-G/T-to-G editing efficiency of 3.5-fold and 1.2-fold, respectively. Compared to prime editing or CGBEs, DAF-BEs expand conversion types of base editors with similar efficiencies, smaller sizes and lower off-target effects. Base editors lacking a deaminase improve thymine and cytosine conversion.},
  archive      = {J_NBT},
  author       = {Ye, Lijun and Zhao, Dongdong and Li, Ju and Wang, Yiran and Li, Bo and Yang, Yuanzhao and Hou, Xueting and Wang, Huibin and Wei, Zhandong and Liu, Xiaoqi and Li, Yaqiu and Li, Siwei and Liu, Yajing and Zhang, Xueli and Bi, Changhao},
  doi          = {10.1038/s41587-023-02050-w},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1538-1547},
  shortjournal = {Nature Biotech.},
  title        = {Glycosylase-based base editors for efficient T-to-G and C-to-G editing in mammalian cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Engineered virus-like particles for transient delivery of
prime editor ribonucleoprotein complexes in vivo. <em>NBT</em>,
<em>42</em>(10), 1526–1537. (<a
href="https://doi.org/10.1038/s41587-023-02078-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing enables precise installation of genomic substitutions, insertions and deletions in living systems. Efficient in vitro and in vivo delivery of prime editing components, however, remains a challenge. Here we report prime editor engineered virus-like particles (PE-eVLPs) that deliver prime editor proteins, prime editing guide RNAs and nicking single guide RNAs as transient ribonucleoprotein complexes. We systematically engineered v3 and v3b PE-eVLPs with 65- to 170-fold higher editing efficiency in human cells compared to a PE-eVLP construct based on our previously reported base editor eVLP architecture. In two mouse models of genetic blindness, single injections of v3 PE-eVLPs resulted in therapeutically relevant levels of prime editing in the retina, protein expression restoration and partial visual function rescue. Optimized PE-eVLPs support transient in vivo delivery of prime editor ribonucleoproteins, enhancing the potential safety of prime editing by reducing off-target editing and obviating the possibility of oncogenic transgene integration. Delivery of prime editors in vivo is improved using virus-like particles.},
  archive      = {J_NBT},
  author       = {An, Meirui and Raguram, Aditya and Du, Samuel W. and Banskota, Samagya and Davis, Jessie R. and Newby, Gregory A. and Chen, Paul Z. and Palczewski, Krzysztof and Liu, David R.},
  doi          = {10.1038/s41587-023-02078-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1526-1537},
  shortjournal = {Nature Biotech.},
  title        = {Engineered virus-like particles for transient delivery of prime editor ribonucleoprotein complexes in vivo},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Combined small-molecule treatment accelerates maturation of
human pluripotent stem cell-derived neurons. <em>NBT</em>,
<em>42</em>(10), 1515–1525. (<a
href="https://doi.org/10.1038/s41587-023-02031-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The maturation of human pluripotent stem cell (hPSC)-derived neurons mimics the protracted timing of human brain development, extending over months to years for reaching adult-like function. Prolonged in vitro maturation presents a major challenge to stem cell-based applications in modeling and treating neurological disease. Therefore, we designed a high-content imaging assay based on morphological and functional readouts in hPSC-derived cortical neurons which identified multiple compounds that drive neuronal maturation including inhibitors of lysine-specific demethylase 1 and disruptor of telomerase-like 1 and activators of calcium-dependent transcription. A cocktail of four factors, GSK2879552, EPZ-5676, N-methyl-d-aspartate and Bay K 8644, collectively termed GENtoniK, triggered maturation across all parameters tested, including synaptic density, electrophysiology and transcriptomics. Maturation effects were further validated in cortical organoids, spinal motoneurons and non-neural lineages including melanocytes and pancreatic β-cells. The effects on maturation observed across a broad range of hPSC-derived cell types indicate that some of the mechanisms controlling the timing of human maturation might be shared across lineages. Neurons derived from stem cells are rapidly matured with a GENtoniK cocktail.},
  archive      = {J_NBT},
  author       = {Hergenreder, Emiliano and Minotti, Andrew P. and Zorina, Yana and Oberst, Polina and Zhao, Zeping and Munguba, Hermany and Calder, Elizabeth L. and Baggiolini, Arianna and Walsh, Ryan M. and Liston, Conor and Levitz, Joshua and Garippa, Ralph and Chen, Shuibing and Ciceri, Gabriele and Studer, Lorenz},
  doi          = {10.1038/s41587-023-02031-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1515-1525},
  shortjournal = {Nature Biotech.},
  title        = {Combined small-molecule treatment accelerates maturation of human pluripotent stem cell-derived neurons},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Encapsulated stem cell–derived β cells exert glucose control
in patients with type 1 diabetes. <em>NBT</em>, <em>42</em>(10),
1507–1514. (<a
href="https://doi.org/10.1038/s41587-023-02055-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clinical studies on the treatment of type 1 diabetes with device-encapsulated pancreatic precursor cells derived from human embryonic stem cells found that insulin output was insufficient for clinical benefit. We are conducting a phase 1/2, open-label, multicenter trial aimed at optimizing cell engraftment (ClinicalTrials.gov identifier: NCT03163511 ). Here we report interim, 1-year outcomes in one study group that received 2–3-fold higher cell doses in devices with an optimized membrane perforation pattern. β cell function was measured by meal-stimulated plasma C-peptide levels at 3-month intervals, and the effect on glucose control was assessed by continuous glucose monitoring (CGM) and insulin dosing. Of 10 patients with undetectable baseline C-peptide, three achieved levels ≥0.1 nmol l−1 from month 6 onwards that correlated with improved CGM measures and reduced insulin dosing, indicating a glucose-controlling effect. The patient with the highest C-peptide (0.23 nmol l−1) increased CGM time-in-range from 55% to 85% at month 12; β cell mass in sentinel devices in this patient at month 6 was 4% of the initial cell mass, indicating directions for improving efficacy. β cells derived from stem cells improve blood glucose control in patients with diabetes.},
  archive      = {J_NBT},
  author       = {Keymeulen, Bart and De Groot, Kaat and Jacobs-Tulleneers-Thevissen, Daniel and Thompson, David M. and Bellin, Melena D. and Kroon, Evert J. and Daniels, Mark and Wang, Richard and Jaiman, Manasi and Kieffer, Timothy J. and Foyt, Howard L. and Pipeleers, Daniel},
  doi          = {10.1038/s41587-023-02055-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1507-1514},
  shortjournal = {Nature Biotech.},
  title        = {Encapsulated stem cell–derived β cells exert glucose control in patients with type 1 diabetes},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). TRGT-ing the dark genome to accurately characterize tandem
repeats at scale. <em>NBT</em>, <em>42</em>(10), 1504–1505. (<a
href="https://doi.org/10.1038/s41587-023-02073-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Repetitive DNA sequences known as tandem repeats (TRs) are linked to dozens of monogenic diseases and to cancer. We developed a computational method to characterize TRs using PacBio HiFi sequencing. This software can be used to discover and profile pathogenic repeat expansions and to catalog genetic and epigenetic variation in TR regions.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02073-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1504-1505},
  shortjournal = {Nature Biotech.},
  title        = {TRGT-ing the dark genome to accurately characterize tandem repeats at scale},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Identification of a chemical cocktail that drives the
maturation of human neurons. <em>NBT</em>, <em>42</em>(10), 1502–1503.
(<a href="https://doi.org/10.1038/s41587-023-02044-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The protracted timeline of maturation is a major bottleneck in generating adult-like neurons from human pluripotent stem cells. We identify a combination of four chemicals that promotes neuronal maturation by repressing epigenetic factors that inhibit it and stimulating activity-dependent factors that promote it.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02044-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1502-1503},
  shortjournal = {Nature Biotech.},
  title        = {Identification of a chemical cocktail that drives the maturation of human neurons},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). From bench to podcast: Chloe fishman. <em>NBT</em>,
<em>42</em>(10), 1501. (<a
href="https://doi.org/10.1038/s41587-024-02433-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02433-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1501},
  shortjournal = {Nature Biotech.},
  title        = {From bench to podcast: Chloe fishman},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). A strategy to load, rethread and read protein sequences
through a nanopore. <em>NBT</em>, <em>42</em>(10), 1499. (<a
href="https://doi.org/10.1038/s41587-024-02443-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02443-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1499},
  shortjournal = {Nature Biotech.},
  title        = {A strategy to load, rethread and read protein sequences through a nanopore},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024k). Cellular shuttles put mislocalized proteins in their right
place. <em>NBT</em>, <em>42</em>(10), 1499. (<a
href="https://doi.org/10.1038/s41587-024-02444-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02444-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1499},
  shortjournal = {Nature Biotech.},
  title        = {Cellular shuttles put mislocalized proteins in their right place},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024ad). Pan-KRAS destruction with a single PROTAC. <em>NBT</em>,
<em>42</em>(10), 1499. (<a
href="https://doi.org/10.1038/s41587-024-02445-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02445-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1499},
  shortjournal = {Nature Biotech.},
  title        = {Pan-KRAS destruction with a single PROTAC},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). T cells. <em>NBT</em>, <em>42</em>(10), 1498. (<a
href="https://doi.org/10.1038/s41587-024-02415-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the manufacture, preparation and use of T cells in treating cancer.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02415-9},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1498},
  shortjournal = {Nature Biotech.},
  title        = {T cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The global patent landscape of functional food innovation.
<em>NBT</em>, <em>42</em>(10), 1493–1497. (<a
href="https://doi.org/10.1038/s41587-024-02410-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of patent documents reveals a trend of increasing interest in functional food innovations that may aid future decision-making in research, business and policymaking.},
  archive      = {J_NBT},
  author       = {Matin, Maima and Hrg, Dalibor and Litvinova, Olena and Łysek-Gładysinska, Małgorzata and Wierzbicka, Agnieszka and Horbańczuk, Jarosław Olav and Jóźwik, Artur and Atanasov, Atanas G.},
  doi          = {10.1038/s41587-024-02410-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1493-1497},
  shortjournal = {Nature Biotech.},
  title        = {The global patent landscape of functional food innovation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The european innovation council supports innovative
portfolios in health biotech. <em>NBT</em>, <em>42</em>(10), 1488–1492.
(<a href="https://doi.org/10.1038/s41587-024-02416-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The European Innovation Council aims to catalyze the formation and growth of science-based businesses in Europe through highly targeted initiatives, actively managed innovative portfolios and a stepped funding pathway of grants and equity investment.},
  archive      = {J_NBT},
  author       = {Arzimanoglou, Iordanis},
  doi          = {10.1038/s41587-024-02416-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1488-1492},
  shortjournal = {Nature Biotech.},
  title        = {The european innovation council supports innovative portfolios in health biotech},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024r). Editor’s pick: Switch therapeutics. <em>NBT</em>,
<em>42</em>(10), 1486–1487. (<a
href="https://doi.org/10.1038/s41587-024-02423-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Switch Therapeutics is rethinking cell-type specificity, building precision RNAi therapeutics for the brain.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02423-9},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1486-1487},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Switch therapeutics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(10), 1480. (<a
href="https://doi.org/10.1038/s41587-024-02436-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02436-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1480},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Niemann–pick double win. <em>NBT</em>, <em>42</em>(10),
1479. (<a href="https://doi.org/10.1038/s41587-024-02450-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02450-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1479},
  shortjournal = {Nature Biotech.},
  title        = {Niemann–Pick double win},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bispecific upstages keytruda. <em>NBT</em>, <em>42</em>(10),
1478. (<a href="https://doi.org/10.1038/s41587-024-02451-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02451-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1478},
  shortjournal = {Nature Biotech.},
  title        = {Bispecific upstages keytruda},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). TCR cell therapies vanquish solid tumors — finally.
<em>NBT</em>, <em>42</em>(10), 1477–1479. (<a
href="https://doi.org/10.1038/s41587-024-02435-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The FDA’s approval of the first genetically modified T cell therapy for treating a rare sarcoma is paving the way for next-generation therapies that tackle other types of solid tumors.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-024-02435-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1477-1479},
  shortjournal = {Nature Biotech.},
  title        = {TCR cell therapies vanquish solid tumors — Finally},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). AI meets real-world patients. <em>NBT</em>, <em>42</em>(10),
1475. (<a href="https://doi.org/10.1038/s41587-024-02455-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The time has come for biotech companies to embrace machine learning for clinical trials, but they should start with compiling the data. That means grappling with real-world patient records.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02455-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1475},
  shortjournal = {Nature Biotech.},
  title        = {AI meets real-world patients},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Plastic-eating bacteria boost growing business of
bioremediation. <em>NBT</em>, <em>42</em>(10), 1481–1485. (<a
href="https://doi.org/10.1038/s41587-024-02401-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacteria, fungi and plants can be grown and engineered to remove plastics, chemicals and pollutants from contaminated soil and water.},
  archive      = {J_NBT},
  author       = {Johnson, Ben},
  doi          = {10.1038/s41587-024-02401-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {10},
  pages        = {1481-1485},
  shortjournal = {Nature Biotech.},
  title        = {Plastic-eating bacteria boost growing business of bioremediation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). People. <em>NBT</em>, <em>42</em>(9), 1474. (<a
href="https://doi.org/10.1038/s41587-024-02379-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02379-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1474},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). 1H24 biotech job picture. <em>NBT</em>, <em>42</em>(9),
1473. (<a href="https://doi.org/10.1038/s41587-024-02378-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A semiannual snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02378-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1473},
  shortjournal = {Nature Biotech.},
  title        = {1H24 biotech job picture},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A virtual case competition approach to bridging business
acumen and biomedical graduate training. <em>NBT</em>, <em>42</em>(9),
1468–1472. (<a
href="https://doi.org/10.1038/s41587-024-02376-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Midwest Healthcare Case Competition lets graduate trainees gain practical experience and exposure to alternative careers in a condensed time frame without taking away from research commitments.},
  archive      = {J_NBT},
  author       = {Donovan, Kara M. and Goldsmith, Robert S. and Ely, Erica V. and Garg, Samridhi and Ruiz, Lloyd P. and Athaiya, Mitre and Meyer, Sophia and Smith, Michelle Sydney and Mawla, Ishtiaq},
  doi          = {10.1038/s41587-024-02376-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1468-1472},
  shortjournal = {Nature Biotech.},
  title        = {A virtual case competition approach to bridging business acumen and biomedical graduate training},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Large-scale evaluation of the ability of
RNA-binding proteins to activate exon inclusion. <em>NBT</em>,
<em>42</em>(9), 1467. (<a
href="https://doi.org/10.1038/s41587-024-02178-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Schmok, Jonathan C. and Jain, Manya and Street, Lena A. and Tankka, Alex T. and Schafer, Danielle and Her, Hsuan-Lin and Elmsaouri, Sara and Gosztyla, Maya L. and Boyle, Evan A. and Jagannatha, Pratibha and Luo, En-Ching and Kwon, Ester J. and Jovanovic, Marko and Yeo, Gene W.},
  doi          = {10.1038/s41587-024-02178-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1467},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Large-scale evaluation of the ability of RNA-binding proteins to activate exon inclusion},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Control of mammalian gene expression by modulation of polyA
signal cleavage at 5′ UTR. <em>NBT</em>, <em>42</em>(9), 1454–1466. (<a
href="https://doi.org/10.1038/s41587-023-01989-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to control gene expression in mammalian cells is crucial for safe and efficacious gene therapies and for elucidating gene functions. Current gene regulation systems have limitations such as harmful immune responses or low efficiency. We describe the pA regulator, an RNA-based switch that controls mammalian gene expression through modulation of a synthetic polyA signal (PAS) cleavage introduced into the 5′ UTR of a transgene. The cleavage is modulated by a ‘dual-mechanism’—(1) aptamer clamping to inhibit PAS cleavage and (2) drug-induced alternative splicing that removes the PAS, both activated by drug binding. This RNA-based methodology circumvents the immune responses observed in other systems and achieves a 900-fold induction with an EC50 of 0.5 µg ml−1 tetracycline (Tc), which is well within the FDA-approved dose range. The pA regulator effectively controls the luciferase transgene in live mice and the endogenous CD133 gene in human cells, in a dose-dependent and reversible manner with long-term stability. Mammalian gene expression is controlled by an RNA switch system.},
  archive      = {J_NBT},
  author       = {Luo, Liming and Jea, Jocelyn Duen-Ya and Wang, Yan and Chao, Pei-Wen and Yen, Laising},
  doi          = {10.1038/s41587-023-01989-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1454-1466},
  shortjournal = {Nature Biotech.},
  title        = {Control of mammalian gene expression by modulation of polyA signal cleavage at 5′ UTR},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An adenine base editor variant expands context
compatibility. <em>NBT</em>, <em>42</em>(9), 1442–1453. (<a
href="https://doi.org/10.1038/s41587-023-01994-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adenine base editors (ABEs) are precise gene-editing agents that convert A:T pairs into G:C through a deoxyinosine intermediate. Existing ABEs function most effectively when the target A is in a TA context. Here we evolve the Escherichia coli transfer RNA-specific adenosine deaminase (TadA) to generate TadA8r, which extends potent deoxyadenosine deamination to RA (R = A or G) and is faster in processing GA than TadA8.20 and TadA8e, the two most active TadA variants reported so far. ABE8r, comprising TadA8r and a Streptococcus pyogenes Cas9 nickase, expands the editing window at the protospacer adjacent motif-distal end and outperforms ABE7.10, ABE8.20 and ABE8e in correcting disease-associated G:C-to-A:T transitions in the human genome, with a controlled off-target profile. We show ABE8r-mediated editing of clinically relevant sites that are poorly accessed by existing editors, including sites in PCSK9, whose disruption reduces low-density lipoprotein cholesterol, and ABCA4-p.Gly1961Glu, the most frequent mutation in Stargardt disease. An improved adenine base editor corrects clinically relevant mutations inaccessible to current editors.},
  archive      = {J_NBT},
  author       = {Xiao, Yu-Lan and Wu, Yuan and Tang, Weixin},
  doi          = {10.1038/s41587-023-01994-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1442-1453},
  shortjournal = {Nature Biotech.},
  title        = {An adenine base editor variant expands context compatibility},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Large-scale evaluation of the ability of RNA-binding
proteins to activate exon inclusion. <em>NBT</em>, <em>42</em>(9),
1429–1441. (<a
href="https://doi.org/10.1038/s41587-023-02014-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) modulate alternative splicing outcomes to determine isoform expression and cellular survival. To identify RBPs that directly drive alternative exon inclusion, we developed tethered function luciferase-based splicing reporters that provide rapid, scalable and robust readouts of exon inclusion changes and used these to evaluate 718 human RBPs. We performed enhanced cross-linking immunoprecipitation, RNA sequencing and affinity purification–mass spectrometry to investigate a subset of candidates with no prior association with splicing. Integrative analysis of these assays indicates surprising roles for TRNAU1AP, SCAF8 and RTCA in the modulation of hundreds of endogenous splicing events. We also leveraged our tethering assays and top candidates to identify potent and compact exon inclusion activation domains for splicing modulation applications. Using these identified domains, we engineered programmable fusion proteins that outperform current artificial splicing factors at manipulating inclusion of reporter and endogenous exons. This tethering approach characterizes the ability of RBPs to induce exon inclusion and yields new molecular parts for programmable splicing control. Alternative splicing is controlled with engineered fusion splicing factors.},
  archive      = {J_NBT},
  author       = {Schmok, Jonathan C. and Jain, Manya and Street, Lena A. and Tankka, Alex T. and Schafer, Danielle and Her, Hsuan-Lin and Elmsaouri, Sara and Gosztyla, Maya L. and Boyle, Evan A. and Jagannatha, Pratibha and Luo, En-Ching and Kwon, Ester J. and Jovanovic, Marko and Yeo, Gene W.},
  doi          = {10.1038/s41587-023-02014-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1429-1441},
  shortjournal = {Nature Biotech.},
  title        = {Large-scale evaluation of the ability of RNA-binding proteins to activate exon inclusion},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Programmable protein expression using a genetically encoded
m6A sensor. <em>NBT</em>, <em>42</em>(9), 1417–1428. (<a
href="https://doi.org/10.1038/s41587-023-01978-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The N6-methyladenosine (m6A) modification is found in thousands of cellular mRNAs and is a critical regulator of gene expression and cellular physiology. m6A dysregulation contributes to several human diseases, and the m6A methyltransferase machinery has emerged as a promising therapeutic target. However, current methods for studying m6A require RNA isolation and do not provide a real-time readout of mRNA methylation in living cells. Here we present a genetically encoded m6A sensor (GEMS) technology, which couples a fluorescent signal with cellular mRNA methylation. GEMS detects changes in m6A caused by pharmacological inhibition of the m6A methyltransferase, giving it potential utility for drug discovery efforts. Additionally, GEMS can be programmed to achieve m6A-dependent delivery of custom protein payloads in cells. Thus, GEMS is a versatile platform for m6A sensing that provides both a simple readout for m6A methylation and a system for m6A-coupled protein expression. m6A modifications are measured in living cells with a fluorescent reporter.},
  archive      = {J_NBT},
  author       = {Marayati, Bahjat F. and Thompson, Matthew G. and Holley, Christopher L. and Horner, Stacy M. and Meyer, Kate D.},
  doi          = {10.1038/s41587-023-01978-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1417-1428},
  shortjournal = {Nature Biotech.},
  title        = {Programmable protein expression using a genetically encoded m6A sensor},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Generation of locus coeruleus norepinephrine neurons from
human pluripotent stem cells. <em>NBT</em>, <em>42</em>(9), 1404–1416.
(<a href="https://doi.org/10.1038/s41587-023-01977-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Central norepinephrine (NE) neurons, located mainly in the locus coeruleus (LC), are implicated in diverse psychiatric and neurodegenerative diseases and are an emerging target for drug discovery. To facilitate their study, we developed a method to generate 40–60% human LC-NE neurons from human pluripotent stem cells. The approach depends on our identification of ACTIVIN A in regulating LC-NE transcription factors in dorsal rhombomere 1 (r1) progenitors. In vitro generated human LC-NE neurons display extensive axonal arborization; release and uptake NE; and exhibit pacemaker activity, calcium oscillation and chemoreceptor activity in response to CO2. Single-nucleus RNA sequencing (snRNA-seq) analysis at multiple timepoints confirmed NE cell identity and revealed the differentiation trajectory from hindbrain progenitors to NE neurons via an ASCL1-expressing precursor stage. LC-NE neurons engineered with an NE sensor reliably reported extracellular levels of NE. The availability of functional human LC-NE neurons enables investigation of their roles in psychiatric and neurodegenerative diseases and provides a tool for therapeutics development. Norepinephrine neurons, a drug target for neurologic diseases, are produced from stem cells.},
  archive      = {J_NBT},
  author       = {Tao, Yunlong and Li, Xueyan and Dong, Qiping and Kong, Linghai and Petersen, Andrew J. and Yan, Yuanwei and Xu, Ke and Zima, Seth and Li, Yanru and Schmidt, Danielle K. and Ayala, Melvin and Mathivanan, Sakthikumar and Sousa, Andre M. M. and Chang, Qiang and Zhang, Su-Chun},
  doi          = {10.1038/s41587-023-01977-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1404-1416},
  shortjournal = {Nature Biotech.},
  title        = {Generation of locus coeruleus norepinephrine neurons from human pluripotent stem cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial host–microbiome sequencing reveals niches in the
mouse gut. <em>NBT</em>, <em>42</em>(9), 1394–1403. (<a
href="https://doi.org/10.1038/s41587-023-01988-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mucosal and barrier tissues, such as the gut, lung or skin, are composed of a complex network of cells and microbes forming a tight niche that prevents pathogen colonization and supports host–microbiome symbiosis. Characterizing these networks at high molecular and cellular resolution is crucial for understanding homeostasis and disease. Here we present spatial host–microbiome sequencing (SHM-seq), an all-sequencing-based approach that captures tissue histology, polyadenylated RNAs and bacterial 16S sequences directly from a tissue by modifying spatially barcoded glass surfaces to enable simultaneous capture of host transcripts and hypervariable regions of the 16S bacterial ribosomal RNA. We applied our approach to the mouse gut as a model system, used a deep learning approach for data mapping and detected spatial niches defined by cellular composition and microbial geography. We show that subpopulations of gut cells express specific gene programs in different microenvironments characteristic of regional commensal bacteria and impact host–bacteria interactions. SHM-seq should enhance the study of native host–microbe interactions in health and disease. Spatial host–microbiome sequencing simultaneously profiles microbes and host transcriptomes from mouse colons.},
  archive      = {J_NBT},
  author       = {Lötstedt, Britta and Stražar, Martin and Xavier, Ramnik and Regev, Aviv and Vickovic, Sanja},
  doi          = {10.1038/s41587-023-01988-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1394-1403},
  shortjournal = {Nature Biotech.},
  title        = {Spatial host–microbiome sequencing reveals niches in the mouse gut},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial metatranscriptomics resolves host–bacteria–fungi
interactomes. <em>NBT</em>, <em>42</em>(9), 1384–1393. (<a
href="https://doi.org/10.1038/s41587-023-01979-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The interactions of microorganisms among themselves and with their multicellular host take place at the microscale, forming complex networks and spatial patterns. Existing technology does not allow the simultaneous investigation of spatial interactions between a host and the multitude of its colonizing microorganisms, which limits our understanding of host–microorganism interactions within a plant or animal tissue. Here we present spatial metatranscriptomics (SmT), a sequencing-based approach that leverages 16S/18S/ITS/poly-d(T) multimodal arrays for simultaneous host transcriptome- and microbiome-wide characterization of tissues at 55-µm resolution. We showcase SmT in outdoor-grown Arabidopsis thaliana leaves as a model system, and find tissue-scale bacterial and fungal hotspots. By network analysis, we study inter- and intrakingdom spatial interactions among microorganisms, as well as the host response to microbial hotspots. SmT provides an approach for answering fundamental questions on host–microbiome interplay. Interactions among plant hosts, bacteria and fungi are measured at high spatial resolution.},
  archive      = {J_NBT},
  author       = {Saarenpää, Sami and Shalev, Or and Ashkenazy, Haim and Carlos, Vanessa and Lundberg, Derek Severi and Weigel, Detlef and Giacomello, Stefania},
  doi          = {10.1038/s41587-023-01979-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1384-1393},
  shortjournal = {Nature Biotech.},
  title        = {Spatial metatranscriptomics resolves host–bacteria–fungi interactomes},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-quality metagenome assembly from long accurate reads
with metaMDBG. <em>NBT</em>, <em>42</em>(9), 1378–1383. (<a
href="https://doi.org/10.1038/s41587-023-01983-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce metaMDBG, a metagenomics assembler for PacBio HiFi reads. MetaMDBG combines a de Bruijn graph assembly in a minimizer space with an iterative assembly over sequences of minimizers to address variations in genome coverage depth and an abundance-based filtering strategy to simplify strain complexity. For complex communities, we obtained up to twice as many high-quality circularized prokaryotic metagenome-assembled genomes as existing methods and had better recovery of viruses and plasmids. An assembler considering variable coverage depths increases the number of near-complete metagenome-assembled genomes.},
  archive      = {J_NBT},
  author       = {Benoit, Gaëtan and Raguideau, Sébastien and James, Robert and Phillippy, Adam M. and Chikhi, Rayan and Quince, Christopher},
  doi          = {10.1038/s41587-023-01983-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1378-1383},
  shortjournal = {Nature Biotech.},
  title        = {High-quality metagenome assembly from long accurate reads with metaMDBG},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inferring super-resolution tissue architecture by
integrating spatial transcriptomics with histology. <em>NBT</em>,
<em>42</em>(9), 1372–1377. (<a
href="https://doi.org/10.1038/s41587-023-02019-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics (ST) has demonstrated enormous potential for generating intricate molecular maps of cells within tissues. Here we present iStar, a method based on hierarchical image feature extraction that integrates ST data and high-resolution histology images to predict spatial gene expression with super-resolution. Our method enhances gene expression resolution to near-single-cell levels in ST and enables gene expression prediction in tissue sections where only histology images are available. iStar predicts gene expression with near-single-cell resolution from histology images.},
  archive      = {J_NBT},
  author       = {Zhang, Daiwei and Schroeder, Amelia and Yan, Hanying and Yang, Haochen and Hu, Jian and Lee, Michelle Y. Y. and Cho, Kyung S. and Susztak, Katalin and Xu, George X. and Feldman, Michael D. and Lee, Edward B. and Furth, Emma E. and Wang, Linghua and Li, Mingyao},
  doi          = {10.1038/s41587-023-02019-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1372-1377},
  shortjournal = {Nature Biotech.},
  title        = {Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A monomeric StayGold fluorescent protein. <em>NBT</em>,
<em>42</em>(9), 1368–1371. (<a
href="https://doi.org/10.1038/s41587-023-02018-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {StayGold is an exceptionally bright and stable fluorescent protein that is highly resistant to photobleaching. Despite favorable fluorescence properties, use of StayGold as a fluorescent tag is limited because it forms a natural dimer. Here we report the 1.6 Å structure of StayGold and generate a derivative, mStayGold, that retains the brightness and photostability of the original protein while being fully monomeric. The bright, photostable fluorescent protein StayGold is converted into a monomer.},
  archive      = {J_NBT},
  author       = {Ivorra-Molla, Esther and Akhuli, Dipayan and McAndrew, Martin B. L. and Scott, William and Kumar, Lokesh and Palani, Saravanan and Mishima, Masanori and Crow, Allister and Balasubramanian, Mohan K.},
  doi          = {10.1038/s41587-023-02018-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1368-1371},
  shortjournal = {Nature Biotech.},
  title        = {A monomeric StayGold fluorescent protein},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A step closer to safe and efficient mammalian gene
regulation. <em>NBT</em>, <em>42</em>(9), 1366–1367. (<a
href="https://doi.org/10.1038/s41587-023-02020-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {pA regulator is an RNA-based, non-immunogenic regulation system of gene expression that achieves up to 900-fold induction by using an inducer drug within the clinically safe dose range.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02020-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1366-1367},
  shortjournal = {Nature Biotech.},
  title        = {A step closer to safe and efficient mammalian gene regulation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Generation of programmable splicing factors using
RNA-binding proteins that activate exon inclusion. <em>NBT</em>,
<em>42</em>(9), 1364–1365. (<a
href="https://doi.org/10.1038/s41587-023-02032-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed luciferase-based splicing reporters to evaluate 718 RNA-binding proteins (RBPs) for the ability to activate exon inclusion. Our screen detected RBPs with no previously characterized roles in splicing and utilized RBP domains displaying potent exon activation to develop programmable splicing modulators with improved strength, generalizability and size over previous versions.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02032-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1364-1365},
  shortjournal = {Nature Biotech.},
  title        = {Generation of programmable splicing factors using RNA-binding proteins that activate exon inclusion},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). GEMS enables mRNA m6A methylation sensing in living cells
and m6A-coupled protein delivery. <em>NBT</em>, <em>42</em>(9),
1362–1363. (<a
href="https://doi.org/10.1038/s41587-023-01997-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thousands of cellular mRNAs contain the methyl modification m6A, which plays critical roles in mRNA processing and gene expression. We developed GEMS, a genetically encoded m6A sensor that provides a simple readout for m6A methylation in living cells and offers a platform for achieving m6A-coupled effector protein expression.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01997-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1362-1363},
  shortjournal = {Nature Biotech.},
  title        = {GEMS enables mRNA m6A methylation sensing in living cells and m6A-coupled protein delivery},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). From bench to podcast: Nicoletta cieri. <em>NBT</em>,
<em>42</em>(9), 1361. (<a
href="https://doi.org/10.1038/s41587-024-02402-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02402-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1361},
  shortjournal = {Nature Biotech.},
  title        = {From bench to podcast: Nicoletta cieri},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial methods for microbiome–host interactions.
<em>NBT</em>, <em>42</em>(9), 1359–1360. (<a
href="https://doi.org/10.1038/s41587-023-01996-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics offers a glimpse of how microbiomes interact with their hosts.},
  archive      = {J_NBT},
  author       = {Ntekas, Ioannis and De Vlaminck, Iwijn},
  doi          = {10.1038/s41587-023-01996-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1359-1360},
  shortjournal = {Nature Biotech.},
  title        = {Spatial methods for microbiome–host interactions},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024y). Intestinal organoids with an autologous immune compartment.
<em>NBT</em>, <em>42</em>(9), 1357. (<a
href="https://doi.org/10.1038/s41587-024-02389-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02389-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1357},
  shortjournal = {Nature Biotech.},
  title        = {Intestinal organoids with an autologous immune compartment},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024ah). Targeting fatty acid metabolism promotes a healthy vaginal
microbiome. <em>NBT</em>, <em>42</em>(9), 1357. (<a
href="https://doi.org/10.1038/s41587-024-02390-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02390-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1357},
  shortjournal = {Nature Biotech.},
  title        = {Targeting fatty acid metabolism promotes a healthy vaginal microbiome},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024s). Engineered parasite delivers therapeutic proteins to the
mouse brain. <em>NBT</em>, <em>42</em>(9), 1357. (<a
href="https://doi.org/10.1038/s41587-024-02399-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02399-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1357},
  shortjournal = {Nature Biotech.},
  title        = {Engineered parasite delivers therapeutic proteins to the mouse brain},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bioremediation. <em>NBT</em>, <em>42</em>(9), 1356. (<a
href="https://doi.org/10.1038/s41587-024-02374-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to compositions and methods for the remediation of polluted and contaminated environments.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02374-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1356},
  shortjournal = {Nature Biotech.},
  title        = {Bioremediation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The prevalence of post-NDA drug patents and their
relationship to the timing of generic approval. <em>NBT</em>,
<em>42</em>(9), 1350–1355. (<a
href="https://doi.org/10.1038/s41587-024-02371-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patent applications filed after a New Drug Application (NDA) is approved are uncommon and do not appear to substantially delay approval of generic drugs in most cases.},
  archive      = {J_NBT},
  author       = {Darrow, Jonathan J. and Van de Wiele, Victor and Brown, Beatrice and Kesselheim, Aaron S.},
  doi          = {10.1038/s41587-024-02371-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1350-1355},
  shortjournal = {Nature Biotech.},
  title        = {The prevalence of post-NDA drug patents and their relationship to the timing of generic approval},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The wild west of spike-in normalization. <em>NBT</em>,
<em>42</em>(9), 1343–1349. (<a
href="https://doi.org/10.1038/s41587-024-02377-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The proper use of spike-in normalization in ChIP-seq improves sensitivity for detecting genome-wide changes between conditions, but improper use is common, calling some biological conclusions into question. A survey of public datasets generates guidelines for implementation of spike-in normalization for future ChIP-seq experiments.},
  archive      = {J_NBT},
  author       = {Patel, Lauren A. and Cao, Yuwei and Mendenhall, Eric M. and Benner, Christopher and Goren, Alon},
  doi          = {10.1038/s41587-024-02377-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1343-1349},
  shortjournal = {Nature Biotech.},
  title        = {The wild west of spike-in normalization},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(9), 1330. (<a
href="https://doi.org/10.1038/s41587-024-02396-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02396-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1330},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Biogenetic blooms: It’s microbial, and it’s art.
<em>NBT</em>, <em>42</em>(9), 1329. (<a
href="https://doi.org/10.1038/s41587-024-02397-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Grushkin, Daniel},
  doi          = {10.1038/s41587-024-02397-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1329},
  shortjournal = {Nature Biotech.},
  title        = {Biogenetic blooms: It’s microbial, and it’s art},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genentech, sangamo ink alzheimer’s deal. <em>NBT</em>,
<em>42</em>(9), 1328. (<a
href="https://doi.org/10.1038/s41587-024-02413-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02413-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1328},
  shortjournal = {Nature Biotech.},
  title        = {Genentech, sangamo ink alzheimer’s deal},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). US selects first ten drugs for price drops. <em>NBT</em>,
<em>42</em>(9), 1327. (<a
href="https://doi.org/10.1038/s41587-024-02407-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02407-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1327},
  shortjournal = {Nature Biotech.},
  title        = {US selects first ten drugs for price drops},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Enzymes boost “rock weathering” to trap CO2 in soil.
<em>NBT</em>, <em>42</em>(9), 1326–1328. (<a
href="https://doi.org/10.1038/s41587-024-02380-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spreading powdered basalt on farmland traps CO2 from the atmosphere, a technology that is cheap, requires no huge steel reactors and, with a biotech boost, may even reduce fertilizer use.},
  archive      = {J_NBT},
  author       = {Peplow, Mark},
  doi          = {10.1038/s41587-024-02380-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1326-1328},
  shortjournal = {Nature Biotech.},
  title        = {Enzymes boost ‘rock weathering’ to trap CO2 in soil},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). FDA approves first IDH-targeted glioma drug. <em>NBT</em>,
<em>42</em>(9), 1325. (<a
href="https://doi.org/10.1038/s41587-024-02408-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02408-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1325},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves first IDH-targeted glioma drug},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Foundation models build on ChatGPT tech to learn the
fundamental language of biology. <em>NBT</em>, <em>42</em>(9),
1323–1325. (<a
href="https://doi.org/10.1038/s41587-024-02400-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists are using ever more sophisticated AI algorithms trained on vast, unlabeled datasets to develop models that can ‘interpret’ biological data to help guide biomolecule design.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-024-02400-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1323-1325},
  shortjournal = {Nature Biotech.},
  title        = {Foundation models build on ChatGPT tech to learn the fundamental language of biology},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pandemic preparedness demands proper prioritization.
<em>NBT</em>, <em>42</em>(9), 1321–1322. (<a
href="https://doi.org/10.1038/s41587-024-02406-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have learned many lessons from COVID-19, but how to get vaccines to low- and middle-income countries effectively isn’t one of them.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02406-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1321-1322},
  shortjournal = {Nature Biotech.},
  title        = {Pandemic preparedness demands proper prioritization},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Editor’s pick: Atomic AI. <em>NBT</em>, <em>42</em>(9),
1341–1342. (<a
href="https://doi.org/10.1038/s41587-024-02375-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Atomic AI generates the structure data it needs for its AI model with the aim of RNA drug discovery.},
  archive      = {J_NBT},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41587-024-02375-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {9},
  pages        = {1341-1342},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Atomic AI},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Editor’s pick: Cargo therapeutics. <em>NBT</em>,
<em>42</em>(9), 1339–1340. (<a
href="https://doi.org/10.1038/s41587-024-02358-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. A rescued chimeric antigen receptor from Cargo Therapeutics for hard-to-treat lymphomas shows promise in patients while a trispecific advances toward human testing.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-024-02358-1},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {9},
  pages        = {1339-1340},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Cargo therapeutics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Biotech financing: Darkest before the dawn. <em>NBT</em>,
<em>42</em>(9), 1331–1338. (<a
href="https://doi.org/10.1038/s41587-024-02357-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After a difficult three years, biotech financing may slowly be returning to health.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-024-02357-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {9},
  pages        = {1331-1338},
  shortjournal = {Nature Biotech.},
  title        = {Biotech financing: Darkest before the dawn},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). People. <em>NBT</em>, <em>42</em>(8), 1320. (<a
href="https://doi.org/10.1038/s41587-024-02352-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02352-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1320},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The rise of teamwork and career prospects in academic
science. <em>NBT</em>, <em>42</em>(8), 1314–1319. (<a
href="https://doi.org/10.1038/s41587-024-02351-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rise in team size in academic science has generated an unintended side effect: junior scientists are less likely to secure research funding or obtain tenure and are more likely to leave academia.},
  archive      = {J_NBT},
  author       = {Andalón, Mabel and de Fontenay, Catherine and Ginther, Donna K. and Lim, Kwanghui},
  doi          = {10.1038/s41587-024-02351-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1314-1319},
  shortjournal = {Nature Biotech.},
  title        = {The rise of teamwork and career prospects in academic science},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Quantifying bias introduced by sample
collection in relative and absolute microbiome measurements.
<em>NBT</em>, <em>42</em>(8), 1313. (<a
href="https://doi.org/10.1038/s41587-024-02341-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Maghini, Dylan G. and Dvorak, Mai and Dahlen, Alex and Roos, Morgan and Doyle, Boryana and Kuersten, Scott and Bhatt, Ami S.},
  doi          = {10.1038/s41587-024-02341-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1313},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Quantifying bias introduced by sample collection in relative and absolute microbiome measurements},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Identification of mobile genetic elements with geNomad.
<em>NBT</em>, <em>42</em>(8), 1303–1312. (<a
href="https://doi.org/10.1038/s41587-023-01953-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying and characterizing mobile genetic elements in sequencing data is essential for understanding their diversity, ecology, biotechnological applications and impact on public health. Here we introduce geNomad, a classification and annotation framework that combines information from gene content and a deep neural network to identify sequences of plasmids and viruses. geNomad uses a dataset of more than 200,000 marker protein profiles to provide functional gene annotation and taxonomic assignment of viral genomes. Using a conditional random field model, geNomad also detects proviruses integrated into host genomes with high precision. In benchmarks, geNomad achieved high classification performance for diverse plasmids and viruses (Matthews correlation coefficient of 77.8% and 95.3%, respectively), substantially outperforming other tools. Leveraging geNomad’s speed and scalability, we processed over 2.7 trillion base pairs of sequencing data, leading to the discovery of millions of viruses and plasmids that are available through the IMG/VR and IMG/PR databases. geNomad is available at https://portal.nersc.gov/genomad . geNomad identifies mobile genetic elements in sequencing data.},
  archive      = {J_NBT},
  author       = {Camargo, Antonio Pedro and Roux, Simon and Schulz, Frederik and Babinski, Michal and Xu, Yan and Hu, Bin and Chain, Patrick S. G. and Nayfach, Stephen and Kyrpides, Nikos C.},
  doi          = {10.1038/s41587-023-01953-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1303-1312},
  shortjournal = {Nature Biotech.},
  title        = {Identification of mobile genetic elements with geNomad},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Detection of transcriptome-wide microRNA–target interactions
in single cells with agoTRIBE. <em>NBT</em>, <em>42</em>(8), 1296–1302.
(<a href="https://doi.org/10.1038/s41587-023-01951-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MicroRNAs (miRNAs) exert their gene regulatory effects on numerous biological processes based on their selection of target transcripts. Current experimental methods available to identify miRNA targets are laborious and require millions of cells. Here we have overcome these limitations by fusing the miRNA effector protein Argonaute2 to the RNA editing domain of ADAR2, allowing the detection of miRNA targets transcriptome-wide in single cells. miRNAs guide the fusion protein to their natural target transcripts, causing them to undergo A&amp;gt;I editing, which can be detected by sensitive single-cell RNA sequencing. We show that agoTRIBE identifies functional miRNA targets, which are supported by evolutionary sequence conservation. In one application of the method we study microRNA interactions in single cells and identify substantial differential targeting across the cell cycle. AgoTRIBE also provides transcriptome-wide measurements of RNA abundance and allows the deconvolution of miRNA targeting in complex tissues at the single-cell level. Functional miRNA targets are identified transcriptome-wide in single cells.},
  archive      = {J_NBT},
  author       = {Sekar, Vaishnovi and Mármol-Sánchez, Emilio and Kalogeropoulos, Panagiotis and Stanicek, Laura and Sagredo, Eduardo A. and Widmark, Albin and Doukoumopoulos, Evangelos and Bonath, Franziska and Biryukova, Inna and Friedländer, Marc R.},
  doi          = {10.1038/s41587-023-01951-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1296-1302},
  shortjournal = {Nature Biotech.},
  title        = {Detection of transcriptome-wide microRNA–target interactions in single cells with agoTRIBE},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Scalable genetic screening for regulatory circuits using
compressed perturb-seq. <em>NBT</em>, <em>42</em>(8), 1282–1295. (<a
href="https://doi.org/10.1038/s41587-023-01964-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pooled CRISPR screens with single-cell RNA sequencing readout (Perturb-seq) have emerged as a key technique in functional genomics, but they are limited in scale by cost and combinatorial complexity. In this study, we modified the design of Perturb-seq by incorporating algorithms applied to random, low-dimensional observations. Compressed Perturb-seq measures multiple random perturbations per cell or multiple cells per droplet and computationally decompresses these measurements by leveraging the sparse structure of regulatory circuits. Applied to 598 genes in the immune response to bacterial lipopolysaccharide, compressed Perturb-seq achieves the same accuracy as conventional Perturb-seq with an order of magnitude cost reduction and greater power to learn genetic interactions. We identified known and novel regulators of immune responses and uncovered evolutionarily constrained genes with downstream targets enriched for immune disease heritability, including many missed by existing genome-wide association studies. Our framework enables new scales of interrogation for a foundational method in functional genomics. Compressed Perturb-seq incorporates compressed sensing to genetic screening for scalable discovery of genetic interactions.},
  archive      = {J_NBT},
  author       = {Yao, Douglas and Binan, Loic and Bezney, Jon and Simonton, Brooke and Freedman, Jahanara and Frangieh, Chris J. and Dey, Kushal and Geiger-Schuller, Kathryn and Eraslan, Basak and Gusev, Alexander and Regev, Aviv and Cleary, Brian},
  doi          = {10.1038/s41587-023-01964-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1282-1295},
  shortjournal = {Nature Biotech.},
  title        = {Scalable genetic screening for regulatory circuits using compressed perturb-seq},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Translocation of linearized full-length proteins through an
engineered nanopore under opposing electrophoretic force. <em>NBT</em>,
<em>42</em>(8), 1275–1281. (<a
href="https://doi.org/10.1038/s41587-023-01954-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanopores have recently been used to identify and fingerprint proteins. However, because proteins, unlike DNA, do not have a uniform charge, the electrophoretic force cannot in general be used to translocate or linearize them. Here we show that the introduction of sets of charges in the lumen of the CytK nanopore spaced by ~1 nm creates an electroosmotic flow that induces the unidirectional transport of unstructured natural polypeptides against a strong electrophoretic force. Molecular dynamics simulations indicate that this electroosmotic-dominated force has a strength of ~20 pN at −100 mV, which is similar to the electric force on single-stranded DNA. Unfolded polypeptides produce current signatures as they traverse the nanopore, which may be used to identify proteins. This approach can be used to translocate and stretch proteins for enzymatic and non-enzymatic protein identification and sequencing. An engineered nanopore translocates untagged full-length proteins by electroosmotic force.},
  archive      = {J_NBT},
  author       = {Sauciuc, Adina and Morozzo della Rocca, Blasco and Tadema, Matthijs Jonathan and Chinappi, Mauro and Maglia, Giovanni},
  doi          = {10.1038/s41587-023-01954-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1275-1281},
  shortjournal = {Nature Biotech.},
  title        = {Translocation of linearized full-length proteins through an engineered nanopore under opposing electrophoretic force},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Killing tumor-associated bacteria with a liposomal
antibiotic generates neoantigens that induce anti-tumor immune
responses. <em>NBT</em>, <em>42</em>(8), 1263–1274. (<a
href="https://doi.org/10.1038/s41587-023-01957-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increasing evidence implicates the tumor microbiota as a factor that can influence cancer progression. In patients with colorectal cancer (CRC), we found that pre-resection antibiotics targeting anaerobic bacteria substantially improved disease-free survival by 25.5%. For mouse studies, we designed an antibiotic silver-tinidazole complex encapsulated in liposomes (LipoAgTNZ) to eliminate tumor-associated bacteria in the primary tumor and liver metastases without causing gut microbiome dysbiosis. Mouse CRC models colonized by tumor-promoting bacteria (Fusobacterium nucleatum spp.) or probiotics (Escherichia coli Nissle spp.) responded to LipoAgTNZ therapy, which enabled more than 70% long-term survival in two F. nucleatum-infected CRC models. The antibiotic treatment generated microbial neoantigens that elicited anti-tumor CD8+ T cells. Heterologous and homologous bacterial epitopes contributed to the immunogenicity, priming T cells to recognize both infected and uninfected tumors. Our strategy targets tumor-associated bacteria to elicit anti-tumoral immunity, paving the way for microbiome–immunotherapy interventions. Killing bacteria in tumors boosts survival in a mouse model of colon cancer.},
  archive      = {J_NBT},
  author       = {Wang, Menglin and Rousseau, Benoit and Qiu, Kunyu and Huang, Guannan and Zhang, Yu and Su, Hang and Le Bihan-Benjamin, Christine and Khati, Ines and Artz, Oliver and Foote, Michael B. and Cheng, Yung-Yi and Lee, Kuo-Hsiung and Miao, Michael Z. and Sun, Yue and Bousquet, Philippe-Jean and Hilmi, Marc and Dumas, Elise and Hamy, Anne-Sophie and Reyal, Fabien and Lin, Lin and Armistead, Paul M. and Song, Wantong and Vargason, Ava and Arthur, Janelle C. and Liu, Yun and Guo, Jianfeng and Zhou, Xuefei and Nguyen, Juliane and He, Yongqun and Ting, Jenny P.-Y. and Anselmo, Aaron C. and Huang, Leaf},
  doi          = {10.1038/s41587-023-01957-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1263-1274},
  shortjournal = {Nature Biotech.},
  title        = {Killing tumor-associated bacteria with a liposomal antibiotic generates neoantigens that induce anti-tumor immune responses},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Direct measurement of engineered cancer mutations and their
transcriptional phenotypes in single cells. <em>NBT</em>,
<em>42</em>(8), 1254–1262. (<a
href="https://doi.org/10.1038/s41587-023-01949-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome sequencing studies have identified numerous cancer mutations across a wide spectrum of tumor types, but determining the phenotypic consequence of these mutations remains a challenge. Here, we developed a high-throughput, multiplexed single-cell technology called TISCC-seq to engineer predesignated mutations in cells using CRISPR base editors, directly delineate their genotype among individual cells and determine each mutation’s transcriptional phenotype. Long-read sequencing of the target gene’s transcript identifies the engineered mutations, and the transcriptome profile from the same set of cells is simultaneously analyzed by short-read sequencing. Through integration, we determine the mutations’ genotype and expression phenotype at single-cell resolution. Using cell lines, we engineer and evaluate the impact of &amp;gt;100 TP53 mutations on gene expression. Based on the single-cell gene expression, we classify the mutations as having a functionally significant phenotype. Engineered cancer mutations are linked with phenotypes in a multiplexed single-cell technology.},
  archive      = {J_NBT},
  author       = {Kim, Heon Seok and Grimes, Susan M. and Chen, Tianqi and Sathe, Anuja and Lau, Billy T. and Hwang, Gue-Ho and Bae, Sangsu and Ji, Hanlee P.},
  doi          = {10.1038/s41587-023-01949-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1254-1262},
  shortjournal = {Nature Biotech.},
  title        = {Direct measurement of engineered cancer mutations and their transcriptional phenotypes in single cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Self-organizing models of human trunk organogenesis
recapitulate spinal cord and spine co-morphogenesis. <em>NBT</em>,
<em>42</em>(8), 1243–1253. (<a
href="https://doi.org/10.1038/s41587-023-01956-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrated in vitro models of human organogenesis are needed to elucidate the multi-systemic events underlying development and disease. Here we report the generation of human trunk-like structures that model the co-morphogenesis, patterning and differentiation of the human spine and spinal cord. We identified differentiation conditions for human pluripotent stem cells favoring the formation of an embryo-like extending antero-posterior (AP) axis. Single-cell and spatial transcriptomics show that somitic and spinal cord differentiation trajectories organize along this axis and can self-assemble into a neural tube surrounded by somites upon extracellular matrix addition. Morphogenesis is coupled with AP patterning mechanisms, which results, at later stages of organogenesis, in in vivo-like arrays of neural subtypes along a neural tube surrounded by spine and muscle progenitors contacted by neuronal projections. This integrated system of trunk development indicates that in vivo-like multi-tissue co-morphogenesis and topographic organization of terminal cell types can be achieved in human organoids, opening windows for the development of more complex models of organogenesis. A patterned neural tube and somites in an embryonic context are generated in vitro.},
  archive      = {J_NBT},
  author       = {Gribaudo, Simona and Robert, Rémi and van Sambeek, Björn and Mirdass, Camil and Lyubimova, Anna and Bouhali, Kamal and Ferent, Julien and Morin, Xavier and van Oudenaarden, Alexander and Nedelec, Stéphane},
  doi          = {10.1038/s41587-023-01956-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1243-1253},
  shortjournal = {Nature Biotech.},
  title        = {Self-organizing models of human trunk organogenesis recapitulate spinal cord and spine co-morphogenesis},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integrated vector genomes may contribute to long-term
expression in primate liver after AAV administration. <em>NBT</em>,
<em>42</em>(8), 1232–1242. (<a
href="https://doi.org/10.1038/s41587-023-01974-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of liver-based adeno-associated virus (AAV) gene therapies is facing concerns about limited efficiency and durability of transgene expression. We evaluated nonhuman primates following intravenous dosing of AAV8 and AAVrh10 vectors for over 2 years to better define the mechanism(s) of transduction that affect performance. High transduction of non-immunogenic transgenes was achieved, although expression declined over the first 90 days to reach a lower but stable steady state. More than 10% of hepatocytes contained single nuclear domains of vector DNA that persisted despite the loss of transgene expression. Greater reductions in vector DNA and RNA were observed with immunogenic transgenes. Genomic integration of vector sequences, including complex concatemeric structures, were detected in 1 out of 100 cells at broadly distributed loci that were not in proximity to genes associated with hepatocellular carcinoma. Our studies suggest that AAV-mediated transgene expression in primate hepatocytes occurs in two phases: high but short-lived expression from episomal genomes, followed by much lower but stable expression, likely from integrated vectors. Long-term expression of AAV transgenes in macaque liver is traced to integrated vectors.},
  archive      = {J_NBT},
  author       = {Greig, Jenny A. and Martins, Kelly M. and Breton, Camilo and Lamontagne, R. Jason and Zhu, Yanqing and He, Zhenning and White, John and Zhu, Jing-Xu and Chichester, Jessica A. and Zheng, Qi and Zhang, Zhe and Bell, Peter and Wang, Lili and Wilson, James M.},
  doi          = {10.1038/s41587-023-01974-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1232-1242},
  shortjournal = {Nature Biotech.},
  title        = {Integrated vector genomes may contribute to long-term expression in primate liver after AAV administration},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-affinity one-step aptamer selection using a non-fouling
porous hydrogel. <em>NBT</em>, <em>42</em>(8), 1224–1231. (<a
href="https://doi.org/10.1038/s41587-023-01973-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aptamers, commonly referred to as chemical antibodies, are used in a wide range of applications including drug delivery and biosensing. However, the process of aptamer selection poses a substantial challenge, as it requires numerous cycles of enrichment and involves issues with nonspecific binding. We present a simple, fast instrument-free method for aptamer enrichment and selection based on a diffusion-binding process in a three-dimensional non-fouling porous hydrogel with immobilized target proteins. Low-affinity aptamer candidates can be rapidly released from the hydrogel, whereas high-affinity candidates are restricted due to their strong binding to the immobilized protein targets. Consequently, a one-step enriched aptamer pool can strongly bind the protein targets. This enrichment is consistent across five proteins with isoelectric points in varying ranges. With thrombin as a representative model, the anti-thrombin aptamer identified from an enriched aptamer pool has been found to have a binding affinity that is comparable to those identified over ten cycles of selection using traditional methods. A non-fouling, porous hydrogel makes aptamer selection and enrichment simple and efficient.},
  archive      = {J_NBT},
  author       = {Singh, Naveen K. and Wang, Yixun and Wen, Connie and Davis, Brandon and Wang, Xuelin and Lee, Kyungsene and Wang, Yong},
  doi          = {10.1038/s41587-023-01973-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1224-1231},
  shortjournal = {Nature Biotech.},
  title        = {High-affinity one-step aptamer selection using a non-fouling porous hydrogel},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Dissecting key regulators of transcriptome kinetics through
scalable single-cell RNA profiling of pooled CRISPR screens.
<em>NBT</em>, <em>42</em>(8), 1218–1223. (<a
href="https://doi.org/10.1038/s41587-023-01948-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a combinatorial indexing method, PerturbSci-Kinetics, for capturing whole transcriptomes, nascent transcriptomes and single guide RNA (sgRNA) identities across hundreds of genetic perturbations at the single-cell level. Profiling a pooled CRISPR screen targeting various biological processes, we show the gene expression regulation during RNA synthesis, processing and degradation, miRNA biogenesis and mitochondrial mRNA processing, systematically decoding the genome-wide regulatory network that underlies RNA temporal dynamics at scale. mRNA kinetics are described by combining single-cell combinatorial indexing with metabolic labeling and pooled CRISPR screens.},
  archive      = {J_NBT},
  author       = {Xu, Zihan and Sziraki, Andras and Lee, Jasper and Zhou, Wei and Cao, Junyue},
  doi          = {10.1038/s41587-023-01948-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1218-1223},
  shortjournal = {Nature Biotech.},
  title        = {Dissecting key regulators of transcriptome kinetics through scalable single-cell RNA profiling of pooled CRISPR screens},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Epigenome editing technologies for discovery and medicine.
<em>NBT</em>, <em>42</em>(8), 1199–1217. (<a
href="https://doi.org/10.1038/s41587-024-02320-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenome editing has rapidly evolved in recent years, with diverse applications that include elucidating gene regulation mechanisms, annotating coding and noncoding genome functions and programming cell state and lineage specification. Importantly, given the ubiquitous role of epigenetics in complex phenotypes, epigenome editing has unique potential to impact a broad spectrum of diseases. By leveraging powerful DNA-targeting technologies, such as CRISPR, epigenome editing exploits the heritable and reversible mechanisms of epigenetics to alter gene expression without introducing DNA breaks, inducing DNA damage or relying on DNA repair pathways. Epigenome editing has versatile applications in biomedical research and disease treatment.},
  archive      = {J_NBT},
  author       = {McCutcheon, Sean R. and Rohm, Dahlia and Iglesias, Nahid and Gersbach, Charles A.},
  doi          = {10.1038/s41587-024-02320-1},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1199-1217},
  shortjournal = {Nature Biotech.},
  title        = {Epigenome editing technologies for discovery and medicine},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Fast and accurate identification of plasmids and viruses in
sequencing data using geNomad. <em>NBT</em>, <em>42</em>(8), 1196–1197.
(<a href="https://doi.org/10.1038/s41587-023-01982-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The detection of mobile genetic elements is crucial for exploring the ecology and evolution of microbial communities, and it has diverse implications in biotechnology and public health. geNomad is a computational framework that enables researchers to precisely identifiy and annotate plasmids and viruses in sequencing data on a large scale.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01982-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1196-1197},
  shortjournal = {Nature Biotech.},
  title        = {Fast and accurate identification of plasmids and viruses in sequencing data using geNomad},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Compressed perturb-seq enables highly efficient genetic
screens. <em>NBT</em>, <em>42</em>(8), 1194–1195. (<a
href="https://doi.org/10.1038/s41587-023-02003-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using an experimental and computational framework inspired by compressed sensing, we greatly reduced the number of measurements needed to run Perturb-seq. Our compressed Perturb-seq strategy relies on collecting measurements comprising random linear combinations of genetic perturbations, followed by deconvolving the perturbation effects on the transcriptome using sparsity-exploiting algorithms.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02003-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1194-1195},
  shortjournal = {Nature Biotech.},
  title        = {Compressed perturb-seq enables highly efficient genetic screens},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Electroosmotic flow across nanopores for single-molecule
protein sequencing. <em>NBT</em>, <em>42</em>(8), 1192–1193. (<a
href="https://doi.org/10.1038/s41587-023-01976-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By using fixed charges to engineer a strong electroosmotic flow, we achieve the unidirectional transport of natural polypeptides across nanopores. Our approach enables native proteins to be transported enzymatically and non-enzymatically in the absence of denaturant and electrophoretic tags, with potential applications for protein sequencing.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01976-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1192-1193},
  shortjournal = {Nature Biotech.},
  title        = {Electroosmotic flow across nanopores for single-molecule protein sequencing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Fast, instrument-free and low-cost selection of
high-affinity aptamers. <em>NBT</em>, <em>42</em>(8), 1190–1191. (<a
href="https://doi.org/10.1038/s41587-023-01980-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We designed a method for fast aptamer selection by integrating biomaterials science, engineering principles and biology. Aptamer candidates dynamically interacting with immobilized targets in a three-dimensional, non-fouling and macroporous polyethylene glycol hydrogel were rapidly enriched and selected with high affinity against five protein targets.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01980-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1190-1191},
  shortjournal = {Nature Biotech.},
  title        = {Fast, instrument-free and low-cost selection of high-affinity aptamers},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). From bench to podcast: Sean johnson. <em>NBT</em>,
<em>42</em>(8), 1189. (<a
href="https://doi.org/10.1038/s41587-024-02310-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02310-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1189},
  shortjournal = {Nature Biotech.},
  title        = {From bench to podcast: Sean johnson},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Antibiotic nanoparticles boost antitumor immunity.
<em>NBT</em>, <em>42</em>(8), 1187–1188. (<a
href="https://doi.org/10.1038/s41587-023-02046-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Liposomes loaded with antibiotics eliminate intracellular bacteria in a colorectal cancer model, unleashing antitumor T cell immunity.},
  archive      = {J_NBT},
  author       = {Han, Kai and Cho, Young Seok and Moon, James J.},
  doi          = {10.1038/s41587-023-02046-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1187-1188},
  shortjournal = {Nature Biotech.},
  title        = {Antibiotic nanoparticles boost antitumor immunity},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Modeling human trunk development. <em>NBT</em>,
<em>42</em>(8), 1185–1186. (<a
href="https://doi.org/10.1038/s41587-023-02048-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A model of the embryonic human trunk attains a more mature developmental stage than previous models.},
  archive      = {J_NBT},
  author       = {Miao, Yuchuan and Pourquié, Olivier},
  doi          = {10.1038/s41587-023-02048-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1185-1186},
  shortjournal = {Nature Biotech.},
  title        = {Modeling human trunk development},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Tracing the fate of AAV vectors in the body. <em>NBT</em>,
<em>42</em>(8), 1183–1184. (<a
href="https://doi.org/10.1038/s41587-023-02047-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study of AAV gene therapy in monkeys proposes that integrated vectors may drive long-term transgene expression.},
  archive      = {J_NBT},
  author       = {Zabaleta, Nerea and Gil-Farina, Irene},
  doi          = {10.1038/s41587-023-02047-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1183-1184},
  shortjournal = {Nature Biotech.},
  title        = {Tracing the fate of AAV vectors in the body},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). A base editor modifies bacterial genes in the guts of
living mice. <em>NBT</em>, <em>42</em>(8), 1182. (<a
href="https://doi.org/10.1038/s41587-024-02363-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02363-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1182},
  shortjournal = {Nature Biotech.},
  title        = {A base editor modifies bacterial genes in the guts of living mice},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024z). Joint profiling of drug–chromatin binding and epigenetic
landscapes in single cells. <em>NBT</em>, <em>42</em>(8), 1182. (<a
href="https://doi.org/10.1038/s41587-024-02364-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02364-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1182},
  shortjournal = {Nature Biotech.},
  title        = {Joint profiling of drug–chromatin binding and epigenetic landscapes in single cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024j). Bridge RNAs direct programmable DNA rearrangements.
<em>NBT</em>, <em>42</em>(8), 1182. (<a
href="https://doi.org/10.1038/s41587-024-02365-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02365-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1182},
  shortjournal = {Nature Biotech.},
  title        = {Bridge RNAs direct programmable DNA rearrangements},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell analysis. <em>NBT</em>, <em>42</em>(8), 1181.
(<a href="https://doi.org/10.1038/s41587-024-02337-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to compositions, methods and systems for single-cell analysis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02337-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1181},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell analysis},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The patentability and bioethics of molecular de-extinction.
<em>NBT</em>, <em>42</em>(8), 1179–1180. (<a
href="https://doi.org/10.1038/s41587-024-02332-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The resurrection of strings of extinct molecules raises intriguing questions of patentability and bioethical considerations in an emerging area of patent law.},
  archive      = {J_NBT},
  author       = {Torrance, Andrew W. and de la Fuente-Nunez, Cesar},
  doi          = {10.1038/s41587-024-02332-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1179-1180},
  shortjournal = {Nature Biotech.},
  title        = {The patentability and bioethics of molecular de-extinction},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving the predictive power of mouse models.
<em>NBT</em>, <em>42</em>(8), 1175–1177. (<a
href="https://doi.org/10.1038/s41587-024-02349-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comparative studies that integrate genetically diverse mouse models and in vitro cell-based assays will accelerate drug discovery for precision medicine.},
  archive      = {J_NBT},
  author       = {Pera, Martin and Greene, Andy and Cardon, Lon and Carter, Gregory W. and Chesler, Elissa J. and Churchill, Gary and Kumar, Vivek and Lutz, Cathleen and Munger, Steven and Murray, Steve and O’Connell, Kristen and Reinholdt, Laura and Rosenthal, Nadia A.},
  doi          = {10.1038/s41587-024-02349-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1175-1177},
  shortjournal = {Nature Biotech.},
  title        = {Improving the predictive power of mouse models},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The rise of patient avatars in precision oncology.
<em>NBT</em>, <em>42</em>(8), 1173–1174. (<a
href="https://doi.org/10.1038/s41587-024-02335-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ehlen, Lukas and Schmueck-Henneresse, Michael},
  doi          = {10.1038/s41587-024-02335-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1173-1174},
  shortjournal = {Nature Biotech.},
  title        = {The rise of patient avatars in precision oncology},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(8), 1166. (<a
href="https://doi.org/10.1038/s41587-024-02362-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02362-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1166},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spanish tortillas go vegan. <em>NBT</em>, <em>42</em>(8),
1165. (<a href="https://doi.org/10.1038/s41587-024-02366-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hoyos Flight, Monica},
  doi          = {10.1038/s41587-024-02366-1},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1165},
  shortjournal = {Nature Biotech.},
  title        = {Spanish tortillas go vegan},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). HIV drug 100. <em>NBT</em>, <em>42</em>(8), 1164. (<a
href="https://doi.org/10.1038/s41587-024-02369-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02369-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1164},
  shortjournal = {Nature Biotech.},
  title        = {HIV drug 100% protective},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Microbes: Next-generation fertilizers. <em>NBT</em>,
<em>42</em>(8), 1163. (<a
href="https://doi.org/10.1038/s41587-024-02368-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02368-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1163},
  shortjournal = {Nature Biotech.},
  title        = {Microbes: Next-generation fertilizers},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Biotechs take multiplexing into animal models to accelerate
drug discovery. <em>NBT</em>, <em>42</em>(8), 1162–1164. (<a
href="https://doi.org/10.1038/s41587-024-02361-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are moving therapeutic testing from plastic wells into pooled in vivo screening, generating physiologically relevant data by screening many drug candidates at once in a single animal.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-024-02361-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1162-1164},
  shortjournal = {Nature Biotech.},
  title        = {Biotechs take multiplexing into animal models to accelerate drug discovery},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). World’s first tax on cows. <em>NBT</em>, <em>42</em>(8),
1161. (<a href="https://doi.org/10.1038/s41587-024-02367-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02367-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1161},
  shortjournal = {Nature Biotech.},
  title        = {World’s first tax on cows},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A new class of mRNA drugs targets poison exons.
<em>NBT</em>, <em>42</em>(8), 1159–1161. (<a
href="https://doi.org/10.1038/s41587-024-02355-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are targeting poison exons embedded in mRNA transcripts as a strategy to restore wild-type protein abundance and cell fitness in severe pediatric epilepsy and other genetic diseases.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-024-02355-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1159-1161},
  shortjournal = {Nature Biotech.},
  title        = {A new class of mRNA drugs targets poison exons},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Slaughter-free meat hits the grocery shelf. <em>NBT</em>,
<em>42</em>(8), 1157. (<a
href="https://doi.org/10.1038/s41587-024-02373-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As cultivated meat begins to enter grocery stores, governments should promote the field’s development in line with broad public goals of sustainability and accessibility.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02373-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1157},
  shortjournal = {Nature Biotech.},
  title        = {Slaughter-free meat hits the grocery shelf},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024q). Editor’s pick: Indapta therapeutics. <em>NBT</em>,
<em>42</em>(8), 1171–1172. (<a
href="https://doi.org/10.1038/s41587-024-02334-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Indapta Therapeutics is using a subset of natural killer cells found in healthy donors to supercharge monoclonal antibodies to fight cancer and autoimmune disease.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02334-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {1171-1172},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Indapta therapeutics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Editor’s pick: Empress therapeutics. <em>NBT</em>,
<em>42</em>(8), 1169–1170. (<a
href="https://doi.org/10.1038/s41587-024-02314-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Empress Therapeutics is reverse-engineering microbial manufacturing.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-024-02314-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {1169-1170},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Empress therapeutics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Editor’s pick: Unnatural products. <em>NBT</em>,
<em>42</em>(8), 1167–1168. (<a
href="https://doi.org/10.1038/s41587-024-02286-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Unnatural Products is rewriting the rules for potent oral therapies.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-024-02286-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {1167-1168},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Unnatural products},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). People. <em>NBT</em>, <em>42</em>(7), 1156. (<a
href="https://doi.org/10.1038/s41587-024-02312-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02312-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1156},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Five questions with ryan flynn. <em>NBT</em>,
<em>42</em>(7), 1155. (<a
href="https://doi.org/10.1038/s41587-024-02311-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A physician-scientist exploring the mechanisms and physiological relevance of glycoRNAs talks about being useful, the challenges of running a laboratory, and translating foundational knowledge into products that will benefit people.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02311-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1155},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with ryan flynn},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). NIH applicant assistance program counteracts decades of
underrepresentation in SBIR grant awardees. <em>NBT</em>,
<em>42</em>(7), 1151–1153. (<a
href="https://doi.org/10.1038/s41587-024-02308-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The NIH Applicant Assistance Program represents a new paradigm guiding the design of meaningful policy interventions for increasing equity of participation in scientific grant funding.},
  archive      = {J_NBT},
  author       = {Korunes, Katharine L. and Cissell, Ethan C. and Xie, Tiger and Garland, Eva R.},
  doi          = {10.1038/s41587-024-02308-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1151-1153},
  shortjournal = {Nature Biotech.},
  title        = {NIH applicant assistance program counteracts decades of underrepresentation in SBIR grant awardees},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Quartet RNA reference materials improve
the quality of transcriptomic data through ratio-based profiling.
<em>NBT</em>, <em>42</em>(7), 1150. (<a
href="https://doi.org/10.1038/s41587-023-02008-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yu, Ying and Hou, Wanwan and Liu, Yaqing and Wang, Haiyan and Dong, Lianhua and Mai, Yuanbang and Chen, Qingwang and Li, Zhihui and Sun, Shanyue and Yang, Jingcheng and Cao, Zehui and Zhang, Peipei and Zi, Yi and Liu, Ruimei and Gao, Jian and Zhang, Naixin and Li, Jingjing and Ren, Luyao and Jiang, He and Shang, Jun and Zhu, Sibo and Wang, Xiaolin and Qing, Tao and Bao, Ding and Li, Bingying and Li, Bin and Suo, Chen and Pi, Yan and Wang, Xia and Dai, Fangping and Scherer, Andreas and Mattila, Pirkko and Han, Jinxiong and Zhang, Lijun and Jiang, Hui and Thierry-Mieg, Danielle and Thierry-Mieg, Jean and Xiao, Wenming and Hong, Huixiao and Tong, Weida and Wang, Jing and Li, Jinming and Fang, Xiang and Jin, Li and Xu, Joshua and Qian, Feng and Zhang, Rui and Shi, Leming and Zheng, Yuanting},
  doi          = {10.1038/s41587-023-02008-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1150},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Quartet RNA reference materials improve the quality of transcriptomic data through ratio-based profiling},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Why japan lacks a vibrant biotech
industry. <em>NBT</em>, <em>42</em>(7), 1150. (<a
href="https://doi.org/10.1038/s41587-024-02296-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kessel, Mark and Vickrey, Chris},
  doi          = {10.1038/s41587-024-02296-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1150},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Why japan lacks a vibrant biotech industry},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multi-omics data integration using ratio-based quantitative
profiling with quartet reference materials. <em>NBT</em>,
<em>42</em>(7), 1133–1149. (<a
href="https://doi.org/10.1038/s41587-023-01934-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterization and integration of the genome, epigenome, transcriptome, proteome and metabolome of different datasets is difficult owing to a lack of ground truth. Here we develop and characterize suites of publicly available multi-omics reference materials of matched DNA, RNA, protein and metabolites derived from immortalized cell lines from a family quartet of parents and monozygotic twin daughters. These references provide built-in truth defined by relationships among the family members and the information flow from DNA to RNA to protein. We demonstrate how using a ratio-based profiling approach that scales the absolute feature values of a study sample relative to those of a concurrently measured common reference sample produces reproducible and comparable data suitable for integration across batches, labs, platforms and omics types. Our study identifies reference-free ‘absolute’ feature quantification as the root cause of irreproducibility in multi-omics measurement and data integration and establishes the advantages of ratio-based multi-omics profiling with common reference materials. Multi-omics data are readily integrated using quartet reference materials.},
  archive      = {J_NBT},
  author       = {Zheng, Yuanting and Liu, Yaqing and Yang, Jingcheng and Dong, Lianhua and Zhang, Rui and Tian, Sha and Yu, Ying and Ren, Luyao and Hou, Wanwan and Zhu, Feng and Mai, Yuanbang and Han, Jinxiong and Zhang, Lijun and Jiang, Hui and Lin, Ling and Lou, Jingwei and Li, Ruiqiang and Lin, Jingchao and Liu, Huafen and Kong, Ziqing and Wang, Depeng and Dai, Fangping and Bao, Ding and Cao, Zehui and Chen, Qiaochu and Chen, Qingwang and Chen, Xingdong and Gao, Yuechen and Jiang, He and Li, Bin and Li, Bingying and Li, Jingjing and Liu, Ruimei and Qing, Tao and Shang, Erfei and Shang, Jun and Sun, Shanyue and Wang, Haiyan and Wang, Xiaolin and Zhang, Naixin and Zhang, Peipei and Zhang, Ruolan and Zhu, Sibo and Scherer, Andreas and Wang, Jiucun and Wang, Jing and Huo, Yinbo and Liu, Gang and Cao, Chengming and Shao, Li and Xu, Joshua and Hong, Huixiao and Xiao, Wenming and Liang, Xiaozhen and Lu, Daru and Jin, Li and Tong, Weida and Ding, Chen and Li, Jinming and Fang, Xiang and Shi, Leming},
  doi          = {10.1038/s41587-023-01934-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1133-1149},
  shortjournal = {Nature Biotech.},
  title        = {Multi-omics data integration using ratio-based quantitative profiling with quartet reference materials},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Quartet RNA reference materials improve the quality of
transcriptomic data through ratio-based profiling. <em>NBT</em>,
<em>42</em>(7), 1118–1132. (<a
href="https://doi.org/10.1038/s41587-023-01867-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Certified RNA reference materials are indispensable for assessing the reliability of RNA sequencing to detect intrinsically small biological differences in clinical settings, such as molecular subtyping of diseases. As part of the Quartet Project for quality control and data integration of multi-omics profiling, we established four RNA reference materials derived from immortalized B-lymphoblastoid cell lines from four members of a monozygotic twin family. Additionally, we constructed ratio-based transcriptome-wide reference datasets between two samples, providing cross-platform and cross-laboratory ‘ground truth’. Investigation of the intrinsically subtle biological differences among the Quartet samples enables sensitive assessment of cross-batch integration of transcriptomic measurements at the ratio level. The Quartet RNA reference materials, combined with the ratio-based reference datasets, can serve as unique resources for assessing and improving the quality of transcriptomic data in clinical and biological settings. A new RNA reference set improves detection of differential expression in clinical settings.},
  archive      = {J_NBT},
  author       = {Yu, Ying and Hou, Wanwan and Liu, Yaqing and Wang, Haiyan and Dong, Lianhua and Mai, Yuanbang and Chen, Qingwang and Li, Zhihui and Sun, Shanyue and Yang, Jingcheng and Cao, Zehui and Zhang, Peipei and Zi, Yi and Liu, Ruimei and Gao, Jian and Zhang, Naixin and Li, Jingjing and Ren, Luyao and Jiang, He and Shang, Jun and Zhu, Sibo and Wang, Xiaolin and Qing, Tao and Bao, Ding and Li, Bingying and Li, Bin and Suo, Chen and Pi, Yan and Wang, Xia and Dai, Fangping and Scherer, Andreas and Mattila, Pirkko and Han, Jinxiong and Zhang, Lijun and Jiang, Hui and Thierry-Mieg, Danielle and Thierry-Mieg, Jean and Xiao, Wenming and Hong, Huixiao and Tong, Weida and Wang, Jing and Li, Jinming and Fang, Xiang and Jin, Li and Xu, Joshua and Qian, Feng and Zhang, Rui and Shi, Leming and Zheng, Yuanting},
  doi          = {10.1038/s41587-023-01867-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1118-1132},
  shortjournal = {Nature Biotech.},
  title        = {Quartet RNA reference materials improve the quality of transcriptomic data through ratio-based profiling},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic discovery of neoepitope–HLA pairs for neoantigens
shared among patients and tumor types. <em>NBT</em>, <em>42</em>(7),
1107–1117. (<a
href="https://doi.org/10.1038/s41587-023-01945-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The broad application of precision cancer immunotherapies is limited by the number of validated neoepitopes that are common among patients or tumor types. To expand the known repertoire of shared neoantigen–human leukocyte antigen (HLA) complexes, we developed a high-throughput platform that coupled an in vitro peptide–HLA binding assay with engineered cellular models expressing individual HLA alleles in combination with a concatenated transgene harboring 47 common cancer neoantigens. From more than 24,000 possible neoepitope–HLA combinations, biochemical and computational assessment yielded 844 unique candidates, of which 86 were verified after immunoprecipitation mass spectrometry analyses of engineered, monoallelic cell lines. To evaluate the potential for immunogenicity, we identified T cell receptors that recognized select neoepitope–HLA pairs and elicited a response after introduction into human T cells. These cellular systems and our data on therapeutically relevant neoepitopes in their HLA contexts will aid researchers studying antigen processing as well as neoepitope targeting therapies. A large resource of shared tumor neoepitopes aims to accelerate cancer immunotherapy.},
  archive      = {J_NBT},
  author       = {Gurung, Hem R. and Heidersbach, Amy J. and Darwish, Martine and Chan, Pamela Pui Fung and Li, Jenny and Beresini, Maureen and Zill, Oliver A. and Wallace, Andrew and Tong, Ann-Jay and Hascall, Dan and Torres, Eric and Chang, Andy and Lou, Kenny ‘Hei-Wai’ and Abdolazimi, Yassan and Hammer, Christian and Xavier-Magalhães, Ana and Marcu, Ana and Vaidya, Samir and Le, Daniel D. and Akhmetzyanova, Ilseyar and Oh, Soyoung A. and Moore, Amanda J. and Uche, Uzodinma N. and Laur, Melanie B. and Notturno, Richard J. and Ebert, Peter J. R. and Blanchette, Craig and Haley, Benjamin and Rose, Christopher M.},
  doi          = {10.1038/s41587-023-01945-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1107-1117},
  shortjournal = {Nature Biotech.},
  title        = {Systematic discovery of neoepitope–HLA pairs for neoantigens shared among patients and tumor types},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integration of spatial and single-cell data across
modalities with weakly linked features. <em>NBT</em>, <em>42</em>(7),
1096–1106. (<a
href="https://doi.org/10.1038/s41587-023-01935-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although single-cell and spatial sequencing methods enable simultaneous measurement of more than one biological modality, no technology can capture all modalities within the same cell. For current data integration methods, the feasibility of cross-modal integration relies on the existence of highly correlated, a priori ‘linked’ features. We describe matching X-modality via fuzzy smoothed embedding (MaxFuse), a cross-modal data integration method that, through iterative coembedding, data smoothing and cell matching, uses all information in each modality to obtain high-quality integration even when features are weakly linked. MaxFuse is modality-agnostic and demonstrates high robustness and accuracy in the weak linkage scenario, achieving 20~70% relative improvement over existing methods under key evaluation metrics on benchmarking datasets. A prototypical example of weak linkage is the integration of spatial proteomic data with single-cell sequencing data. On two example analyses of this type, MaxFuse enabled the spatial consolidation of proteomic, transcriptomic and epigenomic information at single-cell resolution on the same tissue section. MaxFuse enables data integration between modalities even when features are weakly correlated.},
  archive      = {J_NBT},
  author       = {Chen, Shuxiao and Zhu, Bokai and Huang, Sijia and Hickey, John W. and Lin, Kevin Z. and Snyder, Michael and Greenleaf, William J. and Nolan, Garry P. and Zhang, Nancy R. and Ma, Zongming},
  doi          = {10.1038/s41587-023-01935-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1096-1106},
  shortjournal = {Nature Biotech.},
  title        = {Integration of spatial and single-cell data across modalities with weakly linked features},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Supervised discovery of interpretable gene programs from
single-cell data. <em>NBT</em>, <em>42</em>(7), 1084–1095. (<a
href="https://doi.org/10.1038/s41587-023-01940-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Factor analysis decomposes single-cell gene expression data into a minimal set of gene programs that correspond to processes executed by cells in a sample. However, matrix factorization methods are prone to technical artifacts and poor factor interpretability. We address these concerns with Spectra, an algorithm that combines user-provided gene programs with the detection of novel programs that together best explain expression covariation. Spectra incorporates existing gene sets and cell-type labels as prior biological information, explicitly models cell type and represents input gene sets as a gene–gene knowledge graph using a penalty function to guide factorization toward the input graph. We show that Spectra outperforms existing approaches in challenging tumor immune contexts, as it finds factors that change under immune checkpoint therapy, disentangles the highly correlated features of CD8+ T cell tumor reactivity and exhaustion, finds a program that explains continuous macrophage state changes under therapy and identifies cell-type-specific immune metabolic programs. Spectra decomposes gene expression data into interpretable programs using prior knowledge.},
  archive      = {J_NBT},
  author       = {Kunes, Russell Z. and Walle, Thomas and Land, Max and Nawy, Tal and Pe’er, Dana},
  doi          = {10.1038/s41587-023-01940-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1084-1095},
  shortjournal = {Nature Biotech.},
  title        = {Supervised discovery of interpretable gene programs from single-cell data},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A DNA nanodevice for mapping sodium at single-organelle
resolution. <em>NBT</em>, <em>42</em>(7), 1075–1083. (<a
href="https://doi.org/10.1038/s41587-023-01950-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular sodium ion (Na+) homeostasis is integral to organism physiology. Our current understanding of Na+ homeostasis is largely limited to Na+ transport at the plasma membrane. Organelles may also contribute to Na+ homeostasis; however, the direction of Na+ flow across organelle membranes is unknown because organellar Na+ cannot be imaged. Here we report a pH-independent, organelle-targetable, ratiometric probe that reports lumenal Na+. It is a DNA nanodevice containing a Na+-sensitive fluorophore, a reference dye and an organelle-targeting domain. By measuring Na+ at single endosome resolution in mammalian cells and Caenorhabditis elegans, we discovered that lumenal Na+ levels in each stage of the endolysosomal pathway exceed cytosolic levels and decrease as endosomes mature. Further, we find that lysosomal Na+ levels in nematodes are modulated by the Na+/H+ exchanger NHX-5 in response to salt stress. The ability to image subcellular Na+ will unveil mechanisms of Na+ homeostasis at an increased level of cellular detail. A probe reports absolute sodium levels in acidic organelles with single-organelle resolution.},
  archive      = {J_NBT},
  author       = {Zou, Junyi and Mitra, Koushambi and Anees, Palapuravan and Oettinger, Daphne and Ramirez, Joseph R. and Veetil, Aneesh Tazhe and Gupta, Priyanka Dutta and Rao, Rajini and Smith, Jayson J. and Kratsios, Paschalis and Krishnan, Yamuna},
  doi          = {10.1038/s41587-023-01950-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1075-1083},
  shortjournal = {Nature Biotech.},
  title        = {A DNA nanodevice for mapping sodium at single-organelle resolution},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Detecting organelle-specific activity of potassium channels
with a DNA nanodevice. <em>NBT</em>, <em>42</em>(7), 1065–1074. (<a
href="https://doi.org/10.1038/s41587-023-01928-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell surface potassium ion (K+) channels regulate nutrient transport, cell migration and intercellular communication by controlling K+ permeability and are thought to be active only at the plasma membrane. Although these channels transit the trans-Golgi network, early and recycling endosomes, whether they are active in these organelles is unknown. Here we describe a pH-correctable, ratiometric reporter for K+ called pHlicKer, use it to probe the compartment-specific activity of a prototypical voltage-gated K+ channel, Kv11.1, and show that this cell surface channel is active in organelles. Lumenal K+ in organelles increased in cells expressing wild-type Kv11.1 channels but not after treatment with current blockers. Mutant Kv11.1 channels, with impaired transport function, failed to increase K+ levels in recycling endosomes, an effect rescued by pharmacological correction. By providing a way to map the organelle-specific activity of K+ channels, pHlicKer technology could help identify new organellar K+ channels or channel modulators with nuanced functions. A potassium activity reporter detects organelle-specific channel function.},
  archive      = {J_NBT},
  author       = {Anees, Palapuravan and Saminathan, Anand and Rozmus, Ezekiel R. and Di, Anke and Malik, Asrar B. and Delisle, Brian P. and Krishnan, Yamuna},
  doi          = {10.1038/s41587-023-01928-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1065-1074},
  shortjournal = {Nature Biotech.},
  title        = {Detecting organelle-specific activity of potassium channels with a DNA nanodevice},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Imaging brain tissue architecture across millimeter to
nanometer scales. <em>NBT</em>, <em>42</em>(7), 1051–1064. (<a
href="https://doi.org/10.1038/s41587-023-01911-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping the complex and dense arrangement of cells and their connectivity in brain tissue demands nanoscale spatial resolution imaging. Super-resolution optical microscopy excels at visualizing specific molecules and individual cells but fails to provide tissue context. Here we developed Comprehensive Analysis of Tissues across Scales (CATS), a technology to densely map brain tissue architecture from millimeter regional to nanometer synaptic scales in diverse chemically fixed brain preparations, including rodent and human. CATS uses fixation-compatible extracellular labeling and optical imaging, including stimulated emission depletion or expansion microscopy, to comprehensively delineate cellular structures. It enables three-dimensional reconstruction of single synapses and mapping of synaptic connectivity by identification and analysis of putative synaptic cleft regions. Applying CATS to the mouse hippocampal mossy fiber circuitry, we reconstructed and quantified the synaptic input and output structure of identified neurons. We furthermore demonstrate applicability to clinically derived human tissue samples, including formalin-fixed paraffin-embedded routine diagnostic specimens, for visualizing the cellular architecture of brain tissue in health and disease. Mapping fixed brain samples with extracellular labeling and optical microscopy reveals synaptic connections.},
  archive      = {J_NBT},
  author       = {Michalska, Julia M. and Lyudchik, Julia and Velicky, Philipp and Štefaničková, Hana and Watson, Jake F. and Cenameri, Alban and Sommer, Christoph and Amberg, Nicole and Venturino, Alessandro and Roessler, Karl and Czech, Thomas and Höftberger, Romana and Siegert, Sandra and Novarino, Gaia and Jonas, Peter and Danzl, Johann G.},
  doi          = {10.1038/s41587-023-01911-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1051-1064},
  shortjournal = {Nature Biotech.},
  title        = {Imaging brain tissue architecture across millimeter to nanometer scales},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial multimodal analysis of transcriptomes and
metabolomes in tissues. <em>NBT</em>, <em>42</em>(7), 1046–1050. (<a
href="https://doi.org/10.1038/s41587-023-01937-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a spatial omics approach that combines histology, mass spectrometry imaging and spatial transcriptomics to facilitate precise measurements of mRNA transcripts and low-molecular-weight metabolites across tissue regions. The workflow is compatible with commercially available Visium glass slides. We demonstrate the potential of our method using mouse and human brain samples in the context of dopamine and Parkinson’s disease. Metabolites and RNA in a tissue section are profiled simultaneously.},
  archive      = {J_NBT},
  author       = {Vicari, Marco and Mirzazadeh, Reza and Nilsson, Anna and Shariatgorji, Reza and Bjärterot, Patrik and Larsson, Ludvig and Lee, Hower and Nilsson, Mats and Foyer, Julia and Ekvall, Markus and Czarnewski, Paulo and Zhang, Xiaoqun and Svenningsson, Per and Käll, Lukas and Andrén, Per E. and Lundeberg, Joakim},
  doi          = {10.1038/s41587-023-01937-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1046-1050},
  shortjournal = {Nature Biotech.},
  title        = {Spatial multimodal analysis of transcriptomes and metabolomes in tissues},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Targeted genome editing with a DNA-dependent DNA polymerase
and exogenous DNA-containing templates. <em>NBT</em>, <em>42</em>(7),
1039–1045. (<a
href="https://doi.org/10.1038/s41587-023-01947-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reverse transcriptases, used in prime editing systems, exhibit lower fidelity, processivity and dNTP affinity than many DNA-dependent DNA polymerases. We report that a DNA-dependent DNA polymerase (phi29), untethered from Cas9, enables editing from a synthetic, end-stabilized DNA-containing template at up to 60% efficiency in human cells. Compared to prime editing, DNA polymerase editing avoids autoinhibitory intramolecular base pairing of the template, facilitates template synthesis and supports larger insertions (&amp;gt;100 nucleotides). A DNA polymerase editing system offers advantages over prime editing.},
  archive      = {J_NBT},
  author       = {Liu, Bin and Dong, Xiaolong and Zheng, Chunwei and Keener, David and Chen, Zexiang and Cheng, Haoyang and Watts, Jonathan K. and Xue, Wen and Sontheimer, Erik J.},
  doi          = {10.1038/s41587-023-01947-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1039-1045},
  shortjournal = {Nature Biotech.},
  title        = {Targeted genome editing with a DNA-dependent DNA polymerase and exogenous DNA-containing templates},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MaxFuse enables data integration across weakly linked
spatial and single-cell modalities. <em>NBT</em>, <em>42</em>(7),
1036–1037. (<a
href="https://doi.org/10.1038/s41587-023-01943-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data integration between weakly linked single-cell modalities is challenging using existing methods. Therefore, we developed MaxFuse to enable matching and integration between cells from modalities such as single-cell spatial proteomic datasets and single-cell transcriptomic datasets, or other modalities where features are only weakly correlated.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01943-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1036-1037},
  shortjournal = {Nature Biotech.},
  title        = {MaxFuse enables data integration across weakly linked spatial and single-cell modalities},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Decoding the building blocks of cellular processes from
single-cell transcriptomics data. <em>NBT</em>, <em>42</em>(7),
1034–1035. (<a
href="https://doi.org/10.1038/s41587-023-01967-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most features of a cell are determined by gene programs — sets of co-expressed genes that execute a specific function. By incorporating existing knowledge about gene programs and cell types, the Spectra factor analysis method improves how we decode single-cell transcriptomic data and offers insights into challenging tumor immune contexts.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01967-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1034-1035},
  shortjournal = {Nature Biotech.},
  title        = {Decoding the building blocks of cellular processes from single-cell transcriptomics data},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). From bench to podcast: Roman sarrazin-gendron.
<em>NBT</em>, <em>42</em>(7), 1033. (<a
href="https://doi.org/10.1038/s41587-024-02309-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02309-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1033},
  shortjournal = {Nature Biotech.},
  title        = {From bench to podcast: Roman sarrazin-gendron},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Forum: Lancaster and pașca. <em>NBT</em>, <em>42</em>(7),
1033. (<a href="https://doi.org/10.1038/s41587-024-02331-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02331-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1033},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Lancaster and pașca},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DNA nanodevices map intracellular ions. <em>NBT</em>,
<em>42</em>(7), 1030–1032. (<a
href="https://doi.org/10.1038/s41587-023-02054-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two DNA nanodevices, RatiNa and pHlicker, provide precise mapping of sodium levels in acidic organelles and potassium activity at a single-organelle resolution.},
  archive      = {J_NBT},
  author       = {Xu, Haoxing},
  doi          = {10.1038/s41587-023-02054-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1030-1032},
  shortjournal = {Nature Biotech.},
  title        = {DNA nanodevices map intracellular ions},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Brain imaging turned inside out. <em>NBT</em>,
<em>42</em>(7), 1028–1029. (<a
href="https://doi.org/10.1038/s41587-023-02036-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neural connectivity in brain tissue is imaged by labeling the extracellular space.},
  archive      = {J_NBT},
  author       = {Askari, Shahrzad and Misgeld, Thomas},
  doi          = {10.1038/s41587-023-02036-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1028-1029},
  shortjournal = {Nature Biotech.},
  title        = {Brain imaging turned inside out},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024n). Cryo-EM structure of a prime editor complex in multiple
states. <em>NBT</em>, <em>42</em>(7), 1027. (<a
href="https://doi.org/10.1038/s41587-024-02325-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02325-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1027},
  shortjournal = {Nature Biotech.},
  title        = {Cryo-EM structure of a prime editor complex in multiple states},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024ai). Vision–language AI assistance in human pathology.
<em>NBT</em>, <em>42</em>(7), 1027. (<a
href="https://doi.org/10.1038/s41587-024-02326-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02326-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1027},
  shortjournal = {Nature Biotech.},
  title        = {Vision–language AI assistance in human pathology},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024l). Counteracting tissue aging through TERT activation.
<em>NBT</em>, <em>42</em>(7), 1027. (<a
href="https://doi.org/10.1038/s41587-024-02327-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02327-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1027},
  shortjournal = {Nature Biotech.},
  title        = {Counteracting tissue aging through TERT activation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Super-resolution microscopy. <em>NBT</em>, <em>42</em>(7),
1026. (<a href="https://doi.org/10.1038/s41587-024-02304-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to systems and methods for super-resolution microscopy.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02304-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1026},
  shortjournal = {Nature Biotech.},
  title        = {Super-resolution microscopy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). The cost of drug patent expiration date errors.
<em>NBT</em>, <em>42</em>(7), 1024–1025. (<a
href="https://doi.org/10.1038/s41587-024-02298-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Errors in the US Patent and Trademark Office’s automated program to calculate patent expiration dates could result in hundreds of millions of dollars in added costs to drug prices.},
  archive      = {J_NBT},
  author       = {Tu, S. Sean and Cheian, Dinis and Gabriele, Sarah and Rome, Benjamin N. and Kesselheim, Aaron S.},
  doi          = {10.1038/s41587-024-02298-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1024-1025},
  shortjournal = {Nature Biotech.},
  title        = {The cost of drug patent expiration date errors},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A community effort to track commercial single-cell and
spatial ’omic technologies and business trends. <em>NBT</em>,
<em>42</em>(7), 1017–1023. (<a
href="https://doi.org/10.1038/s41587-024-02305-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is an ever-growing choice of single-cell and spatial ’omics platforms for industry and academia. The scTrends Consortium provides a brief historical overview of the established platforms and companies, revealing market trends and presenting possible angles for how technologies may differentiate themselves.},
  archive      = {J_NBT},
  author       = {De Jonghe, Joachim and Opzoomer, James W. and Vilas-Zornoza, Amaia and Crane, Peter and Nilges, Benedikt S. and Vicari, Marco and Lee, Hower and Lara-Astiaso, David and Gross, Torsten and Morf, Jörg and Schneider, Kim and Cudini, Juliana and Ramos-Mucci, Lorenzo and Mooijman, Dylan and Tiklová, Katarína and Salas, Sergio Marco and Langseth, Christoffer Mattsson and Kashikar, Nachiket D. and Roberts, Charles E. S. and Lundeberg, Joakim and Nilsson, Mats and Shalek, Alex K. and Cribbs, Adam P. and Taylor-King, Jake P.},
  doi          = {10.1038/s41587-024-02305-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1017-1023},
  shortjournal = {Nature Biotech.},
  title        = {A community effort to track commercial single-cell and spatial ’omic technologies and business trends},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Effects of the first successful lawsuit against a consumer
neurotechnology company for violating brain data privacy. <em>NBT</em>,
<em>42</em>(7), 1015–1016. (<a
href="https://doi.org/10.1038/s41587-024-02303-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Muñoz, José M. and Marinaro, José Ángel and Iglesias, José Antonio and Sánchez, Moisés and Monti, Natalia and Colombara, Ciro and Girardi, Guido},
  doi          = {10.1038/s41587-024-02303-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1015-1016},
  shortjournal = {Nature Biotech.},
  title        = {Effects of the first successful lawsuit against a consumer neurotechnology company for violating brain data privacy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Increasing access to biotech products for animal agriculture
in sub-saharan africa through partnerships. <em>NBT</em>,
<em>42</em>(7), 1013–1014. (<a
href="https://doi.org/10.1038/s41587-024-02300-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de la Fuente, José and Gortázar, Christian and Contreras, Marinela and Kabi, Frederick and Kasaija, Paul and Mugerwa, Swidiq and Rutaisire, Justus},
  doi          = {10.1038/s41587-024-02300-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1013-1014},
  shortjournal = {Nature Biotech.},
  title        = {Increasing access to biotech products for animal agriculture in sub-saharan africa through partnerships},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(7), 1001. (<a
href="https://doi.org/10.1038/s41587-024-02328-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02328-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1001},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Citizen chickpeas. <em>NBT</em>, <em>42</em>(7), 1000. (<a
href="https://doi.org/10.1038/s41587-024-02333-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02333-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1000},
  shortjournal = {Nature Biotech.},
  title        = {Citizen chickpeas},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bezos backs taste-good lab proteins. <em>NBT</em>,
<em>42</em>(7), 999. (<a
href="https://doi.org/10.1038/s41587-024-02330-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02330-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {999},
  shortjournal = {Nature Biotech.},
  title        = {Bezos backs taste-good lab proteins},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Fungal detox: Investors eye mycelium bioremediation.
<em>NBT</em>, <em>42</em>(7), 998–1000. (<a
href="https://doi.org/10.1038/s41587-024-02315-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The digestive prowess of fungi to decontaminate natural environments from polluting plastics, oil spills, toxic waste and even radiation is gaining traction around the globe.},
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-024-02315-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {998-1000},
  shortjournal = {Nature Biotech.},
  title        = {Fungal detox: Investors eye mycelium bioremediation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Roche bets on RNA exon editor. <em>NBT</em>, <em>42</em>(7),
997. (<a href="https://doi.org/10.1038/s41587-024-02329-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02329-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {997},
  shortjournal = {Nature Biotech.},
  title        = {Roche bets on RNA exon editor},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). CAR-ts sweep into autoimmunity. <em>NBT</em>,
<em>42</em>(7), 995–997. (<a
href="https://doi.org/10.1038/s41587-024-02321-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biotechs are equipping CAR-T cells to destroy B cells — not to treat blood cancers, but to take on multiple sclerosis and a raft of autoimmune disorders.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-024-02321-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {995-997},
  shortjournal = {Nature Biotech.},
  title        = {CAR-ts sweep into autoimmunity},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Breathing new life into in vivo lung editing. <em>NBT</em>,
<em>42</em>(7), 993–994. (<a
href="https://doi.org/10.1038/s41587-024-02338-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New lipid nanoparticle designs make targeted in vivo lung editing possible in mice and may lead to new treatment options for cystic fibrosis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02338-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {993-994},
  shortjournal = {Nature Biotech.},
  title        = {Breathing new life into in vivo lung editing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Editor’s pick: Tenpoint therapeutics. <em>NBT</em>,
<em>42</em>(7), 1011–1012. (<a
href="https://doi.org/10.1038/s41587-024-02281-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Tenpoint Therapeutics optimizes delivery and generation of stem-cell-derived retinal cells for vision repair.},
  archive      = {J_NBT},
  author       = {Schmidt, Charles},
  doi          = {10.1038/s41587-024-02281-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {7},
  pages        = {1011-1012},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Tenpoint therapeutics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Editor’s pick: Tome biosciences. <em>NBT</em>,
<em>42</em>(7), 1009–1010. (<a
href="https://doi.org/10.1038/s41587-024-02280-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Tome Biosciences inserts large DNA sequences into precise genomic locations, overcoming limitations of base and prime editing.},
  archive      = {J_NBT},
  author       = {Schmidt, Charles},
  doi          = {10.1038/s41587-024-02280-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {7},
  pages        = {1009-1010},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Tome biosciences},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Precision radiation opens a new window on cancer therapy.
<em>NBT</em>, <em>42</em>(7), 1003–1008. (<a
href="https://doi.org/10.1038/s41587-024-02295-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted radiopharmaceuticals offer resistance-free potency, pre-treatment imaging and widespread combination potential. Big pharma and investors are piling in.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-024-02295-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {7},
  pages        = {1003-1008},
  shortjournal = {Nature Biotech.},
  title        = {Precision radiation opens a new window on cancer therapy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). People. <em>NBT</em>, <em>42</em>(6), 992. (<a
href="https://doi.org/10.1038/s41587-024-02278-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02278-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {992},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Barriers and enablers to the effective implementation of
omics research in low- and middle-income countries. <em>NBT</em>,
<em>42</em>(6), 988–991. (<a
href="https://doi.org/10.1038/s41587-024-02274-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Addressing the hurdles and opportunities associated with omics research in low- and middle-income countries may inform strategies for its effective execution, and thus increase our ability to tackle health challenges that transcend geographical boundaries.},
  archive      = {J_NBT},
  author       = {Nacis, Jacus S. and Kamande, Patrick and Toni, Alemayehu Teklu and Mudibo, Evans and Musyimi, Robert and Popluechai, Siam and Dailey-Chwalibóg, Trenton and Voskuijl, Wieger and Dable-Tupas, Genevieve and Shahid, Abu Sadat Mohammad Sayeem Bin and Bascos, Neil Andrew and Afroze, Farzana and Chisti, Mohammod Jobayer and Singa, Benson and Ngari, Moses and Tigoi, Caroline and Mhango, Gomezgani and Freitag, Harry and Potani, Isabel and Mukisa, John and Kirolos, Amir and Mutasa, Kuda and Ouédraogo, Lionel Olivier and Prentice, Andrew M. and Girma, Tsinuel and Prendergast, Andrew J. and Njunge, James and Kelly, Paul and Berkley, James A. and Tickell, Kirkby Daniel and Gonzales, Gerard Bryan},
  doi          = {10.1038/s41587-024-02274-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {988-991},
  shortjournal = {Nature Biotech.},
  title        = {Barriers and enablers to the effective implementation of omics research in low- and middle-income countries},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Re-engineering the adenine deaminase
TadA-8e for efficient and specific CRISPR-based cytosine base editing.
<em>NBT</em>, <em>42</em>(6), 987. (<a
href="https://doi.org/10.1038/s41587-024-02252-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chen, Liang and Zhu, Biyun and Ru, Gaomeng and Meng, Haowei and Yan, Yongchang and Hong, Mengjia and Zhang, Dan and Luan, Changming and Zhang, Shun and Wu, Hao and Gao, Hongyi and Bai, Sijia and Li, Changqing and Ding, Ruoyi and Xue, Niannian and Lei, Zhixin and Chen, Yuting and Guan, Yuting and Siwko, Stefan and Cheng, Yiyun and Song, Gaojie and Wang, Liren and Yi, Chengqi and Liu, Mingyao and Li, Dali},
  doi          = {10.1038/s41587-024-02252-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {987},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Re-engineering the adenine deaminase TadA-8e for efficient and specific CRISPR-based cytosine base editing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Author correction: Adenine transversion editors enable
precise, efficient a•t-to-c•g base editing in mammalian cells and
embryos. <em>NBT</em>, <em>42</em>(6), 987. (<a
href="https://doi.org/10.1038/s41587-024-02253-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chen, Liang and Hong, Mengjia and Luan, Changming and Gao, Hongyi and Ru, Gaomeng and Guo, Xinyuan and Zhang, Dujuan and Zhang, Shun and Li, Changwei and Wu, Jun and Randolph, Peyton B. and Sousa, Alexander A. and Qu, Chao and Zhu, Yifan and Guan, Yuting and Wang, Liren and Liu, Mingyao and Feng, Bo and Song, Gaojie and Liu, David R. and Li, Dali},
  doi          = {10.1038/s41587-024-02253-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {987},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Adenine transversion editors enable precise, efficient A•T-to-C•G base editing in mammalian cells and embryos},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Genotoxic effects of base and prime
editing in human hematopoietic stem cells. <em>NBT</em>, <em>42</em>(6),
986. (<a href="https://doi.org/10.1038/s41587-024-02142-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fiumara, Martina and Ferrari, Samuele and Omer-Javed, Attya and Beretta, Stefano and Albano, Luisa and Canarutto, Daniele and Varesi, Angelica and Gaddoni, Chiara and Brombin, Chiara and Cugnata, Federica and Zonari, Erika and Naldini, Matteo Maria and Barcella, Matteo and Gentner, Bernhard and Merelli, Ivan and Naldini, Luigi},
  doi          = {10.1038/s41587-024-02142-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {986},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Genotoxic effects of base and prime editing in human hematopoietic stem cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Protein remote homology detection and structural alignment
using deep learning. <em>NBT</em>, <em>42</em>(6), 975–985. (<a
href="https://doi.org/10.1038/s41587-023-01917-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exploiting sequence–structure–function relationships in biotechnology requires improved methods for aligning proteins that have low sequence similarity to previously annotated proteins. We develop two deep learning methods to address this gap, TM-Vec and DeepBLAST. TM-Vec allows searching for structure–structure similarities in large sequence databases. It is trained to accurately predict TM-scores as a metric of structural similarity directly from sequence pairs without the need for intermediate computation or solution of structures. Once structurally similar proteins have been identified, DeepBLAST can structurally align proteins using only sequence information by identifying structurally homologous regions between proteins. It outperforms traditional sequence alignment methods and performs similarly to structure-based alignment methods. We show the merits of TM-Vec and DeepBLAST on a variety of datasets, including better identification of remotely homologous proteins compared with state-of-the-art sequence alignment and structure prediction methods. Protein sequence and structural similarities in large databases are rapidly identified using machine learning.},
  archive      = {J_NBT},
  author       = {Hamamsy, Tymor and Morton, James T. and Blackwell, Robert and Berenberg, Daniel and Carriero, Nicholas and Gligorijevic, Vladimir and Strauss, Charlie E. M. and Leman, Julia Koehler and Cho, Kyunghyun and Bonneau, Richard},
  doi          = {10.1038/s41587-023-01917-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {975-985},
  shortjournal = {Nature Biotech.},
  title        = {Protein remote homology detection and structural alignment using deep learning},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Joint single-cell profiling resolves 5mC and 5hmC and
reveals their distinct gene regulatory effects. <em>NBT</em>,
<em>42</em>(6), 960–974. (<a
href="https://doi.org/10.1038/s41587-023-01909-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oxidative modification of 5-methylcytosine (5mC) by ten-eleven translocation (TET) DNA dioxygenases generates 5-hydroxymethylcytosine (5hmC), the most abundant form of oxidized 5mC. Existing single-cell bisulfite sequencing methods cannot resolve 5mC and 5hmC, leaving the cell-type-specific regulatory mechanisms of TET and 5hmC largely unknown. Here, we present joint single-nucleus (hydroxy)methylcytosine sequencing (Joint-snhmC-seq), a scalable and quantitative approach that simultaneously profiles 5hmC and true 5mC in single cells by harnessing differential deaminase activity of APOBEC3A toward 5mC and chemically protected 5hmC. Joint-snhmC-seq profiling of single nuclei from mouse brains reveals an unprecedented level of epigenetic heterogeneity of both 5hmC and true 5mC at single-cell resolution. We show that cell-type-specific profiles of 5hmC or true 5mC improve multimodal single-cell data integration, enable accurate identification of neuronal subtypes and uncover context-specific regulatory effects on cell-type-specific genes by TET enzymes. Simultaneous single-cell profiling of 5hmC and 5mC shows their unique regulatory roles.},
  archive      = {J_NBT},
  author       = {Fabyanic, Emily B. and Hu, Peng and Qiu, Qi and Berríos, Kiara N. and Connolly, Daniel R. and Wang, Tong and Flournoy, Jennifer and Zhou, Zhaolan and Kohli, Rahul M. and Wu, Hao},
  doi          = {10.1038/s41587-023-01909-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {960-974},
  shortjournal = {Nature Biotech.},
  title        = {Joint single-cell profiling resolves 5mC and 5hmC and reveals their distinct gene regulatory effects},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell lineage capture across genomic modalities with
CellTag-multi reveals fate-specific gene regulatory changes.
<em>NBT</em>, <em>42</em>(6), 946–959. (<a
href="https://doi.org/10.1038/s41587-023-01931-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex gene regulatory mechanisms underlie differentiation and reprogramming. Contemporary single-cell lineage-tracing (scLT) methods use expressed, heritable DNA barcodes to combine cell lineage readout with single-cell transcriptomics. However, reliance on transcriptional profiling limits adaptation to other single-cell assays. With CellTag-multi, we present an approach that enables direct capture of heritable random barcodes expressed as polyadenylated transcripts, in both single-cell RNA sequencing and single-cell Assay for Transposase Accessible Chromatin using sequencing assays, allowing for independent clonal tracking of transcriptional and epigenomic cell states. We validate CellTag-multi to characterize progenitor cell lineage priming during mouse hematopoiesis. Additionally, in direct reprogramming of fibroblasts to endoderm progenitors, we identify core regulatory programs underlying on-target and off-target fates. Furthermore, we reveal the transcription factor Zfp281 as a regulator of reprogramming outcome, biasing cells toward an off-target mesenchymal fate. Our results establish CellTag-multi as a lineage-tracing method compatible with multiple single-cell modalities and demonstrate its utility in revealing fate-specifying gene regulatory changes across diverse paradigms of differentiation and reprogramming. Lineage tracing using both transcriptomics and chromatin accessibility provides mechanistic insights into cell fate.},
  archive      = {J_NBT},
  author       = {Jindal, Kunal and Adil, Mohd Tayyab and Yamaguchi, Naoto and Yang, Xue and Wang, Helen C. and Kamimoto, Kenji and Rivera-Gonzalez, Guillermo C. and Morris, Samantha A.},
  doi          = {10.1038/s41587-023-01931-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {946-959},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell lineage capture across genomic modalities with CellTag-multi reveals fate-specific gene regulatory changes},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Strand-preferred base editing of organellar and nuclear
genomes using CyDENT. <em>NBT</em>, <em>42</em>(6), 936–945. (<a
href="https://doi.org/10.1038/s41587-023-01910-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription-activator-like effector (TALE)-based tools for base editing of nuclear and organellar DNA rely on double-stranded DNA deaminases, which edit substrate bases on both strands of DNA, reducing editing precision. Here, we present CyDENT base editing, a CRISPR-free, strand-selective, modular base editor. CyDENT comprises a pair of TALEs fused with a FokI nickase, a single-strand-specific cytidine deaminase and an exonuclease to generate a single-stranded DNA substrate for deamination. We demonstrate effective base editing in nuclear, mitochondrial and chloroplast genomes. At certain mitochondrial sites, we show editing efficiencies of 14% and strand specificity of 95%. Furthermore, by exchanging the CyDENT deaminase with one that prefers editing GC motifs, we demonstrate up to 20% mitochondrial base editing at sites that are otherwise inaccessible to editing by other methods. The modular nature of CyDENT enables a suite of bespoke base editors for various applications. A TALE-based modular base editing system shows high strand specificity in multiple organelles.},
  archive      = {J_NBT},
  author       = {Hu, Jiacheng and Sun, Yu and Li, Boshu and Liu, Zhen and Wang, Zhiwei and Gao, Qiang and Guo, Mengyue and Liu, Guanwen and Zhao, Kevin Tianmeng and Gao, Caixia},
  doi          = {10.1038/s41587-023-01910-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {936-945},
  shortjournal = {Nature Biotech.},
  title        = {Strand-preferred base editing of organellar and nuclear genomes using CyDENT},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Predicting transcriptional outcomes of novel multigene
perturbations with GEARS. <em>NBT</em>, <em>42</em>(6), 927–935. (<a
href="https://doi.org/10.1038/s41587-023-01905-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding cellular responses to genetic perturbation is central to numerous biomedical applications, from identifying genetic interactions involved in cancer to developing methods for regenerative medicine. However, the combinatorial explosion in the number of possible multigene perturbations severely limits experimental interrogation. Here, we present graph-enhanced gene activation and repression simulator (GEARS), a method that integrates deep learning with a knowledge graph of gene–gene relationships to predict transcriptional responses to both single and multigene perturbations using single-cell RNA-sequencing data from perturbational screens. GEARS is able to predict outcomes of perturbing combinations consisting of genes that were never experimentally perturbed. GEARS exhibited 40% higher precision than existing approaches in predicting four distinct genetic interaction subtypes in a combinatorial perturbation screen and identified the strongest interactions twice as well as prior approaches. Overall, GEARS can predict phenotypically distinct effects of multigene perturbations and thus guide the design of perturbational experiments. GEARS predicts transcriptional changes of multigene perturbations even for genes that were never experimentally perturbed.},
  archive      = {J_NBT},
  author       = {Roohani, Yusuf and Huang, Kexin and Leskovec, Jure},
  doi          = {10.1038/s41587-023-01905-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {927-935},
  shortjournal = {Nature Biotech.},
  title        = {Predicting transcriptional outcomes of novel multigene perturbations with GEARS},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic benchmarking of single-cell ATAC-sequencing
protocols. <em>NBT</em>, <em>42</em>(6), 916–926. (<a
href="https://doi.org/10.1038/s41587-023-01881-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell assay for transposase-accessible chromatin by sequencing (scATAC-seq) has emerged as a powerful tool for dissecting regulatory landscapes and cellular heterogeneity. However, an exploration of systemic biases among scATAC-seq technologies has remained absent. In this study, we benchmark the performance of eight scATAC-seq methods across 47 experiments using human peripheral blood mononuclear cells (PBMCs) as a reference sample and develop PUMATAC, a universal preprocessing pipeline, to handle the various sequencing data formats. Our analyses reveal significant differences in sequencing library complexity and tagmentation specificity, which impact cell-type annotation, genotype demultiplexing, peak calling, differential region accessibility and transcription factor motif enrichment. Our findings underscore the importance of sample extraction, method selection, data processing and total cost of experiments, offering valuable guidance for future research. Finally, our data and analysis pipeline encompasses 169,000 PBMC scATAC-seq profiles and a best practices code repository for scATAC-seq data analysis, which are freely available to extend this benchmarking effort to future protocols. Benchmarking of scATAC-seq protocols demonstrates the effects of data quality on downstream analyses.},
  archive      = {J_NBT},
  author       = {De Rop, Florian V. and Hulselmans, Gert and Flerin, Chris and Soler-Vila, Paula and Rafels, Albert and Christiaens, Valerie and González-Blas, Carmen Bravo and Marchese, Domenica and Caratù, Ginevra and Poovathingal, Suresh and Rozenblatt-Rosen, Orit and Slyper, Michael and Luo, Wendy and Muus, Christoph and Duarte, Fabiana and Shrestha, Rojesh and Bagdatli, S. Tansu and Corces, M. Ryan and Mamanova, Lira and Knights, Andrew and Meyer, Kerstin B. and Mulqueen, Ryan and Taherinasab, Akram and Maschmeyer, Patrick and Pezoldt, Jörn and Lambert, Camille Lucie Germaine and Iglesias, Marta and Najle, Sebastián R. and Dossani, Zain Y. and Martelotto, Luciano G. and Burkett, Zach and Lebofsky, Ronald and Martin-Subero, José Ignacio and Pillai, Satish and Sebé-Pedrós, Arnau and Deplancke, Bart and Teichmann, Sarah A. and Ludwig, Leif S. and Braun, Theodore P. and Adey, Andrew C. and Greenleaf, William J. and Buenrostro, Jason D. and Regev, Aviv and Aerts, Stein and Heyn, Holger},
  doi          = {10.1038/s41587-023-01881-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {916-926},
  shortjournal = {Nature Biotech.},
  title        = {Systematic benchmarking of single-cell ATAC-sequencing protocols},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A germ-free humanized mouse model shows the contribution of
resident microbiota to human-specific pathogen infection. <em>NBT</em>,
<em>42</em>(6), 905–915. (<a
href="https://doi.org/10.1038/s41587-023-01906-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Germ-free (GF) mice, which are depleted of their resident microbiota, are the gold standard for exploring the role of the microbiome in health and disease; however, they are of limited value in the study of human-specific pathogens because they do not support their replication. Here, we develop GF mice systemically reconstituted with human immune cells and use them to evaluate the role of the resident microbiome in the acquisition, replication and pathogenesis of two human-specific pathogens, Epstein–Barr virus (EBV) and human immunodeficiency virus (HIV). Comparison with conventional (CV) humanized mice showed that resident microbiota enhance the establishment of EBV infection and EBV-induced tumorigenesis and increase mucosal HIV acquisition and replication. HIV RNA levels were higher in plasma and tissues of CV humanized mice compared with GF humanized mice. The frequency of CCR5+ CD4+ T cells throughout the intestine was also higher in CV humanized mice, indicating that resident microbiota govern levels of HIV target cells. Thus, resident microbiota promote the acquisition and pathogenesis of two clinically relevant human-specific pathogens. Resident microbiota contribute to HIV and EBV infection in a germ-free humanized mouse model.},
  archive      = {J_NBT},
  author       = {Wahl, Angela and Yao, Wenbo and Liao, Baolin and Chateau, Morgan and Richardson, Cara and Ling, Lijun and Franks, Adrienne and Senthil, Krithika and Doyon, Genevieve and Li, Fengling and Frost, Josh and Whitehurst, Christopher B. and Pagano, Joseph S. and Fletcher, Craig A. and Azcarate-Peril, M. Andrea and Hudgens, Michael G. and Rogala, Allison R. and Tucker, Joseph D. and McGowan, Ian and Sartor, R. Balfour and Garcia, J. Victor},
  doi          = {10.1038/s41587-023-01906-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {905-915},
  shortjournal = {Nature Biotech.},
  title        = {A germ-free humanized mouse model shows the contribution of resident microbiota to human-specific pathogen infection},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multifunctional microelectronic fibers enable wireless
modulation of gut and brain neural circuits. <em>NBT</em>,
<em>42</em>(6), 892–904. (<a
href="https://doi.org/10.1038/s41587-023-01833-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Progress in understanding brain–viscera interoceptive signaling is hindered by a dearth of implantable devices suitable for probing both brain and peripheral organ neurophysiology during behavior. Here we describe multifunctional neural interfaces that combine the scalability and mechanical versatility of thermally drawn polymer-based fibers with the sophistication of microelectronic chips for organs as diverse as the brain and the gut. Our approach uses meters-long continuous fibers that can integrate light sources, electrodes, thermal sensors and microfluidic channels in a miniature footprint. Paired with custom-fabricated control modules, the fibers wirelessly deliver light for optogenetics and transfer data for physiological recording. We validate this technology by modulating the mesolimbic reward pathway in the mouse brain. We then apply the fibers in the anatomically challenging intestinal lumen and demonstrate wireless control of sensory epithelial cells that guide feeding behaviors. Finally, we show that optogenetic stimulation of vagal afferents from the intestinal lumen is sufficient to evoke a reward phenotype in untethered mice. Crosstalk between the brain and gut is modulated and recorded with microelectronics fibers.},
  archive      = {J_NBT},
  author       = {Sahasrabudhe, Atharva and Rupprecht, Laura E. and Orguc, Sirma and Khudiyev, Tural and Tanaka, Tomo and Sands, Joanna and Zhu, Weikun and Tabet, Anthony and Manthey, Marie and Allen, Harrison and Loke, Gabriel and Antonini, Marc-Joseph and Rosenfeld, Dekel and Park, Jimin and Garwood, Indie C. and Yan, Wei and Niroui, Farnaz and Fink, Yoel and Chandrakasan, Anantha and Bohórquez, Diego V. and Anikeeva, Polina},
  doi          = {10.1038/s41587-023-01833-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {892-904},
  shortjournal = {Nature Biotech.},
  title        = {Multifunctional microelectronic fibers enable wireless modulation of gut and brain neural circuits},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Genotoxic effects of base and prime editing in human
hematopoietic stem cells. <em>NBT</em>, <em>42</em>(6), 877–891. (<a
href="https://doi.org/10.1038/s41587-023-01915-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Base and prime editors (BEs and PEs) may provide more precise genetic engineering than nuclease-based approaches because they bypass the dependence on DNA double-strand breaks. However, little is known about their cellular responses and genotoxicity. Here, we compared state-of-the-art BEs and PEs and Cas9 in human hematopoietic stem and progenitor cells with respect to editing efficiency, cytotoxicity, transcriptomic changes and on-target and genome-wide genotoxicity. BEs and PEs induced detrimental transcriptional responses that reduced editing efficiency and hematopoietic repopulation in xenotransplants and also generated DNA double-strand breaks and genotoxic byproducts, including deletions and translocations, at a lower frequency than Cas9. These effects were strongest for cytidine BEs due to suboptimal inhibition of base excision repair and were mitigated by tailoring delivery timing and editor expression through optimized mRNA design. However, BEs altered the mutational landscape of hematopoietic stem and progenitor cells across the genome by increasing the load and relative proportions of nucleotide variants. These findings raise concerns about the genotoxicity of BEs and PEs and warrant further investigation in view of their clinical application. Base and prime editors induce double-strand breaks, deletions and translocations in hematopoietic cells.},
  archive      = {J_NBT},
  author       = {Fiumara, Martina and Ferrari, Samuele and Omer-Javed, Attya and Beretta, Stefano and Albano, Luisa and Canarutto, Daniele and Varesi, Angelica and Gaddoni, Chiara and Brombin, Chiara and Cugnata, Federica and Zonari, Erika and Naldini, Matteo Maria and Barcella, Matteo and Gentner, Bernhard and Merelli, Ivan and Naldini, Luigi},
  doi          = {10.1038/s41587-023-01915-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {877-891},
  shortjournal = {Nature Biotech.},
  title        = {Genotoxic effects of base and prime editing in human hematopoietic stem cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Near-infrared co-illumination of fluorescent proteins
reduces photobleaching and phototoxicity. <em>NBT</em>, <em>42</em>(6),
872–876. (<a href="https://doi.org/10.1038/s41587-023-01893-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present a method to reduce the photobleaching of fluorescent proteins and the associated phototoxicity. It exploits a photophysical process known as reverse intersystem crossing, which we induce by near-infrared co-illumination during fluorophore excitation. This dual illumination method reduces photobleaching effects 1.5–9.2-fold, can be easily implemented on commercial microscopes and is effective in eukaryotic and prokaryotic cells with a wide range of fluorescent proteins. A dual illumination method reduces photobleaching for green and yellow fluorescent proteins.},
  archive      = {J_NBT},
  author       = {Ludvikova, Lucie and Simon, Emma and Deygas, Mathieu and Panier, Thomas and Plamont, Marie-Aude and Ollion, Jean and Tebo, Alison and Piel, Matthieu and Jullien, Ludovic and Robert, Lydia and Le Saux, Thomas and Espagne, Agathe},
  doi          = {10.1038/s41587-023-01893-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {872-876},
  shortjournal = {Nature Biotech.},
  title        = {Near-infrared co-illumination of fluorescent proteins reduces photobleaching and phototoxicity},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The challenges and promise of sweat sensing. <em>NBT</em>,
<em>42</em>(6), 860–871. (<a
href="https://doi.org/10.1038/s41587-023-02059-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The potential of monitoring biomarkers in sweat for health-related applications has spurred rapid growth in the field of wearable sweat sensors over the past decade. Some of the key challenges have been addressed, including measuring sweat-secretion rate and collecting sufficient sample volumes for real-time, continuous molecular analysis without intense exercise. However, except for assessment of cystic fibrosis and regional nerve function, the ability to accurately measure analytes of interest and their physiological relevance to health metrics remain to be determined. Although sweat is not a crystal ball into every aspect of human health, we expect sweat measurements to continue making inroads into niche applications involving active sweating, such as hydration monitoring for athletes and physical laborers and later for medical and casual health monitoring of relevant drugs and hormones. A comprehensive look at wearable sweat sensors defines their capabilities.},
  archive      = {J_NBT},
  author       = {Davis, Noelle and Heikenfeld, Jason and Milla, Carlos and Javey, Ali},
  doi          = {10.1038/s41587-023-02059-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {860-871},
  shortjournal = {Nature Biotech.},
  title        = {The challenges and promise of sweat sensing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Machine learning predicts cellular response to genetic
perturbation. <em>NBT</em>, <em>42</em>(6), 858–859. (<a
href="https://doi.org/10.1038/s41587-023-01907-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GEARS, a machine learning model informed by biological knowledge of gene–gene relationships, effectively predicts transcriptional responses to multi-gene perturbations. GEARS can predict the effects of perturbing previously unperturbed genes and detects non-additive interactions, such as synergy, when predicting combinatorial perturbation outcomes. Thus, GEARS expands insights gained from perturbational screens.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01907-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {858-859},
  shortjournal = {Nature Biotech.},
  title        = {Machine learning predicts cellular response to genetic perturbation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Benchmarking of single-cell ATAC sequencing tools.
<em>NBT</em>, <em>42</em>(6), 856–857. (<a
href="https://doi.org/10.1038/s41587-023-01897-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A systematic examination of eight different single-cell assay for transposase-accessible chromatin by sequencing (scATAC-seq) technologies revealed marked differences in the complexity of sequencing libraries and the specificity of DNA tagmentation that they achieve. Our pipeline for universal mapping of scATAC-seq data (PUMATAC) allowed a fair benchmarking of existing methods and enables the seamless integration of future datasets and technologies.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01897-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {856-857},
  shortjournal = {Nature Biotech.},
  title        = {Benchmarking of single-cell ATAC sequencing tools},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Germ-free humanized mice reveal a crucial role for the gut
microbiota in HIV and EBV pathogenesis. <em>NBT</em>, <em>42</em>(6),
854–855. (<a href="https://doi.org/10.1038/s41587-023-01908-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directly analyzing the role of the gut microbiota during the course of infection with human-specific pathogens is not possible with existing methods. To overcome this problem, we developed a germ-free precision mouse model, which we used to compare the acquisition, replication and pathogenesis of HIV and Epstein–Barr virus.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01908-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {854-855},
  shortjournal = {Nature Biotech.},
  title        = {Germ-free humanized mice reveal a crucial role for the gut microbiota in HIV and EBV pathogenesis},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Wireless microelectronic fibers to discover gut and brain
pathways. <em>NBT</em>, <em>42</em>(6), 852–853. (<a
href="https://doi.org/10.1038/s41587-023-01850-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created miniature and flexible polymer fibers equipped with optoelectronic microdevices, microfluidics and electrodes that can be implanted in anatomically disparate organs such as the brain and the intestine. The microelectronic fibers can be operated wirelessly to sense and manipulate brain and gut neural circuits in untethered, behaving mice.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01850-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {852-853},
  shortjournal = {Nature Biotech.},
  title        = {Wireless microelectronic fibers to discover gut and brain pathways},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genotoxicity concerns for base and prime editors in
hematopoietic stem cells. <em>NBT</em>, <em>42</em>(6), 850–851. (<a
href="https://doi.org/10.1038/s41587-023-01916-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although base and prime editors can be highly efficient in human hematopoietic stem cells, we find they can cause adverse cellular responses, including reduced engraftment and the generation of DNA double-strand breaks and genotoxic byproducts, albeit at a lower frequency than Cas9. We also find that base editors increase the genome-wide mutagenic load.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01916-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {850-851},
  shortjournal = {Nature Biotech.},
  title        = {Genotoxicity concerns for base and prime editors in hematopoietic stem cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Forum: Anikeeva, courtine and bloch. <em>NBT</em>,
<em>42</em>(6), 849. (<a
href="https://doi.org/10.1038/s41587-024-02294-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02294-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {849},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Anikeeva, courtine and bloch},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024t). Engineered yeast produces the plant-based vaccine adjuvant
QS-21. <em>NBT</em>, <em>42</em>(6), 847. (<a
href="https://doi.org/10.1038/s41587-024-02288-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02288-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {847},
  shortjournal = {Nature Biotech.},
  title        = {Engineered yeast produces the plant-based vaccine adjuvant QS-21},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). A vaccine strategy for inducing broadly neutralizing
antibodies against HIV. <em>NBT</em>, <em>42</em>(6), 847. (<a
href="https://doi.org/10.1038/s41587-024-02289-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02289-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {847},
  shortjournal = {Nature Biotech.},
  title        = {A vaccine strategy for inducing broadly neutralizing antibodies against HIV},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024ac). OpenFold provides insights into AlphaFold2’s learning
behavior. <em>NBT</em>, <em>42</em>(6), 847. (<a
href="https://doi.org/10.1038/s41587-024-02290-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02290-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {847},
  shortjournal = {Nature Biotech.},
  title        = {OpenFold provides insights into AlphaFold2’s learning behavior},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cancer diagnostics. <em>NBT</em>, <em>42</em>(6), 846. (<a
href="https://doi.org/10.1038/s41587-024-02269-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to systems and methods for cancer detection and diagnosis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02269-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {846},
  shortjournal = {Nature Biotech.},
  title        = {Cancer diagnostics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The sufficiency of disclosure of medical artificial
intelligence patents. <em>NBT</em>, <em>42</em>(6), 839–845. (<a
href="https://doi.org/10.1038/s41587-024-02270-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of medical artificial intelligence patents reveals substantial gaps in disclosure, underscoring the need for improved examination guidelines to ensure AI patent specifications contain enabling disclosures.},
  archive      = {J_NBT},
  author       = {Aboy, Mateo and Price, W. Nicholson and Raker, Seth and Liddell, Kathleen},
  doi          = {10.1038/s41587-024-02270-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {839-845},
  shortjournal = {Nature Biotech.},
  title        = {The sufficiency of disclosure of medical artificial intelligence patents},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Should neurotechnologies go into the sandbox? <em>NBT</em>,
<em>42</em>(6), 837–838. (<a
href="https://doi.org/10.1038/s41587-024-02254-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Johnson, Walter G.},
  doi          = {10.1038/s41587-024-02254-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {837-838},
  shortjournal = {Nature Biotech.},
  title        = {Should neurotechnologies go into the sandbox?},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Response to “the perpetual motion machine of AI-generated
data and the distraction of ChatGPT as a ‘scientist’.” <em>NBT</em>,
<em>42</em>(6), 835–836. (<a
href="https://doi.org/10.1038/s41587-024-02230-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Noble, William Stafford},
  doi          = {10.1038/s41587-024-02230-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {835-836},
  shortjournal = {Nature Biotech.},
  title        = {Response to “The perpetual motion machine of AI-generated data and the distraction of ChatGPT as a ‘scientist’”},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(6), 830. (<a
href="https://doi.org/10.1038/s41587-024-02287-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02287-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {830},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Gene therapy restores hearing. <em>NBT</em>, <em>42</em>(6),
829. (<a href="https://doi.org/10.1038/s41587-024-02292-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02292-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {829},
  shortjournal = {Nature Biotech.},
  title        = {Gene therapy restores hearing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Next-generation psychedelics: Should new agents skip the
trip? <em>NBT</em>, <em>42</em>(6), 827–830. (<a
href="https://doi.org/10.1038/s41587-024-02285-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies attract venture funding for redesigned psychedelic drugs and notch clinical trial milestones.},
  archive      = {J_NBT},
  author       = {Peplow, Mark},
  doi          = {10.1038/s41587-024-02285-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {827-830},
  shortjournal = {Nature Biotech.},
  title        = {Next-generation psychedelics: Should new agents skip the trip?},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Honey without bees. <em>NBT</em>, <em>42</em>(6), 826. (<a
href="https://doi.org/10.1038/s41587-024-02279-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-024-02279-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {826},
  shortjournal = {Nature Biotech.},
  title        = {Honey without bees},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bird flu in wastewater. <em>NBT</em>, <em>42</em>(6), 825.
(<a href="https://doi.org/10.1038/s41587-024-02297-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02297-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {825},
  shortjournal = {Nature Biotech.},
  title        = {Bird flu in wastewater},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). First-in-class t cell engager approved for lung cancer.
<em>NBT</em>, <em>42</em>(6), 824. (<a
href="https://doi.org/10.1038/s41587-024-02291-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02291-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {824},
  shortjournal = {Nature Biotech.},
  title        = {First-in-class t cell engager approved for lung cancer},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). B cells as drug factories. <em>NBT</em>, <em>42</em>(6),
823–826. (<a href="https://doi.org/10.1038/s41587-024-02283-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are engineering B cells to produce gene-encoded medicines, but clinical proof of concept is needed to convince the doubters that this platform can deliver.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-024-02283-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {823-826},
  shortjournal = {Nature Biotech.},
  title        = {B cells as drug factories},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cis-editing for all. <em>NBT</em>, <em>42</em>(6), 821. (<a
href="https://doi.org/10.1038/s41587-024-02299-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR technologies offer new opportunities to democratize gene-edited crops and promote global sustainability.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02299-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {821},
  shortjournal = {Nature Biotech.},
  title        = {Cis-editing for all},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Femtechs take on women’s health. <em>NBT</em>,
<em>42</em>(6), 831–834. (<a
href="https://doi.org/10.1038/s41587-024-02272-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An upsurge in biotechs focused on women’s health responds to a fresh understanding of female biology to develop new tools and treatments.},
  archive      = {J_NBT},
  author       = {Johnson, Ben},
  doi          = {10.1038/s41587-024-02272-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {831-834},
  shortjournal = {Nature Biotech.},
  title        = {Femtechs take on women’s health},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). People. <em>NBT</em>, <em>42</em>(5), 820. (<a
href="https://doi.org/10.1038/s41587-024-02237-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02237-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {820},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The impact of gender diversity on junior versus senior
biomedical scientists’ NIH research awards. <em>NBT</em>,
<em>42</em>(5), 815–819. (<a
href="https://doi.org/10.1038/s41587-024-02234-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unearthing within-group inequality in science funding and careers highlights a new mechanism explaining the disadvantages faced by younger generations of women scientists.},
  archive      = {J_NBT},
  author       = {Liu, Christopher C. and Yalcinkaya, Beril and Back, Andy S. and Ding, Waverly W.},
  doi          = {10.1038/s41587-024-02234-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {815-819},
  shortjournal = {Nature Biotech.},
  title        = {The impact of gender diversity on junior versus senior biomedical scientists’ NIH research awards},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Generation of accurate, expandable
phylogenomic trees with uDance. <em>NBT</em>, <em>42</em>(5), 814. (<a
href="https://doi.org/10.1038/s41587-023-02027-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Balaban, Metin and Jiang, Yueyu and Zhu, Qiyun and McDonald, Daniel and Knight, Rob and Mirarab, Siavash},
  doi          = {10.1038/s41587-023-02027-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {814},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Generation of accurate, expandable phylogenomic trees with uDance},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Greengenes2 unifies microbial data in a
single reference tree. <em>NBT</em>, <em>42</em>(5), 813. (<a
href="https://doi.org/10.1038/s41587-023-02026-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {McDonald, Daniel and Jiang, Yueyu and Balaban, Metin and Cantrell, Kalen and Zhu, Qiyun and Gonzalez, Antonio and Morton, James T. and Nicolaou, Giorgia and Parks, Donovan H. and Karst, Søren M. and Albertsen, Mads and Hugenholtz, Philip and DeSantis, Todd and Song, Se Jin and Bartko, Andrew and Havulinna, Aki S. and Jousilahti, Pekka and Cheng, Susan and Inouye, Michael and Niiranen, Teemu and Jain, Mohit and Salomaa, Veikko and Lahti, Leo and Mirarab, Siavash and Knight, Rob},
  doi          = {10.1038/s41587-023-02026-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {813},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Greengenes2 unifies microbial data in a single reference tree},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-nucleotide variant calling in single-cell sequencing
data with monopogen. <em>NBT</em>, <em>42</em>(5), 803–812. (<a
href="https://doi.org/10.1038/s41587-023-01873-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell omics technologies enable molecular characterization of diverse cell types and states, but how the resulting transcriptional and epigenetic profiles depend on the cell’s genetic background remains understudied. We describe Monopogen, a computational tool to detect single-nucleotide variants (SNVs) from single-cell sequencing data. Monopogen leverages linkage disequilibrium from external reference panels to identify germline SNVs and detects putative somatic SNVs using allele cosegregating patterns at the cell population level. It can identify 100 K to 3 M germline SNVs achieving a genotyping accuracy of 95%, together with hundreds of putative somatic SNVs. Monopogen-derived genotypes enable global and local ancestry inference and identification of admixed samples. It identifies variants associated with cardiomyocyte metabolic levels and epigenomic programs. It also improves putative somatic SNV detection that enables clonal lineage tracing in primary human clonal hematopoiesis. Monopogen brings together population genetics, cell lineage tracing and single-cell omics to uncover genetic determinants of cellular processes. Monopogen identifies single-nucleotide variants in single-cell sequencing data.},
  archive      = {J_NBT},
  author       = {Dou, Jinzhuang and Tan, Yukun and Kock, Kian Hong and Wang, Jun and Cheng, Xuesen and Tan, Le Min and Han, Kyung Yeon and Hon, Chung-Chau and Park, Woong-Yang and Shin, Jay W. and Jin, Haijing and Wang, Yujia and Chen, Han and Ding, Li and Prabhakar, Shyam and Navin, Nicholas and Chen, Rui and Chen, Ken},
  doi          = {10.1038/s41587-023-01873-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {803-812},
  shortjournal = {Nature Biotech.},
  title        = {Single-nucleotide variant calling in single-cell sequencing data with monopogen},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BugSigDB captures patterns of differential abundance across
a broad range of host-associated microbial signatures. <em>NBT</em>,
<em>42</em>(5), 790–802. (<a
href="https://doi.org/10.1038/s41587-023-01872-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The literature of human and other host-associated microbiome studies is expanding rapidly, but systematic comparisons among published results of host-associated microbiome signatures of differential abundance remain difficult. We present BugSigDB, a community-editable database of manually curated microbial signatures from published differential abundance studies accompanied by information on study geography, health outcomes, host body site and experimental, epidemiological and statistical methods using controlled vocabulary. The initial release of the database contains &amp;gt;2,500 manually curated signatures from &amp;gt;600 published studies on three host species, enabling high-throughput analysis of signature similarity, taxon enrichment, co-occurrence and coexclusion and consensus signatures. These data allow assessment of microbiome differential abundance within and across experimental conditions, environments or body sites. Database-wide analysis reveals experimental conditions with the highest level of consistency in signatures reported by independent studies and identifies commonalities among disease-associated signatures, including frequent introgression of oral pathobionts into the gut. A database of microbial signatures is used for systematic comparison of differential abundance patterns.},
  archive      = {J_NBT},
  author       = {Geistlinger, Ludwig and Mirzayi, Chloe and Zohra, Fatima and Azhar, Rimsha and Elsafoury, Shaimaa and Grieve, Clare and Wokaty, Jennifer and Gamboa-Tuz, Samuel David and Sengupta, Pratyay and Hecht, Issac and Ravikrishnan, Aarthi and Gonçalves, Rafael S. and Franzosa, Eric and Raman, Karthik and Carey, Vincent and Dowd, Jennifer B. and Jones, Heidi E. and Davis, Sean and Segata, Nicola and Huttenhower, Curtis and Waldron, Levi},
  doi          = {10.1038/s41587-023-01872-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {790-802},
  shortjournal = {Nature Biotech.},
  title        = {BugSigDB captures patterns of differential abundance across a broad range of host-associated microbial signatures},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). PhyloVelo enhances transcriptomic velocity field mapping
using monotonically expressed genes. <em>NBT</em>, <em>42</em>(5),
778–789. (<a href="https://doi.org/10.1038/s41587-023-01887-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing (scRNA-seq) is a powerful approach for studying cellular differentiation, but accurately tracking cell fate transitions can be challenging, especially in disease conditions. Here we introduce PhyloVelo, a computational framework that estimates the velocity of transcriptomic dynamics by using monotonically expressed genes (MEGs) or genes with expression patterns that either increase or decrease, but do not cycle, through phylogenetic time. Through integration of scRNA-seq data with lineage information, PhyloVelo identifies MEGs and reconstructs a transcriptomic velocity field. We validate PhyloVelo using simulated data and Caenorhabditis elegans ground truth data, successfully recovering linear, bifurcated and convergent differentiations. Applying PhyloVelo to seven lineage-traced scRNA-seq datasets, generated using CRISPR–Cas9 editing, lentiviral barcoding or immune repertoire profiling, demonstrates its high accuracy and robustness in inferring complex lineage trajectories while outperforming RNA velocity. Additionally, we discovered that MEGs across tissues and organisms share similar functions in translation and ribosome biogenesis. A refined velocity model improves cell fate mapping with lineage-traced scRNA-seq data.},
  archive      = {J_NBT},
  author       = {Wang, Kun and Hou, Liangzhen and Wang, Xin and Zhai, Xiangwei and Lu, Zhaolian and Zi, Zhike and Zhai, Weiwei and He, Xionglei and Curtis, Christina and Zhou, Da and Hu, Zheng},
  doi          = {10.1038/s41587-023-01887-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {778-789},
  shortjournal = {Nature Biotech.},
  title        = {PhyloVelo enhances transcriptomic velocity field mapping using monotonically expressed genes},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Generation of accurate, expandable phylogenomic trees with
uDance. <em>NBT</em>, <em>42</em>(5), 768–777. (<a
href="https://doi.org/10.1038/s41587-023-01868-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phylogenetic trees provide a framework for organizing evolutionary histories across the tree of life and aid downstream comparative analyses such as metagenomic identification. Methods that rely on single-marker genes such as 16S rRNA have produced trees of limited accuracy with hundreds of thousands of organisms, whereas methods that use genome-wide data are not scalable to large numbers of genomes. We introduce updating trees using divide-and-conquer (uDance), a method that enables updatable genome-wide inference using a divide-and-conquer strategy that refines different parts of the tree independently and can build off of existing trees, with high accuracy and scalability. With uDance, we infer a species tree of roughly 200,000 genomes using 387 marker genes, totaling 42.5 billion amino acid residues. Large, updatable phylogenetic trees are constructed using a divide-and-conquer strategy.},
  archive      = {J_NBT},
  author       = {Balaban, Metin and Jiang, Yueyu and Zhu, Qiyun and McDonald, Daniel and Knight, Rob and Mirarab, Siavash},
  doi          = {10.1038/s41587-023-01868-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {768-777},
  shortjournal = {Nature Biotech.},
  title        = {Generation of accurate, expandable phylogenomic trees with uDance},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). De novo detection of somatic mutations in high-throughput
single-cell profiling data sets. <em>NBT</em>, <em>42</em>(5), 758–767.
(<a href="https://doi.org/10.1038/s41587-023-01863-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterization of somatic mutations at single-cell resolution is essential to study cancer evolution, clonal mosaicism and cell plasticity. Here, we describe SComatic, an algorithm designed for the detection of somatic mutations in single-cell transcriptomic and ATAC-seq (assay for transposase-accessible chromatin sequence) data sets directly without requiring matched bulk or single-cell DNA sequencing data. SComatic distinguishes somatic mutations from polymorphisms, RNA-editing events and artefacts using filters and statistical tests parameterized on non-neoplastic samples. Using &amp;gt;2.6 million single cells from 688 single-cell RNA-seq (scRNA-seq) and single-cell ATAC-seq (scATAC-seq) data sets spanning cancer and non-neoplastic samples, we show that SComatic detects mutations in single cells accurately, even in differentiated cells from polyclonal tissues that are not amenable to mutation detection using existing methods. Validated against matched genome sequencing and scRNA-seq data, SComatic achieves F1 scores between 0.6 and 0.7 across diverse data sets, in comparison to 0.2–0.4 for the second-best performing method. In summary, SComatic permits de novo mutational signature analysis, and the study of clonal heterogeneity and mutational burdens at single-cell resolution. SComatic identifies somatic mutations from scRNA-seq and scATAC data without a matched reference sample.},
  archive      = {J_NBT},
  author       = {Muyas, Francesc and Sauer, Carolin M. and Valle-Inclán, Jose Espejo and Li, Ruoyan and Rahbari, Raheleh and Mitchell, Thomas J. and Hormoz, Sahand and Cortés-Ciriano, Isidro},
  doi          = {10.1038/s41587-023-01863-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {758-767},
  shortjournal = {Nature Biotech.},
  title        = {De novo detection of somatic mutations in high-throughput single-cell profiling data sets},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Evolutionary mining and functional characterization of TnpB
nucleases identify efficient miniature genome editors. <em>NBT</em>,
<em>42</em>(5), 745–757. (<a
href="https://doi.org/10.1038/s41587-023-01857-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As the evolutionary ancestor of Cas12 nuclease, the transposon (IS200/IS605)-encoded TnpB proteins act as compact RNA-guided DNA endonucleases. To explore their evolutionary diversity and potential as genome editors, we screened TnpBs from 64 annotated IS605 members and identified 25 active in Escherichia coli, of which three are active in human cells. Further characterization of these 25 TnpBs enables prediction of the transposon-associated motif (TAM) and the right-end element RNA (reRNA) directly from genomic sequences. We established a framework for annotating TnpB systems in prokaryotic genomes and applied it to identify 14 additional candidates. Among these, ISAam1 (369 amino acids (aa)) and ISYmu1 (382 aa) TnpBs demonstrated robust editing activity across dozens of genomic loci in human cells. Both RNA-guided genome editors demonstrated similar editing efficiency as SaCas9 (1,053 aa) while being substantially smaller. The enormous diversity of TnpBs holds potential for the discovery of additional valuable genome editors. Compact genome editors were discovered by mining a transposon library.},
  archive      = {J_NBT},
  author       = {Xiang, Guanghai and Li, Yuanqing and Sun, Jing and Huo, Yongyuan and Cao, Shiwei and Cao, Yuanwei and Guo, Yanyan and Yang, Ling and Cai, Yujia and Zhang, Yong E. and Wang, Haoyi},
  doi          = {10.1038/s41587-023-01857-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {745-757},
  shortjournal = {Nature Biotech.},
  title        = {Evolutionary mining and functional characterization of TnpB nucleases identify efficient miniature genome editors},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-efficiency transgene integration by homology-directed
repair in human primary cells using DNA-PKcs inhibition. <em>NBT</em>,
<em>42</em>(5), 731–744. (<a
href="https://doi.org/10.1038/s41587-023-01888-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Therapeutic applications of nuclease-based genome editing would benefit from improved methods for transgene integration via homology-directed repair (HDR). To improve HDR efficiency, we screened six small-molecule inhibitors of DNA-dependent protein kinase catalytic subunit (DNA-PKcs), a key protein in the alternative repair pathway of non-homologous end joining (NHEJ), which generates genomic insertions/deletions (INDELs). From this screen, we identified AZD7648 as the most potent compound. The use of AZD7648 significantly increased HDR (up to 50-fold) and concomitantly decreased INDELs across different genomic loci in various therapeutically relevant primary human cell types. In all cases, the ratio of HDR to INDELs markedly increased, and, in certain situations, INDEL-free high-frequency (&amp;gt;50%) targeted integration was achieved. This approach has the potential to improve the therapeutic efficacy of cell-based therapies and broaden the use of targeted integration as a research tool. A small molecule enhances targeted gene integration at therapeutically relevant loci in human primary cells.},
  archive      = {J_NBT},
  author       = {Selvaraj, Sridhar and Feist, William N. and Viel, Sebastien and Vaidyanathan, Sriram and Dudek, Amanda M. and Gastou, Marc and Rockwood, Sarah J. and Ekman, Freja K. and Oseghale, Aluya R. and Xu, Liwen and Pavel-Dinu, Mara and Luna, Sofia E. and Cromer, M. Kyle and Sayana, Ruhi and Gomez-Ospina, Natalia and Porteus, Matthew H.},
  doi          = {10.1038/s41587-023-01888-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {731-744},
  shortjournal = {Nature Biotech.},
  title        = {High-efficiency transgene integration by homology-directed repair in human primary cells using DNA-PKcs inhibition},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Young glial progenitor cells competitively replace aged and
diseased human glia in the adult chimeric mouse brain. <em>NBT</em>,
<em>42</em>(5), 719–730. (<a
href="https://doi.org/10.1038/s41587-023-01798-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Competition among adult brain cells has not been extensively researched. To investigate whether healthy glia can outcompete diseased human glia in the adult forebrain, we engrafted wild-type (WT) human glial progenitor cells (hGPCs) produced from human embryonic stem cells into the striata of adult mice that had been neonatally chimerized with mutant Huntingtin (mHTT)-expressing hGPCs. The WT hGPCs outcompeted and ultimately eliminated their human Huntington’s disease (HD) counterparts, repopulating the host striata with healthy glia. Single-cell RNA sequencing revealed that WT hGPCs acquired a YAP1/MYC/E2F-defined dominant competitor phenotype upon interaction with the host HD glia. WT hGPCs also outcompeted older resident isogenic WT cells that had been transplanted neonatally, suggesting that competitive success depended primarily on the relative ages of competing populations, rather than on the presence of mHTT. These data indicate that aged and diseased human glia may be broadly replaced in adult brain by younger healthy hGPCs, suggesting a therapeutic strategy for the replacement of aged and diseased human glia. Old or diseased human glial cells in the chimeric mouse brain are eliminated by transplanted young cells.},
  archive      = {J_NBT},
  author       = {Vieira, Ricardo and Mariani, John N. and Huynh, Nguyen P. T. and Stephensen, Hans J. T. and Solly, Renee and Tate, Ashley and Schanz, Steven and Cotrupi, Natasha and Mousaei, Marzieh and Sporring, Jon and Benraiss, Abdellatif and Goldman, Steven A.},
  doi          = {10.1038/s41587-023-01798-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {719-730},
  shortjournal = {Nature Biotech.},
  title        = {Young glial progenitor cells competitively replace aged and diseased human glia in the adult chimeric mouse brain},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Greengenes2 unifies microbial data in a single reference
tree. <em>NBT</em>, <em>42</em>(5), 715–718. (<a
href="https://doi.org/10.1038/s41587-023-01845-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies using 16S rRNA and shotgun metagenomics typically yield different results, usually attributed to PCR amplification biases. We introduce Greengenes2, a reference tree that unifies genomic and 16S rRNA databases in a consistent, integrated resource. By inserting sequences into a whole-genome phylogeny, we show that 16S rRNA and shotgun metagenomic data generated from the same samples agree in principal coordinates space, taxonomy and phenotype effect size when analyzed with the same tree. A comprehensive microbial resource reconciles genomic and 16S rRNA data in a single tree.},
  archive      = {J_NBT},
  author       = {McDonald, Daniel and Jiang, Yueyu and Balaban, Metin and Cantrell, Kalen and Zhu, Qiyun and Gonzalez, Antonio and Morton, James T. and Nicolaou, Giorgia and Parks, Donovan H. and Karst, Søren M. and Albertsen, Mads and Hugenholtz, Philip and DeSantis, Todd and Song, Se Jin and Bartko, Andrew and Havulinna, Aki S. and Jousilahti, Pekka and Cheng, Susan and Inouye, Michael and Niiranen, Teemu and Jain, Mohit and Salomaa, Veikko and Lahti, Leo and Mirarab, Siavash and Knight, Rob},
  doi          = {10.1038/s41587-023-01845-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {715-718},
  shortjournal = {Nature Biotech.},
  title        = {Greengenes2 unifies microbial data in a single reference tree},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Detection of phosphorylation post-translational
modifications along single peptides with nanopores. <em>NBT</em>,
<em>42</em>(5), 710–714. (<a
href="https://doi.org/10.1038/s41587-023-01839-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods to detect post-translational modifications of proteins, such as phosphate groups, cannot measure single molecules or differentiate between closely spaced phosphorylation sites. We detect post-translational modifications at the single-molecule level on immunopeptide sequences with cancer-associated phosphate variants by controllably drawing the peptide through the sensing region of a nanopore. We discriminate peptide sequences with one or two closely spaced phosphates with 95% accuracy for individual reads of single molecules. Drawing peptides slowly through a nanopore distinguishes individual phosphorylation sites.},
  archive      = {J_NBT},
  author       = {Nova, Ian C. and Ritmejeris, Justas and Brinkerhoff, Henry and Koenig, Theo J. R. and Gundlach, Jens H. and Dekker, Cees},
  doi          = {10.1038/s41587-023-01839-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {710-714},
  shortjournal = {Nature Biotech.},
  title        = {Detection of phosphorylation post-translational modifications along single peptides with nanopores},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BugSigDB — a database for identifying unusual abundance
patterns in human microbiome studies. <em>NBT</em>, <em>42</em>(5),
708–709. (<a href="https://doi.org/10.1038/s41587-023-01930-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {BugSigDB is a community-editable wiki that harmonizes how key microbial differential abundance methods and results are reported, identifying rare and common patterns across the literature of published host-associated microbiome studies.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01930-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {708-709},
  shortjournal = {Nature Biotech.},
  title        = {BugSigDB — A database for identifying unusual abundance patterns in human microbiome studies},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Detecting somatic mutations in single-cell data sets.
<em>NBT</em>, <em>42</em>(5), 706–707. (<a
href="https://doi.org/10.1038/s41587-023-01883-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an algorithm, SComatic, that can be used to directly detect somatic mutations in single-cell data sets without using a reference sample. This method opens the possibility of studying clonal relationships among cells, mutational processes at single-cell resolution, and the impact of somatic mutations on cell function in development and disease.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01883-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {706-707},
  shortjournal = {Nature Biotech.},
  title        = {Detecting somatic mutations in single-cell data sets},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-throughput screening of TnpB proteins identifies
hypercompact genome editors. <em>NBT</em>, <em>42</em>(5), 704–705. (<a
href="https://doi.org/10.1038/s41587-023-01858-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {TnpB proteins are hypercompact RNA-guided DNA endonucleases. By systematically mining and characterizing TnpB proteins from the IS605 family, we identified potent genome editors and established a framework for high-throughput annotation and screening of TnpB systems in prokaryotic genomes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01858-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {704-705},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput screening of TnpB proteins identifies hypercompact genome editors},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). From bench to podcast: Marcus walker. <em>NBT</em>,
<em>42</em>(5), 703. (<a
href="https://doi.org/10.1038/s41587-024-02260-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02260-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {703},
  shortjournal = {Nature Biotech.},
  title        = {From bench to podcast: Marcus walker},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). From bench to podcast: He, jin and shi. <em>NBT</em>,
<em>42</em>(5), 703. (<a
href="https://doi.org/10.1038/s41587-024-02261-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02261-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {703},
  shortjournal = {Nature Biotech.},
  title        = {From bench to podcast: He, jin and shi},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Survival of the fittest glia. <em>NBT</em>, <em>42</em>(5),
700–702. (<a href="https://doi.org/10.1038/s41587-023-01944-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human glial progenitors transplanted into a chimeric mouse brain replace sick or older human glia, a finding that could one day lead to new treatments for neurological disease.},
  archive      = {J_NBT},
  author       = {Moser, V. Alexandra and Svendsen, Clive N.},
  doi          = {10.1038/s41587-023-01944-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {700-702},
  shortjournal = {Nature Biotech.},
  title        = {Survival of the fittest glia},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024v). FOXO1 enhances CAR t cell fitness and function.
<em>NBT</em>, <em>42</em>(5), 699. (<a
href="https://doi.org/10.1038/s41587-024-02257-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02257-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {699},
  shortjournal = {Nature Biotech.},
  title        = {FOXO1 enhances CAR t cell fitness and function},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). A genetic rescue strategy for timothy syndrome.
<em>NBT</em>, <em>42</em>(5), 699. (<a
href="https://doi.org/10.1038/s41587-024-02258-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02258-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {699},
  shortjournal = {Nature Biotech.},
  title        = {A genetic rescue strategy for timothy syndrome},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024x). Gut microbes may influence cardiovascular health.
<em>NBT</em>, <em>42</em>(5), 699. (<a
href="https://doi.org/10.1038/s41587-024-02259-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02259-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {699},
  shortjournal = {Nature Biotech.},
  title        = {Gut microbes may influence cardiovascular health},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Nanopore sequencing. <em>NBT</em>, <em>42</em>(5), 698. (<a
href="https://doi.org/10.1038/s41587-024-02231-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to devices and methods for the improvement of nanopore sequencing systems.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02231-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {698},
  shortjournal = {Nature Biotech.},
  title        = {Nanopore sequencing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Overcoming unpredictability in biotech patents by tailored
claiming aligned with genuine enablement. <em>NBT</em>, <em>42</em>(5),
695–697. (<a href="https://doi.org/10.1038/s41587-024-02228-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The unpredictability inherent to biological systems poses singular obstacles for biotechnology patents regarding the satisfaction of the enablement and written description requirements.},
  archive      = {J_NBT},
  author       = {Ranbhor, Ranjit S.},
  doi          = {10.1038/s41587-024-02228-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {695-697},
  shortjournal = {Nature Biotech.},
  title        = {Overcoming unpredictability in biotech patents by tailored claiming aligned with genuine enablement},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Why japan lacks a vibrant biotech industry. <em>NBT</em>,
<em>42</em>(5), 694. (<a
href="https://doi.org/10.1038/s41587-024-02227-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kessel, Mark and Vickrey, Chris},
  doi          = {10.1038/s41587-024-02227-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {694},
  shortjournal = {Nature Biotech.},
  title        = {Why japan lacks a vibrant biotech industry},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(5), 693. (<a
href="https://doi.org/10.1038/s41587-024-02247-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02247-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {693},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Industry moves on 3Rs. <em>NBT</em>, <em>42</em>(5), 692.
(<a href="https://doi.org/10.1038/s41587-024-02262-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02262-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {692},
  shortjournal = {Nature Biotech.},
  title        = {Industry moves on 3Rs},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). FDA clears prime editors for testing in humans.
<em>NBT</em>, <em>42</em>(5), 691. (<a
href="https://doi.org/10.1038/s41587-024-02264-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02264-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {691},
  shortjournal = {Nature Biotech.},
  title        = {FDA clears prime editors for testing in humans},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Biopharma foresees a “quantum advantage”: They could be
right. <em>NBT</em>, <em>42</em>(5), 690–692. (<a
href="https://doi.org/10.1038/s41587-024-02251-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are betting on the power of quantum computing to simulate molecular dynamics, solve drug design and even accelerate clinical studies.},
  archive      = {J_NBT},
  author       = {Chandran, Anita},
  doi          = {10.1038/s41587-024-02251-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {690-692},
  shortjournal = {Nature Biotech.},
  title        = {Biopharma foresees a ‘quantum advantage’: They could be right},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Star-studded AI biotech launch. <em>NBT</em>,
<em>42</em>(5), 689. (<a
href="https://doi.org/10.1038/s41587-024-02263-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02263-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {689},
  shortjournal = {Nature Biotech.},
  title        = {Star-studded AI biotech launch},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). A second chance for plant biotechnology in europe.
<em>NBT</em>, <em>42</em>(5), 687–689. (<a
href="https://doi.org/10.1038/s41587-024-02246-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Europe tilts towards gene-edited plants, but progress could be derailed over who owns the patents.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-024-02246-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {687-689},
  shortjournal = {Nature Biotech.},
  title        = {A second chance for plant biotechnology in europe},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An effective MASH drug is good, but biotech can make it
better. <em>NBT</em>, <em>42</em>(5), 685–686. (<a
href="https://doi.org/10.1038/s41587-024-02265-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding this complex disease requires better model systems and large-scale data.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02265-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {685-686},
  shortjournal = {Nature Biotech.},
  title        = {An effective MASH drug is good, but biotech can make it better},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). People. <em>NBT</em>, <em>42</em>(4), 684. (<a
href="https://doi.org/10.1038/s41587-024-02202-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02202-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {684},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Five questions with césar de la fuente. <em>NBT</em>,
<em>42</em>(4), 683. (<a
href="https://doi.org/10.1038/s41587-024-02201-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A pioneer in the emerging fields of AI for antibiotic discovery and molecular de-extinction describes his transdisciplinary background and lifelong passion to understand biology.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02201-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {683},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with césar de la fuente},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Sharing best practices for educational programs on venture
creation and commercialization. <em>NBT</em>, <em>42</em>(4), 675–681.
(<a href="https://doi.org/10.1038/s41587-024-02199-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Longitudinal evaluation of the Biomedical Entrepreneurship Educational Program suggests the program successfully educates and inspires the next generation of life science and biotechnology innovators.},
  archive      = {J_NBT},
  author       = {Eidlisz, Jordan and Hill-Whilton, Zachary and Vizgan, Gabriel and Cobos, Daniel and Chitale, Sadhana and Gillespie, Colleen and Dib, Nabil and Gold-von Simson, Gabrielle},
  doi          = {10.1038/s41587-024-02199-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {675-681},
  shortjournal = {Nature Biotech.},
  title        = {Sharing best practices for educational programs on venture creation and commercialization},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Tumor immunotherapy across MHC barriers
using allogeneic t-cell precursors. <em>NBT</em>, <em>42</em>(4), 674.
(<a href="https://doi.org/10.1038/s41587-024-02205-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zakrzewski, Johannes L. and Suh, David and Markley, John C. and Smith, Odette M. and King, Christopher and Goldberg, Gabrielle L. and Jenq, Robert and Holland, Amanda M. and Grubin, Jeremy and Cabrera-Perez, Javier and Brentjens, Renier J. and Lu, Sydney X. and Rizzuto, Gabrielle and Sant’Angelo, Derek B. and Riviere, Isabelle and Sadelain, Michel and Heller, Glenn and Zúñiga-Pflücker, Juan Carlos and Lu, Chen and van den Brink, Marcel R. M.},
  doi          = {10.1038/s41587-024-02205-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {674},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Tumor immunotherapy across MHC barriers using allogeneic T-cell precursors},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pangenome graph construction from genome alignments with
minigraph-cactus. <em>NBT</em>, <em>42</em>(4), 663–673. (<a
href="https://doi.org/10.1038/s41587-023-01793-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pangenome references address biases of reference genomes by storing a representative set of diverse haplotypes and their alignment, usually as a graph. Alternate alleles determined by variant callers can be used to construct pangenome graphs, but advances in long-read sequencing are leading to widely available, high-quality phased assemblies. Constructing a pangenome graph directly from assemblies, as opposed to variant calls, leverages the graph’s ability to represent variation at different scales. Here we present the Minigraph-Cactus pangenome pipeline, which creates pangenomes directly from whole-genome alignments, and demonstrate its ability to scale to 90 human haplotypes from the Human Pangenome Reference Consortium. The method builds graphs containing all forms of genetic variation while allowing use of current mapping and genotyping tools. We measure the effect of the quality and completeness of reference genomes used for analysis within the pangenomes and show that using the CHM13 reference from the Telomere-to-Telomere Consortium improves the accuracy of our methods. We also demonstrate construction of a Drosophila melanogaster pangenome. Constructing genome graphs directly from genome assemblies overcomes single-reference bias.},
  archive      = {J_NBT},
  author       = {Hickey, Glenn and Monlong, Jean and Ebler, Jana and Novak, Adam M. and Eizenga, Jordan M. and Gao, Yan and Marschall, Tobias and Li, Heng and Paten, Benedict},
  doi          = {10.1038/s41587-023-01793-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {663-673},
  shortjournal = {Nature Biotech.},
  title        = {Pangenome graph construction from genome alignments with minigraph-cactus},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Phage-inclusive profiling of human gut microbiomes with
phanta. <em>NBT</em>, <em>42</em>(4), 651–662. (<a
href="https://doi.org/10.1038/s41587-023-01799-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Due to technical limitations, most gut microbiome studies have focused on prokaryotes, overlooking viruses. Phanta, a virome-inclusive gut microbiome profiling tool, overcomes the limitations of assembly-based viral profiling methods by using customized k-mer-based classification tools and incorporating recently published catalogs of gut viral genomes. Phanta’s optimizations consider the small genome size of viruses, sequence homology with prokaryotes and interactions with other gut microbes. Extensive testing of Phanta on simulated data demonstrates that it quickly and accurately quantifies prokaryotes and viruses. When applied to 245 fecal metagenomes from healthy adults, Phanta identifies ~200 viral species per sample, ~5× more than standard assembly-based methods. We observe a ~2:1 ratio between DNA viruses and bacteria, with higher interindividual variability of the gut virome compared to the gut bacteriome. In another cohort, we observe that Phanta performs equally well on bulk versus virus-enriched metagenomes, making it possible to study prokaryotes and viruses in a single experiment, with a single analysis. Bacteria and viruses are simultaneously quantified in gut metagenomic samples.},
  archive      = {J_NBT},
  author       = {Pinto, Yishay and Chakraborty, Meenakshi and Jain, Navami and Bhatt, Ami S.},
  doi          = {10.1038/s41587-023-01799-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {651-662},
  shortjournal = {Nature Biotech.},
  title        = {Phage-inclusive profiling of human gut microbiomes with phanta},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Adenine transversion editors enable precise, efficient
a•t-to-c•g base editing in mammalian cells and embryos. <em>NBT</em>,
<em>42</em>(4), 638–650. (<a
href="https://doi.org/10.1038/s41587-023-01821-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Base editors have substantial promise in basic research and as therapeutic agents for the correction of pathogenic mutations. The development of adenine transversion editors has posed a particular challenge. Here we report a class of base editors that enable efficient adenine transversion, including precise A•T-to-C•G editing. We found that a fusion of mouse alkyladenine DNA glycosylase (mAAG) with nickase Cas9 and deaminase TadA-8e catalyzed adenosine transversion in specific sequence contexts. Laboratory evolution of mAAG significantly increased A-to-C/T conversion efficiency up to 73% and expanded the targeting scope. Further engineering yielded adenine-to-cytosine base editors (ACBEs), including a high-accuracy ACBE-Q variant, that precisely install A-to-C transversions with minimal Cas9-independent off-targeting effects. ACBEs mediated high-efficiency installation or correction of five pathogenic mutations in mouse embryos and human cell lines. Founder mice showed 44–56% average A-to-C edits and allelic frequencies of up to 100%. Adenosine transversion editors substantially expand the capabilities and possible applications of base editing technology. A base editor for precise adenine transversions is demonstrated in mouse embryos.},
  archive      = {J_NBT},
  author       = {Chen, Liang and Hong, Mengjia and Luan, Changming and Gao, Hongyi and Ru, Gaomeng and Guo, Xinyuan and Zhang, Dujuan and Zhang, Shun and Li, Changwei and Wu, Jun and Randolph, Peyton B. and Sousa, Alexander A. and Qu, Chao and Zhu, Yifan and Guan, Yuting and Wang, Liren and Liu, Mingyao and Feng, Bo and Song, Gaojie and Liu, David R. and Li, Dali},
  doi          = {10.1038/s41587-023-01821-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {638-650},
  shortjournal = {Nature Biotech.},
  title        = {Adenine transversion editors enable precise, efficient A•T-to-C•G base editing in mammalian cells and embryos},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Prediction of on-target and off-target activity of
CRISPR–Cas13d guide RNAs using deep learning. <em>NBT</em>,
<em>42</em>(4), 628–637. (<a
href="https://doi.org/10.1038/s41587-023-01830-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptome engineering applications in living cells with RNA-targeting CRISPR effectors depend on accurate prediction of on-target activity and off-target avoidance. Here we design and test ~200,000 RfxCas13d guide RNAs targeting essential genes in human cells with systematically designed mismatches and insertions and deletions (indels). We find that mismatches and indels have a position- and context-dependent impact on Cas13d activity, and mismatches that result in G–U wobble pairings are better tolerated than other single-base mismatches. Using this large-scale dataset, we train a convolutional neural network that we term targeted inhibition of gene expression via gRNA design (TIGER) to predict efficacy from guide sequence and context. TIGER outperforms the existing models at predicting on-target and off-target activity on our dataset and published datasets. We show that TIGER scoring combined with specific mismatches yields the first general framework to modulate transcript expression, enabling the use of RNA-targeting CRISPRs to precisely control gene dosage. A machine learning model predicts on-target and off-target activity of Cas13d in human cells.},
  archive      = {J_NBT},
  author       = {Wessels, Hans-Hermann and Stirn, Andrew and Méndez-Mancilla, Alejandro and Kim, Eric J. and Hart, Sydney K. and Knowles, David A. and Sanjana, Neville E.},
  doi          = {10.1038/s41587-023-01830-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {628-637},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of on-target and off-target activity of CRISPR–Cas13d guide RNAs using deep learning},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Whole-body cellular mapping in mouse using standard IgG
antibodies. <em>NBT</em>, <em>42</em>(4), 617–627. (<a
href="https://doi.org/10.1038/s41587-023-01846-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-body imaging techniques play a vital role in exploring the interplay of physiological systems in maintaining health and driving disease. We introduce wildDISCO, a new approach for whole-body immunolabeling, optical clearing and imaging in mice, circumventing the need for transgenic reporter animals or nanobody labeling and so overcoming existing technical limitations. We identified heptakis(2,6-di-O-methyl)-β-cyclodextrin as a potent enhancer of cholesterol extraction and membrane permeabilization, enabling deep, homogeneous penetration of standard antibodies without aggregation. WildDISCO facilitates imaging of peripheral nervous systems, lymphatic vessels and immune cells in whole mice at cellular resolution by labeling diverse endogenous proteins. Additionally, we examined rare proliferating cells and the effects of biological perturbations, as demonstrated in germ-free mice. We applied wildDISCO to map tertiary lymphoid structures in the context of breast cancer, considering both primary tumor and metastases throughout the mouse body. An atlas of high-resolution images showcasing mouse nervous, lymphatic and vascular systems is accessible at http://discotechnologies.org/wildDISCO/atlas/index.php . Whole mice are immunolabeled with standard antibodies using wildDISCO.},
  archive      = {J_NBT},
  author       = {Mai, Hongcheng and Luo, Jie and Hoeher, Luciano and Al-Maskari, Rami and Horvath, Izabela and Chen, Ying and Kofler, Florian and Piraud, Marie and Paetzold, Johannes C. and Modamio, Jennifer and Todorov, Mihail and Elsner, Markus and Hellal, Farida and Ertürk, Ali},
  doi          = {10.1038/s41587-023-01846-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {617-627},
  shortjournal = {Nature Biotech.},
  title        = {Whole-body cellular mapping in mouse using standard IgG antibodies},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial imaging of glycoRNA in single cells with ARPLA.
<em>NBT</em>, <em>42</em>(4), 608–616. (<a
href="https://doi.org/10.1038/s41587-023-01801-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Little is known about the biological roles of glycosylated RNAs (glycoRNAs), a recently discovered class of glycosylated molecules, because of a lack of visualization methods. We report sialic acid aptamer and RNA in situ hybridization-mediated proximity ligation assay (ARPLA) to visualize glycoRNAs in single cells with high sensitivity and selectivity. The signal output of ARPLA occurs only when dual recognition of a glycan and an RNA triggers in situ ligation, followed by rolling circle amplification of a complementary DNA, which generates a fluorescent signal by binding fluorophore-labeled oligonucleotides. Using ARPLA, we detect spatial distributions of glycoRNAs on the cell surface and their colocalization with lipid rafts as well as the intracellular trafficking of glycoRNAs through SNARE protein-mediated secretory exocytosis. Studies in breast cell lines suggest that surface glycoRNA is inversely associated with tumor malignancy and metastasis. Investigation of the relationship between glycoRNAs and monocyte–endothelial cell interactions suggests that glycoRNAs may mediate cell–cell interactions during the immune response. GlycoRNA imaging in cells is enabled with a proximity ligation assay.},
  archive      = {J_NBT},
  author       = {Ma, Yuan and Guo, Weijie and Mou, Quanbing and Shao, Xiangli and Lyu, Mingkuan and Garcia, Valeria and Kong, Linggen and Lewis, Whitney and Ward, Carson and Yang, Zhenglin and Pan, Xingxin and Yi, S. Stephen and Lu, Yi},
  doi          = {10.1038/s41587-023-01801-z},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {608-616},
  shortjournal = {Nature Biotech.},
  title        = {Spatial imaging of glycoRNA in single cells with ARPLA},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Design of a mucin-selective protease for targeted
degradation of cancer-associated mucins. <em>NBT</em>, <em>42</em>(4),
597–607. (<a href="https://doi.org/10.1038/s41587-023-01840-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted protein degradation is an emerging strategy for the elimination of classically undruggable proteins. Here, to expand the landscape of targetable substrates, we designed degraders that achieve substrate selectivity via recognition of a discrete peptide and glycan motif and achieve cell-type selectivity via antigen-driven cell-surface binding. We applied this approach to mucins, O-glycosylated proteins that drive cancer progression through biophysical and immunological mechanisms. Engineering of a bacterial mucin-selective protease yielded a variant for fusion to a cancer antigen-binding nanobody. The resulting conjugate selectively degraded mucins on cancer cells, promoted cell death in culture models of mucin-driven growth and survival, and reduced tumor growth in mouse models of breast cancer progression. This work establishes a blueprint for the development of biologics that degrade specific protein glycoforms on target cells. Targeted protein degradation of cell-surface glycoproteins suppresses cancer in mice.},
  archive      = {J_NBT},
  author       = {Pedram, Kayvon and Shon, D. Judy and Tender, Gabrielle S. and Mantuano, Natalia R. and Northey, Jason J. and Metcalf, Kevin J. and Wisnovsky, Simon P. and Riley, Nicholas M. and Forcina, Giovanni C. and Malaker, Stacy A. and Kuo, Angel and George, Benson M. and Miller, Caitlyn L. and Casey, Kerriann M. and Vilches-Moure, José G. and Ferracane, Michael J. and Weaver, Valerie M. and Läubli, Heinz and Bertozzi, Carolyn R.},
  doi          = {10.1038/s41587-023-01840-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {597-607},
  shortjournal = {Nature Biotech.},
  title        = {Design of a mucin-selective protease for targeted degradation of cancer-associated mucins},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell m6A mapping in vivo using picoMeRIP–seq.
<em>NBT</em>, <em>42</em>(4), 591–596. (<a
href="https://doi.org/10.1038/s41587-023-01831-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current N6-methyladenosine (m6A) mapping methods need large amounts of RNA or are limited to cultured cells. Through optimized sample recovery and signal-to-noise ratio, we developed picogram-scale m6A RNA immunoprecipitation and sequencing (picoMeRIP–seq) for studying m6A in vivo in single cells and scarce cell types using standard laboratory equipment. We benchmark m6A mapping on titrations of poly(A) RNA and embryonic stem cells and in single zebrafish zygotes, mouse oocytes and embryos. A picogram-scale m6A mapping method is applied to single mouse oocytes and embryos in vivo.},
  archive      = {J_NBT},
  author       = {Li, Yanjiao and Wang, Yunhao and Vera-Rodriguez, Maria and Lindeman, Leif Christopher and Skuggen, Linda Ellevog and Rasmussen, Erik M. K. and Jermstad, Ingunn and Khan, Shaista and Fosslie, Madeleine and Skuland, Trine and Indahl, Marie and Khodeer, Sherif and Klemsdal, Eva Kristine and Jin, Kang-Xuan and Dalen, Knut Tomas and Fedorcsak, Peter and Greggains, Gareth D. and Lerdrup, Mads and Klungland, Arne and Au, Kin Fai and Dahl, John Arne},
  doi          = {10.1038/s41587-023-01831-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {591-596},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell m6A mapping in vivo using picoMeRIP–seq},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-throughput measurement of the content and properties of
nano-sized bioparticles with single-particle profiler. <em>NBT</em>,
<em>42</em>(4), 587–590. (<a
href="https://doi.org/10.1038/s41587-023-01825-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce a method, single-particle profiler, that provides single-particle information on the content and biophysical properties of thousands of particles in the size range 5–200 nm. We use our single-particle profiler to measure the messenger RNA encapsulation efficiency of lipid nanoparticles, the viral binding efficiencies of different nanobodies, and the biophysical heterogeneity of liposomes, lipoproteins, exosomes and viruses. Lipid nanoparticles, viruses, exosomes and liposomes are characterized by analysis of fluorescence fluctuations.},
  archive      = {J_NBT},
  author       = {Sych, Taras and Schlegel, Jan and Barriga, Hanna M. G. and Ojansivu, Miina and Hanke, Leo and Weber, Florian and Beklem Bostancioglu, R. and Ezzat, Kariem and Stangl, Herbert and Plochberger, Birgit and Laurencikiene, Jurga and El Andaloussi, Samir and Fürth, Daniel and Stevens, Molly M. and Sezgin, Erdinc},
  doi          = {10.1038/s41587-023-01825-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {587-590},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput measurement of the content and properties of nano-sized bioparticles with single-particle profiler},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-throughput RNA isoform sequencing using programmed cDNA
concatenation. <em>NBT</em>, <em>42</em>(4), 582–586. (<a
href="https://doi.org/10.1038/s41587-023-01815-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Full-length RNA-sequencing methods using long-read technologies can capture complete transcript isoforms, but their throughput is limited. We introduce multiplexed arrays isoform sequencing (MAS-ISO-seq), a technique for programmably concatenating complementary DNAs (cDNAs) into molecules optimal for long-read sequencing, increasing the throughput &amp;gt;15-fold to nearly 40 million cDNA reads per run on the Sequel IIe sequencer. When applied to single-cell RNA sequencing of tumor-infiltrating T cells, MAS-ISO-seq demonstrated a 12- to 32-fold increase in the discovery of differentially spliced genes. Programmable concatenation of cDNA molecules increases the throughput of PacBio sequencing about 15-fold.},
  archive      = {J_NBT},
  author       = {Al’Khafaji, Aziz M. and Smith, Jonathan T. and Garimella, Kiran V. and Babadi, Mehrtash and Popic, Victoria and Sade-Feldman, Moshe and Gatzen, Michael and Sarkizova, Siranush and Schwartz, Marc A. and Blaum, Emily M. and Day, Allyson and Costello, Maura and Bowers, Tera and Gabriel, Stacey and Banks, Eric and Philippakis, Anthony A. and Boland, Genevieve M. and Blainey, Paul C. and Hacohen, Nir},
  doi          = {10.1038/s41587-023-01815-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {582-586},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput RNA isoform sequencing using programmed cDNA concatenation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Combining reference genomes into a pangenome graph improves
accuracy and reduces bias. <em>NBT</em>, <em>42</em>(4), 580–581. (<a
href="https://doi.org/10.1038/s41587-023-01828-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Minigraph-Cactus, a method to efficiently combine multiple reference genome assemblies into a pangenome reference graph, can be used to improve accuracy of read mapping and variant calling compared with a single reference in downstream applications.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01828-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {580-581},
  shortjournal = {Nature Biotech.},
  title        = {Combining reference genomes into a pangenome graph improves accuracy and reduces bias},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Efficient a-to-c base editing with high specificity.
<em>NBT</em>, <em>42</em>(4), 578–579. (<a
href="https://doi.org/10.1038/s41587-023-01866-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generating A-to-C transversions in specific targets via base editing technology has been challenging. By fusing an evolved alkyladenine DNA glycosylase with an engineered adenine deaminase TadA-8e variant and nickase Cas9, we have developed A-to-C base editors that generate precise and efficient A-to-C transversions in cells and in mouse embryos, expanding the possible applications of base editing.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01866-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {578-579},
  shortjournal = {Nature Biotech.},
  title        = {Efficient A-to-C base editing with high specificity},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Breaking boundaries in whole-body imaging and disease
understanding with wildDISCO. <em>NBT</em>, <em>42</em>(4), 576–577. (<a
href="https://doi.org/10.1038/s41587-023-01864-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wild three-dimensional imaging of solvent-cleared organs (wildDISCO) uses conventional antibodies for whole-body staining in mice, creating comprehensive biological atlases of nerves, blood vessels and lymphatics. It uncovers pathological changes, such as tertiary lymphoid structures in cancer, and it enables the precise tracking of therapeutic molecules and cells, enhancing our understanding of disease pathology and treatment.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01864-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {576-577},
  shortjournal = {Nature Biotech.},
  title        = {Breaking boundaries in whole-body imaging and disease understanding with wildDISCO},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Imaging glycosylated RNAs at the subcellular scale.
<em>NBT</em>, <em>42</em>(4), 574–575. (<a
href="https://doi.org/10.1038/s41587-023-02021-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recently discovered RNA species on the cell surface is studied by proximity ligation.},
  archive      = {J_NBT},
  author       = {Hristov, Petar and Flynn, Ryan A.},
  doi          = {10.1038/s41587-023-02021-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {574-575},
  shortjournal = {Nature Biotech.},
  title        = {Imaging glycosylated RNAs at the subcellular scale},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A mucin degrader for cancer therapy. <em>NBT</em>,
<em>42</em>(4), 572–573. (<a
href="https://doi.org/10.1038/s41587-023-01984-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new approach to targeting mucins via protein degradation shows promise against cancer.},
  archive      = {J_NBT},
  author       = {Beatson, Richard and Burchell, Joy M.},
  doi          = {10.1038/s41587-023-01984-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {572-573},
  shortjournal = {Nature Biotech.},
  title        = {A mucin degrader for cancer therapy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024af). RoseTTAFold expands to all-atom for biomolecular
prediction and design. <em>NBT</em>, <em>42</em>(4), 571. (<a
href="https://doi.org/10.1038/s41587-024-02211-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02211-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {571},
  shortjournal = {Nature Biotech.},
  title        = {RoseTTAFold expands to all-atom for biomolecular prediction and design},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024o). Customized molecular glue complexes with desired
properties. <em>NBT</em>, <em>42</em>(4), 571. (<a
href="https://doi.org/10.1038/s41587-024-02212-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02212-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {571},
  shortjournal = {Nature Biotech.},
  title        = {Customized molecular glue complexes with desired properties},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024ab). Mapping the landscape of host–microbiome interactions.
<em>NBT</em>, <em>42</em>(4), 571. (<a
href="https://doi.org/10.1038/s41587-024-02219-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02219-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {571},
  shortjournal = {Nature Biotech.},
  title        = {Mapping the landscape of host–microbiome interactions},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Prime editing. <em>NBT</em>, <em>42</em>(4), 570. (<a
href="https://doi.org/10.1038/s41587-024-02197-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods and compositions for prime editing.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02197-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {570},
  shortjournal = {Nature Biotech.},
  title        = {Prime editing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping the global landscape for induced pluripotent stem
cells from patents and clinical trials. <em>NBT</em>, <em>42</em>(4),
563–569. (<a href="https://doi.org/10.1038/s41587-024-02196-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although the clinical application of induced pluripotent stem cells is still in its infancy, the development of iPSC technologies reflected by an increasing number of patents provides hope that they will realize their therapeutic potential.},
  archive      = {J_NBT},
  author       = {Lyu, Liyang and Feng, Ye and Huang, Borong and Xu, Ren-He and Hu, Yuanjia},
  doi          = {10.1038/s41587-024-02196-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {563-569},
  shortjournal = {Nature Biotech.},
  title        = {Mapping the global landscape for induced pluripotent stem cells from patents and clinical trials},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The potential of DAOs for funding and collaborative
development in the life sciences. <em>NBT</em>, <em>42</em>(4), 555–562.
(<a href="https://doi.org/10.1038/s41587-024-02189-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {VitaDAO funds longevity research through a blockchain-based decentralized autonomous organization (DAO), showcasing the potential of collaborative, transparent and alternative systems while also highlighting the challenges of coordination, regulation, biases and skepticism in reshaping traditional research financing methods.},
  archive      = {J_NBT},
  author       = {Fantaccini, Simone and Grassi, Laura and Rampoldi, Andrea},
  doi          = {10.1038/s41587-024-02189-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {555-562},
  shortjournal = {Nature Biotech.},
  title        = {The potential of DAOs for funding and collaborative development in the life sciences},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Making genome editing a success story in africa.
<em>NBT</em>, <em>42</em>(4), 551–554. (<a
href="https://doi.org/10.1038/s41587-024-02187-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Abkallo, Hussein M. and Arbuthnot, Patrick and Auer, Thomas O. and Berger, Dave K. and Burger, Johan and Chakauya, Ereck and Concordet, Jean-Paul and Diabate, Abdoulaye and Di Donato, Vincenzo and Groenewald, Jan-Hendrik and Guindo, Amadou and Koekemoer, Lizette L. and Nazare, Florence and Nolan, Tony and Okumu, Fredros and Orefuwa, Emma and Paemka, Lily and Prieto-Godino, Lucia and Runo, Steven and Sadler, Marie and Tesfaye, Kassahun and Tripathi, Leena and Wondji, Charles},
  doi          = {10.1038/s41587-024-02187-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {551-554},
  shortjournal = {Nature Biotech.},
  title        = {Making genome editing a success story in africa},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Saving cavendish. <em>NBT</em>, <em>42</em>(4), 545. (<a
href="https://doi.org/10.1038/s41587-024-02206-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-024-02206-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {545},
  shortjournal = {Nature Biotech.},
  title        = {Saving cavendish},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(4), 544. (<a
href="https://doi.org/10.1038/s41587-024-02216-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02216-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {544},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). First gene-edited pig kidney transplant. <em>NBT</em>,
<em>42</em>(4), 543. (<a
href="https://doi.org/10.1038/s41587-024-02223-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02223-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {543},
  shortjournal = {Nature Biotech.},
  title        = {First gene-edited pig kidney transplant},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Prime editing deal flurry to nail down patent rights.
<em>NBT</em>, <em>42</em>(4), 542–544. (<a
href="https://doi.org/10.1038/s41587-024-02208-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prime editing field is booming, with companies making strategic decisions to avoid an IP showdown.},
  archive      = {J_NBT},
  author       = {Arnold, Carrie},
  doi          = {10.1038/s41587-024-02208-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {542-544},
  shortjournal = {Nature Biotech.},
  title        = {Prime editing deal flurry to nail down patent rights},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). World’s priciest drug treats MLD. <em>NBT</em>,
<em>42</em>(4), 541. (<a
href="https://doi.org/10.1038/s41587-024-02217-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02217-z},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {541},
  shortjournal = {Nature Biotech.},
  title        = {World’s priciest drug treats MLD},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). FDA approves first MASH drug. <em>NBT</em>, <em>42</em>(4),
540. (<a href="https://doi.org/10.1038/s41587-024-02220-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02220-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {540},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves first MASH drug},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Startups probe hidden viruses in the “dark genome” to treat
disease. <em>NBT</em>, <em>42</em>(4), 539–541. (<a
href="https://doi.org/10.1038/s41587-024-02215-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug hunters are finding that ancient virus-like artifacts in the human genome could offer new avenues to treat neurodegeneration, cancer, autoimmunity and even aging with antibodies, vaccines and antiretroviral agents.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-024-02215-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {539-541},
  shortjournal = {Nature Biotech.},
  title        = {Startups probe hidden viruses in the ‘dark genome’ to treat disease},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Antivenom slithers back to life. <em>NBT</em>,
<em>42</em>(4), 537–538. (<a
href="https://doi.org/10.1038/s41587-024-02221-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Snakebite treatment still relies on antivenom, a 130-year old technology, but biotech is on the job.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02221-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {537-538},
  shortjournal = {Nature Biotech.},
  title        = {Antivenom slithers back to life},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). The glue degraders. <em>NBT</em>, <em>42</em>(4), 546–550.
(<a href="https://doi.org/10.1038/s41587-024-02164-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are hoping to discover small molecules that remove undruggable proteins. It won’t be easy.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-024-02164-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {4},
  pages        = {546-550},
  shortjournal = {Nature Biotech.},
  title        = {The glue degraders},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024j). People. <em>NBT</em>, <em>42</em>(3), 536. (<a
href="https://doi.org/10.1038/s41587-024-02169-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02169-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {536},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). 2H23 biotech job picture. <em>NBT</em>, <em>42</em>(3),
535. (<a href="https://doi.org/10.1038/s41587-024-02167-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A semiannual snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02167-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {535},
  shortjournal = {Nature Biotech.},
  title        = {2H23 biotech job picture},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Lighting the match: How a student-led career mentorship
program can blaze the way for non-academic careers. <em>NBT</em>,
<em>42</em>(3), 530–534. (<a
href="https://doi.org/10.1038/s41587-024-02170-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Academic institutions can address the demand for multiskilled individuals who will be competitive in the non-academic job market by establishing a strong career mentorship program.},
  archive      = {J_NBT},
  author       = {Bagnell, Anna and MacDonald, Alana and Espenshade, Peter and Schenerman, Mark},
  doi          = {10.1038/s41587-024-02170-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {530-534},
  shortjournal = {Nature Biotech.},
  title        = {Lighting the match: How a student-led career mentorship program can blaze the way for non-academic careers},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Author correction: An engineered influenza virus to deliver
antigens for lung cancer vaccination. <em>NBT</em>, <em>42</em>(3), 529.
(<a href="https://doi.org/10.1038/s41587-023-01884-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ji, Dezhong and Zhang, Yuanjie and Sun, Jiaqi and Zhang, Bo and Ma, Wenxiao and Cheng, Boyang and Wang, Xinchen and Li, Yuanhao and Mu, Yu and Xu, Huan and Wang, Qi and Zhang, Chuanling and Xiao, Sulong and Zhang, Lihe and Zhou, Demin},
  doi          = {10.1038/s41587-023-01884-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {529},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: An engineered influenza virus to deliver antigens for lung cancer vaccination},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Predicting the efficiency of prime
editing guide RNAs in human cells. <em>NBT</em>, <em>42</em>(3), 529.
(<a href="https://doi.org/10.1038/s41587-024-02159-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kim, Hui Kwon and Yu, Goosang and Park, Jinman and Min, Seonwoo and Lee, Sungtae and Yoon, Sungroh and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-024-02159-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {529},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Predicting the efficiency of prime editing guide RNAs in human cells},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). An engineered influenza virus to deliver antigens for lung
cancer vaccination. <em>NBT</em>, <em>42</em>(3), 518–528. (<a
href="https://doi.org/10.1038/s41587-023-01796-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of cancer neoantigen vaccines that prime the anti-tumor immune responses has been hindered in part by challenges in delivery of neoantigens to the tumor. Here, using the model antigen ovalbumin (OVA) in a melanoma model, we demonstrate a chimeric antigenic peptide influenza virus (CAP-Flu) system for delivery of antigenic peptides bound to influenza A virus (IAV) to the lung. We conjugated attenuated IAVs with the innate immunostimulatory agent CpG and, after intranasal administration to the mouse lung, observed increased immune cell infiltration to the tumor. OVA was then covalently displayed on IAV-CPG using click chemistry. Vaccination with this construct yielded robust antigen uptake by dendritic cells, a specific immune cell response and a significant increase in tumor-infiltrating lymphocytes compared to peptides alone. Lastly, we engineered the IAV to express anti-PD1-L1 nanobodies that further enhanced regression of lung metastases and prolonged mouse survival after rechallenge. Engineered IAVs can be equipped with any tumor neoantigen of interest to generate lung cancer vaccines. A cancer vaccine is delivered to the lung by an engineered attenuated influenza virus.},
  archive      = {J_NBT},
  author       = {Ji, Dezhong and Zhang, Yuanjie and Sun, Jiaqi and Zhang, Bo and Ma, Wenxiao and Cheng, Boyang and Wang, Xinchen and Li, Yuanhao and Mu, Yu and Xu, Huan and Wang, Qi and Zhang, Chuanling and Xiao, Sulong and Zhang, Lihe and Zhou, Demin},
  doi          = {10.1038/s41587-023-01796-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {518-528},
  shortjournal = {Nature Biotech.},
  title        = {An engineered influenza virus to deliver antigens for lung cancer vaccination},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A microneedle vaccine printer for thermostable COVID-19 mRNA
vaccines. <em>NBT</em>, <em>42</em>(3), 510–517. (<a
href="https://doi.org/10.1038/s41587-023-01774-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Decentralized manufacture of thermostable mRNA vaccines in a microneedle patch (MNP) format could enhance vaccine access in low-resource communities by eliminating the need for a cold chain and trained healthcare personnel. Here we describe an automated process for printing MNP Coronavirus Disease 2019 (COVID-19) mRNA vaccines in a standalone device. The vaccine ink is composed of lipid nanoparticles loaded with mRNA and a dissolvable polymer blend that was optimized for high bioactivity by screening formulations in vitro. We demonstrate that the resulting MNPs are shelf stable for at least 6 months at room temperature when assessed using a model mRNA construct. Vaccine loading efficiency and microneedle dissolution suggest that efficacious, microgram-scale doses of mRNA encapsulated in lipid nanoparticles could be delivered with a single patch. Immunizations in mice using manually produced MNPs with mRNA encoding severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) spike protein receptor-binding domain stimulate long-term immune responses similar to those of intramuscular administration. Automated fabrication of microneedle patch mRNA vaccines for COVID-19 may improve vaccine access.},
  archive      = {J_NBT},
  author       = {vander Straeten, Aurélien and Sarmadi, Morteza and Daristotle, John L. and Kanelli, Maria and Tostanoski, Lisa H. and Collins, Joe and Pardeshi, Apurva and Han, Jooli and Varshney, Dhruv and Eshaghi, Behnaz and Garcia, Johnny and Forster, Timothy A. and Li, Gary and Menon, Nandita and Pyon, Sydney L. and Zhang, Linzixuan and Jacob-Dolan, Catherine and Powers, Olivia C. and Hall, Kevin and Alsaiari, Shahad K. and Wolf, Morris and Tibbitt, Mark W. and Farra, Robert and Barouch, Dan H. and Langer, Robert and Jaklenec, Ana},
  doi          = {10.1038/s41587-023-01774-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {510-517},
  shortjournal = {Nature Biotech.},
  title        = {A microneedle vaccine printer for thermostable COVID-19 mRNA vaccines},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Strand-selective base editing of human mitochondrial DNA
using mitoBEs. <em>NBT</em>, <em>42</em>(3), 498–509. (<a
href="https://doi.org/10.1038/s41587-023-01791-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A number of mitochondrial diseases in humans are caused by point mutations that could be corrected by base editors, but delivery of CRISPR guide RNAs into the mitochondria is difficult. In this study, we present mitochondrial DNA base editors (mitoBEs), which combine a transcription activator-like effector (TALE)-fused nickase and a deaminase for precise base editing in mitochondrial DNA. Combining mitochondria-localized, programmable TALE binding proteins with the nickase MutH or Nt.BspD6I(C) and either the single-stranded DNA-specific adenine deaminase TadA8e or the cytosine deaminase ABOBEC1 and UGI, we achieve A-to-G or C-to-T base editing with up to 77% efficiency and high specificity. We find that mitoBEs are DNA strand-selective mitochondrial base editors, with editing results more likely to be retained on the nonnicked DNA strand. Furthermore, we correct pathogenic mitochondrial DNA mutations in patient-derived cells by delivering mitoBEs encoded in circular RNAs. mitoBEs offer a precise, efficient DNA editing tool with broad applicability for therapy in mitochondrial genetic diseases. Mitochondrial point mutations are corrected in human cells using a transcription activator-like effector (TALE)-based system.},
  archive      = {J_NBT},
  author       = {Yi, Zongyi and Zhang, Xiaoxue and Tang, Wei and Yu, Ying and Wei, Xiaoxu and Zhang, Xue and Wei, Wensheng},
  doi          = {10.1038/s41587-023-01791-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {498-509},
  shortjournal = {Nature Biotech.},
  title        = {Strand-selective base editing of human mitochondrial DNA using mitoBEs},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep learning models to predict the editing efficiencies and
outcomes of diverse base editors. <em>NBT</em>, <em>42</em>(3), 484–497.
(<a href="https://doi.org/10.1038/s41587-023-01792-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Applications of base editing are frequently restricted by the requirement for a protospacer adjacent motif (PAM), and selecting the optimal base editor (BE) and single-guide RNA pair (sgRNA) for a given target can be difficult. To select for BEs and sgRNAs without extensive experimental work, we systematically compared the editing windows, outcomes and preferred motifs for seven BEs, including two cytosine BEs, two adenine BEs and three C•G to G•C BEs at thousands of target sequences. We also evaluated nine Cas9 variants that recognize different PAM sequences and developed a deep learning model, DeepCas9variants, for predicting which variants function most efficiently at sites with a given target sequence. We then develop a computational model, DeepBE, that predicts editing efficiencies and outcomes of 63 BEs that were generated by incorporating nine Cas9 variants as nickase domains into the seven BE variants. The predicted median efficiencies of BEs with DeepBE-based design were 2.9- to 20-fold higher than those of rationally designed SpCas9-containing BEs. The best base editor for specific applications is predicted with a deep learning model.},
  archive      = {J_NBT},
  author       = {Kim, Nahye and Choi, Sungchul and Kim, Sungjae and Song, Myungjae and Seo, Jung Hwa and Min, Seonwoo and Park, Jinman and Cho, Sung-Rae and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-023-01792-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {484-497},
  shortjournal = {Nature Biotech.},
  title        = {Deep learning models to predict the editing efficiencies and outcomes of diverse base editors},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Comparative analysis of cell–cell communication at
single-cell resolution. <em>NBT</em>, <em>42</em>(3), 470–483. (<a
href="https://doi.org/10.1038/s41587-023-01782-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inference of cell–cell communication from single-cell RNA sequencing data is a powerful technique to uncover intercellular communication pathways, yet existing methods perform this analysis at the level of the cell type or cluster, discarding single-cell-level information. Here we present Scriabin, a flexible and scalable framework for comparative analysis of cell–cell communication at single-cell resolution that is performed without cell aggregation or downsampling. We use multiple published atlas-scale datasets, genetic perturbation screens and direct experimental validation to show that Scriabin accurately recovers expected cell–cell communication edges and identifies communication networks that can be obscured by agglomerative methods. Additionally, we use spatial transcriptomic data to show that Scriabin can uncover spatial features of interaction from dissociated data alone. Finally, we demonstrate applications to longitudinal datasets to follow communication pathways operating between timepoints. Our approach represents a broadly applicable strategy to reveal the full structure of niche–phenotype relationships in health and disease. Cell–cell communication pathways are dissected at the level of individual cells.},
  archive      = {J_NBT},
  author       = {Wilk, Aaron J. and Shalek, Alex K. and Holmes, Susan and Blish, Catherine A.},
  doi          = {10.1038/s41587-023-01782-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {470-483},
  shortjournal = {Nature Biotech.},
  title        = {Comparative analysis of cell–cell communication at single-cell resolution},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A highly efficient transgene knock-in technology in
clinically relevant cell types. <em>NBT</em>, <em>42</em>(3), 458–469.
(<a href="https://doi.org/10.1038/s41587-023-01779-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inefficient knock-in of transgene cargos limits the potential of cell-based medicines. In this study, we used a CRISPR nuclease that targets a site within an exon of an essential gene and designed a cargo template so that correct knock-in would retain essential gene function while also integrating the transgene(s) of interest. Cells with non-productive insertions and deletions would undergo negative selection. This technology, called SLEEK (SeLection by Essential-gene Exon Knock-in), achieved knock-in efficiencies of more than 90% in clinically relevant cell types without impacting long-term viability or expansion. SLEEK knock-in rates in T cells are more efficient than state-of-the-art TRAC knock-in with AAV6 and surpass more than 90% efficiency even with non-viral DNA cargos. As a clinical application, natural killer cells generated from induced pluripotent stem cells containing SLEEK knock-in of CD16 and mbIL-15 show substantially improved tumor killing and persistence in vivo. Transgene knock-ins into housekeeping genes lead to high efficiency of cell selection.},
  archive      = {J_NBT},
  author       = {Allen, Alexander G. and Khan, Samia Q. and Margulies, Carrie M. and Viswanathan, Ramya and Lele, Swarali and Blaha, Laura and Scott, Sean N. and Izzo, Kaitlyn M. and Gerew, Alexandra and Pattali, Rithu and Cochran, Nadire R. and Holland, Carl S. and Zhao, Amy H. and Sherman, Stephen E. and Jaskolka, Michael C. and Wu, Meng and Wilson, Aaron C. and Sun, Xiaoqi and Ciulla, Dawn M. and Zhang, Deric and Nelson, Jacqueline D. and Zhang, Peisheng and Mazzucato, Patrizia and Huang, Yan and Giannoukos, Georgia and Marco, Eugenio and Nehil, Michael and Follit, John A. and Chang, Kai-Hsin and Shearman, Mark S. and Wilson, Christopher J. and Zuris, John A.},
  doi          = {10.1038/s41587-023-01779-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {458-469},
  shortjournal = {Nature Biotech.},
  title        = {A highly efficient transgene knock-in technology in clinically relevant cell types},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A fully integrated wearable ultrasound system to monitor
deep tissues in moving subjects. <em>NBT</em>, <em>42</em>(3), 448–457.
(<a href="https://doi.org/10.1038/s41587-023-01800-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in wearable ultrasound technologies have demonstrated the potential for hands-free data acquisition, but technical barriers remain as these probes require wire connections, can lose track of moving targets and create data-interpretation challenges. Here we report a fully integrated autonomous wearable ultrasonic-system-on-patch (USoP). A miniaturized flexible control circuit is designed to interface with an ultrasound transducer array for signal pre-conditioning and wireless data communication. Machine learning is used to track moving tissue targets and assist the data interpretation. We demonstrate that the USoP allows continuous tracking of physiological signals from tissues as deep as 164 mm. On mobile subjects, the USoP can continuously monitor physiological signals, including central blood pressure, heart rate and cardiac output, for as long as 12 h. This result enables continuous autonomous surveillance of deep tissue signals toward the internet-of-medical-things. A wearable ultrasound patch monitors subjects in motion using machine learning and wireless electronics.},
  archive      = {J_NBT},
  author       = {Lin, Muyang and Zhang, Ziyang and Gao, Xiaoxiang and Bian, Yizhou and Wu, Ray S. and Park, Geonho and Lou, Zhiyuan and Zhang, Zhuorui and Xu, Xiangchen and Chen, Xiangjun and Kang, Andrea and Yang, Xinyi and Yue, Wentong and Yin, Lu and Wang, Chonghe and Qi, Baiyan and Zhou, Sai and Hu, Hongjie and Huang, Hao and Li, Mohan and Gu, Yue and Mu, Jing and Yang, Albert and Yaghi, Amer and Chen, Yimu and Lei, Yusheng and Lu, Chengchangfeng and Wang, Ruotao and Wang, Joseph and Xiang, Shu and Kistler, Erik B. and Vasconcelos, Nuno and Xu, Sheng},
  doi          = {10.1038/s41587-023-01800-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {448-457},
  shortjournal = {Nature Biotech.},
  title        = {A fully integrated wearable ultrasound system to monitor deep tissues in moving subjects},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Generation of precision preclinical cancer models using
regulated in vivo base editing. <em>NBT</em>, <em>42</em>(3), 437–447.
(<a href="https://doi.org/10.1038/s41587-023-01900-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although single-nucleotide variants (SNVs) make up the majority of cancer-associated genetic changes and have been comprehensively catalogued, little is known about their impact on tumor initiation and progression. To enable the functional interrogation of cancer-associated SNVs, we developed a mouse system for temporal and regulatable in vivo base editing. The inducible base editing (iBE) mouse carries a single expression-optimized cytosine base editor transgene under the control of a tetracycline response element and enables robust, doxycycline-dependent expression across a broad range of tissues in vivo. Combined with plasmid-based or synthetic guide RNAs, iBE drives efficient engineering of individual or multiple SNVs in intestinal, lung and pancreatic organoids. Temporal regulation of base editor activity allows controlled sequential genome editing ex vivo and in vivo, and delivery of sgRNAs directly to target tissues facilitates generation of in situ preclinical cancer models. The function of single-nucleotide variants can now be tested in a base editor mouse model.},
  archive      = {J_NBT},
  author       = {Katti, Alyna and Vega-Pérez, Adrián and Foronda, Miguel and Zimmerman, Jill and Zafra, Maria Paz and Granowsky, Elizabeth and Goswami, Sukanya and Gardner, Eric E. and Diaz, Bianca J. and Simon, Janelle M. and Wuest, Alexandra and Luan, Wei and Fernandez, Maria Teresa Calvo and Kadina, Anastasia P. and Walker, John A. and Holden, Kevin and Lowe, Scott W. and Sánchez Rivera, Francisco J. and Dow, Lukas E.},
  doi          = {10.1038/s41587-023-01900-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {437-447},
  shortjournal = {Nature Biotech.},
  title        = {Generation of precision preclinical cancer models using regulated in vivo base editing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A prime editor mouse to model a broad spectrum of somatic
mutations in vivo. <em>NBT</em>, <em>42</em>(3), 424–436. (<a
href="https://doi.org/10.1038/s41587-023-01783-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically engineered mouse models only capture a small fraction of the genetic lesions that drive human cancer. Current CRISPR–Cas9 models can expand this fraction but are limited by their reliance on error-prone DNA repair. Here we develop a system for in vivo prime editing by encoding a Cre-inducible prime editor in the mouse germline. This model allows rapid, precise engineering of a wide range of mutations in cell lines and organoids derived from primary tissues, including a clinically relevant Kras mutation associated with drug resistance and Trp53 hotspot mutations commonly observed in pancreatic cancer. With this system, we demonstrate somatic prime editing in vivo using lipid nanoparticles, and we model lung and pancreatic cancer through viral delivery of prime editing guide RNAs or orthotopic transplantation of prime-edited organoids. We believe that this approach will accelerate functional studies of cancer-associated mutations and complex genetic combinations that are challenging to construct with traditional models. Prime-editing mouse models enable the study of specific cancer mutations in vivo.},
  archive      = {J_NBT},
  author       = {Ely, Zackery A. and Mathey-Andrews, Nicolas and Naranjo, Santiago and Gould, Samuel I. and Mercer, Kim L. and Newby, Gregory A. and Cabana, Christina M. and Rideout, William M. and Jaramillo, Grissel Cervantes and Khirallah, Jennifer M. and Holland, Katie and Randolph, Peyton B. and Freed-Pastor, William A. and Davis, Jessie R. and Kulstad, Zachary and Westcott, Peter M. K. and Lin, Lin and Anzalone, Andrew V. and Horton, Brendan L. and Pattada, Nimisha B. and Shanahan, Sean-Luc and Ye, Zhongfeng and Spranger, Stefani and Xu, Qiaobing and Sánchez-Rivera, Francisco J. and Liu, David R. and Jacks, Tyler},
  doi          = {10.1038/s41587-023-01783-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {424-436},
  shortjournal = {Nature Biotech.},
  title        = {A prime editor mouse to model a broad spectrum of somatic mutations in vivo},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Hypoimmune induced pluripotent stem cells survive long term
in fully immunocompetent, allogeneic rhesus macaques. <em>NBT</em>,
<em>42</em>(3), 413–423. (<a
href="https://doi.org/10.1038/s41587-023-01784-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic engineering of allogeneic cell therapeutics that fully prevents rejection by a recipient’s immune system would abolish the requirement for immunosuppressive drugs or encapsulation and support large-scale manufacturing of off-the-shelf cell products. Previously, we generated mouse and human hypoimmune pluripotent (HIP) stem cells by depleting HLA class I and II molecules and overexpressing CD47 (B2M−/−CIITA−/−CD47+). To determine whether this strategy is successful in non-human primates, we engineered rhesus macaque HIP cells and transplanted them intramuscularly into four allogeneic rhesus macaques. The HIP cells survived unrestricted for 16 weeks in fully immunocompetent allogeneic recipients and differentiated into several lineages, whereas allogeneic wild-type cells were vigorously rejected. We also differentiated human HIP cells into endocrinologically active pancreatic islet cells and showed that they survived in immunocompetent, allogeneic diabetic humanized mice for 4 weeks and ameliorated diabetes. HIP-edited primary rhesus macaque islets survived for 40 weeks in an allogeneic rhesus macaque recipient without immunosuppression, whereas unedited islets were quickly rejected. A genetic method to block immune rejection of allogeneic cells is validated in monkeys.},
  archive      = {J_NBT},
  author       = {Hu, Xiaomeng and White, Kathy and Olroyd, Ari G. and DeJesus, Rowena and Dominguez, Antonia A. and Dowdle, William E. and Friera, Annabelle M. and Young, Chi and Wells, Frank and Chu, Elaine Y. and Ito, Cade Ellis and Krishnapura, Harini and Jain, Surbhi and Ankala, Ramya and McGill, Trevor J. and Lin, August and Egenberger, Kyla and Gagnon, Allison and Michael Rukstalis, J. and Hogrebe, Nathaniel J. and Gattis, Corie and Basco, Ron and Millman, Jeffrey R. and Kievit, Paul and Davis, Mark M. and Lanier, Lewis L. and Connolly, Andrew J. and Deuse, Tobias and Schrepfer, Sonja},
  doi          = {10.1038/s41587-023-01784-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {413-423},
  shortjournal = {Nature Biotech.},
  title        = {Hypoimmune induced pluripotent stem cells survive long term in fully immunocompetent, allogeneic rhesus macaques},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Medicare price negotiation and pharmaceutical innovation
following the inflation reduction act. <em>NBT</em>, <em>42</em>(3),
406–412. (<a href="https://doi.org/10.1038/s41587-023-02096-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Inflation Reduction Act (IRA) requires Medicare to negotiate lower prices for some medicines with high Medicare spending. Using historical data from public and proprietary sources to apply the IRA’s negotiation criteria retrospectively, we identify all drugs that met the eligibility criteria from 2012 to 2021 to classify drugs that would have had a negotiated price in effect in 2022 and to calculate associated decreases in industry revenues. Our results suggest that the IRA’s reduction in overall industry revenue will be modest, will not affect most top-selling drugs and will not likely result in large-scale defunding of research and development. Changes in the net present value of drug-development projects will be concentrated in medicines where Medicare is a notable purchaser and where the ratio between expected revenue and development costs was only marginally positive before the IRA. Policymakers considering narrowing or expanding the scope of Medicare negotiation should carefully consider the tradeoffs across medicines with diverse characteristics. A retrospective analysis shows that the IRA will not result in a large-scale defunding of biopharmaceutical innovation.},
  archive      = {J_NBT},
  author       = {Vogel, Matthew and Kakani, Pragya and Chandra, Amitabh and Conti, Rena M.},
  doi          = {10.1038/s41587-023-02096-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {406-412},
  shortjournal = {Nature Biotech.},
  title        = {Medicare price negotiation and pharmaceutical innovation following the inflation reduction act},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). RNA interference in the era of nucleic acid therapeutics.
<em>NBT</em>, <em>42</em>(3), 394–405. (<a
href="https://doi.org/10.1038/s41587-023-02105-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two decades of research on RNA interference (RNAi) have transformed a breakthrough discovery in biology into a robust platform for a new class of medicines that modulate mRNA expression. Here we provide an overview of the trajectory of small-interfering RNA (siRNA) drug development, including the first approval in 2018 of a liver-targeted siRNA interference (RNAi) therapeutic in lipid nanoparticles and subsequent approvals of five more RNAi drugs, which used metabolically stable siRNAs combined with N-acetylgalactosamine ligands for conjugate-based liver delivery. We also consider the remaining challenges in the field, such as delivery to muscle, brain and other extrahepatic organs. Today’s RNAi therapeutics exhibit high specificity, potency and durability, and are transitioning from applications in rare diseases to widespread, chronic conditions. With six approved drugs, siRNA is now an established therapeutic modality poised for expansion.},
  archive      = {J_NBT},
  author       = {Jadhav, Vasant and Vaishnaw, Akshay and Fitzgerald, Kevin and Maier, Martin A.},
  doi          = {10.1038/s41587-023-02105-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {394-405},
  shortjournal = {Nature Biotech.},
  title        = {RNA interference in the era of nucleic acid therapeutics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). CRISPR-free, strand-selective mitochondrial DNA base editing
using a nickase. <em>NBT</em>, <em>42</em>(3), 392–393. (<a
href="https://doi.org/10.1038/s41587-023-01820-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The fusion of a programmable transcription-activator-like effector (TALE) protein with a nickase, in conjunction with a deaminase, enables efficient and strand-selective DNA base editing. This approach has the potential to advance our understanding and treatment of diseases associated with mutations in the mitochondrial or nuclear genome.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01820-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {392-393},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR-free, strand-selective mitochondrial DNA base editing using a nickase},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inferring cell–cell communication at single-cell resolution.
<em>NBT</em>, <em>42</em>(3), 390–391. (<a
href="https://doi.org/10.1038/s41587-023-01834-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing methods to infer cell–cell communication from single-cell RNA-sequencing data fail to leverage the full information structure of the data, generally by operating at the level of the cell type or cluster. We describe a framework called Scriabin to perform this analysis at the level of the individual cell.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01834-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {390-391},
  shortjournal = {Nature Biotech.},
  title        = {Inferring cell–cell communication at single-cell resolution},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Selection by essential-gene exon knock-in for the generation
of efficient cell therapies. <em>NBT</em>, <em>42</em>(3), 388–389. (<a
href="https://doi.org/10.1038/s41587-023-01819-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By linking transgene expression with that of a housekeeping gene, SLEEK (selection by essential-gene exon knock-in) enables efficient knock-in of complex cargos in a variety of clinically relevant cell types. Using SLEEK, we were able to substantially improve the tumor suppression ability and in vivo persistence of a cell therapy.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01819-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {388-389},
  shortjournal = {Nature Biotech.},
  title        = {Selection by essential-gene exon knock-in for the generation of efficient cell therapies},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Wearable ultrasound for continuous deep-tissue monitoring.
<em>NBT</em>, <em>42</em>(3), 386–387. (<a
href="https://doi.org/10.1038/s41587-023-02098-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An integrated wearable and wireless ultrasound device enables the continuous monitoring of physiological information from tissues in moving individuals.},
  archive      = {J_NBT},
  author       = {Kenny, Jon-Emile S.},
  doi          = {10.1038/s41587-023-02098-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {386-387},
  shortjournal = {Nature Biotech.},
  title        = {Wearable ultrasound for continuous deep-tissue monitoring},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cancer variant modeling in vivo. <em>NBT</em>,
<em>42</em>(3), 383–385. (<a
href="https://doi.org/10.1038/s41587-023-02080-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime- and base-editor mouse models enable finely tuned, efficient functional studies of cancer-associated genetic changes.},
  archive      = {J_NBT},
  author       = {Istadi, Aji and Porazinski, Sean and Pajic, Marina},
  doi          = {10.1038/s41587-023-02080-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {383-385},
  shortjournal = {Nature Biotech.},
  title        = {Cancer variant modeling in vivo},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Hypoimmune cells resist rejection in monkeys. <em>NBT</em>,
<em>42</em>(3), 380–382. (<a
href="https://doi.org/10.1038/s41587-023-02013-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic engineering to block immune destruction of transplanted cells succeeds in macaques.},
  archive      = {J_NBT},
  author       = {Borrero Borrego, Asuncion and Gill, Saar},
  doi          = {10.1038/s41587-023-02013-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {380-382},
  shortjournal = {Nature Biotech.},
  title        = {Hypoimmune cells resist rejection in monkeys},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024ae). Predicting the structure of large protein complexes.
<em>NBT</em>, <em>42</em>(3), 379. (<a
href="https://doi.org/10.1038/s41587-024-02183-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02183-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {379},
  shortjournal = {Nature Biotech.},
  title        = {Predicting the structure of large protein complexes},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024ag). Supercharging t cell therapy with cancer mutations.
<em>NBT</em>, <em>42</em>(3), 379. (<a
href="https://doi.org/10.1038/s41587-024-02184-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02184-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {379},
  shortjournal = {Nature Biotech.},
  title        = {Supercharging t cell therapy with cancer mutations},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). A synthetic antibiotic overcomes antimicrobial resistance.
<em>NBT</em>, <em>42</em>(3), 379. (<a
href="https://doi.org/10.1038/s41587-024-02185-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02185-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {379},
  shortjournal = {Nature Biotech.},
  title        = {A synthetic antibiotic overcomes antimicrobial resistance},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Wearable technology and devices. <em>NBT</em>,
<em>42</em>(3), 378. (<a
href="https://doi.org/10.1038/s41587-024-02163-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to wearable technology and devices for patient monitoring and treatment of health conditions.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02163-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {378},
  shortjournal = {Nature Biotech.},
  title        = {Wearable technology and devices},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Biologic patent challenges under the america invents act.
<em>NBT</em>, <em>42</em>(3), 374–377. (<a
href="https://doi.org/10.1038/s41587-024-02156-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Administrative patent review procedures are an effective way of correcting erroneously granted biologic patents and may help promote timely drug competition for the benefit of patients and the US healthcare system.},
  archive      = {J_NBT},
  author       = {Van de Wiele, Victor L. and Kesselheim, Aaron S. and Tu, S. Sean},
  doi          = {10.1038/s41587-024-02156-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {374-377},
  shortjournal = {Nature Biotech.},
  title        = {Biologic patent challenges under the america invents act},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The perpetual motion machine of AI-generated data and the
distraction of ChatGPT as a “scientist.” <em>NBT</em>, <em>42</em>(3),
371–373. (<a href="https://doi.org/10.1038/s41587-023-02103-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Listgarten, Jennifer},
  doi          = {10.1038/s41587-023-02103-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {371-373},
  shortjournal = {Nature Biotech.},
  title        = {The perpetual motion machine of AI-generated data and the distraction of ChatGPT as a ‘scientist’},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Scalable, accessible and reproducible reference genome
assembly and evaluation in galaxy. <em>NBT</em>, <em>42</em>(3),
367–370. (<a href="https://doi.org/10.1038/s41587-023-02100-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Larivière, Delphine and Abueg, Linelle and Brajuka, Nadolina and Gallardo-Alba, Cristóbal and Grüning, Bjorn and Ko, Byung June and Ostrovsky, Alex and Palmada-Flores, Marc and Pickett, Brandon D. and Rabbani, Keon and Antunes, Agostinho and Balacco, Jennifer R. and Chaisson, Mark J. P. and Cheng, Haoyu and Collins, Joanna and Couture, Melanie and Denisova, Alexandra and Fedrigo, Olivier and Gallo, Guido Roberto and Giani, Alice Maria and Gooder, Grenville MacDonald and Horan, Kathleen and Jain, Nivesh and Johnson, Cassidy and Kim, Heebal and Lee, Chul and Marques-Bonet, Tomas and O’Toole, Brian and Rhie, Arang and Secomandi, Simona and Sozzoni, Marcella and Tilley, Tatiana and Uliano-Silva, Marcela and van den Beek, Marius and Williams, Robert W. and Waterhouse, Robert M. and Phillippy, Adam M. and Jarvis, Erich D. and Schatz, Michael C. and Nekrutenko, Anton and Formenti, Giulio},
  doi          = {10.1038/s41587-023-02100-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {367-370},
  shortjournal = {Nature Biotech.},
  title        = {Scalable, accessible and reproducible reference genome assembly and evaluation in galaxy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(3), 354. (<a
href="https://doi.org/10.1038/s41587-024-02177-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02177-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {354},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). US bill targets chinese biotechs. <em>NBT</em>,
<em>42</em>(3), 353. (<a
href="https://doi.org/10.1038/s41587-024-02191-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02191-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {353},
  shortjournal = {Nature Biotech.},
  title        = {US bill targets chinese biotechs},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). In vivo CRISPR agent cuts HAE attacks 95. <em>NBT</em>,
<em>42</em>(3), 352. (<a
href="https://doi.org/10.1038/s41587-024-02188-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02188-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {352},
  shortjournal = {Nature Biotech.},
  title        = {In vivo CRISPR agent cuts HAE attacks 95%},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). After obesity drugs’ success, companies rush to preserve
skeletal muscle. <em>NBT</em>, <em>42</em>(3), 351–353. (<a
href="https://doi.org/10.1038/s41587-024-02176-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A growing number of companies are testing muscle-building agents to counter the side effects of dramatic weight loss and potentially to preserve lean muscle into old age.},
  archive      = {J_NBT},
  author       = {Arnold, Carrie},
  doi          = {10.1038/s41587-024-02176-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {351-353},
  shortjournal = {Nature Biotech.},
  title        = {After obesity drugs’ success, companies rush to preserve skeletal muscle},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Cell-based coffee future-proofs world’s favorite brew.
<em>NBT</em>, <em>42</em>(3), 350. (<a
href="https://doi.org/10.1038/s41587-024-02181-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-024-02181-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {350},
  shortjournal = {Nature Biotech.},
  title        = {Cell-based coffee future-proofs world’s favorite brew},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). World’s first TIL therapy approved. <em>NBT</em>,
<em>42</em>(3), 349. (<a
href="https://doi.org/10.1038/s41587-024-02195-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02195-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {349},
  shortjournal = {Nature Biotech.},
  title        = {World’s first TIL therapy approved},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Innovators want pills to treat sickle cell disease. Can
they match gene therapy? <em>NBT</em>, <em>42</em>(3), 347–350. (<a
href="https://doi.org/10.1038/s41587-024-02179-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although CRISPR-based gene therapy for sickle cell disease offers transformative outcomes, drugmakers are striving to develop treatments that are easy to manufacture and can reach much larger numbers of patients.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-024-02179-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {347-350},
  shortjournal = {Nature Biotech.},
  title        = {Innovators want pills to treat sickle cell disease. can they match gene therapy?},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Growing community across the c-suite. <em>NBT</em>,
<em>42</em>(3), 345. (<a
href="https://doi.org/10.1038/s41587-024-02192-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Women in biotech CEO positions now have resources and networks.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02192-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {345},
  shortjournal = {Nature Biotech.},
  title        = {Growing community across the C-suite},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Cancer-targeting antibody–drug conjugates drive dealmaking
frenzy. <em>NBT</em>, <em>42</em>(3), 362–366. (<a
href="https://doi.org/10.1038/s41587-024-02168-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Improved linker technologies and broadening payload options mark a coming of age for these precision cancer therapies.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-024-02168-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {362-366},
  shortjournal = {Nature Biotech.},
  title        = {Cancer-targeting antibody–drug conjugates drive dealmaking frenzy},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Fresh from the biotech pipeline: Record-breaking FDA
approvals. <em>NBT</em>, <em>42</em>(3), 355–361. (<a
href="https://doi.org/10.1038/s41587-024-02166-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New drug approvals reached an all-time high in 2023, with five gene therapies, the first CRISPR–Cas9-edited therapy and a disease-modifying Alzheimer’s drug.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-024-02166-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {355-361},
  shortjournal = {Nature Biotech.},
  title        = {Fresh from the biotech pipeline: Record-breaking FDA approvals},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024k). People. <em>NBT</em>, <em>42</em>(2), 344. (<a
href="https://doi.org/10.1038/s41587-024-02134-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02134-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {344},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Five questions with erika DeBenedictis. <em>NBT</em>,
<em>42</em>(2), 343. (<a
href="https://doi.org/10.1038/s41587-024-02131-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This computational physicist and molecular biologist wants to solve big problems using evolution.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02131-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {343},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with erika DeBenedictis},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). SPARKing academic technologies across the valley of death.
<em>NBT</em>, <em>42</em>(2), 339–342. (<a
href="https://doi.org/10.1038/s41587-024-02130-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stanford’s SPARK program has established a process that advances academic discoveries to industry partnerships or investigator-initiated clinical trials and provides training that enhances career opportunities for its graduates.},
  archive      = {J_NBT},
  author       = {Kim, Jeewon Sylvia and Kargotich, Stephen and Lee, Sophia H. and Yajima, Rieko and Garcia, Adriana Ann and Ehrenkaufer, Gretchen and Romeo, Mary and Santa Maria, Peter and Grimes, Kevin V. and Mochly-Rosen, Daria},
  doi          = {10.1038/s41587-024-02130-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {339-342},
  shortjournal = {Nature Biotech.},
  title        = {SPARKing academic technologies across the valley of death},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantifying bias introduced by sample collection in relative
and absolute microbiome measurements. <em>NBT</em>, <em>42</em>(2),
328–338. (<a href="https://doi.org/10.1038/s41587-023-01754-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To gain insight into the accuracy of microbial measurements, it is important to evaluate sources of bias related to sample condition, preservative method and bioinformatic analyses. There is increasing evidence that measurement of the total count and concentration of microbes in the gut, or ‘absolute abundance’, provides a richer source of information than relative abundance and can correct some conclusions drawn from relative abundance data. However, little is known about how preservative choice can affect these measurements. In this study, we investigated how two common preservatives and short-term storage conditions impact relative and absolute microbial measurements. OMNIgene GUT OMR-200 yields lower metagenomic taxonomic variation between different storage temperatures, whereas Zymo DNA/RNA Shield yields lower metatranscriptomic taxonomic variation. Absolute abundance quantification reveals two different causes of variable Bacteroidetes:Firmicutes ratios across preservatives. Based on these results, we recommend OMNIgene GUT OMR-200 preservative for field studies and Zymo DNA/RNA Shield for metatranscriptomics studies, and we strongly encourage absolute quantification for microbial measurements. Absolute abundance measurements of microbes reveals information obscured in relative abundance data.},
  archive      = {J_NBT},
  author       = {Maghini, Dylan G. and Dvorak, Mai and Dahlen, Alex and Roos, Morgan and Kuersten, Scott and Bhatt, Ami S.},
  doi          = {10.1038/s41587-023-01754-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {328-338},
  shortjournal = {Nature Biotech.},
  title        = {Quantifying bias introduced by sample collection in relative and absolute microbiome measurements},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Precise integration of large DNA sequences in plant genomes
using PrimeRoot editors. <em>NBT</em>, <em>42</em>(2), 316–327. (<a
href="https://doi.org/10.1038/s41587-023-01769-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A technique for chromosomal insertion of large DNA segments is much needed in plant breeding and synthetic biology to facilitate the introduction of desired agronomic traits and signaling and metabolic pathways. Here we describe PrimeRoot, a genome editing approach to generate targeted precise large DNA insertions in plants. Third-generation PrimeRoot editors employ optimized prime editing guide RNA designs, an enhanced plant prime editor and superior recombinases to enable precise large DNA insertions of up to 11.1 kilobases into plant genomes. We demonstrate the use of PrimeRoot to accurately introduce gene regulatory elements in rice. In this study, we also integrated a gene cassette comprising PigmR, which confers rice blast resistance driven by an Act1 promoter, into a predicted genomic safe harbor site of Kitaake rice and obtain edited plants harboring the expected insertion with an efficiency of 6.3%. We found that these rice plants have increased blast resistance. These results establish PrimeRoot as a promising approach to precisely insert large segments of DNA in plants. Large genetic cargos are precisely inserted into plant genomes.},
  archive      = {J_NBT},
  author       = {Sun, Chao and Lei, Yuan and Li, Boshu and Gao, Qiang and Li, Yunjia and Cao, Wen and Yang, Chao and Li, Hongchao and Wang, Zhiwei and Li, Yan and Wang, Yanpeng and Liu, Jun and Zhao, Kevin Tianmeng and Gao, Caixia},
  doi          = {10.1038/s41587-023-01769-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {316-327},
  shortjournal = {Nature Biotech.},
  title        = {Precise integration of large DNA sequences in plant genomes using PrimeRoot editors},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Efficient engineering of human and mouse primary cells using
peptide-assisted genome editing. <em>NBT</em>, <em>42</em>(2), 305–315.
(<a href="https://doi.org/10.1038/s41587-023-01756-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Simple, efficient and well-tolerated delivery of CRISPR genome editing systems into primary cells remains a major challenge. Here we describe an engineered Peptide-Assisted Genome Editing (PAGE) CRISPR–Cas system for rapid and robust editing of primary cells with minimal toxicity. The PAGE system requires only a 30-min incubation with a cell-penetrating Cas9 or Cas12a and a cell-penetrating endosomal escape peptide to achieve robust single and multiplex genome editing. Unlike electroporation-based methods, PAGE gene editing has low cellular toxicity and shows no significant transcriptional perturbation. We demonstrate rapid and efficient editing of primary cells, including human and mouse T cells, as well as human hematopoietic progenitor cells, with editing efficiencies upwards of 98%. PAGE provides a broadly generalizable platform for next-generation genome engineering in primary cells. Peptide-assisted genome editing enables efficient single and multiplex editing in hematopoietic cells.},
  archive      = {J_NBT},
  author       = {Zhang, Zhen and Baxter, Amy E. and Ren, Diqiu and Qin, Kunhua and Chen, Zeyu and Collins, Sierra M. and Huang, Hua and Komar, Chad A. and Bailer, Peter F. and Parker, Jared B. and Blobel, Gerd A. and Kohli, Rahul M. and Wherry, E. John and Berger, Shelley L. and Shi, Junwei},
  doi          = {10.1038/s41587-023-01756-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {305-315},
  shortjournal = {Nature Biotech.},
  title        = {Efficient engineering of human and mouse primary cells using peptide-assisted genome editing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Dictionary learning for integrative, multimodal and scalable
single-cell analysis. <em>NBT</em>, <em>42</em>(2), 293–304. (<a
href="https://doi.org/10.1038/s41587-023-01767-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping single-cell sequencing profiles to comprehensive reference datasets provides a powerful alternative to unsupervised analysis. However, most reference datasets are constructed from single-cell RNA-sequencing data and cannot be used to annotate datasets that do not measure gene expression. Here we introduce ‘bridge integration’, a method to integrate single-cell datasets across modalities using a multiomic dataset as a molecular bridge. Each cell in the multiomic dataset constitutes an element in a ‘dictionary’, which is used to reconstruct unimodal datasets and transform them into a shared space. Our procedure accurately integrates transcriptomic data with independent single-cell measurements of chromatin accessibility, histone modifications, DNA methylation and protein levels. Moreover, we demonstrate how dictionary learning can be combined with sketching techniques to improve computational scalability and harmonize 8.6 million human immune cell profiles from sequencing and mass cytometry experiments. Our approach, implemented in version 5 of our Seurat toolkit ( http://www.satijalab.org/seurat ), broadens the utility of single-cell reference datasets and facilitates comparisons across diverse molecular modalities. Reference mapping is extended beyond scRNA-seq to single-cell epigenetic and proteomic data.},
  archive      = {J_NBT},
  author       = {Hao, Yuhan and Stuart, Tim and Kowalski, Madeline H. and Choudhary, Saket and Hoffman, Paul and Hartman, Austin and Srivastava, Avi and Molla, Gesmira and Madad, Shaista and Fernandez-Granda, Carlos and Satija, Rahul},
  doi          = {10.1038/s41587-023-01767-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {293-304},
  shortjournal = {Nature Biotech.},
  title        = {Dictionary learning for integrative, multimodal and scalable single-cell analysis},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Stabilized mosaic single-cell data integration using
unshared features. <em>NBT</em>, <em>42</em>(2), 284–292. (<a
href="https://doi.org/10.1038/s41587-023-01766-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Currently available single-cell omics technologies capture many unique features with different biological information content. Data integration aims to place cells, captured with different technologies, onto a common embedding to facilitate downstream analytical tasks. Current horizontal data integration techniques use a set of common features, thereby ignoring non-overlapping features and losing information. Here we introduce StabMap, a mosaic data integration technique that stabilizes mapping of single-cell data by exploiting the non-overlapping features. StabMap first infers a mosaic data topology based on shared features, then projects all cells onto supervised or unsupervised reference coordinates by traversing shortest paths along the topology. We show that StabMap performs well in various simulation contexts, facilitates ‘multi-hop’ mosaic data integration where some datasets do not share any features and enables the use of spatial gene expression features for mapping dissociated single-cell data onto a spatial transcriptomic reference. Mosaic data integration is improved by capturing features that do not overlap between datasets.},
  archive      = {J_NBT},
  author       = {Ghazanfar, Shila and Guibentif, Carolina and Marioni, John C.},
  doi          = {10.1038/s41587-023-01766-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {284-292},
  shortjournal = {Nature Biotech.},
  title        = {Stabilized mosaic single-cell data integration using unshared features},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Efficient evolution of human antibodies from general protein
language models. <em>NBT</em>, <em>42</em>(2), 275–283. (<a
href="https://doi.org/10.1038/s41587-023-01763-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural evolution must explore a vast landscape of possible sequences for desirable yet rare mutations, suggesting that learning from natural evolutionary strategies could guide artificial evolution. Here we report that general protein language models can efficiently evolve human antibodies by suggesting mutations that are evolutionarily plausible, despite providing the model with no information about the target antigen, binding specificity or protein structure. We performed language-model-guided affinity maturation of seven antibodies, screening 20 or fewer variants of each antibody across only two rounds of laboratory evolution, and improved the binding affinities of four clinically relevant, highly mature antibodies up to sevenfold and three unmatured antibodies up to 160-fold, with many designs also demonstrating favorable thermostability and viral neutralization activity against Ebola and severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pseudoviruses. The same models that improve antibody binding also guide efficient evolution across diverse protein families and selection pressures, including antibiotic resistance and enzyme activity, suggesting that these results generalize to many settings. A general protein language model guides protein evolution with 20 or fewer variants needed for testing.},
  archive      = {J_NBT},
  author       = {Hie, Brian L. and Shanker, Varun R. and Xu, Duo and Bruun, Theodora U. J. and Weidenbacher, Payton A. and Tang, Shaogeng and Wu, Wesley and Pak, John E. and Kim, Peter S.},
  doi          = {10.1038/s41587-023-01763-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {275-283},
  shortjournal = {Nature Biotech.},
  title        = {Efficient evolution of human antibodies from general protein language models},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Engineered phage with antibacterial CRISPR–cas selectively
reduce e. Coli burden in mice. <em>NBT</em>, <em>42</em>(2), 265–274.
(<a href="https://doi.org/10.1038/s41587-023-01759-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antibiotic treatments have detrimental effects on the microbiome and lead to antibiotic resistance. To develop a phage therapy against a diverse range of clinically relevant Escherichia coli, we screened a library of 162 wild-type (WT) phages, identifying eight phages with broad coverage of E. coli, complementary binding to bacterial surface receptors, and the capability to stably carry inserted cargo. Selected phages were engineered with tail fibers and CRISPR–Cas machinery to specifically target E. coli. We show that engineered phages target bacteria in biofilms, reduce the emergence of phage-tolerant E. coli and out-compete their ancestral WT phages in coculture experiments. A combination of the four most complementary bacteriophages, called SNIPR001, is well tolerated in both mouse models and minipigs and reduces E. coli load in the mouse gut better than its constituent components separately. SNIPR001 is in clinical development to selectively kill E. coli, which may cause fatal infections in hematological cancer patients. Phage engineered with tail fibers and CRISPR–Cas reduce Escherichia coli load in animals.},
  archive      = {J_NBT},
  author       = {Gencay, Yilmaz Emre and Jasinskytė, Džiuginta and Robert, Camille and Semsey, Szabolcs and Martínez, Virginia and Petersen, Anders Østergaard and Brunner, Katja and de Santiago Torio, Ana and Salazar, Alex and Turcu, Iszabela Cristiana and Eriksen, Melissa Kviesgaard and Koval, Lev and Takos, Adam and Pascal, Ricardo and Schou, Thea Staffeldt and Bayer, Lone and Bryde, Tina and Johansen, Katja Chandelle and Bak, Emilie Glad and Smrekar, Frenk and Doyle, Timothy B. and Satlin, Michael J. and Gram, Aurelie and Carvalho, Joana and Jessen, Lene and Hallström, Björn and Hink, Jonas and Damholt, Birgitte and Troy, Alice and Grove, Mette and Clube, Jasper and Grøndahl, Christian and Haaber, Jakob Krause and van der Helm, Eric and Zdravkovic, Milan and Sommer, Morten Otto Alexander},
  doi          = {10.1038/s41587-023-01759-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {265-274},
  shortjournal = {Nature Biotech.},
  title        = {Engineered phage with antibacterial CRISPR–Cas selectively reduce e. coli burden in mice},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Efficient prime editing in mouse brain, liver and heart with
dual AAVs. <em>NBT</em>, <em>42</em>(2), 253–264. (<a
href="https://doi.org/10.1038/s41587-023-01758-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Realizing the promise of prime editing for the study and treatment of genetic disorders requires efficient methods for delivering prime editors (PEs) in vivo. Here we describe the identification of bottlenecks limiting adeno-associated virus (AAV)-mediated prime editing in vivo and the development of AAV-PE vectors with increased PE expression, prime editing guide RNA stability and modulation of DNA repair. The resulting dual-AAV systems, v1em and v3em PE-AAV, enable therapeutically relevant prime editing in mouse brain (up to 42% efficiency in cortex), liver (up to 46%) and heart (up to 11%). We apply these systems to install putative protective mutations in vivo for Alzheimer’s disease in astrocytes and for coronary artery disease in hepatocytes. In vivo prime editing with v3em PE-AAV caused no detectable off-target effects or significant changes in liver enzymes or histology. Optimized PE-AAV systems support the highest unenriched levels of in vivo prime editing reported to date, facilitating the study and potential treatment of diseases with a genetic component. Prime editors are efficiently delivered to mouse organs with AAV vectors.},
  archive      = {J_NBT},
  author       = {Davis, Jessie R. and Banskota, Samagya and Levy, Jonathan M. and Newby, Gregory A. and Wang, Xiao and Anzalone, Andrew V. and Nelson, Andrew T. and Chen, Peter J. and Hennes, Andrew D. and An, Meirui and Roh, Heejin and Randolph, Peyton B. and Musunuru, Kiran and Liu, David R.},
  doi          = {10.1038/s41587-023-01758-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {253-264},
  shortjournal = {Nature Biotech.},
  title        = {Efficient prime editing in mouse brain, liver and heart with dual AAVs},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ScDesign3 generates realistic in silico data for multimodal
single-cell and spatial omics. <em>NBT</em>, <em>42</em>(2), 247–252.
(<a href="https://doi.org/10.1038/s41587-023-01772-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a statistical simulator, scDesign3, to generate realistic single-cell and spatial omics data, including various cell states, experimental designs and feature modalities, by learning interpretable parameters from real data. Using a unified probabilistic model for single-cell and spatial omics data, scDesign3 infers biologically meaningful parameters; assesses the goodness-of-fit of inferred cell clusters, trajectories and spatial locations; and generates in silico negative and positive controls for benchmarking computational tools. The challenge of simulating multiomic single-cell data is addressed by a probabilistic model.},
  archive      = {J_NBT},
  author       = {Song, Dongyuan and Wang, Qingyang and Yan, Guanao and Liu, Tianyang and Sun, Tianyi and Li, Jingyi Jessica},
  doi          = {10.1038/s41587-023-01772-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {247-252},
  shortjournal = {Nature Biotech.},
  title        = {ScDesign3 generates realistic in silico data for multimodal single-cell and spatial omics},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Fast and accurate protein structure search with foldseek.
<em>NBT</em>, <em>42</em>(2), 243–246. (<a
href="https://doi.org/10.1038/s41587-023-01773-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As structure prediction methods are generating millions of publicly available protein structures, searching these databases is becoming a bottleneck. Foldseek aligns the structure of a query protein against a database by describing tertiary amino acid interactions within proteins as sequences over a structural alphabet. Foldseek decreases computation times by four to five orders of magnitude with 86%, 88% and 133% of the sensitivities of Dali, TM-align and CE, respectively. Foldseek speeds up protein structural search by four to five orders of magnitude.},
  archive      = {J_NBT},
  author       = {van Kempen, Michel and Kim, Stephanie S. and Tumescheit, Charlotte and Mirdita, Milot and Lee, Jeongjae and Gilchrist, Cameron L. M. and Söding, Johannes and Steinegger, Martin},
  doi          = {10.1038/s41587-023-01773-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {243-246},
  shortjournal = {Nature Biotech.},
  title        = {Fast and accurate protein structure search with foldseek},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computational drug development for membrane protein targets.
<em>NBT</em>, <em>42</em>(2), 229–242. (<a
href="https://doi.org/10.1038/s41587-023-01987-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The application of computational biology in drug development for membrane protein targets has experienced a boost from recent developments in deep learning-driven structure prediction, increased speed and resolution of structure elucidation, machine learning structure-based design and the evaluation of big data. Recent protein structure predictions based on machine learning tools have delivered surprisingly reliable results for water-soluble and membrane proteins but have limitations for development of drugs that target membrane proteins. Structural transitions of membrane proteins have a central role during transmembrane signaling and are often influenced by therapeutic compounds. Resolving the structural and functional basis of dynamic transmembrane signaling networks, especially within the native membrane or cellular environment, remains a central challenge for drug development. Tackling this challenge will require an interplay between experimental and computational tools, such as super-resolution optical microscopy for quantification of the molecular interactions of cellular signaling networks and their modulation by potential drugs, cryo-electron microscopy for determination of the structural transitions of proteins in native cell membranes and entire cells, and computational tools for data analysis and prediction of the structure and function of cellular signaling networks, as well as generation of promising drug candidates. Drug discovery is being transformed by advances in computational protein structure prediction and protein design.},
  archive      = {J_NBT},
  author       = {Li, Haijian and Sun, Xiaolin and Cui, Wenqiang and Xu, Marc and Dong, Junlin and Ekundayo, Babatunde Edukpe and Ni, Dongchun and Rao, Zhili and Guo, Liwei and Stahlberg, Henning and Yuan, Shuguang and Vogel, Horst},
  doi          = {10.1038/s41587-023-01987-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {229-242},
  shortjournal = {Nature Biotech.},
  title        = {Computational drug development for membrane protein targets},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Machine learning for functional protein design.
<em>NBT</em>, <em>42</em>(2), 216–228. (<a
href="https://doi.org/10.1038/s41587-024-02127-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent breakthroughs in AI coupled with the rapid accumulation of protein sequence and structure data have radically transformed computational protein design. New methods promise to escape the constraints of natural and laboratory evolution, accelerating the generation of proteins for applications in biotechnology and medicine. To make sense of the exploding diversity of machine learning approaches, we introduce a unifying framework that classifies models on the basis of their use of three core data modalities: sequences, structures and functional labels. We discuss the new capabilities and outstanding challenges for the practical design of enzymes, antibodies, vaccines, nanomachines and more. We then highlight trends shaping the future of this field, from large-scale assays to more robust benchmarks, multimodal foundation models, enhanced sampling strategies and laboratory automation. Notin, Rollins and colleagues discuss advances in computational protein design with a focus on redesign of existing proteins.},
  archive      = {J_NBT},
  author       = {Notin, Pascal and Rollins, Nathan and Gal, Yarin and Sander, Chris and Marks, Debora},
  doi          = {10.1038/s41587-024-02127-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {216-228},
  shortjournal = {Nature Biotech.},
  title        = {Machine learning for functional protein design},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Sparks of function by de novo protein design. <em>NBT</em>,
<em>42</em>(2), 203–215. (<a
href="https://doi.org/10.1038/s41587-024-02133-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Information in proteins flows from sequence to structure to function, with each step causally driven by the preceding one. Protein design is founded on inverting this process: specify a desired function, design a structure executing this function, and find a sequence that folds into this structure. This ‘central dogma’ underlies nearly all de novo protein-design efforts. Our ability to accomplish these tasks depends on our understanding of protein folding and function and our ability to capture this understanding in computational methods. In recent years, deep learning-derived approaches for efficient and accurate structure modeling and enrichment of successful designs have enabled progression beyond the design of protein structures and towards the design of functional proteins. We examine these advances in the broader context of classical de novo protein design and consider implications for future challenges to come, including fundamental capabilities such as sequence and structure co-design and conformational control considering flexibility, and functional objectives such as antibody and enzyme design. Chu and colleagues discuss recent developments in de novo protein design.},
  archive      = {J_NBT},
  author       = {Chu, Alexander E. and Lu, Tianyu and Huang, Po-Ssu},
  doi          = {10.1038/s41587-024-02133-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {203-215},
  shortjournal = {Nature Biotech.},
  title        = {Sparks of function by de novo protein design},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Designing proteins with language models. <em>NBT</em>,
<em>42</em>(2), 200–202. (<a
href="https://doi.org/10.1038/s41587-024-02123-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein language models learn from diverse sequences spanning the evolutionary tree and have proven to be powerful tools for sequence design, variant effect prediction and structure prediction. What are the foundations of protein language models, and how are they applied in protein engineering?},
  archive      = {J_NBT},
  author       = {Ruffolo, Jeffrey A. and Madani, Ali},
  doi          = {10.1038/s41587-024-02123-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {200-202},
  shortjournal = {Nature Biotech.},
  title        = {Designing proteins with language models},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Generative models for protein structures and sequences.
<em>NBT</em>, <em>42</em>(2), 196–199. (<a
href="https://doi.org/10.1038/s41587-023-02115-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Models like ChatGPT and DALL-E2 generate text and images in response to a text prompt. Despite different data and goals, how can generative models be useful for protein engineering?},
  archive      = {J_NBT},
  author       = {Hsu, Chloe and Fannjiang, Clara and Listgarten, Jennifer},
  doi          = {10.1038/s41587-023-02115-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {196-199},
  shortjournal = {Nature Biotech.},
  title        = {Generative models for protein structures and sequences},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Sampling techniques and microbial concentration measurements
influence the results of microbiome studies. <em>NBT</em>,
<em>42</em>(2), 194–195. (<a
href="https://doi.org/10.1038/s41587-023-01804-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most microbiome studies measure relative abundance of the various microorganisms in a sample. This study demonstrates the feasibility of large-scale measurements of absolute microbial concentrations. Moreover, it demonstrates how sample handling and storage methods can alter microbial measurements, potentially introducing bias in conclusions drawn about microbiome–host relationships.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01804-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {194-195},
  shortjournal = {Nature Biotech.},
  title        = {Sampling techniques and microbial concentration measurements influence the results of microbiome studies},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Forum: Deane and grigoryan. <em>NBT</em>, <em>42</em>(2),
192. (<a href="https://doi.org/10.1038/s41587-024-02144-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02144-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {192},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Deane and grigoryan},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integration of multi-modal single-cell data. <em>NBT</em>,
<em>42</em>(2), 190–191. (<a
href="https://doi.org/10.1038/s41587-023-01826-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell data from RNA-seq, chromatin accessibility, DNA methylation and other modalities can be readily integrated using two new methods.},
  archive      = {J_NBT},
  author       = {Lee, Michelle Y. Y. and Li, Mingyao},
  doi          = {10.1038/s41587-023-01826-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {190-191},
  shortjournal = {Nature Biotech.},
  title        = {Integration of multi-modal single-cell data},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An optimized toolkit for prime editing. <em>NBT</em>,
<em>42</em>(2), 187–189. (<a
href="https://doi.org/10.1038/s41587-023-02091-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing in vivo is systematically improved with an eye to clinical translation.},
  archive      = {J_NBT},
  author       = {Marks, Ryan M. and Scott, Ori and Ivakine, Evgueni A. and Cohn, Ronald D.},
  doi          = {10.1038/s41587-023-02091-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {187-189},
  shortjournal = {Nature Biotech.},
  title        = {An optimized toolkit for prime editing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Perfecting antibodies with language models. <em>NBT</em>,
<em>42</em>(2), 185–186. (<a
href="https://doi.org/10.1038/s41587-023-01991-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A general-purpose protein language model rapidly improves antibody properties.},
  archive      = {J_NBT},
  author       = {Outeiral, Carlos and Deane, Charlotte M.},
  doi          = {10.1038/s41587-023-01991-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {185-186},
  shortjournal = {Nature Biotech.},
  title        = {Perfecting antibodies with language models},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). A new class of antibiotics acts as molecular glue.
<em>NBT</em>, <em>42</em>(2), 184. (<a
href="https://doi.org/10.1038/s41587-024-02145-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02145-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {184},
  shortjournal = {Nature Biotech.},
  title        = {A new class of antibiotics acts as molecular glue},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). An improved organoid model of the human cerebellum.
<em>NBT</em>, <em>42</em>(2), 184. (<a
href="https://doi.org/10.1038/s41587-024-02146-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02146-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {184},
  shortjournal = {Nature Biotech.},
  title        = {An improved organoid model of the human cerebellum},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Boosting circulating tumor DNA to improve liquid biopsies.
<em>NBT</em>, <em>42</em>(2), 184. (<a
href="https://doi.org/10.1038/s41587-024-02154-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02154-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {184},
  shortjournal = {Nature Biotech.},
  title        = {Boosting circulating tumor DNA to improve liquid biopsies},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Protein engineering and design. <em>NBT</em>,
<em>42</em>(2), 183. (<a
href="https://doi.org/10.1038/s41587-024-02124-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to systems and methods for designing engineered polypeptides.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02124-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {183},
  shortjournal = {Nature Biotech.},
  title        = {Protein engineering and design},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Contracts and intellectual property rights in translational
r&amp;d: Furthering safeguards in the public interest. <em>NBT</em>,
<em>42</em>(2), 179–182. (<a
href="https://doi.org/10.1038/s41587-023-02117-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intellectual property safeguards should be more widely incorporated into the bilateral contracts that underpin the translational R&amp;amp;D process to preserve the public interest, especially where public funds and resources have spearheaded the innovation process.},
  archive      = {J_NBT},
  author       = {Anderson, Mark and Hawkins, Naomi and Slade, Alison},
  doi          = {10.1038/s41587-023-02117-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {179-182},
  shortjournal = {Nature Biotech.},
  title        = {Contracts and intellectual property rights in translational R&amp;D: Furthering safeguards in the public interest},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Protein design: The experts speak. <em>NBT</em>,
<em>42</em>(2), 175–178. (<a
href="https://doi.org/10.1038/s41587-023-02111-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Following the publication of AlphaFold2 and RoseTTAFold in 2021, the field of protein structure prediction has moved quickly to incorporate these advances into protein engineering.},
  archive      = {J_NBT},
  author       = {Doerr, Anne},
  doi          = {10.1038/s41587-023-02111-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {175-178},
  shortjournal = {Nature Biotech.},
  title        = {Protein design: The experts speak},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). What does it take for an “AlphaFold moment” in functional
protein engineering and design? <em>NBT</em>, <em>42</em>(2), 173–174.
(<a href="https://doi.org/10.1038/s41587-023-02120-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chica, Roberto A. and Ferruz, Noelia},
  doi          = {10.1038/s41587-023-02120-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {173-174},
  shortjournal = {Nature Biotech.},
  title        = {What does it take for an ‘AlphaFold moment’ in functional protein engineering and design?},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). 2023 – biotech back in the mainstream. <em>NBT</em>,
<em>42</em>(2), 170–172. (<a
href="https://doi.org/10.1038/s41587-024-02126-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-024-02126-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {170-172},
  shortjournal = {Nature Biotech.},
  title        = {2023 – biotech back in the mainstream},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Drug pipeline 4Q23 — gene editing delivers. <em>NBT</em>,
<em>42</em>(2), 167–169. (<a
href="https://doi.org/10.1038/s41587-024-02125-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-024-02125-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {167-169},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 4Q23 — Gene editing delivers},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024j). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(2), 166. (<a
href="https://doi.org/10.1038/s41587-024-02139-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02139-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {166},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Merck enlists trispecific killers. <em>NBT</em>,
<em>42</em>(2), 165. (<a
href="https://doi.org/10.1038/s41587-024-02153-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02153-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {165},
  shortjournal = {Nature Biotech.},
  title        = {Merck enlists trispecific killers},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Microbe-made jet fuel. <em>NBT</em>, <em>42</em>(2),
163–166. (<a href="https://doi.org/10.1038/s41587-024-02136-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The aviation industry readies to embrace aviation fuel produced by fermentation.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-024-02136-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {163-166},
  shortjournal = {Nature Biotech.},
  title        = {Microbe-made jet fuel},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Tome launches CRISPR tool for oversized DNA. <em>NBT</em>,
<em>42</em>(2), 163. (<a
href="https://doi.org/10.1038/s41587-024-02152-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02152-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {163},
  shortjournal = {Nature Biotech.},
  title        = {Tome launches CRISPR tool for oversized DNA},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Row over curie building ends on a high note. <em>NBT</em>,
<em>42</em>(2), 162. (<a
href="https://doi.org/10.1038/s41587-024-02140-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02140-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {162},
  shortjournal = {Nature Biotech.},
  title        = {Row over curie building ends on a high note},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). First desmoid tumor drug. <em>NBT</em>, <em>42</em>(2), 161.
(<a href="https://doi.org/10.1038/s41587-024-02151-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02151-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {161},
  shortjournal = {Nature Biotech.},
  title        = {First desmoid tumor drug},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Can single-cell biology realize the promise of precision
medicine? <em>NBT</em>, <em>42</em>(2), 159–162. (<a
href="https://doi.org/10.1038/s41587-024-02138-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biology’s quiet revolution is underway, as single-cell tools fuel the next-wave of drug discoveries and promise to match therapies to the individual.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-024-02138-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {159-162},
  shortjournal = {Nature Biotech.},
  title        = {Can single-cell biology realize the promise of precision medicine?},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spotlight on protein structure design. <em>NBT</em>,
<em>42</em>(2), 157. (<a
href="https://doi.org/10.1038/s41587-024-02150-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New methods for protein design speed up workflows, but issues of training data availability and method optimization remain.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02150-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {157},
  shortjournal = {Nature Biotech.},
  title        = {Spotlight on protein structure design},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024l). People. <em>NBT</em>, <em>42</em>(1), 156. (<a
href="https://doi.org/10.1038/s41587-023-02089-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02089-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {156},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The impact of a graduate training and career outlook program
on diversity in the biotechnology sector. <em>NBT</em>, <em>42</em>(1),
148–155. (<a href="https://doi.org/10.1038/s41587-023-02084-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An intensive summer training program increased enrollment in a Master of Science in Biotechnology degree program, increasing awareness and opportunities among under-represented and underserved groups as a first step to transforming the biotech industry.},
  archive      = {J_NBT},
  author       = {Howard, Angelita P. and Slaughter, Liane S. and Adebowale, Joan and Bentley, Keisha and McPherson, Rebecca},
  doi          = {10.1038/s41587-023-02084-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {148-155},
  shortjournal = {Nature Biotech.},
  title        = {The impact of a graduate training and career outlook program on diversity in the biotechnology sector},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inference of phylogenetic trees directly from raw sequencing
reads using Read2Tree. <em>NBT</em>, <em>42</em>(1), 139–147. (<a
href="https://doi.org/10.1038/s41587-023-01753-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for inference of phylogenetic trees require running complex pipelines at substantial computational and labor costs, with additional constraints in sequencing coverage, assembly and annotation quality, especially for large datasets. To overcome these challenges, we present Read2Tree, which directly processes raw sequencing reads into groups of corresponding genes and bypasses traditional steps in phylogeny inference, such as genome assembly, annotation and all-versus-all sequence comparisons, while retaining accuracy. In a benchmark encompassing a broad variety of datasets, Read2Tree is 10–100 times faster than assembly-based approaches and in most cases more accurate—the exception being when sequencing coverage is high and reference species very distant. Here, to illustrate the broad applicability of the tool, we reconstruct a yeast tree of life of 435 species spanning 590 million years of evolution. We also apply Read2Tree to &amp;gt;10,000 Coronaviridae samples, accurately classifying highly diverse animal samples and near-identical severe acute respiratory syndrome coronavirus 2 sequences on a single tree. The speed, accuracy and versatility of Read2Tree enable comparative genomics at scale. Phylogenetic trees are generated from sequencing reads without genome assembly or annotation.},
  archive      = {J_NBT},
  author       = {Dylus, David and Altenhoff, Adrian and Majidian, Sina and Sedlazeck, Fritz J. and Dessimoz, Christophe},
  doi          = {10.1038/s41587-023-01753-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {139-147},
  shortjournal = {Nature Biotech.},
  title        = {Inference of phylogenetic trees directly from raw sequencing reads using Read2Tree},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Sequencing by avidity enables high accuracy with low reagent
consumption. <em>NBT</em>, <em>42</em>(1), 132–138. (<a
href="https://doi.org/10.1038/s41587-023-01750-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present avidity sequencing, a sequencing chemistry that separately optimizes the processes of stepping along a DNA template and that of identifying each nucleotide within the template. Nucleotide identification uses multivalent nucleotide ligands on dye-labeled cores to form polymerase–polymer–nucleotide complexes bound to clonal copies of DNA targets. These polymer–nucleotide substrates, termed avidites, decrease the required concentration of reporting nucleotides from micromolar to nanomolar and yield negligible dissociation rates. Avidity sequencing achieves high accuracy, with 96.2% and 85.4% of base calls having an average of one error per 1,000 and 10,000 base pairs, respectively. We show that the average error rate of avidity sequencing remained stable following a long homopolymer. A sequencing chemistry that separates nucleotide identification from nucleotide incorporation achieves high accuracy.},
  archive      = {J_NBT},
  author       = {Arslan, Sinan and Garcia, Francisco J. and Guo, Minghao and Kellinger, Matthew W. and Kruglyak, Semyon and LeVieux, Jake A. and Mah, Adeline H. and Wang, Haosen and Zhao, Junhua and Zhou, Chunhong and Altomare, Andrew and Bailey, John and Byrne, Matthew B. and Chang, Chiting and Chen, Steve X. and Cho, Byungrae and Dennler, Claudia N. and Dien, Vivian T. and Fuller, Derek and Kelley, Ryan and Khandan, Omid and Klein, Michael G. and Kim, Michael and Lajoie, Bryan R. and Lin, Bill and Liu, Yu and Lopez, Tyler and Mains, Peter T. and Price, Andrew D. and Robertson, Samantha R. and Taylor-Weiner, Hermes and Tippana, Ramreddy and Tomaney, Austin B. and Zhang, Su and Abtahi, Minna and Ambroso, Mark R. and Bajari, Rosita and Bellizzi, Ava M. and Benitez, Chris B. and Berard, Daniel R. and Berti, Lorenzo and Blease, Kelly N. and Blum, Angela P. and Boddicker, Andrew M. and Bondar, Leo and Brown, Chris and Bui, Chris A. and Calleja-Aguirre, Juan and Cappa, Kevin and Chan, Joshua and Chang, Victor W. and Charov, Katherine and Chen, Xiyi and Constandse, Rodger M. and Damron, Weston and Dawood, Mariam and DeBuono, Nicole and Dimalanta, John D. and Edoli, Laure and Elango, Keerthana and Faustino, Nikka and Feng, Chao and Ferrari, Matthew and Frankie, Keith and Fries, Adam and Galloway, Anne and Gavrila, Vlad and Gemmen, Gregory J. and Ghadiali, James and Ghorbani, Arash and Goddard, Logan A. and Guetter, Adriana Roginski and Hendricks, Garren L. and Hentschel, Jendrik and Honigfort, Daniel J. and Hsieh, Yun-Ting and Hwang Fu, Yu-Hsien and Im, Scott K. and Jin, Chaoyi and Kabu, Shradha and Kincade, Daniel E. and Levy, Shawn and Li, Yu and Liang, Vincent K. and Light, William H. and Lipsher, Jonathan B. and Liu, Tsung-li and Long, Grace and Ma, Rui and Mailloux, John M. and Mandla, Kyle A. and Martinez, Anyssa R. and Mass, Max and McKean, Daniel T. and Meron, Michael and Miller, Edmund A. and Moh, Celyne S. and Moore, Rachel K. and Moreno, Juan and Neysmith, Jordan M. and Niman, Cassandra S. and Nunez, Jesus M. and Ojeda, Micah T. and Ortiz, Sara Espinosa and Owens, Jenna and Piland, Geoffrey and Proctor, Daniel J. and Purba, Josua B. and Ray, Michael and Rong, Daisong and Saade, Virginia M. and Saha, Sanchari and Tomas, Gustav Santo and Scheidler, Nicholas and Sirajudeen, Luqmanal H. and Snow, Samantha and Stengel, Gudrun and Stinson, Ryan and Stone, Michael J. and Sundseth, Keoni J. and Thai, Eileen and Thompson, Connor J. and Tjioe, Marco and Trejo, Christy L. and Trieger, Greg and Truong, Diane Ni and Tse, Ben and Voiles, Benjamin and Vuong, Henry and Wong, Jennifer C. and Wu, Chiung-Ting and Yu, Hua and Yu, Yingxian and Yu, Ming and Zhang, Xi and Zhao, Da and Zheng, Genhua and He, Molly and Previte, Michael},
  doi          = {10.1038/s41587-023-01750-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {132-138},
  shortjournal = {Nature Biotech.},
  title        = {Sequencing by avidity enables high accuracy with low reagent consumption},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). RIP-PEN-seq identifies a class of kink-turn RNAs as splicing
regulators. <em>NBT</em>, <em>42</em>(1), 119–131. (<a
href="https://doi.org/10.1038/s41587-023-01749-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A kink-turn (K-turn) is a three-dimensional RNA structure that exists in all three primary phylogenetic domains. In this study, we developed the RIP-PEN-seq method to identify the full-length sequences of RNAs bound by the K-turn binding protein 15.5K and discovered a previously uncharacterized class of RNAs with backward K-turn motifs (bktRNAs) in humans and mice. All bktRNAs share two consensus sequence motifs at their fixed terminal position and have complex folding properties, expression and evolution patterns. We found that a highly conserved bktRNA1 guides the methyltransferase fibrillarin to install RNA methylation of U12 small nuclear RNA in humans. Depletion of bktRNA1 causes global splicing dysregulation of U12-type introns by impairing the recruitment of ZCRB1 to the minor spliceosome. Most bktRNAs regulate the splicing of local introns by interacting with the 15.5K protein. Taken together, our findings characterize a class of small RNAs and uncover another layer of gene expression regulation that involves crosstalk among bktRNAs, RNA splicing and RNA methylation. Immunoprecipitation coupled with paired-end sequencing of non-coding RNAs identifies a class of splicing regulators.},
  archive      = {J_NBT},
  author       = {Li, Bin and Liu, Shurong and Zheng, Wujian and Liu, Anrui and Yu, Peng and Wu, Di and Zhou, Jie and Zhang, Ping and Liu, Chang and Lin, Qiao and Ye, Jiayi and He, Simeng and Huang, Qiaojuan and Zhou, Hui and Chen, Jianjun and Qu, Lianghu and Yang, Jianhua},
  doi          = {10.1038/s41587-023-01749-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {119-131},
  shortjournal = {Nature Biotech.},
  title        = {RIP-PEN-seq identifies a class of kink-turn RNAs as splicing regulators},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Characterizing expression changes in noncoding RNAs during
aging and heterochronic parabiosis across mouse tissues. <em>NBT</em>,
<em>42</em>(1), 109–118. (<a
href="https://doi.org/10.1038/s41587-023-01751-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular mechanisms of organismal and cell aging remain incompletely understood. We, therefore, generated a body-wide map of noncoding RNA (ncRNA) expression in aging (16 organs at ten timepoints from 1 to 27 months) and rejuvenated mice. We found molecular aging trajectories are largely tissue-specific except for eight broadly deregulated microRNAs (miRNAs). Their individual abundance mirrors their presence in circulating plasma and extracellular vesicles (EVs) whereas tissue-specific ncRNAs were less present. For miR-29c-3p, we observe the largest correlation with aging in solid organs, plasma and EVs. In mice rejuvenated by heterochronic parabiosis, miR-29c-3p was the most prominent miRNA restored to similar levels found in young liver. miR-29c-3p targets the extracellular matrix and secretion pathways, known to be implicated in aging. We provide a map of organism-wide expression of ncRNAs with aging and rejuvenation and identify a set of broadly deregulated miRNAs, which may function as systemic regulators of aging via plasma and EVs. An atlas of noncoding RNA in aging and rejuvenated mice identifies a set of deregulated miRNAs.},
  archive      = {J_NBT},
  author       = {Wagner, Viktoria and Kern, Fabian and Hahn, Oliver and Schaum, Nicholas and Ludwig, Nicole and Fehlmann, Tobias and Engel, Annika and Henn, Dominic and Rishik, Shusruto and Isakova, Alina and Tan, Michelle and Sit, Rene and Neff, Norma and Hart, Martin and Meese, Eckart and Quake, Steve and Wyss-Coray, Tony and Keller, Andreas},
  doi          = {10.1038/s41587-023-01751-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {109-118},
  shortjournal = {Nature Biotech.},
  title        = {Characterizing expression changes in noncoding RNAs during aging and heterochronic parabiosis across mouse tissues},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A relay velocity model infers cell-dependent RNA velocity.
<em>NBT</em>, <em>42</em>(1), 99–108. (<a
href="https://doi.org/10.1038/s41587-023-01728-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA velocity provides an approach for inferring cellular state transitions from single-cell RNA sequencing (scRNA-seq) data. Conventional RNA velocity models infer universal kinetics from all cells in an scRNA-seq experiment, resulting in unpredictable performance in experiments with multi-stage and/or multi-lineage transition of cell states where the assumption of the same kinetic rates for all cells no longer holds. Here we present cellDancer, a scalable deep neural network that locally infers velocity for each cell from its neighbors and then relays a series of local velocities to provide single-cell resolution inference of velocity kinetics. In the simulation benchmark, cellDancer shows robust performance in multiple kinetic regimes, high dropout ratio datasets and sparse datasets. We show that cellDancer overcomes the limitations of existing RNA velocity models in modeling erythroid maturation and hippocampus development. Moreover, cellDancer provides cell-specific predictions of transcription, splicing and degradation rates, which we identify as potential indicators of cell fate in the mouse pancreas. cellDancer enables RNA velocity estimation with cell-specific kinetics.},
  archive      = {J_NBT},
  author       = {Li, Shengyu and Zhang, Pengzhi and Chen, Weiqing and Ye, Lingqun and Brannan, Kristopher W. and Le, Nhat-Tu and Abe, Jun-ichi and Cooke, John P. and Wang, Guangyu},
  doi          = {10.1038/s41587-023-01728-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {99-108},
  shortjournal = {Nature Biotech.},
  title        = {A relay velocity model infers cell-dependent RNA velocity},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Targeted DNA integration in human cells without
double-strand breaks using CRISPR-associated transposases. <em>NBT</em>,
<em>42</em>(1), 87–98. (<a
href="https://doi.org/10.1038/s41587-023-01748-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conventional genome engineering with CRISPR–Cas9 creates double-strand breaks (DSBs) that lead to undesirable byproducts and reduce product purity. Here we report an approach for programmable integration of large DNA sequences in human cells that avoids the generation of DSBs by using Type I-F CRISPR-associated transposases (CASTs). We optimized DNA targeting by the QCascade complex through protein design and developed potent transcriptional activators by exploiting the multi-valent recruitment of the AAA+ ATPase TnsC to genomic sites targeted by QCascade. After initial detection of plasmid-based integration, we screened 15 additional CAST systems from a wide range of bacterial hosts, identified a homolog from Pseudoalteromonas that exhibits improved activity and further increased integration efficiencies. Finally, we discovered that bacterial ClpX enhances genomic integration by multiple orders of magnitude, likely by promoting active disassembly of the post-integration CAST complex, akin to its known role in Mu transposition. Our work highlights the ability to reconstitute complex, multi-component machineries in human cells and establishes a strong foundation to exploit CRISPR-associated transposases for eukaryotic genome engineering. Large DNA sequences are integrated into human cells using RNA-guided CRISPR-associated transposases.},
  archive      = {J_NBT},
  author       = {Lampe, George D. and King, Rebeca T. and Halpin-Healy, Tyler S. and Klompe, Sanne E. and Hogan, Marcus I. and Vo, Phuc Leo H. and Tang, Stephen and Chavez, Alejandro and Sternberg, Samuel H.},
  doi          = {10.1038/s41587-023-01748-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {87-98},
  shortjournal = {Nature Biotech.},
  title        = {Targeted DNA integration in human cells without double-strand breaks using CRISPR-associated transposases},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantitative analysis of tRNA abundance and modifications by
nanopore RNA sequencing. <em>NBT</em>, <em>42</em>(1), 72–86. (<a
href="https://doi.org/10.1038/s41587-023-01743-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transfer RNAs (tRNAs) play a central role in protein translation. Studying them has been difficult in part because a simple method to simultaneously quantify their abundance and chemical modifications is lacking. Here we introduce Nano-tRNAseq, a nanopore-based approach to sequence native tRNA populations that provides quantitative estimates of both tRNA abundances and modification dynamics in a single experiment. We show that default nanopore sequencing settings discard the vast majority of tRNA reads, leading to poor sequencing yields and biased representations of tRNA abundances based on their transcript length. Re-processing of raw nanopore current intensity signals leads to a 12-fold increase in the number of recovered tRNA reads and enables recapitulation of accurate tRNA abundances. We then apply Nano-tRNAseq to Saccharomyces cerevisiae tRNA populations, revealing crosstalks and interdependencies between different tRNA modification types within the same molecule and changes in tRNA populations in response to oxidative stress. tRNA abundance and chemical modifications are measured simultaneously with nanopores.},
  archive      = {J_NBT},
  author       = {Lucas, Morghan C. and Pryszcz, Leszek P. and Medina, Rebeca and Milenkovic, Ivan and Camacho, Noelia and Marchand, Virginie and Motorin, Yuri and Ribas de Pouplana, Lluís and Novoa, Eva Maria},
  doi          = {10.1038/s41587-023-01743-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {72-86},
  shortjournal = {Nature Biotech.},
  title        = {Quantitative analysis of tRNA abundance and modifications by nanopore RNA sequencing},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Reflective multi-immersion microscope objectives inspired by
the schmidt telescope. <em>NBT</em>, <em>42</em>(1), 65–71. (<a
href="https://doi.org/10.1038/s41587-023-01717-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging large, cleared samples requires microscope objectives that combine a large field of view (FOV) with a long working distance (WD) and a high numerical aperture (NA). Ideally, such objectives should be compatible with a wide range of immersion media, which is challenging to achieve with conventional lens-based objective designs. Here we introduce the multi-immersion ‘Schmidt objective’ consisting of a spherical mirror and an aspherical correction plate as a solution to this problem. We demonstrate that a multi-photon variant of the Schmidt objective is compatible with all homogeneous immersion media and achieves an NA of 1.08 at a refractive index of 1.56, 1.1-mm FOV and 11-mm WD. We highlight its versatility by imaging cleared samples in various media ranging from air and water to benzyl alcohol/benzyl benzoate, dibenzyl ether and ethyl cinnamate and by imaging of neuronal activity in larval zebrafish in vivo. In principle, the concept can be extended to any imaging modality, including wide-field, confocal and light-sheet microscopy. Imaging of large, cleared samples in diverse media is achieved using a mirror objective.},
  archive      = {J_NBT},
  author       = {Voigt, Fabian F. and Reuss, Anna Maria and Naert, Thomas and Hildebrand, Sven and Schaettin, Martina and Hotz, Adriana L. and Whitehead, Lachlan and Bahl, Armin and Neuhauss, Stephan C. F. and Roebroeck, Alard and Stoeckli, Esther T. and Lienkamp, Soeren S. and Aguzzi, Adriano and Helmchen, Fritjof},
  doi          = {10.1038/s41587-023-01717-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {65-71},
  shortjournal = {Nature Biotech.},
  title        = {Reflective multi-immersion microscope objectives inspired by the schmidt telescope},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DisP-seq reveals the genome-wide functional organization of
DNA-associated disordered proteins. <em>NBT</em>, <em>42</em>(1), 52–64.
(<a href="https://doi.org/10.1038/s41587-023-01737-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intrinsically disordered regions (IDRs) in DNA-associated proteins are known to influence gene regulation, but their distribution and cooperative functions in genome-wide regulatory programs remain poorly understood. Here we describe DisP-seq (disordered protein precipitation followed by DNA sequencing), an antibody-independent chemical precipitation assay that can simultaneously map endogenous DNA-associated disordered proteins genome-wide through a combination of biotinylated isoxazole precipitation and next-generation sequencing. DisP-seq profiles are composed of thousands of peaks that are associated with diverse chromatin states, are enriched for disordered transcription factors (TFs) and are often arranged in large lineage-specific clusters with high local concentrations of disordered proteins and different combinations of histone modifications linked to regulatory potential. We use DisP-seq to analyze cancer cells and reveal how disordered protein-associated islands enable IDR-dependent mechanisms that control the binding and function of disordered TFs, including oncogene-dependent sequestration of TFs through long-range interactions and the reactivation of differentiation pathways upon loss of oncogenic stimuli in Ewing sarcoma. Mapping DNA-associated disordered proteins across the genome uncovers gene regulatory patterns.},
  archive      = {J_NBT},
  author       = {Xing, Yu-Hang and Dong, Rui and Lee, Lukuo and Rengarajan, Shruthi and Riggi, Nicolò and Boulay, Gaylor and Rivera, Miguel N.},
  doi          = {10.1038/s41587-023-01737-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {52-64},
  shortjournal = {Nature Biotech.},
  title        = {DisP-seq reveals the genome-wide functional organization of DNA-associated disordered proteins},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Dandelion uses the single-cell adaptive immune receptor
repertoire to explore lymphocyte developmental origins. <em>NBT</em>,
<em>42</em>(1), 40–51. (<a
href="https://doi.org/10.1038/s41587-023-01734-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assessment of single-cell gene expression (single-cell RNA sequencing) and adaptive immune receptor (AIR) sequencing (scVDJ-seq) has been invaluable in studying lymphocyte biology. Here we introduce Dandelion, a computational pipeline for scVDJ-seq analysis. It enables the application of standard V(D)J analysis workflows to single-cell datasets, delivering improved V(D)J contig annotation and the identification of nonproductive and partially spliced contigs. We devised a strategy to create an AIR feature space that can be used for both differential V(D)J usage analysis and pseudotime trajectory inference. The application of Dandelion improved the alignment of human thymic development trajectories of double-positive T cells to mature single-positive CD4/CD8 T cells, generating predictions of factors regulating lineage commitment. Dandelion analysis of other cell compartments provided insights into the origins of human B1 cells and ILC/NK cell development, illustrating the power of our approach. Dandelion is available at https://www.github.com/zktuong/dandelion . A computational pipeline enables differential V(D)J usage analysis and pseudotime trajectory inference from single-cell AIR sequencing.},
  archive      = {J_NBT},
  author       = {Suo, Chenqu and Polanski, Krzysztof and Dann, Emma and Lindeboom, Rik G. H. and Vilarrasa-Blasi, Roser and Vento-Tormo, Roser and Haniffa, Muzlifah and Meyer, Kerstin B. and Dratva, Lisa M. and Tuong, Zewen Kelvin and Clatworthy, Menna R. and Teichmann, Sarah A.},
  doi          = {10.1038/s41587-023-01734-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {40-51},
  shortjournal = {Nature Biotech.},
  title        = {Dandelion uses the single-cell adaptive immune receptor repertoire to explore lymphocyte developmental origins},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Exploring tRNAs and their modifications and crosstalk using
nano-tRNAseq. <em>NBT</em>, <em>42</em>(1), 38–39. (<a
href="https://doi.org/10.1038/s41587-023-01755-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nano-tRNAseq is a nanopore-based, cost-effective and high-throughput approach to quantify transfer RNA (tRNA) abundances and modifications simultaneously, providing a framework to study the ‘tRNAome’ at single-molecule resolution. We envision that Nano-tRNAseq will enable us to study the role of tRNA molecules and their modifications in a wide variety of contexts.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01755-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {38-39},
  shortjournal = {Nature Biotech.},
  title        = {Exploring tRNAs and their modifications and crosstalk using nano-tRNAseq},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Imaging the biological microcosmos with a tiny telescope.
<em>NBT</em>, <em>42</em>(1), 36–37. (<a
href="https://doi.org/10.1038/s41587-023-01740-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present the Schmidt objective, a novel astronomy-inspired concept for designing multi-immersion microscope objectives with high numerical aperture, long working distance and large field of view. The Schmidt objective uses a spherical mirror and a correction plate to focus light. It is well suited for high-resolution imaging deep inside cleared biological samples.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01740-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {36-37},
  shortjournal = {Nature Biotech.},
  title        = {Imaging the biological microcosmos with a tiny telescope},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A genome-wide view of disordered proteins. <em>NBT</em>,
<em>42</em>(1), 32–34. (<a
href="https://doi.org/10.1038/s41587-023-01955-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factors containing disordered regions can now be mapped across the genome, aiding functional studies.},
  archive      = {J_NBT},
  author       = {Leslie, Benjamin J. and Lang, Benjamin and Babu, M. Madan},
  doi          = {10.1038/s41587-023-01955-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {32-34},
  shortjournal = {Nature Biotech.},
  title        = {A genome-wide view of disordered proteins},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Designer enhancers for cell-type-specific gene regulation.
<em>NBT</em>, <em>42</em>(1), 31. (<a
href="https://doi.org/10.1038/s41587-023-02112-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Despang, Alexandra},
  doi          = {10.1038/s41587-023-02112-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {31},
  shortjournal = {Nature Biotech.},
  title        = {Designer enhancers for cell-type-specific gene regulation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). N1-methylpseudouridylation affects the fidelity of mRNA
translation. <em>NBT</em>, <em>42</em>(1), 31. (<a
href="https://doi.org/10.1038/s41587-023-02113-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Li, Rong},
  doi          = {10.1038/s41587-023-02113-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {31},
  shortjournal = {Nature Biotech.},
  title        = {N1-methylpseudouridylation affects the fidelity of mRNA translation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Live cell imaging of signaling networks using a conventional
microscope. <em>NBT</em>, <em>42</em>(1), 31. (<a
href="https://doi.org/10.1038/s41587-023-02116-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bishop, Anahita},
  doi          = {10.1038/s41587-023-02116-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {31},
  shortjournal = {Nature Biotech.},
  title        = {Live cell imaging of signaling networks using a conventional microscope},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Xenotransplantation. <em>NBT</em>, <em>42</em>(1), 30. (<a
href="https://doi.org/10.1038/s41587-023-02077-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to generating transgenic animals, tissues and cells for xenotransplantation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02077-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nature Biotech.},
  title        = {Xenotransplantation},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Discovering value: Women’s participation in university and
commercial AI invention. <em>NBT</em>, <em>42</em>(1), 26–29. (<a
href="https://doi.org/10.1038/s41587-023-02075-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More can be done to diversify the AI innovation ecosystem, especially in the emerging field of biotechnology AI, where women’s participation is associated with substantial value.},
  archive      = {J_NBT},
  author       = {Giczy, Alexander V. and Pairolero, Nicholas A. and Toole, Andrew A.},
  doi          = {10.1038/s41587-023-02075-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {26-29},
  shortjournal = {Nature Biotech.},
  title        = {Discovering value: Women’s participation in university and commercial AI invention},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Growth of biotech clusters over several decades through
pioneering, variety and entrepreneurial science. <em>NBT</em>,
<em>42</em>(1), 20–25. (<a
href="https://doi.org/10.1038/s41587-023-02062-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biotechnology (biotech) clusters have evolved over time. Being a top biotech region in the early years 1978–1990 still positively correlates with the cluster’s biotech strength decades later. An entrepreneurial orientation of scientific actors as well as a variety of networks with partners outside the cluster both contribute to sustained biotech activity.},
  archive      = {J_NBT},
  author       = {Van Looy, Bart and Grabowska, Marcelina and Vlaisavljevic, Vesna and Lecocq, Catherine and Debackere, Koenraad},
  doi          = {10.1038/s41587-023-02062-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {20-25},
  shortjournal = {Nature Biotech.},
  title        = {Growth of biotech clusters over several decades through pioneering, variety and entrepreneurial science},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). It takes two to think. <em>NBT</em>, <em>42</em>(1), 18–19.
(<a href="https://doi.org/10.1038/s41587-023-02074-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yanai, Itai and Lercher, Martin J.},
  doi          = {10.1038/s41587-023-02074-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {18-19},
  shortjournal = {Nature Biotech.},
  title        = {It takes two to think},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cell and gene therapy approvals in japan and the need for
international harmonization. <em>NBT</em>, <em>42</em>(1), 13–17. (<a
href="https://doi.org/10.1038/s41587-023-02053-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Arato, Teruyo and Nomura, Kaori},
  doi          = {10.1038/s41587-023-02053-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {13-17},
  shortjournal = {Nature Biotech.},
  title        = {Cell and gene therapy approvals in japan and the need for international harmonization},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Reducing the costs of blockbuster gene and cell therapies
in the global south. <em>NBT</em>, <em>42</em>(1), 8–12. (<a
href="https://doi.org/10.1038/s41587-023-02049-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers in low- and middle-income countries are developing their own IP, scaling up local manufacturing, and looking for biomarkers — all in the hope of bringing costs down and getting therapies to people who need them.},
  archive      = {J_NBT},
  author       = {Johnson, Ben},
  doi          = {10.1038/s41587-023-02049-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {8-12},
  shortjournal = {Nature Biotech.},
  title        = {Reducing the costs of blockbuster gene and cell therapies in the global south},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024k). Biotech news from around the world. <em>NBT</em>,
<em>42</em>(1), 7. (<a
href="https://doi.org/10.1038/s41587-023-02093-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02093-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {7},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Top biopharmas’ zero-carbon drive. <em>NBT</em>,
<em>42</em>(1), 6. (<a
href="https://doi.org/10.1038/s41587-023-02102-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02102-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {6},
  shortjournal = {Nature Biotech.},
  title        = {Top biopharmas’ zero-carbon drive},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Super-resistant CAR ts take on cancers. <em>NBT</em>,
<em>42</em>(1), 5–7. (<a
href="https://doi.org/10.1038/s41587-023-02092-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sophisticated genome editing tools to tweak multiple genes in one step are forging a first-of-its-kind generation of universal CAR T therapeutics, while ‘epitope-edited’ and virus-free manufacturing aim to democratize CAR T cells’ clinical uptake worldwide.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-023-02092-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {5-7},
  shortjournal = {Nature Biotech.},
  title        = {Super-resistant CAR ts take on cancers},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Denmark invests in mucosal vaccines. <em>NBT</em>,
<em>42</em>(1), 5. (<a
href="https://doi.org/10.1038/s41587-023-02104-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02104-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {5},
  shortjournal = {Nature Biotech.},
  title        = {Denmark invests in mucosal vaccines},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). First self-amplifying mRNA vaccine approved. <em>NBT</em>,
<em>42</em>(1), 4. (<a
href="https://doi.org/10.1038/s41587-023-02101-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02101-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nature Biotech.},
  title        = {First self-amplifying mRNA vaccine approved},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Tearing up the traditional biotech playbook. <em>NBT</em>,
<em>42</em>(1), 1. (<a
href="https://doi.org/10.1038/s41587-023-02119-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bigger and more expensive science requires more extensive collaboration and new funding opportunities.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02119-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nature Biotech.},
  title        = {Tearing up the traditional biotech playbook},
  volume       = {42},
  year         = {2024},
}
</textarea>
</details></li>
</ul>

</body>
</html>
