<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>EVLETT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="evlett---44">EVLETT - 44</h2>
<ul>
<li><details>
<summary>
(2023). Germline mutation rate is elevated in young and old parents
in caenorhabditis remanei. <em>EVLETT</em>, <em>7</em>(6), 478–489. (<a
href="https://doi.org/10.1093/evlett/qrad052">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effect of parental age on germline mutation rate across generations is not fully understood. While some studies report a positive linear relationship of mutation rate with increasing age, others suggest that mutation rate varies with age but not in a linear fashion. We investigated the effect of parental age on germline mutations by generating replicated mutation accumulation lines in Caenorhabditis remanei at three parental ages (“Young T1” [Day 1], “Peak T2” [Day 2], and “Old T5” [Day 5] parents). We conducted whole-genome resequencing and variant calling to compare differences in mutation rates after three generations of mutation accumulation. We found that Peak T2 lines had an overall reduced mutation rate compared to Young T1 and Old T5 lines, but this pattern of the effect varied depending on the variant impact. Specifically, we found no high-impact variants in Peak T2 lines, and modifiers and up- and downstream gene variants were less frequent in these lines. These results suggest that animals at the peak of reproduction have better DNA maintenance and repair compared to young and old animals. We propose that C. remanei start to reproduce before they optimize their DNA maintenance and repair, trading the benefits of earlier onset of reproduction against offspring mutation load. The increase in offspring mutation load with age likely represents germline senescence. Germline mutations play a key role in evolution through the generation of novel genotypes. Estimating the mutation rate in species, populations, and individuals is one way to understand the relative timeframe of evolutionary processes, for the timing of historical events and for estimating heritability of traits and diseases. Individual age at reproduction is known to affect the number of mutations being transferred into the next generation and generally mutation rate is thought to increase with increasing parental age. However, preventing mutations in germ cells is potentially costly and it may pay off to optimize germline genome repair and maintenance during peak reproductive periods, and relax it during nonpeak periods. This idea has been put forward to explain for example the reduction of gonad size in seasonally reproducing animals during nonreproductive periods and supported by the finding that the mutation rate seems to be higher in teenage men compared to men during their peak reproductive ages. We further tested this idea of a nonlinear relationship between age and mutation rate by performing a mutation accumulation experiment in a short-lived nematode. We kept experimental lines and allowed adults to reproduce at different ages in different lines, with some lines reproducing before, some during, and some after their reproductive peak. We found that mutation rates are higher in nematode lines reproducing before or after the reproductive peak compared to those reproducing during the peak. Our results therefore support the idea that germline genome maintenance and repair is potentially costly and that the mutation rate does not just increase with age but is optimized during the peak reproductive age of an organism.},
  archive      = {J_EVLETT},
  author       = {Chen, Hwei-yen and Krieg, Therese and Mautz, Brian and Jolly, Cécile and Scofield, Douglas and Maklakov, Alexei A and Immler, Simone},
  doi          = {10.1093/evlett/qrad052},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {478-489},
  shortjournal = {Evol. Lett.},
  title        = {Germline mutation rate is elevated in young and old parents in caenorhabditis remanei},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Host–pathogen coevolution promotes the evolution of general,
broad-spectrum resistance and reduces foreign pathogen spillover risk.
<em>EVLETT</em>, <em>7</em>(6), 467–477. (<a
href="https://doi.org/10.1093/evlett/qrad051">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variation for disease resistance within host populations can strongly impact the spread of endemic pathogens. In plants, recent work has shown that within-population variation in resistance can also affect the transmission of foreign spillover pathogens if that resistance is general. However, most hosts also possess specific resistance mechanisms that provide strong defenses against coevolved endemic pathogens. Here we use a modeling approach to ask how antagonistic coevolution between hosts and their endemic pathogen at the specific resistance locus can affect the frequency of general resistance, and therefore a host’s vulnerability to foreign pathogens. We develop a two-locus model with variable recombination that incorporates both general resistance (effective against all pathogens) and specific resistance (effective against endemic pathogens only). With coevolution, when pathogens can evolve to evade specific resistance, we find that the regions where general resistance can evolve are greatly expanded, decreasing the risk of foreign pathogen invasion. Furthermore, coevolution greatly expands the conditions that maintain polymorphisms at both resistance loci, thereby driving greater genetic diversity within host populations. This genetic diversity often leads to positive correlations between host resistance to foreign and endemic pathogens, similar to those observed in natural populations. However, if resistance loci become linked, the resistance correlations can shift to negative. If we include a third linkage-modifying locus in our model, we find that selection often favors complete linkage. Our model demonstrates how coevolutionary dynamics with an endemic pathogen can mold the resistance structure of host populations in ways that affect its susceptibility to foreign pathogen spillovers, and that the nature of these outcomes depends on resistance costs, as well as the degree of linkage between resistance genes. Spillovers, where a species of pathogen infects a novel type of host, are critically important to biodiversity, human health, and agriculture. Our current understanding of the factors driving spillover risks focuses on species-level factors, such as the degree of genetic relatedness between novel and ancestral hosts. However, recent work has demonstrated that within a species, some individuals can be significantly more vulnerable to foreign pathogens than others, even when the host and foreign pathogen share no coevolutionary history. To gain a more complete understanding of spillover risks, theory must fill a critical gap in our current understanding: how this variation evolves and is maintained. One potential answer lies in the fact that hosts typically have multiple mechanisms of resistance against pathogens. Some of these mechanisms can be highly tuned to a single pathogen, while others are more broadly effective. These broadly effective forms of resistance are much more likely to protect hosts against foreign pathogens where the hosts have not had time to evolve specific resistance. To investigate how multiple forms of resistance may generate diversity in susceptibility to foreign pathogens, we developed a mathematical model incorporating both general resistance (effective against multiple pathogen species) and specific resistance (effective against a single pathogen genotype). In a restrictive setting, where pathogens cannot evolve in response to host evolution, the evolution of specific resistance often prevents the evolution of general resistance. However, when pathogens can themselves evolve in response to host evolution, the coevolutionary arms race between host and pathogen allows for a significant subset of the population to maintain general resistance, and thus resistance to foreign pathogens. At the population level, our model recapitulates patterns of resistance correlation seen in natural populations, where individuals more resistant to endemic pathogens are also likely to be more resistant to foreign pathogens.},
  archive      = {J_EVLETT},
  author       = {Hulse, Samuel V and Antonovics, Janis and Hood, Michael E and Bruns, Emily L},
  doi          = {10.1093/evlett/qrad051},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {467-477},
  shortjournal = {Evol. Lett.},
  title        = {Host–pathogen coevolution promotes the evolution of general, broad-spectrum resistance and reduces foreign pathogen spillover risk},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The evolution of genetic covariance and modularity as a
result of multigenerational environmental fluctuation. <em>EVLETT</em>,
<em>7</em>(6), 457–466. (<a
href="https://doi.org/10.1093/evlett/qrad048">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic covariance between traits can affect the evolution of a population through selection, drift, and migration. Conversely, research has demonstrated the reciprocal effect of evolutionary processes on changing genetic covariances, in part through mutational covariance, correlational selection, and plasticity. In this article, we propose that correlated changes in selective optima over generations can cause the evolution of genetic covariance and the G -matrix in such a way that the population can, in the future, evolve faster. We use individual-based simulations of populations exposed to three types of changing environments that differ in the correlation of the change between selective pressures. Our simulation experiments demonstrate that selection pressures for different traits changing in a correlated pattern over generations can lead to stronger trait correlations compared to the case with independently changing selective optima. Our findings show that correlated selective pressures result in significantly higher genetic trait covariance and that pleiotropy accounts for the majority of the difference in covariance between treatments. We also observe that the mutational variance evolves according to the environment that the populations were exposed to. Moreover, we show that clustered patterns of changes in selection can allow the evolution of genetic modularity. We show that the pattern of change in the selective environment affects the pace at which fitness evolves, with populations experiencing correlated change in optima having on average higher mean fitness than those experiencing uncorrelated environment change. Trait covariance describes the relationship between the variance of two or more traits. The genetic covariance is the heritable portion of the covariance, and it can influence how a biological population changes through time. Researchers have also shown how the reciprocal effect is true, that is, how the evolution of a population may also impact the genetic covariance of traits. Here we are proposing one way in which evolution, specifically evolution through natural selection, can change the genetic covariance of traits. We call the mechanism we are proposing “correlated environmental change” because it describes a changing environment where the amount and direction of change over different traits are the same. Our results show that when the environment changes in such a way, compared to a not correlated way, genetic covariance between traits increases. Additionally, most of the covariance we find is a result of a heritable element (i.e., gene) impacting multiple traits. We also show how the future change given by mutations that impact these traits are more likely to happen in the direction of the changing environment. Finally, according to our results, populations become adapted to the way the environment changes, and are able to reach higher fitness faster when reexposed to the same type of change they were originally exposed to compared to a new type of change.},
  archive      = {J_EVLETT},
  author       = {do O, Isabela and Whitlock, Michael C},
  doi          = {10.1093/evlett/qrad048},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {457-466},
  shortjournal = {Evol. Lett.},
  title        = {The evolution of genetic covariance and modularity as a result of multigenerational environmental fluctuation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Experimental evidence that network topology can accelerate
the spread of beneficial mutations. <em>EVLETT</em>, <em>7</em>(6),
447–456. (<a href="https://doi.org/10.1093/evlett/qrad047">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whether and how the spatial arrangement of a population influences adaptive evolution has puzzled evolutionary biologists. Theoretical models make conflicting predictions about the probability that a beneficial mutation will become fixed in a population for certain topologies like stars, in which “leaf” populations are connected through a central “hub.” To date, these predictions have not been evaluated under realistic experimental conditions. Here, we test the prediction that topology can change the dynamics of fixation both in vitro and in silico by tracking the frequency of a beneficial mutant under positive selection as it spreads through networks of different topologies. Our results provide empirical support that meta-population topology can increase the likelihood that a beneficial mutation spreads, broaden the conditions under which this phenomenon is thought to occur, and points the way toward using network topology to amplify the effects of weakly favored mutations under directed evolution in industrial applications. Most natural populations are spatially structured, meaning that they are geographically subdivided and connected by migration. Whether spatial structure impacts adaptive evolution has been less clear as different theoretical approaches to modeling the spread of beneficial mutations through spatially structured populations make contrasting predictions. Our work uses evolution experiments and numerical simulations to show that spatial structure can impact the pace of adaptive evolution. Certain topologies, specifically a four-patch star with bidirectional migration through a central hub to each of three “leaf” populations, can accelerate the rate at which a beneficial mutation spreads through a population relative to an unstructured, well-mixed population. The cause of this acceleration is a reduced probability that beneficial mutations are stochastically lost when rare because they get concentrated in the central hub and then disperse outwards to the leaves. Our results offer the first experimental support for models of adaptive evolution in space based on evolutionary graph theory, may help understand the spread of invasive species or pathogens, and could be used in industrial settings to selectively enrich desired traits or biomolecules of interest.},
  archive      = {J_EVLETT},
  author       = {Chakraborty, Partha Pratim and Nemzer, Louis R and Kassen, Rees},
  doi          = {10.1093/evlett/qrad047},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {447-456},
  shortjournal = {Evol. Lett.},
  title        = {Experimental evidence that network topology can accelerate the spread of beneficial mutations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Asynchronous life cycles contribute to reproductive
isolation between two alpine butterflies. <em>EVLETT</em>,
<em>7</em>(6), 436–446. (<a
href="https://doi.org/10.1093/evlett/qrad046">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geographic isolation often leads to the emergence of distinct genetic lineages that are at least partially reproductively isolated. Zones of secondary contact between such lineages are natural experiments that allow investigation of how reproductive isolation evolves and co-existence is maintained. While temporal isolation through allochrony has been suggested to promote reproductive isolation in sympatry, its potential for isolation upon secondary contact is far less understood. Sampling two contact zones of a pair of mainly allopatric Alpine butterflies over several years and taking advantage of museum samples, we show that the contact zones have remained geographically stable over several decades. Furthermore, they seem to be maintained by the asynchronous life cycles of the two butterflies, with one reaching adulthood primarily in even and the other primarily in odd years. Genomic inferences document that allochrony is leaky and that gene flow from allopatric sites scales with the degree of geographic isolation. Overall, we show that allochrony has the potential to contribute to the maintenance of secondary contact zones of lineages that diverged in allopatry. Many species have persisted in warmer geographic refugia during the last glaciation period. Since then, they have recolonized formerly glaciated areas, often coming into contact with close relatives from other refugia. Coexistence between these lineages is possible if there are enough isolating barriers that prevent gene flow between them. However, closely related taxa often lack strong barriers and form contact zones instead, where they hybridize to some degree. The factors that maintain such contact zones over time differ among species and are often unknown. Here, we studied two contact zones between a pair of closely related Alpine butterfly lineages. Over the 4 years that we sampled, we found that one lineage occurred predominantly in even and the other one predominantly in odd years, suggesting that their biennial life cycle enables them to spatially coexist in contact zones, with limited interbreeding. Using museum specimens, we confirmed that temporal isolation has been at play for at least several decades, and based on observation record data, we showed that biannual life cycles broadly differed between the two lineages across Switzerland. However, temporal isolation is leaky and hybridization occurs, especially when the contact zone is more connected to other populations. Taken together, our study highlights how asynchronous biennial life cycles, which are common in Alpine butterfly species, have the potential to contribute as a barrier to gene flow and may promote coexistence.},
  archive      = {J_EVLETT},
  author       = {Bouaouina, Selim and Chittaro, Yannick and Willi, Yvonne and Lucek, Kay},
  doi          = {10.1093/evlett/qrad046},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {436-446},
  shortjournal = {Evol. Lett.},
  title        = {Asynchronous life cycles contribute to reproductive isolation between two alpine butterflies},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pollinator sharing and hybridization in a pair of dioecious
figs sheds light on the pathways to speciation. <em>EVLETT</em>,
<em>7</em>(6), 422–435. (<a
href="https://doi.org/10.1093/evlett/qrad045">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dynamics and processes underlying the codiversification of plant–pollinator interactions are of great interest to researchers of biodiversity and evolution. Cospeciation is generally considered a key process driving the diversity of figs and their pollinating wasps. Groups of closely related figs pollinated by separate wasps occur frequently and represent excellent opportunities to study ongoing diversification in this textbook mutualism. We study two closely related sympatric dioecious figs ( Ficus heterostyla and Ficus squamosa ) in Xishuangbanna, southwest China, and aim to document what is likely to be the final stages of speciation between these species using a combination of trait data and experimental manipulation. Volatile profiles at the receptive phase, crucial for attracting pollinators, were analyzed. In total, 37 and 29 volatile compounds were identified from receptive F. heterostyla and F. squamosa figs, respectively. Despite significant interspecific dissimilarity, 25 compounds were shared. Ovipositor lengths lie well within range required for access to heterospecific ovules, facilitating hybridization. Cross introduction of wasps into figs was conducted and hybrid seeds were generated for all donor/recipient combinations. F. heterostyla wasps produce adult offspring in F. squamosa figs. While F. squamosa wasps induce gall development in F. heterostyla figs and their offspring fail to mature in synchrony with their novel host. We record limited geographic barriers, minimal volatile dissimilarity, compatible morphology, complementary reproductive phenologies, and the production of hybrid seeds and wasp offspring. These findings suggest ongoing wasp specialization and reproductive isolation, potentially applicable to other related fig species. The coevolution of flowering plants and their animal pollinators is essential for fostering biodiversity. Exploring the dynamics and processes underlying plant–pollinator interactions is of evolutionary interest and critical for understanding biodiversity origins. Ficus , a diverse genus, is well-known for its species-specific relationships with pollinating wasps. Cospeciation plays an important role in the formation of fig and wasp diversity, as the reciprocal adaptations have led to the development of strong prezygotic barriers. Nevertheless, wasps exhibit much shorter generation times compared to figs, resulting in faster speciation rates. Often, fig complexes or closely related figs coevolve with distinct wasp species, representing the final step toward complete fig speciation and providing an excellent opportunity for studying ongoing mutualistic diversification. In such scenarios, heterospecific visitation of wasps may trigger interspecific pollination. Here, we examined reproductive isolation between two closely related figs, Ficus heterostyla and Ficus squamosa , through trait data and experimental manipulation. Floral volatiles emitted by receptive figs, which are crucial signals for attracting specific wasps, exhibited considerable overlap between these two fig species. The lengths of the wasp ovipositors were well within the range required for access to heterospecific ovules. Hybrid seeds were produced experimentally, with results showing that wasps of F. heterostyla reproduced in F. squamosa figs, while wasps of F. squamosa did not reproduce in F. heterostyla figs. Overall, heterospecific visitation decreased both fig and wasp fitness. Together with geographic barriers and complementary reproductive phenologies, these findings suggest that prezygotic isolation between F. heterostyla and F. squamosa may not yet be established.},
  archive      = {J_EVLETT},
  author       = {Huang, Jian-Feng and Fungjanthuek, Jenjira and Chen, Ming-Bo and Liu, Gui-Xiang and Dong, Yi-Yi and Peng, Yan-Qiong and Wang, Bo and Segar, Simon T},
  doi          = {10.1093/evlett/qrad045},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {422-435},
  shortjournal = {Evol. Lett.},
  title        = {Pollinator sharing and hybridization in a pair of dioecious figs sheds light on the pathways to speciation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Host learning selects for the coevolution of greater egg
mimicry and narrower antiparasitic egg-rejection thresholds.
<em>EVLETT</em>, <em>7</em>(6), 413–421. (<a
href="https://doi.org/10.1093/evlett/qrad041">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Egg rejection is an effective and widespread antiparasitic defense to eliminate foreign eggs from the nests of hosts of brood parasitic birds. Several lines of observational and critical experimental evidence support a role for learning by hosts in the recognition of parasitic versus own eggs; specifically, individual hosts that have had prior or current experience with brood parasitism are more likely to reject foreign eggs. Here we confirm experimentally the role of prior experience in altering subsequent egg-rejection decisions in the American robin Turdus migratorius , a free-living host species of an obligate brood parasite, the brown-headed cowbird Molothrus ater . We then model the coevolutionary trajectory of both the extent of mimicry of host eggs by parasitic eggs and the host’s egg rejection thresholds in response to an increasing role of learning in egg recognition. Critically, with more learning, we see the evolution of both narrower (more discriminating) rejection thresholds in hosts and greater egg mimicry in parasites. Increasing host clutch size (number of eggs/nest) and increasing parasite load (parasitism rate) also have narrowing effects on the egg-rejection threshold. Together, these results suggest that learning from prior experience with egg rejection may play an important role in the coevolution of egg-mimetic lineages of brood parasites and the refined egg rejection defenses of hosts. Obligate brood parasitic birds lay their eggs in the nests of other species. In response to such parasitism, which reduces the fitness of the foster parents, hosts have evolved refined foreign-egg rejection abilities, while parasites have evolved increasing mimicry of the host eggs. But how do hosts recognize and accept their own eggs? Here we hypothesized that hosts learn about their own and foreign eggs through experience with unusual eggs in an early clutch and then they become increasingly better at rejecting the parasite’s egg in subsequent nests and breeding attempts. We confirmed such a learning mechanism with a simple field experiment on the American robin, an egg rejecter host of the brown-headed cowbird. In turn, a complex mathematical model revealed that with more learning by the host we see the evolution of an increasingly fine-tuned egg recognition system by the would-be foster parents and a greater mimetic match of the host eggs by the parasites.},
  archive      = {J_EVLETT},
  author       = {Xu, Kuangyi and Servedio, Maria R and Winnicki, Sarah K and Moskat, Csaba and Hoover, Jeffrey P and Turner, Abbigail M and Hauber, Mark E},
  doi          = {10.1093/evlett/qrad041},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {413-421},
  shortjournal = {Evol. Lett.},
  title        = {Host learning selects for the coevolution of greater egg mimicry and narrower antiparasitic egg-rejection thresholds},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Structural genomic variation and migratory behavior in a
wild songbird. <em>EVLETT</em>, <em>7</em>(6), 401–412. (<a
href="https://doi.org/10.1093/evlett/qrad040">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural variants (SVs) are a major source of genetic variation; and descriptions in natural populations and connections with phenotypic traits are beginning to accumulate in the literature. We integrated advances in genomic sequencing and animal tracking to begin filling this knowledge gap in the Eurasian blackcap. Specifically, we (a) characterized the genome-wide distribution, frequency, and overall fitness effects of SVs using haplotype-resolved assemblies for 79 birds, and (b) used these SVs to study the genetics of seasonal migration. We detected &gt;15 K SVs. Many SVs overlapped repetitive regions and exhibited evidence of purifying selection suggesting they have overall deleterious effects on fitness. We used estimates of genomic differentiation to identify SVs exhibiting evidence of selection in blackcaps with different migratory strategies. Insertions and deletions dominated the SVs we identified and were associated with genes that are either directly (e.g., regulatory motifs that maintain circadian rhythms) or indirectly (e.g., through immune response) related to migration. We also broke migration down into individual traits (direction, distance, and timing) using existing tracking data and tested if genetic variation at the SVs we identified could account for phenotypic variation at these traits. This was only the case for 1 trait—direction—and 1 specific SV (a deletion on chromosome 27) accounted for much of this variation. Our results highlight the evolutionary importance of SVs in natural populations and provide insight into the genetic basis of seasonal migration. Structural variants (SVs) represent a major source of genetic variation that is important for both adaptation and speciation. However, detailed information on how they are distributed in the genome, their frequency, and effects on the fitness of individuals in natural populations (e.g., associations with adaptive traits) is lacking. These knowledge gaps derive in large part from difficulties associated with identifying SVs in the genome (e.g., because they are highly repetitive). We filled this knowledge gap here, using new sequencing technologies to assemble de novo genomes for 79 Eurasian blackcaps. These blackcaps had been tracked on migration in a previous study, allowing us to expand our work to the genetics of migration as well. Migration is a complex behavioral trait that has fascinated researchers and lay people alike for decades as well. The SVs we identified had overall negative effects on fitness but a subset were linked to variation in the migratory behavior of blackcaps. These SVs were linked to genes with functions that may be directly related to migration (e.g., that help maintain circadian rhythms) or are experiencing selection that is secondary to migration (e.g., that help with immune responses).},
  archive      = {J_EVLETT},
  author       = {Delmore, Kira E and Van Doren, Benjamin M and Ullrich, Kristian and Curk, Teja and van der Jeugd, Henk P and Liedvogel, Miriam},
  doi          = {10.1093/evlett/qrad040},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {401-412},
  shortjournal = {Evol. Lett.},
  title        = {Structural genomic variation and migratory behavior in a wild songbird},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Long-term evolution of antibiotic tolerance in pseudomonas
aeruginosa lung infections. <em>EVLETT</em>, <em>7</em>(6), 389–400. (<a
href="https://doi.org/10.1093/evlett/qrad034">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pathogenic bacteria respond to antibiotic pressure with the evolution of resistance but survival can also depend on their ability to tolerate antibiotic treatment, known as tolerance. While a variety of resistance mechanisms and underlying genetics are well characterized in vitro and in vivo, an understanding of the evolution of tolerance, and how it interacts with resistance in situ is lacking. We assayed for tolerance and resistance in isolates of Pseudomonas aeruginosa from chronic cystic fibrosis lung infections spanning up to 40 years of evolution, with 3 clinically relevant antibiotics: meropenem, ciprofloxacin, and tobramycin. We present evidence that tolerance is under positive selection in the lung and that it can act as an evolutionary stepping stone to resistance. However, by examining evolutionary patterns across multiple patients in different clone types, a key result is that the potential for an association between the evolution of resistance and tolerance is not inevitable, and difficult to predict. Antibiotic resistance in bacteria is a major challenge to our ability to treat infections. Understanding how bacteria become resistant is important, but experiments in the lab are not always representative of what actually happens within patients. Studying isolates collected over time from patients may allow us to infer retrospectively how pathogens evolve. We investigated characteristics of bacteria that make them difficult to treat in a unique collection of bacterial isolates spanning 40 years, from Danish individuals with the genetic disease cystic fibrosis. Cystic fibrosis causes the build-up of mucus in the lungs that make bacterial infections difficult to eradicate, despite intensive treatment with antibiotics. We show that the bacteria can evolve “tolerance” to antibiotics even when they are still genetically susceptible by switching their behavior temporarily to a dormant state. Our results suggest that this can allow the bacteria to subsequently evolve genetic resistance, where they are capable of actively growing despite the presence of drugs. In summary, the picture from “real life infections” is more complex than a straightforward expectation of antibiotic treatment leads to selection for resistance, leads to the persistence of infection. Once again, nature proves to be more complex and unpredictable than controlled lab experiments would lead us to expect. But despite the challenges, more studies like this are needed to understand the strategies bacterial cells adopt, and why some infections are hard to treat.},
  archive      = {J_EVLETT},
  author       = {Ghoul, Melanie and Andersen, Sandra B and Marvig, Rasmus L and Johansen, Helle K and Jelsbak, Lars and Molin, Søren and Perron, Gabriel and Griffin, Ashleigh S},
  doi          = {10.1093/evlett/qrad034},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {389-400},
  shortjournal = {Evol. Lett.},
  title        = {Long-term evolution of antibiotic tolerance in pseudomonas aeruginosa lung infections},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Parental care shapes the evolution of molecular genetic
variation. <em>EVLETT</em>, <em>7</em>(6), 379–388. (<a
href="https://doi.org/10.1093/evlett/qrad039">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cooperative social behaviors, such as parental care, have long been hypothesized to relax selection leading to the accumulation of genetic variation in populations. Although the idea has been discussed for decades, there has been relatively little experimental work to investigate how social behavior contributes to genetic variation in populations. Here, we investigate how parental care can shape molecular genetic variation in the subsocial insect, Nicrophorus vespilloides . Using whole-genome sequencing of populations that had evolved in the presence or absence of parental care for 30 generations, we show that parental care maintains levels of standing genetic variation. In contrast, under a harsh environment without care, strong directional selection caused a reduction in genetic variation. Furthermore, we show that adaptation to the loss of care is associated with genetic divergence between populations at loci related to stress, morphological development, and transcriptional regulation. These data reveal how social behavior is linked to the genetic processes that shape and maintain genetic diversity within populations, and provides rare empirical evidence for an old hypothesis. Social behaviors, such as parental care, have long been hypothesized to result in the accumulation of genetic variation in populations. Here, we investigate how parental care can shape molecular genetic variation in a species that performs biparental care, Nicrophorus vespilloides . Using genome sequencing of populations that had evolved in the presence or absence of parental care for 30 generations, we show that parental care maintains levels of standing genetic variation. In contrast, under a harsh environment without care, populations lost genetic variation. Furthermore, we show that adaptation to the loss of care is associated with genetic divergence between populations at genes related to stress, morphological development, and transcriptional regulation. These data reveal how social behavior is linked to the genetic processes that shape and maintain genetic diversity within populations.},
  archive      = {J_EVLETT},
  author       = {Mashoodh, Rahia and Trowsdale, Angela T and Manica, Andrea and Kilner, Rebecca M},
  doi          = {10.1093/evlett/qrad039},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {379-388},
  shortjournal = {Evol. Lett.},
  title        = {Parental care shapes the evolution of molecular genetic variation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High parasite virulence necessary for the maintenance of
host outcrossing via parasite-mediated selection. <em>EVLETT</em>,
<em>7</em>(6), 371–378. (<a
href="https://doi.org/10.1093/evlett/qrad036">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biparental sex is widespread in nature, yet costly relative to uniparental reproduction. It is generally unclear why self-fertilizing or asexual lineages do not readily invade outcrossing populations. The Red Queen hypothesis predicts that coevolving parasites can prevent self-fertilizing or asexual lineages from invading outcrossing host populations. However, only highly virulent parasites are predicted to maintain outcrossing, which may limit the general applicability of the Red Queen hypothesis. Here, we tested whether the ability of coevolving parasites to prevent invasion of self-fertilization within outcrossing host populations was dependent on parasite virulence. We introduced wild-type Caenorhabditis elegans hermaphrodites, capable of both self-fertilization and outcrossing, into C. elegans populations fixed for a mutant allele conferring obligate outcrossing. Replicate C. elegans populations were exposed for 24 host generations to one of four strains of Serratia marcescens parasites that varied in virulence, under three treatments: a heat-killed (control, noninfectious) parasite treatment, a fixed-genotype (nonevolving) parasite treatment, and a copassaged (potentially coevolving) parasite treatment. As predicted, self-fertilization invaded C. elegans host populations in the control and fixed-parasite treatments, regardless of parasite virulence. In the copassaged treatment, selfing invaded host populations coevolving with low- to mid-virulence strains, but remained rare in hosts coevolving with highly virulent bacterial strains. Therefore, we found that only highly virulent coevolving parasites can impede the invasion of selfing. The dominance of biparental sex in plant and animal populations is difficult for evolutionary biologists to explain given the high costs associated with sex. The Red Queen hypothesis provides a possible explanation, predicting that interactions with coevolving parasites produce fluctuating environments that can select for biparental sex despite the costs. However, an important limitation of the Red Queen hypothesis is that coevolving parasites might only impose sufficiently strong selection to maintain biparental sex when parasite virulence is very high. We explored this potential limitation by testing whether uniparental sex (i.e., self-fertilization) could spread into outcrossing populations of the nematode host Caenorhabditis elegans that were exposed to strains of the bacterial parasite Serratia marcescens that varied in virulence. We found that self-fertilization spread in all conditions except when the pathogen was highly virulent and coevolving. Our results support the Red Queen hypothesis by demonstrating that virulent coevolving parasites can select for biparental sex in hosts. They also highlight the limitations of the Red Queen hypothesis, showing that the conditions under which parasites can maintain biparental sex may be limited to cases of high virulence.},
  archive      = {J_EVLETT},
  author       = {Slowinski, Samuel P and Cho, JaeHoon and Penley, McKenna J and Alexander, Laura W and Greenberg, Arielle B and Namburar, Sathvik R and Morran, Levi T},
  doi          = {10.1093/evlett/qrad036},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {371-378},
  shortjournal = {Evol. Lett.},
  title        = {High parasite virulence necessary for the maintenance of host outcrossing via parasite-mediated selection},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The effect of seminal fluid gene expression on paternity.
<em>EVLETT</em>, <em>7</em>(5), 361–369. (<a
href="https://doi.org/10.1093/evlett/qrad033">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When females mate with more than one male, competition between rival ejaculates is expected to favor adaptations that promote fertilization success. There is now compelling evidence that sperm competition selects for increased production and allocation of sperm. However, sperm comes packaged in ejaculates that also contain protein-rich seminal fluids. Predicting how males should allocate individual seminal fluid proteins in response to sperm competition is hampered by our limited knowledge of their precise function. We use gene expression studies and interference RNA to ask how seminal fluid proteins in the ejaculate of a cricket, Teleogryllus oceanicus , affect a male’s paternity share when in competition for fertilizations. We find that the relative expression of one seminal fluid gene, gagein , positively affects the paternity share of competing males and that knockdown of this and two other seminal fluid protein genes renders males mating in the offensive role of sperm competition incapable of fathering living offspring. Despite having a negative effect on offspring viability these seminal fluid genes have been found to be up regulated in response to rival males, consistent with a role in promoting competitive fertilization success. Our data contribute to a growing body of evidence that, like sperm, seminal fluid gene expression is subject to post-mating sexual selection via sperm competition. Females often mate with more than one male so that the sperm of different males must compete to fertilize eggs. Sperm competition has been found to be a powerful force of evolution, favoring increases in sperm production, and sperm form and function. However, sperm comes packaged in ejaculates that can contain hundreds or thousands of different proteins that collectively make up the seminal fluid component of the ejaculate. Despite our growing knowledge of the protein composition of seminal fluid, we still have remarkably limited knowledge of the functional significance of different proteins. In this study, we used genomic manipulations and gene expression analyses to explore the function of seven seminal fluid proteins that are thought to be involved in sperm competition in male crickets. We prevented males from manufacturing a given seminal fluid protein by knocking down gene expression and then asked how many offspring these males fathered when in competition with an unmanipulated rival. We identified four seminal fluid proteins that were essential for males to obtain a share in the paternity of offspring produced by twice-mated females. When both males were unmanipulated, males with a greater production of at least one seminal fluid protein, Gagein, fathered a greater proportion of a female’s offspring. Gagein has been found in previous studies to affect the quality of sperm, and its production is upregulated by males in response to rivals encountered in their environment. Our findings contribute toward a growing understanding of the functional significance of seminal fluid proteins.},
  archive      = {J_EVLETT},
  author       = {Simmons, Leigh W and Lovegrove, Maxine},
  doi          = {10.1093/evlett/qrad033},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {361-369},
  shortjournal = {Evol. Lett.},
  title        = {The effect of seminal fluid gene expression on paternity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Maternal behavioral thermoregulation facilitated
evolutionary transitions from egg laying to live birth. <em>EVLETT</em>,
<em>7</em>(5), 351–360. (<a
href="https://doi.org/10.1093/evlett/qrad031">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Live birth is a key innovation that has evolved from egg-laying ancestors over 100 times in reptiles. However, egg-laying lizards and snakes can have preferred body temperatures that are lethal to developing embryos, which should select against prolonged egg retention. Here, we demonstrate that thermal mismatches between mothers and offspring are widespread across the squamate phylogeny. This mismatch is resolved by gravid females adjusting their body temperature towards the thermal optimum of their embryos. We find that the same response occurs in both live-bearing and egg-laying species, despite the latter only retaining embryos during the early stages of development. Importantly, phylogenetic reconstructions suggest this thermoregulatory behavior in gravid females evolved in egg-laying species prior to the evolution of live birth. Maternal thermoregulatory behavior, therefore, bypasses the constraints imposed by a slowly evolving thermal physiology and has likely been a key facilitator in the repeated transition to live birth. Giving birth to live young is a key innovation that has evolved numerous times across the tree of life. In reptiles alone, there are over 100 independent evolutionary origins of live birth. A paradox that remains to be resolved is how this transition has occurred, given that egg-laying female lizards and snakes frequently have preferred body temperatures that are lethal to developing embryos and should therefore prevent the evolution of egg retention. Using phylogenetic comparative analysis of 224 species of snakes and lizards, we show that a slowly evolving thermal physiology creates widespread and significant mismatches between mothers and embryos. However, evolutionary transitions to live birth occur despite these significant mismatches in the thermal requirements between mothers and offspring. Behavioral thermoregulation by gravid females circumvents a slowly evolving thermal physiology, allowing mothers to meet the thermal requirements of their offspring. Adjustments of body temperatures by gravid females evolved in egg-laying ancestors long before the emergence of live birth, potentially to protect early embryos against thermal stress prior to egg-laying. Our study, therefore, resolves the long-standing question of how live birth has evolved so frequently in a major lineage of egg-laying vertebrates, despite obvious physiological constraints.},
  archive      = {J_EVLETT},
  author       = {Pettersen, Amanda K and Feiner, Nathalie and Noble, Daniel W A and While, Geoffrey M and Uller, Tobias and Cornwallis, Charlie K},
  doi          = {10.1093/evlett/qrad031},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {351-360},
  shortjournal = {Evol. Lett.},
  title        = {Maternal behavioral thermoregulation facilitated evolutionary transitions from egg laying to live birth},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Equilibria and oscillations in cheat–cooperator dynamics.
<em>EVLETT</em>, <em>7</em>(5), 339–350. (<a
href="https://doi.org/10.1093/evlett/qrad032">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cooperative societies can be threatened by cheats, who invest less in cooperation and exploit the contributions of others. The impact of cheats depends on the extent to which they are maintained in the population. However, different empirical studies, across organisms ranging from RNA replicators to bacteria, have shown diverse cheat–cooperator dynamics. These vary from approaching a stable equilibrium to dynamic cyclical oscillations. The reason for this variation remains unclear. Here, we develop a theoretical model to identify the factors that determine whether dynamics should tend toward stable equilibria or cyclical oscillations. Our analyses show that (1) a combination of both periodic population bottlenecks and density-dependent selection on cheating is required to produce cyclical oscillations and (2) the extent of frequency-dependent selection for cheating can influence the amplitude of these oscillations but does not lead to oscillations alone. Furthermore, we show that stochastic group formation (demographic stochasticity) can generate different forms of oscillation, over a longer time scale, across growth cycles. Our results provide experimentally testable hypotheses for the processes underlying cheat–cooperator dynamics. Cooperators can be exploited by cheats. Cheats are individuals who gain the benefits of cooperation without contributing their fair share. Cheats can be found across the tree of life, including viruses, bacteria, slime moulds, and social insects. The outcomes of the interactions between cheats and cooperators can vary widely, from relatively stable proportions of cheats to cyclical oscillations, where the proportion of cheats goes up and down. We developed a theoretical model to explain the dynamics between cheats and cooperators. We discovered that the emergence of cyclical oscillations relies on the simultaneous occurrence of two key mechanisms: periodic population bottlenecks, characterized by recurrent reductions in population size, and density-dependent selection acting on cheating behavior, where the fitness of cheats increases with the abundance of cooperators. We then explored the impact of frequency-dependent selection, where the fitness of cheats increases with the proportion of cooperator within the population. We found that while frequency-dependent selection can influence the amplitude of oscillations, it is insufficient to generate oscillations in the proportion of cheats. Moreover, our examination of stochastic group formation, which is introducing random variations in group composition, revealed another form of oscillation and added an intriguing layer of complexity to cheat–cooperator dynamics. Our results provide hypotheses that could be tested in either experimental or natural populations.},
  archive      = {J_EVLETT},
  author       = {Liu, Ming and Wild, Geoff and West, Stuart A},
  doi          = {10.1093/evlett/qrad032},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {339-350},
  shortjournal = {Evol. Lett.},
  title        = {Equilibria and oscillations in cheat–cooperator dynamics},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pleistocene glaciations caused the latitudinal gradient of
within-species genetic diversity. <em>EVLETT</em>, <em>7</em>(5),
331–338. (<a href="https://doi.org/10.1093/evlett/qrad030">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intraspecific genetic diversity is a key aspect of biodiversity. Quaternary climatic change and glaciation influenced intraspecific genetic diversity by promoting range shifts and population size change. However, the extent to which glaciation affected genetic diversity on a global scale is not well established. Here we quantify nucleotide diversity, a common metric of intraspecific genetic diversity, in more than 38,000 plant and animal species using georeferenced DNA sequences from millions of samples. Results demonstrate that tropical species contain significantly more intraspecific genetic diversity than nontropical species. To explore potential evolutionary processes that may have contributed to this pattern, we calculated summary statistics that measure population demographic change and detected significant correlations between these statistics and latitude. We find that nontropical species are more likely to deviate from neutral expectations, indicating that they have historically experienced dramatic fluctuations in population size likely associated with Pleistocene glacial cycles. By analyzing the most comprehensive data set to date, our results imply that Quaternary climate perturbations may be more important as a process driving the latitudinal gradient in species richness than previously appreciated. It is well appreciated among biologists that species richness is higher near the equator than at temperate latitudes. Using data from 38,000 species, we demonstrate here that a similar gradient is present for intraspecific genetic diversity. Results indicate that nontropical species were more likely to have undergone significant changes in their historical population demography than were tropical species. Decades of phylogeographic work have suggested that Quaternary glaciation forced temperate species into isolated glacial refugia. Under this scenario, genetic diversity would be lost due to reduced population size, population bottlenecks, and genetic drift. Our analyses represent the most comprehensive test of the Quaternary glaciation hypothesis to date and suggest that the latitudinal gradient in intraspecific genetic diversity is at least partially a result of Pleistocene glaciation.},
  archive      = {J_EVLETT},
  author       = {Fonseca, Emanuel M and Pelletier, Tara A and Decker, Sydney K and Parsons, Danielle J and Carstens, Bryan C},
  doi          = {10.1093/evlett/qrad030},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {331-338},
  shortjournal = {Evol. Lett.},
  title        = {Pleistocene glaciations caused the latitudinal gradient of within-species genetic diversity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Signatures of kin selection in a natural population of the
bacteria bacillus subtilis. <em>EVLETT</em>, <em>7</em>(5), 315–330. (<a
href="https://doi.org/10.1093/evlett/qrad029">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Laboratory experiments have suggested that bacteria perform a range of cooperative behaviors, which are favored because they are directed toward relatives (kin selection). However, there is a lack of evidence for cooperation and kin selection in natural bacterial populations. Molecular population genetics offers a promising method to study natural populations because the theory predicts that kin selection will lead to relaxed selection, which will result in increased polymorphism and divergence at cooperative genes. Examining a natural population of Bacillus subtilis , we found consistent evidence that putatively cooperative traits have higher polymorphism and greater divergence than putatively private traits expressed at the same rate. In addition, we were able to eliminate alternative explanations for these patterns and found more deleterious mutations in genes controlling putatively cooperative traits. Overall, our results suggest that cooperation is favored by kin selection, with an average relatedness of r = .79 between interacting individuals. Bacteria produce and secrete a wide range of molecules. The benefits of these molecules can be shared by nearby bacterial cells. For example, secreted molecules that deactivate certain antibiotics provide protection to the whole group, including cells that do not produce the molecule themselves. Laboratory experiments have shown that this is a form of cooperation, which evolves because it benefits closely related cells that share the gene for cooperation (kin selection). However, it has been challenging to find evidence for cooperation and kin selection in natural bacterial populations. To address this, we have used a new way of studying cooperation in natural populations. Evolutionary theory tells us that cooperation leaves a distinct “footprint” in DNA sequence data. We examined the bacterial genomes of the soil-dwelling bacteria Bacillus subtilis taken from a natural population in Dundee, Scotland, and used this theory to look for evidence of cooperation in the DNA sequences. Our results suggest that these shared molecules are indeed cooperative traits that have been favored by kin selection in natural populations.},
  archive      = {J_EVLETT},
  author       = {Belcher, Laurence J and Dewar, Anna E and Hao, Chunhui and Ghoul, Melanie and West, Stuart A},
  doi          = {10.1093/evlett/qrad029},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {315-330},
  shortjournal = {Evol. Lett.},
  title        = {Signatures of kin selection in a natural population of the bacteria bacillus subtilis},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Vertical transmission does not always lead to benign
pathogen–host associations. <em>EVLETT</em>, <em>7</em>(5), 305–314. (<a
href="https://doi.org/10.1093/evlett/qrad028">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the capacity of pathogens to cause severe disease is of fundamental importance to human health and the preservation of biodiversity. Many of those pathogens are not only transmitted horizontally between unrelated hosts but also vertically between parents and their progeny. It is widely accepted that vertical transmission leads to the evolution of less virulent pathogens, but this idea stems from research that neglects the evolutionary response of hosts. Here, we use a game-theory model of coevolution between pathogen and host to show that vertical transmission does not always lead to more benign pathogens. We highlight scenarios in which vertical transmission results in pathogens exhibiting more virulence. However, we also predict that more benign outcomes are still possible (a) when generating new horizontal infections inflicts too much damage on hosts, (b) when clearing an infection is too costly for the host, and (c) when vertical transmission is promoted by a greater growth rate of the host population. Though our work offers a new perspective on the role of vertical transmission in pathogen–host systems, it does agree with previous experimental work.},
  archive      = {J_EVLETT},
  author       = {Shillcock, George and Úbeda, Francisco and Wild, Geoff},
  doi          = {10.1093/evlett/qrad028},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {305-314},
  shortjournal = {Evol. Lett.},
  title        = {Vertical transmission does not always lead to benign pathogen–host associations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ancient hybridization leads to the repeated evolution of red
flowers across a monkeyflower radiation. <em>EVLETT</em>, <em>7</em>(5),
293–304. (<a href="https://doi.org/10.1093/evlett/qrad024">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The reuse of old genetic variation can promote rapid diversification in evolutionary radiations, but in most cases, the historical events underlying this divergence are not known. For example, ancient hybridization can generate new combinations of alleles that sort into descendant lineages, potentially providing the raw material to initiate divergence. In the Mimulus aurantiacus species complex, there is evidence for widespread gene flow among members of this radiation. In addition, allelic variation in the MaMyb2 gene is responsible for differences in flower color between the closely related ecotypes of subspecies puniceus , contributing to reproductive isolation by pollinators. Previous work suggested that MaMyb2 was introgressed into the red-flowered ecotype of puniceus . However, additional taxa within the radiation have independently evolved red flowers from their yellow-flowered ancestors, raising the possibility that this introgression had a more ancient origin. In this study, we used repeated tests of admixture from whole-genome sequence data across this diverse radiation to demonstrate that there has been both ancient and recurrent hybridization in this group. However, most of the signal of this ancient introgression has been removed due to selection, suggesting that widespread barriers to gene flow are in place between taxa. Yet, a roughly 30 kb region that contains the MaMyb2 gene is currently shared only among the red-flowered taxa. Patterns of admixture, sequence divergence, and extended haplotype homozygosity across this region confirm a history of ancient hybridization, where functional variants have been preserved due to positive selection in red-flowered taxa but lost in their yellow-flowered counterparts. The results of this study reveal that selection against gene flow can reduce genomic signatures of ancient hybridization, but that historical introgression can provide essential genetic variation that facilitates the repeated evolution of phenotypic traits between lineages. Hybridization that occurs early during the evolution of a species group can provide the genetic variation necessary to stimulate rapid diversification. However, even though genetic variation can be transferred between species (i.e., introgression), in most cases, this variation will be deleterious and eliminated from populations by natural selection. In this study, we explored the genome-wide effects of hybridization on the evolution of red flowers in a radiation of monkeyflowers. Using genomic data, we demonstrate that there has been both recurrent and ancient hybridization during the divergence of this radiation, but that most of the signal of hybridization across the genome has been removed due to selection. However, even though widespread barriers to gene flow are likely in place, a roughly 30 kb region containing a major-effect gene causing red flowers is shared among the red-flowered taxa and has been preserved by recent natural selection. The results of this study reveal that even though introgressed genetic variation is often deleterious, ancient hybridization can also be a valuable source of functional genetic variation that can promote diversification during evolutionary radiations. Thus, hybridization can be a creative force in evolution, but low fitness in hybrids also can lead to the evolution of reproductive barriers, both of which reveal gene flow’s important role in promoting and maintaining diversity.},
  archive      = {J_EVLETT},
  author       = {Short, Aidan W and Streisfeld, Matthew A},
  doi          = {10.1093/evlett/qrad024},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {293-304},
  shortjournal = {Evol. Lett.},
  title        = {Ancient hybridization leads to the repeated evolution of red flowers across a monkeyflower radiation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Host body size, not host population size, predicts
genome-wide effective population size of parasites. <em>EVLETT</em>,
<em>7</em>(4), 285–292. (<a
href="https://doi.org/10.1093/evlett/qrad026">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effective population size (; ) of an organism is expected to be generally proportional to the total number of individuals in a population. In parasites, we might expect the effective population size to be proportional to host population size and host body size, because both are expected to increase the number of parasite individuals. However, among other factors, parasite populations are sometimes so extremely subdivided that high levels of inbreeding may distort these predicted relationships. Here, we used whole-genome sequence data from dove parasites (71 feather louse species of the genus ; ) and phylogenetic comparative methods to study the relationship between parasite effective population size and host population size and body size. We found that parasite effective population size is largely explained by host body size but not host population size. These results suggest the potential local population size (infrapopulation or deme size) is more predictive of the long-term effective population size of parasites than is the total number of potential parasite infrapopulations (i.e., host individuals).; Parasites, among Earth’s most diverse, threatened, and under-protected animals, play a central role in ecosystem function. The effective population size (; ) of an organism has a profound impact on evolutionary processes, such as the relative contributions of selection and genetic drift to genomic change. Population size is also one of the most important parameters in conservation biology. For free-living organisms, it is expected that ; is generally proportional to the total number of individuals in a population. However, for parasites, among other factors, populations are sometimes so extremely subdivided that high levels of inbreeding may distort these relationships. In this study, we used whole-genome sequence data from dove parasites and phylogenetic comparative methods to investigate the relationship between parasite effective population size (; ) and host population size and body size. Our results revealed a positive relationship between parasite effective population size (; ) and host body size, but not host population size. These results suggest that the size of parasite infrapopulations may be the most important factor when considering parasite effective population size, and have important implications for conservation.},
  archive      = {J_EVLETT},
  author       = {Doña, Jorge and Johnson, Kevin P},
  doi          = {10.1093/evlett/qrad026},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {285-292},
  shortjournal = {Evol. Lett.},
  title        = {Host body size, not host population size, predicts genome-wide effective population size of parasites},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Experimental evolution of metabolism under nutrient
restriction: Enhanced amino acid catabolism and a key role of
branched-chain amino acids. <em>EVLETT</em>, <em>7</em>(4), 273–284. (<a
href="https://doi.org/10.1093/evlett/qrad018">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Periodic food shortage is a common ecological stressor for animals, likely to drive physiological and metabolic adaptations to alleviate its consequences, particularly for juveniles that have no option but to continue to grow and develop despite undernutrition. Here we study changes in metabolism associated with adaptation to nutrient shortage, evolved by replicate ; populations maintained on a nutrient-poor larval diet for over 240 generations. In a factorial metabolomics experiment we showed that both phenotypic plasticity and genetically-based adaptation to the poor diet involved wide-ranging changes in metabolite abundance; however, the plastic response did not predict the evolutionary change. Compared to nonadapted larvae exposed to the poor diet for the first time, the adapted larvae showed lower levels of multiple free amino acids in their tissues—and yet they grew faster. By quantifying accumulation of the nitrogen stable isotope ; N we show that adaptation to the poor diet led to an increased use of amino acids for energy generation. This apparent “waste” of scarce amino acids likely results from the trade-off between acquisition of dietary amino acids and carbohydrates observed in these populations. The three branched-chain amino acids (leucine, isoleucine, and valine) showed a unique pattern of depletion in adapted larvae raised on the poor diet. A diet supplementation experiment demonstrated that these amino acids are limiting for growth on the poor diet, suggesting that their low levels resulted from their expeditious use for protein synthesis. These results demonstrate that selection driven by nutrient shortage not only promotes improved acquisition of limiting nutrients, but also has wide-ranging effects on how the nutrients are used. They also show that the abundance of free amino acids in the tissues does not, in general, reflect the nutritional condition and growth potential of an animal.; Juvenile animals are particularly vulnerable to nutrient deprivation—they usually do not have an option of arresting their development and just trying to survive until food becomes plentiful; rather, they must attempt to grow and develop with whatever nutrients that can scrape. While they could obviously improve their lot by getting better at finding the scarce food, could they also adapt their physiology and metabolism in a way that would alleviate consequences of undernutrition? To find out we let populations of the fruit fly ; adapt during 240 generations to conditions of chronic larval nutrient shortage, and then studied their metabolism. We found that these populations evolved changes in their amino acid metabolism: their larvae are better at extracting amino acids from nutritionally poor diet and are able to grow faster (which requires a higher rate of protein synthesis), while maintaining lower levels of most amino acids in their system. This suggests improved cellular “logistics,” with a higher turnover of raw materials associated with their lower stocks owing to their more efficient and immediate use. However, paradoxically, the malnutrition-adapted larvae also “waste” a substantial fraction of their amino acids by “burning” them for energy. They can afford this because of their improved extraction of scarce amino acid from the diet, but they may also be compelled to do this by a trade-off with absorption of dietary carbohydrates.},
  archive      = {J_EVLETT},
  author       = {Cavigliasso, Fanny and Savary, Loriane and Spangenberg, Jorge E and Gallart-Ayala, Hector and Ivanisevic, Julijana and Kawecki, Tadeusz J},
  doi          = {10.1093/evlett/qrad018},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {273-284},
  shortjournal = {Evol. Lett.},
  title        = {Experimental evolution of metabolism under nutrient restriction: Enhanced amino acid catabolism and a key role of branched-chain amino acids},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tolerance-conferring defensive symbionts and the evolution
of parasite virulence. <em>EVLETT</em>, <em>7</em>(4), 262–272. (<a
href="https://doi.org/10.1093/evlett/qrad015">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Defensive symbionts in the host microbiome can confer protection from infection or reduce the harms of being infected by a parasite. Defensive symbionts are therefore promising agents of biocontrol that could be used to control or ameliorate the impact of infectious diseases. Previous theory has shown how symbionts can evolve along the parasitism–mutualism continuum to confer greater or lesser protection to their hosts and in turn how hosts may coevolve with their symbionts to potentially form a mutualistic relationship. However, the consequences of introducing a defensive symbiont for parasite evolution and how the symbiont may coevolve with the parasite have received relatively little theoretical attention. Here, we investigate the ecological and evolutionary implications of introducing a tolerance-conferring defensive symbiont into an established host–parasite system. We show that while the defensive symbiont may initially have a positive impact on the host population, parasite and symbiont evolution tend to have a net negative effect on the host population in the long term. This is because the introduction of the defensive symbiont always selects for an increase in parasite virulence and may cause diversification into high- and low-virulence strains. Even if the symbiont experiences selection for greater host protection, this simply increases selection for virulence in the parasite, resulting in a net negative effect on the host population. Our results therefore suggest that tolerance-conferring defensive symbionts may be poor biocontrol agents for population-level infectious disease control.; Defensive symbionts—microbes that confer protection to a host against a harmful parasite—are found throughout the natural world and represent promising candidates for biological control to combat infectious diseases. Symbionts can protect their hosts through a variety of mechanisms that may prevent infection (resistance) or increase survival following infection (tolerance), yet our understanding of the ecological and evolutionary impact of defensive symbionts on parasites is limited. Moreover, few theoretical predictions exist for how defensive symbionts are likely to evolve in the presence of parasites and for the net effect on the host population. Using a mathematical model where defensive symbionts reduce parasite virulence (harm to the host), we investigate the impact of their introduction on the evolution of parasite virulence, how selection increases or decreases host protection, and whether such symbionts are beneficial for the host population. We find that this form of defensive symbiosis always selects for higher parasite virulence and that it can cause the parasite to diversify into high- and low-virulence strains which specialize on different host subpopulations. Crucially, we show that the introduction of a defensive symbiont will always lead to a long-term reduction in host population size even if they are beneficial in the short term. Together, our results show that defensive symbionts can have a strong impact on the evolution of virulence and that this form of host protection is not robust, indicating that tolerance-conferring symbionts are likely to be poor candidates for biological control of infectious diseases at the population level.},
  archive      = {J_EVLETT},
  author       = {Smith, Cameron A and Ashby, Ben},
  doi          = {10.1093/evlett/qrad015},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {262-272},
  shortjournal = {Evol. Lett.},
  title        = {Tolerance-conferring defensive symbionts and the evolution of parasite virulence},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Hypermutator emergence in experimental escherichia coli
populations is stress-type dependent. <em>EVLETT</em>, <em>7</em>(4),
252–261. (<a href="https://doi.org/10.1093/evlett/qrad019">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genotypes exhibiting an increased mutation rate, called hypermutators, can propagate in microbial populations because they can have an advantage due to the higher supply of beneficial mutations needed for adaptation. Although this is a frequently observed phenomenon in natural and laboratory populations, little is known about the influence of parameters such as the degree of maladaptation, stress intensity, and the genetic architecture for adaptation on the emergence of hypermutators. To address this knowledge gap, we measured the emergence of hypermutators over ~1,000 generations in experimental ; populations exposed to different levels of osmotic or antibiotic stress. Our stress types were chosen based on the assumption that the genetic architecture for adaptation differs between them. Indeed, we show that the size of the genetic basis for adaptation is larger for osmotic stress compared to antibiotic stress. During our experiment, we observed an increased emergence of hypermutators in populations exposed to osmotic stress but not in those exposed to antibiotic stress, indicating that hypermutator emergence rates are stress type dependent. These results support our hypothesis that hypermutator emergence is linked to the size of the genetic basis for adaptation. In addition, we identified other parameters that covaried with stress type (stress level and IS transposition rates) that might have contributed to an increased hypermutator provision and selection. Our results provide a first comparison of hypermutator emergence rates under varying stress conditions and point towards complex interactions of multiple stress-related factors on the evolution of mutation rates.; Mutation is a double-edged sword. On the one hand, a mutation can have a detrimental effect on an organism by affecting its proper functioning. On the other hand, a mutation can be beneficial when it increases adaptation to an environment. An organism is therefore expected to benefit from having a low mutation rate in an environment to which it is already well adapted. However, when an organism reproduces in an environment to which it is not well adapted, a higher mutation rate can be beneficial as it increases the chances of producing offspring that have an adaptive mutation. Hypermutators are bacteria with a genetically determined high mutation rate. These hypermutators have often been observed in populations that are adapting to stressful environments, but it is currently not well understood which specific conditions lead to their evolution. In this paper, we followed the evolution of hypermutators in ; populations grown in presence of different types and levels of stress. Our results point towards the importance of the number of genes involved in adaptation to a particular stress: the higher the number of genes that contribute to adaptation to a stress, the higher the chances that hypermutators evolve under those stress conditions. But the story is not that simple: we also discovered that certain types of stress cause some mobile genetic elements to jump around more in the genome, and this might also contribute to the evolution of hypermutators.},
  archive      = {J_EVLETT},
  author       = {Callens, Martijn and Rose, Caroline J and Finnegan, Michael and Gatchitch, François and Simon, Léna and Hamet, Jeanne and Pradier, Léa and Dubois, Marie-Pierre and Bedhomme, Stéphanie},
  doi          = {10.1093/evlett/qrad019},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {252-261},
  shortjournal = {Evol. Lett.},
  title        = {Hypermutator emergence in experimental escherichia coli populations is stress-type dependent},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Coprophagy rapidly matures juvenile gut microbiota in a
precocial bird. <em>EVLETT</em>, <em>7</em>(4), 240–251. (<a
href="https://doi.org/10.1093/evlett/qrad021">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coprophagy is a behavior where animals consume feces, and has been observed across a wide range of species, including birds and mammals. The phenomenon is particularly prevalent in juveniles, but the reasons for this remain unclear. One hypothesis is that coprophagy enables offspring to acquire beneficial gut microbes that aid development. However, despite the potential importance of this behavior, studies investigating the effects in juveniles are rare. Here we experimentally test this idea by examining how ingestion of adult feces by ostrich chicks affects their gut microbiota development, growth, feeding behavior, pathogen abundance, and mortality. We conducted extensive longitudinal experiments for 8 weeks, repeated over 2 years. It involved 240 chicks, of which 128 were provided daily access to fresh fecal material from adults and 112 were simultaneously given a control treatment. Repeated measures, behavioral observations, and DNA metabarcoding of the microbial gut community, both prior to and over the course of the experiment, allowed us to evaluate multiple aspects of the behavior. The results show that coprophagy causes (a) marked shifts to the juvenile gut microbiota, including a major increase in diversity and rapid maturation of the microbial composition, (b) higher growth rates (fecal-supplemented chicks became 9.4\% heavier at 8 weeks old), (c) changes to overall feeding behavior but no differences in feed intake, (d) lower abundance of a common gut pathogen (; ), and (e) lower mortality associated with gut disease. Together, our results suggest that the behavior of coprophagy in juveniles is highly beneficial and may have evolved to accelerate the development of gut microbiota.; The behavior of fecal feeding (coprophagy) is widespread among animals, yet our understanding of its causes and consequences is extremely limited. Here we experimentally tested the effects of coprophagy on the development of ostrich chicks by providing treatment groups with adult feces collected daily. We examined how coprophagy influenced the gut microbiome, juvenile growth, feeding behavior, pathogen abundance, and survival over the first 8 weeks of development. Fecal-supplemented individuals readily engaged in coprophagia, which resulted in a rapid maturation of their gut microbiome. Compared to control groups, which had no access to adult feces, treatment groups experienced a drastic increase in microbial diversity and their microbial composition rapidly shifted towards that of adults. Coprophagy also resulted in a higher juvenile growth rate, without increasing feed intake, and reduced mortality associated with gut bacterial infection. Our results suggest that the behavior of coprophagy in juveniles may have evolved to accelerate the development of gut microbiota. These findings have broad implications for animal husbandry and captive breeding programs of wildlife, and contribute to an improved general understanding of how host-associated microbes influence the evolution of animal behavior.},
  archive      = {J_EVLETT},
  author       = {Videvall, Elin and Bensch, Hanna M and Engelbrecht, Anel and Cloete, Schalk and Cornwallis, Charlie K},
  doi          = {10.1093/evlett/qrad021},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {240-251},
  shortjournal = {Evol. Lett.},
  title        = {Coprophagy rapidly matures juvenile gut microbiota in a precocial bird},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolutionary barriers to horizontal gene transfer in
macrophage-associated salmonella. <em>EVLETT</em>, <em>7</em>(4),
227–239. (<a href="https://doi.org/10.1093/evlett/qrad020">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Horizontal gene transfer (HGT) is a powerful evolutionary force facilitating bacterial adaptation and emergence of novel phenotypes. Several factors, including environmental ones, are predicted to restrict HGT, but we lack systematic and experimental data supporting these predictions. Here, we address this gap by measuring the relative fitness of 44 genes horizontally transferred from ; to ; in infection-relevant environments. We estimated the distribution of fitness effects in each environment and identified that dosage-dependent effects across different environments are a significant barrier to HGT. The majority of genes were found to be deleterious. We also found longer genes had stronger negative fitness consequences than shorter ones, showing that gene length was negatively associated with HGT. Furthermore, fitness effects of transferred genes were found to be environmentally dependent. In summary, a substantial fraction of transferred genes had a significant fitness cost on the recipient, with both gene characteristics and the environment acting as evolutionary barriers to HGT.; Bacteria can inherit genes from either the same or different species in the population, in addition to the vertical transfer from parent to offspring. This process of horizontal gene transfer (HGT) can be responsible for the origin of evolutionary novel phenotypes, such as antibiotic resistance, toxins, and virulence factors. While much of our understanding of HGT is from bioinformatic analyses, there is a dearth of systematic experimental studies. To fill this gap in our understanding, we tested this experimentally by mimicking a HGT event by transferring genes from ; to ; . We find that an imbalance in the protein dosage and gene length of transferred genes restricts the success of gene transfer events. Importantly, we find that the environment plays a defining role in the success of transfer events. In summary, our study demonstrates that to gain an in-depth understanding of the gene transfer process, it is important that we consider the influence of both intrinsic gene properties and the environmental conditions of transfer.},
  archive      = {J_EVLETT},
  author       = {Bhatia, Rama P and Kirit, Hande Acar and Lewis, Cecil M and Sankaranarayanan, Krithivasan and Bollback, Jonathan P},
  doi          = {10.1093/evlett/qrad020},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {227-239},
  shortjournal = {Evol. Lett.},
  title        = {Evolutionary barriers to horizontal gene transfer in macrophage-associated salmonella},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Experimental estimates of germline mutation rate in
eukaryotes: A phylogenetic meta-analysis. <em>EVLETT</em>,
<em>7</em>(4), 216–226. (<a
href="https://doi.org/10.1093/evlett/qrad027">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutation is the ultimate source of all genetic variation, and over the last 10 years the ready availability of whole-genome sequencing has permitted direct estimation of mutation rate for many non-model species across the tree of life. In this meta-analysis, we make a comprehensive search of the literature for mutation rate estimates in eukaryotes, identifying 140 mutation accumulation (MA) and parent–offspring (PO) sequencing studies covering 134 species. Based on these data, we revisit differences in the single-nucleotide mutation (SNM) rate between different phylogenetic lineages and update the known relationships between mutation rate and generation time, genome size, and nucleotide diversity—while accounting for phylogenetic nonindependence. We do not find a significant difference between MA and PO in estimated mutation rates, but we confirm that mammal and plant lineages have higher mutation rates than arthropods and that unicellular eukaryotes have the lowest mutation rates. We find that mutation rates are higher in species with longer generation times and larger genome sizes, even when accounting for phylogenetic relationships. Moreover, although nucleotide diversity is positively correlated with mutation rate, the gradient of the relationship is significantly less than one (on a logarithmic scale), consistent with higher mutation rates in populations with smaller effective size. For the 29 species for which data are available, we find that indel mutation rates are positively correlated with nucleotide mutation rates and that short deletions are generally more common than short insertions. Nevertheless, despite recent progress, no estimates of either SNM or indel mutation rates are available for the majority of deeply branching eukaryotic lineages—or even for most animal phyla. Even among charismatic megafauna, experimental mutation rate estimates remain unknown for amphibia and scarce for reptiles and fish.; Over the past decade, the sequencing revolution has led to an ever-increasing number of mutation rate estimates from mutation accumulation or parent–offspring sequencing studies in eukaryotes. However, studies rarely quantify the extent to which mutation rates vary among these species. Also, despite there being strong predictions as to how mutation rate should vary with (e.g.) generation time, there have been few recent or wide-ranging analyses of such predictors that account for the inherent similarity between closely related species. Of particular note, there has been surprisingly little effort to robustly test the “drift barrier” hypothesis that mutation rates should decrease with increasing effective population size. In this study, we used a comprehensive literature search to identify all the available experimental estimates of mutation rate in eukaryotes and subject them to phylogenetic mixed-model analyses. We find that per-nucleotide per-generation mutation rates differ by orders of magnitude among species: plants and mammals tend to have higher mutation rates than arthropods, and unicellular organisms have the lowest mutation rates. Our analysis also shows that mutation rates increase significantly with increasing generation time and genome size, and nucleotide diversity increases with mutation rate with a gradient less than one—as predicted by the drift-barrier hypothesis.},
  archive      = {J_EVLETT},
  author       = {Wang, Yiguan and Obbard, Darren J},
  doi          = {10.1093/evlett/qrad027},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {216-226},
  shortjournal = {Evol. Lett.},
  title        = {Experimental estimates of germline mutation rate in eukaryotes: A phylogenetic meta-analysis},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Damaraland mole-rats do not rely on helpers for reproduction
or survival. <em>EVLETT</em>, <em>7</em>(4), 203–215. (<a
href="https://doi.org/10.1093/evlett/qrad023">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In eusocial invertebrates and obligate cooperative breeders, successful reproduction is dependent on assistance from non-breeding group members. Although naked (; ) and Damaraland mole-rats (; ) are often described as eusocial and their groups are suggested to resemble those of eusocial insects more closely than groups of any other vertebrate, the extent to which breeding individuals benefit from the assistance of non-breeding group members is unclear. Here we show that, in wild Damaraland mole-rats, prospective female breeders usually disperse and settle alone in new burrow systems where they show high survival rates and remain in good body condition—often for several years—before being joined by males. In contrast to many obligate cooperative vertebrates, pairs reproduced successfully without non-breeding helpers, and the breeding success of experimentally formed pairs was similar to that of larger, established groups. Though larger breeding groups recruited slightly more pups than smaller groups, adult survival was independent of group size and group size had mixed effects on the growth of non-breeders. Our results suggest that Damaraland mole-rats do not need groups to survive and that cooperative breeding in the species is not obligate as pairs can—and frequently do—reproduce without the assistance of helpers. While re-emphasizing the importance of ecological constraints on dispersal in social mole-rats, the mixed effects of group size in our study suggest that indirect benefits accrued through cooperative behavior may have played a less prominent role in the evolution of mole-rat group-living than previously thought.; Social mole-rats are subterranean rodents that live in family groups where a single breeding female is responsible for the production of all pups. It has frequently been suggested that her non-breeding offspring act as helpers and increase the survival of all group members through cooperative foraging, which is thought to increase the rate at which tubers and other geophytes—the principal food of mole-rats—are found. Such helper effects are expected to generate positive associations between group size and reproduction, growth, and survival, but have rarely been looked for in wild populations. After monitoring a population of Damaraland mole-rats in the Southern Kalahari over 7 years, we found that the effect of non-breeders on the reproductive output of breeding females was modest: large groups recruited only slightly more offspring than smaller groups, and the experimental creation of breeding pairs showed that newly formed groups can start breeding immediately and reproduce at rates comparable to established groups. Effects of group size on individual growth rates and on individual survival were also limited, with solitary females in particular—females who have dispersed from their natal group and settled alone—showing high survival rates that approached that of breeding females. Taken together, our results suggests that the extent to which breeding females rely on non-breeders as helpers in mole-rat societies may be less pronounced than previously thought. Helper effects appear relatively weak and the principal reason that offspring delay dispersal is likely because of the strong constraints on dispersal in this species.},
  archive      = {J_EVLETT},
  author       = {Thorley, Jack and Bensch, Hanna M and Finn, Kyle and Clutton-Brock, Tim and Zöttl, Markus},
  doi          = {10.1093/evlett/qrad023},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {203-215},
  shortjournal = {Evol. Lett.},
  title        = {Damaraland mole-rats do not rely on helpers for reproduction or survival},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The coevolutionary dynamics of cryptic female choice.
<em>EVLETT</em>, <em>7</em>(4), 191–202. (<a
href="https://doi.org/10.1093/evlett/qrad025">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In contrast to sexual selection on traits that affect interactions between the sexes before mating, little theoretical research has focused on the coevolution of postmating traits via cryptic female choice (when females bias fertilization toward specific males). We used simulation models to ask (a) whether and, if so, how nondirectional cryptic female choice (female-by-male interactions in fertilization success) causes deviations from models that focus exclusively on male-mediated postmating processes, and (b) how the risk of sperm competition, the strength of cryptic female choice, and tradeoffs between sperm number and sperm traits interact to influence the coevolutionary dynamics between cryptic female choice and sperm traits. We found that incorporating cryptic female choice can result in males investing much less in their ejaculates than predicted by models with sperm competition only. We also found that cryptic female choice resulted in the evolution of genetic correlations between cryptic female choice and sperm traits, even when the strength of cryptic female choice was weak, and the risk of sperm competition was low. This suggests that cryptic female choice may be important even in systems with low multiple mating. These genetic correlations increased with the risk of sperm competition and as the strength of cryptic female choice increased. When the strength of cryptic female choice and risk of sperm competition was high, extreme codivergence of sperm traits and cryptic female choice preference occurred even when the sperm trait traded off with sperm number. We also found that male traits lagged behind the evolution of female traits; this lag decreased with increasing strength of cryptic female choice and risk of sperm competition. Overall, our results suggest that cryptic female choice deserves more attention theoretically and may be driving trait evolution in ways just beginning to be explored.; Sexual selection—selection on traits that aid in mating and/or fertilization success—can drive the evolution of extreme ornamentation and elaborate behaviors often in confusing and counterintuitive ways that oppose natural selection. One such example is the elaborate display of the male peacock. While typically thought of occurring before mating, sexual selection also occurs after mating in the form of sperm competition and cryptic female choice—a process in which females bias fertilization toward specific males. Postmating sexual selection can also drive elaborate traits such as in some fruit flies where sperm are often longer than male bodies. Despite decades of research demonstrating cryptic female choice, there exists almost no models to understand the coevolutionary dynamics resulting from this process. We developed models that ask (a) how male ejaculate allocation changes in response to cryptic female choice, and (b) when cryptic female choice will result in extreme trait elaboration. We found that when selection from cryptic female choice is strong, investment in ejaculate production is much lower than predicted by models with sperm competition only. We also found that cryptic female choice results in the correlated evolution of male and female traits even when selection is weak and multiple mating is infrequent, with the potential for extreme trait elaboration. Overall, our results suggest that cryptic female choice deserves more attention theoretically and may drive the evolution of male traits in ways that we are only just beginning to explore.},
  archive      = {J_EVLETT},
  author       = {Kustra, Matthew C and Alonzo, Suzanne H},
  doi          = {10.1093/evlett/qrad025},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {191-202},
  shortjournal = {Evol. Lett.},
  title        = {The coevolutionary dynamics of cryptic female choice},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ornaments indicate parasite load only if they are dynamic or
parasites are contagious. <em>EVLETT</em>, <em>7</em>(3), 176–190. (<a
href="https://doi.org/10.1093/evlett/qrad017">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Choosing to mate with an infected partner has several potential fitness costs, including disease transmission and infection-induced reductions in fecundity and parental care. By instead choosing a mate with no, or few, parasites, animals avoid these costs and may also obtain resistance genes for offspring. Within a population, then, the quality of sexually selected ornaments on which mate choice is based should correlate negatively with the number of parasites with which a host is infected (“parasite load”). However, the hundreds of tests of this prediction yield positive, negative, or no correlation between parasite load and ornament quality. Here, we use phylogenetically controlled meta-analysis of 424 correlations from 142 studies on a wide range of host and parasite taxa to evaluate explanations for this ambiguity. We found that ornament quality is weakly negatively correlated with parasite load overall, but the relationship is more strongly negative among ornaments that can dynamically change in quality, such as behavioral displays and skin pigmentation, and thus can accurately reflect current parasite load. The relationship was also more strongly negative among parasites that can transmit during sex. Thus, the direct benefit of avoiding parasite transmission may be a key driver of parasite-mediated sexual selection. No other moderators, including methodological details and whether males exhibit parental care, explained the substantial heterogeneity in our data set. We hope to stimulate research that more inclusively considers the many and varied ways in which parasites, sexual selection, and epidemiology intersect.; Researchers have long been fascinated by the idea that parasites might affect the mating decisions, and thus the evolutionary trajectory, of their hosts. The role of parasites in the evolution of exaggerated sexually selected ornaments such as brightly colored skin or plumage has received particular attention: Perhaps choosers can use ornaments to selectively mate with individuals with no, or very few, parasites? We extracted data from the published literature testing this idea to evaluate (a) the generality of the pattern and (b) whether any factors affect the extent to which ornaments indicate parasite infection. Our data set, over six times larger than previous such data sets, encompasses hosts from across the animal kingdom (birds, fish, mammals, reptiles, amphibians, insects, and arachnids), infected with a diversity of parasites (including bacteria, viruses, fungi, protists, nematodes, helminths, and arthropods). We found that overall, individual hosts with higher quality ornaments are infected with fewer parasites, but only when ornament quality can change rapidly in response to host condition (like behavior and unlike antlers), or parasites can potentially transmit between mating partners. Together, these results indicate that choosers may be prioritizing the avoidance of potentially contagious partners in their mate choice decisions, and not partner genetic quality. Combined with the fact that there was a large amount of unexplained variation in our data set, we hope this finding stimulates a broader consideration of how mate choice decisions and epidemic dynamics interact.},
  archive      = {J_EVLETT},
  author       = {Dougherty, Liam R and Rovenolt, Faith and Luyet, Alexia and Jokela, Jukka and Stephenson, Jessica F},
  doi          = {10.1093/evlett/qrad017},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {176-190},
  shortjournal = {Evol. Lett.},
  title        = {Ornaments indicate parasite load only if they are dynamic or parasites are contagious},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The evolutionary demise of a social interaction:
Experimentally induced loss of traits involved in the supply and demand
of care. <em>EVLETT</em>, <em>7</em>(3), 168–175. (<a
href="https://doi.org/10.1093/evlett/qrad016">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotypic plasticity enables animals to adjust their behavior flexibly to their social environment—sometimes through the expression of adaptive traits that have not been exhibited for several generations. We investigated how long social adaptations can usefully persist when they are not routinely expressed, by using experimental evolution to document the loss of social traits associated with the supply and demand of parental care. We allowed populations of burying beetles ; to evolve in two different social environments for 48 generations in the lab. In “Full Care” populations, traits associated with the supply and demand of parental care were expressed at every generation, whereas in “No Care” populations we prevented expression of these traits experimentally. We then revived trait expression in the No Care populations at generations 24, 43, and 48 by allowing parents to supply post-hatching care and compared these social traits with those expressed by the Full Care populations. We found that offspring demands for care and male provision of care in the No Care populations were lost sooner than female provision of care. We suggest that this reflects differences in the strength of selection for the expression of alternative traits in offspring, males and females, which can enhance fitness when post-hatching care is disrupted.; Social interactions between animals are suggested to be increasingly vulnerable to breakdown in our changing world. Our experiments assess in real time the durability of social behaviors that are no longer routinely expressed. Our results also have implications for captive breeding programs used in conservation, where compensatory husbandry techniques prevent trait expression and so could inadvertently induce rapid, irreversible trait loss. We investigated how long it took populations to lose the ability to express appropriate social behavior when they had been prevented from doing so for many generations. We did this by evolving replicate populations of burying beetles ; in the laboratory for 48 generations. The burying beetle is a common insect that is well-known for caring for its larvae, although larvae can survive in the lab without any care at all. In two populations (“Full Care”), we allowed parents and offspring to interact during the supply of post-hatching care, as usual. In two other populations (“No Care”), parents were removed before offspring hatched and so could not interact socially with their young. Over the course of 48 generations of experimental evolution, we periodically revived social interactions between parents and offspring in the No Care populations. We assessed the extent to which larval begging behaviors, and parental care behaviors, were lost by comparing their expression with those in the Full Care populations. We found that larval begging behavior and male contributions to care were rapidly lost in No Care populations. In contrast, maternal care was relatively unchanged. We suggest that these differences could be due to differences in the strength of selection on different family members for alternative traits to enhance fitness.},
  archive      = {J_EVLETT},
  author       = {Bladon, Eleanor K and Pascoal, Sonia and Bird, Nancy and Mashoodh, Rahia and Kilner, Rebecca M},
  doi          = {10.1093/evlett/qrad016},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {168-175},
  shortjournal = {Evol. Lett.},
  title        = {The evolutionary demise of a social interaction: Experimentally induced loss of traits involved in the supply and demand of care},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The role of recognition error in the stability of
green-beard genes. <em>EVLETT</em>, <em>7</em>(3), 157–167. (<a
href="https://doi.org/10.1093/evlett/qrad012">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The empirical examples of the green-beard genes, once a conundrum of evolutionary biology, are accumulating, while theoretical analyses of this topic are occasional compared to those concerning (narrow-sense) kin selection. In particular, the recognition error of the green-beard effect that the cooperator fails to accurately recognize the other cooperators or defectors is readily found in numerous green-beard genes. To our knowledge, however, no model up to date has taken that effect into account. In this article, we investigated the effect of recognition error on the fitness of the green-beard gene. By employing theories of evolutionary games, our mathematical model predicts that the fitness of the green-beard gene is frequency dependent (frequency of the green-beard gene), which was corroborated by experiments performed with yeast ; . The experiment also shows that the cells with the green-beard gene (; ) are sturdier under severe stress. We conclude that the low recognition error among the cooperators, the higher reward of cooperation, and the higher cost of defection confer an advantage to the green-beard gene under certain conditions, confirmed by numerical simulation as well. Interestingly, we expect that the recognition error to the defectors may promote the cooperator fitness if the cooperator frequency is low and mutual defection is detrimental. Our ternary approach of mathematical analysis, experiments, and simulation lays the groundwork of the standard model for the green-beard gene that can be generalized to other species.; Cooperators should exclusively cooperate with cooperators while abandoning the defector. If not, defecting free riders will overrun, and cooperators will be extinct. Therefore, successful cooperators should accurately recognize the conspecific cooperators, which defines the characteristics of the “green-beard effect.” Adhesion of the yeasts under external stress is a well-known example of the green-beard effect in that cells that secrete the adhesive materials aggregate. In this case, aggregation is cooperation. However, some cells that do not secrete adhesive materials are included in the aggregate (free riders), which can be considered as a recognition error. As no model until now analyzed this factor, we estimated how the recognition error influences the fitness of the green-beard gene. We theoretically predicted becoming a cooperator would confer much benefit if cooperator frequency is high and the mutual defection is punishing. In the case of yeasts, as defecting cells that do not secrete adhesive materials are less likely to be included in the aggregate, they would die if the external stress is severe. We confirmed these predictions through simulation and actual experiments. Interestingly, we speculate that cooperation with defectors might be advantageous to the cooperators if the cooperator frequency is low. Suppose that there are not many adhesive yeasts but a large aggregate is necessary to endure the stress. In this condition, establishing an aggregate with defectors rather than excluding them would be beneficial to the cooperators.},
  archive      = {J_EVLETT},
  author       = {Choi, Jibeom and Lee, Seoeun and Kim, Hyun and Park, Junpyo},
  doi          = {10.1093/evlett/qrad012},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {157-167},
  shortjournal = {Evol. Lett.},
  title        = {The role of recognition error in the stability of green-beard genes},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sex-biased gene expression at single-cell resolution: Cause
and consequence of sexual dimorphism. <em>EVLETT</em>, <em>7</em>(3),
148–156. (<a href="https://doi.org/10.1093/evlett/qrad013">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression differences between males and females are thought to be key for the evolution of sexual dimorphism, and sex-biased genes are often used to study the molecular footprint of sex-specific selection. However, gene expression is often measured from complex aggregations of diverse cell types, making it difficult to distinguish between sex differences in expression that are due to regulatory rewiring within similar cell types and those that are simply a consequence of developmental differences in cell-type abundance. To determine the role of regulatory versus developmental differences underlying sex-biased gene expression, we use single-cell transcriptomic data from multiple somatic and reproductive tissues of male and female guppies, a species that exhibits extensive phenotypic sexual dimorphism. Our analysis of gene expression at single-cell resolution demonstrates that nonisometric scaling between the cell populations within each tissue and heterogeneity in cell-type abundance between the sexes can influence inferred patterns of sex-biased gene expression by increasing both the false-positive and false-negative rates. Moreover, we show that, at the bulk level, the subset of sex-biased genes that are the product of sex differences in cell-type abundance can significantly confound patterns of coding-sequence evolution. Taken together, our results offer a unique insight into the effects of allometry and cellular heterogeneity on perceived patterns of sex-biased gene expression and highlight the power of single-cell RNA-sequencing in distinguishing between sex-biased genes that are the result of regulatory change and those that stem from sex differences in cell-type abundance, and hence are a consequence rather than a cause of sexual dimorphism.; Males and females of many species display remarkable differences in morphology, physiology, behavior, and other traits, despite sharing an almost identical genome. Differences in gene expression between the sexes, referred to as sex-biased gene expression, are therefore thought to be key for the development of sexually dimorphic traits. However, gene expression is often measured from whole organs or even whole organisms, which represent aggregations of many cell types that can significantly differ in abundance between males and females. Such differences in the relative size or proportion of constituent cell populations may lead to perceived sex-biased gene expression, even if expression within individual cell types is unchanged between the two sexes. As such, it remains unclear to what extent is sex-biased gene expression a consequence of sex differences in cell-type abundance, rather than a cause of sexual dimorphism. To address this, we leverage recent advances in single-cell transcriptomics to compare male and female expression levels across equivalent cell populations from multiple tissues of the guppy, a species that shows extensive sexual dimorphism. We find significant differences in cellular composition between males and females across all sampled tissues and demonstrate that these sex differences in cell-type abundance can alter estimates of sex-biased gene expression. For many genes, expression appears to differ between the sexes at the aggregated whole tissue level but not at the individual cell population level. These sex-biased genes that are a consequence of sex differences in tissue composition can further influence estimates of molecular evolution and studies of selection.},
  archive      = {J_EVLETT},
  author       = {Darolti, Iulia and Mank, Judith E},
  doi          = {10.1093/evlett/qrad013},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {148-156},
  shortjournal = {Evol. Lett.},
  title        = {Sex-biased gene expression at single-cell resolution: Cause and consequence of sexual dimorphism},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Divergent evolution of genetic sex determination mechanisms
along environmental gradients. <em>EVLETT</em>, <em>7</em>(3), 132–147.
(<a href="https://doi.org/10.1093/evlett/qrad011">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sex determination (SD) is a crucial developmental process, but its molecular underpinnings are very diverse, both between and within species. SD mechanisms have traditionally been categorized as either genetic (GSD) or environmental (ESD), depending on the type of cue that triggers sexual differentiation. However, mixed systems, with both genetic and environmental components, are more prevalent than previously thought. Here, we show theoretically that environmental effects on expression levels of genes within SD regulatory mechanisms can easily trigger within-species evolutionary divergence of SD mechanisms. This may lead to the stable coexistence of multiple SD mechanisms and to spatial variation in the occurrence of different SD mechanisms along environmental gradients. We applied the model to the SD system of the housefly, a global species with world-wide latitudinal clines in the frequencies of different SD systems, and found that it correctly predicted these clines if specific genes in the housefly SD system were assumed to have temperature-dependent expression levels. We conclude that environmental sensitivity of gene regulatory networks may play an important role in diversification of SD mechanisms.; Sex determination (SD) is the process whereby individuals commit to developing into females or males. Despite its importance to individual fitness, the mechanisms that direct SD can evolve rapidly. In some species, SD is controlled by environmental factors such as temperature; in other species, genetic factors determine whether an individual becomes male or female. It is becoming increasingly clear that in many species, environmental and genetic factors can both play a role in SD. How this affects the evolution of SD mechanisms is still unclear. We present here a model that explores the evolution of SD mechanisms when an environmental factor affects the expression of a gene involved in SD. We use this model to show how different SD mechanisms may evolve under different environmental conditions, and when variation in environmental conditions can lead to stable gradients in the frequency of different SD mechanisms. We discuss the application of our model to the housefly system, which presents a particularly suitable model system for testing our model, as well as to other species and species groups in which environmental influences may affect SD. These findings shed new light on the potential influence of the environment on the evolution of SD mechanisms, particularly in the context of climate change.},
  archive      = {J_EVLETT},
  author       = {Schenkel, Martijn A and Billeter, Jean-Christophe and Beukeboom, Leo W and Pen, Ido},
  doi          = {10.1093/evlett/qrad011},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {132-147},
  shortjournal = {Evol. Lett.},
  title        = {Divergent evolution of genetic sex determination mechanisms along environmental gradients},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Predicting evolution in experimental range expansions of an
aquatic model system. <em>EVLETT</em>, <em>7</em>(3), 121–131. (<a
href="https://doi.org/10.1093/evlett/qrad010">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting range expansion dynamics is an important goal of both fundamental and applied research in conservation and global change biology. However, this is challenging if ecological and evolutionary processes occur on the same time scale. Using the freshwater ciliate ; , we combined experimental evolution and mathematical modeling to assess the predictability of evolutionary change during range expansions. In the experiment, we followed ecological dynamics and trait evolution in independently replicated microcosm populations in range core and front treatments, where episodes of natural dispersal alternated with periods of population growth. These eco-evolutionary conditions were recreated in a predictive mathematical model, parametrized with dispersal and growth data of the 20 founder strains in the experiment. We found that short-term evolution was driven by selection for increased dispersal in the front treatment and general selection for higher growth rates in all treatments. There was a good quantitative match between predicted and observed trait changes. Phenotypic divergence was further mirrored by genetic divergence between range core and front treatments. In each treatment, we found the repeated fixation of the same cytochrome c oxidase I (COI) marker genotype, carried by strains that also were the most likely winners in our model. Long-term evolution in the experimental range front lines resulted in the emergence of a dispersal syndrome, namely a competition—colonization trade-off. Altogether, both model and experiment highlight the potential importance of dispersal evolution as a driver of range expansions. Thus, evolution at range fronts may follow predictable trajectories, at least for simple scenarios, and predicting these dynamics may be possible from knowledge of few key parameters.; Can we predict species range expansions when ecological and evolutionary processes occur at the same time scale? We addressed this question by combining experimental evolution of the ciliate ; , genetic analysis, and mathematical modeling. We show that information on dispersal and growth characteristic of the founder strains composition allows good predictions of the evolutionary changes in experimental treatments mimicking range expansion fronts. In the long term, we observe the emergence of a dispersal syndrome at the range front, possibly the result of de novo evolution. Predicting evolution at range fronts may be possible in the short run from knowledge of few key parameters, which represents an additional tool for conservation and management strategies in times of environmental global changes.},
  archive      = {J_EVLETT},
  author       = {Zilio, Giacomo and Krenek, Sascha and Gougat-Barbera, Claire and Fronhofer, Emanuel A and Kaltz, Oliver},
  doi          = {10.1093/evlett/qrad010},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {121-131},
  shortjournal = {Evol. Lett.},
  title        = {Predicting evolution in experimental range expansions of an aquatic model system},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Is cooperation favored by horizontal gene transfer?
<em>EVLETT</em>, <em>7</em>(3), 113–120. (<a
href="https://doi.org/10.1093/evlett/qrad003">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It has been hypothesized that horizontal gene transfer on plasmids can facilitate the evolution of cooperation, by allowing genes to jump between bacteria, and hence increase genetic relatedness at the cooperative loci. However, we show theoretically that horizontal gene transfer only appreciably increases relatedness when plasmids are rare, where there are many plasmid-free cells available to infect (many opportunities for horizontal gene transfer). In contrast, when plasmids are common, there are few opportunities for horizontal gene transfer, meaning relatedness is not appreciably increased, and so cooperation is not favored. Plasmids, therefore, evolve to be rare and cooperative, or common and noncooperative, meaning plasmid frequency and cooperativeness are never simultaneously high. The overall level of plasmid-mediated cooperation, given by the product of plasmid frequency and cooperativeness, is therefore consistently negligible or low.; Hamilton’s Rule states that cooperation will evolve if the benefit of cooperation accrued by a recipient, ; , is greater than the cost incurred by an actor, ; , where the benefit ; is weighted by the relatedness, ; , between the actor and recipient, leading to ; &gt; ; . It has long been understood that common descent (kinship) provides a straightforward way to achieve high positive relatedness. In recent years, it has been argued that horizontal gene transfer of plasmids, or other mobile genetic elements, can provide another way to achieve high relatedness. The idea is that plasmid transfer increases relatedness by allowing genes to jump between bacteria that are social partners. However, we show theoretically that horizontal gene transfer only appreciably increases relatedness when plasmids are rare, where there are many plasmid-free cells available to infect (many opportunities for horizontal gene transfer). In contrast, when plasmids are common, there are few opportunities for horizontal gene transfer, meaning relatedness is not appreciably increased. Plasmids, therefore, evolve to be rare and cooperative, or common and noncooperative, meaning plasmid frequency and cooperativeness are never simultaneously high. The overall level of plasmid-mediated cooperation, given by the product of plasmid frequency and cooperativeness, is therefore generally low.},
  archive      = {J_EVLETT},
  author       = {Scott, Thomas W and West, Stuart A and Dewar, Anna E and Wild, Geoff},
  doi          = {10.1093/evlett/qrad003},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {113-120},
  shortjournal = {Evol. Lett.},
  title        = {Is cooperation favored by horizontal gene transfer?},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Recurrent selection and reduction in recombination shape the
genomic landscape of divergence across multiple population pairs of
green-backed tit. <em>EVLETT</em>, <em>7</em>(2), 99–111. (<a
href="https://doi.org/10.1093/evlett/qrad005">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Speciation is fundamental for building and maintaining biodiversity. The formation of the highly differentiated genomic regions between diverging taxa has been interpreted as a result of divergence with gene flow, linked selection, and reduction in recombination. It is challenging to unravel these nonexclusive processes in shaping genomic divergence. Here, we investigate the relative roles of these processes in shaping genomic differentiation in a montane bird, the Green-backed Tit (; ). Our genetic structure and demographic analyses identify that four genetic lineages diverge between 838 and 113 thousand years ago and there is evidence of secondary gene flow. The highly divergent genomic regions do not increase with the divergence time, as we found that the old lineages show relatively fewer numbers and smaller sizes of highly differentiated regions than the young divergent lineages (numbers, 118–138 vs. 156–289; sizes, 5.9–6.9 vs. 7.8–14.5 megabase). Across the genome, the outlier windows show a reduction in nucleotide diversity, absolute genetic divergence, and recombination rate, suggesting recurrent selection in regions with low recombination being the major driver of genomic divergence. Finally, we show that secondary gene flow tends to affect the highly differentiated genomic regions if these regions are less likely to be the targets of selection. Altogether, our study shows how common ancestry, recurrent selection, low recombination rate, and gene flow have contributed to the emergence of genomic islands at different stages of speciation.; Disentangling evolutionary forces that drive genomic divergence is necessary for advancing our understanding of speciation. Previous studies have compared genomic landscapes of divergence across different populations and species. Here we present a genomic divergence at multiple stages of lineage divergence in the Green-backed tit (; ). We found that the lineages that have diverged at different stages differ in the degree to which they display genomic peaks of divergence and in the location of these peaks. The lineages that split a long time ago tend to have fewer but more corresponding genomic islands than the lineages that have split more recently. Recurrent selection in regions with low recombination is the major driver of genomic divergence. The highly differentiated genomic regions under the target of selection are more likely resist to genetic introgression. Overall, this study shows that the combination of common ancestry, recurrent selection, low recombination, and gene flow has shaped genomic divergence at different stages across the speciation.},
  archive      = {J_EVLETT},
  author       = {Jiang, Zhiyong and Song, Gang and Luo, Xu and Zhang, Dezhi and Lei, Fumin and Qu, Yanhua},
  doi          = {10.1093/evlett/qrad005},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {99-111},
  shortjournal = {Evol. Lett.},
  title        = {Recurrent selection and reduction in recombination shape the genomic landscape of divergence across multiple population pairs of green-backed tit},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Not just flowering time: A resurrection approach shows
floral attraction traits are changing over time. <em>EVLETT</em>,
<em>7</em>(2), 88–98. (<a
href="https://doi.org/10.1093/evlett/qrad006">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Contemporary anthropogenic changes in climate and landscape form a complex set of selective pressures acting on natural systems, yet, in many systems, we lack information about both whether and how organisms may adapt to these changes. In plants, research has focused on climate-induced changes in phenology and the resultant potential for disruption of plant-pollinator interactions, however, there remains a paucity of knowledge regarding how other pollinator-mediated traits may be involved in the adaptive response. Here, we use resurrection experiments to investigate the phenotypic basis of adaptation in a mixed-mating system plant, the common morning glory (; ). Specifically, we measure temporal and spatial changes in traits grouped into three categories relevant to plant-pollinator interactions - floral morphology, floral rewards, and floral phenology. We show a significant temporal increase in corolla size and shift to earlier flowering times, as well as a potential for increased investment in floral rewards, all of which are driven primarily by populations at more northern latitudes. Additionally, we find evidence for directional selection on floral morphology and phenology and evidence of balancing selection acting on anther-stigma distance. Overall, these results show an adaptive response in line with greater investment in pollinator attraction rather than self-pollination and fine-scale spatial differences in adaptive potential.; Studies of global change in plants typically examine floral phenology due to the potential for pollinator and plant population mismatches. Several other underinvestigated floral traits also play a critical role in mediating these interactions, however, and show the potential to play an adaptive role in plant response to global change. In this work, we use a resurrection approach where we take seeds of the common morning glory collected from the same populations at two different time points and raise them together in a common garden to measure differences over time in multiple traits related to pollinator interactions and self-pollinating behavior. We find evidence of selection for increased corolla size and earlier flowering times as well as an indication of increased investment over time in floral rewards. We also show that there are clear spatial differences in trait evolution within the relatively narrow range of the southeastern United States, with northern populations displaying greater phenotypic shifts. Overall, our work highlights an instance of evolutionary investment in traits that underlie pollinator attraction and identifies phenotypes beyond phenology that may underlie adaptive responses in plants to the complex and rapid environmental changes associated with contemporary global change and biodiversity declines.},
  archive      = {J_EVLETT},
  author       = {Bishop, Sasha G D and Chang, Shu-Mei and Baucom, Regina S},
  doi          = {10.1093/evlett/qrad006},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {88-98},
  shortjournal = {Evol. Lett.},
  title        = {Not just flowering time: A resurrection approach shows floral attraction traits are changing over time},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Macroecological diversification of ants is linked to
angiosperm evolution. <em>EVLETT</em>, <em>7</em>(2), 79–87. (<a
href="https://doi.org/10.1093/evlett/qrad008">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ants are abundant, diverse, and occupy nearly all habitats and regions of the world. Previous work has demonstrated that ant diversification coincided with the rise of the angiosperms, and that several plant traits evolved as ants began to expand their nesting and foraging habits. In this study, we investigate whether associations with plants enabled niche expansion and are linked to climatic niche evolution in ants. Our analysis of over 1,400 ant species reveals that ancestral expansion from forest floors into the canopy and out into non-forested habitats closely followed evolutionary innovations in angiosperms. Several Paleogene-Neogene ant lineages independently diversified in non-forested habitats on multiple continents, tracking the evolution and expansion of elaiosome-bearing and arid-adapted angiosperms. The evolution of arboreal nesting tracked shifts in angiosperm physiology associated with the onset of everwet tropical rainforests, and climatic optima and rates of climatic niche evolution were linked to nesting location, with arboreally nesting groups having warmer and less seasonal climatic optima, and lower rates of climatic niche evolution. Our work further underscores the varied paths by which niche diversification occurred in ants, and how angiosperms influenced the ecological and evolutionary trajectories of interacting lineages.; The success of numerous lineages has at least in part been attributed to shifts in vegetation. The evolutionary history of ants—an abundant, diverse, and widespread lineage—is deeply entwined with the evolution and spread of flowering plants. Here we further characterize the ecological diversification of ants in the context of flowering plants and biome evolution by determining when ants expanded outward from forested to non-forested biomes, and how the evolution of climatic niche is related to arboreal nesting. The Paleogene-Neogene spread of ants into non-forested biomes likely followed the evolution and expansion of arid-adapted angiosperms that provided food to ants. Shifts to arboreal nesting tracked evolutionary innovations in angiosperm physiology linked to the evolution of everwet tropical rainforests. Differences in climatic optima and climatic niche evolution were also linked to nesting location with arboreal nesters having warmer and less seasonal climatic optima coupled with lower rates of climatic niche evolution. Our results contribute new insights into the diverse means by which flowering plant evolution shaped the ecology and evolution of associated lineages.},
  archive      = {J_EVLETT},
  author       = {Nelsen, Matthew P and Moreau, Corrie S and Kevin Boyce, C and Ree, Richard H},
  doi          = {10.1093/evlett/qrad008},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {79-87},
  shortjournal = {Evol. Lett.},
  title        = {Macroecological diversification of ants is linked to angiosperm evolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Condition dependence and the paradox of missing plasticity
costs. <em>EVLETT</em>, <em>7</em>(2), 67–78. (<a
href="https://doi.org/10.1093/evlett/qrad009">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotypic plasticity plays a key role in adaptation to changing environments. However, plasticity is neither perfect nor ubiquitous, implying that fitness costs may limit the evolution of phenotypic plasticity in nature. The measurement of such costs of plasticity has proved elusive; decades of experiments show that fitness costs of plasticity are often weak or nonexistent. Here, we show that this paradox could potentially be explained by condition dependence. We develop two models differing in their assumptions about how condition dependence arises; both models show that variation in condition can readily mask costs of plasticity even when such costs are substantial. This can be shown simply in a model where plasticity itself evolves condition dependence, which would be expected if costly. Yet similar effects emerge from an alternative model where trait expression itself is condition-dependent. In this more complex model, the average condition in each environment and genetic covariance in condition across environments both determine when costs of plasticity can be revealed. Analogous to the paradox of missing trade-offs between life history traits, our models show that variation in condition can mask costs of plasticity even when costs exist, and suggest this conclusion may be robust to the details of how condition affects trait expression. Our models suggest that condition dependence can also account for the often-observed pattern of elevated plasticity costs inferred in stressful environments, the maintenance of genetic variance in plasticity, and provides insight into experimental and biological scenarios ideal for revealing a cost of phenotypic plasticity.; Phenotypic plasticity—where the same genotype produces different phenotypes when expressed in different environments—is an important source of phenotypic variation. When plasticity is adaptive, organisms can match their traits to their local environment, a clear advantage in variable environments. Yet plasticity is not perfect, in that many traits show limited plasticity and no trait is completely plastic, despite the obvious advantages this would entail. Thus, plasticity is thought to often carry a fitness cost that balances with the benefits of being able to modify trait expression in response to the environment. However, many experiments in a variety of organisms have failed to reveal any fitness cost of plasticity, resulting in a long-standing paradox: plasticity should be costly, yet we rarely can measure such costs. Here, we develop models of condition-dependent trait expression and show that variation in individual condition, or resource acquisition, can mask the appearance of costs. When trait expression in one environment or plasticity itself is condition-dependent, fitness costs of plasticity will rarely be estimable in typical genotype x environment experiments, because this variation in individual quality can overwhelm the fitness variance arising from costs of plasticity. Costs of plasticity will rarely be measurable under such circumstances even when these costs are substantial. We show that these same effects arise even under two very different assumptions about the form of condition dependence, suggesting condition dependence may play a role in explaining the longstanding paradox of missing plasticity costs. Our conclusions are reminiscent of a similar paradox in life history evolution, that of missing costs of reproduction, and point to experimental designs that are likely to reveal costs of plasticity when they are present.},
  archive      = {J_EVLETT},
  author       = {De Lisle, Stephen P and Rowe, Locke},
  doi          = {10.1093/evlett/qrad009},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {67-78},
  shortjournal = {Evol. Lett.},
  title        = {Condition dependence and the paradox of missing plasticity costs},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Limited host availability disrupts the genetic correlation
between virulence and transmission. <em>EVLETT</em>, <em>7</em>(1),
58–66. (<a href="https://doi.org/10.1093/evlett/qrac008">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Virulence is expected to be linked to parasite fitness via transmission. However, it is not clear whether this relationship is genetically determined, nor if it differs when transmission occurs continuously during, or only at the end of, the infection period. Here, we used inbred lines of the macroparasitic spider mite ; to disentangle genetic vs. nongenetic correlations among traits, while varying parasite density and opportunities for transmission. A positive genetic correlation between virulence and the number of transmitting stages produced was found under continuous transmission. However, if transmission occurred only at the end of the infection period, this genetic correlation disappeared. Instead, we observed a negative relationship between virulence and the number of transmitting stages, driven by density dependence. Thus, within-host density dependence caused by reduced opportunities for transmission may hamper selection for higher virulence, providing a novel explanation as to why limited host availability leads to lower virulence.; Parasite fitness increases with parasite growth within hosts and subsequent transmission to uninfected hosts. However, within host parasite replication leads to virulence, that is, parasite-induced harm to hosts. Thus, parasite fitness increases with virulence. However, too much harm can lead to host death, which hampers transmission. Therefore, parasite fitness should be maximized at intermediate virulence levels. Despite the centrality of this virulence transmission trade-off to the study of host-parasite interactions, whether it is genetically determined or shaped by environmental factors, remains an open question. To address this, we experimentally infected bean plants with different densities of individuals from genetically distinct lines of the herbivorous spider-mite ; , a plant macroparasite, and measured replication rate, virulence, and transmission. To test the impact of transmission timing on correlations among traits, we let parasites transmit to uninfected hosts continuously during the infection or only at the end of the infection period. We found a positive genetic correlation between virulence and transmission under continuous transmission, meaning that lines with high virulence transmitted more and lines with low virulence transmitted less. In contrast, when transmission was restricted to the end of the infection, no genetic correlation was found between virulence and transmission. There was, however, a strong effect of parasite density on the relationship between the two variables in this transmission scenario. Indeed, high parasite densities led to strong density dependence and thus a negative correlation between virulence and transmission. Thus, there was a lower transmission from highly harmed hosts, independently of the line infecting it, and the optimal transmission was at intermediate levels of virulence, as expected. Therefore, both the cause of the correlation between virulence and transmission and its form was modulated by opportunities for transmission during infection. This finding has major consequences for the understanding of virulence in natural systems and its potential control.},
  archive      = {J_EVLETT},
  author       = {Godinho, Diogo P and Rodrigues, Leonor R and Lefèvre, Sophie and Delteil, Laurane and Mira, André F and Fragata, Inês R and Magalhães, Sara and Duncan, Alison B},
  doi          = {10.1093/evlett/qrac008},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {58-66},
  shortjournal = {Evol. Lett.},
  title        = {Limited host availability disrupts the genetic correlation between virulence and transmission},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Experimental and theoretical support for costs of plasticity
and phenotype in a nematode cannibalistic trait. <em>EVLETT</em>,
<em>7</em>(1), 48–57. (<a
href="https://doi.org/10.1093/evlett/qrac001">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Developmental plasticity is the ability of a genotype to express multiple phenotypes under different environmental conditions and has been shown to facilitate the evolution of novel traits. However, while the associated cost of plasticity, i.e., the loss in fitness due to the ability to express plasticity in response to environmental change, and the cost of phenotype, i.e., the loss of fitness due to expressing a fixed phenotype across environments, have been theoretically predicted, empirically such costs remain poorly documented and little understood. Here, we use a plasticity model system, hermaphroditic nematode ; , to experimentally measure these costs in wild isolates under controlled laboratory conditions. ; can develop either a bacterial feeding or predatory mouth morph in response to different external stimuli, with natural variation of mouth-morph ratios between strains. We first demonstrated the cost of phenotype by analyzing fecundity and developmental speed in relation to mouth morphs across the ; phylogenetic tree. Then, we exposed ; strains to two distinct microbial diets that induce strain-specific mouth-form ratios. Our results indicate that the plastic strain does shoulder a cost of plasticity, i.e., the diet-induced predatory mouth morph is associated with reduced fecundity and slower developmental speed. In contrast, the non-plastic strain suffers from the cost of phenotype since its phenotype does not change to match the unfavorable bacterial diet but shows increased fitness and higher developmental speed on the favorable diet. Furthermore, using a stage-structured population model based on empirically derived life history parameters, we show how population structure can alleviate the cost of plasticity in ; . The results of the model illustrate the extent to which the costs associated with plasticity and its effect on competition depend on ecological factors. This study provides support for costs of plasticity and phenotype based on empirical and modeling approaches.; A genotype able to express a range of phenotypes in response to environmental conditions, that is to demonstrate developmental plasticity, would be a Darwinian demon, able to infinitely adapt and outcompete those genotypes that require a genetic change to express a phenotype fit to an environment. It has been suggested that the absence of such demons in nature is due to the cost of plasticity, i.e., developmental plasticity results in a reduction of biological fitness compared to a genotype that facultatively expresses a phenotype matching the environment. While conceptually simple, measuring the cost of plasticity in nature has proven a major challenge. We use the nematode ; to measure the cost of plasticity. During its development, ; can assume one of two possible mouth forms: predatory or non-predatory. The likelihood of developing any of these two mouth forms is determined by a gene regulatory network, which itself is affected by a wide range of environmental conditions, including diet. We used two strains of ; and grew them on two different bacterial diets. The plastic strain was capable of switching from non-predatory to predatory mouth form depending on the diet, while the non-plastic strain could only express the predatory mouth form on either of the diets. By measuring the number of eggs laid in both strains on each diet, we show that the plastic response is associated with a reduction in fecundity, thus providing a clear example of the cost of plasticity. We then use a stage-structured model to simulate the population dynamics of the plastic and the non-plastic strains. Our simulation shows that the cost of plasticity is highly context dependent, and its ecological ramifications can be greatly influenced by biotic and abiotic factors.},
  archive      = {J_EVLETT},
  author       = {Dardiry, Mohannad and Piskobulu, Veysi and Kalirad, Ata and Sommer, Ralf J},
  doi          = {10.1093/evlett/qrac001},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {48-57},
  shortjournal = {Evol. Lett.},
  title        = {Experimental and theoretical support for costs of plasticity and phenotype in a nematode cannibalistic trait},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The immediate effects of polyploidization of spirodela
polyrhiza change in a strain-specific way along environmental gradients.
<em>EVLETT</em>, <em>7</em>(1), 37–47. (<a
href="https://doi.org/10.1093/evlett/qrac003">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The immediate effects of plant polyploidization are well characterized and it is generally accepted that these morphological, physiological, developmental, and phenological changes contribute to polyploid establishment. Studies on the environmental dependence of the immediate effects of whole-genome duplication (WGD) are, however, scarce but suggest that these immediate effects are altered by stressful conditions. As polyploid establishment seems to be associated with environmental disturbance, the relationship between ploidy-induced phenotypical changes and environmental conditions is highly relevant. Here, we use a common garden experiment on the greater duckweed ; to test whether the immediate effects of WGD can facilitate the establishment of tetraploid duckweed along gradients of two environmental stressors. Because successful polyploid establishment often depends on recurrent polyploidization events, we include four genetically diverse strains and assess whether these immediate effects are strain-specific. We find evidence that WGD can indeed confer a fitness advantage under stressful conditions and that the environment affects ploidy-induced changes in fitness and trait reaction norms in a strain-specific way.; Polyploidization, resulting from genome duplication and/or merger, is a major force in plant evolution. Despite high numbers of polyploid species and intraspecific karyotypical variation, the challenges faced by a newly formed polyploid are multifarious. Higher resource requirements, difficulties finding compatible polyploid mates, genomic instability, and competition with their well-adapted diploid ancestors all impede establishment. Nevertheless, while many polyploids are short-lived, some of them manage to survive, reproduce, and even thrive. The basis for this inequality amongst polyploids remains largely unknown. Part of the key to polyploid success must lie in the immediate effects of genome doubling. Genome size is correlated with cell size and as such, genome duplication triggers a cell size increase and a whole battery of cascading effects on the phenotype. Although these immediate effects are well characterized, variation between and even within species is considerable. Using a common garden experiment in which we exposed four genetically different diploid-tetraploid pairs of the greater duckweed ; to stress gradients, we hope to improve our understanding of what determines the fate of neopolyploid plants. Here, we show that although the immediate effects of WGD can result in a decreased fitness of the tetraploid in the prevailing environment, they can confer a fitness advantage in another and that ploidy-induced changes in trait plasticity and fitness homeostasis are strain-specific. Consequently, we show that the fate of neopolyploid plants depends at least partially on the interplay of the identity of the individuals present in the polyploid population (the genotypes), and the environmental conditions they face. Our findings support the idea that stressful environments can facilitate polyploid establishment and that recurrent polyploidization can help polyploid establishment through variation in the outcomes of independent polyploidization events. Whenever possible, future studies on nucleotypic effects should include diverse ecotypes and environmental conditions in their designs.},
  archive      = {J_EVLETT},
  author       = {Bafort, Quinten and Wu, Tian and Natran, Annelore and De Clerck, Olivier and Van de Peer, Yves},
  doi          = {10.1093/evlett/qrac003},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {37-47},
  shortjournal = {Evol. Lett.},
  title        = {The immediate effects of polyploidization of spirodela polyrhiza change in a strain-specific way along environmental gradients},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Population genomics of the island thrush elucidates one of
earth’s great archipelagic radiations. <em>EVLETT</em>, <em>7</em>(1),
24–36. (<a href="https://doi.org/10.1093/evlett/qrac006">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tropical islands are renowned as natural laboratories for evolutionary study. Lineage radiations across tropical archipelagos are ideal systems for investigating how colonization, speciation, and extinction processes shape biodiversity patterns. The expansion of the island thrush across the Indo-Pacific represents one of the largest yet most perplexing island radiations of any songbird species. The island thrush exhibits a complex mosaic of pronounced plumage variation across its range and is arguably the world’s most polytypic bird. It is a sedentary species largely restricted to mountain forests, yet it has colonized a vast island region spanning a quarter of the globe. We conducted a comprehensive sampling of island thrush populations and obtained genome-wide SNP data, which we used to reconstruct its phylogeny, population structure, gene flow, and demographic history. The island thrush evolved from migratory Palearctic ancestors and radiated explosively across the Indo-Pacific during the Pleistocene, with numerous instances of gene flow between populations. Its bewildering plumage variation masks a biogeographically intuitive stepping stone colonization path from the Philippines through the Greater Sundas, Wallacea, and New Guinea to Polynesia. The island thrush’s success in colonizing Indo-Pacific mountains can be understood in light of its ancestral mobility and adaptation to cool climates; however, shifts in elevational range, degree of plumage variation and apparent dispersal rates in the eastern part of its range raise further intriguing questions about its biology.; Island radiations are powerful systems for studying evolutionary processes and biodiversity buildup. Here we use population genomics methods to investigate one of the most spectacular and perplexing archipelagic radiations among birds. The island thrush displays a complex mosaic of plumage variation and occupies a vast range across the Indo-Pacific despite being sedentary and largely restricted to mountains. We reconstruct its evolutionary relationships, demographic history, and population structure to provide a detailed picture of an explosive radiation from Asia during the Pleistocene. We illustrate how departure from the mainland and subsequent founder events degraded genetic variation, and detect considerable cross-population gene flow. Our results place focus on the dynamics of populations at the leading edge of radiations vs. those behind it.},
  archive      = {J_EVLETT},
  author       = {Reeve, Andrew Hart and Gower, Graham and Pujolar, José Martín and Smith, Brian Tilston and Petersen, Bent and Olsson, Urban and Haryoko, Tri and Koane, Bonny and Maiah, Gibson and Blom, Mozes P K and Ericson, Per G P and Irestedt, Martin and Racimo, Fernando and Jønsson, Knud Andreas},
  doi          = {10.1093/evlett/qrac006},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {24-36},
  shortjournal = {Evol. Lett.},
  title        = {Population genomics of the island thrush elucidates one of earth’s great archipelagic radiations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Individual- and group-level sex ratios under local mate
competition: Consequences of infanticide and reproductive dominance.
<em>EVLETT</em>, <em>7</em>(1), 13–23. (<a
href="https://doi.org/10.1093/evlett/qrac005">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extremely female-biased sex ratios of parasitoid wasps in multiple-foundress groups challenges evolutionary theory which predicts diminishing bias as foundress numbers increase. Recent theory based on foundress cooperation has achieved qualitative rather than quantitative success in explaining bias among parasitoids in the genus ; . Here, we develop an explanation, expanding the theory of local mate competition, based on the observation that male production seems dominated by some foundresses within groups. Two sex ratio effects arise from such reproductive dominance: an immediate effect via suppression of male production, and a long-term evolutionary response to reproductive skew. We analyze the outcome of these effects at the individual and group level, the latter being more readily observable. Three model scenarios are analyzed: (1) random killing of developing sons in a group by all foundresses, without reproductive skew, (2) the development of reproductive dominance by some foundresses after sex allocation decisions by all foundresses have been implemented, and (3) reproductive dominance within foundress groups before sex allocation decisions are implemented. The 3 scenarios have subtly different implications for sex ratio evolution, with Models 2 and 3 being novel additions to theory, showing how reproductive dominance can alter the outcome of sex ratio evolution. All models match observations in their outcomes better than other recently proposed theory, but Models 2 and 3 are closest to observations in their underlying assumptions. Further, Model 2 shows that differential offspring mortality after parental investment can influence the primary sex ratio even when random with respect to parental and offspring characters, but targeted at entire clutches. The novel models are solved for both diploid and haplodiploid genetic systems, and confirmed with simulations. Overall, these models provide a feasible explanation for the extremely female-biased sex ratios produced by multi-foundress groups and expand the scope of local mate competition theory to consider reproductive dominance.; The evolution of the offspring sex ratio—how many sons ; daughters parents should make—is one of the most widely studied questions in theoretical evolutionary biology. It is also one of the most extensively tested, with a good match between data and theory. However, there are some groups of insects where the theoretical predictions do not match the data. One is the parasitoid genus ; , with many more daughters relative to sons than theory predicts when groups of mothers produce communal broods of offspring. It has recently also come to light that these parasitoids exhibit aggressive behavior and that son production is dominated by a few mothers within each group. We use theory to address the question “How does the predicted sex ratio change if some mothers (‘dominants’) can suppress the ability of other mothers (‘subordinates’) to produce sons”? The answer is far from intuitively obvious, due to two simultaneous effects influencing the observable sex ratio: if some sons are killed, the ratio of sons to daughters becomes lower in the short term; and in the long run, the son-killing behavior may influence the evolution of optimal sex allocation. We find that the combined effect of these two processes does indeed predict very female-biased observable sex ratios. We examine different model setups, where mothers either can or cannot adjust the sex ratio of eggs they lay according to their status as a dominant or subordinate. Overall, we predict that male infanticide can produce sex ratios that are as female-biased as those observed. We also discuss, and explore with theory, reasons why male killing can evolve, and find that it can evolve quite easily, even if the ability of a mother to tell the difference between her own sons and the sons of others is very poor.},
  archive      = {J_EVLETT},
  author       = {Lehtonen, Jussi and Malabusini, Serena and Guo, Xiaomeng and Hardy, Ian C W},
  doi          = {10.1093/evlett/qrac005},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {13-23},
  shortjournal = {Evol. Lett.},
  title        = {Individual- and group-level sex ratios under local mate competition: Consequences of infanticide and reproductive dominance},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Slower-x: Reduced efficiency of selection in the early
stages of x chromosome evolution. <em>EVLETT</em>, <em>7</em>(1), 4–12.
(<a href="https://doi.org/10.1093/evlett/qrac004">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Differentiated X chromosomes are expected to have higher rates of adaptive divergence than autosomes, if new beneficial mutations are recessive (the “faster-X effect”), largely because these mutations are immediately exposed to selection in males. The evolution of X chromosomes after they stop recombining in males, but before they become hemizygous, has not been well explored theoretically. We use the diffusion approximation to infer substitution rates of beneficial and deleterious mutations under such a scenario. Our results show that selection is less efficient on diploid X loci than on autosomal and hemizygous X loci under a wide range of parameters. This “slower-X” effect is stronger for genes affecting primarily (or only) male fitness, and for sexually antagonistic genes. These unusual dynamics suggest that some of the peculiar features of X chromosomes, such as the differential accumulation of genes with sex-specific functions, may start arising earlier than previously appreciated.; In many species sex is determined by a pair of sex chromosomes, usually termed X and Y. Humans share sex chromosomes with all mammals apart from monotremes like platypus and echidna, but sex chromosomes evolved independently from a pair of autosomes many times in evolution. Sex chromosome evolution usually involves the degeneration of the Y chromosome, which loses most of the ancestral gene content. For example, the human Y chromosome contains around 80 genes, compared to 800 on the X. This means that XY males carry the majority of their X-linked genes in only one copy (as opposed to two copies of genes that are on the autosomes). So far, studies of sex chromosome evolution focused on evolutionarily old X chromosomes with degenerated Y chromosomes. This is in part because old sex chromosomes are easier to identify due to the extensive differences between X and Y. With the increasing availability of sequencing, large-scale population genomic studies have detected unusual patterns of evolution on young X chromosomes, but the evolutionary forces at play are currently unknown. Here, we model the evolutionary dynamics of young X chromosomes with a non-degenerated Y and show that young X chromosomes can experience degeneration, and demasculinization, that is, loss of male-important genes. This is in contrast with the prevailing perspective that X chromosomes with non-degenerated Y counterparts have similar evolutionary dynamics to autosomes.},
  archive      = {J_EVLETT},
  author       = {Mrnjavac, Andrea and Khudiakova, Ksenia A and Barton, Nicholas H and Vicoso, Beatriz},
  doi          = {10.1093/evlett/qrac004},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {4-12},
  shortjournal = {Evol. Lett.},
  title        = {Slower-X: Reduced efficiency of selection in the early stages of x chromosome evolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
</ul>

</body>
</html>
