<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATCS_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs---179">NATCS - 179</h2>
<ul>
<li><details>
<summary>
(2021). How connectivity rules and synaptic properties shape the
efficacy of pattern separation in the entorhinal cortex–dentate
gyrus–CA3 network. <em>NATCS</em>, <em>1</em>(12), 830–842. (<a
href="https://doi.org/10.1038/s43588-021-00157-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pattern separation is a fundamental brain computation that converts small differences in input patterns into large differences in output patterns. Several synaptic mechanisms of pattern separation have been proposed, including code expansion, inhibition and plasticity; however, which of these mechanisms play a role in the entorhinal cortex (EC)–dentate gyrus (DG)–CA3 circuit, a classical pattern separation circuit, remains unclear. Here we show that a biologically realistic, full-scale EC–DG–CA3 circuit model, including granule cells (GCs) and parvalbumin-positive inhibitory interneurons (PV+-INs) in the DG, is an efficient pattern separator. Both external gamma-modulated inhibition and internal lateral inhibition mediated by PV+-INs substantially contributed to pattern separation. Both local connectivity and fast signaling at GC–PV+-IN synapses were important for maximum effectiveness. Similarly, mossy fiber synapses with conditional detonator properties contributed to pattern separation. By contrast, perforant path synapses with Hebbian synaptic plasticity and direct EC–CA3 connection shifted the network towards pattern completion. Our results demonstrate that the specific properties of cells and synapses optimize higher-order computations in biological networks and might be useful to improve the deep learning capabilities of technical networks. The authors present a full-scale model of the entorhinal cortex–dentate gyrus–CA3 network based on experimental data to show that fast lateral inhibition plays a key role in pattern separation.},
  archive      = {J_NATCS},
  author       = {Guzman, S. Jose and Schlögl, Alois and Espinoza, Claudia and Zhang, Xiaomin and Suter, Benjamin A. and Jonas, Peter},
  doi          = {10.1038/s43588-021-00157-1},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {830-842},
  shortjournal = {Nat. Comput. Sci},
  title        = {How connectivity rules and synaptic properties shape the efficacy of pattern separation in the entorhinal cortex–dentate gyrus–CA3 network},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A framework for data-driven solution and parameter
estimation of PDEs using conditional generative adversarial networks.
<em>NATCS</em>, <em>1</em>(12), 819–829. (<a
href="https://doi.org/10.1038/s43588-021-00171-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we employ and adapt the image-to-image translation concept based on conditional generative adversarial networks (cGAN) for learning a forward and an inverse solution operator of partial differential equations (PDEs). We focus on steady-state solutions of coupled hydromechanical processes in heterogeneous porous media and present the parameterization of the spatially heterogeneous coefficients, which is exceedingly difficult using standard reduced-order modeling techniques. We show that our framework provides a speed-up of at least 2,000 times compared to a finite-element solver and achieves a relative root-mean-square error (r.m.s.e.) of less than 2\% for forward modeling. For inverse modeling, the framework estimates the heterogeneous coefficients, given an input of pressure and/or displacement fields, with a relative r.m.s.e. of less than 7\%, even for cases where the input data are incomplete and contaminated by noise. The framework also provides a speed-up of 120,000 times compared to a Gaussian prior-based inverse modeling approach while also delivering more accurate results. A data-driven solution of partial differential equations is developed with conditional generative adversarial networks, which could be used in both forward and inverse problems.},
  archive      = {J_NATCS},
  author       = {Kadeethum, Teeratorn and O’Malley, Daniel and Fuhg, Jan Niklas and Choi, Youngsoo and Lee, Jonghyun and Viswanathan, Hari S. and Bouklas, Nikolaos},
  doi          = {10.1038/s43588-021-00171-3},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {819-829},
  shortjournal = {Nat. Comput. Sci},
  title        = {A framework for data-driven solution and parameter estimation of PDEs using conditional generative adversarial networks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cluster learning-assisted directed evolution.
<em>NATCS</em>, <em>1</em>(12), 809–818. (<a
href="https://doi.org/10.1038/s43588-021-00168-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed evolution, a strategy for protein engineering, optimizes protein properties (that is, fitness) by expensive and time-consuming screening or selection of a large mutational sequence space. Machine learning-assisted directed evolution (MLDE), which screens sequence properties in silico, can accelerate the optimization and reduce the experimental burden. This work introduces an MLDE framework, cluster learning-assisted directed evolution (CLADE), which combines hierarchical unsupervised clustering sampling and supervised learning to guide protein engineering. The clustering sampling selectively picks and screens variants in targeted subspaces, which guides the subsequent generation of diverse training sets. In the last stage, accurate predictions via supervised learning models improve the final outcomes. By sequentially screening 480 sequences out of 160,000 in a four-site combinatorial library with five equal experimental batches, CLADE achieves global maximal fitness hit rates of up to 91.0\% and 34.0\% for the GB1 and PhoQ datasets, respectively, improved from the values of 18.6\% and 7.2\% obtained by random sampling-based MLDE. A machine learning-assisted directed evolution method is developed, combining hierarchical unsupervised clustering and supervised learning, to guide protein engineering by iteratively exploring the large mutational sequence space.},
  archive      = {J_NATCS},
  author       = {Qiu, Yuchi and Hu, Jian and Wei, Guo-Wei},
  doi          = {10.1038/s43588-021-00168-y},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {809-818},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cluster learning-assisted directed evolution},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A shortcut to the thermodynamic limit for quantum many-body
calculations of metals. <em>NATCS</em>, <em>1</em>(12), 801–808. (<a
href="https://doi.org/10.1038/s43588-021-00165-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computationally efficient and accurate quantum mechanical approximations to solve the many-electron Schrödinger equation are crucial for computational materials science. Methods such as coupled cluster theory show potential for widespread adoption if computational cost bottlenecks can be removed. For example, extremely dense k-point grids are required to model long-range electronic correlation effects, particularly for metals. Although these grids can be made more effective by averaging calculations over an offset (or twist angle), the resultant cost in time for coupled cluster theory is prohibitive. We show here that a single special twist angle can be found using the transition structure factor, which provides the same benefit as twist averaging with one or two orders of magnitude reduction in computational time. We demonstrate that this not only works for metal systems but also is applicable to a broader range of materials, including insulators and semiconductors. The authors demonstrate an effective approach to lower the computing time required to accurately reach the thermodynamic limit in quantum many-body calculations. This method can be applied to solve problems in a wide range of material systems, including metals, insulators and semiconductors.},
  archive      = {J_NATCS},
  author       = {Mihm, Tina N. and Schäfer, Tobias and Ramadugu, Sai Kumar and Weiler, Laura and Grüneis, Andreas and Shepherd, James J.},
  doi          = {10.1038/s43588-021-00165-1},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {801-808},
  shortjournal = {Nat. Comput. Sci},
  title        = {A shortcut to the thermodynamic limit for quantum many-body calculations of metals},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Emergence of urban growth patterns from human mobility
behavior. <em>NATCS</em>, <em>1</em>(12), 791–800. (<a
href="https://doi.org/10.1038/s43588-021-00160-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cities grow in a bottom-up manner, leading to fractal-like urban morphologies characterized by scaling laws. The correlated percolation model has succeeded in modeling urban geometries by imposing strong spatial correlations; however, the origin of the underlying mechanisms behind spatially correlated urban growth remains largely unknown. Our understanding of human movements has recently been revolutionized thanks to the increasing availability of large-scale human mobility data. This paper introduces a computational urban growth model that captures spatially correlated urban growth with a micro-foundation in human mobility behavior. We compare the proposed model with three empirical datasets, discovering that strong social interactions and long-term memory effects in human movements are two fundamental principles responsible for fractal-like urban morphology, along with the three important laws of urban growth. Our model connects the empirical findings in urban growth patterns and human mobility behavior. The study shows that a memory-aware and socially coupled human movement model can reproduce urban growth patterns at the macro level, providing a bottom-up approach to understand urban growth and to reveal its connection to human mobility behavior.},
  archive      = {J_NATCS},
  author       = {Xu, Fengli and Li, Yong and Jin, Depeng and Lu, Jianhua and Song, Chaoming},
  doi          = {10.1038/s43588-021-00160-6},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {791-800},
  shortjournal = {Nat. Comput. Sci},
  title        = {Emergence of urban growth patterns from human mobility behavior},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Uncovering cell identity through differential stability with
cepo. <em>NATCS</em>, <em>1</em>(12), 784–790. (<a
href="https://doi.org/10.1038/s43588-021-00172-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of single-cell RNA-sequencing (scRNA-seq) allows observation of different cells at multi-tiered complexity in the same microenvironment. To get insights into cell identity using scRNA-seq data, we present Cepo, which generates cell-type-specific gene statistics of differentially stable genes from scRNA-seq data to define cell identity. When applied to multiple datasets, Cepo outperforms current methods in assigning cell identity and enhances several cell identification applications such as cell-type characterisation, spatial mapping of single cells and lineage inference of single cells. Defining cell identity is a fundamental task in dissecting the cellular heterogeneity in single-cell data. Here the authors developed Cepo, a method to uncover cell identity genes and enhance the retrieval of cellular identities from scRNA-seq data.},
  archive      = {J_NATCS},
  author       = {Kim, Hani Jieun and Wang, Kevin and Chen, Carissa and Lin, Yingxin and Tam, Patrick P. L. and Lin, David M. and Yang, Jean Y. H. and Yang, Pengyi},
  doi          = {10.1038/s43588-021-00172-2},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {784-790},
  shortjournal = {Nat. Comput. Sci},
  title        = {Uncovering cell identity through differential stability with cepo},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Insights into hippocampal network function. <em>NATCS</em>,
<em>1</em>(12), 782–783. (<a
href="https://doi.org/10.1038/s43588-021-00159-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study proposes a full-scale model of the entorhinal cortex–dentate gyrus–CA3 network, providing a conceptual overview of the computational properties of this brain network, to show that it is an efficient pattern separator.},
  archive      = {J_NATCS},
  author       = {Aertsen, Ad},
  doi          = {10.1038/s43588-021-00159-z},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {782-783},
  shortjournal = {Nat. Comput. Sci},
  title        = {Insights into hippocampal network function},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Application of wavefunction methods to metals.
<em>NATCS</em>, <em>1</em>(12), 780–781. (<a
href="https://doi.org/10.1038/s43588-021-00175-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accurate determination of correlation energy is a challenging task in many-electron quantum chemistry calculations, especially for metals. A recent work proposes an efficient scheme to speed up the calculation of correlation energy, reducing the computational time by up to two orders of magnitude.},
  archive      = {J_NATCS},
  author       = {Sun, Jianwei},
  doi          = {10.1038/s43588-021-00175-z},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {780-781},
  shortjournal = {Nat. Comput. Sci},
  title        = {Application of wavefunction methods to metals},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Bridging human mobility and urban growth. <em>NATCS</em>,
<em>1</em>(12), 778–779. (<a
href="https://doi.org/10.1038/s43588-021-00166-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A human mobility model that takes into account social interaction and long-term memory mechanisms is proposed, shedding more light onto the interplay between human movements and urban growth.},
  archive      = {J_NATCS},
  author       = {Wang, Pu},
  doi          = {10.1038/s43588-021-00166-0},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {778-779},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bridging human mobility and urban growth},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Pushing the bounds of multiconfigurational methods.
<em>NATCS</em>, <em>1</em>(12), 777. (<a
href="https://doi.org/10.1038/s43588-021-00179-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-021-00179-9},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {777},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pushing the bounds of multiconfigurational methods},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Analyzing noise for quantum advantage. <em>NATCS</em>,
<em>1</em>(12), 776. (<a
href="https://doi.org/10.1038/s43588-021-00178-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00178-w},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {776},
  shortjournal = {Nat. Comput. Sci},
  title        = {Analyzing noise for quantum advantage},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Evolution through the lens of genetic codes.
<em>NATCS</em>, <em>1</em>(12), 775. (<a
href="https://doi.org/10.1038/s43588-021-00177-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00177-x},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {775},
  shortjournal = {Nat. Comput. Sci},
  title        = {Evolution through the lens of genetic codes},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Learning to read. <em>NATCS</em>, <em>1</em>(12), 774. (<a
href="https://doi.org/10.1038/s43588-021-00176-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00176-y},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {774},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning to read},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Thinking fast and slow in disaster decision-making with
smart city digital twins. <em>NATCS</em>, <em>1</em>(12), 771–773. (<a
href="https://doi.org/10.1038/s43588-021-00174-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many cities are vulnerable to disaster-related mortality and economic loss. Smart City Digital Twins can be used to facilitate disaster decision-making and influence policy, but first they must accurately capture, predict, and adapt to the city’s dynamics, including the varying pace at which changes unfold.},
  archive      = {J_NATCS},
  author       = {Mohammadi, Neda and Taylor, John E.},
  doi          = {10.1038/s43588-021-00174-0},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {771-773},
  shortjournal = {Nat. Comput. Sci},
  title        = {Thinking fast and slow in disaster decision-making with smart city digital twins},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fighting COVID-19 with HPC. <em>NATCS</em>, <em>1</em>(12),
769–770. (<a href="https://doi.org/10.1038/s43588-021-00180-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A special prize for outstanding research in high-performance computing (HPC) applied to COVID-19-related challenges sheds light on the computational science community’s efforts to fight the ongoing global crisis.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00180-2},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {769-770},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fighting COVID-19 with HPC},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Biomolecular modeling thrives in the
age of technology. <em>NATCS</em>, <em>1</em>(11), 767. (<a
href="https://doi.org/10.1038/s43588-021-00170-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Schlick, Tamar and Portillo-Ledesma, Stephanie},
  doi          = {10.1038/s43588-021-00170-4},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {767},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Biomolecular modeling thrives in the age of technology},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A rigid body framework for multicellular modeling.
<em>NATCS</em>, <em>1</em>(11), 754–766. (<a
href="https://doi.org/10.1038/s43588-021-00154-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Off-lattice models are a well-established approach in multicellular modeling, where cells are represented as points that are free to move in space. The representation of cells as point objects is useful in a wide range of settings, particularly when large populations are involved; however, a purely point-based representation is not naturally equipped to deal with objects that have length, such as cell boundaries or external membranes. Here we introduce an off-lattice modeling framework that exploits rigid body mechanics to represent objects using a collection of conjoined one-dimensional edges in a viscosity-dominated system. This framework can be used to represent cells as free moving polygons, to allow epithelial layers to smoothly interact with themselves, to model rod-shaped cells such as bacteria and to robustly represent membranes. We demonstrate that this approach offers solutions to the problems that limit the scope of current off-lattice multicellular models. The authors present a framework for modeling cell interactions using rigid bodies, which can used to represent cells as free moving polygons, to allow epithelial layers to smoothly interact, to model bacteria and to robustly represent membranes.},
  archive      = {J_NATCS},
  author       = {Brown, Phillip J. and Green, J. Edward F. and Binder, Benjamin J. and Osborne, James M.},
  doi          = {10.1038/s43588-021-00154-4},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {754-766},
  shortjournal = {Nat. Comput. Sci},
  title        = {A rigid body framework for multicellular modeling},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identifiability and predictability of integer- and
fractional-order epidemiological models using physics-informed neural
networks. <em>NATCS</em>, <em>1</em>(11), 744–753. (<a
href="https://doi.org/10.1038/s43588-021-00158-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We analyze a plurality of epidemiological models through the lens of physics-informed neural networks (PINNs) that enable us to identify time-dependent parameters and data-driven fractional differential operators. In particular, we consider several variations of the classical susceptible-infectious-removed (SIR) model by introducing more compartments and fractional-order and time-delay models. We report the results for the spread of COVID-19 in New York City, Rhode Island and Michigan states and Italy, by simultaneously inferring the unknown parameters and the unobserved dynamics. For integer-order and time-delay models, we fit the available data by identifying time-dependent parameters, which are represented by neural networks. In contrast, for fractional differential models, we fit the data by determining different time-dependent derivative orders for each compartment, which we represent by neural networks. We investigate the structural and practical identifiability of these unknown functions for different datasets, and quantify the uncertainty associated with neural networks and with control measures in forecasting the pandemic. A plurality of epidemiological models are analyzed using physics-informed neural networks to identify time-dependent parameters and data-driven fractional models. The results are reported for different geographical locations by inferring unknown parameters and unobserved dynamics.},
  archive      = {J_NATCS},
  author       = {Kharazmi, Ehsan and Cai, Min and Zheng, Xiaoning and Zhang, Zhen and Lin, Guang and Karniadakis, George Em},
  doi          = {10.1038/s43588-021-00158-0},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {744-753},
  shortjournal = {Nat. Comput. Sci},
  title        = {Identifiability and predictability of integer- and fractional-order epidemiological models using physics-informed neural networks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Physics-driven coarse-grained model for biomolecular phase
separation with near-quantitative accuracy. <em>NATCS</em>,
<em>1</em>(11), 732–743. (<a
href="https://doi.org/10.1038/s43588-021-00155-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Various physics- and data-driven sequence-dependent protein coarse-grained models have been developed to study biomolecular phase separation and elucidate the dominant physicochemical driving forces. Here we present Mpipi, a multiscale coarse-grained model that describes almost quantitatively the change in protein critical temperatures as a function of amino acid sequence. The model is parameterized from both atomistic simulations and bioinformatics data and accounts for the dominant role of π–π and hybrid cation–π/π–π interactions and the much stronger attractive contacts established by arginines than lysines. We provide a comprehensive set of benchmarks for Mpipi and seven other residue-level coarse-grained models against experimental radii of gyration and quantitative in vitro phase diagrams, demonstrating that Mpipi predictions agree well with experiments on both fronts. Moreover, Mpipi can account for protein–RNA interactions, correctly predicts the multiphase behavior of a charge-matched poly-arginine/poly-lysine/RNA system, and recapitulates experimental liquid–liquid phase separation trends for sequence mutations on FUS, DDX4 and LAF-1 proteins. Combining bioinformatics data and atomistic simulations, this study develops a sequence-dependent coarse-grained model for biomolecular phase separation. This model achieves a quantitative agreement with experimental observations. Extensive benchmarks exemplify its performance.},
  archive      = {J_NATCS},
  author       = {Joseph, Jerelle A. and Reinhardt, Aleks and Aguirre, Anne and Chew, Pin Yu and Russell, Kieran O. and Espinosa, Jorge R. and Garaizar, Adiran and Collepardo-Guevara, Rosana},
  doi          = {10.1038/s43588-021-00155-3},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {732-743},
  shortjournal = {Nat. Comput. Sci},
  title        = {Physics-driven coarse-grained model for biomolecular phase separation with near-quantitative accuracy},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modeling galvanostatic charge–discharge of nanoporous
supercapacitors. <em>NATCS</em>, <em>1</em>(11), 725–731. (<a
href="https://doi.org/10.1038/s43588-021-00153-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular modeling has been considered indispensable in studying the energy storage of supercapacitors at the atomistic level. The constant potential method (CPM) allows the electric potential to be kept uniform in the electrode, which is essential for a realistic description of the charge repartition and dynamics process in supercapacitors. However, previous CPM studies have been limited to the potentiostatic mode. Although widely adopted in experiments, the galvanostatic mode has rarely been investigated in CPM simulations because of a lack of effective methods. Here we develop a modeling approach to simulating the galvanostatic charge–discharge process of supercapacitors under constant potential. We show that, for nanoporous electrodes, this modeling approach can capture experimentally consistent dynamics in supercapacitors. It can also delineate, at the molecular scale, the hysteresis in ion adsorption–desorption dynamics during charging and discharging. This approach thus enables the further accurate modeling of the physics and electrochemistry in supercapacitor dynamics. The authors propose a molecular modeling approach to simulating the galvanostatic charge–discharge process of supercapacitors under constant-potential conditions. This model can accurately predict supercapacitor dynamics when compared with experimental observations.},
  archive      = {J_NATCS},
  author       = {Zeng, Liang and Wu, Taizheng and Ye, Ting and Mo, Tangming and Qiao, Rui and Feng, Guang},
  doi          = {10.1038/s43588-021-00153-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {725-731},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling galvanostatic charge–discharge of nanoporous supercapacitors},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Compressing atmospheric data into its real information
content. <em>NATCS</em>, <em>1</em>(11), 713–724. (<a
href="https://doi.org/10.1038/s43588-021-00156-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hundreds of petabytes are produced annually at weather and climate forecast centers worldwide. Compression is essential to reduce storage and to facilitate data sharing. Current techniques do not distinguish the real from the false information in data, leaving the level of meaningful precision unassessed. Here we define the bitwise real information content from information theory for the Copernicus Atmospheric Monitoring Service (CAMS). Most variables contain fewer than 7 bits of real information per value and are highly compressible due to spatio-temporal correlation. Rounding bits without real information to zero facilitates lossless compression algorithms and encodes the uncertainty within the data itself. All CAMS data are 17× compressed relative to 64-bit floats, while preserving 99\% of real information. Combined with four-dimensional compression, factors beyond 60× are achieved. A data compression Turing test is proposed to optimize compressibility while minimizing information loss for the end use of weather and climate forecast data. Climate data are often stored at higher precision than is needed. The proposed compression automatically determines the precision from the data’s bitwise real information, removing any false information and leading to a more efficient compression.},
  archive      = {J_NATCS},
  author       = {Klöwer, Milan and Razinger, Miha and Dominguez, Juan J. and Düben, Peter D. and Palmer, Tim N.},
  doi          = {10.1038/s43588-021-00156-2},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {713-724},
  shortjournal = {Nat. Comput. Sci},
  title        = {Compressing atmospheric data into its real information content},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Advancing data compression via noise detection.
<em>NATCS</em>, <em>1</em>(11), 711–712. (<a
href="https://doi.org/10.1038/s43588-021-00167-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Compressing scientific data is essential to save on storage space, but doing so effectively while ensuring that the conclusions from the data are not affected remains a challenging task. A recent paper proposes a new method to identify numerical noise from floating-point atmospheric data, which can lead to a more effective compression.},
  archive      = {J_NATCS},
  author       = {Hammerling, Dorit M. and Baker, Allison H.},
  doi          = {10.1038/s43588-021-00167-z},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {711-712},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancing data compression via noise detection},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Accuracy and fairness go hand in hand. <em>NATCS</em>,
<em>1</em>(11), 710. (<a
href="https://doi.org/10.1038/s43588-021-00163-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00163-3},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {710},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accuracy and fairness go hand in hand},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Accurate short-term precipitation prediction.
<em>NATCS</em>, <em>1</em>(11), 709. (<a
href="https://doi.org/10.1038/s43588-021-00161-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00161-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {709},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accurate short-term precipitation prediction},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Coupling mechanics into membrane electrochemistry.
<em>NATCS</em>, <em>1</em>(11), 708. (<a
href="https://doi.org/10.1038/s43588-021-00164-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00164-2},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {708},
  shortjournal = {Nat. Comput. Sci},
  title        = {Coupling mechanics into membrane electrochemistry},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Modeling historical impacts of fisheries on biomass.
<em>NATCS</em>, <em>1</em>(11), 707. (<a
href="https://doi.org/10.1038/s43588-021-00162-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-021-00162-4},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {707},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling historical impacts of fisheries on biomass},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). For the greatest benefit to humankind. <em>NATCS</em>,
<em>1</em>(11), 705–706. (<a
href="https://doi.org/10.1038/s43588-021-00169-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How has computational science been recognized throughout Nobel history?},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00169-x},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {705-706},
  shortjournal = {Nat. Comput. Sci},
  title        = {For the greatest benefit to humankind},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: The inflammatory bowel disease
transcriptome and metatranscriptome meta-analysis (IBD TaMMA) framework.
<em>NATCS</em>, <em>1</em>(10), 703. (<a
href="https://doi.org/10.1038/s43588-021-00146-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Massimino, Luca and Lamparelli, Luigi Antonio and Houshyar, Yashar and D’Alessio, Silvia and Peyrin-Biroulet, Laurent and Vetrano, Stefania and Danese, Silvio and Ungaro, Federica},
  doi          = {10.1038/s43588-021-00146-4},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {703},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: The inflammatory bowel disease transcriptome and metatranscriptome meta-analysis (IBD TaMMA) framework},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A poisson binomial-based statistical testing framework for
comorbidity discovery across electronic health record datasets.
<em>NATCS</em>, <em>1</em>(10), 694–702. (<a
href="https://doi.org/10.1038/s43588-021-00141-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering the concomitant occurrence of distinct medical conditions in a patient, also known as comorbidities, is a prerequisite for creating patient outcome prediction tools. Current comorbidity discovery applications are designed for small datasets and use stratification to control for confounding variables such as age, sex or ancestry. Stratification lowers false positive rates, but reduces power, as the size of the study cohort is decreased. Here we describe a Poisson binomial-based approach to comorbidity discovery (PBC) designed for big-data applications that circumvents the need for stratification. PBC adjusts for confounding demographic variables on a per-patient basis and models temporal relationships. We benchmark PBC using two datasets to compute comorbidity statistics on 4,623,841 pairs of potentially comorbid medical terms. The results of this computation are provided as a searchable web resource. Compared with current methods, the PBC approach reduces false positive associations while retaining statistical power to discover true comorbidities. A method for comorbidity discovery informed by each patient’s demographic and medical history is introduced. Statistics for 4,623,841 pairs of potentially comorbid medical terms are provided as a searchable web resource.},
  archive      = {J_NATCS},
  author       = {Lemmon, Gordon and Wesolowski, Sergiusz and Henrie, Alex and Tristani-Firouzi, Martin and Yandell, Mark},
  doi          = {10.1038/s43588-021-00141-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {694-702},
  shortjournal = {Nat. Comput. Sci},
  title        = {A poisson binomial-based statistical testing framework for comorbidity discovery across electronic health record datasets},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Neural network-based order parameter for phase transitions
and its applications in high-entropy alloys. <em>NATCS</em>,
<em>1</em>(10), 686–693. (<a
href="https://doi.org/10.1038/s43588-021-00139-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phase transition is one of the most important phenomena in nature and plays a central role in materials design. All phase transitions are characterized by suitable order parameters, including the order–disorder phase transition. However, finding a representative order parameter for complex systems is non-trivial, such as for high-entropy alloys. Given the strength of dimensionality reduction of a variational autoencoder (VAE), we introduce a VAE-based order parameter. We propose that the Manhattan distance in the VAE latent space can serve as a generic order parameter for order–disorder phase transitions. The physical properties of our order parameter are quantitatively interpreted and demonstrated by multiple refractory high-entropy alloys. Using this order parameter, a generally applicable alloy design concept is proposed by mimicking the natural mixing process of elements. Our physically interpretable VAE-based order parameter provides a computational technique for understanding chemical ordering in alloys, which can facilitate the development of rational alloy design strategies. A variational autoencoder-based order parameter is proposed and demonstrated on various high-entropy alloys, providing a computational technique for understanding chemical ordering in alloys. Ultimately, this has the potential to facilitate the development of rational alloy design strategies.},
  archive      = {J_NATCS},
  author       = {Yin, Junqi and Pei, Zongrui and Gao, Michael C.},
  doi          = {10.1038/s43588-021-00139-3},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {686-693},
  shortjournal = {Nat. Comput. Sci},
  title        = {Neural network-based order parameter for phase transitions and its applications in high-entropy alloys},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Vector-based pedestrian navigation in cities.
<em>NATCS</em>, <em>1</em>(10), 678–685. (<a
href="https://doi.org/10.1038/s43588-021-00130-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How do pedestrians choose their paths within city street networks? Researchers have tried to shed light on this matter through strictly controlled experiments, but an ultimate answer based on real-world mobility data is still lacking. Here, we analyze salient features of human path planning through a statistical analysis of a massive dataset of GPS traces, which reveals that (1) people increasingly deviate from the shortest path when the distance between origin and destination increases and (2) chosen paths are statistically different when origin and destination are swapped. We posit that direction to goal is a main driver of path planning and develop a vector-based navigation model; the resulting trajectories, which we have termed pointiest paths, are a statistically better predictor of human paths than a model based on minimizing distance with stochastic effects. Our findings generalize across two major US cities with different street networks, hinting to the fact that vector-based navigation might be a universal property of human path planning. An analysis of GPS pedestrian traces shows that (1) people increasingly deviate from the shortest path when the distance between origin and destination increases and that (2) chosen paths are statistically different when origin and destination are swapped. Ultimately, this can explain the observed human attitude in selecting different paths upon return trips.},
  archive      = {J_NATCS},
  author       = {Bongiorno, Christian and Zhou, Yulun and Kryven, Marta and Theurel, David and Rizzo, Alessandro and Santi, Paolo and Tenenbaum, Joshua and Ratti, Carlo},
  doi          = {10.1038/s43588-021-00130-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {678-685},
  shortjournal = {Nat. Comput. Sci},
  title        = {Vector-based pedestrian navigation in cities},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Population-scale identification of differential adverse
events before and during a pandemic. <em>NATCS</em>, <em>1</em>(10),
666–677. (<a href="https://doi.org/10.1038/s43588-021-00138-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adverse patient safety events, unintended injuries resulting from medical therapy, were associated with 110,000 deaths in the United States in 2019. A nationwide pandemic (such as COVID-19) further challenges the ability of healthcare systems to ensure safe medication use and the pandemic’s effects on safety events remain poorly understood. Here, we investigate drug safety events across demographic groups before and during a pandemic using a dataset of 1,425,371 reports involving 2,821 drugs and 7,761 adverse events. Among 64 adverse events identified by our analyses, we find 54 increased in frequency during the pandemic, despite a 4.4\% decrease in the total number of reports. Out of 53 adverse events with a pre-pandemic gender gap, 33 have seen their gap increase with the pandemic onset. We find that the number of adverse events with an increased reporting ratio is higher in adults (by 16.8\%) than in older patients. Our findings have implications for safe medication use and preventable healthcare inequality in public health emergencies. An algorithmic approach is developed to analyze large-scale patient safety data and remove the confounders of reporting trajectory and drug inference. Such an approach can be effectively used to investigate demographic disparities of drug safety and to identify at-risk patients during a pandemic.},
  archive      = {J_NATCS},
  author       = {Zhang, Xiang and Sumathipala, Marissa and Zitnik, Marinka},
  doi          = {10.1038/s43588-021-00138-4},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {666-677},
  shortjournal = {Nat. Comput. Sci},
  title        = {Population-scale identification of differential adverse events before and during a pandemic},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Climatic signatures in the different COVID-19 pandemic waves
across both hemispheres. <em>NATCS</em>, <em>1</em>(10), 655–665. (<a
href="https://doi.org/10.1038/s43588-021-00136-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The roles of climate and true seasonal signatures in the epidemiology of emergent pathogens, and that of SARS-CoV-2 in particular, remain poorly understood. With a statistical method designed to detect transitory associations, we show, for COVID-19 cases, strong consistent negative effects of both temperature and absolute humidity at large spatial scales. At finer spatial resolutions, we substantiate these connections during the seasonal rise and fall of COVID-19. Strong disease responses are identified in the first two waves, suggesting clear ranges for temperature and absolute humidity that are similar to those formerly described for seasonal influenza. For COVID-19, in all studied regions and pandemic waves, a process-based model that incorporates a temperature-dependent transmission rate outperforms baseline formulations with no driver or a sinusoidal seasonality. Our results, so far, classify COVID-19 as a seasonal low-temperature infection and suggest an important contribution of the airborne pathway in the transmission of SARS-CoV-2, with implications for the control measures we discuss. Using a statistical method for transient correlations, the waxing and waning in levels of population infection by SARS-CoV-2 are shown to respond to temperature and absolute humidity, across geographical locations and for different temporal and spatial resolutions.},
  archive      = {J_NATCS},
  author       = {Fontal, Alejandro and Bouma, Menno J. and San-José, Adrià and López, Leonardo and Pascual, Mercedes and Rodó, Xavier},
  doi          = {10.1038/s43588-021-00136-6},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {655-665},
  shortjournal = {Nat. Comput. Sci},
  title        = {Climatic signatures in the different COVID-19 pandemic waves across both hemispheres},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computational design of quantum defects in two-dimensional
materials. <em>NATCS</em>, <em>1</em>(10), 646–654. (<a
href="https://doi.org/10.1038/s43588-021-00140-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Missing atoms or atom substitutions (point defects) in crystal lattices in two-dimensional (2D) materials are potential hosts for emerging quantum technologies, such as single-photon emitters and spin quantum bits (qubits). First-principles-guided design of quantum defects in 2D materials is paving the way for rational spin qubit discovery. Here we discuss the frontier of first-principles theory development and the challenges in predicting the critical physical properties of point defects in 2D materials for quantum information technology, in particular for optoelectronic and spin-optotronic properties. Strong many-body interactions at reduced dimensionality require advanced electronic structure methods beyond mean-field theory. The great challenges for developing theoretical methods that are appropriate for strongly correlated defect states, as well as general approaches for predicting spin relaxation and the decoherence time of spin defects, are yet to be addressed. Quantum defects in two-dimensional materials offer promises for the next-generation quantum information technology. However, the rational design of these defects faces challenges, and thus, requires the development of advanced theoretical and computational models.},
  archive      = {J_NATCS},
  author       = {Ping, Yuan and Smart, Tyler J.},
  doi          = {10.1038/s43588-021-00140-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {646-654},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational design of quantum defects in two-dimensional materials},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). To v or not to v. <em>NATCS</em>, <em>1</em>(10), 644–645.
(<a href="https://doi.org/10.1038/s43588-021-00143-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finding a parameter that can accurately identify the order–disorder phase transition, especially for complex physical systems with high-dimensional configurational space, is a challenging task. Recent work proposes a machine learning approach to effectively tackle this challenge.},
  archive      = {J_NATCS},
  author       = {van Nieuwenburg, Evert},
  doi          = {10.1038/s43588-021-00143-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {644-645},
  shortjournal = {Nat. Comput. Sci},
  title        = {To v or not to v},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A new computational model for human navigation.
<em>NATCS</em>, <em>1</em>(10), 642–643. (<a
href="https://doi.org/10.1038/s43588-021-00118-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mobile-phone data reveal a cognitive strategy in human navigation and motivate the development of a new route planning model, with potential implications for traffic forecasting and transportation planning.},
  archive      = {J_NATCS},
  author       = {Alessandretti, Laura},
  doi          = {10.1038/s43588-021-00118-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {642-643},
  shortjournal = {Nat. Comput. Sci},
  title        = {A new computational model for human navigation},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Microbial community dynamics revisited. <em>NATCS</em>,
<em>1</em>(10), 640–641. (<a
href="https://doi.org/10.1038/s43588-021-00144-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A framework called EPICS predicts microbial community structures by estimating effective pairwise interactions in an efficient and scalable way.},
  archive      = {J_NATCS},
  author       = {Ji, Boyang and Herrgård, Markus J. and Nielsen, Jens},
  doi          = {10.1038/s43588-021-00144-6},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {640-641},
  shortjournal = {Nat. Comput. Sci},
  title        = {Microbial community dynamics revisited},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Boosting simulation of quantum computers. <em>NATCS</em>,
<em>1</em>(10), 638–639. (<a
href="https://doi.org/10.1038/s43588-021-00145-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An efficient parallelization technique for tensor network contraction, developed by a careful balance between memory requirement and computational time, speeds up classical simulation of quantum computers.},
  archive      = {J_NATCS},
  author       = {Tura, Jordi},
  doi          = {10.1038/s43588-021-00145-5},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {638-639},
  shortjournal = {Nat. Comput. Sci},
  title        = {Boosting simulation of quantum computers},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Predicting reactivity with a push from theory.
<em>NATCS</em>, <em>1</em>(10), 637. (<a
href="https://doi.org/10.1038/s43588-021-00149-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-021-00149-1},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {637},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting reactivity with a push from theory},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). High-performance modeling toolkit. <em>NATCS</em>,
<em>1</em>(10), 636. (<a
href="https://doi.org/10.1038/s43588-021-00150-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00150-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {636},
  shortjournal = {Nat. Comput. Sci},
  title        = {High-performance modeling toolkit},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Physics-educated DL refines phase mapping. <em>NATCS</em>,
<em>1</em>(10), 635. (<a
href="https://doi.org/10.1038/s43588-021-00148-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00148-2},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {635},
  shortjournal = {Nat. Comput. Sci},
  title        = {Physics-educated DL refines phase mapping},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Computationally guided zeolite phase control.
<em>NATCS</em>, <em>1</em>(10), 634. (<a
href="https://doi.org/10.1038/s43588-021-00151-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-021-00151-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {634},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computationally guided zeolite phase control},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Eva to the rescue. <em>NATCS</em>, <em>1</em>(10), 633. (<a
href="https://doi.org/10.1038/s43588-021-00147-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00147-3},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {633},
  shortjournal = {Nat. Comput. Sci},
  title        = {Eva to the rescue},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Democratizing interactive, immersive experiences for science
education with WebXR. <em>NATCS</em>, <em>1</em>(10), 631–632. (<a
href="https://doi.org/10.1038/s43588-021-00142-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rodríguez, Fabio Cortés and Dal Peraro, Matteo and Abriata, Luciano A.},
  doi          = {10.1038/s43588-021-00142-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {631-632},
  shortjournal = {Nat. Comput. Sci},
  title        = {Democratizing interactive, immersive experiences for science education with WebXR},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Moving towards reproducible machine learning.
<em>NATCS</em>, <em>1</em>(10), 629–630. (<a
href="https://doi.org/10.1038/s43588-021-00152-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide some recommendations on how to report machine learning-based research in order to improve transparency and reproducibility.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00152-6},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {629-630},
  shortjournal = {Nat. Comput. Sci},
  title        = {Moving towards reproducible machine learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An efficient and scalable top-down method for predicting
structures of microbial communities. <em>NATCS</em>, <em>1</em>(9),
619–628. (<a href="https://doi.org/10.1038/s43588-021-00131-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern applications involving multispecies microbial communities rely on the ability to predict structures of such communities in defined environments. The structures depend on pairwise and high-order interactions between species. To unravel these interactions, classical bottom-up approaches examine all possible species subcommunities. Such approaches are not scalable as the number of subcommunities grows exponentially with the number of species, n. Here we present a top-down method wherein the number of subcommunities to be examined grows linearly with n, drastically reducing experimental effort. The method uses steady-state data from leave-one-out subcommunities and mathematical modeling to infer effective pairwise interactions and predict community structures. The accuracy of the method increases with n, making it suitable for large communities. We established the method in silico and validated it against a five-species community from literature and an eight-species community cultured in vitro. Our method offers an efficient and scalable tool for predicting microbial community structures. The authors propose EPICS, a method to predict microbial community structures by estimating effective pairwise interactions that subsume high-order interactions between species. EPICS is more efficient and applicable to larger communities than current approaches.},
  archive      = {J_NATCS},
  author       = {Ansari, Aamir Faisal and Reddy, Yugandhar B. S. and Raut, Janhavi and Dixit, Narendra M.},
  doi          = {10.1038/s43588-021-00131-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {619-628},
  shortjournal = {Nat. Comput. Sci},
  title        = {An efficient and scalable top-down method for predicting structures of microbial communities},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Explainable neural networks that simulate reasoning.
<em>NATCS</em>, <em>1</em>(9), 607–618. (<a
href="https://doi.org/10.1038/s43588-021-00132-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The success of deep neural networks suggests that cognition may emerge from indecipherable patterns of distributed neural activity. Yet these networks are pattern-matching black boxes that cannot simulate higher cognitive functions and lack numerous neurobiological features. Accordingly, they are currently insufficient computational models for understanding neural information processing. Here, we show how neural circuits can directly encode cognitive processes via simple neurobiological principles. To illustrate, we implemented this model in a non-gradient-based machine learning algorithm to train deep neural networks called essence neural networks (ENNs). Neural information processing in ENNs is intrinsically explainable, even on benchmark computer vision tasks. ENNs can also simulate higher cognitive functions such as deliberation, symbolic reasoning and out-of-distribution generalization. ENNs display network properties associated with the brain, such as modularity, distributed and localist firing, and adversarial robustness. ENNs establish a broad computational framework to decipher the neural basis of cognition and pursue artificial general intelligence. The authors demonstrate how neural systems can encode cognitive functions, and use the proposed model to train robust, scalable deep neural networks that are explainable and capable of symbolic reasoning and domain generalization.},
  archive      = {J_NATCS},
  author       = {Blazek, Paul J. and Lin, Milo M.},
  doi          = {10.1038/s43588-021-00132-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {607-618},
  shortjournal = {Nat. Comput. Sci},
  title        = {Explainable neural networks that simulate reasoning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Detecting microstructural deviations in individuals with
deep diffusion MRI tractometry. <em>NATCS</em>, <em>1</em>(9), 598–606.
(<a href="https://doi.org/10.1038/s43588-021-00126-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most diffusion magnetic resonance imaging studies of disease rely on statistical comparisons between large groups of patients and healthy participants to infer altered tissue states in the brain; however, clinical heterogeneity can greatly challenge their discriminative power. There is currently an unmet need to move away from the current approach of group-wise comparisons to methods with the sensitivity to detect altered tissue states at the individual level. This would ultimately enable the early detection and interpretation of microstructural abnormalities in individual patients, an important step towards personalized medicine in translational imaging. To this end, Detect was developed to advance diffusion magnetic resonance imaging tractometry towards single-patient analysis. By operating on the manifold of white-matter pathways and learning normative microstructural features, our framework captures idiosyncrasies in patterns along white-matter pathways. Our approach paves the way from traditional group-based comparisons to true personalized radiology, taking microstructural imaging from the bench to the bedside. The authors propose Detect, a browser-based anomaly detection framework for diffusion magnetic resonance imaging tractometry data. The tool leverages normative microstructural brain features derived from healthy participants using deep autoencoders to detect anomalies at the individual level.},
  archive      = {J_NATCS},
  author       = {Chamberland, Maxime and Genc, Sila and Tax, Chantal M. W. and Shastin, Dmitri and Koller, Kristin and Raven, Erika P. and Cunningham, Adam and Doherty, Joanne and van den Bree, Marianne B. M. and Parker, Greg D. and Hamandi, Khalid and Gray, William P. and Jones, Derek K.},
  doi          = {10.1038/s43588-021-00126-8},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {598-606},
  shortjournal = {Nat. Comput. Sci},
  title        = {Detecting microstructural deviations in individuals with deep diffusion MRI tractometry},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Insights into population behavior during the COVID-19
pandemic from cell phone mobility data and manifold learning.
<em>NATCS</em>, <em>1</em>(9), 588–597. (<a
href="https://doi.org/10.1038/s43588-021-00125-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the complex interplay between human behavior, disease transmission and non-pharmaceutical interventions during the COVID-19 pandemic could provide valuable insights with which to focus future public health efforts. Cell phone mobility data offer a modern measurement instrument to investigate human mobility and behavior at an unprecedented scale. We investigate aggregated and anonymized mobility data, which measure how populations at the census-block-group geographic scale stayed at home in California, Georgia, Texas and Washington from the beginning of the pandemic. Using manifold learning techniques, we show that a low-dimensional embedding enables the identification of patterns of mobility behavior that align with stay-at-home orders, correlate with socioeconomic factors, cluster geographically, reveal subpopulations that probably migrated out of urban areas and, importantly, link to COVID-19 case counts. The analysis and approach provide local epidemiologists a framework for interpreting mobility data and behavior to inform policy makers’ decision-making aimed at curbing the spread of COVID-19. Combining human mobility data and nonlinear mathematical analysis techniques, this study offers insights into the interplay between human behavior, disease transmission and non-pharmaceutical interventions during the COVID-19 pandemic.},
  archive      = {J_NATCS},
  author       = {Levin, Roman and Chao, Dennis L. and Wenger, Edward A. and Proctor, Joshua L.},
  doi          = {10.1038/s43588-021-00125-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {588-597},
  shortjournal = {Nat. Comput. Sci},
  title        = {Insights into population behavior during the COVID-19 pandemic from cell phone mobility data and manifold learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Efficient parallelization of tensor network contraction for
simulating quantum computation. <em>NATCS</em>, <em>1</em>(9), 578–587.
(<a href="https://doi.org/10.1038/s43588-021-00119-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We develop an algorithmic framework for contracting tensor networks and demonstrate its power by classically simulating quantum computation of sizes previously deemed out of reach. Our main contribution, index slicing, is a method that efficiently parallelizes the contraction by breaking it down into much smaller and identically structured subtasks, which can then be executed in parallel without dependencies. We benchmark our algorithm on a class of random quantum circuits, achieving greater than 105 times acceleration over the original estimate of the simulation cost. We then demonstrate applications of the simulation framework for aiding the development of quantum algorithms and quantum error correction. As tensor networks are widely used in computational science, our simulation framework may find further applications. An efficient method for parallelizing the contraction of tensor networks pushes the boundaries for the classical simulation of quantum computation, and aids the development of quantum algorithms and hardware.},
  archive      = {J_NATCS},
  author       = {Huang, Cupjin and Zhang, Fang and Newman, Michael and Ni, Xiaotong and Ding, Dawei and Cai, Junjie and Gao, Xun and Wang, Tenghui and Wu, Feng and Zhang, Gengyan and Ku, Hsiang-Sheng and Tian, Zhengxiong and Wu, Junyin and Xu, Haihong and Yu, Huanjun and Yuan, Bo and Szegedy, Mario and Shi, Yaoyun and Zhao, Hui-Hai and Deng, Chunqing and Chen, Jianxin},
  doi          = {10.1038/s43588-021-00119-7},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {578-587},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient parallelization of tensor network contraction for simulating quantum computation},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fast and accurate bootstrap confidence limits on
genome-scale phylogenies using little bootstraps. <em>NATCS</em>,
<em>1</em>(9), 573–577. (<a
href="https://doi.org/10.1038/s43588-021-00129-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Felsenstein’s bootstrap approach is widely used to assess confidence in species relationships inferred from multiple sequence alignments. It resamples sites randomly with replacement to build alignment replicates of the same size as the original alignment and infers a phylogeny from each replicate dataset. The proportion of phylogenies recovering the same grouping of species is its bootstrap confidence limit. However, standard bootstrap imposes a high computational burden in applications involving long sequence alignments. Here, we introduce the bag of little bootstraps approach to phylogenetics, bootstrapping only a few little samples, each containing a small subset of sites. We report that the median-bagging of bootstrap confidence limits from little samples produces confidence in inferred species relationships similar to standard bootstrap but in a fraction of the computational time and memory. Therefore, the little bootstraps approach can potentially enhance the rigor, efficiency and parallelization of big data phylogenomic analyses. The authors show that accurate bootstrap confidence limits on inferred evolutionary relationships of species can be estimated by bootstrapping a collection of little samples of very long sequence alignments. Little bootstraps take a fraction of computer time and memory compared to the standard bootstrap, enabling big data analytics on personal computers.},
  archive      = {J_NATCS},
  author       = {Sharma, Sudip and Kumar, Sudhir},
  doi          = {10.1038/s43588-021-00129-5},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {573-577},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fast and accurate bootstrap confidence limits on genome-scale phylogenies using little bootstraps},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A handy meta-analysis tool for IBD research. <em>NATCS</em>,
<em>1</em>(9), 571–572. (<a
href="https://doi.org/10.1038/s43588-021-00124-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent work introduces a powerful new web tool that enables a faster and statistically more reliable data mining of transcriptomics and metatranscriptomics for inflammatory bowel disease (IBD) research.},
  archive      = {J_NATCS},
  author       = {Modos, Dezso and Thomas, John P. and Korcsmaros, Tamas},
  doi          = {10.1038/s43588-021-00124-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {571-572},
  shortjournal = {Nat. Comput. Sci},
  title        = {A handy meta-analysis tool for IBD research},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Detect-ing brain anomalies with autoencoders.
<em>NATCS</em>, <em>1</em>(9), 569–570. (<a
href="https://doi.org/10.1038/s43588-021-00128-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A framework called Detect is proposed to detect subtle effects of brain disorders, making it possible to delineate anomalous brain connections within specific individuals.},
  archive      = {J_NATCS},
  author       = {Rokem, Ariel},
  doi          = {10.1038/s43588-021-00128-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {569-570},
  shortjournal = {Nat. Comput. Sci},
  title        = {Detect-ing brain anomalies with autoencoders},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mobility data as a proxy for epidemic measures.
<em>NATCS</em>, <em>1</em>(9), 567–568. (<a
href="https://doi.org/10.1038/s43588-021-00127-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study uses longitudinal mobility data to identify how individuals behave at different stages of the COVID pandemic, elucidating benefits and challenges of using this type of data for decision-making by epidemiologists and policy-makers.},
  archive      = {J_NATCS},
  author       = {Kishore, Nishant},
  doi          = {10.1038/s43588-021-00127-7},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {567-568},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mobility data as a proxy for epidemic measures},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). Shining a light on mutational signatures. <em>NATCS</em>,
<em>1</em>(9), 566. (<a
href="https://doi.org/10.1038/s43588-021-00135-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00135-7},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {566},
  shortjournal = {Nat. Comput. Sci},
  title        = {Shining a light on mutational signatures},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). Reducing qubit budget in quantum machine learning.
<em>NATCS</em>, <em>1</em>(9), 565. (<a
href="https://doi.org/10.1038/s43588-021-00133-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00133-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {565},
  shortjournal = {Nat. Comput. Sci},
  title        = {Reducing qubit budget in quantum machine learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Deep learning boosts RNA prediction. <em>NATCS</em>,
<em>1</em>(9), 564. (<a
href="https://doi.org/10.1038/s43588-021-00134-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00134-8},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {564},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep learning boosts RNA prediction},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Meet the editors. <em>NATCS</em>, <em>1</em>(9), 563. (<a
href="https://doi.org/10.1038/s43588-021-00137-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This month at Nature Computational Science, we welcome a new member to our team. We would like to take this opportunity to briefly describe who we are and what our main responsibilities entail.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00137-5},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {563},
  shortjournal = {Nat. Comput. Sci},
  title        = {Meet the editors},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High performance computing framework for tera-scale database
search of mass spectrometry data. <em>NATCS</em>, <em>1</em>(8),
550–561. (<a href="https://doi.org/10.1038/s43588-021-00113-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Database peptide search algorithms deduce peptides from mass spectrometry data. There has been substantial effort in improving their computational efficiency to achieve larger and more complex systems biology studies. However, modern serial and high-performance computing (HPC) algorithms exhibit suboptimal performance mainly due to their ineffective parallel designs (low resource utilization) and high overhead costs. We present an HPC framework, called HiCOPS, for efficient acceleration of the database peptide search algorithms on distributed-memory supercomputers. HiCOPS provides, on average, more than tenfold improvement in speed and superior parallel performance over several existing HPC database search software. We also formulate a mathematical model for performance analysis and optimization, and report near-optimal results for several key metrics including strong-scale efficiency, hardware utilization, load-balance, inter-process communication and I/O overheads. The core parallel design, techniques and optimizations presented in HiCOPS are search-algorithm-independent and can be extended to efficiently accelerate the existing and future algorithms and software. The study introduces the design and implementation of a parallel computational framework, called HiCOPS, for efficient acceleration of large-scale database peptide search workloads on supercomputers.},
  archive      = {J_NATCS},
  author       = {Haseeb, Muhammad and Saeed, Fahad},
  doi          = {10.1038/s43588-021-00113-z},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {550-561},
  shortjournal = {Nat. Comput. Sci},
  title        = {High performance computing framework for tera-scale database search of mass spectrometry data},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Scalable optical learning operator. <em>NATCS</em>,
<em>1</em>(8), 542–549. (<a
href="https://doi.org/10.1038/s43588-021-00112-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Today’s heavy machine learning tasks are fueled by large datasets. Computing is performed with power-hungry processors whose performance is ultimately limited by the data transfer to and from memory. Optics is a powerful means of communicating and processing information, and there is currently intense interest in optical information processing for realizing high-speed computations. Here we present and experimentally demonstrate an optical computing framework called scalable optical learning operator, which is based on spatiotemporal effects in multimode fibers for a range of learning tasks including classifying COVID-19 X-ray lung images, speech recognition and predicting age from images of faces. The presented framework addresses the energy scaling problem of existing systems without compromising speed. We leverage simultaneous, linear and nonlinear interaction of spatial modes as a computation engine. We numerically and experimentally show the ability of the method to execute several different tasks with accuracy comparable with a digital implementation. Optical computing promises high-speed computations but presents challenges in nonlinear information processing. This Article demonstrates a scalable and energy-efficient nonlinear optical-computing framework that can perform machine learning tasks.},
  archive      = {J_NATCS},
  author       = {Teğin, Uğur and Yıldırım, Mustafa and Oğuz, İlker and Moser, Christophe and Psaltis, Demetri},
  doi          = {10.1038/s43588-021-00112-0},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {542-549},
  shortjournal = {Nat. Comput. Sci},
  title        = {Scalable optical learning operator},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A machine learning-based multiscale model to predict bone
formation in scaffolds. <em>NATCS</em>, <em>1</em>(8), 532–541. (<a
href="https://doi.org/10.1038/s43588-021-00115-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational modeling methods combined with non-invasive imaging technologies have exhibited great potential and unique opportunities to model new bone formation in scaffold tissue engineering, offering an effective alternate and viable complement to laborious and time-consuming in vivo studies. However, existing numerical approaches are still highly demanding computationally in such multiscale problems. To tackle this challenge, we propose a machine learning (ML)-based approach to predict bone ingrowth outcomes in bulk tissue scaffolds. The proposed in silico procedure is developed by correlating with a dedicated longitudinal (12-month) animal study on scaffold treatment of a major segmental defect in sheep tibia. Comparison of the ML-based time-dependent prediction of bone ingrowth with the conventional multilevel finite element (FE2) model demonstrates satisfactory accuracy and efficiency. The ML-based modeling approach provides an effective means for predicting in vivo bone tissue regeneration in a subject-specific scaffolding system. The study develops a machine learning approach for predicting bone regeneration in an additively manufactured bioceramic scaffold, which is correlated with an in vivo sheep model, exhibiting effectiveness for solving such a multiscale modeling problem.},
  archive      = {J_NATCS},
  author       = {Wu, Chi and Entezari, Ali and Zheng, Keke and Fang, Jianguang and Zreiqat, Hala and Steven, Grant P. and Swain, Michael V. and Li, Qing},
  doi          = {10.1038/s43588-021-00115-x},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {532-541},
  shortjournal = {Nat. Comput. Sci},
  title        = {A machine learning-based multiscale model to predict bone formation in scaffolds},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Optimal national prioritization policies for hospital care
during the SARS-CoV-2 pandemic. <em>NATCS</em>, <em>1</em>(8), 521–531.
(<a href="https://doi.org/10.1038/s43588-021-00111-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In response to unprecedented surges in the demand for hospital care during the SARS-CoV-2 pandemic, health systems have prioritized patients with COVID-19 to life-saving hospital care to the detriment of other patients. In contrast to these ad hoc policies, we develop a linear programming framework to optimally schedule elective procedures and allocate hospital beds among all planned and emergency patients to minimize years of life lost. Leveraging a large dataset of administrative patient medical records, we apply our framework to the National Health Service in England and show that an extra 50,750–5,891,608 years of life can be gained compared with prioritization policies that reflect those implemented during the pandemic. Notable health gains are observed for neoplasms, diseases of the digestive system, and injuries and poisoning. Our open-source framework provides a computationally efficient approximation of a large-scale discrete optimization problem that can be applied globally to support national-level care prioritization policies. Countries are using hospital admission policies that prioritize patients with COVID-19 during the pandemic. The authors propose an alternative open-source framework to optimally schedule hospital care for all diseases and patients that can save life years overall.},
  archive      = {J_NATCS},
  author       = {D’Aeth, Josh C. and Ghosal, Shubhechyya and Grimm, Fiona and Haw, David and Koca, Esma and Lau, Krystal and Moret, Stefano and Rizmie, Dheeya and Deeny, Sarah R. and Perez-Guzman, Pablo N. and Ferguson, Neil and Hauck, Katharina and Smith, Peter C. and Forchini, Giovanni and Wiesemann, Wolfram and Miraldo, Marisa},
  doi          = {10.1038/s43588-021-00111-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {521-531},
  shortjournal = {Nat. Comput. Sci},
  title        = {Optimal national prioritization policies for hospital care during the SARS-CoV-2 pandemic},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). From complex datasets to predictive models of embryonic
development. <em>NATCS</em>, <em>1</em>(8), 516–520. (<a
href="https://doi.org/10.1038/s43588-021-00110-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern studies of embryogenesis are increasingly quantitative, powered by rapid advances in imaging, sequencing and genome manipulation technologies. Deriving mechanistic insights from the complex datasets generated by these new tools requires systematic approaches for data-driven analysis of the underlying developmental processes. Here, we use data from our work on signal-dependent gene repression in the Drosophila embryo to illustrate how computational models can compactly summarize quantitative results of live imaging, chromatin immunoprecipitation and optogenetic perturbation experiments. The presented computational approach is ideally suited for integrating rapidly accumulating quantitative data and for guiding future studies of embryogenesis. A computational approach to predictive modeling of embryonic development is presented and validated by integrating imaging, biochemical and genetic studies of tissue patterning in Drosophila embryogenesis.},
  archive      = {J_NATCS},
  author       = {Dutta, Sayantan and Patel, Aleena L. and Keenan, Shannon E. and Shvartsman, Stanislav Y.},
  doi          = {10.1038/s43588-021-00110-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {516-520},
  shortjournal = {Nat. Comput. Sci},
  title        = {From complex datasets to predictive models of embryonic development},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). The inflammatory bowel disease transcriptome and
metatranscriptome meta-analysis (IBD TaMMA) framework. <em>NATCS</em>,
<em>1</em>(8), 511–515. (<a
href="https://doi.org/10.1038/s43588-021-00114-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inflammatory bowel disease (IBD) is a class of chronic disorders whose etiogenesis is still unknown. Despite the high number of IBD-related omics studies, the RNA-sequencing data produced results that are hard to compare because of the experimental variability and different data analysis approaches. We here introduce the IBD Transcriptome and Metatranscriptome Meta-Analysis (TaMMA) framework, a comprehensive survey of publicly available IBD RNA-sequencing datasets. IBD TaMMA is an open-source platform where scientists can explore simultaneously the freely available IBD-associated transcriptomics and microbial profiles thanks to its interactive interface, resulting in a useful tool to the IBD community. Massimino et al. propose the Inflammatory Bowel Disease Transcriptome and Metatranscriptome Meta-Analysis (IBD TaMMA) framework, an open-source platform for expediting the investigation of IBD-specific transcriptomics and metatranscriptomics signatures.},
  archive      = {J_NATCS},
  author       = {Massimino, Luca and Lamparelli, Luigi Antonio and Houshyar, Yashar and D’Alessio, Silvia and Peyrin-Biroulet, Laurent and Vetrano, Stefania and Danese, Silvio and Ungaro, Federica},
  doi          = {10.1038/s43588-021-00114-y},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {511-515},
  shortjournal = {Nat. Comput. Sci},
  title        = {The inflammatory bowel disease transcriptome and metatranscriptome meta-analysis (IBD TaMMA) framework},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Predicting bone regeneration from machine learning.
<em>NATCS</em>, <em>1</em>(8), 509–510. (<a
href="https://doi.org/10.1038/s43588-021-00116-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modeling of the multiscale dynamics of new bone formation in tissue scaffolds is still challenging due to the computational complexity in solving the mechanics–material–biology interactions. Recent work proposes a machine learning approach to address this challenge.},
  archive      = {J_NATCS},
  author       = {Li, Zhiyong},
  doi          = {10.1038/s43588-021-00116-w},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {509-510},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting bone regeneration from machine learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Embryonic development across space and time. <em>NATCS</em>,
<em>1</em>(8), 507–508. (<a
href="https://doi.org/10.1038/s43588-021-00117-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A model for Drosophila embryonic development is presented by integrating several types of experimental data spanning over several layers of space and time.},
  archive      = {J_NATCS},
  author       = {Waymack, Rachel and Wunderlich, Zeba},
  doi          = {10.1038/s43588-021-00117-9},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {507-508},
  shortjournal = {Nat. Comput. Sci},
  title        = {Embryonic development across space and time},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021n). Simulating synthesis of 3D network polymers.
<em>NATCS</em>, <em>1</em>(8), 506. (<a
href="https://doi.org/10.1038/s43588-021-00122-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00122-y},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {506},
  shortjournal = {Nat. Comput. Sci},
  title        = {Simulating synthesis of 3D network polymers},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). A pandemic’s trajectory using PCR testing. <em>NATCS</em>,
<em>1</em>(8), 505. (<a
href="https://doi.org/10.1038/s43588-021-00121-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00121-z},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {505},
  shortjournal = {Nat. Comput. Sci},
  title        = {A pandemic’s trajectory using PCR testing},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Fluid dynamic behavior of deep-sea sponges. <em>NATCS</em>,
<em>1</em>(8), 504. (<a
href="https://doi.org/10.1038/s43588-021-00120-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00120-0},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {504},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fluid dynamic behavior of deep-sea sponges},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fostering multidisciplinary collaborations. <em>NATCS</em>,
<em>1</em>(8), 503. (<a
href="https://doi.org/10.1038/s43588-021-00123-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We discuss the role of computational science as a multidisciplinary field and our editorial practices to promote communication and research across different disciplines.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00123-x},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {503},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fostering multidisciplinary collaborations},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Maximally localized dynamical quantum
embedding for solving many-body correlated systems. <em>NATCS</em>,
<em>1</em>(7), 502. (<a
href="https://doi.org/10.1038/s43588-021-00105-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Lupo, Carla and Jamet, François and Tse, Wai Hei Terence and Rungger, Ivan and Weber, Cedric},
  doi          = {10.1038/s43588-021-00105-z},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {502},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Maximally localized dynamical quantum embedding for solving many-body correlated systems},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modeling gene regulatory networks using neural network
architectures. <em>NATCS</em>, <em>1</em>(7), 491–501. (<a
href="https://doi.org/10.1038/s43588-021-00099-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene regulatory networks (GRNs) encode the complex molecular interactions that govern cell identity. Here we propose DeepSEM, a deep generative model that can jointly infer GRNs and biologically meaningful representation of single-cell RNA sequencing (scRNA-seq) data. In particular, we developed a neural network version of the structural equation model (SEM) to explicitly model the regulatory relationships among genes. Benchmark results show that DeepSEM achieves comparable or better performance on a variety of single-cell computational tasks, such as GRN inference, scRNA-seq data visualization, clustering and simulation, compared with the state-of-the-art methods. In addition, the gene regulations predicted by DeepSEM on cell-type marker genes in the mouse cortex can be validated by epigenetic data, which further demonstrates the accuracy and efficiency of our method. DeepSEM can provide a useful and powerful tool to analyze scRNA-seq data and infer a GRN. The authors propose a deep learning model that analyzes single-cell RNA sequencing (scRNA-seq) data by explicitly modeling gene regulatory networks (GRNs), outperforming the state-of-art methods on various tasks, including GRN inference, scRNA-seq analysis and simulation.},
  archive      = {J_NATCS},
  author       = {Shu, Hantao and Zhou, Jingtian and Lian, Qiuyu and Li, Han and Zhao, Dan and Zeng, Jianyang and Ma, Jianzhu},
  doi          = {10.1038/s43588-021-00099-8},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {491-501},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling gene regulatory networks using neural network architectures},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Simultaneously improving reaction coverage and computational
cost in automated reaction prediction tasks. <em>NATCS</em>,
<em>1</em>(7), 479–490. (<a
href="https://doi.org/10.1038/s43588-021-00101-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Automated reaction prediction has the potential to elucidate complex reaction networks for applications ranging from combustion to materials degradation, but computational cost and inconsistent reaction coverage are still obstacles to exploring deep reaction networks. Here we show that cost can be reduced and reaction coverage can be increased simultaneously by relatively straightforward modifications of the reaction enumeration, geometry initialization and transition state convergence algorithms that are common to many prediction methodologies. These components are implemented in the context of yet another reaction program (YARP), our reaction prediction package with which we report reaction discovery benchmarks for organic single-step reactions, thermal degradation of a γ-ketohydroperoxide, and competing ring-closures in a large organic molecule. Compared with recent benchmarks, YARP (re)discovers both established and unreported reaction pathways and products while simultaneously reducing the cost of reaction characterization by nearly 100-fold and increasing convergence of transition states. This combination of ultra-low cost and high reaction coverage creates opportunities to explore the reactivity of larger systems and more complex reaction networks for applications such as chemical degradation, where computational cost is a bottleneck. This work demonstrates that large gains still exist in accelerating and improving the coverage of reaction prediction algorithms. These advances create opportunities for computationally exploring deeper and broader reaction networks.},
  archive      = {J_NATCS},
  author       = {Zhao, Qiyuan and Savoie, Brett M.},
  doi          = {10.1038/s43588-021-00101-3},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {479-490},
  shortjournal = {Nat. Comput. Sci},
  title        = {Simultaneously improving reaction coverage and computational cost in automated reaction prediction tasks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Evidence-based recommender system for high-entropy alloys.
<em>NATCS</em>, <em>1</em>(7), 470–478. (<a
href="https://doi.org/10.1038/s43588-021-00097-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing data-driven approaches for exploring high-entropy alloys (HEAs) face three challenges: numerous element-combination candidates, designing appropriate descriptors, and limited and biased existing data. To overcome these issues, here we show the development of an evidence-based material recommender system (ERS) that adopts Dempster–Shafer theory, a general framework for reasoning with uncertainty. Herein, without using material descriptors, we model, collect and combine pieces of evidence from data about the HEA phase existence of alloys. To evaluate the ERS, we compared its HEA-recommendation capability with those of matrix-factorization- and supervised-learning-based recommender systems on four widely known datasets of up-to-five-component alloys. The k-fold cross-validation on the datasets suggests that the ERS outperforms all competitors. Furthermore, the ERS shows good extrapolation capabilities in recommending quaternary and quinary HEAs. We experimentally validated the most strongly recommended Fe–Co-based magnetic HEA (namely, FeCoMnNi) and confirmed that its thin film shows a body-centered cubic structure. An evidence-based approach for dealing with insufficient, conflicting and biased materials data is proposed for recommending high-entropy alloys, showing good capabilities for extrapolating the number of components.},
  archive      = {J_NATCS},
  author       = {Ha, Minh-Quyet and Nguyen, Duong-Nguyen and Nguyen, Viet-Cuong and Nagata, Takahiro and Chikyow, Toyohiro and Kino, Hiori and Miyake, Takashi and Denœux, Thierry and Huynh, Van-Nam and Dam, Hieu-Chi},
  doi          = {10.1038/s43588-021-00097-w},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {470-478},
  shortjournal = {Nat. Comput. Sci},
  title        = {Evidence-based recommender system for high-entropy alloys},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fast and effective protein model refinement using deep graph
neural networks. <em>NATCS</em>, <em>1</em>(7), 462–469. (<a
href="https://doi.org/10.1038/s43588-021-00098-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein model refinement is the last step applied to improve the quality of a predicted protein model. Currently, the most successful refinement methods rely on extensive conformational sampling and thus take hours or days to refine even a single protein model. Here, we propose a fast and effective model refinement method that applies graph neural networks (GNNs) to predict a refined inter-atom distance probability distribution from an initial model and then rebuilds three-dimensional models from the predicted distance distribution. Tested on the Critical Assessment of Structure Prediction refinement targets, our method has an accuracy that is comparable to those of two leading human groups (FEIG and BAKER), but runs substantially faster. Our method may refine one protein model within ~11 min on one CPU, whereas BAKER needs ~30 h on 60 CPUs and FEIG needs ~16 h on one GPU. Finally, our study shows that GNN outperforms ResNet (convolutional residual neural networks) for model refinement when very limited conformational sampling is allowed. Deep graph neural networks can refine a predicted protein model efficiently with less computing resources. The accuracy is comparable to that of the leading physics-based methods that rely on time-consuming conformation sampling.},
  archive      = {J_NATCS},
  author       = {Jing, Xiaoyang and Xu, Jinbo},
  doi          = {10.1038/s43588-021-00098-9},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {462-469},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fast and effective protein model refinement using deep graph neural networks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A versatile model for single-cell data analysis.
<em>NATCS</em>, <em>1</em>(7), 460–461. (<a
href="https://doi.org/10.1038/s43588-021-00103-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Making sense of single-cell data requires various computational efforts such as clustering, visualization and gene regulatory network inference, often addressed by different methods. DeepSEM provides an all-in-one solution.},
  archive      = {J_NATCS},
  author       = {Ding, Jun},
  doi          = {10.1038/s43588-021-00103-1},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {460-461},
  shortjournal = {Nat. Comput. Sci},
  title        = {A versatile model for single-cell data analysis},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). From evidence to new high-entropy alloys. <em>NATCS</em>,
<em>1</em>(7), 458–459. (<a
href="https://doi.org/10.1038/s43588-021-00100-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A data-driven approach uses pieces of evidence from existing alloy databases to effectively recommend high-entropy alloys candidates.},
  archive      = {J_NATCS},
  author       = {Zhuang, Houlong},
  doi          = {10.1038/s43588-021-00100-4},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {458-459},
  shortjournal = {Nat. Comput. Sci},
  title        = {From evidence to new high-entropy alloys},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Rapid protein model refinement by deep learning.
<em>NATCS</em>, <em>1</em>(7), 456–457. (<a
href="https://doi.org/10.1038/s43588-021-00104-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A graph-neural-network-based framework is proposed for the refinement of protein structure models, substantially improving the efficacy and efficiency of refining protein models when compared with the state-of-the-art approaches.},
  archive      = {J_NATCS},
  author       = {Abdin, Osama and Kim, Philip M.},
  doi          = {10.1038/s43588-021-00104-0},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {456-457},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rapid protein model refinement by deep learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). COVID-19 prognosis from a longitudinal dataset.
<em>NATCS</em>, <em>1</em>(7), 455. (<a
href="https://doi.org/10.1038/s43588-021-00106-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00106-y},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {455},
  shortjournal = {Nat. Comput. Sci},
  title        = {COVID-19 prognosis from a longitudinal dataset},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Detection of war destruction from satellite images.
<em>NATCS</em>, <em>1</em>(7), 454. (<a
href="https://doi.org/10.1038/s43588-021-00108-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00108-w},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {454},
  shortjournal = {Nat. Comput. Sci},
  title        = {Detection of war destruction from satellite images},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Improving chemical accuracy with neural networks.
<em>NATCS</em>, <em>1</em>(7), 453. (<a
href="https://doi.org/10.1038/s43588-021-00107-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00107-x},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {453},
  shortjournal = {Nat. Comput. Sci},
  title        = {Improving chemical accuracy with neural networks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computational challenges in numerical relativity in the
gravitational-wave era. <em>NATCS</em>, <em>1</em>(7), 450–452. (<a
href="https://doi.org/10.1038/s43588-021-00102-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gravitational-wave discoveries have ignited a new era of astronomy. Numerical relativity plays a crucial role in modeling gravitational-wave sources for current and next-generation observatories, but it doesn’t come without computational challenges.},
  archive      = {J_NATCS},
  author       = {Lovelace, Geoffrey},
  doi          = {10.1038/s43588-021-00102-2},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {450-452},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational challenges in numerical relativity in the gravitational-wave era},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). But is the code (re)usable? <em>NATCS</em>, <em>1</em>(7),
449. (<a href="https://doi.org/10.1038/s43588-021-00109-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While it is crucial to guarantee the reproducibility of the results reported in a paper, let us also not forget about the importance of making research artifacts reusable for the scientific community.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00109-9},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {449},
  shortjournal = {Nat. Comput. Sci},
  title        = {But is the code (re)usable?},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Publisher correction: Single-cell manifold-preserving
feature selection for detecting rare cell populations. <em>NATCS</em>,
<em>1</em>(6), 448. (<a
href="https://doi.org/10.1038/s43588-021-00091-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Liang, Shaoheng and Mohanty, Vakul and Dou, Jinzhuang and Miao, Qi and Huang, Yuefan and Müftüoğlu, Muharrem and Ding, Li and Peng, Weiyi and Chen, Ken},
  doi          = {10.1038/s43588-021-00091-2},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {448},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Single-cell manifold-preserving feature selection for detecting rare cell populations},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Balancing efficient analysis and storage of quantitative
genomics data with the d4 format and d4tools. <em>NATCS</em>,
<em>1</em>(6), 441–447. (<a
href="https://doi.org/10.1038/s43588-021-00085-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern DNA sequencing is used as a readout for diverse assays, with the count of aligned sequences (read depth) representing the quantitative signal for each underlying cellular phenomena. Existing data formats for quantitative genomics assays are, however, limited in either the analysis speeds they enable, the disk space they require or both. We have developed the dense depth data dump (D4) format and tool suite, with the goal of balancing improved analysis speeds with file size. The D4 format is adaptive in that it profiles a random sample of aligned sequence depth from the input sequence file to determine an optimal encoding that enables fast data access. We demonstrate that the D4 format offers substantial speed improvements over existing formats for random access, aggregation and summarization, while also achieving better or comparable file sizes. This performance enables scalable downstream analyses that would be otherwise difficult. The D4 format for genomics datasets employs an adaptive encoding strategy based on the relatively low variance of sequence coverage. This encoding improves the speed at which these datasets can be queried for biological inquiry, while also achieving better or comparable file sizes than existing formats.},
  archive      = {J_NATCS},
  author       = {Hou, Hao and Pedersen, Brent and Quinlan, Aaron},
  doi          = {10.1038/s43588-021-00085-0},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {441-447},
  shortjournal = {Nat. Comput. Sci},
  title        = {Balancing efficient analysis and storage of quantitative genomics data with the d4 format and d4tools},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A survival model generalized to regression learning
algorithms. <em>NATCS</em>, <em>1</em>(6), 433–440. (<a
href="https://doi.org/10.1038/s43588-021-00083-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Survival prediction is an important problem that is encountered widely in industry and medicine. Despite the explosion of artificial intelligence technologies, no uniformed method allows the application of any type of regression learning algorithm to a survival prediction problem. Here, we present a statistical modeling method that is generalized to all types of regression learning algorithm, including deep learning. We present its empirical advantage when it is applied to traditional survival problems. We demonstrate its expanded applications in different types of regression learning algorithm, such as gradient boosted trees, convolutional neural networks and recurrent neural networks. Additionally, we demonstrate its application in clinical informatic data, pathological images and the hardware industry. We expect that this algorithm will be widely applicable for diverse types of survival data, including discrete data types and those suitable for deep learning such as those with time or spatial continuity. A statistical modeling method is proposed to generalize right censored data to a standard regression problem, thus making it possible to apply regression learning algorithms to survival prediction problems.},
  archive      = {J_NATCS},
  author       = {Guan, Yuanfang and Li, Hongyang and Yi, Daiyao and Zhang, Dongdong and Yin, Changchang and Li, Keyu and Zhang, Ping},
  doi          = {10.1038/s43588-021-00083-2},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {433-440},
  shortjournal = {Nat. Comput. Sci},
  title        = {A survival model generalized to regression learning algorithms},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Detection of quantitative trait loci from RNA-seq data with
or without genotypes using BaseQTL. <em>NATCS</em>, <em>1</em>(6),
421–432. (<a href="https://doi.org/10.1038/s43588-021-00087-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detecting genetic variants associated with traits (quantitative trait loci, QTL) requires genotyped study individuals. Here we describe BaseQTL, a Bayesian method that exploits allele-specific expression to map molecular QTL from sequencing reads (eQTL for gene expression) even when no genotypes are available. When used with genotypes to map eQTL, BaseQTL has lower error rates and increased power compared with existing QTL mapping methods. Running without genotypes limits how many tests can be performed, but due to the proximity of QTL variants to gene bodies, the 2.8\% of variants within a 100 kB window that could be tested contained 26\% of eQTL detectable with genotypes. eQTL effect estimates were invariably consistent between analyses performed with and without genotypes. Often, sequencing data may be generated in the absence of genotypes on patients and controls in differential expression studies, and we identified an apparent psoriasis-specific eQTL for GSTP1 in one such dataset, providing new insights into disease-dependent gene regulation. BaseQTL is a Bayesian method to map molecular QTL affecting allele-specific expression even when no genotypes are available. It is well suited to discover eQTLs hidden in a wealth of RNA-seq data to unravel molecular mechanisms underpinning disease.},
  archive      = {J_NATCS},
  author       = {Vigorito, Elena and Lin, Wei-Yu and Starr, Colin and Kirk, Paul D. W. and White, Simon R. and Wallace, Chris},
  doi          = {10.1038/s43588-021-00087-y},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {421-432},
  shortjournal = {Nat. Comput. Sci},
  title        = {Detection of quantitative trait loci from RNA-seq data with or without genotypes using BaseQTL},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Maximally localized dynamical quantum embedding for solving
many-body correlated systems. <em>NATCS</em>, <em>1</em>(6), 410–420.
(<a href="https://doi.org/10.1038/s43588-021-00090-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum computing opens new avenues for modeling correlated materials, which are notoriously challenging to solve due to the presence of large electronic correlations. Quantum embedding approaches, such as dynamical mean-field theory, provide corrections to first-principles calculations for strongly correlated materials, which are poorly described at lower levels of theory. Such embedding approaches are computationally demanding on classical computing architectures and hence remain restricted to small systems, limiting the scope of their applicability. Hitherto, implementations on quantum computers have been limited by hardware constraints. Here, we derive a compact representation, where the number of quantum states is reduced for a given system while retaining a high level of accuracy. We benchmark our method for archetypal quantum states of matter that emerge due to electronic correlations, such as Kondo and Mott physics, both at equilibrium and for quenched systems. We implement this approach on a quantum emulator, demonstrating a reduction of the required number of qubits. Near-term quantum computers hold many promises but remain limited to a moderate number of qubits. This Article presents a pathway for modeling correlated materials with a reduced number of qubits, bringing quantum computing to materials modeling.},
  archive      = {J_NATCS},
  author       = {Lupo, Carla and Jamet, François and Tse, Wai Hei Terence and Rungger, Ivan and Weber, Cedric},
  doi          = {10.1038/s43588-021-00090-3},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {410-420},
  shortjournal = {Nat. Comput. Sci},
  title        = {Maximally localized dynamical quantum embedding for solving many-body correlated systems},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The power of quantum neural networks. <em>NATCS</em>,
<em>1</em>(6), 403–409. (<a
href="https://doi.org/10.1038/s43588-021-00084-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is unknown whether near-term quantum computers are advantageous for machine learning tasks. In this work we address this question by trying to understand how powerful and trainable quantum machine learning models are in relation to popular classical neural networks. We propose the effective dimension—a measure that captures these qualities—and prove that it can be used to assess any statistical model’s ability to generalize on new data. Crucially, the effective dimension is a data-dependent measure that depends on the Fisher information, which allows us to gauge the ability of a model to train. We demonstrate numerically that a class of quantum neural networks is able to achieve a considerably better effective dimension than comparable feedforward networks and train faster, suggesting an advantage for quantum machine learning, which we verify on real quantum hardware. A class of quantum neural networks is presented that outperforms comparable classical feedforward networks. They achieve a higher capacity in terms of effective dimension and at the same time train faster, suggesting a quantum advantage.},
  archive      = {J_NATCS},
  author       = {Abbas, Amira and Sutter, David and Zoufal, Christa and Lucchi, Aurelien and Figalli, Alessio and Woerner, Stefan},
  doi          = {10.1038/s43588-021-00084-1},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {403-409},
  shortjournal = {Nat. Comput. Sci},
  title        = {The power of quantum neural networks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Undisclosed, unmet and neglected challenges in multi-omics
studies. <em>NATCS</em>, <em>1</em>(6), 395–402. (<a
href="https://doi.org/10.1038/s43588-021-00086-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multi-omics approaches have become a reality in both large genomics projects and small laboratories. However, the multi-omics research community still faces a number of issues that have either not been sufficiently discussed or for which current solutions are still limited. In this Perspective, we elaborate on these limitations and suggest points of attention for future research. We finally discuss new opportunities and challenges brought to the field by the rapid development of single-cell high-throughput molecular technologies. Multi-omics studies have been increasingly used to better understand biological samples and infer molecular interactions. Nevertheless, a number of challenges must still be addressed to take full advantage of multi-omics data and to avoid reaching potentially incorrect conclusions.},
  archive      = {J_NATCS},
  author       = {Tarazona, Sonia and Arzalluz-Luque, Angeles and Conesa, Ana},
  doi          = {10.1038/s43588-021-00086-z},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {395-402},
  shortjournal = {Nat. Comput. Sci},
  title        = {Undisclosed, unmet and neglected challenges in multi-omics studies},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Detecting context-dependent gene regulation. <em>NATCS</em>,
<em>1</em>(6), 393–394. (<a
href="https://doi.org/10.1038/s43588-021-00092-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detection of molecular quantitative trait loci (QTL) facilitates mechanistic insights into disease-associated genetic variants. A new study describes BaseQTL, which exploits allele-specific expression to map molecular QTL from sequencing reads even without paired genotype data.},
  archive      = {J_NATCS},
  author       = {Gamazon, Eric R.},
  doi          = {10.1038/s43588-021-00092-1},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {393-394},
  shortjournal = {Nat. Comput. Sci},
  title        = {Detecting context-dependent gene regulation},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards scalable genomic data access. <em>NATCS</em>,
<em>1</em>(6), 391–392. (<a
href="https://doi.org/10.1038/s43588-021-00089-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new format for read depth annotations in genomic data makes access to metadata more scalable and efficient.},
  archive      = {J_NATCS},
  author       = {Hernaez, Mikel},
  doi          = {10.1038/s43588-021-00089-w},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {391-392},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards scalable genomic data access},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Seeking quantum advantage for neural networks.
<em>NATCS</em>, <em>1</em>(6), 389–390. (<a
href="https://doi.org/10.1038/s43588-021-00088-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study based on effective dimension shows that a quantum neural network can have increased capability and trainability as compared to its classical counterpart.},
  archive      = {J_NATCS},
  author       = {Coles, Patrick J.},
  doi          = {10.1038/s43588-021-00088-x},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {389-390},
  shortjournal = {Nat. Comput. Sci},
  title        = {Seeking quantum advantage for neural networks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Mathematically exploring wrinkle evolution. <em>NATCS</em>,
<em>1</em>(6), 388. (<a
href="https://doi.org/10.1038/s43588-021-00094-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00094-z},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {388},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mathematically exploring wrinkle evolution},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Data generation for training biomedical models.
<em>NATCS</em>, <em>1</em>(6), 387. (<a
href="https://doi.org/10.1038/s43588-021-00093-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00093-0},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {387},
  shortjournal = {Nat. Comput. Sci},
  title        = {Data generation for training biomedical models},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Accurate and efficient fluid flow analysis. <em>NATCS</em>,
<em>1</em>(6), 386. (<a
href="https://doi.org/10.1038/s43588-021-00095-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00095-y},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {386},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accurate and efficient fluid flow analysis},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Networking in times of pandemic. <em>NATCS</em>,
<em>1</em>(6), 385. (<a
href="https://doi.org/10.1038/s43588-021-00096-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The shift to virtual meetings has made networking harder, but it has also brought forth benefits to the scientific community that should be embraced moving forward.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00096-x},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {385},
  shortjournal = {Nat. Comput. Sci},
  title        = {Networking in times of pandemic},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Single-cell manifold-preserving feature selection for
detecting rare cell populations. <em>NATCS</em>, <em>1</em>(5), 374–384.
(<a href="https://doi.org/10.1038/s43588-021-00070-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key challenge in studying organisms and diseases is to detect rare molecular programs and rare cell populations that drive development, differentiation and transformation. Molecular features, such as genes and proteins, defining rare cell populations are often unknown and are difficult to detect from unenriched single-cell data using conventional dimensionality reduction and clustering-based approaches. Here, we propose an unsupervised approach, SCMER (‘single-cell manifold-preserving feature selection’), which selects a compact set of molecular features with definitive meanings that preserve the manifold of the data. We apply SCMER in the context of hematopoiesis, lymphogenesis, tumorigenesis and drug resistance and response. We find that SCMER can identify non-redundant features that sensitively delineate both common cell lineages and rare cellular states. SCMER can be used for discovering molecular features in a high-dimensional dataset, designing targeted, cost-effective assays for clinical applications and facilitating multi-modality integration. A manifold-preserving feature selection method was developed for single-cell data analysis, which selects non-redundant features to help detect rare cell populations, design follow-up studies and create targeted panels.},
  archive      = {J_NATCS},
  author       = {Liang, Shaoheng and Mohanty, Vakul and Dou, Jinzhuang and Miao, Qi and Huang, Yuefan and Müftüoğlu, Muharrem and Ding, Li and Peng, Weiyi and Chen, Ken},
  doi          = {10.1038/s43588-021-00070-7},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {374-384},
  shortjournal = {Nat. Comput. Sci},
  title        = {Single-cell manifold-preserving feature selection for detecting rare cell populations},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Rapid assessment of t-cell receptor specificity of the
immune repertoire. <em>NATCS</em>, <em>1</em>(5), 362–373. (<a
href="https://doi.org/10.1038/s43588-021-00076-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate assessment of T-cell-receptor (TCR)–antigen specificity across the whole immune repertoire lies at the heart of improved cancer immunotherapy, but predictive models capable of high-throughput assessment of TCR–peptide pairs are lacking. Recent advances in deep sequencing and crystallography have enriched the data available for studying TCR–peptide systems. Here, we introduce RACER, a pairwise energy model capable of rapid assessment of TCR–peptide affinity for entire immune repertoires. RACER applies supervised machine learning to efficiently and accurately resolve strong TCR–peptide binding pairs from weak ones. The trained parameters further enable a physical interpretation of interacting patterns encoded in each TCR–peptide system. When applied to simulate thymic selection of a major-histocompatibility-complex (MHC)-restricted T-cell repertoire, RACER accurately estimates recognition rates for tumor-associated neoantigens and foreign peptides, thus demonstrating its utility in helping address the computational challenge of reliably identifying properties of tumor antigen-specific T-cells at the level of an individual patient’s immune repertoire. Predicting binding specificity of T-cell receptors (TCRs) and putative antigens can help improve cancer immunotherapy. Lin et al. propose RACER, which efficiently makes use of supervised machine learning to learn important molecular interactions contributing to TCR–peptide binding.},
  archive      = {J_NATCS},
  author       = {Lin, Xingcheng and George, Jason T. and Schafer, Nicholas P. and Ng Chau, Kevin and Birnbaum, Michael E. and Clementi, Cecilia and Onuchic, José N. and Levine, Herbert},
  doi          = {10.1038/s43588-021-00076-1},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {362-373},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rapid assessment of T-cell receptor specificity of the immune repertoire},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Dynamic flux balance analysis of whole-body metabolism for
type 1 diabetes. <em>NATCS</em>, <em>1</em>(5), 348–361. (<a
href="https://doi.org/10.1038/s43588-021-00074-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Type 1 diabetes (T1D) mellitus is a systemic disease triggered by a local autoimmune inflammatory reaction in insulin-producing cells that induce organ-wide, long-term metabolic effects. Mathematical modeling of the whole-body regulatory bihormonal system has helped to identify therapeutic interventions but is limited to a coarse-grained representation of metabolism. To extend the depiction of T1D, we developed a whole-body model of organ-specific regulation and metabolism that highlighted chronic inflammation as a hallmark of the disease, identified processes related to neurodegenerative disorders and suggested calcium channel blockers as adjuvants for diabetes control. In addition, whole-body modeling of a patient population allowed for the assessment of between-individual variability to insulin and suggested that peripheral glucose levels are degenerate biomarkers of the internal metabolic state. Taken together, the organ-resolved, dynamic modeling approach enables modeling and simulation of metabolic disease at greater levels of coverage and precision and the generation of hypothesis from a molecular level up to the population level. A dynamic organ-resolved model is developed by integrating metabolic and regulatory processes in type 1 diabetes, providing a depiction of network dynamics, regulation and response to perturbations in relation to variability in insulin response.},
  archive      = {J_NATCS},
  author       = {Ben Guebila, Marouen and Thiele, Ines},
  doi          = {10.1038/s43588-021-00074-3},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {348-361},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dynamic flux balance analysis of whole-body metabolism for type 1 diabetes},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A probabilistic graphical model foundation for enabling
predictive digital twins at scale. <em>NATCS</em>, <em>1</em>(5),
337–347. (<a href="https://doi.org/10.1038/s43588-021-00069-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A unifying mathematical formulation is needed to move from one-off digital twins built through custom implementations to robust digital twin implementations at scale. This work proposes a probabilistic graphical model as a formal mathematical representation of a digital twin and its associated physical asset. We create an abstraction of the asset–twin system as a set of coupled dynamical systems, evolving over time through their respective state spaces and interacting via observed data and control inputs. The formal definition of this coupled system as a probabilistic graphical model enables us to draw upon well-established theory and methods from Bayesian statistics, dynamical systems and control theory. The declarative and general nature of the proposed digital twin model make it rigorous yet flexible, enabling its application at scale in a diverse range of application areas. We demonstrate how the model is instantiated to enable a structural digital twin of an unmanned aerial vehicle (UAV). The digital twin is calibrated using experimental data from a physical UAV asset. Its use in dynamic decision-making is then illustrated in a synthetic example where the UAV undergoes an in-flight damage event and the digital twin is dynamically updated using sensor data. The graphical model foundation ensures that the digital twin calibration and updating process is principled, unified and able to scale to an entire fleet of digital twins. This work proposes a probabilistic graphical model as a formal mathematical foundation for digital twins, and demonstrates how this model supports principled data assimilation, optimal control and end-to-end uncertainty quantification.},
  archive      = {J_NATCS},
  author       = {Kapteyn, Michael G. and Pretorius, Jacob V. R. and Willcox, Karen E.},
  doi          = {10.1038/s43588-021-00069-0},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {337-347},
  shortjournal = {Nat. Comput. Sci},
  title        = {A probabilistic graphical model foundation for enabling predictive digital twins at scale},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Time- and memory-efficient genome assembly with raven.
<em>NATCS</em>, <em>1</em>(5), 332–336. (<a
href="https://doi.org/10.1038/s43588-021-00073-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole genome sequencing technologies are unable to invariably read DNA molecules intact, a shortcoming that assemblers try to resolve by stitching the obtained fragments back together. Here, we present methods for the improvement of de novo genome assembly from erroneous long reads incorporated into a tool called Raven. Raven maintains similar performance for various genomes and has accuracy on par with other assemblers that support third-generation sequencing data. It is one of the fastest options while having the lowest memory consumption on the majority of benchmarked datasets. Raven is designed to democratize genome assembly, being a simple and efficient tool while keeping high accuracy. Using a method for detection of false overlaps based on graph drawing, it can be employed for various genome sizes.},
  archive      = {J_NATCS},
  author       = {Vaser, Robert and Šikić, Mile},
  doi          = {10.1038/s43588-021-00073-4},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {332-336},
  shortjournal = {Nat. Comput. Sci},
  title        = {Time- and memory-efficient genome assembly with raven},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Biomolecular modeling thrives in the age of technology.
<em>NATCS</em>, <em>1</em>(5), 321–331. (<a
href="https://doi.org/10.1038/s43588-021-00060-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biomolecular modeling field has flourished since its early days in the 1970s due to the rapid adaptation and tailoring of state-of-the-art technology. The resulting dramatic increase in size and timespan of biomolecular simulations has outpaced Moore’s law. Here, we discuss the role of knowledge-based versus physics-based methods and hardware versus software advances in propelling the field forward. This rapid adaptation and outreach suggests a bright future for modeling, where theory, experimentation and simulation define three pillars needed to address future scientific and biomedical challenges. The field of biomolecular modeling has thrived by exploiting state-of-the-art technological advances. In this Perspective, the role of software and hardware advances, and the disparity and synergy between knowledge-based and physics-based methods are discussed and explored.},
  archive      = {J_NATCS},
  author       = {Schlick, Tamar and Portillo-Ledesma, Stephanie},
  doi          = {10.1038/s43588-021-00060-9},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {321-331},
  shortjournal = {Nat. Comput. Sci},
  title        = {Biomolecular modeling thrives in the age of technology},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Scaling digital twins from the artisanal to the industrial.
<em>NATCS</em>, <em>1</em>(5), 313–320. (<a
href="https://doi.org/10.1038/s43588-021-00072-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mathematical modeling and simulation are moving from being powerful development and analysis tools towards having increased roles in operational monitoring, control and decision support, in which models of specific entities are continually updated in the form of a digital twin. However, current digital twins are largely the result of bespoke technical solutions that are difficult to scale. We discuss two exemplar applications that motivate challenges and opportunities for scaling digital twins, and that underscore potential barriers to wider adoption of this technology. Development in digital-twin technology has been rapidly growing across a range of industries and disciplines. However, to ensure a wider and more robust adoption of such technology, various challenges must be addressed by the computational science community.},
  archive      = {J_NATCS},
  author       = {Niederer, Steven A. and Sacks, Michael S. and Girolami, Mark and Willcox, Karen},
  doi          = {10.1038/s43588-021-00072-5},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {313-320},
  shortjournal = {Nat. Comput. Sci},
  title        = {Scaling digital twins from the artisanal to the industrial},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AI tool makes phase identification crystal clear.
<em>NATCS</em>, <em>1</em>(5), 311–312. (<a
href="https://doi.org/10.1038/s43588-021-00071-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping X-ray diffraction patterns to crystal structures is a comprehensive and time-consuming task for chemists and materials scientists. In a recent work, researchers developed a machine-learning tool to make this job more ‘self-driving’.},
  archive      = {J_NATCS},
  author       = {Sun, Wenhao and Toney, Michael F.},
  doi          = {10.1038/s43588-021-00071-6},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {311-312},
  shortjournal = {Nat. Comput. Sci},
  title        = {AI tool makes phase identification crystal clear},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A dynamic metabolic map for diabetes. <em>NATCS</em>,
<em>1</em>(5), 309–310. (<a
href="https://doi.org/10.1038/s43588-021-00075-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue, a large and multiscale whole-body model of organ-specific regulation and metabolism for type 1 diabetes is developed, providing important details on glucose and insulin dynamics.},
  archive      = {J_NATCS},
  author       = {Zhao, Jiao and Xu, Hao and Yang, Laurence},
  doi          = {10.1038/s43588-021-00075-2},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {309-310},
  shortjournal = {Nat. Comput. Sci},
  title        = {A dynamic metabolic map for diabetes},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The digital twin revolution. <em>NATCS</em>, <em>1</em>(5),
307–308. (<a href="https://doi.org/10.1038/s43588-021-00077-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A uniform mathematical framework based on probabilistic graphical models drives the digital twin technologies towards dynamical control with real-time data.},
  archive      = {J_NATCS},
  author       = {San, Omer},
  doi          = {10.1038/s43588-021-00077-0},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {307-308},
  shortjournal = {Nat. Comput. Sci},
  title        = {The digital twin revolution},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Large-scale transport simulation by deep learning.
<em>NATCS</em>, <em>1</em>(5), 306. (<a
href="https://doi.org/10.1038/s43588-021-00080-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00080-5},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {306},
  shortjournal = {Nat. Comput. Sci},
  title        = {Large-scale transport simulation by deep learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Insights on protein condensate formation. <em>NATCS</em>,
<em>1</em>(5), 305. (<a
href="https://doi.org/10.1038/s43588-021-00081-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00081-4},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {305},
  shortjournal = {Nat. Comput. Sci},
  title        = {Insights on protein condensate formation},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Inverse problem via a bayesian approach. <em>NATCS</em>,
<em>1</em>(5), 304. (<a
href="https://doi.org/10.1038/s43588-021-00079-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00079-y},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {304},
  shortjournal = {Nat. Comput. Sci},
  title        = {Inverse problem via a bayesian approach},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). New views of black holes from computational imaging.
<em>NATCS</em>, <em>1</em>(5), 300–303. (<a
href="https://doi.org/10.1038/s43588-021-00078-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The unique challenges associated with imaging a black hole motivated the development of new computational imaging algorithms. As the Event Horizon Telescope continues to expand, these algorithms will need to evolve to keep pace with the increasingly demanding volume and dimensionality of the data.},
  archive      = {J_NATCS},
  author       = {Akiyama, Kazunori and Chael, Andrew and Pesce, Dominic W.},
  doi          = {10.1038/s43588-021-00078-z},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {300-303},
  shortjournal = {Nat. Comput. Sci},
  title        = {New views of black holes from computational imaging},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Theorists and experimentalists must join forces.
<em>NATCS</em>, <em>1</em>(5), 299. (<a
href="https://doi.org/10.1038/s43588-021-00082-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Science can advance at a faster pace when taking advantage of the synergy between theory and experimentation.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00082-3},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {299},
  shortjournal = {Nat. Comput. Sci},
  title        = {Theorists and experimentalists must join forces},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Crystallography companion agent for high-throughput
materials discovery. <em>NATCS</em>, <em>1</em>(4), 290–297. (<a
href="https://doi.org/10.1038/s43588-021-00059-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery of new structural and functional materials is driven by phase identification, often using X-ray diffraction (XRD). Automation has accelerated the rate of XRD measurements, greatly outpacing XRD analysis techniques that remain manual, time-consuming, error-prone and impossible to scale. With the advent of autonomous robotic scientists or self-driving laboratories, contemporary techniques prohibit the integration of XRD. Here, we describe a computer program for the autonomous characterization of XRD data, driven by artificial intelligence (AI), for the discovery of new materials. Starting from structural databases, we train an ensemble model using a physically accurate synthetic dataset, which outputs probabilistic classifications—rather than absolutes—to overcome the overconfidence in traditional neural networks. This AI agent behaves as a companion to the researcher, improving accuracy and offering substantial time savings. It is demonstrated on a diverse set of organic and inorganic materials characterization challenges. This method is directly applicable to inverse design approaches and robotic discovery systems, and can be immediately considered for other forms of characterization such as spectroscopy and the pair distribution function. Crystallography companion agent enables autonomous characterization of powder X-ray diffraction data for organic and inorganic materials.},
  archive      = {J_NATCS},
  author       = {Maffettone, Phillip M. and Banko, Lars and Cui, Peng and Lysogorskiy, Yury and Little, Marc A. and Olds, Daniel and Ludwig, Alfred and Cooper, Andrew I.},
  doi          = {10.1038/s43588-021-00059-2},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {290-297},
  shortjournal = {Nat. Comput. Sci},
  title        = {Crystallography companion agent for high-throughput materials discovery},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Interrogation of clonal tracking data using barcodetrackR.
<em>NATCS</em>, <em>1</em>(4), 280–289. (<a
href="https://doi.org/10.1038/s43588-021-00057-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clonal tracking methods provide quantitative insights into the cellular output of genetically labeled progenitor cells across time and cellular compartments. In the context of gene and cell therapies, clonal tracking methods have enabled the tracking of progenitor cell output both in humans receiving therapies and in corresponding animal models, providing valuable insight into lineage reconstitution, clonal dynamics and vector genotoxicity. However, the absence of a toolbox for analysis of clonal tracking data has precluded the development of standardized analytical frameworks within the field. Thus, we developed barcodetrackR, an R package and accompanying Shiny app containing diverse tools for the analysis and visualization of clonal tracking data. We demonstrate the utility of barcodetrackR in exploring longitudinal clonal patterns and lineage relationships in a number of clonal tracking studies of hematopoietic stem and progenitor cells (HSPCs) in humans receiving HSPC gene therapy and in animals receiving lentivirally transduced HSPC transplants or tumor cells. The R package barcodetrackR facilitates the analysis and visualization of clonal tracking data, and it includes a graphical user interface to allow researchers without programming experience to utilize various quantitative tools.},
  archive      = {J_NATCS},
  author       = {Espinoza, Diego A. and Mortlock, Ryland D. and Koelle, Samson J. and Wu, Chuanfeng and Dunbar, Cynthia E.},
  doi          = {10.1038/s43588-021-00057-4},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {280-289},
  shortjournal = {Nat. Comput. Sci},
  title        = {Interrogation of clonal tracking data using barcodetrackR},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Avoiding genetic racial profiling in criminal DNA profile
databases. <em>NATCS</em>, <em>1</em>(4), 272–279. (<a
href="https://doi.org/10.1038/s43588-021-00058-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA profiling has become an essential tool for crime solving and prevention, and CODIS (Combined DNA Index System) criminal investigation databases have flourished at the national, state and even local level. However, reports suggest that the DNA profiles of all suspects searched in these databases are often retained, which could result in racial profiling. Here, we devise an approach to both enable broad DNA profile searches and preserve exonerated citizens’ privacy through a real-time privacy-preserving procedure to query CODIS databases. Using our approach, an agent can privately and efficiently query a suspect’s DNA profile device in the field, learning only whether the profile matches against any database profile. More importantly, the central database learns nothing about the queried profile, and thus cannot retain it. Our approach paves the way to implement privacy-preserving DNA profile searching in CODIS databases and any CODIS-like system. DNA profiles of suspects that are searched in criminal databases, such as CODIS, are often retained and can result in racial profiling. This work builds a privacy-preserving CODIS query system to support DNA profile searches while preserving individual privacy.},
  archive      = {J_NATCS},
  author       = {Blindenbach, Jacob A. and Jagadeesh, Karthik A. and Bejerano, Gill and Wu, David J.},
  doi          = {10.1038/s43588-021-00058-3},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {272-279},
  shortjournal = {Nat. Comput. Sci},
  title        = {Avoiding genetic racial profiling in criminal DNA profile databases},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Multiple haplotype reconstruction from allele frequency
data. <em>NATCS</em>, <em>1</em>(4), 262–271. (<a
href="https://doi.org/10.1038/s43588-021-00056-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Because haplotype information is of widespread interest in biomedical applications, effort has been put into their reconstruction. Here, we propose an efficient method, called haploSep, that is able to accurately infer major haplotypes and their frequencies just from multiple samples of allele frequency data. Even the accuracy of experimentally obtained allele frequencies can be improved by re-estimating them from our reconstructed haplotypes. From a methodological point of view, we model our problem as a multivariate regression problem where both the design matrix and the coefficient matrix are unknown. Compared to other methods, haploSep is very fast, with linear computational complexity in the haplotype length. We illustrate our method on simulated and real data focusing on experimental evolution and microbial data. haploSep is a computationally efficient method to infer major haplotypes and their frequencies from multiple samples of allele frequency data, and to provide improved estimates of experimentally obtained allele frequencies.},
  archive      = {J_NATCS},
  author       = {Pelizzola, Marta and Behr, Merle and Li, Housen and Munk, Axel and Futschik, Andreas},
  doi          = {10.1038/s43588-021-00056-5},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {262-271},
  shortjournal = {Nat. Comput. Sci},
  title        = {Multiple haplotype reconstruction from allele frequency data},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cell-type-aware analysis of RNA-seq data. <em>NATCS</em>,
<em>1</em>(4), 253–261. (<a
href="https://doi.org/10.1038/s43588-021-00055-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most tissue samples are composed of different cell types. Differential expression analysis without accounting for cell-type composition cannot separate the changes due to cell-type composition or cell type-specific expression. We propose a computational framework to address these limitations: CARseq (cell-type-aware analysis of RNA-seq). CARseq employs a negative binomial distribution that appropriately models the count data from RNA-seq experiments. Simulation studies show that CARseq has substantially higher power than a linear model-based approach and it also provides more accurate estimate of the rankings of differentially expressed genes. We have applied CARseq to compare gene expression of schizophrenia/autism subjects versus controls, and identified the cell types underlying the difference and similarities of these two neuron-developmental diseases. Our results are consistent with the results from differential expression analysis using single-cell RNA-seq data. The CARseq method allows users to assess cell type-specific differential expression using RNA-seq data from bulk tissue samples, which opens up several opportunities for re-analyzing existing RNA-seq data and designing new studies.},
  archive      = {J_NATCS},
  author       = {Jin, Chong and Chen, Mengjie and Lin, Dan-Yu and Sun, Wei},
  doi          = {10.1038/s43588-021-00055-6},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {253-261},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cell-type-aware analysis of RNA-seq data},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A key toolbox for cellular barcoding analysis.
<em>NATCS</em>, <em>1</em>(4), 251–252. (<a
href="https://doi.org/10.1038/s43588-021-00062-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analyzing cellular barcoding data is challenging due to the absence of a flexible, complete and easy-to-use set of tools that can help scientists derive biological meaning from these datasets. barcodetrackR is a promising package for filling this gap in the field.},
  archive      = {J_NATCS},
  author       = {Adair, Jennifer E. and Enstrom, Mark R.},
  doi          = {10.1038/s43588-021-00062-7},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {251-252},
  shortjournal = {Nat. Comput. Sci},
  title        = {A key toolbox for cellular barcoding analysis},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Protecting against racial bias in DNA databasing.
<em>NATCS</em>, <em>1</em>(4), 249–250. (<a
href="https://doi.org/10.1038/s43588-021-00064-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue, a protocol for querying criminal DNA databases is developed to prevent profile discovery, therefore reducing the potential for racial bias.},
  archive      = {J_NATCS},
  author       = {Syndercombe Court, Denise},
  doi          = {10.1038/s43588-021-00064-5},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {249-250},
  shortjournal = {Nat. Comput. Sci},
  title        = {Protecting against racial bias in DNA databasing},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Extracting insights from heterogeneous tissues.
<em>NATCS</em>, <em>1</em>(4), 247–248. (<a
href="https://doi.org/10.1038/s43588-021-00061-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A limitation of data obtained from RNA-seq experiments is the presence of different types of cell expression, making it difficult to identify the contribution of cell-type composition or cell-type-specific expression. A new study addresses this problem by proposing a method for cell-type-aware analysis of RNA-seq data.},
  archive      = {J_NATCS},
  author       = {Aran, Dvir},
  doi          = {10.1038/s43588-021-00061-8},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {247-248},
  shortjournal = {Nat. Comput. Sci},
  title        = {Extracting insights from heterogeneous tissues},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Computationally probing exciton dynamics. <em>NATCS</em>,
<em>1</em>(4), 246. (<a
href="https://doi.org/10.1038/s43588-021-00065-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00065-4},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {246},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computationally probing exciton dynamics},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Comparing single-cell datasets. <em>NATCS</em>,
<em>1</em>(4), 245. (<a
href="https://doi.org/10.1038/s43588-021-00066-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00066-3},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {245},
  shortjournal = {Nat. Comput. Sci},
  title        = {Comparing single-cell datasets},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Can machines learn with hard constraints? <em>NATCS</em>,
<em>1</em>(4), 244. (<a
href="https://doi.org/10.1038/s43588-021-00067-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00067-2},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {244},
  shortjournal = {Nat. Comput. Sci},
  title        = {Can machines learn with hard constraints?},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Designing a biomedical coding program focused on inclusivity
and agency. <em>NATCS</em>, <em>1</em>(4), 242–243. (<a
href="https://doi.org/10.1038/s43588-021-00063-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Mehta, Kala M. and Ceberio, Niquo and Vernard, Sonthonax and Carrasco, Yazmin P. and Bibbins-Domingo, Kirsten and Nguyen, Tung T. and Lê, Peter and Bailey, Jasmón and Parangan-Smith, Audrey and Uwaezuoke, Kelechi and Rivers, Robert C. and Watson, Kenjus and Márquez-Magaña, Leticia},
  doi          = {10.1038/s43588-021-00063-6},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {242-243},
  shortjournal = {Nat. Comput. Sci},
  title        = {Designing a biomedical coding program focused on inclusivity and agency},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). To review or not to review. <em>NATCS</em>, <em>1</em>(4),
241. (<a href="https://doi.org/10.1038/s43588-021-00068-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How do editors at Nature Computational Science evaluate papers before deciding on whether to send them out to peer review?},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00068-1},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {241},
  shortjournal = {Nat. Comput. Sci},
  title        = {To review or not to review},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Publisher correction: Similarity-driven multi-view
embeddings from high-dimensional biomedical data. <em>NATCS</em>,
<em>1</em>(3), 239. (<a
href="https://doi.org/10.1038/s43588-021-00049-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s43588-021-00049-4.},
  archive      = {J_NATCS},
  author       = {Avants, Brian B. and Tustison, Nicholas J. and Stone, James R.},
  doi          = {10.1038/s43588-021-00049-4},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {239},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Similarity-driven multi-view embeddings from high-dimensional biomedical data},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fast inverse design of microstructures via generative
invariance networks. <em>NATCS</em>, <em>1</em>(3), 229–238. (<a
href="https://doi.org/10.1038/s43588-021-00045-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The problem of the efficient design of material microstructures exhibiting desired properties spans a variety of engineering and science applications. The ability to rapidly generate microstructures that exhibit user-specified property distributions can transform the iterative process of traditional microstructure-sensitive design. We reformulate the microstructure design process using a constrained generative adversarial network (GAN) model. This approach explicitly encodes invariance constraints within GANs to generate two-phase morphologies for photovoltaic applications obeying design specifications: specifically, user-defined short-circuit current density and fill factor combinations. Such invariance constraints can be represented by differentiable, deep learning-based surrogates of full physics models mapping microstructures to photovoltaic properties. Furthermore, we propose a multi-fidelity surrogate that reduces expensive label requirements by a factor of five. Our framework enables the incorporation of expensive or non-differentiable constraints for the fast generation of microstructures (in 190 ms) with user-defined properties. Such proposed physics-aware data-driven methods for inverse design problems can be used to considerably accelerate the field of microstructure-sensitive design. Physics-aware deep generative models are used to design material microstructures exhibiting tailored properties. Multi-fidelity data are used to create inexpensive yet accurate machine learning surrogates for evaluating the physics-based constraints within such design frameworks.},
  archive      = {J_NATCS},
  author       = {Lee, Xian Yeow and Waite, Joshua R. and Yang, Chih-Hsuan and Pokuri, Balaji Sesha Sarath and Joshi, Ameya and Balu, Aditya and Hegde, Chinmay and Ganapathysubramanian, Baskar and Sarkar, Soumik},
  doi          = {10.1038/s43588-021-00045-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {229-238},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fast inverse design of microstructures via generative invariance networks},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Random sketch learning for deep neural networks in edge
computing. <em>NATCS</em>, <em>1</em>(3), 221–228. (<a
href="https://doi.org/10.1038/s43588-021-00039-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the great potential of deep neural networks (DNNs), they require massive weights and huge computational resources, creating a vast gap when deploying artificial intelligence at low-cost edge devices. Current lightweight DNNs, achieved by high-dimensional space pre-training and post-compression, present challenges when covering the resources deficit, making tiny artificial intelligence hard to be implemented. Here we report an architecture named random sketch learning, or Rosler, for computationally efficient tiny artificial intelligence. We build a universal compressing-while-training framework that directly learns a compact model and, most importantly, enables computationally efficient on-device learning. As validated on different models and datasets, it attains substantial memory reduction of ~50–90× (16-bits quantization), compared with fully connected DNNs. We demonstrate it on low-cost hardware, whereby the computation is accelerated by &amp;gt;180× and the energy consumption is reduced by ~10×. Our method paves the way for deploying tiny artificial intelligence in many scientific and industrial applications. Developing lightweight deep neural networks, while essential for edge computing, still remains a challenge. Random sketch learning is a method that creates computationally efficient and compact networks, thus paving the way for deploying tiny machine learning (TinyML) in resource-constrained devices.},
  archive      = {J_NATCS},
  author       = {Li, Bin and Chen, Peijun and Liu, Hongfu and Guo, Weisi and Cao, Xianbin and Du, Junzhao and Zhao, Chenglin and Zhang, Jun},
  doi          = {10.1038/s43588-021-00039-6},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {221-228},
  shortjournal = {Nat. Comput. Sci},
  title        = {Random sketch learning for deep neural networks in edge computing},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modeling the electrical double layer at solid-state
electrochemical interfaces. <em>NATCS</em>, <em>1</em>(3), 212–220. (<a
href="https://doi.org/10.1038/s43588-021-00041-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Models of the electrical double layer (EDL) at electrode/liquid-electrolyte interfaces no longer hold for all-solid-state electrochemistry. Here we show a more general model for the EDL at a solid-state electrochemical interface based on the Poisson–Fermi–Dirac equation. By combining this model with density functional theory predictions, the interconnected electronic and ionic degrees of freedom in all-solid-state batteries, including the electronic band bending and defect concentration variation in the space-charge layer, are captured self-consistently. Along with a general mathematical solution, the EDL structure is presented in various materials that are thermodynamically stable in contact with a lithium metal anode: the solid electrolyte Li7La3Zr2O12 (LLZO) and the solid interlayer materials LiF, Li2O and Li2CO3. The model further allows design of the optimum interlayer thicknesses to minimize the electrostatic barrier for lithium ion transport at relevant solid-state battery interfaces. A self-consistent model that bridges electrochemistry and solid-state physics is developed to fully describe the ion and electron distribution at solid electrode/electrolyte interfaces and applied to lower interfacial resistance in batteries.},
  archive      = {J_NATCS},
  author       = {Swift, Michael W. and Swift, James W. and Qi, Yue},
  doi          = {10.1038/s43588-021-00041-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {212-220},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling the electrical double layer at solid-state electrochemical interfaces},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Colocalization for super-resolution microscopy via optimal
transport. <em>NATCS</em>, <em>1</em>(3), 199–211. (<a
href="https://doi.org/10.1038/s43588-021-00050-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Super-resolution fluorescence microscopy is a widely used technique in cell biology. Stimulated emission depletion (STED) microscopy enables the recording of multiple-color images with subdiffraction resolution. The enhanced resolution leads to new challenges regarding colocalization analysis of macromolecule distributions. We demonstrate that well-established methods for the analysis of colocalization in diffraction-limited datasets and for coordinate-stochastic nanoscopy are not equally well suited for the analysis of high-resolution STED images. We propose optimal transport colocalization, which measures the minimal transporting cost below a given spatial scale to match two protein intensity distributions. Its validity on simulated data as well as on dual-color STED recordings of yeast and mammalian cells is demonstrated. We also extend the optimal transport colocalization methodology to coordinate-stochastic nanoscopy. Stimulated emission depletion (STED) microscopy allows images to be captured at a subdiffraction resolution. Here optimal transport colocalization is proposed for analyzing macromolecule distributions in high-resolution STED images.},
  archive      = {J_NATCS},
  author       = {Tameling, Carla and Stoldt, Stefan and Stephan, Till and Naas, Julia and Jakobs, Stefan and Munk, Axel},
  doi          = {10.1038/s43588-021-00050-x},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {199-211},
  shortjournal = {Nat. Comput. Sci},
  title        = {Colocalization for super-resolution microscopy via optimal transport},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Citizen-centered, auditable and privacy-preserving
population genomics. <em>NATCS</em>, <em>1</em>(3), 192–198. (<a
href="https://doi.org/10.1038/s43588-021-00044-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing number of health-data breaches, the use of genomic databases for law enforcement purposes and the lack of transparency of personal genomics companies are raising unprecedented privacy concerns. To enable a secure exploration of genomic datasets with controlled and transparent data access, we propose a citizen-centric approach that combines cryptographic privacy-preserving technologies, such as homomorphic encryption and secure multi-party computation, with the auditability of blockchains. Our open-source implementation supports queries on the encrypted genomic data of hundreds of thousands of individuals, with minimal overhead. We show that real-world adoption of our system alleviates widespread privacy concerns and encourages data access sharing with researchers. Sharing of personal genomics data raises privacy concerns due to the sensitive nature of the data. A citizen-centric approach involving cryptographic privacy-preserving technologies and blockchains is proposed to address this problem.},
  archive      = {J_NATCS},
  author       = {Grishin, Dennis and Raisaro, Jean Louis and Troncoso-Pastoriza, Juan Ramón and Obbad, Kamal and Quinn, Kevin and Misbach, Mickaël and Gollhardt, Jared and Sa, Joao and Fellay, Jacques and Church, George M. and Hubaux, Jean-Pierre},
  doi          = {10.1038/s43588-021-00044-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {192-198},
  shortjournal = {Nat. Comput. Sci},
  title        = {Citizen-centered, auditable and privacy-preserving population genomics},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Machine learning for deciphering cell heterogeneity and gene
regulation. <em>NATCS</em>, <em>1</em>(3), 183–191. (<a
href="https://doi.org/10.1038/s43588-021-00038-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetics studies inheritable and reversible modifications of DNA that allow cells to control gene expression throughout their development and in response to environmental conditions. In computational epigenomics, machine learning is applied to study various epigenetic mechanisms genome wide. Its aim is to expand our understanding of cell differentiation, that is their specialization, in health and disease. Thus far, most efforts focus on understanding the functional encoding of the genome and on unraveling cell-type heterogeneity. Here, we provide an overview of state-of-the-art computational methods and their underlying statistical concepts, which range from matrix factorization and regularized linear regression to deep learning methods. We further show how the rise of single-cell technology leads to new computational challenges and creates opportunities to further our understanding of epigenetic regulation. Massive datasets have been made available to enable systematic studies of gene regulation and its control via epigenetic mechanisms. In this Review, state-of-the-art computational methods used to effectively extract knowledge from these datasets are presented and discussed.},
  archive      = {J_NATCS},
  author       = {Scherer, Michael and Schmidt, Florian and Lazareva, Olga and Walter, Jörn and Baumbach, Jan and Schulz, Marcel H. and List, Markus},
  doi          = {10.1038/s43588-021-00038-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {183-191},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning for deciphering cell heterogeneity and gene regulation},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Efficient deep learning. <em>NATCS</em>, <em>1</em>(3),
181–182. (<a href="https://doi.org/10.1038/s43588-021-00042-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The computational complexity of deep neural networks is a major obstacle of many application scenarios driven by low-power devices, including federated learning. A recent finding shows that random sketches can substantially reduce the model complexity without affecting prediction accuracy.},
  archive      = {J_NATCS},
  author       = {Wang, Shiqiang},
  doi          = {10.1038/s43588-021-00042-x},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {181-182},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient deep learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). On the design of solid-state li-ion batteries.
<em>NATCS</em>, <em>1</em>(3), 179–180. (<a
href="https://doi.org/10.1038/s43588-021-00043-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A model for the electrical double layer at solid-state electrochemical interfaces is reported, shedding some light on the design and optimization of future all-solid-state Li-ion batteries.},
  archive      = {J_NATCS},
  author       = {Muy, Sokseiha and Marzari, Nicola},
  doi          = {10.1038/s43588-021-00043-w},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {179-180},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the design of solid-state li-ion batteries},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Revisiting colocalization via optimal transport.
<em>NATCS</em>, <em>1</em>(3), 177–178. (<a
href="https://doi.org/10.1038/s43588-021-00046-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The advent of STED microscopy, which allows observation at a sub-diffraction resolution, raises a challenge in studying spatial proximities of biomolecules’ distributions. In this issue, researchers have attempted to study colocalization of molecules by employing optimal transport.},
  archive      = {J_NATCS},
  author       = {Wang, Shulei and Yuan, Ming},
  doi          = {10.1038/s43588-021-00046-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {177-178},
  shortjournal = {Nat. Comput. Sci},
  title        = {Revisiting colocalization via optimal transport},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Data privacy through participant empowerment.
<em>NATCS</em>, <em>1</em>(3), 175–176. (<a
href="https://doi.org/10.1038/s43588-021-00047-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sharing of genomic data poses a problem due to privacy concerns and lack of individual’s choice in the matter. In this issue, researchers propose a framework for sharing data where the control lies with the individual.},
  archive      = {J_NATCS},
  author       = {Dankar, Fida K.},
  doi          = {10.1038/s43588-021-00047-6},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {175-176},
  shortjournal = {Nat. Comput. Sci},
  title        = {Data privacy through participant empowerment},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021o). Simulating thermal physics on quantum computers.
<em>NATCS</em>, <em>1</em>(3), 174. (<a
href="https://doi.org/10.1038/s43588-021-00053-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00053-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {174},
  shortjournal = {Nat. Comput. Sci},
  title        = {Simulating thermal physics on quantum computers},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Interpretable predictions of cellular behavior.
<em>NATCS</em>, <em>1</em>(3), 173. (<a
href="https://doi.org/10.1038/s43588-021-00052-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00052-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {173},
  shortjournal = {Nat. Comput. Sci},
  title        = {Interpretable predictions of cellular behavior},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Porting software without affecting scientific results.
<em>NATCS</em>, <em>1</em>(3), 172. (<a
href="https://doi.org/10.1038/s43588-021-00051-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00051-w},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {172},
  shortjournal = {Nat. Comput. Sci},
  title        = {Porting software without affecting scientific results},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). We need to talk about the lack of investment in digital
research infrastructure. <em>NATCS</em>, <em>1</em>(3), 169–171. (<a
href="https://doi.org/10.1038/s43588-021-00048-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research software infrastructure is critical for accelerating science, and yet, these digital public goods are often unsustainably funded. Solving this problem requires an appreciation of the intrinsic value of research software outputs, and greater investment of time and effort into effectively funding maintenance of software at scale.},
  archive      = {J_NATCS},
  author       = {Knowles, Rebecca and Mateen, Bilal A. and Yehudi, Yo},
  doi          = {10.1038/s43588-021-00048-5},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {169-171},
  shortjournal = {Nat. Comput. Sci},
  title        = {We need to talk about the lack of investment in digital research infrastructure},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The imperative of physics-based modeling and inverse theory
in computational science. <em>NATCS</em>, <em>1</em>(3), 166–168. (<a
href="https://doi.org/10.1038/s43588-021-00040-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To best learn from data about large-scale complex systems, physics-based models representing the laws of nature must be integrated into the learning process. Inverse theory provides a crucial perspective for addressing the challenges of ill-posedness, uncertainty, nonlinearity and under-sampling.},
  archive      = {J_NATCS},
  author       = {Willcox, Karen E. and Ghattas, Omar and Heimbach, Patrick},
  doi          = {10.1038/s43588-021-00040-z},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {166-168},
  shortjournal = {Nat. Comput. Sci},
  title        = {The imperative of physics-based modeling and inverse theory in computational science},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Shining a light on the peer review process. <em>NATCS</em>,
<em>1</em>(3), 165. (<a
href="https://doi.org/10.1038/s43588-021-00054-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our authors now have the option to publish the review reports alongside their manuscript.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00054-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {165},
  shortjournal = {Nat. Comput. Sci},
  title        = {Shining a light on the peer review process},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Enabling single-cell trajectory network enrichment.
<em>NATCS</em>, <em>1</em>(2), 153–163. (<a
href="https://doi.org/10.1038/s43588-021-00025-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell sequencing (scRNA-seq) technologies allow the investigation of cellular differentiation processes with unprecedented resolution. Although powerful software packages for scRNA-seq data analysis exist, systems biology-based tools for trajectory analysis are rare and typically difficult to handle. This hampers biological exploration and prevents researchers from gaining deeper insights into the molecular control of developmental processes. Here, to address this, we have developed Scellnetor; a network-constraint time-series clustering algorithm. It allows extraction of temporal differential gene expression network patterns (modules) that explain the difference in regulation of two developmental trajectories. Using well-characterized experimental model systems, we demonstrate the capacity of Scellnetor as a hypothesis generator to identify putative mechanisms driving haematopoiesis or mechanistically interpretable subnetworks driving dysfunctional CD8 T-cell development in chronic infections. Altogether, Scellnetor allows for single-cell trajectory network enrichment, which effectively lifts scRNA-seq data analysis to a systems biology level. Scellnetor is a single-cell network enrichment method that can unravel connected molecular interaction networks to explain the progression and differentiation of developmental trajectories on a systems biology level.},
  archive      = {J_NATCS},
  author       = {Grønning, Alexander G. B. and Oubounyt, Mhaned and Kanev, Kristiyan and Lund, Jesper and Kacprowski, Tim and Zehn, Dietmar and Röttger, Richard and Baumbach, Jan},
  doi          = {10.1038/s43588-021-00025-y},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {153-163},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enabling single-cell trajectory network enrichment},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Similarity-driven multi-view embeddings from
high-dimensional biomedical data. <em>NATCS</em>, <em>1</em>(2),
143–152. (<a href="https://doi.org/10.1038/s43588-021-00029-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diverse, high-dimensional modalities collected in large cohorts present new opportunities for the formulation and testing of integrative scientific hypotheses. Similarity-driven multi-view linear reconstruction (SiMLR) is an algorithm that exploits inter-modality relationships to transform large scientific datasets into smaller, more well-powered and interpretable low-dimensional spaces. SiMLR contributes an objective function to identify joint signal regularization based on sparse matrices representing prior within-modality relationships and an implementation that permits application to joint reduction of large data matrices. We demonstrate that SiMLR outperlforms closely related methods on supervised learning problems in simulation data, a multi-omics cancer survival prediction dataset and multiple modality neuroimaging datasets. Taken together, this collection of results shows that SiMLR may be applied to joint signal estimation from disparate modalities and may yield practically useful results in a variety of application domains. Analysis and interpretation of relationships across different modalities of biomedical datasets is a critical but challenging task. SiMLR is a joint dimensionality reduction algorithm that is designed to perform this task.},
  archive      = {J_NATCS},
  author       = {Avants, Brian B. and Tustison, Nicholas J. and Stone, James R.},
  doi          = {10.1038/s43588-021-00029-8},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {143-152},
  shortjournal = {Nat. Comput. Sci},
  title        = {Similarity-driven multi-view embeddings from high-dimensional biomedical data},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Larger GPU-accelerated brain simulations with procedural
connectivity. <em>NATCS</em>, <em>1</em>(2), 136–142. (<a
href="https://doi.org/10.1038/s43588-020-00022-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Simulations are an important tool for investigating brain function but large models are needed to faithfully reproduce the statistics and dynamics of brain activity. Simulating large spiking neural network models has, until now, needed so much memory for storing synaptic connections that it required high performance computer systems. Here, we present an alternative simulation method we call ‘procedural connectivity’ where connectivity and synaptic weights are generated ‘on the fly’ instead of stored and retrieved from memory. This method is particularly well suited for use on graphical processing units (GPUs)—which are a common fixture in many workstations. Using procedural connectivity and an additional GPU code generation optimization, we can simulate a recent model of the macaque visual cortex with 4.13 × 106 neurons and 24.2 × 109 synapses on a single GPU—a significant step forward in making large-scale brain modeling accessible to more researchers. Spiking neural network simulations are very memory-intensive, limiting large-scale brain simulations to high-performance computer systems. Knight and Nowotny propose using procedural connectivity to substantially reduce the memory footprint of these models, such that they can run on standard GPUs.},
  archive      = {J_NATCS},
  author       = {Knight, James C. and Nowotny, Thomas},
  doi          = {10.1038/s43588-020-00022-7},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {136-142},
  shortjournal = {Nat. Comput. Sci},
  title        = {Larger GPU-accelerated brain simulations with procedural connectivity},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The impact of uncertainty on predictions of the CovidSim
epidemiological code. <em>NATCS</em>, <em>1</em>(2), 128–135. (<a
href="https://doi.org/10.1038/s43588-021-00028-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epidemiological modelling has assisted in identifying interventions that reduce the impact of COVID-19. The UK government relied, in part, on the CovidSim model to guide its policy to contain the rapid spread of the COVID-19 pandemic during March and April 2020; however, CovidSim contains several sources of uncertainty that affect the quality of its predictions: parametric uncertainty, model structure uncertainty and scenario uncertainty. Here we report on parametric sensitivity analysis and uncertainty quantification of the code. From the 940 parameters used as input into CovidSim, we find a subset of 19 to which the code output is most sensitive—imperfect knowledge of these inputs is magnified in the outputs by up to 300\%. The model displays substantial bias with respect to observed data, failing to describe validation data well. Quantifying parametric input uncertainty is therefore not sufficient: the effect of model structure and scenario uncertainty must also be properly understood. Through parametric sensitivity analysis and uncertainty quantification of the CovidSim model, a subset of this model’s parameters is identified to which the code output is most sensitive. Using these allows better and more informed decisions about proposed policies.},
  archive      = {J_NATCS},
  author       = {Edeling, Wouter and Arabnejad, Hamid and Sinclair, Robbie and Suleimenova, Diana and Gopalakrishnan, Krishnakumar and Bosak, Bartosz and Groen, Derek and Mahmood, Imran and Crommelin, Daan and Coveney, Peter V.},
  doi          = {10.1038/s43588-021-00028-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {128-135},
  shortjournal = {Nat. Comput. Sci},
  title        = {The impact of uncertainty on predictions of the CovidSim epidemiological code},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Consistent cross-modal identification of cortical neurons
with coupled autoencoders. <em>NATCS</em>, <em>1</em>(2), 120–127. (<a
href="https://doi.org/10.1038/s43588-021-00030-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Consistent identification of neurons in different experimental modalities is a key problem in neuroscience. Although methods to perform multimodal measurements in the same set of single neurons have become available, parsing complex relationships across different modalities to uncover neuronal identity is a growing challenge. Here we present an optimization framework to learn coordinated representations of multimodal data and apply it to a large multimodal dataset profiling mouse cortical interneurons. Our approach reveals strong alignment between transcriptomic and electrophysiological characterizations, enables accurate cross-modal data prediction, and identifies cell types that are consistent across modalities. Aligning multimodal data for cell type research is a challenging problem in neuroscience. Coupled autoencoders, a deep neural network-based methodology, can be used to effectively address this task.},
  archive      = {J_NATCS},
  author       = {Gala, Rohan and Budzillo, Agata and Baftizadeh, Fahimeh and Miller, Jeremy and Gouwens, Nathan and Arkhipov, Anton and Murphy, Gabe and Tasic, Bosiljka and Zeng, Hongkui and Hawrylycz, Michael and Sümbül, Uygar},
  doi          = {10.1038/s43588-021-00030-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {120-127},
  shortjournal = {Nat. Comput. Sci},
  title        = {Consistent cross-modal identification of cortical neurons with coupled autoencoders},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards practical applications in quantum computational
biology. <em>NATCS</em>, <em>1</em>(2), 114–119. (<a
href="https://doi.org/10.1038/s43588-021-00024-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fascinating progress in understanding our world at the smallest scales moves us to the border of a new technological revolution governed by quantum physics. By taking advantage of quantum phenomena, quantum computing devices allow a speedup in solving diverse tasks. In this Perspective, we discuss the potential impact of quantum computing on computational biology. Bearing in mind the limitations of existing quantum computing devices, we attempt to indicate promising directions for further research in the emerging area of quantum computational biology. Quantum computing has the potential to assist with myriad tasks in science. In this Perspective, the applicability and promising directions of quantum computing in computational biology, genetics and bioinformatics is evaluated and discussed.},
  archive      = {J_NATCS},
  author       = {Fedorov, A. K. and Gelfand, M. S.},
  doi          = {10.1038/s43588-021-00024-z},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {114-119},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards practical applications in quantum computational biology},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The digital revolution of earth-system science.
<em>NATCS</em>, <em>1</em>(2), 104–113. (<a
href="https://doi.org/10.1038/s43588-021-00023-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational science is crucial for delivering reliable weather and climate predictions. However, despite decades of high-performance computing experience, there is serious concern about the sustainability of this application in the post-Moore/Dennard era. Here, we discuss the present limitations in the field and propose the design of a novel infrastructure that is scalable and more adaptable to future, yet unknown computing architectures. There have been substantial developments in weather and climate prediction over the past few decades, attributable to advances in computational science. The rise of new technologies poses challenges to these developments, but also brings opportunities for new progress in the field.},
  archive      = {J_NATCS},
  author       = {Bauer, Peter and Dueben, Peter D. and Hoefler, Torsten and Quintino, Tiago and Schulthess, Thomas C. and Wedi, Nils P.},
  doi          = {10.1038/s43588-021-00023-0},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {104-113},
  shortjournal = {Nat. Comput. Sci},
  title        = {The digital revolution of earth-system science},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Functional change along cellular trajectories.
<em>NATCS</em>, <em>1</em>(2), 102–103. (<a
href="https://doi.org/10.1038/s43588-021-00026-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of genes that are associated with developmental trajectories of cells is an important focus in single-cell transcriptomics. This issue presents Scellnetor, a resource that facilitates this task.},
  archive      = {J_NATCS},
  author       = {Klimm, Florian},
  doi          = {10.1038/s43588-021-00026-x},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {102-103},
  shortjournal = {Nat. Comput. Sci},
  title        = {Functional change along cellular trajectories},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards a consistent neuron classification. <em>NATCS</em>,
<em>1</em>(2), 100–101. (<a
href="https://doi.org/10.1038/s43588-021-00027-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obtaining a consistent taxonomy of neuron types is challenging mainly because of the high dimensionality of the datasets. Coupled autoencoders are a step forward in achieving this goal.},
  archive      = {J_NATCS},
  author       = {Bugeon, Stephane},
  doi          = {10.1038/s43588-021-00027-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {100-101},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards a consistent neuron classification},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantifying the uncertainty of CovidSim. <em>NATCS</em>,
<em>1</em>(2), 98–99. (<a
href="https://doi.org/10.1038/s43588-021-00031-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding and quantifying the uncertainty of predictions from COVID-19 pandemic models is essential to inform public health decision making. This issue presents one such examination using the influential CovidSim model.},
  archive      = {J_NATCS},
  author       = {Leung, Kathy and Wu, Joseph T.},
  doi          = {10.1038/s43588-021-00031-0},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {98-99},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying the uncertainty of CovidSim},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Inspecting treatment benefit in clinical drug trials.
<em>NATCS</em>, <em>1</em>(2), 97. (<a
href="https://doi.org/10.1038/s43588-021-00036-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-021-00036-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {97},
  shortjournal = {Nat. Comput. Sci},
  title        = {Inspecting treatment benefit in clinical drug trials},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Predicting protein structure from cryo-EM data.
<em>NATCS</em>, <em>1</em>(2), 96. (<a
href="https://doi.org/10.1038/s43588-021-00035-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-021-00035-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {96},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting protein structure from cryo-EM data},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021m). Scaling up system size in materials simulation.
<em>NATCS</em>, <em>1</em>(2), 95. (<a
href="https://doi.org/10.1038/s43588-021-00034-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-021-00034-x},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {95},
  shortjournal = {Nat. Comput. Sci},
  title        = {Scaling up system size in materials simulation},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). How community software ecosystems can unlock the potential
of exascale computing. <em>NATCS</em>, <em>1</em>(2), 92–94. (<a
href="https://doi.org/10.1038/s43588-021-00033-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emerging exascale architectures and systems will provide a sizable increase in raw computing power for science. To ensure the full potential of these new and diverse architectures, as well as the longevity and sustainability of science applications, we need to embrace software ecosystems as first-class citizens.},
  archive      = {J_NATCS},
  author       = {McInnes, Lois Curfman and Heroux, Michael A. and Draeger, Erik W. and Siegel, Andrew and Coghlan, Susan and Antypas, Katie},
  doi          = {10.1038/s43588-021-00033-y},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {92-94},
  shortjournal = {Nat. Comput. Sci},
  title        = {How community software ecosystems can unlock the potential of exascale computing},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantum computing opportunities in renewable energy.
<em>NATCS</em>, <em>1</em>(2), 90–91. (<a
href="https://doi.org/10.1038/s43588-021-00032-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Giani, Annarita},
  doi          = {10.1038/s43588-021-00032-z},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {90-91},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantum computing opportunities in renewable energy},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Hail, software! <em>NATCS</em>, <em>1</em>(2), 89. (<a
href="https://doi.org/10.1038/s43588-021-00037-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Software is essential to computational science research, and yet it hasn’t achieved first-class status when it comes to citations. It’s time for all of us in the research community to change this behavior.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-021-00037-8},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {89},
  shortjournal = {Nat. Comput. Sci},
  title        = {Hail, software!},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An efficient and effective method to identify significantly
perturbed subnetworks in cancer. <em>NATCS</em>, <em>1</em>(1), 79–88.
(<a href="https://doi.org/10.1038/s43588-020-00009-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of key functional biological networks from high-dimensional genomics data is pivotal for cancer research. Here, we introduce FDRnet, a method for the detection of molecular subnetworks in cancer, which addresses several challenges in pathway analysis. FDRnet detects key subnetworks by solving a mixed-integer linear programming problem, using a given upper bound of false discovery rate (FDR) as a budget constraint, and minimizing a conductance score to find dense subgraphs around seed genes. A large-scale benchmark study was performed on both simulation and cancer genomics data. FDRnet outperformed other methods in the ability to detect functionally homogeneous subnetworks in a scale-free biological network, to control FDRs of the genes in detected subnetworks, to improve computational efficiency and to integrate multi-omics data. By overcoming the limitations of existing approaches, FDRnet can facilitate the detection of key functional pathways in cancer and other genetic diseases. The FDRnet method addresses several issues in cancer pathway analysis, including those related to detecting functionally homogeneous subnetworks, controlling the false discovery rate of the genes and handling the computational complexity.},
  archive      = {J_NATCS},
  author       = {Yang, Le and Chen, Runpu and Goodison, Steve and Sun, Yijun},
  doi          = {10.1038/s43588-020-00009-4},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {79-88},
  shortjournal = {Nat. Comput. Sci},
  title        = {An efficient and effective method to identify significantly perturbed subnetworks in cancer},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A kinetic ensemble of the alzheimer’s aβ peptide.
<em>NATCS</em>, <em>1</em>(1), 71–78. (<a
href="https://doi.org/10.1038/s43588-020-00003-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The conformational and thermodynamic properties of disordered proteins are commonly described in terms of structural ensembles and free energy landscapes. To provide information on the transition rates between the different states populated by these proteins, it would be desirable to generalize this description to kinetic ensembles. Approaches based on the theory of stochastic processes can be particularly suitable for this purpose. Here, we develop a Markov state model and apply it to determine a kinetic ensemble of Aβ42, a disordered peptide associated with Alzheimer’s disease. Through the Google Compute Engine, we generated 315-µs all-atom molecular dynamics trajectories. Using a probabilistic-based definition of conformational states in a neural network approach, we found that Aβ42 is characterized by inter-state transitions on the microsecond timescale, exhibiting only fully unfolded or short-lived, partially folded states. Our results illustrate how kinetic ensembles provide effective information about the structure, thermodynamics and kinetics of disordered proteins. An approach based on soft Markov state models is capable of constructing interpretable kinetic ensembles of disordered proteins.},
  archive      = {J_NATCS},
  author       = {Löhr, Thomas and Kohlhoff, Kai and Heller, Gabriella T. and Camilloni, Carlo and Vendruscolo, Michele},
  doi          = {10.1038/s43588-020-00003-w},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {71-78},
  shortjournal = {Nat. Comput. Sci},
  title        = {A kinetic ensemble of the alzheimer’s aβ peptide},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Stable maintenance of hidden switches as a strategy to
increase the gene expression stability. <em>NATCS</em>, <em>1</em>(1),
62–70. (<a href="https://doi.org/10.1038/s43588-020-00001-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In response to severe genetic and environmental perturbations, wild-type organisms can express hidden alternative phenotypes adaptive to such adverse conditions. While our theoretical understanding of the population-level fitness advantage and evolution of phenotypic switching under variable environments has grown, the mechanism by which these organisms maintain phenotypic switching capabilities under static environments remains to be elucidated. Here, using computational simulations, we analyzed the evolution of gene circuits under natural selection and found that different strategies evolved to increase the gene expression stability near the optimum level. In a population comprising bistable individuals, a strategy of maintaining bistability and raising the potential barrier separating the bistable regimes was consistently taken. Our results serve as evidence that hidden bistable switches can be stably maintained during environmental stasis—an essential property enabling the timely release of adaptive alternatives with small genetic changes in the event of substantial perturbations. Computational simulations show that selection for high gene expression stability can explain the stable maintenance of obsolete phenotypic switching capabilities under natural selection.},
  archive      = {J_NATCS},
  author       = {Kuwahara, Hiroyuki and Gao, Xin},
  doi          = {10.1038/s43588-020-00001-y},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {62-70},
  shortjournal = {Nat. Comput. Sci},
  title        = {Stable maintenance of hidden switches as a strategy to increase the gene expression stability},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Accelerating computational modeling and design of
high-entropy alloys. <em>NATCS</em>, <em>1</em>(1), 54–61. (<a
href="https://doi.org/10.1038/s43588-020-00006-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-entropy alloys, with N elements and compositions {cν = 1,N} in competing crystal structures, have large design spaces for unique chemical and mechanical properties. Here, to enable computational design, we use a metaheuristic hybrid Cuckoo search (CS) to construct alloy configurational models on the fly that have targeted atomic site and pair probabilities on arbitrary crystal lattices, given by supercell random approximates (SCRAPs) with S sites. Our Hybrid CS permits efficient global solutions for large, discrete combinatorial optimization that scale linearly in a number of parallel processors, and linearly in sites S for SCRAPs. For example, a four-element, 128-site SCRAP is found in seconds—a more than 13,000-fold reduction over current strategies. Our method thus enables computational alloy design that is currently impractical. We qualify the models and showcase application to real alloys with targeted atomic short-range order. Being problem-agnostic, our Hybrid CS offers potential applications in diverse fields. An evolutionary-based algorithm enables modeling of complex solid-solution alloys over exponential search spaces in practical time, accelerating materials design of such high-entropy alloys.},
  archive      = {J_NATCS},
  author       = {Singh, Rahul and Sharma, Aayush and Singh, Prashant and Balasubramanian, Ganesh and Johnson, Duane D.},
  doi          = {10.1038/s43588-020-00006-7},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {54-61},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating computational modeling and design of high-entropy alloys},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Learning properties of ordered and disordered materials from
multi-fidelity data. <em>NATCS</em>, <em>1</em>(1), 46–53. (<a
href="https://doi.org/10.1038/s43588-020-00002-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the properties of a material from the arrangement of its atoms is a fundamental goal in materials science. While machine learning has emerged in recent years as a new paradigm to provide rapid predictions of materials properties, their practical utility is limited by the scarcity of high-fidelity data. Here, we develop multi-fidelity graph networks as a universal approach to achieve accurate predictions of materials properties with small data sizes. As a proof of concept, we show that the inclusion of low-fidelity Perdew–Burke–Ernzerhof band gaps greatly enhances the resolution of latent structural features in materials graphs, leading to a 22–45\% decrease in the mean absolute errors of experimental band gap predictions. We further demonstrate that learned elemental embeddings in materials graph networks provide a natural approach to model disorder in materials, addressing a fundamental gap in the computational prediction of materials properties. Multi-fidelity graph networks learn more effective representations for materials from large data sets of low-fidelity properties, which can then be used to make accurate predictions of high-fidelity properties, such as the band gaps of ordered and disordered crystals and energies of molecules.},
  archive      = {J_NATCS},
  author       = {Chen, Chi and Zuo, Yunxing and Ye, Weike and Li, Xiangguo and Ong, Shyue Ping},
  doi          = {10.1038/s43588-020-00002-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {46-53},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning properties of ordered and disordered materials from multi-fidelity data},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Voice-controlled quantum chemistry. <em>NATCS</em>,
<em>1</em>(1), 42–45. (<a
href="https://doi.org/10.1038/s43588-020-00012-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past decade, artificial intelligence has been propelled forward by advances in machine learning algorithms and computational hardware, opening up myriads of new avenues for scientific research. Nevertheless, virtual assistants and voice control have yet to be widely used in the natural sciences. Here, we present ChemVox, an interactive Amazon Alexa skill that uses speech recognition to perform quantum chemistry calculations. This new application interfaces Alexa with cloud computing and returns the results through a capable device. ChemVox paves the way to making computational chemistry routinely accessible to the wider community. Using voice-based technologies, ChemVox is able to answer quantum chemistry questions in seconds, thus making such complex questions more accessible to the community.},
  archive      = {J_NATCS},
  author       = {Raucci, Umberto and Valentini, Alessio and Pieri, Elisa and Weir, Hayley and Seritan, Stefan and Martínez, Todd J.},
  doi          = {10.1038/s43588-020-00012-9},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {42-45},
  shortjournal = {Nat. Comput. Sci},
  title        = {Voice-controlled quantum chemistry},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Lessons from the COVID-19 pandemic for advancing
computational drug repurposing strategies. <em>NATCS</em>,
<em>1</em>(1), 33–41. (<a
href="https://doi.org/10.1038/s43588-020-00007-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Responding quickly to unknown pathogens is crucial to stop uncontrolled spread of diseases that lead to epidemics, such as the novel coronavirus, and to keep protective measures at a level that causes as little social and economic harm as possible. This can be achieved through computational approaches that significantly speed up drug discovery. A powerful approach is to restrict the search to existing drugs through drug repurposing, which can vastly accelerate the usually long approval process. In this Review, we examine a representative set of currently used computational approaches to identify repurposable drugs for COVID-19, as well as their underlying data resources. Furthermore, we compare drug candidates predicted by computational methods to drugs being assessed by clinical trials. Finally, we discuss lessons learned from the reviewed research efforts, including how to successfully connect computational approaches with experimental studies, and propose a unified drug repurposing strategy for better preparedness in the case of future outbreaks. Computational approaches for drug repurposing can accelerate the identification of treatments during a pandemic. In this Review, the authors discuss this topic in the context of COVID-19 and propose a strategy to make computational drug repurposing more effective in future pandemics.},
  archive      = {J_NATCS},
  author       = {Galindez, Gihanna and Matschinske, Julian and Rose, Tim Daniel and Sadegh, Sepideh and Salgado-Albarrán, Marisol and Späth, Julian and Baumbach, Jan and Pauling, Josch Konstantin},
  doi          = {10.1038/s43588-020-00007-6},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {33-41},
  shortjournal = {Nat. Comput. Sci},
  title        = {Lessons from the COVID-19 pandemic for advancing computational drug repurposing strategies},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantifying causality in data science with
quasi-experiments. <em>NATCS</em>, <em>1</em>(1), 24–32. (<a
href="https://doi.org/10.1038/s43588-020-00005-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimating causality from observational data is essential in many data science questions but can be a challenging task. Here we review approaches to causality that are popular in econometrics and that exploit (quasi) random variation in existing data, called quasi-experiments, and show how they can be combined with machine learning to answer causal questions within typical data science settings. We also highlight how data scientists can help advance these methods to bring causal estimation to high-dimensional data from medicine, industry and society. While estimating causality from observational data is challenging, quasi-experiments provide causal inference methods with plausible assumptions that can be practical to a range of real-world problems.},
  archive      = {J_NATCS},
  author       = {Liu, Tony and Ungar, Lyle and Kording, Konrad},
  doi          = {10.1038/s43588-020-00005-8},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {24-32},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying causality in data science with quasi-experiments},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Redefining false discoveries in cancer data analyses.
<em>NATCS</em>, <em>1</em>(1), 22–23. (<a
href="https://doi.org/10.1038/s43588-020-00008-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The nature of biological networks still brings challenges related to computational complexity, interpretable results and statistical significance. Recent work proposes a new method that paves the way for addressing these issues when analyzing cancer genomic data.},
  archive      = {J_NATCS},
  author       = {Najgebauer, Hanna and Perron, Umberto and Iorio, Francesco},
  doi          = {10.1038/s43588-020-00008-5},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {22-23},
  shortjournal = {Nat. Comput. Sci},
  title        = {Redefining false discoveries in cancer data analyses},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Kinetics of amyloid β from deep learning. <em>NATCS</em>,
<em>1</em>(1), 20–21. (<a
href="https://doi.org/10.1038/s43588-020-00010-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing the aggregation of the peptide amyloid β is essential to better understand Alzheimer’s disease and to find potential targets for drug development. Deep neural networks make it possible to describe the kinetics of this peptide, opening the way for achieving this goal.},
  archive      = {J_NATCS},
  author       = {Meng, Fanjie and Chung, Hoi Sung},
  doi          = {10.1038/s43588-020-00010-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {20-21},
  shortjournal = {Nat. Comput. Sci},
  title        = {Kinetics of amyloid β from deep learning},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Discovering evolutionary hidden treasures. <em>NATCS</em>,
<em>1</em>(1), 18–19. (<a
href="https://doi.org/10.1038/s43588-020-00004-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms facilitating evolutionary adaptation to future challenges are difficult to establish experimentally. Recent computational simulations of 200 cell populations indicate how evolution can hide useless genetic switches with capacity for later use.},
  archive      = {J_NATCS},
  author       = {Balázsi, Gábor},
  doi          = {10.1038/s43588-020-00004-9},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {18-19},
  shortjournal = {Nat. Comput. Sci},
  title        = {Discovering evolutionary hidden treasures},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Modeling higher-order lattice anharmonicity.
<em>NATCS</em>, <em>1</em>(1), 17. (<a
href="https://doi.org/10.1038/s43588-020-00019-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-020-00019-2},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {17},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling higher-order lattice anharmonicity},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Parameter inference for systems biology models.
<em>NATCS</em>, <em>1</em>(1), 16. (<a
href="https://doi.org/10.1038/s43588-020-00020-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-020-00020-9},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {16},
  shortjournal = {Nat. Comput. Sci},
  title        = {Parameter inference for systems biology models},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Characterizing true quantum computing power.
<em>NATCS</em>, <em>1</em>(1), 15. (<a
href="https://doi.org/10.1038/s43588-020-00018-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-020-00018-3},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {15},
  shortjournal = {Nat. Comput. Sci},
  title        = {Characterizing true quantum computing power},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Computationally designing synthesizable molecules.
<em>NATCS</em>, <em>1</em>(1), 14. (<a
href="https://doi.org/10.1038/s43588-020-00017-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-020-00017-4},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {14},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computationally designing synthesizable molecules},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Connecting black women in computational biology.
<em>NATCS</em>, <em>1</em>(1), 11–13. (<a
href="https://doi.org/10.1038/s43588-020-00013-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Jenea Adams, a second year PhD student, found it hard to connect with other Black women in her field of study. She then decided to create the Black Women in Computational Biology Network, which has attracted the support of many researchers. We spoke to her about the Network, as well as gender and racial inequality.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando and Rastogi, Ananya},
  doi          = {10.1038/s43588-020-00013-8},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {11-13},
  shortjournal = {Nat. Comput. Sci},
  title        = {Connecting black women in computational biology},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Data curation during a pandemic and lessons learned from
COVID-19. <em>NATCS</em>, <em>1</em>(1), 9–10. (<a
href="https://doi.org/10.1038/s43588-020-00015-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detailed, accurate data related to a disease outbreak enable informed public health decision making. Given the variety of data types available across different regions, global data curation and standardization efforts are essential to guarantee rapid data integration and dissemination in times of a pandemic.},
  archive      = {J_NATCS},
  author       = {Kraemer, Moritz U. G. and Scarpino, Samuel V. and Marivate, Vukosi and Gutierrez, Bernardo and Xu, Bo and Lee, Graham and Hawkins, Jared B. and Rivers, Caitlin and Pigott, David M. and Katz, Rebecca and Brownstein, John S.},
  doi          = {10.1038/s43588-020-00015-6},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {9-10},
  shortjournal = {Nat. Comput. Sci},
  title        = {Data curation during a pandemic and lessons learned from COVID-19},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The importance of local context in COVID-19 models.
<em>NATCS</em>, <em>1</em>(1), 6–8. (<a
href="https://doi.org/10.1038/s43588-020-00014-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {COVID-19 models have been extensively used to inform public health officials about potential interventions. Nevertheless, careful attention must be taken when extrapolating projections and parameters across different regions, as there is no one-size-fits-all model for the pandemic.},
  archive      = {J_NATCS},
  author       = {Eggo, Rosalind M. and Dawa, Jeanette and Kucharski, Adam J. and Cucunuba, Zulma M.},
  doi          = {10.1038/s43588-020-00014-7},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {6-8},
  shortjournal = {Nat. Comput. Sci},
  title        = {The importance of local context in COVID-19 models},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Promises and perils of computational materials databases.
<em>NATCS</em>, <em>1</em>(1), 3–5. (<a
href="https://doi.org/10.1038/s43588-020-00016-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past decade, the materials science community has fostered the development of materials databases from high-performance computation. While these databases have achieved great success, there are still several challenges to be addressed for the community to realize the full potential of the materials-by-design era.},
  archive      = {J_NATCS},
  author       = {Horton, M. K. and Dwaraknath, S. and Persson, K. A.},
  doi          = {10.1038/s43588-020-00016-5},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {3-5},
  shortjournal = {Nat. Comput. Sci},
  title        = {Promises and perils of computational materials databases},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Open source ecosystems need equitable credit across
contributions. <em>NATCS</em>, <em>1</em>(1), 2. (<a
href="https://doi.org/10.1038/s43588-020-00011-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Casari, Amanda and McLaughlin, Katie and Trujillo, Milo Z. and Young, Jean-Gabriel and Bagrow, James P. and Hébert-Dufresne, Laurent},
  doi          = {10.1038/s43588-020-00011-w},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Comput. Sci},
  title        = {Open source ecosystems need equitable credit across contributions},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Celebrating today, inspiring tomorrow. <em>NATCS</em>,
<em>1</em>(1), 1. (<a
href="https://doi.org/10.1038/s43588-020-00021-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational and mathematical models are at the core of myriad research developments across different domains. We pay tribute to the importance of computational science by providing a dedicated home among the Nature portfolio for this inspiring field.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-020-00021-8},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Comput. Sci},
  title        = {Celebrating today, inspiring tomorrow},
  volume       = {1},
  year         = {2021},
}
</textarea>
</details></li>
</ul>

</body>
</html>
