<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt---30">NBT - 30</h2>
<ul>
<li><details>
<summary>
(2025). Vertex’s opioid-free drug for acute pain wins FDA approval.
<em>NBT</em>, <em>43</em>(3), 287–289. (<a
href="https://doi.org/10.1038/s41587-025-02590-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Society, patients and clinicians welcome a much-needed non-opioid pain medication. Vertex’s first-in-class analgesic Journavx could soon be followed by a new generation of addiction-free pain drugs acting at NaV1.8 sodium channels.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-025-02590-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {287-289},
  shortjournal = {Nature Biotech.},
  title        = {Vertex’s opioid-free drug for acute pain wins FDA approval},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(2), 284. (<a
href="https://doi.org/10.1038/s41587-025-02557-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02557-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {284},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Undergraduate education as catalyst for public–private
partnership in biotechnology. <em>NBT</em>, <em>43</em>(2), 282–283. (<a
href="https://doi.org/10.1038/s41587-025-02552-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An innovative challenge-based and research-driven educational model immerses groups of undergraduate students in the discovery stages of biotech industry research.},
  archive      = {J_NBT},
  author       = {ten Broeke, Toine and Koorman, Thijs and Crnko, Sandra and Voets, Erik and de Goeje, Pauline and Schakelaar, Michael Y. and Rebel, Heggert G. and Pijnappel, Emma W. and Sanders, Jorine G. F. and Meeldijk, Jan and Terveer-Couperus, Rian and Geven, Esther L. and Dilaver, Gönül and van Tilborg, Erik and Beekman, Lucas M. P. and Guelen, Lars and Bovenschen, Niels},
  doi          = {10.1038/s41587-025-02552-9},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {282-283},
  shortjournal = {Nature Biotech.},
  title        = {Undergraduate education as catalyst for public–private partnership in biotechnology},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: The global patent landscape of
functional food innovation. <em>NBT</em>, <em>43</em>(2), 281. (<a
href="https://doi.org/10.1038/s41587-024-02544-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Matin, Maima and Hrg, Dalibor and Litvinova, Olena and Łysek-Gładysinska, Małgorzata and Wierzbicka, Agnieszka and Horbańczuk, Jarosław Olav and Jóźwik, Artur and Atanasov, Atanas G.},
  doi          = {10.1038/s41587-024-02544-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {281},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: The global patent landscape of functional food innovation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The covariance environment defines cellular niches for
spatial inference. <em>NBT</em>, <em>43</em>(2), 269–280. (<a
href="https://doi.org/10.1038/s41587-024-02193-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key challenge of analyzing data from high-resolution spatial profiling technologies is to suitably represent the features of cellular neighborhoods or niches. Here we introduce the covariance environment (COVET), a representation that leverages the gene–gene covariate structure across cells in the niche to capture the multivariate nature of cellular interactions within it. We define a principled optimal transport-based distance metric between COVET niches that scales to millions of cells. Using COVET to encode spatial context, we developed environmental variational inference (ENVI), a conditional variational autoencoder that jointly embeds spatial and single-cell RNA sequencing data into a latent space. ENVI includes two decoders: one to impute gene expression across the spatial modality and a second to project spatial information onto single-cell data. ENVI can confer spatial context to genomics data from single dissociated cells and outperforms alternatives for imputing gene expression on diverse spatial datasets. Using gene–gene covariance structure to represent cellular neighborhoods improves the integration of single-cell RNA sequencing and spatial transcriptomics data.},
  archive      = {J_NBT},
  author       = {Haviv, Doron and Remšík, Ján and Gatie, Mohamed and Snopkowski, Catherine and Takizawa, Meril and Pereira, Nathan and Bashkin, John and Jovanovich, Stevan and Nawy, Tal and Chaligne, Ronan and Boire, Adrienne and Hadjantonakis, Anna-Katerina and Pe’er, Dana},
  doi          = {10.1038/s41587-024-02193-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {269-280},
  shortjournal = {Nature Biotech.},
  title        = {The covariance environment defines cellular niches for spatial inference},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gene trajectory inference for single-cell data by optimal
transport metrics. <em>NBT</em>, <em>43</em>(2), 258–268. (<a
href="https://doi.org/10.1038/s41587-024-02186-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing has been widely used to investigate cell state transitions and gene dynamics of biological processes. Current strategies to infer the sequential dynamics of genes in a process typically rely on constructing cell pseudotime through cell trajectory inference. However, the presence of concurrent gene processes in the same group of cells and technical noise can obscure the true progression of the processes studied. To address this challenge, we present GeneTrajectory, an approach that identifies trajectories of genes rather than trajectories of cells. Specifically, optimal transport distances are calculated between gene distributions across the cell–cell graph to extract gene programs and define their gene pseudotemporal order. Here we demonstrate that GeneTrajectory accurately extracts progressive gene dynamics in myeloid lineage maturation. Moreover, we show that GeneTrajectory deconvolves key gene programs underlying mouse skin hair follicle dermal condensate differentiation that could not be resolved by cell trajectory approaches. GeneTrajectory facilitates the discovery of gene programs that control the changes and activities of biological processes. Gene dynamics of concurrent biological processes are unraveled by considering gene trajectories instead of cell trajectories.},
  archive      = {J_NBT},
  author       = {Qu, Rihao and Cheng, Xiuyuan and Sefik, Esen and Stanley III, Jay S. and Landa, Boris and Strino, Francesco and Platt, Sarah and Garritano, James and Odell, Ian D. and Coifman, Ronald and Flavell, Richard A. and Myung, Peggy and Kluger, Yuval},
  doi          = {10.1038/s41587-024-02186-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {258-268},
  shortjournal = {Nature Biotech.},
  title        = {Gene trajectory inference for single-cell data by optimal transport metrics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inferring gene regulatory networks from single-cell multiome
data using atlas-scale external data. <em>NBT</em>, <em>43</em>(2),
247–257. (<a href="https://doi.org/10.1038/s41587-024-02182-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing methods for gene regulatory network (GRN) inference rely on gene expression data alone or on lower resolution bulk data. Despite the recent integration of chromatin accessibility and RNA sequencing data, learning complex mechanisms from limited independent data points still presents a daunting challenge. Here we present LINGER (Lifelong neural network for gene regulation), a machine-learning method to infer GRNs from single-cell paired gene expression and chromatin accessibility data. LINGER incorporates atlas-scale external bulk data across diverse cellular contexts and prior knowledge of transcription factor motifs as a manifold regularization. LINGER achieves a fourfold to sevenfold relative increase in accuracy over existing methods and reveals a complex regulatory landscape of genome-wide association studies, enabling enhanced interpretation of disease-associated variants and genes. Following the GRN inference from reference single-cell multiome data, LINGER enables the estimation of transcription factor activity solely from bulk or single-cell gene expression data, leveraging the abundance of available gene expression data to identify driver regulators from case-control studies. Accuracy of gene regulatory network inference is increased by combining multiome single-cell and atlas-scale bulk data.},
  archive      = {J_NBT},
  author       = {Yuan, Qiuyue and Duren, Zhana},
  doi          = {10.1038/s41587-024-02182-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {247-257},
  shortjournal = {Nature Biotech.},
  title        = {Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Therapeutic application of circular RNA aptamers in a mouse
model of psoriasis. <em>NBT</em>, <em>43</em>(2), 236–246. (<a
href="https://doi.org/10.1038/s41587-024-02204-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efforts to advance RNA aptamers as a new therapeutic modality have been limited by their susceptibility to degradation and immunogenicity. In a previous study, we demonstrated synthesized short double-stranded region-containing circular RNAs (ds-cRNAs) with minimal immunogenicity targeted to dsRNA-activated protein kinase R (PKR). Here we test the therapeutic potential of ds-cRNAs in a mouse model of imiquimod-induced psoriasis. We find that genetic supplementation of ds-cRNAs leads to inhibition of PKR, resulting in alleviation of downstream interferon-α and dsRNA signals and attenuation of psoriasis phenotypes. Delivery of ds-cRNAs by lipid nanoparticles to the spleen attenuates PKR activity in examined splenocytes, resulting in reduced epidermal thickness. These findings suggest that ds-cRNAs represent a promising approach to mitigate excessive PKR activation for therapeutic purposes. A circular RNA aptamer shows therapeutic efficacy in vivo in a psoriasis model.},
  archive      = {J_NBT},
  author       = {Guo, Si-Kun and Liu, Chu-Xiao and Xu, Yi-Feng and Wang, Xiao and Nan, Fang and Huang, Youkui and Li, Siqi and Nan, Shan and Li, Ling and Kon, Edo and Li, Chen and Wei, Meng-Yuan and Su, Rina and Wei, Jia and Peng, Shiguang and Ad-El, Nitay and Liu, Jiaquan and Peer, Dan and Chen, Ting and Yang, Li and Chen, Ling-Ling},
  doi          = {10.1038/s41587-024-02204-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {236-246},
  shortjournal = {Nature Biotech.},
  title        = {Therapeutic application of circular RNA aptamers in a mouse model of psoriasis},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Starfysh integrates spatial transcriptomic and histologic
data to reveal heterogeneous tumor–immune hubs. <em>NBT</em>,
<em>43</em>(2), 223–235. (<a
href="https://doi.org/10.1038/s41587-024-02173-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved gene expression profiling provides insight into tissue organization and cell–cell crosstalk; however, sequencing-based spatial transcriptomics (ST) lacks single-cell resolution. Current ST analysis methods require single-cell RNA sequencing data as a reference for rigorous interpretation of cell states, mostly do not use associated histology images and are not capable of inferring shared neighborhoods across multiple tissues. Here we present Starfysh, a computational toolbox using a deep generative model that incorporates archetypal analysis and any known cell type markers to characterize known or new tissue-specific cell states without a single-cell reference. Starfysh improves the characterization of spatial dynamics in complex tissues using histology images and enables the comparison of niches as spatial hubs across tissues. Integrative analysis of primary estrogen receptor (ER)-positive breast cancer, triple-negative breast cancer (TNBC) and metaplastic breast cancer (MBC) tissues led to the identification of spatial hubs with patient- and disease-specific cell type compositions and revealed metabolic reprogramming shaping immunosuppressive hubs in aggressive MBC. Reference-free integrative analysis of spatial transcriptomics data is achieved with archetype analysis.},
  archive      = {J_NBT},
  author       = {He, Siyu and Jin, Yinuo and Nazaret, Achille and Shi, Lingting and Chen, Xueer and Rampersaud, Sham and Dhillon, Bahawar S. and Valdez, Izabella and Friend, Lauren E. and Fan, Joy Linyue and Park, Cameron Y. and Mintz, Rachel L. and Lao, Yeh-Hsing and Carrera, David and Fang, Kaylee W. and Mehdi, Kaleem and Rohde, Madeline and McFaline-Figueroa, José L. and Blei, David and Leong, Kam W. and Rudensky, Alexander Y. and Plitas, George and Azizi, Elham},
  doi          = {10.1038/s41587-024-02173-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {223-235},
  shortjournal = {Nature Biotech.},
  title        = {Starfysh integrates spatial transcriptomic and histologic data to reveal heterogeneous tumor–immune hubs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Discovery of tumor-reactive t cell receptors by massively
parallel library synthesis and screening. <em>NBT</em>, <em>43</em>(2),
214–222. (<a href="https://doi.org/10.1038/s41587-024-02210-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cell receptor (TCR) gene therapy is a potent form of cellular immunotherapy in which patient T cells are genetically engineered to express TCRs with defined tumor reactivity. However, the isolation of therapeutic TCRs is complicated by both the general scarcity of tumor-specific T cells among patient T cell repertoires and the patient-specific nature of T cell epitopes expressed on tumors. Here we describe a high-throughput, personalized TCR discovery pipeline that enables the assembly of complex synthetic TCR libraries in a one-pot reaction, followed by pooled expression in reporter T cells and functional genetic screening against patient-derived tumor or antigen-presenting cells. We applied the method to screen thousands of tumor-infiltrating lymphocyte (TIL)-derived TCRs from multiple patients and identified dozens of CD4+ and CD8+ T-cell-derived TCRs with potent tumor reactivity, including TCRs that recognized patient-specific neoantigens. Tumor-specific T cell receptors (TCRs) are identified by functional screening of synthetic TCR libraries.},
  archive      = {J_NBT},
  author       = {Moravec, Ziva and Zhao, Yue and Voogd, Rhianne and Cook, Danielle R. and Kinrot, Seon and Capra, Benjamin and Yang, Haiyan and Raud, Brenda and Ou, Jiayu and Xuan, Jiekun and Wei, Teng and Ren, Lili and Hu, Dandan and Wang, Jun and Haanen, John B.A.G. and Schumacher, Ton N. and Chen, Xi and Porter, Ely and Scheper, Wouter},
  doi          = {10.1038/s41587-024-02210-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {214-222},
  shortjournal = {Nature Biotech.},
  title        = {Discovery of tumor-reactive t cell receptors by massively parallel library synthesis and screening},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome engineering with cas9 and AAV repair templates
generates frequent concatemeric insertions of viral vectors.
<em>NBT</em>, <em>43</em>(2), 204–213. (<a
href="https://doi.org/10.1038/s41587-024-02171-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas9 paired with adeno-associated virus serotype 6 (AAV6) is among the most efficient tools for producing targeted gene knockins. Here, we report that this system can lead to frequent concatemeric insertions of the viral vector genome at the target site that are difficult to detect. Such errors can cause adverse and unreliable phenotypes that are antithetical to the goal of precision genome engineering. The concatemeric knockins occurred regardless of locus, vector concentration, cell line or cell type, including human pluripotent and hematopoietic stem cells. Although these highly abundant errors were found in more than half of the edited cells, they could not be readily detected by common analytical methods. We describe strategies to detect and thoroughly characterize the concatemeric viral vector insertions, and we highlight analytical pitfalls that mask their prevalence. We then describe strategies to prevent the concatemeric inserts by cutting the vector genome after transduction. This approach is compatible with established gene editing pipelines, enabling robust genetic knockins that are safer, more reliable and more reproducible. AAV vectors form difficult-to-detect concatemers at Cas9 target sites.},
  archive      = {J_NBT},
  author       = {Suchy, Fabian P. and Karigane, Daiki and Nakauchi, Yusuke and Higuchi, Maimi and Zhang, Jinyu and Pekrun, Katja and Hsu, Ian and Fan, Amy C. and Nishimura, Toshinobu and Charlesworth, Carsten T. and Bhadury, Joydeep and Nishimura, Toshiya and Wilkinson, Adam C. and Kay, Mark A. and Majeti, Ravindra and Nakauchi, Hiromitsu},
  doi          = {10.1038/s41587-024-02171-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {204-213},
  shortjournal = {Nature Biotech.},
  title        = {Genome engineering with cas9 and AAV repair templates generates frequent concatemeric insertions of viral vectors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Branched chemically modified poly(a) tails enhance the
translation capacity of mRNA. <em>NBT</em>, <em>43</em>(2), 194–203. (<a
href="https://doi.org/10.1038/s41587-024-02174-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although messenger RNA (mRNA) has proved effective as a vaccine, its potential as a general therapeutic modality is limited by its instability and low translation capacity. To increase the duration and level of protein expression from mRNA, we designed and synthesized topologically and chemically modified mRNAs with multiple synthetic poly(A) tails. Here we demonstrate that the optimized multitailed mRNA yielded ~4.7–19.5-fold higher luminescence signals than the control mRNA from 24 to 72 h post transfection in cellulo and 14 days detectable signal versus &amp;lt;7 days signal from the control in vivo. We further achieve efficient multiplexed genome editing of the clinically relevant genes Pcsk9 and Angptl3 in mouse liver at a minimal mRNA dosage. Taken together, these results provide a generalizable approach to synthesize capped branched mRNA with markedly enhanced translation capacity. mRNA with engineered poly(A) tails produces prolonged higher levels of protein.},
  archive      = {J_NBT},
  author       = {Chen, Hongyu and Liu, Dangliang and Guo, Jianting and Aditham, Abhishek and Zhou, Yiming and Tian, Jiakun and Luo, Shuchen and Ren, Jingyi and Hsu, Alvin and Huang, Jiahao and Kostas, Franklin and Wu, Mingrui and Liu, David R. and Wang, Xiao},
  doi          = {10.1038/s41587-024-02174-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {194-203},
  shortjournal = {Nature Biotech.},
  title        = {Branched chemically modified poly(A) tails enhance the translation capacity of mRNA},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of supramolecular anticoagulants with on-demand
reversibility. <em>NBT</em>, <em>43</em>(2), 186–193. (<a
href="https://doi.org/10.1038/s41587-024-02209-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drugs are administered at a dosing schedule set by their therapeutic index, and termination of action is achieved by clearance and metabolism of the drug. In some cases, such as anticoagulant drugs or immunotherapeutics, it is important to be able to quickly reverse the drug’s action. Here, we report a general strategy to achieve on-demand reversibility by designing a supramolecular drug (a noncovalent assembly of two cooperatively interacting drug fragments held together by transient hybridization of peptide nucleic acid (PNA)) that can be reversed with a PNA antidote that outcompetes the hybridization between the fragments. We demonstrate the approach with thrombin-inhibiting anticoagulants, creating very potent and reversible bivalent direct thrombin inhibitors (Ki = 74 pM). The supramolecular inhibitor effectively inhibited thrombus formation in mice in a needle injury thrombosis model, and this activity could be reversed by administration of the PNA antidote. This design is applicable to therapeutic targets where two binding sites can be identified. A supramolecular drug design yields anticoagulants that are rapidly reversible.},
  archive      = {J_NBT},
  author       = {Dockerill, Millicent and Ford, Daniel J. and Angerani, Simona and Alwis, Imala and Dowman, Luke J. and Ripoll-Rozada, Jorge and Smythe, Rhyll E. and Liu, Joanna S. T. and Pereira, Pedro José Barbosa and Jackson, Shaun P. and Payne, Richard J. and Winssinger, Nicolas},
  doi          = {10.1038/s41587-024-02209-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {186-193},
  shortjournal = {Nature Biotech.},
  title        = {Development of supramolecular anticoagulants with on-demand reversibility},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). De novo and somatic structural variant discovery with
SVision-pro. <em>NBT</em>, <em>43</em>(2), 181–185. (<a
href="https://doi.org/10.1038/s41587-024-02190-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read-based de novo and somatic structural variant (SV) discovery remains challenging, necessitating genomic comparison between samples. We developed SVision-pro, a neural-network-based instance segmentation framework that represents genome-to-genome-level sequencing differences visually and discovers SV comparatively between genomes without any prerequisite for inference models. SVision-pro outperforms state-of-the-art approaches, in particular, the resolving of complex SVs is improved, with low Mendelian error rates, high sensitivity of low-frequency SVs and reduced false-positive rates compared with SV merging approaches. SVision-pro enables pairwise genome comparison and improves de novo and somatic structural variant calling.},
  archive      = {J_NBT},
  author       = {Wang, Songbo and Lin, Jiadong and Jia, Peng and Xu, Tun and Li, Xiujuan and Liu, Yuezhuangnan and Xu, Dan and Bush, Stephen J. and Meng, Deyu and Ye, Kai},
  doi          = {10.1038/s41587-024-02190-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {181-185},
  shortjournal = {Nature Biotech.},
  title        = {De novo and somatic structural variant discovery with SVision-pro},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capturing and modeling cellular niches from dissociated
single-cell and spatial data. <em>NBT</em>, <em>43</em>(2), 179–180. (<a
href="https://doi.org/10.1038/s41587-024-02207-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells interact with their local environment to enact global tissue function. By harnessing gene–gene covariation in cellular neighborhoods from spatial transcriptomics data, the covariance environment (COVET) niche representation and the environmental variational inference (ENVI) data integration method model phenotype–microenvironment interplay and reconstruct the spatial context of dissociated single-cell RNA sequencing datasets.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02207-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {179-180},
  shortjournal = {Nature Biotech.},
  title        = {Capturing and modeling cellular niches from dissociated single-cell and spatial data},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Designing drugs with reversible activity. <em>NBT</em>,
<em>43</em>(2), 177–178. (<a
href="https://doi.org/10.1038/s41587-024-02242-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A strategy for creating drugs that can be quickly neutralized is demonstrated for anticoagulants.},
  archive      = {J_NBT},
  author       = {Neri, Dario},
  doi          = {10.1038/s41587-024-02242-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {177-178},
  shortjournal = {Nature Biotech.},
  title        = {Designing drugs with reversible activity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A chromatin-based model for deciphering gene interactions.
<em>NBT</em>, <em>43</em>(2), 176. (<a
href="https://doi.org/10.1038/s41587-025-02573-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Li, Rong},
  doi          = {10.1038/s41587-025-02573-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nature Biotech.},
  title        = {A chromatin-based model for deciphering gene interactions},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Heat-resistant rice without yield compromise. <em>NBT</em>,
<em>43</em>(2), 176. (<a
href="https://doi.org/10.1038/s41587-025-02574-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Despang, Alexandra},
  doi          = {10.1038/s41587-025-02574-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nature Biotech.},
  title        = {Heat-resistant rice without yield compromise},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). De novo-designed protein binders neutralize snake toxins.
<em>NBT</em>, <em>43</em>(2), 176. (<a
href="https://doi.org/10.1038/s41587-025-02575-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bishop, Anahita},
  doi          = {10.1038/s41587-025-02575-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nature Biotech.},
  title        = {De novo-designed protein binders neutralize snake toxins},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Antibodies. <em>NBT</em>, <em>43</em>(2), 175. (<a
href="https://doi.org/10.1038/s41587-024-02547-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to antibody compositions and methods of use.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02547-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {175},
  shortjournal = {Nature Biotech.},
  title        = {Antibodies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining antibody patent protection using paratope mapping
and CDR-scanning. <em>NBT</em>, <em>43</em>(2), 170–174. (<a
href="https://doi.org/10.1038/s41587-024-02549-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patent protection for therapeutic monoclonal antibodies requires a fundamental shift in strategy following Amgen v. Sanofi, which invalidated broad functional claims for antibody genera in the USA.},
  archive      = {J_NBT},
  author       = {Banik, Soma S. R. and Deng, Xiaoxiang and Davidson, Edgar and Storz, Ulrich and Doranz, Benjamin J.},
  doi          = {10.1038/s41587-024-02549-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {170-174},
  shortjournal = {Nature Biotech.},
  title        = {Redefining antibody patent protection using paratope mapping and CDR-scanning},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A platform for the biomedical application of large language
models. <em>NBT</em>, <em>43</em>(2), 166–169. (<a
href="https://doi.org/10.1038/s41587-024-02534-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Lobentanzer, Sebastian and Feng, Shaohong and Bruderer, Noah and Maier, Andreas and Wang, Cankun and Baumbach, Jan and Abreu-Vicente, Jorge and Krehl, Nils and Ma, Qin and Lemberger, Thomas and Saez-Rodriguez, Julio},
  doi          = {10.1038/s41587-024-02534-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {166-169},
  shortjournal = {Nature Biotech.},
  title        = {A platform for the biomedical application of large language models},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>,
<em>43</em>(2), 158. (<a
href="https://doi.org/10.1038/s41587-025-02571-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02571-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {158},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Startup grows egg proteins in potato fields. <em>NBT</em>,
<em>43</em>(2), 157. (<a
href="https://doi.org/10.1038/s41587-025-02563-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02563-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {157},
  shortjournal = {Nature Biotech.},
  title        = {Startup grows egg proteins in potato fields},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gene and cell therapies for parkinson’s make headway.
<em>NBT</em>, <em>43</em>(2), 156. (<a
href="https://doi.org/10.1038/s41587-025-02581-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02581-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {156},
  shortjournal = {Nature Biotech.},
  title        = {Gene and cell therapies for parkinson’s make headway},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). <em>NBT</em>, <em>43</em>(2), 155. (<a
href="https://doi.org/10.1038/s41587-025-02578-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02578-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {155},
  shortjournal = {Nature Biotech.},
  title        = {Colossal raises $200 million to bring back the dodo},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI biotechs launch bioprospecting expeditions with
indigenous groups, agree to share benefits. <em>NBT</em>,
<em>43</em>(2), 154–156. (<a
href="https://doi.org/10.1038/s41587-025-02564-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are striking unconventional benefit-sharing agreements with national governments to scour planet hotspots, gathering thousands of new DNA and protein sequences to develop into new, commercially useful molecules.},
  archive      = {J_NBT},
  author       = {Johnson, Ben},
  doi          = {10.1038/s41587-025-02564-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {154-156},
  shortjournal = {Nature Biotech.},
  title        = {AI biotechs launch bioprospecting expeditions with indigenous groups, agree to share benefits},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Largest proteome study enlists 14 biopharmas. <em>NBT</em>,
<em>43</em>(2), 153. (<a
href="https://doi.org/10.1038/s41587-025-02577-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02577-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {153},
  shortjournal = {Nature Biotech.},
  title        = {Largest proteome study enlists 14 biopharmas},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gains in early-onset dementia with progranulin open new
paths for drug discovery. <em>NBT</em>, <em>43</em>(2), 151–153. (<a
href="https://doi.org/10.1038/s41587-025-02562-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first clinical results in patients with a genetic form of frontotemporal dementia (FTD) show that enhancing progranulin in the brain may halt disease progression. If successful, this potentially disease-modifying approach may uncover new avenues for treating other neurodegenerative diseases.},
  archive      = {J_NBT},
  author       = {Gravitz, Lauren},
  doi          = {10.1038/s41587-025-02562-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {151-153},
  shortjournal = {Nature Biotech.},
  title        = {Gains in early-onset dementia with progranulin open new paths for drug discovery},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sound healing and beyond. <em>NBT</em>, <em>43</em>(2),
149–150. (<a href="https://doi.org/10.1038/s41587-025-02566-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultrasound neurotechnologies are moving quickly into clinical trials in a wide variety of applications, and initiatives to open-source their manufacture will make them more accessible.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02566-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {149-150},
  shortjournal = {Nature Biotech.},
  title        = {Sound healing and beyond},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
