<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng---405">NG - 405</h2>
<ul>
<li><details>
<summary>
(2024b). Publisher correction: Genome-wide association study reveals
mechanisms underlying dilated cardiomyopathy and myocardial resilience.
<em>NG</em>, <em>56</em>(12), 2843. (<a
href="https://doi.org/10.1038/s41588-024-02047-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Jurgens, Sean J. and Rämö, Joel T. and Kramarenko, Daria R. and Wijdeveld, Leonoor F. J. M. and Haas, Jan and Chaffin, Mark D. and Garnier, Sophie and Gaziano, Liam and Weng, Lu-Chen and Lipov, Alex and Zheng, Sean L. and Henry, Albert and Huffman, Jennifer E. and Challa, Saketh and Rühle, Frank and Verdugo, Carmen Diaz and Krijger Juárez, Christian and Kany, Shinwan and van Orsouw, Constance A. and Biddinger, Kiran and Poel, Edwin and Elliott, Amanda L. and Wang, Xin and Francis, Catherine and Ruan, Richard and Koyama, Satoshi and Beekman, Leander and Zimmerman, Dominic S. and Deleuze, Jean-François and Villard, Eric and Trégouët, David-Alexandre and Isnard, Richard and Boomsma, Dorret I. and de Geus, Eco J. C. and Tadros, Rafik and Pinto, Yigal M. and Wilde, Arthur A. M. and Hottenga, Jouke-Jan and Sinisalo, Juha and Niiranen, Teemu and Walsh, Roddy and Schmidt, Amand F. and Choi, Seung Hoan and Chang, Kyong-Mi and Tsao, Philip S. and Matthews, Paul M. and Ware, James S. and Lumbers, R. Thomas and van der Crabben, Saskia and Laukkanen, Jari and Palotie, Aarno and Amin, Ahmad S. and Charron, Philippe and Meder, Benjamin and Ellinor, Patrick T. and Daly, Mark and Aragam, Krishna G. and Bezzina, Connie R.},
  doi          = {10.1038/s41588-024-02047-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2843},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Genome-wide association study reveals mechanisms underlying dilated cardiomyopathy and myocardial resilience},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Brca1 haploinsufficiency promotes early
tumor onset and epigenetic alterations in a mouse model of hereditary
breast cancer. <em>NG</em>, <em>56</em>(12), 2842. (<a
href="https://doi.org/10.1038/s41588-024-02034-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Carman Man-Chung and Cordes, Alyssa and Oliphant, Michael U. J. and Quinn, S. Aidan and Thomas, Mayura and Selfors, Laura M. and Silvestri, Francesca and Girnius, Nomeda and Rinaldi, Gianmarco and Zoeller, Jason J. and Shapiro, Hana and Tsiobikas, Christina and Gupta, Kushali P. and Pathania, Shailja and Regev, Aviv and Kadoch, Cigall and Muthuswamy, Senthil K. and Brugge, Joan S.},
  doi          = {10.1038/s41588-024-02034-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2842},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Brca1 haploinsufficiency promotes early tumor onset and epigenetic alterations in a mouse model of hereditary breast cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ChIP-DIP maps binding of hundreds of proteins to DNA
simultaneously and identifies diverse gene regulatory elements.
<em>NG</em>, <em>56</em>(12), 2827–2841. (<a
href="https://doi.org/10.1038/s41588-024-02000-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression is controlled by dynamic localization of thousands of regulatory proteins to precise genomic regions. Understanding this cell type-specific process has been a longstanding goal yet remains challenging because DNA–protein mapping methods generally study one protein at a time. Here, to address this, we developed chromatin immunoprecipitation done in parallel (ChIP-DIP) to generate genome-wide maps of hundreds of diverse regulatory proteins in a single experiment. ChIP-DIP produces highly accurate maps within large pools (&amp;gt;160 proteins) for all classes of DNA-associated proteins, including modified histones, chromatin regulators and transcription factors and across multiple conditions simultaneously. First, we used ChIP-DIP to measure temporal chromatin dynamics in primary dendritic cells following LPS stimulation. Next, we explored quantitative combinations of histone modifications that define distinct classes of regulatory elements and characterized their functional activity in human and mouse cell lines. Overall, ChIP-DIP generates context-specific protein localization maps at consortium scale within any molecular biology laboratory and experimental system. ChIP-DIP (ChIP done in parallel) is a highly multiplex assay for protein–DNA binding, scalable to hundreds of proteins including modified histones, chromatin regulators and transcription factors, offering a refined view of the cis-regulatory code.},
  archive      = {J_NG},
  author       = {Perez, Andrew A. and Goronzy, Isabel N. and Blanco, Mario R. and Yeh, Benjamin T. and Guo, Jimmy K. and Lopes, Carolina S. and Ettlin, Olivia and Burr, Alex and Guttman, Mitchell},
  doi          = {10.1038/s41588-024-02000-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2827-2841},
  shortjournal = {Nat. Genet.},
  title        = {ChIP-DIP maps binding of hundreds of proteins to DNA simultaneously and identifies diverse gene regulatory elements},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The ZmCPK39–ZmDi19–ZmPR10 immune module regulates
quantitative resistance to multiple foliar diseases in maize.
<em>NG</em>, <em>56</em>(12), 2815–2826. (<a
href="https://doi.org/10.1038/s41588-024-01968-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gray leaf spot, northern leaf blight and southern leaf blight are three of the most destructive foliar diseases affecting maize (Zea mays L.). Here we identified a gene, ZmCPK39, that encodes a calcium-dependent protein kinase and negatively regulates quantitative resistance to these three diseases. The ZmCPK39 allele in the resistant line displayed significantly lower pathogen-induced gene expression than that in the susceptible line. A marked decrease in ZmCPK39 abundance mitigated the phosphorylation and degradation of the transcription factor ZmDi19. This led to elevated expression of ZmPR10, a gene known to encode an antimicrobial protein, thereby enhancing maize resistance to foliar diseases. Moreover, the F1 hybrid with reduced ZmCPK39 expression favored disease resistance, thereby increasing yield. Hence, the discovery of the ZmCPK39–ZmDi19–ZmPR10 immune module provides insight into the mechanisms underlying broad-spectrum quantitative disease resistance and also offers a new avenue for the genetic control of maize foliar diseases. A calcium-dependent protein kinase ZmCPK39 regulates quantitative resistance to multiple foliar diseases in maize through the ZmCPK39–ZmDi19–ZmPR10 immune module.},
  archive      = {J_NG},
  author       = {Zhu, Mang and Zhong, Tao and Xu, Ling and Guo, Chenyu and Zhang, Xiaohui and Liu, Yulin and Zhang, Yan and Li, Yancong and Xie, Zhijian and Liu, Tingting and Jiang, Fuyan and Fan, Xingming and Balint-Kurti, Peter and Xu, Mingliang},
  doi          = {10.1038/s41588-024-01968-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2815-2826},
  shortjournal = {Nat. Genet.},
  title        = {The ZmCPK39–ZmDi19–ZmPR10 immune module regulates quantitative resistance to multiple foliar diseases in maize},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Grapevine pangenome facilitates trait genetics and genomic
breeding. <em>NG</em>, <em>56</em>(12), 2804–2814. (<a
href="https://doi.org/10.1038/s41588-024-01967-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Grapevine breeding is hindered by a limited understanding of the genetic basis of complex agronomic traits. This study constructs a graph-based pangenome reference (Grapepan v.1.0) from 18 newly generated phased telomere-to-telomere assemblies and 11 published assemblies. Using Grapepan v.1.0, we build a variation map with 9,105,787 short variations and 236,449 structural variations (SVs) from the resequencing data of 466 grapevine cultivars. Integrating SVs into a genome-wide association study, we map 148 quantitative trait loci for 29 agronomic traits (50.7% newly identified), with 12 traits significantly contributed by SVs. The estimated heritability improves by 22.78% on average when including SVs. We discovered quantitative trait locus regions under divergent artificial selection in metabolism and berry development between wine and table grapes, respectively. Moreover, significant genetic correlations were detected among the 29 traits. Under a polygenic model, we conducted genomic predictions for each trait. In general, our study facilitates the breeding of superior cultivars via the genomic selection of multiple traits. By constructing a graph-based grapevine pangenome reference (Grapepan v.1.0) and incorporating structural variations and phenotypic maps, the study investigates the genetic basis of agronomic traits, empowering grapevine genomic breeding.},
  archive      = {J_NG},
  author       = {Liu, Zhongjie and Wang, Nan and Su, Ying and Long, Qiming and Peng, Yanling and Shangguan, Lingfei and Zhang, Fan and Cao, Shuo and Wang, Xu and Ge, Mengqing and Xue, Hui and Ma, Zhiyao and Liu, Wenwen and Xu, Xiaodong and Li, Chaochao and Cao, Xuejing and Ahmad, Bilal and Su, Xiangnian and Liu, Yuting and Huang, Guizhou and Du, Mengrui and Liu, Zhenya and Gan, Yu and Sun, Lei and Fan, Xiucai and Zhang, Chuan and Zhong, Haixia and Leng, Xiangpeng and Ren, Yanhua and Dong, Tianyu and Pei, Dan and Wu, Xinyu and Jin, Zhongxin and Wang, Yiwen and Liu, Chonghuai and Chen, Jinfeng and Gaut, Brandon and Huang, Sanwen and Fang, Jinggui and Xiao, Hua and Zhou, Yongfeng},
  doi          = {10.1038/s41588-024-01967-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2804-2814},
  shortjournal = {Nat. Genet.},
  title        = {Grapevine pangenome facilitates trait genetics and genomic breeding},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell multiomics analysis reveals dynamic clonal
evolution and targetable phenotypes in acute myeloid leukemia with
complex karyotype. <em>NG</em>, <em>56</em>(12), 2790–2803. (<a
href="https://doi.org/10.1038/s41588-024-01999-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosomal instability is a major driver of intratumoral heterogeneity (ITH), promoting tumor progression. In the present study, we combined structural variant discovery and nucleosome occupancy profiling with transcriptomic and immunophenotypic changes in single cells to study ITH in complex karyotype acute myeloid leukemia (CK-AML). We observed complex structural variant landscapes within individual cells of patients with CK-AML characterized by linear and circular breakage–fusion–bridge cycles and chromothripsis. We identified three clonal evolution patterns in diagnosis or salvage CK-AML (monoclonal, linear and branched polyclonal), with 75% harboring multiple subclones that frequently displayed ongoing karyotype remodeling. Using patient-derived xenografts, we demonstrated varied clonal evolution of leukemic stem cells (LSCs) and further dissected subclone-specific drug–response profiles to identify LSC-targeting therapies, including BCL-xL inhibition. In paired longitudinal patient samples, we further revealed genetic evolution and cell-type plasticity as mechanisms of disease progression. By dissecting dynamic genomic, phenotypic and functional complexity of CK-AML, our findings offer clinically relevant avenues for characterizing and targeting disease-driving LSCs. An integrated single-cell multiomic analysis of complex karyotype acute myeloid leukemia characterizes intratumoral heterogeneity and highlights links to therapeutic sensitivities.},
  archive      = {J_NG},
  author       = {Leppä, Aino-Maija and Grimes, Karen and Jeong, Hyobin and Huang, Frank Y. and Andrades, Alvaro and Waclawiczek, Alexander and Boch, Tobias and Jauch, Anna and Renders, Simon and Stelmach, Patrick and Müller-Tidow, Carsten and Karpova, Darja and Sohn, Markus and Grünschläger, Florian and Hasenfeld, Patrick and Benito Garagorri, Eva and Thiel, Vera and Dolnik, Anna and Rodriguez-Martin, Bernardo and Bullinger, Lars and Mrózek, Krzysztof and Eisfeld, Ann-Kathrin and Krämer, Alwin and Sanders, Ashley D. and Korbel, Jan O. and Trumpp, Andreas},
  doi          = {10.1038/s41588-024-01999-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2790-2803},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). JAK/STAT signaling maintains an intermediate cell population
during prostate basal cell fate determination. <em>NG</em>,
<em>56</em>(12), 2776–2789. (<a
href="https://doi.org/10.1038/s41588-024-01979-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unipotent basal and luminal stem cells maintain prostate homeostasis, with an intermediate cell population emerging during prostate inflammation or cancer. However, the identities of basal stem cell and intermediate cell population remain unclear. Here we identified a rare intermediate cell population expressing luminal markers (termed Basal-B) with enhanced organoid formation capacity, and a larger basal population (termed Basal-A). Genetic lineage tracing revealed Basal-B cells represented a transient basal stem cell state during prostate homeostasis and androgen-mediated regeneration. Activated JAK/STAT signaling was identified in Basal-B cells, and its inhibition significantly reduced Basal-B markers expression. Inflammation increased Basal-B-to-luminal cell transdifferentiation, but JAK/STAT inhibition notably attenuated this effect. Pten gene deletion increased Nkx3.1-expressing Basal-B-like cell population and led to neoplasia. In humans, h-Basal-B cells were more prevalent in benign prostate hyperplasia. This study reveals the identities of intermediate Basal-B cells and underscores the role of JAK/STAT signaling in prostate cell fate determination. This study identifies Basal-B cells as a transient cell state during prostate homeostasis and androgen-mediated regeneration. These cells exist in humans and increase in number with prostatitis and inflammation.},
  archive      = {J_NG},
  author       = {Guo, Wangxin and Zhang, Xiaoyu and Li, Lin and Shao, Pengfei and Liang, Chao and Zhang, Hongjiong and Liu, Kuo and Wang, Shuoming and Peng, Yunyi and Luo, Jun and Ju, Yi and De Marzo, Angelo M. and Yu, Chen and Chen, Luonan and Zhou, Bin and Gao, Dong},
  doi          = {10.1038/s41588-024-01979-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2776-2789},
  shortjournal = {Nat. Genet.},
  title        = {JAK/STAT signaling maintains an intermediate cell population during prostate basal cell fate determination},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Brca1 haploinsufficiency promotes early tumor onset and
epigenetic alterations in a mouse model of hereditary breast cancer.
<em>NG</em>, <em>56</em>(12), 2763–2775. (<a
href="https://doi.org/10.1038/s41588-024-01958-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Germline BRCA1 mutation carriers face a high breast cancer risk; however, the underlying mechanisms for this risk are not completely understood. Using a new genetically engineered mouse model of germline Brca1 heterozygosity, we demonstrate that early tumor onset in a Brca1 heterozygous background cannot be fully explained by the conventional ‘two-hit’ hypothesis, suggesting the existence of inherent tumor-promoting alterations in the Brca1 heterozygous state. Single-cell RNA sequencing and assay for transposase-accessible chromatin with sequencing analyses uncover a unique set of differentially accessible chromatin regions in ostensibly normal Brca1 heterozygous mammary epithelial cells, distinct from wild-type cells and partially mimicking the chromatin and RNA-level changes in tumor cells. Transcription factor analyses identify loss of ELF5 and gain of AP-1 sites in these epigenetically primed regions; in vivo experiments further implicate AP-1 and Wnt10a as strong promoters of Brca1-related breast cancer. These findings reveal a previously unappreciated epigenetic effect of Brca1 haploinsufficiency in accelerating tumorigenesis, advancing our mechanistic understanding and informing potential therapeutic strategies. A second hit to Brca1 in heterozygous mice leads to accelerated tumor development compared to wild-type mice in which both alleles are simultaneously deleted. This is because of an epigenetic state associated with Brca1 haploinsufficiency that impacts AP-1 and Wnt10a.},
  archive      = {J_NG},
  author       = {Li, Carman Man-Chung and Cordes, Alyssa and Oliphant, Michael U. J. and Quinn, S. Aidan and Thomas, Mayura and Selfors, Laura M. and Silvestri, Francesca and Girnius, Nomeda and Rinaldi, Gianmarco and Zoeller, Jason J. and Shapiro, Hana and Tsiobikas, Christina and Gupta, Kushali P. and Pathania, Shailja and Regev, Aviv and Kadoch, Cigall and Muthuswamy, Senthil K. and Brugge, Joan S.},
  doi          = {10.1038/s41588-024-01958-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2763-2775},
  shortjournal = {Nat. Genet.},
  title        = {Brca1 haploinsufficiency promotes early tumor onset and epigenetic alterations in a mouse model of hereditary breast cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Luminal breast epithelial cells of BRCA1 or BRCA2 mutation
carriers and noncarriers harbor common breast cancer copy number
alterations. <em>NG</em>, <em>56</em>(12), 2753–2762. (<a
href="https://doi.org/10.1038/s41588-024-01988-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prevalence and nature of somatic copy number alterations (CNAs) in breast epithelium and their role in tumor initiation and evolution remain poorly understood. Using single-cell DNA sequencing (49,238 cells) of epithelium from BRCA1 and BRCA2 carriers or wild-type individuals, we identified recurrent CNAs (for example, 1q-gain and 7q, 10q, 16q and 22q-loss) that are present in a rare population of cells across almost all samples (n = 28). In BRCA1/BRCA2 carriers, these occur before loss of heterozygosity (LOH) of wild-type alleles. These CNAs, common in malignant tumors, are enriched in luminal cells but absent in basal myoepithelial cells. Allele-specific analysis of prevalent CNAs reveals that they arose by independent mutational events, consistent with convergent evolution. BRCA1/BRCA2 carriers contained a small percentage of cells with extreme aneuploidy, featuring loss of TP53, BRCA1/BRCA2 LOH and multiple breast cancer-associated CNAs. Our findings suggest that CNAs arising in normal luminal breast epithelium are precursors to clonally expanded tumor genomes. Single-cell DNA sequencing identifies recurrent copy number changes in healthy breast tissue from women with wild-type or germline BRCA1 or BRCA2 mutations.},
  archive      = {J_NG},
  author       = {Williams, Marc J. and Oliphant, Michael U. J. and Au, Vinci and Liu, Cathy and Baril, Caroline and O’Flanagan, Ciara and Lai, Daniel and Beatty, Sean and Van Vliet, Michael and Yiu, Jacky CH and O’Connor, Lauren and Goh, Walter L. and Pollaci, Alicia and Weiner, Adam C. and Grewal, Diljot and McPherson, Andrew and Norton, Klarisa and Moore, McKenna and Prabhakar, Vikas and Agarwal, Shailesh and Garber, Judy E. and Dillon, Deborah A. and Shah, Sohrab P. and Brugge, Joan S. and Aparicio, Samuel},
  doi          = {10.1038/s41588-024-01988-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2753-2762},
  shortjournal = {Nat. Genet.},
  title        = {Luminal breast epithelial cells of BRCA1 or BRCA2 mutation carriers and noncarriers harbor common breast cancer copy number alterations},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell RNA sequencing of peripheral blood links
cell-type-specific regulation of splicing to autoimmune and inflammatory
diseases. <em>NG</em>, <em>56</em>(12), 2739–2752. (<a
href="https://doi.org/10.1038/s41588-024-02019-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alternative splicing contributes to complex traits, but whether this differs in trait-relevant cell types across diverse genetic ancestries is unclear. Here we describe cell-type-specific, sex-biased and ancestry-biased alternative splicing in ~1 M peripheral blood mononuclear cells from 474 healthy donors from the Asian Immune Diversity Atlas. We identify widespread sex-biased and ancestry-biased differential splicing, most of which is cell-type-specific. We identify 11,577 independent cis-splicing quantitative trait loci (sQTLs), 607 trans-sGenes and 107 dynamic sQTLs. Colocalization between cis-eQTLs and trans-sQTLs revealed a cell-type-specific regulatory relationship between HNRNPLL and PTPRC. We observed an enrichment of cis-sQTL effects in autoimmune and inflammatory disease heritability. Specifically, we functionally validated an Asian-specific sQTL disrupting the 5′ splice site of TCHP exon 4 that putatively modulates the risk of Graves’ disease in East Asian populations. Our work highlights the impact of ancestral diversity on splicing and provides a roadmap to dissect its role in complex diseases at single-cell resolution. This analysis of single-cell RNA sequencing data from peripheral blood mononuclear cells for 474 individuals of diverse Asian ancestries in the Asian Immune Diversity Atlas links cell-type-specific splicing variation with autoimmune and inflammatory disease risk.},
  archive      = {J_NG},
  author       = {Tian, Chi and Zhang, Yuntian and Tong, Yihan and Kock, Kian Hong and Sim, Donald Yuhui and Liu, Fei and Dong, Jiaqi and Jing, Zhixuan and Wang, Wenjing and Gao, Junbin and Tan, Le Min and Han, Kyung Yeon and Tomofuji, Yoshihiko and Nakano, Masahiro and Buyamin, Eliora Violain and Sonthalia, Radhika and Ando, Yoshinari and Hatano, Hiroaki and Sonehara, Kyuto and Jin, Xin and Loh, Marie and Chambers, John and Hon, Chung-Chau and Choi, Murim and Park, Jong-Eun and Ishigaki, Kazuyoshi and Okamura, Tomohisa and Fujio, Keishi and Okada, Yukinori and Park, Woong-Yang and Shin, Jay W. and Roca, Xavier and Prabhakar, Shyam and Liu, Boxiang},
  doi          = {10.1038/s41588-024-02019-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2739-2752},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell RNA sequencing of peripheral blood links cell-type-specific regulation of splicing to autoimmune and inflammatory diseases},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Consensus prediction of cell type labels in single-cell data
with popV. <em>NG</em>, <em>56</em>(12), 2731–2738. (<a
href="https://doi.org/10.1038/s41588-024-01993-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell-type classification is a crucial step in single-cell sequencing analysis. Various methods have been proposed for transferring a cell-type label from an annotated reference atlas to unannotated query datasets. Existing methods for transferring cell-type labels lack proper uncertainty estimation for the resulting annotations, limiting interpretability and usefulness. To address this, we propose popular Vote (popV), an ensemble of prediction models with an ontology-based voting scheme. PopV achieves accurate cell-type labeling and provides uncertainty scores. In multiple case studies, popV confidently annotates the majority of cells while highlighting cell populations that are challenging to annotate by label transfer. This additional step helps to reduce the load of manual inspection, which is often a necessary component of the annotation process, and enables one to focus on the most problematic parts of the annotation, streamlining the overall annotation process. Popular Vote (popV) is a simple, ensemble popular vote approach for cell type annotation in single-cell omic data, flexibly incorporating various methods in an open-source Python framework. Across various challenging input datasets, popV offers consistent, accurate performance.},
  archive      = {J_NG},
  author       = {Ergen, Can and Xing, Galen and Xu, Chenling and Kim, Martin and Jayasuriya, Michael and McGeever, Erin and Oliveira Pisco, Angela and Streets, Aaron and Yosef, Nir},
  doi          = {10.1038/s41588-024-01993-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2731-2738},
  shortjournal = {Nat. Genet.},
  title        = {Consensus prediction of cell type labels in single-cell data with popV},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A temporal cortex cell atlas highlights gene expression
dynamics during human brain maturation. <em>NG</em>, <em>56</em>(12),
2718–2730. (<a
href="https://doi.org/10.1038/s41588-024-01990-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human brain undergoes protracted postnatal maturation, guided by dynamic changes in gene expression. Most studies exploring these processes have used bulk tissue analyses, which mask cell-type-specific gene expression dynamics. Here, using single-nucleus RNA sequencing on temporal lobe tissue, including samples of African ancestry, we build a joint pediatric and adult atlas of 75 cell subtypes, which we verify with spatial transcriptomics. We explore the differences between pediatric and adult cell subtypes, revealing the genes and pathways that change during brain maturation. Our results highlight excitatory neuron subtypes, including the LTK and FREM subtypes, that show elevated expression of genes associated with cognition and synaptic plasticity in pediatric tissue. The resources we present here improve our understanding of the brain during its development and contribute to global efforts to build an inclusive brain cell map. This Pediatric Cell Atlas study analyzes temporal cortex single-nucleus RNA sequencing datasets from eight diverse donors from 4 to 50 years of age, describing gene expression dynamics over the course of brain maturation.},
  archive      = {J_NG},
  author       = {Steyn, Christina and Mishi, Ruvimbo and Fillmore, Stephanie and Verhoog, Matthijs B. and More, Jessica and Rohlwink, Ursula K. and Melvill, Roger and Butler, James and Enslin, Johannes M. N. and Jacobs, Muazzam and Sauka-Spengler, Tatjana and Greco, Maria and Quiñones, Sadi and Dulla, Chris G. and Raimondo, Joseph V. and Figaji, Anthony and Hockman, Dorit},
  doi          = {10.1038/s41588-024-01990-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2718-2730},
  shortjournal = {Nat. Genet.},
  title        = {A temporal cortex cell atlas highlights gene expression dynamics during human brain maturation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial and single-nucleus transcriptomic analysis of
genetic and sporadic forms of alzheimer’s disease. <em>NG</em>,
<em>56</em>(12), 2704–2717. (<a
href="https://doi.org/10.1038/s41588-024-01961-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The pathogenesis of Alzheimer’s disease (AD) depends on environmental and heritable factors, with its molecular etiology still unclear. Here we present a spatial transcriptomic (ST) and single-nucleus transcriptomic survey of late-onset sporadic AD and AD in Down syndrome (DSAD). Studying DSAD provides an opportunity to enhance our understanding of the AD transcriptome, potentially bridging the gap between genetic mouse models and sporadic AD. We identified transcriptomic changes that may underlie cortical layer-preferential pathology accumulation. Spatial co-expression network analyses revealed transient and regionally restricted disease processes, including a glial inflammatory program dysregulated in upper cortical layers and implicated in AD genetic risk and amyloid-associated processes. Cell–cell communication analysis further contextualized this gene program in dysregulated signaling networks. Finally, we generated ST data from an amyloid AD mouse model to identify cross-species amyloid-proximal transcriptomic changes with conformational context. Spatial and single-nucleus analyses in human postmortem Alzheimer’s disease (AD) brain tissues at early and late stages from individuals with and without Down syndrome, as well as in AD mouse models, show sex and species-specific phenotypic changes.},
  archive      = {J_NG},
  author       = {Miyoshi, Emily and Morabito, Samuel and Henningfield, Caden M. and Das, Sudeshna and Rahimzadeh, Negin and Shabestari, Sepideh Kiani and Michael, Neethu and Emerson, Nora and Reese, Fairlie and Shi, Zechuan and Cao, Zhenkun and Srinivasan, Shushrruth Sai and Scarfone, Vanessa M. and Arreola, Miguel A. and Lu, Jackie and Wright, Sierra and Silva, Justine and Leavy, Kelsey and Lott, Ira T. and Doran, Eric and Yong, William H. and Shahin, Saba and Perez-Rosendahl, Mari and Head, Elizabeth and Green, Kim N. and Swarup, Vivek},
  doi          = {10.1038/s41588-024-01961-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2704-2717},
  shortjournal = {Nat. Genet.},
  title        = {Spatial and single-nucleus transcriptomic analysis of genetic and sporadic forms of alzheimer’s disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pervasive biases in proxy genome-wide association studies
based on parental history of alzheimer’s disease. <em>NG</em>,
<em>56</em>(12), 2696–2703. (<a
href="https://doi.org/10.1038/s41588-024-01963-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Almost every recent Alzheimer’s disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and nonrandom participation in parental illness surveys, which cause substantial discrepancies between AD GWAS and GWAX results. We demonstrate that the current AD GWAX provide highly misleading genetic correlations between AD risk and higher education, which subsequently affects a variety of genetic epidemiological applications involving AD and cognition. Our study sheds light on potential issues in the design and analysis of middle-aged biobank cohorts and underscores the need for caution when interpreting genetic association results based on proxy-reported parental disease history. Genetic associations based on parental history of Alzheimer’s disease are shown to be impacted by uncorrected survival bias and nonrandom survey participation, underscoring the need for caution when interpreting results based on this study design.},
  archive      = {J_NG},
  author       = {Wu, Yuchang and Sun, Zhongxuan and Zheng, Qinwen and Miao, Jiacheng and Dorn, Stephen and Mukherjee, Shubhabrata and Fletcher, Jason M. and Lu, Qiongshi},
  doi          = {10.1038/s41588-024-01963-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2696-2703},
  shortjournal = {Nat. Genet.},
  title        = {Pervasive biases in proxy genome-wide association studies based on parental history of alzheimer’s disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic architecture of cerebrospinal fluid and brain
metabolite levels and the genetic colocalization of metabolites with
human traits. <em>NG</em>, <em>56</em>(12), 2685–2695. (<a
href="https://doi.org/10.1038/s41588-024-01973-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain metabolism perturbation can contribute to traits and diseases. We conducted a genome-wide association study for cerebrospinal fluid (CSF) and brain metabolite levels, identifying 205 independent associations (47.3% new signals, containing 11 new loci) for 139 CSF metabolites, and 32 independent associations (43.8% new signals, containing 4 new loci) for 31 brain metabolites. Of these, 96.9% (CSF) and 71.4% (brain) of the new signals belonged to previously analyzed metabolites in blood or urine. We integrated the metabolite quantitative trait loci (MQTLs) with 23 neurological, psychiatric and common human traits and diseases through colocalization to identify metabolites and biological processes implicated in these phenotypes. Combining CSF and brain, we identified 71 metabolite–trait associations, such as glycerophosphocholines with Alzheimer’s disease, O-sulfo-l-tyrosine with Parkinson’s disease, glycine, xanthine with waist-to-hip ratio and ergothioneine with inflammatory bowel disease. Our study expanded the knowledge of MQTLs in the central nervous system, providing insights into human traits. Genome-wide association study of cerebrospinal fluid and brain metabolites highlights the unique genetic architecture of metabolite levels and metabolite–trait associations with brain-related phenotypes.},
  archive      = {J_NG},
  author       = {Wang, Ciyang and Yang, Chengran and Western, Daniel and Ali, Muhammad and Wang, Yueyao and Phuah, Chia-Ling and Budde, John and Wang, Lihua and Gorijala, Priyanka and Timsina, Jigyasha and Ruiz, Agustin and Pastor, Pau and Fernandez, Maria Victoria and Panyard, Daniel J. and Engelman, Corinne D. and Deming, Yuetiva and Boada, Merce and Cano, Amanda and Garcia-Gonzalez, Pablo and Graff-Radford, Neill R. and Mori, Hiroshi and Lee, Jae-Hong and Perrin, Richard J. and Ibanez, Laura and Sung, Yun Ju and Cruchaga, Carlos},
  doi          = {10.1038/s41588-024-01973-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2685-2695},
  shortjournal = {Nat. Genet.},
  title        = {Genetic architecture of cerebrospinal fluid and brain metabolite levels and the genetic colocalization of metabolites with human traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Proteogenomic analysis of human cerebrospinal fluid
identifies neurologically relevant regulation and implicates causal
proteins for alzheimer’s disease. <em>NG</em>, <em>56</em>(12),
2672–2684. (<a
href="https://doi.org/10.1038/s41588-024-01972-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The integration of quantitative trait loci (QTLs) with disease genome-wide association studies (GWASs) has proven successful in prioritizing candidate genes at disease-associated loci. QTL mapping has been focused on multi-tissue expression QTLs or plasma protein QTLs (pQTLs). We generated a cerebrospinal fluid (CSF) pQTL atlas by measuring 6,361 proteins in 3,506 samples. We identified 3,885 associations for 1,883 proteins, including 2,885 new pQTLs, demonstrating unique genetic regulation in CSF. We identified CSF-enriched pleiotropic regions on chromosome (chr)3q28 near OSTN and chr19q13.32 near APOE that were enriched for neuron specificity and neurological development. We integrated our associations with Alzheimer’s disease (AD) through proteome-wide association study (PWAS), colocalization and Mendelian randomization and identified 38 putative causal proteins, 15 of which have drugs available. Finally, we developed a proteomics-based AD prediction model that outperforms genetics-based models. These findings will be instrumental to further understand the biology and identify causal and druggable proteins for brain and neurological traits. Proteogenomic analysis of human cerebrospinal fluid by measuring 6,361 proteins in 3,506 individuals identifies new protein QTLs and highlights genetic regulation involved in neurological processes.},
  archive      = {J_NG},
  author       = {Western, Daniel and Timsina, Jigyasha and Wang, Lihua and Wang, Ciyang and Yang, Chengran and Phillips, Bridget and Wang, Yueyao and Liu, Menghan and Ali, Muhammad and Beric, Aleksandra and Gorijala, Priyanka and Kohlfeld, Pat and Budde, John and Levey, Allan I. and Morris, John C. and Perrin, Richard J. and Ruiz, Agustin and Marquié, Marta and Boada, Mercè and de Rojas, Itziar and Rutledge, Jarod and Oh, Hamilton and Wilson, Edward N. and Le Guen, Yann and Reus, Lianne M. and Tijms, Betty and Visser, Pieter Jelle and van der Lee, Sven J. and Pijnenburg, Yolande A. L. and Teunissen, Charlotte E. and del Campo Milan, Marta and Alvarez, Ignacio and Aguilar, Miquel and Greicius, Michael D. and Pastor, Pau and Pulford, David J. and Ibanez, Laura and Wyss-Coray, Tony and Sung, Yun Ju and Cruchaga, Carlos},
  doi          = {10.1038/s41588-024-01972-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2672-2684},
  shortjournal = {Nat. Genet.},
  title        = {Proteogenomic analysis of human cerebrospinal fluid identifies neurologically relevant regulation and implicates causal proteins for alzheimer’s disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide association analyses identify distinct genetic
architectures for age-related macular degeneration across ancestries.
<em>NG</em>, <em>56</em>(12), 2659–2671. (<a
href="https://doi.org/10.1038/s41588-024-01764-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To effectively reduce vision loss due to age-related macular generation (AMD) on a global scale, knowledge of its genetic architecture in diverse populations is necessary. A critical element, AMD risk profiles in African and Hispanic/Latino ancestries, remains largely unknown. We combined data in the Million Veteran Program with five other cohorts to conduct the first multi-ancestry genome-wide association study of AMD and discovered 63 loci (30 novel). We observe marked cross-ancestry heterogeneity at major risk loci, especially in African-ancestry populations which demonstrate a primary signal in a major histocompatibility complex class II haplotype and reduced risk at the established CFH and ARMS2/HTRA1 loci. Dissecting local ancestry in admixed individuals, we find significantly smaller marginal effect sizes for CFH risk alleles in African ancestry haplotypes. Broadening efforts to include ancestrally distinct populations helped uncover genes and pathways that boost risk in an ancestry-dependent manner and are potential targets for corrective therapies. Multi-ancestry genome-wide analyses identify new risk loci for age-related macular degeneration. Ancestry-specific analyses identify distinct effects at major risk loci, including smaller effect sizes for CFH risk alleles in haplotypes of African ancestry.},
  archive      = {J_NG},
  author       = {Gorman, Bryan R. and Voloudakis, Georgios and Igo, Robert P. and Kinzy, Tyler and Halladay, Christopher W. and Bigdeli, Tim B. and Zeng, Biao and Venkatesh, Sanan and Cooke Bailey, Jessica N. and Crawford, Dana C. and Markianos, Kyriacos and Dong, Frederick and Schreiner, Patrick A. and Zhang, Wen and Hadi, Tamer and Anger, Matthew D. and Stockwell, Amy and Melles, Ronald B. and Yin, Jie and Choquet, Hélène and Kaye, Rebecca and Patasova, Karina and Patel, Praveen J. and Yaspan, Brian L. and Jorgenson, Eric and Hysi, Pirro G. and Lotery, Andrew J. and Gaziano, J. Michael and Tsao, Philip S. and Fliesler, Steven J. and Sullivan, Jack M. and Greenberg, Paul B. and Wu, Wen-Chih and Assimes, Themistocles L. and Pyarajan, Saiju and Roussos, Panos and Peachey, Neal S. and Iyengar, Sudha K.},
  doi          = {10.1038/s41588-024-01764-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2659-2671},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses identify distinct genetic architectures for age-related macular degeneration across ancestries},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide association analysis provides insights into the
molecular etiology of dilated cardiomyopathy. <em>NG</em>,
<em>56</em>(12), 2646–2658. (<a
href="https://doi.org/10.1038/s41588-024-01952-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dilated cardiomyopathy (DCM) is a leading cause of heart failure and cardiac transplantation. We report a genome-wide association study and multi-trait analysis of DCM (14,256 cases) and three left ventricular traits (36,203 UK Biobank participants). We identified 80 genomic risk loci and prioritized 62 putative effector genes, including several with rare variant DCM associations (MAP3K7, NEDD4L and SSPN). Using single-nucleus transcriptomics, we identify cellular states, biological pathways, and intracellular communications that drive pathogenesis. We demonstrate that polygenic scores predict DCM in the general population and modify penetrance in carriers of rare DCM variants. Our findings may inform the design of genetic testing strategies that incorporate polygenic background. They also provide insights into the molecular etiology of DCM that may facilitate the development of targeted therapeutics. Genome-wide association analyses comprising 14,256 cases and 1,199,156 controls and incorporating correlated cardiac magnetic resonance imaging traits provide insights into the molecular etiology of dilated cardiomyopathy.},
  archive      = {J_NG},
  author       = {Zheng, Sean L. and Henry, Albert and Cannie, Douglas and Lee, Michael and Miller, David and McGurk, Kathryn A. and Bond, Isabelle and Xu, Xiao and Issa, Hanane and Francis, Catherine and De Marvao, Antonio and Theotokis, Pantazis I. and Buchan, Rachel J. and Speed, Doug and Abner, Erik and Adams, Lance and Aragam, Krishna G. and Ärnlöv, Johan and Raja, Anna Axelsson and Backman, Joshua D. and Baksi, John and Barton, Paul J. R. and Biddinger, Kiran J. and Boersma, Eric and Brandimarto, Jeffrey and Brunak, Søren and Bundgaard, Henning and Carey, David J. and Charron, Philippe and Cook, James P. and Cook, Stuart A. and Denaxas, Spiros and Deleuze, Jean-François and Doney, Alexander S. and Elliott, Perry and Erikstrup, Christian and Esko, Tõnu and Farber-Eger, Eric H. and Finan, Chris and Garnier, Sophie and Ghouse, Jonas and Giedraitis, Vilmantas and Guðbjartsson, Daniel F. and Haggerty, Christopher M. and Halliday, Brian P. and Helgadottir, Anna and Hemingway, Harry and Hillege, Hans L. and Kardys, Isabella and Lind, Lars and Lindgren, Cecilia M. and Lowery, Brandon D. and Manisty, Charlotte and Margulies, Kenneth B. and Moon, James C. and Mordi, Ify R. and Morley, Michael P. and Morris, Andrew D. and Morris, Andrew P. and Morton, Lori and Noursadeghi, Mahdad and Ostrowski, Sisse R. and Owens, Anjali T. and Palmer, Colin N. A. and Pantazis, Antonis and Pedersen, Ole B. V. and Prasad, Sanjay K. and Shekhar, Akshay and Smelser, Diane T. and Srinivasan, Sundararajan and Stefansson, Kari and Sveinbjörnsson, Garðar and Syrris, Petros and Tammesoo, Mari-Liis and Tayal, Upasana and Teder-Laving, Maris and Thorgeirsson, Guðmundur and Thorsteinsdottir, Unnur and Tragante, Vinicius and Trégouët, David-Alexandre and Treibel, Thomas A. and Ullum, Henrik and Valdes, Ana M. and van Setten, Jessica and van Vugt, Marion and Veluchamy, Abirami and Verschuren, W. M. Monique and Villard, Eric and Yang, Yifan and Asselbergs, Folkert W. and Cappola, Thomas P. and Dube, Marie-Pierre and Dunn, Michael E. and Ellinor, Patrick T. and Hingorani, Aroon D. and Lang, Chim C. and Samani, Nilesh J. and Shah, Svati H. and Smith, J. Gustav and Vasan, Ramachandran S. and O’Regan, Declan P. and Holm, Hilma and Noseda, Michela and Wells, Quinn and Ware, James S. and Lumbers, R. Thomas},
  doi          = {10.1038/s41588-024-01952-y},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2646-2658},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analysis provides insights into the molecular etiology of dilated cardiomyopathy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Genome-wide association study reveals mechanisms underlying
dilated cardiomyopathy and myocardial resilience. <em>NG</em>,
<em>56</em>(12), 2636–2645. (<a
href="https://doi.org/10.1038/s41588-024-01975-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dilated cardiomyopathy (DCM) is a heart muscle disease that represents an important cause of morbidity and mortality, yet causal mechanisms remain largely elusive. Here, we perform a large-scale genome-wide association study and multitrait analysis for DCM using 9,365 cases and 946,368 controls. We identify 70 genome-wide significant loci, which show broad replication in independent samples and map to 63 prioritized genes. Tissue, cell type and pathway enrichment analyses highlight the central role of the cardiomyocyte and contractile apparatus in DCM pathogenesis. Polygenic risk scores constructed from our genome-wide association study predict DCM across different ancestry groups, show differing contributions to DCM depending on rare pathogenic variant status and associate with systolic heart failure across various clinical settings. Mendelian randomization analyses reveal actionable potential causes of DCM, including higher bodyweight and higher systolic blood pressure. Our findings provide insights into the genetic architecture and mechanisms underlying DCM and myocardial function more broadly. Genome-wide association and multitrait analyses for dilated cardiomyopathy (DCM) using 9,365 cases and 946,368 controls provide insights into the mechanisms underlying DCM and myocardial resilience},
  archive      = {J_NG},
  author       = {Jurgens, Sean J. and Rämö, Joel T. and Kramarenko, Daria R. and Wijdeveld, Leonoor F. J. M. and Haas, Jan and Chaffin, Mark D. and Garnier, Sophie and Gaziano, Liam and Weng, Lu-Chen and Lipov, Alex and Zheng, Sean L. and Henry, Albert and Huffman, Jennifer E. and Challa, Saketh and Rühle, Frank and Verdugo, Carmen Diaz and Krijger Juárez, Christian and Kany, Shinwan and van Orsouw, Constance A. and Biddinger, Kiran and Poel, Edwin and Elliott, Amanda L. and Wang, Xin and Francis, Catherine and Ruan, Richard and Koyama, Satoshi and Beekman, Leander and Zimmerman, Dominic S. and Deleuze, Jean-François and Villard, Eric and Trégouët, David-Alexandre and Isnard, Richard and Boomsma, Dorret I. and de Geus, Eco J. C. and Tadros, Rafik and Pinto, Yigal M. and Wilde, Arthur A. M. and Hottenga, Jouke-Jan and Sinisalo, Juha and Niiranen, Teemu and Walsh, Roddy and Schmidt, Amand F. and Choi, Seung Hoan and Chang, Kyong-Mi and Tsao, Philip S. and Matthews, Paul M. and Ware, James S. and Lumbers, R. Thomas and van der Crabben, Saskia and Laukkanen, Jari and Palotie, Aarno and Amin, Ahmad S. and Charron, Philippe and Meder, Benjamin and Ellinor, Patrick T. and Daly, Mark and Aragam, Krishna G. and Bezzina, Connie R.},
  doi          = {10.1038/s41588-024-01975-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2636-2645},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study reveals mechanisms underlying dilated cardiomyopathy and myocardial resilience},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Engineering structural variants to interrogate genome
function. <em>NG</em>, <em>56</em>(12), 2623–2635. (<a
href="https://doi.org/10.1038/s41588-024-01981-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural variation, such as deletions, duplications, inversions and complex rearrangements, can have profound effects on gene expression, genome stability, phenotypic diversity and disease susceptibility. Structural variants can encompass up to millions of bases and have the potential to rearrange substantial segments of the genome. They contribute considerably more to genetic diversity in human populations and have larger effects on phenotypic traits than point mutations. Until recently, our understanding of the effects of structural variants was driven mainly by studying naturally occurring variation. New genome-engineering tools capable of generating deletions, insertions, inversions and translocations, together with the discovery of new recombinases and advances in creating synthetic DNA constructs, now enable the design and generation of an extended range of structural variation. Here, we discuss these tools and examples of their application and highlight existing challenges that will need to be overcome to fully harness their potential. Structural variations (SVs) impact gene expression, genome stability and disease susceptibility. This Review discusses recent advances in genome-engineering tools that enable precise SV generation and highlights the challenges that remain.},
  archive      = {J_NG},
  author       = {Koeppel, Jonas and Weller, Juliane and Vanderstichele, Thomas and Parts, Leopold},
  doi          = {10.1038/s41588-024-01981-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2623-2635},
  shortjournal = {Nat. Genet.},
  title        = {Engineering structural variants to interrogate genome function},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A history of multiple denisovan introgression events in
modern humans. <em>NG</em>, <em>56</em>(12), 2612–2622. (<a
href="https://doi.org/10.1038/s41588-024-01960-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of a new hominin group in the Altai mountains called Denisovans was one of the most exciting discoveries in human evolution in the last decade. Unlike Neanderthal remains, the Denisovan fossil record consists of only a finger bone, jawbone, teeth and skull fragments. Leveraging the surviving Denisovan segments in modern human genomes has uncovered evidence of at least three introgression events from distinct Denisovan populations into modern humans in the past. Each of them presents different levels of relatedness to the sequenced Altai Denisovan, indicating a complex relationship between these sister lineages. Here we review the evidence suggesting that several Denisovan populations, who likely had an extensive geographical range, were adapted to distinct environments and introgressed into modern humans multiple times. We further discuss how archaic variants have been affected by demographic history, negative and positive selection and close by proposing possible new lines of future research. This review describes evidence of multiple Denisovan populations, each with varying relatedness to Altai Denisovans, that introgressed into modern humans and highlights how introgression influenced human adaptation to a range of environments.},
  archive      = {J_NG},
  author       = {Ongaro, Linda and Huerta-Sanchez, Emilia},
  doi          = {10.1038/s41588-024-01960-y},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2612-2622},
  shortjournal = {Nat. Genet.},
  title        = {A history of multiple denisovan introgression events in modern humans},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The kinase ZmCPK39 regulates foliar disease resistance in
maize plants. <em>NG</em>, <em>56</em>(12), 2610–2611. (<a
href="https://doi.org/10.1038/s41588-024-01970-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The calcium-dependent protein kinase ZmCPK39 is identified as a key immune component in maize infected with foliar pathogens. Its expression is lower in resistant maize lines than in susceptible lines, leading to stabilization of the transcription factor ZmDi19, elevated expression of the anti-microbial protein ZmPR10 and enhanced resistance to multiple foliar diseases.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01970-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2610-2611},
  shortjournal = {Nat. Genet.},
  title        = {The kinase ZmCPK39 regulates foliar disease resistance in maize plants},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell analysis dissects heterogeneity of leukemias
with complex karyotypes. <em>NG</em>, <em>56</em>(12), 2608–2609. (<a
href="https://doi.org/10.1038/s41588-024-02006-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosomal instability plays a crucial part in tumor progression, shaping cancer cell phenotypes and driving treatment resistance. We harnessed two single-cell multiomics methods to characterize the heterogeneity of acute myeloid leukemia with complex karyotype (CK-AML). Our data link genetic, non-genetic and functional heterogeneity and reveal intriguing therapeutic sensitivities.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-02006-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2608-2609},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell analysis dissects heterogeneity of leukemias with complex karyotypes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Intermediate cells with activated JAK/STAT signaling in
prostate regeneration and diseases. <em>NG</em>, <em>56</em>(12),
2606–2607. (<a
href="https://doi.org/10.1038/s41588-024-01977-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {NKX3.1-expressing intermediate Basal-B cells represent a transient basal stem cell state during prostate regeneration, inflammation and cancer initiation. Remarkably, activation of JAK/STAT signaling is essential in regulating expansion and differentiation of Basal-B-like cells during prostate inflammation, identifying this signaling pathway as a potential therapeutic target in prostatitis associated with increased Basal-B signature.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01977-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2606-2607},
  shortjournal = {Nat. Genet.},
  title        = {Intermediate cells with activated JAK/STAT signaling in prostate regeneration and diseases},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Copy number alterations in normal breast tissues revealed by
single-cell whole-genome sequencing. <em>NG</em>, <em>56</em>(12),
2604–2605. (<a
href="https://doi.org/10.1038/s41588-024-02018-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using single-cell whole-genome sequencing, we reveal the landscape of copy number alterations in normal breast tissue from both BRCA carriers and wild-type individuals.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-02018-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2604-2605},
  shortjournal = {Nat. Genet.},
  title        = {Copy number alterations in normal breast tissues revealed by single-cell whole-genome sequencing},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic control of alternative splicing across blood cell
types from diverse asian donors. <em>NG</em>, <em>56</em>(12),
2602–2603. (<a
href="https://doi.org/10.1038/s41588-024-02020-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing functional genomics datasets are European-centric. The Asian Immune Diversity Atlas incorporated single-cell RNA-sequencing data from approximately 1 million peripheral blood mononuclear cells from around 500 donors of diverse Asian ancestries. Mapping of splicing quantitative trait loci revealed context-specific regulation of alternative splicing, as well as cell-type and ancestry-specific genetic effects on complex diseases.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-02020-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2602-2603},
  shortjournal = {Nat. Genet.},
  title        = {Genetic control of alternative splicing across blood cell types from diverse asian donors},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Estimating uncertainty in reference-based cell type
annotation in single-cell genomics. <em>NG</em>, <em>56</em>(12),
2600–2601. (<a
href="https://doi.org/10.1038/s41588-024-01995-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {PopV is an ensemble method for cell type labeling in single-cell genomics. A Cell Ontology-inspired voting procedure across different algorithms highlights low confidence annotations, thereby guiding human-in-the loop components of the annotation process.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01995-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2600-2601},
  shortjournal = {Nat. Genet.},
  title        = {Estimating uncertainty in reference-based cell type annotation in single-cell genomics},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Exploring gene expression in the maturing human brain at the
single-cell level. <em>NG</em>, <em>56</em>(12), 2598–2599. (<a
href="https://doi.org/10.1038/s41588-024-01991-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advancements in single-cell analysis technologies are enabling exploration of the intricacies of the human brain at unprecedented resolution. However, most research thus far has focused on the adult brain. Here, these tools are applied to reveal cell-type-specific gene-expression dynamics as the brain grows from childhood to adulthood.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01991-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2598-2599},
  shortjournal = {Nat. Genet.},
  title        = {Exploring gene expression in the maturing human brain at the single-cell level},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Split-pool barcoding serves up an epigenomic smorgasbord.
<em>NG</em>, <em>56</em>(12), 2596–2597. (<a
href="https://doi.org/10.1038/s41588-024-01980-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Perez, Goronzy et al. present ChIP-DIP, which enables multiplex genomic mapping of hundreds of epitopes from a single sample. The authors apply ChIP-DIP to localize modified histones, transcription factors and other chromatin-interacting proteins at scale, in both cell lines and primary cells.},
  archive      = {J_NG},
  author       = {Ramani, Vijay},
  doi          = {10.1038/s41588-024-01980-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2596-2597},
  shortjournal = {Nat. Genet.},
  title        = {Split-pool barcoding serves up an epigenomic smorgasbord},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Epigenetic scars of brca1 loss point toward breast cancer
cell of origin. <em>NG</em>, <em>56</em>(12), 2594–2595. (<a
href="https://doi.org/10.1038/s41588-024-02021-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The two-hit hypothesis suggests that a second mutation is necessary for cancer development in cells with a defective tumor-suppressor gene, such as BRCA1. However, a study now shows that the loss of just one Brca1 allele in mice can pre-program cells for cancer-promoting changes, indicating cancer may progress earlier than previously thought.},
  archive      = {J_NG},
  author       = {Lewis, Steven M. and dos Santos, Camila},
  doi          = {10.1038/s41588-024-02021-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2594-2595},
  shortjournal = {Nat. Genet.},
  title        = {Epigenetic scars of brca1 loss point toward breast cancer cell of origin},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial mapping of alzheimer’s disease across genetic
subtypes. <em>NG</em>, <em>56</em>(12), 2592–2593. (<a
href="https://doi.org/10.1038/s41588-024-01895-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alzheimer’s disease is a complex, heterogeneous disorder with multiple genetic subtypes. Spatial and single-cell gene expression analyses of these subtypes have provided new insights into general and subtype-specific cellular and molecular mechanisms of Alzheimer’s disease.},
  archive      = {J_NG},
  author       = {Nott, Alexi and Holtman, Inge R.},
  doi          = {10.1038/s41588-024-01895-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2592-2593},
  shortjournal = {Nat. Genet.},
  title        = {Spatial mapping of alzheimer’s disease across genetic subtypes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A cautionary tale for alzheimer’s disease GWAS by proxy.
<em>NG</em>, <em>56</em>(12), 2590–2591. (<a
href="https://doi.org/10.1038/s41588-024-02023-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using reported parental disease history to decipher the genetics of Alzheimer’s disease may be promising, but this approach is also susceptible to complex selection and information bias that can mislead researchers if not accounted for.},
  archive      = {J_NG},
  author       = {Pedersen, Emil M. and Wimberley, Theresa and Vilhjálmsson, Bjarni J.},
  doi          = {10.1038/s41588-024-02023-y},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2590-2591},
  shortjournal = {Nat. Genet.},
  title        = {A cautionary tale for alzheimer’s disease GWAS by proxy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Improved tool for scRNA-seq analysis. <em>NG</em>,
<em>56</em>(12), 2589. (<a
href="https://doi.org/10.1038/s41588-024-02040-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-02040-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2589},
  shortjournal = {Nat. Genet.},
  title        = {Improved tool for scRNA-seq analysis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Chromatin topology dynamics in gene activation.
<em>NG</em>, <em>56</em>(12), 2589. (<a
href="https://doi.org/10.1038/s41588-024-02041-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-024-02041-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2589},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin topology dynamics in gene activation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). DRAGEN optimizes genomic variant detection. <em>NG</em>,
<em>56</em>(12), 2589. (<a
href="https://doi.org/10.1038/s41588-024-02042-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-02042-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2589},
  shortjournal = {Nat. Genet.},
  title        = {DRAGEN optimizes genomic variant detection},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Functional genomics of height heritability. <em>NG</em>,
<em>56</em>(12), 2589. (<a
href="https://doi.org/10.1038/s41588-024-02043-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-024-02043-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2589},
  shortjournal = {Nat. Genet.},
  title        = {Functional genomics of height heritability},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Toward advances in retinoblastoma genetics in kenya.
<em>NG</em>, <em>56</em>(12), 2585–2588. (<a
href="https://doi.org/10.1038/s41588-024-01974-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite extensive advancements in cancer genetics in North America and Europe, the African continent remains underrepresented in this vital research area. Here we highlight a pioneering collaborative project in Kenya, with a focus on expanding cancer genetics services and research into retinoblastoma, a prototypical heritable cancer syndrome.},
  archive      = {J_NG},
  author       = {Dimaras, Helen and Omweri, Beatrice and Muema, Daniel and Kanda, Loice and Macharia, Rosaline Wanjiru and Gitau, John and Mutinda, Catherine and Kimani, Kahaki and Waweru, Wairimu and Gichuhi, Stephen and Mureithi, Marianne W. and Njambi, Lucy},
  doi          = {10.1038/s41588-024-01974-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2585-2588},
  shortjournal = {Nat. Genet.},
  title        = {Toward advances in retinoblastoma genetics in kenya},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Elucidating the genomic basis of rare pediatric neurological
diseases in central asia and transcaucasia. <em>NG</em>,
<em>56</em>(12), 2582–2584. (<a
href="https://doi.org/10.1038/s41588-024-02016-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Kaiyrzhanov, Rauan and Zharkinbekova, Nazira and Guliyeva, Ulviyya and Ganieva, Manizha and Tavadyan, Zaruhi and Gachechiladze, Tamar and Salayev, Kamran and Guliyeva, Sughra and Isayan, Mariam and Kekenadze, Mariam and Sukhudyan, Biayna and Gevorgyan, Ani and Hakobyan, Artsruni and Ibadova, Rima and Tabatadze, Nazi and Kurua, Ekaterina and Shatirishvili, Teona and Yerkhojayeva, Nigara and Koneev, Kairgali and Zhumakhanov, Dauren and Mukushev, Askhat and Jaxybayeva, Altynshash and Nauryzbayeva, Alissa and Isrofilov, Maksudjon and Badalova, Saadat and Zeyniyeva, Naila and Hajiyeva, Ilaha and Alakbarov, Leyla and Zeynalova, Aynur and Chelban, Viorica and Vandrovcova, Jana and Turchetti, Valentina and Murphy, David and Efthymiou, Stephanie and Alavi, Shahryar and Mohammad, Rahema and Tkemaladze, Tinatin and Shashkin, Chingiz and Tatishvili, Nana Nino and Beridze, Maia and Khachatryan, Samson G. and Melikishvili, Gia and Hardy, John and Maroofian, Reza and Houlden, Henry},
  doi          = {10.1038/s41588-024-02016-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2582-2584},
  shortjournal = {Nat. Genet.},
  title        = {Elucidating the genomic basis of rare pediatric neurological diseases in central asia and transcaucasia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Toward GDPR compliance with the helmholtz munich genotype
imputation server. <em>NG</em>, <em>56</em>(12), 2580–2581. (<a
href="https://doi.org/10.1038/s41588-024-02012-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Rayner, N. William and Park, Young-Chan and Fuchsberger, Christian and Barysenka, Andrei and Zeggini, Eleftheria},
  doi          = {10.1038/s41588-024-02012-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2580-2581},
  shortjournal = {Nat. Genet.},
  title        = {Toward GDPR compliance with the helmholtz munich genotype imputation server},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A lifesaving revolution delayed. <em>NG</em>,
<em>56</em>(12), 2579. (<a
href="https://doi.org/10.1038/s41588-024-02013-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A family tragedy revealed a hidden genetic mutation and transformed my work as a geneticist into a mission to increase awareness of the critical importance of genetic testing.},
  archive      = {J_NG},
  author       = {Liebman, Susan Weiss},
  doi          = {10.1038/s41588-024-02013-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2579},
  shortjournal = {Nat. Genet.},
  title        = {A lifesaving revolution delayed},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Publisher correction: A genome-wide association analysis
reveals new pathogenic pathways in gout. <em>NG</em>, <em>56</em>(11),
2577. (<a href="https://doi.org/10.1038/s41588-024-02017-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Major, Tanya J. and Takei, Riku and Matsuo, Hirotaka and Leask, Megan P. and Sumpter, Nicholas A. and Topless, Ruth K. and Shirai, Yuya and Wang, Wei and Cadzow, Murray J. and Phipps-Green, Amanda J. and Li, Zhiqiang and Ji, Aichang and Merriman, Marilyn E. and Morice, Emily and Kelley, Eric E. and Wei, Wen-Hua and McCormick, Sally P. A. and Bixley, Matthew J. and Reynolds, Richard J. and Saag, Kenneth G. and Fadason, Tayaza and Golovina, Evgenia and O’Sullivan, Justin M. and Stamp, Lisa K. and Dalbeth, Nicola and Abhishek, Abhishek and Doherty, Michael and Roddy, Edward and Jacobsson, Lennart T. H. and Kapetanovic, Meliha C. and Melander, Olle and Andrés, Mariano and Pérez-Ruiz, Fernando and Torres, Rosa J. and Radstake, Timothy and Jansen, Timothy L. and Janssen, Matthijs and Joosten, Leo A. B. and Liu, Ruiqi and Gaal, Orsolya I. and Crişan, Tania O. and Rednic, Simona and Kurreeman, Fina and Huizinga, Tom W. J. and Toes, René and Lioté, Frédéric and Richette, Pascal and Bardin, Thomas and Ea, Hang Korng and Pascart, Tristan and McCarthy, Geraldine M. and Helbert, Laura and Stibůrková, Blanka and Tausche, Anne-K. and Uhlig, Till and Vitart, Véronique and Boutin, Thibaud S. and Hayward, Caroline and Riches, Philip L. and Ralston, Stuart H. and Campbell, Archie and MacDonald, Thomas M. and Nakayama, Akiyoshi and Takada, Tappei and Nakatochi, Masahiro and Shimizu, Seiko and Kawamura, Yusuke and Toyoda, Yu and Nakaoka, Hirofumi and Yamamoto, Ken and Matsuo, Keitaro and Shinomiya, Nariyoshi and Ichida, Kimiyoshi and Lee, Chaeyoung and Bradbury, Linda A. and Brown, Matthew A. and Robinson, Philip C. and Buchanan, Russell R. C. and Hill, Catherine L. and Lester, Susan and Smith, Malcolm D. and Rischmueller, Maureen and Choi, Hyon K. and Stahl, Eli A. and Miner, Jeff N. and Solomon, Daniel H. and Cui, Jing and Giacomini, Kathleen M. and Brackman, Deanna J. and Jorgenson, Eric M. and Liu, Hongbo and Susztak, Katalin and Shringarpure, Suyash and So, Alexander and Okada, Yukinori and Li, Changgui and Shi, Yongyong and Merriman, Tony R.},
  doi          = {10.1038/s41588-024-02017-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2577},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: A genome-wide association analysis reveals new pathogenic pathways in gout},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Publisher correction: Rare variant analyses in 51,256 type
2 diabetes cases and 370,487 controls reveal the pathogenicity spectrum
of monogenic diabetes genes. <em>NG</em>, <em>56</em>(11), 2576. (<a
href="https://doi.org/10.1038/s41588-024-01986-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Huerta-Chagoya, Alicia and Schroeder, Philip and Mandla, Ravi and Li, Jiang and Morris, Lowri and Vora, Maheak and Alkanaq, Ahmed and Nagy, Dorka and Szczerbinski, Lukasz and Madsen, Jesper G. S. and Bonàs-Guarch, Silvia and Mollandin, Fanny and Cole, Joanne B. and Porneala, Bianca and Westerman, Kenneth and Li, Josephine H. and Pollin, Toni I. and Florez, Jose C. and Gloyn, Anna L. and Carey, David J. and Cebola, Inês and Mirshahi, Uyenlinh L. and Manning, Alisa K. and Leong, Aaron and Udler, Miriam and Mercader, Josep M.},
  doi          = {10.1038/s41588-024-01986-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2576},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Rare variant analyses in 51,256 type 2 diabetes cases and 370,487 controls reveal the pathogenicity spectrum of monogenic diabetes genes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Systematic assessment of ISWI subunits
shows that NURF creates local accessibility for CTCF. <em>NG</em>,
<em>56</em>(11), 2574–2575. (<a
href="https://doi.org/10.1038/s41588-024-01985-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Iurlaro, Mario and Masoni, Francesca and Flyamer, Ilya M. and Wirbelauer, Christiane and Iskar, Murat and Burger, Lukas and Giorgetti, Luca and Schübeler, Dirk},
  doi          = {10.1038/s41588-024-01985-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2574-2575},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Systematic assessment of ISWI subunits shows that NURF creates local accessibility for CTCF},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Standardizing variant naming in
literature with VariantValidator to increase diagnostic rates.
<em>NG</em>, <em>56</em>(11), 2574. (<a
href="https://doi.org/10.1038/s41588-024-02001-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Freeman, Peter J. and Wagstaff, John F. and Fokkema, Ivo F. A. C. and Cutting, Garry R. and Rehm, Heidi L. and Davies, Angela C. and den Dunnen, Johan T. and Gretton, Liam J. and Dalgleish, Raymond},
  doi          = {10.1038/s41588-024-02001-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2574},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Standardizing variant naming in literature with VariantValidator to increase diagnostic rates},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Convergence and divergence of diploid and tetraploid cotton
genomes. <em>NG</em>, <em>56</em>(11), 2562–2573. (<a
href="https://doi.org/10.1038/s41588-024-01964-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyploidy is an important driving force in speciation and evolution; however, the genomic basis for parallel selection of a particular trait between polyploids and ancestral diploids remains unexplored. Here we construct graph-based pan-genomes for diploid (A2) and allotetraploid (AD1) cotton species, enabled by an assembly of 50 genomes of genetically diverse accessions. We delineate a mosaic genome map of tetraploid cultivars that illustrates genomic contributions from semi-wild forms into modern cultivars. Pan-genome comparisons identify syntenic and hyper-divergent regions of continued variation between diploid and tetraploid cottons, and suggest an ongoing process of sequence evolution potentially linked to the contrasting genome size change in two subgenomes. We highlight 43% of genetic regulatory relationships for gene expression in diploid encompassing sequence divergence after polyploidy, and specifically characterize six underexplored convergent genetic loci contributing to parallel selection of fiber quality. This study offers a framework for pan-genomic dissection of genetic regulatory components underlying parallel selection of desirable traits in organisms. High-quality assemblies of 15 diploid and 35 allotetraploid cotton accessions are analyzed in graph-based pan-genomes, providing insights into genome dynamics and regulatory control of fiber transcriptomes under varying ploidy and selection pressures.},
  archive      = {J_NG},
  author       = {Li, Jianying and Liu, Zhenping and You, Chunyuan and Qi, Zhengyang and You, Jiaqi and Grover, Corrinne E. and Long, Yuexuan and Huang, Xianhui and Lu, Sifan and Wang, Yuejin and Zhang, Sainan and Wang, Yawen and Bai, Ruizhe and Zhang, Mengke and Jin, Shuangxia and Nie, Xinhui and Wendel, Jonathan F. and Zhang, Xianlong and Wang, Maojun},
  doi          = {10.1038/s41588-024-01964-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2562-2573},
  shortjournal = {Nat. Genet.},
  title        = {Convergence and divergence of diploid and tetraploid cotton genomes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Haplotype-resolved genome assembly and resequencing analysis
provide insights into genome evolution and allelic imbalance in pinus
densiflora. <em>NG</em>, <em>56</em>(11), 2551–2561. (<a
href="https://doi.org/10.1038/s41588-024-01944-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Haplotype-level allelic characterization facilitates research on the functional, evolutionary and breeding-related features of extremely large and complex plant genomes. We report a 21.7-Gb chromosome-level haplotype-resolved assembly in Pinus densiflora. We found genome rearrangements involving translocations and inversions between chromosomes 1 and 3 of Pinus species and a proliferation of specific long terminal repeat (LTR) retrotransposons (LTR-RTs) in P. densiflora. Evolutionary analyses illustrated that tandem and LTR-RT-mediated duplications led to an increment of transcription factor (TF) genes in P. densiflora. The haplotype sequence comparison showed allelic imbalances, including presence–absence variations of genes (PAV genes) and their functional contributions to flowering and abiotic stress-related traits in P. densiflora. Allele-aware resequencing analysis revealed PAV gene diversity across P. densiflora accessions. Our study provides insights into key mechanisms underlying the evolution of genome structure, LTR-RTs and TFs within the Pinus lineage as well as allelic imbalances and diversity across P. densiflora. Chromosome-scale and haplotype-resolved assembly of Pinus densiflora sheds light on Pinus-specific genome enlargement. Comparison between haplotypes and resequencing of 30 wild accessions show allelic imbalance with roles in flowering regulation and stress resistance.},
  archive      = {J_NG},
  author       = {Jang, Min-Jeong and Cho, Hye Jeong and Park, Young-Soo and Lee, Hye-Young and Bae, Eun-Kyung and Jung, Seungmee and Jin, Hongshi and Woo, Jongchan and Park, Eunsook and Kim, Seo-Jin and Choi, Jin-Wook and Chae, Geun Young and Guk, Ji-Yoon and Kim, Do Yeon and Kim, Sun-Hyung and Kang, Min-Jeong and Lee, Hyoshin and Cheon, Kyeong-Seong and Kim, In Sik and Kim, Yong-Min and Kim, Myung-Shin and Ko, Jae-Heung and Kang, Kyu-Suk and Choi, Doil and Park, Eung-Jun and Kim, Seungill},
  doi          = {10.1038/s41588-024-01944-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2551-2561},
  shortjournal = {Nat. Genet.},
  title        = {Haplotype-resolved genome assembly and resequencing analysis provide insights into genome evolution and allelic imbalance in pinus densiflora},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Structural variation reshapes population gene expression and
trait variation in 2,105 brassica napus accessions. <em>NG</em>,
<em>56</em>(11), 2538–2550. (<a
href="https://doi.org/10.1038/s41588-024-01957-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although individual genomic structural variants (SVs) are known to influence gene expression and trait variation, the extent and scale of SV impact across a species remain unknown. In the present study, we constructed a reference library of 334,461 SVs from genome assemblies of 16 representative morphotypes of neopolyploid Brassica napus accessions and detected 258,865 SVs in 2,105 resequenced genomes. Coupling with 5 tissue population transcriptomes, we uncovered 285,976 SV-expression quantitative trait loci (eQTLs) that associate with altered expression of 73,580 genes. We developed a pipeline for the high-throughput joint analyses of SV-genome-wide association studies (SV-GWASs) and transcriptome-wide association studies of phenomic data, eQTLs and eQTL-GWAS colocalization, and identified 726 SV–gene expression–trait variation associations, some of which were verified by transgenics. The pervasive SV impact on how SV reshapes trait variation was demonstrated with the glucosinolate biosynthesis and transport pathway. The study highlighting the impact of genome-wide and species-scale SVs provides a powerful methodological strategy and valuable resources for studying evolution, gene discovery and breeding. Multiomics joint analyses based on a structural variant (SV) map from 16 genome assemblies and 2,105 resequenced accession genomes shed light on the regulatory effect of SVs on gene expression and trait variation in Brassica napus.},
  archive      = {J_NG},
  author       = {Zhang, Yuanyuan and Yang, Zhiquan and He, Yizhou and Liu, Dongxu and Liu, Yueying and Liang, Congyuan and Xie, Meili and Jia, Yupeng and Ke, Qinglin and Zhou, Yongming and Cheng, Xiaohui and Huang, Junyan and Liu, Lijiang and Xiang, Yang and Raman, Harsh and Kliebenstein, Daniel J. and Liu, Shengyi and Yang, Qing-Yong},
  doi          = {10.1038/s41588-024-01957-7},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2538-2550},
  shortjournal = {Nat. Genet.},
  title        = {Structural variation reshapes population gene expression and trait variation in 2,105 brassica napus accessions},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A regulatory network controlling developmental boundaries
and meristem fates contributed to maize domestication. <em>NG</em>,
<em>56</em>(11), 2528–2537. (<a
href="https://doi.org/10.1038/s41588-024-01943-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During domestication, early farmers selected different vegetative and reproductive traits, but identifying the causative loci has been hampered by their epistasis and functional redundancy. Using chromatin immunoprecipitation sequencing combined with genome-wide association analysis, we uncovered a developmental regulator that controls both types of trait while acting upstream of multiple domestication loci. tasselsheath4 (tsh4) is a new maize domestication gene that establishes developmental boundaries and specifies meristem fates despite not being expressed within them. TSH4 accomplishes this by using a double-negative feedback loop that targets and represses the very same microRNAs that negatively regulate it. TSH4 functions redundantly with a pair of homologs to positively regulate a suite of domestication loci while specifying the meristem that doubled seed yield in modern maize. TSH4 has a critical role in yield gain and helped generate ideal crop plant architecture, thus explaining why it was a major domestication target. Maize transcription factor TASSELSHEATH4 establishes the leaf versus meristem cell boundary and vegetative versus floral axillary meristem fates as part of a domestication gene network.},
  archive      = {J_NG},
  author       = {Dong, Zhaobin and Hu, Gaoyuan and Chen, Qiuyue and Shemyakina, Elena A. and Chau, Geeyun and Whipple, Clinton J. and Fletcher, Jennifer C. and Chuck, George},
  doi          = {10.1038/s41588-024-01943-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2528-2537},
  shortjournal = {Nat. Genet.},
  title        = {A regulatory network controlling developmental boundaries and meristem fates contributed to maize domestication},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Interpreting cis-regulatory interactions from large-scale
deep neural networks. <em>NG</em>, <em>56</em>(11), 2517–2527. (<a
href="https://doi.org/10.1038/s41588-024-01923-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rise of large-scale, sequence-based deep neural networks (DNNs) for predicting gene expression has introduced challenges in their evaluation and interpretation. Current evaluations align DNN predictions with orthogonal experimental data, providing insights into generalization but offering limited insights into their decision-making process. Existing model explainability tools focus mainly on motif analysis, which becomes complex when interpreting longer sequences. Here we present cis-regulatory element model explanations (CREME), an in silico perturbation toolkit that interprets the rules of gene regulation learned by a genomic DNN. Applying CREME to Enformer, a state-of-the-art DNN, we identify cis-regulatory elements that enhance or silence gene expression and characterize their complex interactions. CREME can provide interpretations across multiple scales of genomic organization, from cis-regulatory elements to fine-mapped functional sequence elements within them, offering high-resolution insights into the regulatory architecture of the genome. CREME provides a powerful toolkit for translating the predictions of genomic DNNs into mechanistic insights of gene regulation. CREME is an extensible computational tool for investigating cis-regulation via in silico perturbations of neural network-based DNA sequence models such as Enformer, identifying complex interactions between a gene’s regulatory elements.},
  archive      = {J_NG},
  author       = {Toneyan, Shushan and Koo, Peter K.},
  doi          = {10.1038/s41588-024-01923-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2517-2527},
  shortjournal = {Nat. Genet.},
  title        = {Interpreting cis-regulatory interactions from large-scale deep neural networks},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Human DNA polymerase ε is a source of c&gt;t mutations at
CpG dinucleotides. <em>NG</em>, <em>56</em>(11), 2506–2516. (<a
href="https://doi.org/10.1038/s41588-024-01945-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {C-to-T transitions in CpG dinucleotides are the most prevalent mutations in human cancers and genetic diseases. These mutations have been attributed to deamination of 5-methylcytosine (5mC), an epigenetic modification found on CpGs. We recently linked CpG&amp;gt;TpG mutations to replication and hypothesized that errors introduced by polymerase ε (Pol ε) may represent an alternative source of mutations. Here we present a new method called polymerase error rate sequencing (PER-seq) to measure the error spectrum of DNA polymerases in isolation. We find that the most common human cancer-associated Pol ε mutant (P286R) produces an excess of CpG&amp;gt;TpG errors, phenocopying the mutation spectrum of tumors carrying this mutation and deficiencies in mismatch repair. Notably, we also discover that wild-type Pol ε has a sevenfold higher error rate when replicating 5mCpG compared to C in other contexts. Together, our results from PER-seq and human cancers demonstrate that replication errors are a major contributor to CpG&amp;gt;TpG mutagenesis in replicating cells, fundamentally changing our understanding of this important disease-causing mutational mechanism. A new method called polymerase error rate sequencing (PER-seq) can measure the nucleotide misincorporation rate of DNA polymerases. DNA polymerase ε mutants produce an excess of CpG},
  archive      = {J_NG},
  author       = {Tomkova, Marketa and McClellan, Michael John and Crevel, Gilles and Shahid, Akbar Muhammed and Mozumdar, Nandini and Tomek, Jakub and Shepherd, Emelie and Cotterill, Sue and Schuster-Böckler, Benjamin and Kriaucionis, Skirmantas},
  doi          = {10.1038/s41588-024-01945-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2506-2516},
  shortjournal = {Nat. Genet.},
  title        = {Human DNA polymerase ε is a source of C&gt;T mutations at CpG dinucleotides},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic perturbation screens identify regulators of
inflammatory macrophage states and a role for TNF mRNA m6A modification.
<em>NG</em>, <em>56</em>(11), 2493–2505. (<a
href="https://doi.org/10.1038/s41588-024-01962-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Macrophages exhibit remarkable functional plasticity, a requirement for their central role in tissue homeostasis. During chronic inflammation, macrophages acquire sustained inflammatory ‘states’ that contribute to disease, but there is limited understanding of the regulatory mechanisms that drive their generation. Here we describe a systematic functional genomics approach that combines genome-wide phenotypic screening in primary murine macrophages with transcriptional and cytokine profiling of genetic perturbations in primary human macrophages to uncover regulatory circuits of inflammatory states. This process identifies regulators of five distinct states associated with key features of macrophage function. Among these regulators, loss of the N6-methyladenosine (m6A) writer components abolishes m6A modification of TNF transcripts, thereby enhancing mRNA stability and TNF production associated with multiple inflammatory pathologies. Thus, phenotypic characterization of primary murine and human macrophages describes the regulatory circuits underlying distinct inflammatory states, revealing post-transcriptional control of TNF mRNA stability as an immunosuppressive mechanism in innate immunity. Functional screens using mouse and human primary macrophages identify regulators of distinct inflammatory states, including a role for m6A mRNA modification in TNF production.},
  archive      = {J_NG},
  author       = {Haag, Simone M. and Xie, Shiqi and Eidenschenk, Celine and Fortin, Jean-Philippe and Callow, Marinella and Costa, Mike and Lun, Aaron and Cox, Chris and Wu, Sunny Z. and Pradhan, Rachana N. and Lock, Jaclyn and Kuhn, Julia A. and Holokai, Loryn and Thai, Minh and Freund, Emily and Nissenbaum, Ariane and Keir, Mary and Bohlen, Christopher J. and Martin, Scott and Geiger-Schuller, Kathryn and Hejase, Hussein A. and Yaspan, Brian L. and Melo Carlos, Sandra and Turley, Shannon J. and Murthy, Aditya},
  doi          = {10.1038/s41588-024-01962-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2493-2505},
  shortjournal = {Nat. Genet.},
  title        = {Systematic perturbation screens identify regulators of inflammatory macrophage states and a role for TNF mRNA m6A modification},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Base editing screens define the genetic landscape of cancer
drug resistance mechanisms. <em>NG</em>, <em>56</em>(11), 2479–2492. (<a
href="https://doi.org/10.1038/s41588-024-01948-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug resistance is a principal limitation to the long-term efficacy of cancer therapies. Cancer genome sequencing can retrospectively delineate the genetic basis of drug resistance, but this requires large numbers of post-treatment samples to nominate causal variants. Here we prospectively identify genetic mechanisms of resistance to ten oncology drugs from CRISPR base editing mutagenesis screens in four cancer cell lines using a guide RNA library predicted to install 32,476 variants in 11 cancer genes. We identify four functional classes of protein variants modulating drug sensitivity and use single-cell transcriptomics to reveal how these variants operate through distinct mechanisms, including eliciting a drug-addicted cell state. We identify variants that can be targeted with alternative inhibitors to overcome resistance and functionally validate an epidermal growth factor receptor (EGFR) variant that sensitizes lung cancer cells to EGFR inhibitors. Our variant-to-function map has implications for patient stratification, therapy combinations and drug scheduling in cancer treatment. Base editing screens of 11 cancer genes identify four functional classes of variants that collectively underpin sensitivity and resistance to ten commonly used drugs in cancer cell lines.},
  archive      = {J_NG},
  author       = {Coelho, Matthew A. and Strauss, Magdalena E. and Watterson, Alex and Cooper, Sarah and Bhosle, Shriram and Illuzzi, Giuditta and Karakoc, Emre and Dinçer, Cansu and Vieira, Sara F. and Sharma, Mamta and Moullet, Marie and Conticelli, Daniela and Koeppel, Jonas and McCarten, Katrina and Cattaneo, Chiara M. and Veninga, Vivien and Picco, Gabriele and Parts, Leopold and Forment, Josep V. and Voest, Emile E. and Marioni, John C. and Bassett, Andrew and Garnett, Mathew J.},
  doi          = {10.1038/s41588-024-01948-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2479-2492},
  shortjournal = {Nat. Genet.},
  title        = {Base editing screens define the genetic landscape of cancer drug resistance mechanisms},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatially resolved analysis of pancreatic cancer identifies
therapy-associated remodeling of the tumor microenvironment.
<em>NG</em>, <em>56</em>(11), 2466–2478. (<a
href="https://doi.org/10.1038/s41588-024-01890-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In combination with cell-intrinsic properties, interactions in the tumor microenvironment modulate therapeutic response. We leveraged single-cell spatial transcriptomics to dissect the remodeling of multicellular neighborhoods and cell–cell interactions in human pancreatic cancer associated with neoadjuvant chemotherapy and radiotherapy. We developed spatially constrained optimal transport interaction analysis (SCOTIA), an optimal transport model with a cost function that includes both spatial distance and ligand–receptor gene expression. Our results uncovered a marked change in ligand–receptor interactions between cancer-associated fibroblasts and malignant cells in response to treatment, which was supported by orthogonal datasets, including an ex vivo tumoroid coculture system. We identified enrichment in interleukin-6 family signaling that functionally confers resistance to chemotherapy. Overall, this study demonstrates that characterization of the tumor microenvironment using single-cell spatial transcriptomics allows for the identification of molecular interactions that may play a role in the emergence of therapeutic resistance and offers a spatially based analysis framework that can be broadly applied to other contexts. Spatial molecular imaging analysis of human pancreatic adenocarcinomas describes multicellular neighborhoods in the tumor microenvironment. Ligand–receptor analysis using optimal transport-based SCOTIA identifies interleukin-6 as a mediator of chemoresistance.},
  archive      = {J_NG},
  author       = {Shiau, Carina and Cao, Jingyi and Gong, Dennis and Gregory, Mark T. and Caldwell, Nicholas J. and Yin, Xunqin and Cho, Jae-Won and Wang, Peter L. and Su, Jennifer and Wang, Steven and Reeves, Jason W. and Kim, Tae Kyung and Kim, Youngmi and Guo, Jimmy A. and Lester, Nicole A. and Bae, Jung Woo and Zhao, Ryan and Schurman, Nathan and Barth, Jamie L. and Ganci, Maria L. and Weissleder, Ralph and Jacks, Tyler and Qadan, Motaz and Hong, Theodore S. and Wo, Jennifer Y. and Roberts, Hannah and Beechem, Joseph M. and Castillo, Carlos Fernandez-del and Mino-Kenudson, Mari and Ting, David T. and Hemberg, Martin and Hwang, William L.},
  doi          = {10.1038/s41588-024-01890-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2466-2478},
  shortjournal = {Nat. Genet.},
  title        = {Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial transcriptomic analysis of primary and metastatic
pancreatic cancers highlights tumor microenvironmental heterogeneity.
<em>NG</em>, <em>56</em>(11), 2455–2465. (<a
href="https://doi.org/10.1038/s41588-024-01914-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although the spatial, cellular and molecular landscapes of resected pancreatic ductal adenocarcinoma (PDAC) are well documented, the characteristics of its metastatic ecology remain elusive. By applying spatially resolved transcriptomics to matched primary and metastatic PDAC samples, we discovered a conserved continuum of fibrotic, metabolic and immunosuppressive spatial ecotypes across anatomical regions. We observed spatial tumor microenvironment heterogeneity spanning beyond that previously appreciated in PDAC. Through comparative analysis, we show that the spatial ecotypes exhibit distinct enrichment between primary and metastatic sites, implying adaptability to the local environment for survival and progression. The invasive border ecotype exhibits both pro-tumorigenic and anti-tumorigenic cell-type enrichment, suggesting a potential immunotherapy target. The ecotype heterogeneity across patients emphasizes the need to map individual patient landscapes to develop personalized treatment strategies. Collectively, our findings provide critical insights into metastatic PDAC biology and serve as a valuable resource for future therapeutic exploration and molecular investigations. Spot-based spatial transcriptomic analysis of paired primary and metastatic pancreatic cancers identifies cellular, metabolic and fibrotic changes in ecotypes associated with progression, highlighting the contribution of the tumor microenvironment.},
  archive      = {J_NG},
  author       = {Khaliq, Ateeq M. and Rajamohan, Meenakshi and Saeed, Omer and Mansouri, Kimia and Adil, Asif and Zhang, Chi and Turk, Anita and Carstens, Julienne L. and House, Michael and Hayat, Sikander and Nagaraju, Ganji P. and Pappas, Sam G. and Wang, Y. Alan. and Zyromski, Nicholas J. and Opyrchal, Mateusz and Lee, Kelvin P. and O’Hagan, Heather and El Rayes, Bassel and Masood, Ashiq},
  doi          = {10.1038/s41588-024-01914-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2455-2465},
  shortjournal = {Nat. Genet.},
  title        = {Spatial transcriptomic analysis of primary and metastatic pancreatic cancers highlights tumor microenvironmental heterogeneity},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping extrachromosomal DNA amplifications during cancer
progression. <em>NG</em>, <em>56</em>(11), 2447–2454. (<a
href="https://doi.org/10.1038/s41588-024-01949-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand the role of extrachromosomal DNA (ecDNA) amplifications in cancer progression, we detected and classified focal amplifications in 8,060 newly diagnosed primary cancers, untreated metastases and heavily pretreated tumors. The ecDNAs were detected at significantly higher frequency in untreated metastatic and pretreated tumors compared to newly diagnosed cancers. Tumors from chemotherapy-pretreated patients showed significantly higher ecDNA frequency compared to untreated cancers. In particular, tubulin inhibition associated with ecDNA increases, suggesting a role for ecDNA in treatment response. In longitudinally matched tumor samples, ecDNAs were more likely to be retained compared to chromosomal amplifications. EcDNAs shared between time points, and ecDNAs in advanced cancers were more likely to harbor localized hypermutation events compared to private ecDNAs and ecDNAs in newly diagnosed tumors. Relatively high variant allele fractions of ecDNA localized hypermutations implicated early ecDNA mutagenesis. Our findings nominate ecDNAs to provide tumors with competitive advantages during cancer progression and metastasis. A pan-cancer genomic analysis finds an increase of extrachromosomal DNA (ecDNA) in treated and metastatic tumors compared to primary, untreated samples, as well as ecDNA features enriched in advanced disease.},
  archive      = {J_NG},
  author       = {Kim, Hoon and Kim, Soyeon and Wade, Taylor and Yeo, Eunchae and Lipsa, Anuja and Golebiewska, Anna and Johnson, Kevin C. and An, Sepil and Ko, Junyong and Nam, Yoonjoo and Lee, Hwa Yeon and Kang, Seunghyun and Chung, Heesuk and Niclou, Simone P. and Moon, Hyo-Eun and Paek, Sun Ha and Bafna, Vineet and Luebeck, Jens and Verhaak, Roel G. W.},
  doi          = {10.1038/s41588-024-01949-7},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2447-2454},
  shortjournal = {Nat. Genet.},
  title        = {Mapping extrachromosomal DNA amplifications during cancer progression},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multiomic profiling identifies predictors of survival in
african american patients with acute myeloid leukemia. <em>NG</em>,
<em>56</em>(11), 2434–2446. (<a
href="https://doi.org/10.1038/s41588-024-01929-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic profiles and prognostic biomarkers in patients with acute myeloid leukemia (AML) from ancestry-diverse populations are underexplored. We analyzed the exomes and transcriptomes of 100 patients with AML with genomically confirmed African ancestry (Black; Alliance) and compared their somatic mutation frequencies with those of 323 self-reported white patients with AML, 55% of whom had genomically confirmed European ancestry (white; BeatAML). Here we find that 73% of 162 gene mutations recurrent in Black patients, including a hitherto unreported PHIP alteration detected in 7% of patients, were found in one white patient or not detected. Black patients with myelodysplasia-related AML were younger than white patients suggesting intrinsic and/or extrinsic dysplasia-causing stressors. On multivariable analyses of Black patients, NPM1 and NRAS mutations were associated with inferior disease-free and IDH1 and IDH2 mutations with reduced overall survival. Inflammatory profiles, cell type distributions and transcriptional profiles differed between Black and white patients with NPM1 mutations. Incorporation of ancestry-specific risk markers into the 2022 European LeukemiaNet genetic risk stratification changed risk group assignment for one-third of Black patients and improved their outcome prediction. Analysis of exomes and transcriptomes from 100 African American patients with acute myeloid leukemia identifies ancestry-related variation in mutation profiles and survival. Refined risk classification suggests clinical relevance of these ancestry-associated differences.},
  archive      = {J_NG},
  author       = {Stiff, Andrew and Fornerod, Maarten and Kain, Bailee N. and Nicolet, Deedra and Kelly, Benjamin J. and Miller, Katherine E. and Mrózek, Krzysztof and Boateng, Isaiah and Bollas, Audrey and Garfinkle, Elizabeth A. R. and Momoh, Omolegho and Fasola, Foluke A. and Olawumi, Hannah O. and Mencia-Trinchant, Nuria and Kloppers, Jean F. and van Marle, Anne-Cecilia and Hu, Eileen and Wijeratne, Saranga and Wheeler, Gregory and Walker, Christopher J. and Buss, Jill and Heyrosa, Adrienne and Desai, Helee and Laganson, Andrea and Hamp, Ethan and Abu-Shihab, Yazan and Abaza, Hasan and Kronen, Parker and Sen, Sidharth and Johnstone, Megan E. and Quinn, Kate and Wronowski, Ben and Hertlein, Erin and Miles, Linde A. and Mims, Alice S. and Oakes, Christopher C. and Blachly, James S. and Larkin, Karilyn T. and Mundy-Bosse, Bethany and Carroll, Andrew J. and Powell, Bayard L. and Kolitz, Jonathan E. and Stone, Richard M. and Duarte, Cassandra and Abbott, Diana and Amaya, Maria L. and Jordan, Craig T. and Uy, Geoffrey L. and Stock, Wendy and Archer, Kellie J. and Paskett, Electra D. and Guzman, Monica L. and Levine, Ross L. and Menghrajani, Kamal and Chakravarty, Debyani and Berger, Michael F. and Bottomly, Daniel and McWeeney, Shannon K. and Tyner, Jeffrey W. and Byrd, John C. and Salomonis, Nathan and Grimes, H. Leighton and Mardis, Elaine R. and Eisfeld, Ann-Kathrin},
  doi          = {10.1038/s41588-024-01929-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2434-2446},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic profiling identifies predictors of survival in african american patients with acute myeloid leukemia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Gene-based burden tests of rare germline variants identify
six cancer susceptibility genes. <em>NG</em>, <em>56</em>(11),
2422–2433. (<a
href="https://doi.org/10.1038/s41588-024-01966-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovery of cancer risk variants in the sequence of the germline genome can shed light on carcinogenesis. Here we describe gene burden association analyses, aggregating rare missense and loss of function variants, at 22 cancer sites, including 130,991 cancer cases and 733,486 controls from Iceland, Norway and the United Kingdom. We identified four genes associated with increased cancer risk; the pro-apoptotic BIK for prostate cancer, the autophagy involved ATG12 for colorectal cancer, TG for thyroid cancer and CMTR2 for both lung cancer and cutaneous melanoma. Further, we found genes with rare variants that associate with decreased risk of cancer; AURKB for any cancer, irrespective of site, and PPP1R15A for breast cancer, suggesting that inhibition of PPP1R15A may be a preventive strategy for breast cancer. Our findings pinpoint several new cancer risk genes and emphasize autophagy, apoptosis and cell stress response as a focus point for developing new therapeutics. Burden testing in three large European ancestry cohorts identifies new risk genes for a number of common cancer types, including pan-cancer protective variants in AURKB and breast cancer protective variants in PPP1R15A.},
  archive      = {J_NG},
  author       = {Ivarsdottir, Erna V. and Gudmundsson, Julius and Tragante, Vinicius and Sveinbjornsson, Gardar and Kristmundsdottir, Snaedis and Stacey, Simon N. and Halldorsson, Gisli H. and Magnusson, Magnus I. and Oddsson, Asmundur and Walters, G. Bragi and Sigurdsson, Asgeir and Saevarsdottir, Saedis and Beyter, Doruk and Thorleifsson, Gudmar and Halldorsson, Bjarni V. and Melsted, Pall and Stefansson, Hreinn and Jonsdottir, Ingileif and Sørensen, Erik and Pedersen, Ole B. and Erikstrup, Christian and Bøgsted, Martin and Pøhl, Mette and Røder, Andreas and Stroomberg, Hein Vincent and Gögenur, Ismail and Hillingsø, Jens and Bojesen, Stig E. and Lassen, Ulrik and Høgdall, Estrid and Ullum, Henrik and Brunak, Søren and Ostrowski, Sisse R. and Sonderby, Ida Elken and Frei, Oleksandr and Djurovic, Srdjan and Havdahl, Alexandra and Moller, Pal and Dominguez-Valentin, Mev and Haavik, Jan and Andreassen, Ole A. and Hovig, Eivind and Agnarsson, Bjarni A. and Hilmarsson, Rafn and Johannsson, Oskar Th. and Valdimarsson, Trausti and Jonsson, Steinn and Moller, Pall H. and Olafsson, Jon H. and Sigurgeirsson, Bardur and Jonasson, Jon G. and Tryggvason, Geir and Holm, Hilma and Sulem, Patrick and Rafnar, Thorunn and Gudbjartsson, Daniel F. and Stefansson, Kari},
  doi          = {10.1038/s41588-024-01966-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2422-2433},
  shortjournal = {Nat. Genet.},
  title        = {Gene-based burden tests of rare germline variants identify six cancer susceptibility genes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). GWAS of multiple neuropathology endophenotypes identifies
new risk loci and provides insights into the genetic risk of dementia.
<em>NG</em>, <em>56</em>(11), 2407–2421. (<a
href="https://doi.org/10.1038/s41588-024-01939-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS) have identified &amp;gt;80 Alzheimer’s disease and related dementias (ADRD)-associated genetic loci. However, the clinical outcomes used in most previous studies belie the complex nature of underlying neuropathologies. Here we performed GWAS on 11 ADRD-related neuropathology endophenotypes with participants drawn from the following three sources: the National Alzheimer’s Coordinating Center, the Religious Orders Study and Rush Memory and Aging Project, and the Adult Changes in Thought study (n = 7,804 total autopsied participants). We identified eight independent significantly associated loci, of which four were new (COL4A1, PIK3R5, LZTS1 and APOC2). Separately testing known ADRD loci, 19 loci were significantly associated with at least one neuropathology after false-discovery rate adjustment. Genetic colocalization analyses identified pleiotropic effects and quantitative trait loci. Methylation in the cerebral cortex at two sites near APOC2 was associated with cerebral amyloid angiopathy. Studies that include neuropathology endophenotypes are an important step in understanding the mechanisms underlying genetic ADRD risk. Genome-wide analyses identify common variants associated with 11 distinct neuropathology endophenotypes, providing insights into the mechanisms underlying the genetic risk of Alzheimer’s disease and related dementias.},
  archive      = {J_NG},
  author       = {Shade, Lincoln M. P. and Katsumata, Yuriko and Abner, Erin L. and Aung, Khine Zin and Claas, Steven A. and Qiao, Qi and Heberle, Bernardo Aguzzoli and Brandon, J. Anthony and Page, Madeline L. and Hohman, Timothy J. and Mukherjee, Shubhabrata and Mayeux, Richard P. and Farrer, Lindsay A. and Schellenberg, Gerard D. and Haines, Jonathan L. and Kukull, Walter A. and Nho, Kwangsik and Saykin, Andrew J. and Bennett, David A. and Schneider, Julie A. and Ebbert, Mark T. W. and Nelson, Peter T. and Fardo, David W.},
  doi          = {10.1038/s41588-024-01939-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2407-2421},
  shortjournal = {Nat. Genet.},
  title        = {GWAS of multiple neuropathology endophenotypes identifies new risk loci and provides insights into the genetic risk of dementia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A genome-wide association analysis reveals new pathogenic
pathways in gout. <em>NG</em>, <em>56</em>(11), 2392–2406. (<a
href="https://doi.org/10.1038/s41588-024-01921-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gout is a chronic disease that is caused by an innate immune response to deposited monosodium urate crystals in the setting of hyperuricemia. Here, we provide insights into the molecular mechanism of the poorly understood inflammatory component of gout from a genome-wide association study (GWAS) of 2.6 million people, including 120,295 people with prevalent gout. We detected 377 loci and 410 genetically independent signals (149 previously unreported loci in urate and gout). An additional 65 loci with signals in urate (from a GWAS of 630,117 individuals) but not gout were identified. A prioritization scheme identified candidate genes in the inflammatory process of gout, including genes involved in epigenetic remodeling, cell osmolarity and regulation of NOD-like receptor protein 3 (NLRP3) inflammasome activity. Mendelian randomization analysis provided evidence for a causal role of clonal hematopoiesis of indeterminate potential in gout. Our study identifies candidate genes and molecular processes in the inflammatory pathogenesis of gout suitable for follow-up studies. Genome-wide association analysis of gout and urate identifies 148 new loci, implicating biological pathways and prioritizing candidate genes involved in inflammatory processes.},
  archive      = {J_NG},
  author       = {Major, Tanya J. and Takei, Riku and Matsuo, Hirotaka and Leask, Megan P. and Sumpter, Nicholas A. and Topless, Ruth K. and Shirai, Yuya and Wang, Wei and Cadzow, Murray J. and Phipps-Green, Amanda J. and Li, Zhiqiang and Ji, Aichang and Merriman, Marilyn E. and Morice, Emily and Kelley, Eric E. and Wei, Wen-Hua and McCormick, Sally P. A. and Bixley, Matthew J. and Reynolds, Richard J. and Saag, Kenneth G. and Fadason, Tayaza and Golovina, Evgenia and O’Sullivan, Justin M. and Stamp, Lisa K. and Dalbeth, Nicola and Abhishek, Abhishek and Doherty, Michael and Roddy, Edward and Jacobsson, Lennart T. H. and Kapetanovic, Meliha C. and Melander, Olle and Andrés, Mariano and Pérez-Ruiz, Fernando and Torres, Rosa J. and Radstake, Timothy and Jansen, Timothy L. and Janssen, Matthijs and Joosten, Leo A. B. and Liu, Ruiqi and Gaal, Orsolya I. and Crişan, Tania O. and Rednic, Simona and Kurreeman, Fina and Huizinga, Tom W. J. and Toes, René and Lioté, Frédéric and Richette, Pascal and Bardin, Thomas and Ea, Hang Korng and Pascart, Tristan and McCarthy, Geraldine M. and Helbert, Laura and Stibůrková, Blanka and Tausche, Anne-K. and Uhlig, Till and Vitart, Véronique and Boutin, Thibaud S. and Hayward, Caroline and Riches, Philip L. and Ralston, Stuart H. and Campbell, Archie and MacDonald, Thomas M. and Nakayama, Akiyoshi and Takada, Tappei and Nakatochi, Masahiro and Shimizu, Seiko and Kawamura, Yusuke and Toyoda, Yu and Nakaoka, Hirofumi and Yamamoto, Ken and Matsuo, Keitaro and Shinomiya, Nariyoshi and Ichida, Kimiyoshi and Lee, Chaeyoung and Bradbury, Linda A. and Brown, Matthew A. and Robinson, Philip C. and Buchanan, Russell R. C. and Hill, Catherine L. and Lester, Susan and Smith, Malcolm D. and Rischmueller, Maureen and Choi, Hyon K. and Stahl, Eli A. and Miner, Jeff N. and Solomon, Daniel H. and Cui, Jing and Giacomini, Kathleen M. and Brackman, Deanna J. and Jorgenson, Eric M. and Liu, Hongbo and Susztak, Katalin and Shringarpure, Suyash and So, Alexander and Okada, Yukinori and Li, Changgui and Shi, Yongyong and Merriman, Tony R.},
  doi          = {10.1038/s41588-024-01921-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2392-2406},
  shortjournal = {Nat. Genet.},
  title        = {A genome-wide association analysis reveals new pathogenic pathways in gout},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multivariate genomic analysis of 5 million people elucidates
the genetic architecture of shared components of the metabolic syndrome.
<em>NG</em>, <em>56</em>(11), 2380–2391. (<a
href="https://doi.org/10.1038/s41588-024-01933-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metabolic syndrome (MetS) is a complex hereditary condition comprising various metabolic traits as risk factors. Although the genetics of individual MetS components have been investigated actively through large-scale genome-wide association studies, the conjoint genetic architecture has not been fully elucidated. Here, we performed the largest multivariate genome-wide association study of MetS in Europe (nobserved = 4,947,860) by leveraging genetic correlation between MetS components. We identified 1,307 genetic loci associated with MetS that were enriched primarily in brain tissues. Using transcriptomic data, we identified 11 genes associated strongly with MetS. Our phenome-wide association and Mendelian randomization analyses highlighted associations of MetS with diverse diseases beyond cardiometabolic diseases. Polygenic risk score analysis demonstrated better discrimination of MetS and predictive power in European and East Asian populations. Altogether, our findings will guide future studies aimed at elucidating the genetic architecture of MetS. Large-scale multivariate analyses across populations of European ancestry identify risk loci for the metabolic syndrome, improving polygenic prediction models and highlighting associations with diverse traits beyond cardiometabolic diseases.},
  archive      = {J_NG},
  author       = {Park, Sanghyeon and Kim, Soyeon and Kim, Beomsu and Kim, Dan Say and Kim, Jaeyoung and Ahn, Yeeun and Kim, Hyejin and Song, Minku and Shim, Injeong and Jung, Sang-Hyuk and Cho, Chamlee and Lim, Soohyun and Hong, Sanghoon and Jo, Hyeonbin and Fahed, Akl C. and Natarajan, Pradeep and Ellinor, Patrick T. and Torkamani, Ali and Park, Woong-Yang and Yu, Tae Yang and Myung, Woojae and Won, Hong-Hee},
  doi          = {10.1038/s41588-024-01933-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2380-2391},
  shortjournal = {Nat. Genet.},
  title        = {Multivariate genomic analysis of 5 million people elucidates the genetic architecture of shared components of the metabolic syndrome},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Rare variant analyses in 51,256 type 2 diabetes cases and
370,487 controls reveal the pathogenicity spectrum of monogenic diabetes
genes. <em>NG</em>, <em>56</em>(11), 2370–2379. (<a
href="https://doi.org/10.1038/s41588-024-01947-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Type 2 diabetes (T2D) genome-wide association studies (GWASs) often overlook rare variants as a result of previous imputation panels’ limitations and scarce whole-genome sequencing (WGS) data. We used TOPMed imputation and WGS to conduct the largest T2D GWAS meta-analysis involving 51,256 cases of T2D and 370,487 controls, targeting variants with a minor allele frequency as low as 5 × 10−5. We identified 12 new variants, including a rare African/African American-enriched enhancer variant near the LEP gene (rs147287548), associated with fourfold increased T2D risk. We also identified a rare missense variant in HNF4A (p.Arg114Trp), associated with eightfold increased T2D risk, previously reported in maturity-onset diabetes of the young with reduced penetrance, but observed here in a T2D GWAS. We further leveraged these data to analyze 1,634 ClinVar variants in 22 genes related to monogenic diabetes, identifying two additional rare variants in HNF1A and GCK associated with fivefold and eightfold increased T2D risk, respectively, the effects of which were modified by the individual’s polygenic risk score. For 21% of the variants with conflicting interpretations or uncertain significance in ClinVar, we provided support of being benign based on their lack of association with T2D. Our work provides a framework for using rare variant GWASs to identify large-effect variants and assess variant pathogenicity in monogenic diabetes genes. Rare variant analyses identify a new type 2 diabetes risk allele near the LEP gene, which encodes leptin, and other risk alleles of intermediate penetrance in genes previously implicated in monogenic forms of diabetes.},
  archive      = {J_NG},
  author       = {Huerta-Chagoya, Alicia and Schroeder, Philip and Mandla, Ravi and Li, Jiang and Morris, Lowri and Vora, Maheak and Alkanaq, Ahmed and Nagy, Dorka and Szczerbinski, Lukasz and Madsen, Jesper G. S. and Bonàs-Guarch, Silvia and Mollandin, Fanny and Cole, Joanne B. and Porneala, Bianca and Westerman, Kenneth and Li, Josephine H. and Pollin, Toni I. and Florez, Jose C. and Gloyn, Anna L. and Carey, David J. and Cebola, Inês and Mirshahi, Uyenlinh L. and Manning, Alisa K. and Leong, Aaron and Udler, Miriam and Mercader, Josep M.},
  doi          = {10.1038/s41588-024-01947-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2370-2379},
  shortjournal = {Nat. Genet.},
  title        = {Rare variant analyses in 51,256 type 2 diabetes cases and 370,487 controls reveal the pathogenicity spectrum of monogenic diabetes genes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Valid inference for machine learning-assisted genome-wide
association studies. <em>NG</em>, <em>56</em>(11), 2361–2369. (<a
href="https://doi.org/10.1038/s41588-024-01934-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) has become increasingly popular in almost all scientific disciplines, including human genetics. Owing to challenges related to sample collection and precise phenotyping, ML-assisted genome-wide association study (GWAS), which uses sophisticated ML techniques to impute phenotypes and then performs GWAS on the imputed outcomes, have become increasingly common in complex trait genetics research. However, the validity of ML-assisted GWAS associations has not been carefully evaluated. Here, we report pervasive risks for false-positive associations in ML-assisted GWAS and introduce Post-Prediction GWAS (POP-GWAS), a statistical framework that redesigns GWAS on ML-imputed outcomes. POP-GWAS ensures valid and powerful statistical inference irrespective of imputation quality and choice of algorithm, requiring only GWAS summary statistics as input. We employed POP-GWAS to perform a GWAS of bone mineral density derived from dual-energy X-ray absorptiometry imaging at 14 skeletal sites, identifying 89 new loci and revealing skeletal site-specific genetic architecture. Our framework offers a robust analytic solution for future ML-assisted GWAS. Post-prediction genome-wide association study (POP-GWAS) is a statistical framework that uses summary statistics from labeled samples with both observed and imputed phenotypes to debias single-nucleotide polymorphism effect size estimates for unlabeled samples with imputed phenotypes only, leading to valid and powerful inference.},
  archive      = {J_NG},
  author       = {Miao, Jiacheng and Wu, Yixuan and Sun, Zhongxuan and Miao, Xinran and Lu, Tianyuan and Zhao, Jiwei and Lu, Qiongshi},
  doi          = {10.1038/s41588-024-01934-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2361-2369},
  shortjournal = {Nat. Genet.},
  title        = {Valid inference for machine learning-assisted genome-wide association studies},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic architecture reconciles linkage and association
studies of complex traits. <em>NG</em>, <em>56</em>(11), 2352–2360. (<a
href="https://doi.org/10.1038/s41588-024-01940-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Linkage studies have successfully mapped loci underlying monogenic disorders, but mostly failed when applied to common diseases. Conversely, genome-wide association studies (GWASs) have identified replicable associations between thousands of SNPs and complex traits, yet capture less than half of the total heritability. In the present study we reconcile these two approaches by showing that linkage signals of height and body mass index (BMI) from 119,000 sibling pairs colocalize with GWAS-identified loci. Concordant with polygenicity, we observed the following: a genome-wide inflation of linkage test statistics; that GWAS results predict linkage signals; and that adjusting phenotypes for polygenic scores reduces linkage signals. Finally, we developed a method using recombination rate-stratified, identity-by-descent sharing between siblings to unbiasedly estimate heritability of height (0.76 ± 0.05) and BMI (0.55 ± 0.07). Our results imply that substantial heritability remains unaccounted for by GWAS-identified loci and this residual genetic variation is polygenic and enriched near these loci. Analyses of height and body mass index in 119,000 sibling pairs show that linkage and genome-wide association signals colocalize. Further analyses suggest that family-based linkage signals are fully consistent with a highly polygenic architecture.},
  archive      = {J_NG},
  author       = {Sidorenko, Julia and Couvy-Duchesne, Baptiste and Kemper, Kathryn E. and Moen, Gunn-Helen and Bhatta, Laxmi and Åsvold, Bjørn Olav and Mägi, Reedik and Ani, Alireza and Wang, Rujia and Nolte, Ilja M. and Gordon, Scott and Hayward, Caroline and Campbell, Archie and Benjamin, Daniel J. and Cesarini, David and Evans, David M. and Goddard, Michael E. and Haley, Chris S. and Porteous, David and Medland, Sarah E. and Martin, Nicholas G. and Snieder, Harold and Metspalu, Andres and Hveem, Kristian and Brumpton, Ben and Visscher, Peter M. and Yengo, Loic},
  doi          = {10.1038/s41588-024-01940-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2352-2360},
  shortjournal = {Nat. Genet.},
  title        = {Genetic architecture reconciles linkage and association studies of complex traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Yield of genetic association signals from genomes, exomes
and imputation in the UK biobank. <em>NG</em>, <em>56</em>(11),
2345–2351. (<a
href="https://doi.org/10.1038/s41588-024-01930-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome sequencing (WGS), whole-exome sequencing (WES) and array genotyping with imputation (IMP) are common strategies for assessing genetic variation and its association with medically relevant phenotypes. To date, there has been no systematic empirical assessment of the yield of these approaches when applied to hundreds of thousands of samples to enable the discovery of complex trait genetic signals. Using data for 100 complex traits from 149,195 individuals in the UK Biobank, we systematically compare the relative yield of these strategies in genetic association studies. We find that WGS and WES combined with arrays and imputation (WES + IMP) have the largest association yield. Although WGS results in an approximately fivefold increase in the total number of assayed variants over WES + IMP, the number of detected signals differed by only 1% for both single-variant and gene-based association analyses. Given that WES + IMP typically results in savings of lab and computational time and resources expended per sample, we evaluate the potential benefits of applying WES + IMP to larger samples. When we extend our WES + IMP analyses to 468,169 UK Biobank individuals, we observe an approximately fourfold increase in association signals with the threefold increase in sample size. We conclude that prioritizing WES + IMP and large sample sizes rather than contemporary short-read WGS alternatives will maximize the number of discoveries in genetic association studies. Comparison of association signals in UK Biobank using different strategies for assessing genetic variation shows that whole-exome sequencing combined with array genotyping and imputation offers similar performance to whole-genome sequencing at a reduced cost.},
  archive      = {J_NG},
  author       = {Gaynor, Sheila M. and Joseph, Tyler and Bai, Xiaodong and Zou, Yuxin and Boutkov, Boris and Maxwell, Evan K. and Delaneau, Olivier and Hofmeister, Robin J. and Krasheninina, Olga and Balasubramanian, Suganthi and Marcketta, Anthony and Backman, Joshua and Reid, Jeffrey G. and Overton, John D. and Lotta, Luca A. and Marchini, Jonathan and Salerno, William J. and Baras, Aris and Abecasis, Goncalo R. and Thornton, Timothy A.},
  doi          = {10.1038/s41588-024-01930-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2345-2351},
  shortjournal = {Nat. Genet.},
  title        = {Yield of genetic association signals from genomes, exomes and imputation in the UK biobank},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genomic analysis of intracranial and subcortical brain
volumes yields polygenic scores accounting for variation across
ancestries. <em>NG</em>, <em>56</em>(11), 2333–2344. (<a
href="https://doi.org/10.1038/s41588-024-01951-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Subcortical brain structures are involved in developmental, psychiatric and neurological disorders. Here we performed genome-wide association studies meta-analyses of intracranial and nine subcortical brain volumes (brainstem, caudate nucleus, putamen, hippocampus, globus pallidus, thalamus, nucleus accumbens, amygdala and the ventral diencephalon) in 74,898 participants of European ancestry. We identified 254 independent loci associated with these brain volumes, explaining up to 35% of phenotypic variance. We observed gene expression in specific neural cell types across differentiation time points, including genes involved in intracellular signaling and brain aging-related processes. Polygenic scores for brain volumes showed predictive ability when applied to individuals of diverse ancestries. We observed causal genetic effects of brain volumes with Parkinson’s disease and attention-deficit/hyperactivity disorder. Findings implicate specific gene expression patterns in brain development and genetic variants in comorbid neuropsychiatric disorders, which could point to a brain substrate and region of action for risk genes implicated in brain diseases. Genome-wide association analyses of intracranial and nine subcortical brain volumes in 74,898 participants of European ancestry identify 254 independent loci and yield polygenic scores accounting for brain variation across ancestries.},
  archive      = {J_NG},
  author       = {García-Marín, Luis M. and Campos, Adrian I. and Diaz-Torres, Santiago and Rabinowitz, Jill A. and Ceja, Zuriel and Mitchell, Brittany L. and Grasby, Katrina L. and Thorp, Jackson G. and Agartz, Ingrid and Alhusaini, Saud and Ames, David and Amouyel, Philippe and Andreassen, Ole A. and Arfanakis, Konstantinos and Arias-Vasquez, Alejandro and Armstrong, Nicola J. and Athanasiu, Lavinia and Bastin, Mark E. and Beiser, Alexa S. and Bennett, David A. and Bis, Joshua C. and Boks, Marco P. M. and Boomsma, Dorret I. and Brodaty, Henry and Brouwer, Rachel M. and Buitelaar, Jan K. and Burkhardt, Ralph and Cahn, Wiepke and Calhoun, Vince D. and Carmichael, Owen T. and Chakravarty, Mallar and Chen, Qiang and Ching, Christopher R. K. and Cichon, Sven and Crespo-Facorro, Benedicto and Crivello, Fabrice and Dale, Anders M. and Smith, George Davey and de Geus, Eco J. C. and De Jager, Philip L. and de Zubicaray, Greig I. and Debette, Stéphanie and DeCarli, Charles and Depondt, Chantal and Desrivières, Sylvane and Djurovic, Srdjan and Ehrlich, Stefan and Erk, Susanne and Espeseth, Thomas and Fernández, Guillén and Filippi, Irina and Fisher, Simon E. and Fleischman, Debra A. and Fletcher, Evan and Fornage, Myriam and Forstner, Andreas J. and Francks, Clyde and Franke, Barbara and Ge, Tian and Goldman, Aaron L. and Grabe, Hans J. and Green, Robert C. and Grimm, Oliver and Groenewold, Nynke A. and Gruber, Oliver and Gudnason, Vilmundur and Håberg, Asta K. and Haukvik, Unn K. and Heinz, Andreas and Hibar, Derrek P. and Hilal, Saima and Himali, Jayandra J. and Ho, Beng-Choon and Hoehn, David F. and Hoekstra, Pieter J. and Hofer, Edith and Hoffmann, Wolfgang and Holmes, Avram J. and Homuth, Georg and Hosten, Norbert and Ikram, M. Kamran and Ipser, Jonathan C. and Jack Jr, Clifford R. and Jahanshad, Neda and Jönsson, Erik G. and Kahn, Rene S. and Kanai, Ryota and Klein, Marieke and Knol, Maria J. and Launer, Lenore J. and Lawrie, Stephen M. and Hellard, Stephanie Le and Lee, Phil H. and Lemaître, Hervé and Li, Shuo and Liewald, David C. M. and Lin, Honghuang and Longstreth, W. T. and Lopez, Oscar L. and Luciano, Michelle and Maillard, Pauline and Marquand, Andre F. and Martin, Nicholas G. and Martinot, Jean-Luc and Mather, Karen A. and Mattay, Venkata S. and McMahon, Katie L. and Mecocci, Patrizia and Melle, Ingrid and Meyer-Lindenberg, Andreas and Mirza-Schreiber, Nazanin and Milaneschi, Yuri and Mosley, Thomas H. and Mühleisen, Thomas W. and Müller-Myhsok, Bertram and Maniega, Susana Muñoz and Nauck, Matthias and Nho, Kwangsik and Niessen, Wiro J. and Nöthen, Markus M. and Nyquist, Paul A. and Oosterlaan, Jaap and Pandolfo, Massimo and Paus, Tomas and Pausova, Zdenka and Penninx, Brenda W. J. H. and Pike, G. Bruce and Psaty, Bruce M. and Pütz, Benno and Reppermund, Simone and Rietschel, Marcella D. and Risacher, Shannon L. and Romanczuk-Seiferth, Nina and Romero-Garcia, Rafael and Roshchupkin, Gennady V. and Rotter, Jerome I. and Sachdev, Perminder S. and Sämann, Philipp G. and Saremi, Arvin and Sargurupremraj, Muralidharan and Saykin, Andrew J. and Schmaal, Lianne and Schmidt, Helena and Schmidt, Reinhold and Schofield, Peter R. and Scholz, Markus and Schumann, Gunter and Schwarz, Emanuel and Shen, Li and Shin, Jean and Sisodiya, Sanjay M. and Smith, Albert V. and Smoller, Jordan W. and Soininen, Hilkka S. and Steen, Vidar M. and Stein, Dan J. and Stein, Jason L. and Thomopoulos, Sophia I. and Toga, Arthur W. and Tordesillas-Gutiérrez, Diana and Trollor, Julian N. and Valdes-Hernandez, Maria C. and van ′t Ent, Dennis and van Bokhoven, Hans and van der Meer, Dennis and van der Wee, Nic J. A. and Vázquez-Bourgon, Javier and Veltman, Dick J. and Vernooij, Meike W. and Villringer, Arno and Vinke, Louis N. and Völzke, Henry and Walter, Henrik and Wardlaw, Joanna M. and Weinberger, Daniel R. and Weiner, Michael W. and Wen, Wei and Westlye, Lars T. and Westman, Eric and White, Tonya and Witte, A. Veronica and Wolf, Christiane and Yang, Jingyun and Zwiers, Marcel P. and Ikram, M. Arfan and Seshadri, Sudha and Thompson, Paul M. and Satizabal, Claudia L. and Medland, Sarah E. and Rentería, Miguel E.},
  doi          = {10.1038/s41588-024-01951-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2333-2344},
  shortjournal = {Nat. Genet.},
  title        = {Genomic analysis of intracranial and subcortical brain volumes yields polygenic scores accounting for variation across ancestries},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A phenome-wide association study of methylated GC-rich
repeats identifies a GCC repeat expansion in AFF3 associated with
intellectual disability. <em>NG</em>, <em>56</em>(11), 2322–2332. (<a
href="https://doi.org/10.1038/s41588-024-01917-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GC-rich tandem repeat expansions (TREs) are often associated with DNA methylation, gene silencing and folate-sensitive fragile sites, and underlie several congenital and late-onset disorders. Through a combination of DNA-methylation profiling and tandem repeat genotyping, we identified 24 methylated TREs and investigated their effects on human traits using phenome-wide association studies in 168,641 individuals from the UK Biobank, identifying 156 significant TRE–trait associations involving 17 different TREs. Of these, a GCC expansion in the promoter of AFF3 was associated with a 2.4-fold reduced probability of completing secondary education, an effect size comparable to several recurrent pathogenic microdeletions. In a cohort of 6,371 probands with neurodevelopmental problems of suspected genetic etiology, we observed a significant enrichment of AFF3 expansions compared with controls. With a population prevalence that is at least fivefold higher than the TRE that causes fragile X syndrome, AFF3 expansions represent a major cause of neurodevelopmental delay. Phenome-wide analysis in the UK Biobank identifies GC-rich tandem repeat expansions associated with a range of traits, including a GCC expansion in AFF3 contributing to intellectual disability.},
  archive      = {J_NG},
  author       = {Jadhav, Bharati and Garg, Paras and van Vugt, Joke J. F. A. and Ibanez, Kristina and Gagliardi, Delia and Lee, William and Shadrina, Mariya and Mokveld, Tom and Dolzhenko, Egor and Martin-Trujillo, Alejandro and Gies, Scott J. and Altman, Gabrielle and Rocca, Clarissa and Barbosa, Mafalda and Jain, Miten and Lahiri, Nayana and Lachlan, Katherine and Houlden, Henry and Paten, Benedict and Veldink, Jan and Tucci, Arianna and Sharp, Andrew J.},
  doi          = {10.1038/s41588-024-01917-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2322-2332},
  shortjournal = {Nat. Genet.},
  title        = {A phenome-wide association study of methylated GC-rich repeats identifies a GCC repeat expansion in AFF3 associated with intellectual disability},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Obesity-dependent selection of driver mutations in cancer.
<em>NG</em>, <em>56</em>(11), 2318–2321. (<a
href="https://doi.org/10.1038/s41588-024-01969-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obesity is a risk factor for cancer, but whether obesity is linked to specific genomic subtypes of cancer is unknown. We examined the relationship between obesity and tumor genotype in two clinicogenomic corpora. Obesity was associated with specific driver mutations in lung adenocarcinoma, endometrial carcinoma and cancers of unknown primaries, independent of clinical covariates, demographic factors and genetic ancestry. Obesity is therefore a driver of etiological heterogeneity in some cancers. Analysis of pan-cancer clinical genomic sequencing finds that body mass index associates with driver mutations in certain cancer types, including most prominently lung adenocarcinoma. Obesity may thus influence tumor genetics.},
  archive      = {J_NG},
  author       = {Tang, Cerise and Castillon, Venise Jan and Waters, Michele and Fong, Chris and Park, Tricia and Boscenco, Sonia and Kim, Susie and Pekala, Kelly and Carrot-Zhang, Jian and Hakimi, A. Ari and Schultz, Nikolaus and Ostrovnaya, Irina and Gusev, Alexander and Jee, Justin and Reznik, Ed},
  doi          = {10.1038/s41588-024-01969-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2318-2321},
  shortjournal = {Nat. Genet.},
  title        = {Obesity-dependent selection of driver mutations in cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Potential approaches to create ultimate genotypes in crops
and livestock. <em>NG</em>, <em>56</em>(11), 2310–2317. (<a
href="https://doi.org/10.1038/s41588-024-01942-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many thousands and, in some cases, millions of individuals from the major crop and livestock species have been genotyped and phenotyped for the purpose of genomic selection. ‘Ultimate genotypes’, in which the marker allele haplotypes with the most favorable effects on a target trait or traits in the population are combined together in silico, can be constructed from these datasets. Ultimate genotypes display up to six times the performance of the current best individuals in the population, as demonstrated for net profit in dairy cattle (incorporating a range of economic traits), yield in wheat and 100-seed weight in chickpea. However, current breeding strategies that aim to assemble ultimate genotypes through conventional crossing take many generations. As a hypothetical thought piece, here, we contemplate three future pathways for rapidly achieving ultimate genotypes: accelerated recombination with gene editing, direct editing of whole-genome haplotype sequences and synthetic biology. This Perspective discusses approaches to generate ultimate genotypes, combining the best chromosome segments in livestock and crop populations, to increase key production, sustainability and welfare traits, compared with the current best individuals.},
  archive      = {J_NG},
  author       = {Hayes, Ben J. and Mahony, Timothy J. and Villiers, Kira and Warburton, Christie and Kemper, Kathryn E. and Dinglasan, Eric and Robinson, Hannah and Powell, Owen and Voss-Fels, Kai and Godwin, Ian D. and Hickey, Lee T.},
  doi          = {10.1038/s41588-024-01942-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2310-2317},
  shortjournal = {Nat. Genet.},
  title        = {Potential approaches to create ultimate genotypes in crops and livestock},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping drug resistance variants in cancer, one base at a
time. <em>NG</em>, <em>56</em>(11), 2308–2309. (<a
href="https://doi.org/10.1038/s41588-024-01976-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA variants arising in the genome of cancer cells are a major cause of therapy failure, but for most variants, their effects on drug response are unknown. Base-editing screens provide a systematic approach to uncover the functions of cancer variants at scale, which might help to inform the use of precision cancer therapies.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01976-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2308-2309},
  shortjournal = {Nat. Genet.},
  title        = {Mapping drug resistance variants in cancer, one base at a time},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial mapping of primary and metastatic pancreatic tumor
ecosystems. <em>NG</em>, <em>56</em>(11), 2306–2307. (<a
href="https://doi.org/10.1038/s41588-024-01915-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study presents a spatial transcriptomic analysis of matched primary tumors, liver metastases and lymph node metastases from patients with pancreatic ductal adenocarcinoma. Using a tumor ecosystem approach, we uncovered notable tumor microenvironmental heterogeneity and marked differences between primary and metastatic samples, providing key insights into metastatic pancreatic cancer.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01915-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2306-2307},
  shortjournal = {Nat. Genet.},
  title        = {Spatial mapping of primary and metastatic pancreatic tumor ecosystems},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). DNA polymerase ε produces elevated c-to-t mutations at
methylated CpG dinucleotides. <em>NG</em>, <em>56</em>(11), 2304–2305.
(<a href="https://doi.org/10.1038/s41588-024-01946-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {C-to-T mutations in CpG dinucleotides are widespread in cancers and are also observed in normal cells. By developing and using a technique to quantify DNA polymerase errors (polymerase error rate sequencing, PER-seq), we reveal that C-to-T mutations in CpG dinucleotides constitute part of the error signature of both wild-type and mutant cancer-associated DNA polymerase ε.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01946-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2304-2305},
  shortjournal = {Nat. Genet.},
  title        = {DNA polymerase ε produces elevated C-to-T mutations at methylated CpG dinucleotides},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Methylated GCC repeat expansion in AFF3 associates with
intellectual disability. <em>NG</em>, <em>56</em>(11), 2302–2303. (<a
href="https://doi.org/10.1038/s41588-024-01918-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We identified methylated tandem repeat expansions that resemble the FMR1 CGG repeat that causes fragile X syndrome and investigated their association with traits in the UK Biobank. AFF3 expansion carriers had a 2.4-fold reduced probability of completing secondary education and were enriched in a cohort of individuals with intellectual disability.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01918-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2302-2303},
  shortjournal = {Nat. Genet.},
  title        = {Methylated GCC repeat expansion in AFF3 associates with intellectual disability},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic african ancestry modifies the biology of acute
myeloid leukemia. <em>NG</em>, <em>56</em>(11), 2299–2301. (<a
href="https://doi.org/10.1038/s41588-024-01869-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {African American patients were under-represented in the studies that led to the current patient classification system for acute myeloid leukemia (AML). A new in-depth analysis of the genetics of AML in African Americans suggests that this omission has implications for patient care.},
  archive      = {J_NG},
  author       = {Jiagge, Evelyn M.},
  doi          = {10.1038/s41588-024-01869-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2299-2301},
  shortjournal = {Nat. Genet.},
  title        = {Genetic african ancestry modifies the biology of acute myeloid leukemia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Parsing the spectrum of allelic architectures in diabetes.
<em>NG</em>, <em>56</em>(11), 2297–2298. (<a
href="https://doi.org/10.1038/s41588-024-01950-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Distinguishing ordinary diabetes from its monogenic forms has been one of the challenges in optimally managing the disease. Using high-quality imputation of rare variants and large databases, a study now defines the gray zone between the two and lays down a blueprint for objectively evaluating the related variants.},
  archive      = {J_NG},
  author       = {Li, Yangxi and Polychronakos, Constantin},
  doi          = {10.1038/s41588-024-01950-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2297-2298},
  shortjournal = {Nat. Genet.},
  title        = {Parsing the spectrum of allelic architectures in diabetes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Single-cell 3D multi-omics during human brain development.
<em>NG</em>, <em>56</em>(11), 2296. (<a
href="https://doi.org/10.1038/s41588-024-02008-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-024-02008-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2296},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell 3D multi-omics during human brain development},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Multimodal optical pooled screening with CRISPRmap.
<em>NG</em>, <em>56</em>(11), 2296. (<a
href="https://doi.org/10.1038/s41588-024-02010-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-02010-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2296},
  shortjournal = {Nat. Genet.},
  title        = {Multimodal optical pooled screening with CRISPRmap},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Optimizing combination immunotherapy in lung cancer.
<em>NG</em>, <em>56</em>(11), 2296. (<a
href="https://doi.org/10.1038/s41588-024-02011-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-024-02011-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2296},
  shortjournal = {Nat. Genet.},
  title        = {Optimizing combination immunotherapy in lung cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Missense variation affects protein cellular localization.
<em>NG</em>, <em>56</em>(11), 2295. (<a
href="https://doi.org/10.1038/s41588-024-02003-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-02003-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2295},
  shortjournal = {Nat. Genet.},
  title        = {Missense variation affects protein cellular localization},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Spatiotemporal regulation of germline genes by piRNAs.
<em>NG</em>, <em>56</em>(11), 2295. (<a
href="https://doi.org/10.1038/s41588-024-02004-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-024-02004-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2295},
  shortjournal = {Nat. Genet.},
  title        = {Spatiotemporal regulation of germline genes by piRNAs},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Genetics and dietary restriction impact lifespan.
<em>NG</em>, <em>56</em>(11), 2295. (<a
href="https://doi.org/10.1038/s41588-024-02005-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-02005-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2295},
  shortjournal = {Nat. Genet.},
  title        = {Genetics and dietary restriction impact lifespan},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Population survey of repeat expansion mutations.
<em>NG</em>, <em>56</em>(11), 2295. (<a
href="https://doi.org/10.1038/s41588-024-02007-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-024-02007-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2295},
  shortjournal = {Nat. Genet.},
  title        = {Population survey of repeat expansion mutations},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pushing the boundaries of rare disease diagnostics with the
help of the first undiagnosed hackathon. <em>NG</em>, <em>56</em>(11),
2287–2294. (<a
href="https://doi.org/10.1038/s41588-024-01941-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the first-ever Undiagnosed Hackathon, nearly 100 experts from 28 countries combined advanced phenotyping and genomic techniques for 48 hours, ultimately providing diagnoses to 40% of the previously undiagnosed families. This inspiring model demonstrates the power of multidisciplinary collaboration and patient partnership in precision diagnostics.},
  archive      = {J_NG},
  author       = {Delgado-Vega, Angelica Maria and Cederroth, Helene and Taylan, Fulya and Ekholm, Katja and Ek, Marlene and Thonberg, Håkan and Jemt, Anders and Nilsson, Daniel and Eisfeldt, Jesper and Bilgrav Saether, Kristine and Höijer, Ida and Akgun-Dogan, Ozlem and Asano, Yui and Barakat, Tahsin Stefan and Batkovskyte, Dominyka and Baynam, Gareth and Bodamer, Olaf and Chetruengchai, Wanna and Corcoran, Pádraic and Couse, Madeline and Danis, Daniel and Demidov, German and Dohi, Eisuke and Erhardsson, Mattias and Fernandez-Luna, Luis and Fujiwara, Toyofumi and Garg, Neha and Giugliani, Roberto and Gonzaga-Jauregui, Claudia and Grigelioniene, Giedre and Groza, Tudor and Gunnarsson, Cecilia and Hammarsjö, Anna and Hammond, Charles Kumi and Hatirnaz Ng, Özden and Hesketh, Sirisha and Hettiarachchi, Dineshani and Johansson Soller, Maria and Kirmani, Umn Ahmed and Kjellberg, Martin and Kvarnung, Malin and Kvlividze, Oleg and Lagerstedt-Robinson, Kristina and Lasko, Paul and Lassmann, Timo and Lau, Lynette Y. S. and Laurie, Steven and Lim, Weng Khong and Liu, Zhandong and Lysenkova Wiklander, Mariya and Makay, Prince and Maiga, Alassane Baneye and Maya-González, Carolina and Meyn, M. Stephen and Neethiraj, Ramprasad and Nigro, Vincenzo and Nordgren, Felix and Nordlund, Jessica and Orrsjö, Sara and Ottosson, Jesper and Ozbek, Ugur and Özdemir, Özkan and Partin, Clyde and Pearce, David A. and Peck, Raquel and Pedersen, Annie and Pettersson, Maria and Pongpanich, Monnat and Posada de la Paz, Manuel and Ramani, Arun and Romero, Juan Andres and Romero, Vanessa I. and Rosenquist, Richard and Saw, Aung Min and Spencer, Matthew and Stattin, Eva-Lena and Srichomthong, Chalurmpon and Tapia-Paez, Isabel and Taruscio, Domenica and Taylor, Julie P. and Tkemaladze, Tinatin and Tully, Ian and Tümer, Zeynep and van Zelst-Stams, Wendy A. G. and Verloes, Alain and Västerviga, Emma and Wang, Sailan and Yang, Rachel and Yamamoto, Shinya and Yépez, Vicente A. and Zhang, Qing and Shotelersuk, Vorasuk and Wiafe, Samuel Agyei and Alanay, Yasemin and Botto, Lorenzo D. and Kirmani, Salman and Lumaka, Aimé and Palmer, Elizabeth Emma and Puri, Ratna Dua and Wirta, Valtteri and Lindstrand, Anna and Buske, Orion J. and Cederroth, Mikk and Nordgren, Ann},
  doi          = {10.1038/s41588-024-01941-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2287-2294},
  shortjournal = {Nat. Genet.},
  title        = {Pushing the boundaries of rare disease diagnostics with the help of the first undiagnosed hackathon},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Standardizing variant naming in literature with
VariantValidator to increase diagnostic rates. <em>NG</em>,
<em>56</em>(11), 2284–2286. (<a
href="https://doi.org/10.1038/s41588-024-01938-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate naming of genetic variants is essential to identify clinical data that interpret the consequences of such variants. In partnership with the Human Genome Organization, we advocate for integration of VariantValidator in publishing of journals and databases, to improve the quality of shared genetic data and ultimately patient outcomes.},
  archive      = {J_NG},
  author       = {Freeman, Peter J. and Wagstaff, John F. and Fokkema, Ivo F. A. C. and Cutting, Garry R. and Rehm, Heidi L. and Davies, Angela C. and den Dunnen, Johan T. and Gretton, Liam J. and Dalgleish, Raymond},
  doi          = {10.1038/s41588-024-01938-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2284-2286},
  shortjournal = {Nat. Genet.},
  title        = {Standardizing variant naming in literature with VariantValidator to increase diagnostic rates},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving reporting standards for genetic variants.
<em>NG</em>, <em>56</em>(11), 2283. (<a
href="https://doi.org/10.1038/s41588-024-02002-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The standardized naming of gene variants in both databases and publications is crucial to ensure their discoverability and clinical application. Efforts are underway in conjunction with the Human Genome Organization (HUGO) to develop a field standard for variant reporting through the use of validation software prior to publication.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-02002-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2283},
  shortjournal = {Nat. Genet.},
  title        = {Improving reporting standards for genetic variants},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Publisher correction: Statistically and functionally
fine-mapped blood eQTLs and pQTLs from 1,405 humans reveal distinct
regulation patterns and disease relevance. <em>NG</em>, <em>56</em>(10),
2281. (<a href="https://doi.org/10.1038/s41588-024-01959-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Wang, Qingbo S. and Hasegawa, Takanori and Namkoong, Ho and Saiki, Ryunosuke and Edahiro, Ryuya and Sonehara, Kyuto and Tanaka, Hiromu and Azekawa, Shuhei and Chubachi, Shotaro and Takahashi, Yugo and Sakaue, Saori and Namba, Shinichi and Yamamoto, Kenichi and Shiraishi, Yuichi and Chiba, Kenichi and Tanaka, Hiroko and Makishima, Hideki and Nannya, Yasuhito and Zhang, Zicong and Tsujikawa, Rika and Koike, Ryuji and Takano, Tomomi and Ishii, Makoto and Kimura, Akinori and Inoue, Fumitaka and Kanai, Takanori and Fukunaga, Koichi and Ogawa, Seishi and Imoto, Seiya and Miyano, Satoru and Okada, Yukinori},
  doi          = {10.1038/s41588-024-01959-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2281},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Statistically and functionally fine-mapped blood eQTLs and pQTLs from 1,405 humans reveal distinct regulation patterns and disease relevance},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integration of variant annotations using deep set networks
boosts rare variant association testing. <em>NG</em>, <em>56</em>(10),
2271–2280. (<a
href="https://doi.org/10.1038/s41588-024-01919-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rare genetic variants can have strong effects on phenotypes, yet accounting for rare variants in genetic analyses is statistically challenging due to the limited number of allele carriers and the burden of multiple testing. While rich variant annotations promise to enable well-powered rare variant association tests, methods integrating variant annotations in a data-driven manner are lacking. Here we propose deep rare variant association testing (DeepRVAT), a model based on set neural networks that learns a trait-agnostic gene impairment score from rare variant annotations and phenotypes, enabling both gene discovery and trait prediction. On 34 quantitative and 63 binary traits, using whole-exome-sequencing data from UK Biobank, we find that DeepRVAT yields substantial gains in gene discoveries and improved detection of individuals at high genetic risk. Finally, we demonstrate how DeepRVAT enables calibrated and computationally efficient rare variant tests at biobank scale, aiding the discovery of genetic risk factors for human disease traits. Deep rare variant association testing (DeepRVAT) is a deep set neural network model that flexibly integrates rare variant annotations into a trait-agnostic gene impairment score. These scores improve association testing and polygenic risk prediction.},
  archive      = {J_NG},
  author       = {Clarke, Brian and Holtkamp, Eva and Öztürk, Hakime and Mück, Marcel and Wahlberg, Magnus and Meyer, Kayla and Munzlinger, Felix and Brechtmann, Felix and Hölzlwimmer, Florian R. and Lindner, Jonas and Chen, Zhifen and Gagneur, Julien and Stegle, Oliver},
  doi          = {10.1038/s41588-024-01919-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2271-2280},
  shortjournal = {Nat. Genet.},
  title        = {Integration of variant annotations using deep set networks boosts rare variant association testing},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Custom microfluidic chip design enables cost-effective
three-dimensional spatiotemporal transcriptomics with a wide field of
view. <em>NG</em>, <em>56</em>(10), 2259–2270. (<a
href="https://doi.org/10.1038/s41588-024-01906-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomic techniques offer unprecedented insights into the molecular organization of complex tissues. However, integrating cost-effectiveness, high throughput, a wide field of view and compatibility with three-dimensional (3D) volumes has been challenging. Here we introduce microfluidics-assisted grid chips for spatial transcriptome sequencing (MAGIC-seq), a new method that combines carbodiimide chemistry, spatial combinatorial indexing and innovative microfluidics design. This technique allows sensitive and reproducible profiling of diverse tissue types, achieving an eightfold increase in throughput, minimal cost and reduced batch effects. MAGIC-seq breaks conventional microfluidics limits by enhancing barcoding efficiency and enables analysis of whole postnatal mouse sections, providing comprehensive cellular structure elucidation at near single-cell resolution, uncovering transcriptional variations and dynamic trajectories of mouse organogenesis. Our 3D transcriptomic atlas of the developing mouse brain, consisting of 93 sections, reveals the molecular and cellular landscape, serving as a valuable resource for neuroscience and developmental biology. Overall, MAGIC-seq is a high-throughput, cost-effective, large field of view and versatile method for spatial transcriptomic studies. Microfluidics-assisted grid chips for spatial transcriptome sequencing (MAGIC-seq) is a spatial transcriptomics method combining multiple-grid microfluidic design and prefabricated DNA arrays for increased throughput and reduced cost, with applications for large fields of view and 3D spatial mapping.},
  archive      = {J_NG},
  author       = {Zhu, Junjie and Pang, Kun and Hu, Beiyu and He, Ruiqiao and Wang, Ning and Jiang, Zewen and Ji, Peifeng and Zhao, Fangqing},
  doi          = {10.1038/s41588-024-01906-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2259-2270},
  shortjournal = {Nat. Genet.},
  title        = {Custom microfluidic chip design enables cost-effective three-dimensional spatiotemporal transcriptomics with a wide field of view},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-quality genome of a modern soybean cultivar and
resequencing of 547 accessions provide insights into the role of
structural variation. <em>NG</em>, <em>56</em>(10), 2247–2258. (<a
href="https://doi.org/10.1038/s41588-024-01901-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soybean provides protein, oil and multiple health-related compounds. Understanding the effects of structural variations (SVs) on economic traits in modern breeding is important for soybean improvement. Here we assembled the high-quality genome of modern cultivar Nongdadou2 (NDD2) and identified 25,814 SV–gene pairs compared to 29 reported genomes, with 13 NDD2-private SVs validated in 547 deep-resequencing (average = 18.05-fold) accessions, which advances our understanding of genomic variation biology. We found some insertions/deletions involved in seed protein and weight formation, an inversion related to adaptation to drought and a large intertranslocation implicated in a key divergence event in soybean. Of 749,714 SVs from 547 accessions, 6,013 were significantly associated with 22 yield-related and seed-quality-related traits determined in ten location × year environments. We uncovered 1,761 associated SVs that hit genes or regulatory regions, with 12 in GmMQT influencing oil and isoflavone contents. Our work provides resources and insights into SV roles in soybean improvement. High-quality genome of the modern soybean cultivar Nongdadou2 and deep resequencing of 547 accessions provide insights into the role of structural variations in soybean improvement.},
  archive      = {J_NG},
  author       = {Zhang, Caiying and Shao, Zhenqi and Kong, Youbin and Du, Hui and Li, Wenlong and Yang, Zhanwu and Li, Xiangkong and Ke, Huifeng and Sun, Zhengwen and Shao, Jiabiao and Chen, Shiliang and Zhang, Hua and Chu, Jiahao and Xing, Xinzhu and Tian, Rui and Qin, Ning and Li, Junru and Huang, Meihong and Sun, Yaqian and Huo, Xiaobo and Meng, Chengsheng and Wang, Guoning and Liu, Yuan and Ma, Zhiying and Tian, Shilin and Li, Xihuan},
  doi          = {10.1038/s41588-024-01901-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2247-2258},
  shortjournal = {Nat. Genet.},
  title        = {High-quality genome of a modern soybean cultivar and resequencing of 547 accessions provide insights into the role of structural variation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Widespread position-dependent transcriptional regulatory
sequences in plants. <em>NG</em>, <em>56</em>(10), 2238–2246. (<a
href="https://doi.org/10.1038/s41588-024-01907-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Much of what we know about eukaryotic transcription stems from animals and yeast; however, plants evolved separately for over a billion years, leaving ample time for divergence in transcriptional regulation. Here we set out to elucidate fundamental properties of cis-regulatory sequences in plants. Using massively parallel reporter assays across four plant species, we demonstrate the central role of sequences downstream of the transcription start site (TSS) in transcriptional regulation. Unlike animal enhancers that are position independent, plant regulatory elements depend on their position, as altering their location relative to the TSS significantly affects transcription. We highlight the importance of the region downstream of the TSS in regulating transcription by identifying a DNA motif that is conserved across vascular plants and is sufficient to enhance gene expression in a dose-dependent manner. The identification of a large number of position-dependent enhancers points to fundamental differences in gene regulation between plants and animals. Massively parallel reporter assays in four plant species show that transcriptional regulatory elements are position dependent with enrichment downstream of the transcription start site, particularly GATC motifs with strong effects in vascular plants.},
  archive      = {J_NG},
  author       = {Voichek, Yoav and Hristova, Gabriela and Mollá-Morales, Almudena and Weigel, Detlef and Nordborg, Magnus},
  doi          = {10.1038/s41588-024-01907-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2238-2246},
  shortjournal = {Nat. Genet.},
  title        = {Widespread position-dependent transcriptional regulatory sequences in plants},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Antagonism between H3K27me3 and genome–lamina association
drives atypical spatial genome organization in the totipotent embryo.
<em>NG</em>, <em>56</em>(10), 2228–2237. (<a
href="https://doi.org/10.1038/s41588-024-01902-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In mammals, early embryonic development exhibits highly unusual spatial positioning of genomic regions at the nuclear lamina, but the mechanisms underpinning this atypical genome organization remain elusive. Here, we generated single-cell profiles of lamina-associated domains (LADs) coupled with transcriptomics, which revealed a striking overlap between preimplantation-specific LAD dissociation and noncanonical broad domains of H3K27me3. Loss of H3K27me3 resulted in a restoration of canonical LAD profiles, suggesting an antagonistic relationship between lamina association and H3K27me3. Tethering of H3K27me3 to the nuclear periphery showed that the resultant relocalization is partially dependent on the underlying DNA sequence. Collectively, our results suggest that the atypical organization of LADs in early developmental stages is the result of a tug-of-war between intrinsic affinity for the nuclear lamina and H3K27me3, constrained by the available space at the nuclear periphery. This study provides detailed insight into the molecular mechanisms regulating nuclear organization during early mammalian development. Single-cell profiling of lamina-associated domains (LADs) during early mouse development reveals an overlap between preimplantation-specific LAD dissociation and noncanonical broad H3K27me3 domains. Loss of H3K27me3 restores canonical LAD profiles.},
  archive      = {J_NG},
  author       = {Guerreiro, Isabel and Rang, Franka J. and Kawamura, Yumiko K. and Kroon-Veenboer, Carla and Korving, Jeroen and Groenveld, Femke C. and van Beek, Ramada E. and Lochs, Silke J. A. and Boele, Ellen and Peters, Antoine H. M. F. and Kind, Jop},
  doi          = {10.1038/s41588-024-01902-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2228-2237},
  shortjournal = {Nat. Genet.},
  title        = {Antagonism between H3K27me3 and genome–lamina association drives atypical spatial genome organization in the totipotent embryo},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pharmacological restriction of genomic binding sites
redirects PU.1 pioneer transcription factor activity. <em>NG</em>,
<em>56</em>(10), 2213–2227. (<a
href="https://doi.org/10.1038/s41588-024-01911-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factor (TF) DNA-binding dynamics govern cell fate and identity. However, our ability to pharmacologically control TF localization is limited. Here we leverage chemically driven binding site restriction leading to robust and DNA-sequence-specific redistribution of PU.1, a pioneer TF pertinent to many hematopoietic malignancies. Through an innovative technique, ‘CLICK-on-CUT&amp;amp;Tag’, we characterize the hierarchy of de novo PU.1 motifs, predicting occupancy in the PU.1 cistrome under binding site restriction. Temporal and single-molecule studies of binding site restriction uncover the pioneering dynamics of native PU.1 and identify the paradoxical activation of an alternate target gene set driven by PU.1 localization to second-tier binding sites. These transcriptional changes were corroborated by genetic blockade and site-specific reporter assays. Binding site restriction and subsequent PU.1 network rewiring causes primary human leukemia cells to differentiate. In summary, pharmacologically induced TF redistribution can be harnessed to govern TF localization, actuate alternate gene networks and direct cell fate. Chemically driven blockade of PU.1 binding sites leads to its genome-wide redistribution. PU.1 network rewiring causes human acute myeloid leukemia cells to differentiate.},
  archive      = {J_NG},
  author       = {Taylor, Samuel J. and Stauber, Jacob and Bohorquez, Oliver and Tatsumi, Goichi and Kumari, Rajni and Chakraborty, Joyeeta and Bartholdy, Boris A. and Schwenger, Emily and Sundaravel, Sriram and Farahat, Abdelbasset A. and Wheat, Justin C. and Goldfinger, Mendel and Verma, Amit and Kumar, Arvind and Boykin, David W. and Stengel, Kristy R. and Poon, Gregory M. K. and Steidl, Ulrich},
  doi          = {10.1038/s41588-024-01911-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2213-2227},
  shortjournal = {Nat. Genet.},
  title        = {Pharmacological restriction of genomic binding sites redirects PU.1 pioneer transcription factor activity},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Context transcription factors establish cooperative
environments and mediate enhancer communication. <em>NG</em>,
<em>56</em>(10), 2199–2212. (<a
href="https://doi.org/10.1038/s41588-024-01892-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many enhancers control gene expression by assembling regulatory factor clusters, also referred to as condensates. This process is vital for facilitating enhancer communication and establishing cellular identity. However, how DNA sequence and transcription factor (TF) binding instruct the formation of high regulatory factor environments remains poorly understood. Here we developed a new approach leveraging enhancer-centric chromatin accessibility quantitative trait loci (caQTLs) to nominate regulatory factor clusters genome-wide. By analyzing TF-binding signatures within the context of caQTLs and comparing episomal versus endogenous enhancer activities, we discovered a class of regulators, ‘context-only’ TFs, that amplify the activity of cell type-specific caQTL-binding TFs, that is, ‘context-initiator’ TFs. Similar to super-enhancers, enhancers enriched for context-only TF-binding sites display high coactivator binding and sensitivity to bromodomain-inhibiting molecules. We further show that binding sites for context-only and context-initiator TFs underlie enhancer coordination, providing a mechanistic rationale for how a loose TF syntax confers regulatory specificity. This study identifies context-only transcription factors (TFs), a TF class that enhances DNA accessibility initiated by cell type-specific TFs and establishes cooperative environments. Enhancers enriched with motifs of both TF classes show high coactivator binding, enhanced coordination and sensitivity to bromodomain inhibitors.},
  archive      = {J_NG},
  author       = {Kribelbauer-Swietek, Judith F. and Pushkarev, Olga and Gardeux, Vincent and Faltejskova, Katerina and Russeil, Julie and van Mierlo, Guido and Deplancke, Bart},
  doi          = {10.1038/s41588-024-01892-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2199-2212},
  shortjournal = {Nat. Genet.},
  title        = {Context transcription factors establish cooperative environments and mediate enhancer communication},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Stepwise de novo establishment of inactive x chromosome
architecture in early development. <em>NG</em>, <em>56</em>(10),
2185–2198. (<a
href="https://doi.org/10.1038/s41588-024-01897-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {X chromosome inactivation triggers a dramatic reprogramming of transcription and chromosome architecture. However, how the chromatin organization of inactive X chromosome is established de novo in vivo remains elusive. Here, we identified an Xist-separated megadomain structure (X-megadomains) on the inactive X chromosome in mouse extraembryonic lineages and extraembryonic endoderm (XEN) cell lines, and transiently in the embryonic lineages, before Dxz4-delineated megadomain formation at later stages in a strain-specific manner. X-megadomain boundary coincides with strong enhancer activities and cohesin binding in an Xist regulatory region required for proper Xist activation in early embryos. Xist regulatory region disruption or cohesin degradation impaired X-megadomains in extraembryonic endoderm cells and caused ectopic activation of regulatory elements and genes near Xist, indicating that cohesin loading at regulatory elements promotes X-megadomains and confines local gene activities. These data reveal stepwise X chromosome folding and transcriptional regulation to achieve both essential gene activation and global silencing during the early stages of X chromosome inactivation. Hi-C analysis identifies Xist-separated megadomains (X-megadomains) on the inactive X chromosome in mouse early embryos. Cohesin loading at Xist regulatory elements promotes X-megadomain formation and restricts nearby gene activity.},
  archive      = {J_NG},
  author       = {Du, Zhenhai and Hu, Liangjun and Zou, Zhuoning and Liu, Meishuo and Li, Zihan and Lu, Xukun and Harris, Clair and Xiang, Yunlong and Chen, Fengling and Yu, Guang and Xu, Kai and Kong, Feng and Xu, Qianhua and Huang, Bo and Liu, Ling and Fan, Qiang and Wang, Haifeng and Kalantry, Sundeep and Xie, Wei},
  doi          = {10.1038/s41588-024-01897-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2185-2198},
  shortjournal = {Nat. Genet.},
  title        = {Stepwise de novo establishment of inactive x chromosome architecture in early development},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Defining heritability, plasticity, and transition dynamics
of cellular phenotypes in somatic evolution. <em>NG</em>,
<em>56</em>(10), 2174–2184. (<a
href="https://doi.org/10.1038/s41588-024-01920-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell sequencing has characterized cell state heterogeneity across diverse healthy and malignant tissues. However, the plasticity or heritability of these cell states remains largely unknown. To address this, we introduce PATH (phylogenetic analysis of trait heritability), a framework to quantify cell state heritability versus plasticity and infer cell state transition and proliferation dynamics from single-cell lineage tracing data. Applying PATH to a mouse model of pancreatic cancer, we observed heritability at the ends of the epithelial-to-mesenchymal transition spectrum, with higher plasticity at more intermediate states. In primary glioblastoma, we identified bidirectional transitions between stem- and mesenchymal-like cells, which use the astrocyte-like state as an intermediary. Finally, we reconstructed a phylogeny from single-cell whole-genome sequencing in B cell acute lymphoblastic leukemia and delineated the heritability of B cell differentiation states linked with genetic drivers. Altogether, PATH replaces qualitative conceptions of plasticity with quantitative measures, offering a framework to study somatic evolution. Phylogenetic analysis of trait heritability (PATH) applies phylogenetic correlations to single-cell lineage tracing data, quantifying cell state plasticity and transition probabilities. PATH offers insights into cell state heritability and transition dynamics in cancers.},
  archive      = {J_NG},
  author       = {Schiffman, Joshua S. and D’Avino, Andrew R. and Prieto, Tamara and Pang, Yakun and Fan, Yilin and Rajagopalan, Srinivas and Potenski, Catherine and Hara, Toshiro and Suvà, Mario L. and Gawad, Charles and Landau, Dan A.},
  doi          = {10.1038/s41588-024-01920-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2174-2184},
  shortjournal = {Nat. Genet.},
  title        = {Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Self-sustaining long-term 3D epithelioid cultures reveal
drivers of clonal expansion in esophageal epithelium. <em>NG</em>,
<em>56</em>(10), 2158–2173. (<a
href="https://doi.org/10.1038/s41588-024-01875-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aging epithelia are colonized by somatic mutations, which are subjected to selection influenced by intrinsic and extrinsic factors. The lack of suitable culture systems has slowed the study of this and other long-term biological processes. Here, we describe epithelioids, a facile, cost-effective method of culturing multiple mouse and human epithelia. Esophageal epithelioids self-maintain without passaging for at least 1 year, maintaining a three-dimensional structure with proliferative basal cells that differentiate into suprabasal cells, which eventually shed and retain genomic stability. Live imaging over 5 months showed that epithelioids replicate in vivo cell dynamics. Epithelioids support genetic manipulation and enable the study of mutant cell competition and selection in three-dimensional epithelia, and show how anti-cancer treatments modulate competition between transformed and wild-type cells. Finally, a targeted CRISPR–Cas9 screen shows that epithelioids recapitulate mutant gene selection in aging human esophagus and identifies additional drivers of clonal expansion, resolving the genetic networks underpinning competitive fitness. Epithelioids are genetically stable, self-sustaining three-dimensional cultures. They may be used to investigate various aspects of epithelial biology over several months without need for passaging. In this paper, mouse epithelioids are used to identify drivers of clonal expansion in the esophagus.},
  archive      = {J_NG},
  author       = {Herms, Albert and Fernandez-Antoran, David and Alcolea, Maria P. and Kalogeropoulou, Argyro and Banerjee, Ujjwal and Piedrafita, Gabriel and Abby, Emilie and Valverde-Lopez, Jose Antonio and Ferreira, Inês S. and Caseda, Irene and Bejar, Maria T. and Dentro, Stefan C. and Vidal-Notari, Sara and Ong, Swee Hoe and Colom, Bartomeu and Murai, Kasumi and King, Charlotte and Mahbubani, Krishnaa and Saeb-Parsy, Kourosh and Lowe, Alan R. and Gerstung, Moritz and Jones, Philip H.},
  doi          = {10.1038/s41588-024-01875-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2158-2173},
  shortjournal = {Nat. Genet.},
  title        = {Self-sustaining long-term 3D epithelioid cultures reveal drivers of clonal expansion in esophageal epithelium},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Organismal metabolism regulates the expansion of oncogenic
PIK3CA mutant clones in normal esophagus. <em>NG</em>, <em>56</em>(10),
2144–2157. (<a
href="https://doi.org/10.1038/s41588-024-01891-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oncogenic PIK3CA mutations generate large clones in aging human esophagus. Here we investigate the behavior of Pik3ca mutant clones in the normal esophageal epithelium of transgenic mice. Expression of a heterozygous Pik3caH1047R mutation drives clonal expansion by tilting cell fate toward proliferation. CRISPR screening and inhibitor treatment of primary esophageal keratinocytes confirmed the PI3K–mTOR pathway increased mutant cell competitive fitness. The antidiabetic drug metformin reduced mutant cell advantage in vivo and in vitro. Conversely, metabolic conditions such as type 1 diabetes or diet-induced obesity enhanced the competitive fitness of Pik3caH1047R cells. Consistently, we found a higher density of PIK3CA gain-of-function mutations in the esophagus of individuals with high body mass index compared with those with normal weight. We conclude that the metabolic environment selectively influences the evolution of the normal epithelial mutational landscape. Clinically feasible interventions to even out signaling imbalances between wild-type and mutant cells may limit the expansion of oncogenic mutants in normal tissues. Reducing the competitive advantage conferred by driver mutations can abrogate expansions of mutant clones in healthy tissue in mice. This suggests ways to prevent cancer and other diseases that are associated with somatic mutations in humans.},
  archive      = {J_NG},
  author       = {Herms, Albert and Colom, Bartomeu and Piedrafita, Gabriel and Kalogeropoulou, Argyro and Banerjee, Ujjwal and King, Charlotte and Abby, Emilie and Murai, Kasumi and Caseda, Irene and Fernandez-Antoran, David and Ong, Swee Hoe and Hall, Michael W. J. and Bryant, Christopher and Sood, Roshan K. and Fowler, Joanna C. and Pol, Albert and Frezza, Christian and Vanhaesebroeck, Bart and Jones, Philip H.},
  doi          = {10.1038/s41588-024-01891-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2144-2157},
  shortjournal = {Nat. Genet.},
  title        = {Organismal metabolism regulates the expansion of oncogenic PIK3CA mutant clones in normal esophagus},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). NSD2 is a requisite subunit of the AR/FOXA1 neo-enhanceosome
in promoting prostate tumorigenesis. <em>NG</em>, <em>56</em>(10),
2132–2143. (<a
href="https://doi.org/10.1038/s41588-024-01893-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Androgen receptor (AR) is a ligand-responsive transcription factor that drives terminal differentiation of the prostatic luminal epithelia. By contrast, in tumors originating from these cells, AR chromatin occupancy is extensively reprogrammed to activate malignant phenotypes, the molecular mechanisms of which remain unknown. Here, we show that tumor-specific AR enhancers are critically reliant on H3K36 dimethyltransferase activity of NSD2. NSD2 expression is abnormally induced in prostate cancer, where its inactivation impairs AR transactivation potential by disrupting over 65% of its cistrome. NSD2-dependent AR sites distinctively harbor the chimeric FOXA1:AR half-motif, which exclusively comprise tumor-specific AR enhancer circuitries defined from patient specimens. NSD2 inactivation also engenders increased dependency on the NSD1 paralog, and a dual NSD1/2 PROTAC degrader is preferentially cytotoxic in AR-dependent prostate cancer models. Altogether, we characterize NSD2 as an essential AR neo-enhanceosome subunit that enables its oncogenic activity, and position NSD1/2 as viable co-targets in advanced prostate cancer. CRISPR screen identifies coactivators of the androgen receptor (AR) complex, including NSD2. NSD2 contributes to AR cistrome reprogramming during prostate cancer progression, and its degradation via a novel PROTAC reduces prostate cancer cell viability in vitro.},
  archive      = {J_NG},
  author       = {Parolia, Abhijit and Eyunni, Sanjana and Verma, Brijesh Kumar and Young, Eleanor and Liu, Yihan and Liu, Lianchao and George, James and Aras, Shweta and Das, Chandan Kanta and Mannan, Rahul and ur Rasool, Reyaz and Mitchell-Velasquez, Erick and Mahapatra, Somnath and Luo, Jie and Carson, Sandra E. and Xiao, Lanbo and Gajjala, Prathibha R. and Venkatesh, Sharan and Jaber, Mustapha and Wang, Xiaoju and He, Tongchen and Qiao, Yuanyuan and Pang, Matthew and Zhang, Yuping and Tien, Jean Ching-Yi and Louw, Micheala and Alhusayan, Mohammed and Cao, Xuhong and Su, Fengyun and Tavana, Omid and Hou, Caiyun and Wang, Zhen and Ding, Ke and Chinnaiyan, Arul M. and Asangani, Irfan A.},
  doi          = {10.1038/s41588-024-01893-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2132-2143},
  shortjournal = {Nat. Genet.},
  title        = {NSD2 is a requisite subunit of the AR/FOXA1 neo-enhanceosome in promoting prostate tumorigenesis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MHC hammer reveals genetic and non-genetic HLA disruption in
cancer evolution. <em>NG</em>, <em>56</em>(10), 2121–2131. (<a
href="https://doi.org/10.1038/s41588-024-01883-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Disruption of the class I human leukocyte antigen (HLA) molecules has important implications for immune evasion and tumor evolution. We developed major histocompatibility complex loss of heterozygosity (LOH), allele-specific mutation and measurement of expression and repression (MHC Hammer). We identified extensive variability in HLA allelic expression and pervasive HLA alternative splicing in normal lung and breast tissue. In lung TRACERx and lung and breast TCGA cohorts, 61% of lung adenocarcinoma (LUAD), 76% of lung squamous cell carcinoma (LUSC) and 35% of estrogen receptor-positive (ER+) cancers harbored class I HLA transcriptional repression, while HLA tumor-enriched alternative splicing occurred in 31%, 11% and 15% of LUAD, LUSC and ER+ cancers. Consistent with the importance of HLA dysfunction in tumor evolution, in LUADs, HLA LOH was associated with metastasis and LUAD primary tumor regions seeding a metastasis had a lower effective neoantigen burden than non-seeding regions. These data highlight the extent and importance of HLA transcriptomic disruption, including repression and alternative splicing in cancer evolution. Major histocompatibility complex (MHC) loss of heterozygosity, allele-specific mutation and measurement of expression and repression (MHC Hammer) detects disruption to human leukocyte antigens due to mutations, loss of heterogeneity, altered gene expression or alternative splicing. Applied to lung and breast cancer datasets, the tool shows that these aberrations are common across cancer and can have clinical implications.},
  archive      = {J_NG},
  author       = {Puttick, Clare and Jones, Thomas P. and Leung, Michelle M. and Galvez-Cancino, Felipe and Liu, Jiali and Varas-Godoy, Manuel and Rowan, Andrew and Pich, Oriol and Martinez-Ruiz, Carlos and Bentham, Robert and Dijkstra, Krijn K. and Black, James R. M. and Rosenthal, Rachel and Kanu, Nnennaya and Litchfield, Kevin and Salgado, Roberto and Moore, David A. and Van Loo, Peter and Jamal-Hanjani, Mariam and Quezada, Sergio A. and Swanton, Charles and McGranahan, Nicholas},
  doi          = {10.1038/s41588-024-01883-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2121-2131},
  shortjournal = {Nat. Genet.},
  title        = {MHC hammer reveals genetic and non-genetic HLA disruption in cancer evolution},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Five latent factors underlie response to immunotherapy.
<em>NG</em>, <em>56</em>(10), 2112–2120. (<a
href="https://doi.org/10.1038/s41588-024-01899-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Only a subset of patients treated with immune checkpoint inhibitors (CPIs) respond to the treatment, and distinguishing responders from non-responders is a major challenge. Many proposed biomarkers of CPI response and survival probably represent alternative measurements of the same aspects of the tumor, its microenvironment or the host. Thus, we currently ignore how many truly independent biomarkers there are. With an unbiased analysis of genomics, transcriptomics and clinical data of a cohort of patients with metastatic tumors (n = 479), we discovered five orthogonal latent factors: tumor mutation burden, T cell effective infiltration, transforming growth factor-beta activity in the microenvironment, prior treatment and tumor proliferative potential. Their association with CPI response and survival was observed across all tumor types and validated across six independent cohorts (n = 1,491). These five latent factors constitute a frame of reference to organize current and future knowledge on biomarkers of CPI response and survival. Analysis of human tumor datasets shows that all features that appear significantly associated with immunotherapy response and survival may be collapsed into five latent factors: tumor mutation burden, T cell effective infiltration, TGF-β activity in the microenvironment, prior treatment and tumor proliferative potential.},
  archive      = {J_NG},
  author       = {Usset, Joseph and Rosendahl Huber, Axel and Andrianova, Maria A. and Batlle, Eduard and Carles, Joan and Cuppen, Edwin and Elez, Elena and Felip, Enriqueta and Gómez-Rey, Marina and Lo Giacco, Deborah and Martinez-Jimenez, Francisco and Muñoz-Couselo, Eva and Siu, Lillian L. and Tabernero, Josep and Vivancos, Ana and Muiños, Ferran and Gonzalez-Perez, Abel and Lopez-Bigas, Nuria},
  doi          = {10.1038/s41588-024-01899-0},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2112-2120},
  shortjournal = {Nat. Genet.},
  title        = {Five latent factors underlie response to immunotherapy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic prioritization of functional variants and
effector genes underlying colorectal cancer risk. <em>NG</em>,
<em>56</em>(10), 2104–2111. (<a
href="https://doi.org/10.1038/s41588-024-01900-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies of colorectal cancer (CRC) have identified 170 autosomal risk loci. However, for most of these, the functional variants and their target genes are unknown. Here, we perform statistical fine-mapping incorporating tissue-specific epigenetic annotations and massively parallel reporter assays to systematically prioritize functional variants for each CRC risk locus. We identify plausible causal variants for the 170 risk loci, with a single variant for 40. We link these variants to 208 target genes by analyzing colon-specific quantitative trait loci and implementing the activity-by-contact model, which integrates epigenomic features and Micro-C data, to predict enhancer–gene connections. By deciphering CRC risk loci, we identify direct links between risk variants and target genes, providing further insight into the molecular basis of CRC susceptibility and highlighting potential pharmaceutical targets for prevention and treatment. This study uses a combination of in silico and experimental techniques to ascribe target genes to 170 risk loci for colorectal cancer.},
  archive      = {J_NG},
  author       = {Law, Philip J. and Studd, James and Smith, James and Vijayakrishnan, Jayaram and Harris, Bradley T. and Mandelia, Maria and Mills, Charlie and Dunlop, Malcolm G. and Houlston, Richard S.},
  doi          = {10.1038/s41588-024-01900-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2104-2111},
  shortjournal = {Nat. Genet.},
  title        = {Systematic prioritization of functional variants and effector genes underlying colorectal cancer risk},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Heterogeneous genetic architectures of prostate cancer
susceptibility in sub-saharan africa. <em>NG</em>, <em>56</em>(10),
2093–2103. (<a
href="https://doi.org/10.1038/s41588-024-01931-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Men of African descent have the highest prostate cancer incidence and mortality rates, yet the genetic basis of prostate cancer in African men has been understudied. We used genomic data from 3,963 cases and 3,509 controls from Ghana, Nigeria, Senegal, South Africa and Uganda to infer ancestry-specific genetic architectures and fine-map disease associations. Fifteen independent associations at 8q24.21, 6q22.1 and 11q13.3 reached genome-wide significance, including four new associations. Intriguingly, multiple lead associations are private alleles, a pattern arising from recent mutations and the out-of-Africa bottleneck. These African-specific alleles contribute to haplotypes with odds ratios above 2.4. We found that the genetic architecture of prostate cancer differs across Africa, with effect size differences contributing more to this heterogeneity than allele frequency differences. Population genetic analyses reveal that African prostate cancer associations are largely governed by neutral evolution. Collectively, our findings emphasize the utility of conducting genetic studies that use diverse populations. Genome-wide association analyses of prostate cancer in men from sub-Saharan Africa identify population-specific risk variants and regional differences in effect sizes. Founder effects contribute to continental differences in the genetic architecture of prostate cancer.},
  archive      = {J_NG},
  author       = {Janivara, Rohini and Chen, Wenlong C. and Hazra, Ujani and Baichoo, Shakuntala and Agalliu, Ilir and Kachambwa, Paidamoyo and Simonti, Corrine N. and Brown, Lyda M. and Tambe, Saanika P. and Kim, Michelle S. and Harlemon, Maxine and Jalloh, Mohamed and Muzondiwa, Dillon and Naidoo, Daphne and Ajayi, Olabode O. and Snyper, Nana Yaa and Niang, Lamine and Diop, Halimatou and Ndoye, Medina and Mensah, James E. and Abrahams, Afua O. D. and Biritwum, Richard and Adjei, Andrew A. and Adebiyi, Akindele O. and Shittu, Olayiwola and Ogunbiyi, Olufemi and Adebayo, Sikiru and Nwegbu, Maxwell M. and Ajibola, Hafees O. and Oluwole, Olabode P. and Jamda, Mustapha A. and Pentz, Audrey and Haiman, Christopher A. and Spies, Petrus V. and van der Merwe, André and Cook, Michael B. and Chanock, Stephen J. and Berndt, Sonja I. and Watya, Stephen and Lubwama, Alexander and Muchengeti, Mazvita and Doherty, Sean and Smyth, Natalie and Lounsbury, David and Fortier, Brian and Rohan, Thomas E. and Jacobson, Judith S. and Neugut, Alfred I. and Hsing, Ann W. and Gusev, Alexander and Aisuodionoe-Shadrach, Oseremen I. and Joffe, Maureen and Adusei, Ben and Gueye, Serigne M. and Fernandez, Pedro W. and McBride, Jo and Andrews, Caroline and Petersen, Lindsay N. and Lachance, Joseph and Rebbeck, Timothy R.},
  doi          = {10.1038/s41588-024-01931-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2093-2103},
  shortjournal = {Nat. Genet.},
  title        = {Heterogeneous genetic architectures of prostate cancer susceptibility in sub-saharan africa},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Variants in tubule epithelial regulatory elements mediate
most heritable differences in human kidney function. <em>NG</em>,
<em>56</em>(10), 2078–2092. (<a
href="https://doi.org/10.1038/s41588-024-01904-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Kidney failure, the decrease of kidney function below a threshold necessary to support life, is a major cause of morbidity and mortality. We performed a genome-wide association study (GWAS) of 406,504 individuals in the UK Biobank, identifying 430 loci affecting kidney function in middle-aged adults. To investigate the cell types affected by these loci, we integrated the GWAS with human kidney candidate cis-regulatory elements (cCREs) identified using single-cell assay for transposase-accessible chromatin sequencing (scATAC–seq). Overall, 56% of kidney function heritability localized to kidney tubule epithelial cCREs and an additional 7% to kidney podocyte cCREs. Thus, most heritable differences in adult kidney function are a result of altered gene expression in these two cell types. Using enhancer assays, allele-specific scATAC–seq and machine learning, we found that many kidney function variants alter tubule epithelial cCRE chromatin accessibility and function. Using CRISPRi, we determined which genes some of these cCREs regulate, implicating NDRG1, CCNB1 and STC1 in human kidney function. Integration of genome-wide association analysis of 406,504 individuals in the UK Biobank and scATAC–seq data reveals that variants in tubule epithelial regulatory elements mediate most heritable differences in human kidney function.},
  archive      = {J_NG},
  author       = {Loeb, Gabriel B. and Kathail, Pooja and Shuai, Richard W. and Chung, Ryan and Grona, Reinier J. and Peddada, Sailaja and Sevim, Volkan and Federman, Scot and Mader, Karl and Chu, Audrey Y. and Davitte, Jonathan and Du, Juan and Gupta, Alexander R. and Ye, Chun Jimmie and Shafer, Shawn and Przybyla, Laralynne and Rapiteanu, Radu and Ioannidis, Nilah M. and Reiter, Jeremy F.},
  doi          = {10.1038/s41588-024-01904-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2078-2092},
  shortjournal = {Nat. Genet.},
  title        = {Variants in tubule epithelial regulatory elements mediate most heritable differences in human kidney function},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Identifying genetic variants that influence the abundance of
cell states in single-cell data. <em>NG</em>, <em>56</em>(10),
2068–2077. (<a
href="https://doi.org/10.1038/s41588-024-01909-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Disease risk alleles influence the composition of cells present in the body, but modeling genetic effects on the cell states revealed by single-cell profiling is difficult because variant-associated states may reflect diverse combinations of the profiled cell features that are challenging to predefine. We introduce Genotype–Neighborhood Associations (GeNA), a statistical tool to identify cell-state abundance quantitative trait loci (csaQTLs) in high-dimensional single-cell datasets. Instead of testing associations to predefined cell states, GeNA flexibly identifies the cell states whose abundance is most associated with genetic variants. In a genome-wide survey of single-cell RNA sequencing peripheral blood profiling from 969 individuals, GeNA identifies five independent loci associated with shifts in the relative abundance of immune cell states. For example, rs3003-T (P = 1.96 × 10−11) associates with increased abundance of natural killer cells expressing tumor necrosis factor response programs. This csaQTL colocalizes with increased risk for psoriasis, an autoimmune disease that responds to anti-tumor necrosis factor treatments. Flexibly characterizing csaQTLs for granular cell states may help illuminate how genetic background alters cellular composition to confer disease risk. GeNA identifies cell-state abundance quantitative trait loci (csaQTLs) in single-cell RNA sequencing data. Applied to OneK1K, GeNA identifies natural killer cell and myeloid csaQTLs and implicates interferon-α-related cell states using a polygenic risk score for systemic lupus erythematosus.},
  archive      = {J_NG},
  author       = {Rumker, Laurie and Sakaue, Saori and Reshef, Yakir and Kang, Joyce B. and Yazar, Seyhan and Alquicira-Hernandez, Jose and Valencia, Cristian and Lagattuta, Kaitlyn A. and Mah-Som, Annelise and Nathan, Aparna and Powell, Joseph E. and Loh, Po-Ru and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-024-01909-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2068-2077},
  shortjournal = {Nat. Genet.},
  title        = {Identifying genetic variants that influence the abundance of cell states in single-cell data},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Statistically and functionally fine-mapped blood eQTLs and
pQTLs from 1,405 humans reveal distinct regulation patterns and disease
relevance. <em>NG</em>, <em>56</em>(10), 2054–2067. (<a
href="https://doi.org/10.1038/s41588-024-01896-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studying the genetic regulation of protein expression (through protein quantitative trait loci (pQTLs)) offers a deeper understanding of regulatory variants uncharacterized by mRNA expression regulation (expression QTLs (eQTLs)) studies. Here we report cis-eQTL and cis-pQTL statistical fine-mapping from 1,405 genotyped samples with blood mRNA and 2,932 plasma samples of protein expression, as part of the Japan COVID-19 Task Force (JCTF). Fine-mapped eQTLs (n = 3,464) were enriched for 932 variants validated with a massively parallel reporter assay. Fine-mapped pQTLs (n = 582) were enriched for missense variations on structured and extracellular domains, although the possibility of epitope-binding artifacts remains. Trans-eQTL and trans-pQTL analysis highlighted associations of class I HLA allele variation with KIR genes. We contrast the multi-tissue origin of plasma protein with blood mRNA, contributing to the limited colocalization level, distinct regulatory mechanisms and trait relevance of eQTLs and pQTLs. We report a negative correlation between ABO mRNA and protein expression because of linkage disequilibrium between distinct nearby eQTLs and pQTLs. Statistical fine-mapping of mRNA and protein quantitative trait loci in blood samples from the Japan COVID-19 Task Force sheds light on regulatory mechanisms and disease associations.},
  archive      = {J_NG},
  author       = {Wang, Qingbo S. and Hasegawa, Takanori and Namkoong, Ho and Saiki, Ryunosuke and Edahiro, Ryuya and Sonehara, Kyuto and Tanaka, Hiromu and Azekawa, Shuhei and Chubachi, Shotaro and Takahashi, Yugo and Sakaue, Saori and Namba, Shinichi and Yamamoto, Kenichi and Shiraishi, Yuichi and Chiba, Kenichi and Tanaka, Hiroko and Makishima, Hideki and Nannya, Yasuhito and Zhang, Zicong and Tsujikawa, Rika and Koike, Ryuji and Takano, Tomomi and Ishii, Makoto and Kimura, Akinori and Inoue, Fumitaka and Kanai, Takanori and Fukunaga, Koichi and Ogawa, Seishi and Imoto, Seiya and Miyano, Satoru and Okada, Yukinori},
  doi          = {10.1038/s41588-024-01896-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2054-2067},
  shortjournal = {Nat. Genet.},
  title        = {Statistically and functionally fine-mapped blood eQTLs and pQTLs from 1,405 humans reveal distinct regulation patterns and disease relevance},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Federated analysis of autosomal recessive coding variants in
29,745 developmental disorder patients from diverse populations.
<em>NG</em>, <em>56</em>(10), 2046–2053. (<a
href="https://doi.org/10.1038/s41588-024-01910-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autosomal recessive coding variants are well-known causes of rare disorders. We quantified the contribution of these variants to developmental disorders in a large, ancestrally diverse cohort comprising 29,745 trios, of whom 20.4% had genetically inferred non-European ancestries. The estimated fraction of patients attributable to exome-wide autosomal recessive coding variants ranged from ~2–19% across genetically inferred ancestry groups and was significantly correlated with average autozygosity. Established autosomal recessive developmental disorder-associated (ARDD) genes explained 84.0% of the total autosomal recessive coding burden, and 34.4% of the burden in these established genes was explained by variants not already reported as pathogenic in ClinVar. Statistical analyses identified two novel ARDD genes: KBTBD2 and ZDHHC16. This study expands our understanding of the genetic architecture of developmental disorders across diverse genetically inferred ancestry groups and suggests that improving strategies for interpreting missense variants in known ARDD genes may help diagnose more patients than discovering the remaining genes. Analysis of autosomal recessive coding variants in 29,745 trios from the DDD study and GeneDx provides insights into the genetic architecture of developmental disorders across ancestrally diverse populations.},
  archive      = {J_NG},
  author       = {Chundru, V. Kartik and Zhang, Zhancheng and Walter, Klaudia and Lindsay, Sarah J. and Danecek, Petr and Eberhardt, Ruth Y. and Gardner, Eugene J. and Malawsky, Daniel S. and Wigdor, Emilie M. and Torene, Rebecca and Retterer, Kyle and Wright, Caroline F. and Ólafsdóttir, Hildur and Guillen Sacoto, Maria J. and Ayaz, Akif and Akbeyaz, Ismail Hakki and Türkdoğan, Dilşad and Al Balushi, Aaisha Ibrahim and Bertoli-Avella, Aida and Bauer, Peter and Szenker-Ravi, Emmanuelle and Reversade, Bruno and McWalter, Kirsty and Sheridan, Eamonn and Firth, Helen V. and Hurles, Matthew E. and Samocha, Kaitlin E. and Ustach, Vincent D. and Martin, Hilary C.},
  doi          = {10.1038/s41588-024-01910-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2046-2053},
  shortjournal = {Nat. Genet.},
  title        = {Federated analysis of autosomal recessive coding variants in 29,745 developmental disorder patients from diverse populations},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Gene discovery and biological insights into anxiety
disorders from a large-scale multi-ancestry genome-wide association
study. <em>NG</em>, <em>56</em>(10), 2036–2045. (<a
href="https://doi.org/10.1038/s41588-024-01908-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We leveraged information from more than 1.2 million participants, including 97,383 cases, to investigate the genetics of anxiety disorders across five continental groups. Through ancestry-specific and cross-ancestry genome-wide association studies, we identified 51 anxiety-associated loci, 39 of which were novel. In addition, polygenic risk scores derived from individuals of European descent were associated with anxiety in African, admixed American and East Asian groups. The heritability of anxiety was enriched for genes expressed in the limbic system, cerebral cortex, cerebellum, metencephalon, entorhinal cortex and brain stem. Transcriptome-wide and proteome-wide analyses highlighted 115 genes associated with anxiety through brain-specific and cross-tissue regulation. Anxiety also showed global and local genetic correlations with depression, schizophrenia and bipolar disorder and widespread pleiotropy with several physical health domains. Overall, this study expands our knowledge regarding the genetic risk and pathogenesis of anxiety disorders, highlighting the importance of investigating diverse populations and integrating multi-omics information. A large-scale multi-ancestry genome-wide association study of European, African, admixed American, South Asian and East Asian ancestries provides insights into the pathogenesis of anxiety disorders.},
  archive      = {J_NG},
  author       = {Friligkou, Eleni and Løkhammer, Solveig and Cabrera-Mendoza, Brenda and Shen, Jie and He, Jun and Deiana, Giovanni and Zanoaga, Mihaela Diana and Asgel, Zeynep and Pilcher, Abigail and Di Lascio, Luciana and Makharashvili, Ana and Koller, Dora and Tylee, Daniel S. and Pathak, Gita A. and Polimanti, Renato},
  doi          = {10.1038/s41588-024-01908-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2036-2045},
  shortjournal = {Nat. Genet.},
  title        = {Gene discovery and biological insights into anxiety disorders from a large-scale multi-ancestry genome-wide association study},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Population-specific putative causal variants shape
quantitative traits. <em>NG</em>, <em>56</em>(10), 2027–2035. (<a
href="https://doi.org/10.1038/s41588-024-01913-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human genetic variants are associated with many traits through largely unknown mechanisms. Here, combining approximately 260,000 Japanese study participants, a Japanese-specific genotype reference panel and statistical fine-mapping, we identified 4,423 significant loci across 63 quantitative traits, among which 601 were new, and 9,406 putatively causal variants. New associations included Japanese-specific coding, splicing and noncoding variants, exemplified by a damaging missense variant rs730881101 in TNNT2 associated with lower heart function and increased risk for heart failure (P = 1.4 × 10−15 and odds ratio = 4.5, 95% confidence interval = 3.1–6.5). Putative causal noncoding variants were supported by state-of-art in silico functional assays and had comparable effect sizes to coding variants. A plausible example of new mechanisms of causal variants is an enrichment of causal variants in 3′ untranslated regions (UTRs), including the Japanese-specific rs13306436 in IL6 associated with pro-inflammatory traits and protection against tuberculosis. We experimentally showed that transcripts with rs13306436 are resistant to mRNA degradation by regnase-1, an RNA-binding protein. Our study provides a list of fine-mapped causal variants to be tested for functionality and underscores the importance of sequencing, genotyping and association efforts in diverse populations. Genome-wide association and fine-mapping analyses in approximately 260,000 Japanese individuals combined with a newly constructed Japanese-specific genotype reference panel identify hundreds of new loci and putative causal variants for 63 quantitative traits.},
  archive      = {J_NG},
  author       = {Koyama, Satoshi and Liu, Xiaoxi and Koike, Yoshinao and Hikino, Keiko and Koido, Masaru and Li, Wei and Akaki, Kotaro and Tomizuka, Kohei and Ito, Shuji and Otomo, Nao and Suetsugu, Hiroyuki and Yoshino, Soichiro and Akiyama, Masato and Saito, Kohei and Ishikawa, Yuki and Benner, Christian and Natarajan, Pradeep and Ellinor, Patrick T. and Mushiroda, Taisei and Horikoshi, Momoko and Ikeda, Masashi and Iwata, Nakao and Matsuda, Koichi and Niida, Shumpei and Ozaki, Kouichi and Momozawa, Yukihide and Ikegawa, Shiro and Takeuchi, Osamu and Ito, Kaoru and Terao, Chikashi},
  doi          = {10.1038/s41588-024-01913-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2027-2035},
  shortjournal = {Nat. Genet.},
  title        = {Population-specific putative causal variants shape quantitative traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Aneuploidy as a driver of human cancer. <em>NG</em>,
<em>56</em>(10), 2014–2026. (<a
href="https://doi.org/10.1038/s41588-024-01916-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aneuploidy, an abnormal chromosome composition, is a major contributor to cancer development and progression and an important determinant of cancer therapeutic responses and clinical outcomes. Despite being recognized as a hallmark of human cancer, the exact role of aneuploidy as a ‘driver’ of cancer is still largely unknown. Identifying the specific genetic elements that underlie the recurrence of common aneuploidies remains a major challenge of cancer genetics. In this Review, we discuss recurrent aneuploidies and their function as drivers of tumor development. We then delve into the context-dependent identification and functional characterization of the driver genes underlying driver aneuploidies and examine emerging strategies to uncover these driver genes using cancer genomics data and cancer models. Lastly, we explore opportunities for targeting driver aneuploidies in cancer by leveraging the functional consequences of these common genetic alterations. This Review discusses recurrent aneuploidies driving human cancer, methods to identify them and strategies to uncover underlying driver genes. It highlights genomic and experimental approaches to study and ultimately target driver aneuploidies.},
  archive      = {J_NG},
  author       = {Sdeor, Eran and Okada, Hajime and Saad, Ron and Ben-Yishay, Tal and Ben-David, Uri},
  doi          = {10.1038/s41588-024-01916-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2014-2026},
  shortjournal = {Nat. Genet.},
  title        = {Aneuploidy as a driver of human cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Chemical restriction of PU.1 genomic binding sites activates
alternate gene networks. <em>NG</em>, <em>56</em>(10), 2012–2013. (<a
href="https://doi.org/10.1038/s41588-024-01912-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Blockade of primary genomic binding sites with small molecules causes redistribution of the transcription factor PU.1 to alternative binding sites; its transcriptional activity at these sites activates secondary gene networks that drive myeloid cell differentiation.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01912-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2012-2013},
  shortjournal = {Nat. Genet.},
  title        = {Chemical restriction of PU.1 genomic binding sites activates alternate gene networks},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Long-term 3D primary epithelioid cultures reveal genes that
regulate esophageal cell fitness. <em>NG</em>, <em>56</em>(10),
2010–2011. (<a
href="https://doi.org/10.1038/s41588-024-01887-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Primary cell cultures need to be frequently passaged, which limits the study of long-term biological processes, such as how mutant clones colonize aging epithelia. Esophageal epithelioids self-maintain for months, recapitulating progenitor cell behavior in vivo. Epithelioid CRISPR–Cas9 screens reveal genes encoding molecules that control cell fitness.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01887-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2010-2011},
  shortjournal = {Nat. Genet.},
  title        = {Long-term 3D primary epithelioid cultures reveal genes that regulate esophageal cell fitness},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic and non-genetic HLA disruption is widespread in lung
and breast tumors. <em>NG</em>, <em>56</em>(10), 2008–2009. (<a
href="https://doi.org/10.1038/s41588-024-01886-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immune recognition of cancers can be inhibited if the molecules that present cancer cell-specific antigens are disrupted. We have developed a tool that can detect four different types of disruption. Overall, we find that both genetic and non-genetic disruption of these molecules is common in lung and breast tumors.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01886-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2008-2009},
  shortjournal = {Nat. Genet.},
  title        = {Genetic and non-genetic HLA disruption is widespread in lung and breast tumors},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pan-african analysis identifies genetic differences in
prostate cancer risk. <em>NG</em>, <em>56</em>(10), 2006–2007. (<a
href="https://doi.org/10.1038/s41588-024-01932-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand the genetic basis of disease, it is essential to study diverse populations. We conducted the largest study to date of African men to evaluate the evolutionary genetics and causes of prostate cancer. Our findings reveal novel genetic associations, including those that were not observed in studies of non-African populations.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01932-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2006-2007},
  shortjournal = {Nat. Genet.},
  title        = {Pan-african analysis identifies genetic differences in prostate cancer risk},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Structural remodeling of the inactive x chromosome during
early mouse development. <em>NG</em>, <em>56</em>(10), 2004–2005. (<a
href="https://doi.org/10.1038/s41588-024-01936-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mammalian inactive X chromosome shows unusual folding dominated by large-scale structures. A study finds a megadomain structure with a boundary at the Xist locus, preceding the well-known Dxz4-separated megadomains in somatic cells.},
  archive      = {J_NG},
  author       = {Schulz, Edda G. and Martitz, Alexandra},
  doi          = {10.1038/s41588-024-01936-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2004-2005},
  shortjournal = {Nat. Genet.},
  title        = {Structural remodeling of the inactive x chromosome during early mouse development},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Host physiology shapes the mutational landscape of normal
and carcinogenic tissue. <em>NG</em>, <em>56</em>(10), 2002–2003. (<a
href="https://doi.org/10.1038/s41588-024-01922-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations accrue with age as patches of mutant clones arise in otherwise histologically normal tissue. The clones’ persistence, expansion and roles in physiology and tumorigenesis are unclear. New work on the behavior of Pik3caH1047R mutant esophageal clones shows that host-dependent metabolic features underpin their expansion.},
  archive      = {J_NG},
  author       = {Nasreddin, Nadia and Sansom, Owen J.},
  doi          = {10.1038/s41588-024-01922-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2002-2003},
  shortjournal = {Nat. Genet.},
  title        = {Host physiology shapes the mutational landscape of normal and carcinogenic tissue},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Context-specific targeting of the androgen receptor in
prostate cancer. <em>NG</em>, <em>56</em>(10), 2000–2001. (<a
href="https://doi.org/10.1038/s41588-024-01935-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A co-factor for the androgen receptor, NSD2, provides insights into context-specific functions of the androgen receptor and is a new target for intervention.},
  archive      = {J_NG},
  author       = {Abate-Shen, Cory},
  doi          = {10.1038/s41588-024-01935-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2000-2001},
  shortjournal = {Nat. Genet.},
  title        = {Context-specific targeting of the androgen receptor in prostate cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Defining genome access of transcription factors.
<em>NG</em>, <em>56</em>(10), 1999. (<a
href="https://doi.org/10.1038/s41588-024-01953-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-024-01953-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1999},
  shortjournal = {Nat. Genet.},
  title        = {Defining genome access of transcription factors},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Personalizing pangenome graphs with k-mers. <em>NG</em>,
<em>56</em>(10), 1999. (<a
href="https://doi.org/10.1038/s41588-024-01954-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-01954-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1999},
  shortjournal = {Nat. Genet.},
  title        = {Personalizing pangenome graphs with k-mers},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Autoreactive t cells target neoself-antigens in lupus.
<em>NG</em>, <em>56</em>(10), 1999. (<a
href="https://doi.org/10.1038/s41588-024-01955-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-024-01955-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1999},
  shortjournal = {Nat. Genet.},
  title        = {Autoreactive t cells target neoself-antigens in lupus},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Reprogramming-based gene therapy promotes anti-tumor
immunity in vivo. <em>NG</em>, <em>56</em>(10), 1999. (<a
href="https://doi.org/10.1038/s41588-024-01956-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-024-01956-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1999},
  shortjournal = {Nat. Genet.},
  title        = {Reprogramming-based gene therapy promotes anti-tumor immunity in vivo},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A model for a dual function of n6-methyladenosine in r-loop
regulation. <em>NG</em>, <em>56</em>(10), 1995–1998. (<a
href="https://doi.org/10.1038/s41588-024-01905-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {R-loops contain DNA:RNA hybrids and an unpaired single-stranded DNA. N6-methyladenosine (m6A) has been reported to modulate R-loop levels, but with varying outcomes (R-loop resolution versus stabilization). We propose that in different contexts, m6A may either directly prevent R-loop accumulation or stabilize R-loops via the formation of RNA abasic sites.},
  archive      = {J_NG},
  author       = {Abakir, Abdulkadir and Ruzov, Alexey},
  doi          = {10.1038/s41588-024-01905-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1995-1998},
  shortjournal = {Nat. Genet.},
  title        = {A model for a dual function of n6-methyladenosine in R-loop regulation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Enhancing the polygenic score catalog with tools for score
calculation and ancestry normalization. <em>NG</em>, <em>56</em>(10),
1989–1994. (<a
href="https://doi.org/10.1038/s41588-024-01937-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polygenic scores (PGSs) have transformed human genetic research and have numerous potential clinical applications. Here we present a series of recent enhancements to the PGS Catalog and highlight the PGS Catalog Calculator, an open-source, scalable and portable pipeline for reproducibly calculating PGSs that democratizes equitable PGS applications.},
  archive      = {J_NG},
  author       = {Lambert, Samuel A. and Wingfield, Benjamin and Gibson, Joel T. and Gil, Laurent and Ramachandran, Santhi and Yvon, Florent and Saverimuttu, Shirin and Tinsley, Emily and Lewis, Elizabeth and Ritchie, Scott C. and Wu, Jingqin and Cánovas, Rodrigo and McMahon, Aoife and Harris, Laura W. and Parkinson, Helen and Inouye, Michael},
  doi          = {10.1038/s41588-024-01937-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1989-1994},
  shortjournal = {Nat. Genet.},
  title        = {Enhancing the polygenic score catalog with tools for score calculation and ancestry normalization},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Defining and pursuing diversity in human genetic studies.
<em>NG</em>, <em>56</em>(10), 1985–1988. (<a
href="https://doi.org/10.1038/s41588-024-01903-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Calls for more diverse data in genetics studies typically fall short of offering further guidance. Here we summarize a policy framework from the Global Alliance for Genomics and Health designed to fill this gap. The framework prompts researchers to consider both what types of diversity are needed and why, and how aims can be achieved through choices made throughout the data life cycle.},
  archive      = {J_NG},
  author       = {Raven-Adams, Maili C. and Hernandez-Boussard, Tina and Joly, Yann and Knoppers, Bartha Maria and Chandrasekharan, Subhashini and Thorogood, Adrian and Kumuthini, Judit and Ho, Calvin Wai Loon and Gonzlez, Ariana and Nelson, Sarah C. and Bombard, Yvonne and Thaldar, Donrich and Liu, Hanshi and Costa, Alessia and Muralidharan, Vijaytha and Henriques, Sasha and Nasir, Jamal and Lumaka, Aimé and Kaiser, Beatrice and Jamuar, Saumya Shekhar and Lewis, Anna C. F.},
  doi          = {10.1038/s41588-024-01903-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1985-1988},
  shortjournal = {Nat. Genet.},
  title        = {Defining and pursuing diversity in human genetic studies},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Chloroplast and whole-genome sequencing shed light on the
evolutionary history and phenotypic diversification of peanuts.
<em>NG</em>, <em>56</em>(9), 1975–1984. (<a
href="https://doi.org/10.1038/s41588-024-01876-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivated peanut (Arachis hypogaea L.) is a widely grown oilseed crop worldwide; however, the events leading to its origin and diversification are not fully understood. Here by combining chloroplast and whole-genome sequence data from a large germplasm collection, we show that the two subspecies of A. hypogaea (hypogaea and fastigiata) likely arose from distinct allopolyploidization and domestication events. Peanut genetic clusters were then differentiated in relation to dissemination routes and breeding efforts. A combination of linkage mapping and genome-wide association studies allowed us to characterize genes and genomic regions related to main peanut morpho-agronomic traits, namely flowering pattern, inner tegument color, growth habit, pod/seed weight and oil content. Together, our findings shed light on the evolutionary history and phenotypic diversification of peanuts and might be of broad interest to plant breeders. Chloroplast and whole-genome sequencing of a global germplasm panel of 355 Arachis accessions encompassing various species sheds light on the evolutionary history and phenotypic diversification of peanuts.},
  archive      = {J_NG},
  author       = {Zheng, Zheng and Sun, Ziqi and Qi, Feiyan and Fang, Yuanjin and Lin, Ke and Pavan, Stefano and Huang, Bingyan and Dong, Wenzhao and Du, Pei and Tian, Mengdi and Shi, Lei and Xu, Jing and Han, Suoyi and Liu, Hua and Qin, Li and Zhang, Zhongxin and Dai, Xiaodong and Miao, Lijuan and Zhao, Ruifang and Wang, Juan and Liao, Yanlin and Li, Alun and Ruan, Jue and Delvento, Chiara and Aiese Cigliano, Riccardo and Maliepaard, Chris and Bai, Yuling and Visser, Richard G. F. and Zhang, Xinyou},
  doi          = {10.1038/s41588-024-01876-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1975-1984},
  shortjournal = {Nat. Genet.},
  title        = {Chloroplast and whole-genome sequencing shed light on the evolutionary history and phenotypic diversification of peanuts},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Reference genome sequence and population genomic analysis of
peas provide insights into the genetic basis of mendelian and other
agronomic traits. <em>NG</em>, <em>56</em>(9), 1964–1974. (<a
href="https://doi.org/10.1038/s41588-024-01867-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peas are essential for human nutrition and played a crucial role in the discovery of Mendelian laws of inheritance. In this study, we assembled the genome of the elite vegetable pea cultivar ‘Zhewan No. 1’ at the chromosome level and analyzed resequencing data from 314 accessions, creating a comprehensive map of genetic variation in peas. We identified 235 candidate loci associated with 57 important agronomic traits through genome-wide association studies. Notably, we pinpointed the causal gene haplotypes responsible for four Mendelian traits: stem length (Le/le), flower color (A/a), cotyledon color (I/i) and seed shape (R/r). Additionally, we discovered the genes controlling pod form (Mendelian P/p) and hilum color. Our study also involved constructing a gene expression atlas across 22 tissues, highlighting key gene modules related to pod and seed development. These findings provide valuable pea genomic information and will facilitate the future genome-informed improvement of pea crops. Chromosome-scale genome assembly of a vegetable-type elite pea cultivar ZW1 and population analyses using resequencing data of 314 accessions provide insights into the genetic basis of Mendelian and other agronomic traits.},
  archive      = {J_NG},
  author       = {Liu, Na and Lyu, Xiaolong and Zhang, Xueying and Zhang, Guwen and Zhang, Ziqian and Guan, Xueying and Chen, Xiaoyang and Yang, Xiaoming and Feng, Zhijuan and Gao, Qiang and Shi, Wanghong and Deng, Yayuan and Sheng, Kuang and Ou, Jinwen and Zhu, Yumeng and Wang, Bin and Bu, Yuanpeng and Zhang, Mingfang and Zhang, Liangsheng and Zhao, Ting and Gong, Yaming},
  doi          = {10.1038/s41588-024-01867-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1964-1974},
  shortjournal = {Nat. Genet.},
  title        = {Reference genome sequence and population genomic analysis of peas provide insights into the genetic basis of mendelian and other agronomic traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A telomere-to-telomere cotton genome assembly reveals
centromere evolution and a mutator transposon-linked module regulating
embryo development. <em>NG</em>, <em>56</em>(9), 1953–1963. (<a
href="https://doi.org/10.1038/s41588-024-01877-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assembly of complete genomes can reveal functional genetic elements missing from draft sequences. Here we present the near-complete telomere-to-telomere and contiguous genome of the cotton species Gossypium raimondii. Our assembly identified gaps and misoriented or misassembled regions in previous assemblies and produced 13 centromeres, with 25 chromosomal ends having telomeres. In contrast to satellite-rich Arabidopsis and rice centromeres, cotton centromeres lack phased CENH3 nucleosome positioning patterns and probably evolved by invasion from long terminal repeat retrotransposons. In-depth expression profiling of transposable elements revealed a previously unannotated DNA transposon (MuTC01) that interacts with miR2947 to produce trans-acting small interfering RNAs (siRNAs), one of which targets the newly evolved LEC2 (LEC2b) to produce phased siRNAs. Systematic genome editing experiments revealed that this tripartite module, miR2947–MuTC01–LEC2b, controls the morphogenesis of complex folded embryos characteristic of Gossypium and its close relatives in the cotton tribe. Our study reveals a trans-acting siRNA-based tripartite regulatory pathway for embryo development in higher plants. A telomere-to-telomere genome assembly of the diploid cotton species Gossypium raimondii reveals centromere evolution and a Mutator transposon-linked tripartite module, miR2947–MuTC01–LEC2b that regulates cotton embryo development.},
  archive      = {J_NG},
  author       = {Huang, Gai and Bao, Zhigui and Feng, Li and Zhai, Jixian and Wendel, Jonathan F. and Cao, Xiaofeng and Zhu, Yuxian},
  doi          = {10.1038/s41588-024-01877-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1953-1963},
  shortjournal = {Nat. Genet.},
  title        = {A telomere-to-telomere cotton genome assembly reveals centromere evolution and a mutator transposon-linked module regulating embryo development},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). YY1-controlled regulatory connectivity and transcription are
influenced by the cell cycle. <em>NG</em>, <em>56</em>(9), 1938–1952.
(<a href="https://doi.org/10.1038/s41588-024-01871-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Few transcription factors have been examined for their direct roles in physically connecting enhancers and promoters. Here acute degradation of Yin Yang 1 (YY1) in erythroid cells revealed its requirement for the maintenance of numerous enhancer–promoter loops, but not compartments or domains. Despite its reported ability to interact with cohesin, the formation of YY1-dependent enhancer–promoter loops does not involve stalling of cohesin-mediated loop extrusion. Integrating mitosis-to-G1-phase dynamics, we observed partial retention of YY1 on mitotic chromatin, predominantly at gene promoters, followed by rapid rebinding during mitotic exit, coinciding with enhancer–promoter loop establishment. YY1 degradation during the mitosis-to-G1-phase interval revealed a set of enhancer–promoter loops that require YY1 for establishment during G1-phase entry but not for maintenance in interphase, suggesting that cell cycle stage influences YY1’s architectural function. Thus, as revealed here for YY1, chromatin architectural functions of transcription factors can vary in their interplay with CTCF and cohesin as well as by cell cycle stage. Yin Yang 1 (YY1) aids in the formation of enhancer–promoter (E–P) loops independently of cohesin. YY1 maintains a subset of E–P interactions in interphase and establishes an overlapping yet distinct set after mitotic exit.},
  archive      = {J_NG},
  author       = {Lam, Jessica C. and Aboreden, Nicholas G. and Midla, Susannah C. and Wang, Siqing and Huang, Anran and Keller, Cheryl A. and Giardine, Belinda and Henderson, Kate A. and Hardison, Ross C. and Zhang, Haoyue and Blobel, Gerd A.},
  doi          = {10.1038/s41588-024-01871-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1938-1952},
  shortjournal = {Nat. Genet.},
  title        = {YY1-controlled regulatory connectivity and transcription are influenced by the cell cycle},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An integrated single-cell reference atlas of the human
endometrium. <em>NG</em>, <em>56</em>(9), 1925–1937. (<a
href="https://doi.org/10.1038/s41588-024-01873-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The complex and dynamic cellular composition of the human endometrium remains poorly understood. Previous endometrial single-cell atlases profiled few donors and lacked consensus in defining cell types. We introduce the Human Endometrial Cell Atlas (HECA), a high-resolution single-cell reference atlas (313,527 cells) combining published and new endometrial single-cell transcriptomics datasets of 63 women with and without endometriosis. HECA assigns consensus and identifies previously unreported cell types, mapped in situ using spatial transcriptomics and validated using a new independent single-nuclei dataset (312,246 nuclei, 63 donors). In the functionalis, we identify intricate stromal–epithelial cell coordination via transforming growth factor beta (TGFβ) signaling. In the basalis, we define signaling between fibroblasts and an epithelial population expressing progenitor markers. Integration of HECA with large-scale endometriosis genome-wide association study data pinpoints decidualized stromal cells and macrophages as most likely dysregulated in endometriosis. The HECA is a valuable resource for studying endometrial physiology and disorders, and for guiding microphysiological in vitro systems development. The Human Endometrial Cell Atlas integrates single-cell transcriptomic datasets from women with and without endometriosis. Novel and known cell types are registered using spatial transcriptomics to provide a comprehensive map of the human endometrium in controls and endometriosis cases.},
  archive      = {J_NG},
  author       = {Marečková, Magda and Garcia-Alonso, Luz and Moullet, Marie and Lorenzi, Valentina and Petryszak, Robert and Sancho-Serra, Carmen and Oszlanczi, Agnes and Icoresi Mazzeo, Cecilia and Wong, Frederick C. K. and Kelava, Iva and Hoffman, Sophie and Krassowski, Michał and Garbutt, Kurtis and Gaitskell, Kezia and Yancheva, Slaveya and Woon, Ee Von and Male, Victoria and Granne, Ingrid and Hellner, Karin and Mahbubani, Krishnaa T. and Saeb-Parsy, Kourosh and Lotfollahi, Mohammad and Prigmore, Elena and Southcombe, Jennifer and Dragovic, Rebecca A. and Becker, Christian M. and Zondervan, Krina T. and Vento-Tormo, Roser},
  doi          = {10.1038/s41588-024-01873-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1925-1937},
  shortjournal = {Nat. Genet.},
  title        = {An integrated single-cell reference atlas of the human endometrium},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-scale quantification and prediction of pathogenic
stop codon readthrough by small molecules. <em>NG</em>, <em>56</em>(9),
1914–1924. (<a
href="https://doi.org/10.1038/s41588-024-01878-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Premature termination codons (PTCs) cause ~10–20% of inherited diseases and are a major mechanism of tumor suppressor gene inactivation in cancer. A general strategy to alleviate the effects of PTCs would be to promote translational readthrough. Nonsense suppression by small molecules has proven effective in diverse disease models, but translation into the clinic is hampered by ineffective readthrough of many PTCs. Here we directly tackle the challenge of defining drug efficacy by quantifying the readthrough of ~5,800 human pathogenic stop codons by eight drugs. We find that different drugs promote the readthrough of complementary subsets of PTCs defined by local sequence context. This allows us to build interpretable models that accurately predict drug-induced readthrough genome-wide, and we validate these models by quantifying endogenous stop codon readthrough. Accurate readthrough quantification and prediction will empower clinical trial design and the development of personalized nonsense suppression therapies. Small molecules can promote translational readthrough of premature termination codons, reducing their pathological effect. This study quantifies the readthrough of ~5,800 human pathogenic stop codons by eight drugs and builds models to predict drug-induced readthrough genome-wide.},
  archive      = {J_NG},
  author       = {Toledano, Ignasi and Supek, Fran and Lehner, Ben},
  doi          = {10.1038/s41588-024-01878-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1914-1924},
  shortjournal = {Nat. Genet.},
  title        = {Genome-scale quantification and prediction of pathogenic stop codon readthrough by small molecules},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Epigenetic therapy potentiates transposable element
transcription to create tumor-enriched antigens in glioblastoma cells.
<em>NG</em>, <em>56</em>(9), 1903–1913. (<a
href="https://doi.org/10.1038/s41588-024-01880-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inhibiting epigenetic modulators can transcriptionally reactivate transposable elements (TEs). These TE transcripts often generate unique peptides that can serve as immunogenic antigens for immunotherapy. Here, we ask whether TEs activated by epigenetic therapy could appreciably increase the antigen repertoire in glioblastoma, an aggressive brain cancer with low mutation and neoantigen burden. We treated patient-derived primary glioblastoma stem cell lines, an astrocyte cell line and primary fibroblast cell lines with epigenetic drugs, and identified treatment-induced, TE-derived transcripts that are preferentially expressed in cancer cells. We verified that these transcripts could produce human leukocyte antigen class I-presented antigens using liquid chromatography with tandem mass spectrometry pulldown experiments. Importantly, many TEs were also transcribed, even in proliferating nontumor cell lines, after epigenetic therapy, which suggests that targeted strategies like CRISPR-mediated activation could minimize potential side effects of activating unwanted genomic regions. The results highlight both the need for caution and the promise of future translational efforts in harnessing treatment-induced TE-derived antigens for targeted immunotherapy. Treatment of primary glioblastoma cell lines with epigenetic therapy reactivates transposable elements (TEs). TE-derived transcripts can produce human leukocyte antigen class I-presented antigens, which could potentially be therapeutically targeted.},
  archive      = {J_NG},
  author       = {Jang, H. Josh and Shah, Nakul M. and Maeng, Ju Heon and Liang, Yonghao and Basri, Noah L. and Ge, Jiaxin and Qu, Xuan and Mahlokozera, Tatenda and Tzeng, Shin-Cheng and Williams, Russell B. and Moore, Michael J. and Annamalai, Devi and Chen, Justin Y. and Lee, Hyung Joo and DeSouza, Patrick A. and Li, Daofeng and Xing, Xiaoyun and Kim, Albert H. and Wang, Ting},
  doi          = {10.1038/s41588-024-01880-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1903-1913},
  shortjournal = {Nat. Genet.},
  title        = {Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). In vivo CRISPR screens identify a dual function of MEN1 in
regulating tumor–microenvironment interactions. <em>NG</em>,
<em>56</em>(9), 1890–1902. (<a
href="https://doi.org/10.1038/s41588-024-01874-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Functional genomic screens in two-dimensional cell culture models are limited in identifying therapeutic targets that influence the tumor microenvironment. By comparing targeted CRISPR–Cas9 screens in a two-dimensional culture with xenografts derived from the same cell line, we identified MEN1 as the top hit that confers differential dropout effects in vitro and in vivo. MEN1 knockout in multiple solid cancer types does not impact cell proliferation in vitro but significantly promotes or inhibits tumor growth in immunodeficient or immunocompetent mice, respectively. Mechanistically, MEN1 knockout redistributes MLL1 chromatin occupancy, increasing H3K4me3 at repetitive genomic regions, activating double-stranded RNA expression and increasing neutrophil and CD8+ T cell infiltration in immunodeficient and immunocompetent mice, respectively. Pharmacological inhibition of the menin–MLL interaction reduces tumor growth in a CD8+ T cell-dependent manner. These findings reveal tumor microenvironment-dependent oncogenic and tumor-suppressive functions of MEN1 and provide a rationale for targeting MEN1 in solid cancers. Loss of MEN1 affects tumor growth, varing with the components of the tumor microenvironment. These tumors show redistribution of MLL1 on chromatin and the activation of a viral mimicry response.},
  archive      = {J_NG},
  author       = {Su, Peiran and Liu, Yin and Chen, Tianyi and Xue, Yibo and Zeng, Yong and Zhu, Guanghui and Chen, Sujun and Teng, Mona and Ci, Xinpei and Guo, Mengdi and He, Michael Y. and Hao, Jun and Chu, Vivian and Xu, Wenxi and Wang, Shiyan and Mehdipour, Parinaz and Xu, Xin and Marhon, Sajid A. and Soares, Fraser and Pham, Nhu-An and Wu, Bell Xi and Her, Peter Hyunwuk and Feng, Shengrui and Alshamlan, Najd and Khalil, Maryam and Krishnan, Rehna and Yu, Fangyou and Chen, Chang and Burrows, Francis and Hakem, Razqallah and Lupien, Mathieu and Harding, Shane and Lok, Benjamin H. and O’Brien, Catherine and Berlin, Alejandro and De Carvalho, Daniel D. and Brooks, David G. and Schramek, Daniel and Tsao, Ming-Sound and He, Housheng Hansen},
  doi          = {10.1038/s41588-024-01874-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1890-1902},
  shortjournal = {Nat. Genet.},
  title        = {In vivo CRISPR screens identify a dual function of MEN1 in regulating tumor–microenvironment interactions},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Comprehensive molecular profiling of multiple myeloma
identifies refined copy number and expression subtypes. <em>NG</em>,
<em>56</em>(9), 1878–1889. (<a
href="https://doi.org/10.1038/s41588-024-01853-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple myeloma is a treatable, but currently incurable, hematological malignancy of plasma cells characterized by diverse and complex tumor genetics for which precision medicine approaches to treatment are lacking. The Multiple Myeloma Research Foundation’s Relating Clinical Outcomes in Multiple Myeloma to Personal Assessment of Genetic Profile study ( NCT01454297 ) is a longitudinal, observational clinical study of newly diagnosed patients with multiple myeloma (n = 1,143) where tumor samples are characterized using whole-genome sequencing, whole-exome sequencing and RNA sequencing at diagnosis and progression, and clinical data are collected every 3 months. Analyses of the baseline cohort identified genes that are the target of recurrent gain-of-function and loss-of-function events. Consensus clustering identified 8 and 12 unique copy number and expression subtypes of myeloma, respectively, identifying high-risk genetic subtypes and elucidating many of the molecular underpinnings of these unique biological groups. Analysis of serial samples showed that 25.5% of patients transition to a high-risk expression subtype at progression. We observed robust expression of immunotherapy targets in this subtype, suggesting a potential therapeutic option. Longitudinal genomic and transcriptomic profiling of 1,143 patients with multiple myeloma by the Relating Clinical Outcomes in Multiple Myeloma to Personal Assessment of Genetic Profile study yields an improved copy number and gene expression subtype scheme, most notably a high-risk proliferative subtype associated with complete loss of RB1 or MAX.},
  archive      = {J_NG},
  author       = {Skerget, Sheri and Penaherrera, Daniel and Chari, Ajai and Jagannath, Sundar and Siegel, David S. and Vij, Ravi and Orloff, Gregory and Jakubowiak, Andrzej and Niesvizky, Ruben and Liles, Darla and Berdeja, Jesus and Levy, Moshe and Wolf, Jeffrey and Usmani, Saad Z. and Christofferson, Austin W. and Nasser, Sara and Aldrich, Jessica L. and Legendre, Christophe and Benard, Brooks and Miller, Chase and Turner, Bryce and Kurdoglu, Ahmet and Washington, Megan and Yellapantula, Venkata and Adkins, Jonathan R. and Cuyugan, Lori and Boateng, Martin and Helland, Adrienne and Kyman, Shari and McDonald, Jackie and Reiman, Rebecca and Stephenson, Kristi and Tassone, Erica and Blanski, Alex and Livermore, Brianne and Kirchhoff, Meghan and Rohrer, Daniel C. and D’Agostino, Mattia and Gamella, Manuela and Collison, Kimberly and Stumph, Jennifer and Kidd, Pam and Donnelly, Andrea and Zaugg, Barbara and Toone, Maureen and McBride, Kyle and DeRome, Mary and Rogers, Jennifer and Craig, David and Liang, Winnie S. and Gutierrez, Norma C. and Jewell, Scott D. and Carpten, John and Anderson, Kenneth C. and Cho, Hearn Jay and Auclair, Daniel and Lonial, Sagar and Keats, Jonathan J.},
  doi          = {10.1038/s41588-024-01853-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1878-1889},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive molecular profiling of multiple myeloma identifies refined copy number and expression subtypes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Analysis of 10,478 cancer genomes identifies candidate
driver genes and opportunities for precision oncology. <em>NG</em>,
<em>56</em>(9), 1868–1877. (<a
href="https://doi.org/10.1038/s41588-024-01785-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor genomic profiling is increasingly seen as a prerequisite to guide the treatment of patients with cancer. To explore the value of whole-genome sequencing (WGS) in broadening the scope of cancers potentially amenable to a precision therapy, we analysed whole-genome sequencing data on 10,478 patients spanning 35 cancer types recruited to the UK 100,000 Genomes Project. We identified 330 candidate driver genes, including 74 that are new to any cancer. We estimate that approximately 55% of patients studied harbor at least one clinically relevant mutation, predicting either sensitivity or resistance to certain treatments or clinical trial eligibility. By performing computational chemogenomic analysis of cancer mutations we identify additional targets for compounds that represent attractive candidates for future clinical trials. This study represents one of the most comprehensive efforts thus far to identify cancer driver genes in the real world setting and assess their impact on informing precision oncology. Analysis of whole-genome sequencing data from over 10,000 tumor samples spanning 35 cancer types identifies putative driver genes and highlights new therapeutic opportunities.},
  archive      = {J_NG},
  author       = {Kinnersley, Ben and Sud, Amit and Everall, Andrew and Cornish, Alex J. and Chubb, Daniel and Culliford, Richard and Gruber, Andreas J. and Lärkeryd, Adrian and Mitsopoulos, Costas and Wedge, David and Houlston, Richard},
  doi          = {10.1038/s41588-024-01785-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1868-1877},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of 10,478 cancer genomes identifies candidate driver genes and opportunities for precision oncology},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic factors associated with reasons for clinical trial
stoppage. <em>NG</em>, <em>56</em>(9), 1862–1867. (<a
href="https://doi.org/10.1038/s41588-024-01854-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many drug discovery projects are started but few progress fully through clinical trials to approval. Previous work has shown that human genetics support for the therapeutic hypothesis increases the chance of trial progression. Here, we applied natural language processing to classify the free-text reasons for 28,561 clinical trials that stopped before their endpoints were met. We then evaluated these classes in light of the underlying evidence for the therapeutic hypothesis and target properties. We found that trials are more likely to stop because of a lack of efficacy in the absence of strong genetic evidence from human populations or genetically modified animal models. Furthermore, certain trials are more likely to stop for safety reasons if the drug target gene is highly constrained in human populations and if the gene is broadly expressed across tissues. These results support the growing use of human genetics to evaluate targets for drug discovery programs. Natural language processing of stopped clinical trials finds associations with the strength and extent of genetic evidence, suggesting that incorporating such evidence during target selection could help avoid negative outcomes.},
  archive      = {J_NG},
  author       = {Razuvayevskaya, Olesya and Lopez, Irene and Dunham, Ian and Ochoa, David},
  doi          = {10.1038/s41588-024-01854-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1862-1867},
  shortjournal = {Nat. Genet.},
  title        = {Genetic factors associated with reasons for clinical trial stoppage},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Global impact of unproductive splicing on human gene
expression. <em>NG</em>, <em>56</em>(9), 1851–1861. (<a
href="https://doi.org/10.1038/s41588-024-01872-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alternative splicing (AS) in human genes is widely viewed as a mechanism for enhancing proteomic diversity. AS can also impact gene expression levels without increasing protein diversity by producing ‘unproductive’ transcripts that are targeted for rapid degradation by nonsense-mediated decay (NMD). However, the relative importance of this regulatory mechanism remains underexplored. To better understand the impact of AS–NMD relative to other regulatory mechanisms, we analyzed population-scale genomic data across eight molecular assays, covering various stages from transcription to cytoplasmic decay. We report threefold more unproductive splicing compared with prior estimates using steady-state RNA. This unproductive splicing compounds across multi-intronic genes, resulting in 15% of transcript molecules from protein-coding genes being unproductive. Leveraging genetic variation across cell lines, we find that GWAS trait-associated loci explained by AS are as often associated with NMD-induced expression level differences as with differences in protein isoform usage. Our findings suggest that much of the impact of AS is mediated by NMD-induced changes in gene expression rather than diversification of the proteome. Genomic analyses suggest that ~15% of transcript molecules are spliced into unproductive transcripts targeted by nonsense-mediated decay, which have a larger effect on gene expression than previously thought.},
  archive      = {J_NG},
  author       = {Fair, Benjamin and Buen Abad Najar, Carlos F. and Zhao, Junxing and Lozano, Stephanie and Reilly, Austin and Mossian, Gabriela and Staley, Jonathan P. and Wang, Jingxin and Li, Yang I.},
  doi          = {10.1038/s41588-024-01872-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1851-1861},
  shortjournal = {Nat. Genet.},
  title        = {Global impact of unproductive splicing on human gene expression},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Fine-mapping across diverse ancestries drives the discovery
of putative causal variants underlying human complex traits and
diseases. <em>NG</em>, <em>56</em>(9), 1841–1850. (<a
href="https://doi.org/10.1038/s41588-024-01870-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS) of human complex traits or diseases often implicate genetic loci that span hundreds or thousands of genetic variants, many of which have similar statistical significance. While statistical fine-mapping in individuals of European ancestry has made important discoveries, cross-population fine-mapping has the potential to improve power and resolution by capitalizing on the genomic diversity across ancestries. Here we present SuSiEx, an accurate and computationally efficient method for cross-population fine-mapping. SuSiEx integrates data from an arbitrary number of ancestries, explicitly models population-specific allele frequencies and linkage disequilibrium patterns, accounts for multiple causal variants in a genomic region and can be applied to GWAS summary statistics. We comprehensively assessed the performance of SuSiEx using simulations. We further showed that SuSiEx improves the fine-mapping of a range of quantitative traits available in both the UK Biobank and Taiwan Biobank, and improves the fine-mapping of schizophrenia-associated loci by integrating GWAS across East Asian and European ancestries. The cross-population Sum of Single Effects (SuSiEx) model is a robust and computationally efficient method for conducting multi-ancestry fine-mapping of genome-wide association signals, producing smaller credible sets and capturing population-specific causal variants.},
  archive      = {J_NG},
  author       = {Yuan, Kai and Longchamps, Ryan J. and Pardiñas, Antonio F. and Yu, Mingrui and Chen, Tzu-Ting and Lin, Shu-Chin and Chen, Yu and Lam, Max and Liu, Ruize and Xia, Yan and Guo, Zhenglin and Shi, Wenzhao and Shen, Chengguo and Daly, Mark J. and Neale, Benjamin M. and Feng, Yen-Chen A. and Lin, Yen-Feng and Chen, Chia-Yen and O’Donovan, Michael C. and Ge, Tian and Huang, Hailiang},
  doi          = {10.1038/s41588-024-01870-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1841-1850},
  shortjournal = {Nat. Genet.},
  title        = {Fine-mapping across diverse ancestries drives the discovery of putative causal variants underlying human complex traits and diseases},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic architecture of telomere length in 462,666 UK
biobank whole-genome sequences. <em>NG</em>, <em>56</em>(9), 1832–1840.
(<a href="https://doi.org/10.1038/s41588-024-01884-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Telomeres protect chromosome ends from damage and their length is linked with human disease and aging. We developed a joint telomere length metric, combining quantitative PCR and whole-genome sequencing measurements from 462,666 UK Biobank participants. This metric increased SNP heritability, suggesting that it better captures genetic regulation of telomere length. Exome-wide rare-variant and gene-level collapsing association studies identified 64 variants and 30 genes significantly associated with telomere length, including allelic series in ACD and RTEL1. Notably, 16% of these genes are known drivers of clonal hematopoiesis—an age-related somatic mosaicism associated with myeloid cancers and several nonmalignant diseases. Somatic variant analyses revealed gene-specific associations with telomere length, including lengthened telomeres in individuals with large SRSF2-mutant clones, compared with shortened telomeres in individuals with clonal expansions driven by other genes. Collectively, our findings demonstrate the impact of rare variants on telomere length, with larger effects observed among genes also associated with clonal hematopoiesis. Genome-wide association analysis of an improved telomere length score, calculated from quantitative PCR and whole-genome sequencing measurements in 462,666 individuals in the UK Biobank, identifies novel genes and variants underlying this trait.},
  archive      = {J_NG},
  author       = {Burren, Oliver S. and Dhindsa, Ryan S. and Deevi, Sri V. V. and Wen, Sean and Nag, Abhishek and Mitchell, Jonathan and Hu, Fengyuan and Loesch, Douglas P. and Smith, Katherine R. and Razdan, Neetu and Olsson, Henric and Platt, Adam and Vitsios, Dimitrios and Wu, Qiang and Codd, Veryan and Nelson, Christopher P. and Samani, Nilesh J. and March, Ruth E. and Wasilewski, Sebastian and Carss, Keren and Fabre, Margarete and Wang, Quanli and Pangalos, Menelas N. and Petrovski, Slavé},
  doi          = {10.1038/s41588-024-01884-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1832-1840},
  shortjournal = {Nat. Genet.},
  title        = {Genetic architecture of telomere length in 462,666 UK biobank whole-genome sequences},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Disease prediction with multi-omics and biomarkers empowers
case–control genetic discoveries in the UK biobank. <em>NG</em>,
<em>56</em>(9), 1821–1831. (<a
href="https://doi.org/10.1038/s41588-024-01898-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of biobank-level datasets offers new opportunities to discover novel biomarkers and develop predictive algorithms for human disease. Here, we present an ensemble machine-learning framework (machine learning with phenotype associations, MILTON) utilizing a range of biomarkers to predict 3,213 diseases in the UK Biobank. Leveraging the UK Biobank’s longitudinal health record data, MILTON predicts incident disease cases undiagnosed at time of recruitment, largely outperforming available polygenic risk scores. We further demonstrate the utility of MILTON in augmenting genetic association analyses in a phenome-wide association study of 484,230 genome-sequenced samples, along with 46,327 samples with matched plasma proteomics data. This resulted in improved signals for 88 known (P &amp;lt; 1 × 10−8) gene–disease relationships alongside 182 gene–disease relationships that did not achieve genome-wide significance in the nonaugmented baseline cohorts. We validated these discoveries in the FinnGen biobank alongside two orthogonal machine-learning methods built for gene–disease prioritization. All extracted gene–disease associations and incident disease predictive biomarkers are publicly available ( http://milton.public.cgr.astrazeneca.com ). MILTON uses phenotype information in the UK Biobank to identify clinical biomarkers and other quantitative traits that characterize diseases. It then constructs augmented cohorts by predicting undiagnosed individuals, improving power to discover gene–disease relationships.},
  archive      = {J_NG},
  author       = {Garg, Manik and Karpinski, Marcin and Matelska, Dorota and Middleton, Lawrence and Burren, Oliver S. and Hu, Fengyuan and Wheeler, Eleanor and Smith, Katherine R. and Fabre, Margarete A. and Mitchell, Jonathan and O’Neill, Amanda and Ashley, Euan A. and Harper, Andrew R. and Wang, Quanli and Dhindsa, Ryan S. and Petrovski, Slavé and Vitsios, Dimitrios},
  doi          = {10.1038/s41588-024-01898-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1821-1831},
  shortjournal = {Nat. Genet.},
  title        = {Disease prediction with multi-omics and biomarkers empowers case–control genetic discoveries in the UK biobank},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Rare coding variant analysis for human diseases across
biobanks and ancestries. <em>NG</em>, <em>56</em>(9), 1811–1820. (<a
href="https://doi.org/10.1038/s41588-024-01894-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale sequencing has enabled unparalleled opportunities to investigate the role of rare coding variation in human phenotypic variability. Here, we present a pan-ancestry analysis of sequencing data from three large biobanks, including the All of Us research program. Using mixed-effects models, we performed gene-based rare variant testing for 601 diseases across 748,879 individuals, including 155,236 with ancestry dissimilar to European. We identified 363 significant associations, which highlighted core genes for the human disease phenome and identified potential novel associations, including UBR3 for cardiometabolic disease and YLPM1 for psychiatric disease. Pan-ancestry burden testing represented an inclusive and useful approach for discovery in diverse datasets, although we also highlight the importance of ancestry-specific sensitivity analyses in this setting. Finally, we found that effect sizes for rare protein-disrupting variants were concordant between samples similar to European ancestry and other genetic ancestries (βDeming = 0.7–1.0). Our results have implications for multi-ancestry and cross-biobank approaches in sequencing association studies for human disease. Gene-based rare variant analyses for 601 diseases across 748,879 individuals from three biobanks identify 363 significant associations and highlight important considerations for multi-ancestry and cross-biobank sequencing studies.},
  archive      = {J_NG},
  author       = {Jurgens, Sean J. and Wang, Xin and Choi, Seung Hoan and Weng, Lu-Chen and Koyama, Satoshi and Pirruccello, James P. and Nguyen, Trang and Smadbeck, Patrick and Jang, Dongkeun and Chaffin, Mark and Walsh, Roddy and Roselli, Carolina and Elliott, Amanda L. and Wijdeveld, Leonoor F. J. M. and Biddinger, Kiran J. and Kany, Shinwan and Rämö, Joel T. and Natarajan, Pradeep and Aragam, Krishna G. and Flannick, Jason and Burtt, Noël P. and Bezzina, Connie R. and Lubitz, Steven A. and Lunetta, Kathryn L. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-024-01894-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1811-1820},
  shortjournal = {Nat. Genet.},
  title        = {Rare coding variant analysis for human diseases across biobanks and ancestries},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Homozygosity for a stop-gain variant in CCDC201 causes
primary ovarian insufficiency. <em>NG</em>, <em>56</em>(9), 1804–1810.
(<a href="https://doi.org/10.1038/s41588-024-01885-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Age at menopause (AOM) has a substantial impact on fertility and disease risk. While many loci with variants that associate with AOM have been identified through genome-wide association studies (GWAS) under an additive model, other genetic models are rarely considered1. Here through GWAS meta-analysis under the recessive model of 174,329 postmenopausal women from Iceland, Denmark, the United Kingdom (UK; UK Biobank) and Norway, we study low-frequency variants with a large effect on AOM. We discovered that women homozygous for the stop-gain variant rs117316434 (A) in CCDC201 (p.(Arg162Ter), minor allele frequency ~1%) reached menopause 9 years earlier than other women (P = 1.3 × 10−15). The genotype is present in one in 10,000 northern European women and leads to primary ovarian insufficiency in close to half of them. Consequently, homozygotes have fewer children, and the age at last childbirth is 5 years earlier (P = 3.8 × 10−5). The CCDC201 gene was only found in humans in 2022 and is highly expressed in oocytes. Homozygosity for CCDC201 loss-of-function has a substantial impact on female reproductive health, and homozygotes would benefit from reproductive counseling and treatment for symptoms of early menopause. Genome-wide analysis of age at menopause under a recessive model identifies a stop-gain variant in CCDC201 associated with primary ovarian insufficiency. This homozygous genotype is present in 1 in 10,000 women of northern European ancestry.},
  archive      = {J_NG},
  author       = {Oddsson, Asmundur and Steinthorsdottir, Valgerdur and Oskarsson, Gudjon R. and Styrkarsdottir, Unnur and Moore, Kristjan H. S. and Isberg, Salvor and Halldorsson, Gisli H. and Sveinbjornsson, Gardar and Westergaard, David and Nielsen, Henriette Svarre and Fridriksdottir, Run and Jensson, Brynjar O. and Arnadottir, Gudny A. and Jonsson, Hakon and Sturluson, Arni and Snaebjarnarson, Audunn S. and Andreassen, Ole A. and Walters, G. Bragi and Nyegaard, Mette and Erikstrup, Christian and Steingrimsdottir, Thora and Lie, Rolv T. and Melsted, Pall and Jonsdottir, Ingileif and Halldorsson, Bjarni V. and Thorleifsson, Gudmar and Saemundsdottir, Jona and Magnusson, Olafur Th. and Banasik, Karina and Sorensen, Erik and Masson, Gisli and Pedersen, Ole Birger and Tryggvadottir, Laufey and Haavik, Jan and Ostrowski, Sisse Rye and Stefansson, Hreinn and Holm, Hilma and Rafnar, Thorunn and Gudbjartsson, Daniel F. and Sulem, Patrick and Stefansson, Kari},
  doi          = {10.1038/s41588-024-01885-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1804-1810},
  shortjournal = {Nat. Genet.},
  title        = {Homozygosity for a stop-gain variant in CCDC201 causes primary ovarian insufficiency},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A genomics england haplotype reference panel and imputation
of UK biobank. <em>NG</em>, <em>56</em>(9), 1800–1803. (<a
href="https://doi.org/10.1038/s41588-024-01868-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We built a reference panel with 342 million autosomal variants using 78,195 individuals from the Genomics England (GEL) dataset, achieving a phasing switch error rate of 0.18% for European samples and imputation quality of r2 = 0.75 for variants with minor allele frequencies as low as 2 × 10−4 in white British samples. The GEL-imputed UK Biobank genome-wide association analysis identified 70% of associations found by direct exome sequencing (P &amp;lt; 2.18 × 10−11), while extending testing of rare variants to the entire genome. Coding variants dominated the rare-variant genome-wide association results, implying less disruptive effects of rare non-coding variants. A Genomics England haplotype reference panel constructed using sequence data from 78,195 individuals improves phasing and imputation accuracy. Imputation of the UK Biobank using this panel enables genome-wide rare-variant association analyses.},
  archive      = {J_NG},
  author       = {Shi, Sinan and Rubinacci, Simone and Hu, Sile and Moutsianas, Loukas and Stuckey, Alex and Need, Anna C. and Palamara, Pier Francesco and Caulfield, Mark and Marchini, Jonathan and Myers, Simon},
  doi          = {10.1038/s41588-024-01868-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1800-1803},
  shortjournal = {Nat. Genet.},
  title        = {A genomics england haplotype reference panel and imputation of UK biobank},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unlocking plant genetics with telomere-to-telomere genome
assemblies. <em>NG</em>, <em>56</em>(9), 1788–1799. (<a
href="https://doi.org/10.1038/s41588-024-01830-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Contiguous genome sequence assemblies will help us to realize the full potential of crop translational genomics. Recent advances in sequencing technologies, especially long-read sequencing strategies, have made it possible to construct gapless telomere-to-telomere (T2T) assemblies, thus offering novel insights into genome organization and function. Plant genomes pose unique challenges, such as a continuum of ancient to recent polyploidy and abundant highly similar and long repetitive elements. Owing to progress in sequencing approaches, for most crop plants, chromosome-scale reference genome assemblies are available, but T2T assembly construction remains challenging. Here we describe methods for haplotype-resolved, gapless T2T assembly construction in plants, including various crop species. We outline the impact of T2T assemblies in elucidating the roles of repetitive elements in gene regulation, as well as in pangenomics, functional genomics, genome-assisted breeding and targeted genome manipulation. In conjunction with sequence-enriched germplasm repositories, T2T assemblies thus hold great promise for basic and applied plant sciences. This Review explores the challenges and strategies for developing gapless telomere-to-telomere genome assemblies to enhance our understanding of genome organization and improve crops through genome-assisted breeding or gene editing.},
  archive      = {J_NG},
  author       = {Garg, Vanika and Bohra, Abhishek and Mascher, Martin and Spannagl, Manuel and Xu, Xun and Bevan, Michael W. and Bennetzen, Jeffrey L. and Varshney, Rajeev K.},
  doi          = {10.1038/s41588-024-01830-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1788-1799},
  shortjournal = {Nat. Genet.},
  title        = {Unlocking plant genetics with telomere-to-telomere genome assemblies},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Promoting equity in polygenic risk assessment through global
collaboration. <em>NG</em>, <em>56</em>(9), 1780–1787. (<a
href="https://doi.org/10.1038/s41588-024-01843-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The long delay before genomic technologies become available in low- and middle-income countries is a concern from both scientific and ethical standpoints. Polygenic risk scores (PRSs), a relatively recent advance in genomics, could have a substantial impact on promoting health by improving disease risk prediction and guiding preventive strategies. However, clinical use of PRSs in their current forms might widen global health disparities, as their portability to diverse groups is limited. This Perspective highlights the need for global collaboration to develop and implement PRSs that perform equitably across the world. Such collaboration requires capacity building and the generation of new data in low-resource settings, the sharing of harmonized genotype and phenotype data securely across borders, novel population genetics and statistical methods to improve PRS performance, and thoughtful clinical implementation in diverse settings. All this needs to occur while considering the ethical, legal and social implications, with support from regulatory and funding agencies and policymakers. The clinical use of current polygenic risk scores (PRSs) might widen global health disparities, as their portability to diverse groups is limited, emphasizing the need for global collaboration to develop PRSs that perform equitably across the world.},
  archive      = {J_NG},
  author       = {Kullo, Iftikhar J.},
  doi          = {10.1038/s41588-024-01843-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1780-1787},
  shortjournal = {Nat. Genet.},
  title        = {Promoting equity in polygenic risk assessment through global collaboration},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Guidance for estimating penetrance of monogenic
disease-causing variants in population cohorts. <em>NG</em>,
<em>56</em>(9), 1772–1779. (<a
href="https://doi.org/10.1038/s41588-024-01842-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Penetrance is the probability that an individual with a pathogenic genetic variant develops a specific disease. Knowing the penetrance of variants for monogenic disorders is important for counseling of individuals. Until recently, estimates of penetrance have largely relied on affected individuals and their at-risk family members being clinically referred for genetic testing, a ‘phenotype-first’ approach. This approach substantially overestimates the penetrance of variants because of ascertainment bias. The recent availability of whole-genome sequencing data in individuals from very-large-scale population-based cohorts now allows ‘genotype-first’ estimates of penetrance for many conditions. Although this type of population-based study can underestimate penetrance owing to recruitment biases, it provides more accurate estimates of penetrance for secondary or incidental findings. Here, we provide guidance for the conduct of penetrance studies to ensure that robust genotypes and phenotypes are used to accurately estimate penetrance of variants and groups of similarly annotated variants from population-based studies. This Perspective explores challenges in calculating the penetrance of monogenic disease-associated variants in the context of large-scale population sequencing cohorts.},
  archive      = {J_NG},
  author       = {Wright, Caroline F. and Sharp, Luke N. and Jackson, Leigh and Murray, Anna and Ware, James S. and MacArthur, Daniel G. and Rehm, Heidi L. and Patel, Kashyap A. and Weedon, Michael N.},
  doi          = {10.1038/s41588-024-01842-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1772-1779},
  shortjournal = {Nat. Genet.},
  title        = {Guidance for estimating penetrance of monogenic disease-causing variants in population cohorts},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Activating the dark genome to illuminate cancer vaccine
targets. <em>NG</em>, <em>56</em>(9), 1770–1771. (<a
href="https://doi.org/10.1038/s41588-024-01850-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetic therapy triggers myriad transposable elements to generate new antigens that could prime tumor cells for immunotherapy. A study of glioblastoma discovers indiscriminate activation in healthy cells as well, and presents a more selective strategy for potential therapeutic targeting.},
  archive      = {J_NG},
  author       = {Kwok, Darwin W. and Okada, Hideho and Costello, Joseph F.},
  doi          = {10.1038/s41588-024-01850-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1770-1771},
  shortjournal = {Nat. Genet.},
  title        = {Activating the dark genome to illuminate cancer vaccine targets},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Harnessing cancer genomes for precision oncology.
<em>NG</em>, <em>56</em>(9), 1768–1769. (<a
href="https://doi.org/10.1038/s41588-024-01879-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precision oncology has just received a boost: a report on the prevalence of mutations in cancer driver genes based on whole genome sequencing of 10,000 clinical cases. The challenge ahead lies in how to explore the data to accelerate new discoveries in cancer biology while advancing precision oncology.},
  archive      = {J_NG},
  author       = {Chanock, Stephen J.},
  doi          = {10.1038/s41588-024-01879-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1768-1769},
  shortjournal = {Nat. Genet.},
  title        = {Harnessing cancer genomes for precision oncology},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Stopped clinical trials give evidence for the value of
genetics. <em>NG</em>, <em>56</em>(9), 1766–1767. (<a
href="https://doi.org/10.1038/s41588-024-01834-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new analysis of clinical trials that were stopped early generates insights on the role of genetics in drug development and provides a new resource for researchers aiming to improve success rates of clinical trials.},
  archive      = {J_NG},
  author       = {King, Emily},
  doi          = {10.1038/s41588-024-01834-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1766-1767},
  shortjournal = {Nat. Genet.},
  title        = {Stopped clinical trials give evidence for the value of genetics},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). X chromosome dosage shapes renal cell carcinoma risk.
<em>NG</em>, <em>56</em>(9), 1765. (<a
href="https://doi.org/10.1038/s41588-024-01924-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-024-01924-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1765},
  shortjournal = {Nat. Genet.},
  title        = {X chromosome dosage shapes renal cell carcinoma risk},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Polygenic basis for seedless grapes. <em>NG</em>,
<em>56</em>(9), 1765. (<a
href="https://doi.org/10.1038/s41588-024-01926-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-024-01926-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1765},
  shortjournal = {Nat. Genet.},
  title        = {Polygenic basis for seedless grapes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Human intestinal immuno-organoids. <em>NG</em>,
<em>56</em>(9), 1765. (<a
href="https://doi.org/10.1038/s41588-024-01927-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-024-01927-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1765},
  shortjournal = {Nat. Genet.},
  title        = {Human intestinal immuno-organoids},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). The genomics of childhood t-lineage acute lymphoblastic
leukemia. <em>NG</em>, <em>56</em>(9), 1765. (<a
href="https://doi.org/10.1038/s41588-024-01928-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-024-01928-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1765},
  shortjournal = {Nat. Genet.},
  title        = {The genomics of childhood T-lineage acute lymphoblastic leukemia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Publisher correction: MYC activity at enhancers drives
prognostic transcriptional programs through an epigenetic switch.
<em>NG</em>, <em>56</em>(8), 1764. (<a
href="https://doi.org/10.1038/s41588-024-01856-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Jakobsen, Simon T. and Jensen, Rikke A. M. and Madsen, Maria S. and Ravnsborg, Tina and Vaagenso, Christian S. and Siersbæk, Majken S. and Einarsson, Hjorleifur and Andersson, Robin and Jensen, Ole N. and Siersbæk, Rasmus},
  doi          = {10.1038/s41588-024-01856-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1764},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: MYC activity at enhancers drives prognostic transcriptional programs through an epigenetic switch},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Publisher correction: Understanding the genetic complexity
of puberty timing across the allele frequency spectrum. <em>NG</em>,
<em>56</em>(8), 1763–1764. (<a
href="https://doi.org/10.1038/s41588-024-01857-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Kentistou, Katherine A. and Kaisinger, Lena R. and Stankovic, Stasa and Vaudel, Marc and Mendes de Oliveira, Edson and Messina, Andrea and Walters, Robin G. and Liu, Xiaoxi and Busch, Alexander S. and Helgason, Hannes and Thompson, Deborah J. and Santoni, Federico and Petricek, Konstantin M. and Zouaghi, Yassine and Huang-Doran, Isabel and Gudbjartsson, Daniel F. and Bratland, Eirik and Lin, Kuang and Gardner, Eugene J. and Zhao, Yajie and Jia, Raina Y. and Terao, Chikashi and Riggan, Marjorie J. and Bolla, Manjeet K. and Yazdanpanah, Mojgan and Yazdanpanah, Nahid and Bradfield, Jonathan P. and Broer, Linda and Campbell, Archie and Chasman, Daniel I. and Cousminer, Diana L. and Franceschini, Nora and Franke, Lude H. and Girotto, Giorgia and He, Chunyan and Järvelin, Marjo-Riitta and Joshi, Peter K. and Kamatani, Yoichiro and Karlsson, Robert and Luan, Jian’an and Lunetta, Kathryn L. and Mägi, Reedik and Mangino, Massimo and Medland, Sarah E. and Meisinger, Christa and Noordam, Raymond and Nutile, Teresa and Concas, Maria Pina and Polašek, Ozren and Porcu, Eleonora and Ring, Susan M. and Sala, Cinzia and Smith, Albert V. and Tanaka, Toshiko and van der Most, Peter J. and Vitart, Veronique and Wang, Carol A. and Willemsen, Gonneke and Zygmunt, Marek and Ahearn, Thomas U. and Andrulis, Irene L. and Anton-Culver, Hoda and Antoniou, Antonis C. and Auer, Paul L. and Barnes, Catriona L. K. and Beckmann, Matthias W. and Berrington de Gonzalez, Amy and Bogdanova, Natalia V. and Bojesen, Stig E. and Brenner, Hermann and Buring, Julie E. and Canzian, Federico and Chang-Claude, Jenny and Couch, Fergus J. and Cox, Angela and Crisponi, Laura and Czene, Kamila and Daly, Mary B. and Demerath, Ellen W. and Dennis, Joe and Devilee, Peter and De Vivo, Immaculata and Dörk, Thilo and Dunning, Alison M. and Dwek, Miriam and Eriksson, Johan G. and Fasching, Peter A. and Fernandez-Rhodes, Lindsay and Ferreli, Liana and Fletcher, Olivia and Gago-Dominguez, Manuela and García-Closas, Montserrat and García-Sáenz, José A. and González-Neira, Anna and Grallert, Harald and Guénel, Pascal and Haiman, Christopher A. and Hall, Per and Hamann, Ute and Hakonarson, Hakon and Hart, Roger J. and Hickey, Martha and Hooning, Maartje J. and Hoppe, Reiner and Hopper, John L. and Hottenga, Jouke-Jan and Hu, Frank B. and Huebner, Hanna and Hunter, David J. and Jernström, Helena and John, Esther M. and Karasik, David and Khusnutdinova, Elza K. and Kristensen, Vessela N. and Lacey, James V. and Lambrechts, Diether and Launer, Lenore J. and Lind, Penelope A. and Lindblom, Annika and Magnusson, Patrik K. E. and Mannermaa, Arto and McCarthy, Mark I. and Meitinger, Thomas and Menni, Cristina and Michailidou, Kyriaki and Millwood, Iona Y. and Milne, Roger L. and Montgomery, Grant W. and Nevanlinna, Heli and Nolte, Ilja M. and Nyholt, Dale R. and Obi, Nadia and O’Brien, Katie M. and Offit, Kenneth and Oldehinkel, Albertine J. and Ostrowski, Sisse R. and Palotie, Aarno and Pedersen, Ole B. and Peters, Annette and Pianigiani, Giulia and Plaseska-Karanfilska, Dijana and Pouta, Anneli and Pozarickij, Alfred and Radice, Paolo and Rennert, Gad and Rosendaal, Frits R. and Ruggiero, Daniela and Saloustros, Emmanouil and Sandler, Dale P. and Schipf, Sabine and Schmidt, Carsten O. and Schmidt, Marjanka K. and Small, Kerrin and Spedicati, Beatrice and Stampfer, Meir and Stone, Jennifer and Tamimi, Rulla M. and Teras, Lauren R. and Tikkanen, Emmi and Turman, Constance and Vachon, Celine M. and Wang, Qin and Winqvist, Robert and Wolk, Alicja and Zemel, Babette S. and Zheng, Wei and van Dijk, Ko W. and Alizadeh, Behrooz Z. and Bandinelli, Stefania and Boerwinkle, Eric and Boomsma, Dorret I. and Ciullo, Marina and Chenevix-Trench, Georgia and Cucca, Francesco and Esko, Tõnu and Gieger, Christian and Grant, Struan F. A. and Gudnason, Vilmundur and Hayward, Caroline and Kolčić, Ivana and Kraft, Peter and Lawlor, Deborah A. and Martin, Nicholas G. and Nøhr, Ellen A. and Pedersen, Nancy L. and Pennell, Craig E. and Ridker, Paul M. and Robino, Antonietta and Snieder, Harold and Sovio, Ulla and Spector, Tim D. and Stöckl, Doris and Sudlow, Cathie and Timpson, Nic J. and Toniolo, Daniela and Uitterlinden, André and Ulivi, Sheila and Völzke, Henry and Wareham, Nicholas J. and Widen, Elisabeth and Wilson, James F. and Pharoah, Paul D. P. and Li, Liming and Easton, Douglas F. and Njølstad, Pål R. and Sulem, Patrick and Murabito, Joanne M. and Murray, Anna and Manousaki, Despoina and Juul, Anders and Erikstrup, Christian and Stefansson, Kari and Horikoshi, Momoko and Chen, Zhengming and Farooqi, I. Sadaf and Pitteloud, Nelly and Johansson, Stefan and Day, Felix R. and Perry, John R. B. and Ong, Ken K.},
  doi          = {10.1038/s41588-024-01857-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1763-1764},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Understanding the genetic complexity of puberty timing across the allele frequency spectrum},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: LINE-1 transcription activates
long-range gene expression. <em>NG</em>, <em>56</em>(8), 1762. (<a
href="https://doi.org/10.1038/s41588-024-01866-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Xiufeng and Bie, Luyao and Wang, Yang and Hong, Yaqiang and Zhou, Ziqiang and Fan, Yiming and Yan, Xiaohan and Tao, Yibing and Huang, Chunyi and Zhang, Yongyan and Sun, Xueyan and Li, John Xiao He and Zhang, Jing and Chang, Zai and Xi, Qiaoran and Meng, Anming and Shen, Xiaohua and Xie, Wei and Liu, Nian},
  doi          = {10.1038/s41588-024-01866-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1762},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: LINE-1 transcription activates long-range gene expression},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Telomere-to-telomere citrullus super-pangenome provides
direction for watermelon breeding. <em>NG</em>, <em>56</em>(8),
1750–1761. (<a
href="https://doi.org/10.1038/s41588-024-01823-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To decipher the genetic diversity within the cucurbit genus Citrullus, we generated telomere-to-telomere (T2T) assemblies of 27 distinct genotypes, encompassing all seven Citrullus species. This T2T super-pangenome has expanded the previously published reference genome, T2T-G42, by adding 399.2 Mb and 11,225 genes. Comparative analysis has unveiled gene variants and structural variations (SVs), shedding light on watermelon evolution and domestication processes that enhanced attributes such as bitterness and sugar content while compromising disease resistance. Multidisease-resistant loci from Citrullus amarus and Citrullus mucosospermus were successfully introduced into cultivated Citrullus lanatus. The SVs identified in C. lanatus have not only been inherited from cordophanus but also from C. mucosospermus, suggesting additional ancestors beyond cordophanus in the lineage of cultivated watermelon. Our investigation substantially improves the comprehension of watermelon genome diversity, furnishing comprehensive reference genomes for all Citrullus species. This advancement aids in the exploration and genetic enhancement of watermelon using its wild relatives. Citrullus super-pangenome constructed using 27 telomere-to-telomere assemblies encompassing all seven Citrullus species highlights genomic diversity across wild and cultivated watermelons and the potential for crop improvement.},
  archive      = {J_NG},
  author       = {Zhang, Yilin and Zhao, Mingxia and Tan, Jingsheng and Huang, Minghan and Chu, Xiao and Li, Yan and Han, Xue and Fang, Taohong and Tian, Yao and Jarret, Robert and Lu, Dongdong and Chen, Yijun and Xue, Lifang and Li, Xiaoni and Qin, Guochen and Li, Bosheng and Sun, Yudong and Deng, Xing Wang and Deng, Yun and Zhang, Xingping and He, Hang},
  doi          = {10.1038/s41588-024-01823-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1750-1761},
  shortjournal = {Nat. Genet.},
  title        = {Telomere-to-telomere citrullus super-pangenome provides direction for watermelon breeding},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Condensin i folds the caenorhabditis elegans genome.
<em>NG</em>, <em>56</em>(8), 1737–1749. (<a
href="https://doi.org/10.1038/s41588-024-01832-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The structural maintenance of chromosome (SMC) complexes—cohesin and condensins—are crucial for chromosome separation and compaction during cell division. During the interphase, mammalian cohesins additionally fold the genome into loops and domains. Here we show that, in Caenorhabditis elegans, a species with holocentric chromosomes, condensin I is the primary, long-range loop extruder. The loss of condensin I and its X-specific variant, condensin IDC, leads to genome-wide decompaction, chromosome mixing and disappearance of X-specific topologically associating domains, while reinforcing fine-scale epigenomic compartments. In addition, condensin I/IDC inactivation led to the upregulation of X-linked genes and unveiled nuclear bodies grouping together binding sites for the X-targeting loading complex of condensin IDC. C. elegans condensin I/IDC thus uniquely organizes holocentric interphase chromosomes, akin to cohesin in mammals, as well as regulates X-chromosome gene expression. Inactivation of somatic SMC complexes in Caenorhabditis elegans shows that condensin I is the major long-range genome loop extruder, while cohesin forms small loops. Inactivation of cohesin, condensin II and condensin I/IDC causes minor transcriptional changes in autosomes.},
  archive      = {J_NG},
  author       = {Das, Moushumi and Semple, Jennifer I. and Haemmerli, Anja and Volodkina, Valeriia and Scotton, Janik and Gitchev, Todor and Annan, Ahrmad and Campos, Julie and Statzer, Cyril and Dakhovnik, Alexander and Ewald, Collin Y. and Mozziconacci, Julien and Meister, Peter},
  doi          = {10.1038/s41588-024-01832-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1737-1749},
  shortjournal = {Nat. Genet.},
  title        = {Condensin i folds the caenorhabditis elegans genome},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping spatially resolved transcriptomes in human and mouse
pulmonary fibrosis. <em>NG</em>, <em>56</em>(8), 1725–1736. (<a
href="https://doi.org/10.1038/s41588-024-01819-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Idiopathic pulmonary fibrosis (IPF) is a progressive lung disease with poor prognosis and limited treatment options. Efforts to identify effective treatments are thwarted by limited understanding of IPF pathogenesis and poor translatability of available preclinical models. Here we generated spatially resolved transcriptome maps of human IPF (n = 4) and bleomycin-induced mouse pulmonary fibrosis (n = 6) to address these limitations. We uncovered distinct fibrotic niches in the IPF lung, characterized by aberrant alveolar epithelial cells in a microenvironment dominated by transforming growth factor beta signaling alongside predicted regulators, such as TP53 and APOE. We also identified a clear divergence between the arrested alveolar regeneration in the IPF fibrotic niches and the active tissue repair in the acutely fibrotic mouse lung. Our study offers in-depth insights into the IPF transcriptional landscape and proposes alveolar regeneration as a promising therapeutic strategy for IPF. Spatially resolved transcriptome analysis of human and mouse idiopathic pulmonary fibrosis identifies disease-associated niches and a role for aberrant alveolar epithelial cells in human disease pathogenesis.},
  archive      = {J_NG},
  author       = {Franzén, Lovisa and Olsson Lindvall, Martina and Hühn, Michael and Ptasinski, Victoria and Setyo, Laura and Keith, Benjamin P. and Collin, Astrid and Oag, Steven and Volckaert, Thomas and Borde, Annika and Lundeberg, Joakim and Lindgren, Julia and Belfield, Graham and Jackson, Sonya and Ollerstam, Anna and Stamou, Marianna and Ståhl, Patrik L. and Hornberg, Jorrit J.},
  doi          = {10.1038/s41588-024-01819-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1725-1736},
  shortjournal = {Nat. Genet.},
  title        = {Mapping spatially resolved transcriptomes in human and mouse pulmonary fibrosis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell multi-omic and spatial profiling of human
kidneys implicates the fibrotic microenvironment in kidney disease
progression. <em>NG</em>, <em>56</em>(8), 1712–1724. (<a
href="https://doi.org/10.1038/s41588-024-01802-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Kidneys are intricate three-dimensional structures in the body, yet the spatial and molecular principles of kidney health and disease remain inadequately understood. We generated high-quality datasets for 81 samples, including single-cell, single-nuclear, spot-level (Visium) and single-cell resolution (CosMx) spatial-RNA expression and single-nuclear open chromatin, capturing cells from healthy, diabetic and hypertensive diseased human kidneys. Combining these data, we identify cell types and map them to their locations within the tissue. Unbiased deconvolution of the spatial data identifies the following four distinct microenvironments: glomerular, immune, tubule and fibrotic. We describe the complex organization of microenvironments in health and disease and find that the fibrotic microenvironment is able to molecularly classify human kidneys and offers an improved prognosis compared to traditional histopathology. We provide a comprehensive spatially resolved molecular roadmap of the human kidney and the fibrotic process, demonstrating the clinical utility of spatial transcriptomics. A spatial transcriptomic analysis of healthy kidneys and those from individuals with chronic kidney disease characterizes the fibrotic microenvironment and highlights features that could be used to predict prognosis.},
  archive      = {J_NG},
  author       = {Abedini, Amin and Levinsohn, Jonathan and Klötzer, Konstantin A. and Dumoulin, Bernhard and Ma, Ziyuan and Frederick, Julia and Dhillon, Poonam and Balzer, Michael S. and Shrestha, Rojesh and Liu, Hongbo and Vitale, Steven and Bergeson, Andi M. and Devalaraja-Narashimha, Kishor and Grandi, Paola and Bhattacharyya, Tanmoy and Hu, Erding and Pullen, Steven S. and Boustany-Kari, Carine M. and Guarnieri, Paolo and Karihaloo, Anil and Traum, Daniel and Yan, Hanying and Coleman, Kyle and Palmer, Matthew and Sarov-Blat, Lea and Morton, Lori and Hunter, Christopher A. and Kaestner, Klaus H. and Li, Mingyao and Susztak, Katalin},
  doi          = {10.1038/s41588-024-01802-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1712-1724},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-omic and spatial profiling of human kidneys implicates the fibrotic microenvironment in kidney disease progression},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). GAGE-seq concurrently profiles multiscale 3D genome
organization and gene expression in single cells. <em>NG</em>,
<em>56</em>(8), 1701–1711. (<a
href="https://doi.org/10.1038/s41588-024-01745-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The organization of mammalian genomes features a complex, multiscale three-dimensional (3D) architecture, whose functional significance remains elusive because of limited single-cell technologies that can concurrently profile genome organization and transcriptional activities. Here, we introduce genome architecture and gene expression by sequencing (GAGE-seq), a scalable, robust single-cell co-assay measuring 3D genome structure and transcriptome simultaneously within the same cell. Applied to mouse brain cortex and human bone marrow CD34+ cells, GAGE-seq characterized the intricate relationships between 3D genome and gene expression, showing that multiscale 3D genome features inform cell-type-specific gene expression and link regulatory elements to target genes. Integration with spatial transcriptomic data revealed in situ 3D genome variations in mouse cortex. Observations in human hematopoiesis unveiled discordant changes between 3D genome organization and gene expression, underscoring a complex, temporal interplay at the single-cell level. GAGE-seq provides a powerful, cost-effective approach for exploring genome structure and gene expression relationships at the single-cell level across diverse biological contexts. GAGE-seq is a joint assay for 3D genome and transcriptome in single cells using combinatorial indexing to increase throughput. Applied to complex tissues, GAGE-seq enables the analysis of links between 3D organization and gene expression in rare cell types.},
  archive      = {J_NG},
  author       = {Zhou, Tianming and Zhang, Ruochi and Jia, Deyong and Doty, Raymond T. and Munday, Adam D. and Gao, Daniel and Xin, Li and Abkowitz, Janis L. and Duan, Zhijun and Ma, Jian},
  doi          = {10.1038/s41588-024-01745-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1701-1711},
  shortjournal = {Nat. Genet.},
  title        = {GAGE-seq concurrently profiles multiscale 3D genome organization and gene expression in single cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integrated analyses highlight interactions between the
three-dimensional genome and DNA, RNA and epigenomic alterations in
metastatic prostate cancer. <em>NG</em>, <em>56</em>(8), 1689–1700. (<a
href="https://doi.org/10.1038/s41588-024-01826-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of variations in the three-dimensional structure of the genome has been recognized, but solid cancer tissue studies are limited. Here, we performed integrated deep Hi-C sequencing with matched whole-genome sequencing, whole-genome bisulfite sequencing, 5-hydroxymethylcytosine (5hmC) sequencing and RNA sequencing across a cohort of 80 biopsy samples from patients with metastatic castration-resistant prostate cancer. Dramatic differences were present in gene expression, 5-methylcytosine/5hmC methylation and in structural variation versus mutation rate between A and B (open and closed) chromatin compartments. A subset of tumors exhibited depleted regional chromatin contacts at the AR locus, linked to extrachromosomal circular DNA (ecDNA) and worse response to AR signaling inhibitors. We also identified topological subtypes associated with stark differences in methylation structure, gene expression and prognosis. Our data suggested that DNA interactions may predispose to structural variant formation, exemplified by the recurrent TMPRSS2–ERG fusion. This comprehensive integrated sequencing effort represents a unique clinical tumor resource. A multiomic approach profiles the three-dimensional, epigenetic and mutational landscapes of 80 metastatic prostate cancer biopsies. Hi-C experiments identify an extrachromosomal circular DNA at the AR locus associated with therapy resistance.},
  archive      = {J_NG},
  author       = {Zhao, Shuang G. and Bootsma, Matthew and Zhou, Stanley and Shrestha, Raunak and Moreno-Rodriguez, Thaidy and Lundberg, Arian and Pan, Chu and Arlidge, Christopher and Hawley, James R. and Foye, Adam and Weinstein, Alana S. and Sjöström, Martin and Zhang, Meng and Li, Haolong and Chesner, Lisa N. and Rydzewski, Nicholas R. and Helzer, Kyle T. and Shi, Yue and Lynch, Molly and Dehm, Scott M. and Lang, Joshua M. and Alumkal, Joshi J. and He, Hansen H. and Wyatt, Alexander W. and Aggarwal, Rahul and Zwart, Wilbert and Small, Eric J. and Quigley, David A. and Lupien, Mathieu and Feng, Felix Y.},
  doi          = {10.1038/s41588-024-01826-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1689-1700},
  shortjournal = {Nat. Genet.},
  title        = {Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Competition shapes the landscape of x-chromosome-linked
genetic diversity. <em>NG</em>, <em>56</em>(8), 1678–1688. (<a
href="https://doi.org/10.1038/s41588-024-01840-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {X chromosome inactivation (XCI) generates clonal heterogeneity within XX individuals. Combined with sequence variation between human X chromosomes, XCI gives rise to intra-individual clonal diversity, whereby two sets of clones express mutually exclusive sequence variants present on one or the other X chromosome. Here we ask whether such clones merely co-exist or potentially interact with each other to modulate the contribution of X-linked diversity to organismal development. Focusing on X-linked coding variation in the human STAG2 gene, we show that Stag2variant clones contribute to most tissues at the expected frequencies but fail to form lymphocytes in Stag2WT Stag2variant mouse models. Unexpectedly, the absence of Stag2variant clones from the lymphoid compartment is due not solely to cell-intrinsic defects but requires continuous competition by Stag2WT clones. These findings show that interactions between epigenetically diverse clones can operate in an XX individual to shape the contribution of X-linked genetic diversity in a cell-type-specific manner. Clones expressing variants of the X-chromosome-linked STAG2 gene only form lymphocytes in the absence of Stag2WT clones. Interactions between epigenetic clones can thus modulate the impact of X-chromosome-linked genetic diversity in a cell-type-specific manner.},
  archive      = {J_NG},
  author       = {Buenaventura, Teresa and Bagci, Hakan and Patrascan, Ilinca and Graham, Joshua J. and Hipwell, Kelsey D. and Oldenkamp, Roel and King, James W. D. and Urtasun, Jesus and Young, George and Mouzo, Daniel and Gomez-Cabrero, David and Rowland, Benjamin D. and Panne, Daniel and Fisher, Amanda G. and Merkenschlager, Matthias},
  doi          = {10.1038/s41588-024-01840-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1678-1688},
  shortjournal = {Nat. Genet.},
  title        = {Competition shapes the landscape of X-chromosome-linked genetic diversity},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mitochondrial DNA mosaicism in normal human somatic cells.
<em>NG</em>, <em>56</em>(8), 1665–1677. (<a
href="https://doi.org/10.1038/s41588-024-01838-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic cells accumulate genomic alterations with age; however, our understanding of mitochondrial DNA (mtDNA) mosaicism remains limited. Here we investigated the genomes of 2,096 clones derived from three cell types across 31 donors, identifying 6,451 mtDNA variants with heteroplasmy levels of ≳0.3%. While the majority of these variants were unique to individual clones, suggesting stochastic acquisition with age, 409 variants (6%) were shared across multiple embryonic lineages, indicating their origin from heteroplasmy in fertilized eggs. The mutational spectrum exhibited replication-strand bias, implicating mtDNA replication as a major mutational process. We evaluated the mtDNA mutation rate (5.0 × 10−8 per base pair) and a turnover frequency of 10–20 per year, which are fundamental components shaping the landscape of mtDNA mosaicism over a lifetime. The expansion of mtDNA-truncating mutations toward homoplasmy was substantially suppressed. Our findings provide comprehensive insights into the origins, dynamics and functional consequences of mtDNA mosaicism in human somatic cells. Analysis of 2,096 single-cell clones from three tissues of 31 healthy donors characterizes mitochondrial DNA mosaicism and highlights the following two origins of mtDNA variants: heteroplasmy in the fertilized egg and postzygotic mutations.},
  archive      = {J_NG},
  author       = {An, Jisong and Nam, Chang Hyun and Kim, Ryul and Lee, Yunah and Won, Hyein and Park, Seongyeol and Lee, Won Hee and Park, Hansol and Yoon, Christopher J. and An, Yohan and Kim, Jie-Hyun and Jun, Jong Kwan and Bae, Jeong Mo and Shin, Eui-Cheol and Kim, Bun and Cha, Yong Jun and Kwon, Hyun Woo and Oh, Ji Won and Park, Jee Yoon and Kim, Min Jung and Ju, Young Seok},
  doi          = {10.1038/s41588-024-01838-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1665-1677},
  shortjournal = {Nat. Genet.},
  title        = {Mitochondrial DNA mosaicism in normal human somatic cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cohesin prevents cross-domain gene coactivation.
<em>NG</em>, <em>56</em>(8), 1654–1664. (<a
href="https://doi.org/10.1038/s41588-024-01852-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The contrast between the disruption of genome topology after cohesin loss and the lack of downstream gene expression changes instigates intense debates regarding the structure–function relationship between genome and gene regulation. Here, by analyzing transcriptome and chromatin accessibility at the single-cell level, we discover that, instead of dictating population-wide gene expression levels, cohesin supplies a general function to neutralize stochastic coexpression tendencies of cis-linked genes in single cells. Notably, cohesin loss induces widespread gene coactivation and chromatin co-opening tens of million bases apart in cis. Spatial genome and protein imaging reveals that cohesin prevents gene co-bursting along the chromosome and blocks spatial mixing of transcriptional hubs. Single-molecule imaging shows that cohesin confines the exploration of diverse enhancer and core promoter binding transcriptional regulators. Together, these results support that cohesin arranges nuclear topology to control gene coexpression in single cells. Single-cell analyses in RAD21-depleted mouse embryonic stem cells show that cohesin limits the coexpression of genes located in different accessible chromatin domains on the same chromosome by preventing co-bursting and blocking spatial mixing of different transcriptional hubs.},
  archive      = {J_NG},
  author       = {Dong, Peng and Zhang, Shu and Gandin, Valentina and Xie, Liangqi and Wang, Lihua and Lemire, Andrew L. and Li, Wenhong and Otsuna, Hideo and Kawase, Takashi and Lander, Arthur D. and Chang, Howard Y. and Liu, Zhe J.},
  doi          = {10.1038/s41588-024-01852-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1654-1664},
  shortjournal = {Nat. Genet.},
  title        = {Cohesin prevents cross-domain gene coactivation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Next-generation phenotyping integrated in a national
framework for patients with ultrarare disorders improves genetic
diagnostics and yields new molecular findings. <em>NG</em>,
<em>56</em>(8), 1644–1653. (<a
href="https://doi.org/10.1038/s41588-024-01836-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Individuals with ultrarare disorders pose a structural challenge for healthcare systems since expert clinical knowledge is required to establish diagnoses. In TRANSLATE NAMSE, a 3-year prospective study, we evaluated a novel diagnostic concept based on multidisciplinary expertise in Germany. Here we present the systematic investigation of the phenotypic and molecular genetic data of 1,577 patients who had undergone exome sequencing and were partially analyzed with next-generation phenotyping approaches. Molecular genetic diagnoses were established in 32% of the patients totaling 370 distinct molecular genetic causes, most with prevalence below 1:50,000. During the diagnostic process, 34 novel and 23 candidate genotype–phenotype associations were identified, mainly in individuals with neurodevelopmental disorders. Sequencing data of the subcohort that consented to computer-assisted analysis of their facial images with GestaltMatcher could be prioritized more efficiently compared with approaches based solely on clinical features and molecular scores. Our study demonstrates the synergy of using next-generation sequencing and phenotyping for diagnosing ultrarare diseases in routine healthcare and discovering novel etiologies by multidisciplinary teams. Exome sequencing within a structured diagnostic process for rare diseases in Germany shows how facial image analysis and machine learning can guide variant prioritization and uncover many ultrarare diseases.},
  archive      = {J_NG},
  author       = {Schmidt, Axel and Danyel, Magdalena and Grundmann, Kathrin and Brunet, Theresa and Klinkhammer, Hannah and Hsieh, Tzung-Chien and Engels, Hartmut and Peters, Sophia and Knaus, Alexej and Moosa, Shahida and Averdunk, Luisa and Boschann, Felix and Sczakiel, Henrike Lisa and Schwartzmann, Sarina and Mensah, Martin Atta and Pantel, Jean Tori and Holtgrewe, Manuel and Bösch, Annemarie and Weiß, Claudia and Weinhold, Natalie and Suter, Aude-Annick and Stoltenburg, Corinna and Neugebauer, Julia and Kallinich, Tillmann and Kaindl, Angela M. and Holzhauer, Susanne and Bührer, Christoph and Bufler, Philip and Kornak, Uwe and Ott, Claus-Eric and Schülke, Markus and Nguyen, Hoa Huu Phuc and Hoffjan, Sabine and Grasemann, Corinna and Rothoeft, Tobias and Brinkmann, Folke and Matar, Nora and Sivalingam, Sugirthan and Perne, Claudia and Mangold, Elisabeth and Kreiss, Martina and Cremer, Kirsten and Betz, Regina C. and Mücke, Martin and Grigull, Lorenz and Klockgether, Thomas and Spier, Isabel and Heimbach, André and Bender, Tim and Brand, Fabian and Stieber, Christiane and Morawiec, Alexandra Marzena and Karakostas, Pantelis and Schäfer, Valentin S. and Bernsen, Sarah and Weydt, Patrick and Castro-Gomez, Sergio and Aziz, Ahmad and Grobe-Einsler, Marcus and Kimmich, Okka and Kobeleva, Xenia and Önder, Demet and Lesmann, Hellen and Kumar, Sheetal and Tacik, Pawel and Basin, Meghna Ahuja and Incardona, Pietro and Lee-Kirsch, Min Ae and Berner, Reinhard and Schuetz, Catharina and Körholz, Julia and Kretschmer, Tanita and Di Donato, Nataliya and Schröck, Evelin and Heinen, André and Reuner, Ulrike and Hanßke, Amalia-Mihaela and Kaiser, Frank J. and Manka, Eva and Munteanu, Martin and Kuechler, Alma and Cordula, Kiewert and Hirtz, Raphael and Schlapakow, Elena and Schlein, Christian and Lisfeld, Jasmin and Kubisch, Christian and Herget, Theresia and Hempel, Maja and Weiler-Normann, Christina and Ullrich, Kurt and Schramm, Christoph and Rudolph, Cornelia and Rillig, Franziska and Groffmann, Maximilian and Muntau, Ania and Tibelius, Alexandra and Schwaibold, Eva M. C. and Schaaf, Christian P. and Zawada, Michal and Kaufmann, Lilian and Hinderhofer, Katrin and Okun, Pamela M. and Kotzaeridou, Urania and Hoffmann, Georg F. and Choukair, Daniela and Bettendorf, Markus and Spielmann, Malte and Ripke, Annekatrin and Pauly, Martje and Münchau, Alexander and Lohmann, Katja and Hüning, Irina and Hanker, Britta and Bäumer, Tobias and Herzog, Rebecca and Hellenbroich, Yorck and Westphal, Dominik S. and Strom, Tim and Kovacs, Reka and Riedhammer, Korbinian M. and Mayerhanser, Katharina and Graf, Elisabeth and Brugger, Melanie and Hoefele, Julia and Oexle, Konrad and Mirza-Schreiber, Nazanin and Berutti, Riccardo and Schatz, Ulrich and Krenn, Martin and Makowski, Christine and Weigand, Heike and Schröder, Sebastian and Rohlfs, Meino and Vill, Katharina and Hauck, Fabian and Borggraefe, Ingo and Müller-Felber, Wolfgang and Kurth, Ingo and Elbracht, Miriam and Knopp, Cordula and Begemann, Matthias and Kraft, Florian and Lemke, Johannes R. and Hentschel, Julia and Platzer, Konrad and Strehlow, Vincent and Abou Jamra, Rami and Kehrer, Martin and Demidov, German and Beck-Wödl, Stefanie and Graessner, Holm and Sturm, Marc and Zeltner, Lena and Schöls, Ludger J. and Magg, Janine and Bevot, Andrea and Kehrer, Christiane and Kaiser, Nadja and Turro, Ernest and Horn, Denise and Grüters-Kieslich, Annette and Klein, Christoph and Mundlos, Stefan and Nöthen, Markus and Riess, Olaf and Meitinger, Thomas and Krude, Heiko and Krawitz, Peter M. and Haack, Tobias and Ehmke, Nadja and Wagner, Matias},
  doi          = {10.1038/s41588-024-01836-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1644-1653},
  shortjournal = {Nat. Genet.},
  title        = {Next-generation phenotyping integrated in a national framework for patients with ultrarare disorders improves genetic diagnostics and yields new molecular findings},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bayesian estimation of gene constraint from an evolutionary
model with gene features. <em>NG</em>, <em>56</em>(8), 1632–1643. (<a
href="https://doi.org/10.1038/s41588-024-01820-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Measures of selective constraint on genes have been used for many applications, including clinical interpretation of rare coding variants, disease gene discovery and studies of genome evolution. However, widely used metrics are severely underpowered at detecting constraints for the shortest ~25% of genes, potentially causing important pathogenic mutations to be overlooked. Here we developed a framework combining a population genetics model with machine learning on gene features to enable accurate inference of an interpretable constraint metric, shet. Our estimates outperform existing metrics for prioritizing genes important for cell essentiality, human disease and other phenotypes, especially for short genes. Our estimates of selective constraint should have wide utility for characterizing genes relevant to human disease. Finally, our inference framework, GeneBayes, provides a flexible platform that can improve the estimation of many gene-level properties, such as rare variant burden or gene expression differences. GeneBayes is a Bayesian approach incorporating a Wright–Fisher population model with machine learning of gene features to infer an interpretable gene constraint metric that has a broad range of uses in downstream analysis.},
  archive      = {J_NG},
  author       = {Zeng, Tony and Spence, Jeffrey P. and Mostafavi, Hakhamanesh and Pritchard, Jonathan K.},
  doi          = {10.1038/s41588-024-01820-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1632-1643},
  shortjournal = {Nat. Genet.},
  title        = {Bayesian estimation of gene constraint from an evolutionary model with gene features},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The correlation between CpG methylation and gene expression
is driven by sequence variants. <em>NG</em>, <em>56</em>(8), 1624–1631.
(<a href="https://doi.org/10.1038/s41588-024-01851-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene promoter and enhancer sequences are bound by transcription factors and are depleted of methylated CpG sites (cytosines preceding guanines in DNA). The absence of methylated CpGs in these sequences typically correlates with increased gene expression, indicating a regulatory role for methylation. We used nanopore sequencing to determine haplotype-specific methylation rates of 15.3 million CpG units in 7,179 whole-blood genomes. We identified 189,178 methylation depleted sequences where three or more proximal CpGs were unmethylated on at least one haplotype. A total of 77,789 methylation depleted sequences (~41%) associated with 80,503 cis-acting sequence variants, which we termed allele-specific methylation quantitative trait loci (ASM-QTLs). RNA sequencing of 896 samples from the same blood draws used to perform nanopore sequencing showed that the ASM-QTL, that is, DNA sequence variability, drives most of the correlation found between gene expression and CpG methylation. ASM-QTLs were enriched 40.2-fold (95% confidence interval 32.2, 49.9) among sequence variants associating with hematological traits, demonstrating that ASM-QTLs are important functional units in the noncoding genome. Allele-specific DNA methylation data in whole blood from 7,179 individuals sequenced by Nanopore, and gene expression profiles from 896 samples, show that DNA sequence variability accounts for most of the correlation between CpG methylation and gene expression.},
  archive      = {J_NG},
  author       = {Stefansson, Olafur Andri and Sigurpalsdottir, Brynja Dogg and Rognvaldsson, Solvi and Halldorsson, Gisli Hreinn and Juliusson, Kristinn and Sveinbjornsson, Gardar and Gunnarsson, Bjarni and Beyter, Doruk and Jonsson, Hakon and Gudjonsson, Sigurjon Axel and Olafsdottir, Thorunn Asta and Saevarsdottir, Saedis and Magnusson, Magnus Karl and Lund, Sigrun Helga and Tragante, Vinicius and Oddsson, Asmundur and Hardarson, Marteinn Thor and Eggertsson, Hannes Petur and Gudmundsson, Reynir L. and Sverrisson, Sverrir and Frigge, Michael L. and Zink, Florian and Holm, Hilma and Stefansson, Hreinn and Rafnar, Thorunn and Jonsdottir, Ingileif and Sulem, Patrick and Helgason, Agnar and Gudbjartsson, Daniel F. and Halldorsson, Bjarni V. and Thorsteinsdottir, Unnur and Stefansson, Kari},
  doi          = {10.1038/s41588-024-01851-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1624-1631},
  shortjournal = {Nat. Genet.},
  title        = {The correlation between CpG methylation and gene expression is driven by sequence variants},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Metabolic gene function discovery platform GeneMAP
identifies SLC25A48 as necessary for mitochondrial choline import.
<em>NG</em>, <em>56</em>(8), 1614–1623. (<a
href="https://doi.org/10.1038/s41588-024-01827-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organisms maintain metabolic homeostasis through the combined functions of small-molecule transporters and enzymes. While many metabolic components have been well established, a substantial number remains without identified physiological substrates. To bridge this gap, we have leveraged large-scale plasma metabolome genome-wide association studies (GWAS) to develop a multiomic Gene–Metabolite Association Prediction (GeneMAP) discovery platform. GeneMAP can generate accurate predictions and even pinpoint genes that are distant from the variants implicated by GWAS. In particular, our analysis identified solute carrier family 25 member 48 (SLC25A48) as a genetic determinant of plasma choline levels. Mechanistically, SLC25A48 loss strongly impairs mitochondrial choline import and synthesis of its downstream metabolite betaine. Integrative rare variant and polygenic score analyses in UK Biobank provide strong evidence that the SLC25A48 causal effects on human disease may in part be mediated by the effects of choline. Altogether, our study provides a discovery platform for metabolic gene function and proposes SLC25A48 as a mitochondrial choline transporter. This study presents a multiomic Gene–Metabolite Association Prediction (GeneMAP) platform for discovery of metabolic gene function and identifies SLC25A48 as a mediator of mitochondrial choline import.},
  archive      = {J_NG},
  author       = {Khan, Artem and Unlu, Gokhan and Lin, Phillip and Liu, Yuyang and Kilic, Ece and Kenny, Timothy C. and Birsoy, Kıvanç and Gamazon, Eric R.},
  doi          = {10.1038/s41588-024-01827-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1614-1623},
  shortjournal = {Nat. Genet.},
  title        = {Metabolic gene function discovery platform GeneMAP identifies SLC25A48 as necessary for mitochondrial choline import},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unsupervised representation learning on high-dimensional
clinical data improves genomic discovery and prediction. <em>NG</em>,
<em>56</em>(8), 1604–1613. (<a
href="https://doi.org/10.1038/s41588-024-01831-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although high-dimensional clinical data (HDCD) are increasingly available in biobank-scale datasets, their use for genetic discovery remains challenging. Here we introduce an unsupervised deep learning model, Representation Learning for Genetic Discovery on Low-Dimensional Embeddings (REGLE), for discovering associations between genetic variants and HDCD. REGLE leverages variational autoencoders to compute nonlinear disentangled embeddings of HDCD, which become the inputs to genome-wide association studies (GWAS). REGLE can uncover features not captured by existing expert-defined features and enables the creation of accurate disease-specific polygenic risk scores (PRSs) in datasets with very few labeled data. We apply REGLE to perform GWAS on respiratory and circulatory HDCD—spirograms measuring lung function and photoplethysmograms measuring blood volume changes. REGLE replicates known loci while identifying others not previously detected. REGLE are predictive of overall survival, and PRSs constructed from REGLE loci improve disease prediction across multiple biobanks. Overall, REGLE contain clinically relevant information beyond that captured by existing expert-defined features, leading to improved genetic discovery and disease prediction. Representation Learning for Genetic Discovery on Low-Dimensional Embeddings (REGLE) uses machine learning to generate low-dimensional representations of healthcare data. Applied to lung spirograms and blood volume photoplethysmograms, REGLE factors capture additional information beyond expert-defined features, suggesting the utility of this approach.},
  archive      = {J_NG},
  author       = {Yun, Taedong and Cosentino, Justin and Behsaz, Babak and McCaw, Zachary R. and Hill, Davin and Luben, Robert and Lai, Dongbing and Bates, John and Yang, Howard and Schwantes-An, Tae-Hwi and Zhou, Yuchen and Khawaja, Anthony P. and Carroll, Andrew and Hobbs, Brian D. and Cho, Michael H. and McLean, Cory Y. and Hormozdiari, Farhad},
  doi          = {10.1038/s41588-024-01831-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1604-1613},
  shortjournal = {Nat. Genet.},
  title        = {Unsupervised representation learning on high-dimensional clinical data improves genomic discovery and prediction},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide association meta-analysis identifies five loci
associated with postpartum hemorrhage. <em>NG</em>, <em>56</em>(8),
1597–1603. (<a
href="https://doi.org/10.1038/s41588-024-01839-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bleeding in early pregnancy and postpartum hemorrhage (PPH) bear substantial risks, with the former closely associated with pregnancy loss and the latter being the foremost cause of maternal death, underscoring the severe impact on maternal–fetal health. We identified five genetic loci linked to PPH in a meta-analysis. Functional annotation analysis indicated candidate genes HAND2, TBX3 and RAP2C/FRMD7 at three loci and showed that at each locus, associated variants were located within binding sites for progesterone receptors. There were strong genetic correlations with birth weight, gestational duration and uterine fibroids. Bleeding in early pregnancy yielded no genome-wide association signals but showed strong genetic correlation with various human traits, suggesting a potentially complex, polygenic etiology. Our results suggest that PPH is related to progesterone signaling dysregulation, whereas early bleeding is a complex trait associated with underlying health and possibly socioeconomic status and may include genetic factors that have not yet been identified. A European ancestry genome-wide meta-analysis of pregnancy-associated bleeding traits identifies five novel loci associated with postpartum hemorrhage, but none with early bleeding. Functional analysis highlights a role for progesterone receptor-mediated signaling.},
  archive      = {J_NG},
  author       = {Westergaard, David and Steinthorsdottir, Valgerdur and Stefansdottir, Lilja and Rohde, Palle Duun and Wu, Xiaoping and Geller, Frank and Tyrmi, Jaakko and Havulinna, Aki S. and Solé-Navais, Pol and Flatley, Christopher and Ostrowski, Sisse Rye and Pedersen, Ole Birger and Erikstrup, Christian and Sørensen, Erik and Mikkelsen, Christina and Bruun, Mie Topholm and Aagaard Jensen, Bitten and Brodersen, Thorsten and Ullum, Henrik and Magnus, Per and Andreassen, Ole A. and Njolstad, Pål R. and Kolte, Astrid Marie and Krebs, Lone and Nyegaard, Mette and Hansen, Thomas Folkmann and Feenstra, Bjarke and Daly, Mark and Lindgren, Cecilia M. and Thorleifsson, Gudmar and Stefansson, Olafur A. and Sveinbjornsson, Gardar and Gudbjartsson, Daniel F. and Thorsteinsdottir, Unnur and Banasik, Karina and Jacobsson, Bo and Laisk, Triin and Laivuori, Hannele and Stefansson, Kari and Brunak, Søren and Nielsen, Henriette Svarre},
  doi          = {10.1038/s41588-024-01839-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1597-1603},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analysis identifies five loci associated with postpartum hemorrhage},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic risk factors for COVID-19 and influenza are largely
distinct. <em>NG</em>, <em>56</em>(8), 1592–1596. (<a
href="https://doi.org/10.1038/s41588-024-01844-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coronavirus disease 2019 (COVID-19) and influenza are respiratory illnesses caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and influenza viruses, respectively. Both diseases share symptoms and clinical risk factors1, but the extent to which these conditions have a common genetic etiology is unknown. This is partly because host genetic risk factors are well characterized for COVID-19 but not for influenza, with the largest published genome-wide association studies for these conditions including &amp;gt;2 million individuals2 and about 1,000 individuals3–6, respectively. Shared genetic risk factors could point to targets to prevent or treat both infections. Through a genetic study of 18,334 cases with a positive test for influenza and 276,295 controls, we show that published COVID-19 risk variants are not associated with influenza. Furthermore, we discovered and replicated an association between influenza infection and noncoding variants in B3GALT5 and ST6GAL1, neither of which was associated with COVID-19. In vitro small interfering RNA knockdown of ST6GAL1—an enzyme that adds sialic acid to the cell surface, which is used for viral entry—reduced influenza infectivity by 57%. These results mirror the observation that variants that downregulate ACE2, the SARS-CoV-2 receptor, protect against COVID-19 (ref. 7). Collectively, these findings highlight downregulation of key cell surface receptors used for viral entry as treatment opportunities to prevent COVID-19 and influenza. Genome-wide analyses identify variants in B3GALT5 and ST6GAL1 associated with influenza susceptibility. Knockdown of ST6GAL1 in cell culture reduces influenza infectivity, likely by interfering with the glycoprotein modifications required for viral entry.},
  archive      = {J_NG},
  author       = {Kosmicki, Jack A. and Marcketta, Anthony and Sharma, Deepika and Di Gioia, Silvio Alessandro and Batista, Samantha and Yang, Xiao-Man and Tzoneva, Gannie and Martinez, Hector and Sidore, Carlo and Kessler, Michael D. and Horowitz, Julie E. and Roberts, Genevieve H. L. and Justice, Anne E. and Banerjee, Nilanjana and Coignet, Marie V. and Leader, Joseph B. and Park, Danny S. and Lanche, Rouel and Maxwell, Evan and Knight, Spencer C. and Bai, Xiaodong and Guturu, Harendra and Baltzell, Asher and Girshick, Ahna R. and McCurdy, Shannon R. and Partha, Raghavendran and Mansfield, Adam J. and Turissini, David A. and Zhang, Miao and Mbatchou, Joelle and Watanabe, Kyoko and Verma, Anurag and Sirugo, Giorgio and Ritchie, Marylyn D. and Salerno, William J. and Shuldiner, Alan R. and Rader, Daniel J. and Mirshahi, Tooraj and Marchini, Jonathan and Overton, John D. and Carey, David J. and Habegger, Lukas and Reid, Jeffrey G. and Economides, Aris and Kyratsous, Christos and Karalis, Katia and Baum, Alina and Cantor, Michael N. and Rand, Kristin A. and Hong, Eurie L. and Ball, Catherine A. and Siminovitch, Katherine and Baras, Aris and Abecasis, Goncalo R. and Ferreira, Manuel A. R.},
  doi          = {10.1038/s41588-024-01844-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1592-1596},
  shortjournal = {Nat. Genet.},
  title        = {Genetic risk factors for COVID-19 and influenza are largely distinct},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A sequence of SVA retrotransposon insertions in ASIP shaped
human pigmentation. <em>NG</em>, <em>56</em>(8), 1583–1591. (<a
href="https://doi.org/10.1038/s41588-024-01841-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Retrotransposons comprise about 45% of the human genome1, but their contributions to human trait variation and evolution are only beginning to be explored2,3. Here, we find that a sequence of SVA retrotransposon insertions in an early intron of the ASIP (agouti signaling protein) gene has probably shaped human pigmentation several times. In the UK Biobank (n = 169,641), a recent 3.3-kb SVA insertion polymorphism associated strongly with lighter skin pigmentation (0.22 [0.21–0.23] s.d.; P = 2.8 × 10−351) and increased skin cancer risk (odds ratio = 1.23 [1.18–1.27]; P = 1.3 × 10−28), appearing to underlie one of the strongest common genetic influences on these phenotypes within European populations4–6. ASIP expression in skin displayed the same association pattern, with the SVA insertion allele exhibiting 2.2-fold (1.9–2.6) increased expression. This effect had an unusual apparent mechanism: an earlier, nonpolymorphic, human-specific SVA retrotransposon 3.9 kb upstream appeared to have caused ASIP hypofunction by nonproductive splicing, which the new (polymorphic) SVA insertion largely eliminated. Extended haplotype homozygosity indicated that the insertion allele has risen to allele frequencies up to 11% in European populations over the past several thousand years. These results indicate that a sequence of retrotransposon insertions contributed to a species-wide increase, then a local decrease, of human pigmentation. Genomic analysis identifies an SVA retrotransposon insertion in an intron of ASIP as a likely causal variant influencing human pigmentation. This insertion appears to mitigate the effects of an older, nonpolymorphic SVA insertion in the same intron.},
  archive      = {J_NG},
  author       = {Kamitaki, Nolan and Hujoel, Margaux L. A. and Mukamel, Ronen E. and Gebara, Edward and McCarroll, Steven A. and Loh, Po-Ru},
  doi          = {10.1038/s41588-024-01841-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1583-1591},
  shortjournal = {Nat. Genet.},
  title        = {A sequence of SVA retrotransposon insertions in ASIP shaped human pigmentation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Rewiring gene circuitry for plant improvement. <em>NG</em>,
<em>56</em>(8), 1574–1582. (<a
href="https://doi.org/10.1038/s41588-024-01806-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aspirations for high crop growth and yield, nutritional quality and bioproduction of materials are challenged by climate change and limited adoption of new technologies. Here, we review recent advances in approaches to profile and model gene regulatory activity over developmental and response time in specific cells, which have revealed the basis of variation in plant phenotypes: both redeployment of key regulators to new contexts and their repurposing to control different slates of genes. New synthetic biology tools allow tunable, spatiotemporal regulation of transgenes, while recent gene-editing technologies enable manipulation of the regulation of native genes. Ultimately, understanding how gene circuitry is wired to control form and function across varied plant species, combined with advanced technology to rewire that circuitry, will unlock solutions to our greatest challenges in agriculture, energy and the environment. This Review discusses gene regulatory networks in plants and their function across evolution and highlights how their manipulation with synthetic biology and gene editing can promote plant productivity in the face of climate change.},
  archive      = {J_NG},
  author       = {Borowsky, Alexander T. and Bailey-Serres, Julia},
  doi          = {10.1038/s41588-024-01806-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1574-1582},
  shortjournal = {Nat. Genet.},
  title        = {Rewiring gene circuitry for plant improvement},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The ruminant telomere-to-telomere (RT2T) consortium.
<em>NG</em>, <em>56</em>(8), 1566–1573. (<a
href="https://doi.org/10.1038/s41588-024-01835-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Telomere-to-telomere (T2T) assemblies reveal new insights into the structure and function of the previously ‘invisible’ parts of the genome and allow comparative analyses of complete genomes across entire clades. We present here an open collaborative effort, termed the ‘Ruminant T2T Consortium’ (RT2T), that aims to generate complete diploid assemblies for numerous species of the Artiodactyla suborder Ruminantia to examine chromosomal evolution in the context of natural selection and domestication of species used as livestock. Here we describe an open collaborative effort termed the ‘Ruminant T2T Consortium’. It aims to generate complete diploid assemblies for many species of ruminants to examine chromosomal evolution in the context of natural selection and domestication.},
  archive      = {J_NG},
  author       = {Kalbfleisch, Theodore S. and McKay, Stephanie D. and Murdoch, Brenda M. and Adelson, David L. and Almansa-Villa, Diego and Becker, Gabrielle and Beckett, Linda M. and Benítez-Galeano, María José and Biase, Fernando and Casey, Theresa and Chuong, Edward and Clark, Emily and Clarke, Shannon and Cockett, Noelle and Couldrey, Christine and Davis, Brian W. and Elsik, Christine G. and Faraut, Thomas and Gao, Yahui and Genet, Carine and Grady, Patrick and Green, Jonathan and Green, Richard and Guan, Dailu and Hagen, Darren and Hartley, Gabrielle A. and Heaton, Mike and Hoyt, Savannah J. and Huang, Wen and Jarvis, Erich and Kalleberg, Jenna and Khatib, Hasan and Koepfi, Klaus-Peter and Koltes, James and Koren, Sergey and Kuehn, Christa and Leeb, Tosso and Leonard, Alexander and Liu, George E. and Low, Wai Yee and McConnell, Hunter and McRae, Kathryn and Miga, Karen and Mousel, Michelle and Neibergs, Holly and Olagunju, Temitayo and Pennell, Matt and Petry, Bruna and Pewsner, Mirjam and Phillippy, Adam M. and Pickett, Brandon D. and Pineda, Paulene and Potapova, Tamara and Rachagani, Satyanarayana and Rhie, Arang and Rijnkels, Monique and Robic, Annie and Rodriguez Osorio, Nelida and Safonova, Yana and Schettini, Gustavo and Schnabel, Robert D. and Sirpu Natesh, Nagabhishek and Stegemiller, Morgan and Storer, Jessica and Stothard, Paul and Stull, Caleb and Tosser-Klopp, Gwenola and Traglia, Germán M. and Tuggle, Christopher K. and Van Tassell, Curtis P. and Watson, Corey and Weikard, Rosemarie and Wimmers, Klaus and Xie, Shangqian and Yang, Liu and Smith, Timothy P. L. and O’Neill, Rachel J. and Rosen, Benjamin D.},
  doi          = {10.1038/s41588-024-01835-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1566-1573},
  shortjournal = {Nat. Genet.},
  title        = {The ruminant telomere-to-telomere (RT2T) consortium},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Establishing african genomics and bioinformatics programs
through annual regional workshops. <em>NG</em>, <em>56</em>(8),
1556–1565. (<a
href="https://doi.org/10.1038/s41588-024-01807-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The African BioGenome Project (AfricaBP) Open Institute for Genomics and Bioinformatics aims to overcome barriers to capacity building through its distributed African regional workshops and prioritizes the exchange of grassroots knowledge and innovation in biodiversity genomics and bioinformatics. In 2023, we implemented 28 workshops on biodiversity genomics and bioinformatics, covering 11 African countries across the 5 African geographical regions. These regional workshops trained 408 African scientists in hands-on molecular biology, genomics and bioinformatics techniques as well as the ethical, legal and social issues associated with acquiring genetic resources. Here, we discuss the implementation of transformative strategies, such as expanding the regional workshop model of AfricaBP to involve multiple countries, institutions and partners, including the proposed creation of an African digital database with sequence information relating to both biodiversity and agriculture. This will ultimately help create a critical mass of skilled genomics and bioinformatics scientists across Africa. The African BioGenome Project (AfricaBP) Open Institute for Genomics and Bioinformatics established a series of regional workshops in 2023 to exchange knowledge and overcome barriers, which could serve as a model for other scientific communities.},
  archive      = {J_NG},
  author       = {Sharaf, Abdoallah and Nesengani, Lucky Tendani and Hayah, Ichrak and Kuja, Josiah Ochieng and Mdyogolo, Sinebongo and Omotoriogun, Taiwo Crossby and Odogwu, Blessing Adanta and Beedessee, Girish and Smith, Rae Marvin and Barakat, Abdelhamid and Moila, Acclaim M. and El Hamouchi, Adil and Benkahla, Alia and Boukteb, Amal and Elmouhtadi, Amine and Mafwila, Antoine Lusala and Abushady, Asmaa Mohammed and Elsherif, Assem Kadry and Ahmed, Bulbul and Wairuri, Charles and Ndiribe, Charlotte C. and Ebuzome, Chukwuike and Kinnear, Craig J. and Ndlovu, Deborah-Fay and Iraqi, Driss and El Fahime, Elmostafa and Assefa, Ermias and Ouardi, Faissal and Belharfi, Fatima Zohra and Tmimi, Fatim Zohra and Markey, Fatu Badiane and Radouani, Fouzia and Zeukeng, Francis and Mvumbi, Georges Lelo and Ganesan, Hamilton and Hanachi, Mariem and Nigussie, Helen and Charoute, Hicham and Benamri, Ichrak and Mkedder, Ikram and Haddadi, Imane and Meftah-Kadmiri, Issam and Mubiru, Jackson Franco and Domelevo Entfellner, Jean-Baka Kodjo and Rokani, Joan Bayowa and Ogwang, Joel and Daiga, Jude Bigoga and Omumbo, Judy and Ideozu, Justin Eze and Errafii, Khaoula and Labuschagne, Kim and Komi, Komi Koukoura and Tonfack, Libert Brice and Hadjeras, Lydia and Ramantswana, Madeleine and Chaisi, Mamohale and Botes, Marietjie W. and Kilian, Mariëtte and Kvas, Marija and Melloul, Marouane and Chaouch, Melek and Khyatti, Meriem and Abdo, Michael and Phasha-Muchemenye, Mmatshepho and Hijri, Mohamed and Mediouni, Mohammed Rida and Hassan, Mohammed Ahmed and Piro, Mohammed and Mwale, Monica and Maaloum, Mossaab and Mavhunga, Mudzuli and Olivier, Nicholas Abraham and Aminou, Oumaima and Arbani, Oumayma and Souiai, Oussema and Djocgoue, Pierre François and Mentag, Rachid and Zipfel, Renate Dorothea and Tata, Rolland Bantar and Megnekou, Rosette and Muzemil, Sadik and Paez, Sadye and Salifu, Samson Pandam and Kagame, Samuel Paul and Selka, Sarra and Edwards, Sean and Gaouar, Semir Bechir Suheil and Reda, Shaimaa Roshdy Abdullah and Fellahi, Siham and Khayi, Slimane and Ayed, Soumia and Madisha, Thabang and Sahil, Tulsi and Udensi, Ogbuagu Ugorji and Ras, Verena and Ezebuiro, Victor and Duru, Vincent C. and David, Xavier and Geberemichael, Yonas and Tchiechoua, Yves H. and Mungloo-Dilmohamud, Zahra and Chen, Zhiliang and Happi, Christian and Kariuki, Thomas and Ziyomo, Cathrine and Djikeng, Appolinaire and Badaoui, Bouabid and Mapholi, Ntanganedzeni and Muigai, Anne and Osuji, Julian O. and Ebenezer, ThankGod Echezona},
  doi          = {10.1038/s41588-024-01807-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1556-1565},
  shortjournal = {Nat. Genet.},
  title        = {Establishing african genomics and bioinformatics programs through annual regional workshops},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unravelling the origins and forces that shape mtDNA
mutations in human cells. <em>NG</em>, <em>56</em>(8), 1554–1555. (<a
href="https://doi.org/10.1038/s41588-024-01837-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As we age, our cells acquire DNA mutations, resulting in cell-to-cell genomic heterogeneity. We characterized the landscape of mitochondrial DNA (mtDNA) heterogeneity in healthy human cells. Our observations provide deeper insight into the frequency of new mitochondrial mutations and the mechanisms that propagate low-level mutations in mtDNA over a lifetime.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01837-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1554-1555},
  shortjournal = {Nat. Genet.},
  title        = {Unravelling the origins and forces that shape mtDNA mutations in human cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). GeneMAP enables discovery of metabolic gene function.
<em>NG</em>, <em>56</em>(8), 1552–1553. (<a
href="https://doi.org/10.1038/s41588-024-01828-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying substrates of metabolic gene products is important to understand their function in physiology and disease. We developed GeneMAP, a multiomics platform for predicting metabolic gene functions using models of gene expression. We experimentally validated a top-scoring gene–metabolite association, thereby revealing a role for SLC25A48 in mitochondrial choline import.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01828-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1552-1553},
  shortjournal = {Nat. Genet.},
  title        = {GeneMAP enables discovery of metabolic gene function},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping the lung fibrosis transcriptome of mice and men.
<em>NG</em>, <em>56</em>(8), 1550–1551. (<a
href="https://doi.org/10.1038/s41588-024-01833-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Idiopathic pulmonary fibrosis (IPF), a deadly lung disease of unclear etiology, lacks sufficient therapeutic options. We extensively mapped the spatial transcriptomes of patient lungs and made translational comparisons with a mouse model of lung fibrosis, providing insights into disease mechanisms and the utility of the animal model for drug discovery.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01833-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1550-1551},
  shortjournal = {Nat. Genet.},
  title        = {Mapping the lung fibrosis transcriptome of mice and men},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell omics to unravel chronic kidney disease.
<em>NG</em>, <em>56</em>(8), 1548–1549. (<a
href="https://doi.org/10.1038/s41588-024-01804-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a multi-omic map of the human kidney in health and disease, using single-cell RNA, single-nuclei RNA, single-nuclei assay for transposase-accessible chromatin sequencing, spot-based and single-cell spatial transcriptomics. This comprehensive approach enables an in-depth understanding of the microenvironments in the diseased kidney.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01804-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1548-1549},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell omics to unravel chronic kidney disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mobile element insertions affect human pigmentation and skin
cancer risk. <em>NG</em>, <em>56</em>(8), 1546–1547. (<a
href="https://doi.org/10.1038/s41588-024-01849-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nearly 50% of the sequence of mammalian genomes is derived from mobile elements that inserted into the genome over millions of years of evolution. A recent mobile element insertion, found only in some individuals with European genetic ancestry, contributes to decreased skin pigmentation and increased sunburn frequency and skin cancer risk.},
  archive      = {J_NG},
  author       = {Kidd, Jeffrey M.},
  doi          = {10.1038/s41588-024-01849-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1546-1547},
  shortjournal = {Nat. Genet.},
  title        = {Mobile element insertions affect human pigmentation and skin cancer risk},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving estimates of loss-of-function constraint for short
genes. <em>NG</em>, <em>56</em>(8), 1544–1545. (<a
href="https://doi.org/10.1038/s41588-024-01829-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic constraint identifies genes under selection against loss-of-function, but existing methods are inaccurate for shorter genes. A new study overcomes this key limitation to ascribe more confident predictions to all human protein-coding genes.},
  archive      = {J_NG},
  author       = {Whiffin, Nicola},
  doi          = {10.1038/s41588-024-01829-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1544-1545},
  shortjournal = {Nat. Genet.},
  title        = {Improving estimates of loss-of-function constraint for short genes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell technologies meet hi-c. <em>NG</em>,
<em>56</em>(8), 1542–1543. (<a
href="https://doi.org/10.1038/s41588-024-01855-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new single-cell multiomic technique, GAGE-seq, pushes the frontier of single-cell 4D genome analysis by enabling analysis of rare cell populations in complex tissues.},
  archive      = {J_NG},
  author       = {Hughes, Jim R. and Davies, James O. J.},
  doi          = {10.1038/s41588-024-01855-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1542-1543},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell technologies meet hi-C},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). G6PD deficiency and diabetes complications. <em>NG</em>,
<em>56</em>(8), 1541. (<a
href="https://doi.org/10.1038/s41588-024-01881-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-024-01881-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1541},
  shortjournal = {Nat. Genet.},
  title        = {G6PD deficiency and diabetes complications},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). RNU4-2 variants cause neurodevelopmental disorders.
<em>NG</em>, <em>56</em>(8), 1541. (<a
href="https://doi.org/10.1038/s41588-024-01882-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-024-01882-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1541},
  shortjournal = {Nat. Genet.},
  title        = {RNU4-2 variants cause neurodevelopmental disorders},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Silencing prion protein expression in the brain.
<em>NG</em>, <em>56</em>(8), 1541. (<a
href="https://doi.org/10.1038/s41588-024-01888-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-024-01888-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1541},
  shortjournal = {Nat. Genet.},
  title        = {Silencing prion protein expression in the brain},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Chromosome organization from the ice age. <em>NG</em>,
<em>56</em>(8), 1541. (<a
href="https://doi.org/10.1038/s41588-024-01889-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-024-01889-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1541},
  shortjournal = {Nat. Genet.},
  title        = {Chromosome organization from the ice age},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Large-scale genotype prediction from RNA sequence data
necessitates a new ethical and policy framework. <em>NG</em>,
<em>56</em>(8), 1537–1540. (<a
href="https://doi.org/10.1038/s41588-024-01825-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genotype prediction from RNA sequencing (RNA-seq) data has become widespread, but there is a lack of clarity in current policy and inconsistency in data handling. To address this we call for a framework consisting of registered access for RNA-seq data, controlled access for genotypes, a code of conduct and enhanced downstream protections.},
  archive      = {J_NG},
  author       = {Majumder, Mary A. and Leek, Jeffrey T. and Hansen, Kasper D. and Razi, Afrooz and McGuire, Amy L.},
  doi          = {10.1038/s41588-024-01825-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1537-1540},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale genotype prediction from RNA sequence data necessitates a new ethical and policy framework},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Synthetic surrogates improve power for genome-wide
association studies of partially missing phenotypes in population
biobanks. <em>NG</em>, <em>56</em>(7), 1527–1536. (<a
href="https://doi.org/10.1038/s41588-024-01793-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Within population biobanks, incomplete measurement of certain traits limits the power for genetic discovery. Machine learning is increasingly used to impute the missing values from the available data. However, performing genome-wide association studies (GWAS) on imputed traits can introduce spurious associations, identifying genetic variants that are not associated with the original trait. Here we introduce a new method, synthetic surrogate (SynSurr) analysis, which makes GWAS on imputed phenotypes robust to imputation errors. Rather than replacing missing values, SynSurr jointly analyzes the original and imputed traits. We show that SynSurr estimates the same genetic effect as standard GWAS and improves power in proportion to the quality of the imputations. SynSurr requires a commonly made missing-at-random assumption but relaxes the requirements of existing imputation methods by not requiring correct model specification. We present extensive simulations and ablation analyses to validate SynSurr and apply it to empower the GWAS of dual-energy X-ray absorptiometry traits within the UK Biobank. This study presents a synthetic surrogate (SynSurr) method for imputing missing phenotypes in biobank datasets. Joint analysis of the partially observed and imputed surrogate phenotype improves power in genome-wide association studies while being robust to imputation errors.},
  archive      = {J_NG},
  author       = {McCaw, Zachary R. and Gao, Jianhui and Lin, Xihong and Gronsbell, Jessica},
  doi          = {10.1038/s41588-024-01793-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1527-1536},
  shortjournal = {Nat. Genet.},
  title        = {Synthetic surrogates improve power for genome-wide association studies of partially missing phenotypes in population biobanks},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The elite haplotype OsGATA8-h coordinates nitrogen uptake
and productive tiller formation in rice. <em>NG</em>, <em>56</em>(7),
1516–1526. (<a
href="https://doi.org/10.1038/s41588-024-01795-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Excessive nitrogen promotes the formation of nonproductive tillers in rice, which decreases nitrogen use efficiency (NUE). Developing high-NUE rice cultivars through balancing nitrogen uptake and the formation of productive tillers remains a long-standing challenge, yet how these two processes are coordinated in rice remains elusive. Here we identify the transcription factor OsGATA8 as a key coordinator of nitrogen uptake and tiller formation in rice. OsGATA8 negatively regulates nitrogen uptake by repressing transcription of the ammonium transporter gene OsAMT3.2. Meanwhile, it promotes tiller formation by repressing the transcription of OsTCP19, a negative modulator of tillering. We identify OsGATA8-H as a high-NUE haplotype with enhanced nitrogen uptake and a higher proportion of productive tillers. The geographical distribution of OsGATA8-H and its frequency change in historical accessions suggest its adaption to the fertile soil. Overall, this study provides molecular and evolutionary insights into the regulation of NUE and facilitates the breeding of rice cultivars with higher NUE. The study identifies transcription factor OsGATA8 as a negative regulator of nitrogen uptake and tiller formation in rice and OsGATA8-H as an elite haplotype with reduced expression and enhanced nitrogen use efficiency.},
  archive      = {J_NG},
  author       = {Wu, Wei and Dong, Xiaoou and Chen, Gaoming and Lin, Zhixi and Chi, Wenchao and Tang, Weijie and Yu, Jun and Wang, Saisai and Jiang, Xingzhou and Liu, Xiaolan and Wu, Yujun and Wang, Chunyuan and Cheng, Xinran and Zhang, Wei and Xuan, Wei and Terzaghi, William and Ronald, Pamela C. and Wang, Haiyang and Wang, Chunming and Wan, Jianmin},
  doi          = {10.1038/s41588-024-01795-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1516-1526},
  shortjournal = {Nat. Genet.},
  title        = {The elite haplotype OsGATA8-H coordinates nitrogen uptake and productive tiller formation in rice},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Complexin-1 enhances ultrasound neurotransmission in the
mammalian auditory pathway. <em>NG</em>, <em>56</em>(7), 1503–1515. (<a
href="https://doi.org/10.1038/s41588-024-01781-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike megabats, which rely on well-developed vision, microbats use ultrasonic echolocation to navigate and locate prey. To study ultrasound perception, here we compared the auditory cortices of microbats and megabats by constructing reference genomes and single-nucleus atlases for four species. We found that parvalbumin (PV)+ neurons exhibited evident cross-species differences and could respond to ultrasound signals, whereas their silencing severely affected ultrasound perception in the mouse auditory cortex. Moreover, megabat PV+ neurons expressed low levels of complexins (CPLX1–CPLX4), which can facilitate neurotransmitter release, while microbat PV+ neurons highly expressed CPLX1, which improves neurotransmission efficiency. Further perturbation of Cplx1 in PV+ neurons impaired ultrasound perception in the mouse auditory cortex. In addition, CPLX1 functioned in other parts of the auditory pathway in microbats but not megabats and exhibited convergent evolution between echolocating microbats and whales. Altogether, we conclude that CPLX1 expression throughout the entire auditory pathway can enhance mammalian ultrasound neurotransmission. High-quality genomes and single-cell atlases of auditory cortices from microbat and megabat species identify neuronal populations related to ultrasound perception and implicate complexin-1 as a key component of ultrasound transmission in mammals.},
  archive      = {J_NG},
  author       = {Liu, Meiling and Wang, Changliang and Huo, Lifang and Cao, Jie and Mao, Xiuguang and He, Ziqing and Hu, Chuanxia and Sun, Haijian and Deng, Wenjun and He, Weiya and Chen, Yifu and Gu, Meifeng and Liao, Jiayu and Guo, Ning and He, Xiangyang and Wu, Qian and Chen, Jiekai and Zhang, Libiao and Wang, Xiaoqun and Shang, Congping and Dong, Ji},
  doi          = {10.1038/s41588-024-01781-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1503-1515},
  shortjournal = {Nat. Genet.},
  title        = {Complexin-1 enhances ultrasound neurotransmission in the mammalian auditory pathway},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). LINE-1 transcription activates long-range gene expression.
<em>NG</em>, <em>56</em>(7), 1494–1502. (<a
href="https://doi.org/10.1038/s41588-024-01789-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long interspersed nuclear element-1 (LINE-1 or L1) is a retrotransposon group that constitutes 17% of the human genome and shows variable expression across cell types. However, the control of L1 expression and its function in gene regulation are incompletely understood. Here we show that L1 transcription activates long-range gene expression. Genome-wide CRISPR–Cas9 screening using a reporter driven by the L1 5′ UTR in human cells identifies functionally diverse genes affecting L1 expression. Unexpectedly, altering L1 expression by knockout of regulatory genes impacts distant gene expression. L1s can physically contact their distal target genes, with these interactions becoming stronger upon L1 activation and weaker when L1 is silenced. Remarkably, L1s contact and activate genes essential for zygotic genome activation (ZGA), and L1 knockdown impairs ZGA, leading to developmental arrest in mouse embryos. These results characterize the regulation and function of L1 in long-range gene activation and reveal its importance in mammalian ZGA. A CRISPR–Cas9 screen identifies genes that modulate long interspersed nuclear element-1 (LINE-1) expression in human cells. LINE-1 5′ UTRs have enhancer features and can activate long-range gene expression, including during zygotic genome activation.},
  archive      = {J_NG},
  author       = {Li, Xiufeng and Bie, Luyao and Wang, Yang and Hong, Yaqiang and Zhou, Ziqiang and Fan, Yiming and Yan, Xiaohan and Tao, Yibing and Huang, Chunyi and Zhang, Yongyan and Sun, Xueyan and Li, John Xiao He and Zhang, Jing and Chang, Zai and Xi, Qiaoran and Meng, Anming and Shen, Xiaohua and Xie, Wei and Liu, Nian},
  doi          = {10.1038/s41588-024-01789-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1494-1502},
  shortjournal = {Nat. Genet.},
  title        = {LINE-1 transcription activates long-range gene expression},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-resolution genome-wide mapping of chromosome-arm-scale
truncations induced by CRISPR–Cas9 editing. <em>NG</em>, <em>56</em>(7),
1482–1493. (<a
href="https://doi.org/10.1038/s41588-024-01758-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clustered regularly interspaced short palindromic repeats (CRISPR)–CRISPR-associated protein 9 (Cas9) is a powerful tool for introducing targeted mutations in DNA, but recent studies have shown that it can have unintended effects such as structural changes. However, these studies have not yet looked genome wide or across data types. Here we performed a phenotypic CRISPR–Cas9 scan targeting 17,065 genes in primary human cells, revealing a ‘proximity bias’ in which CRISPR knockouts show unexpected similarities to unrelated genes on the same chromosome arm. This bias was found to be consistent across cell types, laboratories, Cas9 delivery methods and assay modalities, and the data suggest that it is caused by telomeric truncations of chromosome arms, with cell cycle and apoptotic pathways playing a mediating role. Additionally, a simple correction is demonstrated to mitigate this pervasive bias while preserving biological relationships. This previously uncharacterized effect has implications for functional genomic studies using CRISPR–Cas9, with applications in discovery biology, drug-target identification, cell therapies and genetic therapeutics. This study finds that CRISPR-knockout phenotypes from genome-wide screens systematically show increased similarity to knockouts of unrelated genomically proximal genes located on the same chromosome arm. Multiple lines of evidence suggest that this proximity bias is caused by telomeric truncations of chromosome arms and is consistent across cell types, labs and Cas9 delivery methods.},
  archive      = {J_NG},
  author       = {Lazar, Nathan H. and Celik, Safiye and Chen, Lu and Fay, Marta M. and Irish, Jonathan C. and Jensen, James and Tillinghast, Conor A. and Urbanik, John and Bone, William P. and Gibson, Christopher C. and Haque, Imran S.},
  doi          = {10.1038/s41588-024-01758-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1482-1493},
  shortjournal = {Nat. Genet.},
  title        = {High-resolution genome-wide mapping of chromosome-arm-scale truncations induced by CRISPR–Cas9 editing},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A single-cell transcriptome atlas of human euploid and
aneuploid blastocysts. <em>NG</em>, <em>56</em>(7), 1468–1481. (<a
href="https://doi.org/10.1038/s41588-024-01788-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aneuploidy is frequently detected in early human embryos as a major cause of early pregnancy failure. However, how aneuploidy affects cellular function remains elusive. Here, we profiled the transcriptomes of 14,908 single cells from 203 human euploid and aneuploid blastocysts involving autosomal and sex chromosomes. Nearly all of the blastocysts contained four lineages. In aneuploid chromosomes, 19.5% ± 1.2% of the expressed genes showed a dosage effect, and 90 dosage-sensitive domains were identified. Aneuploidy leads to prevalent genome-wide transcriptome alterations. Common effects, including apoptosis, were identified, especially in monosomies, partially explaining the lower cell numbers in autosomal monosomies. We further identified lineage-specific effects causing unstable epiblast development in aneuploidies, which was accompanied by the downregulation of TGF-β and FGF signaling, which resulted in insufficient trophectoderm maturation. Our work provides crucial insights into the molecular basis of human aneuploid blastocysts and may shed light on the cellular interaction during blastocyst development. Single-cell transcriptome profiling of human aneuploid blastocysts highlights global transcriptomic alteration and identifies novel dosage-sensitive genes in aneuploidy. Aneuploid embryos undergo unstable epiblast maturation caused by defects in the TGF-β and FGF pathways that impact trophectoderm progression.},
  archive      = {J_NG},
  author       = {Wang, Shengpeng and Leng, Lizhi and Wang, Quanlei and Gu, Yifan and Li, Jinlu and An, Yanru and Deng, Qiuting and Xie, Pingyuan and Cheng, Can and Chen, Xueqin and Zhou, Qinwei and Lu, Jia and Chen, Fang and Liu, Longqi and Yang, Huanming and Wang, Jian and Xu, Xun and Hou, Yong and Gong, Fei and Hu, Liang and Lu, Guangxiu and Shang, Zhouchun and Lin, Ge},
  doi          = {10.1038/s41588-024-01788-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1468-1481},
  shortjournal = {Nat. Genet.},
  title        = {A single-cell transcriptome atlas of human euploid and aneuploid blastocysts},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Non-stem cell lineages as an alternative origin of
intestinal tumorigenesis in the context of inflammation. <em>NG</em>,
<em>56</em>(7), 1456–1467. (<a
href="https://doi.org/10.1038/s41588-024-01801-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {According to conventional views, colon cancer originates from stem cells. However, inflammation, a key risk factor for colon cancer, has been shown to suppress intestinal stemness. Here, we used Paneth cells as a model to assess the capacity of differentiated lineages to trigger tumorigenesis in the context of inflammation in mice. Upon inflammation, Paneth cell-specific Apc mutations led to intestinal tumors reminiscent not only of those arising in patients with inflammatory bowel disease, but also of a larger fraction of human sporadic colon cancers. The latter is possibly because of the inflammatory consequences of western-style dietary habits, a major colon cancer risk factor. Machine learning methods designed to predict the cell-of-origin of cancer from patient-derived tumor samples confirmed that, in a substantial fraction of sporadic cases, the origins of colon cancer reside in secretory lineages and not in stem cells. Upon inflammation and targeted gene mutation, some fully differentiated secretory and postmitotic intestinal epithelial lineages dedifferentiate to acquire stem-like features and promote tumor formation.},
  archive      = {J_NG},
  author       = {Verhagen, Mathijs P. and Joosten, Rosalie and Schmitt, Mark and Välimäki, Niko and Sacchetti, Andrea and Rajamäki, Kristiina and Choi, Jiahn and Procopio, Paola and Silva, Sara and van der Steen, Berdine and van den Bosch, Thierry P. P. and Seinstra, Danielle and de Vries, Annemarie C. and Doukas, Michail and Augenlicht, Leonard H. and Aaltonen, Lauri A. and Fodde, Riccardo},
  doi          = {10.1038/s41588-024-01801-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1456-1467},
  shortjournal = {Nat. Genet.},
  title        = {Non-stem cell lineages as an alternative origin of intestinal tumorigenesis in the context of inflammation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Saturation genome editing maps the functional spectrum of
pathogenic VHL alleles. <em>NG</em>, <em>56</em>(7), 1446–1455. (<a
href="https://doi.org/10.1038/s41588-024-01800-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To maximize the impact of precision medicine approaches, it is critical to identify genetic variants underlying disease and to accurately quantify their functional effects. A gene exemplifying the challenge of variant interpretation is the von Hippel–Lindautumor suppressor (VHL). VHL encodes an E3 ubiquitin ligase that regulates the cellular response to hypoxia. Germline pathogenic variants in VHL predispose patients to tumors including clear cell renal cell carcinoma (ccRCC) and pheochromocytoma, and somatic VHL mutations are frequently observed in sporadic renal cancer. Here we optimize and apply saturation genome editing to assay nearly all possible single-nucleotide variants (SNVs) across VHL’s coding sequence. To delineate mechanisms, we quantify mRNA dosage effects and compare functional effects in isogenic cell lines. Function scores for 2,268 VHL SNVs identify a core set of pathogenic alleles driving ccRCC with perfect accuracy, inform differential risk across tumor types and reveal new mechanisms by which variants impact function. These results have immediate utility for classifying VHL variants encountered clinically and illustrate how precise functional measurements can resolve pleiotropic and dosage-dependent genotype–phenotype relationships across complete genes. Saturation genome editing characterizes von Hippel–Lindau (VHL) coding variants and their associations with diseases. Function scores for 2,268 VHL single-nucleotide variants (SNVs) classify pathogenic alleles driving renal cell carcinoma and suggest new mechanisms by which variants impact function.},
  archive      = {J_NG},
  author       = {Buckley, Megan and Terwagne, Chloé and Ganner, Athina and Cubitt, Laura and Brewer, Reid and Kim, Dong-Kyu and Kajba, Christina M. and Forrester, Nicole and Dace, Phoebe and De Jonghe, Joachim and Shepherd, Scott T. C. and Sawyer, Chelsea and McEwen, Mairead and Diederichs, Sven and Neumann-Haefelin, Elke and Turajlic, Samra and Ivakine, Evgueni A. and Findlay, Gregory M.},
  doi          = {10.1038/s41588-024-01800-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1446-1455},
  shortjournal = {Nat. Genet.},
  title        = {Saturation genome editing maps the functional spectrum of pathogenic VHL alleles},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Saturation genome editing of BAP1 functionally classifies
somatic and germline variants. <em>NG</em>, <em>56</em>(7), 1434–1445.
(<a href="https://doi.org/10.1038/s41588-024-01799-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many variants that we inherit from our parents or acquire de novo or somatically are rare, limiting the precision with which we can associate them with disease. We performed exhaustive saturation genome editing (SGE) of BAP1, the disruption of which is linked to tumorigenesis and altered neurodevelopment. We experimentally characterized 18,108 unique variants, of which 6,196 were found to have abnormal functions, and then used these data to evaluate phenotypic associations in the UK Biobank. We also characterized variants in a large population-ascertained tumor collection, in cancer pedigrees and ClinVar, and explored the behavior of cancer-associated variants compared to that of variants linked to neurodevelopmental phenotypes. Our analyses demonstrated that disruptive germline BAP1 variants were significantly associated with higher circulating levels of the mitogen IGF-1, suggesting a possible pathological mechanism and therapeutic target. Furthermore, we built a variant classifier with &amp;gt;98% sensitivity and specificity and quantify evidence strengths to aid precision variant interpretation. Saturation genome editing characterizes BAP1 variants and their association with disease presentation. A phenome-wide association analysis in the UK finds that BAP1 variants identified as deleterious in the study are associated with higher serum IGF-1 levels.},
  archive      = {J_NG},
  author       = {Waters, Andrew J. and Brendler-Spaeth, Timothy and Smith, Danielle and Offord, Victoria and Tan, Hong Kee and Zhao, Yajie and Obolenski, Sofia and Nielsen, Maartje and van Doorn, Remco and Murphy, Jo-Ellen and Gupta, Prashant and Rowlands, Charlie F. and Hanson, Helen and Delage, Erwan and Thomas, Mark and Radford, Elizabeth J. and Gerety, Sebastian S. and Turnbull, Clare and Perry, John R. B. and Hurles, Matthew E. and Adams, David J.},
  doi          = {10.1038/s41588-024-01799-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1434-1445},
  shortjournal = {Nat. Genet.},
  title        = {Saturation genome editing of BAP1 functionally classifies somatic and germline variants},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Homopolymer switches mediate adaptive mutability in mismatch
repair-deficient colorectal cancer. <em>NG</em>, <em>56</em>(7),
1420–1433. (<a
href="https://doi.org/10.1038/s41588-024-01777-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mismatch repair (MMR)-deficient cancer evolves through the stepwise erosion of coding homopolymers in target genes. Curiously, the MMR genes MutS homolog 6 (MSH6) and MutS homolog 3 (MSH3) also contain coding homopolymers, and these are frequent mutational targets in MMR-deficient cancers. The impact of incremental MMR mutations on MMR-deficient cancer evolution is unknown. Here we show that microsatellite instability modulates DNA repair by toggling hypermutable mononucleotide homopolymer runs in MSH6 and MSH3 through stochastic frameshift switching. Spontaneous mutation and reversion modulate subclonal mutation rate, mutation bias and HLA and neoantigen diversity. Patient-derived organoids corroborate these observations and show that MMR homopolymer sequences drift back into reading frame in the absence of immune selection, suggesting a fitness cost of elevated mutation rates. Combined experimental and simulation studies demonstrate that subclonal immune selection favors incremental MMR mutations. Overall, our data demonstrate that MMR-deficient colorectal cancers fuel intratumor heterogeneity by adapting subclonal mutation rate and diversity to immune selection. Mismatch repair-deficient colorectal cancer clones adapt their mutation landscape by toggling homopolymer sequences in MutS homolog 3 (MSH3) and MutS homolog 6 (MSH6). This increases the subclonal mutation rate and clonal diversity, favoring immune escape and tumor growth.},
  archive      = {J_NG},
  author       = {Kayhanian, Hamzeh and Cross, William and van der Horst, Suzanne E. M. and Barmpoutis, Panagiotis and Lakatos, Eszter and Caravagna, Giulio and Zapata, Luis and Van Hoeck, Arne and Middelkamp, Sjors and Litchfield, Kevin and Steele, Christopher and Waddingham, William and Patel, Dominic and Milite, Salvatore and Jin, Chen and Baker, Ann-Marie and Alexander, Daniel C. and Khan, Khurum and Hochhauser, Daniel and Novelli, Marco and Werner, Benjamin and van Boxtel, Ruben and Hageman, Joris H. and Buissant des Amorie, Julian R. and Linares, Josep and Ligtenberg, Marjolijn J. L. and Nagtegaal, Iris D. and Laclé, Miangela M. and Moons, Leon M. G. and Brosens, Lodewijk A. A. and Pillay, Nischalan and Sottoriva, Andrea and Graham, Trevor A. and Rodriguez-Justo, Manuel and Shiu, Kai-Keen and Snippert, Hugo J. G. and Jansen, Marnix},
  doi          = {10.1038/s41588-024-01777-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1420-1433},
  shortjournal = {Nat. Genet.},
  title        = {Homopolymer switches mediate adaptive mutability in mismatch repair-deficient colorectal cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Exome sequence analysis identifies rare coding variants
associated with a machine learning-based marker for coronary artery
disease. <em>NG</em>, <em>56</em>(7), 1412–1419. (<a
href="https://doi.org/10.1038/s41588-024-01791-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coronary artery disease (CAD) exists on a spectrum of disease represented by a combination of risk factors and pathogenic processes. An in silico score for CAD built using machine learning and clinical data in electronic health records captures disease progression, severity and underdiagnosis on this spectrum and could enhance genetic discovery efforts for CAD. Here we tested associations of rare and ultrarare coding variants with the in silico score for CAD in the UK Biobank, All of Us Research Program and BioMe Biobank. We identified associations in 17 genes; of these, 14 show at least moderate levels of prior genetic, biological and/or clinical support for CAD. We also observed an excess of ultrarare coding variants in 321 aggregated CAD genes, suggesting more ultrarare variant associations await discovery. These results expand our understanding of the genetic etiology of CAD and illustrate how digital markers can enhance genetic association investigations for complex diseases. A machine learning-based, continuous in silico coronary artery disease (CAD) score built using electronic health record data is applied to rare variant association analysis of CAD, implicating novel candidate genes and biological mechanisms.},
  archive      = {J_NG},
  author       = {Petrazzini, Ben Omega and Forrest, Iain S. and Rocheleau, Ghislain and Vy, Ha My T. and Márquez-Luna, Carla and Duffy, Áine and Chen, Robert and Park, Joshua K. and Gibson, Kyle and Goonewardena, Sascha N. and Malick, Waqas A. and Rosenson, Robert S. and Jordan, Daniel M. and Do, Ron},
  doi          = {10.1038/s41588-024-01791-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1412-1419},
  shortjournal = {Nat. Genet.},
  title        = {Exome sequence analysis identifies rare coding variants associated with a machine learning-based marker for coronary artery disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Understanding the genetic complexity of puberty timing
across the allele frequency spectrum. <em>NG</em>, <em>56</em>(7),
1397–1411. (<a
href="https://doi.org/10.1038/s41588-024-01798-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pubertal timing varies considerably and is associated with later health outcomes. We performed multi-ancestry genetic analyses on ~800,000 women, identifying 1,080 signals for age at menarche. Collectively, these explained 11% of trait variance in an independent sample. Women at the top and bottom 1% of polygenic risk exhibited ~11 and ~14-fold higher risks of delayed and precocious puberty, respectively. We identified several genes harboring rare loss-of-function variants in ~200,000 women, including variants in ZNF483, which abolished the impact of polygenic risk. Variant-to-gene mapping approaches and mouse gonadotropin-releasing hormone neuron RNA sequencing implicated 665 genes, including an uncharacterized G-protein-coupled receptor, GPR83, which amplified the signaling of MC3R, a key nutritional sensor. Shared signals with menopause timing at genes involved in DNA damage response suggest that the ovarian reserve might signal centrally to trigger puberty. We also highlight body size-dependent and independent mechanisms that potentially link reproductive timing to later life disease. A multi-ancestry genome-wide association study for age at menarche followed by fine mapping and downstream analysis implicates 665 pubertal timing genes, such as the G-protein-coupled receptor 83 (GPR83) and other genes expressed in the ovaries involved in the DNA damage response.},
  archive      = {J_NG},
  author       = {Kentistou, Katherine A. and Kaisinger, Lena R. and Stankovic, Stasa and Vaudel, Marc and Mendes de Oliveira, Edson and Messina, Andrea and Walters, Robin G. and Liu, Xiaoxi and Busch, Alexander S. and Helgason, Hannes and Thompson, Deborah J. and Santoni, Federico and Petricek, Konstantin M. and Zouaghi, Yassine and Huang-Doran, Isabel and Gudbjartsson, Daniel F. and Bratland, Eirik and Lin, Kuang and Gardner, Eugene J. and Zhao, Yajie and Jia, Raina Y. and Terao, Chikashi and Riggan, Marjorie J. and Bolla, Manjeet K. and Yazdanpanah, Mojgan and Yazdanpanah, Nahid and Bradfield, Jonathan P. and Broer, Linda and Campbell, Archie and Chasman, Daniel I. and Cousminer, Diana L. and Franceschini, Nora and Franke, Lude H. and Girotto, Giorgia and He, Chunyan and Järvelin, Marjo-Riitta and Joshi, Peter K. and Kamatani, Yoichiro and Karlsson, Robert and Luan, Jian’an and Lunetta, Kathryn L. and Mägi, Reedik and Mangino, Massimo and Medland, Sarah E. and Meisinger, Christa and Noordam, Raymond and Nutile, Teresa and Concas, Maria Pina and Polašek, Ozren and Porcu, Eleonora and Ring, Susan M. and Sala, Cinzia and Smith, Albert V. and Tanaka, Toshiko and van der Most, Peter J. and Vitart, Veronique and Wang, Carol A. and Willemsen, Gonneke and Zygmunt, Marek and Ahearn, Thomas U. and Andrulis, Irene L. and Anton-Culver, Hoda and Antoniou, Antonis C. and Auer, Paul L. and Barnes, Catriona L. K. and Beckmann, Matthias W. and Berrington de Gonzalez, Amy and Bogdanova, Natalia V. and Bojesen, Stig E. and Brenner, Hermann and Buring, Julie E. and Canzian, Federico and Chang-Claude, Jenny and Couch, Fergus J. and Cox, Angela and Crisponi, Laura and Czene, Kamila and Daly, Mary B. and Demerath, Ellen W. and Dennis, Joe and Devilee, Peter and De Vivo, Immaculata and Dörk, Thilo and Dunning, Alison M. and Dwek, Miriam and Eriksson, Johan G. and Fasching, Peter A. and Fernandez-Rhodes, Lindsay and Ferreli, Liana and Fletcher, Olivia and Gago-Dominguez, Manuela and García-Closas, Montserrat and García-Sáenz, José A. and González-Neira, Anna and Grallert, Harald and Guénel, Pascal and Haiman, Christopher A. and Hall, Per and Hamann, Ute and Hakonarson, Hakon and Hart, Roger J. and Hickey, Martha and Hooning, Maartje J. and Hoppe, Reiner and Hopper, John L. and Hottenga, Jouke-Jan and Hu, Frank B. and Huebner, Hanna and Hunter, David J. and Jernström, Helena and John, Esther M. and Karasik, David and Khusnutdinova, Elza K. and Kristensen, Vessela N. and Lacey, James V. and Lambrechts, Diether and Launer, Lenore J. and Lind, Penelope A. and Lindblom, Annika and Magnusson, Patrik K. E. and Mannermaa, Arto and McCarthy, Mark I. and Meitinger, Thomas and Menni, Cristina and Michailidou, Kyriaki and Millwood, Iona Y. and Milne, Roger L. and Montgomery, Grant W. and Nevanlinna, Heli and Nolte, Ilja M. and Nyholt, Dale R. and Obi, Nadia and O’Brien, Katie M. and Offit, Kenneth and Oldehinkel, Albertine J. and Ostrowski, Sisse R. and Palotie, Aarno and Pedersen, Ole B. and Peters, Annette and Pianigiani, Giulia and Plaseska-Karanfilska, Dijana and Pouta, Anneli and Pozarickij, Alfred and Radice, Paolo and Rennert, Gad and Rosendaal, Frits R. and Ruggiero, Daniela and Saloustros, Emmanouil and Sandler, Dale P. and Schipf, Sabine and Schmidt, Carsten O. and Schmidt, Marjanka K. and Small, Kerrin and Spedicati, Beatrice and Stampfer, Meir and Stone, Jennifer and Tamimi, Rulla M. and Teras, Lauren R. and Tikkanen, Emmi and Turman, Constance and Vachon, Celine M. and Wang, Qin and Winqvist, Robert and Wolk, Alicja and Zemel, Babette S. and Zheng, Wei and van Dijk, Ko W. and Alizadeh, Behrooz Z. and Bandinelli, Stefania and Boerwinkle, Eric and Boomsma, Dorret I. and Ciullo, Marina and Chenevix-Trench, Georgia and Cucca, Francesco and Esko, Tõnu and Gieger, Christian and Grant, Struan F. A. and Gudnason, Vilmundur and Hayward, Caroline and Kolčić, Ivana and Kraft, Peter and Lawlor, Deborah A. and Martin, Nicholas G. and Nøhr, Ellen A. and Pedersen, Nancy L. and Pennell, Craig E. and Ridker, Paul M. and Robino, Antonietta and Snieder, Harold and Sovio, Ulla and Spector, Tim D. and Stöckl, Doris and Sudlow, Cathie and Timpson, Nic J. and Toniolo, Daniela and Uitterlinden, André and Ulivi, Sheila and Völzke, Henry and Wareham, Nicholas J. and Widen, Elisabeth and Wilson, James F. and Pharoah, Paul D. P. and Li, Liming and Easton, Douglas F. and Njølstad, Pål R. and Sulem, Patrick and Murabito, Joanne M. and Murray, Anna and Manousaki, Despoina and Juul, Anders and Erikstrup, Christian and Stefansson, Kari and Horikoshi, Momoko and Chen, Zhengming and Farooqi, I. Sadaf and Pitteloud, Nelly and Johansson, Stefan and Day, Felix R. and Perry, John R. B. and Ong, Ken K.},
  doi          = {10.1038/s41588-024-01798-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1397-1411},
  shortjournal = {Nat. Genet.},
  title        = {Understanding the genetic complexity of puberty timing across the allele frequency spectrum},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Calibrated prediction intervals for polygenic scores across
diverse contexts. <em>NG</em>, <em>56</em>(7), 1386–1396. (<a
href="https://doi.org/10.1038/s41588-024-01792-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polygenic scores (PGS) have emerged as the tool of choice for genomic prediction in a wide range of fields. We show that PGS performance varies broadly across contexts and biobanks. Contexts such as age, sex and income can impact PGS accuracy with similar magnitudes as genetic ancestry. Here we introduce an approach (CalPred) that models all contexts jointly to produce prediction intervals that vary across contexts to achieve calibration (include the trait with 90% probability), whereas existing methods are miscalibrated. In analyses of 72 traits across large and diverse biobanks (All of Us and UK Biobank), we find that prediction intervals required adjustment by up to 80% for quantitative traits. For disease traits, PGS-based predictions were miscalibrated across socioeconomic contexts such as annual household income levels, further highlighting the need of accounting for context information in PGS-based prediction across diverse populations. CalPred is a framework that adjusts polygenic score (PGS) prediction intervals based on joint modeling of multiple contexts, such as age, sex and genetic ancestry. PGS show pervasive context-specific accuracy, suggesting that accounting for this will improve portability across contexts.},
  archive      = {J_NG},
  author       = {Hou, Kangcheng and Xu, Ziqi and Ding, Yi and Mandla, Ravi and Shi, Zhuozheng and Boulier, Kristin and Harpak, Arbel and Pasaniuc, Bogdan},
  doi          = {10.1038/s41588-024-01792-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1386-1396},
  shortjournal = {Nat. Genet.},
  title        = {Calibrated prediction intervals for polygenic scores across diverse contexts},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Interaction between MED12 and ΔNp63 activates basal identity
in pancreatic ductal adenocarcinoma. <em>NG</em>, <em>56</em>(7),
1377–1385. (<a
href="https://doi.org/10.1038/s41588-024-01790-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The presence of basal lineage characteristics signifies hyperaggressive human adenocarcinomas of the breast, bladder and pancreas. However, the biochemical mechanisms that maintain this aberrant cell state are poorly understood. Here we performed marker-based genetic screens in search of factors needed to maintain basal identity in pancreatic ductal adenocarcinoma (PDAC). This approach revealed MED12 as a powerful regulator of the basal cell state in this disease. Using biochemical reconstitution and epigenomics, we show that MED12 carries out this function by bridging the transcription factor ΔNp63, a known master regulator of the basal lineage, with the Mediator complex to activate lineage-specific enhancer elements. Consistent with this finding, the growth of basal-like PDAC is hypersensitive to MED12 loss when compared to PDAC cells lacking basal characteristics. Taken together, our genetic screens have revealed a biochemical interaction that sustains basal identity in human cancer, which could serve as a target for tumor lineage-directed therapeutics. Marker-based CRISPR screens in pancreatic cancer cells followed by functional validation highlight a role for MED12 in bridging ΔNp63 and components of the Mediator family. This interaction helps drive basal cell identity in pancreatic ductal adenocarcinoma.},
  archive      = {J_NG},
  author       = {Maia-Silva, Diogo and Cunniff, Patrick J. and Schier, Allison C. and Skopelitis, Damianos and Trousdell, Marygrace C. and Moresco, Philip and Gao, Yuan and Kechejian, Vahag and He, Xue-Yan and Sahin, Yunus and Wan, Ledong and Alpsoy, Aktan and Liverpool, Jynelle and Krainer, Adrian R. and Egeblad, Mikala and Spector, David L. and Fearon, Douglas T. and dos Santos, Camila O. and Taatjes, Dylan J. and Vakoc, Christopher R.},
  doi          = {10.1038/s41588-024-01790-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1377-1385},
  shortjournal = {Nat. Genet.},
  title        = {Interaction between MED12 and ΔNp63 activates basal identity in pancreatic ductal adenocarcinoma},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic rare variant analyses identify RAB32 as a
susceptibility gene for familial parkinson’s disease. <em>NG</em>,
<em>56</em>(7), 1371–1376. (<a
href="https://doi.org/10.1038/s41588-024-01787-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite substantial progress, causal variants are identified only for a minority of familial Parkinson’s disease (PD) cases, leaving high-risk pathogenic variants unidentified1,2. To identify such variants, we uniformly processed exome sequencing data of 2,184 index familial PD cases and 69,775 controls. Exome-wide analyses converged on RAB32 as a novel PD gene identifying c.213C &amp;gt; G/p.S71R as a high-risk variant presenting in ~0.7% of familial PD cases while observed in only 0.004% of controls (odds ratio of 65.5). This variant was confirmed in all cases via Sanger sequencing and segregated with PD in three families. RAB32 encodes a small GTPase known to interact with LRRK2 (refs. 3,4). Functional analyses showed that RAB32 S71R increases LRRK2 kinase activity, as indicated by increased autophosphorylation of LRRK2 S1292. Here our results implicate mutant RAB32 in a key pathological mechanism in PD—LRRK2 kinase activity5–7—and thus provide novel insights into the mechanistic connections between RAB family biology, LRRK2 and PD risk. Analysis of exome sequencing data identifies a missense variant in RAB32 associated with high risk of familial Parkinson’s disease. Functional studies show that this risk variant increases LRRK2 kinase activity.},
  archive      = {J_NG},
  author       = {Hop, Paul J. and Lai, Dongbing and Keagle, Pamela J. and Baron, Desiree M. and Kenna, Brendan J. and Kooyman, Maarten and Shankaracharya and Halter, Cheryl and Straniero, Letizia and Asselta, Rosanna and Bonvegna, Salvatore and Soto-Beasley, Alexandra I. and Wszolek, Zbigniew K. and Uitti, Ryan J. and Isaias, Ioannis Ugo and Pezzoli, Gianni and Ticozzi, Nicola and Ross, Owen A. and Veldink, Jan H. and Foroud, Tatiana M. and Kenna, Kevin P. and Landers, John E.},
  doi          = {10.1038/s41588-024-01787-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1371-1376},
  shortjournal = {Nat. Genet.},
  title        = {Systematic rare variant analyses identify RAB32 as a susceptibility gene for familial parkinson’s disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A common flanking variant is associated with enhanced
stability of the FGF14-SCA27B repeat locus. <em>NG</em>, <em>56</em>(7),
1366–1370. (<a
href="https://doi.org/10.1038/s41588-024-01808-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The factors driving or preventing pathological expansion of tandem repeats remain largely unknown. Here, we assessed the FGF14 (GAA)·(TTC) repeat locus in 2,530 individuals by long-read and Sanger sequencing and identified a common 5′-flanking variant in 70.34% of alleles analyzed (3,463/4,923) that represents the phylogenetically ancestral allele and is present on all major haplotypes. This common sequence variation is present nearly exclusively on nonpathogenic alleles with fewer than 30 GAA-pure triplets and is associated with enhanced stability of the repeat locus upon intergenerational transmission and increased Fiber-seq chromatin accessibility. Analysis of the FGF14-SCA27B repeat locus identifies a common 5′-flanking insertion that represents the major allele, is present exclusively on nonpathogenic alleles, enhances repeat stability and increases chromatin accessibility.},
  archive      = {J_NG},
  author       = {Pellerin, David and Del Gobbo, Giulia F. and Couse, Madeline and Dolzhenko, Egor and Nageshwaran, Sathiji K. and Cheung, Warren A. and Xu, Isaac R. L. and Dicaire, Marie-Josée and Spurdens, Guinevere and Matos-Rodrigues, Gabriel and Stevanovski, Igor and Scriba, Carolin K. and Rebelo, Adriana and Roth, Virginie and Wandzel, Marion and Bonnet, Céline and Ashton, Catherine and Agarwal, Aman and Peter, Cyril and Hasson, Dan and Tsankova, Nadejda M. and Dewar, Ken and Lamont, Phillipa J. and Laing, Nigel G. and Renaud, Mathilde and Houlden, Henry and Synofzik, Matthis and Usdin, Karen and Nussenzweig, Andre and Napierala, Marek and Chen, Zhao and Jiang, Hong and Deveson, Ira W. and Ravenscroft, Gianina and Akbarian, Schahram and Eberle, Michael A. and Boycott, Kym M. and Pastinen, Tomi and Brais, Bernard and Zuchner, Stephan and Danzi, Matt C.},
  doi          = {10.1038/s41588-024-01808-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1366-1370},
  shortjournal = {Nat. Genet.},
  title        = {A common flanking variant is associated with enhanced stability of the FGF14-SCA27B repeat locus},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mitochondrial genetics through the lens of single-cell
multi-omics. <em>NG</em>, <em>56</em>(7), 1355–1365. (<a
href="https://doi.org/10.1038/s41588-024-01794-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mitochondria carry their own genetic information encoding for a subset of protein-coding genes and translational machinery essential for cellular respiration and metabolism. Despite its small size, the mitochondrial genome, its natural genetic variation and molecular phenotypes have been challenging to study using bulk sequencing approaches, due to its variation in cellular copy number, non-Mendelian modes of inheritance and propensity for mutations. Here we highlight emerging strategies designed to capture mitochondrial genetic variation across individual cells for lineage tracing and studying mitochondrial genetics in primary human cells and clinical specimens. We review recent advances surrounding single-cell mitochondrial genome sequencing and its integration with functional genomic readouts, including leveraging somatic mitochondrial DNA mutations as clonal markers that can resolve cellular population dynamics in complex human tissues. Finally, we discuss how single-cell whole mitochondrial genome sequencing approaches can be utilized to investigate mitochondrial genetics and its contribution to cellular heterogeneity and disease. This Review discusses emerging technologies and insights from mitochondrial DNA variant profiling obtained by single-cell multi-omics, demonstrating how the powerhouse of the cell may record clonal lineages and contribute to disease heterogeneity.},
  archive      = {J_NG},
  author       = {Nitsch, Lena and Lareau, Caleb A. and Ludwig, Leif S.},
  doi          = {10.1038/s41588-024-01794-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1355-1365},
  shortjournal = {Nat. Genet.},
  title        = {Mitochondrial genetics through the lens of single-cell multi-omics},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The potential and translational application of infant
genetic research. <em>NG</em>, <em>56</em>(7), 1346–1354. (<a
href="https://doi.org/10.1038/s41588-024-01822-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the current genomic revolution, the infancy life stage is the most neglected. Although clinical genetics recognizes the value of early identification in infancy of rare genetic causes of disorders and delay, common genetic variation is almost completely ignored in research on infant behavioral and neurodevelopmental traits. In this Perspective, we argue for a much-needed surge in research on common genetic variation influencing infant neurodevelopment and behavior, findings that would be relevant for all children. We now see convincing evidence from different research designs to suggest that developmental milestones, skills and behaviors of infants are heritable and thus are suitable candidates for gene-discovery research. We highlight the resources available to the field, including genotyped infant cohorts, and we outline, with recommendations, special considerations needed for infant data. Therefore, infant genetic research has the potential to impact basic science and to affect educational policy, public health and clinical practice. Infant genetic research is currently underexplored but, as highlighted in this Perspective, has the potential to impact basic science and affect educational policy, public health and clinical practice.},
  archive      = {J_NG},
  author       = {Ronald, Angelica and Gui, Anna},
  doi          = {10.1038/s41588-024-01822-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1346-1354},
  shortjournal = {Nat. Genet.},
  title        = {The potential and translational application of infant genetic research},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A proposal for an inclusive working definition of genetic
discrimination to promote a more coherent debate. <em>NG</em>,
<em>56</em>(7), 1339–1345. (<a
href="https://doi.org/10.1038/s41588-024-01786-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic discrimination is an evolving phenomenon that impacts fundamental human rights such as dignity, justice and equity. Although, in the past, various definitions to better conceptualize genetic discrimination have been proposed, these have been unable to capture several key facets of the phenomenon. In this Perspective, we explore definitions of genetic discrimination across disciplines, consider criticisms of such definitions and show how other forms of discrimination and stigmatization can compound genetic discrimination in a way that affects individuals, groups and systems. We propose a nuanced and inclusive definition of genetic discrimination, which reflects its multifaceted impact that should remain relevant in the face of an evolving social context and advancing science. We argue that our definition should be adopted as a guiding academic framework to facilitate scientific and policy discussions about genetic discrimination and support the development of laws and industry policies seeking to address the phenomenon. Kaiser et al. propose a nuanced and inclusive definition of genetic discrimination that reflects its multifaceted impact. This adaptive definition is intended to remain relevant in the face of an evolving social context and advancing genomic science.},
  archive      = {J_NG},
  author       = {Kaiser, Beatrice and Uberoi, Diya and Raven-Adams, Maili C. and Cheung, Katherine and Bruns, Andreas and Chandrasekharan, Subhashini and Otlowski, Margaret and Prince, Anya E. R. and Tiller, Jane and Ahmed, Arzoo and Bombard, Yvonne and Dupras, Charles and Moreno, Palmira Granados and Ryan, Rosalyn and Valderrama-Aguirre, Augusto and Joly, Yann},
  doi          = {10.1038/s41588-024-01786-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1339-1345},
  shortjournal = {Nat. Genet.},
  title        = {A proposal for an inclusive working definition of genetic discrimination to promote a more coherent debate},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Nitrogen uptake and productive tiller formation in rice.
<em>NG</em>, <em>56</em>(7), 1337–1338. (<a
href="https://doi.org/10.1038/s41588-024-01796-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We identify an elite haplotype of the transcription factor gene OsGATA8 in rice that is associated with enhanced nitrogen uptake and a higher proportion of productive tillers. Revealing how OsGATA8 regulates nitrogen use efficiency (NUE) enables insights into coordination of nitrogen uptake and productive tiller formation to achieve high NUE in rice.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01796-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1337-1338},
  shortjournal = {Nat. Genet.},
  title        = {Nitrogen uptake and productive tiller formation in rice},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ultrasound perception mechanisms revealed by comparisons of
microbats and megabats. <em>NG</em>, <em>56</em>(7), 1335–1336. (<a
href="https://doi.org/10.1038/s41588-024-01803-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbats utilize ultrasonic echolocation to navigate and locate prey, whereas megabats primarily perceive human-audible sound in daily life. High-quality genomes and single-cell atlases of auditory cortices from microbat and megabat species identify parvalbumin-positive inhibitory neurons and the complexin-1 gene to be crucial in mammalian ultrasound perception.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01803-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1335-1336},
  shortjournal = {Nat. Genet.},
  title        = {Ultrasound perception mechanisms revealed by comparisons of microbats and megabats},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic gear switches drive cancer immune evasion.
<em>NG</em>, <em>56</em>(7), 1333–1334. (<a
href="https://doi.org/10.1038/s41588-024-01778-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Relentless accumulation of somatic mutations renders mismatch repair (MMR)-deficient cancers immunogenic. The evolutionary strategies that these hypermutator tumors use to drive immune evasion remain unknown. We identify repetitive homopolymer sequences in MMR genes as genetic ON/OFF switches, which vary mutation rate and bias during tumor evolution to fuel intratumor heterogeneity.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01778-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1333-1334},
  shortjournal = {Nat. Genet.},
  title        = {Genetic gear switches drive cancer immune evasion},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A ΔNp63–MED12 axis drives basal-like identity in pancreatic
cancer. <em>NG</em>, <em>56</em>(7), 1331–1332. (<a
href="https://doi.org/10.1038/s41588-024-01797-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive dissection of the transcriptional basis of basal-like differentiation of pancreatic ductal adenocarcinoma (PDAC) revealed MED12 as a core regulator of this abnormal identity. Mechanistic studies demonstrate that MED12 cooperates with the basal-like master regulator transcription factor ΔNp63 by bridging it with the critical Mediator co-activator complex.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01797-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1331-1332},
  shortjournal = {Nat. Genet.},
  title        = {A ΔNp63–MED12 axis drives basal-like identity in pancreatic cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). LINE1 mediates long-range DNA interactions. <em>NG</em>,
<em>56</em>(7), 1328–1330. (<a
href="https://doi.org/10.1038/s41588-024-01824-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of transposable elements (TEs) on gene regulatory networks and how they are governed remain poorly understood. A new study identifies a role for LINE1, a specific subset of TEs, in the long-range regulation of gene expression, particularly during embryonic development.},
  archive      = {J_NG},
  author       = {Liang, Yonghao and Wang, Ting},
  doi          = {10.1038/s41588-024-01824-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1328-1330},
  shortjournal = {Nat. Genet.},
  title        = {LINE1 mediates long-range DNA interactions},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Measuring nascent transcription in single cells.
<em>NG</em>, <em>56</em>(7), 1327. (<a
href="https://doi.org/10.1038/s41588-024-01845-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-024-01845-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1327},
  shortjournal = {Nat. Genet.},
  title        = {Measuring nascent transcription in single cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Understanding x chromosome loss. <em>NG</em>,
<em>56</em>(7), 1327. (<a
href="https://doi.org/10.1038/s41588-024-01846-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-024-01846-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1327},
  shortjournal = {Nat. Genet.},
  title        = {Understanding x chromosome loss},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A one-stop shop for 3D spatial transcriptomics.
<em>NG</em>, <em>56</em>(7), 1327. (<a
href="https://doi.org/10.1038/s41588-024-01847-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-024-01847-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1327},
  shortjournal = {Nat. Genet.},
  title        = {A one-stop shop for 3D spatial transcriptomics},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Finding cancer mutagens using signature analysis.
<em>NG</em>, <em>56</em>(7), 1327. (<a
href="https://doi.org/10.1038/s41588-024-01848-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-01848-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1327},
  shortjournal = {Nat. Genet.},
  title        = {Finding cancer mutagens using signature analysis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving transparency of computational tools for variant
effect prediction. <em>NG</em>, <em>56</em>(7), 1324–1326. (<a
href="https://doi.org/10.1038/s41588-024-01821-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efforts to integrate computational tools for variant effect prediction into the process of clinical decision-making are in progress. However, for such efforts to succeed and help to provide more informed clinical decisions, it is necessary to enhance transparency and address the current limitations of computational predictors.},
  archive      = {J_NG},
  author       = {Karchin, Rachel and Radivojac, Predrag and O’Donnell-Luria, Anne and Greenblatt, Marc S. and Tolstorukov, Michael Y. and Sonkin, Dmitriy},
  doi          = {10.1038/s41588-024-01821-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1324-1326},
  shortjournal = {Nat. Genet.},
  title        = {Improving transparency of computational tools for variant effect prediction},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mining the diagnosis of rare disease with limited resources.
<em>NG</em>, <em>56</em>(7), 1323. (<a
href="https://doi.org/10.1038/s41588-024-01818-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {From a young age, my fascination with life sciences led me to follow in the footsteps of my grandmother, a pediatrician, whose name I proudly bear. After graduating from Tbilisi State Medical University in 2004, I pursued my PhD in neuroscience under the guidance of my lifelong mentor Elene Abzianidze. However, it was not until my clinical training in neurology in 2015, when seeing children with rare neurological diseases, that I realized the importance of precise diagnosis.},
  archive      = {J_NG},
  author       = {Tkemaladze, Tinatin},
  doi          = {10.1038/s41588-024-01818-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1323},
  shortjournal = {Nat. Genet.},
  title        = {Mining the diagnosis of rare disease with limited resources},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Pathway level subtyping identifies a
slow-cycling biological phenotype associated with poor clinical outcomes
in colorectal cancer. <em>NG</em>, <em>56</em>(6), 1321. (<a
href="https://doi.org/10.1038/s41588-024-01809-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Malla, Sudhir B. and Byrne, Ryan M. and Lafarge, Maxime W. and Corry, Shania M. and Fisher, Natalie C. and Tsantoulis, Petros K. and Mills, Megan L. and Ridgway, Rachel A. and Lannagan, Tamsin R. M. and Najumudeen, Arafath K. and Gilroy, Kathryn L. and Amirkhah, Raheleh and Maguire, Sarah L. and Mulholland, Eoghan J. and Belnoue-Davis, Hayley L. and Grassi, Elena and Viviani, Marco and Rogan, Emily and Redmond, Keara L. and Sakhnevych, Svetlana and McCooey, Aoife J. and Bull, Courtney and Hoey, Emily and Sinevici, Nicoleta and Hall, Holly and Ahmaderaghi, Baharak and Domingo, Enric and Blake, Andrew and Richman, Susan D. and Isella, Claudio and Miller, Crispin and Bertotti, Andrea and Trusolino, Livio and Loughrey, Maurice B. and Kerr, Emma M. and Tejpar, Sabine and Maughan, Timothy S. and Lawler, Mark and Campbell, Andrew D. and Leedham, Simon J. and Koelzer, Viktor H. and Sansom, Owen J. and Dunne, Philip D.},
  doi          = {10.1038/s41588-024-01809-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1321},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Pathway level subtyping identifies a slow-cycling biological phenotype associated with poor clinical outcomes in colorectal cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Cicer super-pangenome provides insights
into species evolution and agronomic trait loci for crop improvement in
chickpea. <em>NG</em>, <em>56</em>(6), 1320. (<a
href="https://doi.org/10.1038/s41588-024-01813-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Khan, Aamir W. and Garg, Vanika and Sun, Shuai and Gupta, Saurabh and Dudchenko, Olga and Roorkiwal, Manish and Chitikineni, Annapurna and Bayer, Philipp E. and Shi, Chengcheng and Upadhyaya, Hari D. and Bohra, Abhishek and Bharadwaj, Chellapilla and Mir, Reyazul Rouf and Baruch, Kobi and Yang, Bicheng and Coyne, Clarice J. and Bansal, Kailash C. and Nguyen, Henry T. and Ronen, Gil and Aiden, Erez Lieberman and Veneklaas, Erik and Siddique, Kadambot H. M. and Liu, Xin and Edwards, David and Varshney, Rajeev K.},
  doi          = {10.1038/s41588-024-01813-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1320},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Cicer super-pangenome provides insights into species evolution and agronomic trait loci for crop improvement in chickpea},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Single-cell multi-ome regression models
identify functional and disease-associated enhancers and enable
chromatin potential analysis. <em>NG</em>, <em>56</em>(6), 1319. (<a
href="https://doi.org/10.1038/s41588-024-01805-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Mitra, Sneha and Malik, Rohan and Wong, Wilfred and Rahman, Afsana and Hartemink, Alexander J. and Pritykin, Yuri and Dey, Kushal K. and Leslie, Christina S.},
  doi          = {10.1038/s41588-024-01805-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1319},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Single-cell multi-ome regression models identify functional and disease-associated enhancers and enable chromatin potential analysis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improved functional mapping of complex trait heritability
with GSA-MiXeR implicates biologically specific gene sets. <em>NG</em>,
<em>56</em>(6), 1310–1318. (<a
href="https://doi.org/10.1038/s41588-024-01771-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While genome-wide association studies are increasingly successful in discovering genomic loci associated with complex human traits and disorders, the biological interpretation of these findings remains challenging. Here we developed the GSA-MiXeR analytical tool for gene set analysis (GSA), which fits a model for the heritability of individual genes, accounting for linkage disequilibrium across variants and allowing the quantification of partitioned heritability and fold enrichment for small gene sets. We validated the method using extensive simulations and sensitivity analyses. When applied to a diverse selection of complex traits and disorders, including schizophrenia, GSA-MiXeR prioritizes gene sets with greater biological specificity compared to standard GSA approaches, implicating voltage-gated calcium channel function and dopaminergic signaling for schizophrenia. Such biologically relevant gene sets, often with fewer than ten genes, are more likely to provide insights into the pathobiology of complex diseases and highlight potential drug targets. GSA-MiXeR models gene heritability and variant linkage disequilibrium for improved gene set enrichment testing. GSA-MiXeR implicates relevant sets of fewer than ten genes in schizophrenia, providing more nuanced insights into trait biology.},
  archive      = {J_NG},
  author       = {Frei, Oleksandr and Hindley, Guy and Shadrin, Alexey A. and van der Meer, Dennis and Akdeniz, Bayram C. and Hagen, Espen and Cheng, Weiqiu and O’Connell, Kevin S. and Bahrami, Shahram and Parker, Nadine and Smeland, Olav B. and Holland, Dominic and de Leeuw, Christiaan and Posthuma, Danielle and Andreassen, Ole A. and Dale, Anders M.},
  doi          = {10.1038/s41588-024-01771-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1310-1318},
  shortjournal = {Nat. Genet.},
  title        = {Improved functional mapping of complex trait heritability with GSA-MiXeR implicates biologically specific gene sets},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Direct transposition of native DNA for sensitive multimodal
single-molecule sequencing. <em>NG</em>, <em>56</em>(6), 1300–1309. (<a
href="https://doi.org/10.1038/s41588-024-01748-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concurrent readout of sequence and base modifications from long unamplified DNA templates by Pacific Biosciences of California (PacBio) single-molecule sequencing requires large amounts of input material. Here we adapt Tn5 transposition to introduce hairpin oligonucleotides and fragment (tagment) limiting quantities of DNA for generating PacBio-compatible circular molecules. We developed two methods that implement tagmentation and use 90–99% less input than current protocols: (1) single-molecule real-time sequencing by tagmentation (SMRT-Tag), which allows detection of genetic variation and CpG methylation; and (2) single-molecule adenine-methylated oligonucleosome sequencing assay by tagmentation (SAMOSA-Tag), which uses exogenous adenine methylation to add a third channel for probing chromatin accessibility. SMRT-Tag of 40 ng or more human DNA (approximately 7,000 cell equivalents) yielded data comparable to gold standard whole-genome and bisulfite sequencing. SAMOSA-Tag of 30,000–50,000 nuclei resolved single-fiber chromatin structure, CTCF binding and DNA methylation in patient-derived prostate cancer xenografts and uncovered metastasis-associated global epigenome disorganization. Tagmentation thus promises to enable sensitive, scalable and multimodal single-molecule genomics for diverse basic and clinical applications. Two low-input tagmentation-based long-read sequencing methods, single-molecule real-time sequencing by tagmentation (SMRT-Tag), which identifies genetic variation and CpG methylation, and single-molecule adenine-methylated oligonucleosome sequencing assay by tagmentation (SAMOSA-Tag), which detects chromatin accessibility, are presented. Application of SAMOSA-Tag to prostate cancer patient-derived xenograft samples identifies metastasis-associated epigenomic alterations.},
  archive      = {J_NG},
  author       = {Nanda, Arjun S. and Wu, Ke and Irkliyenko, Iryna and Woo, Brian and Ostrowski, Megan S. and Clugston, Andrew S. and Sayles, Leanne C. and Xu, Lingru and Satpathy, Ansuman T. and Nguyen, Hao G. and Alejandro Sweet-Cordero, E. and Goodarzi, Hani and Kasinathan, Sivakanthan and Ramani, Vijay},
  doi          = {10.1038/s41588-024-01748-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1300-1309},
  shortjournal = {Nat. Genet.},
  title        = {Direct transposition of native DNA for sensitive multimodal single-molecule sequencing},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-throughput fitness experiments reveal specific
vulnerabilities of human-adapted salmonella during stress and infection.
<em>NG</em>, <em>56</em>(6), 1288–1299. (<a
href="https://doi.org/10.1038/s41588-024-01779-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Salmonella enterica is comprised of genetically distinct ‘serovars’ that together provide an intriguing model for exploring the genetic basis of pathogen evolution. Although the genomes of numerous Salmonella isolates with broad variations in host range and human disease manifestations have been sequenced, the functional links between genetic and phenotypic differences among these serovars remain poorly understood. Here, we conduct high-throughput functional genomics on both generalist (Typhimurium) and human-restricted (Typhi and Paratyphi A) Salmonella at unprecedented scale in the study of this enteric pathogen. Using a comprehensive systems biology approach, we identify gene networks with serovar-specific fitness effects across 25 host-associated stresses encountered at key stages of human infection. By experimentally perturbing these networks, we characterize previously undescribed pseudogenes in human-adapted Salmonella. Overall, this work highlights specific vulnerabilities encoded within human-restricted Salmonella that are linked to the degradation of their genomes, shedding light into the evolution of this enteric pathogen. Random barcoded transposon sequencing screens of generalist and typhoidal Salmonella determine the fitness effects of genes in a range of stress conditions and during macrophage infection, characterizing unknown genes and identifying typhoidal-specific vulnerabilities.},
  archive      = {J_NG},
  author       = {Wang, Benjamin X. and Leshchiner, Dmitry and Luo, Lijuan and Tuncel, Miles and Hokamp, Karsten and Hinton, Jay C. D. and Monack, Denise M.},
  doi          = {10.1038/s41588-024-01779-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1288-1299},
  shortjournal = {Nat. Genet.},
  title        = {High-throughput fitness experiments reveal specific vulnerabilities of human-adapted salmonella during stress and infection},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pan-transcriptome reveals a large accessory genome
contribution to gene expression variation in yeast. <em>NG</em>,
<em>56</em>(6), 1278–1287. (<a
href="https://doi.org/10.1038/s41588-024-01769-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression is an essential step in the translation of genotypes into phenotypes. However, little is known about the transcriptome architecture and the underlying genetic effects at the species level. Here we generated and analyzed the pan-transcriptome of ~1,000 yeast natural isolates across 4,977 core and 1,468 accessory genes. We found that the accessory genome is an underappreciated driver of transcriptome divergence. Global gene expression patterns combined with population structure showed that variation in heritable expression mainly lies within subpopulation-specific signatures, for which accessory genes are overrepresented. Genome-wide association analyses consistently highlighted that accessory genes are associated with proportionally more variants with larger effect sizes, illustrating the critical role of the accessory genome on the transcriptional landscape within and between populations. Insight from the transcriptomes of 1,032 Saccharomyces cerevisiae natural isolates emphasizes the essential contribution of accessory genes to the species-level transcriptional landscape.},
  archive      = {J_NG},
  author       = {Caudal, Élodie and Loegler, Victor and Dutreux, Fabien and Vakirlis, Nikolaos and Teyssonnière, Élie and Caradec, Claudia and Friedrich, Anne and Hou, Jing and Schacherer, Joseph},
  doi          = {10.1038/s41588-024-01769-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1278-1287},
  shortjournal = {Nat. Genet.},
  title        = {Pan-transcriptome reveals a large accessory genome contribution to gene expression variation in yeast},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Long noncoding RNAs underlie multiple domestication traits
and leafhopper resistance in soybean. <em>NG</em>, <em>56</em>(6),
1270–1277. (<a
href="https://doi.org/10.1038/s41588-024-01738-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The origin and functionality of long noncoding RNA (lncRNA) remain poorly understood. Here, we show that multiple quantitative trait loci modulating distinct domestication traits in soybeans are pleiotropic effects of a locus composed of two tandem lncRNA genes. These lncRNA genes, each containing two inverted repeats, originating from coding sequences of the MYB genes, function in wild soybeans by generating clusters of small RNA (sRNA) species that inhibit the expression of their MYB gene relatives through post-transcriptional regulation. By contrast, the expression of lncRNA genes in cultivated soybeans is severely repressed, and, consequently, the corresponding MYB genes are highly expressed, shaping multiple distinct domestication traits as well as leafhopper resistance. The inverted repeats were formed before the divergence of the Glycine genus from the Phaseolus–Vigna lineage and exhibit strong structure–function constraints. This study exemplifies a type of target for selection during plant domestication and identifies mechanisms of lncRNA formation and action. Multiple domestication-related traits in cultivated soybeans are pleiotropic effects of a locus composed of two tandemly duplicated long noncoding RNAs that act via sRNA repression of MYB transcription factors.},
  archive      = {J_NG},
  author       = {Wang, Weidong and Duan, Jingbo and Wang, Xutong and Feng, Xingxing and Chen, Liyang and Clark, Chancelor B. and Swarm, Stephen A. and Wang, Jinbin and Lin, Sen and Nelson, Randall L. and Meyers, Blake C. and Feng, Xianzhong and Ma, Jianxin},
  doi          = {10.1038/s41588-024-01738-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1270-1277},
  shortjournal = {Nat. Genet.},
  title        = {Long noncoding RNAs underlie multiple domestication traits and leafhopper resistance in soybean},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Variation in TaSPL6-d confers salinity tolerance in bread
wheat by activating TaHKT1;5-d while preserving yield-related traits.
<em>NG</em>, <em>56</em>(6), 1257–1269. (<a
href="https://doi.org/10.1038/s41588-024-01762-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Na+ exclusion from above-ground tissues via the Na+-selective transporter HKT1;5 is a major salt-tolerance mechanism in crops. Using the expression genome-wide association study and yeast-one-hybrid screening, we identified TaSPL6-D, a transcriptional suppressor of TaHKT1;5-D in bread wheat. SPL6 also targeted HKT1;5 in rice and Brachypodium. A 47-bp insertion in the first exon of TaSPL6-D resulted in a truncated peptide, TaSPL6-DIn, disrupting TaHKT1;5-D repression exhibited by TaSPL6-DDel. Overexpressing TaSPL6-DDel, but not TaSPL6-DIn, led to inhibited TaHKT1;5-D expression and increased salt sensitivity. Knockout of TaSPL6-DDel in two wheat genotypes enhanced salinity tolerance, which was attenuated by a further TaHKT1;5-D knockdown. Spike development was preserved in Taspl6-dd mutants but not in Taspl6-aabbdd mutants. TaSPL6-DIn was mainly present in landraces, and molecular-assisted introduction of TaSPL6-DIn from a landrace into a leading wheat cultivar successfully improved yield on saline soils. The SPL6-HKT1;5 module offers a target for the molecular breeding of salt-tolerant crops. The study identifies TaSPL6-D as a negative regulator of TaHKT1;5-D and salinity tolerance in bread wheat. An insertion variation of TaSPL6-D, mainly hidden in landraces, shows the potential for breeding salt-tolerant crops.},
  archive      = {J_NG},
  author       = {Wang, Meng and Cheng, Jie and Wu, Jianhui and Chen, Jiefei and Liu, Dan and Wang, Chenyang and Ma, Shengwei and Guo, Weiwei and Li, Guangjie and Di, Dongwei and Zhang, Yumei and Han, Dejun and Kronzucker, Herbert J. and Xia, Guangmin and Shi, Weiming},
  doi          = {10.1038/s41588-024-01762-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1257-1269},
  shortjournal = {Nat. Genet.},
  title        = {Variation in TaSPL6-D confers salinity tolerance in bread wheat by activating TaHKT1;5-D while preserving yield-related traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Seedling root system adaptation to water availability during
maize domestication and global expansion. <em>NG</em>, <em>56</em>(6),
1245–1256. (<a
href="https://doi.org/10.1038/s41588-024-01761-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The maize root system has been reshaped by indirect selection during global adaptation to new agricultural environments. In this study, we characterized the root systems of more than 9,000 global maize accessions and its wild relatives, defining the geographical signature and genomic basis of variation in seminal root number. We demonstrate that seminal root number has increased during maize domestication followed by a decrease in response to limited water availability in locally adapted varieties. By combining environmental and phenotypic association analyses with linkage mapping, we identified genes linking environmental variation and seminal root number. Functional characterization of the transcription factor ZmHb77 and in silico root modeling provides evidence that reshaping root system architecture by reducing the number of seminal roots and promoting lateral root density is beneficial for the resilience of maize seedlings to drought. Characterization of seminal root number variation in the root systems of &amp;gt;9,000 global maize accessions and its wild relatives provides insights into root trait adaptation to environments during domestication and global expansion.},
  archive      = {J_NG},
  author       = {Yu, Peng and Li, Chunhui and Li, Meng and He, Xiaoming and Wang, Danning and Li, Hongjie and Marcon, Caroline and Li, Yu and Perez-Limón, Sergio and Chen, Xinping and Delgado-Baquerizo, Manuel and Koller, Robert and Metzner, Ralf and van Dusschoten, Dagmar and Pflugfelder, Daniel and Borisjuk, Ljudmilla and Plutenko, Iaroslav and Mahon, Audrey and Resende, Marcio F. R. and Salvi, Silvio and Akale, Asegidew and Abdalla, Mohanned and Ahmed, Mutez Ali and Bauer, Felix Maximilian and Schnepf, Andrea and Lobet, Guillaume and Heymans, Adrien and Suresh, Kiran and Schreiber, Lukas and McLaughlin, Chloee M. and Li, Chunjian and Mayer, Manfred and Schön, Chris-Carolin and Bernau, Vivian and von Wirén, Nicolaus and Sawers, Ruairidh J. H. and Wang, Tianyu and Hochholdinger, Frank},
  doi          = {10.1038/s41588-024-01761-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1245-1256},
  shortjournal = {Nat. Genet.},
  title        = {Seedling root system adaptation to water availability during maize domestication and global expansion},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genomic analyses reveal the stepwise domestication and
genetic mechanism of curd biogenesis in cauliflower. <em>NG</em>,
<em>56</em>(6), 1235–1244. (<a
href="https://doi.org/10.1038/s41588-024-01744-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cauliflower (Brassica oleracea L. var. botrytis) is a distinctive vegetable that supplies a nutrient-rich edible inflorescence meristem for the human diet. However, the genomic bases of its selective breeding have not been studied extensively. Herein, we present a high-quality reference genome assembly C-8 (V2) and a comprehensive genomic variation map consisting of 971 diverse accessions of cauliflower and its relatives. Genomic selection analysis and deep-mined divergences were used to explore a stepwise domestication process for cauliflower that initially evolved from broccoli (Curd-emergence and Curd-improvement), revealing that three MADS-box genes, CAULIFLOWER1 (CAL1), CAL2 and FRUITFULL (FUL2), could have essential roles during curd formation. Genome-wide association studies identified nine loci significantly associated with morphological and biological characters and demonstrated that a zinc-finger protein (BOB06G135460) positively regulates stem height in cauliflower. This study offers valuable genomic resources for better understanding the genetic bases of curd biogenesis and florescent development in crops. A high-quality reference genome assembly of cauliflower C-8 (V2) and genomic analyses of 971 diverse accessions and their relatives reveal the stepwise domestication and the genetic mechanism of curd biogenesis.},
  archive      = {J_NG},
  author       = {Chen, Rui and Chen, Ke and Yao, Xingwei and Zhang, Xiaoli and Yang, Yingxia and Su, Xiao and Lyu, Mingjie and Wang, Qian and Zhang, Guan and Wang, Mengmeng and Li, Yanhao and Duan, Lijin and Xie, Tianyu and Li, Haichao and Yang, Yuyao and Zhang, Hong and Guo, Yutong and Jia, Guiying and Ge, Xianhong and Sarris, Panagiotis F. and Lin, Tao and Sun, Deling},
  doi          = {10.1038/s41588-024-01744-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1235-1244},
  shortjournal = {Nat. Genet.},
  title        = {Genomic analyses reveal the stepwise domestication and genetic mechanism of curd biogenesis in cauliflower},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Cicer super-pangenome provides insights into species
evolution and agronomic trait loci for crop improvement in chickpea.
<em>NG</em>, <em>56</em>(6), 1225–1234. (<a
href="https://doi.org/10.1038/s41588-024-01760-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chickpea (Cicer arietinum L.)—an important legume crop cultivated in arid and semiarid regions—has limited genetic diversity. Efforts are being undertaken to broaden its diversity by utilizing its wild relatives, which remain largely unexplored. Here, we present the Cicer super-pangenome based on the de novo genome assemblies of eight annual Cicer wild species. We identified 24,827 gene families, including 14,748 core, 2,958 softcore, 6,212 dispensable and 909 species-specific gene families. The dispensable genome was enriched for genes related to key agronomic traits. Structural variations between cultivated and wild genomes were used to construct a graph-based genome, revealing variations in genes affecting traits such as flowering time, vernalization and disease resistance. These variations will facilitate the transfer of valuable traits from wild Cicer species into elite chickpea varieties through marker-assisted selection or gene-editing. This study offers valuable insights into the genetic diversity and potential avenues for crop improvement in chickpea. Cicer super-pangenome constructed using genome assemblies of eight Cicer annual wild species and two cultivated chickpea species provides insights into the genetic diversity and agronomic trait loci for chickpea improvement.},
  archive      = {J_NG},
  author       = {Khan, Aamir W. and Garg, Vanika and Sun, Shuai and Gupta, Saurabh and Dudchenko, Olga and Roorkiwal, Manish and Chitikineni, Annapurna and Bayer, Philipp E. and Shi, Chengcheng and Upadhyaya, Hari D. and Bohra, Abhishek and Bharadwaj, Chellapilla and Mir, Reyazul Rouf and Baruch, Kobi and Yang, Bicheng and Coyne, Clarice J. and Bansal, Kailash C. and Nguyen, Henry T. and Ronen, Gil and Aiden, Erez Lieberman and Veneklaas, Erik and Siddique, Kadambot H. M. and Liu, Xin and Edwards, David and Varshney, Rajeev K.},
  doi          = {10.1038/s41588-024-01760-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1225-1234},
  shortjournal = {Nat. Genet.},
  title        = {Cicer super-pangenome provides insights into species evolution and agronomic trait loci for crop improvement in chickpea},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome folding principles uncovered in condensin-depleted
mitotic chromosomes. <em>NG</em>, <em>56</em>(6), 1213–1224. (<a
href="https://doi.org/10.1038/s41588-024-01759-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During mitosis, condensin activity is thought to interfere with interphase chromatin structures. To investigate genome folding principles in the absence of chromatin loop extrusion, we codepleted condensin I and condensin II, which triggered mitotic chromosome compartmentalization in ways similar to that in interphase. However, two distinct euchromatic compartments, indistinguishable in interphase, emerged upon condensin loss with different interaction preferences and dependencies on H3K27ac. Constitutive heterochromatin gradually self-aggregated and cocompartmentalized with facultative heterochromatin, contrasting with their separation during interphase. Notably, some cis-regulatory element contacts became apparent even in the absence of CTCF/cohesin-mediated structures. Heterochromatin protein 1 (HP1) proteins, which are thought to partition constitutive heterochromatin, were absent from mitotic chromosomes, suggesting, surprisingly, that constitutive heterochromatin can self-aggregate without HP1. Indeed, in cells traversing from M to G1 phase in the combined absence of HP1α, HP1β and HP1γ, constitutive heterochromatin compartments are normally re-established. In sum, condensin-deficient mitotic chromosomes illuminate forces of genome compartmentalization not identified in interphase cells. Condensin-depleted mitotic chromosomes compartmentalize and form contacts among regulatory elements despite lacking transcription and most chromatin-associated factors. Heterochromatin protein 1 (HP1) proteins are surprisingly dispensable for compartmentalizing constitutive heterochromatin.},
  archive      = {J_NG},
  author       = {Zhao, Han and Lin, Yinzhi and Lin, En and Liu, Fuhai and Shu, Lirong and Jing, Dannan and Wang, Baiyue and Wang, Manzhu and Shan, Fengnian and Zhang, Lin and Lam, Jessica C. and Midla, Susannah C. and Giardine, Belinda M. and Keller, Cheryl A. and Hardison, Ross C. and Blobel, Gerd A. and Zhang, Haoyue},
  doi          = {10.1038/s41588-024-01759-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1213-1224},
  shortjournal = {Nat. Genet.},
  title        = {Genome folding principles uncovered in condensin-depleted mitotic chromosomes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Systematic assessment of ISWI subunits shows that NURF
creates local accessibility for CTCF. <em>NG</em>, <em>56</em>(6),
1203–1212. (<a
href="https://doi.org/10.1038/s41588-024-01767-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Catalytic activity of the imitation switch (ISWI) family of remodelers is critical for nucleosomal organization and DNA binding of certain transcription factors, including the insulator protein CTCF. Here we define the contribution of individual subcomplexes by deriving a panel of isogenic mouse stem cell lines, each lacking one of six ISWI accessory subunits. Individual deletions of subunits of either CERF, RSF, ACF, WICH or NoRC subcomplexes only moderately affect the chromatin landscape, while removal of the NURF-specific subunit BPTF leads to a strong reduction in chromatin accessibility and SNF2H ATPase localization around CTCF sites. This affects adjacent nucleosome occupancy and CTCF binding. At a group of sites with reduced chromatin accessibility, CTCF binding persists but cohesin occupancy is reduced, resulting in decreased insulation. These results suggest that CTCF binding can be separated from its function as an insulator in nuclear organization and identify a specific role for NURF in mediating SNF2H localization and chromatin opening at bound CTCF sites. Systematic deletions of individual imitation switch (ISWI) subunits in mouse embryonic stem cell lines highlight the role of BPTF in regulating chromatin accessibility at most CTCF sites and binding at a subset, with a modest effect on its insulating function.},
  archive      = {J_NG},
  author       = {Iurlaro, Mario and Masoni, Francesca and Flyamer, Ilya M. and Wirbelauer, Christiane and Iskar, Murat and Burger, Lukas and Giorgetti, Luca and Schübeler, Dirk},
  doi          = {10.1038/s41588-024-01767-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1203-1212},
  shortjournal = {Nat. Genet.},
  title        = {Systematic assessment of ISWI subunits shows that NURF creates local accessibility for CTCF},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inseparable RNA binding and chromatin modification
activities of a nucleosome-interacting surface in EZH2. <em>NG</em>,
<em>56</em>(6), 1193–1202. (<a
href="https://doi.org/10.1038/s41588-024-01740-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polycomb repressive complex 2 (PRC2) interacts with RNA in cells, but there is no consensus on how RNA regulates PRC2 canonical functions, including chromatin modification and the maintenance of transcription programs in lineage-committed cells. We assayed two separation-of-function mutants of the PRC2 catalytic subunit EZH2, defective in RNA binding but functional in methyltransferase activity. We find that part of the RNA-binding surface of EZH2 is required for chromatin modification, yet this activity is independent of RNA. Mechanistically, the RNA-binding surface within EZH2 is required for chromatin modification in vitro and in cells, through interactions with nucleosomal DNA. Contrarily, an RNA-binding-defective mutant exhibited normal chromatin modification activity in vitro and in lineage-committed cells, accompanied by normal gene repression activity. Collectively, we show that part of the RNA-binding surface of EZH2, rather than the RNA-binding activity per se, is required for the histone methylation in vitro and in cells, through interactions with the substrate nucleosome. Analysis of EZH2 separation-of-function mutants indicates that part of the RNA-binding surface of EZH2 is necessary for histone modification independently of RNA. A specific RNA-binding-defective mutant shows normal enzymatic activity in vitro and in lineage-committed cells.},
  archive      = {J_NG},
  author       = {Gail, Emma H. and Healy, Evan and Flanigan, Sarena F. and Jones, Natasha and Ng, Xiao Han and Uckelmann, Michael and Levina, Vitalina and Zhang, Qi and Davidovich, Chen},
  doi          = {10.1038/s41588-024-01740-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1193-1202},
  shortjournal = {Nat. Genet.},
  title        = {Inseparable RNA binding and chromatin modification activities of a nucleosome-interacting surface in EZH2},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Comparative cofactor screens show the influence of
transactivation domains and core promoters on the mechanisms of
transcription. <em>NG</em>, <em>56</em>(6), 1181–1192. (<a
href="https://doi.org/10.1038/s41588-024-01749-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Eukaryotic transcription factors (TFs) activate gene expression by recruiting cofactors to promoters. However, the relationships between TFs, promoters and their associated cofactors remain poorly understood. Here we combine GAL4-transactivation assays with comparative CRISPR–Cas9 screens to identify the cofactors used by nine different TFs and core promoters in human cells. Using this dataset, we associate TFs with cofactors, classify cofactors as ubiquitous or specific and discover transcriptional co-dependencies. Through a reductionistic, comparative approach, we demonstrate that TFs do not display discrete mechanisms of activation. Instead, each TF depends on a unique combination of cofactors, which influences distinct steps in transcription. By contrast, the influence of core promoters appears relatively discrete. Different promoter classes are constrained by either initiation or pause-release, which influences their dynamic range and compatibility with cofactors. Overall, our comparative cofactor screens characterize the interplay between TFs, cofactors and core promoters, identifying general principles by which they influence transcription. Systematic assessment of cofactor dependencies of nine transcription factors (TFs) and promoters finds that TFs use unique cofactor combinations to modulate distinct steps in transcription, whereas promoter elements fit into discrete groups where their rate-limiting step for activation influences cofactor compatibility.},
  archive      = {J_NG},
  author       = {Bell, Charles C. and Balic, Jesse J. and Talarmain, Laure and Gillespie, Andrea and Scolamiero, Laura and Lam, Enid Y. N. and Ang, Ching-Seng and Faulkner, Geoffrey J. and Gilan, Omer and Dawson, Mark A.},
  doi          = {10.1038/s41588-024-01749-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1181-1192},
  shortjournal = {Nat. Genet.},
  title        = {Comparative cofactor screens show the influence of transactivation domains and core promoters on the mechanisms of transcription},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic epigenome editing captures the context-dependent
instructive function of chromatin modifications. <em>NG</em>,
<em>56</em>(6), 1168–1180. (<a
href="https://doi.org/10.1038/s41588-024-01706-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin modifications are linked with regulating patterns of gene expression, but their causal role and context-dependent impact on transcription remains unresolved. Here we develop a modular epigenome editing platform that programs nine key chromatin modifications, or combinations thereof, to precise loci in living cells. We couple this with single-cell readouts to systematically quantitate the magnitude and heterogeneity of transcriptional responses elicited by each specific chromatin modification. Among these, we show that installing histone H3 lysine 4 trimethylation (H3K4me3) at promoters can causally instruct transcription by hierarchically remodeling the chromatin landscape. We further dissect how DNA sequence motifs influence the transcriptional impact of chromatin marks, identifying switch-like and attenuative effects within distinct cis contexts. Finally, we examine the interplay of combinatorial modifications, revealing that co-targeted H3K27 trimethylation (H3K27me3) and H2AK119 monoubiquitination (H2AK119ub) maximizes silencing penetrance across single cells. Our precision-perturbation strategy unveils the causal principles of how chromatin modification(s) influence transcription and dissects how quantitative responses are calibrated by contextual interactions. The authors develop and harness a suite of epigenome editing tools to explore the role of different epigenetic marks in modulating transcription. In particular, H3K4me3 deposition on promoter sequences is shown to directly promote transcription activation in mouse embryonic stem cells.},
  archive      = {J_NG},
  author       = {Policarpi, Cristina and Munafò, Marzia and Tsagkris, Stylianos and Carlini, Valentina and Hackett, Jamie A.},
  doi          = {10.1038/s41588-024-01706-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1168-1180},
  shortjournal = {Nat. Genet.},
  title        = {Systematic epigenome editing captures the context-dependent instructive function of chromatin modifications},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic decoding of cis gene regulation defines
context-dependent control of the multi-gene costimulatory receptor locus
in human t cells. <em>NG</em>, <em>56</em>(6), 1156–1167. (<a
href="https://doi.org/10.1038/s41588-024-01743-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cis-regulatory elements (CREs) interact with trans regulators to orchestrate gene expression, but how transcriptional regulation is coordinated in multi-gene loci has not been experimentally defined. We sought to characterize the CREs controlling dynamic expression of the adjacent costimulatory genes CD28, CTLA4 and ICOS, encoding regulators of T cell-mediated immunity. Tiling CRISPR interference (CRISPRi) screens in primary human T cells, both conventional and regulatory subsets, uncovered gene-, cell subset- and stimulation-specific CREs. Integration with CRISPR knockout screens and assay for transposase-accessible chromatin with sequencing (ATAC-seq) profiling identified trans regulators influencing chromatin states at specific CRISPRi-responsive elements to control costimulatory gene expression. We then discovered a critical CCCTC-binding factor (CTCF) boundary that reinforces CRE interaction with CTLA4 while also preventing promiscuous activation of CD28. By systematically mapping CREs and associated trans regulators directly in primary human T cell subsets, this work overcomes longstanding experimental limitations to decode context-dependent gene regulatory programs in a complex, multi-gene locus critical to immune homeostasis. Functional characterization of the regulatory landscape of the adjacent costimulatory genes CD28, CTLA4 and ICOS in primary human T cell subsets identifies context-dependent programs controlling this locus critical for immune homeostasis.},
  archive      = {J_NG},
  author       = {Mowery, Cody T. and Freimer, Jacob W. and Chen, Zeyu and Casaní-Galdón, Salvador and Umhoefer, Jennifer M. and Arce, Maya M. and Gjoni, Ketrin and Daniel, Bence and Sandor, Katalin and Gowen, Benjamin G. and Nguyen, Vinh and Simeonov, Dimitre R. and Garrido, Christian M. and Curie, Gemma L. and Schmidt, Ralf and Steinhart, Zachary and Satpathy, Ansuman T. and Pollard, Katherine S. and Corn, Jacob E. and Bernstein, Bradley E. and Ye, Chun Jimmie and Marson, Alexander},
  doi          = {10.1038/s41588-024-01743-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1156-1167},
  shortjournal = {Nat. Genet.},
  title        = {Systematic decoding of cis gene regulation defines context-dependent control of the multi-gene costimulatory receptor locus in human t cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Analysis of somatic mutations in whole blood from 200,618
individuals identifies pervasive positive selection and novel drivers of
clonal hematopoiesis. <em>NG</em>, <em>56</em>(6), 1147–1155. (<a
href="https://doi.org/10.1038/s41588-024-01755-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human aging is marked by the emergence of a tapestry of clonal expansions in dividing tissues, particularly evident in blood as clonal hematopoiesis (CH). CH, linked to cancer risk and aging-related phenotypes, often stems from somatic mutations in a set of established genes. However, the majority of clones lack known drivers. Here we infer gene-level positive selection in whole blood exomes from 200,618 individuals in UK Biobank. We identify 17 additional genes, ZBTB33, ZNF318, ZNF234, SPRED2, SH2B3, SRCAP, SIK3, SRSF1, CHEK2, CCDC115, CCL22, BAX, YLPM1, MYD88, MTA2, MAGEC3 and IGLL5, under positive selection at a population level, and validate this selection pattern in 10,837 whole genomes from single-cell-derived hematopoietic colonies. Clones with mutations in these genes grow in frequency and size with age, comparable to classical CH drivers. They correlate with heightened risk of infection, death and hematological malignancy, highlighting the significance of these additional genes in the aging process. Fitness-based analysis of 200,618 UK Biobank exomes and single-cell-derived hematopoietic clones identifies 17 genes under positive selection, including novel drivers of clonal hematopoiesis.},
  archive      = {J_NG},
  author       = {Bernstein, Nicholas and Spencer Chapman, Michael and Nyamondo, Kudzai and Chen, Zhenghao and Williams, Nicholas and Mitchell, Emily and Campbell, Peter J. and Cohen, Robert L. and Nangalia, Jyoti},
  doi          = {10.1038/s41588-024-01755-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1147-1155},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cell-type-specific consequences of mosaic structural
variants in hematopoietic stem and progenitor cells. <em>NG</em>,
<em>56</em>(6), 1134–1146. (<a
href="https://doi.org/10.1038/s41588-024-01754-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The functional impact and cellular context of mosaic structural variants (mSVs) in normal tissues is understudied. Utilizing Strand-seq, we sequenced 1,133 single-cell genomes from 19 human donors of increasing age, and discovered the heterogeneous mSV landscapes of hematopoietic stem and progenitor cells. While mSVs are continuously acquired throughout life, expanded subclones in our cohort are confined to individuals &amp;gt;60. Cells already harboring mSVs are more likely to acquire additional somatic structural variants, including megabase-scale segmental aneuploidies. Capitalizing on comprehensive single-cell micrococcal nuclease digestion with sequencing reference data, we conducted high-resolution cell-typing for eight hematopoietic stem and progenitor cells. Clonally expanded mSVs disrupt normal cellular function by dysregulating diverse cellular pathways, and enriching for myeloid progenitors. Our findings underscore the contribution of mSVs to the cellular and molecular phenotypes associated with the aging hematopoietic system, and establish a foundation for deciphering the molecular links between mSVs, aging and disease susceptibility in normal tissues. This study uses Strand-seq to explore the landscape of mosaic structural variants (mSVs) in human hematopoietic stem and progenitor cells from people of different ages. The analysis highlights patterns of enrichment for mSVs in specific cell types, with associated phenotypes, and suggests that clonal expansions due to mSVs are generally restricted to older individuals.},
  archive      = {J_NG},
  author       = {Grimes, Karen and Jeong, Hyobin and Amoah, Amanda and Xu, Nuo and Niemann, Julian and Raeder, Benjamin and Hasenfeld, Patrick and Stober, Catherine and Rausch, Tobias and Benito, Eva and Jann, Johann-Christoph and Nowak, Daniel and Emini, Ramiz and Hoenicka, Markus and Liebold, Andreas and Ho, Anthony and Shuai, Shimin and Geiger, Hartmut and Sanders, Ashley D. and Korbel, Jan O.},
  doi          = {10.1038/s41588-024-01754-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1134-1146},
  shortjournal = {Nat. Genet.},
  title        = {Cell-type-specific consequences of mosaic structural variants in hematopoietic stem and progenitor cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spatial genomic, biochemical and cellular mechanisms
underlying meningioma heterogeneity and evolution. <em>NG</em>,
<em>56</em>(6), 1121–1133. (<a
href="https://doi.org/10.1038/s41588-024-01747-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intratumor heterogeneity underlies cancer evolution and treatment resistance, but targetable mechanisms driving intratumor heterogeneity are poorly understood. Meningiomas are the most common primary intracranial tumors and are resistant to all medical therapies, and high-grade meningiomas have significant intratumor heterogeneity. Here we use spatial approaches to identify genomic, biochemical and cellular mechanisms linking intratumor heterogeneity to the molecular, temporal and spatial evolution of high-grade meningiomas. We show that divergent intratumor gene and protein expression programs distinguish high-grade meningiomas that are otherwise grouped together by current classification systems. Analyses of matched pairs of primary and recurrent meningiomas reveal spatial expansion of subclonal copy number variants associated with treatment resistance. Multiplexed sequential immunofluorescence and deconvolution of meningioma spatial transcriptomes using cell types from single-cell RNA sequencing show decreased immune infiltration, decreased MAPK signaling, increased PI3K–AKT signaling and increased cell proliferation, which are associated with meningioma recurrence. To translate these findings to preclinical models, we use CRISPR interference and lineage tracing approaches to identify combination therapies that target intratumor heterogeneity in meningioma cell co-cultures. Comprehensive spatial multiomic profiling of high-grade meningiomas identifies intratumor and primary–recurrence subclonal heterogeneity. Cell line models recapitulating intratumor heterogeneity show differential sensitivity in drug screens.},
  archive      = {J_NG},
  author       = {Lucas, Calixto-Hope G. and Mirchia, Kanish and Seo, Kyounghee and Najem, Hinda and Chen, William C. and Zakimi, Naomi and Foster, Kyla and Eaton, Charlotte D. and Cady, Martha A. and Choudhury, Abrar and Liu, S. John and Phillips, Joanna J. and Magill, Stephen T. and Horbinski, Craig M. and Solomon, David A. and Perry, Arie and Vasudevan, Harish N. and Heimberger, Amy B. and Raleigh, David R.},
  doi          = {10.1038/s41588-024-01747-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1121-1133},
  shortjournal = {Nat. Genet.},
  title        = {Spatial genomic, biochemical and cellular mechanisms underlying meningioma heterogeneity and evolution},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cross-ancestry genome-wide association studies of brain
imaging phenotypes. <em>NG</em>, <em>56</em>(6), 1110–1120. (<a
href="https://doi.org/10.1038/s41588-024-01766-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies of brain imaging phenotypes are mainly performed in European populations, but other populations are severely under-represented. Here, we conducted Chinese-alone and cross-ancestry genome-wide association studies of 3,414 brain imaging phenotypes in 7,058 Chinese Han and 33,224 white British participants. We identified 38 new associations in Chinese-alone analyses and 486 additional new associations in cross-ancestry meta-analyses at P &amp;lt; 1.46 × 10−11 for discovery and P &amp;lt; 0.05 for replication. We pooled significant autosomal associations identified by single- or cross-ancestry analyses into 6,443 independent associations, which showed uneven distribution in the genome and the phenotype subgroups. We further divided them into 44 associations with different effect sizes and 3,557 associations with similar effect sizes between ancestries. Loci of these associations were shared with 15 brain-related non-imaging traits including cognition and neuropsychiatric disorders. Our results provide a valuable catalog of genetic associations for brain imaging phenotypes in more diverse populations. Cross-ancestry genome-wide association studies of 3,414 brain imaging phenotypes in Chinese Han and white British participants identify autosomal and X-chromosomal associations in more diverse populations.},
  archive      = {J_NG},
  author       = {Fu, Jilian and Zhang, Quan and Wang, Jianhua and Wang, Meiyun and Zhang, Bing and Zhu, Wenzhen and Qiu, Shijun and Geng, Zuojun and Cui, Guangbin and Yu, Yongqiang and Liao, Weihua and Zhang, Hui and Gao, Bo and Xu, Xiaojun and Han, Tong and Yao, Zhenwei and Qin, Wen and Liu, Feng and Liang, Meng and Wang, Sijia and Xu, Qiang and Xu, Jiayuan and Zhang, Peng and Li, Wei and Shi, Dapeng and Wang, Caihong and Lui, Su and Yan, Zhihan and Chen, Feng and Zhang, Jing and Li, Jiance and Shen, Wen and Miao, Yanwei and Wang, Dawei and Xian, Junfang and Gao, Jia-Hong and Zhang, Xiaochu and Xu, Kai and Zuo, Xi-Nian and Zhang, Longjiang and Ye, Zhaoxiang and Cheng, Jingliang and Li, Mulin Jun and Yu, Chunshui},
  doi          = {10.1038/s41588-024-01766-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1110-1120},
  shortjournal = {Nat. Genet.},
  title        = {Cross-ancestry genome-wide association studies of brain imaging phenotypes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Body mass index stratification optimizes polygenic
prediction of type 2 diabetes in cross-biobank analyses. <em>NG</em>,
<em>56</em>(6), 1100–1109. (<a
href="https://doi.org/10.1038/s41588-024-01782-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Type 2 diabetes (T2D) shows heterogeneous body mass index (BMI) sensitivity. Here, we performed stratification based on BMI to optimize predictions for BMI-related diseases. We obtained BMI-stratified datasets using data from more than 195,000 individuals (nT2D = 55,284) from BioBank Japan (BBJ) and UK Biobank. T2D heritability in the low-BMI group was greater than that in the high-BMI group. Polygenic predictions of T2D toward low-BMI targets had pseudo-R2 values that were more than 22% higher than BMI-unstratified targets. Polygenic risk scores (PRSs) from low-BMI discovery outperformed PRSs from high BMI, while PRSs from BMI-unstratified discovery performed best. Pathway-specific PRSs demonstrated the biological contributions of pathogenic pathways. Low-BMI T2D cases showed higher rates of neuropathy and retinopathy. Combining BMI stratification and a method integrating cross-population effects, T2D predictions showed greater than 37% improvements over unstratified-matched-population prediction. We replicated findings in the Tohoku Medical Megabank (n = 26,000) and the second BBJ cohort (n = 33,096). Our findings suggest that target stratification based on existing traits can improve the polygenic prediction of heterogeneous diseases. Analyses in BioBank Japan, UK Biobank and Tohoku Medical Megabank show that polygenic scores constructed from subgroups stratified by body mass index (BMI) can optimize type 2 diabetes risk prediction, particularly among individuals with low BMI.},
  archive      = {J_NG},
  author       = {Ojima, Takafumi and Namba, Shinichi and Suzuki, Ken and Yamamoto, Kenichi and Sonehara, Kyuto and Narita, Akira and Kamatani, Yoichiro and Tamiya, Gen and Yamamoto, Masayuki and Yamauchi, Toshimasa and Kadowaki, Takashi and Okada, Yukinori},
  doi          = {10.1038/s41588-024-01782-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1100-1109},
  shortjournal = {Nat. Genet.},
  title        = {Body mass index stratification optimizes polygenic prediction of type 2 diabetes in cross-biobank analyses},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide meta-analyses of restless legs syndrome yield
insights into genetic architecture, disease biology and risk prediction.
<em>NG</em>, <em>56</em>(6), 1090–1099. (<a
href="https://doi.org/10.1038/s41588-024-01763-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Restless legs syndrome (RLS) affects up to 10% of older adults. Their healthcare is impeded by delayed diagnosis and insufficient treatment. To advance disease prediction and find new entry points for therapy, we performed meta-analyses of genome-wide association studies in 116,647 individuals with RLS (cases) and 1,546,466 controls of European ancestry. The pooled analysis increased the number of risk loci eightfold to 164, including three on chromosome X. Sex-specific meta-analyses revealed largely overlapping genetic predispositions of the sexes (rg = 0.96). Locus annotation prioritized druggable genes such as glutamate receptors 1 and 4, and Mendelian randomization indicated RLS as a causal risk factor for diabetes. Machine learning approaches combining genetic and nongenetic information performed best in risk prediction (area under the curve (AUC) = 0.82–0.91). In summary, we identified targets for drug development and repurposing, prioritized potential causal relationships between RLS and relevant comorbidities and risk factors for follow-up and provided evidence that nonlinear interactions are likely relevant to RLS risk prediction. Pooled and sex-specific genome-wide association analyses identify new risk loci for restless legs syndrome and candidates for drug repurposing. Machine learning models combining genetic and other information show improved risk prediction performance.},
  archive      = {J_NG},
  author       = {Schormair, Barbara and Zhao, Chen and Bell, Steven and Didriksen, Maria and Nawaz, Muhammad S. and Schandra, Nathalie and Stefani, Ambra and Högl, Birgit and Dauvilliers, Yves and Bachmann, Cornelius G. and Kemlink, David and Sonka, Karel and Paulus, Walter and Trenkwalder, Claudia and Oertel, Wolfgang H. and Hornyak, Magdolna and Teder-Laving, Maris and Metspalu, Andres and Hadjigeorgiou, Georgios M. and Polo, Olli and Fietze, Ingo and Ross, Owen A. and Wszolek, Zbigniew K. and Ibrahim, Abubaker and Bergmann, Melanie and Kittke, Volker and Harrer, Philip and Dowsett, Joseph and Chenini, Sofiene and Ostrowski, Sisse Rye and Sørensen, Erik and Erikstrup, Christian and Pedersen, Ole B. and Topholm Bruun, Mie and Nielsen, Kaspar R. and Butterworth, Adam S. and Soranzo, Nicole and Ouwehand, Willem H. and Roberts, David J. and Danesh, John and Burchell, Brendan and Furlotte, Nicholas A. and Nandakumar, Priyanka and Earley, Christopher J. and Ondo, William G. and Xiong, Lan and Desautels, Alex and Perola, Markus and Vodicka, Pavel and Dina, Christian and Stoll, Monika and Franke, Andre and Lieb, Wolfgang and Stewart, Alexandre F. R. and Shah, Svati H. and Gieger, Christian and Peters, Annette and Rye, David B. and Rouleau, Guy A. and Berger, Klaus and Stefansson, Hreinn and Ullum, Henrik and Stefansson, Kari and Hinds, David A. and Di Angelantonio, Emanuele and Oexle, Konrad and Winkelmann, Juliane},
  doi          = {10.1038/s41588-024-01763-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1090-1099},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide meta-analyses of restless legs syndrome yield insights into genetic architecture, disease biology and risk prediction},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A GGC-repeat expansion in ZFHX3 encoding polyglycine causes
spinocerebellar ataxia type 4 and impairs autophagy. <em>NG</em>,
<em>56</em>(6), 1080–1089. (<a
href="https://doi.org/10.1038/s41588-024-01719-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite linkage to chromosome 16q in 1996, the mutation causing spinocerebellar ataxia type 4 (SCA4), a late-onset sensory and cerebellar ataxia, remained unknown. Here, using long-read single-strand whole-genome sequencing (LR-GS), we identified a heterozygous GGC-repeat expansion in a large Utah pedigree encoding polyglycine (polyG) in zinc finger homeobox protein 3 (ZFHX3), also known as AT-binding transcription factor 1 (ATBF1). We queried 6,495 genome sequencing datasets and identified the repeat expansion in seven additional pedigrees. Ultrarare DNA variants near the repeat expansion indicate a common distant founder event in Sweden. Intranuclear ZFHX3–p62–ubiquitin aggregates were abundant in SCA4 basis pontis neurons. In fibroblasts and induced pluripotent stem cells, the GGC expansion led to increased ZFHX3 protein levels and abnormal autophagy, which were normalized with small interfering RNA-mediated ZFHX3 knockdown in both cell types. Improving autophagy points to a therapeutic avenue for this novel polyG disease. The coding GGC-repeat expansion in an extremely G+C-rich region was not detectable by short-read whole-exome sequencing, which demonstrates the power of LR-GS for variant discovery. Long-read sequencing identifies a GGC-repeat expansion in the coding region of ZFHX3 as the cause of spinocerebellar ataxia type 4. The expansion encodes polyglycine and results in intranuclear aggregates and abnormal autophagy.},
  archive      = {J_NG},
  author       = {Figueroa, Karla P. and Gross, Caspar and Buena-Atienza, Elena and Paul, Sharan and Gandelman, Mandi and Kakar, Naseebullah and Sturm, Marc and Casadei, Nicolas and Admard, Jakob and Park, Joohyun and Zühlke, Christine and Hellenbroich, Yorck and Pozojevic, Jelena and Balachandran, Saranya and Händler, Kristian and Zittel, Simone and Timmann, Dagmar and Erdlenbruch, Friedrich and Herrmann, Laura and Feindt, Thomas and Zenker, Martin and Klopstock, Thomas and Dufke, Claudia and Scoles, Daniel R. and Koeppen, Arnulf and Spielmann, Malte and Riess, Olaf and Ossowski, Stephan and Haack, Tobias B. and Pulst, Stefan M.},
  doi          = {10.1038/s41588-024-01719-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1080-1089},
  shortjournal = {Nat. Genet.},
  title        = {A GGC-repeat expansion in ZFHX3 encoding polyglycine causes spinocerebellar ataxia type 4 and impairs autophagy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Harnessing clonal gametes in hybrid crops to engineer
polyploid genomes. <em>NG</em>, <em>56</em>(6), 1075–1079. (<a
href="https://doi.org/10.1038/s41588-024-01750-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Heterosis boosts crop yield; however, harnessing additional progressive heterosis in polyploids is challenging for breeders. We bioengineered a ‘mitosis instead of meiosis’ (MiMe) system that generates unreduced, clonal gametes in three hybrid tomato genotypes and used it to establish polyploid genome design. Through the hybridization of MiMe hybrids, we generated ‘4-haplotype’ plants that encompassed the complete genetics of their four inbred grandparents, providing a blueprint for exploiting polyploidy in crops. An approach to generate unreduced, clonal gametes in hybrid tomato genotypes enables polyploid genome design through controlled combination of four predefined genome haplotypes, thereby establishing a framework for exploiting progressive heterosis in crops.},
  archive      = {J_NG},
  author       = {Wang, Yazhong and Fuentes, Roven Rommel and van Rengs, Willem M. J. and Effgen, Sieglinde and Zaidan, Mohd Waznul Adly Mohd and Franzen, Rainer and Susanto, Tamara and Fernandes, Joiselle Blanche and Mercier, Raphael and Underwood, Charles J.},
  doi          = {10.1038/s41588-024-01750-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1075-1079},
  shortjournal = {Nat. Genet.},
  title        = {Harnessing clonal gametes in hybrid crops to engineer polyploid genomes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Reply to: Phylogenomic and comparative genomic analyses
support a single evolutionary origin of flatfish asymmetry. <em>NG</em>,
<em>56</em>(6), 1073–1074. (<a
href="https://doi.org/10.1038/s41588-024-01783-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lü, Zhenming and Li, Haorong and Jiang, Hui and Luo, Hairong and Wang, Wen and Kong, Xiaoyu and Li, Yongxin},
  doi          = {10.1038/s41588-024-01783-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1073-1074},
  shortjournal = {Nat. Genet.},
  title        = {Reply to: Phylogenomic and comparative genomic analyses support a single evolutionary origin of flatfish asymmetry},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Phylogenomic and comparative genomic analyses support a
single evolutionary origin of flatfish asymmetry. <em>NG</em>,
<em>56</em>(6), 1069–1072. (<a
href="https://doi.org/10.1038/s41588-024-01784-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Duarte-Ribeiro, Emanuell and Rosas-Puchuri, Ulises and Friedman, Matt and Woodruff, Gavin C. and Hughes, Lily C. and Carpenter, Kent E. and White, William T. and Pogonoski, John J. and Westneat, Mark and Diaz de Astarloa, Juan Martin and Williams, Jeffrey T. and Santos, Mudjekeewis D. and Domínguez-Domínguez, Omar and Ortí, Guillermo and Arcila, Dahiana and Betancur-R, Ricardo},
  doi          = {10.1038/s41588-024-01784-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1069-1072},
  shortjournal = {Nat. Genet.},
  title        = {Phylogenomic and comparative genomic analyses support a single evolutionary origin of flatfish asymmetry},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genotype × environment interactions in gene regulation and
complex traits. <em>NG</em>, <em>56</em>(6), 1057–1068. (<a
href="https://doi.org/10.1038/s41588-024-01776-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genotype × environment interactions (GxE) have long been recognized as a key mechanism underlying human phenotypic variation. Technological developments over the past 15 years have dramatically expanded our appreciation of the role of GxE in both gene regulation and complex traits. The richness and complexity of these datasets also required parallel efforts to develop robust and sensitive statistical and computational approaches. Although our understanding of the genetic architecture of molecular and complex traits has been maturing, a large proportion of complex trait heritability remains unexplained. Furthermore, there are increasing efforts to characterize the effect of environmental exposure on human health. We therefore review GxE in human gene regulation and complex traits, advocating for a comprehensive approach that jointly considers genetic and environmental factors in human health and disease. Genotype × environment interactions are a key mechanism underlying human phenotypic variation and contribute to our understanding of the genetic architecture of human traits, with possible applications in personalized medicine.},
  archive      = {J_NG},
  author       = {Boye, Carly and Nirmalan, Shreya and Ranjbaran, Ali and Luca, Francesca},
  doi          = {10.1038/s41588-024-01776-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1057-1068},
  shortjournal = {Nat. Genet.},
  title        = {Genotype × environment interactions in gene regulation and complex traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Transposition enables low-input single-molecule concurrent
genomics and epigenomics. <em>NG</em>, <em>56</em>(6), 1055–1056. (<a
href="https://doi.org/10.1038/s41588-024-01753-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We adapted Tn5 transposition to reduce the amount of input DNA required for amplification-free single-molecule real-time Pacific Biosciences sequencing. We applied transposition-based library preparation in two sensitive methods for concurrently resolving genomes and epigenomes using limiting amounts of native DNA.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01753-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1055-1056},
  shortjournal = {Nat. Genet.},
  title        = {Transposition enables low-input single-molecule concurrent genomics and epigenomics},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). High-throughput fitness screens link genes to unique
phenotypes in human-restricted salmonella. <em>NG</em>, <em>56</em>(6),
1053–1054. (<a
href="https://doi.org/10.1038/s41588-024-01780-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We used high-throughput transposon screens to examine the responses of generalist and host-adapted Salmonella enterica serovars to 25 stress conditions that recapitulate key stages of human infection. We identified and characterized numerous typhoid-specific gene networks, revealing a role for specific pseudogenes in shaping bacterial fitness outcomes.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01780-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1053-1054},
  shortjournal = {Nat. Genet.},
  title        = {High-throughput fitness screens link genes to unique phenotypes in human-restricted salmonella},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Chromatin modifications integrate cis genomic context to
instruct transcriptional outputs. <em>NG</em>, <em>56</em>(6),
1051–1052. (<a
href="https://doi.org/10.1038/s41588-024-01705-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By developing a modular system for precision epigenome editing, we were able to delineate the causal and quantitative role of chromatin modifications in transcription regulation. The precise effect of chromatin modifications is influenced by multiple contextual factors, including the underlying DNA sequence, transcription factor occupancy and genomic positioning.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01705-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1051-1052},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin modifications integrate cis genomic context to instruct transcriptional outputs},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Novel somatic mutations in blood driving age-related clonal
hematopoiesis. <em>NG</em>, <em>56</em>(6), 1049–1050. (<a
href="https://doi.org/10.1038/s41588-024-01756-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of somatic mutations in blood exomes from 200,618 individuals revealed previously unrecognized genes driving clonal hematopoiesis. Mutations in these genes showed age-dependent clonal expansion and their presence correlated with heightened risks of infection, death and hematological malignancy.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01756-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1049-1050},
  shortjournal = {Nat. Genet.},
  title        = {Novel somatic mutations in blood driving age-related clonal hematopoiesis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unlocking the secrets of structural variants in a healthy
tissue. <em>NG</em>, <em>56</em>(6), 1047–1048. (<a
href="https://doi.org/10.1038/s41588-024-01757-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our study explored cell-specific functional consequences and clonal expansions of mosaic structural variants in distinct hematopoietic stem and progenitor cells by utilizing advanced single-cell sequencing techniques. Our single-cell multi-omics approach paves the way for future studies to focus on the roles of somatic structural variants in aging and disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01757-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1047-1048},
  shortjournal = {Nat. Genet.},
  title        = {Unlocking the secrets of structural variants in a healthy tissue},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Clonal gametes enable polyploid genome design. <em>NG</em>,
<em>56</em>(6), 1045–1046. (<a
href="https://doi.org/10.1038/s41588-024-01751-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many plant products eaten daily in human diets — such as potato or banana — are polyploid and are notoriously difficult to breed. In this study, the fusion of clonal gametes from distinct diploid tomato parents is used as a blueprint for the design of polyploid genomes in crops.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01751-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1045-1046},
  shortjournal = {Nat. Genet.},
  title        = {Clonal gametes enable polyploid genome design},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). How cauliflower got its curd. <em>NG</em>, <em>56</em>(6),
1042–1044. (<a
href="https://doi.org/10.1038/s41588-024-01768-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For a variety of reasons, genetic understanding of the steps leading to domestication of the nutrient-rich edible arrested inflorescence of cauliflower — its curd — has proven relatively intractable. A genomic study now unravels the details.},
  archive      = {J_NG},
  author       = {Fernie, Alisdair R. and Bulut, Mustafa},
  doi          = {10.1038/s41588-024-01768-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1042-1044},
  shortjournal = {Nat. Genet.},
  title        = {How cauliflower got its curd},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An unexpected polyglycine route to spinocerebellar ataxia.
<em>NG</em>, <em>56</em>(6), 1039–1041. (<a
href="https://doi.org/10.1038/s41588-024-01770-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Trinucleotide repeat expansions, notably CAG repeats translated into toxic polyglutamine-containing proteins, are the leading cause of spinocerebellar ataxia (SCA). New work points to a GGC repeat expansion that encodes a polyglycine-containing protein as a cause of SCA4, highlighting polyglycine disorders as an emerging human genetic disease class.},
  archive      = {J_NG},
  author       = {Charlet-Berguerand, Nicolas},
  doi          = {10.1038/s41588-024-01770-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1039-1041},
  shortjournal = {Nat. Genet.},
  title        = {An unexpected polyglycine route to spinocerebellar ataxia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Exploring the origins of pancreatic cancer. <em>NG</em>,
<em>56</em>(6), 1038. (<a
href="https://doi.org/10.1038/s41588-024-01814-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-024-01814-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1038},
  shortjournal = {Nat. Genet.},
  title        = {Exploring the origins of pancreatic cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Library size normalization affects spatial domain
identification. <em>NG</em>, <em>56</em>(6), 1038. (<a
href="https://doi.org/10.1038/s41588-024-01815-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-01815-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1038},
  shortjournal = {Nat. Genet.},
  title        = {Library size normalization affects spatial domain identification},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). RNA editing with CRISPR. <em>NG</em>, <em>56</em>(6), 1038.
(<a href="https://doi.org/10.1038/s41588-024-01816-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-024-01816-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1038},
  shortjournal = {Nat. Genet.},
  title        = {RNA editing with CRISPR},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Metagenomics of the human gut mycobiome. <em>NG</em>,
<em>56</em>(6), 1038. (<a
href="https://doi.org/10.1038/s41588-024-01817-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-024-01817-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1038},
  shortjournal = {Nat. Genet.},
  title        = {Metagenomics of the human gut mycobiome},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The egypt genome project. <em>NG</em>, <em>56</em>(6),
1035–1037. (<a
href="https://doi.org/10.1038/s41588-024-01739-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recently launched Egyptian Genome Project aims to sequence genomic variants of 100,000 apparently healthy Egyptian adults, with around 8,000 individuals suspected to have a genetic disease, as well as 200 ancient Egyptian mummies. The project will provide the first comprehensive genomic dataset from Egypt and North Africa.},
  archive      = {J_NG},
  author       = {Elmonem, Mohamed A. and Soliman, Neveen A. and Moustafa, Ahmed and Gad, Yehia Z. and Hassan, Wael A. and Taha, Tarek and El-Feky, Gina and Sakr, Mahmoud and Amer, Khaled},
  doi          = {10.1038/s41588-024-01739-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1035-1037},
  shortjournal = {Nat. Genet.},
  title        = {The egypt genome project},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Author correction: Cell-type-specific and
disease-associated expression quantitative trait loci in the human lung.
<em>NG</em>, <em>56</em>(5), 1034. (<a
href="https://doi.org/10.1038/s41588-024-01765-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Natri, Heini M. and Del Azodi, Christina B. and Peter, Lance and Taylor, Chase J. and Chugh, Sagrika and Kendle, Robert and Chung, Mei-i and Flaherty, David K. and Matlock, Brittany K. and Calvi, Carla L. and Blackwell, Timothy S. and Ware, Lorraine B. and Bacchetta, Matthew and Walia, Rajat and Shaver, Ciara M. and Kropski, Jonathan A. and McCarthy, Davis J. and Banovich, Nicholas E.},
  doi          = {10.1038/s41588-024-01765-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1034},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Cell-type-specific and disease-associated expression quantitative trait loci in the human lung},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: New genetic signals for lung function
highlight pathways and chronic obstructive pulmonary disease
associations across multiple ancestries. <em>NG</em>, <em>56</em>(5),
1032–1033. (<a
href="https://doi.org/10.1038/s41588-024-01752-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Shrine, Nick and Guyatt, Anna L. and Erzurumluoglu, A. Mesut and Jackson, Victoria E. and Hobbs, Brian D. and Melbourne, Carl A. and Batini, Chiara and Fawcett, Katherine A. and Song, Kijoung and Sakornsakolpat, Phuwanat and Li, Xingnan and Boxall, Ruth and Reeve, Nicola F. and Obeidat, Ma’en and Zhao, Jing Hua and Wielscher, Matthias and Weiss, Stefan and Kentistou, Katherine A. and Cook, James P. and Sun, Benjamin B. and Zhou, Jian and Hui, Jennie and Karrasch, Stefan and Imboden, Medea and Harris, Sarah E. and Marten, Jonathan and Enroth, Stefan and Kerr, Shona M. and Surakka, Ida and Vitart, Veronique and Lehtimäki, Terho and Allen, Richard J. and Bakke, Per S. and Beaty, Terri H. and Bleecker, Eugene R. and Bossé, Yohan and Brandsma, Corry-Anke and Chen, Zhengming and Crapo, James D. and Danesh, John and DeMeo, Dawn L. and Dudbridge, Frank and Ewert, Ralf and Gieger, Christian and Gulsvik, Amund and Hansell, Anna L. and Hao, Ke and Hoffman, Joshua D. and Hokanson, John E. and Homuth, Georg and Joshi, Peter K. and Joubert, Philippe and Langenberg, Claudia and Li, Xuan and Li, Liming and Lin, Kuang and Lind, Lars and Locantore, Nicholas and Luan, Jian’an and Mahajan, Anubha and Maranville, Joseph C. and Murray, Alison and Nickle, David C. and Packer, Richard and Parker, Margaret M. and Paynton, Megan L. and Porteous, David J. and Prokopenko, Dmitry and Qiao, Dandi and Rawal, Rajesh and Runz, Heiko and Sayers, Ian and Sin, Don D. and Smith, Blair H. and Artigas, María Soler and Sparrow, David and Tal-Singer, Ruth and Timmers, Paul R. H. J. and Van den Berge, Maarten and Whittaker, John C. and Woodruff, Prescott G. and Yerges-Armstrong, Laura M. and Troyanskaya, Olga G. and Raitakari, Olli T. and Kähönen, Mika and Polašek, Ozren and Gyllensten, Ulf and Rudan, Igor and Deary, Ian J. and Probst-Hensch, Nicole M. and Schulz, Holger and James, Alan L. and Wilson, James F. and Stubbe, Beate and Zeggini, Eleftheria and Jarvelin, Marjo-Riitta and Wareham, Nick and Silverman, Edwin K. and Hayward, Caroline and Morris, Andrew P. and Butterworth, Adam S. and Scott, Robert A. and Walters, Robin G. and Meyers, Deborah A. and Cho, Michael H. and Strachan, David P. and Hall, Ian P. and Tobin, Martin D. and Wain, Louise V.},
  doi          = {10.1038/s41588-024-01752-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1032-1033},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: New genetic signals for lung function highlight pathways and chronic obstructive pulmonary disease associations across multiple ancestries},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genomes of multicellular algal sisters to land plants
illuminate signaling network evolution. <em>NG</em>, <em>56</em>(5),
1018–1031. (<a
href="https://doi.org/10.1038/s41588-024-01737-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zygnematophyceae are the algal sisters of land plants. Here we sequenced four genomes of filamentous Zygnematophyceae, including chromosome-scale assemblies for three strains of Zygnema circumcarinatum. We inferred traits in the ancestor of Zygnematophyceae and land plants that might have ushered in the conquest of land by plants: expanded genes for signaling cascades, environmental response, and multicellular growth. Zygnematophyceae and land plants share all the major enzymes for cell wall synthesis and remodifications, and gene gains shaped this toolkit. Co-expression network analyses uncover gene cohorts that unite environmental signaling with multicellular developmental programs. Our data shed light on a molecular chassis that balances environmental response and growth modulation across more than 600 million years of streptophyte evolution. Genome assemblies of four filamentous Zygnematophyceae and co-expression network analyses shed light on the evolutionary roots of the mechanism for balancing environmental responses and multicellular growth.},
  archive      = {J_NG},
  author       = {Feng, Xuehuan and Zheng, Jinfang and Irisarri, Iker and Yu, Huihui and Zheng, Bo and Ali, Zahin and de Vries, Sophie and Keller, Jean and Fürst-Jansen, Janine M. R. and Dadras, Armin and Zegers, Jaccoline M. S. and Rieseberg, Tim P. and Dhabalia Ashok, Amra and Darienko, Tatyana and Bierenbroodspot, Maaike J. and Gramzow, Lydia and Petroll, Romy and Haas, Fabian B. and Fernandez-Pozo, Noe and Nousias, Orestis and Li, Tang and Fitzek, Elisabeth and Grayburn, W. Scott and Rittmeier, Nina and Permann, Charlotte and Rümpler, Florian and Archibald, John M. and Theißen, Günter and Mower, Jeffrey P. and Lorenz, Maike and Buschmann, Henrik and von Schwartzenberg, Klaus and Boston, Lori and Hayes, Richard D. and Daum, Chris and Barry, Kerrie and Grigoriev, Igor V. and Wang, Xiyin and Li, Fay-Wei and Rensing, Stefan A. and Ben Ari, Julius and Keren, Noa and Mosquna, Assaf and Holzinger, Andreas and Delaux, Pierre-Marc and Zhang, Chi and Huang, Jinling and Mutwil, Marek and de Vries, Jan and Yin, Yanbin},
  doi          = {10.1038/s41588-024-01737-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1018-1031},
  shortjournal = {Nat. Genet.},
  title        = {Genomes of multicellular algal sisters to land plants illuminate signaling network evolution},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genomic variation in weedy and cultivated broomcorn millet
accessions uncovers the genetic architecture of agronomic traits.
<em>NG</em>, <em>56</em>(5), 1006–1017. (<a
href="https://doi.org/10.1038/s41588-024-01718-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale genomic variations are fundamental resources for crop genetics and breeding. Here we sequenced 1,904 genomes of broomcorn millet to an average of 40× sequencing depth and constructed a comprehensive variation map of weedy and cultivated accessions. Being one of the oldest cultivated crops, broomcorn millet has extremely low nucleotide diversity and remarkably rapid decay of linkage disequilibrium. Genome-wide association studies identified 186 loci for 12 agronomic traits. Many causative candidate genes, such as PmGW8 for grain size and PmLG1 for panicle shape, showed strong selection signatures during domestication. Weedy accessions contained many beneficial variations for the grain traits that are largely lost in cultivated accessions. Weedy and cultivated broomcorn millet have adopted different loci controlling flowering time for regional adaptation in parallel. Our study uncovers the unique population genomic features of broomcorn millet and provides an agronomically important resource for cereal crops. A comprehensive variation map constructed by deep sequencing 1,904 accessions of weedy and cultivated broomcorn millet sheds light on the genetic architecture of agronomic traits during domestication.},
  archive      = {J_NG},
  author       = {Lu, Qiong and Zhao, Hainan and Zhang, Zhengquan and Bai, Yuhe and Zhao, Haiming and Liu, Guoqing and Liu, Minxuan and Zheng, Yunxiao and Zhao, Haiyue and Gong, Huihui and Chen, Lingwei and Deng, Xizhen and Hong, Xiangde and Liu, Tianxiang and Li, Baichuan and Lu, Ping and Wen, Feng and Wang, Lun and Li, Zhijiang and Li, Hai and Li, Haiquan and Zhang, Like and Ma, Wenhui and Liu, Chunqing and Bai, Yan and Xin, Beibei and Chen, Jian and E., Lizhu and Lai, Jinsheng and Song, Weibin},
  doi          = {10.1038/s41588-024-01718-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1006-1017},
  shortjournal = {Nat. Genet.},
  title        = {Genomic variation in weedy and cultivated broomcorn millet accessions uncovers the genetic architecture of agronomic traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Differential selection of yield and quality traits has
shaped genomic signatures of cowpea domestication and improvement.
<em>NG</em>, <em>56</em>(5), 992–1005. (<a
href="https://doi.org/10.1038/s41588-024-01722-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cowpeas (tropical legumes) are important in ensuring food and nutritional security in developing countries, especially in sub-Saharan Africa. Herein, we report two high-quality genome assemblies of grain and vegetable cowpeas and we re-sequenced 344 accessions to characterize the genomic variations landscape. We identified 39 loci for ten important agronomic traits and more than 541 potential loci that underwent selection during cowpea domestication and improvement. In particular, the synchronous selections of the pod-shattering loci and their neighboring stress-relevant loci probably led to the enhancement of pod-shattering resistance and the compromise of stress resistance during the domestication from grain to vegetable cowpeas. Moreover, differential selections on multiple loci associated with pod length, grain number per pod, seed weight, pod and seed soluble sugars, and seed crude proteins shaped the yield and quality diversity in cowpeas. Our findings provide genomic insights into cowpea domestication and improvement footprints, enabling further genome-informed cultivar improvement of cowpeas. High-quality genome assemblies of grain and vegetable cowpeas and re-sequencing of 344 accessions characterize genomic variations between cowpea subspecies and their domestication and improvement under selection.},
  archive      = {J_NG},
  author       = {Wu, Xinyi and Hu, Zhongyuan and Zhang, Yan and Li, Mao and Liao, Nanqiao and Dong, Junyang and Wang, Baogen and Wu, Jian and Wu, Xiaohua and Wang, Ying and Wang, Jian and Lu, Zhongfu and Yang, Yi and Sun, Yuyan and Dong, Wenqi and Zhang, Mingfang and Li, Guojing},
  doi          = {10.1038/s41588-024-01722-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {992-1005},
  shortjournal = {Nat. Genet.},
  title        = {Differential selection of yield and quality traits has shaped genomic signatures of cowpea domestication and improvement},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A pan-genome of 69 arabidopsis thaliana accessions reveals a
conserved genome structure throughout the global species range.
<em>NG</em>, <em>56</em>(5), 982–991. (<a
href="https://doi.org/10.1038/s41588-024-01715-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although originally primarily a system for functional biology, Arabidopsis thaliana has, owing to its broad geographical distribution and adaptation to diverse environments, developed into a powerful model in population genomics. Here we present chromosome-level genome assemblies of 69 accessions from a global species range. We found that genomic colinearity is very conserved, even among geographically and genetically distant accessions. Along chromosome arms, megabase-scale rearrangements are rare and typically present only in a single accession. This indicates that the karyotype is quasi-fixed and that rearrangements in chromosome arms are counter-selected. Centromeric regions display higher structural dynamics, and divergences in core centromeres account for most of the genome size variations. Pan-genome analyses uncovered 32,986 distinct gene families, 60% being present in all accessions and 40% appearing to be dispensable, including 18% private to a single accession, indicating unexplored genic diversity. These 69 new Arabidopsis thaliana genome assemblies will empower future genetic research. A pan-genome of Arabidopsis thaliana constructed using chromosome-level genome assemblies of 69 diverse accessions reveals a conserved genome structure throughout the global species range.},
  archive      = {J_NG},
  author       = {Lian, Qichao and Huettel, Bruno and Walkemeier, Birgit and Mayjonade, Baptiste and Lopez-Roques, Céline and Gil, Lisa and Roux, Fabrice and Schneeberger, Korbinian and Mercier, Raphael},
  doi          = {10.1038/s41588-024-01715-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {982-991},
  shortjournal = {Nat. Genet.},
  title        = {A pan-genome of 69 arabidopsis thaliana accessions reveals a conserved genome structure throughout the global species range},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). New genes helped acorn barnacles adapt to a sessile
lifestyle. <em>NG</em>, <em>56</em>(5), 970–981. (<a
href="https://doi.org/10.1038/s41588-024-01733-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Barnacles are the only sessile lineages among crustaceans, and their sessile life begins with the settlement of swimming larvae (cyprids) and the formation of protective shells. These processes are crucial for adaptation to a sessile lifestyle, but the underlying molecular mechanisms remain poorly understood. While investigating these mechanisms in the acorn barnacle, Amphibalanus amphitrite, we discovered a new gene, bcs-6, which is involved in the energy metabolism of cyprid settlement and originated from a transposon by acquiring the promoter and cis-regulatory element. Unlike mollusks, the barnacle shell comprises alternate layers of chitin and calcite and requires another new gene, bsf, which generates silk-like fibers that efficiently bind chitin and aggregate calcite in the aquatic environment. Our findings highlight the importance of exploring new genes in unique adaptative scenarios, and the results will provide important insights into gene origin and material development. A new acorn barnacles genome assembly, together with transcriptomic and proteomic datasets and functional experiments, identify bcs-6 and bsf, as new genes involved in the settlement process by facilitating energy metabolism and stabilizing the alternating layers of chitin and calcites, respectively.},
  archive      = {J_NG},
  author       = {Han, Zhaofang and Wang, Zhixuan and Rittschof, Daniel and Huang, Zekun and Chen, Liying and Hao, Huanhuan and Yao, Shanshan and Su, Pei and Huang, Miaoqin and Zhang, Yuan-Ye and Ke, Caihuan and Feng, Danqing},
  doi          = {10.1038/s41588-024-01733-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {970-981},
  shortjournal = {Nat. Genet.},
  title        = {New genes helped acorn barnacles adapt to a sessile lifestyle},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A spatiotemporal atlas of mouse liver homeostasis and
regeneration. <em>NG</em>, <em>56</em>(5), 953–969. (<a
href="https://doi.org/10.1038/s41588-024-01709-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanism by which mammalian liver cell responses are coordinated during tissue homeostasis and perturbation is poorly understood, representing a major obstacle in our understanding of many diseases. This knowledge gap is caused by the difficulty involved with studying multiple cell types in different states and locations, particularly when these are transient. We have combined Stereo-seq (spatiotemporal enhanced resolution omics-sequencing) with single-cell transcriptomic profiling of 473,290 cells to generate a high-definition spatiotemporal atlas of mouse liver homeostasis and regeneration at the whole-lobe scale. Our integrative study dissects in detail the molecular gradients controlling liver cell function, systematically defining how gene networks are dynamically modulated through intercellular communication to promote regeneration. Among other important regulators, we identified the transcriptional cofactor TBL1XR1 as a rheostat linking inflammation to Wnt/β-catenin signaling for facilitating hepatocyte proliferation. Our data and analytical pipelines lay the foundation for future high-definition tissue-scale atlases of organ physiology and malfunction. A Stereo-seq and scRNA-seq atlas of mouse liver in homeostasis and regeneration after partial hepatectomy identifies zonated genes, pathways, cell–cell interactions and gene regulatory networks. Functional validation finds that cooperation between TBL1XR1 and β-catenin activates hepatocyte proliferation.},
  archive      = {J_NG},
  author       = {Xu, Jiangshan and Guo, Pengcheng and Hao, Shijie and Shangguan, Shuncheng and Shi, Quan and Volpe, Giacomo and Huang, Keke and Zuo, Jing and An, Juan and Yuan, Yue and Cheng, Mengnan and Deng, Qiuting and Zhang, Xiao and Lai, Guangyao and Nan, Haitao and Wu, Baihua and Shentu, Xinyi and Wu, Liang and Wei, Xiaoyu and Jiang, Yujia and Huang, Xin and Pan, Fengyu and Song, Yumo and Li, Ronghai and Wang, Zhifeng and Liu, Chuanyu and Liu, Shiping and Li, Yuxiang and Yang, Tao and Xu, Zhicheng and Du, Wensi and Li, Ling and Ahmed, Tanveer and You, Kai and Dai, Zhen and Li, Li and Qin, Baoming and Li, Yinxiong and Lai, Liangxue and Qin, Dajiang and Chen, Junling and Fan, Rong and Li, Yongyin and Hou, Jinlin and Ott, Michael and Sharma, Amar Deep and Cantz, Tobias and Schambach, Axel and Kristiansen, Karsten and Hutchins, Andrew P. and Göttgens, Berthold and Maxwell, Patrick H. and Hui, Lijian and Xu, Xun and Liu, Longqi and Chen, Ao and Lai, Yiwei and Esteban, Miguel A.},
  doi          = {10.1038/s41588-024-01709-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {953-969},
  shortjournal = {Nat. Genet.},
  title        = {A spatiotemporal atlas of mouse liver homeostasis and regeneration},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A spatiotemporal atlas of cholestatic injury and repair in
mice. <em>NG</em>, <em>56</em>(5), 938–952. (<a
href="https://doi.org/10.1038/s41588-024-01687-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cholestatic liver injuries, characterized by regional damage around the bile ductular region, lack curative therapies and cause considerable mortality. Here we generated a high-definition spatiotemporal atlas of gene expression during cholestatic injury and repair in mice by integrating spatial enhanced resolution omics sequencing and single-cell transcriptomics. Spatiotemporal analyses revealed a key role of cholangiocyte-driven signaling correlating with the periportal damage-repair response. Cholangiocytes express genes related to recruitment and differentiation of lipid-associated macrophages, which generate feedback signals enhancing ductular reaction. Moreover, cholangiocytes express high TGFβ in association with the conversion of liver progenitor-like cells into cholangiocytes during injury and the dampened proliferation of periportal hepatocytes during recovery. Notably, Atoh8 restricts hepatocyte proliferation during 3,5-diethoxycarbonyl-1,4-dihydro-collidin damage and is quickly downregulated after injury withdrawal, allowing hepatocytes to respond to growth signals. Our findings lay a keystone for in-depth studies of cellular dynamics and molecular mechanisms of cholestatic injuries, which may further develop into therapies for cholangiopathies. A comprehensive time series characterization of a mouse model of cholestatic liver injury with spatial enhanced resolution omics sequencing and single-cell RNA sequencing identifies zonal responses to insult, such as cholangiocyte signaling recruiting lipid-associated macrophages.},
  archive      = {J_NG},
  author       = {Wu, Baihua and Shentu, Xinyi and Nan, Haitao and Guo, Pengcheng and Hao, Shijie and Xu, Jiangshan and Shangguan, Shuncheng and Cui, Lei and Cen, Jin and Deng, Qiuting and Wu, Yan and Liu, Chang and Song, Yumo and Lin, Xiumei and Wang, Zhifeng and Yuan, Yue and Ma, Wen and Li, Ronghai and Li, Yikang and Qian, Qiwei and Du, Wensi and Lai, Tingting and Yang, Tao and Liu, Chuanyu and Ma, Xiong and Chen, Ao and Xu, Xun and Lai, Yiwei and Liu, Longqi and Esteban, Miguel A. and Hui, Lijian},
  doi          = {10.1038/s41588-024-01687-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {938-952},
  shortjournal = {Nat. Genet.},
  title        = {A spatiotemporal atlas of cholestatic injury and repair in mice},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Joint genotypic and phenotypic outcome modeling improves
base editing variant effect quantification. <em>NG</em>, <em>56</em>(5),
925–937. (<a href="https://doi.org/10.1038/s41588-024-01726-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR base editing screens enable analysis of disease-associated variants at scale; however, variable efficiency and precision confounds the assessment of variant-induced phenotypes. Here, we provide an integrated experimental and computational pipeline that improves estimation of variant effects in base editing screens. We use a reporter construct to measure guide RNA (gRNA) editing outcomes alongside their phenotypic consequences and introduce base editor screen analysis with activity normalization (BEAN), a Bayesian network that uses per-guide editing outcomes provided by the reporter and target site chromatin accessibility to estimate variant impacts. BEAN outperforms existing tools in variant effect quantification. We use BEAN to pinpoint common regulatory variants that alter low-density lipoprotein (LDL) uptake, implicating previously unreported genes. Additionally, through saturation base editing of LDLR, we accurately quantify missense variant pathogenicity that is consistent with measurements in UK Biobank patients and identify underlying structural mechanisms. This work provides a widely applicable approach to improve the power of base editing screens for disease-associated variant characterization. BEAN is a Bayesian approach for analyzing base editing screens with improved effect size quantification and variant classification. Applied to low-density lipoprotein (LDL)-associated common variants and saturation base editing of LDLR, BEAN identifies new LDL uptake genes and offers insights into variant structure–pathogenicity mechanisms.},
  archive      = {J_NG},
  author       = {Ryu, Jayoung and Barkal, Sam and Yu, Tian and Jankowiak, Martin and Zhou, Yunzhuo and Francoeur, Matthew and Phan, Quang Vinh and Li, Zhijian and Tognon, Manuel and Brown, Lara and Love, Michael I. and Bhat, Vineel and Lettre, Guillaume and Ascher, David B. and Cassa, Christopher A. and Sherwood, Richard I. and Pinello, Luca},
  doi          = {10.1038/s41588-024-01726-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {925-937},
  shortjournal = {Nat. Genet.},
  title        = {Joint genotypic and phenotypic outcome modeling improves base editing variant effect quantification},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-mitosis dissection of acute and chronic DNA
mutagenesis and repair. <em>NG</em>, <em>56</em>(5), 913–924. (<a
href="https://doi.org/10.1038/s41588-024-01712-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How chronic mutational processes and punctuated bursts of DNA damage drive evolution of the cancer genome is poorly understood. Here, we demonstrate a strategy to disentangle and quantify distinct mechanisms underlying genome evolution in single cells, during single mitoses and at single-strand resolution. To distinguish between chronic (reactive oxygen species (ROS)) and acute (ultraviolet light (UV)) mutagenesis, we microfluidically separate pairs of sister cells from the first mitosis following burst UV damage. Strikingly, UV mutations manifest as sister-specific events, revealing mirror-image mutation phasing genome-wide. In contrast, ROS mutagenesis in transcribed regions is reduced strand agnostically. Successive rounds of genome replication over persisting UV damage drives multiallelic variation at CC dinucleotides. Finally, we show that mutation phasing can be resolved to single strands across the entire genome of liver tumors from F1 mice. This strategy can be broadly used to distinguish the contributions of overlapping cancer relevant mutational processes. A single-cell-based approach allows the daughters of a damaged cell to be separately tracked following single mitotic events. This technique highlights the different ways in which ultraviolet light and reactive oxygen species cause mutagenesis.},
  archive      = {J_NG},
  author       = {Ginno, Paul Adrian and Borgers, Helena and Ernst, Christina and Schneider, Anja and Behm, Mikaela and Aitken, Sarah J. and Taylor, Martin S. and Odom, Duncan T.},
  doi          = {10.1038/s41588-024-01712-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {913-924},
  shortjournal = {Nat. Genet.},
  title        = {Single-mitosis dissection of acute and chronic DNA mutagenesis and repair},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Chromosome evolution screens recapitulate tissue-specific
tumor aneuploidy patterns. <em>NG</em>, <em>56</em>(5), 900–912. (<a
href="https://doi.org/10.1038/s41588-024-01665-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole chromosome and arm-level copy number alterations occur at high frequencies in tumors, but their selective advantages, if any, are poorly understood. Here, utilizing unbiased whole chromosome genetic screens combined with in vitro evolution to generate arm- and subarm-level events, we iteratively selected the fittest karyotypes from aneuploidized human renal and mammary epithelial cells. Proliferation-based karyotype selection in these epithelial lines modeled tissue-specific tumor aneuploidy patterns in patient cohorts in the absence of driver mutations. Hi-C-based translocation mapping revealed that arm-level events usually emerged in multiples of two via centromeric translocations and occurred more frequently in tetraploids than diploids, contributing to the increased diversity in evolving tetraploid populations. Isogenic clonal lineages enabled elucidation of pro-tumorigenic mechanisms associated with common copy number alterations, revealing Notch signaling potentiation as a driver of 1q gain in breast cancer. We propose that intrinsic, tissue-specific proliferative effects underlie tumor copy number patterns in cancer. This study examines karyotypic selection and evolution in vitro using immortalized mammary and kidney epithelial cell lines, observing aneuploidy patterns specific to each origin tissue that are correlated with frequencies in patient tumors and independent of drivers such as TP53 mutation.},
  archive      = {J_NG},
  author       = {Watson, Emma V. and Lee, Jake June-Koo and Gulhan, Doga C. and Melloni, Giorgio E. M. and Venev, Sergey V. and Magesh, Rayna Y. and Frederick, Abdulrazak and Chiba, Kunitoshi and Wooten, Eric C. and Naxerova, Kamila and Dekker, Job and Park, Peter J. and Elledge, Stephen J.},
  doi          = {10.1038/s41588-024-01665-2},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {900-912},
  shortjournal = {Nat. Genet.},
  title        = {Chromosome evolution screens recapitulate tissue-specific tumor aneuploidy patterns},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-cell mtDNA dynamics in tumors is driven by
coregulation of nuclear and mitochondrial genomes. <em>NG</em>,
<em>56</em>(5), 889–899. (<a
href="https://doi.org/10.1038/s41588-024-01724-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The extent of cell-to-cell variation in tumor mitochondrial DNA (mtDNA) copy number and genotype, and the phenotypic and evolutionary consequences of such variation, are poorly characterized. Here we use amplification-free single-cell whole-genome sequencing (Direct Library Prep (DLP+)) to simultaneously assay mtDNA copy number and nuclear DNA (nuDNA) in 72,275 single cells derived from immortalized cell lines, patient-derived xenografts and primary human tumors. Cells typically contained thousands of mtDNA copies, but variation in mtDNA copy number was extensive and strongly associated with cell size. Pervasive whole-genome doubling events in nuDNA associated with stoichiometrically balanced adaptations in mtDNA copy number, implying that mtDNA-to-nuDNA ratio, rather than mtDNA copy number itself, mediated downstream phenotypes. Finally, multimodal analysis of DLP+ and single-cell RNA sequencing identified both somatic loss-of-function and germline noncoding variants in mtDNA linked to heteroplasmy-dependent changes in mtDNA copy number and mitochondrial transcription, revealing phenotypic adaptations to disrupted nuclear/mitochondrial balance. Amplification-free single-cell whole-genome sequencing shows that genomic, evolutionary and biophysical factors collectively drive cell-to-cell variation in mitochondrial DNA copy number.},
  archive      = {J_NG},
  author       = {Kim, Minsoo and Gorelick, Alexander N. and Vàzquez-García, Ignacio and Williams, Marc J. and Salehi, Sohrab and Shi, Hongyu and Weiner, Adam C. and Ceglia, Nick and Funnell, Tyler and Park, Tricia and Boscenco, Sonia and O’Flanagan, Ciara H. and Jiang, Hui and Grewal, Diljot and Tang, Cerise and Rusk, Nicole and Gammage, Payam A. and McPherson, Andrew and Aparicio, Sam and Shah, Sohrab P. and Reznik, Ed},
  doi          = {10.1038/s41588-024-01724-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {889-899},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell mtDNA dynamics in tumors is driven by coregulation of nuclear and mitochondrial genomes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). STR mutations on chromosome 15q cause thyrotropin resistance
by activating a primate-specific enhancer of MIR7-2/MIR1179.
<em>NG</em>, <em>56</em>(5), 877–888. (<a
href="https://doi.org/10.1038/s41588-024-01717-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thyrotropin (TSH) is the master regulator of thyroid gland growth and function. Resistance to TSH (RTSH) describes conditions with reduced sensitivity to TSH. Dominantly inherited RTSH has been linked to a locus on chromosome 15q, but its genetic basis has remained elusive. Here we show that non-coding mutations in a (TTTG)4 short tandem repeat (STR) underlie dominantly inherited RTSH in all 82 affected participants from 12 unrelated families. The STR is contained in a primate-specific Alu retrotransposon with thyroid-specific cis-regulatory chromatin features. Fiber-seq and RNA-seq studies revealed that the mutant STR activates a thyroid-specific enhancer cluster, leading to haplotype-specific upregulation of the bicistronic MIR7-2/MIR1179 locus 35 kb downstream and overexpression of its microRNA products in the participants’ thyrocytes. An imbalance in signaling pathways targeted by these micro-RNAs provides a working model for this cause of RTSH. This finding broadens our current knowledge of genetic defects altering pituitary–thyroid feedback regulation. Short tandem repeat mutations in a primate Alu element on chromosome 15q cause activation of a thyroid-specific enhancer, upregulating MIR7-2/MIR1179. This results in defective thyroid proliferation and thyrotropin resistance.},
  archive      = {J_NG},
  author       = {Grasberger, Helmut and Dumitrescu, Alexandra M. and Liao, Xiao-Hui and Swanson, Elliott G. and Weiss, Roy E. and Srichomkwun, Panudda and Pappa, Theodora and Chen, Junfeng and Yoshimura, Takashi and Hoffmann, Phillip and França, Monica Malheiros and Tagett, Rebecca and Onigata, Kazumichi and Costagliola, Sabine and Ranchalis, Jane and Vollger, Mitchell R. and Stergachis, Andrew B. and Chong, Jessica X. and Bamshad, Michael J. and Smits, Guillaume and Vassart, Gilbert and Refetoff, Samuel},
  doi          = {10.1038/s41588-024-01717-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {877-888},
  shortjournal = {Nat. Genet.},
  title        = {STR mutations on chromosome 15q cause thyrotropin resistance by activating a primate-specific enhancer of MIR7-2/MIR1179},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Functional variants in a TTTG microsatellite on 15q26.1
cause familial nonautoimmune thyroid abnormalities. <em>NG</em>,
<em>56</em>(5), 869–876. (<a
href="https://doi.org/10.1038/s41588-024-01735-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Insufficient thyroid hormone production in newborns is referred to as congenital hypothyroidism. Multinodular goiter (MNG), characterized by an enlarged thyroid gland with multiple nodules, is usually seen in adults and is recognized as a separate disorder from congenital hypothyroidism. Here we performed a linkage analysis of a family with both nongoitrous congenital hypothyroidism and MNG and identified a signal at 15q26.1. Follow-up analyses with whole-genome sequencing and genetic screening in congenital hypothyroidism and MNG cohorts showed that changes in a noncoding TTTG microsatellite on 15q26.1 were frequently observed in congenital hypothyroidism (137 in 989) and MNG (3 in 33) compared with controls (3 in 38,722). Characterization of the noncoding variants with epigenomic data and in vitro experiments suggested that the microsatellite is located in a thyroid-specific transcriptional repressor, and its activity is disrupted by the variants. Collectively, we presented genetic evidence linking nongoitrous congenital hypothyroidism and MNG, providing unique insights into thyroid abnormalities. Noncoding variants in a TTTG microsatellite on 15q26.1 are identified in Japanese patients with childhood and adult-onset thyroid abnormalities. Functional analyses suggest that these variants affect the role of the microsatellite as a potential regulator of thyroid cell growth.},
  archive      = {J_NG},
  author       = {Narumi, Satoshi and Nagasaki, Keisuke and Kiriya, Mitsuo and Uehara, Erika and Akiba, Kazuhisa and Tanase-Nakao, Kanako and Shimura, Kazuhiro and Abe, Kiyomi and Sugisawa, Chiho and Ishii, Tomohiro and Miyako, Kenichi and Hasegawa, Yukihiro and Maruo, Yoshihiro and Muroya, Koji and Watanabe, Natsuko and Nishihara, Eijun and Ito, Yuka and Kogai, Takahiko and Kameyama, Kaori and Nakabayashi, Kazuhiko and Hata, Kenichiro and Fukami, Maki and Shima, Hirohito and Kikuchi, Atsuo and Takayama, Jun and Tamiya, Gen and Hasegawa, Tomonobu},
  doi          = {10.1038/s41588-024-01735-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {869-876},
  shortjournal = {Nat. Genet.},
  title        = {Functional variants in a TTTG microsatellite on 15q26.1 cause familial nonautoimmune thyroid abnormalities},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic modifiers of rare variants in monogenic
developmental disorder loci. <em>NG</em>, <em>56</em>(5), 861–868. (<a
href="https://doi.org/10.1038/s41588-024-01710-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rare damaging variants in a large number of genes are known to cause monogenic developmental disorders (DDs) and have also been shown to cause milder subclinical phenotypes in population cohorts. Here, we show that carrying multiple (2−5) rare damaging variants across 599 dominant DD genes has an additive adverse effect on numerous cognitive and socioeconomic traits in UK Biobank, which can be partially counterbalanced by a higher educational attainment polygenic score (EA-PGS). Phenotypic deviators from expected EA-PGS could be partly explained by the enrichment or depletion of rare DD variants. Among carriers of rare DD variants, those with a DD-related clinical diagnosis had a substantially lower EA-PGS and more severe phenotype than those without a clinical diagnosis. Our results suggest that the overall burden of both rare and common variants can modify the expressivity of a phenotype, which may then influence whether an individual reaches the threshold for clinical disease. Analysis of genetic modifiers of 599 developmental disorder genes in the UK Biobank found that rare variant burden within this set, as well as the common polygenic background, can alter the expressivity of cognitive and socioeconomic traits in an additive manner.},
  archive      = {J_NG},
  author       = {Kingdom, Rebecca and Beaumont, Robin N. and Wood, Andrew R. and Weedon, Michael N. and Wright, Caroline F.},
  doi          = {10.1038/s41588-024-01710-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {861-868},
  shortjournal = {Nat. Genet.},
  title        = {Genetic modifiers of rare variants in monogenic developmental disorder loci},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Analysis of blood methylation quantitative trait loci in
east asians reveals ancestry-specific impacts on complex traits.
<em>NG</em>, <em>56</em>(5), 846–860. (<a
href="https://doi.org/10.1038/s41588-023-01494-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methylation quantitative trait loci (mQTLs) are essential for understanding the role of DNA methylation changes in genetic predisposition, yet they have not been fully characterized in East Asians (EAs). Here we identified mQTLs in whole blood from 3,523 Chinese individuals and replicated them in additional 1,858 Chinese individuals from two cohorts. Over 9% of mQTLs displayed specificity to EAs, facilitating the fine-mapping of EA-specific genetic associations, as shown for variants associated with height. Trans-mQTL hotspots revealed biological pathways contributing to EA-specific genetic associations, including an ERG-mediated 233 trans-mCpG network, implicated in hematopoietic cell differentiation, which likely reflects binding efficiency modulation of the ERG protein complex. More than 90% of mQTLs were shared between different blood cell lineages, with a smaller fraction of lineage-specific mQTLs displaying preferential hypomethylation in the respective lineages. Our study provides new insights into the mQTL landscape across genetic ancestries and their downstream effects on cellular processes and diseases/traits. DNA methylation data of whole blood from Han Chinese individuals are used to map mQTLs, finding that cis- and trans-mQTLs show distinct patterns of East Asians (EA) ancestry-specific colocalization with complex trait variation.},
  archive      = {J_NG},
  author       = {Peng, Qianqian and Liu, Xinxuan and Li, Wenran and Jing, Han and Li, Jiarui and Gao, Xingjian and Luo, Qi and Breeze, Charles E. and Pan, Siyu and Zheng, Qiwen and Li, Guochao and Qian, Jiaqiang and Yuan, Liyun and Yuan, Na and You, Chenglong and Du, Siyuan and Zheng, Yuanting and Yuan, Ziyu and Tan, Jingze and Jia, Peilin and Wang, Jiucun and Zhang, Guoqing and Lu, Xianping and Shi, Leming and Guo, Shicheng and Liu, Yun and Ni, Ting and Wen, Bo and Zeng, Changqing and Jin, Li and Teschendorff, Andrew E. and Liu, Fan and Wang, Sijia},
  doi          = {10.1038/s41588-023-01494-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {846-860},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of blood methylation quantitative trait loci in east asians reveals ancestry-specific impacts on complex traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic mapping across autoimmune diseases reveals shared
associations and mechanisms. <em>NG</em>, <em>56</em>(5), 838–845. (<a
href="https://doi.org/10.1038/s41588-024-01732-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autoimmune and inflammatory diseases are polygenic disorders of the immune system. Many genomic loci harbor risk alleles for several diseases, but the limited resolution of genetic mapping prevents determining whether the same allele is responsible, indicating a shared underlying mechanism. Here, using a collection of 129,058 cases and controls across 6 diseases, we show that ~40% of overlapping associations are due to the same allele. We improve fine-mapping resolution for shared alleles twofold by combining cases and controls across diseases, allowing us to identify more expression quantitative trait loci driven by the shared alleles. The patterns indicate widespread sharing of pathogenic mechanisms but not a single global autoimmune mechanism. Our approach can be applied to any set of traits and is particularly valuable as sample collections become depleted. Joint likelihood mapping across six autoimmune diseases identifies shared and distinct association signals and improves fine-mapping resolution at loci with shared effects, yielding insights into the underlying biological mechanisms.},
  archive      = {J_NG},
  author       = {Lincoln, Matthew R. and Connally, Noah and Axisa, Pierre-Paul and Gasperi, Christiane and Mitrovic, Mitja and van Heel, David and Wijmenga, Cisca and Withoff, Sebo and Jonkers, Iris H. and Padyukov, Leonid and Rich, Stephen S. and Graham, Robert R. and Gaffney, Patrick M. and Langefeld, Carl D. and Vyse, Timothy J. and Hafler, David A. and Chun, Sung and Sunyaev, Shamil R. and Cotsapas, Chris},
  doi          = {10.1038/s41588-024-01732-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {838-845},
  shortjournal = {Nat. Genet.},
  title        = {Genetic mapping across autoimmune diseases reveals shared associations and mechanisms},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integrative common and rare variant analyses provide
insights into the genetic architecture of liver cirrhosis. <em>NG</em>,
<em>56</em>(5), 827–837. (<a
href="https://doi.org/10.1038/s41588-024-01720-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report a multi-ancestry genome-wide association study on liver cirrhosis and its associated endophenotypes, alanine aminotransferase (ALT) and γ-glutamyl transferase. Using data from 12 cohorts, including 18,265 cases with cirrhosis, 1,782,047 controls, up to 1 million individuals with liver function tests and a validation cohort of 21,689 cases and 617,729 controls, we identify and validate 14 risk associations for cirrhosis. Many variants are located near genes involved in hepatic lipid metabolism. One of these, PNPLA3 p.Ile148Met, interacts with alcohol intake, obesity and diabetes on the risk of cirrhosis and hepatocellular carcinoma (HCC). We develop a polygenic risk score that associates with the progression from cirrhosis to HCC. By focusing on prioritized genes from common variant analyses, we find that rare coding variants in GPAM associate with lower ALT, supporting GPAM as a potential target for therapeutic inhibition. In conclusion, this study provides insights into the genetic underpinnings of cirrhosis. A multi-ancestry genome-wide association study of liver cirrhosis and its associated endophenotypes identifies and validates 14 risk variants. Integrative common and rare variant analyses provide insights into the genetic architecture of liver cirrhosis.},
  archive      = {J_NG},
  author       = {Ghouse, Jonas and Sveinbjörnsson, Gardar and Vujkovic, Marijana and Seidelin, Anne-Sofie and Gellert-Kristensen, Helene and Ahlberg, Gustav and Tragante, Vinicius and Rand, Søren A. and Brancale, Joseph and Vilarinho, Silvia and Lundegaard, Pia Rengtved and Sørensen, Erik and Erikstrup, Christian and Bruun, Mie Topholm and Jensen, Bitten Aagaard and Brunak, Søren and Banasik, Karina and Ullum, Henrik and Verweij, Niek and Lotta, Luca and Baras, Aris and Mirshahi, Tooraj and Carey, David J. and Kaplan, David E. and Lynch, Julie and Morgan, Timothy and Schwantes-An, Tae-Hwi and Dochtermann, Daniel R. and Pyarajan, Saiju and Tsao, Philip S. and Laisk, Triin and Mägi, Reedik and Kozlitina, Julia and Tybjærg-Hansen, Anne and Jones, David and Knowlton, Kirk U. and Nadauld, Lincoln and Ferkingstad, Egil and Björnsson, Einar S. and Ulfarsson, Magnus O. and Sturluson, Árni and Sulem, Patrick and Pedersen, Ole B. and Ostrowski, Sisse R. and Gudbjartsson, Daniel F. and Stefansson, Kari and Olesen, Morten Salling and Chang, Kyong-Mi and Holm, Hilma and Bundgaard, Henning and Stender, Stefan},
  doi          = {10.1038/s41588-024-01720-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {827-837},
  shortjournal = {Nat. Genet.},
  title        = {Integrative common and rare variant analyses provide insights into the genetic architecture of liver cirrhosis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide association analyses of breast cancer in women
of african ancestry identify new susceptibility loci and improve risk
prediction. <em>NG</em>, <em>56</em>(5), 819–826. (<a
href="https://doi.org/10.1038/s41588-024-01736-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We performed genome-wide association studies of breast cancer including 18,034 cases and 22,104 controls of African ancestry. Genetic variants at 12 loci were associated with breast cancer risk (P &amp;lt; 5 × 10−8), including associations of a low-frequency missense variant rs61751053 in ARHGEF38 with overall breast cancer (odds ratio (OR) = 1.48) and a common variant rs76664032 at chromosome 2q14.2 with triple-negative breast cancer (TNBC) (OR = 1.30). Approximately 15.4% of cases with TNBC carried six risk alleles in three genome-wide association study-identified TNBC risk variants, with an OR of 4.21 (95% confidence interval = 2.66–7.03) compared with those carrying fewer than two risk alleles. A polygenic risk score (PRS) showed an area under the receiver operating characteristic curve of 0.60 for the prediction of breast cancer risk, which outperformed PRS derived using data from females of European ancestry. Our study markedly increases the population diversity in genetic studies for breast cancer and demonstrates the utility of PRS for risk prediction in females of African ancestry. Genome-wide association analyses identify risk loci for breast cancer in women of African ancestry. Polygenic risk scores derived from these data improve ancestry-specific risk prediction.},
  archive      = {J_NG},
  author       = {Jia, Guochong and Ping, Jie and Guo, Xingyi and Yang, Yaohua and Tao, Ran and Li, Bingshan and Ambs, Stefan and Barnard, Mollie E. and Chen, Yu and Garcia-Closas, Montserrat and Gu, Jian and Hu, Jennifer J. and Huo, Dezheng and John, Esther M. and Li, Christopher I. and Li, James L. and Nathanson, Katherine L. and Nemesure, Barbara and Olopade, Olufunmilayo I. and Pal, Tuya and Press, Michael F. and Sanderson, Maureen and Sandler, Dale P. and Shu, Xiao-Ou and Troester, Melissa A. and Yao, Song and Adejumo, Prisca O. and Ahearn, Thomas and Brewster, Abenaa M. and Hennis, Anselm J. M. and Makumbi, Timothy and Ndom, Paul and O’Brien, Katie M. and Olshan, Andrew F. and Oluwasanu, Mojisola M. and Reid, Sonya and Butler, Ebonee N. and Huang, Maosheng and Ntekim, Atara and Qian, Huijun and Zhang, Haoyu and Ambrosone, Christine B. and Cai, Qiuyin and Long, Jirong and Palmer, Julie R. and Haiman, Christopher A. and Zheng, Wei},
  doi          = {10.1038/s41588-024-01736-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {819-826},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses of breast cancer in women of african ancestry identify new susceptibility loci and improve risk prediction},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multi-ancestry genome-wide association study of kidney
cancer identifies 63 susceptibility regions. <em>NG</em>,
<em>56</em>(5), 809–818. (<a
href="https://doi.org/10.1038/s41588-024-01725-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here, in a multi-ancestry genome-wide association study meta-analysis of kidney cancer (29,020 cases and 835,670 controls), we identified 63 susceptibility regions (50 novel) containing 108 independent risk loci. In analyses stratified by subtype, 52 regions (78 loci) were associated with clear cell renal cell carcinoma (RCC) and 6 regions (7 loci) with papillary RCC. Notably, we report a variant common in African ancestry individuals ( rs7629500 ) in the 3′ untranslated region of VHL, nearly tripling clear cell RCC risk (odds ratio 2.72, 95% confidence interval 2.23–3.30). In cis-expression quantitative trait locus analyses, 48 variants from 34 regions point toward 83 candidate genes. Enrichment of hypoxia-inducible factor-binding sites underscores the importance of hypoxia-related mechanisms in kidney cancer. Our results advance understanding of the genetic architecture of kidney cancer, provide clues for functional investigation and enable generation of a validated polygenic risk score with an estimated area under the curve of 0.65 (0.74 including risk factors) among European ancestry individuals. A multi-ancestry genome-wide association meta-analysis of kidney cancer identifies 63 regions associated with disease susceptibility including one locus that was associated with increased risk in individuals with African ancestry.},
  archive      = {J_NG},
  author       = {Purdue, Mark P. and Dutta, Diptavo and Machiela, Mitchell J. and Gorman, Bryan R. and Winter, Timothy and Okuhara, Dayne and Cleland, Sara and Ferreiro-Iglesias, Aida and Scheet, Paul and Liu, Aoxing and Wu, Chao and Antwi, Samuel O. and Larkin, James and Zequi, Stênio C. and Sun, Maxine and Hikino, Keiko and Hajiran, Ali and Lawson, Keith A. and Cárcano, Flavio and Blanchet, Odile and Shuch, Brian and Nepple, Kenneth G. and Margue, Gaëlle and Sundi, Debasish and Diver, W. Ryan and Folgueira, Maria A. A. K. and van Bokhoven, Adrie and Neffa, Florencia and Brown, Kevin M. and Hofmann, Jonathan N. and Rhee, Jongeun and Yeager, Meredith and Cole, Nathan R. and Hicks, Belynda D. and Manning, Michelle R. and Hutchinson, Amy A. and Rothman, Nathaniel and Huang, Wen-Yi and Linehan, W. Marston and Lori, Adriana and Ferragu, Matthieu and Zidane-Marinnes, Merzouka and Serrano, Sérgio V. and Magnabosco, Wesley J. and Vilas, Ana and Decia, Ricardo and Carusso, Florencia and Graham, Laura S. and Anderson, Kyra and Bilen, Mehmet A. and Arciero, Cletus and Pellegrin, Isabelle and Ricard, Solène and Scelo, Ghislaine and Banks, Rosamonde E. and Vasudev, Naveen S. and Soomro, Naeem and Stewart, Grant D. and Adeyoju, Adebanji and Bromage, Stephen and Hrouda, David and Gibbons, Norma and Patel, Poulam and Sullivan, Mark and Protheroe, Andrew and Nugent, Francesca I. and Fournier, Michelle J. and Zhang, Xiaoyu and Martin, Lisa J. and Komisarenko, Maria and Eisen, Timothy and Cunningham, Sonia A. and Connolly, Denise C. and Uzzo, Robert G. and Zaridze, David and Mukeria, Anush and Holcatova, Ivana and Hornakova, Anna and Foretova, Lenka and Janout, Vladimir and Mates, Dana and Jinga, Viorel and Rascu, Stefan and Mijuskovic, Mirjana and Savic, Slavisa and Milosavljevic, Sasa and Gaborieau, Valérie and Abedi-Ardekani, Behnoush and McKay, James and Johansson, Mattias and Phouthavongsy, Larry and Hayman, Lindsay and Li, Jason and Lungu, Ilinca and Bezerra, Stephania M. and Souza, Aline G. and Sares, Claudia T. G. and Reis, Rodolfo B. and Gallucci, Fabio P. and Cordeiro, Mauricio D. and Pomerantz, Mark and Lee, Gwo-Shu M. and Freedman, Matthew L. and Jeong, Anhyo and Greenberg, Samantha E. and Sanchez, Alejandro and Thompson, R. Houston and Sharma, Vidit and Thiel, David D. and Ball, Colleen T. and Abreu, Diego and Lam, Elaine T. and Nahas, William C. and Master, Viraj A. and Patel, Alpa V. and Bernhard, Jean-Christophe and Freedman, Neal D. and Bigot, Pierre and Reis, Rui M. and Colli, Leandro M. and Finelli, Antonio and Manley, Brandon J. and Terao, Chikashi and Choueiri, Toni K. and Carraro, Dirce M. and Houlston, Richard and Eckel-Passow, Jeanette E. and Abbosh, Philip H. and Ganna, Andrea and Brennan, Paul and Gu, Jian and Chanock, Stephen J.},
  doi          = {10.1038/s41588-024-01725-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {809-818},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genome-wide association study of kidney cancer identifies 63 susceptibility regions},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide association analyses identify 95 risk loci and
provide insights into the neurobiology of post-traumatic stress
disorder. <em>NG</em>, <em>56</em>(5), 792–808. (<a
href="https://doi.org/10.1038/s41588-024-01707-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-traumatic stress disorder (PTSD) genetics are characterized by lower discoverability than most other psychiatric disorders. The contribution to biological understanding from previous genetic studies has thus been limited. We performed a multi-ancestry meta-analysis of genome-wide association studies across 1,222,882 individuals of European ancestry (137,136 cases) and 58,051 admixed individuals with African and Native American ancestry (13,624 cases). We identified 95 genome-wide significant loci (80 new). Convergent multi-omic approaches identified 43 potential causal genes, broadly classified as neurotransmitter and ion channel synaptic modulators (for example, GRIA1, GRM8 and CACNA1E), developmental, axon guidance and transcription factors (for example, FOXP2, EFNA5 and DCC), synaptic structure and function genes (for example, PCLO, NCAM1 and PDE4B) and endocrine or immune regulators (for example, ESR1, TRAF3 and TANK). Additional top genes influence stress, immune, fear and threat-related processes, previously hypothesized to underlie PTSD neurobiology. These findings strengthen our understanding of neurobiological systems relevant to PTSD pathophysiology, while also opening new areas for investigation. Multi-ancestry genome-wide analyses identify 95 loci associated with post-traumatic stress disorder and implicate candidate genes, pathways and neurobiological systems underlying its pathophysiology.},
  archive      = {J_NG},
  author       = {Nievergelt, Caroline M. and Maihofer, Adam X. and Atkinson, Elizabeth G. and Chen, Chia-Yen and Choi, Karmel W. and Coleman, Jonathan R. I. and Daskalakis, Nikolaos P. and Duncan, Laramie E. and Polimanti, Renato and Aaronson, Cindy and Amstadter, Ananda B. and Andersen, Soren B. and Andreassen, Ole A. and Arbisi, Paul A. and Ashley-Koch, Allison E. and Austin, S. Bryn and Avdibegoviç, Esmina and Babić, Dragan and Bacanu, Silviu-Alin and Baker, Dewleen G. and Batzler, Anthony and Beckham, Jean C. and Belangero, Sintia and Benjet, Corina and Bergner, Carisa and Bierer, Linda M. and Biernacka, Joanna M. and Bierut, Laura J. and Bisson, Jonathan I. and Boks, Marco P. and Bolger, Elizabeth A. and Brandolino, Amber and Breen, Gerome and Bressan, Rodrigo Affonseca and Bryant, Richard A. and Bustamante, Angela C. and Bybjerg-Grauholm, Jonas and Bækvad-Hansen, Marie and Børglum, Anders D. and Børte, Sigrid and Cahn, Leah and Calabrese, Joseph R. and Caldas-de-Almeida, Jose Miguel and Chatzinakos, Chris and Cheema, Sheraz and Clouston, Sean A. P. and Colodro-Conde, Lucía and Coombes, Brandon J. and Cruz-Fuentes, Carlos S. and Dale, Anders M. and Dalvie, Shareefa and Davis, Lea K. and Deckert, Jürgen and Delahanty, Douglas L. and Dennis, Michelle F. and Desarnaud, Frank and DiPietro, Christopher P. and Disner, Seth G. and Docherty, Anna R. and Domschke, Katharina and Dyb, Grete and Kulenović, Alma Džubur and Edenberg, Howard J. and Evans, Alexandra and Fabbri, Chiara and Fani, Negar and Farrer, Lindsay A. and Feder, Adriana and Feeny, Norah C. and Flory, Janine D. and Forbes, David and Franz, Carol E. and Galea, Sandro and Garrett, Melanie E. and Gelaye, Bizu and Gelernter, Joel and Geuze, Elbert and Gillespie, Charles F. and Goleva, Slavina B. and Gordon, Scott D. and Goçi, Aferdita and Grasser, Lana Ruvolo and Guindalini, Camila and Haas, Magali and Hagenaars, Saskia and Hauser, Michael A. and Heath, Andrew C. and Hemmings, Sian M. J. and Hesselbrock, Victor and Hickie, Ian B. and Hogan, Kelleigh and Hougaard, David Michael and Huang, Hailiang and Huckins, Laura M. and Hveem, Kristian and Jakovljević, Miro and Javanbakht, Arash and Jenkins, Gregory D. and Johnson, Jessica and Jones, Ian and Jovanovic, Tanja and Karstoft, Karen-Inge and Kaufman, Milissa L. and Kennedy, James L. and Kessler, Ronald C. and Khan, Alaptagin and Kimbrel, Nathan A. and King, Anthony P. and Koen, Nastassja and Kotov, Roman and Kranzler, Henry R. and Krebs, Kristi and Kremen, William S. and Kuan, Pei-Fen and Lawford, Bruce R. and Lebois, Lauren A. M. and Lehto, Kelli and Levey, Daniel F. and Lewis, Catrin and Liberzon, Israel and Linnstaedt, Sarah D. and Logue, Mark W. and Lori, Adriana and Lu, Yi and Luft, Benjamin J. and Lupton, Michelle K. and Luykx, Jurjen J. and Makotkine, Iouri and Maples-Keller, Jessica L. and Marchese, Shelby and Marmar, Charles and Martin, Nicholas G. and Martínez-Levy, Gabriela A. and McAloney, Kerrie and McFarlane, Alexander and McLaughlin, Katie A. and McLean, Samuel A. and Medland, Sarah E. and Mehta, Divya and Meyers, Jacquelyn and Michopoulos, Vasiliki and Mikita, Elizabeth A. and Milani, Lili and Milberg, William and Miller, Mark W. and Morey, Rajendra A. and Morris, Charles Phillip and Mors, Ole and Mortensen, Preben Bo and Mufford, Mary S. and Nelson, Elliot C. and Nordentoft, Merete and Norman, Sonya B. and Nugent, Nicole R. and O’Donnell, Meaghan and Orcutt, Holly K. and Pan, Pedro M. and Panizzon, Matthew S. and Pathak, Gita A. and Peters, Edward S. and Peterson, Alan L. and Peverill, Matthew and Pietrzak, Robert H. and Polusny, Melissa A. and Porjesz, Bernice and Powers, Abigail and Qin, Xue-Jun and Ratanatharathorn, Andrew and Risbrough, Victoria B. and Roberts, Andrea L. and Rothbaum, Alex O. and Rothbaum, Barbara O. and Roy-Byrne, Peter and Ruggiero, Kenneth J. and Rung, Ariane and Runz, Heiko and Rutten, Bart P. F. and de Viteri, Stacey Saenz and Salum, Giovanni Abrahão and Sampson, Laura and Sanchez, Sixto E. and Santoro, Marcos and Seah, Carina and Seedat, Soraya and Seng, Julia S. and Shabalin, Andrey and Sheerin, Christina M. and Silove, Derrick and Smith, Alicia K. and Smoller, Jordan W. and Sponheim, Scott R. and Stein, Dan J. and Stensland, Synne and Stevens, Jennifer S. and Sumner, Jennifer A. and Teicher, Martin H. and Thompson, Wesley K. and Tiwari, Arun K. and Trapido, Edward and Uddin, Monica and Ursano, Robert J. and Valdimarsdóttir, Unnur and Van Hooff, Miranda and Vermetten, Eric and Vinkers, Christiaan H. and Voisey, Joanne and Wang, Yunpeng and Wang, Zhewu and Waszczuk, Monika and Weber, Heike and Wendt, Frank R. and Werge, Thomas and Williams, Michelle A. and Williamson, Douglas E. and Winsvold, Bendik S. and Winternitz, Sherry and Wolf, Christiane and Wolf, Erika J. and Xia, Yan and Xiong, Ying and Yehuda, Rachel and Young, Keith A. and Young, Ross McD and Zai, Clement C. and Zai, Gwyneth C. and Zervas, Mark and Zhao, Hongyu and Zoellner, Lori A. and Zwart, John-Anker and deRoon-Cassini, Terri and van Rooij, Sanne J. H. and van den Heuvel, Leigh L. and Stein, Murray B. and Ressler, Kerry J. and Koenen, Karestan C.},
  doi          = {10.1038/s41588-024-01707-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {792-808},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses identify 95 risk loci and provide insights into the neurobiology of post-traumatic stress disorder},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide analysis in over 1 million individuals of
european ancestry yields improved polygenic risk scores for blood
pressure traits. <em>NG</em>, <em>56</em>(5), 778–791. (<a
href="https://doi.org/10.1038/s41588-024-01714-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertension affects more than one billion people worldwide. Here we identify 113 novel loci, reporting a total of 2,103 independent genetic signals (P &amp;lt; 5 × 10−8) from the largest single-stage blood pressure (BP) genome-wide association study to date (n = 1,028,980 European individuals). These associations explain more than 60% of single nucleotide polymorphism-based BP heritability. Comparing top versus bottom deciles of polygenic risk scores (PRSs) reveals clinically meaningful differences in BP (16.9 mmHg systolic BP, 95% CI, 15.5–18.2 mmHg, P = 2.22 × 10−126) and more than a sevenfold higher odds of hypertension risk (odds ratio, 7.33; 95% CI, 5.54–9.70; P = 4.13 × 10−44) in an independent dataset. Adding PRS into hypertension-prediction models increased the area under the receiver operating characteristic curve (AUROC) from 0.791 (95% CI, 0.781–0.801) to 0.826 (95% CI, 0.817–0.836, ∆AUROC, 0.035, P = 1.98 × 10−34). We compare the 2,103 loci results in non-European ancestries and show significant PRS associations in a large African-American sample. Secondary analyses implicate 500 genes previously unreported for BP. Our study highlights the role of increasingly large genomic studies for precision health research. Genome-wide association analysis in over one million individuals of European ancestry identifies 2,103 independent genetic signals (including 113 new loci) associated with blood pressure traits.},
  archive      = {J_NG},
  author       = {Keaton, Jacob M. and Kamali, Zoha and Xie, Tian and Vaez, Ahmad and Williams, Ariel and Goleva, Slavina B. and Ani, Alireza and Evangelou, Evangelos and Hellwege, Jacklyn N. and Yengo, Loic and Young, William J. and Traylor, Matthew and Giri, Ayush and Zheng, Zhili and Zeng, Jian and Chasman, Daniel I. and Morris, Andrew P. and Caulfield, Mark J. and Hwang, Shih-Jen and Kooner, Jaspal S. and Conen, David and Attia, John R. and Morrison, Alanna C. and Loos, Ruth J. F. and Kristiansson, Kati and Schmidt, Reinhold and Hicks, Andrew A. and Pramstaller, Peter P. and Nelson, Christopher P. and Samani, Nilesh J. and Risch, Lorenz and Gyllensten, Ulf and Melander, Olle and Riese, Harriette and Wilson, James F. and Campbell, Harry and Rich, Stephen S. and Psaty, Bruce M. and Lu, Yingchang and Rotter, Jerome I. and Guo, Xiuqing and Rice, Kenneth M. and Vollenweider, Peter and Sundström, Johan and Langenberg, Claudia and Tobin, Martin D. and Giedraitis, Vilmantas and Luan, Jian’an and Tuomilehto, Jaakko and Kutalik, Zoltan and Ripatti, Samuli and Salomaa, Veikko and Girotto, Giorgia and Trompet, Stella and Jukema, J. Wouter and van der Harst, Pim and Ridker, Paul M. and Giulianini, Franco and Vitart, Veronique and Goel, Anuj and Watkins, Hugh and Harris, Sarah E. and Deary, Ian J. and van der Most, Peter J. and Oldehinkel, Albertine J. and Keavney, Bernard D. and Hayward, Caroline and Campbell, Archie and Boehnke, Michael and Scott, Laura J. and Boutin, Thibaud and Mamasoula, Chrysovalanto and Järvelin, Marjo-Riitta and Peters, Annette and Gieger, Christian and Lakatta, Edward G. and Cucca, Francesco and Hui, Jennie and Knekt, Paul and Enroth, Stefan and De Borst, Martin H. and Polašek, Ozren and Concas, Maria Pina and Catamo, Eulalia and Cocca, Massimiliano and Li-Gao, Ruifang and Hofer, Edith and Schmidt, Helena and Spedicati, Beatrice and Waldenberger, Melanie and Strachan, David P. and Laan, Maris and Teumer, Alexander and Dörr, Marcus and Gudnason, Vilmundur and Cook, James P. and Ruggiero, Daniela and Kolcic, Ivana and Boerwinkle, Eric and Traglia, Michela and Lehtimäki, Terho and Raitakari, Olli T. and Johnson, Andrew D. and Newton-Cheh, Christopher and Brown, Morris J. and Dominiczak, Anna F. and Sever, Peter J. and Poulter, Neil and Chambers, John C. and Elosua, Roberto and Siscovick, David and Esko, Tõnu and Metspalu, Andres and Strawbridge, Rona J. and Laakso, Markku and Hamsten, Anders and Hottenga, Jouke-Jan and de Geus, Eco and Morris, Andrew D. and Palmer, Colin N. A. and Nolte, Ilja M. and Milaneschi, Yuri and Marten, Jonathan and Wright, Alan and Zeggini, Eleftheria and Howson, Joanna M. M. and O’Donnell, Christopher J. and Spector, Tim and Nalls, Mike A. and Simonsick, Eleanor M. and Liu, Yongmei and van Duijn, Cornelia M. and Butterworth, Adam S. and Danesh, John N. and Menni, Cristina and Wareham, Nicholas J. and Khaw, Kay-Tee and Sun, Yan V. and Wilson, Peter W. F. and Cho, Kelly and Visscher, Peter M. and Denny, Joshua C. and Levy, Daniel and Edwards, Todd L. and Munroe, Patricia B. and Snieder, Harold and Warren, Helen R.},
  doi          = {10.1038/s41588-024-01714-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {778-791},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analysis in over 1 million individuals of european ancestry yields improved polygenic risk scores for blood pressure traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Leveraging functional genomic annotations and genome
coverage to improve polygenic prediction of complex traits within and
between ancestries. <em>NG</em>, <em>56</em>(5), 767–777. (<a
href="https://doi.org/10.1038/s41588-024-01704-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We develop a method, SBayesRC, that integrates genome-wide association study (GWAS) summary statistics with functional genomic annotations to improve polygenic prediction of complex traits. Our method is scalable to whole-genome variant analysis and refines signals from functional annotations by allowing them to affect both causal variant probability and causal effect distribution. We analyze 50 complex traits and diseases using ∼7 million common single-nucleotide polymorphisms (SNPs) and 96 annotations. SBayesRC improves prediction accuracy by 14% in European ancestry and up to 34% in cross-ancestry prediction compared to the baseline method SBayesR, which does not use annotations, and outperforms other methods, including LDpred2, LDpred-funct, MegaPRS, PolyPred-S and PRS-CSx. Investigation of factors affecting prediction accuracy identifies a significant interaction between SNP density and annotation information, suggesting whole-genome sequence variants with annotations may further improve prediction. Functional partitioning analysis highlights a major contribution of evolutionary constrained regions to prediction accuracy and the largest per-SNP contribution from nonsynonymous SNPs. SBayesRC integrates genome-wide association summary statistics with functional annotations to improve polygenic prediction of complex traits. Functional partitioning highlights a major contribution of evolutionarily constrained regions to prediction accuracy.},
  archive      = {J_NG},
  author       = {Zheng, Zhili and Liu, Shouye and Sidorenko, Julia and Wang, Ying and Lin, Tian and Yengo, Loic and Turley, Patrick and Ani, Alireza and Wang, Rujia and Nolte, Ilja M. and Snieder, Harold and Yang, Jian and Wray, Naomi R. and Goddard, Michael E. and Visscher, Peter M. and Zeng, Jian},
  doi          = {10.1038/s41588-024-01704-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {767-777},
  shortjournal = {Nat. Genet.},
  title        = {Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integrating population genetics, stem cell biology and
cellular genomics to study complex human diseases. <em>NG</em>,
<em>56</em>(5), 758–766. (<a
href="https://doi.org/10.1038/s41588-024-01731-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human pluripotent stem (hPS) cells can, in theory, be differentiated into any cell type, making them a powerful in vitro model for human biology. Recent technological advances have facilitated large-scale hPS cell studies that allow investigation of the genetic regulation of molecular phenotypes and their contribution to high-order phenotypes such as human disease. Integrating hPS cells with single-cell sequencing makes identifying context-dependent genetic effects during cell development or upon experimental manipulation possible. Here we discuss how the intersection of stem cell biology, population genetics and cellular genomics can help resolve the functional consequences of human genetic variation. We examine the critical challenges of integrating these fields and approaches to scaling them cost-effectively and practically. We highlight two areas of human biology that can particularly benefit from population-scale hPS cell studies, elucidating mechanisms underlying complex disease risk loci and evaluating relationships between common genetic variation and pharmacotherapeutic phenotypes. Integrating human pluripotent stem cell models with population genetics and cellular genomics can help elucidate functional mechanisms underlying complex disease risk loci and uncover relationships between common genetic variation and pharmacotherapeutic phenotypes.},
  archive      = {J_NG},
  author       = {Farbehi, Nona and Neavin, Drew R. and Cuomo, Anna S. E. and Studer, Lorenz and MacArthur, Daniel G. and Powell, Joseph E.},
  doi          = {10.1038/s41588-024-01731-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {758-766},
  shortjournal = {Nat. Genet.},
  title        = {Integrating population genetics, stem cell biology and cellular genomics to study complex human diseases},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Advancing genomics to improve health equity. <em>NG</em>,
<em>56</em>(5), 752–757. (<a
href="https://doi.org/10.1038/s41588-024-01711-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Health equity is the state in which everyone has fair and just opportunities to attain their highest level of health. The field of human genomics has fallen short in increasing health equity, largely because the diversity of the human population has been inadequately reflected among participants of genomics research. This lack of diversity leads to disparities that can have scientific and clinical consequences. Achieving health equity related to genomics will require greater effort in addressing inequities within the field. As part of the commitment of the National Human Genome Research Institute (NHGRI) to advancing health equity, it convened experts in genomics and health equity research to make recommendations and performed a review of current literature to identify the landscape of gaps and opportunities at the interface between human genomics and health equity research. This Perspective describes these findings and examines health equity within the context of human genomics and genomic medicine. This Perspective discusses how addressing disparities in genomics can drive improvements in health equity more broadly.},
  archive      = {J_NG},
  author       = {Madden, Ebony B. and Hindorff, Lucia A. and Bonham, Vence L. and Akintobi, Tabia Henry and Burchard, Esteban G. and Baker, Kellan E. and Begay, Rene L. and Carpten, John D. and Cox, Nancy J. and Di Francesco, Valentina and Dillard, Denise A. and Fletcher, Faith E. and Fullerton, Stephanie M. and Garrison, Nanibaa’ A. and Hammack-Aviran, Catherine M. and Hiratsuka, Vanessa Y. and Hildreth, James E. K. and Horowitz, Carol R. and Hughes Halbert, Chanita A. and Inouye, Michael and Jackson, Amber and Landry, Latrice G. and Kittles, Rick A. and Leek, Jeff T. and Limdi, Nita A. and Lockhart, Nicole C. and Ofili, Elizabeth O. and Pérez-Stable, Eliseo J. and Sabatello, Maya and Saulsberry, Loren and Schools, Lorjetta E. and Troyer, Jennifer L. and Wilfond, Benjamin S. and Wojcik, Genevieve L. and Cho, Judy H. and Lee, Sandra S.-J. and Green, Eric D.},
  doi          = {10.1038/s41588-024-01711-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {752-757},
  shortjournal = {Nat. Genet.},
  title        = {Advancing genomics to improve health equity},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). New genes are involved in the adaptation of acorn barnacles
to sessile life. <em>NG</em>, <em>56</em>(5), 750–751. (<a
href="https://doi.org/10.1038/s41588-024-01734-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We construct a chromosome-level genome assembly of the acorn barnacle Amphibalanus amphitrite. Using this genome together with multi-omics datasets and functional evidence, we reveal the evolution and function of two lineage-specific genes, bcs-6 and bsf, that help A. amphitrite adapt to a sessile lifestyle.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01734-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {750-751},
  shortjournal = {Nat. Genet.},
  title        = {New genes are involved in the adaptation of acorn barnacles to sessile life},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Constants and variations in 69 diverse genomes of
arabidopsis thaliana. <em>NG</em>, <em>56</em>(5), 748–749. (<a
href="https://doi.org/10.1038/s41588-024-01716-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosome-level genome sequences of 69 diverse Arabidopsis thaliana strains reveal a quasi-fixed genome structure worldwide, in which large rearrangement is limited almost exclusively to the centromeric regions. Pan-genome analysis uncovered substantial diversity in gene content that, together with the genome assemblies, will fuel future genetic research.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01716-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {748-749},
  shortjournal = {Nat. Genet.},
  title        = {Constants and variations in 69 diverse genomes of arabidopsis thaliana},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Finding the lost sister: Mirror-image mutation phasing in
mammalian mitotic sister cells. <em>NG</em>, <em>56</em>(5), 746–747.
(<a href="https://doi.org/10.1038/s41588-024-01713-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study establishes in vitro and in vivo systems to study mutagenesis and cancer genome evolution. Using microfluidics and mouse models, the method enables the dissection of co-occurring mutational processes and reveals that acute damage results in mirror-image mutation phasing between sister cells after a single mitosis.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01713-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {746-747},
  shortjournal = {Nat. Genet.},
  title        = {Finding the lost sister: Mirror-image mutation phasing in mammalian mitotic sister cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Experimental evolution of cancer chromosomal changes.
<em>NG</em>, <em>56</em>(5), 743–745. (<a
href="https://doi.org/10.1038/s41588-024-01742-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimental genome evolution of normal human cells reveals an intrinsic propensity to develop cancer-associated chromosomal alterations.},
  archive      = {J_NG},
  author       = {Guscott, Molly A. and McClelland, Sarah E.},
  doi          = {10.1038/s41588-024-01742-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {743-745},
  shortjournal = {Nat. Genet.},
  title        = {Experimental evolution of cancer chromosomal changes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Understanding liver repair through space and time.
<em>NG</em>, <em>56</em>(5), 740–742. (<a
href="https://doi.org/10.1038/s41588-024-01741-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By integrating spatially resolved single-cell RNA sequencing (scRNA-seq) and Stereo-seq data, two studies in this issue characterize molecular signatures of liver cell types and their interactions in homeostasis, damage, repair and regeneration.},
  archive      = {J_NG},
  author       = {Belicova, Lenka and Van Hul, Noemi and Andersson, Emma R.},
  doi          = {10.1038/s41588-024-01741-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {740-742},
  shortjournal = {Nat. Genet.},
  title        = {Understanding liver repair through space and time},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Contraction or sequence variant of an intergenic repeat-alu
element leads to inherited thyroid disease. <em>NG</em>, <em>56</em>(5),
738–739. (<a href="https://doi.org/10.1038/s41588-024-01723-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic and epigenomic techniques identify a new variation type causing Mendelian disease by altering the non-coding regulatory network in thyroid cells — solving a hidden cause linked for 20 years.},
  archive      = {J_NG},
  author       = {Cortese, Andrea and Vegezzi, Elisa and Houlden, Henry},
  doi          = {10.1038/s41588-024-01723-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {738-739},
  shortjournal = {Nat. Genet.},
  title        = {Contraction or sequence variant of an intergenic repeat-alu element leads to inherited thyroid disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Spatial and molecular exploration of glioblastoma
heterogeneity. <em>NG</em>, <em>56</em>(5), 737. (<a
href="https://doi.org/10.1038/s41588-024-01772-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-01772-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {737},
  shortjournal = {Nat. Genet.},
  title        = {Spatial and molecular exploration of glioblastoma heterogeneity},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Regulation of chromatin domains by BAZ2A. <em>NG</em>,
<em>56</em>(5), 737. (<a
href="https://doi.org/10.1038/s41588-024-01773-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-024-01773-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {737},
  shortjournal = {Nat. Genet.},
  title        = {Regulation of chromatin domains by BAZ2A},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Polyploid sugarcane genome architecture. <em>NG</em>,
<em>56</em>(5), 737. (<a
href="https://doi.org/10.1038/s41588-024-01774-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-01774-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {737},
  shortjournal = {Nat. Genet.},
  title        = {Polyploid sugarcane genome architecture},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Extreme RNA longevity in the brain. <em>NG</em>,
<em>56</em>(5), 737. (<a
href="https://doi.org/10.1038/s41588-024-01775-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-024-01775-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {737},
  shortjournal = {Nat. Genet.},
  title        = {Extreme RNA longevity in the brain},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The 1+million genomes minimal dataset for cancer.
<em>NG</em>, <em>56</em>(5), 733–736. (<a
href="https://doi.org/10.1038/s41588-024-01721-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Defining minimal standards for data collection is key to creating interoperative, searchable genomic and clinical databases. We highlight here the 1+Million Genomes Minimal Dataset for Cancer, encompassing 140 items in 8 domains to foster the collection of cancer data, inform transnational cooperation and advance precision cancer medicine.},
  archive      = {J_NG},
  author       = {Riba, Michela and Sala, Cinzia and Culhane, Aedin C. and Flobak, Åsmund and Patocs, Attila and Boye, Kjetil and Plevova, Karla and Pospíšilová, Šárka and Gandolfi, Giorgia and Morelli, Marco J. and Bucci, Gabriele and Edsjö, Anders and Lassen, Ulrik and Al-Shahrour, Fátima and Lopez-Bigas, Nuria and Hovland, Randi and Cuppen, Edwin and Valencia, Alfonso and Poirel, Helene A. and Rosenquist, Richard and Scollen, Serena and Arenas Marquez, Juan and Belien, Jeroen and De Nicolo, Arcangela and De Maria, Ruggero and Torrents, David and Tonon, Giovanni},
  doi          = {10.1038/s41588-024-01721-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {733-736},
  shortjournal = {Nat. Genet.},
  title        = {The 1+Million genomes minimal dataset for cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The genome and population genomics of allopolyploid coffea
arabica reveal the diversification history of modern coffee cultivars.
<em>NG</em>, <em>56</em>(4), 721–731. (<a
href="https://doi.org/10.1038/s41588-024-01695-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coffea arabica, an allotetraploid hybrid of Coffea eugenioides and Coffea canephora, is the source of approximately 60% of coffee products worldwide, and its cultivated accessions have undergone several population bottlenecks. We present chromosome-level assemblies of a di-haploid C. arabica accession and modern representatives of its diploid progenitors, C. eugenioides and C. canephora. The three species exhibit largely conserved genome structures between diploid parents and descendant subgenomes, with no obvious global subgenome dominance. We find evidence for a founding polyploidy event 350,000–610,000 years ago, followed by several pre-domestication bottlenecks, resulting in narrow genetic variation. A split between wild accessions and cultivar progenitors occurred ~30.5 thousand years ago, followed by a period of migration between the two populations. Analysis of modern varieties, including lines historically introgressed with C. canephora, highlights their breeding histories and loci that may contribute to pathogen resistance, laying the groundwork for future genomics-based breeding of C. arabica. Chromosome-level genome assemblies of allotetraploid Coffea arabica and representatives of its diploid progenitors, Coffea eugenioides and Coffea canephora, provide insights into Arabica’s diversification history.},
  archive      = {J_NG},
  author       = {Salojärvi, Jarkko and Rambani, Aditi and Yu, Zhe and Guyot, Romain and Strickler, Susan and Lepelley, Maud and Wang, Cui and Rajaraman, Sitaram and Rastas, Pasi and Zheng, Chunfang and Muñoz, Daniella Santos and Meidanis, João and Paschoal, Alexandre Rossi and Bawin, Yves and Krabbenhoft, Trevor J. and Wang, Zhen Qin and Fleck, Steven J. and Aussel, Rudy and Bellanger, Laurence and Charpagne, Aline and Fournier, Coralie and Kassam, Mohamed and Lefebvre, Gregory and Métairon, Sylviane and Moine, Déborah and Rigoreau, Michel and Stolte, Jens and Hamon, Perla and Couturon, Emmanuel and Tranchant-Dubreuil, Christine and Mukherjee, Minakshi and Lan, Tianying and Engelhardt, Jan and Stadler, Peter and Correia De Lemos, Samara Mireza and Suzuki, Suzana Ivamoto and Sumirat, Ucu and Wai, Ching Man and Dauchot, Nicolas and Orozco-Arias, Simon and Garavito, Andrea and Kiwuka, Catherine and Musoli, Pascal and Nalukenge, Anne and Guichoux, Erwan and Reinout, Havinga and Smit, Martin and Carretero-Paulet, Lorenzo and Filho, Oliveiro Guerreiro and Braghini, Masako Toma and Padilha, Lilian and Sera, Gustavo Hiroshi and Ruttink, Tom and Henry, Robert and Marraccini, Pierre and Van de Peer, Yves and Andrade, Alan and Domingues, Douglas and Giuliano, Giovanni and Mueller, Lukas and Pereira, Luiz Filipe and Plaisance, Stephane and Poncet, Valerie and Rombauts, Stephane and Sankoff, David and Albert, Victor A. and Crouzillat, Dominique and de Kochko, Alexandre and Descombes, Patrick},
  doi          = {10.1038/s41588-024-01695-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {721-731},
  shortjournal = {Nat. Genet.},
  title        = {The genome and population genomics of allopolyploid coffea arabica reveal the diversification history of modern coffee cultivars},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome assemblies of 11 bamboo species highlight
diversification induced by dynamic subgenome dominance. <em>NG</em>,
<em>56</em>(4), 710–720. (<a
href="https://doi.org/10.1038/s41588-024-01683-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyploidy (genome duplication) is a pivotal force in evolution. However, the interactions between parental genomes in a polyploid nucleus, frequently involving subgenome dominance, are poorly understood. Here we showcase analyses of a bamboo system (Poaceae: Bambusoideae) comprising a series of lineages from diploid (herbaceous) to tetraploid and hexaploid (woody), with 11 chromosome-level de novo genome assemblies and 476 transcriptome samples. We find that woody bamboo subgenomes exhibit stunning karyotype stability, with parallel subgenome dominance in the two tetraploid clades and a gradual shift of dominance in the hexaploid clade. Allopolyploidization and subgenome dominance have shaped the evolution of tree-like lignified culms, rapid growth and synchronous flowering characteristic of woody bamboos as large grasses. Our work provides insights into genome dominance in a remarkable polyploid system, including its dependence on genomic context and its ability to switch which subgenomes are dominant over evolutionary time. Chromosome-level genome assemblies of 11 bamboo species comprising lineages from diploid (herbaceous) to tetraploid and hexaploid (woody) provide insights into dynamic subgenome dominance in bamboos.},
  archive      = {J_NG},
  author       = {Ma, Peng-Fei and Liu, Yun-Long and Guo, Cen and Jin, Guihua and Guo, Zhen-Hua and Mao, Ling and Yang, Yi-Zhou and Niu, Liang-Zhong and Wang, Yu-Jiao and Clark, Lynn G. and Kellogg, Elizabeth A. and Xu, Zu-Chang and Ye, Xia-Ying and Liu, Jing-Xia and Zhou, Meng-Yuan and Luo, Yan and Yang, Yang and Soltis, Douglas E. and Bennetzen, Jeffrey L. and Soltis, Pamela S. and Li, De-Zhu},
  doi          = {10.1038/s41588-024-01683-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {710-720},
  shortjournal = {Nat. Genet.},
  title        = {Genome assemblies of 11 bamboo species highlight diversification induced by dynamic subgenome dominance},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The homeobox transcription factor DUXBL controls exit from
totipotency. <em>NG</em>, <em>56</em>(4), 697–709. (<a
href="https://doi.org/10.1038/s41588-024-01692-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In mice, exit from the totipotent two-cell (2C) stage embryo requires silencing of the 2C-associated transcriptional program. However, the molecular mechanisms involved in this process remain poorly understood. Here we demonstrate that the 2C-specific transcription factor double homeobox protein (DUX) mediates an essential negative feedback loop by inducing the expression of DUXBL to promote this silencing. We show that DUXBL gains accessibility to DUX-bound regions specifically upon DUX expression. Furthermore, we determine that DUXBL interacts with TRIM24 and TRIM33, members of the TRIM superfamily involved in gene silencing, and colocalizes with them in nuclear foci upon DUX expression. Importantly, DUXBL overexpression impairs 2C-associated transcription, whereas Duxbl inactivation in mouse embryonic stem cells increases DUX-dependent induction of the 2C-transcriptional program. Consequently, DUXBL deficiency in embryos results in sustained expression of 2C-associated transcripts leading to early developmental arrest. Our study identifies DUXBL as an essential regulator of totipotency exit enabling the first divergence of cell fates. The transcription factor double homeobox protein (DUX) induces a totipotency-specific regulatory program, including the upregulation of DUXBL. DUXBL subsequently accesses DUX-bound regions and interacts with TRIM24 and TRIM33, thus contributing to totipotency exit.},
  archive      = {J_NG},
  author       = {Vega-Sendino, Maria and Lüttmann, Felipe F. and Olbrich, Teresa and Chen, Yanpu and Kuenne, Carsten and Stein, Paula and Tillo, Desiree and Carey, Grace I. and Zhong, Jiasheng and Savy, Virginia and Radonova, Lenka and Lu, Tianlin and Saykali, Bechara and Kim, Kee-Pyo and Domingo, Catherine N. and Schüler, Leah and Günther, Stefan and Bentsen, Mette and Bosnakovski, Darko and Schöler, Hans and Kyba, Michael and Maity, Tapan K. and Jenkins, Lisa M. and Looso, Mario and Williams, Carmen J. and Kim, Johnny and Ruiz, Sergio},
  doi          = {10.1038/s41588-024-01692-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {697-709},
  shortjournal = {Nat. Genet.},
  title        = {The homeobox transcription factor DUXBL controls exit from totipotency},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Enhancer–promoter interactions become more instructive in
the transition from cell-fate specification to tissue differentiation.
<em>NG</em>, <em>56</em>(4), 686–696. (<a
href="https://doi.org/10.1038/s41588-024-01678-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To regulate expression, enhancers must come in proximity to their target gene. However, the relationship between the timing of enhancer–promoter (E–P) proximity and activity remains unclear, with examples of uncoupled, anticorrelated and correlated interactions. To assess this, we selected 600 characterized enhancers or promoters with tissue-specific activity in Drosophila embryos and performed Capture-C in FACS-purified myogenic or neurogenic cells during specification and tissue differentiation. This enabled direct comparison between E–P proximity and activity transitioning from OFF-to-ON and ON-to-OFF states across developmental conditions. This showed remarkably similar E–P topologies between specified muscle and neuronal cells, which are uncoupled from activity. During tissue differentiation, many new distal interactions emerge where changes in E–P proximity reflect changes in activity. The mode of E–P regulation therefore appears to change as embryogenesis proceeds, from largely permissive topologies during cell-fate specification to more instructive regulation during terminal tissue differentiation, when E–P proximity is coupled to activation. Analysis of enhancer–promoter (E–P) interactions during Drosophila embryogenesis suggests that the relationship between E–P proximity and activity depends on the developmental stage. Increased E–P proximity is associated with activity during differentiation but not specification.},
  archive      = {J_NG},
  author       = {Pollex, Tim and Rabinowitz, Adam and Gambetta, Maria Cristina and Marco-Ferreres, Raquel and Viales, Rebecca R. and Jankowski, Aleksander and Schaub, Christoph and Furlong, Eileen E. M.},
  doi          = {10.1038/s41588-024-01678-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {686-696},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer–promoter interactions become more instructive in the transition from cell-fate specification to tissue differentiation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Increased enhancer–promoter interactions during
developmental enhancer activation in mammals. <em>NG</em>,
<em>56</em>(4), 675–685. (<a
href="https://doi.org/10.1038/s41588-024-01681-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Remote enhancers are thought to interact with their target promoters via physical proximity, yet the importance of this proximity for enhancer function remains unclear. Here we investigate the three-dimensional (3D) conformation of enhancers during mammalian development by generating high-resolution tissue-resolved contact maps for nearly a thousand enhancers with characterized in vivo activities in ten murine embryonic tissues. Sixty-one percent of developmental enhancers bypass their neighboring genes, which are often marked by promoter CpG methylation. The majority of enhancers display tissue-specific 3D conformations, and both enhancer–promoter and enhancer–enhancer interactions are moderately but consistently increased upon enhancer activation in vivo. Less than 14% of enhancer–promoter interactions form stably across tissues; however, these invariant interactions form in the absence of the enhancer and are likely mediated by adjacent CTCF binding. Our results highlight the general importance of enhancer–promoter physical proximity for developmental gene activation in mammals. A catalog of enhancer–promoter (E–P) interactions across ten mouse embryonic tissues, supported by in vivo functional experiments, shows that E–P proximity increases upon enhancer activation during development.},
  archive      = {J_NG},
  author       = {Chen, Zhuoxin and Snetkova, Valentina and Bower, Grace and Jacinto, Sandra and Clock, Benjamin and Dizehchi, Atrin and Barozzi, Iros and Mannion, Brandon J. and Alcaina-Caro, Ana and Lopez-Rios, Javier and Dickel, Diane E. and Visel, Axel and Pennacchio, Len A. and Kvon, Evgeny Z.},
  doi          = {10.1038/s41588-024-01681-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {675-685},
  shortjournal = {Nat. Genet.},
  title        = {Increased enhancer–promoter interactions during developmental enhancer activation in mammals},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). MYC activity at enhancers drives prognostic transcriptional
programs through an epigenetic switch. <em>NG</em>, <em>56</em>(4),
663–674. (<a href="https://doi.org/10.1038/s41588-024-01676-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The transcription factor MYC is overexpressed in most cancers, where it drives multiple hallmarks of cancer progression. MYC is known to promote oncogenic transcription by binding to active promoters. In addition, MYC has also been shown to invade distal enhancers when expressed at oncogenic levels, but this enhancer binding has been proposed to have low gene-regulatory potential. Here, we demonstrate that MYC directly regulates enhancer activity to promote cancer type-specific gene programs predictive of poor patient prognosis. MYC induces transcription of enhancer RNA through recruitment of RNA polymerase II (RNAPII), rather than regulating RNAPII pause-release, as is the case at promoters. This process is mediated by MYC-induced H3K9 demethylation and acetylation by GCN5, leading to enhancer-specific BRD4 recruitment through its bromodomains, which facilitates RNAPII recruitment. We propose that MYC drives prognostic cancer type-specific gene programs through induction of an enhancer-specific epigenetic switch, which can be targeted by BET and GCN5 inhibitors. This paper highlights the mechanisms underlying MYC-dependent gene regulation from transcriptional enhancers, which are distinct to the function of MYC at promoters. This process takes place in a cancer type-specific way, and the resulting transcriptional programs can predict prognosis.},
  archive      = {J_NG},
  author       = {Jakobsen, Simon T. and Jensen, Rikke A. M. and Madsen, Maria S. and Ravnsborg, Tina and Vaagenso, Christian S. and Siersbæk, Majken S. and Einarsson, Hjorleifur and Andersson, Robin and Jensen, Ole N. and Siersbæk, Rasmus},
  doi          = {10.1038/s41588-024-01676-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {663-674},
  shortjournal = {Nat. Genet.},
  title        = {MYC activity at enhancers drives prognostic transcriptional programs through an epigenetic switch},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A single-cell atlas enables mapping of homeostatic cellular
shifts in the adult human breast. <em>NG</em>, <em>56</em>(4), 652–662.
(<a href="https://doi.org/10.1038/s41588-024-01688-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we use single-cell RNA sequencing to compile a human breast cell atlas assembled from 55 donors that had undergone reduction mammoplasties or risk reduction mastectomies. From more than 800,000 cells we identified 41 cell subclusters across the epithelial, immune and stromal compartments. The contribution of these different clusters varied according to the natural history of the tissue. Age, parity and germline mutations, known to modulate the risk of developing breast cancer, affected the homeostatic cellular state of the breast in different ways. We found that immune cells from BRCA1 or BRCA2 carriers had a distinct gene expression signature indicative of potential immune exhaustion, which was validated by immunohistochemistry. This suggests that immune-escape mechanisms could manifest in non-cancerous tissues very early during tumor initiation. This atlas is a rich resource that can be used to inform novel approaches for early detection and prevention of breast cancer. Single-cell RNA sequencing analysis of over 800,000 human adult breast cells from 55 female donors identifies 41 cell subtypes and highlights age- and parity-dependent effects. Samples from healthy women with germline mutations in BRCA1 or BRCA2 showed signs of T cell exhaustion.},
  archive      = {J_NG},
  author       = {Reed, Austin D. and Pensa, Sara and Steif, Adi and Stenning, Jack and Kunz, Daniel J. and Porter, Linsey J. and Hua, Kui and He, Peng and Twigger, Alecia-Jane and Siu, Abigail J. Q. and Kania, Katarzyna and Barrow-McGee, Rachel and Goulding, Iain and Gomm, Jennifer J. and Speirs, Valerie and Jones, J Louise and Marioni, John C. and Khaled, Walid T.},
  doi          = {10.1038/s41588-024-01688-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {652-662},
  shortjournal = {Nat. Genet.},
  title        = {A single-cell atlas enables mapping of homeostatic cellular shifts in the adult human breast},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Proteogenomic insights into early-onset endometrioid
endometrial carcinoma: Predictors for fertility-sparing therapy
response. <em>NG</em>, <em>56</em>(4), 637–651. (<a
href="https://doi.org/10.1038/s41588-024-01703-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Endometrial carcinoma remains a public health concern with a growing incidence, particularly in younger women. Preserving fertility is a crucial consideration in the management of early-onset endometrioid endometrial carcinoma (EEEC), particularly in patients under 40 who maintain both reproductive desire and capacity. To illuminate the molecular characteristics of EEEC, we undertook a large-scale multi-omics study of 215 patients with endometrial carcinoma, including 81 with EEEC. We reveal an unexpected association between exposome-related mutational signature and EEEC, characterized by specific CTNNB1 and SIGLEC10 hotspot mutations and disruption of downstream pathways. Interestingly, SIGLEC10Q144K mutation in EEECs resulted in aberrant SIGLEC-10 protein expression and promoted progestin resistance by interacting with estrogen receptor alpha. We also identified potential protein biomarkers for progestin response in fertility-sparing treatment for EEEC. Collectively, our study establishes a proteogenomic resource of EEECs, uncovering the interactions between exposome and genomic susceptibilities that contribute to the development of primary prevention and early detection strategies for EEECs. This study of Chinese endometrioid endometrial carcinomas describes the proteogenomic differences between early-onset and late-onset tumors, finding that SIGLEC10 mutation may contribute to tumorigenesis and progestin resistance in early cases.},
  archive      = {J_NG},
  author       = {Hu, Zhe and Wu, Zimeng and Liu, Wei and Ning, Yan and Liu, Jingbo and Ding, Wencheng and Fan, Junpeng and Cai, Shuyan and Li, Qinlan and Li, Wenting and Yang, Xiaohang and Dou, Yingyu and Wang, Wei and Peng, Wenju and Lu, Funian and Zhuang, Xucui and Qin, Tianyu and Kang, Xiaoyan and Feng, Chenzhao and Xu, Zhiying and Lv, Qiaoying and Wang, Qian and Wang, Chao and Wang, Xinyu and Wang, Zhiqi and Wang, Jianliu and Jiang, Jie and Wang, Beibei and Mills, Gordon B. and Ma, Ding and Gao, Qinglei and Li, Kezhen and Chen, Gang and Chen, Xiaojun and Sun, Chaoyang},
  doi          = {10.1038/s41588-024-01703-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {637-651},
  shortjournal = {Nat. Genet.},
  title        = {Proteogenomic insights into early-onset endometrioid endometrial carcinoma: Predictors for fertility-sparing therapy response},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Single-cell multi-ome regression models identify functional
and disease-associated enhancers and enable chromatin potential
analysis. <em>NG</em>, <em>56</em>(4), 627–636. (<a
href="https://doi.org/10.1038/s41588-024-01689-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a gene-level regulatory model, single-cell ATAC + RNA linking (SCARlink), which predicts single-cell gene expression and links enhancers to target genes using multi-ome (scRNA-seq and scATAC–seq co-assay) sequencing data. The approach uses regularized Poisson regression on tile-level accessibility data to jointly model all regulatory effects at a gene locus, avoiding the limitations of pairwise gene–peak correlations and dependence on peak calling. SCARlink outperformed existing gene scoring methods for imputing gene expression from chromatin accessibility across high-coverage multi-ome datasets while giving comparable to improved performance on low-coverage datasets. Shapley value analysis on trained models identified cell-type-specific gene enhancers that are validated by promoter capture Hi-C and are 11× to 15× and 5× to 12× enriched in fine-mapped eQTLs and fine-mapped genome-wide association study (GWAS) variants, respectively. We further show that SCARlink-predicted and observed gene expression vectors provide a robust way to compute a chromatin potential vector field to enable developmental trajectory analysis. Single-cell ATAC + RNA linking (SCARlink) predicts gene expression by jointly modeling local tiled chromatin accessibility using regularized Poisson regression on multi-ome data. SCARlink predictions can be used to identify cell-type-specific enhancers and perform chromatin potential analysis.},
  archive      = {J_NG},
  author       = {Mitra, Sneha and Malik, Rohan and Wong, Wilfred and Rahman, Afsana and Hartemink, Alexander J. and Pritykin, Yuri and Dey, Kushal K. and Leslie, Christina S.},
  doi          = {10.1038/s41588-024-01689-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {627-636},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-ome regression models identify functional and disease-associated enhancers and enable chromatin potential analysis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Tissue-specific enhancer–gene maps from multimodal
single-cell data identify causal disease alleles. <em>NG</em>,
<em>56</em>(4), 615–626. (<a
href="https://doi.org/10.1038/s41588-024-01682-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Translating genome-wide association study (GWAS) loci into causal variants and genes requires accurate cell-type-specific enhancer–gene maps from disease-relevant tissues. Building enhancer–gene maps is essential but challenging with current experimental methods in primary human tissues. Here we developed a nonparametric statistical method, SCENT (single-cell enhancer target gene mapping), that models association between enhancer chromatin accessibility and gene expression in single-cell or nucleus multimodal RNA sequencing and ATAC sequencing data. We applied SCENT to 9 multimodal datasets including &amp;gt;120,000 single cells or nuclei and created 23 cell-type-specific enhancer–gene maps. These maps were highly enriched for causal variants in expression quantitative loci and GWAS for 1,143 diseases and traits. We identified likely causal genes for both common and rare diseases and linked somatic mutation hotspots to target genes. We demonstrate that application of SCENT to multimodal data from disease-relevant human tissue enables the scalable construction of accurate cell-type-specific enhancer–gene maps, essential for defining noncoding variant function. SCENT is a nonparametric method that models association between chromatin accessibility and gene expression in single-cell multimodal datasets, enabling construction of cell-type-specific enhancer–gene maps to aid mapping of candidate causal variants and genes for common diseases.},
  archive      = {J_NG},
  author       = {Sakaue, Saori and Weinand, Kathryn and Isaac, Shakson and Dey, Kushal K. and Jagadeesh, Karthik and Kanai, Masahiro and Watts, Gerald F. M. and Zhu, Zhu and Brenner, Michael B. and McDavid, Andrew and Donlin, Laura T. and Wei, Kevin and Price, Alkes L. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-024-01682-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {615-626},
  shortjournal = {Nat. Genet.},
  title        = {Tissue-specific enhancer–gene maps from multimodal single-cell data identify causal disease alleles},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cell subtype-specific effects of genetic variation in the
alzheimer’s disease brain. <em>NG</em>, <em>56</em>(4), 605–614. (<a
href="https://doi.org/10.1038/s41588-024-01685-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The relationship between genetic variation and gene expression in brain cell types and subtypes remains understudied. Here, we generated single-nucleus RNA sequencing data from the neocortex of 424 individuals of advanced age; we assessed the effect of genetic variants on RNA expression in cis (cis-expression quantitative trait loci) for seven cell types and 64 cell subtypes using 1.5 million transcriptomes. This effort identified 10,004 eGenes at the cell type level and 8,099 eGenes at the cell subtype level. Many eGenes are only detected within cell subtypes. A new variant influences APOE expression only in microglia and is associated with greater cerebral amyloid angiopathy but not Alzheimer’s disease pathology, after adjusting for APOEε4, providing mechanistic insights into both pathologies. Furthermore, only a TMEM106B variant affects the proportion of cell subtypes. Integration of these results with genome-wide association studies highlighted the targeted cell type and probable causal gene within Alzheimer’s disease, schizophrenia, educational attainment and Parkinson’s disease loci. Single-nucleus RNA sequencing from the dorsolateral prefrontal cortex of 424 aging individuals, and mapping the effect of genetic variation on gene expression, identified a large number of cis-expression quantitative trait loci at the level of cell types and cell subtypes.},
  archive      = {J_NG},
  author       = {Fujita, Masashi and Gao, Zongmei and Zeng, Lu and McCabe, Cristin and White, Charles C. and Ng, Bernard and Green, Gilad Sahar and Rozenblatt-Rosen, Orit and Phillips, Devan and Amir-Zilberstein, Liat and Lee, Hyo and Pearse, Richard V. and Khan, Atlas and Vardarajan, Badri N. and Kiryluk, Krzysztof and Ye, Chun Jimmie and Klein, Hans-Ulrich and Wang, Gao and Regev, Aviv and Habib, Naomi and Schneider, Julie A. and Wang, Yanling and Young-Pearse, Tracy and Mostafavi, Sara and Bennett, David A. and Menon, Vilas and De Jager, Philip L.},
  doi          = {10.1038/s41588-024-01685-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {605-614},
  shortjournal = {Nat. Genet.},
  title        = {Cell subtype-specific effects of genetic variation in the alzheimer’s disease brain},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Cell-type-specific and disease-associated expression
quantitative trait loci in the human lung. <em>NG</em>, <em>56</em>(4),
595–604. (<a href="https://doi.org/10.1038/s41588-024-01702-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common genetic variants confer substantial risk for chronic lung diseases, including pulmonary fibrosis. Defining the genetic control of gene expression in a cell-type-specific and context-dependent manner is critical for understanding the mechanisms through which genetic variation influences complex traits and disease pathobiology. To this end, we performed single-cell RNA sequencing of lung tissue from 66 individuals with pulmonary fibrosis and 48 unaffected donors. Using a pseudobulk approach, we mapped expression quantitative trait loci (eQTLs) across 38 cell types, observing both shared and cell-type-specific regulatory effects. Furthermore, we identified disease interaction eQTLs and demonstrated that this class of associations is more likely to be cell-type-specific and linked to cellular dysregulation in pulmonary fibrosis. Finally, we connected lung disease risk variants to their regulatory targets in disease-relevant cell types. These results indicate that cellular context determines the impact of genetic variation on gene expression and implicates context-specific eQTLs as key regulators of lung homeostasis and disease. Single-cell transcriptomics and expression quantitative trait locus mapping in 114 lung tissue samples, including 66 with interstitial lung disease, highlight the cell-type-specific functions of risk variants contributing to disease pathobiology.},
  archive      = {J_NG},
  author       = {Natri, Heini M. and Del Azodi, Christina B. and Peter, Lance and Taylor, Chase J. and Chugh, Sagrika and Kendle, Robert and Chung, Mei-i and Flaherty, David K. and Matlock, Brittany K. and Calvi, Carla L. and Blackwell, Timothy S. and Ware, Lorraine B. and Bacchetta, Matthew and Walia, Rajat and Shaver, Ciara M. and Kropski, Jonathan A. and McCarthy, Davis J. and Banovich, Nicholas E.},
  doi          = {10.1038/s41588-024-01702-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {595-604},
  shortjournal = {Nat. Genet.},
  title        = {Cell-type-specific and disease-associated expression quantitative trait loci in the human lung},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Comprehensive whole-genome sequence analyses provide
insights into the genomic architecture of cerebral palsy. <em>NG</em>,
<em>56</em>(4), 585–594. (<a
href="https://doi.org/10.1038/s41588-024-01686-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We performed whole-genome sequencing (WGS) in 327 children with cerebral palsy (CP) and their biological parents. We classified 37 of 327 (11.3%) children as having pathogenic/likely pathogenic (P/LP) variants and 58 of 327 (17.7%) as having variants of uncertain significance. Multiple classes of P/LP variants included single-nucleotide variants (SNVs)/indels (6.7%), copy number variations (3.4%) and mitochondrial mutations (1.5%). The COL4A1 gene had the most P/LP SNVs. We also analyzed two pediatric control cohorts (n = 203 trios and n = 89 sib-pair families) to provide a baseline for de novo mutation rates and genetic burden analyses, the latter of which demonstrated associations between de novo deleterious variants and genes related to the nervous system. An enrichment analysis revealed previously undescribed plausible candidate CP genes (SMOC1, KDM5B, BCL11A and CYP51A1). A multifactorial CP risk profile and substantial presence of P/LP variants combine to support WGS in the diagnostic work-up across all CP and related phenotypes. Whole-genome sequencing in a Canadian cohort of 327 children with cerebral palsy compared to pediatric controls identifies novel pathogenic single-nucleotide variants/indels and copy number variations. In addition, mitochondrial variants in known disease genes were identified. This highlights the importance of genomic testing for individuals with cerebral palsy.},
  archive      = {J_NG},
  author       = {Fehlings, Darcy L. and Zarrei, Mehdi and Engchuan, Worrawat and Sondheimer, Neal and Thiruvahindrapuram, Bhooma and MacDonald, Jeffrey R. and Higginbotham, Edward J. and Thapa, Ritesh and Behlim, Tarannum and Aimola, Sabrina and Switzer, Lauren and Ng, Pamela and Wei, John and Danthi, Prakroothi S. and Pellecchia, Giovanna and Lamoureux, Sylvia and Ho, Karen and Pereira, Sergio L. and de Rijke, Jill and Sung, Wilson W. L. and Mowjoodi, Alireza and Howe, Jennifer L. and Nalpathamkalam, Thomas and Manshaei, Roozbeh and Ghaffari, Siavash and Whitney, Joseph and Patel, Rohan V. and Hamdan, Omar and Shaath, Rulan and Trost, Brett and Knights, Shannon and Samdup, Dawa and McCormick, Anna and Hunt, Carolyn and Kirton, Adam and Kawamura, Anne and Mesterman, Ronit and Gorter, Jan Willem and Dlamini, Nomazulu and Merico, Daniele and Hilali, Murto and Hirschfeld, Kyle and Grover, Kritika and Bautista, Nelson X. and Han, Kara and Marshall, Christian R. and Yuen, Ryan K. C. and Subbarao, Padmaja and Azad, Meghan B. and Turvey, Stuart E. and Mandhane, Piush and Moraes, Theo J. and Simons, Elinor and Maxwell, George and Shevell, Michael and Costain, Gregory and Michaud, Jacques L. and Hamdan, Fadi F. and Gauthier, Julie and Uguen, Kevin and Stavropoulos, Dimitri J. and Wintle, Richard F. and Oskoui, Maryam and Scherer, Stephen W.},
  doi          = {10.1038/s41588-024-01686-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {585-594},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive whole-genome sequence analyses provide insights into the genomic architecture of cerebral palsy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Protein-truncating variants in BSN are associated with
severe adult-onset obesity, type 2 diabetes and fatty liver disease.
<em>NG</em>, <em>56</em>(4), 579–584. (<a
href="https://doi.org/10.1038/s41588-024-01694-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obesity is a major risk factor for many common diseases and has a substantial heritable component. To identify new genetic determinants, we performed exome-sequence analyses for adult body mass index (BMI) in up to 587,027 individuals. We identified rare loss-of-function variants in two genes (BSN and APBA1) with effects substantially larger than those of well-established obesity genes such as MC4R. In contrast to most other obesity-related genes, rare variants in BSN and APBA1 were not associated with normal variation in childhood adiposity. Furthermore, BSN protein-truncating variants (PTVs) magnified the influence of common genetic variants associated with BMI, with a common variant polygenic score exhibiting an effect twice as large in BSN PTV carriers than in noncarriers. Finally, we explored the plasma proteomic signatures of BSN PTV carriers as well as the functional consequences of BSN deletion in human induced pluripotent stem cell-derived hypothalamic neurons. Collectively, our findings implicate degenerative processes in synaptic function in the etiology of adult-onset obesity. Analyses of whole-exome sequencing data identify rare loss-of-function variants in BSN associated with adult-onset obesity, type 2 diabetes and fatty liver disease, with stronger effect sizes than those observed for variants in known obesity risk genes such as MC4R.},
  archive      = {J_NG},
  author       = {Zhao, Yajie and Chukanova, Maria and Kentistou, Katherine A. and Fairhurst-Hunter, Zammy and Siegert, Anna Maria and Jia, Raina Y. and Dowsett, Georgina K. C. and Gardner, Eugene J. and Lawler, Katherine and Day, Felix R. and Kaisinger, Lena R. and Tung, Yi-Chun Loraine and Lam, Brian Yee Hong and Chen, Hsiao-Jou Cortina and Wang, Quanli and Berumen-Campos, Jaime and Kuri-Morales, Pablo and Tapia-Conyer, Roberto and Alegre-Diaz, Jesus and Barroso, Inês and Emberson, Jonathan and Torres, Jason M. and Collins, Rory and Saleheen, Danish and Smith, Katherine R. and Paul, Dirk S. and Merkle, Florian and Farooqi, I. Sadaf and Wareham, Nick J. and Petrovski, Slavé and O’Rahilly, Stephen and Ong, Ken K. and Yeo, Giles S. H. and Perry, John R. B.},
  doi          = {10.1038/s41588-024-01694-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {579-584},
  shortjournal = {Nat. Genet.},
  title        = {Protein-truncating variants in BSN are associated with severe adult-onset obesity, type 2 diabetes and fatty liver disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Protein-altering variants at copy number-variable regions
influence diverse human phenotypes. <em>NG</em>, <em>56</em>(4),
569–578. (<a href="https://doi.org/10.1038/s41588-024-01684-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Copy number variants (CNVs) are among the largest genetic variants, yet CNVs have not been effectively ascertained in most genetic association studies. Here we ascertained protein-altering CNVs from UK Biobank whole-exome sequencing data (n = 468,570) using haplotype-informed methods capable of detecting subexonic CNVs and variation within segmental duplications. Incorporating CNVs into analyses of rare variants predicted to cause gene loss of function (LOF) identified 100 associations of predicted LOF variants with 41 quantitative traits. A low-frequency partial deletion of RGL3 exon 6 conferred one of the strongest protective effects of gene LOF on hypertension risk (odds ratio = 0.86 (0.82–0.90)). Protein-coding variation in rapidly evolving gene families within segmental duplications—previously invisible to most analysis methods—generated some of the human genome’s largest contributions to variation in type 2 diabetes risk, chronotype and blood cell traits. These results illustrate the potential for new genetic insights from genomic variation that has escaped large-scale analysis to date. Incorporating protein-altering copy number variants ascertained from UK Biobank whole-exome sequencing data into analyses of rare predicted loss-of-function variants identifies complex trait associations not detectable using standard analysis methods.},
  archive      = {J_NG},
  author       = {Hujoel, Margaux L. A. and Handsaker, Robert E. and Sherman, Maxwell A. and Kamitaki, Nolan and Barton, Alison R. and Mukamel, Ronen E. and Terao, Chikashi and McCarroll, Steven A. and Loh, Po-Ru},
  doi          = {10.1038/s41588-024-01684-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {569-578},
  shortjournal = {Nat. Genet.},
  title        = {Protein-altering variants at copy number-variable regions influence diverse human phenotypes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MYC activates transcriptional enhancers to drive cancer
progression. <em>NG</em>, <em>56</em>(4), 567–568. (<a
href="https://doi.org/10.1038/s41588-024-01677-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We show that in addition to promoter activation, MYC drives cancer progression by activating transcriptional enhancers via a distinct mechanism. MYC cooperates with several other proteins at these cis-regulatory regions to change the epigenome and promote recruitment of RNA polymerase II and enhancer transcription.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01677-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {567-568},
  shortjournal = {Nat. Genet.},
  title        = {MYC activates transcriptional enhancers to drive cancer progression},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping single-cell gene expression in the healthy human
breast. <em>NG</em>, <em>56</em>(4), 565–566. (<a
href="https://doi.org/10.1038/s41588-024-01691-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have curated a comprehensive single-cell reference map of the human breast. Our data explore how age, parity and germline mutations might influence cellular dynamics, revealing unexpected signs of immune exhaustion in healthy tissues from carriers of BRCA1 or BRCA2 germline mutations.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01691-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {565-566},
  shortjournal = {Nat. Genet.},
  title        = {Mapping single-cell gene expression in the healthy human breast},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Exit from totipotency is controlled by DUXBL in mice.
<em>NG</em>, <em>56</em>(4), 563–564. (<a
href="https://doi.org/10.1038/s41588-024-01690-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In mice, zygotic genome activation occurs at onset of the two-cell stage in embryonic development and coincides with the exit from totipotency. Our work shows that the transcription factor DUXBL participates in silencing part of the stage-specific two-cell-associated transcriptional program and is required for development to proceed.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01690-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {563-564},
  shortjournal = {Nat. Genet.},
  title        = {Exit from totipotency is controlled by DUXBL in mice},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Identifying regulatory loci across 38 lung cell types.
<em>NG</em>, <em>56</em>(4), 561–562. (<a
href="https://doi.org/10.1038/s41588-024-01701-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using single-cell RNA-sequencing (scRNA-seq) of lung tissue, expression quantitative trait loci (eQTLs) were mapped across 38 cell types, revealing both shared and cell-type-specific effects. Highly cell-type-specific disease-interaction eQTLs were linked to cellular dysregulation in lung disease and lung disease risk variants were connected to their regulatory targets in relevant cell types.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01701-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {561-562},
  shortjournal = {Nat. Genet.},
  title        = {Identifying regulatory loci across 38 lung cell types},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Enhancer contacts during embryonic development show diverse
interaction modes and modest yet significant increases upon gene
activation. <em>NG</em>, <em>56</em>(4), 558–560. (<a
href="https://doi.org/10.1038/s41588-024-01700-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The exact relationship between 3D chromatin interactions and enhancer function is unclear. By probing three-dimensional enhancer interactions in developing embryos, two studies now show nuanced dynamics in tissue-specific contexts and reveal how moderately increased enhancer–promoter interactions coincide with functionality.},
  archive      = {J_NG},
  author       = {Ibrahim, Daniel M.},
  doi          = {10.1038/s41588-024-01700-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {558-560},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer contacts during embryonic development show diverse interaction modes and modest yet significant increases upon gene activation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024f). Quantitative proteomics of protein–nucleosome interactions.
<em>NG</em>, <em>56</em>(4), 557. (<a
href="https://doi.org/10.1038/s41588-024-01727-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-01727-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {557},
  shortjournal = {Nat. Genet.},
  title        = {Quantitative proteomics of protein–nucleosome interactions},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Role of gene–gene loops in fine-tuning cross-regulation.
<em>NG</em>, <em>56</em>(4), 557. (<a
href="https://doi.org/10.1038/s41588-024-01728-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-024-01728-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {557},
  shortjournal = {Nat. Genet.},
  title        = {Role of gene–gene loops in fine-tuning cross-regulation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Genetic associations of human metabolic traits.
<em>NG</em>, <em>56</em>(4), 557. (<a
href="https://doi.org/10.1038/s41588-024-01729-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-01729-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {557},
  shortjournal = {Nat. Genet.},
  title        = {Genetic associations of human metabolic traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Implementing polygenic risk scores in the clinic.
<em>NG</em>, <em>56</em>(4), 557. (<a
href="https://doi.org/10.1038/s41588-024-01730-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-024-01730-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {557},
  shortjournal = {Nat. Genet.},
  title        = {Implementing polygenic risk scores in the clinic},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Guidance on use of race, ethnicity, and geographic origin as
proxies for genetic ancestry groups in biomedical publications.
<em>NG</em>, <em>56</em>(4), 555–556. (<a
href="https://doi.org/10.1038/s41588-024-01708-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Feero, W. Gregory and Steiner, Robert D. and Slavotinek, Anne and Faial, Tiago and Bamshad, Michael J. and Austin, Jehannine and Korf, Bruce R. and Flanagin, Annette and Bibbins-Domingo, Kirsten},
  doi          = {10.1038/s41588-024-01708-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {555-556},
  shortjournal = {Nat. Genet.},
  title        = {Guidance on use of race, ethnicity, and geographic origin as proxies for genetic ancestry groups in biomedical publications},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Combined effects of host genetics and
diet on human gut microbiota and incident disease in a single population
cohort. <em>NG</em>, <em>56</em>(3), 554. (<a
href="https://doi.org/10.1038/s41588-024-01693-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Qin, Youwen and Havulinna, Aki S. and Liu, Yang and Jousilahti, Pekka and Ritchie, Scott C. and Tokolyi, Alex and Sanders, Jon G. and Valsta, Liisa and Brożyńska, Marta and Zhu, Qiyun and Tripathi, Anupriya and Vázquez-Baeza, Yoshiki and Loomba, Rohit and Cheng, Susan and Jain, Mohit and Niiranen, Teemu and Lahti, Leo and Knight, Rob and Salomaa, Veikko and Inouye, Michael and Méric, Guillaume},
  doi          = {10.1038/s41588-024-01693-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {554},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Combined effects of host genetics and diet on human gut microbiota and incident disease in a single population cohort},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: GATK-gCNV enables the discovery of rare
copy number variants from exome sequencing data. <em>NG</em>,
<em>56</em>(3), 553. (<a
href="https://doi.org/10.1038/s41588-024-01663-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Babadi, Mehrtash and Fu, Jack M. and Lee, Samuel K. and Smirnov, Andrey N. and Gauthier, Laura D. and Walker, Mark and Benjamin, David I. and Zhao, Xuefang and Karczewski, Konrad J. and Wong, Isaac and Collins, Ryan L. and Sanchis-Juan, Alba and Brand, Harrison and Banks, Eric and Talkowski, Michael E.},
  doi          = {10.1038/s41588-024-01663-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {553},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: GATK-gCNV enables the discovery of rare copy number variants from exome sequencing data},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Systematic profiling of conditional
pathway activation identifies context-dependent synthetic lethalities.
<em>NG</em>, <em>56</em>(3), 553. (<a
href="https://doi.org/10.1038/s41588-024-01680-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Chang, Liang and Jung, Nancy Y. and Atari, Adel and Rodriguez, Diego J. and Kesar, Devishi and Song, Tian-Yu and Rees, Matthew G. and Ronan, Melissa and Li, Ruitong and Ruiz, Paloma and Chaturantabut, Saireudee and Ito, Takahiro and van Tienen, Laurens M. and Tseng, Yuen-Yi and Roth, Jennifer A. and Sellers, William R.},
  doi          = {10.1038/s41588-024-01680-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {553},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Systematic profiling of conditional pathway activation identifies context-dependent synthetic lethalities},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accurate and sensitive mutational signature analysis with
MuSiCal. <em>NG</em>, <em>56</em>(3), 541–552. (<a
href="https://doi.org/10.1038/s41588-024-01659-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational signature analysis is a recent computational approach for interpreting somatic mutations in the genome. Its application to cancer data has enhanced our understanding of mutational forces driving tumorigenesis and demonstrated its potential to inform prognosis and treatment decisions. However, methodological challenges remain for discovering new signatures and assigning proper weights to existing signatures, thereby hindering broader clinical applications. Here we present Mutational Signature Calculator (MuSiCal), a rigorous analytical framework with algorithms that solve major problems in the standard workflow. Our simulation studies demonstrate that MuSiCal outperforms state-of-the-art algorithms for both signature discovery and assignment. By reanalyzing more than 2,700 cancer genomes, we provide an improved catalog of signatures and their assignments, discover nine indel signatures absent in the current catalog, resolve long-standing issues with the ambiguous ‘flat’ signatures and give insights into signatures with unknown etiologies. We expect MuSiCal and the improved catalog to be a step towards establishing best practices for mutational signature analysis. MuSiCal is a mutational signature analysis tool combining minimum-volume nonnegative matrix factorization with other algorithmic innovations. Applied to PCAWG data, MuSiCal gives more accurate results, including resolving ambiguous flat signatures.},
  archive      = {J_NG},
  author       = {Jin, Hu and Gulhan, Doga C. and Geiger, Benedikt and Ben-Isvy, Daniel and Geng, David and Ljungström, Viktor and Park, Peter J.},
  doi          = {10.1038/s41588-024-01659-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {541-552},
  shortjournal = {Nat. Genet.},
  title        = {Accurate and sensitive mutational signature analysis with MuSiCal},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A genomic variation map provides insights into peanut
diversity in china and associations with 28 agronomic traits.
<em>NG</em>, <em>56</em>(3), 530–540. (<a
href="https://doi.org/10.1038/s41588-024-01660-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peanut (Arachis hypogaea L.) is an important allotetraploid oil and food legume crop. China is one of the world’s largest peanut producers and consumers. However, genomic variations underlying the migration and divergence of peanuts in China remain unclear. Here we reported a genome-wide variation map based on the resequencing of 390 peanut accessions, suggesting that peanuts might have been introduced into southern and northern China separately, forming two cultivation centers. Selective sweep analysis highlights asymmetric selection between the two subgenomes during peanut improvement. A classical pedigree from South China offers a context for the examination of the impact of artificial selection on peanut genome. Genome-wide association studies identified 22,309 significant associations with 28 agronomic traits, including candidate genes for plant architecture and oil biosynthesis. Our findings shed light on peanut migration and diversity in China and provide valuable genomic resources for peanut improvement. Resequencing of 390 peanut accessions provides insights into peanut migration and diversity in China. Genome-wide association analysis identifies loci associated with 28 agronomic traits.},
  archive      = {J_NG},
  author       = {Lu, Qing and Huang, Lu and Liu, Hao and Garg, Vanika and Gangurde, Sunil S. and Li, Haifen and Chitikineni, Annapurna and Guo, Dandan and Pandey, Manish K. and Li, Shaoxiong and Liu, Haiyan and Wang, Runfeng and Deng, Quanqing and Du, Puxuan and Varshney, Rajeev K. and Liang, Xuanqiang and Hong, Yanbin and Chen, Xiaoping},
  doi          = {10.1038/s41588-024-01660-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {530-540},
  shortjournal = {Nat. Genet.},
  title        = {A genomic variation map provides insights into peanut diversity in china and associations with 28 agronomic traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Large-scale gene expression alterations introduced by
structural variation drive morphotype diversification in brassica
oleracea. <em>NG</em>, <em>56</em>(3), 517–529. (<a
href="https://doi.org/10.1038/s41588-024-01655-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brassica oleracea, globally cultivated for its vegetable crops, consists of very diverse morphotypes, characterized by specialized enlarged organs as harvested products. This makes B. oleracea an ideal model for studying rapid evolution and domestication. We constructed a B. oleracea pan-genome from 27 high-quality genomes representing all morphotypes and their wild relatives. We identified structural variations (SVs) among these genomes and characterized these in 704 B. oleracea accessions using graph-based genome tools. We show that SVs exert bidirectional effects on the expression of numerous genes, either suppressing through DNA methylation or promoting probably by harboring transcription factor-binding elements. The following examples illustrate the role of SVs modulating gene expression: SVs promoting BoPNY and suppressing BoCKX3 in cauliflower/broccoli, suppressing BoKAN1 and BoACS4 in cabbage and promoting BoMYBtf in ornamental kale. These results provide solid evidence for the role of SVs as dosage regulators of gene expression, driving B. oleracea domestication and diversification. De novo genome assemblies of 22 Brassica oleracea accessions and pan-genome analyses highlight the effects of structural variations on gene expression and their contributions to morphotype diversification.},
  archive      = {J_NG},
  author       = {Li, Xing and Wang, Yong and Cai, Chengcheng and Ji, Jialei and Han, Fengqing and Zhang, Lei and Chen, Shumin and Zhang, Lingkui and Yang, Yinqing and Tang, Qi and Bucher, Johan and Wang, Xuelin and Yang, Limei and Zhuang, Mu and Zhang, Kang and Lv, Honghao and Bonnema, Guusje and Zhang, Yangyong and Cheng, Feng},
  doi          = {10.1038/s41588-024-01655-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {517-529},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale gene expression alterations introduced by structural variation drive morphotype diversification in brassica oleracea},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Near-gapless and haplotype-resolved apple genomes provide
insights into the genetic basis of rootstock-induced dwarfing.
<em>NG</em>, <em>56</em>(3), 505–516. (<a
href="https://doi.org/10.1038/s41588-024-01657-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dwarfing rootstocks have transformed the production of cultivated apples; however, the genetic basis of rootstock-induced dwarfing remains largely unclear. We have assembled chromosome-level, near-gapless and haplotype-resolved genomes for the popular dwarfing rootstock ‘M9’, the semi-vigorous rootstock ‘MM106’ and ‘Fuji’, one of the most commonly grown apple cultivars. The apple orthologue of auxin response factor 3 (MdARF3) is in the Dw1 region of ‘M9’, the major locus for rootstock-induced dwarfing. Comparing ‘M9’ and ‘MM106’ genomes revealed a 9,723-bp allele-specific long terminal repeat retrotransposon/gypsy insertion, DwTE, located upstream of MdARF3. DwTE is cosegregated with the dwarfing trait in two segregating populations, suggesting its prospective utility in future dwarfing rootstock breeding. In addition, our pipeline discovered mobile mRNAs that may contribute to the development of dwarfed scion architecture. Our research provides valuable genomic resources and applicable methodology, which have the potential to accelerate breeding dwarfing rootstocks for apple and other perennial woody fruit trees. Near-gapless and haplotype-resolved genome assemblies of the dwarfing ‘M9’ and semi-vigorous ‘MM106’ rootstocks and a major apple cultivar ‘Fuji’ provide insights into the genetic basis of rootstock-induced dwarfing traits.},
  archive      = {J_NG},
  author       = {Li, Wei and Chu, Chong and Li, Hui and Zhang, Hengtao and Sun, Haochen and Wang, Shiyao and Wang, Zijun and Li, Yuqi and Foster, Toshi M. and López-Girona, Elena and Yu, Jiaxin and Li, Yi and Ma, Yue and Zhang, Ke and Han, Yongming and Zhou, Bowen and Fan, Xingqiang and Xiong, Yao and Deng, Cecilia H. and Wang, Yi and Xu, Xuefeng and Han, Zhenhai},
  doi          = {10.1038/s41588-024-01657-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {505-516},
  shortjournal = {Nat. Genet.},
  title        = {Near-gapless and haplotype-resolved apple genomes provide insights into the genetic basis of rootstock-induced dwarfing},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Polycomb repression of hox genes involves spatial feedback
but not domain compaction or phase transition. <em>NG</em>,
<em>56</em>(3), 493–504. (<a
href="https://doi.org/10.1038/s41588-024-01661-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polycomb group proteins have a critical role in silencing transcription during development. It is commonly proposed that Polycomb-dependent changes in genome folding, which compact chromatin, contribute directly to repression by blocking the binding of activating complexes. Recently, it has also been argued that liquid–liquid demixing of Polycomb proteins facilitates this compaction and repression by phase-separating target genes into a membraneless compartment. To test these models, we used Optical Reconstruction of Chromatin Architecture to trace the Hoxa gene cluster, a canonical Polycomb target, in thousands of single cells. Across multiple cell types, we find that Polycomb-bound chromatin frequently explores decompact states and partial mixing with neighboring chromatin, while remaining uniformly repressed, challenging the repression-by-compaction or phase-separation models. Using polymer simulations, we show that these observed flexible ensembles can be explained by ‘spatial feedback’—transient contacts that contribute to the propagation of the epigenetic state (epigenetic memory), without inducing a globular organization. In vivo tracing of the single-cell 3D structure at the Hoxa cluster shows that Polycomb-associated domains in individual cells are flexible and often decompact while maintaining a repressive transcriptional state.},
  archive      = {J_NG},
  author       = {Murphy, Sedona Eve and Boettiger, Alistair Nicol},
  doi          = {10.1038/s41588-024-01661-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {493-504},
  shortjournal = {Nat. Genet.},
  title        = {Polycomb repression of hox genes involves spatial feedback but not domain compaction or phase transition},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). In vitro reconstitution of chromatin domains shows a role
for nucleosome positioning in 3D genome organization. <em>NG</em>,
<em>56</em>(3), 483–492. (<a
href="https://doi.org/10.1038/s41588-023-01649-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Eukaryotic genomes are organized into chromatin domains. The molecular mechanisms driving the formation of these domains are difficult to dissect in vivo and remain poorly understood. Here we reconstitute Saccharomyces cerevisiae chromatin in vitro and determine its 3D organization at subnucleosome resolution by micrococcal nuclease-based chromosome conformation capture and molecular dynamics simulations. We show that regularly spaced and phased nucleosome arrays form chromatin domains in vitro that resemble domains in vivo. This demonstrates that neither loop extrusion nor transcription is required for basic domain formation in yeast. In addition, we find that the boundaries of reconstituted domains correspond to nucleosome-free regions and that insulation strength scales with their width. Finally, we show that domain compaction depends on nucleosome linker length, with longer linkers forming more compact structures. Together, our results demonstrate that regular nucleosome positioning is important for the formation of chromatin domains and provide a proof-of-principle for bottom-up 3D genome studies. Bottom-up in vitro reconstitution of ~10-kb chromatin domains shows that nucleosome positioning, rather than loop extrusion or transcription, determines domain formation in yeast.},
  archive      = {J_NG},
  author       = {Oberbeckmann, Elisa and Quililan, Kimberly and Cramer, Patrick and Oudelaar, A. Marieke},
  doi          = {10.1038/s41588-023-01649-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {483-492},
  shortjournal = {Nat. Genet.},
  title        = {In vitro reconstitution of chromatin domains shows a role for nucleosome positioning in 3D genome organization},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genome-wide ATAC-see screening identifies TFDP1 as a
modulator of global chromatin accessibility. <em>NG</em>,
<em>56</em>(3), 473–482. (<a
href="https://doi.org/10.1038/s41588-024-01658-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin accessibility is a hallmark of active regulatory regions and is functionally linked to transcriptional networks and cell identity. However, the molecular mechanisms and networks that govern chromatin accessibility have not been thoroughly studied. Here we conducted a genome-wide CRISPR screening combined with an optimized ATAC-see protocol to identify genes that modulate global chromatin accessibility. In addition to known chromatin regulators like CREBBP and EP400, we discovered a number of previously unrecognized proteins that modulate chromatin accessibility, including TFDP1, HNRNPU, EIF3D and THAP11 belonging to diverse biological pathways. ATAC-seq analysis upon their knockouts revealed their distinct and specific effects on chromatin accessibility. Remarkably, we found that TFDP1, a transcription factor, modulates global chromatin accessibility through transcriptional regulation of canonical histones. In addition, our findings highlight the manipulation of chromatin accessibility as an approach to enhance various cell engineering applications, including genome editing and induced pluripotent stem cell reprogramming. A genome-wide CRISPR knockout screen in eHAP cells using ATAC-see to fluorescently label chromatin identifies novel regulators of accessibility, such as the transcription factor TFDP1 via regulation of canonical histone expression.},
  archive      = {J_NG},
  author       = {Ishii, Satoko and Kakizuka, Taishi and Park, Sung-Joon and Tagawa, Ayako and Sanbo, Chiaki and Tanabe, Hideyuki and Ohkawa, Yasuyuki and Nakanishi, Mahito and Nakai, Kenta and Miyanari, Yusuke},
  doi          = {10.1038/s41588-024-01658-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {473-482},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide ATAC-see screening identifies TFDP1 as a modulator of global chromatin accessibility},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Pathway level subtyping identifies a slow-cycling
biological phenotype associated with poor clinical outcomes in
colorectal cancer. <em>NG</em>, <em>56</em>(3), 458–472. (<a
href="https://doi.org/10.1038/s41588-024-01654-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular stratification using gene-level transcriptional data has identified subtypes with distinctive genotypic and phenotypic traits, as exemplified by the consensus molecular subtypes (CMS) in colorectal cancer (CRC). Here, rather than gene-level data, we make use of gene ontology and biological activation state information for initial molecular class discovery. In doing so, we defined three pathway-derived subtypes (PDS) in CRC: PDS1 tumors, which are canonical/LGR5+ stem-rich, highly proliferative and display good prognosis; PDS2 tumors, which are regenerative/ANXA1+ stem-rich, with elevated stromal and immune tumor microenvironmental lineages; and PDS3 tumors, which represent a previously overlooked slow-cycling subset of tumors within CMS2 with reduced stem populations and increased differentiated lineages, particularly enterocytes and enteroendocrine cells, yet display the worst prognosis in locally advanced disease. These PDS3 phenotypic traits are evident across numerous bulk and single-cell datasets, and demark a series of subtle biological states that are currently under-represented in pre-clinical models and are not identified using existing subtyping classifiers. Analysis of colorectal cancer bulk gene expression data at the pathway level identifies a poor-prognosis subtype associated with cell differentiation. The subtypes are reproducible in single-cell data and offer biological insights beyond existing stratification strategies.},
  archive      = {J_NG},
  author       = {Malla, Sudhir B. and Byrne, Ryan M. and Lafarge, Maxime W. and Corry, Shania M. and Fisher, Natalie C. and Tsantoulis, Petros K. and Mills, Megan L. and Ridgway, Rachel A. and Lannagan, Tamsin R. M. and Najumudeen, Arafath K. and Gilroy, Kathryn L. and Amirkhah, Raheleh and Maguire, Sarah L. and Mulholland, Eoghan J. and Belnoue-Davis, Hayley L. and Grassi, Elena and Viviani, Marco and Rogan, Emily and Redmond, Keara L. and Sakhnevych, Svetlana and McCooey, Aoife J. and Bull, Courtney and Hoey, Emily and Sinevici, Nicoleta and Hall, Holly and Ahmaderaghi, Baharak and Domingo, Enric and Blake, Andrew and Richman, Susan D. and Isella, Claudio and Miller, Crispin and Bertotti, Andrea and Trusolino, Livio and Loughrey, Maurice B. and Kerr, Emma M. and Tejpar, Sabine and Maughan, Timothy S. and Lawler, Mark and Campbell, Andrew D. and Leedham, Simon J. and Koelzer, Viktor H. and Sansom, Owen J. and Dunne, Philip D.},
  doi          = {10.1038/s41588-024-01654-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {458-472},
  shortjournal = {Nat. Genet.},
  title        = {Pathway level subtyping identifies a slow-cycling biological phenotype associated with poor clinical outcomes in colorectal cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multi-omic profiling of clear cell renal cell carcinoma
identifies metabolic reprogramming associated with disease progression.
<em>NG</em>, <em>56</em>(3), 442–457. (<a
href="https://doi.org/10.1038/s41588-024-01662-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clear cell renal cell carcinoma (ccRCC) is a complex disease with remarkable immune and metabolic heterogeneity. Here we perform genomic, transcriptomic, proteomic, metabolomic and spatial transcriptomic and metabolomic analyses on 100 patients with ccRCC from the Tongji Hospital RCC (TJ-RCC) cohort. Our analysis identifies four ccRCC subtypes including De-clear cell differentiated (DCCD)-ccRCC, a subtype with distinctive metabolic features. DCCD cancer cells are characterized by fewer lipid droplets, reduced metabolic activity, enhanced nutrient uptake capability and a high proliferation rate, leading to poor prognosis. Using single-cell and spatial trajectory analysis, we demonstrate that DCCD is a common mode of ccRCC progression. Even among stage I patients, DCCD is associated with worse outcomes and higher recurrence rate, suggesting that it cannot be cured by nephrectomy alone. Our study also suggests a treatment strategy based on subtype-specific immune cell infiltration that could guide the clinical management of ccRCC. Multi-omic analysis of 100 clear cell renal cell carcinomas identifies four transcriptomic subgroups (IM1–IM4). IM4 is a high-risk subtype characterized by specific metabolic changes and a loss of lipid droplets.},
  archive      = {J_NG},
  author       = {Hu, Junyi and Wang, Shao-Gang and Hou, Yaxin and Chen, Zhaohui and Liu, Lilong and Li, Ruizhi and Li, Nisha and Zhou, Lijie and Yang, Yu and Wang, Liping and Wang, Liang and Yang, Xiong and Lei, Yichen and Deng, Changqi and Li, Yang and Deng, Zhiyao and Ding, Yuhong and Kuang, Yingchun and Yao, Zhipeng and Xun, Yang and Li, Fan and Li, Heng and Hu, Jia and Liu, Zheng and Wang, Tao and Hao, Yi and Jiao, Xuanmao and Guan, Wei and Tao, Zhen and Ren, Shancheng and Chen, Ke},
  doi          = {10.1038/s41588-024-01662-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {442-457},
  shortjournal = {Nat. Genet.},
  title        = {Multi-omic profiling of clear cell renal cell carcinoma identifies metabolic reprogramming associated with disease progression},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BANKSY unifies cell typing and tissue domain segmentation
for scalable spatial omics data analysis. <em>NG</em>, <em>56</em>(3),
431–441. (<a href="https://doi.org/10.1038/s41588-024-01664-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics data are clustered to define both cell types and tissue domains. We present Building Aggregates with a Neighborhood Kernel and Spatial Yardstick (BANKSY), an algorithm that unifies these two spatial clustering problems by embedding cells in a product space of their own and the local neighborhood transcriptome, representing cell state and microenvironment, respectively. BANKSY’s spatial feature augmentation strategy improved performance on both tasks when tested on diverse RNA (imaging, sequencing) and protein (imaging) datasets. BANKSY revealed unexpected niche-dependent cell states in the mouse brain and outperformed competing methods on domain segmentation and cell typing benchmarks. BANKSY can also be used for quality control of spatial transcriptomics data and for spatially aware batch effect correction. Importantly, it is substantially faster and more scalable than existing methods, enabling the processing of millions of cell datasets. In summary, BANKSY provides an accurate, biologically motivated, scalable and versatile framework for analyzing spatially resolved omics data. BANKSY is an algorithm with R and Python implementations that identifies both cell types and tissue domains from spatially resolved omics data by incorporating spatial kernels capturing microenvironmental information. It is applicable to a range of technologies and is scalable to millions of cells.},
  archive      = {J_NG},
  author       = {Singhal, Vipul and Chou, Nigel and Lee, Joseph and Yue, Yifei and Liu, Jinyue and Chock, Wan Kee and Lin, Li and Chang, Yun-Ching and Teo, Erica Mei Ling and Aow, Jonathan and Lee, Hwee Kuan and Chen, Kok Hao and Prabhakar, Shyam},
  doi          = {10.1038/s41588-024-01664-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {431-441},
  shortjournal = {Nat. Genet.},
  title        = {BANKSY unifies cell typing and tissue domain segmentation for scalable spatial omics data analysis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Functional dissection of human cardiac enhancers and
noncoding de novo variants in congenital heart disease. <em>NG</em>,
<em>56</em>(3), 420–430. (<a
href="https://doi.org/10.1038/s41588-024-01669-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rare coding mutations cause ∼45% of congenital heart disease (CHD). Noncoding mutations that perturb cis-regulatory elements (CREs) likely contribute to the remaining cases, but their identification has been problematic. Using a lentiviral massively parallel reporter assay (lentiMPRA) in human induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs), we functionally evaluated 6,590 noncoding de novo variants (ncDNVs) prioritized from the whole-genome sequencing of 750 CHD trios. A total of 403 ncDNVs substantially affected cardiac CRE activity. A majority increased enhancer activity, often at regions with undetectable reference sequence activity. Of ten DNVs tested by introduction into their native genomic context, four altered the expression of neighboring genes and iPSC-CM transcriptional state. To prioritize future DNVs for functional testing, we used the MPRA data to develop a regression model, EpiCard. Analysis of an independent CHD cohort by EpiCard found enrichment of DNVs. Together, we developed a scalable system to measure the effect of ncDNVs on CRE activity and deployed it to systematically assess the contribution of ncDNVs to CHD. Lentiviral massively parallel reporter assay (lentiMPRA) analysis of cardiac cis-regulatory elements characterizes the effects of noncoding de novo variants identified in congenital heart disease. EpiCard is a model for variant prioritization.},
  archive      = {J_NG},
  author       = {Xiao, Feng and Zhang, Xiaoran and Morton, Sarah U. and Kim, Seong Won and Fan, Youfei and Gorham, Joshua M. and Zhang, Huan and Berkson, Paul J. and Mazumdar, Neil and Cao, Yangpo and Chen, Jian and Hagen, Jacob and Liu, Xujie and Zhou, Pingzhu and Richter, Felix and Shen, Yufeng and Ward, Tarsha and Gelb, Bruce D. and Seidman, Jonathan G. and Seidman, Christine E. and Pu, William T.},
  doi          = {10.1038/s41588-024-01669-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {420-430},
  shortjournal = {Nat. Genet.},
  title        = {Functional dissection of human cardiac enhancers and noncoding de novo variants in congenital heart disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Epigenetic variation impacts individual differences in the
transcriptional response to influenza infection. <em>NG</em>,
<em>56</em>(3), 408–419. (<a
href="https://doi.org/10.1038/s41588-024-01668-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Humans display remarkable interindividual variation in their immune response to identical challenges. Yet, our understanding of the genetic and epigenetic factors contributing to such variation remains limited. Here we performed in-depth genetic, epigenetic and transcriptional profiling on primary macrophages derived from individuals of European and African ancestry before and after infection with influenza A virus. We show that baseline epigenetic profiles are strongly predictive of the transcriptional response to influenza A virus across individuals. Quantitative trait locus (QTL) mapping revealed highly coordinated genetic effects on gene regulation, with many cis-acting genetic variants impacting concomitantly gene expression and multiple epigenetic marks. These data reveal that ancestry-associated differences in the epigenetic landscape can be genetically controlled, even more than gene expression. Lastly, among QTL variants that colocalized with immune-disease loci, only 7% were gene expression QTL, while the remaining genetic variants impact epigenetic marks, stressing the importance of considering molecular phenotypes beyond gene expression in disease-focused studies. Epigenetic profiles can be predictive of macrophage transcriptional responses to influenza A virus infection in individuals of European and African ancestry. Ancestry-linked epigenetic differences appear to be genetically controlled.},
  archive      = {J_NG},
  author       = {Aracena, Katherine A. and Lin, Yen-Lung and Luo, Kaixuan and Pacis, Alain and Gona, Saideep and Mu, Zepeng and Yotova, Vania and Sindeaux, Renata and Pramatarova, Albena and Simon, Marie-Michelle and Chen, Xun and Groza, Cristian and Lougheed, David and Gregoire, Romain and Brownlee, David and Boye, Carly and Pique-Regi, Roger and Li, Yang and He, Xin and Bujold, David and Pastinen, Tomi and Bourque, Guillaume and Barreiro, Luis B.},
  doi          = {10.1038/s41588-024-01668-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {408-419},
  shortjournal = {Nat. Genet.},
  title        = {Epigenetic variation impacts individual differences in the transcriptional response to influenza infection},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Digenic inheritance involving a muscle-specific protein
kinase and the giant titin protein causes a skeletal muscle myopathy.
<em>NG</em>, <em>56</em>(3), 395–407. (<a
href="https://doi.org/10.1038/s41588-023-01651-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In digenic inheritance, pathogenic variants in two genes must be inherited together to cause disease. Only very few examples of digenic inheritance have been described in the neuromuscular disease field. Here we show that predicted deleterious variants in SRPK3, encoding the X-linked serine/argenine protein kinase 3, lead to a progressive early onset skeletal muscle myopathy only when in combination with heterozygous variants in the TTN gene. The co-occurrence of predicted deleterious SRPK3/TTN variants was not seen among 76,702 healthy male individuals, and statistical modeling strongly supported digenic inheritance as the best-fitting model. Furthermore, double-mutant zebrafish (srpk3−/−; ttn.1+/−) replicated the myopathic phenotype and showed myofibrillar disorganization. Transcriptome data suggest that the interaction of srpk3 and ttn.1 in zebrafish occurs at a post-transcriptional level. We propose that digenic inheritance of deleterious changes impacting both the protein kinase SRPK3 and the giant muscle protein titin causes a skeletal myopathy and might serve as a model for other genetic diseases. Digenic inheritance of deleterious variants in serine/arginine protein kinase 3 (SRPK3) and titin (TTN) leads to a progressive early onset skeletal muscle myopathy. Zebrafish double mutants exhibit a similar myopathy phenotype accompanied by myofibrillar disorganization.},
  archive      = {J_NG},
  author       = {Töpf, Ana and Cox, Dan and Zaharieva, Irina T. and Di Leo, Valeria and Sarparanta, Jaakko and Jonson, Per Harald and Sealy, Ian M. and Smolnikov, Andrei and White, Richard J. and Vihola, Anna and Savarese, Marco and Merteroglu, Munise and Wali, Neha and Laricchia, Kristen M. and Venturini, Cristina and Vroling, Bas and Stenton, Sarah L. and Cummings, Beryl B. and Harris, Elizabeth and Marini-Bettolo, Chiara and Diaz-Manera, Jordi and Henderson, Matt and Barresi, Rita and Duff, Jennifer and England, Eleina M. and Patrick, Jane and Al-Husayni, Sundos and Biancalana, Valerie and Beggs, Alan H. and Bodi, Istvan and Bommireddipalli, Shobhana and Bönnemann, Carsten G. and Cairns, Anita and Chiew, Mei-Ting and Claeys, Kristl G. and Cooper, Sandra T. and Davis, Mark R. and Donkervoort, Sandra and Erasmus, Corrie E. and Fassad, Mahmoud R. and Genetti, Casie A. and Grosmann, Carla and Jungbluth, Heinz and Kamsteeg, Erik-Jan and Lornage, Xavière and Löscher, Wolfgang N. and Malfatti, Edoardo and Manzur, Adnan and Martí, Pilar and Mongini, Tiziana E. and Muelas, Nuria and Nishikawa, Atsuko and O’Donnell-Luria, Anne and Ogonuki, Narumi and O’Grady, Gina L. and O’Heir, Emily and Paquay, Stéphanie and Phadke, Rahul and Pletcher, Beth A. and Romero, Norma B. and Schouten, Meyke and Shah, Snehal and Smuts, Izelle and Sznajer, Yves and Tasca, Giorgio and Taylor, Robert W. and Tuite, Allysa and Van den Bergh, Peter and VanNoy, Grace and Voermans, Nicol C. and Wanschitz, Julia V. and Wraige, Elizabeth and Yoshimura, Kimihiko and Oates, Emily C. and Nakagawa, Osamu and Nishino, Ichizo and Laporte, Jocelyn and Vilchez, Juan J. and MacArthur, Daniel G. and Sarkozy, Anna and Cordell, Heather J. and Udd, Bjarne and Busch-Nentwich, Elisabeth M. and Muntoni, Francesco and Straub, Volker},
  doi          = {10.1038/s41588-023-01651-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {395-407},
  shortjournal = {Nat. Genet.},
  title        = {Digenic inheritance involving a muscle-specific protein kinase and the giant titin protein causes a skeletal muscle myopathy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cell-type-specific CAG repeat expansions and toxicity of
mutant huntingtin in human striatum and cerebellum. <em>NG</em>,
<em>56</em>(3), 383–394. (<a
href="https://doi.org/10.1038/s41588-024-01653-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain region-specific degeneration and somatic expansions of the mutant Huntingtin (mHTT) CAG tract are key features of Huntington’s disease (HD). However, the relationships among CAG expansions, death of specific cell types and molecular events associated with these processes are not established. Here, we used fluorescence-activated nuclear sorting (FANS) and deep molecular profiling to gain insight into the properties of cell types of the human striatum and cerebellum in HD and control donors. CAG expansions arise at mHTT in striatal medium spiny neurons (MSNs), cholinergic interneurons and cerebellar Purkinje neurons, and at mutant ATXN3 in MSNs from SCA3 donors. CAG expansions in MSNs are associated with higher levels of MSH2 and MSH3 (forming MutSβ), which can inhibit nucleolytic excision of CAG slip-outs by FAN1. Our data support a model in which CAG expansions are necessary but may not be sufficient for cell death and identify transcriptional changes associated with somatic CAG expansions and striatal toxicity. Fluorescence-activated nuclear sorting combined with deep profiling shows that Huntington’s disease repeat expansions arise in specific cell types and are associated with elevated MSH2 and MSH3, which promote expansions in vitro by inhibiting excision of CAG slip-outs by FAN1.},
  archive      = {J_NG},
  author       = {Mätlik, Kert and Baffuto, Matthew and Kus, Laura and Deshmukh, Amit Laxmikant and Davis, David A. and Paul, Matthew R. and Carroll, Thomas S. and Caron, Marie-Christine and Masson, Jean-Yves and Pearson, Christopher E. and Heintz, Nathaniel},
  doi          = {10.1038/s41588-024-01653-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {383-394},
  shortjournal = {Nat. Genet.},
  title        = {Cell-type-specific CAG repeat expansions and toxicity of mutant huntingtin in human striatum and cerebellum},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Distinct and shared genetic architectures of gestational
diabetes mellitus and type 2 diabetes. <em>NG</em>, <em>56</em>(3),
377–382. (<a href="https://doi.org/10.1038/s41588-023-01607-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gestational diabetes mellitus (GDM) is a common metabolic disorder affecting more than 16 million pregnancies annually worldwide1,2. GDM is related to an increased lifetime risk of type 2 diabetes (T2D)1–3, with over a third of women developing T2D within 15 years of their GDM diagnosis. The diseases are hypothesized to share a genetic predisposition1–7, but few studies have sought to uncover the genetic underpinnings of GDM. Most studies have evaluated the impact of T2D loci only8–10, and the three prior genome-wide association studies of GDM11–13 have identified only five loci, limiting the power to assess to what extent variants or biological pathways are specific to GDM. We conducted the largest genome-wide association study of GDM to date in 12,332 cases and 131,109 parous female controls in the FinnGen study and identified 13 GDM-associated loci, including nine new loci. Genetic features distinct from T2D were identified both at the locus and genomic scale. Our results suggest that the genetics of GDM risk falls into the following two distinct categories: one part conventional T2D polygenic risk and one part predominantly influencing mechanisms disrupted in pregnancy. Loci with GDM-predominant effects map to genes related to islet cells, central glucose homeostasis, steroidogenesis and placental expression. Genome-wide association analyses identify 13 loci associated with gestational diabetes, showing partial overlap with type 2 diabetes risk loci but also distinct genetic architecture predominantly influencing pregnancy-related mechanisms.},
  archive      = {J_NG},
  author       = {Elliott, Amanda and Walters, Raymond K. and Pirinen, Matti and Kurki, Mitja and Junna, Nella and Goldstein, Jacqueline I. and Reeve, Mary Pat and Siirtola, Harri and Lemmelä, Susanna M. and Turley, Patrick and Lahtela, Elisa and Mehtonen, Juha and Reis, Kadri and Elnahas, Abdelrahman G. and Reigo, Anu and Palta, Priit and Esko, Tõnu and Mägi, Reedik and Palotie, Aarno and Daly, Mark J. and Widén, Elisabeth},
  doi          = {10.1038/s41588-023-01607-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {377-382},
  shortjournal = {Nat. Genet.},
  title        = {Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A combinatorial genetic strategy for exploring complex
genotype–phenotype associations in cancer. <em>NG</em>, <em>56</em>(3),
371–376. (<a href="https://doi.org/10.1038/s41588-024-01674-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Available genetically defined cancer models are limited in genotypic and phenotypic complexity and underrepresent the heterogeneity of human cancer. Here, we describe a combinatorial genetic strategy applied to an organoid transformation assay to rapidly generate diverse, clinically relevant bladder and prostate cancer models. Importantly, the clonal architecture of the resultant tumors can be resolved using single-cell or spatially resolved next-generation sequencing to uncover polygenic drivers of cancer phenotypes. An approach combining infection of primary human epithelial cells with a barcoded lentiviral-based library followed by engraftment into mice yields biologically relevant models of bladder and prostate cancer harboring complex genetic perturbations.},
  archive      = {J_NG},
  author       = {Li, Shan and Wong, Alicia and Sun, Huiyun and Bhatia, Vipul and Javier, Gerardo and Jana, Sujata and Wu, Qian and Montgomery, Robert B. and Wright, Jonathan L. and Lam, Hung-Ming and Hsieh, Andrew C. and Faltas, Bishoy M. and Haffner, Michael C. and Lee, John K.},
  doi          = {10.1038/s41588-024-01674-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {371-376},
  shortjournal = {Nat. Genet.},
  title        = {A combinatorial genetic strategy for exploring complex genotype–phenotype associations in cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Structural variations fine-tune gene expression to steer
brassica oleracea diversification. <em>NG</em>, <em>56</em>(3), 369–370.
(<a href="https://doi.org/10.1038/s41588-024-01656-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We constructed a pan-genome using 27 high-quality representative Brassica oleracea genomes. Using this pan-genome, together with multi-omics datasets from large-scale populations, we uncovered the important role of structural variations as dosage regulators of gene expression, which drives the morphotype diversification in B. oleracea.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01656-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {369-370},
  shortjournal = {Nat. Genet.},
  title        = {Structural variations fine-tune gene expression to steer brassica oleracea diversification},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Proteins shaping global chromatin accessibility.
<em>NG</em>, <em>56</em>(3), 367–368. (<a
href="https://doi.org/10.1038/s41588-024-01667-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide CRISPR screening coupled with ATAC-see uncovered modulators that influence global chromatin accessibility. Notably, TFDP1 emerged as a pivotal modulator of chromatin accessibility that acts by controlling histone transcription. Depletion of TFDP1 induced a global elevation in accessibility, enhancing the efficiency of genome editing and iPS cell reprogramming.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-01667-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {367-368},
  shortjournal = {Nat. Genet.},
  title        = {Proteins shaping global chromatin accessibility},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improved identification of cancer mutational processes.
<em>NG</em>, <em>56</em>(3), 365–366. (<a
href="https://doi.org/10.1038/s41588-024-01679-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational signatures help to deconvolve the different processes that shape cancer genomes. A new tool now alleviates some of the persistent challenges in the field.},
  archive      = {J_NG},
  author       = {Kaufmann, Tom L. and Schwarz, Roland F.},
  doi          = {10.1038/s41588-024-01679-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {365-366},
  shortjournal = {Nat. Genet.},
  title        = {Improved identification of cancer mutational processes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cellular neighborhood analysis in spatial omics reveals new
tissue domains and cell subtypes. <em>NG</em>, <em>56</em>(3), 362–364.
(<a href="https://doi.org/10.1038/s41588-023-01646-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics enables the molecular profiling of cells with the tissue context preserved. A new analytic approach shows how cellular neighborhood analysis and feature augmentation can spatially connect and cluster millions of cells into higher-order functional units.},
  archive      = {J_NG},
  author       = {Ruitenberg, Marc J. and Nguyen, Quan H.},
  doi          = {10.1038/s41588-023-01646-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {362-364},
  shortjournal = {Nat. Genet.},
  title        = {Cellular neighborhood analysis in spatial omics reveals new tissue domains and cell subtypes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Building chromatin from the ground up. <em>NG</em>,
<em>56</em>(3), 360–361. (<a
href="https://doi.org/10.1038/s41588-024-01666-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A common architectural feature of the genome in many organisms is chromatin domains. A synthetic biology approach now builds chromatin domains from scratch and identifies some of the minimal components needed for their formation.},
  archive      = {J_NG},
  author       = {Busby, Theodore and Misteli, Tom},
  doi          = {10.1038/s41588-024-01666-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {360-361},
  shortjournal = {Nat. Genet.},
  title        = {Building chromatin from the ground up},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). New insights into the genetics of diabetes in pregnancy.
<em>NG</em>, <em>56</em>(3), 358–359. (<a
href="https://doi.org/10.1038/s41588-024-01675-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gestational diabetes is a complex metabolic condition thought to have a strong genetic predisposition. A large genome-wide association study of participants from Finland sheds light on the genetic contributors, opening avenues for research into mechanisms that underlie glucose regulation in pregnancy to improve the health of mothers and babies.},
  archive      = {J_NG},
  author       = {Cissé, Aminata Hallimat and Freathy, Rachel M.},
  doi          = {10.1038/s41588-024-01675-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {358-359},
  shortjournal = {Nat. Genet.},
  title        = {New insights into the genetics of diabetes in pregnancy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). How transposable elements are spliced out. <em>NG</em>,
<em>56</em>(3), 357. (<a
href="https://doi.org/10.1038/s41588-024-01696-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-024-01696-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {357},
  shortjournal = {Nat. Genet.},
  title        = {How transposable elements are spliced out},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Transcription factor binding site affinity and the link to
phenotype. <em>NG</em>, <em>56</em>(3), 357. (<a
href="https://doi.org/10.1038/s41588-024-01697-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-01697-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {357},
  shortjournal = {Nat. Genet.},
  title        = {Transcription factor binding site affinity and the link to phenotype},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Genetic contribution to heterogeneity in type 2 diabetes.
<em>NG</em>, <em>56</em>(3), 357. (<a
href="https://doi.org/10.1038/s41588-024-01698-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-01698-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {357},
  shortjournal = {Nat. Genet.},
  title        = {Genetic contribution to heterogeneity in type 2 diabetes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Implicating XIST in sex-biased autoimmunity. <em>NG</em>,
<em>56</em>(3), 357. (<a
href="https://doi.org/10.1038/s41588-024-01699-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-024-01699-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {357},
  shortjournal = {Nat. Genet.},
  title        = {Implicating XIST in sex-biased autoimmunity},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Conditional transcriptome-wide association study for
fine-mapping candidate causal genes. <em>NG</em>, <em>56</em>(2),
348–356. (<a href="https://doi.org/10.1038/s41588-023-01645-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptome-wide association studies (TWASs) aim to integrate genome-wide association studies with expression-mapping studies to identify genes with genetically predicted expression (GReX) associated with a complex trait. In the present report, we develop a method, GIFT (gene-based integrative fine-mapping through conditional TWAS), that performs conditional TWAS analysis by explicitly controlling for GReX of all other genes residing in a local region to fine-map putatively causal genes. GIFT is frequentist in nature, explicitly models both expression correlation and cis-single nucleotide polymorphism linkage disequilibrium across multiple genes and uses a likelihood framework to account for expression prediction uncertainty. As a result, GIFT produces calibrated P values and is effective for fine-mapping. We apply GIFT to analyze six traits in the UK Biobank, where GIFT narrows down the set size of putatively causal genes by 32.16–91.32% compared with existing TWAS fine-mapping approaches. The genes identified by GIFT highlight the importance of vessel regulation in determining blood pressures and lipid metabolism for regulating lipid levels. GIFT fine-maps candidate causal genes in a transcription-wide association study by conditioning on predicted expression of nearby genes, leading to improved statistical power and enhanced mapping resolution when applied to complex traits.},
  archive      = {J_NG},
  author       = {Liu, Lu and Yan, Ran and Guo, Ping and Ji, Jiadong and Gong, Weiming and Xue, Fuzhong and Yuan, Zhongshang and Zhou, Xiang},
  doi          = {10.1038/s41588-023-01645-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {348-356},
  shortjournal = {Nat. Genet.},
  title        = {Conditional transcriptome-wide association study for fine-mapping candidate causal genes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Adjusting for genetic confounders in transcriptome-wide
association studies improves discovery of risk genes of complex traits.
<em>NG</em>, <em>56</em>(2), 336–347. (<a
href="https://doi.org/10.1038/s41588-023-01648-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many methods have been developed to leverage expression quantitative trait loci (eQTL) data to nominate candidate genes from genome-wide association studies. These methods, including colocalization, transcriptome-wide association studies (TWAS) and Mendelian randomization-based methods; however, all suffer from a key problem—when assessing the role of a gene in a trait using its eQTLs, nearby variants and genetic components of other genes’ expression may be correlated with these eQTLs and have direct effects on the trait, acting as potential confounders. Our extensive simulations showed that existing methods fail to account for these ‘genetic confounders’, resulting in severe inflation of false positives. Our new method, causal-TWAS (cTWAS), borrows ideas from statistical fine-mapping and allows us to adjust all genetic confounders. cTWAS showed calibrated false discovery rates in simulations, and its application on several common traits discovered new candidate genes. In conclusion, cTWAS provides a robust statistical framework for gene discovery. Causal-TWAS (cTWAS) is a statistical framework that adjusts for genetic confounders in transcriptome-wide association studies. Application of cTWAS on common traits leads to reliable detection of candidate causal genes.},
  archive      = {J_NG},
  author       = {Zhao, Siming and Crouse, Wesley and Qian, Sheng and Luo, Kaixuan and Stephens, Matthew and He, Xin},
  doi          = {10.1038/s41588-023-01648-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {336-347},
  shortjournal = {Nat. Genet.},
  title        = {Adjusting for genetic confounders in transcriptome-wide association studies improves discovery of risk genes of complex traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Public platform with 39,472 exome control samples enables
association studies without genotype sharing. <em>NG</em>,
<em>56</em>(2), 327–335. (<a
href="https://doi.org/10.1038/s41588-023-01637-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acquiring a sufficiently powered cohort of control samples matched to a case sample can be time-consuming or, in some cases, impossible. Accordingly, an ability to leverage genetic data from control samples that were already collected elsewhere could dramatically improve power in genetic association studies. Sharing of control samples can pose significant challenges, since most human genetic data are subject to strict sharing regulations. Here, using the properties of singular value decomposition and subsampling algorithm, we developed a method allowing selection of the best-matching controls in an external pool of samples compliant with personal data protection and eliminating the need for genotype sharing. We provide access to a library of 39,472 exome sequencing controls at http://dnascore.net enabling association studies for case cohorts lacking control subjects. Using this approach, control sets can be selected from this online library with a prespecified matching accuracy, ensuring well-calibrated association analysis for both rare and common variants. A new method allows selection of matched controls from an external pool of samples without genotype sharing. This method has been implemented in an online repository containing 39,472 exome sequencing controls that can be used for association analyses.},
  archive      = {J_NG},
  author       = {Artomov, Mykyta and Loboda, Alexander A. and Artyomov, Maxim N. and Daly, Mark J.},
  doi          = {10.1038/s41588-023-01637-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {327-335},
  shortjournal = {Nat. Genet.},
  title        = {Public platform with 39,472 exome control samples enables association studies without genotype sharing},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The ZmWAKL–ZmWIK–ZmBLK1–ZmRBOH4 module provides quantitative
resistance to gray leaf spot in maize. <em>NG</em>, <em>56</em>(2),
315–326. (<a href="https://doi.org/10.1038/s41588-023-01644-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gray leaf spot (GLS), caused by the fungal pathogens Cercospora zeae-maydis and Cercospora zeina, is a major foliar disease of maize worldwide (Zea mays L.). Here we demonstrate that ZmWAKL encoding cell-wall-associated receptor kinase-like protein is the causative gene at the major quantitative disease resistance locus against GLS. The ZmWAKLY protein, encoded by the resistance allele, can self-associate and interact with a leucine-rich repeat immune-related kinase ZmWIK on the plasma membrane. The ZmWAKLY/ZmWIK receptor complex interacts with and phosphorylates the receptor-like cytoplasmic kinase (RLCK) ZmBLK1, which in turn phosphorylates its downstream NADPH oxidase ZmRBOH4. Upon pathogen infection, ZmWAKLY phosphorylation activity is transiently increased, initiating immune signaling from ZmWAKLY, ZmWIK, ZmBLK1 to ZmRBOH4, ultimately triggering a reactive oxygen species burst. Our study thus uncovers the role of the maize ZmWAKL–ZmWIK–ZmBLK1–ZmRBOH4 receptor/signaling/executor module in perceiving the pathogen invasion, transducing immune signals, activating defense responses and conferring increased resistance to GLS. ZmWAKL, which encodes a cell-wall-associated receptor kinase-like protein, regulates quantitative disease resistance to gray leaf spot in maize through the ZmWAKL–ZmWIK–ZmBLK1–ZmRBOH4 module.},
  archive      = {J_NG},
  author       = {Zhong, Tao and Zhu, Mang and Zhang, Qianqian and Zhang, Yan and Deng, Suining and Guo, Chenyu and Xu, Ling and Liu, Tingting and Li, Yancong and Bi, Yaqi and Fan, Xingming and Balint-Kurti, Peter and Xu, Mingliang},
  doi          = {10.1038/s41588-023-01644-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {315-326},
  shortjournal = {Nat. Genet.},
  title        = {The ZmWAKL–ZmWIK–ZmBLK1–ZmRBOH4 module provides quantitative resistance to gray leaf spot in maize},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Boundary stacking interactions enable cross-TAD
enhancer–promoter communication during limb development. <em>NG</em>,
<em>56</em>(2), 306–314. (<a
href="https://doi.org/10.1038/s41588-023-01641-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although promoters and their enhancers are frequently contained within a topologically associating domain (TAD), some developmentally important genes have their promoter and enhancers within different TADs. Hypotheses about molecular mechanisms enabling cross-TAD interactions remain to be assessed. To test these hypotheses, we used optical reconstruction of chromatin architecture to characterize the conformations of the Pitx1 locus on single chromosomes in developing mouse limbs. Our data support a model in which neighboring boundaries are stacked as a result of loop extrusion, bringing boundary-proximal cis-elements into contact. This stacking interaction also contributes to the appearance of architectural stripes in the population average maps. Through molecular dynamics simulations, we found that increasing boundary strengths facilitates the formation of the stacked boundary conformation, counter-intuitively facilitating border bypass. This work provides a revised view of the TAD borders’ function, both facilitating and preventing cis-regulatory interactions, and introduces a framework to distinguish border-crossing from border-respecting enhancer–promoter pairs. A combination of single-cell imaging and dynamic polymer simulation shows that stacked boundary conformation facilitates cis-regulatory elements communication across topologically associating domain (TAD) borders at the Pitx1 locus in developing mouse limbs.},
  archive      = {J_NG},
  author       = {Hung, Tzu-Chiao and Kingsley, David M. and Boettiger, Alistair N.},
  doi          = {10.1038/s41588-023-01641-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {306-314},
  shortjournal = {Nat. Genet.},
  title        = {Boundary stacking interactions enable cross-TAD enhancer–promoter communication during limb development},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Single-nucleus multi-omic profiling of human placental
syncytiotrophoblasts identifies cellular trajectories during pregnancy.
<em>NG</em>, <em>56</em>(2), 294–305. (<a
href="https://doi.org/10.1038/s41588-023-01647-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human placenta has a vital role in ensuring a successful pregnancy. Despite the growing body of knowledge about its cellular compositions and functions, there has been limited research on the heterogeneity of the billions of nuclei within the syncytiotrophoblast (STB), a multinucleated entity primarily responsible for placental function. Here we conducted integrated single-nucleus RNA sequencing and single-nucleus ATAC sequencing analyses of human placentas from early and late pregnancy. Our findings demonstrate the dynamic heterogeneity and developmental trajectories of STB nuclei and their correspondence with human trophoblast stem cell (hTSC)-derived STB. Furthermore, we identified transcription factors associated with diverse STB nuclear lineages through their gene regulatory networks and experimentally confirmed their function in hTSC and trophoblast organoid-derived STBs. Together, our data provide insights into the heterogeneity of human STB and represent a valuable resource for interpreting associated pregnancy complications. Multi-omic analysis of single nuclei from 12 human placentas collected during early-stage and late-stage pregnancy characterizes syncytiotrophoblast diversity. Gene regulatory network analysis implicates candidate lineage regulators such as STAT5A and CEBPB.},
  archive      = {J_NG},
  author       = {Wang, Meijiao and Liu, Yawei and Sun, Run and Liu, Fenting and Li, Jiaqian and Yan, Long and Zhang, Jixiang and Xie, Xinwei and Li, Dongxu and Wang, Yiming and Li, Shiwen and Zhu, Xili and Li, Rong and Lu, Falong and Xiao, Zhenyu and Wang, Hongmei},
  doi          = {10.1038/s41588-023-01647-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {294-305},
  shortjournal = {Nat. Genet.},
  title        = {Single-nucleus multi-omic profiling of human placental syncytiotrophoblasts identifies cellular trajectories during pregnancy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A new genomic framework to categorize pediatric acute
myeloid leukemia. <em>NG</em>, <em>56</em>(2), 281–293. (<a
href="https://doi.org/10.1038/s41588-023-01640-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent studies on pediatric acute myeloid leukemia (pAML) have revealed pediatric-specific driver alterations, many of which are underrepresented in the current classification schemas. To comprehensively define the genomic landscape of pAML, we systematically categorized 887 pAML into 23 mutually distinct molecular categories, including new major entities such as UBTF or BCL11B, covering 91.4% of the cohort. These molecular categories were associated with unique expression profiles and mutational patterns. For instance, molecular categories characterized by specific HOXA or HOXB expression signatures showed distinct mutation patterns of RAS pathway genes, FLT3 or WT1, suggesting shared biological mechanisms. We show that molecular categories were strongly associated with clinical outcomes using two independent cohorts, leading to the establishment of a new prognostic framework for pAML based on these updated molecular categories and minimal residual disease. Together, this comprehensive diagnostic and prognostic framework forms the basis for future classification of pAML and treatment strategies. Genomic profiling of 887 pediatric AML samples highlights 23 groups with distinct molecular and clinical features.},
  archive      = {J_NG},
  author       = {Umeda, Masayuki and Ma, Jing and Westover, Tamara and Ni, Yonghui and Song, Guangchun and Maciaszek, Jamie L. and Rusch, Michael and Rahbarinia, Delaram and Foy, Scott and Huang, Benjamin J. and Walsh, Michael P. and Kumar, Priyadarshini and Liu, Yanling and Yang, Wenjian and Fan, Yiping and Wu, Gang and Baker, Sharyn D. and Ma, Xiaotu and Wang, Lu and Alonzo, Todd A. and Rubnitz, Jeffrey E. and Pounds, Stanley and Klco, Jeffery M.},
  doi          = {10.1038/s41588-023-01640-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {281-293},
  shortjournal = {Nat. Genet.},
  title        = {A new genomic framework to categorize pediatric acute myeloid leukemia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inherited polygenic effects on common hematological traits
influence clonal selection on JAK2V617F and the development of
myeloproliferative neoplasms. <em>NG</em>, <em>56</em>(2), 273–280. (<a
href="https://doi.org/10.1038/s41588-023-01638-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Myeloproliferative neoplasms (MPNs) are chronic cancers characterized by overproduction of mature blood cells. Their causative somatic mutations, for example, JAK2V617F, are common in the population, yet only a minority of carriers develop MPN. Here we show that the inherited polygenic loci that underlie common hematological traits influence JAK2V617F clonal expansion. We identify polygenic risk scores (PGSs) for monocyte count and plateletcrit as new risk factors for JAK2V617F positivity. PGSs for several hematological traits influenced the risk of different MPN subtypes, with low PGSs for two platelet traits also showing protective effects in JAK2V617F carriers, making them two to three times less likely to have essential thrombocythemia than carriers with high PGSs. We observed that extreme hematological PGSs may contribute to an MPN diagnosis in the absence of somatic driver mutations. Our study showcases how polygenic backgrounds underlying common hematological traits influence both clonal selection on somatic mutations and the subsequent phenotype of cancer. Inherited polygenic scores for blood cell traits are associated with an increased risk of JAK2V617F clonal expansion and influence clinical phenotypes in individuals with myeloproliferative neoplasms.},
  archive      = {J_NG},
  author       = {Guo, Jing and Walter, Klaudia and Quiros, Pedro M. and Gu, Muxin and Baxter, E. Joanna and Danesh, John and Di Angelantonio, Emanuele and Roberts, David and Guglielmelli, Paola and Harrison, Claire N. and Godfrey, Anna L. and Green, Anthony R. and Vassiliou, George S. and Vuckovic, Dragana and Nangalia, Jyoti and Soranzo, Nicole},
  doi          = {10.1038/s41588-023-01638-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {273-280},
  shortjournal = {Nat. Genet.},
  title        = {Inherited polygenic effects on common hematological traits influence clonal selection on JAK2V617F and the development of myeloproliferative neoplasms},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Integrative functional genomic analyses identify genetic
variants influencing skin pigmentation in africans. <em>NG</em>,
<em>56</em>(2), 258–272. (<a
href="https://doi.org/10.1038/s41588-023-01626-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Skin color is highly variable in Africans, yet little is known about the underlying molecular mechanism. Here we applied massively parallel reporter assays to screen 1,157 candidate variants influencing skin pigmentation in Africans and identified 165 single-nucleotide polymorphisms showing differential regulatory activities between alleles. We combine Hi-C, genome editing and melanin assays to identify regulatory elements for MFSD12, HMG20B, OCA2, MITF, LEF1, TRPS1, BLOC1S6 and CYB561A3 that impact melanin levels in vitro and modulate human skin color. We found that independent mutations in an OCA2 enhancer contribute to the evolution of human skin color diversity and detect signals of local adaptation at enhancers of MITF, LEF1 and TRPS1, which may contribute to the light skin color of Khoesan-speaking populations from Southern Africa. Additionally, we identified CYB561A3 as a novel pigmentation regulator that impacts genes involved in oxidative phosphorylation and melanogenesis. These results provide insights into the mechanisms underlying human skin color diversity and adaptive evolution. Massively parallel reporter assays identify 165 functional variants associated with skin pigmentation in ethnically diverse Africans. Functional characterization of eight variants demonstrates their impact in regulating melanin levels and validates CYB561A3 as a novel gene involved in melanogenesis and pigmentation.},
  archive      = {J_NG},
  author       = {Feng, Yuanqing and Xie, Ning and Inoue, Fumitaka and Fan, Shaohua and Saskin, Joshua and Zhang, Chao and Zhang, Fang and Hansen, Matthew E. B. and Nyambo, Thomas and Mpoloka, Sununguko Wata and Mokone, Gaonyadiwe George and Fokunang, Charles and Belay, Gurja and Njamnshi, Alfred K. and Marks, Michael S. and Oancea, Elena and Ahituv, Nadav and Tishkoff, Sarah A.},
  doi          = {10.1038/s41588-023-01626-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {258-272},
  shortjournal = {Nat. Genet.},
  title        = {Integrative functional genomic analyses identify genetic variants influencing skin pigmentation in africans},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic architecture of cardiac dynamic flow volumes.
<em>NG</em>, <em>56</em>(2), 245–257. (<a
href="https://doi.org/10.1038/s41588-023-01587-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cardiac blood flow is a critical determinant of human health. However, the definition of its genetic architecture is limited by the technical challenge of capturing dynamic flow volumes from cardiac imaging at scale. We present DeepFlow, a deep-learning system to extract cardiac flow and volumes from phase-contrast cardiac magnetic resonance imaging. A mixed-linear model applied to 37,653 individuals from the UK Biobank reveals genome-wide significant associations across cardiac dynamic flow volumes spanning from aortic forward velocity to aortic regurgitation fraction. Mendelian randomization reveals a causal role for aortic root size in aortic valve regurgitation. Among the most significant contributing variants, localizing genes (near ELN, PRDM6 and ADAMTS7) are implicated in connective tissue and blood pressure pathways. Here we show that DeepFlow cardiac flow phenotyping at scale, combined with genotyping data, reinforces the contribution of connective tissue genes, blood pressure and root size to aortic valve function. DeepFlow uses deep learning to extract cardiac flow phenotypes from phase-contrast magnetic resonance images. Genome-wide analyses of cardiac flow traits in UK Biobank highlight the contribution of connective tissue genes to aortic valve function.},
  archive      = {J_NG},
  author       = {Gomes, Bruna and Singh, Aditya and O’Sullivan, Jack W. and Schnurr, Theresia M. and Goddard, Pagé C. and Loong, Shaun and Amar, David and Hughes, J. Weston and Kostur, Mykhailo and Haddad, Francois and Salerno, Michael and Foo, Roger and Montgomery, Stephen B. and Parikh, Victoria N. and Meder, Benjamin and Ashley, Euan A.},
  doi          = {10.1038/s41588-023-01587-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {245-257},
  shortjournal = {Nat. Genet.},
  title        = {Genetic architecture of cardiac dynamic flow volumes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Polygenic profiles define aspects of clinical heterogeneity
in attention deficit hyperactivity disorder. <em>NG</em>,
<em>56</em>(2), 234–244. (<a
href="https://doi.org/10.1038/s41588-023-01593-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attention deficit hyperactivity disorder (ADHD) is a complex disorder that manifests variability in long-term outcomes and clinical presentations. The genetic contributions to such heterogeneity are not well understood. Here we show several genetic links to clinical heterogeneity in ADHD in a case-only study of 14,084 diagnosed individuals. First, we identify one genome-wide significant locus by comparing cases with ADHD and autism spectrum disorder (ASD) to cases with ADHD but not ASD. Second, we show that cases with ASD and ADHD, substance use disorder and ADHD, or first diagnosed with ADHD in adulthood have unique polygenic score (PGS) profiles that distinguish them from complementary case subgroups and controls. Finally, a PGS for an ASD diagnosis in ADHD cases predicted cognitive performance in an independent developmental cohort. Our approach uncovered evidence of genetic heterogeneity in ADHD, helping us to understand its etiology and providing a model for studies of other disorders. Genome-wide analyses yield insights into the polygenic effects contributing to clinical heterogeneity in attention deficit hyperactivity disorder, advancing understanding of its genetic etiology and serving as a model for future studies in other complex disorders.},
  archive      = {J_NG},
  author       = {LaBianca, Sonja and Brikell, Isabell and Helenius, Dorte and Loughnan, Robert and Mefford, Joel and Palmer, Clare E. and Walker, Rebecca and Gådin, Jesper R. and Krebs, Morten and Appadurai, Vivek and Vaez, Morteza and Agerbo, Esben and Pedersen, Marianne Giørtz and Børglum, Anders D. and Hougaard, David M. and Mors, Ole and Nordentoft, Merete and Mortensen, Preben Bo and Kendler, Kenneth S. and Jernigan, Terry L. and Geschwind, Daniel H. and Ingason, Andrés and Dahl, Andrew W. and Zaitlen, Noah and Dalsgaard, Søren and Werge, Thomas M. and Schork, Andrew J.},
  doi          = {10.1038/s41588-023-01593-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {234-244},
  shortjournal = {Nat. Genet.},
  title        = {Polygenic profiles define aspects of clinical heterogeneity in attention deficit hyperactivity disorder},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multi-ancestry genome-wide association study of major
depression aids locus discovery, fine mapping, gene prioritization and
causal inference. <em>NG</em>, <em>56</em>(2), 222–233. (<a
href="https://doi.org/10.1038/s41588-023-01596-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most genome-wide association studies (GWAS) of major depression (MD) have been conducted in samples of European ancestry. Here we report a multi-ancestry GWAS of MD, adding data from 21 cohorts with 88,316 MD cases and 902,757 controls to previously reported data. This analysis used a range of measures to define MD and included samples of African (36% of effective sample size), East Asian (26%) and South Asian (6%) ancestry and Hispanic/Latin American participants (32%). The multi-ancestry GWAS identified 53 significantly associated novel loci. For loci from GWAS in European ancestry samples, fewer than expected were transferable to other ancestry groups. Fine mapping benefited from additional sample diversity. A transcriptome-wide association study identified 205 significantly associated novel genes. These findings suggest that, for MD, increasing ancestral and global diversity in genetic studies may be particularly important to ensure discovery of core genes and inform about transferability of findings. Multi-ancestry genome-wide association meta-analysis of major depression identifies new risk loci, assesses the transferability of risk loci across ancestry groups, and improves fine-mapping resolution and prioritization of candidate effector genes.},
  archive      = {J_NG},
  author       = {Meng, Xiangrui and Navoly, Georgina and Giannakopoulou, Olga and Levey, Daniel F. and Koller, Dora and Pathak, Gita A. and Koen, Nastassja and Lin, Kuang and Adams, Mark J. and Rentería, Miguel E. and Feng, Yanzhe and Gaziano, J. Michael and Stein, Dan J. and Zar, Heather J. and Campbell, Megan L. and van Heel, David A. and Trivedi, Bhavi and Finer, Sarah and McQuillin, Andrew and Bass, Nick and Chundru, V. Kartik and Martin, Hilary C. and Huang, Qin Qin and Valkovskaya, Maria and Chu, Chia-Yi and Kanjira, Susan and Kuo, Po-Hsiu and Chen, Hsi-Chung and Tsai, Shih-Jen and Liu, Yu-Li and Kendler, Kenneth S. and Peterson, Roseann E. and Cai, Na and Fang, Yu and Sen, Srijan and Scott, Laura J. and Burmeister, Margit and Loos, Ruth J. F. and Preuss, Michael H. and Actkins, Ky’Era V. and Davis, Lea K. and Uddin, Monica and Wani, Agaz H. and Wildman, Derek E. and Aiello, Allison E. and Ursano, Robert J. and Kessler, Ronald C. and Kanai, Masahiro and Okada, Yukinori and Sakaue, Saori and Rabinowitz, Jill A. and Maher, Brion S. and Uhl, George and Eaton, William and Cruz-Fuentes, Carlos S. and Martinez-Levy, Gabriela A. and Campos, Adrian I. and Millwood, Iona Y. and Chen, Zhengming and Li, Liming and Wassertheil-Smoller, Sylvia and Jiang, Yunxuan and Tian, Chao and Martin, Nicholas G. and Mitchell, Brittany L. and Byrne, Enda M. and Awasthi, Swapnil and Coleman, Jonathan R. I. and Ripke, Stephan and Sofer, Tamar and Walters, Robin G. and McIntosh, Andrew M. and Polimanti, Renato and Dunn, Erin C. and Stein, Murray B. and Gelernter, Joel and Lewis, Cathryn M. and Kuchenbaecker, Karoline},
  doi          = {10.1038/s41588-023-01596-4},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {222-233},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genome-wide association study of major depression aids locus discovery, fine mapping, gene prioritization and causal inference},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Comprehensive genetic study of the insulin resistance marker
TG: HDL-c in the UK biobank. <em>NG</em>, <em>56</em>(2), 212–221. (<a
href="https://doi.org/10.1038/s41588-023-01625-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Insulin resistance (IR) is a well-established risk factor for metabolic disease. The ratio of triglycerides to high-density lipoprotein cholesterol (TG:HDL-C) is a surrogate marker of IR. We conducted a genome-wide association study of the TG:HDL-C ratio in 402,398 Europeans within the UK Biobank. We identified 369 independent SNPs, of which 114 had a false discovery rate-adjusted P value &amp;lt; 0.05 in other genome-wide studies of IR making them high-confidence IR-associated loci. Seventy-two of these 114 loci have not been previously associated with IR. These 114 loci cluster into five groups upon phenome-wide analysis and are enriched for candidate genes important in insulin signaling, adipocyte physiology and protein metabolism. We created a polygenic-risk score from the high-confidence IR-associated loci using 51,550 European individuals in the Michigan Genomics Initiative. We identified associations with diabetes, hyperglyceridemia, hypertension, nonalcoholic fatty liver disease and ischemic heart disease. Collectively, this study provides insight into the genes, pathways, tissues and subtypes critical in IR. Genome-wide association analysis of triglycerides to high-density lipoprotein cholesterol (TG:HDL-C) ratio within the UK Biobank identifies candidate insulin resistance-associated loci linked to metabolic pathways and insulin biology. A polygenic risk score derived from these results shows an association with multiple cardiometabolic traits.},
  archive      = {J_NG},
  author       = {Oliveri, Antonino and Rebernick, Ryan J and Kuppa, Annapurna and Pant, Asmita and Chen, Yanhua and Du, Xiaomeng and Cushing, Kelly C. and Bell, Hannah N and Raut, Chinmay and Prabhu, Ponnandy and Chen, Vincent L. and Halligan, Brian D. and Speliotes, Elizabeth K.},
  doi          = {10.1038/s41588-023-01625-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {212-221},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive genetic study of the insulin resistance marker TG: HDL-C in the UK biobank},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Distinguishing between driver and passenger mechanisms of
aging. <em>NG</em>, <em>56</em>(2), 204–211. (<a
href="https://doi.org/10.1038/s41588-023-01627-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding why we age is a long-standing question, and many mechanistic theories of aging have been proposed. Owing to limitations in studying the aging process, including a lack of adequate quantitative measurements, its mechanistic basis remains a subject of debate. Here, I explore theories of aging from the perspective of causal relationships. Many aging-related changes have been observed and touted as drivers of aging, including molecular changes in the genome, telomeres, mitochondria, epigenome and proteins and cellular changes affecting stem cells, the immune system and senescent cell buildup. Determining which changes are drivers and not passengers of aging remains a challenge, however, and I discuss how animal models and human genetic studies have been used empirically to infer causality. Overall, our understanding of the drivers of human aging is still inadequate; yet with a global aging population, elucidating the causes of aging has the potential to revolutionize biomedical research. This Review explores mechanistic theories of aging, discusses challenges in establishing causality of these mechanisms and suggests that genetically informed investigation will help address this gap.},
  archive      = {J_NG},
  author       = {de Magalhães, João Pedro},
  doi          = {10.1038/s41588-023-01627-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {204-211},
  shortjournal = {Nat. Genet.},
  title        = {Distinguishing between driver and passenger mechanisms of aging},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). New genomic classification of pediatric acute myeloid
leukemia. <em>NG</em>, <em>56</em>(2), 202–203. (<a
href="https://doi.org/10.1038/s41588-023-01639-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic background of pediatric acute myeloid leukemia (AML) does not fit with classification systems developed for adult AML. This study investigates the genetic background of pediatric AML and proposes a genomic framework for improved classification and risk stratification based on the driver alterations.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01639-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {202-203},
  shortjournal = {Nat. Genet.},
  title        = {New genomic classification of pediatric acute myeloid leukemia},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unravelling the molecular mechanisms of skin color diversity
in africans. <em>NG</em>, <em>56</em>(2), 200–201. (<a
href="https://doi.org/10.1038/s41588-023-01643-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Skin color is highly variable in Africans, but the underlying molecular mechanisms remain poorly understood. Using population genetics and functional genomics, we identified key genetic variants, regulatory elements and genes that affect skin pigmentation, an adaptive trait, which provides valuable insights into the mechanisms underlying human skin color diversity and evolution.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01643-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {200-201},
  shortjournal = {Nat. Genet.},
  title        = {Unravelling the molecular mechanisms of skin color diversity in africans},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Genetic analysis of cardiac dynamic flow volumes identifies
loci mapping aortic root size. <em>NG</em>, <em>56</em>(2), 197–199. (<a
href="https://doi.org/10.1038/s41588-023-01650-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An open-source automated algorithm called DeepFlow enables large-scale derivation of aortic flow measurements, and genetic analysis of aortic flow, structural and functional traits demonstrates a causal relationship between aortic size and aortic valve regurgitation.},
  archive      = {J_NG},
  author       = {Munroe, Patricia B. and Aung, Nay and Ramírez, Julia},
  doi          = {10.1038/s41588-023-01650-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {197-199},
  shortjournal = {Nat. Genet.},
  title        = {Genetic analysis of cardiac dynamic flow volumes identifies loci mapping aortic root size},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Connecting clinical and genetic heterogeneity in ADHD.
<em>NG</em>, <em>56</em>(2), 195–196. (<a
href="https://doi.org/10.1038/s41588-024-01652-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding clinical heterogeneity in attention deficit hyperactivity disorder (ADHD) is important for improving personalized care and long-term outcomes. A study exploits the large scale and breadth of phenotyping of the iPSYCH cohort to link clinical heterogeneity to genetic heterogeneity in ADHD.},
  archive      = {J_NG},
  author       = {Yap, Chloe X. and Gratten, Jacob},
  doi          = {10.1038/s41588-024-01652-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {195-196},
  shortjournal = {Nat. Genet.},
  title        = {Connecting clinical and genetic heterogeneity in ADHD},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Establishment of DNA replication timing during mammalian
development. <em>NG</em>, <em>56</em>(2), 194. (<a
href="https://doi.org/10.1038/s41588-024-01670-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-024-01670-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {194},
  shortjournal = {Nat. Genet.},
  title        = {Establishment of DNA replication timing during mammalian development},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024h). Rare CTR9 variants and myeloid malignancies. <em>NG</em>,
<em>56</em>(2), 194. (<a
href="https://doi.org/10.1038/s41588-024-01671-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-024-01671-4},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {194},
  shortjournal = {Nat. Genet.},
  title        = {Rare CTR9 variants and myeloid malignancies},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). Linking host genetics to gut microbial variation.
<em>NG</em>, <em>56</em>(2), 194. (<a
href="https://doi.org/10.1038/s41588-024-01672-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-024-01672-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {194},
  shortjournal = {Nat. Genet.},
  title        = {Linking host genetics to gut microbial variation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). The genetic origins of multiple sclerosis. <em>NG</em>,
<em>56</em>(2), 194. (<a
href="https://doi.org/10.1038/s41588-024-01673-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-024-01673-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {194},
  shortjournal = {Nat. Genet.},
  title        = {The genetic origins of multiple sclerosis},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Advancing diagnosis and research for rare genetic diseases
in indigenous peoples. <em>NG</em>, <em>56</em>(2), 189–193. (<a
href="https://doi.org/10.1038/s41588-023-01642-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Achieving a diagnosis for Indigenous people living with a rare, often genetic, disease is crucial for equitable healthcare. The International Rare Disease Research Consortium convened a global Task Force to bridge the gap in diagnosing Indigenous rare diseases, and identify solutions to tackle the health inequity faced by Indigenous people.},
  archive      = {J_NG},
  author       = {Baynam, Gareth and Julkowska, Daria and Bowdin, Sarah and Hermes, Azure and McMaster, Christopher R. and Prichep, Elissa and Richer, Étienne and van der Westhuizen, Francois H. and Repetto, Gabriela M. and Malherbe, Helen and Reichardt, Juergen K. V. and Arbour, Laura and Hudson, Maui and du Plessis, Kelly and Haendel, Melissa and Wilcox, Phillip and Lynch, Sally Ann and Rind, Shamir and Easteal, Simon and Estivill, Xavier and Caron, Nadine and Chongo, Meck and Thomas, Yarlalu and Letinturier, Mary Catherine V. and Vorster, Barend Christiaan},
  doi          = {10.1038/s41588-023-01642-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {189-193},
  shortjournal = {Nat. Genet.},
  title        = {Advancing diagnosis and research for rare genetic diseases in indigenous peoples},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Chromosome-level genomes of three key
allium crops and their trait evolution. <em>NG</em>, <em>56</em>(1),
187. (<a href="https://doi.org/10.1038/s41588-023-01610-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Hao, Fei and Liu, Xue and Zhou, Botong and Tian, Zunzhe and Zhou, Lina and Zong, Hang and Qi, Jiyan and He, Juan and Zhang, Yongting and Zeng, Peng and Li, Qiong and Wang, Kai and Xia, Keke and Guo, Xing and Li, Li and Shao, Wenwen and Zhang, Bohan and Li, Shengkang and Yang, Haifeng and Hui, Linchong and Chen, Wei and Peng, Lixin and Liu, Feipeng and Rong, Zi-Qiang and Peng, Yingmei and Zhu, Wenbo and McCallum, John A. and Li, Zhen and Xu, Xun and Yang, Hui and Macknight, Richard C. and Wang, Wen and Cai, Jing},
  doi          = {10.1038/s41588-023-01610-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {187},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Chromosome-level genomes of three key allium crops and their trait evolution},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). BridgePRS leverages shared genetic effects across ancestries
to increase polygenic risk score portability. <em>NG</em>,
<em>56</em>(1), 180–186. (<a
href="https://doi.org/10.1038/s41588-023-01583-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present BridgePRS, a novel Bayesian polygenic risk score (PRS) method that leverages shared genetic effects across ancestries to increase PRS portability. We evaluate BridgePRS via simulations and real UK Biobank data across 19 traits in individuals of African, South Asian and East Asian ancestry, using both UK Biobank and Biobank Japan genome-wide association study summary statistics; out-of-cohort validation is performed in the Mount Sinai (New York) BioMe biobank. BridgePRS is compared with the leading alternative, PRS-CSx, and two other PRS methods. Simulations suggest that the performance of BridgePRS relative to PRS-CSx increases as uncertainty increases: with lower trait heritability, higher polygenicity and greater between-population genetic diversity; and when causal variants are not present in the data. In real data, BridgePRS has a 61% larger average R2 than PRS-CSx in out-of-cohort prediction of African ancestry samples in BioMe (P = 6 × 10−5). BridgePRS is a computationally efficient, user-friendly and powerful approach for PRS analyses in non-European ancestries. A powerful Bayesian method, BridgePRS, leverages shared genetic effects across ancestries to increase polygenic risk score portability in non-European populations.},
  archive      = {J_NG},
  author       = {Hoggart, Clive J. and Choi, Shing Wan and García-González, Judit and Souaiaia, Tade and Preuss, Michael and O’Reilly, Paul F.},
  doi          = {10.1038/s41588-023-01583-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {180-186},
  shortjournal = {Nat. Genet.},
  title        = {BridgePRS leverages shared genetic effects across ancestries to increase polygenic risk score portability},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MESuSiE enables scalable and powerful multi-ancestry
fine-mapping of causal variants in genome-wide association studies.
<em>NG</em>, <em>56</em>(1), 170–179. (<a
href="https://doi.org/10.1038/s41588-023-01604-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fine-mapping in genome-wide association studies attempts to identify causal SNPs from a set of candidate SNPs in a local genomic region of interest and is commonly performed in one genetic ancestry at a time. Here, we present multi-ancestry sum of the single effects model (MESuSiE), a probabilistic multi-ancestry fine-mapping method, to improve the accuracy and resolution of fine-mapping by leveraging association information across ancestries. MESuSiE uses summary statistics as input, accounts for the diverse linkage disequilibrium pattern observed in different ancestries, explicitly models both shared and ancestry-specific causal SNPs, and relies on a variational inference algorithm for scalable computation. We evaluated the performance of MESuSiE through comprehensive simulations and multi-ancestry fine-mapping of four lipid traits with both European and African samples. In the real data, MESuSiE improves fine-mapping resolution by 19.0% to 72.0% compared to existing approaches, is an order of magnitude faster, and captures and categorizes shared and ancestry-specific causal signals with enhanced functional enrichment. MESuSiE extends fine-mapping approaches to multi-ancestry analysis using LD-aware bivariate normal mixture models with a variational algorithm to identify shared and ancestry-specific causal variants.},
  archive      = {J_NG},
  author       = {Gao, Boran and Zhou, Xiang},
  doi          = {10.1038/s41588-023-01604-7},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {170-179},
  shortjournal = {Nat. Genet.},
  title        = {MESuSiE enables scalable and powerful multi-ancestry fine-mapping of causal variants in genome-wide association studies},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving fine-mapping by modeling infinitesimal effects.
<em>NG</em>, <em>56</em>(1), 162–169. (<a
href="https://doi.org/10.1038/s41588-023-01597-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fine-mapping aims to identify causal genetic variants for phenotypes. Bayesian fine-mapping algorithms (for example, SuSiE, FINEMAP, ABF and COJO-ABF) are widely used, but assessing posterior probability calibration remains challenging in real data, where model misspecification probably exists, and true causal variants are unknown. We introduce replication failure rate (RFR), a metric to assess fine-mapping consistency by downsampling. SuSiE, FINEMAP and COJO-ABF show high RFR, indicating potential overconfidence in their output. Simulations reveal that nonsparse genetic architecture can lead to miscalibration, while imputation noise, nonuniform distribution of causal variants and quality control filters have minimal impact. Here we present SuSiE-inf and FINEMAP-inf, fine-mapping methods modeling infinitesimal effects alongside fewer larger causal effects. Our methods show improved calibration, RFR and functional enrichment, competitive recall and computational efficiency. Notably, using our methods’ posterior effect sizes substantially increases polygenic risk score accuracy over SuSiE and FINEMAP. Our work improves causal variant identification for complex traits, a fundamental goal of human genetics. A downsampling approach to assess causal variant fine-mapping, replication failure rate, finds that commonly used methods may be miscalibrated. Simulations suggest this is probably due to a nonsparse genetic architecture model misspecification. Incorporating infinitesimal effects in the SuSiE and FINEMAP frameworks improves performance.},
  archive      = {J_NG},
  author       = {Cui, Ran and Elzur, Roy A. and Kanai, Masahiro and Ulirsch, Jacob C. and Weissbrod, Omer and Daly, Mark J. and Neale, Benjamin M. and Fan, Zhou and Finucane, Hilary K.},
  doi          = {10.1038/s41588-023-01597-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {162-169},
  shortjournal = {Nat. Genet.},
  title        = {Improving fine-mapping by modeling infinitesimal effects},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Inferring compound heterozygosity from large-scale exome
sequencing data. <em>NG</em>, <em>56</em>(1), 152–161. (<a
href="https://doi.org/10.1038/s41588-023-01608-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recessive diseases arise when both copies of a gene are impacted by a damaging genetic variant. When a patient carries two potentially causal variants in a gene, accurate diagnosis requires determining that these variants occur on different copies of the chromosome (that is, are in trans) rather than on the same copy (that is, in cis). However, current approaches for determining phase, beyond parental testing, are limited in clinical settings. Here we developed a strategy for inferring phase for rare variant pairs within genes, leveraging genotypes observed in the Genome Aggregation Database (v2, n = 125,748 exomes). Our approach estimates phase with 96% accuracy, both in trio data and in patients with Mendelian conditions and presumed causal compound heterozygous variants. We provide a public resource of phasing estimates for coding variants and counts per gene of rare variants in trans that can aid interpretation of rare co-occurring variants in the context of recessive disease. A strategy for inferring phase for rare variant pairs is applied to exome sequencing data for 125,748 individuals from the Genome Aggregation Database (gnomAD). This resource will aid interpretation of rare co-occurring variants in the context of recessive disease.},
  archive      = {J_NG},
  author       = {Guo, Michael H. and Francioli, Laurent C. and Stenton, Sarah L. and Goodrich, Julia K. and Watts, Nicholas A. and Singer-Berk, Moriel and Groopman, Emily and Darnowsky, Philip W. and Solomonson, Matthew and Baxter, Samantha and Tiao, Grace and Neale, Benjamin M. and Hirschhorn, Joel N. and Rehm, Heidi L. and Daly, Mark J. and O’Donnell-Luria, Anne and Karczewski, Konrad J. and MacArthur, Daniel G. and Samocha, Kaitlin E.},
  doi          = {10.1038/s41588-023-01608-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {152-161},
  shortjournal = {Nat. Genet.},
  title        = {Inferring compound heterozygosity from large-scale exome sequencing data},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accurate detection of identity-by-descent segments in human
ancient DNA. <em>NG</em>, <em>56</em>(1), 143–151. (<a
href="https://doi.org/10.1038/s41588-023-01582-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long DNA segments shared between two individuals, known as identity-by-descent (IBD), reveal recent genealogical connections. Here we introduce ancIBD, a method for identifying IBD segments in ancient human DNA (aDNA) using a hidden Markov model and imputed genotype probabilities. We demonstrate that ancIBD accurately identifies IBD segments &amp;gt;8 cM for aDNA data with an average depth of &amp;gt;0.25× for whole-genome sequencing or &amp;gt;1× for 1240k single nucleotide polymorphism capture data. Applying ancIBD to 4,248 ancient Eurasian individuals, we identify relatives up to the sixth degree and genealogical connections between archaeological groups. Notably, we reveal long IBD sharing between Corded Ware and Yamnaya groups, indicating that the Yamnaya herders of the Pontic-Caspian Steppe and the Steppe-related ancestry in various European Corded Ware groups share substantial co-ancestry within only a few hundred years. These results show that detecting IBD segments can generate powerful insights into the growing aDNA record, both on a small scale relevant to life stories and on a large scale relevant to major cultural-historical events. ancIBD identifies identity-by-descent regions in ancient DNA using a hidden Markov model optimized for these low-coverage data. Analysis of 4,248 individuals demonstrates that ancIBD can identify up to sixth-degree relatives and provides genealogical insights into ancient populations.},
  archive      = {J_NG},
  author       = {Ringbauer, Harald and Huang, Yilei and Akbari, Ali and Mallick, Swapan and Olalde, Iñigo and Patterson, Nick and Reich, David},
  doi          = {10.1038/s41588-023-01582-w},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {143-151},
  shortjournal = {Nat. Genet.},
  title        = {Accurate detection of identity-by-descent segments in human ancient DNA},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Origin and evolution of the triploid cultivated banana
genome. <em>NG</em>, <em>56</em>(1), 136–142. (<a
href="https://doi.org/10.1038/s41588-023-01589-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most fresh bananas belong to the Cavendish and Gros Michel subgroups. Here, we report chromosome-scale genome assemblies of Cavendish (1.48 Gb) and Gros Michel (1.33 Gb), defining three subgenomes, Ban, Dh and Ze, with Musa acuminata ssp. banksii, malaccensis and zebrina as their major ancestral contributors, respectively. The insertion of repeat sequences in the Fusarium oxysporum f. sp. cubense (Foc) tropical race 4 RGA2 (resistance gene analog 2) promoter was identified in most diploid and triploid bananas. We found that the receptor-like protein (RLP) locus, including Foc race 1-resistant genes, is absent in the Gros Michel Ze subgenome. We identified two NAP (NAC-like, activated by apetala3/pistillata) transcription factor homologs specifically and highly expressed in fruit that directly bind to the promoters of many fruit ripening genes and may be key regulators of fruit ripening. Our genome data should facilitate the breeding and super-domestication of bananas. Chromosome-scale genome assemblies of triploid Cavendish and Gros Michel reveal the banana cultivars’ origins, disease resistance and fruit ripening mechanism.},
  archive      = {J_NG},
  author       = {Li, Xiuxiu and Yu, Sheng and Cheng, Zhihao and Chang, Xiaojun and Yun, Yingzi and Jiang, Mengwei and Chen, Xuequn and Wen, Xiaohui and Li, Hua and Zhu, Wenjun and Xu, Shiyao and Xu, Yanbing and Wang, Xianjun and Zhang, Chen and Wu, Qiong and Hu, Jin and Lin, Zhenguo and Aury, Jean-Marc and Van de Peer, Yves and Wang, Zonghua and Zhou, Xiaofan and Wang, Jihua and Lü, Peitao and Zhang, Liangsheng},
  doi          = {10.1038/s41588-023-01589-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {136-142},
  shortjournal = {Nat. Genet.},
  title        = {Origin and evolution of the triploid cultivated banana genome},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computational prediction and experimental validation
identify functionally conserved lncRNAs from zebrafish to human.
<em>NG</em>, <em>56</em>(1), 124–135. (<a
href="https://doi.org/10.1038/s41588-023-01620-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Functional studies of long noncoding RNAs (lncRNAs) have been hindered by the lack of methods to assess their evolution. Here we present lncRNA Homology Explorer (lncHOME), a computational pipeline that identifies a unique class of long noncoding RNAs (lncRNAs) with conserved genomic locations and patterns of RNA-binding protein (RBP) binding sites (coPARSE-lncRNAs). Remarkably, several hundred human coPARSE-lncRNAs can be evolutionarily traced to zebrafish. Using CRISPR–Cas12a knockout and rescue assays, we found that knocking out many human coPARSE-lncRNAs led to cell proliferation defects, which were subsequently rescued by predicted zebrafish homologs. Knocking down coPARSE-lncRNAs in zebrafish embryos caused severe developmental delays that were rescued by human homologs. Furthermore, we verified that human, mouse and zebrafish coPARSE-lncRNA homologs tend to bind similar RBPs with their conserved functions relying on specific RBP-binding sites. Overall, our study demonstrates a comprehensive approach for studying the functional conservation of lncRNAs and implicates numerous lncRNAs in regulating vertebrate physiology. A new computational method coupled with a CRISPR–Cas12a screen identifies human long noncoding RNAs (lncRNAs) that lead to cell proliferation defects, which can be rescued by zebrafish homologs. Knockdown of four zebrafish lncRNAs that perturb embryonic development can be rescued by human homologs.},
  archive      = {J_NG},
  author       = {Huang, Wenze and Xiong, Tuanlin and Zhao, Yuting and Heng, Jian and Han, Ge and Wang, Pengfei and Zhao, Zhihua and Shi, Ming and Li, Juan and Wang, Jiazhen and Wu, Yixia and Liu, Feng and Xi, Jianzhong Jeff and Wang, Yangming and Zhang, Qiangfeng Cliff},
  doi          = {10.1038/s41588-023-01620-7},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {124-135},
  shortjournal = {Nat. Genet.},
  title        = {Computational prediction and experimental validation identify functionally conserved lncRNAs from zebrafish to human},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A compendium of genetic regulatory effects across pig
tissues. <em>NG</em>, <em>56</em>(1), 112–123. (<a
href="https://doi.org/10.1038/s41588-023-01585-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Farm Animal Genotype-Tissue Expression (FarmGTEx) project has been established to develop a public resource of genetic regulatory variants in livestock, which is essential for linking genetic polymorphisms to variation in phenotypes, helping fundamental biological discovery and exploitation in animal breeding and human biomedicine. Here we show results from the pilot phase of PigGTEx by processing 5,457 RNA-sequencing and 1,602 whole-genome sequencing samples passing quality control from pigs. We build a pig genotype imputation panel and associate millions of genetic variants with five types of transcriptomic phenotypes in 34 tissues. We evaluate tissue specificity of regulatory effects and elucidate molecular mechanisms of their action using multi-omics data. Leveraging this resource, we decipher regulatory mechanisms underlying 207 pig complex phenotypes and demonstrate the similarity of pigs to humans in gene expression and the genetic regulation behind complex phenotypes, supporting the importance of pigs as a human biomedical model. The pilot phase of PigGTEx, re-analyzing 5,457 published RNA-seq samples, presents a pan-tissue catalog of molecular quantitative trait loci. Cross-species comparisons identify traits with shared genetic regulation in humans.},
  archive      = {J_NG},
  author       = {Teng, Jinyan and Gao, Yahui and Yin, Hongwei and Bai, Zhonghao and Liu, Shuli and Zeng, Haonan and Bai, Lijing and Cai, Zexi and Zhao, Bingru and Li, Xiujin and Xu, Zhiting and Lin, Qing and Pan, Zhangyuan and Yang, Wenjing and Yu, Xiaoshan and Guan, Dailu and Hou, Yali and Keel, Brittney N. and Rohrer, Gary A. and Lindholm-Perry, Amanda K. and Oliver, William T. and Ballester, Maria and Crespo-Piazuelo, Daniel and Quintanilla, Raquel and Canela-Xandri, Oriol and Rawlik, Konrad and Xia, Charley and Yao, Yuelin and Zhao, Qianyi and Yao, Wenye and Yang, Liu and Li, Houcheng and Zhang, Huicong and Liao, Wang and Chen, Tianshuo and Karlskov-Mortensen, Peter and Fredholm, Merete and Amills, Marcel and Clop, Alex and Giuffra, Elisabetta and Wu, Jun and Cai, Xiaodian and Diao, Shuqi and Pan, Xiangchun and Wei, Chen and Li, Jinghui and Cheng, Hao and Wang, Sheng and Su, Guosheng and Sahana, Goutam and Lund, Mogens Sandø and Dekkers, Jack C. M. and Kramer, Luke and Tuggle, Christopher K. and Corbett, Ryan and Groenen, Martien A. M. and Madsen, Ole and Gòdia, Marta and Rocha, Dominique and Charles, Mathieu and Li, Cong-jun and Pausch, Hubert and Hu, Xiaoxiang and Frantz, Laurent and Luo, Yonglun and Lin, Lin and Zhou, Zhongyin and Zhang, Zhe and Chen, Zitao and Cui, Leilei and Xiang, Ruidong and Shen, Xia and Li, Pinghua and Huang, Ruihua and Tang, Guoqing and Li, Mingzhou and Zhao, Yunxiang and Yi, Guoqiang and Tang, Zhonglin and Jiang, Jicai and Zhao, Fuping and Yuan, Xiaolong and Liu, Xiaohong and Chen, Yaosheng and Xu, Xuewen and Zhao, Shuhong and Zhao, Pengju and Haley, Chris and Zhou, Huaijun and Wang, Qishan and Pan, Yuchun and Ding, Xiangdong and Ma, Li and Li, Jiaqi and Navarro, Pau and Zhang, Qin and Li, Bingjie and Tenesa, Albert and Li, Kui and Liu, George E. and Fang, Lingzhao},
  doi          = {10.1038/s41588-023-01585-7},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {112-123},
  shortjournal = {Nat. Genet.},
  title        = {A compendium of genetic regulatory effects across pig tissues},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The BAF chromatin remodeler synergizes with RNA polymerase
II and transcription factors to evict nucleosomes. <em>NG</em>,
<em>56</em>(1), 100–111. (<a
href="https://doi.org/10.1038/s41588-023-01603-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin accessibility is a hallmark of active transcription and entails ATP-dependent nucleosome remodeling, which is carried out by complexes such as Brahma-associated factor (BAF). However, the mechanistic links between transcription, nucleosome remodeling and chromatin accessibility are unclear. Here, we used a chemical–genetic approach coupled with time-resolved chromatin profiling to dissect the interplay between RNA Polymerase II (RNAPII), BAF and DNA-sequence-specific transcription factors in mouse embryonic stem cells. We show that BAF dynamically unwraps and evicts nucleosomes at accessible chromatin regions, while RNAPII promoter-proximal pausing stabilizes BAF chromatin occupancy and enhances ATP-dependent nucleosome eviction by BAF. We find that although RNAPII and BAF dynamically probe both transcriptionally active and Polycomb-repressed genomic regions, pluripotency transcription factor chromatin binding confers locus specificity for productive chromatin remodeling and nucleosome eviction by BAF. Our study suggests a paradigm for how functional synergy between dynamically acting chromatin factors regulates locus-specific nucleosome organization and chromatin accessibility. A chemical-genetic approach coupled with temporally resolved chromatin profiling shows that RNAPII promoter-proximal pausing stabilizes BAF complex occupancy and promotes nucleosome eviction.},
  archive      = {J_NG},
  author       = {Brahma, Sandipan and Henikoff, Steven},
  doi          = {10.1038/s41588-023-01603-8},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {100-111},
  shortjournal = {Nat. Genet.},
  title        = {The BAF chromatin remodeler synergizes with RNA polymerase II and transcription factors to evict nucleosomes},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A chromatin-regulated biphasic circuit coordinates
IL-1β-mediated inflammation. <em>NG</em>, <em>56</em>(1), 85–99. (<a
href="https://doi.org/10.1038/s41588-023-01598-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inflammation is characterized by a biphasic cycle consisting initially of a proinflammatory phase that is subsequently resolved by anti-inflammatory processes. Interleukin-1β (IL-1β) is a master regulator of proinflammation and is encoded within the same topologically associating domain (TAD) as IL-37, which is an anti-inflammatory cytokine that opposes the function of IL-1β. Within this TAD, we identified a long noncoding RNA called AMANZI, which negatively regulates IL-1β expression and trained immunity through the induction of IL37 transcription. We found that the activation of IL37 occurs through the formation of a dynamic long-range chromatin contact that leads to the temporal delay of anti-inflammatory responses. The common variant rs16944 present in AMANZI augments this regulatory circuit, predisposing individuals to enhanced proinflammation or immunosuppression. Our work illuminates a chromatin-mediated biphasic circuit coordinating expression of IL-1β and IL-37, thereby regulating two functionally opposed states of inflammation from within a single TAD. The long noncoding RNA AMANZI downregulates IL-1β expression and trained immunity by inducing IL-37 transcription via long-range chromatin contacts. The common variant rs16944 present in AMANZI modulates proinflammation or immunosuppression risk.},
  archive      = {J_NG},
  author       = {Fok, Ezio T. and Moorlag, Simone J. C. F. M. and Negishi, Yutaka and Groh, Laszlo A. and dos Santos, Jéssica Cristina and Gräwe, Cathrin and Monge, Valerie Villacorta and Craenmehr, Daphne D. D. and van Roosmalen, Mellanie and da Cunha Jolvino, David Pablo and Migliorini, Letícia Busato and Neto, Ary Serpa and Severino, Patricia and Vermeulen, Michiel and Joosten, Leo A. B. and Netea, Mihai G. and Fanucchi, Stephanie and Mhlanga, Musa M.},
  doi          = {10.1038/s41588-023-01598-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {85-99},
  shortjournal = {Nat. Genet.},
  title        = {A chromatin-regulated biphasic circuit coordinates IL-1β-mediated inflammation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). CellCharter reveals spatial cell niches associated with
tissue remodeling and cell plasticity. <em>NG</em>, <em>56</em>(1),
74–84. (<a href="https://doi.org/10.1038/s41588-023-01588-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissues are organized in cellular niches, the composition and interactions of which can be investigated using spatial omics technologies. However, systematic analyses of tissue composition are challenged by the scale and diversity of the data. Here we present CellCharter, an algorithmic framework to identify, characterize, and compare cellular niches in spatially resolved datasets. CellCharter outperformed existing approaches and effectively identified cellular niches across datasets generated using different technologies, and comprising hundreds of samples and millions of cells. In multiple human lung cancer cohorts, CellCharter uncovered a cellular niche composed of tumor-associated neutrophil and cancer cells expressing markers of hypoxia and cell migration. This cancer cell state was spatially segregated from more proliferative tumor cell clusters and was associated with tumor-associated neutrophil infiltration and poor prognosis in independent patient cohorts. Overall, CellCharter enables systematic analyses across data types and technologies to decode the link between spatial tissue architectures and cell plasticity. CellCharter is a flexible, platform-agnostic method for identifying cell niches in spatially resolved data. Analysis of lung cancers demonstrates the importance of considering spatial information, exemplified by a neutrophil-associated niche that correlates with an aggressive cancer cell state and patient prognosis.},
  archive      = {J_NG},
  author       = {Varrone, Marco and Tavernari, Daniele and Santamaria-Martínez, Albert and Walsh, Logan A. and Ciriello, Giovanni},
  doi          = {10.1038/s41588-023-01588-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {74-84},
  shortjournal = {Nat. Genet.},
  title        = {CellCharter reveals spatial cell niches associated with tissue remodeling and cell plasticity},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The role of APOBEC3B in lung tumor evolution and targeted
cancer therapy resistance. <em>NG</em>, <em>56</em>(1), 60–73. (<a
href="https://doi.org/10.1038/s41588-023-01592-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, the impact of the apolipoprotein B mRNA-editing catalytic subunit-like (APOBEC) enzyme APOBEC3B (A3B) on epidermal growth factor receptor (EGFR)-driven lung cancer was assessed. A3B expression in EGFR mutant (EGFRmut) non-small-cell lung cancer (NSCLC) mouse models constrained tumorigenesis, while A3B expression in tumors treated with EGFR-targeted cancer therapy was associated with treatment resistance. Analyses of human NSCLC models treated with EGFR-targeted therapy showed upregulation of A3B and revealed therapy-induced activation of nuclear factor kappa B (NF-κB) as an inducer of A3B expression. Significantly reduced viability was observed with A3B deficiency, and A3B was required for the enrichment of APOBEC mutation signatures, in targeted therapy-treated human NSCLC preclinical models. Upregulation of A3B was confirmed in patients with NSCLC treated with EGFR-targeted therapy. This study uncovers the multifaceted roles of A3B in NSCLC and identifies A3B as a potential target for more durable responses to targeted cancer therapy. Analyses of in vivo models, cell lines and patient-derived samples show that apolipoprotein B mRNA-editing catalytic subunit 3B (APOBEC3B) not only restrains lung tumor initiation but also that its upregulation is associated with resistance to targeted therapies. This study highlights the complex and context-dependent role of APOBEC3B in lung cancer.},
  archive      = {J_NG},
  author       = {Caswell, Deborah R. and Gui, Philippe and Mayekar, Manasi K. and Law, Emily K. and Pich, Oriol and Bailey, Chris and Boumelha, Jesse and Kerr, D. Lucas and Blakely, Collin M. and Manabe, Tadashi and Martinez-Ruiz, Carlos and Bakker, Bjorn and De Dios Palomino Villcas, Juan and I. Vokes, Natalie and Dietzen, Michelle and Angelova, Mihaela and Gini, Beatrice and Tamaki, Whitney and Allegakoen, Paul and Wu, Wei and Humpton, Timothy J. and Hill, William and Tomaschko, Mona and Lu, Wei-Ting and Haderk, Franziska and Al Bakir, Maise and Nagano, Ai and Gimeno-Valiente, Francisco and de Carné Trécesson, Sophie and Vendramin, Roberto and Barbè, Vittorio and Mugabo, Miriam and Weeden, Clare E. and Rowan, Andrew and McCoach, Caroline E. and Almeida, Bruna and Green, Mary and Gomez, Carlos and Nanjo, Shigeki and Barbosa, Dora and Moore, Chris and Przewrocka, Joanna and Black, James R. M. and Grönroos, Eva and Suarez-Bonnet, Alejandro and Priestnall, Simon L. and Zverev, Caroline and Lighterness, Scott and Cormack, James and Olivas, Victor and Cech, Lauren and Andrews, Trisha and Rule, Brandon and Jiao, Yuwei and Zhang, Xinzhu and Ashford, Paul and Durfee, Cameron and Venkatesan, Subramanian and Temiz, Nuri Alpay and Tan, Lisa and Larson, Lindsay K. and Argyris, Prokopios P. and Brown, William L. and Yu, Elizabeth A. and Rotow, Julia K. and Guha, Udayan and Roper, Nitin and Yu, Johnny and Vogel, Rachel I. and Thomas, Nicholas J. and Marra, Antonio and Selenica, Pier and Yu, Helena and Bakhoum, Samuel F. and Chew, Su Kit and Reis-Filho, Jorge S. and Jamal-Hanjani, Mariam and Vousden, Karen H. and McGranahan, Nicholas and Van Allen, Eliezer M. and Kanu, Nnennaya and Harris, Reuben S. and Downward, Julian and Bivona, Trever G. and Swanton, Charles},
  doi          = {10.1038/s41588-023-01592-8},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {60-73},
  shortjournal = {Nat. Genet.},
  title        = {The role of APOBEC3B in lung tumor evolution and targeted cancer therapy resistance},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Development of a human genetics-guided priority score for
19,365 genes and 399 drug indications. <em>NG</em>, <em>56</em>(1),
51–59. (<a href="https://doi.org/10.1038/s41588-023-01609-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies have shown that drug targets with human genetic support are more likely to succeed in clinical trials. Hence, a tool integrating genetic evidence to prioritize drug target genes is beneficial for drug discovery. We built a genetic priority score (GPS) by integrating eight genetic features with drug indications from the Open Targets and SIDER databases. The top 0.83%, 0.28% and 0.19% of the GPS conferred a 5.3-, 9.9- and 11.0-fold increased effect of having an indication, respectively. In addition, we observed that targets in the top 0.28% of the score were 1.7-, 3.7- and 8.8-fold more likely to advance from phase I to phases II, III and IV, respectively. Complementary to the GPS, we incorporated the direction of genetic effect and drug mechanism into a directional version of the score called the GPS with direction of effect. We applied our method to 19,365 protein-coding genes and 399 drug indications and made all results available through a web portal. A human genetics-informed drug prioritization tool, genetic priority score (GPS), combines genetic features and drug datasets. GPS-supported indications are more likely to progress through clinical trials, suggesting the utility of this score for target prioritization.},
  archive      = {J_NG},
  author       = {Duffy, Áine and Petrazzini, Ben Omega and Stein, David and Park, Joshua K. and Forrest, Iain S. and Gibson, Kyle and Vy, Ha My and Chen, Robert and Márquez-Luna, Carla and Mort, Matthew and Verbanck, Marie and Schlessinger, Avner and Itan, Yuval and Cooper, David N. and Rocheleau, Ghislain and Jordan, Daniel M. and Do, Ron},
  doi          = {10.1038/s41588-023-01609-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {51-59},
  shortjournal = {Nat. Genet.},
  title        = {Development of a human genetics-guided priority score for 19,365 genes and 399 drug indications},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Perturbational phenotyping of human blood cells reveals
genetically determined latent traits associated with subsets of common
diseases. <em>NG</em>, <em>56</em>(1), 37–50. (<a
href="https://doi.org/10.1038/s41588-023-01600-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although genome-wide association studies (GWAS) have successfully linked genetic risk loci to various disorders, identifying underlying cellular biological mechanisms remains challenging due to the complex nature of common diseases. We established a framework using human peripheral blood cells, physical, chemical and pharmacological perturbations, and flow cytometry-based functional readouts to reveal latent cellular processes and performed GWAS based on these evoked traits in up to 2,600 individuals. We identified 119 genomic loci implicating 96 genes associated with these cellular responses and discovered associations between evoked blood phenotypes and subsets of common diseases. We found a population of pro-inflammatory anti-apoptotic neutrophils prevalent in individuals with specific subsets of cardiometabolic disease. Multigenic models based on this trait predicted the risk of developing chronic kidney disease in type 2 diabetes patients. By expanding the phenotypic space for human genetic studies, we could identify variants associated with large effect response differences, stratify patients and efficiently characterize the underlying biology. Genome-wide analyses of blood cell phenotypes derived from perturbations coupled with flow cytometry-based functional readouts identify loci associated with latent cellular traits, yielding insights into biological mechanisms underlying common diseases.},
  archive      = {J_NG},
  author       = {Homilius, Max and Zhu, Wandi and Eddy, Samuel S. and Thompson, Patrick C. and Zheng, Huahua and Warren, Caleb N. and Evans, Chiara G. and Kim, David D. and Xuan, Lucius L. and Nsubuga, Cissy and Strecker, Zachary and Pettit, Christopher J. and Cho, Jungwoo and Howie, Mikayla N. and Thaler, Alexandra S. and Wilson, Evan and Wollison, Bruce and Smith, Courtney and Nascimben, Julia B. and Nascimben, Diana N. and Lunati, Gabriella M. and Folks, Hassan C. and Cupelo, Matthew and Sridaran, Suriya and Rheinstein, Carolyn and McClennen, Taylor and Goto, Shinichi and Truslow, James G. and Vandenwijngaert, Sara and MacRae, Calum A. and Deo, Rahul C.},
  doi          = {10.1038/s41588-023-01600-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {37-50},
  shortjournal = {Nat. Genet.},
  title        = {Perturbational phenotyping of human blood cells reveals genetically determined latent traits associated with subsets of common diseases},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multi-ancestry genome-wide association meta-analysis of
parkinson’s disease. <em>NG</em>, <em>56</em>(1), 27–36. (<a
href="https://doi.org/10.1038/s41588-023-01584-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although over 90 independent risk variants have been identified for Parkinson’s disease using genome-wide association studies, most studies have been performed in just one population at a time. Here we performed a large-scale multi-ancestry meta-analysis of Parkinson’s disease with 49,049 cases, 18,785 proxy cases and 2,458,063 controls including individuals of European, East Asian, Latin American and African ancestry. In a meta-analysis, we identified 78 independent genome-wide significant loci, including 12 potentially novel loci (MTF2, PIK3CA, ADD1, SYBU, IRS2, USP8, PIGL, FASN, MYLK2, USP25, EP300 and PPP6R2) and fine-mapped 6 putative causal variants at 6 known PD loci. By combining our results with publicly available eQTL data, we identified 25 putative risk genes in these novel loci whose expression is associated with PD risk. This work lays the groundwork for future efforts aimed at identifying PD loci in non-European populations. Multi-ancestry genome-wide association analyses identify new risk loci for Parkinson’s disease, and fine-mapping and co-localization analyses implicate candidate genes whose expression is associated with disease susceptibility.},
  archive      = {J_NG},
  author       = {Kim, Jonggeol Jeffrey and Vitale, Dan and Otani, Diego Véliz and Lian, Michelle Mulan and Heilbron, Karl and Iwaki, Hirotaka and Lake, Julie and Solsberg, Caroline Warly and Leonard, Hampton and Makarious, Mary B. and Tan, Eng-King and Singleton, Andrew B. and Bandres-Ciga, Sara and Noyce, Alastair J. and Blauwendraat, Cornelis and Nalls, Mike A. and Foo, Jia Nee and Mata, Ignacio},
  doi          = {10.1038/s41588-023-01584-8},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {27-36},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genome-wide association meta-analysis of parkinson’s disease},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The chemotherapeutic drug CX-5461 is a potent mutagen in
cultured human cells. <em>NG</em>, <em>56</em>(1), 23–26. (<a
href="https://doi.org/10.1038/s41588-023-01602-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The chemotherapeutic agent CX-5461, or pidnarulex, has been fast-tracked by the United States Food and Drug Administration for early-stage clinical studies of BRCA1-, BRCA2- and PALB2-mutated cancers. It is under investigation in phase I and II trials. Here, we find that, although CX-5461 exhibits synthetic lethality in BRCA1-/BRCA2-deficient cells, it also causes extensive, nonselective, collateral mutagenesis in all three cell lines tested, to magnitudes that exceed known environmental carcinogens. The chemotherapeutic agent CX-5461 is shown to be a potent mutagen in hTERT-RPE1, HAP1 and human induced pluripotent stem cells. The compound generates distinct mutational patterns of single- and double-base substitutions, as well as of small insertions and deletions, that were detectable following a single exposure.},
  archive      = {J_NG},
  author       = {Koh, Gene Ching Chiek and Boushaki, Soraya and Zhao, Salome Jingchen and Pregnall, Andrew Marcel and Sadiyah, Firas and Badja, Cherif and Memari, Yasin and Georgakopoulos-Soares, Ilias and Nik-Zainal, Serena},
  doi          = {10.1038/s41588-023-01602-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {23-26},
  shortjournal = {Nat. Genet.},
  title        = {The chemotherapeutic drug CX-5461 is a potent mutagen in cultured human cells},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A lncRNA–chromatin circuit regulates IL-1β-driven
inflammation. <em>NG</em>, <em>56</em>(1), 21–22. (<a
href="https://doi.org/10.1038/s41588-023-01605-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polymorphisms in the non-coding genome affect genetic circuits and result in variable immune responses across individuals. Here we report a genetic circuit involving a long non-coding RNA (lncRNA) that spatially coordinates chromatin contacts to control pro- and anti-inflammatory gene expression and shape immune responses of healthy individuals to pathogens or vaccination.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01605-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {21-22},
  shortjournal = {Nat. Genet.},
  title        = {A lncRNA–chromatin circuit regulates IL-1β-driven inflammation},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). RNA polymerase II promoter-proximal pausing promotes BAF
chromatin binding and remodeling. <em>NG</em>, <em>56</em>(1), 19–20.
(<a href="https://doi.org/10.1038/s41588-023-01628-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New research reports that paused RNA polymerase II (RNAPII) enhances the targeting and activity of BAF chromatin remodelers. These findings suggest a new paradigm for understanding how the collaborative action of chromatin remodelers and the transcriptional machinery govern cell-type-specific chromatin accessibility.},
  archive      = {J_NG},
  author       = {Chick, Brent Y. and Hargreaves, Diana C.},
  doi          = {10.1038/s41588-023-01628-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {19-20},
  shortjournal = {Nat. Genet.},
  title        = {RNA polymerase II promoter-proximal pausing promotes BAF chromatin binding and remodeling},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cell perturbation and lasers illuminate the genetics of
latent blood cell traits. <em>NG</em>, <em>56</em>(1), 16–18. (<a
href="https://doi.org/10.1038/s41588-023-01623-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study combining experimental treatments of human blood cells from thousands of individuals with flow-cytometry-based phenotyping and then genome-wide association analyses identifies genetic loci associated with non-resting cell states. Integrating the results with disease association signals yields insights into the underlying biology.},
  archive      = {J_NG},
  author       = {Johnson, Andrew D.},
  doi          = {10.1038/s41588-023-01623-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {16-18},
  shortjournal = {Nat. Genet.},
  title        = {Cell perturbation and lasers illuminate the genetics of latent blood cell traits},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Charting multicellular tissue structure cell-to-cell.
<em>NG</em>, <em>56</em>(1), 14–15. (<a
href="https://doi.org/10.1038/s41588-023-01624-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The spatial biology revolution promises deep insights into tissue organization, but deriving this knowledge from diverse, complex data remains a major obstacle. Data-driven discovery of the multicellular organization of tissues is now achieved by transforming multimodal spatial imaging data using deep learning.},
  archive      = {J_NG},
  author       = {Schrader, Ellen and Ali, H. Raza},
  doi          = {10.1038/s41588-023-01624-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {14-15},
  shortjournal = {Nat. Genet.},
  title        = {Charting multicellular tissue structure cell-to-cell},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Extraordinary collateral mutagenesis induced by CX-5461.
<em>NG</em>, <em>56</em>(1), 12–13. (<a
href="https://doi.org/10.1038/s41588-023-01611-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CX-5461 (also known as pidnarulex), currently in phase 1/2 trials, induces selective killing of homologous-recombination-deficient or BRCA1- or BRCA2-mutated tumors in preclinical models. New work confirms these findings but shows it to be a remarkably potent mutagen that induces extensive genetic changes in cultured human cells with or without BRCA1/2 mutations, raising substantial safety issues.},
  archive      = {J_NG},
  author       = {Boulton, Simon J.},
  doi          = {10.1038/s41588-023-01611-8},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {12-13},
  shortjournal = {Nat. Genet.},
  title        = {Extraordinary collateral mutagenesis induced by CX-5461},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Shared chromosomal segments connect ancient human societies.
<em>NG</em>, <em>56</em>(1), 10–11. (<a
href="https://doi.org/10.1038/s41588-023-01606-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long segments of the genome that are shared ‘identical by descent’ (IBD) demonstrate recent relatedness between individuals. A new computational method robustly identifies shared IBD segments in human ancient DNA data, providing insights into the mobility and demography of prehistoric human societies.},
  archive      = {J_NG},
  author       = {Bergström, Anders},
  doi          = {10.1038/s41588-023-01606-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {10-11},
  shortjournal = {Nat. Genet.},
  title        = {Shared chromosomal segments connect ancient human societies},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024j). Whole-embryo scRNA-seq of mouse development. <em>NG</em>,
<em>56</em>(1), 9. (<a
href="https://doi.org/10.1038/s41588-023-01633-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01633-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nat. Genet.},
  title        = {Whole-embryo scRNA-seq of mouse development},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Single-cell atlas of mutant zebrafish embryos. <em>NG</em>,
<em>56</em>(1), 9. (<a
href="https://doi.org/10.1038/s41588-023-01634-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01634-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell atlas of mutant zebrafish embryos},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). African-ancestry-associated genomic differences in cancer.
<em>NG</em>, <em>56</em>(1), 9. (<a
href="https://doi.org/10.1038/s41588-023-01635-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01635-0},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nat. Genet.},
  title        = {African-ancestry-associated genomic differences in cancer},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024e). PNPLA3-associated mitochondrial dysfunction. <em>NG</em>,
<em>56</em>(1), 9. (<a
href="https://doi.org/10.1038/s41588-023-01636-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01636-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nat. Genet.},
  title        = {PNPLA3-associated mitochondrial dysfunction},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024i). Refining the apple pan-genome. <em>NG</em>, <em>56</em>(1),
8. (<a href="https://doi.org/10.1038/s41588-023-01629-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01629-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nat. Genet.},
  title        = {Refining the apple pan-genome},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024g). Reshaping the brain during pregnancy. <em>NG</em>,
<em>56</em>(1), 8. (<a
href="https://doi.org/10.1038/s41588-023-01630-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-023-01630-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nat. Genet.},
  title        = {Reshaping the brain during pregnancy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Gene therapy for parkinson’s disease models. <em>NG</em>,
<em>56</em>(1), 8. (<a
href="https://doi.org/10.1038/s41588-023-01632-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01632-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nat. Genet.},
  title        = {Gene therapy for parkinson’s disease models},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Clonal dynamics of blood stem cells after gene therapy.
<em>NG</em>, <em>56</em>(1), 8. (<a
href="https://doi.org/10.1038/s41588-023-01631-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01631-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nat. Genet.},
  title        = {Clonal dynamics of blood stem cells after gene therapy},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The ongoing evolution of UShER during the SARS-CoV-2
pandemic. <em>NG</em>, <em>56</em>(1), 4–7. (<a
href="https://doi.org/10.1038/s41588-023-01622-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While the number of SARS-CoV-2 genome sequences grew to over 15 million, the Ultrafast Sample placement on Existing tRees (UShER) tool suite maintained a comprehensive phylogenetic tree in near real time. This experience, and critical performance improvements throughout the pandemic, provide valuable lessons for rapidly scaling analyses.},
  archive      = {J_NG},
  author       = {Hinrichs, Angie and Ye, Cheng and Turakhia, Yatish and Corbett-Detig, Russell},
  doi          = {10.1038/s41588-023-01622-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {4-7},
  shortjournal = {Nat. Genet.},
  title        = {The ongoing evolution of UShER during the SARS-CoV-2 pandemic},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Planting the milestones of human genetics in senegal.
<em>NG</em>, <em>56</em>(1), 2–3. (<a
href="https://doi.org/10.1038/s41588-023-01581-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Implementing human genetics research: an itinerary of a researcher from low-income settings.},
  archive      = {J_NG},
  author       = {Ndiaye, Rokhaya},
  doi          = {10.1038/s41588-023-01581-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2-3},
  shortjournal = {Nat. Genet.},
  title        = {Planting the milestones of human genetics in senegal},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A call for spatial omics submissions. <em>NG</em>,
<em>56</em>(1), 1. (<a
href="https://doi.org/10.1038/s41588-023-01621-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The field of spatial omics is developing rapidly, with a potentially transformative effect across many areas of biology. Nature Genetics invites authors to submit papers that use these techniques to answer questions of broad interest to researchers working in genetics and genomics.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01621-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {A call for spatial omics submissions},
  volume       = {56},
  year         = {2024},
}
</textarea>
</details></li>
</ul>

</body>
</html>
