<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth---328">NMETH - 328</h2>
<ul>
<li><details>
<summary>
(2021b). Publisher correction: High-speed, cortex-wide volumetric
recording of neuroactivity at cellular resolution using light beads
microscopy. <em>NMETH</em>, <em>18</em>(12), 1552. (<a
href="https://doi.org/10.1038/s41592-021-01337-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Demas, Jeffrey and Manley, Jason and Tejera, Frank and Barber, Kevin and Kim, Hyewon and Traub, Francisca Martínez and Chen, Brandon and Vaziri, Alipasha},
  doi          = {10.1038/s41592-021-01337-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1552},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: High-speed, cortex-wide volumetric recording of neuroactivity at cellular resolution using light beads microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Anterograde transneuronal tracing and genetic control with
engineered yellow fever vaccine YFV-17D. <em>NMETH</em>,
<em>18</em>(12), 1542–1551. (<a
href="https://doi.org/10.1038/s41592-021-01319-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transneuronal viruses are powerful tools for tracing neuronal circuits or delivering genes to specific neurons in the brain. While there are multiple retrograde viruses, few anterograde viruses are available. Further, available anterograde viruses often have limitations such as retrograde transport, high neuronal toxicity or weak signals. We developed an anterograde viral system based on a live attenuated vaccine for yellow fever—YFV-17D. Replication- or packaging-deficient mutants of YFV-17D can be reconstituted in the brain, leading to efficient synapse-specific and anterograde-only transneuronal spreading, which can be controlled to achieve either monosynaptic or polysynaptic tracing. Moreover, inducible transient replication of YFV-17D mutant is sufficient to induce permanent transneuronal genetic modifications without causing neuronal toxicity. The engineered YFV-17D systems can be used to express fluorescent markers, sensors or effectors in downstream neurons, thus providing versatile tools for mapping and functionally controlling neuronal circuits. Anterograde transneuronal tracing can be achieved with engineered variants of the yellow fever vaccine YFV-17D. Both monosynaptic and polysynaptic tracing are demonstrated in different circuits of the mouse brain.},
  archive      = {J_NMETH},
  author       = {Li, Elizabeth and Guo, Jun and Oh, So Jung and Luo, Yi and Oliveros, Heankel Cantu and Du, Wenqin and Arano, Rachel and Kim, Yerim and Chen, Yuh-Tarng and Eitson, Jennifer and Lin, Da-Ting and Li, Ying and Roberts, Todd and Schoggins, John W. and Xu, Wei},
  doi          = {10.1038/s41592-021-01319-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1542-1551},
  shortjournal = {Nat. Meth.},
  title        = {Anterograde transneuronal tracing and genetic control with engineered yellow fever vaccine YFV-17D},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Imaging intact human organs with local resolution of
cellular structures using hierarchical phase-contrast tomography.
<em>NMETH</em>, <em>18</em>(12), 1532–1541. (<a
href="https://doi.org/10.1038/s41592-021-01317-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging intact human organs from the organ to the cellular scale in three dimensions is a goal of biomedical imaging. To meet this challenge, we developed hierarchical phase-contrast tomography (HiP-CT), an X-ray phase propagation technique using the European Synchrotron Radiation Facility (ESRF)’s Extremely Brilliant Source (EBS). The spatial coherence of the ESRF-EBS combined with our beamline equipment, sample preparation and scanning developments enabled us to perform non-destructive, three-dimensional (3D) scans with hierarchically increasing resolution at any location in whole human organs. We applied HiP-CT to image five intact human organ types: brain, lung, heart, kidney and spleen. HiP-CT provided a structural overview of each whole organ followed by multiple higher-resolution volumes of interest, capturing organotypic functional units and certain individual specialized cells within intact human organs. We demonstrate the potential applications of HiP-CT through quantification and morphometry of glomeruli in an intact human kidney and identification of regional changes in the tissue architecture in a lung from a deceased donor with coronavirus disease 2019 (COVID-19). Hierarchical phase-contrast tomography (HiP-CT) enables multiscale imaging of any region within an intact human organ down to cellular resolution. HiP-CT of five organ types revealed 3D morphological features in healthy and diseased tissue.},
  archive      = {J_NMETH},
  author       = {Walsh, C. L. and Tafforeau, P. and Wagner, W. L. and Jafree, D. J. and Bellier, A. and Werlein, C. and Kühnel, M. P. and Boller, E. and Walker-Samuel, S. and Robertus, J. L. and Long, D. A. and Jacob, J. and Marussi, S. and Brown, E. and Holroyd, N. and Jonigk, D. D. and Ackermann, M. and Lee, P. D.},
  doi          = {10.1038/s41592-021-01317-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1532-1541},
  shortjournal = {Nat. Meth.},
  title        = {Imaging intact human organs with local resolution of cellular structures using hierarchical phase-contrast tomography},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Spectral entropy outperforms MS/MS dot product similarity
for small-molecule compound identification. <em>NMETH</em>,
<em>18</em>(12), 1524–1531. (<a
href="https://doi.org/10.1038/s41592-021-01331-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Compound identification in small-molecule research, such as untargeted metabolomics or exposome research, relies on matching tandem mass spectrometry (MS/MS) spectra against experimental or in silico mass spectral libraries. Most software programs use dot product similarity scores. Here we introduce the concept of MS/MS spectral entropy to improve scoring results in MS/MS similarity searches via library matching. Entropy similarity outperformed 42 alternative similarity algorithms, including dot product similarity, when searching 434,287 spectra against the high-quality NIST20 library. Entropy similarity scores proved to be highly robust even when we added different levels of noise ions. When we applied entropy levels to 37,299 experimental spectra of natural products, false discovery rates of less than 10\% were observed at entropy similarity score 0.75. Experimental human gut metabolome data were used to confirm that entropy similarity largely improved the accuracy of MS-based annotations in small-molecule research to false discovery rates below 10\%, annotated new compounds and provided the basis to automatically flag poor-quality, noisy spectra. This manuscript proposes the use of spectral entropy similarity as a measure of similarity for small-molecule mass spectra and the use of a one-bond-difference approach for improving false discovery rates.},
  archive      = {J_NMETH},
  author       = {Li, Yuanyue and Kind, Tobias and Folz, Jacob and Vaniya, Arpana and Mehta, Sajjan Singh and Fiehn, Oliver},
  doi          = {10.1038/s41592-021-01331-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1524-1531},
  shortjournal = {Nat. Meth.},
  title        = {Spectral entropy outperforms MS/MS dot product similarity for small-molecule compound identification},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Precise, fast and comprehensive analysis of intact
glycopeptides and modified glycans with pGlyco3. <em>NMETH</em>,
<em>18</em>(12), 1515–1523. (<a
href="https://doi.org/10.1038/s41592-021-01306-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Great advances have been made in mass spectrometric data interpretation for intact glycopeptide analysis. However, accurate identification of intact glycopeptides and modified saccharide units at the site-specific level and with fast speed remains challenging. Here, we present a glycan-first glycopeptide search engine, pGlyco3, to comprehensively analyze intact N- and O-glycopeptides, including glycopeptides with modified saccharide units. A glycan ion-indexing algorithm developed for glycan-first search makes pGlyco3 5–40 times faster than other glycoproteomic search engines without decreasing accuracy or sensitivity. By combining electron-based dissociation spectra, pGlyco3 integrates a dynamic programming-based algorithm termed pGlycoSite for site-specific glycan localization. Our evaluation shows that the site-specific glycan localization probabilities estimated by pGlycoSite are suitable to localize site-specific glycans. With pGlyco3, we confidently identified N-glycopeptides and O-mannose glycopeptides that were extensively modified by ammonia adducts in yeast samples. The freely available pGlyco3 is an accurate and flexible tool that can be used to identify glycopeptides and modified saccharide units. pGlyco3 is a glycan-first glycopeptide search engine for the identification and localization of site-specific N- and O-glycopeptides, including glycopeptides with modified glycans.},
  archive      = {J_NMETH},
  author       = {Zeng, Wen-Feng and Cao, Wei-Qian and Liu, Ming-Qi and He, Si-Min and Yang, Peng-Yuan},
  doi          = {10.1038/s41592-021-01306-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1515-1523},
  shortjournal = {Nat. Meth.},
  title        = {Precise, fast and comprehensive analysis of intact glycopeptides and modified glycans with pGlyco3},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mapping single-cell-resolution cell phylogeny reveals cell
population dynamics during organ development. <em>NMETH</em>,
<em>18</em>(12), 1506–1514. (<a
href="https://doi.org/10.1038/s41592-021-01325-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping the cell phylogeny of a complex multicellular organism relies on somatic mutations accumulated from zygote to adult. Available cell barcoding methods can record about three mutations per barcode, enabling only low-resolution mapping of the cell phylogeny of complex organisms. Here we developed SMALT, a substitution mutation-aided lineage-tracing system that outperforms the available cell barcoding methods in mapping cell phylogeny. We applied SMALT to Drosophila melanogaster and obtained on average more than 20 mutations on a three-kilobase-pair barcoding sequence in early-adult cells. Using the barcoding mutations, we obtained high-quality cell phylogenetic trees, each comprising several thousand internal nodes with 84–93\% median bootstrap support. The obtained cell phylogenies enabled a population genetic analysis that estimates the longitudinal dynamics of the number of actively dividing parental cells (Np) in each organ through development. The Np dynamics revealed the trajectory of cell births and provided insight into the balance of symmetric and asymmetric cell division. SMALT, a base-editing lineage-tracing method, enables whole-organism phylogenetic analyses at single-cell resolution.},
  archive      = {J_NMETH},
  author       = {Liu, Kehui and Deng, Shanjun and Ye, Chang and Yao, Zeqi and Wang, Jianguo and Gong, Han and Liu, Li and He, Xionglei},
  doi          = {10.1038/s41592-021-01325-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1506-1514},
  shortjournal = {Nat. Meth.},
  title        = {Mapping single-cell-resolution cell phylogeny reveals cell population dynamics during organ development},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Persistence and plasticity in bacterial gene regulation.
<em>NMETH</em>, <em>18</em>(12), 1499–1505. (<a
href="https://doi.org/10.1038/s41592-021-01312-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organisms orchestrate cellular functions through transcription factor (TF) interactions with their target genes, although these regulatory relationships are largely unknown in most species. Here we report a high-throughput approach for characterizing TF–target gene interactions across species and its application to 354 TFs across 48 bacteria, generating 17,000 genome-wide binding maps. This dataset revealed themes of ancient conservation and rapid evolution of regulatory modules. We observed rewiring, where the TF sensing and regulatory role is maintained while the arrangement and identity of target genes diverges, in some cases encoding entirely new functions. We further integrated phenotypic information to define new functional regulatory modules and pathways. Finally, we identified 242 new TF DNA binding motifs, including a 70\% increase of known Escherichia coli motifs and the first annotation in Pseudomonas simiae, revealing deep conservation in bacterial promoter architecture. Our method provides a versatile tool for functional characterization of genetic pathways in prokaryotes and eukaryotes. This work presents biotin-DNA affinity purification (DAP) sequencing, that is, an in vitro, clone-free workflow to profile transcription factor (TF) DNA binding, as well as multiDAP to simultaneously characterize TF DNA binding in multiple bacterial genomes.},
  archive      = {J_NMETH},
  author       = {Baumgart, Leo A. and Lee, Ji Eun and Salamov, Asaf and Dilworth, David J. and Na, Hyunsoo and Mingay, Matthew and Blow, Matthew J. and Zhang, Yu and Yoshinaga, Yuko and Daum, Chris G. and O’Malley, Ronan C.},
  doi          = {10.1038/s41592-021-01312-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1499-1505},
  shortjournal = {Nat. Meth.},
  title        = {Persistence and plasticity in bacterial gene regulation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). OME-NGFF: A next-generation file format for expanding
bioimaging data-access strategies. <em>NMETH</em>, <em>18</em>(12),
1496–1498. (<a
href="https://doi.org/10.1038/s41592-021-01326-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid pace of innovation in biological imaging and the diversity of its applications have prevented the establishment of a community-agreed standardized data format. We propose that complementing established open formats such as OME-TIFF and HDF5 with a next-generation file format such as Zarr will satisfy the majority of use cases in bioimaging. Critically, a common metadata format used in all these vessels can deliver truly findable, accessible, interoperable and reusable bioimaging data. OME’s next-generation file format (OME-NGFF) provides a cloud-native complement to OME-TIFF and HDF5 for storing and accessing bioimaging data at scale and works toward the goal of findable, accessible, interoperable and reusable bioimaging data.},
  archive      = {J_NMETH},
  author       = {Moore, Josh and Allan, Chris and Besson, Sébastien and Burel, Jean-Marie and Diel, Erin and Gault, David and Kozlowski, Kevin and Lindner, Dominik and Linkert, Melissa and Manz, Trevor and Moore, Will and Pape, Constantin and Tischer, Christian and Swedlow, Jason R.},
  doi          = {10.1038/s41592-021-01326-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1496-1498},
  shortjournal = {Nat. Meth.},
  title        = {OME-NGFF: A next-generation file format for expanding bioimaging data-access strategies},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Micro-meta app: An interactive tool for collecting
microscopy metadata based on community specifications. <em>NMETH</em>,
<em>18</em>(12), 1489–1495. (<a
href="https://doi.org/10.1038/s41592-021-01315-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For quality, interpretation, reproducibility and sharing value, microscopy images should be accompanied by detailed descriptions of the conditions that were used to produce them. Micro-Meta App is an intuitive, highly interoperable, open-source software tool that was developed in the context of the 4D Nucleome (4DN) consortium and is designed to facilitate the extraction and collection of relevant microscopy metadata as specified by the recent 4DN-BINA-OME tiered-system of Microscopy Metadata specifications. In addition to substantially lowering the burden of quality assurance, the visual nature of Micro-Meta App makes it particularly suited for training purposes. Micro-Meta App is an intuitive, highly interoperable, open-source software tool designed to facilitate the extraction and collection of relevant microscopy metadata as specified by recent community guidelines.},
  archive      = {J_NMETH},
  author       = {Rigano, Alessandro and Ehmsen, Shannon and Öztürk, Serkan Utku and Ryan, Joel and Balashov, Alexander and Hammer, Mathias and Kirli, Koray and Boehm, Ulrike and Brown, Claire M. and Bellve, Karl and Chambers, James J. and Cosolo, Andrea and Coleman, Robert A. and Faklaris, Orestis and Fogarty, Kevin E. and Guilbert, Thomas and Hamacher, Anna B. and Itano, Michelle S. and Keeley, Daniel P. and Kunis, Susanne and Lacoste, Judith and Laude, Alex and Ma, Willa Y. and Marcello, Marco and Montero-Llopis, Paula and Nelson, Glyn and Nitschke, Roland and Pimentel, Jaime A. and Weidtkamp-Peters, Stefanie and Park, Peter J. and Alver, Burak H. and Grunwald, David and Strambio-De-Castillia, Caterina},
  doi          = {10.1038/s41592-021-01315-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1489-1495},
  shortjournal = {Nat. Meth.},
  title        = {Micro-meta app: An interactive tool for collecting microscopy metadata based on community specifications},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A 3D structural SARS-CoV-2–human interactome to explore
genetic and drug perturbations. <em>NMETH</em>, <em>18</em>(12),
1477–1488. (<a
href="https://doi.org/10.1038/s41592-021-01318-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emergence of new viral agents is driven by evolution of interactions between viral proteins and host targets. For instance, increased infectivity of SARS-CoV-2 compared to SARS-CoV-1 arose in part through rapid evolution along the interface between the spike protein and its human receptor ACE2, leading to increased binding affinity. To facilitate broader exploration of how pathogen–host interactions might impact transmission and virulence in the ongoing COVID-19 pandemic, we performed state-of-the-art interface prediction followed by molecular docking to construct a three-dimensional structural interactome between SARS-CoV-2 and human. We additionally carried out downstream meta-analyses to investigate enrichment of sequence divergence between SARS-CoV-1 and SARS-CoV-2 or human population variants along viral–human protein-interaction interfaces, predict changes in binding affinity by these mutations/variants and further prioritize drug repurposing candidates predicted to competitively bind human targets. We believe this resource ( http://3D-SARS2.yulab.org ) will aid in development and testing of informed hypotheses for SARS-CoV-2 etiology and treatments. This three-dimensional structural SARS-CoV-2–human interactome web server allows hypothesis-driven exploration of the mechanisms of SARS-CoV-2 pathology and host response.},
  archive      = {J_NMETH},
  author       = {Wierbowski, Shayne D. and Liang, Siqi and Liu, Yuan and Chen, You and Gupta, Shagun and Andre, Nicole M. and Lipkin, Steven M. and Whittaker, Gary R. and Yu, Haiyuan},
  doi          = {10.1038/s41592-021-01318-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1477-1488},
  shortjournal = {Nat. Meth.},
  title        = {A 3D structural SARS-CoV-2–human interactome to explore genetic and drug perturbations},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Best practices and tools for reporting reproducible
fluorescence microscopy methods. <em>NMETH</em>, <em>18</em>(12),
1463–1476. (<a
href="https://doi.org/10.1038/s41592-021-01156-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although fluorescence microscopy is ubiquitous in biomedical research, microscopy methods reporting is inconsistent and perhaps undervalued. We emphasize the importance of appropriate microscopy methods reporting and seek to educate researchers about how microscopy metadata impact data interpretation. We provide comprehensive guidelines and resources to enable accurate reporting for the most common fluorescence light microscopy modalities. We aim to improve microscopy reporting, thus improving the quality, rigor and reproducibility of image-based science. Comprehensive guidelines and resources to enable accurate reporting for the most common fluorescence light microscopy modalities are reported with the goal of improving microscopy reporting, rigor and reproducibility.},
  archive      = {J_NMETH},
  author       = {Montero Llopis, Paula and Senft, Rebecca A. and Ross-Elliott, Tim J. and Stephansky, Ryan and Keeley, Daniel P. and Koshar, Preman and Marqués, Guillermo and Gao, Ya-Sheng and Carlson, Benjamin R. and Pengo, Thomas and Sanders, Mark A. and Cameron, Lisa A. and Itano, Michelle S.},
  doi          = {10.1038/s41592-021-01156-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1463-1476},
  shortjournal = {Nat. Meth.},
  title        = {Best practices and tools for reporting reproducible fluorescence microscopy methods},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A repurposed vaccine grants durable anterograde access to
neural circuits. <em>NMETH</em>, <em>18</em>(12), 1459–1461. (<a
href="https://doi.org/10.1038/s41592-021-01345-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How a decades-old vaccine for yellow fever may help scientists study the brain connectome.},
  archive      = {J_NMETH},
  author       = {Treweek, Jennifer B.},
  doi          = {10.1038/s41592-021-01345-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1459-1461},
  shortjournal = {Nat. Meth.},
  title        = {A repurposed vaccine grants durable anterograde access to neural circuits},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Glycoproteomic software solutions spotlight glycans.
<em>NMETH</em>, <em>18</em>(12), 1457–1458. (<a
href="https://doi.org/10.1038/s41592-021-01310-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Software solutions pGlyco3 and StrucGP both aim to better assign the glycan part of a glycopeptide beyond simple glycosyl composition, but they differ in their strategies, their requirement for a glycan library and their applicability to O-glycopeptides.},
  archive      = {J_NMETH},
  author       = {Khoo, Kay-Hooi},
  doi          = {10.1038/s41592-021-01310-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1457-1458},
  shortjournal = {Nat. Meth.},
  title        = {Glycoproteomic software solutions spotlight glycans},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell metabolomics hits its stride. <em>NMETH</em>,
<em>18</em>(12), 1452–1456. (<a
href="https://doi.org/10.1038/s41592-021-01333-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An array of new techniques allow researchers to catalog the chemical contents of a single cell, or even a single organelle.},
  archive      = {J_NMETH},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41592-021-01333-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1452-1456},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell metabolomics hits its stride},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Tracking protein conformation in live cells. <em>NMETH</em>,
<em>18</em>(12), 1451. (<a
href="https://doi.org/10.1038/s41592-021-01354-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A simple ‘binder/tag’ approach allows tracking of individual protein conformation changes in live cells.},
  archive      = {J_NMETH},
  author       = {Doerr, Allison},
  doi          = {10.1038/s41592-021-01354-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1451},
  shortjournal = {Nat. Meth.},
  title        = {Tracking protein conformation in live cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Reverse-engineering tissues. <em>NMETH</em>,
<em>18</em>(12), 1450. (<a
href="https://doi.org/10.1038/s41592-021-01353-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A framework to reconstruct tissue architecture from imaging data.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01353-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1450},
  shortjournal = {Nat. Meth.},
  title        = {Reverse-engineering tissues},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). Profiling double-strand break repair. <em>NMETH</em>,
<em>18</em>(12), 1449. (<a
href="https://doi.org/10.1038/s41592-021-01352-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Repair-seq enables efficient profiling of genetic factors involved in DNA double-strand break repair.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01352-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1449},
  shortjournal = {Nat. Meth.},
  title        = {Profiling double-strand break repair},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021r). Sketching open and closed chromatin. <em>NMETH</em>,
<em>18</em>(12), 1448. (<a
href="https://doi.org/10.1038/s41592-021-01351-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell GET-seq (scGET-seq) enables the probing of compacted as well as accessible chromatin in single cells, covering a greater portion of the genome, which provides insights into genomic and epigenetic dynamics.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01351-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1448},
  shortjournal = {Nat. Meth.},
  title        = {Sketching open and closed chromatin},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Dissecting subcellular architecture of whole cells.
<em>NMETH</em>, <em>18</em>(12), 1447. (<a
href="https://doi.org/10.1038/s41592-021-01350-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technological advances in volume electron microscopy allow imaging and analysis of whole cells at record resolution.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01350-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1447},
  shortjournal = {Nat. Meth.},
  title        = {Dissecting subcellular architecture of whole cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A global view of standards for open image data formats and
repositories. <em>NMETH</em>, <em>18</em>(12), 1440–1446. (<a
href="https://doi.org/10.1038/s41592-021-01113-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging technologies are used throughout the life and biomedical sciences to understand mechanisms in biology and diagnosis and therapy in animal and human medicine. We present criteria for globally applicable guidelines for open image data tools and resources for the rapidly developing fields of biological and biomedical imaging.},
  archive      = {J_NMETH},
  author       = {Swedlow, Jason R. and Kankaanpää, Pasi and Sarkans, Ugis and Goscinski, Wojtek and Galloway, Graham and Malacrida, Leonel and Sullivan, Ryan P. and Härtel, Steffen and Brown, Claire M. and Wood, Christopher and Keppler, Antje and Paina, Federica and Loos, Ben and Zullino, Sara and Longo, Dario Livio and Aime, Silvio and Onami, Shuichi},
  doi          = {10.1038/s41592-021-01113-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1440-1446},
  shortjournal = {Nat. Meth.},
  title        = {A global view of standards for open image data formats and repositories},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards community-driven metadata standards for light
microscopy: Tiered specifications extending the OME model.
<em>NMETH</em>, <em>18</em>(12), 1427–1440. (<a
href="https://doi.org/10.1038/s41592-021-01327-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rigorous record-keeping and quality control are required to ensure the quality, reproducibility and value of imaging data. The 4DN Initiative and BINA here propose light Microscopy Metadata Specifications that extend the OME Data Model, scale with experimental intent and complexity, and make it possible for scientists to create comprehensive records of imaging experiments.},
  archive      = {J_NMETH},
  author       = {Hammer, Mathias and Huisman, Maximiliaan and Rigano, Alessandro and Boehm, Ulrike and Chambers, James J. and Gaudreault, Nathalie and North, Alison J. and Pimentel, Jaime A. and Sudar, Damir and Bajcsy, Peter and Brown, Claire M. and Corbett, Alexander D. and Faklaris, Orestis and Lacoste, Judith and Laude, Alex and Nelson, Glyn and Nitschke, Roland and Farzam, Farzin and Smith, Carlas S. and Grunwald, David and Strambio-De-Castillia, Caterina},
  doi          = {10.1038/s41592-021-01327-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1427-1440},
  shortjournal = {Nat. Meth.},
  title        = {Towards community-driven metadata standards for light microscopy: Tiered specifications extending the OME model},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). QUAREP-LiMi: A community endeavor to advance quality
assessment and reproducibility in light microscopy. <em>NMETH</em>,
<em>18</em>(12), 1423–1426. (<a
href="https://doi.org/10.1038/s41592-021-01162-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The community-driven initiative Quality Assessment and Reproducibility for Instruments &amp;amp; Images in Light Microscopy (QUAREP-LiMi) wants to improve reproducibility for light microscopy image data through quality control (QC) management of instruments and images. It aims for a common set of QC guidelines for hardware calibration and image acquisition, management and analysis.},
  archive      = {J_NMETH},
  author       = {Boehm, Ulrike and Nelson, Glyn and Brown, Claire M. and Bagley, Steve and Bajcsy, Peter and Bischof, Johanna and Dauphin, Aurelien and Dobbie, Ian M. and Eriksson, John E. and Faklaris, Orestis and Fernandez-Rodriguez, Julia and Ferrand, Alexia and Gelman, Laurent and Gheisari, Ali and Hartmann, Hella and Kukat, Christian and Laude, Alex and Mitkovski, Miso and Munck, Sebastian and North, Alison J. and Rasse, Tobias M. and Resch-Genger, Ute and Schuetz, Lucas C. and Seitz, Arne and Strambio-De-Castillia, Caterina and Swedlow, Jason R. and Nitschke, Roland},
  doi          = {10.1038/s41592-021-01162-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1423-1426},
  shortjournal = {Nat. Meth.},
  title        = {QUAREP-LiMi: A community endeavor to advance quality assessment and reproducibility in light microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). REMBI: Recommended metadata for biological images—enabling
reuse of microscopy data in biology. <em>NMETH</em>, <em>18</em>(12),
1418–1422. (<a
href="https://doi.org/10.1038/s41592-021-01166-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioimaging data have significant potential for reuse, but unlocking this potential requires systematic archiving of data and metadata in public databases. We propose draft metadata guidelines to begin addressing the needs of diverse communities within light and electron microscopy. We hope this publication and the proposed Recommended Metadata for Biological Images (REMBI) will stimulate discussions about their implementation and future extension.},
  archive      = {J_NMETH},
  author       = {Sarkans, Ugis and Chiu, Wah and Collinson, Lucy and Darrow, Michele C. and Ellenberg, Jan and Grunwald, David and Hériché, Jean-Karim and Iudin, Andrii and Martins, Gabriel G. and Meehan, Terry and Narayan, Kedar and Patwardhan, Ardan and Russell, Matthew Robert Geoffrey and Saibil, Helen R. and Strambio-De-Castillia, Caterina and Swedlow, Jason R. and Tischer, Christian and Uhlmann, Virginie and Verkade, Paul and Barlow, Mary and Bayraktar, Omer and Birney, Ewan and Catavitello, Cesare and Cawthorne, Christopher and Wagner-Conrad, Stephan and Duke, Elizabeth and Paul-Gilloteaux, Perrine and Gustin, Emmanuel and Harkiolaki, Maria and Kankaanpää, Pasi and Lemberger, Thomas and McEntyre, Jo and Moore, Josh and Nicholls, Andrew W. and Onami, Shuichi and Parkinson, Helen and Parsons, Maddy and Romanchikova, Marina and Sofroniew, Nicholas and Swoger, Jim and Utz, Nadine and Voortman, Lenard M. and Wong, Frances and Zhang, Peijun and Kleywegt, Gerard J. and Brazma, Alvis},
  doi          = {10.1038/s41592-021-01166-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1418-1422},
  shortjournal = {Nat. Meth.},
  title        = {REMBI: Recommended metadata for biological images—enabling reuse of microscopy data in biology},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MDEmic: A metadata annotation tool to facilitate management
of FAIR image data in the bioimaging community. <em>NMETH</em>,
<em>18</em>(12), 1416–1417. (<a
href="https://doi.org/10.1038/s41592-021-01288-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kunis, Susanne and Hänsch, Sebastian and Schmidt, Christian and Wong, Frances and Strambio-De-Castillia, Caterina and Weidtkamp-Peters, Stefanie},
  doi          = {10.1038/s41592-021-01288-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1416-1417},
  shortjournal = {Nat. Meth.},
  title        = {MDEmic: A metadata annotation tool to facilitate management of FAIR image data in the bioimaging community},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MethodsJ2: A software tool to capture metadata and generate
comprehensive microscopy methods text. <em>NMETH</em>, <em>18</em>(12),
1414–1416. (<a
href="https://doi.org/10.1038/s41592-021-01290-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Ryan, Joel and Pengo, Thomas and Rigano, Alex and Llopis, Paula Montero and Itano, Michelle S. and Cameron, Lisa A. and Marqués, Guillermo and Strambio-De-Castillia, Caterina and Sanders, Mark A. and Brown, Claire M.},
  doi          = {10.1038/s41592-021-01290-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1414-1416},
  shortjournal = {Nat. Meth.},
  title        = {MethodsJ2: A software tool to capture metadata and generate comprehensive microscopy methods text},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Caterina strambio-de-castillia. <em>NMETH</em>,
<em>18</em>(12), 1413. (<a
href="https://doi.org/10.1038/s41592-021-01342-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sharing, teaching and singing to bring people together on microscopy standards.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01342-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1413},
  shortjournal = {Nat. Meth.},
  title        = {Caterina strambio-de-castillia},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Minding microscopy metadata. <em>NMETH</em>,
<em>18</em>(12), 1411. (<a
href="https://doi.org/10.1038/s41592-021-01347-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Guidelines for improving reporting and reproduciblity in microscopy take center stage in this month’s Focus issue.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01347-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1411},
  shortjournal = {Nat. Meth.},
  title        = {Minding microscopy metadata},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Deep learning enables fast and dense
single-molecule localization with high accuracy. <em>NMETH</em>,
<em>18</em>(11), 1410. (<a
href="https://doi.org/10.1038/s41592-021-01305-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Speiser, Artur and Müller, Lucas-Raphael and Hoess, Philipp and Matti, Ulf and Obara, Christopher J. and Legant, Wesley R. and Kreshuk, Anna and Macke, Jakob H. and Ries, Jonas and Turaga, Srinivas C.},
  doi          = {10.1038/s41592-021-01305-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1410},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Deep learning enables fast and dense single-molecule localization with high accuracy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: DOME: Recommendations for supervised
machine learning validation in biology. <em>NMETH</em>, <em>18</em>(11),
1409–1410. (<a
href="https://doi.org/10.1038/s41592-021-01304-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Walsh, Ian and Fishman, Dmytro and Garcia-Gasulla, Dario and Titma, Tiina and Pollastri, Gianluca and Harrow, Jennifer and Psomopoulos, Fotis E. and Tosatto, Silvio C. E.},
  doi          = {10.1038/s41592-021-01304-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1409-1410},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: DOME: recommendations for supervised machine learning validation in biology},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Removing independent noise in systems neuroscience data
using DeepInterpolation. <em>NMETH</em>, <em>18</em>(11), 1401–1408. (<a
href="https://doi.org/10.1038/s41592-021-01285-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Progress in many scientific disciplines is hindered by the presence of independent noise. Technologies for measuring neural activity (calcium imaging, extracellular electrophysiology and functional magnetic resonance imaging (fMRI)) operate in domains in which independent noise (shot noise and/or thermal noise) can overwhelm physiological signals. Here, we introduce DeepInterpolation, a general-purpose denoising algorithm that trains a spatiotemporal nonlinear interpolation model using only raw noisy samples. Applying DeepInterpolation to two-photon calcium imaging data yielded up to six times more neuronal segments than those computed from raw data with a 15-fold increase in the single-pixel signal-to-noise ratio (SNR), uncovering single-trial network dynamics that were previously obscured by noise. Extracellular electrophysiology recordings processed with DeepInterpolation yielded 25\% more high-quality spiking units than those computed from raw data, while DeepInterpolation produced a 1.6-fold increase in the SNR of individual voxels in fMRI datasets. Denoising was attained without sacrificing spatial or temporal resolution and without access to ground truth training data. We anticipate that DeepInterpolation will provide similar benefits in other domains in which independent noise contaminates spatiotemporally structured datasets. DeepInterpolation is a self-supervised deep learning-based denoising approach for calcium imaging, electrophysiology and fMRI data. The approach increases the signal-to-noise ratio and allows extraction of more information from the processed data than from the raw data.},
  archive      = {J_NMETH},
  author       = {Lecoq, Jérôme and Oliver, Michael and Siegle, Joshua H. and Orlova, Natalia and Ledochowitsch, Peter and Koch, Christof},
  doi          = {10.1038/s41592-021-01285-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1401-1408},
  shortjournal = {Nat. Meth.},
  title        = {Removing independent noise in systems neuroscience data using DeepInterpolation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reinforcing neuron extraction and spike inference in calcium
imaging using deep self-supervised denoising. <em>NMETH</em>,
<em>18</em>(11), 1395–1400. (<a
href="https://doi.org/10.1038/s41592-021-01225-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Calcium imaging has transformed neuroscience research by providing a methodology for monitoring the activity of neural circuits with single-cell resolution. However, calcium imaging is inherently susceptible to detection noise, especially when imaging with high frame rate or under low excitation dosage. Here we developed DeepCAD, a self-supervised deep-learning method for spatiotemporal enhancement of calcium imaging data that does not require any high signal-to-noise ratio (SNR) observations. DeepCAD suppresses detection noise and improves the SNR more than tenfold, which reinforces the accuracy of neuron extraction and spike inference and facilitates the functional analysis of neural circuits. DeepCAD is a self-supervised deep-learning approach for denoising calcium imaging data. DeepCAD improved SNR and facilitates neuron extraction and spike inference.},
  archive      = {J_NMETH},
  author       = {Li, Xinyang and Zhang, Guoxun and Wu, Jiamin and Zhang, Yuanlong and Zhao, Zhifeng and Lin, Xing and Qiao, Hui and Xie, Hao and Wang, Haoqian and Fang, Lu and Dai, Qionghai},
  doi          = {10.1038/s41592-021-01225-0},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1395-1400},
  shortjournal = {Nat. Meth.},
  title        = {Reinforcing neuron extraction and spike inference in calcium imaging using deep self-supervised denoising},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning improves macromolecule identification in 3D
cellular cryo-electron tomograms. <em>NMETH</em>, <em>18</em>(11),
1386–1394. (<a
href="https://doi.org/10.1038/s41592-021-01275-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron tomography (cryo-ET) visualizes the 3D spatial distribution of macromolecules at nanometer resolution inside native cells. However, automated identification of macromolecules inside cellular tomograms is challenged by noise and reconstruction artifacts, as well as the presence of many molecular species in the crowded volumes. Here, we present DeepFinder, a computational procedure that uses artificial neural networks to simultaneously localize multiple classes of macromolecules. Once trained, the inference stage of DeepFinder is faster than template matching and performs better than other competitive deep learning methods at identifying macromolecules of various sizes in both synthetic and experimental datasets. On cellular cryo-ET data, DeepFinder localized membrane-bound and cytosolic ribosomes (roughly 3.2 MDa), ribulose 1,5-bisphosphate carboxylase–oxygenase (roughly 560 kDa soluble complex) and photosystem II (roughly 550 kDa membrane complex) with an accuracy comparable to expert-supervised ground truth annotations. DeepFinder is therefore a promising algorithm for the semiautomated analysis of a wide range of molecular targets in cellular tomograms. DeepFinder is a deep learning-based tool for identifying macromolecules in cellular cryo-electron tomograms. DeepFinder performs with an accuracy comparable to expert-supervised ground truth annotations on multiple experimental datasets.},
  archive      = {J_NMETH},
  author       = {Moebel, Emmanuel and Martinez-Sanchez, Antonio and Lamm, Lorenz and Righetto, Ricardo D. and Wietrzynski, Wojciech and Albert, Sahradha and Larivière, Damien and Fourmentin, Eric and Pfeffer, Stefan and Ortiz, Julio and Baumeister, Wolfgang and Peng, Tingying and Engel, Benjamin D. and Kervrann, Charles},
  doi          = {10.1038/s41592-021-01275-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1386-1394},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning improves macromolecule identification in 3D cellular cryo-electron tomograms},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Metabolite discovery through global annotation of untargeted
metabolomics data. <em>NMETH</em>, <em>18</em>(11), 1377–1385. (<a
href="https://doi.org/10.1038/s41592-021-01303-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Liquid chromatography–high-resolution mass spectrometry (LC-MS)-based metabolomics aims to identify and quantify all metabolites, but most LC-MS peaks remain unidentified. Here we present a global network optimization approach, NetID, to annotate untargeted LC-MS metabolomics data. The approach aims to generate, for all experimentally observed ion peaks, annotations that match the measured masses, retention times and (when available) tandem mass spectrometry fragmentation patterns. Peaks are connected based on mass differences reflecting adduction, fragmentation, isotopes, or feasible biochemical transformations. Global optimization generates a single network linking most observed ion peaks, enhances peak assignment accuracy, and produces chemically informative peak–peak relationships, including for peaks lacking tandem mass spectrometry spectra. Applying this approach to yeast and mouse data, we identified five previously unrecognized metabolites (thiamine derivatives and N-glucosyl-taurine). Isotope tracer studies indicate active flux through these metabolites. Thus, NetID applies existing metabolomic knowledge and global optimization to substantially improve annotation coverage and accuracy in untargeted metabolomics datasets, facilitating metabolite discovery. The NetID algorithm annotates untargeted LC-MS metabolomics data by combining known biochemical and metabolomic principles with a global network optimization strategy.},
  archive      = {J_NMETH},
  author       = {Chen, Li and Lu, Wenyun and Wang, Lin and Xing, Xi and Chen, Ziyang and Teng, Xin and Zeng, Xianfeng and Muscarella, Antonio D. and Shen, Yihui and Cowan, Alexis and McReynolds, Melanie R. and Kennedy, Brandon J. and Lato, Ashley M. and Campagna, Shawn R. and Singh, Mona and Rabinowitz, Joshua D.},
  doi          = {10.1038/s41592-021-01303-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1377-1385},
  shortjournal = {Nat. Meth.},
  title        = {Metabolite discovery through global annotation of untargeted metabolomics data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). HERMES: A molecular-formula-oriented method to target the
metabolome. <em>NMETH</em>, <em>18</em>(11), 1370–1376. (<a
href="https://doi.org/10.1038/s41592-021-01307-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive metabolome analyses are essential for biomedical, environmental, and biotechnological research. However, current MS1- and MS2-based acquisition and data analysis strategies in untargeted metabolomics result in low identification rates of metabolites. Here we present HERMES, a molecular-formula-oriented and peak-detection-free method that uses raw LC/MS1 information to optimize MS2 acquisition. Investigating environmental water, Escherichia coli, and human plasma extracts with HERMES, we achieved an increased biological specificity of MS2 scans, leading to improved mass spectral similarity scoring and identification rates when compared with a state-of-the-art data-dependent acquisition (DDA) approach. Thus, HERMES improves sensitivity, selectivity, and annotation of metabolites. HERMES is available as an R package with a user-friendly graphical interface for data analysis and visualization. HERMES is a molecular-formula-oriented and peak-detection-free method that uses LC/MS1 information to optimize MS2 acquisition for LC/MS-based metabolomic analysis.},
  archive      = {J_NMETH},
  author       = {Giné, Roger and Capellades, Jordi and Badia, Josep M. and Vughs, Dennis and Schwaiger-Haber, Michaela and Alexandrov, Theodore and Vinaixa, Maria and Brunner, Andrea M. and Patti, Gary J. and Yanes, Oscar},
  doi          = {10.1038/s41592-021-01307-z},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1370-1376},
  shortjournal = {Nat. Meth.},
  title        = {HERMES: A molecular-formula-oriented method to target the metabolome},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). DeepLC can predict retention times for peptides that carry
as-yet unseen modifications. <em>NMETH</em>, <em>18</em>(11), 1363–1369.
(<a href="https://doi.org/10.1038/s41592-021-01301-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The inclusion of peptide retention time prediction promises to remove peptide identification ambiguity in complex liquid chromatography–mass spectrometry identification workflows. However, due to the way peptides are encoded in current prediction models, accurate retention times cannot be predicted for modified peptides. This is especially problematic for fledgling open searches, which will benefit from accurate retention time prediction for modified peptides to reduce identification ambiguity. We present DeepLC, a deep learning peptide retention time predictor using peptide encoding based on atomic composition that allows the retention time of (previously unseen) modified peptides to be predicted accurately. We show that DeepLC performs similarly to current state-of-the-art approaches for unmodified peptides and, more importantly, accurately predicts retention times for modifications not seen during training. Moreover, we show that DeepLC’s ability to predict retention times for any modification enables potentially incorrect identifications to be flagged in an open search of a wide variety of proteome data. DeepLC, a deep learning-based peptide retention time predictor, can predict retention times for unmodified peptides as well as peptides with previously unseen modifications.},
  archive      = {J_NMETH},
  author       = {Bouwmeester, Robbin and Gabriels, Ralf and Hulstaert, Niels and Martens, Lennart and Degroeve, Sven},
  doi          = {10.1038/s41592-021-01301-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1363-1369},
  shortjournal = {Nat. Meth.},
  title        = {DeepLC can predict retention times for peptides that carry as-yet unseen modifications},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning and alignment of spatially resolved
single-cell transcriptomes with tangram. <em>NMETH</em>,
<em>18</em>(11), 1352–1362. (<a
href="https://doi.org/10.1038/s41592-021-01264-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Charting an organs’ biological atlas requires us to spatially resolve the entire single-cell transcriptome, and to relate such cellular features to the anatomical scale. Single-cell and single-nucleus RNA-seq (sc/snRNA-seq) can profile cells comprehensively, but lose spatial information. Spatial transcriptomics allows for spatial measurements, but at lower resolution and with limited sensitivity. Targeted in situ technologies solve both issues, but are limited in gene throughput. To overcome these limitations we present Tangram, a method that aligns sc/snRNA-seq data to various forms of spatial data collected from the same region, including MERFISH, STARmap, smFISH, Spatial Transcriptomics (Visium) and histological images. Tangram can map any type of sc/snRNA-seq data, including multimodal data such as those from SHARE-seq, which we used to reveal spatial patterns of chromatin accessibility. We demonstrate Tangram on healthy mouse brain tissue, by reconstructing a genome-wide anatomically integrated spatial map at single-cell resolution of the visual and somatomotor areas. Tangram is a versatile tool for aligning single-cell and single-nucleus RNA-seq data to spatially resolved transcriptomics data using deep learning.},
  archive      = {J_NMETH},
  author       = {Biancalani, Tommaso and Scalia, Gabriele and Buffoni, Lorenzo and Avasthi, Raghav and Lu, Ziqing and Sanger, Aman and Tokcan, Neriman and Vanderburg, Charles R. and Segerstolpe, Åsa and Zhang, Meng and Avraham-Davidi, Inbal and Vickovic, Sanja and Nitzan, Mor and Ma, Sai and Subramanian, Ayshwarya and Lipinski, Michal and Buenrostro, Jason and Brown, Nik Bear and Fanelli, Duccio and Zhuang, Xiaowei and Macosko, Evan Z. and Regev, Aviv},
  doi          = {10.1038/s41592-021-01264-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1352-1362},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning and alignment of spatially resolved single-cell transcriptomes with tangram},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SpaGCN: Integrating gene expression, spatial location and
histology to identify spatial domains and spatially variable genes by
graph convolutional network. <em>NMETH</em>, <em>18</em>(11), 1342–1351.
(<a href="https://doi.org/10.1038/s41592-021-01255-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in spatially resolved transcriptomics (SRT) technologies have enabled comprehensive characterization of gene expression patterns in the context of tissue microenvironment. To elucidate spatial gene expression variation, we present SpaGCN, a graph convolutional network approach that integrates gene expression, spatial location and histology in SRT data analysis. Through graph convolution, SpaGCN aggregates gene expression of each spot from its neighboring spots, which enables the identification of spatial domains with coherent expression and histology. The subsequent domain guided differential expression (DE) analysis then detects genes with enriched expression patterns in the identified domains. Analyzing seven SRT datasets using SpaGCN, we show it can detect genes with much more enriched spatial expression patterns than competing methods. Furthermore, genes detected by SpaGCN are transferrable and can be utilized to study spatial variation of gene expression in other datasets. SpaGCN is computationally fast, platform independent, making it a desirable tool for diverse SRT studies. SpaGCN is a spatially resolved transcriptomics data analysis tool for identifying spatial domains and spatially variable genes using graph convolutional networks.},
  archive      = {J_NMETH},
  author       = {Hu, Jian and Li, Xiangjie and Coleman, Kyle and Schroeder, Amelia and Ma, Nan and Irwin, David J. and Lee, Edward B. and Shinohara, Russell T. and Li, Mingyao},
  doi          = {10.1038/s41592-021-01255-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1342-1351},
  shortjournal = {Nat. Meth.},
  title        = {SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell chromatin state analysis with signac.
<em>NMETH</em>, <em>18</em>(11), 1333–1341. (<a
href="https://doi.org/10.1038/s41592-021-01282-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recent development of experimental methods for measuring chromatin state at single-cell resolution has created a need for computational tools capable of analyzing these datasets. Here we developed Signac, a comprehensive toolkit for the analysis of single-cell chromatin data. Signac enables an end-to-end analysis of single-cell chromatin data, including peak calling, quantification, quality control, dimension reduction, clustering, integration with single-cell gene expression datasets, DNA motif analysis and interactive visualization. Through its seamless compatibility with the Seurat package, Signac facilitates the analysis of diverse multimodal single-cell chromatin data, including datasets that co-assay DNA accessibility with gene expression, protein abundance and mitochondrial genotype. We demonstrate scaling of the Signac framework to analyze datasets containing over 700,000 cells. The Signac framework enables the end-to-end analysis of single-cell chromatin data and interoperability with the Seurat package for multimodal analysis.},
  archive      = {J_NMETH},
  author       = {Stuart, Tim and Srivastava, Avi and Madad, Shaista and Lareau, Caleb A. and Satija, Rahul},
  doi          = {10.1038/s41592-021-01282-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1333-1341},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell chromatin state analysis with signac},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Haplotype-aware variant calling with
PEPPER-margin-DeepVariant enables high accuracy in nanopore long-reads.
<em>NMETH</em>, <em>18</em>(11), 1322–1332. (<a
href="https://doi.org/10.1038/s41592-021-01299-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing has the potential to transform variant detection by reaching currently difficult-to-map regions and routinely linking together adjacent variations to enable read-based phasing. Third-generation nanopore sequence data have demonstrated a long read length, but current interpretation methods for their novel pore-based signal have unique error profiles, making accurate analysis challenging. Here, we introduce a haplotype-aware variant calling pipeline, PEPPER-Margin-DeepVariant, that produces state-of-the-art variant calling results with nanopore data. We show that our nanopore-based method outperforms the short-read-based single-nucleotide-variant identification method at the whole-genome scale and produces high-quality single-nucleotide variants in segmental duplications and low-mappability regions where short-read-based genotyping fails. We show that our pipeline can provide highly contiguous phase blocks across the genome with nanopore reads, contiguously spanning between 85\% and 92\% of annotated genes across six samples. We also extend PEPPER-Margin-DeepVariant to PacBio HiFi data, providing an efficient solution with superior performance over the current WhatsHap-DeepVariant standard. Finally, we demonstrate de novo assembly polishing methods that use nanopore and PacBio HiFi reads to produce diploid assemblies with high accuracy (Q35+ nanopore-polished and Q40+ PacBio HiFi-polished). The PEPPER-Margin-DeepVariant pipeline achieves highly accurate variant calling using nanopore and other long-read sequencing data.},
  archive      = {J_NMETH},
  author       = {Shafin, Kishwar and Pesout, Trevor and Chang, Pi-Chuan and Nattestad, Maria and Kolesnikov, Alexey and Goel, Sidharth and Baid, Gunjan and Kolmogorov, Mikhail and Eizenga, Jordan M. and Miga, Karen H. and Carnevali, Paolo and Jain, Miten and Carroll, Andrew and Paten, Benedict},
  doi          = {10.1038/s41592-021-01299-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1322-1332},
  shortjournal = {Nat. Meth.},
  title        = {Haplotype-aware variant calling with PEPPER-margin-DeepVariant enables high accuracy in nanopore long-reads},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An analytical framework for interpretable and generalizable
single-cell data analysis. <em>NMETH</em>, <em>18</em>(11), 1317–1321.
(<a href="https://doi.org/10.1038/s41592-021-01286-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scaling of single-cell data exploratory analysis with the rapidly growing diversity and quantity of single-cell omics datasets demands more interpretable and robust data representation that is generalizable across datasets. Here, we have developed a ‘linearly interpretable’ framework that combines the interpretability and transferability of linear methods with the representational power of non-linear methods. Within this framework we introduce a data representation and visualization method, GraphDR, and a structure discovery method, StructDR, that unifies cluster, trajectory and surface estimation and enables their confidence set inference. A linearly interpretable framework for analyzing single-cell omics data improves data representation, visualization and structure discovery.},
  archive      = {J_NMETH},
  author       = {Zhou, Jian and Troyanskaya, Olga G.},
  doi          = {10.1038/s41592-021-01286-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1317-1321},
  shortjournal = {Nat. Meth.},
  title        = {An analytical framework for interpretable and generalizable single-cell data analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Community evaluation of glycoproteomics informatics
solutions reveals high-performance search strategies for serum
glycopeptide analysis. <em>NMETH</em>, <em>18</em>(11), 1304–1316. (<a
href="https://doi.org/10.1038/s41592-021-01309-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glycoproteomics is a powerful yet analytically challenging research tool. Software packages aiding the interpretation of complex glycopeptide tandem mass spectra have appeared, but their relative performance remains untested. Conducted through the HUPO Human Glycoproteomics Initiative, this community study, comprising both developers and users of glycoproteomics software, evaluates solutions for system-wide glycopeptide analysis. The same mass spectrometry based glycoproteomics datasets from human serum were shared with participants and the relative team performance for N﻿- and O-glycopeptide data analysis was comprehensively established by orthogonal performance tests. Although the results were variable, several high-performance glycoproteomics informatics strategies were identified. Deep analysis of the data revealed key performance-associated search parameters and led to recommendations for improved ‘high-coverage’ and ‘high-accuracy’ glycoproteomics search solutions. This study concludes that diverse software packages for comprehensive glycopeptide data analysis exist, points to several high-performance search strategies and specifies key variables that will guide future software developments and assist informatics decision-making in glycoproteomics. This analysis presents the results of a community-based evaluation of existing software for large-scale glycopeptide data analysis.},
  archive      = {J_NMETH},
  author       = {Kawahara, Rebeca and Chernykh, Anastasia and Alagesan, Kathirvel and Bern, Marshall and Cao, Weiqian and Chalkley, Robert J. and Cheng, Kai and Choo, Matthew S. and Edwards, Nathan and Goldman, Radoslav and Hoffmann, Marcus and Hu, Yingwei and Huang, Yifan and Kim, Jin Young and Kletter, Doron and Liquet, Benoit and Liu, Mingqi and Mechref, Yehia and Meng, Bo and Neelamegham, Sriram and Nguyen-Khuong, Terry and Nilsson, Jonas and Pap, Adam and Park, Gun Wook and Parker, Benjamin L. and Pegg, Cassandra L. and Penninger, Josef M. and Phung, Toan K. and Pioch, Markus and Rapp, Erdmann and Sakalli, Enes and Sanda, Miloslav and Schulz, Benjamin L. and Scott, Nichollas E. and Sofronov, Georgy and Stadlmann, Johannes and Vakhrushev, Sergey Y. and Woo, Christina M. and Wu, Hung-Yi and Yang, Pengyuan and Ying, Wantao and Zhang, Hui and Zhang, Yong and Zhao, Jingfu and Zaia, Joseph and Haslam, Stuart M. and Palmisano, Giuseppe and Yoo, Jong Shin and Larson, Göran and Khoo, Kai-Hooi and Medzihradszky, Katalin F. and Kolarich, Daniel and Packer, Nicolle H. and Thaysen-Andersen, Morten},
  doi          = {10.1038/s41592-021-01309-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1304-1316},
  shortjournal = {Nat. Meth.},
  title        = {Community evaluation of glycoproteomics informatics solutions reveals high-performance search strategies for serum glycopeptide analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MISpheroID: A knowledgebase and transparency tool for
minimum information in spheroid identity. <em>NMETH</em>,
<em>18</em>(11), 1294–1303. (<a
href="https://doi.org/10.1038/s41592-021-01291-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spheroids are three-dimensional cellular models with widespread basic and translational application across academia and industry. However, methodological transparency and guidelines for spheroid research have not yet been established. The MISpheroID Consortium developed a crowdsourcing knowledgebase that assembles the experimental parameters of 3,058 published spheroid-related experiments. Interrogation of this knowledgebase identified heterogeneity in the methodological setup of spheroids. Empirical evaluation and interlaboratory validation of selected variations in spheroid methodology revealed diverse impacts on spheroid metrics. To facilitate interpretation, stimulate transparency and increase awareness, the Consortium defines the MISpheroID string, a minimum set of experimental parameters required to report spheroid research. Thus, MISpheroID combines a valuable resource and a tool for three-dimensional cellular models to mine experimental parameters and to improve reproducibility. A knowledgebase developed for increased the transparency of reporting in spheroid research.},
  archive      = {J_NMETH},
  author       = {Peirsman, Arne and Blondeel, Eva and Ahmed, Tasdiq and Anckaert, Jasper and Audenaert, Dominique and Boterberg, Tom and Buzas, Krisztina and Carragher, Neil and Castellani, Gastone and Castro, Flávia and Dangles-Marie, Virginie and Dawson, John and De Tullio, Pascal and De Vlieghere, Elly and Dedeyne, Sándor and Depypere, Herman and Diosdi, Akos and Dmitriev, Ruslan I. and Dolznig, Helmut and Fischer, Suzanne and Gespach, Christian and Goossens, Vera and Heino, Jyrki and Hendrix, An and Horvath, Peter and Kunz-Schughart, Leoni A. and Maes, Sebastiaan and Mangodt, Christophe and Mestdagh, Pieter and Michlíková, Soňa and Oliveira, Maria José and Pampaloni, Francesco and Piccinini, Filippo and Pinheiro, Cláudio and Rahn, Jennifer and Robbins, Stephen M. and Siljamäki, Elina and Steigemann, Patrick and Sys, Gwen and Takayama, Shuichi and Tesei, Anna and Tulkens, Joeri and Van Waeyenberge, Michiel and Vandesompele, Jo and Wagemans, Glenn and Weindorfer, Claudia and Yigit, Nurten and Zablowsky, Nina and Zanoni, Michele and Blondeel, Phillip and De Wever, Olivier},
  doi          = {10.1038/s41592-021-01291-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1294-1303},
  shortjournal = {Nat. Meth.},
  title        = {MISpheroID: A knowledgebase and transparency tool for minimum information in spheroid identity},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Time for NanoNeuro. <em>NMETH</em>, <em>18</em>(11),
1287–1293. (<a
href="https://doi.org/10.1038/s41592-021-01270-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study of electronic properties of materials at the nanoscale has unveiled physical laws and generated materials such as nanoparticles, quantum dots, nanodiamonds, nanoelectrodes, and nanoprobes. Independently, large-scale public and private neuroscience programs have been launched to develop methods to measure and manipulate neural circuits in living animals and humans. Here, we review an upcoming field, NanoNeuro, defined as the intersection of nanoscience and neuroscience, that aims to develop nanoscale methods to record and stimulate neuronal activity. Because of their unique physical properties, nanomaterials have intrinsic advantages as biosensors and actuators, and they may be applicable to humans without the need for genetic modifications. Thus, nanoscience could make major methodological contributions to the future of neuroscience and, more generally, to biomedical sciences. This Perspective discusses how neuroscience can benefit from harnessing nanoscience-based tools for manipulating and recording neuronal activity.},
  archive      = {J_NMETH},
  author       = {Garcia-Etxarri, Aitzol and Yuste, Rafael},
  doi          = {10.1038/s41592-021-01270-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1287-1293},
  shortjournal = {Nat. Meth.},
  title        = {Time for NanoNeuro},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards spheroid-omics. <em>NMETH</em>, <em>18</em>(11),
1283–1285. (<a
href="https://doi.org/10.1038/s41592-021-01311-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The MISpheroID knowledgebase records and organizes experimental parameters from thousands of cancer spheroid experiments, revealing heterogeneity and a lack of transparency in key spheroid research reporting practices.},
  archive      = {J_NMETH},
  author       = {Downing, Timothy L.},
  doi          = {10.1038/s41592-021-01311-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1283-1285},
  shortjournal = {Nat. Meth.},
  title        = {Towards spheroid-omics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Integrative analysis methods for spatial transcriptomics.
<em>NMETH</em>, <em>18</em>(11), 1282–1283. (<a
href="https://doi.org/10.1038/s41592-021-01272-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational methods use different integrative strategies to tackle the challenges of spatially resolved transcriptomics data analysis.},
  archive      = {J_NMETH},
  author       = {Lu, Shaina and Fürth, Daniel and Gillis, Jesse},
  doi          = {10.1038/s41592-021-01272-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1282-1283},
  shortjournal = {Nat. Meth.},
  title        = {Integrative analysis methods for spatial transcriptomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Microscopy made to order. <em>NMETH</em>, <em>18</em>(11),
1277–1281. (<a
href="https://doi.org/10.1038/s41592-021-01313-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The proliferation of versatile open software and hardware for microscopy is helping to democratize biological imaging for both current and aspiring scientists.},
  archive      = {J_NMETH},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41592-021-01313-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1277-1281},
  shortjournal = {Nat. Meth.},
  title        = {Microscopy made to order},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Expanding RNA target effectors. <em>NMETH</em>,
<em>18</em>(11), 1276. (<a
href="https://doi.org/10.1038/s41592-021-01323-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers identified a single-effector Cas, Cas7-11, that architecturally diverges from previous CRISPR single- or multi-component systems and can possess RNA knockdown activity without detectable collateral activity in mammalian cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01323-z},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1276},
  shortjournal = {Nat. Meth.},
  title        = {Expanding RNA target effectors},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). DELIVERing transgenes into muscle. <em>NMETH</em>,
<em>18</em>(11), 1275. (<a
href="https://doi.org/10.1038/s41592-021-01322-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed evolution of adeno-associated virus capsids with the DELIVER approach generates tissue-specific viruses.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01322-0},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1275},
  shortjournal = {Nat. Meth.},
  title        = {DELIVERing transgenes into muscle},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Synthetic tumor microenvironments. <em>NMETH</em>,
<em>18</em>(11), 1274. (<a
href="https://doi.org/10.1038/s41592-021-01321-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An artificial system models the mechanical properties of the tumor microenvironment in vitro.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01321-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1274},
  shortjournal = {Nat. Meth.},
  title        = {Synthetic tumor microenvironments},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Revealing the secretome. <em>NMETH</em>, <em>18</em>(11),
1273. (<a href="https://doi.org/10.1038/s41592-021-01320-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Four independent research groups have used proximity ligation techniques to profile the in vivo mouse secretome.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01320-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1273},
  shortjournal = {Nat. Meth.},
  title        = {Revealing the secretome},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021m). Morten thaysen-andersen. <em>NMETH</em>, <em>18</em>(11),
1269. (<a href="https://doi.org/10.1038/s41592-021-01314-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sydney’s beauty, candy-store lollies and a community view of glycoproteomics software.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01314-0},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1269},
  shortjournal = {Nat. Meth.},
  title        = {Morten thaysen-andersen},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The art of microscopy. <em>NMETH</em>, <em>18</em>(11),
1267. (<a href="https://doi.org/10.1038/s41592-021-01324-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This month, we celebrate the value and beauty of images of biological processes captured by microscopes.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01324-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1267},
  shortjournal = {Nat. Meth.},
  title        = {The art of microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The class imbalance problem. <em>NMETH</em>,
<em>18</em>(11), 1270–1272. (<a
href="https://doi.org/10.1038/s41592-021-01302-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The exception proves the rule.},
  archive      = {J_NMETH},
  author       = {Megahed, Fadel M. and Chen, Ying-Ju and Megahed, Aly and Ong, Yuya and Altman, Naomi and Krzywinski, Martin},
  doi          = {10.1038/s41592-021-01302-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {11},
  pages        = {1270-1272},
  shortjournal = {Nat. Meth.},
  title        = {The class imbalance problem},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Comparison of the hi-c, GAM and
SPRITE methods using polymer models of chromatin. <em>NMETH</em>,
<em>18</em>(10), 1266. (<a
href="https://doi.org/10.1038/s41592-021-01251-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Fiorillo, Luca and Musella, Francesco and Conte, Mattia and Kempfer, Rieke and Chiariello, Andrea M. and Bianco, Simona and Kukalev, Alexander and Irastorza-Azcarate, Ibai and Esposito, Andrea and Abraham, Alex and Prisco, Antonella and Pombo, Ana and Nicodemi, Mario},
  doi          = {10.1038/s41592-021-01251-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1266},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Comparison of the hi-c, GAM and SPRITE methods using polymer models of chromatin},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Publisher correction: Towards resolving proteomes in single
cells. <em>NMETH</em>, <em>18</em>(10), 1266. (<a
href="https://doi.org/10.1038/s41592-021-01298-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01298-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1266},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Towards resolving proteomes in single cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: PaintSHOP enables the interactive design
of transcriptome- and genome-scale oligonucleotide FISH experiments.
<em>NMETH</em>, <em>18</em>(10), 1265. (<a
href="https://doi.org/10.1038/s41592-021-01273-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hershberg, Elliot A. and Camplisson, Conor K. and Close, Jennie L. and Attar, Sahar and Chern, Ryan and Liu, Yuzhen and Akilesh, Shreeram and Nicovich, Philip R. and Beliveau, Brian J.},
  doi          = {10.1038/s41592-021-01273-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1265},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: PaintSHOP enables the interactive design of transcriptome- and genome-scale oligonucleotide FISH experiments},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Graphical assessment of tests and
classifiers. <em>NMETH</em>, <em>18</em>(10), 1265. (<a
href="https://doi.org/10.1038/s41592-021-01289-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Altman, Naomi and Krzywinski, Martin},
  doi          = {10.1038/s41592-021-01289-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1265},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Graphical assessment of tests and classifiers},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An adaptive optics module for deep tissue multiphoton
imaging in vivo. <em>NMETH</em>, <em>18</em>(10), 1259–1264. (<a
href="https://doi.org/10.1038/s41592-021-01279-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding complex biological systems requires visualizing structures and processes deep within living organisms. We developed a compact adaptive optics module and incorporated it into two- and three-photon fluorescence microscopes, to measure and correct tissue-induced aberrations. We resolved synaptic structures in deep cortical and subcortical areas of the mouse brain, and demonstrated high-resolution imaging of neuronal structures and somatosensory-evoked calcium responses in the mouse spinal cord at great depths in vivo. A compact adaptive optics module corrects aberrations in two-photon and three-photon microscopy, enabling structural and functional imaging deep in the mouse brain, the mouse spinal cord and the zebrafish larva.},
  archive      = {J_NMETH},
  author       = {Rodríguez, Cristina and Chen, Anderson and Rivera, José A. and Mohr, Manuel A. and Liang, Yajie and Natan, Ryan G. and Sun, Wenzhi and Milkie, Daniel E. and Bifano, Thomas G. and Chen, Xiaoke and Ji, Na},
  doi          = {10.1038/s41592-021-01279-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1259-1264},
  shortjournal = {Nat. Meth.},
  title        = {An adaptive optics module for deep tissue multiphoton imaging in vivo},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-resolution structural and functional deep brain imaging
using adaptive optics three-photon microscopy. <em>NMETH</em>,
<em>18</em>(10), 1253–1258. (<a
href="https://doi.org/10.1038/s41592-021-01257-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiphoton microscopy has become a powerful tool with which to visualize the morphology and function of neural cells and circuits in the intact mammalian brain. However, tissue scattering, optical aberrations and motion artifacts degrade the imaging performance at depth. Here we describe a minimally invasive intravital imaging methodology based on three-photon excitation, indirect adaptive optics (AO) and active electrocardiogram gating to advance deep-tissue imaging. Our modal-based, sensorless AO approach is robust to low signal-to-noise ratios as commonly encountered in deep scattering tissues such as the mouse brain, and permits AO correction over large axial fields of view. We demonstrate near-diffraction-limited imaging of deep cortical spines and (sub)cortical dendrites up to a depth of 1.4 mm (the edge of the mouse CA1 hippocampus). In addition, we show applications to deep-layer calcium imaging of astrocytes, including fibrous astrocytes that reside in the highly scattering corpus callosum. Three-photon microscopy in combination with adaptive optics-based aberration correction and ECG-triggered gating allows high-resolution imaging of neurons and astrocytes up to a depth of 1.4 mm in the mouse brain.},
  archive      = {J_NMETH},
  author       = {Streich, Lina and Boffi, Juan Carlos and Wang, Ling and Alhalaseh, Khaleel and Barbieri, Matteo and Rehm, Ronja and Deivasigamani, Senthilkumar and Gross, Cornelius T. and Agarwal, Amit and Prevedel, Robert},
  doi          = {10.1038/s41592-021-01257-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1253-1258},
  shortjournal = {Nat. Meth.},
  title        = {High-resolution structural and functional deep brain imaging using adaptive optics three-photon microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mass photometry enables label-free tracking and mass
measurement of single proteins on lipid bilayers. <em>NMETH</em>,
<em>18</em>(10), 1247–1252. (<a
href="https://doi.org/10.1038/s41592-021-01261-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The quantification of membrane-associated biomolecular interactions is crucial to our understanding of various cellular processes. State-of-the-art single-molecule approaches rely largely on the addition of fluorescent labels, which complicates the quantification of the involved stoichiometries and dynamics because of low temporal resolution and the inherent limitations associated with labeling efficiency, photoblinking and photobleaching. Here, we demonstrate dynamic mass photometry, a method for label-free imaging, tracking and mass measurement of individual membrane-associated proteins diffusing on supported lipid bilayers. Application of this method to the membrane remodeling GTPase, dynamin-1, reveals heterogeneous mixtures of dimer-based oligomers, oligomer-dependent mobilities, membrane affinities and (dis)association of individual complexes. These capabilities, together with assay-based advances for studying integral membrane proteins, will enable the elucidation of biomolecular mechanisms in and on lipid bilayers. Dynamic mass photometry allows label-free tracking and mass measurement of individual membrane-associated proteins diffusing on supported lipid bilayers. The approach can be used to monitor dynamic (dis)assembly of protein complexes.},
  archive      = {J_NMETH},
  author       = {Foley, Eric D. B. and Kushwah, Manish S. and Young, Gavin and Kukura, Philipp},
  doi          = {10.1038/s41592-021-01261-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1247-1252},
  shortjournal = {Nat. Meth.},
  title        = {Mass photometry enables label-free tracking and mass measurement of single proteins on lipid bilayers},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mass-sensitive particle tracking to elucidate the
membrane-associated MinDE reaction cycle. <em>NMETH</em>,
<em>18</em>(10), 1239–1246. (<a
href="https://doi.org/10.1038/s41592-021-01260-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In spite of their great importance in biology, methods providing access to spontaneous molecular interactions with and on biological membranes have been sparse. The recent advent of mass photometry to quantify mass distributions of unlabeled biomolecules landing on surfaces raised hopes that this approach could be transferred to membranes. Here, by introducing a new interferometric scattering (iSCAT) image processing and analysis strategy adapted to diffusing particles, we enable mass-sensitive particle tracking (MSPT) of single unlabeled biomolecules on a supported lipid bilayer. We applied this approach to the highly nonlinear reaction cycles underlying MinDE protein self-organization. MSPT allowed us to determine the stoichiometry and turnover of individual membrane-bound MinD/MinDE protein complexes and to quantify their size-dependent diffusion. This study demonstrates the potential of MSPT to enhance our quantitative understanding of membrane-associated biological systems. An iSCAT image processing and analysis strategy enables mass-sensitive particle tracking (MSPT) of single unlabeled biomolecules on a supported lipid bilayer. MSPT was used to observe the (dis-)assembly of membrane complexes in real-time.},
  archive      = {J_NMETH},
  author       = {Heermann, Tamara and Steiert, Frederik and Ramm, Beatrice and Hundt, Nikolas and Schwille, Petra},
  doi          = {10.1038/s41592-021-01260-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1239-1246},
  shortjournal = {Nat. Meth.},
  title        = {Mass-sensitive particle tracking to elucidate the membrane-associated MinDE reaction cycle},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Image-guided MALDI mass spectrometry for high-throughput
single-organelle characterization. <em>NMETH</em>, <em>18</em>(10),
1233–1238. (<a
href="https://doi.org/10.1038/s41592-021-01277-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peptidergic dense-core vesicles are involved in packaging and releasing neuropeptides and peptide hormones—critical processes underlying brain, endocrine and exocrine function. Yet, the heterogeneity within these organelles, even for morphologically defined vesicle types, is not well characterized because of their small volumes. We present image-guided, high-throughput mass spectrometry-based protocols to chemically profile large populations of both dense-core vesicles and lucent vesicles for their lipid and peptide contents, allowing observation of the chemical heterogeneity within and between these two vesicle populations. The proteolytic processing products of four prohormones are observed within the dense-core vesicles, and the mass spectral features corresponding to the specific peptide products suggest three distinct dense-core vesicle populations. Notable differences in the lipid mass range are observed between the dense-core and lucent vesicles. These single-organelle mass spectrometry approaches are adaptable to characterize a range of subcellular structures. This article presents a workflow for image-guided MALDI MS characterization of peptide and lipid content in single organelles.},
  archive      = {J_NMETH},
  author       = {Castro, Daniel C. and Xie, Yuxuan Richard and Rubakhin, Stanislav S. and Romanova, Elena V. and Sweedler, Jonathan V.},
  doi          = {10.1038/s41592-021-01277-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1233-1238},
  shortjournal = {Nat. Meth.},
  title        = {Image-guided MALDI mass spectrometry for high-throughput single-organelle characterization},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SEAM is a spatial single nuclear metabolomics method for
dissecting tissue microenvironment. <em>NMETH</em>, <em>18</em>(10),
1223–1232. (<a
href="https://doi.org/10.1038/s41592-021-01276-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial metabolomics can reveal intercellular heterogeneity and tissue organization. Here we report on the spatial single nuclear metabolomics (SEAM) method, a flexible platform combining high-spatial-resolution imaging mass spectrometry and a set of computational algorithms that can display multiscale and multicolor tissue tomography together with identification and clustering of single nuclei by their in situ metabolic fingerprints. We first applied SEAM to a range of wild-type mouse tissues, then delineated a consistent pattern of metabolic zonation in mouse liver. We further studied the spatial metabolic profile in the human fibrotic liver. We discovered subpopulations of hepatocytes with special metabolic features associated with their proximity to the fibrotic niche, and validated this finding by spatial transcriptomics with Geo-seq. These demonstrations highlighted SEAM’s ability to explore the spatial metabolic profile and tissue histology at the single-cell level, leading to a deeper understanding of tissue metabolic organization. SEAM is a platform for the analysis of high-resolution secondary ion mass spectrometry imaging that allows spatially resolved nuclear metabolomic profiling at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Yuan, Zhiyuan and Zhou, Qiming and Cai, Lesi and Pan, Lin and Sun, Weiliang and Qumu, Shiwei and Yu, Si and Feng, Jiaxin and Zhao, Hansen and Zheng, Yongchang and Shi, Minglei and Li, Shao and Chen, Yang and Zhang, Xinrong and Zhang, Michael Q.},
  doi          = {10.1038/s41592-021-01276-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1223-1232},
  shortjournal = {Nat. Meth.},
  title        = {SEAM is a spatial single nuclear metabolomics method for dissecting tissue microenvironment},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Systematic calibration of epitranscriptomic maps using a
synthetic modification-free RNA library. <em>NMETH</em>,
<em>18</em>(10), 1213–1222. (<a
href="https://doi.org/10.1038/s41592-021-01280-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent years have witnessed rapid progress in the field of epitranscriptomics. Functional interpretation of the epitranscriptome relies on sequencing technologies that determine the location and stoichiometry of various RNA modifications. However, contradictory results have been reported among studies, bringing the biological impacts of certain RNA modifications into doubt. Here, we develop a synthetic RNA library resembling the endogenous transcriptome but without any RNA modification. By incorporating this modification-free RNA library into established mapping techniques as a negative control, we reveal abundant false positives resulting from sequence bias or RNA structure. After calibration, precise and quantitative mapping expands the understanding of two representative modification types, N6-methyladenosine (m6A) and 5-methylcytosine (m5C). We propose that this approach provides a systematic solution for the calibration of various RNA-modification mappings and holds great promise in epitranscriptomic studies. This work describes the generation of a modification-free RNA library that resembles endogenous transcriptome sequence and expression level, which can be used as a negative control in epitranscriptomic sequencing methods to obtain high-confidence and quantitative maps of various RNA modifications.},
  archive      = {J_NMETH},
  author       = {Zhang, Zhang and Chen, Tao and Chen, Hong-Xuan and Xie, Ying-Yuan and Chen, Li-Qian and Zhao, Yu-Li and Liu, Biao-Di and Jin, Lingmei and Zhang, Wutong and Liu, Chang and Ma, Dong-Zhao and Chai, Guo-Shi and Zhang, Ying and Zhao, Wen-Shuo and Ng, Wen Hui and Chen, Jiekai and Jia, Guifang and Yang, Jianhua and Luo, Guan-Zheng},
  doi          = {10.1038/s41592-021-01280-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1213-1222},
  shortjournal = {Nat. Meth.},
  title        = {Systematic calibration of epitranscriptomic maps using a synthetic modification-free RNA library},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Joint single-cell measurements of nuclear proteins and RNA
in vivo. <em>NMETH</em>, <em>18</em>(10), 1204–1212. (<a
href="https://doi.org/10.1038/s41592-021-01278-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying gene-regulatory targets of nuclear proteins in tissues is a challenge. Here we describe intranuclear cellular indexing of transcriptomes and epitopes (inCITE-seq), a scalable method that measures multiplexed intranuclear protein levels and the transcriptome in parallel across thousands of nuclei, enabling joint analysis of transcription factor (TF) levels and gene expression in vivo. We apply inCITE-seq to characterize cell state-related changes upon pharmacological induction of neuronal activity in the mouse brain. Modeling gene expression as a linear combination of quantitative protein levels revealed genome-wide associations of each TF and recovered known gene targets. TF-associated genes were coexpressed as distinct modules that each reflected positive or negative TF levels, showing that our approach can disentangle relative putative contributions of TFs to gene expression and add interpretability to inferred gene networks. inCITE-seq can illuminate how combinations of nuclear proteins shape gene expression in native tissue contexts, with direct applications to solid or frozen tissues and clinical specimens. This work describes inCITE-seq that jointly measures intranuclear protein levels and the transcriptome in single nuclei, which is applied to mouse brain tissue to relate quantitative protein levels of TFs to gene expression programs.},
  archive      = {J_NMETH},
  author       = {Chung, Hattie and Parkhurst, Christopher N. and Magee, Emma M. and Phillips, Devan and Habibi, Ehsan and Chen, Fei and Yeung, Bertrand Z. and Waldman, Julia and Artis, David and Regev, Aviv},
  doi          = {10.1038/s41592-021-01278-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1204-1212},
  shortjournal = {Nat. Meth.},
  title        = {Joint single-cell measurements of nuclear proteins and RNA in vivo},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Effective gene expression prediction from sequence by
integrating long-range interactions. <em>NMETH</em>, <em>18</em>(10),
1196–1203. (<a
href="https://doi.org/10.1038/s41592-021-01252-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How noncoding DNA determines gene expression in different cell types is a major unsolved problem, and critical downstream applications in human genetics depend on improved solutions. Here, we report substantially improved gene expression prediction accuracy from DNA sequences through the use of a deep learning architecture, called Enformer, that is able to integrate information from long-range interactions (up to 100 kb away) in the genome. This improvement yielded more accurate variant effect predictions on gene expression for both natural genetic variants and saturation mutagenesis measured by massively parallel reporter assays. Furthermore, Enformer learned to predict enhancer–promoter interactions directly from the DNA sequence competitively with methods that take direct experimental data as input. We expect that these advances will enable more effective fine-mapping of human disease associations and provide a framework to interpret cis-regulatory evolution. By using a new deep learning architecture, Enformer leverages long-range information to improve prediction of gene expression on the basis of DNA sequence.},
  archive      = {J_NMETH},
  author       = {Avsec, Žiga and Agarwal, Vikram and Visentin, Daniel and Ledsam, Joseph R. and Grabska-Barwinska, Agnieszka and Taylor, Kyle R. and Assael, Yannis and Jumper, John and Kohli, Pushmeet and Kelley, David R.},
  doi          = {10.1038/s41592-021-01252-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1196-1203},
  shortjournal = {Nat. Meth.},
  title        = {Effective gene expression prediction from sequence by integrating long-range interactions},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). DeepImageJ: A user-friendly environment to run deep learning
models in ImageJ. <em>NMETH</em>, <em>18</em>(10), 1192–1195. (<a
href="https://doi.org/10.1038/s41592-021-01262-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DeepImageJ is a user-friendly solution that enables the generic use of pre-trained deep learning models for biomedical image analysis in ImageJ. The deepImageJ environment gives access to the largest bioimage repository of pre-trained deep learning models (BioImage Model Zoo). Hence, nonexperts can easily perform common image processing tasks in life-science research with deep learning-based tools including pixel and object classification, instance segmentation, denoising or virtual staining. DeepImageJ is compatible with existing state of the art solutions and it is equipped with utility tools for developers to include new models. Very recently, several training frameworks have adopted the deepImageJ format to deploy their work in one of the most used softwares in the field (ImageJ). Beyond its direct use, we expect deepImageJ to contribute to the broader dissemination and reuse of deep learning models in life sciences applications and bioimage informatics. DeepImageJ offers a user-friendly solution in ImageJ to run trained deep learning models for biomedical image analysis. It includes guiding tools for reliable analyses, contributing to the democratization of deep learning in microscopy.},
  archive      = {J_NMETH},
  author       = {Gómez-de-Mariscal, Estibaliz and García-López-de-Haro, Carlos and Ouyang, Wei and Donati, Laurène and Lundberg, Emma and Unser, Michael and Muñoz-Barrutia, Arrate and Sage, Daniel},
  doi          = {10.1038/s41592-021-01262-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1192-1195},
  shortjournal = {Nat. Meth.},
  title        = {DeepImageJ: A user-friendly environment to run deep learning models in ImageJ},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Systematic investigation of cytokine signaling activity at
the tissue and single-cell levels. <em>NMETH</em>, <em>18</em>(10),
1181–1191. (<a
href="https://doi.org/10.1038/s41592-021-01274-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytokines are critical for intercellular communication in human health and disease, but the investigation of cytokine signaling activity has remained challenging due to the short half-lives of cytokines and the complexity/redundancy of cytokine functions. To address these challenges, we developed the Cytokine Signaling Analyzer (CytoSig; https://cytosig.ccr.cancer.gov/ ), providing both a database of target genes modulated by cytokines and a predictive model of cytokine signaling cascades from transcriptomic profiles. We collected 20,591 transcriptome profiles for human cytokine, chemokine and growth factor responses. This atlas of transcriptional patterns induced by cytokines enabled the reliable prediction of signaling activities in distinct cell populations in infectious diseases, chronic inflammation and cancer using bulk and single-cell transcriptomic data. CytoSig revealed previously unidentified roles of many cytokines, such as BMP6 as an anti-inflammatory factor, and identified candidate therapeutic targets in human inflammatory diseases, such as CXCL8 for severe coronavirus disease 2019. CytoSig, an interactive database and model to predict cytokine signaling activity at bulk and single-cell levels.},
  archive      = {J_NMETH},
  author       = {Jiang, Peng and Zhang, Yu and Ru, Beibei and Yang, Yuan and Vu, Trang and Paul, Rohit and Mirza, Amer and Altan-Bonnet, Grégoire and Liu, Lingrui and Ruppin, Eytan and Wakefield, Lalage and Wucherpfennig, Kai W.},
  doi          = {10.1038/s41592-021-01274-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1181-1191},
  shortjournal = {Nat. Meth.},
  title        = {Systematic investigation of cytokine signaling activity at the tissue and single-cell levels},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Differentiable biology: Using deep learning for
biophysics-based and data-driven modeling of molecular mechanisms.
<em>NMETH</em>, <em>18</em>(10), 1169–1180. (<a
href="https://doi.org/10.1038/s41592-021-01283-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning using neural networks relies on a class of machine-learnable models constructed using ‘differentiable programs’. These programs can combine mathematical equations specific to a particular domain of natural science with general-purpose, machine-learnable components trained on experimental data. Such programs are having a growing impact on molecular and cellular biology. In this Perspective, we describe an emerging ‘differentiable biology’ in which phenomena ranging from the small and specific (for example, one experimental assay) to the broad and complex (for example, protein folding) can be modeled effectively and efficiently, often by exploiting knowledge about basic natural phenomena to overcome the limitations of sparse, incomplete and noisy data. By distilling differentiable biology into a small set of conceptual primitives and illustrative vignettes, we show how it can help to address long-standing challenges in integrating multimodal data from diverse experiments across biological scales. This promises to benefit fields as diverse as biophysics and functional genomics. This Perspective describes advances in computer science that enable the integration of deep learning with traditional knowledge-based modeling in biological sciences, and discusses how such integration might overcome the challenges of modeling sparse, incomplete and noisy experimental data.},
  archive      = {J_NMETH},
  author       = {AlQuraishi, Mohammed and Sorger, Peter K.},
  doi          = {10.1038/s41592-021-01283-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1169-1180},
  shortjournal = {Nat. Meth.},
  title        = {Differentiable biology: Using deep learning for biophysics-based and data-driven modeling of molecular mechanisms},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reproducible, scalable, and shareable analysis pipelines
with bioinformatics workflow managers. <em>NMETH</em>, <em>18</em>(10),
1161–1168. (<a
href="https://doi.org/10.1038/s41592-021-01254-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid growth of high-throughput technologies has transformed biomedical research. With the increasing amount and complexity of data, scalability and reproducibility have become essential not just for experiments, but also for computational analysis. However, transforming data into information involves running a large number of tools, optimizing parameters, and integrating dynamically changing reference data. Workflow managers were developed in response to such challenges. They simplify pipeline development, optimize resource usage, handle software installation and versions, and run on different compute platforms, enabling workflow portability and sharing. In this Perspective, we highlight key features of workflow managers, compare commonly used approaches for bioinformatics workflows, and provide a guide for computational and noncomputational users. We outline community-curated pipeline initiatives that enable novice and experienced users to perform complex, best-practice analyses without having to manually assemble workflows. In sum, we illustrate how workflow managers contribute to making computational analysis in biomedical research shareable, scalable, and reproducible. This Perspective highlights workflow managers, which are useful for developing and managing complex bioinformatics pipelines.},
  archive      = {J_NMETH},
  author       = {Wratten, Laura and Wilm, Andreas and Göke, Jonathan},
  doi          = {10.1038/s41592-021-01254-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1161-1168},
  shortjournal = {Nat. Meth.},
  title        = {Reproducible, scalable, and shareable analysis pipelines with bioinformatics workflow managers},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Weighing single protein complexes on the go. <em>NMETH</em>,
<em>18</em>(10), 1159–1160. (<a
href="https://doi.org/10.1038/s41592-021-01263-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamic mass photometry, a method based on optical imaging of unlabeled proteins, enables direct observation and tracking of single-protein interactions on lipid membranes.},
  archive      = {J_NMETH},
  author       = {Vala, Milan and Piliarik, Marek},
  doi          = {10.1038/s41592-021-01263-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1159-1160},
  shortjournal = {Nat. Meth.},
  title        = {Weighing single protein complexes on the go},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mass spectrometry comes of age for subcellular organelles.
<em>NMETH</em>, <em>18</em>(10), 1157–1158. (<a
href="https://doi.org/10.1038/s41592-021-01287-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technological innovations in optical object recognition and high-throughput ultrasensitive mass spectrometry are enabling subcellular metabolomics and peptidomics, providing unprecedented opportunities to study small-molecule mediators of cellular function with important implications in health and disease.},
  archive      = {J_NMETH},
  author       = {Nemes, Peter},
  doi          = {10.1038/s41592-021-01287-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1157-1158},
  shortjournal = {Nat. Meth.},
  title        = {Mass spectrometry comes of age for subcellular organelles},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A wider field of view to predict expression. <em>NMETH</em>,
<em>18</em>(10), 1155–1156. (<a
href="https://doi.org/10.1038/s41592-021-01259-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A gene sequence-to-expression machine learning model achieves improved accuracy by incorporating information about potential long-range interactions.},
  archive      = {J_NMETH},
  author       = {Lu, Yang Young and Noble, William Stafford},
  doi          = {10.1038/s41592-021-01259-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1155-1156},
  shortjournal = {Nat. Meth.},
  title        = {A wider field of view to predict expression},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021n). Neuroscientists go wireless. <em>NMETH</em>,
<em>18</em>(10), 1150–1154. (<a
href="https://doi.org/10.1038/s41592-021-01281-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To study the neural complexities of animals moving though physical and social space, labs increasingly turn to wireless technology.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01281-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1150-1154},
  shortjournal = {Nat. Meth.},
  title        = {Neuroscientists go wireless},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Nanopore-detectable reporter proteins. <em>NMETH</em>,
<em>18</em>(10), 1149. (<a
href="https://doi.org/10.1038/s41592-021-01297-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {NanoporeTERs, engineered reporter proteins, can be detected on MinION nanopore sensor arrays.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01297-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1149},
  shortjournal = {Nat. Meth.},
  title        = {Nanopore-detectable reporter proteins},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Reprogrammed anti-tumor NK cells. <em>NMETH</em>,
<em>18</em>(10), 1148. (<a
href="https://doi.org/10.1038/s41592-021-01296-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reprogrammed natural killer cells show enhanced functional properties and anti-tumor efficacy.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01296-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1148},
  shortjournal = {Nat. Meth.},
  title        = {Reprogrammed anti-tumor NK cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Spike inference with CASCADE. <em>NMETH</em>,
<em>18</em>(10), 1147. (<a
href="https://doi.org/10.1038/s41592-021-01295-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A deep-learning-based tool and a large ground truth dataset enable spike inference from calcium imaging data acquired in a variety of experimental conditions.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01295-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1147},
  shortjournal = {Nat. Meth.},
  title        = {Spike inference with CASCADE},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Mapping genome structures in single cells. <em>NMETH</em>,
<em>18</em>(10), 1146. (<a
href="https://doi.org/10.1038/s41592-021-01294-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers develop single-cell SPRITE to detect higher-order 3D genome structures in single cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01294-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1146},
  shortjournal = {Nat. Meth.},
  title        = {Mapping genome structures in single cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). A multimodal whole-body atlas. <em>NMETH</em>,
<em>18</em>(10), 1145. (<a
href="https://doi.org/10.1038/s41592-021-01293-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers have registered a gene expression atlas to a whole-body EM volume of a marine bristle worm.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01293-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1145},
  shortjournal = {Nat. Meth.},
  title        = {A multimodal whole-body atlas},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Avoiding a replication crisis in deep-learning-based
bioimage analysis. <em>NMETH</em>, <em>18</em>(10), 1136–1144. (<a
href="https://doi.org/10.1038/s41592-021-01284-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning algorithms are powerful tools for analyzing, restoring and transforming bioimaging data. One promise of deep learning is parameter-free one-click image analysis with expert-level performance in a fraction of the time previously required. However, as with most emerging technologies, the potential for inappropriate use is raising concerns among the research community. In this Comment, we discuss key concepts that we believe are important for researchers to consider when using deep learning for their microscopy studies. We describe how results obtained using deep learning can be validated and propose what should, in our view, be considered when choosing a suitable tool. We also suggest what aspects of a deep learning analysis should be reported in publications to ensure reproducibility. We hope this perspective will foster further discussion among developers, image analysis specialists, users and journal editors to define adequate guidelines and ensure the appropriate use of this transformative technology.},
  archive      = {J_NMETH},
  author       = {Laine, Romain F. and Arganda-Carreras, Ignacio and Henriques, Ricardo and Jacquemet, Guillaume},
  doi          = {10.1038/s41592-021-01284-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1136-1144},
  shortjournal = {Nat. Meth.},
  title        = {Avoiding a replication crisis in deep-learning-based bioimage analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reproducibility standards for machine learning in the life
sciences. <em>NMETH</em>, <em>18</em>(10), 1132–1135. (<a
href="https://doi.org/10.1038/s41592-021-01256-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To make machine-learning analyses in the life sciences more computationally reproducible, we propose standards based on data, model and code publication, programming best practices and workflow automation. By meeting these standards, the community of researchers applying machine-learning methods in the life sciences can ensure that their analyses are worthy of trust.},
  archive      = {J_NMETH},
  author       = {Heil, Benjamin J. and Hoffman, Michael M. and Markowetz, Florian and Lee, Su-In and Greene, Casey S. and Hicks, Stephanie C.},
  doi          = {10.1038/s41592-021-01256-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1132-1135},
  shortjournal = {Nat. Meth.},
  title        = {Reproducibility standards for machine learning in the life sciences},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The AIMe registry for artificial intelligence in biomedical
research. <em>NMETH</em>, <em>18</em>(10), 1128–1131. (<a
href="https://doi.org/10.1038/s41592-021-01241-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present the AIMe registry, a community-driven reporting platform for AI in biomedicine. It aims to enhance the accessibility, reproducibility and usability of biomedical AI models, and allows future revisions by the community.},
  archive      = {J_NMETH},
  author       = {Matschinske, Julian and Alcaraz, Nicolas and Benis, Arriel and Golebiewski, Martin and Grimm, Dominik G. and Heumos, Lukas and Kacprowski, Tim and Lazareva, Olga and List, Markus and Louadi, Zakaria and Pauling, Josch K. and Pfeifer, Nico and Röttger, Richard and Schwämmle, Veit and Sturm, Gregor and Traverso, Alberto and Van Steen, Kristel and de Freitas, Martiela Vaz and Villalba Silva, Gerda Cristal and Wee, Leonard and Wenke, Nina K. and Zanin, Massimiliano and Zolotareva, Olga and Baumbach, Jan and Blumenthal, David B.},
  doi          = {10.1038/s41592-021-01241-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1128-1131},
  shortjournal = {Nat. Meth.},
  title        = {The AIMe registry for artificial intelligence in biomedical research},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). DOME: Recommendations for supervised machine learning
validation in biology. <em>NMETH</em>, <em>18</em>(10), 1122–1127. (<a
href="https://doi.org/10.1038/s41592-021-01205-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DOME is a set of community-wide recommendations for reporting supervised machine learning–based analyses applied to biological studies. Broad adoption of these recommendations will help improve machine learning assessment and reproducibility.},
  archive      = {J_NMETH},
  author       = {Walsh, Ian and Fishman, Dmytro and Garcia-Gasulla, Dario and Titma, Tiina and Pollastri, Gianluca and Harrow, Jennifer and Psomopoulos, Fotis E. and Tosatto, Silvio C. E.},
  doi          = {10.1038/s41592-021-01205-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1122-1127},
  shortjournal = {Nat. Meth.},
  title        = {DOME: Recommendations for supervised machine learning validation in biology},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Keeping checks on machine learning. <em>NMETH</em>,
<em>18</em>(10), 1119. (<a
href="https://doi.org/10.1038/s41592-021-01300-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Community-driven initiatives are proposing standards to improve the reporting and reproducibility of machine learning in biology. We support these developments, some of which are described in this month’s special issue.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01300-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1119},
  shortjournal = {Nat. Meth.},
  title        = {Keeping checks on machine learning},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Jonathan sweedler. <em>NMETH</em>, <em>18</em>(10), 1121.
(<a href="https://doi.org/10.1038/s41592-021-01292-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scuba diving and jazz bring together chemistry, neuroscience and life inland.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01292-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {10},
  pages        = {1121},
  shortjournal = {Nat. Meth.},
  title        = {Jonathan sweedler},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chronically implantable LED arrays for behavioral
optogenetics in primates. <em>NMETH</em>, <em>18</em>(9), 1112–1116. (<a
href="https://doi.org/10.1038/s41592-021-01238-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optogenetic methods have been widely used in rodent brains, but remain relatively under-developed for nonhuman primates such as rhesus macaques, an animal model with a large brain expressing sophisticated sensory, motor and cognitive behaviors. To address challenges in behavioral optogenetics in large brains, we developed Opto-Array, a chronically implantable array of light-emitting diodes for high-throughput optogenetic perturbation. We demonstrated that optogenetic silencing in the macaque primary visual cortex with the help of the Opto-Array results in reliable retinotopic visual deficits in a luminance discrimination task. We separately confirmed that Opto-Array illumination results in local neural silencing, and that behavioral effects are not due to tissue heating. These results demonstrate the effectiveness of the Opto-Array for behavioral optogenetic applications in large brains. An implantable array of LEDs called the Opto-Array facilitates optogenetic manipulations in the nonhuman primate brain.},
  archive      = {J_NMETH},
  author       = {Rajalingham, Rishi and Sorenson, Michael and Azadi, Reza and Bohn, Simon and DiCarlo, James J. and Afraz, Arash},
  doi          = {10.1038/s41592-021-01238-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1112-1116},
  shortjournal = {Nat. Meth.},
  title        = {Chronically implantable LED arrays for behavioral optogenetics in primates},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). High-speed, cortex-wide volumetric recording of
neuroactivity at cellular resolution using light beads microscopy.
<em>NMETH</em>, <em>18</em>(9), 1103–1111. (<a
href="https://doi.org/10.1038/s41592-021-01239-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two-photon microscopy has enabled high-resolution imaging of neuroactivity at depth within scattering brain tissue. However, its various realizations have not overcome the tradeoffs between speed and spatiotemporal sampling that would be necessary to enable mesoscale volumetric recording of neuroactivity at cellular resolution and speed compatible with resolving calcium transients. Here, we introduce light beads microscopy (LBM), a scalable and spatiotemporally optimal acquisition approach limited only by fluorescence lifetime, where a set of axially separated and temporally distinct foci record the entire axial imaging range near-simultaneously, enabling volumetric recording at 1.41 × 108 voxels per second. Using LBM, we demonstrate mesoscopic and volumetric imaging at multiple scales in the mouse cortex, including cellular-resolution recordings within ~3 × 5 × 0.5 mm volumes containing &amp;gt;200,000 neurons at ~5 Hz and recordings of populations of ~1 million neurons within ~5.4 × 6 × 0.5 mm volumes at ~2 Hz, as well as higher speed (9.6 Hz) subcellular-resolution volumetric recordings. LBM provides an opportunity for discovering the neurocomputations underlying cortex-wide encoding and processing of information in the mammalian brain. Light beads microscopy is a two-photon microscopy approach that allows high-speed volumetric imaging of neuronal activity at the mesoscale.},
  archive      = {J_NMETH},
  author       = {Demas, Jeffrey and Manley, Jason and Tejera, Frank and Barber, Kevin and Kim, Hyewon and Traub, Francisca Martínez and Chen, Brandon and Vaziri, Alipasha},
  doi          = {10.1038/s41592-021-01239-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1103-1111},
  shortjournal = {Nat. Meth.},
  title        = {High-speed, cortex-wide volumetric recording of neuroactivity at cellular resolution using light beads microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Automated segmentation and tracking of mitochondria in
live-cell time-lapse images. <em>NMETH</em>, <em>18</em>(9), 1091–1102.
(<a href="https://doi.org/10.1038/s41592-021-01234-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mitochondria display complex morphology and movements, which complicates their segmentation and tracking in time-lapse images. Here, we introduce Mitometer, an algorithm for fast, unbiased, and automated segmentation and tracking of mitochondria in live-cell two-dimensional and three-dimensional time-lapse images. Mitometer requires only the pixel size and the time between frames to identify mitochondrial motion and morphology, including fusion and fission events. The segmentation algorithm isolates individual mitochondria via a shape- and size-preserving background removal process. The tracking algorithm links mitochondria via differences in morphological features and displacement, followed by a gap-closing scheme. Using Mitometer, we show that mitochondria of triple-negative breast cancer cells are faster, more directional, and more elongated than those in their receptor-positive counterparts. Furthermore, we show that mitochondrial motility and morphology in breast cancer, but not in normal breast epithelia, correlate with metabolic activity. Mitometer is an unbiased and user-friendly tool that will help resolve fundamental questions regarding mitochondrial form and function. Mitometer enables efficient, rapid, and accurate automated segmentation and tracking of mitochondria from time-lapse images. Mitometer performs well on diverse input images and can be used to monitor dynamic fission and fusion events.},
  archive      = {J_NMETH},
  author       = {Lefebvre, Austin E. Y. T. and Ma, Dennis and Kessenbrock, Kai and Lawson, Devon A. and Digman, Michelle A.},
  doi          = {10.1038/s41592-021-01234-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1091-1102},
  shortjournal = {Nat. Meth.},
  title        = {Automated segmentation and tracking of mitochondria in live-cell time-lapse images},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Deep learning enables fast and dense single-molecule
localization with high accuracy. <em>NMETH</em>, <em>18</em>(9),
1082–1090. (<a
href="https://doi.org/10.1038/s41592-021-01236-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy (SMLM) has had remarkable success in imaging cellular structures with nanometer resolution, but standard analysis algorithms require sparse emitters, which limits imaging speed and labeling density. Here, we overcome this major limitation using deep learning. We developed DECODE (deep context dependent), a computational tool that can localize single emitters at high density in three dimensions with highest accuracy for a large range of imaging modalities and conditions. In a public software benchmark competition, it outperformed all other fitters on 12 out of 12 datasets when comparing both detection accuracy and localization error, often by a substantial margin. DECODE allowed us to acquire fast dynamic live-cell SMLM data with reduced light exposure and to image microtubules at ultra-high labeling density. Packaged for simple installation and use, DECODE will enable many laboratories to reduce imaging times and increase localization density in SMLM. DECODE uses deep learning for localizing single emitters in high-density two-dimensional and three-dimensional single-molecule localization microscopy data. DECODE outperforms available methods and enables fast live-cell SMLM of dynamic processes.},
  archive      = {J_NMETH},
  author       = {Speiser, Artur and Müller, Lucas-Raphael and Hoess, Philipp and Matti, Ulf and Obara, Christopher J. and Legant, Wesley R. and Kreshuk, Anna and Macke, Jakob H. and Ries, Jonas and Turaga, Srinivas C.},
  doi          = {10.1038/s41592-021-01236-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1082-1090},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning enables fast and dense single-molecule localization with high accuracy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Augmenting and directing long-range CRISPR-mediated
activation in human cells. <em>NMETH</em>, <em>18</em>(9), 1075–1081.
(<a href="https://doi.org/10.1038/s41592-021-01224-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetic editing is an emerging technology that uses artificial transcription factors (aTFs) to regulate expression of a target gene. Although human genes can be robustly upregulated by targeting aTFs to promoters, the activation induced by directing aTFs to distal transcriptional enhancers is substantially less robust and consistent. Here we show that long-range activation using CRISPR-based aTFs in human cells can be made more efficient and reliable by concurrently targeting an aTF to the target gene promoter. We used this strategy to direct target gene choice for enhancers capable of regulating more than one promoter and to achieve allele-selective activation of human genes by targeting aTFs to single-nucleotide polymorphisms embedded in distally located sequences. Our results broaden the potential applications of the epigenetic editing toolbox for research and therapeutics. Long-range CRISPR activation can be enhanced by concurrent recruitment of artificial TFs to the enhancer and promoter of a target gene. This CRISPR activation system can be employed to achieve allele-selective gene upregulation by differentially targeting single-nucleotide polymorphisms embedded in enhancers or other distally located sequences.},
  archive      = {J_NMETH},
  author       = {Tak, Y. Esther and Horng, Joy E. and Perry, Nicholas T. and Schultz, Hayley T. and Iyer, Sowmya and Yao, Qiuming and Zou, Luli S. and Aryee, Martin J. and Pinello, Luca and Joung, J. Keith},
  doi          = {10.1038/s41592-021-01224-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1075-1081},
  shortjournal = {Nat. Meth.},
  title        = {Augmenting and directing long-range CRISPR-mediated activation in human cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Nascent ribo-seq measures ribosomal loading time and reveals
kinetic impact on ribosome density. <em>NMETH</em>, <em>18</em>(9),
1068–1074. (<a
href="https://doi.org/10.1038/s41592-021-01250-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In general, mRNAs are assumed to be loaded with ribosomes instantly upon entry into the cytoplasm. To measure ribosome density (RD) on nascent mRNA, we developed nascent Ribo-Seq by combining Ribo-Seq with progressive 4-thiouridine labeling. In mouse macrophages, we determined experimentally the lag between the appearance of nascent mRNA and its association with ribosomes, which was calculated to be 20–22 min for bulk mRNA. In mouse embryonic stem cells, nRibo-Seq revealed an even stronger lag of 35–38 min in ribosome loading. After stimulation of macrophages with lipopolysaccharide, the lag between cytoplasmic and translated mRNA leads to uncoupling between input and ribosome-protected fragments, which gives rise to distorted RD measurements under conditions where mRNA amounts are far from steady-state expression. As a result, we demonstrate that transcriptional changes affect RD in a passive way. This work describes nascent Ribo-Seq, which measures the speed of ribosome loading and polysome assembly on endogenous mRNA through a combination of 4-thiouridine labeling and ribosome footprint sequencing.},
  archive      = {J_NMETH},
  author       = {Schott, Johanna and Reitter, Sonja and Lindner, Doris and Grosser, Jan and Bruer, Marius and Shenoy, Anjana and Geiger, Tamar and Mathes, Arthur and Dobreva, Gergana and Stoecklin, Georg},
  doi          = {10.1038/s41592-021-01250-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1068-1074},
  shortjournal = {Nat. Meth.},
  title        = {Nascent ribo-seq measures ribosomal loading time and reveals kinetic impact on ribosome density},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Multiplexed profiling facilitates robust m6A quantification
at site, gene and sample resolution. <em>NMETH</em>, <em>18</em>(9),
1060–1067. (<a
href="https://doi.org/10.1038/s41592-021-01242-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {N6-methyladenosine (m6A) is the most prevalent modification of messenger RNA in mammals. To interrogate its functions and dynamics, there is a critical need to quantify m6A at three levels: site, gene and sample. Current approaches address these needs in a limited manner. Here we develop m6A-seq2, relying on multiplexed m6A-immunoprecipitation of barcoded and pooled samples. m6A-seq2 allows a big increase in throughput while reducing technical variability, requirements of input material and cost. m6A-seq2 is furthermore uniquely capable of providing sample-level relative quantitations of m6A, serving as an orthogonal alternative to mass spectrometry-based approaches. Finally, we develop a computational approach for gene-level quantitation of m6A. We demonstrate that using this metric, roughly 30\% of the variability in RNA half life in mouse embryonic stem cells can be explained, establishing m6A as a main driver of RNA stability. m6A-seq2 thus provides an experimental and analytic framework for dissecting m6A-mediated regulation at three different levels. This work describes m6A-seq2, which utilizes multiplexed N6-methyladenosine (m6A) immunoprecipitation of pre-barcoded and pooled samples to reduce the technical variability and allows direct comparison of m6A at the site, gene and sample level.},
  archive      = {J_NMETH},
  author       = {Dierks, David and Garcia-Campos, Miguel Angel and Uzonyi, Anna and Safra, Modi and Edelheit, Sarit and Rossi, Alice and Sideri, Theodora and Varier, Radhika A. and Brandis, Alexander and Stelzer, Yonatan and van Werven, Folkert and Scherz-Shouval, Ruth and Schwartz, Schraga},
  doi          = {10.1038/s41592-021-01242-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1060-1067},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed profiling facilitates robust m6A quantification at site, gene and sample resolution},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SnapHiC: A computational pipeline to identify chromatin
loops from single-cell hi-c data. <em>NMETH</em>, <em>18</em>(9),
1056–1059. (<a
href="https://doi.org/10.1038/s41592-021-01231-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell Hi-C (scHi-C) analysis has been increasingly used to map chromatin architecture in diverse tissue contexts, but computational tools to define chromatin loops at high resolution from scHi-C data are still lacking. Here, we describe Single-Nucleus Analysis Pipeline for Hi-C (SnapHiC), a method that can identify chromatin loops at high resolution and accuracy from scHi-C data. Using scHi-C data from 742 mouse embryonic stem cells, we benchmark SnapHiC against a number of computational tools developed for mapping chromatin loops and interactions from bulk Hi-C. We further demonstrate its use by analyzing single-nucleus methyl-3C-seq data from 2,869 human prefrontal cortical cells, which uncovers cell type-specific chromatin loops and predicts putative target genes for noncoding sequence variants associated with neuropsychiatric disorders. Our results indicate that SnapHiC could facilitate the analysis of cell type-specific chromatin architecture and gene regulatory programs in complex tissues. SnapHiC offers a computational tool for improving detection of chromatin loops from single-cell Hi-C data.},
  archive      = {J_NMETH},
  author       = {Yu, Miao and Abnousi, Armen and Zhang, Yanxiao and Li, Guoqiang and Lee, Lindsay and Chen, Ziyin and Fang, Rongxin and Lagler, Taylor M. and Yang, Yuchen and Wen, Jia and Sun, Quan and Li, Yun and Ren, Bing and Hu, Ming},
  doi          = {10.1038/s41592-021-01231-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1056-1059},
  shortjournal = {Nat. Meth.},
  title        = {SnapHiC: A computational pipeline to identify chromatin loops from single-cell hi-C data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Systematic evaluation of chromosome conformation capture
assays. <em>NMETH</em>, <em>18</em>(9), 1046–1055. (<a
href="https://doi.org/10.1038/s41592-021-01248-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosome conformation capture (3C) assays are used to map chromatin interactions genome-wide. Chromatin interaction maps provide insights into the spatial organization of chromosomes and the mechanisms by which they fold. Hi-C and Micro-C are widely used 3C protocols that differ in key experimental parameters including cross-linking chemistry and chromatin fragmentation strategy. To understand how the choice of experimental protocol determines the ability to detect and quantify aspects of chromosome folding we have performed a systematic evaluation of 3C experimental parameters. We identified optimal protocol variants for either loop or compartment detection, optimizing fragment size and cross-linking chemistry. We used this knowledge to develop a greatly improved Hi-C protocol (Hi-C 3.0) that can detect both loops and compartments relatively effectively. In addition to providing benchmarked protocols, this work produced ultra-deep chromatin interaction maps using Micro-C, conventional Hi-C and Hi-C 3.0 for key cell lines used by the 4D Nucleome project. This analysis systematically evaluates cross-linking chemistry and chromatin fragmentation strategies commonly used in 3C assays and introduces an improved Hi-C protocol for detecting loops and compartments.},
  archive      = {J_NMETH},
  author       = {Akgol Oksuz, Betul and Yang, Liyan and Abraham, Sameer and Venev, Sergey V. and Krietenstein, Nils and Parsi, Krishna Mohan and Ozadam, Hakan and Oomen, Marlies E. and Nand, Ankita and Mao, Hui and Genga, Ryan M. J. and Maehr, Rene and Rando, Oliver J. and Mirny, Leonid A. and Gibcus, Johan H. and Dekker, Job},
  doi          = {10.1038/s41592-021-01248-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1046-1055},
  shortjournal = {Nat. Meth.},
  title        = {Systematic evaluation of chromosome conformation capture assays},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). LIVECell—a large-scale dataset for label-free live cell
segmentation. <em>NMETH</em>, <em>18</em>(9), 1038–1045. (<a
href="https://doi.org/10.1038/s41592-021-01249-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light microscopy combined with well-established protocols of two-dimensional cell culture facilitates high-throughput quantitative imaging to study biological phenomena. Accurate segmentation of individual cells in images enables exploration of complex biological questions, but can require sophisticated imaging processing pipelines in cases of low contrast and high object density. Deep learning-based methods are considered state-of-the-art for image segmentation but typically require vast amounts of annotated data, for which there is no suitable resource available in the field of label-free cellular imaging. Here, we present LIVECell, a large, high-quality, manually annotated and expert-validated dataset of phase-contrast images, consisting of over 1.6 million cells from a diverse set of cell morphologies and culture densities. To further demonstrate its use, we train convolutional neural network-based models using LIVECell and evaluate model segmentation accuracy with a proposed a suite of benchmarks. The LIVECell dataset comprises annotated phase-contrast images of over 1.6 million cells from different cell lines during growth from sparse seeding to confluence for improved training of deep learning-based models of image segmentation.},
  archive      = {J_NMETH},
  author       = {Edlund, Christoffer and Jackson, Timothy R. and Khalid, Nabeel and Bevan, Nicola and Dale, Timothy and Dengel, Andreas and Ahmed, Sheraz and Trygg, Johan and Sjögren, Rickard},
  doi          = {10.1038/s41592-021-01249-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1038-1045},
  shortjournal = {Nat. Meth.},
  title        = {LIVECell—A large-scale dataset for label-free live cell segmentation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A guide to the optogenetic regulation of endogenous
molecules. <em>NMETH</em>, <em>18</em>(9), 1027–1037. (<a
href="https://doi.org/10.1038/s41592-021-01240-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded tools for the regulation of endogenous molecules (RNA, DNA elements and protein) are needed to study and control biological processes with minimal interference caused by protein overexpression and overactivation of signaling pathways. Here we focus on light-controlled optogenetic tools (OTs) that allow spatiotemporally precise regulation of gene expression and protein function. To control endogenous molecules, OTs combine light-sensing modules from natural photoreceptors with specific protein or nucleic acid binders. We discuss OT designs and group OTs according to the principles of their regulation. We outline characteristics of OT performance, discuss considerations for their use in vivo and review available OTs and their applications in cells and in vivo. Finally, we provide a brief outlook on the development of OTs. This Review discusses optogenetic tools for manipulating endogenous targets such as genes and signaling pathways in a physiological range.},
  archive      = {J_NMETH},
  author       = {Manoilov, Kyrylo Yu. and Verkhusha, Vladislav V. and Shcherbakova, Daria M.},
  doi          = {10.1038/s41592-021-01240-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1027-1037},
  shortjournal = {Nat. Meth.},
  title        = {A guide to the optogenetic regulation of endogenous molecules},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The power of imaging to understand extracellular vesicle
biology in vivo. <em>NMETH</em>, <em>18</em>(9), 1013–1026. (<a
href="https://doi.org/10.1038/s41592-021-01206-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extracellular vesicles (EVs) are nano-sized lipid bilayer vesicles released by virtually every cell type. EVs have diverse biological activities, ranging from roles in development and homeostasis to cancer progression, which has spurred the development of EVs as disease biomarkers and drug nanovehicles. Owing to the small size of EVs, however, most studies have relied on isolation and biochemical analysis of bulk EVs separated from biofluids. Although informative, these approaches do not capture the dynamics of EV release, biodistribution, and other contributions to pathophysiology. Recent advances in live and high-resolution microscopy techniques, combined with innovative EV labeling strategies and reporter systems, provide new tools to study EVs in vivo in their physiological environment and at the single-vesicle level. Here we critically review the latest advances and challenges in EV imaging, and identify urgent, outstanding questions in our quest to unravel EV biology and therapeutic applications. This Review describes the state of the art in imaging extracellular vesicles in animals to study their release, biodistribution and uptake, and covers labeling strategies, microscopy methods and discoveries made in model organisms.},
  archive      = {J_NMETH},
  author       = {Verweij, Frederik J. and Balaj, Leonora and Boulanger, Chantal M. and Carter, David R. F. and Compeer, Ewoud B. and D’Angelo, Gisela and El Andaloussi, Samir and Goetz, Jacky G. and Gross, Julia Christina and Hyenne, Vincent and Krämer-Albers, Eva-Maria and Lai, Charles P. and Loyer, Xavier and Marki, Alex and Momma, Stefan and Nolte-‘t Hoen, Esther N. M. and Pegtel, D. Michiel and Peinado, Hector and Raposo, Graça and Rilla, Kirsi and Tahara, Hidetoshi and Théry, Clotilde and van Royen, Martin E. and Vandenbroucke, Roosmarijn E. and Wehman, Ann M. and Witwer, Kenneth and Wu, Zhiwei and Wubbolts, Richard and van Niel, Guillaume},
  doi          = {10.1038/s41592-021-01206-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1013-1026},
  shortjournal = {Nat. Meth.},
  title        = {The power of imaging to understand extracellular vesicle biology in vivo},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Spatial omics and multiplexed imaging to explore cancer
biology. <em>NMETH</em>, <em>18</em>(9), 997–1012. (<a
href="https://doi.org/10.1038/s41592-021-01203-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding intratumoral heterogeneity—the molecular variation among cells within a tumor—promises to address outstanding questions in cancer biology and improve the diagnosis and treatment of specific cancer subtypes. Single-cell analyses, especially RNA sequencing and other genomics modalities, have been transformative in revealing novel biomarkers and molecular regulators associated with tumor growth, metastasis and drug resistance. However, these approaches fail to provide a complete picture of tumor biology, as information on cellular location within the tumor microenvironment is lost. New technologies leveraging multiplexed fluorescence, DNA, RNA and isotope labeling enable the detection of tens to thousands of cancer subclones or molecular biomarkers within their native spatial context. The expeditious growth in these techniques, along with methods for multiomics data integration, promises to yield a more comprehensive understanding of cell-to-cell variation within and between individual tumors. Here we provide the current state and future perspectives on the spatial technologies expected to drive the next generation of research and diagnostic and therapeutic strategies for cancer. This Review describes spatial omics and multiplexed imaging technologies and their current and future impact in studying tumor heterogeneity and cancer biology.},
  archive      = {J_NMETH},
  author       = {Lewis, Sabrina M. and Asselin-Labat, Marie-Liesse and Nguyen, Quan and Berthelet, Jean and Tan, Xiao and Wimmer, Verena C. and Merino, Delphine and Rogers, Kelly L. and Naik, Shalin H.},
  doi          = {10.1038/s41592-021-01203-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {997-1012},
  shortjournal = {Nat. Meth.},
  title        = {Spatial omics and multiplexed imaging to explore cancer biology},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021v). Tools to cut the sweet layer-cake that is glycoproteomics.
<em>NMETH</em>, <em>18</em>(9), 991–995. (<a
href="https://doi.org/10.1038/s41592-021-01253-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteins and sugars interact in sweet, complex ways. More tools are emerging, but working in this field still takes endurance.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01253-w},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {991-995},
  shortjournal = {Nat. Meth.},
  title        = {Tools to cut the sweet layer-cake that is glycoproteomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Engineered protein–protein toggle network. <em>NMETH</em>,
<em>18</em>(9), 990. (<a
href="https://doi.org/10.1038/s41592-021-01269-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sensitive synthetic protein-phosphorylation networks manifest fast bistable dynamics.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01269-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {990},
  shortjournal = {Nat. Meth.},
  title        = {Engineered protein–protein toggle network},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Thermodynamic profiling of single molecules.
<em>NMETH</em>, <em>18</em>(9), 989. (<a
href="https://doi.org/10.1038/s41592-021-01268-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A single-molecule calorimeter measures the thermodynamic properties of individual proteins.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01268-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {989},
  shortjournal = {Nat. Meth.},
  title        = {Thermodynamic profiling of single molecules},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Proteomic analysis with MaxDIA. <em>NMETH</em>,
<em>18</em>(9), 988. (<a
href="https://doi.org/10.1038/s41592-021-01267-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A software platform for analyzing data-independent acquisition (DIA) proteomics data.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01267-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {988},
  shortjournal = {Nat. Meth.},
  title        = {Proteomic analysis with MaxDIA},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Recapitulating oogenesis in vitro. <em>NMETH</em>,
<em>18</em>(9), 987. (<a
href="https://doi.org/10.1038/s41592-021-01266-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A co-culture system to mimic the development of functional mouse oocytes in vitro.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01266-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {987},
  shortjournal = {Nat. Meth.},
  title        = {Recapitulating oogenesis in vitro},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Measuring condensation forces. <em>NMETH</em>,
<em>18</em>(9), 986. (<a
href="https://doi.org/10.1038/s41592-021-01265-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers use an in vitro assay in combination with a polymer model to study protein–DNA condensation and how condensates bring distant DNA elements into close proximity.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01265-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {986},
  shortjournal = {Nat. Meth.},
  title        = {Measuring condensation forces},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). What makes an author. <em>NMETH</em>, <em>18</em>(9), 983.
(<a href="https://doi.org/10.1038/s41592-021-01271-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Constructing a fair and accurate author list can be one of the most fraught aspects of manuscript publication. We provide some advice and resources for authors at all career levels.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01271-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {983},
  shortjournal = {Nat. Meth.},
  title        = {What makes an author},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Michelle digman. <em>NMETH</em>, <em>18</em>(9), 985. (<a
href="https://doi.org/10.1038/s41592-021-01258-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {She connects equity, samurai swords, imaging and an automated way to count and track mitochondria.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01258-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {9},
  pages        = {985},
  shortjournal = {Nat. Meth.},
  title        = {Michelle digman},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). LiftPose3D, a deep learning-based approach for transforming
two-dimensional to three-dimensional poses in laboratory animals.
<em>NMETH</em>, <em>18</em>(8), 975–981. (<a
href="https://doi.org/10.1038/s41592-021-01226-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Markerless three-dimensional (3D) pose estimation has become an indispensable tool for kinematic studies of laboratory animals. Most current methods recover 3D poses by multi-view triangulation of deep network-based two-dimensional (2D) pose estimates. However, triangulation requires multiple synchronized cameras and elaborate calibration protocols that hinder its widespread adoption in laboratory studies. Here we describe LiftPose3D, a deep network-based method that overcomes these barriers by reconstructing 3D poses from a single 2D camera view. We illustrate LiftPose3D’s versatility by applying it to multiple experimental systems using flies, mice, rats and macaques, and in circumstances where 3D triangulation is impractical or impossible. Our framework achieves accurate lifting for stereotypical and nonstereotypical behaviors from different camera angles. Thus, LiftPose3D permits high-quality 3D pose estimation in the absence of complex camera arrays and tedious calibration procedures and despite occluded body parts in freely behaving animals. LiftPose3D infers three-dimensional poses from two-dimensional data or from limited three-dimensional data. The approach is illustrated for videos of behaving Drosophila, mice, rats and macaques.},
  archive      = {J_NMETH},
  author       = {Gosztolai, Adam and Günel, Semih and Lobato-Ríos, Victor and Pietro Abrate, Marco and Morales, Daniel and Rhodin, Helge and Fua, Pascal and Ramdya, Pavan},
  doi          = {10.1038/s41592-021-01226-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {975-981},
  shortjournal = {Nat. Meth.},
  title        = {LiftPose3D, a deep learning-based approach for transforming two-dimensional to three-dimensional poses in laboratory animals},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Transgenic mice for in vivo epigenome editing with
CRISPR-based systems. <em>NMETH</em>, <em>18</em>(8), 965–974. (<a
href="https://doi.org/10.1038/s41592-021-01207-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR-Cas9 technologies have dramatically increased the ease of targeting DNA sequences in the genomes of living systems. The fusion of chromatin-modifying domains to nuclease-deactivated Cas9 (dCas9) has enabled targeted epigenome editing in both cultured cells and animal models. However, delivering large dCas9 fusion proteins to target cells and tissues is an obstacle to the widespread adoption of these tools for in vivo studies. Here, we describe the generation and characterization of two conditional transgenic mouse lines for epigenome editing, Rosa26:LSL-dCas9-p300 for gene activation and Rosa26:LSL-dCas9-KRAB for gene repression. By targeting the guide RNAs to transcriptional start sites or distal enhancer elements, we demonstrate regulation of target genes and corresponding changes to epigenetic states and downstream phenotypes in the brain and liver in vivo, and in T cells and fibroblasts ex vivo. These mouse lines are convenient and valuable tools for facile, temporally controlled, and tissue-restricted epigenome editing and manipulation of gene expression in vivo. Two conditional transgenic mouse lines based on CRISPRa and CRISPRi enable epigenome editing in vivo.},
  archive      = {J_NMETH},
  author       = {Gemberling, Matthew P. and Siklenka, Keith and Rodriguez, Erica and Tonn-Eisinger, Katherine R. and Barrera, Alejandro and Liu, Fang and Kantor, Ariel and Li, Liqing and Cigliola, Valentina and Hazlett, Mariah F. and Williams, Courtney A. and Bartelt, Luke C. and Madigan, Victoria J. and Bodle, Josephine C. and Daniels, Heather and Rouse, Douglas C. and Hilton, Isaac B. and Asokan, Aravind and Ciofani, Maria and Poss, Kenneth D. and Reddy, Timothy E. and West, Anne E. and Gersbach, Charles A.},
  doi          = {10.1038/s41592-021-01207-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {965-974},
  shortjournal = {Nat. Meth.},
  title        = {Transgenic mice for in vivo epigenome editing with CRISPR-based systems},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Clear optically matched panoramic access channel technique
(COMPACT) for large-volume deep brain imaging. <em>NMETH</em>,
<em>18</em>(8), 959–964. (<a
href="https://doi.org/10.1038/s41592-021-01230-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand neural circuit mechanisms underlying behavior, it is crucial to observe the dynamics of neuronal structure and function in different regions of the brain. Since current noninvasive imaging technologies allow cellular-resolution imaging of neurons only within ~1 mm below the cortical surface, the majority of mouse brain tissue remains inaccessible. While miniature optical imaging probes allow access to deep brain regions, cellular-resolution imaging is typically restricted to a small tissue volume. To increase the tissue access volume, we developed a clear optically matched panoramic access channel technique (COMPACT). With probe dimensions comparable to those of common gradient-index lenses, COMPACT enables a two to three orders of magnitude greater tissue access volume. We demonstrated the capabilities of COMPACT by multiregional calcium imaging in mice during sleep. We believe that large-volume in vivo imaging with COMPACT will be valuable to a variety of deep tissue imaging applications. COMPACT allows imaging of large volumes in the brain through a periscope-like probe inside a glass capillary.},
  archive      = {J_NMETH},
  author       = {Wei, Bowen and Wang, Chenmao and Cheng, Zongyue and Lai, Baoling and Gan, Wen-Biao and Cui, Meng},
  doi          = {10.1038/s41592-021-01230-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {959-964},
  shortjournal = {Nat. Meth.},
  title        = {Clear optically matched panoramic access channel technique (COMPACT) for large-volume deep brain imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Universal autofocus for quantitative volumetric microscopy
of whole mouse brains. <em>NMETH</em>, <em>18</em>(8), 953–958. (<a
href="https://doi.org/10.1038/s41592-021-01208-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unbiased quantitative analysis of macroscopic biological samples demands fast imaging systems capable of maintaining high resolution across large volumes. Here we introduce RAPID (rapid autofocusing via pupil-split image phase detection), a real-time autofocus method applicable in every widefield-based microscope. RAPID-enabled light-sheet microscopy reliably reconstructs intact, cleared mouse brains with subcellular resolution, and allowed us to characterize the three-dimensional (3D) spatial clustering of somatostatin-positive neurons in the whole encephalon, including densely labeled areas. Furthermore, it enabled 3D morphological analysis of microglia across the entire brain. Beyond light-sheet microscopy, we demonstrate that RAPID maintains high image quality in various settings, from in vivo fluorescence imaging to 3D tracking of fast-moving organisms. RAPID thus provides a flexible autofocus solution that is suitable for traditional automated microscopy tasks as well as for quantitative analysis of large biological specimens. RAPID (rapid autofocusing via pupil-split image phase detection) is a sample-agnostic real-time autofocus method for widefield microscopy. RAPID removes most image degradation in large, cleared samples for enhanced quantitative analyses.},
  archive      = {J_NMETH},
  author       = {Silvestri, L. and Müllenbroich, M. C. and Costantini, I. and Di Giovanna, A. P. and Mazzamuto, G. and Franceschini, A. and Kutra, D. and Kreshuk, A. and Checcucci, C. and Toresano, L. O. and Frasconi, P. and Sacconi, L. and Pavone, F. S.},
  doi          = {10.1038/s41592-021-01208-1},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {953-958},
  shortjournal = {Nat. Meth.},
  title        = {Universal autofocus for quantitative volumetric microscopy of whole mouse brains},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultrasensitive ultrasound imaging of gene expression with
signal unmixing. <em>NMETH</em>, <em>18</em>(8), 945–952. (<a
href="https://doi.org/10.1038/s41592-021-01229-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acoustic reporter genes (ARGs) that encode air-filled gas vesicles enable ultrasound-based imaging of gene expression in genetically modified bacteria and mammalian cells, facilitating the study of cellular function in deep tissues. Despite the promise of this technology for biological research and potential clinical applications, the sensitivity with which ARG-expressing cells can be visualized is currently limited. Here we present burst ultrasound reconstructed with signal templates (BURST)—an ARG imaging paradigm that improves the cellular detection limit by more than 1,000-fold compared to conventional methods. BURST takes advantage of the unique temporal signal pattern produced by gas vesicles as they collapse under acoustic pressure above a threshold defined by the ARG. By extracting the unique pattern of this signal from total scattering, BURST boosts the sensitivity of ultrasound to image ARG-expressing cells, as demonstrated in vitro and in vivo in the mouse gastrointestinal tract and liver. Furthermore, in dilute cell suspensions, BURST imaging enables the detection of gene expression in individual bacteria and mammalian cells. The resulting abilities of BURST expand the potential use of ultrasound for non-invasive imaging of cellular functions. Acoustic reporter genes can be imaged with high sensitivity using the BURST imaging sequence, allowing the detection of single cells under optimal conditions.},
  archive      = {J_NMETH},
  author       = {Sawyer, Daniel P. and Bar-Zion, Avinoam and Farhadi, Arash and Shivaei, Shirin and Ling, Bill and Lee-Gosselin, Audrey and Shapiro, Mikhail G.},
  doi          = {10.1038/s41592-021-01229-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {945-952},
  shortjournal = {Nat. Meth.},
  title        = {Ultrasensitive ultrasound imaging of gene expression with signal unmixing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). PaintSHOP enables the interactive design of transcriptome-
and genome-scale oligonucleotide FISH experiments. <em>NMETH</em>,
<em>18</em>(8), 937–944. (<a
href="https://doi.org/10.1038/s41592-021-01187-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence in situ hybridization (FISH) allows researchers to visualize the spatial position and quantity of nucleic acids in fixed samples. Recently, considerable progress has been made in developing oligonucleotide (oligo)-based FISH methods that have enabled researchers to study the three-dimensional organization of the genome at super-resolution and visualize the spatial patterns of gene expression for thousands of genes in individual cells. However, there are few existing computational tools to support the bioinformatics workflows necessary to carry out these experiments using oligo FISH probes. Here, we introduce paint server and homology optimization pipeline (PaintSHOP), an interactive platform for the design of oligo FISH experiments. PaintSHOP enables researchers to identify probes for their experimental targets efficiently, to incorporate additional necessary sequences such as primer pairs and to easily generate files documenting library design. PaintSHOP democratizes and standardizes the process of designing complex probe sets for the oligo FISH community. Paint server and homology optimization pipeline (PaintSHOP), an interactive platform for the design of oligo fluorescence in situ hybridization (FISH) experiments, democratizes and standardizes the process of designing complex probe sets for the oligo FISH community.},
  archive      = {J_NMETH},
  author       = {Hershberg, Elliot A. and Camplisson, Conor K. and Close, Jennie L. and Attar, Sahar and Chern, Ryan and Liu, Yuzhen and Akilesh, Shreeram and Nicovich, Philip R. and Beliveau, Brian J.},
  doi          = {10.1038/s41592-021-01187-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {937-944},
  shortjournal = {Nat. Meth.},
  title        = {PaintSHOP enables the interactive design of transcriptome- and genome-scale oligonucleotide FISH experiments},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning-based mixed-dimensional gaussian mixture model
for characterizing variability in cryo-EM. <em>NMETH</em>,
<em>18</em>(8), 930–936. (<a
href="https://doi.org/10.1038/s41592-021-01220-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural flexibility and/or dynamic interactions with other molecules is a critical aspect of protein function. Cryogenic electron microscopy (cryo-EM) provides direct visualization of individual macromolecules sampling different conformational and compositional states. While numerous methods are available for computational classification of discrete states, characterization of continuous conformational changes or large numbers of discrete state without human supervision remains challenging. Here we present e2gmm, a machine learning algorithm to determine a conformational landscape for proteins or complexes using a three-dimensional Gaussian mixture model mapped onto two-dimensional particle images in known orientations. Using a deep neural network architecture, e2gmm can automatically resolve the structural heterogeneity within the protein complex and map particles onto a small latent space describing conformational and compositional changes. This system presents a more intuitive and flexible representation than other manifold methods currently in use. We demonstrate this method on both simulated data and three biological systems to explore compositional and conformational changes at a range of scales. The software is distributed as part of EMAN2. e2gmm uses a deep neural network with a Gaussian representation to resolve the compositional and conformational variability within biomolecules using cryo-EM data.},
  archive      = {J_NMETH},
  author       = {Chen, Muyuan and Ludtke, Steven J.},
  doi          = {10.1038/s41592-021-01220-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {930-936},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning-based mixed-dimensional gaussian mixture model for characterizing variability in cryo-EM},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). StrucGP: De novo structural sequencing of site-specific
n-glycan on glycoproteins using a modularization strategy.
<em>NMETH</em>, <em>18</em>(8), 921–929. (<a
href="https://doi.org/10.1038/s41592-021-01209-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precision mapping of glycans at structural and site-specific level is still one of the most challenging tasks in the glycobiology field. Here, we describe a modularization strategy for de novo interpretation of N-glycan structures on intact glycopeptides using tandem mass spectrometry. An algorithm named StrucGP is also developed to automate the interpretation process for large-scale analysis. By dividing an N-glycan into three modules and identifying each module using distinct patterns of Y ions or a combination of distinguishable B/Y ions, the method enables determination of detailed glycan structures on thousands of glycosites in mouse brain, which comprise four types of core structure and 17 branch structures with three glycan subtypes. Owing to the database-independent glycan mapping strategy, StrucGP also facilitates the identification of rare/new glycan structures. The approach will be greatly beneficial for in-depth structural and functional study of glycoproteins in the biomedical research. StrucGP offers a de novo glycan mapping method to determine detailed N-glycan structures at the site-specific level.},
  archive      = {J_NMETH},
  author       = {Shen, Jiechen and Jia, Li and Dang, Liuyi and Su, Yuanjie and Zhang, Jie and Xu, Yintai and Zhu, Bojing and Chen, Zexuan and Wu, Jingyu and Lan, Rongxia and Hao, Zhifang and Ma, Chen and Zhao, Ting and Gao, Ni and Bai, Jieyun and Zhi, Yuan and Li, Jun and Zhang, Junying and Sun, Shisheng},
  doi          = {10.1038/s41592-021-01209-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {921-929},
  shortjournal = {Nat. Meth.},
  title        = {StrucGP: De novo structural sequencing of site-specific N-glycan on glycoproteins using a modularization strategy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An unsupervised method for physical cell interaction
profiling of complex tissues. <em>NMETH</em>, <em>18</em>(8), 912–920.
(<a href="https://doi.org/10.1038/s41592-021-01196-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular identity in complex multicellular organisms is determined in part by the physical organization of cells. However, large-scale investigation of the cellular interactome remains technically challenging. Here we develop cell interaction by multiplet sequencing (CIM-seq), an unsupervised and high-throughput method to analyze direct physical cell–cell interactions between cell types present in a tissue. CIM-seq is based on RNA sequencing of incompletely dissociated cells, followed by computational deconvolution into constituent cell types. CIM-seq estimates parameters such as number of cells and cell types in each multiplet directly from sequencing data, making it compatible with high-throughput droplet-based methods. When applied to gut epithelium or whole dissociated lung and spleen, CIM-seq correctly identifies known interactions, including those between different cell lineages and immune cells. In the colon, CIM-seq identifies a previously unrecognized goblet cell subtype expressing the wound-healing marker Plet1, which is directly adjacent to colonic stem cells. Our results demonstrate that CIM-seq is broadly applicable to unsupervised profiling of cell-type interactions in different tissue types. CIM-seq offers an unsupervised deconvolution method to profile cell–cell interactions by sequencing cell multiplets of a given tissue, and was employed to analyze diverse tissues such as the intestinal epithelium, lung and spleen.},
  archive      = {J_NMETH},
  author       = {Andrews, Nathanael and Serviss, Jason T. and Geyer, Natalie and Andersson, Agneta B. and Dzwonkowska, Ewa and Šutevski, Iva and Heijboer, Rosan and Baryawno, Ninib and Gerling, Marco and Enge, Martin},
  doi          = {10.1038/s41592-021-01196-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {912-920},
  shortjournal = {Nat. Meth.},
  title        = {An unsupervised method for physical cell interaction profiling of complex tissues},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SCITO-seq: Single-cell combinatorial indexed cytometry
sequencing. <em>NMETH</em>, <em>18</em>(8), 903–911. (<a
href="https://doi.org/10.1038/s41592-021-01222-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of DNA-barcoded antibodies to tag cell surface molecules has enabled the use of droplet-based single-cell sequencing (dsc-seq) to profile protein abundances from thousands of cells simultaneously. As compared to flow and mass cytometry, the high per cell cost of current dsc-seq-based workflows precludes their use in clinical applications and large-scale pooled screens. Here, we introduce SCITO-seq, a workflow that uses splint oligonucleotides (oligos) to enable combinatorially indexed dsc-seq of DNA-barcoded antibodies from over 105 cells per reaction using commercial microfluidics. By encoding sample barcodes into splint oligos, we demonstrate that multiplexed SCITO-seq produces reproducible estimates of cellular composition and surface protein expression comparable to those from mass cytometry. We further demonstrate two modified splint oligo designs that extend SCITO-seq to achieve compatibility with commercial DNA-barcoded antibodies and simultaneous expression profiling of the transcriptome and surface proteins from the same cell. These results demonstrate SCITO-seq as a flexible and ultra-high-throughput platform for sequencing-based single-cell protein and multimodal profiling. Single-cell combinatorial indexed cytometry sequencing (SCITO-seq) combines split-pool indexing and droplet-based single-cell sequencing for single-cell protein profiling.},
  archive      = {J_NMETH},
  author       = {Hwang, Byungjin and Lee, David S. and Tamaki, Whitney and Sun, Yang and Ogorodnikov, Anton and Hartoularos, George C. and Winters, Aidan and Yeung, Bertrand Z. and Nazor, Kristopher L. and Song, Yun S. and Chow, Eric D. and Spitzer, Matthew H. and Ye, Chun Jimmie},
  doi          = {10.1038/s41592-021-01222-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {903-911},
  shortjournal = {Nat. Meth.},
  title        = {SCITO-seq: Single-cell combinatorial indexed cytometry sequencing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A 4D single-cell protein atlas of transcription factors
delineates spatiotemporal patterning during embryogenesis.
<em>NMETH</em>, <em>18</em>(8), 893–902. (<a
href="https://doi.org/10.1038/s41592-021-01216-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex biological processes such as embryogenesis require precise coordination of cell differentiation programs across both space and time. Using protein-fusion fluorescent reporters and four-dimensional live imaging, we present a protein expression atlas of transcription factors (TFs) mapped onto developmental cell lineages during Caenorhabditis elegans embryogenesis, at single-cell resolution. This atlas reveals a spatiotemporal combinatorial code of TF expression, and a cascade of lineage-specific, tissue-specific and time-specific TFs that specify developmental states. The atlas uncovers regulators of embryogenesis, including an unexpected role of a skin specifier in neurogenesis and the critical function of an uncharacterized TF in convergent muscle differentiation. At the systems level, the atlas provides an opportunity to model cell state–fate relationships, revealing a lineage-dependent state diversity within functionally related cells and a winding trajectory of developmental state progression. Collectively, this single-cell protein atlas represents a valuable resource for elucidating metazoan embryogenesis at the molecular and systems levels. A protein expression atlas of transcription factors charted onto cell lineage maps of Caenorhabditis elegans development that uncovers mechanisms of spatiotemporal cell fate patterning and regulators of embryogenesis.},
  archive      = {J_NMETH},
  author       = {Ma, Xuehua and Zhao, Zhiguang and Xiao, Long and Xu, Weina and Kou, Yahui and Zhang, Yanping and Wu, Gang and Wang, Yangyang and Du, Zhuo},
  doi          = {10.1038/s41592-021-01216-1},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {893-902},
  shortjournal = {Nat. Meth.},
  title        = {A 4D single-cell protein atlas of transcription factors delineates spatiotemporal patterning during embryogenesis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-throughput and single-cell t cell receptor sequencing
technologies. <em>NMETH</em>, <em>18</em>(8), 881–892. (<a
href="https://doi.org/10.1038/s41592-021-01201-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells express T cell receptors (TCRs) composed of somatically recombined TCRα and TCRβ chains, which mediate recognition of major histocompatibility complex (MHC)–antigen complexes and drive the antigen-specific adaptive immune response to pathogens and cancer. The TCR repertoire in each individual is highly diverse, which allows for recognition of a wide array of foreign antigens, but also presents a challenge in analyzing this response using conventional methods. Recent studies have developed high-throughput sequencing technologies to identify TCR sequences, analyze their antigen specificities using experimental and computational tools, and pair TCRs with transcriptional and epigenetic cell state phenotypes in single cells. In this Review, we highlight these technological advances and describe how they have been applied to discover fundamental insights into T cell-mediated immunity. This Review discusses methodological advances for high-throughput analysis of the TCR repertoire at bulk and single-cell resolution.},
  archive      = {J_NMETH},
  author       = {Pai, Joy A. and Satpathy, Ansuman T.},
  doi          = {10.1038/s41592-021-01201-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {881-892},
  shortjournal = {Nat. Meth.},
  title        = {High-throughput and single-cell t cell receptor sequencing technologies},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). T cell antigen discovery. <em>NMETH</em>, <em>18</em>(8),
873–880. (<a href="https://doi.org/10.1038/s41592-020-0867-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells respond to threats in an antigen-specific manner using T cell receptors (TCRs) that recognize short peptide antigens presented on major histocompatibility complex (MHC) proteins. The TCR–peptide-MHC interaction mediated between a T cell and its target cell dictates its function and thereby influences its role in disease. A lack of approaches for antigen discovery has limited the fundamental understanding of the antigenic landscape of the overall T cell response. Recent advances in high-throughput sequencing, mass cytometry, microfluidics and computational biology have led to a surge in approaches to address the challenge of T cell antigen discovery. Here, we summarize the scope of this challenge, discuss in depth the recent exciting work and highlight the outstanding questions and remaining technical hurdles in this field. A review on recent method development for discovering T cell antigens.},
  archive      = {J_NMETH},
  author       = {Joglekar, Alok V. and Li, Guideng},
  doi          = {10.1038/s41592-020-0867-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {873-880},
  shortjournal = {Nat. Meth.},
  title        = {T cell antigen discovery},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Neural networks learn the motions of molecular machines.
<em>NMETH</em>, <em>18</em>(8), 869–871. (<a
href="https://doi.org/10.1038/s41592-021-01235-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New computational approaches capture molecular motion from cryo-EM images and provide a more complete understanding of protein dynamics.},
  archive      = {J_NMETH},
  author       = {Grant, Timothy},
  doi          = {10.1038/s41592-021-01235-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {869-871},
  shortjournal = {Nat. Meth.},
  title        = {Neural networks learn the motions of molecular machines},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Triangulating spatial relationships from single-cell
interaction maps. <em>NMETH</em>, <em>18</em>(8), 867–869. (<a
href="https://doi.org/10.1038/s41592-021-01221-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A combined experimental and computational approach to transcriptomic profiling of cell ‘multiplets’ enables the reconstruction of cell–cell interactions and higher-order structural features of biological tissues.},
  archive      = {J_NMETH},
  author       = {Wagner, Daniel E.},
  doi          = {10.1038/s41592-021-01221-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {867-869},
  shortjournal = {Nat. Meth.},
  title        = {Triangulating spatial relationships from single-cell interaction maps},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The long and winding road of development: A coordinated song
of transcription factors. <em>NMETH</em>, <em>18</em>(8), 866–867. (<a
href="https://doi.org/10.1038/s41592-021-01219-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A four-dimensional single-cell atlas of transcription factor expression in Caenorhabditis elegans allows the identification of novel regulators of embryo development and the generation of molecular models of cell fate specification.},
  archive      = {J_NMETH},
  author       = {Makhlouf, Aly and N. Shahbazi, Marta},
  doi          = {10.1038/s41592-021-01219-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {866-867},
  shortjournal = {Nat. Meth.},
  title        = {The long and winding road of development: A coordinated song of transcription factors},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Exploring the diverse, intimate lives of plants.
<em>NMETH</em>, <em>18</em>(8), 861–865. (<a
href="https://doi.org/10.1038/s41592-021-01228-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New ways to assess plant–microbe interactions can yield unexpected paths to biodiversity.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01228-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {861-865},
  shortjournal = {Nat. Meth.},
  title        = {Exploring the diverse, intimate lives of plants},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Bespoke antibodies fast tracked. <em>NMETH</em>,
<em>18</em>(8), 860. (<a
href="https://doi.org/10.1038/s41592-021-01247-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A yeast cell culture approach mimics the benefits of somatic hypermutation to accelerate the discovery of desirable antibodies.},
  archive      = {J_NMETH},
  author       = {Marcinkiewicz, Katarzyna},
  doi          = {10.1038/s41592-021-01247-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {860},
  shortjournal = {Nat. Meth.},
  title        = {Bespoke antibodies fast tracked},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Atomic force microscopy in super-resolution.
<em>NMETH</em>, <em>18</em>(8), 859. (<a
href="https://doi.org/10.1038/s41592-021-01246-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Applying principles from super-resolution fluorescence microscopy increases the resolution of atomic force microscopy.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01246-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {859},
  shortjournal = {Nat. Meth.},
  title        = {Atomic force microscopy in super-resolution},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Arsenal of single-cell multi-omics methods expanded.
<em>NMETH</em>, <em>18</em>(8), 858. (<a
href="https://doi.org/10.1038/s41592-021-01245-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin accessibility, proteins levels and other features measured from the same cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01245-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {858},
  shortjournal = {Nat. Meth.},
  title        = {Arsenal of single-cell multi-omics methods expanded},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021s). Trapping virus in a shell. <em>NMETH</em>, <em>18</em>(8),
857. (<a href="https://doi.org/10.1038/s41592-021-01244-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers design DNA shells to trap virus particles and thus prevent interactions between viruses and host cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01244-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {857},
  shortjournal = {Nat. Meth.},
  title        = {Trapping virus in a shell},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Towards resolving proteomes in single cells.
<em>NMETH</em>, <em>18</em>(8), 856. (<a
href="https://doi.org/10.1038/s41592-021-01243-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Development of methods for single-cell proteomics increases depth and throughput},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01243-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {856},
  shortjournal = {Nat. Meth.},
  title        = {Towards resolving proteomes in single cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Imaging africa: A strategic approach to optical microscopy
training in africa. <em>NMETH</em>, <em>18</em>(8), 847–855. (<a
href="https://doi.org/10.1038/s41592-021-01227-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Life scientists in Africa have had limited opportunity to participate in international advanced scientific training programs and workshops, which largely benefit researchers in North America, Europe and the Asia–Pacific region. Here, we chronicle the strategies adopted and challenges encountered in organizing Imaging Africa, an all-expenses-paid, continent-wide practical workshop in optical microscopy hosted in South Africa from 13 to 17 January 2020. Our experience can help steer other groups who similarly seek to organize impactful and sustainable training initiatives in Africa.},
  archive      = {J_NMETH},
  author       = {Reiche, Michael A. and Warner, Digby F. and Aaron, Jesse and Khuon, Satya and Fletcher, Daniel A. and Hahn, Klaus and Rogers, Kelly L. and Mhlanga, Musa and Koch, Anastasia and Quaye, Wendye and Chew, Teng-Leong},
  doi          = {10.1038/s41592-021-01227-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {847-855},
  shortjournal = {Nat. Meth.},
  title        = {Imaging africa: A strategic approach to optical microscopy training in africa},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CloudReg: Automatic terabyte-scale cross-modal brain volume
registration. <em>NMETH</em>, <em>18</em>(8), 845–846. (<a
href="https://doi.org/10.1038/s41592-021-01218-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Chandrashekhar, Vikram and Tward, Daniel J. and Crowley, Devin and Crow, Ailey K. and Wright, Matthew A. and Hsueh, Brian Y. and Gore, Felicity and Machado, Timothy A. and Branch, Audrey and Rosenblum, Jared S. and Deisseroth, Karl and Vogelstein, Joshua T.},
  doi          = {10.1038/s41592-021-01218-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {845-846},
  shortjournal = {Nat. Meth.},
  title        = {CloudReg: Automatic terabyte-scale cross-modal brain volume registration},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). GEAR: Gene expression analysis resource portal for
community-driven, multi-omic data exploration. <em>NMETH</em>,
<em>18</em>(8), 843–844. (<a
href="https://doi.org/10.1038/s41592-021-01200-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Orvis, Joshua and Gottfried, Brian and Kancherla, Jayaram and Adkins, Ricky S. and Song, Yang and Dror, Amiel A. and Olley, Dustin and Rose, Kevin and Chrysostomou, Elena and Kelly, Michael C. and Milon, Beatrice and Matern, Maggie S. and Azaiez, Hela and Herb, Brian and Colantuoni, Carlo and Carter, Robert L. and Ament, Seth A. and Kelley, Matthew W. and White, Owen and Bravo, Hector Corrada and Mahurkar, Anup and Hertzano, Ronna},
  doi          = {10.1038/s41592-021-01200-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {843-844},
  shortjournal = {Nat. Meth.},
  title        = {GEAR: Gene expression analysis resource portal for community-driven, multi-omic data exploration},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021o). Pavan ramdya. <em>NMETH</em>, <em>18</em>(8), 839. (<a
href="https://doi.org/10.1038/s41592-021-01237-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A neuroscientist who jams, plays and builds a way to capture animal movement.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01237-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {839},
  shortjournal = {Nat. Meth.},
  title        = {Pavan ramdya},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Going pro. <em>NMETH</em>, <em>18</em>(8), 837. (<a
href="https://doi.org/10.1038/s41592-021-01233-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Professional editing can be a rewarding career. Here we discuss what the job entails, highs and lows, how to prepare for this career, the interview process and more.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01233-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {837},
  shortjournal = {Nat. Meth.},
  title        = {Going pro},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Graphical assessment of tests and classifiers.
<em>NMETH</em>, <em>18</em>(8), 840–842. (<a
href="https://doi.org/10.1038/s41592-021-01232-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {I do not think you can start with anything precise. You have to achieve such precision as you can, as you go along. —Bertrand Russell},
  archive      = {J_NMETH},
  author       = {Altman, Naomi and Krzywinski, Martin},
  doi          = {10.1038/s41592-021-01232-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {8},
  pages        = {840-842},
  shortjournal = {Nat. Meth.},
  title        = {Graphical assessment of tests and classifiers},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Simultaneous cortex-wide fluorescence
ca2+ imaging and whole-brain fMRI. <em>NMETH</em>, <em>18</em>(7), 835.
(<a href="https://doi.org/10.1038/s41592-021-01217-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Lake, Evelyn M. R. and Ge, Xinxin and Shen, Xilin and Herman, Peter and Hyder, Fahmeed and Cardin, Jessica A. and Higley, Michael J. and Scheinost, Dustin and Papademetris, Xenophon and Crair, Michael C. and Constable, R. Todd},
  doi          = {10.1038/s41592-021-01217-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {835},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Simultaneous cortex-wide fluorescence ca2+ imaging and whole-brain fMRI},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Publisher correction: The triumphs and limitations of
computational methods for scRNA-seq. <em>NMETH</em>, <em>18</em>(7),
835. (<a href="https://doi.org/10.1038/s41592-021-01223-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kharchenko, Peter V.},
  doi          = {10.1038/s41592-021-01223-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {835},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: The triumphs and limitations of computational methods for scRNA-seq},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Real-time multi-angle projection imaging of biological
dynamics. <em>NMETH</em>, <em>18</em>(7), 829–834. (<a
href="https://doi.org/10.1038/s41592-021-01175-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce a cost-effective and easily implementable scan unit that converts any camera-based microscope with optical sectioning capability into a multi-angle projection imaging system. Projection imaging reduces data overhead and accelerates imaging by a factor of &amp;gt;100, while also allowing users to readily view biological phenomena of interest from multiple perspectives on the fly. By rapidly interrogating the sample from just two perspectives, our method also enables real-time stereoscopic imaging and three-dimensional particle localization. We demonstrate projection imaging with spinning disk confocal, lattice light-sheet, multidirectional illumination light-sheet and oblique plane microscopes on specimens that range from organelles in single cells to the vasculature of a zebrafish embryo. Furthermore, we leverage our projection method to rapidly image cancer cell morphodynamics and calcium signaling in cultured neurons at rates up to 119 Hz as well as to simultaneously image orthogonal views of a beating embryonic zebrafish heart. Multiangle projection imaging accelerates volumetric imaging by up to two orders of magnitude and is readily implemented on diverse microscopes, including spinning disk confocal and light-sheet microscopes.},
  archive      = {J_NMETH},
  author       = {Chang, Bo-Jui and Manton, James D. and Sapoznik, Etai and Pohlkamp, Theresa and Terrones, Tamara S. and Welf, Erik S. and Murali, Vasanth S. and Roudot, Philippe and Hake, Kayley and Whitehead, Lachlan and York, Andrew G. and Dean, Kevin M. and Fiolka, Reto},
  doi          = {10.1038/s41592-021-01175-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {829-834},
  shortjournal = {Nat. Meth.},
  title        = {Real-time multi-angle projection imaging of biological dynamics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Structured illumination microscopy with noise-controlled
image reconstructions. <em>NMETH</em>, <em>18</em>(7), 821–828. (<a
href="https://doi.org/10.1038/s41592-021-01167-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Super-resolution structured illumination microscopy (SIM) has become a widely used method for biological imaging. Standard reconstruction algorithms, however, are prone to generate noise-specific artifacts that limit their applicability for lower signal-to-noise data. Here we present a physically realistic noise model that explains the structured noise artifact, which we then use to motivate new complementary reconstruction approaches. True-Wiener-filtered SIM optimizes contrast given the available signal-to-noise ratio, and flat-noise SIM fully overcomes the structured noise artifact while maintaining resolving power. Both methods eliminate ad hoc user-adjustable reconstruction parameters in favor of physical parameters, enhancing objectivity. The new reconstructions point to a trade-off between contrast and a natural noise appearance. This trade-off can be partly overcome by further notch filtering but at the expense of a decrease in signal-to-noise ratio. The benefits of the proposed approaches are demonstrated on focal adhesion and tubulin samples in two and three dimensions, and on nanofabricated fluorescent test patterns. Super-resolution structured illumination microscopy reconstruction algorithms are described that can handle structured noise artifacts in two and three dimensions. The algorithms lack adjustable parameters and enhance objective representation of imaged objects.},
  archive      = {J_NMETH},
  author       = {Smith, Carlas S. and Slotman, Johan A. and Schermelleh, Lothar and Chakrova, Nadya and Hari, Sangeetha and Vos, Yoram and Hagen, Cornelis W. and Müller, Marcel and van Cappellen, Wiggert and Houtsmuller, Adriaan B. and Hoogenboom, Jacob P. and Stallinga, Sjoerd},
  doi          = {10.1038/s41592-021-01167-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {821-828},
  shortjournal = {Nat. Meth.},
  title        = {Structured illumination microscopy with noise-controlled image reconstructions},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ABEL-FRET: Tether-free single-molecule FRET with
hydrodynamic profiling. <em>NMETH</em>, <em>18</em>(7), 816–820. (<a
href="https://doi.org/10.1038/s41592-021-01173-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule Förster resonance energy transfer (smFRET) has become a versatile and widespread method to probe nanoscale conformation and dynamics. However, current experimental modalities often resort to molecule immobilization for long observation times and do not always approach the resolution limit of FRET-based nanoscale metrology. Here we present ABEL-FRET, an immobilization-free platform for smFRET measurements with ultrahigh resolving power in FRET efficiency. Importantly, single-molecule diffusivity is used to provide additional size and shape information for hydrodynamic profiling of individual molecules, which, together with the concurrently measured intramolecular conformation through FRET, enables a holistic and dynamic view of biomolecules and their complexes. ABEL-FRET combines anti-Brownian trapping for tether-free observation of biomolecules in solution with smFRET. ABEL-FRET offers ultrahigh resolution of FRET efficiency and enables simultaneous hydrodynamic profiling of molecular composition.},
  archive      = {J_NMETH},
  author       = {Wilson, Hugh and Wang, Quan},
  doi          = {10.1038/s41592-021-01173-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {816-820},
  shortjournal = {Nat. Meth.},
  title        = {ABEL-FRET: Tether-free single-molecule FRET with hydrodynamic profiling},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Meta-analysis defines principles for the design and analysis
of co-fractionation mass spectrometry experiments. <em>NMETH</em>,
<em>18</em>(7), 806–815. (<a
href="https://doi.org/10.1038/s41592-021-01194-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Co-fractionation mass spectrometry (CF-MS) has emerged as a powerful technique for interactome mapping. However, there is little consensus on optimal strategies for the design of CF-MS experiments or their computational analysis. Here, we reanalyzed a total of 206 CF-MS experiments to generate a uniformly processed resource containing over 11 million measurements of protein abundance. We used this resource to benchmark experimental designs for CF-MS studies and systematically optimize computational approaches to network inference. We then applied this optimized methodology to reconstruct a draft-quality human interactome by CF-MS and predict over 700,000 protein–protein interactions across 27 eukaryotic species or clades. Our work defines new resources to illuminate proteome organization over evolutionary timescales and establishes best practices for the design and analysis of CF-MS studies. Meta-analysis of more than 200 co-fractionation mass spectrometry experiments provides expanded protein–protein interaction resources and helps establish optimal protocols for experimental design and data analysis.},
  archive      = {J_NMETH},
  author       = {Skinnider, Michael A. and Foster, Leonard J.},
  doi          = {10.1038/s41592-021-01194-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {806-815},
  shortjournal = {Nat. Meth.},
  title        = {Meta-analysis defines principles for the design and analysis of co-fractionation mass spectrometry experiments},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SpaceM reveals metabolic states of single cells.
<em>NMETH</em>, <em>18</em>(7), 799–805. (<a
href="https://doi.org/10.1038/s41592-021-01198-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A growing appreciation of the importance of cellular metabolism and revelations concerning the extent of cell–cell heterogeneity demand metabolic characterization of individual cells. We present SpaceM, an open-source method for in situ single-cell metabolomics that detects &amp;gt;100 metabolites from &amp;gt;1,000 individual cells per hour, together with a fluorescence-based readout and retention of morpho-spatial features. We validated SpaceM by predicting the cell types of cocultured human epithelial cells and mouse fibroblasts. We used SpaceM to show that stimulating human hepatocytes with fatty acids leads to the emergence of two coexisting subpopulations outlined by distinct cellular metabolic states. Inducing inflammation with the cytokine interleukin-17A perturbs the balance of these states in a process dependent on NF-κB signaling. The metabolic state markers were reproduced in a murine model of nonalcoholic steatohepatitis. We anticipate SpaceM to be broadly applicable for investigations of diverse cellular models and to democratize single-cell metabolomics. SpaceM combines light microscopy and MALDI-imaging mass spectrometry to enable single-cell metabolic profiling of cultured cells. SpaceM reveals coexisting metabolic states of hepatocytes and aims to democratize single-cell metabolomics.},
  archive      = {J_NMETH},
  author       = {Rappez, Luca and Stadler, Mira and Triana, Sergio and Gathungu, Rose Muthoni and Ovchinnikova, Katja and Phapale, Prasad and Heikenwalder, Mathias and Alexandrov, Theodore},
  doi          = {10.1038/s41592-021-01198-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {799-805},
  shortjournal = {Nat. Meth.},
  title        = {SpaceM reveals metabolic states of single cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Metabolomic profiling of single enlarged lysosomes.
<em>NMETH</em>, <em>18</em>(7), 788–798. (<a
href="https://doi.org/10.1038/s41592-021-01182-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lysosomes are critical for cellular metabolism and are heterogeneously involved in various cellular processes. The ability to measure lysosomal metabolic heterogeneity is essential for understanding their physiological roles. We therefore built a single-lysosome mass spectrometry (SLMS) platform integrating lysosomal patch-clamp recording and induced nano-electrospray ionization (nanoESI)/mass spectrometry (MS) that enables concurrent metabolic and electrophysiological profiling of individual enlarged lysosomes. The accuracy and reliability of this technique were validated by supporting previous findings, such as the transportability of lysosomal cationic amino acids transporters such as PQLC2 and the lysosomal trapping of lysosomotropic, hydrophobic weak base drugs such as lidocaine. We derived metabolites from single lysosomes in various cell types and classified lysosomes into five major subpopulations based on their chemical and biological divergence. Senescence and carcinoma altered metabolic profiles of lysosomes in a type-specific manner. Thus, SLMS can open more avenues for investigating heterogeneous lysosomal metabolic changes during physiological and pathological processes. Single-lysosome mass spectrometry (SLMS) integrates lysosomal patch-clamp recording and induced nanoESI/MS for concurrent metabolic and electrophysiological profiling of individual enlarged lysosomes.},
  archive      = {J_NMETH},
  author       = {Zhu, Hongying and Li, Qianqian and Liao, Tiepeng and Yin, Xiang and Chen, Qi and Wang, Ziyi and Dai, Meifang and Yi, Lin and Ge, Siyuan and Miao, Chenjian and Zeng, Wenping and Qu, Lili and Ju, Zhenyu and Huang, Guangming and Cang, Chunlei and Xiong, Wei},
  doi          = {10.1038/s41592-021-01182-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {788-798},
  shortjournal = {Nat. Meth.},
  title        = {Metabolomic profiling of single enlarged lysosomes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). DecoID improves identification rates in metabolomics through
database-assisted MS/MS deconvolution. <em>NMETH</em>, <em>18</em>(7),
779–787. (<a href="https://doi.org/10.1038/s41592-021-01195-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric MS/MS spectra contain fragments from multiple precursor ions and therefore hinder compound identification in metabolomics. Historically, deconvolution of these chimeric spectra has been challenging and relied on specific experimental methods that introduce variation in the ratios of precursor ions between multiple tandem mass spectrometry (MS/MS) scans. DecoID provides a complementary, method-independent approach where database spectra are computationally mixed to match an experimentally acquired spectrum by using LASSO regression. We validated that DecoID increases the number of identified metabolites in MS/MS datasets from both data-independent and data-dependent acquisition without increasing the false discovery rate. We applied DecoID to publicly available data from the MetaboLights repository and to data from human plasma, where DecoID increased the number of identified metabolites from data-dependent acquisition data by over 30\% compared to direct spectral matching. DecoID is compatible with any user-defined MS/MS database and provides automated searching for some of the largest MS/MS databases currently available. DecoID provides acquisition-independent, database-assisted deconvolution of metabolomic MS/MS spectra for improved metabolite identification.},
  archive      = {J_NMETH},
  author       = {Stancliffe, Ethan and Schwaiger-Haber, Michaela and Sindelar, Miriam and Patti, Gary J.},
  doi          = {10.1038/s41592-021-01195-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {779-787},
  shortjournal = {Nat. Meth.},
  title        = {DecoID improves identification rates in metabolomics through database-assisted MS/MS deconvolution},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). QSIPrep: An integrative platform for preprocessing and
reconstructing diffusion MRI data. <em>NMETH</em>, <em>18</em>(7),
775–778. (<a href="https://doi.org/10.1038/s41592-021-01185-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diffusion-weighted magnetic resonance imaging (dMRI) is the primary method for noninvasively studying the organization of white matter in the human brain. Here we introduce QSIPrep, an integrative software platform for the processing of diffusion images that is compatible with nearly all dMRI sampling schemes. Drawing on a diverse set of software suites to capitalize on their complementary strengths, QSIPrep facilitates the implementation of best practices for processing of diffusion images. QSIPrep is a software platform for processing of most diffusion MRI datasets and ensures that adequate workflows are used.},
  archive      = {J_NMETH},
  author       = {Cieslak, Matthew and Cook, Philip A. and He, Xiaosong and Yeh, Fang-Cheng and Dhollander, Thijs and Adebimpe, Azeez and Aguirre, Geoffrey K. and Bassett, Danielle S. and Betzel, Richard F. and Bourque, Josiane and Cabral, Laura M. and Davatzikos, Christos and Detre, John A. and Earl, Eric and Elliott, Mark A. and Fadnavis, Shreyas and Fair, Damien A. and Foran, Will and Fotiadis, Panagiotis and Garyfallidis, Eleftherios and Giesbrecht, Barry and Gur, Ruben C. and Gur, Raquel E. and Kelz, Max B. and Keshavan, Anisha and Larsen, Bart S. and Luna, Beatriz and Mackey, Allyson P. and Milham, Michael P. and Oathes, Desmond J. and Perrone, Anders and Pines, Adam R. and Roalf, David R. and Richie-Halford, Adam and Rokem, Ariel and Sydnor, Valerie J. and Tapera, Tinashe M. and Tooley, Ursula A. and Vettel, Jean M. and Yeatman, Jason D. and Grafton, Scott T. and Satterthwaite, Theodore D.},
  doi          = {10.1038/s41592-021-01185-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {775-778},
  shortjournal = {Nat. Meth.},
  title        = {QSIPrep: An integrative platform for preprocessing and reconstructing diffusion MRI data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Automatic detection of synaptic partners in a whole-brain
drosophila electron microscopy data set. <em>NMETH</em>, <em>18</em>(7),
771–774. (<a href="https://doi.org/10.1038/s41592-021-01183-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We develop an automatic method for synaptic partner identification in insect brains and use it to predict synaptic partners in a whole-brain electron microscopy dataset of the fruit fly. The predictions can be used to infer a connectivity graph with high accuracy, thus allowing fast identification of neural pathways. To facilitate circuit reconstruction using our results, we develop CIRCUITMAP, a user interface add-on for the circuit annotation tool CATMAID. A deep-learning-based approach enables automatic identification of synaptically connected neurons in electron microscopy datasets of the fly brain.},
  archive      = {J_NMETH},
  author       = {Buhmann, Julia and Sheridan, Arlo and Malin-Mayor, Caroline and Schlegel, Philipp and Gerhard, Stephan and Kazimiers, Tom and Krause, Renate and Nguyen, Tri M. and Heinrich, Larissa and Lee, Wei-Chung Allen and Wilson, Rachel and Saalfeld, Stephan and Jefferis, Gregory S. X. E. and Bock, Davi D. and Turaga, Srinivas C. and Cook, Matthew and Funke, Jan},
  doi          = {10.1038/s41592-021-01183-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {771-774},
  shortjournal = {Nat. Meth.},
  title        = {Automatic detection of synaptic partners in a whole-brain drosophila electron microscopy data set},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Universal spectrum identifier for mass spectra.
<em>NMETH</em>, <em>18</em>(7), 768–770. (<a
href="https://doi.org/10.1038/s41592-021-01184-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectra provide the ultimate evidence to support the findings of mass spectrometry proteomics studies in publications, and it is therefore crucial to be able to trace the conclusions back to the spectra. The Universal Spectrum Identifier (USI) provides a standardized mechanism for encoding a virtual path to any mass spectrum contained in datasets deposited to public proteomics repositories. USI enables greater transparency of spectral evidence, with more than 1 billion USI identifications from over 3 billion spectra already available through ProteomeXchange repositories. Universal Spectrum Identifier (USI) provides a standardized mechanism for encoding a virtual path to mass spectra deposited to public repositories or contained in public spectral libraries.},
  archive      = {J_NMETH},
  author       = {Deutsch, Eric W. and Perez-Riverol, Yasset and Carver, Jeremy and Kawano, Shin and Mendoza, Luis and Van Den Bossche, Tim and Gabriels, Ralf and Binz, Pierre-Alain and Pullman, Benjamin and Sun, Zhi and Shofstahl, Jim and Bittremieux, Wout and Mak, Tytus D. and Klein, Joshua and Zhu, Yunping and Lam, Henry and Vizcaíno, Juan Antonio and Bandeira, Nuno},
  doi          = {10.1038/s41592-021-01184-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {768-770},
  shortjournal = {Nat. Meth.},
  title        = {Universal spectrum identifier for mass spectra},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Huang et al. reply. <em>NMETH</em>, <em>18</em>(7), 763–767.
(<a href="https://doi.org/10.1038/s41592-021-01179-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Huang, Jun X. and Wu, David and Stein, Benjamin D. and Moellering, Raymond E.},
  doi          = {10.1038/s41592-021-01179-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {763-767},
  shortjournal = {Nat. Meth.},
  title        = {Huang et al. reply},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identification of phosphosites that alter protein thermal
stability. <em>NMETH</em>, <em>18</em>(7), 760–762. (<a
href="https://doi.org/10.1038/s41592-021-01178-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Smith, Ian R. and Hess, Kyle N. and Bakhtina, Anna A. and Valente, Anthony S. and Rodríguez-Mias, Ricard A. and Villén, Judit},
  doi          = {10.1038/s41592-021-01178-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {760-762},
  shortjournal = {Nat. Meth.},
  title        = {Identification of phosphosites that alter protein thermal stability},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Impact of phosphorylation on thermal stability of proteins.
<em>NMETH</em>, <em>18</em>(7), 757–759. (<a
href="https://doi.org/10.1038/s41592-021-01177-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Potel, Clément M. and Kurzawa, Nils and Becher, Isabelle and Typas, Athanasios and Mateus, André and Savitski, Mikhail M.},
  doi          = {10.1038/s41592-021-01177-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {757-759},
  shortjournal = {Nat. Meth.},
  title        = {Impact of phosphorylation on thermal stability of proteins},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mass spectrometry-based metabolomics: A guide for
annotation, quantification and best reporting practices. <em>NMETH</em>,
<em>18</em>(7), 747–756. (<a
href="https://doi.org/10.1038/s41592-021-01197-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectrometry-based metabolomics approaches can enable detection and quantification of many thousands of metabolite features simultaneously. However, compound identification and reliable quantification are greatly complicated owing to the chemical complexity and dynamic range of the metabolome. Simultaneous quantification of many metabolites within complex mixtures can additionally be complicated by ion suppression, fragmentation and the presence of isomers. Here we present guidelines covering sample preparation, replication and randomization, quantification, recovery and recombination, ion suppression and peak misidentification, as a means to enable high-quality reporting of liquid chromatography– and gas chromatography–mass spectrometry-based metabolomics-derived data. This Perspective, from a large group of metabolomics experts, provides best practices and simplified reporting guidelines for practitioners of liquid chromatography– and gas chromatography–mass spectrometry-based metabolomics.},
  archive      = {J_NMETH},
  author       = {Alseekh, Saleh and Aharoni, Asaph and Brotman, Yariv and Contrepois, Kévin and D’Auria, John and Ewald, Jan and C. Ewald, Jennifer and Fraser, Paul D. and Giavalisco, Patrick and Hall, Robert D. and Heinemann, Matthias and Link, Hannes and Luo, Jie and Neumann, Steffen and Nielsen, Jens and Perez de Souza, Leonardo and Saito, Kazuki and Sauer, Uwe and Schroeder, Frank C. and Schuster, Stefan and Siuzdak, Gary and Skirycz, Aleksandra and Sumner, Lloyd W. and Snyder, Michael P. and Tang, Huiru and Tohge, Takayuki and Wang, Yulan and Wen, Weiwei and Wu, Si and Xu, Guowang and Zamboni, Nicola and Fernie, Alisdair R.},
  doi          = {10.1038/s41592-021-01197-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {747-756},
  shortjournal = {Nat. Meth.},
  title        = {Mass spectrometry-based metabolomics: A guide for annotation, quantification and best reporting practices},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultra-high-performance liquid chromatography high-resolution
mass spectrometry variants for metabolomics research. <em>NMETH</em>,
<em>18</em>(7), 733–746. (<a
href="https://doi.org/10.1038/s41592-021-01116-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultra-high-performance liquid chromatography high-resolution mass spectrometry (UHPLC–HRMS) variants currently represent the best tools to tackle the challenges of complexity and lack of comprehensive coverage of the metabolome. UHPLC offers flexible and efficient separation coupled with high-sensitivity detection via HRMS, allowing for the detection and identification of a broad range of metabolites. Here we discuss current common strategies for UHPLC–HRMS-based metabolomics, with a focus on expanding metabolome coverage. This Review surveys ultra-high-performance liquid chromatography high-resolution mass spectrometry (UHPLC–HRMS), a highly sensitive, high-throughput technique that is used for analyzing a broad range of metabolites.},
  archive      = {J_NMETH},
  author       = {Perez de Souza, Leonardo and Alseekh, Saleh and Scossa, Federico and Fernie, Alisdair R.},
  doi          = {10.1038/s41592-021-01116-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {733-746},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-performance liquid chromatography high-resolution mass spectrometry variants for metabolomics research},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). The triumphs and limitations of computational methods for
scRNA-seq. <em>NMETH</em>, <em>18</em>(7), 723–732. (<a
href="https://doi.org/10.1038/s41592-021-01171-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid progress of protocols for sequencing single-cell transcriptomes over the past decade has been accompanied by equally impressive advances in the computational methods for analysis of such data. As capacity and accuracy of the experimental techniques grew, the emerging algorithm developments revealed increasingly complex facets of the underlying biology, from cell type composition to gene regulation to developmental dynamics. At the same time, rapid growth has forced continuous reevaluation of the underlying statistical models, experimental aims, and sheer volumes of data processing that are handled by these computational tools. Here, I review key computational steps of single-cell RNA sequencing (scRNA-seq) analysis, examine assumptions made by different approaches, and highlight successes, remaining ambiguities, and limitations that are important to keep in mind as scRNA-seq becomes a mainstream technique for studying biology. This review provides an overview of recent computational developments in scRNA-seq analysis and highlights packages and tools applied in executing these analyses.},
  archive      = {J_NMETH},
  author       = {Kharchenko, Peter V.},
  doi          = {10.1038/s41592-021-01171-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {723-732},
  shortjournal = {Nat. Meth.},
  title        = {The triumphs and limitations of computational methods for scRNA-seq},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Towards reproducibility in large-scale analysis of
protein–protein interactions. <em>NMETH</em>, <em>18</em>(7), 720–721.
(<a href="https://doi.org/10.1038/s41592-021-01202-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Co-fractionation mass spectrometry (CF-MS) has the potential to measure thousands of protein complexes in a single experiment, but the field is still in its infancy. A meta-analysis of CF-MS data yields a core CF-MS interactome and a tool allowing researchers to align new results to published data.},
  archive      = {J_NMETH},
  author       = {Lund-Johansen, Fridtjof and Tran, Trung and Mehta, Adi},
  doi          = {10.1038/s41592-021-01202-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {720-721},
  shortjournal = {Nat. Meth.},
  title        = {Towards reproducibility in large-scale analysis of protein–protein interactions},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Biology begins to tangle with quantum computing.
<em>NMETH</em>, <em>18</em>(7), 715–719. (<a
href="https://doi.org/10.1038/s41592-021-01199-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum computing promises plenty, such as how it can massively accelerate some bioinformatics calculations.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01199-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {715-719},
  shortjournal = {Nat. Meth.},
  title        = {Biology begins to tangle with quantum computing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Mapping human hematopoiesis. <em>NMETH</em>,
<em>18</em>(7), 714. (<a
href="https://doi.org/10.1038/s41592-021-01214-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lineage tracing using somatic mutations as barcodes helps decode human hematopoiesis.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01214-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {714},
  shortjournal = {Nat. Meth.},
  title        = {Mapping human hematopoiesis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Subcellular proteome map of human cells. <em>NMETH</em>,
<em>18</em>(7), 713. (<a
href="https://doi.org/10.1038/s41592-021-01213-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {BioID using organelle-specific bait proteins reveals the subcellular HEK293 proteome.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01213-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {713},
  shortjournal = {Nat. Meth.},
  title        = {Subcellular proteome map of human cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Optogenetic silencing at synaptic terminals.
<em>NMETH</em>, <em>18</em>(7), 712. (<a
href="https://doi.org/10.1038/s41592-021-01212-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GPCR-based opsins can be harnessed to inhibit neurotransmission via inhibitory G protein signaling.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01212-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {712},
  shortjournal = {Nat. Meth.},
  title        = {Optogenetic silencing at synaptic terminals},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021n). Sequencing diverse RNA modifications. <em>NMETH</em>,
<em>18</em>(7), 711. (<a
href="https://doi.org/10.1038/s41592-021-01211-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanopore sequencing enables quantitative profiling of pseudouridine and 2′-O-methylation modifications in native RNAs.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01211-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {711},
  shortjournal = {Nat. Meth.},
  title        = {Sequencing diverse RNA modifications},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Breaking barriers for intravital imaging. <em>NMETH</em>,
<em>18</em>(7), 710. (<a
href="https://doi.org/10.1038/s41592-021-01210-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in adaptive optics combined with a twist on light field microscopy enable high-quality imaging.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01210-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {710},
  shortjournal = {Nat. Meth.},
  title        = {Breaking barriers for intravital imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantum computing at the frontiers of biological sciences.
<em>NMETH</em>, <em>18</em>(7), 701–709. (<a
href="https://doi.org/10.1038/s41592-020-01004-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computing plays a critical role in the biological sciences but faces increasing challenges of scale and complexity. Quantum computing, a computational paradigm exploiting the unique properties of quantum mechanical analogs of classical bits, seeks to address many of these challenges. We discuss the potential for quantum computing to aid in the merging of insights across different areas of biological sciences.},
  archive      = {J_NMETH},
  author       = {Emani, Prashant S. and Warrell, Jonathan and Anticevic, Alan and Bekiranov, Stefan and Gandal, Michael and McConnell, Michael J. and Sapiro, Guillermo and Aspuru-Guzik, Alán and Baker, Justin T. and Bastiani, Matteo and Murray, John D. and Sotiropoulos, Stamatios N. and Taylor, Jacob and Senthil, Geetha and Lehner, Thomas and Gerstein, Mark B. and Harrow, Aram W.},
  doi          = {10.1038/s41592-020-01004-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {701-709},
  shortjournal = {Nat. Meth.},
  title        = {Quantum computing at the frontiers of biological sciences},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The ENIGMA toolbox: Multiscale neural contextualization of
multisite neuroimaging datasets. <em>NMETH</em>, <em>18</em>(7),
698–700. (<a href="https://doi.org/10.1038/s41592-021-01186-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Larivière, Sara and Paquola, Casey and Park, Bo-yong and Royer, Jessica and Wang, Yezhou and Benkarim, Oualid and Vos de Wael, Reinder and Valk, Sofie L. and Thomopoulos, Sophia I. and Kirschner, Matthias and Lewis, Lindsay B. and Evans, Alan C. and Sisodiya, Sanjay M. and McDonald, Carrie R. and Thompson, Paul M. and Bernhardt, Boris C.},
  doi          = {10.1038/s41592-021-01186-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {698-700},
  shortjournal = {Nat. Meth.},
  title        = {The ENIGMA toolbox: Multiscale neural contextualization of multisite neuroimaging datasets},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computation and biology: A partnership. <em>NMETH</em>,
<em>18</em>(7), 695. (<a
href="https://doi.org/10.1038/s41592-021-01215-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computation goes hand in hand with contemporary biological studies. We describe a few trends in computational science that are helping drive new biological knowledge.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01215-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {695},
  shortjournal = {Nat. Meth.},
  title        = {Computation and biology: A partnership},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021r). Reto fiolka. <em>NMETH</em>, <em>18</em>(7), 697. (<a
href="https://doi.org/10.1038/s41592-021-01204-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A flight path into biology and fast 3D projection of sample dynamics.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01204-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {7},
  pages        = {697},
  shortjournal = {Nat. Meth.},
  title        = {Reto fiolka},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Three-dimensional adaptive optical nanoscopy for thick
specimen imaging at sub-50-nm resolution. <em>NMETH</em>,
<em>18</em>(6), 688–693. (<a
href="https://doi.org/10.1038/s41592-021-01149-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding cellular organization demands the best possible spatial resolution in all three dimensions. In fluorescence microscopy, this is achieved by 4Pi nanoscopy methods that combine the concepts of using two opposing objectives for optimal diffraction-limited 3D resolution with switching fluorescent molecules between bright and dark states to break the diffraction limit. However, optical aberrations have limited these nanoscopes to thin samples and prevented their application in thick specimens. Here we have developed an improved iso-stimulated emission depletion nanoscope, which uses an advanced adaptive optics strategy to achieve sub-50-nm isotropic resolution of structures such as neuronal synapses and ring canals previously inaccessible in tissue. The adaptive optics scheme presented in this work is generally applicable to any microscope with a similar beam path geometry involving two opposing objectives to optimize resolution when imaging deep in aberrating specimens. The combination of adaptive optics with an improved isoSTED nanoscope allows imaging of cells and tissues with sub-50-nm isotropic resolution.},
  archive      = {J_NMETH},
  author       = {Hao, Xiang and Allgeyer, Edward S. and Lee, Dong-Ryoung and Antonello, Jacopo and Watters, Katherine and Gerdes, Julianne A. and Schroeder, Lena K. and Bottanelli, Francesca and Zhao, Jiaxi and Kidd, Phylicia and Lessard, Mark D. and Rothman, James E. and Cooley, Lynn and Biederer, Thomas and Booth, Martin J. and Bewersdorf, Joerg},
  doi          = {10.1038/s41592-021-01149-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {688-693},
  shortjournal = {Nat. Meth.},
  title        = {Three-dimensional adaptive optical nanoscopy for thick specimen imaging at sub-50-nm resolution},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Three-dimensional residual channel attention networks
denoise and sharpen fluorescence microscopy image volumes.
<em>NMETH</em>, <em>18</em>(6), 678–687. (<a
href="https://doi.org/10.1038/s41592-021-01155-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We demonstrate residual channel attention networks (RCAN) for the restoration and enhancement of volumetric time-lapse (four-dimensional) fluorescence microscopy data. First we modify RCAN to handle image volumes, showing that our network enables denoising competitive with three other state-of-the-art neural networks. We use RCAN to restore noisy four-dimensional super-resolution data, enabling image capture of over tens of thousands of images (thousands of volumes) without apparent photobleaching. Second, using simulations we show that RCAN enables resolution enhancement equivalent to, or better than, other networks. Third, we exploit RCAN for denoising and resolution improvement in confocal microscopy, enabling ~2.5-fold lateral resolution enhancement using stimulated emission depletion microscopy ground truth. Fourth, we develop methods to improve spatial resolution in structured illumination microscopy using expansion microscopy data as ground truth, achieving improvements of ~1.9-fold laterally and ~3.6-fold axially. Finally, we characterize the limits of denoising and resolution enhancement, suggesting practical benchmarks for evaluation and further enhancement of network performance. Three-dimensional residual channel attention networks (RCAN) enable improved image denoising and resolution enhancement on volumetric time-lapse fluorescence microscopy data, allowing longitudinal super-resolution imaging of living samples.},
  archive      = {J_NMETH},
  author       = {Chen, Jiji and Sasaki, Hideki and Lai, Hoyin and Su, Yijun and Liu, Jiamin and Wu, Yicong and Zhovmer, Alexander and Combs, Christian A. and Rey-Suarez, Ivan and Chang, Hung-Yu and Huang, Chi Chou and Li, Xuesong and Guo, Min and Nizambad, Srineil and Upadhyaya, Arpita and Lee, Shih-Jong J. and Lucas, Luciano A. G. and Shroff, Hari},
  doi          = {10.1038/s41592-021-01155-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {678-687},
  shortjournal = {Nat. Meth.},
  title        = {Three-dimensional residual channel attention networks denoise and sharpen fluorescence microscopy image volumes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A pairwise distance distribution correction (DDC) algorithm
to eliminate blinking-caused artifacts in SMLM. <em>NMETH</em>,
<em>18</em>(6), 669–677. (<a
href="https://doi.org/10.1038/s41592-021-01154-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy (SMLM) relies on the blinking behavior of a fluorophore, which is the stochastic switching between fluorescent and dark states. Blinking creates multiple localizations belonging to the same fluorophore, confounding quantitative analyses and interpretations. Here we present a method, termed distance distribution correction (DDC), to eliminate blinking-caused repeat localizations without any additional calibrations. The approach relies on obtaining the true pairwise distance distribution of different fluorophores naturally from the imaging sequence by using distances between localizations separated by a time much longer than the average fluorescence survival time. We show that, using the true pairwise distribution, we can define and maximize the likelihood, obtaining a set of localizations void of blinking artifacts. DDC results in drastic improvements in obtaining the closest estimate of the true spatial organization and number of fluorescent emitters in a wide range of applications, enabling accurate reconstruction and quantification of SMLM images. Distance distribution correction (DDC) eliminates repeat localizations caused by fluorophore blinking without the need for calibrations. Use of DDC yields accurate and quantifiable single-molecule localization microscopy data.},
  archive      = {J_NMETH},
  author       = {Bohrer, Christopher H. and Yang, Xinxing and Thakur, Shreyasi and Weng, Xiaoli and Tenner, Brian and McQuillen, Ryan and Ross, Brian and Wooten, Matthew and Chen, Xin and Zhang, Jin and Roberts, Elijah and Lakadamyali, Melike and Xiao, Jie},
  doi          = {10.1038/s41592-021-01154-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {669-677},
  shortjournal = {Nat. Meth.},
  title        = {A pairwise distance distribution correction (DDC) algorithm to eliminate blinking-caused artifacts in SMLM},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide detection of enhancer-hijacking events from
chromatin interaction data in rearranged genomes. <em>NMETH</em>,
<em>18</em>(6), 661–668. (<a
href="https://doi.org/10.1038/s41592-021-01164-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent efforts have shown that structural variations (SVs) can disrupt three-dimensional genome organization and induce enhancer hijacking, yet no computational tools exist to identify such events from chromatin interaction data. Here, we develop NeoLoopFinder, a computational framework to identify the chromatin interactions induced by SVs, including interchromosomal translocations, large deletions and inversions. Our framework can automatically resolve complex SVs, reconstruct local Hi-C maps surrounding the breakpoints, normalize copy number variation and allele effects and predict chromatin loops induced by SVs. We applied NeoLoopFinder in Hi-C data from 50 cancer cell lines and primary tumors and identified tens of recurrent genes associated with enhancer hijacking. To experimentally validate NeoLoopFinder, we deleted the hijacked enhancers in prostate adenocarcinoma cells using CRISPR–Cas9, which significantly reduced expression of the target oncogene. In summary, NeoLoopFinder enables identification of critical oncogenic regulatory elements that can potentially reveal therapeutic targets. This work presents NeoLoopFinder, a computational method, for identifying chromatin interactions of structurally rearranged genomes. NeoLoopFinder was applied in 50 cancer datasets and identified genes associated with enhancer-hijacking events.},
  archive      = {J_NMETH},
  author       = {Wang, Xiaotao and Xu, Jie and Zhang, Baozhen and Hou, Ye and Song, Fan and Lyu, Huijue and Yue, Feng},
  doi          = {10.1038/s41592-021-01164-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {661-668},
  shortjournal = {Nat. Meth.},
  title        = {Genome-wide detection of enhancer-hijacking events from chromatin interaction data in rearranged genomes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell joint detection of chromatin occupancy and
transcriptome enables higher-dimensional epigenomic reconstructions.
<em>NMETH</em>, <em>18</em>(6), 652–660. (<a
href="https://doi.org/10.1038/s41592-021-01129-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering mechanisms in cell-fate decisions requires single-cell holistic reconstructions of multidimensional epigenomic states in transcriptional regulation. Here we develop CoTECH, a combinatorial barcoding method allowing high-throughput single-cell joint detection of chromatin occupancy and transcriptome. We used CoTECH to examine bivalent histone marks (H3K4me3 and H3K27me3) with transcription from naive to primed mouse embryonic stem cells. We also derived concurrent bivalent marks in pseudosingle cells using transcriptome as an anchor for resolving pseudotemporal bivalency trajectories and disentangling a context-specific interplay between H3K4me3/H3K27me3 and transcription level. Next, we revealed the regulatory basis of endothelial-to-hematopoietic transition in two waves of hematopoietic cells and distinctive enhancer-gene-linking schemes guiding hemogenic endothelial cell emergence, indicating a unique epigenetic control of transcriptional regulation for hematopoietic stem cell priming. CoTECH provides an efficient framework for single-cell coassay of chromatin occupancy and transcription, thus enabling higher-dimensional epigenomic reconstructions. This paper reports CoTECH, which couples chromatin binding enrichment with RNA sequencing for concurrent measurements of histone modification and transcriptome in single cells, offering a multiomics tool for studying epigenetic regulations.},
  archive      = {J_NMETH},
  author       = {Xiong, Haiqing and Luo, Yingjie and Wang, Qianhao and Yu, Xianhong and He, Aibin},
  doi          = {10.1038/s41592-021-01129-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {652-660},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell joint detection of chromatin occupancy and transcriptome enables higher-dimensional epigenomic reconstructions},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Detect-seq reveals out-of-protospacer editing and
target-strand editing by cytosine base editors. <em>NMETH</em>,
<em>18</em>(6), 643–651. (<a
href="https://doi.org/10.1038/s41592-021-01172-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine base editors (CBEs) have the potential to correct human pathogenic point mutations. However, their genome-wide specificity remains poorly understood. Here we report Detect-seq for the evaluation of CBE specificity. It enables sensitive detection of CBE-induced off-target sites at the genome-wide level. Detect-seq leverages chemical labeling and biotin pulldown to trace the editing intermediate deoxyuridine, thereby revealing the editome of CBE. In addition to Cas9-independent and typical Cas9-dependent off-target sites, we discovered edits outside the protospacer sequence (that is, out-of-protospacer) and on the target strand (which pairs with the single-guide RNA). Such unexpected off-target edits are prevalent and can exhibit a high editing ratio, while their occurrences exhibit cell-type dependency and cannot be predicted based on the sgRNA sequence. Moreover, we found out-of-protospacer and target-strand edits nearby the on-target sites tested, challenging the general knowledge that CBEs do not induce proximal off-target mutations. Collectively, our approaches allow unbiased analysis of the CBE editome and provide a widely applicable tool for specificity evaluation of various emerging genome editing tools. Detect-seq, built upon chemical labeling and enrichment of intermediate deoxyuridine, offers an approach to profile different types of off-target mutation induced by cytosine base editors including unexpected edits outside of protospacer and on the target strand.},
  archive      = {J_NMETH},
  author       = {Lei, Zhixin and Meng, Haowei and Lv, Zhicong and Liu, Menghao and Zhao, Huanan and Wu, Hao and Zhang, Xiaoxue and Liu, Lulu and Zhuang, Yuan and Yin, Kailin and Yan, Yongchang and Yi, Chengqi},
  doi          = {10.1038/s41592-021-01172-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {643-651},
  shortjournal = {Nat. Meth.},
  title        = {Detect-seq reveals out-of-protospacer editing and target-strand editing by cytosine base editors},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultra-high-throughput single-cell RNA sequencing and
perturbation screening with combinatorial fluidic indexing.
<em>NMETH</em>, <em>18</em>(6), 635–642. (<a
href="https://doi.org/10.1038/s41592-021-01153-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell atlas projects and high-throughput perturbation screens require single-cell sequencing at a scale that is challenging with current technology. To enable cost-effective single-cell sequencing for millions of individual cells, we developed ‘single-cell combinatorial fluidic indexing’ (scifi). The scifi-RNA-seq assay combines one-step combinatorial preindexing of entire transcriptomes inside permeabilized cells with subsequent single-cell RNA-seq using microfluidics. Preindexing allows us to load several cells per droplet and computationally demultiplex their individual expression profiles. Thereby, scifi-RNA-seq massively increases the throughput of droplet-based single-cell RNA-seq, and provides a straightforward way of multiplexing thousands of samples in a single experiment. Compared with multiround combinatorial indexing, scifi-RNA-seq provides an easy and efficient workflow. Compared to cell hashing methods, which flag and discard droplets containing more than one cell, scifi-RNA-seq resolves and retains individual transcriptomes from overloaded droplets. We benchmarked scifi-RNA-seq on various human and mouse cell lines, validated it for primary human T cells and applied it in a highly multiplexed CRISPR screen with single-cell transcriptome readout of T cell receptor activation. Combining whole-transcriptome preindexing with standard droplet microfluidics, scifi-RNA-seq enables single-cell RNA-seq with massive throughput and built-in sample multiplexing.},
  archive      = {J_NMETH},
  author       = {Datlinger, Paul and Rendeiro, André F. and Boenke, Thorina and Senekowitsch, Martin and Krausgruber, Thomas and Barreca, Daniele and Bock, Christoph},
  doi          = {10.1038/s41592-021-01153-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {635-642},
  shortjournal = {Nat. Meth.},
  title        = {Ultra-high-throughput single-cell RNA sequencing and perturbation screening with combinatorial fluidic indexing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). L1CAM is not associated with extracellular vesicles in human
cerebrospinal fluid or plasma. <em>NMETH</em>, <em>18</em>(6), 631–634.
(<a href="https://doi.org/10.1038/s41592-021-01174-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {L1CAM is a transmembrane protein expressed on neurons that was presumed to be found on neuron-derived extracellular vesicles (NDEVs) in human biofluids. We developed a panel of single-molecule array assays to evaluate the use of L1CAM for NDEV isolation. We demonstrate that L1CAM is not associated with extracellular vesicles in human plasma or cerebrospinal fluid and therefore recommend against its use as a marker in NDEV isolation protocols. L1CAM was presumed to be a marker for neuron-derived extracellular vesicles, but this is now shown not to be the case.},
  archive      = {J_NMETH},
  author       = {Norman, Maia and Ter-Ovanesyan, Dmitry and Trieu, Wendy and Lazarovits, Roey and Kowal, Emma J. K. and Lee, Ju Hyun and Chen-Plotkin, Alice S. and Regev, Aviv and Church, George M. and Walt, David R.},
  doi          = {10.1038/s41592-021-01174-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {631-634},
  shortjournal = {Nat. Meth.},
  title        = {L1CAM is not associated with extracellular vesicles in human cerebrospinal fluid or plasma},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). TRUST4: Immune repertoire reconstruction from bulk and
single-cell RNA-seq data. <em>NMETH</em>, <em>18</em>(6), 627–630. (<a
href="https://doi.org/10.1038/s41592-021-01142-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce the TRUST4 open-source algorithm for reconstruction of immune receptor repertoires in αβ/γδ T cells and B cells from RNA-sequencing (RNA-seq) data. Compared with competing methods, TRUST4 supports both FASTQ and BAM format and is faster and more sensitive in assembling longer—even full-length—receptor repertoires. TRUST4 can also call repertoire sequences from single-cell RNA-seq (scRNA-seq) data without V(D)J enrichment, and is compatible with both SMART-seq and 5′ 10x Genomics platforms. TRUST4 is a computational tool for reconstructing T-cell and B-cell receptor repertoires using bulk and single-cell RNA-seq data.},
  archive      = {J_NMETH},
  author       = {Song, Li and Cohen, David and Ouyang, Zhangyi and Cao, Yang and Hu, Xihao and Liu, X. Shirley},
  doi          = {10.1038/s41592-021-01142-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {627-630},
  shortjournal = {Nat. Meth.},
  title        = {TRUST4: Immune repertoire reconstruction from bulk and single-cell RNA-seq data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Challenges in benchmarking metagenomic profilers.
<em>NMETH</em>, <em>18</em>(6), 618–626. (<a
href="https://doi.org/10.1038/s41592-021-01141-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate microbial identification and abundance estimation are crucial for metagenomics analysis. Various methods for classification of metagenomic data and estimation of taxonomic profiles, broadly referred to as metagenomic profilers, have been developed. Nevertheless, benchmarking of metagenomic profilers remains challenging because some tools are designed to report relative sequence abundance while others report relative taxonomic abundance. Here we show how misleading conclusions can be drawn by neglecting this distinction between relative abundance types when benchmarking metagenomic profilers. Moreover, we show compelling evidence that interchanging sequence abundance and taxonomic abundance will influence both per-sample summary statistics and cross-sample comparisons. We suggest that the microbiome research community pay attention to potentially misleading biological conclusions arising from this issue when benchmarking metagenomic profilers, by carefully considering the type of abundance data that were analyzed and interpreted and clearly stating the strategy used for metagenomic profiling. Many computational tools for metagenomic profiling have been developed, with different algorithms and features. This analysis shows that, when comparing these tools, the distinction of different types of relative sequence abundance should be taken into consideration.},
  archive      = {J_NMETH},
  author       = {Sun, Zheng and Huang, Shi and Zhang, Meng and Zhu, Qiyun and Haiminen, Niina and Carrieri, Anna Paola and Vázquez-Baeza, Yoshiki and Parida, Laxmi and Kim, Ho-Cheol and Knight, Rob and Liu, Yang-Yu},
  doi          = {10.1038/s41592-021-01141-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {618-626},
  shortjournal = {Nat. Meth.},
  title        = {Challenges in benchmarking metagenomic profilers},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The emerging landscape of single-molecule protein sequencing
technologies. <em>NMETH</em>, <em>18</em>(6), 604–617. (<a
href="https://doi.org/10.1038/s41592-021-01143-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell profiling methods have had a profound impact on the understanding of cellular heterogeneity. While genomes and transcriptomes can be explored at the single-cell level, single-cell profiling of proteomes is not yet established. Here we describe new single-molecule protein sequencing and identification technologies alongside innovations in mass spectrometry that will eventually enable broad sequence coverage in single-cell profiling. These technologies will in turn facilitate biological discovery and open new avenues for ultrasensitive disease diagnostics. This Perspective describes new single-molecule protein sequencing and identification technologies alongside innovations in mass spectrometry that will eventually enable broad sequence coverage in single-cell proteomics.},
  archive      = {J_NMETH},
  author       = {Alfaro, Javier Antonio and Bohländer, Peggy and Dai, Mingjie and Filius, Mike and Howard, Cecil J. and van Kooten, Xander F. and Ohayon, Shilo and Pomorski, Adam and Schmid, Sonja and Aksimentiev, Aleksei and Anslyn, Eric V. and Bedran, Georges and Cao, Chan and Chinappi, Mauro and Coyaud, Etienne and Dekker, Cees and Dittmar, Gunnar and Drachman, Nicholas and Eelkema, Rienk and Goodlett, David and Hentz, Sébastien and Kalathiya, Umesh and Kelleher, Neil L. and Kelly, Ryan T. and Kelman, Zvi and Kim, Sung Hyun and Kuster, Bernhard and Rodriguez-Larrea, David and Lindsay, Stuart and Maglia, Giovanni and Marcotte, Edward M. and Marino, John P. and Masselon, Christophe and Mayer, Michael and Samaras, Patroklos and Sarthak, Kumar and Sepiashvili, Lusia and Stein, Derek and Wanunu, Meni and Wilhelm, Mathias and Yin, Peng and Meller, Amit and Joo, Chirlmin},
  doi          = {10.1038/s41592-021-01143-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {604-617},
  shortjournal = {Nat. Meth.},
  title        = {The emerging landscape of single-molecule protein sequencing technologies},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell multiomics to probe relationships between
histone modifications and transcription. <em>NMETH</em>, <em>18</em>(6),
602–603. (<a href="https://doi.org/10.1038/s41592-021-01147-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two new technologies enable the profiling of single cells for RNA transcription as well as modifications to histone proteins. Each takes a similar strategy to capture both properties, but with different methods for cell indexing, and they are applied to two different areas: neuroscience and developmental biology, respectively.},
  archive      = {J_NMETH},
  author       = {Adey, Andrew C.},
  doi          = {10.1038/s41592-021-01147-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {602-603},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell multiomics to probe relationships between histone modifications and transcription},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021u). Tick-tock, it’s RNA o’clock. <em>NMETH</em>,
<em>18</em>(6), 597–601. (<a
href="https://doi.org/10.1038/s41592-021-01180-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There’s an evolving choice of ways to track the temporal dynamic of RNA biographies.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01180-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {597-601},
  shortjournal = {Nat. Meth.},
  title        = {Tick-tock, it’s RNA o’clock},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Organoid systems to recapitulate cervical cancer.
<em>NMETH</em>, <em>18</em>(6), 596. (<a
href="https://doi.org/10.1038/s41592-021-01191-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human cervical organoids mimic dynamics of cancer and sexually transmitted infections.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01191-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {596},
  shortjournal = {Nat. Meth.},
  title        = {Organoid systems to recapitulate cervical cancer},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Integration of massive single-cell datasets.
<em>NMETH</em>, <em>18</em>(6), 595. (<a
href="https://doi.org/10.1038/s41592-021-01190-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The iNMF algorithm efficiently integrates large-scale, continually arriving single-cell multi-omic datasets.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01190-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {595},
  shortjournal = {Nat. Meth.},
  title        = {Integration of massive single-cell datasets},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Accelerating imaging transcriptomics. <em>NMETH</em>,
<em>18</em>(6), 594. (<a
href="https://doi.org/10.1038/s41592-021-01189-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Composite in situ imaging leverages gene expression patterns to improve the efficiency of highly multiplexed single-molecule FISH measurements by orders of magnitude.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01189-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {594},
  shortjournal = {Nat. Meth.},
  title        = {Accelerating imaging transcriptomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Cell–cell interactions revealed with RABID-seq.
<em>NMETH</em>, <em>18</em>(6), 593. (<a
href="https://doi.org/10.1038/s41592-021-01192-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An approach involving barcoded viral tracing provides insights into cell–cell interactions at single-cell resolution.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01192-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {593},
  shortjournal = {Nat. Meth.},
  title        = {Cell–cell interactions revealed with RABID-seq},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Prime editing progress. <em>NMETH</em>, <em>18</em>(6),
592. (<a href="https://doi.org/10.1038/s41592-021-01188-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers work towards improving the efficiency and specificity of prime editors, as well as their delivery system in plants and animals.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01188-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {592},
  shortjournal = {Nat. Meth.},
  title        = {Prime editing progress},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Diversity in immunogenomics: The value and the challenge.
<em>NMETH</em>, <em>18</em>(6), 588–591. (<a
href="https://doi.org/10.1038/s41592-021-01169-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immunogenomics studies have been largely limited to individuals of European ancestry, restricting the ability to identify variation in human adaptive immune responses across populations. Inclusion of a greater diversity of individuals in immunogenomics studies will substantially enhance our understanding of human immunology.},
  archive      = {J_NMETH},
  author       = {Peng, Kerui and Safonova, Yana and Shugay, Mikhail and Popejoy, Alice B. and Rodriguez, Oscar L. and Breden, Felix and Brodin, Petter and Burkhardt, Amanda M. and Bustamante, Carlos and Cao-Lormeau, Van-Mai and Corcoran, Martin M. and Duffy, Darragh and Fuentes-Guajardo, Macarena and Fujita, Ricardo and Greiff, Victor and Jönsson, Vanessa D. and Liu, Xiao and Quintana-Murci, Lluis and Rossetti, Maura and Xie, Jianming and Yaari, Gur and Zhang, Wei and Abedalthagafi, Malak S. and Adekoya, Khalid O. and Ahmed, Rahaman A. and Chang, Wei-Chiao and Gray, Clive and Nakamura, Yusuke and Lees, William D. and Khatri, Purvesh and Alachkar, Houda and Scheepers, Cathrine and Watson, Corey T. and Karlsson Hedestam, Gunilla B. and Mangul, Serghei},
  doi          = {10.1038/s41592-021-01169-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {588-591},
  shortjournal = {Nat. Meth.},
  title        = {Diversity in immunogenomics: The value and the challenge},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Molecular omics resources should require sex annotation: A
call for action. <em>NMETH</em>, <em>18</em>(6), 585–588. (<a
href="https://doi.org/10.1038/s41592-021-01168-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The most commonly used omics databases are a compilation of results from primarily male-only and sex-agnostic studies. The pervasive use of these databases critically hinders progress toward fully accounting for the biology of sex differences.},
  archive      = {J_NMETH},
  author       = {Bond, Kamila M. and McCarthy, Margaret M. and Rubin, Joshua B. and Swanson, Kristin R.},
  doi          = {10.1038/s41592-021-01168-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {585-588},
  shortjournal = {Nat. Meth.},
  title        = {Molecular omics resources should require sex annotation: A call for action},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PYMEVisualize: An open-source tool for exploring 3D
super-resolution data. <em>NMETH</em>, <em>18</em>(6), 582–584. (<a
href="https://doi.org/10.1038/s41592-021-01165-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Marin, Zach and Graff, Michael and Barentine, Andrew E. S. and Soeller, Christian and Chung, Kenny Kwok Hin and Fuentes, Lukas A. and Baddeley, David},
  doi          = {10.1038/s41592-021-01165-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {582-584},
  shortjournal = {Nat. Meth.},
  title        = {PYMEVisualize: An open-source tool for exploring 3D super-resolution data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PANOPLY: A cloud-based platform for automated and
reproducible proteogenomic data analysis. <em>NMETH</em>,
<em>18</em>(6), 580–582. (<a
href="https://doi.org/10.1038/s41592-021-01176-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mani, D. R. and Maynard, Myranda and Kothadia, Ramani and Krug, Karsten and Christianson, Karen E. and Heiman, David and Clauser, Karl R. and Birger, Chet and Getz, Gad and Carr, Steven A.},
  doi          = {10.1038/s41592-021-01176-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {580-582},
  shortjournal = {Nat. Meth.},
  title        = {PANOPLY: A cloud-based platform for automated and reproducible proteogenomic data analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Accounting for diversity. <em>NMETH</em>, <em>18</em>(6),
577. (<a href="https://doi.org/10.1038/s41592-021-01193-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers, funders, database managers and journals all have a role to play in accounting for diversity and prioritizing inclusion at the basic science level.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01193-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {577},
  shortjournal = {Nat. Meth.},
  title        = {Accounting for diversity},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Jie xiao. <em>NMETH</em>, <em>18</em>(6), 579. (<a
href="https://doi.org/10.1038/s41592-021-01181-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reaching ground truth in super-res microscopy, powered by a love of cooking and food.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01181-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {6},
  pages        = {579},
  shortjournal = {Nat. Meth.},
  title        = {Jie xiao},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Retraction note: A fast, high-affinity fluorescent
serotonin biosensor engineered from a tick lipocalin. <em>NMETH</em>,
<em>18</em>(5), 575. (<a
href="https://doi.org/10.1038/s41592-021-01148-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zhang, Shen and Li, Xinyu and Zhao, Shengyu and Drobizhev, Mikhail and Ai, Hui-wang},
  doi          = {10.1038/s41592-021-01148-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {575},
  shortjournal = {Nat. Meth.},
  title        = {Retraction note: A fast, high-affinity fluorescent serotonin biosensor engineered from a tick lipocalin},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Publisher correction: The cryo-EM method microcrystal
electron diffraction (MicroED). <em>NMETH</em>, <em>18</em>(5), 574. (<a
href="https://doi.org/10.1038/s41592-021-01114-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41592-021-01114-6.},
  archive      = {J_NMETH},
  author       = {Nannenga, Brent L. and Gonen, Tamir},
  doi          = {10.1038/s41592-021-01114-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {574},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: The cryo-EM method microcrystal electron diffraction (MicroED)},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Geometric deep learning enables 3D kinematic profiling
across species and environments. <em>NMETH</em>, <em>18</em>(5),
564–573. (<a href="https://doi.org/10.1038/s41592-021-01106-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive descriptions of animal behavior require precise three-dimensional (3D) measurements of whole-body movements. Although two-dimensional approaches can track visible landmarks in restrictive environments, performance drops in freely moving animals, due to occlusions and appearance changes. Therefore, we designed DANNCE to robustly track anatomical landmarks in 3D across species and behaviors. DANNCE uses projective geometry to construct inputs to a convolutional neural network that leverages learned 3D geometric reasoning. We trained and benchmarked DANNCE using a dataset of nearly seven million frames that relates color videos and rodent 3D poses. In rats and mice, DANNCE robustly tracked dozens of landmarks on the head, trunk, and limbs of freely moving animals in naturalistic settings. We extended DANNCE to datasets from rat pups, marmosets, and chickadees, and demonstrate quantitative profiling of behavioral lineage during development. DANNCE enables robust 3D tracking of animals’ limbs and other features in naturalistic environments by making use of a deep learning approach that incorporates geometric reasoning. DANNCE is demonstrated on behavioral sequences from rodents, marmosets, and chickadees.},
  archive      = {J_NMETH},
  author       = {Dunn, Timothy W. and Marshall, Jesse D. and Severson, Kyle S. and Aldarondo, Diego E. and Hildebrand, David G. C. and Chettih, Selmaan N. and Wang, William L. and Gellis, Amanda J. and Carlson, David E. and Aronov, Dmitriy and Freiwald, Winrich A. and Wang, Fan and Ölveczky, Bence P.},
  doi          = {10.1038/s41592-021-01106-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {564-573},
  shortjournal = {Nat. Meth.},
  title        = {Geometric deep learning enables 3D kinematic profiling across species and environments},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning-enhanced light-field imaging with continuous
validation. <em>NMETH</em>, <em>18</em>(5), 557–563. (<a
href="https://doi.org/10.1038/s41592-021-01136-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Visualizing dynamic processes over large, three-dimensional fields of view at high speed is essential for many applications in the life sciences. Light-field microscopy (LFM) has emerged as a tool for fast volumetric image acquisition, but its effective throughput and widespread use in biology has been hampered by a computationally demanding and artifact-prone image reconstruction process. Here, we present a framework for artificial intelligence–enhanced microscopy, integrating a hybrid light-field light-sheet microscope and deep learning–based volume reconstruction. In our approach, concomitantly acquired, high-resolution two-dimensional light-sheet images continuously serve as training data and validation for the convolutional neural network reconstructing the raw LFM data during extended volumetric time-lapse imaging experiments. Our network delivers high-quality three-dimensional reconstructions at video-rate throughput, which can be further refined based on the high-resolution light-sheet images. We demonstrate the capabilities of our approach by imaging medaka heart dynamics and zebrafish neural activity with volumetric imaging rates up to 100 Hz. A deep learning–based algorithm enables efficient reconstruction of light-field microscopy data at video rate. In addition, concurrently acquired light-sheet microscopy data provide ground truth data for training, validation and refinement of the algorithm.},
  archive      = {J_NMETH},
  author       = {Wagner, Nils and Beuttenmueller, Fynn and Norlin, Nils and Gierten, Jakob and Boffi, Juan Carlos and Wittbrodt, Joachim and Weigert, Martin and Hufnagel, Lars and Prevedel, Robert and Kreshuk, Anna},
  doi          = {10.1038/s41592-021-01136-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {557-563},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning-enhanced light-field imaging with continuous validation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Real-time volumetric reconstruction of biological dynamics
with light-field microscopy and deep learning. <em>NMETH</em>,
<em>18</em>(5), 551–556. (<a
href="https://doi.org/10.1038/s41592-021-01058-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light-field microscopy has emerged as a technique of choice for high-speed volumetric imaging of fast biological processes. However, artifacts, nonuniform resolution and a slow reconstruction speed have limited its full capabilities for in toto extraction of dynamic spatiotemporal patterns in samples. Here, we combined a view-channel-depth (VCD) neural network with light-field microscopy to mitigate these limitations, yielding artifact-free three-dimensional image sequences with uniform spatial resolution and high-video-rate reconstruction throughput. We imaged neuronal activities across moving Caenorhabditis elegans and blood flow in a beating zebrafish heart at single-cell resolution with volumetric imaging rates up to 200 Hz. Reconstruction of light-field microscopy data with a deep-learning network achieves high reconstruction speed and reduces artifacts, as illustrated for moving C. elegans and beating zebrafish hearts.},
  archive      = {J_NMETH},
  author       = {Wang, Zhaoqiang and Zhu, Lanxin and Zhang, Hao and Li, Guo and Yi, Chengqiang and Li, Yi and Yang, Yicong and Ding, Yichen and Zhen, Mei and Gao, Shangbang and Hsiai, Tzung K. and Fei, Peng},
  doi          = {10.1038/s41592-021-01058-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {551-556},
  shortjournal = {Nat. Meth.},
  title        = {Real-time volumetric reconstruction of biological dynamics with light-field microscopy and deep learning},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Phasor s-FLIM: A new paradigm for fast and robust spectral
fluorescence lifetime imaging. <em>NMETH</em>, <em>18</em>(5), 542–550.
(<a href="https://doi.org/10.1038/s41592-021-01108-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence lifetime imaging microscopy (FLIM) and spectral imaging are two broadly applied methods for increasing dimensionality in microscopy. However, their combination is typically inefficient and slow in terms of acquisition and processing. By integrating technological and computational advances, we developed a robust and unbiased spectral FLIM (S-FLIM) system. Our method, Phasor S-FLIM, combines true parallel multichannel digital frequency domain electronics with a multidimensional phasor approach to extract detailed and precise information about the photophysics of fluorescent specimens at optical resolution. To show the flexibility of the Phasor S-FLIM technology and its applications to the biological and biomedical field, we address four common, yet challenging, problems: the blind unmixing of spectral and lifetime signatures from multiple unknown species, the unbiased bleedthrough- and background-free Förster resonance energy transfer analysis of biosensors, the photophysical characterization of environment-sensitive probes in living cells and parallel four-color FLIM imaging in tumor spheroids. Phasor S-FLIM combines novel electronics for multichannel fluorescence lifetime acquisition and a phasor-based unmixing algorithm for real-time analysis of reliable spectral lifetime imaging data, enabling new biological observations.},
  archive      = {J_NMETH},
  author       = {Scipioni, Lorenzo and Rossetta, Alessandro and Tedeschi, Giulia and Gratton, Enrico},
  doi          = {10.1038/s41592-021-01108-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {542-550},
  shortjournal = {Nat. Meth.},
  title        = {Phasor S-FLIM: A new paradigm for fast and robust spectral fluorescence lifetime imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A versatile polypharmacology platform promotes
cytoprotection and viability of human pluripotent and differentiated
cells. <em>NMETH</em>, <em>18</em>(5), 528–541. (<a
href="https://doi.org/10.1038/s41592-021-01126-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human pluripotent stem cells (hPSCs) are capable of extensive self-renewal yet remain highly sensitive to environmental perturbations in vitro, posing challenges to their therapeutic use. There is an urgent need to advance strategies that ensure safe and robust long-term growth and functional differentiation of these cells. Here, we deployed high-throughput screening strategies to identify a small-molecule cocktail that improves viability of hPSCs and their differentiated progeny. The combination of chroman 1, emricasan, polyamines, and trans-ISRIB (CEPT) enhanced cell survival of genetically stable hPSCs by simultaneously blocking several stress mechanisms that otherwise compromise cell structure and function. CEPT provided strong improvements for several key applications in stem-cell research, including routine cell passaging, cryopreservation of pluripotent and differentiated cells, embryoid body (EB) and organoid formation, single-cell cloning, and genome editing. Thus, CEPT represents a unique poly-pharmacological strategy for comprehensive cytoprotection, providing a rationale for efficient and safe utilization of hPSCs. The CEPT cocktail comprising four small molecules enhances pluripotent stem cell survival, biobanking, organoid formation, and single-cell cloning efficiency by reducing cellular stress.},
  archive      = {J_NMETH},
  author       = {Chen, Yu and Tristan, Carlos A. and Chen, Lu and Jovanovic, Vukasin M. and Malley, Claire and Chu, Pei-Hsuan and Ryu, Seungmi and Deng, Tao and Ormanoglu, Pinar and Tao, Dingyin and Fang, Yuhong and Slamecka, Jaroslav and Hong, Hyenjong and LeClair, Christopher A. and Michael, Sam and Austin, Christopher P. and Simeonov, Anton and Singeç, Ilyas},
  doi          = {10.1038/s41592-021-01126-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {528-541},
  shortjournal = {Nat. Meth.},
  title        = {A versatile polypharmacology platform promotes cytoprotection and viability of human pluripotent and differentiated cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PCprophet: A framework for protein complex prediction and
differential analysis using proteomic data. <em>NMETH</em>,
<em>18</em>(5), 520–527. (<a
href="https://doi.org/10.1038/s41592-021-01107-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the availability of methods for analyzing protein complexes, systematic analysis of complexes under multiple conditions remains challenging. Approaches based on biochemical fractionation of intact, native complexes and correlation of protein profiles have shown promise. However, most approaches for interpreting cofractionation datasets to yield complex composition and rearrangements between samples depend considerably on protein–protein interaction inference. We introduce PCprophet, a toolkit built on size exclusion chromatography–sequential window acquisition of all theoretical mass spectrometry (SEC-SWATH-MS) data to predict protein complexes and characterize their changes across experimental conditions. We demonstrate improved performance of PCprophet over state-of-the-art approaches and introduce a Bayesian approach to analyze altered protein–protein interactions across conditions. We provide both command-line and graphical interfaces to support the application of PCprophet to any cofractionation MS dataset, independent of separation or quantitative liquid chromatography–MS workflow, for the detection and quantitative tracking of protein complexes and their physiological dynamics. PCprophet combines complex-level scoring and machine learning to predict novel protein complexes from protein cofractionation mass spectrometry data and to perform differential analysis across experimental conditions.},
  archive      = {J_NMETH},
  author       = {Fossati, Andrea and Li, Chen and Uliana, Federico and Wendt, Fabian and Frommelt, Fabian and Sykacek, Peter and Heusel, Moritz and Hallal, Mahmoud and Bludau, Isabell and Capraz, Tümay and Xue, Peng and Song, Jiangning and Wollscheid, Bernd and Purcell, Anthony W. and Gstaiger, Matthias and Aebersold, Ruedi},
  doi          = {10.1038/s41592-021-01107-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {520-527},
  shortjournal = {Nat. Meth.},
  title        = {PCprophet: A framework for protein complex prediction and differential analysis using proteomic data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Robust single-cell discovery of RNA targets of RNA-binding
proteins and ribosomes. <em>NMETH</em>, <em>18</em>(5), 507–519. (<a
href="https://doi.org/10.1038/s41592-021-01128-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) are critical regulators of gene expression and RNA processing that are required for gene function. Yet the dynamics of RBP regulation in single cells is unknown. To address this gap in understanding, we developed STAMP (Surveying Targets by APOBEC-Mediated Profiling), which efficiently detects RBP–RNA interactions. STAMP does not rely on ultraviolet cross-linking or immunoprecipitation and, when coupled with single-cell capture, can identify RBP-specific and cell-type–specific RNA–protein interactions for multiple RBPs and cell types in single, pooled experiments. Pairing STAMP with long-read sequencing yields RBP target sites in an isoform-specific manner. Finally, Ribo-STAMP leverages small ribosomal subunits to measure transcriptome-wide ribosome association in single cells. STAMP enables the study of RBP–RNA interactomes and translational landscapes with unprecedented cellular resolution. Surveying targets by APOBEC-mediated profiling identifies binding sites of RBPs by C-to-U RNA editing. STAMP is isoform specific, can be multiplexed and enables detection of ribosome association in single cells.},
  archive      = {J_NMETH},
  author       = {Brannan, Kristopher W. and Chaim, Isaac A. and Marina, Ryan J. and Yee, Brian A. and Kofman, Eric R. and Lorenz, Daniel A. and Jagannatha, Pratibha and Dong, Kevin D. and Madrigal, Assael A. and Underwood, Jason G. and Yeo, Gene W.},
  doi          = {10.1038/s41592-021-01128-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {507-519},
  shortjournal = {Nat. Meth.},
  title        = {Robust single-cell discovery of RNA targets of RNA-binding proteins and ribosomes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Programmable RNA editing with compact CRISPR–Cas13 systems
from uncultivated microbes. <em>NMETH</em>, <em>18</em>(5), 499–506. (<a
href="https://doi.org/10.1038/s41592-021-01124-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Competitive coevolution between microbes and viruses has led to the diversification of CRISPR–Cas defense systems against infectious agents. By analyzing metagenomic terabase datasets, we identified two compact families (775 to 803 amino acids (aa)) of CRISPR–Cas ribonucleases from hypersaline samples, named Cas13X and Cas13Y. We engineered Cas13X.1 (775 aa) for RNA interference experiments in mammalian cell lines. We found Cas13X.1 could tolerate single-nucleotide mismatches in RNA recognition, facilitating prophylactic RNA virus inhibition. Moreover, a minimal RNA base editor, composed of engineered deaminase (385 aa) and truncated Cas13X.1 (445 aa), exhibited robust editing efficiency and high specificity to induce RNA base conversions. Our results suggest that there exist untapped bacterial defense systems in natural microbes that can function efficiently in mammalian cells, and thus potentially are useful for RNA-editing-based research. This work describes the identification of Cas13 proteins from two families by mining public metagenomic data. The newly identified Cas13X.1 shows efficient target knockdown and can be used to degrade SARS-CoV-2 and H1N1 genomes. In addition, the truncated Cas13X.1 offers an advantage in generating mini-RNA base editors.},
  archive      = {J_NMETH},
  author       = {Xu, Chunlong and Zhou, Yingsi and Xiao, Qingquan and He, Bingbing and Geng, Guannan and Wang, Zikang and Cao, Birong and Dong, Xue and Bai, Weiya and Wang, Yifan and Wang, Xiang and Zhou, Dongming and Yuan, Tanglong and Huo, Xiaona and Lai, Jinsheng and Yang, Hui},
  doi          = {10.1038/s41592-021-01124-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {499-506},
  shortjournal = {Nat. Meth.},
  title        = {Programmable RNA editing with compact CRISPR–Cas13 systems from uncultivated microbes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Discovering multiple types of DNA methylation from bacteria
and microbiome using nanopore sequencing. <em>NMETH</em>,
<em>18</em>(5), 491–498. (<a
href="https://doi.org/10.1038/s41592-021-01109-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacterial DNA methylation occurs at diverse sequence contexts and plays important functional roles in cellular defense and gene regulation. Existing methods for detecting DNA modification from nanopore sequencing data do not effectively support de novo study of unknown bacterial methylomes. In this work, we observed that a nanopore sequencing signal displays complex heterogeneity across methylation events of the same type. To enable nanopore sequencing for broadly applicable methylation discovery, we generated a training dataset from an assortment of bacterial species and developed a method, named nanodisco ( https://github.com/fanglab/nanodisco ), that couples the identification and fine mapping of the three forms of methylation into a multi-label classification framework. We applied it to individual bacteria and the mouse gut microbiome for reliable methylation discovery. In addition, we demonstrated the use of DNA methylation for binning metagenomic contigs, associating mobile genetic elements with their host genomes and identifying misassembled metagenomic contigs. This work describes nanodisco, a tool for de novo identifying DNA methylation in bacterial species and microbiomes using nanopore sequencing and for performing metagenomic binning using microbial DNA methylation patterns.},
  archive      = {J_NMETH},
  author       = {Tourancheau, Alan and Mead, Edward A. and Zhang, Xue-Song and Fang, Gang},
  doi          = {10.1038/s41592-021-01109-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {491-498},
  shortjournal = {Nat. Meth.},
  title        = {Discovering multiple types of DNA methylation from bacteria and microbiome using nanopore sequencing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Comparison of the hi-c, GAM and SPRITE methods using
polymer models of chromatin. <em>NMETH</em>, <em>18</em>(5), 482–490.
(<a href="https://doi.org/10.1038/s41592-021-01135-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hi-C, split-pool recognition of interactions by tag extension (SPRITE) and genome architecture mapping (GAM) are powerful technologies utilized to probe chromatin interactions genome wide, but how faithfully they capture three-dimensional (3D) contacts and how they perform relative to each other is unclear, as no benchmark exists. Here, we compare these methods in silico in a simplified, yet controlled, framework against known 3D structures of polymer models of murine and human loci, which can recapitulate Hi-C, GAM and SPRITE experiments and multiplexed fluorescence in situ hybridization (FISH) single-molecule conformations. We find that in silico Hi-C, GAM and SPRITE bulk data are faithful to the reference 3D structures whereas single-cell data reflect strong variability among single molecules. The minimal number of cells required in replicate experiments to return statistically similar contacts is different across the technologies, being lowest in SPRITE and highest in GAM under the same conditions. Noise-to-signal levels follow an inverse power law with detection efficiency and grow with genomic distance differently among the three methods, being lowest in GAM for genomic separations &amp;gt;1 Mb. This Analysis reports a computational approach to implement Hi-C, SPRITE and GAM, which allows researchers to assess the performances of the three technologies to capture DNA contacts in chromatin three-dimensional models.},
  archive      = {J_NMETH},
  author       = {Fiorillo, Luca and Musella, Francesco and Conte, Mattia and Kempfer, Rieke and Chiariello, Andrea M. and Bianco, Simona and Kukalev, Alexander and Irastorza-Azcarate, Ibai and Esposito, Andrea and Abraham, Alex and Prisco, Antonella and Pombo, Ana and Nicodemi, Mario},
  doi          = {10.1038/s41592-021-01135-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {482-490},
  shortjournal = {Nat. Meth.},
  title        = {Comparison of the hi-c, GAM and SPRITE methods using polymer models of chromatin},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Critical assessment of protein intrinsic disorder
prediction. <em>NMETH</em>, <em>18</em>(5), 472–481. (<a
href="https://doi.org/10.1038/s41592-021-01117-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intrinsically disordered proteins, defying the traditional protein structure–function paradigm, are a challenge to study experimentally. Because a large part of our knowledge rests on computational predictions, it is crucial that their accuracy is high. The Critical Assessment of protein Intrinsic Disorder prediction (CAID) experiment was established as a community-based blind test to determine the state of the art in prediction of intrinsically disordered regions and the subset of residues involved in binding. A total of 43 methods were evaluated on a dataset of 646 proteins from DisProt. The best methods use deep learning techniques and notably outperform physicochemical methods. The top disorder predictor has Fmax = 0.483 on the full dataset and Fmax = 0.792 following filtering out of bona fide structured regions. Disordered binding regions remain hard to predict, with Fmax = 0.231. Interestingly, computing times among methods can vary by up to four orders of magnitude. Results are presented from the first Critical Assessment of protein Intrinsic Disorder prediction (CAID) experiment, a community-based blind test to determine the state of the art in predicting intrinsically disordered regions in proteins.},
  archive      = {J_NMETH},
  author       = {Necci, Marco and Piovesan, Damiano and Tosatto, Silvio C. E.},
  doi          = {10.1038/s41592-021-01117-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {472-481},
  shortjournal = {Nat. Meth.},
  title        = {Critical assessment of protein intrinsic disorder prediction},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Understanding the invisible hands of sample preparation for
cryo-EM. <em>NMETH</em>, <em>18</em>(5), 463–471. (<a
href="https://doi.org/10.1038/s41592-021-01130-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron microscopy (cryo-EM) is rapidly becoming an attractive method in the field of structural biology. With the exploding popularity of cryo-EM, sample preparation must evolve to prevent congestion in the workflow. The dire need for improved microscopy samples has led to a diversification of methods. This Review aims to categorize and explain the principles behind various techniques in the preparation of vitrified samples for the electron microscope. Various aspects and challenges in the workflow are discussed, from sample optimization and carriers to deposition and vitrification. Reliable and versatile specimen preparation remains a challenge, and we hope to give guidelines and posit future directions for improvement. The quality of structural data obtained in cryo-EM is affected by multiple factors pertaining to sample preparation. This Review discusses available techniques and current challenges.},
  archive      = {J_NMETH},
  author       = {Weissenberger, Giulia and Henderikx, Rene J. M. and Peters, Peter J.},
  doi          = {10.1038/s41592-021-01130-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {463-471},
  shortjournal = {Nat. Meth.},
  title        = {Understanding the invisible hands of sample preparation for cryo-EM},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The DANNCE of the rats: A new toolkit for 3D tracking of
animal behavior. <em>NMETH</em>, <em>18</em>(5), 460–462. (<a
href="https://doi.org/10.1038/s41592-021-01110-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new approach tracks animal movements in 3D from multiple camera views using volumetric triangulation, reconciling occlusions and ambiguities present in any one camera view.},
  archive      = {J_NMETH},
  author       = {Karashchuk, Pierre and Tuthill, John C. and Brunton, Bingni W.},
  doi          = {10.1038/s41592-021-01110-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {460-462},
  shortjournal = {Nat. Meth.},
  title        = {The DANNCE of the rats: A new toolkit for 3D tracking of animal behavior},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep-learning-enhanced light-field microscopy.
<em>NMETH</em>, <em>18</em>(5), 459–460. (<a
href="https://doi.org/10.1038/s41592-021-01151-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light-field microscopes can image three-dimensional dynamics of biological samples at unprecedented speed, but the computational reconstruction necessary for image formation is artifact-prone and time-consuming. Deep learning closes this gap between imaging and reconstruction speed.},
  archive      = {J_NMETH},
  author       = {Wang, Kai},
  doi          = {10.1038/s41592-021-01151-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {459-460},
  shortjournal = {Nat. Meth.},
  title        = {Deep-learning-enhanced light-field microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A small-molecule cocktail that beats cellular stress.
<em>NMETH</em>, <em>18</em>(5), 457–458. (<a
href="https://doi.org/10.1038/s41592-021-01127-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CEPT, a small-molecule cocktail, improves the viability of human pluripotent stem cells, protects cells during culture and cryopreservation, and promotes in vitro differentiation and organoid formation.},
  archive      = {J_NMETH},
  author       = {Pal, Rajarshi},
  doi          = {10.1038/s41592-021-01127-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {457-458},
  shortjournal = {Nat. Meth.},
  title        = {A small-molecule cocktail that beats cellular stress},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Benchmarking experiments with polymer modeling.
<em>NMETH</em>, <em>18</em>(5), 456–457. (<a
href="https://doi.org/10.1038/s41592-021-01134-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study applies polymer physics to assess the advantages and limitations of three sequencing-based approaches for determining the structure of genomes and genomic domains.},
  archive      = {J_NMETH},
  author       = {Marti-Renom, Marc A.},
  doi          = {10.1038/s41592-021-01134-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {456-457},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking experiments with polymer modeling},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A community effort to bring structure to disorder.
<em>NMETH</em>, <em>18</em>(5), 454–455. (<a
href="https://doi.org/10.1038/s41592-021-01123-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With protein structure prediction recently getting a seismic boost in accuracy, hopes are also up to better predict unstructured protein regions that can adopt diverse conformations. CAID, a community effort to revive systematic benchmarking, should help.},
  archive      = {J_NMETH},
  author       = {Lang, Benjamin and Babu, M. Madan},
  doi          = {10.1038/s41592-021-01123-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {454-455},
  shortjournal = {Nat. Meth.},
  title        = {A community effort to bring structure to disorder},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021s). Scientists set out to connect the dots on long COVID.
<em>NMETH</em>, <em>18</em>(5), 449–453. (<a
href="https://doi.org/10.1038/s41592-021-01145-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With grit, urgency and creativity, scientists address the puzzle of long-haul COVID.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01145-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {449-453},
  shortjournal = {Nat. Meth.},
  title        = {Scientists set out to connect the dots on long COVID},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). 50 years of PDB: A catalyst in structural biology.
<em>NMETH</em>, <em>18</em>(5), 448–449. (<a
href="https://doi.org/10.1038/s41592-021-01138-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrative structural biology, the culmination of experimental and theoretical methods, will provide a holistic view of molecular processes.},
  archive      = {J_NMETH},
  author       = {Bonvin, Alexandre M. J. J.},
  doi          = {10.1038/s41592-021-01138-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {448-449},
  shortjournal = {Nat. Meth.},
  title        = {50 years of PDB: A catalyst in structural biology},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Genome-wide analysis of structural variation.
<em>NMETH</em>, <em>18</em>(5), 448. (<a
href="https://doi.org/10.1038/s41592-021-01161-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phased genomes from diverse human populations are assembled using multiple sequencing technologies.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01161-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {448},
  shortjournal = {Nat. Meth.},
  title        = {Genome-wide analysis of structural variation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). ISCAT gets a signal boost. <em>NMETH</em>, <em>18</em>(5),
447. (<a href="https://doi.org/10.1038/s41592-021-01160-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photonic resonator interferometric scattering microscopy offers label-free imaging of biomolecules in vitro with improved signal-to-noise ratio.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01160-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {447},
  shortjournal = {Nat. Meth.},
  title        = {ISCAT gets a signal boost},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Integrating experiments, theory and simulations into
whole-cell models. <em>NMETH</em>, <em>18</em>(5), 446–447. (<a
href="https://doi.org/10.1038/s41592-021-01150-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-cell models, still in the early stages of development, are poised to open the next frontier in integrative modeling.},
  archive      = {J_NMETH},
  author       = {Luthey-Schulten, Zaida},
  doi          = {10.1038/s41592-021-01150-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {446-447},
  shortjournal = {Nat. Meth.},
  title        = {Integrating experiments, theory and simulations into whole-cell models},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021u). Walking through chromatin modifications. <em>NMETH</em>,
<em>18</em>(5), 446. (<a
href="https://doi.org/10.1038/s41592-021-01159-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Cell-TALKING technique probes DNA modifications around a histone modification in fixed cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01159-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {446},
  shortjournal = {Nat. Meth.},
  title        = {Walking through chromatin modifications},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). 3D in vitro models of human blastocysts. <em>NMETH</em>,
<em>18</em>(5), 445. (<a
href="https://doi.org/10.1038/s41592-021-01158-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two independent teams have developed the first in vitro human blastocysts.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01158-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {445},
  shortjournal = {Nat. Meth.},
  title        = {3D in vitro models of human blastocysts},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computational protein modeling and the next viral pandemic.
<em>NMETH</em>, <em>18</em>(5), 444–445. (<a
href="https://doi.org/10.1038/s41592-021-01144-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational protein modeling rapidly advances structural knowledge of viral proteins, but methods for modeling protein complexes still need improvement.},
  archive      = {J_NMETH},
  author       = {Narykov, Oleksandr and Srinivasan, Suhas and Korkin, Dmitry},
  doi          = {10.1038/s41592-021-01144-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {444-445},
  shortjournal = {Nat. Meth.},
  title        = {Computational protein modeling and the next viral pandemic},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Studying divergence in human–chimp hybrid cells.
<em>NMETH</em>, <em>18</em>(5), 444. (<a
href="https://doi.org/10.1038/s41592-021-01157-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fusions of human and chimp iPS cells help study how divergence in gene expression affects phenotypic traits.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01157-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {444},
  shortjournal = {Nat. Meth.},
  title        = {Studying divergence in human–chimp hybrid cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Molecular visualization of cellular complexity.
<em>NMETH</em>, <em>18</em>(5), 442–443. (<a
href="https://doi.org/10.1038/s41592-021-01131-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-ET, which yields both structural and spatial information, will help unlock the molecular architecture of cells.},
  archive      = {J_NMETH},
  author       = {Wozny, Michael R. and Kukulski, Wanda},
  doi          = {10.1038/s41592-021-01131-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {442-443},
  shortjournal = {Nat. Meth.},
  title        = {Molecular visualization of cellular complexity},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cryo-electron tomography: Observing the cell at the atomic
level. <em>NMETH</em>, <em>18</em>(5), 440–441. (<a
href="https://doi.org/10.1038/s41592-021-01133-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interest in cryo-ET is rapidly growing, but many technical challenges still require solutions.},
  archive      = {J_NMETH},
  author       = {Li, Xueming},
  doi          = {10.1038/s41592-021-01133-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {440-441},
  shortjournal = {Nat. Meth.},
  title        = {Cryo-electron tomography: Observing the cell at the atomic level},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). RNA structure: A renaissance begins? <em>NMETH</em>,
<em>18</em>(5), 439. (<a
href="https://doi.org/10.1038/s41592-021-01132-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in cryo-EM technology will open a new era of RNA-only 3D structure determination.},
  archive      = {J_NMETH},
  author       = {Das, Rhiju},
  doi          = {10.1038/s41592-021-01132-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {439},
  shortjournal = {Nat. Meth.},
  title        = {RNA structure: A renaissance begins?},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Expanding capabilities and infrastructure for cryo-EM.
<em>NMETH</em>, <em>18</em>(5), 437–438. (<a
href="https://doi.org/10.1038/s41592-021-01139-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many challenges and considerations must be evaluated when expanding and supporting new cryo-electron microscopy facilities.},
  archive      = {J_NMETH},
  author       = {Vinothkumar, Kutti R.},
  doi          = {10.1038/s41592-021-01139-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {437-438},
  shortjournal = {Nat. Meth.},
  title        = {Expanding capabilities and infrastructure for cryo-EM},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). From structure to mechanism: Skiing the energy landscape.
<em>NMETH</em>, <em>18</em>(5), 435–436. (<a
href="https://doi.org/10.1038/s41592-021-01140-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is time for structural biologists to embrace the challenge of quantitatively describing functional energy landscapes.},
  archive      = {J_NMETH},
  author       = {Kern, Dorothee},
  doi          = {10.1038/s41592-021-01140-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {435-436},
  shortjournal = {Nat. Meth.},
  title        = {From structure to mechanism: Skiing the energy landscape},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A new era of synchrotron-enabled macromolecular
crystallography. <em>NMETH</em>, <em>18</em>(5), 433–434. (<a
href="https://doi.org/10.1038/s41592-021-01146-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The future of macromolecular crystallography includes new X-ray sources, enhanced remote-accessible capabilities and time-resolved methods to capture intermediate structures along reaction pathways.},
  archive      = {J_NMETH},
  author       = {Cohen, Aina E.},
  doi          = {10.1038/s41592-021-01146-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {433-434},
  shortjournal = {Nat. Meth.},
  title        = {A new era of synchrotron-enabled macromolecular crystallography},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). WwPDB biocuration: On the front line of structural biology.
<em>NMETH</em>, <em>18</em>(5), 431–432. (<a
href="https://doi.org/10.1038/s41592-021-01137-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biocurators, the backbone of the wwPDB, manage structural biology data deposition, quality, and integrity, and provide integral support to the research community worldwide.},
  archive      = {J_NMETH},
  author       = {Young, Jasmine Y. and Berrisford, John and Chen, Minyu},
  doi          = {10.1038/s41592-021-01137-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {431-432},
  shortjournal = {Nat. Meth.},
  title        = {WwPDB biocuration: On the front line of structural biology},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Mario nicodemi. <em>NMETH</em>, <em>18</em>(5), 429. (<a
href="https://doi.org/10.1038/s41592-021-01152-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polymer physics, comparing ways to assess chromatin structure and taking in tales at the Italian seaside.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01152-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {429},
  shortjournal = {Nat. Meth.},
  title        = {Mario nicodemi},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A celebration of structural biology. <em>NMETH</em>,
<em>18</em>(5), 427. (<a
href="https://doi.org/10.1038/s41592-021-01163-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Protein Data Bank (PDB) is the primary data resource for structural biology. On its 50th anniversary, we celebrate the future of this ever-growing field.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01163-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {427},
  shortjournal = {Nat. Meth.},
  title        = {A celebration of structural biology},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Miniaturized head-mounted microscope for whole-cortex
mesoscale imaging in freely behaving mice. <em>NMETH</em>,
<em>18</em>(4), 417–425. (<a
href="https://doi.org/10.1038/s41592-021-01104-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The advent of genetically encoded calcium indicators, along with surgical preparations such as thinned skulls or refractive-index-matched skulls, has enabled mesoscale cortical activity imaging in head-fixed mice. However, neural activity during unrestrained behavior substantially differs from neural activity in head-fixed animals. For whole-cortex imaging in freely behaving mice, we present the ‘mini-mScope’, a widefield, miniaturized, head-mounted fluorescence microscope that is compatible with transparent polymer skull preparations. With a field of view of 8 × 10 mm2 and weighing less than 4 g, the mini-mScope can image most of the mouse dorsal cortex with resolutions ranging from 39 to 56 µm. We used the mini-mScope to record mesoscale calcium activity across the dorsal cortex during sensory-evoked stimuli, open field behaviors, social interactions and transitions from wakefulness to sleep. The mini-mScope is a miniature microscope that can image neural activity at the mesoscale in most of the dorsal cortex of freely behaving mice.},
  archive      = {J_NMETH},
  author       = {Rynes, Mathew L. and Surinach, Daniel A. and Linn, Samantha and Laroque, Michael and Rajendran, Vijay and Dominguez, Judith and Hadjistamoulou, Orestes and Navabi, Zahra S. and Ghanbari, Leila and Johnson, Gregory W. and Nazari, Mojtaba and Mohajerani, Majid H. and Kodandaramaiah, Suhasa B.},
  doi          = {10.1038/s41592-021-01104-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {417-425},
  shortjournal = {Nat. Meth.},
  title        = {Miniaturized head-mounted microscope for whole-cortex mesoscale imaging in freely behaving mice},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep learning-based point-scanning super-resolution imaging.
<em>NMETH</em>, <em>18</em>(4), 406–416. (<a
href="https://doi.org/10.1038/s41592-021-01080-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Point-scanning imaging systems are among the most widely used tools for high-resolution cellular and tissue imaging, benefiting from arbitrarily defined pixel sizes. The resolution, speed, sample preservation and signal-to-noise ratio (SNR) of point-scanning systems are difficult to optimize simultaneously. We show these limitations can be mitigated via the use of deep learning-based supersampling of undersampled images acquired on a point-scanning system, which we term point-scanning super-resolution (PSSR) imaging. We designed a ‘crappifier’ that computationally degrades high SNR, high-pixel resolution ground truth images to simulate low SNR, low-resolution counterparts for training PSSR models that can restore real-world undersampled images. For high spatiotemporal resolution fluorescence time-lapse data, we developed a ‘multi-frame’ PSSR approach that uses information in adjacent frames to improve model predictions. PSSR facilitates point-scanning image acquisition with otherwise unattainable resolution, speed and sensitivity. All the training data, models and code for PSSR are publicly available at 3DEM.org. Point-scanning super-resolution imaging uses deep learning to supersample undersampled images and enable time-lapse imaging of subcellular events. An accompanying ‘crappifier’ rapidly generates quality training data for robust performance.},
  archive      = {J_NMETH},
  author       = {Fang, Linjing and Monroe, Fred and Novak, Sammy Weiser and Kirk, Lyndsey and Schiavon, Cara R. and Yu, Seungyoon B. and Zhang, Tong and Wu, Melissa and Kastner, Kyle and Latif, Alaa Abdel and Lin, Zijun and Shaw, Andrew and Kubota, Yoshiyuki and Mendenhall, John and Zhang, Zhao and Pekkurnaz, Gulcin and Harris, Kristen and Howard, Jeremy and Manor, Uri},
  doi          = {10.1038/s41592-021-01080-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {406-416},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning-based point-scanning super-resolution imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-molecule FRET imaging of GPCR dimers in living cells.
<em>NMETH</em>, <em>18</em>(4), 397–405. (<a
href="https://doi.org/10.1038/s41592-021-01081-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Class C G protein-coupled receptors (GPCRs) are known to form stable homodimers or heterodimers critical for function, but the oligomeric status of class A and B receptors, which constitute &amp;gt;90\% of all GPCRs, remains hotly debated. Single-molecule fluorescence resonance energy transfer (smFRET) is a powerful approach with the potential to reveal valuable insights into GPCR organization but has rarely been used in living cells to study protein systems. Here, we report generally applicable methods for using smFRET to detect and track transmembrane proteins diffusing within the plasma membrane of mammalian cells. We leverage this in-cell smFRET approach to show agonist-induced structural dynamics within individual metabotropic glutamate receptor dimers. We apply these methods to representative class A, B and C receptors, finding evidence for receptor monomers, density-dependent dimers and constitutive dimers, respectively. A suite of generally applicable methods and tools, developed to enable single-molecule FRET-based studies of transmembrane proteins diffusing in the cell membrane of living cells, was used to study the oligomerization and dynamics of GPCRs.},
  archive      = {J_NMETH},
  author       = {Asher, Wesley B. and Geggier, Peter and Holsey, Michael D. and Gilmore, Grant T. and Pati, Avik K. and Meszaros, Jozsef and Terry, Daniel S. and Mathiasen, Signe and Kaliszewski, Megan J. and McCauley, Mitchell D. and Govindaraju, Alekhya and Zhou, Zhou and Harikumar, Kaleeckal G. and Jaqaman, Khuloud and Miller, Laurence J. and Smith, Adam W. and Blanchard, Scott C. and Javitch, Jonathan A.},
  doi          = {10.1038/s41592-021-01081-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {397-405},
  shortjournal = {Nat. Meth.},
  title        = {Single-molecule FRET imaging of GPCR dimers in living cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Low-n protein engineering with data-efficient deep learning.
<em>NMETH</em>, <em>18</em>(4), 389–396. (<a
href="https://doi.org/10.1038/s41592-021-01100-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein engineering has enormous academic and industrial potential. However, it is limited by the lack of experimental assays that are consistent with the design goal and sufficiently high throughput to find rare, enhanced variants. Here we introduce a machine learning-guided paradigm that can use as few as 24 functionally assayed mutant sequences to build an accurate virtual fitness landscape and screen ten million sequences via in silico directed evolution. As demonstrated in two dissimilar proteins, GFP from Aequorea victoria (avGFP) and E. coli strain TEM-1 β-lactamase, top candidates from a single round are diverse and as active as engineered mutants obtained from previous high-throughput efforts. By distilling information from natural protein sequence landscapes, our model learns a latent representation of ‘unnaturalness’, which helps to guide search away from nonfunctional sequence neighborhoods. Subsequent low-N supervision then identifies improvements to the activity of interest. In sum, our approach enables efficient use of resource-intensive high-fidelity assays without sacrificing throughput, and helps to accelerate engineered proteins into the fermenter, field and clinic. A deep-learning-guided approach enables protein engineering using only a small number (‘low N’) of functionally characterized variants of target proteins.},
  archive      = {J_NMETH},
  author       = {Biswas, Surojit and Khimulya, Grigory and Alley, Ethan C. and Esvelt, Kevin M. and Church, George M.},
  doi          = {10.1038/s41592-021-01100-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {389-396},
  shortjournal = {Nat. Meth.},
  title        = {Low-N protein engineering with data-efficient deep learning},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Martini 3: A general purpose force field for coarse-grained
molecular dynamics. <em>NMETH</em>, <em>18</em>(4), 382–388. (<a
href="https://doi.org/10.1038/s41592-021-01098-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The coarse-grained Martini force field is widely used in biomolecular simulations. Here we present the refined model, Martini 3 ( http://cgmartini.nl ), with an improved interaction balance, new bead types and expanded ability to include specific interactions representing, for example, hydrogen bonding and electronic polarizability. The updated model allows more accurate predictions of molecular packing and interactions in general, which is exemplified with a vast and diverse set of applications, ranging from oil/water partitioning and miscibility data to complex molecular systems, involving protein–protein and protein–lipid interactions and material science applications as ionic liquids and aedamers. Martini 3.0 is an updated and reparametrized force field for coarse-grained molecular dynamics simulations with new bead types and an expanded ability to model molecular packing and interactions.},
  archive      = {J_NMETH},
  author       = {Souza, Paulo C. T. and Alessandri, Riccardo and Barnoud, Jonathan and Thallmair, Sebastian and Faustino, Ignacio and Grünewald, Fabian and Patmanidis, Ilias and Abdizadeh, Haleh and Bruininks, Bart M. H. and Wassenaar, Tsjerk A. and Kroon, Peter C. and Melcr, Josef and Nieto, Vincent and Corradi, Valentina and Khan, Hanif M. and Domański, Jan and Javanainen, Matti and Martinez-Seara, Hector and Reuter, Nathalie and Best, Robert B. and Vattulainen, Ilpo and Monticelli, Luca and Periole, Xavier and Tieleman, D. Peter and de Vries, Alex H. and Marrink, Siewert J.},
  doi          = {10.1038/s41592-021-01098-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {382-388},
  shortjournal = {Nat. Meth.},
  title        = {Martini 3: A general purpose force field for coarse-grained molecular dynamics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A three-dimensional virtual mouse generates synthetic
training data for behavioral analysis. <em>NMETH</em>, <em>18</em>(4),
378–381. (<a href="https://doi.org/10.1038/s41592-021-01103-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a three-dimensional (3D) synthetic animated mouse based on computed tomography scans that is actuated using animation and semirandom, joint-constrained movements to generate synthetic behavioral data with ground-truth label locations. Image-domain translation produced realistic synthetic videos used to train two-dimensional (2D) and 3D pose estimation models with accuracy similar to typical manual training datasets. The outputs from the 3D model-based pose estimation yielded better definition of behavioral clusters than 2D videos and may facilitate automated ethological classification. Bolaños et al. present a realistic three-dimensional virtual mouse model that can be animated and that facilitates the training of pose estimation algorithms.},
  archive      = {J_NMETH},
  author       = {Bolaños, Luis A. and Xiao, Dongsheng and Ford, Nancy L. and LeDue, Jeff M. and Gupta, Pankaj K. and Doebeli, Carlos and Hu, Hao and Rhodin, Helge and Murphy, Timothy H.},
  doi          = {10.1038/s41592-021-01103-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {378-381},
  shortjournal = {Nat. Meth.},
  title        = {A three-dimensional virtual mouse generates synthetic training data for behavioral analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SNT: A unifying toolbox for quantification of neuronal
anatomy. <em>NMETH</em>, <em>18</em>(4), 374–377. (<a
href="https://doi.org/10.1038/s41592-021-01105-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SNT is an end-to-end framework for neuronal morphometry and whole-brain connectomics that supports tracing, proof-editing, visualization, quantification and modeling of neuroanatomy. With an open architecture, a large user base, community-based documentation, support for complex imagery and several model organisms, SNT is a flexible resource for the broad neuroscience community. SNT is both a desktop application and multi-language scripting library, and it is available through the Fiji distribution of ImageJ. SNT is a toolbox neuronal morphometry and connectomics and provides various analysis, visualization, quantification and modeling tools.},
  archive      = {J_NMETH},
  author       = {Arshadi, Cameron and Günther, Ulrik and Eddison, Mark and Harrington, Kyle I. S. and Ferreira, Tiago A.},
  doi          = {10.1038/s41592-021-01105-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {374-377},
  shortjournal = {Nat. Meth.},
  title        = {SNT: A unifying toolbox for quantification of neuronal anatomy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Molecular-scale axial localization by repetitive optical
selective exposure. <em>NMETH</em>, <em>18</em>(4), 369–373. (<a
href="https://doi.org/10.1038/s41592-021-01099-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce an axial localization with repetitive optical selective exposure (ROSE-Z) method for super-resolution imaging. By using an asymmetric optical scheme to generate interference fringes, a &amp;lt;2 nm axial localization precision was achieved with only ~3,000 photons, which is an approximately sixfold improvement compared to previous astigmatism methods. Nanoscale three-dimensional and two-color imaging was demonstrated, illustrating how this method achieves superior performance and facilitates the investigation of cellular nanostructures. ROSE-Z achieves axial interference through an asymmetrical optical scheme, yielding 2 nm axial localization precision with ~3,000 photons and a single objective, which offers improved multicolor three-dimensional localization microscopy for cellular structures.},
  archive      = {J_NMETH},
  author       = {Gu, Lusheng and Li, Yuanyuan and Zhang, Shuwen and Zhou, Maoge and Xue, Yanhong and Li, Weixing and Xu, Tao and Ji, Wei},
  doi          = {10.1038/s41592-021-01099-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {369-373},
  shortjournal = {Nat. Meth.},
  title        = {Molecular-scale axial localization by repetitive optical selective exposure},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Sensitive protein alignments at tree-of-life scale using
DIAMOND. <em>NMETH</em>, <em>18</em>(4), 366–368. (<a
href="https://doi.org/10.1038/s41592-021-01101-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We are at the beginning of a genomic revolution in which all known species are planned to be sequenced. Accessing such data for comparative analyses is crucial in this new age of data-driven biology. Here, we introduce an improved version of DIAMOND that greatly exceeds previous search performances and harnesses supercomputing to perform tree-of-life scale protein alignments in hours, while matching the sensitivity of the gold standard BLASTP. An updated version of DIAMOND uses improved algorithmic procedures and a customized high-performance computing framework to make seemingly prohibitive large-scale protein sequence alignments feasible.},
  archive      = {J_NMETH},
  author       = {Buchfink, Benjamin and Reuter, Klaus and Drost, Hajk-Georg},
  doi          = {10.1038/s41592-021-01101-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {366-368},
  shortjournal = {Nat. Meth.},
  title        = {Sensitive protein alignments at tree-of-life scale using DIAMOND},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Imaging-based screens of pool-synthesized cell libraries.
<em>NMETH</em>, <em>18</em>(4), 358–365. (<a
href="https://doi.org/10.1038/s41592-020-01053-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping a genetic perturbation to a change in phenotype is at the core of biological research. Advances in microscopy have transformed these studies, but they have largely been confined to examining a few strains or cell lines at a time. In parallel, there has been a revolution in creating synthetic libraries of genetically altered cells with relative ease. Here we describe methods that combine these powerful tools to perform live-cell imaging of pool-generated strain libraries for improved biological discovery. This Perspective describes technological advances for imaging-based screening of diverse cellular phenotypes.},
  archive      = {J_NMETH},
  author       = {Lawson, Michael and Elf, Johan},
  doi          = {10.1038/s41592-020-01053-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {358-365},
  shortjournal = {Nat. Meth.},
  title        = {Imaging-based screens of pool-synthesized cell libraries},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Directed evolution in mammalian cells. <em>NMETH</em>,
<em>18</em>(4), 346–357. (<a
href="https://doi.org/10.1038/s41592-021-01090-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed evolution experiments are typically carried out using in vitro systems, bacteria, or yeast—even when the goal is to probe or modulate mammalian biology. Performing directed evolution in systems that do not match the intended mammalian environment severely constrains the scope and functionality of the targets that can be evolved. We review new platforms that are now making it possible to use the mammalian cell itself as the setting for directed evolution and present an overview of frontier challenges and high-impact targets for this approach. This Perspective describes advances that have enabled robust directed evolution in mammalian cells. These approaches are poised to improve the development of new generations of tools to probe or modulate mammalian biology.},
  archive      = {J_NMETH},
  author       = {Hendel, Samuel J. and Shoulders, Matthew D.},
  doi          = {10.1038/s41592-021-01090-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {346-357},
  shortjournal = {Nat. Meth.},
  title        = {Directed evolution in mammalian cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-molecule FRET on its way to structural biology in
live cells. <em>NMETH</em>, <em>18</em>(4), 344–345. (<a
href="https://doi.org/10.1038/s41592-021-01084-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule FRET is making stepwise progress toward the realization of its full potential: becoming the reference technique to monitor protein structural dynamics in live cells.},
  archive      = {J_NMETH},
  author       = {Quast, Robert B. and Margeat, Emmanuel},
  doi          = {10.1038/s41592-021-01084-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {344-345},
  shortjournal = {Nat. Meth.},
  title        = {Single-molecule FRET on its way to structural biology in live cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Martini 3: A coarse-grained force field with an eye for
atomic detail. <em>NMETH</em>, <em>18</em>(4), 342–343. (<a
href="https://doi.org/10.1038/s41592-021-01111-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Souza et al. present the latest release of the popular and widely used force field for near-atomistic coarse-grained biomolecular simulations.},
  archive      = {J_NMETH},
  author       = {Risselada, H. Jelger},
  doi          = {10.1038/s41592-021-01111-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {342-343},
  shortjournal = {Nat. Meth.},
  title        = {Martini 3: A coarse-grained force field with an eye for atomic detail},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Cancer researchers get a grip on immune cell plasticity.
<em>NMETH</em>, <em>18</em>(4), 337–341. (<a
href="https://doi.org/10.1038/s41592-021-01112-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immune cells can transform themselves with plasticity that cancer labs seek to harness.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01112-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {337-341},
  shortjournal = {Nat. Meth.},
  title        = {Cancer researchers get a grip on immune cell plasticity},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). RNA-binding protein kinetics. <em>NMETH</em>,
<em>18</em>(4), 335. (<a
href="https://doi.org/10.1038/s41592-021-01122-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {KIN-CLIP allows determination of binding and dissociation kinetics of RNA-binding proteins in cells.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01122-6},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {335},
  shortjournal = {Nat. Meth.},
  title        = {RNA-binding protein kinetics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021p). Single-cell profiling of microbes. <em>NMETH</em>,
<em>18</em>(4), 334. (<a
href="https://doi.org/10.1038/s41592-021-01120-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The split-pool barcoding strategy has been extended to single-cell RNA sequencing (scRNA-seq) of microbes.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01120-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {334},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell profiling of microbes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). High-resolution optical tweezers. <em>NMETH</em>,
<em>18</em>(4), 333. (<a
href="https://doi.org/10.1038/s41592-021-01121-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Germanium nanoparticles enable optical trapping at high spatiotemporal resolution.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01121-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {333},
  shortjournal = {Nat. Meth.},
  title        = {High-resolution optical tweezers},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Expanding views of the transcriptome. <em>NMETH</em>,
<em>18</em>(4), 332. (<a
href="https://doi.org/10.1038/s41592-021-01119-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In situ long-read sequencing combined with expansion microscopy enables precise views of transcriptomes in intact biological systems.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01119-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {332},
  shortjournal = {Nat. Meth.},
  title        = {Expanding views of the transcriptome},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Recapitulating early cardiogenesis in vitro.
<em>NMETH</em>, <em>18</em>(4), 331. (<a
href="https://doi.org/10.1038/s41592-021-01118-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two independent studies describe the reproduction of early events in cardiac development in vitro.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01118-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {331},
  shortjournal = {Nat. Meth.},
  title        = {Recapitulating early cardiogenesis in vitro},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chunkflow: Hybrid cloud processing of large 3D images by
convolutional nets. <em>NMETH</em>, <em>18</em>(4), 328–330. (<a
href="https://doi.org/10.1038/s41592-021-01088-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Wu, Jingpeng and Silversmith, William M. and Lee, Kisuk and Seung, H. Sebastian},
  doi          = {10.1038/s41592-021-01088-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {328-330},
  shortjournal = {Nat. Meth.},
  title        = {Chunkflow: Hybrid cloud processing of large 3D images by convolutional nets},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). User-friendly, scalable tools and workflows for single-cell
RNA-seq analysis. <em>NMETH</em>, <em>18</em>(4), 327–328. (<a
href="https://doi.org/10.1038/s41592-021-01102-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Moreno, Pablo and Huang, Ni and Manning, Jonathan R. and Mohammed, Suhaib and Solovyev, Andrey and Polanski, Krzysztof and Bacon, Wendi and Chazarra, Ruben and Talavera-López, Carlos and Doyle, Maria A. and Marnier, Guilhem and Grüning, Björn and Rasche, Helena and George, Nancy and Fexova, Silvie Korena and Alibi, Mohamed and Miao, Zhichao and Perez-Riverol, Yasset and Haeussler, Maximilian and Brazma, Alvis and Teichmann, Sarah and Meyer, Kerstin B. and Papatheodorou, Irene},
  doi          = {10.1038/s41592-021-01102-w},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {327-328},
  shortjournal = {Nat. Meth.},
  title        = {User-friendly, scalable tools and workflows for single-cell RNA-seq analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021t). Tiago ferreira. <em>NMETH</em>, <em>18</em>(4), 325. (<a
href="https://doi.org/10.1038/s41592-021-01115-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How computational neuroanatomy, vintage gear and fado fit together.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01115-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {325},
  shortjournal = {Nat. Meth.},
  title        = {Tiago ferreira},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Editing during COVID-19: One year later. <em>NMETH</em>,
<em>18</em>(4), 323–324. (<a
href="https://doi.org/10.1038/s41592-021-01125-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The past year has been long and challenging for all of us. Here’s how Nature Methods has been faring since COVID-19 was declared a pandemic.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01125-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {323-324},
  shortjournal = {Nat. Meth.},
  title        = {Editing during COVID-19: One year later},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: The SEIRS model for infectious disease
dynamics. <em>NMETH</em>, <em>18</em>(3), 321. (<a
href="https://doi.org/10.1038/s41592-021-01079-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41592-021-01079-6.},
  archive      = {J_NMETH},
  author       = {Bjørnstad, Ottar N. and Shea, Katriona and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-021-01079-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {321},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: The SEIRS model for infectious disease dynamics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mass spectrometry imaging of untreated wet cell membranes in
solution using single-layer graphene. <em>NMETH</em>, <em>18</em>(3),
316–320. (<a href="https://doi.org/10.1038/s41592-020-01055-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report a means by which atomic and molecular secondary ions, including cholesterol and fatty acids, can be sputtered through single-layer graphene to enable secondary ion mass spectrometry (SIMS) imaging of untreated wet cell membranes in solution at subcellular spatial resolution. We can observe the intrinsic molecular distribution of lipids, such as cholesterol, phosphoethanolamine and various fatty acids, in untreated wet cell membranes without any labeling. We show that graphene-covered cells prepared on a wet substrate with a cell culture medium reservoir are alive and that their cellular membranes do not disintegrate during SIMS imaging in an ultra-high-vacuum environment. Ab initio molecular dynamics calculations and ion dose-dependence studies suggest that sputtering through single-layer graphene occurs through a transient hole generated in the graphene layer. Cholesterol imaging shows that methyl-β-cyclodextrin preferentially extracts cholesterol molecules from the cholesterol-enriched regions in cell membranes. Application of a single layer of graphene to untreated wet mammalian cells enables mass spectrometry imaging of cellular membranes of live cells in solution at subcellular resolution.},
  archive      = {J_NMETH},
  author       = {Lim, Heejin and Lee, Sun Young and Park, Yereum and Jin, Hyeonggyu and Seo, Daeha and Jang, Yun Hee and Moon, Dae Won},
  doi          = {10.1038/s41592-020-01055-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {316-320},
  shortjournal = {Nat. Meth.},
  title        = {Mass spectrometry imaging of untreated wet cell membranes in solution using single-layer graphene},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-definition imaging using line-illumination modulation
microscopy. <em>NMETH</em>, <em>18</em>(3), 309–315. (<a
href="https://doi.org/10.1038/s41592-021-01074-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The microscopic visualization of large-scale three-dimensional (3D) samples by optical microscopy requires overcoming challenges in imaging quality and speed and in big data acquisition and management. We report a line-illumination modulation (LiMo) technique for imaging thick tissues with high throughput and low background. Combining LiMo with thin tissue sectioning, we further develop a high-definition fluorescent micro-optical sectioning tomography (HD-fMOST) method that features an average signal-to-noise ratio of 110, leading to substantial improvement in neuronal morphology reconstruction. We achieve a &amp;gt;30-fold lossless data compression at a voxel resolution of 0.32 × 0.32 × 1.00 μm3, enabling online data storage to a USB drive or in the cloud, and high-precision (95\% accuracy) brain-wide 3D cell counting in real time. These results highlight the potential of HD-fMOST to facilitate large-scale acquisition and analysis of whole-brain high-resolution datasets. HD-fMOST is a microscopy technique for imaging large samples at high throughput and with high definition, which is achieved with a line-illumination modulation approach. The technology is illustrated by imaging fluorescently labeled neurons in whole mouse brains.},
  archive      = {J_NMETH},
  author       = {Zhong, Qiuyuan and Li, Anan and Jin, Rui and Zhang, Dejie and Li, Xiangning and Jia, Xueyan and Ding, Zhangheng and Luo, Pan and Zhou, Can and Jiang, Chenyu and Feng, Zhao and Zhang, Zhihong and Gong, Hui and Yuan, Jing and Luo, Qingming},
  doi          = {10.1038/s41592-021-01074-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {309-315},
  shortjournal = {Nat. Meth.},
  title        = {High-definition imaging using line-illumination modulation microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Decoding the protein composition of whole nucleosomes with
nuc-MS. <em>NMETH</em>, <em>18</em>(3), 303–308. (<a
href="https://doi.org/10.1038/s41592-020-01052-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current proteomic approaches disassemble and digest nucleosome particles, blurring readouts of the ‘histone code’. To preserve nucleosome-level information, we developed Nuc-MS, which displays the landscape of histone variants and their post-translational modifications (PTMs) in a single mass spectrum. Combined with immunoprecipitation, Nuc-MS quantified nucleosome co-occupancy of histone H3.3 with variant H2A.Z (sixfold over bulk) and the co-occurrence of oncogenic H3.3K27M with euchromatic marks (for example, a &amp;gt;15-fold enrichment of dimethylated H3K79me2). Nuc-MS is highly concordant with chromatin immunoprecipitation-sequencing (ChIP-seq) and offers a new readout of nucleosome-level biology. Nuc-MS makes use of top–down mass spectrometry in ‘native’ mode to quantitatively interrogate histone proteoforms and their post-translational modifications in a single experiment.},
  archive      = {J_NMETH},
  author       = {Schachner, Luis F. and Jooß, Kevin and Morgan, Marc A. and Piunti, Andrea and Meiners, Matthew J. and Kafader, Jared O. and Lee, Alexander S. and Iwanaszko, Marta and Cheek, Marcus A. and Burg, Jonathan M. and Howard, Sarah A. and Keogh, Michael-Christopher and Shilatifard, Ali and Kelleher, Neil L.},
  doi          = {10.1038/s41592-020-01052-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {303-308},
  shortjournal = {Nat. Meth.},
  title        = {Decoding the protein composition of whole nucleosomes with nuc-MS},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Dual DNA and protein tagging of open chromatin unveils
dynamics of epigenomic landscapes in leukemia. <em>NMETH</em>,
<em>18</em>(3), 293–302. (<a
href="https://doi.org/10.1038/s41592-021-01077-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The architecture of chromatin regulates eukaryotic cell states by controlling transcription factor access to sites of gene regulation. Here we describe a dual transposase–peroxidase approach, integrative DNA and protein tagging (iDAPT), which detects both DNA (iDAPT-seq) and protein (iDAPT-MS) associated with accessible regions of chromatin. In addition to direct identification of bound transcription factors, iDAPT enables the inference of their gene regulatory networks, protein interactors and regulation of chromatin accessibility. We applied iDAPT to profile the epigenomic consequences of granulocytic differentiation of acute promyelocytic leukemia, yielding previously undescribed mechanistic insights. Our findings demonstrate the power of iDAPT as a platform for studying the dynamic epigenomic landscapes and their transcription factor components associated with biological phenomena and disease. This work reports a dual transposase–peroxidase fusion to survey the accessible chromatin regions and the proximal proteome in one assay, providing a tool to capture both the genomic and proteomic contents of open chromatin.},
  archive      = {J_NMETH},
  author       = {Lee, Jonathan D. and Paulo, Joao A. and Posey, Ryan R. and Mugoni, Vera and Kong, Nikki R. and Cheloni, Giulia and Lee, Yu-Ru and Slack, Frank J. and Tenen, Daniel G. and Clohessy, John G. and Gygi, Steven P. and Pandolfi, Pier Paolo},
  doi          = {10.1038/s41592-021-01077-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {293-302},
  shortjournal = {Nat. Meth.},
  title        = {Dual DNA and protein tagging of open chromatin unveils dynamics of epigenomic landscapes in leukemia},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Joint profiling of histone modifications and transcriptome
in single cells from mouse brain. <em>NMETH</em>, <em>18</em>(3),
283–292. (<a href="https://doi.org/10.1038/s41592-021-01060-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide profiling of histone modifications can reveal not only the location and activity state of regulatory elements, but also the regulatory mechanisms involved in cell-type-specific gene expression during development and disease pathology. Conventional assays to profile histone modifications in bulk tissues lack single-cell resolution. Here we describe an ultra-high-throughput method, Paired-Tag, for joint profiling of histone modifications and transcriptome in single cells to produce cell-type-resolved maps of chromatin state and transcriptome in complex tissues. We used this method to profile five histone modifications jointly with transcriptome in the adult mouse frontal cortex and hippocampus. Integrative analysis of the resulting maps identified distinct groups of genes subject to divergent epigenetic regulatory mechanisms. Our single-cell multiomics approach enables comprehensive analysis of chromatin state and gene regulation in complex tissues and characterization of gene regulatory programs in the constituent cell types. Paired-Tag offers a multiomics assay for joint profiling of histone modifications and gene expression in single nuclei, and is applied to mouse frontal cortex and hippocampus for measuring cell-type-resolved chromatin state and transcriptome.},
  archive      = {J_NMETH},
  author       = {Zhu, Chenxu and Zhang, Yanxiao and Li, Yang Eric and Lucero, Jacinta and Behrens, M. Margarita and Ren, Bing},
  doi          = {10.1038/s41592-021-01060-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {283-292},
  shortjournal = {Nat. Meth.},
  title        = {Joint profiling of histone modifications and transcriptome in single cells from mouse brain},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Joint probabilistic modeling of single-cell multi-omic data
with totalVI. <em>NMETH</em>, <em>18</em>(3), 272–282. (<a
href="https://doi.org/10.1038/s41592-020-01050-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The paired measurement of RNA and surface proteins in single cells with cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) is a promising approach to connect transcriptional variation with cell phenotypes and functions. However, combining these paired views into a unified representation of cell state is made challenging by the unique technical characteristics of each measurement. Here we present Total Variational Inference (totalVI; https://scvi-tools.org ), a framework for end-to-end joint analysis of CITE-seq data that probabilistically represents the data as a composite of biological and technical factors, including protein background and batch effects. To evaluate totalVI’s performance, we profiled immune cells from murine spleen and lymph nodes with CITE-seq, measuring over 100 surface proteins. We demonstrate that totalVI provides a cohesive solution for common analysis tasks such as dimensionality reduction, the integration of datasets with different measured proteins, estimation of correlations between molecules and differential expression testing. Total Variational Inference is a framework for end-to-end analysis of paired transcriptome and protein measurements such as CITE-seq data in single cells.},
  archive      = {J_NMETH},
  author       = {Gayoso, Adam and Steier, Zoë and Lopez, Romain and Regier, Jeffrey and Nazor, Kristopher L. and Streets, Aaron and Yosef, Nir},
  doi          = {10.1038/s41592-020-01050-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {272-282},
  shortjournal = {Nat. Meth.},
  title        = {Joint probabilistic modeling of single-cell multi-omic data with totalVI},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fast searches of large collections of single-cell data using
scfind. <em>NMETH</em>, <em>18</em>(3), 262–271. (<a
href="https://doi.org/10.1038/s41592-021-01076-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell technologies have made it possible to profile millions of cells, but for these resources to be useful they must be easy to query and access. To facilitate interactive and intuitive access to single-cell data we have developed scfind, a single-cell analysis tool that facilitates fast search of biologically or clinically relevant marker genes in cell atlases. Using transcriptome data from six mouse cell atlases, we show how scfind can be used to evaluate marker genes, perform in silico gating, and identify both cell-type-specific and housekeeping genes. Moreover, we have developed a subquery optimization routine to ensure that long and complex queries return meaningful results. To make scfind more user friendly, we use indices of PubMed abstracts and techniques from natural language processing to allow for arbitrary queries. Finally, we show how scfind can be used for multi-omics analyses by combining single-cell ATAC-seq data with transcriptome data. Advances in single-cell sequencing technologies enable generation of datasets of millions of cells. scfind facilitates efficient and sophisticated gene search in massive single-cell datasets.},
  archive      = {J_NMETH},
  author       = {Lee, Jimmy Tsz Hang and Patikas, Nikolaos and Kiselev, Vladimir Yu and Hemberg, Martin},
  doi          = {10.1038/s41592-021-01076-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {262-271},
  shortjournal = {Nat. Meth.},
  title        = {Fast searches of large collections of single-cell data using scfind},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). RETRACTED ARTICLE: A fast, high-affinity fluorescent
serotonin biosensor engineered from a tick lipocalin. <em>NMETH</em>,
<em>18</em>(3), 258–261. (<a
href="https://doi.org/10.1038/s41592-021-01078-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Serotonin (5-HT) is an important signaling monoamine and neurotransmitter. We report structure-guided engineering of a green fluorescent, genetically encoded serotonin sensor (G-GESS) from a 5-HT-binding lipocalin in the soft tick Argas monolakensis. G-GESS shows fast response kinetics and high affinity, specificity, brightness and photostability. We used G-GESS to image 5-HT dynamics in cultured cells, brain slices and behaving mice. G-GESS is a serotonin biosensor, derived from a tick serotonin-binding protein. The biosensor has been applied here in cell culture, primary neurons, mouse brain slices and in vivo in the mouse brain.},
  archive      = {J_NMETH},
  author       = {Zhang, Shen and Li, Xinyu and Zhao, Shengyu and Drobizhev, Mikhail and Ai, Hui-wang},
  doi          = {10.1038/s41592-021-01078-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {258-261},
  shortjournal = {Nat. Meth.},
  title        = {RETRACTED ARTICLE: A fast, high-affinity fluorescent serotonin biosensor engineered from a tick lipocalin},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Photoblueing of organic dyes can cause artifacts in
super-resolution microscopy. <em>NMETH</em>, <em>18</em>(3), 253–257.
(<a href="https://doi.org/10.1038/s41592-021-01061-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Illumination of fluorophores can induce a loss of the ability to fluoresce, known as photobleaching. Interestingly, some fluorophores photoconvert to a blue-shifted fluorescent molecule as an intermediate on the photobleaching pathway, which can complicate multicolor fluorescence imaging, especially under the intense laser irradiation used in super-resolution fluorescence imaging. Here, we discuss the mechanisms of photoblueing of fluorophores and its impact on fluorescence imaging, and show how it can be prevented. Commonly used organic dyes can photoconvert to blue-shifted fluorescent species, especially under intense illumination. The mechanism of this process and how to avoid unwanted artifacts in super-resolution microscopy are explored here.},
  archive      = {J_NMETH},
  author       = {Helmerich, Dominic A. and Beliu, Gerti and Matikonda, Siddharth S. and Schnermann, Martin J. and Sauer, Markus},
  doi          = {10.1038/s41592-021-01061-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {253-257},
  shortjournal = {Nat. Meth.},
  title        = {Photoblueing of organic dyes can cause artifacts in super-resolution microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-scale deconvolution of RNA structure ensembles.
<em>NMETH</em>, <em>18</em>(3), 249–252. (<a
href="https://doi.org/10.1038/s41592-021-01075-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA structure heterogeneity is a major challenge when querying RNA structures with chemical probing. We introduce DRACO, an algorithm for the deconvolution of coexisting RNA conformations from mutational profiling experiments. Analysis of the SARS-CoV-2 genome using dimethyl sulfate mutational profiling with sequencing (DMS-MaPseq) and DRACO, identifies multiple regions that fold into two mutually exclusive conformations, including a conserved structural switch in the 3′ untranslated region. This work may open the way to dissecting the heterogeneity of the RNA structurome. The DRACO algorithm deconvolutes coexisting RNA structures from mutational profiling experiments, and can be applied to bacterial regulatory structures and elements from the SARS-CoV-2 genome.},
  archive      = {J_NMETH},
  author       = {Morandi, Edoardo and Manfredonia, Ilaria and Simon, Lisa M. and Anselmi, Francesca and van Hemert, Martijn J. and Oliviero, Salvatore and Incarnato, Danny},
  doi          = {10.1038/s41592-021-01075-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {249-252},
  shortjournal = {Nat. Meth.},
  title        = {Genome-scale deconvolution of RNA structure ensembles},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: Limitations of the iterative electron density
reconstruction algorithm from solution scattering data. <em>NMETH</em>,
<em>18</em>(3), 246–248. (<a
href="https://doi.org/10.1038/s41592-021-01083-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Grant, Thomas D.},
  doi          = {10.1038/s41592-021-01083-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {246-248},
  shortjournal = {Nat. Meth.},
  title        = {Reply to: Limitations of the iterative electron density reconstruction algorithm from solution scattering data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Limitations of the iterative electron density reconstruction
algorithm from solution scattering data. <em>NMETH</em>, <em>18</em>(3),
244–245. (<a href="https://doi.org/10.1038/s41592-021-01082-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Konarev, Petr V. and Svergun, Dmitri I.},
  doi          = {10.1038/s41592-021-01082-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {244-245},
  shortjournal = {Nat. Meth.},
  title        = {Limitations of the iterative electron density reconstruction algorithm from solution scattering data},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Molecular cellophane. <em>NMETH</em>, <em>18</em>(3),
242–243. (<a href="https://doi.org/10.1038/s41592-021-01064-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vacuum, a monolayer graphene cover enables imaging mass spectrometry of living, wet cells.},
  archive      = {J_NMETH},
  author       = {Heeren, Ron M. A.},
  doi          = {10.1038/s41592-021-01064-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {242-243},
  shortjournal = {Nat. Meth.},
  title        = {Molecular cellophane},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Making a tick protein talk as a serotonin sensor.
<em>NMETH</em>, <em>18</em>(3), 240–241. (<a
href="https://doi.org/10.1038/s41592-021-01085-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A genetically encoded neurotransmitter sensor, engineered from a tick lipocalin, detects serotonin in the living brain.},
  archive      = {J_NMETH},
  author       = {Morgan, Charles W. and Ren, Jing},
  doi          = {10.1038/s41592-021-01085-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {240-241},
  shortjournal = {Nat. Meth.},
  title        = {Making a tick protein talk as a serotonin sensor},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). Model organisms on roads less traveled. <em>NMETH</em>,
<em>18</em>(3), 235–239. (<a
href="https://doi.org/10.1038/s41592-021-01086-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Beyond the well-known pantheon of model organisms are others. A shift is underway to level the playing field.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01086-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {235-239},
  shortjournal = {Nat. Meth.},
  title        = {Model organisms on roads less traveled},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Bottom-up de novo protein design. <em>NMETH</em>,
<em>18</em>(3), 233. (<a
href="https://doi.org/10.1038/s41592-021-01097-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {TopoBuilder enables de novo design of proteins bearing complex structural motifs.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01097-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {233},
  shortjournal = {Nat. Meth.},
  title        = {Bottom-up de novo protein design},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). Super-resolution 3D live cell imaging. <em>NMETH</em>,
<em>18</em>(3), 232. (<a
href="https://doi.org/10.1038/s41592-021-01096-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {3D pRESOLFT enables volumetric live cell imaging with high spatial and temporal resolution.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01096-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {232},
  shortjournal = {Nat. Meth.},
  title        = {Super-resolution 3D live cell imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021q). Single-cell recording of metastasis. <em>NMETH</em>,
<em>18</em>(3), 231. (<a
href="https://doi.org/10.1038/s41592-021-01095-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metastatic dynamics are revealed by integrating single-cell lineage tracing and RNA sequencing.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01095-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {231},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell recording of metastasis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Ordering protein arrays on cells. <em>NMETH</em>,
<em>18</em>(3), 230. (<a
href="https://doi.org/10.1038/s41592-021-01094-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers report a method to generate two-component protein arrays on living cells, which enables controlled receptor clustering.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01094-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {230},
  shortjournal = {Nat. Meth.},
  title        = {Ordering protein arrays on cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Human tonsils in a dish. <em>NMETH</em>, <em>18</em>(3),
229. (<a href="https://doi.org/10.1038/s41592-021-01093-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human tonsil organoids recapitulate key features of the antigen-specific B cell response.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01093-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {229},
  shortjournal = {Nat. Meth.},
  title        = {Human tonsils in a dish},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Pycro-manager: Open-source software for customized and
reproducible microscope control. <em>NMETH</em>, <em>18</em>(3),
226–228. (<a href="https://doi.org/10.1038/s41592-021-01087-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Pinkard, Henry and Stuurman, Nico and Ivanov, Ivan E. and Anthony, Nicholas M. and Ouyang, Wei and Li, Bin and Yang, Bin and Tsuchida, Mark A. and Chhun, Bryant and Zhang, Grace and Mei, Ryan and Anderson, Michael and Shepherd, Douglas P. and Hunt-Isaak, Ian and Dunn, Raymond L. and Jahr, Wiebke and Kato, Saul and Royer, Loïc A. and Thiagarajah, Jay R. and Eliceiri, Kevin W. and Lundberg, Emma and Mehta, Shalin B. and Waller, Laura},
  doi          = {10.1038/s41592-021-01087-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {226-228},
  shortjournal = {Nat. Meth.},
  title        = {Pycro-manager: Open-source software for customized and reproducible microscope control},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Testing for rare conditions. <em>NMETH</em>,
<em>18</em>(3), 224–225. (<a
href="https://doi.org/10.1038/s41592-021-01089-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {All that glitters is not gold. —William Shakespeare},
  archive      = {J_NMETH},
  author       = {Altman, Naomi and Krzywinski, Martin},
  doi          = {10.1038/s41592-021-01089-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {224-225},
  shortjournal = {Nat. Meth.},
  title        = {Testing for rare conditions},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021q). Qingming luo. <em>NMETH</em>, <em>18</em>(3), 223. (<a
href="https://doi.org/10.1038/s41592-021-01091-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Connecting the Ming dynasty to sharper views in brain imaging.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01091-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {223},
  shortjournal = {Nat. Meth.},
  title        = {Qingming luo},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Let’s chat. <em>NMETH</em>, <em>18</em>(3), 221. (<a
href="https://doi.org/10.1038/s41592-021-01092-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {At Nature Methods we are happy to engage with prospective authors and readers. Here we go through the why, what, who, when and how of talking to journal editors.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01092-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {221},
  shortjournal = {Nat. Meth.},
  title        = {Let’s chat},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Miniature two-photon microscopy for
enlarged field-of-view, multi-plane and long-term brain imaging.
<em>NMETH</em>, <em>18</em>(2), 220. (<a
href="https://doi.org/10.1038/s41592-021-01066-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41592-021-01066-x},
  archive      = {J_NMETH},
  author       = {Zong, Weijian and Wu, Runlong and Chen, Shiyuan and Wu, Junjie and Wang, Hanbin and Zhao, Zhe and Chen, Guoqing and Tu, Rui and Wu, Danlei and Hu, Yanhui and Xu, Yangyang and Wang, Yao and Duan, Zhuoli and Wu, Haitao and Zhang, Yunfeng and Zhang, Jue and Wang, Aimin and Chen, Liangyi and Cheng, Heping},
  doi          = {10.1038/s41592-021-01066-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {220},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Miniature two-photon microscopy for enlarged field-of-view, multi-plane and long-term brain imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: DeepCell kiosk: Scaling deep
learning–enabled cellular image analysis with kubernetes.
<em>NMETH</em>, <em>18</em>(2), 219. (<a
href="https://doi.org/10.1038/s41592-021-01059-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41592-021-01059-w},
  archive      = {J_NMETH},
  author       = {Bannon, Dylan and Moen, Erick and Schwartz, Morgan and Borba, Enrico and Kudo, Takamasa and Greenwald, Noah and Vijayakumar, Vibha and Chang, Brian and Pao, Edward and Osterman, Erik and Graf, William and Van Valen, David},
  doi          = {10.1038/s41592-021-01059-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {219},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: DeepCell kiosk: scaling deep learning–enabled cellular image analysis with kubernetes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021p). Publisher correction: Method of the year: Spatially
resolved transcriptomics. <em>NMETH</em>, <em>18</em>(2), 219. (<a
href="https://doi.org/10.1038/s41592-021-01065-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41592-021-01065-y},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01065-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {219},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: method of the year: spatially resolved transcriptomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: CRISPR-assisted detection of RNA–protein
interactions in living cells. <em>NMETH</em>, <em>18</em>(2), 219. (<a
href="https://doi.org/10.1038/s41592-021-01067-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41592-021-01067-w},
  archive      = {J_NMETH},
  author       = {Yi, Wenkai and Li, Jingyu and Zhu, Xiaoxuan and Wang, Xi and Fan, Ligang and Sun, Wenju and Liao, Linbu and Zhang, Jilin and Li, Xiaoyu and Ye, Jing and Chen, Fulin and Taipale, Jussi and Chan, Kui Ming and Zhang, Liang and Yan, Jian},
  doi          = {10.1038/s41592-021-01067-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {219},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: CRISPR-assisted detection of RNA–protein interactions in living cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Exosome detection via the ultrafast-isolation system:
EXODUS. <em>NMETH</em>, <em>18</em>(2), 212–218. (<a
href="https://doi.org/10.1038/s41592-020-01034-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exosomes have shown great potential in disease diagnostics and therapeutics. However, current isolation approaches are burdensome and suffer from low speed, yield and purity, limiting basic research and clinical applications. Here, we describe an efficient exosome detection method via the ultrafast-isolation system (EXODUS) that allows automated label-free purification of exosomes from varied biofluids. We obtained the ultra-efficient purification of exosomes by negative pressure oscillation and double coupled harmonic oscillator–enabled membrane vibration. Our two coupled oscillators generate dual-frequency transverse waves on the membranes, enabling EXODUS to outperform other isolation techniques in speed, purity and yield. We demonstrated EXODUS by purifying exosomes from urine samples of 113 patients and validated the practical relevance in exosomal RNA profiling with the high-resolution capability and high-throughput analysis. EXODUS is a high-speed isolation method for the enrichment of exosome from biological fluids with high purity and yield.},
  archive      = {J_NMETH},
  author       = {Chen, Yuchao and Zhu, Qingfu and Cheng, Liming and Wang, Yong and Li, Meng and Yang, Qinsi and Hu, Liang and Lou, Doudou and Li, Jiaoyuan and Dong, Xianjun and Lee, Luke P. and Liu, Fei},
  doi          = {10.1038/s41592-020-01034-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {212-218},
  shortjournal = {Nat. Meth.},
  title        = {Exosome detection via the ultrafast-isolation system: EXODUS},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). NnU-net: A self-configuring method for deep learning-based
biomedical image segmentation. <em>NMETH</em>, <em>18</em>(2), 203–211.
(<a href="https://doi.org/10.1038/s41592-020-01008-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biomedical imaging is a driver of scientific discovery and a core component of medical care and is being stimulated by the field of deep learning. While semantic segmentation algorithms enable image analysis and quantification in many applications, the design of respective specialized solutions is non-trivial and highly dependent on dataset properties and hardware conditions. We developed nnU-Net, a deep learning-based segmentation method that automatically configures itself, including preprocessing, network architecture, training and post-processing for any new task. The key design choices in this process are modeled as a set of fixed parameters, interdependent rules and empirical decisions. Without manual intervention, nnU-Net surpasses most existing approaches, including highly specialized solutions on 23 public datasets used in international biomedical segmentation competitions. We make nnU-Net publicly available as an out-of-the-box tool, rendering state-of-the-art segmentation accessible to a broad audience by requiring neither expert knowledge nor computing resources beyond standard network training. nnU-Net is a deep learning-based image segmentation method that automatically configures itself for diverse biological and medical image segmentation tasks. nnU-Net offers state-of-the-art performance as an out-of-the-box tool.},
  archive      = {J_NMETH},
  author       = {Isensee, Fabian and Jaeger, Paul F. and Kohl, Simon A. A. and Petersen, Jens and Maier-Hein, Klaus H.},
  doi          = {10.1038/s41592-020-01008-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {203-211},
  shortjournal = {Nat. Meth.},
  title        = {NnU-net: A self-configuring method for deep learning-based biomedical image segmentation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Evaluation and development of deep neural networks for image
super-resolution in optical microscopy. <em>NMETH</em>, <em>18</em>(2),
194–202. (<a href="https://doi.org/10.1038/s41592-020-01048-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep neural networks have enabled astonishing transformations from low-resolution (LR) to super-resolved images. However, whether, and under what imaging conditions, such deep-learning models outperform super-resolution (SR) microscopy is poorly explored. Here, using multimodality structured illumination microscopy (SIM), we first provide an extensive dataset of LR–SR image pairs and evaluate the deep-learning SR models in terms of structural complexity, signal-to-noise ratio and upscaling factor. Second, we devise the deep Fourier channel attention network (DFCAN), which leverages the frequency content difference across distinct features to learn precise hierarchical representations of high-frequency information about diverse biological structures. Third, we show that DFCAN’s Fourier domain focalization enables robust reconstruction of SIM images under low signal-to-noise ratio conditions. We demonstrate that DFCAN achieves comparable image quality to SIM over a tenfold longer duration in multicolor live-cell imaging experiments, which reveal the detailed structures of mitochondrial cristae and nucleoids and the interaction dynamics of organelles and cytoskeleton. This study explores the performance of deep-learning models for super-resolution imaging and introduces models that utilize frequency content information in the Fourier domain to improve SIM reconstruction under low-SNR conditions.},
  archive      = {J_NMETH},
  author       = {Qiao, Chang and Li, Di and Guo, Yuting and Liu, Chong and Jiang, Tao and Dai, Qionghai and Li, Dong},
  doi          = {10.1038/s41592-020-01048-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {194-202},
  shortjournal = {Nat. Meth.},
  title        = {Evaluation and development of deep neural networks for image super-resolution in optical microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Multi-particle cryo-EM refinement with m visualizes
ribosome-antibiotic complex at 3.5 å in cells. <em>NMETH</em>,
<em>18</em>(2), 186–193. (<a
href="https://doi.org/10.1038/s41592-020-01054-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron microscopy (cryo-EM) enables macromolecular structure determination in vitro and inside cells. In addition to aligning individual particles, accurate registration of sample motion and three-dimensional deformation during exposures are crucial for achieving high-resolution reconstructions. Here we describe M, a software tool that establishes a reference-based, multi-particle refinement framework for cryo-EM data and couples a comprehensive spatial deformation model to in silico correction of electron-optical aberrations. M provides a unified optimization framework for both frame-series and tomographic tilt-series data. We show that tilt-series data can provide the same resolution as frame-series data on a purified protein specimen, indicating that the alignment step no longer limits the resolution obtainable from tomographic data. In combination with Warp and RELION, M resolves to residue level a 70S ribosome bound to an antibiotic inside intact bacterial cells. Our work provides a computational tool that facilitates structural biology in cells. The software M establishes a reference-based multi-particle refinement framework for cryo-EM data. Combined with CTF correction and map denoising, M enables residue-level structure determination inside cells.},
  archive      = {J_NMETH},
  author       = {Tegunov, Dimitry and Xue, Liang and Dienemann, Christian and Cramer, Patrick and Mahamid, Julia},
  doi          = {10.1038/s41592-020-01054-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {186-193},
  shortjournal = {Nat. Meth.},
  title        = {Multi-particle cryo-EM refinement with m visualizes ribosome-antibiotic complex at 3.5 Å in cells},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CryoDRGN: Reconstruction of heterogeneous cryo-EM structures
using neural networks. <em>NMETH</em>, <em>18</em>(2), 176–185. (<a
href="https://doi.org/10.1038/s41592-020-01049-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron microscopy (cryo-EM) single-particle analysis has proven powerful in determining the structures of rigid macromolecules. However, many imaged protein complexes exhibit conformational and compositional heterogeneity that poses a major challenge to existing three-dimensional reconstruction methods. Here, we present cryoDRGN, an algorithm that leverages the representation power of deep neural networks to directly reconstruct continuous distributions of 3D density maps and map per-particle heterogeneity of single-particle cryo-EM datasets. Using cryoDRGN, we uncovered residual heterogeneity in high-resolution datasets of the 80S ribosome and the RAG complex, revealed a new structural state of the assembling 50S ribosome, and visualized large-scale continuous motions of a spliceosome complex. CryoDRGN contains interactive tools to visualize a dataset’s distribution of per-particle variability, generate density maps for exploratory analysis, extract particle subsets for use with other tools and generate trajectories to visualize molecular motions. CryoDRGN is open-source software freely available at http://cryodrgn.csail.mit.edu . CryoDRGN is an unsupervised machine learning algorithm that reconstructs continuous distributions of three-dimensional density maps from heterogeneous single-particle cryo-EM data.},
  archive      = {J_NMETH},
  author       = {Zhong, Ellen D. and Bepler, Tristan and Berger, Bonnie and Davis, Joseph H.},
  doi          = {10.1038/s41592-020-01049-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {176-185},
  shortjournal = {Nat. Meth.},
  title        = {CryoDRGN: Reconstruction of heterogeneous cryo-EM structures using neural networks},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Haplotype-resolved de novo assembly using phased assembly
graphs with hifiasm. <em>NMETH</em>, <em>18</em>(2), 170–175. (<a
href="https://doi.org/10.1038/s41592-020-01056-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Haplotype-resolved de novo assembly is the ultimate solution to the study of sequence variations in a genome. However, existing algorithms either collapse heterozygous alleles into one consensus copy or fail to cleanly separate the haplotypes to produce high-quality phased assemblies. Here we describe hifiasm, a de novo assembler that takes advantage of long high-fidelity sequence reads to faithfully represent the haplotype information in a phased assembly graph. Unlike other graph-based assemblers that only aim to maintain the contiguity of one haplotype, hifiasm strives to preserve the contiguity of all haplotypes. This feature enables the development of a graph trio binning algorithm that greatly advances over standard trio binning. On three human and five nonhuman datasets, including California redwood with a ~30-Gb hexaploid genome, we show that hifiasm frequently delivers better assemblies than existing tools and consistently outperforms others on haplotype-resolved assembly. Hifiasm is a haplotype-resolved de novo genome assembler for long-read high-fidelity sequencing data based on phased assembly graphs.},
  archive      = {J_NMETH},
  author       = {Cheng, Haoyu and Concepcion, Gregory T. and Feng, Xiaowen and Zhang, Haowen and Li, Heng},
  doi          = {10.1038/s41592-020-01056-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {170-175},
  shortjournal = {Nat. Meth.},
  title        = {Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-accuracy long-read amplicon sequences using unique
molecular identifiers with nanopore or PacBio sequencing.
<em>NMETH</em>, <em>18</em>(2), 165–169. (<a
href="https://doi.org/10.1038/s41592-020-01041-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-throughput amplicon sequencing of large genomic regions remains challenging for short-read technologies. Here, we report a high-throughput amplicon sequencing approach combining unique molecular identifiers (UMIs) with Oxford Nanopore Technologies (ONT) or Pacific Biosciences circular consensus sequencing, yielding high-accuracy single-molecule consensus sequences of large genomic regions. We applied our approach to sequence ribosomal RNA operon amplicons (~4,500 bp) and genomic sequences (&amp;gt;10,000 bp) of reference microbial communities in which we observed a chimera rate &amp;lt;0.02\%. To reach a mean UMI consensus error rate &amp;lt;0.01\%, a UMI read coverage of 15× (ONT R10.3), 25× (ONT R9.4.1) and 3× (Pacific Biosciences circular consensus sequencing) is needed, which provides a mean error rate of 0.0042\%, 0.0041\% and 0.0007\%, respectively. This work presents a sequencing strategy based on unique molecular identifiers that improves long-read consensus sequence accuracy of targeted amplicons as well as shotgun whole-genome fragments.},
  archive      = {J_NMETH},
  author       = {Karst, Søren M. and Ziels, Ryan M. and Kirkegaard, Rasmus H. and Sørensen, Emil A. and McDonald, Daniel and Zhu, Qiyun and Knight, Rob and Albertsen, Mads},
  doi          = {10.1038/s41592-020-01041-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {165-169},
  shortjournal = {Nat. Meth.},
  title        = {High-accuracy long-read amplicon sequences using unique molecular identifiers with nanopore or PacBio sequencing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cryo-EM model validation recommendations based on outcomes
of the 2019 EMDataResource challenge. <em>NMETH</em>, <em>18</em>(2),
156–164. (<a href="https://doi.org/10.1038/s41592-020-01051-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This paper describes outcomes of the 2019 Cryo-EM Model Challenge. The goals were to (1) assess the quality of models that can be produced from cryogenic electron microscopy (cryo-EM) maps using current modeling software, (2) evaluate reproducibility of modeling results from different software developers and users and (3) compare performance of current metrics used for model evaluation, particularly Fit-to-Map metrics, with focus on near-atomic resolution. Our findings demonstrate the relatively high accuracy and reproducibility of cryo-EM models derived by 13 participating teams from four benchmark maps, including three forming a resolution series (1.8 to 3.1 Å). The results permit specific recommendations to be made about validating near-atomic cryo-EM structures both in the context of individual experiments and structure data archives such as the Protein Data Bank. We recommend the adoption of multiple scoring parameters to provide full and objective annotation and assessment of the model, reflective of the observed cryo-EM map density. A multi-laboratory study in the form of a community challenge assesses the quality of models that can be produced from cryo-EM maps using different software tools, the reproducibility of models generated by different users and the performance of metrics used for model validation.},
  archive      = {J_NMETH},
  author       = {Lawson, Catherine L. and Kryshtafovych, Andriy and Adams, Paul D. and Afonine, Pavel V. and Baker, Matthew L. and Barad, Benjamin A. and Bond, Paul and Burnley, Tom and Cao, Renzhi and Cheng, Jianlin and Chojnowski, Grzegorz and Cowtan, Kevin and Dill, Ken A. and DiMaio, Frank and Farrell, Daniel P. and Fraser, James S. and Herzik, Mark A. and Hoh, Soon Wen and Hou, Jie and Hung, Li-Wei and Igaev, Maxim and Joseph, Agnel P. and Kihara, Daisuke and Kumar, Dilip and Mittal, Sumit and Monastyrskyy, Bohdan and Olek, Mateusz and Palmer, Colin M. and Patwardhan, Ardan and Perez, Alberto and Pfab, Jonas and Pintilie, Grigore D. and Richardson, Jane S. and Rosenthal, Peter B. and Sarkar, Daipayan and Schäfer, Luisa U. and Schmid, Michael F. and Schröder, Gunnar F. and Shekhar, Mrinal and Si, Dong and Singharoy, Abishek and Terashi, Genki and Terwilliger, Thomas C. and Vaiana, Andrea and Wang, Liguo and Wang, Zhe and Wankowicz, Stephanie A. and Williams, Christopher J. and Winn, Martyn and Wu, Tianqi and Yu, Xiaodi and Zhang, Kaiming and Berman, Helen M. and Chiu, Wah},
  doi          = {10.1038/s41592-020-01051-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {156-164},
  shortjournal = {Nat. Meth.},
  title        = {Cryo-EM model validation recommendations based on outcomes of the 2019 EMDataResource challenge},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A practical guide to cancer subclonal reconstruction from
DNA sequencing. <em>NMETH</em>, <em>18</em>(2), 144–155. (<a
href="https://doi.org/10.1038/s41592-020-01013-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Subclonal reconstruction from bulk tumor DNA sequencing has become a pillar of cancer evolution studies, providing insight into the clonality and relative ordering of mutations and mutational processes. We provide an outline of the complex computational approaches used for subclonal reconstruction from single and multiple tumor samples. We identify the underlying assumptions and uncertainties in each step and suggest best practices for analysis and quality assessment. This guide provides a pragmatic resource for the growing user community of subclonal reconstruction methods. The Perspective reviews the computational methods for subclonal reconstruction and recommends best practices for analyses and quality assessments.},
  archive      = {J_NMETH},
  author       = {Tarabichi, Maxime and Salcedo, Adriana and Deshwar, Amit G. and Ni Leathlobhair, Máire and Wintersinger, Jeff and Wedge, David C. and Van Loo, Peter and Morris, Quaid D. and Boutros, Paul C.},
  doi          = {10.1038/s41592-020-01013-2},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {144-155},
  shortjournal = {Nat. Meth.},
  title        = {A practical guide to cancer subclonal reconstruction from DNA sequencing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deciphering molecular interactions by proximity labeling.
<em>NMETH</em>, <em>18</em>(2), 133–143. (<a
href="https://doi.org/10.1038/s41592-020-01010-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many biological processes are executed and regulated through the molecular interactions of proteins and nucleic acids. Proximity labeling (PL) is a technology for tagging the endogenous interaction partners of specific protein ‘baits’, via genetic fusion to promiscuous enzymes that catalyze the generation of diffusible reactive species in living cells. Tagged molecules that interact with baits can then be enriched and identified by mass spectrometry or nucleic acid sequencing. Here we review the development of PL technologies and highlight studies that have applied PL to the discovery and analysis of molecular interactions. In particular, we focus on the use of PL for mapping protein–protein, protein–RNA and protein–DNA interactions in living cells and organisms. This Review describes proximity labeling methods that make use of peroxidases (APEX) or biotin ligases (TurboID, BioID), and their applications to studying protein–protein and protein–nucleic acid interactions in living systems.},
  archive      = {J_NMETH},
  author       = {Qin, Wei and Cho, Kelvin F. and Cavanagh, Peter E. and Ting, Alice Y.},
  doi          = {10.1038/s41592-020-01010-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {133-143},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering molecular interactions by proximity labeling},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The promise and peril of deep learning in microscopy.
<em>NMETH</em>, <em>18</em>(2), 131–132. (<a
href="https://doi.org/10.1038/s41592-020-01035-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hoffman, David P. and Slavitt, Isaac and Fitzpatrick, Casey A.},
  doi          = {10.1038/s41592-020-01035-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {131-132},
  shortjournal = {Nat. Meth.},
  title        = {The promise and peril of deep learning in microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improving cryo-EM structure validation. <em>NMETH</em>,
<em>18</em>(2), 130–131. (<a
href="https://doi.org/10.1038/s41592-021-01062-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A community-wide challenge yields recommendations for improving cryo-EM structure validation.},
  archive      = {J_NMETH},
  author       = {Rohou, Alexis},
  doi          = {10.1038/s41592-021-01062-1},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {130-131},
  shortjournal = {Nat. Meth.},
  title        = {Improving cryo-EM structure validation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Long road to long-read assembly. <em>NMETH</em>,
<em>18</em>(2), 125–129. (<a
href="https://doi.org/10.1038/s41592-021-01057-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome assembly projects get a boost from high-accuracy long-read sequencing.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01057-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {125-129},
  shortjournal = {Nat. Meth.},
  title        = {Long road to long-read assembly},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Real-time behavioral analysis. <em>NMETH</em>,
<em>18</em>(2), 123. (<a
href="https://doi.org/10.1038/s41592-021-01072-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DeepLabCut-Live! performs pose estimation in real time and allows closed-loop feedback on behavior.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01072-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {123},
  shortjournal = {Nat. Meth.},
  title        = {Real-time behavioral analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). DNA-PAINT takes a left turn. <em>NMETH</em>,
<em>18</em>(2), 122. (<a
href="https://doi.org/10.1038/s41592-021-01071-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA-PAINT with left-handed DNA oligomers enables imaging of nuclear structures with reduced background.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01071-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {122},
  shortjournal = {Nat. Meth.},
  title        = {DNA-PAINT takes a left turn},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021o). Sequencing DNA bendability. <em>NMETH</em>, <em>18</em>(2),
121. (<a href="https://doi.org/10.1038/s41592-021-01070-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Loop-seq is a high-throughput sequencing assay that measures DNA looping and can help explain how DNA bendability contributes to nucleosome organization.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01070-1},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {121},
  shortjournal = {Nat. Meth.},
  title        = {Sequencing DNA bendability},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Probing dynamic proteomes. <em>NMETH</em>, <em>18</em>(2),
120. (<a href="https://doi.org/10.1038/s41592-021-01069-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global protein structure readouts help detect protein functional changes.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01069-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {120},
  shortjournal = {Nat. Meth.},
  title        = {Probing dynamic proteomes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Assembling organs in vitro. <em>NMETH</em>, <em>18</em>(2),
119. (<a href="https://doi.org/10.1038/s41592-021-01068-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assembloids recapitulate organ spatial organization and interaction with high fidelity.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01068-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {119},
  shortjournal = {Nat. Meth.},
  title        = {Assembling organs in vitro},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Julia mahamid. <em>NMETH</em>, <em>18</em>(2), 117. (<a
href="https://doi.org/10.1038/s41592-021-01063-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Jumping down rabbit holes for a 3.5-Ångstrom peek at molecules.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01063-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {117},
  shortjournal = {Nat. Meth.},
  title        = {Julia mahamid},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Open access at nature methods. <em>NMETH</em>,
<em>18</em>(2), 115. (<a
href="https://doi.org/10.1038/s41592-021-01073-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New year, big changes: Nature Methods now offers authors the ability to publish research papers on an open access basis, including via a Guided Open Access pilot. Here’s how it works.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01073-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {115},
  shortjournal = {Nat. Meth.},
  title        = {Open access at nature methods},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Highly specific multiplexed RNA imaging
in tissues with split-FISH. <em>NMETH</em>, <em>18</em>(1), 114. (<a
href="https://doi.org/10.1038/s41592-020-01019-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Goh, Jolene Jie Lin and Chou, Nigel and Seow, Wan Yi and Ha, Norbert and Cheng, Chung Pui Paul and Chang, Yun-Ching and Zhao, Ziqing Winston and Chen, Kok Hao},
  doi          = {10.1038/s41592-020-01019-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {114},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Highly specific multiplexed RNA imaging in tissues with split-FISH},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Click-ExM enables expansion microscopy for all biomolecules.
<em>NMETH</em>, <em>18</em>(1), 107–113. (<a
href="https://doi.org/10.1038/s41592-020-01005-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expansion microscopy (ExM) allows super-resolution imaging on conventional fluorescence microscopes, but has been limited to proteins and nucleic acids. Here we develop click-ExM, which integrates click labeling into ExM to enable a ‘one-stop-shop’ method for nanoscale imaging of various types of biomolecule. By click labeling with biotin and staining with fluorescently labeled streptavidin, a large range of biomolecules can be imaged by the standard ExM procedure normally used for proteins. Using 18 clickable labels, we demonstrate click-ExM on lipids, glycans, proteins, DNA, RNA and small molecules. We demonstrate that click-ExM is applicable in cell culture systems and for tissue imaging. We further show that click-ExM is compatible with signal-amplification techniques and two-color imaging. Click-ExM thus provides a convenient and versatile method for super-resolution imaging, which may be routinely used for cell and tissue samples. Click-ExM uses click-chemistry-based labeling to increase the versatility of expansion microscopy. Click-ExM enables imaging of numerous classes of biomolecules including lipids, glycans, proteins, DNA, RNA and small molecules.},
  archive      = {J_NMETH},
  author       = {Sun, De-en and Fan, Xinqi and Shi, Yujie and Zhang, Hao and Huang, Zhimin and Cheng, Bo and Tang, Qi and Li, Wei and Zhu, Yuntao and Bai, Jinyi and Liu, Wei and Li, Yuan and Wang, Xiaoting and Lei, Xiaoguang and Chen, Xing},
  doi          = {10.1038/s41592-020-01005-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {107-113},
  shortjournal = {Nat. Meth.},
  title        = {Click-ExM enables expansion microscopy for all biomolecules},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cellpose: A generalist algorithm for cellular segmentation.
<em>NMETH</em>, <em>18</em>(1), 100–106. (<a
href="https://doi.org/10.1038/s41592-020-01018-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many biological applications require the segmentation of cell bodies, membranes and nuclei from microscopy images. Deep learning has enabled great progress on this problem, but current methods are specialized for images that have large training datasets. Here we introduce a generalist, deep learning-based segmentation method called Cellpose, which can precisely segment cells from a wide range of image types and does not require model retraining or parameter adjustments. Cellpose was trained on a new dataset of highly varied images of cells, containing over 70,000 segmented objects. We also demonstrate a three-dimensional (3D) extension of Cellpose that reuses the two-dimensional (2D) model and does not require 3D-labeled data. To support community contributions to the training data, we developed software for manual labeling and for curation of the automated results. Periodically retraining the model on the community-contributed data will ensure that Cellpose improves constantly. Cellpose is a generalist, deep learning-based approach for segmenting structures in a wide range of image types. Cellpose does not require parameter adjustment or model retraining and outperforms established methods on 2D and 3D datasets.},
  archive      = {J_NMETH},
  author       = {Stringer, Carsen and Wang, Tim and Michaelos, Michalis and Pachitariu, Marius},
  doi          = {10.1038/s41592-020-01018-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {100-106},
  shortjournal = {Nat. Meth.},
  title        = {Cellpose: A generalist algorithm for cellular segmentation},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mapping the functional landscape of t cell receptor
repertoires by single-t cell transcriptomics. <em>NMETH</em>,
<em>18</em>(1), 92–99. (<a
href="https://doi.org/10.1038/s41592-020-01020-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many experimental and bioinformatics approaches have been developed to characterize the human T cell receptor (TCR) repertoire. However, the unknown functional relevance of TCR profiling hinders unbiased interpretation of the biology of T cells. To address this inadequacy, we developed tessa, a tool to integrate TCRs with gene expression of T cells to estimate the effect that TCRs confer on the phenotypes of T cells. Tessa leveraged techniques combining single-cell RNA-sequencing with TCR sequencing. We validated tessa and showed its superiority over existing approaches that investigate only the TCR sequences. With tessa, we demonstrated that TCR similarity constrains the phenotypes of T cells to be similar and dictates a gradient in antigen targeting efficiency of T cell clonotypes with convergent TCRs. We showed this constraint could predict a functional dichotomization of T cells postimmunotherapy treatment and is weakened in tumor contexts. Tessa, a Bayesian model, integrates TCR sequencing data with gene expression data to analyze the effect of the TCR repertoire on T cell functionality.},
  archive      = {J_NMETH},
  author       = {Zhang, Ze and Xiong, Danyi and Wang, Xinlei and Liu, Hongyu and Wang, Tao},
  doi          = {10.1038/s41592-020-01020-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {92-99},
  shortjournal = {Nat. Meth.},
  title        = {Mapping the functional landscape of t cell receptor repertoires by single-T cell transcriptomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cell surface thermal proteome profiling tracks perturbations
and drug targets on the plasma membrane. <em>NMETH</em>, <em>18</em>(1),
84–91. (<a href="https://doi.org/10.1038/s41592-020-01022-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerous drugs and endogenous ligands bind to cell surface receptors leading to modulation of downstream signaling cascades and frequently to adaptation of the plasma membrane proteome. In-depth analysis of dynamic processes at the cell surface is challenging due to biochemical properties and low abundances of plasma membrane proteins. Here we introduce cell surface thermal proteome profiling for the comprehensive characterization of ligand-induced changes in protein abundances and thermal stabilities at the plasma membrane. We demonstrate drug binding to extracellular receptors and transporters, discover stimulation-dependent remodeling of T cell receptor complexes and describe a competition-based approach to measure target engagement of G-protein-coupled receptor antagonists. Remodeling of the plasma membrane proteome in response to treatment with the TGFB receptor inhibitor SB431542 leads to partial internalization of the monocarboxylate transporters MCT1/3 explaining the antimetastatic effects of the drug. Cell surface thermal proteome profiling allows characterization of ligand-induced changes in protein abundances and thermal stabilities at the plasma membrane.},
  archive      = {J_NMETH},
  author       = {Kalxdorf, Mathias and Günthner, Ina and Becher, Isabelle and Kurzawa, Nils and Knecht, Sascha and Savitski, Mikhail M. and Eberl, H. Christian and Bantscheff, Marcus},
  doi          = {10.1038/s41592-020-01022-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {84-91},
  shortjournal = {Nat. Meth.},
  title        = {Cell surface thermal proteome profiling tracks perturbations and drug targets on the plasma membrane},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Defining the carrier proteome limit for single-cell
proteomics. <em>NMETH</em>, <em>18</em>(1), 76–83. (<a
href="https://doi.org/10.1038/s41592-020-01002-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell proteomics by mass spectrometry (SCoPE-MS) is a recently introduced method to quantify multiplexed single-cell proteomes. While this technique has generated great excitement, the underlying technologies (isobaric labeling and mass spectrometry (MS)) have technical limitations with the potential to affect data quality and biological interpretation. These limitations are particularly relevant when a carrier proteome, a sample added at 25–500× the amount of a single-cell proteome, is used to enable peptide identifications. Here we perform controlled experiments with increasing carrier proteome amounts and evaluate quantitative accuracy, as it relates to mass analyzer dynamic range, multiplexing level and number of ions sampled. We demonstrate that an increase in carrier proteome level requires a concomitant increase in the number of ions sampled to maintain quantitative accuracy. Lastly, we introduce Single-Cell Proteomics Companion (SCPCompanion), a software tool that enables rapid evaluation of single-cell proteomic data and recommends instrument and data analysis parameters for improved data quality. A careful analysis of how carrier proteome levels used in the SCoPE-MS method affect the quantitative accuracy of single-cell proteomics results, yields guidelines for method users.},
  archive      = {J_NMETH},
  author       = {Cheung, Tommy K. and Lee, Chien-Yun and Bayer, Florian P. and McCoy, Atticus and Kuster, Bernhard and Rose, Christopher M.},
  doi          = {10.1038/s41592-020-01002-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {76-83},
  shortjournal = {Nat. Meth.},
  title        = {Defining the carrier proteome limit for single-cell proteomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A “build and retrieve” methodology to simultaneously solve
cryo-EM structures of membrane proteins. <em>NMETH</em>, <em>18</em>(1),
69–75. (<a href="https://doi.org/10.1038/s41592-020-01021-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-particle cryo-electron microscopy (cryo-EM) has become a powerful technique in the field of structural biology. However, the inability to reliably produce pure, homogeneous membrane protein samples hampers the progress of their structural determination. Here, we develop a bottom-up iterative method, Build and Retrieve (BaR), that enables the identification and determination of cryo-EM structures of a variety of inner and outer membrane proteins, including membrane protein complexes of different sizes and dimensions, from a heterogeneous, impure protein sample. We also use the BaR methodology to elucidate structural information from Escherichia coli K12 crude membrane and raw lysate. The findings demonstrate that it is possible to solve high-resolution structures of a number of relatively small (&amp;lt;100 kDa) and less abundant (&amp;lt;10\%) unidentified membrane proteins within a single, heterogeneous sample. Importantly, these results highlight the potential of cryo-EM for systems structural proteomics. The iterative Build and Retrieve (BaR) methodology facilitates the solving of cryo-EM structures of multiple membrane (and soluble) proteins simultaneously, including small and low-abundance membrane proteins.},
  archive      = {J_NMETH},
  author       = {Su, Chih-Chia and Lyu, Meinan and Morgan, Christopher E. and Bolla, Jani Reddy and Robinson, Carol V. and Yu, Edward W.},
  doi          = {10.1038/s41592-020-01021-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {69-75},
  shortjournal = {Nat. Meth.},
  title        = {A ‘Build and retrieve’ methodology to simultaneously solve cryo-EM structures of membrane proteins},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Megabodies expand the nanobody toolkit for protein structure
determination by single-particle cryo-EM. <em>NMETH</em>,
<em>18</em>(1), 60–68. (<a
href="https://doi.org/10.1038/s41592-020-01001-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanobodies are popular and versatile tools for structural biology. They have a compact single immunoglobulin domain organization, bind target proteins with high affinities while reducing their conformational heterogeneity and stabilize multi-protein complexes. Here we demonstrate that engineered nanobodies can also help overcome two major obstacles that limit the resolution of single-particle cryo-electron microscopy reconstructions: particle size and preferential orientation at the water–air interfaces. We have developed and characterized constructs, termed megabodies, by grafting nanobodies onto selected protein scaffolds to increase their molecular weight while retaining the full antigen-binding specificity and affinity. We show that the megabody design principles are applicable to different scaffold proteins and recognition domains of compatible geometries and are amenable for efficient selection from yeast display libraries. Moreover, we demonstrate that megabodies can be used to obtain three-dimensional reconstructions for membrane proteins that suffer from severe preferential orientation or are otherwise too small to allow accurate particle alignment. Megabodies, built by grafting nanobodies onto larger protein scaffolds, help alleviate problems of particle size and preferential orientation at the water–air interfaces during cryo-EM based structure determination experiments and are shown to be generalizable to soluble and membrane-bound proteins.},
  archive      = {J_NMETH},
  author       = {Uchański, Tomasz and Masiulis, Simonas and Fischer, Baptiste and Kalichuk, Valentina and López-Sánchez, Uriel and Zarkadas, Eleftherios and Weckener, Miriam and Sente, Andrija and Ward, Philip and Wohlkönig, Alexandre and Zögg, Thomas and Remaut, Han and Naismith, James H. and Nury, Hugues and Vranken, Wim and Aricescu, A. Radu and Pardon, Els and Steyaert, Jan},
  doi          = {10.1038/s41592-020-01001-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {60-68},
  shortjournal = {Nat. Meth.},
  title        = {Megabodies expand the nanobody toolkit for protein structure determination by single-particle cryo-EM},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Screening for functional circular RNAs using the
CRISPR–Cas13 system. <em>NMETH</em>, <em>18</em>(1), 51–59. (<a
href="https://doi.org/10.1038/s41592-020-01011-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circular RNAs (circRNAs) produced from back-spliced exons are widely expressed, but individual circRNA functions remain poorly understood owing to the lack of adequate methods for distinguishing circRNAs from cognate messenger RNAs with overlapping exons. Here, we report that CRISPR–RfxCas13d can effectively discriminate circRNAs from mRNAs by using guide RNAs targeting sequences spanning back-splicing junction (BSJ) sites featured in RNA circles. Using a lentiviral library that targets sequences across BSJ sites of highly expressed human circRNAs, we show that a group of circRNAs are important for cell growth mostly in a cell-type-specific manner and that a common oncogenic circRNA, circFAM120A, promotes cell proliferation by preventing the mRNA for family with sequence similarity 120A (FAM120A) from binding the translation inhibitor IGF2BP2. Further application of RfxCas13d–BSJ-gRNA screening has uncovered circMan1a2, which has regulatory potential in mouse embryo preimplantation development. Together, these results establish CRISPR–RfxCas13d as a useful tool for the discovery and functional study of circRNAs at both individual and large-scale levels. This paper describes a CRISPR–Cas13 system to effectively target circRNAs and screen their functions in vitro and in vivo, which enables the study of relevant circRNA phenotypes in human cell proliferation and in mouse embryogenesis.},
  archive      = {J_NMETH},
  author       = {Li, Siqi and Li, Xiang and Xue, Wei and Zhang, Lin and Yang, Liang-Zhong and Cao, Shi-Meng and Lei, Yun-Ni and Liu, Chu-Xiao and Guo, Si-Kun and Shan, Lin and Wu, Man and Tao, Xiao and Zhang, Jia-Lin and Gao, Xiang and Zhang, Jun and Wei, Jia and Li, Jinsong and Yang, Li and Chen, Ling-Ling},
  doi          = {10.1038/s41592-020-01011-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {51-59},
  shortjournal = {Nat. Meth.},
  title        = {Screening for functional circular RNAs using the CRISPR–Cas13 system},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Miniature two-photon microscopy for enlarged field-of-view,
multi-plane and long-term brain imaging. <em>NMETH</em>, <em>18</em>(1),
46–49. (<a href="https://doi.org/10.1038/s41592-020-01024-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have developed a miniature two-photon microscope equipped with an axial scanning mechanism and a long-working-distance miniature objective to enable multi-plane imaging over a volume of 420 × 420 × 180 μm3 at a lateral resolution of ~1 μm. Together with the detachable design that permits long-term recurring imaging, our miniature two-photon microscope can help decipher neuronal mechanisms in freely behaving animals. A two-photon miniature microscope with enlarged field of view and axial scanning capabilities has been developed and applied in freely moving mice.},
  archive      = {J_NMETH},
  author       = {Zong, Weijian and Wu, Runlong and Chen, Shiyuan and Wu, Junjie and Wang, Hanbin and Zhao, Zhe and Chen, Guoqing and Tu, Rui and Wu, Danlei and Hu, Yanhui and Xu, Yangyang and Wang, Yao and Duan, Zhuoli and Wu, Haitao and Zhang, Yunfeng and Zhang, Jue and Wang, Aimin and Chen, Liangyi and Cheng, Heping},
  doi          = {10.1038/s41592-020-01024-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {46-49},
  shortjournal = {Nat. Meth.},
  title        = {Miniature two-photon microscopy for enlarged field-of-view, multi-plane and long-term brain imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). DeepCell kiosk: Scaling deep learning–enabled cellular
image analysis with kubernetes. <em>NMETH</em>, <em>18</em>(1), 43–45.
(<a href="https://doi.org/10.1038/s41592-020-01023-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning is transforming the analysis of biological images, but applying these models to large datasets remains challenging. Here we describe the DeepCell Kiosk, cloud-native software that dynamically scales deep learning workflows to accommodate large imaging datasets. To demonstrate the scalability and affordability of this software, we identified cell nuclei in 106 1-megapixel images in ~5.5 h for ~US$250, with a cost below US$100 achievable depending on cluster configuration. The DeepCell Kiosk can be downloaded at https://github.com/vanvalenlab/kiosk-console ; a persistent deployment is available at https://deepcell.org/ . DeepCell Kiosk is a user-friendly tool that enables fast and affordable deep learning–based analysis of large imaging datasets on Google cloud.},
  archive      = {J_NMETH},
  author       = {Bannon, Dylan and Moen, Erick and Schwartz, Morgan and Borba, Enrico and Kudo, Takamasa and Greenwald, Noah and Vijayakumar, Vibha and Chang, Brian and Pao, Edward and Osterman, Erik and Graf, William and Van Valen, David},
  doi          = {10.1038/s41592-020-01023-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {43-45},
  shortjournal = {Nat. Meth.},
  title        = {DeepCell kiosk: Scaling deep learning–enabled cellular image analysis with kubernetes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: Examining microbe–metabolite correlations by
linear methods. <em>NMETH</em>, <em>18</em>(1), 40–41. (<a
href="https://doi.org/10.1038/s41592-020-01007-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Morton, James T. and McDonald, Daniel and Aksenov, Alexander A. and Nothias, Louis Felix and Foulds, James R. and Quinn, Robert A. and Badri, Michelle H. and Swenson, Tami L. and Van Goethem, Marc W. and Northen, Trent R. and Vazquez-Baeza, Yoshiki and Wang, Mingxun and Bokulich, Nicholas A. and Watters, Aaron and Song, Se Jin and Bonneau, Richard and Dorrestein, Pieter C. and Knight, Rob},
  doi          = {10.1038/s41592-020-01007-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {40-41},
  shortjournal = {Nat. Meth.},
  title        = {Reply to: Examining microbe–metabolite correlations by linear methods},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Examining microbe–metabolite correlations by linear methods.
<em>NMETH</em>, <em>18</em>(1), 37–39. (<a
href="https://doi.org/10.1038/s41592-020-01006-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Quinn, Thomas P. and Erb, Ionas},
  doi          = {10.1038/s41592-020-01006-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {37-39},
  shortjournal = {Nat. Meth.},
  title        = {Examining microbe–metabolite correlations by linear methods},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). T cell development in a dish. <em>NMETH</em>,
<em>18</em>(1), 35. (<a
href="https://doi.org/10.1038/s41592-020-01047-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial thymic organoid systems recapitulate murine thymopoiesis in vitro.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-01047-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {35},
  shortjournal = {Nat. Meth.},
  title        = {T cell development in a dish},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Covalent inhibitor design using phage display.
<em>NMETH</em>, <em>18</em>(1), 34. (<a
href="https://doi.org/10.1038/s41592-020-01046-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phage-based natural selection helps identify highly specific covalent binders.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-01046-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {34},
  shortjournal = {Nat. Meth.},
  title        = {Covalent inhibitor design using phage display},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021m). Scaling up multiple-genome alignments. <em>NMETH</em>,
<em>18</em>(1), 33. (<a
href="https://doi.org/10.1038/s41592-020-01045-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Progressive Cactus enables reference-free multiple-genome alignment for massive datasets.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-01045-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {33},
  shortjournal = {Nat. Meth.},
  title        = {Scaling up multiple-genome alignments},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Calcium imaging in the NIR. <em>NMETH</em>, <em>18</em>(1),
32. (<a href="https://doi.org/10.1038/s41592-020-01044-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The near-infrared calcium sensor iGECI shows promise for imaging neuronal activity in vivo.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-01044-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {32},
  shortjournal = {Nat. Meth.},
  title        = {Calcium imaging in the NIR},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Multiomics sequencing goes spatial. <em>NMETH</em>,
<em>18</em>(1), 31. (<a
href="https://doi.org/10.1038/s41592-020-01043-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microfluidic channels provide a means to deliver barcodes encoding spatial information to a tissue, which allows co-profiling of gene expression and proteins of interest in a spatially resolved manner.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-01043-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {31},
  shortjournal = {Nat. Meth.},
  title        = {Multiomics sequencing goes spatial},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Diving into the TCR repertoire. <em>NMETH</em>,
<em>18</em>(1), 30. (<a
href="https://doi.org/10.1038/s41592-020-01031-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational approaches help us explore complexities of the T cell receptor repertoire.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-01031-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Diving into the TCR repertoire},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Organic dyes for live imaging. <em>NMETH</em>,
<em>18</em>(1), 30. (<a
href="https://doi.org/10.1038/s41592-020-01032-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers are putting new spins on familiar dyes and showing their versatility for labeling living systems.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-01032-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Organic dyes for live imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). More complete genomes. <em>NMETH</em>, <em>18</em>(1), 29.
(<a href="https://doi.org/10.1038/s41592-020-01029-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Method development pushes the limit of completeness of genome sequences.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-01029-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {More complete genomes},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Automated behavioral analysis. <em>NMETH</em>,
<em>18</em>(1), 29. (<a
href="https://doi.org/10.1038/s41592-020-01030-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Behavioral analysis has come a long way from tedious manual annotation, and further strides in automation are expected.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-01030-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {Automated behavioral analysis},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Single-objective light sheet microscopy. <em>NMETH</em>,
<em>18</em>(1), 28. (<a
href="https://doi.org/10.1038/s41592-020-01027-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-objective light sheet fluorescence microscopes are driving innovation in volumetric imaging.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-01027-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {28},
  shortjournal = {Nat. Meth.},
  title        = {Single-objective light sheet microscopy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Glycoproteomics. <em>NMETH</em>, <em>18</em>(1), 28. (<a
href="https://doi.org/10.1038/s41592-020-01028-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glycoproteomics is coming of age, thanks to advances in instrumentation, experimental methodologies and computational search algorithms.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-01028-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {28},
  shortjournal = {Nat. Meth.},
  title        = {Glycoproteomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021t). Versatile genome editing. <em>NMETH</em>, <em>18</em>(1),
27. (<a href="https://doi.org/10.1038/s41592-020-01025-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editors enable different types of small mutations to be introduced into genomes.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-01025-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {27},
  shortjournal = {Nat. Meth.},
  title        = {Versatile genome editing},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Assembloids. <em>NMETH</em>, <em>18</em>(1), 27. (<a
href="https://doi.org/10.1038/s41592-020-01026-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoids generated by spatially organizing multiple cell types, called assembloids, will enable deeper insights into tissue function.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-01026-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {27},
  shortjournal = {Nat. Meth.},
  title        = {Assembloids},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Spatially resolved transcriptomics in neuroscience.
<em>NMETH</em>, <em>18</em>(1), 23–25. (<a
href="https://doi.org/10.1038/s41592-020-01040-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One major challenge in neuroscience is to gain a systematic understanding of the extraordinary diversity of brain cell types and how they contribute to brain function. Spatially resolved transcriptomics holds unmatched promise in unraveling the organization of brain cell types and their relationship with connectivity, circuit dynamics, behavior and disease. Here we discuss neuroscience applications of various spatially resolved transcriptomics methods, as well as technical challenges that need to be overcome to realize their full potentials.},
  archive      = {J_NMETH},
  author       = {Close, Jennie L. and Long, Brian R. and Zeng, Hongkui},
  doi          = {10.1038/s41592-020-01040-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {23-25},
  shortjournal = {Nat. Meth.},
  title        = {Spatially resolved transcriptomics in neuroscience},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Spatially resolved single-cell genomics and transcriptomics
by imaging. <em>NMETH</em>, <em>18</em>(1), 18–22. (<a
href="https://doi.org/10.1038/s41592-020-01037-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recent advent of genome-scale imaging has enabled single-cell omics analysis in a spatially resolved manner in intact cells and tissues. These advances allow gene expression profiling of individual cells, and hence in situ identification and spatial mapping of cell types, in complex tissues. The high spatial resolution of these approaches further allows determination of the spatial organizations of the genome and transcriptome inside cells, both of which are key regulatory mechanisms for gene expression.},
  archive      = {J_NMETH},
  author       = {Zhuang, Xiaowei},
  doi          = {10.1038/s41592-020-01037-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {18-22},
  shortjournal = {Nat. Meth.},
  title        = {Spatially resolved single-cell genomics and transcriptomics by imaging},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Spatially resolved transcriptomics adds a new dimension to
genomics. <em>NMETH</em>, <em>18</em>(1), 15–18. (<a
href="https://doi.org/10.1038/s41592-020-01038-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As single-cell omics continue to advance, the field of spatially resolved transcriptomics has emerged with a set of experimental and computational methods to map out the positions of cells and their gene expression profiles in space. Here we summarize current transcriptome-wide and sequencing-based methodologies and their applications in genomics research.},
  archive      = {J_NMETH},
  author       = {Larsson, Ludvig and Frisén, Jonas and Lundeberg, Joakim},
  doi          = {10.1038/s41592-020-01038-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {15-18},
  shortjournal = {Nat. Meth.},
  title        = {Spatially resolved transcriptomics adds a new dimension to genomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Method of the year: Spatially resolved transcriptomics.
<em>NMETH</em>, <em>18</em>(1), 9–14. (<a
href="https://doi.org/10.1038/s41592-020-01033-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Methods has crowned spatially resolved transcriptomics Method of the Year 2020.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-01033-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {9-14},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year: Spatially resolved transcriptomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The standardization fallacy. <em>NMETH</em>, <em>18</em>(1),
5–7. (<a href="https://doi.org/10.1038/s41592-020-01036-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {“We demand rigidly defined areas of doubt and uncertainty!” —D. Adams},
  archive      = {J_NMETH},
  author       = {Voelkl, Bernhard and Würbel, Hanno and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-020-01036-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {5-7},
  shortjournal = {Nat. Meth.},
  title        = {The standardization fallacy},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Method of the year 2020: Spatially resolved transcriptomics.
<em>NMETH</em>, <em>18</em>(1), 1. (<a
href="https://doi.org/10.1038/s41592-020-01042-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved transcriptomics methods are changing the way we understand complex tissues.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-01042-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year 2020: Spatially resolved transcriptomics},
  volume       = {18},
  year         = {2021},
}
</textarea>
</details></li>
</ul>

</body>
</html>
