<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng---38">NG - 38</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Functional innovation through new
genes as a general evolutionary process. <em>NG</em>, <em>57</em>(3),
764. (<a href="https://doi.org/10.1038/s41588-025-02116-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Xia, Shengqian and Chen, Jianhai and Arsala, Deanna and Emerson, J. J. and Long, Manyuan},
  doi          = {10.1038/s41588-025-02116-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {764},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Functional innovation through new genes as a general evolutionary process},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The NAT1–bHLH110–CER1/CER1L module
regulates heat stress tolerance in rice. <em>NG</em>, <em>57</em>(3),
763. (<a href="https://doi.org/10.1038/s41588-025-02106-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lu, Hai-Ping and Liu, Xue-Huan and Wang, Mei-Jing and Zhu, Qiao-Yun and Lyu, Yu-Shu and Xu, Jian-Hang and Liu, Jian-Xiang},
  doi          = {10.1038/s41588-025-02106-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The NAT1–bHLH110–CER1/CER1L module regulates heat stress tolerance in rice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Pangenome graphs and their applications
in biodiversity genomics. <em>NG</em>, <em>57</em>(3), 763. (<a
href="https://doi.org/10.1038/s41588-025-02132-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Secomandi, Simona and Gallo, Guido Roberto and Rossi, Riccardo and Rodríguez Fernandes, Carlos and Jarvis, Erich D. and Bonisoli-Alquati, Andrea and Gianfranceschi, Luca and Formenti, Giulio},
  doi          = {10.1038/s41588-025-02132-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Pangenome graphs and their applications in biodiversity genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Origin and de novo domestication of sweet orange.
<em>NG</em>, <em>57</em>(3), 754–762. (<a
href="https://doi.org/10.1038/s41588-025-02122-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sweet orange is cultivated worldwide but suffers from various devastating diseases because of its monogenetic background. The elucidation of the origin of a crop facilitates the domestication of new crops that may better cope with new challenges. Here we collected and sequenced 226 citrus accessions and assembled telomere-to-telomere phased diploid genomes of sweet orange and sour orange. On the basis of a high-resolution haplotype-resolved genome analysis, we inferred that sweet orange originated from a sour orange × mandarin cross and confirmed this model using artificial hybridization experiments. We identified defense-related metabolites that potently inhibited the growth of multiple industrially important pathogenic bacteria. We introduced diversity to sweet orange, which showed wide segregation in fruit flavor and disease resistance and produced canker-resistant sweet orange by selecting defense-related metabolites. Our findings elucidate the origin of sweet orange and de novo domesticated disease-resistant sweet oranges, illuminating a strategy for the rapid domestication of perennial crops. Genomic analyses of Citrus species including haplotype-resolved genomes of Citrus sinensis and Citrus aurantium highlight the origin of sweet orange and provide a strategy for de novo domestication of perennial crops.},
  archive      = {J_NG},
  author       = {Liu, Shengjun and Xu, Yuantao and Yang, Kun and Huang, Yue and Lu, Zhihao and Chen, Shulin and Gao, Xiang and Xiao, Gongao and Chen, Peng and Zeng, Xiuli and Wang, Lun and Zheng, Weikang and Liu, Zishuang and Liao, Guanglian and He, Fa and Liu, Junjie and Wan, Pengfei and Ding, Fang and Ye, Junli and Jiao, Wenbiao and Chai, Lijun and Pan, Zhiyong and Zhang, Fei and Lin, Zongcheng and Zan, Yanjun and Guo, Wenwu and Larkin, Robert M. and Xie, Zongzhou and Wang, Xia and Deng, Xiuxin and Xu, Qiang},
  doi          = {10.1038/s41588-025-02122-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {754-762},
  shortjournal = {Nat. Genet.},
  title        = {Origin and de novo domestication of sweet orange},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super pangenome of vitis empowers identification of downy
mildew resistance genes for grapevine improvement. <em>NG</em>,
<em>57</em>(3), 741–753. (<a
href="https://doi.org/10.1038/s41588-025-02111-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Grapevine (Vitis) is one of the oldest domesticated fruit crops with great cultural and economic importance. Here we assembled and annotated haplotype-resolved genomes of 72 global Vitis accessions including 25 wild and 47 cultivated grapevines, among which genomes for 60 grapevines are newly released. Haplotype-aware phylogenomics disentangled the mysterious hybridization history of grapevines, revealing the enormous genetic diversity of the Vitis genus. Pangenomic analysis reveals that European cultivars, more susceptible to the destructive disease downy mildew (DM), have a smaller repertoire of resistance genes in the NLR family encoding the TIR-NBARC-LRR domain. Through extensive structural variation (SV) characterization, phenotyping, DM-infection transcriptome profiling of 113 Vitis accessions, and SV–expression quantitative trait loci analysis, we have identified over 63 SVs and their relevant genes significantly associated with DM resistance, exemplified by a lysine histidine transporter, VvLHT8. This haplotype-resolved super pangenome of the Vitis genus will accelerate breeding and enrich our understanding of the evolution and biology of grapevines. Super pangenome of the Vitis genus assembled from haplotype-resolved genomes of 72 global Vitis accessions characterizes the landscape of structural variants (SVs) and identifies SV–expression quantitative trait loci for downy mildew resistance.},
  archive      = {J_NG},
  author       = {Guo, Li and Wang, Xiangfeng and Ayhan, Dilay Hazal and Rhaman, Mohammad Saidur and Yan, Ming and Jiang, Jianfu and Wang, Dongyue and Zheng, Wei and Mei, Junjie and Ji, Wei and Jiao, Jian and Chen, Shaoying and Sun, Jie and Yi, Shu and Meng, Dian and Wang, Jing and Bhuiyan, Mohammad Nasim and Qin, Guochen and Guo, Linling and Yang, Qingxian and Zhang, Xuenan and Sun, Haisheng and Liu, Chonghuai and Deng, Xing Wang and Ye, Wenxiu},
  doi          = {10.1038/s41588-025-02111-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {741-753},
  shortjournal = {Nat. Genet.},
  title        = {Super pangenome of vitis empowers identification of downy mildew resistance genes for grapevine improvement},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The marchantia polymorpha pangenome reveals ancient
mechanisms of plant adaptation to the environment. <em>NG</em>,
<em>57</em>(3), 729–740. (<a
href="https://doi.org/10.1038/s41588-024-02071-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant adaptation to terrestrial life started 450 million years ago and has played a major role in the evolution of life on Earth. The genetic mechanisms allowing this adaptation to a diversity of terrestrial constraints have been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the model bryophyte Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, a genome–environment association study and a pangenome. We identified adaptive features, such as peroxidases or nucleotide-binding and leucine-rich repeats (NLRs), also observed in flowering plants, likely inherited from the first land plants. The M. polymorpha pangenome also harbors lineage-specific accessory genes absent from seed plants. We conclude that different land plant lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage-specific polymorphisms and gene family evolution. Pangenome analyses of 133 wild accessions of the model bryophyte Marchantia polymorpha identify adaptive features and provide insights into the mechanisms of plant adaptation to the terrestrial environment.},
  archive      = {J_NG},
  author       = {Beaulieu, Chloé and Libourel, Cyril and Mbadinga Zamar, Duchesse Lacourt and El Mahboubi, Karima and Hoey, David J. and Greiff, George R. L. and Keller, Jean and Girou, Camille and San Clemente, Helene and Diop, Issa and Amblard, Emilie and Castel, Baptiste and Théron, Anthony and Cauet, Stéphane and Rodde, Nathalie and Zachgo, Sabine and Halpape, Wiebke and Meierhenrich, Anja and Laker, Bianca and Bräutigam, Andrea and Szovenyi, Peter and Cheng, Shifeng and Tanizawa, Yasuhiro and Aziz, Simon and Leebens-Mack, James H. and Schmutz, Jeremy and Webber, Jenell and Grimwood, Jane and Jacquet, Christophe and Dunand, Christophe and Nelson, Jessica M. and Roux, Fabrice and Philippe, Hervé and Schornack, Sebastian and Bonhomme, Maxime and Delaux, Pierre-Marc},
  doi          = {10.1038/s41588-024-02071-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {729-740},
  shortjournal = {Nat. Genet.},
  title        = {The marchantia polymorpha pangenome reveals ancient mechanisms of plant adaptation to the environment},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Functional analysis of cancer-associated germline risk
variants. <em>NG</em>, <em>57</em>(3), 718–728. (<a
href="https://doi.org/10.1038/s41588-024-02070-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nucleotide variants (SNVs) in regulatory DNA are linked to inherited cancer risk. Massively parallel reporter assays of 4,041 SNVs linked to 13 neoplasms comprising &amp;gt;90% of human malignancies were performed in pertinent primary human cell types and then integrated with matching chromatin accessibility, DNA looping and expression quantitative trait loci data to nominate 380 potentially regulatory SNVs and their putative target genes. The latter highlighted specific protein networks in lifetime cancer risk, including mitochondrial translation, DNA damage repair and Rho GTPase activity. A CRISPR knockout screen demonstrated that a subset of germline putative risk genes also enables the growth of established cancers. Editing one SNV, rs10411210 , showed that its risk allele increases rhophilin RHPN2 expression and stimulus-responsive RhoA activation, indicating that individual SNVs may upregulate cancer-linked pathways. These functional data are a resource for variant prioritization efforts and further interrogation of the mechanisms underlying inherited risk for cancer. Analysis of 4,041 single-nucleotide variants (SNVs) linked to 13 cancers performed in primary human cell types identifies 380 potentially regulatory SNVs and their putative target genes. Editing one SNV, rs10411210 , revealed that the risk allele increases RHPN2 expression and stimulus-responsive RhoA activation.},
  archive      = {J_NG},
  author       = {Kellman, Laura N. and Neela, Poornima H. and Srinivasan, Suhas and Siprashvili, Zurab and Shanderson, Ronald L. and Hong, Audrey W. and Rao, Deepti and Porter, Douglas F. and Reynolds, David L. and Meyers, Robin M. and Guo, Margaret G. and Yang, Xue and Zhao, Yang and Wozniak, Glenn G. and Donohue, Laura K. H. and Shenoy, Rajani and Ko, Lisa A. and Nguyen, Duy T. and Mondal, Smarajit and Garcia, Omar S. and Elcavage, Lara E. and Elfaki, Ibtihal and Abell, Nathan S. and Tao, Shiying and Lopez, Christopher M. and Montgomery, Stephen B. and Khavari, Paul A.},
  doi          = {10.1038/s41588-024-02070-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {718-728},
  shortjournal = {Nat. Genet.},
  title        = {Functional analysis of cancer-associated germline risk variants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying cell divisions along evolutionary lineages in
cancer. <em>NG</em>, <em>57</em>(3), 706–717. (<a
href="https://doi.org/10.1038/s41588-025-02078-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell division drives somatic evolution but is challenging to quantify. We developed a framework to count cell divisions with DNA replication-related mutations in polyguanine homopolymers. Analyzing 505 samples from 37 patients, we studied the milestones of colorectal cancer evolution. Primary tumors diversify at ~250 divisions from the founder cell, while distant metastasis divergence occurs significantly later, at ~500 divisions. Notably, distant but not lymph node metastases originate from primary tumor regions that have undergone surplus divisions, tying subclonal expansion to metastatic capacity. Then, we analyzed a cohort of 73 multifocal lung cancers and showed that the cell division burden of the tumors’ common ancestor distinguishes independent primary tumors from intrapulmonary metastases and correlates with patient survival. In lung cancer too, metastatic capacity is tied to more extensive proliferation. The cell division history of human cancers is easily accessible using our simple framework and contains valuable biological and clinical information. This work presents a framework for estimating cell division numbers using DNA replication-associated polyguanine tract mutations, with applications for understanding tumor natural histories and origins.},
  archive      = {J_NG},
  author       = {Blohmer, Martin and Cheek, David M. and Hung, Wei-Ting and Kessler, Maria and Chatzidimitriou, Foivos and Wang, Jiahe and Hung, William and Lee, I-Hsiu and Gorelick, Alexander N. and Wassenaar, Emma CE and Yang, Ching-Yeuh and Yeh, Yi-Chen and Ho, Hsiang-Ling and Speiser, Dorothee and Karsten, Maria M. and Lanuti, Michael and Pai, Sara I. and Kranenburg, Onno and Lennerz, Jochen K. and Chou, Teh-Ying and Kloor, Matthias and Naxerova, Kamila},
  doi          = {10.1038/s41588-025-02078-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {706-717},
  shortjournal = {Nat. Genet.},
  title        = {Quantifying cell divisions along evolutionary lineages in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ImmuneLENS characterizes systemic immune dysregulation in
aging and cancer. <em>NG</em>, <em>57</em>(3), 694–705. (<a
href="https://doi.org/10.1038/s41588-025-02086-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recognition and elimination of pathogens and cancer cells depend on the adaptive immune system. Thus, accurate quantification of immune subsets is vital for precision medicine. We present immune lymphocyte estimation from nucleotide sequencing (ImmuneLENS), which estimates T cell and B cell fractions, class switching and clonotype diversity from whole-genome sequencing data at depths as low as 5× coverage. By applying ImmuneLENS to the 100,000 Genomes Project, we identify genes enriched with somatic mutations in T cell-rich tumors, significant sex-based differences in circulating T cell fraction and demonstrated that the circulating T cell fraction in patients with cancer is significantly lower than in healthy individuals. Low circulating B cell fraction was linked to increased cancer incidence. Finally, circulating T cell abundance was more prognostic of 5-year cancer survival than infiltrating T cells. Immune lymphocyte estimation from nucleotide sequencing (ImmuneLENS) infers B cell and T cell fractions from whole-genome sequencing data. Applied to the 100,000 Genomes Project datasets, circulating T cell fraction provides sex-dependent and prognostic insights in patients.},
  archive      = {J_NG},
  author       = {Bentham, Robert and Jones, Thomas P. and Black, James R. M. and Martinez-Ruiz, Carlos and Dietzen, Michelle and Litovchenko, Maria and Thol, Kerstin and Watkins, Thomas B. K. and Bailey, Chris and Pich, Oriol and Zhang, Zhihui and Van Loo, Peter and Swanton, Charles and McGranahan, Nicholas},
  doi          = {10.1038/s41588-025-02086-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {694-705},
  shortjournal = {Nat. Genet.},
  title        = {ImmuneLENS characterizes systemic immune dysregulation in aging and cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Alterations in PD-l1 succinylation shape anti-tumor immune
responses in melanoma. <em>NG</em>, <em>57</em>(3), 680–693. (<a
href="https://doi.org/10.1038/s41588-025-02077-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumors undergo metabolic reprogramming to meet the energetic, synthetic and redox demands essential for malignancy, often characterized by increased glycolysis and lactate production. However, the role of mitochondrial metabolism in tumor immunity remains unclear. The present study integrates spatial transcriptomics, bulk transcriptomics and proteomics, revealing a strong link between the metabolite succinyl-CoA and tumor immunity as well as the efficacy of anti-programmed cell death protein-1 (PD-1) therapy in patients with melanoma. Elevated succinyl-CoA levels, through α-ketoglutarate or succinate supplementation, enhanced T cell-mediated tumor elimination, both in vitro and in vivo. Mechanistically, succinylation of the ligand of PD-1 (PD-L1) at lysine 129 led to its degradation. Increased carnitine palmitoyltransferase 1A (CPT1A), identified as a succinyltransferase for PD-L1, boosted anti-tumor activity. Preclinically, bezafibrate, a hyperlipidemia drug, upregulated CPT1A and synergized with CTLA-4 monoclonal antibody to inhibit tumor growth. Clinically, higher PD-L1 and lower CPT1A levels in tumors correlated with better anti-PD-1 therapy responses, suggesting potential biomarkers for prediction of treatment efficacy. Succinylation of PD-L1 by carnitine palmitoyltransferase 1A (CPT1A) in melanoma leads to its degradation and enhanced T cell-dependent killing in vitro. Increasing CPT1A levels synergizes with anti-CTLA-4 treatment to suppress tumor growth in a mouse melanoma model.},
  archive      = {J_NG},
  author       = {Liang, Long and Kuang, Xinwei and He, Yi and Zhu, Lin and Lau, Poyee and Li, Xin and Luo, Dingan and Gong, Lan and Zhou, Wenbin and Zhang, Fanglin and Liang, Xiaowei and Li, Zhuofeng and Hu, Bin and Liu, Dandan and Ding, Tao and Li, Hui and Zhao, Shuang and Su, Juan and Hung, Mien-Chie and Liu, Jing and Liu, Hong and Chen, Xiang},
  doi          = {10.1038/s41588-025-02077-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {680-693},
  shortjournal = {Nat. Genet.},
  title        = {Alterations in PD-l1 succinylation shape anti-tumor immune responses in melanoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Active repression of cell fate plasticity by PROX1
safeguards hepatocyte identity and prevents liver tumorigenesis.
<em>NG</em>, <em>57</em>(3), 668–679. (<a
href="https://doi.org/10.1038/s41588-025-02081-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell fate plasticity enables development, yet unlocked plasticity is a cancer hallmark. While transcription master regulators induce lineage-specific genes to restrict plasticity, it remains unclear whether plasticity is actively suppressed by lineage-specific repressors. Here we computationally predict so-called safeguard repressors for 18 cell types that block phenotypic plasticity lifelong. We validated hepatocyte-specific candidates using reprogramming, revealing that prospero homeobox protein 1 (PROX1) enhanced hepatocyte identity by direct repression of alternative fate master regulators. In mice, Prox1 was required for efficient hepatocyte regeneration after injury and was sufficient to prevent liver tumorigenesis. In line with patient data, Prox1 depletion caused hepatocyte fate loss in vivo and enabled the transition of hepatocellular carcinoma to cholangiocarcinoma. Conversely, overexpression promoted cholangiocarcinoma to hepatocellular carcinoma transdifferentiation. Our findings provide evidence for PROX1 as a hepatocyte-specific safeguard and support a model where cell-type-specific repressors actively suppress plasticity throughout life to safeguard lineage identity and thus prevent disease. This study identifies candidate safeguard repressor transcription factors that repress alternate lineages in mature cell types and provides functional evidence that Prox1 performs such a function in hepatocytes during reprogramming, regeneration and in cancer.},
  archive      = {J_NG},
  author       = {Lim, Bryce and Kamal, Aryan and Gomez Ramos, Borja and Adrian Segarra, Juan M. and Ibarra, Ignacio L. and Dignas, Lennart and Kindinger, Tim and Volz, Kai and Rahbari, Mohammad and Rahbari, Nuh and Poisel, Eric and Kafetzopoulou, Kanela and Böse, Lio and Breinig, Marco and Heide, Danijela and Gallage, Suchira and Barragan Avila, Jose E. and Wiethoff, Hendrik and Berest, Ivan and Schnabellehner, Sarah and Schneider, Martin and Becker, Jonas and Helm, Dominic and Grimm, Dirk and Mäkinen, Taija and Tschaharganeh, Darjus F. and Heikenwalder, Mathias and Zaugg, Judith B. and Mall, Moritz},
  doi          = {10.1038/s41588-025-02081-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {668-679},
  shortjournal = {Nat. Genet.},
  title        = {Active repression of cell fate plasticity by PROX1 safeguards hepatocyte identity and prevents liver tumorigenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis of multi-condition single-cell data with latent
embedding multivariate regression. <em>NG</em>, <em>57</em>(3), 659–667.
(<a href="https://doi.org/10.1038/s41588-024-01996-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying gene expression differences in heterogeneous tissues across conditions is a fundamental biological task, enabled by multi-condition single-cell RNA sequencing (RNA-seq). Current data analysis approaches divide the constituent cells into clusters meant to represent cell types, but such discrete categorization tends to be an unsatisfactory model of the underlying biology. Here, we introduce latent embedding multivariate regression (LEMUR), a model that operates without, or before, commitment to discrete categorization. LEMUR (1) integrates data from different conditions, (2) predicts each cell’s gene expression changes as a function of the conditions and its position in latent space and (3) for each gene, identifies a compact neighborhood of cells with consistent differential expression. We apply LEMUR to cancer, zebrafish development and spatial gradients in Alzheimer’s disease, demonstrating its broad applicability. Latent embedding multivariate regression models multi-condition single-cell RNA-seq using a continuous latent space, enabling data integration, per-cell gene expression prediction and clustering-free differential expression analysis.},
  archive      = {J_NG},
  author       = {Ahlmann-Eltze, Constantin and Huber, Wolfgang},
  doi          = {10.1038/s41588-024-01996-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {659-667},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of multi-condition single-cell data with latent embedding multivariate regression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial transcriptomics identifies molecular niche
dysregulation associated with distal lung remodeling in pulmonary
fibrosis. <em>NG</em>, <em>57</em>(3), 647–658. (<a
href="https://doi.org/10.1038/s41588-025-02080-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale changes in the structure and cellular makeup of the distal lung are a hallmark of pulmonary fibrosis (PF), but the spatial contexts that contribute to disease pathogenesis have remained uncertain. Using image-based spatial transcriptomics, we analyzed the gene expression of 1.6 million cells from 35 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell types, established the cellular and molecular basis of classical PF histopathologic features and identified a diversity of distinct molecularly defined spatial niches in control and PF lungs. Using machine learning and trajectory analysis to segment and rank airspaces on a gradient of remodeling severity, we identified compositional and molecular changes associated with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially resolved view of PF and establish methods that could be applied to other spatial transcriptomic studies. Xenium spatial transcriptomic profiling of pulmonary fibrosis characterizes cell composition dynamics and histopathological features associated with the disease.},
  archive      = {J_NG},
  author       = {Vannan, Annika and Lyu, Ruqian and Williams, Arianna L. and Negretti, Nicholas M. and Mee, Evan D. and Hirsh, Joseph and Hirsh, Samuel and Hadad, Niran and Nichols, David S. and Calvi, Carla L. and Taylor, Chase J. and Polosukhin, Vasiliy. V. and Serezani, Ana P. M. and McCall, A. Scott and Gokey, Jason J. and Shim, Heejung and Ware, Lorraine B. and Bacchetta, Matthew J. and Shaver, Ciara M. and Blackwell, Timothy S. and Walia, Rajat and Sucre, Jennifer M. S. and Kropski, Jonathan A. and McCarthy, Davis J. and Banovich, Nicholas E.},
  doi          = {10.1038/s41588-025-02080-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {647-658},
  shortjournal = {Nat. Genet.},
  title        = {Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Plasma proteome variation and its genetic determinants in
children and adolescents. <em>NG</em>, <em>57</em>(3), 635–646. (<a
href="https://doi.org/10.1038/s41588-025-02089-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our current understanding of the determinants of plasma proteome variation during pediatric development remains incomplete. Here, we show that genetic variants, age, sex and body mass index significantly influence this variation. Using a streamlined and highly quantitative mass spectrometry-based proteomics workflow, we analyzed plasma from 2,147 children and adolescents, identifying 1,216 proteins after quality control. Notably, the levels of 70% of these were associated with at least one of the aforementioned factors, with protein levels also being predictive. Quantitative trait loci (QTLs) regulated at least one-third of the proteins; between a few percent and up to 30-fold. Together with excellent replication in an additional 1,000 children and 558 adults, this reveals substantial genetic effects on plasma protein levels, persisting from childhood into adulthood. Through Mendelian randomization and colocalization analyses, we identified 41 causal genes for 33 cardiometabolic traits, emphasizing the value of protein QTLs in drug target identification and disease understanding. This mass spectrometry-based proteomic study profiles the plasma proteome in 2,147 children and adolescents and reveals its association with age, sex, puberty, body mass index and genetics.},
  archive      = {J_NG},
  author       = {Niu, Lili and Stinson, Sara Elizabeth and Holm, Louise Aas and Lund, Morten Asp Vonsild and Fonvig, Cilius Esmann and Cobuccio, Leonardo and Meisner, Jonas and Juel, Helene Bæk and Fadista, Joao and Thiele, Maja and Krag, Aleksander and Holm, Jens-Christian and Rasmussen, Simon and Hansen, Torben and Mann, Matthias},
  doi          = {10.1038/s41588-025-02089-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {635-646},
  shortjournal = {Nat. Genet.},
  title        = {Plasma proteome variation and its genetic determinants in children and adolescents},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Whole-genome sequencing analysis identifies rare,
large-effect noncoding variants and regulatory regions associated with
circulating protein levels. <em>NG</em>, <em>57</em>(3), 626–634. (<a
href="https://doi.org/10.1038/s41588-025-02095-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The contribution of rare noncoding genetic variation to common phenotypes is largely unknown, as a result of a historical lack of population-scale whole-genome sequencing data and the difficulty of categorizing noncoding variants into functionally similar groups. To begin addressing these challenges, we performed a cis association analysis using whole-genome sequencing data, consisting of 1.1 billion variants, 123 million noncoding aggregate-based tests and 2,907 circulating protein levels in ~50,000 UK Biobank participants. We identified 604 independent rare noncoding single-variant associations with circulating protein levels. Unlike protein-coding variation, rare noncoding genetic variation was almost as likely to increase or decrease protein levels. Rare noncoding aggregate testing identified 357 conditionally independent associated regions. Of these, 74 (21%) were not detectable by single-variant testing alone. Our findings have important implications for the identification, and role, of rare noncoding genetic variation associated with common human phenotypes, including the importance of testing aggregates of noncoding variants. Rare variant association analysis of plasma proteins using whole-genome sequencing data in 54,306 individuals in the UK Biobank demonstrates that combining both single-variant and aggregate testing will be needed for noncoding variant analysis.},
  archive      = {J_NG},
  author       = {Hawkes, Gareth and Chundru, Kartik and Jackson, Leigh and Patel, Kashyap A. and Murray, Anna and Wood, Andrew R. and Wright, Caroline F. and Weedon, Michael N. and Frayling, Timothy M. and Beaumont, Robin N.},
  doi          = {10.1038/s41588-025-02095-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {626-634},
  shortjournal = {Nat. Genet.},
  title        = {Whole-genome sequencing analysis identifies rare, large-effect noncoding variants and regulatory regions associated with circulating protein levels},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The contribution of genetic determinants of blood gene
expression and splicing to molecular phenotypes and health outcomes.
<em>NG</em>, <em>57</em>(3), 616–625. (<a
href="https://doi.org/10.1038/s41588-025-02096-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biological mechanisms through which most nonprotein-coding genetic variants affect disease risk are unknown. To investigate gene-regulatory mechanisms, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA sequencing in 4,732 participants and integrated protein, metabolite and lipid data from the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Colocalization analyses revealed 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access resource on the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans ( https://IntervalRNA.org.uk ). Integrative analyses of whole-blood gene expression and splicing QTLs with protein, metabolite and lipid QTLs from the same individuals shed light on the shared genetic architecture of molecular traits and health outcomes.},
  archive      = {J_NG},
  author       = {Tokolyi, Alex and Persyn, Elodie and Nath, Artika P. and Burnham, Katie L. and Marten, Jonathan and Vanderstichele, Thomas and Tardaguila, Manuel and Stacey, David and Farr, Ben and Iyer, Vivek and Jiang, Xilin and Lambert, Samuel A. and Noell, Guillaume and Quail, Michael A. and Rajan, Diana and Ritchie, Scott C. and Sun, Benjamin B. and Thurston, Scott A. J. and Xu, Yu and Whelan, Christopher D. and Runz, Heiko and Petrovski, Slavé and Gaffney, Daniel J. and Roberts, David J. and Di Angelantonio, Emanuele and Peters, James E. and Soranzo, Nicole and Danesh, John and Butterworth, Adam S. and Inouye, Michael and Davenport, Emma E. and Paul, Dirk S.},
  doi          = {10.1038/s41588-025-02096-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {616-625},
  shortjournal = {Nat. Genet.},
  title        = {The contribution of genetic determinants of blood gene expression and splicing to molecular phenotypes and health outcomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-read RNA sequencing atlas of human microglia isoforms
elucidates disease-associated genetic regulation of splicing.
<em>NG</em>, <em>57</em>(3), 604–615. (<a
href="https://doi.org/10.1038/s41588-025-02099-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microglia, the innate immune cells of the central nervous system, have been genetically implicated in multiple neurodegenerative diseases. Mapping the genetics of gene expression in human microglia has identified several loci associated with disease-associated genetic variants in microglia-specific regulatory elements. However, identifying genetic effects on splicing is challenging because of the use of short sequencing reads. Here, we present the isoform-centric microglia genomic atlas (isoMiGA), which leverages long-read RNA sequencing to identify 35,879 novel microglia isoforms. We show that these isoforms are involved in stimulation response and brain region specificity. We then quantified the expression of both known and novel isoforms in a multi-ancestry meta-analysis of 555 human microglia short-read RNA sequencing samples from 391 donors, and found associations with genetic risk loci in Alzheimer’s and Parkinson’s disease. We nominate several loci that may act through complex changes in isoform and splice-site usage. This isoform-centric microglia genomic atlas includes 35,879 novel human microglia isoforms identified by long-read RNA sequencing. A multi-ancestry quantitative trait locus meta-analysis of known and novel isoforms in 555 samples from 391 donors finds associations with genetic risk loci in Alzheimer’s and Parkinson’s diseases.},
  archive      = {J_NG},
  author       = {Humphrey, Jack and Brophy, Erica and Kosoy, Roman and Zeng, Biao and Coccia, Elena and Mattei, Daniele and Ravi, Ashvin and Naito, Tatsuhiko and Efthymiou, Anastasia G. and Navarro, Elisa and De Sanctis, Claudia and Flores-Almazan, Victoria and Muller, Benjamin Z. and Snijders, Gijsje J. L. J. and Allan, Amanda and Münch, Alexandra and Kitata, Reta Birhanu and Kleopoulos, Steven P. and Argyriou, Stathis and Malakates, Periklis and Psychogyiou, Konstantina and Shao, Zhiping and Francoeur, Nancy and Tsai, Chia-Feng and Gritsenko, Marina A. and Monroe, Matthew E. and Paurus, Vanessa L. and Weitz, Karl K. and Shi, Tujin and Sebra, Robert and Liu, Tao and de Witte, Lot D. and Goate, Alison M. and Bennett, David A. and Haroutunian, Vahram and Hoffman, Gabriel E. and Fullard, John F. and Roussos, Panos and Raj, Towfique},
  doi          = {10.1038/s41588-025-02099-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {604-615},
  shortjournal = {Nat. Genet.},
  title        = {Long-read RNA sequencing atlas of human microglia isoforms elucidates disease-associated genetic regulation of splicing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiomic single-cell profiling identifies critical
regulators of postnatal brain. <em>NG</em>, <em>57</em>(3), 591–603. (<a
href="https://doi.org/10.1038/s41588-025-02083-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human brain development spans from embryogenesis to adulthood, with dynamic gene expression controlled by cell-type-specific cis-regulatory element activity and three-dimensional genome organization. To advance our understanding of postnatal brain development, we simultaneously profiled gene expression and chromatin accessibility in 101,924 single nuclei from four brain regions across ten donors, covering five key postnatal stages from infancy to late adulthood. Using this dataset and chromosome conformation capture data, we constructed enhancer-based gene regulatory networks to identify cell-type-specific regulators of brain development and interpret genome-wide association study loci for ten main brain disorders. Our analysis connected 2,318 cell-specific loci to 1,149 unique genes, representing 41% of loci linked to the investigated traits, and highlighted 55 genes influencing several disease phenotypes. Pseudotime analysis revealed distinct stages of postnatal oligodendrogenesis and their regulatory programs. These findings provide a comprehensive dataset of cell-type-specific gene regulation at critical timepoints in postnatal brain development. Simultaneous profiling of RNA and chromatin accessibility in single nuclei isolated from human postnatal brain regions from infancy to late adulthood identifies key cellular regulators and nominates target genes and mechanisms for brain-related diseases and disorders.},
  archive      = {J_NG},
  author       = {Clarence, Tereza and Bendl, Jaroslav and Cao, Xuan and Wang, Xinyi and Zheng, Shiwei and Hoffman, Gabriel E. and Kozlenkov, Alexey and Hong, Aram and Iskhakova, Marina and Jaiswal, Manoj K. and Murphy, Sarah and Yu, Alexander and Haroutunian, Vahram and Dracheva, Stella and Akbarian, Schahram and Fullard, John F. and Yuan, Guo-Cheng and Lee, Donghoon and Roussos, Panos},
  doi          = {10.1038/s41588-025-02083-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {591-603},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic single-cell profiling identifies critical regulators of postnatal brain},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparison of the multivariate genetic architecture of eight
major psychiatric disorders across sex. <em>NG</em>, <em>57</em>(3),
583–590. (<a href="https://doi.org/10.1038/s41588-025-02093-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Differences in the patterning of genetic sharing between groups of individuals may arise from biological pathways, social mechanisms, phenotyping and ascertainment. We expand genomic structural equation modeling to allow for testing genomic structural invariance (GSI), that is, the formal comparison of multivariate genetic architecture across groups. We apply GSI to compare the autosomal multivariate genetic architecture of eight psychiatric disorders spanning three factors (psychotic, neurodevelopmental and internalizing) between cisgender males and females. We find that the genetic factor structure is largely similar across sex, permitting meaningful comparisons of associations at the level of the factors. However, in females, problematic alcohol use and posttraumatic stress disorder loaded more strongly on the internalizing factor, while the neurodevelopmental disorder factor exhibited weaker genetic correlations with the other factors. Four phenotypes (educational attainment, insomnia, smoking and deprivation) showed significant, albeit small, sex-differentiated associations with the psychotic factor. As genome-wide association study samples grow and diversify, GSI will become increasingly valuable for comparing multivariate genetic architecture across groups. Genomic structural invariance, an extension of genomic structural equation modeling, identifies similarities and differences between males and females in the genetic architectures of eight psychiatric disorders spanning three broad factors.},
  archive      = {J_NG},
  author       = {Schwaba, Ted and Mallard, Travis T. and Maihofer, Adam X. and Rhemtulla, Mijke and Lee, Phil H. and Smoller, Jordan W. and Davis, Lea K. and Nivard, Michel G. and Grotzinger, Andrew D. and Tucker-Drob, Elliot M.},
  doi          = {10.1038/s41588-025-02093-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {583-590},
  shortjournal = {Nat. Genet.},
  title        = {Comparison of the multivariate genetic architecture of eight major psychiatric disorders across sex},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparative analysis of the mexico city prospective study
and the UK biobank identifies ancestry-specific effects on clonal
hematopoiesis. <em>NG</em>, <em>57</em>(3), 572–582. (<a
href="https://doi.org/10.1038/s41588-025-02085-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of genetic ancestry on the development of clonal hematopoiesis (CH) remains largely unexplored. Here, we compared CH in 136,401 participants from the Mexico City Prospective Study (MCPS) to 416,118 individuals from the UK Biobank (UKB) and observed CH to be significantly less common in MCPS compared to UKB (adjusted odds ratio = 0.59, 95% confidence interval (CI) = [0.57, 0.61], P = 7.31 × 10−185). Among MCPS participants, CH frequency was positively correlated with the percentage of European ancestry (adjusted beta = 0.84, 95% CI = [0.66, 1.03], P = 7.35 × 10−19). Genome-wide and exome-wide association analyses in MCPS identified ancestry-specific variants in the TCL1B locus with opposing effects on DNMT3A-CH versus non-DNMT3A-CH. Meta-analysis of MCPS and UKB identified five novel loci associated with CH, including polymorphisms at PARP11/CCND2, MEIS1 and MYCN. Our CH study, the largest in a non-European population to date, demonstrates the power of cross-ancestry comparisons to derive novel insights into CH pathogenesis. This paper compares the frequency and genetic basis of clonal hematopoiesis in 136,401 admixed American participants from the Mexico City Prospective Cohort with 416,118 largely European ancestry individuals from the UK Biobank cohort.},
  archive      = {J_NG},
  author       = {Wen, Sean and Kuri-Morales, Pablo and Hu, Fengyuan and Nag, Abhishek and Tachmazidou, Ioanna and Deevi, Sri V. V. and Taiy, Haeyam and Smith, Katherine R. and Loesch, Douglas P. and Burren, Oliver S. and Dhindsa, Ryan S. and Wasilewski, Sebastian and Alegre-Díaz, Jesus and Berumen, Jaime and Emberson, Jonathan and Torres, Jason M. and Collins, Rory and Carss, Keren and Wang, Quanli and Petrovski, Slavé and Tapia-Conyer, Roberto and Fabre, Margarete A. and Harper, Andrew R. and Vassiliou, George S. and Mitchell, Jonathan},
  doi          = {10.1038/s41588-025-02085-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {572-582},
  shortjournal = {Nat. Genet.},
  title        = {Comparative analysis of the mexico city prospective study and the UK biobank identifies ancestry-specific effects on clonal hematopoiesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evaluation of polygenic scores for hypertrophic
cardiomyopathy in the general population and across clinical settings.
<em>NG</em>, <em>57</em>(3), 563–571. (<a
href="https://doi.org/10.1038/s41588-025-02094-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertrophic cardiomyopathy (HCM) is an important cause of morbidity and mortality, with pathogenic variants found in about a third of cases. Large-scale genome-wide association studies (GWAS) demonstrate that common genetic variation contributes to HCM risk. Here we derive polygenic scores (PGS) from HCM GWAS and genetically correlated traits and test their performance in the UK Biobank, 100,000 Genomes Project, and clinical cohorts. We show that higher PGS significantly increases the risk of HCM in the general population, particularly among pathogenic variant carriers, where HCM penetrance differs 10-fold between those in the highest and lowest PGS quintiles. Among relatives of HCM probands, PGS stratifies risks of developing HCM and adverse outcomes. Finally, among HCM cases, PGS strongly predicts the risk of adverse outcomes and death. These findings support the broad utility of PGS across clinical settings, enabling tailored screening and surveillance and stratification of risk of adverse outcomes. Polygenic scores derived from genome-wide association analyses of hypertrophic cardiomyopathy and genetically correlated left ventricular traits effectively stratify risk of disease and adverse outcomes in the general population and across clinical cohorts.},
  archive      = {J_NG},
  author       = {Zheng, Sean L. and Jurgens, Sean J. and McGurk, Kathryn A. and Xu, Xiao and Grace, Chris and Theotokis, Pantazis I. and Buchan, Rachel J. and Francis, Catherine and de Marvao, Antonio and Curran, Lara and Bai, Wenjia and Pua, Chee Jian and Tang, Hak Chiaw and Jorda, Paloma and van Slegtenhorst, Marjon A. and Verhagen, Judith M. A. and Harper, Andrew R. and Ormondroyd, Elizabeth and Chin, Calvin W. L. and Pantazis, Antonis and Baksi, John and Halliday, Brian P. and Matthews, Paul and Pinto, Yigal M. and Walsh, Roddy and Amin, Ahmad S. and Wilde, Arthur A. M. and Cook, Stuart A. and Prasad, Sanjay K. and Barton, Paul J. R. and O’Regan, Declan P. and Lumbers, R. T. and Goel, Anuj and Tadros, Rafik and Michels, Michelle and Watkins, Hugh and Bezzina, Connie R. and Ware, James S.},
  doi          = {10.1038/s41588-025-02094-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {563-571},
  shortjournal = {Nat. Genet.},
  title        = {Evaluation of polygenic scores for hypertrophic cardiomyopathy in the general population and across clinical settings},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sequencing in over 50,000 cases identifies coding and
structural variation underlying atrial fibrillation risk. <em>NG</em>,
<em>57</em>(3), 548–562. (<a
href="https://doi.org/10.1038/s41588-025-02074-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atrial fibrillation (AF) is a prevalent and morbid abnormality of the heart rhythm with a strong genetic component. Here, we meta-analyzed genome and exome sequencing data from 36 studies that included 52,416 AF cases and 277,762 controls. In burden tests of rare coding variation, we identified novel associations between AF and the genes MYBPC3, LMNA, PKP2, FAM189A2 and KDM5B. We further identified associations between AF and rare structural variants owing to deletions in CTNNA3 and duplications of GATA4. We broadly replicated our findings in independent samples from MyCode, deCODE and UK Biobank. Finally, we found that CRISPR knockout of KDM5B in stem-cell-derived atrial cardiomyocytes led to a shortening of the action potential duration and widespread transcriptomic dysregulation of genes relevant to atrial homeostasis and conduction. Our results highlight the contribution of rare coding and structural variants to AF, including genetic links between AF and cardiomyopathies, and expand our understanding of the rare variant architecture for this common arrhythmia. Analyses of genome and exome sequencing data identify rare coding and structural variants associated with atrial fibrillation and highlight genetic links between atrial fibrillation and cardiomyopathies.},
  archive      = {J_NG},
  author       = {Choi, Seung Hoan and Jurgens, Sean J. and Xiao, Ling and Hill, Matthew C. and Haggerty, Christopher M. and Sveinbjörnsson, Garðar and Morrill, Valerie N. and Marston, Nicholas A. and Weng, Lu-Chen and Pirruccello, James P. and Arnar, David O. and Gudbjartsson, Daniel Fannar and Mantineo, Helene and von Falkenhausen, Aenne S. and Natale, Andrea and Tveit, Arnljot and Geelhoed, Bastiaan and Roselli, Carolina and Van Wagoner, David R. and Darbar, Dawood and Haase, Doreen and Soliman, Elsayed Z. and Davogustto, Giovanni E. and Jun, Goo and Calkins, Hugh and Anderson, Jeffrey L. and Brody, Jennifer A. and Halford, Jennifer L. and Barnard, John and Hokanson, John E. and Smith, Jonathan D. and Bis, Joshua C. and Young, Kendra and Johnson, Linda S. B. and Risch, Lorenz and Gula, Lorne J. and Kwee, Lydia Coulter and Chaffin, Mark D. and Kühne, Michael and Preuss, Michael and Gupta, Namrata and Nafissi, Navid A. and Smith, Nicholas L. and Nilsson, Peter M. and van der Harst, Pim and Wells, Quinn S. and Judy, Renae L. and Schnabel, Renate B. and Johnson, Renee and Smit, Roelof A. J. and Gabriel, Stacey and Knight, Stacey and Furukawa, Tetsushi and Blackwell, Thomas W. and Nauffal, Victor and Wang, Xin and Min, Yuan-I and Yoneda, Zachary T. and Laksman, Zachary W. M. and Bezzina, Connie R. and Alonso, Alvaro and Psaty, Bruce M. and Albert, Christine M. and Arking, Dan E. and Roden, Dan M. and Chasman, Daniel I. and Rader, Daniel J. and Conen, David and McManus, David D. and Fatkin, Diane and Benjamin, Emelia J. and Boerwinkle, Eric and Marcus, Gregory M. and Christophersen, Ingrid E. and Smith, J. Gustav and Roberts, Jason D. and Raffield, Laura M. and Shoemaker, M. Benjamin and Cho, Michael H. and Cutler, Michael J. and Rienstra, Michiel and Chung, Mina K. and S. Olesen, Morten and Sinner, Moritz F. and Sotoodehnia, Nona and Kirchhof, Paulus and Loos, Ruth J. F. and Nazarian, Saman and Mohanty, Sanghamitra and Damrauer, Scott M. and Kaab, Stefan and Heckbert, Susan R. and Redline, Susan and Shah, Svati H. and Tanaka, Toshihiro and Ebana, Yusuke and Holm, Hilma and Stefansson, Kari and Ruff, Christian T. and Sabatine, Marc S. and Lunetta, Kathryn L. and Lubitz, Steven A. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-025-02074-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {548-562},
  shortjournal = {Nat. Genet.},
  title        = {Sequencing in over 50,000 cases identifies coding and structural variation underlying atrial fibrillation risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Meta-analysis of genome-wide associations and polygenic risk
prediction for atrial fibrillation in more than 180,000 cases.
<em>NG</em>, <em>57</em>(3), 539–547. (<a
href="https://doi.org/10.1038/s41588-024-02072-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atrial fibrillation (AF) is the most common heart rhythm abnormality and is a leading cause of heart failure and stroke. This large-scale meta-analysis of genome-wide association studies increased the power to detect single-nucleotide variant associations and found more than 350 AF-associated genetic loci. We identified candidate genes related to muscle contractility, cardiac muscle development and cell–cell communication at 139 loci. Furthermore, we assayed chromatin accessibility using assay for transposase-accessible chromatin with sequencing and histone H3 lysine 4 trimethylation in stem cell-derived atrial cardiomyocytes. We observed a marked increase in chromatin accessibility for our sentinel variants and prioritized genes in atrial cardiomyocytes. Finally, a polygenic risk score (PRS) based on our updated effect estimates improved AF risk prediction compared to the CHARGE-AF clinical risk score and a previously reported PRS for AF. The doubling of known risk loci will facilitate a greater understanding of the pathways underlying AF. Genome-wide association meta-analyses identify more than 350 loci associated with atrial fibrillation. A polygenic score derived from these results improves atrial fibrillation risk prediction compared to published clinical and polygenic scores.},
  archive      = {J_NG},
  author       = {Roselli, Carolina and Surakka, Ida and Olesen, Morten S. and Sveinbjornsson, Gardar and Marston, Nicholas A. and Choi, Seung Hoan and Holm, Hilma and Chaffin, Mark and Gudbjartsson, Daniel and Hill, Matthew C. and Aegisdottir, Hildur and Albert, Christine M. and Alonso, Alvaro and Anderson, Christopher D. and Arking, Dan E. and Arnar, David O. and Barnard, John and Benjamin, Emelia J. and Braunwald, Eugene and Brumpton, Ben and Campbell, Archie and Chami, Nathalie and Chasman, Daniel I. and Cho, Kelly and Choi, Eue-Keun and Christophersen, Ingrid E. and Chung, Mina K. and Conen, David and Crijns, Harry J. and Cutler, Michael J. and Czuba, Tomasz and Damrauer, Scott M. and Dichgans, Martin and Dörr, Marcus and Dudink, Elton and Duong, ThuyVy and Erikstrup, Christian and Esko, Tõnu and Fatkin, Diane and Faul, Jessica D. and Ferreira, Manuel and Freitag, Daniel F. and Ganesh, Santhi K. and Gaziano, J. Michael and Geelhoed, Bastiaan and Ghouse, Jonas and Gieger, Christian and Giulianini, Franco and Graham, Sarah E. and Gudnason, Vilmundur and Guo, Xiuqing and Haggerty, Christopher and Hayward, Caroline and Heckbert, Susan R. and Hveem, Kristian and Ito, Kaoru and Johnson, Renee and Jukema, J. Wouter and Jurgens, Sean J. and Kääb, Stefan and Kane, John P. and Kany, Shinwan and Kardia, Sharon L. R. and Kavousi, Maryam and Khurshid, Shaan and Kamanu, Frederick K. and Kirchhof, Paulus and Kleber, Marcus E. and Knight, Stacey and Komuro, Issei and Krieger, Jose E. and Launer, Lenore J. and Li, Dadong and Lin, Honghuang and Lin, Henry J. and Loos, Ruth J. F. and Lotta, Luca and Lubitz, Steven A. and Lunetta, Kathryn L. and Macfarlane, Peter W. and Magnusson, Patrik K. E. and Malik, Rainer and Mantineo, Helene and Marcus, Gregory M. and März, Winfried and McManus, David D. and Melander, Olle and Melloni, Giorgio E. M. and Meyre, Pascal B. and Miyazawa, Kazuo and Mohanty, Sanghamitra and Monfort, Laia M. and Müller-Nurasyid, Martina and Nafissi, Navid A. and Natale, Andrea and Nazarian, Saman and Ostrowski, Sisse R. and Pak, Hui-Nam and Pang, Shichao and Pedersen, Ole B. and Pedersen, Nancy L. and Pereira, Alexandre C. and Pirruccello, James P. and Preuss, Michael and Psaty, Bruce M. and Pullinger, Clive R. and Rader, Daniel J. and Rämö, Joel T. and Ridker, Paul M. and Rienstra, Michiel and Risch, Lorenz and Roden, Dan M. and Rotter, Jerome I. and Sabatine, Marc S. and Schunkert, Heribert and Shah, Svati H. and Shim, Jaemin and Shoemaker, M. Benjamin and Simonson, Bridget and Sinner, Moritz F. and Smit, Roelof A. J. and Smith, Jennifer A. and Smith, Nicholas L. and Smith, J. Gustav and Soliman, Elsayed Z. and Sørensen, Erik and Sotoodehnia, Nona and Strbian, Daniel and Stricker, Bruno H. and Teder-Laving, Maris and Sun, Yan V. and Thériault, Sébastien and Thorolfsdottir, Rosa B. and Thorsteinsdottir, Unnur and Tveit, Arnljot and van der Harst, Pim and van Meurs, Joyce and Wang, Biqi and Weiss, Stefan and Wells, Quinn S. and Weng, Lu-Chen and Wilson, Peter W. and Xiao, Ling and Yang, Pil-Sung and Yao, Jie and Yoneda, Zachary T. and Zeller, Tanja and Zeng, Lingyao and Zhao, Wei and Zhou, Xiang and Zöllner, Sebastian and Ruff, Christian T. and Bundgaard, Henning and Willer, Cristen and Stefansson, Kari and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-024-02072-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {539-547},
  shortjournal = {Nat. Genet.},
  title        = {Meta-analysis of genome-wide associations and polygenic risk prediction for atrial fibrillation in more than 180,000 cases},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large-scale genome-wide association analyses identify novel
genetic loci and mechanisms in hypertrophic cardiomyopathy. <em>NG</em>,
<em>57</em>(3), 530–538. (<a
href="https://doi.org/10.1038/s41588-025-02087-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertrophic cardiomyopathy (HCM) is an important cause of morbidity and mortality with both monogenic and polygenic components. Here, we report results from a large genome-wide association study and multitrait analysis including 5,900 HCM cases, 68,359 controls and 36,083 UK Biobank participants with cardiac magnetic resonance imaging. We identified 70 loci (50 novel) associated with HCM and 62 loci (20 novel) associated with relevant left ventricular traits. Among the prioritized genes in the HCM loci, we identify a novel HCM disease gene, SVIL, which encodes the actin-binding protein supervillin, showing that rare truncating SVIL variants confer a roughly tenfold increased risk of HCM. Mendelian randomization analyses support a causal role of increased left ventricular contractility in both obstructive and nonobstructive forms of HCM, suggesting common disease mechanisms and anticipating shared response to therapy. Taken together, these findings increase our understanding of the genetic basis of HCM, with potential implications for disease management. Genome-wide and multitrait analyses identify novel loci associated with hypertrophic cardiomyopathy and relevant left ventricular traits. Gene-level burden analyses show that rare truncating SVIL variants are associated with high risk of hypertrophic cardiomyopathy.},
  archive      = {J_NG},
  author       = {Tadros, Rafik and Zheng, Sean L. and Grace, Christopher and Jordà, Paloma and Francis, Catherine and West, Dominique M. and Jurgens, Sean J. and Thomson, Kate L. and Harper, Andrew R. and Ormondroyd, Elizabeth and Xu, Xiao and Theotokis, Pantazis I. and Buchan, Rachel J. and McGurk, Kathryn A. and Mazzarotto, Francesco and Boschi, Beatrice and Pelo, Elisabetta and Lee, Michael and Noseda, Michela and Varnava, Amanda and Vermeer, Alexa M. C. and Walsh, Roddy and Amin, Ahmad S. and van Slegtenhorst, Marjon A. and Roslin, Nicole M. and Strug, Lisa J. and Salvi, Erika and Lanzani, Chiara and de Marvao, Antonio and Roberts, Jason D. and Tremblay-Gravel, Maxime and Giraldeau, Genevieve and Cadrin-Tourigny, Julia and L’Allier, Philippe L. and Garceau, Patrick and Talajic, Mario and Gagliano Taliun, Sarah A. and Pinto, Yigal M. and Rakowski, Harry and Pantazis, Antonis and Bai, Wenjia and Baksi, John and Halliday, Brian P. and Prasad, Sanjay K. and Barton, Paul J. R. and O’Regan, Declan P. and Cook, Stuart A. and de Boer, Rudolf A. and Christiaans, Imke and Michels, Michelle and Kramer, Christopher M. and Ho, Carolyn Y. and Neubauer, Stefan and Matthews, Paul M. and Wilde, Arthur A. M. and Tardif, Jean-Claude and Olivotto, Iacopo and Adler, Arnon and Goel, Anuj and Ware, James S. and Bezzina, Connie R. and Watkins, Hugh},
  doi          = {10.1038/s41588-025-02087-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {530-538},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale genome-wide association analyses identify novel genetic loci and mechanisms in hypertrophic cardiomyopathy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The transcriptomic architecture of common cancers reflects
synthetic lethal interactions. <em>NG</em>, <em>57</em>(3), 522–529. (<a
href="https://doi.org/10.1038/s41588-025-02108-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from &amp;gt;9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection. Tumor cells upregulate compensatory buffering genes following tumor suppressor loss. These genes may represent new synthetic lethal partners that could be harnessed therapeutically.},
  archive      = {J_NG},
  author       = {Haider, Syed and Brough, Rachel and Madera, Santiago and Iacovacci, Jacopo and Gulati, Aditi and Wicks, Andrew and Alexander, John and Pettitt, Stephen J. and Tutt, Andrew N. J. and Lord, Christopher J.},
  doi          = {10.1038/s41588-025-02108-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {522-529},
  shortjournal = {Nat. Genet.},
  title        = {The transcriptomic architecture of common cancers reflects synthetic lethal interactions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer-independent somatic mutation of the wild-type NF1
allele in normal tissues in neurofibromatosis type 1. <em>NG</em>,
<em>57</em>(3), 515–521. (<a
href="https://doi.org/10.1038/s41588-025-02097-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer predisposition syndromes mediated by recessive cancer genes generate tumors via somatic variants (second hits) in the unaffected allele. Second hits may or may not be sufficient for neoplastic transformation. Here we performed whole-genome and whole-exome sequencing on 479 tissue biopsies from a child with neurofibromatosis type 1, a multisystem cancer-predisposing syndrome mediated by constitutive monoallelic NF1 inactivation. We identified multiple independent NF1 driver variants in histologically normal tissues, but not in 610 biopsies from two nonpredisposed children. We corroborated this finding using targeted duplex sequencing, including a further nine adults with the same syndrome. Overall, truncating NF1 mutations were under positive selection in normal tissues from individuals with neurofibromatosis type 1. We demonstrate that normal tissues in neurofibromatosis type 1 commonly harbor second hits in NF1, the extent and pattern of which may underpin the syndrome’s cancer phenotype. Histologically normal tissues of individuals with a germline NF1 mutation exhibited multiple second NF1 hits, unrelated to their tumors, that conferred a selective advantage.},
  archive      = {J_NG},
  author       = {Oliver, Thomas R. W. and Lawson, Andrew R. J. and Lee-Six, Henry and Tollit, Anna and Jung, Hyunchul and Hooks, Yvette and Sanghvi, Rashesh and Young, Matthew D. and Butler, Timothy M. and Nicola, Pantelis A. and Treger, Taryn D. and Lensing, Stefanie V. and Burke, G. A. Amos and Aquilina, Kristian and Löbel, Ulrike and Cortes-Ciriano, Isidro and Hargrave, Darren and Jorgensen, Mette and Jessop, Flora A. and Coorens, Tim H. H. and Flanagan, Adrienne M. and Allinson, Kieren and Martincorena, Inigo and Jacques, Thomas S. and Behjati, Sam},
  doi          = {10.1038/s41588-025-02097-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {515-521},
  shortjournal = {Nat. Genet.},
  title        = {Cancer-independent somatic mutation of the wild-type NF1 allele in normal tissues in neurofibromatosis type 1},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genomics of rare diseases in the greater middle east.
<em>NG</em>, <em>57</em>(3), 505–514. (<a
href="https://doi.org/10.1038/s41588-025-02075-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Greater Middle East (GME) represents a concentrated region of unparalleled genetic diversity, characterized by an abundance of distinct alleles, founder mutations and extensive autozygosity driven by high consanguinity rates. These genetic hallmarks present a unique, yet vastly untapped resource for genomic research on Mendelian diseases. Despite this immense potential, the GME continues to face substantial challenges in comprehensive data collection and analysis. This Perspective highlights the region’s unique position as a natural laboratory for genetic discovery and explores the challenges that have stifled progress thus far. Importantly, we propose strategic solutions, advocating for an all-inclusive research approach. With targeted investment and focused efforts, the latent genetic wealth in the GME can be transformed into a global hub for genomic research that will redefine and advance our understanding of the human genome. The Greater Middle East is home to a rich genetic diversity that remains largely untapped in genomic research. This Perspective explores its potential for rare disease research, obstacles limiting progress and strategies to drive genomic discoveries.},
  archive      = {J_NG},
  author       = {Chekroun, Ikram and Shenbagam, Shruti and Almarri, Mohamed A. and Mokrab, Younes and Uddin, Mohammed and Alkhnbashi, Omer S. and Zaki, Maha S. and Najmabadi, Hossein and Kahrizi, Kimia and Fakhro, Khalid A. and Almontashiri, Naif A. M. and Ali, Fahad R. and Özbek, Uğur and Reversade, Bruno and Alkuraya, Fowzan S. and Alsheikh-Ali, Alawi and Abou Tayoun, Ahmad N.},
  doi          = {10.1038/s41588-025-02075-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {505-514},
  shortjournal = {Nat. Genet.},
  title        = {Genomics of rare diseases in the greater middle east},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The african animal breeding network as a pathway towards
genetic improvement of livestock. <em>NG</em>, <em>57</em>(3), 498–504.
(<a href="https://doi.org/10.1038/s41588-025-02079-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transforming Africa’s agricultural production and food systems is an imperative to achieve the United Nations Sustainable Development Goals and deliver on the Africa Union’s 2063 vision, ‘the Africa we want’. Transforming livestock systems through genetic improvement will sustainably increase productivity and proffer an inclusive socioeconomic development of farming communities. The African Animal Breeding Network (AABNet) is a platform of highly knowledgeable geneticists, animal breeders and professionals willing to provide information, training, advice and support across the continent. It will leverage available human resources among its members, facilitate partnerships and investment and develop infrastructure for innovative livestock genetic improvement in Africa. This Perspective introduces the African Animal Breeding Network as a platform to develop infrastructure for innovative livestock genetic improvement in Africa and discusses the challenges in implementing successful animal breeding programs.},
  archive      = {J_NG},
  author       = {Djikeng, Appolinaire and Olori, Victor E. and Houaga, Isidore and Aggrey, Samuel E. and Mwai, Okeyo and Ibeagha-Awemu, Eveline M. and Mrode, Raphael and Chagunda, Mizeck G. G. and Tiambo, Christian K. and Rekaya, Romdhane and Nash, Oyenkanmi and Nziku, Zabron and Opoola, Oluyinka and Ntanganedzeni, Mapholi and Ekine-Dzivenu, Chinyere and Kahi, Alexander and Okeno, Tobias and Hickey, John M. and Enyew, Negussie and Rege, Edward J. O.},
  doi          = {10.1038/s41588-025-02079-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {498-504},
  shortjournal = {Nat. Genet.},
  title        = {The african animal breeding network as a pathway towards genetic improvement of livestock},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unraveling the role of succinyl-CoA in tumor immunity.
<em>NG</em>, <em>57</em>(3), 496–497. (<a
href="https://doi.org/10.1038/s41588-025-02076-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study highlights the crucial role of succinyl-CoA in tumor immunity by inducing the succinylation of PD-L1. We identified the succinyltransferase CPT1A as a potential therapeutic target. CPT1A could be a valuable biomarker for evaluating the efficacy of anti-PD-1 therapy and suggests a promising treatment strategy.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02076-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {496-497},
  shortjournal = {Nat. Genet.},
  title        = {Unraveling the role of succinyl-CoA in tumor immunity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Safeguard repressor locks hepatocyte identity and blocks
liver cancer. <em>NG</em>, <em>57</em>(3), 494–495. (<a
href="https://doi.org/10.1038/s41588-025-02082-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {PROX1 acts as a molecular ‘cell fate guardian’ in hepatocytes, preserving liver cell identity throughout life by actively repressing unwanted plasticity. PROX1 loss impairs liver regeneration and fuels tumor formation, whereas PROX1 activation halts cancer formation and progression, revealing unexpected therapeutic potential.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02082-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {494-495},
  shortjournal = {Nat. Genet.},
  title        = {Safeguard repressor locks hepatocyte identity and blocks liver cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Novel mRNA isoforms in human microglia refine genetic
associations with neurodegeneration. <em>NG</em>, <em>57</em>(3),
492–493. (<a href="https://doi.org/10.1038/s41588-025-02112-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read RNA sequencing of human microglia expands the catalog of mRNA isoforms in this important cell type and assists with the interpretation of genetic associations with Alzheimer’s disease and Parkinson’s disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02112-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {492-493},
  shortjournal = {Nat. Genet.},
  title        = {Novel mRNA isoforms in human microglia refine genetic associations with neurodegeneration},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Polyguanine microsatellites are robust replication clocks in
cancer. <em>NG</em>, <em>57</em>(3), 489–491. (<a
href="https://doi.org/10.1038/s41588-025-02098-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer phylogenetic trees describe the evolutionary relationship between primary tumors and metastatic sites. A study now shows that mutations in guanine homopolymer microsatellites represent accurate molecular clocks, revealing the number of cell divisions that have occurred during cancer development and progression.},
  archive      = {J_NG},
  author       = {Gejman, Ron S. and Izar, Benjamin},
  doi          = {10.1038/s41588-025-02098-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {489-491},
  shortjournal = {Nat. Genet.},
  title        = {Polyguanine microsatellites are robust replication clocks in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mass spectrometry-based mapping of plasma protein QTLs in
children and adolescents. <em>NG</em>, <em>57</em>(3), 487–488. (<a
href="https://doi.org/10.1038/s41588-025-02088-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectrometry-based proteomics demonstrated its increasing analytical power and unique strengths in measuring genome-wide plasma proteomes and identifying protein quantitative trait loci (pQTLs), as reported in a study of over 3,000 Danish children and adolescents.},
  archive      = {J_NG},
  author       = {Liu, Yansheng},
  doi          = {10.1038/s41588-025-02088-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {487-488},
  shortjournal = {Nat. Genet.},
  title        = {Mass spectrometry-based mapping of plasma protein QTLs in children and adolescents},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell mapping of HTT somatic repeat expansions.
<em>NG</em>, <em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02142-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02142-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell mapping of HTT somatic repeat expansions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput screening identifies modulators of gene
bursting. <em>NG</em>, <em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02143-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02143-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {High-throughput screening identifies modulators of gene bursting},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding immune escape in lung cancer. <em>NG</em>,
<em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02144-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02144-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Understanding immune escape in lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hyperactive histone genes fuel cancer. <em>NG</em>,
<em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02145-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02145-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Hyperactive histone genes fuel cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The federated european genome–phenome archive as a global
network for sharing human genomics data. <em>NG</em>, <em>57</em>(3),
481–485. (<a href="https://doi.org/10.1038/s41588-025-02101-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this era of rapidly expanding human genomics in research and healthcare, efficient data reuse is essential to maximize benefits for society. In response, the Federated European Genome–Phenome Archive (FEGA) was launched in 2022, and as of 2024, the FEGA network was composed of seven national nodes. Here we describe the complexities, challenges and achievements of FEGA, unravelling the dynamic interplay of regulatory frameworks, technical challenges and the shared vision of advancing genomic research.},
  archive      = {J_NG},
  author       = {D’Altri, Teresa and Freeberg, Mallory Ann and Curwin, Amy J. and Alonso, Ana and Freitas, Ana T. and Capella-Gutierrez, Salvador and Gadelha, Luiz and Hagwall, Anna and Hovig, Eivind and Kerry, Giselle and Kirli, Koray and Kochel, Krzysztof and Kohlbacher, Oliver and Korbel, Jan O. and Leinonen, Jaakko and Marciniak, Blazej and Oliveira, Jorge S. and Petersen, Kjell and Silva, Mário J. and Stegle, Oliver and Valencia, Alfonso and Viklund, Johan and Guigo, Roderic and Parkinson, Helen and Navarro, Arcadi and Rambla, Jordi and Keane, Thomas M.},
  doi          = {10.1038/s41588-025-02101-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {481-485},
  shortjournal = {Nat. Genet.},
  title        = {The federated european Genome–Phenome archive as a global network for sharing human genomics data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
