<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>JCC_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="jcc---206">JCC - 206</h2>
<ul>
<li><details>
<summary>
(2021). Local perturbations of periodic systems. Chemisorption and
point defects by GoGreenGo. <em>JCC</em>, <em>42</em>(32), 2352–2368.
(<a href="https://doi.org/10.1002/jcc.26766">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a software package GoGreenGo—an overlay aimed to model local perturbations of periodic systems due to either chemisorption or point defects. The electronic structure of an ideal crystal is obtained by worldwide-distributed standard quantum physics/chemistry codes, and then processed by various tools performing projection to atomic orbital basis sets. Starting from this, the perturbation is addressed by GoGreenGowith use of the Green&#39;s functions formalism, which allows evaluating its effect on the electronic structure, density matrix, and energy of the system. In the present contribution, the main accent is made on processes of chemical nature, such as chemisorption or doping. We address a general theory and its computational implementation supported by a series of test calculations of the electronic structure perturbations for benchmark model solids: simple, face-centered, and body-centered cubium systems. In addition, more realistic problems of local perturbations in graphene lattice, such as lattice substitution, vacancy, and “on-top” chemisorption, are considered.},
  archive      = {J_JCC},
  author       = {Ilya V. Popov and Timofei S. Kushnir and Andrei L. Tchougréeff},
  doi          = {10.1002/jcc.26766},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2352-2368},
  shortjournal = {J. Comput. Chem.},
  title        = {Local perturbations of periodic systems. chemisorption and point defects by GoGreenGo},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). XEDA, a fast and multipurpose energy decomposition analysis
program. <em>JCC</em>, <em>42</em>(32), 2341–2351. (<a
href="https://doi.org/10.1002/jcc.26765">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fast and multipurpose energy decomposition analysis (EDA) program, called XEDA, is introduced for quantitative analysis of intermolecular interactions. This program contains a series of variational EDA methods, including LMO-EDA, GKS-EDA and their extensions, to analyze non-covalent interactions and strong chemical bonds in various environments. XEDA is highly efficient with a similar computational scaling of single point energy calculations. Its efficiency and universality are validated by a series of test examples including van der Waals interactions, hydrogen bonds, radical–radical interactions and strong covalent bonds.},
  archive      = {J_JCC},
  author       = {Zhen Tang and Yanlin Song and Shu Zhang and Wei Wang and Yuan Xu and Di Wu and Wei Wu and Peifeng Su},
  doi          = {10.1002/jcc.26765},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2341-2351},
  shortjournal = {J. Comput. Chem.},
  title        = {XEDA, a fast and multipurpose energy decomposition analysis program},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Feshbach resonances in d + HD(v = 1, j = 0) reaction at low
collision energies. <em>JCC</em>, <em>42</em>(32), 2334–2340. (<a
href="https://doi.org/10.1002/jcc.26763">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Feshbach resonances in D + HD(v= 1,j= 0) reaction are studied by using the time-independent quantum method. The integral cross section (ICS) results present three Feshbach resonance peaks, which are different from H + HD(v= 1,j= 0) reaction dominated by only one peak. These resonances are attributed to coupling with adiabatic effective potentials of D + HD(v= 1,j= 1) reaction, and the most obvious peak is contributed byJ= 1 at 83.16 cm−1collision energy. ForJ= 0 and 2, the resonances are related with the sameLpartial wave and present a double-peak structure in total ICS. The characteristics of product angular distribution show that the resonance ofJ= 1 is long-lived, while the lifetimes are relatively shorter for the resonance ofJ= 0 and 2.},
  archive      = {J_JCC},
  author       = {Yuwen Bai and Bayaer Buren and Zijiang Yang and Boyi Zhou and Maodu Chen},
  doi          = {10.1002/jcc.26763},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2334-2340},
  shortjournal = {J. Comput. Chem.},
  title        = {Feshbach resonances in d + HD(v = 1, j = 0) reaction at low collision energies},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Advances in vibrational configuration interaction theory -
part 1: Efficient calculation of vibrational angular momentum terms.
<em>JCC</em>, <em>42</em>(32), 2321–2333. (<a
href="https://doi.org/10.1002/jcc.26762">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finite basis vibrational configuration interaction theory (VCI) is a highly accurate method for the variational calculation of state energies and related properties, but suffers from fast growing computational costs in dependence of the size of the correlation space. In this series of papers, concepts and techniques will be presented, which diminish the computational demands and thus broaden the applicability of this method to larger molecules or more complex situations. This first part focuses on a highly efficient implementation of the vibrational angular momentum (VAM) terms as occurring in the Watson Hamiltonian and the prediagonalization of initial subspaces within an iterative configuration selective VCI implementation. Working equations and benchmark calculations are provided, the latter demonstrating the increased performance of the new algorithm.},
  archive      = {J_JCC},
  author       = {Tina Mathea and Guntram Rauhut},
  doi          = {10.1002/jcc.26762},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2321-2333},
  shortjournal = {J. Comput. Chem.},
  title        = {Advances in vibrational configuration interaction theory - part 1: Efficient calculation of vibrational angular momentum terms},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Evaluation of a new series of pyrazole derivatives as a
potent epidermal growth factor receptor inhibitory activity: QSAR
modeling using quantum-chemical descriptors. <em>JCC</em>,
<em>42</em>(32), 2306–2320. (<a
href="https://doi.org/10.1002/jcc.26761">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pyrazole derivatives correspond to a family of heterocycle molecules with important pharmacological and physiological applications. At present, we perform a density functional theory (DFT) calculations and a quantitative structure–activity relationship (QSAR) evaluation on a series of 1-(4,5-dihydro-1H-pyrazol-1-yl) ethan-1-one and 4,5-dihydro-1H-pyrazole-1-carbothioamide derivatives as an epidermal growth factor receptor (EGFR) inhibitory activity. We thus propose a virtual screening protocol based on a machine-learning study. This theoretical model relates the studied compounds&#39; biological activity to their calculated physicochemical descriptors. Moreover, the linear regression function is used to validate the model via the evaluation of Q2extand Q2cvparameters for external and internal validations, respectively. Our QSAR model shows a good correlation between observed activities IC50and predicted ones. Our model allows us to mitigate time-consuming problems and waste chemical and biological products in the preclinical phases.},
  archive      = {J_JCC},
  author       = {Ridha Ben Said and Riadh Hanachi and Seyfeddine Rahali and Mohammed A. M. Alkhalifah and Faisal Alresheedi and Bahoueddine Tangour and Majdi Hochlaf},
  doi          = {10.1002/jcc.26761},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2306-2320},
  shortjournal = {J. Comput. Chem.},
  title        = {Evaluation of a new series of pyrazole derivatives as a potent epidermal growth factor receptor inhibitory activity: QSAR modeling using quantum-chemical descriptors},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A complex gaussian approach to molecular photoionization.
<em>JCC</em>, <em>42</em>(32), 2294–2305. (<a
href="https://doi.org/10.1002/jcc.26760">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We develop and implement a Gaussian approach to calculate partial cross-sections and asymmetry parameters for molecular photoionization. Optimal sets of complex Gaussian-type orbitals (cGTOs) are first obtained by nonlinear optimization, to best fit sets of Coulomb or distorted continuum wave functions for relevant orbital quantum numbers. This allows us to represent the radial wavefunction for the outgoing electron with accurate cGTO expansions. Within a time-independent partial wave approach, we show that all the necessary transition integrals become analytical, in both length and velocity gauges, thus facilitating the numerical evaluation of photoionization observables. Illustrative results, presented for NH3and H2O within a one-active-electron monocentric model, validate numerically the proposed strategy based on a complex Gaussian representation of continuum states.},
  archive      = {J_JCC},
  author       = {Abdallah Ammar and Lorenzo Ugo Ancarani and Arnaud Leclerc},
  doi          = {10.1002/jcc.26760},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2294-2305},
  shortjournal = {J. Comput. Chem.},
  title        = {A complex gaussian approach to molecular photoionization},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Repurposing fusion inhibitor peptide against SARS-CoV-2.
<em>JCC</em>, <em>42</em>(32), 2283–2293. (<a
href="https://doi.org/10.1002/jcc.26758">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is continuously evolving. Although several vaccines were approved, this pandemic is still a major threat to public life. Till date, no established therapies are available against SARS-CoV-2. Peptide inhibitors hold great promise for this viral pathogen due to their efficacy, safety, and specificity. In this study, seventeen antiviral peptides which were known to inhibit SARS-CoV-1 are collected and computationally screened against heptad repeat 1 (HR1) of the SARS-CoV-2 spike protein (S2). Out of 17 peptides, Fp13 and Fp14 showed better binding affinity toward HR1 compared to a control peptide EK1 (a modified pan-coronavirus fusion inhibitor) in molecular docking. To explore the time-dependent interactions of the fusion peptide with HR1, molecular dynamics simulation was performed incorporating lipid membrane. During 100 ns MD simulation, structural and energy parameters of Fp13-HR1 and Fp14-HR1 complexes demonstrated lower fluctuations compared to the control EK1-HR1 complex. Furthermore, principal component analysis and free energy landscape study revealed that these two peptides (Fp13 and Fp14) strongly bind to the HR1 with higher affinity than that of control EK1. Tyr917, Asn919, Gln926, lys933, and Gln949 residues in HR1 protein were found to be crucial residues for peptide interaction. Notably, Fp13, Fp14 showed reasonably better binding free energy and hydrogen bond contribution than that of EK1. Taken together, Fp13 and Fp14 peptides may be highly specific for HR1 which can potentially prevent the formation of the fusion core and could be further developed as therapeutics for treatment or prophylaxis of SARS-CoV-2 infection.},
  archive      = {J_JCC},
  author       = {Faiyaz Md. Efaz and Shafiqul Islam and Shafi Ahmad Talukder and Shaila Akter and Md. Zakaria Tashrif and Md. Ackas Ali and Md. Abu Sufian and Md. Rimon Parves and Md. Jahirul Islam and Mohammad A. Halim},
  doi          = {10.1002/jcc.26758},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2283-2293},
  shortjournal = {J. Comput. Chem.},
  title        = {Repurposing fusion inhibitor peptide against SARS-CoV-2},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An automatized workflow from molecular dynamic simulation to
quantum chemical methods to identify elementary reactions and compute
reaction constants. <em>JCC</em>, <em>42</em>(32), 2264–2282. (<a
href="https://doi.org/10.1002/jcc.26757">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an automatized workflow which, starting from molecular dynamics simulations, identifies reaction events, filters them, and prepares them for accurate quantum chemical calculations using, for example, Density Functional Theory (DFT) or Coupled Cluster methods. The capabilities of the automatized workflow are demonstrated by the example of simulations for the combustion of some polycyclic aromatic hydrocarbons (PAHs). It is shown how key elementary reaction candidates are filtered out of a much larger set of redundant reactions and refined further. The molecular species in question are optimized using DFT and reaction energies, barrier heights, and reaction rates are calculated. The setup is general enough to include at this stage configurational sampling, which can be exploited in the future. Using the introduced machinery, we investigate how the observed reaction types depend on the gas atmosphere used in the molecular dynamics simulation. For the re-optimization on the DFT level, we show how the additional information needed to switch from reactive force-field to electronic structure calculations can be filled in and study how well ReaxFF and DFT agree with each other and shine light on the perspective of using more accurate semi-empirical methods in the MD simulation.},
  archive      = {J_JCC},
  author       = {Gunnar Schmitz and Özlem Yönder and Bastian Schnieder and Rochus Schmid and Christof Hättig},
  doi          = {10.1002/jcc.26757},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2264-2282},
  shortjournal = {J. Comput. Chem.},
  title        = {An automatized workflow from molecular dynamic simulation to quantum chemical methods to identify elementary reactions and compute reaction constants},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Normal-mode driven exploration of protein domain motions.
<em>JCC</em>, <em>42</em>(31), 2250–2257. (<a
href="https://doi.org/10.1002/jcc.26755">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Domain motions involved in the function of proteins can often be well described as a combination of motions along a handfull of low-frequency modes, that is, with the values of a few normal coordinates. This means that, when the functional motion of a protein is unknown, it should prove possible to predict it, since it amounts to guess a few values. However, without the help of additional experimental data, using normal coordinates for generating accurate conformers far away from the initial one is not so straightforward. To do so, a new approach is proposed: instead of building conformers directly with the values of a subset of normal coordinates, they are built in two steps, the conformer built with normal coordinates being just used for defining a set of distance constraints, the final conformer being built so as to match them. Note that this approach amounts to transform the problem of generating accurate protein conformers using normal coordinates into a better known one: the distance-geometry problem, which is herein solved with the help of the ROSETTA software. In the present study, this approach allowed to rebuild accurately six large amplitude conformational changes, using at most six low-frequency normal coordinates. As a consequence of the low-dimensionality of the corresponding subspace, random exploration also proved enough for generating low-energy conformers close to the known end-point of the conformational change of the LAO binding protein, lysozyme T4 and adenylate kinase.},
  archive      = {J_JCC},
  author       = {Yves-Henri Sanejouand},
  doi          = {10.1002/jcc.26755},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2250-2257},
  shortjournal = {J. Comput. Chem.},
  title        = {Normal-mode driven exploration of protein domain motions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A new protocol for the identification of singlet fission
sensitizers through computational screening. <em>JCC</em>,
<em>42</em>(31), 2241–2249. (<a
href="https://doi.org/10.1002/jcc.26753">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although singlet fission presents deep advantages or the generation of solar energy, the list of efficient singlet fission sensitizers is still very short, encouraging the theoreticians to focus their efforts on selecting and designing new candidates. Here, it is presented a computational protocol for the efficient screening of databases to select those species matching the energy requirements for singlet fission. Hence, out of the initial 29,123 species, 254 molecules (0.87\%) were found to match singlet fission energy conditions. The consideration of practical concerns such as availability or stability reduced the number to just 24 (0.08\%), among which the aminoanthraquinone derivatives are found particularly promising. The proposed protocol correct the deficiencies of the preceding ones, reaching DFT accuracy and minimizing the risk of getting false negatives especially at the early stages of the screening. In addition, this protocol can be used in future high-throughput investigations using datasets composed of millions of species.},
  archive      = {J_JCC},
  author       = {Diego López-Carballeira and Tomáš Polcar},
  doi          = {10.1002/jcc.26753},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2241-2249},
  shortjournal = {J. Comput. Chem.},
  title        = {A new protocol for the identification of singlet fission sensitizers through computational screening},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Boosting the conformational sampling by combining replica
exchange with solute tempering and well-sliced metadynamics.
<em>JCC</em>, <em>42</em>(31), 2233–2240. (<a
href="https://doi.org/10.1002/jcc.26752">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods that combine collective variable (CV) based enhanced sampling and global tempering approaches are used in speeding-up the conformational sampling and free energy calculation of large and soft systems with a plethora of energy minima. In this paper, a new method of this kind is proposed in which the well-sliced metadynamics approach (WSMTD) is united with replica exchange with solute tempering (REST2) method. WSMTD employs a divide-and-conquer strategy wherein high-dimensional slices of a free energy surface are independently sampled and combined. The method enables one to accomplish a controlled exploration of the CV-space with a restraining bias as in umbrella sampling, and enhance-sampling of one or more orthogonal CVs using a metadynamics like bias. The new hybrid method proposed here enables boosting the sampling of more slow degrees of freedom in WSMTD simulations, without the need to specify associated CVs, through a replica exchange scheme within the framework of REST2. The high-dimensional slices of the probability distributions of CVs computed from the united WSMTD and REST2 simulations are subsequently combined using the weighted histogram analysis method to obtain the free energy surface. We show that the new method proposed here is accurate, improves the conformational sampling, and achieves quick convergence in free energy estimates. We demonstrate this by computing the conformational free energy landscapes of solvated alanine tripeptide and Trp-cage mini protein in explicit water.},
  archive      = {J_JCC},
  author       = {Anji Babu Kapakayala and Nisanth N. Nair},
  doi          = {10.1002/jcc.26752},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2233-2240},
  shortjournal = {J. Comput. Chem.},
  title        = {Boosting the conformational sampling by combining replica exchange with solute tempering and well-sliced metadynamics},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Free energy of micellization of dodecyl phosphocholine (DPC)
from molecular simulation: Hybrid PEACH-BAR method. <em>JCC</em>,
<em>42</em>(31), 2221–2232. (<a
href="https://doi.org/10.1002/jcc.26751">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new method to extract the free energy of aggregation versus aggregate size from molecular simulation data is proposed and applied to a united atom model of the zwitterionic surfactant dodecyl phosphocholine in water. This system&#39;s slow dissociation rate and low critical micelle concentration (CMC of approximately 1–2 mM) make extraction of cluster free energies directly from simulation results using the “partition-enabled analysis of cluster histogram” (PEACH) method impractical. The new approach applies PEACH to a model with weakened attractions between aggregants, which allows sampling of a continuous range of cluster sizes, then recovers the free energy of aggregation under the original fully-attractive force field using the BAR free energy difference method. PEACH-BAR results are compared with free energy differences calculated via umbrella sampling, and are used to make predictions of CMC, average cluster size, and SAXS scattering profiles that are in fair agreement with experiment.},
  archive      = {J_JCC},
  author       = {Xiaokun Zhang and James T. Kindt},
  doi          = {10.1002/jcc.26751},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2221-2232},
  shortjournal = {J. Comput. Chem.},
  title        = {Free energy of micellization of dodecyl phosphocholine (DPC) from molecular simulation: Hybrid PEACH-BAR method},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Temperature-dependent structural fluctuation and its effect
on the electronic structure and charge transport in hybrid perovskite
CH3NH3PbI3. <em>JCC</em>, <em>42</em>(31), 2213–2220. (<a
href="https://doi.org/10.1002/jcc.26750">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recently discovered hybrid organic–inorganic perovskites have been suggested for high-performance optoelectronic applications. Owing to the mechanical flexibility of these compounds, they demonstrate structural fluctuation at finite temperatures that have been widely discussed with respect to their optical properties. However, the effect of temperature-induced structural fluctuation is not clear until now, with respect to the equally important charge transport properties. In the present study, throughab initiomolecular dynamics simulations of cubic-phase CH3NH3PbI3at different temperatures, the temperature-dependent electronic structure and charge carrier transport properties are examined. Compared with the significant structural fluctuation of organic cations, the structural change of the inorganic framework is minor. In addition, because the band edge states atRpoint are mainly influenced by the anti-bonding character of the Pb-I bond, CH3NH3PbI3demonstrates relatively small deformation potentials as well as low temperature dependence of band gaps (ΔEg≈ 50 meV from 330 K to 400 K) and electron–phonon coupling strengths, despite the large structural fluctuation of organic cations. Furthermore, the effective mass of the valence band increases with the increase of temperature. The predicted mobilities of CH3NH3PbI3can reach above 75 cm2V−1s−1near room temperature, exhibiting an appropriate optoelectronic potential, while the temperature dependence is steeper thanT−1.5of the traditional semiconductors because of the enhanced effective masses.},
  archive      = {J_JCC},
  author       = {Jinyang Xi and Liangliang Zheng and Shenghao Wang and Jiong Yang and Wenqing Zhang},
  doi          = {10.1002/jcc.26750},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2213-2220},
  shortjournal = {J. Comput. Chem.},
  title        = {Temperature-dependent structural fluctuation and its effect on the electronic structure and charge transport in hybrid perovskite CH3NH3PbI3},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The role of electric field on decomposition of CL-20/HMX
cocrystal: A reactive molecular dynamics study. <em>JCC</em>,
<em>42</em>(31), 2202–2212. (<a
href="https://doi.org/10.1002/jcc.26748">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electric field can initiate decomposition or detonation of explosives, but underlying mechanism is unclear. Here, we performed ReaxFF molecular dynamics simulation for decomposition of a cocrystal, formed by 2,4,6,8,10,12-hexanitro-2,4,6,8,10,12-hexaazaisowurtzitane (CL-20) and 1,3,5,7-tetranitro-1,3,5,7-tetrazacyclooctane (HMX), solely induced by electric field. A new analytical method was proposed to obtain detailed decomposition mechanism. Results show that electric fields play important roles in decomposition of CL-20/HMX cocrystal, such as heating the system and causing the explosive to decompose. Strong constant field makes CL-20 molecules in the cocrystal decompose at significantly lower temperature, which greatly increases sensitivity. This is ascribed to the distinct decomposition mechanism that CN bond rupture dominates the initial step of CL-20&#39;s decomposition. Contrarily, oscillating field has a stronger heating effect but weaker influence on sensitivity. Moreover, HMX exhibits desensitizing effect in CL-20/HMX cocrystal under electric field. These results enhance our understanding of sensitivity mechanism beyond mechanical stimuli in explosives.},
  archive      = {J_JCC},
  author       = {Jidong Zhang and Wei Guo},
  doi          = {10.1002/jcc.26748},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2202-2212},
  shortjournal = {J. Comput. Chem.},
  title        = {The role of electric field on decomposition of CL-20/HMX cocrystal: A reactive molecular dynamics study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Cover image. <em>JCC</em>, <em>42</em>(30), i. (<a
href="https://doi.org/10.1002/jcc.26771">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Yuxia Mao and Shiliang Li and Bojie Gong and Luhua Lai and Gaoqi He and Honglin Li},
  doi          = {10.1002/jcc.26771},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). EPharmer: An integrated and graphical software for
pharmacophore-based virtual screening. <em>JCC</em>, <em>42</em>(30),
2181–2195. (<a href="https://doi.org/10.1002/jcc.26743">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pharmacophore-based virtual screening (VS) has emerged as an efficient computer-aided drug design technique when appraising multiple ligands with similar structures or targets with unknown crystal structures. Current pharmacophore modeling and analysis software suffers from inadequate integration of mainstream methods and insufficient user-friendly program interface. In this study, we propose a stand-alone, integrated, graphical software for pharmacophore-based VS, termed ePharmer. Both ligand-based and structure-based pharmacophore generation methods were integrated into a compact architecture. Fine-grained modules were carefully organized into the computing, integration, and visualization layers. Graphical design covered the global user interface and specific user operations including editing, evaluation, and task management. Metabolites prediction analysis with the chosen VS result is provided for preselection of wet experiments. Moreover, the underlying computing units largely adopted the preliminary work of our research team. The presented software is currently in client use and will be released for both professional and nonexpert users. Experimental results verified the favorable computing capability, user convenience, and case performance of the proposed software.},
  archive      = {J_JCC},
  author       = {Yuxia Mao and Shiliang Li and Bojie Gong and Luhua Lai and Gaoqi He and Honglin Li},
  doi          = {10.1002/jcc.26743},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2181-2195},
  shortjournal = {J. Comput. Chem.},
  title        = {EPharmer: An integrated and graphical software for pharmacophore-based virtual screening},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mixed-solvent molecular dynamics simulation-based discovery
of a putative allosteric site on regulator of g protein signaling 4.
<em>JCC</em>, <em>42</em>(30), 2170–2180. (<a
href="https://doi.org/10.1002/jcc.26747">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Regulator of G protein signaling 4 (RGS4) is an intracellular protein that binds to the Gαsubunit ofheterotrimeric G proteins and aids in terminating G protein coupled receptor signaling. RGS4 has been implicated in pain, schizophrenia, and the control of cardiac contractility. Inhibitors of RGS4 have been developed but bind covalently to cysteine residues on the protein. Therefore, we sought to identify alternative druggable sites on RGS4 using mixed-solvent molecular dynamics simulations, which employ low concentrations of organic probes to identify druggable hotspots on the protein. Pseudo-ligands were placed in consensus hotspots, and perturbation with normal mode analysis led to the identification and characterization of a putative allosteric site, which would be invaluable for structure-based drug design of non-covalent, small molecule inhibitors. Future studies on the mechanism of this allostery will aid in the development of novel therapeutics targeting RGS4.},
  archive      = {J_JCC},
  author       = {Wallace K. B. Chan and Debarati DasGupta and Heather A. Carlson and John R. Traynor},
  doi          = {10.1002/jcc.26747},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2170-2180},
  shortjournal = {J. Comput. Chem.},
  title        = {Mixed-solvent molecular dynamics simulation-based discovery of a putative allosteric site on regulator of g protein signaling 4},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A reaction route network for methanol decomposition on a
pt(111) surface. <em>JCC</em>, <em>42</em>(30), 2163–2169. (<a
href="https://doi.org/10.1002/jcc.26746">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A reaction route network for the decomposition reaction of methanol on a Pt(111) surface was constructed by using the artificial force-induced reaction (AFIR) method, which can search for reaction paths automatically and systematically. Then, the network was kinetically analyzed by applying the rate constant matrix contraction (RCMC) method. Specifically, the time hierarchy of the network, the time evolution of the population initially given to CH3OH to the other species on the network, and the most favorable route from CH3OH to major and minor products were investigated by the RCMC method. Consistently to previous studies, the major product on the network was CO+4H, and the most favorable route proceeded through the following steps: CH3OH → CH2OH+H → HCOH+2H → HCO+3H → CO+4H. Furthermore, paths to byproducts found on the network and their kinetic importance were discussed. The present procedure combining AFIR and RCMC was thus successful in explaining the title reaction without using any information on its product or the reaction mechanism.},
  archive      = {J_JCC},
  author       = {Kanami Sugiyama and Kenichiro Saita and Satoshi Maeda},
  doi          = {10.1002/jcc.26746},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2163-2169},
  shortjournal = {J. Comput. Chem.},
  title        = {A reaction route network for methanol decomposition on a pt(111) surface},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Property-oriented basis sets for computation of atomization
energies. <em>JCC</em>, <em>42</em>(30), 2154–2162. (<a
href="https://doi.org/10.1002/jcc.26745">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The SBO4-DZ(d,p)-3G basis sets introduced in a previous paper have been modified to improve their performance in the calculation of the atomization energies of organic molecules (pure or substituted hydrocarbons). The first step was to explore the possibility of improving those basis sets by adding a second D shell. The scale factor for an additional “D-3G” shell was first studied by minimizing the total energies. In a second step, the scale factor was calculated by optimizing atomization energies directly (instead total energies). This way the results obtained were very similar to those of the cc-pV5Z basis sets. Finally, we optimized simplified box orbital (SBO) basis sets for different methods obtaining optimal SBO basis sets for HF, DFTs, and MP2.},
  archive      = {J_JCC},
  author       = {Victor García and David Zorrilla and Manuel Fernández and Jesús Sánchez-Márquez},
  doi          = {10.1002/jcc.26745},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2154-2162},
  shortjournal = {J. Comput. Chem.},
  title        = {Property-oriented basis sets for computation of atomization energies},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Another look at energetically quasi-degenerate structures of
the gold cluster au27q with q = 1, 0, −1. <em>JCC</em>, <em>42</em>(30),
2145–2153. (<a href="https://doi.org/10.1002/jcc.26744">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum chemical computations were used to reinvestigate the geometries, spectroscopic, and energetic properties of the gold clusters Au27qin three charge states (q = 1, 0, −1). Density functional theory (DFT) and the domain-based local pair natural orbital modification of the coupled-cluster theory DLPNO-CCSD(T) calculations revealed that, at variance with earlier reports in the literature, while the anion Au27−tends to exist in a tube-like form, both the lowest-energy Au27and Au27+isomers exhibit a pyramidal shape. However, several isomers were found to lie very close in energy, thus rendering a structural transition and their coexistence easy to occur. More specifically, the equilibrium geometry of the neutral Au27is a core-shell pyramid-like structure with one gold atom located inside. We also identified a novel ground state for the anion Au27−and located for the first time the global minimum of the cation Au27+. The vertical detachment energies of the neutral and anionic states were also computed and used to assign the available experimental photoelectron spectra. Although many Au27isomers were predicted to be energetically quasi-degenerate, the corresponding distinctive vibrational signatures can be used as fingerprints for the identification of cluster geometrical features.},
  archive      = {J_JCC},
  author       = {Pham Vu Nhat and Nguyen Thanh Si and Vitaly G. Kiselev and Minh Tho Nguyen},
  doi          = {10.1002/jcc.26744},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2145-2153},
  shortjournal = {J. Comput. Chem.},
  title        = {Another look at energetically quasi-degenerate structures of the gold cluster au27q with q = 1, 0, −1},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Momentum removal to obtain the position-dependent diffusion
constant in constrained molecular dynamics simulation. <em>JCC</em>,
<em>42</em>(30), 2136–2144. (<a
href="https://doi.org/10.1002/jcc.26742">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The position-dependent diffusion coefficient along with free energy profile are important parameters needed to study mass transport in heterogeneous systems such as biological and polymer membranes, and molecular dynamics (MD) calculation is a popular tool to obtain them. Among many methodologies, the Marrink–Berendsen (MB) method is often employed to calculate the position-dependent diffusion coefficient, in which the autocorrelation function of the force on a fixed molecule is related to the friction on the molecule. However, the diffusion coefficient is shown to be affected by the period of the removal of the center-of-mass velocity, which is necessary when performing MD calculations using the Ewald method for Coulombic interaction. We have clarified theoretically in this study how this operation affects the diffusion coefficient calculated by the MB method, and the theoretical predictions are proven by MD calculations. Therefore, we succeeded in providing guidance on how to select an appropriatevalue in estimating the position-dependent diffusion coefficient by the MB method. This guideline is applicable also to the Woolf–Roux method.},
  archive      = {J_JCC},
  author       = {Kazushi Fujimoto and Tetsuro Nagai and Tsuyoshi Yamaguchi},
  doi          = {10.1002/jcc.26742},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2136-2144},
  shortjournal = {J. Comput. Chem.},
  title        = {Momentum removal to obtain the position-dependent diffusion constant in constrained molecular dynamics simulation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Cover image. <em>JCC</em>, <em>42</em>(29), i. (<a
href="https://doi.org/10.1002/jcc.26756">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Sara Tortorella and Emanuele Carosati and Giulia Sorbi and Giovanni Bocci and Simon Cross and Gabriele Cruciani and Loriano Storchi},
  doi          = {10.1002/jcc.26756},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). DJMol: An open-source modeling platform for computational
chemistry and materials science with a python interpreter. <em>JCC</em>,
<em>42</em>(29), 2116–2129. (<a
href="https://doi.org/10.1002/jcc.26740">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a modular and extendable software suite, DJMol, for performing molecular simulations and it is demonstrated with DFTB+, Siesta, Atomic Simulation Environment, and OpenMD codes. It supports many of the standard features of an integrated development environment and consists of a structure builder and viewer, which could be connected with these electronic structure codes along with a set of data analyzers. This program comprises Java and Python modules and its libraries to carry out a different set of modeling tasks in materials science and chemistry. By adopting a Python interpreter into the software, a range of scriptable Python codes, such as Pymatgen can be incorporated into this programmable modeling platform. DJMol, through its common application programming interface (API), supports multiple modeling codes in the backend and several post-processing tools. It benefits an experienced user by increasing efficiency, while a nonexpert user by easy to use API.},
  archive      = {J_JCC},
  author       = {Krishnamohan G. Prasanna and Rahul Sunil and Kapil Gupta and Seung-Cheol Lee},
  doi          = {10.1002/jcc.26740},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {2116-2129},
  shortjournal = {J. Comput. Chem.},
  title        = {DJMol: An open-source modeling platform for computational chemistry and materials science with a python interpreter},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Tuning the metal–ligand bond in the σ-complexes of
stannylenes and azabenzenes. <em>JCC</em>, <em>42</em>(29), 2103–2115.
(<a href="https://doi.org/10.1002/jcc.26741">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The metal–ligand bond in a set of 60σ-complexes has been investigated by electronic structure computations. Theseσ-complexes originate from the unique combination of 12 stannylenes (SnX2) with five azabenzene ligands (pyridine, pyrazine, pyrimidine, pyridazine, and s-triazine), where the nitrogen center of the ligand acts asσ-donor and the tin(II) center asσ-acceptor in a 1:1 fashion. The Sn ← N bond and the total interaction between the stannylene and azabenzene moieties of theσ-complexes are characterized in depth to relate the Sn ← N strength to the substitution pattern at SnX2and to the number and the positioning of N atoms in the azabenzenes. Such X substituents as (iso)cyano and trifluoromethyl groups enhance the interaction strength, while the presence of alkyl, phenyl, and silyl substituents in SnX2diminishes the stability ofσ-complexes. A gradual weakening of the total interaction is associated with the growing number of N atoms in the azabenzenes, while the N-atom positioning in pyridazine is particularly effective in strengthening the interaction with stannylenes. Variations in the Sn ← N bond strength usually follow those in the total interaction between the moieties but the interacting quantum atoms picture of Sn ← N reveals certain intriguing exceptions.},
  archive      = {J_JCC},
  author       = {Barbara Bankiewicz and Stephan Kupfer and Piotr Matczak},
  doi          = {10.1002/jcc.26741},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {2103-2115},
  shortjournal = {J. Comput. Chem.},
  title        = {Tuning the metal–ligand bond in the σ-complexes of stannylenes and azabenzenes},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Pseudospectral implementations of long-range corrected
density functional theory. <em>JCC</em>, <em>42</em>(29), 2089–2102. (<a
href="https://doi.org/10.1002/jcc.26739">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have implemented pseudospectral density-functional theory (DFT) with long-range corrected DFT functionals (PS-LRC) in quantum mechanics package Jaguar, and applied it in the calculations of geometry optimizations, dimmer interaction energies, polarizabilities and first-order hyperpolarizabilities, harmonic vibrational frequencies, S1and T1excitation energies, singlet-triplet gaps, charge transfer numbers, oscillator strengths, reaction barrier heights, electron-transfer couplings, and charge-transfer excitation energies. From our accuracy benchmark analysis, PS grids, PS dealiasing functions, PS atomic corrections, PS multigrid strategy, PS length scales, and PS cutoff scheme perform well in PS DFT with LRC density functionals with very small and ignorable deviations when compared to the conventional spectral (CS) method. The timing benchmark study of S1excitation energy calculations of fullerenes (Cn,nup to 540) demonstrates that PS-LRC achieves 1.4–8.4-fold speedups in SCF, 22–32-fold speedups in Tamm-Dancoff approximation, and 6–15-fold speedups in total wall clock time with an average error 0.004 eV of excitation energies compared to the CS method.},
  archive      = {J_JCC},
  author       = {Yixiang Cao and Mathew D. Halls and Tati Reddy Vadicherla and Richard A. Friesner},
  doi          = {10.1002/jcc.26739},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {2089-2102},
  shortjournal = {J. Comput. Chem.},
  title        = {Pseudospectral implementations of long-range corrected density functional theory},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Dependence of the substituent energy on the level of theory.
<em>JCC</em>, <em>42</em>(29), 2079–2088. (<a
href="https://doi.org/10.1002/jcc.26738">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most often, the substituent effects are described using rather troublesome Hammett constants. Quite recently, it has been proposed to use the so-called substituent energy, which is based on total energies of the X-substituted polycyclic aromatic hydrocarbon and phenyl. This article concerns the influence of the applied level of theory (i.e., both the basis set and the method) on the determined values of the substituent energies. For this purpose, the energies of theNH2andNO2groups in 16 unique positions of naphthalene, anthracene, tetracene, phenanthrene, and pyrene were calculated using 10 different basis sets and 23 various exchange-correlation functionals representing the entire Jacob&#39;s Ladder, from local, through gradient- and meta-gradient-corrected, to hybrid and double-hybrid ones. Additionally, using even larger 6-311++G(2df,2p) basis set, the energies ofNH2,NO2,CN, andCl were also computed. Both the basis set and the method used have little effect on the substituent energy if the substituent is in the benzene-like position. On the contrary, the effect of the level of theory is pronounced especially in the case of the most spatially crowded 4-substituted phenanthrene. Substituent energies have been shown to be very useful theoretical parameters describing the proximity effect in the substituted derivatives of polycyclic aromatic hydrocarbons.},
  archive      = {J_JCC},
  author       = {Mirosław Jabłoński and Tadeusz M. Krygowski},
  doi          = {10.1002/jcc.26738},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {2079-2088},
  shortjournal = {J. Comput. Chem.},
  title        = {Dependence of the substituent energy on the level of theory},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Combining machine learning and quantum mechanics yields
more chemically aware molecular descriptors for medicinal chemistry
applications. <em>JCC</em>, <em>42</em>(29), 2068–2078. (<a
href="https://doi.org/10.1002/jcc.26737">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular interaction fields (MIFs), describing molecules in terms of their ability to interact with any chemical entity, are one of the most established and versatile concepts in drug discovery. Improvement of this molecular description is highly desirable for in silico drug discovery and medicinal chemistry applications. In this work, we revised a well-established molecular mechanics&#39; force field and applied a hybrid quantum mechanics and machine learning approach to parametrize the hydrogen-bonding (HB) potentials of small molecules, improving this aspect of the molecular description. Approximately 66,000 molecules were chosen from available drug databases and subjected to density functional theory calculations (DFT). For each atom, the molecular electrostatic potential (EP) was extracted and used to derive new HB energy contributions; this was subsequently combined with a fingerprint-based description of the structural environment via partial least squares modeling, enabling the new potentials to be used for molecules outside of the training set. We demonstrate that parameter prediction for molecules outside of the training set correlates with their DFT-derived EP, and that there is correlation of the new potentials with hydrogen-bond acidity and basicity scales. We show the newly derived MIFs vary in strength for various ring substitution in accordance with chemical intuition. Finally, we report that this derived parameter, when extended to non-HB atoms, can also be used to estimate sites of reaction.},
  archive      = {J_JCC},
  author       = {Sara Tortorella and Emanuele Carosati and Giulia Sorbi and Giovanni Bocci and Simon Cross and Gabriele Cruciani and Loriano Storchi},
  doi          = {10.1002/jcc.26737},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {2068-2078},
  shortjournal = {J. Comput. Chem.},
  title        = {Combining machine learning and quantum mechanics yields more chemically aware molecular descriptors for medicinal chemistry applications},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Pseudopotentials for coarse-grained cross-link-assisted
modeling of protein structures. <em>JCC</em>, <em>42</em>(29),
2054–2067. (<a href="https://doi.org/10.1002/jcc.26736">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pseudopotentials for the chemical cross-links comprising the glutamic- and aspartic-acid side chains bridged with adipic- (ADH) or pimelic-acid hydrazide (PDH), and the lysine side chains bridged with glutaric (BS2G) or suberic acid (BS3) for coarse-grained cross-link-assisted simulations were determined by canonical molecular dynamics with the Amber14sb force field. The potentials depend on the distance between side-chain ends and on side-chain orientation, this preventing from making cross-link contacts across the globule in simulations. The potentials were implemented in the UNRES coarse-grained force field and their effect on the quality of models was assessed with 11 monomeric and 1 dimeric proteins, using synthetic or experimental cross-link data. Simulations with the new potentials resulted in improvement of the generated models compared to unrestrained simulations in more instances compared to those with the statistical potentials.},
  archive      = {J_JCC},
  author       = {Mateusz Kogut and Zhou Gong and Chun Tang and Adam Liwo},
  doi          = {10.1002/jcc.26736},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {2054-2067},
  shortjournal = {J. Comput. Chem.},
  title        = {Pseudopotentials for coarse-grained cross-link-assisted modeling of protein structures},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AutoMeKin2021: An open-source program for automated reaction
discovery. <em>JCC</em>, <em>42</em>(28), 2036–2048. (<a
href="https://doi.org/10.1002/jcc.26734">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AutoMeKin2021 is an updated version of tsscds2018, a program for the automated discovery of reaction mechanisms (J. Comput. Chem.2018,39, 1922). This release features a number of new capabilities: rare-event molecular dynamics simulations to enhance reaction discovery, extension of the original search algorithm to study van der Waals complexes, use of chemical knowledge, a new search algorithm based on bond-order time series analysis, statistics of the chemical reaction networks, a web application to submit jobs, and other features. The source code, manual, installation instructions and the website link are available at:https://rxnkin.usc.es/index.php/AutoMeKin},
  archive      = {J_JCC},
  author       = {Emilio Martínez-Núñez and George L. Barnes and David R. Glowacki and Sabine Kopec and Daniel Peláez and Aurelio Rodríguez and Roberto Rodríguez-Fernández and Robin J. Shannon and James J. P. Stewart and Pablo G. Tahoces and Saulo A. Vazquez},
  doi          = {10.1002/jcc.26734},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {2036-2048},
  shortjournal = {J. Comput. Chem.},
  title        = {AutoMeKin2021: An open-source program for automated reaction discovery},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CAN: A new program to streamline preparation of molecular
coordinate files for molecular dynamics simulations. <em>JCC</em>,
<em>42</em>(28), 2031–2035. (<a
href="https://doi.org/10.1002/jcc.26729">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Preparing molecular coordinate files for molecular dynamics (MD) simulations can be a very time-consuming process. Herein we present the development of a user-friendly program that drastically reduces the time required to prepare these molecular coordinate files for MD software packages such as AmberTools. Our program, known as charge atomtype naming (CAN), creates and uses a library of structures such as amino acid monomers to update the charge, atom type, and name of atoms in any molecular structure (mol2) file. We demonstrate the utility of this new program by rapidly preparing structural files for MD simulations for polypeptides ranging from small molecules to large protein structures. Both native and non-native amino acid residues are easily handled by this new program.},
  archive      = {J_JCC},
  author       = {Jacob A. Parkman and Caleb A. Barksdale and David J. Michaelis},
  doi          = {10.1002/jcc.26729},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {2031-2035},
  shortjournal = {J. Comput. Chem.},
  title        = {CAN: A new program to streamline preparation of molecular coordinate files for molecular dynamics simulations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Two novel semiconducting B2CO monolayers with high carrier
mobilities. <em>JCC</em>, <em>42</em>(28), 2024–2030. (<a
href="https://doi.org/10.1002/jcc.26735">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The design of new two-dimensional (2D) materials with moderate band gaps and high carrier mobility is an important aspiration for materials innovation. Recent studies have shown that boron and oxygen atoms can be integrated into the graphene lattice to form a stable B–C–O monolayer structure. To search for the most energetically stable configuration for 2D B–C–O, here, we theoretically propose two new 2D B–C–O crystal structures with a stoichiometric ratio of 2:1:1, namely monolayer (1 L) C3v-and C2v-B2CO. Two configurations have 0.09 and 0.03 eV/unit cell lower energies than the reported 1 L Cs-B2CO configuration (Nanoscale2016, 8, 8910). This result is further confirmed by particle swarm optimization (PSO) calculations. According to the chemical bonding analysis, 1 L C3v-B2CO with a quasi-planar configuration has the lowest energy, which is consisted of three strong B′–O σ-bonds, three B″–C σ-bonds, and one B′–C σ-bond. As a result, 2D B2CO has an ultra-high mechanical strength of ~366 J m−2, comparable to graphene ~352 J m−2. In addition, 1 L C3v-B2CO is a semiconductor with an HSE06 bandgap of 2.57 eV, and it has a high electron mobility of up to ~150 cm2v−1s−1. The high kinetic and thermodynamic stabilities of both 1 L C3v-and C2v-B2CO were confirmed according to phonon dispersion and molecular dynamic simulation. Comparable to that of crystalline silicon, 1 L C3v-B2CO also shows a high light absorption intensity in the 400–550 nm region. Therefore, 2D C3v-B2CO will have promising applications in semiconductor devices and photodetectors.},
  archive      = {J_JCC},
  author       = {Feng Li and Weibo Han and Zhi Cao and Shilei Ji and Haiyun Wang and Lixia Wang and Hong Wu and Yuping Zhu and Yong Pu},
  doi          = {10.1002/jcc.26735},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {2024-2030},
  shortjournal = {J. Comput. Chem.},
  title        = {Two novel semiconducting B2CO monolayers with high carrier mobilities},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Hydration of selenolate moiety: Ab initio investigation of
properties of o–h⋯se(–) hydrogen bonds in CH3Se(–)⋯(H2O)n clusters.
<em>JCC</em>, <em>42</em>(28), 2014–2023. (<a
href="https://doi.org/10.1002/jcc.26733">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This work is devoted to investigations of the influence of O–H···Se(–) hydrogen bonds on the electronic shells of selenolate R–Se(–) fragment (R═CH3). The geometric, energetic and nuclear magnetic resonance (NMR) spectral parameters for various conformers of CH3Se(–)⋯(H2O)nclusters withn= 0–6 were calculated at CCSD/aug-cc-pVDZ level of theory. For selenolate anion CH3Se(–) solvation free energy was calculated, and for water media it is equal to −71.41 kcal/mol. For O–H···Se(–) hydrogen bonds the proportionality coefficients between QTAIM parameters at (3; −1) bond critical point and the strength of an individual hydrogen bond ∆Ewere proposed. It was shown, that despite a relative weakness of O–H···Se(–) hydrogen bonds, the outer electronic shell of the selenium atom changes significantly upon formation of each hydrogen bond. This, in turn, cause the dramatic change of NMR parameters of selenium nuclei.},
  archive      = {J_JCC},
  author       = {Valerii V. Karpov and Aleksandra M. Puzyk and Peter M. Tolstoy and Elena Yu. Tupikina},
  doi          = {10.1002/jcc.26733},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {2014-2023},
  shortjournal = {J. Comput. Chem.},
  title        = {Hydration of selenolate moiety: Ab initio investigation of properties of O–H⋯Se(–) hydrogen bonds in CH3Se(–)⋯(H2O)n clusters},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MCML: Combining physical constraints with experimental data
for a multi-purpose meta-generalized gradient approximation.
<em>JCC</em>, <em>42</em>(28), 2004–2013. (<a
href="https://doi.org/10.1002/jcc.26732">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The predictive power of density functional theory for materials properties can be improved without increasing the overall computational complexity by extending the generalized gradient approximation (GGA) for electronic exchange and correlation to density functionals depending on the electronic kinetic energy density in addition to the charge density and its gradient, resulting in a meta-GGA. Here, we propose an empirical meta-GGA model that is based both on physical constraints and on experimental and quantum chemistry reference data. The resulting optimized meta-GGA MCML yields improved surface and gas phase reaction energetics without sacrificing the accuracy of bulk property predictions of existing meta-GGA approaches.},
  archive      = {J_JCC},
  author       = {Kristopher Brown and Yasheng Maimaiti and Kai Trepte and Thomas Bligaard and Johannes Voss},
  doi          = {10.1002/jcc.26732},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {2004-2013},
  shortjournal = {J. Comput. Chem.},
  title        = {MCML: Combining physical constraints with experimental data for a multi-purpose meta-generalized gradient approximation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mean force based temperature accelerated sliced sampling:
Efficient reconstruction of high dimensional free energy landscapes.
<em>JCC</em>, <em>42</em>(28), 1996–2003. (<a
href="https://doi.org/10.1002/jcc.26727">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Temperature accelerated sliced sampling (TASS) is an efficient method to compute high dimensional free energy landscapes. The original TASS method employs the weighted histogram analysis method (WHAM) which is an iterative post-processing to reweight and stitch high dimensional probability distributions in sliced windows that are obtained in the presence of restraining biases. The WHAM necessitates that TASS windows lie close to each other for proper overlap of distributions and span the collective variable space of interest. On the other hand, increase in number of TASS windows implies more number of simulations, and thus it affects the efficiency of the method. To overcome this problem, we propose herein a new mean-force (MF) based reweighting scheme called TASS-MF, which enables accurate computation with a fewer number of windows devoid of the WHAM post-processing. Application of the technique is demonstrated for alanine di- and tripeptides in vacuo to compute their two- and four-dimensional free energy landscapes, the latter of which is formidable in conventional umbrella sampling and metadynamics. The landscapes are computed within a kcal mol−1accuracy, ensuring a safe usage for broad applications in computational chemistry.},
  archive      = {J_JCC},
  author       = {Asit Pal and Subhendu Pal and Shivani Verma and Motoyuki Shiga and Nisanth N. Nair},
  doi          = {10.1002/jcc.26727},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {1996-2003},
  shortjournal = {J. Comput. Chem.},
  title        = {Mean force based temperature accelerated sliced sampling: Efficient reconstruction of high dimensional free energy landscapes},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Cover image. <em>JCC</em>, <em>42</em>(27), i. (<a
href="https://doi.org/10.1002/jcc.26749">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Zhen Chao and Dexuan Xie},
  doi          = {10.1002/jcc.26749},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Arene and functionalized arene based two dimensional
organic–inorganic hybrid perovskites for photovoltaic applications.
<em>JCC</em>, <em>42</em>(27), 1982–1990. (<a
href="https://doi.org/10.1002/jcc.26731">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recently, two-dimensional organic–inorganic hybrid perovskites have attracted great attention for their outstanding performances in solar energy conversion devices. By using first principles calculations, we explored the structural, electronic and optical properties of recently synthesized (PEA)2PbI4and (PEA)2SnI4organic–inorganic hybrid perovskites to understand the photovoltaic performances of these systems. Our study reveals that both the perovskites are direct band gap semiconductors and possess desirable band gap for solar energy absorption. We have further extended our study to fluoro-, chloro-, and bromo-functionalized phenethylammonium (PEA) cations based [X(X = F, Cl, Br)PEA]2A(A = Pb, Sn)I4perovskite materials. The halogenated benzene moiety confers an ultrahydrophobic character and protects the perovskites from ambient moisture. The halogen functionalized perovskites remain direct band gap semiconductors and all the perovskites show very strong optical absorption (∼7 × 105cm−1) across UV–visible region. We have further calculated the photo-conversion efficiency (PCE) of both arene and functionalized arene based perovskites. The halogen-functionalized PEA-based perovskites also exhibit high PCE as like pristine ones and finally achieve high PCE of up to 24.30\%, making them competitive with other previously reported perovskite-based photovoltaic devices.},
  archive      = {J_JCC},
  author       = {Moumita Kar and Atish Ghosh and Ritabrata Sarkar and Sougata Pal and Pranab Sarkar},
  doi          = {10.1002/jcc.26731},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1982-1990},
  shortjournal = {J. Comput. Chem.},
  title        = {Arene and functionalized arene based two dimensional organic–inorganic hybrid perovskites for photovoltaic applications},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Density functionals with full nonlocal exchange, nonlocal
rung-3.5 correlation, and d3 dispersion: Combined accuracy for general
main-group thermochemistry, kinetics, and noncovalent interactions.
<em>JCC</em>, <em>42</em>(27), 1974–1981. (<a
href="https://doi.org/10.1002/jcc.26728">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce the HF-R35-D3(BJ) functional combining full nonlocal exact (Hartree-Fock-like, HF) exchange, inexpensive rung-3.5 correlation constructed from nonlocal one-electron operators, and nonlocal D3 dispersion corrections. HF-R35-D3(BJ) is among the first full-exact-exchange functionals offering competitive accuracy for general main-group thermochemistry, kinetics, and noncovalent interactions. HF-R35-D3(BJ) gives weighted mean absolute deviation WTMAD-2 8.5 kcal/mol across the entire GMTKN55 dataset, outperforming most dispersion-corrected semilocal functionals and approaching the accuracy of dispersion-corrected global hybrids. This requires six fitted parameters, three each in the nonlocal correlation and dispersion corrections. Full nonlocal exchange appears to help give accurate binding energies and reasonable energy orderings for water hexamers. These results motivate continued exploration of inexpensive nonlocal correlation corrections to nonlocal exchange.},
  archive      = {J_JCC},
  author       = {Chloe Ramos and Jeremiah Muehlbrad and Benjamin G. Janesko},
  doi          = {10.1002/jcc.26728},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1974-1981},
  shortjournal = {J. Comput. Chem.},
  title        = {Density functionals with full nonlocal exchange, nonlocal rung-3.5 correlation, and d3 dispersion: Combined accuracy for general main-group thermochemistry, kinetics, and noncovalent interactions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Implicit solvation in domain based pair natural orbital
coupled cluster (DLPNO-CCSD) theory. <em>JCC</em>, <em>42</em>(27),
1959–1973. (<a href="https://doi.org/10.1002/jcc.26726">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A nearly linear scaling implementation of coupled-cluster with singles and doubles excitations (CCSD) can be achieved by means of the domain-based local pair natural orbital (DLPNO) method. The combination of DLPNO-CCSD with implicit solvation methods allows the calculation of accurate energies and chemical properties of solvated systems at an affordable computational cost. We have efficiently implemented different schemes within the conductor-like polarizable continuum model (C-PCM) for DLPNO-CCSD in the ORCA quantum chemistry suite. In our implementation, the overhead due to the additional solvent terms amounts to less than 5\% of the time the equivalent gas phase job takes. Our results for organic neutrals and open-shell ions in water show that for most systems, adding solvation terms to the coupled-cluster amplitudes equations and to the energy leads to small changes in the total energy compared to only considering solvated orbitals and corrections to the reference energy. However, when the solute contains certain functional groups, such as carbonyl or nitrile groups, the changes in the energy are larger and estimated to be around 0.04 and 0.02 kcal/mol for each carbonyl and nitrile group in the solute, respectively. For solutes containing metals, the use of accurate CC/C-PCM schemes is crucial to account for correlation solvation effects. Simultaneously, we have calculated the electrostatic component of the solvation energy for neutrals and ions in water for the different DLPNO-CCSD/C-PCM schemes. We observe negligible changes in the deviation between DLPNO-CCSD and canonical-CCSD data. Here, DLPNO-CCSD results outperform those for Hartree-Fock and density functional theory calculations.},
  archive      = {J_JCC},
  author       = {Miquel Garcia-Ratés and Ute Becker and Frank Neese},
  doi          = {10.1002/jcc.26726},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1959-1973},
  shortjournal = {J. Comput. Chem.},
  title        = {Implicit solvation in domain based pair natural orbital coupled cluster (DLPNO-CCSD) theory},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Role of atomicity in the oxygen reduction reaction activity
of platinum sub nanometer clusters: A global optimization study.
<em>JCC</em>, <em>42</em>(27), 1944–1958. (<a
href="https://doi.org/10.1002/jcc.26725">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metal nanoclusters are an important class of materials for catalytic applications. Sub nanometer clusters are relatively less explored for their catalytic activity on account of undercoordinated surface structure. Taking this into account, we studied platinum-based sub nanometer clusters for their catalytic activity for oxygen reduction reaction (ORR). A comprehensive analysis with global optimization is carried out for structural prediction of the platinum clusters. The energetic and electronic properties of interactions of clusters with reaction intermediates are investigated. The role of structural sensitivity in the dynamics of clusters is unraveled, and unique intermediate specific interactions are identified. ORR energetics is examined, and exceptional activity for sub nanometer clusters are observed. An inverse size versus activity relationship is identified, challenging the conventional trends followed by larger nanoclusters. The principal role of atomicity in governing the catalytic activity of nanoclusters is illustrated. The structural norms governing the sub nanometer cluster activity are shown to be markedly different from larger nanoclusters.},
  archive      = {J_JCC},
  author       = {Akhil S. Nair and Anakuthil Anoop and Rajeev Ahuja and Biswarup Pathak},
  doi          = {10.1002/jcc.26725},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1944-1958},
  shortjournal = {J. Comput. Chem.},
  title        = {Role of atomicity in the oxygen reduction reaction activity of platinum sub nanometer clusters: A global optimization study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). An improved poisson-nernst-planck ion channel model and
numerical studies on effects of boundary conditions, membrane charges,
and bulk concentrations. <em>JCC</em>, <em>42</em>(27), 1929–1943. (<a
href="https://doi.org/10.1002/jcc.26723">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper, an improved Poisson-Nernst-Planck ion channel (PNPic) model is presented, along with its effective finite element solver and software package for an ion channel protein in a solution of multiple ionic species. Numerical studies are then done on the effects of boundary value conditions, membrane charges, and bulk concentrations on electrostatics and ionic concentrations for an ion channel protein, a gramicidin A (gA), and five different ionic solvents with up to four species. Numerical results indicate that the cation selectivity property of gA occurs within a central portion of ion channel pore, insensitively to any change of boundary value condition, membrane charge, or bulk concentration. Moreover, a numerical scheme for computing the electric currents induced by ion transports across membrane via an ion channel pore is presented and implemented as a part of the PNPic finite element package. It is then applied to the calculation of current–voltage curves, well validating the PNPic model and finite element package by electric current experimental data.},
  archive      = {J_JCC},
  author       = {Zhen Chao and Dexuan Xie},
  doi          = {10.1002/jcc.26723},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1929-1943},
  shortjournal = {J. Comput. Chem.},
  title        = {An improved poisson-nernst-planck ion channel model and numerical studies on effects of boundary conditions, membrane charges, and bulk concentrations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Insights into the electronic structure and mechanism of
norcarane hydroxylation by OxoMn(v) porphyrin complexes: A density
functional theory study. <em>JCC</em>, <em>42</em>(27), 1920–1928. (<a
href="https://doi.org/10.1002/jcc.26715">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Norcarane hydroxylation by neutral [PorMn(V)O-L] (L═OH−, F−) and cationic [PorMn(V)O-L]+(L═H2O, imidazole) oxoMn(V) porphyrin complex models has been investigated by density functional theory calculations to better understand the reaction mechanism and electronic structure. We found that the energy barriers of norcarane hydroxylation by cationic oxoMn(V) porphyrin complexes are lower than those by neutral oxoMn(V) porphyrin complexes. This indicates that cationic oxoMn(V) porphyrin complexes enhance norcarane hydroxylation compared with neutral oxoMn(V) porphyrin complexes. According to electronic structure analysis, in the C─H activation step, electron transfer occurs through initial interaction between the σCHand rich-oxygen π(Mn═O) orbitals to form real donor orbitals, followed by transfer to the acceptor π*(Mn═O) orbitals. Moreover, single electron shifts from norcarane (CH) to Mn atom during C─H activation. The positive charge of the cationic complex stabilizes the acceptor orbital more than the donor orbital, reducing the energy gap between these orbitals, thus lowering the reaction barrier.},
  archive      = {J_JCC},
  author       = {Zhifeng Ma and Naoki Nakatani and Masahiko Hada},
  doi          = {10.1002/jcc.26715},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1920-1928},
  shortjournal = {J. Comput. Chem.},
  title        = {Insights into the electronic structure and mechanism of norcarane hydroxylation by OxoMn(V) porphyrin complexes: A density functional theory study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Excited state distribution function for probing
herzberg–teller vibronic coupling using linear optical response theory:
Application to glassy pheophytin a. <em>JCC</em>, <em>42</em>(26),
1905–1913. (<a href="https://doi.org/10.1002/jcc.26724">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The goal of the present work is to develop an excited-state distribution function that can be used to calculate electronic transition dipole moment time correlation functions at a considerably low computational cost. An additional merit of the distribution function is its capability to probe the Hertzberg–Teller vibronic coupling effect in terms of the previously reported Condon correlation functions in the literature without having to start from the equilibrium density operator to probe spectral non-Condon effects, thereby exploring Hertzberg–Teller vibronic coupling by building on the Condon regime. It is easily extendable to anharmonic systems. Model calculations are reported to show the high degree of accuracy and computational efficiency of the presented approach. Application to a photosynthetic system such aspheophytin ain triethylamine is provided.},
  archive      = {J_JCC},
  author       = {Mohamad Toutounji},
  doi          = {10.1002/jcc.26724},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1905-1913},
  shortjournal = {J. Comput. Chem.},
  title        = {Excited state distribution function for probing Herzberg–Teller vibronic coupling using linear optical response theory: Application to glassy pheophytin a},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Small lithium-chloride clusters: Superalkalis,
superhalogens, supersalts and nanocrystals. <em>JCC</em>,
<em>42</em>(26), 1895–1904. (<a
href="https://doi.org/10.1002/jcc.26722">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the present paper, the results of the theoretical investigation of the small lithium-chloride clusters are reported. The geometrical structures, electronic and thermodynamic stability of superalkalis, superhalogens, and their single and double charged ions are obtained using efficient and accurate quantum chemistry methods. Further, low-lying isomers of the LinCln() clusters and their stability parameters are calculated. Two ways of formation of the LinClnclusters, polymerization of LiCl fragments and combination of superalkalis and superhalogen clusters, are compared. By examination the lattice energy and the average Li-Cl bond length in rectangular LinCln() clusters, it was concluded that already 50 LiCl are enough to mostly resembles the structure and stability of the bulk LiCl.},
  archive      = {J_JCC},
  author       = {Milan Milovanović},
  doi          = {10.1002/jcc.26722},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1895-1904},
  shortjournal = {J. Comput. Chem.},
  title        = {Small lithium-chloride clusters: Superalkalis, superhalogens, supersalts and nanocrystals},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Efficient workflow for the investigation of the catalytic
cycle of water oxidation catalysts: Combining GFN-xTB and density
functional theory. <em>JCC</em>, <em>42</em>(26), 1885–1894. (<a
href="https://doi.org/10.1002/jcc.26721">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photocatalytic water oxidation remains the bottleneck in many artificial photosynthesis devices. The efficiency of this challenging process is inherently linked to the thermodynamic and electronic properties of the chromophore and the water oxidation catalyst (WOC). Computational investigations can facilitate the search for favorable chromophore-catalyst combinations. However, this remains a demanding task due to the requirements on the computational method that should be able to correctly describe different spin and oxidation states of the transition metal, the influence of solvation and the different rates of the charge transfer and water oxidation processes. To determine a suitable method with favorable cost/accuracy ratios, the full catalytic cycle of a molecular ruthenium based WOC is investigated using different computational methods, including density functional theory (DFT) with different functionals (GGA, Hybrid, Double Hybrid) as well as the semi-empirical tight binding approach GFN-xTB. A workflow with low computational cost is proposed that combines GFN-xTB and DFT and provides reliable results. GFN-xTB geometries and frequencies combined with single-point DFT energies give free energy changes along the catalytic cycle that closely follow the full DFT results and show satisfactory agreement with experiment, while significantly decreasing the computational cost. This workflow allows for cost efficient determination of energetic, thermodynamic and dynamic properties of WOCs.},
  archive      = {J_JCC},
  author       = {Jan Paul Menzel and Martijn Kloppenburg and Jelena Belić and Huub J. M. de Groot and Lucas Visscher and Francesco Buda},
  doi          = {10.1002/jcc.26721},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1885-1894},
  shortjournal = {J. Comput. Chem.},
  title        = {Efficient workflow for the investigation of the catalytic cycle of water oxidation catalysts: Combining GFN-xTB and density functional theory},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Synthesis, identification, density functional and hirshfeld
surface studies of
2,2′-disulfanediylbis(tetrahydro-4H-cyclopenta[d][1,3,2]dioxaphosphole-2-sulfide).
<em>JCC</em>, <em>42</em>(26), 1873–1884. (<a
href="https://doi.org/10.1002/jcc.26720">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The new compound 2,2′-disulfanediylbis (tetrahydro-4H-cyclo penta[d][1,3,2]dioxaphosphole 2-sulfide), the dimeric form of 2-mercaptotetrahydro-4H-cyclopenta[d] [1,3,2] dioxaphosphole 2-sulfide, has been synthesized and characterized by elemental analysis, molecular weight determination and spectral data (1H-NMR,13C-NMR,31P-NMR, FTIR). The molecular geometry was confirmed by single X-Ray crystallography. The ground state property was examined by PBE0 and B3LYP density functionals using aug-cc-pV(Q+d)Z basis set in the gas phase and in DMSO solution. The preference of PBE0 functional was statistically established. Thermodynamic parameters and standard heat of dissociation reaction () have been established. The calculated equilibrium constants at different temperatures reflect the stability of the dimer over the monomers at low temperatures and vice versa. Valency and Fukui indices calculations showed that the monomer is more reactive than the dimer. 2D-fingerprint revealed that, while the H…X; [X = H, O and S] nonbonding intermolecular interactions and reciprocals play a crucial role in strengthening of molecules packing in the crystal unit cell while the S…S ones contribute negatively on it.},
  archive      = {J_JCC},
  author       = {Ahmed M. Mkadmh and Zaki S. Safi and Adnan A. Elkhaldy and Richard J. Staples and Savaş Kaya and Goncagül Serdaroğlu},
  doi          = {10.1002/jcc.26720},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1873-1884},
  shortjournal = {J. Comput. Chem.},
  title        = {Synthesis, identification, density functional and hirshfeld surface studies of 2,2′-disulfanediylbis(tetrahydro-4H-cyclopenta[d][1,3,2]dioxaphosphole-2-sulfide)},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identification of novel inhibitors of SARS-CoV-2 main
protease (mpro) from withania sp. By molecular docking and molecular
dynamics simulation. <em>JCC</em>, <em>42</em>(26), 1861–1872. (<a
href="https://doi.org/10.1002/jcc.26717">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since December 2019, coronavirus disease (COVID-19) has claimed the lives of millions of people across the globe. To date, no medicine is available for the responsible virus SARS-CoV-2. 3CLpro, that is, 3-chymotrypsin-like protease, the main protease (Mpro), has an important role in cleaving pp1a and pp1ab polyproteins. This Mproserves as an important target in drug designing against COVID-19. Herein, the study includes the investigation, screening, and identification of potent leads from (Withaniasps.), against SARS-CoV-2, using virtual screening, molecular docking, and molecular dynamics (MD) simulations. Seventy-three natural compounds from this important medicinal plant were screened. The Binding affinity was used to identify the most probable target to inhibit the Mpro, compounds 27-hydroxywithanolide F (W32, −11.5 kcal/mol), withanolide A (W56, −11.4 kcal/mol), and withacoagulin H (W30, −11.1 kcal/mol) showed highest binding energy. Lipinski&#39;s rule, followed by drug-likability and likeness screening, resulted in 36 molecules. Further, MD simulation of 50 ns predicted withacoagulin H possessing strong binding affinity and hydrogen-bonding interactions with the active site. The binding free energy calculation showed the most negative energy of withacoagulin H (−63.463 KJ/mol) compared to other selected compounds. The study also compared the bonding energy of already reported repurposed and newly synthesized drugs. Further, absorption, distribution, metabolism, and excretion predictions were made to found a good balance of potency. Hence the following screened compounds fromWithaniasps. could serve as the potential leads for drug development against COVID-19.},
  archive      = {J_JCC},
  author       = {Surjeet Verma and Chirag N. Patel and Muktesh Chandra},
  doi          = {10.1002/jcc.26717},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1861-1872},
  shortjournal = {J. Comput. Chem.},
  title        = {Identification of novel inhibitors of SARS-CoV-2 main protease (Mpro) from withania sp. by molecular docking and molecular dynamics simulation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ARGOS: An adaptive refinement goal-oriented solver for the
linearized poisson–boltzmann equation. <em>JCC</em>, <em>42</em>(26),
1832–1860. (<a href="https://doi.org/10.1002/jcc.26716">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An adaptive finite element solver for the numerical calculation of the electrostatic coupling between molecules in a solvent environment is developed and tested. At the heart of the solver is a goal-oriented a posteriori error estimate for the electrostatic coupling, derived and implemented in the present work, that gives rise to an orders of magnitude improved precision and a shorter computational time as compared to standard finite difference solvers. The accuracy of the new solver ARGOS is evaluated by numerical experiments on a series of problems with analytically known solutions. In addition, the solver is used to calculate electrostatic couplings between two chromophores, linked to polyproline helices of different lengths and between the spike protein of SARS-CoV-2 and the ACE2 receptor. All the calculations are repeated by using the well-known finite difference solvers MEAD and APBS, revealing the advantages of the present finite element solver.},
  archive      = {J_JCC},
  author       = {Svetoslav Nakov and Ekaterina Sobakinskaya and Thomas Renger and Johannes Kraus},
  doi          = {10.1002/jcc.26716},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1832-1860},
  shortjournal = {J. Comput. Chem.},
  title        = {ARGOS: An adaptive refinement goal-oriented solver for the linearized Poisson–Boltzmann equation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Cover image. <em>JCC</em>, <em>42</em>(25), i. (<a
href="https://doi.org/10.1002/jcc.26730">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Tomomi Kondo and Takehiko Sasaki and Motoyuki Shiga},
  doi          = {10.1002/jcc.26730},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comparison of group 14 elements in sp3 and sp2 environment
by fragment structure energy analysis. <em>JCC</em>, <em>42</em>(25),
1817–1825. (<a href="https://doi.org/10.1002/jcc.26718">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To explore the characteristics of group 14 elements in sp3and sp2hybrid environments systematically, the concept of fragment structure energy (FSE) was proposed through ab initio molecular orbital method. A fragment in this study means a building block of molecular structure and the energy of each kind of fragments (FSE) is obtained from the total electronic energy of different size of model compounds by just a simple algebraic method. The model compounds are the hydrogen-terminated diamond and graphene analogs as the typical compounds in theidealsp3- and sp2-environment, respectively. As a result, the sp3environment was found to be more stable than the sp2for all the group 14 elements (C–Pb) in the ideal hybrid environment. Furthermore, a mixing effect of carbon and heavier group 14 elements (E = Si, Ge, Sn, and Pb) was investigated by comparing the E/C-alternately mixed- and the pure FSEs.},
  archive      = {J_JCC},
  author       = {Taiji Nakamura and Tamotsu Uchiyama and Takako Kudo},
  doi          = {10.1002/jcc.26718},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1817-1825},
  shortjournal = {J. Comput. Chem.},
  title        = {Comparison of group 14 elements in sp3 and sp2 environment by fragment structure energy analysis},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Intramolecular interactions play key role in stabilization
of pHLIP at acidic conditions. <em>JCC</em>, <em>42</em>(25), 1809–1816.
(<a href="https://doi.org/10.1002/jcc.26719">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The pH-Low Insertion Peptide (pHLIP) is a membrane-active peptide that spontaneously folds into a transmembraneα-helix upon acidification. This activity enables pHLIP to potentially act as a vector for drugs related to diseases characterized by acidosis such as cancer or heart ischemia. Presently, due to aggregation-based effects, formulations of pHLIP are only viable at near-μM concentrations. In addition, since most of pHLIP&#39;s measurable qualities involve a membrane, probing the details of pHLIP in the interstitial region is difficult. In attempts to shed light on these issues, we performed constant pH molecular dynamics simulations on pHLIP as well as P20G, a variant with increased helicity, in solution at 0 and 150 mM NaCl over a broad range of pHs. In general, the addition of ions reduced the effective pKaof the acidic residues in pHLIP. P20G exhibits a higher helicity than pHLIP in general and is more compact than pHLIP at pH values under 4. In terms of charge effects, sodium cations localized predominantly to the C-terminus of the peptide with a high density of acidic residues. Additionally, the salt bridge between R11 and D14 is by far the most favored and particularly so with pHLIP at 150 mM NaCl. We expect that this approach will be a valuable tool to screen variants of pHLIP for favorable properties in solution, an aspect of pHLIP design that to this point has largely been neglected.},
  archive      = {J_JCC},
  author       = {Nicolas Frazee and Blake Mertz},
  doi          = {10.1002/jcc.26719},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1809-1816},
  shortjournal = {J. Comput. Chem.},
  title        = {Intramolecular interactions play key role in stabilization of pHLIP at acidic conditions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Photophysical properties of heavy atom containing
tetrasulfonyl phthalocyanines as possible photosensitizers in
photodynamic therapy. <em>JCC</em>, <em>42</em>(25), 1803–1808. (<a
href="https://doi.org/10.1002/jcc.26714">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The excitation energies, singlet-triplet energy gap and spin-orbit coupling constants for Zn-, GaCl-, Pd-, and Pt- tetrasulfonyl phthalocyanines complexes (ZnPc, GaClPc, PdPc, and PtPc) have been computed by using the density functional theory and employing the M06 exchange-correlation functional. Results show that these systems possess interesting photophysical properties, which make them possible photosensitizers to be proposed in photodynamic therapy (PDT). Absorption energies of all the complexes examined have been found falling inside the so-called therapeutic window (550–800 nm). Singlet-triplet energy gap values are higher than those required for the production of cytotoxic molecular oxygen and the spin-orbit coupling constants are such as to ensure an efficient spin orbit intersystem crossing. The obtained data are consistent with the experimental oxygen singlet quantum yields. The platinum complex appears to be the most effective candidate to propose for PDT.},
  archive      = {J_JCC},
  author       = {Bruna Clara De Simone and Marta Erminia Alberto and Nino Russo and Marirosa Toscano},
  doi          = {10.1002/jcc.26714},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1803-1808},
  shortjournal = {J. Comput. Chem.},
  title        = {Photophysical properties of heavy atom containing tetrasulfonyl phthalocyanines as possible photosensitizers in photodynamic therapy},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Influence of the solvent on the lewis acidity of antimony
pentahalides and group 13 lewis acids toward acetonitrile and pyridine.
<em>JCC</em>, <em>42</em>(25), 1792–1802. (<a
href="https://doi.org/10.1002/jcc.26713">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Energetic effects of solvation of SbF5, SbCl5, and 21 group 13 Lewis acids (LA) and their molecular complexes with acetonitrile and pyridine are evaluated using SMD approach. Compared to the gas phase, solvation increases the stability of boron- and aluminum-containing complexes but decreases the stability of gallium and indium-containing homologs due to larger solation energies of free LA. New Lewis acidity scales, based on the Gibbs energy of dissociation of the molecular complexes LA·pyridine and LA·acetonitrile in the gas phase, in benzene and acetonitrile solutions, are proposed.},
  archive      = {J_JCC},
  author       = {Anna V. Pomogaeva and Alexey Y. Timoshkin},
  doi          = {10.1002/jcc.26713},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1792-1802},
  shortjournal = {J. Comput. Chem.},
  title        = {Influence of the solvent on the lewis acidity of antimony pentahalides and group 13 lewis acids toward acetonitrile and pyridine},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). The mechanism of sorbitol dehydration in hot acidic
solutions. <em>JCC</em>, <em>42</em>(25), 1783–1791. (<a
href="https://doi.org/10.1002/jcc.26710">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sugar alcohol dehydration in hot water is an important reaction that allows for environmentally friendly biomass conversion without the use of organic solvents. Here, we report a free-energy analysis by metadynamics (MTD) simulations based on ab initio density functional theory and semiempirical density-functional tight-binding method to understand the mechanism of dehydration reactions ofd-sorbitol (SBT) in hot acidic water. Comparing the results of ab initio and semiempirical MTD, it was found that the latter gives a reliable free energy surface of SBT dehydration reaction, although the results vary upon the inclusion of dispersion correction. It was found that the reaction proceeds consistently via an SN2 mechanism, whereby the free energy of protonation of the hydroxyl group created as an intermediate is affected by the acidic species. This mechanism was further verified by real-time trajectories started from the transition state using ab initio molecular dynamics simulations. The free energy barriers of the reaction pathways leading to five-membered ether products are lower than those leading to six-membered ether products, in agreement with experiment. This outcome can be ascribed, in part, to our finding that the reaction barrier of the pathway is correlated to the stability of the SBT confined conformation at the initial stage of the reaction.},
  archive      = {J_JCC},
  author       = {Tomomi Kondo and Takehiko Sasaki and Motoyuki Shiga},
  doi          = {10.1002/jcc.26710},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1783-1791},
  shortjournal = {J. Comput. Chem.},
  title        = {The mechanism of sorbitol dehydration in hot acidic solutions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Physical achirality in geometrically chiral rotamers of
hydrazine and boranylborane molecules. <em>JCC</em>, <em>42</em>(25),
1772–1782. (<a href="https://doi.org/10.1002/jcc.26709">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The diagonal components and the trace of tensors which account for chiroptical response of the hydrazine molecule N2H4, that is, static anapole magnetizability and frequency-dependent electric dipole-magnetic dipole polarisability, are a function of theH─N─N─H dihedral angle. They vanish for symmetry reasons atϕ= 0° andϕ= 180°, corresponding respectively toC2vandC2hpoint group symmetries, that is,cisandtransconformers characterized by the presence of molecular symmetry planes. Nonetheless, vanishing diagonal components have been observed also in the proximity ofH─N─N─H = 90°, in which the point group symmetry isC2and hydrazine is unquestionably chiral. In the boranylborane molecule B2H4, assuming the B─B bond in theydirection, theayycomponent of the anapole magnetizability tensor approximately vanishes for dihedral anglesH─B─B─H corresponding to chiral rotamers which belong toD2symmetry. Such anomalous effects have been ascribed tophysical achiralityof these conformers, that is, to their inability to sustain electronic current densities inducing either anapole moments, or electric and magnetic dipole moments, about the chiral axis connecting heavier atoms, as well as perpendicular directions. In other terms, the structure of certaingeometrically chiralrotamers may be such that neither toroidal nor helical flow, which determine chiroptical phenomenology, can take place in the presence of perturbing fields parallel or orthogonal to the chiral axis.},
  archive      = {J_JCC},
  author       = {Gabriel I. Pagola and Marta B. Ferraro and Patricio F. Provasi and Stefano Pelloni and Paolo Lazzeretti},
  doi          = {10.1002/jcc.26709},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1772-1782},
  shortjournal = {J. Comput. Chem.},
  title        = {Physical achirality in geometrically chiral rotamers of hydrazine and boranylborane molecules},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PyUNIxMD: A python-based excited state molecular dynamics
package. <em>JCC</em>, <em>42</em>(24), 1755–1766. (<a
href="https://doi.org/10.1002/jcc.26711">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Theoretical/computational description of excited state molecular dynamics is nowadays a crucial tool for understanding light-matter interactions in many materials. Here we present an open-source Python-based nonadiabatic molecular dynamics program package, namely PyUNIxMD, to deal with mixed quantum-classical dynamics for correlated electron-nuclear propagation. The PyUNIxMD provides many interfaces for quantum chemical calculation methods with commercial and noncommercial ab initio and semiempirical quantum chemistry programs. In addition, the PyUNIxMD offers many nonadiabatic molecular dynamics algorithms such as fewest-switch surface hopping and its derivatives as well as decoherence-induced surface hopping based on the exact factorization (DISH-XF) and coupled-trajectory mixed quantum-classical dynamics (CTMQC) for general purposes. Detailed structures and flows of PyUNIxMD are explained for the further implementations by developers. We perform a nonadiabatic molecular dynamics simulation for a molecular motor system as a simple demonstration.},
  archive      = {J_JCC},
  author       = {In Seong Lee and Jong-Kwon Ha and Daeho Han and Tae In Kim and Sung Wook Moon and Seung Kyu Min},
  doi          = {10.1002/jcc.26711},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1755-1766},
  shortjournal = {J. Comput. Chem.},
  title        = {PyUNIxMD: A python-based excited state molecular dynamics package},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SEQCROW: A ChimeraX bundle to facilitate quantum chemical
applications to complex molecular systems. <em>JCC</em>,
<em>42</em>(24), 1750–1754. (<a
href="https://doi.org/10.1002/jcc.26700">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe a bundle for UCSF ChimeraX called SEQCROW that provides advanced structure editing capabilities and quantum chemistry utilities designed for complex organic and organometallic compounds. SEQCROW includes graphical presets and bond editing tools that facilitate the generation of publication-quality molecular structure figures while also allowing users to build molecular structures quickly and efficiently by mapping new ligands onto existing organometallic complexes as well as adding rings and substituents. Other capabilities include the ability to visualize vibrational modes and simulated IR spectra, to compute and visualize molecular descriptors including percent buried volume, ligand cone angles, and Sterimol parameters, to process thermochemical corrections from quantum mechanical computations, to generate input files for ORCA, Psi4, and Gaussian, and to run and manage computational jobs.},
  archive      = {J_JCC},
  author       = {Anthony J. Schaefer and Victoria M. Ingman and Steven E. Wheeler},
  doi          = {10.1002/jcc.26700},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1750-1754},
  shortjournal = {J. Comput. Chem.},
  title        = {SEQCROW: A ChimeraX bundle to facilitate quantum chemical applications to complex molecular systems},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Exploration of potential inhibitors for tuberculosis via
structure-based drug design, molecular docking, and molecular dynamics
simulation studies. <em>JCC</em>, <em>42</em>(24), 1736–1749. (<a
href="https://doi.org/10.1002/jcc.26712">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug resistance in tuberculosis is major threat to human population. In the present investigation, we aimed to identify novel and potent benzimidazole molecules to overcome the resistance management. A series of 20 benzimidazole derivatives were examined for its activity as selective antitubercular agents. Initially, AutodockVina algorithm was performed to assess the efficacy of the molecules. The results are further enriched by redocking by means of Glide algorithm. The binding free energies of the compounds were then calculated by MM-generalized-born surface area method. Molecular docking studies elucidated that benzimidazole derivatives has revealed formation of hydrogen bond and strong binding affinity in the active site ofMycobacterium tuberculosisprotein. Note that ARG308, GLY189, VAL312, LEU403, and LEU190 amino acid residues ofMycobacterium tuberculosisprotein PrpR are involved in binding with ligands of benzimidazoles. Interestingly, the ligands exhibited same binding potential to the active site of protein complex PrpR in both the docking programs. In essence, the result portrays that benzimidazole derivatives such as1p,1q, and1 tcould be potent and selective antitubercular agents than the standard drug isoniazid. These compounds were then subjected to molecular dynamics simulation to validate the dynamics activity of the compounds against PrpR. Finally, the inhibitory behavior of compounds was predicted using a machine learning algorithm trained on a data collection of 15,000 compounds utilizing graph-based signatures. Overall, the study concludes that designed benzimidazoles can be employed as antitubercular agents. Indeed, the results are helpful for the experimental biologists to develop safe and non-toxic drugs against tuberculosis.},
  archive      = {J_JCC},
  author       = {Sreerama Rajasekhar and Ramanathan Karuppasamy and Kaushik Chanda},
  doi          = {10.1002/jcc.26712},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1736-1749},
  shortjournal = {J. Comput. Chem.},
  title        = {Exploration of potential inhibitors for tuberculosis via structure-based drug design, molecular docking, and molecular dynamics simulation studies},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Rational design of pincer-nickel complexes for catalytic
cyanomethylation of benzaldehyde: A systematic DFT study. <em>JCC</em>,
<em>42</em>(24), 1728–1735. (<a
href="https://doi.org/10.1002/jcc.26708">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The current study dwells upon the efforts to computationally probe a phosphine-free pincer-nickel complex that would demonstrate an efficiency better than the reported phosphine-based pincer-nickel complex (iPr2POCNEt2)Ni(CH2CN) for cyanomethylation reaction. For this purpose, the mechanism of cyanomethylation of benzaldehyde was studied quantum mechanically for a series of 11 pincer-nickel complexes. The energetics of various intermediates and transition states involved in the catalytic cycle for each catalyst was compared with the corresponding energetics of the Miller&#39;s catalyst (iPr2POCNEt2)Ni(CH2CN) that is reported to accomplish the cyanomethylation at room temperature. While pincer complexes (iPr4NNN)Ni(CH2CN) and (iPr4NCN)Ni(CH2CN) containing strongσ-donating amines were found to fare poorly, pincer-nickel complexes (iPr2NCN)Ni(CH2CN) and (dmPheboxNCN)Ni(CH2CN) based on weakerσ-donating imines had energetics more favorable than the reported efficient catalyst (iPr2POCNEt2)Ni(CH2CN). While strongtrans-influencing C as the pincer central atom was found to be pivotal for lowering the cyanomethylation kinetics, presence of a poortrans-influencing N proved to be detrimental on the overall energetics.},
  archive      = {J_JCC},
  author       = {Moumita Dutta and Hemant Kumar Srivastava and Akshai Kumar},
  doi          = {10.1002/jcc.26708},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1728-1735},
  shortjournal = {J. Comput. Chem.},
  title        = {Rational design of pincer-nickel complexes for catalytic cyanomethylation of benzaldehyde: A systematic DFT study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Searching for local order parameters to classify water
structures at triple points. <em>JCC</em>, <em>42</em>(24), 1720–1727.
(<a href="https://doi.org/10.1002/jcc.26707">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The diversity of ice polymorphs is of interest in condensed-matter physics, engineering, astronomy, and biosphere and climate studies. In particular, their triple points are critical to elucidate the formation of each phase and transitions among phases. However, an approach to distinguish their molecular structures is lacking. When precise molecular geometries are given, order parameters are often computed to quantify the degree of structural ordering and to classify the structures. Many order parameters have been developed for specific or multiple purposes, but their capabilities have not been exhaustively investigated for distinguishing ice polymorphs. Here, 493 order parameters and their combinations are considered for two triple points involving the ice polymorphs ice III-V-liquid and ice V-VI-liquid. Supervised machine learning helps automatic and systematic searching of the parameters. For each triple point, the best set of two order parameters was found that distinguishes three structures with high accuracy. A set of three order parameters is also suggested for better accuracy.},
  archive      = {J_JCC},
  author       = {Hideo Doi and Kazuaki Z. Takahashi and Takeshi Aoyagi},
  doi          = {10.1002/jcc.26707},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1720-1727},
  shortjournal = {J. Comput. Chem.},
  title        = {Searching for local order parameters to classify water structures at triple points},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). First-principles based theoretical investigation of impact
of polyolefin structure on photooxidation behavior. <em>JCC</em>,
<em>42</em>(24), 1710–1719. (<a
href="https://doi.org/10.1002/jcc.26702">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite their mass production and large applications, polyolefins&#39; stability and durability toward the air, moisture, and weather resistance is a challenge for the ecosystem. After long-term exposure to ultraviolet (UV) radiation or high-temperature or erosion, polyolefins undergo degradation generating microplastics (MPs). The MPs generated after the degradation of these polyolefins are hazardous for the ecosystem. In the present work, we have carried out density functional theory (DFT) studies to investigate the photodegradation of six different polyolefins ranging from polyethylene to polydecene, differing in side-chain. Herein, we have investigated photooxidized derivatives of different polyolefins and analyzed their relative stability, conformations, UV–visible spectral behavior, and carbonyl index. The photooxidized derivatives of various polyolefins formed during degradation are examined. The time-dependent density functional theory analysis confirms that the carbonyl groups of photooxidized products show absorption peak in Infrared (IR) and visible region, acting as light-absorbing species. The relative stabilities of hydroperoxide formed during photo/thermal oxidation of different polyolefins have been evaluated to explain the degradation behavior. The oligomerization and stabilization energies of their corresponding hydroperoxide&#39;s were computed and analyzed to explain the degradation behavior of the polyolefins. The computed results suggest that polyolefins in their pristine state are stable toward photooxidation, but chemical impurities like carbonyl, unsaturated carbonyl, carboxylic acid, and hydroperoxide derivatives make them prone to undergo degradation, a fundamental process leading to generation of MPs. The comparative results confirmed that the side-chain length affects the stability and degradation of different polyolefins toward photooxidation.},
  archive      = {J_JCC},
  author       = {Vikas Khatri and Harender S. Dhattarwal and Hemant K. Kashyap and Gurmeet Singh},
  doi          = {10.1002/jcc.26702},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1710-1719},
  shortjournal = {J. Comput. Chem.},
  title        = {First-principles based theoretical investigation of impact of polyolefin structure on photooxidation behavior},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A molecular dynamics study of a fully zwitterionic
copolymer/ionic liquid-based electrolyte: Li+ transport mechanisms and
ionic interactions. <em>JCC</em>, <em>42</em>(23), 1689–1703. (<a
href="https://doi.org/10.1002/jcc.26706">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of polymer electrolytes (PEs) is crucial for advancing safe, high-energy density batteries, such as lithium-metal and other beyond lithium-ion chemistries. However, reaching the optimum balance between mechanical stiffness and ionic conductivity is not a straightforward task. Zwitterionic (ZI) gel electrolytes comprising lithium salt and ionic liquid (IL) solutions within a fully ZI polymer network can, in this context, provide useful properties. Although such materials have shown compatibility with lithium metal in batteries, several fundamental structure-dynamic relationships regarding ionic transport and the Li+coordination environment remain unclear. To better resolve such issues, molecular dynamics simulations were carried out for two IL-based electrolyte systems, N-butyl-N-methylpyrrolidinium bis(trifluoromethylsulfonyl)imide ([BMP][TFSI]) with 1 M LiTFSI salt and a ZI gel electrolyte containing the IL and a ZI copolymer: poly(2-methacryloyloxyethyl phosphorylcholine-co-sulfobetaine vinylimidazole), poly(MPC-co-SBVI). The addition of ZI polymer decreases the [TFSI]−–[Li]+interactions and increases the IL ion diffusivities, and consequently, the overall ZI gel ionic conductivity. The structural analyses showed a large preference for lithium-ion interactions with the polymer phosphonate groups, while the [TFSI]−anions interact directly with the sulfonate group and the [BMP]+cations only display secondary interactions with the polymer. In contrast to previous experimental data on the same system, the simulated transference numbers showed smaller [Li]+contributions to the overall ionic conductivities, mainly due to negatively charged lithium aggregates and the strong lithium-ion interactions in the systems.},
  archive      = {J_JCC},
  author       = {Tuanan C. Lourenço and Mahsa Ebadi and Matthew J. Panzer and Daniel Brandell and Luciano T. Costa},
  doi          = {10.1002/jcc.26706},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1689-1703},
  shortjournal = {J. Comput. Chem.},
  title        = {A molecular dynamics study of a fully zwitterionic copolymer/ionic liquid-based electrolyte: Li+ transport mechanisms and ionic interactions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Links among the fukui potential, the alchemical hardness and
the local hardness of an atom in a molecule. <em>JCC</em>,
<em>42</em>(23), 1681–1688. (<a
href="https://doi.org/10.1002/jcc.26705">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This paper presents a brief summary of the difficulty that resides in the definition of the elusive concept of local chemical hardness. We argue that a definition of local hardness should be useful to a reactivity principle and not just as a mere definition. We then continue with a formal discussion about the benefits and difficulties of using the Fukui potential, which is interpreted as an alchemical derivative (alchemical hardness), as descriptor of local hardness of molecules. Computational evidence shows that the alchemical hardness is at least as good a descriptor as the combination of other two well-stabilized descriptors of local hardness, such as the Fukui function and grand canonical local hardness. Although our results are auspicious for the alchemical hardness as descriptor of local hardness, we finish by calling the attention of the community on the importance of discussing theraison d&#39;êtreof a local hardness function and its main characteristics. We suggest that an axiomatic construction of local hardness could be they way of constructing a local hardness which is both useful and free of arbitrariness.},
  archive      = {J_JCC},
  author       = {Tatiana Gómez and Patricio Fuentealba and Andrés Robles-Navarro and Carlos Cárdenas},
  doi          = {10.1002/jcc.26705},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1681-1688},
  shortjournal = {J. Comput. Chem.},
  title        = {Links among the fukui potential, the alchemical hardness and the local hardness of an atom in a molecule},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Exploring the molecular basis of UG-rich RNA recognition by
the human splicing factor TDP-43 using molecular dynamics simulation and
free energy calculation. <em>JCC</em>, <em>42</em>(23), 1670–1680. (<a
href="https://doi.org/10.1002/jcc.26704">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transactivation response element RNA/DNA-binding protein 43 (TDP-43) is involved in the regulation of alternative splicing of human neurodegenerative disease-related genes through binding to long UG-rich RNA sequences. Mutations in TDP-43, most in the homeodomain, cause neurological disorders such as amyotrophic lateral sclerosis and fronto temporal lobar degeneration. Several mutants destabilize the structure and disrupt RNA-binding activity. The biological functions of these mutants have been characterized, but the structural basis behind the loss of RNA-binding activity is unclear. Focused on the specific TDP-43-ssRNA complex (PDB code 4BS2), we applied molecular dynamics simulations and the molecular mechanics Poisson-Boltzmann surface area free energy calculation to characterize and explore the structural and dynamic effects between ssRNA and TDP-43. The energetic analysis indicated that the intermolecular van der Waals interaction and nonpolar solvation energy play an important role in the binding process of TDP-43 and ssRNA. Compared with the wild-type TDP-43, the reduction of the polar or non-polar interaction between all the mutants F149A, D105A/S254A, R171A/D174A, F147L/F149L/F229L/F231L and ssRNA is the main reason for the reduction of its binding free energy. Decomposing energies suggested that the extensive interactions between TDP-43 and the nitrogenous bases of ssRNA are responsible for the specific ssRNA recognition by TDP-43. These results elucidated the TDP-43-ssRNA interaction comprehensively and further extended our understanding of the previous experimental data. The uncovering of TDP-43-ssRNA recognition mechanism will provide us useful insights and new chances for the development of anti-neurodegenerative drugs.},
  archive      = {J_JCC},
  author       = {Han Sun and Wei Chen and Lin Chen and Wenqing Zheng},
  doi          = {10.1002/jcc.26704},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1670-1680},
  shortjournal = {J. Comput. Chem.},
  title        = {Exploring the molecular basis of UG-rich RNA recognition by the human splicing factor TDP-43 using molecular dynamics simulation and free energy calculation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An analysis of valence electronic structure from a viewpoint
of resonance theory: Tautomerization of formamide and diazadiboretidine.
<em>JCC</em>, <em>42</em>(23), 1662–1669. (<a
href="https://doi.org/10.1002/jcc.26703">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The resonance theory is still very useful in understanding the valence electron structure. However, such a viewpoint is not usually obtained by general-purpose quantum chemical calculations, instead requires rather special treatment such as valence bond methods. In this study, we propose a method based on second quantization to analyze the results obtained by general-purpose quantum chemical calculations from the local point of view of electronic structure and analyze diazadiboretidine and the tautomerization of formamide. This method requires only the “PS”-matrix, consisting of the density matrix (P-matrix) and overlap matrix, and can be computed with a comparable load to that of Mulliken population analysis. A key feature of the method is that, unlike other methods proposed so far, it makes direct use of the results of general-purpose quantum chemical calculations.},
  archive      = {J_JCC},
  author       = {Kaho Nakatani and Masahiro Higashi and Ryoichi Fukuda and Hirofumi Sato},
  doi          = {10.1002/jcc.26703},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1662-1669},
  shortjournal = {J. Comput. Chem.},
  title        = {An analysis of valence electronic structure from a viewpoint of resonance theory: Tautomerization of formamide and diazadiboretidine},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Parameterizing elastic network models to capture the
dynamics of proteins. <em>JCC</em>, <em>42</em>(23), 1643–1661. (<a
href="https://doi.org/10.1002/jcc.26701">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coarse-grained normal mode analyses of protein dynamics rely on the idea that the geometry of a protein structure contains enough information for computing its fluctuations around its equilibrium conformation. This geometry is captured in the form of an elastic network (EN), namely a network of edges between its residues. The normal modes of a protein are then identified with the normal modes of its EN. Different approaches have been proposed to construct ENs, focusing on the choice of the edges that they are comprised of, and on their parameterizations by the force constants associated with those edges. Here we propose new tools to guide choices on these two facets of EN. We study first different geometric models for ENs. We compare cutoff-based ENs, whose edges have lengths that are smaller than a cutoff distance, with Delaunay-based ENs and find that the latter provide better representations of the geometry of protein structures. We then derive an analytical method for the parameterization of the EN such that its dynamics leads to atomic fluctuations that agree with experimental B-factors. To limit overfitting, we attach a parameter referred to as flexibility constant to each atom instead of to each edge in the EN. The parameterization is expressed as a non-linear optimization problem whose parameters describe both rigid-body and internal motions. We show that this parameterization leads to improved ENs, whose dynamics mimic MD simulations better than ENs with uniform force constants, and reduces the number of normal modes needed to reproduce functional conformational changes.},
  archive      = {J_JCC},
  author       = {Patrice Koehl and Henri Orland and Marc Delarue},
  doi          = {10.1002/jcc.26701},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1643-1661},
  shortjournal = {J. Comput. Chem.},
  title        = {Parameterizing elastic network models to capture the dynamics of proteins},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CHAMPION: Chalmers hierarchical atomic, molecular, polymeric
and ionic analysis toolkit. <em>JCC</em>, <em>42</em>(23), 1632–1642.
(<a href="https://doi.org/10.1002/jcc.26699">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present CHAMPION (Chalmers hierarchical atomic, molecular, polymeric, and ionic analysis toolkit): a software developed to automatically detect time-dependent bonds between atoms based on their dynamics, classify the local graph topology around them, and analyze the physicochemical properties of these topologies by statistical physics. In stark contrast to methodologies where bonds are detected based on static conditions such as cut-off distances, CHAMPION considers pairs of atoms to be bound only if they move together and act as a bound pair over time. Furthermore, the time-dependent global bond graph is possible to split into dynamically shifting connected components or subgraphs around a certain chemical motif and thereby allow the physicochemical properties of each such topology to be analyzed by statistical physics. Applicable to condensed matter and liquids in general, and electrolytes in particular, this allows both quantitative and qualitative descriptions of local structure, as well as dynamical processes such as speciation and diffusion. We present here a detailed overview of CHAMPION, including its underlying methodology, implementation, and capabilities.},
  archive      = {J_JCC},
  author       = {Rasmus Andersson and Fabian Årén and Alejandro A. Franco and Patrik Johansson},
  doi          = {10.1002/jcc.26699},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1632-1642},
  shortjournal = {J. Comput. Chem.},
  title        = {CHAMPION: Chalmers hierarchical atomic, molecular, polymeric and ionic analysis toolkit},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). New developments in the GDIS simulation package: Integration
of VASP and USPEX. <em>JCC</em>, <em>42</em>(22), 1602–1626. (<a
href="https://doi.org/10.1002/jcc.26697">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A popular first principles simulation code, the Vienna Ab initio Simulation Package (VASP), and a crystal structure prediction (CSP) package, the Universal Structure Predictor: Evolutionary Xtallography (USPEX) have been integrated into the GDIS visualization software. The aim of this integration is to provide users with a unique and simple interface through which most of the steps of a typical crystal optimization or prediction work. This involved, for the latter, not only setting up a CSP calculation with complete support for the latest version of USPEX, but also displaying the many structure results by linking each structure geometry and its energy via interactive graphics. For the optimization part, any structure displayed by GDIS can now be the starting point for VASP calculations, with support for its most commonly used parameters. Atomic and electronic structures can be displayed as well as dynamic properties such as total energy, force, volume, and pressure for each ionic step. It is not only possible to start calculations from the GDIS visualization software, using an in-place task manager, but a running calculation can also be followed, allowing a greater control of the simulation process. The GDIS software is available under the GNU public license in its second version.},
  archive      = {J_JCC},
  author       = {Hubert Okadome Valencia and Busheng Wang and Gilles Frapper and Andrew Lloyd Rohl},
  doi          = {10.1002/jcc.26697},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1602-1626},
  shortjournal = {J. Comput. Chem.},
  title        = {New developments in the GDIS simulation package: Integration of VASP and USPEX},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Evaluation of density functional theory for a large and
diverse set of organic and inorganic equilibrium structures.
<em>JCC</em>, <em>42</em>(22), 1590–1601. (<a
href="https://doi.org/10.1002/jcc.26698">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Density functional theory (DFT) has been extensively benchmarked for energetic properties; however, less attention has been given to equilibrium structures and the effect of using a certain DFT geometry on subsequent energetic properties. We evaluate the performance of 52 contemporary DFT methods for obtaining the structures of 122 species in the W4-11-GEOM database. This dataset includes a total of 246 unique bonds: 117 H─X, 65 X─Y, 49 X═Y, and 15 XY bonds (where X and Y are first- and second-row atoms) and 133 key bond angles: 96 X-Y-H, 22 X-Y-Z, and 15 H-X-H angles. The reference geometries are optimized at the CCSD(T)/jul-cc-pV(n+d)Z level of theory (n= 5, 6). The performance of DFT is evaluated in conjunction with the Def2-nZVPP (n= T, Q), cc-pV(T+d)Z, and jul-cc-pV(T+d)Z basis sets. The root-mean-square deviations (RMSDs) over the bond distances of the best performing functionals from each rung of Jacob&#39;s Ladder are 0.0086 (SOGGA11), 0.0088 (τ-HCTH), 0.0059 (B3LYP), 0.0054 (TPSSh), and 0.0032 (DSD-PBEP86) Å. We evaluate the effect of the choice of the DFT geometry on subsequent molecular energies calculated with W1-F12 theory. Geometries obtained with GGA and MGGA methods result in large RMSDs in the subsequent W1-F12 energies; however, six hybrid GGA functionals (B3LYP, B3P86, mPW3PBE, B3PW91, mPW1LYP, and X3LYP) result in an excellent performance with RMSDs between 0.25 and 0.30 kJ mol−1relative to the CCSD(T)/CBS reference geometries. The B2GP-PLYP and mPW2-PLYP DHDFT methods result in near-CCSD(T) accuracy with RMSDs of 0.11 and 0.10 kJ mol−1, respectively.},
  archive      = {J_JCC},
  author       = {Amir Karton and Peter R. Spackman},
  doi          = {10.1002/jcc.26698},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1590-1601},
  shortjournal = {J. Comput. Chem.},
  title        = {Evaluation of density functional theory for a large and diverse set of organic and inorganic equilibrium structures},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ESCASA: Analytical estimation of atomic coordinates from
coarse-grained geometry for nuclear-magnetic-resonance-assisted protein
structure modeling. I. Backbone and hβ protons. <em>JCC</em>,
<em>42</em>(22), 1579–1589. (<a
href="https://doi.org/10.1002/jcc.26695">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method for the estimation of coordinates of atoms in proteins from coarse-grained geometry by simple analytical formulas (ESCASA), for use in nuclear-magnetic-resonance (NMR) data-assisted coarse-grained simulations of proteins is proposed. In this paper, the formulas for the backbone Hαand amide (HN) protons, and the side-chain Hβprotons, given the Cα-trace, have been derived and parameterized, by using the interproton distances calculated from a set of 140 high-resolution non-homologous protein structures. The mean standard deviation over all types of proton pairs in the set was 0.44 Å after fitting. Validation against a set of 41 proteins with NMR-determined structures, which were not considered in parameterization, resulted in average standard deviation from average proton–proton distances of the NMR-determined structures of 0.25 Å, compared to 0.21 Å obtained with the PULCHRA all-atom-chain reconstruction algorithm and to the 0.12 Å standard deviation of the average-structure proton–proton distance of NMR-determined ensembles. The formulas provide analytical forces and can, therefore, be used in coarse-grained molecular dynamics.},
  archive      = {J_JCC},
  author       = {Emilia A. Lubecka and Adam Liwo},
  doi          = {10.1002/jcc.26695},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1579-1589},
  shortjournal = {J. Comput. Chem.},
  title        = {ESCASA: Analytical estimation of atomic coordinates from coarse-grained geometry for nuclear-magnetic-resonance-assisted protein structure modeling. i. backbone and hβ protons},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Novel 6-hydroxyquinolinone derivatives: Design, synthesis,
antimicrobial evaluation, in silico study and toxicity profiling.
<em>JCC</em>, <em>42</em>(22), 1561–1578. (<a
href="https://doi.org/10.1002/jcc.26693">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Infectious diseases of bacteria and fungi have become a major risk to public health because of antibiotic and antifungal resistance. However, the availability of effective antibacterial and antifungal agents is becoming increasingly limited with growing resistance to existing drugs. In response to that, novel agents are critically needed to overcome such resistance. A new series of 6-hydroxyquinolinone3, 4, 5a, 5b, 6a and 6bbearing different side chains were synthesized and evaluated as antimicrobials against numbers of bacteria and fungi, using inhibition zone technique. As one of these derivatives, compound3was identified as a potent antibacterial and antifungal agent against all tested microorganisms with good minimum inhibitory concentration values comparable to reference drugs. Molecular docking studies were performed on antibacterial and antifungal targets; microbial DNA gyrase B ofStaphylococcus aureus(PDB ID: 4URO);N-myristoyltransferase ofCandida albicans(PDB ID: 1IYK), respectively, to predict the most probable type of interaction at the active site of the target protein in addition to binding affinities and orientations of docked ligands. Additionally, in silico prediction in terms of detailed physicochemical ADME and toxicity profile relating drug-likeness as well as medicinal chemistry friendliness was performed to all synthesized compounds. The results indicated that a novel 4,6-dihydroxyquinolin-2(1H)-one(3)is likely to be a newly synthesized drug candidate, indicating low toxicity in addition to good in silico absorption. In order to pave the way for more logical production of such compounds, structure–activity and toxicity relationships are also discussed.},
  archive      = {J_JCC},
  author       = {Marwa A. Saleh and Mohamed A. El-Badry and Rogy R. Ezz Eldin},
  doi          = {10.1002/jcc.26693},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1561-1578},
  shortjournal = {J. Comput. Chem.},
  title        = {Novel 6-hydroxyquinolinone derivatives: Design, synthesis, antimicrobial evaluation, in silico study and toxicity profiling},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comparison of the MSMS and NanoShaper molecular surface
triangulation codes in the TABI poisson–boltzmann solver. <em>JCC</em>,
<em>42</em>(22), 1552–1560. (<a
href="https://doi.org/10.1002/jcc.26692">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Poisson–Boltzmann (PB) implicit solvent model is a popular framework for studying the electrostatics of solvated biomolecules. In this model the dielectric interface between the biomolecule and solvent is often taken to be the molecular surface or solvent-excluded surface (SES), and the quality of the SES triangulation is critical in boundary element simulations of the model. This work compares the performance of the MSMS and NanoShaper surface triangulation codes for a set of 38 biomolecules. While MSMS produces triangles of exceedingly small area and large aspect ratio, the two codes yield comparable values for the SES surface area and electrostatic solvation energy, where the latter calculations were performed using the treecode-accelerated boundary integral (TABI) PB solver. However we found that NanoShaper is computationally more efficient and reliable than MSMS, especially when parameters are set to produce highly resolved triangulations.},
  archive      = {J_JCC},
  author       = {Leighton Wilson and Robert Krasny},
  doi          = {10.1002/jcc.26692},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1552-1560},
  shortjournal = {J. Comput. Chem.},
  title        = {Comparison of the MSMS and NanoShaper molecular surface triangulation codes in the TABI Poisson–Boltzmann solver},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modifying the catalytic preference of alpha-amylase toward
n-alkanes for bioremediation purposes using in silico strategies.
<em>JCC</em>, <em>42</em>(22), 1540–1551. (<a
href="https://doi.org/10.1002/jcc.26562">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since the beginning of oil exploration, whole ecosystems have been affected by accidents and bad practices involving petroleum compounds. In this sense, bioremediation stands out as the cheapest and most eco-friendly alternatives to reverse the damage done in oil-impacted areas. However, more efforts must be made to engineer enzymes that could be used in the bioremediation process. Interestingly, a recent work described thatα-amylase, one of the most evolutionary conserved enzymes, was able to promiscuously degraden-alkanes, a class of molecules abundant in the petroleum admixture. Considering thatα-amylase is expressed in almost all known organisms, and employed in numerous biotechnological processes, using it can be a great leap toward more efficient applications of enzyme or microorganism-consortia bioremediation approaches. In this work, we employed a strict computational approach to design newα-amylase mutants with potentially enhanced catalytic efficiency towardn-alkanes. Using in silico techniques, such as molecular docking, molecular dynamics, metadynamics, and residue–residue interaction networks, we generated mutants potentially more efficient for degradingn-alkanes, L183Y, and N314A. Our results indicate that the new mutants have an increased binding rate for tetradecane, the longestn-alkane previously tested, which can reside in the catalytic center for more extended periods. Additionally, molecular dynamics and network analysis showed that the new mutations have no negative impact on protein structure than the WT. Our results aid in solidifying this enzyme as one more tool in the petroleum bioremediation toolbox.},
  archive      = {J_JCC},
  author       = {Éderson Sales Moreira Pinto and Bruno César Feltes and Conrado Pedebos and Márcio Dorn},
  doi          = {10.1002/jcc.26562},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1540-1551},
  shortjournal = {J. Comput. Chem.},
  title        = {Modifying the catalytic preference of alpha-amylase toward n-alkanes for bioremediation purposes using in silico strategies},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MonteCarbo: A software to generate and dock
multifunctionalized ring molecules. <em>JCC</em>, <em>42</em>(21),
1526–1534. (<a href="https://doi.org/10.1002/jcc.26559">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MonteCarbo is an open-source software to construct simple 5-, 6-, and 7-membered ring multifunctionalized monosaccharides and nucleobases and dock them into the active site of carbohydrate-active enzymes. The core bash script executes simple orders to generate theZ-matrix of the neutral molecule of interest. After that, a Fortran90 code based on a pseudo-random number generator (Monte Carlo method) is executed to assign dihedral angles to the different rotamers present in the structure (ring and rotating functional groups). The program also has a generalized internal coordinates (GIC) implementation of the Cremer and Pople puckering coordinates ring. Once the structures are generated and optimized, a second code is ready to execute in serial the docking of multiple conformers in the active site of a wide family of enzymes.},
  archive      = {J_JCC},
  author       = {Santiago Alonso-Gil},
  doi          = {10.1002/jcc.26559},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1526-1534},
  shortjournal = {J. Comput. Chem.},
  title        = {MonteCarbo: A software to generate and dock multifunctionalized ring molecules},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Evolution of the linker in microhydrated hydrogen dinitrate
anions: From h+ to H5O2+. <em>JCC</em>, <em>42</em>(21), 1514–1525. (<a
href="https://doi.org/10.1002/jcc.26560">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hydrogen dinitrate anion, HNO3(NO3−), is a proton-bound dimer with a very strong hydrogen bond. By employing ab initio molecular dynamics (AIMD) method, we studied the effects of the proton transfer and the rotation of the nitrates on the vibrational profiles of HNO3(NO3−)(H2O)n(n =0–2). The AIMD results indicate that the structure of then =0 cluster is very flexible, even though its hydrogen bond is quite strong. Significant rotations around the hydrogen bond and frequent transfers of proton from HNO3to NO3−are observed in AIMD simulations. Dynamic changes are therefore an important factor in understanding the broadening of vibrational features. Forn =1, the extent of structural fluctuation increases further, as H2O could move around the anion while the HNO3(NO3−) core also goes through structural changes. Its vibrational spectrum can be understood as a mixture of many isomers visited during AIMD simulations. Byn =2, the structure is stabilized around one isomer, with the linker between the two nitrates being H5O2+, rather than H+. Due to strong hydrogen bonds between nitrates and water molecules, this H5O2+takes the extraordinary structure with the H+localized on one H2O, rather than being shared. While this novel structure is stable during AIMD simulations, the dynamic fluctuations in hydrogen bond distances still produce significant broadening in its vibrational profile.},
  archive      = {J_JCC},
  author       = {Shaojun Gui and Ling Jiang and Zhi-Feng Liu},
  doi          = {10.1002/jcc.26560},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1514-1525},
  shortjournal = {J. Comput. Chem.},
  title        = {Evolution of the linker in microhydrated hydrogen dinitrate anions: From h+ to H5O2+},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Solid-state quantum chemistry with ΘΦ (ThetaPhi):
Spin-liquids, superconductors, and magnetic superstructures made
computationally available. <em>JCC</em>, <em>42</em>(21), 1498–1513. (<a
href="https://doi.org/10.1002/jcc.26561">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a standalone ΘΦ (ThetaPhi) package capable to read the results of ab initio DFT/PAW quantum-chemical solid-state calculations processed through various tools projecting them to the atomic basis states as an input and to perform on top of this an analysis of so derived electronic structure which includes (among other options) the possibility to obtain a superconducting (Bardeen-Cooper-Schrieffer, BCS), spin-liquid (resonating valence bond, RVB) states/phases as solutions of the electronic structure problem along with the magnetically ordered phases with an arbitrary pitch (magnetic superstructure) vector. Remarkably, different solutions of electronic-structure problems come out as temperature-dependent (exemplified by various superconducting and spin-liquid phases) which feature is as well implemented. All that is exemplified by model calculations on 1D chain, 2D square lattice as well as on more realistic superconducting doped graphene, magnetic phases of iron, and spin-liquid and magnetically ordered states of a simplest nitrogen-based copper pseudo-oxide, CuNCN, resembling socalled metal-oxide framework (MOF) phases by the atomic interlinkage.},
  archive      = {J_JCC},
  author       = {Andrei Tchougréeff and Evgeny Plekhanov and Richard Dronskowski},
  doi          = {10.1002/jcc.26561},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1498-1513},
  shortjournal = {J. Comput. Chem.},
  title        = {Solid-state quantum chemistry with ΘΦ (ThetaPhi): Spin-liquids, superconductors, and magnetic superstructures made computationally available},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Accurate prediction of the properties of materials using the
CAM-B3LYP density functional. <em>JCC</em>, <em>42</em>(21), 1486–1497.
(<a href="https://doi.org/10.1002/jcc.26558">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Density functionals with asymptotic corrections to the long-range potential provide entry-level methods for calculations on molecules that can sustain charge transfer, but similar applications in materials science are rare. We describe an implementation of the CAM-B3LYP range-separated functional within the Vienna Ab-initio Simulation Package (VASP) framework, together with its analytical functional derivatives. Results obtained for eight representative materials: aluminum, diamond, graphene, silicon, NaCl, MgO, 2D h-BN, and 3D h-BN, indicate that CAM-B3LYP predictions embody mean-absolute deviations (MAD) compared to HSE06 that are reduced by a factor of six for lattice parameters, four for quasiparticle band gaps, three for the lowest optical excitation energies, and six for exciton binding energies. Further, CAM-B3LYP appears competitive compared to ab initio G0W0and Bethe-Salpeter equation approaches. The CAM-B3LYP implementation in VASP was verified by comparison of optimized geometries and reaction energies for isolated molecules taken from the ACCDB database, evaluated in large periodic unit cells, to analogous results obtained using Gaussian basis sets. Using standard GW pseudopotentials and energy cutoffs for the plane-wave calculations and the aug-cc-pV5Z basis set for the atomic-basis ones, the MAD in energy for 1738 chemical reactions was 0.34 kcal mol−1, while for 480 unique bond lengths this was 0.0036 Å; these values reduced to 0.28 kcal mol−1(largest error 0.94 kcal mol−1) and 0.0009 Å by increasing the plane-wave cutoff energy to 850 eV.},
  archive      = {J_JCC},
  author       = {Musen Li and Jeffrey R. Reimers and Michael J. Ford and Rika Kobayashi and Roger D. Amos},
  doi          = {10.1002/jcc.26558},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1486-1497},
  shortjournal = {J. Comput. Chem.},
  title        = {Accurate prediction of the properties of materials using the CAM-B3LYP density functional},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Local current analysis on defective zigzag graphene
nanoribbons devices for biosensor material applications. <em>JCC</em>,
<em>42</em>(21), 1475–1485. (<a
href="https://doi.org/10.1002/jcc.26557">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this contribution, we aim at investigating the mechanism of biosensing in graphene-based materials from first principles. Inspired by recent experiments, we construct an atomistic model composed of a pyrene molecule serving as a linker fragment, which is used in experiment to attach certain aptamers, and a defective zigzag graphene nanoribbons (ZGNRs). Density functional theory including dispersive interaction is employed to study the energetics of the linker absorption on the defective ZGNRs. Combining non-equilibrium Green&#39;s function and the Landauer formalism, the total current-bias voltage dependence through the device is evaluated. Modifying the distance between the linker molecule and the nanojunction plane reveals a quantitative change in the total current-bias voltage dependence, which correlates to the experimental measurements. In order to illuminate the geometric origin of these variation observed in the considered systems, the local currents through the device are investigated using the method originally introduced by Evers and co-workers. In our new implementation, the numerical efficiency is improved by applying sparse matrix storage and spectral filtering techniques, without compromising the resolution of the local currents. Local current density maps qualitatively demonstrate the local variation of the interference between the linker molecule and the nanojunction plane.},
  archive      = {J_JCC},
  author       = {Jingjing Shao and Beate Paulus and Jean Christophe Tremblay},
  doi          = {10.1002/jcc.26557},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1475-1485},
  shortjournal = {J. Comput. Chem.},
  title        = {Local current analysis on defective zigzag graphene nanoribbons devices for biosensor material applications},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Exploring the quality of protein structural models from a
bayesian perspective. <em>JCC</em>, <em>42</em>(21), 1466–1474. (<a
href="https://doi.org/10.1002/jcc.26556">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We explore how ideas and practices common in Bayesian modeling can be applied to help assess the quality of 3D protein structural models. The basic premise of our approach is that the evaluation of a Bayesian statistical model&#39;s fit may reveal aspects of the quality of a structure when the fitted data is related to protein structural properties. Therefore, we fit a Bayesian hierarchical linear regression model to experimental and theoretical13Cαchemical shifts. Then, we propose two complementary approaches for the evaluation of such fitting: (a) in terms of theexpected differencesbetween experimental and posterior predicted values; (b) in terms of theleave-one-out cross-validation point-wise predictive accuracy. Finally, we present visualizations that can help interpret these evaluations. The analyses presented in this article are aimed to aid in detecting problematic residues in protein structures. The code developed for this work is available on:https://github.com/BIOS-IMASL/Hierarchical-Bayes-NMR-Validation.},
  archive      = {J_JCC},
  author       = {Agustina Arroyuelo and Jorge A. Vila and Osvaldo A. Martin},
  doi          = {10.1002/jcc.26556},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1466-1474},
  shortjournal = {J. Comput. Chem.},
  title        = {Exploring the quality of protein structural models from a bayesian perspective},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). QSARINS-chem standalone version: A new platform-independent
software to profile chemicals for physico-chemical properties, fate, and
toxicity. <em>JCC</em>, <em>42</em>(20), 1452–1460. (<a
href="https://doi.org/10.1002/jcc.26551">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The new software QSARINS-Chem standalone version is a multiplatform tool, freely downloadable, for thein silicoprofiling of multiple properties and activities of organic chemicals. This software, which is based on the concept of the QSARINS-chem module embedded in the QSARINS software, has been fully redesigned and redeveloped in the Java™ language. In addition to a selection of models included in the old module, the new software predicts biotransformation rates and aquatic toxicities of pharmaceuticals and personal care products in multiple organisms, and offers a suite of tools for the analysis of predictions. Furthermore, a comprehensive and transparent database of molecular structures is provided. The new QSARINS-Chem standalone version is an informative and solid tool, which is useful to support the assessment of the potential hazard and risks related to organic chemicals and is dedicated to users which are interested in the application of QSARs to generate reliable predictions.},
  archive      = {J_JCC},
  author       = {Nicola Chirico and Alessandro Sangion and Paola Gramatica and Linda Bertato and Ilaria Casartelli and Ester Papa},
  doi          = {10.1002/jcc.26551},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1452-1460},
  shortjournal = {J. Comput. Chem.},
  title        = {QSARINS-chem standalone version: A new platform-independent software to profile chemicals for physico-chemical properties, fate, and toxicity},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Unveiling gas-phase oxidative coupling of methane via data
analysis. <em>JCC</em>, <em>42</em>(20), 1447–1451. (<a
href="https://doi.org/10.1002/jcc.26554">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unveiling the details of the mechanisms of a chemical reaction is a difficult task as reaction mechanisms are strongly coupled with reaction conditions. Here, catalysts informatics combined with high-throughput experimental data is implemented to understand the oxidative coupling of methane (OCM) reaction. In particular, pairwise correlation and data visualization are performed to reveal the relation between reaction conditions and selectivity/conversion. In addition, machine learning is used to fill the gap between experimental data points; thus, a more detailed understanding of the OCM reaction against reaction conditions can be achieved. Therefore, catalysts informatics is proposed for understanding the details of the reaction mechanism, thereby aiding reaction design.},
  archive      = {J_JCC},
  author       = {Sora Ishioka and Itsuki Miyazato and Lauren Takahashi and Thanh Nhat Nguyen and Toshiaki Taniike and Keisuke Takahashi},
  doi          = {10.1002/jcc.26554},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1447-1451},
  shortjournal = {J. Comput. Chem.},
  title        = {Unveiling gas-phase oxidative coupling of methane via data analysis},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Multimolecular complexes of the phosphodiester anion with
zn(II) or mg(II) and water molecules—preliminary validations of a
polarizable potential by ab initio quantum chemistry. <em>JCC</em>,
<em>42</em>(20), 1430–1446. (<a
href="https://doi.org/10.1002/jcc.26555">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dimethyl phosphate (DMP−) is a model for the phosphodiester backbone of DNA, RNA, and phospholipids. It is central for the binding of divalent cations and water along the backbone of nucleic acids. Significant polarization and charge-transfer contributions and nonadditivity come into play in the multimolecular complexes organized around phosphate. Prior to large-scale molecular dynamics (MD) with advanced polarizable potentials, it is essential to evaluate how well the values and trends of intermolecular interaction energies (ΔE) from ab initio quantum chemistry (QC) and their individual contributions are reproduced in a diversity of such complexes. These differ by the starting binding modes of a divalent cation, Zn(II), namely direct, bi- or mono-dentate to anionic and/or ester oxygens, versus through-water binding. We present first the results from automated refinements of the individual contributions of the SIBFA potential with respect to their QC counterparts using a Zn(II) or a water probe. This is followed by validations on eight relaxed multimolecular complexes of DMP−with Zn(II) or Mg(II) and seven waters, then on sixteen complexes of DMP−with Zn(II) and eight waters in arrangements extracted from MD or energy-minimization on a droplet of sixty-four waters. This monitors the compared evolutions of SIBFA and QC ΔEand their individual contributions in the competing arrangements. Some waters, bridging Zn(II) and DMP−, were found to have exceptionally large dipole moments, of up to 3.8 Debye. The perspectives of extension to a flexible phosphodiester backbone are discussed in the context of the SIBFA potential for DNA and RNA.},
  archive      = {J_JCC},
  author       = {Nohad Gresh and David Perahia},
  doi          = {10.1002/jcc.26555},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1430-1446},
  shortjournal = {J. Comput. Chem.},
  title        = {Multimolecular complexes of the phosphodiester anion with Zn(II) or Mg(II) and water molecules—Preliminary validations of a polarizable potential by ab initio quantum chemistry},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Describing ground and excited state potential energy
surfaces for molecular photoswitches using coupled cluster models.
<em>JCC</em>, <em>42</em>(20), 1419–1429. (<a
href="https://doi.org/10.1002/jcc.26553">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this article, we use two extensively studied systems, a retinal model system and azobenzene, to explore the use of coupled cluster models for describing ground and singlet excited state potential energy surfaces of photoswitchable systems. While not being suitable for describing nuclear dynamics of photoisomerization, coupled cluster models have useful attributes, such as the inclusion of dynamical correlation, their black box nature, and the systematic improvement offered by truncation level. Results for the studied systems show that when triple excitations (here through the CC3 model) are included, ground and excited state potential energy surfaces for isomerization paths may reliably be generated, also for states of doubly excited character. For ground state equilibrium cis- and trans-azobenzene, the molecular geometry and basis set is seen to significantly impact the vertical excitation energies for the two lowest excited states. Efficient implementations of coupled cluster models can therefore constitute valuable tools for investigating photoswitchable systems and can be used for preliminary black box studies to gather information before more complicated excited state dynamics approaches are pursued.},
  archive      = {J_JCC},
  author       = {Anders Hutcheson and Alexander Christian Paul and Rolf H. Myhre and Henrik Koch and Ida-Marie Høyvik},
  doi          = {10.1002/jcc.26553},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1419-1429},
  shortjournal = {J. Comput. Chem.},
  title        = {Describing ground and excited state potential energy surfaces for molecular photoswitches using coupled cluster models},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Geometry dependence of excitonic couplings and the
consequences for configuration-space sampling. <em>JCC</em>,
<em>42</em>(20), 1402–1418. (<a
href="https://doi.org/10.1002/jcc.26552">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Excitonic coupling plays a key role for the understanding of excitonic energy transport (EET) in, for example, organic photovoltaics. However, the calculation of realistic systems is often beyond the applicability range of accurate wavefunction methods so that lower-scaling semi-empirical methods are used to model EET events. In the present work, the distance and angle dependence of excitonic couplings of dimers of selected organic molecules are evaluated for the semi-empirical long-range corrected density functional based tight binding (LC-DFTB) method and spin opposite scaled second order approximate coupled cluster singles and doubles (SOS-CC2). While semi-empirically scaled methods can lead to slightly increased deviations for excitation energies, the excitonic couplings and their dependence on the dimer geometry are reproduced. LC-DFTB yields a similar accuracy range as density-functional theory (DFT) employing theωB97X functional while the computation time is reduced by several orders of magnitude. The dependence of the exchange contributions to the excitonic couplings on the dimer geometry is analyzed assessing the calculation of Coulombic excitonic couplings from monomer local excited states only, which reduces the computational effort significantly. The present work is a necessary first step toward the simulation of excitonic energy transport using semi-empirical methods.},
  archive      = {J_JCC},
  author       = {Nils Schieschke and Beatrix M. Bold and Philipp M. Dohmen and Daniel Wehl and Marvin Hoffmann and Andreas Dreuw and Marcus Elstner and Sebastian Höfener},
  doi          = {10.1002/jcc.26552},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1402-1418},
  shortjournal = {J. Comput. Chem.},
  title        = {Geometry dependence of excitonic couplings and the consequences for configuration-space sampling},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Hybrid localized graph kernel for machine learning
energy-related properties of molecules and solids. <em>JCC</em>,
<em>42</em>(20), 1390–1401. (<a
href="https://doi.org/10.1002/jcc.26550">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nowadays, the coupling of electronic structure and machine learning techniques serves as a powerful tool to predict chemical and physical properties of a broad range of systems. With the aim of improving the accuracy of predictions, a large number of representations for molecules and solids for machine learning applications has been developed. In this work we propose a novel descriptor based on the notion of molecular graph. While graphs are largely employed in classification problems in cheminformatics or bioinformatics, they are not often used in regression problem, especially of energy-related properties. Our method is based on a local decomposition of atomic environments and on the hybridization of two kernel functions: a graph kernel contribution that describes the chemical pattern and a Coulomb label contribution that encodes finer details of the local geometry. The accuracy of this new kernel method in energy predictions of molecular and condensed phase systems is demonstrated by considering the popular QM7 and BA10 datasets. These examples show that the hybrid localized graph kernel outperforms traditional approaches such as, for example, the smooth overlap of atomic positions and the Coulomb matrices.},
  archive      = {J_JCC},
  author       = {Bastien Casier and Mauricio Chagas da Silva and Michael Badawi and Fabien Pascale and Tomáš Bučko and Sébastien Lebègue and Dario Rocca},
  doi          = {10.1002/jcc.26550},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1390-1401},
  shortjournal = {J. Comput. Chem.},
  title        = {Hybrid localized graph kernel for machine learning energy-related properties of molecules and solids},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The extended eighth-shell method for periodic boundary
conditions with rotational symmetry. <em>JCC</em>, <em>42</em>(19),
1373–1383. (<a href="https://doi.org/10.1002/jcc.26545">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Eighth-Shell method for parallelization of molecular dynamics simulations has previously been shown to be the most optimal for efficiency at large process counts. However, in its current formulation only theP1 space group is supported for periodic boundary conditions (PBC) and thus reflection and/or rotational crystal symmetries are not supported. In this work, we outline the development and implementation of the Extended Eighth-Shell (EES) method that allows rotational symmetry by using an extended import region compared to the ES method. It simulates only the asymmetric unit and communicates coordinates and forces with images that correspond toP21PBC. TheP21PBC has application in lipid bilayer simulations as it can be used to allow lipids to switch leaftlets, thus rapidly balancing the chemical potential difference between the two layers. Our results show that the EES method scales efficiently over large number of processes and can be used for simulations withP21symmetry in an orthorhombic crystal.},
  archive      = {J_JCC},
  author       = {Samarjeet Prasad and Andrew C. Simmonett and Rubén Meana-Pañeda and Bernard R. Brooks},
  doi          = {10.1002/jcc.26545},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1373-1383},
  shortjournal = {J. Comput. Chem.},
  title        = {The extended eighth-shell method for periodic boundary conditions with rotational symmetry},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Understanding the intermolecular diels–alder cycloaddition
promotion: Activation strain model/energy decomposition analysis model
and conceptual density functional theory viewpoints. <em>JCC</em>,
<em>42</em>(19), 1364–1372. (<a
href="https://doi.org/10.1002/jcc.26548">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The present work reports the computational study of the major Diels–Alder reaction between 2-bromocycloalkenone and a variety of mono- and di-substituted dienes. Through density functional theory (DFT) calculations and subsequent activation strain model/energy decomposition analysis/conceptual DFT (C-DFT) analyses, the key factors governing the activation barriers heights, and thus reactivity, are characterized. In contrast with a previous study, steric effects do not appear to control reactivity. Conversely, in all presented cases, a subtle interplay between deformation and interaction energies is evidenced at transition states. In the end, neither term alone is enough to explain or predict reactivity. Yet a simple C-DFT descriptor allows to predict with a reasonable efficiency the activation barriers: the excitation energy needed to observe a charge transfer from the diene to the dienophile. Theoretical elements are provided to support the use of this descriptor.},
  archive      = {J_JCC},
  author       = {Lynda Merzoud and Frédéric Guégan and Henry Chermette and Christophe Morell},
  doi          = {10.1002/jcc.26548},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1364-1372},
  shortjournal = {J. Comput. Chem.},
  title        = {Understanding the intermolecular Diels–Alder cycloaddition promotion: Activation strain model/energy decomposition analysis model and conceptual density functional theory viewpoints},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The comparison of structure, nature of bond, and electronic
transitions in [m(η5-cp)(η5-C60Me5)] (m = fe2+, ru2+, os2+) hybrids and
corresponding metallocenes; a theoretical study. <em>JCC</em>,
<em>42</em>(19), 1354–1363. (<a
href="https://doi.org/10.1002/jcc.26542">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper, metallocene-fullerene hybrid complexes, [M(η5-Cp)(η5-C60Me5)] (M = Fe2+, Ru2+, Os2+), as well as corresponding classical metallocenes have been studied theoretically at BP86/def2-SVP and M06L/def2-SVP levels of theory. With considering these metal complexes as an ABA′ system (B is the central metal ion and A and A′ are relatedη5-ligands), the total interaction energies were calculated using common methods, as well as by calculating the interaction energies between the four defined pairs of fragments including A–B, B–A′, A–BA′, and AB–A′. The resulting data clearly showed that in all complexes there is a strong anticoopertivity between two metal-(η5-ligand) bonds. In order to understand the origin of difference in values of various calculated interactions in above two types of complexes, the nature of metal–ligand bonds was also studied using energy decomposition analysis-natural orbital for chemical valence calculations. The results showed that in hybrid complexes, in contrast to metallocenes, the orbital interactions are considerably larger than electrostatic interactions.},
  archive      = {J_JCC},
  author       = {Samaneh Hokmi and Sadegh Salehzadeh and Yasin Gholiee},
  doi          = {10.1002/jcc.26542},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1354-1363},
  shortjournal = {J. Comput. Chem.},
  title        = {The comparison of structure, nature of bond, and electronic transitions in [M(η5-cp)(η5-C60Me5)] (M = fe2+, ru2+, os2+) hybrids and corresponding metallocenes; a theoretical study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Dissociation of hydrogen peroxide in water and methanol
through a biased molecular dynamics investigation. <em>JCC</em>,
<em>42</em>(19), 1344–1353. (<a
href="https://doi.org/10.1002/jcc.26539">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The two dissociation channels of HOOH, namely, HOOH and HOOH, in water and methanol are investigated using umbrella-sampling ab initio molecular dynamics. Our potential of mean force calculations reveals the HOOH dissociation to be more favorable in methanol with a free energy barrier of 7.56 kcal/mol, while the HOOH dissociation possesses a free energy barrier of 11.46 kcal/mol. In water, the HOOH dissociation channel is more favorable (8.25 kcal/mol), while the HOOH dissociation process requires a higher free energy (11.28 kcal/mol). Such reaction favorability can be explained by inspecting the formation of secondary radical species during the course of multiple hydrogen donating-accepting processes in each reaction channel. The radical species, that is, H3O•(observed in water) and CH3OH2•(observed in methanol), are the first subordinate species upon the HOOH dissociation. For the HOOH dissociation channel in methanol, the secondary species such as water and formaldehyde can be observed, while the re-generation of HOOH in water can be spotted.},
  archive      = {J_JCC},
  author       = {Hieu C. Dong and Thi H. Ho and Thu M. Nguyen and Yoshiyuki Kawazoe and Hung M. Le},
  doi          = {10.1002/jcc.26539},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1344-1353},
  shortjournal = {J. Comput. Chem.},
  title        = {Dissociation of hydrogen peroxide in water and methanol through a biased molecular dynamics investigation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Response to comment on “superposition of waves or
densities: Which is the nature of chemical resonance?” <em>JCC</em>,
<em>42</em>(19), 1341–1343. (<a
href="https://doi.org/10.1002/jcc.26523">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {I reply to the comment by Weinhold and Glendening on the article (J. Comput. Chem.2021,42, 412). I provide further explanation and an additional numerical example to support my previous assertion that the present form of natural resonance theory is fundamentally flawed, at least within the DFT framework.},
  archive      = {J_JCC},
  author       = {Yang Wang},
  doi          = {10.1002/jcc.26523},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1341-1343},
  shortjournal = {J. Comput. Chem.},
  title        = {Response to comment on “Superposition of waves or densities: Which is the nature of chemical resonance?”},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comment on “superposition of waves or densities: Which is
the nature of chemical resonance?” [J. Comput. Chem. 2021, 42, 412–417].
<em>JCC</em>, <em>42</em>(19), 1338–1340. (<a
href="https://doi.org/10.1002/jcc.26696">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Frank Weinhold and Eric D. Glendening},
  doi          = {10.1002/jcc.26696},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1338-1340},
  shortjournal = {J. Comput. Chem.},
  title        = {Comment on “Superposition of waves or densities: Which is the nature of chemical resonance?” [J. comput. chem. 2021, 42, 412–417]},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MoSDeF cassandra: A complete python interface for the
cassandra monte carlo software. <em>JCC</em>, <em>42</em>(18),
1321–1331. (<a href="https://doi.org/10.1002/jcc.26544">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce a new Python interface for the Cassandra Monte Carlo software, molecular simulation design framework (MoSDeF) Cassandra. MoSDeF Cassandra provides a simplified user interface, offers broader interoperability with other molecular simulation codes, enables the construction of programmatic and reproducible molecular simulation workflows, and builds the infrastructure necessary for high-throughput Monte Carlo studies. Many of the capabilities of MoSDeF Cassandra are enabled via tight integration with MoSDeF. We discuss the motivation and design of MoSDeF Cassandra and proceed to demonstrate both simple use-cases and more complex workflows, including adsorption in porous media and a combined molecular dynamics – Monte Carlo workflow for computing lateral diffusivity in graphene slit pores. The examples presented herein demonstrate how even relatively complex simulation workflows can be reduced to, at most, a few files of Python code that can be version-controlled and shared with other researchers. We believe this paradigm will enable more rapid research advances and represents the future of molecular simulations.},
  archive      = {J_JCC},
  author       = {Ryan S. DeFever and Ray A. Matsumoto and Alexander W. Dowling and Peter T. Cummings and Edward J. Maginn},
  doi          = {10.1002/jcc.26544},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1321-1331},
  shortjournal = {J. Comput. Chem.},
  title        = {MoSDeF cassandra: A complete python interface for the cassandra monte carlo software},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-order harmonic generation by static coherent states
method in single-electron atomic and molecular systems. <em>JCC</em>,
<em>42</em>(18), 1312–1320. (<a
href="https://doi.org/10.1002/jcc.26549">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We solve the time-dependent Schrodinger equation using the coherent states as basis sets for computing high harmonic generation (HHG) in a full-dimensional single-electron “realistic” system. We apply the static coherent states (SCS) method to investigate HHG in the hydrogen molecular ion induced by a linearly polarized laser field. We show that SCS gives reasonable agreement compared to the three dimensional unitary split-operator approach. Next, we study isolated attosecond pulse generation in. To do so, we employ the well-known polarization gating technique, which combines two delayed counter-rotating circular laser pulses, and opens up a gate at the central portion of the superposed pulse. Our results suggest that the SCS method can be used for full-dimensional quantum simulation of higher dimensional systems such as the hydrogen molecule in the presence of an external laser field.},
  archive      = {J_JCC},
  author       = {Mohammadreza Eidi and Mohsen Vafaee and Hamed Koochaki Kelardeh and Alexandra Landsman},
  doi          = {10.1002/jcc.26549},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1312-1320},
  shortjournal = {J. Comput. Chem.},
  title        = {High-order harmonic generation by static coherent states method in single-electron atomic and molecular systems},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A computational investigation of the selectivity and
mechanism of the lewis acid catalyzed oxa-diels–alder cycloaddition of
substituted diene with benzaldehyde. <em>JCC</em>, <em>42</em>(18),
1296–1311. (<a href="https://doi.org/10.1002/jcc.26547">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The selectivity and the mechanism of the uncatalyzed and AlCl3catalyzed hetero-Diels–Alder reaction (HDR) between ([E]-4-methylpenta-2,4-dienyloxy)(tert-butyl)dimethylsilane1and benzaldehyde2have been studied using density functional theory at the MPWB1K/6-31G(d) level of theory. The uncatalyzed HDR between diene1and alkene2is characterized by a polar character and proceeds via an asynchronous one-step mechanism for themetapaths and synchronous for theorthoones. In the presence of AlCl3catalyst, the mechanism changes to be stepwise, while the first step is the rate-determining step. The activation energies widely decrease, and the polar character increases dramatically. A large analysis of the mechanism is performed using the activation strain model/energy decomposition analysis (ASM/EDA) model, the natural bond orbital (NBO) and state specific dual descriptors (SSDDs). The obtained results indicate that the combined interaction energy associated with the distortion of the reactants in these HDR are at the origin of the observed kinetics. NBO analyses were applied to estimate the Lewis-acid catalyst donor-acceptor interaction with the molecular system. The SSDD analysis shed light into the orientation effects on the reaction kinetics by providing important information about charge transfer interactions during the chemical reaction. It indicates that the more favorable HDR pathway have the lowest excitation energies, facilitating the interaction between diene1and benzaldehyde2moieties. Non-covalent interaction (NCI) and QTAIM analyses of themeta-endostructure indicate that the presence of several weak NCIs formed at this approach is at the origin of themeta-endoselectivity.},
  archive      = {J_JCC},
  author       = {Abdelmalek Khorief Nacereddine and Lynda Merzoud and Christophe Morell and Henry Chermette},
  doi          = {10.1002/jcc.26547},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1296-1311},
  shortjournal = {J. Comput. Chem.},
  title        = {A computational investigation of the selectivity and mechanism of the lewis acid catalyzed oxa-Diels–Alder cycloaddition of substituted diene with benzaldehyde},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Uncertainty of exchange-correlation functionals in density
functional theory calculations for lithium-based solid electrolytes on
the case study of lithium phosphorus oxynitride. <em>JCC</em>,
<em>42</em>(18), 1283–1295. (<a
href="https://doi.org/10.1002/jcc.26546">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amorphous lithium phosphorus oxynitride (LIPON) has emerged as a promising solid electrolyte for all-solid-state thin-film lithium batteries. In this context, the use of theoretical modeling to characterize, understand, or screen material properties is becoming increasingly important to complement experimental analysis or elucidate features at atomistic level that are difficult to obtain through experimental studies. Density functional theory (DFT) is the method of choice for quantum mechanical material modeling at the atomistic scale. The current state of the art represents DFT values, such as the formation or migration energies relevant for bulk phase of materials, as absolute numbers. Estimating the accuracy or fluctuation range of the different density functionals is challenging. In order to investigate the thermodynamic and kinetic properties of LIPON by DFT, an approach to describe the fluctuation range caused by the choice of the exchange-correlation (XC) functional is developed. Three different model systems were chosen to characterize various structural features of amorphous LIPON, which are distinguished by the cross-linking of the POuN4-u-structural units. The uncertaintyŨis introduced as a parameter describing the fluctuation range of energy values. The uncertainty approach does not determine the accuracy of DFT results, but rather a fluctuation range in the DFT results without the need for a reference value from a higher level of theory or experiment. The uncertainty was determined for both the thermodynamic Li-vacancy formation energies and the kinetic Li-vacancy migration energies in LIPON. We assume that the uncertainty approach can be applied to different material systems with different density functionals.},
  archive      = {J_JCC},
  author       = {Pascal Henkel and Doreen Mollenhauer},
  doi          = {10.1002/jcc.26546},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1283-1295},
  shortjournal = {J. Comput. Chem.},
  title        = {Uncertainty of exchange-correlation functionals in density functional theory calculations for lithium-based solid electrolytes on the case study of lithium phosphorus oxynitride},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). On the use of multiple-time-step algorithms to save
computing effort in molecular dynamics simulations of proteins.
<em>JCC</em>, <em>42</em>(18), 1263–1282. (<a
href="https://doi.org/10.1002/jcc.26541">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computer simulation of proteins in aqueous solution at the atomic level of resolution is still limited in time span and system size due to limited computing power available and thus employs a variety of time-saving techniques that trade some accuracy against computational effort. Examples of such time-saving techniques are the application of constraints to particular degrees of freedom or the use of a multiple-time-step (MTS) algorithm distinguishing between particular forces when integrating Newton&#39;s equations of motion. The application of two types of MTS algorithms to bond-stretching forces versus the remaining forces in molecular dynamics (MD) simulations of a protein in aqueous solution or of liquid water is investigated and the results in terms of total energy conservation and the influence on various other properties are compared to those of MD simulations of the same systems using bond-length, and for water bond-angle, constraints. At comparable computational effort, the use of bond-length constraints in proteins leads to better energy conservation and less distorted properties than the two MTS algorithms investigated.},
  archive      = {J_JCC},
  author       = {Maria Pechlaner and Chris Oostenbrink and Wilfred F. van Gunsteren},
  doi          = {10.1002/jcc.26541},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1263-1282},
  shortjournal = {J. Comput. Chem.},
  title        = {On the use of multiple-time-step algorithms to save computing effort in molecular dynamics simulations of proteins},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The best density functional theory functional for the
prediction of 1H and 13C chemical shifts of protonated alkylpyrroles.
<em>JCC</em>, <em>42</em>(18), 1248–1262. (<a
href="https://doi.org/10.1002/jcc.26540">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prediction of13C chemical shifts can be challenging with density functional theory (DFT). In this study 39 different functionals and three different basis sets were tested on three neutral alkylpyrroles and their corresponding protonated species. The calculated shielding constants were compared to experimental data and results from previous calculations at the MP2. We find that the meta-hybrid functional TPSSh with either the Pople style basis set 6-311++G(2d,p) or the polarization consistent basis set pcSseg-1 gives the best results for the13C chemical shifts, whereas for the1H chemical shifts it is the TPSSh functional with either the 6-311++G(2d,p) or pcSseg-2 basis set. Including an explicit solvent molecule hydrogen bonded to NH in the alkylpyrroles improves the results slightly for the13C chemical shifts. On the other hand, for1H chemical shifts the opposite is true. Compared to calculations at the MP2 level none of the DFT functionals can compete with MP2 for the13C chemical shifts but for the1H chemical shifts the investigated DFT functionals are shown to give better agreement with experiment than MP2 calculations.},
  archive      = {J_JCC},
  author       = {Sarah L. V. Zahn and Ole Hammerich and Poul Erik Hansen and Stephan P. A. Sauer},
  doi          = {10.1002/jcc.26540},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1248-1262},
  shortjournal = {J. Comput. Chem.},
  title        = {The best density functional theory functional for the prediction of 1H and 13C chemical shifts of protonated alkylpyrroles},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modeling ligand electrochemical parameters by
repulsion-corrected eigenvalues. <em>JCC</em>, <em>42</em>(17),
1236–1242. (<a href="https://doi.org/10.1002/jcc.26536">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ligand electrochemical parameters,EL, more commonly known as Lever parameters, have played a major research role in understanding redox processes involved in inorganic electrochemistry, enzymatic reactions, catalysis, solar cells, biochemistry, and materials science. Despite their broad usefulness, Lever parameters are not well understood at a first-principles level. Using density functional theory, we demonstrate in this contribution that a ligand&#39;s Lever parameter is fundamentally related to the ligand&#39;s ability to alter the eigenvalue of the electroactive spin-orbital in an octahedral transition metal complex. Our analysis furthers a first-principles understanding of the nature of Lever parameters.},
  archive      = {J_JCC},
  author       = {Pirouz Kiani and Elaine S. Dodsworth and A. B. P. Lever and William J. Pietro},
  doi          = {10.1002/jcc.26536},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1236-1242},
  shortjournal = {J. Comput. Chem.},
  title        = {Modeling ligand electrochemical parameters by repulsion-corrected eigenvalues},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Equilibrium between tri- and tetra-coordinate
chalcogenuranes is critical for cysteine protease inhibition.
<em>JCC</em>, <em>42</em>(17), 1225–1235. (<a
href="https://doi.org/10.1002/jcc.26535">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There have been significant advances in the biological use of hypervalent selenium and tellurium compounds as cysteine protease inhibitors. However, the full understanding of their reaction mechanisms for and cysteine proteases inhibition is still elusive. Kinetic studies suggest an irreversible inhibition mechanism, which was explained by forming a covalent bond between the enzyme sulfhydryl group and the chalcogen atom at its hypervalent state (+4). In this work, we performed a theoretical investigation using density functional theory to propose the active inhibitor form in an aqueous solution. To this end, we investigated chloride ligand exchange reactions by oxygen and sulfur nucleophiles on hypervalent selenium and tellurium compounds. All tetra- and tri-coordinated chalcogen compounds and distinct protonation states of the nucleophiles were considered, totaling 34 unique species, 7 nucleophiles, and 155 free energies reactions. We discovered that chloride is easily replaced by a nonprotonated nucleophile (SH−or OH−) in R2SeCl2. We also found that tri-coordinate species are more stable than their tetra-coordinate counterparts, with selenoxide (R2SeO) protonation being strongly exergonic in acid pH. The thermodynamic and kinetic results suggest that the protonated selenoxide (R2SeOH+) is the most probable active chemical species in biological media. The computed energetic profiles paint a possible picture for selenuranes activity, with successive exergonic steps leading to a covalent inhibition of thiol-dependent enzymes, like cysteine proteases. A second pathway has also been uncovered, with a direct reaction to chalcogenonium cation (R2SeCl+) as the inhibition step. Tellurium compounds showed similar trends but formed telluroxide in a pH-independent fashion.},
  archive      = {J_JCC},
  author       = {Gabriela Dias-da-Silva and Rodrigo L. O. R. Cunha and Mauricio D. Coutinho-Neto},
  doi          = {10.1002/jcc.26535},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1225-1235},
  shortjournal = {J. Comput. Chem.},
  title        = {Equilibrium between tri- and tetra-coordinate chalcogenuranes is critical for cysteine protease inhibition},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modeling UV–vis spectra of low dimensional materials using
electrostatic embedding: The case of CdSe. <em>JCC</em>,
<em>42</em>(17), 1212–1224. (<a
href="https://doi.org/10.1002/jcc.26534">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a generalization of a self-consistent electrostatic embedding approach (SC-Ewald) devised to investigate the photophysical properties of 3D periodic materials, to systems in one- or two-dimensional (2D) reduced periodicity. In this approach, calculations are carried out on a small finite molecular cluster extracted from a periodic model, while the crystalline environment is accounted for by an array of point charges which are fitted to reproduce the exact electrostatic potential (at ground or the excited state) of the infinite periodic system. Periodic density functional theory (DFT) calculations are combined with time dependent DFT calculations to simulate absorption and emission properties of the extended system under investigation. We apply this method to compute the UV–Vis. spectra of bulk and quantum-confined 0D quantum dots and 2D extended nanoplatelets of CdSe, due to their relevance as sensitizers in solar cells technologies. The influence of the size and shape of the finite cluster model chosen in the excited state calculations was also investigated and revealed that, although the long-range electrostatics of the environment are important for the calculation of the UV–Vis, a subtle balance between short- and long-range effects exists. These encouraging results demonstrate that this self-consistent electrostatic embedding approach, when applied in different dimensions, can successfully model the photophysical properties of diverse material classes, making it an attractive low-cost alternative to far more computationally demanding electronic structure methods for excited state calculations.},
  archive      = {J_JCC},
  author       = {Davide Luise and Liam Wilbraham and Frédéric Labat and Ilaria Ciofini},
  doi          = {10.1002/jcc.26534},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1212-1224},
  shortjournal = {J. Comput. Chem.},
  title        = {Modeling UV–Vis spectra of low dimensional materials using electrostatic embedding: The case of CdSe},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Free energy change estimation: The divide and conquer MBAR
method. <em>JCC</em>, <em>42</em>(17), 1204–1211. (<a
href="https://doi.org/10.1002/jcc.26533">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the present study, the Divide and Conquer MBAR (DC-MBAR) method is proposed to predict the free energies based on the data sampled by multi-states simulations. For DC-MBAR method, the overlap between any two alchemical states is calculated first and those with sufficient overlap are defined as the adjacent states. Unlike the traditional MBAR method, which calculates the free energy of each state using all the data at once, DC-MBAR focuses on predicting the free energy changes between adjacent states. To estimate the free energy changes accurately, the other states with overlaps with the two adjacent states bigger than the defined threshold are included in the MBAR equation. At a specific threshold, the free energies predicted by DC-MBAR are very close to those calculated by the traditional MABR method. Furthermore, DC-MBAR scheme can reduce both the computation and memory cost. One important characteristic of DC-MBAR method is linear scaling, which means the CPU time with the change of the number of states is a straight-line relation. As the pair-based calculations are mutually independent and parallelizable, all accessible CPU cores on the HPC cluster could be utilized, which makes DC-MBAR strategy more efficient.},
  archive      = {J_JCC},
  author       = {Xiangyu Jia and Hu Ge and Ye Mei},
  doi          = {10.1002/jcc.26533},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1204-1211},
  shortjournal = {J. Comput. Chem.},
  title        = {Free energy change estimation: The divide and conquer MBAR method},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Understanding the molecular mechanism of γ-elimination of
nitrous acid in the framework of the molecular electron density theory.
<em>JCC</em>, <em>42</em>(17), 1195–1203. (<a
href="https://doi.org/10.1002/jcc.26532">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The reactions of γ-dehydronitration of furaxanenitrolic acids have been studied within the density functional theory using molecular electron density theory scheme at the MPWB1K(PCM)/6-311G(d,p) level of theory. The alteration of bonding along the course of the reaction is studied in the topology of the electron density functional within the bonding evolution theory perspective. The characteristics of electron density changes indicate that we can distinguish six different phases in the nitrous acid extrusion from furaxanenitrolic acid1a. These different phases related to the intrinsic reaction coordinate path of the analyzed reaction denote the non-concerted nature of the molecular mechanism.},
  archive      = {J_JCC},
  author       = {Agnieszka Kącka-Zych and Radomir Jasiński},
  doi          = {10.1002/jcc.26532},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1195-1203},
  shortjournal = {J. Comput. Chem.},
  title        = {Understanding the molecular mechanism of γ-elimination of nitrous acid in the framework of the molecular electron density theory},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fast non-iterative calculation of solvation energies for
water and non-aqueous solvents. <em>JCC</em>, <em>42</em>(17),
1184–1194. (<a href="https://doi.org/10.1002/jcc.26531">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We propose an efficient and accurate non-iterative method, dubbed uESE, for calculating solvation free energies. Apart from a COSMO-like electrostatic term, the model takes into account non-electrostatic contributions, which depend on atomic surfaces, induced surface charge densities, and the molecular volume. uESE is tested on 35 polar and 57 non-polar solvents. The calculated and experimental solvation free energies are compared for 2892 systems. The method exhibits an excellent performance, which is superior to major solvation methods. The mean absolute error of predicted solvation energies is found below 1 kcal/mol for neutral solutes and below 3 kcal/mol for ions. The calculated data are almost independent of the quantum-chemical method or/and basis sets employed.},
  archive      = {J_JCC},
  author       = {Sergei F. Vyboishchikov and Alexander A. Voityuk},
  doi          = {10.1002/jcc.26531},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1184-1194},
  shortjournal = {J. Comput. Chem.},
  title        = {Fast non-iterative calculation of solvation energies for water and non-aqueous solvents},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Stabilization of group 14 elements e = c, si, ge by
hetero-bileptic ligands cAAC, MCOn with push-pull mechanism.
<em>JCC</em>, <em>42</em>(16), 1159–1177. (<a
href="https://doi.org/10.1002/jcc.26530">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The stability and bonding of a series of hetero-diatomic molecules with general formula (cAAC)EM(CO)n, where cAAC = cyclic alkyl(amino) carbene; E = group 14 elements (C, Si, and Ge); M = transition metal (Ni, Fe, and Cr) have been studied by quantum chemical calculations using density functional theory (DFT) and energy decomposition analysis-natural orbital chemical valence (EDA-NOCV). The equilibrium geometries were calculated at the BP86/def2-TZVPP level of theory. The tri-coordinated group 14 complex (1a,4a, and7a) in which one of the CO groups is migrated to the central group 14 element from adjacent metal is theoretically found to be more stable when the central atom (E) is carbon. On the other hand, the two-coordinate group 14 element containing metal-complexes (2,5,8,3,6, and9) are found to be more stable with their corresponding heavier analogues. The electronic structures of all the molecules have been analyzed by molecular orbital, topological analysis of electron density and natural bond orbital (NBO) analysis at the M06/def2-TZVPP//BP86/def2-TZVPP level of theory. The nature of the cAACE and EM bonds has been studied by EDA-NOCV calculations at BP86-D3(BJ)/TZ2P level of theory. The EDA analysis suggests that the bonding of cAACC(CO) can be best represented by electron sharing σ and π interactions, whereas, C(CO)M(CO)n−1by dative σ and π interactions. On the other hand, EDA-NOCV calculations suggests both dative σ and π interactions for cAACE and EM(CO)nbonds of the corresponding Si and Ge analogues having stronger σ- and relatively weaker π-bonds. The topological analysis of electron density supports the closed-shell interaction for the Si and Ge complexes and open-shell interaction for the carbon complexes. The calculated proton affinity and hydride affinity values corroborated well with the present bonding description. This class of complexes might act as efficient future catalysts for different organic transformations due to the presence of electron rich group 14 element and metal carbonyl.},
  archive      = {J_JCC},
  author       = {Sai Manoj N. V. T. Gorantla and Pattiyil Parameswaran and Kartik Chandra Mondal},
  doi          = {10.1002/jcc.26530},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1159-1177},
  shortjournal = {J. Comput. Chem.},
  title        = {Stabilization of group 14 elements e = c, si, ge by hetero-bileptic ligands cAAC, MCOn with push-pull mechanism},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). [10]annulene: Electrocyclization mechanisms. <em>JCC</em>,
<em>42</em>(16), 1150–1158. (<a
href="https://doi.org/10.1002/jcc.26529">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanism of 6-π electrocyclization of all-cis, mono-trans, and double-trans[10]annulene to yield 4a,8a-dihydronaphthalene has been explored at various quantum-chemical methods. The mono-transconfiguration cyclizes preferentially totrans-4a,8a-dihydronaphthalene, in agreement with the experimental results. The cyclization of the all-cisconfiguration requires firstly a bond-shifting to the naphthalene-like conformation of double-trans[10]annulene, which is the rate-limiting step, and finally its azulene-like conformation electrocyclizes quickly tocis-4a,8a-dihydronaphthalene. Its experimental rate coefficient is consistent with the computed one for the cyclization of the all-cisconfiguration, unlike the calculated one for the double-transconfiguration. These results confirm the configurations assigned by Masamune et al. to the two isomers which they isolated.},
  archive      = {J_JCC},
  author       = {Raúl Palmeiro and Obis Castaño},
  doi          = {10.1002/jcc.26529},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1150-1158},
  shortjournal = {J. Comput. Chem.},
  title        = {[10]annulene: Electrocyclization mechanisms},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comparative first principles-based molecular dynamics study
of catalytic mechanism and reaction energetics of water oxidation
reaction on 2D-surface. <em>JCC</em>, <em>42</em>(16), 1138–1149. (<a
href="https://doi.org/10.1002/jcc.26528">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study of the water-splitting process, which can proceed in 2e−as well as 4e−pathway, reveals that the process is entirely an uphill process, and the third step, that is, the oxooxo bond formation is the rate-determining step. The kinetic barrier of the oxygen evolution reaction (OER) on the 2D material catalysts in the presence of explicit solvents is scarcely studied. Here, we investigate the dynamics of the OER on the undoped graphene and the activation energy barrier of each step using first principles molecular dynamics simulations. Here we provide a detailed analysis of the kinetics of all the 4e−transfer steps of OER on the graphene surface. We also compare the accuracy of one of the density functional theory (DFT) functionals and density functional based tight binding (DFTB) method in explaining the OER steps. The comparative study reveals that DFTB can be used for performing metadynamics simulations quipped with much less computational cost than DFT functionals. By both Perdew–Burke–Ernzerhof and DFTB methods, the third step is revealed to be the rate-determining step with an energy barrier of 21.19 ± 0.51 and 20.23 ± 0.20 kcal mol−1, respectively. DFTB gives an impression of being successful in predicting the energy barriers of OER in 4e- transfer pathway and comparable to the DFT method, and we would like to extend the use of DFTB for further studies with a sizable and complex system.},
  archive      = {J_JCC},
  author       = {Adyasa Priyadarsini and Bhabani S. Mallik},
  doi          = {10.1002/jcc.26528},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1138-1149},
  shortjournal = {J. Comput. Chem.},
  title        = {Comparative first principles-based molecular dynamics study of catalytic mechanism and reaction energetics of water oxidation reaction on 2D-surface},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quadruple chemical bonding in the diatomic anions TcN−,
RuC−, RhB−, and PdBe−. <em>JCC</em>, <em>42</em>(16), 1126–1137. (<a
href="https://doi.org/10.1002/jcc.26527">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quadruple bonding is uncommon for main group elements and the identification of species forming such bonds is remarkably interesting particularly in diatomic anions for which there is a lack of information. Here, it is found that the MX−anions, TcN−, RuC−, RhB−, and PdBe−, present quadruple bonding, as do the corresponding MX neutrals, even though a different type of σ2bond is involved in ∑+states of neutral and anions. Specifically, the ground states (X2Δ orX2∑+) of the four anions and their first excited states (A2∑+orA2Δ) of TcN−, RuC−, and RhB−present quadruple bonds consisting of two σ and two π bonds: (4dz2− 2pz)2, 5pz0← 2s2, (4dxz− 2px)2, and (4dyz− 2py)2. Bond lengths, dissociation energies, spectroscopic data and electron affinities were calculated via high-level multireference and coupled-cluster methodology using the aug–cc–pV5ZX(-PP)Mbasis set. Strong bonding results in short bond lengths ranging from 1.602 (TcN−) to 1.944 (PdBe−) Å. Adiabatic (diabatic) binding energies reach up to 139 (184) kcal/mol. Electron affinities (EA) were calculated at 1.368 (TcN), 1.242 (RuC), 0.873 (RhB), 0.743 (PdBe) eV. Only for RhB has EA been measured experimentally at 0.961 eV, in good agreement with the value reported here.},
  archive      = {J_JCC},
  author       = {Demeter Tzeli},
  doi          = {10.1002/jcc.26527},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1126-1137},
  shortjournal = {J. Comput. Chem.},
  title        = {Quadruple chemical bonding in the diatomic anions TcN−, RuC−, RhB−, and PdBe−},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The density polarization reveals directions of electron
displacements due to the substituent effect: Analysis performed on a
metal-organic mo-oxo catalyst. <em>JCC</em>, <em>42</em>(16), 1118–1125.
(<a href="https://doi.org/10.1002/jcc.26526">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Some Mo-oxo complexes bearing pyridine rings have the capability for dihydrogen production from water. However, energy barrier and overall energy vary depending on the effect exerted by several substituent groups located at different positions around one or more pyridine rings which are ligands of these compounds. Based on the Karunadasa and coworkers investigation where thepara-position was experimentally tested in compounds derivatised from the 2,6-bis[1,1-bis(2-pyridil)ethyl]-pyridine oxo-molybdenum complex synthesized (Karunadasa et al., Nature, 2010, 464, 1329), we tested the combined effect of electron-withdrawing and electron-donating groups simulated as perturbations represented by point-charges. Then, we used the density polarization concept,δρ(r), a local reactivity descriptor corresponding to the partially integrated linear response function,χ(r,r′) (a non-local reactivity descriptor), which is able to reveal different displacements ofπ-electrons on molecular structures. We perturbed thepara-positions in the pentadentate ligand 2,6-bis[1,1-bis(2-pyridil)ethyl]-pyridine in the Mo-based complex by means of point-charges. They were located in three different configurations of the organic ligand (trans, geminal, andcis) which could help to explain energy barriers and overall energy of reactions catalyzed by this type of Mo-complexes. Our results indicate that thetransconfiguration of point-charges induces the most amount of fraction of electron shifted on the complex. A Mo-based complex bearing the sametransconfiguration for electron-withdrawing and electron-donating substituent groups (cyano and amino, respectively), leads to a kinetically more favorable H2release than thecisor geminal configuration of the substituent groups aforementioned.},
  archive      = {J_JCC},
  author       = {Jorge I. Martínez-Araya and Christophe Morell},
  doi          = {10.1002/jcc.26526},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1118-1125},
  shortjournal = {J. Comput. Chem.},
  title        = {The density polarization reveals directions of electron displacements due to the substituent effect: Analysis performed on a metal-organic mo-oxo catalyst},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Benchmark calculations of proton affinity and gas-phase
basicity using multilevel (G4 and G3B3), B3LYP and MP2 computational
methods of para-substituted benzaldehyde compounds. <em>JCC</em>,
<em>42</em>(16), 1106–1117. (<a
href="https://doi.org/10.1002/jcc.26538">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study presents the benchmark calculations of proton affinities (PAs) and gas-phase basicities (GBs) of8-para substituted benzaldehyde compounds using the multilevel model chemistries (G3B3 and G4), density-functional quantum model (B3LYP) and ab initio model (MP2). The results show that the computed properties are strongly correlated with the available experimental data. The PAs and the GBs of other eight para-substituted benzaldehyde compounds, for which the experimental data does not currently exist, have been calculated using G3B3 and B3LYP methods. The correlations between the experimental PAs and GBs with the computed properties such as PA, GB, chemical properties (bond lengths, electron density and δ1H NMR chemical shift) of the investigated benzaldehydes have been studied and statistically analyzed. The influence of the substituted groups has been discussed in terms of inductive effect and electron donating and electron withdrawing effect. The results obtained show that the chemical properties of the benzaldehyde compounds are controlled by the strong coupling between theCHO group and the nature of the para-substituent groups through the benzene ring as a conducting linkage.},
  archive      = {J_JCC},
  author       = {Zaki S. Safi and Nuha Wazzan},
  doi          = {10.1002/jcc.26538},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1106-1117},
  shortjournal = {J. Comput. Chem.},
  title        = {Benchmark calculations of proton affinity and gas-phase basicity using multilevel (G4 and G3B3), B3LYP and MP2 computational methods of para-substituted benzaldehyde compounds},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). LSLOpt: An open-source implementation of the step-length
controlled LSL-BFGS algorithm. <em>JCC</em>, <em>42</em>(15), 1095–1100.
(<a href="https://doi.org/10.1002/jcc.26522">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerical optimization is a common technique in various areas of computational chemistry, molecular modeling and drug design. It is a key element of 3D techniques, for example, the optimization of protein–ligand poses and small-molecule conformers. Here, often the BFGS algorithm or variants thereof are used. However, the BFGS algorithm tends to make unreasonable large changes to the optimized system under certain circumstances. This behavior has been known for a long time and different solutions have been suggested. Recently, we have analyzed the optimization behavior of our novel JAMDA scoring function in detail and proposed the limited step length (LSL)-BFGS algorithm as a new solution to the problem of excessively large steps during optimization. The LSL-BFGS algorithm allows to control the step sizes during optimization. Its unique feature is the inclusion of arbitrary domain knowledge into the selection of the step sizes. Here, we introduce the open-source LSLOpt C++ library that implements this LSL-BFGS algorithm and demonstrate its usage.},
  archive      = {J_JCC},
  author       = {Florian Flachsenberg and Matthias Rarey},
  doi          = {10.1002/jcc.26522},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1095-1100},
  shortjournal = {J. Comput. Chem.},
  title        = {LSLOpt: An open-source implementation of the step-length controlled LSL-BFGS algorithm},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A strategy for proline and glycine mutations to proteins
with alchemical free energy calculations. <em>JCC</em>, <em>42</em>(15),
1088–1094. (<a href="https://doi.org/10.1002/jcc.26525">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computation of the thermodynamic consequences of protein mutations holds great promise in protein biophysics and design. Alchemical free energy methods can give improved estimates of mutational free energies, and are already widely used in calculations of relative and absolute binding free energies in small molecule design problems. In principle, alchemical methods can address any amino acid mutation with an appropriate alchemical pathway, but identifying a strategy that produces such a path for proline and glycine mutations is an ongoing challenge. Most current strategies perturb only side chain atoms, while proline and glycine mutations also alter the backbone parameters and backbone ring topology. Some strategies also perturb backbone parameters and enable glycine mutations. This work presents a strategy that enables both proline and glycine mutations and comprises two key elements: a dual backbone with restraints and scaling of bonded terms, facilitating backbone parameter changes, and a soft bond in the proline ring, enabling ring topology changes in proline mutations. These elements also have utility for core hopping and macrocycle studies in computer-aided drug design. This new strategy shows slight improvements over an alternative side chain perturbation strategy for a set T4 lysozyme mutations lacking proline and glycine, and yields good agreement with experiment for a set of T4 lysozyme proline and glycine mutations not previously studied. To our knowledge this is the first report comparing alchemical predictions of proline mutations with experiment. With this strategy in hand, alchemical methods now have access to the full palette of amino acid mutations.},
  archive      = {J_JCC},
  author       = {Ryan L. Hayes and Charles L. Brooks},
  doi          = {10.1002/jcc.26525},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1088-1094},
  shortjournal = {J. Comput. Chem.},
  title        = {A strategy for proline and glycine mutations to proteins with alchemical free energy calculations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Algorithm to minimize MPI communications in the parallelized
fast multipole method combined with molecular dynamics calculations.
<em>JCC</em>, <em>42</em>(15), 1073–1087. (<a
href="https://doi.org/10.1002/jcc.26524">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the era of exascale supercomputers, large-scale, and long-time molecular dynamics (MD) calculations are expected to make breakthroughs in various fields of science and technology. Here, we propose a new algorithm to improve the parallelization performance of message passing interface (MPI)-communication in the MPI-parallelized fast multipole method (FMM) combined with MD calculations under three-dimensional periodic boundary conditions. Our approach enables a drastic reduction in the amount of communication data, including the atomic coordinates and multipole coefficients, both of which are required to calculate the electrostatic interaction by using the FMM. In communications of multipole coefficients, the reduction rate of communication data in the new algorithm relative to the amount of data in the conventional one increases as both the number of FMM levels and the number of MPI processes increase. The aforementioned rate increase could exceed 50\% as the number of MPI processes becomes larger for very large systems. The proposed algorithm, named the minimum-transferred data (MTD) method, should enable large-scale and long-time MD calculations to be calculated efficiently, under the condition of massive MPI-parallelization on exascale supercomputers.},
  archive      = {J_JCC},
  author       = {Yoshimichi Andoh and Shin-ichi Ichikawa and Tatsuya Sakashita and Noriyuki Yoshii and Susumu Okazaki},
  doi          = {10.1002/jcc.26524},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1073-1087},
  shortjournal = {J. Comput. Chem.},
  title        = {Algorithm to minimize MPI communications in the parallelized fast multipole method combined with molecular dynamics calculations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). BSSE-corrected consistent gaussian basis sets of triple-zeta
valence with polarization quality of the sixth period for solid-state
calculations. <em>JCC</em>, <em>42</em>(15), 1064–1072. (<a
href="https://doi.org/10.1002/jcc.26521">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Consistent basis sets of triple-zeta valence with polarization quality for the elements Cs-Po were derived for periodic quantum-chemical solid-state calculations. They are an extension of the pob-TZVP-rev2 [Vilela Oliveira, D.; Laun, J.; Peintinger, M. F. and Bredow, T., J. Comput. Chem., 2019, 40 (27), 2364–2376] basis sets and are based on the fully relativistic effective core potentials (ECPs) of the Stuttgart/Cologne group and on the def2-TZVP valence basis of the Ahlrichs group. The basis sets are constructed to minimize the basis set superposition error (BSSE) in crystalline systems. The contraction scheme, the orbital exponents, and contraction coefficients were optimized in order to ensure robust and stable self-consistent-field (SCF) convergence for a set of compounds and metals. For the applied PW1PW hybrid functional, the average deviations of the calculated lattice constants from experimental references are smaller with pob-TZVP-rev2 than with standard basis sets available from the CRYSTAL basis set database.},
  archive      = {J_JCC},
  author       = {Joachim Laun and Thomas Bredow},
  doi          = {10.1002/jcc.26521},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1064-1072},
  shortjournal = {J. Comput. Chem.},
  title        = {BSSE-corrected consistent gaussian basis sets of triple-zeta valence with polarization quality of the sixth period for solid-state calculations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A combined monte carlo/DFT approach to simulate UV-vis
spectra of molecules and aggregates: Merocyanine dyes as a case study.
<em>JCC</em>, <em>42</em>(15), 1054–1063. (<a
href="https://doi.org/10.1002/jcc.26505">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The combination of a Monte Carlo (MC) sampling of the configurational space with time dependent-density functional theory (TD-DFT) to estimate vertical excitations energies has been applied to compute the absorption spectra of a family of merocyanine dyes in both their monomeric and dimeric forms. These results have been compared to those obtained using a static DFT/TD-DFT approach as well as to the available experimental spectra. Though suffering of the limitations related to the use of DFT and TD-DFT for this type of systems, our data clearly show that the classical MC sampling provides a suitable alternative to classical molecular dynamics to explore the structural flexibility of these donor-acceptor (D-π-A) chromophores enabling a realistic description of the potential energy surface of both their monomers and aggregates (here dimers) and thus of their spectra. Overall, the combination of MC sampling with quantum mechanics (TD-DFT) calculations, carried out in implicit dioxane solvent on random snapshots, provides a workable compromise to solve the combined challenge of accuracy and time-consuming problem not only for merocyanines momers, but also for their dimers, up to now less investigated. Indeed, the simulated absorption spectra fairly agree with the experimental ones, suggesting the general reliability of the method.},
  archive      = {J_JCC},
  author       = {Bernardino Tirri and Gloria Mazzone and Alistar Ottochian and Jerôme Gomar and Umberto Raucci and Carlo Adamo and Ilaria Ciofini},
  doi          = {10.1002/jcc.26505},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1054-1063},
  shortjournal = {J. Comput. Chem.},
  title        = {A combined monte Carlo/DFT approach to simulate UV-vis spectra of molecules and aggregates: Merocyanine dyes as a case study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Evaluation of replica exchange with repulsive scaling
approach for docking glycosaminoglycans. <em>JCC</em>, <em>42</em>(15),
1040–1053. (<a href="https://doi.org/10.1002/jcc.26496">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glycosaminoglycans (GAGs), long linear periodic anionic polysaccharides, are key molecules in the extracellular matrix (ECM). Therefore, deciphering their role in the biologically relevant context is important for fundamental understanding of the processes ongoing in ECM and for establishing new strategies in the regenerative medicine. Although GAGs represent a number of computational challenges, molecular docking is a powerful tool for analysis of their interactions. Despite the recent development of GAG-specific docking approaches, there is plenty of room for improvement. Here, replica exchange molecular dynamics with repulsive scaling (REMD-RS) recently proved to be a successful approach for protein–protein complexes, was applied to dock GAGs. In this method, effective pairwise radii are increased in different Hamiltonian replicas. REMD-RS is shown to be an attractive alternative to classical docking approaches for GAGs. This work contributes to setting up of GAG-specific computational protocols and provides new insights into the nature of these biological systems.},
  archive      = {J_JCC},
  author       = {Martyna Maszota-Zieleniak and Mateusz Marcisz and Małgorzata M. Kogut and Till Siebenmorgen and Martin Zacharias and Sergey A. Samsonov},
  doi          = {10.1002/jcc.26496},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1040-1053},
  shortjournal = {J. Comput. Chem.},
  title        = {Evaluation of replica exchange with repulsive scaling approach for docking glycosaminoglycans},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). LIMONADA: A database dedicated to the simulation of
biological membranes. <em>JCC</em>, <em>42</em>(14), 1028–1033. (<a
href="https://doi.org/10.1002/jcc.26511">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular membranes are composed of a wide diversity of lipid species in varying proportions and these compositions are representative of the organism, cellular type and organelle to which they belong. Because models of these molecular systems simulated by MD steadily gain in size and complexity, they are increasingly representative of specific compositions and behaviors of biological membranes. Due to the number of lipid species involved, of force fields and topologies and because of the complexity of membrane objects that have been simulated, LIMONADA has been developed as an open database allowing to handle the various aspects of lipid membrane simulation. LIMONADA presents published membrane patches with their simulation files and the cellular membrane it models. Their compositions are then detailed based on the lipid identification from LIPID MAPS database plus the lipid topologies and the force field used. LIMONADA is freely accessible on the web athttps://limonada.univ-reims.fr/.},
  archive      = {J_JCC},
  author       = {Jean-Marc Crowet and Sébastien Buchoux and Nicolas Belloy and Catherine Sarazin and Laurence Lins and Manuel Dauchez},
  doi          = {10.1002/jcc.26511},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {1028-1033},
  shortjournal = {J. Comput. Chem.},
  title        = {LIMONADA: A database dedicated to the simulation of biological membranes},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Non long-range corrected density functionals incorrectly
describe the intensity of the c?h stretching band in polycyclic aromatic
hydrocarbons. <em>JCC</em>, <em>42</em>(14), 1018–1027. (<a
href="https://doi.org/10.1002/jcc.26520">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a comprehensive study of the most relevant numerical aspects influencing frequencies and intensities in the infrared spectrum of isolated polycyclic aromatic hydrocarbons (PAHs) regarding the overestimate of the IR CH-stretching bands. We use naphthalene as benchmark and show the validity of our results to different members of the PAH family. Our analysis relies on widely employed density functional theory methods and second-order vibrational perturbational theory for the computation of vibrational eigenstates. We have focused on the elucidation of the origin of the systematic overestimate of the intensities in the CH-stretching region. To rule out nonfundamental numerical errors, we have initially considered the influence of the electronic basis set and various other parameters on the different stages of the vibrational analysis. In a second stage, we have benchmarked the results of different density functional theory functionals with respect to the aforementioned overestimate taken as the ratio between the most prominent features of the spectrum, the CH-bending and the CH-stretching bands. Our results unambiguously indicate that the long-range correction plays a major role in this spurious numerical issue. More specifically, this phenomenon is due to an incorrect description of the charge distribution (and hence dipole) within the symmetrically relevant CH bonds. Long-range correction specifically remedies this issue. It improves the description of the intensities in the stretching region while at the same time it does not perturb significantly the rest of the spectrum. With respect to the frequencies, we have observed an overall improvement when compared to noncorrected functionals.},
  archive      = {J_JCC},
  author       = {Hugo Geindre and Abdul-Rahman Allouche and Daniel Peláez},
  doi          = {10.1002/jcc.26520},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {1018-1027},
  shortjournal = {J. Comput. Chem.},
  title        = {Non long-range corrected density functionals incorrectly describe the intensity of the C?H stretching band in polycyclic aromatic hydrocarbons},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ClassicalGSG: Prediction of log p using classical molecular
force fields and geometric scattering for graphs. <em>JCC</em>,
<em>42</em>(14), 1006–1017. (<a
href="https://doi.org/10.1002/jcc.26519">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This work examines methods for predicting the partition coefficient (logP) for a dataset of small molecules. Here, we use atomic attributes such as radius and partial charge, which are typically used as force field parameters in classical molecular dynamics simulations. These atomic attributes are transformed into index-invariant molecular features using a recently developed method called geometric scattering for graphs (GSG). We call this approach “ClassicalGSG” and examine its performance under a broad range of conditions and hyperparameters. We train ClassicalGSG logPpredictors with neural networks using 10,722 molecules from the OpenChem dataset and apply them to predict the logPvalues from four independent test sets. The ClassicalGSG method&#39;s performance is compared to a baseline model that employs graph convolutional networks. Our results show that the best prediction accuracies are obtained using atomic attributes generated with the CHARMM generalized force field and 2D molecular structures.},
  archive      = {J_JCC},
  author       = {Nazanin Donyapour and Matthew Hirn and Alex Dickson},
  doi          = {10.1002/jcc.26519},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {1006-1017},
  shortjournal = {J. Comput. Chem.},
  title        = {ClassicalGSG: Prediction of log p using classical molecular force fields and geometric scattering for graphs},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comparison of many-particle representations for selected-CI
i: A tree based approach. <em>JCC</em>, <em>42</em>(14), 982–1005. (<a
href="https://doi.org/10.1002/jcc.26518">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The full configuration interaction (FCI) method is only applicable to small molecules with few electrons in moderate size basis sets. One of the main alternatives to obtain approximate FCI energies for bigger molecules and larger basis sets is selected CI. However, due to: (a) the lack of a well-defined structure in a selected CI Hamiltonian, (b) the potentially large number of electrons together with c) potentially large orbital spaces, a computationally and memory efficient algorithm is difficult to construct. In the present series of papers, we describe our attempts to address these issues by exploring tree-based approaches. At the same time, we devote special attention to the issue of obtaining eigenfunctions of the total spin squared operator since this is of particular importance in tackling magnetic properties of complex open shell systems. Dedicated algorithms are designed to tackle the CI problem in terms of determinant, configuration (CFG) and configuration state function many-particle bases by effective use of the tree representation. In this paper we describe the underlying logic of our algorithm design and discuss the advantages and disadvantages of the different many particle bases. We demonstrate by the use of small examples how the use of the tree simplifies many key algorithms required for the design of an efficient selected CI program. Our selected CI algorithm, called the iterative configuration expansion, is presented in the penultimate part. Finally, we discuss the limitations and scaling characteristics of the present approach.},
  archive      = {J_JCC},
  author       = {Vijay Gopal Chilkuri and Frank Neese},
  doi          = {10.1002/jcc.26518},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {982-1005},
  shortjournal = {J. Comput. Chem.},
  title        = {Comparison of many-particle representations for selected-CI i: A tree based approach},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Assessing challenging intra- and inter-molecular
charge-transfer excitations energies with double-hybrid density
functionals. <em>JCC</em>, <em>42</em>(14), 970–981. (<a
href="https://doi.org/10.1002/jcc.26517">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We investigate the performance of a set of recently introduced range-separated double-hybrid functionals, namelyωB2-PLYP,ωB2GP-PLYP, RSX-0DH, and RSX-QIDH models for hard-to-calculate excitation energies. We compare with the parent (B2-PLYP, B2GP-PLYP, PBE0-DH, and PBE-QIDH) and other (DSD-PBEP86) double-hybrid models as well as with some of the most widely employed hybrid functionals (B3LYP, PBE0, M06-2X, andωB97X). For this purpose, we select a number of medium-sized intra- and inter-molecular charge-transfer excitations, which are known to be challenging to calculate using time-dependent density-functional theory (TD-DFT) and for which accurate reference values are available. We assess whether the high accuracy shown by the newest double-hybrid models is also confirmed for those cases too. We find that asymptotically corrected double-hybrid models yield a superior performance, especially for the inter-molecular charge-transfer excitation energies, as compared to standard double-hybrid models. Overall, the PBE-QIDH and its corresponding range-separated RSX-QIDH functional are recommended for general-purpose TD-DFT applications, depending on whether long-range effects are expected to play a significant role.},
  archive      = {J_JCC},
  author       = {Éric Brémond and Alistar Ottochian and Ángel José Pérez-Jiménez and Ilaria Ciofini and Giovanni Scalmani and Michael J. Frisch and Juan Carlos Sancho-García and Carlo Adamo},
  doi          = {10.1002/jcc.26517},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {970-981},
  shortjournal = {J. Comput. Chem.},
  title        = {Assessing challenging intra- and inter-molecular charge-transfer excitations energies with double-hybrid density functionals},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Generalized–ensemble method study: A helix-mimetic compound
inhibits protein–protein interaction by long-range and short-range
intermolecular interactions. <em>JCC</em>, <em>42</em>(14), 956–969. (<a
href="https://doi.org/10.1002/jcc.26516">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A heterocyclic compound mS-11 is a helix-mimetic designed to inhibit binding of an intrinsic disordered protein neural restrictive silence factor/repressor element 1 silencing factor (NRSF/REST) to a receptor protein mSin3B. We apply a generalized ensemble method, multi-dimensional virtual-system coupled molecular dynamics developed by ourselves recently, to a system consisting of mS-11 and mSin3B, and obtain a thermally equilibrated distribution, which is comprised of the bound and unbound states extensively. The lowest free-energy position of mS-11 coincides with the NRSF/REST position in the experimentally-determined NRSF/REST-mSin3B complex. Importantly, the molecular orientation of mS-11 is ordering in a wide region around mSin3B. The resultant binding scenario is: When mS-11 is distant from the binding site of mSin3B, mS-11 descends the free-energy slope toward the binding site maintaining the molecular orientation to be advantageous for binding. Then, finally a long and flexible hydrophobic sidechain of mS-11 fits into the binding site, which is the lowest-free-energy complex structure inhibiting NRSF/REST binding to mSin3B.},
  archive      = {J_JCC},
  author       = {Junichi Higo and Hajime Takashima and Yoshifumi Fukunishi and Atsushi Yoshimori},
  doi          = {10.1002/jcc.26516},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {956-969},
  shortjournal = {J. Comput. Chem.},
  title        = {Generalized–ensemble method study: A helix-mimetic compound inhibits protein–protein interaction by long-range and short-range intermolecular interactions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PyRad: A software shell for simulating radiolysis with qball
package. <em>JCC</em>, <em>42</em>(13), 944–950. (<a
href="https://doi.org/10.1002/jcc.26509">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The assessment of the radiolytic stability of media is an important task in the fields of nuclear power engineering and radiochemistry. Such studies must be carried out in special laboratory conditions with the use of sources of ionizing radiation, which may increase personal doses of the staff. In addition, difficulties arise in studying the products of irradiated media. While it is impossible to abandon experiments to obtain reliable results in this area, computational methods of quantum chemistry can reduce the number of experiments and help understand the mechanisms of the reactions that occur during radiolysis. Here we would like to present a software shell of the Qb@ll program performing time-dependent density functional theory simulations of the radiolysis process.},
  archive      = {J_JCC},
  author       = {Nikolai Andreadi and Artem Mitrofanov and Artem Eliseev and Petr Matveev and Stepan Kalmykov and Vladimir Petrov},
  doi          = {10.1002/jcc.26509},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {944-950},
  shortjournal = {J. Comput. Chem.},
  title        = {PyRad: A software shell for simulating radiolysis with qball package},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MLLPA: A machine learning-assisted python module to study
phase-specific events in lipid membranes. <em>JCC</em>, <em>42</em>(13),
930–943. (<a href="https://doi.org/10.1002/jcc.26508">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine Learning-assisted Lipid Phase Analysis (MLLPA) is a new Python 3 module developed to analyze phase domains in a lipid membrane based on lipid molecular states. Reading standard simulation coordinate and trajectory files, the software first analyze the phase composition of the lipid membrane by using machine learning tools to label each individual molecules with respect to their state, and then decompose the simulation box using Voronoi tessellations to analyze the local environment of all the molecules of interest. MLLPA is versatile as it can read from multiple format (e.g., GROMACS, LAMMPS) and from either all-atom (e.g., CHARMM36) or coarse-grain models (e.g.,Martini). It can also analyze multiple geometries of membranes (e.g., bilayers, vesicles). Finally, the software allows for training with more than two phases, allowing for multiple phase coexistence analysis.},
  archive      = {J_JCC},
  author       = {Vivien Walter and Céline Ruscher and Olivier Benzerara and Fabrice Thalmann},
  doi          = {10.1002/jcc.26508},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {930-943},
  shortjournal = {J. Comput. Chem.},
  title        = {MLLPA: A machine learning-assisted python module to study phase-specific events in lipid membranes},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Developing a variation of 3D-QSAR/MD method in drug design.
<em>JCC</em>, <em>42</em>(13), 917–929. (<a
href="https://doi.org/10.1002/jcc.26514">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In continuation of the previous reports on a combination of 3D-quantitative structure–activity relationships (QSAR) with computational molecular dynamics (MD) studies, a new variation of 3D-QSAR/MD method has been employed for drug-design as an alternative or supplementary for the typical experimental methods. The presented method is more cost-effective and less time-consuming than the previous methods and avoids several restrictions of experimental methods, such as validity estimation, and predictability. For this purpose, seven inhibitors for bromodomain (BRD)-containing protein, as an important protein in the development of different types of cancer and responsible for oncogenic rearrangements, have been selected to study of their interactions by docking and MD simulations using molecular mechanics/generalized born surface area (MM/GBSA) method. To build the proposed model, a common variant of 3D-QSAR methods, comparative molecular field analysis has been employed using a dataset of 100 MD-extracted ligand conformations and their corresponding MM/GBSA BRD4-binding energies. The results showed excellent predictability of the generated model for both the training set and test groups. Finally, two new inhibitors were selected among total 4000 designed derivatives (generated through evolutionary techniques) using the proposed 3D-QSAR-MD model. The potentials of these inhibitors were assessed by MD simulations, which showed the higher inhibitory of these compounds than the previous inhibitors. Therefore, this method showed high potentials for acceleration of the procedure of drug design and a basis for joining researchers in computational biology and pharmaceutical sciences.},
  archive      = {J_JCC},
  author       = {Hamed Haghshenas and Bita Kaviani and Monireh Firouzeh and Hossein Tavakol},
  doi          = {10.1002/jcc.26514},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {917-929},
  shortjournal = {J. Comput. Chem.},
  title        = {Developing a variation of 3D-QSAR/MD method in drug design},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The competition and cooperativity of hydrogen/halogen bond
and π-hole bond involving the heteronuclear ethylene analogues.
<em>JCC</em>, <em>42</em>(13), 908–916. (<a
href="https://doi.org/10.1002/jcc.26513">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The noncovalent interactions involving heteronuclear ethylene analogues H2CEH2(E = Si, Ge and Sn) have been studied by the Møller–Plesset perturbation theory to investigate the competition and cooperativity between the hydrogen/halogen bond and π-hole bond. H2CEH2has a dual role of being a Lewis base and acid with the region of π-electron accumulation above the carbon atom and the region of π-electron depletion (π-hole) above the E atom to participate in the NCX···CE (X = H and Cl) hydrogen/halogen bond and CE···NCY (Y = H, Cl, Li and Na) π-hole bond, respectively. When HCN/ClCN interacts with H2CEH2by two sites, the strength of hydrogen bond/halogen bond is stronger than that of π-hole bond. The π-hole bond becomes obviously stronger when the metal substituent of YCN (Y = Li and Na) interacting with H2CEH2, showing the character of partial covalent, its strength is much greater than that of hydrogen/halogen bond. In the ternary complexes, both hydrogen/halogen bond and π-hole bond are simultaneously strengthened compared to those in the binary complexes, especially in the systems containing alkali metal.},
  archive      = {J_JCC},
  author       = {Shaojie Shen and Xinyue Jing and Xueying Zhang and Xiaoyan Li and Yanli Zeng},
  doi          = {10.1002/jcc.26513},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {908-916},
  shortjournal = {J. Comput. Chem.},
  title        = {The competition and cooperativity of hydrogen/halogen bond and π-hole bond involving the heteronuclear ethylene analogues},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Drug repositioning to target NSP15 protein on SARS-CoV-2 as
possible COVID-19 treatment. <em>JCC</em>, <em>42</em>(13), 897–907. (<a
href="https://doi.org/10.1002/jcc.26512">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SARS-CoV and SARS-CoV-2 belong to the subfamily Coronaviridae and infect humans, they are constituted by four structural proteins: Spike glycoprotein (S), membrane (M), envelope (E) and nucleocapsid (N), and nonstructural proteins, such as Nsp15 protein which is exclusively present on nidoviruses and is absent in other RNA viruses, making it an ideal target in the field of drug design. A virtual screening strategy to search for potential drugs was proposed, using molecular docking to explore a library of approved drugs available in the DrugBank database in order to identify possible NSP15 inhibitors to treat Covid19 disease. We found from the docking analysis that the antiviral drugs: Paritaprevir and Elbasvir, currently both approved for hepatitis C treatment which showed some of the lowest free binding energy values were considered as repositioning drugs to combat SARS-CoV-2. Furthermore, molecular dynamics simulations of the Apo and Holo-Nsp15 systems were performed in order to get insights about the stability of these protein-ligand complexes.},
  archive      = {J_JCC},
  author       = {Yudibeth Sixto-López and Marlet Martínez-Archundia},
  doi          = {10.1002/jcc.26512},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {897-907},
  shortjournal = {J. Comput. Chem.},
  title        = {Drug repositioning to target NSP15 protein on SARS-CoV-2 as possible COVID-19 treatment},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Judicious design functionalized 3D-COF to enhance CO2
adsorption and separation. <em>JCC</em>, <em>42</em>(13), 888–896. (<a
href="https://doi.org/10.1002/jcc.26510">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effects of functional groups (includingOH,OCH3,NH2,CH2NH2,COOH,SO3H,OCO(CH2)2COOH(E-COOH), and(CH2)4COOH(c-COOH)) in 3D covalent organic frameworks (3D-COFs) on CO2adsorption and separation are investigated by grand canonical Monte Carlo (GCMC) simulations and density functional theory calculations. The results indicate that interaction between CO2and the framework is the main factor for determining CO2uptakes at low pressure, while pore size becomes the decisive factor at high pressure. The binding energy of CO2with functionalized linker is correlated to CO2uptake at 0.3 bar and 298 K on 3D-COF-1, suggesting functional groups play a key role in CO2capture in microporous 3D-COFs. Moreover, CO2selectivity over CH4, N2, and H2can be significantly enhanced by functionalization, whereCH2NH2,COOH,SO3H, andE-COOH enhance CO2adsorption more effectively at 1 bar. Among them,SO3H is the most promising functional group in 3D-COFs for CO2separation.},
  archive      = {J_JCC},
  author       = {Fang Yuan and Zhifang Yang and Xiaoying Zhang and Cuiyan Tong and Godefroid Gahungu and Wenliang Li and Jingping Zhang},
  doi          = {10.1002/jcc.26510},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {888-896},
  shortjournal = {J. Comput. Chem.},
  title        = {Judicious design functionalized 3D-COF to enhance CO2 adsorption and separation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The explicit role of electron exchange in the hydrogen
bonded molecular complexes. <em>JCC</em>, <em>42</em>(12), 870–882. (<a
href="https://doi.org/10.1002/jcc.26507">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We applied a set of advanced bonding descriptors to establish the hidden electron density features and binding energy characteristics of intermolecular DH∙∙∙A hydrogen bonds (OH∙∙∙O, NH∙∙∙O and SH∙∙∙O) in 150 isolated and solvated molecular complexes. The exchange-correlation and Pauli potentials as well as corresponding local one-electron forces allowed us to explicitly ascertain how electron exchange defines the bonding picture in the proximity of the H-bond critical point. The electron density features of DH∙∙∙A interaction are governed by alterations in the electron localization in the H-bond region displaying itself in the exchange hole. At that, they do not depend on the variations in the exchange hole mobility. The electrostatic interaction mainly defines the energy of H-bonds of different types, whereas the strengthening/weakening of H-bonds in complexes with varying substituents depends on the barrier height of the exchange potential near the bond critical point. Energy variations between H-bonds in isolated and solvated systems are also caused the electron exchange peculiarities as follows from the corresponding potential and the interacting quantum atom analyses complemented by electron delocalization index calculations. Our approach is based on the bonding descriptors associated with the characteristics of the observable electron density and can be recommended for in-depth studies of non-covalent bonding.},
  archive      = {J_JCC},
  author       = {Elena O. Levina and Maria G. Khrenova and Vladimir G. Tsirelson},
  doi          = {10.1002/jcc.26507},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {870-882},
  shortjournal = {J. Comput. Chem.},
  title        = {The explicit role of electron exchange in the hydrogen bonded molecular complexes},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Efficient mesh refinement for the poisson-boltzmann equation
with boundary elements. <em>JCC</em>, <em>42</em>(12), 855–869. (<a
href="https://doi.org/10.1002/jcc.26506">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Poisson-Boltzmann equation is a widely used model to study electrostatics in molecular solvation. Its numerical solution using a boundary integral formulation requires a mesh on the molecular surface only, yielding accurate representations of the solute, which is usually a complicated geometry. Here, we utilize adjoint-based analyses to form two goal-oriented error estimates that allow us to determine the contribution of each discretization element (panel) to the numerical error in the solvation free energy. This information is useful to identify high-error panels to then refine them adaptively to find optimal surface meshes. We present results for spheres and real molecular geometries, and see that elements with large error tend to be in regions where there is a high electrostatic potential. We also find that even though both estimates predict different total errors, they have similar performance as part of an adaptive mesh refinement scheme. Our test cases suggest that the adaptive mesh refinement scheme is very effective, as we are able to reduce the error one order of magnitude by increasing the mesh size less than 20\% and come out to be more efficient than uniform refinement when computing error estimations. This result sets the basis toward efficient automatic mesh refinement schemes that produce optimal meshes for solvation energy calculations.},
  archive      = {J_JCC},
  author       = {Vicente Ramm and Jehanzeb H. Chaudhry and Christopher D. Cooper},
  doi          = {10.1002/jcc.26506},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {855-869},
  shortjournal = {J. Comput. Chem.},
  title        = {Efficient mesh refinement for the poisson-boltzmann equation with boundary elements},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). New insights in chemical reactivity from quantum chemical
topology. <em>JCC</em>, <em>42</em>(12), 840–854. (<a
href="https://doi.org/10.1002/jcc.26504">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Based on the quantum chemical topology of the modified electron localization function ELFx, an efficient and robust mechanistic methodology designed to identify the favorable reaction pathway between two reactants is proposed. We first recall and reshape how the supermolecular interaction energy can be evaluated from only three distinct terms, namely the intermolecular coulomb energy, the intermolecular exchange-correlation energy and the intramolecular energies of reactants. Thereafter, we show that the reactivity between the reactants is driven by the first-order variation in the coulomb intermolecular energy defined in terms of the response to changes in the number of electrons. Illustrative examples with the formation of the dative bond B-N involved in the BH3NH3molecule and the typical formation of the hydrogen bond in the canonical water dimer are presented. For these selected systems, our approach unveils a noticeable mimicking of Edualonto the DFT intermolecular interaction energy surface calculated between the both reactants. An automated reaction-path algorithm aimed to determine the most favorable relative orientations when the two molecules approach each other is also outlined.},
  archive      = {J_JCC},
  author       = {Johanna Klein and Paul Fleurat-Lessard and Julien Pilmé},
  doi          = {10.1002/jcc.26504},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {840-854},
  shortjournal = {J. Comput. Chem.},
  title        = {New insights in chemical reactivity from quantum chemical topology},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Efficient hierarchical models for reactivity of organic
layers on semiconductor surfaces. <em>JCC</em>, <em>42</em>(12),
827–839. (<a href="https://doi.org/10.1002/jcc.26503">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational modeling of organic interface formation on semiconductors poses a challenge to a density functional theory-based description due to structural and chemical complexity. A hierarchical approach is presented, where parts of the interface are successively removed in order to increase computational efficiency while maintaining the necessary accuracy. First, a benchmark is performed to probe the validity of this approach for three model reactions and five dispersion corrected density functionals. Reaction energies are generally well reproduced by generalized gradient approximation-type functionals but accurate reaction barriers require the use of hybrid functionals. Best performance is found for the model system that does not explicitly consider the substrate but includes its templating effects. Finally, this efficient model is used to provide coverage dependent reaction energies and suggest synthetic principles for the prevention of unwanted growth termination reactions for organic layers on semiconductor surfaces.},
  archive      = {J_JCC},
  author       = {Jan-Niclas Luy and Mahlet Molla and Lisa Pecher and Ralf Tonner},
  doi          = {10.1002/jcc.26503},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {827-839},
  shortjournal = {J. Comput. Chem.},
  title        = {Efficient hierarchical models for reactivity of organic layers on semiconductor surfaces},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A computational study of the reaction mechanism involved in
the fast cleavage of an unconstrained amide bond assisted by an amine
intramolecular nucleophilic attack. <em>JCC</em>, <em>42</em>(12),
818–826. (<a href="https://doi.org/10.1002/jcc.26501">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the present work, the fast amide bond cleavage of [3-((1R,5S,7s)-3-azabicyclo[3.3.1]nonane-7-carbonyl)-3-azabicyclo[3.3.1]nonane-7-carboxylic acid (bi-ATDO)], through an intramolecular nucleophilic attack of an amine group is evaluated. First, six possible peptide bond cleavage mechanisms, two of them including a water molecule, are described at the ωB97XD/6–311 + G(d,p)//MP2/6–311 + G(d,p) level of theory. The reaction consisting of an intramolecular nitrogen nucleophilic attack followed by a proton transfer and the amide bond cleavage is determined as the most favorable mechanism. The activation free energy computed for the latter is 20.5 kcal mol−1, which agrees with the reported experimental result of 24.8 kcal mol−1. Inclusion of a water molecule to assist the first step of the reaction results in an activation free energy increase of about 17 kcal mol−1. All the steps in the most favorable mechanism are studied more in detail employing intrinsic reaction coordinate as well as the reaction force and reaction electronic flux analysis.},
  archive      = {J_JCC},
  author       = {Sebastián Adolfo Cuesta and Luis Rincón and F. Javier Torres and Vladimir Rodríguez and José Ramón Mora},
  doi          = {10.1002/jcc.26501},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {818-826},
  shortjournal = {J. Comput. Chem.},
  title        = {A computational study of the reaction mechanism involved in the fast cleavage of an unconstrained amide bond assisted by an amine intramolecular nucleophilic attack},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Interstitial carbon defects in silicon. A quantum mechanical
characterization through the infrared and raman spectra. <em>JCC</em>,
<em>42</em>(12), 806–817. (<a
href="https://doi.org/10.1002/jcc.26500">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Infrared (IR) and Raman spectra of various interstitial carbon defects in silicon are computed at the quantum mechanical level by using anall electronGaussian type basis set, the hybrid B3LYP functional and the supercell approach, as implemented in the CRYSTAL code (Dovesi et al. J. Chem. Phys. 2020, 152, 204111). The list includes two 〈100〉 split interstitialIXYdefects, namelyICCandICSi, a couple of related defects that we indicate asIXIY, the so calledin its A and B form, as well as SiCiSi and CsCiCs, in which the interstitial carbon atom is twofold coordinated. The second undergoes a large relaxation, and the final configuration is close toICCCs. Geometries, relative stabilities, electronic, and vibrational properties are analysed. All these defects show characteristic features in their IR spectrum (above 730 cm−1), whereas the Raman spectrum is dominated, in most of the cases, by the pristine silicon peak at 530 cm−1, that hides the defect peaks.},
  archive      = {J_JCC},
  author       = {Alexander Platonenko and Francesco Silvio Gentile and Fabien Pascale and Philippe D&#39;Arco and Roberto Dovesi},
  doi          = {10.1002/jcc.26500},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {806-817},
  shortjournal = {J. Comput. Chem.},
  title        = {Interstitial carbon defects in silicon. a quantum mechanical characterization through the infrared and raman spectra},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deciphering excited state properties utilizing algebraic
diagrammatic construction schemes of decreasing order. <em>JCC</em>,
<em>42</em>(11), 793–800. (<a
href="https://doi.org/10.1002/jcc.26499">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Excited state properties are difficult to trace back to the common molecular orbital picture when the excited state wavefunction is a linear combination of two or more Slater determinants. Here, a theoretical methodology is introduced based on the algebraic diagrammatic construction scheme for the polarization propagator (ADC(n)) that allows to make this connection and to eventually derive structure–function relationships. The usefulness of this approach is demonstrated by an analysis of the transition dipole moments of the low-lying1B3uand2B3ustates of anthracene and (1,4,5,8)-tetraazaanthracene.},
  archive      = {J_JCC},
  author       = {Marvin Hoffmann and Andreas Dreuw},
  doi          = {10.1002/jcc.26499},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {793-800},
  shortjournal = {J. Comput. Chem.},
  title        = {Deciphering excited state properties utilizing algebraic diagrammatic construction schemes of decreasing order},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). To the fast calculation of the solvation free energy.
Combining expanded ensembles with L2MC. <em>JCC</em>, <em>42</em>(11),
787–792. (<a href="https://doi.org/10.1002/jcc.26498">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A more efficient version of the Expanded Ensembles method for calculation of free energy in molecular-mechanical simulations is proposed. The method is based on the Horowitz L2MC approach to accelerate movement along the “alchemical” coordinate. It is possible to achieve the same efficiency of the algorithm both with the optimal number of “windows” and with a larger number of them compared to the original algorithm. Since the optimal number of windows is unknowna priory, the proposed algorithm is more robust than the traditional one. We can choose the number of windows in excess and do not worry about the loss of efficiency. We illustrate the method&#39;s efficiency with the computation of the hydration free energies of pyridine and water.},
  archive      = {J_JCC},
  author       = {Alexei Nikitin and Vladislava Milchevskaya and Alexander Lyubartsev},
  doi          = {10.1002/jcc.26498},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {787-792},
  shortjournal = {J. Comput. Chem.},
  title        = {To the fast calculation of the solvation free energy. combining expanded ensembles with L2MC},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Multipolar electrostatics for hairpin and pseudoknots in
RNA: Improving the accuracy of force field potential energy function.
<em>JCC</em>, <em>42</em>(11), 771–786. (<a
href="https://doi.org/10.1002/jcc.26497">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular dynamics (MD) simulations that rely on force field methods has been widely used to explore the structure and function of RNAs. However, the current commonly used force fields are limited by the electrostatic description offered by atomic charge, dipole and at most quadrupole moments, failing to capture the anisotropic picture of electronic features. Actually, the distribution of electrons around atomic nuclei is not spherically symmetric but is geometry dependent. A multipolar electrostatic model based on high rank multipole moments is described in this work, which allows us to combine polarizability and anisotropy of electron density. RNA secondary structure was taken as a research system, and its substructures including stem, loops (hairpin loop, bulge loop, internal loop, and multi-branch loop), and pseudoknots (H-type and K-type) were investigated, respectively, as well as the hairpin. First, the atom–atom electrostatic properties derived from one chain of a duplex RNA 2MVY in our previous work (Ref. 58) were measured by the pilot RNA systems of hairpin, hairpin loop, stem, and H-type pseudoknot, respectively. The prediction results were not satisfactory. Consequently, to obtain a general set of electrostatic parameters for RNA force fields, the convergence behavior of the atom–atom electrostatic interactions in the pilot RNA systems was explored using high rank atomic multipole moments. The pilot RNA systems were cut into four types of different-sized molecular fragments, and the single nucleotide fragment and nucleotide-paired fragment proved to be the most reasonable systems for base-unpairing regions and base-pairing regions to investigate the convergence behavior of all types of atom–atom electrostatic interactions, respectively. Transferability of the electrostatic properties drawn from the pilot RNA systems to the corresponding test systems was also investigated. Furthermore, the convergence behavior of atomic electrostatic interactions in other substructures including bulge loop, internal loop, multi-branch loop, and K-type pseudoknot was expected to be modeled via the hairpin.},
  archive      = {J_JCC},
  author       = {Yongna Yuan and Shaowei Fu and Dongxu Huo and Wei Su and Ruisheng Zhang and Jiaxuan Wei},
  doi          = {10.1002/jcc.26497},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {771-786},
  shortjournal = {J. Comput. Chem.},
  title        = {Multipolar electrostatics for hairpin and pseudoknots in RNA: Improving the accuracy of force field potential energy function},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Simple position and orientation preconditioning scheme for
minimum energy path calculations. <em>JCC</em>, <em>42</em>(11),
761–770. (<a href="https://doi.org/10.1002/jcc.26495">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Minimum-energy path (MEP) calculations, such as those typified by the nudged elastic band method, require input of reactant and product molecular configurations at initialization. In the case of reactions involving more than one molecule, generating initial reactant and product configurations requires careful consideration of the relative position and orientations of the reactive molecules in order to ensure that the resulting MEP calculation proceeds without converging on an alternative reaction-path, and without requiring excessive numbers of optimization iterations; as such, this initial system set-up is most commonly performed “by hand,” with an expert user arranging reactive molecules in space to ensure that the following MEP calculation runs smoothly. In this Article, we introduce a simple preconditioning scheme which replaces this labor-intensive, human-knowledge-based step with an automated deterministic computational scheme. In our approach, initial reactant and product configurations are generated such that steric hindrance between reactive molecules is minimized in the reactant and product configurations, while also simultaneously requiring minimal structural differences between the reactants and products. The method is demonstrated using a benchmark test-set of&gt;3400 organic molecular reactions, where comparison of the reactant/product configurations generated using our approach compare very well to initial configurations which were generated on an ad hoc basis.},
  archive      = {J_JCC},
  author       = {Christopher Robertson and Scott Habershon},
  doi          = {10.1002/jcc.26495},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {761-770},
  shortjournal = {J. Comput. Chem.},
  title        = {Simple position and orientation preconditioning scheme for minimum energy path calculations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Toward efficient generation, correction, and properties
control of unique drug-like structures. <em>JCC</em>, <em>42</em>(11),
746–760. (<a href="https://doi.org/10.1002/jcc.26494">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efficient design and screening of the novel molecules is a major challenge in drug and material design. This paper focuses on a multi-stage pipeline, in which several deep neural network models are combined to map discrete molecular representations into continuous vector space to later generate from it new molecular structures with desired properties. Here, the Attention-based Sequence-to-Sequence model is added to “spellcheck” and correct generated structures, while the oversampling in the continuous space allows generating candidate structures with desired distribution for properties and molecular descriptors, even for a small reference datasets. We further use computer simulation to validate the desired properties in the numerical experiment. With the focus on the drug design, such a pipeline allows generating novel structures with a control of Synthetic Accessibility Score and a series of metrics that assess the drug-likeliness. Our code is available athttps://github.com/SoftServeInc/novel-molecule-generation.},
  archive      = {J_JCC},
  author       = {Maksym Druchok and Dzvenymyra Yarish and Oleksandr Gurbych and Mykola Maksymenko},
  doi          = {10.1002/jcc.26494},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {746-760},
  shortjournal = {J. Comput. Chem.},
  title        = {Toward efficient generation, correction, and properties control of unique drug-like structures},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Application of artificial neural network and global
optimization techniques for high throughput modeling of the crystal
structure of stannites and kesterites. <em>JCC</em>, <em>42</em>(11),
740–745. (<a href="https://doi.org/10.1002/jcc.26493">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study aims to apply artificial neural networks for the prediction of the lattice parameters of materials with stannite- and kesterite-type structure, and to compare the results of predictions with that obtained in the calculations exploiting the density functional theory. Crystallographic data for 49 compounds with stannite-type structure and for four compounds with the kesterite-type structure are found and, based on it, crystal structures are calculated using the density functional theory (DFT) method in a two-step relaxation procedure for all compounds. An multilayer Perceptron is constructed, which then is trained on gathered crystallographic data. Values predicted by a neural network (lattice parameters) are compared with experimental data and with results of DFT calculations. Moreover, a global optimization method (the Uspex code) is used to find potentially novel crystal structures for investigated chemical compositions. The results are discussed in the term of advantages and disadvantages of each method.},
  archive      = {J_JCC},
  author       = {Grzegorz Matyszczak and Krzysztof Zberecki},
  doi          = {10.1002/jcc.26493},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {740-745},
  shortjournal = {J. Comput. Chem.},
  title        = {Application of artificial neural network and global optimization techniques for high throughput modeling of the crystal structure of stannites and kesterites},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Cover image. <em>JCC</em>, <em>42</em>(10), C1. (<a
href="https://doi.org/10.1002/jcc.26515">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Lucas de Azevedo Santos and Teodorico C. Ramalho and Trevor A. Hamlin and F. Matthias Bickelhaupt},
  doi          = {10.1002/jcc.26515},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {C1},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Electrostatically embedded molecules-in-molecules approach
and its application to molecular clusters. <em>JCC</em>,
<em>42</em>(10), 719–734. (<a
href="https://doi.org/10.1002/jcc.26492">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report the application of our fragment-based quantum chemistry model MIM (Molecules-In-Molecules) with electrostatic embedding. The method is termed “EE-MIM (ElectrostaticallyEmbeddedMolecules-In-Molecules)” and accounts for the missing electrostatic interactions in the subsystems resulting from fragmentation. Point charges placed at the atomic positions are used to represent the interaction of each subsystem with the rest of the molecule with minimal increase in the computational cost. We have carefully calibrated this model on a range of different sizes of clusters containing up to 57 water molecules. The fragmentation methods have been applied with the goal of reproducing the unfragmented total energy at the MP2/6-311G(d,p) level. Comparative analysis has been carried out between MIM and EE-MIM to gauge the impact of electrostatic embedding. Performance of several different parameters such as the type of charge and levels of fragmentation are analyzed for the prediction of absolute energies. The use of background charges in subsystem calculations improves the performance of both one- and two-layer MIM while it is noticeably important in the case of one-layer MIM. Embedded charges for two-layer MIM are obtained from a full system calculation at the low-level. For one-layer MIM, in the absence of a full system calculation, two different types of embedded charges, namely, Geometry dependent (GD) and geometry independent (GI) charges, are used. A self-consistent procedure is employed to obtain GD charges. We have further tested our method on challenging charged systems with stronger intermolecular interactions, namely, protonated ammonia clusters (containing up to 30 ammonia molecules). The observations are similar to water clusters with improved performance using embedded charges. Overall, the performance of NPA charges as embedded charges is found to be the best.},
  archive      = {J_JCC},
  author       = {Vikrant Tripathy and Arjun Saha and Krishnan Raghavachari},
  doi          = {10.1002/jcc.26492},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {719-734},
  shortjournal = {J. Comput. Chem.},
  title        = {Electrostatically embedded molecules-in-molecules approach and its application to molecular clusters},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The CH···HC interaction in biphenyl is a delocalized,
molecular-wide and entirely non-classical interaction: Results from
FALDI analysis. <em>JCC</em>, <em>42</em>(10), 706–718. (<a
href="https://doi.org/10.1002/jcc.26491">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study we aim to determine the origin of the electron density describing a CH···HC interaction in planar and twisted conformers of biphenyl. In order to achieve this, the fragment, atomic, localized, delocalized, intra- and inter-atomic (FALDI) decomposition scheme was utilized to decompose the density in the inter-nuclear region between theortho-hydrogens in both conformers. Importantly, the structural integrity, hence also topological properties, were fully preserved as no &#39;artificial&#39; partitioning of molecules was implemented. FALDI-based qualitative and quantitative analysis revealed that the majority of electron density arises from two, non-classical and non-local effects: strong overlap oforthoCH σ-bonds, and long-range electron delocalization between the phenyl rings andorthocarbons and hydrogens. These effects resulted in a delocalized electron channel, that is, a density bridge or a bond path in a QTAIM terminology, linking the H-atoms in the planar conformer. The same effects and phenomena are present in both conformers of biphenyl. We show that the CH···HC interaction is a molecular-wide event due to large and long-range electron delocalization, and caution against approaches that investigate CH···HC interactions without fully taking into account the remainder of the molecule.},
  archive      = {J_JCC},
  author       = {Thomas G. Bates and Jurgens H. de Lange and Ignacy Cukrowski},
  doi          = {10.1002/jcc.26491},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {706-718},
  shortjournal = {J. Comput. Chem.},
  title        = {The CH···HC interaction in biphenyl is a delocalized, molecular-wide and entirely non-classical interaction: Results from FALDI analysis},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). New strontium titanate polymorphs under high pressure.
<em>JCC</em>, <em>42</em>(10), 699–705. (<a
href="https://doi.org/10.1002/jcc.26490">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report six new dynamically stable structures of SrTiO3at various pressures ranging from 0 to 200 GPa. These structures were found by exploring the enthalpy surface with the Minima Hopping structure prediction method. The potential energy surface was generated by a machine learned potential, the charge equilibration via neural network technique (CENT), based on an extensive training data set of highly diverse SrTiO3periodic and cluster structures. All our CENT structures were validated at the level of density functional theory. For our new structures, we performed phonon calculations and NVT molecular dynamics calculations to investigate their dynamical stability. Finally, X-ray diffraction patterns were simulated to help to identify our predicted structures in experiments.},
  archive      = {J_JCC},
  author       = {Ehsan Rahmatizad Khajehpasha and Stefan Goedecker and S. Alireza Ghasemi},
  doi          = {10.1002/jcc.26490},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {699-705},
  shortjournal = {J. Comput. Chem.},
  title        = {New strontium titanate polymorphs under high pressure},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Chalcogen bonds: Hierarchical ab initio benchmark and
density functional theory performance study. <em>JCC</em>,
<em>42</em>(10), 688–698. (<a
href="https://doi.org/10.1002/jcc.26489">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have performed a hierarchicalab initiobenchmark and DFT performance study of D2Ch•••A−chalcogen bonds (Ch = S, Se; D, A = F, Cl). Theab initiobenchmark study is based on a series of ZORA-relativistic quantum chemical methods [HF, MP2, CCSD, CCSD(T)], and all-electron relativistically contracted variants of Karlsruhe basis sets (ZORA-def2-SVP, ZORA-def2-TZVPP, ZORA-def2-QZVPP) with and without diffuse functions. The highest-level ZORA-CCSD(T)/ma-ZORA-def2-QZVPP counterpoise-corrected complexation energies (ΔECPC) are converged within 1.1–3.4 kcal mol−1and 1.5–3.1 kcal mol−1with respect to the method and basis set, respectively. Next, we used the ZORA-CCSD(T)/ma-ZORA-def2-QZVPP (ΔECPC) as reference data for analyzing the performance of 13 different ZORA-relativistic DFT approaches in combination with the Slater-type QZ4P basis set. We find that the three-best performing functionals are M06-2X, B3LYP, and M06, with mean absolute errors (MAE) of 4.1, 4.2, and 4.3 kcal mol−1, respectively. The MAE for BLYP-D3(BJ) and PBE amount to 8.5 and 9.3 kcal mol−1, respectively.},
  archive      = {J_JCC},
  author       = {Lucas de Azevedo Santos and Teodorico C. Ramalho and Trevor A. Hamlin and F. Matthias Bickelhaupt},
  doi          = {10.1002/jcc.26489},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {688-698},
  shortjournal = {J. Comput. Chem.},
  title        = {Chalcogen bonds: Hierarchical ab initio benchmark and density functional theory performance study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Challenging the electrostatic σ-hole picture of halogen
bonding using minimal models and the interacting quantum atoms approach.
<em>JCC</em>, <em>42</em>(10), 676–687. (<a
href="https://doi.org/10.1002/jcc.26488">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Among the different noncovalent interactions, halogen bonds have captured wide attention in the last years. Their stability has been rationalized in electrostatic terms by appealing to theσ-hole concept, a charge-depleted region that is able to interact favorably with electron rich moieties. This interpretation has been questioned, and in this work a set of anionic halogen model systems are used to shed some light on this issue. We use the interacting quantum atoms method, which provides an orbital invariant energy decomposition in which pure electrostatic terms are well isolated, and we complement our insights with the analysis of electrostatic potentials (ESPs) as well as with traditional descriptors of charge accumulation like the Laplacian of the electron density. The total electrostatic interaction between the interacting species is surprisingly destabilizing in many of the systems examined, demonstrating that althoughσ-holes might be qualitatively helpful, much care has to be taken in ascribing the stability of these systems to electrostatics. It is clearly shown that electron delocalization is essential to understand the stability of the complexes. The evolution of atomic charges as the aggregates forms reveals a charge transfer picture in which the central,σ-hole bearing halogen acts as a mere spectator. These systems may then be not far from engaging in a classical 3c-4e interaction. Since the presence of aσ-hole as characterized by the ESP mapped on a suitable molecular envelope isosurface does not guarantee attractive electrostatic interactions, we encourage to employ a wider perspective that takes into account the full charge distribution.},
  archive      = {J_JCC},
  author       = {Fernando Jiménez-Grávalos and Miguel Gallegos and Ángel Martín Pendás and Alexander S. Novikov},
  doi          = {10.1002/jcc.26488},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {676-687},
  shortjournal = {J. Comput. Chem.},
  title        = {Challenging the electrostatic σ-hole picture of halogen bonding using minimal models and the interacting quantum atoms approach},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Anisotropic numerical potentials for coarse-grained modeling
from high-speed multidimensional lookup table and interpolation
algorithms. <em>JCC</em>, <em>42</em>(10), 666–675. (<a
href="https://doi.org/10.1002/jcc.26487">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A high-speed numerical potential delivering computational performance comparable with complex coarse-grained analytic potentials makes available models that have greater degrees of physical and chemical accuracy. This opens the possibility of increased accuracy in classical molecular dynamics simulations of anisotropic systems. In this work, we report the development of a high-speed lookup table (LUT) of four-dimensional gridded data, that uses cubic B-spline interpolations to derive off grid values and their associated partial derivatives that are located between the known grid data points. The accuracy of the coarse-grained numerical potential using a LUT from uniaxial Gay–Berne (GB) potential produced array of values is within a 3\% and a 5\% margin of error respectively for the interpolation of the uniaxial GB potential and its partial derivatives. The numerical potential model and partial derivatives speedup is made competitive with the analytical potential by exploiting graphics processing units on board functionality. The capability of the numerical potential is demonstrated by comparing minimizations of a box of 500 naphthalene molecules. The minimizations using a full atomistic (NAMD/CHARMM force field), a biaxial GB and a numerical potential from a LUT using data from the CHARMM pair potential was done. The numerical potential model is significantly more accurate in its approximation of the atomistic local minimum configuration than is the biaxial GB analytical potential function. This demonstrates that using a numerical potential founded on a direct lookup of the atomistic potential landscape significantly improves coarse grain (CG) modeling of complex molecules, possibly paving the way for accurate anisotropic system CG modeling.},
  archive      = {J_JCC},
  author       = {Ananya Gangopadhyay and Simon Winberg and Kevin J. Naidoo},
  doi          = {10.1002/jcc.26487},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {666-675},
  shortjournal = {J. Comput. Chem.},
  title        = {Anisotropic numerical potentials for coarse-grained modeling from high-speed multidimensional lookup table and interpolation algorithms},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ISPECTRON: A simulation interface for linear and nonlinear
spectra with ab-initio quantum chemistry software. <em>JCC</em>,
<em>42</em>(9), 644–659. (<a
href="https://doi.org/10.1002/jcc.26485">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce iSPECTRON, a program that parses data from common quantum chemistry software (NWChem, OpenMolcas, Gaussian, Cobramm, etc.), produces the input files for the simulation of linear and nonlinear spectroscopy of molecules with the Spectron code, and analyzes the spectra with a broad range of tools. Vibronic spectra are expressed in term of the electronic eigenstates, obtained from quantum chemistry computations, and vibrational/bath effects are incorporated in the framework of the displaced harmonic oscillator model, where all required quantities are computed at the Franck-Condon point. The program capabilities are illustrated by simulating linear absorption, transient absorption and two dimensional electronic spectra of the pyrene molecule. Calculations at two levels of electronic structure theory, time-dependent density functional theory (with NWChem) and RASSCF/RASPT2 (with OpenMolcas) are presented and compared where possible. The iSPECTRON program is available online athttps://github.com/ispectrongit/iSPECTRON/and distributed open source under the terms of the Educational Community License version 2.0 (ECL 2.0).},
  archive      = {J_JCC},
  author       = {Francesco Segatta and Artur Nenov and Daniel R. Nascimento and Niranjan Govind and Shaul Mukamel and Marco Garavelli},
  doi          = {10.1002/jcc.26485},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {644-659},
  shortjournal = {J. Comput. Chem.},
  title        = {ISPECTRON: A simulation interface for linear and nonlinear spectra with ab-initio quantum chemistry software},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PorE: A code for deterministic and systematic analyses of
porosities. <em>JCC</em>, <em>42</em>(9), 630–643. (<a
href="https://doi.org/10.1002/jcc.26484">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate numerical calculations of porosities and related properties are of importance when analyzing metal–organic frameworks (MOFs). We presentporE, an open-source, general-purpose implementation to compute such properties and discuss all results regarding their sensitivity to numerical parameters. Our code combines the numerical efficiency ofFortranwith the user-friendliness ofPython. Three different approaches to calculate porosities are implemented inporE, and their advantages and drawbacks are discussed. In contrast to commonly used implementations, our approaches are entirely deterministic and do not require any stochastic averaging. In addition to the calculation of porosities,porEcan calculate pore size distributions and offers the possibility to analyze pore windows. The underlying approaches are outlined, and pore windows are discussed concerning their impact on the analyzed porosities. Comparisons with reference values aim for a clear differentiation between void and accessible porosities, which we provide for a small benchmark set consisting of eight MOFs. In addition, our approaches are used for a bigger benchmark set containing 370 MOFs, where we determine linear relationships within our approaches as well as to reference values. We show how these relationships can be used to derive corrections to a give porosity approach, minimizing its mean error. As a highlight we show how complex workflows can be designed with a few lines ofPythoncode usingporE.},
  archive      = {J_JCC},
  author       = {Kai Trepte and Sebastian Schwalbe},
  doi          = {10.1002/jcc.26484},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {630-643},
  shortjournal = {J. Comput. Chem.},
  title        = {PorE: A code for deterministic and systematic analyses of porosities},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Energy-based automatic determination of buffer region in the
divide-and-conquer second-order møller–plesset perturbation theory.
<em>JCC</em>, <em>42</em>(9), 620–629. (<a
href="https://doi.org/10.1002/jcc.26486">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the linear-scaling divide-and-conquer (DC) electronic structure method, each subsystem is calculated together with the neighboring buffer region, the size of which affects the energy error introduced by the fragmentation in the DC method. The DC self-consistent field calculation utilizes a scheme to automatically determine the appropriate buffer region that is as compact as possible for reducing the computational time while maintaining acceptable accuracy (J. Comput. Chem.2018,39, 909). To extend the automatic determination scheme of the buffer region to the DC second-order Møller–Plesset perturbation (MP2) calculation, a scheme for estimating the subsystem MP2 correlation energy contribution from each atom in the buffer region is proposed. The estimation is based on the atomic orbital Laplace MP2 formalism. Based on this, an automatic buffer determination scheme for the DC-MP2 calculation is constructed and its performance for several types of systems is assessed.},
  archive      = {J_JCC},
  author       = {Toshikazu Fujimori and Masato Kobayashi and Tetsuya Taketsugu},
  doi          = {10.1002/jcc.26486},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {620-629},
  shortjournal = {J. Comput. Chem.},
  title        = {Energy-based automatic determination of buffer region in the divide-and-conquer second-order Møller–Plesset perturbation theory},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Iodido equatorial ligands influence on the mechanism of
action of pt(IV) and pt(II) anti-cancer complexes: A DFT computational
study. <em>JCC</em>, <em>42</em>(9), 608–619. (<a
href="https://doi.org/10.1002/jcc.26483">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A detailed computational exploration of the most relevant steps of iodido Pt(IV) complexes reduction and Pt(II) drugs mechanism of action and eventual deactivation is presented here inspired by the recent findings on iodido Pt(II) complexes and surprising re-evaluation of their cytotoxic activity. Pt(II) and Pt(IV) model systems are investigated and compared with cisplatin and its Pt(IV) derivative. Both monodeprotonated ascorbic acid andl-cysteine are used as reducing agents in the inner-sphere reduction mechanism of Pt(IV) complexes. Aquation mechanism of iodido Pt(II) complexes, interaction with guanine and sulfur containing compounds and reaction with the model protein hen egg white lysozyme are explored, due to a detected different behavior with respect to classical platinum drugs. The outcomes of such exploration allow to shed light on the role that the increasedsoftcharacter together with bridging and leaving abilities of iodide over chloride could play in determining the cytotoxic profile of iodido Pt drugs.},
  archive      = {J_JCC},
  author       = {Stefano Scoditti and Vincenzo Vigna and Eslam Dabbish and Emilia Sicilia},
  doi          = {10.1002/jcc.26483},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {608-619},
  shortjournal = {J. Comput. Chem.},
  title        = {Iodido equatorial ligands influence on the mechanism of action of Pt(IV) and Pt(II) anti-cancer complexes: A DFT computational study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The nature of persistent interactions in two model β-grasp
proteins reveals the advantage of symmetry in stability. <em>JCC</em>,
<em>42</em>(9), 600–607. (<a
href="https://doi.org/10.1002/jcc.26477">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two proteins within the β-grasp superfamily, the B1-domain of protein G and the small archaeal modifier protein 1, were investigated to elucidate the key determinants of structural stability at the level of individual interactions. These symmetrical proteins both contain two β-hairpins which form a sheet flanked by a central α-helix. They were subjected to high temperature molecular dynamics simulations and the detailed behavior of each long-range interaction was characterized. The results revealed that in GB1 the most stable region was the C-terminal hairpin and in SAMP1 it was the opposite, the N-terminal hairpin. Experimental results for GB1 support this finding. In conclusion, it appears that the difference in the location and number of hydrophobic interactions dictate the differential stability which is accommodated due to structural symmetry of the β-grasp fold. Thus, the hairpins are interchangeable and in nature this lends itself to adaptability and flexibility.},
  archive      = {J_JCC},
  author       = {John T. Bedford and Jennifer Poutsma and Norou Diawara and Lesley H. Greene},
  doi          = {10.1002/jcc.26477},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {600-607},
  shortjournal = {J. Comput. Chem.},
  title        = {The nature of persistent interactions in two model β-grasp proteins reveals the advantage of symmetry in stability},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Extended diffusion theory: Recovering dynamics from
biased/accelerated molecular simulations. <em>JCC</em>, <em>42</em>(9),
586–599. (<a href="https://doi.org/10.1002/jcc.26474">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamical properties are of great importance in determining the behavior of synthetic and natural molecules, but capturing them by computational methods is a nontrivial task. Very often the time scales of the relevant phenomena are far beyond the typical time windows accessible by classical Molecular Dynamics (MD) simulations, currently limited to the order of microseconds on standard laboratory workstations. On the other hand, biased and accelerated simulations allow for fast and thorough exploration of the molecular conformational space, but they lose the dynamic information. The problem of recovering dynamics from biased/accelerated simulations is a very active field of research, but no totally robust/reliable solutions have been given yet. In this paper it is shown how the Smoluchowski equation, in the framework of Diffusion Theory (DT), can be used to bridge this gap, and dynamical properties, in the form of time correlation functions (TCFs), can be extracted also from such kind of simulations. DT is first extended (EDT) to express the mobility tensors entering the Smoluchowski operator in terms of a recently introduced unified and regularized Rotne–Prager–Yamakawa approximation, [P. J. Zuk, E. Wajnryb, K. A. Mizerski, P. Szymczak,J. Fluid. Mech.2014,741, R5, 1–13] also involving mixed rotation-translation contributions, and rotation-rotation terms beside the classical translation-translation ones, so far used in DT. Then, the method is applied to recover the dynamics of a nontrivial example of a peptide in explicit water from the first 200 ns of a Replica Exchange Molecular Dynamics simulation, which is a popular computational method that destroys the long time dynamics. EDT dynamics were found to favorably compare against those coming from a standard MD simulation of the same system, requiring a time window of 30 μs to converge. This result shows that EDT is a tool of practical value to recover the long time dynamics of systems in diffusive regimes from biased/accelerated simulations, to be exploited in those cases when direct evaluation by standard MD is unfeasible.},
  archive      = {J_JCC},
  author       = {Arnaldo Rapallo and Roberto Gaspari and Gianvito Grasso and Andrea Danani},
  doi          = {10.1002/jcc.26474},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {586-599},
  shortjournal = {J. Comput. Chem.},
  title        = {Extended diffusion theory: Recovering dynamics from biased/accelerated molecular simulations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Cover image. <em>JCC</em>, <em>42</em>(8), C1. (<a
href="https://doi.org/10.1002/jcc.26502">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Shoichi Tanimoto and Koichi Tamura and Shigehiko Hayashi and Norio Yoshida and Haruyuki Nakano},
  doi          = {10.1002/jcc.26502},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {C1},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Introducing pep McConst—a user-friendly peptide modeler for
biophysical applications. <em>JCC</em>, <em>42</em>(8), 572–580. (<a
href="https://doi.org/10.1002/jcc.26479">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We are introducingPep McConst—a software that employs a Monte-Carlo algorithm to construct 3D structures of polypeptide chains which could subsequently be studied as stand-alone macromolecules or complement the structure of known proteins. Using an approach to avoid steric clashes,Pep McConst allows to create multiple structures for a predefined primary sequence of amino acids. These structures could then effectively be used for further structural analysis and investigations. The article introduces the algorithm and describes its user-friendly approach that was made possible through the VIKING online platform. Finally, the manuscript provides several highlight examples wherePep McConst was used to predict the structure of the C-terminal of a known protein, generate a missing bit of already crystallized protein structures and simply generate short polypeptide chains.},
  archive      = {J_JCC},
  author       = {Fabian Schuhmann and Vasili Korol and Ilia A. Solov&#39;yov},
  doi          = {10.1002/jcc.26479},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {572-580},
  shortjournal = {J. Comput. Chem.},
  title        = {Introducing pep McConst—A user-friendly peptide modeler for biophysical applications},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Estimations of FH···x hydrogen bond energies from IR
intensities: Iogansen’s rule revisited. <em>JCC</em>, <em>42</em>(8),
564–571. (<a href="https://doi.org/10.1002/jcc.26482">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this work the possibility of using the IR intensity of the stretching vibrationνsof proton donor group for estimation of hydrogen bond strength was investigated. For a set of complexes with FH···X (X = F, N, O) hydrogen bonds in the wide range of energies (0.1–49.2 kcal/mol) vibrational frequenciesνsand their intensitiesAwere calculated (CCSD at complete basis set limit). The validity of the previously proposed linear proportionality between the intensification of the stretching vibrationνsin IR spectra and hydrogen bond enthalpy –ΔH= 12.2(A. V. Iogansen, Spectrochimica Acta A1999) was examined. It is shown that for a range of similar hydrogen bond types with complexation energies ∆E&lt;15 kcal/mol the ∆E() function remains similar to that proposed in the Iogansen&#39;s work, while upon strengthening this dependency becomes significantly nonlinear. We examined two other parameters (and) related to IR intensity as descriptors of hydrogen bond strength which are proportional to transition dipole moment matrix element and mass-independent dipole moment derivative. It was found that the dependency ∆E() stays linear in the whole studied range of complexation energies and it can be used for evaluation of ∆Efrom infrared spectral data with the accuracy about 2 kcal/mol. The mass-independent productis an appropriate descriptor for sets of complexes with various hydrogen bond types. Simple equations proposed in this work can be used for estimations of hydrogen bond strength in various systems, where experimental thermodynamic methods or direct calculations are difficult or even impossible.},
  archive      = {J_JCC},
  author       = {Elena Yu. Tupikina and Peter M. Tolstoy and Anna A. Titova and Mikhail A. Kostin and Gleb S. Denisov},
  doi          = {10.1002/jcc.26482},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {564-571},
  shortjournal = {J. Comput. Chem.},
  title        = {Estimations of FH···X hydrogen bond energies from IR intensities: Iogansen&#39;s rule revisited},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). A computational method to simulate global conformational
changes of proteins induced by cosolvent. <em>JCC</em>, <em>42</em>(8),
552–563. (<a href="https://doi.org/10.1002/jcc.26481">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A computational method to investigate the global conformational change of a protein is proposed by combining the linear response path following (LRPF) method and three-dimensional reference interaction site model (3D-RISM) theory, which is referred to as the LRPF/3D-RISM method. The proposed method makes it possible to efficiently simulate protein conformational changes caused by either solutions of varying concentrations or the presence of cosolvent species by taking advantage of the LRPF and 3D-RISM. The proposed method is applied to the urea-induced denaturation of ubiquitin. The LRPF/3D-RISM trajectories successfully simulate the early stage of the denaturation process within the simulation time of 300 ns, whereas no significant structural change is observed even in the 1 μs standard MD simulation. The obtained LRPF/3D-RISM trajectories reproduce the mechanism of the urea denaturation of ubiquitin reported in previous studies, and demonstrate the high efficiency of the method.},
  archive      = {J_JCC},
  author       = {Shoichi Tanimoto and Koichi Tamura and Shigehiko Hayashi and Norio Yoshida and Haruyuki Nakano},
  doi          = {10.1002/jcc.26481},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {552-563},
  shortjournal = {J. Comput. Chem.},
  title        = {A computational method to simulate global conformational changes of proteins induced by cosolvent},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Rational design of the nickel-borane complex for efficient
hydrogenation of styrene. <em>JCC</em>, <em>42</em>(8), 545–551. (<a
href="https://doi.org/10.1002/jcc.26480">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Ni-B complex1BCFwith a facilely accessible monophosphine (PtBu3) unit was theoretically designed, which was found to be more active than that with an ambiphilic ligand for hydrogenation of styrene. Substituting PtBu3with a stronger electron donating ligand N-heterocyclic carbene largely improves the activity of the Ni-B complex.},
  archive      = {J_JCC},
  author       = {Lei Zhang and Qin Zhu and Liuzhou Gao and Linlin Yang and Wei Li and Shuhua Li and Jun Zhu and Wei Wang and Guixiang Zeng},
  doi          = {10.1002/jcc.26480},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {545-551},
  shortjournal = {J. Comput. Chem.},
  title        = {Rational design of the nickel-borane complex for efficient hydrogenation of styrene},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Massively parallel quantum chemical density matrix
renormalization group method. <em>JCC</em>, <em>42</em>(8), 534–544. (<a
href="https://doi.org/10.1002/jcc.26476">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present, to the best of our knowledge, the first attempt to exploit the super-computer platform for quantum chemical density matrix renormalization group (QC-DMRG) calculations. We have developed the parallel scheme based on the in-house MPI global memory library, which combines operator and symmetry sector parallelisms, and tested its performance on three different molecules, all typical candidates for QC-DMRG calculations. In case of the largest calculation, which is the nitrogenase FeMo cofactor cluster with the active space comprising 113 electrons in 76 orbitals and bond dimension equal to 6000, our parallel approach scales up to approximately 2000 CPU cores.},
  archive      = {J_JCC},
  author       = {Jiri Brabec and Jan Brandejs and Karol Kowalski and Sotiris Xantheas and Örs Legeza and Libor Veis},
  doi          = {10.1002/jcc.26476},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {534-544},
  shortjournal = {J. Comput. Chem.},
  title        = {Massively parallel quantum chemical density matrix renormalization group method},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Global double hybrids do not work for charge transfer: A
comment on “double hybrids and time-dependent density functional theory:
An implementation and benchmark on charge transfer excited states.”
<em>JCC</em>, <em>42</em>(8), 528–533. (<a
href="https://doi.org/10.1002/jcc.26478">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We comment on the results published by Ottochian et al. inJ. Comput Chem.2020,41, 1242. Therein, the authors claim that the second-order, perturbative correlation correction applied to the time-dependent version of the PBE-QIDH global double-hybrid functional approximation (DHDFA) enables the description of charge-transfer (CT) excitations. Herein, we point out some inadvertent oversights related to what had already been previously known and achieved in the field of time-dependent DHDFAs. Exemplified for the same four systems that Ottochian et al. have used to analyze intermolecular CT excitations, we show how a systematic and unacceptably large redshift in global DHDFAs is rectified when using the latest long-range corrected DHDFAs published earlier inJ. Chem. Theory Comput.2019,15, 4735.},
  archive      = {J_JCC},
  author       = {Marcos Casanova-Páez and Lars Goerigk},
  doi          = {10.1002/jcc.26478},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {528-533},
  shortjournal = {J. Comput. Chem.},
  title        = {Global double hybrids do not work for charge transfer: a comment on “Double hybrids and time-dependent density functional theory: an implementation and benchmark on charge transfer excited states”},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comment on “exploring nature and predicting strength of
hydrogen bonds: A correlation analysis between atoms-in-molecules
descriptors, binding energies, and energy components of symmetry-adapted
perturbation theory.” <em>JCC</em>, <em>42</em>(7), 516–521. (<a
href="https://doi.org/10.1002/jcc.26475">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We evaluate the correlation between binding energy (BE) and electron densityρ(r)at the bond critical point for 28 neutral hydrogen bonds, recently reported by Emamian and co-workers (J. Comput. Chem.,2019,40, 2868). As an efficient tool, we use local stretching force constantderived from the local vibrational mode theory of Konkoli and Cremer. We compare the physical nature of BE versus, and provide an important explanation for cases with significant deviation in the BE–relation as well as in the BE–ρ(r)correlation. We also show that care has to be taken when different hydrogen bond strength measures are compared. The BE is a cumulative hydrogen bond strength measure whileis a local measure of hydrogen bond strength covering different aspects of bonding. A simplified and unified description of hydrogen bonding is not always possible and needs an in-depth understanding of the systems involved.},
  archive      = {J_JCC},
  author       = {Sadisha Nanayakkara and Yunwen Tao and Elfi Kraka},
  doi          = {10.1002/jcc.26475},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {516-521},
  shortjournal = {J. Comput. Chem.},
  title        = {Comment on “Exploring nature and predicting strength of hydrogen bonds: A correlation analysis between atoms-in-molecules descriptors, binding energies, and energy components of symmetry-adapted perturbation theory”},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Assessment of long-range corrected density functional theory
on the absorption and vibrationally resolved fluorescence spectrum of
carbon nanobelts. <em>JCC</em>, <em>42</em>(7), 505–515. (<a
href="https://doi.org/10.1002/jcc.26473">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Time-dependent (TD) density functional theory (DFT) and Franck-Condon Hertzberg-Teller (FCHT) calculations of various DFT functionals [B3LYP, CAM-B3LYP,ωB97XD, and optimally tuned (OT) long-range corrected (LC)-BLYP] were performed to examine how well DFT functionals can predict the experimental absorption and fluorescence spectra of a 12-carbon nanobelt (CNB). OT-LC-BLYP reproduced the experimental absorption spectrum well in terms of the peak position and intensity in the case of using a basis set with a diffuse function, such as 6–31+G(d,p) and 6–311+G(d,p), whereas B3LYP showed a red-shift in the peak positions and CAM-B3LYP andωB97XD, which have a long-range HF exchange, showed blue shifts. Regarding the fluorescence spectrum calculations with FCHT using 6–311+G(d,p), the OT-LC-BLYP reproduced both the peak intensities and positions closest to the experimental spectrum. B3LYP, however, showed red-shifted peaks, andωB97XD showed blue-shifted peaks. CAM-B3LYP provided less blue-shifted peaks, but the relative peak intensities mismatched the experimental ones. Furthermore, calculations of the absorption and vibrationally resolved fluorescence spectra of 16-CNB and 24-CNB using OT-LC-BLYP/6–311+G(d,p) showed absorption and fluorescence spectra close to the experimental spectra with high accuracy. Moreover, the application of a polarizable continuum model (dichloromethane) produced a red shift in the peak positions of the absorption spectrum with increasing intensity but an increase in the peak intensities of the fluorescence calculations without shifting the peak position.},
  archive      = {J_JCC},
  author       = {Dae-Hwan Ahn and Jong-Won Song},
  doi          = {10.1002/jcc.26473},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {505-515},
  shortjournal = {J. Comput. Chem.},
  title        = {Assessment of long-range corrected density functional theory on the absorption and vibrationally resolved fluorescence spectrum of carbon nanobelts},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A diagonalization-free optimization algorithm for solving
kohn–sham equations of closed-shell molecules. <em>JCC</em>,
<em>42</em>(7), 492–504. (<a
href="https://doi.org/10.1002/jcc.26472">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A local optimization algorithm for solving the Kohn–Sham equations is presented. It is based on a direct minimization of the energy functional under the equality constraints representing the Grassmann Manifold. The algorithm does not require an eigendecomposition, which may be advantageous in large-scale computations. It is optimized to reduce the number of Kohn–Sham matrix evaluations to one per iteration to be competitive with standard self-consistent field (SCF) approach accelerated by direct inversion of the iterative subspace (DIIS). Numerical experiments include a comparison of the algorithm with DIIS. A high reliability of the algorithm is observed in configurations where SCF iterations fail to converge or find a wrong solution corresponding to a stationary point different from the global minimum. The local optimization algorithm itself does not guarantee that the found minimum is global. However, a randomization of the initial approximation shows a convergence to the right minimum in the vast majority of cases.},
  archive      = {J_JCC},
  author       = {Martin Mrovec and J. A. Berger},
  doi          = {10.1002/jcc.26472},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {492-504},
  shortjournal = {J. Comput. Chem.},
  title        = {A diagonalization-free optimization algorithm for solving Kohn–Sham equations of closed-shell molecules},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computational prediction on the catalytic activity of
heterobimetallic complex featuring m?Mˊ triple bond in acetylene
cyclotrimerization: Mechanistic study. <em>JCC</em>, <em>42</em>(7),
484–491. (<a href="https://doi.org/10.1002/jcc.26471">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A detailed reaction mechanism of acetylene cyclotrimerization catalyzed by V(iPrNPMe2)3Fe-PMe3(denote as CAT), a heterobimetallic complex featuring V-Fe triple bond, was computationally investigated using density functional theory. The calculated results show that the first acetylene firstly attaches to the V atom of CAT to get a four-membered ring structure through [2 + 2] cycloaddition reaction. For the second acetylene addition, there are two cyclotrimerization mechanisms, outer sphere mechanism and inner mechanism. The inner sphere reaction pathway is the main reaction pathway. By replacing the V with Nb and Ta, Fe with Ru and Os, a series of new catalysts are screened computationally. The calculated results show that, all of the nine heterobimetallic complexes show high activity at mild condition. The energy barrier of the rate determining step is related to the natural population analysis (NPA) charge of Mˊ and the Wiberg bond index (WBI) of M-Mˊ bond. The more negative NPA charge of Mˊ and the smaller WBI of M-Mˊ bond, the lower energy barrier is.},
  archive      = {J_JCC},
  author       = {Na Zhang and Jinhui Mi and Suhong Huo and Lingpeng Meng and Xiaoyan Li},
  doi          = {10.1002/jcc.26471},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {484-491},
  shortjournal = {J. Comput. Chem.},
  title        = {Computational prediction on the catalytic activity of heterobimetallic complex featuring M?Mˊ triple bond in acetylene cyclotrimerization: Mechanistic study},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fragmentation of natural orbital occupation numbers-based
diagnostic of differential multireference character in complexes with
hydrogen bonds. <em>JCC</em>, <em>42</em>(7), 475–483. (<a
href="https://doi.org/10.1002/jcc.26470">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We explore the possible route to approximate natural orbital occupation numbers-based diagnostic of differential multireference character of noncovalent energy differences by techniques based on many-body expansion. It turns out that two-body fragmentation of monomers may lead to a reasonable approximation of such a diagnostic in hydrogen-bonded complexes. The results are useful, for example, for assessment of the expected bias cancellation in energy differences of larger systems obtained by single-reference methods.},
  archive      = {J_JCC},
  author       = {Martin Šulka and Matúš Dubecký},
  doi          = {10.1002/jcc.26470},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {475-483},
  shortjournal = {J. Comput. Chem.},
  title        = {Fragmentation of natural orbital occupation numbers-based diagnostic of differential multireference character in complexes with hydrogen bonds},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Revisiting the effect of f-functions in predicting the right
reaction mechanism for hypervalent iodine reagents. <em>JCC</em>,
<em>42</em>(7), 470–474. (<a
href="https://doi.org/10.1002/jcc.26469">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand the effect off-functions in predicting the right reaction mechanism for hypervalent iodine reagents, we adopt the Ahlrichs basis set family def2-SVP and def2-TZVP to revisit the potential energy surfaces ofIBX-mediated oxidation andTogni I&#39;s isomerisation. Our results further prove thatf-functions (in either Pople, Dunning, or Ahlrichs basis set series) are indispensable to predict the correct rate-determining step of hypervalent iodine reagents. Thef-functions have a significant impact on the predicted reaction barriers for processes involving the IX (X = O, OH, CF3, etc.) bond cleavage and formation, for example, in the reductive elimination step or the hypervalent twist step. We furthermore explore two hypervalent twist modes that account for the different influences off-functions forIBXandTogni I. Our findings may be helpful for theoretical chemists to appropriately study the reaction mechanism of hypervalent iodine reagents.},
  archive      = {J_JCC},
  author       = {Tian-Yu Sun and Kai Chen and Huakang Zhou and Tingting You and Penggang Yin and Xiao Wang},
  doi          = {10.1002/jcc.26469},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {470-474},
  shortjournal = {J. Comput. Chem.},
  title        = {Revisiting the effect of f-functions in predicting the right reaction mechanism for hypervalent iodine reagents},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Cover image. <em>JCC</em>, <em>42</em>(6), C1. (<a
href="https://doi.org/10.1002/jcc.26245">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Bartosz Błasiak and Joanna D. Bednarska and Marta Chołuj and Robert W. Góra and Wojciech Bartkowiak},
  doi          = {10.1002/jcc.26245},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {C1},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). IOData: A python library for reading, writing, and
converting computational chemistry file formats and generating input
files. <em>JCC</em>, <em>42</em>(6), 458–464. (<a
href="https://doi.org/10.1002/jcc.26468">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {IODatais a free and open-source Python library for parsing, storing, and converting various file formats commonly used by quantum chemistry, molecular dynamics, and plane-wave density-functional-theory software programs. In addition,IODatasupports a flexible framework for generating input files for various software packages. While designed and released for stand-alone use, its original purpose was to facilitate the interoperability of various modules in theHORTONandChemToolssoftware packages with external (third-party) molecular quantum chemistry and solid-state density-functional-theory packages.IODatais designed to be easy to use, maintain, and extend; this is why we wroteIODatain Python and adopted many principles of modern software development, including comprehensive documentation, extensive testing, continuous integration/delivery protocols, and package management. This article is the official release note of theIODatalibrary.},
  archive      = {J_JCC},
  author       = {Toon Verstraelen and William Adams and Leila Pujal and Alireza Tehrani and Braden D. Kelly and Luis Macaya and Fanwang Meng and Michael Richer and Raymundo Hernández-Esparza and Xiaotian Derrick Yang and Matthew Chan and Taewon David Kim and Maarten Cools-Ceuppens and Valerii Chuiko and Esteban Vöhringer-Martinez and Paul W. Ayers and Farnaz Heidar-Zadeh},
  doi          = {10.1002/jcc.26468},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {458-464},
  shortjournal = {J. Comput. Chem.},
  title        = {IOData: A python library for reading, writing, and converting computational chemistry file formats and generating input files},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). QMCube (QM3): An all-purpose suite for multiscale QM/MM
calculations. <em>JCC</em>, <em>42</em>(6), 447–457. (<a
href="https://doi.org/10.1002/jcc.26465">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {QMCube (QM3) is a suite written in the Python programming language, initially focused on multiscale QM/MM simulations of biological systems, but open enough to address other kinds of problems. It allows the user to combine highly efficient QM and MM programs, providing unified access to a wide range of computational methods. The suite also supplies additional modules with extra functionalities. These modules facilitate common tasks such as performing the setup of the models or process the data generated during the simulations. The design of QM3has been carried out considering the least number of external dependencies (only an algebra library, already included in the distribution), which makes it extremely portable. Also, the modular structure of the suite should help to expand and develop new computational methods.},
  archive      = {J_JCC},
  author       = {Sergio Martí},
  doi          = {10.1002/jcc.26465},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {447-457},
  shortjournal = {J. Comput. Chem.},
  title        = {QMCube (QM3): An all-purpose suite for multiscale QM/MM calculations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PyVisA: Visualization and analysis of path sampling
trajectories. <em>JCC</em>, <em>42</em>(6), 435–446. (<a
href="https://doi.org/10.1002/jcc.26467">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rare event methods applied to molecular simulations are growing in popularity, accessible and customizable software solutions have thus been developed and released. One of the most recent is PyRETIS, an open Python library for performing path sampling simulations. Here, we introduce PyVisA, a postprocessing package for path sampling simulations, which includes visualization and analysis tools for interpreting path sampling outputs. PyVisA integrates PyRETIS functionalities and aims to facilitate the determination of: (a) the correlation of the order parameter with other descriptors; (b) the presence of latent variables; and (c) intermediate meta-stable states. To illustrate some of the main PyVisA features, we investigate the proton transfer reaction in a protonated water trimer simulated via a simple polarizable model (Stillinger−David).},
  archive      = {J_JCC},
  author       = {Ola Aarøen and Henrik Kiær and Enrico Riccardi},
  doi          = {10.1002/jcc.26467},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {435-446},
  shortjournal = {J. Comput. Chem.},
  title        = {PyVisA: Visualization and analysis of path sampling trajectories},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A method to apply bond-angle constraints in molecular
dynamics simulations. <em>JCC</em>, <em>42</em>(6), 418–434. (<a
href="https://doi.org/10.1002/jcc.26466">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An algorithm to apply bond-angle constraints in molecular dynamics simulations of macromolecules or molecular liquids is presented. It uses Cartesian coordinates and determines the Lagrange multipliers required for maintaining the constraints iteratively. It constitutes an alternative to the use of only distance constraints (DCs) between particles to maintain a particular geometry. DCs are unsuitable to maintain particular, for example, linear or flat, geometries of molecules. The proposed algorithm can easily handle bond-length, bond-angle, and dihedral-angle constraints simultaneously, as when calculating a potential of mean force along a dihedral-angle degree of freedom.},
  archive      = {J_JCC},
  author       = {Maria Pechlaner and Andreas P. Dorta and Zhixiong Lin and Victor H. Rusu and Wilfred F. van Gunsteren},
  doi          = {10.1002/jcc.26466},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {418-434},
  shortjournal = {J. Comput. Chem.},
  title        = {A method to apply bond-angle constraints in molecular dynamics simulations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Superposition of waves or densities: Which is the nature of
chemical resonance? <em>JCC</em>, <em>42</em>(6), 412–417. (<a
href="https://doi.org/10.1002/jcc.26463">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resonance is a fundamental and widely used concept in chemistry, but there exist two distinct theories of chemical resonance, based on quite different and incompatible premises: the wave-function-based resonance theory (WFRT), assuming the superposition of wave functions, versus the density-matrix-based resonance theory (DMRT), which interprets the resonance phenomenon as the superposition of density matrices. The latter theory, best known to the chemistry community as the natural resonance theory (NRT), has received much more popularity than the WFRT. In this contribution, the DMRT is shown to be inherently inadequate: (i) the exact density matrix expansion is mathematically impossible unless unphysical negative weights are introduced; (ii) any approximate density matrix representing the resonance hybrid lacks the idempotent property. Therefore, the validity of the NRT ansatz should be seriously questioned. The WFRT seems the only reasonable explanation of resonance so far, and has been shown to provide valuable insights into diverse chemical problems.},
  archive      = {J_JCC},
  author       = {Yang Wang},
  doi          = {10.1002/jcc.26463},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {412-417},
  shortjournal = {J. Comput. Chem.},
  title        = {Superposition of waves or densities: Which is the nature of chemical resonance?},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Ab initio effective one-electron potential operators:
Applications for charge-transfer energy in effective fragment
potentials. <em>JCC</em>, <em>42</em>(6), 398–411. (<a
href="https://doi.org/10.1002/jcc.26462">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The concept of effective one-electron potentials (EOPs) has proven to be extremely useful in efficient description of electronic structure of chemical systems, especially extended molecular aggregates such as interacting molecules in condensed phases. Here, a general method for EOP-based elimination of electron repulsion integrals is presented, that is tuned toward the fragment-based calculation methodologies such as the second generation of the effective fragment potentials (EFP2) method. Two general types of the EOP operator matrix elements are distinguished and treated either via the distributed multipole expansion or the extended density fitting (DF) schemes developed in this work. The EOP technique is then applied to reduce the high computational costs of the effective fragment charge-transfer (CT) terms being the bottleneck of EFP2 potentials. The alternative EOP-based CT energy model is proposed, derived within the framework of intermolecular perturbation theory with Hartree–Fock noninteracting reference wavefunctions, compatible with the original EFP2 formulation. It is found that the computational cost of the EFP2 total interaction energy calculation can be reduced by up to 38 times when using the EOP-based formulation of CT energy, as compared to the original EFP2 scheme, without compromising the accuracy for a wide range of weakly interacting neutral and ionic molecular fragments. The proposed model can thus be used routinely within the EFP2 framework.},
  archive      = {J_JCC},
  author       = {Bartosz Błasiak and Joanna D. Bednarska and Marta Chołuj and Robert W. Góra and Wojciech Bartkowiak},
  doi          = {10.1002/jcc.26462},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {398-411},
  shortjournal = {J. Comput. Chem.},
  title        = {Ab initio effective one-electron potential operators: Applications for charge-transfer energy in effective fragment potentials},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CATKINAS: A large-scale catalytic microkinetic analysis
software for mechanism auto-analysis and catalyst screening.
<em>JCC</em>, <em>42</em>(5), 379–391. (<a
href="https://doi.org/10.1002/jcc.26464">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As an effective method to analyze complex catalytic reaction networks, microkinetic modeling is gaining increasing popularity in the catalytic activity evaluation and rational design of heterogeneous catalysts. An automated simulator with stable and reliable performance is especially useful and in great request. Here we introduce the CATKINAS package developed for large-scale microkinetic modeling and analysis. Featuring with a multilevel solver and a multifunctional analyzer, CATKINAS can provide both accurate solutions and various quantitative and automatic analysis for a wide range of catalytic systems. The structure and the basic workflow are overviewed with the multilevel solver particularly illustrated. Also, we take the CO methanation reaction as an example to illustrate the application and efficiency of the CATKINAS package.},
  archive      = {J_JCC},
  author       = {Jianfu Chen and Menglei Jia and Peijun Hu and Haifeng Wang},
  doi          = {10.1002/jcc.26464},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {379-391},
  shortjournal = {J. Comput. Chem.},
  title        = {CATKINAS: A large-scale catalytic microkinetic analysis software for mechanism auto-analysis and catalyst screening},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A new release of MOPAC incorporating the INDO/s
semiempirical model with CI excited states. <em>JCC</em>,
<em>42</em>(5), 365–378. (<a
href="https://doi.org/10.1002/jcc.26455">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The semiempirical INDO/S Hamiltonian is incorporated into a new release of MOPAC2016. The MOPAC2016 software package has long been at the forefront of semiempirical quantum chemical methods (SEQMs) for small molecules, proteins, and solids and until this release has included only NDDO-type SEQMs. The new code enables the calculation of excited states using the INDO/S Hamiltonian combined with a configuration interaction (CI) approach using single excitations (CIS), single and double excitations (CISD), or multiple reference determinants (MRCI) where reference determinants are generated using a complete active space (CAS) approach. The capacity to perform excited-state calculations beyond the CIS level makes INDO/CI one of the few low-cost computational methods capable of accurately modeling states with substantial double-excitation character. Solvent corrections to the ground-state and excited-state energies can be computed using the COSMO implicit solvent model, incorporating state-specific corrections to the excited states based on the solvent refractive index. This code produces physically reasonable electronic structures, absorption spectra, and solvatochromic shifts at low computational costs for systems up to hundreds of atoms, and for both organic molecules and metal clusters.},
  archive      = {J_JCC},
  author       = {Rebecca L. M. Gieseking},
  doi          = {10.1002/jcc.26455},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {365-378},
  shortjournal = {J. Comput. Chem.},
  title        = {A new release of MOPAC incorporating the INDO/S semiempirical model with CI excited states},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Accelerating atomistic simulations of proteins using
multiscale enhanced sampling with independent tempering. <em>JCC</em>,
<em>42</em>(5), 358–364. (<a
href="https://doi.org/10.1002/jcc.26461">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efficient sampling of the conformational space is essential for quantitative simulations of proteins. The multiscale enhanced sampling (MSES) method accelerates atomistic sampling by coupling it to a coarse-grained (CG) simulation. Bias from coupling to the CG model is removed using Hamiltonian replica exchange, such that one could benefit simultaneously from the high accuracy of atomistic models and fast dynamics of CG ones. Here, we extend MSES to allow independent control of the effective temperatures of atomistic and CG simulations, by directly scaling the atomistic and CG Hamiltonians. The new algorithm, named MSES with independent tempering (MSES-IT), supports more sophisticated Hamiltonian and temperature replica exchange protocols to further improve the sampling efficiency. Using a small but nontrivial β-hairpin, we show that setting the effective temperature of CG model in all conditions to its melting temperature maximizes structural transition rates at the CG level and promotes more efficient replica exchange and diffusion in the condition space. As the result, MSES-IT drive faster reversible transitions at the atomic level and leads to significant improvement in generating converged conformational ensembles compared to the original MSES scheme.},
  archive      = {J_JCC},
  author       = {Xiaorong Liu and Xiping Gong and Jianhan Chen},
  doi          = {10.1002/jcc.26461},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {358-364},
  shortjournal = {J. Comput. Chem.},
  title        = {Accelerating atomistic simulations of proteins using multiscale enhanced sampling with independent tempering},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Molecular dynamic simulations, GIAO-NMR and TD-DFT
spectroscopy analyze for zwitterionic isoleucine (ILE)n, 1 ≤ n ≤ 6, in
water solution. <em>JCC</em>, <em>42</em>(5), 344–357. (<a
href="https://doi.org/10.1002/jcc.26460">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this article, we investigate the effects of the isoleucine (ILE)Namino acid chain growth, N = 1.0.6, the ILE conformational effect as well as the solvent presence on the electrical and magnetic spectroscopic properties when these compounds are in aqueous solution. Computational molecular dynamics simulations were performed to include the solvent medium and generate uncorrelated configurations involving solute-solvent structures. The charge point model for solvent was used to obtain the results for quantum mechanical calculation, in special DFT calculations, for (ILE)Nstructures. Our results for the magnetic shielding constant obtained via GIAO-DFT-NMR calculations show that there is evidence of a magnetic behavior that characterizes the number of peptide bonds and, therefore, how the N isoleucine polypeptide chain is composed. TD-DFT results also show an absorption band shift to larger wavelengths indicating a dependence on N growth.},
  archive      = {J_JCC},
  author       = {Karinna Mendanha and Richard Costa Prado and Leonardo Bruno Assis Oliveira and Guilherme Colherinhas},
  doi          = {10.1002/jcc.26460},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {344-357},
  shortjournal = {J. Comput. Chem.},
  title        = {Molecular dynamic simulations, GIAO-NMR and TD-DFT spectroscopy analyze for zwitterionic isoleucine (ILE)N, 1 ≤ n ≤ 6, in water solution},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Orbital energies and nuclear forces in DFT: Interpretation
and validation. <em>JCC</em>, <em>42</em>(5), 334–343. (<a
href="https://doi.org/10.1002/jcc.26459">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The bonding and antibonding character of individual molecular orbitals has been previously shown to be related to their orbital energy derivatives with respect to nuclear coordinates, known as dynamical orbital forces. Albeit usually derived from Koopmans&#39; theorem, in this work we show a more general derivation from conceptual DFT, which justifies application in a broader context. The consistency of the approach is validated numerically for valence orbitals in Kohn–Sham DFT. Then, we illustrate its usefulness by showcasing applications in aromatic and antiaromatic systems and in excited state chemistry. Overall, dynamical orbital forces can be used to interpret the results of routine ab initio calculations, be it wavefunction or density based, in terms of forces and occupations.},
  archive      = {J_JCC},
  author       = {Rubén Laplaza and Carlos Cárdenas and Patrick Chaquin and Julia Contreras-García and Paul W. Ayers},
  doi          = {10.1002/jcc.26459},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {334-343},
  shortjournal = {J. Comput. Chem.},
  title        = {Orbital energies and nuclear forces in DFT: Interpretation and validation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Electronic couplings for singlet fission: Orbital choice and
extrapolation to the complete basis set limit. <em>JCC</em>,
<em>42</em>(5), 326–333. (<a
href="https://doi.org/10.1002/jcc.26458">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For the search for promising singlet fission candidates, the calculation of the effective electronic coupling, which is required to estimate the singlet fission rate between the initially excited state (S0S1) and the multiexcitonic state (1TT, two triplets on neighboring molecules, coupled into a singlet), should be sufficiently reliable and fast enough to explore the configuration space. We propose here to modify the calculation of the effective electronic coupling using a nonorthogonal configuration interaction approach by: (a) using only one set of orbitals, optimized for the triplet state of the molecules, to describe all molecular electronic states, and (b) only taking the leading configurations into consideration. Furthermore, we also studied the basis set convergence of the electronic coupling, and we found, by comparison to the complete basis set limit obtained using the cc-pVnZ series of basis sets, that both the aug-cc-pVDZ and 6–311++G** basis sets are a good compromise between accuracy and computational feasibility. The proposed approach enables future work on larger clusters of molecules than dimers.},
  archive      = {J_JCC},
  author       = {Tom Speelman and Ana V. Cunha and R. K. Kathir and Remco W. A. Havenith},
  doi          = {10.1002/jcc.26458},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {326-333},
  shortjournal = {J. Comput. Chem.},
  title        = {Electronic couplings for singlet fission: Orbital choice and extrapolation to the complete basis set limit},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Neutral gold clusters studied by the isothermal
brownian-type molecular dynamics and metadynamics molecular dynamics
simulations. <em>JCC</em>, <em>42</em>(5), 310–325. (<a
href="https://doi.org/10.1002/jcc.26457">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The DFTB theory was combined with the isothermal Brownian-type molecular dynamics (MD) and metadynamics molecular dynamics (MMD) algorithms to perform simulation studies for Au clusters. Two representative DFTB parametrizations were investigated. In one parametrization, the DFTB-A, the Slater–Koster parameters in the DFTB energy function were determined focusing on the ionic repulsive energy part,Erepand the other, the DFTB-B, due attention was paid to the electronic band-structure energy part,Eband. Minimized structures of these two parametrizations were separately applied in MD and MMD simulations to generate unbiased and biased trajectories in collective variable (CV) space, respectively. Here, we found the MD simulations monitored at 300Kmanifest fluxional characteristics in planar cluster Au9/DFTB-A, but give no discernible tracts of fluxionality for planar Au8/DFTB-A and Au8/DFTB-B, for nonplanar Au10/DFTB-A and, to some extent, for nonplanar Au9/DFTB-B; they are plausibly being hindered by higher-thankBTenergy barriers. Very recent FIR-MPD spectroscopy measurements, however, were reported to have detected at 300Kboth the planar and nonplanar neutral Aunclusters in the size range 5 ≤n≤ 13. The failure of MD simulations has prompted us to apply the MMD simulation and construct the free energy landscape (FEL) inCVspace. Through scrutinizing the FELs of these clusters and their associated structures, we examine the relative importance ofErep/DFTB-A andEband/DFTB-B in unraveling the covalent-like behavior of valence electrons in Aun. Most important of all, we shall evaluate the DFTB parametrization in MMD strategy through comparing extensively the simulation data recorded with the gas-phase experimental data.},
  archive      = {J_JCC},
  author       = {San Kiong Lai and Chong Chiat Lim},
  doi          = {10.1002/jcc.26457},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {310-325},
  shortjournal = {J. Comput. Chem.},
  title        = {Neutral gold clusters studied by the isothermal brownian-type molecular dynamics and metadynamics molecular dynamics simulations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Variation of the optical properties with size and
composition of small, isolated CdxSey+ clusters. <em>JCC</em>,
<em>42</em>(5), 303–309. (<a
href="https://doi.org/10.1002/jcc.26456">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global energy minimum structures and optoelectronic properties are presented for isolated CdxSey+clusters withx+y≤ 26. The compositional- and size-dependent variation of optical, electronic and geometric properties is systematically studied within the framework of ground state and time-dependent density functional theory. The applied methods are justified by benchmarks with experimental data. It is shown that the optical gap can be tuned by more than 2 eV by only changing the composition for a fixed number of atoms. The stoichiometric species reveal an unexpected size-dependent behavior in comparison to larger colloidal CdSe quantum dots, that is, a redshift of the optical gap was observed with decreasing cluster size in contrast to predictions by quantum-size effects. This unexpected result is discussed in detail taking the positive charge of the clusters into account.},
  archive      = {J_JCC},
  author       = {Marc Jäger and Rolf Schäfer},
  doi          = {10.1002/jcc.26456},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {303-309},
  shortjournal = {J. Comput. Chem.},
  title        = {Variation of the optical properties with size and composition of small, isolated CdxSey+ clusters},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Unraveling individual host–guest interactions in molecular
recognition from first principles quantum mechanics: Insights into the
nature of nicotinic acetylcholine receptor agonist binding.
<em>JCC</em>, <em>42</em>(5), 293–302. (<a
href="https://doi.org/10.1002/jcc.26454">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug binding to a protein target is governed by a complex pattern of noncovalent interactions between the ligand and the residues in the protein&#39;s binding pocket. Here we introduce a generally applicable, parameter-free, computational method that allows for the identification, quantification, and analysis of the key ligand–residue interactions responsible for molecular recognition. Our strategy relies on Local Energy Decomposition analysis at the “gold-standard” coupled cluster DLPNO-CCSD(T) level. In the study case shown in this paper, nicotine and imidacloprid binding to the nicotinic acetylcholine receptor, our approach provides new insights into how individual amino acids in the active site determine sensitivity and selectivity of the ligands, extending and refining classical pharmacophore hypotheses. By inference, the method is applicable to any kind of host/guest interactions with potential applications in industrial biocatalysis and protein engineering.},
  archive      = {J_JCC},
  author       = {Michael Edmund Beck and Christoph Riplinger and Frank Neese and Giovanni Bistoni},
  doi          = {10.1002/jcc.26454},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {293-302},
  shortjournal = {J. Comput. Chem.},
  title        = {Unraveling individual host–guest interactions in molecular recognition from first principles quantum mechanics: Insights into the nature of nicotinic acetylcholine receptor agonist binding},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Protein secondary structure motifs: A kinematic
construction. <em>JCC</em>, <em>42</em>(5), 271–292. (<a
href="https://doi.org/10.1002/jcc.26448">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The kinematic geometry of protein backbone structures, constrained by either single or multiple hydrogen bonds (H-bonds), possibly in a periodic array, is discussed. These structures include regular secondary structure elementsα-helices andβ-sheets but also include other short H-bond stabilized irregular structural elements likeβ-turns. The work here shows that the variations observed in such structures have simple geometrical correlations consistent with constrained motion kinematics. A new classification of the ideal helices is given, in terms of the parameterα, the angle at aCαatom to its two neighboringCα&#39;s along the helix, and shown how it can be generalized to include nonideal helices. Specifically, we derive an analytical expression of the backbone dihedrals,(ϕ,ψ), in terms of the parameterαsubject to the constraint that the peptide planes are parallel to the helical axis. Helices constructed in this way exhibit near-vertical alignment of theC = OandN − Hunits and are the canonical objects of this study. These expressions are easily modifiable to include perturbations of parameters relevant to nonplanar peptide units and noncanonical angles. The addition of a second parameter,ε0, inclination of successive peptide planes along a helix with respect to the helical axis leads to a generalization of the previous expression and provides an efficient parametrization of such structures in terms of coordinates consistent with H-bond parameters. An analogs parametrization ofβ-turns, using inverse kinematic methods, is also given. Besides offering a unifying viewpoint, our results may find useful applications to protein and peptide design.},
  archive      = {J_JCC},
  author       = {Mosavverul Hassan and Evangelos A. Coutsias},
  doi          = {10.1002/jcc.26448},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {271-292},
  shortjournal = {J. Comput. Chem.},
  title        = {Protein secondary structure motifs: A kinematic construction},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comment on “martini force field for protonated
polyethyleneimine.” <em>JCC</em>, <em>42</em>(4), 261–263. (<a
href="https://doi.org/10.1002/jcc.26453">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We comment on the recently published Martini forcefield for linear polyethylenimine (Beu et al.,J. Comput. Chem.,2020,41, 349). With supporting evidence, we demonstrate that the new coarse-graining methodology presented in that work is inconsistent with the Martini methodology and lacks important validation steps which is usually required by Martini forcefields.},
  archive      = {J_JCC},
  author       = {Subhamoy Mahajan and Tian Tang},
  doi          = {10.1002/jcc.26453},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {261-263},
  shortjournal = {J. Comput. Chem.},
  title        = {Comment on “Martini force field for protonated polyethyleneimine”},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Prediction of lattice energy of benzene crystals: A robust
theoretical approach. <em>JCC</em>, <em>42</em>(4), 248–260. (<a
href="https://doi.org/10.1002/jcc.26452">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an inexpensive and robust theoretical approach based on the fragment molecular orbital methodology and the spin-ratio scaled second-order Møller–Plesset perturbation theory to predict the lattice energy of benzene crystals within 2 kJ⋅mol−1. Inspired by the Harrison method to estimate the Madelung constant, the proposed approach calculates the lattice energy as a sum of two- and three-body interaction energies between a reference molecule and the surrounding molecules arranged in a sphere. The lattice energy converges rapidly at a radius of 13 Å. Adding the corrections to account for a higher correlated level of theory and basis set superposition for the Hartree Fock (HF) level produced a lattice energy of−57.5 kJ⋅mol−1for the benzene crystal structure at 138 K. This estimate is within 1.6 kJ⋅mol−1off the best theoretical prediction of−55.9 kJ⋅mol−1. We applied this approach to calculate lattice energies of the crystal structures of phase I and phase II—polymorphs of benzene—observed at a higher temperature of 295 K. The stability of these polymorphs was correctly predicted, with phase II being energetically preferred by 3.7 kJ⋅mol−1over phase I. The proposed approach gives a tremendous potential to predict stability of other molecular crystal polymorphs.},
  archive      = {J_JCC},
  author       = {Anh L. P. Nguyen and Thomas G. Mason and Benny D. Freeman and Ekaterina I. Izgorodina},
  doi          = {10.1002/jcc.26452},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {248-260},
  shortjournal = {J. Comput. Chem.},
  title        = {Prediction of lattice energy of benzene crystals: A robust theoretical approach},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Argentophillic interactions in argentum chalcogenides: First
principles calculations and topological analysis of electron density.
<em>JCC</em>, <em>42</em>(4), 242–247. (<a
href="https://doi.org/10.1002/jcc.26451">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The plasticity of Ag2S and Ag2Se crystals important for applications is associated mainly with AgAg metallophilic bonds, which are related to van der Waals interactions and therefore are not directed. This is demonstrated by the first principles DFT M06 LCAO calculations of Ag2X (X = S, Se) crystals (periodic model) and hypothetical molecules XnAg2n(X = S, Se;n= 1, 2, 4) in gas phase (molecular model). A topological analysis of the calculated electron density, was performed both for periodic and molecular models of Ag2X (X = S, Se). It was found that Ag–Ag interatomic distances are close in periodic and molecular models. The numerical values of electron density, its Laplacian, kinetic, and potential energy densities are also close in both models and confirm the existence of AgAg metallophilic bonds.},
  archive      = {J_JCC},
  author       = {Robert A. Evarestov and Andrej I. Panin and Yuri S. Tverjanovich},
  doi          = {10.1002/jcc.26451},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {242-247},
  shortjournal = {J. Comput. Chem.},
  title        = {Argentophillic interactions in argentum chalcogenides: First principles calculations and topological analysis of electron density},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). New parallel computing algorithm of molecular dynamics for
extremely huge scale biological systems. <em>JCC</em>, <em>42</em>(4),
231–241. (<a href="https://doi.org/10.1002/jcc.26450">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper, we address high performance extreme-scale molecular dynamics (MD) algorithm in the GENESIS software to perform cellular-scale molecular dynamics (MD) simulations with more than 100,000 CPU cores. It includes (1) the new algorithm of real-space nonbonded interactions maximizing the performance on ARM CPU architecture, (2) reciprocal-space nonbonded interactions minimizing communicational cost, (3) accurate temperature/pressure evaluations that allows a large time step, and (4) effective parallel file inputs/outputs (I/O) for MD simulations of extremely huge systems. The largest system that contains 1.6 billion atoms was simulated using MD with a performance of 8.30 ns/day on Fugaku supercomputer. It extends the available size and time of MD simulations to answer unresolved questions of biomacromolecules in a living cell.},
  archive      = {J_JCC},
  author       = {Jaewoon Jung and Chigusa Kobayashi and Kento Kasahara and Cheng Tan and Akiyoshi Kuroda and Kazuo Minami and Shigeru Ishiduki and Tatsuo Nishiki and Hikaru Inoue and Yutaka Ishikawa and Michael Feig and Yuji Sugita},
  doi          = {10.1002/jcc.26450},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {231-241},
  shortjournal = {J. Comput. Chem.},
  title        = {New parallel computing algorithm of molecular dynamics for extremely huge scale biological systems},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Thermochemical stabilities of giant fullerenes using density
functional tight binding theory and isodesmic-type reactions.
<em>JCC</em>, <em>42</em>(4), 222–230. (<a
href="https://doi.org/10.1002/jcc.26449">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a systematic assessment of the density functional tight binding (DFTB) method for calculating heats of formation of fullerenes with isodesmic-type reaction schemes. We show that DFTB3-D/3ob can accurately predict ΔfHvalues of the 1812 structural isomers of C60, reproduce subtle trends in ΔfHvalues for 24 isolated pentagon rule (IPR) isomers of C84, and predict ΔfHvalues of giant fullerenes that are in effectively exact agreement with benchmark DSD-PBEP86/def2-QZVPP calculations. For fullerenes up to C320, DFTB ΔfHvalues are within 1.0 kJ mol−1of DSD-PBEP86/def2-QZVPP values per carbon atom, and on a per carbon atom basis DFTB3-D/3ob yields exactly the same numerical trend of (ΔfH[per carbon] = 722n−0.72+ 5.2 kJ mol−1). DFTB3-D/3ob is therefore an accurate replacement for high-level DHDFT and composite thermochemical methods in predicting of thermochemical stabilities of giant fullerenes and analogous nanocarbon architectures.},
  archive      = {J_JCC},
  author       = {Simone L. Waite and Amir Karton and Bun Chan and Alister J. Page},
  doi          = {10.1002/jcc.26449},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {222-230},
  shortjournal = {J. Comput. Chem.},
  title        = {Thermochemical stabilities of giant fullerenes using density functional tight binding theory and isodesmic-type reactions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Instanton calculations of tunneling splittings in chiral
molecules. <em>JCC</em>, <em>42</em>(4), 210–221. (<a
href="https://doi.org/10.1002/jcc.26447">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report the ground state tunneling splittings (ΔE±) of a number of axially chiral molecules using the ring-polymer instanton (RPI) method (J. Chem. Phys.,2011,134, 054109). The list includes isotopomers of hydrogen dichalcogenides H2X2(X = O, S, Se, Te, and Po), hydrogen thioperoxide HSOH and dichlorodisulfane S2Cl2. Ab initio electronic-structure calculations have been performed on the level of second-order Møller–Plesset perturbation (MP2) theory either with split-valance basis sets or augmented correlation-consistent basis sets on H, O, S, and Cl atoms. Energy-consistent pseudopotential and corresponding triple zeta basis sets of the Stuttgart group are used on Se, Te, and Po atoms. The results are further improved using single point calculations performed at the coupled cluster level with iterative singles and doubles and perturbative triples amplitudes. When available for comparison, our computed values of ΔE±are found to lie within the same order of magnitude as values reported in the literature, although RPI also provides predictions for H2Po2and S2Cl2, which have not previously been directly calculated. Since RPI is a single-shot method which does not require detailed prior knowledge of the optimal tunneling path, it offers an effective way for estimating the tunneling dynamics of more complex chiral molecules, and especially those with small tunneling splittings.},
  archive      = {J_JCC},
  author       = {Nityananda Sahu and Jeremy O. Richardson and Robert Berger},
  doi          = {10.1002/jcc.26447},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {210-221},
  shortjournal = {J. Comput. Chem.},
  title        = {Instanton calculations of tunneling splittings in chiral molecules},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High performance global exploration of isomers and
isomerization channels on quantum chemical potential energy surface of
H5C2NO2. <em>JCC</em>, <em>42</em>(3), 192–204. (<a
href="https://doi.org/10.1002/jcc.26446">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High performance global exploration of isomers and isomerization channels on the quantum chemical potential energy surface (PES) is performed for H5C2NO2by using the scaled hypersphere search-anharmonic downward distortion following (SHS-ADDF) method. A multi-node operation, NeoGRRM, has achieved high performance exploration calculations for the large system by submitting SHS-ADDF sub-jobs into many cores in parallel and unifying the results of sub-jobs into the total lists of the main-job. Global exploration of equilibrium (EQ) and transition-state structures at the level of B3LYP/6-31G(d) gave 3210 EQs and 23278 TSs. Nine compounds were found in the low energy regions of 0–100 kJ/mol; the lowest energy compound isN-methylcarbamic acid, the second is methyl carbamate, and the third is glycine (the most fundamental amino acid). Interconversion pathways between the conformers of each of the low energy compounds were surveyed. Isomerization channels around glycine were explored in detail. The lowest energy barriers around some of the EQs turned to be negative after zero-point energy corrections. This indicates that those structures cannot exist as independent structures because they spontaneously collapse into more stable structures. The global PES search showed various interesting dissociating channels which indicate synthon reaction pathways in the reverse directions.},
  archive      = {J_JCC},
  author       = {Koichi Ohno and Naoki Kishimoto and Takeaki Iwamoto and Hiroko Satoh and Hiromasa Watanabe},
  doi          = {10.1002/jcc.26446},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {192-204},
  shortjournal = {J. Comput. Chem.},
  title        = {High performance global exploration of isomers and isomerization channels on quantum chemical potential energy surface of H5C2NO2},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comparative DFT study of vibrational, electronic, and
optical properties of energetic alkali metal salts based on
nitrogen-rich 5-aminotetrazole. <em>JCC</em>, <em>42</em>(3), 180–191.
(<a href="https://doi.org/10.1002/jcc.26445">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This article presents a thorough density functional theory based comparative study on nitrogen-rich 5-aminotetrazole alkali metal salts M 5-At (M = Li, K, Rb, Cs). The calculated structural parameters using plane-wave pseudopotential method are consistent with the experimental results. The computed vibrational frequencies at ambient pressure show that vibrational modes in high energy region are due the NH bond of NH2group. Pressure variation IR spectra of these materials show clear frequency shifts where Li 5-At shows an overall red shift below 900 cm−1contrary to the blue shift seen in other materials in this range. The born effective charge values reveal the presence of strong covalency between N, H, and C atoms whereas an increased ionic nature is seen as the atomic number of metal atoms increases. Furthermore, we used full potential linear augmented plane wave (FP-LAPW) method for calculating electronic structure and optical properties with TB-mBJ potential which provides an enhanced band gap for all materials compared to standard GGA functional. Electronic structure calculation reveals that all the compounds are indirect band gap insulators with the exception of Li 5-At. The computed partial density of states show mixed ionic-covalent nature in metal-N/C bonds and covalent nature in NC bonds. In addition, we are also presenting the optical properties such as real and imaginary dielectric constant, absorption, refraction, reflection, loss spectrum as functions of photon energy. From the optical properties we can conclude that all the studied compounds are optically anisotropic in nature and are good absorbers in the ultraviolet (UV) region.},
  archive      = {J_JCC},
  author       = {Prathap Kumar Jharapla and Subrata Mondal and Ganapathy Vaitheeswaran},
  doi          = {10.1002/jcc.26445},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {180-191},
  shortjournal = {J. Comput. Chem.},
  title        = {Comparative DFT study of vibrational, electronic, and optical properties of energetic alkali metal salts based on nitrogen-rich 5-aminotetrazole},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A model electronic hamiltonian to describe low-lying d–d and
metal-to-ligand charge-transfer excited states of [fe(bpy)3]2+.
<em>JCC</em>, <em>42</em>(3), 166–179. (<a
href="https://doi.org/10.1002/jcc.26444">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A simple practical method to compute both d–d and metal-to-ligand charge-transfer (MLCT) excited states of iron(II) polypyridyl complexes is proposed for use in simulation studies. Specifically, a model electronic Hamiltonian developed previously for d–d excited states of [Fe(bpy)3]2+is extended to deal with low-lying MLCT excited states simultaneously by including the MLCT electronic configurations into the basis functions of the model Hamiltonian. As a first attempt, parameters in the model Hamiltonian matrix elements are determined by using density functional theory (DFT) and time-dependent (TD-)DFT calculation results as benchmarks. To examine the performance of the model Hamiltonian, the potential energy curves along the interpolation between the lowest singlet and quintet state structures are compared to those from the (TD-)DFT calculations and to those from CASPT2 calculations in literature. The electronic absorption spectrum computed through molecular dynamics simulation is compared to the experimental spectrum. The spin-orbit couplings at the ground state structure are also compared to those from wavefunction-based ab initio electronic structure calculations. The results indicate that the constructed model Hamiltonian provides reasonable information on both the low-lying d–d and MLCT excited states of [Fe(bpy)3]2+.},
  archive      = {J_JCC},
  author       = {Satoru Iuchi and Nobuaki Koga},
  doi          = {10.1002/jcc.26444},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {166-179},
  shortjournal = {J. Comput. Chem.},
  title        = {A model electronic hamiltonian to describe low-lying d–d and metal-to-ligand charge-transfer excited states of [Fe(bpy)3]2+},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Refined metadynamics through canonical sampling using
time-invariant bias potential: A study of polyalcohol dehydration in hot
acidic solutions. <em>JCC</em>, <em>42</em>(3), 156–165. (<a
href="https://doi.org/10.1002/jcc.26443">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We propose a canonical sampling method to refine metadynamics simulations a posteriori, where the hills obtained from metadynamics are used as a time-invariant bias potential. In this way, the statistical error in the computed reaction barriers is reduced by an efficient sampling of the collective variable space at the free energy level of interest. This simple approach could be useful particularly when two or more free energy barriers are to be compared among chemical reactions in different or competing conditions. The method was then applied to study the acid dependence of polyalcohol dehydration reactions in high-temperature aqueous solutions. It was found that the reaction proceeds consistently via an SN2 mechanism, whereby the free energy of protonation of the hydroxyl group created as an intermediate is affected significantly by the acidic species. Although demonstration is shown for a specific problem, the computational method suggested herein could be generally used for simulations of complex reactions in the condensed phase.},
  archive      = {J_JCC},
  author       = {Tomomi Kondo and Takehiko Sasaki and Sergi Ruiz-Barragan and Jordi Ribas-Ariño and Motoyuki Shiga},
  doi          = {10.1002/jcc.26443},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {156-165},
  shortjournal = {J. Comput. Chem.},
  title        = {Refined metadynamics through canonical sampling using time-invariant bias potential: A study of polyalcohol dehydration in hot acidic solutions},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Assessment of local coupled cluster methods for excited
states of BODIPY/aza-BODIPY families. <em>JCC</em>, <em>42</em>(3),
144–155. (<a href="https://doi.org/10.1002/jcc.26442">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It was previously reported that Laplace transformed local CC2 (LCC2*) provided the best agreement (MAE = 0.145 eV) when comparing vertical excitation energies to experimentalλmaxfor a benchmark set of 17 BODIPY/Aza-BODIPY molecules. However, these energies did not agree with values obtained from canonical CC2. Here we report LCC2* computations of vertical excitation energies on the same benchmark set of molecules using a newly implemented treatment of the ground state. Comparison with resolution-of-identity approximate coupled cluster to second-order (RI-CC2) results demonstrate that the new LCC2* results agree quantitatively. Furthermore, these values can easily be corrected empirically to also provide excellent agreement with the experiment. We show that the local algebraic diagrammatic construction to second-order (LADC(2)) method exhibits the same differences between implementations as seen for LCC2. The source of the difference is traced to an improved treatment of the ground state in the local methods, which decreases agreement with the experiment (as attributed to a fortuitous cancellation of errors) but significantly improves agreement with RI-CC2. While the absolute vertical excitation energies now show larger deviations, there remains a strong linear correlation between the LCC2* results and the experiment. For the 17 BODIPY/Aza-BODIPY molecules vertical excitation energies are determined using DLPNO-STEOM-CCSD and shown to have excellent agreement with experimentalλmax(MAE = 0.145 eV), which is the best of all the single-reference methods. The vertical excitation energies are determined using LCC2*, empirically corrected LCC2*, and RI-CC2 for a series of eight large BODIPYs and Aza-BODIPYs.},
  archive      = {J_JCC},
  author       = {Milica Feldt and Alex Brown},
  doi          = {10.1002/jcc.26442},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {144-155},
  shortjournal = {J. Comput. Chem.},
  title        = {Assessment of local coupled cluster methods for excited states of BODIPY/Aza-BODIPY families},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). De novo generation of optically active small organic
molecules using monte carlo tree search combined with recurrent neural
network. <em>JCC</em>, <em>42</em>(3), 136–143. (<a
href="https://doi.org/10.1002/jcc.26441">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optically active small organic molecules are computationally designed using the ChemTS python library developed by Tsuda and collaborators, which utilizes a combined Monte Carlo tree search (MCTS) and recurrent neural network model. Geometry optimization and excited-state calculations are performed for each generated molecule, following which the excitation energy and dissymmetry factors are computed to evaluate the score function in the MCTS process. Using this procedure, molecules not contained in existing databases are generated. Molecules having either high dissymmetry factors or high transition dipole strengths can be generated depending on the choice of the score function. In a single trajectory with 100,000 trials, mutually similar high-scoring molecules are generated frequently after the initial 15,000–20,000 trials. This indicates that it is better to sample high-scoring molecules from several trajectories having a modest number of trials each than from a single trajectory having a large number of trials.},
  archive      = {J_JCC},
  author       = {Motomichi Tashiro and Yutaka Imamura and Michio Katouda},
  doi          = {10.1002/jcc.26441},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {136-143},
  shortjournal = {J. Comput. Chem.},
  title        = {De novo generation of optically active small organic molecules using monte carlo tree search combined with recurrent neural network},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). On the stability and chemical bond of noble gas halide
cations NgX+ (ng = he – rn; x = f – i). <em>JCC</em>, <em>42</em>(2),
124–129. (<a href="https://doi.org/10.1002/jcc.26440">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the belief that noble gases (Ng) are completely inert and cannot form stable molecules, a variety of Ng compounds have been reported under laboratory conditions and others were recently detected in the interstellar media, raising interest in knowing and studying their bond nature and the physicochemical properties associated with their stability. In the present work, a systematic analysis of the thermodynamic stability of noble gas halide cations (NgX+) at the CCSD(T)/def2-QZVP level have been performed. In addition, chemical bond was characterized through Natural Bonding Theory (NBO), Quantum Theory of Atoms in Molecules (QTAIM) and Energy Decomposition Analysis (EDA) with relativistic corrections. All methods suggest that NgX+compounds possess a strong covalent bond. However, results show that only compounds containing Ar–Rn atoms are thermodynamic stable with a highly energetic and endergonic dissociation process. For these reasons, it is possible to suggest that several compounds that have not yet been reported could be obtained at the laboratory level or observed in the interstellar medium.},
  archive      = {J_JCC},
  author       = {Lily Arrué and Ricardo Pino-Rios},
  doi          = {10.1002/jcc.26440},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {124-129},
  shortjournal = {J. Comput. Chem.},
  title        = {On the stability and chemical bond of noble gas halide cations NgX+ (Ng = he – rn; x = f – i)},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Estimating the ligand-binding affinity via λ-dependent
umbrella sampling simulations. <em>JCC</em>, <em>42</em>(2), 117–123.
(<a href="https://doi.org/10.1002/jcc.26439">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The umbrella sampling (US) approach has been demonstrated to be a very efficient method for estimating the ligand-binding affinity. However, most of the calculated values overestimate experimental ones that are probably caused by the inaccurate representation of the interaction between the ligand and the surrounding molecules. The issue can be resolved via the implementation aspects ofλ-alteration simulation into the US approach, which we call theλ-dependent umbrella sampling (λUS) scheme. In particular, the electrostatic and van der Waals interactions were simultaneously changed by using the coupling parameterλduringλUS simulations. The mean value of obtained results,kcal mol−1, is in good fitting to the mean value of respective experiments,∆GEXP=  − 11.26 ± 0.89kcal mol−1. Moreover, the correlation between the proposed approach and experiment is quite good with a value of. TheλUS scheme significantly enhances the calculated accuracy since the RMSE of the proposed scheme is smaller than traditional US simulations,kcal mol−1versuskcal mol−1. Furthermore, the precision is increased since the computed error viaλUS approach,kcal mol−1, was smaller than those of the US simulation,kcal mol−1. Overall, the proposed approach perhaps provides an efficient way to accurately and precisely estimate the ligand-binding free energy.},
  archive      = {J_JCC},
  author       = {Son Tung Ngo},
  doi          = {10.1002/jcc.26439},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {117-123},
  shortjournal = {J. Comput. Chem.},
  title        = {Estimating the ligand-binding affinity via λ-dependent umbrella sampling simulations},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). On the many-body nature of intramolecular forces in FFLUX
and its implications. <em>JCC</em>, <em>42</em>(2), 107–116. (<a
href="https://doi.org/10.1002/jcc.26438">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {FFLUX is a biomolecular force field under construction, based on Quantum Chemical Topology (QCT) and machine learning (kriging), with a minimalistic and physically motivated design. A detailed analysis of the forces within the kriging models as treated in FFLUX is presented, taking as a test example a liquid water model. The energies of topological atoms are modeled as 3Natoms-6 dimensional potential energy surfaces, using atomic local frames to represent the internal degrees of freedom. As a result, the forces within the kriging models in FFLUX are inherently N-body in nature whereNrefers toNatoms. This provides a fuller picture that is closer to a true quantum mechanical representation of interactions between atoms. The presented computational example quantitatively showcases the non-negligible (as much as 9\%) three-body nature of bonded forces and angular forces in a water molecule. We discuss the practical impact on the pressure calculation with N-body forces and periodic boundary conditions (PBC) in molecular dynamics, as opposed to classical force fields with two-body forces. The equivalence between the PBC-related correction terms in the general virial equation is shown mathematically.},
  archive      = {J_JCC},
  author       = {Anton Konovalov and Benjamin C.B. Symons and Paul L.A. Popelier},
  doi          = {10.1002/jcc.26438},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {107-116},
  shortjournal = {J. Comput. Chem.},
  title        = {On the many-body nature of intramolecular forces in FFLUX and its implications},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Intermolecular interactions of the extended recognition site
of VIM-2 metallo-β-lactamase with 1,2,4-triazole-3-thione inhibitors.
Validations of a polarizable molecular mechanics potential by ab initio
QC. <em>JCC</em>, <em>42</em>(2), 86–106. (<a
href="https://doi.org/10.1002/jcc.26437">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular dynamics on the complexes of inhibitors with Zn-metalloproteins are a privileged area of applications of polarizable molecular mechanics potentials. With which accuracy could these reproduce the QC intermolecular interaction energies in the two mono-zinc cores and in the dizinc core, toward full-fledged MD simulations on the entire protein complexes? We considered the complexes of the extended recognition site of a Zn-dependent metallo-β-lactamase, VIM-2, produced by bacteria responsible for nosocomial infections, with five newly synthesized inhibitors sharing an original dizinc binding group, 1,2,4-triazole-3-thione (TZT). We considered the energy-minimized structures of each of the five VIM-2 complexes obtained with the SIBFA potential. Energy decomposition analyses (EDA) at the HF level enabled to compare the QC and the SIBFA ΔEvalues and their contributions in the zinc cores, with and without TZT, totaling 30 complexes. With one exception, the ΔE(QC) values were reproduced with relative errors &lt;1.5\%. We next considered the complex of the entire inhibitors with an extended model of VIM-2 recognition site, totaling up to 280 atoms. ΔE(SIBFA) could closely reproduce ΔE(QC). EDA analyses were resumed on the complexes of each inhibitor arm with its interacting VIM-2 residues. As a last step, EDA results at correlated levels were analyzed for the mono- and dizinc sites enabling comparisons with dispersion-augmented ΔE(SIBFA) and correlated multipoles and polarizabilities. Closely reproducing ΔE(QC) and the contrasting trends of its individual contributions should enable for dependable free energy perturbation studies and comparisons to recent experimental ΔGvalues, limiting as much as possible the reliance on error compensations.},
  archive      = {J_JCC},
  author       = {Karolina Kwapien and Laurent Gavara and Jean-Denis Docquier and Dorothée Berthomieu and Jean-François Hernandez and Nohad Gresh},
  doi          = {10.1002/jcc.26437},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {86-106},
  shortjournal = {J. Comput. Chem.},
  title        = {Intermolecular interactions of the extended recognition site of VIM-2 metallo-β-lactamase with 1,2,4-triazole-3-thione inhibitors. validations of a polarizable molecular mechanics potential by ab initio QC},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The active e4 structure of nitrogenase studied with
different DFT functionals. <em>JCC</em>, <em>42</em>(2), 81–85. (<a
href="https://doi.org/10.1002/jcc.26435">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The present study concerns the technical aspects of obtaining the energetics for the E4 state of nitrogenase, the enzyme that fixes N2in nature. EPR experiments have shown that the critical E4 structure that activates N2should contain two bridging hydrides in the FeMo-cofactor. It is furthermore in equilibrium with a structure where the two hydrides have been released and N2binds. These observations led to the suggestion that E4 should have two bridging hydrides and two protonated sulfides. It is important to note that the structure for E4 has not been determined, but only suggested. For a long time, no DFT study led to the suggested structure, independent of which functional was used. However, in two recent DFT studies a good agreement with the experimental suggestion was claimed to have been obtained. In one of them the TPSS functional was used. That was the first out of 11 functionals tried that led to the experimentally suggested structure. In the second of the recent DFT studies, a similar conclusion was reached using the TPSSh functional. The conclusions in the recent studies have here been studied in detail, by calculating a critical energetic value strongly implied by the same EPR experiments. Both the TPSS and TPSSh functionals have been used. The present calculations suggest that those DFT functionals would not lead to agreement with the experimental EPR results either.},
  archive      = {J_JCC},
  author       = {Wen-Jie Wei and Per E. M. Siegbahn},
  doi          = {10.1002/jcc.26435},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {81-85},
  shortjournal = {J. Comput. Chem.},
  title        = {The active e4 structure of nitrogenase studied with different DFT functionals},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Structure, stability and bonding of the leapfrog b240,±1,±2.
<em>JCC</em>, <em>42</em>(2), 72–80. (<a
href="https://doi.org/10.1002/jcc.26434">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two new structural motifs of the B24clusters are constructed by use of the leapfrog transformation. The resulting leapfrog B24has either a bowl shape with a square vacancy or a quasi-planar 2D close-packed triangular boron sheet. The neutral and ionic forms of the latter are found to be more stable than their homologous leapfrog bowl clusters, with the exception of the dicationic B24+2. While the leapfrog isomer is less stable than the tubular double ring in the neutral state, it becomes competitive in some ionic states. The nucleus independent chemical shift, electron localization function, ring current maps and the electronic structure of leapfrog B24clusters reveal them to behave as aromatics.},
  archive      = {J_JCC},
  author       = {Jules Tshishimbi Muya and Bienfait Kabuyaya Isamura and Issofa Patouossa and Minh Tho Nguyen and Arnout Ceulemans},
  doi          = {10.1002/jcc.26434},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {72-80},
  shortjournal = {J. Comput. Chem.},
  title        = {Structure, stability and bonding of the leapfrog b240,±1,±2},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Cover image. <em>JCC</em>, <em>42</em>(1), i. (<a
href="https://doi.org/10.1002/jcc.26225">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Shuichi Ebisawa and Takuro Tsutsumi and Tetsuya Taketsugu},
  doi          = {10.1002/jcc.26225},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Interaction of graphene with antipsychotic drugs: Is there
any charge transfer process? <em>JCC</em>, <em>42</em>(1), 60–65. (<a
href="https://doi.org/10.1002/jcc.26433">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antipsychotics represent an effective therapy for schizophrenia (a chronic mental disorder). Their benefits are related to the interaction of the drugs with dopamine D2 receptor (D2R). Antipsychotics are classified as agonists or antagonists. One of the working hypotheses is that there is a charge transfer process between the drugs and the receptors, which is different for agonists and antagonists. To have more insight into the nature of the interaction of these molecules and the differences between agonists and antagonists, we analyze the interaction of graphene with three molecules: dopamine, pramipexole (an agonist of dopamine), and risperidone (an antagonist of dopamine). The idea is to use graphene as a simple model to analyze the charge transfer process of these three drugs. Optimized structures, atomic charges, and Density of States results indicate that global charges of dopamine and pramipexole are similar, while for risperidone, it is more than double. Pramipexole is an agonist, and the charge transfer process is similar to that of dopamine. Risperidone is an antagonist, and the charge transfer process is different from dopamine. The charge transfer is more significant with risperidone than with dopamine, and this could be related to the mechanism of action. This is in agreement with the working hypotheses that establish that it is possible to distinguish between agonists and antagonists since they have different capacity to transfer charge.},
  archive      = {J_JCC},
  author       = {Roxana M. del Castillo and Estrella Ramos and Ana Martínez},
  doi          = {10.1002/jcc.26433},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {60-65},
  shortjournal = {J. Comput. Chem.},
  title        = {Interaction of graphene with antipsychotic drugs: Is there any charge transfer process?},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ProteinUnet—an efficient alternative to SPIDER3-single for
sequence-based prediction of protein secondary structures. <em>JCC</em>,
<em>42</em>(1), 50–59. (<a
href="https://doi.org/10.1002/jcc.26432">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting protein function and structure from sequence remains an unsolved problem in bioinformatics. The best performing methods rely heavily on evolutionary information from multiple sequence alignments, which means their accuracy deteriorates for sequences with a few homologs, and given the increasing sequence database sizes requires long computation times. Here, a single-sequence-based prediction method is presented, called ProteinUnet, leveraging an U-Net convolutional network architecture. It is compared to SPIDER3-Single model, based on long short-term memory-bidirectional recurrent neural networks architecture. Both methods achieve similar results for prediction of secondary structures (both three- and eight-state), half-sphere exposure, and contact number, but ProteinUnet has two times fewer parameters, 17 times shorter inference time, and can be trained 11 times faster. Moreover, ProteinUnet tends to be better for short sequences and residues with a low number of local contacts. Additionally, the method of loss weighting is presented as an effective way of increasing accuracy for rare secondary structures.},
  archive      = {J_JCC},
  author       = {Krzysztof Kotowski and Tomasz Smolarczyk and Irena Roterman-Konieczna and Katarzyna Stapor},
  doi          = {10.1002/jcc.26432},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {50-59},
  shortjournal = {J. Comput. Chem.},
  title        = {ProteinUnet—An efficient alternative to SPIDER3-single for sequence-based prediction of protein secondary structures},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Bonding and metastability for group 12 dications.
<em>JCC</em>, <em>42</em>(1), 40–49. (<a
href="https://doi.org/10.1002/jcc.26431">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic structure and bonding properties of the Group 12 dications M22+(M = Zn, Cd, Hg) are investigated and electron density-derived quantities are used to characterize the metastability of these species. Of particular interest are the complementary descriptions afforded by the Laplacian of the electron density∇2ρ(r)and the one-electron Bohm quantum potential (Q=) along the bond path. Further, properties derived from the pair density including the localization-delocalization matrices (LDMs) and the interacting quantum atoms (IQA) energies are analyzed within the framework of the quantum theory of atoms in molecules (QTAIM). From the crossing points of the singlet (ground) and triplet (excited) potential energy curves, the barriers for dissociation (BFD) are estimated to be 25.2 kcal/mol (1.09 eV) for Zn22+, 22.8 kcal/mol (0.99 eV) for Cd22+, and 26.4 kcal/mol (1.14 eV) for Hg22+. For comparison and benchmarking purposes, the case of N22+is considered as a texbook example of metastability. At the equilibrium geometries, LDMs, which are used here as an electronic fingerprinting tool, discriminate and group together Group 12 M22+from its isoelectronic Group 11 M2. While “classical” bonding indices are inconclusive in establishing regions of metastability in the bonding, it is shown that the one-electron Bohm quantum potential is promising in this regard.},
  archive      = {J_JCC},
  author       = {SeyedAbdolreza Sadjadi and Chérif F. Matta and Ian P. Hamilton},
  doi          = {10.1002/jcc.26431},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {40-49},
  shortjournal = {J. Comput. Chem.},
  title        = {Bonding and metastability for group 12 dications},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Geometric analysis of anharmonic downward distortion
following paths. <em>JCC</em>, <em>42</em>(1), 27–39. (<a
href="https://doi.org/10.1002/jcc.26430">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A mathematical aspect of the anharmonic downward distortion following (ADDF) path is discussed. The ADDF method is utilized as an automated reaction path search method, which can explore transition state geometries on a potential energy surface from a potential minimum. We show that the maximum number of the ADD stationary paths intersecting the potential minimum is2f+ 1− 2, wherefdenotes the degree of freedom of the system. We also show that the bifurcation of the ADD stationary path is essential to detect all the transition states connected from a given minimum. The ADDF computation is demonstrated for aH2Omolecule in which pitchfork bifurcation is observed.},
  archive      = {J_JCC},
  author       = {Shuichi Ebisawa and Takuro Tsutsumi and Tetsuya Taketsugu},
  doi          = {10.1002/jcc.26430},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {27-39},
  shortjournal = {J. Comput. Chem.},
  title        = {Geometric analysis of anharmonic downward distortion following paths},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Atomistic detailed free-energy landscape of intrinsically
disordered protein studied by multi-scale divide-and-conquer molecular
dynamics simulation. <em>JCC</em>, <em>42</em>(1), 19–26. (<a
href="https://doi.org/10.1002/jcc.26429">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Calcineurin (CaN) is a eukaryotic serine/threonine protein phosphatase activated by both Ca2+and calmodulin (CaM), including intrinsically disordered region (IDR). The region undergoes folding into an α-helix form in the presence Ca2+-loaded CaM. To sample the ordered structure of the IDR by conventional all atom model (AAM) molecular dynamics (MD) simulation, the IDR and Ca2+-loaded CaM must be simultaneously treated. However, it is time-consuming task because the coupled folding and binding should include repeated binding and dissociation. Then, in this study, we propose novel multi-scale divide-and-conquer MD (MSDC-MD), which combines AAM-MD and coarse-grained model MD (CGM-MD). To speed up the conformation sampling, MSDC-MD simulation first treats the IDR by CGM to sample conformations from wide conformation space; then, multiple AAM-MD in a limited area is initiated using the resultant CGM conformation, which is reconstructed by homology modeling method. To investigate performance, we sampled the ordered conformation of the IDR using MSDC-MD; the root-mean-square distance (RMSD) with respect to the experimental structure was 2.23 Å.},
  archive      = {J_JCC},
  author       = {Hiromitsu Shimoyama and Yasushige Yonezawa},
  doi          = {10.1002/jcc.26429},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {19-26},
  shortjournal = {J. Comput. Chem.},
  title        = {Atomistic detailed free-energy landscape of intrinsically disordered protein studied by multi-scale divide-and-conquer molecular dynamics simulation},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Atomistic hybrid particle-field molecular dynamics combined
with slip-springs: Restoring entangled dynamics to simulations of
polymer melts. <em>JCC</em>, <em>42</em>(1), 6–18. (<a
href="https://doi.org/10.1002/jcc.26428">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In hybrid particle-field (hPF) simulations (J. Chem. Phys.,2009130, 214106), the entangled dynamics of polymer melts is lost due to chain crossability. Chains cross, because the field-treatment of thenonbondedinteractions makes them effectively soft-core. We introduce a multi-chain slip-spring model (J. Chem. Phys.,2013138, 104907) into the hPF scheme to mimic the topological constraints of entanglements. The structure of the polymer chains is consistent with that of regular molecular dynamics simulations and is not affected by the introduction of slip-springs. Although slight deviations are seen at short times, dynamical properties such as mean-square displacements and reorientational relaxation times are in good agreement with traditional molecular dynamics simulations and theoretical predictions at long times.},
  archive      = {J_JCC},
  author       = {Zhenghao Wu and Andreas Kalogirou and Antonio De Nicola and Giuseppe Milano and Florian Müller-Plathe},
  doi          = {10.1002/jcc.26428},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {6-18},
  shortjournal = {J. Comput. Chem.},
  title        = {Atomistic hybrid particle-field molecular dynamics combined with slip-springs: Restoring entangled dynamics to simulations of polymer melts},
  volume       = {42},
  year         = {2021},
}
</textarea>
</details></li>
</ul>

</body>
</html>
