<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATMI_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natmi---132">NATMI - 132</h2>
<ul>
<li><details>
<summary>
(2020). Exploring the cloud of variable importance for the set of
all good models. <em>NATMI</em>, <em>2</em>(12), 810–824. (<a
href="https://doi.org/10.1038/s42256-020-00264-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Variable importance is central to scientific studies, including the social sciences and causal inference, healthcare and other domains. However, current notions of variable importance are often tied to a specific predictive model. This is problematic: what if there were multiple well-performing predictive models, and a specific variable is important to some of them but not to others? In that case, we cannot tell from a single well-performing model if a variable is always important, sometimes important, never important or perhaps only important when another variable is not important. Ideally, we would like to explore variable importance for all approximately equally accurate predictive models within the same model class. In this way, we can understand the importance of a variable in the context of other variables, and for many good models. This work introduces the concept of a variable importance cloud, which maps every variable to its importance for every good predictive model. We show properties of the variable importance cloud and draw connections to other areas of statistics. We introduce variable importance diagrams as a projection of the variable importance cloud into two dimensions for visualization purposes. Experiments with criminal justice, marketing data and image classification tasks illustrate how variables can change dramatically in importance for approximately equally accurate predictive models. A set of predictive models can exist that predict equally well; however, the specific variables underlying these models may be important to some of them but not to others. Jiayun Dong and Cynthia Rudin demonstrate a method to visualize and quantify this effect of variable importance.},
  archive      = {J_NATMI},
  author       = {Dong, Jiayun and Rudin, Cynthia},
  doi          = {10.1038/s42256-020-00264-0},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {810-824},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Exploring the cloud of variable importance for the set of all good models},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Gene set inference from single-cell sequencing data using a
hybrid of matrix factorization and variational autoencoders.
<em>NATMI</em>, <em>2</em>(12), 800–809. (<a
href="https://doi.org/10.1038/s42256-020-00269-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in single-cell RNA sequencing have driven the simultaneous measurement of the expression of thousands of genes in thousands of single cells. These growing datasets allow us to model gene sets in biological networks at an unprecedented level of detail, in spite of heterogeneous cell populations. Here, we propose a deep neural network model that is a hybrid of matrix factorization and variational autoencoders, which we call restricted latent variational autoencoder (resVAE). The model uses weights as factorized matrices to obtain gene sets, while class-specific inputs to the latent variable space facilitate a plausible identification of cell types. This artificial neural network model seamlessly integrates functional gene set inference, experimental covariate effect isolation, and static gene identification, which we conceptually demonstrate here for four single-cell RNA sequencing datasets. The wealth of data generated by single-cell RNA sequencing can be used to identify gene sets across cells, as well as to identify specific cells. Lukassen and colleagues propose a method combining matrix factorization and variational auto encoders that can capture both cross-cell and cell-specific information.},
  archive      = {J_NATMI},
  author       = {Lukassen, Soeren and Ten, Foo Wei and Adam, Lukas and Eils, Roland and Conrad, Christian},
  doi          = {10.1038/s42256-020-00269-9},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {800-809},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Gene set inference from single-cell sequencing data using a hybrid of matrix factorization and variational autoencoders},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Network of evolvable neural units can learn synaptic
learning rules and spiking dynamics. <em>NATMI</em>, <em>2</em>(12),
791–799. (<a href="https://doi.org/10.1038/s42256-020-00267-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although deep neural networks have seen great success in recent years through various changes in overall architectures and optimization strategies, their fundamental underlying design remains largely unchanged. Computational neuroscience may provide more biologically realistic models of neural processing mechanisms, but they are still high-level abstractions of empirical behaviour. Here we propose an evolvable neural unit (ENU) that can evolve individual somatic and synaptic compartment models of neurons in a scalable manner. We demonstrate that ENUs can evolve to mimic integrate-and-fire neurons and synaptic spike-timing-dependent plasticity. Furthermore, by constructing a network where an ENU takes the place of each synapse and neuron, we evolve an agent capable of learning to solve a T-maze environment task. This network independently discovers spiking dynamics and reinforcement-type learning rules, opening up a new path towards biologically inspired artificial intelligence. Bertens and Lee propose an evolvable neural unit, a recurrent neural network-based module that can evolve individual somatic and synaptic compartment models of neurons. By constructing networks of these evolvable neural units, they can evolve agents that learn synaptic update rules and the spiking dynamics of neurons.},
  archive      = {J_NATMI},
  author       = {Bertens, Paul and Lee, Seong-Whan},
  doi          = {10.1038/s42256-020-00267-x},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {791-799},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Network of evolvable neural units can learn synaptic learning rules and spiking dynamics},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Search and rescue with airborne optical sectioning.
<em>NATMI</em>, <em>2</em>(12), 783–790. (<a
href="https://doi.org/10.1038/s42256-020-00261-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the future, rescuing lost, ill or injured persons will increasingly be carried out by autonomous drones. However, discovering humans in densely forested terrain is challenging because of occlusion, and robust detection mechanisms are required. We show that automated person detection under occlusion conditions can be notably improved by combining multi-perspective images before classification. Here, we employ image integration by airborne optical sectioning (AOS)—a synthetic aperture imaging technique that uses camera drones to capture unstructured thermal light fields—to achieve this with a precision and recall of 96\% and 93\%, respectively. Finding lost or injured people in dense forests is not generally feasible with thermal recordings, but becomes practical with the use of AOS integral images. Our findings lay the foundation for effective future search-and-rescue technologies that can be applied in combination with autonomous or manned aircraft. They can also be beneficial for other fields that currently suffer from inaccurate classification of partially occluded people, animals or objects. Autonomous drones can help find injured or missing people when a large or hard to traverse area has to be searched, but their view can be obscured in dense forests. David Schedl and colleagues have developed a method to reveal humans in thermal imaging recordings, even in the presence of dense foliage.},
  archive      = {J_NATMI},
  author       = {Schedl, David C. and Kurmi, Indrajit and Bimber, Oliver},
  doi          = {10.1038/s42256-020-00261-3},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {783-790},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Search and rescue with airborne optical sectioning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Concept whitening for interpretable image recognition.
<em>NATMI</em>, <em>2</em>(12), 772–782. (<a
href="https://doi.org/10.1038/s42256-020-00265-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What does a neural network encode about a concept as we traverse through the layers? Interpretability in machine learning is undoubtedly important, but the calculations of neural networks are very challenging to understand. Attempts to see inside their hidden layers can be misleading, unusable or rely on the latent space to possess properties that it may not have. Here, rather than attempting to analyse a neural network post hoc, we introduce a mechanism, called concept whitening (CW), to alter a given layer of the network to allow us to better understand the computation leading up to that layer. When a concept whitening module is added to a convolutional neural network, the latent space is whitened (that is, decorrelated and normalized) and the axes of the latent space are aligned with known concepts of interest. By experiment, we show that CW can provide us with a much clearer understanding of how the network gradually learns concepts over layers. CW is an alternative to a batch normalization layer in that it normalizes, and also decorrelates (whitens), the latent space. CW can be used in any layer of the network without hurting predictive performance. There is much interest in ‘explainable’ AI, but most efforts concern post hoc methods. Instead, a neural network can be made inherently interpretable, with an approach that involves making human-understandable concepts (aeroplane, bed, lamp and so on) align along the axes of its latent space.},
  archive      = {J_NATMI},
  author       = {Chen, Zhi and Bei, Yijie and Rudin, Cynthia},
  doi          = {10.1038/s42256-020-00265-z},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {772-782},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Concept whitening for interpretable image recognition},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Personalized deep learning of individual immunopeptidomes to
identify neoantigens for cancer vaccines. <em>NATMI</em>,
<em>2</em>(12), 764–771. (<a
href="https://doi.org/10.1038/s42256-020-00260-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumour-specific neoantigens play a major role for developing personal vaccines in cancer immunotherapy. We propose a personalized de novo peptide sequencing workflow to identify HLA-I and HLA-II neoantigens directly and solely from mass spectrometry data. Our workflow trains a personal deep learning model on the immunopeptidome of an individual patient and then uses it to predict mutated neoantigens of that patient. This personalized learning and mass spectrometry-based approach enables comprehensive and accurate identification of neoantigens. We applied the workflow to datasets of five patients with melanoma and expanded their predicted immunopeptidomes by 5–15\%. Subsequently, we discovered neoantigens of both HLA-I and HLA-II, including those with validated T-cell responses and those that had not been reported in previous studies. Neoantigens play a critical role in cancer immunotherapy. Tran et al. show how training a personalized deep learning model for each individual patient can improve the accuracy and identification rate of mutated neoantigens.},
  archive      = {J_NATMI},
  author       = {Tran, Ngoc Hieu and Qiao, Rui and Xin, Lei and Chen, Xin and Shan, Baozhen and Li, Ming},
  doi          = {10.1038/s42256-020-00260-4},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {764-771},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Personalized deep learning of individual immunopeptidomes to identify neoantigens for cancer vaccines},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evolutionary training and abstraction yields algorithmic
generalization of neural computers. <em>NATMI</em>, <em>2</em>(12),
753–763. (<a href="https://doi.org/10.1038/s42256-020-00255-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key feature of intelligent behaviour is the ability to learn abstract strategies that scale and transfer to unfamiliar problems. An abstract strategy solves every sample from a problem class, no matter its representation or complexity—similar to algorithms in computer science. Neural networks are powerful models for processing sensory data, discovering hidden patterns and learning complex functions, but they struggle to learn such iterative, sequential or hierarchical algorithmic strategies. Extending neural networks with external memories has increased their capacities to learn such strategies, but they are still prone to data variations, struggle to learn scalable and transferable solutions, and require massive training data. We present the neural Harvard computer, a memory-augmented network-based architecture that employs abstraction by decoupling algorithmic operations from data manipulations, realized by splitting the information flow and separated modules. This abstraction mechanism and evolutionary training enable the learning of robust and scalable algorithmic solutions. On a diverse set of 11 algorithms with varying complexities, we show that the neural Harvard computer reliably learns algorithmic solutions with strong generalization and abstraction, achieves perfect generalization and scaling to arbitrary task configurations and complexities far beyond seen during training, and independence of the data representation and the task domain. A hallmark of intelligent behaviour is the ability to learn abstract strategies that can be transferred across different tasks, but it has been challenging to incorporate this ability in artificial systems. The authors present a modular architecture for the learning of algorithmic solutions, and demonstrate generalization and scaling on 11 diverse algorithms.},
  archive      = {J_NATMI},
  author       = {Tanneberg, Daniel and Rueckert, Elmar and Peters, Jan},
  doi          = {10.1038/s42256-020-00255-1},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {753-763},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Evolutionary training and abstraction yields algorithmic generalization of neural computers},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reusability report: Designing organic photoelectronic
molecules with descriptor conditional recurrent neural networks.
<em>NATMI</em>, <em>2</em>(12), 749–752. (<a
href="https://doi.org/10.1038/s42256-020-00268-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Mohapatra, Somesh and Yang, Tzuhsiung and Gómez-Bombarelli, Rafael},
  doi          = {10.1038/s42256-020-00268-w},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {749-752},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Reusability report: Designing organic photoelectronic molecules with descriptor conditional recurrent neural networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep learning for tomographic image reconstruction.
<em>NATMI</em>, <em>2</em>(12), 737–748. (<a
href="https://doi.org/10.1038/s42256-020-00273-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep-learning-based tomographic imaging is an important application of artificial intelligence and a new frontier of machine learning. Deep learning has been widely used in computer vision and image analysis, which deal with existing images, improve these images, and produce features from them. Since 2016, deep learning techniques have been actively researched for tomographic imaging, especially in the context of biomedicine, with impressive results and great potential. Tomographic reconstruction produces images of multi-dimensional structures from externally measured ‘encoded’ data in the form of various tomographic transforms (integrals, harmonics, echoes and so on). In this Review, we provide a general background, highlight representative results with an emphasis on medical imaging, and discuss key issues that need to be addressed in this emerging field. In particular, tomographic imaging is an integral part of modern medicine, and will play a key role in personalized, preventive and precision medicine and make it intelligent, inexpensive and indiscriminate. The popularity of deep learning is leading to new areas in biomedical applications. Wang and colleagues summarize in this Review the recent development and future directions of deep neural networks for superior image quality in the tomographic imaging field.},
  archive      = {J_NATMI},
  author       = {Wang, Ge and Ye, Jong Chul and De Man, Bruno},
  doi          = {10.1038/s42256-020-00273-z},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {737-748},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning for tomographic image reconstruction},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Understanding adversarial examples requires a theory of
artefacts for deep learning. <em>NATMI</em>, <em>2</em>(12), 731–736.
(<a href="https://doi.org/10.1038/s42256-020-00266-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep neural networks are currently the most widespread and successful technology in artificial intelligence. However, these systems exhibit bewildering new vulnerabilities: most notably a susceptibility to adversarial examples. Here, I review recent empirical research on adversarial examples that suggests that deep neural networks may be detecting in them features that are predictively useful, though inscrutable to humans. To understand the implications of this research, we should contend with some older philosophical puzzles about scientific reasoning, helping us to determine whether these features are reliable targets of scientific investigation or just the distinctive processing artefacts of deep neural networks. DNN classifiers are vulnerable to small, specific perturbations in an input that seem benign to humans. To understand this phenomenon, Buckner argues that it may be necessary to treat the patterns that DNNs detect in these adversarial examples as artefacts, which may contain predictive information.},
  archive      = {J_NATMI},
  author       = {Buckner, Cameron},
  doi          = {10.1038/s42256-020-00266-y},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {731-736},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Understanding adversarial examples requires a theory of artefacts for deep learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Let’s talk about digital death. <em>NATMI</em>,
<em>2</em>(12), 730. (<a
href="https://doi.org/10.1038/s42256-020-00274-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Nowaczyk-Basińska, Katarzyna and Cave, Stephen},
  doi          = {10.1038/s42256-020-00274-y},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {730},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Let’s talk about digital death},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Research, reuse, repeat. <em>NATMI</em>, <em>2</em>(12),
729. (<a href="https://doi.org/10.1038/s42256-020-00277-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A research paper makes the most impact when its methods, data and code are available for others to use and build on. We highlight the benefits of good sharing practices with a new type of article, reusability reports.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-00277-9},
  journal      = {Nature Machine Intelligence},
  number       = {12},
  pages        = {729},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Research, reuse, repeat},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Database-independent molecular
formula annotation using gibbs sampling through ZODIAC. <em>NATMI</em>,
<em>2</em>(11), 727. (<a
href="https://doi.org/10.1038/s42256-020-00259-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NATMI},
  author       = {Ludwig, Marcus and Nothias, Louis-Félix and Dührkop, Kai and Koester, Irina and Fleischauer, Markus and Hoffmann, Martin A. and Petras, Daniel and Vargas, Fernando and Morsy, Mustafa and Aluwihare, Lihini and Dorrestein, Pieter C. and Böcker, Sebastian},
  doi          = {10.1038/s42256-020-00259-x},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {727},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Database-independent molecular formula annotation using gibbs sampling through ZODIAC},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Self-supervised retinal thickness prediction enables deep
learning from unlabelled data to boost classification of diabetic
retinopathy. <em>NATMI</em>, <em>2</em>(11), 719–726. (<a
href="https://doi.org/10.1038/s42256-020-00247-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Access to large, annotated samples represents a considerable challenge for training accurate deep-learning models in medical imaging. Although at present transfer learning from pre-trained models can help with cases lacking data, this limits design choices and generally results in the use of unnecessarily large models. Here we propose a self-supervised training scheme for obtaining high-quality, pre-trained networks from unlabelled, cross-modal medical imaging data, which will allow the creation of accurate and efficient models. We demonstrate the utility of the scheme by accurately predicting retinal thickness measurements based on optical coherence tomography from simple infrared fundus images. Subsequently, learned representations outperformed advanced classifiers on a separate diabetic retinopathy classification task in a scenario of scarce training data. Our cross-modal, three-stage scheme effectively replaced 26,343 diabetic retinopathy annotations with 1,009 semantic segmentations on optical coherence tomography and reached the same classification accuracy using only 25\% of fundus images, without any drawbacks, since optical coherence tomography is not required for predictions. We expect this concept to apply to other multimodal clinical imaging, health records and genomics data, and to corresponding sample-starved learning problems. The thickness of the retina is an important medical indicator for diabetic retinopathy. Holmberg and colleagues present a self-supervised deep-learning method that uses cross-modal data to predict retinal thickness maps from easily obtainable fundus images.},
  archive      = {J_NATMI},
  author       = {Holmberg, Olle G. and Köhler, Niklas D. and Martins, Thiago and Siedlecki, Jakob and Herold, Tina and Keidel, Leonie and Asani, Ben and Schiefelbein, Johannes and Priglinger, Siegfried and Kortuem, Karsten U. and Theis, Fabian J.},
  doi          = {10.1038/s42256-020-00247-1},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {719-726},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Self-supervised retinal thickness prediction enables deep learning from unlabelled data to boost classification of diabetic retinopathy},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Chemically programmable microrobots weaving a web from
hormones. <em>NATMI</em>, <em>2</em>(11), 711–718. (<a
href="https://doi.org/10.1038/s42256-020-00248-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The occurrence of synthetic and natural hormones in an aqueous environment poses significant risks to humans because of their endocrine-disrupting activity. Autonomous self-propelled and remotely actuated nano/microrobots have emerged as a new field that encompasses a wide range of potential applications, including sensing, detection and elimination/degradation of emerging pollutants. In this work, we develop programmable polypyrrole-based (PPy, outer functional layer) microrobots incorporated with a Pt catalytic layer and paramagnetic iron nanoparticles (Fe3O4) to provide self-propulsion and a magnetic response for the highly efficient removal of oestrogenic pollutants. As the pH of the tested water alters, the surface charge of PPy/Fe3O4/Pt microrobots gradually changes, leading to affinity modulation. As microrobots move inside the solution, they collect oestrogen fibres and subsequently weave macroscopic webs on the surface. Our results suggest that motion-controllable microrobots with adjustable surface chemistry could provide a suitable platform for the highly efficient removal of hormonal pollutants. Microrobots are usually too small to contain traditional computing substrates that could control their behaviour. Dekanovsky and colleagues have developed a microrobot swarm that removes hormonal pollutants when it senses a chemical signal in its environment.},
  archive      = {J_NATMI},
  author       = {Dekanovsky, Lukas and Khezri, Bahareh and Rottnerova, Zdeňka and Novotny, Filip and Plutnar, Jan and Pumera, Martin},
  doi          = {10.1038/s42256-020-00248-0},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {711-718},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Chemically programmable microrobots weaving a web from hormones},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Prediction of water stability of metal–organic frameworks
using machine learning. <em>NATMI</em>, <em>2</em>(11), 704–710. (<a
href="https://doi.org/10.1038/s42256-020-00249-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Owing to their highly tunable structures, metal–organic frameworks (MOFs) are considered suitable candidates for a range of applications, including adsorption, separation, sensing and catalysis. However, MOFs must be stable in water vapour to be considered industrially viable. It is currently challenging to predict water stability in MOFs; experiments involve time-intensive MOF synthesis, while modelling techniques do not reliably capture the water stability behaviour. Here, we build a machine learning-based model to accurately and instantly classify MOFs as stable or unstable depending on the target application, or the amount of water exposed. The model is trained using an empirically measured dataset of water stabilities for over 200 MOFs, and uses a comprehensive set of chemical features capturing information about their constituent metal node, organic ligand and metal–ligand molar ratios. In addition to screening stable MOF candidates for future experiments, the trained models were used to extract a number of simple water stability trends in MOFs. This approach is general and can also be used to screen MOFs for other design criteria. Metal–organic frameworks (MOFs) are attractive materials for gas capture, separation, sensing and catalysis. Determining their water stability is important, but time-intensive. Batra et al. use machine learning to screen water-stable MOFs and identify chemical features supporting their stability.},
  archive      = {J_NATMI},
  author       = {Batra, Rohit and Chen, Carmen and Evans, Tania G. and Walton, Krista S. and Ramprasad, Rampi},
  doi          = {10.1038/s42256-020-00249-z},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {704-710},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Prediction of water stability of metal–organic frameworks using machine learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An interpretable deep-learning architecture of capsule
networks for identifying cell-type gene expression programs from
single-cell RNA-sequencing data. <em>NATMI</em>, <em>2</em>(11),
693–703. (<a href="https://doi.org/10.1038/s42256-020-00244-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing (scRNA-seq) technologies are used to characterize the heterogeneity of cells in cell types, developmental stages and spatial positions. The rapid accumulation of scRNA-seq data has enabled single-cell-type labelling to transform single-cell transcriptome analysis. Here we propose an interpretable deep-learning architecture using capsule networks (called scCapsNet). A capsule structure (a neuron vector representing a set of properties of a specific object) captures hierarchical relations. By utilizing competitive single-cell-type recognition, the scCapsNet model is able to perform feature selection to identify groups of genes encoding different subcellular types. The RNA expression signatures, which enable subcellular-type recognition, are effectively integrated into the parameter matrices of scCapsNet. This characteristic enables the discovery of gene regulatory modules in which genes interact with each other and are closely related in function, but present distinct expression patterns. The wealth of data gathered from single-cell RNA sequencing can be processed with deep learning techniques, but often those methods are too opaque to reveal why a single cell is labelled to be a certain cell type. Lifei Wang and colleagues present an RNA-sequencing analysis method that uses capsule networks and is interpretable enough to allow for identification of cell-type-specific genes.},
  archive      = {J_NATMI},
  author       = {Wang, Lifei and Nie, Rui and Yu, Zeyang and Xin, Ruyue and Zheng, Caihong and Zhang, Zhang and Zhang, Jiang and Cai, Jun},
  doi          = {10.1038/s42256-020-00244-4},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {693-703},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An interpretable deep-learning architecture of capsule networks for identifying cell-type gene expression programs from single-cell RNA-sequencing data},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Interpretable socioeconomic status inference from aerial
imagery through urban patterns. <em>NATMI</em>, <em>2</em>(11), 684–692.
(<a href="https://doi.org/10.1038/s42256-020-00243-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Urbanization is a great challenge for modern societies, promising better access to economic opportunities, but widening socioeconomic inequalities. Accurately tracking this process as it unfolds has been challenging for traditional data collection methods, but remote sensing information offers an alternative way to gather a more complete view of these societal changes. By feeding neural networks with satellite images, the socioeconomic information associated with that area can be recovered. However, these models lack the ability to explain how visual features contained in a sample trigger a given prediction. Here, we close this gap by predicting socioeconomic status across France from aerial images and interpreting class activation mappings in terms of urban topology. We show that trained models disregard the spatial correlations existing between urban class and socioeconomic status to derive their predictions. These results pave the way to build more interpretable models, which may help to better track and understand urbanization and its consequences. Neural network models can predict the socioeconomic wealth of an area from aerial views, but fall short of explaining how visual features trigger a given prediction. The authors develop a pipeline for projecting class activation maps onto the underlying urban topology, to help interpret such predictions.},
  archive      = {J_NATMI},
  author       = {Abitbol, Jacob Levy and Karsai, Márton},
  doi          = {10.1038/s42256-020-00243-5},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {684-692},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Interpretable socioeconomic status inference from aerial imagery through urban patterns},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Moving beyond generalization to accurate interpretation of
flexible models. <em>NATMI</em>, <em>2</em>(11), 674–683. (<a
href="https://doi.org/10.1038/s42256-020-00242-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning optimizes flexible models to predict data. In scientific applications, there is a rising interest in interpreting these flexible models to derive hypotheses from data. However, it is unknown whether good data prediction guarantees the accurate interpretation of flexible models. Here, we test this connection using a flexible, yet intrinsically interpretable framework for modelling neural dynamics. We find that many models discovered during optimization predict data equally well, yet they fail to match the correct hypothesis. We develop an alternative approach that identifies models with correct interpretation by comparing model features across data samples to separate true features from noise. We illustrate our findings using recordings of spiking activity from the visual cortex of monkeys performing a fixation task. Our results reveal that good predictions cannot substitute for accurate interpretation of flexible models and offer a principled approach to identify models with correct interpretation. Across disciplines, there is a rising interest in interpreting machine learning models to derive scientific knowledge from data. Genkin and Engel show that models optimized for predicting data can disagree with the ground truth and propose a new model selection principle to prioritize accurate interpretation.},
  archive      = {J_NATMI},
  author       = {Genkin, Mikhail and Engel, Tatiana A.},
  doi          = {10.1038/s42256-020-00242-6},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {674-683},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Moving beyond generalization to accurate interpretation of flexible models},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Shortcut learning in deep neural networks. <em>NATMI</em>,
<em>2</em>(11), 665–673. (<a
href="https://doi.org/10.1038/s42256-020-00257-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning has triggered the current rise of artificial intelligence and is the workhorse of today’s machine intelligence. Numerous success stories have rapidly spread all over science, industry and society, but its limitations have only recently come into focus. In this Perspective we seek to distil how many of deep learning’s failures can be seen as different symptoms of the same underlying problem: shortcut learning. Shortcuts are decision rules that perform well on standard benchmarks but fail to transfer to more challenging testing conditions, such as real-world scenarios. Related issues are known in comparative psychology, education and linguistics, suggesting that shortcut learning may be a common characteristic of learning systems, biological and artificial alike. Based on these observations, we develop a set of recommendations for model interpretation and benchmarking, highlighting recent advances in machine learning to improve robustness and transferability from the lab to real-world applications. Deep learning has resulted in impressive achievements, but under what circumstances does it fail, and why? The authors propose that its failures are a consequence of shortcut learning, a common characteristic across biological and artificial systems in which strategies that appear to have solved a problem fail unexpectedly under different circumstances.},
  archive      = {J_NATMI},
  author       = {Geirhos, Robert and Jacobsen, Jörn-Henrik and Michaelis, Claudio and Zemel, Richard and Brendel, Wieland and Bethge, Matthias and Wichmann, Felix A.},
  doi          = {10.1038/s42256-020-00257-z},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {665-673},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Shortcut learning in deep neural networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Embodied intelligence weaves a better future.
<em>NATMI</em>, <em>2</em>(11), 663–664. (<a
href="https://doi.org/10.1038/s42256-020-00250-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microrobots can interact intelligently with their environment and complete specific tasks by well-designed incorporation of responsive materials. Recent work demonstrates how swarms of microbots with specifically tuned surface chemistry can remove a hormone pollutant from a solution by coalescing it into a web.},
  archive      = {J_NATMI},
  author       = {Jin, Dongdong and Zhang, Li},
  doi          = {10.1038/s42256-020-00250-6},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {663-664},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Embodied intelligence weaves a better future},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). C. Elegans inspires self-driving cars. <em>NATMI</em>,
<em>2</em>(11), 661–662. (<a
href="https://doi.org/10.1038/s42256-020-00245-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autonomous driving technology is improving, although doubts about their reliability remain. Controllers based on compact neural architectures could help improve their interpretability and robustness.},
  archive      = {J_NATMI},
  author       = {Milford, Michael},
  doi          = {10.1038/s42256-020-00245-3},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {661-662},
  shortjournal = {Nat. Mach. Intell.},
  title        = {C. elegans inspires self-driving cars},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Skills for physical artificial intelligence. <em>NATMI</em>,
<em>2</em>(11), 658–660. (<a
href="https://doi.org/10.1038/s42256-020-00258-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Synthesizing robots via physical artificial intelligence is a multidisciplinary challenge for future robotics research. An education methodology is needed for researchers to develop a combination of skills in physical artificial intelligence.},
  archive      = {J_NATMI},
  author       = {Miriyev, Aslan and Kovač, Mirko},
  doi          = {10.1038/s42256-020-00258-y},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {658-660},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Skills for physical artificial intelligence},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). More than a few bad apps. <em>NATMI</em>, <em>2</em>(11),
655–657. (<a href="https://doi.org/10.1038/s42256-020-00246-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Addressing the problems caused by AI applications in society with ethics frameworks is futile until we confront the political structure of such applications.},
  archive      = {J_NATMI},
  author       = {Sadowski, Jathan and Andrejevic, Mark},
  doi          = {10.1038/s42256-020-00246-2},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {655-657},
  shortjournal = {Nat. Mach. Intell.},
  title        = {More than a few bad apps},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Beware of fictional AI narratives. <em>NATMI</em>,
<em>2</em>(11), 654. (<a
href="https://doi.org/10.1038/s42256-020-00256-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Hermann, Isabella},
  doi          = {10.1038/s42256-020-00256-0},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {654},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Beware of fictional AI narratives},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Materializing artificial intelligence. <em>NATMI</em>,
<em>2</em>(11), 653. (<a
href="https://doi.org/10.1038/s42256-020-00262-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence can be defined as intelligence demonstrated by machines. But what counts as intelligence, and how intelligence is implemented in different kinds of machines, robots and software varies across disciplines and over time.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-00262-2},
  journal      = {Nature Machine Intelligence},
  number       = {11},
  pages        = {653},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Materializing artificial intelligence},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Neural circuit policies enabling auditable autonomy.
<em>NATMI</em>, <em>2</em>(10), 642–652. (<a
href="https://doi.org/10.1038/s42256-020-00237-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central goal of artificial intelligence in high-stakes decision-making applications is to design a single algorithm that simultaneously expresses generalizability by learning coherent representations of their world and interpretable explanations of its dynamics. Here, we combine brain-inspired neural computation principles and scalable deep learning architectures to design compact neural controllers for task-specific compartments of a full-stack autonomous vehicle control system. We discover that a single algorithm with 19 control neurons, connecting 32 encapsulated input features to outputs by 253 synapses, learns to map high-dimensional inputs into steering commands. This system shows superior generalizability, interpretability and robustness compared with orders-of-magnitude larger black-box learning systems. The obtained neural agents enable high-fidelity autonomy for task-specific parts of a complex autonomous system. Inspired by the brain of the roundworm Caenorhabditis elegans, the authors design a highly compact neural network controller directly from raw input pixels. Compared with larger networks, this compact controller demonstrates improved generalization, robustness and interpretability on a lane-keeping task.},
  archive      = {J_NATMI},
  author       = {Lechner, Mathias and Hasani, Ramin and Amini, Alexander and Henzinger, Thomas A. and Rus, Daniela and Grosu, Radu},
  doi          = {10.1038/s42256-020-00237-3},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {642-652},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Neural circuit policies enabling auditable autonomy},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Database-independent molecular formula annotation using
gibbs sampling through ZODIAC. <em>NATMI</em>, <em>2</em>(10), 629–641.
(<a href="https://doi.org/10.1038/s42256-020-00234-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The confident high-throughput identification of small molecules is one of the most challenging tasks in mass spectrometry-based metabolomics. Annotating the molecular formula of a compound is the first step towards its structural elucidation. Yet even the annotation of molecular formulas remains highly challenging. This is particularly so for large compounds above 500 daltons, and for de novo annotations, for which we consider all chemically feasible formulas. Here we present ZODIAC, a network-based algorithm for the de novo annotation of molecular formulas. Uniquely, it enables fully automated and swift processing of complete experimental runs, providing high-quality, high-confidence molecular formula annotations. This allows us to annotate novel molecular formulas that are absent from even the largest public structure databases. Our method re-ranks molecular formula candidates by considering joint fragments and losses between fragmentation trees. We employ Bayesian statistics and Gibbs sampling. Thorough algorithm engineering ensures fast processing in practice. We evaluate ZODIAC on five datasets, producing results substantially (up to 16.5-fold) better than for several other methods, including SIRIUS, which is the state-of-the-art algorithm for molecular formula annotation at present. Finally, we report and verify several novel molecular formulas annotated by ZODIAC. To infer a previously unknown molecular formula from mass spectrometry data is a challenging, yet neglected problem. Ludwig and colleagues present a network-based approach to ranking possible formulas.},
  archive      = {J_NATMI},
  author       = {Ludwig, Marcus and Nothias, Louis-Félix and Dührkop, Kai and Koester, Irina and Fleischauer, Markus and Hoffmann, Martin A. and Petras, Daniel and Vargas, Fernando and Morsy, Mustafa and Aluwihare, Lihini and Dorrestein, Pieter C. and Böcker, Sebastian},
  doi          = {10.1038/s42256-020-00234-6},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {629-641},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Database-independent molecular formula annotation using gibbs sampling through ZODIAC},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Integration of mechanistic immunological knowledge into a
machine learning pipeline improves predictions. <em>NATMI</em>,
<em>2</em>(10), 619–628. (<a
href="https://doi.org/10.1038/s42256-020-00232-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dense network of interconnected cellular signalling responses that are quantifiable in peripheral immune cells provides a wealth of actionable immunological insights. Although high-throughput single-cell profiling techniques, including polychromatic flow and mass cytometry, have matured to a point that enables detailed immune profiling of patients in numerous clinical settings, the limited cohort size and high dimensionality of data increase the possibility of false-positive discoveries and model overfitting. We introduce a generalizable machine learning platform, the immunological Elastic-Net (iEN), which incorporates immunological knowledge directly into the predictive models. Importantly, the algorithm maintains the exploratory nature of the high-dimensional dataset, allowing for the inclusion of immune features with strong predictive capabilities even if not consistent with prior knowledge. In three independent studies our method demonstrates improved predictions for clinically relevant outcomes from mass cytometry data generated from whole blood, as well as a large simulated dataset. The iEN is available under an open-source licence. Recent advances have increased the dimensionality and complexity of immunological data. The authors developed a machine learning approach to incorporate prior immunological knowledge and applied it on clinical examples and a simulation study. The approach may be useful for high-dimensional datasets in clinical settings where the cohort size is limited.},
  archive      = {J_NATMI},
  author       = {Culos, Anthony and Tsai, Amy S. and Stanley, Natalie and Becker, Martin and Ghaemi, Mohammad S. and McIlwain, David R. and Fallahzadeh, Ramin and Tanada, Athena and Nassar, Huda and Espinosa, Camilo and Xenochristou, Maria and Ganio, Edward and Peterson, Laura and Han, Xiaoyuan and Stelzer, Ina A. and Ando, Kazuo and Gaudilliere, Dyani and Phongpreecha, Thanaphong and Marić, Ivana and Chang, Alan L. and Shaw, Gary M. and Stevenson, David K. and Bendall, Sean and Davis, Kara L. and Fantl, Wendy and Nolan, Garry P. and Hastie, Trevor and Tibshirani, Robert and Angst, Martin S. and Gaudilliere, Brice and Aghaeepour, Nima},
  doi          = {10.1038/s42256-020-00232-8},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {619-628},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Integration of mechanistic immunological knowledge into a machine learning pipeline improves predictions},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Iterative transfer learning with neural network for
clustering and cell type classification in single-cell RNA-seq analysis.
<em>NATMI</em>, <em>2</em>(10), 607–618. (<a
href="https://doi.org/10.1038/s42256-020-00233-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clustering and cell type classification are important steps in single-cell RNA-seq (scRNA-seq) analysis. As more and more scRNA-seq data are becoming available, supervised cell type classification methods that utilize external well-annotated source data start to gain popularity over unsupervised clustering algorithms; however, the performance of existing supervised methods is highly dependent on source data quality and they often have limited accuracy to classify cell types that are missing in the source data. We developed ItClust to overcome these limitations, a transfer learning algorithm that borrows ideas from supervised cell type classification algorithms, but also leverages information in target data to ensure sensitivity in classifying cells that are only present in the target data. Through extensive evaluations using data from different species and tissues generated with diverse scRNA-seq protocols, we show that ItClust considerably improves clustering and cell type classification accuracy over popular unsupervised clustering and supervised cell type classification algorithms. Classifying cells from single-cell RNA sequences is challenging for cells where only limited data is available. Hu and colleagues show here that a clustering approach using transfer learning can use the variability of one dataset to cluster a smaller target dataset with high-quality results.},
  archive      = {J_NATMI},
  author       = {Hu, Jian and Li, Xiangjie and Hu, Gang and Lyu, Yafei and Susztak, Katalin and Li, Mingyao},
  doi          = {10.1038/s42256-020-00233-7},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {607-618},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Iterative transfer learning with neural network for clustering and cell type classification in single-cell RNA-seq analysis},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Enabling the future of colonoscopy with intelligent and
autonomous magnetic manipulation. <em>NATMI</em>, <em>2</em>(10),
595–606. (<a href="https://doi.org/10.1038/s42256-020-00231-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Early diagnosis of colorectal cancer substantially improves survival. However, over half of cases are diagnosed late due to the demand for colonoscopy—the ‘gold standard’ for screening—exceeding capacity. Colonoscopy is limited by the outdated design of conventional endoscopes, which are associated with high complexity of use, cost and pain. Magnetic endoscopes are a promising alternative and overcome the drawbacks of pain and cost, but they struggle to reach the translational stage as magnetic manipulation is complex and unintuitive. In this work, we use machine vision to develop intelligent and autonomous control of a magnetic endoscope, enabling non-expert users to effectively perform magnetic colonoscopy in vivo. We combine the use of robotics, computer vision and advanced control to offer an intuitive and effective endoscopic system. Moreover, we define the characteristics required to achieve autonomy in robotic endoscopy. The paradigm described here can be adopted in a variety of applications where navigation in unstructured environments is required, such as catheters, pancreatic endoscopy, bronchoscopy and gastroscopy. This work brings alternative endoscopic technologies closer to the translational stage, increasing the availability of early-stage cancer treatments. Magnetic endoscopes have the potential to improve access, reduce patient discomfort and enhance safety. While navigation of magnetic endoscopes can be challenging for the operator, a new approach by Martin, Scaglioni and colleagues explores how to reduce this burden by offering different levels of autonomy in robotic colonoscopy.},
  archive      = {J_NATMI},
  author       = {Martin, James W. and Scaglioni, Bruno and Norton, Joseph C. and Subramanian, Venkataraman and Arezzo, Alberto and Obstein, Keith L. and Valdastri, Pietro},
  doi          = {10.1038/s42256-020-00231-9},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {595-606},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Enabling the future of colonoscopy with intelligent and autonomous magnetic manipulation},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Semantic segmentation of microscopic neuroanatomical data by
combining topological priors with encoder–decoder deep networks.
<em>NATMI</em>, <em>2</em>(10), 585–594. (<a
href="https://doi.org/10.1038/s42256-020-0227-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding of neuronal circuitry at cellular resolution within the brain has relied on neuron tracing methods that involve careful observation and interpretation by experienced neuroscientists. With recent developments in imaging and digitization, this approach is no longer feasible with the large-scale (terabyte to petabyte range) images. Machine-learning-based techniques, using deep networks, provide an efficient alternative to the problem. However, these methods rely on very large volumes of annotated images for training and have error rates that are too high for scientific data analysis, and thus requires a substantial volume of human-in-the-loop proofreading. Here we introduce a hybrid architecture combining prior structure in the form of topological data analysis methods, based on discrete Morse theory, with the best-in-class deep-net architectures for the neuronal connectivity analysis. We show significant performance gains using our hybrid architecture on detection of topological structure (for example, connectivity of neuronal processes and local intensity maxima on axons corresponding to synaptic swellings) with precision and recall close to 90\% compared with human observers. We have adapted our architecture to a high-performance pipeline capable of semantic segmentation of light-microscopic whole-brain image data into a hierarchy of neuronal compartments. We expect that the hybrid architecture incorporating discrete Morse techniques into deep nets will generalize to other data domains. Advances in large-scale connectivity mapping of the brain require efficient computational tools to detect fine structures across large volumes of images, which poses challenges. The authors introduce a hybrid architecture that incorporates topological priors of neuronal structures with deep learning models to improve semantic segmentation of neuroanatomical image data.},
  archive      = {J_NATMI},
  author       = {Banerjee, Samik and Magee, Lucas and Wang, Dingkang and Li, Xu and Huo, Bing-Xing and Jayakumar, Jaikishan and Matho, Katherine and Lin, Meng-Kuan and Ram, Keerthi and Sivaprakasam, Mohanasankar and Huang, Josh and Wang, Yusu and Mitra, Partha P.},
  doi          = {10.1038/s42256-020-0227-9},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {585-594},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Semantic segmentation of microscopic neuroanatomical data by combining topological priors with encoder–decoder deep networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Drug discovery with explainable artificial intelligence.
<em>NATMI</em>, <em>2</em>(10), 573–584. (<a
href="https://doi.org/10.1038/s42256-020-00236-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning bears promise for drug discovery, including advanced image analysis, prediction of molecular structure and function, and automated generation of innovative chemical entities with bespoke properties. Despite the growing number of successful prospective applications, the underlying mathematical models often remain elusive to interpretation by the human mind. There is a demand for ‘explainable’ deep learning methods to address the need for a new narrative of the machine language of the molecular sciences. This Review summarizes the most prominent algorithmic concepts of explainable artificial intelligence, and forecasts future opportunities, potential applications as well as several remaining challenges. We also hope it encourages additional efforts towards the development and acceptance of explainable artificial intelligence techniques. Drug discovery has recently profited greatly from the use of deep learning models. However, these models can be notoriously hard to interpret. In this Review, Jiménez-Luna and colleagues summarize recent approaches to use explainable artificial intelligence techniques in drug discovery.},
  archive      = {J_NATMI},
  author       = {Jiménez-Luna, José and Grisoni, Francesca and Schneider, Gisbert},
  doi          = {10.1038/s42256-020-00236-4},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {573-584},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Drug discovery with explainable artificial intelligence},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The rise of robots in surgical environments during COVID-19.
<em>NATMI</em>, <em>2</em>(10), 566–572. (<a
href="https://doi.org/10.1038/s42256-020-00238-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COVID-19 pandemic has changed our world and impacted multiple layers of our society. All frontline workers and in particular those in direct contact with patients have been exposed to major risk. To mitigate pathogen spread and protect healthcare workers and patients, medical services have been largely restricted, including cancellation of elective surgeries, which has posed a substantial burden for patients and immense economic loss for various hospitals. The integration of a robot as a shielding layer, physically separating the healthcare worker and patient, is a powerful tool to combat the omnipresent fear of pathogen contamination and maintain surgical volumes. In this Perspective, we outline detailed scenarios in the pre-, intra- and postoperative care, in which the use of robots and artificial intelligence can mitigate infectious contamination and aid patient management in the surgical environment during times of immense patient influx. We also discuss cost-effectiveness and benefits of surgical robotic systems beyond their use in pandemics. The current pandemic creates unprecedented demands for hospitals. Digitization and machine intelligence are gaining significance in healthcare to combat the virus. Their legacy may well outlast the pandemic and revolutionize surgical performance and management. Robots could play an important part in transforming healthcare to cope with the COVID-19 pandemic. This Perspective highlights how robotic technology integrated in a range of tasks in the surgical environment could help to ensure a continuation of medical services while reducing the risk of infection.},
  archive      = {J_NATMI},
  author       = {Zemmar, Ajmal and Lozano, Andres M. and Nelson, Bradley J.},
  doi          = {10.1038/s42256-020-00238-2},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {566-572},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The rise of robots in surgical environments during COVID-19},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Accelerating evidence-informed decision-making for the
sustainable development goals using machine learning. <em>NATMI</em>,
<em>2</em>(10), 559–565. (<a
href="https://doi.org/10.1038/s42256-020-00235-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The United Nations Sustainable Development Goal 2 (SDG 2) is to achieve zero hunger by 2030. We have designed Persephone, a machine learning model, to support a diverse volunteer network of 77 researchers from 23 countries engaged in creating interdisciplinary evidence syntheses in support of SDG 2. Such evidence syntheses, whatever the specific topic, assess original studies to determine the effectiveness of interventions. By gathering and summarizing current evidence and providing objective recommendations they can be valuable aids to decision-makers. However, they are time-consuming; estimates range from 18 months to three years to produce a single review. Persephone analysed 500,000 unstructured text summaries from prominent sources of agricultural research, determining with 90\% accuracy the subset of studies that would eventually be selected by expert researchers. We demonstrate that machine learning models can be invaluable in placing evidence into the hands of policymakers. Evidence syntheses produced from the scientific literature are important tools for policymakers. Producing such evidence syntheses can be highly time- and labour-consuming but machine learning models can help as already demonstrated in the health and medical sciences. This Perspective describes a machine learning-based framework specifically designed to support evidence syntheses in the area of agricultural research, for tackling the UN Sustainable Development Goal 2: zero hunger by 2030.},
  archive      = {J_NATMI},
  author       = {Porciello, Jaron and Ivanina, Maryia and Islam, Maidul and Einarson, Stefan and Hirsh, Haym},
  doi          = {10.1038/s42256-020-00235-5},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {559-565},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accelerating evidence-informed decision-making for the sustainable development goals using machine learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Solving optimization tasks in condensed matter.
<em>NATMI</em>, <em>2</em>(10), 557–558. (<a
href="https://doi.org/10.1038/s42256-020-00240-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finding states of matter with properties that are just right is a main challenge from metallurgy to quantum computing. A data-driven optimization approach based on gaming strategies could help.},
  archive      = {J_NATMI},
  author       = {Greplova, Eliska},
  doi          = {10.1038/s42256-020-00240-8},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {557-558},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Solving optimization tasks in condensed matter},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Improving the quality of machine learning in health
applications and clinical research. <em>NATMI</em>, <em>2</em>(10),
554–556. (<a href="https://doi.org/10.1038/s42256-020-00239-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For machine learning developers, the use of prediction tools in real-world clinical settings can be a distant goal. Recently published guidelines for reporting clinical research that involves machine learning will help connect clinical and computer science communities, and realize the full potential of machine learning tools.},
  archive      = {J_NATMI},
  author       = {Mateen, Bilal A. and Liley, James and Denniston, Alastair K. and Holmes, Chris C. and Vollmer, Sebastian J.},
  doi          = {10.1038/s42256-020-00239-1},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {554-556},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Improving the quality of machine learning in health applications and clinical research},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Call to action for robotics. <em>NATMI</em>, <em>2</em>(10),
553. (<a href="https://doi.org/10.1038/s42256-020-00241-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Robots can relieve humans of dangerous tasks. With the pandemic making physical contact potentially dangerous due to the risk of contagion, a new focus for robotic applications in healthcare has come into view.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-00241-7},
  journal      = {Nature Machine Intelligence},
  number       = {10},
  pages        = {553},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Call to action for robotics},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Predicting drug–protein interaction
using quasi-visual question answering system. <em>NATMI</em>,
<em>2</em>(9), 551. (<a
href="https://doi.org/10.1038/s42256-020-0224-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NATMI},
  author       = {Zheng, Shuangjia and Li, Yongjian and Chen, Sheng and Xu, Jun and Yang, Yuedong},
  doi          = {10.1038/s42256-020-0224-z},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {551},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Predicting drug–protein interaction using quasi-visual question answering system},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Protein function prediction is improved by creating
synthetic feature samples with generative adversarial networks.
<em>NATMI</em>, <em>2</em>(9), 540–550. (<a
href="https://doi.org/10.1038/s42256-020-0222-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein function prediction is a challenging but important task in bioinformatics. Many prediction methods have been developed, but are still limited by the bottleneck on training sample quantity. Therefore, it is valuable to develop a data augmentation method that can generate high-quality synthetic samples to further improve the accuracy of prediction methods. In this work, we propose a novel generative adversarial networks-based method, FFPred-GAN, to accurately learn the high-dimensional distributions of protein sequence-based biophysical features and also generate high-quality synthetic protein feature samples. The experimental results suggest that the synthetic protein feature samples are successful in improving the prediction accuracy for all three domains of Gene Ontology through augmentation of the original training protein feature samples. Training machine learning models to predict the function of proteins is limited by the availability of only a small amount of labelled training data. Training can be improved by employing generative adversarial networks to generate additional synthetic protein samples.},
  archive      = {J_NATMI},
  author       = {Wan, Cen and Jones, David T.},
  doi          = {10.1038/s42256-020-0222-1},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {540-550},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Protein function prediction is improved by creating synthetic feature samples with generative adversarial networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Remote explainability faces the bouncer problem.
<em>NATMI</em>, <em>2</em>(9), 529–539. (<a
href="https://doi.org/10.1038/s42256-020-0216-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The concept of explainability is envisioned to satisfy society’s demands for transparency about machine learning decisions. The concept is simple: like humans, algorithms should explain the rationale behind their decisions so that their fairness can be assessed. Although this approach is promising in a local context (for example, the model creator explains it during debugging at the time of training), we argue that this reasoning cannot simply be transposed to a remote context, where a model trained by a service provider is only accessible to a user through a network and its application programming interface. This is problematic, as it constitutes precisely the target use case requiring transparency from a societal perspective. Through an analogy with a club bouncer (who may provide untruthful explanations upon customer rejection), we show that providing explanations cannot prevent a remote service from lying about the true reasons leading to its decisions. More precisely, we observe the impossibility of remote explainability for single explanations by constructing an attack on explanations that hides discriminatory features from the querying user. We provide an example implementation of this attack. We then show that the probability that an observer spots the attack, using several explanations for attempting to find incoherences, is low in practical settings. This undermines the very concept of remote explainability in general. When automated decisions are provided by a company without providing the full model, users and law makers might demand a ‘right to an explanation’. Le Merrer and Trédan show that malicious manipulations of these explanations are hard to detect, even for simple strategies to obscure the model’s decisions.},
  archive      = {J_NATMI},
  author       = {Le Merrer, Erwan and Trédan, Gilles},
  doi          = {10.1038/s42256-020-0216-z},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {529-539},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Remote explainability faces the bouncer problem},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Using online verification to prevent autonomous vehicles
from causing accidents. <em>NATMI</em>, <em>2</em>(9), 518–528. (<a
href="https://doi.org/10.1038/s42256-020-0225-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ensuring that autonomous vehicles do not cause accidents remains a challenge. We present a formal verification technique for guaranteeing legal safety in arbitrary urban traffic situations. Legal safety means that autonomous vehicles never cause accidents although other traffic participants are allowed to perform any behaviour in accordance with traffic rules. Our technique serves as a safety layer for existing motion planning frameworks that provide intended trajectories for autonomous vehicles. We verify whether intended trajectories comply with legal safety and provide fallback solutions in safety-critical situations. The benefits of our verification technique are demonstrated in critical urban scenarios, which have been recorded in real traffic. The autonomous vehicle executed only safe trajectories, even when using an intended trajectory planner that was not aware of other traffic participants. Our results indicate that our online verification technique can drastically reduce the number of traffic accidents. Recent accidents with autonomous test vehicles have eroded trust in such self-driving cars. A shift in approach is required to ensure autonomous vehicles can never be the cause of accidents. An online verification technique is presented that guarantees provably safe motions, including fallback solutions in safety-critical situations, for any intended trajectory calculated by the underlying motion planner.},
  archive      = {J_NATMI},
  author       = {Pek, Christian and Manzinger, Stefanie and Koschi, Markus and Althoff, Matthias},
  doi          = {10.1038/s42256-020-0225-y},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {518-528},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Using online verification to prevent autonomous vehicles from causing accidents},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Finding the ground state of spin hamiltonians with
reinforcement learning. <em>NATMI</em>, <em>2</em>(9), 509–517. (<a
href="https://doi.org/10.1038/s42256-020-0226-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reinforcement learning (RL) has become a proven method for optimizing a procedure for which success has been defined, but the specific actions needed to achieve it have not. Using a method we call ‘controlled online optimization learning’ (COOL), we apply the so-called ‘black box’ method of RL to simulated annealing (SA), demonstrating that an RL agent based on proximal policy optimization can, through experience alone, arrive at a temperature schedule that surpasses the performance of standard heuristic temperature schedules for two classes of Hamiltonians. When the system is initialized at a cool temperature, the RL agent learns to heat the system to ‘melt’ it and then slowly cool it in an effort to anneal to the ground state; if the system is initialized at a high temperature, the algorithm immediately cools the system. We investigate the performance of our RL-driven SA agent in generalizing to all Hamiltonians of a specific class. When trained on random Hamiltonians of nearest-neighbour spin glasses, the RL agent is able to control the SA process for other Hamiltonians, reaching the ground state with a higher probability than a simple linear annealing schedule. Furthermore, the scaling performance (with respect to system size) of the RL approach is far more favourable, achieving a performance improvement of almost two orders of magnitude on L = 142 systems. We demonstrate the robustness of the RL approach when the system operates in a ‘destructive observation’ mode, an allusion to a quantum system where measurements destroy the state of the system. The success of the RL agent could have far-reaching impacts, from classical optimization, to quantum annealing and to the simulation of physical systems. Reinforcement learning has become a popular method in various domains, for problems where an agent must learn what actions must be taken to reach a particular goal. An interesting example where the technique can be applied is simulated annealing in condensed matter physics, where a procedure is determined for slowly cooling a complex system to its ground state. A reinforcement learning approach has been developed that can learn a temperature scheduling protocol to find the ground state of spin glasses, magnetic systems with strong spin–spin interactions between neighbouring atoms.},
  archive      = {J_NATMI},
  author       = {Mills, Kyle and Ronagh, Pooya and Tamblyn, Isaac},
  doi          = {10.1038/s42256-020-0226-x},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {509-517},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Finding the ground state of spin hamiltonians with reinforcement learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Ensemble deep learning in bioinformatics. <em>NATMI</em>,
<em>2</em>(9), 500–508. (<a
href="https://doi.org/10.1038/s42256-020-0217-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The remarkable flexibility and adaptability of ensemble methods and deep learning models have led to the proliferation of their application in bioinformatics research. Traditionally, these two machine learning techniques have largely been treated as independent methodologies in bioinformatics applications. However, the recent emergence of ensemble deep learning—wherein the two machine learning techniques are combined to achieve synergistic improvements in model accuracy, stability and reproducibility—has prompted a new wave of research and application. Here, we share recent key developments in ensemble deep learning and look at how their contribution has benefited a wide range of bioinformatics research from basic sequence analysis to systems biology. While the application of ensemble deep learning in bioinformatics is diverse and multifaceted, we identify and discuss the common challenges and opportunities in the context of bioinformatics research. We hope this Review Article will bring together the broader community of machine learning researchers, bioinformaticians and biologists to foster future research and development in ensemble deep learning, and inspire novel bioinformatics applications that are unattainable by traditional methods. Recent developments in machine learning have seen the merging of ensemble and deep learning techniques. The authors review advances in ensemble deep learning methods and their applications in bioinformatics, and discuss the challenges and opportunities going forward.},
  archive      = {J_NATMI},
  author       = {Cao, Yue and Geddes, Thomas Andrew and Yang, Jean Yee Hwa and Yang, Pengyi},
  doi          = {10.1038/s42256-020-0217-y},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {500-508},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Ensemble deep learning in bioinformatics},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Disentangling automatic and semi-automatic approaches to the
optimization-based design of control software for robot swarms.
<em>NATMI</em>, <em>2</em>(9), 494–499. (<a
href="https://doi.org/10.1038/s42256-020-0215-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optimization-based design is an effective and promising approach to realizing collective behaviours for robot swarms. Unfortunately, the domain literature often remains vague about the exact role played by the human designer, if any. It is our contention that two cases should be disentangled: semi-automatic design, in which a human designer operates and steers an optimization process (for example, by fine-tuning the parameters of the optimization algorithm); and (fully) automatic design, in which the optimization process does not involve, need or allow any human intervention. In this Perspective, we briefly review the relevant literature; illustrate the hypotheses, characteristics and core challenges of semi-automatic and automatic design; and sketch out the context in which they could be ideally applied. Developing swarm robots for a specific application is a time consuming process and can be alleviated by automated optimization of the behaviour. Birattari and colleagues discuss that there are two fundamentally different design approaches; a semi-autonomous one, which allows for situation specific tuning from human engineers and one that needs to be entirely autonomous.},
  archive      = {J_NATMI},
  author       = {Birattari, Mauro and Ligot, Antoine and Hasselmann, Ken},
  doi          = {10.1038/s42256-020-0215-0},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {494-499},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Disentangling automatic and semi-automatic approaches to the optimization-based design of control software for robot swarms},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Estimating time with neural networks. <em>NATMI</em>,
<em>2</em>(9), 492–493. (<a
href="https://doi.org/10.1038/s42256-020-00229-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The proper response to an ever-changing environment depends on the ability to quantify elapsed time, memorize short intervals and forecast when an upcoming experience may occur. A recent study describes the encoding principles of these three types of time using computational modelling.},
  archive      = {J_NATMI},
  author       = {Merchant, Hugo and Pérez, Oswaldo},
  doi          = {10.1038/s42256-020-00229-3},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {492-493},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Estimating time with neural networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Domesticating the techno-racial project. <em>NATMI</em>,
<em>2</em>(9), 491. (<a
href="https://doi.org/10.1038/s42256-020-00228-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Mohamed, Shakir},
  doi          = {10.1038/s42256-020-00228-4},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {491},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Domesticating the techno-racial project},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An embedded ethics approach for AI development.
<em>NATMI</em>, <em>2</em>(9), 488–490. (<a
href="https://doi.org/10.1038/s42256-020-0214-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is a need to consider how AI developers can be practically assisted in identifying and addressing ethical issues. In this Comment, a group of AI engineers, ethicists and social scientists suggest embedding ethicists into the development team as one way of improving the consideration of ethical issues during AI development.},
  archive      = {J_NATMI},
  author       = {McLennan, Stuart and Fiske, Amelia and Celi, Leo Anthony and Müller, Ruth and Harder, Jan and Ritt, Konstantin and Haddadin, Sami and Buyx, Alena},
  doi          = {10.1038/s42256-020-0214-1},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {488-490},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An embedded ethics approach for AI development},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Algorithms to live by. <em>NATMI</em>, <em>2</em>(9), 487.
(<a href="https://doi.org/10.1038/s42256-020-00230-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The challenge of practically integrating an ethical and social approach in the development and implementation of AI needs to be urgently addressed, to help restore public trust in technology.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-00230-w},
  journal      = {Nature Machine Intelligence},
  number       = {9},
  pages        = {487},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Algorithms to live by},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Making deep neural networks right for the right scientific
reasons by interacting with their explanations. <em>NATMI</em>,
<em>2</em>(8), 476–486. (<a
href="https://doi.org/10.1038/s42256-020-0212-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep neural networks have demonstrated excellent performances in many real-world applications. Unfortunately, they may show Clever Hans-like behaviour (making use of confounding factors within datasets) to achieve high performance. In this work we introduce the novel learning setting of explanatory interactive learning and illustrate its benefits on a plant phenotyping research task. Explanatory interactive learning adds the scientist into the training loop, who interactively revises the original model by providing feedback on its explanations. Our experimental results demonstrate that explanatory interactive learning can help to avoid Clever Hans moments in machine learning and encourages (or discourages, if appropriate) trust in the underlying model. Deep learning approaches can show excellent performance but still have limited practical use if they learn to predict based on confounding factors in a dataset, for instance text labels in the corner of images. By using an explanatory interactive learning approach, with a human expert in the loop during training, it becomes possible to avoid predictions based on confounding factors.},
  archive      = {J_NATMI},
  author       = {Schramowski, Patrick and Stammer, Wolfgang and Teso, Stefano and Brugger, Anna and Herbert, Franziska and Shao, Xiaoting and Luigs, Hans-Georg and Mahlein, Anne-Katrin and Kersting, Kristian},
  doi          = {10.1038/s42256-020-0212-3},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {476-486},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Making deep neural networks right for the right scientific reasons by interacting with their explanations},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Elucidation of DNA methylation on n6-adenine with deep
learning. <em>NATMI</em>, <em>2</em>(8), 466–475. (<a
href="https://doi.org/10.1038/s42256-020-0211-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research on DNA methylation on N6-adenine (6mA) in eukaryotes has received much recent attention. Recent studies have generated a large amount of 6mA genomic data, yet the role of DNA 6mA in eukaryotes remains elusive, or even controversial. We argue that the sparsity of DNA 6mA in eukaryotes, the limitations of current biotechnologies for 6mA detection and the sophistication of the 6mA regulatory mechanism together pose great challenges for elucidation of DNA 6mA. To exploit existing 6mA genomic data and address this challenge, here we develop a deep-learning-based algorithm for predicting potential DNA 6mA sites de novo from sequence at single-nucleotide resolution, with application to three representative model organisms, Arabidopsis thaliana, Drosophila melanogaster and Escherichia coli. Extensive experiments demonstrate the accuracy of our algorithm and its superior performance compared with conventional k-mer-based approaches. Furthermore, our saliency maps-based context analysis protocol reveals interesting cis-regulatory patterns around the 6mA sites that are missed by conventional motif analysis. Our proposed analytical tools and findings will help to elucidate the regulatory mechanisms of 6mA and benefit the in-depth exploration of their functional effects. Finally, we offer a complete catalogue of potential 6mA sites based on in silico whole-genome prediction. The role of DNA methylation on N6-adenine (6mA) in eukaryotes is a challenging research problem. Tan et al. develop a deep-learning-based algorithm to predict 6mA sites from sequences at single-nucleotide resolution, and apply the method to three representative model organisms. The method is further developed to visualize regulatory patterns around 6mA sites.},
  archive      = {J_NATMI},
  author       = {Tan, Fei and Tian, Tian and Hou, Xiurui and Yu, Xiang and Gu, Lei and Mafra, Fernanda and Gregory, Brian D. and Wei, Zhi and Hakonarson, Hakon},
  doi          = {10.1038/s42256-020-0211-4},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {466-475},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Elucidation of DNA methylation on n6-adenine with deep learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Minimal-uncertainty prediction of general drug-likeness
based on bayesian neural networks. <em>NATMI</em>, <em>2</em>(8),
457–465. (<a href="https://doi.org/10.1038/s42256-020-0209-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Triaging unpromising lead molecules early in the drug discovery process is essential for accelerating its pace while avoiding the costs of unwarranted biological and clinical testing. Accordingly, medicinal chemists have been trying for decades to develop metrics—ranging from heuristic measures to machine-learning models—that could rapidly distinguish potential drugs from small molecules that lack drug-like features. However, none of these metrics has gained universal acceptance and the very idea of ‘drug-likeness’ has recently been put into question. Here, we evaluate drug-likeness using different sets of descriptors and different state-of-the-art classifiers, reaching an out-of-sample accuracy of 87–88\%. Remarkably, because these individual classifiers yield different Bayesian error distributions, their combination and selection of minimal-variance predictions can increase the accuracy of distinguishing drug-like from non-drug-like molecules to 93\%. Because total variance is comparable with its aleatoric contribution reflecting irreducible error inherent to the dataset (as opposed to the epistemic contribution due to the model itself), this level of accuracy is probably the upper limit achievable with the currently known collection of drugs. When designing new drugs, there are countless ways to create molecules, yet only a few interact with biological targets. Beker and colleagues provide here a graph neural network based metric for drug-likeness that can guide the search.},
  archive      = {J_NATMI},
  author       = {Beker, Wiktor and Wołos, Agnieszka and Szymkuć, Sara and Grzybowski, Bartosz A.},
  doi          = {10.1038/s42256-020-0209-y},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {457-465},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Minimal-uncertainty prediction of general drug-likeness based on bayesian neural networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A unified framework for integrative study of heterogeneous
gene regulatory mechanisms. <em>NATMI</em>, <em>2</em>(8), 447–456. (<a
href="https://doi.org/10.1038/s42256-020-0205-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression is regulated by a large variety of mechanisms. Previous studies attempting to model the quantitative relationships between gene expression levels and regulatory mechanisms have considered only one or a few mechanisms at a time, which cannot provide a full picture of the complex interactions among different mechanisms. This was partially due to the heterogeneity of the mechanisms, which involve different types of biological objects and data representations, making it hard to study them in a unified way. Here, we describe a flexible framework that can integrate very different types of data for studying their joint effects on gene expression. In this framework, domain knowledge is represented by metapaths, while the manifestations of their effects in actual data are summarized by an embedding of the biological objects in a latent space. We demonstrate the use of our framework in integrating several diverse types of data that are related to gene expression in different ways, including DNA contacts in three-dimensional genome architecture, protein–protein interactions, genomic neighbourhoods and broad chromatin accessibility domains. The modelling results reveal that these several types of data are able to model gene expression fairly well individually, but even better when integrated. Gene expression is regulated by a variety of mechanisms, which have been difficult to study in a unified way. The authors propose a flexible framework that can integrate different types of data for studying their joint effects on gene expression. The framework uses a general network representation for data integration, metapaths for inputting prior knowledge of gene regulatory mechanisms, and embedding techniques for capturing complex structures in the data.},
  archive      = {J_NATMI},
  author       = {Cao, Qin and Zhang, Zhenghao and Fu, Alexander Xi and Wu, Qiong and Lee, Tin-Lap and Lo, Eric and Cheng, Alfred S. L. and Cheng, Chao and Leung, Danny and Yip, Kevin Y.},
  doi          = {10.1038/s42256-020-0205-2},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {447-456},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A unified framework for integrative study of heterogeneous gene regulatory mechanisms},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Origami-inspired miniature manipulator for teleoperated
microsurgery. <em>NATMI</em>, <em>2</em>(8), 437–446. (<a
href="https://doi.org/10.1038/s42256-020-0203-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of a structure with a remote fixed point around which a mechanism can rotate is called remote centre of motion (RCM). The technique is widely used in minimally invasive surgery to avoid excess force on the incision site during the robot’s motion. Here we describe the design, fabrication and characterization of an origami-inspired miniature RCM manipulator for teleoperated microsurgery (the mini-RCM has mass 2.4 g and size 50 mm × 70 mm × 50 mm), which is actuated by three independently controlled linear actuators with concomitant sensing (each mini-LA has mass 0.41 g and size 28 mm × 7 mm × 3.6 mm). The mini-RCM has a payload capacity of approximately 27 mN and attains a positional precision of 26.4 μm. We demonstrate its potential utility as a precise tool for teleoperated microsurgery by performing 0.5-mm-square tracing and micro-cannulation teleoperated microsurgical procedures under a microscope. Teleoperation using the mini-RCM reduced the deviation from the desired trajectory by 68\% compared to manual operation. In addition, the mini-RCM allows gravity compensation and back drivability for safety. Its compact, simple structure facilitates manufacture. Robot-assisted microsurgery promises high stability and accuracy for instance in eye- or neurosurgery applications. A new miniature robotics device, based on an origami-inspired design, can make complex 3D motions and reaches a precision of around 26 micrometres.},
  archive      = {J_NATMI},
  author       = {Suzuki, Hiroyuki and Wood, Robert J.},
  doi          = {10.1038/s42256-020-0203-4},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {437-446},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Origami-inspired miniature manipulator for teleoperated microsurgery},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Increasing generality in machine learning through procedural
content generation. <em>NATMI</em>, <em>2</em>(8), 428–436. (<a
href="https://doi.org/10.1038/s42256-020-0208-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Procedural content generation (PCG) refers to the practice of generating game content, such as levels, quests or characters, algorithmically. Motivated by the need to make games replayable, as well as to reduce authoring burden and enable particular aesthetics, many PCG methods have been devised. At the same time that researchers are adapting methods from machine learning (ML) to PCG problems, the ML community has become more interested in PCG-inspired methods. One reason for this development is that ML algorithms often only work for a particular version of a particular task with particular initial parameters. In response, researchers have begun exploring randomization of problem parameters to counteract such overfitting and to allow trained policies to more easily transfer from one environment to another, such as from a simulated robot to a robot in the real world. Here we review existing work on PCG, its overlap with current efforts in ML, and promising new research directions such as procedurally generated learning environments. Although originating in games, we believe PCG algorithms are critical to creating more general machine intelligence. This review covers the history of procedural content generation (PCG) approaches for video games, and how these approaches are now used to generate training data and environments for machine learning models. The authors then discuss how PCG may be crucial for training agents which generalise well.},
  archive      = {J_NATMI},
  author       = {Risi, Sebastian and Togelius, Julian},
  doi          = {10.1038/s42256-020-0208-z},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {428-436},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Increasing generality in machine learning through procedural content generation},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). When causal inference meets deep learning. <em>NATMI</em>,
<em>2</em>(8), 426–427. (<a
href="https://doi.org/10.1038/s42256-020-0218-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bayesian networks can capture causal relations, but learning such a network from data is NP-hard. Recent work has made it possible to approximate this problem as a continuous optimization task that can be solved efficiently with well-established numerical techniques.},
  archive      = {J_NATMI},
  author       = {Luo, Yunan and Peng, Jian and Ma, Jianzhu},
  doi          = {10.1038/s42256-020-0218-x},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {426-427},
  shortjournal = {Nat. Mach. Intell.},
  title        = {When causal inference meets deep learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The carbon impact of artificial intelligence.
<em>NATMI</em>, <em>2</em>(8), 423–425. (<a
href="https://doi.org/10.1038/s42256-020-0219-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The part that artificial intelligence plays in climate change has come under scrutiny, including from tech workers themselves who joined the global climate strike last year. Much can be done by developing tools to quantify the carbon cost of machine learning models and by switching to a sustainable artificial intelligence infrastructure.},
  archive      = {J_NATMI},
  author       = {Dhar, Payal},
  doi          = {10.1038/s42256-020-0219-9},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {423-425},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The carbon impact of artificial intelligence},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A checklist for safe robot swarms. <em>NATMI</em>,
<em>2</em>(8), 420–422. (<a
href="https://doi.org/10.1038/s42256-020-0213-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As robot swarms move from the laboratory to real-world applications, a routine checklist of questions could help ensure their safe operation.},
  archive      = {J_NATMI},
  author       = {Hunt, Edmund R. and Hauert, Sabine},
  doi          = {10.1038/s42256-020-0213-2},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {420-422},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A checklist for safe robot swarms},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Next chapter in artificial writing. <em>NATMI</em>,
<em>2</em>(8), 419. (<a
href="https://doi.org/10.1038/s42256-020-0223-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {OpenAI released a beta version of its language model, GPT-3. As artificial writing permeates our lives, the challenge is how to think clearly about what it is and what impact it could have on society.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0223-0},
  journal      = {Nature Machine Intelligence},
  number       = {8},
  pages        = {419},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Next chapter in artificial writing},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). High-accuracy prostate cancer pathology using deep learning.
<em>NATMI</em>, <em>2</em>(7), 411–418. (<a
href="https://doi.org/10.1038/s42256-020-0200-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning (DL) is a powerful methodology for the recognition and classification of tissue structures in digital pathology. Its performance in prostate cancer pathology is still under intensive investigation. Here we develop DL-based models for the detection of prostate cancer tissue in whole-slide images based on a large high-quality annotated training dataset and a modern state-of-the-art convolutional network architecture (NASNetLarge). The overall accuracy of our model for tumour detection in two validation cohorts is comparable to that of pathologists and reaches 97.3\% in a native version and more than 98\% using the suggested DL-based augmentation strategies. As a second step, we suggest a new biologically meaningful DL-based algorithm for Gleason grading of prostatic adenocarcinomas with high, human-level performance in prognostic stratification of patients when tested in several well-characterized validation cohorts. Furthermore, we determine the optimal minimal tumour size (real size of approximately 560 × 560 µm) for robust Gleason grading representative of the whole tumour focus. Our approach is realized in the unified digital pathology pipeline, which delivers all the relevant tumour metrics for a pathology report. Deep learning methods can be a powerful part of digital pathology workflows, provided well-annotated training datasets are available. Tolkach and colleagues develop a deep learning model to recognize and grade prostate cancer, based on a convolution neural network and a dataset with high-quality labels at gland-level precision.},
  archive      = {J_NATMI},
  author       = {Tolkach, Yuri and Dohmgörgen, Tilmann and Toma, Marieta and Kristiansen, Glen},
  doi          = {10.1038/s42256-020-0200-7},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {411-418},
  shortjournal = {Nat. Mach. Intell.},
  title        = {High-accuracy prostate cancer pathology using deep learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Actor neural networks for the robust control of partially
measured nonlinear systems showcased for image propagation through
diffuse media. <em>NATMI</em>, <em>2</em>(7), 403–410. (<a
href="https://doi.org/10.1038/s42256-020-0199-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The output of physical systems, such as the scrambled pattern formed by shining the spot of a laser pointer through fog, is often easily accessible by direct measurements. However, selection of the input of such a system to obtain a desired output is difficult, because it is an ill-posed problem; that is, there are multiple inputs yielding the same output. Information transmission through scattering media is an example of this problem. Machine learning approaches for imaging have been implemented very successfully in photonics to recover the original input phase and amplitude objects of the scattering system from the distorted intensity diffraction pattern outputs. However, controlling the output of such a system, without having examples of inputs that can produce outputs in the class of the output objects the user wants to produce, is a challenging problem. Here, we propose an online learning approach for the projection of arbitrary shapes through a multimode fibre when a sample of intensity-only measurements is taken at the output. This projection system is nonlinear, because the intensity, not the complex amplitude, is detected. We show an image projection fidelity as high as ~90\%, which is on par with the gold-standard methods that characterize the system fully by phase and amplitude measurements. The generality and simplicity of the proposed approach could potentially provide a new way of target-oriented control in real-world applications when only partial measurements are available. Machine learning has become popular in solving complex optical problems such as recovering the input phase and amplitude for a specific pattern or image measured through a scattering medium. In a more challenging application, Rahmani et al. consider the problem of also producing desired outputs for such a nonlinear system when only some intensity-only measurements of example outputs are available. They develop a neural network approach that can ensure the transmission of images through a highly nonlinear system—a multimode fibre—with a 90\% fidelity.},
  archive      = {J_NATMI},
  author       = {Rahmani, Babak and Loterie, Damien and Kakkava, Eirini and Borhani, Navid and Teğin, Uğur and Psaltis, Demetri and Moser, Christophe},
  doi          = {10.1038/s42256-020-0199-9},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {403-410},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Actor neural networks for the robust control of partially measured nonlinear systems showcased for image propagation through diffuse media},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Quantum approximate bayesian computation for NMR model
inference. <em>NATMI</em>, <em>2</em>(7), 396–402. (<a
href="https://doi.org/10.1038/s42256-020-0198-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advances may lead to the development of small-scale quantum computers that are capable of solving problems that cannot be tackled with classical computers. A limited number of algorithms have been proposed and their relevance to real-world problems is a subject of active investigation. Analysis of many-body quantum systems is particularly challenging for classical computers due to the exponential scaling of the Hilbert space dimension with the number of particles. Hence, solving the problems relevant to chemistry and condensed-matter physics is expected to be the first successful application of quantum computers. In this Article, we propose another class of problems from the quantum realm that can be solved efficiently on quantum computers: model inference for nuclear magnetic resonance (NMR) spectroscopy, which is important for biological and medical research. Our results are based on three interconnected studies. First, we use methods from classical machine learning to analyse a dataset of NMR spectra of small molecules. We perform stochastic neighbourhood embedding and identify clusters of spectra, and demonstrate that these clusters are correlated with the covalent structure of the molecules. Second, we propose a simple and efficient method, aided by a quantum simulator, to extract the NMR spectrum of any hypothetical molecule described by a parametric Heisenberg model. Third, we propose a simple variational Bayesian inference procedure for estimating the Hamiltonian parameters of experimentally relevant NMR spectra. Currently available quantum hardware is limited by noise, so practical implementations often involve a combination with classical approaches. Sels et al. identify a promising application for such a quantum–classic hybrid approach, namely inferring molecular structure from NMR spectra, by employing a range of machine learning tools in combination with a quantum simulator.},
  archive      = {J_NATMI},
  author       = {Sels, Dries and Dashti, Hesam and Mora, Samia and Demler, Olga and Demler, Eugene},
  doi          = {10.1038/s42256-020-0198-x},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {396-402},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Quantum approximate bayesian computation for NMR model inference},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Gaussian embedding for large-scale gene set analysis.
<em>NATMI</em>, <em>2</em>(7), 387–395. (<a
href="https://doi.org/10.1038/s42256-020-0193-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene sets, including protein complexes and signalling pathways, have proliferated greatly, in large part as a result of high-throughput biological data. Leveraging gene sets to gain insight into biological discovery requires computational methods for converting them into a useful form for available machine learning models. Here, we study the problem of embedding gene sets as compact features that are compatible with available machine learning codes. We present Set2Gaussian, a novel network-based gene set embedding approach, which represents each gene set as a multivariate Gaussian distribution rather than a single point in the low-dimensional space, according to the proximity of these genes in a protein–protein interaction network. We demonstrate that Set2Gaussian improves gene set member identification, accurately stratifies tumours, and finds concise gene sets for gene set enrichment analysis. We further show how Set2Gaussian allows us to identify a clinical prognostic and predictive subnetwork around neurofilament medium in sarcoma, which we validate in independent cohorts. Gene sets can provide valuable information for gaining insight into disease mechanisms and cellular functions. In this paper, the authors use a Gaussian approach to represent gene sets and gene networks in a low-dimensional space, allowing for accurate prediction and decreased computational complexity.},
  archive      = {J_NATMI},
  author       = {Wang, Sheng and Flynn, Emily R. and Altman, Russ B.},
  doi          = {10.1038/s42256-020-0193-2},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {387-395},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Gaussian embedding for large-scale gene set analysis},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep learning decodes the principles of differential gene
expression. <em>NATMI</em>, <em>2</em>(7), 376–386. (<a
href="https://doi.org/10.1038/s42256-020-0201-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying the molecular mechanisms that control differential gene expression (DE) is a major goal of basic and disease biology. Here, we develop a systems biology model to predict DE and mine the biological basis of the factors that influence predicted gene expression to understand how it may be generated. This model, called DEcode, utilizes deep learning to predict DE based on genome-wide binding sites on RNAs and promoters. Ranking predictive factors from DEcode indicates that clinically relevant expression changes between thousands of individuals can be predicted mainly through the joint action of post-transcriptional RNA-binding factors. We also show the broad potential applications of DEcode to generate biological insights, by predicting DE between tissues, differential transcript usage, and drivers of ageing throughout the human lifespan, of gene co-expression relationships on a genome-wide scale, and of frequently differentially expressed genes across diverse conditions. DEcode is freely available to researchers to identify influential molecular mechanisms for any human expression data. A goal of biology is to identify the molecular mechanisms that control differential gene expression. Tasaki et al. have developed a framework that integrates genomic data into a deep learning model of transcriptome regulations to predict multiple transcriptional effects in tissue- and person-specific transcriptomes.},
  archive      = {J_NATMI},
  author       = {Tasaki, Shinya and Gaiteri, Chris and Mostafavi, Sara and Wang, Yanling},
  doi          = {10.1038/s42256-020-0201-6},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {376-386},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning decodes the principles of differential gene expression},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Causal inference and counterfactual prediction in machine
learning for actionable healthcare. <em>NATMI</em>, <em>2</em>(7),
369–375. (<a href="https://doi.org/10.1038/s42256-020-0197-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Big data, high-performance computing, and (deep) machine learning are increasingly becoming key to precision medicine—from identifying disease risks and taking preventive measures, to making diagnoses and personalizing treatment for individuals. Precision medicine, however, is not only about predicting risks and outcomes, but also about weighing interventions. Interventional clinical predictive models require the correct specification of cause and effect, and the calculation of so-called counterfactuals, that is, alternative scenarios. In biomedical research, observational studies are commonly affected by confounding and selection bias. Without robust assumptions, often requiring a priori domain knowledge, causal inference is not feasible. Data-driven prediction models are often mistakenly used to draw causal effects, but neither their parameters nor their predictions necessarily have a causal interpretation. Therefore, the premise that data-driven prediction models lead to trustable decisions/interventions for precision medicine is questionable. When pursuing intervention modelling, the bio-health informatics community needs to employ causal approaches and learn causal structures. Here we discuss how target trials (algorithmic emulation of randomized studies), transportability (the licence to transfer causal effects from one population to another) and prediction invariance (where a true causal model is contained in the set of all prediction models whose accuracy does not vary across different settings) are linchpins to developing and testing intervention models. Machine learning models are commonly used to predict risks and outcomes in biomedical research. But healthcare often requires information about cause–effect relations and alternative scenarios, that is, counterfactuals. Prosperi et al. discuss the importance of interventional and counterfactual models, as opposed to purely predictive models, in the context of precision medicine.},
  archive      = {J_NATMI},
  author       = {Prosperi, Mattia and Guo, Yi and Sperrin, Matt and Koopman, James S. and Min, Jae S. and He, Xing and Rich, Shannan and Wang, Mo and Buchan, Iain E. and Bian, Jiang},
  doi          = {10.1038/s42256-020-0197-y},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {369-375},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Causal inference and counterfactual prediction in machine learning for actionable healthcare},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Embedding gene sets in low-dimensional space.
<em>NATMI</em>, <em>2</em>(7), 367–368. (<a
href="https://doi.org/10.1038/s42256-020-0204-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An important task in system biology is to understand cellular processes through the lens of gene sets and their expression patterns. Machine learning can help, but genes form complex interaction networks, and levarging this information in machine learning applications requires a sophisticated data representation.},
  archive      = {J_NATMI},
  author       = {Hoinka, Jan and Przytycka, Teresa M.},
  doi          = {10.1038/s42256-020-0204-3},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {367-368},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Embedding gene sets in low-dimensional space},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Artificial intelligence in a crisis needs ethics with
urgency. <em>NATMI</em>, <em>2</em>(7), 365–366. (<a
href="https://doi.org/10.1038/s42256-020-0195-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence tools can help save lives in a pandemic. However, the need to implement technological solutions rapidly raises challenging ethical issues. We need new approaches for ethics with urgency, to ensure AI can be safely and beneficially used in the COVID-19 response and beyond.},
  archive      = {J_NATMI},
  author       = {Tzachor, Asaf and Whittlestone, Jess and Sundaram, Lalitha and hÉigeartaigh, Seán Ó},
  doi          = {10.1038/s42256-020-0195-0},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {365-366},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Artificial intelligence in a crisis needs ethics with urgency},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Why it is time to stop ostracizing social robots.
<em>NATMI</em>, <em>2</em>(7), 364. (<a
href="https://doi.org/10.1038/s42256-020-0202-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Aymerich-Franch, Laura},
  doi          = {10.1038/s42256-020-0202-5},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {364},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Why it is time to stop ostracizing social robots},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Technology can’t fix this. <em>NATMI</em>, <em>2</em>(7),
363. (<a href="https://doi.org/10.1038/s42256-020-0210-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AI tools used in society often enhance inequality, affecting Black lives disproportionally. Addressing this issue will require more than technological solutions. Researchers and experts in the field are overwhelmingly white and need to engage and listen to those experiencing the harm.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0210-5},
  journal      = {Nature Machine Intelligence},
  number       = {7},
  pages        = {363},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Technology can’t fix this},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Predicting tumour mutational burden from histopathological
images using multiscale deep learning. <em>NATMI</em>, <em>2</em>(6),
356–362. (<a href="https://doi.org/10.1038/s42256-020-0190-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumour mutational burden (TMB) is an important biomarker for predicting the response to immunotherapy in patients with cancer. Gold-standard measurement of TMB is performed using whole exome sequencing (WES), which is not available at most hospitals because of its high cost, operational complexity and long turnover times. We have developed a machine learning algorithm, Image2TMB, which can predict TMB from readily available lung adenocarcinoma histopathological images. Image2TMB integrates the predictions of three deep learning models that operate at different resolution scales (×5, ×10 and ×20 magnification) to determine if the TMB of a cancer is high or low. On a held-out set of patients, Image2TMB achieves an area under the precision recall curve of 0.92, an average precision of 0.89, and has the predictive power of a targeted sequencing panel of ~100 genes. This study demonstrates that it is possible to infer genomic features from histopathology images, and potentially opens avenues for exploring genotype–phenotype relationships. Tumour mutational burden (TMB) shows promise as a biomarker in cancer immunotherapy, but it usually requires whole-exome sequencing, which is costly, time-consuming and unavailable at most hospitals. The authors develop a machine learning algorithm that uses standard H&amp;amp;E histopathological images to quickly, inexpensively and accurately predict TMB. The approach may have applications as a tool to screen and prioritize patient samples and subsequent treatments.},
  archive      = {J_NATMI},
  author       = {Jain, Mika S. and Massoud, Tarik F.},
  doi          = {10.1038/s42256-020-0190-5},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {356-362},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting tumour mutational burden from histopathological images using multiscale deep learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A novel machine learning framework for automated biomedical
relation extraction from large-scale literature repositories.
<em>NATMI</em>, <em>2</em>(6), 347–355. (<a
href="https://doi.org/10.1038/s42256-020-0189-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Knowledge about the relations between biomedical entities (such as drugs and targets) is widely distributed in more than 30 million research articles and consistently plays an important role in the development of biomedical science. In this work, we propose a novel machine learning framework, named BERE, for automatically extracting biomedical relations from large-scale literature repositories. BERE uses a hybrid encoding network to better represent each sentence from both semantic and syntactic aspects, and employs a feature aggregation network to make predictions after considering all relevant statements. More importantly, BERE can also be trained without any human annotation via a distant supervision technique. Through extensive tests, BERE has demonstrated promising performance in extracting biomedical relations, and can also find meaningful relations that were not reported in existing databases, thus providing useful hints to guide wet-lab experiments and advance the biological knowledge discovery process. A lot of scientific literature is unstructured, which makes extracting information for biomedical databases difficult. Hong and colleagues show that a distant supervision approach, using latent tree learning and recurrent units, can extract drug–target interactions from literature that were previously unknown.},
  archive      = {J_NATMI},
  author       = {Hong, Lixiang and Lin, Jinjian and Li, Shuya and Wan, Fangping and Yang, Hui and Jiang, Tao and Zhao, Dan and Zeng, Jianyang},
  doi          = {10.1038/s42256-020-0189-y},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {347-355},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A novel machine learning framework for automated biomedical relation extraction from large-scale literature repositories},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Augmenting vascular disease diagnosis by vasculature-aware
unsupervised learning. <em>NATMI</em>, <em>2</em>(6), 337–346. (<a
href="https://doi.org/10.1038/s42256-020-0188-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vascular disease is one of the leading causes of death and threatens human health worldwide. Imaging examination of vascular pathology with reduced invasiveness is challenging due to the intrinsic vasculature complexity and non-uniform scattering from bio-tissues. Here, we report VasNet, a vasculature-aware unsupervised learning algorithm that augments pathovascular recognition from small sets of unlabelled fluorescence and digital subtraction angiography images. VasNet adopts a multi-scale fusion strategy with a domain adversarial neural network loss function that induces biased pattern reconstruction by strengthening features relevant to the retinal vasculature reference while weakening irrelevant features. VasNet delivers the outputs ‘Structure + X’ (where X refers to multi-dimensional features such as blood flows, the distinguishment of blood dilation and its suspicious counterparts, and the dependence of new pattern emergence on disease progression). Therefore, explainable imaging output from VasNet and other algorithm extensions holds the promise to augment medical diagnosis, as it improves performance while reducing the cost of human expertise, equipment and time consumption. Vascular abnormalities are challenging for diagnostic imaging due to the complexity of vasculature and the non-uniform scattering from biological tissues. The authors present an unsupervised learning algorithm for vascular feature recognition from small sets of biomedical images acquired from different modalities. They demonstrate the utility of their diagnostic approach on vascular images of thrombosis, internal bleeding and colitis.},
  archive      = {J_NATMI},
  author       = {Wang, Yong and Ji, Mengqi and Jiang, Shengwei and Wang, Xukang and Wu, Jiamin and Duan, Feng and Fan, Jingtao and Huang, Laiqiang and Ma, Shaohua and Fang, Lu and Dai, Qionghai},
  doi          = {10.1038/s42256-020-0188-z},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {337-346},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Augmenting vascular disease diagnosis by vasculature-aware unsupervised learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep learning incorporating biologically inspired neural
dynamics and in-memory computing. <em>NATMI</em>, <em>2</em>(6),
325–336. (<a href="https://doi.org/10.1038/s42256-020-0187-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spiking neural networks (SNNs) incorporating biologically plausible neurons hold great promise because of their unique temporal dynamics and energy efficiency. However, SNNs have developed separately from artificial neural networks (ANNs), limiting the impact of deep learning advances for SNNs. Here, we present an alternative perspective of the spiking neuron that incorporates its neural dynamics into a recurrent ANN unit called a spiking neural unit (SNU). SNUs may operate as SNNs, using a step function activation, or as ANNs, using continuous activations. We demonstrate the advantages of SNU dynamics through simulations on multiple tasks and obtain accuracies comparable to, or better than, those of ANNs. The SNU concept enables an efficient implementation with in-memory acceleration for both training and inference. We experimentally demonstrate its efficacy for a music-prediction task in an in-memory-based SNN accelerator prototype using 52,800 phase-change memory devices. Our results open up an avenue for broad adoption of biologically inspired neural dynamics in challenging applications and acceleration with neuromorphic hardware. Spiking neural networks and in-memory computing are both promising routes towards energy-efficient hardware for deep learning. Woźniak et al. incorporate the biologically inspired dynamics of spiking neurons into conventional recurrent neural network units and in-memory computing, and show how this allows for accurate and energy-efficient deep learning.},
  archive      = {J_NATMI},
  author       = {Woźniak, Stanisław and Pantazi, Angeliki and Bohnstingl, Thomas and Eleftheriou, Evangelos},
  doi          = {10.1038/s42256-020-0187-0},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {325-336},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning incorporating biologically inspired neural dynamics and in-memory computing},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Finding key players in complex networks through deep
reinforcement learning. <em>NATMI</em>, <em>2</em>(6), 317–324. (<a
href="https://doi.org/10.1038/s42256-020-0177-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finding an optimal set of nodes, called key players, whose activation (or removal) would maximally enhance (or degrade) a certain network functionality, is a fundamental class of problems in network science. Potential applications include network immunization, epidemic control, drug design and viral marketing. Due to their general NP-hard nature, these problems typically cannot be solved by exact algorithms with polynomial time complexity. Many approximate and heuristic strategies have been proposed to deal with specific application scenarios. Yet, we still lack a unified framework to efficiently solve this class of problems. Here, we introduce a deep reinforcement learning framework FINDER, which can be trained purely on small synthetic networks generated by toy models and then applied to a wide spectrum of application scenarios. Extensive experiments under various problem settings demonstrate that FINDER significantly outperforms existing methods in terms of solution quality. Moreover, it is several orders of magnitude faster than existing methods for large networks. The presented framework opens up a new direction of using deep learning techniques to understand the organizing principle of complex networks, which enables us to design more robust networks against both attacks and failures. A fundamental problem in network science is how to find an optimal set of key players whose activation or removal significantly impacts network functionality. The authors propose a deep reinforcement learning framework that can be trained on small networks to understand the organizing principles of complex networked systems.},
  archive      = {J_NATMI},
  author       = {Fan, Changjun and Zeng, Li and Sun, Yizhou and Liu, Yang-Yu},
  doi          = {10.1038/s42256-020-0177-2},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {317-324},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Finding key players in complex networks through deep reinforcement learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Towards a new generation of artificial intelligence in
china. <em>NATMI</em>, <em>2</em>(6), 312–316. (<a
href="https://doi.org/10.1038/s42256-020-0183-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence has become a main driving force for a new round of industrial transformation around the world. Many countries including China are seizing the opportunity of the AI revolution to promote domestic economic and technological development. This Perspective briefly introduces the New Generation Artificial Intelligence Development Plan of China (2015–2030) from the point of view of the authors, a group of AI experts from academia and industry who have been involved in various stages of the plan. China’s AI development plan outlines a strategy for science and technology as well as education, tackling a number of challenges such as retaining talent, advancing fundamental research and exploring ethical issues. The New Generation Artificial Intelligence Development Plan is intended to be a blueprint for a complete AI ecosystem for the country. China’s New Generation Artificial Intelligence Development Plan was launched in 2017 and lays out an ambitious strategy, which intends to make China one of the world’s premier AI innovation centre by 2030. This Perspective presents the view from a group of Chinese AI experts from academia and industry about the origins of the plan, the motivations and main focus for attention from research and industry.},
  archive      = {J_NATMI},
  author       = {Wu, Fei and Lu, Cewu and Zhu, Mingjie and Chen, Hao and Zhu, Jun and Yu, Kai and Li, Lei and Li, Ming and Chen, Qianfeng and Li, Xi and Cao, Xudong and Wang, Zhongyuan and Zha, Zhengjun and Zhuang, Yueting and Pan, Yunhe},
  doi          = {10.1038/s42256-020-0183-4},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {312-316},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards a new generation of artificial intelligence in china},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Secure, privacy-preserving and federated machine learning in
medical imaging. <em>NATMI</em>, <em>2</em>(6), 305–311. (<a
href="https://doi.org/10.1038/s42256-020-0186-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The broad application of artificial intelligence techniques in medicine is currently hindered by limited dataset availability for algorithm training and validation, due to the absence of standardized electronic medical records, and strict legal and ethical requirements to protect patient privacy. In medical imaging, harmonized data exchange formats such as Digital Imaging and Communication in Medicine and electronic data storage are the standard, partially addressing the first issue, but the requirements for privacy preservation are equally strict. To prevent patient privacy compromise while promoting scientific research on large datasets that aims to improve patient care, the implementation of technical solutions to simultaneously address the demands for data protection and utilization is mandatory. Here we present an overview of current and next-generation methods for federated, secure and privacy-preserving artificial intelligence with a focus on medical imaging applications, alongside potential attack vectors and future prospects in medical imaging and beyond. Medical imaging data is often subject to privacy and intellectual property restrictions. AI techniques can help out by offering tools like federated learning to bridge the gap between personal data protection and data utilisation for research and clinical routine, but these tools need to be secure.},
  archive      = {J_NATMI},
  author       = {Kaissis, Georgios A. and Makowski, Marcus R. and Rückert, Daniel and Braren, Rickmer F.},
  doi          = {10.1038/s42256-020-0186-1},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {305-311},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Secure, privacy-preserving and federated machine learning in medical imaging},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Tracking the debate on COVID-19 surveillance tools.
<em>NATMI</em>, <em>2</em>(6), 301–304. (<a
href="https://doi.org/10.1038/s42256-020-0194-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Contact-tracing apps could help keep countries open before a vaccine is available. But do we have a sufficient understanding of their efficacy, and can we balance protecting public health with safeguarding civil rights? We interviewed five experts, with backgrounds in digital health ethics, internet law and social sciences.},
  archive      = {J_NATMI},
  author       = {Sweeney, Yann},
  doi          = {10.1038/s42256-020-0194-1},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {301-304},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Tracking the debate on COVID-19 surveillance tools},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The challenges of deploying artificial intelligence models
in a rapidly evolving pandemic. <em>NATMI</em>, <em>2</em>(6), 298–300.
(<a href="https://doi.org/10.1038/s42256-020-0185-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The attention and resources of AI researchers have been captured by COVID-19. However, successful adoption of AI models in the fight against the pandemic is facing various challenges, including moving clinical needs as the epidemic progresses and the necessity to translate models to local healthcare situations.},
  archive      = {J_NATMI},
  author       = {Hu, Yipeng and Jacob, Joseph and Parker, Geoffrey J. M. and Hawkes, David J. and Hurst, John R. and Stoyanov, Danail},
  doi          = {10.1038/s42256-020-0185-2},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {298-300},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The challenges of deploying artificial intelligence models in a rapidly evolving pandemic},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Artificial intelligence cooperation to support the global
response to COVID-19. <em>NATMI</em>, <em>2</em>(6), 295–297. (<a
href="https://doi.org/10.1038/s42256-020-0184-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In an unprecedented effort of scientific collaboration, researchers across fields are racing to support the response to COVID-19. Making a global impact with AI tools will require scalable approaches for data, model and code sharing; adapting applications to local contexts; and cooperation across borders.},
  archive      = {J_NATMI},
  author       = {Luengo-Oroz, Miguel and Hoffmann Pham, Katherine and Bullock, Joseph and Kirkpatrick, Robert and Luccioni, Alexandra and Rubel, Sasha and Wachholz, Cedric and Chakchouk, Moez and Biggs, Phillippa and Nguyen, Tim and Purnat, Tina and Mariano, Bernardo},
  doi          = {10.1038/s42256-020-0184-3},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {295-297},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Artificial intelligence cooperation to support the global response to COVID-19},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Machine learning for COVID-19 needs global collaboration and
data-sharing. <em>NATMI</em>, <em>2</em>(6), 293–294. (<a
href="https://doi.org/10.1038/s42256-020-0181-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COVID-19 pandemic poses a historical challenge to society. The profusion of data requires machine learning to improve and accelerate COVID-19 diagnosis, prognosis and treatment. However, a global and open approach is necessary to avoid pitfalls in these applications.},
  archive      = {J_NATMI},
  author       = {Peiffer-Smadja, Nathan and Maatoug, Redwan and Lescure, François-Xavier and D’Ortenzio, Eric and Pineau, Joëlle and King, Jean-Rémi},
  doi          = {10.1038/s42256-020-0181-6},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {293-294},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine learning for COVID-19 needs global collaboration and data-sharing},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Speeding up coral reef conservation with AI-aided automated
image analysis. <em>NATMI</em>, <em>2</em>(6), 292. (<a
href="https://doi.org/10.1038/s42256-020-0192-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Nunes, José Anchieta C. C. and Cruz, Igor C. S. and Nunes, André and Pinheiro, Hudson T.},
  doi          = {10.1038/s42256-020-0192-3},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {292},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Speeding up coral reef conservation with AI-aided automated image analysis},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Finding a role for AI in the pandemic. <em>NATMI</em>,
<em>2</em>(6), 291. (<a
href="https://doi.org/10.1038/s42256-020-0196-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expectations are high for AI to help fight COVID-19. But before AI tools can make an impact, global collaboration and high-quality data and model sharing are needed.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0196-z},
  journal      = {Nature Machine Intelligence},
  number       = {6},
  pages        = {291},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Finding a role for AI in the pandemic},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: Pandemic data challenges.
<em>NATMI</em>, <em>2</em>(5), 289. (<a
href="https://doi.org/10.1038/s42256-020-0179-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0179-0},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {289},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Pandemic data challenges},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An interpretable mortality prediction model for COVID-19
patients. <em>NATMI</em>, <em>2</em>(5), 283–288. (<a
href="https://doi.org/10.1038/s42256-020-0180-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The sudden increase in COVID-19 cases is putting high pressure on healthcare services worldwide. At this stage, fast, accurate and early clinical assessment of the disease severity is vital. To support decision making and logistical planning in healthcare systems, this study leverages a database of blood samples from 485 infected patients in the region of Wuhan, China, to identify crucial predictive biomarkers of disease mortality. For this purpose, machine learning tools selected three biomarkers that predict the mortality of individual patients more than 10 days in advance with more than 90\% accuracy: lactic dehydrogenase (LDH), lymphocyte and high-sensitivity C-reactive protein (hs-CRP). In particular, relatively high levels of LDH alone seem to play a crucial role in distinguishing the vast majority of cases that require immediate medical attention. This finding is consistent with current medical knowledge that high LDH levels are associated with tissue breakdown occurring in various diseases, including pulmonary disorders such as pneumonia. Overall, this Article suggests a simple and operable decision rule to quickly predict patients at the highest risk, allowing them to be prioritized and potentially reducing the mortality rate. Early and accurate clinical assessment of disease severity in COVID-19 patients is essential for planning the allocation of scarce hospital resources. An explainable machine learning tool trained on blood sample data from 485 patients from Wuhan selected three biomarkers for predicting mortality of individual patients with high accuracy.},
  archive      = {J_NATMI},
  author       = {Yan, Li and Zhang, Hai-Tao and Goncalves, Jorge and Xiao, Yang and Wang, Maolin and Guo, Yuqi and Sun, Chuan and Tang, Xiuchuan and Jing, Liang and Zhang, Mingyang and Huang, Xiang and Xiao, Ying and Cao, Haosen and Chen, Yanyan and Ren, Tongxin and Wang, Fang and Xiao, Yaru and Huang, Sufang and Tan, Xi and Huang, Niannian and Jiao, Bo and Cheng, Cheng and Zhang, Yong and Luo, Ailin and Mombaerts, Laurent and Jin, Junyang and Cao, Zhiguo and Li, Shusheng and Xu, Hui and Yuan, Ye},
  doi          = {10.1038/s42256-020-0180-7},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {283-288},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An interpretable mortality prediction model for COVID-19 patients},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A shallow convolutional neural network predicts prognosis of
lung cancer patients in multi-institutional computed tomography image
datasets. <em>NATMI</em>, <em>2</em>(5), 274–282. (<a
href="https://doi.org/10.1038/s42256-020-0173-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lung cancer is the most common fatal malignancy in adults worldwide, and non-small-cell lung cancer (NSCLC) accounts for 85\% of lung cancer diagnoses. Computed tomography is routinely used in clinical practice to determine lung cancer treatment and assess prognosis. Here, we developed LungNet, a shallow convolutional neural network for predicting outcomes of patients with NSCLC. We trained and evaluated LungNet on four independent cohorts of patients with NSCLC from four medical centres: Stanford Hospital (n = 129), H. Lee Moffitt Cancer Center and Research Institute (n = 185), MAASTRO Clinic (n = 311) and Charité – Universitätsmedizin, Berlin (n = 84). We show that outcomes from LungNet are predictive of overall survival in all four independent survival cohorts as measured by concordance indices of 0.62, 0.62, 0.62 and 0.58 on cohorts 1, 2, 3 and 4, respectively. Furthermore, the survival model can be used, via transfer learning, for classifying benign versus malignant nodules on the Lung Image Database Consortium (n = 1,010), with improved performance (AUC = 0.85) versus training from scratch (AUC = 0.82). LungNet can be used as a non-invasive predictor for prognosis in patients with NSCLC and can facilitate interpretation of computed tomography images for lung cancer stratification and prognostication. Predicting overall survival for patients with confirmed non-small-cell lung cancer is an important issue in clinical practice. The authors developed and validated in four independent patient cohorts a shallow convolutional neural network that can predict the outcomes of individuals using pre-treatment CT images. The authors further show that the survival model can be used, via transfer learning, for classifying benign versus malignant nodules.},
  archive      = {J_NATMI},
  author       = {Mukherjee, Pritam and Zhou, Mu and Lee, Edward and Schicht, Anne and Balagurunathan, Yoganand and Napel, Sandy and Gillies, Robert and Wong, Simon and Thieme, Alexander and Leung, Ann and Gevaert, Olivier},
  doi          = {10.1038/s42256-020-0173-6},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {274-282},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A shallow convolutional neural network predicts prognosis of lung cancer patients in multi-institutional computed tomography image datasets},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Improving healthcare operations management with machine
learning. <em>NATMI</em>, <em>2</em>(5), 266–273. (<a
href="https://doi.org/10.1038/s42256-020-0176-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Healthcare institutions need modern and powerful technology to provide high-quality, cost-effective care to patients. However, despite the considerable progress in the computerization and digitization of medicine, efficient and robust management tools have yet to materialize. One important reason for this is the extreme complexity and variability of healthcare operations, the needs of which have outgrown conventional management. Machine learning algorithms, scalable and adaptive to complex patterns, may be particularly well suited to solving these problems. Two major advantages of machine learning—the power of building strong models from a large number of weakly predictive features, and the ability to identify key factors in complex feature sets—have a particularly direct connection to the principal operational challenges. The main goal of this work was to study this relationship using two major types of operational problems: predicting operational events, and identifying key workflow drivers. Using practical examples, we demonstrate how machine learning can improve human ability to understand and manage healthcare operations, leading to more efficient healthcare. While computerization and digitization of medicine have advanced substantially, management tools in healthcare have not yet benefited much from these developments due to the extreme complexity and variability of healthcare operations. The ability of machine learning algorithms to build strong models from a large number of weakly predictive features, and to identify key factors in complex feature sets, is tested in operational problems involving hospital datasets on workflow and patient waiting time.},
  archive      = {J_NATMI},
  author       = {Pianykh, Oleg S. and Guitron, Steven and Parke, Darren and Zhang, Chengzhao and Pandharipande, Pari and Brink, James and Rosenthal, Daniel},
  doi          = {10.1038/s42256-020-0176-3},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {266-273},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Improving healthcare operations management with machine learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Direct steering of de novo molecular generation with
descriptor conditional recurrent neural networks. <em>NATMI</em>,
<em>2</em>(5), 254–265. (<a
href="https://doi.org/10.1038/s42256-020-0174-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning has acquired considerable momentum over the past couple of years in the domain of de novo drug design. Here, we propose a simple approach to the task of focused molecular generation for drug design purposes by constructing a conditional recurrent neural network (cRNN). We aggregate selected molecular descriptors and transform them into the initial memory state of the network before starting the generation of alphanumeric strings that describe molecules. We thus tackle the inverse design problem directly, as the cRNNs may generate molecules near the specified conditions. Moreover, we exemplify a novel way of assessing the focus of the conditional output of such a model using negative log-likelihood plots. The output is more focused than traditional unbiased RNNs, yet less focused than autoencoders, thus representing a novel method with intermediate output specificity between well-established methods. Conceptually, our architecture shows promise for the generalized problem of steering of sequential data generation with recurrent neural networks. The rise of deep neural networks allows for new ways to design molecules that interact with biological structures. An approach that uses conditional recurrent neural networks generates molecules with properties near specified conditions.},
  archive      = {J_NATMI},
  author       = {Kotsias, Panagiotis-Christos and Arús-Pous, Josep and Chen, Hongming and Engkvist, Ola and Tyrchan, Christian and Bjerrum, Esben Jannik},
  doi          = {10.1038/s42256-020-0174-5},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {254-265},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Complex sequential understanding through the awareness of
spatial and temporal concepts. <em>NATMI</em>, <em>2</em>(5), 245–253.
(<a href="https://doi.org/10.1038/s42256-020-0168-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding sequential information is a fundamental task for artificial intelligence. Current neural networks attempt to learn spatial and temporal information as a whole, limiting their abilities to represent large-scale spatial representations over long-range sequences. Here, we introduce a new modelling strategy—‘semi-coupled structure’ (SCS)—which consists of deep neural networks that decouple the complex spatial and temporal concepts during learning. SCS can learn to implicitly separate input information into independent parts and process these parts separately. Experiments demonstrate that SCS can successfully sequentially annotate the outline of an object in images and perform video action recognition. As an example of sequence-to-sequence problems, SCS can predict future meteorological radar echo images based on observed images. Taken together, our results demonstrate that SCS has the capacity to improve the performance of long short-term memory (LSTM)-like models on large-scale sequential tasks. Current neural networks attempt to learn spatial and temporal information as a whole, limiting their ability to process complex video data. Pang et al. improve performance by introducing a network structure which learns to implicitly decouple complex spatial and temporal concepts.},
  archive      = {J_NATMI},
  author       = {Pang, Bo and Zha, Kaiwen and Cao, Hanwen and Tang, Jiajun and Yu, Minghui and Lu, Cewu},
  doi          = {10.1038/s42256-020-0168-3},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {245-253},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Complex sequential understanding through the awareness of spatial and temporal concepts},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Contributions from the catholic church to ethical
reflections in the digital era. <em>NATMI</em>, <em>2</em>(5), 242–244.
(<a href="https://doi.org/10.1038/s42256-020-0175-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Catholic Church is challenged by scientific and technological innovation but can help to integrate multiple voices in the ongoing dialogue regarding AI and machine ethics. In this context, a multidisciplinary working group brought together by the Church reflected on roboethics, explored the themes of embodiment, agency and intelligence.},
  archive      = {J_NATMI},
  author       = {Sinibaldi, Edoardo and Gastmans, Chris and Yáñez, Miguel and Lerner, Richard M. and Kovács, László and Casalone, Carlo and Pegoraro, Renzo and Paglia, Vincenzo},
  doi          = {10.1038/s42256-020-0175-4},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {242-244},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Contributions from the catholic church to ethical reflections in the digital era},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Robot teams stay safe with blockchains. <em>NATMI</em>,
<em>2</em>(5), 240–241. (<a
href="https://doi.org/10.1038/s42256-020-0178-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To deploy robot swarms in our daily lives, they need to be resilient to malfunctioning errors and protected against malicious attacks. Blockchain technology could provide an essential level of protection.},
  archive      = {J_NATMI},
  author       = {Reina, Andreagiovanni},
  doi          = {10.1038/s42256-020-0178-1},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {240-241},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Robot teams stay safe with blockchains},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A match for virtual conferences. <em>NATMI</em>,
<em>2</em>(5), 239. (<a
href="https://doi.org/10.1038/s42256-020-0182-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists have been getting concerned about the carbon footprint of international meetings and have been asking whether travelling to conferences is the best use of their time and funds. 2020 is turning out to be the year that many organizers decide to go virtual — and this was before COVID-19.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0182-5},
  journal      = {Nature Machine Intelligence},
  number       = {5},
  pages        = {239},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A match for virtual conferences},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: Intelligent feature engineering and
ontological mapping of brain tumour histomorphologies by deep learning.
<em>NATMI</em>, <em>2</em>(4), 237. (<a
href="https://doi.org/10.1038/s42256-019-0133-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NATMI},
  author       = {Faust, Kevin and Bala, Sudarshan and van Ommeren, Randy and Portante, Alessia and Qawahmed, Raniah Al and Djuric, Ugljesa and Diamandis, Phedias},
  doi          = {10.1038/s42256-019-0133-1},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {237},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Publisher correction: Intelligent feature engineering and ontological mapping of brain tumour histomorphologies by deep learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Evaluation of deep learning in non-coding
RNA classification. <em>NATMI</em>, <em>2</em>(4), 236. (<a
href="https://doi.org/10.1038/s42256-019-0143-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NATMI},
  author       = {Amin, Noorul and McGrath, Annette and Chen, Yi-Ping Phoebe},
  doi          = {10.1038/s42256-019-0143-z},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {236},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Author correction: Evaluation of deep learning in non-coding RNA classification},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep variational network for rapid 4D flow MRI
reconstruction. <em>NATMI</em>, <em>2</em>(4), 228–235. (<a
href="https://doi.org/10.1038/s42256-020-0165-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phase-contrast magnetic resonance imaging (MRI) provides time-resolved quantification of blood flow dynamics that can aid clinical diagnosis. Long in vivo scan times due to repeated three-dimensional (3D) volume sampling over cardiac phases and breathing cycles necessitate accelerated imaging techniques that leverage data correlations. Standard compressed sensing reconstruction methods require tuning of hyperparameters and are computationally expensive, which diminishes the potential reduction of examination times. We propose an efficient model-based deep neural reconstruction network and evaluate its performance on clinical aortic flow data. The network is shown to reconstruct undersampled 4D flow MRI data in under a minute on standard consumer hardware. Remarkably, the relatively low amounts of tunable parameters allowed the network to be trained on images from 11 reference scans while generalizing well to retrospective and prospective undersampled data for various acceleration factors and anatomies. 4D MRI scans can reconstruct cardiovascular flow, although they typically take many minutes, hindering real-time assessment. Vishnevskiy et al. develop a deep variational network to permit high-fidelity image reconstruction in a matter of seconds, allowing integration of 4D flow MRI into clinical workflows.},
  archive      = {J_NATMI},
  author       = {Vishnevskiy, Valery and Walheim, Jonas and Kozerke, Sebastian},
  doi          = {10.1038/s42256-020-0165-6},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {228-235},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep variational network for rapid 4D flow MRI reconstruction},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Exploring the limit of using a deep neural network on pileup
data for germline variant calling. <em>NATMI</em>, <em>2</em>(4),
220–227. (<a href="https://doi.org/10.1038/s42256-020-0167-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule sequencing technologies have emerged in recent years and revolutionized structural variant calling, complex genome assembly and epigenetic mark detection. However, the lack of a highly accurate small variant caller has limited these technologies from being more widely used. Here, we present Clair, the successor to Clairvoyante, a program for fast and accurate germline small variant calling, using single-molecule sequencing data. For Oxford Nanopore Technology data, Clair achieves better precision, recall and speed than several competing programs, including Clairvoyante, Longshot and Medaka. Through studying the missed variants and benchmarking intentionally overfitted models, we found that Clair may be approaching the limit of possible accuracy for germline small variant calling using pileup data and deep neural networks. Clair requires only a conventional central processing unit (CPU) for variant calling and is an open-source project available at https://github.com/HKU-BAL/Clair. A lack of accurate and efficient variant calling methods has held back single-molecule sequencing technologies from clinical applications. The authors present a deep-learning method for fast and accurate germline small variant calling, using single-molecule sequencing data.},
  archive      = {J_NATMI},
  author       = {Luo, Ruibang and Wong, Chak-Lim and Wong, Yat-Sing and Tang, Chi-Ian and Liu, Chi-Man and Leung, Chi-Ming and Lam, Tak-Wah},
  doi          = {10.1038/s42256-020-0167-4},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {220-227},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Exploring the limit of using a deep neural network on pileup data for germline variant calling},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A neural network trained for prediction mimics diverse
features of biological neurons and perception. <em>NATMI</em>,
<em>2</em>(4), 210–219. (<a
href="https://doi.org/10.1038/s42256-020-0170-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent work has shown that convolutional neural networks (CNNs) trained on image recognition tasks can serve as valuable models for predicting neural responses in primate visual cortex. However, these models typically require biologically infeasible levels of labelled training data, so this similarity must at least arise via different paths. In addition, most popular CNNs are solely feedforward, lacking a notion of time and recurrence, whereas neurons in visual cortex produce complex time-varying responses, even to static inputs. Towards addressing these inconsistencies with biology, here we study the emergent properties of a recurrent generative network that is trained to predict future video frames in a self-supervised manner. Remarkably, the resulting model is able to capture a wide variety of seemingly disparate phenomena observed in visual cortex, ranging from single-unit response dynamics to complex perceptual motion illusions, even when subjected to highly impoverished stimuli. These results suggest potentially deep connections between recurrent predictive neural network models and computations in the brain, providing new leads that can enrich both fields. The deep convolutional recurrent neural network ‘PredNet’ can be trained to predict future video frames in a self-supervised manner. A surprising result is that it captures a wide array of phenomena observed in natural neuronal systems, ranging from low-level visual cortical neuron response properties to high-level perceptual illusions, hinting at potential similarities between recurrent predictive neural network models and computations in the brain.},
  archive      = {J_NATMI},
  author       = {Lotter, William and Kreiman, Gabriel and Cox, David},
  doi          = {10.1038/s42256-020-0170-9},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {210-219},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A neural network trained for prediction mimics diverse features of biological neurons and perception},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Combining automated microfluidic experimentation with
machine learning for efficient polymerization design. <em>NATMI</em>,
<em>2</em>(4), 200–209. (<a
href="https://doi.org/10.1038/s42256-020-0166-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding polymerization reactions has challenges relating to the complexity of the systems, the hazards associated with the reagents, the environmental footprint of the operations and the highly nonlinear topologies of reaction spaces. In this work, we aim to present a new methodology for studying polymerization reactions using machine-learning-assisted automated microchemical reactors. A custom-designed rapidly prototyped microreactor is used in conjunction with automation and in situ infrared thermography for efficient, high-speed experimentation to map the reaction space of a zirconocene polymerization catalyst and obtain fundamental kinetic parameters. Chemical waste is decreased by two orders of magnitude and catalytic discovery is reduced from weeks to hours. Bayesian regularization backpropagation is used in conjunction with kinetic modelling to understand the reaction space and resultant technoeconomic topology. Here, we show that efficient microfluidic technology can be coupled with machine-learning algorithms to obtain high-fidelity datasets on a complex chemical reaction. Finding the best ratio of ingredients for polymerization reactions can be time consuming and wasteful. An automated microreactor process with integrated machine learning analysis initiates reactions, measures the resulting yield and cleans itself without human intervention. It can test concentrations of reagents systematically to find the combination with the highest production, while producing a low amount of waste.},
  archive      = {J_NATMI},
  author       = {Rizkin, Benjamin A. and Shkolnik, Albert S. and Ferraro, Neil J. and Hartman, Ryan L.},
  doi          = {10.1038/s42256-020-0166-5},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {200-209},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Combining automated microfluidic experimentation with machine learning for efficient polymerization design},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The imperative of interpretable machines. <em>NATMI</em>,
<em>2</em>(4), 197–199. (<a
href="https://doi.org/10.1038/s42256-020-0171-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As artificial intelligence becomes prevalent in society, a framework is needed to connect interpretability and trust in algorithm-assisted decisions, for a range of stakeholders.},
  archive      = {J_NATMI},
  author       = {Stoyanovich, Julia and Van Bavel, Jay J. and West, Tessa V.},
  doi          = {10.1038/s42256-020-0171-8},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {197-199},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The imperative of interpretable machines},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Exploring public opinion about liability and responsibility
in surgical robotics. <em>NATMI</em>, <em>2</em>(4), 194–196. (<a
href="https://doi.org/10.1038/s42256-020-0169-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As robotic systems become more autonomous, it gets less straightforward to determine liability when humans are harmed. This is an emerging challenge, with legal implications, in the field of surgical robotic systems. The iRobotSurgeon Survey explores public opinions about responsibility and liability in the area of surgical robotics.},
  archive      = {J_NATMI},
  author       = {Jamjoom, Aimun A. B. and Jamjoom, Ammer M. A. and Marcus, Hani J.},
  doi          = {10.1038/s42256-020-0169-2},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {194-196},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Exploring public opinion about liability and responsibility in surgical robotics},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Pandemic data challenges. <em>NATMI</em>, <em>2</em>(4),
193. (<a href="https://doi.org/10.1038/s42256-020-0172-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The worldwide outbreak of COVID-19 has led to great tragedy and poses unprecedented challenges for countries’ healthcare systems. Data has become an important instrument in the global fight against the unprecedented spread of the virus. But how will we ensure a return to previous forms of data privacy once the pandemic subsides?},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0172-7},
  journal      = {Nature Machine Intelligence},
  number       = {4},
  pages        = {193},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Pandemic data challenges},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Rapid online learning and robust recall in a neuromorphic
olfactory circuit. <em>NATMI</em>, <em>2</em>(3), 181–191. (<a
href="https://doi.org/10.1038/s42256-020-0159-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a neural algorithm for the rapid online learning and identification of odourant samples under noise, based on the architecture of the mammalian olfactory bulb and implemented on the Intel Loihi neuromorphic system. As with biological olfaction, the spike timing-based algorithm utilizes distributed, event-driven computations and rapid (one shot) online learning. Spike timing-dependent plasticity rules operate iteratively over sequential gamma-frequency packets to construct odour representations from the activity of chemosensor arrays mounted in a wind tunnel. Learned odourants then are reliably identified despite strong destructive interference. Noise resistance is further enhanced by neuromodulation and contextual priming. Lifelong learning capabilities are enabled by adult neurogenesis. The algorithm is applicable to any signal identification problem in which high-dimensional signals are embedded in unknown backgrounds. Integrating knowledge about the circuit-level organization of the brain into neuromorphic artificial systems is a challenging research problem. The authors present a neural algorithm for the learning of odourant signals and their robust identification under noise, based on the architecture of the mammalian olfactory bulb and implemented on the Intel Loihi neuromorphic system.},
  archive      = {J_NATMI},
  author       = {Imam, Nabil and Cleland, Thomas A.},
  doi          = {10.1038/s42256-020-0159-4},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {181-191},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Rapid online learning and robust recall in a neuromorphic olfactory circuit},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Generative molecular design in low data regimes.
<em>NATMI</em>, <em>2</em>(3), 171–180. (<a
href="https://doi.org/10.1038/s42256-020-0160-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative machine learning models sample molecules from chemical space without the need for explicit design rules. To enable the generative design of innovative molecular entities with limited training data, a deep learning framework for customized compound library generation is presented that aims to enrich and expand the pharmacologically relevant chemical space with drug-like molecular entities on demand. This de novo design approach combines best practices and was used to generate molecules that incorporate features of both bioactive synthetic compounds and natural products, which are a primary source of inspiration for drug discovery. The results show that the data-driven machine intelligence acquires implicit chemical knowledge and generates novel molecules with bespoke properties and structural diversity. The method is available as an open-access tool for medicinal and bioorganic chemistry. With the aid of deep learning, the space of chemical molecules, such as candidates for drugs, can be constrained to find new bioactive molecules. A new open source tool can generate libraries of novel molecules with user defined properties.},
  archive      = {J_NATMI},
  author       = {Moret, Michael and Friedrich, Lukas and Grisoni, Francesca and Merk, Daniel and Schneider, Gisbert},
  doi          = {10.1038/s42256-020-0160-y},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {171-180},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Generative molecular design in low data regimes},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Predicting the state of charge and health of batteries using
data-driven machine learning. <em>NATMI</em>, <em>2</em>(3), 161–170.
(<a href="https://doi.org/10.1038/s42256-020-0156-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning is a specific application of artificial intelligence that allows computers to learn and improve from data and experience via sets of algorithms, without the need for reprogramming. In the field of energy storage, machine learning has recently emerged as a promising modelling approach to determine the state of charge, state of health and remaining useful life of batteries. First, we review the two most studied types of battery models in the literature for battery state prediction: the equivalent circuit and physics-based models. Based on the current limitations of these models, we showcase the promise of various machine learning techniques for fast and accurate battery state prediction. Finally, we highlight the major challenges involved, especially in accurate modelling over length and time, performing in situ calculations and high-throughput data generation. Overall, this work provides insights into real-time, explainable machine learning for battery production, management and optimization in the future. Predicting the properties of batteries, such as their state of charge and remaining lifetime, is crucial for improving battery manufacturing, usage and optimisation for energy storage. The authors discuss how machine learning methods and high-throughput experimentation provide a data-driven approach to this problem, and highlight challenges in building models which provide fast and accurate battery state predictions.},
  archive      = {J_NATMI},
  author       = {Ng, Man-Fai and Zhao, Jin and Yan, Qingyu and Conduit, Gareth J. and Seh, Zhi Wei},
  doi          = {10.1038/s42256-020-0156-7},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {161-170},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting the state of charge and health of batteries using data-driven machine learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Human ownership of artificial creativity. <em>NATMI</em>,
<em>2</em>(3), 157–160. (<a
href="https://doi.org/10.1038/s42256-020-0161-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in generative algorithms have enhanced the quality and accessibility of artificial intelligence (AI) as a tool in building synthetic datasets. By generating photorealistic images and videos, these networks can pose a major technological disruption to a broad range of industries from medical imaging to virtual reality. However, as artwork developed by generative algorithms and cognitive robotics enters the arena, the notion of human-driven creativity has been thoroughly tested. When creativity is automated by the programmer, in a style determined by the trainer, using features from information available in public and private datasets, who is the proprietary owner of the rights in AI-generated artworks and designs? This Perspective seeks to provide an answer by systematically exploring the key issues in copyright law that arise at each phase of artificial creativity, from programming to deployment. Ultimately, four guiding actions are established for artists, programmers and end users that utilize AI as a tool such that they may be appropriately awarded the necessary proprietary rights. As artists are beginning to employ deep learning techniques to create new and interesting art, questions arise about how copyright and ownership apply to those works. This Perspective discusses how artists, programmers and users can ensure clarity about the ownership of their creations.},
  archive      = {J_NATMI},
  author       = {Eshraghian, Jason K.},
  doi          = {10.1038/s42256-020-0161-x},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {157-160},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Human ownership of artificial creativity},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An alternative to backpropagation through time.
<em>NATMI</em>, <em>2</em>(3), 155–156. (<a
href="https://doi.org/10.1038/s42256-020-0162-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recurrent networks can be trained using a generalization of backpropagation, called backpropagation through time, but a gap exists between the mathematics of this learning algorithm and biological plausibility. E-prop is a biologically inspired alternative that opens up possibilities for a new generation of online training algorithms for recurrent networks.},
  archive      = {J_NATMI},
  author       = {Manneschi, Luca and Vasilaki, Eleni},
  doi          = {10.1038/s42256-020-0162-9},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {155-156},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An alternative to backpropagation through time},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The path of foundational research in robotics.
<em>NATMI</em>, <em>2</em>(3), 153–154. (<a
href="https://doi.org/10.1038/s42256-020-0163-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Martial Hebert is the director of the Robotics Institute and dean of the School of Computer Science at Carnegie Mellon University. We spoke with him at the O’Reilly AI Conference in New York in April 2019, where he delivered a keynote talk, ‘AI and the robotics revolution’, and in a follow-up conversation.},
  archive      = {J_NATMI},
  author       = {Jerde, Trenton},
  doi          = {10.1038/s42256-020-0163-8},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {153-154},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The path of foundational research in robotics},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Queering machines. <em>NATMI</em>, <em>2</em>(3), 152. (<a
href="https://doi.org/10.1038/s42256-020-0157-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Poulsen, Adam and Fosch-Villaronga, Eduard and Søraa, Roger Andre},
  doi          = {10.1038/s42256-020-0157-6},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {152},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Queering machines},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Into the latent space. <em>NATMI</em>, <em>2</em>(3), 151.
(<a href="https://doi.org/10.1038/s42256-020-0164-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative deep learning can produce artificial, natural-looking images and other data, which has many promising applications in research — and in art. But the wide availability of generative models poses a challenge for society, which needs tools and best practices to distinguish between real and synthetic data.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0164-7},
  journal      = {Nature Machine Intelligence},
  number       = {3},
  pages        = {151},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Into the latent space},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep-learning-based prediction of late age-related macular
degeneration progression. <em>NATMI</em>, <em>2</em>(2), 141–150. (<a
href="https://doi.org/10.1038/s42256-020-0154-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Both genetic and environmental factors influence the etiology of age-related macular degeneration (AMD), a leading cause of blindness. AMD severity is primarily measured by images of the fundus of the retina and recently developed machine learning methods can successfully predict AMD progression using image data. However, none of these methods have used both genetic and image data for predicting AMD progression. Here we used both genotypes and fundus images to predict whether an eye had progressed to late AMD with a modified deep convolutional neural network. In total, we used 31,262 fundus images and 52 AMD-associated genetic variants from 1,351 subjects from the Age-Related Eye Disease Study, which provided disease severity phenotypes and fundus images available at baseline and follow-up visits over a period of 12 years. Our results showed that fundus images coupled with genotypes could predict late AMD progression with an averaged area-under-the-curve value of 0.85 (95\% confidence interval 0.83–0.86). The results using fundus images alone showed an averaged area under the receiver operating characteristic curve value of 0.81 (95\% confidence interval 0.80–0.83). We implemented our model in a cloud-based application for individual risk assessment. Age-related macular degeneration is a serious eye disease which should be detected as early as possible. Using both fundus images and genetic information, a deep neural network is able to detect the severity of the disease and predict its progression seven years into the future.},
  archive      = {J_NATMI},
  author       = {Yan, Qi and Weeks, Daniel E. and Xin, Hongyi and Swaroop, Anand and Chew, Emily Y. and Huang, Heng and Ding, Ying and Chen, Wei},
  doi          = {10.1038/s42256-020-0154-9},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {141-150},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep-learning-based prediction of late age-related macular degeneration progression},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Predicting drug–protein interaction using quasi-visual
question answering system. <em>NATMI</em>, <em>2</em>(2), 134–140. (<a
href="https://doi.org/10.1038/s42256-020-0152-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying novel drug–protein interactions is crucial for drug discovery. For this purpose, many machine learning-based methods have been developed based on drug descriptors and one-dimensional protein sequences. However, protein sequences cannot accurately reflect the interactions in three-dimensional space. However, direct input of three-dimensional structure is of low efficiency due to the sparse three-dimensional matrix, and is also prevented by the limited number of co-crystal structures available for training. Here we propose an end-to-end deep learning framework to predict the interactions by representing proteins with a two-dimensional distance map from monomer structures (Image) and drugs with molecular linear notation (String), following the visual question answering mode. For efficient training of the system, we introduce a dynamic attentive convolutional neural network to learn fixed-size representations from the variable-length distance maps and a self-attentional sequential model to automatically extract semantic features from the linear notations. Extensive experiments demonstrate that our model obtains competitive performance against state-of-the-art baselines on the directory of useful decoys, enhanced (DUD-E), human and BindingDB benchmark datasets. Further attention visualization provides biological interpretation to depict highlighted regions of both protein and drug molecules. When predicting the interaction of proteins with potential drugs, the protein can be encoded as its one-dimensional sequence or a three-dimensional structure, which can capture more relevant features of the protein, but also makes the task to predict the interactions harder. A new method predicts these interactions using a two-dimensional distance matrix representation of a protein, which can be processed like a two-dimensional image, striking a balance between the data being simple to process and rich in relevant structures.},
  archive      = {J_NATMI},
  author       = {Zheng, Shuangjia and Li, Yongjian and Chen, Sheng and Xu, Jun and Yang, Yuedong},
  doi          = {10.1038/s42256-020-0152-y},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {134-140},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Predicting drug–protein interaction using quasi-visual question answering system},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep learning of circulating tumour cells. <em>NATMI</em>,
<em>2</em>(2), 124–133. (<a
href="https://doi.org/10.1038/s42256-020-0153-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circulating tumour cells (CTCs) found in the blood of cancer patients are a promising biomarker in precision medicine. However, their use is currently hindered by their low frequency, tedious manual scoring and extensive cell heterogeneities. Those challenges limit the effectiveness of classical machine-learning methods for automated CTC analysis. Here, we combine autoencoding convolutional neural networks with advanced visualization techniques. This provides a very informative view on the data that opens the way for new biomedical research questions. We unravel hidden information in the raw image data of fluorescent images of blood samples enriched for CTCs. Our network classifies fluorescent images of single cells in five different classes with an accuracy, sensitivity and specificity of over 96\%, and the obtained CTC counts predict the overall survival of cancer patients as well as state-of-the-art manual counts. Moreover, our network excelled in identifying different important subclasses of objects. Deep learning was faster and superior to classical image analysis approaches and enabled the identification of new biological phenomena. Counting different types of circulating tumour cells can give valuable information on the severity of the disease and on whether treatments are effective for a specific patient. In this work, the authors show that their method based on autoencoders can identify and count cells more accurately and faster than human experts.},
  archive      = {J_NATMI},
  author       = {Zeune, Leonie L. and Boink, Yoeri E. and van Dalum, Guus and Nanou, Afroditi and de Wit, Sanne and Andree, Kiki C. and Swennenhuis, Joost F. and van Gils, Stephan A. and Terstappen, Leon W.M.M. and Brune, Christoph},
  doi          = {10.1038/s42256-020-0153-x},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {124-133},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning of circulating tumour cells},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A topology-based network tree for the prediction of
protein–protein binding affinity changes following mutation.
<em>NATMI</em>, <em>2</em>(2), 116–123. (<a
href="https://doi.org/10.1038/s42256-020-0149-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to predict protein–protein interactions is crucial to our understanding of a wide range of biological activities and functions in the human body, and for guiding drug discovery. Despite considerable efforts to develop suitable computational methods, predicting protein–protein interaction binding affinity changes following mutation (ΔΔG) remains a severe challenge. Algebraic topology, a champion in recent worldwide competitions for protein–ligand binding affinity predictions, is a promising approach to simplifying the complexity of biological structures. Here we introduce element- and site-specific persistent homology (a new branch of algebraic topology) to simplify the structural complexity of protein–protein complexes and embed crucial biological information into topological invariants. We also propose a new deep learning algorithm called NetTree to take advantage of convolutional neural networks and gradient-boosting trees. A topology-based network tree is constructed by integrating the topological representation and NetTree for predicting protein–protein interaction ΔΔG. Tests on major benchmark datasets indicate that the proposed topology-based network tree is an important improvement over the current state of the art in predicting ΔΔG. Persistent homology provides an efficient approach to simplifying the complexity of protein structure. Wang et al. combine this approach with convolutional neural networks and gradient-boosting trees to improve predictions of protein–protein interactions.},
  archive      = {J_NATMI},
  author       = {Wang, Menglun and Cang, Zixuan and Wei, Guo-Wei},
  doi          = {10.1038/s42256-020-0149-6},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {116-123},
  shortjournal = {Nat. Mach. Intell.},
  title        = {A topology-based network tree for the prediction of protein–protein binding affinity changes following mutation},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep learning robotic guidance for autonomous vascular
access. <em>NATMI</em>, <em>2</em>(2), 104–115. (<a
href="https://doi.org/10.1038/s42256-020-0148-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Medical robots have demonstrated the ability to manipulate percutaneous instruments into soft tissue anatomy while working beyond the limits of human perception and dexterity. Robotic technologies further offer the promise of autonomy in carrying out critical tasks with minimal supervision when resources are limited. Here, we present a portable robotic device capable of introducing needles and catheters into deformable tissues such as blood vessels to draw blood or deliver fluids autonomously. Robotic cannulation is driven by predictions from a series of deep convolutional neural networks that encode spatiotemporal information from multimodal image sequences to guide real-time servoing. We demonstrate, through imaging and robotic tracking studies in volunteers, the ability of the device to segment, classify, localize and track peripheral vessels in the presence of anatomical variability and motion. We then evaluate robotic performance in phantom and animal models of difficult vascular access and show that the device can improve success rates and procedure times compared to manual cannulations by trained operators, particularly in challenging physiological conditions. These results suggest the potential for autonomous systems to outperform humans on complex visuomotor tasks, and demonstrate a step in the translation of such capabilities into clinical use. Getting safe and fast access to blood vessels is vital to many methods of treatment and diagnosis in medicine. Robot-assisted or even fully autonomous methods can potentially do the task more reliably than humans, especially when veins are hard to detect. In this work, a method is tested that uses deep learning to find blood vessels and track the movement of a patient’s arm.},
  archive      = {J_NATMI},
  author       = {Chen, Alvin I. and Balter, Max L. and Maguire, Timothy J. and Yarmush, Martin L.},
  doi          = {10.1038/s42256-020-0148-7},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {104-115},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep learning robotic guidance for autonomous vascular access},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Machine learning for active matter. <em>NATMI</em>,
<em>2</em>(2), 94–103. (<a
href="https://doi.org/10.1038/s42256-020-0146-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The availability of large datasets has boosted the application of machine learning in many fields and is now starting to shape active-matter research as well. Machine learning techniques have already been successfully applied to active-matter data—for example, deep neural networks to analyse images and track objects, and recurrent nets and random forests to analyse time series. Yet machine learning can also help to disentangle the complexity of biological active matter, helping, for example, to establish a relation between genetic code and emergent bacterial behaviour, to find navigation strategies in complex environments, and to map physical cues to animal behaviours. In this Review, we highlight the current state of the art in the application of machine learning to active matter and discuss opportunities and challenges that are emerging. We also emphasize how active matter and machine learning can work together for mutual benefit. This Review surveys machine learning techniques that are currently developed for a range of research topics in biological and artificial active matter and also discusses challenges and exciting opportunities. This research direction promises to help disentangle the complexity of active matter and gain fundamental insights for instance in collective behaviour of systems at many length scales from colonies of bacteria to animal flocks.},
  archive      = {J_NATMI},
  author       = {Cichos, Frank and Gustavsson, Kristian and Mehlig, Bernhard and Volpe, Giovanni},
  doi          = {10.1038/s42256-020-0146-9},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {94-103},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine learning for active matter},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Conceptual systems align to aid concept learning.
<em>NATMI</em>, <em>2</em>(2), 92–93. (<a
href="https://doi.org/10.1038/s42256-020-0145-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our understanding of concepts can differ depending on the modality — such as vision, text or speech — through which we learn this concept. A recent study uses computational modelling to demonstrate how conceptual understanding aligns across modalities.},
  archive      = {J_NATMI},
  author       = {Horst, Jessica S. and Bird, Chris M.},
  doi          = {10.1038/s42256-020-0145-x},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {92-93},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Conceptual systems align to aid concept learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Advancing impact assessment for intelligent systems.
<em>NATMI</em>, <em>2</em>(2), 89–91. (<a
href="https://doi.org/10.1038/s42256-020-0151-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the rise of AI technologies in society, we need a human impact assessment for technology.},
  archive      = {J_NATMI},
  author       = {Calvo, Rafael A. and Peters, Dorian and Cave, Stephen},
  doi          = {10.1038/s42256-020-0151-z},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {89-91},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Advancing impact assessment for intelligent systems},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An online platform for interactive feedback in biomedical
machine learning. <em>NATMI</em>, <em>2</em>(2), 86–88. (<a
href="https://doi.org/10.1038/s42256-020-0147-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning models have great potential in biomedical applications. A new platform called GradioHub offers an interactive and intuitive way for clinicians and biomedical researchers to try out models and test their reliability on real-world, out-of-training data.},
  archive      = {J_NATMI},
  author       = {Abid, Abubakar and Abdalla, Ali and Abid, Ali and Khan, Dawood and Alfozan, Abdulrahman and Zou, James},
  doi          = {10.1038/s42256-020-0147-8},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {86-88},
  shortjournal = {Nat. Mach. Intell.},
  title        = {An online platform for interactive feedback in biomedical machine learning},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The citizen at the centre of ethics. <em>NATMI</em>,
<em>2</em>(2), 85. (<a
href="https://doi.org/10.1038/s42256-020-0150-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Spatola, Nicolas},
  doi          = {10.1038/s42256-020-0150-0},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {85},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The citizen at the centre of ethics},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Machine learning could improve innovation policy.
<em>NATMI</em>, <em>2</em>(2), 84. (<a
href="https://doi.org/10.1038/s42256-020-0155-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATMI},
  author       = {Furman, Jeffrey L. and Teodoridis, Florenta},
  doi          = {10.1038/s42256-020-0155-8},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {84},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Machine learning could improve innovation policy},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The only game in town. <em>NATMI</em>, <em>2</em>(2), 83.
(<a href="https://doi.org/10.1038/s42256-020-0158-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In a recent workshop at the Conference on Neural Information Processing Systems (NeurIPS), future directions at the intersection of neuroscience and AI were considered. A panel discussion at the end of the day started with a provocative question: do we need AI to understand the brain?},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0158-5},
  journal      = {Nature Machine Intelligence},
  number       = {2},
  pages        = {83},
  shortjournal = {Nat. Mach. Intell.},
  title        = {The only game in town},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Learning as the unsupervised alignment of conceptual
systems. <em>NATMI</em>, <em>2</em>(1), 76–82. (<a
href="https://doi.org/10.1038/s42256-019-0132-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concept induction requires the extraction and naming of concepts from noisy perceptual experience. For supervised approaches, as the number of concepts grows, so does the number of required training examples. Philosophers, psychologists and computer scientists have long recognized that children can learn to label objects without being explicitly taught. In a series of computational experiments, we highlight how information in the environment can be used to build and align conceptual systems. Unlike supervised learning, the learning problem becomes easier the more concepts and systems there are to master. The key insight is that each concept has a unique signature within one conceptual system (for example, images) that is recapitulated in other systems (for example, text or audio). As predicted, children’s early concepts form readily aligned systems. By assembling conceptual systems from real-word datasets of text, images and audio, Roads and Love propose that objects embedded within a conceptual system have a unique signature that allows for conceptual systems to be aligned in an unsupervised fashion.},
  archive      = {J_NATMI},
  author       = {Roads, Brett D. and Love, Bradley C.},
  doi          = {10.1038/s42256-019-0132-2},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {76-82},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning as the unsupervised alignment of conceptual systems},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Large-scale automated investigation of free-falling paper
shapes via iterative physical experimentation. <em>NATMI</em>,
<em>2</em>(1), 68–75. (<a
href="https://doi.org/10.1038/s42256-019-0135-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Free-falling paper shapes exhibit rich, complex and varied behaviours that are extremely challenging to model analytically. Physical experimentation aids in system understanding, but is time-consuming, sensitive to initial conditions and reliant on subjective visual behavioural classification. In this study, robotics, computer vision and machine learning are used to autonomously fabricate, drop, analyse and classify the behaviours of hundreds of shapes. The system is validated by reproducing results for falling discs, which exhibit four falling styles: tumbling, chaotic, steady and periodic. A previously determined mapping from a non-dimensional parameter space to behaviour groups is shown to be consistent with these new experiments for tumbling and chaotic behaviours. However, steady or periodic behaviours are observed in previously unseen areas of the parameter space. More complex hexagon, square and cross shapes are investigated, showing that the non-dimensional parameter space generalizes to these shapes. The system highlights the potential of robotics for the investigation of complex physical systems, of which falling paper is one example, and provides a template for future investigation of such systems. The dynamics of paper shapes in free fall are still not fully understood, despite being discussed for more than 150 years. Collecting large amounts of data has the potential to give us new insights and a robotics system could generate and analyse data in large quantities.},
  archive      = {J_NATMI},
  author       = {Howison, Toby and Hughes, Josie and Iida, Fumiya},
  doi          = {10.1038/s42256-019-0135-z},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {68-75},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Large-scale automated investigation of free-falling paper shapes via iterative physical experimentation},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). From local explanations to global understanding with
explainable AI for trees. <em>NATMI</em>, <em>2</em>(1), 56–67. (<a
href="https://doi.org/10.1038/s42256-019-0138-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tree-based machine learning models such as random forests, decision trees and gradient boosted trees are popular nonlinear predictive models, yet comparatively little attention has been paid to explaining their predictions. Here we improve the interpretability of tree-based models through three main contributions. (1) A polynomial time algorithm to compute optimal explanations based on game theory. (2) A new type of explanation that directly measures local feature interaction effects. (3) A new set of tools for understanding global model structure based on combining many local explanations of each prediction. We apply these tools to three medical machine learning problems and show how combining many high-quality local explanations allows us to represent global structure while retaining local faithfulness to the original model. These tools enable us to (1) identify high-magnitude but low-frequency nonlinear mortality risk factors in the US population, (2) highlight distinct population subgroups with shared risk characteristics, (3) identify nonlinear interaction effects among risk factors for chronic kidney disease and (4) monitor a machine learning model deployed in a hospital by identifying which features are degrading the model’s performance over time. Given the popularity of tree-based machine learning models, these improvements to their interpretability have implications across a broad set of domains. Tree-based machine learning models are widely used in domains such as healthcare, finance and public services. The authors present an explanation method for trees that enables the computation of optimal local explanations for individual predictions, and demonstrate their method on three medical datasets.},
  archive      = {J_NATMI},
  author       = {Lundberg, Scott M. and Erion, Gabriel and Chen, Hugh and DeGrave, Alex and Prutkin, Jordan M. and Nair, Bala and Katz, Ronit and Himmelfarb, Jonathan and Bansal, Nisha and Lee, Su-In},
  doi          = {10.1038/s42256-019-0138-9},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {56-67},
  shortjournal = {Nat. Mach. Intell.},
  title        = {From local explanations to global understanding with explainable AI for trees},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Constructing energy-efficient mixed-precision neural
networks through principal component analysis for edge intelligence.
<em>NATMI</em>, <em>2</em>(1), 43–55. (<a
href="https://doi.org/10.1038/s42256-019-0134-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ‘Internet of Things’ has brought increased demand for artificial intelligence-based edge computing in applications ranging from healthcare monitoring systems to autonomous vehicles. Quantization is a powerful tool to address the growing computational cost of such applications and yields significant compression over full-precision networks. However, quantization can result in substantial loss of performance for complex image classification tasks. To address this, we propose a principal component analysis (PCA)-driven methodology to identify the important layers of a binary network, and design mixed-precision networks. The proposed Hybrid-Net achieves a more than 10\% improvement in classification accuracy over binary networks such as XNOR-Net for ResNet and VGG architectures on CIFAR-100 and ImageNet datasets, while still achieving up to 94\% of the energy efficiency of XNOR-Nets. This work advances the feasibility of using highly compressed neural networks for energy-efficient neural computing in edge devices. Neural networks are often implemented with reduced precision in order to meet the tight energy and memory budget required by edge computing devices. Chakraborty et al. develop a technique for assessing which layers can be quantized, and by how much, without sacrificing too much on performance.},
  archive      = {J_NATMI},
  author       = {Chakraborty, Indranil and Roy, Deboleena and Garg, Isha and Ankit, Aayush and Roy, Kaushik},
  doi          = {10.1038/s42256-019-0134-0},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {43-55},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Constructing energy-efficient mixed-precision neural networks through principal component analysis for edge intelligence},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Assessing the importance of magnetic resonance contrasts
using collaborative generative adversarial networks. <em>NATMI</em>,
<em>2</em>(1), 34–42. (<a
href="https://doi.org/10.1038/s42256-019-0137-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A unique advantage of magnetic resonance imaging (MRI) is its mechanism for generating various image contrasts depending on tissue-specific parameters, which provides useful clinical information. Unfortunately, a complete set of MR contrasts is often difficult to obtain in a real clinical environment. Recently, there have been claims that generative models such as generative adversarial networks (GANs) can synthesize MR contrasts that are not acquired. However, the poor scalability of existing GAN-based image synthesis poses a fundamental challenge to understanding the nature of MR contrasts: which contrasts matter, and which cannot be synthesized by generative models? Here, we show that these questions can be addressed systematically by learning the joint manifold of multiple MR contrasts using collaborative generative adversarial networks. Our experimental results show that the exogenous contrast provided by contrast agents is not replaceable, but endogenous contrasts such as T1 and T2 can be synthesized from other contrasts. These findings provide important guidance for the acquisition-protocol design of MR in clinical environments. Magnetic resonance scans use different contrast agents to generate different images, each giving specific clinical information. Lee et al. use a collaborative generative model to synthesize some magnetic resonance contrasts from others, providing guidance for how clinical imaging times can be reduced.},
  archive      = {J_NATMI},
  author       = {Lee, Dongwook and Moon, Won-Jin and Ye, Jong Chul},
  doi          = {10.1038/s42256-019-0137-x},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {34-42},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Assessing the importance of magnetic resonance contrasts using collaborative generative adversarial networks},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). AmoebaContact and GDFold as a pipeline for rapid de novo
protein structure prediction. <em>NATMI</em>, <em>2</em>(1), 25–33. (<a
href="https://doi.org/10.1038/s42256-019-0130-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the structures of proteins from amino acid sequences is of great importance. Recently, the accuracy of de novo protein structure prediction has been substantially improved when assisted by information about the contact between residues, which is also predictable from the sequence. Here, we present a novel pipeline for rapid protein structure prediction, which consists of a residue contact predictor, AmoebaContact, and a contact-assisted folder, GDFold. Unlike mainstream contact predictors that utilize simple, regularized neural networks, AmoebaContact adopts a set of network architectures that are optimized for contact prediction through automatic searching, and it predicts contacts at a series of cutoffs. Unlike conventional contact-assisted folders that only use top-scored contact pairs, GDFold considers all residue pairs from the prediction results of AmoebaContact in a differentiable loss function and optimizes atom coordinates using the gradient descent algorithm. The combination of AmoebaContact and GDFold allows quick modelling of the protein structure with acceptable model quality. Predicting the structure of proteins from amino acid sequences is a hard problem. Convolutional neural networks can learn to predict a map of distances between amino acid residues that can be turned into a three-dimensional structure. With a combination of approaches, including an evolutionary technique to find the best neural network architecture and a tool to find the atom coordinates in the folded structure, a pipeline for rapid prediction of three-dimensional protein structures is demonstrated.},
  archive      = {J_NATMI},
  author       = {Mao, Wenzhi and Ding, Wenze and Xing, Yaoguang and Gong, Haipeng},
  doi          = {10.1038/s42256-019-0130-4},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {25-33},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AmoebaContact and GDFold as a pipeline for rapid de novo protein structure prediction},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Validity of machine learning in biology and medicine
increased through collaborations across fields of expertise.
<em>NATMI</em>, <em>2</em>(1), 18–24. (<a
href="https://doi.org/10.1038/s42256-019-0139-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) has become an essential asset for the life sciences and medicine. We selected 250 articles describing ML applications from 17 journals sampling 26 different fields between 2011 and 2016. Independent evaluation by two readers highlighted three results. First, only half of the articles shared software, 64\% shared data and 81\% applied any kind of evaluation. Although crucial for ensuring the validity of ML applications, these aspects were met more by publications in lower-ranked journals. Second, the authors’ scientific backgrounds highly influenced how technical aspects were addressed: reproducibility and computational evaluation methods were more prominent with computational co-authors; experimental proofs more with experimentalists. Third, 73\% of the ML applications resulted from interdisciplinary collaborations comprising authors from at least two of the three disciplines: computational sciences, biology, and medicine. The results suggested collaborations between computational and experimental scientists to generate more scientifically sound and impactful work integrating knowledge from both domains. Although scientifically more valid solutions and collaborations involving diverse expertise did not correlate with impact factors, such collaborations provide opportunities to both sides: computational scientists are given access to novel and challenging real-world biological data, increasing the scientific impact of their research, and experimentalists benefit from more in-depth computational analyses improving the technical correctness of work. Applications of machine learning in the life sciences and medicine require expertise in computational methods and in scientific subject matter. The authors surveyed articles in the life sciences that included machine learning applications, and found that interdisciplinary collaborations increased the scientific validity of published research.},
  archive      = {J_NATMI},
  author       = {Littmann, Maria and Selig, Katharina and Cohen-Lavi, Liel and Frank, Yotam and Hönigschmid, Peter and Kataka, Evans and Mösch, Anja and Qian, Kun and Ron, Avihai and Schmid, Sebastian and Sorbie, Adam and Szlak, Liran and Dagan-Wiener, Ayana and Ben-Tal, Nir and Niv, Masha Y. and Razansky, Daniel and Schuller, Björn W. and Ankerst, Donna and Hertz, Tomer and Rost, Burkhard},
  doi          = {10.1038/s42256-019-0139-8},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {18-24},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Validity of machine learning in biology and medicine increased through collaborations across fields of expertise},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Learning with explainable trees. <em>NATMI</em>,
<em>2</em>(1), 16–17. (<a
href="https://doi.org/10.1038/s42256-019-0142-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tree-based models are among the most popular and successful machine learning algorithms in practice. New tools allow us to explain the predictions and gain insight into the global behaviour of these models.},
  archive      = {J_NATMI},
  author       = {Samek, Wojciech},
  doi          = {10.1038/s42256-019-0142-0},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {16-17},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Learning with explainable trees},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Accurate data-driven prediction does not mean high
reproducibility. <em>NATMI</em>, <em>2</em>(1), 13–15. (<a
href="https://doi.org/10.1038/s42256-019-0140-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A valid machine model is predictive, but a predictive model may not be valid. The gap between these two can be larger than many practitioners may expect.},
  archive      = {J_NATMI},
  author       = {Li, Jiuyong and Liu, Lin and Le, Thuc Duy and Liu, Jixue},
  doi          = {10.1038/s42256-019-0140-2},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {13-15},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Accurate data-driven prediction does not mean high reproducibility},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Towards ethical and socio-legal governance in AI.
<em>NATMI</em>, <em>2</em>(1), 10–12. (<a
href="https://doi.org/10.1038/s42256-019-0136-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many high-level ethics guidelines for AI have been produced in the past few years. It is time to work towards concrete policies within the context of existing moral, legal and cultural values, say Andreas Theodorou and Virginia Dignum.},
  archive      = {J_NATMI},
  author       = {Theodorou, Andreas and Dignum, Virginia},
  doi          = {10.1038/s42256-019-0136-y},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {10-12},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards ethical and socio-legal governance in AI},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). AI reflections in 2019. <em>NATMI</em>, <em>2</em>(1), 2–9.
(<a href="https://doi.org/10.1038/s42256-019-0141-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is no shortage of opinions on the impact of artificial intelligence and deep learning. We invited authors of Comment and Perspective articles that we published in roughly the first half of 2019 to look back at the year and give their thoughts on how the issue they wrote about developed.},
  archive      = {J_NATMI},
  author       = {Rich, Alexander S. and Rudin, Cynthia and Jacoby, David M. P. and Freeman, Robin and Wearn, Oliver R. and Shevlin, Henry and Dihal, Kanta and ÓhÉigeartaigh, Seán S. and Butcher, James and Lippi, Marco and Palka, Przemyslaw and Torroni, Paolo and Wongvibulsin, Shannon and Begoli, Edmon and Schneider, Gisbert and Cave, Stephen and Sloane, Mona and Moss, Emmanuel and Rahwan, Iyad and Goldberg, Ken and Howard, David and Floridi, Luciano and Stilgoe, Jack},
  doi          = {10.1038/s42256-019-0141-1},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {2-9},
  shortjournal = {Nat. Mach. Intell.},
  title        = {AI reflections in 2019},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Let’s go 2020. <em>NATMI</em>, <em>2</em>(1), 1. (<a
href="https://doi.org/10.1038/s42256-020-0144-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Welcome to the new year, new decade and second volume of Nature Machine Intelligence.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-020-0144-y},
  journal      = {Nature Machine Intelligence},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Let’s go 2020},
  volume       = {2},
  year         = {2020},
}
</textarea>
</details></li>
</ul>

</body>
</html>
