<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt---408">NBT - 408</h2>
<ul>
<li><details>
<summary>
(2023f). The world’s first CRISPR therapy is approved: Who will
receive it? <em>NBT</em>, <em>42</em>(1), 3–4. (<a
href="https://doi.org/10.1038/d41587-023-00016-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The go-ahead for Vertex’s gene editing therapy in sickle cell disease and β-thalassemia is a historic milestone, but this one-time treatment is costly. The go-ahead for Vertex’s gene editing therapy in sickle cell disease and β-thalassemia is a historic milestone, but this one-time treatment is costly.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-023-00016-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {1},
  pages        = {3-4},
  shortjournal = {Nature Biotech.},
  title        = {The world’s first CRISPR therapy is approved: Who will receive it?},
  volume       = {42},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). People. <em>NBT</em>, <em>41</em>(12), 1834. (<a
href="https://doi.org/10.1038/s41587-023-02058-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02058-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1834},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Collectively building a mentor–mentee relationship through a
one-day workshop. <em>NBT</em>, <em>41</em>(12), 1829–1833. (<a
href="https://doi.org/10.1038/s41587-023-02056-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A workshop based on collective intelligence lays the foundations of mentor–mentee interactions by providing tools to build a valuable relationship.},
  archive      = {J_NBT},
  author       = {Foncy, Julie and Kwapisz, Marta and Knibiehler, Martine and Batut, Julie},
  doi          = {10.1038/s41587-023-02056-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1829-1833},
  shortjournal = {Nature Biotech.},
  title        = {Collectively building a mentor–mentee relationship through a one-day workshop},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Contamination source modeling with SCRuB improves cancer
phenotype prediction from microbiome data. <em>NBT</em>,
<em>41</em>(12), 1820–1828. (<a
href="https://doi.org/10.1038/s41587-023-01696-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing-based approaches for the analysis of microbial communities are susceptible to contamination, which could mask biological signals or generate artifactual ones. Methods for in silico decontamination using controls are routinely used, but do not make optimal use of information shared across samples and cannot handle taxa that only partially originate in contamination or leakage of biological material into controls. Here we present Source tracking for Contamination Removal in microBiomes (SCRuB), a probabilistic in silico decontamination method that incorporates shared information across multiple samples and controls to precisely identify and remove contamination. We validate the accuracy of SCRuB in multiple data-driven simulations and experiments, including induced contamination, and demonstrate that it outperforms state-of-the-art methods by an average of 15–20 times. We showcase the robustness of SCRuB across multiple ecosystems, data types and sequencing depths. Demonstrating its applicability to microbiome research, SCRuB facilitates improved predictions of host phenotypes, most notably the prediction of treatment response in melanoma patients using decontaminated tumor microbiome data. Modeling contamination sources over multiple samples enhances the analysis of microbiome data.},
  archive      = {J_NBT},
  author       = {Austin, George I. and Park, Heekuk and Meydan, Yoli and Seeram, Dwayne and Sezin, Tanya and Lou, Yue Clare and Firek, Brian A. and Morowitz, Michael J. and Banfield, Jillian F. and Christiano, Angela M. and Pe’er, Itsik and Uhlemann, Anne-Catrin and Shenhav, Liat and Korem, Tal},
  doi          = {10.1038/s41587-023-01696-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1820-1828},
  shortjournal = {Nature Biotech.},
  title        = {Contamination source modeling with SCRuB improves cancer phenotype prediction from microbiome data},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Protein structure prediction with in-cell photo-crosslinking
mass spectrometry and deep learning. <em>NBT</em>, <em>41</em>(12),
1810–1819. (<a
href="https://doi.org/10.1038/s41587-023-01704-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While AlphaFold2 can predict accurate protein structures from the primary sequence, challenges remain for proteins that undergo conformational changes or for which few homologous sequences are known. Here we introduce AlphaLink, a modified version of the AlphaFold2 algorithm that incorporates experimental distance restraint information into its network architecture. By employing sparse experimental contacts as anchor points, AlphaLink improves on the performance of AlphaFold2 in predicting challenging targets. We confirm this experimentally by using the noncanonical amino acid photo-leucine to obtain information on residue–residue contacts inside cells by crosslinking mass spectrometry. The program can predict distinct conformations of proteins on the basis of the distance restraints provided, demonstrating the value of experimental data in driving protein structure prediction. The noise-tolerant framework for integrating data in protein structure prediction presented here opens a path to accurate characterization of protein structures from in-cell data. Limitations of Alphafold2 structure prediction are addressed by including experimentally determined distance constraints.},
  archive      = {J_NBT},
  author       = {Stahl, Kolja and Graziadei, Andrea and Dau, Therese and Brock, Oliver and Rappsilber, Juri},
  doi          = {10.1038/s41587-023-01704-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1810-1819},
  shortjournal = {Nature Biotech.},
  title        = {Protein structure prediction with in-cell photo-crosslinking mass spectrometry and deep learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Quantification of absolute transcription factor binding
affinities in the native chromatin context using BANC-seq. <em>NBT</em>,
<em>41</em>(12), 1801–1809. (<a
href="https://doi.org/10.1038/s41587-023-01715-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factor binding across the genome is regulated by DNA sequence and chromatin features. However, it is not yet possible to quantify the impact of chromatin context on transcription factor binding affinities. Here, we report a method called binding affinities to native chromatin by sequencing (BANC-seq) to determine absolute apparent binding affinities of transcription factors to native DNA across the genome. In BANC-seq, a concentration range of a tagged transcription factor is added to isolated nuclei. Concentration-dependent binding is then measured per sample to quantify apparent binding affinities across the genome. BANC-seq adds a quantitative dimension to transcription factor biology, which enables stratification of genomic targets based on transcription factor concentration and prediction of transcription factor binding sites under non-physiological conditions, such as disease-associated overexpression of (onco)genes. Notably, whereas consensus DNA binding motifs for transcription factors are important to establish high-affinity binding sites, these motifs are not always strictly required to generate nanomolar-affinity interactions in the genome. The impact of chromatin context on transcription factor binding affinities is quantified.},
  archive      = {J_NBT},
  author       = {Neikes, Hannah K. and Kliza, Katarzyna W. and Gräwe, Cathrin and Wester, Roelof A. and Jansen, Pascal W. T. C. and Lamers, Lieke A. and Baltissen, Marijke P. and van Heeringen, Simon J. and Logie, Colin and Teichmann, Sarah A. and Lindeboom, Rik G. H. and Vermeulen, Michiel},
  doi          = {10.1038/s41587-023-01715-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1801-1809},
  shortjournal = {Nature Biotech.},
  title        = {Quantification of absolute transcription factor binding affinities in the native chromatin context using BANC-seq},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Epicardioid single-cell genomics uncovers principles of
human epicardium biology in heart development and disease. <em>NBT</em>,
<em>41</em>(12), 1787–1800. (<a
href="https://doi.org/10.1038/s41587-023-01718-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The epicardium, the mesothelial envelope of the vertebrate heart, is the source of multiple cardiac cell lineages during embryonic development and provides signals that are essential to myocardial growth and repair. Here we generate self-organizing human pluripotent stem cell-derived epicardioids that display retinoic acid-dependent morphological, molecular and functional patterning of the epicardium and myocardium typical of the left ventricular wall. By combining lineage tracing, single-cell transcriptomics and chromatin accessibility profiling, we describe the specification and differentiation process of different cell lineages in epicardioids and draw comparisons to human fetal development at the transcriptional and morphological levels. We then use epicardioids to investigate the functional cross-talk between cardiac cell types, gaining new insights into the role of IGF2/IGF1R and NRP2 signaling in human cardiogenesis. Finally, we show that epicardioids mimic the multicellular pathogenesis of congenital or stress-induced hypertrophy and fibrotic remodeling. As such, epicardioids offer a unique testing ground of epicardial activity in heart development, disease and regeneration. The lineage of human epicardium is studied using time course single-cell analysis of epicardioids.},
  archive      = {J_NBT},
  author       = {Meier, Anna B. and Zawada, Dorota and De Angelis, Maria Teresa and Martens, Laura D. and Santamaria, Gianluca and Zengerle, Sophie and Nowak-Imialek, Monika and Kornherr, Jessica and Zhang, Fangfang and Tian, Qinghai and Wolf, Cordula M. and Kupatt, Christian and Sahara, Makoto and Lipp, Peter and Theis, Fabian J. and Gagneur, Julien and Goedel, Alexander and Laugwitz, Karl-Ludwig and Dorn, Tatjana and Moretti, Alessandra},
  doi          = {10.1038/s41587-023-01718-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1787-1800},
  shortjournal = {Nature Biotech.},
  title        = {Epicardioid single-cell genomics uncovers principles of human epicardium biology in heart development and disease},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Global detection of human variants and isoforms by deep
proteome sequencing. <em>NBT</em>, <em>41</em>(12), 1776–1786. (<a
href="https://doi.org/10.1038/s41587-023-01714-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An average shotgun proteomics experiment detects approximately 10,000 human proteins from a single sample. However, individual proteins are typically identified by peptide sequences representing a small fraction of their total amino acids. Hence, an average shotgun experiment fails to distinguish different protein variants and isoforms. Deeper proteome sequencing is therefore required for the global discovery of protein isoforms. Using six different human cell lines, six proteases, deep fractionation and three tandem mass spectrometry fragmentation methods, we identify a million unique peptides from 17,717 protein groups, with a median sequence coverage of approximately 80\%. Direct comparison with RNA expression data provides evidence for the translation of most nonsynonymous variants. We have also hypothesized that undetected variants likely arise from mutation-induced protein instability. We further observe comparable detection rates for exon–exon junction peptides representing constitutive and alternative splicing events. Our dataset represents a resource for proteoform discovery and provides direct evidence that most frame-preserving alternatively spliced isoforms are translated. Deep proteome sequencing achieves ~80\% coverage of the human proteome.},
  archive      = {J_NBT},
  author       = {Sinitcyn, Pavel and Richards, Alicia L. and Weatheritt, Robert J. and Brademan, Dain R. and Marx, Harald and Shishkova, Evgenia and Meyer, Jesse G. and Hebert, Alexander S. and Westphall, Michael S. and Blencowe, Benjamin J. and Cox, Jürgen and Coon, Joshua J.},
  doi          = {10.1038/s41587-023-01714-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1776-1786},
  shortjournal = {Nature Biotech.},
  title        = {Global detection of human variants and isoforms by deep proteome sequencing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multimodal spatiotemporal phenotyping of human retinal
organoid development. <em>NBT</em>, <em>41</em>(12), 1765–1775. (<a
href="https://doi.org/10.1038/s41587-023-01747-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoids generated from human pluripotent stem cells provide experimental systems to study development and disease, but quantitative measurements across different spatial scales and molecular modalities are lacking. In this study, we generated multiplexed protein maps over a retinal organoid time course and primary adult human retinal tissue. We developed a toolkit to visualize progenitor and neuron location, the spatial arrangements of extracellular and subcellular components and global patterning in each organoid and primary tissue. In addition, we generated a single-cell transcriptome and chromatin accessibility timecourse dataset and inferred a gene regulatory network underlying organoid development. We integrated genomic data with spatially segmented nuclei into a multimodal atlas to explore organoid patterning and retinal ganglion cell (RGC) spatial neighborhoods, highlighting pathways involved in RGC cell death and showing that mosaic genetic perturbations in retinal organoids provide insight into cell fate regulation. A multimodal atlas of retinal organoid development shows spatial interactions over time.},
  archive      = {J_NBT},
  author       = {Wahle, Philipp and Brancati, Giovanna and Harmel, Christoph and He, Zhisong and Gut, Gabriele and del Castillo, Jacobo Sarabia and Xavier da Silveira dos Santos, Aline and Yu, Qianhui and Noser, Pascal and Fleck, Jonas Simon and Gjeta, Bruno and Pavlinić, Dinko and Picelli, Simone and Hess, Max and Schmidt, Gregor W. and Lummen, Tom T. A. and Hou, Yanyan and Galliker, Patricia and Goldblum, David and Balogh, Marton and Cowan, Cameron S. and Scholl, Hendrik P. N. and Roska, Botond and Renner, Magdalena and Pelkmans, Lucas and Treutlein, Barbara and Camp, J. Gray},
  doi          = {10.1038/s41587-023-01747-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1765-1775},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal spatiotemporal phenotyping of human retinal organoid development},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tuning plant phenotypes by precise, graded downregulation of
gene expression. <em>NBT</em>, <em>41</em>(12), 1758–1764. (<a
href="https://doi.org/10.1038/s41587-023-01707-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to control gene expression and generate quantitative phenotypic changes is essential for breeding new and desired traits into crops. Here we report an efficient, facile method for downregulating gene expression to predictable, desired levels by engineering upstream open reading frames (uORFs). We used base editing or prime editing to generate de novo uORFs or to extend existing uORFs by mutating their stop codons. By combining these approaches, we generated a suite of uORFs that incrementally downregulate the translation of primary open reading frames (pORFs) to 2.5–84.9\% of the wild-type level. By editing the 5′ untranslated region of OsDLT, which encodes a member of the GRAS family and is involved in the brassinosteroid transduction pathway, we obtained, as predicted, a series of rice plants with varied plant heights and tiller numbers. These methods offer an efficient way to obtain genome-edited plants with graded expression of traits. Plant traits are fine-tuned by stepwise downregulation of gene expression.},
  archive      = {J_NBT},
  author       = {Xue, Chenxiao and Qiu, Fengti and Wang, Yuxiang and Li, Boshu and Zhao, Kevin Tianmeng and Chen, Kunling and Gao, Caixia},
  doi          = {10.1038/s41587-023-01707-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1758-1764},
  shortjournal = {Nature Biotech.},
  title        = {Tuning plant phenotypes by precise, graded downregulation of gene expression},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SEACells infers transcriptional and epigenomic cellular
states from single-cell genomics data. <em>NBT</em>, <em>41</em>(12),
1746–1757. (<a
href="https://doi.org/10.1038/s41587-023-01716-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metacells are cell groupings derived from single-cell sequencing data that represent highly granular, distinct cell states. Here we present single-cell aggregation of cell states (SEACells), an algorithm for identifying metacells that overcome the sparsity of single-cell data while retaining heterogeneity obscured by traditional cell clustering. SEACells outperforms existing algorithms in identifying comprehensive, compact and well-separated metacells in both RNA and assay for transposase-accessible chromatin (ATAC) modalities across datasets with discrete cell types and continuous trajectories. We demonstrate the use of SEACells to improve gene–peak associations, compute ATAC gene scores and infer the activities of critical regulators during differentiation. Metacell-level analysis scales to large datasets and is particularly well suited for patient cohorts, where per-patient aggregation provides more robust units for data integration. We use our metacells to reveal expression dynamics and gradual reconfiguration of the chromatin landscape during hematopoietic differentiation and to uniquely identify CD4 T cell differentiation and activation states associated with disease onset and severity in a Coronavirus Disease 2019 (COVID-19) patient cohort. Single-cell aggregation with archetypes provides intermediate granularity for downstream analyses.},
  archive      = {J_NBT},
  author       = {Persad, Sitara and Choo, Zi-Ning and Dien, Christine and Sohail, Noor and Masilionis, Ignas and Chaligné, Ronan and Nawy, Tal and Brown, Chrysothemis C. and Sharma, Roshan and Pe’er, Itsik and Setty, Manu and Pe’er, Dana},
  doi          = {10.1038/s41587-023-01716-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1746-1757},
  shortjournal = {Nature Biotech.},
  title        = {SEACells infers transcriptional and epigenomic cellular states from single-cell genomics data},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetically encoded barcodes for correlative volume electron
microscopy. <em>NBT</em>, <em>41</em>(12), 1734–1745. (<a
href="https://doi.org/10.1038/s41587-023-01713-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While genetically encoded reporters are common for fluorescence microscopy, equivalent multiplexable gene reporters for electron microscopy (EM) are still scarce. Here, by installing a variable number of fixation-stable metal-interacting moieties in the lumen of encapsulin nanocompartments of different sizes, we developed a suite of spherically symmetric and concentric barcodes (EMcapsulins) that are readable by standard EM techniques. Six classes of EMcapsulins could be automatically segmented and differentiated. The coding capacity was further increased by arranging several EMcapsulins into distinct patterns via a set of rigid spacers of variable length. Fluorescent EMcapsulins were expressed to monitor subcellular structures in light and EM. Neuronal expression in Drosophila and mouse brains enabled the automatic identification of genetically defined cells in EM. EMcapsulins are compatible with transmission EM, scanning EM and focused ion beam scanning EM. The expandable palette of genetically controlled EM-readable barcodes can augment anatomical EM images with multiplexed gene expression maps. Multiplexable barcodes for electron microscopy are applied to brain imaging.},
  archive      = {J_NBT},
  author       = {Sigmund, Felix and Berezin, Oleksandr and Beliakova, Sofia and Magerl, Bernhard and Drawitsch, Martin and Piovesan, Alberto and Gonçalves, Filipa and Bodea, Silviu-Vasile and Winkler, Stefanie and Bousraou, Zoe and Grosshauser, Martin and Samara, Eleni and Pujol-Martí, Jesús and Schädler, Sebastian and So, Chun and Irsen, Stephan and Walch, Axel and Kofler, Florian and Piraud, Marie and Kornfeld, Joergen and Briggman, Kevin and Westmeyer, Gil Gregor},
  doi          = {10.1038/s41587-023-01713-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1734-1745},
  shortjournal = {Nature Biotech.},
  title        = {Genetically encoded barcodes for correlative volume electron microscopy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Trans-segmental imaging in the spinal cord of behaving mice.
<em>NBT</em>, <em>41</em>(12), 1729–1733. (<a
href="https://doi.org/10.1038/s41587-023-01700-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spinal cord circuits play crucial roles in transmitting pain, but the underlying activity patterns within and across spinal segments in behaving mice have remained elusive. We developed a wearable widefield macroscope with a 7.9-mm2 field of view, ~3- to 4-μm lateral resolution, 2.7-mm working distance and &amp;lt;10-g overall weight and show that highly localized painful mechanical stimuli evoke widespread, coordinated astrocyte excitation across multiple spinal segments. A wearable macroscope enables cellular imaging across multiple spinal segments in behaving mice.},
  archive      = {J_NBT},
  author       = {Shekhtmeyster, Pavel and Duarte, Daniela and Carey, Erin M. and Ngo, Alexander and Gao, Grace and Olmstead, Jack A. and Nelson, Nicholas A. and Nimmerjahn, Axel},
  doi          = {10.1038/s41587-023-01700-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1729-1733},
  shortjournal = {Nature Biotech.},
  title        = {Trans-segmental imaging in the spinal cord of behaving mice},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Soil microbiome engineering for sustainability in a changing
environment. <em>NBT</em>, <em>41</em>(12), 1716–1728. (<a
href="https://doi.org/10.1038/s41587-023-01932-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in microbial ecology and synthetic biology have the potential to mitigate damage caused by anthropogenic activities that are deleteriously impacting Earth’s soil ecosystems. Here, we discuss challenges and opportunities for harnessing natural and synthetic soil microbial communities, focusing on plant growth promotion under different scenarios. We explore current needs for microbial solutions in soil ecosystems, how these solutions are being developed and applied, and the potential for new biotechnology breakthroughs to tailor and target microbial products for specific applications. We highlight several scientific and technological advances in soil microbiome engineering, including characterization of microbes that impact soil ecosystems, directing how microbes assemble to interact in soil environments, and the developing suite of gene-engineering approaches. This Review underscores the need for an interdisciplinary approach to understand the composition, dynamics and deployment of beneficial soil microbiomes to drive efforts to mitigate or reverse environmental damage by restoring and protecting healthy soil ecosystems. Challenges and opportunities for engineering and studying the soil microbiome are discussed.},
  archive      = {J_NBT},
  author       = {Jansson, Janet K. and McClure, Ryan and Egbert, Robert G.},
  doi          = {10.1038/s41587-023-01932-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1716-1728},
  shortjournal = {Nature Biotech.},
  title        = {Soil microbiome engineering for sustainability in a changing environment},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The chemistry of next-generation sequencing. <em>NBT</em>,
<em>41</em>(12), 1709–1715. (<a
href="https://doi.org/10.1038/s41587-023-01986-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first large genome fully sequenced by next-generation sequencing (NGS) was that of a bacteriophage using sequencing by synthesis (SBS) as a paradigm. SBS in NGS is underpinned by ‘reversible-terminator chemistry’. To grow from proof of concept to being both affordable and practical, SBS needed to overcome a series of challenges, each of which required the invention of new chemistries. These included the design and synthesis of unnatural deoxynucleotide triphosphates (dNTPs), engineering a suitable polymerase, a new surface chemistry and an ingenious molecular solution to neutralize copying errors inherent to all polymerases. In this historical Perspective, we discuss how NGS was developed from Sanger sequencing, highlighting the chemistry behind this technology, which has impacted biology in unprecedented ways. A historical perspective on how next-generation sequencing chemistry was developed.},
  archive      = {J_NBT},
  author       = {Rodriguez, Raphaël and Krishnan, Yamuna},
  doi          = {10.1038/s41587-023-01986-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1709-1715},
  shortjournal = {Nature Biotech.},
  title        = {The chemistry of next-generation sequencing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Expanding access to CAR t cell therapies through local
manufacturing. <em>NBT</em>, <em>41</em>(12), 1698–1708. (<a
href="https://doi.org/10.1038/s41587-023-01981-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor (CAR) T cells are changing the therapeutic landscape for hematological malignancies. To date, all six CAR T cell products approved by the US Food and Drug Administration (FDA) are autologous and centrally manufactured. As the numbers of approved products and indications continue to grow, new strategies to increase cell-manufacturing capacity are urgently needed to ensure patient access. Distributed manufacturing at the point of care or at other local manufacturing sites would go a long way toward meeting the rising demand. To ensure successful implementation, it is imperative to harness novel technologies to achieve uniform product quality across geographically dispersed facilities. This includes the use of automated cell-production systems, in-line sensors and process simulation for enhanced quality control and efficient supply chain management. A comprehensive effort to understand the critical quality attributes of CAR T cells would enable better definition of widely attainable release criteria. To supplement oversight by national regulatory agencies, we recommend expansion of the role of accreditation bodies. Moreover, regulatory standards may need to be amended to accommodate the unique characteristics of distributed manufacturing models. Shortages of CAR T cells should be alleviated by distributed manufacturing, according to Elsallab and Maus.},
  archive      = {J_NBT},
  author       = {Elsallab, Magdi and Maus, Marcela V.},
  doi          = {10.1038/s41587-023-01981-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1698-1708},
  shortjournal = {Nature Biotech.},
  title        = {Expanding access to CAR t cell therapies through local manufacturing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Measuring the impact of chromatin context on transcription
factor binding affinities. <em>NBT</em>, <em>41</em>(12), 1696–1697. (<a
href="https://doi.org/10.1038/s41587-023-01752-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have designed a method, binding affinities to native chromatin by sequencing (BANC-seq), to determine the transcription factor concentrations required for binding to regulatory elements across the genome. Our study shows that chromatin context and DNA accessibility are key regulators of transcription factor binding.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01752-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1696-1697},
  shortjournal = {Nature Biotech.},
  title        = {Measuring the impact of chromatin context on transcription factor binding affinities},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineered upstream open reading frames predictably
downregulate mRNA translation in plants. <em>NBT</em>, <em>41</em>(12),
1694–1695. (<a
href="https://doi.org/10.1038/s41587-023-01729-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fine-tuning gene expression is crucial for generating quantitative phenotypic changes and crop improvement. Using CRISPR–Cas base editing and prime editing, we engineered upstream open reading frames — eukaryotic translational control elements — in rice to incrementally downregulate translation to predictable and desired levels.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01729-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1694-1695},
  shortjournal = {Nature Biotech.},
  title        = {Engineered upstream open reading frames predictably downregulate mRNA translation in plants},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Augmenting electron microscopy with barcoded gene reporters.
<em>NBT</em>, <em>41</em>(12), 1692–1693. (<a
href="https://doi.org/10.1038/s41587-023-01731-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded concentric barcodes function as multiplexed electron microscopy (EM) gene reporters (EMcapsulins) for cell culture and in vivo models. These barcodes can be used to visualize gene expression information as automatically labeled overlays on volume EM data.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01731-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1692-1693},
  shortjournal = {Nature Biotech.},
  title        = {Augmenting electron microscopy with barcoded gene reporters},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Subtle cell states resolved in single-cell data.
<em>NBT</em>, <em>41</em>(12), 1690–1691. (<a
href="https://doi.org/10.1038/s41587-023-01797-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SEACells identifies rare cell states in large datasets, enabling atlas-scale studies.},
  archive      = {J_NBT},
  author       = {Lareau, Caleb},
  doi          = {10.1038/s41587-023-01797-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1690-1691},
  shortjournal = {Nature Biotech.},
  title        = {Subtle cell states resolved in single-cell data},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A neuroprosthesis for parkinson’s disease. <em>NBT</em>,
<em>41</em>(12), 1689. (<a
href="https://doi.org/10.1038/s41587-023-02067-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bishop, Anahita},
  doi          = {10.1038/s41587-023-02067-1},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1689},
  shortjournal = {Nature Biotech.},
  title        = {A neuroprosthesis for parkinson’s disease},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Wireless multisite physiological sensor with clinical-grade
accuracy. <em>NBT</em>, <em>41</em>(12), 1689. (<a
href="https://doi.org/10.1038/s41587-023-02068-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Li, Rong},
  doi          = {10.1038/s41587-023-02068-0},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1689},
  shortjournal = {Nature Biotech.},
  title        = {Wireless multisite physiological sensor with clinical-grade accuracy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Yeast with a half-synthetic genome. <em>NBT</em>,
<em>41</em>(12), 1689. (<a
href="https://doi.org/10.1038/s41587-023-02069-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Doerr, Anne},
  doi          = {10.1038/s41587-023-02069-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1689},
  shortjournal = {Nature Biotech.},
  title        = {Yeast with a half-synthetic genome},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Metagenomics. <em>NBT</em>, <em>41</em>(12), 1688. (<a
href="https://doi.org/10.1038/s41587-023-02052-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods of characterizing metagenomic samples and analyzing and processing metagenomic data.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02052-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1688},
  shortjournal = {Nature Biotech.},
  title        = {Metagenomics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The global patent landscape of artificial intelligence
applications for cancer. <em>NBT</em>, <em>41</em>(12), 1679–1687. (<a
href="https://doi.org/10.1038/s41587-023-02051-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A review of the emerging patent landscape can assist key actors in making informed strategic decisions about AI applications in oncology.},
  archive      = {J_NBT},
  author       = {Braga, Luiza and Lopes, Renato and Alves, Luiz and Mota, Fabio},
  doi          = {10.1038/s41587-023-02051-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1679-1687},
  shortjournal = {Nature Biotech.},
  title        = {The global patent landscape of artificial intelligence applications for cancer},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Nature biotechnology’s academic spinouts 2022. <em>NBT</em>,
<em>41</em>(12), 1669–1678. (<a
href="https://doi.org/10.1038/s41587-023-02023-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology’s annual survey highlights academic start ups that are, among other things, correcting misfolded or disordered proteins, creating second-generation GPCR agonists, building a new gene delivery platform and mining cancer genomes for novel targets.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael and Schmidt, Charles and DeFrancesco, Laura},
  doi          = {10.1038/s41587-023-02023-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1669-1678},
  shortjournal = {Nature Biotech.},
  title        = {Nature biotechnology’s academic spinouts 2022},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(12), 1668. (<a
href="https://doi.org/10.1038/s41587-023-02066-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02066-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1668},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chikungunya vaccine approved. <em>NBT</em>, <em>41</em>(12),
1667. (<a href="https://doi.org/10.1038/s41587-023-02071-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02071-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1667},
  shortjournal = {Nature Biotech.},
  title        = {Chikungunya vaccine approved},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Bioengineered spider silk without critters. <em>NBT</em>,
<em>41</em>(12), 1665–1667. (<a
href="https://doi.org/10.1038/s41587-023-02064-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New tools are enabling companies to develop genetically engineered spider silk fibers that are as tough as steel, and as elastic as rubber.},
  archive      = {J_NBT},
  author       = {Hughes-Castleberry, Kenna},
  doi          = {10.1038/s41587-023-02064-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1665-1667},
  shortjournal = {Nature Biotech.},
  title        = {Bioengineered spider silk without critters},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). First in vivo base editing lowers cholesterol. <em>NBT</em>,
<em>41</em>(12), 1665. (<a
href="https://doi.org/10.1038/s41587-023-02070-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02070-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1665},
  shortjournal = {Nature Biotech.},
  title        = {First in vivo base editing lowers cholesterol},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). From AI to the y chromosome (and everything in between).
<em>NBT</em>, <em>41</em>(12), 1661–1662. (<a
href="https://doi.org/10.1038/s41587-023-02076-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology editors pick their favorite research articles from 2023.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02076-0},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1661-1662},
  shortjournal = {Nature Biotech.},
  title        = {From AI to the y chromosome (and everything in between)},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Top ten news stories in 2023. <em>NBT</em>,
<em>41</em>(12), 1663–1664. (<a
href="https://doi.org/10.1038/s41587-023-02063-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our ranking is out, and it has a few surprises. In pole position — predictably — is Wegovy, the weight-slashing GLP-1 agonist, with AI-hallucinated proteins a close second. These are followed by drugging options for RNA. The dark horses are virus-free gene therapies and a sweet protein.},
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-023-02063-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1663-1664},
  shortjournal = {Nature Biotech.},
  title        = {Top ten news stories in 2023},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). People. <em>NBT</em>, <em>41</em>(11), 1660. (<a
href="https://doi.org/10.1038/s41587-023-02017-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02017-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1660},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Five questions with isidro cortés-ciriano. <em>NBT</em>,
<em>41</em>(11), 1659. (<a
href="https://doi.org/10.1038/s41587-023-02012-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After growing up collecting fossils and reading science books, this group leader’s research now lies at the interface of biology, artificial intelligence and genomics.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-023-02012-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1659},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with isidro cortés-ciriano},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ChatGPT and medicine: How AI language models are shaping the
future and health related careers. <em>NBT</em>, <em>41</em>(11),
1657–1658. (<a
href="https://doi.org/10.1038/s41587-023-02011-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {ChatGPT holds transformative potential for medicine across clinical, research and educational domains.},
  archive      = {J_NBT},
  author       = {Yan, Michael and Cerri, Giovanni G. and Moraes, Fabio Y.},
  doi          = {10.1038/s41587-023-02011-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1657-1658},
  shortjournal = {Nature Biotech.},
  title        = {ChatGPT and medicine: How AI language models are shaping the future and health related careers},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Enhancing untargeted metabolomics using
metadata-based source annotation. <em>NBT</em>, <em>41</em>(11), 1656.
(<a href="https://doi.org/10.1038/s41587-023-02025-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Gauglitz, Julia M. and West, Kiana A. and Bittremieux, Wout and Williams, Candace L. and Weldon, Kelly C. and Panitchpakdi, Morgan and Di Ottavio, Francesca and Aceves, Christine M. and Brown, Elizabeth and Sikora, Nicole C. and Jarmusch, Alan K. and Martino, Cameron and Tripathi, Anupriya and Meehan, Michael J. and Dorrestein, Kathleen and Shaffer, Justin P. and Coras, Roxana and Vargas, Fernando and Goldasich, Lindsay DeRight and Schwartz, Tara and Bryant, MacKenzie and Humphrey, Gregory and Johnson, Abigail J. and Spengler, Katharina and Belda-Ferre, Pedro and Diaz, Edgar and McDonald, Daniel and Zhu, Qiyun and Elijah, Emmanuel O. and Wang, Mingxun and Marotz, Clarisse and Sprecher, Kate E. and Vargas-Robles, Daniela and Withrow, Dana and Ackermann, Gail and Herrera, Lourdes and Bradford, Barry J. and Marques, Lucas Maciel Mauriz and Amaral, Juliano Geraldo and Silva, Rodrigo Moreira and Veras, Flavio Protasio and Cunha, Thiago Mattar and Oliveira, Rene Donizeti Ribeiro and Louzada-Junior, Paulo and Mills, Robert H. and Piotrowski, Paulina K. and Servetas, Stephanie L. and Da Silva, Sandra M. and Jones, Christina M. and Lin, Nancy J. and Lippa, Katrice A. and Jackson, Scott A. and Daouk, Rima Kaddurah and Galasko, Douglas and Dulai, Parambir S. and Kalashnikova, Tatyana I. and Wittenberg, Curt and Terkeltaub, Robert and Doty, Megan M. and Kim, Jae H. and Rhee, Kyung E. and Beauchamp-Walters, Julia and Wright, Kenneth P. and Dominguez-Bello, Maria Gloria and Manary, Mark and Oliveira, Michelli F. and Boland, Brigid S. and Lopes, Norberto Peporine and Guma, Monica and Swafford, Austin D. and Dutton, Rachel J. and Knight, Rob and Dorrestein, Pieter C.},
  doi          = {10.1038/s41587-023-02025-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1656},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Enhancing untargeted metabolomics using metadata-based source annotation},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Deconvolution of clinical variance in
CAR-t cell pharmacology and response. <em>NBT</em>, <em>41</em>(11),
1655. (<a href="https://doi.org/10.1038/s41587-023-01816-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kirouac, Daniel C. and Zmurchok, Cole and Deyati, Avisek and Sicherman, Jordan and Bond, Chris and Zandstra, Peter W.},
  doi          = {10.1038/s41587-023-01816-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1655},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Deconvolution of clinical variance in CAR-T cell pharmacology and response},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Efficient c•g-to-g•c base editors
developed using CRISPRi screens, target-library analysis, and machine
learning. <em>NBT</em>, <em>41</em>(11), 1655. (<a
href="https://doi.org/10.1038/s41587-023-02028-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Koblan, Luke W. and Arbab, Mandana and Shen, Max W. and Hussmann, Jeffrey A. and Anzalone, Andrew V. and Doman, Jordan L. and Newby, Gregory A. and Yang, Dian and Mok, Beverly and Replogle, Joseph M. and Xu, Albert and Sisley, Tyler A. and Weissman, Jonathan S. and Adamson, Britt and Liu, David R.},
  doi          = {10.1038/s41587-023-02028-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1655},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Efficient C•G-to-G•C base editors developed using CRISPRi screens, target-library analysis, and machine learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ScPrisma infers, filters and enhances topological signals in
single-cell data using spectral template matching. <em>NBT</em>,
<em>41</em>(11), 1645–1654. (<a
href="https://doi.org/10.1038/s41587-023-01663-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing has been instrumental in uncovering cellular spatiotemporal context. This task is challenging as cells simultaneously encode multiple, potentially cross-interfering, biological signals. Here we propose scPrisma, a spectral computational method that uses topological priors to decouple, enhance and filter different classes of biological processes in single-cell data, such as periodic and linear signals. We apply scPrisma to the analysis of the cell cycle in HeLa cells, circadian rhythm and spatial zonation in liver lobules, diurnal cycle in Chlamydomonas and circadian rhythm in the suprachiasmatic nucleus in the brain. scPrisma can be used to distinguish mixed cellular populations by specific characteristics such as cell type and uncover regulatory networks and cell–cell interactions specific to predefined biological signals, such as the circadian rhythm. We show scPrisma’s flexibility in incorporating prior knowledge, inference of topologically informative genes and generalization to additional diverse templates and systems. scPrisma can be used as a stand-alone workflow for signal analysis and as a prior step for downstream single-cell analysis. scPrisma deconvolutes different biological processes in single-cell RNA-sequencing data.},
  archive      = {J_NBT},
  author       = {Karin, Jonathan and Bornfeld, Yonathan and Nitzan, Mor},
  doi          = {10.1038/s41587-023-01663-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1645-1654},
  shortjournal = {Nature Biotech.},
  title        = {ScPrisma infers, filters and enhances topological signals in single-cell data using spectral template matching},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Extending and improving metagenomic taxonomic profiling with
uncharacterized species using MetaPhlAn 4. <em>NBT</em>,
<em>41</em>(11), 1633–1644. (<a
href="https://doi.org/10.1038/s41587-023-01688-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metagenomic assembly enables new organism discovery from microbial communities, but it can only capture few abundant organisms from most metagenomes. Here we present MetaPhlAn 4, which integrates information from metagenome assemblies and microbial isolate genomes for more comprehensive metagenomic taxonomic profiling. From a curated collection of 1.01 M prokaryotic reference and metagenome-assembled genomes, we define unique marker genes for 26,970 species-level genome bins, 4,992 of them taxonomically unidentified at the species level. MetaPhlAn 4 explains ~20\% more reads in most international human gut microbiomes and &amp;gt;40\% in less-characterized environments such as the rumen microbiome and proves more accurate than available alternatives on synthetic evaluations while also reliably quantifying organisms with no cultured isolates. Application of the method to &amp;gt;24,500 metagenomes highlights previously undetected species to be strong biomarkers for host conditions and lifestyles in human and mouse microbiomes and shows that even previously uncharacterized species can be genetically profiled at the resolution of single microbial strains. Integration of metagenomic assemblies and microbial isolate genomes improves profiling of uncharacterized species.},
  archive      = {J_NBT},
  author       = {Blanco-Míguez, Aitor and Beghini, Francesco and Cumbo, Fabio and McIver, Lauren J. and Thompson, Kelsey N. and Zolfo, Moreno and Manghi, Paolo and Dubois, Leonard and Huang, Kun D. and Thomas, Andrew Maltez and Nickols, William A. and Piccinno, Gianmarco and Piperni, Elisa and Punčochář, Michal and Valles-Colomer, Mireia and Tett, Adrian and Giordano, Francesca and Davies, Richard and Wolf, Jonathan and Berry, Sarah E. and Spector, Tim D. and Franzosa, Eric A. and Pasolli, Edoardo and Asnicar, Francesco and Huttenhower, Curtis and Segata, Nicola},
  doi          = {10.1038/s41587-023-01688-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1633-1644},
  shortjournal = {Nature Biotech.},
  title        = {Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell transcriptomic atlas-guided development of CAR-t
cells for the treatment of acute myeloid leukemia. <em>NBT</em>,
<em>41</em>(11), 1618–1632. (<a
href="https://doi.org/10.1038/s41587-023-01684-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor T cells (CAR-T cells) have emerged as a powerful treatment option for individuals with B cell malignancies but have yet to achieve success in treating acute myeloid leukemia (AML) due to a lack of safe targets. Here we leveraged an atlas of publicly available RNA-sequencing data of over 500,000 single cells from 15 individuals with AML and tissue from 9 healthy individuals for prediction of target antigens that are expressed on malignant cells but lacking on healthy cells, including T cells. Aided by this high-resolution, single-cell expression approach, we computationally identify colony-stimulating factor 1 receptor and cluster of differentiation 86 as targets for CAR-T cell therapy in AML. Functional validation of these established CAR-T cells shows robust in vitro and in vivo efficacy in cell line- and human-derived AML models with minimal off-target toxicity toward relevant healthy human tissues. This provides a strong rationale for further clinical development. A single-cell screening approach identifies targets for CAR-T cells in acute myeloid leukemia.},
  archive      = {J_NBT},
  author       = {Gottschlich, Adrian and Thomas, Moritz and Grünmeier, Ruth and Lesch, Stefanie and Rohrbacher, Lisa and Igl, Veronika and Briukhovetska, Daria and Benmebarek, Mohamed-Reda and Vick, Binje and Dede, Sertac and Müller, Katharina and Xu, Tao and Dhoqina, Dario and Märkl, Florian and Robinson, Sophie and Sendelhofert, Andrea and Schulz, Heiko and Umut, Öykü and Kavaka, Vladyslav and Tsiverioti, Christina Angeliki and Carlini, Emanuele and Nandi, Sayantan and Strzalkowski, Thaddäus and Lorenzini, Theo and Stock, Sophia and Müller, Philipp Jie and Dörr, Janina and Seifert, Matthias and Cadilha, Bruno L. and Brabenec, Ruben and Röder, Natalie and Rataj, Felicitas and Nüesch, Manuel and Modemann, Franziska and Wellbrock, Jasmin and Fiedler, Walter and Kellner, Christian and Beltrán, Eduardo and Herold, Tobias and Paquet, Dominik and Jeremias, Irmela and von Baumgarten, Louisa and Endres, Stefan and Subklewe, Marion and Marr, Carsten and Kobold, Sebastian},
  doi          = {10.1038/s41587-023-01684-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1618-1632},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell transcriptomic atlas-guided development of CAR-T cells for the treatment of acute myeloid leukemia},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Deconvolution of clinical variance in CAR-t cell
pharmacology and response. <em>NBT</em>, <em>41</em>(11), 1606–1617. (<a
href="https://doi.org/10.1038/s41587-023-01687-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor T cell (CAR-T) expansion and persistence vary widely among patients and predict both efficacy and toxicity. However, the mechanisms underlying clinical outcomes and patient variability are poorly defined. In this study, we developed a mathematical description of T cell responses wherein transitions among memory, effector and exhausted T cell states are coordinately regulated by tumor antigen engagement. The model is trained using clinical data from CAR-T products in different hematological malignancies and identifies cell-intrinsic differences in the turnover rate of memory cells and cytotoxic potency of effectors as the primary determinants of clinical response. Using a machine learning workflow, we demonstrate that product-intrinsic differences can accurately predict patient outcomes based on pre-infusion transcriptomes, and additional pharmacological variance arises from cellular interactions with patient tumors. We found that transcriptional signatures outperform T cell immunophenotyping as predictive of clinical response for two CD19-targeted CAR-T products in three indications, enabling a new phase of predictive CAR-T product development. A model of T cell differentiation control deconvolutes biological mechanisms underlying clinical variance.},
  archive      = {J_NBT},
  author       = {Kirouac, Daniel C. and Zmurchok, Cole and Deyati, Avisek and Sicherman, Jordan and Bond, Chris and Zandstra, Peter W.},
  doi          = {10.1038/s41587-023-01687-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1606-1617},
  shortjournal = {Nature Biotech.},
  title        = {Deconvolution of clinical variance in CAR-T cell pharmacology and response},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell mapping of combinatorial target antigens for CAR
switches using logic gates. <em>NBT</em>, <em>41</em>(11), 1593–1605.
(<a href="https://doi.org/10.1038/s41587-023-01686-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of optimal target antigens that distinguish cancer cells from normal surrounding tissue cells remains a key challenge in chimeric antigen receptor (CAR) cell therapy for tumors with intratumoral heterogeneity. In this study, we dissected tissue complexity to the level of individual cells through the construction of a single-cell expression atlas that integrates ~1.4 million tumor, tumor-infiltrating normal and reference normal cells from 412 tumors and 12 normal organs. We used a two-step screening method using random forest and convolutional neural networks to select gene pairs that contribute most to discrimination between individual malignant and normal cells. Tumor coverage and specificity are evaluated for the AND, OR and NOT logic gates based on the combinatorial expression pattern of the pairing genes across individual single cells. Single-cell transcriptome-coupled epitope profiling validates the AND, OR and NOT switch targets identified in ovarian cancer and colorectal cancer. CAR-T targets are identified using logic gates and single-cell expression data.},
  archive      = {J_NBT},
  author       = {Kwon, Joonha and Kang, Junho and Jo, Areum and Seo, Kayoung and An, Dohyeon and Baykan, Mert Yakup and Lee, Jun Hyeong and Kim, Nayoung and Eum, Hye Hyeon and Hwang, Sohyun and Lee, Ji Min and Park, Woong-Yang and An, Hee Jung and Lee, Hae-Ock and Park, Jong-Eun and Choi, Jung Kyoon},
  doi          = {10.1038/s41587-023-01686-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1593-1605},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell mapping of combinatorial target antigens for CAR switches using logic gates},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Retrospective analysis of enhancer activity and
transcriptome history. <em>NBT</em>, <em>41</em>(11), 1582–1592. (<a
href="https://doi.org/10.1038/s41587-023-01683-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell state changes in development and disease are controlled by gene regulatory networks, the dynamics of which are difficult to track in real time. In this study, we used an inducible DCM–RNA polymerase subunit b fusion protein which labels active genes and enhancers with a bacterial methylation mark that does not affect gene transcription and is propagated in S-phase. This DCM–RNA polymerase fusion protein enables transcribed genes and active enhancers to be tagged and then examined at later stages of development or differentiation. We apply this DCM-time machine (DCM-TM) technology to study intestinal homeostasis, revealing rapid and coordinated activation of enhancers and nearby genes during enterocyte differentiation. We provide new insights in absorptive–secretory lineage decision-making in intestinal stem cell (ISC) differentiation and show that ISCs retain a unique chromatin landscape required to maintain ISC identity and delineate future expression of differentiation-associated genes. DCM-TM has wide applicability in tracking cell states, providing new insights in the regulatory networks underlying cell state changes. Labeling transcribed genes and active enhancers with a bacterial methylation mark traces cell state changes.},
  archive      = {J_NBT},
  author       = {Boers, Ruben and Boers, Joachim and Tan, Beatrice and van Leeuwen, Marieke E. and Wassenaar, Evelyne and Sanchez, Erlantz Gonzalez and Sleddens, Esther and Tenhagen, Yasha and Mulugeta, Eskeatnaf and Laven, Joop and Creyghton, Menno and Baarends, Willy and van IJcken, Wilfred F. J. and Gribnau, Joost},
  doi          = {10.1038/s41587-023-01683-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1582-1592},
  shortjournal = {Nature Biotech.},
  title        = {Retrospective analysis of enhancer activity and transcriptome history},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineered human hepatocyte organoids enable CRISPR-based
target discovery and drug screening for steatosis. <em>NBT</em>,
<em>41</em>(11), 1567–1581. (<a
href="https://doi.org/10.1038/s41587-023-01680-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The lack of registered drugs for nonalcoholic fatty liver disease (NAFLD) is partly due to the paucity of human-relevant models for target discovery and compound screening. Here we use human fetal hepatocyte organoids to model the first stage of NAFLD, steatosis, representing three different triggers: free fatty acid loading, interindividual genetic variability (PNPLA3 I148M) and monogenic lipid disorders (APOB and MTTP mutations). Screening of drug candidates revealed compounds effective at resolving steatosis. Mechanistic evaluation of effective drugs uncovered repression of de novo lipogenesis as the convergent molecular pathway. We present FatTracer, a CRISPR screening platform to identify steatosis modulators and putative targets using APOB−/− and MTTP−/− organoids. From a screen targeting 35 genes implicated in lipid metabolism and/or NAFLD risk, FADS2 (fatty acid desaturase 2) emerged as an important determinant of hepatic steatosis. Enhancement of FADS2 expression increases polyunsaturated fatty acid abundancy which, in turn, reduces de novo lipogenesis. These organoid models facilitate study of steatosis etiology and drug targets. Organoid models of early liver disease aid target discovery and drug screening.},
  archive      = {J_NBT},
  author       = {Hendriks, Delilah and Brouwers, Jos F. and Hamer, Karien and Geurts, Maarten H. and Luciana, Léa and Massalini, Simone and López-Iglesias, Carmen and Peters, Peter J. and Rodríguez-Colman, Maria J. and Chuva de Sousa Lopes, Susana and Artegiani, Benedetta and Clevers, Hans},
  doi          = {10.1038/s41587-023-01680-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1567-1581},
  shortjournal = {Nature Biotech.},
  title        = {Engineered human hepatocyte organoids enable CRISPR-based target discovery and drug screening for steatosis},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Microfluidics-free single-cell genomics with templated
emulsification. <em>NBT</em>, <em>41</em>(11), 1557–1566. (<a
href="https://doi.org/10.1038/s41587-023-01685-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current single-cell RNA-sequencing approaches have limitations that stem from the microfluidic devices or fluid handling steps required for sample processing. We develop a method that does not require specialized microfluidic devices, expertise or hardware. Our approach is based on particle-templated emulsification, which allows single-cell encapsulation and barcoding of cDNA in uniform droplet emulsions with only a vortexer. Particle-templated instant partition sequencing (PIP-seq) accommodates a wide range of emulsification formats, including microwell plates and large-volume conical tubes, enabling thousands of samples or millions of cells to be processed in minutes. We demonstrate that PIP-seq produces high-purity transcriptomes in mouse–human mixing studies, is compatible with multiomics measurements and can accurately characterize cell types in human breast tissue compared to a commercial microfluidic platform. Single-cell transcriptional profiling of mixed phenotype acute leukemia using PIP-seq reveals the emergence of heterogeneity within chemotherapy-resistant cell subsets that were hidden by standard immunophenotyping. PIP-seq is a simple, flexible and scalable next-generation workflow that extends single-cell sequencing to new applications. A microfluidics-free, scalable single-cell RNA-sequencing method produces high-quality transcriptomes.},
  archive      = {J_NBT},
  author       = {Clark, Iain C. and Fontanez, Kristina M. and Meltzer, Robert H. and Xue, Yi and Hayford, Corey and May-Zhang, Aaron and D’Amato, Chris and Osman, Ahmad and Zhang, Jesse Q. and Hettige, Pabodha and Ishibashi, Jacob S. A. and Delley, Cyrille L. and Weisgerber, Daniel W. and Replogle, Joseph M. and Jost, Marco and Phong, Kiet T. and Kennedy, Vanessa E. and Peretz, Cheryl A. C. and Kim, Esther A. and Song, Siyou and Karlon, William and Weissman, Jonathan S. and Smith, Catherine C. and Gartner, Zev J. and Abate, Adam R.},
  doi          = {10.1038/s41587-023-01685-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1557-1566},
  shortjournal = {Nature Biotech.},
  title        = {Microfluidics-free single-cell genomics with templated emulsification},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An integrated platform for high-throughput nanoscopy.
<em>NBT</em>, <em>41</em>(11), 1549–1556. (<a
href="https://doi.org/10.1038/s41587-023-01702-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy enables three-dimensional fluorescence imaging at tens-of-nanometer resolution, but requires many camera frames to reconstruct a super-resolved image. This limits the typical throughput to tens of cells per day. While frame rates can now be increased by over an order of magnitude, the large data volumes become limiting in existing workflows. Here we present an integrated acquisition and analysis platform leveraging microscopy-specific data compression, distributed storage and distributed analysis to enable an acquisition and analysis throughput of 10,000 cells per day. The platform facilitates graphically reconfigurable analyses to be automatically initiated from the microscope during acquisition and remotely executed, and can even feed back and queue new acquisition tasks on the microscope. We demonstrate the utility of this framework by imaging hundreds of cells per well in multi-well sample formats. Our platform, implemented within the PYthon-Microscopy Environment (PYME), is easily configurable to control custom microscopes, and includes a plugin framework for user-defined extensions. A fast data processing platform enables super-resolution microscopy with increased throughput.},
  archive      = {J_NBT},
  author       = {Barentine, Andrew E. S. and Lin, Yu and Courvan, Edward M. and Kidd, Phylicia and Liu, Miao and Balduf, Leonhard and Phan, Timy and Rivera-Molina, Felix and Grace, Michael R. and Marin, Zach and Lessard, Mark and Rios Chen, Juliana and Wang, Siyuan and Neugebauer, Karla M. and Bewersdorf, Joerg and Baddeley, David},
  doi          = {10.1038/s41587-023-01702-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1549-1556},
  shortjournal = {Nature Biotech.},
  title        = {An integrated platform for high-throughput nanoscopy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-resolution alignment of single-cell and spatial
transcriptomes with CytoSPACE. <em>NBT</em>, <em>41</em>(11), 1543–1548.
(<a href="https://doi.org/10.1038/s41587-023-01697-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent studies have emphasized the importance of single-cell spatial biology, yet available assays for spatial transcriptomics have limited gene recovery or low spatial resolution. Here we introduce CytoSPACE, an optimization method for mapping individual cells from a single-cell RNA sequencing atlas to spatial expression profiles. Across diverse platforms and tissue types, we show that CytoSPACE outperforms previous methods with respect to noise tolerance and accuracy, enabling tissue cartography at single-cell resolution. CytoSPACE maps individual cells from a reference single-cell RNA sequencing atlas to spatial transcriptomics data.},
  archive      = {J_NBT},
  author       = {Vahid, Milad R. and Brown, Erin L. and Steen, Chloé B. and Zhang, Wubing and Jeon, Hyun Soo and Kang, Minji and Gentles, Andrew J. and Newman, Aaron M.},
  doi          = {10.1038/s41587-023-01697-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1543-1548},
  shortjournal = {Nature Biotech.},
  title        = {High-resolution alignment of single-cell and spatial transcriptomes with CytoSPACE},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precision RNA base editing with engineered and endogenous
effectors. <em>NBT</em>, <em>41</em>(11), 1526–1542. (<a
href="https://doi.org/10.1038/s41587-023-01927-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA base editing refers to the rewriting of genetic information within an intact RNA molecule and serves various functions, such as evasion of the endogenous immune system and regulation of protein function. To achieve this, certain enzymes have been discovered in human cells that catalyze the conversion of one nucleobase into another. This natural process could be exploited to manipulate and recode any base in a target transcript. In contrast to DNA base editing, analogous changes introduced in RNA are not permanent or inheritable but rather allow reversible and doseable effects that appeal to various therapeutic applications. The current practice of RNA base editing involves the deamination of adenosines and cytidines, which are converted to inosines and uridines, respectively. In this Review, we summarize current site-directed RNA base-editing strategies and highlight recent achievements to improve editing efficiency, precision, codon-targeting scope and in vivo delivery into disease-relevant tissues. Besides engineered editing effectors, we focus on strategies to harness endogenous adenosine deaminases acting on RNA (ADAR) enzymes and discuss limitations and future perspectives to apply the tools in basic research and as a therapeutic modality. We expect the field to realize the first RNA base-editing drug soon, likely on a well-defined genetic disease. However, the long-term challenge will be to carve out the sweet spot of the technology where its unique ability is exploited to modulate signaling cues, metabolism or other clinically relevant processes in a safe and doseable manner. Pfeiffer and Stafforst review and discuss the state-of-the-art tools for precision editing of RNA bases.},
  archive      = {J_NBT},
  author       = {Pfeiffer, Laura S. and Stafforst, Thorsten},
  doi          = {10.1038/s41587-023-01927-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1526-1542},
  shortjournal = {Nature Biotech.},
  title        = {Precision RNA base editing with engineered and endogenous effectors},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The next generation of single-cell sequencing methods can be
microfluidics-free. <em>NBT</em>, <em>41</em>(11), 1524–1525. (<a
href="https://doi.org/10.1038/s41587-023-01710-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing individual cells in a sample enables scientists to infer the unique characteristics of important subsets. Single-cell sequencing methods that rely on microfluidics for cell barcoding are limited in speed, scale and flexibility. We developed a technique that uses particle-templated emulsification instead of microfluidics and can process millions of cells within minutes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01710-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1524-1525},
  shortjournal = {Nature Biotech.},
  title        = {The next generation of single-cell sequencing methods can be microfluidics-free},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Imaging large cell populations with fast, automated
super-resolution. <em>NBT</em>, <em>41</em>(11), 1522–1523. (<a
href="https://doi.org/10.1038/s41587-023-01711-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leveraging advances in hardware and probes, we have combined innovations in optics and algorithms to allow automated single-molecule-based super-resolution imaging at unprecedented throughput. This approach allows us to obtain nanoscale information from large cell populations, bridging the gap between imaging and indirect ensemble methods.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01711-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1522-1523},
  shortjournal = {Nature Biotech.},
  title        = {Imaging large cell populations with fast, automated super-resolution},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Gene therapy for walking recovery after paralysis.
<em>NBT</em>, <em>41</em>(11), 1521. (<a
href="https://doi.org/10.1038/s41587-023-02037-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Despang, Alexandra},
  doi          = {10.1038/s41587-023-02037-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1521},
  shortjournal = {Nature Biotech.},
  title        = {Gene therapy for walking recovery after paralysis},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Precise intraoperative brain tumor classification.
<em>NBT</em>, <em>41</em>(11), 1521. (<a
href="https://doi.org/10.1038/s41587-023-02038-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Li, Rong},
  doi          = {10.1038/s41587-023-02038-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1521},
  shortjournal = {Nature Biotech.},
  title        = {Precise intraoperative brain tumor classification},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A bacterial assist for CAR-t cells. <em>NBT</em>,
<em>41</em>(11), 1521. (<a
href="https://doi.org/10.1038/s41587-023-02042-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Aschheim, Kathryn},
  doi          = {10.1038/s41587-023-02042-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1521},
  shortjournal = {Nature Biotech.},
  title        = {A bacterial assist for CAR-T cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Antibody engineering. <em>NBT</em>, <em>41</em>(11), 1520.
(<a href="https://doi.org/10.1038/s41587-023-02010-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the manufacture and design of engineered immunoglobulin polypeptides.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02010-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1520},
  shortjournal = {Nature Biotech.},
  title        = {Antibody engineering},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Do patent applications and cooperative research and
development agreements between the national cancer institute and
industry serve the public interest? <em>NBT</em>, <em>41</em>(11),
1517–1519. (<a
href="https://doi.org/10.1038/s41587-023-02004-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Policy changes are needed to ensure that Cooperative Research and Development Agreements and patent applications filed by the US National Institutes of Health are aligned with the interests of the American public.},
  archive      = {J_NBT},
  author       = {Wesevich, Austin and Ratain, Mark J.},
  doi          = {10.1038/s41587-023-02004-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1517-1519},
  shortjournal = {Nature Biotech.},
  title        = {Do patent applications and cooperative research and development agreements between the national cancer institute and industry serve the public interest?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sir ian wilmut 1944–2023. <em>NBT</em>, <em>41</em>(11),
1515–1516. (<a
href="https://doi.org/10.1038/s41587-023-02009-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Trounson, Alan and Cibelli, Jose},
  doi          = {10.1038/s41587-023-02009-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1515-1516},
  shortjournal = {Nature Biotech.},
  title        = {Sir ian wilmut 1944–2023},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The digital and analog worlds of protein engineering.
<em>NBT</em>, <em>41</em>(11), 1513–1514. (<a
href="https://doi.org/10.1038/s41587-023-01938-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Nuzhna, Lada and van Stekelenburg, Tess},
  doi          = {10.1038/s41587-023-01938-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1513-1514},
  shortjournal = {Nature Biotech.},
  title        = {The digital and analog worlds of protein engineering},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Scalable genomic data exchange and analytics with sBeacon.
<em>NBT</em>, <em>41</em>(11), 1510–1512. (<a
href="https://doi.org/10.1038/s41587-023-01972-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Wickramarachchi, Anuradha and Hosking, Brendan and Jain, Yatish and Grimes, John and O’Brien, Mitchell J. and Wright, Tracey and Burgess, Mark A. and Lin, Victor San Kho and Reisinger, Florian and Hofmann, Oliver and Lawley, Michael and Wilson, Laurence O. W. and Twine, Natalie A. and Bauer, Denis C.},
  doi          = {10.1038/s41587-023-01972-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1510-1512},
  shortjournal = {Nature Biotech.},
  title        = {Scalable genomic data exchange and analytics with sBeacon},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). 3Q23 — VCs feed “older children” as IPO markets stagnate.
<em>NBT</em>, <em>41</em>(11), 1501–1503. (<a
href="https://doi.org/10.1038/s41587-023-02016-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-02016-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1501-1503},
  shortjournal = {Nature Biotech.},
  title        = {3Q23 — VCs feed ‘older children’ as IPO markets stagnate},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Drug pipeline 3Q23 — ERT, bispecifics and CRISPR in sickle
cell disease. <em>NBT</em>, <em>41</em>(11), 1498–1500. (<a
href="https://doi.org/10.1038/s41587-023-02015-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-02015-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1498-1500},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 3Q23 — ERT, bispecifics and CRISPR in sickle cell disease},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(11), 1497. (<a
href="https://doi.org/10.1038/s41587-023-02035-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02035-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1497},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). NKs for lupus poised for trial. <em>NBT</em>,
<em>41</em>(11), 1496. (<a
href="https://doi.org/10.1038/s41587-023-02043-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02043-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1496},
  shortjournal = {Nature Biotech.},
  title        = {NKs for lupus poised for trial},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chan zuckerberg hubs multiply. <em>NBT</em>,
<em>41</em>(11), 1495. (<a
href="https://doi.org/10.1038/s41587-023-02039-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02039-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1495},
  shortjournal = {Nature Biotech.},
  title        = {Chan zuckerberg hubs multiply},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Most effective malaria vaccine yet. <em>NBT</em>,
<em>41</em>(11), 1493. (<a
href="https://doi.org/10.1038/s41587-023-02041-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02041-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1493},
  shortjournal = {Nature Biotech.},
  title        = {Most effective malaria vaccine yet},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). All that nobel glitters is not biotech gold. <em>NBT</em>,
<em>41</em>(11), 1489. (<a
href="https://doi.org/10.1038/s41587-023-02045-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum dots won the Nobel Prize, but proved to be a miss in the biomedical space.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02045-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1489},
  shortjournal = {Nature Biotech.},
  title        = {All that nobel glitters is not biotech gold},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). DNA writing technologies moving toward synthetic genomes.
<em>NBT</em>, <em>41</em>(11), 1504–1509. (<a
href="https://doi.org/10.1038/s41587-023-02006-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As new technologies hit the market, synthetic DNA is available faster and cheaper than ever before. Regulators are preparing to step in to limit opportunities for misuse.},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-023-02006-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {11},
  pages        = {1504-1509},
  shortjournal = {Nature Biotech.},
  title        = {DNA writing technologies moving toward synthetic genomes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). AI for drug discovery is booming, but who owns the patents?
<em>NBT</em>, <em>41</em>(11), 1494–1496. (<a
href="https://doi.org/10.1038/s41587-023-02029-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biopharma companies debate whether to patent their algorithms for finding medicines.},
  archive      = {J_NBT},
  author       = {Hutson, Matthew},
  doi          = {10.1038/s41587-023-02029-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {11},
  pages        = {1494-1496},
  shortjournal = {Nature Biotech.},
  title        = {AI for drug discovery is booming, but who owns the patents?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Can gene editing drive out HIV and hepatitis viruses from
inside cells? <em>NBT</em>, <em>41</em>(11), 1491–1493. (<a
href="https://doi.org/10.1038/s41587-023-02022-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Armed with gene technologies and CAR-Ts, scientists are attempting to eliminate viruses that escape immune detection and lurk in tissues for years.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-023-02022-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {11},
  pages        = {1491-1493},
  shortjournal = {Nature Biotech.},
  title        = {Can gene editing drive out HIV and hepatitis viruses from inside cells?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). People. <em>NBT</em>, <em>41</em>(10), 1488. (<a
href="https://doi.org/10.1038/s41587-023-01975-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01975-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1488},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A strategy to incentivize innovation in the health care
system: The innovation RVU. <em>NBT</em>, <em>41</em>(10), 1485–1487.
(<a href="https://doi.org/10.1038/s41587-023-01971-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Encouraging innovation will allow physicians and their academic medical centers to pursue the development of technologies and healthcare practices that will improve patient care.},
  archive      = {J_NBT},
  author       = {Mallipeddi, Nathan V. and Chandra, Jay and Succi, Marc D.},
  doi          = {10.1038/s41587-023-01971-w},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1485-1487},
  shortjournal = {Nature Biotech.},
  title        = {A strategy to incentivize innovation in the health care system: The innovation RVU},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Unidirectional single-file transport of
full-length proteins through a nanopore. <em>NBT</em>, <em>41</em>(10),
1483. (<a href="https://doi.org/10.1038/s41587-023-01995-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yu, Luning and Kang, Xinqi and Li, Fanjun and Mehrafrooz, Behzad and Makhamreh, Amr and Fallahi, Ali and Foster, Joshua C. and Aksimentiev, Aleksei and Chen, Min and Wanunu, Meni},
  doi          = {10.1038/s41587-023-01995-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1483},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Unidirectional single-file transport of full-length proteins through a nanopore},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Telomere-to-telomere assembly of diploid chromosomes with
verkko. <em>NBT</em>, <em>41</em>(10), 1474–1482. (<a
href="https://doi.org/10.1038/s41587-023-01662-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Telomere-to-Telomere consortium recently assembled the first truly complete sequence of a human genome. To resolve the most complex repeats, this project relied on manual integration of ultra-long Oxford Nanopore sequencing reads with a high-resolution assembly graph built from long, accurate PacBio high-fidelity reads. We have improved and automated this strategy in Verkko, an iterative, graph-based pipeline for assembling complete, diploid genomes. Verkko begins with a multiplex de Bruijn graph built from long, accurate reads and progressively simplifies this graph by integrating ultra-long reads and haplotype-specific markers. The result is a phased, diploid assembly of both haplotypes, with many chromosomes automatically assembled from telomere to telomere. Running Verkko on the HG002 human genome resulted in 20 of 46 diploid chromosomes assembled without gaps at 99.9997\% accuracy. The complete assembly of diploid genomes is a critical step towards the construction of comprehensive pangenome databases and chromosome-scale comparative genomics. Integration of long and ultra-long reads results in improved phased, diploid assemblies.},
  archive      = {J_NBT},
  author       = {Rautiainen, Mikko and Nurk, Sergey and Walenz, Brian P. and Logsdon, Glennis A. and Porubsky, David and Rhie, Arang and Eichler, Evan E. and Phillippy, Adam M. and Koren, Sergey},
  doi          = {10.1038/s41587-023-01662-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1474-1482},
  shortjournal = {Nature Biotech.},
  title        = {Telomere-to-telomere assembly of diploid chromosomes with verkko},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). TACCO unifies annotation transfer and decomposition of cell
identities for single-cell and spatial omics. <em>NBT</em>,
<em>41</em>(10), 1465–1473. (<a
href="https://doi.org/10.1038/s41587-023-01657-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transferring annotations of single-cell-, spatial- and multi-omics data is often challenging owing both to technical limitations, such as low spatial resolution or high dropout fraction, and to biological variations, such as continuous spectra of cell states. Based on the concept that these data are often best described as continuous mixtures of cells or molecules, we present a computational framework for the transfer of annotations to cells and their combinations (TACCO), which consists of an optimal transport model extended with different wrappers to annotate a wide variety of data. We apply TACCO to identify cell types and states, decipher spatiomolecular tissue structure at the cell and molecular level and resolve differentiation trajectories using synthetic and biological datasets. While matching or exceeding the accuracy of specialized tools for the individual tasks, TACCO reduces the computational requirements by up to an order of magnitude and scales to larger datasets (for example, considering the runtime of annotation transfer for 1 M simulated dropout observations). Annotation transfer from reference to new datasets is improved with a probabilistic approach.},
  archive      = {J_NBT},
  author       = {Mages, Simon and Moriel, Noa and Avraham-Davidi, Inbal and Murray, Evan and Watter, Jan and Chen, Fei and Rozenblatt-Rosen, Orit and Klughammer, Johanna and Regev, Aviv and Nitzan, Mor},
  doi          = {10.1038/s41587-023-01657-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1465-1473},
  shortjournal = {Nature Biotech.},
  title        = {TACCO unifies annotation transfer and decomposition of cell identities for single-cell and spatial omics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Simultaneous sequencing of genetic and epigenetic bases in
DNA. <em>NBT</em>, <em>41</em>(10), 1457–1464. (<a
href="https://doi.org/10.1038/s41587-022-01652-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA comprises molecular information stored in genetic and epigenetic bases, both of which are vital to our understanding of biology. Most DNA sequencing approaches address either genetics or epigenetics and thus capture incomplete information. Methods widely used to detect epigenetic DNA bases fail to capture common C-to-T mutations or distinguish 5-methylcytosine from 5-hydroxymethylcytosine. We present a single base-resolution sequencing methodology that sequences complete genetics and the two most common cytosine modifications in a single workflow. DNA is copied and bases are enzymatically converted. Coupled decoding of bases across the original and copy strand provides a phased digital readout. Methods are demonstrated on human genomic DNA and cell-free DNA from a blood sample of a patient with cancer. The approach is accurate, requires low DNA input and has a simple workflow and analysis pipeline. Simultaneous, phased reading of genetic and epigenetic bases provides a more complete picture of the information stored in genomes and has applications throughout biomedicine. A six-letter sequencing workflow can simultaneously detect genetic and epigenetic bases.},
  archive      = {J_NBT},
  author       = {Füllgrabe, Jens and Gosal, Walraj S. and Creed, Páidí and Liu, Sidong and Lumby, Casper K. and Morley, David J. and Ost, Tobias W. B. and Vilella, Albert J. and Yu, Shirong and Bignell, Helen and Burns, Philippa and Charlesworth, Tom and Fu, Beiyuan and Fordham, Howerd and Harding, Nicolas J. and Gandelman, Olga and Golder, Paula and Hodson, Christopher and Li, Mengjie and Lila, Marjana and Liu, Yang and Mason, Joanne and Mellad, Jason and Monahan, Jack M. and Nentwich, Oliver and Palmer, Alexandra and Steward, Michael and Taipale, Minna and Vandomme, Audrey and San-Bento, Rita Santo and Singhal, Ankita and Vivian, Julia and Wójtowicz, Natalia and Williams, Nathan and Walker, Nicolas J. and Wong, Nicola C. H. and Yalloway, Gary N. and Holbrook, Joanna D. and Balasubramanian, Shankar},
  doi          = {10.1038/s41587-022-01652-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1457-1464},
  shortjournal = {Nature Biotech.},
  title        = {Simultaneous sequencing of genetic and epigenetic bases in DNA},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Prediction of prime editing insertion efficiencies using
sequence features and DNA repair determinants. <em>NBT</em>,
<em>41</em>(10), 1446–1456. (<a
href="https://doi.org/10.1038/s41587-023-01678-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most short sequences can be precisely written into a selected genomic target using prime editing; however, it remains unclear what factors govern insertion. We design a library of 3,604 sequences of various lengths and measure the frequency of their insertion into four genomic sites in three human cell lines, using different prime editor systems in varying DNA repair contexts. We find that length, nucleotide composition and secondary structure of the insertion sequence all affect insertion rates. We also discover that the 3′ flap nucleases TREX1 and TREX2 suppress the insertion of longer sequences. Combining the sequence and repair features into a machine learning model, we can predict relative frequency of insertions into a site with R = 0.70. Finally, we demonstrate how our accurate prediction and user-friendly software help choose codon variants of common fusion tags that insert at high efficiency, and provide a catalog of empirically determined insertion rates for over a hundred useful sequences. Prime editing efficiency is affected by insertion sequence features, secondary structure and DNA repair context.},
  archive      = {J_NBT},
  author       = {Koeppel, Jonas and Weller, Juliane and Peets, Elin Madli and Pallaseni, Ananth and Kuzmin, Ivan and Raudvere, Uku and Peterson, Hedi and Liberante, Fabio Giuseppe and Parts, Leopold},
  doi          = {10.1038/s41587-023-01678-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1446-1456},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of prime editing insertion efficiencies using sequence features and DNA repair determinants},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Synapse-tuned CARs enhance immune cell anti-tumor activity.
<em>NBT</em>, <em>41</em>(10), 1434–1445. (<a
href="https://doi.org/10.1038/s41587-022-01650-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor (CAR) technologies have been clinically implemented for the treatment of hematological malignancies; however, solid tumors remain resilient to CAR therapeutics. Natural killer (NK) cells may provide an optimal class of immune cells for CAR-based approaches due to their inherent anti-tumor functionality. In this study, we sought to tune CAR immune synapses by adding an intracellular scaffolding protein binding site to the CAR. We employ a PDZ binding motif (PDZbm) that enables additional scaffolding crosslinks that enhance synapse formation and NK CAR cell polarization. Combined effects of this CAR design result in increased effector cell functionality in vitro and in vivo. Additionally, we used T cells and observed similar global enhancements in effector function. Synapse-tuned CAR immune cells exhibit amplified synaptic strength, number and abundance of secreted cytokines, enhanced killing of tumor cells and prolonged survival in numerous different tumor models, including solid tumors. Synapse-tuned CAR immune cells effectively kill solid tumor cells.},
  archive      = {J_NBT},
  author       = {Chockley, Peter J. and Ibanez-Vega, Jorge and Krenciute, Giedre and Talbot, Lindsay J. and Gottschalk, Stephen},
  doi          = {10.1038/s41587-022-01650-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1434-1445},
  shortjournal = {Nature Biotech.},
  title        = {Synapse-tuned CARs enhance immune cell anti-tumor activity},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-throughput microbial culturomics using automation and
machine learning. <em>NBT</em>, <em>41</em>(10), 1424–1433. (<a
href="https://doi.org/10.1038/s41587-023-01674-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pure bacterial cultures remain essential for detailed experimental and mechanistic studies in microbiome research, and traditional methods to isolate individual bacteria from complex microbial ecosystems are labor-intensive, difficult-to-scale and lack phenotype–genotype integration. Here we describe an open-source high-throughput robotic strain isolation platform for the rapid generation of isolates on demand. We develop a machine learning approach that leverages colony morphology and genomic data to maximize the diversity of microbes isolated and enable targeted picking of specific genera. Application of this platform on fecal samples from 20 humans yields personalized gut microbiome biobanks totaling 26,997 isolates that represented &amp;gt;80\% of all abundant taxa. Spatial analysis on &amp;gt;100,000 visually captured colonies reveals cogrowth patterns between Ruminococcaceae, Bacteroidaceae, Coriobacteriaceae and Bifidobacteriaceae families that suggest important microbial interactions. Comparative analysis of 1,197 high-quality genomes from these biobanks shows interesting intra- and interpersonal strain evolution, selection and horizontal gene transfer. This culturomics framework should empower new research efforts to systematize the collection and quantitative analysis of imaging-based phenotypes with high-resolution genomics data for many emerging microbiome studies. A machine learning isolation and genotyping platform enable high-throughput bacterial culture generation.},
  archive      = {J_NBT},
  author       = {Huang, Yiming and Sheth, Ravi U. and Zhao, Shijie and Cohen, Lucas A. and Dabaghi, Kendall and Moody, Thomas and Sun, Yiwei and Ricaurte, Deirdre and Richardson, Miles and Velez-Cortes, Florencia and Blazejewski, Tomasz and Kaufman, Andrew and Ronda, Carlotta and Wang, Harris H.},
  doi          = {10.1038/s41587-023-01674-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1424-1433},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput microbial culturomics using automation and machine learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GutSMASH predicts specialized primary metabolic pathways
from the human gut microbiota. <em>NBT</em>, <em>41</em>(10), 1416–1423.
(<a href="https://doi.org/10.1038/s41587-023-01675-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The gut microbiota produce hundreds of small molecules, many of which modulate host physiology. Although efforts have been made to identify biosynthetic genes for secondary metabolites, the chemical output of the gut microbiome consists predominantly of primary metabolites. Here we introduce the gutSMASH algorithm for identification of primary metabolic gene clusters, and we used it to systematically profile gut microbiome metabolism, identifying 19,890 gene clusters in 4,240 high-quality microbial genomes. We found marked differences in pathway distribution among phyla, reflecting distinct strategies for energy capture. These data explain taxonomic differences in short-chain fatty acid production and suggest a characteristic metabolic niche for each taxon. Analysis of 1,135 individuals from a Dutch population-based cohort shows that the level of microbiome-derived metabolites in plasma and feces is almost completely uncorrelated with the metagenomic abundance of corresponding metabolic genes, indicating a crucial role for pathway-specific gene regulation and metabolite flux. This work is a starting point for understanding differences in how bacterial taxa contribute to the chemistry of the microbiome. Taxon-specific primary metabolic pathways are identified using profile hidden Markov models.},
  archive      = {J_NBT},
  author       = {Pascal Andreu, Victòria and Augustijn, Hannah E. and Chen, Lianmin and Zhernakova, Alexandra and Fu, Jingyuan and Fischbach, Michael A. and Dodd, Dylan and Medema, Marnix H.},
  doi          = {10.1038/s41587-023-01675-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1416-1423},
  shortjournal = {Nature Biotech.},
  title        = {GutSMASH predicts specialized primary metabolic pathways from the human gut microbiota},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Combinatorial design of nanoparticles for pulmonary mRNA
delivery and genome editing. <em>NBT</em>, <em>41</em>(10), 1410–1415.
(<a href="https://doi.org/10.1038/s41587-023-01679-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The expanding applications of nonviral genomic medicines in the lung remain restricted by delivery challenges. Here, leveraging a high-throughput platform, we synthesize and screen a combinatorial library of biodegradable ionizable lipids to build inhalable delivery vehicles for messenger RNA and CRISPR–Cas9 gene editors. Lead lipid nanoparticles are amenable for repeated intratracheal dosing and could achieve efficient gene editing in lung epithelium, providing avenues for gene therapy of congenital lung diseases. A high-throughput screen improves lipid nanoparticle delivery of gene editors in the lung.},
  archive      = {J_NBT},
  author       = {Li, Bowen and Manan, Rajith Singh and Liang, Shun-Qing and Gordon, Akiva and Jiang, Allen and Varley, Andrew and Gao, Guangping and Langer, Robert and Xue, Wen and Anderson, Daniel},
  doi          = {10.1038/s41587-023-01679-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1410-1415},
  shortjournal = {Nature Biotech.},
  title        = {Combinatorial design of nanoparticles for pulmonary mRNA delivery and genome editing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-plex protein and whole transcriptome co-mapping at
cellular resolution with spatial CITE-seq. <em>NBT</em>,
<em>41</em>(10), 1405–1409. (<a
href="https://doi.org/10.1038/s41587-023-01676-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, we extended co-indexing of transcriptomes and epitopes (CITE) to the spatial dimension and demonstrated high-plex protein and whole transcriptome co-mapping. We profiled 189 proteins and whole transcriptome in multiple mouse tissue types with spatial CITE sequencing and then further applied the method to measure 273 proteins and transcriptome in human tissues, revealing spatially distinct germinal center reactions in tonsil and early immune activation in skin at the Coronavirus Disease 2019 mRNA vaccine injection site. Co-indexing of transcriptomes and epitopes is extended to the spatial dimension with large protein panels.},
  archive      = {J_NBT},
  author       = {Liu, Yang and DiStasio, Marcello and Su, Graham and Asashima, Hiromitsu and Enninful, Archibald and Qin, Xiaoyu and Deng, Yanxiang and Nam, Jungmin and Gao, Fu and Bordignon, Pino and Cassano, Marco and Tomayko, Mary and Xu, Mina and Halene, Stephanie and Craft, Joseph E. and Hafler, David and Fan, Rong},
  doi          = {10.1038/s41587-023-01676-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1405-1409},
  shortjournal = {Nature Biotech.},
  title        = {High-plex protein and whole transcriptome co-mapping at cellular resolution with spatial CITE-seq},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Interstrand crosslinking of homologous repair template DNA
enhances gene editing in human cells. <em>NBT</em>, <em>41</em>(10),
1398–1404. (<a
href="https://doi.org/10.1038/s41587-022-01654-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe a strategy to boost the efficiency of gene editing via homology-directed repair (HDR) by covalently modifying the template DNA with interstrand crosslinks. Crosslinked templates (xHDRTs) increase Cas9-mediated editing efficiencies by up to fivefold in K562, HEK293T, U2OS, iPS and primary T cells. Increased editing from xHDRTs is driven by events on the template molecule and requires ataxia telangiectasia and Rad3-related (ATR) kinase and components of the Fanconi anemia pathway. The efficiency of homologous recombination is increased by crosslinking template DNA.},
  archive      = {J_NBT},
  author       = {Ghasemi, Hannah I. and Bacal, Julien and Yoon, Amanda C. and Tavasoli, Katherine U. and Cruz, Carmen and Vu, Jonathan T. and Gardner, Brooke M. and Richardson, Chris D.},
  doi          = {10.1038/s41587-022-01654-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1398-1404},
  shortjournal = {Nature Biotech.},
  title        = {Interstrand crosslinking of homologous repair template DNA enhances gene editing in human cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Computational framework decomposes and annotates single-cell
and spatial omics. <em>NBT</em>, <em>41</em>(10), 1396–1397. (<a
href="https://doi.org/10.1038/s41587-023-01658-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed TACCO, a versatile and efficient computational annotation framework, leveraging growing single-cell and spatial omics data to identify cell types and states, decipher molecular and cellular tissue structure, and resolve differentiation trajectories.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01658-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1396-1397},
  shortjournal = {Nature Biotech.},
  title        = {Computational framework decomposes and annotates single-cell and spatial omics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Lipid nanoparticle-enabled gene editing in the lung via
inhalation. <em>NBT</em>, <em>41</em>(10), 1394–1395. (<a
href="https://doi.org/10.1038/s41587-023-01689-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to create in vivo genomic medicines for tissues other than the liver has been impeded by difficulties in delivery. Using a high-throughput platform, we developed lipid nanoparticles that can effectively deliver mRNA and CRISPR–Cas9 gene editing tools to the lungs through intratracheal administration, expanding the potential clinical uses of gene editing and mRNA-based technologies.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01689-9},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1394-1395},
  shortjournal = {Nature Biotech.},
  title        = {Lipid nanoparticle-enabled gene editing in the lung via inhalation},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Speed reading the epigenome and genome. <em>NBT</em>,
<em>41</em>(10), 1392–1393. (<a
href="https://doi.org/10.1038/s41587-023-01757-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dual sequencing of the epigenome and genome could have broad implications in oncology.},
  archive      = {J_NBT},
  author       = {George, James M. and Chinnaiyan, Arul M.},
  doi          = {10.1038/s41587-023-01757-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1392-1393},
  shortjournal = {Nature Biotech.},
  title        = {Speed reading the epigenome and genome},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Human microbiome cultivation expands with AI. <em>NBT</em>,
<em>41</em>(10), 1389–1391. (<a
href="https://doi.org/10.1038/s41587-023-01852-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An automated system for picking bacterial colonies is used to create a biobank of personalized microbiomes.},
  archive      = {J_NBT},
  author       = {Selma-Royo, Marta and Segata, Nicola and Ricci, Liviana},
  doi          = {10.1038/s41587-023-01852-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1389-1391},
  shortjournal = {Nature Biotech.},
  title        = {Human microbiome cultivation expands with AI},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CRISPR editing in the lung with novel lipids. <em>NBT</em>,
<em>41</em>(10), 1387–1388. (<a
href="https://doi.org/10.1038/s41587-023-01744-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR RNA is delivered to the mouse lung by inhalation using improved lipid nanoparticles.},
  archive      = {J_NBT},
  author       = {Elia, Uri and Kon, Edo and Peer, Dan},
  doi          = {10.1038/s41587-023-01744-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1387-1388},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR editing in the lung with novel lipids},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Implantable biosensor tracks temperature to detect early
transplant rejection in rats. <em>NBT</em>, <em>41</em>(10), 1386. (<a
href="https://doi.org/10.1038/s41587-023-01998-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bishop, Anahita},
  doi          = {10.1038/s41587-023-01998-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1386},
  shortjournal = {Nature Biotech.},
  title        = {Implantable biosensor tracks temperature to detect early transplant rejection in rats},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Advancing missense variant pathogenicity prediction.
<em>NBT</em>, <em>41</em>(10), 1386. (<a
href="https://doi.org/10.1038/s41587-023-01999-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Li, Rong},
  doi          = {10.1038/s41587-023-01999-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1386},
  shortjournal = {Nature Biotech.},
  title        = {Advancing missense variant pathogenicity prediction},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Monitoring protein proximity with fluorogens. <em>NBT</em>,
<em>41</em>(10), 1386. (<a
href="https://doi.org/10.1038/s41587-023-02000-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Despang, Alexandra},
  doi          = {10.1038/s41587-023-02000-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1386},
  shortjournal = {Nature Biotech.},
  title        = {Monitoring protein proximity with fluorogens},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Antimicrobials. <em>NBT</em>, <em>41</em>(10), 1385. (<a
href="https://doi.org/10.1038/s41587-023-01970-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the production of antimicrobial agents and materials.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01970-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1385},
  shortjournal = {Nature Biotech.},
  title        = {Antimicrobials},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Are ecological processes that select beneficial traits in
agricultural microbes nature’s intellectual property rights?
<em>NBT</em>, <em>41</em>(10), 1381–1384. (<a
href="https://doi.org/10.1038/s41587-023-01966-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Novel beneficial traits in agricultural microbes represent inventive steps of nature, but the inability of patent laws to reward nonhuman inventors has led to conflicts over microbial ownership rights and presents barriers to the sharing of benefits.},
  archive      = {J_NBT},
  author       = {Kothamasi, David and Vermeylen, Saskia and Deepika, Sharma},
  doi          = {10.1038/s41587-023-01966-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1381-1384},
  shortjournal = {Nature Biotech.},
  title        = {Are ecological processes that select beneficial traits in agricultural microbes nature’s intellectual property rights?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A new chance for genome editing in europe. <em>NBT</em>,
<em>41</em>(10), 1378–1380. (<a
href="https://doi.org/10.1038/s41587-023-01969-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Vanderschuren, Hervé and Chatukuta, Patience and Weigel, Detlef and Mehta, Devang},
  doi          = {10.1038/s41587-023-01969-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1378-1380},
  shortjournal = {Nature Biotech.},
  title        = {A new chance for genome editing in europe},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tenure time loopers. <em>NBT</em>, <em>41</em>(10),
1375–1377. (<a
href="https://doi.org/10.1038/s41587-023-01924-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yachie, Nozomu and Shakiba, Nika},
  doi          = {10.1038/s41587-023-01924-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1375-1377},
  shortjournal = {Nature Biotech.},
  title        = {Tenure time loopers},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The long game of tenure. <em>NBT</em>, <em>41</em>(10),
1373–1374. (<a
href="https://doi.org/10.1038/s41587-023-01929-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Treangen, Todd J. and Vardi, Moshe Y.},
  doi          = {10.1038/s41587-023-01929-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1373-1374},
  shortjournal = {Nature Biotech.},
  title        = {The long game of tenure},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(10), 1364. (<a
href="https://doi.org/10.1038/s41587-023-01993-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01993-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1364},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Autoimmune neuroinflammation newco. <em>NBT</em>,
<em>41</em>(10), 1363. (<a
href="https://doi.org/10.1038/s41587-023-02007-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02007-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1363},
  shortjournal = {Nature Biotech.},
  title        = {Autoimmune neuroinflammation newco},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Upswing in AI drug-discovery deals. <em>NBT</em>,
<em>41</em>(10), 1361. (<a
href="https://doi.org/10.1038/s41587-023-02002-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02002-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1361},
  shortjournal = {Nature Biotech.},
  title        = {Upswing in AI drug-discovery deals},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). New guise ADCs couple antibodies to degraders. <em>NBT</em>,
<em>41</em>(10), 1360. (<a
href="https://doi.org/10.1038/s41587-023-02001-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02001-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1360},
  shortjournal = {Nature Biotech.},
  title        = {New guise ADCs couple antibodies to degraders},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The community of the DAO. <em>NBT</em>, <em>41</em>(10),
1357. (<a href="https://doi.org/10.1038/s41587-023-02005-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Decentralized autonomous organizations are growing as alternative research funding models, but are also strong scientific communities. We should get on board.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-02005-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1357},
  shortjournal = {Nature Biotech.},
  title        = {The community of the DAO},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Pharma deal making: A bright spot amid the gloom.
<em>NBT</em>, <em>41</em>(10), 1365–1372. (<a
href="https://doi.org/10.1038/s41587-023-01958-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biopharma M&amp;amp;A is back and IPOs are possible, but most biotechs aren’t out of the woods.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-023-01958-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {10},
  pages        = {1365-1372},
  shortjournal = {Nature Biotech.},
  title        = {Pharma deal making: A bright spot amid the gloom},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). From air to your plate: Tech startups making food from
atmospheric CO2. <em>NBT</em>, <em>41</em>(10), 1362–1364. (<a
href="https://doi.org/10.1038/s41587-023-01992-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Food tech companies are taking bacteria that capture CO2 from air to make edible and nutritious food and drink.},
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-023-01992-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {10},
  pages        = {1362-1364},
  shortjournal = {Nature Biotech.},
  title        = {From air to your plate: Tech startups making food from atmospheric CO2},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Scientists hone tools to measure aging and rejuvenation
interventions. <em>NBT</em>, <em>41</em>(10), 1359–1361. (<a
href="https://doi.org/10.1038/d41587-023-00008-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aging is malleable, at least in animal models. But to prove the efficacy of interventions aimed at extending healthspan and to test life-extending approaches in humans, longevity researchers first need to agree on the best measuring tools. With the power of ’omics, AI and large biobanks, improved biomarkers may be forthcoming. Companies and academic researchers are rallying forces, but how far have they gotten? Aging is malleable, at least in animal models. But to prove the efficacy of interventions aimed at extending healthspan and to test life-extending approaches in humans, longevity researchers first need to agree on the best measuring tools. With the power of ’omics, AI and large biobanks, improved biomarkers may be forthcoming. Companies and academic researchers are rallying forces, but how far have they gotten?},
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/d41587-023-00008-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {10},
  pages        = {1359-1361},
  shortjournal = {Nature Biotech.},
  title        = {Scientists hone tools to measure aging and rejuvenation interventions},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). People. <em>NBT</em>, <em>41</em>(9), 1356. (<a
href="https://doi.org/10.1038/s41587-023-01941-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01941-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1356},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Five questions with mor nitzan. <em>NBT</em>,
<em>41</em>(9), 1355. (<a
href="https://doi.org/10.1038/s41587-023-01926-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An expert in network dynamics and cell interactions describes her interdisciplinary background, creativity and building community in her lab during a pandemic.},
  archive      = {J_NBT},
  author       = {Chuong, Julie},
  doi          = {10.1038/s41587-023-01926-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1355},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with mor nitzan},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Bridging the gap in african biodiversity genomics and
bioinformatics. <em>NBT</em>, <em>41</em>(9), 1348–1354. (<a
href="https://doi.org/10.1038/s41587-023-01933-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Open Institute of the African BioGenome Project empowers African scientists and institutions with the skill sets, capacity and infrastructure to advance scientific knowledge and innovation and drive economic growth.},
  archive      = {J_NBT},
  author       = {Sharaf, Abdoallah and Ndiribe, Charlotte C. and Omotoriogun, Taiwo Crossby and Abueg, Linelle and Badaoui, Bouabid and Badiane Markey, Fatu J. and Beedessee, Girish and Diouf, Diaga and Duru, Vincent C. and Ebuzome, Chukwuike and Eziuzor, Samuel C. and Jaufeerally Fakim, Yasmina and Formenti, Giulio and Ghanmi, Nidhal and Guerfali, Fatma Zahra and Houaga, Isidore and Ideozu, Justin Eze and Katee, Sally Mueni and Khayi, Slimane and Kuja, Josiah O. and Kwon-Ndung, Emmanuel Hala and Marks, Rose A. and Moila, Acclaim M. and Mungloo-Dilmohamud, Zahra and Muzemil, Sadik and Nigussie, Helen and Osuji, Julian O. and Ras, Verena and Tchiechoua, Yves H. and Zoclanclounon, Yedomon Ange Bovys and Tolley, Krystal A. and Ziyomo, Cathrine and Mapholi, Ntanganedzeni and Muigai, Anne W. T. and Djikeng, Appolinaire and Ebenezer, ThankGod Echezona},
  doi          = {10.1038/s41587-023-01933-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1348-1354},
  shortjournal = {Nature Biotech.},
  title        = {Bridging the gap in african biodiversity genomics and bioinformatics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Guidelines for public database
submission of uncultivated virus genome sequences for taxonomic
classification. <em>NBT</em>, <em>41</em>(9), 1346. (<a
href="https://doi.org/10.1038/s41587-023-01952-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Adriaenssens, Evelien M. and Roux, Simon and Brister, J. Rodney and Karsch-Mizrachi, Ilene and Kuhn, Jens H. and Varsani, Arvind and Yigang, Tong and Reyes, Alejandro and Lood, Cédric and Lefkowitz, Elliot J. and Sullivan, Matthew B. and Edwards, Robert A. and Simmonds, Peter and Rubino, Luisa and Sabanadzovic, Sead and Krupovic, Mart and Dutilh, Bas E.},
  doi          = {10.1038/s41587-023-01952-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1346},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Guidelines for public database submission of uncultivated virus genome sequences for taxonomic classification},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Massively parallel single-cell
mitochondrial DNA genotyping and chromatin profiling. <em>NBT</em>,
<em>41</em>(9), 1345. (<a
href="https://doi.org/10.1038/s41587-023-01942-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Lareau, Caleb A. and Ludwig, Leif S. and Muus, Christoph and Gohil, Satyen H. and Zhao, Tongtong and Chiang, Zachary and Pelka, Karin and Verboon, Jeffrey M. and Luo, Wendy and Christian, Elena and Rosebrock, Daniel and Getz, Gad and Boland, Genevieve M. and Chen, Fei and Buenrostro, Jason D. and Hacohen, Nir and Wu, Catherine J. and Aryee, Martin J. and Regev, Aviv and Sankaran, Vijay G.},
  doi          = {10.1038/s41587-023-01942-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1345},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Massively parallel single-cell mitochondrial DNA genotyping and chromatin profiling},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Droplet-based transcriptome profiling of individual
synapses. <em>NBT</em>, <em>41</em>(9), 1332–1344. (<a
href="https://doi.org/10.1038/s41587-022-01635-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Synapses are crucial structures that mediate signal transmission between neurons in complex neural circuits and display considerable morphological and electrophysiological heterogeneity. So far we still lack a high-throughput method to profile the molecular heterogeneity among individual synapses. In the present study, we develop a droplet-based single-cell (sc) total-RNA-sequencing platform, called Multiple-Annealing-and-Tailing-based Quantitative scRNA-seq in Droplets, for transcriptome profiling of individual neurites, primarily composed of synaptosomes. In the synaptosome transcriptome, or ‘synaptome’, profiling of both mouse and human brain samples, we detect subclusters among synaptosomes that are associated with neuronal subtypes and characterize the landscape of transcript splicing that occurs within synapses. We extend synaptome profiling to synaptopathy in an Alzheimer’s disease (AD) mouse model and discover AD-associated synaptic gene expression changes that cannot be detected by single-nucleus transcriptome profiling. Overall, our results show that this platform provides a high-throughput, single-synaptosome transcriptome profiling tool that will facilitate future discoveries in neuroscience. High-throughput profiling of the transcriptomes of individual synapses shows molecular heterogeneity.},
  archive      = {J_NBT},
  author       = {Niu, Muchun and Cao, Wenjian and Wang, Yongcheng and Zhu, Qiangyuan and Luo, Jiayi and Wang, Baiping and Zheng, Hui and Weitz, David A. and Zong, Chenghang},
  doi          = {10.1038/s41587-022-01635-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1332-1344},
  shortjournal = {Nature Biotech.},
  title        = {Droplet-based transcriptome profiling of individual synapses},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-scale metabolic reconstruction of 7,302 human
microorganisms for personalized medicine. <em>NBT</em>, <em>41</em>(9),
1320–1331. (<a
href="https://doi.org/10.1038/s41587-022-01628-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human microbiome influences the efficacy and safety of a wide variety of commonly prescribed drugs. Designing precision medicine approaches that incorporate microbial metabolism would require strain- and molecule-resolved, scalable computational modeling. Here, we extend our previous resource of genome-scale metabolic reconstructions of human gut microorganisms with a greatly expanded version. AGORA2 (assembly of gut organisms through reconstruction and analysis, version 2) accounts for 7,302 strains, includes strain-resolved drug degradation and biotransformation capabilities for 98 drugs, and was extensively curated based on comparative genomics and literature searches. The microbial reconstructions performed very well against three independently assembled experimental datasets with an accuracy of 0.72 to 0.84, surpassing other reconstruction resources and predicted known microbial drug transformations with an accuracy of 0.81. We demonstrate that AGORA2 enables personalized, strain-resolved modeling by predicting the drug conversion potential of the gut microbiomes from 616 patients with colorectal cancer and controls, which greatly varied between individuals and correlated with age, sex, body mass index and disease stages. AGORA2 serves as a knowledge base for the human microbiome and paves the way to personalized, predictive analysis of host–microbiome metabolic interactions. A resource of microbial metabolic models reveals how an individual’s gut microbiome influences drug metabolism.},
  archive      = {J_NBT},
  author       = {Heinken, Almut and Hertel, Johannes and Acharya, Geeta and Ravcheev, Dmitry A. and Nyga, Malgorzata and Okpala, Onyedika Emmanuel and Hogan, Marcus and Magnúsdóttir, Stefanía and Martinelli, Filippo and Nap, Bram and Preciat, German and Edirisinghe, Janaka N. and Henry, Christopher S. and Fleming, Ronan M. T. and Thiele, Ines},
  doi          = {10.1038/s41587-022-01628-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1320-1331},
  shortjournal = {Nature Biotech.},
  title        = {Genome-scale metabolic reconstruction of 7,302 human microorganisms for personalized medicine},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Three-dimensional structured illumination microscopy with
enhanced axial resolution. <em>NBT</em>, <em>41</em>(9), 1307–1319. (<a
href="https://doi.org/10.1038/s41587-022-01651-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The axial resolution of three-dimensional structured illumination microscopy (3D SIM) is limited to ∼300 nm. Here we present two distinct, complementary methods to improve axial resolution in 3D SIM with minimal or no modification to the optical system. We show that placing a mirror directly opposite the sample enables four-beam interference with higher spatial frequency content than 3D SIM illumination, offering near-isotropic imaging with ∼120-nm lateral and 160-nm axial resolution. We also developed a deep learning method achieving ∼120-nm isotropic resolution. This method can be combined with denoising to facilitate volumetric imaging spanning dozens of timepoints. We demonstrate the potential of these advances by imaging a variety of cellular samples, delineating the nanoscale distribution of vimentin and microtubule filaments, observing the relative positions of caveolar coat proteins and lysosomal markers and visualizing cytoskeletal dynamics within T cells in the early stages of immune synapse formation. Enhanced axial resolution for 3D SIM is achieved with deep learning or four-beam interference.},
  archive      = {J_NBT},
  author       = {Li, Xuesong and Wu, Yicong and Su, Yijun and Rey-Suarez, Ivan and Matthaeus, Claudia and Updegrove, Taylor B. and Wei, Zhuang and Zhang, Lixia and Sasaki, Hideki and Li, Yue and Guo, Min and Giannini, John P. and Vishwasrao, Harshad D. and Chen, Jiji and Lee, Shih-Jong J. and Shao, Lin and Liu, Huafeng and Ramamurthi, Kumaran S. and Taraska, Justin W. and Upadhyaya, Arpita and La Riviere, Patrick and Shroff, Hari},
  doi          = {10.1038/s41587-022-01651-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1307-1319},
  shortjournal = {Nature Biotech.},
  title        = {Three-dimensional structured illumination microscopy with enhanced axial resolution},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Control of acute myeloid leukemia by a trifunctional
NKp46-CD16a-NK cell engager targeting CD123. <em>NBT</em>,
<em>41</em>(9), 1296–1306. (<a
href="https://doi.org/10.1038/s41587-022-01626-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CD123, the alpha chain of the IL-3 receptor, is an attractive target for acute myeloid leukemia (AML) treatment. However, cytotoxic antibodies or T cell engagers targeting CD123 had insufficient efficacy or safety in clinical trials. We show that expression of CD64, the high-affinity receptor for human IgG, on AML blasts confers resistance to anti-CD123 antibody-dependent cell cytotoxicity (ADCC) in vitro. We engineer a trifunctional natural killer cell engager (NKCE) that targets CD123 on AML blasts and NKp46 and CD16a on NK cells (CD123-NKCE). CD123-NKCE has potent antitumor activity against primary AML blasts regardless of CD64 expression and induces NK cell activation and cytokine secretion only in the presence of AML cells. Its antitumor activity in a mouse CD123+ tumor model exceeds that of the benchmark ADCC-enhanced antibody. In nonhuman primates, it had prolonged pharmacodynamic effects, depleting CD123+ cells for more than 10 days with no signs of toxicity and very low inflammatory cytokine induction over a large dose range. These results support clinical development of CD123-NKCE. Leukemia cells resistant to CD123-targeted therapies are susceptible to a trifunctional NK cell engager.},
  archive      = {J_NBT},
  author       = {Gauthier, Laurent and Virone-Oddos, Angela and Beninga, Jochen and Rossi, Benjamin and Nicolazzi, Céline and Amara, Céline and Blanchard-Alvarez, Audrey and Gourdin, Nicolas and Courta, Jacqueline and Basset, Alexandra and Agnel, Magali and Guillot, Franceline and Grondin, Gwendoline and Bonnevaux, Hélène and Bauchet, Anne-Laure and Morel, Ariane and Morel, Yannis and Chiron, Marielle and Vivier, Eric},
  doi          = {10.1038/s41587-022-01626-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1296-1306},
  shortjournal = {Nature Biotech.},
  title        = {Control of acute myeloid leukemia by a trifunctional NKp46-CD16a-NK cell engager targeting CD123},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tension-tuned receptors for synthetic mechanotransduction
and intercellular force detection. <em>NBT</em>, <em>41</em>(9),
1287–1295. (<a
href="https://doi.org/10.1038/s41587-022-01638-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells interpret mechanical stimuli from their environments and neighbors, but the ability to engineer customized mechanosensing capabilities has remained a synthetic and mechanobiology challenge. Here we introduce tension-tuned synthetic Notch (SynNotch) receptors to convert extracellular and intercellular forces into specifiable gene expression changes. By elevating the tension requirements of SynNotch activation, in combination with structure-guided mutagenesis, we designed a set of receptors with mechanical sensitivities spanning the physiologically relevant picoNewton range. Cells expressing these receptors can distinguish between varying tensile forces and respond by enacting customizable transcriptional programs. We applied these tools to design a decision-making circuit, through which fibroblasts differentiate into myoblasts upon stimulation with distinct tension magnitudes. We also characterize cell-generated forces transmitted between cells during Notch signaling. Overall, this work provides insight into how mechanically induced changes in protein structure can be used to transduce physical forces into biochemical signals. The system should facilitate the further programming and dissection of force-related phenomena in biological systems. Synthetic sensors of extra and intracellular mechanical forces that activate gene expression are designed.},
  archive      = {J_NBT},
  author       = {Sloas, D. Christopher and Tran, Jeremy C. and Marzilli, Alexander M. and Ngo, John T.},
  doi          = {10.1038/s41587-022-01638-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1287-1295},
  shortjournal = {Nature Biotech.},
  title        = {Tension-tuned receptors for synthetic mechanotransduction and intercellular force detection},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial transcriptomics for profiling the tropism of viral
vectors in tissues. <em>NBT</em>, <em>41</em>(9), 1272–1286. (<a
href="https://doi.org/10.1038/s41587-022-01648-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A barrier to advancing engineered adeno-associated viral vectors (AAVs) for precision access to cell subtypes is a lack of high-throughput, high-resolution assays to characterize in vivo transduction profiles. In this study, we developed an ultrasensitive, sequential fluorescence in situ hybridization (USeqFISH) method for spatial transcriptomic profiling of endogenous and viral RNA with a short barcode in intact tissue volumes by integrating hydrogel-based tissue clearing, enhanced signal amplification and multiplexing using sequential labeling. Using USeqFISH, we investigated the transduction and cell subtype tropisms across mouse brain regions of six systemic AAVs, including AAV-PHP.AX, a new variant that transduces robustly and efficiently across neurons and astrocytes. Here we reveal distinct cell subtype biases of each AAV variant, including a bias of AAV-PHP.N toward excitatory neurons. USeqFISH also enables profiling of pooled regulatory cargos, as we show for a 13-variant pool of microRNA target sites in AAV genomes. Lastly, we demonstrate potential applications of USeqFISH for in situ AAV profiling and multimodal single-cell analysis in non-human primates. Dual mapping shows localization patterns of AAV and RNA in intact tissues.},
  archive      = {J_NBT},
  author       = {Jang, Min J. and Coughlin, Gerard M. and Jackson, Cameron R. and Chen, Xinhong and Chuapoco, Miguel R. and Vendemiatti, Julia L. and Wang, Alexander Z. and Gradinaru, Viviana},
  doi          = {10.1038/s41587-022-01648-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1272-1286},
  shortjournal = {Nature Biotech.},
  title        = {Spatial transcriptomics for profiling the tropism of viral vectors in tissues},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chem-map profiles drug binding to chromatin in cells.
<em>NBT</em>, <em>41</em>(9), 1265–1271. (<a
href="https://doi.org/10.1038/s41587-022-01636-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing drug–target engagement is essential to understand how small molecules influence cellular functions. Here we present Chem-map for in situ mapping of small molecules that interact with DNA or chromatin-associated proteins, utilizing small-molecule-directed transposase Tn5 tagmentation. We demonstrate Chem-map for three distinct drug-binding modalities as follows: molecules that target a chromatin protein, a DNA secondary structure or that intercalate in DNA. We map the BET bromodomain protein-binding inhibitor JQ1 and provide interaction maps for DNA G-quadruplex structure-binding molecules PDS and PhenDC3. Moreover, we determine the binding sites of the widely used anticancer drug doxorubicin in human leukemia cells; using the Chem-map of doxorubicin in cells exposed to the histone deacetylase inhibitor tucidinostat reveals the potential clinical advantages of this combination therapy. In situ mapping with Chem-map of small-molecule interactions with DNA and chromatin proteins provides insights that will enhance understanding of genome and chromatin function and therapeutic interventions. How small molecules bind chromatin and DNA is determined by Chem-map.},
  archive      = {J_NBT},
  author       = {Yu, Zutao and Spiegel, Jochen and Melidis, Larry and Hui, Winnie W. I. and Zhang, Xiaoyun and Radzevičius, Antanas and Balasubramanian, Shankar},
  doi          = {10.1038/s41587-022-01636-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1265-1271},
  shortjournal = {Nature Biotech.},
  title        = {Chem-map profiles drug binding to chromatin in cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precise transcript targeting by CRISPR-csm complexes.
<em>NBT</em>, <em>41</em>(9), 1256–1264. (<a
href="https://doi.org/10.1038/s41587-022-01649-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Robust and precise transcript targeting in mammalian cells remains a difficult challenge using existing approaches due to inefficiency, imprecision and subcellular compartmentalization. Here we show that the clustered regularly interspaced short palindromic repeats (CRISPR)-Csm complex, a multiprotein effector from type III CRISPR immune systems in prokaryotes, provides surgical RNA ablation of both nuclear and cytoplasmic transcripts. As part of the most widely occurring CRISPR adaptive immune pathway, CRISPR-Csm uses a programmable RNA-guided mechanism to find and degrade target RNA molecules without inducing indiscriminate trans-cleavage of cellular RNAs, giving it an important advantage over the CRISPR-Cas13 family of enzymes. Using single-vector delivery of the Streptococcus thermophilus Csm complex, we observe high-efficiency RNA knockdown (90–99\%) and minimal off-target effects in human cells, outperforming existing technologies including short hairpin RNA- and Cas13-mediated knockdown. We also find that catalytically inactivated Csm achieves specific and durable RNA binding, a property we harness for live-cell RNA imaging. These results establish the feasibility and efficacy of multiprotein CRISPR-Cas effector complexes as RNA-targeting tools in eukaryotes. The bacterial Csm complex efficiently knocks down eukaryotic nuclear and cytoplasmic RNAs.},
  archive      = {J_NBT},
  author       = {Colognori, David and Trinidad, Marena and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-022-01649-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1256-1264},
  shortjournal = {Nature Biotech.},
  title        = {Precise transcript targeting by CRISPR-csm complexes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Massively parallel knock-in engineering of human t cells.
<em>NBT</em>, <em>41</em>(9), 1239–1255. (<a
href="https://doi.org/10.1038/s41587-022-01639-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The efficiency of targeted knock-in for cell therapeutic applications is generally low, and the scale is limited. In this study, we developed CLASH, a system that enables high-efficiency, high-throughput knock-in engineering. In CLASH, Cas12a/Cpf1 mRNA combined with pooled adeno-associated viruses mediate simultaneous gene editing and precise transgene knock-in using massively parallel homology-directed repair, thereby producing a pool of stably integrated mutant variants each with targeted gene editing. We applied this technology in primary human T cells and performed time-coursed CLASH experiments in blood cancer and solid tumor models using CD3, CD8 and CD4 T cells, enabling pooled generation and unbiased selection of favorable CAR-T variants. Emerging from CLASH experiments, a unique CRISPR RNA (crRNA) generates an exon3 skip mutant of PRDM1 in CAR-Ts, which leads to increased proliferation, stem-like properties, central memory and longevity in these cells, resulting in higher efficacy in vivo across multiple cancer models, including a solid tumor model. The versatility of CLASH makes it broadly applicable to diverse cellular and therapeutic engineering applications. Unbiased selection of CAR-T cells is achieved by massively parallel knock-in engineering.},
  archive      = {J_NBT},
  author       = {Dai, Xiaoyun and Park, Jonathan J. and Du, Yaying and Na, Zhenkun and Lam, Stanley Z. and Chow, Ryan D. and Renauer, Paul A. and Gu, Jianlei and Xin, Shan and Chu, Zhiyuan and Liao, Cun and Clark, Paul and Zhao, Hongyu and Slavoff, Sarah and Chen, Sidi},
  doi          = {10.1038/s41587-022-01639-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1239-1255},
  shortjournal = {Nature Biotech.},
  title        = {Massively parallel knock-in engineering of human t cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precision targeting of autoantigen-specific b cells in
muscle-specific tyrosine kinase myasthenia gravis with chimeric
autoantibody receptor t cells. <em>NBT</em>, <em>41</em>(9), 1229–1238.
(<a href="https://doi.org/10.1038/s41587-022-01637-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Muscle-specific tyrosine kinase myasthenia gravis (MuSK MG) is an autoimmune disease that causes life-threatening muscle weakness due to anti-MuSK autoantibodies that disrupt neuromuscular junction signaling. To avoid chronic immunosuppression from current therapies, we engineered T cells to express a MuSK chimeric autoantibody receptor with CD137-CD3ζ signaling domains (MuSK-CAART) for precision targeting of B cells expressing anti-MuSK autoantibodies. MuSK-CAART demonstrated similar efficacy as anti-CD19 chimeric antigen receptor T cells for depletion of anti-MuSK B cells and retained cytolytic activity in the presence of soluble anti-MuSK antibodies. In an experimental autoimmune MG mouse model, MuSK-CAART reduced anti-MuSK IgG without decreasing B cells or total IgG levels, reflecting MuSK-specific B cell depletion. Specific off-target interactions of MuSK-CAART were not identified in vivo, in primary human cell screens or by high-throughput human membrane proteome array. These data contributed to an investigational new drug application and phase 1 clinical study design for MuSK-CAART for the treatment of MuSK autoantibody-positive MG. A muscle autoimmune disease is treated in mice with CAAR T cells.},
  archive      = {J_NBT},
  author       = {Oh, Sangwook and Mao, Xuming and Manfredo-Vieira, Silvio and Lee, Jinmin and Patel, Darshil and Choi, Eun Jung and Alvarado, Andrea and Cottman-Thomas, Ebony and Maseda, Damian and Tsao, Patricia Y. and Ellebrecht, Christoph T. and Khella, Sami L. and Richman, David P. and O’Connor, Kevin C. and Herzberg, Uri and Binder, Gwendolyn K. and Milone, Michael C. and Basu, Samik and Payne, Aimee S.},
  doi          = {10.1038/s41587-022-01637-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1229-1238},
  shortjournal = {Nature Biotech.},
  title        = {Precision targeting of autoantigen-specific b cells in muscle-specific tyrosine kinase myasthenia gravis with chimeric autoantibody receptor t cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reply to: Methodological concerns and lack of evidence for
single-synapse RNA-seq. <em>NBT</em>, <em>41</em>(9), 1225–1228. (<a
href="https://doi.org/10.1038/s41587-023-01878-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Niu, Muchun and Zong, Chenghang},
  doi          = {10.1038/s41587-023-01878-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1225-1228},
  shortjournal = {Nature Biotech.},
  title        = {Reply to: Methodological concerns and lack of evidence for single-synapse RNA-seq},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Methodological concerns and lack of evidence for
single-synapse RNA-seq. <em>NBT</em>, <em>41</em>(9), 1221–1224. (<a
href="https://doi.org/10.1038/s41587-023-01877-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hobson, Benjamin D. and Herzog, Etienne},
  doi          = {10.1038/s41587-023-01877-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1221-1224},
  shortjournal = {Nature Biotech.},
  title        = {Methodological concerns and lack of evidence for single-synapse RNA-seq},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Wearable sensors for monitoring marine environments and
their inhabitants. <em>NBT</em>, <em>41</em>(9), 1208–1220. (<a
href="https://doi.org/10.1038/s41587-023-01827-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human societies depend on marine ecosystems, but their degradation continues. Toward mitigating this decline, new and more effective ways to precisely measure the status and condition of marine environments are needed alongside existing rebuilding strategies. Here, we provide an overview of how sensors and wearable technology developed for humans could be adapted to improve marine monitoring. We describe barriers that have slowed the transition of this technology from land to sea, update on the developments in sensors to advance ocean observation and advocate for more widespread use of wearables on marine organisms in the wild and in aquaculture. We propose that large-scale use of wearables could facilitate the concept of an ‘internet of marine life’ that might contribute to a more robust and effective observation system for the oceans and commercial aquaculture operations. These observations may aid in rationalizing strategies toward conservation and restoration of marine communities and habitats. Wearable sensors inform on aquaculture and marine-conservation strategies.},
  archive      = {J_NBT},
  author       = {Kaidarova, Altynay and Geraldi, Nathan R. and Wilson, Rory P. and Kosel, Jürgen and Meekan, Mark G. and Eguíluz, Víctor M. and Hussain, Muhammad Mustafa and Shamim, Atif and Liao, Hanguang and Srivastava, Mani and Saha, Swapnil Sayan and Strano, Michael S. and Zhang, Xiangliang and Ooi, Boon S. and Holton, Mark and Hopkins, Lloyd W. and Jin, Xiaojia and Gong, Xun and Quintana, Flavio and Tovasarov, Adylkhan and Tasmagambetova, Assel and Duarte, Carlos M.},
  doi          = {10.1038/s41587-023-01827-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1208-1220},
  shortjournal = {Nature Biotech.},
  title        = {Wearable sensors for monitoring marine environments and their inhabitants},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial tropism profiling of AAV vectors by ultrasensitive
sequential FISH in tissue. <em>NBT</em>, <em>41</em>(9), 1206–1207. (<a
href="https://doi.org/10.1038/s41587-023-01681-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultrasensitive sequential fluorescence in situ hybridization (USeqFISH) enables multiplexed detection of the expression of endogenous and exogenous genes delivered by adeno-associated virus (AAV) vectors in intact tissue. USeqFISH provides a spatial map of AAV tropism with high throughput and resolution.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01681-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1206-1207},
  shortjournal = {Nature Biotech.},
  title        = {Spatial tropism profiling of AAV vectors by ultrasensitive sequential FISH in tissue},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CRISPR-csm for eukaryotic RNA knockdown and imaging without
toxicity. <em>NBT</em>, <em>41</em>(9), 1204–1205. (<a
href="https://doi.org/10.1038/s41587-023-01665-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted RNA degradation remains a challenge using existing methods owing to off-target effects and toxicity. We adapted the CRISPR-Csm complex, a multi-protein effector from type III CRISPR systems, for precise knockdown of nuclear or cytoplasmic transcripts in eukaryotic cells with minimal side effects.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01665-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1204-1205},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR-csm for eukaryotic RNA knockdown and imaging without toxicity},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CLASH enables large-scale parallel knock-in for cell
engineering. <em>NBT</em>, <em>41</em>(9), 1202–1203. (<a
href="https://doi.org/10.1038/s41587-023-01664-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CLASH is a CRISPR-based platform that enables the parallel knock-in via homology-directed repair of a large pool of transgene variants encoded in adeno-associated virus vectors. CLASH can be applied to the systematic and unbiased selection of favorable features in T cells and, in principle, other cell types.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01664-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1202-1203},
  shortjournal = {Nature Biotech.},
  title        = {CLASH enables large-scale parallel knock-in for cell engineering},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Mining extinct proteomes for antimicrobial peptides.
<em>NBT</em>, <em>41</em>(9), 1201. (<a
href="https://doi.org/10.1038/s41587-023-01959-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Cheifet, Barbara},
  doi          = {10.1038/s41587-023-01959-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1201},
  shortjournal = {Nature Biotech.},
  title        = {Mining extinct proteomes for antimicrobial peptides},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A biosensor detects tumor DNA in vivo. <em>NBT</em>,
<em>41</em>(9), 1201. (<a
href="https://doi.org/10.1038/s41587-023-01960-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Doerr, Anne},
  doi          = {10.1038/s41587-023-01960-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1201},
  shortjournal = {Nature Biotech.},
  title        = {A biosensor detects tumor DNA in vivo},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Targeting hematopoietic stem cells in vivo. <em>NBT</em>,
<em>41</em>(9), 1201. (<a
href="https://doi.org/10.1038/s41587-023-01961-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Aschheim, Kathryn},
  doi          = {10.1038/s41587-023-01961-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1201},
  shortjournal = {Nature Biotech.},
  title        = {Targeting hematopoietic stem cells in vivo},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Microbiomics. <em>NBT</em>, <em>41</em>(9), 1200. (<a
href="https://doi.org/10.1038/s41587-023-01922-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to microbiome classification, modulation and treatment of diseases.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01922-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1200},
  shortjournal = {Nature Biotech.},
  title        = {Microbiomics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The global patent landscape of mRNA for diagnosis and
therapy. <em>NBT</em>, <em>41</em>(9), 1193–1199. (<a
href="https://doi.org/10.1038/s41587-023-01925-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COVID-19 pandemic brought mRNA vaccines to market in a short period, pointing the entire drug development field in the direction of mRNA treatment.},
  archive      = {J_NBT},
  author       = {Lyu, Mengru and Chen, Jiyuan and Peng, Yeheng and Han, Fang and Gong, Luyao and Guo, Jingfei and Tian, Lijuan and Gao, Yuan},
  doi          = {10.1038/s41587-023-01925-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1193-1199},
  shortjournal = {Nature Biotech.},
  title        = {The global patent landscape of mRNA for diagnosis and therapy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A way forward for diagnosis of patients with extremely rare
genetic mutations. <em>NBT</em>, <em>41</em>(9), 1190–1192. (<a
href="https://doi.org/10.1038/s41587-023-01879-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Crooke, Stanley T. and Kim-McManus, Olivia S. and Dalby, Kelley},
  doi          = {10.1038/s41587-023-01879-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1190-1192},
  shortjournal = {Nature Biotech.},
  title        = {A way forward for diagnosis of patients with extremely rare genetic mutations},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Advancing artificial intelligence to help feed the world.
<em>NBT</em>, <em>41</em>(9), 1188–1189. (<a
href="https://doi.org/10.1038/s41587-023-01898-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hayes, Ben J. and Chen, Chensong and Powell, Owen and Dinglasan, Eric and Villiers, Kira and Kemper, Kathryn E. and Hickey, Lee T.},
  doi          = {10.1038/s41587-023-01898-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1188-1189},
  shortjournal = {Nature Biotech.},
  title        = {Advancing artificial intelligence to help feed the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Plant cell-based drug delivery enhances affordability of
biologics. <em>NBT</em>, <em>41</em>(9), 1186–1187. (<a
href="https://doi.org/10.1038/s41587-023-01899-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Daniell, Henry and Kulchar, Rachel J. and Herzog, Roland W. and Kulis, Mike and Leong, Kam W.},
  doi          = {10.1038/s41587-023-01899-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1186-1187},
  shortjournal = {Nature Biotech.},
  title        = {Plant cell-based drug delivery enhances affordability of biologics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Public biotech in 2022 — the numbers. <em>NBT</em>,
<em>41</em>(9), 1185. (<a
href="https://doi.org/10.1038/s41587-023-01923-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura and Lähteenmäki, Riku},
  doi          = {10.1038/s41587-023-01923-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1185},
  shortjournal = {Nature Biotech.},
  title        = {Public biotech in 2022 — the numbers},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(9), 1184. (<a
href="https://doi.org/10.1038/s41587-023-01946-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01946-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1184},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Parkinson’s iPSC trial. <em>NBT</em>, <em>41</em>(9), 1183.
(<a href="https://doi.org/10.1038/s41587-023-01965-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01965-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1183},
  shortjournal = {Nature Biotech.},
  title        = {Parkinson’s iPSC trial},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Drugs catch a ride through the blood–brain barrier.
<em>NBT</em>, <em>41</em>(9), 1182–1184. (<a
href="https://doi.org/10.1038/s41587-023-01936-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new era of brain-penetrant therapies is opening up, as companies hook up drugs to transferrin receptors or use viral capsids to hitch a ride into previously inaccessible regions.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-023-01936-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1182-1184},
  shortjournal = {Nature Biotech.},
  title        = {Drugs catch a ride through the blood–brain barrier},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Henrietta lacks lawsuits continue. <em>NBT</em>,
<em>41</em>(9), 1181. (<a
href="https://doi.org/10.1038/s41587-023-01963-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01963-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1181},
  shortjournal = {Nature Biotech.},
  title        = {Henrietta lacks lawsuits continue},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). FDA approves first-in-class bispecific for multiple myeloma.
<em>NBT</em>, <em>41</em>(9), 1180. (<a
href="https://doi.org/10.1038/s41587-023-01962-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01962-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1180},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves first-in-class bispecific for multiple myeloma},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Hair loss treatments take aim at the immune system.
<em>NBT</em>, <em>41</em>(9), 1179–1181. (<a
href="https://doi.org/10.1038/s41587-023-01939-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drugs commonly used in rheumatology, the JAK inhibitors, are opening up treatment options for people with autoimmune-driven hair loss, and new mechanistic knowledge is also helping scientists tackle androgenic alopecia, the commonest cause of baldness.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-023-01939-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1179-1181},
  shortjournal = {Nature Biotech.},
  title        = {Hair loss treatments take aim at the immune system},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). When are your cells no longer your own? <em>NBT</em>,
<em>41</em>(9), 1177. (<a
href="https://doi.org/10.1038/s41587-023-01968-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell lines from human samples have historically benefitted scientific research but continue to raise questions about consent.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01968-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1177},
  shortjournal = {Nature Biotech.},
  title        = {When are your cells no longer your own?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). People. <em>NBT</em>, <em>41</em>(8), 1176. (<a
href="https://doi.org/10.1038/s41587-023-01896-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01896-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1176},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). 1H23 biotech job picture. <em>NBT</em>, <em>41</em>(8),
1175. (<a href="https://doi.org/10.1038/s41587-023-01895-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A semiannual snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-023-01895-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1175},
  shortjournal = {Nature Biotech.},
  title        = {1H23 biotech job picture},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Grand challenges in bioinformatics education and training.
<em>NBT</em>, <em>41</em>(8), 1171–1174. (<a
href="https://doi.org/10.1038/s41587-023-01891-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the increasing need for bioinformatics expertise in the life sciences, a coordinated effort is needed to proactively and critically reflect on the challenges of educating and training life scientists with the necessary skills and competencies.},
  archive      = {J_NBT},
  author       = {Işık, Esra Büşra and Brazas, Michelle D. and Schwartz, Russell and Gaeta, Bruno and Palagi, Patricia M. and van Gelder, Celia W. G. and Suravajhala, Prashanth and Singh, Harpreet and Morgan, Sarah L. and Zahroh, Hilyatuz and Ling, Maurice and Satagopam, Venkata P. and McGrath, Annette and Nakai, Kenta and Tan, Tin Wee and Gao, Ge and Mulder, Nicola and Schönbach, Christian and Zheng, Yun and De Las Rivas, Javier and Khan, Asif M.},
  doi          = {10.1038/s41587-023-01891-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1171-1174},
  shortjournal = {Nature Biotech.},
  title        = {Grand challenges in bioinformatics education and training},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Real-time, volumetric imaging of radiation dose delivery
deep into the liver during cancer treatment. <em>NBT</em>,
<em>41</em>(8), 1160–1167. (<a
href="https://doi.org/10.1038/s41587-022-01593-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ionizing radiation acoustic imaging (iRAI) allows online monitoring of radiation’s interactions with tissues during radiation therapy, providing real-time, adaptive feedback for cancer treatments. We describe an iRAI volumetric imaging system that enables mapping of the three-dimensional (3D) radiation dose distribution in a complex clinical radiotherapy treatment. The method relies on a two-dimensional matrix array transducer and a matching multi-channel preamplifier board. The feasibility of imaging temporal 3D dose accumulation was first validated in a tissue-mimicking phantom. Next, semiquantitative iRAI relative dose measurements were verified in vivo in a rabbit model. Finally, real-time visualization of the 3D radiation dose delivered to a patient with liver metastases was accomplished with a clinical linear accelerator. These studies demonstrate the potential of iRAI to monitor and quantify the 3D radiation dose deposition during treatment, potentially improving radiotherapy treatment efficacy using real-time adaptive treatment. Radiation dose delivery is measured in real time for a patient with cancer.},
  archive      = {J_NBT},
  author       = {Zhang, Wei and Oraiqat, Ibrahim and Litzenberg, Dale and Chang, Kai-Wei and Hadley, Scott and Sunbul, Noora Ba and Matuszak, Martha M. and Tichacek, Christopher J. and Moros, Eduardo G. and Carson, Paul L. and Cuneo, Kyle C. and Wang, Xueding and El Naqa, Issam},
  doi          = {10.1038/s41587-022-01593-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1160-1167},
  shortjournal = {Nature Biotech.},
  title        = {Real-time, volumetric imaging of radiation dose delivery deep into the liver during cancer treatment},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Predicting prime editing efficiency and product purity by
deep learning. <em>NBT</em>, <em>41</em>(8), 1151–1159. (<a
href="https://doi.org/10.1038/s41587-022-01613-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing is a versatile genome editing tool but requires experimental optimization of the prime editing guide RNA (pegRNA) to achieve high editing efficiency. Here we conducted a high-throughput screen to analyze prime editing outcomes of 92,423 pegRNAs on a highly diverse set of 13,349 human pathogenic mutations that include base substitutions, insertions and deletions. Based on this dataset, we identified sequence context features that influence prime editing and trained PRIDICT (prime editing guide prediction), an attention-based bidirectional recurrent neural network. PRIDICT reliably predicts editing rates for all small-sized genetic changes with a Spearman’s R of 0.85 and 0.78 for intended and unintended edits, respectively. We validated PRIDICT on endogenous editing sites as well as an external dataset and showed that pegRNAs with high (&amp;gt;70) versus low (&amp;lt;70) PRIDICT scores showed substantially increased prime editing efficiencies in different cell types in vitro (12-fold) and in hepatocytes in vivo (tenfold), highlighting the value of PRIDICT for basic and for translational research applications. The design of prime editing guide RNAs is optimized by deep learning.},
  archive      = {J_NBT},
  author       = {Mathis, Nicolas and Allam, Ahmed and Kissling, Lucas and Marquart, Kim Fabiano and Schmidheini, Lukas and Solari, Cristina and Balázs, Zsolt and Krauthammer, Michael and Schwank, Gerald},
  doi          = {10.1038/s41587-022-01613-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1151-1159},
  shortjournal = {Nature Biotech.},
  title        = {Predicting prime editing efficiency and product purity by deep learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cell-type-specific prediction of 3D chromatin organization
enables high-throughput in silico genetic screening. <em>NBT</em>,
<em>41</em>(8), 1140–1150. (<a
href="https://doi.org/10.1038/s41587-022-01612-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Investigating how chromatin organization determines cell-type-specific gene expression remains challenging. Experimental methods for measuring three-dimensional chromatin organization, such as Hi-C, are costly and have technical limitations, restricting their broad application particularly in high-throughput genetic perturbations. We present C.Origami, a multimodal deep neural network that performs de novo prediction of cell-type-specific chromatin organization using DNA sequence and two cell-type-specific genomic features—CTCF binding and chromatin accessibility. C.Origami enables in silico experiments to examine the impact of genetic changes on chromatin interactions. We further developed an in silico genetic screening approach to assess how individual DNA elements may contribute to chromatin organization and to identify putative cell-type-specific trans-acting regulators that collectively determine chromatin architecture. Applying this approach to leukemia cells and normal T cells, we demonstrate that cell-type-specific in silico genetic screening, enabled by C.Origami, can be used to systematically discover novel chromatin regulation circuits in both normal and disease-related biological systems. Deep learning predicts cell-type-specific 3D chromatin contacts from DNA sequence, chromatin accessibility and CTCF binding.},
  archive      = {J_NBT},
  author       = {Tan, Jimin and Shenker-Tauris, Nina and Rodriguez-Hernaez, Javier and Wang, Eric and Sakellaropoulos, Theodore and Boccalatte, Francesco and Thandapani, Palaniraja and Skok, Jane and Aifantis, Iannis and Fenyö, David and Xia, Bo and Tsirigos, Aristotelis},
  doi          = {10.1038/s41587-022-01612-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1140-1150},
  shortjournal = {Nature Biotech.},
  title        = {Cell-type-specific prediction of 3D chromatin organization enables high-throughput in silico genetic screening},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Unidirectional single-file transport of full-length
proteins through a nanopore. <em>NBT</em>, <em>41</em>(8), 1130–1139.
(<a href="https://doi.org/10.1038/s41587-022-01598-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The electrical current blockade of a peptide or protein threading through a nanopore can be used as a fingerprint of the molecule in biosensor applications. However, threading of full-length proteins has only been achieved using enzymatic unfolding and translocation. Here we describe an enzyme-free approach for unidirectional, slow transport of full-length proteins through nanopores. We show that the combination of a chemically resistant biological nanopore, α-hemolysin (narrowest part is ~1.4 nm in diameter), and a high concentration guanidinium chloride buffer enables unidirectional, single-file protein transport propelled by an electroosmotic effect. We show that the mean protein translocation velocity depends linearly on the applied voltage and translocation times depend linearly on length, resembling the translocation dynamics of ssDNA. Using a supervised machine-learning classifier, we demonstrate that single-translocation events contain sufficient information to distinguish their threading orientation and identity with accuracies larger than 90\%. Capture rates of protein are increased substantially when either a genetically encoded charged peptide tail or a DNA tag is added to a protein. Full-length, unfolded proteins are slowly translocated through nanopores without enzymes and fingerprinted.},
  archive      = {J_NBT},
  author       = {Yu, Luning and Kang, Xinqi and Li, Fanjun and Mehrafrooz, Behzad and Makhamreh, Amr and Fallahi, Ali and Foster, Joshua C. and Aksimentiev, Aleksei and Chen, Min and Wanunu, Meni},
  doi          = {10.1038/s41587-022-01598-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1130-1139},
  shortjournal = {Nature Biotech.},
  title        = {Unidirectional single-file transport of full-length proteins through a nanopore},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A universal deep-learning model for zinc finger design
enables transcription factor reprogramming. <em>NBT</em>,
<em>41</em>(8), 1117–1129. (<a
href="https://doi.org/10.1038/s41587-022-01624-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cys2His2 zinc finger (ZF) domains engineered to bind specific target sequences in the genome provide an effective strategy for programmable regulation of gene expression, with many potential therapeutic applications. However, the structurally intricate engagement of ZF domains with DNA has made their design challenging. Here we describe the screening of 49 billion protein–DNA interactions and the development of a deep-learning model, ZFDesign, that solves ZF design for any genomic target. ZFDesign is a modern machine learning method that models global and target-specific differences induced by a range of library environments and specifically takes into account compatibility of neighboring fingers using a novel hierarchical transformer architecture. We demonstrate the versatility of designed ZFs as nucleases as well as activators and repressors by seamless reprogramming of human transcription factors. These factors could be used to upregulate an allele of haploinsufficiency, downregulate a gain-of-function mutation or test the consequence of regulation of a single gene as opposed to the many genes that a transcription factor would normally influence. Zinc finger design is facilitated with a deep-learning model.},
  archive      = {J_NBT},
  author       = {Ichikawa, David M. and Abdin, Osama and Alerasool, Nader and Kogenaru, Manjunatha and Mueller, April L. and Wen, Han and Giganti, David O. and Goldberg, Gregory W. and Adams, Samantha and Spencer, Jeffrey M. and Razavi, Rozita and Nim, Satra and Zheng, Hong and Gionco, Courtney and Clark, Finnegan T. and Strokach, Alexey and Hughes, Timothy R. and Lionnet, Timothee and Taipale, Mikko and Kim, Philip M. and Noyes, Marcus B.},
  doi          = {10.1038/s41587-022-01624-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1117-1129},
  shortjournal = {Nature Biotech.},
  title        = {A universal deep-learning model for zinc finger design enables transcription factor reprogramming},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). RNA recording in single bacterial cells using reprogrammed
tracrRNAs. <em>NBT</em>, <em>41</em>(8), 1107–1116. (<a
href="https://doi.org/10.1038/s41587-022-01604-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Capturing an individual cell’s transcriptional history is a challenge exacerbated by the functional heterogeneity of cellular communities. Here, we leverage reprogrammed tracrRNAs (Rptrs) to record selected cellular transcripts as stored DNA edits in single living bacterial cells. Rptrs are designed to base pair with sensed transcripts, converting them into guide RNAs. The guide RNAs then direct a Cas9 base editor to target an introduced DNA target. The extent of base editing can then be read in the future by sequencing. We use this approach, called TIGER (transcribed RNAs inferred by genetically encoded records), to record heterologous and endogenous transcripts in individual bacterial cells. TIGER can quantify relative expression, distinguish single-nucleotide differences, record multiple transcripts simultaneously and read out single-cell phenomena. We further apply TIGER to record metabolic bet hedging and antibiotic resistance mobilization in Escherichia coli as well as host cell invasion by Salmonella. Through RNA recording, TIGER connects current cellular states with past transcriptional states to decipher complex cellular responses in single cells. RNA transcripts in bacteria are recorded as DNA modifications using a base editor.},
  archive      = {J_NBT},
  author       = {Jiao, Chunlei and Reckstadt, Claas and König, Fabian and Homberger, Christina and Yu, Jiaqi and Vogel, Jörg and Westermann, Alexander J. and Sharma, Cynthia M. and Beisel, Chase L.},
  doi          = {10.1038/s41587-022-01604-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1107-1116},
  shortjournal = {Nature Biotech.},
  title        = {RNA recording in single bacterial cells using reprogrammed tracrRNAs},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Large language models generate functional protein sequences
across diverse families. <em>NBT</em>, <em>41</em>(8), 1099–1106. (<a
href="https://doi.org/10.1038/s41587-022-01618-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep-learning language models have shown promise in various biotechnological applications, including protein design and engineering. Here we describe ProGen, a language model that can generate protein sequences with a predictable function across large protein families, akin to generating grammatically and semantically correct natural language sentences on diverse topics. The model was trained on 280 million protein sequences from &amp;gt;19,000 families and is augmented with control tags specifying protein properties. ProGen can be further fine-tuned to curated sequences and tags to improve controllable generation performance of proteins from families with sufficient homologous samples. Artificial proteins fine-tuned to five distinct lysozyme families showed similar catalytic efficiencies as natural lysozymes, with sequence identity to natural proteins as low as 31.4\%. ProGen is readily adapted to diverse protein families, as we demonstrate with chorismate mutase and malate dehydrogenase. A generative deep-learning model designs artificial proteins with desired enzymatic activities.},
  archive      = {J_NBT},
  author       = {Madani, Ali and Krause, Ben and Greene, Eric R. and Subramanian, Subu and Mohr, Benjamin P. and Holton, James M. and Olmos, Jose Luis and Xiong, Caiming and Sun, Zachary Z. and Socher, Richard and Fraser, James S. and Naik, Nikhil},
  doi          = {10.1038/s41587-022-01618-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1099-1106},
  shortjournal = {Nature Biotech.},
  title        = {Large language models generate functional protein sequences across diverse families},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineered live bacteria suppress pseudomonas aeruginosa
infection in mouse lung and dissolve endotracheal-tube biofilms.
<em>NBT</em>, <em>41</em>(8), 1089–1098. (<a
href="https://doi.org/10.1038/s41587-022-01584-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered live bacteria could provide a new modality for treating lung infections, a major cause of mortality worldwide. In the present study, we engineered a genome-reduced human lung bacterium, Mycoplasma pneumoniae, to treat ventilator-associated pneumonia, a disease with high hospital mortality when associated with Pseudomonas aeruginosa biofilms. After validating the biosafety of an attenuated M. pneumoniae chassis in mice, we introduced four transgenes into the chromosome by transposition to implement bactericidal and biofilm degradation activities. We show that this engineered strain has high efficacy against an acute P. aeruginosa lung infection in a mouse model. In addition, we demonstrated that the engineered strain could dissolve biofilms formed in endotracheal tubes of patients with ventilator-associated pneumonia and be combined with antibiotics targeting the peptidoglycan layer to increase efficacy against Gram-positive and Gram-negative bacteria. We expect our M. pneumoniae-engineered strain to be able to treat biofilm-associated infections in the respiratory tract. A bacterium expressing antibacterial genes treats a lung infection associated with mechanical ventilators.},
  archive      = {J_NBT},
  author       = {Mazzolini, Rocco and Rodríguez-Arce, Irene and Fernández-Barat, Laia and Piñero-Lambea, Carlos and Garrido, Victoria and Rebollada-Merino, Agustín and Motos, Anna and Torres, Antoni and Grilló, Maria Jesús and Serrano, Luis and Lluch-Senar, Maria},
  doi          = {10.1038/s41587-022-01584-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1089-1098},
  shortjournal = {Nature Biotech.},
  title        = {Engineered live bacteria suppress pseudomonas aeruginosa infection in mouse lung and dissolve endotracheal-tube biofilms},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Solid-phase capture and profiling of open chromatin by
spatial ATAC. <em>NBT</em>, <em>41</em>(8), 1085–1088. (<a
href="https://doi.org/10.1038/s41587-022-01603-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for epigenomic profiling are limited in their ability to obtain genome-wide information with spatial resolution. We introduce spatial ATAC, a method that integrates transposase-accessible chromatin profiling in tissue sections with barcoded solid-phase capture to perform spatially resolved epigenomics. We show that spatial ATAC enables the discovery of the regulatory programs underlying spatial gene expression during mouse organogenesis, lineage differentiation and in human pathology. Chromatin accessibility profiles are spatially resolved in tissue sections.},
  archive      = {J_NBT},
  author       = {Llorens-Bobadilla, Enric and Zamboni, Margherita and Marklund, Maja and Bhalla, Nayanika and Chen, Xinsong and Hartman, Johan and Frisén, Jonas and Ståhl, Patrik L.},
  doi          = {10.1038/s41587-022-01603-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1085-1088},
  shortjournal = {Nature Biotech.},
  title        = {Solid-phase capture and profiling of open chromatin by spatial ATAC},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Programmable a-to-y base editing by fusing an adenine base
editor with an n-methylpurine DNA glycosylase. <em>NBT</em>,
<em>41</em>(8), 1080–1084. (<a
href="https://doi.org/10.1038/s41587-022-01595-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we developed an adenine transversion base editor, AYBE, for A-to-C and A-to-T transversion editing in mammalian cells by fusing an adenine base editor (ABE) with hypoxanthine excision protein N-methylpurine DNA glycosylase (MPG). We also engineered AYBE variants enabling targeted editing at genomic loci with higher transversion editing activity (up to 72\% for A-to-C or A-to-T editing). A base editor is engineered for A-to-T and A-to-C transversion editing.},
  archive      = {J_NBT},
  author       = {Tong, Huawei and Wang, Xuchen and Liu, Yuanhua and Liu, Nana and Li, Yun and Luo, Jiamin and Ma, Qian and Wu, Danni and Li, Jiyong and Xu, Chunlong and Yang, Hui},
  doi          = {10.1038/s41587-022-01595-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1080-1084},
  shortjournal = {Nature Biotech.},
  title        = {Programmable A-to-Y base editing by fusing an adenine base editor with an N-methylpurine DNA glycosylase},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Protein identification using a digestion-free nanopore
approach. <em>NBT</em>, <em>41</em>(8), 1078–1079. (<a
href="https://doi.org/10.1038/s41587-022-01599-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing of proteins is a technically difficult task that typically requires digestion into short peptides before detection and identification. We developed a digestion-free method to chemically unfold and ‘scan’ full-length proteins through a nanopore, producing electrical fingerprints unique to individual protein molecules that are useful in their identification.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01599-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1078-1079},
  shortjournal = {Nature Biotech.},
  title        = {Protein identification using a digestion-free nanopore approach},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell recording of cellular RNAs in bacteria.
<em>NBT</em>, <em>41</em>(8), 1076–1077. (<a
href="https://doi.org/10.1038/s41587-022-01625-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The collection of RNAs in a cell reflects its identity and behavior. Although probing these RNAs in the present is commonplace, peering into the past remains challenging. We developed TIGER (transcribed RNAs inferred by genetically encoded records) to record selected RNAs at the single-cell level, enabling us to connect the past with the present.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01625-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1076-1077},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell recording of cellular RNAs in bacteria},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Forum: Boyden and danzl. <em>NBT</em>, <em>41</em>(8),
1075. (<a href="https://doi.org/10.1038/s41587-023-01901-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01901-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1075},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Boyden and danzl},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Hallucinating functional protein sequences. <em>NBT</em>,
<em>41</em>(8), 1073–1074. (<a
href="https://doi.org/10.1038/s41587-022-01634-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Functional proteins with limited homology to natural proteins are designed using a large language model.},
  archive      = {J_NBT},
  author       = {Belanger, David and Colwell, Lucy J.},
  doi          = {10.1038/s41587-022-01634-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1073-1074},
  shortjournal = {Nature Biotech.},
  title        = {Hallucinating functional protein sequences},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023p). Spatial transcriptomics of the human heart. <em>NBT</em>,
<em>41</em>(8), 1072. (<a
href="https://doi.org/10.1038/s41587-023-01918-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01918-1},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1072},
  shortjournal = {Nature Biotech.},
  title        = {Spatial transcriptomics of the human heart},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Diffusion model expands RoseTTAFold’s power. <em>NBT</em>,
<em>41</em>(8), 1072. (<a
href="https://doi.org/10.1038/s41587-023-01919-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01919-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1072},
  shortjournal = {Nature Biotech.},
  title        = {Diffusion model expands RoseTTAFold’s power},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). Models of the early human embryo. <em>NBT</em>,
<em>41</em>(8), 1072. (<a
href="https://doi.org/10.1038/s41587-023-01920-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01920-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1072},
  shortjournal = {Nature Biotech.},
  title        = {Models of the early human embryo},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). RNA-based therapies. <em>NBT</em>, <em>41</em>(8), 1071. (<a
href="https://doi.org/10.1038/s41587-023-01885-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the composition, preparation and optimization of RNA-based therapies.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01885-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1071},
  shortjournal = {Nature Biotech.},
  title        = {RNA-based therapies},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Comparing efficiency of patent production between US
institutions using a hybrid NIH–USPTO dataset. <em>NBT</em>,
<em>41</em>(8), 1062–1070. (<a
href="https://doi.org/10.1038/s41587-023-01890-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A database that links patents to NIH awards enables evaluation of key milestones along the research translation pathway.},
  archive      = {J_NBT},
  author       = {DiSanto, Rocco M. and Van Dyke, Mark and Barker, Michael J. and Gourdie, Robert G.},
  doi          = {10.1038/s41587-023-01890-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1062-1070},
  shortjournal = {Nature Biotech.},
  title        = {Comparing efficiency of patent production between US institutions using a hybrid NIH–USPTO dataset},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). FAIR data policies can benefit biotech startups.
<em>NBT</em>, <em>41</em>(8), 1060–1061. (<a
href="https://doi.org/10.1038/s41587-023-01892-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hardy, Kate and Heyse, Stephan},
  doi          = {10.1038/s41587-023-01892-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1060-1061},
  shortjournal = {Nature Biotech.},
  title        = {FAIR data policies can benefit biotech startups},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Democratizing knowledge representation with BioCypher.
<em>NBT</em>, <em>41</em>(8), 1056–1059. (<a
href="https://doi.org/10.1038/s41587-023-01848-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Lobentanzer, Sebastian and Aloy, Patrick and Baumbach, Jan and Bohar, Balazs and Carey, Vincent J. and Charoentong, Pornpimol and Danhauser, Katharina and Doğan, Tunca and Dreo, Johann and Dunham, Ian and Farr, Elias and Fernandez-Torras, Adrià and Gyori, Benjamin M. and Hartung, Michael and Hoyt, Charles Tapley and Klein, Christoph and Korcsmaros, Tamas and Maier, Andreas and Mann, Matthias and Ochoa, David and Pareja-Lorente, Elena and Popp, Ferdinand and Preusse, Martin and Probul, Niklas and Schwikowski, Benno and Sen, Bünyamin and Strauss, Maximilian T. and Turei, Denes and Ulusoy, Erva and Waltemath, Dagmar and Wodke, Judith A. H. and Saez-Rodriguez, Julio},
  doi          = {10.1038/s41587-023-01848-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1056-1059},
  shortjournal = {Nature Biotech.},
  title        = {Democratizing knowledge representation with BioCypher},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Defining a longevity biotechnology company. <em>NBT</em>,
<em>41</em>(8), 1053–1055. (<a
href="https://doi.org/10.1038/s41587-023-01854-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Boekstein, Nicola and Barzilai, Nir and Bertram, André and Betts-LaCroix, Joe and Fortney, Kristen and Helliwell, Stephen B. and Hufford, Michael and Mannick, Joan and McLaughlin, Jerry and Mellon, Jim and Morgen, Eric and Regge, Nils and Robinton, Daisy A. and Sinclair, David A. and Young, Sergey and Starr, Risa and Zhavoronkov, Alex and Peyer, James},
  doi          = {10.1038/s41587-023-01854-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1053-1055},
  shortjournal = {Nature Biotech.},
  title        = {Defining a longevity biotechnology company},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(8), 1040. (<a
href="https://doi.org/10.1038/s41587-023-01904-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01904-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1040},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Gates, wellcome rekindle TB vaccine. <em>NBT</em>,
<em>41</em>(8), 1039. (<a
href="https://doi.org/10.1038/s41587-023-01914-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01914-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1039},
  shortjournal = {Nature Biotech.},
  title        = {Gates, wellcome rekindle TB vaccine},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CRISPR upgrades insect proteins for feed. <em>NBT</em>,
<em>41</em>(8), 1038–1040. (<a
href="https://doi.org/10.1038/s41587-023-01902-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new generation of companies is pursuing insect larvae as a protein source for animal feed, fertilizer, biofuels and even as ingredients for burgers and shakes. The insects promise to deliver cheaper and more sustainable alternatives to soy and fishmeal, with the added benefit that larvae can be raised on organic waste.},
  archive      = {J_NBT},
  author       = {Gruber, Karl and Melton, Lisa},
  doi          = {10.1038/s41587-023-01902-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1038-1040},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR upgrades insect proteins for feed},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). US approves “no kill” meat for sale. <em>NBT</em>,
<em>41</em>(8), 1038. (<a
href="https://doi.org/10.1038/s41587-023-01913-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01913-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1038},
  shortjournal = {Nature Biotech.},
  title        = {US approves ‘no kill’ meat for sale},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Fungal architecture is a thing, and it’s eye-catching.
<em>NBT</em>, <em>41</em>(8), 1037. (<a
href="https://doi.org/10.1038/s41587-023-01903-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-023-01903-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1037},
  shortjournal = {Nature Biotech.},
  title        = {Fungal architecture is a thing, and it’s eye-catching},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). First cell therapy for diabetes approved. <em>NBT</em>,
<em>41</em>(8), 1036. (<a
href="https://doi.org/10.1038/s41587-023-01912-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01912-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1036},
  shortjournal = {Nature Biotech.},
  title        = {First cell therapy for diabetes approved},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Innovators hit by ruling in dispute over cholesterol
antibody patent. <em>NBT</em>, <em>41</em>(8), 1035–1036. (<a
href="https://doi.org/10.1038/s41587-023-01894-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The US Supreme Court decision in the Amgen vs. Sanofi patent dispute over PCSK9 inhibitors puts a decisive end to broad patents claiming rights over all molecules directed at a given target.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-023-01894-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1035-1036},
  shortjournal = {Nature Biotech.},
  title        = {Innovators hit by ruling in dispute over cholesterol antibody patent},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Not-so-creepy crawlies. <em>NBT</em>, <em>41</em>(8), 1033.
(<a href="https://doi.org/10.1038/s41587-023-01921-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biotech is embracing insect protein as a meat alternative. Will the rest of the population follow?},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01921-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1033},
  shortjournal = {Nature Biotech.},
  title        = {Not-so-creepy crawlies},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Bat biotech takes flight. <em>NBT</em>, <em>41</em>(8),
1047–1052. (<a
href="https://doi.org/10.1038/s41587-023-01880-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Harnessing the unusual biology of bats, researchers aim to turn drug discovery upside-down.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-023-01880-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {1047-1052},
  shortjournal = {Nature Biotech.},
  title        = {Bat biotech takes flight},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). 2Q23 — merger premiums reveal valuation gulf. <em>NBT</em>,
<em>41</em>(8), 1044–1046. (<a
href="https://doi.org/10.1038/s41587-023-01889-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-01889-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {1044-1046},
  shortjournal = {Nature Biotech.},
  title        = {2Q23 — merger premiums reveal valuation gulf},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Drug pipeline 2Q23 — controversy and complexity.
<em>NBT</em>, <em>41</em>(8), 1041–1043. (<a
href="https://doi.org/10.1038/s41587-023-01886-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-01886-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {1041-1043},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 2Q23 — controversy and complexity},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). People. <em>NBT</em>, <em>41</em>(7), 1032. (<a
href="https://doi.org/10.1038/s41587-023-01859-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01859-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1032},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Five questions with sanja vicković. <em>NBT</em>,
<em>41</em>(7), 1031. (<a
href="https://doi.org/10.1038/s41587-023-01856-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A pioneer in spatially resolved transcriptomics and genomics methods recounts her journey from professional ice hockey player to running her own lab.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-023-01856-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1031},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with sanja vicković},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The need for need-finding in medical education.
<em>NBT</em>, <em>41</em>(7), 1028–1030. (<a
href="https://doi.org/10.1038/s41587-023-01855-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The paradigm of developing a novel technology and applying it to a clinically relevant problem may be dominant in academic medicine, but more emphasis is needed in identifying important clinical problems before looking for solutions.},
  archive      = {J_NBT},
  author       = {Maddineni, Sainiteesh and Kumar, Wasan},
  doi          = {10.1038/s41587-023-01855-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1028-1030},
  shortjournal = {Nature Biotech.},
  title        = {The need for need-finding in medical education},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Author correction: A spatial genome aligner for resolving
chromatin architectures from multiplexed DNA FISH. <em>NBT</em>,
<em>41</em>(7), 1027. (<a
href="https://doi.org/10.1038/s41587-023-01673-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Jia, Bojing Blair and Jussila, Adam and Kern, Colin and Zhu, Quan and Ren, Bing},
  doi          = {10.1038/s41587-023-01673-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1027},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: A spatial genome aligner for resolving chromatin architectures from multiplexed DNA FISH},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Discovery of drug–omics associations in
type 2 diabetes with generative deep-learning models. <em>NBT</em>,
<em>41</em>(7), 1026. (<a
href="https://doi.org/10.1038/s41587-023-01805-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Allesøe, Rosa Lundbye and Lundgaard, Agnete Troen and Hernández Medina, Ricardo and Aguayo-Orozco, Alejandro and Johansen, Joachim and Nissen, Jakob Nybo and Brorsson, Caroline and Mazzoni, Gianluca and Niu, Lili and Biel, Jorge Hernansanz and Leal Rodríguez, Cristina and Brasas, Valentas and Webel, Henry and Benros, Michael Eriksen and Pedersen, Anders Gorm and Chmura, Piotr Jaroslaw and Jacobsen, Ulrik Plesner and Mari, Andrea and Koivula, Robert and Mahajan, Anubha and Vinuela, Ana and Tajes, Juan Fernandez and Sharma, Sapna and Haid, Mark and Hong, Mun-Gwan and Musholt, Petra B. and De Masi, Federico and Vogt, Josef and Pedersen, Helle Krogh and Gudmundsdottir, Valborg and Jones, Angus and Kennedy, Gwen and Bell, Jimmy and Thomas, E. Louise and Frost, Gary and Thomsen, Henrik and Hansen, Elizaveta and Hansen, Tue Haldor and Vestergaard, Henrik and Muilwijk, Mirthe and Blom, Marieke T. and ‘t Hart, Leen M. and Pattou, Francois and Raverdy, Violeta and Brage, Soren and Kokkola, Tarja and Heggie, Alison and McEvoy, Donna and Mourby, Miranda and Kaye, Jane and Hattersley, Andrew and McDonald, Timothy and Ridderstråle, Martin and Walker, Mark and Forgie, Ian and Giordano, Giuseppe N. and Pavo, Imre and Ruetten, Hartmut and Pedersen, Oluf and Hansen, Torben and Dermitzakis, Emmanouil and Franks, Paul W. and Schwenk, Jochen M. and Adamski, Jerzy and McCarthy, Mark I. and Pearson, Ewan and Banasik, Karina and Rasmussen, Simon and Brunak, Søren},
  doi          = {10.1038/s41587-023-01805-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1026},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Discovery of drug–omics associations in type 2 diabetes with generative deep-learning models},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dynamic, adaptive sampling during nanopore sequencing using
bayesian experimental design. <em>NBT</em>, <em>41</em>(7), 1018–1025.
(<a href="https://doi.org/10.1038/s41587-022-01580-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanopore sequencers can select which DNA molecules to sequence, rejecting a molecule after analysis of a small initial part. Currently, selection is based on predetermined regions of interest that remain constant throughout an experiment. Sequencing efforts, thus, cannot be re-focused on molecules likely contributing most to experimental success. Here we present BOSS-RUNS, an algorithmic framework and software to generate dynamically updated decision strategies. We quantify uncertainty at each genome position with real-time updates from data already observed. For each DNA fragment, we decide whether the expected decrease in uncertainty that it would provide warrants fully sequencing it, thus optimizing information gain. BOSS-RUNS mitigates coverage bias between and within members of a microbial community, leading to improved variant calling; for example, low-coverage sites of a species at 1\% abundance were reduced by 87.5\%, with 12.5\% more single-nucleotide polymorphisms detected. Such data-driven updates to molecule selection are applicable to many sequencing scenarios, such as enriching for regions with increased divergence or low coverage, reducing time-to-answer. Nanopore selective sequencing with real-time decision updates mitigates coverage bias.},
  archive      = {J_NBT},
  author       = {Weilguny, Lukas and De Maio, Nicola and Munro, Rory and Manser, Charlotte and Birney, Ewan and Loose, Matthew and Goldman, Nick},
  doi          = {10.1038/s41587-022-01580-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1018-1025},
  shortjournal = {Nature Biotech.},
  title        = {Dynamic, adaptive sampling during nanopore sequencing using bayesian experimental design},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A spatial genome aligner for resolving chromatin
architectures from multiplexed DNA FISH. <em>NBT</em>, <em>41</em>(7),
1004–1017. (<a
href="https://doi.org/10.1038/s41587-022-01568-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed fluorescence in situ hybridization (FISH) is a widely used approach for analyzing three-dimensional genome organization, but it is challenging to derive chromosomal conformations from noisy fluorescence signals, and tracing chromatin is not straightforward. Here we report a spatial genome aligner that parses true chromatin signal from noise by aligning signals to a DNA polymer model. Using genomic distances separating imaged loci, our aligner estimates spatial distances expected to separate loci on a polymer in three-dimensional space. Our aligner then evaluates the physical probability observed signals belonging to these loci are connected, thereby tracing chromatin structures. We demonstrate that this spatial genome aligner can efficiently model chromosome architectures from DNA FISH data across multiple scales and be used to predict chromosome ploidies de novo in interphase cells. Reprocessing of previous whole-genome chromosome tracing data with this method indicates the spatial aggregation of sister chromatids in S/G2 phase cells in asynchronous mouse embryonic stem cells and provides evidence for extranumerary chromosomes that remain tightly paired in postmitotic neurons of the adult mouse cortex. The architectures of chromatin fibers are resolved using a spatial genome aligner.},
  archive      = {J_NBT},
  author       = {Jia, Bojing Blair and Jussila, Adam and Kern, Colin and Zhu, Quan and Ren, Bing},
  doi          = {10.1038/s41587-022-01568-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1004-1017},
  shortjournal = {Nature Biotech.},
  title        = {A spatial genome aligner for resolving chromatin architectures from multiplexed DNA FISH},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Transcriptome-wide profiling and quantification of
n6-methyladenosine by enzyme-assisted adenosine deamination.
<em>NBT</em>, <em>41</em>(7), 993–1003. (<a
href="https://doi.org/10.1038/s41587-022-01587-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {N6-methyladenosine (m6A), the most abundant internal messenger RNA modification in higher eukaryotes, serves myriad roles in regulating cellular processes. Functional dissection of m6A is, however, hampered in part by the lack of high-resolution and quantitative detection methods. Here we present evolved TadA-assisted N6-methyladenosine sequencing (eTAM-seq), an enzyme-assisted sequencing technology that detects and quantifies m6A by global adenosine deamination. With eTAM-seq, we analyze the transcriptome-wide distribution of m6A in HeLa and mouse embryonic stem cells. The enzymatic deamination route employed by eTAM-seq preserves RNA integrity, facilitating m6A detection from limited input samples. In addition to transcriptome-wide m6A profiling, we demonstrate site-specific, deep-sequencing-free m6A quantification with as few as ten cells, an input demand orders of magnitude lower than existing quantitative profiling methods. We envision that eTAM-seq will enable researchers to not only survey the m6A landscape at unprecedented resolution, but also detect m6A at user-specified loci with a simple workflow. N6-methyladenosine is mapped at single-base resolution in as few as ten cells.},
  archive      = {J_NBT},
  author       = {Xiao, Yu-Lan and Liu, Shun and Ge, Ruiqi and Wu, Yuan and He, Chuan and Chen, Mengjie and Tang, Weixin},
  doi          = {10.1038/s41587-022-01587-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {993-1003},
  shortjournal = {Nature Biotech.},
  title        = {Transcriptome-wide profiling and quantification of n6-methyladenosine by enzyme-assisted adenosine deamination},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-throughput, targeted MHC class i immunopeptidomics
using a functional genetics screening platform. <em>NBT</em>,
<em>41</em>(7), 980–992. (<a
href="https://doi.org/10.1038/s41587-022-01566-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of CD8+ T cell epitopes is critical for the development of immunotherapeutics. Existing methods for major histocompatibility complex class I (MHC class I) ligand discovery are time intensive, specialized and unable to interrogate specific proteins on a large scale. Here, we present EpiScan, which uses surface MHC class I levels as a readout for whether a genetically encoded peptide is an MHC class I ligand. Predetermined starting pools composed of &amp;gt;100,000 peptides can be designed using oligonucleotide synthesis, permitting large-scale MHC class I screening. We exploit this programmability of EpiScan to uncover an unappreciated role for cysteine that increases the number of predicted ligands by 9–21\%, reveal affinity hierarchies by analysis of biased anchor peptide libraries and screen viral proteomes for MHC class I ligands. Using these data, we generate and iteratively refine peptide binding predictions to create EpiScan Predictor. EpiScan Predictor performs comparably to other state-of-the-art MHC class I peptide binding prediction algorithms without suffering from underrepresentation of cysteine-containing peptides. Thus, targeted immunopeptidomics using EpiScan will accelerate CD8+ T cell epitope discovery toward the goal of individual-specific immunotherapeutics. Peptides that bind MHC class I are readily identified with a programmable, high-throughput method.},
  archive      = {J_NBT},
  author       = {Bruno, Peter M. and Timms, Richard T. and Abdelfattah, Nouran S. and Leng, Yumei and Lelis, Felipe J. N. and Wesemann, Duane R. and Yu, Xu G. and Elledge, Stephen J.},
  doi          = {10.1038/s41587-022-01566-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {980-992},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput, targeted MHC class i immunopeptidomics using a functional genetics screening platform},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precise cut-and-paste DNA insertion using engineered type
v-k CRISPR-associated transposases. <em>NBT</em>, <em>41</em>(7),
968–979. (<a href="https://doi.org/10.1038/s41587-022-01574-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR-associated transposases (CASTs) enable recombination-independent, multi-kilobase DNA insertions at RNA-programmed genomic locations. However, the utility of type V-K CASTs is hindered by high off-target integration and a transposition mechanism that results in a mixture of desired simple cargo insertions and undesired plasmid cointegrate products. Here we overcome both limitations by engineering new CASTs with improved integration product purity and genome-wide specificity. To do so, we engineered a nicking homing endonuclease fusion to TnsB (named HELIX) to restore the 5′ nicking capability needed for cargo excision on the DNA donor. HELIX enables cut-and-paste DNA insertion with up to 99.4\% simple insertion product purity, while retaining robust integration efficiencies on genomic targets. HELIX has substantially higher on-target specificity than canonical CASTs, and we identify several novel factors that further regulate targeted and genome-wide integration. Finally, we extend HELIX to other type V-K orthologs and demonstrate the feasibility of HELIX-mediated integration in human cell contexts. Engineered CRISPR-associated transposases improve the purity and specificity of DNA insertions.},
  archive      = {J_NBT},
  author       = {Tou, Connor J. and Orr, Benno and Kleinstiver, Benjamin P.},
  doi          = {10.1038/s41587-022-01574-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {968-979},
  shortjournal = {Nature Biotech.},
  title        = {Precise cut-and-paste DNA insertion using engineered type V-K CRISPR-associated transposases},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Heritable transgene-free genome editing in plants by
grafting of wild-type shoots to transgenic donor rootstocks.
<em>NBT</em>, <em>41</em>(7), 958–967. (<a
href="https://doi.org/10.1038/s41587-022-01585-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generation of stable gene-edited plant lines using clustered regularly interspaced short palindromic repeats (CRISPR)–CRISPR-associated protein 9 (Cas9) requires a lengthy process of outcrossing to eliminate CRISPR–Cas9-associated sequences and produce transgene-free lines. We have addressed this issue by designing fusions of Cas9 and guide RNA transcripts to tRNA-like sequence motifs that move RNAs from transgenic rootstocks to grafted wild-type shoots (scions) and achieve heritable gene editing, as demonstrated in wild-type Arabidopsis thaliana and Brassica rapa. The graft-mobile gene editing system enables the production of transgene-free offspring in one generation without the need for transgene elimination, culture recovery and selection, or use of viral editing vectors. We anticipate that using graft-mobile editing systems for transgene-free plant production may be applied to a wide range of breeding programs and crop plants. Gene-edited plants free of CRISPR-associated sequences are generated by grafting.},
  archive      = {J_NBT},
  author       = {Yang, Lei and Machin, Frank and Wang, Shuangfeng and Saplaoura, Eleftheria and Kragler, Friedrich},
  doi          = {10.1038/s41587-022-01585-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {958-967},
  shortjournal = {Nature Biotech.},
  title        = {Heritable transgene-free genome editing in plants by grafting of wild-type shoots to transgenic donor rootstocks},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A genetically encoded sensor measures temporal oxytocin
release from different neuronal compartments. <em>NBT</em>,
<em>41</em>(7), 944–957. (<a
href="https://doi.org/10.1038/s41587-022-01561-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oxytocin (OT), a peptide hormone and neuromodulator, is involved in diverse physiological and pathophysiological processes in the central nervous system and the periphery. However, the regulation and functional sequences of spatial OT release in the brain remain poorly understood. We describe a genetically encoded G-protein-coupled receptor activation-based (GRAB) OT sensor called GRABOT1.0. In contrast to previous methods, GRABOT1.0 enables imaging of OT release ex vivo and in vivo with suitable sensitivity, specificity and spatiotemporal resolution. Using this sensor, we visualize stimulation-induced OT release from specific neuronal compartments in mouse brain slices and discover that N-type calcium channels predominantly mediate axonal OT release, whereas L-type calcium channels mediate somatodendritic OT release. We identify differences in the fusion machinery of OT release for axon terminals versus somata and dendrites. Finally, we measure OT dynamics in various brain regions in mice during male courtship behavior. Thus, GRABOT1.0 provides insights into the role of compartmental OT release in physiological and behavioral functions. A sensor for oxytocin reveals mechanisms of oxytocin action with high spatiotemporal precision.},
  archive      = {J_NBT},
  author       = {Qian, Tongrui and Wang, Huan and Wang, Peng and Geng, Lan and Mei, Long and Osakada, Takuya and Wang, Lei and Tang, Yan and Kania, Alan and Grinevich, Valery and Stoop, Ron and Lin, Dayu and Luo, Minmin and Li, Yulong},
  doi          = {10.1038/s41587-022-01561-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {944-957},
  shortjournal = {Nature Biotech.},
  title        = {A genetically encoded sensor measures temporal oxytocin release from different neuronal compartments},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Facile repurposing of peptide–MHC-restricted antibodies for
cancer immunotherapy. <em>NBT</em>, <em>41</em>(7), 932–943. (<a
href="https://doi.org/10.1038/s41587-022-01567-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monoclonal antibodies (Abs) that recognize major histocompatability complex (MHC)-presented tumor antigens in a manner similar to T cell receptors (TCRs) have great potential as cancer immunotherapeutics. However, isolation of ‘TCR-mimic’ (TCRm) Abs is laborious because Abs have not evolved the structurally nuanced peptide–MHC restriction of αβ-TCRs. Here, we present a strategy for rapid isolation of highly peptide-specific and ‘MHC-restricted’ Abs by re-engineering preselected Abs that engage peptide–MHC in a manner structurally similar to that of conventional αβ-TCRs. We created structure-based libraries focused on the peptide-interacting residues of TCRm Ab complementarity-determining region (CDR) loops, and rapidly generated MHC-restricted Abs to both mouse and human tumor antigens that specifically killed target cells when formatted as IgG, bispecific T cell engager (BiTE) and chimeric antigen receptor-T (CAR-T). Crystallographic analysis of one selected pMHC-restricted Ab revealed highly peptide-specific recognition, validating the engineering strategy. This approach can yield tumor antigen-specific antibodies in several weeks, potentially enabling rapid clinical translation. A method to identify antibodies against peptide–MHC complexes could enable patient-specific cancer immunotherapeutics.},
  archive      = {J_NBT},
  author       = {Yang, Xinbo and Nishimiya, Daisuke and Löchte, Sara and Jude, Kevin M. and Borowska, Marta and Savvides, Christina S. and Dougan, Michael and Su, Leon and Zhao, Xiang and Piehler, Jacob and Garcia, K. Christopher},
  doi          = {10.1038/s41587-022-01567-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {932-943},
  shortjournal = {Nature Biotech.},
  title        = {Facile repurposing of peptide–MHC-restricted antibodies for cancer immunotherapy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genomically mined acoustic reporter genes for real-time in
vivo monitoring of tumors and tumor-homing bacteria. <em>NBT</em>,
<em>41</em>(7), 919–931. (<a
href="https://doi.org/10.1038/s41587-022-01581-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultrasound allows imaging at a much greater depth than optical methods, but existing genetically encoded acoustic reporters for in vivo cellular imaging have been limited by poor sensitivity, specificity and in vivo expression. Here we describe two acoustic reporter genes (ARGs)—one for use in bacteria and one for use in mammalian cells—identified through a phylogenetic screen of candidate gas vesicle gene clusters from diverse bacteria and archaea that provide stronger ultrasound contrast, produce non-linear signals distinguishable from background tissue and have stable long-term expression. Compared to their first-generation counterparts, these improved bacterial and mammalian ARGs produce 9-fold and 38-fold stronger non-linear contrast, respectively. Using these new ARGs, we non-invasively imaged in situ tumor colonization and gene expression in tumor-homing therapeutic bacteria, tracked the progression of tumor gene expression and growth in a mouse model of breast cancer, and performed gene-expression-guided needle biopsies of a genetically mosaic tumor, demonstrating non-invasive access to dynamic biological processes at centimeter depth. Gene expression is imaged in deep tissues in mice using ultrasound.},
  archive      = {J_NBT},
  author       = {Hurt, Robert C. and Buss, Marjorie T. and Duan, Mengtong and Wong, Katie and You, Mei Yi and Sawyer, Daniel P. and Swift, Margaret B. and Dutka, Przemysław and Barturen-Larrea, Pierina and Mittelstein, David R. and Jin, Zhiyang and Abedi, Mohamad H. and Farhadi, Arash and Deshpande, Ramya and Shapiro, Mikhail G.},
  doi          = {10.1038/s41587-022-01581-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {919-931},
  shortjournal = {Nature Biotech.},
  title        = {Genomically mined acoustic reporter genes for real-time in vivo monitoring of tumors and tumor-homing bacteria},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accurate isoform discovery with IsoQuant using long reads.
<em>NBT</em>, <em>41</em>(7), 915–918. (<a
href="https://doi.org/10.1038/s41587-022-01565-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Annotating newly sequenced genomes and determining alternative isoforms from long-read RNA data are complex and incompletely solved problems. Here we present IsoQuant—a computational tool using intron graphs that accurately reconstructs transcripts both with and without reference genome annotation. For novel transcript discovery, IsoQuant reduces the false-positive rate fivefold and 2.5-fold for Oxford Nanopore reference-based or reference-free mode, respectively. IsoQuant also improves performance for Pacific Biosciences data. IsoQuant predicts novel isoforms from long-read RNA sequencing.},
  archive      = {J_NBT},
  author       = {Prjibelski, Andrey D. and Mikheenko, Alla and Joglekar, Anoushka and Smetanin, Alexander and Jarroux, Julien and Lapidus, Alla L. and Tilgner, Hagen U.},
  doi          = {10.1038/s41587-022-01565-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {915-918},
  shortjournal = {Nature Biotech.},
  title        = {Accurate isoform discovery with IsoQuant using long reads},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Real-time monitoring of tumor-homing bacteria and tumor
cells in vivo using ultrasound. <em>NBT</em>, <em>41</em>(7), 913–914.
(<a href="https://doi.org/10.1038/s41587-022-01590-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By screening natural prokaryotic gas vesicle gene clusters, we found and engineered new acoustic reporter genes (ARGs) that give bacteria and mammalian cells brighter ultrasound contrast for real-time noninvasive imaging. Expressing these ARGs in engineered cells enabled us to image tumor-homing bacteria and perform genetically guided tumor biopsies in vivo.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01590-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {913-914},
  shortjournal = {Nature Biotech.},
  title        = {Real-time monitoring of tumor-homing bacteria and tumor cells in vivo using ultrasound},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Staple crops equipped for alkaline soils. <em>NBT</em>,
<em>41</em>(7), 911–912. (<a
href="https://doi.org/10.1038/s41587-023-01832-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene editing boosts the yields of sorghum, rice and millet grown in alkaline soils.},
  archive      = {J_NBT},
  author       = {Byrt, Caitlin S. and Millar, A. Harvey and Munns, Rana},
  doi          = {10.1038/s41587-023-01832-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {911-912},
  shortjournal = {Nature Biotech.},
  title        = {Staple crops equipped for alkaline soils},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CRISPR-edited plants by grafting. <em>NBT</em>,
<em>41</em>(7), 909–910. (<a
href="https://doi.org/10.1038/s41587-022-01516-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Edited, transgene-free plants are produced without tissue culture by RNA migration from rootstocks to grafts.},
  archive      = {J_NBT},
  author       = {Hu, Jiacheng and Gao, Caixia},
  doi          = {10.1038/s41587-022-01516-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {909-910},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR-edited plants by grafting},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Transplanting soil microbial communities improves climate
tolerance in trees. <em>NBT</em>, <em>41</em>(7), 908. (<a
href="https://doi.org/10.1038/s41587-023-01874-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chuong, Julie N.},
  doi          = {10.1038/s41587-023-01874-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {908},
  shortjournal = {Nature Biotech.},
  title        = {Transplanting soil microbial communities improves climate tolerance in trees},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Fluorescence microscopy resolution increases to near-atomic
level. <em>NBT</em>, <em>41</em>(7), 908. (<a
href="https://doi.org/10.1038/s41587-023-01875-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chuong, Julie N.},
  doi          = {10.1038/s41587-023-01875-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {908},
  shortjournal = {Nature Biotech.},
  title        = {Fluorescence microscopy resolution increases to near-atomic level},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Multiplex-GAM detects complex chromatin interactions.
<em>NBT</em>, <em>41</em>(7), 908. (<a
href="https://doi.org/10.1038/s41587-023-01876-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chuong, Julie N.},
  doi          = {10.1038/s41587-023-01876-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {908},
  shortjournal = {Nature Biotech.},
  title        = {Multiplex-GAM detects complex chromatin interactions},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine learning in drug discovery. <em>NBT</em>,
<em>41</em>(7), 907. (<a
href="https://doi.org/10.1038/s41587-023-01851-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to machine learning in drug discovery and screening.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01851-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {907},
  shortjournal = {Nature Biotech.},
  title        = {Machine learning in drug discovery},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The prevalence of drug patent term extensions in the united
states, 2000–2018. <em>NBT</em>, <em>41</em>(7), 903–906. (<a
href="https://doi.org/10.1038/s41587-023-01847-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug patents that are given patent term extension are important bellwether indicators for generic drug market entry.},
  archive      = {J_NBT},
  author       = {Van de Wiele, Victor L. and Kesselheim, Aaron S. and Nagar, Sarosh and Tu, S. Sean},
  doi          = {10.1038/s41587-023-01847-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {903-906},
  shortjournal = {Nature Biotech.},
  title        = {The prevalence of drug patent term extensions in the united states, 2000–2018},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Guidelines for public database submission of uncultivated
virus genome sequences for taxonomic classification. <em>NBT</em>,
<em>41</em>(7), 898–902. (<a
href="https://doi.org/10.1038/s41587-023-01844-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Adriaenssens, Evelien M. and Roux, Simon and Brister, J. Rodney and Karsch-Mizrachi, Ilene and Kuhn, Jens H. and Varsani, Arvind and Yigang, Tong and Reyes, Alejandro and Lood, Cédric and Lefkowitz, Elliot J. and Sullivan, Matthew B. and Edwards, Robert A. and Simmonds, Peter and Rubino, Luisa and Sabanadzovic, Sead and Krupovic, Mart and Dutilh, Bas E.},
  doi          = {10.1038/s41587-023-01844-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {898-902},
  shortjournal = {Nature Biotech.},
  title        = {Guidelines for public database submission of uncultivated virus genome sequences for taxonomic classification},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A call for an “asilomar” for cultivated meat and seafood.
<em>NBT</em>, <em>41</em>(7), 895–897. (<a
href="https://doi.org/10.1038/s41587-023-01849-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bottini, Sveva and Fuoco, Claudia and Schiavo, Nike and Bertero, Alessandro and Biressi, Stefano and Conti, Luciano and Gargioli, Cesare},
  doi          = {10.1038/s41587-023-01849-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {895-897},
  shortjournal = {Nature Biotech.},
  title        = {A call for an ‘Asilomar’ for cultivated meat and seafood},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Plant molecular farming in the wake of the closure of
medicago inc. <em>NBT</em>, <em>41</em>(7), 893–894. (<a
href="https://doi.org/10.1038/s41587-023-01812-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Benvenuto, Eugenio and Broer, Inge and D’Aoust, Marc-André and Hitzeroth, Inga and Hundleby, Penny and Menassa, Rima and Oksman-Caldentey, Kirsi-Marja and Peyret, Hadrien and Salgueiro, Sancha and Saxena, Pooja and Stander, Jennifer and Warzecha, Heribert and Ma, Julian},
  doi          = {10.1038/s41587-023-01812-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {893-894},
  shortjournal = {Nature Biotech.},
  title        = {Plant molecular farming in the wake of the closure of medicago inc},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biotech patenting — 2022. <em>NBT</em>, <em>41</em>(7),
891–892. (<a href="https://doi.org/10.1038/s41587-023-01853-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Paisner, Kathryn and DeFrancesco, Laura},
  doi          = {10.1038/s41587-023-01853-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {891-892},
  shortjournal = {Nature Biotech.},
  title        = {Biotech patenting — 2022},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(7), 890. (<a
href="https://doi.org/10.1038/s41587-023-01861-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01861-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {890},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). So sweet, no spike. <em>NBT</em>, <em>41</em>(7), 889. (<a
href="https://doi.org/10.1038/s41587-023-01865-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-023-01865-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {889},
  shortjournal = {Nature Biotech.},
  title        = {So sweet, no spike},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Eight diseases for gene therapy accelerator. <em>NBT</em>,
<em>41</em>(7), 888. (<a
href="https://doi.org/10.1038/s41587-023-01871-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01871-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {888},
  shortjournal = {Nature Biotech.},
  title        = {Eight diseases for gene therapy accelerator},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Verve to edit gene for coronary risk lipid lp(a).
<em>NBT</em>, <em>41</em>(7), 885. (<a
href="https://doi.org/10.1038/s41587-023-01870-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01870-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {885},
  shortjournal = {Nature Biotech.},
  title        = {Verve to edit gene for coronary risk lipid lp(a)},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Targeted glioma therapy makes strides. <em>NBT</em>,
<em>41</em>(7), 884. (<a
href="https://doi.org/10.1038/s41587-023-01869-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01869-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {884},
  shortjournal = {Nature Biotech.},
  title        = {Targeted glioma therapy makes strides},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). “Clicked” drugs: Researchers prove the remarkable chemistry
in humans. <em>NBT</em>, <em>41</em>(7), 883–885. (<a
href="https://doi.org/10.1038/s41587-023-01860-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioorthogonal click chemistry is being used in patients to help target cancer medicines and diagnostic imaging agents},
  archive      = {J_NBT},
  author       = {Peplow, Mark},
  doi          = {10.1038/s41587-023-01860-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {883-885},
  shortjournal = {Nature Biotech.},
  title        = {‘Clicked’ drugs: Researchers prove the remarkable chemistry in humans},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Prize-winning spinoffs. <em>NBT</em>, <em>41</em>(7), 881.
(<a href="https://doi.org/10.1038/s41587-023-01882-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Early-stage researchers win Spinoff Prize with a clinical bioadhesive.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01882-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {881},
  shortjournal = {Nature Biotech.},
  title        = {Prize-winning spinoffs},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Unprecedented blood biomarker enables ALS drug approval.
<em>NBT</em>, <em>41</em>(7), 886–888. (<a
href="https://doi.org/10.1038/s41587-023-01862-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biogen’s drug Qalsody is the first drug to gain approval for a genetically defined form of ALS, but there is more to come as developers zoom in on a wide range of disease targets.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-023-01862-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {7},
  pages        = {886-888},
  shortjournal = {Nature Biotech.},
  title        = {Unprecedented blood biomarker enables ALS drug approval},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). People. <em>NBT</em>, <em>41</em>(6), 880. (<a
href="https://doi.org/10.1038/s41587-023-01823-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01823-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {880},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). What is hindering the drug development industry from
achieving diversity, equity and inclusion in its workforce?
<em>NBT</em>, <em>41</em>(6), 878–879. (<a
href="https://doi.org/10.1038/s41587-023-01817-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Establishing accountability by implementing measurable metrics to track and monitor DEI dynamics can ensure that it becomes central to an organization’s strategy.},
  archive      = {J_NBT},
  author       = {Kim, Jennifer Y. and Botto, Emily},
  doi          = {10.1038/s41587-023-01817-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {878-879},
  shortjournal = {Nature Biotech.},
  title        = {What is hindering the drug development industry from achieving diversity, equity and inclusion in its workforce?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Control-independent mosaic single nucleotide variant
detection with DeepMosaic. <em>NBT</em>, <em>41</em>(6), 870–877. (<a
href="https://doi.org/10.1038/s41587-022-01559-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mosaic variants (MVs) reflect mutagenic processes during embryonic development and environmental exposure, accumulate with aging and underlie diseases such as cancer and autism. The detection of noncancer MVs has been computationally challenging due to the sparse representation of nonclonally expanded MVs. Here we present DeepMosaic, combining an image-based visualization module for single nucleotide MVs and a convolutional neural network-based classification module for control-independent MV detection. DeepMosaic was trained on 180,000 simulated or experimentally assessed MVs, and was benchmarked on 619,740 simulated MVs and 530 independent biologically tested MVs from 16 genomes and 181 exomes. DeepMosaic achieved higher accuracy compared with existing methods on biological data, with a sensitivity of 0.78, specificity of 0.83 and positive predictive value of 0.96 on noncancer whole-genome sequencing data, as well as doubling the validation rate over previous best-practice methods on noncancer whole-exome sequencing data (0.43 versus 0.18). DeepMosaic represents an accurate MV classifier for noncancer samples that can be implemented as an alternative or complement to existing methods. A deep learning model detects low frequency mosaic variants with improved accuracy.},
  archive      = {J_NBT},
  author       = {Yang, Xiaoxu and Xu, Xin and Breuss, Martin W. and Antaki, Danny and Ball, Laurel L. and Chung, Changuk and Shen, Jiawei and Li, Chen and George, Renee D. and Wang, Yifan and Bae, Taejeong and Cheng, Yuhe and Abyzov, Alexej and Wei, Liping and Alexandrov, Ludmil B. and Sebat, Jonathan L. and Gleeson, Joseph G.},
  doi          = {10.1038/s41587-022-01559-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {870-877},
  shortjournal = {Nature Biotech.},
  title        = {Control-independent mosaic single nucleotide variant detection with DeepMosaic},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Magnify is a universal molecular anchoring strategy for
expansion microscopy. <em>NBT</em>, <em>41</em>(6), 858–869. (<a
href="https://doi.org/10.1038/s41587-022-01546-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expansion microscopy enables nanoimaging with conventional microscopes by physically and isotropically magnifying preserved biological specimens embedded in a crosslinked water-swellable hydrogel. Current expansion microscopy protocols require prior treatment with reactive anchoring chemicals to link specific labels and biomolecule classes to the gel. We describe a strategy called Magnify, which uses a mechanically sturdy gel that retains nucleic acids, proteins and lipids without the need for a separate anchoring step. Magnify expands biological specimens up to 11 times and facilitates imaging of cells and tissues with effectively around 25-nm resolution using a diffraction-limited objective lens of about 280 nm on conventional optical microscopes or with around 15 nm effective resolution if combined with super-resolution optical fluctuation imaging. We demonstrate Magnify on a broad range of biological specimens, providing insight into nanoscopic subcellular structures, including synaptic proteins from mouse brain, podocyte foot processes in formalin-fixed paraffin-embedded human kidney and defects in cilia and basal bodies in drug-treated human lung organoids. Expansion microscopy with gel-anchored molecules enables simultaneous retention of nucleic acids, proteins and lipids in diverse tissue types.},
  archive      = {J_NBT},
  author       = {Klimas, Aleksandra and Gallagher, Brendan R. and Wijesekara, Piyumi and Fekir, Sinda and DiBernardo, Emma F. and Cheng, Zhangyu and Stolz, Donna B. and Cambi, Franca and Watkins, Simon C. and Brody, Steven L. and Horani, Amjad and Barth, Alison L. and Moore, Christopher I. and Ren, Xi and Zhao, Yongxin},
  doi          = {10.1038/s41587-022-01546-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {858-869},
  shortjournal = {Nature Biotech.},
  title        = {Magnify is a universal molecular anchoring strategy for expansion microscopy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A proteome-wide atlas of drug mechanism of action.
<em>NBT</em>, <em>41</em>(6), 845–857. (<a
href="https://doi.org/10.1038/s41587-022-01539-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Defining the cellular response to pharmacological agents is critical for understanding the mechanism of action of small molecule perturbagens. Here, we developed a 96-well-plate-based high-throughput screening infrastructure for quantitative proteomics and profiled 875 compounds in a human cancer cell line with near-comprehensive proteome coverage. Examining the 24-h proteome changes revealed ligand-induced changes in protein expression and uncovered rules by which compounds regulate their protein targets while identifying putative dihydrofolate reductase and tankyrase inhibitors. We used protein–protein and compound–compound correlation networks to uncover mechanisms of action for several compounds, including the adrenergic receptor antagonist JP1302, which we show disrupts the FACT complex and degrades histone H1. By profiling many compounds with overlapping targets covering a broad chemical space, we linked compound structure to mechanisms of action and highlighted off-target polypharmacology for molecules within the library. A proteome-wide atlas of the effect of 875 compounds reveals mechanisms of action and off-target effects.},
  archive      = {J_NBT},
  author       = {Mitchell, Dylan C. and Kuljanin, Miljan and Li, Jiaming and Van Vranken, Jonathan G. and Bulloch, Nathan and Schweppe, Devin K. and Huttlin, Edward L. and Gygi, Steven P.},
  doi          = {10.1038/s41587-022-01539-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {845-857},
  shortjournal = {Nature Biotech.},
  title        = {A proteome-wide atlas of drug mechanism of action},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Functional analysis of structural variants in single cells
using strand-seq. <em>NBT</em>, <em>41</em>(6), 832–844. (<a
href="https://doi.org/10.1038/s41587-022-01551-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic structural variants (SVs) are widespread in cancer, but their impact on disease evolution is understudied due to a lack of methods to directly characterize their functional consequences. We present a computational method, scNOVA, which uses Strand-seq to perform haplotype-aware integration of SV discovery and molecular phenotyping in single cells by using nucleosome occupancy to infer gene expression as a readout. Application to leukemias and cell lines identifies local effects of copy-balanced rearrangements on gene deregulation, and consequences of SVs on aberrant signaling pathways in subclones. We discovered distinct SV subclones with dysregulated Wnt signaling in a chronic lymphocytic leukemia patient. We further uncovered the consequences of subclonal chromothripsis in T cell acute lymphoblastic leukemia, which revealed c-Myb activation, enrichment of a primitive cell state and informed successful targeting of the subclone in cell culture, using a Notch inhibitor. By directly linking SVs to their functional effects, scNOVA enables systematic single-cell multiomic studies of structural variation in heterogeneous cell populations. Linking structural variants to gene expression information identifies aberrant signaling pathways.},
  archive      = {J_NBT},
  author       = {Jeong, Hyobin and Grimes, Karen and Rauwolf, Kerstin K. and Bruch, Peter-Martin and Rausch, Tobias and Hasenfeld, Patrick and Benito, Eva and Roider, Tobias and Sabarinathan, Radhakrishnan and Porubsky, David and Herbst, Sophie A. and Erarslan-Uysal, Büşra and Jann, Johann-Christoph and Marschall, Tobias and Nowak, Daniel and Bourquin, Jean-Pierre and Kulozik, Andreas E. and Dietrich, Sascha and Bornhauser, Beat and Sanders, Ashley D. and Korbel, Jan O.},
  doi          = {10.1038/s41587-022-01551-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {832-844},
  shortjournal = {Nature Biotech.},
  title        = {Functional analysis of structural variants in single cells using strand-seq},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). In vivo development of immune tissue in human intestinal
organoids transplanted into humanized mice. <em>NBT</em>,
<em>41</em>(6), 824–831. (<a
href="https://doi.org/10.1038/s41587-022-01558-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human intestinal organoids (HIOs) derived from pluripotent stem cells provide a valuable model for investigating human intestinal organogenesis and physiology, but they lack the immune components required to fully recapitulate the complexity of human intestinal biology and diseases. To address this issue and to begin to decipher human intestinal–immune crosstalk during development, we generated HIOs containing immune cells by transplanting HIOs under the kidney capsule of mice with a humanized immune system. We found that human immune cells temporally migrate to the mucosa and form cellular aggregates that resemble human intestinal lymphoid follicles. Moreover, after microbial exposure, epithelial microfold cells are increased in number, leading to immune cell activation determined by the secretion of IgA antibodies in the HIO lumen. This in vivo HIO system with human immune cells provides a framework for future studies on infection- or allergen-driven intestinal diseases. Human intestinal organoids are endowed with immune cells by transplantation into humanized mice.},
  archive      = {J_NBT},
  author       = {Bouffi, Carine and Wikenheiser-Brokamp, Kathryn A. and Chaturvedi, Praneet and Sundaram, Nambirajan and Goddard, Gillian R. and Wunderlich, Mark and Brown, Nicole E. and Staab, Janet F. and Latanich, Rachel and Zachos, Nicholas C. and Holloway, Emily M. and Mahe, Maxime M. and Poling, Holly M. and Vales, Simon and Fisher, Garrett W. and Spence, Jason R. and Mulloy, James C. and Zorn, Aaron M. and Wells, James M. and Helmrath, Michael A.},
  doi          = {10.1038/s41587-022-01558-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {824-831},
  shortjournal = {Nature Biotech.},
  title        = {In vivo development of immune tissue in human intestinal organoids transplanted into humanized mice},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ScChIX-seq infers dynamic relationships between histone
modifications in single cells. <em>NBT</em>, <em>41</em>(6), 813–823.
(<a href="https://doi.org/10.1038/s41587-022-01560-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Regulation of chromatin states involves the dynamic interplay between different histone modifications to control gene expression. Recent advances have enabled mapping of histone marks in single cells, but most methods are constrained to profile only one histone mark per cell. Here, we present an integrated experimental and computational framework, scChIX-seq (single-cell chromatin immunocleavage and unmixing sequencing), to map several histone marks in single cells. scChIX-seq multiplexes two histone marks together in single cells, then computationally deconvolves the signal using training data from respective histone mark profiles. This framework learns the cell-type-specific correlation structure between histone marks, and therefore does not require a priori assumptions of their genomic distributions. Using scChIX-seq, we demonstrate multimodal analysis of histone marks in single cells across a range of mark combinations. Modeling dynamics of in vitro macrophage differentiation enables integrated analysis of chromatin velocity. Overall, scChIX-seq unlocks systematic interrogation of the interplay between histone modifications in single cells. Analysis of two histone marks in single cells reveals interplay between modifications.},
  archive      = {J_NBT},
  author       = {Yeung, Jake and Florescu, Maria and Zeller, Peter and de Barbanson, Buys Anton and Wellenstein, Max D. and van Oudenaarden, Alexander},
  doi          = {10.1038/s41587-022-01560-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {813-823},
  shortjournal = {Nature Biotech.},
  title        = {ScChIX-seq infers dynamic relationships between histone modifications in single cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Nanobody-tethered transposition enables multifactorial
chromatin profiling at single-cell resolution. <em>NBT</em>,
<em>41</em>(6), 806–812. (<a
href="https://doi.org/10.1038/s41587-022-01588-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin states are functionally defined by a complex combination of histone modifications, transcription factor binding, DNA accessibility and other factors. Current methods for defining chromatin states cannot measure more than one aspect in a single experiment at single-cell resolution. Here we introduce nanobody-tethered transposition followed by sequencing (NTT-seq), an assay capable of measuring the genome-wide presence of up to three histone modifications and protein–DNA binding sites at single-cell resolution. NTT-seq uses recombinant Tn5 transposase fused to a set of secondary nanobodies (nb). Each nb–Tn5 fusion protein specifically binds to different immunoglobulin-G antibodies, enabling a mixture of primary antibodies binding different epitopes to be used in a single experiment. We apply bulk-cell and single-cell NTT-seq to generate high-resolution multimodal maps of chromatin states in cell culture and in human immune cells. We also extend NTT-seq to enable simultaneous profiling of cell surface protein expression and multimodal chromatin states to study cells of the immune system. Cell surface protein expression and multiple epigenetic features are mapped simultaneously in single cells.},
  archive      = {J_NBT},
  author       = {Stuart, Tim and Hao, Stephanie and Zhang, Bingjie and Mekerishvili, Levan and Landau, Dan A. and Maniatis, Silas and Satija, Rahul and Raimondi, Ivan},
  doi          = {10.1038/s41587-022-01588-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {806-812},
  shortjournal = {Nature Biotech.},
  title        = {Nanobody-tethered transposition enables multifactorial chromatin profiling at single-cell resolution},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multimodal chromatin profiling using nanobody-based
single-cell CUT&amp;tag. <em>NBT</em>, <em>41</em>(6), 794–805. (<a
href="https://doi.org/10.1038/s41587-022-01535-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Probing histone modifications at a single-cell level in thousands of cells has been enabled by technologies such as single-cell CUT&amp;amp;Tag. Here we describe nano-CUT&amp;amp;Tag (nano-CT), which allows simultaneous mapping of up to three epigenomic modalities at single-cell resolution using nanobody-Tn5 fusion proteins. Multimodal nano-CT is compatible with starting materials as low as 25,000–200,000 cells and has significantly higher sensitivity and number of fragments per cell than single-cell CUT&amp;amp;Tag. We use nano-CT to simultaneously profile chromatin accessibility, H3K27ac, and H3K27me3 in juvenile mouse brain, allowing for discrimination of more cell types and states than unimodal single-cell CUT&amp;amp;Tag. We also infer chromatin velocity between assay for transposase-accessible chromatin (ATAC) and H3K27ac in the oligodendrocyte lineage and deconvolute H3K27me3 repressive states, finding two sequential waves of H3K27me3 repression at distinct gene modules during oligodendrocyte lineage progression. Given its high resolution, versatility, and multimodal features, nano-CT allows unique insights in epigenetic landscapes in complex biological systems at the single-cell level. Three epigenetic features are mapped at single-cell resolution using Tn5-nanobody fusion proteins},
  archive      = {J_NBT},
  author       = {Bartosovic, Marek and Castelo-Branco, Gonçalo},
  doi          = {10.1038/s41587-022-01535-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {794-805},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal chromatin profiling using nanobody-based single-cell CUT&amp;Tag},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integration of whole transcriptome spatial profiling with
protein markers. <em>NBT</em>, <em>41</em>(6), 788–793. (<a
href="https://doi.org/10.1038/s41587-022-01536-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics and proteomics provide complementary information that independently transformed our understanding of complex biological processes. However, experimental integration of these modalities is limited. To overcome this, we developed Spatial PrOtein and Transcriptome Sequencing (SPOTS) for high-throughput simultaneous spatial transcriptomics and protein profiling. Compared with unimodal measurements, SPOTS substantially improves signal resolution and cell clustering and enhances the discovery power in differential gene expression analysis across tissue regions. Visium spatial transcriptomics is combined with markers for more than 30 proteins.},
  archive      = {J_NBT},
  author       = {Ben-Chetrit, Nir and Niu, Xiang and Swett, Ariel D. and Sotelo, Jesus and Jiao, Maria S. and Stewart, Caitlin M. and Potenski, Catherine and Mielinis, Paulius and Roelli, Patrick and Stoeckius, Marlon and Landau, Dan A.},
  doi          = {10.1038/s41587-022-01536-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {788-793},
  shortjournal = {Nature Biotech.},
  title        = {Integration of whole transcriptome spatial profiling with protein markers},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identification of patient-specific CD4+ and CD8+ t cell
neoantigens through HLA-unbiased genetic screens. <em>NBT</em>,
<em>41</em>(6), 783–787. (<a
href="https://doi.org/10.1038/s41587-022-01547-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer neoantigens that arise from tumor mutations are drivers of tumor-specific T cell responses, but identification of T cell-recognized neoantigens in individual patients is challenging. Previous methods have restricted antigen discovery to selected HLA alleles, thereby limiting the breadth of neoantigen repertoires that can be uncovered. Here, we develop a genetic neoantigen screening system that allows sensitive identification of CD4+ and CD8+ T cell-recognized neoantigens across patients’ complete HLA genotypes. Patient-specific tumor neoantigens are recognized by T cells.},
  archive      = {J_NBT},
  author       = {Cattaneo, Chiara M. and Battaglia, Thomas and Urbanus, Jos and Moravec, Ziva and Voogd, Rhianne and de Groot, Rosa and Hartemink, Koen J. and Haanen, John B. A. G. and Voest, Emile E. and Schumacher, Ton N. and Scheper, Wouter},
  doi          = {10.1038/s41587-022-01547-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {783-787},
  shortjournal = {Nature Biotech.},
  title        = {Identification of patient-specific CD4+ and CD8+ t cell neoantigens through HLA-unbiased genetic screens},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The expanding vistas of spatial transcriptomics.
<em>NBT</em>, <em>41</em>(6), 773–782. (<a
href="https://doi.org/10.1038/s41587-022-01448-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The formation and maintenance of tissue integrity requires complex, coordinated activities by thousands of genes and their encoded products. Until recently, transcript levels could only be quantified for a few genes in tissues, but advances in DNA sequencing, oligonucleotide synthesis and fluorescence microscopy have enabled the invention of a suite of spatial transcriptomics technologies capable of measuring the expression of many, or all, genes in situ. These technologies have evolved rapidly in sensitivity, multiplexing and throughput. As such, they have enabled the determination of the cell-type architecture of tissues, the querying of cell–cell interactions and the monitoring of molecular interactions between tissue components. The rapidly evolving spatial genomics landscape will enable generalized high-throughput genomic measurements and perturbations to be performed in the context of tissues. These advances will empower hypothesis generation and biological discovery and bridge the worlds of tissue biology and genomics. Spatial transcriptomics workflows, metrics and limitations are reviewed and discussed.},
  archive      = {J_NBT},
  author       = {Tian, Luyi and Chen, Fei and Macosko, Evan Z.},
  doi          = {10.1038/s41587-022-01448-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {773-782},
  shortjournal = {Nature Biotech.},
  title        = {The expanding vistas of spatial transcriptomics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell multi-omics allows functional characterization
of structural variants. <em>NBT</em>, <em>41</em>(6), 771–772. (<a
href="https://doi.org/10.1038/s41587-022-01552-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a computational method based on Strand-seq that combines nucleosome occupancy and structural variation analysis in single cells to identify the functional effects of somatic structural variants in leukemia. The method revealed the molecular consequences of somatic driver mutations and informed specific therapeutic targeting of a chromothriptic leukemic subclone.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01552-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {771-772},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell multi-omics allows functional characterization of structural variants},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiplexed mapping of chromatin features at single-cell
resolution. <em>NBT</em>, <em>41</em>(6), 769–770. (<a
href="https://doi.org/10.1038/s41587-022-01589-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell nanobody-tethered transposition followed by sequencing (scNTT-seq) is a new assay that measures the genome-wide presence of multiple histone modifications and protein–DNA binding sites at single-cell resolution. scNTT-seq generates high-resolution multimodal maps of chromatin states with high accuracy and sensitivity.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01589-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {769-770},
  shortjournal = {Nature Biotech.},
  title        = {Multiplexed mapping of chromatin features at single-cell resolution},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell nanobody-based profiles of multiple epigenetic
modalities and chromatin velocity. <em>NBT</em>, <em>41</em>(6),
767–768. (<a href="https://doi.org/10.1038/s41587-022-01596-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanobody–Tn5 transposase fusion proteins enable detection of two histone marks and open chromatin regions at the same time, with single-cell resolution.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01596-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {767-768},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell nanobody-based profiles of multiple epigenetic modalities and chromatin velocity},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Human gastrointestinal-tract variation catalogued.
<em>NBT</em>, <em>41</em>(6), 766. (<a
href="https://doi.org/10.1038/s41587-023-01841-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01841-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {766},
  shortjournal = {Nature Biotech.},
  title        = {Human gastrointestinal-tract variation catalogued},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Combining 47 human genomes into a single pangenome.
<em>NBT</em>, <em>41</em>(6), 766. (<a
href="https://doi.org/10.1038/s41587-023-01842-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01842-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {766},
  shortjournal = {Nature Biotech.},
  title        = {Combining 47 human genomes into a single pangenome},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Electronic skin with onboard sensory feedback system.
<em>NBT</em>, <em>41</em>(6), 766. (<a
href="https://doi.org/10.1038/s41587-023-01843-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01843-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {766},
  shortjournal = {Nature Biotech.},
  title        = {Electronic skin with onboard sensory feedback system},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Drug delivery. <em>NBT</em>, <em>41</em>(6), 765. (<a
href="https://doi.org/10.1038/s41587-023-01813-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to compositions, methods and devices for drug delivery.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01813-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {765},
  shortjournal = {Nature Biotech.},
  title        = {Drug delivery},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The global patent landscape of HER2-targeted biologics.
<em>NBT</em>, <em>41</em>(6), 756–764. (<a
href="https://doi.org/10.1038/s41587-023-01814-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global patents on HER2-targeted biologics show rapid growth and a diversified technology pattern.},
  archive      = {J_NBT},
  author       = {Li, Qingjian and Xu, Jiaqi and Sun, Qianshu and Zhang, Zebang and Hu, Yuanjia and Yao, Herui},
  doi          = {10.1038/s41587-023-01814-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {756-764},
  shortjournal = {Nature Biotech.},
  title        = {The global patent landscape of HER2-targeted biologics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). How the monty hall problem is similar to the false discovery
rate in high-throughput data analysis. <em>NBT</em>, <em>41</em>(6),
754–755. (<a href="https://doi.org/10.1038/s41587-023-01794-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Li, Jingyi Jessica},
  doi          = {10.1038/s41587-023-01794-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {754-755},
  shortjournal = {Nature Biotech.},
  title        = {How the monty hall problem is similar to the false discovery rate in high-throughput data analysis},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). New precision-breeding law unlocks gene editing in england.
<em>NBT</em>, <em>41</em>(6), 752–753. (<a
href="https://doi.org/10.1038/s41587-023-01795-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Caccamo, Mario},
  doi          = {10.1038/s41587-023-01795-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {752-753},
  shortjournal = {Nature Biotech.},
  title        = {New precision-breeding law unlocks gene editing in england},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). How will generative AI disrupt data science in drug
discovery? <em>NBT</em>, <em>41</em>(6), 750–751. (<a
href="https://doi.org/10.1038/s41587-023-01789-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Vert, Jean-Philippe},
  doi          = {10.1038/s41587-023-01789-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {750-751},
  shortjournal = {Nature Biotech.},
  title        = {How will generative AI disrupt data science in drug discovery?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(6), 744. (<a
href="https://doi.org/10.1038/s41587-023-01822-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01822-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {744},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). The instant spray-on dress. <em>NBT</em>, <em>41</em>(6),
743. (<a href="https://doi.org/10.1038/s41587-023-01829-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-023-01829-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {743},
  shortjournal = {Nature Biotech.},
  title        = {The instant spray-on dress},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Novo and moderna seal base-editing pacts with life edit.
<em>NBT</em>, <em>41</em>(6), 741. (<a
href="https://doi.org/10.1038/s41587-023-01838-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01838-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {741},
  shortjournal = {Nature Biotech.},
  title        = {Novo and moderna seal base-editing pacts with life edit},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). After GLP-1, what’s next for weight loss? <em>NBT</em>,
<em>41</em>(6), 740–743. (<a
href="https://doi.org/10.1038/s41587-023-01818-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The GLP-1 agonist Wegovy has re-energized the hunt for obesity treatments. Alternatives — ranging from bitter taste compounds to lean muscle boosters and bacteria — are already in the clinic.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-023-01818-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {740-743},
  shortjournal = {Nature Biotech.},
  title        = {After GLP-1, what’s next for weight loss?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ADCs’ revival. <em>NBT</em>, <em>41</em>(6), 740. (<a
href="https://doi.org/10.1038/s41587-023-01836-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01836-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {740},
  shortjournal = {Nature Biotech.},
  title        = {ADCs’ revival},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). First herpesvirus gene therapy. <em>NBT</em>,
<em>41</em>(6), 739. (<a
href="https://doi.org/10.1038/s41587-023-01835-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01835-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {739},
  shortjournal = {Nature Biotech.},
  title        = {First herpesvirus gene therapy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Why gene therapies must go virus-free. <em>NBT</em>,
<em>41</em>(6), 737–739. (<a
href="https://doi.org/10.1038/s41587-023-01824-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene therapies have made spectacular progress in delivering new cures for previously intractable disease, but they remain the world’s most expensive treatments. Now companies are replacing the virus in gene therapies with new delivery technologies that promise not only to overcome the limitations of viral vectors but to slash production costs too.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-023-01824-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {737-739},
  shortjournal = {Nature Biotech.},
  title        = {Why gene therapies must go virus-free},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Can microbes save the planet? <em>NBT</em>, <em>41</em>(6),
735. (<a href="https://doi.org/10.1038/s41587-023-01837-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic engineering and accelerated evolution of microbes aim to help climate change.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01837-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {735},
  shortjournal = {Nature Biotech.},
  title        = {Can microbes save the planet?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Drugging RNA. <em>NBT</em>, <em>41</em>(6), 745–749. (<a
href="https://doi.org/10.1038/s41587-023-01790-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Small molecules that target messenger RNA have therapeutic potential, but the field still lacks an unqualified success. Companies differ on how to move the resurgent field forward.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-023-01790-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {745-749},
  shortjournal = {Nature Biotech.},
  title        = {Drugging RNA},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). People. <em>NBT</em>, <em>41</em>(5), 734. (<a
href="https://doi.org/10.1038/s41587-023-01785-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01785-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {734},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A single-center assessment of mental health and well-being
in a biomedical sciences graduate program. <em>NBT</em>, <em>41</em>(5),
728–733. (<a href="https://doi.org/10.1038/s41587-023-01778-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A diversity of factors influence student mental health, arguing for the importance of longitudinal monitoring of, and accountability for, student mental health at graduate institutions.},
  archive      = {J_NBT},
  author       = {Jachim, Sarah K. and Bowles, Bradley S. and Panicker, Anjali J. and Yousaf, Iris and Brown, Alyssa D. and Zoroufy, Alexander J. and Boehmer, Kasey R. and Stonnington, Cynthia M. and Vadeboncoeur, Tyler F. and Lujan, J. Luis and Ehlers, Shawna L. and Schulze, Autumn J.},
  doi          = {10.1038/s41587-023-01778-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {728-733},
  shortjournal = {Nature Biotech.},
  title        = {A single-center assessment of mental health and well-being in a biomedical sciences graduate program},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Protection of cell therapeutics from antibody-mediated
killing by CD64 overexpression. <em>NBT</em>, <em>41</em>(5), 717–727.
(<a href="https://doi.org/10.1038/s41587-022-01540-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Allogeneic cell therapeutics for cancer therapy or regenerative medicine are susceptible to antibody-mediated killing, which diminishes their efficacy. Here we report a strategy to protect cells from antibody-mediated killing that relies on engineered overexpression of the IgG receptor CD64. We show that human and mouse iPSC-derived endothelial cells (iECs) overexpressing CD64 escape antibody-dependent cellular cytotoxicity (ADCC) and complement-dependent cytotoxicity from IgG antibodies in vitro and in ADCC-enabled mice. When CD64 expression was combined with hypoimmune genetic modifications known to protect against cellular immunity, B2M−/−CIITA−/− CD47/CD64-transgenic iECs were resistant to both IgG antibody-mediated and cellular immune killing in vitro and in humanized mice. Mechanistic studies demonstrated that CD64 or its intracellularly truncated analog CD64t effectively capture monomeric IgG and occupy their Fc, and the IgG bind and occupy their target antigens. In three applications of the approach, human CD64t-engineered thyroid epithelial cells, pancreatic beta cells and CAR T cells withstood clinically relevant levels of graft-directed antibodies and fully evaded antibody-mediated killing. Hypoimmune engineering of cell therapies is extended to protection from host antibodies.},
  archive      = {J_NBT},
  author       = {Gravina, Alessia and Tediashvili, Grigol and Rajalingam, Raja and Quandt, Zoe and Deisenroth, Chad and Schrepfer, Sonja and Deuse, Tobias},
  doi          = {10.1038/s41587-022-01540-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {717-727},
  shortjournal = {Nature Biotech.},
  title        = {Protection of cell therapeutics from antibody-mediated killing by CD64 overexpression},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multifactorial profiling of epigenetic landscapes at
single-cell resolution using MulTI-tag. <em>NBT</em>, <em>41</em>(5),
708–716. (<a href="https://doi.org/10.1038/s41587-022-01522-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin profiling at locus resolution uncovers gene regulatory features that define cell types and developmental trajectories, but it remains challenging to map and compare different chromatin-associated proteins in the same sample. Here we describe Multiple Target Identification by Tagmentation (MulTI-Tag), an antibody barcoding approach for profiling multiple chromatin features simultaneously in single cells. We optimized MulTI-Tag to retain high sensitivity and specificity, and we demonstrate detection of up to three histone modifications in the same cell: H3K27me3, H3K4me1/2 and H3K36me3. We apply MulTI-Tag to resolve distinct cell types and developmental trajectories; to distinguish unique, coordinated patterns of active and repressive element regulatory usage associated with differentiation outcomes; and to uncover associations between histone marks. Multifactorial epigenetic profiling holds promise for comprehensively characterizing cell-specific gene regulatory landscapes in development and disease. A scalable barcoding method measures multiple chromatin-associated proteins in single cells.},
  archive      = {J_NBT},
  author       = {Meers, Michael P. and Llagas, Geneva and Janssens, Derek H. and Codomo, Christine A. and Henikoff, Steven},
  doi          = {10.1038/s41587-022-01522-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {708-716},
  shortjournal = {Nature Biotech.},
  title        = {Multifactorial profiling of epigenetic landscapes at single-cell resolution using MulTI-tag},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Programmable eukaryotic protein synthesis with RNA sensors
by harnessing ADAR. <em>NBT</em>, <em>41</em>(5), 698–707. (<a
href="https://doi.org/10.1038/s41587-022-01534-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Programmable approaches to sense and respond to the presence of specific RNAs in biological systems have broad applications in research, diagnostics, and therapeutics. Here we engineer a programmable RNA-sensing technology, reprogrammable ADAR sensors (RADARS), which harnesses RNA editing by adenosine deaminases acting on RNA (ADAR) to gate translation of a cargo protein by the presence of endogenous RNA transcripts. Introduction of a stop codon in a guide upstream of the cargo makes translation contingent on binding of an endogenous transcript to the guide, leading to ADAR editing of the stop codon and allowing translational readthrough. Through systematic sensor engineering, we achieve 277 fold improvement in sensor activation and engineer RADARS with diverse cargo proteins, including luciferases, fluorescent proteins, recombinases, and caspases, enabling detection sensitivity on endogenous transcripts expressed at levels as low as 13 transcripts per million. We show that RADARS are functional as either expressed DNA or synthetic mRNA and with either exogenous or endogenous ADAR. We apply RADARS in multiple contexts, including tracking transcriptional states, RNA-sensing-induced cell death, cell-type identification, and control of synthetic mRNA translation. An RNA sensor harnesses ADAR to control translation of a target protein.},
  archive      = {J_NBT},
  author       = {Jiang, Kaiyi and Koob, Jeremy and Chen, Xi Dawn and Krajeski, Rohan N. and Zhang, Yifan and Volf, Verena and Zhou, Wenyuan and Sgrizzi, Samantha R. and Villiger, Lukas and Gootenberg, Jonathan S. and Chen, Fei and Abudayyeh, Omar O.},
  doi          = {10.1038/s41587-022-01534-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {698-707},
  shortjournal = {Nature Biotech.},
  title        = {Programmable eukaryotic protein synthesis with RNA sensors by harnessing ADAR},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Improved cytosine base editors generated from TadA variants.
<em>NBT</em>, <em>41</em>(5), 686–697. (<a
href="https://doi.org/10.1038/s41587-022-01611-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine base editors (CBEs) enable programmable genomic C·G-to-T·A transition mutations and typically comprise a modified CRISPR–Cas enzyme, a naturally occurring cytidine deaminase, and an inhibitor of uracil repair. Previous studies have shown that CBEs utilizing naturally occurring cytidine deaminases may cause unguided, genome-wide cytosine deamination. While improved CBEs that decrease stochastic genome-wide off-targets have subsequently been reported, these editors can suffer from suboptimal on-target performance. Here, we report the generation and characterization of CBEs that use engineered variants of TadA (CBE-T) that enable high on-target C·G to T·A across a sequence-diverse set of genomic loci, demonstrate robust activity in primary cells and cause no detectable elevation in genome-wide mutation. Additionally, we report cytosine and adenine base editors (CABEs) catalyzing both A-to-I and C-to-U editing (CABE-Ts). Together with ABEs, CBE-Ts and CABE-Ts enable the programmable installation of all transition mutations using laboratory-evolved TadA variants with improved properties relative to previously reported CBEs. Improved cytosine base editors are generated by directed evolution of an adenosine deaminase.},
  archive      = {J_NBT},
  author       = {Lam, Dieter K. and Feliciano, Patricia R. and Arif, Amena and Bohnuud, Tanggis and Fernandez, Thomas P. and Gehrke, Jason M. and Grayson, Phil and Lee, Kin D. and Ortega, Manuel A. and Sawyer, Courtney and Schwaegerle, Noah D. and Peraro, Leila and Young, Lauren and Lee, Seung-Joo and Ciaramella, Giuseppe and Gaudelli, Nicole M.},
  doi          = {10.1038/s41587-022-01611-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {686-697},
  shortjournal = {Nature Biotech.},
  title        = {Improved cytosine base editors generated from TadA variants},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolution of an adenine base editor into a small, efficient
cytosine base editor with low off-target activity. <em>NBT</em>,
<em>41</em>(5), 673–685. (<a
href="https://doi.org/10.1038/s41587-022-01533-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine base editors (CBEs) are larger and can suffer from higher off-target activity or lower on-target editing efficiency than current adenine base editors (ABEs). To develop a CBE that retains the small size, low off-target activity and high on-target activity of current ABEs, we evolved the highly active deoxyadenosine deaminase TadA-8e to perform cytidine deamination using phage-assisted continuous evolution. Evolved TadA cytidine deaminases contain mutations at DNA-binding residues that alter enzyme selectivity to strongly favor deoxycytidine over deoxyadenosine deamination. Compared to commonly used CBEs, TadA-derived cytosine base editors (TadCBEs) offer similar or higher on-target activity, smaller size and substantially lower Cas-independent DNA and RNA off-target editing activity. We also identified a TadA dual base editor (TadDE) that performs equally efficient cytosine and adenine base editing. TadCBEs support single or multiplexed base editing at therapeutically relevant genomic loci in primary human T cells and primary human hematopoietic stem and progenitor cells. TadCBEs expand the utility of CBEs for precision gene editing. Improved cytosine base editors are generated by phage-assisted evolution of a deoxyadenosine deaminase.},
  archive      = {J_NBT},
  author       = {Neugebauer, Monica E. and Hsu, Alvin and Arbab, Mandana and Krasnow, Nicholas A. and McElroy, Amber N. and Pandey, Smriti and Doman, Jordan L. and Huang, Tony P. and Raguram, Aditya and Banskota, Samagya and Newby, Gregory A. and Tolar, Jakub and Osborn, Mark J. and Liu, David R.},
  doi          = {10.1038/s41587-022-01533-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {673-685},
  shortjournal = {Nature Biotech.},
  title        = {Evolution of an adenine base editor into a small, efficient cytosine base editor with low off-target activity},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Re-engineering the adenine deaminase TadA-8e for efficient
and specific CRISPR-based cytosine base editing. <em>NBT</em>,
<em>41</em>(5), 663–672. (<a
href="https://doi.org/10.1038/s41587-022-01532-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine base editors (CBEs) efficiently generate precise C·G-to-T·A base conversions, but the activation-induced cytidine deaminase/apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like (AID/APOBEC) protein family deaminase component induces considerable off-target effects and indels. To explore unnatural cytosine deaminases, we repurpose the adenine deaminase TadA-8e for cytosine conversion. The introduction of an N46L variant in TadA-8e eliminates its adenine deaminase activity and results in a TadA-8e-derived C-to-G base editor (Td-CGBE) capable of highly efficient and precise C·G-to-G·C editing. Through fusion with uracil glycosylase inhibitors and further introduction of additional variants, a series of Td-CBEs was obtained either with a high activity similar to that of BE4max or with higher precision compared to other reported accurate CBEs. Td-CGBE/Td-CBEs show very low indel effects and a background level of Cas9-dependent or Cas9-independent DNA/RNA off-target editing. Moreover, Td-CGBE/Td-CBEs are more efficient in generating accurate edits in homopolymeric cytosine sites in cells or mouse embryos, suggesting their accuracy and safety for gene therapy and other applications. Improved cytosine base editors are created by repurposing an adenine deaminase.},
  archive      = {J_NBT},
  author       = {Chen, Liang and Zhu, Biyun and Ru, Gaomeng and Meng, Haowei and Yan, Yongchang and Hong, Mengjia and Zhang, Dan and Luan, Changming and Zhang, Shun and Wu, Hao and Gao, Hongyi and Bai, Sijia and Li, Changqing and Ding, Ruoyi and Xue, Niannian and Lei, Zhixin and Chen, Yuting and Guan, Yuting and Siwko, Stefan and Cheng, Yiyun and Song, Gaojie and Wang, Liren and Yi, Chengqi and Liu, Mingyao and Li, Dali},
  doi          = {10.1038/s41587-022-01532-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {663-672},
  shortjournal = {Nature Biotech.},
  title        = {Re-engineering the adenine deaminase TadA-8e for efficient and specific CRISPR-based cytosine base editing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Wireless, closed-loop, smart bandage with integrated sensors
and stimulators for advanced wound care and accelerated healing.
<em>NBT</em>, <em>41</em>(5), 652–662. (<a
href="https://doi.org/10.1038/s41587-022-01528-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {‘Smart’ bandages based on multimodal wearable devices could enable real-time physiological monitoring and active intervention to promote healing of chronic wounds. However, there has been limited development in incorporation of both sensors and stimulators for the current smart bandage technologies. Additionally, while adhesive electrodes are essential for robust signal transduction, detachment of existing adhesive dressings can lead to secondary damage to delicate wound tissues without switchable adhesion. Here we overcome these issues by developing a flexible bioelectronic system consisting of wirelessly powered, closed-loop sensing and stimulation circuits with skin-interfacing hydrogel electrodes capable of on-demand adhesion and detachment. In mice, we demonstrate that our wound care system can continuously monitor skin impedance and temperature and deliver electrical stimulation in response to the wound environment. Across preclinical wound models, the treatment group healed ~25\% more rapidly and with ~50\% enhancement in dermal remodeling compared with control. Further, we observed activation of proregenerative genes in monocyte and macrophage cell populations, which may enhance tissue regeneration, neovascularization and dermal recovery. A wireless ‘smart’ bandage stimulates wound healing.},
  archive      = {J_NBT},
  author       = {Jiang, Yuanwen and Trotsyuk, Artem A. and Niu, Simiao and Henn, Dominic and Chen, Kellen and Shih, Chien-Chung and Larson, Madelyn R. and Mermin-Bunnell, Alana M. and Mittal, Smiti and Lai, Jian-Cheng and Saberi, Aref and Beard, Ethan and Jing, Serena and Zhong, Donglai and Steele, Sydney R. and Sun, Kefan and Jain, Tanish and Zhao, Eric and Neimeth, Christopher R. and Viana, Willian G. and Tang, Jing and Sivaraj, Dharshan and Padmanabhan, Jagannath and Rodrigues, Melanie and Perrault, David P. and Chattopadhyay, Arhana and Maan, Zeshaan N. and Leeolou, Melissa C. and Bonham, Clark A. and Kwon, Sun Hyung and Kussie, Hudson C. and Fischer, Katharina S. and Gurusankar, Gurupranav and Liang, Kui and Zhang, Kailiang and Nag, Ronjon and Snyder, Michael P. and Januszyk, Michael and Gurtner, Geoffrey C. and Bao, Zhenan},
  doi          = {10.1038/s41587-022-01528-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {652-662},
  shortjournal = {Nature Biotech.},
  title        = {Wireless, closed-loop, smart bandage with integrated sensors and stimulators for advanced wound care and accelerated healing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Recording of cellular physiological histories along
optically readable self-assembling protein chains. <em>NBT</em>,
<em>41</em>(5), 640–651. (<a
href="https://doi.org/10.1038/s41587-022-01586-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Observing cellular physiological histories is key to understanding normal and disease-related processes. Here we describe expression recording islands—a fully genetically encoded approach that enables both continual digital recording of biological information within cells and subsequent high-throughput readout in fixed cells. The information is stored in growing intracellular protein chains made of self-assembling subunits, human-designed filament-forming proteins bearing different epitope tags that each correspond to a different cellular state or function (for example, gene expression downstream of neural activity or pharmacological exposure), allowing the physiological history to be read out along the ordered subunits of protein chains with conventional optical microscopy. We use expression recording islands to record gene expression timecourse downstream of specific pharmacological and physiological stimuli in cultured neurons and in living mouse brain, with a time resolution of a fraction of a day, over periods of days to weeks. A history of cellular events is recorded in self-assembling protein chains.},
  archive      = {J_NBT},
  author       = {Linghu, Changyang and An, Bobae and Shpokayte, Monika and Celiker, Orhan T. and Shmoel, Nava and Zhang, Ruihan and Zhang, Chi and Park, Demian and Park, Won Min and Ramirez, Steve and Boyden, Edward S.},
  doi          = {10.1038/s41587-022-01586-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {640-651},
  shortjournal = {Nature Biotech.},
  title        = {Recording of cellular physiological histories along optically readable self-assembling protein chains},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Time-tagged ticker tapes for intracellular recordings.
<em>NBT</em>, <em>41</em>(5), 631–639. (<a
href="https://doi.org/10.1038/s41587-022-01524-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recording transcriptional histories of a cell would enable deeper understanding of cellular developmental trajectories and responses to external perturbations. Here we describe an engineered protein fiber that incorporates diverse fluorescent marks during its growth to store a ticker tape-like history. An embedded HaloTag reporter incorporates user-supplied dyes, leading to colored stripes that map the growth of each individual fiber to wall clock time. A co-expressed eGFP tag driven by a promoter of interest records a history of transcriptional activation. High-resolution multi-spectral imaging on fixed samples reads the cellular histories, and interpolation of eGFP marks relative to HaloTag timestamps provides accurate absolute timing. We demonstrate recordings of doxycycline-induced transcription in HEK cells and cFos promoter activation in cultured neurons, with a single-cell absolute accuracy of 30–40 minutes over a 12-hour recording. The protein-based ticker tape design we present here could be generalized to achieve massively parallel single-cell recordings of diverse physiological modalities. Reporter gene expression history in cells is recorded with engineered fluorescent protein fibers.},
  archive      = {J_NBT},
  author       = {Lin, Dingchang and Li, Xiuyuan and Moult, Eric and Park, Pojeong and Tang, Benjamin and Shen, Hao and Grimm, Jonathan B. and Falco, Natalie and Jia, Bill Z. and Baker, David and Lavis, Luke D. and Cohen, Adam E.},
  doi          = {10.1038/s41587-022-01524-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {631-639},
  shortjournal = {Nature Biotech.},
  title        = {Time-tagged ticker tapes for intracellular recordings},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-throughput retrieval of target sequences from complex
clone libraries using CRISPRi. <em>NBT</em>, <em>41</em>(5), 626–630.
(<a href="https://doi.org/10.1038/s41587-022-01531-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The capture of metagenomic DNA in large clone libraries provides the opportunity to study microbial diversity that is inaccessible using culture-dependent methods. In this study, we harnessed nuclease-deficient Cas9 to establish a CRISPR counter-selection interruption circuit (CCIC) that can be used to retrieve target clones from complex libraries. Combining modern sequencing methods with CCIC cloning allows for rapid physical access to the genetic diversity present in natural ecosystems. A CRISPRi method retrieves sequences of interest from large metagenomic libraries.},
  archive      = {J_NBT},
  author       = {Burian, Ján and Libis, Vincent K. and Hernandez, Yozen A. and Guerrero-Porras, Liliana and Ternei, Melinda A. and Brady, Sean F.},
  doi          = {10.1038/s41587-022-01531-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {626-630},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput retrieval of target sequences from complex clone libraries using CRISPRi},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Profiling chromatin–protein crosstalk at locus resolution in
single cells. <em>NBT</em>, <em>41</em>(5), 624–625. (<a
href="https://doi.org/10.1038/s41587-022-01557-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Development is orchestrated in part by myriad chromatin-associated proteins that decorate the genome and physically coordinate in ways that are often unclear. An epigenomics technique called MulTI-Tag maps the genomic locations of different chromatin proteins in the same individual cells, helping to unveil their interactions.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01557-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {624-625},
  shortjournal = {Nature Biotech.},
  title        = {Profiling chromatin–protein crosstalk at locus resolution in single cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Healing chronic wounds with a wireless smart bandage with
integrated sensors and stimulators. <em>NBT</em>, <em>41</em>(5),
622–623. (<a href="https://doi.org/10.1038/s41587-022-01564-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a wireless smart bandage with sensors that can detect the condition of a wound and stimulators that can deliver electric cues. Application of our device on wounds in mouse models reduced infection and accelerated wound healing.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01564-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {622-623},
  shortjournal = {Nature Biotech.},
  title        = {Healing chronic wounds with a wireless smart bandage with integrated sensors and stimulators},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rapid mining of (meta)genomic biodiversity by repurposing
CRISPRi for positive selection. <em>NBT</em>, <em>41</em>(5), 620–621.
(<a href="https://doi.org/10.1038/s41587-022-01563-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The global microbiota contains an enormous, previously inaccessible reservoir of biodiversity that can now be captured as large DNA fragments in clone libraries. To mine cloned biodiversity more effectively, we developed a CRISPR-interference-based counter-selection interruption circuit for the high-throughput retrieval of desired clones within complex libraries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01563-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {620-621},
  shortjournal = {Nature Biotech.},
  title        = {Rapid mining of (meta)genomic biodiversity by repurposing CRISPRi for positive selection},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Forum: Alexandrov and le. <em>NBT</em>, <em>41</em>(5),
619. (<a href="https://doi.org/10.1038/s41587-023-01806-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01806-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {619},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Alexandrov and le},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Writing cellular history in protein chains. <em>NBT</em>,
<em>41</em>(5), 617–618. (<a
href="https://doi.org/10.1038/s41587-022-01597-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elongating protein fibers record the transcriptional activity of single cells for later readout by imaging.},
  archive      = {J_NBT},
  author       = {Kwon, Hyung-Bae},
  doi          = {10.1038/s41587-022-01597-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {617-618},
  shortjournal = {Nature Biotech.},
  title        = {Writing cellular history in protein chains},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Bespoke protein delivery via bacterial injection systems.
<em>NBT</em>, <em>41</em>(5), 616. (<a
href="https://doi.org/10.1038/s41587-023-01809-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01809-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {616},
  shortjournal = {Nature Biotech.},
  title        = {Bespoke protein delivery via bacterial injection systems},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023n). Protein sequences absent in nature generated with diffusion
model. <em>NBT</em>, <em>41</em>(5), 616. (<a
href="https://doi.org/10.1038/s41587-023-01810-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01810-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {616},
  shortjournal = {Nature Biotech.},
  title        = {Protein sequences absent in nature generated with diffusion model},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023q). Transposon–gene chimeras move up as cancer epitopes.
<em>NBT</em>, <em>41</em>(5), 616. (<a
href="https://doi.org/10.1038/s41587-023-01811-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01811-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {616},
  shortjournal = {Nature Biotech.},
  title        = {Transposon–gene chimeras move up as cancer epitopes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Proteomics. <em>NBT</em>, <em>41</em>(5), 615. (<a
href="https://doi.org/10.1038/s41587-023-01777-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to protein identification, characterization and analysis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01777-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {615},
  shortjournal = {Nature Biotech.},
  title        = {Proteomics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Who counts as an inventor? Seniority and gender in 430,000
biomedical inventor–researcher teams. <em>NBT</em>, <em>41</em>(5),
610–614. (<a href="https://doi.org/10.1038/s41587-023-01771-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of the overlap of authors and inventors in a comprehensive citation map of patents to the biomedical research literature reveals significant and persistent disparities within the commercialization process.},
  archive      = {J_NBT},
  author       = {Manjunath, Anoop and Kahrobai, Nathan and Manjunath, Jaya and Seffens, Angelina and Gowda, Arya and Umbarkar, Rohaan and Umbarkar, Esha and Zou, James Y. and Kumar, Ishan},
  doi          = {10.1038/s41587-023-01771-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {610-614},
  shortjournal = {Nature Biotech.},
  title        = {Who counts as an inventor? seniority and gender in 430,000 biomedical inventor–researcher teams},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Challenges and solutions to advancing health equity with
medical devices. <em>NBT</em>, <em>41</em>(5), 607–609. (<a
href="https://doi.org/10.1038/s41587-023-01746-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kadakia, Kushal T. and Rathi, Vinay K. and Ramachandran, Reshma and Johnston, James L. and Ross, Joseph S. and Dhruva, Sanket S.},
  doi          = {10.1038/s41587-023-01746-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {607-609},
  shortjournal = {Nature Biotech.},
  title        = {Challenges and solutions to advancing health equity with medical devices},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The scverse project provides a computational ecosystem for
single-cell omics data analysis. <em>NBT</em>, <em>41</em>(5), 604–606.
(<a href="https://doi.org/10.1038/s41587-023-01733-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Virshup, Isaac and Bredikhin, Danila and Heumos, Lukas and Palla, Giovanni and Sturm, Gregor and Gayoso, Adam and Kats, Ilia and Koutrouli, Mikaela and Berger, Bonnie and Pe’er, Dana and Regev, Aviv and Teichmann, Sarah A. and Finotello, Francesca and Wolf, F. Alexander and Yosef, Nir and Stegle, Oliver and Theis, Fabian J.},
  doi          = {10.1038/s41587-023-01733-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {604-606},
  shortjournal = {Nature Biotech.},
  title        = {The scverse project provides a computational ecosystem for single-cell omics data analysis},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). 1Q23 — biotech financing woes continue. <em>NBT</em>,
<em>41</em>(5), 594–596. (<a
href="https://doi.org/10.1038/s41587-023-01776-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-01776-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {594-596},
  shortjournal = {Nature Biotech.},
  title        = {1Q23 — biotech financing woes continue},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Drug pipeline 1Q23 — everything everywhere all over the
place. <em>NBT</em>, <em>41</em>(5), 591–593. (<a
href="https://doi.org/10.1038/s41587-023-01775-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-01775-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {591-593},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 1Q23 — everything everywhere all over the place},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(5), 590. (<a
href="https://doi.org/10.1038/s41587-023-01786-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01786-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {590},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). FDA approves cord-blood therapy. <em>NBT</em>,
<em>41</em>(5), 589. (<a
href="https://doi.org/10.1038/s41587-023-01808-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01808-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {589},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves cord-blood therapy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CAR-t cells for solid tumors. <em>NBT</em>, <em>41</em>(5),
588. (<a href="https://doi.org/10.1038/s41587-023-01803-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01803-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {588},
  shortjournal = {Nature Biotech.},
  title        = {CAR-T cells for solid tumors},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). NASH therapies head toward landmark approval. <em>NBT</em>,
<em>41</em>(5), 587–590. (<a
href="https://doi.org/10.1038/s41587-023-01787-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drugs targeting metabolic drivers of fatty liver disease show promise at resolving inflammation, reducing damage and reversing fibrosis.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-023-01787-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {587-590},
  shortjournal = {Nature Biotech.},
  title        = {NASH therapies head toward landmark approval},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Elizabeth holmes prison delayed. <em>NBT</em>,
<em>41</em>(5), 587. (<a
href="https://doi.org/10.1038/s41587-023-01802-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01802-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {587},
  shortjournal = {Nature Biotech.},
  title        = {Elizabeth holmes prison delayed},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The promise of phages. <em>NBT</em>, <em>41</em>(5), 583.
(<a href="https://doi.org/10.1038/s41587-023-01807-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phage therapy is an exciting alternative to antibiotics. Why now?},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01807-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {583},
  shortjournal = {Nature Biotech.},
  title        = {The promise of phages},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Precision financing. <em>NBT</em>, <em>41</em>(5), 597–603.
(<a href="https://doi.org/10.1038/d41587-023-00001-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Private biotech financing in 2023 is more discriminate, disciplined and demanding. Strong data and efficiency are de rigueur. Private biotech financing in 2023 is more discriminate, disciplined and demanding. Strong data and efficiency are de rigueur.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/d41587-023-00001-z},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {5},
  pages        = {597-603},
  shortjournal = {Nature Biotech.},
  title        = {Precision financing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Drug discovery companies are customizing ChatGPT: Here’s
how. <em>NBT</em>, <em>41</em>(5), 585–586. (<a
href="https://doi.org/10.1038/s41587-023-01788-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models are helping scientists to converse with artificial intelligence and even to generate potential drug targets.},
  archive      = {J_NBT},
  author       = {Savage, Neil},
  doi          = {10.1038/s41587-023-01788-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {5},
  pages        = {585-586},
  shortjournal = {Nature Biotech.},
  title        = {Drug discovery companies are customizing ChatGPT: Here’s how},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). People. <em>NBT</em>, <em>41</em>(4), 582. (<a
href="https://doi.org/10.1038/s41587-023-01741-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01741-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {582},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integrating a healthcare innovation bootcamp into an
international medical conference to democratize innovation learning.
<em>NBT</em>, <em>41</em>(4), 579–581. (<a
href="https://doi.org/10.1038/s41587-023-01738-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integration of an intensive innovation education bootcamp into a large international conference demonstrates statistically significant increases in participant understanding and knowledge of healthcare innovation.},
  archive      = {J_NBT},
  author       = {Succi, Marc D. and Cheng, Debby and Andriole, Katherine P. and Fintelmann, Florian J. and Flores, Efren J. and Zhang, Haipeng (Mark) and Gee, Michael S. and Coburn, Christopher M. and Brink, James A.},
  doi          = {10.1038/s41587-023-01738-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {579-581},
  shortjournal = {Nature Biotech.},
  title        = {Integrating a healthcare innovation bootcamp into an international medical conference to democratize innovation learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Publisher correction: Butler enables rapid cloud-based
analysis of thousands of human genomes. <em>NBT</em>, <em>41</em>(4),
578. (<a href="https://doi.org/10.1038/s41587-020-0448-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Yakneen, Sergei and Waszak, Sebastian M. and Gertz, Michael and Korbel, Jan O.},
  doi          = {10.1038/s41587-020-0448-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {578},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Butler enables rapid cloud-based analysis of thousands of human genomes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Butler enables rapid cloud-based
analysis of thousands of human genomes. <em>NBT</em>, <em>41</em>(4),
577. (<a href="https://doi.org/10.1038/s41587-022-01554-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yakneen, Sergei and Waszak, Sebastian M. and Gertz, Michael and Korbel, Jan O.},
  doi          = {10.1038/s41587-022-01554-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {577},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Butler enables rapid cloud-based analysis of thousands of human genomes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Modular, programmable RNA sensing using
ADAR editing in living cells. <em>NBT</em>, <em>41</em>(4), 577. (<a
href="https://doi.org/10.1038/s41587-022-01617-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kaseniit, K. Eerik and Katz, Noa and Kolber, Natalie S. and Call, Connor C. and Wengier, Diego L. and Cody, Will B. and Sattely, Elizabeth S. and Gao, Xiaojing J.},
  doi          = {10.1038/s41587-022-01617-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {577},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Modular, programmable RNA sensing using ADAR editing in living cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). MINSTED nanoscopy enters the ångström localization range.
<em>NBT</em>, <em>41</em>(4), 569–576. (<a
href="https://doi.org/10.1038/s41587-022-01519-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Super-resolution techniques have achieved localization precisions in the nanometer regime. Here we report all-optical, room temperature localization of fluorophores with precision in the Ångström range. We built on the concept of MINSTED nanoscopy where precision is increased by encircling the fluorophore with the low-intensity central region of a stimulated emission depletion (STED) donut beam while constantly increasing the absolute donut power. By blue-shifting the STED beam and separating fluorophores by on/off switching, individual fluorophores bound to a DNA strand are localized with σ = 4.7 Å, corresponding to a fraction of the fluorophore size, with only 2,000 detected photons. MINSTED fluorescence nanoscopy with single-digit nanometer resolution is exemplified by imaging nuclear pore complexes and the distribution of nuclear lamin in mammalian cells labeled by transient DNA hybridization. Because our experiments yield a localization precision σ = 2.3 Å, estimated for 10,000 detected photons, we anticipate that MINSTED will open up new areas of application in the study of macromolecular complexes in cells. Sub-nanometer localization precision is achieved with all-optical microscopy.},
  archive      = {J_NBT},
  author       = {Weber, Michael and von der Emde, Henrik and Leutenegger, Marcel and Gunkel, Philip and Sambandan, Sivakumar and Khan, Taukeer A. and Keller-Findeisen, Jan and Cordes, Volker C. and Hell, Stefan W.},
  doi          = {10.1038/s41587-022-01519-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {569-576},
  shortjournal = {Nature Biotech.},
  title        = {MINSTED nanoscopy enters the Ångström localization range},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An engineered t7 RNA polymerase that produces mRNA free of
immunostimulatory byproducts. <em>NBT</em>, <em>41</em>(4), 560–568. (<a
href="https://doi.org/10.1038/s41587-022-01525-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vitro transcription (IVT) is a DNA-templated process for synthesizing long RNA transcripts, including messenger RNA (mRNA). For many research and commercial applications, IVT of mRNA is typically performed using bacteriophage T7 RNA polymerase (T7 RNAP) owing to its ability to produce full-length RNA transcripts with high fidelity; however, T7 RNAP can also produce immunostimulatory byproducts such as double-stranded RNA that can affect protein expression. Such byproducts require complex purification processes, using methods such as reversed-phase high-performance liquid chromatography, to yield safe and effective mRNA-based medicines. To minimize the need for downstream purification processes, we rationally and computationally engineered a double mutant of T7 RNAP that produces substantially less immunostimulatory RNA during IVT compared with wild-type T7 RNAP. The resulting mutant allows for a simplified production process with similar mRNA potency, lower immunostimulatory content and quicker manufacturing time compared with wild-type T7 RNAP. Herein, we describe the computational design and development of this improved T7 RNAP variant. The purity of synthetic mRNA is improved with a double-mutant T7 RNA polymerase.},
  archive      = {J_NBT},
  author       = {Dousis, Athanasios and Ravichandran, Kanchana and Hobert, Elissa M. and Moore, Melissa J. and Rabideau, Amy E.},
  doi          = {10.1038/s41587-022-01525-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {560-568},
  shortjournal = {Nature Biotech.},
  title        = {An engineered t7 RNA polymerase that produces mRNA free of immunostimulatory byproducts},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Extending fluorescence anisotropy to large complexes using
reversibly switchable proteins. <em>NBT</em>, <em>41</em>(4), 552–559.
(<a href="https://doi.org/10.1038/s41587-022-01489-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The formation of macromolecular complexes can be measured by detection of changes in rotational mobility using time-resolved fluorescence anisotropy. However, this method is limited to relatively small molecules (~0.1–30 kDa), excluding the majority of the human proteome and its complexes. We describe selective time-resolved anisotropy with reversibly switchable states (STARSS), which overcomes this limitation and extends the observable mass range by more than three orders of magnitude. STARSS is based on long-lived reversible molecular transitions of switchable fluorescent proteins to resolve the relatively slow rotational diffusivity of large complexes. We used STARSS to probe the rotational mobility of several molecular complexes in cells, including chromatin, the retroviral Gag lattice and activity-regulated cytoskeleton-associated protein oligomers. Because STARSS can probe arbitrarily large structures, it is generally applicable to the entire human proteome. STARSS measures rotational dynamics of large molecules and their complexes.},
  archive      = {J_NBT},
  author       = {Volpato, Andrea and Ollech, Dirk and Alvelid, Jonatan and Damenti, Martina and Müller, Barbara and York, Andrew G and Ingaramo, Maria and Testa, Ilaria},
  doi          = {10.1038/s41587-022-01489-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {552-559},
  shortjournal = {Nature Biotech.},
  title        = {Extending fluorescence anisotropy to large complexes using reversibly switchable proteins},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Targeting MYC with modular synthetic transcriptional
repressors derived from bHLH DNA-binding domains. <em>NBT</em>,
<em>41</em>(4), 541–551. (<a
href="https://doi.org/10.1038/s41587-022-01504-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite unequivocal roles in disease, transcription factors (TFs) remain largely untapped as pharmacologic targets due to the challenges in targeting protein–protein and protein–DNA interactions. Here we report a chemical strategy to generate modular synthetic transcriptional repressors (STRs) derived from the bHLH domain of MAX. Our synthetic approach yields chemically stabilized tertiary domain mimetics that cooperatively bind the MYC/MAX consensus E-box motif with nanomolar affinity, exhibit specificity that is equivalent to or beyond that of full-length TFs and directly compete with MYC/MAX protein for DNA binding. A lead STR directly inhibits MYC binding in cells, downregulates MYC-dependent expression programs at the proteome level and inhibits MYC-dependent cell proliferation. Co-crystallization and structure determination of a STR:E-box DNA complex confirms retention of DNA recognition in a near identical manner as full-length bHLH TFs. We additionally demonstrate structure-blind design of STRs derived from alternative bHLH-TFs, confirming that STRs can be used to develop highly specific mimetics of TFs targeting other gene regulatory elements. Synthetic transcriptional repressors are engineered from natural DNA-binding domains.},
  archive      = {J_NBT},
  author       = {Speltz, Thomas E. and Qiao, Zeyu and Swenson, Colin S. and Shangguan, Xianghang and Coukos, John S. and Lee, Christopher W. and Thomas, Deborah M. and Santana, Jesse and Fanning, Sean W. and Greene, Geoffrey L. and Moellering, Raymond E.},
  doi          = {10.1038/s41587-022-01504-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {541-551},
  shortjournal = {Nature Biotech.},
  title        = {Targeting MYC with modular synthetic transcriptional repressors derived from bHLH DNA-binding domains},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A split, conditionally active mimetic of IL-2 reduces the
toxicity of systemic cytokine therapy. <em>NBT</em>, <em>41</em>(4),
532–540. (<a href="https://doi.org/10.1038/s41587-022-01510-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The therapeutic potential of recombinant cytokines has been limited by the severe side effects of systemic administration. We describe a strategy to reduce the dose-limiting toxicities of monomeric cytokines by designing two components that require colocalization for activity and that can be independently targeted to restrict activity to cells expressing two surface markers. We demonstrate the approach with a previously designed mimetic of cytokines interleukin-2 and interleukin-15—Neoleukin-2/15 (Neo-2/15)—both for trans-activating immune cells surrounding targeted tumor cells and for cis-activating directly targeted immune cells. In trans-activation mode, tumor antigen targeting of the two components enhanced antitumor activity and attenuated toxicity compared with systemic treatment in syngeneic mouse melanoma models. In cis-activation mode, immune cell targeting of the two components selectively expanded CD8+ T cells in a syngeneic mouse melanoma model and promoted chimeric antigen receptor T cell activation in a lymphoma xenograft model, enhancing antitumor efficacy in both cases. IL-2 cytokine therapy is made safer by splitting a designed mimetic in two for tumor targeting.},
  archive      = {J_NBT},
  author       = {Quijano-Rubio, Alfredo and Bhuiyan, Aladdin M. and Yang, Huilin and Leung, Isabel and Bello, Elisa and Ali, Lestat R. and Zhangxu, Kevin and Perkins, Jilliane and Chun, Jung-Ho and Wang, Wentao and Lajoie, Marc J. and Ravichandran, Rashmi and Kuo, Yun-Huai and Dougan, Stephanie K. and Riddell, Stanley R. and Spangler, Jamie B. and Dougan, Michael and Silva, Daniel-Adriano and Baker, David},
  doi          = {10.1038/s41587-022-01510-z},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {532-540},
  shortjournal = {Nature Biotech.},
  title        = {A split, conditionally active mimetic of IL-2 reduces the toxicity of systemic cytokine therapy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-yield genome engineering in primary cells using a
hybrid ssDNA repair template and small-molecule cocktails. <em>NBT</em>,
<em>41</em>(4), 521–531. (<a
href="https://doi.org/10.1038/s41587-022-01418-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enhancing CRISPR-mediated site-specific transgene insertion efficiency by homology-directed repair (HDR) using high concentrations of double-stranded DNA (dsDNA) with Cas9 target sequences (CTSs) can be toxic to primary cells. Here, we develop single-stranded DNA (ssDNA) HDR templates (HDRTs) incorporating CTSs with reduced toxicity that boost knock-in efficiency and yield by an average of around two- to threefold relative to dsDNA CTSs. Using small-molecule combinations that enhance HDR, we could further increase knock-in efficiencies by an additional roughly two- to threefold on average. Our method works across a variety of target loci, knock-in constructs and primary human cell types, reaching HDR efficiencies of &amp;gt;80–90\%. We demonstrate application of this approach for both pathogenic gene variant modeling and gene-replacement strategies for IL2RA and CTLA4 mutations associated with Mendelian disorders. Finally, we develop a good manufacturing practice (GMP)-compatible process for nonviral chimeric antigen receptor-T cell manufacturing, with knock-in efficiencies (46–62\%) and yields (&amp;gt;1.5 × 109 modified cells) exceeding those of conventional approaches. Combinations of single-stranded DNA repair templates and small molecules markedly enhance genome editing.},
  archive      = {J_NBT},
  author       = {Shy, Brian R. and Vykunta, Vivasvan S. and Ha, Alvin and Talbot, Alexis and Roth, Theodore L. and Nguyen, David N. and Pfeifer, Wolfgang G. and Chen, Yan Yi and Blaeschke, Franziska and Shifrut, Eric and Vedova, Shane and Mamedov, Murad R. and Chung, Jing-Yi Jing and Li, Hong and Yu, Ruby and Wu, David and Wolf, Jeffrey and Martin, Thomas G. and Castro, Carlos E. and Ye, Lumeng and Esensten, Jonathan H. and Eyquem, Justin and Marson, Alexander},
  doi          = {10.1038/s41587-022-01418-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {521-531},
  shortjournal = {Nature Biotech.},
  title        = {High-yield genome engineering in primary cells using a hybrid ssDNA repair template and small-molecule cocktails},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial mapping of the total transcriptome by in situ
polyadenylation. <em>NBT</em>, <em>41</em>(4), 513–520. (<a
href="https://doi.org/10.1038/s41587-022-01517-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics reveals the spatial context of gene expression, but current methods are limited to assaying polyadenylated (A-tailed) RNA transcripts. Here we demonstrate that enzymatic in situ polyadenylation of RNA enables detection of the full spectrum of RNAs, expanding the scope of sequencing-based spatial transcriptomics to the total transcriptome. We demonstrate that our spatial total RNA-sequencing (STRS) approach captures coding RNAs, noncoding RNAs and viral RNAs. We apply STRS to study skeletal muscle regeneration and viral-induced myocarditis. Our analyses reveal the spatial patterns of noncoding RNA expression with near-cellular resolution, identify spatially defined expression of noncoding transcripts in skeletal muscle regeneration and highlight host transcriptional responses associated with local viral RNA abundance. STRS requires adding only one step to the widely used Visium spatial total RNA-sequencing protocol from 10x Genomics, and thus could be easily adopted to enable new insights into spatial gene regulation and biology. Spatial RNA sequencing is extended beyond poly-A transcripts to capture the full transcriptome.},
  archive      = {J_NBT},
  author       = {McKellar, David W. and Mantri, Madhav and Hinchman, Meleana M. and Parker, John S. L. and Sethupathy, Praveen and Cosgrove, Benjamin D. and De Vlaminck, Iwijn},
  doi          = {10.1038/s41587-022-01517-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {513-520},
  shortjournal = {Nature Biotech.},
  title        = {Spatial mapping of the total transcriptome by in situ polyadenylation},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Drag-and-drop genome insertion of large sequences without
double-strand DNA cleavage using CRISPR-directed integrases.
<em>NBT</em>, <em>41</em>(4), 500–512. (<a
href="https://doi.org/10.1038/s41587-022-01527-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Programmable genome integration of large, diverse DNA cargo without DNA repair of exposed DNA double-strand breaks remains an unsolved challenge in genome editing. We present programmable addition via site-specific targeting elements (PASTE), which uses a CRISPR–Cas9 nickase fused to both a reverse transcriptase and serine integrase for targeted genomic recruitment and integration of desired payloads. We demonstrate integration of sequences as large as ~36 kilobases at multiple genomic loci across three human cell lines, primary T cells and non-dividing primary human hepatocytes. To augment PASTE, we discovered 25,614 serine integrases and cognate attachment sites from metagenomes and engineered orthologs with higher activity and shorter recognition sequences for efficient programmable integration. PASTE has editing efficiencies similar to or exceeding those of homology-directed repair and non-homologous end joining-based methods, with activity in non-dividing cells and in vivo with fewer detectable off-target events. PASTE expands the capabilities of genome editing by allowing large, multiplexed gene insertion without reliance on DNA repair pathways. Large sequences are integrated site specifically into the human genome without double-strand DNA cleavage.},
  archive      = {J_NBT},
  author       = {Yarnall, Matthew T. N. and Ioannidi, Eleonora I. and Schmitt-Ulms, Cian and Krajeski, Rohan N. and Lim, Justin and Villiger, Lukas and Zhou, Wenyuan and Jiang, Kaiyi and Garushyants, Sofya K. and Roberts, Nathaniel and Zhang, Liyang and Vakulskas, Christopher A. and Walker, John A. and Kadina, Anastasia P. and Zepeda, Adrianna E. and Holden, Kevin and Ma, Hong and Xie, Jun and Gao, Guangping and Foquet, Lander and Bial, Greg and Donnelly, Sara K. and Miyata, Yoshinari and Radiloff, Daniel R. and Henderson, Jordana M. and Ujita, Andrew and Abudayyeh, Omar O. and Gootenberg, Jonathan S.},
  doi          = {10.1038/s41587-022-01527-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {500-512},
  shortjournal = {Nature Biotech.},
  title        = {Drag-and-drop genome insertion of large sequences without double-strand DNA cleavage using CRISPR-directed integrases},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Systematic discovery of recombinases for efficient
integration of large DNA sequences into the human genome. <em>NBT</em>,
<em>41</em>(4), 488–499. (<a
href="https://doi.org/10.1038/s41587-022-01494-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large serine recombinases (LSRs) are DNA integrases that facilitate the site-specific integration of mobile genetic elements into bacterial genomes. Only a few LSRs, such as Bxb1 and PhiC31, have been characterized to date, with limited efficiency as tools for DNA integration in human cells. In this study, we developed a computational approach to identify thousands of LSRs and their DNA attachment sites, expanding known LSR diversity by &amp;gt;100-fold and enabling the prediction of their insertion site specificities. We tested their recombination activity in human cells, classifying them as landing pad, genome-targeting or multi-targeting LSRs. Overall, we achieved up to seven-fold higher recombination than Bxb1 and genome integration efficiencies of 40–75\% with cargo sizes over 7 kb. We also demonstrate virus-free, direct integration of plasmid or amplicon libraries for improved functional genomics applications. This systematic discovery of recombinases directly from microbial sequencing data provides a resource of over 60 LSRs experimentally characterized in human cells for large-payload genome insertion without exposed DNA double-stranded breaks. Screening recombinases identifies tools for inserting large sequences into the human genome.},
  archive      = {J_NBT},
  author       = {Durrant, Matthew G. and Fanton, Alison and Tycko, Josh and Hinks, Michaela and Chandrasekaran, Sita S. and Perry, Nicholas T. and Schaepe, Julia and Du, Peter P. and Lotfy, Peter and Bassik, Michael C. and Bintu, Lacramioara and Bhatt, Ami S. and Hsu, Patrick D.},
  doi          = {10.1038/s41587-022-01494-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {488-499},
  shortjournal = {Nature Biotech.},
  title        = {Systematic discovery of recombinases for efficient integration of large DNA sequences into the human genome},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Modular, programmable RNA sensing using ADAR editing in
living cells. <em>NBT</em>, <em>41</em>(4), 482–487. (<a
href="https://doi.org/10.1038/s41587-022-01493-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the increasing availability of single-cell transcriptomes, RNA signatures offer a promising basis for targeting living cells. Molecular RNA sensors would enable the study of and therapeutic interventions for specific cell types/states in diverse contexts, particularly in human patients and non-model organisms. Here we describe a modular, programmable system for live RNA sensing using adenosine deaminases acting on RNA (RADAR). We validate, and then expand, our basic design, characterize its performance, and analyze its compatibility with human and mouse transcriptomes. We identify strategies to boost output levels and improve the dynamic range. Additionally, we show that RADAR enables compact AND logic. In addition to responding to transcript levels, RADAR can distinguish disease-relevant sequence alterations of transcript identities, such as point mutations and fusions. Finally, we demonstrate that RADAR is a self-contained system with the potential to function in diverse organisms. Specific RNAs are detected in live cells by harnessing endogenous ADAR enzymes.},
  archive      = {J_NBT},
  author       = {Kaseniit, K. Eerik and Katz, Noa and Kolber, Natalie S. and Call, Connor C. and Wengier, Diego L. and Cody, Will B. and Sattely, Elizabeth S. and Gao, Xiaojing J.},
  doi          = {10.1038/s41587-022-01493-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {482-487},
  shortjournal = {Nature Biotech.},
  title        = {Modular, programmable RNA sensing using ADAR editing in living cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Synthetic biologics provide a path to target bHLH
transcription factors. <em>NBT</em>, <em>41</em>(4), 480–481. (<a
href="https://doi.org/10.1038/s41587-022-01509-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to target the majority of human transcription factors remains an as yet unachieved goal in chemical biology and medicine. We developed a modular, synthetic transcriptional repressor platform that recapitulates the DNA-binding properties of native basic helix-loop-helix (bHLH) domains and can block the DNA-binding functions of an important oncogenic transcription factor target, MYC.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01509-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {480-481},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic biologics provide a path to target bHLH transcription factors},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Improving non-viral knock-in with modified single-stranded
DNAs and small molecules. <em>NBT</em>, <em>41</em>(4), 478–479. (<a
href="https://doi.org/10.1038/s41587-022-01420-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The addition of Cas9 target sequences to long single-stranded DNAs, combined with cocktails of small molecules to boost homology-directed repair, leads to marked enhancement of non-viral knock-in efficiency and yield in primary human T cells and other hematopoietic cell types.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01420-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {478-479},
  shortjournal = {Nature Biotech.},
  title        = {Improving non-viral knock-in with modified single-stranded DNAs and small molecules},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial total RNA-sequencing maps coding, noncoding and
viral RNAs in tissues. <em>NBT</em>, <em>41</em>(4), 476–477. (<a
href="https://doi.org/10.1038/s41587-022-01562-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial total RNA-sequencing (STRS) combines in situ polyadenylation with existing spatial transcriptomics technologies to enable a broader view of the transcriptome in tissues. We use STRS to spatially map coding, noncoding and nonhost RNAs in models of skeletal muscle regeneration and viral myocarditis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01562-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {476-477},
  shortjournal = {Nature Biotech.},
  title        = {Spatial total RNA-sequencing maps coding, noncoding and viral RNAs in tissues},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Optical microscopy gets down to angstroms. <em>NBT</em>,
<em>41</em>(4), 473–474. (<a
href="https://doi.org/10.1038/s41587-022-01544-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Super-resolution microscopy at ångström precision could pave the way to optical structural biology in cells.},
  archive      = {J_NBT},
  author       = {Tortarolo, Giorgio and Manley, Suliana},
  doi          = {10.1038/s41587-022-01544-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {473-474},
  shortjournal = {Nature Biotech.},
  title        = {Optical microscopy gets down to angstroms},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Novel recombinases for large DNA insertions. <em>NBT</em>,
<em>41</em>(4), 471–472. (<a
href="https://doi.org/10.1038/s41587-022-01600-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioinformatic mining and engineering of serine recombinases yields improved genome editing tools.},
  archive      = {J_NBT},
  author       = {Lampe, George D. and Sternberg, Samuel H.},
  doi          = {10.1038/s41587-022-01600-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {471-472},
  shortjournal = {Nature Biotech.},
  title        = {Novel recombinases for large DNA insertions},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023o). Retrofitted e. Coli genome confers phage resistance.
<em>NBT</em>, <em>41</em>(4), 470. (<a
href="https://doi.org/10.1038/s41587-023-01764-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01764-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {470},
  shortjournal = {Nature Biotech.},
  title        = {Retrofitted e. coli genome confers phage resistance},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Long COVID immune phenotypes identified for blockade.
<em>NBT</em>, <em>41</em>(4), 470. (<a
href="https://doi.org/10.1038/s41587-023-01765-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01765-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {470},
  shortjournal = {Nature Biotech.},
  title        = {Long COVID immune phenotypes identified for blockade},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Base editor restores CD3δ-dependent SCID mutation.
<em>NBT</em>, <em>41</em>(4), 470. (<a
href="https://doi.org/10.1038/s41587-023-01768-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01768-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {470},
  shortjournal = {Nature Biotech.},
  title        = {Base editor restores CD3δ-dependent SCID mutation},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Microfluidics. <em>NBT</em>, <em>41</em>(4), 469. (<a
href="https://doi.org/10.1038/s41587-023-01739-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to microfluidic devices and methods for high-throughput phenotyping and sample preparation methods.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01739-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {469},
  shortjournal = {Nature Biotech.},
  title        = {Microfluidics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping the patent landscape of medical machine learning.
<em>NBT</em>, <em>41</em>(4), 461–468. (<a
href="https://doi.org/10.1038/s41587-023-01735-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patent office data show robust and rising patenting of AI inventions in the medical field, contrary to fears that medical machine learning patents might be largely unavailable because of challenges to their subject-matter eligibility.},
  archive      = {J_NBT},
  author       = {Aboy, Mateo and Price, W. Nicholson and Raker, Seth},
  doi          = {10.1038/s41587-023-01735-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {461-468},
  shortjournal = {Nature Biotech.},
  title        = {Mapping the patent landscape of medical machine learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Xenotransplantation: How close are we? <em>NBT</em>,
<em>41</em>(4), 452–460. (<a
href="https://doi.org/10.1038/s41587-023-01730-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A group of experts reflects on what was learned from the first human transplants of genetically engineered pig organs and what the future of xenotransplantation may hold.},
  archive      = {J_NBT},
  author       = {Aschheim, Kathryn and DeFrancesco, Laura},
  doi          = {10.1038/s41587-023-01730-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {452-460},
  shortjournal = {Nature Biotech.},
  title        = {Xenotransplantation: How close are we?},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Make science disruptive again. <em>NBT</em>, <em>41</em>(4),
450–451. (<a href="https://doi.org/10.1038/s41587-023-01736-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yanai, Itai and Lercher, Martin J.},
  doi          = {10.1038/s41587-023-01736-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {450-451},
  shortjournal = {Nature Biotech.},
  title        = {Make science disruptive again},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integrative analysis of multimodal mass spectrometry data in
MZmine 3. <em>NBT</em>, <em>41</em>(4), 447–449. (<a
href="https://doi.org/10.1038/s41587-023-01690-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Schmid, Robin and Heuckeroth, Steffen and Korf, Ansgar and Smirnov, Aleksandr and Myers, Owen and Dyrlund, Thomas S. and Bushuiev, Roman and Murray, Kevin J. and Hoffmann, Nils and Lu, Miaoshan and Sarvepalli, Abinesh and Zhang, Zheng and Fleischauer, Markus and Dührkop, Kai and Wesner, Mark and Hoogstra, Shawn J. and Rudt, Edward and Mokshyna, Olena and Brungs, Corinna and Ponomarov, Kirill and Mutabdžija, Lana and Damiani, Tito and Pudney, Chris J. and Earll, Mark and Helmer, Patrick O. and Fallon, Timothy R. and Schulze, Tobias and Rivas-Ubach, Albert and Bilbao, Aivett and Richter, Henning and Nothias, Louis-Félix and Wang, Mingxun and Orešič, Matej and Weng, Jing-Ke and Böcker, Sebastian and Jeibmann, Astrid and Hayen, Heiko and Karst, Uwe and Dorrestein, Pieter C. and Petras, Daniel and Du, Xiuxia and Pluskal, Tomáš},
  doi          = {10.1038/s41587-023-01690-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {447-449},
  shortjournal = {Nature Biotech.},
  title        = {Integrative analysis of multimodal mass spectrometry data in MZmine 3},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(4), 440. (<a
href="https://doi.org/10.1038/s41587-023-01742-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01742-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {440},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biotech mega-deals boost pfizer, sanofi pipelines.
<em>NBT</em>, <em>41</em>(4), 439. (<a
href="https://doi.org/10.1038/s41587-023-01761-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01761-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {439},
  shortjournal = {Nature Biotech.},
  title        = {Biotech mega-deals boost pfizer, sanofi pipelines},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Microbiome-friendly phages join the campaign for better
antimicrobials. <em>NBT</em>, <em>41</em>(4), 438–440. (<a
href="https://doi.org/10.1038/s41587-023-01732-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biotech companies are racing to test bacteriophages — some as found in nature, others armed with CRISPR–Cas — to destroy drug-resistant bacteria selectively while keeping the microbiome intact.},
  archive      = {J_NBT},
  author       = {Johnson, Ben},
  doi          = {10.1038/s41587-023-01732-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {438-440},
  shortjournal = {Nature Biotech.},
  title        = {Microbiome-friendly phages join the campaign for better antimicrobials},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). EU gets first hemophilia b gene therapy. <em>NBT</em>,
<em>41</em>(4), 438. (<a
href="https://doi.org/10.1038/s41587-023-01760-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01760-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {438},
  shortjournal = {Nature Biotech.},
  title        = {EU gets first hemophilia b gene therapy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Silicon valley bank collapse sends shivers down biotech
spines. <em>NBT</em>, <em>41</em>(4), 437. (<a
href="https://doi.org/10.1038/s41587-023-01762-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01762-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {437},
  shortjournal = {Nature Biotech.},
  title        = {Silicon valley bank collapse sends shivers down biotech spines},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Genetic medicines aim straight for the heart. <em>NBT</em>,
<em>41</em>(4), 435–437. (<a
href="https://doi.org/10.1038/s41587-023-01745-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Promising gene therapy, gene editing and antisense oligonucleotide approaches herald a new era of cardiac medicine for patients with certain so-far intractable heart conditions.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-023-01745-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {435-437},
  shortjournal = {Nature Biotech.},
  title        = {Genetic medicines aim straight for the heart},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Data sharing in the age of deep learning. <em>NBT</em>,
<em>41</em>(4), 433. (<a
href="https://doi.org/10.1038/s41587-023-01770-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How can we protect personal information and the integrity of artificial intelligence models when sharing data?},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01770-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {433},
  shortjournal = {Nature Biotech.},
  title        = {Data sharing in the age of deep learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Personalized medicine is having its day. <em>NBT</em>,
<em>41</em>(4), 441–446. (<a
href="https://doi.org/10.1038/s41587-023-01724-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {From made-to-order genetic therapies to model organisms engineered to be ‘patient avatars’, the technology exists right now to save patients with rare diseases.},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-023-01724-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {4},
  pages        = {441-446},
  shortjournal = {Nature Biotech.},
  title        = {Personalized medicine is having its day},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). People. <em>NBT</em>, <em>41</em>(3), 432. (<a
href="https://doi.org/10.1038/s41587-023-01706-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01706-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {432},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Longitudinal data support university-based biomedical
entrepreneurship education programs. <em>NBT</em>, <em>41</em>(3),
429–431. (<a href="https://doi.org/10.1038/s41587-023-01701-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Early-stage researchers can greatly benefit from education and skills development in venture creation and commercialization of discoveries.},
  archive      = {J_NBT},
  author       = {Vizgan, Gabriel and Hill-Whilton, Zachary and Achuonjei, Joy and Schweickart, Tucker and Chitale, Sadhana and Gillespie, Colleen and Gold-von Simson, Gabrielle},
  doi          = {10.1038/s41587-023-01701-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {429-431},
  shortjournal = {Nature Biotech.},
  title        = {Longitudinal data support university-based biomedical entrepreneurship education programs},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Haplotype-aware analysis of somatic copy number variations
from single-cell transcriptomes. <em>NBT</em>, <em>41</em>(3), 417–426.
(<a href="https://doi.org/10.1038/s41587-022-01468-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome instability and aberrant alterations of transcriptional programs both play important roles in cancer. Single-cell RNA sequencing (scRNA-seq) has the potential to investigate both genetic and nongenetic sources of tumor heterogeneity in a single assay. Here we present a computational method, Numbat, that integrates haplotype information obtained from population-based phasing with allele and expression signals to enhance detection of copy number variations from scRNA-seq. Numbat exploits the evolutionary relationships between subclones to iteratively infer single-cell copy number profiles and tumor clonal phylogeny. Analysis of 22 tumor samples, including multiple myeloma, gastric, breast and thyroid cancers, shows that Numbat can reconstruct the tumor copy number profile and precisely identify malignant cells in the tumor microenvironment. We identify genetic subpopulations with transcriptional signatures relevant to tumor progression and therapy resistance. Numbat requires neither sample-matched DNA data nor a priori genotyping, and is applicable to a wide range of experimental settings and cancer types. Numbat integrates expression, allele and haplotype information to detect copy number variations in scRNA-seq.},
  archive      = {J_NBT},
  author       = {Gao, Teng and Soldatov, Ruslan and Sarkar, Hirak and Kurkiewicz, Adam and Biederstedt, Evan and Loh, Po-Ru and Kharchenko, Peter V.},
  doi          = {10.1038/s41587-022-01468-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {417-426},
  shortjournal = {Nature Biotech.},
  title        = {Haplotype-aware analysis of somatic copy number variations from single-cell transcriptomes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precise DNA cleavage using CRISPR-SpRYgests. <em>NBT</em>,
<em>41</em>(3), 409–416. (<a
href="https://doi.org/10.1038/s41587-022-01492-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods for in vitro DNA cleavage and molecular cloning remain unable to precisely cleave DNA directly adjacent to bases of interest. Restriction enzymes (REs) must bind specific motifs, whereas wild-type CRISPR–Cas9 or CRISPR–Cas12 nucleases require protospacer adjacent motifs (PAMs). Here we explore the utility of our previously reported near-PAMless SpCas9 variant, named SpRY, to serve as a universal DNA cleavage tool for various cloning applications. By performing SpRY DNA digests (SpRYgests) using more than 130 guide RNAs (gRNAs) sampling a wide diversity of PAMs, we discovered that SpRY is PAMless in vitro and can cleave DNA at practically any sequence, including sites refractory to cleavage with wild-type SpCas9. We illustrate the versatility and effectiveness of SpRYgests to improve the precision of several cloning workflows, including those not possible with REs or canonical CRISPR nucleases. We also optimize a rapid and simple one-pot gRNA synthesis protocol to streamline SpRYgest implementation. Together, SpRYgests can improve various DNA engineering applications that benefit from precise DNA breaks. A PAMless CRISPR nuclease is applied for precision DNA cleavage and cloning.},
  archive      = {J_NBT},
  author       = {Christie, Kathleen A. and Guo, Jimmy A. and Silverstein, Rachel A. and Doll, Roman M. and Mabuchi, Megumu and Stutzman, Hannah E. and Lin, Jiecong and Ma, Linyuan and Walton, Russell T. and Pinello, Luca and Robb, G. Brett and Kleinstiver, Benjamin P.},
  doi          = {10.1038/s41587-022-01492-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {409-416},
  shortjournal = {Nature Biotech.},
  title        = {Precise DNA cleavage using CRISPR-SpRYgests},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Discovery of drug–omics associations in type 2 diabetes
with generative deep-learning models. <em>NBT</em>, <em>41</em>(3),
399–408. (<a href="https://doi.org/10.1038/s41587-022-01520-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The application of multiple omics technologies in biomedical cohorts has the potential to reveal patient-level disease characteristics and individualized response to treatment. However, the scale and heterogeneous nature of multi-modal data makes integration and inference a non-trivial task. We developed a deep-learning-based framework, multi-omics variational autoencoders (MOVE), to integrate such data and applied it to a cohort of 789 people with newly diagnosed type 2 diabetes with deep multi-omics phenotyping from the DIRECT consortium. Using in silico perturbations, we identified drug–omics associations across the multi-modal datasets for the 20 most prevalent drugs given to people with type 2 diabetes with substantially higher sensitivity than univariate statistical tests. From these, we among others, identified novel associations between metformin and the gut microbiota as well as opposite molecular responses for the two statins, simvastatin and atorvastatin. We used the associations to quantify drug–drug similarities, assess the degree of polypharmacy and conclude that drug effects are distributed across the multi-omics modalities. Clinical multi-omics data are integrated and analyzed using a generative deep-learning model.},
  archive      = {J_NBT},
  author       = {Allesøe, Rosa Lundbye and Lundgaard, Agnete Troen and Hernández Medina, Ricardo and Aguayo-Orozco, Alejandro and Johansen, Joachim and Nissen, Jakob Nybo and Brorsson, Caroline and Mazzoni, Gianluca and Niu, Lili and Biel, Jorge Hernansanz and Leal Rodríguez, Cristina and Brasas, Valentas and Webel, Henry and Benros, Michael Eriksen and Pedersen, Anders Gorm and Chmura, Piotr Jaroslaw and Jacobsen, Ulrik Plesner and Mari, Andrea and Koivula, Robert and Mahajan, Anubha and Vinuela, Ana and Tajes, Juan Fernandez and Sharma, Sapna and Haid, Mark and Hong, Mun-Gwan and Musholt, Petra B. and De Masi, Federico and Vogt, Josef and Pedersen, Helle Krogh and Gudmundsdottir, Valborg and Jones, Angus and Kennedy, Gwen and Bell, Jimmy and Thomas, E. Louise and Frost, Gary and Thomsen, Henrik and Hansen, Elizaveta and Hansen, Tue Haldor and Vestergaard, Henrik and Muilwijk, Mirthe and Blom, Marieke T. and ‘t Hart, Leen M. and Pattou, Francois and Raverdy, Violeta and Brage, Soren and Kokkola, Tarja and Heggie, Alison and McEvoy, Donna and Mourby, Miranda and Kaye, Jane and Hattersley, Andrew and McDonald, Timothy and Ridderstråle, Martin and Walker, Mark and Forgie, Ian and Giordano, Giuseppe N. and Pavo, Imre and Ruetten, Hartmut and Pedersen, Oluf and Hansen, Torben and Dermitzakis, Emmanouil and Franks, Paul W. and Schwenk, Jochen M. and Adamski, Jerzy and McCarthy, Mark I. and Pearson, Ewan and Banasik, Karina and Rasmussen, Simon and Brunak, Søren},
  doi          = {10.1038/s41587-022-01520-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {399-408},
  shortjournal = {Nature Biotech.},
  title        = {Discovery of drug–omics associations in type 2 diabetes with generative deep-learning models},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multi-omic single-cell velocity models
epigenome–transcriptome interactions and improves cell fate prediction.
<em>NBT</em>, <em>41</em>(3), 387–398. (<a
href="https://doi.org/10.1038/s41587-022-01476-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multi-omic single-cell datasets, in which multiple molecular modalities are profiled within the same cell, offer an opportunity to understand the temporal relationship between epigenome and transcriptome. To realize this potential, we developed MultiVelo, a differential equation model of gene expression that extends the RNA velocity framework to incorporate epigenomic data. MultiVelo uses a probabilistic latent variable model to estimate the switch time and rate parameters of chromatin accessibility and gene expression and improves the accuracy of cell fate prediction compared to velocity estimates from RNA only. Application to multi-omic single-cell datasets from brain, skin and blood cells reveals two distinct classes of genes distinguished by whether chromatin closes before or after transcription ceases. We also find four types of cell states: two states in which epigenome and transcriptome are coupled and two distinct decoupled states. Finally, we identify time lags between transcription factor expression and binding site accessibility and between disease-associated SNP accessibility and expression of the linked genes. MultiVelo is available on PyPI, Bioconda and GitHub ( https://github.com/welch-lab/MultiVelo ). MultiVelo extends the concept of RNA velocity to multimodal single-cell measurements.},
  archive      = {J_NBT},
  author       = {Li, Chen and Virgilio, Maria C. and Collins, Kathleen L. and Welch, Joshua D.},
  doi          = {10.1038/s41587-022-01476-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {387-398},
  shortjournal = {Nature Biotech.},
  title        = {Multi-omic single-cell velocity models epigenome–transcriptome interactions and improves cell fate prediction},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precision mitochondrial DNA editing with high-fidelity
DddA-derived base editors. <em>NBT</em>, <em>41</em>(3), 378–386. (<a
href="https://doi.org/10.1038/s41587-022-01486-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacterial toxin DddA-derived cytosine base editors (DdCBEs)—composed of split DddAtox (a cytosine deaminase specific to double-stranded DNA), custom-designed TALE (transcription activator-like effector) DNA-binding proteins, and a uracil glycosylase inhibitor—enable mitochondrial DNA (mtDNA) editing in human cells, which may pave the way for therapeutic correction of pathogenic mtDNA mutations in patients. The utility of DdCBEs has been limited by off-target activity, which is probably caused by spontaneous assembly of the split DddAtox deaminase enzyme, independent of DNA-binding interactions. We engineered high-fidelity DddA-derived cytosine base editors (HiFi-DdCBEs) with minimal off-target activity by substituting alanine for amino acid residues at the interface between the split DddAtox halves. The resulting domains cannot form a functional deaminase without binding of their linked TALE proteins at adjacent sites on DNA. Whole mitochondrial genome sequencing shows that, unlike conventional DdCBEs, which induce hundreds of unwanted off-target C-to-T conversions in human mtDNA, HiFi-DdCBEs are highly efficient and precise, avoiding collateral off-target mutations, and as such, they will probably be desirable for therapeutic applications. Off-target activity of DddA-derived base editors is minimized through protein engineering.},
  archive      = {J_NBT},
  author       = {Lee, Seonghyun and Lee, Hyunji and Baek, Gayoung and Kim, Jin-Soo},
  doi          = {10.1038/s41587-022-01486-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {378-386},
  shortjournal = {Nature Biotech.},
  title        = {Precision mitochondrial DNA editing with high-fidelity DddA-derived base editors},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rationalized deep learning super-resolution microscopy for
sustained live imaging of rapid subcellular processes. <em>NBT</em>,
<em>41</em>(3), 367–377. (<a
href="https://doi.org/10.1038/s41587-022-01471-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The goal when imaging bioprocesses with optical microscopy is to acquire the most spatiotemporal information with the least invasiveness. Deep neural networks have substantially improved optical microscopy, including image super-resolution and restoration, but still have substantial potential for artifacts. In this study, we developed rationalized deep learning (rDL) for structured illumination microscopy and lattice light sheet microscopy (LLSM) by incorporating prior knowledge of illumination patterns and, thereby, rationally guiding the network to denoise raw images. Here we demonstrate that rDL structured illumination microscopy eliminates spectral bias-induced resolution degradation and reduces model uncertainty by five-fold, improving the super-resolution information by more than ten-fold over other computational approaches. Moreover, rDL applied to LLSM enables self-supervised training by using the spatial or temporal continuity of noisy data itself, yielding results similar to those of supervised methods. We demonstrate the utility of rDL by imaging the rapid kinetics of motile cilia, nucleolar protein condensation during light-sensitive mitosis and long-term interactions between membranous and membrane-less organelles. Rationalized deep learning improves image reconstruction by incorporating prior knowledge of illumination patterns.},
  archive      = {J_NBT},
  author       = {Qiao, Chang and Li, Di and Liu, Yong and Zhang, Siwei and Liu, Kan and Liu, Chong and Guo, Yuting and Jiang, Tao and Fang, Chuyu and Li, Nan and Zeng, Yunmin and He, Kangmin and Zhu, Xueliang and Lippincott-Schwartz, Jennifer and Dai, Qionghai and Li, Dong},
  doi          = {10.1038/s41587-022-01471-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {367-377},
  shortjournal = {Nature Biotech.},
  title        = {Rationalized deep learning super-resolution microscopy for sustained live imaging of rapid subcellular processes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Absolute quantification of single-base m6A methylation in
the mammalian transcriptome using GLORI. <em>NBT</em>, <em>41</em>(3),
355–366. (<a href="https://doi.org/10.1038/s41587-022-01487-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {N6-methyladenosine (m6A) is the most abundant RNA modification in mammalian cells and the best-studied epitranscriptomic mark. Despite the development of various tools to map m6A, a transcriptome-wide method that enables absolute quantification of m6A at single-base resolution is lacking. Here we use glyoxal and nitrite-mediated deamination of unmethylated adenosines (GLORI) to develop an absolute m6A quantification method that is conceptually similar to bisulfite-sequencing-based quantification of DNA 5-methylcytosine. We apply GLORI to quantify the m6A methylomes of mouse and human cells and reveal clustered m6A modifications with differential distribution and stoichiometry. In addition, we characterize m6A dynamics under stress and examine the quantitative landscape of m6A modification in gene expression regulation. GLORI is an unbiased, convenient method for the absolute quantification of the m6A methylome. The m6A modification is mapped transcriptome-wide at single-base resolution in mammalian cells.},
  archive      = {J_NBT},
  author       = {Liu, Cong and Sun, Hanxiao and Yi, Yunpeng and Shen, Weiguo and Li, Kai and Xiao, Ye and Li, Fei and Li, Yuchen and Hou, Yongkang and Lu, Bo and Liu, Wenqing and Meng, Haowei and Peng, Jinying and Yi, Chengqi and Wang, Jing},
  doi          = {10.1038/s41587-022-01487-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {355-366},
  shortjournal = {Nature Biotech.},
  title        = {Absolute quantification of single-base m6A methylation in the mammalian transcriptome using GLORI},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Quantitative sequencing using BID-seq uncovers abundant
pseudouridines in mammalian mRNA at base resolution. <em>NBT</em>,
<em>41</em>(3), 344–354. (<a
href="https://doi.org/10.1038/s41587-022-01505-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Functional characterization of pseudouridine (Ψ) in mammalian mRNA has been hampered by the lack of a quantitative method that maps Ψ in the whole transcriptome. We report bisulfite-induced deletion sequencing (BID-seq), which uses a bisulfite-mediated reaction to convert pseudouridine stoichiometrically into deletion upon reverse transcription without cytosine deamination. BID-seq enables detection of abundant Ψ sites with stoichiometry information in several human cell lines and 12 different mouse tissues using 10–20 ng input RNA. We uncover consensus sequences for Ψ in mammalian mRNA and assign different ‘writer’ proteins to individual Ψ deposition. Our results reveal a transcript stabilization role of Ψ sites installed by TRUB1 in human cancer cells. We also detect the presence of Ψ within stop codons of mammalian mRNA and confirm the role of Ψ in promoting stop codon readthrough in vivo. BID-seq will enable future investigations of the roles of Ψ in diverse biological processes. Pseudouridine sites in mRNA are detected at base resolution and functionally investigated.},
  archive      = {J_NBT},
  author       = {Dai, Qing and Zhang, Li-Sheng and Sun, Hui-Lung and Pajdzik, Kinga and Yang, Lei and Ye, Chang and Ju, Cheng-Wei and Liu, Shun and Wang, Yuru and Zheng, Zhong and Zhang, Linda and Harada, Bryan T. and Dou, Xiaoyang and Irkliyenko, Iryna and Feng, Xinran and Zhang, Wen and Pan, Tao and He, Chuan},
  doi          = {10.1038/s41587-022-01505-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {344-354},
  shortjournal = {Nature Biotech.},
  title        = {Quantitative sequencing using BID-seq uncovers abundant pseudouridines in mammalian mRNA at base resolution},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineered CRISPR prime editors with compact, untethered
reverse transcriptases. <em>NBT</em>, <em>41</em>(3), 337–343. (<a
href="https://doi.org/10.1038/s41587-022-01473-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The CRISPR prime editor PE2 consists of a Streptococcus pyogenes Cas9 nickase (nSpCas9) fused at its C-terminus to a Moloney murine leukemia virus reverse transcriptase (MMLV-RT). Here we show that separated nSpCas9 and MMLV-RT proteins function as efficiently as intact PE2 in human cells. We use this Split-PE system to rapidly identify and engineer more compact prime editor architectures that also broaden the types of RTs used for prime editing. A split prime editor architecture facilitates screening and engineering of improved variants.},
  archive      = {J_NBT},
  author       = {Grünewald, Julian and Miller, Bret R. and Szalay, Regan N. and Cabeceiras, Peter K. and Woodilla, Christopher J. and Holtz, Eliza Jane B. and Petri, Karl and Joung, J. Keith},
  doi          = {10.1038/s41587-022-01473-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {337-343},
  shortjournal = {Nature Biotech.},
  title        = {Engineered CRISPR prime editors with compact, untethered reverse transcriptases},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Modeling intercellular communication in tissues using
spatial graphs of cells. <em>NBT</em>, <em>41</em>(3), 332–336. (<a
href="https://doi.org/10.1038/s41587-022-01467-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Models of intercellular communication in tissues are based on molecular profiles of dissociated cells, are limited to receptor–ligand signaling and ignore spatial proximity in situ. We present node-centric expression modeling, a method based on graph neural networks that estimates the effects of niche composition on gene expression in an unbiased manner from spatial molecular profiling data. We recover signatures of molecular processes known to underlie cell communication. How cells in a tissue communicate is modeled with a graph neural network.},
  archive      = {J_NBT},
  author       = {Fischer, David S. and Schaar, Anna C. and Theis, Fabian J.},
  doi          = {10.1038/s41587-022-01467-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {332-336},
  shortjournal = {Nature Biotech.},
  title        = {Modeling intercellular communication in tissues using spatial graphs of cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mathematical modeling of epigenetic gene regulation during
cell differentiation. <em>NBT</em>, <em>41</em>(3), 330–331. (<a
href="https://doi.org/10.1038/s41587-022-01488-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells differentiate to their final fates through sequential epigenetic and transcriptional changes. A mathematical model fit on multi-omic single-cell data yields insights into the temporal relationships between chromatin accessibility and gene expression during cell differentiation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01488-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {330-331},
  shortjournal = {Nature Biotech.},
  title        = {Mathematical modeling of epigenetic gene regulation during cell differentiation},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Combining a deep neural network with physical properties for
super-resolution live imaging. <em>NBT</em>, <em>41</em>(3), 328–329.
(<a href="https://doi.org/10.1038/s41587-022-01508-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We integrated the pre-characterized physical model of super-resolution (SR) microscopy into a deep neural network to guide the denoising of raw images for high-quality SR image reconstruction. This approach enabled us to investigate a wide variety of fragile and rapidly evolving bioprocesses at ultrahigh spatiotemporal resolution over extended imaging times.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01508-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {328-329},
  shortjournal = {Nature Biotech.},
  title        = {Combining a deep neural network with physical properties for super-resolution live imaging},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Forum: Fischbach and mazmanian. <em>NBT</em>,
<em>41</em>(3), 326. (<a
href="https://doi.org/10.1038/s41587-023-01719-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01719-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {326},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Fischbach and mazmanian},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping mRNA modifications for functional studies.
<em>NBT</em>, <em>41</em>(3), 324–325. (<a
href="https://doi.org/10.1038/s41587-022-01537-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New methods for mapping the mRNA modifications N6-methyladenosine and pseudouridine enable large-scale functional analyses.},
  archive      = {J_NBT},
  author       = {Jones, Joshua D. and Eyler, Daniel E. and Koutmou, Kristin S.},
  doi          = {10.1038/s41587-022-01537-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {324-325},
  shortjournal = {Nature Biotech.},
  title        = {Mapping mRNA modifications for functional studies},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023m). Plant-inspired design of self-burying seed carrier.
<em>NBT</em>, <em>41</em>(3), 323. (<a
href="https://doi.org/10.1038/s41587-023-01720-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01720-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {323},
  shortjournal = {Nature Biotech.},
  title        = {Plant-inspired design of self-burying seed carrier},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Cardiac defect corrected in vivo with CRISPR. <em>NBT</em>,
<em>41</em>(3), 323. (<a
href="https://doi.org/10.1038/s41587-023-01721-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01721-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {323},
  shortjournal = {Nature Biotech.},
  title        = {Cardiac defect corrected in vivo with CRISPR},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). Metatranscriptomes reveal increased diversity of
viroid-like circular RNAs. <em>NBT</em>, <em>41</em>(3), 323. (<a
href="https://doi.org/10.1038/s41587-023-01722-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01722-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {323},
  shortjournal = {Nature Biotech.},
  title        = {Metatranscriptomes reveal increased diversity of viroid-like circular RNAs},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Synthetic biology. <em>NBT</em>, <em>41</em>(3), 322. (<a
href="https://doi.org/10.1038/s41587-023-01712-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods, systems and toolkits related to synthetic biology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01712-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {322},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic biology},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Distinguishing and predicting drug patents. <em>NBT</em>,
<em>41</em>(3), 317–321. (<a
href="https://doi.org/10.1038/s41587-023-01703-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug patents are different. To improve their quality ex ante, regulators can use predictive models.},
  archive      = {J_NBT},
  author       = {Chien, Colleen V. and Halkowski, Nicholas and Kuhn, Jeffrey},
  doi          = {10.1038/s41587-023-01703-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {317-321},
  shortjournal = {Nature Biotech.},
  title        = {Distinguishing and predicting drug patents},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rapid-response manufacturing of adenovirus-vectored
vaccines. <em>NBT</em>, <em>41</em>(3), 314–316. (<a
href="https://doi.org/10.1038/s41587-023-01682-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Joe, Carina C. D. and Chopra, Nitin and Nestola, Piergiuseppe and Niemann, Julia and Douglas, Alexander D.},
  doi          = {10.1038/s41587-023-01682-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {314-316},
  shortjournal = {Nature Biotech.},
  title        = {Rapid-response manufacturing of adenovirus-vectored vaccines},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(3), 309. (<a
href="https://doi.org/10.1038/s41587-023-01708-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01708-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {309},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Shoot the messenger: RNA editing is here. <em>NBT</em>,
<em>41</em>(3), 306–308. (<a
href="https://doi.org/10.1038/s41587-023-01709-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA editing is rapidly gaining prominence as its transient and reversible changes promise a safer and more flexible option to reverse disease-causing mutations than DNA editing.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-023-01709-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {306-308},
  shortjournal = {Nature Biotech.},
  title        = {Shoot the messenger: RNA editing is here},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GM forests for carbon removal planted in US. <em>NBT</em>,
<em>41</em>(3), 306. (<a
href="https://doi.org/10.1038/s41587-023-01726-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01726-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {306},
  shortjournal = {Nature Biotech.},
  title        = {GM forests for carbon removal planted in US},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Big pharma craves slice of AI-based RNA drug discovery.
<em>NBT</em>, <em>41</em>(3), 305. (<a
href="https://doi.org/10.1038/s41587-023-01725-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01725-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {305},
  shortjournal = {Nature Biotech.},
  title        = {Big pharma craves slice of AI-based RNA drug discovery},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biotech commission accused of conflicts of interest.
<em>NBT</em>, <em>41</em>(3), 304. (<a
href="https://doi.org/10.1038/s41587-023-01723-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01723-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {304},
  shortjournal = {Nature Biotech.},
  title        = {Biotech commission accused of conflicts of interest},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Women build strength in numbers. <em>NBT</em>,
<em>41</em>(3), 301. (<a
href="https://doi.org/10.1038/s41587-023-01727-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Women’s networks are strong, and making a difference.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01727-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Women build strength in numbers},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cancer debugged. <em>NBT</em>, <em>41</em>(3), 310–313. (<a
href="https://doi.org/10.1038/s41587-023-01677-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumors are rife with bacteria and fungi. Their ubiquity is proving useful in detecting cancers, categorizing them, and even determining whether certain interventions will work.},
  archive      = {J_NBT},
  author       = {Brouillette, Monique},
  doi          = {10.1038/s41587-023-01677-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {310-313},
  shortjournal = {Nature Biotech.},
  title        = {Cancer debugged},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). AI-enhanced protein design makes proteins that have never
existed. <em>NBT</em>, <em>41</em>(3), 303–305. (<a
href="https://doi.org/10.1038/s41587-023-01705-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein engineers are drawing on rapidly evolving machine learning tools, deep reservoirs of data, and the structure-predicting firepower of AlphaFold2 to pursue more sophisticated de novo protein designs.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-023-01705-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {303-305},
  shortjournal = {Nature Biotech.},
  title        = {AI-enhanced protein design makes proteins that have never existed},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). People. <em>NBT</em>, <em>41</em>(2), 300. (<a
href="https://doi.org/10.1038/s41587-023-01667-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01667-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {300},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Fourth-quarter biotech job picture. <em>NBT</em>,
<em>41</em>(2), 299. (<a
href="https://doi.org/10.1038/s41587-023-01666-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-023-01666-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {299},
  shortjournal = {Nature Biotech.},
  title        = {Fourth-quarter biotech job picture},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Retaining postdocs by recognizing their worth. <em>NBT</em>,
<em>41</em>(2), 296–298. (<a
href="https://doi.org/10.1038/s41587-023-01656-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data from a postdoc salary and benefits survey may assist in retaining PhD holders interested in academia but considering alternate options owing to financial considerations.},
  archive      = {J_NBT},
  author       = {Yalcin, Esra and Martinez-Corral, Rosa and Chugh, Mayank},
  doi          = {10.1038/s41587-023-01656-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {296-298},
  shortjournal = {Nature Biotech.},
  title        = {Retaining postdocs by recognizing their worth},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Engineering circular RNA for enhanced
protein production. <em>NBT</em>, <em>41</em>(2), 293. (<a
href="https://doi.org/10.1038/s41587-022-01472-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chen, Robert and Wang, Sean K. and Belk, Julia A. and Amaya, Laura and Li, Zhijian and Cardenas, Angel and Abe, Brian T. and Chen, Chun-Kan and Wender, Paul A. and Chang, Howard Y.},
  doi          = {10.1038/s41587-022-01472-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {293},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Engineering circular RNA for enhanced protein production},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Real-time denoising enables high-sensitivity fluorescence
time-lapse imaging beyond the shot-noise limit. <em>NBT</em>,
<em>41</em>(2), 282–292. (<a
href="https://doi.org/10.1038/s41587-022-01450-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fundamental challenge in fluorescence microscopy is the photon shot noise arising from the inevitable stochasticity of photon detection. Noise increases measurement uncertainty and limits imaging resolution, speed and sensitivity. To achieve high-sensitivity fluorescence imaging beyond the shot-noise limit, we present DeepCAD-RT, a self-supervised deep learning method for real-time noise suppression. Based on our previous framework DeepCAD, we reduced the number of network parameters by 94\%, memory consumption by 27-fold and processing time by a factor of 20, allowing real-time processing on a two-photon microscope. A high imaging signal-to-noise ratio can be acquired with tenfold fewer photons than in standard imaging approaches. We demonstrate the utility of DeepCAD-RT in a series of photon-limited experiments, including in vivo calcium imaging of mice, zebrafish larva and fruit flies, recording of three-dimensional (3D) migration of neutrophils after acute brain injury and imaging of 3D dynamics of cortical ATP release. DeepCAD-RT will facilitate the morphological and functional interrogation of biological dynamics with a minimal photon budget. DeepCAD-RT denoises fluorescence time-lapse images in real time.},
  archive      = {J_NBT},
  author       = {Li, Xinyang and Li, Yixin and Zhou, Yiliang and Wu, Jiamin and Zhao, Zhifeng and Fan, Jiaqi and Deng, Fei and Wu, Zhaofa and Xiao, Guihua and He, Jing and Zhang, Yuanlong and Zhang, Guoxun and Hu, Xiaowan and Chen, Xingye and Zhang, Yi and Qiao, Hui and Xie, Hao and Li, Yulong and Wang, Haoqian and Fang, Lu and Dai, Qionghai},
  doi          = {10.1038/s41587-022-01450-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {282-292},
  shortjournal = {Nature Biotech.},
  title        = {Real-time denoising enables high-sensitivity fluorescence time-lapse imaging beyond the shot-noise limit},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Modular cytokine receptor-targeting chimeras for targeted
degradation of cell surface and extracellular proteins. <em>NBT</em>,
<em>41</em>(2), 273–281. (<a
href="https://doi.org/10.1038/s41587-022-01456-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted degradation of cell surface and extracellular proteins via lysosomal delivery is an important means to modulate extracellular biology. However, these approaches have limitations due to lack of modularity, ease of development, restricted tissue targeting and applicability to both cell surface and extracellular proteins. We describe a lysosomal degradation strategy, termed cytokine receptor-targeting chimeras (KineTACs), that addresses these limitations. KineTACs are fully genetically encoded bispecific antibodies consisting of a cytokine arm, which binds its cognate cytokine receptor, and a target-binding arm for the protein of interest. We show that KineTACs containing the cytokine CXCL12 can use the decoy recycling receptor, CXCR7, to target a variety of target proteins to the lysosome for degradation. Additional KineTACs were designed to harness other CXCR7-targeting cytokines, CXCL11 and vMIPII, and the interleukin-2 (IL-2) receptor-targeting cytokine IL-2. Thus, KineTACs represent a general, modular, selective and simple genetically encoded strategy for inducing lysosomal delivery of extracellular and cell surface targets with broad or tissue-specific distribution. KineTACs are modular bispecific antibodies for degradation of extracellular and cell-surface proteins.},
  archive      = {J_NBT},
  author       = {Pance, Katarina and Gramespacher, Josef A. and Byrnes, James R. and Salangsang, Fernando and Serrano, Juan-Antonio C. and Cotton, Adam D. and Steri, Veronica and Wells, James A.},
  doi          = {10.1038/s41587-022-01456-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {273-281},
  shortjournal = {Nature Biotech.},
  title        = {Modular cytokine receptor-targeting chimeras for targeted degradation of cell surface and extracellular proteins},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Engineering circular RNA for enhanced protein production.
<em>NBT</em>, <em>41</em>(2), 262–272. (<a
href="https://doi.org/10.1038/s41587-022-01393-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circular RNAs (circRNAs) are stable and prevalent RNAs in eukaryotic cells that arise from back-splicing. Synthetic circRNAs and some endogenous circRNAs can encode proteins, raising the promise of circRNA as a platform for gene expression. In this study, we developed a systematic approach for rapid assembly and testing of features that affect protein production from synthetic circRNAs. To maximize circRNA translation, we optimized five elements: vector topology, 5′ and 3′ untranslated regions, internal ribosome entry sites and synthetic aptamers recruiting translation initiation machinery. Together, these design principles improve circRNA protein yields by several hundred-fold, provide increased translation over messenger RNA in vitro, provide more durable translation in vivo and are generalizable across multiple transgenes. Protein expression from circular RNAs is enhanced several hundred-fold by optimizing vector design.},
  archive      = {J_NBT},
  author       = {Chen, Robert and Wang, Sean K. and Belk, Julia A. and Amaya, Laura and Li, Zhijian and Cardenas, Angel and Abe, Brian T. and Chen, Chun-Kan and Wender, Paul A. and Chang, Howard Y.},
  doi          = {10.1038/s41587-022-01393-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {262-272},
  shortjournal = {Nature Biotech.},
  title        = {Engineering circular RNA for enhanced protein production},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Human ureteric bud organoids recapitulate branching
morphogenesis and differentiate into functional collecting duct cell
types. <em>NBT</em>, <em>41</em>(2), 252–261. (<a
href="https://doi.org/10.1038/s41587-022-01429-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed differentiation of human pluripotent stem cells (hPSCs) into functional ureteric and collecting duct (CD) epithelia is essential to kidney regenerative medicine. Here we describe highly efficient, serum-free differentiation of hPSCs into ureteric bud (UB) organoids and functional CD cells. The hPSCs are first induced into pronephric progenitor cells at 90\% efficiency and then aggregated into spheres with a molecular signature similar to the nephric duct. In a three-dimensional matrix, the spheres form UB organoids that exhibit branching morphogenesis similar to the fetal UB and correct distal tip localization of RET expression. Organoid-derived cells incorporate into the UB tips of the progenitor niche in chimeric fetal kidney explant culture. At later stages, the UB organoids differentiate into CD organoids, which contain &amp;gt;95\% CD cell types as estimated by single-cell RNA sequencing. The CD epithelia demonstrate renal electrophysiologic functions, with ENaC-mediated vectorial sodium transport by principal cells and V-type ATPase proton pump activity by FOXI1-induced intercalated cells. Kidney collecting duct cells with physiologic functions are generated from human pluripotent stem cells.},
  archive      = {J_NBT},
  author       = {Shi, Min and McCracken, Kyle W. and Patel, Ankit B. and Zhang, Weitao and Ester, Lioba and Valerius, M. Todd and Bonventre, Joseph V.},
  doi          = {10.1038/s41587-022-01429-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {252-261},
  shortjournal = {Nature Biotech.},
  title        = {Human ureteric bud organoids recapitulate branching morphogenesis and differentiate into functional collecting duct cell types},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Post-translational modifications reshape the antigenic
landscape of the MHC i immunopeptidome in tumors. <em>NBT</em>,
<em>41</em>(2), 239–251. (<a
href="https://doi.org/10.1038/s41587-022-01464-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modification (PTM) of antigens provides an additional source of specificities targeted by immune responses to tumors or pathogens, but identifying antigen PTMs and assessing their role in shaping the immunopeptidome is challenging. Here we describe the Protein Modification Integrated Search Engine (PROMISE), an antigen discovery pipeline that enables the analysis of 29 different PTM combinations from multiple clinical cohorts and cell lines. We expanded the antigen landscape, uncovering human leukocyte antigen class I binding motifs defined by specific PTMs with haplotype-specific binding preferences and revealing disease-specific modified targets, including thousands of new cancer-specific antigens that can be shared between patients and across cancer types. Furthermore, we uncovered a subset of modified peptides that are specific to cancer tissue and driven by post-translational changes that occurred in the tumor proteome. Our findings highlight principles of PTM-driven antigenicity, which may have broad implications for T cell-mediated therapies in cancer and beyond. A computational pipeline identifies tumor antigen post-translational modifications guiding immune responses.},
  archive      = {J_NBT},
  author       = {Kacen, Assaf and Javitt, Aaron and Kramer, Matthias P. and Morgenstern, David and Tsaban, Tomer and Shmueli, Merav D. and Teo, Guo Ci and da Veiga Leprevost, Felipe and Barnea, Eilon and Yu, Fengchao and Admon, Arie and Eisenbach, Lea and Samuels, Yardena and Schueler-Furman, Ora and Levin, Yishai and Nesvizhskii, Alexey I. and Merbl, Yifat},
  doi          = {10.1038/s41587-022-01464-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {239-251},
  shortjournal = {Nature Biotech.},
  title        = {Post-translational modifications reshape the antigenic landscape of the MHC i immunopeptidome in tumors},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). DeepConsensus improves the accuracy of sequences with a
gap-aware sequence transformer. <em>NBT</em>, <em>41</em>(2), 232–238.
(<a href="https://doi.org/10.1038/s41587-022-01435-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circular consensus sequencing with Pacific Biosciences (PacBio) technology generates long (10–25 kilobases), accurate ‘HiFi’ reads by combining serial observations of a DNA molecule into a consensus sequence. The standard approach to consensus generation, pbccs, uses a hidden Markov model. We introduce DeepConsensus, which uses an alignment-based loss to train a gap-aware transformer–encoder for sequence correction. Compared to pbccs, DeepConsensus reduces read errors by 42\%. This increases the yield of PacBio HiFi reads at Q20 by 9\%, at Q30 by 27\% and at Q40 by 90\%. With two SMRT Cells of HG003, reads from DeepConsensus improve hifiasm assembly contiguity (﻿NG50 4.9 megabases (Mb) to 17.2 Mb), increase gene completeness (94\% to 97\%), reduce the false gene duplication rate (1.1\% to 0.5\%), improve assembly base accuracy (Q43 to Q45) and reduce variant-calling errors by 24\%. DeepConsensus models could be trained to the general problem of analyzing the alignment of other types of sequences, such as unique molecular identifiers or genome assemblies. Deep learning reduces errors in sequences from PacBio HiFi reads.},
  archive      = {J_NBT},
  author       = {Baid, Gunjan and Cook, Daniel E. and Shafin, Kishwar and Yun, Taedong and Llinares-López, Felipe and Berthet, Quentin and Belyaeva, Anastasiya and Töpfer, Armin and Wenger, Aaron M. and Rowell, William J. and Yang, Howard and Kolesnikov, Alexey and Ammar, Waleed and Vert, Jean-Philippe and Vaswani, Ashish and McLean, Cory Y. and Nattestad, Maria and Chang, Pi-Chuan and Carroll, Andrew},
  doi          = {10.1038/s41587-022-01435-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {232-238},
  shortjournal = {Nature Biotech.},
  title        = {DeepConsensus improves the accuracy of sequences with a gap-aware sequence transformer},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Scalable in situ single-cell profiling by electrophoretic
capture of mRNA using EEL FISH. <em>NBT</em>, <em>41</em>(2), 222–231.
(<a href="https://doi.org/10.1038/s41587-022-01455-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to spatially profile the transcriptome are dominated by a trade-off between resolution and throughput. Here we develop a method named Enhanced ELectric Fluorescence in situ Hybridization (EEL FISH) that can rapidly process large tissue samples without compromising spatial resolution. By electrophoretically transferring RNA from a tissue section onto a capture surface, EEL speeds up data acquisition by reducing the amount of imaging needed, while ensuring that RNA molecules move straight down toward the surface, preserving single-cell resolution. We apply EEL on eight entire sagittal sections of the mouse brain and measure the expression patterns of up to 440 genes to reveal complex tissue organization. Moreover, EEL can be used to study challenging human samples by removing autofluorescent lipofuscin, enabling the spatial transcriptome of the human visual cortex to be visualized. We provide full hardware specifications, all protocols and complete software for instrument control, image processing, data analysis and visualization. EEL FISH profiles spatial transcriptomes in single cells in large, complex tissue samples.},
  archive      = {J_NBT},
  author       = {Borm, Lars E. and Mossi Albiach, Alejandro and Mannens, Camiel C. A. and Janusauskas, Jokubas and Özgün, Ceren and Fernández-García, David and Hodge, Rebecca and Castillo, Francisca and Hedin, Charlotte R. H. and Villablanca, Eduardo J. and Uhlén, Per and Lein, Ed S. and Codeluppi, Simone and Linnarsson, Sten},
  doi          = {10.1038/s41587-022-01455-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {222-231},
  shortjournal = {Nature Biotech.},
  title        = {Scalable in situ single-cell profiling by electrophoretic capture of mRNA using EEL FISH},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiplexed, single-molecule, epigenetic analysis of
plasma-isolated nucleosomes for cancer diagnostics. <em>NBT</em>,
<em>41</em>(2), 212–221. (<a
href="https://doi.org/10.1038/s41587-022-01447-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The analysis of cell-free DNA (cfDNA) in plasma provides information on pathological processes in the body. Blood cfDNA is in the form of nucleosomes, which maintain their tissue- and cancer-specific epigenetic state. We developed a single-molecule multiparametric assay to comprehensively profile the epigenetics of plasma-isolated nucleosomes (EPINUC), DNA methylation and cancer-specific protein biomarkers. Our system allows for high-resolution detection of six active and repressive histone modifications and their ratios and combinatorial patterns on millions of individual nucleosomes by single-molecule imaging. In addition, our system provides sensitive and quantitative data on plasma proteins, including detection of non-secreted tumor-specific proteins, such as mutant p53. EPINUC analysis of a cohort of 63 colorectal cancer, 10 pancreatic cancer and 33 healthy plasma samples detected cancer with high accuracy and sensitivity, even at early stages. Finally, combining EPINUC with direct single-molecule DNA sequencing revealed the tissue of origin of colorectal, pancreatic, lung and breast tumors. EPINUC provides multilayered information of potential clinical relevance from limited (&amp;lt;1 ml) liquid biopsy material. Single-molecule imaging of cell-free nucleosomes detects colorectal cancer with high accuracy and sensitivity.},
  archive      = {J_NBT},
  author       = {Fedyuk, Vadim and Erez, Nir and Furth, Noa and Beresh, Olga and Andreishcheva, Ekaterina and Shinde, Abhijeet and Jones, Daniel and Zakai, Barak Bar and Mavor, Yael and Peretz, Tamar and Hubert, Ayala and Cohen, Jonathan E. and Salah, Azzam and Temper, Mark and Grinshpun, Albert and Maoz, Myriam and Zick, Aviad and Ron, Guy and Shema, Efrat},
  doi          = {10.1038/s41587-022-01447-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {212-221},
  shortjournal = {Nature Biotech.},
  title        = {Multiplexed, single-molecule, epigenetic analysis of plasma-isolated nucleosomes for cancer diagnostics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mostly natural sequencing-by-synthesis for scRNA-seq using
ultima sequencing. <em>NBT</em>, <em>41</em>(2), 204–211. (<a
href="https://doi.org/10.1038/s41587-022-01452-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we introduce a mostly natural sequencing-by-synthesis (mnSBS) method for single-cell RNA sequencing (scRNA-seq), adapted to the Ultima genomics platform, and systematically benchmark it against current scRNA-seq technology. mnSBS uses mostly natural, unmodified nucleotides and only a low fraction of fluorescently labeled nucleotides, which allows for high polymerase processivity and lower costs. We demonstrate successful application in four scRNA-seq case studies of different technical and biological types, including 5′ and 3′ scRNA-seq, human peripheral blood mononuclear cells from a single individual and in multiplex, as well as Perturb-Seq. Benchmarking shows that results from mnSBS-based scRNA-seq are very similar to those using Illumina sequencing, with minor differences in results related to the position of reads relative to annotated gene boundaries, owing to single-end reads of Ultima being closer to gene ends than reads from Illumina. The method is thus compatible with state-of-the-art scRNA-seq libraries independent of the sequencing technology. We expect mnSBS to be of particular utility for cost-effective large-scale scRNA-seq projects. A mostly natural sequencing-by-synthesis method is effective for scRNA-seq, particularly in large-scale studies.},
  archive      = {J_NBT},
  author       = {Simmons, Sean K. and Lithwick-Yanai, Gila and Adiconis, Xian and Oberstrass, Florian and Iremadze, Nika and Geiger-Schuller, Kathryn and Thakore, Pratiksha I. and Frangieh, Chris J. and Barad, Omer and Almogy, Gilad and Rozenblatt-Rosen, Orit and Regev, Aviv and Lipson, Doron and Levin, Joshua Z.},
  doi          = {10.1038/s41587-022-01452-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {204-211},
  shortjournal = {Nature Biotech.},
  title        = {Mostly natural sequencing-by-synthesis for scRNA-seq using ultima sequencing},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A competitive precision CRISPR method to identify the
fitness effects of transcription factor binding sites. <em>NBT</em>,
<em>41</em>(2), 197–203. (<a
href="https://doi.org/10.1038/s41587-022-01444-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we describe a competitive genome editing method that measures the effect of mutations on molecular functions, based on precision CRISPR editing using template libraries with either the original or altered sequence, and a sequence tag, enabling direct comparison between original and mutated cells. Using the example of the MYC oncogene, we identify important transcriptional targets and show that E-box mutations at MYC target gene promoters reduce cellular fitness. A competitive CRISPR method discovers targets and phenotypic effects of the MYC oncogene.},
  archive      = {J_NBT},
  author       = {Pihlajamaa, Päivi and Kauko, Otto and Sahu, Biswajyoti and Kivioja, Teemu and Taipale, Jussi},
  doi          = {10.1038/s41587-022-01444-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {197-203},
  shortjournal = {Nature Biotech.},
  title        = {A competitive precision CRISPR method to identify the fitness effects of transcription factor binding sites},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Capturing the diversity of protein modifications on
presented tumor antigens. <em>NBT</em>, <em>41</em>(2), 195–196. (<a
href="https://doi.org/10.1038/s41587-022-01465-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational modifications (PTMs) alter the structure, properties and functions of proteins in all aspects of biology. A new computational pipeline, termed protein modification integrated search engine (PROMISE), reveals the impact these modifications might have on the presentation of cancer antigens to T cells.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01465-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {195-196},
  shortjournal = {Nature Biotech.},
  title        = {Capturing the diversity of protein modifications on presented tumor antigens},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Capturing haplotype variation in populations using
pangenome references. <em>NBT</em>, <em>41</em>(2), 194. (<a
href="https://doi.org/10.1038/s41587-023-01691-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01691-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {194},
  shortjournal = {Nature Biotech.},
  title        = {Capturing haplotype variation in populations using pangenome references},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Consequences of cis-regulatory sequence variation.
<em>NBT</em>, <em>41</em>(2), 194. (<a
href="https://doi.org/10.1038/s41587-023-01692-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fudge, Jared B.},
  doi          = {10.1038/s41587-023-01692-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {194},
  shortjournal = {Nature Biotech.},
  title        = {Consequences of cis-regulatory sequence variation},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). DISCO-MS combines spatial proteomics with whole-organ
imaging. <em>NBT</em>, <em>41</em>(2), 194. (<a
href="https://doi.org/10.1038/s41587-023-01699-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Doerr, Anne},
  doi          = {10.1038/s41587-023-01699-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {194},
  shortjournal = {Nature Biotech.},
  title        = {DISCO-MS combines spatial proteomics with whole-organ imaging},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Base editors. <em>NBT</em>, <em>41</em>(2), 193. (<a
href="https://doi.org/10.1038/s41587-023-01672-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to compositions and methods for targeted editing of nucleic acids.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01672-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {193},
  shortjournal = {Nature Biotech.},
  title        = {Base editors},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Drug licensing as evidence of evolution, diffusion and
catch-up in east asia. <em>NBT</em>, <em>41</em>(2), 189–192. (<a
href="https://doi.org/10.1038/s41587-023-01659-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data-driven results map a holistic landscape of drug licensing and biotechnology diffusion in the last two decades.},
  archive      = {J_NBT},
  author       = {Huang, Jianan},
  doi          = {10.1038/s41587-023-01659-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {189-192},
  shortjournal = {Nature Biotech.},
  title        = {Drug licensing as evidence of evolution, diffusion and catch-up in east asia},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Delivering 3 billion doses of comirnaty in 2021.
<em>NBT</em>, <em>41</em>(2), 183–188. (<a
href="https://doi.org/10.1038/s41587-022-01643-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pfizer created a ‘light-speed’ approach to meet the challenge of vaccinating the world against COVID-19. It involved developing new strategies for all aspects of vaccine development, from sourcing materials and scaling up manufacturing to transportation and dosing.},
  archive      = {J_NBT},
  author       = {Warne, Nicholas and Ruesch, Margaret and Siwik, Pamela and Mensah, Paul and Ludwig, John and Hripcsak, Michael and Godavarti, Ranga and Prigodich, Andrew and Dolsten, Mikael},
  doi          = {10.1038/s41587-022-01643-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {183-188},
  shortjournal = {Nature Biotech.},
  title        = {Delivering 3 billion doses of comirnaty in 2021},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). 2022 – toughing out the trough. <em>NBT</em>,
<em>41</em>(2), 170–173. (<a
href="https://doi.org/10.1038/s41587-023-01661-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-01661-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {170-173},
  shortjournal = {Nature Biotech.},
  title        = {2022 – toughing out the trough},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Drug pipeline 4Q22 — sticking around. <em>NBT</em>,
<em>41</em>(2), 167–169. (<a
href="https://doi.org/10.1038/s41587-023-01660-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-023-01660-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {167-169},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 4Q22 — sticking around},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(2), 166. (<a
href="https://doi.org/10.1038/s41587-023-01669-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01669-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {166},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Putting CRISPR into african hands to future-proof crops.
<em>NBT</em>, <em>41</em>(2), 165–166. (<a
href="https://doi.org/10.1038/s41587-023-01668-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists from seven countries embark on training to use gene editing tools to improve the continent’s neglected crops.},
  archive      = {J_NBT},
  author       = {Nordling, Linda},
  doi          = {10.1038/s41587-023-01668-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {165-166},
  shortjournal = {Nature Biotech.},
  title        = {Putting CRISPR into african hands to future-proof crops},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). <em>NBT</em>, <em>41</em>(2), 165. (<a
href="https://doi.org/10.1038/s41587-023-01698-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01698-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {165},
  shortjournal = {Nature Biotech.},
  title        = {Amgen pledges $8 million for diversity},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). First bee vaccine. <em>NBT</em>, <em>41</em>(2), 163. (<a
href="https://doi.org/10.1038/s41587-023-01694-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01694-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {163},
  shortjournal = {Nature Biotech.},
  title        = {First bee vaccine},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Small innovators advance microbes as alternatives to
chemical crop sprays. <em>NBT</em>, <em>41</em>(2), 162–164. (<a
href="https://doi.org/10.1038/s41587-023-01670-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As farmers around the globe embrace microorganisms to treat and protect crops, ag giants sort out how they will participate in the industry, coming to different conclusions.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-023-01670-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {162-164},
  shortjournal = {Nature Biotech.},
  title        = {Small innovators advance microbes as alternatives to chemical crop sprays},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). UK, BioNTech test mRNA against cancer. <em>NBT</em>,
<em>41</em>(2), 162. (<a
href="https://doi.org/10.1038/s41587-023-01693-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01693-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {162},
  shortjournal = {Nature Biotech.},
  title        = {UK, BioNTech test mRNA against cancer},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Industry appetite for natural killer cells intensifies.
<em>NBT</em>, <em>41</em>(2), 159–161. (<a
href="https://doi.org/10.1038/s41587-023-01671-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural killer cells are attractive as cancer immunotherapy agents because — unlike T cells — they evade immune rejection and do not induce cytokine storms. But capturing their activity in effective therapies remains a work in progress.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-023-01671-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {159-161},
  shortjournal = {Nature Biotech.},
  title        = {Industry appetite for natural killer cells intensifies},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Generating “smarter” biotechnology. <em>NBT</em>,
<em>41</em>(2), 157. (<a
href="https://doi.org/10.1038/s41587-023-01695-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative AI is moving full speed ahead in the biomedical space.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-023-01695-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {157},
  shortjournal = {Nature Biotech.},
  title        = {Generating ‘smarter’ biotechnology},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Fresh from the biotech pipeline: Fewer approvals, but
biologics gain share. <em>NBT</em>, <em>41</em>(2), 174–182. (<a
href="https://doi.org/10.1038/s41587-022-01630-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {2022 saw a 25\% drop in new drug approvals and a less predictable regulator.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-022-01630-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {174-182},
  shortjournal = {Nature Biotech.},
  title        = {Fresh from the biotech pipeline: Fewer approvals, but biologics gain share},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). People. <em>NBT</em>, <em>41</em>(1), 156. (<a
href="https://doi.org/10.1038/s41587-022-01629-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01629-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {156},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Increasing the success potential of promising biotech
companies. <em>NBT</em>, <em>41</em>(1), 154–155. (<a
href="https://doi.org/10.1038/s41587-022-01627-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biotech startups benefit society by solving unmet scientific and medical needs, but more can be done to move their technologies to market more quickly.},
  archive      = {J_NBT},
  author       = {Marrus, Stephanie K. and Blaho, John A.},
  doi          = {10.1038/s41587-022-01627-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {154-155},
  shortjournal = {Nature Biotech.},
  title        = {Increasing the success potential of promising biotech companies},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: M6A RNA modifications are measured at
single-base resolution across the mammalian transcriptome. <em>NBT</em>,
<em>41</em>(1), 150. (<a
href="https://doi.org/10.1038/s41587-022-01616-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hu, Lulu and Liu, Shun and Peng, Yong and Ge, Ruiqi and Su, Rui and Senevirathne, Chamara and Harada, Bryan T. and Dai, Qing and Wei, Jiangbo and Zhang, Lisheng and Hao, Ziyang and Luo, Liangzhi and Wang, Huanyu and Wang, Yuru and Luo, Minkui and Chen, Mengjie and Chen, Jianjun and He, Chuan},
  doi          = {10.1038/s41587-022-01616-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {150},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: M6A RNA modifications are measured at single-base resolution across the mammalian transcriptome},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A proteome-scale map of the SARS-CoV-2–human contactome.
<em>NBT</em>, <em>41</em>(1), 140–149. (<a
href="https://doi.org/10.1038/s41587-022-01475-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the mechanisms of coronavirus disease 2019 (COVID-19) disease severity to efficiently design therapies for emerging virus variants remains an urgent challenge of the ongoing pandemic. Infection and immune reactions are mediated by direct contacts between viral molecules and the host proteome, and the vast majority of these virus–host contacts (the ‘contactome’) have not been identified. Here, we present a systematic contactome map of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) with the human host encompassing more than 200 binary virus–host and intraviral protein–protein interactions. We find that host proteins genetically associated with comorbidities of severe illness and long COVID are enriched in SARS-CoV-2 targeted network communities. Evaluating contactome-derived hypotheses, we demonstrate that viral NSP14 activates nuclear factor κB (NF-κB)-dependent transcription, even in the presence of cytokine signaling. Moreover, for several tested host proteins, genetic knock-down substantially reduces viral replication. Additionally, we show for USP25 that this effect is phenocopied by the small-molecule inhibitor AZ1. Our results connect viral proteins to human genetic architecture for COVID-19 severity and offer potential therapeutic targets. A SARS-CoV-2-human protein contactome map uncovers an inhibitor of viral replication.},
  archive      = {J_NBT},
  author       = {Kim, Dae-Kyum and Weller, Benjamin and Lin, Chung-Wen and Sheykhkarimli, Dayag and Knapp, Jennifer J. and Dugied, Guillaume and Zanzoni, Andreas and Pons, Carles and Tofaute, Marie J. and Maseko, Sibusiso B. and Spirohn, Kerstin and Laval, Florent and Lambourne, Luke and Kishore, Nishka and Rayhan, Ashyad and Sauer, Mayra and Young, Veronika and Halder, Hridi and la Rosa, Nora Marín-de and Pogoutse, Oxana and Strobel, Alexandra and Schwehn, Patrick and Li, Roujia and Rothballer, Simin T. and Altmann, Melina and Cassonnet, Patricia and Coté, Atina G. and Vergara, Lena Elorduy and Hazelwood, Isaiah and Liu, Betty B. and Nguyen, Maria and Pandiarajan, Ramakrishnan and Dohai, Bushra and Coloma, Patricia A. Rodriguez and Poirson, Juline and Giuliana, Paolo and Willems, Luc and Taipale, Mikko and Jacob, Yves and Hao, Tong and Hill, David E. and Brun, Christine and Twizere, Jean-Claude and Krappmann, Daniel and Heinig, Matthias and Falter, Claudia and Aloy, Patrick and Demeret, Caroline and Vidal, Marc and Calderwood, Michael A. and Roth, Frederick P. and Falter-Braun, Pascal},
  doi          = {10.1038/s41587-022-01475-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {140-149},
  shortjournal = {Nature Biotech.},
  title        = {A proteome-scale map of the SARS-CoV-2–human contactome},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A comprehensive SARS-CoV-2–human protein–protein interactome
reveals COVID-19 pathobiology and potential host therapeutic targets.
<em>NBT</em>, <em>41</em>(1), 128–139. (<a
href="https://doi.org/10.1038/s41587-022-01474-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studying viral–host protein–protein interactions can facilitate the discovery of therapies for viral infection. We use high-throughput yeast two-hybrid experiments and mass spectrometry to generate a comprehensive SARS-CoV-2–human protein–protein interactome network consisting of 739 high-confidence binary and co-complex interactions, validating 218 known SARS-CoV-2 host factors and revealing 361 novel ones. Our results show the highest overlap of interaction partners between published datasets and of genes differentially expressed in samples from COVID-19 patients. We identify an interaction between the viral protein ORF3a and the human transcription factor ZNF579, illustrating a direct viral impact on host transcription. We perform network-based screens of &amp;gt;2,900 FDA-approved or investigational drugs and identify 23 with significant network proximity to SARS-CoV-2 host factors. One of these drugs, carvedilol, shows clinical benefits for COVID-19 patients in an electronic health records analysis and antiviral properties in a human lung cell line infected with SARS-CoV-2. Our study demonstrates the value of network systems biology to understand human–virus interactions and provides hits for further research on COVID-19 therapeutics. A SARS-CoV-2–human interactome network reveals potential human drug targets.},
  archive      = {J_NBT},
  author       = {Zhou, Yadi and Liu, Yuan and Gupta, Shagun and Paramo, Mauricio I. and Hou, Yuan and Mao, Chengsheng and Luo, Yuan and Judd, Julius and Wierbowski, Shayne and Bertolotti, Marta and Nerkar, Mriganka and Jehi, Lara and Drayman, Nir and Nicolaescu, Vlad and Gula, Haley and Tay, Savaş and Randall, Glenn and Wang, Peihui and Lis, John T. and Feschotte, Cédric and Erzurum, Serpil C. and Cheng, Feixiong and Yu, Haiyuan},
  doi          = {10.1038/s41587-022-01474-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {128-139},
  shortjournal = {Nature Biotech.},
  title        = {A comprehensive SARS-CoV-2–human protein–protein interactome reveals COVID-19 pathobiology and potential host therapeutic targets},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The role of transposon inverted repeats in balancing drought
tolerance and yield-related traits in maize. <em>NBT</em>,
<em>41</em>(1), 120–127. (<a
href="https://doi.org/10.1038/s41587-022-01470-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genomic basis underlying the selection for environmental adaptation and yield-related traits in maize remains poorly understood. Here we carried out genome-wide profiling of the small RNA (sRNA) transcriptome (sRNAome) and transcriptome landscapes of a global maize diversity panel under dry and wet conditions and uncover dozens of environment-specific regulatory hotspots. Transgenic and molecular studies of Drought-Related Environment-specific Super eQTL Hotspot on chromosome 8 (DRESH8) and ZmMYBR38, a target of DRESH8-derived small interfering RNAs, revealed a transposable element-mediated inverted repeats (TE-IR)-derived sRNA- and gene-regulatory network that balances plant drought tolerance with yield-related traits. A genome-wide scan revealed that TE-IRs associate with drought response and yield-related traits that were positively selected and expanded during maize domestication. These results indicate that TE-IR-mediated posttranscriptional regulation is a key molecular mechanism underlying the tradeoff between crop environmental adaptation and yield-related traits, providing potential genomic targets for the breeding of crops with greater stress tolerance but uncompromised yield. Transposon-associated inverted repeats play a role in drought resistance and yield-related traits in maize.},
  archive      = {J_NBT},
  author       = {Sun, Xiaopeng and Xiang, Yanli and Dou, Nannan and Zhang, Hui and Pei, Surui and Franco, Arcadio Valdes and Menon, Mitra and Monier, Brandon and Ferebee, Taylor and Liu, Tao and Liu, Sanyang and Gao, Yuchi and Wang, Jubin and Terzaghi, William and Yan, Jianbing and Hearne, Sarah and Li, Lin and Li, Feng and Dai, Mingqiu},
  doi          = {10.1038/s41587-022-01470-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {120-127},
  shortjournal = {Nature Biotech.},
  title        = {The role of transposon inverted repeats in balancing drought tolerance and yield-related traits in maize},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-fidelity cas13 variants for targeted RNA degradation
with minimal collateral effects. <em>NBT</em>, <em>41</em>(1), 108–119.
(<a href="https://doi.org/10.1038/s41587-022-01419-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas13 systems have recently been used for targeted RNA degradation in various organisms. However, collateral degradation of bystander RNAs has limited their in vivo applications. Here, we design a dual-fluorescence reporter system for detecting collateral effects and screening Cas13 variants in mammalian cells. Among over 200 engineered variants, several Cas13 variants including Cas13d and Cas13X exhibit efficient on-target activity but markedly reduced collateral activity. Furthermore, transcriptome-wide off-targets and cell growth arrest induced by Cas13 are absent for these variants. High-fidelity Cas13 variants show similar RNA knockdown activity to wild-type Cas13 but no detectable collateral damage in transgenic mice or adeno-associated-virus-mediated somatic cell targeting. Thus, high-fidelity Cas13 variants with minimal collateral effects are now available for targeted degradation of RNAs in basic research and therapeutic applications. Several engineered Cas13 variants achieve targeted RNA degradation with minimal collateral effects.},
  archive      = {J_NBT},
  author       = {Tong, Huawei and Huang, Jia and Xiao, Qingquan and He, Bingbing and Dong, Xue and Liu, Yuanhua and Yang, Xiali and Han, Dingyi and Wang, Zikang and Wang, Xuchen and Ying, Wenqin and Zhang, Runze and Wei, Yu and Xu, Chunlong and Zhou, Yingsi and Li, Yanfei and Cai, Minqing and Wang, Qifang and Xue, Mingxing and Li, Guoling and Fang, Kailun and Zhang, Hainan and Yang, Hui},
  doi          = {10.1038/s41587-022-01419-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {108-119},
  shortjournal = {Nature Biotech.},
  title        = {High-fidelity cas13 variants for targeted RNA degradation with minimal collateral effects},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-throughput continuous evolution of compact cas9
variants targeting single-nucleotide-pyrimidine PAMs. <em>NBT</em>,
<em>41</em>(1), 96–107. (<a
href="https://doi.org/10.1038/s41587-022-01410-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the availability of Cas9 variants with varied protospacer-adjacent motif (PAM) compatibilities, some genomic loci—especially those with pyrimidine-rich PAM sequences—remain inaccessible by high-activity Cas9 proteins. Moreover, broadening PAM sequence compatibility through engineering can increase off-target activity. With directed evolution, we generated four Cas9 variants that together enable targeting of most pyrimidine-rich PAM sequences in the human genome. Using phage-assisted noncontinuous evolution and eVOLVER-supported phage-assisted continuous evolution, we evolved Nme2Cas9, a compact Cas9 variant, into variants that recognize single-nucleotide pyrimidine-PAM sequences. We developed a general selection strategy that requires functional editing with fully specified target protospacers and PAMs. We applied this selection to evolve high-activity variants eNme2-T.1, eNme2-T.2, eNme2-C and eNme2-C.NR. Variants eNme2-T.1 and eNme2-T.2 offer access to N4TN PAM sequences with comparable editing efficiencies as existing variants, while eNme2-C and eNme2-C.NR offer less restrictive PAM requirements, comparable or higher activity in a variety of human cell types and lower off-target activity at N4CN PAM sequences. Directed evolution creates new compact Cas9 variants that target most single pyrimidine nucleotide PAMs.},
  archive      = {J_NBT},
  author       = {Huang, Tony P. and Heins, Zachary J. and Miller, Shannon M. and Wong, Brandon G. and Balivada, Pallavi A. and Wang, Tina and Khalil, Ahmad S. and Liu, David R.},
  doi          = {10.1038/s41587-022-01410-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {96-107},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput continuous evolution of compact cas9 variants targeting single-nucleotide-pyrimidine PAMs},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Removing unwanted variation from large-scale RNA sequencing
data with PRPS. <em>NBT</em>, <em>41</em>(1), 82–95. (<a
href="https://doi.org/10.1038/s41587-022-01440-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate identification and effective removal of unwanted variation is essential to derive meaningful biological results from RNA sequencing (RNA-seq) data, especially when the data come from large and complex studies. Using RNA-seq data from The Cancer Genome Atlas (TCGA), we examined several sources of unwanted variation and demonstrate here how these can significantly compromise various downstream analyses, including cancer subtype identification, association between gene expression and survival outcomes and gene co-expression analysis. We propose a strategy, called pseudo-replicates of pseudo-samples (PRPS), for deploying our recently developed normalization method, called removing unwanted variation III (RUV-III), to remove the variation caused by library size, tumor purity and batch effects in TCGA RNA-seq data. We illustrate the value of our approach by comparing it to the standard TCGA normalizations on several TCGA RNA-seq datasets. RUV-III with PRPS can be used to integrate and normalize other large transcriptomic datasets coming from multiple laboratories or platforms. Cancer RNA-seq data are normalized with respect to library size, tumor purity and batch effects.},
  archive      = {J_NBT},
  author       = {Molania, Ramyar and Foroutan, Momeneh and Gagnon-Bartsch, Johann A. and Gandolfo, Luke C. and Jain, Aryan and Sinha, Abhishek and Olshansky, Gavriel and Dobrovic, Alexander and Papenfuss, Anthony T. and Speed, Terence P.},
  doi          = {10.1038/s41587-022-01440-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {82-95},
  shortjournal = {Nature Biotech.},
  title        = {Removing unwanted variation from large-scale RNA sequencing data with PRPS},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Bioengineered corneal tissue for minimally invasive vision
restoration in advanced keratoconus in two clinical cohorts.
<em>NBT</em>, <em>41</em>(1), 70–81. (<a
href="https://doi.org/10.1038/s41587-022-01408-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Visual impairment from corneal stromal disease affects millions worldwide. We describe a cell-free engineered corneal tissue, bioengineered porcine construct, double crosslinked (BPCDX) and a minimally invasive surgical method for its implantation. In a pilot feasibility study in India and Iran (clinicaltrials.gov no. NCT04653922 ), we implanted BPCDX in 20 advanced keratoconus subjects to reshape the native corneal stroma without removing existing tissue or using sutures. During 24 months of follow-up, no adverse event was observed. We document improvements in corneal thickness (mean increase of 209 ± 18 µm in India, 285 ± 99 µm in Iran), maximum keratometry (mean decrease of 13.9 ± 7.9 D in India and 11.2 ± 8.9 D in Iran) and visual acuity (to a mean contact-lens-corrected acuity of 20/26 in India and spectacle-corrected acuity of 20/58 in Iran). Fourteen of 14 initially blind subjects had a final mean best-corrected vision (spectacle or contact lens) of 20/36 and restored tolerance to contact lens wear. This work demonstrates restoration of vision using an approach that is potentially equally effective, safer, simpler and more broadly available than donor cornea transplantation. A bioengineered cornea made of collagen restores vision in a pilot clinical study.},
  archive      = {J_NBT},
  author       = {Rafat, Mehrdad and Jabbarvand, Mahmoud and Sharma, Namrata and Xeroudaki, Maria and Tabe, Shideh and Omrani, Raha and Thangavelu, Muthukumar and Mukwaya, Anthony and Fagerholm, Per and Lennikov, Anton and Askarizadeh, Farshad and Lagali, Neil},
  doi          = {10.1038/s41587-022-01408-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {70-81},
  shortjournal = {Nature Biotech.},
  title        = {Bioengineered corneal tissue for minimally invasive vision restoration in advanced keratoconus in two clinical cohorts},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Uncovering the mode of action of engineered t cells in
patient cancer organoids. <em>NBT</em>, <em>41</em>(1), 60–69. (<a
href="https://doi.org/10.1038/s41587-022-01397-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extending the success of cellular immunotherapies against blood cancers to the realm of solid tumors will require improved in vitro models that reveal therapeutic modes of action at the molecular level. Here we describe a system, called BEHAV3D, developed to study the dynamic interactions of immune cells and patient cancer organoids by means of imaging and transcriptomics. We apply BEHAV3D to live-track &amp;gt;150,000 engineered T cells cultured with patient-derived, solid-tumor organoids, identifying a ‘super engager’ behavioral cluster comprising T cells with potent serial killing capacity. Among other T cell concepts we also study cancer metabolome-sensing engineered T cells (TEGs) and detect behavior-specific gene signatures that include a group of 27 genes with no previously described T cell function that are expressed by super engager killer TEGs. We further show that type I interferon can prime resistant organoids for TEG-mediated killing. BEHAV3D is a promising tool for the characterization of behavioral-phenotypic heterogeneity of cellular immunotherapies and may support the optimization of personalized solid-tumor-targeting cell therapies. The dynamics and molecular mechanisms of T cell therapies are probed in cancer organoids.},
  archive      = {J_NBT},
  author       = {Dekkers, Johanna F. and Alieva, Maria and Cleven, Astrid and Keramati, Farid and Wezenaar, Amber K. L. and van Vliet, Esmée J. and Puschhof, Jens and Brazda, Peter and Johanna, Inez and Meringa, Angelo D. and Rebel, Heggert G. and Buchholz, Maj-Britt and Barrera Román, Mario and Zeeman, Amber L. and de Blank, Sam and Fasci, Domenico and Geurts, Maarten H. and Cornel, Annelisa M. and Driehuis, Else and Millen, Rosemary and Straetemans, Trudy and Nicolasen, Mara J. T. and Aarts-Riemens, Tineke and Ariese, Hendrikus C. R. and Johnson, Hannah R. and van Ineveld, Ravian L. and Karaiskaki, Froso and Kopper, Oded and Bar-Ephraim, Yotam E. and Kretzschmar, Kai and Eggermont, Alexander M. M. and Nierkens, Stefan and Wehrens, Ellen J. and Stunnenberg, Henk G. and Clevers, Hans and Kuball, Jürgen and Sebestyen, Zsolt and Rios, Anne C.},
  doi          = {10.1038/s41587-022-01397-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {60-69},
  shortjournal = {Nature Biotech.},
  title        = {Uncovering the mode of action of engineered t cells in patient cancer organoids},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Increasing the throughput of sensitive proteomics by
plexDIA. <em>NBT</em>, <em>41</em>(1), 50–59. (<a
href="https://doi.org/10.1038/s41587-022-01389-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current mass spectrometry methods enable high-throughput proteomics of large sample amounts, but proteomics of low sample amounts remains limited in depth and throughput. To increase the throughput of sensitive proteomics, we developed an experimental and computational framework, called plexDIA, for simultaneously multiplexing the analysis of peptides and samples. Multiplexed analysis with plexDIA increases throughput multiplicatively with the number of labels without reducing proteome coverage or quantitative accuracy. By using three-plex non-isobaric mass tags, plexDIA enables quantification of threefold more protein ratios among nanogram-level samples. Using 1-hour active gradients, plexDIA quantified ~8,000 proteins in each sample of labeled three-plex sets and increased data completeness, reducing missing data more than twofold across samples. Applied to single human cells, plexDIA quantified ~1,000 proteins per cell and achieved 98\% data completeness within a plexDIA set while using ~5 minutes of active chromatography per cell. These results establish a general framework for increasing the throughput of sensitive and quantitative protein analysis. Proteomics of small sample sizes using data-independent acquisition methods achieves higher throughput with multiplexing.},
  archive      = {J_NBT},
  author       = {Derks, Jason and Leduc, Andrew and Wallmann, Georg and Huffman, R. Gray and Willetts, Matthew and Khan, Saad and Specht, Harrison and Ralser, Markus and Demichev, Vadim and Slavov, Nikolai},
  doi          = {10.1038/s41587-022-01389-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {50-59},
  shortjournal = {Nature Biotech.},
  title        = {Increasing the throughput of sensitive proteomics by plexDIA},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Automated reconstruction of whole-embryo cell lineages by
learning from sparse annotations. <em>NBT</em>, <em>41</em>(1), 44–49.
(<a href="https://doi.org/10.1038/s41587-022-01427-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a method to automatically identify and track nuclei in time-lapse microscopy recordings of entire developing embryos. The method combines deep learning and global optimization. On a mouse dataset, it reconstructs 75.8\% of cell lineages spanning 1 h, as compared to 31.8\% for the competing method. Our approach improves understanding of where and when cell fate decisions are made in developing embryos, tissues, and organs. Cell lineages in developing embryos are reconstructed from time-lapse microscopy images.},
  archive      = {J_NBT},
  author       = {Malin-Mayor, Caroline and Hirsch, Peter and Guignard, Leo and McDole, Katie and Wan, Yinan and Lemon, William C. and Kainmueller, Dagmar and Keller, Philipp J. and Preibisch, Stephan and Funke, Jan},
  doi          = {10.1038/s41587-022-01427-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {44-49},
  shortjournal = {Nature Biotech.},
  title        = {Automated reconstruction of whole-embryo cell lineages by learning from sparse annotations},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Prediction of peptide mass spectral libraries with machine
learning. <em>NBT</em>, <em>41</em>(1), 33–43. (<a
href="https://doi.org/10.1038/s41587-022-01424-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recent development of machine learning methods to identify peptides in complex mass spectrometric data constitutes a major breakthrough in proteomics. Longstanding methods for peptide identification, such as search engines and experimental spectral libraries, are being superseded by deep learning models that allow the fragmentation spectra of peptides to be predicted from their amino acid sequence. These new approaches, including recurrent neural networks and convolutional neural networks, use predicted in silico spectral libraries rather than experimental libraries to achieve higher sensitivity and/or specificity in the analysis of proteomics data. Machine learning is galvanizing applications that involve large search spaces, such as immunopeptidomics and proteogenomics. Current challenges in the field include the prediction of spectra for peptides with post-translational modifications and for cross-linked pairs of peptides. Permeation of machine-learning-based spectral prediction into search engines and spectrum-centric data-independent acquisition workflows for diverse peptide classes and measurement conditions will continue to push sensitivity and dynamic range in proteomics applications in the coming years. Proteomics is being transformed by deep learning methods that predict peptide fragmentation spectra.},
  archive      = {J_NBT},
  author       = {Cox, Jürgen},
  doi          = {10.1038/s41587-022-01424-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {33-43},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of peptide mass spectral libraries with machine learning},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). BEHAV3D: An imaging and transcriptomics platform that
unravels t cell antitumor activity. <em>NBT</em>, <em>41</em>(1), 31–32.
(<a href="https://doi.org/10.1038/s41587-022-01398-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study reveals a high level of behavioral heterogeneity in engineered T cells that target tumor organoids. Behavior-guided transcriptomics identified the gene signatures of T cells with highly potent serial cytotoxicity. Integration of the organoid and T cell transcriptomic data enabled the design of patient-specific combinatorial treatment to achieve an optimized response to cancer immunotherapies.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01398-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {31-32},
  shortjournal = {Nature Biotech.},
  title        = {BEHAV3D: An imaging and transcriptomics platform that unravels t cell antitumor activity},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineered cas13 variants with minimal collateral RNA
targeting. <em>NBT</em>, <em>41</em>(1), 29–30. (<a
href="https://doi.org/10.1038/s41587-022-01423-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The application of CRISPR–Cas13 for targeted RNA degradation is hindered by the existence of collateral effects induced by Cas13. We screened hundreds of engineered Cas13 variants using a convenient and sensitive reporter system and obtained several variants with markedly reduced collateral activity and efficient on-target activity, suitable for in vivo applications.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01423-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {29-30},
  shortjournal = {Nature Biotech.},
  title        = {Engineered cas13 variants with minimal collateral RNA targeting},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Normalizing cancer RNA-seq data for library size, tumor
purity and batch effects. <em>NBT</em>, <em>41</em>(1), 27–28. (<a
href="https://doi.org/10.1038/s41587-022-01441-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate identification and effective removal of unwanted variation is essential to derive meaningful biological results from large and complex RNA-seq studies. Technical replicates together with negative and positive control genes are key tools for carrying out this task. We show how to proceed when technical replicates are unavailable.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01441-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {27-28},
  shortjournal = {Nature Biotech.},
  title        = {Normalizing cancer RNA-seq data for library size, tumor purity and batch effects},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accessible bioengineered corneal tissue to address a
blinding disease globally. <em>NBT</em>, <em>41</em>(1), 25–26. (<a
href="https://doi.org/10.1038/s41587-022-01409-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Porcine dermal collagen was chemically and photochemically bioengineered into an implantable tissue mimicking the human corneal extracellular matrix. The implant presents a simpler and safer method than donor cornea transplantation while delivering equivalent outcomes, and has restored vision to people with advanced keratoconus in resource-limited regions, where the burden of blindness is highest.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01409-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {25-26},
  shortjournal = {Nature Biotech.},
  title        = {Accessible bioengineered corneal tissue to address a blinding disease globally},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Framework for multiplicative scaling of single-cell
proteomics. <em>NBT</em>, <em>41</em>(1), 23–24. (<a
href="https://doi.org/10.1038/s41587-022-01411-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many biomedical questions demand scalable, deep, and accurate proteome analysis of small samples, including single cells. A scalable framework of multiplexed data-independent acquisition for mass spectrometry enables time saving by parallel analysis of both peptide ions and protein samples, thereby realizing multiplicative gains in throughput.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01411-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {23-24},
  shortjournal = {Nature Biotech.},
  title        = {Framework for multiplicative scaling of single-cell proteomics},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Forum: Locke and sykes. <em>NBT</em>, <em>41</em>(1), 22.
(<a href="https://doi.org/10.1038/s41587-022-01653-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01653-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {22},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Locke and sykes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Transfer of a plant-derived photosynthetic system to
mammalian cells. <em>NBT</em>, <em>41</em>(1), 21. (<a
href="https://doi.org/10.1038/s41587-022-01645-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Doerr, Anne},
  doi          = {10.1038/s41587-022-01645-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {21},
  shortjournal = {Nature Biotech.},
  title        = {Transfer of a plant-derived photosynthetic system to mammalian cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Measuring elusive protein complexes in single cells.
<em>NBT</em>, <em>41</em>(1), 21. (<a
href="https://doi.org/10.1038/s41587-022-01646-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Cheifet, Barbara},
  doi          = {10.1038/s41587-022-01646-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {21},
  shortjournal = {Nature Biotech.},
  title        = {Measuring elusive protein complexes in single cells},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Linking genetic variants to cellular function and disease.
<em>NBT</em>, <em>41</em>(1), 21. (<a
href="https://doi.org/10.1038/s41587-022-01647-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Despang, Alexandra},
  doi          = {10.1038/s41587-022-01647-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {21},
  shortjournal = {Nature Biotech.},
  title        = {Linking genetic variants to cellular function and disease},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Intellectual property and assisted reproductive technology.
<em>NBT</em>, <em>41</em>(1), 14–20. (<a
href="https://doi.org/10.1038/s41587-022-01592-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past fifty years, intellectual property has not played a major role in the spread of assisted reproductive technology, but with in vitro gametogenesis — a technique likely to dominate the future of reproduction — it might.},
  archive      = {J_NBT},
  author       = {Cyranoski, David and Contreras, Jorge L. and Carrington, Victoria T.},
  doi          = {10.1038/s41587-022-01592-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {14-20},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property and assisted reproductive technology},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Exploring the value of a global gene drive project registry.
<em>NBT</em>, <em>41</em>(1), 9–13. (<a
href="https://doi.org/10.1038/s41587-022-01591-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Taitingfong, Riley I. and Triplett, Cynthia and Vásquez, Váleri N. and Rajagopalan, Ramya M. and Raban, Robyn and Roberts, Aaron and Terradas, Gerard and Baumgartner, Bridget and Emerson, Claudia and Gould, Fred and Okumu, Fredros and Schairer, Cynthia E. and Bossin, Hervé C. and Buchman, Leah and Campbell, Karl J. and Clark, Anna and Delborne, Jason and Esvelt, Kevin and Fisher, Joshua and Friedman, Robert M. and Gronvall, Gigi and Gurfield, Nikos and Heitman, Elizabeth and Kofler, Natalie and Kuiken, Todd and Kuzma, Jennifer and Manrique-Saide, Pablo and Marshall, John M. and Montague, Michael and Morrison, Amy C. and Opesen, Chris C. and Phelan, Ryan and Piaggio, Antoinette and Quemada, Hector and Rudenko, Larisa and Sawadogo, Natéwindé and Smith, Robert and Tuten, Holly and Ullah, Anika and Vorsino, Adam and Windbichler, Nikolai and Akbari, Omar S. and Long, Kanya and Lavery, James V. and Evans, Sam Weiss and Tountas, Karen and Bloss, Cinnamon S.},
  doi          = {10.1038/s41587-022-01591-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {9-13},
  shortjournal = {Nature Biotech.},
  title        = {Exploring the value of a global gene drive project registry},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). Biotech news from around the world. <em>NBT</em>,
<em>41</em>(1), 7. (<a
href="https://doi.org/10.1038/s41587-022-01642-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01642-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {7},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Drug for delaying diabetes wins landmark approval.
<em>NBT</em>, <em>41</em>(1), 6–7. (<a
href="https://doi.org/10.1038/s41587-022-01633-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The November approval of Tzield (teplizumab) marks a turning point in the management of type 1 diabetes.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01633-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {6-7},
  shortjournal = {Nature Biotech.},
  title        = {Drug for delaying diabetes wins landmark approval},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chikungunya and lyme vaccines make headway. <em>NBT</em>,
<em>41</em>(1), 6. (<a
href="https://doi.org/10.1038/s41587-022-01641-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01641-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {6},
  shortjournal = {Nature Biotech.},
  title        = {Chikungunya and lyme vaccines make headway},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). FDA okays first human stool therapy. <em>NBT</em>,
<em>41</em>(1), 5. (<a
href="https://doi.org/10.1038/s41587-022-01640-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01640-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {5},
  shortjournal = {Nature Biotech.},
  title        = {FDA okays first human stool therapy},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). UK launches whole-genome sequencing pilot for babies.
<em>NBT</em>, <em>41</em>(1), 4. (<a
href="https://doi.org/10.1038/s41587-022-01644-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01644-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nature Biotech.},
  title        = {UK launches whole-genome sequencing pilot for babies},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Illumina faces short-read rivals. <em>NBT</em>,
<em>41</em>(1), 3–5. (<a
href="https://doi.org/10.1038/s41587-022-01632-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New sequencers are poised to disrupt Illumina’s dominance. Users will need to navigate the opportunities and challenges on offer in an increasingly crowded market.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-022-01632-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {3-5},
  shortjournal = {Nature Biotech.},
  title        = {Illumina faces short-read rivals},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The next giant step for microbes. <em>NBT</em>,
<em>41</em>(1), 1. (<a
href="https://doi.org/10.1038/s41587-022-01655-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A microbiome therapy is approved by the FDA for treatment of Clostridioides difficile. What’s next?},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01655-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nature Biotech.},
  title        = {The next giant step for microbes},
  volume       = {41},
  year         = {2023},
}
</textarea>
</details></li>
</ul>

</body>
</html>
