<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng---233">NG - 233</h2>
<ul>
<li><details>
<summary>
(2021b). Publisher correction: BRD4 orchestrates genome folding to
promote neural crest differentiation. <em>NG</em>, <em>53</em>(12),
1723. (<a href="https://doi.org/10.1038/s41588-021-00980-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Linares-Saldana, Ricardo and Kim, Wonho and Bolar, Nikhita A. and Zhang, Haoyue and Koch-Bojalad, Bailey A. and Yoon, Sora and Shah, Parisha P. and Karnay, Ashley and Park, Daniel S. and Luppino, Jennifer M. and Nguyen, Son C. and Padmanabhan, Arun and Smith, Cheryl L. and Poleshko, Andrey and Wang, Qiaohong and Li, Li and Srivastava, Deepak and Vahedi, Golnaz and Eom, Gwang Hyeon and Blobel, Gerd A. and Joyce, Eric F. and Jain, Rajan},
  doi          = {10.1038/s41588-021-00980-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1723},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: BRD4 orchestrates genome folding to promote neural crest differentiation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Author correction: A genome-wide association study with
1,126,563 individuals identifies new risk loci for alzheimer’s disease.
<em>NG</em>, <em>53</em>(12), 1722. (<a
href="https://doi.org/10.1038/s41588-021-00977-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Wightman, Douglas P. and Jansen, Iris E. and Savage, Jeanne E. and Shadrin, Alexey A. and Bahrami, Shahram and Holland, Dominic and Rongve, Arvid and Børte, Sigrid and Winsvold, Bendik S. and Drange, Ole Kristian and Martinsen, Amy E. and Skogholt, Anne Heidi and Willer, Cristen and Bråthen, Geir and Bosnes, Ingunn and Nielsen, Jonas Bille and Fritsche, Lars G. and Thomas, Laurent F. and Pedersen, Linda M. and Gabrielsen, Maiken E. and Johnsen, Marianne Bakke and Meisingset, Tore Wergeland and Zhou, Wei and Proitsi, Petroula and Hodges, Angela and Dobson, Richard and Velayudhan, Latha and Heilbron, Karl and Auton, Adam and Sealock, Julia M. and Davis, Lea K. and Pedersen, Nancy L. and Reynolds, Chandra A. and Karlsson, Ida K. and Magnusson, Sigurdur and Stefansson, Hreinn and Thordardottir, Steinunn and Jonsson, Palmi V. and Snaedal, Jon and Zettergren, Anna and Skoog, Ingmar and Kern, Silke and Waern, Margda and Zetterberg, Henrik and Blennow, Kaj and Stordal, Eystein and Hveem, Kristian and Zwart, John-Anker and Athanasiu, Lavinia and Selnes, Per and Saltvedt, Ingvild and Sando, Sigrid B. and Ulstein, Ingun and Djurovic, Srdjan and Fladby, Tormod and Aarsland, Dag and Selbæk, Geir and Ripke, Stephan and Stefansson, Kari and Andreassen, Ole A. and Posthuma, Danielle},
  doi          = {10.1038/s41588-021-00977-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1722},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A genome-wide association study with 1,126,563 individuals identifies new risk loci for alzheimer’s disease},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Author correction: A high-resolution HLA reference panel
capturing global population diversity enables multi-ancestry
fine-mapping in HIV host response. <em>NG</em>, <em>53</em>(12), 1722.
(<a href="https://doi.org/10.1038/s41588-021-00979-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Luo, Yang and Kanai, Masahiro and Choi, Wanson and Li, Xinyi and Sakaue, Saori and Yamamoto, Kenichi and Ogawa, Kotaro and Gutierrez-Arcelus, Maria and Gregersen, Peter K. and Stuart, Philip E. and Elder, James T. and Forer, Lukas and Schönherr, Sebastian and Fuchsberger, Christian and Smith, Albert V. and Fellay, Jacques and Carrington, Mary and Haas, David W. and Guo, Xiuqing and Palmer, Nicholette D. and Chen, Yii-Der Ida and Rotter, Jerome I. and Taylor, Kent D. and Rich, Stephen S. and Correa, Adolfo and Wilson, James G. and Kathiresan, Sekar and Cho, Michael H. and Metspalu, Andres and Esko, Tonu and Okada, Yukinori and Han, Buhm and McLaren, Paul J. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-021-00979-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1722},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Large-scale integration of the plasma proteome with genetics
and disease. <em>NG</em>, <em>53</em>(12), 1712–1721. (<a
href="https://doi.org/10.1038/s41588-021-00978-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The plasma proteome can help bridge the gap between the genome and diseases. Here we describe genome-wide association studies (GWASs) of plasma protein levels measured with 4,907 aptamers in 35,559 Icelanders. We found 18,084 associations between sequence variants and levels of proteins in plasma (protein quantitative trait loci; pQTL), of which 19% were with rare variants (minor allele frequency (MAF) &amp;lt; 1%). We tested plasma protein levels for association with 373 diseases and other traits and identified 257,490 associations. We integrated pQTL and genetic associations with diseases and other traits and found that 12% of 45,334 lead associations in the GWAS Catalog are with variants in high linkage disequilibrium with pQTL. We identified 938 genes encoding potential drug targets with variants that influence levels of possible biomarkers. Combining proteomics, genomics and transcriptomics, we provide a valuable resource that can be used to improve understanding of disease pathogenesis and to assist with drug discovery and development. A genome-wide association study of plasma protein levels measured with 4,907 aptamers in 35,559 Icelanders highlights links with over 370 disease endpoints and other traits.},
  archive      = {J_NG},
  author       = {Ferkingstad, Egil and Sulem, Patrick and Atlason, Bjarni A. and Sveinbjornsson, Gardar and Magnusson, Magnus I. and Styrmisdottir, Edda L. and Gunnarsdottir, Kristbjorg and Helgason, Agnar and Oddsson, Asmundur and Halldorsson, Bjarni V. and Jensson, Brynjar O. and Zink, Florian and Halldorsson, Gisli H. and Masson, Gisli and Arnadottir, Gudny A. and Katrinardottir, Hildigunnur and Juliusson, Kristinn and Magnusson, Magnus K. and Magnusson, Olafur Th. and Fridriksdottir, Run and Saevarsdottir, Saedis and Gudjonsson, Sigurjon A. and Stacey, Simon N. and Rognvaldsson, Solvi and Eiriksdottir, Thjodbjorg and Olafsdottir, Thorunn A. and Steinthorsdottir, Valgerdur and Tragante, Vinicius and Ulfarsson, Magnus O. and Stefansson, Hreinn and Jonsdottir, Ingileif and Holm, Hilma and Rafnar, Thorunn and Melsted, Pall and Saemundsdottir, Jona and Norddahl, Gudmundur L. and Lund, Sigrun H. and Gudbjartsson, Daniel F. and Thorsteinsdottir, Unnur and Stefansson, Kari},
  doi          = {10.1038/s41588-021-00978-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1712-1721},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale integration of the plasma proteome with genetics and disease},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mapping the temporal and spatial dynamics of the human
endometrium in vivo and in vitro. <em>NG</em>, <em>53</em>(12),
1698–1711. (<a
href="https://doi.org/10.1038/s41588-021-00972-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The endometrium, the mucosal lining of the uterus, undergoes dynamic changes throughout the menstrual cycle in response to ovarian hormones. We have generated dense single-cell and spatial reference maps of the human uterus and three-dimensional endometrial organoid cultures. We dissect the signaling pathways that determine cell fate of the epithelial lineages in the lumenal and glandular microenvironments. Our benchmark of the endometrial organoids reveals the pathways and cell states regulating differentiation of the secretory and ciliated lineages both in vivo and in vitro. In vitro downregulation of WNT or NOTCH pathways increases the differentiation efficiency along the secretory and ciliated lineages, respectively. We utilize our cellular maps to deconvolute bulk data from endometrial cancers and endometriotic lesions, illuminating the cell types dominating in each of these disorders. These mechanistic insights provide a platform for future development of treatments for common conditions including endometriosis and endometrial carcinoma. Single-cell and spatial transcriptomic profiling of the human endometrium highlights pathways governing the proliferative and secretory phases of the menstrual cycle. Analyses of endometrial organoids show that WNT and NOTCH signaling modulate differentiation into the secretory and ciliated epithelial lineages, respectively.},
  archive      = {J_NG},
  author       = {Garcia-Alonso, Luz and Handfield, Louis-François and Roberts, Kenny and Nikolakopoulou, Konstantina and Fernando, Ridma C. and Gardner, Lucy and Woodhams, Benjamin and Arutyunyan, Anna and Polanski, Krzysztof and Hoo, Regina and Sancho-Serra, Carmen and Li, Tong and Kwakwa, Kwasi and Tuck, Elizabeth and Lorenzi, Valentina and Massalha, Hassan and Prete, Martin and Kleshchevnikov, Vitalii and Tarkowska, Aleksandra and Porter, Tarryn and Mazzeo, Cecilia Icoresi and van Dongen, Stijn and Dabrowska, Monika and Vaskivskyi, Vasyl and Mahbubani, Krishnaa T. and Park, Jong-eun and Jimenez-Linan, Mercedes and Campos, Lia and Kiselev, Vladimir Yu. and Lindskog, Cecilia and Ayuk, Paul and Prigmore, Elena and Stratton, Michael R. and Saeb-Parsy, Kourosh and Moffett, Ashley and Moore, Luiza and Bayraktar, Omer A. and Teichmann, Sarah A. and Turco, Margherita Y. and Vento-Tormo, Roser},
  doi          = {10.1038/s41588-021-00972-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1698-1711},
  shortjournal = {Nat. Genet.},
  title        = {Mapping the temporal and spatial dynamics of the human endometrium in vivo and in vitro},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A cis-acting mechanism mediates transcriptional memory at
polycomb target genes in mammals. <em>NG</em>, <em>53</em>(12),
1686–1697. (<a
href="https://doi.org/10.1038/s41588-021-00964-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetic inheritance of gene expression states enables a single genome to maintain distinct cellular identities. How histone modifications contribute to this process remains unclear. Using global chromatin perturbations and local, time-controlled modulation of transcription, we establish the existence of epigenetic memory of transcriptional activation for genes that can be silenced by the Polycomb group. This property emerges during cell differentiation and allows genes to be stably switched after a transient transcriptional stimulus. This transcriptional memory state at Polycomb targets operates in cis; however, rather than relying solely on read-and-write propagation of histone modifications, the memory is also linked to the strength of activating inputs opposing Polycomb proteins, and therefore varies with the cellular context. Our data and computational simulations suggest a model whereby transcriptional memory arises from double-negative feedback between Polycomb-mediated silencing and active transcription. Transcriptional memory at Polycomb targets thus depends not only on histone modifications but also on the gene-regulatory network and underlying identity of a cell. Polycomb proteins can regulate epigenetic transcriptional memory during cell differentiation. This memory operates in cis and is linked to the strength of activating inputs.},
  archive      = {J_NG},
  author       = {Holoch, Daniel and Wassef, Michel and Lövkvist, Cecilia and Zielinski, Dina and Aflaki, Setareh and Lombard, Bérangère and Héry, Tiphaine and Loew, Damarys and Howard, Martin and Margueron, Raphaël},
  doi          = {10.1038/s41588-021-00964-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1686-1697},
  shortjournal = {Nat. Genet.},
  title        = {A cis-acting mechanism mediates transcriptional memory at polycomb target genes in mammals},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chromothripsis followed by circular recombination drives
oncogene amplification in human cancer. <em>NG</em>, <em>53</em>(12),
1673–1685. (<a
href="https://doi.org/10.1038/s41588-021-00951-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms behind the evolution of complex genomic amplifications in cancer have remained largely unclear. Using whole-genome sequencing data of the pediatric tumor neuroblastoma, we here identified a type of amplification, termed ‘seismic amplification’, that is characterized by multiple rearrangements and discontinuous copy number levels. Overall, seismic amplifications occurred in 9.9% (274 of 2,756) of cases across 38 cancer types, and were associated with massively increased copy numbers and elevated oncogene expression. Reconstruction of the development of seismic amplification showed a stepwise evolution, starting with a chromothripsis event, followed by formation of circular extrachromosomal DNA that subsequently underwent repetitive rounds of circular recombination. The resulting amplicons persisted as extrachromosomal DNA circles or had reintegrated into the genome in overt tumors. Together, our data indicate that the sequential occurrence of chromothripsis and circular recombination drives oncogene amplification and overexpression in a substantial fraction of human malignancies. Seismic amplifications arise from several cycles of circular recombination of circular extrachromosomal DNA formed as a result of chromothripsis. The process provides a mechanism for oncogene amplification in a number of different human tumor types.},
  archive      = {J_NG},
  author       = {Rosswog, Carolina and Bartenhagen, Christoph and Welte, Anne and Kahlert, Yvonne and Hemstedt, Nadine and Lorenz, Witali and Cartolano, Maria and Ackermann, Sandra and Perner, Sven and Vogel, Wenzel and Altmüller, Janine and Nürnberg, Peter and Hertwig, Falk and Göhring, Gudrun and Lilienweiss, Esther and Stütz, Adrian M. and Korbel, Jan O. and Thomas, Roman K. and Peifer, Martin and Fischer, Matthias},
  doi          = {10.1038/s41588-021-00951-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1673-1685},
  shortjournal = {Nat. Genet.},
  title        = {Chromothripsis followed by circular recombination drives oncogene amplification in human cancer},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Paralog knockout profiling identifies DUSP4 and DUSP6 as a
digenic dependence in MAPK pathway-driven cancers. <em>NG</em>,
<em>53</em>(12), 1664–1672. (<a
href="https://doi.org/10.1038/s41588-021-00967-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although single-gene perturbation screens have revealed a number of new targets, vulnerabilities specific to frequently altered drivers have not been uncovered. An important question is whether the compensatory relationship between functionally redundant genes masks potential therapeutic targets in single-gene perturbation studies. To identify digenic dependencies, we developed a CRISPR paralog targeting library to investigate the viability effects of disrupting 3,284 genes, 5,065 paralog pairs and 815 paralog families. We identified that dual inactivation of DUSP4 and DUSP6 selectively impairs growth in NRAS and BRAF mutant cells through the hyperactivation of MAPK signaling. Furthermore, cells resistant to MAPK pathway therapeutics become cross-sensitized to DUSP4 and DUSP6 perturbations such that the mechanisms of resistance to the inhibitors reinforce this mechanism of vulnerability. Together, multigene perturbation technologies unveil previously unrecognized digenic vulnerabilities that may be leveraged as new therapeutic targets in cancer. A CRISPR paralog targeting library profiling 815 paralog families across 11 cell lines identifies DUSP4 and DUSP6 as paralog pairs whose combined inactivation confers sensitivity to cells resistant to MAPK inhibitors or cells harboring NRAS or BRAF mutations.},
  archive      = {J_NG},
  author       = {Ito, Takahiro and Young, Michael J. and Li, Ruitong and Jain, Sidharth and Wernitznig, Andreas and Krill-Burger, John M. and Lemke, Christopher T. and Monducci, Davide and Rodriguez, Diego J. and Chang, Liang and Dutta, Sanjukta and Pal, Debjani and Paolella, Brenton R. and Rothberg, Michael V. and Root, David E. and Johannessen, Cory M. and Parida, Laxmi and Getz, Gad and Vazquez, Francisca and Doench, John G. and Zamanighomi, Mahdi and Sellers, William R.},
  doi          = {10.1038/s41588-021-00967-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1664-1672},
  shortjournal = {Nat. Genet.},
  title        = {Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Renal plasticity revealed through reversal of polycystic
kidney disease in mice. <em>NG</em>, <em>53</em>(12), 1649–1663. (<a
href="https://doi.org/10.1038/s41588-021-00946-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Initiation of cyst formation in autosomal dominant polycystic kidney disease (ADPKD) occurs when kidney tubule cells are rendered null for either PKD1 or PKD2 by somatic ‘second hit’ mutations. Subsequent cyst progression remodels the organ through changes in tubule cell shape, proliferation and secretion. The kidney develops inflammation and fibrosis. We constructed a mouse model in which adult inactivation of either Pkd gene can be followed by reactivation of the gene at a later time. Using this model, we show that re-expression of Pkd genes in cystic kidneys results in rapid reversal of ADPKD. Cyst cell proliferation is reduced, autophagy is activated and cystic tubules with expanded lumina lined by squamoid cells revert to normal lumina lined by cuboidal cells. Increases in inflammation, extracellular matrix deposition and myofibroblast activation are reversed, and the kidneys become smaller. We conclude that phenotypic features of ADPKD are reversible and that the kidney has an unexpected capacity for plasticity controlled at least in part by ADPKD gene function. Re-expression of Pkd genes in cystic kidneys results in rapid reversal of autosomal dominant polycystic kidney disease phenotypes in mice, revealing an unexpected capacity for renal plasticity under the control of Pkd gene function.},
  archive      = {J_NG},
  author       = {Dong, Ke and Zhang, Chao and Tian, Xin and Coman, Daniel and Hyder, Fahmeed and Ma, Ming and Somlo, Stefan},
  doi          = {10.1038/s41588-021-00946-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1649-1663},
  shortjournal = {Nat. Genet.},
  title        = {Renal plasticity revealed through reversal of polycystic kidney disease in mice},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Common and rare variant association analyses in amyotrophic
lateral sclerosis identify 15 risk loci with distinct genetic
architectures and neuron-specific biology. <em>NG</em>, <em>53</em>(12),
1636–1648. (<a
href="https://doi.org/10.1038/s41588-021-00973-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease with a lifetime risk of one in 350 people and an unmet need for disease-modifying therapies. We conducted a cross-ancestry genome-wide association study (GWAS) including 29,612 patients with ALS and 122,656 controls, which identified 15 risk loci. When combined with 8,953 individuals with whole-genome sequencing (6,538 patients, 2,415 controls) and a large cortex-derived expression quantitative trait locus (eQTL) dataset (MetaBrain), analyses revealed locus-specific genetic architectures in which we prioritized genes either through rare variants, short tandem repeats or regulatory effects. ALS-associated risk loci were shared with multiple traits within the neurodegenerative spectrum but with distinct enrichment patterns across brain regions and cell types. Of the environmental and lifestyle risk factors obtained from the literature, Mendelian randomization analyses indicated a causal role for high cholesterol levels. The combination of all ALS-associated signals reveals a role for perturbations in vesicle-mediated transport and autophagy and provides evidence for cell-autonomous disease initiation in glutamatergic neurons. A cross-ancestry genome-wide association meta-analysis of amyotrophic lateral sclerosis (ALS) including 29,612 patients with ALS and 122,656 controls identifies 15 risk loci with distinct genetic architectures and neuron-specific biology.},
  archive      = {J_NG},
  author       = {van Rheenen, Wouter and van der Spek, Rick A. A. and Bakker, Mark K. and van Vugt, Joke J. F. A. and Hop, Paul J. and Zwamborn, Ramona A. J. and de Klein, Niek and Westra, Harm-Jan and Bakker, Olivier B. and Deelen, Patrick and Shireby, Gemma and Hannon, Eilis and Moisse, Matthieu and Baird, Denis and Restuadi, Restuadi and Dolzhenko, Egor and Dekker, Annelot M. and Gawor, Klara and Westeneng, Henk-Jan and Tazelaar, Gijs H. P. and van Eijk, Kristel R. and Kooyman, Maarten and Byrne, Ross P. and Doherty, Mark and Heverin, Mark and Al Khleifat, Ahmad and Iacoangeli, Alfredo and Shatunov, Aleksey and Ticozzi, Nicola and Cooper-Knock, Johnathan and Smith, Bradley N. and Gromicho, Marta and Chandran, Siddharthan and Pal, Suvankar and Morrison, Karen E. and Shaw, Pamela J. and Hardy, John and Orrell, Richard W. and Sendtner, Michael and Meyer, Thomas and Başak, Nazli and van der Kooi, Anneke J. and Ratti, Antonia and Fogh, Isabella and Gellera, Cinzia and Lauria, Giuseppe and Corti, Stefania and Cereda, Cristina and Sproviero, Daisy and D’Alfonso, Sandra and Sorarù, Gianni and Siciliano, Gabriele and Filosto, Massimiliano and Padovani, Alessandro and Chiò, Adriano and Calvo, Andrea and Moglia, Cristina and Brunetti, Maura and Canosa, Antonio and Grassano, Maurizio and Beghi, Ettore and Pupillo, Elisabetta and Logroscino, Giancarlo and Nefussy, Beatrice and Osmanovic, Alma and Nordin, Angelica and Lerner, Yossef and Zabari, Michal and Gotkine, Marc and Baloh, Robert H. and Bell, Shaughn and Vourc’h, Patrick and Corcia, Philippe and Couratier, Philippe and Millecamps, Stéphanie and Meininger, Vincent and Salachas, François and Mora Pardina, Jesus S. and Assialioui, Abdelilah and Rojas-García, Ricardo and Dion, Patrick A. and Ross, Jay P. and Ludolph, Albert C. and Weishaupt, Jochen H. and Brenner, David and Freischmidt, Axel and Bensimon, Gilbert and Brice, Alexis and Durr, Alexandra and Payan, Christine A. M. and Saker-Delye, Safa and Wood, Nicholas W. and Topp, Simon and Rademakers, Rosa and Tittmann, Lukas and Lieb, Wolfgang and Franke, Andre and Ripke, Stephan and Braun, Alice and Kraft, Julia and Whiteman, David C. and Olsen, Catherine M. and Uitterlinden, Andre G. and Hofman, Albert and Rietschel, Marcella and Cichon, Sven and Nöthen, Markus M. and Amouyel, Philippe and Traynor, Bryan J. and Singleton, Andrew B. and Mitne Neto, Miguel and Cauchi, Ruben J. and Ophoff, Roel A. and Wiedau-Pazos, Martina and Lomen-Hoerth, Catherine and van Deerlin, Vivianna M. and Grosskreutz, Julian and Roediger, Annekathrin and Gaur, Nayana and Jörk, Alexander and Barthel, Tabea and Theele, Erik and Ilse, Benjamin and Stubendorff, Beatrice and Witte, Otto W. and Steinbach, Robert and Hübner, Christian A. and Graff, Caroline and Brylev, Lev and Fominykh, Vera and Demeshonok, Vera and Ataulina, Anastasia and Rogelj, Boris and Koritnik, Blaž and Zidar, Janez and Ravnik-Glavač, Metka and Glavač, Damjan and Stević, Zorica and Drory, Vivian and Povedano, Monica and Blair, Ian P. and Kiernan, Matthew C. and Benyamin, Beben and Henderson, Robert D. and Furlong, Sarah and Mathers, Susan and McCombe, Pamela A. and Needham, Merrilee and Ngo, Shyuan T. and Nicholson, Garth A. and Pamphlett, Roger and Rowe, Dominic B. and Steyn, Frederik J. and Williams, Kelly L. and Mather, Karen A. and Sachdev, Perminder S. and Henders, Anjali K. and Wallace, Leanne and de Carvalho, Mamede and Pinto, Susana and Petri, Susanne and Weber, Markus and Rouleau, Guy A. and Silani, Vincenzo and Curtis, Charles J. and Breen, Gerome and Glass, Jonathan D. and Brown, Robert H. and Landers, John E. and Shaw, Christopher E. and Andersen, Peter M. and Groen, Ewout J. N. and van Es, Michael A. and Pasterkamp, R. Jeroen and Fan, Dongsheng and Garton, Fleur C. and McRae, Allan F. and Davey Smith, George and Gaunt, Tom R. and Eberle, Michael A. and Mill, Jonathan and McLaughlin, Russell L. and Hardiman, Orla and Kenna, Kevin P. and Wray, Naomi R. and Tsai, Ellen and Runz, Heiko and Franke, Lude and Al-Chalabi, Ammar and Van Damme, Philip and van den Berg, Leonard H. and Veldink, Jan H.},
  doi          = {10.1038/s41588-021-00973-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1636-1648},
  shortjournal = {Nat. Genet.},
  title        = {Common and rare variant association analyses in amyotrophic lateral sclerosis identify 15 risk loci with distinct genetic architectures and neuron-specific biology},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: On powerful GWAS in admixed populations.
<em>NG</em>, <em>53</em>(12), 1634–1635. (<a
href="https://doi.org/10.1038/s41588-021-00975-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Atkinson, Elizabeth G. and Bloemendal, Alex and Maihofer, Adam X. and Nievergelt, Caroline M. and Daly, Mark J. and Neale, Benjamin M.},
  doi          = {10.1038/s41588-021-00975-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1634-1635},
  shortjournal = {Nat. Genet.},
  title        = {Reply to: On powerful GWAS in admixed populations},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). On powerful GWAS in admixed populations. <em>NG</em>,
<em>53</em>(12), 1631–1633. (<a
href="https://doi.org/10.1038/s41588-021-00953-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Hou, Kangcheng and Bhattacharya, Arjun and Mester, Rachel and Burch, Kathryn S. and Pasaniuc, Bogdan},
  doi          = {10.1038/s41588-021-00953-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1631-1633},
  shortjournal = {Nat. Genet.},
  title        = {On powerful GWAS in admixed populations},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Using advanced spatial and single-cell transcriptomics to
characterize the human endometrium. <em>NG</em>, <em>53</em>(12),
1628–1630. (<a
href="https://doi.org/10.1038/s41588-021-00982-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human endometrium exhibits complex signaling cascades mediated by spatial–temporal cellular interactions. Understanding the normal endometrial microenvironment is the first step towards understanding diseases of endometrial dysfunction.},
  archive      = {J_NG},
  author       = {Wang, Xiyin and Hawkins, Shannon M.},
  doi          = {10.1038/s41588-021-00982-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1628-1630},
  shortjournal = {Nat. Genet.},
  title        = {Using advanced spatial and single-cell transcriptomics to characterize the human endometrium},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Decoding complex patterns of oncogene amplification.
<em>NG</em>, <em>53</em>(12), 1626–1627. (<a
href="https://doi.org/10.1038/s41588-021-00974-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oncogene amplification is a major driver of tumorigenesis; yet, the mechanisms generating amplification are only partially understood. New research reports on the identification of a new focal amplification pattern termed ‘seismic amplification’ that is hypothesized to originate from recombination between extrachromosomal DNA circles.},
  archive      = {J_NG},
  author       = {Pellman, David and Zhang, Cheng-Zhong},
  doi          = {10.1038/s41588-021-00974-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1626-1627},
  shortjournal = {Nat. Genet.},
  title        = {Decoding complex patterns of oncogene amplification},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Translating amyotrophic lateral sclerosis genes into drug
development leads. <em>NG</em>, <em>53</em>(12), 1624–1626. (<a
href="https://doi.org/10.1038/s41588-021-00981-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although clear genetic connections to ALS were first established three decades ago, there has been negligible progress in the development of disease-modifying treatments for this disease. Despite tremendous unmet need, industry has often been largely baffled by a disease seemingly designed to thwart current effective drug development approaches. In the largest genetic study of ALS to date, van Rheenen and colleagues use sophisticated analyses to gain novel insights into its pathogenesis.},
  archive      = {J_NG},
  author       = {Sahdeo, Sunil and Goldstein, David B.},
  doi          = {10.1038/s41588-021-00981-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1624-1626},
  shortjournal = {Nat. Genet.},
  title        = {Translating amyotrophic lateral sclerosis genes into drug development leads},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reversing polycystic kidney disease. <em>NG</em>,
<em>53</em>(12), 1623–1624. (<a
href="https://doi.org/10.1038/s41588-021-00963-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study shows that re-expressing PKD genes early in the course of the disease can fully reverse polycystic kidney disease in mice. These results reveal an unexpected ability of the kidney to regenerate following genetic rescue of polycystin function.},
  archive      = {J_NG},
  author       = {Boletta, Alessandra},
  doi          = {10.1038/s41588-021-00963-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1623-1624},
  shortjournal = {Nat. Genet.},
  title        = {Reversing polycystic kidney disease},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Genetics of 35 blood and urine
biomarkers in the UK biobank. <em>NG</em>, <em>53</em>(11), 1622. (<a
href="https://doi.org/10.1038/s41588-021-00956-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Sinnott-Armstrong, Nasa and Tanigawa, Yosuke and Amar, David and Mars, Nina and Benner, Christian and Aguirre, Matthew and Venkataraman, Guhan Ram and Wainberg, Michael and Ollila, Hanna M. and Kiiskinen, Tuomo and Havulinna, Aki S. and Pirruccello, James P. and Qian, Junyang and Shcherbina, Anna and Rodriguez, Fatima and Assimes, Themistocles L. and Agarwala, Vineeta and Tibshirani, Robert and Hastie, Trevor and Ripatti, Samuli and Pritchard, Jonathan K. and Daly, Mark J. and Rivas, Manuel A.},
  doi          = {10.1038/s41588-021-00956-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1622},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Genetics of 35 blood and urine biomarkers in the UK biobank},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Epigenomic contributions to tumor
cell heterogeneity and plasticity. <em>NG</em>, <em>53</em>(11), 1622.
(<a href="https://doi.org/10.1038/s41588-021-00966-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Mathur, Radhika and Costello, Joseph F.},
  doi          = {10.1038/s41588-021-00966-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1622},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Epigenomic contributions to tumor cell heterogeneity and plasticity},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A generalized linear mixed model association tool for
biobank-scale data. <em>NG</em>, <em>53</em>(11), 1616–1621. (<a
href="https://doi.org/10.1038/s41588-021-00954-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Compared with linear mixed model-based genome-wide association (GWA) methods, generalized linear mixed model (GLMM)-based methods have better statistical properties when applied to binary traits but are computationally much slower. In the present study, leveraging efficient sparse matrix-based algorithms, we developed a GLMM-based GWA tool, fastGWA-GLMM, that is severalfold to orders of magnitude faster than the state-of-the-art tools when applied to the UK Biobank (UKB) data and scalable to cohorts with millions of individuals. We show by simulation that the fastGWA-GLMM test statistics of both common and rare variants are well calibrated under the null, even for traits with extreme case–control ratios. We applied fastGWA-GLMM to the UKB data of 456,348 individuals, 11,842,647 variants and 2,989 binary traits (full summary statistics available at http://fastgwa.info/ukbimpbin ), and identified 259 rare variants associated with 75 traits, demonstrating the use of imputed genotype data in a large cohort to discover rare variants for binary complex traits. FastGWA-GLMM is a fast, scalable generalized linear mixed model method for genetic association testing for binary traits in large cohorts that is robust to variant frequency and case–control imbalance.},
  archive      = {J_NG},
  author       = {Jiang, Longda and Zheng, Zhili and Fang, Hailing and Yang, Jian},
  doi          = {10.1038/s41588-021-00954-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1616-1621},
  shortjournal = {Nat. Genet.},
  title        = {A generalized linear mixed model association tool for biobank-scale data},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identification of LZTFL1 as a candidate effector gene at a
COVID-19 risk locus. <em>NG</em>, <em>53</em>(11), 1606–1615. (<a
href="https://doi.org/10.1038/s41588-021-00955-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The severe acute respiratory syndrome coronavirus 2 (SARS‑CoV‑2) disease (COVID-19) pandemic has caused millions of deaths worldwide. Genome-wide association studies identified the 3p21.31 region as conferring a twofold increased risk of respiratory failure. Here, using a combined multiomics and machine learning approach, we identify the gain-of-function risk A allele of an SNP, rs17713054G&amp;gt;A, as a probable causative variant. We show with chromosome conformation capture and gene-expression analysis that the rs17713054-affected enhancer upregulates the interacting gene, leucine zipper transcription factor like 1 (LZTFL1). Selective spatial transcriptomic analysis of lung biopsies from patients with COVID-19 shows the presence of signals associated with epithelial–mesenchymal transition (EMT), a viral response pathway that is regulated by LZTFL1. We conclude that pulmonary epithelial cells undergoing EMT, rather than immune cells, are likely responsible for the 3p21.31-associated risk. Since the 3p21.31 effect is conferred by a gain-of-function, LZTFL1 may represent a therapeutic target. SNP rs17713054 in the 3p21.31 COVID-19 risk locus is identified as a probable causative variant for disease association. Chromatin conformation and gene expression data indicate that LZTFL1 is impacted by rs17713054 in pulmonary epithelial cells.},
  archive      = {J_NG},
  author       = {Downes, Damien J. and Cross, Amy R. and Hua, Peng and Roberts, Nigel and Schwessinger, Ron and Cutler, Antony J. and Munis, Altar M. and Brown, Jill and Mielczarek, Olga and de Andrea, Carlos E. and Melero, Ignacio and Gill, Deborah R. and Hyde, Stephen C. and Knight, Julian C. and Todd, John A. and Sansom, Stephen N. and Issa, Fadi and Davies, James O. J. and Hughes, Jim R.},
  doi          = {10.1038/s41588-021-00955-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1606-1615},
  shortjournal = {Nat. Genet.},
  title        = {Identification of LZTFL1 as a candidate effector gene at a COVID-19 risk locus},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Synonymous mutations reveal genome-wide levels of positive
selection in healthy tissues. <em>NG</em>, <em>53</em>(11), 1597–1605.
(<a href="https://doi.org/10.1038/s41588-021-00957-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic alterations under positive selection in healthy tissues have implications for cancer risk. However, total levels of positive selection across the genome remain unknown. Passenger mutations are influenced by all driver mutations, regardless of type or location in the genome. Therefore, the total number of passengers can be used to estimate the total number of drivers—including unidentified drivers outside of cancer genes that are traditionally missed. Here we analyze the variant allele frequency spectrum of synonymous mutations from healthy blood and esophagus to quantify levels of missing positive selection. In blood, we find that only 30% of passengers can be explained by single-nucleotide variants in driver genes, suggesting high levels of positive selection for mutations elsewhere in the genome. In contrast, more than half of all passengers in the esophagus can be explained by just the two driver genes NOTCH1 and TP53, suggesting little positive selection elsewhere. Synonymous passenger mutations are used to measure levels of positive selection in healthy blood and esophagus. This approach can quantify missing selection due to unidentified drivers.},
  archive      = {J_NG},
  author       = {Poon, Gladys Y. P. and Watson, Caroline J. and Fisher, Daniel S. and Blundell, Jamie R.},
  doi          = {10.1038/s41588-021-00957-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1597-1605},
  shortjournal = {Nat. Genet.},
  title        = {Synonymous mutations reveal genome-wide levels of positive selection in healthy tissues},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Automated CUT&amp;tag profiling of chromatin heterogeneity
in mixed-lineage leukemia. <em>NG</em>, <em>53</em>(11), 1586–1596. (<a
href="https://doi.org/10.1038/s41588-021-00941-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acute myeloid and lymphoid leukemias often harbor chromosomal translocations involving the KMT2A gene, encoding the KMT2A lysine methyltransferase (also known as mixed-lineage leukemia-1), and produce in-frame fusions of KMT2A to other chromatin-regulatory proteins. Here we map fusion-specific targets across the genome for diverse KMT2A oncofusion proteins in cell lines and patient samples. By modifying CUT&amp;amp;Tag chromatin profiling for full automation, we identify common and tumor-subtype-specific sites of aberrant chromatin regulation induced by KMT2A oncofusion proteins. A subset of KMT2A oncofusion-binding sites are marked by bivalent (H3K4me3 and H3K27me3) chromatin signatures, and single-cell CUT&amp;amp;Tag profiling reveals that these sites display cell-to-cell heterogeneity suggestive of lineage plasticity. In addition, we find that aberrant enrichment of H3K4me3 in gene bodies is sensitive to Menin inhibitors, demonstrating the utility of automated chromatin profiling for identifying therapeutic vulnerabilities. Thus, integration of automated and single-cell CUT&amp;amp;Tag can uncover epigenomic heterogeneity within patient samples and predict sensitivity to therapeutic agents. Automated and single-cell CUT&amp;amp;Tag is used to characterize the effects of KMT2A fusion proteins on chromatin in human primary leukemia samples, identifying oncogenic networks and fusion-specific therapeutic vulnerabilities.},
  archive      = {J_NG},
  author       = {Janssens, Derek H. and Meers, Michael P. and Wu, Steven J. and Babaeva, Ekaterina and Meshinchi, Soheil and Sarthy, Jay F. and Ahmad, Kami and Henikoff, Steven},
  doi          = {10.1038/s41588-021-00941-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1586-1596},
  shortjournal = {Nat. Genet.},
  title        = {Automated CUT&amp;Tag profiling of chromatin heterogeneity in mixed-lineage leukemia},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The context-specific role of germline pathogenicity in
tumorigenesis. <em>NG</em>, <em>53</em>(11), 1577–1585. (<a
href="https://doi.org/10.1038/s41588-021-00949-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human cancers arise from environmental, heritable and somatic factors, but how these mechanisms interact in tumorigenesis is poorly understood. Studying 17,152 prospectively sequenced patients with cancer, we identified pathogenic germline variants in cancer predisposition genes, and assessed their zygosity and co-occurring somatic alterations in the concomitant tumors. Two major routes to tumorigenesis were apparent. In carriers of pathogenic germline variants in high-penetrance genes (5.1% overall), lineage-dependent patterns of biallelic inactivation led to tumors exhibiting mechanism-specific somatic phenotypes and fewer additional somatic oncogenic drivers. Nevertheless, 27% of cancers in these patients, and most tumors in patients with pathogenic germline variants in lower-penetrance genes, lacked particular hallmarks of tumorigenesis associated with the germline allele. The dependence of tumors on pathogenic germline variants is variable and often dictated by both penetrance and lineage, a finding with implications for clinical management. A study of 17,152 patients with cancer identified pathogenic germline variants in cancer predisposition genes. Although tumors showed biallelic inactivation for high-penetrance genes, this was not the case in most patients with pathogenic variants in low-penetrance genes, suggesting alternative routes to tumorigenesis.},
  archive      = {J_NG},
  author       = {Srinivasan, Preethi and Bandlamudi, Chaitanya and Jonsson, Philip and Kemel, Yelena and Chavan, Shweta S. and Richards, Allison L. and Penson, Alexander V. and Bielski, Craig M. and Fong, Christopher and Syed, Aijazuddin and Jayakumaran, Gowtham and Prasad, Meera and Hwee, Jason and Sumer, Selcuk Onur and de Bruijn, Ino and Li, Xiang and Gao, JianJiong and Schultz, Nikolaus and Cambria, Roy and Galle, Jesse and Mukherjee, Semanti and Vijai, Joseph and Cadoo, Karen A. and Carlo, Maria I. and Walsh, Michael F. and Mandelker, Diana and Ceyhan-Birsoy, Ozge and Shia, Jinru and Zehir, Ahmet and Ladanyi, Marc and Hyman, David M. and Zhang, Liying and Offit, Kenneth and Robson, Mark E. and Solit, David B. and Stadler, Zsofia K. and Berger, Michael F. and Taylor, Barry S.},
  doi          = {10.1038/s41588-021-00949-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1577-1585},
  shortjournal = {Nat. Genet.},
  title        = {The context-specific role of germline pathogenicity in tumorigenesis},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The dynamic, combinatorial cis-regulatory lexicon of
epidermal differentiation. <em>NG</em>, <em>53</em>(11), 1564–1576. (<a
href="https://doi.org/10.1038/s41588-021-00947-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factors bind DNA sequence motif vocabularies in cis-regulatory elements (CREs) to modulate chromatin state and gene expression during cell state transitions. A quantitative understanding of how motif lexicons influence dynamic regulatory activity has been elusive due to the combinatorial nature of the cis-regulatory code. To address this, we undertook multiomic data profiling of chromatin and expression dynamics across epidermal differentiation to identify 40,103 dynamic CREs associated with 3,609 dynamically expressed genes, then applied an interpretable deep-learning framework to model the cis-regulatory logic of chromatin accessibility. This analysis framework identified cooperative DNA sequence rules in dynamic CREs regulating synchronous gene modules with diverse roles in skin differentiation. Massively parallel reporter assay analysis validated temporal dynamics and cooperative cis-regulatory logic. Variants linked to human polygenic skin disease were enriched in these time-dependent combinatorial motif rules. This integrative approach shows the combinatorial cis-regulatory lexicon of epidermal differentiation and represents a general framework for deciphering the organizational principles of the cis-regulatory code of dynamic gene regulation. A deep-learning framework interprets multiomic data across epidermal differentiation, identifying cooperative DNA sequence rules that regulate gene modules. Massively parallel reporter assay analysis validates temporal dynamics and cis-regulatory logic.},
  archive      = {J_NG},
  author       = {Kim, Daniel S. and Risca, Viviana I. and Reynolds, David L. and Chappell, James and Rubin, Adam J. and Jung, Namyoung and Donohue, Laura K. H. and Lopez-Pajares, Vanessa and Kathiria, Arwa and Shi, Minyi and Zhao, Zhixin and Deep, Harsh and Sharmin, Mahfuza and Rao, Deepti and Lin, Shin and Chang, Howard Y. and Snyder, Michael P. and Greenleaf, William J. and Kundaje, Anshul and Khavari, Paul A.},
  doi          = {10.1038/s41588-021-00947-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1564-1576},
  shortjournal = {Nat. Genet.},
  title        = {The dynamic, combinatorial cis-regulatory lexicon of epidermal differentiation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mutational signatures in esophageal squamous cell carcinoma
from eight countries with varying incidence. <em>NG</em>,
<em>53</em>(11), 1553–1563. (<a
href="https://doi.org/10.1038/s41588-021-00928-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Esophageal squamous cell carcinoma (ESCC) shows remarkable variation in incidence that is not fully explained by known lifestyle and environmental risk factors. It has been speculated that an unknown exogenous exposure(s) could be responsible. Here we combine the fields of mutational signature analysis with cancer epidemiology to study 552 ESCC genomes from eight countries with varying incidence rates. Mutational profiles were similar across all countries studied. Associations between specific mutational signatures and ESCC risk factors were identified for tobacco, alcohol, opium and germline variants, with modest impacts on mutation burden. We find no evidence of a mutational signature indicative of an exogenous exposure capable of explaining differences in ESCC incidence. Apolipoprotein B mRNA-editing enzyme, catalytic polypeptide-like (APOBEC)-associated mutational signatures single-base substitution (SBS)2 and SBS13 were present in 88% and 91% of cases, respectively, and accounted for 25% of the mutation burden on average, indicating that APOBEC activation is a crucial step in ESCC tumor development. The incidence of esophageal squamous cell carcinoma varies significantly across different geographical regions. Mutational signature analysis of tumors sampled from high- and low-incidence areas suggests that these variations may not be explained by mutagenic exposures.},
  archive      = {J_NG},
  author       = {Moody, Sarah and Senkin, Sergey and Islam, S. M. Ashiqul and Wang, Jingwei and Nasrollahzadeh, Dariush and Cortez Cardoso Penha, Ricardo and Fitzgerald, Stephen and Bergstrom, Erik N. and Atkins, Joshua and He, Yudou and Khandekar, Azhar and Smith-Byrne, Karl and Carreira, Christine and Gaborieau, Valerie and Latimer, Calli and Thomas, Emily and Abnizova, Irina and Bucciarelli, Pauline E. and Jones, David and Teague, Jon W. and Abedi-Ardekani, Behnoush and Serra, Stefano and Scoazec, Jean-Yves and Saffar, Hiva and Azmoudeh-Ardalan, Farid and Sotoudeh, Masoud and Nikmanesh, Arash and Poustchi, Hossein and Niavarani, Ahmadreza and Gharavi, Samad and Eden, Michael and Richman, Paul and Campos, Lia S. and Fitzgerald, Rebecca C. and Ribeiro, Luis Felipe and Soares-Lima, Sheila Coelho and Dzamalala, Charles and Mmbaga, Blandina Theophil and Shibata, Tatsuhiro and Menya, Diana and Goldstein, Alisa M. and Hu, Nan and Malekzadeh, Reza and Fazel, Abdolreza and McCormack, Valerie and McKay, James and Perdomo, Sandra and Scelo, Ghislaine and Chanudet, Estelle and Humphreys, Laura and Alexandrov, Ludmil B. and Brennan, Paul and Stratton, Michael R.},
  doi          = {10.1038/s41588-021-00928-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1553-1563},
  shortjournal = {Nat. Genet.},
  title        = {Mutational signatures in esophageal squamous cell carcinoma from eight countries with varying incidence},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide analysis of 53,400 people with irritable bowel
syndrome highlights shared genetic pathways with mood and anxiety
disorders. <em>NG</em>, <em>53</em>(11), 1543–1552. (<a
href="https://doi.org/10.1038/s41588-021-00950-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Irritable bowel syndrome (IBS) results from disordered brain–gut interactions. Identifying susceptibility genes could highlight the underlying pathophysiological mechanisms. We designed a digestive health questionnaire for UK Biobank and combined identified cases with IBS with independent cohorts. We conducted a genome-wide association study with 53,400 cases and 433,201 controls and replicated significant associations in a 23andMe panel (205,252 cases and 1,384,055 controls). Our study identified and confirmed six genetic susceptibility loci for IBS. Implicated genes included NCAM1, CADM2, PHF2/FAM120A, DOCK9, CKAP2/TPTE2P3 and BAG6. The first four are associated with mood and anxiety disorders, expressed in the nervous system, or both. Mirroring this, we also found strong genome-wide correlation between the risk of IBS and anxiety, neuroticism and depression (rg &amp;gt; 0.5). Additional analyses suggested this arises due to shared pathogenic pathways rather than, for example, anxiety causing abdominal symptoms. Implicated mechanisms require further exploration to help understand the altered brain–gut interactions underlying IBS. Genome-wide association analysis of irritable bowel syndrome identifies genetic susceptibility loci and highlights shared pathways with mood and anxiety disorders.},
  archive      = {J_NG},
  author       = {Eijsbouts, Chris and Zheng, Tenghao and Kennedy, Nicholas A. and Bonfiglio, Ferdinando and Anderson, Carl A. and Moutsianas, Loukas and Holliday, Joanne and Shi, Jingchunzi and Shringarpure, Suyash and Voda, Alexandru-Ioan and Farrugia, Gianrico and Franke, Andre and Hübenthal, Matthias and Abecasis, Gonçalo and Zawistowski, Matthew and Skogholt, Anne Heidi and Ness-Jensen, Eivind and Hveem, Kristian and Esko, Tõnu and Teder-Laving, Maris and Zhernakova, Alexandra and Camilleri, Michael and Boeckxstaens, Guy and Whorwell, Peter J. and Spiller, Robin and McVean, Gil and D’Amato, Mauro and Jostins, Luke and Parkes, Miles},
  doi          = {10.1038/s41588-021-00950-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1543-1552},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analysis of 53,400 people with irritable bowel syndrome highlights shared genetic pathways with mood and anxiety disorders},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide association analyses highlight etiological
differences underlying newly defined subtypes of diabetes. <em>NG</em>,
<em>53</em>(11), 1534–1542. (<a
href="https://doi.org/10.1038/s41588-021-00948-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Type 2 diabetes has been reproducibly clustered into five subtypes with different disease progression and risk of complications; however, etiological differences are unknown. We used genome-wide association and genetic risk score (GRS) analysis to compare the underlying genetic drivers. Individuals from the Swedish ANDIS (All New Diabetics In Scania) study were compared to individuals without diabetes; the Finnish DIREVA (Diabetes register in Vasa) and Botnia studies were used for replication. We show that subtypes differ with regard to family history of diabetes and association with GRS for diabetes-related traits. The severe insulin-resistant subtype was uniquely associated with GRS for fasting insulin but not with variants in the TCF7L2 locus or GRS reflecting insulin secretion. Further, an SNP (rs10824307) near LRMDA was uniquely associated with mild obesity-related diabetes. Therefore, we conclude that the subtypes have partially distinct genetic backgrounds indicating etiological differences. Genome-wide association and genetic risk score analyses highlight differences in genetic architecture across five subtypes of diabetes.},
  archive      = {J_NG},
  author       = {Mansour Aly, Dina and Dwivedi, Om Prakash and Prasad, Rashmi B. and Käräjämäki, Annemari and Hjort, Rebecka and Thangam, Manonanthini and Åkerlund, Mikael and Mahajan, Anubha and Udler, Miriam S. and Florez, Jose C. and McCarthy, Mark I. and Brosnan, Julia and Melander, Olle and Carlsson, Sofia and Hansson, Ola and Tuomi, Tiinamaija and Groop, Leif and Ahlqvist, Emma},
  doi          = {10.1038/s41588-021-00948-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1534-1542},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An open approach to systematically prioritize causal
variants and genes at all published human GWAS trait-associated loci.
<em>NG</em>, <em>53</em>(11), 1527–1533. (<a
href="https://doi.org/10.1038/s41588-021-00945-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWASs) have identified many variants associated with complex traits, but identifying the causal gene(s) is a major challenge. In the present study, we present an open resource that provides systematic fine mapping and gene prioritization across 133,441 published human GWAS loci. We integrate genetics (GWAS Catalog and UK Biobank) with transcriptomic, proteomic and epigenomic data, including systematic disease–disease and disease–molecular trait colocalization results across 92 cell types and tissues. We identify 729 loci fine mapped to a single-coding causal variant and colocalized with a single gene. We trained a machine-learning model using the fine-mapped genetics and functional genomics data and 445 gold-standard curated GWAS loci to distinguish causal genes from neighboring genes, outperforming a naive distance-based model. Our prioritized genes were enriched for known approved drug targets (odds ratio = 8.1, 95% confidence interval = 5.7, 11.5). These results are publicly available through a web portal ( http://genetics.opentargets.org ), enabling users to easily prioritize genes at disease-associated loci and assess their potential as drug targets. Open Targets Genetics is a community resource that provides systematic fine mapping at human GWAS loci, enabling users to prioritize genes at disease-associated regions and assess their potential as drug targets.},
  archive      = {J_NG},
  author       = {Mountjoy, Edward and Schmidt, Ellen M. and Carmona, Miguel and Schwartzentruber, Jeremy and Peat, Gareth and Miranda, Alfredo and Fumis, Luca and Hayhurst, James and Buniello, Annalisa and Karim, Mohd Anisul and Wright, Daniel and Hercules, Andrew and Papa, Eliseo and Fauman, Eric B. and Barrett, Jeffrey C. and Todd, John A. and Ochoa, David and Dunham, Ian and Ghoussaini, Maya},
  doi          = {10.1038/s41588-021-00945-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1527-1533},
  shortjournal = {Nat. Genet.},
  title        = {An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Shining light on dark selection in healthy human tissues.
<em>NG</em>, <em>53</em>(11), 1525–1526. (<a
href="https://doi.org/10.1038/s41588-021-00959-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genomes of cells across human tissues are riddled by changes to their DNA1–3. Many of these mutations do not alter the properties of a cell, and are neutral passengers. However, a small proportion can change a cell’s fitness, and increase or decrease the progeny that originate from the mutated cell4. How many of these alterations under positive selection (drivers) exist in total is unknown. Whether the current list of drivers is almost complete, or whether large proportions of positively selected drivers in the human genome remain undetected is yet to be determined.},
  archive      = {J_NG},
  author       = {Werner, Benjamin},
  doi          = {10.1038/s41588-021-00959-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1525-1526},
  shortjournal = {Nat. Genet.},
  title        = {Shining light on dark selection in healthy human tissues},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). How the germline informs the somatic landscape. <em>NG</em>,
<em>53</em>(11), 1523–1525. (<a
href="https://doi.org/10.1038/s41588-021-00960-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How somatic and germline mutations interact in cancer remains largely unexplored. A study of 17,152 patients with cancer suggests that the relative contribution of pathogenic germline mutations is governed by lineage and penetrance.},
  archive      = {J_NG},
  author       = {Chanock, Stephen J.},
  doi          = {10.1038/s41588-021-00960-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1523-1525},
  shortjournal = {Nat. Genet.},
  title        = {How the germline informs the somatic landscape},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Esophageal cancer mutational signatures around the world.
<em>NG</em>, <em>53</em>(11), 1522–1523. (<a
href="https://doi.org/10.1038/s41588-021-00958-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational signatures can shed light onto mechanisms of carcinogenesis and reveal the mutagenic impact of novel and established environmental risk factors. A new study examines the mutational spectra of esophageal squamous cell cancer together with exposure information in regions of high and low incidence of the disease, and demonstrates both the limitations and potential of mutational signature analyses.},
  archive      = {J_NG},
  author       = {Giannakis, Marios and Peters, Ulrike},
  doi          = {10.1038/s41588-021-00958-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1522-1523},
  shortjournal = {Nat. Genet.},
  title        = {Esophageal cancer mutational signatures around the world},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Misuse of the term “trans-ethnic” in genomics research.
<em>NG</em>, <em>53</em>(11), 1520–1521. (<a
href="https://doi.org/10.1038/s41588-021-00952-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A concerning trend in genetics is the common use of the term ‘trans-ethnic’ to describe analyses that combine or compare several ancestrally diverse populations. In this commentary, we discuss how this term is inaccurate and alienating. We propose that geneticists avoid using the term trans-ethnic entirely and that researchers across disciplines reach a new consensus about the best terms to use to describe the populations we study.},
  archive      = {J_NG},
  author       = {Kamariza, Mireille and Crawford, Lorin and Jones, David and Finucane, Hilary},
  doi          = {10.1038/s41588-021-00952-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1520-1521},
  shortjournal = {Nat. Genet.},
  title        = {Misuse of the term ‘trans-ethnic’ in genomics research},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The language of genetics. <em>NG</em>, <em>53</em>(11),
1519. (<a href="https://doi.org/10.1038/s41588-021-00965-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The language used in genetic and medical research to describe populations has a fraught history, and current practices must be sensitively considered when reporting on genetic cohorts and analyses.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-021-00965-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1519},
  shortjournal = {Nat. Genet.},
  title        = {The language of genetics},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Direct characterization of
cis-regulatory elements and functional dissection of complex genetic
associations using HCR–FlowFISH. <em>NG</em>, <em>53</em>(10), 1517. (<a
href="https://doi.org/10.1038/s41588-021-00943-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Reilly, Steven K. and Gosai, Sager J. and Gutierrez, Alan and Mackay-Smith, Ava and Ulirsch, Jacob C. and Kanai, Masahiro and Mouri, Kousuke and Berenzy, Daniel and Kales, Susan and Butler, Gina M. and Gladden-Young, Adrianne and Bhuiyan, Redwan M. and Stitzel, Michael L. and Finucane, Hilary K. and Sabeti, Pardis C. and Tewhey, Ryan},
  doi          = {10.1038/s41588-021-00943-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1517},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Direct characterization of cis-regulatory elements and functional dissection of complex genetic associations using HCR–FlowFISH},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). A high-resolution HLA reference panel capturing global
population diversity enables multi-ancestry fine-mapping in HIV host
response. <em>NG</em>, <em>53</em>(10), 1504–1516. (<a
href="https://doi.org/10.1038/s41588-021-00935-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fine-mapping to plausible causal variation may be more effective in multi-ancestry cohorts, particularly in the MHC, which has population-specific structure. To enable such studies, we constructed a large (n = 21,546) HLA reference panel spanning five global populations based on whole-genome sequences. Despite population-specific long-range haplotypes, we demonstrated accurate imputation at G-group resolution (94.2%, 93.7%, 97.8% and 93.7% in admixed African (AA), East Asian (EAS), European (EUR) and Latino (LAT) populations). Applying HLA imputation to genome-wide association study data for HIV-1 viral load in three populations (EUR, AA and LAT), we obviated effects of previously reported associations from population-specific HIV studies and discovered a novel association at position 156 in HLA-B. We pinpointed the MHC association to three amino acid positions (97, 67 and 156) marking three consecutive pockets (C, B and D) within the HLA-B peptide-binding groove, explaining 12.9% of trait variance. A high-resolution reference panel based on whole-genome sequencing data enables accurate imputation of HLA alleles across diverse populations and fine-mapping of HLA association signals for HIV-1 host response.},
  archive      = {J_NG},
  author       = {Luo, Yang and Kanai, Masahiro and Choi, Wanson and Li, Xinyi and Sakaue, Saori and Yamamoto, Kenichi and Ogawa, Kotaro and Gutierrez-Arcelus, Maria and Gregersen, Peter K. and Stuart, Philip E. and Elder, James T. and Forer, Lukas and Schönherr, Sebastian and Fuchsberger, Christian and Smith, Albert V. and Fellay, Jacques and Carrington, Mary and Haas, David W. and Guo, Xiuqing and Palmer, Nicholette D. and Chen, Yii-Der Ida and Rotter, Jerome I. and Taylor, Kent D. and Rich, Stephen S. and Correa, Adolfo and Wilson, James G. and Kathiresan, Sekar and Cho, Michael H. and Metspalu, Andres and Esko, Tonu and Okada, Yukinori and Han, Buhm and McLaren, Paul J. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-021-00935-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1504-1516},
  shortjournal = {Nat. Genet.},
  title        = {A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Parallel subgenome structure and divergent expression
evolution of allo-tetraploid common carp and goldfish. <em>NG</em>,
<em>53</em>(10), 1493–1503. (<a
href="https://doi.org/10.1038/s41588-021-00933-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How two subgenomes in allo-tetraploids adapt to coexistence and coordinate through structure and expression evolution requires extensive studies. In the present study, we report an improved genome assembly of allo-tetraploid common carp, an updated genome annotation of allo-tetraploid goldfish and the chromosome-scale assemblies of a progenitor-like diploid Puntius tetrazona and an outgroup diploid Paracanthobrama guichenoti. Parallel subgenome structure evolution in the allo-tetraploids was featured with equivalent chromosome components, higher protein identities, similar transposon divergence and contents, homoeologous exchanges, better synteny level, strong sequence compensation and symmetric purifying selection. Furthermore, we observed subgenome expression divergence processes in the allo-tetraploids, including inter-/intrasubgenome trans-splicing events, expression dominance, decreased expression levels, dosage compensation, stronger expression correlation, dynamic functionalization and balancing of differential expression. The potential disorders introduced by different progenitors in the allo-tetraploids were hypothesized to be alleviated by increasing structural homogeneity and performing versatile expression processes. Resequencing three common carp strains revealed two major ecotypes and uncovered candidate genes relevant to growth and survival rate. Genomic analysis of allo-tetraploid common carp and goldfish identifies parallel subgenome structure and divergent expression processes.},
  archive      = {J_NG},
  author       = {Li, Jiong-Tang and Wang, Qi and Huang Yang, Mei-Di and Li, Qing-Song and Cui, Ming-Shu and Dong, Zai-Jie and Wang, Hong-Wei and Yu, Ju-Hua and Zhao, Yu-Jie and Yang, Chen-Ru and Wang, Ya-Xin and Sun, Xiao-Qing and Zhang, Yan and Zhao, Ran and Jia, Zhi-Ying and Wang, Xi-Yin},
  doi          = {10.1038/s41588-021-00933-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1493-1503},
  shortjournal = {Nat. Genet.},
  title        = {Parallel subgenome structure and divergent expression evolution of allo-tetraploid common carp and goldfish},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). BRD4 orchestrates genome folding to promote neural crest
differentiation. <em>NG</em>, <em>53</em>(10), 1480–1492. (<a
href="https://doi.org/10.1038/s41588-021-00934-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Higher-order chromatin structure regulates gene expression, and mutations in proteins mediating genome folding underlie developmental disorders known as cohesinopathies. However, the relationship between three-dimensional genome organization and embryonic development remains unclear. Here we define a role for bromodomain-containing protein 4 (BRD4) in genome folding, and leverage it to understand the importance of genome folding in neural crest progenitor differentiation. Brd4 deletion in neural crest results in cohesinopathy-like phenotypes. BRD4 interacts with NIPBL, a cohesin agonist, and BRD4 depletion or loss of the BRD4–NIPBL interaction reduces NIPBL occupancy, suggesting that BRD4 stabilizes NIPBL on chromatin. Chromatin interaction mapping and imaging experiments demonstrate that BRD4 depletion results in compromised genome folding and loop extrusion. Finally, mutation of individual BRD4 amino acids that mediate an interaction with NIPBL impedes neural crest differentiation into smooth muscle. Remarkably, loss of WAPL, a cohesin antagonist, rescues attenuated smooth muscle differentiation resulting from BRD4 loss. Collectively, our data reveal that BRD4 choreographs genome folding and illustrates the relevance of balancing cohesin activity for progenitor differentiation. Depletion of BRD4 reduces the chromatin occupancy of NIPBL, resulting in aberrant genome folding. Loss of BRD4 impedes neural crest differentiation, which can be rescued by depletion of WAPL.},
  archive      = {J_NG},
  author       = {Linares-Saldana, Ricardo and Kim, Wonho and Bolar, Nikhita A. and Zhang, Haoyue and Koch-Bojalad, Bailey A. and Yoon, Sora and Shah, Parisha P. and Karnay, Ashley and Park, Daniel S. and Luppino, Jennifer M. and Nguyen, Son C. and Padmanabhan, Arun and Smith, Cheryl L. and Poleshko, Andrey and Wang, Qiaohong and Li, Li and Srivastava, Deepak and Vahedi, Golnaz and Eom, Gwang Hyeon and Blobel, Gerd A. and Joyce, Eric F. and Jain, Rajan},
  doi          = {10.1038/s41588-021-00934-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1480-1492},
  shortjournal = {Nat. Genet.},
  title        = {BRD4 orchestrates genome folding to promote neural crest differentiation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Epigenetic encoding, heritability and plasticity of glioma
transcriptional cell states. <em>NG</em>, <em>53</em>(10), 1469–1479.
(<a href="https://doi.org/10.1038/s41588-021-00927-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing has revealed extensive transcriptional cell state diversity in cancer, often observed independently of genetic heterogeneity, raising the central question of how malignant cell states are encoded epigenetically. To address this, here we performed multiomics single-cell profiling—integrating DNA methylation, transcriptome and genotype within the same cells—of diffuse gliomas, tumors characterized by defined transcriptional cell state diversity. Direct comparison of the epigenetic profiles of distinct cell states revealed key switches for state transitions recapitulating neurodevelopmental trajectories and highlighted dysregulated epigenetic mechanisms underlying gliomagenesis. We further developed a quantitative framework to directly measure cell state heritability and transition dynamics based on high-resolution lineage trees in human samples. We demonstrated heritability of malignant cell states, with key differences in hierarchal and plastic cell state architectures in IDH-mutant glioma versus IDH-wild-type glioblastoma, respectively. This work provides a framework anchoring transcriptional cancer cell states in their epigenetic encoding, inheritance and transition dynamics. Multimodal DNA methylation and transcriptome profiling of single glioma cells links tumor cell transcriptional states to epigenetics via interaction with PRC2 and shows that these states are heritable and important for tumor plasticity.},
  archive      = {J_NG},
  author       = {Chaligne, Ronan and Gaiti, Federico and Silverbush, Dana and Schiffman, Joshua S. and Weisman, Hannah R. and Kluegel, Lloyd and Gritsch, Simon and Deochand, Sunil D. and Gonzalez Castro, L. Nicolas and Richman, Alyssa R. and Klughammer, Johanna and Biancalani, Tommaso and Muus, Christoph and Sheridan, Caroline and Alonso, Alicia and Izzo, Franco and Park, Jane and Rozenblatt-Rosen, Orit and Regev, Aviv and Suvà, Mario L. and Landau, Dan A.},
  doi          = {10.1038/s41588-021-00927-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1469-1479},
  shortjournal = {Nat. Genet.},
  title        = {Epigenetic encoding, heritability and plasticity of glioma transcriptional cell states},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell multimodal glioma analyses identify epigenetic
regulators of cellular plasticity and environmental stress response.
<em>NG</em>, <em>53</em>(10), 1456–1468. (<a
href="https://doi.org/10.1038/s41588-021-00926-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glioma intratumoral heterogeneity enables adaptation to challenging microenvironments and contributes to therapeutic resistance. We integrated 914 single-cell DNA methylomes, 55,284 single-cell transcriptomes and bulk multi-omic profiles across 11 adult IDH mutant or IDH wild-type gliomas to delineate sources of intratumoral heterogeneity. We showed that local DNA methylation disorder is associated with cell–cell DNA methylation differences, is elevated in more aggressive tumors, links with transcriptional disruption and is altered during the environmental stress response. Glioma cells under in vitro hypoxic and irradiation stress increased local DNA methylation disorder and shifted cell states. We identified a positive association between genetic and epigenetic instability that was supported in bulk longitudinally collected DNA methylation data. Increased DNA methylation disorder associated with accelerated disease progression and recurrently selected DNA methylation changes were enriched for environmental stress response pathways. Our work identified an epigenetically facilitated adaptive stress response process and highlights the importance of epigenetic heterogeneity in shaping therapeutic outcomes. Single-cell DNA methylation and transcriptomic glioma analyses link local DNA methylation disorder and cellular plasticity. Increases in disorder are associated with stress and disease progression, suggesting a role in shaping the therapeutic response.},
  archive      = {J_NG},
  author       = {Johnson, Kevin C. and Anderson, Kevin J. and Courtois, Elise T. and Gujar, Amit D. and Barthel, Floris P. and Varn, Frederick S. and Luo, Diane and Seignon, Martine and Yi, Eunhee and Kim, Hoon and Estecio, Marcos R. H. and Zhao, Dacheng and Tang, Ming and Navin, Nicholas E. and Maurya, Rahul and Ngan, Chew Yee and Verburg, Niels and de Witt Hamer, Philip C. and Bulsara, Ketan and Samuels, Michael L. and Das, Sunit and Robson, Paul and Verhaak, Roel G. W.},
  doi          = {10.1038/s41588-021-00926-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1456-1468},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multimodal glioma analyses identify epigenetic regulators of cellular plasticity and environmental stress response},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mutational synergy during leukemia induction remodels
chromatin accessibility, histone modifications and three-dimensional DNA
topology to alter gene expression. <em>NG</em>, <em>53</em>(10),
1443–1455. (<a
href="https://doi.org/10.1038/s41588-021-00925-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Altered transcription is a cardinal feature of acute myeloid leukemia (AML); however, exactly how mutations synergize to remodel the epigenetic landscape and rewire three-dimensional DNA topology is unknown. Here, we apply an integrated genomic approach to a murine allelic series that models the two most common mutations in AML: Flt3-ITD and Npm1c. We then deconvolute the contribution of each mutation to alterations of the epigenetic landscape and genome organization, and infer how mutations synergize in the induction of AML. Our studies demonstrate that Flt3-ITD signals to chromatin to alter the epigenetic environment and synergizes with mutations in Npm1c to alter gene expression and drive leukemia induction. These analyses also allow the identification of long-range cis-regulatory circuits, including a previously unknown superenhancer of Hoxa locus, as well as larger and more detailed gene-regulatory networks, driven by transcription factors including PU.1 and IRF8, whose importance we demonstrate through perturbation of network members. Mice bearing mutations in Flt3-ITD and Npm1c, which are commonly found in acute myeloid leukemia, are used to characterize the cooperative effects of these cancer drivers on the cellular epigenome and three-dimensional genome conformation during tumor development.},
  archive      = {J_NG},
  author       = {Yun, Haiyang and Narayan, Nisha and Vohra, Shabana and Giotopoulos, George and Mupo, Annalisa and Madrigal, Pedro and Sasca, Daniel and Lara-Astiaso, David and Horton, Sarah J. and Agrawal-Singh, Shuchi and Meduri, Eshwar and Basheer, Faisal and Marando, Ludovica and Gozdecka, Malgorzata and Dovey, Oliver M. and Castillo-Venzor, Aracely and Wang, Xiaonan and Gallipoli, Paolo and Müller-Tidow, Carsten and Osborne, Cameron S. and Vassiliou, George S. and Huntly, Brian J. P.},
  doi          = {10.1038/s41588-021-00925-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1443-1455},
  shortjournal = {Nat. Genet.},
  title        = {Mutational synergy during leukemia induction remodels chromatin accessibility, histone modifications and three-dimensional DNA topology to alter gene expression},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Increased somatic mutation burdens in normal human cells due
to defective DNA polymerases. <em>NG</em>, <em>53</em>(10), 1434–1442.
(<a href="https://doi.org/10.1038/s41588-021-00930-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutation accumulation in somatic cells contributes to cancer development and is proposed as a cause of aging. DNA polymerases Pol ε and Pol δ replicate DNA during cell division. However, in some cancers, defective proofreading due to acquired POLE/POLD1 exonuclease domain mutations causes markedly elevated somatic mutation burdens with distinctive mutational signatures. Germline POLE/POLD1 mutations cause familial cancer predisposition. Here, we sequenced normal tissue and tumor DNA from individuals with germline POLE/POLD1 mutations. Increased mutation burdens with characteristic mutational signatures were found in normal adult somatic cell types, during early embryogenesis and in sperm. Thus human physiology can tolerate ubiquitously elevated mutation burdens. Except for increased cancer risk, individuals with germline POLE/POLD1 mutations do not exhibit overt features of premature aging. These results do not support a model in which all features of aging are attributable to widespread cell malfunction directly resulting from somatic mutation burdens accrued during life. Healthy tissues from individuals with germline mutations in POLE or POLD1 show increased mutational burden, suggesting that normal cells are capable of tolerating high mutation rates.},
  archive      = {J_NG},
  author       = {Robinson, Philip S. and Coorens, Tim H. H. and Palles, Claire and Mitchell, Emily and Abascal, Federico and Olafsson, Sigurgeir and Lee, Bernard C. H. and Lawson, Andrew R. J. and Lee-Six, Henry and Moore, Luiza and Sanders, Mathijs A. and Hewinson, James and Martin, Lynn and Pinna, Claudia M. A. and Galavotti, Sara and Rahbari, Raheleh and Campbell, Peter J. and Martincorena, Iñigo and Tomlinson, Ian and Stratton, Michael R.},
  doi          = {10.1038/s41588-021-00930-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1434-1442},
  shortjournal = {Nat. Genet.},
  title        = {Increased somatic mutation burdens in normal human cells due to defective DNA polymerases},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Polygenic basis and biomedical consequences of telomere
length variation. <em>NG</em>, <em>53</em>(10), 1425–1433. (<a
href="https://doi.org/10.1038/s41588-021-00944-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Telomeres, the end fragments of chromosomes, play key roles in cellular proliferation and senescence. Here we characterize the genetic architecture of naturally occurring variation in leukocyte telomere length (LTL) and identify causal links between LTL and biomedical phenotypes in 472,174 well-characterized UK Biobank participants. We identified 197 independent sentinel variants associated with LTL at 138 genomic loci (108 new). Genetically determined differences in LTL were associated with multiple biological traits, ranging from height to bone marrow function, as well as several diseases spanning neoplastic, vascular and inflammatory pathologies. Finally, we estimated that, at the age of 40 years, people with an LTL &amp;gt;1 s.d. shorter than the population mean had a 2.5-year-lower life expectancy compared with the group with ≥1 s.d. longer LDL. Overall, we furnish new insights into the genetic regulation of LTL, reveal wide-ranging influences of LTL on physiological traits, diseases and longevity, and provide a powerful resource available to the global research community. Genome-wide association and Mendelian randomization analyses in the UK Biobank identify genetic variants associated with leukocyte telomere length and highlight putative causal links between telomere length and biomedical phenotypes.},
  archive      = {J_NG},
  author       = {Codd, Veryan and Wang, Qingning and Allara, Elias and Musicha, Crispin and Kaptoge, Stephen and Stoma, Svetlana and Jiang, Tao and Hamby, Stephen E. and Braund, Peter S. and Bountziouka, Vasiliki and Budgeon, Charley A. and Denniff, Matthew and Swinfield, Chloe and Papakonstantinou, Manolo and Sheth, Shilpi and Nanus, Dominika E. and Warner, Sophie C. and Wang, Minxian and Khera, Amit V. and Eales, James and Ouwehand, Willem H. and Thompson, John R. and Di Angelantonio, Emanuele and Wood, Angela M. and Butterworth, Adam S. and Danesh, John N. and Nelson, Christopher P. and Samani, Nilesh J.},
  doi          = {10.1038/s41588-021-00944-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1425-1433},
  shortjournal = {Nat. Genet.},
  title        = {Polygenic basis and biomedical consequences of telomere length variation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A cross-population atlas of genetic associations for 220
human phenotypes. <em>NG</em>, <em>53</em>(10), 1415–1424. (<a
href="https://doi.org/10.1038/s41588-021-00931-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current genome-wide association studies do not yet capture sufficient diversity in populations and scope of phenotypes. To expand an atlas of genetic associations in non-European populations, we conducted 220 deep-phenotype genome-wide association studies (diseases, biomarkers and medication usage) in BioBank Japan (n = 179,000), by incorporating past medical history and text-mining of electronic medical records. Meta-analyses with the UK Biobank and FinnGen (ntotal = 628,000) identified ~5,000 new loci, which improved the resolution of the genomic map of human traits. This atlas elucidated the landscape of pleiotropy as represented by the major histocompatibility complex locus, where we conducted HLA fine-mapping. Finally, we performed statistical decomposition of matrices of phenome-wide summary statistics, and identified latent genetic components, which pinpointed responsible variants and biological mechanisms underlying current disease classifications across populations. The decomposed components enabled genetically informed subtyping of similar diseases (for example, allergic diseases). Our study suggests a potential avenue for hypothesis-free re-investigation of human diseases through genetics. Genome-wide analyses in BioBank Japan, UK Biobank and FinnGen identify ~5,000 new loci associated with 220 human traits. Statistical decomposition of matrices of phenome-wide summary statistics further highlights variants underpinning diseases across populations.},
  archive      = {J_NG},
  author       = {Sakaue, Saori and Kanai, Masahiro and Tanigawa, Yosuke and Karjalainen, Juha and Kurki, Mitja and Koshiba, Seizo and Narita, Akira and Konuma, Takahiro and Yamamoto, Kenichi and Akiyama, Masato and Ishigaki, Kazuyoshi and Suzuki, Akari and Suzuki, Ken and Obara, Wataru and Yamaji, Ken and Takahashi, Kazuhisa and Asai, Satoshi and Takahashi, Yasuo and Suzuki, Takao and Shinozaki, Nobuaki and Yamaguchi, Hiroki and Minami, Shiro and Murayama, Shigeo and Yoshimori, Kozo and Nagayama, Satoshi and Obata, Daisuke and Higashiyama, Masahiko and Masumoto, Akihide and Koretsune, Yukihiro and Ito, Kaoru and Terao, Chikashi and Yamauchi, Toshimasa and Komuro, Issei and Kadowaki, Takashi and Tamiya, Gen and Yamamoto, Masayuki and Nakamura, Yusuke and Kubo, Michiaki and Murakami, Yoshinori and Yamamoto, Kazuhiko and Kamatani, Yoichiro and Palotie, Aarno and Rivas, Manuel A. and Daly, Mark J. and Matsuda, Koichi and Okada, Yukinori},
  doi          = {10.1038/s41588-021-00931-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1415-1424},
  shortjournal = {Nat. Genet.},
  title        = {A cross-population atlas of genetic associations for 220 human phenotypes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The first wave of the COVID-19 epidemic in spain was
associated with early introductions and fast spread of a dominating
genetic variant. <em>NG</em>, <em>53</em>(10), 1405–1414. (<a
href="https://doi.org/10.1038/s41588-021-00936-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The coronavirus disease 2019 (COVID-19) pandemic has affected the world radically since 2020. Spain was one of the European countries with the highest incidence during the first wave. As a part of a consortium to monitor and study the evolution of the epidemic, we sequenced 2,170 samples, diagnosed mostly before lockdown measures. Here, we identified at least 500 introductions from multiple international sources and documented the early rise of two dominant Spanish epidemic clades (SECs), probably amplified by superspreading events. Both SECs were related closely to the initial Asian variants of SARS-CoV-2 and spread widely across Spain. We inferred a substantial reduction in the effective reproductive number of both SECs due to public-health interventions (Re &amp;lt; 1), also reflected in the replacement of SECs by a new variant over the summer of 2020. In summary, we reveal a notable difference in the initial genetic makeup of SARS-CoV-2 in Spain compared with other European countries and show evidence to support the effectiveness of lockdown measures in controlling virus spread, even for the most successful genetic variants. Analysis of 2,170 SARS-CoV-2 sequences from the first wave of the COVID-19 epidemic in Spain provides insights into transmission patterns and the effects of lockdown on the emergence of new variants.},
  archive      = {J_NG},
  author       = {López, Mariana G. and Chiner-Oms, Álvaro and García de Viedma, Darío and Ruiz-Rodriguez, Paula and Bracho, Maria Alma and Cancino-Muñoz, Irving and D’Auria, Giuseppe and de Marco, Griselda and García-González, Neris and Goig, Galo Adrian and Gómez-Navarro, Inmaculada and Jiménez-Serrano, Santiago and Martinez-Priego, Llúcia and Ruiz-Hueso, Paula and Ruiz-Roldán, Lidia and Torres-Puente, Manuela and Alberola, Juan and Albert, Eliseo and Aranzamendi Zaldumbide, Maitane and Bea-Escudero, María Pilar and Boga, Jose Antonio and Bordoy, Antoni E. and Canut-Blasco, Andrés and Carvajal, Ana and Cilla Eguiluz, Gustavo and Cordón Rodríguez, Maria Luz and Costa-Alcalde, José J. and de Toro, María and de Toro Peinado, Inmaculada and del Pozo, Jose Luis and Duchêne, Sebastián and Fernández-Pinero, Jovita and Fuster Escrivá, Begoña and Gimeno Cardona, Concepción and González Galán, Verónica and Gonzalo Jiménez, Nieves and Hernáez Crespo, Silvia and Herranz, Marta and Lepe, José Antonio and López-Causapé, Carla and López-Hontangas, José Luis and Martín, Vicente and Martró, Elisa and Milagro Beamonte, Ana and Montes Ros, Milagrosa and Moreno-Muñoz, Rosario and Navarro, David and Navarro-Marí, José María and Not, Anna and Oliver, Antonio and Palop-Borrás, Begoña and Parra Grande, Mónica and Pedrosa-Corral, Irene and Pérez González, Maria Carmen and Pérez-Lago, Laura and Pérez-Ruiz, Mercedes and Piñeiro Vázquez, Luis and Rabella, Nuria and Rezusta, Antonio and Robles Fonseca, Lorena and Rodríguez-Villodres, Ángel and Sanbonmatsu-Gámez, Sara and Sicilia, Jon and Soriano, Alex and Tirado Balaguer, María Dolores and Torres, Ignacio and Tristancho, Alexander and Marimón, José María and Coscolla, Mireia and González-Candelas, Fernando and Comas, Iñaki},
  doi          = {10.1038/s41588-021-00936-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1405-1414},
  shortjournal = {Nat. Genet.},
  title        = {The first wave of the COVID-19 epidemic in spain was associated with early introductions and fast spread of a dominating genetic variant},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Epigenomic contributions to tumor cell heterogeneity and
plasticity. <em>NG</em>, <em>53</em>(10), 1403–1404. (<a
href="https://doi.org/10.1038/s41588-021-00932-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Heterogeneity in brain tumors has been viewed through many lenses—from microscopes and experimental models to ‘omic’ analysis at the tissue and single-cell levels. Two studies now characterize patterns of DNA methylation and gene expression in single cells to reveal epigenomic underpinnings of cellular heterogeneity and plasticity in exquisite detail, including mechanistic insight into cellular transitions between stem-like and differentiated-like states.},
  archive      = {J_NG},
  author       = {Mathur, Radhika and Costello, Joseph F.},
  doi          = {10.1038/s41588-021-00932-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1403-1404},
  shortjournal = {Nat. Genet.},
  title        = {Epigenomic contributions to tumor cell heterogeneity and plasticity},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genomic insights into the origin, domestication and
diversification of brassica juncea. <em>NG</em>, <em>53</em>(9),
1392–1402. (<a
href="https://doi.org/10.1038/s41588-021-00922-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite early domestication around 3000 BC, the evolutionary history of the ancient allotetraploid species Brassica juncea (L.) Czern &amp;amp; Coss remains uncertain. Here, we report a chromosome-scale de novo assembly of a yellow-seeded B. juncea genome by integrating long-read and short-read sequencing, optical mapping and Hi-C technologies. Nuclear and organelle phylogenies of 480 accessions worldwide supported that B. juncea is most likely a single origin in West Asia, 8,000–14,000 years ago, via natural interspecific hybridization. Subsequently, new crop types evolved through spontaneous gene mutations and introgressions along three independent routes of eastward expansion. Selective sweeps, genome-wide trait associations and tissue-specific RNA-sequencing analysis shed light on the domestication history of flowering time and seed weight, and on human selection for morphological diversification in this versatile species. Our data provide a comprehensive insight into the origin and domestication and a foundation for genomics-based breeding of B. juncea. A chromosome-scale de novo assembly of a yellow-seeded Brassica juncea genome and population analyses of 480 accessions from 38 countries provide insights into the origin, domestication history and morphological diversification of B. juncea.},
  archive      = {J_NG},
  author       = {Kang, Lei and Qian, Lunwen and Zheng, Ming and Chen, Liyang and Chen, Hao and Yang, Liu and You, Liang and Yang, Bin and Yan, Mingli and Gu, Yuanguo and Wang, Tianyi and Schiessl, Sarah-Veronica and An, Hong and Blischak, Paul and Liu, Xianjun and Lu, Hongfeng and Zhang, Dawei and Rao, Yong and Jia, Donghai and Zhou, Dinggang and Xiao, Huagui and Wang, Yonggang and Xiong, Xinghua and Mason, Annaliese S. and Chris Pires, J. and Snowdon, Rod J. and Hua, Wei and Liu, Zhongsong},
  doi          = {10.1038/s41588-021-00922-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1392-1402},
  shortjournal = {Nat. Genet.},
  title        = {Genomic insights into the origin, domestication and diversification of brassica juncea},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-quality genome assembly and resequencing of modern
cotton cultivars provide resources for crop improvement. <em>NG</em>,
<em>53</em>(9), 1385–1391. (<a
href="https://doi.org/10.1038/s41588-021-00910-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cotton produces natural fiber for the textile industry. The genetic effects of genomic structural variations underlying agronomic traits remain unclear. Here, we generate two high-quality genomes of Gossypium hirsutum cv. NDM8 and Gossypium barbadense acc. Pima90, and identify large-scale structural variations in the two species and 1,081 G. hirsutum accessions. The density of structural variations is higher in the D-subgenome than in the A-subgenome, indicating that the D-subgenome undergoes stronger selection during species formation and variety development. Many structural variations in genes and/or regulatory regions potentially influencing agronomic traits were discovered. Of 446 significantly associated structural variations, those for fiber quality and Verticillium wilt resistance are located mainly in the D-subgenome and those for yield mainly in the A-subgenome. Our research provides insight into the role of structural variations in genotype-to-phenotype relationships and their potential utility in crop improvement. High-quality genomes of two cultivated tetraploid cottons Gossypium hirsutum cv. NDM8 and Gossypium barbadense acc. Pima90 and resequencing of 1,081 G. hirsutum accessions provide insights into the role of structural variations.},
  archive      = {J_NG},
  author       = {Ma, Zhiying and Zhang, Yan and Wu, Liqiang and Zhang, Guiyin and Sun, Zhengwen and Li, Zhikun and Jiang, Yafei and Ke, Huifeng and Chen, Bin and Liu, Zhengwen and Gu, Qishen and Wang, Zhicheng and Wang, Guoning and Yang, Jun and Wu, Jinhua and Yan, Yuanyuan and Meng, Chengsheng and Li, Lihua and Li, Xiuxin and Mo, Shaojing and Wu, Nan and Ma, Limei and Chen, Liting and Zhang, Man and Si, Aijun and Yang, Zhanwu and Wang, Nan and Wu, Lizhu and Zhang, Dongmei and Cui, Yanru and Cui, Jing and Lv, Xing and Li, Yang and Shi, Rongkang and Duan, Yihong and Tian, Shilin and Wang, Xingfen},
  doi          = {10.1038/s41588-021-00910-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1385-1391},
  shortjournal = {Nat. Genet.},
  title        = {High-quality genome assembly and resequencing of modern cotton cultivars provide resources for crop improvement},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The bowfin genome illuminates the developmental evolution of
ray-finned fishes. <em>NG</em>, <em>53</em>(9), 1373–1384. (<a
href="https://doi.org/10.1038/s41588-021-00914-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The bowfin (Amia calva) is a ray-finned fish that possesses a unique suite of ancestral and derived phenotypes, which are key to understanding vertebrate evolution. The phylogenetic position of bowfin as a representative of neopterygian fishes, its archetypical body plan and its unduplicated and slowly evolving genome make bowfin a central species for the genomic exploration of ray-finned fishes. Here we present a chromosome-level genome assembly for bowfin that enables gene-order analyses, settling long-debated neopterygian phylogenetic relationships. We examine chromatin accessibility and gene expression through bowfin development to investigate the evolution of immune, scale, respiratory and fin skeletal systems and identify hundreds of gene-regulatory loci conserved across vertebrates. These resources connect developmental evolution among bony fishes, further highlighting the bowfin’s importance for illuminating vertebrate biology and diversity in the genomic era. Analysis of a chromosome-level bowfin genome assembly sheds light into neopterygian fish evolution. Chromatin accessibility and gene expression profiling provides insight into bowfin embryonic development.},
  archive      = {J_NG},
  author       = {Thompson, Andrew W. and Hawkins, M. Brent and Parey, Elise and Wcisel, Dustin J. and Ota, Tatsuya and Kawasaki, Kazuhiko and Funk, Emily and Losilla, Mauricio and Fitch, Olivia E. and Pan, Qiaowei and Feron, Romain and Louis, Alexandra and Montfort, Jérôme and Milhes, Marine and Racicot, Brett L. and Childs, Kevin L. and Fontenot, Quenton and Ferrara, Allyse and David, Solomon R. and McCune, Amy R. and Dornburg, Alex and Yoder, Jeffrey A. and Guiguen, Yann and Roest Crollius, Hugues and Berthelot, Camille and Harris, Matthew P. and Braasch, Ingo},
  doi          = {10.1038/s41588-021-00914-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1373-1384},
  shortjournal = {Nat. Genet.},
  title        = {The bowfin genome illuminates the developmental evolution of ray-finned fishes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Somatic mutations of GNA11 and GNAQ in CTNNB1-mutant
aldosterone-producing adenomas presenting in puberty, pregnancy or
menopause. <em>NG</em>, <em>53</em>(9), 1360–1372. (<a
href="https://doi.org/10.1038/s41588-021-00906-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most aldosterone-producing adenomas (APAs) have gain-of-function somatic mutations of ion channels or transporters. However, their frequency in aldosterone-producing cell clusters of normal adrenal gland suggests a requirement for codriver mutations in APAs. Here we identified gain-of-function mutations in both CTNNB1 and GNA11 by whole-exome sequencing of 3/41 APAs. Further sequencing of known CTNNB1-mutant APAs led to a total of 16 of 27 (59%) with a somatic p.Gln209His, p.Gln209Pro or p.Gln209Leu mutation of GNA11 or GNAQ. Solitary GNA11 mutations were found in hyperplastic zona glomerulosa adjacent to double-mutant APAs. Nine of ten patients in our UK/Irish cohort presented in puberty, pregnancy or menopause. Among multiple transcripts upregulated more than tenfold in double-mutant APAs was LHCGR, the receptor for luteinizing or pregnancy hormone (human chorionic gonadotropin). Transfections of adrenocortical cells demonstrated additive effects of GNA11 and CTNNB1 mutations on aldosterone secretion and expression of genes upregulated in double-mutant APAs. In adrenal cortex, GNA11/Q mutations appear clinically silent without a codriver mutation of CTNNB1. Sequence analysis identifies gain-of-function somatic mutations in GNA11 or GNAQ in CTNNB1-mutant aldosterone-producing adenomas. Most patients with these mutations presented during puberty, pregnancy or menopause, with elevated LHCGR expression.},
  archive      = {J_NG},
  author       = {Zhou, Junhua and Azizan, Elena A. B. and Cabrera, Claudia P. and Fernandes-Rosa, Fabio L. and Boulkroun, Sheerazed and Argentesi, Giulia and Cottrell, Emily and Amar, Laurence and Wu, Xilin and O’Toole, Sam and Goodchild, Emily and Marker, Alison and Senanayake, Russell and Garg, Sumedha and Åkerström, Tobias and Backman, Samuel and Jordan, Suzanne and Polubothu, Satyamaanasa and Berney, Daniel M. and Gluck, Anna and Lines, Kate E. and Thakker, Rajesh V. and Tuthill, Antoinette and Joyce, Caroline and Kaski, Juan Pablo and Karet Frankl, Fiona E. and Metherell, Lou A. and Teo, Ada E. D. and Gurnell, Mark and Parvanta, Laila and Drake, William M. and Wozniak, Eva and Klinzing, David and Kuan, Jyn Ling and Tiang, Zenia and Gomez Sanchez, Celso E. and Hellman, Per and Foo, Roger S. Y. and Mein, Charles A. and Kinsler, Veronica A. and Björklund, Peyman and Storr, Helen L. and Zennaro, Maria-Christina and Brown, Morris J.},
  doi          = {10.1038/s41588-021-00906-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1360-1372},
  shortjournal = {Nat. Genet.},
  title        = {Somatic mutations of GNA11 and GNAQ in CTNNB1-mutant aldosterone-producing adenomas presenting in puberty, pregnancy or menopause},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genomic and evolutionary classification of lung cancer in
never smokers. <em>NG</em>, <em>53</em>(9), 1348–1359. (<a
href="https://doi.org/10.1038/s41588-021-00920-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lung cancer in never smokers (LCINS) is a common cause of cancer mortality but its genomic landscape is poorly characterized. Here high-coverage whole-genome sequencing of 232 LCINS showed 3 subtypes defined by copy number aberrations. The dominant subtype (piano), which is rare in lung cancer in smokers, features somatic UBA1 mutations, germline AR variants and stem cell-like properties, including low mutational burden, high intratumor heterogeneity, long telomeres, frequent KRAS mutations and slow growth, as suggested by the occurrence of cancer drivers’ progenitor cells many years before tumor diagnosis. The other subtypes are characterized by specific amplifications and EGFR mutations (mezzo-forte) and whole-genome doubling (forte). No strong tobacco smoking signatures were detected, even in cases with exposure to secondhand tobacco smoke. Genes within the receptor tyrosine kinase–Ras pathway had distinct impacts on survival; five genomic alterations independently doubled mortality. These findings create avenues for personalized treatment in LCINS. Whole-genome sequencing of lung cancer in never smokers identifies different copy number subtypes and shows a lack of tobacco smoking signatures, even in cases exposed to secondhand smoke.},
  archive      = {J_NG},
  author       = {Zhang, Tongwu and Joubert, Philippe and Ansari-Pour, Naser and Zhao, Wei and Hoang, Phuc H. and Lokanga, Rachel and Moye, Aaron L. and Rosenbaum, Jennifer and Gonzalez-Perez, Abel and Martínez-Jiménez, Francisco and Castro, Andrea and Muscarella, Lucia Anna and Hofman, Paul and Consonni, Dario and Pesatori, Angela C. and Kebede, Michael and Li, Mengying and Gould Rothberg, Bonnie E. and Peneva, Iliana and Schabath, Matthew B. and Poeta, Maria Luana and Costantini, Manuela and Hirsch, Daniela and Heselmeyer-Haddad, Kerstin and Hutchinson, Amy and Olanich, Mary and Lawrence, Scott M. and Lenz, Petra and Duggan, Maire and Bhawsar, Praphulla M. S. and Sang, Jian and Kim, Jung and Mendoza, Laura and Saini, Natalie and Klimczak, Leszek J. and Islam, S. M. Ashiqul and Otlu, Burcak and Khandekar, Azhar and Cole, Nathan and Stewart, Douglas R. and Choi, Jiyeon and Brown, Kevin M. and Caporaso, Neil E. and Wilson, Samuel H. and Pommier, Yves and Lan, Qing and Rothman, Nathaniel and Almeida, Jonas S. and Carter, Hannah and Ried, Thomas and Kim, Carla F. and Lopez-Bigas, Nuria and Garcia-Closas, Montserrat and Shi, Jianxin and Bossé, Yohan and Zhu, Bin and Gordenin, Dmitry A. and Alexandrov, Ludmil B. and Chanock, Stephen J. and Wedge, David C. and Landi, Maria Teresa},
  doi          = {10.1038/s41588-021-00920-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1348-1359},
  shortjournal = {Nat. Genet.},
  title        = {Genomic and evolutionary classification of lung cancer in never smokers},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A single-cell and spatially resolved atlas of human breast
cancers. <em>NG</em>, <em>53</em>(9), 1334–1347. (<a
href="https://doi.org/10.1038/s41588-021-00911-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breast cancers are complex cellular ecosystems where heterotypic interactions play central roles in disease progression and response to therapy. However, our knowledge of their cellular composition and organization is limited. Here we present a single-cell and spatially resolved transcriptomics analysis of human breast cancers. We developed a single-cell method of intrinsic subtype classification (SCSubtype) to reveal recurrent neoplastic cell heterogeneity. Immunophenotyping using cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) provides high-resolution immune profiles, including new PD-L1/PD-L2+ macrophage populations associated with clinical outcome. Mesenchymal cells displayed diverse functions and cell-surface protein expression through differentiation within three major lineages. Stromal-immune niches were spatially organized in tumors, offering insights into antitumor immune regulation. Using single-cell signatures, we deconvoluted large breast cancer cohorts to stratify them into nine clusters, termed ‘ecotypes’, with unique cellular compositions and clinical outcomes. This study provides a comprehensive transcriptional atlas of the cellular architecture of breast cancer. A multi-omic atlas of breast cancers, integrating single-cell RNA sequencing, spatial transcriptomics and immunophenotyping, identifies nine ecotypes associated with cellular heterogeneity and prognosis.},
  archive      = {J_NG},
  author       = {Wu, Sunny Z. and Al-Eryani, Ghamdan and Roden, Daniel Lee and Junankar, Simon and Harvey, Kate and Andersson, Alma and Thennavan, Aatish and Wang, Chenfei and Torpy, James R. and Bartonicek, Nenad and Wang, Taopeng and Larsson, Ludvig and Kaczorowski, Dominik and Weisenfeld, Neil I. and Uytingco, Cedric R. and Chew, Jennifer G. and Bent, Zachary W. and Chan, Chia-Ling and Gnanasambandapillai, Vikkitharan and Dutertre, Charles-Antoine and Gluch, Laurence and Hui, Mun N. and Beith, Jane and Parker, Andrew and Robbins, Elizabeth and Segara, Davendra and Cooper, Caroline and Mak, Cindy and Chan, Belinda and Warrier, Sanjay and Ginhoux, Florent and Millar, Ewan and Powell, Joseph E. and Williams, Stephen R. and Liu, X. Shirley and O’Toole, Sandra and Lim, Elgene and Lundeberg, Joakim and Perou, Charles M. and Swarbrick, Alexander},
  doi          = {10.1038/s41588-021-00911-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1334-1347},
  shortjournal = {Nat. Genet.},
  title        = {A single-cell and spatially resolved atlas of human breast cancers},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mapping the genetic architecture of human traits to cell
types in the kidney identifies mechanisms of disease and potential
treatments. <em>NG</em>, <em>53</em>(9), 1322–1333. (<a
href="https://doi.org/10.1038/s41588-021-00909-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The functional interpretation of genome-wide association studies (GWAS) is challenging due to the cell-type-dependent influences of genetic variants. Here, we generated comprehensive maps of expression quantitative trait loci (eQTLs) for 659 microdissected human kidney samples and identified cell-type-eQTLs by mapping interactions between cell type abundances and genotypes. By partitioning heritability using stratified linkage disequilibrium score regression to integrate GWAS with single-cell RNA sequencing and single-nucleus assay for transposase-accessible chromatin with high-throughput sequencing data, we prioritized proximal tubules for kidney function and endothelial cells and distal tubule segments for blood pressure pathogenesis. Bayesian colocalization analysis nominated more than 200 genes for kidney function and hypertension. Our study clarifies the mechanism of commonly used antihypertensive and renal-protective drugs and identifies drug repurposing opportunities for kidney disease. Cell-type-specific eQTL maps in the human kidney generated from the analysis of over 600 microdissected kidney samples, together with single-cell RNA sequencing and single-nucleus ATAC-seq, prioritize cell types influencing kidney function, hypertension and other traits.},
  archive      = {J_NG},
  author       = {Sheng, Xin and Guan, Yuting and Ma, Ziyuan and Wu, Junnan and Liu, Hongbo and Qiu, Chengxiang and Vitale, Steven and Miao, Zhen and Seasock, Matthew J. and Palmer, Matthew and Shin, Myung K. and Duffin, Kevin L. and Pullen, Steven S. and Edwards, Todd L. and Hellwege, Jacklyn N. and Hung, Adriana M. and Li, Mingyao and Voight, Benjamin F. and Coffman, Thomas M. and Brown, Christopher D. and Susztak, Katalin},
  doi          = {10.1038/s41588-021-00909-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1322-1333},
  shortjournal = {Nat. Genet.},
  title        = {Mapping the genetic architecture of human traits to cell types in the kidney identifies mechanisms of disease and potential treatments},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genomic and phenotypic insights from an atlas of genetic
effects on DNA methylation. <em>NG</em>, <em>53</em>(9), 1311–1321. (<a
href="https://doi.org/10.1038/s41588-021-00923-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing genetic influences on DNA methylation (DNAm) provides an opportunity to understand mechanisms underpinning gene regulation and disease. In the present study, we describe results of DNAm quantitative trait locus (mQTL) analyses on 32,851 participants, identifying genetic variants associated with DNAm at 420,509 DNAm sites in blood. We present a database of &amp;gt;270,000 independent mQTLs, of which 8.5% comprise long-range (trans) associations. Identified mQTL associations explain 15–17% of the additive genetic variance of DNAm. We show that the genetic architecture of DNAm levels is highly polygenic. Using shared genetic control between distal DNAm sites, we constructed networks, identifying 405 discrete genomic communities enriched for genomic annotations and complex traits. Shared genetic variants are associated with both DNAm levels and complex diseases, but only in a minority of cases do these associations reflect causal relationships from DNAm to trait or vice versa, indicating a more complex genotype–phenotype map than previously anticipated. DNA methylation quantitative trait locus (mQTL) analyses on 32,851 participants identify genetic variants associated with DNA methylation at 420,509 sites in blood, resulting in a database of &amp;gt;270,000 independent mQTLs.},
  archive      = {J_NG},
  author       = {Min, Josine L. and Hemani, Gibran and Hannon, Eilis and Dekkers, Koen F. and Castillo-Fernandez, Juan and Luijk, René and Carnero-Montoro, Elena and Lawson, Daniel J. and Burrows, Kimberley and Suderman, Matthew and Bretherick, Andrew D. and Richardson, Tom G. and Klughammer, Johanna and Iotchkova, Valentina and Sharp, Gemma and Al Khleifat, Ahmad and Shatunov, Aleksey and Iacoangeli, Alfredo and McArdle, Wendy L. and Ho, Karen M. and Kumar, Ashish and Söderhäll, Cilla and Soriano-Tárraga, Carolina and Giralt-Steinhauer, Eva and Kazmi, Nabila and Mason, Dan and McRae, Allan F. and Corcoran, David L. and Sugden, Karen and Kasela, Silva and Cardona, Alexia and Day, Felix R. and Cugliari, Giovanni and Viberti, Clara and Guarrera, Simonetta and Lerro, Michael and Gupta, Richa and Bollepalli, Sailalitha and Mandaviya, Pooja and Zeng, Yanni and Clarke, Toni-Kim and Walker, Rosie M. and Schmoll, Vanessa and Czamara, Darina and Ruiz-Arenas, Carlos and Rezwan, Faisal I. and Marioni, Riccardo E. and Lin, Tian and Awaloff, Yvonne and Germain, Marine and Aïssi, Dylan and Zwamborn, Ramona and van Eijk, Kristel and Dekker, Annelot and van Dongen, Jenny and Hottenga, Jouke-Jan and Willemsen, Gonneke and Xu, Cheng-Jian and Barturen, Guillermo and Català-Moll, Francesc and Kerick, Martin and Wang, Carol and Melton, Phillip and Elliott, Hannah R. and Shin, Jean and Bernard, Manon and Yet, Idil and Smart, Melissa and Gorrie-Stone, Tyler and Shaw, Chris and Al Chalabi, Ammar and Ring, Susan M. and Pershagen, Göran and Melén, Erik and Jiménez-Conde, Jordi and Roquer, Jaume and Lawlor, Deborah A. and Wright, John and Martin, Nicholas G. and Montgomery, Grant W. and Moffitt, Terrie E. and Poulton, Richie and Esko, Tõnu and Milani, Lili and Metspalu, Andres and Perry, John R. B. and Ong, Ken K. and Wareham, Nicholas J. and Matullo, Giuseppe and Sacerdote, Carlotta and Panico, Salvatore and Caspi, Avshalom and Arseneault, Louise and Gagnon, France and Ollikainen, Miina and Kaprio, Jaakko and Felix, Janine F. and Rivadeneira, Fernando and Tiemeier, Henning and van IJzendoorn, Marinus H. and Uitterlinden, André G. and Jaddoe, Vincent W. V. and Haley, Chris and McIntosh, Andrew M. and Evans, Kathryn L. and Murray, Alison and Räikkönen, Katri and Lahti, Jari and Nohr, Ellen A. and Sørensen, Thorkild I. A. and Hansen, Torben and Morgen, Camilla S. and Binder, Elisabeth B. and Lucae, Susanne and Gonzalez, Juan Ramon and Bustamante, Mariona and Sunyer, Jordi and Holloway, John W. and Karmaus, Wilfried and Zhang, Hongmei and Deary, Ian J. and Wray, Naomi R. and Starr, John M. and Beekman, Marian and van Heemst, Diana and Slagboom, P. Eline and Morange, Pierre-Emmanuel and Trégouët, David-Alexandre and Veldink, Jan H. and Davies, Gareth E. and de Geus, Eco J. C. and Boomsma, Dorret I. and Vonk, Judith M. and Brunekreef, Bert and Koppelman, Gerard H. and Alarcón-Riquelme, Marta E. and Huang, Rae-Chi and Pennell, Craig E. and van Meurs, Joyce and Ikram, M. Arfan and Hughes, Alun D. and Tillin, Therese and Chaturvedi, Nish and Pausova, Zdenka and Paus, Tomas and Spector, Timothy D. and Kumari, Meena and Schalkwyk, Leonard C. and Visscher, Peter M. and Davey Smith, George and Bock, Christoph and Gaunt, Tom R. and Bell, Jordana T. and Heijmans, Bastiaan T. and Mill, Jonathan and Relton, Caroline L.},
  doi          = {10.1038/s41588-021-00923-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1311-1321},
  shortjournal = {Nat. Genet.},
  title        = {Genomic and phenotypic insights from an atlas of genetic effects on DNA methylation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Large-scale cis- and trans-eQTL analyses identify thousands
of genetic loci and polygenic scores that regulate blood gene
expression. <em>NG</em>, <em>53</em>(9), 1300–1310. (<a
href="https://doi.org/10.1038/s41588-021-00913-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Trait-associated genetic variants affect complex phenotypes primarily via regulatory mechanisms on the transcriptome. To investigate the genetics of gene expression, we performed cis- and trans-expression quantitative trait locus (eQTL) analyses using blood-derived expression from 31,684 individuals through the eQTLGen Consortium. We detected cis-eQTL for 88% of genes, and these were replicable in numerous tissues. Distal trans-eQTL (detected for 37% of 10,317 trait-associated variants tested) showed lower replication rates, partially due to low replication power and confounding by cell type composition. However, replication analyses in single-cell RNA-seq data prioritized intracellular trans-eQTL. Trans-eQTL exerted their effects via several mechanisms, primarily through regulation by transcription factors. Expression of 13% of the genes correlated with polygenic scores for 1,263 phenotypes, pinpointing potential drivers for those traits. In summary, this work represents a large eQTL resource, and its results serve as a starting point for in-depth interpretation of complex phenotypes. Analyses of expression profiles from whole blood of 31,684 individuals identify cis-expression quantitative trait loci (eQTL) effects for 88% of genes and trans-eQTL effects for 37% of trait-associated variants.},
  archive      = {J_NG},
  author       = {Võsa, Urmo and Claringbould, Annique and Westra, Harm-Jan and Bonder, Marc Jan and Deelen, Patrick and Zeng, Biao and Kirsten, Holger and Saha, Ashis and Kreuzhuber, Roman and Yazar, Seyhan and Brugge, Harm and Oelen, Roy and de Vries, Dylan H. and van der Wijst, Monique G. P. and Kasela, Silva and Pervjakova, Natalia and Alves, Isabel and Favé, Marie-Julie and Agbessi, Mawussé and Christiansen, Mark W. and Jansen, Rick and Seppälä, Ilkka and Tong, Lin and Teumer, Alexander and Schramm, Katharina and Hemani, Gibran and Verlouw, Joost and Yaghootkar, Hanieh and Sönmez Flitman, Reyhan and Brown, Andrew and Kukushkina, Viktorija and Kalnapenkis, Anette and Rüeger, Sina and Porcu, Eleonora and Kronberg, Jaanika and Kettunen, Johannes and Lee, Bernett and Zhang, Futao and Qi, Ting and Hernandez, Jose Alquicira and Arindrarto, Wibowo and Beutner, Frank and Dmitrieva, Julia and Elansary, Mahmoud and Fairfax, Benjamin P. and Georges, Michel and Heijmans, Bastiaan T. and Hewitt, Alex W. and Kähönen, Mika and Kim, Yungil and Knight, Julian C. and Kovacs, Peter and Krohn, Knut and Li, Shuang and Loeffler, Markus and Marigorta, Urko M. and Mei, Hailang and Momozawa, Yukihide and Müller-Nurasyid, Martina and Nauck, Matthias and Nivard, Michel G. and Penninx, Brenda W. J. H. and Pritchard, Jonathan K. and Raitakari, Olli T. and Rotzschke, Olaf and Slagboom, Eline P. and Stehouwer, Coen D. A. and Stumvoll, Michael and Sullivan, Patrick and ’t Hoen, Peter A. C. and Thiery, Joachim and Tönjes, Anke and van Dongen, Jenny and van Iterson, Maarten and Veldink, Jan H. and Völker, Uwe and Warmerdam, Robert and Wijmenga, Cisca and Swertz, Morris and Andiappan, Anand and Montgomery, Grant W. and Ripatti, Samuli and Perola, Markus and Kutalik, Zoltan and Dermitzakis, Emmanouil and Bergmann, Sven and Frayling, Timothy and van Meurs, Joyce and Prokisch, Holger and Ahsan, Habibul and Pierce, Brandon L. and Lehtimäki, Terho and Boomsma, Dorret I. and Psaty, Bruce M. and Gharib, Sina A. and Awadalla, Philip and Milani, Lili and Ouwehand, Willem H. and Downes, Kate and Stegle, Oliver and Battle, Alexis and Visscher, Peter M. and Yang, Jian and Scholz, Markus and Powell, Joseph and Gibson, Greg and Esko, Tõnu and Franke, Lude},
  doi          = {10.1038/s41588-021-00913-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1300-1310},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale cis- and trans-eQTL analyses identify thousands of genetic loci and polygenic scores that regulate blood gene expression},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A compendium of uniformly processed human gene expression
and splicing quantitative trait loci. <em>NG</em>, <em>53</em>(9),
1290–1299. (<a
href="https://doi.org/10.1038/s41588-021-00924-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many gene expression quantitative trait locus (eQTL) studies have published their summary statistics, which can be used to gain insight into complex human traits by downstream analyses, such as fine mapping and co-localization. However, technical differences between these datasets are a barrier to their widespread use. Consequently, target genes for most genome-wide association study (GWAS) signals have still not been identified. In the present study, we present the eQTL Catalogue ( https://www.ebi.ac.uk/eqtl ), a resource of quality-controlled, uniformly re-computed gene expression and splicing QTLs from 21 studies. We find that, for matching cell types and tissues, the eQTL effect sizes are highly reproducible between studies. Although most QTLs were shared between most bulk tissues, we identified a greater diversity of cell-type-specific QTLs from purified cell types, a subset of which also manifested as new disease co-localizations. Our summary statistics are freely available to enable the systematic interpretation of human GWAS associations across many cell types and tissues. The eQTL Catalogue provides uniform processing of 21 eQTL studies, allowing the identification of highly reproducible eQTLs affecting whole gene and transcript expression levels.},
  archive      = {J_NG},
  author       = {Kerimov, Nurlan and Hayhurst, James D. and Peikova, Kateryna and Manning, Jonathan R. and Walter, Peter and Kolberg, Liis and Samoviča, Marija and Sakthivel, Manoj Pandian and Kuzmin, Ivan and Trevanion, Stephen J. and Burdett, Tony and Jupp, Simon and Parkinson, Helen and Papatheodorou, Irene and Yates, Andrew D. and Zerbino, Daniel R. and Alasoo, Kaur},
  doi          = {10.1038/s41588-021-00924-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1290-1299},
  shortjournal = {Nat. Genet.},
  title        = {A compendium of uniformly processed human gene expression and splicing quantitative trait loci},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Sex differences in genetic architecture in the UK biobank.
<em>NG</em>, <em>53</em>(9), 1283–1289. (<a
href="https://doi.org/10.1038/s41588-021-00912-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Males and females present differences in complex traits and in the risk of a wide array of diseases. Genotype by sex (GxS) interactions are thought to account for some of these differences. However, the extent and basis of GxS are poorly understood. In the present study, we provide insights into both the scope and the mechanism of GxS across the genome of about 450,000 individuals of European ancestry and 530 complex traits in the UK Biobank. We found small yet widespread differences in genetic architecture across traits. We also found that, in some cases, sex-agnostic analyses may be missing trait-associated loci and looked into possible improvements in the prediction of high-level phenotypes. Finally, we studied the potential functional role of the differences observed through sex-biased gene expression and gene-level analyses. Our results suggest the need to consider sex-aware analyses for future studies to shed light onto possible sex-specific molecular mechanisms. Sex-stratified GWAS analyses for 530 traits within 452,264 individuals of European ancestry from the UK Biobank provide insights into the scope of genotype by sex (GxS) interactions and the genetics of sex differences.},
  archive      = {J_NG},
  author       = {Bernabeu, Elena and Canela-Xandri, Oriol and Rawlik, Konrad and Talenti, Andrea and Prendergast, James and Tenesa, Albert},
  doi          = {10.1038/s41588-021-00912-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1283-1289},
  shortjournal = {Nat. Genet.},
  title        = {Sex differences in genetic architecture in the UK biobank},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). A genome-wide association study with 1,126,563 individuals
identifies new risk loci for alzheimer’s disease. <em>NG</em>,
<em>53</em>(9), 1276–1282. (<a
href="https://doi.org/10.1038/s41588-021-00921-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Late-onset Alzheimer’s disease is a prevalent age-related polygenic disease that accounts for 50–70% of dementia cases. Currently, only a fraction of the genetic variants underlying Alzheimer’s disease have been identified. Here we show that increased sample sizes allowed identification of seven previously unidentified genetic loci contributing to Alzheimer’s disease. This study highlights microglia, immune cells and protein catabolism as relevant to late-onset Alzheimer’s disease, while identifying and prioritizing previously unidentified genes of potential interest. We anticipate that these results can be included in larger meta-analyses of Alzheimer’s disease to identify further genetic variants that contribute to Alzheimer’s pathology. A genome-wide association study performed in 1,126,563 individuals identifies seven new loci associated with Alzheimer’s disease and implicates microglia and immune cells in late-onset disease.},
  archive      = {J_NG},
  author       = {Wightman, Douglas P. and Jansen, Iris E. and Savage, Jeanne E. and Shadrin, Alexey A. and Bahrami, Shahram and Holland, Dominic and Rongve, Arvid and Børte, Sigrid and Winsvold, Bendik S. and Drange, Ole Kristian and Martinsen, Amy E. and Skogholt, Anne Heidi and Willer, Cristen and Bråthen, Geir and Bosnes, Ingunn and Nielsen, Jonas Bille and Fritsche, Lars G. and Thomas, Laurent F. and Pedersen, Linda M. and Gabrielsen, Maiken E. and Johnsen, Marianne Bakke and Meisingset, Tore Wergeland and Zhou, Wei and Proitsi, Petroula and Hodges, Angela and Dobson, Richard and Velayudhan, Latha and Heilbron, Karl and Auton, Adam and Sealock, Julia M. and Davis, Lea K. and Pedersen, Nancy L. and Reynolds, Chandra A. and Karlsson, Ida K. and Magnusson, Sigurdur and Stefansson, Hreinn and Thordardottir, Steinunn and Jonsson, Palmi V. and Snaedal, Jon and Zettergren, Anna and Skoog, Ingmar and Kern, Silke and Waern, Margda and Zetterberg, Henrik and Blennow, Kaj and Stordal, Eystein and Hveem, Kristian and Zwart, John-Anker and Athanasiu, Lavinia and Selnes, Per and Saltvedt, Ingvild and Sando, Sigrid B. and Ulstein, Ingun and Djurovic, Srdjan and Fladby, Tormod and Aarsland, Dag and Selbæk, Geir and Ripke, Stephan and Stefansson, Kari and Andreassen, Ole A. and Posthuma, Danielle},
  doi          = {10.1038/s41588-021-00921-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1276-1282},
  shortjournal = {Nat. Genet.},
  title        = {A genome-wide association study with 1,126,563 individuals identifies new risk loci for alzheimer’s disease},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). RNA demethylation for increased crop yields. <em>NG</em>,
<em>53</em>(9), 1275. (<a
href="https://doi.org/10.1038/s41588-021-00937-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00937-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1275},
  shortjournal = {Nat. Genet.},
  title        = {RNA demethylation for increased crop yields},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). A single-cell-type transcriptomics map of human tissues.
<em>NG</em>, <em>53</em>(9), 1275. (<a
href="https://doi.org/10.1038/s41588-021-00938-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00938-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1275},
  shortjournal = {Nat. Genet.},
  title        = {A single-cell-type transcriptomics map of human tissues},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Bottleneck size and selection level reproducibly impact
evolution of antibiotic resistance. <em>NG</em>, <em>53</em>(9), 1275.
(<a href="https://doi.org/10.1038/s41588-021-00939-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00939-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1275},
  shortjournal = {Nat. Genet.},
  title        = {Bottleneck size and selection level reproducibly impact evolution of antibiotic resistance},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). In silico saturation mutagenesis of cancer genes.
<em>NG</em>, <em>53</em>(9), 1275. (<a
href="https://doi.org/10.1038/s41588-021-00940-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00940-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1275},
  shortjournal = {Nat. Genet.},
  title        = {In silico saturation mutagenesis of cancer genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). FoGS provides a public FAQ repository for social and
behavioral genomic discoveries. <em>NG</em>, <em>53</em>(9), 1272–1274.
(<a href="https://doi.org/10.1038/s41588-021-00929-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we introduce ‘FAQs on Genomic Studies’ (FoGS), an open-access repository of explanatory documents that accompany genomic analyses in social and behavioral genomics. For fields such as social and behavioral genomics that are shaped by an ugly history and uncertain future, socially and ethically responsible research and research communication are crucial. FoGS amplifies one such approach towards responsible research communication.},
  archive      = {J_NG},
  author       = {Martschenko, Daphne Oluwaseun and Domingue, Benjamin W. and Matthews, Lucas J. and Trejo, Sam},
  doi          = {10.1038/s41588-021-00929-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1272-1274},
  shortjournal = {Nat. Genet.},
  title        = {FoGS provides a public FAQ repository for social and behavioral genomic discoveries},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Embracing communication. <em>NG</em>, <em>53</em>(9), 1271.
(<a href="https://doi.org/10.1038/s41588-021-00942-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although it should be a given that scholarly communication must be clear and accurate, researchers, particularly those in the field of human genetics, can also promote the responsible reporting of their findings to a broader public audience in ways that heighten understanding and reduce misinterpretation.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-021-00942-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1271},
  shortjournal = {Nat. Genet.},
  title        = {Embracing communication},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Publisher correction: Germline de novo mutation clusters
arise during oocyte aging in genomic regions with high
double-strand-break incidence. <em>NG</em>, <em>53</em>(8), 1270. (<a
href="https://doi.org/10.1038/s41588-021-00905-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Goldmann, Jakob M. and Seplyarskiy, Vladimir B. and Wong, Wendy S. W. and Vilboux, Thierry and Neerincx, Pieter B. and Bodian, Dale L. and Solomon, Benjamin D. and Veltman, Joris A. and Deeken, John F. and Gilissen, Christian and Niederhuber, John E.},
  doi          = {10.1038/s41588-021-00905-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1270},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Germline de novo mutation clusters arise during oocyte aging in genomic regions with high double-strand-break incidence},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Whole-exome imputation within UK biobank powers rare coding
variant association and fine-mapping analyses. <em>NG</em>,
<em>53</em>(8), 1260–1269. (<a
href="https://doi.org/10.1038/s41588-021-00892-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exome association studies to date have generally been underpowered to systematically evaluate the phenotypic impact of very rare coding variants. We leveraged extensive haplotype sharing between 49,960 exome-sequenced UK Biobank participants and the remainder of the cohort (total n ≈ 500,000) to impute exome-wide variants with accuracy R2 &amp;gt; 0.5 down to minor allele frequency (MAF) ~0.00005. Association and fine-mapping analyses of 54 quantitative traits identified 1,189 significant associations (P &amp;lt; 5 × 10−8) involving 675 distinct rare protein-altering variants (MAF &amp;lt; 0.01) that passed stringent filters for likely causality. Across all traits, 49% of associations (578/1,189) occurred in genes with two or more hits; follow-up analyses of these genes identified allelic series containing up to 45 distinct ‘likely-causal’ variants. Our results demonstrate the utility of within-cohort imputation in population-scale genome-wide association studies, provide a catalog of likely-causal, large-effect coding variant associations and foreshadow the insights that will be revealed as genetic biobank studies continue to grow. Imputation of rare coding variants in the UK Biobank enables association and fine-mapping analyses of rare (minor allele frequency (MAF) = 0.00005) genotypes, identifying 600 new variant–trait associations, including long allelic series in individual genes.},
  archive      = {J_NG},
  author       = {Barton, Alison R. and Sherman, Maxwell A. and Mukamel, Ronen E. and Loh, Po-Ru},
  doi          = {10.1038/s41588-021-00892-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1260-1269},
  shortjournal = {Nat. Genet.},
  title        = {Whole-exome imputation within UK biobank powers rare coding variant association and fine-mapping analyses},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Haplotype-resolved genome assembly provides insights into
evolutionary history of the tea plant camellia sinensis. <em>NG</em>,
<em>53</em>(8), 1250–1259. (<a
href="https://doi.org/10.1038/s41588-021-00895-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tea is an important global beverage crop and is largely clonally propagated. Despite previous studies on the species, its genetic and evolutionary history deserves further research. Here, we present a haplotype-resolved assembly of an Oolong tea cultivar, Tieguanyin. Analysis of allele-specific expression suggests a potential mechanism in response to mutation load during long-term clonal propagation. Population genomic analysis using 190 Camellia accessions uncovered independent evolutionary histories and parallel domestication in two widely cultivated varieties, var. sinensis and var. assamica. It also revealed extensive intra- and interspecific introgressions contributing to genetic diversity in modern cultivars. Strong signatures of selection were associated with biosynthetic and metabolic pathways that contribute to flavor characteristics as well as genes likely involved in the Green Revolution in the tea industry. Our results offer genetic and molecular insights into the evolutionary history of Camellia sinensis and provide genomic resources to further facilitate gene editing to enhance desirable traits in tea crops. Haplotype-resolved genome assembly of an Oolong tea cultivar Tieguanyin and population genomic analyses of 190 Camellia accessions provide insights into the evolutionary history of the tea plant Camellia sinensis.},
  archive      = {J_NG},
  author       = {Zhang, Xingtan and Chen, Shuai and Shi, Longqing and Gong, Daping and Zhang, Shengcheng and Zhao, Qian and Zhan, Dongliang and Vasseur, Liette and Wang, Yibin and Yu, Jiaxin and Liao, Zhenyang and Xu, Xindan and Qi, Rui and Wang, Wenling and Ma, Yunran and Wang, Pengjie and Ye, Naixing and Ma, Dongna and Shi, Yan and Wang, Haifeng and Ma, Xiaokai and Kong, Xiangrui and Lin, Jing and Wei, Liufeng and Ma, Yaying and Li, Ruoyu and Hu, Guiping and He, Haifang and Zhang, Lin and Ming, Ray and Wang, Gang and Tang, Haibao and You, Minsheng},
  doi          = {10.1038/s41588-021-00895-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1250-1259},
  shortjournal = {Nat. Genet.},
  title        = {Haplotype-resolved genome assembly provides insights into evolutionary history of the tea plant camellia sinensis},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The distribution of common-variant effect sizes.
<em>NG</em>, <em>53</em>(8), 1243–1249. (<a
href="https://doi.org/10.1038/s41588-021-00901-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic effect-size distribution of a disease describes the number of risk variants, the range of their effect sizes and sample sizes that will be required to discover them. Accurate estimation has been a challenge. Here I propose Fourier Mixture Regression (FMR), validating that it accurately estimates real and simulated effect-size distributions. Applied to summary statistics for ten diseases (average $$N_{\textrm{eff}} = 169,000$$ ), FMR estimates that 100,000–1,000,000 cases will be required for genome-wide significant SNPs to explain 50% of SNP heritability. In such large studies, genome-wide significance becomes increasingly conservative, and less stringent thresholds achieve high true positive rates if confounding is controlled. Across traits, polygenicity varies, but the range of their effect sizes is similar. Compared with effect sizes in the top 10% of heritability, including most discovered thus far, those in the bottom 10–50% are orders of magnitude smaller and more numerous, spanning a large fraction of the genome. Fourier Mixture Regression (FMR) is a method for estimating common-variant effect-size distributions. Applying FMR to summary statistics for complex traits from the UK Biobank shows that heritability is spread across a wide range of effect sizes.},
  archive      = {J_NG},
  author       = {O’Connor, Luke J.},
  doi          = {10.1038/s41588-021-00901-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1243-1249},
  shortjournal = {Nat. Genet.},
  title        = {The distribution of common-variant effect sizes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Variably methylated retrotransposons are refractory to a
range of environmental perturbations. <em>NG</em>, <em>53</em>(8),
1233–1242. (<a
href="https://doi.org/10.1038/s41588-021-00898-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The agouti viable yellow (Avy) allele is an insertional mutation in the mouse genome caused by a variably methylated intracisternal A particle (VM-IAP) retrotransposon. Avy expressivity is sensitive to a range of early-life chemical exposures and nutritional interventions, suggesting that environmental perturbations can have long-lasting effects on the methylome. However, the extent to which VM-IAP elements are environmentally labile with phenotypic implications is unknown. Using a recently identified repertoire of VM-IAPs, we assessed the epigenetic effects of different environmental contexts. A longitudinal aging analysis indicated that VM-IAPs are stable across the murine lifespan, with only small increases in DNA methylation detected for a subset of loci. No significant effects were observed after maternal exposure to the endocrine disruptor bisphenol A, an obesogenic diet or methyl donor supplementation. A genetic mouse model of abnormal folate metabolism exhibited shifted VM-IAP methylation levels and altered VM-IAP-associated gene expression, yet these effects are likely largely driven by differential targeting by polymorphic KRAB zinc finger proteins. We conclude that epigenetic variability at retrotransposons is not predictive of environmental susceptibility. Variably methylated intracisternal A particle (VM-IAP) retrotransposons are stable across the murine lifespan. VM-IAP retrotransposons are unaffected by maternal exposure to the endocrine disruptor bisphenol A, an obesogenic diet or methyl donor supplementation.},
  archive      = {J_NG},
  author       = {Bertozzi, Tessa M. and Becker, Jessica L. and Blake, Georgina E. T. and Bansal, Amita and Nguyen, Duy K. and Fernandez-Twinn, Denise S. and Ozanne, Susan E. and Bartolomei, Marisa S. and Simmons, Rebecca A. and Watson, Erica D. and Ferguson-Smith, Anne C.},
  doi          = {10.1038/s41588-021-00898-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1233-1242},
  shortjournal = {Nat. Genet.},
  title        = {Variably methylated retrotransposons are refractory to a range of environmental perturbations},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Simultaneous disruption of PRC2 and enhancer function
underlies histone h3.3-K27M oncogenic activity in human hindbrain neural
stem cells. <em>NG</em>, <em>53</em>(8), 1221–1232. (<a
href="https://doi.org/10.1038/s41588-021-00897-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Driver mutations in genes encoding histone H3 proteins resulting in p.Lys27Met substitutions (H3-K27M) are frequent in pediatric midline brain tumors. However, the precise mechanisms by which H3-K27M causes tumor initiation remain unclear. Here, we use human hindbrain neural stem cells to model the consequences of H3.3-K27M on the epigenomic landscape in a relevant developmental context. Genome-wide mapping of epitope-tagged histone H3.3 revealed that both the wild type and the K27M mutant incorporate abundantly at pre-existing active enhancers and promoters, and to a lesser extent at Polycomb repressive complex 2 (PRC2)-bound regions. At active enhancers, H3.3-K27M leads to focal H3K27ac loss, decreased chromatin accessibility and reduced transcriptional expression of nearby neurodevelopmental genes. In addition, H3.3-K27M deposition at a subset of PRC2 target genes leads to increased PRC2 and PRC1 binding and augmented transcriptional repression that can be partially reversed by PRC2 inhibitors. Our work suggests that, rather than imposing de novo transcriptional circuits, H3.3-K27M drives tumorigenesis by locking initiating cells in their pre-existing, immature epigenomic state, via disruption of PRC2 and enhancer functions. The oncohistone H3.3-K27M decreases chromatin accessibility and H3K27ac at some active enhancers and downregulates nearby neurodevelopmental genes, while increasing transcriptional repression of a subset of PRC2-bound neurodevelopment genes.},
  archive      = {J_NG},
  author       = {Brien, Gerard L. and Bressan, Raul Bardini and Monger, Craig and Gannon, Dáire and Lagan, Eimear and Doherty, Anthony M. and Healy, Evan and Neikes, Hannah and Fitzpatrick, Darren J. and Deevy, Orla and Grant, Vivien and Marqués-Torrejón, Maria-Angeles and Alfazema, Neza and Pollard, Steven M. and Bracken, Adrian P.},
  doi          = {10.1038/s41588-021-00897-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1221-1232},
  shortjournal = {Nat. Genet.},
  title        = {Simultaneous disruption of PRC2 and enhancer function underlies histone h3.3-K27M oncogenic activity in human hindbrain neural stem cells},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). P53 convergently activates dux/DUX4 in embryonic stem cells
and in facioscapulohumeral muscular dystrophy cell models. <em>NG</em>,
<em>53</em>(8), 1207–1220. (<a
href="https://doi.org/10.1038/s41588-021-00893-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In mammalian embryos, proper zygotic genome activation (ZGA) underlies totipotent development. Double homeobox (DUX)-family factors participate in ZGA, and mouse Dux is required for forming cultured two-cell (2C)-like cells. Remarkably, in mouse embryonic stem cells, Dux is activated by the tumor suppressor p53, and Dux expression promotes differentiation into expanded-fate cell types. Long-read sequencing and assembly of the mouse Dux locus reveals its complex chromatin regulation including putative positive and negative feedback loops. We show that the p53–DUX/DUX4 regulatory axis is conserved in humans. Furthermore, we demonstrate that cells derived from patients with facioscapulohumeral muscular dystrophy (FSHD) activate human DUX4 during p53 signaling via a p53-binding site in a primate-specific subtelomeric long terminal repeat (LTR)10C element. In summary, our work shows that p53 activation convergently evolved to couple p53 to Dux/DUX4 activation in embryonic stem cells, embryos and cells from patients with FSHD, potentially uniting the developmental and disease regulation of DUX-family factors and identifying evidence-based therapeutic opportunities for FSHD. p53 activates Dux in mouse embryos and embryonic stem cells, as well as DUX4 in human facioscapulohumeral muscular dystrophy cell models.},
  archive      = {J_NG},
  author       = {Grow, Edward J. and Weaver, Bradley D. and Smith, Christina M. and Guo, Jingtao and Stein, Paula and Shadle, Sean C. and Hendrickson, Peter G. and Johnson, Nicholas E. and Butterfield, Russell J. and Menafra, Roberta and Kloet, Susan L. and van der Maarel, Silvère M. and Williams, Carmen J. and Cairns, Bradley R.},
  doi          = {10.1038/s41588-021-00893-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1207-1220},
  shortjournal = {Nat. Genet.},
  title        = {P53 convergently activates Dux/DUX4 in embryonic stem cells and in facioscapulohumeral muscular dystrophy cell models},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-scale screens identify factors regulating tumor cell
responses to natural killer cells. <em>NG</em>, <em>53</em>(8),
1196–1206. (<a
href="https://doi.org/10.1038/s41588-021-00889-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To systematically define molecular features in human tumor cells that determine their degree of sensitivity to human allogeneic natural killer (NK) cells, we quantified the NK cell responsiveness of hundreds of molecularly annotated ‘DNA-barcoded’ solid tumor cell lines in multiplexed format and applied genome-scale CRISPR-based gene-editing screens in several solid tumor cell lines, to functionally interrogate which genes in tumor cells regulate the response to NK cells. In these orthogonal studies, NK cell–sensitive tumor cells tend to exhibit ‘mesenchymal-like’ transcriptional programs; high transcriptional signature for chromatin remodeling complexes; high levels of B7-H6 (NCR3LG1); and low levels of HLA-E/antigen presentation genes. Importantly, transcriptional signatures of NK cell–sensitive tumor cells correlate with immune checkpoint inhibitor (ICI) resistance in clinical samples. This study provides a comprehensive map of mechanisms regulating tumor cell responses to NK cells, with implications for future biomarker-driven applications of NK cell immunotherapies. The use of natural killer (NK) cells in immunotherapy as an alternative to allogeneic T cells is gaining ground. Here, two genome-scale high-throughput platforms are used to identify genes that modulate the sensitivity of multiple solid tumor cell lines to NK-mediated killing.},
  archive      = {J_NG},
  author       = {Sheffer, Michal and Lowry, Emily and Beelen, Nicky and Borah, Minasri and Amara, Suha Naffar-Abu and Mader, Chris C. and Roth, Jennifer A. and Tsherniak, Aviad and Freeman, Samuel S. and Dashevsky, Olga and Gandolfi, Sara and Bender, Samantha and Bryan, Jordan G. and Zhu, Cong and Wang, Li and Tariq, Ifrah and Kamath, Govinda M. and Simoes, Ricardo De Matos and Dhimolea, Eugen and Yu, Channing and Hu, Yiguo and Dufva, Olli and Giannakis, Marios and Syrgkanis, Vasilis and Fraenkel, Ernest and Golub, Todd and Romee, Rizwan and Mustjoki, Satu and Culhane, Aedin C. and Wieten, Lotte and Mitsiades, Constantine S.},
  doi          = {10.1038/s41588-021-00889-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1196-1206},
  shortjournal = {Nat. Genet.},
  title        = {Genome-scale screens identify factors regulating tumor cell responses to natural killer cells},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reconstructing single-cell karyotype alterations in
colorectal cancer identifies punctuated and gradual diversification
patterns. <em>NG</em>, <em>53</em>(8), 1187–1195. (<a
href="https://doi.org/10.1038/s41588-021-00891-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Central to tumor evolution is the generation of genetic diversity. However, the extent and patterns by which de novo karyotype alterations emerge and propagate within human tumors are not well understood, especially at single-cell resolution. Here, we present 3D Live-Seq—a protocol that integrates live-cell imaging of tumor organoid outgrowth and whole-genome sequencing of each imaged cell to reconstruct evolving tumor cell karyotypes across consecutive cell generations. Using patient-derived colorectal cancer organoids and fresh tumor biopsies, we demonstrate that karyotype alterations of varying complexity are prevalent and can arise within a few cell generations. Sub-chromosomal acentric fragments were prone to replication and collective missegregation across consecutive cell divisions. In contrast, gross genome-wide karyotype alterations were generated in a single erroneous cell division, providing support that aneuploid tumor genomes can evolve via punctuated evolution. Mapping the temporal dynamics and patterns of karyotype diversification in cancer enables reconstructions of evolutionary paths to malignant fitness. Analysis of live-cell imaging and single-cell genome sequencing data of colorectal cancer organoids identifies temporal dynamics of sub-chromosomal copy-number amplifications.},
  archive      = {J_NG},
  author       = {Bollen, Yannik and Stelloo, Ellen and van Leenen, Petra and van den Bos, Myrna and Ponsioen, Bas and Lu, Bingxin and van Roosmalen, Markus J. and Bolhaqueiro, Ana C. F. and Kimberley, Christopher and Mossner, Maximilian and Cross, William C. H. and Besselink, Nicolle J. M. and van der Roest, Bastiaan and Boymans, Sander and Oost, Koen C. and de Vries, Sippe G. and Rehmann, Holger and Cuppen, Edwin and Lens, Susanne M. A. and Kops, Geert J. P. L. and Kloosterman, Wigard P. and Terstappen, Leon W. M. M. and Barnes, Chris P. and Sottoriva, Andrea and Graham, Trevor A. and Snippert, Hugo J. G.},
  doi          = {10.1038/s41588-021-00891-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1187-1195},
  shortjournal = {Nat. Genet.},
  title        = {Reconstructing single-cell karyotype alterations in colorectal cancer identifies punctuated and gradual diversification patterns},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Activation of γ-globin gene expression by GATA1 and NF-y in
hereditary persistence of fetal hemoglobin. <em>NG</em>, <em>53</em>(8),
1177–1186. (<a
href="https://doi.org/10.1038/s41588-021-00904-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hereditary persistence of fetal hemoglobin (HPFH) ameliorates β-hemoglobinopathies by inhibiting the developmental switch from γ-globin (HBG1/HBG2) to β-globin (HBB) gene expression. Some forms of HPFH are associated with γ-globin promoter variants that either disrupt binding motifs for transcriptional repressors or create new motifs for transcriptional activators. How these variants sustain γ-globin gene expression postnatally remains undefined. We mapped γ-globin promoter sequences functionally in erythroid cells harboring different HPFH variants. Those that disrupt a BCL11A repressor binding element induce γ-globin expression by facilitating the recruitment of nuclear transcription factor Y (NF-Y) to a nearby proximal CCAAT box and GATA1 to an upstream motif. The proximal CCAAT element becomes dispensable for HPFH variants that generate new binding motifs for activators NF-Y or KLF1, but GATA1 recruitment remains essential. Our findings define distinct mechanisms through which transcription factors and their cis-regulatory elements activate γ-globin expression in different forms of HPFH, some of which are being recreated by therapeutic genome editing. Introduction of hereditary persistence of fetal hemoglobin variants into the γ-globin promoter by using CRISPR mutagenesis and editing provides insights into transcription factor interplay, with implications for gene therapies targeting this element.},
  archive      = {J_NG},
  author       = {Doerfler, Phillip A. and Feng, Ruopeng and Li, Yichao and Palmer, Lance E. and Porter, Shaina N. and Bell, Henry W. and Crossley, Merlin and Pruett-Miller, Shondra M. and Cheng, Yong and Weiss, Mitchell J.},
  doi          = {10.1038/s41588-021-00904-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1177-1186},
  shortjournal = {Nat. Genet.},
  title        = {Activation of γ-globin gene expression by GATA1 and NF-Y in hereditary persistence of fetal hemoglobin},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Direct characterization of cis-regulatory elements and
functional dissection of complex genetic associations using
HCR–FlowFISH. <em>NG</em>, <em>53</em>(8), 1166–1176. (<a
href="https://doi.org/10.1038/s41588-021-00900-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Effective interpretation of genome function and genetic variation requires a shift from epigenetic mapping of cis-regulatory elements (CREs) to characterization of endogenous function. We developed hybridization chain reaction fluorescence in situ hybridization coupled with flow cytometry (HCR–FlowFISH), a broadly applicable approach to characterize CRISPR-perturbed CREs via accurate quantification of native transcripts, alongside CRISPR activity screen analysis (CASA), a hierarchical Bayesian model to quantify CRE activity. Across &amp;gt;325,000 perturbations, we provide evidence that CREs can regulate multiple genes, skip over the nearest gene and display activating and/or silencing effects. At the cholesterol-level-associated FADS locus, we combine endogenous screens with reporter assays to exhaustively characterize multiple genome-wide association signals, functionally nominate causal variants and, importantly, identify their target genes. HCR–FlowFISH is a new approach to characterize CRISPR-perturbed cis-regulatory elements (CREs) via accurate quantification of native transcripts, alongside CRISPR activity screen analysis (CASA), a hierarchical Bayesian model to quantify CRE activity.},
  archive      = {J_NG},
  author       = {Reilly, Steven K. and Gosai, Sager J. and Gutierrez, Alan and Mackay-Smith, Ava and Ulirsch, Jacob C. and Kanai, Masahiro and Mouri, Kousuke and Berenzy, Daniel and Kales, Susan and Butler, Gina M. and Gladden-Young, Adrianne and Bhuiyan, Redwan M. and Stitzel, Michael L. and Finucane, Hilary K. and Sabeti, Pardis C. and Tewhey, Ryan},
  doi          = {10.1038/s41588-021-00900-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1166-1176},
  shortjournal = {Nat. Genet.},
  title        = {Direct characterization of cis-regulatory elements and functional dissection of complex genetic associations using HCR–FlowFISH},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genetic drivers of m6A methylation in human brain, lung,
heart and muscle. <em>NG</em>, <em>53</em>(8), 1156–1165. (<a
href="https://doi.org/10.1038/s41588-021-00890-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The most prevalent post-transcriptional mRNA modification, N6-methyladenosine (m6A), plays diverse RNA-regulatory roles, but its genetic control in human tissues remains uncharted. Here we report 129 transcriptome-wide m6A profiles, covering 91 individuals and 4 tissues (brain, lung, muscle and heart) from GTEx/eGTEx. We integrate these with interindividual genetic and expression variation, revealing 8,843 tissue-specific and 469 tissue-shared m6A quantitative trait loci (QTLs), which are modestly enriched in, but mostly orthogonal to, expression QTLs. We integrate m6A QTLs with disease genetics, identifying 184 GWAS-colocalized m6A QTL, including brain m6A QTLs underlying neuroticism, depression, schizophrenia and anxiety; lung m6A QTLs underlying expiratory flow and asthma; and muscle/heart m6A QTLs underlying coronary artery disease. Last, we predict novel m6A regulators that show preferential binding in m6A QTLs, protein interactions with known m6A regulators and expression correlation with the m6A levels of their targets. Our results provide important insights and resources for understanding both cis and trans regulation of epitranscriptomic modifications, their interindividual variation and their roles in human disease. Analysis of 129 N6-methyladenosine (m6A) profiles across 4 tissues (brain, lung, muscle and heart) identifies 8,843 tissue-specific and 469 tissue-shared m6A quantitative trait loci (QTLs). Of these, 184 m6A QTLs colocalize with GWAS signals.},
  archive      = {J_NG},
  author       = {Xiong, Xushen and Hou, Lei and Park, Yongjin P. and Molinie, Benoit and Gregory, Richard I. and Kellis, Manolis},
  doi          = {10.1038/s41588-021-00890-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1156-1165},
  shortjournal = {Nat. Genet.},
  title        = {Genetic drivers of m6A methylation in human brain, lung, heart and muscle},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-nucleus chromatin accessibility and transcriptomic
characterization of alzheimer’s disease. <em>NG</em>, <em>53</em>(8),
1143–1155. (<a
href="https://doi.org/10.1038/s41588-021-00894-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The gene-regulatory landscape of the brain is highly dynamic in health and disease, coordinating a menagerie of biological processes across distinct cell types. Here, we present a multi-omic single-nucleus study of 191,890 nuclei in late-stage Alzheimer’s disease (AD), accessible through our web portal, profiling chromatin accessibility and gene expression in the same biological samples and uncovering vast cellular heterogeneity. We identified cell-type-specific, disease-associated candidate cis-regulatory elements and their candidate target genes, including an oligodendrocyte-associated regulatory module containing links to APOE and CLU. We describe cis-regulatory relationships in specific cell types at a subset of AD risk loci defined by genome-wide association studies, demonstrating the utility of this multi-omic single-nucleus approach. Trajectory analysis of glial populations identified disease-relevant transcription factors, such as SREBF1, and their regulatory targets. Finally, we introduce single-nucleus consensus weighted gene coexpression analysis, a coexpression network analysis strategy robust to sparse single-cell data, and perform a systems-level analysis of the AD transcriptome. An integrative analysis of single-nucleus assay for transposase-accessible chromatin with sequencing and RNA sequencing in normal and Alzheimer’s disease brain tissue identifies cell-type-specific cis-regulatory elements and candidate target genes at disease-associated loci.},
  archive      = {J_NG},
  author       = {Morabito, Samuel and Miyoshi, Emily and Michael, Neethu and Shahin, Saba and Martini, Alessandra Cadete and Head, Elizabeth and Silva, Justine and Leavy, Kelsey and Perez-Rosendahl, Mari and Swarup, Vivek},
  doi          = {10.1038/s41588-021-00894-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1143-1155},
  shortjournal = {Nat. Genet.},
  title        = {Single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer’s disease},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Distinction between the effects of parental and fetal
genomes on fetal growth. <em>NG</em>, <em>53</em>(8), 1135–1142. (<a
href="https://doi.org/10.1038/s41588-021-00896-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Birth weight is a common measure of fetal growth that is associated with a range of health outcomes. It is directly affected by the fetal genome and indirectly by the maternal genome. We performed genome-wide association studies on birth weight in the genomes of the child and parents and further analyzed birth length and ponderal index, yielding a total of 243 fetal growth variants. We clustered those variants based on the effects of transmitted and nontransmitted alleles on birth weight. Out of 141 clustered variants, 22 were consistent with parent-of-origin-specific effects. We further used haplotype-specific polygenic risk scores to directly test the relationship between adult traits and birth weight. Our results indicate that the maternal genome contributes to increased birth weight through blood-glucose-raising alleles while blood-pressure-raising alleles reduce birth weight largely through the fetal genome. Genome-wide association analyses using parental and offspring genotypes provide insights into fetal and maternal genetic effects on fetal growth. The results show that maternal and fetal genomes influence birth weight through distinct mechanisms.},
  archive      = {J_NG},
  author       = {Juliusdottir, Thorhildur and Steinthorsdottir, Valgerdur and Stefansdottir, Lilja and Sveinbjornsson, Gardar and Ivarsdottir, Erna V. and Thorolfsdottir, Rosa B. and Sigurdsson, Jon K. and Tragante, Vinicius and Hjorleifsson, Kristjan E. and Helgadottir, Anna and Frigge, Michael L. and Thorgeirsson, Gudmundur and Benediktsson, Rafn and Sigurdsson, Emil L. and Arnar, David O. and Steingrimsdottir, Thora and Jonsdottir, Ingileif and Holm, Hilma and Gudbjartsson, Daniel F. and Thorleifsson, Gudmar and Thorsteinsdottir, Unnur and Stefansson, Kari},
  doi          = {10.1038/s41588-021-00896-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1135-1142},
  shortjournal = {Nat. Genet.},
  title        = {Distinction between the effects of parental and fetal genomes on fetal growth},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Recent ultra-rare inherited variants implicate new autism
candidate risk genes. <em>NG</em>, <em>53</em>(8), 1125–1134. (<a
href="https://doi.org/10.1038/s41588-021-00899-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autism is a highly heritable complex disorder in which de novo mutation (DNM) variation contributes significantly to risk. Using whole-genome sequencing data from 3,474 families, we investigate another source of large-effect risk variation, ultra-rare variants. We report and replicate a transmission disequilibrium of private, likely gene-disruptive (LGD) variants in probands but find that 95% of this burden resides outside of known DNM-enriched genes. This variant class more strongly affects multiplex family probands and supports a multi-hit model for autism. Candidate genes with private LGD variants preferentially transmitted to probands converge on the E3 ubiquitin–protein ligase complex, intracellular transport and Erb signaling protein networks. We estimate that these variants are approximately 2.5 generations old and significantly younger than other variants of similar type and frequency in siblings. Overall, private LGD variants are under strong purifying selection and appear to act on a distinct set of genes not yet associated with autism. Analysis of whole-genome sequence data from 3,474 families finds an excess of private, likely gene-disrupting variants in individuals with autism. These variants are under purifying selection and suggest candidate genes not previously associated with autism.},
  archive      = {J_NG},
  author       = {Wilfert, Amy B. and Turner, Tychele N. and Murali, Shwetha C. and Hsieh, PingHsun and Sulovari, Arvis and Wang, Tianyun and Coe, Bradley P. and Guo, Hui and Hoekzema, Kendra and Bakken, Trygve E. and Winterkorn, Lara H. and Evani, Uday S. and Byrska-Bishop, Marta and Earl, Rachel K. and Bernier, Raphael A. and Zody, Michael C. and Eichler, Evan E.},
  doi          = {10.1038/s41588-021-00899-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1125-1134},
  shortjournal = {Nat. Genet.},
  title        = {Recent ultra-rare inherited variants implicate new autism candidate risk genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Metastable epialleles are stable in their instability.
<em>NG</em>, <em>53</em>(8), 1121–1123. (<a
href="https://doi.org/10.1038/s41588-021-00907-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metastable epialleles refer to loci with variable methylation states among individuals without underlying genetic differences. Although these loci have generally been assumed to be vulnerable to environmental influence, a new study reports their remarkable metastable epigenetic robustness toward a range of physiological, chemical and dietary disruptions in mammals.},
  archive      = {J_NG},
  author       = {Bourc’his, Deborah},
  doi          = {10.1038/s41588-021-00907-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1121-1123},
  shortjournal = {Nat. Genet.},
  title        = {Metastable epialleles are stable in their instability},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Shedding light on the genetics of fetal growth. <em>NG</em>,
<em>53</em>(8), 1120–1121. (<a
href="https://doi.org/10.1038/s41588-021-00902-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have identified genetic variants in maternal and fetal genomes associated with early-life growth traits but have been limited by the paucity of large-scale family-based cohorts that would enable the resolution of informative transmissions between parents and their offspring. A new study uses extensive pedigree data from the Icelandic population to identify genetic effects on birth weight that differ according to parental origin and to demarcate distinct contributions from the maternal intrauterine environment and offspring genetics on fetal growth.},
  archive      = {J_NG},
  author       = {Evans, David M. and Freathy, Rachel M.},
  doi          = {10.1038/s41588-021-00902-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1120-1121},
  shortjournal = {Nat. Genet.},
  title        = {Shedding light on the genetics of fetal growth},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Cultural and genetic interplay. <em>NG</em>,
<em>53</em>(8), 1119. (<a
href="https://doi.org/10.1038/s41588-021-00915-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00915-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1119},
  shortjournal = {Nat. Genet.},
  title        = {Cultural and genetic interplay},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Coronavirus footprints. <em>NG</em>, <em>53</em>(8), 1119.
(<a href="https://doi.org/10.1038/s41588-021-00916-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00916-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1119},
  shortjournal = {Nat. Genet.},
  title        = {Coronavirus footprints},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Hybrid potato genetics. <em>NG</em>, <em>53</em>(8), 1119.
(<a href="https://doi.org/10.1038/s41588-021-00917-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00917-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1119},
  shortjournal = {Nat. Genet.},
  title        = {Hybrid potato genetics},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Sickle-cell anemia gene therapy. <em>NG</em>,
<em>53</em>(8), 1119. (<a
href="https://doi.org/10.1038/s41588-021-00918-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00918-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1119},
  shortjournal = {Nat. Genet.},
  title        = {Sickle-cell anemia gene therapy},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Single-cell spatial sequencing. <em>NG</em>,
<em>53</em>(8), 1119. (<a
href="https://doi.org/10.1038/s41588-021-00919-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alam, Ornob},
  doi          = {10.1038/s41588-021-00919-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1119},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell spatial sequencing},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Stephen t. Warren 1953–2021. <em>NG</em>, <em>53</em>(8),
1117–1118. (<a
href="https://doi.org/10.1038/s41588-021-00908-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stephen T. Warren was a key contributor to the 1991 discovery of an unstable trinucleotide repeat that expands in families and causes loss of function in fragile X syndrome.},
  archive      = {J_NG},
  author       = {Nelson, David L.},
  doi          = {10.1038/s41588-021-00908-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1117-1118},
  shortjournal = {Nat. Genet.},
  title        = {Stephen t. warren 1953–2021},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A call for direct sequencing of full-length RNAs to identify
all modifications. <em>NG</em>, <em>53</em>(8), 1113–1116. (<a
href="https://doi.org/10.1038/s41588-021-00903-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For most organisms, DNA sequences are available, but the complete RNA sequences are not. Here, we call for technologies to sequence full-length RNAs with all their modifications.},
  archive      = {J_NG},
  author       = {Alfonzo, Juan D. and Brown, Jessica A. and Byers, Peter H. and Cheung, Vivian G. and Maraia, Richard J. and Ross, Robert L.},
  doi          = {10.1038/s41588-021-00903-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1113-1116},
  shortjournal = {Nat. Genet.},
  title        = {A call for direct sequencing of full-length RNAs to identify all modifications},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Rapid genotype imputation from sequence with reference
panels. <em>NG</em>, <em>53</em>(7), 1104–1111. (<a
href="https://doi.org/10.1038/s41588-021-00877-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inexpensive genotyping methods are essential to modern genomics. Here we present QUILT, which performs diploid genotype imputation using low-coverage whole-genome sequence data. QUILT employs Gibbs sampling to partition reads into maternal and paternal sets, facilitating rapid haploid imputation using large reference panels. We show this partitioning to be accurate over many megabases, enabling highly accurate imputation close to theoretical limits and outperforming existing methods. Moreover, QUILT can impute accurately using diverse technologies, including long reads from Oxford Nanopore Technologies, and a new form of low-cost barcoded Illumina sequencing called haplotagging, with the latter showing improved accuracy at low coverages. Relative to DNA genotyping microarrays, QUILT offers improved accuracy at reduced cost, particularly for diverse populations that are traditionally underserved in modern genomic analyses, with accuracy nearly doubling at rare SNPs. Finally, QUILT can accurately impute (four-digit) human leukocyte antigen types, the first such method from low-coverage sequence data. QUILT is a method for rapid genotype imputation and phasing from low-coverage whole-genome sequence data using a large haplotype reference panel. QUILT enables highly accurate imputation across a range of coverages and data types.},
  archive      = {J_NG},
  author       = {Davies, Robert W. and Kucka, Marek and Su, Dingwen and Shi, Sinan and Flanagan, Maeve and Cunniff, Christopher M. and Chan, Yingguang Frank and Myers, Simon},
  doi          = {10.1038/s41588-021-00877-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1104-1111},
  shortjournal = {Nat. Genet.},
  title        = {Rapid genotype imputation from sequence with reference panels},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computationally efficient whole-genome regression for
quantitative and binary traits. <em>NG</em>, <em>53</em>(7), 1097–1103.
(<a href="https://doi.org/10.1038/s41588-021-00870-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association analysis of cohorts with thousands of phenotypes is computationally expensive, particularly when accounting for sample relatedness or population structure. Here we present a novel machine-learning method called REGENIE for fitting a whole-genome regression model for quantitative and binary phenotypes that is substantially faster than alternatives in multi-trait analyses while maintaining statistical efficiency. The method naturally accommodates parallel analysis of multiple phenotypes and requires only local segments of the genotype matrix to be loaded in memory, in contrast to existing alternatives, which must load genome-wide matrices into memory. This results in substantial savings in compute time and memory usage. We introduce a fast, approximate Firth logistic regression test for unbalanced case–control phenotypes. The method is ideally suited to take advantage of distributed computing frameworks. We demonstrate the accuracy and computational benefits of this approach using the UK Biobank dataset with up to 407,746 individuals. REGENIE is a whole-genome regression method based on ridge regression that enables highly parallelized analysis of quantitative and binary traits in biobank-scale data with reduced computational requirements.},
  archive      = {J_NG},
  author       = {Mbatchou, Joelle and Barnard, Leland and Backman, Joshua and Marcketta, Anthony and Kosmicki, Jack A. and Ziyatdinov, Andrey and Benner, Christian and O’Dushlaine, Colm and Barber, Mathew and Boutkov, Boris and Habegger, Lukas and Ferreira, Manuel and Baras, Aris and Reid, Jeffrey and Abecasis, Goncalo and Maxwell, Evan and Marchini, Jonathan},
  doi          = {10.1038/s41588-021-00870-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1097-1103},
  shortjournal = {Nat. Genet.},
  title        = {Computationally efficient whole-genome regression for quantitative and binary traits},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Radiotherapy is associated with a deletion signature that
contributes to poor outcomes in patients with cancer. <em>NG</em>,
<em>53</em>(7), 1088–1096. (<a
href="https://doi.org/10.1038/s41588-021-00874-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ionizing radiation causes DNA damage and is a mainstay for cancer treatment, but understanding of its genomic impact is limited. We analyzed mutational spectra following radiotherapy in 190 paired primary and recurrent gliomas from the Glioma Longitudinal Analysis Consortium and 3,693 post-treatment metastatic tumors from the Hartwig Medical Foundation. We identified radiotherapy-associated significant increases in the burden of small deletions (5–15 bp) and large deletions (20+ bp to chromosome-arm length). Small deletions were characterized by a larger span size, lacking breakpoint microhomology and were genomically more dispersed when compared to pre-existing deletions and deletions in non-irradiated tumors. Mutational signature analysis implicated classical non-homologous end-joining-mediated DNA damage repair and APOBEC mutagenesis following radiotherapy. A high radiation-associated deletion burden was associated with worse clinical outcomes, suggesting that effective repair of radiation-induced DNA damage is detrimental to patient survival. These results may be leveraged to predict sensitivity to radiation therapy in recurrent cancer. Radiotherapy induces small and large deletions as well as inversions across the genome in multiple cancer types. The genomic changes associated with radiotherapy correlate with poorer clinical outcomes.},
  archive      = {J_NG},
  author       = {Kocakavuk, Emre and Anderson, Kevin J. and Varn, Frederick S. and Johnson, Kevin C. and Amin, Samirkumar B. and Sulman, Erik. P. and Lolkema, Martijn P. and Barthel, Floris P. and Verhaak, Roel G. W.},
  doi          = {10.1038/s41588-021-00874-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1088-1096},
  shortjournal = {Nat. Genet.},
  title        = {Radiotherapy is associated with a deletion signature that contributes to poor outcomes in patients with cancer},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Three-dimensional folding dynamics of the xenopus tropicalis
genome. <em>NG</em>, <em>53</em>(7), 1075–1087. (<a
href="https://doi.org/10.1038/s41588-021-00878-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal interphase chromosomes are organized into topologically associating domains (TADs). How TADs are formed is not fully understood. Here, we combined high-throughput chromosome conformation capture and gene silencing to obtain insights into TAD dynamics in Xenopus tropicalis embryos. First, TAD establishment in X. tropicalis is similar to that in mice and flies and does not depend on zygotic genome transcriptional activation. This process is followed by further refinements in active and repressive chromatin compartments and the appearance of loops and stripes. Second, within TADs, higher self-interaction frequencies at one end of the boundary are associated with higher DNA occupancy of the architectural proteins CTCF and Rad21. Third, the chromatin remodeling factor ISWI is required for de novo TAD formation. Finally, TAD structures are variable in different tissues. Our work shows that X. tropicalis is a powerful model for chromosome architecture analysis and suggests that chromatin remodeling plays an essential role in de novo TAD establishment. Hi-C and single-molecule sequencing analysis provide an improved assembly of the Xenopus tropicalis genome and insights into three-dimensional genome dynamics throughout embryogenesis.},
  archive      = {J_NG},
  author       = {Niu, Longjian and Shen, Wei and Shi, Zhaoying and Tan, Yongjun and He, Na and Wan, Jing and Sun, Jialei and Zhang, Yuedong and Huang, Yingzhang and Wang, Wenjing and Fang, Chao and Li, Jiashuo and Zheng, Piaopiao and Cheung, Edwin and Chen, Yonglong and Li, Li and Hou, Chunhui},
  doi          = {10.1038/s41588-021-00878-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1075-1087},
  shortjournal = {Nat. Genet.},
  title        = {Three-dimensional folding dynamics of the xenopus tropicalis genome},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CTCF mediates dosage- and sequence-context-dependent
transcriptional insulation by forming local chromatin domains.
<em>NG</em>, <em>53</em>(7), 1064–1074. (<a
href="https://doi.org/10.1038/s41588-021-00863-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Insulators play a critical role in spatiotemporal gene regulation in animals. The evolutionarily conserved CCCTC-binding factor (CTCF) is required for insulator function in mammals, but not all of its binding sites act as insulators. Here we explore the sequence requirements of CTCF-mediated transcriptional insulation using a sensitive insulator reporter in mouse embryonic stem cells. We find that insulation potency depends on the number of CTCF-binding sites in tandem. Furthermore, CTCF-mediated insulation is dependent on upstream flanking sequences at its binding sites. CTCF-binding sites at topologically associating domain boundaries are more likely to function as insulators than those outside topologically associating domain boundaries, independently of binding strength. We demonstrate that insulators form local chromatin domain boundaries and weaken enhancer–promoter contacts. Taken together, our results provide genetic, molecular and structural evidence connecting chromatin topology to the action of insulators in the mammalian genome. The insulation potency of CTCF depends on the number of binding sites in tandem and on upstream flanking sequences. Insulators form local chromatin domain boundaries and weaken enhancer–promoter contacts.},
  archive      = {J_NG},
  author       = {Huang, Hui and Zhu, Quan and Jussila, Adam and Han, Yuanyuan and Bintu, Bogdan and Kern, Colin and Conte, Mattia and Zhang, Yanxiao and Bianco, Simona and Chiariello, Andrea M. and Yu, Miao and Hu, Rong and Tastemel, Melodi and Juric, Ivan and Hu, Ming and Nicodemi, Mario and Zhuang, Xiaowei and Ren, Bing},
  doi          = {10.1038/s41588-021-00863-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1064-1074},
  shortjournal = {Nat. Genet.},
  title        = {CTCF mediates dosage- and sequence-context-dependent transcriptional insulation by forming local chromatin domains},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The SWI/SNF chromatin remodeling complex helps resolve
r-loop-mediated transcription–replication conflicts. <em>NG</em>,
<em>53</em>(7), 1050–1063. (<a
href="https://doi.org/10.1038/s41588-021-00867-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {ATP-dependent chromatin remodelers are commonly mutated in human cancer. Mammalian SWI/SNF complexes comprise three conserved multisubunit chromatin remodelers (cBAF, ncBAF and PBAF) that share the BRG1 (also known as SMARCA4) subunit responsible for the main ATPase activity. BRG1 is the most frequently mutated Snf2-like ATPase in cancer. In the present study, we have investigated the role of SWI/SNF in genome instability, a hallmark of cancer cells, given its role in transcription, DNA replication and DNA-damage repair. We show that depletion of BRG1 increases R-loops and R-loop-dependent DNA breaks, as well as transcription–replication (T-R) conflicts. BRG1 colocalizes with R-loops and replication fork blocks, as determined by FANCD2 foci, with BRG1 depletion being epistatic to FANCD2 silencing. Our study, extended to other components of SWI/SNF, uncovers a key role of the SWI/SNF complex, in particular cBAF, in helping resolve R-loop-mediated T-R conflicts, thus, unveiling a new mechanism by which chromatin remodeling protects genome integrity. The SWI/SNF complex helps resolve R-loop-mediated transcription–replication conflicts, as depletion of SWI/SNF complex member BRG1 increases R-loops, R-loop-dependent DNA breaks and transcription–replication conflicts.},
  archive      = {J_NG},
  author       = {Bayona-Feliu, Aleix and Barroso, Sonia and Muñoz, Sergio and Aguilera, Andrés},
  doi          = {10.1038/s41588-021-00867-2},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1050-1063},
  shortjournal = {Nat. Genet.},
  title        = {The SWI/SNF chromatin remodeling complex helps resolve R-loop-mediated transcription–replication conflicts},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Orphan CpG islands amplify poised enhancer regulatory
activity and determine target gene responsiveness. <em>NG</em>,
<em>53</em>(7), 1036–1049. (<a
href="https://doi.org/10.1038/s41588-021-00888-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CpG islands (CGIs) represent a widespread feature of vertebrate genomes, being associated with ~70% of all gene promoters. CGIs control transcription initiation by conferring nearby promoters with unique chromatin properties. In addition, there are thousands of distal or orphan CGIs (oCGIs) whose functional relevance is barely known. Here we show that oCGIs are an essential component of poised enhancers that augment their long-range regulatory activity and control the responsiveness of their target genes. Using a knock-in strategy in mouse embryonic stem cells, we introduced poised enhancers with or without oCGIs within topologically associating domains harboring genes with different types of promoters. Analysis of the resulting cell lines revealed that oCGIs act as tethering elements that promote the physical and functional communication between poised enhancers and distally located genes, particularly those with large CGI clusters in their promoters. Therefore, by acting as genetic determinants of gene–enhancer compatibility, CGIs can contribute to gene expression control under both physiological and potentially pathological conditions. Genetic manipulation of poised enhancers (PEs) shows that orphan CpG islands promote physical and functional communication between PEs and distally located developmental genes.},
  archive      = {J_NG},
  author       = {Pachano, Tomas and Sánchez-Gaya, Víctor and Ealo, Thais and Mariner-Faulí, Maria and Bleckwehl, Tore and Asenjo, Helena G. and Respuela, Patricia and Cruz-Molina, Sara and Muñoz-San Martín, María and Haro, Endika and van IJcken, Wilfred F. J. and Landeira, David and Rada-Iglesias, Alvaro},
  doi          = {10.1038/s41588-021-00888-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1036-1049},
  shortjournal = {Nat. Genet.},
  title        = {Orphan CpG islands amplify poised enhancer regulatory activity and determine target gene responsiveness},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The HIF complex recruits the histone methyltransferase SET1B
to activate specific hypoxia-inducible genes. <em>NG</em>,
<em>53</em>(7), 1022–1035. (<a
href="https://doi.org/10.1038/s41588-021-00887-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypoxia-inducible transcription factors (HIFs) are fundamental to cellular adaptation to low oxygen levels, but it is unclear how they interact with chromatin and activate their target genes. Here, we use genome-wide mutagenesis to identify genes involved in HIF transcriptional activity, and define a requirement for the histone H3 lysine 4 (H3K4) methyltransferase SET1B. SET1B loss leads to a selective reduction in transcriptional activation of HIF target genes, resulting in impaired cell growth, angiogenesis and tumor establishment in SET1B-deficient xenografts. Mechanistically, we show that SET1B accumulates on chromatin in hypoxia, and is recruited to HIF target genes by the HIF complex. The selective induction of H3K4 trimethylation at HIF target loci is both HIF- and SET1B-dependent and, when impaired, correlates with decreased promoter acetylation and gene expression. Together, these findings show SET1B as a determinant of site-specific histone methylation and provide insight into how HIF target genes are differentially regulated. The histone H3K4 methyltransferase SET1B is recruited to a subset of hypoxia-inducible genes by the HIF complex. Loss of SET1B reduces HIF transcriptional activity in hypoxia and impairs tumor formation in xenograft models.},
  archive      = {J_NG},
  author       = {Ortmann, Brian M. and Burrows, Natalie and Lobb, Ian T. and Arnaiz, Esther and Wit, Niek and Bailey, Peter S. J. and Jordon, Louise H. and Lombardi, Olivia and Peñalver, Ana and McCaffrey, James and Seear, Rachel and Mole, David R. and Ratcliffe, Peter J. and Maxwell, Patrick H. and Nathan, James A.},
  doi          = {10.1038/s41588-021-00887-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1022-1035},
  shortjournal = {Nat. Genet.},
  title        = {The HIF complex recruits the histone methyltransferase SET1B to activate specific hypoxia-inducible genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Pathogenic SPTBN1 variants cause an autosomal dominant
neurodevelopmental syndrome. <em>NG</em>, <em>53</em>(7), 1006–1021. (<a
href="https://doi.org/10.1038/s41588-021-00886-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SPTBN1 encodes βII-spectrin, the ubiquitously expressed β-spectrin that forms micrometer-scale networks associated with plasma membranes. Mice deficient in neuronal βII-spectrin have defects in cortical organization, developmental delay and behavioral deficiencies. These phenotypes, while less severe, are observed in haploinsufficient animals, suggesting that individuals carrying heterozygous SPTBN1 variants may also show measurable compromise of neural development and function. Here we identify heterozygous SPTBN1 variants in 29 individuals with developmental, language and motor delays; mild to severe intellectual disability; autistic features; seizures; behavioral and movement abnormalities; hypotonia; and variable dysmorphic facial features. We show that these SPTBN1 variants lead to effects that affect βII-spectrin stability, disrupt binding to key molecular partners, and disturb cytoskeleton organization and dynamics. Our studies define SPTBN1 variants as the genetic basis of a neurodevelopmental syndrome, expand the set of spectrinopathies affecting the brain and underscore the critical role of βII-spectrin in the central nervous system. SPTBN1 mutations cause a neurodevelopmental syndrome characterized by intellectual disability, language and motor delays, autism, seizures and other features. The variants disrupt βII-spectrin function and disturb cytoskeletal organization and dynamics.},
  archive      = {J_NG},
  author       = {Cousin, Margot A. and Creighton, Blake A. and Breau, Keith A. and Spillmann, Rebecca C. and Torti, Erin and Dontu, Sruthi and Tripathi, Swarnendu and Ajit, Deepa and Edwards, Reginald J. and Afriyie, Simone and Bay, Julia C. and Harper, Kathryn M. and Beltran, Alvaro A. and Munoz, Lorena J. and Falcon Rodriguez, Liset and Stankewich, Michael C. and Person, Richard E. and Si, Yue and Normand, Elizabeth A. and Blevins, Amy and May, Alison S. and Bier, Louise and Aggarwal, Vimla and Mancini, Grazia M. S. and van Slegtenhorst, Marjon A. and Cremer, Kirsten and Becker, Jessica and Engels, Hartmut and Aretz, Stefan and MacKenzie, Jennifer J. and Brilstra, Eva and van Gassen, Koen L. I. and van Jaarsveld, Richard H. and Oegema, Renske and Parsons, Gretchen M. and Mark, Paul and Helbig, Ingo and McKeown, Sarah E. and Stratton, Robert and Cogne, Benjamin and Isidor, Bertrand and Cacheiro, Pilar and Smedley, Damian and Firth, Helen V. and Bierhals, Tatjana and Kloth, Katja and Weiss, Deike and Fairley, Cecilia and Shieh, Joseph T. and Kritzer, Amy and Jayakar, Parul and Kurtz-Nelson, Evangeline and Bernier, Raphael A. and Wang, Tianyun and Eichler, Evan E. and van de Laar, Ingrid M. B. H. and McConkie-Rosell, Allyn and McDonald, Marie T. and Kemppainen, Jennifer and Lanpher, Brendan C. and Schultz-Rogers, Laura E. and Gunderson, Lauren B. and Pichurin, Pavel N. and Yoon, Grace and Zech, Michael and Jech, Robert and Winkelmann, Juliane and Beltran, Adriana S. and Zimmermann, Michael T. and Temple, Brenda and Moy, Sheryl S. and Klee, Eric W. and Tan, Queenie K.-G. and Lorenzo, Damaris N.},
  doi          = {10.1038/s41588-021-00886-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1006-1021},
  shortjournal = {Nat. Genet.},
  title        = {Pathogenic SPTBN1 variants cause an autosomal dominant neurodevelopmental syndrome},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An atlas of alternative polyadenylation quantitative trait
loci contributing to complex trait and disease heritability.
<em>NG</em>, <em>53</em>(7), 994–1005. (<a
href="https://doi.org/10.1038/s41588-021-00864-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have identified thousands of noncoding variants associated with human traits and diseases. However, the functional interpretation of these variants is a major challenge. Here, we constructed a multi-tissue atlas of human 3′UTR alternative polyadenylation (APA) quantitative trait loci (3′aQTLs), containing approximately 0.4 million common genetic variants associated with the APA of target genes, identified in 46 tissues isolated from 467 individuals (Genotype-Tissue Expression Project). Mechanistically, 3′aQTLs can alter poly(A) motifs, RNA secondary structure and RNA-binding protein–binding sites, leading to thousands of APA changes. Our CRISPR-based experiments indicate that such 3′aQTLs can alter APA regulation. Furthermore, we demonstrate that mapping 3′aQTLs can identify APA regulators, such as La-related protein 4. Finally, 3′aQTLs are colocalized with approximately 16.1% of trait-associated variants and are largely distinct from other QTLs, such as expression QTLs. Together, our findings show that 3′aQTLs contribute substantially to the molecular mechanisms underlying human complex traits and diseases. A multi-tissue atlas of alternative polyadenylation (APA) quantitative trait loci (3′aQTLs) identifies approximately 0.4 million common genetic variants associated with the APA of target genes. Approximately 16% of trait-associated variants colocalize with 3′aQTLs.},
  archive      = {J_NG},
  author       = {Li, Lei and Huang, Kai-Lieh and Gao, Yipeng and Cui, Ya and Wang, Gao and Elrod, Nathan D. and Li, Yumei and Chen, Yiling Elaine and Ji, Ping and Peng, Fanglue and Russell, William K. and Wagner, Eric J. and Li, Wei},
  doi          = {10.1038/s41588-021-00864-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {994-1005},
  shortjournal = {Nat. Genet.},
  title        = {An atlas of alternative polyadenylation quantitative trait loci contributing to complex trait and disease heritability},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An atlas of mitochondrial DNA genotype–phenotype
associations in the UK biobank. <em>NG</em>, <em>53</em>(7), 982–993.
(<a href="https://doi.org/10.1038/s41588-021-00868-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mitochondrial DNA (mtDNA) variation in common diseases has been underexplored, partly due to a lack of genotype calling and quality-control procedures. Developing an at-scale workflow for mtDNA variant analyses, we show correlations between nuclear and mitochondrial genomic structures within subpopulations of Great Britain and establish a UK Biobank reference atlas of mtDNA–phenotype associations. A total of 260 mtDNA–phenotype associations were new (P &amp;lt; 1 × 10−5), including rs2853822 /m.8655 C&amp;gt;T (MT-ATP6) with type 2 diabetes, rs878966690 /m.13117 A&amp;gt;G (MT-ND5) with multiple sclerosis, 6 mtDNA associations with adult height, 24 mtDNA associations with 2 liver biomarkers and 16 mtDNA associations with parameters of renal function. Rare-variant gene-based tests implicated complex I genes modulating mean corpuscular volume and mean corpuscular hemoglobin. Seven traits had both rare and common mtDNA associations, where rare variants tended to have larger effects than common variants. Our work illustrates the value of studying mtDNA variants in common complex diseases and lays foundations for future large-scale mtDNA association studies. An analysis of the UK Biobank identifies 227 new associations between mitochondrial DNA (mtDNA) variants and phenotypes. mtDNA genetic architecture reflects regional UK nuclear genome ancestry.},
  archive      = {J_NG},
  author       = {Yonova-Doing, Ekaterina and Calabrese, Claudia and Gomez-Duran, Aurora and Schon, Katherine and Wei, Wei and Karthikeyan, Savita and Chinnery, Patrick F. and Howson, Joanna M. M.},
  doi          = {10.1038/s41588-021-00868-1},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {982-993},
  shortjournal = {Nat. Genet.},
  title        = {An atlas of mitochondrial DNA genotype–phenotype associations in the UK biobank},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A unified framework identifies new links between plasma
lipids and diseases from electronic medical records across large-scale
cohorts. <em>NG</em>, <em>53</em>(7), 972–981. (<a
href="https://doi.org/10.1038/s41588-021-00879-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plasma lipids are known heritable risk factors for cardiovascular disease, but increasing evidence also supports shared genetics with diseases of other organ systems. We devised a comprehensive three-phase framework to identify new lipid-associated genes and study the relationships among lipids, genotypes, gene expression and hundreds of complex human diseases from the Electronic Medical Records and Genomics (347 traits) and the UK Biobank (549 traits). Aside from 67 new lipid-associated genes with strong replication, we found evidence for pleiotropic SNPs/genes between lipids and diseases across the phenome. These include discordant pleiotropy in the HLA region between lipids and multiple sclerosis and putative causal paths between triglycerides and gout, among several others. Our findings give insights into the genetic basis of the relationship between plasma lipids and diseases on a phenome-wide scale and can provide context for future prevention and treatment strategies. An analytical framework based on transcriptome-wide association and electronic medical records provides insights into the relationship between plasma lipids and complex diseases on a phenome-wide scale.},
  archive      = {J_NG},
  author       = {Veturi, Yogasudha and Lucas, Anastasia and Bradford, Yuki and Hui, Daniel and Dudek, Scott and Theusch, Elizabeth and Verma, Anurag and Miller, Jason E. and Kullo, Iftikhar and Hakonarson, Hakon and Sleiman, Patrick and Schaid, Daniel and Stein, Charles M. and Edwards, Digna R. Velez and Feng, QiPing and Wei, Wei-Qi and Medina, Marisa W. and Krauss, Ronald M. and Hoffmann, Thomas J. and Risch, Neil and Voight, Benjamin F. and Rader, Daniel J. and Ritchie, Marylyn D.},
  doi          = {10.1038/s41588-021-00879-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {972-981},
  shortjournal = {Nat. Genet.},
  title        = {A unified framework identifies new links between plasma lipids and diseases from electronic medical records across large-scale cohorts},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fine-mapping, trans-ancestral and genomic analyses identify
causal variants, cells, genes and drug targets for type 1 diabetes.
<em>NG</em>, <em>53</em>(7), 962–971. (<a
href="https://doi.org/10.1038/s41588-021-00880-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report the largest and most diverse genetic study of type 1 diabetes (T1D) to date (61,427 participants), yielding 78 genome-wide-significant (P &amp;lt; 5 × 10−8) regions, including 36 that are new. We define credible sets of T1D-associated variants and show that they are enriched in immune-cell accessible chromatin, particularly CD4+ effector T cells. Using chromatin-accessibility profiling of CD4+ T cells from 115 individuals, we map chromatin-accessibility quantitative trait loci and identify five regions where T1D risk variants co-localize with chromatin-accessibility quantitative trait loci. We highlight rs72928038 in BACH2 as a candidate causal T1D variant leading to decreased enhancer accessibility and BACH2 expression in T cells. Finally, we prioritize potential drug targets by integrating genetic evidence, functional genomic maps and immune protein–protein interactions, identifying 12 genes implicated in T1D that have been targeted in clinical trials for autoimmune diseases. These findings provide an expanded genomic landscape for T1D. A large-scale genetic analysis of type 1 diabetes identifies new susceptibility variants, highlights potential regulatory mechanisms and provides genetic support for therapeutic targets for immune intervention.},
  archive      = {J_NG},
  author       = {Robertson, Catherine C. and Inshaw, Jamie R. J. and Onengut-Gumuscu, Suna and Chen, Wei-Min and Santa Cruz, David Flores and Yang, Hanzhi and Cutler, Antony J. and Crouch, Daniel J. M. and Farber, Emily and Bridges, S. Louis and Edberg, Jeffrey C. and Kimberly, Robert P. and Buckner, Jane H. and Deloukas, Panos and Divers, Jasmin and Dabelea, Dana and Lawrence, Jean M. and Marcovina, Santica and Shah, Amy S. and Greenbaum, Carla J. and Atkinson, Mark A. and Gregersen, Peter K. and Oksenberg, Jorge R. and Pociot, Flemming and Rewers, Marian J. and Steck, Andrea K. and Dunger, David B. and Wicker, Linda S. and Concannon, Patrick and Todd, John A. and Rich, Stephen S.},
  doi          = {10.1038/s41588-021-00880-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {962-971},
  shortjournal = {Nat. Genet.},
  title        = {Fine-mapping, trans-ancestral and genomic analyses identify causal variants, cells, genes and drug targets for type 1 diabetes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). PHYTOCHROME-INTERACTING FACTORs trigger environmentally
responsive chromatin dynamics in plants. <em>NG</em>, <em>53</em>(7),
955–961. (<a href="https://doi.org/10.1038/s41588-021-00882-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The interplay between light receptors and PHYTOCHROME-INTERACTING FACTORs (PIFs) serves as a regulatory hub that perceives and integrates environmental cues into transcriptional networks of plants1,2. Although occupancy of the histone variant H2A.Z and acetylation of histone H3 have emerged as regulators of environmentally responsive gene networks, how these epigenomic features interface with PIF activity is poorly understood3–7. By taking advantage of rapid and reversible light-mediated manipulation of PIF7 subnuclear localization and phosphorylation, we simultaneously assayed the DNA-binding properties of PIF7, as well as its impact on chromatin dynamics genome wide. We found that PIFs act rapidly to reshape the H2A.Z and H3K9ac epigenetic landscape in response to a change in light quality. Furthermore, we discovered that PIFs achieve H2A.Z removal through direct interaction with EIN6 ENHANCER (EEN), the Arabidopsis thaliana homolog of the chromatin remodeling complex subunit INO80 Subunit 6 (Ies6). Thus, we describe a PIF–INO80 regulatory module that is an intermediate step for allowing plants to change their growth trajectory in response to environmental changes. PHYTOCHROME-INTERACTING FACTORs reshape the H2A.Z epigenetic landscape in response to light quality changes in Arabidopsis thaliana, through an interaction with the INO80 chromatin remodeling complex.},
  archive      = {J_NG},
  author       = {Willige, Björn C. and Zander, Mark and Yoo, Chan Yul and Phan, Amy and Garza, Renee M. and Wanamaker, Shelly A. and He, Yupeng and Nery, Joseph R. and Chen, Huaming and Chen, Meng and Ecker, Joseph R. and Chory, Joanne},
  doi          = {10.1038/s41588-021-00882-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {955-961},
  shortjournal = {Nat. Genet.},
  title        = {PHYTOCHROME-INTERACTING FACTORs trigger environmentally responsive chromatin dynamics in plants},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Non-additive association analysis using proxy phenotypes
identifies novel cattle syndromes. <em>NG</em>, <em>53</em>(7), 949–954.
(<a href="https://doi.org/10.1038/s41588-021-00872-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian species carry ~100 loss-of-function variants per individual1,2, where ~1–5 of these impact essential genes and cause embryonic lethality or severe disease when homozygous3. The functions of the remainder are more difficult to resolve, although the assumption is that these variants impact fitness in less manifest ways. Here we report one of the largest sequence-resolution screens of cattle to date, targeting discovery and validation of non-additive effects in 130,725 animals. We highlight six novel recessive loci with impacts generally exceeding the largest-effect variants identified from additive genome-wide association studies, presenting analogs of human diseases and hitherto-unrecognized disorders. These loci present compelling missense (PLCD4, MTRF1 and DPF2), premature stop (MUS81) and splice-disrupting (GALNT2 and FGD4) mutations, together explaining substantial proportions of inbreeding depression. These results demonstrate that the frequency distribution of deleterious alleles segregating in selected species can afford sufficient power to directly map novel disorders, presenting selection opportunities to minimize the incidence of genetic disease. An association model that estimates dominance and additive effects applied to imputed whole-genome data from cattle allows for the mapping of recessive syndromes in the absence of disease classification by using proxy phenotypes such as body weight.},
  archive      = {J_NG},
  author       = {Reynolds, Edwardo G. M. and Neeley, Catherine and Lopdell, Thomas J. and Keehan, Michael and Dittmer, Keren and Harland, Chad S. and Couldrey, Christine and Johnson, Thomas J. J. and Tiplady, Kathryn and Worth, Gemma and Walker, Mark and Davis, Stephen R. and Sherlock, Richard G. and Carnie, Katie and Harris, Bevin L. and Charlier, Carole and Georges, Michel and Spelman, Richard J. and Garrick, Dorian J. and Littlejohn, Mathew D.},
  doi          = {10.1038/s41588-021-00872-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {949-954},
  shortjournal = {Nat. Genet.},
  title        = {Non-additive association analysis using proxy phenotypes identifies novel cattle syndromes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Advancing human genetics research and drug discovery through
exome sequencing of the UK biobank. <em>NG</em>, <em>53</em>(7),
942–948. (<a href="https://doi.org/10.1038/s41588-021-00885-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The UK Biobank Exome Sequencing Consortium (UKB-ESC) is a private–public partnership between the UK Biobank (UKB) and eight biopharmaceutical companies that will complete the sequencing of exomes for all ~500,000 UKB participants. Here, we describe the early results from ~200,000 UKB participants and the features of this project that enabled its success. The biopharmaceutical industry has increasingly used human genetics to improve success in drug discovery. Recognizing the need for large-scale human genetics data, as well as the unique value of the data access and contribution terms of the UKB, the UKB-ESC was formed. As a result, exome data from 200,643 UKB enrollees are now available. These data include ~10 million exonic variants—a rich resource of rare coding variation that is particularly valuable for drug discovery. The UKB-ESC precompetitive collaboration has further strengthened academic and industry ties and has provided teams with an opportunity to interact with and learn from the wider research community. The UK Biobank Exome Sequencing Consortium aims to sequence all the exomes of approximately 500,000 UK Biobank participants. This Perspective describes the results from approximately 200,000 exomes and discusses the lessons learned from this UK Biobank–biopharmaceutical company collaboration.},
  archive      = {J_NG},
  author       = {Szustakowski, Joseph D. and Balasubramanian, Suganthi and Kvikstad, Erika and Khalid, Shareef and Bronson, Paola G. and Sasson, Ariella and Wong, Emily and Liu, Daren and Wade Davis, J. and Haefliger, Carolina and Katrina Loomis, A. and Mikkilineni, Rajesh and Noh, Hyun Ji and Wadhawan, Samir and Bai, Xiaodong and Hawes, Alicia and Krasheninina, Olga and Ulloa, Ricardo and Lopez, Alex E. and Smith, Erin N. and Waring, Jeffrey F. and Whelan, Christopher D. and Tsai, Ellen A. and Overton, John D. and Salerno, William J. and Jacob, Howard and Szalma, Sandor and Runz, Heiko and Hinkle, Gregory and Nioi, Paul and Petrovski, Slavé and Miller, Melissa R. and Baras, Aris and Mitnaul, Lyndon J. and Reid, Jeffrey G.},
  doi          = {10.1038/s41588-021-00885-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {942-948},
  shortjournal = {Nat. Genet.},
  title        = {Advancing human genetics research and drug discovery through exome sequencing of the UK biobank},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). SWI/SNF complexes as determinants of r-loop metabolism.
<em>NG</em>, <em>53</em>(7), 940–941. (<a
href="https://doi.org/10.1038/s41588-021-00884-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {R-loops—nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA—are abundant in the genome and may impair progression of the replication fork, thus leading to accumulation of DNA damage and genome instability. A new study demonstrates that SWI/SNF chromatin-remodeling complexes are instrumental in resolving such R-loop-mediated transcription–replication conflicts, highlighting the importance of these chromatin remodelers in R-loop homeostasis and the maintenance of genome integrity.},
  archive      = {J_NG},
  author       = {Abakir, Abdulkadir and Ruzov, Alexey},
  doi          = {10.1038/s41588-021-00884-1},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {940-941},
  shortjournal = {Nat. Genet.},
  title        = {SWI/SNF complexes as determinants of R-loop metabolism},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Going home: The challenges and rewards of genetics research
in mexico. <em>NG</em>, <em>53</em>(7), 937–939. (<a
href="https://doi.org/10.1038/s41588-021-00876-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The decision to move back home was very appealing but still took substantial thought.},
  archive      = {J_NG},
  author       = {Robles-Espinoza, Carla Daniela},
  doi          = {10.1038/s41588-021-00876-1},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {937-939},
  shortjournal = {Nat. Genet.},
  title        = {Going home: The challenges and rewards of genetics research in mexico},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Distinct dynamics and functions of
H2AK119ub1 and H3K27me3 in mouse preimplantation embryos. <em>NG</em>,
<em>53</em>(6), 936. (<a
href="https://doi.org/10.1038/s41588-021-00871-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00871-6.},
  archive      = {J_NG},
  author       = {Chen, Zhiyuan and Djekidel, Mohamed Nadhir and Zhang, Yi},
  doi          = {10.1038/s41588-021-00871-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {936},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Distinct dynamics and functions of H2AK119ub1 and H3K27me3 in mouse preimplantation embryos},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Author correction: ArchR is a scalable software package for
integrative single-cell chromatin accessibility analysis. <em>NG</em>,
<em>53</em>(6), 935. (<a
href="https://doi.org/10.1038/s41588-021-00850-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00850-x.},
  archive      = {J_NG},
  author       = {Granja, Jeffrey M. and Corces, M. Ryan and Pierce, Sarah E. and Bagdatli, S. Tansu and Choudhry, Hani and Chang, Howard Y. and Greenleaf, William J.},
  doi          = {10.1038/s41588-021-00850-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {935},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: The NCI genomic data commons.
<em>NG</em>, <em>53</em>(6), 935. (<a
href="https://doi.org/10.1038/s41588-021-00883-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Heath, Allison P. and Ferretti, Vincent and Agrawal, Stuti and An, Maksim and Angelakos, James C. and Arya, Renuka and Bajari, Rosita and Baqar, Bilal and Barnowski, Justin H. B. and Burt, Jeffrey and Catton, Ann and Chan, Brandon F. and Chu, Fay and Cullion, Kim and Davidsen, Tanja and Do, Phuong-My and Dompierre, Christian and Ferguson, Martin L. and Fitzsimons, Michael S. and Ford, Michael and Fukuma, Miyuki and Gaheen, Sharon and Ganji, Gajanan L. and Garcia, Tzintzuni I. and George, Sameera S. and Gerhard, Daniela S. and Gerthoffert, Francois and Gomez, Fauzi and Han, Kang and Hernandez, Kyle M. and Issac, Biju and Jackson, Richard and Jensen, Mark A. and Joshi, Sid and Kadam, Ajinkya and Khurana, Aishmit and Kim, Kyle M. J. and Kraft, Victoria E. and Li, Shenglai and Lichtenberg, Tara M. and Lodato, Janice and Lolla, Laxmi and Martinov, Plamen and Mazzone, Jeffrey A. and Miller, Daniel P. and Miller, Ian and Miller, Joshua S. and Miyauchi, Koji and Murphy, Mark W. and Nullet, Thomas and Ogwara, Rowland O. and Ortuño, Francisco M. and Pedrosa, Jesús and Pham, Phuong L. and Popov, Maxim Y. and Porter, James J. and Powell, Raymond and Rademacher, Karl and Reid, Colin P. and Rich, Samantha and Rogel, Bessie and Sahni, Himanso and Savage, Jeremiah H. and Schmitt, Kyle A. and Simmons, Trevar J. and Sislow, Joseph and Spring, Jonathan and Stein, Lincoln and Sullivan, Sean and Tang, Yajing and Thiagarajan, Mathangi and Troyer, Heather D. and Wang, Chang and Wang, Zhining and West, Bedford L. and Wilmer, Alex and Wilson, Shane and Wu, Kaman and Wysocki, William P. and Xiang, Linda and Yamada, Joseph T. and Yang, Liming and Yu, Christine and Yung, Christina K. and Zenklusen, Jean Claude and Zhang, Junjun and Zhang, Zhenyu and Zhao, Yuanheng and Zubair, Ariz and Staudt, Louis M. and Grossman, Robert L.},
  doi          = {10.1038/s41588-021-00883-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {935},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The NCI genomic data commons},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Alternative splicing during mammalian organ development.
<em>NG</em>, <em>53</em>(6), 925–934. (<a
href="https://doi.org/10.1038/s41588-021-00851-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alternative splicing (AS) is pervasive in mammalian genomes, yet cross-species comparisons have been largely restricted to adult tissues and the functionality of most AS events remains unclear. We assessed AS patterns across pre- and postnatal development of seven organs in six mammals and a bird. Our analyses revealed that developmentally dynamic AS events, which are especially prevalent in the brain, are substantially more conserved than nondynamic ones. Cassette exons with increasing inclusion frequencies during development show the strongest signals of conserved and regulated AS. Newly emerged cassette exons are typically incorporated late in testis development, but those retained during evolution are predominantly brain specific. Our work suggests that an intricate interplay of programs controlling gene expression levels and AS is fundamental to organ development, especially for the brain and heart. In these regulatory networks, AS affords substantial functional diversification of genes through the generation of tissue- and time-specific isoforms from broadly expressed genes. Analysis of RNA-seq datasets from seven organs across seven species generates an alternative splicing (AS) atlas and shows that AS events provide functional gene diversification through generation of tissue- and time-specific transcript isoforms.},
  archive      = {J_NG},
  author       = {Mazin, Pavel V. and Khaitovich, Philipp and Cardoso-Moreira, Margarida and Kaessmann, Henrik},
  doi          = {10.1038/s41588-021-00851-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {925-934},
  shortjournal = {Nat. Genet.},
  title        = {Alternative splicing during mammalian organ development},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The genomic basis of geographic differentiation and fiber
improvement in cultivated cotton. <em>NG</em>, <em>53</em>(6), 916–924.
(<a href="https://doi.org/10.1038/s41588-021-00844-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale genomic surveys of crop germplasm are important for understanding the genetic architecture of favorable traits. The genomic basis of geographic differentiation and fiber improvement in cultivated cotton is poorly understood. Here, we analyzed 3,248 tetraploid cotton genomes and confirmed that the extensive chromosome inversions on chromosomes A06 and A08 underlies the geographic differentiation in cultivated Gossypium hirsutum. We further revealed that the haplotypic diversity originated from landraces, which might be essential for understanding adaptative evolution in cultivated cotton. Introgression and association analyses identified new fiber quality-related loci and demonstrated that the introgressed alleles from two diploid cottons had a large effect on fiber quality improvement. These loci provided the potential power to overcome the bottleneck in fiber quality improvement. Our study uncovered several critical genomic signatures generated by historical breeding effects in cotton and a wealth of data that enrich genomic resources for the research community. Genomic variations in 3,248 tetraploid cotton germplasms and multi-environmental genome-wide association analyses provide insights into the basis of geographic differentiation and fiber improvement in cultivated cotton.},
  archive      = {J_NG},
  author       = {He, Shoupu and Sun, Gaofei and Geng, Xiaoli and Gong, Wenfang and Dai, Panhong and Jia, Yinhua and Shi, Weijun and Pan, Zhaoe and Wang, Junduo and Wang, Liyuan and Xiao, Songhua and Chen, Baojun and Cui, Shufang and You, Chunyuan and Xie, Zongming and Wang, Feng and Sun, Jie and Fu, Guoyong and Peng, Zhen and Hu, Daowu and Wang, Liru and Pang, Baoyin and Du, Xiongming},
  doi          = {10.1038/s41588-021-00844-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {916-924},
  shortjournal = {Nat. Genet.},
  title        = {The genomic basis of geographic differentiation and fiber improvement in cultivated cotton},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A plasma membrane transporter coordinates phosphate
reallocation and grain filling in cereals. <em>NG</em>, <em>53</em>(6),
906–915. (<a href="https://doi.org/10.1038/s41588-021-00855-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phosphate (Pi) is essential to plant growth and crop yield. However, it remains unknown how Pi homeostasis is maintained during cereal grain filling. Here, we identified a rice grain-filling-controlling PHO1-type Pi transporter, OsPHO1;2, through map-based cloning. Pi efflux activity and its localization to the plasma membrane of seed tissues implicated a specific role for OsPHO1;2 in Pi reallocation during grain filling. Indeed, Pi over-accumulated in developing seeds of the Ospho1;2 mutant, which inhibited the activity of ADP-glucose pyrophosphorylase (AGPase), important for starch synthesis, and the grain-filling defect was alleviated by overexpression of AGPase in Ospho1;2-mutant plants. A conserved function was recognized for the maize transporter ZmPHO1;2. Importantly, ectopic overexpression of OsPHO1;2 enhanced grain yield, especially under low-Pi conditions. Collectively, we discovered a mechanism underlying Pi transport, grain filling and P-use efficiency, providing an efficient strategy for improving grain yield with minimal P-fertilizer input in cereals. A plasma membrane transporter OsPHO1;2 coordinates phosphate reallocation essential for starch biosynthesis during grain filling of cereal crops, providing a potential breeding target for improving phosphate-use efficiency.},
  archive      = {J_NG},
  author       = {Ma, Bin and Zhang, Lin and Gao, Qifei and Wang, Junmin and Li, Xiaoyuan and Wang, Hu and Liu, Yu and Lin, Hui and Liu, Jiyun and Wang, Xin and Li, Qun and Deng, Yiwen and Tang, Weihua and Luan, Sheng and He, Zuhua},
  doi          = {10.1038/s41588-021-00855-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {906-915},
  shortjournal = {Nat. Genet.},
  title        = {A plasma membrane transporter coordinates phosphate reallocation and grain filling in cereals},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chromothripsis as an on-target consequence of CRISPR–Cas9
genome editing. <em>NG</em>, <em>53</em>(6), 895–905. (<a
href="https://doi.org/10.1038/s41588-021-00838-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing has therapeutic potential for treating genetic diseases and cancer. However, the currently most practicable approaches rely on the generation of DNA double-strand breaks (DSBs), which can give rise to a poorly characterized spectrum of chromosome structural abnormalities. Here, using model cells and single-cell whole-genome sequencing, as well as by editing at a clinically relevant locus in clinically relevant cells, we show that CRISPR–Cas9 editing generates structural defects of the nucleus, micronuclei and chromosome bridges, which initiate a mutational process called chromothripsis. Chromothripsis is extensive chromosome rearrangement restricted to one or a few chromosomes that can cause human congenital disease and cancer. These results demonstrate that chromothripsis is a previously unappreciated on-target consequence of CRISPR–Cas9-generated DSBs. As genome editing is implemented in the clinic, the potential for extensive chromosomal rearrangements should be considered and monitored. Chromothripsis, a chromosomal shattering event, can be elicited by micronuclei and chromosome bridges formed by CRISPR–Cas9-generated double-stranded breaks. Extensive chromosomal rearrangements may thus be an on-target effect of genome editing.},
  archive      = {J_NG},
  author       = {Leibowitz, Mitchell L. and Papathanasiou, Stamatis and Doerfler, Phillip A. and Blaine, Logan J. and Sun, Lili and Yao, Yu and Zhang, Cheng-Zhong and Weiss, Mitchell J. and Pellman, David},
  doi          = {10.1038/s41588-021-00838-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {895-905},
  shortjournal = {Nat. Genet.},
  title        = {Chromothripsis as an on-target consequence of CRISPR–Cas9 genome editing},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reprogramming of the esophageal squamous carcinoma epigenome
by SOX2 promotes ADAR1 dependence. <em>NG</em>, <em>53</em>(6), 881–894.
(<a href="https://doi.org/10.1038/s41588-021-00859-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Esophageal squamous cell carcinomas (ESCCs) harbor recurrent chromosome 3q amplifications that target the transcription factor SOX2. Beyond its role as an oncogene in ESCC, SOX2 acts in development of the squamous esophagus and maintenance of adult esophageal precursor cells. To compare Sox2 activity in normal and malignant tissue, we developed engineered murine esophageal organoids spanning normal esophagus to Sox2-induced squamous cell carcinoma and mapped Sox2 binding and the epigenetic and transcriptional landscape with evolution from normal to cancer. While oncogenic Sox2 largely maintains actions observed in normal tissue, Sox2 overexpression with p53 and p16 inactivation promotes chromatin remodeling and evolution of the Sox2 cistrome. With Klf5, oncogenic Sox2 acquires new binding sites and enhances activity of oncogenes such as Stat3. Moreover, oncogenic Sox2 activates endogenous retroviruses, inducing expression of double-stranded RNA and dependence on the RNA editing enzyme ADAR1. These data reveal SOX2 functions in ESCC, defining targetable vulnerabilities. The transition from normal esophageal tissue to squamous carcinoma is characterized by an altered SOX2 cistrome. This transcriptional reprogramming activates endogenous retroviruses and double-stranded RNA expression, creating a dependency on the RNA editing enzyme ADAR1.},
  archive      = {J_NG},
  author       = {Wu, Zhong and Zhou, Jin and Zhang, Xiaoyang and Zhang, Zhouwei and Xie, Yingtian and Liu, Jie bin and Ho, Zandra V. and Panda, Arpit and Qiu, Xintao and Cejas, Paloma and Cañadas, Israel and Akarca, Fahire Goknur and McFarland, James M. and Nagaraja, Ankur K. and Goss, Louisa B. and Kesten, Nikolas and Si, Longlong and Lim, Klothilda and Liu, Yanli and Zhang, Yanxi and Baek, Ji Yeon and Liu, Yang and Patil, Deepa T. and Katz, Jonathan P. and Hai, Josephine and Bao, Chunyang and Stachler, Matthew and Qi, Jun and Ishizuka, Jeffrey J. and Nakagawa, Hiroshi and Rustgi, Anil K. and Wong, Kwok-Kin and Meyerson, Matthew and Barbie, David A. and Brown, Myles and Long, Henry and Bass, Adam J.},
  doi          = {10.1038/s41588-021-00859-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {881-894},
  shortjournal = {Nat. Genet.},
  title        = {Reprogramming of the esophageal squamous carcinoma epigenome by SOX2 promotes ADAR1 dependence},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-nucleotide-level mapping of DNA regulatory elements
that control fetal hemoglobin expression. <em>NG</em>, <em>53</em>(6),
869–880. (<a href="https://doi.org/10.1038/s41588-021-00861-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pinpointing functional noncoding DNA sequences and defining their contributions to health-related traits is a major challenge for modern genetics. We developed a high-throughput framework to map noncoding DNA functions with single-nucleotide resolution in four loci that control erythroid fetal hemoglobin (HbF) expression, a genetically determined trait that modifies sickle cell disease (SCD) phenotypes. Specifically, we used the adenine base editor ABEmax to introduce 10,156 separate A•T to G•C conversions in 307 predicted regulatory elements and quantified the effects on erythroid HbF expression. We identified numerous regulatory elements, defined their epigenomic structures and linked them to low-frequency variants associated with HbF expression in an SCD cohort. Targeting a newly discovered γ-globin gene repressor element in SCD donor CD34+ hematopoietic progenitors raised HbF levels in the erythroid progeny, inhibiting hypoxia-induced sickling. Our findings reveal previously unappreciated genetic complexities of HbF regulation and provide potentially therapeutic insights into SCD. An ABEmax-based screen identifies regulatory noncoding nucleotides in four loci that control erythroid fetal hemoglobin (HbF) expression. Targeting a repressor element raises HbF levels in cells from patients with sickle cell disease.},
  archive      = {J_NG},
  author       = {Cheng, Li and Li, Yichao and Qi, Qian and Xu, Peng and Feng, Ruopeng and Palmer, Lance and Chen, Jingjing and Wu, Ruiqiong and Yee, Tiffany and Zhang, Jingjing and Yao, Yu and Sharma, Akshay and Hardison, Ross C. and Weiss, Mitchell J. and Cheng, Yong},
  doi          = {10.1038/s41588-021-00861-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {869-880},
  shortjournal = {Nat. Genet.},
  title        = {Single-nucleotide-level mapping of DNA regulatory elements that control fetal hemoglobin expression},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A map of transcriptional heterogeneity and regulatory
variation in human microglia. <em>NG</em>, <em>53</em>(6), 861–868. (<a
href="https://doi.org/10.1038/s41588-021-00875-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microglia, the tissue-resident macrophages of the central nervous system (CNS), play critical roles in immune defense, development and homeostasis. However, isolating microglia from humans in large numbers is challenging. Here, we profiled gene expression variation in primary human microglia isolated from 141 patients undergoing neurosurgery. Using single-cell and bulk RNA sequencing, we identify how age, sex and clinical pathology influence microglia gene expression and which genetic variants have microglia-specific functions using expression quantitative trait loci (eQTL) mapping. We follow up one of our findings using a human induced pluripotent stem cell-based macrophage model to fine-map a candidate causal variant for Alzheimer’s disease at the BIN1 locus. Our study provides a population-scale transcriptional map of a critically important cell for human CNS development and disease. A population-scale map of gene expression in primary human microglia provides a systematic exploration of microglia diversity and how age, sex, pathology, cortical anatomy and common germline genetic variation influence the microglia transcriptome.},
  archive      = {J_NG},
  author       = {Young, Adam M. H. and Kumasaka, Natsuhiko and Calvert, Fiona and Hammond, Timothy R. and Knights, Andrew and Panousis, Nikolaos and Park, Jun Sung and Schwartzentruber, Jeremy and Liu, Jimmy and Kundu, Kousik and Segel, Michael and Murphy, Natalia A. and McMurran, Christopher E. and Bulstrode, Harry and Correia, Jason and Budohoski, Karol P. and Joannides, Alexis and Guilfoyle, Mathew R. and Trivedi, Rikin and Kirollos, Ramez and Morris, Robert and Garnett, Matthew R. and Timofeev, Ivan and Jalloh, Ibrahim and Holland, Katherine and Mannion, Richard and Mair, Richard and Watts, Colin and Price, Stephen J. and Kirkpatrick, Peter J. and Santarius, Thomas and Mountjoy, Edward and Ghoussaini, Maya and Soranzo, Nicole and Bayraktar, Omer A. and Stevens, Beth and Hutchinson, Peter J. and Franklin, Robin J. M. and Gaffney, Daniel J.},
  doi          = {10.1038/s41588-021-00875-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {861-868},
  shortjournal = {Nat. Genet.},
  title        = {A map of transcriptional heterogeneity and regulatory variation in human microglia},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The trans-ancestral genomic architecture of glycemic traits.
<em>NG</em>, <em>53</em>(6), 840–860. (<a
href="https://doi.org/10.1038/s41588-021-00852-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glycemic traits are used to diagnose and monitor type 2 diabetes and cardiometabolic health. To date, most genetic studies of glycemic traits have focused on individuals of European ancestry. Here we aggregated genome-wide association studies comprising up to 281,416 individuals without diabetes (30% non-European ancestry) for whom fasting glucose, 2-h glucose after an oral glucose challenge, glycated hemoglobin and fasting insulin data were available. Trans-ancestry and single-ancestry meta-analyses identified 242 loci (99 novel; P &amp;lt; 5 × 10−8), 80% of which had no significant evidence of between-ancestry heterogeneity. Analyses restricted to individuals of European ancestry with equivalent sample size would have led to 24 fewer new loci. Compared with single-ancestry analyses, equivalent-sized trans-ancestry fine-mapping reduced the number of estimated variants in 99% credible sets by a median of 37.5%. Genomic-feature, gene-expression and gene-set analyses revealed distinct biological signatures for each trait, highlighting different underlying biological pathways. Our results increase our understanding of diabetes pathophysiology by using trans-ancestry studies for improved power and resolution. A trans-ancestry meta-analysis of GWAS of glycemic traits in up to 281,416 individuals identifies 99 novel loci, of which one quarter was found due to the multi-ancestry approach, which also improves fine-mapping of credible variant sets.},
  archive      = {J_NG},
  author       = {Chen, Ji and Spracklen, Cassandra N. and Marenne, Gaëlle and Varshney, Arushi and Corbin, Laura J. and Luan, Jian’an and Willems, Sara M. and Wu, Ying and Zhang, Xiaoshuai and Horikoshi, Momoko and Boutin, Thibaud S. and Mägi, Reedik and Waage, Johannes and Li-Gao, Ruifang and Chan, Kei Hang Katie and Yao, Jie and Anasanti, Mila D. and Chu, Audrey Y. and Claringbould, Annique and Heikkinen, Jani and Hong, Jaeyoung and Hottenga, Jouke-Jan and Huo, Shaofeng and Kaakinen, Marika A. and Louie, Tin and März, Winfried and Moreno-Macias, Hortensia and Ndungu, Anne and Nelson, Sarah C. and Nolte, Ilja M. and North, Kari E. and Raulerson, Chelsea K. and Ray, Debashree and Rohde, Rebecca and Rybin, Denis and Schurmann, Claudia and Sim, Xueling and Southam, Lorraine and Stewart, Isobel D. and Wang, Carol A. and Wang, Yujie and Wu, Peitao and Zhang, Weihua and Ahluwalia, Tarunveer S. and Appel, Emil V. R. and Bielak, Lawrence F. and Brody, Jennifer A. and Burtt, Noël P. and Cabrera, Claudia P. and Cade, Brian E. and Chai, Jin Fang and Chai, Xiaoran and Chang, Li-Ching and Chen, Chien-Hsiun and Chen, Brian H. and Chitrala, Kumaraswamy Naidu and Chiu, Yen-Feng and de Haan, Hugoline G. and Delgado, Graciela E. and Demirkan, Ayse and Duan, Qing and Engmann, Jorgen and Fatumo, Segun A. and Gayán, Javier and Giulianini, Franco and Gong, Jung Ho and Gustafsson, Stefan and Hai, Yang and Hartwig, Fernando P. and He, Jing and Heianza, Yoriko and Huang, Tao and Huerta-Chagoya, Alicia and Hwang, Mi Yeong and Jensen, Richard A. and Kawaguchi, Takahisa and Kentistou, Katherine A. and Kim, Young Jin and Kleber, Marcus E. and Kooner, Ishminder K. and Lai, Shuiqing and Lange, Leslie A. and Langefeld, Carl D. and Lauzon, Marie and Li, Man and Ligthart, Symen and Liu, Jun and Loh, Marie and Long, Jirong and Lyssenko, Valeriya and Mangino, Massimo and Marzi, Carola and Montasser, May E. and Nag, Abhishek and Nakatochi, Masahiro and Noce, Damia and Noordam, Raymond and Pistis, Giorgio and Preuss, Michael and Raffield, Laura and Rasmussen-Torvik, Laura J. and Rich, Stephen S. and Robertson, Neil R. and Rueedi, Rico and Ryan, Kathleen and Sanna, Serena and Saxena, Richa and Schraut, Katharina E. and Sennblad, Bengt and Setoh, Kazuya and Smith, Albert V. and Sparsø, Thomas and Strawbridge, Rona J. and Takeuchi, Fumihiko and Tan, Jingyi and Trompet, Stella and van den Akker, Erik and van der Most, Peter J. and Verweij, Niek and Vogel, Mandy and Wang, Heming and Wang, Chaolong and Wang, Nan and Warren, Helen R. and Wen, Wanqing and Wilsgaard, Tom and Wong, Andrew and Wood, Andrew R. and Xie, Tian and Zafarmand, Mohammad Hadi and Zhao, Jing-Hua and Zhao, Wei and Amin, Najaf and Arzumanyan, Zorayr and Astrup, Arne and Bakker, Stephan J. L. and Baldassarre, Damiano and Beekman, Marian and Bergman, Richard N. and Bertoni, Alain and Blüher, Matthias and Bonnycastle, Lori L. and Bornstein, Stefan R. and Bowden, Donald W. and Cai, Qiuyin and Campbell, Archie and Campbell, Harry and Chang, Yi Cheng and de Geus, Eco J. C. and Dehghan, Abbas and Du, Shufa and Eiriksdottir, Gudny and Farmaki, Aliki Eleni and Frånberg, Mattias and Fuchsberger, Christian and Gao, Yutang and Gjesing, Anette P. and Goel, Anuj and Han, Sohee and Hartman, Catharina A. and Herder, Christian and Hicks, Andrew A. and Hsieh, Chang-Hsun and Hsueh, Willa A. and Ichihara, Sahoko and Igase, Michiya and Ikram, M. Arfan and Johnson, W. Craig and Jørgensen, Marit E. and Joshi, Peter K. and Kalyani, Rita R. and Kandeel, Fouad R. and Katsuya, Tomohiro and Khor, Chiea Chuen and Kiess, Wieland and Kolcic, Ivana and Kuulasmaa, Teemu and Kuusisto, Johanna and Läll, Kristi and Lam, Kelvin and Lawlor, Deborah A. and Lee, Nanette R. and Lemaitre, Rozenn N. and Li, Honglan and Lin, Shih-Yi and Lindström, Jaana and Linneberg, Allan and Liu, Jianjun and Lorenzo, Carlos and Matsubara, Tatsuaki and Matsuda, Fumihiko and Mingrone, Geltrude and Mooijaart, Simon and Moon, Sanghoon and Nabika, Toru and Nadkarni, Girish N. and Nadler, Jerry L. and Nelis, Mari and Neville, Matt J. and Norris, Jill M. and Ohyagi, Yasumasa and Peters, Annette and Peyser, Patricia A. and Polasek, Ozren and Qi, Qibin and Raven, Dennis and Reilly, Dermot F. and Reiner, Alex and Rivideneira, Fernando and Roll, Kathryn and Rudan, Igor and Sabanayagam, Charumathi and Sandow, Kevin and Sattar, Naveed and Schürmann, Annette and Shi, Jinxiu and Stringham, Heather M. and Taylor, Kent D. and Teslovich, Tanya M. and Thuesen, Betina and Timmers, Paul R. H. J. and Tremoli, Elena and Tsai, Michael Y. and Uitterlinden, Andre and van Dam, Rob M. and van Heemst, Diana and van Hylckama Vlieg, Astrid and van Vliet-Ostaptchouk, Jana V. and Vangipurapu, Jagadish and Vestergaard, Henrik and Wang, Tao and Willems van Dijk, Ko and Zemunik, Tatijana and Abecasis, Gonçalo R. and Adair, Linda S. and Aguilar-Salinas, Carlos Alberto and Alarcón-Riquelme, Marta E. and An, Ping and Aviles-Santa, Larissa and Becker, Diane M. and Beilin, Lawrence J. and Bergmann, Sven and Bisgaard, Hans and Black, Corri and Boehnke, Michael and Boerwinkle, Eric and Böhm, Bernhard O. and Bønnelykke, Klaus and Boomsma, D. I. and Bottinger, Erwin P. and Buchanan, Thomas A. and Canouil, Mickaël and Caulfield, Mark J. and Chambers, John C. and Chasman, Daniel I. and Chen, Yii-Der Ida and Cheng, Ching-Yu and Collins, Francis S. and Correa, Adolfo and Cucca, Francesco and de Silva, H. Janaka and Dedoussis, George and Elmståhl, Sölve and Evans, Michele K. and Ferrannini, Ele and Ferrucci, Luigi and Florez, Jose C. and Franks, Paul W. and Frayling, Timothy M. and Froguel, Philippe and Gigante, Bruna and Goodarzi, Mark O. and Gordon-Larsen, Penny and Grallert, Harald and Grarup, Niels and Grimsgaard, Sameline and Groop, Leif and Gudnason, Vilmundur and Guo, Xiuqing and Hamsten, Anders and Hansen, Torben and Hayward, Caroline and Heckbert, Susan R. and Horta, Bernardo L. and Huang, Wei and Ingelsson, Erik and James, Pankow S. and Jarvelin, Marjo-Ritta and Jonas, Jost B. and Jukema, J. Wouter and Kaleebu, Pontiano and Kaplan, Robert and Kardia, Sharon L. R. and Kato, Norihiro and Keinanen-Kiukaanniemi, Sirkka M. and Kim, Bong-Jo and Kivimaki, Mika and Koistinen, Heikki A. and Kooner, Jaspal S. and Körner, Antje and Kovacs, Peter and Kuh, Diana and Kumari, Meena and Kutalik, Zoltan and Laakso, Markku and Lakka, Timo A. and Launer, Lenore J. and Leander, Karin and Li, Huaixing and Lin, Xu and Lind, Lars and Lindgren, Cecilia and Liu, Simin and Loos, Ruth J. F. and Magnusson, Patrik K. E. and Mahajan, Anubha and Metspalu, Andres and Mook-Kanamori, Dennis O. and Mori, Trevor A. and Munroe, Patricia B. and Njølstad, Inger and O’Connell, Jeffrey R. and Oldehinkel, Albertine J. and Ong, Ken K. and Padmanabhan, Sandosh and Palmer, Colin N. A. and Palmer, Nicholette D. and Pedersen, Oluf and Pennell, Craig E. and Porteous, David J. and Pramstaller, Peter P. and Province, Michael A. and Psaty, Bruce M. and Qi, Lu and Raffel, Leslie J. and Rauramaa, Rainer and Redline, Susan and Ridker, Paul M. and Rosendaal, Frits R. and Saaristo, Timo E. and Sandhu, Manjinder and Saramies, Jouko and Schneiderman, Neil and Schwarz, Peter and Scott, Laura J. and Selvin, Elizabeth and Sever, Peter and Shu, Xiao-ou and Slagboom, P. Eline and Small, Kerrin S. and Smith, Blair H. and Snieder, Harold and Sofer, Tamar and Sørensen, Thorkild I. A. and Spector, Tim D. and Stanton, Alice and Steves, Claire J. and Stumvoll, Michael and Sun, Liang and Tabara, Yasuharu and Tai, E. Shyong and Timpson, Nicholas J. and Tönjes, Anke and Tuomilehto, Jaakko and Tusie, Teresa and Uusitupa, Matti and van der Harst, Pim and van Duijn, Cornelia and Vitart, Veronique and Vollenweider, Peter and Vrijkotte, Tanja G. M. and Wagenknecht, Lynne E. and Walker, Mark and Wang, Ya X. and Wareham, Nick J. and Watanabe, Richard M. and Watkins, Hugh and Wei, Wen B. and Wickremasinghe, Ananda R. and Willemsen, Gonneke and Wilson, James F. and Wong, Tien-Yin and Wu, Jer-Yuarn and Xiang, Anny H. and Yanek, Lisa R. and Yengo, Loïc and Yokota, Mitsuhiro and Zeggini, Eleftheria and Zheng, Wei and Zonderman, Alan B. and Rotter, Jerome I. and Gloyn, Anna L. and McCarthy, Mark I. and Dupuis, Josée and Meigs, James B. and Scott, Robert A. and Prokopenko, Inga and Leong, Aaron and Liu, Ching-Ti and Parker, Stephen C. J. and Mohlke, Karen L. and Langenberg, Claudia and Wheeler, Eleanor and Morris, Andrew P. and Barroso, Inês},
  doi          = {10.1038/s41588-021-00852-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {840-860},
  shortjournal = {Nat. Genet.},
  title        = {The trans-ancestral genomic architecture of glycemic traits},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Shared heritability of human face and brain shape.
<em>NG</em>, <em>53</em>(6), 830–839. (<a
href="https://doi.org/10.1038/s41588-021-00827-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evidence from model organisms and clinical genetics suggests coordination between the developing brain and face, but the role of this link in common genetic variation remains unknown. We performed a multivariate genome-wide association study of cortical surface morphology in 19,644 individuals of European ancestry, identifying 472 genomic loci influencing brain shape, of which 76 are also linked to face shape. Shared loci include transcription factors involved in craniofacial development, as well as members of signaling pathways implicated in brain–face cross-talk. Brain shape heritability is equivalently enriched near regulatory regions active in either forebrain organoids or facial progenitors. However, we do not detect significant overlap between shared brain–face genome-wide association study signals and variants affecting behavioral–cognitive traits. These results suggest that early in embryogenesis, the face and brain mutually shape each other through both structural effects and paracrine signaling, but this interplay may not impact later brain development associated with cognitive function. A multivariate genome-wide association study highlighting loci that influence both face and brain shape suggesting shared developmental axes during early embryogenesis. These loci did not overlap with those governing behavioral–cognitive traits or neuropsychiatric risk indicating divergence between early brain development and cognitive function.},
  archive      = {J_NG},
  author       = {Naqvi, Sahin and Sleyp, Yoeri and Hoskens, Hanne and Indencleef, Karlijne and Spence, Jeffrey P. and Bruffaerts, Rose and Radwan, Ahmed and Eller, Ryan J. and Richmond, Stephen and Shriver, Mark D. and Shaffer, John R. and Weinberg, Seth M. and Walsh, Susan and Thompson, James and Pritchard, Jonathan K. and Sunaert, Stefan and Peeters, Hilde and Wysocka, Joanna and Claes, Peter},
  doi          = {10.1038/s41588-021-00827-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {830-839},
  shortjournal = {Nat. Genet.},
  title        = {Shared heritability of human face and brain shape},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide association study of more than 40,000 bipolar
disorder cases provides new insights into the underlying biology.
<em>NG</em>, <em>53</em>(6), 817–829. (<a
href="https://doi.org/10.1038/s41588-021-00857-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bipolar disorder is a heritable mental illness with complex etiology. We performed a genome-wide association study of 41,917 bipolar disorder cases and 371,549 controls of European ancestry, which identified 64 associated genomic loci. Bipolar disorder risk alleles were enriched in genes in synaptic signaling pathways and brain-expressed genes, particularly those with high specificity of expression in neurons of the prefrontal cortex and hippocampus. Significant signal enrichment was found in genes encoding targets of antipsychotics, calcium channel blockers, antiepileptics and anesthetics. Integrating expression quantitative trait locus data implicated 15 genes robustly linked to bipolar disorder via gene expression, encoding druggable targets such as HTR6, MCHR1, DCLK3 and FURIN. Analyses of bipolar disorder subtypes indicated high but imperfect genetic correlation between bipolar disorder type I and II and identified additional associated loci. Together, these results advance our understanding of the biological etiology of bipolar disorder, identify novel therapeutic leads and prioritize genes for functional follow-up studies. Genome-wide association analyses of 41,917 bipolar disorder cases and 371,549 controls of European ancestry provide new insights into the etiology of this disorder and identify novel therapeutic leads and potential opportunities for drug repurposing.},
  archive      = {J_NG},
  author       = {Mullins, Niamh and Forstner, Andreas J. and O’Connell, Kevin S. and Coombes, Brandon and Coleman, Jonathan R. I. and Qiao, Zhen and Als, Thomas D. and Bigdeli, Tim B. and Børte, Sigrid and Bryois, Julien and Charney, Alexander W. and Drange, Ole Kristian and Gandal, Michael J. and Hagenaars, Saskia P. and Ikeda, Masashi and Kamitaki, Nolan and Kim, Minsoo and Krebs, Kristi and Panagiotaropoulou, Georgia and Schilder, Brian M. and Sloofman, Laura G. and Steinberg, Stacy and Trubetskoy, Vassily and Winsvold, Bendik S. and Won, Hong-Hee and Abramova, Liliya and Adorjan, Kristina and Agerbo, Esben and Al Eissa, Mariam and Albani, Diego and Alliey-Rodriguez, Ney and Anjorin, Adebayo and Antilla, Verneri and Antoniou, Anastasia and Awasthi, Swapnil and Baek, Ji Hyun and Bækvad-Hansen, Marie and Bass, Nicholas and Bauer, Michael and Beins, Eva C. and Bergen, Sarah E. and Birner, Armin and Bøcker Pedersen, Carsten and Bøen, Erlend and Boks, Marco P. and Bosch, Rosa and Brum, Murielle and Brumpton, Ben M. and Brunkhorst-Kanaan, Nathalie and Budde, Monika and Bybjerg-Grauholm, Jonas and Byerley, William and Cairns, Murray and Casas, Miquel and Cervantes, Pablo and Clarke, Toni-Kim and Cruceanu, Cristiana and Cuellar-Barboza, Alfredo and Cunningham, Julie and Curtis, David and Czerski, Piotr M. and Dale, Anders M. and Dalkner, Nina and David, Friederike S. and Degenhardt, Franziska and Djurovic, Srdjan and Dobbyn, Amanda L. and Douzenis, Athanassios and Elvsåshagen, Torbjørn and Escott-Price, Valentina and Ferrier, I. Nicol and Fiorentino, Alessia and Foroud, Tatiana M. and Forty, Liz and Frank, Josef and Frei, Oleksandr and Freimer, Nelson B. and Frisén, Louise and Gade, Katrin and Garnham, Julie and Gelernter, Joel and Giørtz Pedersen, Marianne and Gizer, Ian R. and Gordon, Scott D. and Gordon-Smith, Katherine and Greenwood, Tiffany A. and Grove, Jakob and Guzman-Parra, José and Ha, Kyooseob and Haraldsson, Magnus and Hautzinger, Martin and Heilbronner, Urs and Hellgren, Dennis and Herms, Stefan and Hoffmann, Per and Holmans, Peter A. and Huckins, Laura and Jamain, Stéphane and Johnson, Jessica S. and Kalman, Janos L. and Kamatani, Yoichiro and Kennedy, James L. and Kittel-Schneider, Sarah and Knowles, James A. and Kogevinas, Manolis and Koromina, Maria and Kranz, Thorsten M. and Kranzler, Henry R. and Kubo, Michiaki and Kupka, Ralph and Kushner, Steven A. and Lavebratt, Catharina and Lawrence, Jacob and Leber, Markus and Lee, Heon-Jeong and Lee, Phil H. and Levy, Shawn E. and Lewis, Catrin and Liao, Calwing and Lucae, Susanne and Lundberg, Martin and MacIntyre, Donald J. and Magnusson, Sigurdur H. and Maier, Wolfgang and Maihofer, Adam and Malaspina, Dolores and Maratou, Eirini and Martinsson, Lina and Mattheisen, Manuel and McCarroll, Steven A. and McGregor, Nathaniel W. and McGuffin, Peter and McKay, James D. and Medeiros, Helena and Medland, Sarah E. and Millischer, Vincent and Montgomery, Grant W. and Moran, Jennifer L. and Morris, Derek W. and Mühleisen, Thomas W. and O’Brien, Niamh and O’Donovan, Claire and Olde Loohuis, Loes M. and Oruc, Lilijana and Papiol, Sergi and Pardiñas, Antonio F. and Perry, Amy and Pfennig, Andrea and Porichi, Evgenia and Potash, James B. and Quested, Digby and Raj, Towfique and Rapaport, Mark H. and DePaulo, J. Raymond and Regeer, Eline J. and Rice, John P. and Rivas, Fabio and Rivera, Margarita and Roth, Julian and Roussos, Panos and Ruderfer, Douglas M. and Sánchez-Mora, Cristina and Schulte, Eva C. and Senner, Fanny and Sharp, Sally and Shilling, Paul D. and Sigurdsson, Engilbert and Sirignano, Lea and Slaney, Claire and Smeland, Olav B. and Smith, Daniel J. and Sobell, Janet L. and Søholm Hansen, Christine and Soler Artigas, Maria and Spijker, Anne T. and Stein, Dan J. and Strauss, John S. and Świątkowska, Beata and Terao, Chikashi and Thorgeirsson, Thorgeir E. and Toma, Claudio and Tooney, Paul and Tsermpini, Evangelia-Eirini and Vawter, Marquis P. and Vedder, Helmut and Walters, James T. R. and Witt, Stephanie H. and Xi, Simon and Xu, Wei and Yang, Jessica Mei Kay and Young, Allan H. and Young, Hannah and Zandi, Peter P. and Zhou, Hang and Zillich, Lea and Adolfsson, Rolf and Agartz, Ingrid and Alda, Martin and Alfredsson, Lars and Babadjanova, Gulja and Backlund, Lena and Baune, Bernhard T. and Bellivier, Frank and Bengesser, Susanne and Berrettini, Wade H. and Blackwood, Douglas H. R. and Boehnke, Michael and Børglum, Anders D. and Breen, Gerome and Carr, Vaughan J. and Catts, Stanley and Corvin, Aiden and Craddock, Nicholas and Dannlowski, Udo and Dikeos, Dimitris and Esko, Tõnu and Etain, Bruno and Ferentinos, Panagiotis and Frye, Mark and Fullerton, Janice M. and Gawlik, Micha and Gershon, Elliot S. and Goes, Fernando S. and Green, Melissa J. and Grigoroiu-Serbanescu, Maria and Hauser, Joanna and Henskens, Frans and Hillert, Jan and Hong, Kyung Sue and Hougaard, David M. and Hultman, Christina M. and Hveem, Kristian and Iwata, Nakao and Jablensky, Assen V. and Jones, Ian and Jones, Lisa A. and Kahn, René S. and Kelsoe, John R. and Kirov, George and Landén, Mikael and Leboyer, Marion and Lewis, Cathryn M. and Li, Qingqin S. and Lissowska, Jolanta and Lochner, Christine and Loughland, Carmel and Martin, Nicholas G. and Mathews, Carol A. and Mayoral, Fermin and McElroy, Susan L. and McIntosh, Andrew M. and McMahon, Francis J. and Melle, Ingrid and Michie, Patricia and Milani, Lili and Mitchell, Philip B. and Morken, Gunnar and Mors, Ole and Mortensen, Preben Bo and Mowry, Bryan and Müller-Myhsok, Bertram and Myers, Richard M. and Neale, Benjamin M. and Nievergelt, Caroline M. and Nordentoft, Merete and Nöthen, Markus M. and O’Donovan, Michael C. and Oedegaard, Ketil J. and Olsson, Tomas and Owen, Michael J. and Paciga, Sara A. and Pantelis, Chris and Pato, Carlos and Pato, Michele T. and Patrinos, George P. and Perlis, Roy H. and Posthuma, Danielle and Ramos-Quiroga, Josep Antoni and Reif, Andreas and Reininghaus, Eva Z. and Ribasés, Marta and Rietschel, Marcella and Ripke, Stephan and Rouleau, Guy A. and Saito, Takeo and Schall, Ulrich and Schalling, Martin and Schofield, Peter R. and Schulze, Thomas G. and Scott, Laura J. and Scott, Rodney J. and Serretti, Alessandro and Shannon Weickert, Cynthia and Smoller, Jordan W. and Stefansson, Hreinn and Stefansson, Kari and Stordal, Eystein and Streit, Fabian and Sullivan, Patrick F. and Turecki, Gustavo and Vaaler, Arne E. and Vieta, Eduard and Vincent, John B. and Waldman, Irwin D. and Weickert, Thomas W. and Werge, Thomas and Wray, Naomi R. and Zwart, John-Anker and Biernacka, Joanna M. and Nurnberger, John I. and Cichon, Sven and Edenberg, Howard J. and Stahl, Eli A. and McQuillin, Andrew and Di Florio, Arianna and Ophoff, Roel A. and Andreassen, Ole A.},
  doi          = {10.1038/s41588-021-00857-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {817-829},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study of more than 40,000 bipolar disorder cases provides new insights into the underlying biology},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultrafast sample placement on existing tRees (UShER) enables
real-time phylogenetics for the SARS-CoV-2 pandemic. <em>NG</em>,
<em>53</em>(6), 809–816. (<a
href="https://doi.org/10.1038/s41588-021-00862-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As the SARS-CoV-2 virus spreads through human populations, the unprecedented accumulation of viral genome sequences is ushering in a new era of ‘genomic contact tracing’—that is, using viral genomes to trace local transmission dynamics. However, because the viral phylogeny is already so large—and will undoubtedly grow many fold—placing new sequences onto the tree has emerged as a barrier to real-time genomic contact tracing. Here, we resolve this challenge by building an efficient tree-based data structure encoding the inferred evolutionary history of the virus. We demonstrate that our approach greatly improves the speed of phylogenetic placement of new samples and data visualization, making it possible to complete the placements under the constraints of real-time contact tracing. Thus, our method addresses an important need for maintaining a fully updated reference phylogeny. We make these tools available to the research community through the University of California Santa Cruz SARS-CoV-2 Genome Browser to enable rapid cross-referencing of information in new virus sequences with an ever-expanding array of molecular and structural biology data. The methods described here will empower research and genomic contact tracing for SARS-CoV-2 specifically for laboratories worldwide. Ultrafast Sample placement on Existing tRees (UShER) is an efficient method that facilitates the addition of new SARS-CoV-2 genome sequences onto the existing phylogeny, aiding in real-time analysis of viral evolution during the COVID-19 pandemic.},
  archive      = {J_NG},
  author       = {Turakhia, Yatish and Thornlow, Bryan and Hinrichs, Angie S. and De Maio, Nicola and Gozashti, Landen and Lanfear, Robert and Haussler, David and Corbett-Detig, Russell},
  doi          = {10.1038/s41588-021-00862-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {809-816},
  shortjournal = {Nat. Genet.},
  title        = {Ultrafast sample placement on existing tRees (UShER) enables real-time phylogenetics for the SARS-CoV-2 pandemic},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Trans-ancestry analysis reveals genetic and nongenetic
associations with COVID-19 susceptibility and severity. <em>NG</em>,
<em>53</em>(6), 801–808. (<a
href="https://doi.org/10.1038/s41588-021-00854-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {COVID-19 presents with a wide range of severity, from asymptomatic in some individuals to fatal in others. Based on a study of 1,051,032 23andMe research participants, we report genetic and nongenetic associations with testing positive for SARS-CoV-2, respiratory symptoms and hospitalization. Using trans-ancestry genome-wide association studies, we identified a strong association between blood type and COVID-19 diagnosis, as well as a gene-rich locus on chromosome 3p21.31 that is more strongly associated with outcome severity. Hospitalization risk factors include advancing age, male sex, obesity, lower socioeconomic status, non-European ancestry and preexisting cardiometabolic conditions. While non-European ancestry was a significant risk factor for hospitalization after adjusting for sociodemographics and preexisting health conditions, we did not find evidence that these two primary genetic associations explain risk differences between populations for severe COVID-19 outcomes. A study of more than one million 23andMe research participants identifies genetic and nongenetic associations with COVID-19 susceptibility and severity.},
  archive      = {J_NG},
  author       = {Shelton, Janie F. and Shastri, Anjali J. and Ye, Chelsea and Weldon, Catherine H. and Filshtein-Sonmez, Teresa and Coker, Daniella and Symons, Antony and Esparza-Gordillo, Jorge and Aslibekyan, Stella and Auton, Adam},
  doi          = {10.1038/s41588-021-00854-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {801-808},
  shortjournal = {Nat. Genet.},
  title        = {Trans-ancestry analysis reveals genetic and nongenetic associations with COVID-19 susceptibility and severity},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Two competing mechanisms of DNMT3A recruitment regulate the
dynamics of de novo DNA methylation at PRC1-targeted CpG islands.
<em>NG</em>, <em>53</em>(6), 794–800. (<a
href="https://doi.org/10.1038/s41588-021-00856-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precise deposition of CpG methylation is critical for mammalian development and tissue homeostasis and is often dysregulated in human diseases. The localization of de novo DNA methyltransferase DNMT3A is facilitated by its PWWP domain recognizing histone H3 lysine 36 (H3K36) methylation1,2 and is normally depleted at CpG islands (CGIs)3. However, methylation of CGIs regulated by Polycomb repressive complexes (PRCs) has also been observed4–8. Here, we report that DNMT3A PWWP domain mutations identified in paragangliomas9 and microcephalic dwarfism10 promote aberrant localization of DNMT3A to CGIs in a PRC1-dependent manner. DNMT3A PWWP mutants accumulate at regions containing PRC1-mediated formation of monoubiquitylated histone H2A lysine 119 (H2AK119ub), irrespective of the amounts of PRC2-catalyzed formation of trimethylated histone H3 lysine 27 (H3K27me3). DNMT3A interacts with H2AK119ub-modified nucleosomes through a putative amino-terminal ubiquitin-dependent recruitment region, providing an alternative form of DNMT3A genomic targeting that is augmented by the loss of PWWP reader function. Ablation of PRC1 abrogates localization of DNMT3A PWWP mutants to CGIs and prevents aberrant DNA hypermethylation. Our study implies that a balance between DNMT3A recruitment by distinct reader domains guides de novo CpG methylation and may underlie the abnormal DNA methylation landscapes observed in select human cancer subtypes and developmental disorders. DNMT3A PWWP domain mutations promote localization of DNMT3A to CpG islands in a PRC1-dependent manner. DNMT3A interacts with H2AK119ub-modified nucleosomes.},
  archive      = {J_NG},
  author       = {Weinberg, Daniel N. and Rosenbaum, Phillip and Chen, Xiao and Barrows, Douglas and Horth, Cynthia and Marunde, Matthew R. and Popova, Irina K. and Gillespie, Zachary B. and Keogh, Michael-Christopher and Lu, Chao and Majewski, Jacek and Allis, C. David},
  doi          = {10.1038/s41588-021-00856-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {794-800},
  shortjournal = {Nat. Genet.},
  title        = {Two competing mechanisms of DNMT3A recruitment regulate the dynamics of de novo DNA methylation at PRC1-targeted CpG islands},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide survival study identifies a novel synaptic locus
and polygenic score for cognitive progression in parkinson’s disease.
<em>NG</em>, <em>53</em>(6), 787–793. (<a
href="https://doi.org/10.1038/s41588-021-00847-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key driver of patients’ well-being and clinical trials for Parkinson’s disease (PD) is the course that the disease takes over time (progression and prognosis). To assess how genetic variation influences the progression of PD over time to dementia, a major determinant for quality of life, we performed a longitudinal genome-wide survival study of 11.2 million variants in 3,821 patients with PD over 31,053 visits. We discover RIMS2 as a progression locus and confirm this in a replicate population (hazard ratio (HR) = 4.77, P = 2.78 × 10−11), identify suggestive evidence for TMEM108 (HR = 2.86, P = 2.09 × 10−8) and WWOX (HR = 2.12, P = 2.37 × 10−8) as progression loci, and confirm associations for GBA (HR = 1.93, P = 0.0002) and APOE (HR = 1.48, P = 0.001). Polygenic progression scores exhibit a substantial aggregate association with dementia risk, while polygenic susceptibility scores are not predictive. This study identifies a novel synaptic locus and polygenic score for cognitive disease progression in PD and proposes diverging genetic architectures of progression and susceptibility. A genome-wide survival study identifies variants at RIMS2 associated with progression of Parkinson’s disease to dementia and highlights divergence in the genetic architecture of disease onset and progression.},
  archive      = {J_NG},
  author       = {Liu, Ganqiang and Peng, Jiajie and Liao, Zhixiang and Locascio, Joseph J. and Corvol, Jean-Christophe and Zhu, Frank and Dong, Xianjun and Maple-Grødem, Jodi and Campbell, Meghan C. and Elbaz, Alexis and Lesage, Suzanne and Brice, Alexis and Mangone, Graziella and Growdon, John H. and Hung, Albert Y. and Schwarzschild, Michael A. and Hayes, Michael T. and Wills, Anne-Marie and Herrington, Todd M. and Ravina, Bernard and Shoulson, Ira and Taba, Pille and Kõks, Sulev and Beach, Thomas G. and Cormier-Dequaire, Florence and Alves, Guido and Tysnes, Ole-Bjørn and Perlmutter, Joel S. and Heutink, Peter and Amr, Sami S. and van Hilten, Jacobus J. and Kasten, Meike and Mollenhauer, Brit and Trenkwalder, Claudia and Klein, Christine and Barker, Roger A. and Williams-Gray, Caroline H. and Marinus, Johan and Scherzer, Clemens R.},
  doi          = {10.1038/s41588-021-00847-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {787-793},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide survival study identifies a novel synaptic locus and polygenic score for cognitive progression in parkinson’s disease},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Long-read sequencing of 3,622 icelanders provides insight
into the role of structural variants in human diseases and other traits.
<em>NG</em>, <em>53</em>(6), 779–786. (<a
href="https://doi.org/10.1038/s41588-021-00865-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing (LRS) promises to improve the characterization of structural variants (SVs). We generated LRS data from 3,622 Icelanders and identified a median of 22,636 SVs per individual (a median of 13,353 insertions and 9,474 deletions). We discovered a set of 133,886 reliably genotyped SV alleles and imputed them into 166,281 individuals to explore their effects on diseases and other traits. We discovered an association of a rare deletion in PCSK9 with lower low-density lipoprotein (LDL) cholesterol levels, compared to the population average. We also discovered an association of a multiallelic SV in ACAN with height; we found 11 alleles that differed in the number of a 57-bp-motif repeat and observed a linear relationship between the number of repeats carried and height. These results show that SVs can be accurately characterized at the population scale using LRS data in a genome-wide non-targeted approach and demonstrate how SVs impact phenotypes. Analysis of long-read sequencing data from 3,622 Icelanders identifies a set of high-confidence structural variants and provides insights into their effect on human traits and diseases.},
  archive      = {J_NG},
  author       = {Beyter, Doruk and Ingimundardottir, Helga and Oddsson, Asmundur and Eggertsson, Hannes P. and Bjornsson, Eythor and Jonsson, Hakon and Atlason, Bjarni A. and Kristmundsdottir, Snaedis and Mehringer, Svenja and Hardarson, Marteinn T. and Gudjonsson, Sigurjon A. and Magnusdottir, Droplaug N. and Jonasdottir, Aslaug and Jonasdottir, Adalbjorg and Kristjansson, Ragnar P. and Sverrisson, Sverrir T. and Holley, Guillaume and Palsson, Gunnar and Stefansson, Olafur A. and Eyjolfsson, Gudmundur and Olafsson, Isleifur and Sigurdardottir, Olof and Torfason, Bjarni and Masson, Gisli and Helgason, Agnar and Thorsteinsdottir, Unnur and Holm, Hilma and Gudbjartsson, Daniel F. and Sulem, Patrick and Magnusson, Olafur T. and Halldorsson, Bjarni V. and Stefansson, Kari},
  doi          = {10.1038/s41588-021-00865-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {779-786},
  shortjournal = {Nat. Genet.},
  title        = {Long-read sequencing of 3,622 icelanders provides insight into the role of structural variants in human diseases and other traits},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Separating measurement and expression models clarifies
confusion in single-cell RNA sequencing analysis. <em>NG</em>,
<em>53</em>(6), 770–777. (<a
href="https://doi.org/10.1038/s41588-021-00873-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The high proportion of zeros in typical single-cell RNA sequencing datasets has led to widespread but inconsistent use of terminology such as dropout and missing data. Here, we argue that much of this terminology is unhelpful and confusing, and outline simple ideas to help to reduce confusion. These include: (1) observed single-cell RNA sequencing counts reflect both true gene expression levels and measurement error, and carefully distinguishing between these contributions helps to clarify thinking; and (2) method development should start with a Poisson measurement model, rather than more complex models, because it is simple and generally consistent with existing data. We outline how several existing methods can be viewed within this framework and highlight how these methods differ in their assumptions about expression variation. We also illustrate how our perspective helps to address questions of biological interest, such as whether messenger RNA expression levels are multimodal among cells. Numerous statistical models have been used to analyze single-cell RNA sequencing data. This Perspective proposes that a Poisson measurement model is sufficient and suitably flexible for this task, and its use would resolve current controversies.},
  archive      = {J_NG},
  author       = {Sarkar, Abhishek and Stephens, Matthew},
  doi          = {10.1038/s41588-021-00873-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {770-777},
  shortjournal = {Nat. Genet.},
  title        = {Separating measurement and expression models clarifies confusion in single-cell RNA sequencing analysis},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CRISPR–Cas9 can cause chromothripsis. <em>NG</em>,
<em>53</em>(6), 768–769. (<a
href="https://doi.org/10.1038/s41588-021-00881-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing with CRISPR–Cas9 is beginning to be used clinically; promising results to date inspire hope for broad medical impact and mindfulness about safety. A new study shows that when Cas9 cuts its target, a fraction of the time, the target chromosome experiences a breakage process known as chromothripsis, thus prompting efforts to understand the potential negative consequences of this phenomenon and ways to mitigate them.},
  archive      = {J_NG},
  author       = {Urnov, Fyodor D.},
  doi          = {10.1038/s41588-021-00881-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {768-769},
  shortjournal = {Nat. Genet.},
  title        = {CRISPR–Cas9 can cause chromothripsis},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Scrutinizing disease states and regulation in human
microglia. <em>NG</em>, <em>53</em>(6), 766–767. (<a
href="https://doi.org/10.1038/s41588-021-00826-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell and bulk transcriptomics of adult human microglia from a population of individuals reveals activated states across several brain disorders and maps Alzheimer’s disease variants to microglia-enriched genes.},
  archive      = {J_NG},
  author       = {Kelley, Kevin W. and Pașca, Sergiu P.},
  doi          = {10.1038/s41588-021-00826-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {766-767},
  shortjournal = {Nat. Genet.},
  title        = {Scrutinizing disease states and regulation in human microglia},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Our faces and brains. <em>NG</em>, <em>53</em>(6), 765–766.
(<a href="https://doi.org/10.1038/s41588-021-00858-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neurological, cognitive and genetic sciences share a longstanding interest in studying the relationship between the shape of the human skull and of the brain. However, the downstream biological genetic underpinnings of such interactions, if they exist, remain largely unknown. A new genetic study now reports on a set of genes that may be involved in substantially shaping the profile of our face and the physical shape of our brain, but not so much its capabilities.},
  archive      = {J_NG},
  author       = {van den Heuvel, Martijn P. and Posthuma, Danielle},
  doi          = {10.1038/s41588-021-00858-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {765-766},
  shortjournal = {Nat. Genet.},
  title        = {Our faces and brains},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Publisher correction: Clinical use of current polygenic risk
scores may exacerbate health disparities. <em>NG</em>, <em>53</em>(5),
763. (<a href="https://doi.org/10.1038/s41588-021-00797-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00797-z.},
  archive      = {J_NG},
  author       = {Martin, Alicia R. and Kanai, Masahiro and Kamatani, Yoichiro and Okada, Yukinori and Neale, Benjamin M. and Daly, Mark J.},
  doi          = {10.1038/s41588-021-00797-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Clinical use of current polygenic risk scores may exacerbate health disparities},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Mutation load at a mimicry supergene
sheds new light on the evolution of inversion polymorphisms.
<em>NG</em>, <em>53</em>(5), 763. (<a
href="https://doi.org/10.1038/s41588-021-00810-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00810-5.},
  archive      = {J_NG},
  author       = {Jay, Paul and Chouteau, Mathieu and Whibley, Annabel and Bastide, Héloïse and Parrinello, Hugues and Llaurens, Violaine and Joron, Mathieu},
  doi          = {10.1038/s41588-021-00810-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Mutation load at a mimicry supergene sheds new light on the evolution of inversion polymorphisms},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Publisher correction: Discovery of rare variants associated
with blood pressure regulation through meta-analysis of 1.3 million
individuals. <em>NG</em>, <em>53</em>(5), 762. (<a
href="https://doi.org/10.1038/s41588-021-00832-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00832-z.},
  archive      = {J_NG},
  author       = {Surendran, Praveen and Feofanova, Elena V. and Lahrouchi, Najim and Ntalla, Ioanna and Karthikeyan, Savita and Cook, James and Chen, Lingyan and Mifsud, Borbala and Yao, Chen and Kraja, Aldi T. and Cartwright, James H. and Hellwege, Jacklyn N. and Giri, Ayush and Tragante, Vinicius and Thorleifsson, Gudmar and Liu, Dajiang J. and Prins, Bram P. and Stewart, Isobel D. and Cabrera, Claudia P. and Eales, James M. and Akbarov, Artur and Auer, Paul L. and Bielak, Lawrence F. and Bis, Joshua C. and Braithwaite, Vickie S. and Brody, Jennifer A. and Daw, E. Warwick and Warren, Helen R. and Drenos, Fotios and Nielsen, Sune Fallgaard and Faul, Jessica D. and Fauman, Eric B. and Fava, Cristiano and Ferreira, Teresa and Foley, Christopher N. and Franceschini, Nora and Gao, He and Giannakopoulou, Olga and Giulianini, Franco and Gudbjartsson, Daniel F. and Guo, Xiuqing and Harris, Sarah E. and Havulinna, Aki S. and Helgadottir, Anna and Huffman, Jennifer E. and Hwang, Shih-Jen and Kanoni, Stavroula and Kontto, Jukka and Larson, Martin G. and Li-Gao, Ruifang and Lindström, Jaana and Lotta, Luca A. and Lu, Yingchang and Luan, Jian’an and Mahajan, Anubha and Malerba, Giovanni and Masca, Nicholas G. D. and Mei, Hao and Menni, Cristina and Mook-Kanamori, Dennis O. and Mosen-Ansorena, David and Müller-Nurasyid, Martina and Paré, Guillaume and Paul, Dirk S. and Perola, Markus and Poveda, Alaitz and Rauramaa, Rainer and Richard, Melissa and Richardson, Tom G. and Sepúlveda, Nuno and Sim, Xueling and Smith, Albert V. and Smith, Jennifer A. and Staley, James R. and Stanáková, Alena and Sulem, Patrick and Thériault, Sébastien and Thorsteinsdottir, Unnur and Trompet, Stella and Varga, Tibor V. and Velez Edwards, Digna R. and Veronesi, Giovanni and Weiss, Stefan and Willems, Sara M. and Yao, Jie and Young, Robin and Yu, Bing and Zhang, Weihua and Zhao, Jing-Hua and Zhao, Wei and Evangelou, Evangelos and Aeschbacher, Stefanie and Asllanaj, Eralda and Blankenberg, Stefan and Bonnycastle, Lori L. and Bork-Jensen, Jette and Brandslund, Ivan and Braund, Peter S. and Burgess, Stephen and Cho, Kelly and Christensen, Cramer and Connell, John and Mutsert, Renée de and Dominiczak, Anna F. and Dörr, Marcus and Eiriksdottir, Gudny and Farmaki, Aliki-Eleni and Gaziano, J. Michael and Grarup, Niels and Grove, Megan L. and Hallmans, Göran and Hansen, Torben and Have, Christian T. and Heiss, Gerardo and Jørgensen, Marit E. and Jousilahti, Pekka and Kajantie, Eero and Kamat, Mihir and Käräjämäki, AnneMari and Karpe, Fredrik and Koistinen, Heikki A. and Kovesdy, Csaba P. and Kuulasmaa, Kari and Laatikainen, Tiina and Lannfelt, Lars and Lee, I-Te and Lee, Wen-Jane and Linneberg, Allan and Martin, Lisa W. and Moitry, Marie and Nadkarni, Girish and Neville, Matt J. and Palmer, Colin N. A. and Papanicolaou, George J. and Pedersen, Oluf and Peters, James and Poulter, Neil and Rasheed, Asif and Rasmussen, Katrine L. and Rayner, N. William and Mägi, Reedik and Renström, Frida and Rettig, Rainer and Rossouw, Jacques and Schreiner, Pamela J. and Sever, Peter S. and Sigurdsson, Emil L. and Skaaby, Tea and Sun, Yan V. and Sundstrom, Johan and Thorgeirsson, Gudmundur and Esko, Tõnu and Trabetti, Elisabetta and Tsao, Philip S. and Tuomi, Tiinamaija and Turner, Stephen T. and Tzoulaki, Ioanna and Vaartjes, Ilonca and Vergnaud, Anne-Claire and Willer, Cristen J. and Wilson, Peter W. F. and Witte, Daniel R. and Yonova-Doing, Ekaterina and Zhang, He and Aliya, Naheed and Almgren, Peter and Amouyel, Philippe and Asselbergs, Folkert W. and Barnes, Michael R. and Blakemore, Alexandra I. and Boehnke, Michael and Bots, Michiel L. and Bottinger, Erwin P. and Buring, Julie E. and Chambers, John C. and Chen, Yii-Der Ida and Chowdhury, Rajiv and Conen, David and Correa, Adolfo and Davey Smith, George and Boer, Rudolf A. de and Deary, Ian J. and Dedoussis, George and Deloukas, Panos and Di Angelantonio, Emanuele and Elliott, Paul and Felix, Stephan B. and Ferrières, Jean and Ford, Ian and Fornage, Myriam and Franks, Paul W. and Franks, Stephen and Frossard, Philippe and Gambaro, Giovanni and Gaunt, Tom R. and Groop, Leif and Gudnason, Vilmundur and Harris, Tamara B. and Hayward, Caroline and Hennig, Branwen J. and Herzig, Karl-Heinz and Ingelsson, Erik and Tuomilehto, Jaakko and Järvelin, Marjo-Riitta and Jukema, J. Wouter and Kardia, Sharon L. R. and Kee, Frank and Kooner, Jaspal S. and Kooperberg, Charles and Launer, Lenore J. and Lind, Lars and Loos, Ruth J. F. and Majumder, Abdulla al Shafi. and Laakso, Markku and McCarthy, Mark I. and Melander, Olle and Mohlke, Karen L. and Murray, Alison D. and Nordestgaard, Børge Grønne and Orho-Melander, Marju and Packard, Chris J. and Padmanabhan, Sandosh and Palmas, Walter and Polasek, Ozren and Porteous, David J. and Prentice, Andrew M. and Province, Michael A. and Relton, Caroline L. and Rice, Kenneth and Ridker, Paul M. and Rolandsson, Olov and Rosendaal, Frits R. and Rotter, Jerome I. and Rudan, Igor and Salomaa, Veikko and Samani, Nilesh J. and Sattar, Naveed and Sheu, Wayne H.-H. and Smith, Blair H. and Soranzo, Nicole and Spector, Timothy D. and Starr, John M. and Sebert, Sylvain and Taylor, Kent D. and Lakka, Timo A. and Timpson, Nicholas J. and Tobin, Martin D. and van der Harst, Pim and van der Meer, Peter and Ramachandran, Vasan S. and Verweij, Niek and Virtamo, Jarmo and Völker, Uwe and Weir, David R. and Zeggini, Eleftheria and Charchar, Fadi J. and Wareham, Nicholas J. and Langenberg, Claudia and Tomaszewski, Maciej and Butterworth, Adam S. and Caulfield, Mark J. and Danesh, John and Edwards, Todd L. and Holm, Hilma and Hung, Adriana M. and Lindgren, Cecilia M. and Liu, Chunyu and Manning, Alisa K. and Morris, Andrew P. and Morrison, Alanna C. and O’Donnell, Christopher J. and Psaty, Bruce M. and Saleheen, Danish and Stefansson, Kari and Boerwinkle, Eric and Chasman, Daniel I. and Levy, Daniel and Newton-Cheh, Christopher and Munroe, Patricia B. and Howson, Joanna M. M.},
  doi          = {10.1038/s41588-021-00832-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {762},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Discovery of rare variants associated with blood pressure regulation through meta-analysis of 1.3 million individuals},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Conservation of copy number profiles
during engraftment and passaging of patient-derived cancer xenografts.
<em>NG</em>, <em>53</em>(5), 761. (<a
href="https://doi.org/10.1038/s41588-021-00811-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00811-4.},
  archive      = {J_NG},
  author       = {Woo, Xing Yi and Giordano, Jessica and Srivastava, Anuj and Zhao, Zi-Ming and Lloyd, Michael W. and de Bruijn, Roebi and Suh, Yun-Suhk and Patidar, Rajesh and Chen, Li and Scherer, Sandra and Bailey, Matthew H. and Yang, Chieh-Hsiang and Cortes-Sanchez, Emilio and Xi, Yuanxin and Wang, Jing and Wickramasinghe, Jayamanna and Kossenkov, Andrew V. and Rebecca, Vito W. and Sun, Hua and Mashl, R. Jay and Davies, Sherri R. and Jeon, Ryan and Frech, Christian and Randjelovic, Jelena and Rosains, Jacqueline and Galimi, Francesco and Bertotti, Andrea and Lafferty, Adam and O’Farrell, Alice C. and Modave, Elodie and Lambrechts, Diether and ter Brugge, Petra and Serra, Violeta and Marangoni, Elisabetta and El Botty, Rania and Kim, Hyunsoo and Kim, Jong-Il and Yang, Han-Kwang and Lee, Charles and Dean, Dennis A. and Davis-Dusenbery, Brandi and Evrard, Yvonne A. and Doroshow, James H. and Welm, Alana L. and Welm, Bryan E. and Lewis, Michael T. and Fang, Bingliang and Roth, Jack A. and Meric-Bernstam, Funda and Herlyn, Meenhard and Davies, Michael A. and Ding, Li and Li, Shunqiang and Govindan, Ramaswamy and Isella, Claudio and Moscow, Jeffrey A. and Trusolino, Livio and Byrne, Annette T. and Jonkers, Jos and Bult, Carol J. and Medico, Enzo and Chuang, Jeffrey H.},
  doi          = {10.1038/s41588-021-00811-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {761},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Conservation of copy number profiles during engraftment and passaging of patient-derived cancer xenografts},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Enhancer–promoter interactions and
transcription. <em>NG</em>, <em>53</em>(5), 761. (<a
href="https://doi.org/10.1038/s41588-021-00815-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00815-0.},
  archive      = {J_NG},
  author       = {Higgs, Douglas R.},
  doi          = {10.1038/s41588-021-00815-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {761},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Enhancer–promoter interactions and transcription},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Whole-genome resequencing of 445 lactuca accessions reveals
the domestication history of cultivated lettuce. <em>NG</em>,
<em>53</em>(5), 752–760. (<a
href="https://doi.org/10.1038/s41588-021-00831-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lettuce (Lactuca sativa) is an important vegetable crop worldwide. Cultivated lettuce is believed to be domesticated from L. serriola; however, its origins and domestication history remain to be elucidated. Here, we sequenced a total of 445 Lactuca accessions, including major lettuce crop types and wild relative species, and generated a comprehensive map of lettuce genome variations. In-depth analyses of population structure and demography revealed that lettuce was first domesticated near the Caucasus, which was marked by loss of seed shattering. We also identified the genetic architecture of other domestication traits and wild introgressions in major resistance clusters in the lettuce genome. This study provides valuable genomic resources for crop breeding and sheds light on the domestication history of cultivated lettuce. Whole-genome resequencing of 445 Lactuca accessions, including major lettuce crop types and wild relative species, provides a comprehensive map of lettuce genome variations and sheds light on the domestication history of cultivated lettuce.},
  archive      = {J_NG},
  author       = {Wei, Tong and van Treuren, Rob and Liu, Xinjiang and Zhang, Zhaowu and Chen, Jiongjiong and Liu, Yang and Dong, Shanshan and Sun, Peinan and Yang, Ting and Lan, Tianming and Wang, Xiaogang and Xiong, Zhouquan and Liu, Yaqiong and Wei, Jinpu and Lu, Haorong and Han, Shengping and Chen, Jason C. and Ni, Xuemei and Wang, Jian and Yang, Huanming and Xu, Xun and Kuang, Hanhui and van Hintum, Theo and Liu, Xin and Liu, Huan},
  doi          = {10.1038/s41588-021-00831-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {752-760},
  shortjournal = {Nat. Genet.},
  title        = {Whole-genome resequencing of 445 lactuca accessions reveals the domestication history of cultivated lettuce},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Large-scale sequencing of flatfish genomes provides insights
into the polyphyletic origin of their specialized body plan.
<em>NG</em>, <em>53</em>(5), 742–751. (<a
href="https://doi.org/10.1038/s41588-021-00836-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolutionary and genetic origins of the specialized body plan of flatfish are largely unclear. We analyzed the genomes of 11 flatfish species representing 9 of the 14 Pleuronectiforme families and conclude that Pleuronectoidei and Psettodoidei do not form a monophyletic group, suggesting independent origins from different percoid ancestors. Genomic and transcriptomic data indicate that genes related to WNT and retinoic acid pathways, hampered musculature and reduced lipids might have functioned in the evolution of the specialized body plan of Pleuronectoidei. Evolution of Psettodoidei involved similar but not identical genes. Our work provides valuable resources and insights for understanding the genetic origins of the unusual body plan of flatfishes. Analysis of 11 flatfish genomes indicates that Pleuronectoidei and Psettodoidei are not a monophyletic group. Genomic and transcriptomic data suggest that the WNT and RA pathways played a role in the evolution of the specialized body of flatfish.},
  archive      = {J_NG},
  author       = {Lü, Zhenming and Gong, Li and Ren, Yandong and Chen, Yongjiu and Wang, Zhongkai and Liu, Liqin and Li, Haorong and Chen, Xianqing and Li, Zhenzhu and Luo, Hairong and Jiang, Hui and Zeng, Yan and Wang, Yifan and Wang, Kun and Zhang, Chen and Jiang, Haifeng and Wan, Wenting and Qin, Yanli and Zhang, Jianshe and Zhu, Liang and Shi, Wei and He, Shunping and Mao, Bingyu and Wang, Wen and Kong, Xiaoyu and Li, Yongxin},
  doi          = {10.1038/s41588-021-00836-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {742-751},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale sequencing of flatfish genomes provides insights into the polyphyletic origin of their specialized body plan},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Functional and structural basis of extreme conservation in
vertebrate 5′ untranslated regions. <em>NG</em>, <em>53</em>(5),
729–741. (<a href="https://doi.org/10.1038/s41588-021-00830-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The lack of knowledge about extreme conservation in genomes remains a major gap in our understanding of the evolution of gene regulation. Here, we reveal an unexpected role of extremely conserved 5′ untranslated regions (UTRs) in noncanonical translational regulation that is linked to the emergence of essential developmental features in vertebrate species. Endogenous deletion of conserved elements within these 5′ UTRs decreased gene expression, and extremely conserved 5′ UTRs possess cis-regulatory elements that promote cell-type-specific regulation of translation. We further developed in-cell mutate-and-map (icM2), a new methodology that maps RNA structure inside cells. Using icM2, we determined that an extremely conserved 5′ UTR encodes multiple alternative structures and that each single nucleotide within the conserved element maintains the balance of alternative structures important to control the dynamic range of protein expression. These results explain how extreme sequence conservation can lead to RNA-level biological functions encoded in the untranslated regions of vertebrate genomes. Extremely conserved 5′ UTRs act as cis-regulatory elements, playing an unsuspected role in translation regulation. A new in-cell mutational method, icM2, shows that these 5′ UTRs adopt alternative structures that depend on RNA helicase activity.},
  archive      = {J_NG},
  author       = {Byeon, Gun Woo and Cenik, Elif Sarinay and Jiang, Lihua and Tang, Hua and Das, Rhiju and Barna, Maria},
  doi          = {10.1038/s41588-021-00830-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {729-741},
  shortjournal = {Nat. Genet.},
  title        = {Functional and structural basis of extreme conservation in vertebrate 5′ untranslated regions},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ZNF410 represses fetal globin by singular control of CHD4.
<em>NG</em>, <em>53</em>(5), 719–728. (<a
href="https://doi.org/10.1038/s41588-021-00843-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Known fetal hemoglobin (HbF) silencers have potential on-target liabilities for rational β-hemoglobinopathy therapeutic inhibition. Here, through transcription factor (TF) CRISPR screening, we identify zinc-finger protein (ZNF) 410 as an HbF repressor. ZNF410 does not bind directly to the genes encoding γ-globins, but rather its chromatin occupancy is concentrated solely at CHD4, encoding the NuRD nucleosome remodeler, which is itself required for HbF repression. CHD4 has two ZNF410-bound regulatory elements with 27 combined ZNF410 binding motifs constituting unparalleled genomic clusters. These elements completely account for the effects of ZNF410 on fetal globin repression. Knockout of ZNF410 or its mouse homolog Zfp410 reduces CHD4 levels by 60%, enough to substantially de-repress HbF while eluding cellular or organismal toxicity. These studies suggest a potential target for HbF induction for β-hemoglobin disorders with a wide therapeutic index. More broadly, ZNF410 represents a special class of gene regulator, a conserved TF with singular devotion to regulation of a chromatin subcomplex. A CRISPR screen combined with in vivo data identify ZNF410 as an indirect repressor of fetal hemoglobin. ZNF410 binds to and regulates CHD4 expression, which in turn silences fetal hemoglobin.},
  archive      = {J_NG},
  author       = {Vinjamur, Divya S. and Yao, Qiuming and Cole, Mitchel A. and McGuckin, Connor and Ren, Chunyan and Zeng, Jing and Hossain, Mir and Luk, Kevin and Wolfe, Scot A. and Pinello, Luca and Bauer, Daniel E.},
  doi          = {10.1038/s41588-021-00843-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {719-728},
  shortjournal = {Nat. Genet.},
  title        = {ZNF410 represses fetal globin by singular control of CHD4},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Minor intron retention drives clonal hematopoietic disorders
and diverse cancer predisposition. <em>NG</em>, <em>53</em>(5), 707–718.
(<a href="https://doi.org/10.1038/s41588-021-00828-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most eukaryotes harbor two distinct pre-mRNA splicing machineries: the major spliceosome, which removes &amp;gt;99% of introns, and the minor spliceosome, which removes rare, evolutionarily conserved introns. Although hypothesized to serve important regulatory functions, physiologic roles of the minor spliceosome are not well understood. For example, the minor spliceosome component ZRSR2 is subject to recurrent, leukemia-associated mutations, yet functional connections among minor introns, hematopoiesis and cancers are unclear. Here, we identify that impaired minor intron excision via ZRSR2 loss enhances hematopoietic stem cell self-renewal. CRISPR screens mimicking nonsense-mediated decay of minor intron-containing mRNA species converged on LZTR1, a regulator of RAS-related GTPases. LZTR1 minor intron retention was also discovered in the RASopathy Noonan syndrome, due to intronic mutations disrupting splicing and diverse solid tumors. These data uncover minor intron recognition as a regulator of hematopoiesis, noncoding mutations within minor introns as potential cancer drivers and links among ZRSR2 mutations, LZTR1 regulation and leukemias. Loss of function of the minor spliceosome component ZRSR2 enhances hematopoietic stem cell self-renewal through minor intron retention of its target LZTR1, which is a regulator of RAS-related GTPases. Minor intron retention of LZTR1 was also identified in Noonan syndrome and diverse solid tumor types.},
  archive      = {J_NG},
  author       = {Inoue, Daichi and Polaski, Jacob T. and Taylor, Justin and Castel, Pau and Chen, Sisi and Kobayashi, Susumu and Hogg, Simon J. and Hayashi, Yasutaka and Pineda, Jose Mario Bello and El Marabti, Ettaib and Erickson, Caroline and Knorr, Katherine and Fukumoto, Miki and Yamazaki, Hiromi and Tanaka, Atsushi and Fukui, Chie and Lu, Sydney X. and Durham, Benjamin H. and Liu, Bo and Wang, Eric and Mehta, Sanjoy and Zakheim, Daniel and Garippa, Ralph and Penson, Alex and Chew, Guo-Liang and McCormick, Frank and Bradley, Robert K. and Abdel-Wahab, Omar},
  doi          = {10.1038/s41588-021-00828-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {707-718},
  shortjournal = {Nat. Genet.},
  title        = {Minor intron retention drives clonal hematopoietic disorders and diverse cancer predisposition},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell transcriptomics of human embryos identifies
multiple sympathoblast lineages with potential implications for
neuroblastoma origin. <em>NG</em>, <em>53</em>(5), 694–706. (<a
href="https://doi.org/10.1038/s41588-021-00818-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterization of the progression of cellular states during human embryogenesis can provide insights into the origin of pediatric diseases. We examined the transcriptional states of neural crest– and mesoderm-derived lineages differentiating into adrenal glands, kidneys, endothelium and hematopoietic tissue between post-conception weeks 6 and 14 of human development. Our results reveal transitions connecting the intermediate mesoderm and progenitors of organ primordia, the hematopoietic system and endothelial subtypes. Unexpectedly, by using a combination of single-cell transcriptomics and lineage tracing, we found that intra-adrenal sympathoblasts at that stage are directly derived from nerve-associated Schwann cell precursors, similarly to local chromaffin cells, whereas the majority of extra-adrenal sympathoblasts arise from the migratory neural crest. In humans, this process persists during several weeks of development within the large intra-adrenal ganglia-like structures, which may also serve as reservoirs of originating cells in neuroblastoma. Single-cell transcriptome profiling of human embryonic sympathoadrenal tissues identifies developmental transitions and suggests that intra-adrenal sympathoblasts arising from Schwann cell precursors are a potential neuroblastoma cell of origin.},
  archive      = {J_NG},
  author       = {Kameneva, Polina and Artemov, Artem V. and Kastriti, Maria Eleni and Faure, Louis and Olsen, Thale K. and Otte, Jörg and Erickson, Alek and Semsch, Bettina and Andersson, Emma R. and Ratz, Michael and Frisén, Jonas and Tischler, Arthur S. and de Krijger, Ronald R. and Bouderlique, Thibault and Akkuratova, Natalia and Vorontsova, Maria and Gusev, Oleg and Fried, Kaj and Sundström, Erik and Mei, Shenglin and Kogner, Per and Baryawno, Ninib and Kharchenko, Peter V. and Adameyko, Igor},
  doi          = {10.1038/s41588-021-00818-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {694-706},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell transcriptomics of human embryos identifies multiple sympathoblast lineages with potential implications for neuroblastoma origin},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell transcriptomic analyses provide insights into
the developmental origins of neuroblastoma. <em>NG</em>, <em>53</em>(5),
683–693. (<a href="https://doi.org/10.1038/s41588-021-00806-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuroblastoma is a pediatric tumor of the developing sympathetic nervous system. However, the cellular origin of neuroblastoma has yet to be defined. Here we studied the single-cell transcriptomes of neuroblastomas and normal human developing adrenal glands at various stages of embryonic and fetal development. We defined normal differentiation trajectories from Schwann cell precursors over intermediate states to neuroblasts or chromaffin cells and showed that neuroblastomas transcriptionally resemble normal fetal adrenal neuroblasts. Importantly, neuroblastomas with varying clinical phenotypes matched different temporal states along normal neuroblast differentiation trajectories, with the degree of differentiation corresponding to clinical prognosis. Our work highlights the roles of oncogenic MYCN and loss of TFAP2B in blocking differentiation and may provide the basis for designing therapeutic interventions to overcome differentiation blocks. A single-cell transcriptomic analysis of neuroblastomas and healthy adrenal glands defines cell types and lineage trajectories during different developmental stages. Comparisons with the transcriptomes of neuroblastoma cells show that their transcriptomes most closely resemble those of developing neuroblasts of the adrenal gland.},
  archive      = {J_NG},
  author       = {Jansky, Selina and Sharma, Ashwini Kumar and Körber, Verena and Quintero, Andrés and Toprak, Umut H. and Wecht, Elisa M. and Gartlgruber, Moritz and Greco, Alessandro and Chomsky, Elad and Grünewald, Thomas G. P. and Henrich, Kai-Oliver and Tanay, Amos and Herrmann, Carl and Höfer, Thomas and Westermann, Frank},
  doi          = {10.1038/s41588-021-00806-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {683-693},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell transcriptomic analyses provide insights into the developmental origins of neuroblastoma},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Silencing of LINE-1 retrotransposons is a selective
dependency of myeloid leukemia. <em>NG</em>, <em>53</em>(5), 672–682.
(<a href="https://doi.org/10.1038/s41588-021-00829-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposable elements or transposons are major players in genetic variability and genome evolution. Aberrant activation of long interspersed element-1 (LINE-1 or L1) retrotransposons is common in human cancers, yet their tumor-type-specific functions are poorly characterized. We identified MPHOSPH8/MPP8, a component of the human silencing hub (HUSH) complex, as an acute myeloid leukemia (AML)-selective dependency by epigenetic regulator-focused CRISPR screening. Although MPP8 is dispensable for steady-state hematopoiesis, MPP8 loss inhibits AML development by reactivating L1s to induce the DNA damage response and cell cycle exit. Activation of endogenous or ectopic L1s mimics the phenotype of MPP8 loss, whereas blocking retrotransposition abrogates MPP8-deficiency-induced phenotypes. Expression of AML oncogenic mutations promotes L1 suppression, and enhanced L1 silencing is associated with poor prognosis in human AML. Hence, while retrotransposons are commonly recognized for their cancer-promoting functions, we describe a tumor-suppressive role for L1 retrotransposons in myeloid leukemia. MPHOSPH8 loss inhibits acute myeloid leukemia (AML) development by reactivating LINE-1 retrotransposons. LINE-1 suppression is associated with therapy resistance and poor prognosis in patients with AML.},
  archive      = {J_NG},
  author       = {Gu, Zhimin and Liu, Yuxuan and Zhang, Yuannyu and Cao, Hui and Lyu, Junhua and Wang, Xun and Wylie, Annika and Newkirk, Simon J. and Jones, Amanda E. and Lee, Michael and Botten, Giovanni A. and Deng, Mi and Dickerson, Kathryn E. and Zhang, Cheng Cheng and An, Wenfeng and Abrams, John M. and Xu, Jian},
  doi          = {10.1038/s41588-021-00829-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {672-682},
  shortjournal = {Nat. Genet.},
  title        = {Silencing of LINE-1 retrotransposons is a selective dependency of myeloid leukemia},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genetic analyses identify widespread sex-differential
participation bias. <em>NG</em>, <em>53</em>(5), 663–671. (<a
href="https://doi.org/10.1038/s41588-021-00846-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic association results are often interpreted with the assumption that study participation does not affect downstream analyses. Understanding the genetic basis of participation bias is challenging since it requires the genotypes of unseen individuals. Here we demonstrate that it is possible to estimate comparative biases by performing a genome-wide association study contrasting one subgroup versus another. For example, we showed that sex exhibits artifactual autosomal heritability in the presence of sex-differential participation bias. By performing a genome-wide association study of sex in approximately 3.3 million males and females, we identified over 158 autosomal loci spuriously associated with sex and highlighted complex traits underpinning differences in study participation between the sexes. For example, the body mass index–increasing allele at FTO was observed at higher frequency in males compared to females (odds ratio = 1.02, P = 4.4 × 10−36). Finally, we demonstrated how these biases can potentially lead to incorrect inferences in downstream analyses and propose a conceptual framework for addressing such biases. Our findings highlight a new challenge that genetic studies may face as sample sizes continue to grow. Genetic analyses identify widespread sex-differential participation bias in population-based studies and show how this bias can lead to incorrect inferences. These findings highlight new challenges for association studies as sample sizes continue to grow.},
  archive      = {J_NG},
  author       = {Pirastu, Nicola and Cordioli, Mattia and Nandakumar, Priyanka and Mignogna, Gianmarco and Abdellaoui, Abdel and Hollis, Benjamin and Kanai, Masahiro and Rajagopal, Veera M. and Parolo, Pietro Della Briotta and Baya, Nikolas and Carey, Caitlin E. and Karjalainen, Juha and Als, Thomas D. and Van der Zee, Matthijs D. and Day, Felix R. and Ong, Ken K. and Morisaki, Takayuki and de Geus, Eco and Bellocco, Rino and Okada, Yukinori and Børglum, Anders D. and Joshi, Peter and Auton, Adam and Hinds, David and Neale, Benjamin M. and Walters, Raymond K. and Nivard, Michel G. and Perry, John R. B. and Ganna, Andrea},
  doi          = {10.1038/s41588-021-00846-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {663-671},
  shortjournal = {Nat. Genet.},
  title        = {Genetic analyses identify widespread sex-differential participation bias},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Histone acetylation dynamics modulates chromatin
conformation and allele-specific interactions at oncogenic loci.
<em>NG</em>, <em>53</em>(5), 650–662. (<a
href="https://doi.org/10.1038/s41588-021-00842-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In cancer cells, enhancer hijacking mediated by chromosomal alterations and/or increased deposition of acetylated histone H3 lysine 27 (H3K27ac) can support oncogene expression. However, how the chromatin conformation of enhancer–promoter interactions is affected by these events is unclear. In the present study, by comparing chromatin structure and H3K27ac levels in normal and lymphoma B cells, we show that enhancer–promoter-interacting regions assume different conformations according to the local abundance of H3K27ac. Genetic or pharmacological depletion of H3K27ac decreases the frequency and the spreading of these interactions, altering oncogene expression. Moreover, enhancer hijacking mediated by chromosomal translocations influences the epigenetic status of the regions flanking the breakpoint, prompting the formation of distinct intrachromosomal interactions in the two homologous chromosomes. These interactions are accompanied by allele-specific gene expression changes. Overall, our work indicates that H3K27ac dynamics modulates interaction frequency between regulatory regions and can lead to allele-specific chromatin configurations to sustain oncogene expression. Enhancer–promoter three-dimensional interactions at oncogenic loci are modulated by H3K27ac dynamics. Enhancer hijacking mediated by chromosomal translocations leads to distinct chromatin states, intrachromosomal interactions and allele-specific gene expression.},
  archive      = {J_NG},
  author       = {Sungalee, Stephanie and Liu, Yuanlong and Lambuta, Ruxandra A. and Katanayeva, Natalya and Donaldson Collier, Maria and Tavernari, Daniele and Roulland, Sandrine and Ciriello, Giovanni and Oricchio, Elisa},
  doi          = {10.1038/s41588-021-00842-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {650-662},
  shortjournal = {Nat. Genet.},
  title        = {Histone acetylation dynamics modulates chromatin conformation and allele-specific interactions at oncogenic loci},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A genome-wide atlas of co-essential modules assigns function
to uncharacterized genes. <em>NG</em>, <em>53</em>(5), 638–649. (<a
href="https://doi.org/10.1038/s41588-021-00840-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central question in the post-genomic era is how genes interact to form biological pathways. Measurements of gene dependency across hundreds of cell lines have been used to cluster genes into ‘co-essential’ pathways, but this approach has been limited by ubiquitous false positives. In the present study, we develop a statistical method that enables robust identification of gene co-essentiality and yields a genome-wide set of functional modules. This atlas recapitulates diverse pathways and protein complexes, and predicts the functions of 108 uncharacterized genes. Validating top predictions, we show that TMEM189 encodes plasmanylethanolamine desaturase, a key enzyme for plasmalogen synthesis. We also show that C15orf57 encodes a protein that binds the AP2 complex, localizes to clathrin-coated pits and enables efficient transferrin uptake. Finally, we provide an interactive webtool for the community to explore our results, which establish co-essentiality profiling as a powerful resource for biological pathway identification and discovery of new gene functions. A new statistical approach to gene co-essentiality mapping identifies a genome-wide set of functional modules. This analysis predicts the functions of 108 uncharacterized genes.},
  archive      = {J_NG},
  author       = {Wainberg, Michael and Kamber, Roarke A. and Balsubramani, Akshay and Meyers, Robin M. and Sinnott-Armstrong, Nasa and Hornburg, Daniel and Jiang, Lihua and Chan, Joanne and Jian, Ruiqi and Gu, Mingxin and Shcherbina, Anna and Dubreuil, Michael M. and Spees, Kaitlyn and Meuleman, Wouter and Snyder, Michael P. and Bassik, Michael C. and Kundaje, Anshul},
  doi          = {10.1038/s41588-021-00840-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {638-649},
  shortjournal = {Nat. Genet.},
  title        = {A genome-wide atlas of co-essential modules assigns function to uncharacterized genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Uncovering genetic mechanisms of hypertension through
multi-omic analysis of the kidney. <em>NG</em>, <em>53</em>(5), 630–637.
(<a href="https://doi.org/10.1038/s41588-021-00835-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The kidney is an organ of key relevance to blood pressure (BP) regulation, hypertension and antihypertensive treatment. However, genetically mediated renal mechanisms underlying susceptibility to hypertension remain poorly understood. We integrated genotype, gene expression, alternative splicing and DNA methylation profiles of up to 430 human kidneys to characterize the effects of BP index variants from genome-wide association studies (GWASs) on renal transcriptome and epigenome. We uncovered kidney targets for 479 (58.3%) BP-GWAS variants and paired 49 BP-GWAS kidney genes with 210 licensed drugs. Our colocalization and Mendelian randomization analyses identified 179 unique kidney genes with evidence of putatively causal effects on BP. Through Mendelian randomization, we also uncovered effects of BP on renal outcomes commonly affecting patients with hypertension. Collectively, our studies identified genetic variants, kidney genes, molecular mechanisms and biological pathways of key relevance to the genetic regulation of BP and inherited susceptibility to hypertension. Gene expression, alternative splicing and DNA methylation profiles from human kidney samples provide insights into the effects of common variants influencing blood pressure. Mendelian randomization uncovers the effects of blood pressure on renal outcomes.},
  archive      = {J_NG},
  author       = {Eales, James M. and Jiang, Xiao and Xu, Xiaoguang and Saluja, Sushant and Akbarov, Artur and Cano-Gamez, Eddie and McNulty, Michelle T. and Finan, Christopher and Guo, Hui and Wystrychowski, Wojciech and Szulinska, Monika and Thomas, Huw B. and Pramanik, Sanjeev and Chopade, Sandesh and Prestes, Priscilla R. and Wise, Ingrid and Evangelou, Evangelos and Salehi, Mahan and Shakanti, Yusif and Ekholm, Mikael and Denniff, Matthew and Nazgiewicz, Alicja and Eichinger, Felix and Godfrey, Bradley and Antczak, Andrzej and Glyda, Maciej and Król, Robert and Eyre, Stephen and Brown, Jason and Berzuini, Carlo and Bowes, John and Caulfield, Mark and Zukowska-Szczechowska, Ewa and Zywiec, Joanna and Bogdanski, Pawel and Kretzler, Matthias and Woolf, Adrian S. and Talavera, David and Keavney, Bernard and Maffia, Pasquale and Guzik, Tomasz J. and O’Keefe, Raymond T. and Trynka, Gosia and Samani, Nilesh J. and Hingorani, Aroon and Sampson, Matthew G. and Morris, Andrew P. and Charchar, Fadi J. and Tomaszewski, Maciej},
  doi          = {10.1038/s41588-021-00835-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {630-637},
  shortjournal = {Nat. Genet.},
  title        = {Uncovering genetic mechanisms of hypertension through multi-omic analysis of the kidney},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genetic and spatial organization of the unusual chromosomes
of the dinoflagellate symbiodinium microadriaticum. <em>NG</em>,
<em>53</em>(5), 618–629. (<a
href="https://doi.org/10.1038/s41588-021-00841-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dinoflagellates are main primary producers in the oceans, the cause of algal blooms and endosymbionts of marine invertebrates. Much remains to be understood about their biology, including their peculiar crystalline chromosomes. We assembled 94 chromosome-scale scaffolds of the genome of the coral endosymbiont Symbiodinium microadriaticum and analyzed their organization. Genes are enriched towards the ends of chromosomes and are arranged in alternating unidirectional blocks. Some chromosomes are enriched for genes involved in specific biological processes. The chromosomes fold as linear rods and each is composed of a series of structural domains separated by boundaries. Domain boundaries are positioned at sites where transcription of two gene blocks converges and disappear when cells are treated with chemicals that block transcription, indicating correlations between gene orientation, transcription and chromosome folding. The description of the genetic and spatial organization of the S. microadriaticum genome provides a foundation for deeper exploration of the extraordinary biology of dinoflagellates and their chromosomes. Genome assembly of the coral endosymbiont Symbiodinium microadriaticum shows that genes are arranged in alternating unidirectional blocks and are enriched at the ends of chromosomes. Chromosomes are composed of structural domains separated by boundaries that disappear when transcription is blocked.},
  archive      = {J_NG},
  author       = {Nand, Ankita and Zhan, Ye and Salazar, Octavio R. and Aranda, Manuel and Voolstra, Christian R. and Dekker, Job},
  doi          = {10.1038/s41588-021-00841-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {618-629},
  shortjournal = {Nat. Genet.},
  title        = {Genetic and spatial organization of the unusual chromosomes of the dinoflagellate symbiodinium microadriaticum},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Transcription-dependent domain-scale three-dimensional
genome organization in the dinoflagellate breviolum minutum.
<em>NG</em>, <em>53</em>(5), 613–617. (<a
href="https://doi.org/10.1038/s41588-021-00848-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dinoflagellate chromosomes represent a unique evolutionary experiment, as they exist in a permanently condensed, liquid crystalline state; are not packaged by histones; and contain genes organized into tandem gene arrays, with minimal transcriptional regulation. We analyze the three-dimensional genome of Breviolum minutum, and find large topological domains (dinoflagellate topologically associating domains, which we term ‘dinoTADs’) without chromatin loops, which are demarcated by convergent gene array boundaries. Transcriptional inhibition disrupts dinoTADs, implicating transcription-induced supercoiling as the primary topological force in dinoflagellates. Analysis of the three-dimensional architecture of the dinoflagellate Breviolum minutum genome identifies large topological domains (dinoTADs) demarcated by convergent gene array boundaries. Transcriptional inhibition disrupts dinoTADs.},
  archive      = {J_NG},
  author       = {Marinov, Georgi K. and Trevino, Alexandro E. and Xiang, Tingting and Kundaje, Anshul and Grossman, Arthur R. and Greenleaf, William J.},
  doi          = {10.1038/s41588-021-00848-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {613-617},
  shortjournal = {Nat. Genet.},
  title        = {Transcription-dependent domain-scale three-dimensional genome organization in the dinoflagellate breviolum minutum},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Engineering three-dimensional genome folding. <em>NG</em>,
<em>53</em>(5), 602–611. (<a
href="https://doi.org/10.1038/s41588-021-00860-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal genomes are partitioned and folded at various scales that contribute distinctly to nuclear processes. While structural features have been disrupted either globally or at select loci in loss-of-function studies, gain-of-function studies that probe the role of genome architecture have lagged behind. Here we examine recent advances in experimentally creating chromatin loops, contact domains, boundaries and compartments. Furthermore, we explore parallels between this emerging theme and natural evolution of mammalian genomes with increasing architectural complexity. Finally, we provide a perspective on how insights arising from recent gain-of-function studies may inform future endeavors toward engineering the three-dimensional genome. Recent technologies allow experimental manipulation of chromatin conformation. This Perspective discusses the insights obtained from gain-of-function studies that engineer the three-dimensional genome.},
  archive      = {J_NG},
  author       = {Zhang, Di and Lam, Jessica and Blobel, Gerd A.},
  doi          = {10.1038/s41588-021-00860-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {602-611},
  shortjournal = {Nat. Genet.},
  title        = {Engineering three-dimensional genome folding},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Advancing crop genomics from lab to field. <em>NG</em>,
<em>53</em>(5), 595–601. (<a
href="https://doi.org/10.1038/s41588-021-00866-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crop genomics remains a key element in ensuring scientific progress to secure global food security. It has been two decades since the sequence of the first plant genome, that of Arabidopsis thaliana, was released, and soon after that the draft sequencing of the rice genome was completed. Since then, the genomes of more than 100 crops have been sequenced, plant genome research has expanded across multiple fronts and the next few years promise to bring further advances spurred by the advent of new technologies and approaches. We are likely to see continued innovations in crop genome sequencing, genetic mapping and the acquisition of multiple levels of biological data. There will be exciting opportunities to integrate genome-scale information across multiple scales of biological organization, leading to advances in our mechanistic understanding of crop biological processes, which will, in turn, provide greater impetus for translation of laboratory results to the field. This Perspective on crop genomics discusses the promises of new technologies and approaches to help minimize food insecurity and to lay the foundation for sustainable agricultural systems needed to feed the world.},
  archive      = {J_NG},
  author       = {Purugganan, Michael D. and Jackson, Scott A.},
  doi          = {10.1038/s41588-021-00866-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {595-601},
  shortjournal = {Nat. Genet.},
  title        = {Advancing crop genomics from lab to field},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Linking human sympathoadrenal development and neuroblastoma.
<em>NG</em>, <em>53</em>(5), 593–594. (<a
href="https://doi.org/10.1038/s41588-021-00845-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells resembling human fetal adrenal neuroblasts have been identified as major neuroblastoma cancer cells through single-cell mRNA comparison. Tumor risk stratification correlates with the differentiation of neuroblast-like neuroblastoma cells.},
  archive      = {J_NG},
  author       = {Rohrer, Hermann},
  doi          = {10.1038/s41588-021-00845-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {593-594},
  shortjournal = {Nat. Genet.},
  title        = {Linking human sympathoadrenal development and neuroblastoma},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Extreme conservation encodes the structural dynamics and
function of 5′ UTRs. <em>NG</em>, <em>53</em>(5), 591–592. (<a
href="https://doi.org/10.1038/s41588-021-00869-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA is dynamic and can fold into multiple alternative conformations. A new study shows that cells control mRNA translation by regulating the stoichiometry of alternative RNA structures formed from extremely conserved sequences of 5′ untranslated regions and that this control may contribute to embryonic development.},
  archive      = {J_NG},
  author       = {Gupta, Amit and Beaudoin, Jean-Denis},
  doi          = {10.1038/s41588-021-00869-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {591-592},
  shortjournal = {Nat. Genet.},
  title        = {Extreme conservation encodes the structural dynamics and function of 5′ UTRs},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Targeting ZNF410 as a potential β-hemoglobinopathy therapy.
<em>NG</em>, <em>53</em>(5), 589–590. (<a
href="https://doi.org/10.1038/s41588-021-00817-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The nucleosome remodeling and deacetylase (NuRD) complex is a chromatin modifier with a key role in the switch from fetal to adult hemoglobin. In a new study, Vinjamur et al. identify a fetal hemoglobin repressor, ZNF410, which does not directly bind the γ-globin promoter but acts through highly specific regulation of CHD4, a protein subunit of the NuRD complex, thus presenting a potential approach for therapeutic reactivation of fetal hemoglobin.},
  archive      = {J_NG},
  author       = {Tumburu, Laxminath and Thein, Swee Lay},
  doi          = {10.1038/s41588-021-00817-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {589-590},
  shortjournal = {Nat. Genet.},
  title        = {Targeting ZNF410 as a potential β-hemoglobinopathy therapy},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Human–chimpanzee fused cells reveal
cis-regulatory divergence underlying skeletal evolution. <em>NG</em>,
<em>53</em>(4), 587. (<a
href="https://doi.org/10.1038/s41588-021-00849-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00849-4.},
  archive      = {J_NG},
  author       = {Gokhman, David and Agoglia, Rachel M. and Kinnebrew, Maia and Gordon, Wei and Sun, Danqiong and Bajpai, Vivek K. and Naqvi, Sahin and Chen, Coral and Chan, Anthony and Chen, Chider and Petrov, Dmitri A. and Ahituv, Nadav and Zhang, Honghao and Mishina, Yuji and Wysocka, Joanna and Rohatgi, Rajat and Fraser, Hunter B.},
  doi          = {10.1038/s41588-021-00849-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {587},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Human–chimpanzee fused cells reveal cis-regulatory divergence underlying skeletal evolution},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Transcription factor competition at the
γ-globin promoters controls hemoglobin switching. <em>NG</em>,
<em>53</em>(4), 586. (<a
href="https://doi.org/10.1038/s41588-021-00834-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00834-x.},
  archive      = {J_NG},
  author       = {Liu, Nan and Xu, Shuqian and Yao, Qiuming and Zhu, Qian and Kai, Yan and Hsu, Jonathan Y. and Sakon, Phraew and Pinello, Luca and Yuan, Guo-Cheng and Bauer, Daniel E. and Orkin, Stuart H.},
  doi          = {10.1038/s41588-021-00834-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {586},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Transcription factor competition at the γ-globin promoters controls hemoglobin switching},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Genome-wide meta-analysis, fine-mapping
and integrative prioritization implicate new alzheimer’s disease risk
genes. <em>NG</em>, <em>53</em>(4), 585–586. (<a
href="https://doi.org/10.1038/s41588-021-00822-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00822-1.},
  archive      = {J_NG},
  author       = {Schwartzentruber, Jeremy and Cooper, Sarah and Liu, Jimmy Z. and Barrio-Hernandez, Inigo and Bello, Erica and Kumasaka, Natsuhiko and Young, Adam M. H. and Franklin, Robin J. M. and Johnson, Toby and Estrada, Karol and Gaffney, Daniel J. and Beltrao, Pedro and Bassett, Andrew},
  doi          = {10.1038/s41588-021-00822-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {585-586},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Genome-wide meta-analysis, fine-mapping and integrative prioritization implicate new alzheimer’s disease risk genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A high-quality genome assembly highlights rye genomic
characteristics and agronomically important genes. <em>NG</em>,
<em>53</em>(4), 574–584. (<a
href="https://doi.org/10.1038/s41588-021-00808-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rye is a valuable food and forage crop, an important genetic resource for wheat and triticale improvement and an indispensable material for efficient comparative genomic studies in grasses. Here, we sequenced the genome of Weining rye, an elite Chinese rye variety. The assembled contigs (7.74 Gb) accounted for 98.47% of the estimated genome size (7.86 Gb), with 93.67% of the contigs (7.25 Gb) assigned to seven chromosomes. Repetitive elements constituted 90.31% of the assembled genome. Compared to previously sequenced Triticeae genomes, Daniela, Sumaya and Sumana retrotransposons showed strong expansion in rye. Further analyses of the Weining assembly shed new light on genome-wide gene duplications and their impact on starch biosynthesis genes, physical organization of complex prolamin loci, gene expression features underlying early heading trait and putative domestication-associated chromosomal regions and loci in rye. This genome sequence promises to accelerate genomic and breeding studies in rye and related cereal crops. A high-quality genome assembly of Weining rye sheds new light on gene duplications and their effects on starch biosynthesis genes, gene expression features underlying early heading trait and putative domestication-associated chromosomal regions.},
  archive      = {J_NG},
  author       = {Li, Guangwei and Wang, Lijian and Yang, Jianping and He, Hang and Jin, Huaibing and Li, Xuming and Ren, Tianheng and Ren, Zhenglong and Li, Feng and Han, Xue and Zhao, Xiaoge and Dong, Lingli and Li, Yiwen and Song, Zhongping and Yan, Zehong and Zheng, Nannan and Shi, Cuilan and Wang, Zhaohui and Yang, Shuling and Xiong, Zijun and Zhang, Menglan and Sun, Guanghua and Zheng, Xu and Gou, Mingyue and Ji, Changmian and Du, Junkai and Zheng, Hongkun and Doležel, Jaroslav and Deng, Xing Wang and Stein, Nils and Yang, Qinghua and Zhang, Kunpu and Wang, Daowen},
  doi          = {10.1038/s41588-021-00808-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {574-584},
  shortjournal = {Nat. Genet.},
  title        = {A high-quality genome assembly highlights rye genomic characteristics and agronomically important genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chromosome-scale genome assembly provides insights into rye
biology, evolution and agronomic potential. <em>NG</em>, <em>53</em>(4),
564–573. (<a href="https://doi.org/10.1038/s41588-021-00807-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rye (Secale cereale L.) is an exceptionally climate-resilient cereal crop, used extensively to produce improved wheat varieties via introgressive hybridization and possessing the entire repertoire of genes necessary to enable hybrid breeding. Rye is allogamous and only recently domesticated, thus giving cultivated ryes access to a diverse and exploitable wild gene pool. To further enhance the agronomic potential of rye, we produced a chromosome-scale annotated assembly of the 7.9-gigabase rye genome and extensively validated its quality by using a suite of molecular genetic resources. We demonstrate applications of this resource with a broad range of investigations. We present findings on cultivated rye’s incomplete genetic isolation from wild relatives, mechanisms of genome structural evolution, pathogen resistance, low-temperature tolerance, fertility control systems for hybrid breeding and the yield benefits of rye–wheat introgressions. A chromosome-scale genome assembly of rye inbred line ‘Lo7’ provides insights into its incomplete genetic isolation from wild relatives, mechanisms of genome structural evolution and the yield benefits of rye–wheat introgressions.},
  archive      = {J_NG},
  author       = {Rabanus-Wallace, M. Timothy and Hackauf, Bernd and Mascher, Martin and Lux, Thomas and Wicker, Thomas and Gundlach, Heidrun and Baez, Mariana and Houben, Andreas and Mayer, Klaus F. X. and Guo, Liangliang and Poland, Jesse and Pozniak, Curtis J. and Walkowiak, Sean and Melonek, Joanna and Praz, Coraline R. and Schreiber, Mona and Budak, Hikmet and Heuberger, Matthias and Steuernagel, Burkhard and Wulff, Brande and Börner, Andreas and Byrns, Brook and Čížková, Jana and Fowler, D. Brian and Fritz, Allan and Himmelbach, Axel and Kaithakottil, Gemy and Keilwagen, Jens and Keller, Beat and Konkin, David and Larsen, Jamie and Li, Qiang and Myśków, Beata and Padmarasu, Sudharsan and Rawat, Nidhi and Sesiz, Uğur and Biyiklioglu-Kaya, Sezgi and Sharpe, Andy and Šimková, Hana and Small, Ian and Swarbreck, David and Toegelová, Helena and Tsvetkova, Natalia and Voylokov, Anatoly V. and Vrána, Jan and Bauer, Eva and Bolibok-Bragoszewska, Hanna and Doležel, Jaroslav and Hall, Anthony and Jia, Jizeng and Korzun, Viktor and Laroche, André and Ma, Xue-Feng and Ordon, Frank and Özkan, Hakan and Rakoczy-Trojanowska, Monika and Scholz, Uwe and Schulman, Alan H. and Siekmann, Dörthe and Stojałowski, Stefan and Tiwari, Vijay K. and Spannagl, Manuel and Stein, Nils},
  doi          = {10.1038/s41588-021-00807-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {564-573},
  shortjournal = {Nat. Genet.},
  title        = {Chromosome-scale genome assembly provides insights into rye biology, evolution and agronomic potential},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Distinct dynamics and functions of H2AK119ub1 and H3K27me3
in mouse preimplantation embryos. <em>NG</em>, <em>53</em>(4), 551–563.
(<a href="https://doi.org/10.1038/s41588-021-00821-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polycomb repressive complexes 1 and 2 (PRC1/2) maintain transcriptional silencing of developmental genes largely by catalyzing the formation of mono-ubiquitinated histone H2A at lysine 119 (H2AK119ub1) and trimethylated histone H3 at lysine 27 (H3K27me3), respectively. How Polycomb domains are reprogrammed during mammalian preimplantation development remains largely unclear. Here we show that, although H2AK119ub1 and H3K27me3 are highly colocalized in gametes, they undergo differential reprogramming dynamics following fertilization. H3K27me3 maintains thousands of maternally biased domains until the blastocyst stage, whereas maternally biased H2AK119ub1 distribution in zygotes is largely equalized at the two-cell stage. Notably, while maternal PRC2 depletion has a limited effect on global H2AK119ub1 in early embryos, it disrupts allelic H2AK119ub1 at H3K27me3 imprinting loci including Xist. By contrast, acute H2AK119ub1 depletion in zygotes does not affect H3K27me3 imprinting maintenance, at least by the four-cell stage. Importantly, loss of H2AK119ub1, but not H3K27me3, causes premature activation of developmental genes during zygotic genome activation (ZGA) and subsequent embryonic arrest. Thus, our study reveals distinct dynamics and functions of H3K27me3 and H2AK119ub1 in mouse preimplantation embryos. H2AK119ub1 and H3K27me3 have different genome-wide dynamics in mouse preimplantation embryos. Loss of H2AK119ub1, but not H3K27me3, causes premature activation of developmental genes during zygotic genome activation.},
  archive      = {J_NG},
  author       = {Chen, Zhiyuan and Djekidel, Mohamed Nadhir and Zhang, Yi},
  doi          = {10.1038/s41588-021-00821-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {551-563},
  shortjournal = {Nat. Genet.},
  title        = {Distinct dynamics and functions of H2AK119ub1 and H3K27me3 in mouse preimplantation embryos},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). H2AK119ub1 guides maternal inheritance and zygotic
deposition of H3K27me3 in mouse embryos. <em>NG</em>, <em>53</em>(4),
539–550. (<a href="https://doi.org/10.1038/s41588-021-00820-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Parental epigenomes are established during gametogenesis. While they are largely reset after fertilization, broad domains of Polycomb repressive complex 2 (PRC2)-mediated formation of lysine 27–trimethylated histone H3 (H3K27me3) are inherited from oocytes in mice. How maternal H3K27me3 is established and inherited by embryos remains elusive. Here, we show that PRC1-mediated formation of lysine 119–monoubiquititinated histone H2A (H2AK119ub1) confers maternally heritable H3K27me3. Temporal profiling of H2AK119ub1 dynamics revealed that atypically broad H2AK119ub1 domains are established, along with H3K27me3, during oocyte growth. From the two-cell stage, H2AK119ub1 is progressively deposited at typical Polycomb targets and precedes H3K27me3. Reduction of H2AK119ub1 by depletion of Polycomb group ring finger 1 (PCGF1) and PCGF6—essential components of variant PRC1 (vPRC1)—leads to H3K27me3 loss at a subset of genes in oocytes. The gene-selective H3K27me3 deficiency is irreversibly inherited by embryos, causing loss of maternal H3K27me3-dependent imprinting, embryonic sublethality and placental enlargement at term. Collectively, our study unveils preceding dynamics of H2AK119ub1 over H3K27me3 at the maternal-to-zygotic transition, and identifies PCGF1/6–vPRC1 as an essential player in maternal epigenetic inheritance. In early mouse embryos, PRC1-mediated H2AK119ub1 deposition precedes H3K27me3. Deficiency in variant PRC1 reduces H2AK119ub1 and leads to gene-selective loss of H3K27me3 in oocytes, which is inherited by embryos.},
  archive      = {J_NG},
  author       = {Mei, Hailiang and Kozuka, Chisayo and Hayashi, Ryoya and Kumon, Mami and Koseki, Haruhiko and Inoue, Azusa},
  doi          = {10.1038/s41588-021-00820-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {539-550},
  shortjournal = {Nat. Genet.},
  title        = {H2AK119ub1 guides maternal inheritance and zygotic deposition of H3K27me3 in mouse embryos},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A first-generation pediatric cancer dependency map.
<em>NG</em>, <em>53</em>(4), 529–538. (<a
href="https://doi.org/10.1038/s41588-021-00819-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exciting therapeutic targets are emerging from CRISPR-based screens of high mutational-burden adult cancers. A key question, however, is whether functional genomic approaches will yield new targets in pediatric cancers, known for remarkably few mutations, which often encode proteins considered challenging drug targets. To address this, we created a first-generation pediatric cancer dependency map representing 13 pediatric solid and brain tumor types. Eighty-two pediatric cancer cell lines were subjected to genome-scale CRISPR–Cas9 loss-of-function screening to identify genes required for cell survival. In contrast to the finding that pediatric cancers harbor fewer somatic mutations, we found a similar complexity of genetic dependencies in pediatric cancer cell lines compared to that in adult models. Findings from the pediatric cancer dependency map provide preclinical support for ongoing precision medicine clinical trials. The vulnerabilities observed in pediatric cancers were often distinct from those in adult cancer, indicating that repurposing adult oncology drugs will be insufficient to address childhood cancers. A pediatric cancer dependency map generated with genome-scale CRISPR–Cas9 loss-of-function screens in 82 pediatric cancer cell lines highlights genetic dependencies across a range of tumor types.},
  archive      = {J_NG},
  author       = {Dharia, Neekesh V. and Kugener, Guillaume and Guenther, Lillian M. and Malone, Clare F. and Durbin, Adam D. and Hong, Andrew L. and Howard, Thomas P. and Bandopadhayay, Pratiti and Wechsler, Caroline S. and Fung, Iris and Warren, Allison C. and Dempster, Joshua M. and Krill-Burger, John M. and Paolella, Brenton R. and Moh, Phoebe and Jha, Nishant and Tang, Andrew and Montgomery, Philip and Boehm, Jesse S. and Hahn, William C. and Roberts, Charles W. M. and McFarland, James M. and Tsherniak, Aviad and Golub, Todd R. and Vazquez, Francisca and Stegmaier, Kimberly},
  doi          = {10.1038/s41588-021-00819-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {529-538},
  shortjournal = {Nat. Genet.},
  title        = {A first-generation pediatric cancer dependency map},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultraconserved enhancer function does not require perfect
sequence conservation. <em>NG</em>, <em>53</em>(4), 521–528. (<a
href="https://doi.org/10.1038/s41588-021-00812-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultraconserved enhancer sequences show perfect conservation between human and rodent genomes, suggesting that their functions are highly sensitive to mutation. However, current models of enhancer function do not sufficiently explain this extreme evolutionary constraint. We subjected 23 ultraconserved enhancers to different levels of mutagenesis, collectively introducing 1,547 mutations, and examined their activities in transgenic mouse reporter assays. Overall, we find that the regulatory properties of ultraconserved enhancers are robust to mutation. Upon mutagenesis, nearly all (19/23, 83%) still functioned as enhancers at one developmental stage, as did most of those tested again later in development (5/9, 56%). Replacement of endogenous enhancers with mutated alleles in mice corroborated results of transgenic assays, including the functional resilience of ultraconserved enhancers to mutation. Our findings show that the currently known activities of ultraconserved enhancers do not necessarily require the perfect conservation observed in evolution and suggest that additional regulatory or other functions contribute to their sequence constraint. Mutagenesis of 23 ultraconserved enhancers and examination of their activities in transgenic mouse reporter assays show that overall their regulatory properties are robust to mutation. Manipulation of endogenous loci in mice corroborates reporter assay data.},
  archive      = {J_NG},
  author       = {Snetkova, Valentina and Ypsilanti, Athena R. and Akiyama, Jennifer A. and Mannion, Brandon J. and Plajzer-Frick, Ingrid and Novak, Catherine S. and Harrington, Anne N. and Pham, Quan T. and Kato, Momoe and Zhu, Yiwen and Godoy, Janeth and Meky, Eman and Hunter, Riana D. and Shi, Marie and Kvon, Evgeny Z. and Afzal, Veena and Tran, Stella and Rubenstein, John L. R. and Visel, Axel and Pennacchio, Len A. and Dickel, Diane E.},
  doi          = {10.1038/s41588-021-00812-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {521-528},
  shortjournal = {Nat. Genet.},
  title        = {Ultraconserved enhancer function does not require perfect sequence conservation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Transcription factor competition at the γ-globin promoters
controls hemoglobin switching. <em>NG</em>, <em>53</em>(4), 511–520. (<a
href="https://doi.org/10.1038/s41588-021-00798-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {BCL11A, the major regulator of fetal hemoglobin (HbF, α2γ2) level, represses γ-globin expression through direct promoter binding in adult erythroid cells in a switch to adult hemoglobin (HbA, α2β2). To uncover how BCL11A initiates repression, we used CRISPR–Cas9, dCas9, dCas9-KRAB and dCas9-VP64 screens to dissect the γ-globin promoters and identified an activator element near the BCL11A-binding site. Using CUT&amp;amp;RUN and base editing, we demonstrate that a proximal CCAAT box is occupied by the activator NF-Y. BCL11A competes with NF-Y binding through steric hindrance to initiate repression. Occupancy of NF-Y is rapidly established following BCL11A depletion, and precedes γ-globin derepression and locus control region (LCR)–globin loop formation. Our findings reveal that the switch from fetal to adult globin gene expression within the &amp;gt;50-kb β-globin gene cluster is initiated by competition between a stage-selective repressor and a ubiquitous activating factor within a remarkably discrete region of the γ-globin promoters. CRISPR–Cas9 tiled screens of the β-globin gene cluster identify an NF-Y bound activator element at the γ-globin promoter. Binding competition by the transcriptional repressor BCL11A leads to NF-Y eviction and a switch from fetal to adult globin gene expression.},
  archive      = {J_NG},
  author       = {Liu, Nan and Xu, Shuqian and Yao, Qiuming and Zhu, Qian and Kai, Yan and Hsu, Jonathan Y. and Sakon, Phraew and Pinello, Luca and Yuan, Guo-Cheng and Bauer, Daniel E. and Orkin, Stuart H.},
  doi          = {10.1038/s41588-021-00798-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {511-520},
  shortjournal = {Nat. Genet.},
  title        = {Transcription factor competition at the γ-globin promoters controls hemoglobin switching},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Gain-of-function variants in SYK cause immune dysregulation
and systemic inflammation in humans and mice. <em>NG</em>,
<em>53</em>(4), 500–510. (<a
href="https://doi.org/10.1038/s41588-021-00803-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spleen tyrosine kinase (SYK) is a critical immune signaling molecule and therapeutic target. We identified damaging monoallelic SYK variants in six patients with immune deficiency, multi-organ inflammatory disease such as colitis, arthritis and dermatitis, and diffuse large B cell lymphomas. The SYK variants increased phosphorylation and enhanced downstream signaling, indicating gain of function. A knock-in (SYK-Ser544Tyr) mouse model of a patient variant (p.Ser550Tyr) recapitulated aspects of the human disease that could be partially treated with a SYK inhibitor or transplantation of bone marrow from wild-type mice. Our studies demonstrate that SYK gain-of-function variants result in a potentially treatable form of inflammatory disease. Individuals with SYK gain-of-function variants develop immunodeficiency and systemic inflammation, which are recapitulated in a knock-in mouse model. Treatment of these mice with bone marrow transplantation or with a SYK inhibitor ameliorates disease symptoms, highlighting potential therapeutic strategies for patients with SYK mutations.},
  archive      = {J_NG},
  author       = {Wang, Lin and Aschenbrenner, Dominik and Zeng, Zhiyang and Cao, Xiya and Mayr, Daniel and Mehta, Meera and Capitani, Melania and Warner, Neil and Pan, Jie and Wang, Liren and Li, Qi and Zuo, Tao and Cohen-Kedar, Sarit and Lu, Jiawei and Ardy, Rico Chandra and Mulder, Daniel J. and Dissanayake, Dilan and Peng, Kaiyue and Huang, Zhiheng and Li, Xiaoqin and Wang, Yuesheng and Wang, Xiaobing and Li, Shuchao and Bullers, Samuel and Gammage, Anís N. and Warnatz, Klaus and Schiefer, Ana-Iris and Krivan, Gergely and Goda, Vera and Kahr, Walter H. A. and Lemaire, Mathieu and Lu, Chien-Yi and Siddiqui, Iram and Surette, Michael G. and Kotlarz, Daniel and Engelhardt, Karin R. and Griffin, Helen R. and Rottapel, Robert and Decaluwe, Hélène and Laxer, Ronald M. and Proietti, Michele and Hambleton, Sophie and Elcombe, Suzanne and Guo, Cong-Hui and Grimbacher, Bodo and Dotan, Iris and Ng, Siew C. and Freeman, Spencer A. and Snapper, Scott B. and Klein, Christoph and Boztug, Kaan and Huang, Ying and Li, Dali and Uhlig, Holm H. and Muise, Aleixo M.},
  doi          = {10.1038/s41588-021-00803-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {500-510},
  shortjournal = {Nat. Genet.},
  title        = {Gain-of-function variants in SYK cause immune dysregulation and systemic inflammation in humans and mice},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Independence of chromatin conformation and gene regulation
during drosophila dorsoventral patterning. <em>NG</em>, <em>53</em>(4),
487–499. (<a href="https://doi.org/10.1038/s41588-021-00799-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The relationship between chromatin organization and gene regulation remains unclear. While disruption of chromatin domains and domain boundaries can lead to misexpression of developmental genes, acute depletion of regulators of genome organization has a relatively small effect on gene expression. It is therefore uncertain whether gene expression and chromatin state drive chromatin organization or whether changes in chromatin organization facilitate cell-type-specific activation of gene expression. Here, using the dorsoventral patterning of the Drosophila melanogaster embryo as a model system, we provide evidence for the independence of chromatin organization and dorsoventral gene expression. We define tissue-specific enhancers and link them to expression patterns using single-cell RNA-seq. Surprisingly, despite tissue-specific chromatin states and gene expression, chromatin organization is largely maintained across tissues. Our results indicate that tissue-specific chromatin conformation is not necessary for tissue-specific gene expression but rather acts as a scaffold facilitating gene expression when enhancers become active. Chromatin conformation is largely independent of dorsoventral gene expression during early embryonic development in Drosophila. Despite tissue-specific differences in chromatin state and gene expression, three-dimensional chromatin organization is maintained across tissues.},
  archive      = {J_NG},
  author       = {Ing-Simmons, Elizabeth and Vaid, Roshan and Bing, Xin Yang and Levine, Michael and Mannervik, Mattias and Vaquerizas, Juan M.},
  doi          = {10.1038/s41588-021-00799-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {487-499},
  shortjournal = {Nat. Genet.},
  title        = {Independence of chromatin conformation and gene regulation during drosophila dorsoventral patterning},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cis-regulatory chromatin loops arise before TADs and gene
activation, and are independent of cell fate during early drosophila
development. <em>NG</em>, <em>53</em>(4), 477–486. (<a
href="https://doi.org/10.1038/s41588-021-00816-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acquisition of cell fate is thought to rely on the specific interaction of remote cis-regulatory modules (CRMs), for example, enhancers and target promoters. However, the precise interplay between chromatin structure and gene expression is still unclear, particularly within multicellular developing organisms. In the present study, we employ Hi-M, a single-cell spatial genomics approach, to detect CRM–promoter looping interactions within topologically associating domains (TADs) during early Drosophila development. By comparing cis-regulatory loops in alternate cell types, we show that physical proximity does not necessarily instruct transcriptional states. Moreover, multi-way analyses reveal that multiple CRMs spatially coalesce to form hubs. Loops and CRM hubs are established early during development, before the emergence of TADs. Moreover, CRM hubs are formed, in part, via the action of the pioneer transcription factor Zelda and precede transcriptional activation. Our approach provides insight into the role of CRM–promoter interactions in defining transcriptional states, as well as distinct cell types. Single-cell analysis of Drosophila development with Hi-M suggests that physical proximity between regulatory regions does not necessarily instruct transcriptional states. Multi-way analyses identify the existence of regulatory hubs that emerge before topologically associating domains.},
  archive      = {J_NG},
  author       = {Espinola, Sergio Martin and Götz, Markus and Bellec, Maelle and Messina, Olivier and Fiche, Jean-Bernard and Houbron, Christophe and Dejean, Matthieu and Reim, Ingolf and Cardozo Gizzi, Andrés M. and Lagha, Mounia and Nollmann, Marcelo},
  doi          = {10.1038/s41588-021-00816-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {477-486},
  shortjournal = {Nat. Genet.},
  title        = {Cis-regulatory chromatin loops arise before TADs and gene activation, and are independent of cell fate during early drosophila development},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Human–chimpanzee fused cells reveal cis-regulatory
divergence underlying skeletal evolution. <em>NG</em>, <em>53</em>(4),
467–476. (<a href="https://doi.org/10.1038/s41588-021-00804-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene regulatory divergence is thought to play a central role in determining human-specific traits. However, our ability to link divergent regulation to divergent phenotypes is limited. Here, we utilized human–chimpanzee hybrid induced pluripotent stem cells to study gene expression separating these species. The tetraploid hybrid cells allowed us to separate cis- from trans-regulatory effects, and to control for nongenetic confounding factors. We differentiated these cells into cranial neural crest cells, the primary cell type giving rise to the face. We discovered evidence of lineage-specific selection on the hedgehog signaling pathway, including a human-specific sixfold down-regulation of EVC2 (LIMBIN), a key hedgehog gene. Inducing a similar down-regulation of EVC2 substantially reduced hedgehog signaling output. Mice and humans lacking functional EVC2 show striking phenotypic parallels to human–chimpanzee craniofacial differences, suggesting that the regulatory divergence of hedgehog signaling may have contributed to the unique craniofacial morphology of humans. Human–chimpanzee tetraploid fusions serve as a model to study gene expression differences between these species, allowing for separation of cis- from trans-regulatory effects and analysis of unique craniofacial morphologies.},
  archive      = {J_NG},
  author       = {Gokhman, David and Agoglia, Rachel M. and Kinnebrew, Maia and Gordon, Wei and Sun, Danqiong and Bajpai, Vivek K. and Naqvi, Sahin and Chen, Coral and Chan, Anthony and Chen, Chider and Petrov, Dmitri A. and Ahituv, Nadav and Zhang, Honghao and Mishina, Yuji and Wysocka, Joanna and Rohatgi, Rajat and Fraser, Hunter B.},
  doi          = {10.1038/s41588-021-00804-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {467-476},
  shortjournal = {Nat. Genet.},
  title        = {Human–chimpanzee fused cells reveal cis-regulatory divergence underlying skeletal evolution},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell chromatin accessibility identifies pancreatic
islet cell type– and state-specific regulatory programs of diabetes
risk. <em>NG</em>, <em>53</em>(4), 455–466. (<a
href="https://doi.org/10.1038/s41588-021-00823-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nucleus assay for transposase-accessible chromatin using sequencing (snATAC-seq) creates new opportunities to dissect cell type–specific mechanisms of complex diseases. Since pancreatic islets are central to type 2 diabetes (T2D), we profiled 15,298 islet cells by using combinatorial barcoding snATAC-seq and identified 12 clusters, including multiple alpha, beta and delta cell states. We cataloged 228,873 accessible chromatin sites and identified transcription factors underlying lineage- and state-specific regulation. We observed state-specific enrichment of fasting glucose and T2D genome-wide association studies for beta cells and enrichment for other endocrine cell types. At T2D signals localized to islet-accessible chromatin, we prioritized variants with predicted regulatory function and co-accessibility with target genes. A causal T2D variant rs231361 at the KCNQ1 locus had predicted effects on a beta cell enhancer co-accessible with INS and genome editing in embryonic stem cell–derived beta cells affected INS levels. Together our findings demonstrate the power of single-cell epigenomics for interpreting complex disease genetics. Single-cell ATAC-seq analysis of human pancreatic islet cells identifies different cell clusters and transcription factors that underlie lineage- and state-specific regulation and helps prioritize type 2 diabetes risk variants.},
  archive      = {J_NG},
  author       = {Chiou, Joshua and Zeng, Chun and Cheng, Zhang and Han, Jee Yun and Schlichting, Michael and Miller, Michael and Mendez, Robert and Huang, Serina and Wang, Jinzhao and Sui, Yinghui and Deogaygay, Allison and Okino, Mei-Lin and Qiu, Yunjiang and Sun, Ying and Kudtarkar, Parul and Fang, Rongxin and Preissl, Sebastian and Sander, Maike and Gorkin, David U. and Gaulton, Kyle J.},
  doi          = {10.1038/s41588-021-00823-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {455-466},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell chromatin accessibility identifies pancreatic islet cell type– and state-specific regulatory programs of diabetes risk},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identifying loci with different allele frequencies among
cases of eight psychiatric disorders using CC-GWAS. <em>NG</em>,
<em>53</em>(4), 445–454. (<a
href="https://doi.org/10.1038/s41588-021-00787-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Psychiatric disorders are highly genetically correlated, but little research has been conducted on the genetic differences between disorders. We developed a new method (case–case genome-wide association study; CC-GWAS) to test for differences in allele frequency between cases of two disorders using summary statistics from the respective case–control GWAS, transcending current methods that require individual-level data. Simulations and analytical computations confirm that CC-GWAS is well powered with effective control of type I error. We applied CC-GWAS to publicly available summary statistics for schizophrenia, bipolar disorder, major depressive disorder and five other psychiatric disorders. CC-GWAS identified 196 independent case–case loci, including 72 CC-GWAS-specific loci that were not significant at the genome-wide level in the input case–control summary statistics; two of the CC-GWAS-specific loci implicate the genes KLF6 and KLF16 (from the Krüppel-like family of transcription factors), which have been linked to neurite outgrowth and axon regeneration. CC-GWAS loci replicated convincingly in applications to datasets with independent replication data. Identification of the genetic differences between two different disorders has been hampered by a need for individual-level data from cases of both disorders. CC-GWAS enables the comparison of allele frequencies among cases of two disorders using case–control GWAS summary statistics.},
  archive      = {J_NG},
  author       = {Peyrot, Wouter J. and Price, Alkes L.},
  doi          = {10.1038/s41588-021-00787-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {445-454},
  shortjournal = {Nat. Genet.},
  title        = {Identifying loci with different allele frequencies among cases of eight psychiatric disorders using CC-GWAS},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide CRISPR screening identifies TMEM106B as a
proviral host factor for SARS-CoV-2. <em>NG</em>, <em>53</em>(4),
435–444. (<a href="https://doi.org/10.1038/s41588-021-00805-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ongoing COVID-19 pandemic has caused a global economic and health crisis. To identify host factors essential for coronavirus infection, we performed genome-wide functional genetic screens with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and human coronavirus 229E. These screens uncovered virus-specific as well as shared host factors, including TMEM41B and PI3K type 3. We discovered that SARS-CoV-2 requires the lysosomal protein TMEM106B to infect human cell lines and primary lung cells. TMEM106B overexpression enhanced SARS-CoV-2 infection as well as pseudovirus infection, suggesting a role in viral entry. Furthermore, single-cell RNA-sequencing of airway cells from patients with COVID-19 demonstrated that TMEM106B expression correlates with SARS-CoV-2 infection. The present study uncovered a collection of coronavirus host factors that may be exploited to develop drugs against SARS-CoV-2 infection or future zoonotic coronavirus outbreaks. Genome-wide CRISPR screens for proviral host factors of SARS-CoV-2 and HCoV-229E human coronaviruses show that the lysosomal protein TMEM106B is required for SARS-CoV-2 infection.},
  archive      = {J_NG},
  author       = {Baggen, Jim and Persoons, Leentje and Vanstreels, Els and Jansen, Sander and Van Looveren, Dominique and Boeckx, Bram and Geudens, Vincent and De Man, Julie and Jochmans, Dirk and Wauters, Joost and Wauters, Els and Vanaudenaerde, Bart M. and Lambrechts, Diether and Neyts, Johan and Dallmeier, Kai and Thibaut, Hendrik Jan and Jacquemyn, Maarten and Maes, Piet and Daelemans, Dirk},
  doi          = {10.1038/s41588-021-00805-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {435-444},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide CRISPR screening identifies TMEM106B as a proviral host factor for SARS-CoV-2},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The sickening consequences of too much SYK signaling.
<em>NG</em>, <em>53</em>(4), 432–434. (<a
href="https://doi.org/10.1038/s41588-021-00837-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immune responses require a delicate balance: a weak response can cause immunodeficiency, whereas an excessive response can lead to hyperinflammatory disease and hematological malignancy. Because spleen tyrosine kinase has roles in multiple signaling pathways, its gain-of-function alterations in humans cause hypogammaglobulinemia as well as autoinflammation and predisposition to B cell lymphoma.},
  archive      = {J_NG},
  author       = {Aksentijevich, Ivona},
  doi          = {10.1038/s41588-021-00837-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {432-434},
  shortjournal = {Nat. Genet.},
  title        = {The sickening consequences of too much SYK signaling},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Finding hidden treasures in summary statistics from
genome-wide association studies. <em>NG</em>, <em>53</em>(4), 431–432.
(<a href="https://doi.org/10.1038/s41588-021-00824-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Case–case genome-wide association studies (GWAS) within a single genotyped cohort have proven useful in identifying genetic variants explaining different health outcomes, yet they are limited by data availability. A new study by Peyrot and Price proposes a clever statistical method to overcome this problem by inferring case–case GWAS results from a pair of standard case–control GWAS summary statistics that need not be from the same cohort.},
  archive      = {J_NG},
  author       = {Privé, Florian and Zhu, Zhihong and Vilhjalmsson, Bjarni J.},
  doi          = {10.1038/s41588-021-00824-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {431-432},
  shortjournal = {Nat. Genet.},
  title        = {Finding hidden treasures in summary statistics from genome-wide association studies},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultraconservation of enhancers is not ultranecessary.
<em>NG</em>, <em>53</em>(4), 429–430. (<a
href="https://doi.org/10.1038/s41588-021-00839-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stretches of non-coding DNA that have remained identical across millions of years of evolution are typically assumed to have functional regulatory roles that would be compromised by any amount of nucleotide substitution. A new study finds that these ultraconserved regions are more robust to mutagenesis than their level of conservation would suggest.},
  archive      = {J_NG},
  author       = {Pittman, Maureen and Pollard, Katherine S.},
  doi          = {10.1038/s41588-021-00839-6},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {429-430},
  shortjournal = {Nat. Genet.},
  title        = {Ultraconservation of enhancers is not ultranecessary},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The polycomb landscape in mouse development. <em>NG</em>,
<em>53</em>(4), 427–429. (<a
href="https://doi.org/10.1038/s41588-021-00833-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polycomb-group proteins assemble into two primary complexes—Polycomb repressive complex (PRC) 1 and 2—that safeguard cell fate by repressing gene transcription. Two new studies explore the PRC1 landscape during the transition from gametes to embryos in mice, thus providing insight into the intergenerational transmission of epigenetic information and gene regulation dynamics as embryos prepare for gastrulation.},
  archive      = {J_NG},
  author       = {Richard Albert, Julien and Greenberg, Maxim V. C.},
  doi          = {10.1038/s41588-021-00833-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {427-429},
  shortjournal = {Nat. Genet.},
  title        = {The polycomb landscape in mouse development},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chromatin architecture is a flexible foundation for gene
expression. <em>NG</em>, <em>53</em>(4), 426–427. (<a
href="https://doi.org/10.1038/s41588-021-00813-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The structure of chromatin is associated with its function, but precisely how is unclear. New data show that the higher-order architecture of the genome is similar among cell types with widely variant fates and gene expression patterns, thus challenging the view that chromatin domains determine function in the genome.},
  archive      = {J_NG},
  author       = {Misteli, Tom and Finn, Elizabeth H.},
  doi          = {10.1038/s41588-021-00813-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {426-427},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin architecture is a flexible foundation for gene expression},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The polygenic score catalog as an open database for
reproducibility and systematic evaluation. <em>NG</em>, <em>53</em>(4),
420–425. (<a href="https://doi.org/10.1038/s41588-021-00783-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present the Polygenic Score (PGS) Catalog ( https://www.PGSCatalog.org ), an open resource of published scores (including variants, alleles and weights) and consistently curated metadata required for reproducibility and independent applications. The PGS Catalog has capabilities for user deposition, expert curation and programmatic access, thus providing the community with a platform for PGS dissemination, research and translation.},
  archive      = {J_NG},
  author       = {Lambert, Samuel A. and Gil, Laurent and Jupp, Simon and Ritchie, Scott C. and Xu, Yu and Buniello, Annalisa and McMahon, Aoife and Abraham, Gad and Chapman, Michael and Parkinson, Helen and Danesh, John and MacArthur, Jacqueline A. L. and Inouye, Michael},
  doi          = {10.1038/s41588-021-00783-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {420-425},
  shortjournal = {Nat. Genet.},
  title        = {The polygenic score catalog as an open database for reproducibility and systematic evaluation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A resource of targeted mutant mouse lines for 5,061 genes.
<em>NG</em>, <em>53</em>(4), 416–419. (<a
href="https://doi.org/10.1038/s41588-021-00825-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The International Mouse Phenotyping Consortium reports the generation of new mouse mutant strains for more than 5,000 genes, including 2,850 novel null, 2,987 novel conditional-ready and 4,433 novel reporter alleles.},
  archive      = {J_NG},
  author       = {Birling, Marie-Christine and Yoshiki, Atsushi and Adams, David J. and Ayabe, Shinya and Beaudet, Arthur L. and Bottomley, Joanna and Bradley, Allan and Brown, Steve D. M. and Bürger, Antje and Bushell, Wendy and Chiani, Francesco and Chin, Hsian-Jean Genie and Christou, Skevoulla and Codner, Gemma F. and DeMayo, Francesco J. and Dickinson, Mary E. and Doe, Brendan and Donahue, Leah Rae and Fray, Martin D. and Gambadoro, Alessia and Gao, Xiang and Gertsenstein, Marina and Gomez-Segura, Alba and Goodwin, Leslie O. and Heaney, Jason D. and Hérault, Yann and de Angelis, Martin Hrabe and Jiang, Si-Tse and Justice, Monica J. and Kasparek, Petr and King, Ruairidh E. and Kühn, Ralf and Lee, Ho and Lee, Young Jae and Liu, Zhiwei and Lloyd, K. C. Kent and Lorenzo, Isabel and Mallon, Ann-Marie and McKerlie, Colin and Meehan, Terrence F. and Fuentes, Violeta Munoz and Newman, Stuart and Nutter, Lauryl M. J. and Oh, Goo Taeg and Pavlovic, Guillaume and Ramirez-Solis, Ramiro and Rosen, Barry and Ryder, Edward J. and Santos, Luis A. and Schick, Joel and Seavitt, John R. and Sedlacek, Radislav and Seisenberger, Claudia and Seong, Je Kyung and Skarnes, William C. and Sorg, Tania and Steel, Karen P. and Tamura, Masaru and Tocchini-Valentini, Glauco P. and Wang, Chi-Kuang Leo and Wardle-Jones, Hannah and Wattenhofer-Donzé, Marie and Wells, Sara and Wiles, Michael V. and Willis, Brandon J. and Wood, Joshua A. and Wurst, Wolfgang and Xu, Ying and Teboul, Lydia and Murray, Stephen A.},
  doi          = {10.1038/s41588-021-00825-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {416-419},
  shortjournal = {Nat. Genet.},
  title        = {A resource of targeted mutant mouse lines for 5,061 genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Guiding authors to open access. <em>NG</em>, <em>53</em>(4),
415. (<a href="https://doi.org/10.1038/s41588-021-00853-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Guided Open Access is a new publishing option offered at Nature Genetics. Authors can submit once and be simultaneously considered by three journals. Editorial collaboration and a single submission system combine to make the publication process easier and faster.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-021-00853-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {415},
  shortjournal = {Nat. Genet.},
  title        = {Guiding authors to open access},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Publisher correction: Trans-ancestry genome-wide
association meta-analysis of prostate cancer identifies new
susceptibility loci and informs genetic risk prediction. <em>NG</em>,
<em>53</em>(3), 413. (<a
href="https://doi.org/10.1038/s41588-021-00786-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00786-2.},
  archive      = {J_NG},
  author       = {Conti, David V. and Darst, Burcu F. and Moss, Lilit C. and Saunders, Edward J. and Sheng, Xin and Chou, Alisha and Schumacher, Fredrick R. and Olama, Ali Amin Al and Benlloch, Sara and Dadaev, Tokhir and Brook, Mark N. and Sahimi, Ali and Hoffmann, Thomas J. and Takahashi, Atushi and Matsuda, Koichi and Momozawa, Yukihide and Fujita, Masashi and Muir, Kenneth and Lophatananon, Artitaya and Wan, Peggy and Le Marchand, Loic and Wilkens, Lynne R. and Stevens, Victoria L. and Gapstur, Susan M. and Carter, Brian D. and Schleutker, Johanna and Tammela, Teuvo L. J. and Sipeky, Csilla and Auvinen, Anssi and Giles, Graham G. and Southey, Melissa C. and MacInnis, Robert J. and Cybulski, Cezary and Wokołorczyk, Dominika and Lubiński, Jan and Neal, David E. and Donovan, Jenny L. and Hamdy, Freddie C. and Martin, Richard M. and Nordestgaard, Børge G. and Nielsen, Sune F. and Weischer, Maren and Bojesen, Stig E. and Røder, Martin Andreas and Iversen, Peter and Batra, Jyotsna and Chambers, Suzanne and Moya, Leire and Horvath, Lisa and Clements, Judith A. and Tilley, Wayne and Risbridger, Gail P. and Gronberg, Henrik and Aly, Markus and Szulkin, Robert and Eklund, Martin and Nordström, Tobias and Pashayan, Nora and Dunning, Alison M. and Ghoussaini, Maya and Travis, Ruth C. and Key, Tim J. and Riboli, Elio and Park, Jong Y. and Sellers, Thomas A. and Lin, Hui-Yi and Albanes, Demetrius and Weinstein, Stephanie J. and Mucci, Lorelei A. and Giovannucci, Edward and Lindstrom, Sara and Kraft, Peter and Hunter, David J. and Penney, Kathryn L. and Turman, Constance and Tangen, Catherine M. and Goodman, Phyllis J. and Thompson, Ian M. and Hamilton, Robert J. and Fleshner, Neil E. and Finelli, Antonio and Parent, Marie-Élise and Stanford, Janet L. and Ostrander, Elaine A. and Geybels, Milan S. and Koutros, Stella and Freeman, Laura E. Beane and Stampfer, Meir and Wolk, Alicja and Håkansson, Niclas and Andriole, Gerald L. and Hoover, Robert N. and Machiela, Mitchell J. and Sørensen, Karina Dalsgaard and Borre, Michael and Blot, William J. and Zheng, Wei and Yeboah, Edward D. and Mensah, James E. and Lu, Yong-Jie and Zhang, Hong-Wei and Feng, Ninghan and Mao, Xueying and Wu, Yudong and Zhao, Shan-Chao and Sun, Zan and Thibodeau, Stephen N. and McDonnell, Shannon K. and Schaid, Daniel J. and West, Catharine M. L. and Burnet, Neil and Barnett, Gill and Maier, Christiane and Schnoeller, Thomas and Luedeke, Manuel and Kibel, Adam S. and Drake, Bettina F. and Cussenot, Olivier and Cancel-Tassin, Géraldine and Menegaux, Florence and Truong, Thérèse and Koudou, Yves Akoli and John, Esther M. and Grindedal, Eli Marie and Maehle, Lovise and Khaw, Kay-Tee and Ingles, Sue A. and Stern, Mariana C. and Vega, Ana and Gómez-Caamaño, Antonio and Fachal, Laura and Rosenstein, Barry S. and Kerns, Sarah L. and Ostrer, Harry and Teixeira, Manuel R. and Paulo, Paula and Brandão, Andreia and Watya, Stephen and Lubwama, Alexander and Bensen, Jeannette T. and Fontham, Elizabeth T. H. and Mohler, James and Taylor, Jack A. and Kogevinas, Manolis and Llorca, Javier and Castaño-Vinyals, Gemma and Cannon-Albright, Lisa and Teerlink, Craig C. and Huff, Chad D. and Strom, Sara S. and Multigner, Luc and Blanchet, Pascal and Brureau, Laurent and Kaneva, Radka and Slavov, Chavdar and Mitev, Vanio and Leach, Robin J. and Weaver, Brandi and Brenner, Hermann and Cuk, Katarina and Holleczek, Bernd and Saum, Kai-Uwe and Klein, Eric A. and Hsing, Ann W. and Kittles, Rick A. and Murphy, Adam B. and Logothetis, Christopher J. and Kim, Jeri and Neuhausen, Susan L. and Steele, Linda and Ding, Yuan Chun and Isaacs, William B. and Nemesure, Barbara and Hennis, Anselm J. M. and Carpten, John and Pandha, Hardev and Michael, Agnieszka and De Ruyck, Kim and De Meerleer, Gert and Ost, Piet and Xu, Jianfeng and Razack, Azad and Lim, Jasmine and Teo, Soo-Hwang and Newcomb, Lisa F. and Lin, Daniel W. and Fowke, Jay H. and Neslund-Dudas, Christine and Rybicki, Benjamin A. and Gamulin, Marija and Lessel, Davor and Kulis, Tomislav and Usmani, Nawaid and Singhal, Sandeep and Parliament, Matthew and Claessens, Frank and Joniau, Steven and Van den Broeck, Thomas and Gago-Dominguez, Manuela and Castelao, Jose Esteban and Martinez, Maria Elena and Larkin, Samantha and Townsend, Paul A. and Aukim-Hastie, Claire and Bush, William S. and Aldrich, Melinda C. and Crawford, Dana C. and Srivastava, Shiv and Cullen, Jennifer C. and Petrovics, Gyorgy and Casey, Graham and Roobol, Monique J. and Jenster, Guido and van Schaik, Ron H. N. and Hu, Jennifer J. and Sanderson, Maureen and Varma, Rohit and McKean-Cowdin, Roberta and Torres, Mina and Mancuso, Nicholas and Berndt, Sonja I. and Van Den Eeden, Stephen K. and Easton, Douglas F. and Chanock, Stephen J. and Cook, Michael B. and Wiklund, Fredrik and Nakagawa, Hidewaki and Witte, John S. and Eeles, Rosalind A. and Kote-Jarai, Zsofia and Haiman, Christopher A.},
  doi          = {10.1038/s41588-021-00786-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {413},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Mutations disrupting neuritogenesis genes
confer risk for cerebral palsy. <em>NG</em>, <em>53</em>(3), 412. (<a
href="https://doi.org/10.1038/s41588-021-00780-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-021-00780-8.},
  archive      = {J_NG},
  author       = {Jin, Sheng Chih and Lewis, Sara A. and Bakhtiari, Somayeh and Zeng, Xue and Sierant, Michael C. and Shetty, Sheetal and Nordlie, Sandra M. and Elie, Aureliane and Corbett, Mark A. and Norton, Bethany Y. and van Eyk, Clare L. and Haider, Shozeb and Guida, Brandon S. and Magee, Helen and Liu, James and Pastore, Stephen and Vincent, John B. and Brunstrom-Hernandez, Janice and Papavasileiou, Antigone and Fahey, Michael C. and Berry, Jesia G. and Harper, Kelly and Zhou, Chongchen and Zhang, Junhui and Li, Boyang and Zhao, Hongyu and Heim, Jennifer and Webber, Dani L. and Frank, Mahalia S. B. and Xia, Lei and Xu, Yiran and Zhu, Dengna and Zhang, Bohao and Sheth, Amar H. and Knight, James R. and Castaldi, Christopher and Tikhonova, Irina R. and López-Giráldez, Francesc and Keren, Boris and Whalen, Sandra and Buratti, Julien and Doummar, Diane and Cho, Megan and Retterer, Kyle and Millan, Francisca and Wang, Yangong and Waugh, Jeff L. and Rodan, Lance and Cohen, Julie S. and Fatemi, Ali and Lin, Angela E. and Phillips, John P. and Feyma, Timothy and MacLennan, Suzanna C. and Vaughan, Spencer and Crompton, Kylie E. and Reid, Susan M. and Reddihough, Dinah S. and Shang, Qing and Gao, Chao and Novak, Iona and Badawi, Nadia and Wilson, Yana A. and McIntyre, Sarah J. and Mane, Shrikant M. and Wang, Xiaoyang and Amor, David J. and Zarnescu, Daniela C. and Lu, Qiongshi and Xing, Qinghe and Zhu, Changlian and Bilguvar, Kaya and Padilla-Lopez, Sergio and Lifton, Richard P. and Gecz, Jozef and MacLennan, Alastair H. and Kruer, Michael C.},
  doi          = {10.1038/s41588-021-00780-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {412},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Mutations disrupting neuritogenesis genes confer risk for cerebral palsy},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Efficient phasing and imputation of
low-coverage sequencing data using large reference panels. <em>NG</em>,
<em>53</em>(3), 412. (<a
href="https://doi.org/10.1038/s41588-021-00788-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Rubinacci, Simone and Ribeiro, Diogo M. and Hofmeister, Robin J. and Delaneau, Olivier},
  doi          = {10.1038/s41588-021-00788-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {412},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Efficient phasing and imputation of low-coverage sequencing data using large reference panels},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). ArchR is a scalable software package for integrative
single-cell chromatin accessibility analysis. <em>NG</em>,
<em>53</em>(3), 403–411. (<a
href="https://doi.org/10.1038/s41588-021-00790-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The advent of single-cell chromatin accessibility profiling has accelerated the ability to map gene regulatory landscapes but has outpaced the development of scalable software to rapidly extract biological meaning from these data. Here we present a software suite for single-cell analysis of regulatory chromatin in R (ArchR; https://www.archrproject.com/ ) that enables fast and comprehensive analysis of single-cell chromatin accessibility data. ArchR provides an intuitive, user-focused interface for complex single-cell analyses, including doublet removal, single-cell clustering and cell type identification, unified peak set generation, cellular trajectory identification, DNA element-to-gene linkage, transcription factor footprinting, mRNA expression level prediction from chromatin accessibility and multi-omic integration with single-cell RNA sequencing (scRNA-seq). Enabling the analysis of over 1.2 million single cells within 8 h on a standard Unix laptop, ArchR is a comprehensive software suite for end-to-end analysis of single-cell chromatin accessibility that will accelerate the understanding of gene regulation at the resolution of individual cells. ArchR is a software suite that enables efficient and end-to-end analysis of single-cell chromatin accessibility data (scATAC-seq).},
  archive      = {J_NG},
  author       = {Granja, Jeffrey M. and Corces, M. Ryan and Pierce, Sarah E. and Bagdatli, S. Tansu and Choudhry, Hani and Chang, Howard Y. and Greenleaf, William J.},
  doi          = {10.1038/s41588-021-00790-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {403-411},
  shortjournal = {Nat. Genet.},
  title        = {ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Genome-wide meta-analysis, fine-mapping and integrative
prioritization implicate new alzheimer’s disease risk genes.
<em>NG</em>, <em>53</em>(3), 392–402. (<a
href="https://doi.org/10.1038/s41588-020-00776-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have discovered numerous genomic loci associated with Alzheimer’s disease (AD); yet the causal genes and variants are incompletely identified. We performed an updated genome-wide AD meta-analysis, which identified 37 risk loci, including new associations near CCDC6, TSPAN14, NCK2 and SPRED2. Using three SNP-level fine-mapping methods, we identified 21 SNPs with &amp;gt;50% probability each of being causally involved in AD risk and others strongly suggested by functional annotation. We followed this with colocalization analyses across 109 gene expression quantitative trait loci datasets and prioritization of genes by using protein interaction networks and tissue-specific expression. Combining this information into a quantitative score, we found that evidence converged on likely causal genes, including the above four genes, and those at previously discovered AD loci, including BIN1, APH1B, PTK2B, PILRA and CASS4. Genome-wide meta-analysis, fine-mapping and integrative prioritization using expression quantitative trait loci, protein interaction networks and tissue-specific expression implicate new candidate susceptibility genes for Alzheimer’s disease.},
  archive      = {J_NG},
  author       = {Schwartzentruber, Jeremy and Cooper, Sarah and Liu, Jimmy Z. and Barrio-Hernandez, Inigo and Bello, Erica and Kumasaka, Natsuhiko and Young, Adam M. H. and Franklin, Robin J. M. and Johnson, Toby and Estrada, Karol and Gaffney, Daniel J. and Beltrao, Pedro and Bassett, Andrew},
  doi          = {10.1038/s41588-020-00776-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {392-402},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide meta-analysis, fine-mapping and integrative prioritization implicate new alzheimer’s disease risk genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A unique bipartite polycomb signature regulates
stimulus-response transcription during development. <em>NG</em>,
<em>53</em>(3), 379–391. (<a
href="https://doi.org/10.1038/s41588-021-00789-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid cellular responses to environmental stimuli are fundamental for development and maturation. Immediate early genes can be transcriptionally induced within minutes in response to a variety of signals. How their induction levels are regulated and their untimely activation by spurious signals prevented during development is poorly understood. We found that in developing sensory neurons, before perinatal sensory-activity-dependent induction, immediate early genes are embedded into a unique bipartite Polycomb chromatin signature, carrying active H3K27ac on promoters but repressive Ezh2-dependent H3K27me3 on gene bodies. This bipartite signature is widely present in developing cell types, including embryonic stem cells. Polycomb marking of gene bodies inhibits mRNA elongation, dampening productive transcription, while still allowing for fast stimulus-dependent mark removal and bipartite gene induction. We reveal a developmental epigenetic mechanism regulating the rapidity and amplitude of the transcriptional response to relevant stimuli, while preventing inappropriate activation of stimulus-response genes. Immediate early genes in developing sensory neurons have a unique bipartite chromatin signature: H3K27ac on promoters and enhancers, and H3K27me3 on gene bodies. Polycomb marking of gene bodies inhibits productive mRNA elongation.},
  archive      = {J_NG},
  author       = {Kitazawa, Taro and Machlab, Dania and Joshi, Onkar and Maiorano, Nicola and Kohler, Hubertus and Ducret, Sebastien and Kessler, Sandra and Gezelius, Henrik and Soneson, Charlotte and Papasaikas, Panagiotis and López-Bendito, Guillermina and Stadler, Michael B. and Rijli, Filippo M.},
  doi          = {10.1038/s41588-021-00789-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {379-391},
  shortjournal = {Nat. Genet.},
  title        = {A unique bipartite polycomb signature regulates stimulus-response transcription during development},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Liquid chromatin hi-c characterizes compartment-dependent
chromatin interaction dynamics. <em>NG</em>, <em>53</em>(3), 367–378.
(<a href="https://doi.org/10.1038/s41588-021-00784-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nuclear compartmentalization of active and inactive chromatin is thought to occur through microphase separation mediated by interactions between loci of similar type. The nature and dynamics of these interactions are not known. We developed liquid chromatin Hi-C to map the stability of associations between loci. Before fixation and Hi-C, chromosomes are fragmented, which removes strong polymeric constraint, enabling detection of intrinsic locus–locus interaction stabilities. Compartmentalization is stable when fragments are larger than 10–25 kb. Fragmentation of chromatin into pieces smaller than 6 kb leads to gradual loss of genome organization. Lamin-associated domains are most stable, whereas interactions for speckle- and polycomb-associated loci are more dynamic. Cohesin-mediated loops dissolve after fragmentation. Liquid chromatin Hi-C provides a genome-wide view of chromosome interaction dynamics. Liquid chromatin Hi-C maps the intrinsic stability of associations between loci. Lamin-associated domains are most stable, whereas interactions for speckle- and polycomb-associated loci are more dynamic.},
  archive      = {J_NG},
  author       = {Belaghzal, Houda and Borrman, Tyler and Stephens, Andrew D. and Lafontaine, Denis L. and Venev, Sergey V. and Weng, Zhiping and Marko, John F. and Dekker, Job},
  doi          = {10.1038/s41588-021-00784-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {367-378},
  shortjournal = {Nat. Genet.},
  title        = {Liquid chromatin hi-C characterizes compartment-dependent chromatin interaction dynamics},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Base-resolution models of transcription-factor binding
reveal soft motif syntax. <em>NG</em>, <em>53</em>(3), 354–366. (<a
href="https://doi.org/10.1038/s41588-021-00782-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The arrangement (syntax) of transcription factor (TF) binding motifs is an important part of the cis-regulatory code, yet remains elusive. We introduce a deep learning model, BPNet, that uses DNA sequence to predict base-resolution chromatin immunoprecipitation (ChIP)–nexus binding profiles of pluripotency TFs. We develop interpretation tools to learn predictive motif representations and identify soft syntax rules for cooperative TF binding interactions. Strikingly, Nanog preferentially binds with helical periodicity, and TFs often cooperate in a directional manner, which we validate using clustered regularly interspaced short palindromic repeat (CRISPR)-induced point mutations. Our model represents a powerful general approach to uncover the motifs and syntax of cis-regulatory sequences in genomics data. BPNet is an interpretable deep learning tool that predicts transcription-factor binding profiles from DNA sequence at base-pair resolution, enabling the identification of motifs and the regulatory syntax underlying transcription-factor binding.},
  archive      = {J_NG},
  author       = {Avsec, Žiga and Weilert, Melanie and Shrikumar, Avanti and Krueger, Sabrina and Alexandari, Amr and Dalal, Khyati and Fropf, Robin and McAnany, Charles and Gagneur, Julien and Kundaje, Anshul and Zeitlinger, Julia},
  doi          = {10.1038/s41588-021-00782-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {354-366},
  shortjournal = {Nat. Genet.},
  title        = {Base-resolution models of transcription-factor binding reveal soft motif syntax},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comprehensive characterization of protein–protein
interactions perturbed by disease mutations. <em>NG</em>,
<em>53</em>(3), 342–353. (<a
href="https://doi.org/10.1038/s41588-020-00774-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technological and computational advances in genomics and interactomics have made it possible to identify how disease mutations perturb protein–protein interaction (PPI) networks within human cells. Here, we show that disease-associated germline variants are significantly enriched in sequences encoding PPI interfaces compared to variants identified in healthy participants from the projects 1000 Genomes and ExAC. Somatic missense mutations are also significantly enriched in PPI interfaces compared to noninterfaces in 10,861 tumor exomes. We computationally identified 470 putative oncoPPIs in a pan-cancer analysis and demonstrate that oncoPPIs are highly correlated with patient survival and drug resistance/sensitivity. We experimentally validate the network effects of 13 oncoPPIs using a systematic binary interaction assay, and also demonstrate the functional consequences of two of these on tumor cell growth. In summary, this human interactome network framework provides a powerful tool for prioritization of alleles with PPI-perturbing mutations to inform pathobiological mechanism- and genotype-based therapeutic discovery. Human disease mutations affect protein–protein interfaces in a three-dimensional structurally resolved interaction network. Predicted oncoPPIs in cancer correlate with survival and drug sensitivity, and affect growth in vitro, supporting their relevance to disease pathogenesis.},
  archive      = {J_NG},
  author       = {Cheng, Feixiong and Zhao, Junfei and Wang, Yang and Lu, Weiqiang and Liu, Zehui and Zhou, Yadi and Martin, William R. and Wang, Ruisheng and Huang, Jin and Hao, Tong and Yue, Hong and Ma, Jing and Hou, Yuan and Castrillon, Jessica A. and Fang, Jiansong and Lathia, Justin D. and Keri, Ruth A. and Lightstone, Felice C. and Antman, Elliott Marshall and Rabadan, Raul and Hill, David E. and Eng, Charis and Vidal, Marc and Loscalzo, Joseph},
  doi          = {10.1038/s41588-020-00774-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {342-353},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive characterization of protein–protein interactions perturbed by disease mutations},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Multimodal pooled perturb-CITE-seq screens in patient models
define mechanisms of cancer immune evasion. <em>NG</em>, <em>53</em>(3),
332–341. (<a href="https://doi.org/10.1038/s41588-021-00779-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resistance to immune checkpoint inhibitors (ICIs) is a key challenge in cancer therapy. To elucidate underlying mechanisms, we developed Perturb-CITE-sequencing (Perturb-CITE-seq), enabling pooled clustered regularly interspaced short palindromic repeat (CRISPR)–Cas9 perturbations with single-cell transcriptome and protein readouts. In patient-derived melanoma cells and autologous tumor-infiltrating lymphocyte (TIL) co-cultures, we profiled transcriptomes and 20 proteins in ~218,000 cells under ~750 perturbations associated with cancer cell-intrinsic ICI resistance (ICR). We recover known mechanisms of resistance, including defects in the interferon-γ (IFN-γ)–JAK/STAT and antigen-presentation pathways in RNA, protein and perturbation space, and new ones, including loss/downregulation of CD58. Loss of CD58 conferred immune evasion in multiple co-culture models and was downregulated in tumors of melanoma patients with ICR. CD58 protein expression was not induced by IFN-γ signaling, and CD58 loss conferred immune evasion without compromising major histocompatibility complex (MHC) expression, suggesting that it acts orthogonally to known mechanisms of ICR. This work provides a framework for the deciphering of complex mechanisms by large-scale perturbation screens with multimodal, single-cell readouts, and discovers potentially clinically relevant mechanisms of immune evasion. Pooled CRISPR perturbation screens with multimodal RNA and protein single-cell profiling readout (Perturb-CITE-seq) applied to patient-derived melanoma and tumor-infiltrating lymphocyte co-cultures identifies new tumor immune evasion mechanisms.},
  archive      = {J_NG},
  author       = {Frangieh, Chris J. and Melms, Johannes C. and Thakore, Pratiksha I. and Geiger-Schuller, Kathryn R. and Ho, Patricia and Luoma, Adrienne M. and Cleary, Brian and Jerby-Arnon, Livnat and Malu, Shruti and Cuoco, Michael S. and Zhao, Maryann and Ager, Casey R. and Rogava, Meri and Hovey, Lila and Rotem, Asaf and Bernatchez, Chantale and Wucherpfennig, Kai W. and Johnson, Bruce E. and Rozenblatt-Rosen, Orit and Schadendorf, Dirk and Regev, Aviv and Izar, Benjamin},
  doi          = {10.1038/s41588-021-00779-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {332-341},
  shortjournal = {Nat. Genet.},
  title        = {Multimodal pooled perturb-CITE-seq screens in patient models define mechanisms of cancer immune evasion},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Characterizing the molecular regulation of inhibitory immune
checkpoints with multimodal single-cell screens. <em>NG</em>,
<em>53</em>(3), 322–331. (<a
href="https://doi.org/10.1038/s41588-021-00778-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The expression of inhibitory immune checkpoint molecules, such as programmed death-ligand (PD-L)1, is frequently observed in human cancers and can lead to the suppression of T cell–mediated immune responses. Here, we apply expanded CRISPR-compatible (EC)CITE-seq, a technology that combines pooled CRISPR screens with single-cell mRNA and surface protein measurements, to explore the molecular networks that regulate PD-L1 expression. We also develop a computational framework, mixscape, that substantially improves the signal-to-noise ratio in single-cell perturbation screens by identifying and removing confounding sources of variation. Applying these tools, we identify and validate regulators of PD-L1 and leverage our multimodal data to identify both transcriptional and post-transcriptional modes of regulation. Specifically, we discover that the Kelch-like protein KEAP1 and the transcriptional activator NRF2 mediate the upregulation of PD-L1 after interferon (IFN)-γ stimulation. Our results identify a new mechanism for the regulation of immune checkpoints and present a powerful analytical framework for the analysis of multimodal single-cell perturbation screens. ECCITE-seq, which combines pooled CRISPR screens with single-cell mRNA and surface protein measurements, and the computational framework mixscape identify new regulation mechanisms of PD-L1 expression.},
  archive      = {J_NG},
  author       = {Papalexi, Efthymia and Mimitou, Eleni P. and Butler, Andrew W. and Foster, Samantha and Bracken, Bernadette and Mauck, William M. and Wessels, Hans-Hermann and Hao, Yuhan and Yeung, Bertrand Z. and Smibert, Peter and Satija, Rahul},
  doi          = {10.1038/s41588-021-00778-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {322-331},
  shortjournal = {Nat. Genet.},
  title        = {Characterizing the molecular regulation of inhibitory immune checkpoints with multimodal single-cell screens},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identification of rare and common regulatory variants in
pluripotent cells using population-scale transcriptomics. <em>NG</em>,
<em>53</em>(3), 313–321. (<a
href="https://doi.org/10.1038/s41588-021-00800-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Induced pluripotent stem cells (iPSCs) are an established cellular system to study the impact of genetic variants in derived cell types and developmental contexts. However, in their pluripotent state, the disease impact of genetic variants is less well known. Here, we integrate data from 1,367 human iPSC lines to comprehensively map common and rare regulatory variants in human pluripotent cells. Using this population-scale resource, we report hundreds of new colocalization events for human traits specific to iPSCs, and find increased power to identify rare regulatory variants compared with somatic tissues. Finally, we demonstrate how iPSCs enable the identification of causal genes for rare diseases. Integrative data analysis of 1,367 human iPSC lines maps common and rare regulatory variants that colocalize with loci associated with human traits and diseases.},
  archive      = {J_NG},
  author       = {Bonder, Marc Jan and Smail, Craig and Gloudemans, Michael J. and Frésard, Laure and Jakubosky, David and D’Antonio, Matteo and Li, Xin and Ferraro, Nicole M. and Carcamo-Orive, Ivan and Mirauta, Bogdan and Seaton, Daniel D. and Cai, Na and Vakili, Dara and Horta, Danilo and Zhao, Chunli and Zastrow, Diane B. and Bonner, Devon E. and Wheeler, Matthew T. and Kilpinen, Helena and Knowles, Joshua W. and Smith, Erin N. and Frazer, Kelly A. and Montgomery, Stephen B. and Stegle, Oliver},
  doi          = {10.1038/s41588-021-00800-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {313-321},
  shortjournal = {Nat. Genet.},
  title        = {Identification of rare and common regulatory variants in pluripotent cells using population-scale transcriptomics},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Population-scale single-cell RNA-seq profiling across
dopaminergic neuron differentiation. <em>NG</em>, <em>53</em>(3),
304–312. (<a href="https://doi.org/10.1038/s41588-021-00801-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studying the function of common genetic variants in primary human tissues and during development is challenging. To address this, we use an efficient multiplexing strategy to differentiate 215 human induced pluripotent stem cell (iPSC) lines toward a midbrain neural fate, including dopaminergic neurons, and use single-cell RNA sequencing (scRNA-seq) to profile over 1 million cells across three differentiation time points. The proportion of neurons produced by each cell line is highly reproducible and is predictable by robust molecular markers expressed in pluripotent cells. Expression quantitative trait loci (eQTL) were characterized at different stages of neuronal development and in response to rotenone-induced oxidative stress. Of these, 1,284 eQTL colocalize with known neurological trait risk loci, and 46% are not found in the Genotype–Tissue Expression (GTEx) catalog. Our study illustrates how coupling scRNA-seq with long-term iPSC differentiation enables mechanistic studies of human trait-associated genetic variants in otherwise inaccessible cell states. Single-cell RNA-seq analysis of iPSC neural differentiation identifies markers that predict line-to-line differences in cell fate potential and eQTLs that are specific to different stages of differentiation and that overlap with GWAS risk variants for neurological traits.},
  archive      = {J_NG},
  author       = {Jerber, Julie and Seaton, Daniel D. and Cuomo, Anna S. E. and Kumasaka, Natsuhiko and Haldane, James and Steer, Juliette and Patel, Minal and Pearce, Daniel and Andersson, Malin and Bonder, Marc Jan and Mountjoy, Ed and Ghoussaini, Maya and Lancaster, Madeline A. and Marioni, John C. and Merkle, Florian T. and Gaffney, Daniel J. and Stegle, Oliver},
  doi          = {10.1038/s41588-021-00801-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {304-312},
  shortjournal = {Nat. Genet.},
  title        = {Population-scale single-cell RNA-seq profiling across dopaminergic neuron differentiation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome sequencing analysis identifies new loci associated
with lewy body dementia and provides insights into its genetic
architecture. <em>NG</em>, <em>53</em>(3), 294–303. (<a
href="https://doi.org/10.1038/s41588-021-00785-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic basis of Lewy body dementia (LBD) is not well understood. Here, we performed whole-genome sequencing in large cohorts of LBD cases and neurologically healthy controls to study the genetic architecture of this understudied form of dementia, and to generate a resource for the scientific community. Genome-wide association analysis identified five independent risk loci, whereas genome-wide gene-aggregation tests implicated mutations in the gene GBA. Genetic risk scores demonstrate that LBD shares risk profiles and pathways with Alzheimer’s disease and Parkinson’s disease, providing a deeper molecular understanding of the complex genetic architecture of this age-related neurodegenerative condition. Whole-genome sequence analysis identifies five independent risk loci for Lewy body dementia and demonstrates overlapping genetic architecture with Alzheimer’s and Parkinson’s diseases.},
  archive      = {J_NG},
  author       = {Chia, Ruth and Sabir, Marya S. and Bandres-Ciga, Sara and Saez-Atienzar, Sara and Reynolds, Regina H. and Gustavsson, Emil and Walton, Ronald L. and Ahmed, Sarah and Viollet, Coralie and Ding, Jinhui and Makarious, Mary B. and Diez-Fairen, Monica and Portley, Makayla K. and Shah, Zalak and Abramzon, Yevgeniya and Hernandez, Dena G. and Blauwendraat, Cornelis and Stone, David J. and Eicher, John and Parkkinen, Laura and Ansorge, Olaf and Clark, Lorraine and Honig, Lawrence S. and Marder, Karen and Lemstra, Afina and St George-Hyslop, Peter and Londos, Elisabet and Morgan, Kevin and Lashley, Tammaryn and Warner, Thomas T. and Jaunmuktane, Zane and Galasko, Douglas and Santana, Isabel and Tienari, Pentti J. and Myllykangas, Liisa and Oinas, Minna and Cairns, Nigel J. and Morris, John C. and Halliday, Glenda M. and Van Deerlin, Vivianna M. and Trojanowski, John Q. and Grassano, Maurizio and Calvo, Andrea and Mora, Gabriele and Canosa, Antonio and Floris, Gianluca and Bohannan, Ryan C. and Brett, Francesca and Gan-Or, Ziv and Geiger, Joshua T. and Moore, Anni and May, Patrick and Krüger, Rejko and Goldstein, David S. and Lopez, Grisel and Tayebi, Nahid and Sidransky, Ellen and Norcliffe-Kaufmann, Lucy and Palma, Jose-Alberto and Kaufmann, Horacio and Shakkottai, Vikram G. and Perkins, Matthew and Newell, Kathy L. and Gasser, Thomas and Schulte, Claudia and Landi, Francesco and Salvi, Erika and Cusi, Daniele and Masliah, Eliezer and Kim, Ronald C. and Caraway, Chad A. and Monuki, Edwin S. and Brunetti, Maura and Dawson, Ted M. and Rosenthal, Liana S. and Albert, Marilyn S. and Pletnikova, Olga and Troncoso, Juan C. and Flanagan, Margaret E. and Mao, Qinwen and Bigio, Eileen H. and Rodríguez-Rodríguez, Eloy and Infante, Jon and Lage, Carmen and González-Aramburu, Isabel and Sanchez-Juan, Pascual and Ghetti, Bernardino and Keith, Julia and Black, Sandra E. and Masellis, Mario and Rogaeva, Ekaterina and Duyckaerts, Charles and Brice, Alexis and Lesage, Suzanne and Xiromerisiou, Georgia and Barrett, Matthew J. and Tilley, Bension S. and Gentleman, Steve and Logroscino, Giancarlo and Serrano, Geidy E. and Beach, Thomas G. and McKeith, Ian G. and Thomas, Alan J. and Attems, Johannes and Morris, Christopher M. and Palmer, Laura and Love, Seth and Troakes, Claire and Al-Sarraj, Safa and Hodges, Angela K. and Aarsland, Dag and Klein, Gregory and Kaiser, Scott M. and Woltjer, Randy and Pastor, Pau and Bekris, Lynn M. and Leverenz, James B. and Besser, Lilah M. and Kuzma, Amanda and Renton, Alan E. and Goate, Alison and Bennett, David A. and Scherzer, Clemens R. and Morris, Huw R. and Ferrari, Raffaele and Albani, Diego and Pickering-Brown, Stuart and Faber, Kelley and Kukull, Walter A. and Morenas-Rodriguez, Estrella and Lleó, Alberto and Fortea, Juan and Alcolea, Daniel and Clarimon, Jordi and Nalls, Mike A. and Ferrucci, Luigi and Resnick, Susan M. and Tanaka, Toshiko and Foroud, Tatiana M. and Graff-Radford, Neill R. and Wszolek, Zbigniew K. and Ferman, Tanis and Boeve, Bradley F. and Hardy, John A. and Topol, Eric J. and Torkamani, Ali and Singleton, Andrew B. and Ryten, Mina and Dickson, Dennis W. and Chiò, Adriano and Ross, Owen A. and Gibbs, J. Raphael and Dalgard, Clifton L. and Traynor, Bryan J. and Scholz, Sonja W.},
  doi          = {10.1038/s41588-021-00785-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {294-303},
  shortjournal = {Nat. Genet.},
  title        = {Genome sequencing analysis identifies new loci associated with lewy body dementia and provides insights into its genetic architecture},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Mutation load at a mimicry supergene sheds new light on the
evolution of inversion polymorphisms. <em>NG</em>, <em>53</em>(3),
288–293. (<a href="https://doi.org/10.1038/s41588-020-00771-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosomal inversions are ubiquitous in genomes and often coordinate complex phenotypes, such as the covariation of behavior and morphology in many birds, fishes, insects or mammals1–11. However, why and how inversions become associated with polymorphic traits remains obscure. Here we show that despite a strong selective advantage when they form, inversions accumulate recessive deleterious mutations that generate frequency-dependent selection and promote their maintenance at intermediate frequency. Combining genomics and in vivo fitness analyses in a model butterfly for wing-pattern polymorphism, Heliconius numata, we reveal that three ecologically advantageous inversions have built up a heavy mutational load from the sequential accumulation of deleterious mutations and transposable elements. Inversions associate with sharply reduced viability when homozygous, which prevents them from replacing ancestral chromosome arrangements. Our results suggest that other complex polymorphisms, rather than representing adaptations to competing ecological optima, could evolve because chromosomal rearrangements are intrinsically prone to carrying recessive harmful mutations. Chromosomal inversions at the mimicry locus of Heliconius numata butterflies accumulate deleterious variants. This leads to selection against homozygosity, in opposition to positive selection for mimicry, resulting in intermediate allele frequency.},
  archive      = {J_NG},
  author       = {Jay, Paul and Chouteau, Mathieu and Whibley, Annabel and Bastide, Héloïse and Parrinello, Hugues and Llaurens, Violaine and Joron, Mathieu},
  doi          = {10.1038/s41588-020-00771-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {288-293},
  shortjournal = {Nat. Genet.},
  title        = {Mutation load at a mimicry supergene sheds new light on the evolution of inversion polymorphisms},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mammalian SWI/SNF continuously restores local accessibility
to chromatin. <em>NG</em>, <em>53</em>(3), 279–287. (<a
href="https://doi.org/10.1038/s41588-020-00768-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin accessibility is a hallmark of regulatory regions, entails transcription factor (TF) binding and requires nucleosomal reorganization. However, it remains unclear how dynamic this process is. In the present study, we use small-molecule inhibition of the catalytic subunit of the mouse SWI/SNF remodeler complex to show that accessibility and reduced nucleosome presence at TF-binding sites rely on persistent activity of nucleosome remodelers. Within minutes of remodeler inhibition, accessibility and TF binding decrease. Although this is irrespective of TF function, we show that the activating TF OCT4 (POU5F1) exhibits a faster response than the repressive TF REST. Accessibility, nucleosome depletion and gene expression are rapidly restored on inhibitor removal, suggesting that accessible chromatin is regenerated continuously and in a largely cell-autonomous fashion. We postulate that TF binding to chromatin and remodeler-mediated nucleosomal removal do not represent a stable situation, but instead accessible chromatin reflects an average of a dynamic process under continued renewal. Chemical inhibition of the SWI/SNF remodeling complex shows decreased accessibility and transcription factor binding within minutes. These changes are rapidly restored on inhibitor removal suggesting that accessible chromatin is regenerated continuously.},
  archive      = {J_NG},
  author       = {Iurlaro, Mario and Stadler, Michael B. and Masoni, Francesca and Jagani, Zainab and Galli, Giorgio G. and Schübeler, Dirk},
  doi          = {10.1038/s41588-020-00768-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {279-287},
  shortjournal = {Nat. Genet.},
  title        = {Mammalian SWI/SNF continuously restores local accessibility to chromatin},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Acute BAF perturbation causes immediate changes in chromatin
accessibility. <em>NG</em>, <em>53</em>(3), 269–278. (<a
href="https://doi.org/10.1038/s41588-021-00777-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer-associated, loss-of-function mutations in genes encoding subunits of the BRG1/BRM-associated factor (BAF) chromatin-remodeling complexes1–8 often cause drastic chromatin accessibility changes, especially in important regulatory regions9–19. However, it remains unknown how these changes are established over time (for example, immediate consequences or long-term adaptations), and whether they are causative for intracomplex synthetic lethalities, abrogating the formation or activity of BAF complexes9,20–24. In the present study, we use the dTAG system to induce acute degradation of BAF subunits and show that chromatin alterations are established faster than the duration of one cell cycle. Using a pharmacological inhibitor and a chemical degrader of the BAF complex ATPase subunits25,26, we show that maintaining genome accessibility requires constant ATP-dependent remodeling. Completely abolishing BAF complex function by acute degradation of a synthetic lethal subunit in a paralog-deficient background results in an almost complete loss of chromatin accessibility at BAF-controlled sites, especially also at superenhancers, providing a mechanism for intracomplex synthetic lethalities. Acute degradation of the BAF subunit SMARCA4 shows rapid chromatin accessibility changes, thus suggesting that maintenance of genome accessibility requires constant ATP-dependent remodeling.},
  archive      = {J_NG},
  author       = {Schick, Sandra and Grosche, Sarah and Kohl, Katharina Eva and Drpic, Danica and Jaeger, Martin G. and Marella, Nara C. and Imrichova, Hana and Lin, Jung-Ming G. and Hofstätter, Gerald and Schuster, Michael and Rendeiro, André F. and Koren, Anna and Petronczki, Mark and Bock, Christoph and Müller, André C. and Winter, Georg E. and Kubicek, Stefan},
  doi          = {10.1038/s41588-021-00777-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {269-278},
  shortjournal = {Nat. Genet.},
  title        = {Acute BAF perturbation causes immediate changes in chromatin accessibility},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deciphering cis-regulatory grammar with deep learning.
<em>NG</em>, <em>53</em>(3), 266–268. (<a
href="https://doi.org/10.1038/s41588-021-00814-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study builds a novel deep-learning approach to unravel the syntax of transcription-factor binding from high-resolution ChIP–nexus data. In silico simulations lead to experimental validation of complex sequence-based predictions: helical periodicity and directional cooperativity between transcription factors.},
  archive      = {J_NG},
  author       = {Miraldi, Emily R. and Chen, Xiaoting and Weirauch, Matthew T.},
  doi          = {10.1038/s41588-021-00814-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {266-268},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering cis-regulatory grammar with deep learning},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Metamorphosis of a butterfly inversion. <em>NG</em>,
<em>53</em>(3), 264–266. (<a
href="https://doi.org/10.1038/s41588-021-00809-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosomal inversions frequently underlie distinct phenotypic variation. A new study shows that in butterflies, inversion haplotypes accumulate deleterious mutations that prevent fixation in natural populations.},
  archive      = {J_NG},
  author       = {Küpper, Clemens},
  doi          = {10.1038/s41588-021-00809-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {264-266},
  shortjournal = {Nat. Genet.},
  title        = {Metamorphosis of a butterfly inversion},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chromatin openness requires continuous SWI/SNF activity.
<em>NG</em>, <em>53</em>(3), 263–264. (<a
href="https://doi.org/10.1038/s41588-021-00781-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin structure, specifically sites of open or accessible chromatin, regulates transcription-factor binding, thereby determining cell-type-specific gene expression. Two new studies identify a constant requirement for SWI/SNF-complex remodeling to maintain open chromatin. In both studies, acute inhibition or degradation of the BRG1 or BRM ATPase subunits through chemical or genetic methods led to a loss of chromatin accessibility, in some cases affecting transcription-factor binding and altered gene expression.},
  archive      = {J_NG},
  author       = {Hargreaves, Diana C.},
  doi          = {10.1038/s41588-021-00781-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {263-264},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin openness requires continuous SWI/SNF activity},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). The NCI genomic data commons. <em>NG</em>, <em>53</em>(3),
257–262. (<a href="https://doi.org/10.1038/s41588-021-00791-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The National Cancer Institute (NCI) Genomic Data Commons (GDC) contains more than 2.9 petabytes of genomic and associated clinical data from more than 60 NCI-funded and other contributed cancer genomics research projects. The GDC consists of five applications over a common data model and a common application programming interface.},
  archive      = {J_NG},
  author       = {Heath, Allison P. and Ferretti, Vincent and Agrawal, Stuti and An, Maksim and Angelakos, James C. and Arya, Renuka and Bajari, Rosita and Baqar, Bilal and Barnowski, Justin H. B. and Burt, Jeffrey and Catton, Ann and Chan, Brandon F. and Chu, Fay and Cullion, Kim and Davidsen, Tanja and Do, Phuong-My and Dompierre, Christian and Ferguson, Martin L. and Fitzsimons, Michael S. and Ford, Michael and Fukuma, Miyuki and Gaheen, Sharon and Ganji, Gajanan L. and Garcia, Tzintzuni I. and George, Sameera S. and Gerhard, Daniela S. and Gerthoffert, Francois and Gomez, Fauzi and Han, Kang and Hernandez, Kyle M. and Issac, Biju and Jackson, Richard and Jensen, Mark A. and Joshi, Sid and Kadam, Ajinkya and Khurana, Aishmit and Kim, Kyle M. J. and Kraft, Victoria E. and Li, Shenglai and Lichtenberg, Tara M. and Lodato, Janice and Lolla, Laxmi and Martinov, Plamen and Mazzone, Jeffrey A. and Miller, Daniel P. and Miller, Ian and Miller, Joshua S. and Miyauchi, Koji and Murphy, Mark W. and Nullet, Thomas and Ogwara, Rowland O. and Ortuño, Francisco M. and Pedrosa, Jesús and Pham, Phuong L. and Popov, Maxim Y. and Porter, James J. and Powell, Raymond and Rademacher, Karl and Reid, Colin P. and Rich, Samantha and Rogel, Bessie and Sahni, Himanso and Savage, Jeremiah H. and Schmitt, Kyle A. and Simmons, Trevar J. and Sislow, Joseph and Spring, Jonathan and Stein, Lincoln and Sullivan, Sean and Tang, Yajing and Thiagarajan, Mathangi and Troyer, Heather D. and Wang, Chang and Wang, Zhining and West, Bedford L. and Wilmer, Alex and Wilson, Shane and Wu, Kaman and Wysocki, William P. and Xiang, Linda and Yamada, Joseph T. and Yang, Liming and Yu, Christine and Yung, Christina K. and Zenklusen, Jean Claude and Zhang, Junjun and Zhang, Zhenyu and Zhao, Yuanheng and Zubair, Ariz and Staudt, Louis M. and Grossman, Robert L.},
  doi          = {10.1038/s41588-021-00791-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {257-262},
  shortjournal = {Nat. Genet.},
  title        = {The NCI genomic data commons},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genes do not operate in a vacuum, and neither should our
research. <em>NG</em>, <em>53</em>(3), 255–256. (<a
href="https://doi.org/10.1038/s41588-021-00802-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It’s time for a paradigm shift in the scientific enterprise. Our social responsibilities, especially as stakeholders in a field such as genetics, are central to the responsible conduct of research.},
  archive      = {J_NG},
  author       = {Martschenko, Daphne Oluwaseun and Smith, Markia},
  doi          = {10.1038/s41588-021-00802-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {255-256},
  shortjournal = {Nat. Genet.},
  title        = {Genes do not operate in a vacuum, and neither should our research},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Genome-wide association study of
intracranial aneurysms identifies 17 risk loci and genetic overlap with
clinical risk factors. <em>NG</em>, <em>53</em>(2), 254. (<a
href="https://doi.org/10.1038/s41588-020-00760-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Bakker, Mark K. and van der Spek, Rick A. A. and van Rheenen, Wouter and Morel, Sandrine and Bourcier, Romain and Hostettler, Isabel C. and Alg, Varinder S. and van Eijk, Kristel R. and Koido, Masaru and Akiyama, Masato and Terao, Chikashi and Matsuda, Koichi and Walters, Robin G. and Lin, Kuang and Li, Liming and Millwood, Iona Y. and Chen, Zhengming and Rouleau, Guy A. and Zhou, Sirui and Rannikmäe, Kristiina and Sudlow, Cathie L. M. and Houlden, Henry and van den Berg, Leonard H. and Dina, Christian and Naggara, Olivier and Gentric, Jean-Christophe and Shotar, Eimad and Eugène, François and Desal, Hubert and Winsvold, Bendik S. and Børte, Sigrid and Johnsen, Marianne Bakke and Brumpton, Ben M. and Sandvei, Marie Søfteland and Willer, Cristen J. and Hveem, Kristian and Zwart, John-Anker and Verschuren, W. M. Monique and Friedrich, Christoph M. and Hirsch, Sven and Schilling, Sabine and Dauvillier, Jérôme and Martin, Olivier and Jones, Gregory T. and Bown, Matthew J. and Ko, Nerissa U. and Kim, Helen and Coleman, Jonathan R. I. and Breen, Gerome and Zaroff, Jonathan G. and Klijn, Catharina J. M. and Malik, Rainer and Dichgans, Martin and Sargurupremraj, Muralidharan and Tatlisumak, Turgut and Amouyel, Philippe and Debette, Stéphanie and Rinkel, Gabriel J. E. and Worrall, Bradford B. and Pera, Joanna and Slowik, Agnieszka and Gaál-Paavola, Emília I. and Niemelä, Mika and Jääskeläinen, Juha E. and von Und Zu Fraunberg, Mikael and Lindgren, Antti and Broderick, Joseph P. and Werring, David J. and Woo, Daniel and Redon, Richard and Bijlenga, Philippe and Kamatani, Yoichiro and Veldink, Jan H. and Ruigrok, Ynte M.},
  doi          = {10.1038/s41588-020-00760-4},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {254},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Genome-wide association study of intracranial aneurysms identifies 17 risk loci and genetic overlap with clinical risk factors},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Exploring the structural distribution of
genetic variation in SARS-CoV-2 with the COVID-3D online resource.
<em>NG</em>, <em>53</em>(2), 254. (<a
href="https://doi.org/10.1038/s41588-020-00775-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41588-020-00775-x.},
  archive      = {J_NG},
  author       = {Portelli, Stephanie and Olshansky, Moshe and Rodrigues, Carlos H. M. and D’Souza, Elston N. and Myung, Yoochan and Silk, Michael and Alavi, Azadeh and Pires, Douglas E. V. and Ascher, David B.},
  doi          = {10.1038/s41588-020-00775-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {254},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Exploring the structural distribution of genetic variation in SARS-CoV-2 with the COVID-3D online resource},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A quantitative genomics map of rice provides genetic
insights and guides breeding. <em>NG</em>, <em>53</em>(2), 243–253. (<a
href="https://doi.org/10.1038/s41588-020-00769-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extensive allelic variation in agronomically important genes serves as the basis of rice breeding. Here, we present a comprehensive map of rice quantitative trait nucleotides (QTNs) and inferred QTN effects based on eight genome-wide association study cohorts. Population genetic analyses revealed that domestication, local adaptation and heterosis are all associated with QTN allele frequency changes. A genome navigation system, RiceNavi, was developed for QTN pyramiding and breeding route optimization, and implemented in the improvement of a widely cultivated indica variety. This work presents an efficient platform that bridges ever-increasing genomic knowledge and diverse improvement needs in rice. A comprehensive rice quantitative trait nucleotide map provides new genetic insights and serves as the basis for RiceNavi, a tool to optimize breeding schemes.},
  archive      = {J_NG},
  author       = {Wei, Xin and Qiu, Jie and Yong, Kaicheng and Fan, Jiongjiong and Zhang, Qi and Hua, Hua and Liu, Jie and Wang, Qin and Olsen, Kenneth M. and Han, Bin and Huang, Xuehui},
  doi          = {10.1038/s41588-020-00769-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {243-253},
  shortjournal = {Nat. Genet.},
  title        = {A quantitative genomics map of rice provides genetic insights and guides breeding},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Noncoding RNA processing by DIS3 regulates chromosomal
architecture and somatic hypermutation in b cells. <em>NG</em>,
<em>53</em>(2), 230–242. (<a
href="https://doi.org/10.1038/s41588-020-00772-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Noncoding RNAs are exquisitely titrated by the cellular RNA surveillance machinery for regulating diverse biological processes. The RNA exosome, the predominant 3′ RNA exoribonuclease in mammalian cells, is composed of nine core and two catalytic subunits. Here, we developed a mouse model with a conditional allele to study the RNA exosome catalytic subunit DIS3. In DIS3-deficient B cells, integrity of the immunoglobulin heavy chain (Igh) locus in its topologically associating domain is affected, with accumulation of DNA-associated RNAs flanking CTCF-binding elements, decreased CTCF binding to CTCF-binding elements and disorganized cohesin localization. DIS3-deficient B cells also accumulate activation-induced cytidine deaminase–mediated asymmetric nicks, altering somatic hypermutation patterns and increasing microhomology-mediated end-joining DNA repair. Altered mutation patterns and Igh architectural defects in DIS3-deficient B cells lead to decreased class-switch recombination but increased chromosomal translocations. Our observations of DIS3-mediated architectural regulation at the Igh locus are reflected genome wide, thus providing evidence that noncoding RNA processing is an important mechanism for controlling genome organization. DIS3 deficiency alters CTCF and cohesin binding at the Igh locus in B cells. These cells show decreased class-switch recombination but increased chromosomal translocations.},
  archive      = {J_NG},
  author       = {Laffleur, Brice and Lim, Junghyun and Zhang, Wanwei and Chen, Yiyun and Pefanis, Evangelos and Bizarro, Jonathan and Batista, Carolina R. and Wu, Lijing and Economides, Aris N. and Wang, Jiguang and Basu, Uttiya},
  doi          = {10.1038/s41588-020-00772-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {230-242},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding RNA processing by DIS3 regulates chromosomal architecture and somatic hypermutation in b cells},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Metabolic control of DNA methylation in naive pluripotent
cells. <em>NG</em>, <em>53</em>(2), 215–229. (<a
href="https://doi.org/10.1038/s41588-020-00770-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Naive epiblast and embryonic stem cells (ESCs) give rise to all cells of adults. Such developmental plasticity is associated with genome hypomethylation. Here, we show that LIF–Stat3 signaling induces genomic hypomethylation via metabolic reconfiguration. Stat3−/− ESCs show decreased α-ketoglutarate production from glutamine, leading to increased Dnmt3a and Dnmt3b expression and DNA methylation. Notably, genome methylation is dynamically controlled through modulation of α-ketoglutarate availability or Stat3 activation in mitochondria. Alpha-ketoglutarate links metabolism to the epigenome by reducing the expression of Otx2 and its targets Dnmt3a and Dnmt3b. Genetic inactivation of Otx2 or Dnmt3a and Dnmt3b results in genomic hypomethylation even in the absence of active LIF–Stat3. Stat3−/− ESCs show increased methylation at imprinting control regions and altered expression of cognate transcripts. Single-cell analyses of Stat3−/− embryos confirmed the dysregulated expression of Otx2, Dnmt3a and Dnmt3b as well as imprinted genes. Several cancers display Stat3 overactivation and abnormal DNA methylation; therefore, the molecular module that we describe might be exploited under pathological conditions. Stat3 depletion in pluripotent cells decreases α-ketoglutarate and increases the expression of Otx2 and its targets Dnmt3a and Dnmt3b, leading to global DNA hypermethylation.},
  archive      = {J_NG},
  author       = {Betto, Riccardo M. and Diamante, Linda and Perrera, Valentina and Audano, Matteo and Rapelli, Stefania and Lauria, Andrea and Incarnato, Danny and Arboit, Mattia and Pedretti, Silvia and Rigoni, Giovanni and Guerineau, Vincent and Touboul, David and Stirparo, Giuliano Giuseppe and Lohoff, Tim and Boroviak, Thorsten and Grumati, Paolo and Soriano, Maria E. and Nichols, Jennifer and Mitro, Nico and Oliviero, Salvatore and Martello, Graziano},
  doi          = {10.1038/s41588-020-00770-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {215-229},
  shortjournal = {Nat. Genet.},
  title        = {Metabolic control of DNA methylation in naive pluripotent cells},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A novel ACE2 isoform is expressed in human respiratory
epithelia and is upregulated in response to interferons and RNA
respiratory virus infection. <em>NG</em>, <em>53</em>(2), 205–214. (<a
href="https://doi.org/10.1038/s41588-020-00759-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Angiotensin-converting enzyme 2 (ACE2) is the main entry point in airway epithelial cells for SARS-CoV-2. ACE2 binding to the SARS-CoV-2 protein spike triggers viral fusion with the cell plasma membrane, resulting in viral RNA genome delivery into the host. Despite ACE2’s critical role in SARS-CoV-2 infection, full understanding of ACE2 expression, including in response to viral infection, remains unclear. ACE2 was thought to encode five transcripts and one protein of 805 amino acids. In the present study, we identify a novel short isoform of ACE2 expressed in the airway epithelium, the main site of SARS-CoV-2 infection. Short ACE2 is substantially upregulated in response to interferon stimulation and rhinovirus infection, but not SARS-CoV-2 infection. This short isoform lacks SARS-CoV-2 spike high-affinity binding sites and, altogether, our data are consistent with a model where short ACE2 is unlikely to directly contribute to host susceptibility to SARS-CoV-2 infection. A short isoform of the SARS-CoV-2 host receptor ACE2, expressed in human nasal and bronchial respiratory epithelia, is upregulated in response to interferon treatment and rhinovirus infection, but not SARS-CoV-2 infection.},
  archive      = {J_NG},
  author       = {Blume, Cornelia and Jackson, Claire L. and Spalluto, Cosma Mirella and Legebeke, Jelmer and Nazlamova, Liliya and Conforti, Franco and Perotin, Jeanne-Marie and Frank, Martin and Butler, John and Crispin, Max and Coles, Janice and Thompson, James and Ridley, Robert A. and Dean, Lareb S. N. and Loxham, Matthew and Reikine, Stephanie and Azim, Adnan and Tariq, Kamran and Johnston, David A. and Skipp, Paul J. and Djukanovic, Ratko and Baralle, Diana and McCormick, Christopher J. and Davies, Donna E. and Lucas, Jane S. and Wheway, Gabrielle and Mennella, Vito},
  doi          = {10.1038/s41588-020-00759-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {205-214},
  shortjournal = {Nat. Genet.},
  title        = {A novel ACE2 isoform is expressed in human respiratory epithelia and is upregulated in response to interferons and RNA respiratory virus infection},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Tractor uses local ancestry to enable the inclusion of
admixed individuals in GWAS and to boost power. <em>NG</em>,
<em>53</em>(2), 195–204. (<a
href="https://doi.org/10.1038/s41588-020-00766-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Admixed populations are routinely excluded from genomic studies due to concerns over population structure. Here, we present a statistical framework and software package, Tractor, to facilitate the inclusion of admixed individuals in association studies by leveraging local ancestry. We test Tractor with simulated and empirical two-way admixed African–European cohorts. Tractor generates accurate ancestry-specific effect-size estimates and P values, can boost genome-wide association study (GWAS) power and improves the resolution of association signals. Using a local ancestry-aware regression model, we replicate known hits for blood lipids, discover novel hits missed by standard GWAS and localize signals closer to putative causal variants. Tractor is a statistical framework that facilitates the inclusion of admixed individuals in association studies by leveraging local ancestry. Tractor generates accurate ancestry-specific effect-size estimates and improves the resolution of association signals.},
  archive      = {J_NG},
  author       = {Atkinson, Elizabeth G. and Maihofer, Adam X. and Kanai, Masahiro and Martin, Alicia R. and Karczewski, Konrad J. and Santoro, Marcos L. and Ulirsch, Jacob C. and Kamatani, Yoichiro and Okada, Yukinori and Finucane, Hilary K. and Koenen, Karestan C. and Nievergelt, Caroline M. and Daly, Mark J. and Neale, Benjamin M.},
  doi          = {10.1038/s41588-020-00766-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {195-204},
  shortjournal = {Nat. Genet.},
  title        = {Tractor uses local ancestry to enable the inclusion of admixed individuals in GWAS and to boost power},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Genetics of 35 blood and urine biomarkers in the UK
biobank. <em>NG</em>, <em>53</em>(2), 185–194. (<a
href="https://doi.org/10.1038/s41588-020-00757-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clinical laboratory tests are a critical component of the continuum of care. We evaluate the genetic basis of 35 blood and urine laboratory measurements in the UK Biobank (n = 363,228 individuals). We identify 5,794 independent loci associated with at least one trait (p &amp;lt; 5 × 10−9), containing 3,374 fine-mapped associations and additional sets of large-effect (&amp;gt;0.1 s.d.) protein-altering, human leukocyte antigen (HLA) and copy number variant (CNV) associations. Through Mendelian randomization (MR) analysis, we discover 51 causal relationships, including previously known agonistic effects of urate on gout and cystatin C on stroke. Finally, we develop polygenic risk scores (PRSs) for each biomarker and build ‘multi-PRS’ models for diseases using 35 PRSs simultaneously, which improved chronic kidney disease, type 2 diabetes, gout and alcoholic cirrhosis genetic risk stratification in an independent dataset (FinnGen; n = 135,500) relative to single-disease PRSs. Together, our results delineate the genetic basis of biomarkers and their causal influences on diseases and improve genetic risk stratification for common diseases. Genetic analysis of 35 blood and urine laboratory measurements from 363,228 individuals identifies 1,857 loci associated with at least one laboratory value.},
  archive      = {J_NG},
  author       = {Sinnott-Armstrong, Nasa and Tanigawa, Yosuke and Amar, David and Mars, Nina and Benner, Christian and Aguirre, Matthew and Venkataraman, Guhan Ram and Wainberg, Michael and Ollila, Hanna M. and Kiiskinen, Tuomo and Havulinna, Aki S. and Pirruccello, James P. and Qian, Junyang and Shcherbina, Anna and Rodriguez, Fatima and Assimes, Themistocles L. and Agarwala, Vineeta and Tibshirani, Robert and Hastie, Trevor and Ripatti, Samuli and Pritchard, Jonathan K. and Daly, Mark J. and Rivas, Manuel A.},
  doi          = {10.1038/s41588-020-00757-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {185-194},
  shortjournal = {Nat. Genet.},
  title        = {Genetics of 35 blood and urine biomarkers in the UK biobank},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide association analyses of post-traumatic stress
disorder and its symptom subdomains in the million veteran program.
<em>NG</em>, <em>53</em>(2), 174–184. (<a
href="https://doi.org/10.1038/s41588-020-00767-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted genome-wide association analyses of over 250,000 participants of European (EUR) and African (AFR) ancestry from the Million Veteran Program using electronic health record-validated post-traumatic stress disorder (PTSD) diagnosis and quantitative symptom phenotypes. Applying genome-wide multiple testing correction, we identified three significant loci in European case-control analyses and 15 loci in quantitative symptom analyses. Genomic structural equation modeling indicated tight coherence of a PTSD symptom factor that shares genetic variance with a distinct internalizing (mood–anxiety–neuroticism) factor. Partitioned heritability indicated enrichment in several cortical and subcortical regions, and imputed genetically regulated gene expression in these regions was used to identify potential drug repositioning candidates. These results validate the biological coherence of the PTSD syndrome, inform its relationship to comorbid anxiety and depressive disorders and provide new considerations for treatment. Genome-wide association analyses of post-traumatic stress disorder and its symptom subdomains in individuals of European and African ancestry provide insights into its relationship with anxiety and depressive disorders and identify potential candidates for drug repositioning.},
  archive      = {J_NG},
  author       = {Stein, Murray B. and Levey, Daniel F. and Cheng, Zhongshan and Wendt, Frank R. and Harrington, Kelly and Pathak, Gita A. and Cho, Kelly and Quaden, Rachel and Radhakrishnan, Krishnan and Girgenti, Matthew J. and Ho, Yuk-Lam Anne and Posner, Daniel and Aslan, Mihaela and Duman, Ronald S. and Zhao, Hongyu and Polimanti, Renato and Concato, John and Gelernter, Joel},
  doi          = {10.1038/s41588-020-00767-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {174-184},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses of post-traumatic stress disorder and its symptom subdomains in the million veteran program},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide landscape of RNA-binding protein target site
dysregulation reveals a major impact on psychiatric disorder risk.
<em>NG</em>, <em>53</em>(2), 166–173. (<a
href="https://doi.org/10.1038/s41588-020-00761-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the strong genetic basis of psychiatric disorders, the underlying molecular mechanisms are largely unmapped. RNA-binding proteins (RBPs) are responsible for most post-transcriptional regulation, from splicing to translation to localization. RBPs thus act as key gatekeepers of cellular homeostasis, especially in the brain. However, quantifying the pathogenic contribution of noncoding variants impacting RBP target sites is challenging. Here, we leverage a deep learning approach that can accurately predict the RBP target site dysregulation effects of mutations and discover that RBP dysregulation is a principal contributor to psychiatric disorder risk. RBP dysregulation explains a substantial amount of heritability not captured by large-scale molecular quantitative trait loci studies and has a stronger impact than common coding region variants. We share the genome-wide profiles of RBP dysregulation, which we use to identify DDHD2 as a candidate schizophrenia risk gene. This resource provides a new analytical framework to connect the full range of RNA regulation to complex disease. Genome-wide analysis of RNA-binding protein (RBP) target sites identifies a major role for RBP dysregulation in complex psychiatric disorders and implicates DDHD2 as a candidate risk gene for schizophrenia.},
  archive      = {J_NG},
  author       = {Park, Christopher Y. and Zhou, Jian and Wong, Aaron K. and Chen, Kathleen M. and Theesfeld, Chandra L. and Darnell, Robert B. and Troyanskaya, Olga G.},
  doi          = {10.1038/s41588-020-00761-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {166-173},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide landscape of RNA-binding protein target site dysregulation reveals a major impact on psychiatric disorder risk},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Large-scale association analyses identify host factors
influencing human gut microbiome composition. <em>NG</em>,
<em>53</em>(2), 156–165. (<a
href="https://doi.org/10.1038/s41588-020-00763-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To study the effect of host genetics on gut microbiome composition, the MiBioGen consortium curated and analyzed genome-wide genotypes and 16S fecal microbiome data from 18,340 individuals (24 cohorts). Microbial composition showed high variability across cohorts: only 9 of 410 genera were detected in more than 95% of samples. A genome-wide association study of host genetic variation regarding microbial taxa identified 31 loci affecting the microbiome at a genome-wide significant (P &amp;lt; 5 × 10−8) threshold. One locus, the lactase (LCT) gene locus, reached study-wide significance (genome-wide association study signal: P = 1.28 × 10−20), and it showed an age-dependent association with Bifidobacterium abundance. Other associations were suggestive (1.95 × 10−10 &amp;lt; P &amp;lt; 5 × 10−8) but enriched for taxa showing high heritability and for genes expressed in the intestine and brain. A phenome-wide association study and Mendelian randomization identified enrichment of microbiome trait loci in the metabolic, nutrition and environment domains and suggested the microbiome might have causal effects in ulcerative colitis and rheumatoid arthritis. Analysis of human genotypes and 16S microbiome data of 18,473 individuals from 25 cohorts through a genome-wide association study, a phenome-wide association study and Mendelian randomization identifies host genetic and microbial trait associations.},
  archive      = {J_NG},
  author       = {Kurilshikov, Alexander and Medina-Gomez, Carolina and Bacigalupe, Rodrigo and Radjabzadeh, Djawad and Wang, Jun and Demirkan, Ayse and Le Roy, Caroline I. and Raygoza Garay, Juan Antonio and Finnicum, Casey T. and Liu, Xingrong and Zhernakova, Daria V. and Bonder, Marc Jan and Hansen, Tue H. and Frost, Fabian and Rühlemann, Malte C. and Turpin, Williams and Moon, Jee-Young and Kim, Han-Na and Lüll, Kreete and Barkan, Elad and Shah, Shiraz A. and Fornage, Myriam and Szopinska-Tokov, Joanna and Wallen, Zachary D. and Borisevich, Dmitrii and Agreus, Lars and Andreasson, Anna and Bang, Corinna and Bedrani, Larbi and Bell, Jordana T. and Bisgaard, Hans and Boehnke, Michael and Boomsma, Dorret I. and Burk, Robert D. and Claringbould, Annique and Croitoru, Kenneth and Davies, Gareth E. and van Duijn, Cornelia M. and Duijts, Liesbeth and Falony, Gwen and Fu, Jingyuan and van der Graaf, Adriaan and Hansen, Torben and Homuth, Georg and Hughes, David A. and Ijzerman, Richard G. and Jackson, Matthew A. and Jaddoe, Vincent W. V. and Joossens, Marie and Jørgensen, Torben and Keszthelyi, Daniel and Knight, Rob and Laakso, Markku and Laudes, Matthias and Launer, Lenore J. and Lieb, Wolfgang and Lusis, Aldons J. and Masclee, Ad A. M. and Moll, Henriette A. and Mujagic, Zlatan and Qibin, Qi and Rothschild, Daphna and Shin, Hocheol and Sørensen, Søren J. and Steves, Claire J. and Thorsen, Jonathan and Timpson, Nicholas J. and Tito, Raul Y. and Vieira-Silva, Sara and Völker, Uwe and Völzke, Henry and Võsa, Urmo and Wade, Kaitlin H. and Walter, Susanna and Watanabe, Kyoko and Weiss, Stefan and Weiss, Frank U. and Weissbrod, Omer and Westra, Harm-Jan and Willemsen, Gonneke and Payami, Haydeh and Jonkers, Daisy M. A. E. and Arias Vasquez, Alejandro and de Geus, Eco J. C. and Meyer, Katie A. and Stokholm, Jakob and Segal, Eran and Org, Elin and Wijmenga, Cisca and Kim, Hyung-Lae and Kaplan, Robert C. and Spector, Tim D. and Uitterlinden, Andre G. and Rivadeneira, Fernando and Franke, Andre and Lerch, Markus M. and Franke, Lude and Sanna, Serena and D’Amato, Mauro and Pedersen, Oluf and Paterson, Andrew D. and Kraaij, Robert and Raes, Jeroen and Zhernakova, Alexandra},
  doi          = {10.1038/s41588-020-00763-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {156-165},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale association analyses identify host factors influencing human gut microbiome composition},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide association study in 8,956 german individuals
identifies influence of ABO histo-blood groups on gut microbiome.
<em>NG</em>, <em>53</em>(2), 147–155. (<a
href="https://doi.org/10.1038/s41588-020-00747-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The intestinal microbiome is implicated as an important modulating factor in multiple inflammatory1,2, neurologic3 and neoplastic diseases4. Recent genome-wide association studies yielded inconsistent, underpowered and rarely replicated results such that the role of human host genetics as a contributing factor to microbiome assembly and structure remains uncertain5–11. Nevertheless, twin studies clearly suggest host genetics as a driver of microbiome composition11. In a genome-wide association analysis of 8,956 German individuals, we identified 38 genetic loci to be associated with single bacteria and overall microbiome composition. Further analyses confirm the identified associations of ABO histo-blood groups and FUT2 secretor status with Bacteroides and Faecalibacterium spp. Mendelian randomization analysis suggests causative and protective effects of gut microbes, with clade-specific effects on inflammatory bowel disease. This holistic investigative approach of the host, its genetics and its associated microbial communities as a ‘metaorganism’ broaden our understanding of disease etiology, and emphasize the potential for implementing microbiota in disease treatment and management. Genome-wide association analysis of 8,956 German individuals identifies 38 genetic loci associated with single bacteria and overall microbiome composition.},
  archive      = {J_NG},
  author       = {Rühlemann, Malte Christoph and Hermes, Britt Marie and Bang, Corinna and Doms, Shauni and Moitinho-Silva, Lucas and Thingholm, Louise Bruun and Frost, Fabian and Degenhardt, Frauke and Wittig, Michael and Kässens, Jan and Weiss, Frank Ulrich and Peters, Annette and Neuhaus, Klaus and Völker, Uwe and Völzke, Henry and Homuth, Georg and Weiss, Stefan and Grallert, Harald and Laudes, Matthias and Lieb, Wolfgang and Haller, Dirk and Lerch, Markus M. and Baines, John F. and Franke, Andre},
  doi          = {10.1038/s41588-020-00747-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {147-155},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study in 8,956 german individuals identifies influence of ABO histo-blood groups on gut microbiome},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Integrating human brain proteomes with genome-wide
association data implicates new proteins in alzheimer’s disease
pathogenesis. <em>NG</em>, <em>53</em>(2), 143–146. (<a
href="https://doi.org/10.1038/s41588-020-00773-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS) have identified many risk loci for Alzheimer’s disease (AD)1,2, but how these loci confer AD risk is unclear. Here, we aimed to identify loci that confer AD risk through their effects on brain protein abundance to provide new insights into AD pathogenesis. To that end, we integrated AD GWAS results with human brain proteomes to perform a proteome-wide association study (PWAS) of AD, followed by Mendelian randomization and colocalization analysis. We identified 11 genes that are consistent with being causal in AD, acting via their cis-regulated brain protein abundance. Nine replicated in a confirmation PWAS and eight represent new AD risk genes not identified before by AD GWAS. Furthermore, we demonstrated that our results were independent of APOE e4. Together, our findings provide new insights into AD pathogenesis and promising targets for further mechanistic and therapeutic studies. Integrating human brain proteomes with genome-wide association data followed by Mendelian randomization identifies 11 genes with potentially causal roles in Alzheimer’s disease pathogenesis.},
  archive      = {J_NG},
  author       = {Wingo, Aliza P. and Liu, Yue and Gerasimov, Ekaterina S. and Gockley, Jake and Logsdon, Benjamin A. and Duong, Duc M. and Dammer, Eric B. and Robins, Chloe and Beach, Thomas G. and Reiman, Eric M. and Epstein, Michael P. and De Jager, Philip L. and Lah, James J. and Bennett, David A. and Seyfried, Nicholas T. and Levey, Allan I. and Wingo, Thomas S.},
  doi          = {10.1038/s41588-020-00773-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {143-146},
  shortjournal = {Nat. Genet.},
  title        = {Integrating human brain proteomes with genome-wide association data implicates new proteins in alzheimer’s disease pathogenesis},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Common genetic variants and modifiable risk factors underpin
hypertrophic cardiomyopathy susceptibility and expressivity.
<em>NG</em>, <em>53</em>(2), 135–142. (<a
href="https://doi.org/10.1038/s41588-020-00764-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertrophic cardiomyopathy (HCM) is a common, serious, genetic heart disorder. Rare pathogenic variants in sarcomere genes cause HCM, but with unexplained phenotypic heterogeneity. Moreover, most patients do not carry such variants. We report a genome-wide association study of 2,780 cases and 47,486 controls that identified 12 genome-wide-significant susceptibility loci for HCM. Single-nucleotide polymorphism heritability indicated a strong polygenic influence, especially for sarcomere-negative HCM (64% of cases; h2g = 0.34 ± 0.02). A genetic risk score showed substantial influence on the odds of HCM in a validation study, halving the odds in the lowest quintile and doubling them in the highest quintile, and also influenced phenotypic severity in sarcomere variant carriers. Mendelian randomization identified diastolic blood pressure (DBP) as a key modifiable risk factor for sarcomere-negative HCM, with a one standard deviation increase in DBP increasing the HCM risk fourfold. Common variants and modifiable risk factors have important roles in HCM that we suggest will be clinically actionable. Genome-wide association analyses identify 12 susceptibility loci for hypertrophic cardiomyopathy (HCM). A genetic risk score for HCM was associated with disease status in a validation study and influenced phenotypic severity in carriers of risk variants in sarcomere genes.},
  archive      = {J_NG},
  author       = {Harper, Andrew R. and Goel, Anuj and Grace, Christopher and Thomson, Kate L. and Petersen, Steffen E. and Xu, Xiao and Waring, Adam and Ormondroyd, Elizabeth and Kramer, Christopher M. and Ho, Carolyn Y. and Neubauer, Stefan and Tadros, Rafik and Ware, James S. and Bezzina, Connie R. and Farrall, Martin and Watkins, Hugh},
  doi          = {10.1038/s41588-020-00764-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {135-142},
  shortjournal = {Nat. Genet.},
  title        = {Common genetic variants and modifiable risk factors underpin hypertrophic cardiomyopathy susceptibility and expressivity},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Shared genetic pathways contribute to risk of hypertrophic
and dilated cardiomyopathies with opposite directions of effect.
<em>NG</em>, <em>53</em>(2), 128–134. (<a
href="https://doi.org/10.1038/s41588-020-00762-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The heart muscle diseases hypertrophic (HCM) and dilated (DCM) cardiomyopathies are leading causes of sudden death and heart failure in young, otherwise healthy, individuals. We conducted genome-wide association studies and multi-trait analyses in HCM (1,733 cases), DCM (5,521 cases) and nine left ventricular (LV) traits (19,260 UK Biobank participants with structurally normal hearts). We identified 16 loci associated with HCM, 13 with DCM and 23 with LV traits. We show strong genetic correlations between LV traits and cardiomyopathies, with opposing effects in HCM and DCM. Two-sample Mendelian randomization supports a causal association linking increased LV contractility with HCM risk. A polygenic risk score explains a significant portion of phenotypic variability in carriers of HCM-causing rare variants. Our findings thus provide evidence that polygenic risk score may account for variability in Mendelian diseases. More broadly, we provide insights into how genetic pathways may lead to distinct disorders through opposing genetic effects. Genome-wide analyses identify multiple loci associated with hypertrophic cardiomyopathy (HCM), dilated cardiomyopathy (DCM) and left ventricular (LV) traits. Cardiomyopathies exhibit strong genetic correlations with LV traits, with opposing effects in HCM and DCM.},
  archive      = {J_NG},
  author       = {Tadros, Rafik and Francis, Catherine and Xu, Xiao and Vermeer, Alexa M. C. and Harper, Andrew R. and Huurman, Roy and Kelu Bisabu, Ken and Walsh, Roddy and Hoorntje, Edgar T. and te Rijdt, Wouter P. and Buchan, Rachel J. and van Velzen, Hannah G. and van Slegtenhorst, Marjon A. and Vermeulen, Jentien M. and Offerhaus, Joost Allard and Bai, Wenjia and de Marvao, Antonio and Lahrouchi, Najim and Beekman, Leander and Karper, Jacco C. and Veldink, Jan H. and Kayvanpour, Elham and Pantazis, Antonis and Baksi, A. John and Whiffin, Nicola and Mazzarotto, Francesco and Sloane, Geraldine and Suzuki, Hideaki and Schneider-Luftman, Deborah and Elliott, Paul and Richard, Pascale and Ader, Flavie and Villard, Eric and Lichtner, Peter and Meitinger, Thomas and Tanck, Michael W. T. and van Tintelen, J. Peter and Thain, Andrew and McCarty, David and Hegele, Robert A. and Roberts, Jason D. and Amyot, Julie and Dubé, Marie-Pierre and Cadrin-Tourigny, Julia and Giraldeau, Geneviève and L’Allier, Philippe L. and Garceau, Patrick and Tardif, Jean-Claude and Boekholdt, S. Matthijs and Lumbers, R. Thomas and Asselbergs, Folkert W. and Barton, Paul J. R. and Cook, Stuart A. and Prasad, Sanjay K. and O’Regan, Declan P. and van der Velden, Jolanda and Verweij, Karin J. H. and Talajic, Mario and Lettre, Guillaume and Pinto, Yigal M. and Meder, Benjamin and Charron, Philippe and de Boer, Rudolf A. and Christiaans, Imke and Michels, Michelle and Wilde, Arthur A. M. and Watkins, Hugh and Matthews, Paul M. and Ware, James S. and Bezzina, Connie R.},
  doi          = {10.1038/s41588-020-00762-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {128-134},
  shortjournal = {Nat. Genet.},
  title        = {Shared genetic pathways contribute to risk of hypertrophic and dilated cardiomyopathies with opposite directions of effect},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). X-linked inflammatory syndrome. <em>NG</em>, <em>53</em>(2),
127. (<a href="https://doi.org/10.1038/s41588-021-00792-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-021-00792-4},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {127},
  shortjournal = {Nat. Genet.},
  title        = {X-linked inflammatory syndrome},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cancer exceptional responders. <em>NG</em>, <em>53</em>(2),
127. (<a href="https://doi.org/10.1038/s41588-021-00793-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-021-00793-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {127},
  shortjournal = {Nat. Genet.},
  title        = {Cancer exceptional responders},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chromothripsis and drug resistance. <em>NG</em>,
<em>53</em>(2), 127. (<a
href="https://doi.org/10.1038/s41588-021-00794-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-021-00794-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {127},
  shortjournal = {Nat. Genet.},
  title        = {Chromothripsis and drug resistance},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Perturbing autism risk genes. <em>NG</em>, <em>53</em>(2),
127. (<a href="https://doi.org/10.1038/s41588-021-00795-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-021-00795-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {127},
  shortjournal = {Nat. Genet.},
  title        = {Perturbing autism risk genes},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Adaptation to nitrogen. <em>NG</em>, <em>53</em>(2), 127.
(<a href="https://doi.org/10.1038/s41588-021-00796-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-021-00796-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {127},
  shortjournal = {Nat. Genet.},
  title        = {Adaptation to nitrogen},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Efficient phasing and imputation of low-coverage sequencing
data using large reference panels. <em>NG</em>, <em>53</em>(1), 120–126.
(<a href="https://doi.org/10.1038/s41588-020-00756-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Low-coverage whole-genome sequencing followed by imputation has been proposed as a cost-effective genotyping approach for disease and population genetics studies. However, its competitiveness against SNP arrays is undermined because current imputation methods are computationally expensive and unable to leverage large reference panels. Here, we describe a method, GLIMPSE, for phasing and imputation of low-coverage sequencing datasets from modern reference panels. We demonstrate its remarkable performance across different coverages and human populations. GLIMPSE achieves imputation of a genome for less than US$1 in computational cost, considerably outperforming other methods and improving imputation accuracy over the full allele frequency range. As a proof of concept, we show that 1× coverage enables effective gene expression association studies and outperforms dense SNP arrays in rare variant burden tests. Overall, this study illustrates the promising potential of low-coverage imputation and suggests a paradigm shift in the design of future genomic studies. GLIMPSE is a new method for haplotype phasing and genotype imputation of low-coverage sequencing datasets from large reference panels. GLIMPSE shows remarkable performance across different coverages and human populations.},
  archive      = {J_NG},
  author       = {Rubinacci, Simone and Ribeiro, Diogo M. and Hofmeister, Robin J. and Delaneau, Olivier},
  doi          = {10.1038/s41588-020-00756-0},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {120-126},
  shortjournal = {Nat. Genet.},
  title        = {Efficient phasing and imputation of low-coverage sequencing data using large reference panels},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Promoter-interacting expression quantitative trait loci are
enriched for functional genetic variants. <em>NG</em>, <em>53</em>(1),
110–119. (<a href="https://doi.org/10.1038/s41588-020-00745-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expression quantitative trait loci (eQTLs) studies provide associations of genetic variants with gene expression but fall short of pinpointing functionally important eQTLs. Here, using H3K27ac HiChIP assays, we mapped eQTLs overlapping active cis-regulatory elements that interact with their target gene promoters (promoter-interacting eQTLs, pieQTLs) in five common immune cell types (Database of Immune Cell Expression, Expression quantitative trait loci and Epigenomics (DICE) cis-interactome project). This approach allowed us to identify functionally important eQTLs and show mechanisms that explain their cell-type restriction. We also devised an approach to eQTL discovery that relies on HiChIP-based promoter interaction maps as a structural framework for deciding which SNPs to test for association with gene expression, and observe ultra-long-distance pieQTLs (&amp;gt;1 megabase away), including several disease-risk variants. We validated the functional role of pieQTLs using reporter assays, CRISPRi, dCas9-tiling guides and Cas9-mediated base-pair editing. In this article we present a method for functional eQTL discovery and provide insights into relevance of noncoding variants for cell-specific gene regulation and for disease association beyond conventional eQTL mapping. H3K27ac HiChIP analysis helps to identify promoter-interacting expression quantitative trait loci (pieQTLs) in five common immune cell types. Some pieQTLs overlap with nontranscribed promoters that act as enhancers.},
  archive      = {J_NG},
  author       = {Chandra, Vivek and Bhattacharyya, Sourya and Schmiedel, Benjamin J. and Madrigal, Ariel and Gonzalez-Colin, Cristian and Fotsing, Stephanie and Crinklaw, Austin and Seumois, Gregory and Mohammadi, Pejman and Kronenberg, Mitchell and Peters, Bjoern and Ay, Ferhat and Vijayanand, Pandurangan},
  doi          = {10.1038/s41588-020-00745-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {110-119},
  shortjournal = {Nat. Genet.},
  title        = {Promoter-interacting expression quantitative trait loci are enriched for functional genetic variants},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). WAPL maintains a cohesin loading cycle to preserve
cell-type-specific distal gene regulation. <em>NG</em>, <em>53</em>(1),
100–109. (<a href="https://doi.org/10.1038/s41588-020-00744-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cohesin complex has an essential role in maintaining genome organization. However, its role in gene regulation remains largely unresolved. Here we report that the cohesin release factor WAPL creates a pool of free cohesin, in a process known as cohesin turnover, which reloads it to cell-type-specific binding sites. Paradoxically, stabilization of cohesin binding, following WAPL ablation, results in depletion of cohesin from these cell-type-specific regions, loss of gene expression and differentiation. Chromosome conformation capture experiments show that cohesin turnover is important for maintaining promoter–enhancer loops. Binding of cohesin to cell-type-specific sites is dependent on the pioneer transcription factors OCT4 (POU5F1) and SOX2, but not NANOG. We show the importance of cohesin turnover in controlling transcription and propose that a cycle of cohesin loading and off-loading, instead of static cohesin binding, mediates promoter and enhancer interactions critical for gene regulation. WAPL creates a pool of free cohesin that binds to cell-type-specific sites. This cohesin turnover is important for maintaining promoter–enhancer loops.},
  archive      = {J_NG},
  author       = {Liu, Ning Qing and Maresca, Michela and van den Brand, Teun and Braccioli, Luca and Schijns, Marijne M. G. A. and Teunissen, Hans and Bruneau, Benoit G. and Nora, Elphѐge P. and de Wit, Elzo},
  doi          = {10.1038/s41588-020-00744-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {100-109},
  shortjournal = {Nat. Genet.},
  title        = {WAPL maintains a cohesin loading cycle to preserve cell-type-specific distal gene regulation},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Conservation of copy number profiles during engraftment and
passaging of patient-derived cancer xenografts. <em>NG</em>,
<em>53</em>(1), 86–99. (<a
href="https://doi.org/10.1038/s41588-020-00750-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patient-derived xenografts (PDXs) are resected human tumors engrafted into mice for preclinical studies and therapeutic testing. It has been proposed that the mouse host affects tumor evolution during PDX engraftment and propagation, affecting the accuracy of PDX modeling of human cancer. Here, we exhaustively analyze copy number alterations (CNAs) in 1,451 PDX and matched patient tumor (PT) samples from 509 PDX models. CNA inferences based on DNA sequencing and microarray data displayed substantially higher resolution and dynamic range than gene expression-based inferences, and they also showed strong CNA conservation from PTs through late-passage PDXs. CNA recurrence analysis of 130 colorectal and breast PT/PDX-early/PDX-late trios confirmed high-resolution CNA retention. We observed no significant enrichment of cancer-related genes in PDX-specific CNAs across models. Moreover, CNA differences between patient and PDX tumors were comparable to variations in multiregion samples within patients. Our study demonstrates the lack of systematic copy number evolution driven by the PDX mouse host. Analysis of copy number alterations in 1,451 patient-derived xenografts (PDXs) and matched patient tumor samples shows strong conservation from patient tumors through late-passage PDXs and a lack of systematic copy number evolution driven by the mouse host.},
  archive      = {J_NG},
  author       = {Woo, Xing Yi and Giordano, Jessica and Srivastava, Anuj and Zhao, Zi-Ming and Lloyd, Michael W. and de Bruijn, Roebi and Suh, Yun-Suhk and Patidar, Rajesh and Chen, Li and Scherer, Sandra and Bailey, Matthew H. and Yang, Chieh-Hsiang and Cortes-Sanchez, Emilio and Xi, Yuanxin and Wang, Jing and Wickramasinghe, Jayamanna and Kossenkov, Andrew V. and Rebecca, Vito W. and Sun, Hua and Mashl, R. Jay and Davies, Sherri R. and Jeon, Ryan and Frech, Christian and Randjelovic, Jelena and Rosains, Jacqueline and Galimi, Francesco and Bertotti, Andrea and Lafferty, Adam and O’Farrell, Alice C. and Modave, Elodie and Lambrechts, Diether and ter Brugge, Petra and Serra, Violeta and Marangoni, Elisabetta and El Botty, Rania and Kim, Hyunsoo and Kim, Jong-Il and Yang, Han-Kwang and Lee, Charles and Dean, Dennis A. and Davis-Dusenbery, Brandi and Evrard, Yvonne A. and Doroshow, James H. and Welm, Alana L. and Welm, Bryan E. and Lewis, Michael T. and Fang, Bingliang and Roth, Jack A. and Meric-Bernstam, Funda and Herlyn, Meenhard and Davies, Michael A. and Ding, Li and Li, Shunqiang and Govindan, Ramaswamy and Isella, Claudio and Moscow, Jeffrey A. and Trusolino, Livio and Byrne, Annette T. and Jonkers, Jos and Bult, Carol J. and Medico, Enzo and Chuang, Jeffrey H.},
  doi          = {10.1038/s41588-020-00750-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {86-99},
  shortjournal = {Nat. Genet.},
  title        = {Conservation of copy number profiles during engraftment and passaging of patient-derived cancer xenografts},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genetic screening for single-cell variability modulators
driving therapy resistance. <em>NG</em>, <em>53</em>(1), 76–85. (<a
href="https://doi.org/10.1038/s41588-020-00749-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular plasticity describes the ability of cells to transition from one set of phenotypes to another. In melanoma, transient fluctuations in the molecular state of tumor cells mark the formation of rare cells primed to survive BRAF inhibition and reprogram into a stably drug-resistant fate. However, the biological processes governing cellular priming remain unknown. We used CRISPR–Cas9 genetic screens to identify genes that affect cell fate decisions by altering cellular plasticity. We found that many factors can independently affect cellular priming and fate decisions. We discovered a new plasticity-based mode of increasing resistance to BRAF inhibition that pushes cells towards a more differentiated state. Manipulating cellular plasticity through inhibition of DOT1L before the addition of the BRAF inhibitor resulted in more therapy resistance than concurrent administration. Our results indicate that modulating cellular plasticity can alter cell fate decisions and may prove useful for treating drug resistance in other cancers. A genetic screen designed to capture modulators of single-cell state variability identifies a new mode of altering resistance to BRAF inhibition in melanoma that pushes cells toward a more differentiated state. Modulating cellular plasticity may provide a new avenue to overcome drug resistance.},
  archive      = {J_NG},
  author       = {Torre, Eduardo A. and Arai, Eri and Bayatpour, Sareh and Jiang, Connie L. and Beck, Lauren E. and Emert, Benjamin L. and Shaffer, Sydney M. and Mellis, Ian A. and Fane, Mitchell E. and Alicea, Gretchen M. and Budinich, Krista A. and Weeraratna, Ashani T. and Shi, Junwei and Raj, Arjun},
  doi          = {10.1038/s41588-020-00749-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {76-85},
  shortjournal = {Nat. Genet.},
  title        = {Genetic screening for single-cell variability modulators driving therapy resistance},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Trans-ancestry genome-wide association meta-analysis of
prostate cancer identifies new susceptibility loci and informs genetic
risk prediction. <em>NG</em>, <em>53</em>(1), 65–75. (<a
href="https://doi.org/10.1038/s41588-020-00748-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prostate cancer is a highly heritable disease with large disparities in incidence rates across ancestry populations. We conducted a multiancestry meta-analysis of prostate cancer genome-wide association studies (107,247 cases and 127,006 controls) and identified 86 new genetic risk variants independently associated with prostate cancer risk, bringing the total to 269 known risk variants. The top genetic risk score (GRS) decile was associated with odds ratios that ranged from 5.06 (95% confidence interval (CI), 4.84–5.29) for men of European ancestry to 3.74 (95% CI, 3.36–4.17) for men of African ancestry. Men of African ancestry were estimated to have a mean GRS that was 2.18-times higher (95% CI, 2.14–2.22), and men of East Asian ancestry 0.73-times lower (95% CI, 0.71–0.76), than men of European ancestry. These findings support the role of germline variation contributing to population differences in prostate cancer risk, with the GRS offering an approach for personalized risk prediction. A meta-analysis of genome-wide association studies across different populations highlights new risk loci and provides a genetic risk score that can stratify prostate cancer risk across ancestries.},
  archive      = {J_NG},
  author       = {Conti, David V. and Darst, Burcu F. and Moss, Lilit C. and Saunders, Edward J. and Sheng, Xin and Chou, Alisha and Schumacher, Fredrick R. and Olama, Ali Amin Al and Benlloch, Sara and Dadaev, Tokhir and Brook, Mark N. and Sahimi, Ali and Hoffmann, Thomas J. and Takahashi, Atushi and Matsuda, Koichi and Momozawa, Yukihide and Fujita, Masashi and Muir, Kenneth and Lophatananon, Artitaya and Wan, Peggy and Le Marchand, Loic and Wilkens, Lynne R. and Stevens, Victoria L. and Gapstur, Susan M. and Carter, Brian D. and Schleutker, Johanna and Tammela, Teuvo L. J. and Sipeky, Csilla and Auvinen, Anssi and Giles, Graham G. and Southey, Melissa C. and MacInnis, Robert J. and Cybulski, Cezary and Wokołorczyk, Dominika and Lubiński, Jan and Neal, David E. and Donovan, Jenny L. and Hamdy, Freddie C. and Martin, Richard M. and Nordestgaard, Børge G. and Nielsen, Sune F. and Weischer, Maren and Bojesen, Stig E. and Røder, Martin Andreas and Iversen, Peter and Batra, Jyotsna and Chambers, Suzanne and Moya, Leire and Horvath, Lisa and Clements, Judith A. and Tilley, Wayne and Risbridger, Gail P. and Gronberg, Henrik and Aly, Markus and Szulkin, Robert and Eklund, Martin and Nordström, Tobias and Pashayan, Nora and Dunning, Alison M. and Ghoussaini, Maya and Travis, Ruth C. and Key, Tim J. and Riboli, Elio and Park, Jong Y. and Sellers, Thomas A. and Lin, Hui-Yi and Albanes, Demetrius and Weinstein, Stephanie J. and Mucci, Lorelei A. and Giovannucci, Edward and Lindstrom, Sara and Kraft, Peter and Hunter, David J. and Penney, Kathryn L. and Turman, Constance and Tangen, Catherine M. and Goodman, Phyllis J. and Thompson, Ian M. and Hamilton, Robert J. and Fleshner, Neil E. and Finelli, Antonio and Parent, Marie-Élise and Stanford, Janet L. and Ostrander, Elaine A. and Geybels, Milan S. and Koutros, Stella and Freeman, Laura E. Beane and Stampfer, Meir and Wolk, Alicja and Håkansson, Niclas and Andriole, Gerald L. and Hoover, Robert N. and Machiela, Mitchell J. and Sørensen, Karina Dalsgaard and Borre, Michael and Blot, William J. and Zheng, Wei and Yeboah, Edward D. and Mensah, James E. and Lu, Yong-Jie and Zhang, Hong-Wei and Feng, Ninghan and Mao, Xueying and Wu, Yudong and Zhao, Shan-Chao and Sun, Zan and Thibodeau, Stephen N. and McDonnell, Shannon K. and Schaid, Daniel J. and West, Catharine M. L. and Burnet, Neil and Barnett, Gill and Maier, Christiane and Schnoeller, Thomas and Luedeke, Manuel and Kibel, Adam S. and Drake, Bettina F. and Cussenot, Olivier and Cancel-Tassin, Géraldine and Menegaux, Florence and Truong, Thérèse and Koudou, Yves Akoli and John, Esther M. and Grindedal, Eli Marie and Maehle, Lovise and Khaw, Kay-Tee and Ingles, Sue A. and Stern, Mariana C. and Vega, Ana and Gómez-Caamaño, Antonio and Fachal, Laura and Rosenstein, Barry S. and Kerns, Sarah L. and Ostrer, Harry and Teixeira, Manuel R. and Paulo, Paula and Brandão, Andreia and Watya, Stephen and Lubwama, Alexander and Bensen, Jeannette T. and Fontham, Elizabeth T. H. and Mohler, James and Taylor, Jack A. and Kogevinas, Manolis and Llorca, Javier and Castaño-Vinyals, Gemma and Cannon-Albright, Lisa and Teerlink, Craig C. and Huff, Chad D. and Strom, Sara S. and Multigner, Luc and Blanchet, Pascal and Brureau, Laurent and Kaneva, Radka and Slavov, Chavdar and Mitev, Vanio and Leach, Robin J. and Weaver, Brandi and Brenner, Hermann and Cuk, Katarina and Holleczek, Bernd and Saum, Kai-Uwe and Klein, Eric A. and Hsing, Ann W. and Kittles, Rick A. and Murphy, Adam B. and Logothetis, Christopher J. and Kim, Jeri and Neuhausen, Susan L. and Steele, Linda and Ding, Yuan Chun and Isaacs, William B. and Nemesure, Barbara and Hennis, Anselm J. M. and Carpten, John and Pandha, Hardev and Michael, Agnieszka and De Ruyck, Kim and De Meerleer, Gert and Ost, Piet and Xu, Jianfeng and Razack, Azad and Lim, Jasmine and Teo, Soo-Hwang and Newcomb, Lisa F. and Lin, Daniel W. and Fowke, Jay H. and Neslund-Dudas, Christine and Rybicki, Benjamin A. and Gamulin, Marija and Lessel, Davor and Kulis, Tomislav and Usmani, Nawaid and Singhal, Sandeep and Parliament, Matthew and Claessens, Frank and Joniau, Steven and Van den Broeck, Thomas and Gago-Dominguez, Manuela and Castelao, Jose Esteban and Martinez, Maria Elena and Larkin, Samantha and Townsend, Paul A. and Aukim-Hastie, Claire and Bush, William S. and Aldrich, Melinda C. and Crawford, Dana C. and Srivastava, Shiv and Cullen, Jennifer C. and Petrovics, Gyorgy and Casey, Graham and Roobol, Monique J. and Jenster, Guido and van Schaik, Ron H. N. and Hu, Jennifer J. and Sanderson, Maureen and Varma, Rohit and McKean-Cowdin, Roberta and Torres, Mina and Mancuso, Nicholas and Berndt, Sonja I. and Van Den Eeden, Stephen K. and Easton, Douglas F. and Chanock, Stephen J. and Cook, Michael B. and Wiklund, Fredrik and Nakagawa, Hidewaki and Witte, John S. and Eeles, Rosalind A. and Kote-Jarai, Zsofia and Haiman, Christopher A.},
  doi          = {10.1038/s41588-020-00748-0},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {65-75},
  shortjournal = {Nat. Genet.},
  title        = {Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A cross-platform approach identifies genetic regulators of
human metabolism and health. <em>NG</em>, <em>53</em>(1), 54–64. (<a
href="https://doi.org/10.1038/s41588-020-00751-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In cross-platform analyses of 174 metabolites, we identify 499 associations (P &amp;lt; 4.9 × 10−10) characterized by pleiotropy, allelic heterogeneity, large and nonlinear effects and enrichment for nonsynonymous variation. We identify a signal at GLP2R (p.Asp470Asn) shared among higher citrulline levels, body mass index, fasting glucose-dependent insulinotropic peptide and type 2 diabetes, with β-arrestin signaling as the underlying mechanism. Genetically higher serine levels are shown to reduce the likelihood (by 95%) and predict development of macular telangiectasia type 2, a rare degenerative retinal disease. Integration of genomic and small molecule data across platforms enables the discovery of regulators of human metabolism and translation into clinical insights. A large-scale genome-wide meta-analysis conducted across different platforms identifies genetic loci regulating levels of circulating metabolites.},
  archive      = {J_NG},
  author       = {Lotta, Luca A. and Pietzner, Maik and Stewart, Isobel D. and Wittemans, Laura B. L. and Li, Chen and Bonelli, Roberto and Raffler, Johannes and Biggs, Emma K. and Oliver-Williams, Clare and Auyeung, Victoria P. W. and Luan, Jian’an and Wheeler, Eleanor and Paige, Ellie and Surendran, Praveen and Michelotti, Gregory A. and Scott, Robert A. and Burgess, Stephen and Zuber, Verena and Sanderson, Eleanor and Koulman, Albert and Imamura, Fumiaki and Forouhi, Nita G. and Khaw, Kay-Tee and Griffin, Julian L. and Wood, Angela M. and Kastenmüller, Gabi and Danesh, John and Butterworth, Adam S. and Gribble, Fiona M. and Reimann, Frank and Bahlo, Melanie and Fauman, Eric and Wareham, Nicholas J. and Langenberg, Claudia},
  doi          = {10.1038/s41588-020-00751-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {54-64},
  shortjournal = {Nat. Genet.},
  title        = {A cross-platform approach identifies genetic regulators of human metabolism and health},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Insights into the genetic architecture of the human face.
<em>NG</em>, <em>53</em>(1), 45–53. (<a
href="https://doi.org/10.1038/s41588-020-00741-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human face is complex and multipartite, and characterization of its genetic architecture remains challenging. Using a multivariate genome-wide association study meta-analysis of 8,246 European individuals, we identified 203 genome-wide-significant signals (120 also study-wide significant) associated with normal-range facial variation. Follow-up analyses indicate that the regions surrounding these signals are enriched for enhancer activity in cranial neural crest cells and craniofacial tissues, several regions harbor multiple signals with associations to different facial phenotypes, and there is evidence for potential coordinated actions of variants. In summary, our analyses provide insights into the understanding of how complex morphological traits are shaped by both individual and coordinated genetic actions. A multivariate genome-wide association study identifies 203 signals associated with facial variation. These signals are enriched for enhancer activity in cranial neural crest cells and craniofacial tissues.},
  archive      = {J_NG},
  author       = {White, Julie D. and Indencleef, Karlijne and Naqvi, Sahin and Eller, Ryan J. and Hoskens, Hanne and Roosenboom, Jasmien and Lee, Myoung Keun and Li, Jiarui and Mohammed, Jaaved and Richmond, Stephen and Quillen, Ellen E. and Norton, Heather L. and Feingold, Eleanor and Swigut, Tomek and Marazita, Mary L. and Peeters, Hilde and Hens, Greet and Shaffer, John R. and Wysocka, Joanna and Walsh, Susan and Weinberg, Seth M. and Shriver, Mark D. and Claes, Peter},
  doi          = {10.1038/s41588-020-00741-7},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {45-53},
  shortjournal = {Nat. Genet.},
  title        = {Insights into the genetic architecture of the human face},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Investigating the genetic architecture of noncognitive
skills using GWAS-by-subtraction. <em>NG</em>, <em>53</em>(1), 35–44.
(<a href="https://doi.org/10.1038/s41588-020-00754-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Little is known about the genetic architecture of traits affecting educational attainment other than cognitive ability. We used genomic structural equation modeling and prior genome-wide association studies (GWASs) of educational attainment (n = 1,131,881) and cognitive test performance (n = 257,841) to estimate SNP associations with educational attainment variation that is independent of cognitive ability. We identified 157 genome-wide-significant loci and a polygenic architecture accounting for 57% of genetic variance in educational attainment. Noncognitive genetics were enriched in the same brain tissues and cell types as cognitive performance, but showed different associations with gray-matter brain volumes. Noncognitive genetics were further distinguished by associations with personality traits, less risky behavior and increased risk for certain psychiatric disorders. For socioeconomic success and longevity, noncognitive and cognitive-performance genetics demonstrated associations of similar magnitude. By conducting a GWAS of a phenotype that was not directly measured, we offer a view of genetic architecture of noncognitive skills influencing educational success. Genomic structural equation modeling of genome-wide association data for educational attainment and cognitive test performance is used to estimate the genetic component of variation in educational attainment that is independent of cognitive ability. The study finds that noncognitive skills account for 57% of genetic variation in educational attainment.},
  archive      = {J_NG},
  author       = {Demange, Perline A. and Malanchini, Margherita and Mallard, Travis T. and Biroli, Pietro and Cox, Simon R. and Grotzinger, Andrew D. and Tucker-Drob, Elliot M. and Abdellaoui, Abdel and Arseneault, Louise and van Bergen, Elsje and Boomsma, Dorret I. and Caspi, Avshalom and Corcoran, David L. and Domingue, Benjamin W. and Harris, Kathleen Mullan and Ip, Hill F. and Mitchell, Colter and Moffitt, Terrie E. and Poulton, Richie and Prinz, Joseph A. and Sugden, Karen and Wertz, Jasmin and Williams, Benjamin S. and de Zeeuw, Eveline L. and Belsky, Daniel W. and Harden, K. Paige and Nivard, Michel G.},
  doi          = {10.1038/s41588-020-00754-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {35-44},
  shortjournal = {Nat. Genet.},
  title        = {Investigating the genetic architecture of noncognitive skills using GWAS-by-subtraction},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Differences between germline genomes of monozygotic twins.
<em>NG</em>, <em>53</em>(1), 27–34. (<a
href="https://doi.org/10.1038/s41588-020-00755-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the important role that monozygotic twins have played in genetics research, little is known about their genomic differences. Here we show that monozygotic twins differ on average by 5.2 early developmental mutations and that approximately 15% of monozygotic twins have a substantial number of these early developmental mutations specific to one of them. Using the parents and offspring of twins, we identified pre-twinning mutations. We observed instances where a twin was formed from a single cell lineage in the pre-twinning cell mass and instances where a twin was formed from several cell lineages. CpG&amp;gt;TpG mutations increased in frequency with embryonic development, coinciding with an increase in DNA methylation. Our results indicate that allocations of cells during development shapes genomic differences between monozygotic twins. Whole-genome sequencing of monozygotic twins, along with their parents, spouses and children, identifies postzygotic mutations present in the somatic tissue of one twin, but not the other, and characterizes differences in the number and timing of these mutations.},
  archive      = {J_NG},
  author       = {Jonsson, Hakon and Magnusdottir, Erna and Eggertsson, Hannes P. and Stefansson, Olafur A. and Arnadottir, Gudny A. and Eiriksson, Ogmundur and Zink, Florian and Helgason, Einar A. and Jonsdottir, Ingileif and Gylfason, Arnaldur and Jonasdottir, Adalbjorg and Jonasdottir, Aslaug and Beyter, Doruk and Steingrimsdottir, Thora and Norddahl, Gudmundur L. and Magnusson, Olafur Th. and Masson, Gisli and Halldorsson, Bjarni V. and Thorsteinsdottir, Unnur and Helgason, Agnar and Sulem, Patrick and Gudbjartsson, Daniel F. and Stefansson, Kari},
  doi          = {10.1038/s41588-020-00755-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {27-34},
  shortjournal = {Nat. Genet.},
  title        = {Differences between germline genomes of monozygotic twins},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The amino acid transporter SLC7A5 is required for efficient
growth of KRAS-mutant colorectal cancer. <em>NG</em>, <em>53</em>(1),
16–26. (<a href="https://doi.org/10.1038/s41588-020-00753-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oncogenic KRAS mutations and inactivation of the APC tumor suppressor co-occur in colorectal cancer (CRC). Despite efforts to target mutant KRAS directly, most therapeutic approaches focus on downstream pathways, albeit with limited efficacy. Moreover, mutant KRAS alters the basal metabolism of cancer cells, increasing glutamine utilization to support proliferation. We show that concomitant mutation of Apc and Kras in the mouse intestinal epithelium profoundly rewires metabolism, increasing glutamine consumption. Furthermore, SLC7A5, a glutamine antiporter, is critical for colorectal tumorigenesis in models of both early- and late-stage metastatic disease. Mechanistically, SLC7A5 maintains intracellular amino acid levels following KRAS activation through transcriptional and metabolic reprogramming. This supports the increased demand for bulk protein synthesis that underpins the enhanced proliferation of KRAS-mutant cells. Moreover, targeting protein synthesis, via inhibition of the mTORC1 regulator, together with Slc7a5 deletion abrogates the growth of established Kras-mutant tumors. Together, these data suggest SLC7A5 as an attractive target for therapy-resistant KRAS-mutant CRC. Colorectal tumors with mutated KRAS and APC require the amino acid transporter SLC7A5 to drive tumorigenesis. Mechanistically, SLC7A5 drives transcriptional and metabolic reprogramming by maintaining intracellular amino acid levels, leading to enhanced protein synthesis.},
  archive      = {J_NG},
  author       = {Najumudeen, Arafath K. and Ceteci, Fatih and Fey, Sigrid K. and Hamm, Gregory and Steven, Rory T. and Hall, Holly and Nikula, Chelsea J. and Dexter, Alex and Murta, Teresa and Race, Alan M. and Sumpton, David and Vlahov, Nikola and Gay, David M. and Knight, John R. P. and Jackstadt, Rene and Leach, Joshua D. G. and Ridgway, Rachel A. and Johnson, Emma R. and Nixon, Colin and Hedley, Ann and Gilroy, Kathryn and Clark, William and Malla, Sudhir B. and Dunne, Philip D. and Rodriguez-Blanco, Giovanny and Critchlow, Susan E. and Mrowinska, Agata and Malviya, Gaurav and Solovyev, Dmitry and Brown, Gavin and Lewis, David Y. and Mackay, Gillian M. and Strathdee, Douglas and Tardito, Saverio and Gottlieb, Eyal and Takats, Zoltan and Barry, Simon T. and Goodwin, Richard J. A. and Bunch, Josephine and Bushell, Martin and Campbell, Andrew D. and Sansom, Owen J.},
  doi          = {10.1038/s41588-020-00753-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {16-26},
  shortjournal = {Nat. Genet.},
  title        = {The amino acid transporter SLC7A5 is required for efficient growth of KRAS-mutant colorectal cancer},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The association between tumor mutational burden and
prognosis is dependent on treatment context. <em>NG</em>,
<em>53</em>(1), 11–15. (<a
href="https://doi.org/10.1038/s41588-020-00752-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In multiple cancer types, high tumor mutational burden (TMB) is associated with longer survival after treatment with immune checkpoint inhibitors (ICIs). The association of TMB with survival outside of the immunotherapy context is poorly understood. We analyzed 10,233 patients (80% non-ICI-treated, 20% ICI-treated) with 17 cancer types before/without ICI treatment or after ICI treatment. In non-ICI-treated patients, higher TMB (higher percentile within cancer type) was not associated with better prognosis; in fact, in many cancer types, higher TMB was associated with poorer survival, in contrast to ICI-treated patients in whom higher TMB was associated with longer survival. A high tumor mutational burden (TMB) is associated with improved immunotherapy response in many tumor types. This analysis of 10,233 individuals shows that, in contrast, high TMB is associated with poorer survival in patients that have not been treated with immune checkpoint inhibitors.},
  archive      = {J_NG},
  author       = {Valero, Cristina and Lee, Mark and Hoen, Douglas and Wang, Jingming and Nadeem, Zaineb and Patel, Neal and Postow, Michael A. and Shoushtari, Alexander N. and Plitas, George and Balachandran, Vinod P. and Smith, J. Joshua and Crago, Aimee M. and Long Roche, Kara C. and Kelly, Daniel W. and Samstein, Robert M. and Rana, Satshil and Ganly, Ian and Wong, Richard J. and Hakimi, A. Ari and Berger, Michael F. and Zehir, Ahmet and Solit, David B. and Ladanyi, Marc and Riaz, Nadeem and Chan, Timothy A. and Seshan, Venkatraman E. and Morris, Luc G. T.},
  doi          = {10.1038/s41588-020-00752-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {11-15},
  shortjournal = {Nat. Genet.},
  title        = {The association between tumor mutational burden and prognosis is dependent on treatment context},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Exploiting KRAS-mediated metabolic reprogramming as a
therapeutic target. <em>NG</em>, <em>53</em>(1), 9–10. (<a
href="https://doi.org/10.1038/s41588-020-00758-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Half of all colorectal cancers bear KRAS-activating mutations that affect the metabolic dependencies of cancer cells and drive resistance to commonly used drugs. A new study provides insights into KRAS-driven metabolic rewiring and identifies a new therapeutic target for KRAS-mutant cancers.},
  archive      = {J_NG},
  author       = {Bootsma, Sanne and van Neerven, Sanne M. and Vermeulen, Louis},
  doi          = {10.1038/s41588-020-00758-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {9-10},
  shortjournal = {Nat. Genet.},
  title        = {Exploiting KRAS-mediated metabolic reprogramming as a therapeutic target},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Toward better governance of human genomic data. <em>NG</em>,
<em>53</em>(1), 2–8. (<a
href="https://doi.org/10.1038/s41588-020-00742-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here, we argue that, in line with the dramatic increase in the collection, storage and curation of human genomic data for biomedical research, genomic data repositories and consortia have adopted governance frameworks to both enable wide access and protect against possible harms. However, the merits and limitations of different governance frameworks in achieving these twin aims are a matter of ongoing debate in the scientific community; indeed, best practices and points for consideration are notably absent in devising governance frameworks for genomic databases. According to our collective experience in devising and assessing governance frameworks, we identify five key functions of ‘good governance’ (or ‘better governance’) and three areas in which trade-offs should be considered when specifying policies within those functions. We apply these functions as a benchmark to describe, as an example, the governance frameworks of six large-scale international genomic projects.},
  archive      = {J_NG},
  author       = {O’Doherty, Kieran C. and Shabani, Mahsa and Dove, Edward S. and Bentzen, Heidi Beate and Borry, Pascal and Burgess, Michael M. and Chalmers, Don and De Vries, Jantina and Eckstein, Lisa and Fullerton, Stephanie M. and Juengst, Eric and Kato, Kazuto and Kaye, Jane and Knoppers, Bartha Maria and Koenig, Barbara A. and Manson, Spero M. and McGrail, Kimberlyn M. and McGuire, Amy L. and Meslin, Eric M. and Nicol, Dianne and Prainsack, Barbara and Terry, Sharon F. and Thorogood, Adrian and Burke, Wylie},
  doi          = {10.1038/s41588-020-00742-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2-8},
  shortjournal = {Nat. Genet.},
  title        = {Toward better governance of human genomic data},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Welcoming 2021. <em>NG</em>, <em>53</em>(1), 1. (<a
href="https://doi.org/10.1038/s41588-020-00765-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After a year turned upside-down by the COVID-19 pandemic, and now with promising vaccine news on the horizon, we can start looking forward to a new year with the hope that 2021 will bring exciting things to the science community. Here, Nature Genetics shares what we are awaiting in 2021.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-00765-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {Welcoming 2021},
  volume       = {53},
  year         = {2021},
}
</textarea>
</details></li>
</ul>

</body>
</html>
