<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth---346">NMETH - 346</h2>
<ul>
<li><details>
<summary>
(2020o). Ling-ling chen. <em>NMETH</em>, <em>18</em>(1), 3. (<a
href="https://doi.org/10.1038/s41592-020-01039-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Creativity, passion, inclusion and a targeted way to study circular RNAs.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-01039-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {1},
  pages        = {3},
  shortjournal = {Nat. Meth.},
  title        = {Ling-ling chen},
  volume       = {18},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Simultaneous cortex-wide fluorescence ca2+ imaging and
whole-brain fMRI. <em>NMETH</em>, <em>17</em>(12), 1262–1271. (<a
href="https://doi.org/10.1038/s41592-020-00984-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Achieving a comprehensive understanding of brain function requires multiple imaging modalities with complementary strengths. We present an approach for concurrent widefield optical and functional magnetic resonance imaging. By merging these modalities, we can simultaneously acquire whole-brain blood-oxygen-level-dependent (BOLD) and whole-cortex calcium-sensitive fluorescent measures of brain activity. In a transgenic murine model, we show that calcium predicts the BOLD signal, using a model that optimizes a gamma-variant transfer function. We find consistent predictions across the cortex, which are best at low frequency (0.009–0.08 Hz). Furthermore, we show that the relationship between modality connectivity strengths varies by region. Our approach links cell-type-specific optical measurements of activity to the most widely used method for assessing human brain function. Simultaneous widefield calcium imaging and fMRI provide insight into neural activity at multiple scales and can be used to decipher the cellular origin of BOLD activity.},
  archive      = {J_NMETH},
  author       = {Lake, Evelyn M. R. and Ge, Xinxin and Shen, Xilin and Herman, Peter and Hyder, Fahmeed and Cardin, Jessica A. and Higley, Michael J. and Scheinost, Dustin and Papademetris, Xenophon and Crair, Michael C. and Constable, R. Todd},
  doi          = {10.1038/s41592-020-00984-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1262-1271},
  shortjournal = {Nat. Meth.},
  title        = {Simultaneous cortex-wide fluorescence ca2+ imaging and whole-brain fMRI},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). BAcTrace, a tool for retrograde tracing of neuronal circuits
in drosophila. <em>NMETH</em>, <em>17</em>(12), 1254–1261. (<a
href="https://doi.org/10.1038/s41592-020-00989-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal behavior is encoded in neuronal circuits in the brain. To elucidate the function of these circuits, it is necessary to identify, record from and manipulate networks of connected neurons. Here we present BAcTrace (Botulinum-Activated Tracer), a genetically encoded, retrograde, transsynaptic labeling system. BAcTrace is based on Clostridium botulinum neurotoxin A, Botox, which we engineered to travel retrogradely between neurons to activate an otherwise silent transcription factor. We validated BAcTrace at three neuronal connections in the Drosophila olfactory system. We show that BAcTrace-mediated labeling allows electrophysiological recording of connected neurons. Finally, in a challenging circuit with highly divergent connections, BAcTrace correctly identified 12 of 16 connections that were previously observed by electron microscopy. BAcTrace is a predominantly retrograde tracing tool for connectomic analyses in Drosophila.},
  archive      = {J_NMETH},
  author       = {Cachero, Sebastian and Gkantia, Marina and Bates, Alexander S. and Frechter, Shahar and Blackie, Laura and McCarthy, Amy and Sutcliffe, Ben and Strano, Alessio and Aso, Yoshinori and Jefferis, Gregory S. X. E.},
  doi          = {10.1038/s41592-020-00989-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1254-1261},
  shortjournal = {Nat. Meth.},
  title        = {BAcTrace, a tool for retrograde tracing of neuronal circuits in drosophila},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Targeted deubiquitination rescues distinct
trafficking-deficient ion channelopathies. <em>NMETH</em>,
<em>17</em>(12), 1245–1253. (<a
href="https://doi.org/10.1038/s41592-020-00992-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Impaired protein stability or trafficking underlies diverse ion channelopathies and represents an unexploited unifying principle for developing common treatments for otherwise dissimilar diseases. Ubiquitination limits ion channel surface density, but targeting this pathway for the purposes of basic study or therapy is challenging because of its prevalent role in proteostasis. We developed engineered deubiquitinases (enDUBs) that enable selective ubiquitin chain removal from target proteins to rescue the functional expression of disparate mutant ion channels that underlie long QT syndrome (LQT) and cystic fibrosis (CF). In an LQT type 1 (LQT1) cardiomyocyte model, enDUB treatment restored delayed rectifier potassium currents and normalized action potential duration. CF-targeted enDUBs synergistically rescued common (ΔF508) and pharmacotherapy-resistant (N1303K) CF mutations when combined with the US Food and Drug Administation (FDA)-approved drugs Orkambi (lumacaftor/ivacaftor) and Trikafta (elexacaftor/tezacaftor/ivacaftor and ivacaftor). Altogether, targeted deubiquitination via enDUBs provides a powerful protein stabilization method that not only corrects diverse diseases caused by impaired ion channel trafficking, but also introduces a new tool for deconstructing the ubiquitin code in situ. Engineered deubiquitinases (enDUBs) use nanobodies to bring deubiquitinases to target proteins for selective ubiquitin chain removal. enDUBs can rescue the functional expression of mutant ion channels like those responsible for cystic fibrosis.},
  archive      = {J_NMETH},
  author       = {Kanner, Scott A. and Shuja, Zunaira and Choudhury, Papiya and Jain, Ananya and Colecraft, Henry M.},
  doi          = {10.1038/s41592-020-00992-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1245-1253},
  shortjournal = {Nat. Meth.},
  title        = {Targeted deubiquitination rescues distinct trafficking-deficient ion channelopathies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Avant-garde: An automated data-driven DIA data curation
tool. <em>NMETH</em>, <em>17</em>(12), 1237–1244. (<a
href="https://doi.org/10.1038/s41592-020-00986-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several challenges remain in data-independent acquisition (DIA) data analysis, such as to confidently identify peptides, define integration boundaries, remove interferences, and control false discovery rates. In practice, a visual inspection of the signals is still required, which is impractical with large datasets. We present Avant-garde as a tool to refine DIA (and parallel reaction monitoring) data. Avant-garde uses a novel data-driven scoring strategy: signals are refined by learning from the dataset itself, using all measurements in all samples to achieve the best optimization. We evaluate the performance of Avant-garde using benchmark DIA datasets and show that it can determine the quantitative suitability of a peptide peak, and reach the same levels of selectivity, accuracy, and reproducibility as manual validation. Avant-garde is complementary to existing DIA analysis engines and aims to establish a strong foundation for subsequent analysis of quantitative mass spectrometry data. A computational tool, Avant-garde, automates refinement of data-independent acquisition mass spectrometry-based quantitative proteomics data.},
  archive      = {J_NMETH},
  author       = {Vaca Jacome, Alvaro Sebastian and Peckner, Ryan and Shulman, Nicholas and Krug, Karsten and DeRuff, Katherine C. and Officer, Adam and Christianson, Karen E. and MacLean, Brendan and MacCoss, Michael J. and Carr, Steven A. and Jaffe, Jacob D.},
  doi          = {10.1038/s41592-020-00986-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1237-1244},
  shortjournal = {Nat. Meth.},
  title        = {Avant-garde: An automated data-driven DIA data curation tool},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DiaPASEF: Parallel accumulation–serial fragmentation
combined with data-independent acquisition. <em>NMETH</em>,
<em>17</em>(12), 1229–1236. (<a
href="https://doi.org/10.1038/s41592-020-00998-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data-independent acquisition modes isolate and concurrently fragment populations of different precursors by cycling through segments of a predefined precursor m/z range. Although these selection windows collectively cover the entire m/z range, overall, only a few per cent of all incoming ions are isolated for mass analysis. Here, we make use of the correlation of molecular weight and ion mobility in a trapped ion mobility device (timsTOF Pro) to devise a scan mode that samples up to 100\% of the peptide precursor ion current in m/z and mobility windows. We extend an established targeted data extraction workflow by inclusion of the ion mobility dimension for both signal extraction and scoring and thereby increase the specificity for precursor identification. Data acquired from whole proteome digests and mixed organism samples demonstrate deep proteome coverage and a high degree of reproducibility as well as quantitative accuracy, even from 10 ng sample amounts. diaPASEF makes use of the correlation between the ion mobility and the m/z of peptides to trap and release precursor ions in a TIMS-TOF mass spectrometer for an almost complete sampling of the precursor ion beam with data-independent acquisition.},
  archive      = {J_NMETH},
  author       = {Meier, Florian and Brunner, Andreas-David and Frank, Max and Ha, Annie and Bludau, Isabell and Voytik, Eugenia and Kaspar-Schoenefeld, Stephanie and Lubeck, Markus and Raether, Oliver and Bache, Nicolai and Aebersold, Ruedi and Collins, Ben C. and Röst, Hannes L. and Mann, Matthias},
  doi          = {10.1038/s41592-020-00998-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1229-1236},
  shortjournal = {Nat. Meth.},
  title        = {DiaPASEF: Parallel accumulation–serial fragmentation combined with data-independent acquisition},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Quantitative shotgun proteome analysis by direct infusion.
<em>NMETH</em>, <em>17</em>(12), 1222–1228. (<a
href="https://doi.org/10.1038/s41592-020-00999-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Liquid chromatography–mass spectrometry (LC–MS) delivers sensitive peptide analysis for proteomics but requires extensive analysis time, reducing throughput. Here, we demonstrate that gas-phase peptide separation instead of LC enables fast proteome analysis. Using direct infusion–shotgun proteome analysis (DI-SPA) by data-independent acquisition mass spectrometry (DIA-MS), we demonstrate the targeted quantification of over 500 proteins within minutes of MS data collection (~3.5 proteins per second). We show the utility of this technology in performing a complex multifactorial proteomic study of interactions between nutrients, genotype and mitochondrial toxins in a collection of cultured human cells. More than 45,000 quantitative protein measurements from 132 samples were achieved in only ~4.4 h of MS data collection. Enabling fast, unbiased proteome quantification without LC, DI-SPA offers an approach to boost throughput, critical to drug and biomarker discovery studies that require analysis of thousands of proteomes. Direct infusion–shotgun proteome analysis (DI-SPA) using data-independent acquisition mass spectrometry (DIA-MS) achieves fast and reproducible results by omitting the liquid-chromatography fractionation step and directly performing gas-phase peptide fractionation by ion mobility.},
  archive      = {J_NMETH},
  author       = {Meyer, Jesse G. and Niemi, Natalie M. and Pagliarini, David J. and Coon, Joshua J.},
  doi          = {10.1038/s41592-020-00999-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1222-1228},
  shortjournal = {Nat. Meth.},
  title        = {Quantitative shotgun proteome analysis by direct infusion},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Non-uniform refinement: Adaptive regularization improves
single-particle cryo-EM reconstruction. <em>NMETH</em>, <em>17</em>(12),
1214–1221. (<a
href="https://doi.org/10.1038/s41592-020-00990-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron microscopy (cryo-EM) is widely used to study biological macromolecules that comprise regions with disorder, flexibility or partial occupancy. For example, membrane proteins are often kept in solution with detergent micelles and lipid nanodiscs that are locally disordered. Such spatial variability negatively impacts computational three-dimensional (3D) reconstruction with existing iterative refinement algorithms that assume rigidity. We introduce non-uniform refinement, an algorithm based on cross-validation optimization, which automatically regularizes 3D density maps during refinement to account for spatial variability. Unlike common shift-invariant regularizers, non-uniform refinement systematically removes noise from disordered regions, while retaining signal useful for aligning particle images, yielding dramatically improved resolution and 3D map quality in many cases. We obtain high-resolution reconstructions for multiple membrane proteins as small as 100 kDa, demonstrating increased effectiveness of cryo-EM for this class of targets critical in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package. Membrane proteins exhibit spatial variation in rigidity and disorder, which poses a challenge for traditional cryo-EM reconstruction algorithms. Non-uniform refinement accounts for this spatial variability, yielding improved 3D reconstruction quality even for small membrane proteins.},
  archive      = {J_NMETH},
  author       = {Punjani, Ali and Zhang, Haowei and Fleet, David J.},
  doi          = {10.1038/s41592-020-00990-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1214-1221},
  shortjournal = {Nat. Meth.},
  title        = {Non-uniform refinement: Adaptive regularization improves single-particle cryo-EM reconstruction},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Quantify and control reproducibility in high-throughput
experiments. <em>NMETH</em>, <em>17</em>(12), 1207–1213. (<a
href="https://doi.org/10.1038/s41592-020-00978-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ensuring reproducibility of results in high-throughput experiments is crucial for biomedical research. Here, we propose a set of computational methods, INTRIGUE, to evaluate and control reproducibility in high-throughput settings. Our approaches are built on a new definition of reproducibility that emphasizes directional consistency when experimental units are assessed with signed effect size estimates. The proposed methods are designed to (1) assess the overall reproducible quality of multiple studies and (2) evaluate reproducibility at the individual experimental unit levels. We demonstrate the proposed methods in detecting unobserved batch effects via simulations. We further illustrate the versatility of the proposed methods in transcriptome-wide association studies: in addition to reproducible quality control, they are also suited to investigating genuine biological heterogeneity. Finally, we discuss the potential extensions of the proposed methods in other vital areas of reproducible research (for example, publication bias and conceptual replications). INTRIGUE is a statistical framework based on the directional consistency criterion for quantifying and controlling reproducibility in high-throughput experiments.},
  archive      = {J_NMETH},
  author       = {Zhao, Yi and Sampson, Matthew G. and Wen, Xiaoquan},
  doi          = {10.1038/s41592-020-00978-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1207-1213},
  shortjournal = {Nat. Meth.},
  title        = {Quantify and control reproducibility in high-throughput experiments},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). MARS: Discovering novel cell types across heterogeneous
single-cell experiments. <em>NMETH</em>, <em>17</em>(12), 1200–1206. (<a
href="https://doi.org/10.1038/s41592-020-00979-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although tremendous effort has been put into cell-type annotation, identification of previously uncharacterized cell types in heterogeneous single-cell RNA-seq data remains a challenge. Here we present MARS, a meta-learning approach for identifying and annotating known as well as new cell types. MARS overcomes the heterogeneity of cell types by transferring latent cell representations across multiple datasets. MARS uses deep learning to learn a cell embedding function as well as a set of landmarks in the cell embedding space. The method has a unique ability to discover cell types that have never been seen before and annotate experiments that are as yet unannotated. We apply MARS to a large mouse cell atlas and show its ability to accurately identify cell types, even when it has never seen them before. Further, MARS automatically generates interpretable names for new cell types by probabilistically defining a cell type in the embedding space. MARS uses a meta-learning strategy for annotating known cell types and identifying novel ones across single-cell RNA-seq datasets.},
  archive      = {J_NMETH},
  author       = {Brbić, Maria and Zitnik, Marinka and Wang, Sheng and Pisco, Angela O. and Altman, Russ B. and Darmanis, Spyros and Leskovec, Jure},
  doi          = {10.1038/s41592-020-00979-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1200-1206},
  shortjournal = {Nat. Meth.},
  title        = {MARS: Discovering novel cell types across heterogeneous single-cell experiments},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Simultaneous profiling of chromatin accessibility and
methylation on human cell lines with nanopore sequencing.
<em>NMETH</em>, <em>17</em>(12), 1191–1199. (<a
href="https://doi.org/10.1038/s41592-020-01000-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Probing epigenetic features on DNA has tremendous potential to advance our understanding of the phased epigenome. In this study, we use nanopore sequencing to evaluate CpG methylation and chromatin accessibility simultaneously on long strands of DNA by applying GpC methyltransferase to exogenously label open chromatin. We performed nanopore sequencing of nucleosome occupancy and methylome (nanoNOMe) on four human cell lines (GM12878, MCF-10A, MCF-7 and MDA-MB-231). The single-molecule resolution allows footprinting of protein and nucleosome binding, and determination of the combinatorial promoter epigenetic signature on individual molecules. Long-read sequencing makes it possible to robustly assign reads to haplotypes, allowing us to generate a fully phased human epigenome, consisting of chromosome-level allele-specific profiles of CpG methylation and chromatin accessibility. We further apply this to a breast cancer model to evaluate differential methylation and accessibility between cancerous and noncancerous cells. Using nanopore sequencing as readout, nanoNOMe–seq enables chromosome-level allele-specific profiles of CpG methylation and chromatin accessibility on human cells, in which the chromatin accessibility is profiled through exogenous GpC methylation introduced by a GpC methyltransferase.},
  archive      = {J_NMETH},
  author       = {Lee, Isac and Razaghi, Roham and Gilpatrick, Timothy and Molnar, Michael and Gershman, Ariel and Sadowski, Norah and Sedlazeck, Fritz J. and Hansen, Kasper D. and Simpson, Jared T. and Timp, Winston},
  doi          = {10.1038/s41592-020-01000-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1191-1199},
  shortjournal = {Nat. Meth.},
  title        = {Simultaneous profiling of chromatin accessibility and methylation on human cell lines with nanopore sequencing},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A compact cascade–Cas3 system for targeted genome
engineering. <em>NMETH</em>, <em>17</em>(12), 1183–1190. (<a
href="https://doi.org/10.1038/s41592-020-00980-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas technologies have enabled programmable gene editing in eukaryotes and prokaryotes. However, the leading Cas9 and Cas12a enzymes are limited in their ability to make large deletions. Here, we used the processive nuclease Cas3, together with a minimal Type I-C Cascade-based system for targeted genome engineering in bacteria. DNA cleavage guided by a single CRISPR RNA generated large deletions (7–424 kilobases) in Pseudomonas aeruginosa with near-100\% efficiency, while Cas9 yielded small deletions and point mutations. Cas3 generated bidirectional deletions originating from the programmed site, which was exploited to reduce the P. aeruginosa genome by 837 kb (13.5\%). Large deletion boundaries were efficiently specified by a homology-directed repair template during editing with Cascade–Cas3, but not Cas9. A transferable ‘all-in-one’ vector was functional in Escherichia coli, Pseudomonas syringae and Klebsiella pneumoniae, and endogenous CRISPR–Cas use was enhanced with an ‘anti-anti-CRISPR’ strategy. P. aeruginosa Type I-C Cascade–Cas3 (PaeCas3c) facilitates rapid strain manipulation with applications in synthetic biology, genome minimization and the removal of large genomic regions. This work repurposes the Type I-C Cascade–Cas3 system from Pseudomonas aeruginosa to achieve large deletions in bacterial genomes.},
  archive      = {J_NMETH},
  author       = {Csörgő, Bálint and León, Lina M. and Chau-Ly, Ilea J. and Vasquez-Rifo, Alejandro and Berry, Joel D. and Mahendra, Caroline and Crawford, Emily D. and Lewis, Jennifer D. and Bondy-Denomy, Joseph},
  doi          = {10.1038/s41592-020-00980-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1183-1190},
  shortjournal = {Nat. Meth.},
  title        = {A compact Cascade–Cas3 system for targeted genome engineering},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Guide RNAs: It’s good to be choosy. <em>NMETH</em>,
<em>17</em>(12), 1179–1182. (<a
href="https://doi.org/10.1038/s41592-020-01003-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR, now a Nobel-Prize-winning technology, comes in many flavors. And so do gRNAs.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-01003-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1179-1182},
  shortjournal = {Nat. Meth.},
  title        = {Guide RNAs: It’s good to be choosy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020aj). Stamping RNA age. <em>NMETH</em>, <em>17</em>(12), 1177.
(<a href="https://doi.org/10.1038/s41592-020-01016-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA timestamps are molecular recorders that enable one to infer the age of an RNA molecule and record transcriptional activity in a temporal manner.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-01016-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1177},
  shortjournal = {Nat. Meth.},
  title        = {Stamping RNA age},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). Electrical functional imaging. <em>NMETH</em>,
<em>17</em>(12), 1176. (<a
href="https://doi.org/10.1038/s41592-020-01015-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-density microelectrode arrays with dual-mode capabilities can ‘image’ electrical activity in neuronal cultures or slice preparations.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-01015-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1176},
  shortjournal = {Nat. Meth.},
  title        = {Electrical functional imaging},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Cryo-EM goes atomic. <em>NMETH</em>, <em>17</em>(12), 1175.
(<a href="https://doi.org/10.1038/s41592-020-01014-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technological advances push the limits of single-particle cryo-EM to achieve atomic resolution.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-01014-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1175},
  shortjournal = {Nat. Meth.},
  title        = {Cryo-EM goes atomic},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). RefMet: A reference nomenclature for metabolomics.
<em>NMETH</em>, <em>17</em>(12), 1173–1174. (<a
href="https://doi.org/10.1038/s41592-020-01009-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Fahy, Eoin and Subramaniam, Shankar},
  doi          = {10.1038/s41592-020-01009-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1173-1174},
  shortjournal = {Nat. Meth.},
  title        = {RefMet: A reference nomenclature for metabolomics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The method comes first. <em>NMETH</em>, <em>17</em>(12),
1169. (<a href="https://doi.org/10.1038/s41592-020-01017-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new method should be thoroughly tested, applied, described — and peer-reviewed — before biological discoveries generated using the method are published.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-01017-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1169},
  shortjournal = {Nat. Meth.},
  title        = {The method comes first},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). Henry colecraft. <em>NMETH</em>, <em>17</em>(12), 1171. (<a
href="https://doi.org/10.1038/s41592-020-01012-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A soccer striker and his ion channel rescue tools.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-01012-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {12},
  pages        = {1171},
  shortjournal = {Nat. Meth.},
  title        = {Henry colecraft},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Quantitative mapping and minimization of
super-resolution optical imaging artifacts. <em>NMETH</em>,
<em>17</em>(11), 1167. (<a
href="https://doi.org/10.1038/s41592-020-00983-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Culley, Siân and Albrecht, David and Jacobs, Caron and Pereira, Pedro Matos and Leterrier, Christophe and Mercer, Jason and Henriques, Ricardo},
  doi          = {10.1038/s41592-020-00983-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1167},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Quantitative mapping and minimization of super-resolution optical imaging artifacts},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Genetically encoded tags for direct
synthesis of EM-visible gold nanoparticles in cells. <em>NMETH</em>,
<em>17</em>(11), 1167. (<a
href="https://doi.org/10.1038/s41592-020-00996-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Jiang, Zhaodi and Jin, Xiumei and Li, Yuhua and Liu, Sitong and Liu, Xiao-Man and Wang, Ying-Ying and Zhao, Pei and Cai, Xinbin and Liu, Ying and Tang, Yaqi and Sun, Xiaobin and Liu, Yan and Hu, Yanyong and Li, Ming and Cai, Gaihong and Qi, Xiangbing and Chen, She and Du, Li-Lin and He, Wanzhong},
  doi          = {10.1038/s41592-020-00996-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1167},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Genetically encoded tags for direct synthesis of EM-visible gold nanoparticles in cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Next-generation GRAB sensors for monitoring dopaminergic
activity in vivo. <em>NMETH</em>, <em>17</em>(11), 1156–1166. (<a
href="https://doi.org/10.1038/s41592-020-00981-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dopamine (DA) plays a critical role in the brain, and the ability to directly measure dopaminergic activity is essential for understanding its physiological functions. We therefore developed red fluorescent G-protein-coupled receptor-activation-based DA (GRABDA) sensors and optimized versions of green fluorescent GRABDA sensors. In response to extracellular DA, both the red and green GRABDA sensors exhibit a large increase in fluorescence, with subcellular resolution, subsecond kinetics and nanomolar-to-submicromolar affinity. Moreover, the GRABDA sensors resolve evoked DA release in mouse brain slices, detect evoked compartmental DA release from a single neuron in live flies and report optogenetically elicited nigrostriatal DA release as well as mesoaccumbens dopaminergic activity during sexual behavior in freely behaving mice. Coexpressing red GRABDA with either green GRABDA or the calcium indicator GCaMP6s allows tracking of dopaminergic signaling and neuronal activity in distinct circuits in vivo. Red and improved green versions of the genetically encoded dopamine sensor GRABDA have been developed. These neurotransmitter sensors are used alone or in combination with, for example, calcium sensors in behaving fruit flies and rodents.},
  archive      = {J_NMETH},
  author       = {Sun, Fangmiao and Zhou, Jingheng and Dai, Bing and Qian, Tongrui and Zeng, Jianzhi and Li, Xuelin and Zhuo, Yizhou and Zhang, Yajun and Wang, Yipan and Qian, Cheng and Tan, Ke and Feng, Jiesi and Dong, Hui and Lin, Dayu and Cui, Guohong and Li, Yulong},
  doi          = {10.1038/s41592-020-00981-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1156-1166},
  shortjournal = {Nat. Meth.},
  title        = {Next-generation GRAB sensors for monitoring dopaminergic activity in vivo},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An expanded palette of dopamine sensors for multiplex
imaging in vivo. <em>NMETH</em>, <em>17</em>(11), 1147–1155. (<a
href="https://doi.org/10.1038/s41592-020-0936-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded dopamine sensors based on green fluorescent protein (GFP) enable high-resolution imaging of dopamine dynamics in behaving animals. However, these GFP-based variants cannot be readily combined with commonly used optical sensors and actuators, due to spectral overlap. We therefore engineered red-shifted variants of dopamine sensors called RdLight1, based on mApple. RdLight1 can be combined with GFP-based sensors with minimal interference and shows high photostability, permitting prolonged continuous imaging. We demonstrate the utility of RdLight1 for receptor-specific pharmacological analysis in cell culture, simultaneous assessment of dopamine release and cell-type-specific neuronal activity and simultaneous subsecond monitoring of multiple neurotransmitters in freely behaving rats. Dual-color photometry revealed that dopamine release in the nucleus accumbens evoked by reward-predictive cues is accompanied by a rapid suppression of glutamate release. By enabling multiplexed imaging of dopamine with other circuit components in vivo, RdLight1 opens avenues for understanding many aspects of dopamine biology. Red and yellow versions of the genetically encoded dopamine sensor dLight1 have been developed and allow multiplexed imaging of dopamine with neurotransmitter or cell-type-specific calcium combined with green sensors or actuators, as demonstrated ex vivo and in behaving rodents.},
  archive      = {J_NMETH},
  author       = {Patriarchi, Tommaso and Mohebi, Ali and Sun, Junqing and Marley, Aaron and Liang, Ruqiang and Dong, Chunyang and Puhger, Kyle and Mizuno, Grace Or and Davis, Carolyn M. and Wiltgen, Brian and von Zastrow, Mark and Berke, Joshua D. and Tian, Lin},
  doi          = {10.1038/s41592-020-0936-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1147-1155},
  shortjournal = {Nat. Meth.},
  title        = {An expanded palette of dopamine sensors for multiplex imaging in vivo},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An optimized acetylcholine sensor for monitoring in vivo
cholinergic activity. <em>NMETH</em>, <em>17</em>(11), 1139–1146. (<a
href="https://doi.org/10.1038/s41592-020-0953-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to directly measure acetylcholine (ACh) release is an essential step toward understanding its physiological function. Here we optimized the GRABACh (GPCR-activation-based ACh) sensor to achieve substantially improved sensitivity in ACh detection, as well as reduced downstream coupling to intracellular pathways. The improved version of the ACh sensor retains the subsecond response kinetics, physiologically relevant affinity and precise molecular specificity for ACh of its predecessor. Using this sensor, we revealed compartmental ACh signals in the olfactory center of transgenic flies in response to external stimuli including odor and body shock. Using fiber photometry recording and two-photon imaging, our ACh sensor also enabled sensitive detection of single-trial ACh dynamics in multiple brain regions in mice performing a variety of behaviors. A genetically encoded acetylcholine sensor with improved sensitivity allows detection of cholinergic neurotransmission in vivo in the Drosophila and mouse brain.},
  archive      = {J_NMETH},
  author       = {Jing, Miao and Li, Yuexuan and Zeng, Jianzhi and Huang, Pengcheng and Skirzewski, Miguel and Kljakic, Ornela and Peng, Wanling and Qian, Tongrui and Tan, Ke and Zou, Jing and Trinh, Simon and Wu, Runlong and Zhang, Shichen and Pan, Sunlei and Hires, Samuel A. and Xu, Min and Li, Haohong and Saksida, Lisa M. and Prado, Vania F. and Bussey, Timothy J. and Prado, Marco A. M. and Chen, Liangyi and Cheng, Heping and Li, Yulong},
  doi          = {10.1038/s41592-020-0953-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1139-1146},
  shortjournal = {Nat. Meth.},
  title        = {An optimized acetylcholine sensor for monitoring in vivo cholinergic activity},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). O-pair search with MetaMorpheus for o-glycopeptide
characterization. <em>NMETH</em>, <em>17</em>(11), 1133–1138. (<a
href="https://doi.org/10.1038/s41592-020-00985-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report O-Pair Search, an approach to identify O-glycopeptides and localize O-glycosites. Using paired collision- and electron-based dissociation spectra, O-Pair Search identifies O-glycopeptides via an ion-indexed open modification search and localizes O-glycosites using graph theory and probability-based localization. O-Pair Search reduces search times more than 2,000-fold compared to current O-glycopeptide processing software, while defining O-glycosite localization confidence levels and generating more O-glycopeptide identifications. Beyond the mucin-type O-glycopeptides discussed here, O-Pair Search also accepts user-defined glycan databases, making it compatible with many types of O-glycosylation. O-Pair Search is freely available within the open-source MetaMorpheus platform at https://github.com/smith-chem-wisc/MetaMorpheus . O-Pair search identifies O-glycopeptides and localizes O-glycosites using a fragment-ion-indexed open modification search combined with a graph-based approach. It also introduces a classification scheme to unify data reporting for glycoproteomics.},
  archive      = {J_NMETH},
  author       = {Lu, Lei and Riley, Nicholas M. and Shortreed, Michael R. and Bertozzi, Carolyn R. and Smith, Lloyd M.},
  doi          = {10.1038/s41592-020-00985-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1133-1138},
  shortjournal = {Nat. Meth.},
  title        = {O-pair search with MetaMorpheus for O-glycopeptide characterization},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Fast and comprehensive n- and o-glycoproteomics analysis
with MSFragger-glyco. <em>NMETH</em>, <em>17</em>(11), 1125–1132. (<a
href="https://doi.org/10.1038/s41592-020-0967-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in methods for enrichment and mass spectrometric analysis of intact glycopeptides have produced large-scale glycoproteomics datasets, but interpreting these data remains challenging. We present MSFragger-Glyco, a glycoproteomics mode of the MSFragger search engine, for fast and sensitive identification of N- and O-linked glycopeptides and open glycan searches. Reanalysis of recent N-glycoproteomics data resulted in annotation of 80\% more glycopeptide spectrum matches (glycoPSMs) than previously reported. In published O-glycoproteomics data, our method more than doubled the number of glycoPSMs annotated when searching the same glycans as the original search, and yielded 4- to 6-fold increases when expanding searches to include additional glycan compositions and other modifications. Expanded searches also revealed many sulfated and complex glycans that remained hidden to the original search. With greatly improved spectral annotation, coupled with the speed of index-based scoring, MSFragger-Glyco makes it possible to comprehensively interrogate glycoproteomics data and illuminate the many roles of glycosylation. MSFragger-Glyco allows identification of N- and O-linked glycopeptides using the localization-aware open search strategy of the MSFragger search engine.},
  archive      = {J_NMETH},
  author       = {Polasky, Daniel A. and Yu, Fengchao and Teo, Guo Ci and Nesvizhskii, Alexey I.},
  doi          = {10.1038/s41592-020-0967-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1125-1132},
  shortjournal = {Nat. Meth.},
  title        = {Fast and comprehensive N- and O-glycoproteomics analysis with MSFragger-glyco},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DeepC: Predicting 3D genome folding using megabase-scale
transfer learning. <em>NMETH</em>, <em>17</em>(11), 1118–1124. (<a
href="https://doi.org/10.1038/s41592-020-0960-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the impact of noncoding genetic variation requires interpreting it in the context of three-dimensional genome architecture. We have developed deepC, a transfer-learning-based deep neural network that accurately predicts genome folding from megabase-scale DNA sequence. DeepC predicts domain boundaries at high resolution, learns the sequence determinants of genome folding and predicts the impact of both large-scale structural and single base-pair variations. DeepC uses transfer learning-based deep neural networks for predicting genome folding from megabase-scale DNA sequence.},
  archive      = {J_NMETH},
  author       = {Schwessinger, Ron and Gosden, Matthew and Downes, Damien and Brown, Richard C. and Oudelaar, A. Marieke and Telenius, Jelena and Teh, Yee Whye and Lunter, Gerton and Hughes, Jim R.},
  doi          = {10.1038/s41592-020-0960-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1118-1124},
  shortjournal = {Nat. Meth.},
  title        = {DeepC: Predicting 3D genome folding using megabase-scale transfer learning},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Predicting 3D genome folding from DNA sequence with akita.
<em>NMETH</em>, <em>17</em>(11), 1111–1117. (<a
href="https://doi.org/10.1038/s41592-020-0958-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In interphase, the human genome sequence folds in three dimensions into a rich variety of locus-specific contact patterns. Cohesin and CTCF (CCCTC-binding factor) are key regulators; perturbing the levels of either greatly disrupts genome-wide folding as assayed by chromosome conformation capture methods. Still, how a given DNA sequence encodes a particular locus-specific folding pattern remains unknown. Here we present a convolutional neural network, Akita, that accurately predicts genome folding from DNA sequence alone. Representations learned by Akita underscore the importance of an orientation-specific grammar for CTCF binding sites. Akita learns predictive nucleotide-level features of genome folding, revealing effects of nucleotides beyond the core CTCF motif. Once trained, Akita enables rapid in silico predictions. Accounting for this, we demonstrate how Akita can be used to perform in silico saturation mutagenesis, interpret eQTLs, make predictions for structural variants and probe species-specific genome folding. Collectively, these results enable decoding genome function from sequence through structure. Akita enables three-dimensional genome folding predictions from DNA sequence using a convolutional neural network.},
  archive      = {J_NMETH},
  author       = {Fudenberg, Geoff and Kelley, David R. and Pollard, Katherine S.},
  doi          = {10.1038/s41592-020-0958-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1111-1117},
  shortjournal = {Nat. Meth.},
  title        = {Predicting 3D genome folding from DNA sequence with akita},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). MetaFlye: Scalable long-read metagenome assembly using
repeat graphs. <em>NMETH</em>, <em>17</em>(11), 1103–1110. (<a
href="https://doi.org/10.1038/s41592-020-00971-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing technologies have substantially improved the assemblies of many isolate bacterial genomes as compared to fragmented short-read assemblies. However, assembling complex metagenomic datasets remains difficult even for state-of-the-art long-read assemblers. Here we present metaFlye, which addresses important long-read metagenomic assembly challenges, such as uneven bacterial composition and intra-species heterogeneity. First, we benchmarked metaFlye using simulated and mock bacterial communities and show that it consistently produces assemblies with better completeness and contiguity than state-of-the-art long-read assemblers. Second, we performed long-read sequencing of the sheep microbiome and applied metaFlye to reconstruct 63 complete or nearly complete bacterial genomes within single contigs. Finally, we show that long-read assembly of human microbiomes enables the discovery of full-length biosynthetic gene clusters that encode biomedically important natural products. Long-read metagenomics offers a valuable approach for profiling bacterial communities. This work presents a long-read assembler, metaFlye, that specifically addresses the challenges of assembling metagenomes.},
  archive      = {J_NMETH},
  author       = {Kolmogorov, Mikhail and Bickhart, Derek M. and Behsaz, Bahar and Gurevich, Alexey and Rayko, Mikhail and Shin, Sung Bong and Kuhn, Kristen and Yuan, Jeffrey and Polevikov, Evgeny and Smith, Timothy P. L. and Pevzner, Pavel A.},
  doi          = {10.1038/s41592-020-00971-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1103-1110},
  shortjournal = {Nat. Meth.},
  title        = {MetaFlye: Scalable long-read metagenome assembly using repeat graphs},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genuage: Visualize and analyze multidimensional
single-molecule point cloud data in virtual reality. <em>NMETH</em>,
<em>17</em>(11), 1100–1102. (<a
href="https://doi.org/10.1038/s41592-020-0946-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimentally recorded point cloud data, such as those generated by single-molecule localization microscopy, are continuously increasing in size and dimension. Gaining an intuitive understanding and facilitating the analysis of such multidimensional data remains challenging. Here we report a new open-source software platform, Genuage, that enables the easy perception of, interaction with and analysis of multidimensional point clouds in virtual reality. Genuage is compatible with arbitrary multidimensional data extending beyond single-molecule localization microscopy. An open-source software platform, Genuage, enables the easy perception of, interaction with and analysis of multidimensional point clouds in virtual reality.},
  archive      = {J_NMETH},
  author       = {Blanc, Thomas and El Beheiry, Mohamed and Caporal, Clément and Masson, Jean-Baptiste and Hajj, Bassam},
  doi          = {10.1038/s41592-020-0946-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1100-1102},
  shortjournal = {Nat. Meth.},
  title        = {Genuage: Visualize and analyze multidimensional single-molecule point cloud data in virtual reality},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). VLUME: 3D virtual reality for single-molecule localization
microscopy. <em>NMETH</em>, <em>17</em>(11), 1097–1099. (<a
href="https://doi.org/10.1038/s41592-020-0962-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {vLUME is a virtual reality software package designed to render large three-dimensional single-molecule localization microscopy datasets. vLUME features include visualization, segmentation, bespoke analysis of complex local geometries and exporting features. vLUME can perform complex analysis on real three-dimensional biological samples that would otherwise be impossible by using regular flat-screen visualization programs. vLUME is a complete virtual reality environment for visualizing, analyzing and interacting with three-dimensional single-molecule localization microscopy data.},
  archive      = {J_NMETH},
  author       = {Spark, Alexander and Kitching, Alexandre and Esteban-Ferrer, Daniel and Handa, Anoushka and Carr, Alexander R. and Needham, Lisa-Maria and Ponjavic, Aleks and Santos, Ana Mafalda and McColl, James and Leterrier, Christophe and Davis, Simon J. and Henriques, Ricardo and Lee, Steven F.},
  doi          = {10.1038/s41592-020-0962-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1097-1099},
  shortjournal = {Nat. Meth.},
  title        = {VLUME: 3D virtual reality for single-molecule localization microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An efficient KRAB domain for CRISPRi applications in human
cells. <em>NMETH</em>, <em>17</em>(11), 1093–1096. (<a
href="https://doi.org/10.1038/s41592-020-0966-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clustered regularly interspaced short palindromic repeat interference (CRISPRi), based on the fusion of inactive Cas9 (dCas9) to the Krüppel-associated box (KRAB) repressor, is a powerful platform for silencing gene expression. However, it suffers from incomplete silencing of target genes. We assayed 57 KRAB domains for their repressive potency and identified the ZIM3 KRAB domain as an exceptionally potent repressor. We establish that ZIM3 KRAB–dCas9 fusion silences gene expression more efficiently than existing platforms. CRISPRi activities are measured from dCas9 fusions with 57 KRAB domains, which lead to the identification of ZIM3 KRAB–dCas9 fusion as a superior repressor.},
  archive      = {J_NMETH},
  author       = {Alerasool, Nader and Segal, Dmitri and Lee, Hunsang and Taipale, Mikko},
  doi          = {10.1038/s41592-020-0966-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1093-1096},
  shortjournal = {Nat. Meth.},
  title        = {An efficient KRAB domain for CRISPRi applications in human cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A systematic evaluation of the design and context
dependencies of massively parallel reporter assays. <em>NMETH</em>,
<em>17</em>(11), 1083–1091. (<a
href="https://doi.org/10.1038/s41592-020-0965-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Massively parallel reporter assays (MPRAs) functionally screen thousands of sequences for regulatory activity in parallel. To date, there are limited studies that systematically compare differences in MPRA design. Here, we screen a library of 2,440 candidate liver enhancers and controls for regulatory activity in HepG2 cells using nine different MPRA designs. We identify subtle but significant differences that correlate with epigenetic and sequence-level features, as well as differences in dynamic range and reproducibility. We also validate that enhancer activity is largely independent of orientation, at least for our library and designs. Finally, we assemble and test the same enhancers as 192-mers, 354-mers and 678-mers and observe sizable differences. This work provides a framework for the experimental design of high-throughput reporter assays, suggesting that the extended sequence context of tested elements and to a lesser degree the precise assay, influence MPRA results. Massively parallel reporter assays (MPRAs) enable high-throughput assessments of regulatory elements in single experiments. This work compares nine MPRA designs and reports how differences in reporter assays influence the results of MPRAs.},
  archive      = {J_NMETH},
  author       = {Klein, Jason C. and Agarwal, Vikram and Inoue, Fumitaka and Keith, Aidan and Martin, Beth and Kircher, Martin and Ahituv, Nadav and Shendure, Jay},
  doi          = {10.1038/s41592-020-0965-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1083-1091},
  shortjournal = {Nat. Meth.},
  title        = {A systematic evaluation of the design and context dependencies of massively parallel reporter assays},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Getting more for less: New software solutions for
glycoproteomics. <em>NMETH</em>, <em>17</em>(11), 1081–1082. (<a
href="https://doi.org/10.1038/s41592-020-00987-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Algorithms from the Nesvizhskii and Smith groups facilitate faster and more comprehensive glycopeptide assignments from mass spectrometry datasets.},
  archive      = {J_NMETH},
  author       = {Praissman, Jeremy L. and Wells, Lance},
  doi          = {10.1038/s41592-020-00987-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1081-1082},
  shortjournal = {Nat. Meth.},
  title        = {Getting more for less: New software solutions for glycoproteomics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Smart solutions for automated imaging. <em>NMETH</em>,
<em>17</em>(11), 1075–1079. (<a
href="https://doi.org/10.1038/s41592-020-00988-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Algorithms trained to interpret microscope data can greatly extend the information that can be derived from the resulting images, or even optimize how imaging experiments are conducted.},
  archive      = {J_NMETH},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41592-020-00988-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1075-1079},
  shortjournal = {Nat. Meth.},
  title        = {Smart solutions for automated imaging},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). De novo design of small-molecule-binding proteins.
<em>NMETH</em>, <em>17</em>(11), 1073. (<a
href="https://doi.org/10.1038/s41592-020-00995-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {van der Mers (vdMs), a unit of protein structure, facilitate de novo design of proteins that bind small molecules.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-00995-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1073},
  shortjournal = {Nat. Meth.},
  title        = {De novo design of small-molecule-binding proteins},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020z). X-ray connectomics. <em>NMETH</em>, <em>17</em>(11), 1072.
(<a href="https://doi.org/10.1038/s41592-020-00994-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {X-ray holographic nano-tomography is a promising complement to electron microscopy for connectomics studies.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-00994-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1072},
  shortjournal = {Nat. Meth.},
  title        = {X-ray connectomics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Metabolic profiling of CD8+ t cells at the single-cell
level. <em>NMETH</em>, <em>17</em>(11), 1071. (<a
href="https://doi.org/10.1038/s41592-020-00993-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A single-cell approach to metabolomics links metabolic profiles to cellular identity.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-00993-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1071},
  shortjournal = {Nat. Meth.},
  title        = {Metabolic profiling of CD8+ t cells at the single-cell level},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Non-animal-derived monoclonal antibodies are not ready to
substitute current hybridoma technology. <em>NMETH</em>,
<em>17</em>(11), 1069–1070. (<a
href="https://doi.org/10.1038/s41592-020-00977-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {González-Fernández, África and Bermúdez Silva, Francisco J. and López-Hoyos, Marcos and Cobaleda, César and Montoliu, Lluís and Del Val, Margarita and Leech, Kirk},
  doi          = {10.1038/s41592-020-00977-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1069-1070},
  shortjournal = {Nat. Meth.},
  title        = {Non-animal-derived monoclonal antibodies are not ready to substitute current hybridoma technology},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The power of a picture. <em>NMETH</em>, <em>17</em>(11),
1065. (<a href="https://doi.org/10.1038/s41592-020-00997-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This month, we celebrate the beauty and power of microscopy images.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-00997-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1065},
  shortjournal = {Nat. Meth.},
  title        = {The power of a picture},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020n). Lin tian. <em>NMETH</em>, <em>17</em>(11), 1067. (<a
href="https://doi.org/10.1038/s41592-020-00991-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neurochemistry sensors for brain research and life with a collaborative frame of mind.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-00991-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {11},
  pages        = {1067},
  shortjournal = {Nat. Meth.},
  title        = {Lin tian},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Addendum: Parameter-free image resolution estimation based
on decorrelation analysis. <em>NMETH</em>, <em>17</em>(10), 1061–1063.
(<a href="https://doi.org/10.1038/s41592-020-0963-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Descloux, A. and Grußmayer, K. S. and Radenovic, A.},
  doi          = {10.1038/s41592-020-0963-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1061-1063},
  shortjournal = {Nat. Meth.},
  title        = {Addendum: Parameter-free image resolution estimation based on decorrelation analysis},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Molecular motion on ice.
<em>NMETH</em>, <em>17</em>(10), 1061. (<a
href="https://doi.org/10.1038/s41592-020-0970-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Dance, Amber},
  doi          = {10.1038/s41592-020-0970-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1061},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Molecular motion on ice},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: High-sensitivity and high-specificity
biomechanical imaging by stimulated brillouin scattering microscopy.
<em>NMETH</em>, <em>17</em>(10), 1060. (<a
href="https://doi.org/10.1038/s41592-020-0956-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Remer, Itay and Shaashoua, Roni and Shemesh, Netta and Ben-Zvi, Anat and Bilenca, Alberto},
  doi          = {10.1038/s41592-020-0956-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1060},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: High-sensitivity and high-specificity biomechanical imaging by stimulated brillouin scattering microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Nested designs. <em>NMETH</em>,
<em>17</em>(10), 1060. (<a
href="https://doi.org/10.1038/s41592-020-0969-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Krzywinski, Martin and Altman, Naomi and Blainey, Paul},
  doi          = {10.1038/s41592-020-0969-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1060},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Nested designs},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). EthoLoop: Automated closed-loop neuroethology in
naturalistic environments. <em>NMETH</em>, <em>17</em>(10), 1052–1059.
(<a href="https://doi.org/10.1038/s41592-020-0961-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate tracking and analysis of animal behavior is crucial for modern systems neuroscience. However, following freely moving animals in naturalistic, three-dimensional (3D) or nocturnal environments remains a major challenge. Here, we present EthoLoop, a framework for studying the neuroethology of freely roaming animals. Combining real-time optical tracking and behavioral analysis with remote-controlled stimulus-reward boxes, this system allows direct interactions with animals in their habitat. EthoLoop continuously provides close-up views of the tracked individuals and thus allows high-resolution behavioral analysis using deep-learning methods. The behaviors detected on the fly can be automatically reinforced either by classical conditioning or by optogenetic stimulation via wirelessly controlled portable devices. Finally, by combining 3D tracking with wireless neurophysiology we demonstrate the existence of place-cell-like activity in the hippocampus of freely moving primates. Taken together, we show that the EthoLoop framework enables interactive, well-controlled and reproducible neuroethological studies in large-field naturalistic settings. EthoLoop enables real-time tracking and behavioral analysis of animals in naturalistic environments and can be combined with behavioral conditioning, optogenetic stimulation or wireless recording of neural activity. The system is illustrated with freely behaving mice and mouse lemurs.},
  archive      = {J_NMETH},
  author       = {Nourizonoz, Ali and Zimmermann, Robert and Ho, Chun Lum Andy and Pellat, Sebastien and Ormen, Yannick and Prévost-Solié, Clément and Reymond, Gilles and Pifferi, Fabien and Aujard, Fabienne and Herrel, Anthony and Huber, Daniel},
  doi          = {10.1038/s41592-020-0961-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1052-1059},
  shortjournal = {Nat. Meth.},
  title        = {EthoLoop: Automated closed-loop neuroethology in naturalistic environments},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Scale-free vertical tracking microscopy. <em>NMETH</em>,
<em>17</em>(10), 1040–1051. (<a
href="https://doi.org/10.1038/s41592-020-0924-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The behavior and microscale processes associated with freely suspended organisms, along with sinking particles underlie key ecological processes in the ocean. Mechanistically studying such multiscale processes in the laboratory presents a considerable challenge for microscopy: how to measure single cells at microscale resolution, while allowing them to freely move hundreds of meters in the vertical direction? Here we present a solution in the form of a scale-free, vertical tracking microscope, based on a ‘hydrodynamic treadmill’ with no bounds for motion along the axis of gravity. Using this method to bridge spatial scales, we assembled a multiscale behavioral dataset of nonadherent planktonic cells and organisms. Furthermore, we demonstrate a ‘virtual-reality system for single cells’, wherein cell behavior directly controls its ambient environmental parameters, enabling quantitative behavioral assays. Our method and results exemplify a new paradigm of multiscale measurement, wherein one can observe and probe macroscale and ecologically relevant phenomena at microscale resolution. Beyond the marine context, we foresee that our method will allow biological measurements of cells and organisms in a suspended state by freeing them from the confines of the coverslip. Scale-free vertical tracking microscopy based on a ‘hydrodynamic treadmill’ enables measuring long-range movements of freely suspended organisms with high spatiotemporal resolution.},
  archive      = {J_NMETH},
  author       = {Krishnamurthy, Deepak and Li, Hongquan and Benoit du Rey, François and Cambournac, Pierre and Larson, Adam G. and Li, Ethan and Prakash, Manu},
  doi          = {10.1038/s41592-020-0924-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1040-1051},
  shortjournal = {Nat. Meth.},
  title        = {Scale-free vertical tracking microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A temporal decomposition method for identifying venous
effects in task-based fMRI. <em>NMETH</em>, <em>17</em>(10), 1033–1039.
(<a href="https://doi.org/10.1038/s41592-020-0941-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The spatial resolution of functional magnetic resonance imaging (fMRI) is fundamentally limited by effects from large draining veins. Here we describe an analysis method that provides data-driven estimates of these effects in task-based fMRI. The method involves fitting a one-dimensional manifold that characterizes variation in response timecourses observed in a given dataset, and then using identified early and late timecourses as basis functions for decomposing responses into components related to the microvasculature (capillaries and small venules) and the macrovasculature (large veins), respectively. We show the removal of late components substantially reduces the superficial cortical depth bias of fMRI responses and helps eliminate artifacts in cortical activity maps. This method provides insight into the origins of the fMRI signal and can be used to improve the spatial accuracy of fMRI. Temporal decomposition through manifold fitting (TDM) is an analysis technique that decomposes blood oxygenation level dependent (BOLD) responses in task-based fMRI into different components that likely correspond to microvasculature- and macrovasculature-driven signals.},
  archive      = {J_NMETH},
  author       = {Kay, Kendrick and Jamison, Keith W. and Zhang, Ru-Yuan and Uğurbil, Kamil},
  doi          = {10.1038/s41592-020-0941-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1033-1039},
  shortjournal = {Nat. Meth.},
  title        = {A temporal decomposition method for identifying venous effects in task-based fMRI},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). In vivo detection of antigen-specific CD8+ t cells by
immuno-positron emission tomography. <em>NMETH</em>, <em>17</em>(10),
1025–1032. (<a href="https://doi.org/10.1038/s41592-020-0934-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The immune system’s ability to recognize peptides on major histocompatibility molecules contributes to the eradication of cancers and pathogens. Tracking these responses in vivo could help evaluate the efficacy of immune interventions and improve mechanistic understanding of immune responses. For this purpose, we employ synTacs, which are dimeric major histocompatibility molecule scaffolds of defined composition. SynTacs, when labeled with positron-emitting isotopes, can noninvasively image antigen-specific CD8+ T cells in vivo. Using radiolabeled synTacs loaded with the appropriate peptides, we imaged human papillomavirus-specific CD8+ T cells by positron emission tomography in mice bearing human papillomavirus-positive tumors, as well as influenza A virus–specific CD8+ T cells in the lungs of influenza A virus–infected mice. It is thus possible to visualize antigen-specific CD8+ T-cell populations in vivo, which may serve prognostic and diagnostic roles. Antigen-specific CD8+ T cells can be imaged by immunoPET with the help of synTacs, MHC-based tools that bind to relevant T-cell receptors.},
  archive      = {J_NMETH},
  author       = {Woodham, Andrew W. and Zeigler, Stad H. and Zeyang, Ella L. and Kolifrath, Stephen C. and Cheloha, Ross W. and Rashidian, Mohammad and Chaparro, Rodolfo J. and Seidel, Ronald D. and Garforth, Scott J. and Dearling, Jason L. and Mesyngier, Maia and Duddempudi, Phaneendra K. and Packard, Alan B. and Almo, Steven C. and Ploegh, Hidde L.},
  doi          = {10.1038/s41592-020-0934-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1025-1032},
  shortjournal = {Nat. Meth.},
  title        = {In vivo detection of antigen-specific CD8+ t cells by immuno-positron emission tomography},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Live-cell super-resolved PAINT imaging of piconewton
cellular traction forces. <em>NMETH</em>, <em>17</em>(10), 1018–1024.
(<a href="https://doi.org/10.1038/s41592-020-0929-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the vital role of mechanical forces in biology, it still remains a challenge to image cellular force with sub-100-nm resolution. Here, we present tension points accumulation for imaging in nanoscale topography (tPAINT), integrating molecular tension probes with the DNA points accumulation for imaging in nanoscale topography (DNA-PAINT) technique to map piconewton mechanical events with ~25-nm resolution. To perform live-cell dynamic tension imaging, we engineered reversible probes with a cryptic docking site revealed only when the probe experiences forces exceeding a defined mechanical threshold (~7–21 pN). Additionally, we report a second type of irreversible tPAINT probe that exposes its cryptic docking site permanently and thus integrates force history over time, offering improved spatial resolution in exchange for temporal dynamics. We applied both types of tPAINT probes to map integrin receptor forces in live human platelets and mouse embryonic fibroblasts. Importantly, tPAINT revealed a link between platelet forces at the leading edge of cells and the dynamic actin-rich ring nucleated by the Arp2/3 complex. Tension-PAINT integrates molecular tension probes with DNA-PAINT to enable ~25-nm-resolution mapping of piconewton mechanical events. Tension-PAINT can be used to study dynamic forces, and an irreversible variant integrates force history over time.},
  archive      = {J_NMETH},
  author       = {Brockman, Joshua M. and Su, Hanquan and Blanchard, Aaron T. and Duan, Yuxin and Meyer, Travis and Quach, M. Edward and Glazier, Roxanne and Bazrafshan, Alisina and Bender, Rachel L. and Kellner, Anna V. and Ogasawara, Hiroaki and Ma, Rong and Schueder, Florian and Petrich, Brian G. and Jungmann, Ralf and Li, Renhao and Mattheyses, Alexa L. and Ke, Yonggang and Salaita, Khalid},
  doi          = {10.1038/s41592-020-0929-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1018-1024},
  shortjournal = {Nat. Meth.},
  title        = {Live-cell super-resolved PAINT imaging of piconewton cellular traction forces},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Plasmonic scattering imaging of single proteins and binding
kinetics. <em>NMETH</em>, <em>17</em>(10), 1010–1017. (<a
href="https://doi.org/10.1038/s41592-020-0947-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Measuring the binding kinetics of single proteins represents one of the most important and challenging tasks in protein analysis. Here we show that this is possible using a surface plasmon resonance (SPR) scattering technique. SPR is a popular label-free detection technology because of its extraordinary sensitivity, but it has never been used for imaging single proteins. We overcome this limitation by imaging scattering of surface plasmonic waves by proteins. This allows us to image single proteins, measure their sizes and identify them based on their specific binding to antibodies. We further show that it is possible to quantify protein binding kinetics by counting the binding of individual molecules, providing a digital method to measure binding kinetics and analyze heterogeneity of protein behavior. We anticipate that this imaging method will become an important tool for single protein analysis, especially for low volume samples, such as single cells. Plasmonic scattering microscopy (PSM) enables the imaging of single proteins on SPR instruments. The method enables measurement of protein size and binding kinetics and is fully compatible with simultaneous traditional SPR measurements.},
  archive      = {J_NMETH},
  author       = {Zhang, Pengfei and Ma, Guangzhong and Dong, Wei and Wan, Zijian and Wang, Shaopeng and Tao, Nongjian},
  doi          = {10.1038/s41592-020-0947-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1010-1017},
  shortjournal = {Nat. Meth.},
  title        = {Plasmonic scattering imaging of single proteins and binding kinetics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Detecting chromatin interactions between and along sister
chromatids with SisterC. <em>NMETH</em>, <em>17</em>(10), 1002–1009. (<a
href="https://doi.org/10.1038/s41592-020-0930-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosome segregation requires both compaction and disentanglement of sister chromatids. We describe SisterC, a chromosome conformation capture assay that distinguishes interactions between and along identical sister chromatids. SisterC employs 5-bromo-2′-deoxyuridine (BrdU) incorporation during S-phase to label newly replicated strands, followed by Hi-C and then the destruction of 5-bromodeoxyuridine-containing strands via Hoechst/ultraviolet treatment. After sequencing of the remaining intact strands, this allows assignment of Hi-C products as inter- and intra-sister interactions based on the strands that reads are mapped to. We performed SisterC on mitotic Saccharomyces cerevisiae cells. We find precise alignment of sister chromatids at centromeres. Along arms, sister chromatids are less precisely aligned, with inter-sister connections every ~35 kilobase (kb). Inter-sister interactions occur between cohesin binding sites that are often offset by 5 to 25 kb. Along sister chromatids, cohesin results in the formation of loops of up to 50 kb. SisterC allows study of the complex interplay between sister chromatid compaction and their segregation during mitosis. It remains impossible using conventional Hi-C to differentiate interactions between and along sister chromatids. SisterC relies on selective destruction of nascent DNA and in combination with Hi-C offers a means to study intra- and inter-sister interactions independently.},
  archive      = {J_NMETH},
  author       = {Oomen, Marlies E. and Hedger, Adam K. and Watts, Jonathan K. and Dekker, Job},
  doi          = {10.1038/s41592-020-0930-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1002-1009},
  shortjournal = {Nat. Meth.},
  title        = {Detecting chromatin interactions between and along sister chromatids with SisterC},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Massively parallel and time-resolved RNA sequencing in
single cells with scNT-seq. <em>NMETH</em>, <em>17</em>(10), 991–1001.
(<a href="https://doi.org/10.1038/s41592-020-0935-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing offers snapshots of whole transcriptomes but obscures the temporal RNA dynamics. Here we present single-cell metabolically labeled new RNA tagging sequencing (scNT-seq), a method for massively parallel analysis of newly transcribed and pre-existing mRNAs from the same cell. This droplet microfluidics-based method enables high-throughput chemical conversion on barcoded beads, efficiently marking newly transcribed mRNAs with T-to-C substitutions. Using scNT-seq, we jointly profiled new and old transcriptomes in ~55,000 single cells. These data revealed time-resolved transcription factor activities and cell-state trajectories at the single-cell level in response to neuronal activation. We further determined rates of RNA biogenesis and decay to uncover RNA regulatory strategies during stepwise conversion between pluripotent and rare totipotent two-cell embryo (2C)-like stem cell states. Finally, integrating scNT-seq with genetic perturbation identifies DNA methylcytosine dioxygenase as an epigenetic barrier into the 2C-like cell state. Time-resolved single-cell transcriptomic analysis thus opens new lines of inquiry regarding cell-type-specific RNA regulatory mechanisms. ScNT-seq leverages metabolic labeling of nascent RNA and droplet-based sequencing for parallel analysis of newly transcribed and pre-existing mRNAs, which enables time-resolved analysis of dynamic biological processes in single cells.},
  archive      = {J_NMETH},
  author       = {Qiu, Qi and Hu, Peng and Qiu, Xiaojie and Govek, Kiya W. and Cámara, Pablo G. and Wu, Hao},
  doi          = {10.1038/s41592-020-0935-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {991-1001},
  shortjournal = {Nat. Meth.},
  title        = {Massively parallel and time-resolved RNA sequencing in single cells with scNT-seq},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Structure-based validation can drastically underestimate
error rate in proteome-wide cross-linking mass spectrometry studies.
<em>NMETH</em>, <em>17</em>(10), 985–988. (<a
href="https://doi.org/10.1038/s41592-020-0959-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thorough quality assessment of novel interactions identified by proteome-wide cross-linking mass spectrometry (XL-MS) studies is critical. Almost all current XL-MS studies have validated cross-links against known three-dimensional structures of representative protein complexes. Here, we provide theoretical and experimental evidence demonstrating that this approach can drastically underestimate error rates for proteome-wide XL-MS datasets, and propose a comprehensive set of four data-quality metrics to address this issue. The current standard approach for estimating error in proteome-scale cross-linking mass spectrometry datasets has severe limitations. A proposed set of data-quality metrics provides a more accurate assessment of error rate.},
  archive      = {J_NMETH},
  author       = {Yugandhar, Kumar and Wang, Ting-Yi and Wierbowski, Shayne D. and Shayhidin, Elnur Elyar and Yu, Haiyuan},
  doi          = {10.1038/s41592-020-0959-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {985-988},
  shortjournal = {Nat. Meth.},
  title        = {Structure-based validation can drastically underestimate error rate in proteome-wide cross-linking mass spectrometry studies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). MassIVE.quant: A community resource of quantitative mass
spectrometry–based proteomics datasets. <em>NMETH</em>, <em>17</em>(10),
981–984. (<a href="https://doi.org/10.1038/s41592-020-0955-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MassIVE.quant is a repository infrastructure and data resource for reproducible quantitative mass spectrometry–based proteomics, which is compatible with all mass spectrometry data acquisition types and computational analysis tools. A branch structure enables MassIVE.quant to systematically store raw experimental data, metadata of the experimental design, scripts of the quantitative analysis workflow, intermediate input and output files, as well as alternative reanalyses of the same dataset. MassIVE.quant is a data repository and data resource for reproducible quantitative mass spectrometry–based proteomics.},
  archive      = {J_NMETH},
  author       = {Choi, Meena and Carver, Jeremy and Chiva, Cristina and Tzouros, Manuel and Huang, Ting and Tsai, Tsung-Heng and Pullman, Benjamin and Bernhardt, Oliver M. and Hüttenhain, Ruth and Teo, Guo Ci and Perez-Riverol, Yasset and Muntel, Jan and Müller, Maik and Goetze, Sandra and Pavlou, Maria and Verschueren, Erik and Wollscheid, Bernd and Nesvizhskii, Alexey I. and Reiter, Lukas and Dunkley, Tom and Sabidó, Eduard and Bandeira, Nuno and Vitek, Olga},
  doi          = {10.1038/s41592-020-0955-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {981-984},
  shortjournal = {Nat. Meth.},
  title        = {MassIVE.quant: A community resource of quantitative mass spectrometry–based proteomics datasets},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Advances and challenges for fluorescence nanothermometry.
<em>NMETH</em>, <em>17</em>(10), 967–980. (<a
href="https://doi.org/10.1038/s41592-020-0957-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescent nanothermometers can probe changes in local temperature in living cells and in vivo and reveal fundamental insights into biological properties. This field has attracted global efforts in developing both temperature-responsive materials and detection procedures to achieve sub-degree temperature resolution in biosystems. Recent generations of nanothermometers show superior performance to earlier ones and also offer multifunctionality, enabling state-of-the-art functional imaging with improved spatial, temporal and temperature resolutions for monitoring the metabolism of intracellular organelles and internal organs. Although progress in this field has been rapid, it has not been without controversy, as recent studies have shown possible biased sensing during fluorescence-based detection. Here, we introduce the design principles and advances in fluorescence nanothermometry, highlight application achievements, discuss scenarios that may lead to biased sensing, analyze the challenges ahead in terms of both fundamental issues and practical implementations, and point to new directions for improving this interdisciplinary field. This Review on nanothermometry introduces the many types of nanothermometers and their cellular and in vivo applications, as well as best practices for accurate measurements.},
  archive      = {J_NMETH},
  author       = {Zhou, Jiajia and del Rosal, Blanca and Jaque, Daniel and Uchiyama, Seiichi and Jin, Dayong},
  doi          = {10.1038/s41592-020-0957-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {967-980},
  shortjournal = {Nat. Meth.},
  title        = {Advances and challenges for fluorescence nanothermometry},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Plankton in a hamster wheel. <em>NMETH</em>,
<em>17</em>(10), 965–966. (<a
href="https://doi.org/10.1038/s41592-020-0939-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plankton regularly travel vast distances up and down in the ocean. A water-filled hamster wheel with glass windows now enables detailed microscopic lab observations of individual aquatic microorganisms during their vertical migrations.},
  archive      = {J_NMETH},
  author       = {Taute, Katja M.},
  doi          = {10.1038/s41592-020-0939-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {965-966},
  shortjournal = {Nat. Meth.},
  title        = {Plankton in a hamster wheel},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020t). Reality check for organoids in neuroscience.
<em>NMETH</em>, <em>17</em>(10), 961–964. (<a
href="https://doi.org/10.1038/s41592-020-0964-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To better study human neurodevelopment, researchers advance organoids as brain models.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0964-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {961-964},
  shortjournal = {Nat. Meth.},
  title        = {Reality check for organoids in neuroscience},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020n). GFP lights up amyloid fibrils. <em>NMETH</em>,
<em>17</em>(10), 959. (<a
href="https://doi.org/10.1038/s41592-020-00976-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescent proteins can bind amyloid fibrils formed from natural peptides and proteins.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-00976-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {959},
  shortjournal = {Nat. Meth.},
  title        = {GFP lights up amyloid fibrils},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020x). Modeling the e. Coli cell. <em>NMETH</em>, <em>17</em>(10),
958. (<a href="https://doi.org/10.1038/s41592-020-00974-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An integrative modeling framework enables cross-validation of existing E. coli datasets and prediction of uncharacterized parameters.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-00974-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {958},
  shortjournal = {Nat. Meth.},
  title        = {Modeling the e. coli cell},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Label-free t cell characterization. <em>NMETH</em>,
<em>17</em>(10), 957. (<a
href="https://doi.org/10.1038/s41592-020-00975-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A machine learning model uses autofluorescence of key metabolites to predict T cell activation state.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-00975-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {957},
  shortjournal = {Nat. Meth.},
  title        = {Label-free t cell characterization},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020x). Volumetric imaging with confocal light field microscopy.
<em>NMETH</em>, <em>17</em>(10), 956. (<a
href="https://doi.org/10.1038/s41592-020-00973-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conferring optical sectioning capabilities on light field microscopy improves signal-to-noise ratio while maintaining high imaging speed.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-00973-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {956},
  shortjournal = {Nat. Meth.},
  title        = {Volumetric imaging with confocal light field microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). Inducing phase separation using artificial disordered
proteins. <em>NMETH</em>, <em>17</em>(10), 955. (<a
href="https://doi.org/10.1038/s41592-020-00972-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered intrinsically disordered proteins can induce phase separation and be used to modulate cellular function.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-00972-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {955},
  shortjournal = {Nat. Meth.},
  title        = {Inducing phase separation using artificial disordered proteins},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). METLIN MS2 molecular standards database: A broad chemical
and biological resource. <em>NMETH</em>, <em>17</em>(10), 953–954. (<a
href="https://doi.org/10.1038/s41592-020-0942-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Xue, Jingchuan and Guijas, Carlos and Benton, H. Paul and Warth, Benedikt and Siuzdak, Gary},
  doi          = {10.1038/s41592-020-0942-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {953-954},
  shortjournal = {Nat. Meth.},
  title        = {METLIN MS2 molecular standards database: A broad chemical and biological resource},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Vote, for science! <em>NMETH</em>, <em>17</em>(10), 949. (<a
href="https://doi.org/10.1038/s41592-020-00982-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The US elections take place in a little over a month. How its citizens vote will have a major impact on the scientific community and on the integrity of the scientific enterprise in the United States.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-00982-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {949},
  shortjournal = {Nat. Meth.},
  title        = {Vote, for science!},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020p). Manu prakash. <em>NMETH</em>, <em>17</em>(10), 951. (<a
href="https://doi.org/10.1038/s41592-020-0968-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Frugally built technology to study the ocean’s microbes, and engineering for societal good.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0968-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {10},
  pages        = {951},
  shortjournal = {Nat. Meth.},
  title        = {Manu prakash},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Analysis of the human protein atlas
image classification competition. <em>NMETH</em>, <em>17</em>(9), 948.
(<a href="https://doi.org/10.1038/s41592-020-0937-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Ouyang, Wei and Winsnes, Casper F. and Hjelmare, Martin and Cesnik, Anthony J. and Åkesson, Lovisa and Xu, Hao and Sullivan, Devin P. and Dai, Shubin and Lan, Jun and Jinmo, Park and Galib, Shaikat M. and Henkel, Christof and Hwang, Kevin and Poplavskiy, Dmytro and Tunguz, Bojan and Wolfinger, Russel D. and Gu, Yinzheng and Li, Chuanpeng and Xie, Jinbin and Buslov, Dmitry and Fironov, Sergei and Kiselev, Alexander and Panchenko, Dmytro and Cao, Xuan and Wei, Runmin and Wu, Yuanhao and Zhu, Xun and Tseng, Kuan-Lun and Gao, Zhifeng and Ju, Cheng and Yi, Xiaohan and Zheng, Hongdong and Kappel, Constantin and Lundberg, Emma},
  doi          = {10.1038/s41592-020-0937-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {948},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Analysis of the human protein atlas image classification competition},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020r). Publisher correction: Change-makers bring on recombinant
antibodies. <em>NMETH</em>, <em>17</em>(9), 948. (<a
href="https://doi.org/10.1038/s41592-020-0944-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0944-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {948},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Change-makers bring on recombinant antibodies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Highly specific multiplexed RNA imaging
in tissues with split-FISH. <em>NMETH</em>, <em>17</em>(9), 947. (<a
href="https://doi.org/10.1038/s41592-020-0931-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Goh, Jolene Jie Lin and Chou, Nigel and Seow, Wan Yi and Ha, Norbert and Cheng, Chung Pui Paul and Chang, Yun-Ching and Zhao, Ziqing Winston and Chen, Kok Hao},
  doi          = {10.1038/s41592-020-0931-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {947},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Highly specific multiplexed RNA imaging in tissues with split-FISH},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Comparing phenotypic variation between
inbred and outbred mice. <em>NMETH</em>, <em>17</em>(9), 947. (<a
href="https://doi.org/10.1038/s41592-020-0932-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Tuttle, Alexander H. and Philip, Vivek M. and Chesler, Elissa J. and Mogil, Jeffrey S.},
  doi          = {10.1038/s41592-020-0932-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {947},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Comparing phenotypic variation between inbred and outbred mice},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Genetically encoded tags for direct synthesis of EM-visible
gold nanoparticles in cells. <em>NMETH</em>, <em>17</em>(9), 937–946.
(<a href="https://doi.org/10.1038/s41592-020-0911-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded tags for single-molecule imaging in electron microscopy (EM) are long-awaited. Here, we report an approach for directly synthesizing EM-visible gold nanoparticles (AuNPs) on cysteine-rich tags for single-molecule visualization in cells. We first uncovered an auto-nucleation suppression mechanism that allows specific synthesis of AuNPs on isolated tags. Next, we exploited this mechanism to develop approaches for single-molecule detection of proteins in prokaryotic cells and achieved an unprecedented labeling efficiency. We then expanded it to more complicated eukaryotic cells and successfully detected the proteins targeted to various organelles, including the membranes of endoplasmic reticulum (ER) and nuclear envelope, ER lumen, nuclear pores, spindle pole bodies and mitochondrial matrices. We further implemented cysteine-rich tag–antibody fusion proteins as new immuno-EM probes. Thus, our approaches should allow biologists to address a wide range of biological questions at the single-molecule level in cellular ultrastructural contexts. Genetically encoded cysteine-rich tags enable formation of gold nanoparticles in situ for single-molecule imaging of individual proteins in the context of cellular ultrastructure in bacterial, yeast and mammalian cells.},
  archive      = {J_NMETH},
  author       = {Jiang, Zhaodi and Jin, Xiumei and Li, Yuhua and Liu, Sitong and Liu, Xiao-Man and Wang, Ying-Ying and Zhao, Pei and Cai, Xinbin and Liu, Ying and Tang, Yaqi and Sun, Xiaobin and Liu, Yan and Hu, Yanyong and Li, Ming and Cai, Gaihong and Qi, Xiangbing and Chen, She and Du, Li-Lin and He, Wanzhong},
  doi          = {10.1038/s41592-020-0911-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {937-946},
  shortjournal = {Nat. Meth.},
  title        = {Genetically encoded tags for direct synthesis of EM-visible gold nanoparticles in cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Rational design and implementation of a chemically inducible
heterotrimerization system. <em>NMETH</em>, <em>17</em>(9), 928–936. (<a
href="https://doi.org/10.1038/s41592-020-0913-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemically inducible dimerization (CID) uses a small molecule to induce binding of two different proteins. CID tools such as the FK506-binding protein–FKBP–rapamycin-binding– (FKBP–FRB)–rapamycin system have been widely used to probe molecular events inside and outside cells. While various CID tools are available, chemically inducible trimerization (CIT) does not exist, due to inherent challenges in designing a chemical that simultaneously binds three proteins with high affinity and specificity. Here, we developed CIT by rationally splitting FRB and FKBP. Cellular and structural datasets showed efficient trimerization of split pairs of FRB or FKBP with full-length FKBP or FRB, respectively, by rapamycin. CIT rapidly induced tri-organellar junctions and perturbed intended membrane lipids exclusively at select membrane contact sites. By conferring one additional condition to what is achievable with CID, CIT expands the types of manipulation in single live cells to address cell biology questions otherwise intractable and engineer cell functions for future synthetic biology applications. Chemically inducible trimerization tools based on split FRB or FKBP with full-length FKBP or FRB, respectively, expand the chemogenetics toolbox. Their efficiency and fast kinetics enable new types of protein manipulation in live cells.},
  archive      = {J_NMETH},
  author       = {Wu, Helen D. and Kikuchi, Masaki and Dagliyan, Onur and Aragaki, Adam K. and Nakamura, Hideki and Dokholyan, Nikolay V. and Umehara, Takashi and Inoue, Takanari},
  doi          = {10.1038/s41592-020-0913-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {928-936},
  shortjournal = {Nat. Meth.},
  title        = {Rational design and implementation of a chemically inducible heterotrimerization system},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Improvement of cryo-EM maps by density modification.
<em>NMETH</em>, <em>17</em>(9), 923–927. (<a
href="https://doi.org/10.1038/s41592-020-0914-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A density-modification procedure for improving maps from single-particle electron cryogenic microscopy (cryo-EM) is presented. The theoretical basis of the method is identical to that of maximum-likelihood density modification, previously used to improve maps from macromolecular X-ray crystallography. Key differences from applications in crystallography are that the errors in Fourier coefficients are largely in the phases in crystallography but in both phases and amplitudes in cryo-EM, and that half-maps with independent errors are available in cryo-EM. These differences lead to a distinct approach for combination of information from starting maps with information obtained in the density-modification process. The density-modification procedure was applied to a set of 104 datasets and improved map-model correlation and increased the visibility of details in many of the maps. The procedure requires two unmasked half-maps and a sequence file or other source of information on the volume of the macromolecule that has been imaged. A method based on maximum likelihood density modification, adapted from X-ray crystallography, improves cryo-EM maps.},
  archive      = {J_NMETH},
  author       = {Terwilliger, Thomas C. and Ludtke, Steven J. and Read, Randy J. and Adams, Paul D. and Afonine, Pavel V.},
  doi          = {10.1038/s41592-020-0914-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {923-927},
  shortjournal = {Nat. Meth.},
  title        = {Improvement of cryo-EM maps by density modification},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Actin chromobody imaging reveals sub-organellar actin
dynamics. <em>NMETH</em>, <em>17</em>(9), 917–921. (<a
href="https://doi.org/10.1038/s41592-020-0926-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The actin cytoskeleton plays multiple critical roles in cells, from cell migration to organelle dynamics. The small and transient actin structures regulating organelle dynamics are challenging to detect with fluorescence microscopy, making it difficult to determine whether actin filaments are directly associated with specific membranes. To address these limitations, we developed fluorescent-protein-tagged actin nanobodies, termed ‘actin chromobodies’ (ACs), targeted to organelle membranes to enable high-resolution imaging of sub-organellar actin dynamics. Genetically encoded fluorescent-protein-tagged actin nanobodies fused to organelle membrane-targeting sequences enables high-resolution imaging of transient organelle–actin contact sites in live cells.},
  archive      = {J_NMETH},
  author       = {Schiavon, Cara R. and Zhang, Tong and Zhao, Bing and Moore, Andrew S. and Wales, Pauline and Andrade, Leonardo R. and Wu, Melissa and Sung, Tsung-Chang and Dayn, Yelena and Feng, Jasmine W. and Quintero, Omar A. and Shadel, Gerald S. and Grosse, Robert and Manor, Uri},
  doi          = {10.1038/s41592-020-0926-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {917-921},
  shortjournal = {Nat. Meth.},
  title        = {Actin chromobody imaging reveals sub-organellar actin dynamics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). High-sensitivity and high-specificity biomechanical imaging
by stimulated brillouin scattering microscopy. <em>NMETH</em>,
<em>17</em>(9), 913–916. (<a
href="https://doi.org/10.1038/s41592-020-0882-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Label-free, non-contact imaging with mechanical contrast and optical sectioning is a substantial challenge in microscopy. Spontaneous Brillouin scattering microscopy meets this challenge, but encounters a trade-off between acquisition speed and the specificity for biomechanical constituents with overlapping Brillouin bands. Stimulated Brillouin scattering microscopy overcomes this trade-off and enables the cross-sectional imaging of live Caenorhabditis elegans at the organ and subcellular levels, with both elasticity and viscosity contrasts at high specificity and with practical recording times. Stimulated Brillouin scattering microscopy overcomes the trade-off between acquisition speed and spectral resolution in spontaneous Brillouin scattering microscopy and allows visualization of elasticity and viscosity, as shown in C. elegans.},
  archive      = {J_NMETH},
  author       = {Remer, Itay and Shaashoua, Roni and Shemesh, Netta and Ben-Zvi, Anat and Bilenca, Alberto},
  doi          = {10.1038/s41592-020-0882-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {913-916},
  shortjournal = {Nat. Meth.},
  title        = {High-sensitivity and high-specificity biomechanical imaging by stimulated brillouin scattering microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Optimizing imaging speed and excitation intensity for
single-molecule localization microscopy. <em>NMETH</em>, <em>17</em>(9),
909–912. (<a href="https://doi.org/10.1038/s41592-020-0918-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High laser powers are common practice in single-molecule localization microscopy to speed up data acquisition. Here we systematically quantified how excitation intensity influences localization precision and labeling density, the two main factors determining data quality. We found a strong trade-off between imaging speed and quality and present optimized imaging protocols for high-throughput, multicolor and three-dimensional single-molecule localization microscopy with greatly improved resolution and effective labeling efficiency. A systematic evaluation shows that excitation intensity has a dramatic impact on image quality in localization microscopy and reveals the benefits of lower excitation intensity for improved labeling efficiency and localization precision.},
  archive      = {J_NMETH},
  author       = {Diekmann, Robin and Kahnwald, Maurice and Schoenit, Andreas and Deschamps, Joran and Matti, Ulf and Ries, Jonas},
  doi          = {10.1038/s41592-020-0918-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {909-912},
  shortjournal = {Nat. Meth.},
  title        = {Optimizing imaging speed and excitation intensity for single-molecule localization microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Feature-based molecular networking in the GNPS analysis
environment. <em>NMETH</em>, <em>17</em>(9), 905–908. (<a
href="https://doi.org/10.1038/s41592-020-0933-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular networking has become a key method to visualize and annotate the chemical space in non-targeted mass spectrometry data. We present feature-based molecular networking (FBMN) as an analysis method in the Global Natural Products Social Molecular Networking (GNPS) infrastructure that builds on chromatographic feature detection and alignment tools. FBMN enables quantitative analysis and resolution of isomers, including from ion mobility spectrometry. Feature-based molecular networking allows the generation of molecular networks for mass spectrometry data that can recognize isomers, incorporate relative quantification and integrate ion mobility data.},
  archive      = {J_NMETH},
  author       = {Nothias, Louis-Félix and Petras, Daniel and Schmid, Robin and Dührkop, Kai and Rainer, Johannes and Sarvepalli, Abinesh and Protsyuk, Ivan and Ernst, Madeleine and Tsugawa, Hiroshi and Fleischauer, Markus and Aicheler, Fabian and Aksenov, Alexander A. and Alka, Oliver and Allard, Pierre-Marie and Barsch, Aiko and Cachet, Xavier and Caraballo-Rodriguez, Andres Mauricio and Da Silva, Ricardo R. and Dang, Tam and Garg, Neha and Gauglitz, Julia M. and Gurevich, Alexey and Isaac, Giorgis and Jarmusch, Alan K. and Kameník, Zdeněk and Kang, Kyo Bin and Kessler, Nikolas and Koester, Irina and Korf, Ansgar and Le Gouellec, Audrey and Ludwig, Marcus and Martin H., Christian and McCall, Laura-Isobel and McSayles, Jonathan and Meyer, Sven W. and Mohimani, Hosein and Morsy, Mustafa and Moyne, Oriane and Neumann, Steffen and Neuweger, Heiko and Nguyen, Ngoc Hung and Nothias-Esposito, Melissa and Paolini, Julien and Phelan, Vanessa V. and Pluskal, Tomáš and Quinn, Robert A. and Rogers, Simon and Shrestha, Bindesh and Tripathi, Anupriya and van der Hooft, Justin J. J. and Vargas, Fernando and Weldon, Kelly C. and Witting, Michael and Yang, Heejung and Zhang, Zheng and Zubeil, Florian and Kohlbacher, Oliver and Böcker, Sebastian and Alexandrov, Theodore and Bandeira, Nuno and Wang, Mingxun and Dorrestein, Pieter C.},
  doi          = {10.1038/s41592-020-0933-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {905-908},
  shortjournal = {Nat. Meth.},
  title        = {Feature-based molecular networking in the GNPS analysis environment},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). ReDU: A framework to find and reanalyze public mass
spectrometry data. <em>NMETH</em>, <em>17</em>(9), 901–904. (<a
href="https://doi.org/10.1038/s41592-020-0916-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present ReDU ( https://redu.ucsd.edu/ ), a system for metadata capture of public mass spectrometry-based metabolomics data, with validated controlled vocabularies. Systematic capture of knowledge enables the reanalysis of public data and/or co-analysis of one’s own data. ReDU enables multiple types of analyses, including finding chemicals and associated metadata, comparing the shared and different chemicals between groups of samples, and metadata-filtered, repository-scale molecular networking. Repository-scale reanalysis of public mass spectrometry-based metabolomics data is facilitated by the Reanalysis of Data User (ReDU) interface, a system that uses consistent formatting and controlled vocabularies for metadata capture.},
  archive      = {J_NMETH},
  author       = {Jarmusch, Alan K. and Wang, Mingxun and Aceves, Christine M. and Advani, Rohit S. and Aguirre, Shaden and Aksenov, Alexander A. and Aleti, Gajender and Aron, Allegra T. and Bauermeister, Anelize and Bolleddu, Sanjana and Bouslimani, Amina and Caraballo Rodriguez, Andres Mauricio and Chaar, Rama and Coras, Roxana and Elijah, Emmanuel O. and Ernst, Madeleine and Gauglitz, Julia M. and Gentry, Emily C. and Husband, Makhai and Jarmusch, Scott A. and Jones, Kenneth L. and Kamenik, Zdenek and Le Gouellec, Audrey and Lu, Aileen and McCall, Laura-Isobel and McPhail, Kerry L. and Meehan, Michael J. and Melnik, Alexey V. and Menezes, Riya C. and Montoya Giraldo, Yessica Alejandra and Nguyen, Ngoc Hung and Nothias, Louis Felix and Nothias-Esposito, Mélissa and Panitchpakdi, Morgan and Petras, Daniel and Quinn, Robert A. and Sikora, Nicole and van der Hooft, Justin J. J. and Vargas, Fernando and Vrbanac, Alison and Weldon, Kelly C. and Knight, Rob and Bandeira, Nuno and Dorrestein, Pieter C.},
  doi          = {10.1038/s41592-020-0916-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {901-904},
  shortjournal = {Nat. Meth.},
  title        = {ReDU: A framework to find and reanalyze public mass spectrometry data},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Time-resolved cryo-EM using spotiton. <em>NMETH</em>,
<em>17</em>(9), 897–900. (<a
href="https://doi.org/10.1038/s41592-020-0925-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an approach for preparing cryo-electron microscopy (cryo-EM) grids to study short-lived molecular states. Using piezoelectric dispensing, two independent streams of ~50-pl droplets of sample are deposited within 10 ms of each other onto the surface of a nanowire EM grid, and the mixing reaction stops when the grid is vitrified in liquid ethane ~100 ms later. We demonstrate this approach for four biological systems where short-lived states are of high interest. A spraying-mixing approach for preparing cryo-EM grids using the Spotiton robot allows time-resolved observations of short-lived biomolecular states.},
  archive      = {J_NMETH},
  author       = {Dandey, Venkata P. and Budell, William C. and Wei, Hui and Bobe, Daija and Maruthi, Kashyap and Kopylov, Mykhailo and Eng, Edward T. and Kahn, Peter A. and Hinshaw, Jenny E. and Kundu, Nidhi and Nimigean, Crina M. and Fan, Chen and Sukomon, Nattakan and Darst, Seth A. and Saecker, Ruth M. and Chen, James and Malone, Brandon and Potter, Clinton S. and Carragher, Bridget},
  doi          = {10.1038/s41592-020-0925-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {897-900},
  shortjournal = {Nat. Meth.},
  title        = {Time-resolved cryo-EM using spotiton},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). On the cutting edge: Protease-based methods for sensing and
controlling cell biology. <em>NMETH</em>, <em>17</em>(9), 885–896. (<a
href="https://doi.org/10.1038/s41592-020-0891-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequence-specific proteases have proven to be versatile building blocks for tools that report or control cellular function. Reporting methods link protease activity to biochemical signals, whereas control methods rely on engineering proteases to respond to exogenous inputs such as light or chemicals. In turn, proteases have inherent control abilities, as their native functions are to release, activate or destroy proteins by cleavage, with the irreversibility of proteolysis allowing sustained downstream effects. As a result, protease-based synthetic circuits have been created for diverse uses such as reporting cellular signaling, tuning protein expression, controlling viral replication and detecting cancer states. Here, we comprehensively review the development and application of protease-based methods for reporting and controlling cellular function in eukaryotes. This Review focuses on protease-based tools for sensing and controlling protein function in cell biology.},
  archive      = {J_NMETH},
  author       = {Chung, H. Kay and Lin, Michael Z.},
  doi          = {10.1038/s41592-020-0891-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {885-896},
  shortjournal = {Nat. Meth.},
  title        = {On the cutting edge: Protease-based methods for sensing and controlling cell biology},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Molecular motion on ice. <em>NMETH</em>, <em>17</em>(9),
879–883. (<a href="https://doi.org/10.1038/s41592-020-0940-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With advanced approaches to sample preparation and data analysis, scientists doing cryo-EM are starting to infer molecular movements from samples that are, literally, frozen.},
  archive      = {J_NMETH},
  author       = {Dance, Amber},
  doi          = {10.1038/s41592-020-0940-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {879-883},
  shortjournal = {Nat. Meth.},
  title        = {Molecular motion on ice},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). Extracellular protein degradation on demand.
<em>NMETH</em>, <em>17</em>(9), 877. (<a
href="https://doi.org/10.1038/s41592-020-0952-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A protein degradation platform enables the targeted depletion of extracellular and membrane-bound proteins.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0952-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {877},
  shortjournal = {Nat. Meth.},
  title        = {Extracellular protein degradation on demand},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ak). Standardizing population genetics simulations.
<em>NMETH</em>, <em>17</em>(9), 876. (<a
href="https://doi.org/10.1038/s41592-020-0951-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stdpopsim facilitates realistic population genetics simulations with a catalog of widely used models.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0951-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {876},
  shortjournal = {Nat. Meth.},
  title        = {Standardizing population genetics simulations},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Recreating the blood–CNS barrier in vitro. <em>NMETH</em>,
<em>17</em>(9), 875. (<a
href="https://doi.org/10.1038/s41592-020-0950-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stem-cell-derived organoids reproduce the in vivo secretory and functional properties of the choroid plexus.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-0950-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {875},
  shortjournal = {Nat. Meth.},
  title        = {Recreating the blood–CNS barrier in vitro},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A dual-constriction biological nanopore. <em>NMETH</em>,
<em>17</em>(9), 874. (<a
href="https://doi.org/10.1038/s41592-020-0949-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Addition of a second constriction to CsgG-family nanopores improves the accuracy of homopolymer sequencing.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0949-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {874},
  shortjournal = {Nat. Meth.},
  title        = {A dual-constriction biological nanopore},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Connectomics in high throughput. <em>NMETH</em>,
<em>17</em>(9), 873. (<a
href="https://doi.org/10.1038/s41592-020-0948-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {BRICseq provides access to brain-wide connectivity and can be combined with functional and transcriptomic studies for a comprehensive view of neuronal circuitry.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0948-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {873},
  shortjournal = {Nat. Meth.},
  title        = {Connectomics in high throughput},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). SMAP: A modular super-resolution microscopy analysis
platform for SMLM data. <em>NMETH</em>, <em>17</em>(9), 870–872. (<a
href="https://doi.org/10.1038/s41592-020-0938-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Ries, Jonas},
  doi          = {10.1038/s41592-020-0938-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {870-872},
  shortjournal = {Nat. Meth.},
  title        = {SMAP: A modular super-resolution microscopy analysis platform for SMLM data},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Philosopher: A versatile toolkit for shotgun proteomics data
analysis. <em>NMETH</em>, <em>17</em>(9), 869–870. (<a
href="https://doi.org/10.1038/s41592-020-0912-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {da Veiga Leprevost, Felipe and Haynes, Sarah E. and Avtonomov, Dmitry M. and Chang, Hui-Yin and Shanmugam, Avinash K. and Mellacheruvu, Dattatreya and Kong, Andy T. and Nesvizhskii, Alexey I.},
  doi          = {10.1038/s41592-020-0912-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {869-870},
  shortjournal = {Nat. Meth.},
  title        = {Philosopher: A versatile toolkit for shotgun proteomics data analysis},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Science on WeChat. <em>NMETH</em>, <em>17</em>(9), 863. (<a
href="https://doi.org/10.1038/s41592-020-0954-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We distributed an informal questionnaire to learn from scientists about their professional use of WeChat. We share some of their answers and discuss the multipurpose platform offered by WeChat.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0954-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {863},
  shortjournal = {Nat. Meth.},
  title        = {Science on WeChat},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Uncertainty and the management of epidemics. <em>NMETH</em>,
<em>17</em>(9), 867–868. (<a
href="https://doi.org/10.1038/s41592-020-0943-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {“I have no idea what’s awaiting me, or what will happen when this all ends. For the moment I know this: there are sick people and they need curing.” ―Albert Camus, The Plague},
  archive      = {J_NMETH},
  author       = {Shea, Katriona and Bjørnstad, Ottar N. and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-020-0943-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {9},
  pages        = {867-868},
  shortjournal = {Nat. Meth.},
  title        = {Uncertainty and the management of epidemics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020u). Uri manor. <em>NMETH</em>, <em>17</em>(9), 865. (<a
href="https://doi.org/10.1038/s41592-020-0945-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new way to see actin in action, hearing as others don’t, and saved by the guitar.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0945-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {9},
  pages        = {865},
  shortjournal = {Nat. Meth.},
  title        = {Uri manor},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: GPCRmd uncovers the dynamics of the
3D-GPCRome. <em>NMETH</em>, <em>17</em>(8), 861–862. (<a
href="https://doi.org/10.1038/s41592-020-0928-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Rodríguez-Espigares, Ismael and Torrens-Fontanals, Mariona and Tiemann, Johanna K. S. and Aranda-García, David and Ramírez-Anguita, Juan Manuel and Stepniewski, Tomasz Maciej and Worp, Nathalie and Varela-Rial, Alejandro and Morales-Pastor, Adrián and Medel-Lacruz, Brian and Pándy-Szekeres, Gáspár and Mayol, Eduardo and Giorgino, Toni and Carlsson, Jens and Deupi, Xavier and Filipek, Slawomir and Filizola, Marta and Gómez-Tamayo, José Carlos and Gonzalez, Angel and Gutiérrez-de-Terán, Hugo and Jiménez-Rosés, Mireia and Jespers, Willem and Kapla, Jon and Khelashvili, George and Kolb, Peter and Latek, Dorota and Marti-Solano, Maria and Matricon, Pierre and Matsoukas, Minos-Timotheos and Miszta, Przemyslaw and Olivella, Mireia and Perez-Benito, Laura and Provasi, Davide and Ríos, Santiago and R. Torrecillas, Iván and Sallander, Jessica and Sztyler, Agnieszka and Vasile, Silvana and Weinstein, Harel and Zachariae, Ulrich and Hildebrand, Peter W. and De Fabritiis, Gianni and Sanz, Ferran and Gloriam, David E. and Cordomi, Arnau and Guixà-González, Ramon and Selent, Jana},
  doi          = {10.1038/s41592-020-0928-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {861-862},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: GPCRmd uncovers the dynamics of the 3D-GPCRome},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Novel NanoLuc substrates enable bright two-population
bioluminescence imaging in animals. <em>NMETH</em>, <em>17</em>(8),
852–860. (<a href="https://doi.org/10.1038/s41592-020-0889-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sensitive detection of two biological events in vivo has long been a goal in bioluminescence imaging. Antares, a fusion of the luciferase NanoLuc to the orange fluorescent protein CyOFP, has emerged as a bright bioluminescent reporter with orthogonal substrate specificity to firefly luciferase (FLuc) and its derivatives such as AkaLuc. However, the brightness of Antares in mice is limited by the poor solubility and bioavailability of the NanoLuc substrate furimazine. Here, we report a new substrate, hydrofurimazine, whose enhanced aqueous solubility allows delivery of higher doses to mice. In the liver, Antares with hydrofurimazine exhibited similar brightness to AkaLuc with its substrate AkaLumine. Further chemical exploration generated a second substrate, fluorofurimazine, with even higher brightness in vivo. We used Antares with fluorofurimazine to track tumor size and AkaLuc with AkaLumine to visualize CAR-T cells within the same mice, demonstrating the ability to perform two-population imaging with these two luciferase systems. NanoLuc substrates with improved solubility and bioavailability, hydrofurimazine and fluorofurimazine, strongly enhance bioluminescence signals in vivo and enable bright dual-color bioluminescent imaging with AkaLuc and AkaLumine.},
  archive      = {J_NMETH},
  author       = {Su, Yichi and Walker, Joel R. and Park, Yunhee and Smith, Thomas P. and Liu, Lan Xiang and Hall, Mary P. and Labanieh, Louai and Hurst, Robin and Wang, David C. and Encell, Lance P. and Kim, Namdoo and Zhang, Feijie and Kay, Mark A. and Casey, Kerriann M. and Majzner, Robbie G. and Cochran, Jennifer R. and Mackall, Crystal L. and Kirkland, Thomas A. and Lin, Michael Z.},
  doi          = {10.1038/s41592-020-0889-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {852-860},
  shortjournal = {Nat. Meth.},
  title        = {Novel NanoLuc substrates enable bright two-population bioluminescence imaging in animals},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Mid-infrared metabolic imaging with vibrational probes.
<em>NMETH</em>, <em>17</em>(8), 844–851. (<a
href="https://doi.org/10.1038/s41592-020-0883-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding metabolism is indispensable in unraveling the mechanistic basis of many physiological and pathological processes. However, in situ metabolic imaging tools are still lacking. Here we introduce a framework for mid-infrared (MIR) metabolic imaging by coupling the emerging high-information-throughput MIR microscopy with specifically designed IR-active vibrational probes. We present three categories of small vibrational tags including azide bond, 13C-edited carbonyl bond and deuterium-labeled probes to interrogate various metabolic activities in cells, small organisms and mice. Two MIR imaging platforms are implemented including broadband Fourier transform infrared microscopy and discrete frequency infrared microscopy with a newly incorporated spectral region (2,000–2,300 cm−1). Our technique is uniquely suited to metabolic imaging with high information throughput. In particular, we performed single-cell metabolic profiling including heterogeneity characterization, and large-area metabolic imaging at tissue or organ level with rich spectral information. Small vibrational tags (azide, 13C-edited carbonyl and deuterium-labeled probes) were introduced as metabolic probes for mid-infrared imaging. The tags allow unprecedented in situ visualization of metabolism in cells and animals with high information throughput.},
  archive      = {J_NMETH},
  author       = {Shi, Lixue and Liu, Xinwen and Shi, Lingyan and Stinson, H. Ted and Rowlette, Jeremy and Kahl, Lisa J. and Evans, Christopher R. and Zheng, Chaogu and Dietrich, Lars E. P. and Min, Wei},
  doi          = {10.1038/s41592-020-0883-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {844-851},
  shortjournal = {Nat. Meth.},
  title        = {Mid-infrared metabolic imaging with vibrational probes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). ZipSeq: Barcoding for real-time mapping of single cell
transcriptomes. <em>NMETH</em>, <em>17</em>(8), 833–843. (<a
href="https://doi.org/10.1038/s41592-020-0880-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics seeks to integrate single cell transcriptomic data within the three-dimensional space of multicellular biology. Current methods to correlate a cell’s position with its transcriptome in living tissues have various limitations. We developed an approach, called ‘ZipSeq’, that uses patterned illumination and photocaged oligonucleotides to serially print barcodes (‘zipcodes’) onto live cells in intact tissues, in real time and with an on-the-fly selection of patterns. Using ZipSeq, we mapped gene expression in three settings: in vitro wound healing, live lymph node sections and a live tumor microenvironment. In all cases, we discovered new gene expression patterns associated with histological structures. In the tumor microenvironment, this demonstrated a trajectory of myeloid and T cell differentiation from the periphery inward. A combinatorial variation of ZipSeq efficiently scales in the number of regions defined, providing a pathway for complete mapping of live tissues, subsequent to real-time imaging or perturbation. ZipSeq uses patterned illumination and photocaged oligonucleotides to serially print barcodes onto live, intact tissues for spatially mapping single cell transcriptomes.},
  archive      = {J_NMETH},
  author       = {Hu, Kenneth H. and Eichorst, John P. and McGinnis, Chris S. and Patterson, David M. and Chow, Eric D. and Kersten, Kelly and Jameson, Stephen C. and Gartner, Zev J. and Rao, Arjun A. and Krummel, Matthew F.},
  doi          = {10.1038/s41592-020-0880-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {833-843},
  shortjournal = {Nat. Meth.},
  title        = {ZipSeq: Barcoding for real-time mapping of single cell transcriptomes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). 3D mapping and accelerated super-resolution imaging of the
human genome using in situ sequencing. <em>NMETH</em>, <em>17</em>(8),
822–832. (<a href="https://doi.org/10.1038/s41592-020-0890-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is a need for methods that can image chromosomes with genome-wide coverage, as well as greater genomic and optical resolution. We introduce OligoFISSEQ, a suite of three methods that leverage fluorescence in situ sequencing (FISSEQ) of barcoded Oligopaint probes to enable the rapid visualization of many targeted genomic regions. Applying OligoFISSEQ to human diploid fibroblast cells, we show how four rounds of sequencing are sufficient to produce 3D maps of 36 genomic targets across six chromosomes in hundreds to thousands of cells, implying a potential to image thousands of targets in only five to eight rounds of sequencing. We also use OligoFISSEQ to trace chromosomes at finer resolution, following the path of the X chromosome through 46 regions, with separate studies showing compatibility of OligoFISSEQ with immunocytochemistry. Finally, we combined OligoFISSEQ with OligoSTORM, laying the foundation for accelerated single-molecule super-resolution imaging of large swaths of, if not entire, human genomes. OligoFISSEQ combines Oligopaints with fluorescence in situ sequencing to enable the 3D mapping of many regions across the genome in human cells to interrogate genome organization at improved genomic resolution. OligoFISSEQ is compatible with immunochemistry and OligoSTORM for super-resolution imaging.},
  archive      = {J_NMETH},
  author       = {Nguyen, Huy Q. and Chattoraj, Shyamtanu and Castillo, David and Nguyen, Son C. and Nir, Guy and Lioutas, Antonios and Hershberg, Elliot A. and Martins, Nuno M. C. and Reginato, Paul L. and Hannan, Mohammed and Beliveau, Brian J. and Church, George M. and Daugharthy, Evan R. and Marti-Renom, Marc A. and Wu, C.-ting},
  doi          = {10.1038/s41592-020-0890-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {822-832},
  shortjournal = {Nat. Meth.},
  title        = {3D mapping and accelerated super-resolution imaging of the human genome using in situ sequencing},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A general method to optimize and functionalize red-shifted
rhodamine dyes. <em>NMETH</em>, <em>17</em>(8), 815–821. (<a
href="https://doi.org/10.1038/s41592-020-0909-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expanding the palette of fluorescent dyes is vital to push the frontier of biological imaging. Although rhodamine dyes remain the premier type of small-molecule fluorophore owing to their bioavailability and brightness, variants excited with far-red or near-infrared light suffer from poor performance due to their propensity to adopt a lipophilic, nonfluorescent form. We report a framework for rationalizing rhodamine behavior in biological environments and a general chemical modification for rhodamines that optimizes long-wavelength variants and enables facile functionalization with different chemical groups. This strategy yields red-shifted ‘Janelia Fluor’ (JF) dyes useful for biological imaging experiments in cells and in vivo. A general tuning strategy is introduced for improving the utility of rhodamines for biological imaging applications. The strategy yielded bright, versatile and bioavailable far-red and near-infrared ‘Janelia Fluor’ dyes.},
  archive      = {J_NMETH},
  author       = {Grimm, Jonathan B. and Tkachuk, Ariana N. and Xie, Liangqi and Choi, Heejun and Mohar, Boaz and Falco, Natalie and Schaefer, Kathy and Patel, Ronak and Zheng, Qinsi and Liu, Zhe and Lippincott-Schwartz, Jennifer and Brown, Timothy A. and Lavis, Luke D.},
  doi          = {10.1038/s41592-020-0909-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {815-821},
  shortjournal = {Nat. Meth.},
  title        = {A general method to optimize and functionalize red-shifted rhodamine dyes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Supervised enhancer prediction with epigenetic pattern
recognition and targeted validation. <em>NMETH</em>, <em>17</em>(8),
807–814. (<a href="https://doi.org/10.1038/s41592-020-0907-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enhancers are important non-coding elements, but they have traditionally been hard to characterize experimentally. The development of massively parallel assays allows the characterization of large numbers of enhancers for the first time. Here, we developed a framework using Drosophila STARR-seq to create shape-matching filters based on meta-profiles of epigenetic features. We integrated these features with supervised machine-learning algorithms to predict enhancers. We further demonstrated that our model could be transferred to predict enhancers in mammals. We comprehensively validated the predictions using a combination of in vivo and in vitro approaches, involving transgenic assays in mice and transduction-based reporter assays in human cell lines (153 enhancers in total). The results confirmed that our model can accurately predict enhancers in different species without re-parameterization. Finally, we examined the transcription factor binding patterns at predicted enhancers versus promoters. We demonstrated that these patterns enable the construction of a secondary model that effectively distinguishes enhancers and promoters. Supervised machine-learning models trained using Drosophila epigenetic and STARR-seq data can be transferred to predict mouse and human enhancers.},
  archive      = {J_NMETH},
  author       = {Sethi, Anurag and Gu, Mengting and Gumusgoz, Emrah and Chan, Landon and Yan, Koon-Kiu and Rozowsky, Joel and Barozzi, Iros and Afzal, Veena and Akiyama, Jennifer A. and Plajzer-Frick, Ingrid and Yan, Chengfei and Novak, Catherine S. and Kato, Momoe and Garvin, Tyler H. and Pham, Quan and Harrington, Anne and Mannion, Brandon J. and Lee, Elizabeth A. and Fukuda-Yuzawa, Yoko and Visel, Axel and Dickel, Diane E. and Yip, Kevin Y. and Sutton, Richard and Pennacchio, Len A. and Gerstein, Mark},
  doi          = {10.1038/s41592-020-0907-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {807-814},
  shortjournal = {Nat. Meth.},
  title        = {Supervised enhancer prediction with epigenetic pattern recognition and targeted validation},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Investigating higher-order interactions in single-cell data
with scHOT. <em>NMETH</em>, <em>17</em>(8), 799–806. (<a
href="https://doi.org/10.1038/s41592-020-0885-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell genomics has transformed our ability to examine cell fate choice. Examining cells along a computationally ordered ‘pseudotime’ offers the potential to unpick subtle changes in variability and covariation among key genes. We describe an approach, scHOT—single-cell higher-order testing—which provides a flexible and statistically robust framework for identifying changes in higher-order interactions among genes. scHOT can be applied for cells along a continuous trajectory or across space and accommodates various higher-order measurements including variability or correlation. We demonstrate the use of scHOT by studying coordinated changes in higher-order interactions during embryonic development of the mouse liver. Additionally, scHOT identifies subtle changes in gene–gene correlations across space using spatially resolved transcriptomics data from the mouse olfactory bulb. scHOT meaningfully adds to first-order differential expression testing and provides a framework for interrogating higher-order interactions using single-cell data. scHOT enables detection of changes in higher-order interactions in single-cell gene expression data.},
  archive      = {J_NMETH},
  author       = {Ghazanfar, Shila and Lin, Yingxin and Su, Xianbin and Lin, David Ming and Patrick, Ellis and Han, Ze-Guang and Marioni, John C. and Yang, Jean Yee Hwa},
  doi          = {10.1038/s41592-020-0885-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {799-806},
  shortjournal = {Nat. Meth.},
  title        = {Investigating higher-order interactions in single-cell data with scHOT},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Cumulus provides cloud-based data analysis for large-scale
single-cell and single-nucleus RNA-seq. <em>NMETH</em>, <em>17</em>(8),
793–798. (<a href="https://doi.org/10.1038/s41592-020-0905-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Massively parallel single-cell and single-nucleus RNA sequencing has opened the way to systematic tissue atlases in health and disease, but as the scale of data generation is growing, so is the need for computational pipelines for scaled analysis. Here we developed Cumulus—a cloud-based framework for analyzing large-scale single-cell and single-nucleus RNA sequencing datasets. Cumulus combines the power of cloud computing with improvements in algorithm and implementation to achieve high scalability, low cost, user-friendliness and integrated support for a comprehensive set of features. We benchmark Cumulus on the Human Cell Atlas Census of Immune Cells dataset of bone marrow cells and show that it substantially improves efficiency over conventional frameworks, while maintaining or improving the quality of results, enabling large-scale studies. Cumulus is a cloud-based framework enabling large-scale single-cell and single-nucleus RNA sequencing data analysis.},
  archive      = {J_NMETH},
  author       = {Li, Bo and Gould, Joshua and Yang, Yiming and Sarkizova, Siranush and Tabaka, Marcin and Ashenberg, Orr and Rosen, Yanay and Slyper, Michal and Kowalczyk, Monika S. and Villani, Alexandra-Chloé and Tickle, Timothy and Hacohen, Nir and Rozenblatt-Rosen, Orit and Regev, Aviv},
  doi          = {10.1038/s41592-020-0905-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {793-798},
  shortjournal = {Nat. Meth.},
  title        = {Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Up to 100-fold speed-up and multiplexing in optimized
DNA-PAINT. <em>NMETH</em>, <em>17</em>(8), 789–791. (<a
href="https://doi.org/10.1038/s41592-020-0869-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA-PAINT’s imaging speed has recently been significantly enhanced by optimized sequence design and buffer conditions. However, this implementation has not reached an ultimate speed limit and is only applicable to imaging of single targets. To further improve acquisition speed, we introduce concatenated, periodic DNA sequence motifs, yielding up to 100-fold-faster sampling in comparison to traditional DNA-PAINT. We extend this approach to six orthogonal sequence motifs, now enabling speed-optimized multiplexed imaging. Hundred-fold-faster DNA-PAINT imaging is enabled by the introduction of concatenated, periodic DNA sequence motifs in the docking strand. Six orthogonal sequences are described for speed-optimized and highly multiplexed cellular imaging.},
  archive      = {J_NMETH},
  author       = {Strauss, Sebastian and Jungmann, Ralf},
  doi          = {10.1038/s41592-020-0869-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {789-791},
  shortjournal = {Nat. Meth.},
  title        = {Up to 100-fold speed-up and multiplexing in optimized DNA-PAINT},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). GPCRmd uncovers the dynamics of the 3D-GPCRome.
<em>NMETH</em>, <em>17</em>(8), 777–787. (<a
href="https://doi.org/10.1038/s41592-020-0884-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {G-protein-coupled receptors (GPCRs) are involved in numerous physiological processes and are the most frequent targets of approved drugs. The explosion in the number of new three-dimensional (3D) molecular structures of GPCRs (3D-GPCRome) over the last decade has greatly advanced the mechanistic understanding and drug design opportunities for this protein family. Molecular dynamics (MD) simulations have become a widely established technique for exploring the conformational landscape of proteins at an atomic level. However, the analysis and visualization of MD simulations require efficient storage resources and specialized software. Here we present GPCRmd ( http://gpcrmd.org/ ), an online platform that incorporates web-based visualization capabilities as well as a comprehensive and user-friendly analysis toolbox that allows scientists from different disciplines to visualize, analyze and share GPCR MD data. GPCRmd originates from a community-driven effort to create an open, interactive and standardized database of GPCR MD simulations. GPCRmd is a community-driven online platform to visualize, analyze and share G-protein-coupled receptor (GPCR) molecular dynamics data. It currently contains simulation data representing 100\% of GPCR classes, 71\% of receptor subtypes and 80\% of GPCR families.},
  archive      = {J_NMETH},
  author       = {Rodríguez-Espigares, Ismael and Torrens-Fontanals, Mariona and Tiemann, Johanna K. S. and Aranda-García, David and Ramírez-Anguita, Juan Manuel and Stepniewski, Tomasz Maciej and Worp, Nathalie and Varela-Rial, Alejandro and Morales-Pastor, Adrián and Medel-Lacruz, Brian and Pándy-Szekeres, Gáspár and Mayol, Eduardo and Giorgino, Toni and Carlsson, Jens and Deupi, Xavier and Filipek, Slawomir and Filizola, Marta and Gómez-Tamayo, José Carlos and Gonzalez, Angel and Gutiérrez-de-Terán, Hugo and Jiménez-Rosés, Mireia and Jespers, Willem and Kapla, Jon and Khelashvili, George and Kolb, Peter and Latek, Dorota and Marti-Solano, Maria and Matricon, Pierre and Matsoukas, Minos-Timotheos and Miszta, Przemyslaw and Olivella, Mireia and Perez-Benito, Laura and Provasi, Davide and Ríos, Santiago and R. Torrecillas, Iván and Sallander, Jessica and Sztyler, Agnieszka and Vasile, Silvana and Weinstein, Harel and Zachariae, Ulrich and Hildebrand, Peter W. and De Fabritiis, Gianni and Sanz, Ferran and Gloriam, David E. and Cordomi, Arnau and Guixà-González, Ramon and Selent, Jana},
  doi          = {10.1038/s41592-020-0884-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {777-787},
  shortjournal = {Nat. Meth.},
  title        = {GPCRmd uncovers the dynamics of the 3D-GPCRome},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Mechanistic modeling of chromatin folding to understand
function. <em>NMETH</em>, <em>17</em>(8), 767–775. (<a
href="https://doi.org/10.1038/s41592-020-0852-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimental approaches have been applied to address questions in understanding three-dimensional chromatin organization and function. As datasets increase in size and complexity, it becomes a challenge to reach a mechanistic interpretation of experimental results. Polymer simulations and mechanistic modeling have been applied to explain experimental observations and their links to different aspects of genome function. Here we provide a guide for biologists, explaining different simulation approaches and the contexts in which they have been used. This Perspective highlights recently developed computational models for studying chromosome organization, with a focus on how mechanistic modeling helps biologists to interpret the biological function behind the genome structures.},
  archive      = {J_NMETH},
  author       = {Brackey, Chris A. and Marenduzzo, Davide and Gilbert, Nick},
  doi          = {10.1038/s41592-020-0852-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {767-775},
  shortjournal = {Nat. Meth.},
  title        = {Mechanistic modeling of chromatin folding to understand function},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Change-makers bring on recombinant antibodies.
<em>NMETH</em>, <em>17</em>(8), 763–766. (<a
href="https://doi.org/10.1038/s41592-020-0915-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There’s a growing affinity for recombinant antibodies. Some say it’s also high time for animal-free recombinants.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0915-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {763-766},
  shortjournal = {Nat. Meth.},
  title        = {Change-makers bring on recombinant antibodies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020v). Trapping fluorescence in the soma. <em>NMETH</em>,
<em>17</em>(8), 761. (<a
href="https://doi.org/10.1038/s41592-020-0923-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tethering fluorescent proteins to ribosomes clears up background signals when imaging neural circuits.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0923-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {761},
  shortjournal = {Nat. Meth.},
  title        = {Trapping fluorescence in the soma},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020v). Mapping RNA–RNA interactions. <em>NMETH</em>,
<em>17</em>(8), 760. (<a
href="https://doi.org/10.1038/s41592-020-0922-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RIC-seq enables in situ mapping of intra- and intermolecular RNA–RNA interactions.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0922-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {760},
  shortjournal = {Nat. Meth.},
  title        = {Mapping RNA–RNA interactions},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Multiplex fate mapping. <em>NMETH</em>, <em>17</em>(8),
759. (<a href="https://doi.org/10.1038/s41592-020-0921-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {barRNAseq allows detailed explorations of signaling pathways that control embryonic differentiation.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-0921-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {759},
  shortjournal = {Nat. Meth.},
  title        = {Multiplex fate mapping},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ad). Printing in vivo. <em>NMETH</em>, <em>17</em>(8), 758. (<a
href="https://doi.org/10.1038/s41592-020-0920-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intravital three-dimensional bioprinting enables the biofabrication of constructs in a pre-existing 3D matrix, such as inside tissues of live animals.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0920-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {758},
  shortjournal = {Nat. Meth.},
  title        = {Printing in vivo},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). Imaging single glycan molecules. <em>NMETH</em>,
<em>17</em>(8), 757. (<a
href="https://doi.org/10.1038/s41592-020-0919-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Direct imaging of carbohydrates using electrospray ion beam deposition and scanning tunneling microscopy.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0919-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {757},
  shortjournal = {Nat. Meth.},
  title        = {Imaging single glycan molecules},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Animal-derived-antibody generation faces strict reform in
accordance with european union policy on animal use. <em>NMETH</em>,
<em>17</em>(8), 755–756. (<a
href="https://doi.org/10.1038/s41592-020-0906-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Gray, Alison C. and Bradbury, Andrew R. M. and Knappik, Achim and Plückthun, Andreas and Borrebaeck, Carl A. K. and Dübel, Stefan},
  doi          = {10.1038/s41592-020-0906-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {755-756},
  shortjournal = {Nat. Meth.},
  title        = {Animal-derived-antibody generation faces strict reform in accordance with european union policy on animal use},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Content type revamp. <em>NMETH</em>, <em>17</em>(8), 751.
(<a href="https://doi.org/10.1038/s41592-020-0927-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We’ve made some recent updates to our various content type offerings at Nature Methods. Here is a cheat sheet.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0927-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {751},
  shortjournal = {Nat. Meth.},
  title        = {Content type revamp},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). Jana selent. <em>NMETH</em>, <em>17</em>(8), 753. (<a
href="https://doi.org/10.1038/s41592-020-0917-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A platform for exploring GPCR secrets, powered by community, music and sports.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0917-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {8},
  pages        = {753},
  shortjournal = {Nat. Meth.},
  title        = {Jana selent},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Kethoxal-assisted single-stranded DNA
sequencing captures global transcription dynamics and enhancer activity
in situ. <em>NMETH</em>, <em>17</em>(7), 749. (<a
href="https://doi.org/10.1038/s41592-020-0881-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Wu, Tong and Lyu, Ruitu and You, Qiancheng and He, Chuan},
  doi          = {10.1038/s41592-020-0881-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {749},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Kethoxal-assisted single-stranded DNA sequencing captures global transcription dynamics and enhancer activity in situ},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: DeepSTORM3D: Dense 3D localization
microscopy and PSF design by deep learning. <em>NMETH</em>,
<em>17</em>(7), 749. (<a
href="https://doi.org/10.1038/s41592-020-0910-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Nehme, Elias and Freedman, Daniel and Gordon, Racheli and Ferdman, Boris and Weiss, Lucien E. and Alalouf, Onit and Naor, Tal and Orange, Reut and Michaeli, Tomer and Shechtman, Yoav},
  doi          = {10.1038/s41592-020-0910-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {749},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: DeepSTORM3D: dense 3D localization microscopy and PSF design by deep learning},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Real-time 3D movement correction for two-photon imaging in
behaving animals. <em>NMETH</em>, <em>17</em>(7), 741–748. (<a
href="https://doi.org/10.1038/s41592-020-0851-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two-photon microscopy is widely used to investigate brain function across multiple spatial scales. However, measurements of neural activity are compromised by brain movement in behaving animals. Brain motion-induced artifacts are typically corrected using post hoc processing of two-dimensional images, but this approach is slow and does not correct for axial movements. Moreover, the deleterious effects of brain movement on high-speed imaging of small regions of interest and photostimulation cannot be corrected post hoc. To address this problem, we combined random-access three-dimensional (3D) laser scanning using an acousto-optic lens and rapid closed-loop field programmable gate array processing to track 3D brain movement and correct motion artifacts in real time at up to 1 kHz. Our recordings from synapses, dendrites and large neuronal populations in behaving mice and zebrafish demonstrate real-time movement-corrected 3D two-photon imaging with submicrometer precision. Real-time 3D movement correction by tracking a fluorescent bead in the field of view enables functional imaging with 3D two-photon random-access microscopy in behaving mice and zebrafish.},
  archive      = {J_NMETH},
  author       = {Griffiths, Victoria A. and Valera, Antoine M. and Lau, Joanna YN. and Roš, Hana and Younts, Thomas J. and Marin, Bóris and Baragli, Chiara and Coyle, Diccon and Evans, Geoffrey J. and Konstantinou, George and Koimtzis, Theo and Nadella, K. M. Naga Srinivas and Punde, Sameer A. and Kirkby, Paul A. and Bianco, Isaac H. and Silver, R. Angus},
  doi          = {10.1038/s41592-020-0851-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {741-748},
  shortjournal = {Nat. Meth.},
  title        = {Real-time 3D movement correction for two-photon imaging in behaving animals},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). DeepSTORM3D: Dense 3D localization microscopy and PSF
design by deep learning. <em>NMETH</em>, <em>17</em>(7), 734–740. (<a
href="https://doi.org/10.1038/s41592-020-0853-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An outstanding challenge in single-molecule localization microscopy is the accurate and precise localization of individual point emitters in three dimensions in densely labeled samples. One established approach for three-dimensional single-molecule localization is point-spread-function (PSF) engineering, in which the PSF is engineered to vary distinctively with emitter depth using additional optical elements. However, images of dense emitters, which are desirable for improving temporal resolution, pose a challenge for algorithmic localization of engineered PSFs, due to lateral overlap of the emitter PSFs. Here we train a neural network to localize multiple emitters with densely overlapping Tetrapod PSFs over a large axial range. We then use the network to design the optimal PSF for the multi-emitter case. We demonstrate our approach experimentally with super-resolution reconstructions of mitochondria and volumetric imaging of fluorescently labeled telomeres in cells. Our approach, DeepSTORM3D, enables the study of biological processes in whole cells at timescales that are rarely explored in localization microscopy. DeepSTORM3D uses deep learning for accurate localization of point emitters in densely labeled samples in three dimensions for volumetric localization microscopy with high temporal resolution, as well as for optimal point-spread function design.},
  archive      = {J_NMETH},
  author       = {Nehme, Elias and Freedman, Daniel and Gordon, Racheli and Ferdman, Boris and Weiss, Lucien E. and Alalouf, Onit and Naor, Tal and Orange, Reut and Michaeli, Tomer and Shechtman, Yoav},
  doi          = {10.1038/s41592-020-0853-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {734-740},
  shortjournal = {Nat. Meth.},
  title        = {DeepSTORM3D: Dense 3D localization microscopy and PSF design by deep learning},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Homogeneous multifocal excitation for high-throughput
super-resolution imaging. <em>NMETH</em>, <em>17</em>(7), 726–733. (<a
href="https://doi.org/10.1038/s41592-020-0859-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Super-resolution microscopies have become an established tool in biological research. However, imaging throughput remains a main bottleneck in acquiring large datasets required for quantitative biology. Here we describe multifocal flat illumination for field-independent imaging (mfFIFI). By integrating mfFIFI into an instant structured illumination microscope (iSIM), we extend the field of view (FOV) to &amp;gt;100 × 100 µm2 while maintaining high-speed, multicolor, volumetric imaging at double the diffraction-limited resolution. We further extend the effective FOV by stitching adjacent images for fast live-cell super-resolution imaging of dozens of cells. Finally, we combine our flat-fielded iSIM with ultrastructure expansion microscopy to collect three-dimensional (3D) images of hundreds of centrioles in human cells, or thousands of purified Chlamydomonas reinhardtii centrioles, per hour at an effective resolution of ~35 nm. Classification and particle averaging of these large datasets enables 3D mapping of posttranslational modifications of centriolar microtubules, revealing differences in their coverage and positioning. Multifocal flat illumination for field-independent imaging (mfFIFI) enables patterned illumination over an extended field of view. Integration with instant structured illumination microscope allowed for high-speed, multicolor, volumetric super-resolution imaging over 100 × 100 µm2.},
  archive      = {J_NMETH},
  author       = {Mahecic, Dora and Gambarotto, Davide and Douglass, Kyle M. and Fortun, Denis and Banterle, Niccoló and Ibrahim, Khalid A. and Le Guennec, Maeva and Gönczy, Pierre and Hamel, Virginie and Guichard, Paul and Manley, Suliana},
  doi          = {10.1038/s41592-020-0859-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {726-733},
  shortjournal = {Nat. Meth.},
  title        = {Homogeneous multifocal excitation for high-throughput super-resolution imaging},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Optogenetic control of gene expression in plants in the
presence of ambient white light. <em>NMETH</em>, <em>17</em>(7),
717–725. (<a href="https://doi.org/10.1038/s41592-020-0868-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optogenetics is the genetic approach for controlling cellular processes with light. It provides spatiotemporal, quantitative and reversible control over biological signaling and metabolic processes, overcoming limitations of chemically inducible systems. However, optogenetics lags in plant research because ambient light required for growth leads to undesired system activation. We solved this issue by developing plant usable light-switch elements (PULSE), an optogenetic tool for reversibly controlling gene expression in plants under ambient light. PULSE combines a blue-light-regulated repressor with a red-light-inducible switch. Gene expression is only activated under red light and remains inactive under white light or in darkness. Supported by a quantitative mathematical model, we characterized PULSE in protoplasts and achieved high induction rates, and we combined it with CRISPR–Cas9-based technologies to target synthetic signaling and developmental pathways. We applied PULSE to control immune responses in plant leaves and generated Arabidopsis transgenic plants. PULSE opens broad experimental avenues in plant research and biotechnology. PULSE is an optogenetic tool that consists of two modules with different wavelength sensitivities. Their interplay enables optogenetic access to gene expression in plants independently of ambient light.},
  archive      = {J_NMETH},
  author       = {Ochoa-Fernandez, Rocio and Abel, Nikolaj B. and Wieland, Franz-Georg and Schlegel, Jenia and Koch, Leonie-Alexa and Miller, J. Benjamin and Engesser, Raphael and Giuriani, Giovanni and Brandl, Simon M. and Timmer, Jens and Weber, Wilfried and Ott, Thomas and Simon, Rüdiger and Zurbriggen, Matias D.},
  doi          = {10.1038/s41592-020-0868-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {717-725},
  shortjournal = {Nat. Meth.},
  title        = {Optogenetic control of gene expression in plants in the presence of ambient white light},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Multilayered VBC score predicts sgRNAs that efficiently
generate loss-of-function alleles. <em>NMETH</em>, <em>17</em>(7),
708–716. (<a href="https://doi.org/10.1038/s41592-020-0850-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas9 screens have emerged as a transformative approach to systematically probe gene functions. The quality and success of these screens depends on the frequencies of loss-of-function alleles, particularly in negative-selection screens widely applied for probing essential genes. Using optimized screening workflows, we performed essentialome screens in cancer cell lines and embryonic stem cells and achieved dropout efficiencies that could not be explained by common frameshift frequencies. We find that these superior effect sizes are mainly determined by the impact of in-frame mutations on protein function, which can be predicted based on amino acid composition and conservation. We integrate protein features into a ‘Bioscore’ and fuse it with improved predictors of single-guide RNA activity and indel formation to establish a score that captures all relevant processes in CRISPR–Cas9 mutagenesis. This Vienna Bioactivity CRISPR score ( www.vbc-score.org ) outperforms previous prediction tools and enables the selection of sgRNAs that effectively produce loss-of-function alleles. Guide RNA selection plays an essential role in CRISPR screens, the Vienna Bioactivity CRISPR (VBC) scoring system provides an improved selection for sgRNAs that generate loss-of-function alleles.},
  archive      = {J_NMETH},
  author       = {Michlits, Georg and Jude, Julian and Hinterndorfer, Matthias and de Almeida, Melanie and Vainorius, Gintautas and Hubmann, Maria and Neumann, Tobias and Schleiffer, Alexander and Burkard, Thomas Rainer and Fellner, Michaela and Gijsbertsen, Max and Traunbauer, Anna and Zuber, Johannes and Elling, Ulrich},
  doi          = {10.1038/s41592-020-0850-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {708-716},
  shortjournal = {Nat. Meth.},
  title        = {Multilayered VBC score predicts sgRNAs that efficiently generate loss-of-function alleles},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Accelerated cryo-EM-guided determination of
three-dimensional RNA-only structures. <em>NMETH</em>, <em>17</em>(7),
699–707. (<a href="https://doi.org/10.1038/s41592-020-0878-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery and design of biologically important RNA molecules is outpacing three-dimensional structural characterization. Here, we demonstrate that cryo-electron microscopy can routinely resolve maps of RNA-only systems and that these maps enable subnanometer-resolution coordinate estimation when complemented with multidimensional chemical mapping and Rosetta DRRAFTER computational modeling. This hybrid ‘Ribosolve’ pipeline detects and falsifies homologies and conformational rearrangements in 11 previously unknown 119- to 338-nucleotide protein-free RNA structures: full-length Tetrahymena ribozyme, hc16 ligase with and without substrate, full-length Vibrio cholerae and Fusobacterium nucleatum glycine riboswitch aptamers with and without glycine, Mycobacterium SAM-IV riboswitch with and without S-adenosylmethionine, and the computer-designed ATP-TTR-3 aptamer with and without AMP. Simulation benchmarks, blind challenges, compensatory mutagenesis, cross-RNA homologies and internal controls demonstrate that Ribosolve can accurately resolve the global architectures of RNA molecules but does not resolve atomic details. These tests offer guidelines for making inferences in future RNA structural studies with similarly accelerated throughput. The Ribosolve pipeline combines single-particle cryo-EM, M2-seq biochemical analysis and Rosetta auto-DRRAFTER modeling to guide three-dimensional RNA structure determination.},
  archive      = {J_NMETH},
  author       = {Kappel, Kalli and Zhang, Kaiming and Su, Zhaoming and Watkins, Andrew M. and Kladwang, Wipapat and Li, Shanshan and Pintilie, Grigore and Topkar, Ved V. and Rangan, Ramya and Zheludev, Ivan N. and Yesselman, Joseph D. and Chiu, Wah and Das, Rhiju},
  doi          = {10.1038/s41592-020-0878-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {699-707},
  shortjournal = {Nat. Meth.},
  title        = {Accelerated cryo-EM-guided determination of three-dimensional RNA-only structures},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). JYCaMP: An optimized calcium indicator for two-photon
imaging at fiber laser wavelengths. <em>NMETH</em>, <em>17</em>(7),
694–697. (<a href="https://doi.org/10.1038/s41592-020-0835-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Femtosecond lasers at fixed wavelengths above 1,000 nm are powerful, stable and inexpensive, making them promising sources for two-photon microscopy. Biosensors optimized for these wavelengths are needed for both next-generation microscopes and affordable turn-key systems. Here we report jYCaMP1, a yellow variant of the calcium indicator jGCaMP7 that outperforms its parent in mice and flies at excitation wavelengths above 1,000 nm and enables improved two-color calcium imaging with red fluorescent protein-based indicators. jYCaMP1, a yellow variant of the calcium indicator jGCaMP7, enables fast multicolor two-photon imaging at excitation wavelengths above 1,000 nm for use with popular ytterbium-doped fiber and modelocked semiconductor lasers.},
  archive      = {J_NMETH},
  author       = {Mohr, Manuel Alexander and Bushey, Daniel and Aggarwal, Abhi and Marvin, Jonathan S. and Kim, Jeong Jun and Marquez, Emiliano Jimenez and Liang, Yajie and Patel, Ronak and Macklin, John J. and Lee, Chi-Yu and Tsang, Arthur and Tsegaye, Getahun and Ahrens, Allison M. and Chen, Jerry L. and Kim, Douglas S. and Wong, Allan M. and Looger, Loren L. and Schreiter, Eric R. and Podgorski, Kaspar},
  doi          = {10.1038/s41592-020-0835-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {694-697},
  shortjournal = {Nat. Meth.},
  title        = {JYCaMP: An optimized calcium indicator for two-photon imaging at fiber laser wavelengths},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Highly specific multiplexed RNA imaging in tissues with
split-FISH. <em>NMETH</em>, <em>17</em>(7), 689–693. (<a
href="https://doi.org/10.1038/s41592-020-0858-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present split-FISH, a multiplexed fluorescence in situ hybridization method that leverages a split-probe design to achieve enhanced specificity. Split-FISH reduces off-target background fluorescence, decreases false positives and enables accurate RNA profiling in uncleared tissues. We demonstrate the efficacy of split-FISH on various mouse tissues by quantifying the distribution and abundance of 317 genes in single cells and reveal diverse localization patterns for spatial regulation of the transcriptome in complex tissues. Split-FISH leverages a split-probe design to achieve enhanced specificity in multiplexed fluorescence in situ hybridization. Split-FISH reduces background and false positives, enabling transcriptome profiling in uncleared tissues.},
  archive      = {J_NMETH},
  author       = {Goh, Jolene Jie Lin and Chou, Nigel and Seow, Wan Yi and Ha, Norbert and Cheng, Chung Pui Paul and Chang, Yun-Ching and Zhao, Ziqing Winston and Chen, Kok Hao},
  doi          = {10.1038/s41592-020-0858-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {689-693},
  shortjournal = {Nat. Meth.},
  title        = {Highly specific multiplexed RNA imaging in tissues with split-FISH},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). CRISPR-assisted detection of RNA–protein interactions in
living cells. <em>NMETH</em>, <em>17</em>(7), 685–688. (<a
href="https://doi.org/10.1038/s41592-020-0866-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have developed CRISPR-assisted RNA–protein interaction detection method (CARPID), which leverages CRISPR–CasRx-based RNA targeting and proximity labeling to identify binding proteins of specific long non-coding RNAs (lncRNAs) in the native cellular context. We applied CARPID to the nuclear lncRNA XIST, and it captured a list of known interacting proteins and multiple previously uncharacterized binding proteins. We generalized CARPID to explore binders of the lncRNAs DANCR and MALAT1, revealing the method’s wide applicability in identifying RNA-binding proteins. CARPID uses CRISPR technology to navigate biotin ligase to specific lncRNAs, which allows proximal labeling and thus the querying of RNA–protein interactions in living cells.},
  archive      = {J_NMETH},
  author       = {Yi, Wenkai and Li, Jingyu and Zhu, Xiaoxuan and Wang, Xi and Fan, Ligang and Sun, Wenju and Liao, Linbu and Zhang, Jilin and Li, Xiaoyu and Ye, Jing and Chen, Fulin and Taipale, Jussi and Chan, Kui Ming and Zhang, Liang and Yan, Jian},
  doi          = {10.1038/s41592-020-0866-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {685-688},
  shortjournal = {Nat. Meth.},
  title        = {CRISPR-assisted detection of RNA–protein interactions in living cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Illumination guidelines for ultrafast pump–probe experiments
by serial femtosecond crystallography. <em>NMETH</em>, <em>17</em>(7),
681–684. (<a href="https://doi.org/10.1038/s41592-020-0847-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Time-resolved crystallography with X-ray free-electron lasers enables structural characterization of light-induced reactions on ultrafast timescales. To be biologically and chemically relevant, such studies must be carried out in an appropriate photoexcitation regime to avoid multiphoton artifacts, a common issue in recent studies. We describe numerical and experimental approaches to determine how many photons are needed for single-photon excitation in microcrystals, taking into account losses by scattering. Theoretical and experimental guidelines to determine laser intensity for single-photon excitation in microcrystals for ultrafast pump–probe experiments by serial femtosecond crystallography.},
  archive      = {J_NMETH},
  author       = {Grünbein, Marie Luise and Stricker, Miriam and Nass Kovacs, Gabriela and Kloos, Marco and Doak, R. Bruce and Shoeman, Robert L. and Reinstein, Jochen and Lecler, Sylvain and Haacke, Stefan and Schlichting, Ilme},
  doi          = {10.1038/s41592-020-0847-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {681-684},
  shortjournal = {Nat. Meth.},
  title        = {Illumination guidelines for ultrafast pump–probe experiments by serial femtosecond crystallography},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Macromolecular modeling and design in rosetta: Recent
methods and frameworks. <em>NMETH</em>, <em>17</em>(7), 665–680. (<a
href="https://doi.org/10.1038/s41592-020-0848-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Rosetta software for macromolecular modeling, docking and design is extensively used in laboratories worldwide. During two decades of development by a community of laboratories at more than 60 institutions, Rosetta has been continuously refactored and extended. Its advantages are its performance and interoperability between broad modeling capabilities. Here we review tools developed in the last 5 years, including over 80 methods. We discuss improvements to the score function, user interfaces and usability. Rosetta is available at http://www.rosettacommons.org . This Perspective reviews tools developed over the past five years in the macromolecular modeling, docking and design software Rosetta.},
  archive      = {J_NMETH},
  author       = {Leman, Julia Koehler and Weitzner, Brian D. and Lewis, Steven M. and Adolf-Bryfogle, Jared and Alam, Nawsad and Alford, Rebecca F. and Aprahamian, Melanie and Baker, David and Barlow, Kyle A. and Barth, Patrick and Basanta, Benjamin and Bender, Brian J. and Blacklock, Kristin and Bonet, Jaume and Boyken, Scott E. and Bradley, Phil and Bystroff, Chris and Conway, Patrick and Cooper, Seth and Correia, Bruno E. and Coventry, Brian and Das, Rhiju and De Jong, René M. and DiMaio, Frank and Dsilva, Lorna and Dunbrack, Roland and Ford, Alexander S. and Frenz, Brandon and Fu, Darwin Y. and Geniesse, Caleb and Goldschmidt, Lukasz and Gowthaman, Ragul and Gray, Jeffrey J. and Gront, Dominik and Guffy, Sharon and Horowitz, Scott and Huang, Po-Ssu and Huber, Thomas and Jacobs, Tim M. and Jeliazkov, Jeliazko R. and Johnson, David K. and Kappel, Kalli and Karanicolas, John and Khakzad, Hamed and Khar, Karen R. and Khare, Sagar D. and Khatib, Firas and Khramushin, Alisa and King, Indigo C. and Kleffner, Robert and Koepnick, Brian and Kortemme, Tanja and Kuenze, Georg and Kuhlman, Brian and Kuroda, Daisuke and Labonte, Jason W. and Lai, Jason K. and Lapidoth, Gideon and Leaver-Fay, Andrew and Lindert, Steffen and Linsky, Thomas and London, Nir and Lubin, Joseph H. and Lyskov, Sergey and Maguire, Jack and Malmström, Lars and Marcos, Enrique and Marcu, Orly and Marze, Nicholas A. and Meiler, Jens and Moretti, Rocco and Mulligan, Vikram Khipple and Nerli, Santrupti and Norn, Christoffer and Ó’Conchúir, Shane and Ollikainen, Noah and Ovchinnikov, Sergey and Pacella, Michael S. and Pan, Xingjie and Park, Hahnbeom and Pavlovicz, Ryan E. and Pethe, Manasi and Pierce, Brian G. and Pilla, Kala Bharath and Raveh, Barak and Renfrew, P. Douglas and Burman, Shourya S. Roy and Rubenstein, Aliza and Sauer, Marion F. and Scheck, Andreas and Schief, William and Schueler-Furman, Ora and Sedan, Yuval and Sevy, Alexander M. and Sgourakis, Nikolaos G. and Shi, Lei and Siegel, Justin B. and Silva, Daniel-Adriano and Smith, Shannon and Song, Yifan and Stein, Amelie and Szegedy, Maria and Teets, Frank D. and Thyme, Summer B. and Wang, Ray Yu-Ruei and Watkins, Andrew and Zimmerman, Lior and Bonneau, Richard},
  doi          = {10.1038/s41592-020-0848-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {665-680},
  shortjournal = {Nat. Meth.},
  title        = {Macromolecular modeling and design in rosetta: Recent methods and frameworks},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A new way to see RNAs. <em>NMETH</em>, <em>17</em>(7),
663–664. (<a href="https://doi.org/10.1038/s41592-020-0888-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Producing reliable atomic- or close-to-atomic-resolution structures of RNA-only molecules has been a formidable task. Ribosolve can solve sub-nanometer-resolution cryo-EM structures of unbound RNA molecules with unprecedented accuracy and speed.},
  archive      = {J_NMETH},
  author       = {Richardson, Jane S.},
  doi          = {10.1038/s41592-020-0888-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {663-664},
  shortjournal = {Nat. Meth.},
  title        = {A new way to see RNAs},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020v). When computational pipelines go “clank.” <em>NMETH</em>,
<em>17</em>(7), 659–662. (<a
href="https://doi.org/10.1038/s41592-020-0886-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {‘Omics pipeline builders and users face options and tough decisions.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0886-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {659-662},
  shortjournal = {Nat. Meth.},
  title        = {When computational pipelines go ‘clank’},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020q). Selective cell-surface n-glycan editing. <em>NMETH</em>,
<em>17</em>(7), 657. (<a
href="https://doi.org/10.1038/s41592-020-0896-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A two-step chemoenzymatic method for N-glycan subtype-selective editing.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0896-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {657},
  shortjournal = {Nat. Meth.},
  title        = {Selective cell-surface N-glycan editing},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). GemSpot allows modeling of ligands in cryo-EM maps.
<em>NMETH</em>, <em>17</em>(7), 656. (<a
href="https://doi.org/10.1038/s41592-020-0900-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0900-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {656},
  shortjournal = {Nat. Meth.},
  title        = {GemSpot allows modeling of ligands in cryo-EM maps},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020p). Improving single-cell RNA counting. <em>NMETH</em>,
<em>17</em>(7), 656. (<a
href="https://doi.org/10.1038/s41592-020-0901-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0901-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {656},
  shortjournal = {Nat. Meth.},
  title        = {Improving single-cell RNA counting},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Benchmarked spike sorting. <em>NMETH</em>, <em>17</em>(7),
656. (<a href="https://doi.org/10.1038/s41592-020-0902-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0902-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {656},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarked spike sorting},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A clearer view of mitophagy. <em>NMETH</em>,
<em>17</em>(7), 656. (<a
href="https://doi.org/10.1038/s41592-020-0903-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0903-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {656},
  shortjournal = {Nat. Meth.},
  title        = {A clearer view of mitophagy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Disease heritability explained by eQTLs. <em>NMETH</em>,
<em>17</em>(7), 655. (<a
href="https://doi.org/10.1038/s41592-020-0897-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0897-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {655},
  shortjournal = {Nat. Meth.},
  title        = {Disease heritability explained by eQTLs},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). Expanding worms. <em>NMETH</em>, <em>17</em>(7), 655. (<a
href="https://doi.org/10.1038/s41592-020-0898-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0898-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {655},
  shortjournal = {Nat. Meth.},
  title        = {Expanding worms},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020u). The mouse reference brain in 3D. <em>NMETH</em>,
<em>17</em>(7), 655. (<a
href="https://doi.org/10.1038/s41592-020-0899-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0899-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {655},
  shortjournal = {Nat. Meth.},
  title        = {The mouse reference brain in 3D},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Design of improved pET expression plasmids. <em>NMETH</em>,
<em>17</em>(7), 655. (<a
href="https://doi.org/10.1038/s41592-020-0904-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0904-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {655},
  shortjournal = {Nat. Meth.},
  title        = {Design of improved pET expression plasmids},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Contamination in sequence databases. <em>NMETH</em>,
<em>17</em>(7), 654. (<a
href="https://doi.org/10.1038/s41592-020-0895-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fast algorithm detects unexpected contamination events in public databases.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0895-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {654},
  shortjournal = {Nat. Meth.},
  title        = {Contamination in sequence databases},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020z). Tuning transparency in human cells. <em>NMETH</em>,
<em>17</em>(7), 653. (<a
href="https://doi.org/10.1038/s41592-020-0894-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteins that control the optical properties of cephalopod skin cells can modulate the transparency of mammalian cells.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0894-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {653},
  shortjournal = {Nat. Meth.},
  title        = {Tuning transparency in human cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ae). Radial genome organization. <em>NMETH</em>,
<em>17</em>(7), 652. (<a
href="https://doi.org/10.1038/s41592-020-0893-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing gradually digested chromatin along the nuclear radius enables the mapping of radial organization of chromatin in human cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0893-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {652},
  shortjournal = {Nat. Meth.},
  title        = {Radial genome organization},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Sugar-coating dendritic cells. <em>NMETH</em>,
<em>17</em>(7), 651. (<a
href="https://doi.org/10.1038/s41592-020-0892-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In situ labeling with azido sugars enables in vivo immunomodulation of dendritic cells.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-0892-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {651},
  shortjournal = {Nat. Meth.},
  title        = {Sugar-coating dendritic cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The GlyCosmos portal: A unified and comprehensive web
resource for the glycosciences. <em>NMETH</em>, <em>17</em>(7), 649–650.
(<a href="https://doi.org/10.1038/s41592-020-0879-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Yamada, Issaku and Shiota, Masaaki and Shinmachi, Daisuke and Ono, Tamiko and Tsuchiya, Shinichiro and Hosoda, Masae and Fujita, Akihiro and Aoki, Nobuyuki P. and Watanabe, Yu and Fujita, Noriaki and Angata, Kiyohiko and Kaji, Hiroyuki and Narimatsu, Hisashi and Okuda, Shujiro and Aoki-Kinoshita, Kiyoko F.},
  doi          = {10.1038/s41592-020-0879-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {649-650},
  shortjournal = {Nat. Meth.},
  title        = {The GlyCosmos portal: A unified and comprehensive web resource for the glycosciences},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Why black lives matter in science. <em>NMETH</em>,
<em>17</em>(7), 645. (<a
href="https://doi.org/10.1038/s41592-020-0908-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Systemic racism has held back under-represented minorities, especially Black people, in scientific research. We’re committed in our roles as journal editors to help correct this inequality.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0908-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {645},
  shortjournal = {Nat. Meth.},
  title        = {Why black lives matter in science},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020m). Kaspar podgorski. <em>NMETH</em>, <em>17</em>(7), 647. (<a
href="https://doi.org/10.1038/s41592-020-0887-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A yellow fluorescent sensor to study the brain, and the joy and pain of climbing.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0887-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {7},
  pages        = {647},
  shortjournal = {Nat. Meth.},
  title        = {Kaspar podgorski},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: Science in the time of coronavirus.
<em>NMETH</em>, <em>17</em>(6), 643. (<a
href="https://doi.org/10.1038/s41592-020-0849-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0849-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {643},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Science in the time of coronavirus},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Pooled CRISPR screens with imaging on microraft arrays
reveals stress granule-regulatory factors. <em>NMETH</em>,
<em>17</em>(6), 636–642. (<a
href="https://doi.org/10.1038/s41592-020-0826-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic screens using pooled CRISPR-based approaches are scalable and inexpensive, but restricted to standard readouts, including survival, proliferation and sortable markers. However, many biologically relevant cell states involve cellular and subcellular changes that are only accessible by microscopic visualization, and are currently impossible to screen with pooled methods. Here we combine pooled CRISPR–Cas9 screening with microraft array technology and high-content imaging to screen image-based phenotypes (CRaft-ID; CRISPR-based microRaft followed by guide RNA identification). By isolating microrafts that contain genetic clones harboring individual guide RNAs (gRNA), we identify RNA-binding proteins (RBPs) that influence the formation of stress granules, the punctate protein–RNA assemblies that form during stress. To automate hit identification, we developed a machine-learning model trained on nuclear morphology to remove unhealthy cells or imaging artifacts. In doing so, we identified and validated previously uncharacterized RBPs that modulate stress granule abundance, highlighting the applicability of our approach to facilitate image-based pooled CRISPR screens. CRISPR-based microraft followed by guide RNA identification (CRaft-ID) combines microraft arrays, microscopy and CRISPR–Cas9 technology for high-content image-based phenotyping. CRaft-ID was used to identify proteins involved in stress granule formation.},
  archive      = {J_NMETH},
  author       = {Wheeler, Emily C. and Vu, Anthony Q. and Einstein, Jaclyn M. and DiSalvo, Matthew and Ahmed, Noorsher and Van Nostrand, Eric L. and Shishkin, Alexander A. and Jin, Wenhao and Allbritton, Nancy L. and Yeo, Gene W.},
  doi          = {10.1038/s41592-020-0826-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {636-642},
  shortjournal = {Nat. Meth.},
  title        = {Pooled CRISPR screens with imaging on microraft arrays reveals stress granule-regulatory factors},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Targeted perturb-seq enables genome-scale genetic screens in
single cells. <em>NMETH</em>, <em>17</em>(6), 629–635. (<a
href="https://doi.org/10.1038/s41592-020-0837-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The transcriptome contains rich information on molecular, cellular and organismal phenotypes. However, experimental and statistical limitations constrain sensitivity and throughput of genetic screening with single-cell transcriptomics readout. To overcome these limitations, we introduce targeted Perturb-seq (TAP-seq), a sensitive, inexpensive and platform-independent method focusing single-cell RNA-seq coverage on genes of interest, thereby increasing the sensitivity and scale of genetic screens by orders of magnitude. TAP-seq permits routine analysis of thousands of CRISPR-mediated perturbations within a single experiment, detects weak effects and lowly expressed genes, and decreases sequencing requirements by up to 50-fold. We apply TAP-seq to generate perturbation-based enhancer–target gene maps for 1,778 enhancers within 2.5\% of the human genome. We thereby show that enhancer–target association is jointly determined by three-dimensional contact frequency and epigenetic states, allowing accurate prediction of enhancer targets throughout the genome. In addition, we demonstrate that TAP-seq can identify cell subtypes with only 100 sequencing reads per cell. Targeted sequencing of perturbation effects offers a sensitive approach to capture genes of interest in CRISPR-mediated screens, enabling genome-scale screens at higher scale and lower cost than whole-transcriptome Perturb-seq.},
  archive      = {J_NMETH},
  author       = {Schraivogel, Daniel and Gschwind, Andreas R. and Milbank, Jennifer H. and Leonce, Daniel R. and Jakob, Petra and Mathur, Lukas and Korbel, Jan O. and Merten, Christoph A. and Velten, Lars and Steinmetz, Lars M.},
  doi          = {10.1038/s41592-020-0837-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {629-635},
  shortjournal = {Nat. Meth.},
  title        = {Targeted perturb-seq enables genome-scale genetic screens in single cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Putative cell type discovery from single-cell gene
expression data. <em>NMETH</em>, <em>17</em>(6), 621–628. (<a
href="https://doi.org/10.1038/s41592-020-0825-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present the Single-Cell Clustering Assessment Framework, a method for the automated identification of putative cell types from single-cell RNA sequencing (scRNA-seq) data. By iteratively applying a machine learning approach to a given set of cells, we simultaneously identify distinct cell groups and a weighted list of feature genes for each group. The differentially expressed feature genes discriminate the given cell group from other cells. Each such group of cells corresponds to a putative cell type or state, characterized by the feature genes as markers. Benchmarking using expert-annotated scRNA-seq datasets shows that our method automatically identifies the ‘ground truth’ cell assignments with high accuracy. SCCAF automates the discovery of putative cell types and their feature genes using scRNA-seq data.},
  archive      = {J_NMETH},
  author       = {Miao, Zhichao and Moreno, Pablo and Huang, Ni and Papatheodorou, Irene and Brazma, Alvis and Teichmann, Sarah A.},
  doi          = {10.1038/s41592-020-0825-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {621-628},
  shortjournal = {Nat. Meth.},
  title        = {Putative cell type discovery from single-cell gene expression data},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Souporcell: Robust clustering of single-cell RNA-seq data by
genotype without reference genotypes. <em>NMETH</em>, <em>17</em>(6),
615–620. (<a href="https://doi.org/10.1038/s41592-020-0820-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to deconvolve single-cell RNA-sequencing (scRNA-seq) data are necessary for samples containing a mixture of genotypes, whether they are natural or experimentally combined. Multiplexing across donors is a popular experimental design that can avoid batch effects, reduce costs and improve doublet detection. By using variants detected in scRNA-seq reads, it is possible to assign cells to their donor of origin and identify cross-genotype doublets that may have highly similar transcriptional profiles, precluding detection by transcriptional profile. More subtle cross-genotype variant contamination can be used to estimate the amount of ambient RNA. Ambient RNA is caused by cell lysis before droplet partitioning and is an important confounder of scRNA-seq analysis. Here we develop souporcell, a method to cluster cells using the genetic variants detected within the scRNA-seq reads. We show that it achieves high accuracy on genotype clustering, doublet detection and ambient RNA estimation, as demonstrated across a range of challenging scenarios. Souporcell clusters single-cell RNA-seq data using genotype information without the use of a genotype reference.},
  archive      = {J_NMETH},
  author       = {Heaton, Haynes and Talman, Arthur M. and Knights, Andrew and Imaz, Maria and Gaffney, Daniel J. and Durbin, Richard and Hemberg, Martin and Lawniczak, Mara K. N.},
  doi          = {10.1038/s41592-020-0820-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {615-620},
  shortjournal = {Nat. Meth.},
  title        = {Souporcell: Robust clustering of single-cell RNA-seq data by genotype without reference genotypes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Elasticizing tissues for reversible shape transformation and
accelerated molecular labeling. <em>NMETH</em>, <em>17</em>(6), 609–613.
(<a href="https://doi.org/10.1038/s41592-020-0823-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed entangled link-augmented stretchable tissue-hydrogel (ELAST), a technology that transforms tissues into elastic hydrogels to enhance macromolecular accessibility and mechanical stability simultaneously. ELASTicized tissues are highly stretchable and compressible, which enables reversible shape transformation and faster delivery of probes into intact tissue specimens via mechanical thinning. This universal platform may facilitate rapid and scalable molecular phenotyping of large-scale biological systems, such as human organs. The ELAST technology renders large tissue samples elastic through embedding in hydrogel and facilitates sample handling and rapid molecular labeling.},
  archive      = {J_NMETH},
  author       = {Ku, Taeyun and Guan, Webster and Evans, Nicholas B. and Sohn, Chang Ho and Albanese, Alexandre and Kim, Joon-Goon and Frosch, Matthew P. and Chung, Kwanghun},
  doi          = {10.1038/s41592-020-0823-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {609-613},
  shortjournal = {Nat. Meth.},
  title        = {Elasticizing tissues for reversible shape transformation and accelerated molecular labeling},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deep three-photon imaging of the brain in intact adult
zebrafish. <em>NMETH</em>, <em>17</em>(6), 605–608. (<a
href="https://doi.org/10.1038/s41592-020-0819-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Behaviors emerge from activity throughout the brain, but noninvasive optical access in adult vertebrate brains is limited. We show that three-photon (3P) imaging through the head of intact adult zebrafish allows structural and functional imaging at cellular resolution throughout the telencephalon and deep into the cerebellum and optic tectum. With 3P imaging, considerable portions of the brain become noninvasively accessible from embryo to sexually mature adult in a vertebrate model. Three-photon microscopy provides access to most of the adult zebrafish brain for both structural and functional imaging, as well as to the Danionella dracula brain.},
  archive      = {J_NMETH},
  author       = {Chow, Dawnis M. and Sinefeld, David and Kolkman, Kristine E. and Ouzounov, Dimitre G. and Akbari, Najva and Tatarsky, Rose and Bass, Andrew and Xu, Chris and Fetcho, Joseph R.},
  doi          = {10.1038/s41592-020-0819-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {605-608},
  shortjournal = {Nat. Meth.},
  title        = {Deep three-photon imaging of the brain in intact adult zebrafish},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A rationally engineered cytosine base editor retains high
on-target activity while reducing both DNA and RNA off-target effects.
<em>NMETH</em>, <em>17</em>(6), 600–604. (<a
href="https://doi.org/10.1038/s41592-020-0832-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine base editors (CBEs) offer a powerful tool for correcting point mutations, yet their DNA and RNA off-target activities have caused concerns in biomedical applications. We describe screens of 23 rationally engineered CBE variants, which reveal mutation residues in the predicted DNA-binding site can dramatically decrease the Cas9-independent off-target effects. Furthermore, we obtained a CBE variant—YE1-BE3-FNLS—that retains high on-target editing efficiency while causing extremely low off-target edits and bystander edits. Structural and biochemical insights help engineer a cytosine base editor variant that possesses improved on-target activity with minimal DNA and RNA off-target editing.},
  archive      = {J_NMETH},
  author       = {Zuo, Erwei and Sun, Yidi and Yuan, Tanglong and He, Bingbing and Zhou, Changyang and Ying, Wenqin and Liu, Jing and Wei, Wu and Zeng, Rong and Li, Yixue and Yang, Hui},
  doi          = {10.1038/s41592-020-0832-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {600-604},
  shortjournal = {Nat. Meth.},
  title        = {A rationally engineered cytosine base editor retains high on-target activity while reducing both DNA and RNA off-target effects},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Intelligent image-based deformation-assisted cell sorting
with molecular specificity. <em>NMETH</em>, <em>17</em>(6), 595–599. (<a
href="https://doi.org/10.1038/s41592-020-0831-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although label-free cell sorting is desirable for providing pristine cells for further analysis or use, current approaches lack molecular specificity and speed. Here, we combine real-time fluorescence and deformability cytometry with sorting based on standing surface acoustic waves and transfer molecular specificity to image-based sorting using an efficient deep neural network. In addition to general performance, we demonstrate the utility of this method by sorting neutrophils from whole blood without labels. Sorting RT-FDC combines real-time fluorescence and deformability cytometry with sorting based on standing surface acoustic waves to transfer molecular specificity to label-free, image-based cell sorting using an efficient deep neural network.},
  archive      = {J_NMETH},
  author       = {Nawaz, Ahmad Ahsan and Urbanska, Marta and Herbig, Maik and Nötzel, Martin and Kräter, Martin and Rosendahl, Philipp and Herold, Christoph and Toepfner, Nicole and Kubánková, Markéta and Goswami, Ruchi and Abuhattum, Shada and Reichel, Felix and Müller, Paul and Taubenberger, Anna and Girardo, Salvatore and Jacobi, Angela and Guck, Jochen},
  doi          = {10.1038/s41592-020-0831-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {595-599},
  shortjournal = {Nat. Meth.},
  title        = {Intelligent image-based deformation-assisted cell sorting with molecular specificity},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A comparison of microfluidic methods for high-throughput
cell deformability measurements. <em>NMETH</em>, <em>17</em>(6),
587–593. (<a href="https://doi.org/10.1038/s41592-020-0818-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanical phenotype of a cell is an inherent biophysical marker of its state and function, with many applications in basic and applied biological research. Microfluidics-based methods have enabled single-cell mechanophenotyping at throughputs comparable to those of flow cytometry. Here, we present a standardized cross-laboratory study comparing three microfluidics-based approaches for measuring cell mechanical phenotype: constriction-based deformability cytometry (cDC), shear flow deformability cytometry (sDC) and extensional flow deformability cytometry (xDC). All three methods detect cell deformability changes induced by exposure to altered osmolarity. However, a dose-dependent deformability increase upon latrunculin B-induced actin disassembly was detected only with cDC and sDC, which suggests that when exposing cells to the higher strain rate imposed by xDC, cellular components other than the actin cytoskeleton dominate the response. The direct comparison presented here furthers our understanding of the applicability of the different deformability cytometry methods and provides context for the interpretation of deformability measurements performed using different platforms. This Analysis compares microfluidics-based methods for assessing mechanical properties of cells in high throughput.},
  archive      = {J_NMETH},
  author       = {Urbanska, Marta and Muñoz, Hector E. and Shaw Bagnall, Josephine and Otto, Oliver and Manalis, Scott R. and Di Carlo, Dino and Guck, Jochen},
  doi          = {10.1038/s41592-020-0818-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {587-593},
  shortjournal = {Nat. Meth.},
  title        = {A comparison of microfluidic methods for high-throughput cell deformability measurements},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Aggregating transcript-level analyses for single-cell
differential gene expression. <em>NMETH</em>, <em>17</em>(6), 583–585.
(<a href="https://doi.org/10.1038/s41592-020-0854-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Becht, Etienne and Zhao, Edward and Amezquita, Robert and Gottardo, Raphael},
  doi          = {10.1038/s41592-020-0854-4},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {583-585},
  shortjournal = {Nat. Meth.},
  title        = {Aggregating transcript-level analyses for single-cell differential gene expression},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Scanless two-photon excitation with temporal focusing.
<em>NMETH</em>, <em>17</em>(6), 571–581. (<a
href="https://doi.org/10.1038/s41592-020-0795-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Temporal focusing, with its ability to focus light in time, enables scanless illumination of large surface areas at the sample with micrometer axial confinement and robust propagation through scattering tissue. In conventional two-photon microscopy, widely used for the investigation of intact tissue in live animals, images are formed by point scanning of a spatially focused pulsed laser beam, resulting in limited temporal resolution of the excitation. Replacing point scanning with temporally focused widefield illumination removes this limitation and represents an important milestone in two-photon microscopy. Temporal focusing uses a diffusive or dispersive optical element placed in a plane conjugate to the objective focal plane to generate position-dependent temporal pulse broadening that enables axially confined multiphoton absorption, without the need for tight spatial focusing. Many techniques have benefitted from temporal focusing, including scanless imaging, super-resolution imaging, photolithography, uncaging of caged neurotransmitters and control of neuronal activity via optogenetics. This Review discusses temporal focusing microscopy and its applications in neuroscience for imaging and optogenetic activation.},
  archive      = {J_NMETH},
  author       = {Papagiakoumou, Eirini and Ronzitti, Emiliano and Emiliani, Valentina},
  doi          = {10.1038/s41592-020-0795-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {571-581},
  shortjournal = {Nat. Meth.},
  title        = {Scanless two-photon excitation with temporal focusing},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Cell biology befriends soft matter physics. <em>NMETH</em>,
<em>17</em>(6), 567–570. (<a
href="https://doi.org/10.1038/s41592-020-0855-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phase separation creates complex condensates in eukaryotic cells. To study these mysterious droplets, many disciplines come together.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0855-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {567-570},
  shortjournal = {Nat. Meth.},
  title        = {Cell biology befriends soft matter physics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Designing protein logic gates. <em>NMETH</em>,
<em>17</em>(6), 565. (<a
href="https://doi.org/10.1038/s41592-020-0865-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo–designed protein logic gates allow post-translational regulation of protein function.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0865-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {565},
  shortjournal = {Nat. Meth.},
  title        = {Designing protein logic gates},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Representation matters for t cell responses.
<em>NMETH</em>, <em>17</em>(6), 564. (<a
href="https://doi.org/10.1038/s41592-020-0874-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-0874-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {564},
  shortjournal = {Nat. Meth.},
  title        = {Representation matters for t cell responses},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020o). Optimized tissue clearing. <em>NMETH</em>, <em>17</em>(6),
564. (<a href="https://doi.org/10.1038/s41592-020-0875-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0875-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {564},
  shortjournal = {Nat. Meth.},
  title        = {Optimized tissue clearing},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020u). Mapping DNA single-strand breaks. <em>NMETH</em>,
<em>17</em>(6), 564. (<a
href="https://doi.org/10.1038/s41592-020-0876-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0876-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {564},
  shortjournal = {Nat. Meth.},
  title        = {Mapping DNA single-strand breaks},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020m). Fast analysis of single-molecule data. <em>NMETH</em>,
<em>17</em>(6), 564. (<a
href="https://doi.org/10.1038/s41592-020-0877-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0877-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {564},
  shortjournal = {Nat. Meth.},
  title        = {Fast analysis of single-molecule data},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Chromatin dynamics get mapped. <em>NMETH</em>,
<em>17</em>(6), 563. (<a
href="https://doi.org/10.1038/s41592-020-0870-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0870-4},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {563},
  shortjournal = {Nat. Meth.},
  title        = {Chromatin dynamics get mapped},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020m). Harmonizing cancer variant knowledgebases. <em>NMETH</em>,
<em>17</em>(6), 563. (<a
href="https://doi.org/10.1038/s41592-020-0871-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0871-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {563},
  shortjournal = {Nat. Meth.},
  title        = {Harmonizing cancer variant knowledgebases},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Identifying phosphorylation-site-specific kinases.
<em>NMETH</em>, <em>17</em>(6), 563. (<a
href="https://doi.org/10.1038/s41592-020-0872-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0872-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {563},
  shortjournal = {Nat. Meth.},
  title        = {Identifying phosphorylation-site-specific kinases},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A digital embryo. <em>NMETH</em>, <em>17</em>(6), 563. (<a
href="https://doi.org/10.1038/s41592-020-0873-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0873-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {563},
  shortjournal = {Nat. Meth.},
  title        = {A digital embryo},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020al). Taxonomy of bacteria and archaea. <em>NMETH</em>,
<em>17</em>(6), 562. (<a
href="https://doi.org/10.1038/s41592-020-0863-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Improvements to the Genome Taxonomy Database provide a complete bacterial and archaeal taxonomy.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0863-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {562},
  shortjournal = {Nat. Meth.},
  title        = {Taxonomy of bacteria and archaea},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Macrophages enter CAR immunotherapy. <em>NMETH</em>,
<em>17</em>(6), 561. (<a
href="https://doi.org/10.1038/s41592-020-0862-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CAR+ macrophages modulate the tumor milieu and lead to antitumor immune responses.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-0862-4},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {561},
  shortjournal = {Nat. Meth.},
  title        = {Macrophages enter CAR immunotherapy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020y). Watching embryos develop. <em>NMETH</em>, <em>17</em>(6),
560. (<a href="https://doi.org/10.1038/s41592-020-0864-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryonic development is typically hidden from view, but a window preparation technique now sheds light on this phase in the life of a mouse.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0864-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {560},
  shortjournal = {Nat. Meth.},
  title        = {Watching embryos develop},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020aa). PAM-less is more. <em>NMETH</em>, <em>17</em>(6), 559. (<a
href="https://doi.org/10.1038/s41592-020-0861-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structure-guided engineering allows researchers access to an expanding number of Cas9 variants that relax the constraint imposed by target site recognition of a protospacer-adjacent motif (PAM).},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0861-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {559},
  shortjournal = {Nat. Meth.},
  title        = {PAM-less is more},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). The SEIRS model for infectious disease dynamics.
<em>NMETH</em>, <em>17</em>(6), 557–558. (<a
href="https://doi.org/10.1038/s41592-020-0856-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Realistic models of epidemics account for latency, loss of immunity, births and deaths.},
  archive      = {J_NMETH},
  author       = {Bjørnstad, Ottar N. and Shea, Katriona and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-020-0856-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {557-558},
  shortjournal = {Nat. Meth.},
  title        = {The SEIRS model for infectious disease dynamics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Tweet, tweet. <em>NMETH</em>, <em>17</em>(6), 553. (<a
href="https://doi.org/10.1038/s41592-020-0860-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted an informal survey to learn how scientists are using Twitter. We share some of the responses and discuss why being active on Twitter offers real benefits.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0860-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {553},
  shortjournal = {Nat. Meth.},
  title        = {Tweet, tweet},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). Hui yang. <em>NMETH</em>, <em>17</em>(6), 555. (<a
href="https://doi.org/10.1038/s41592-020-0857-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A better base editor with fewer off-target changes, from a die-hard Manchester United fan.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0857-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {6},
  pages        = {555},
  shortjournal = {Nat. Meth.},
  title        = {Hui yang},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Genetic tool development in marine
protists: Emerging model organisms for experimental cell biology.
<em>NMETH</em>, <em>17</em>(5), 551. (<a
href="https://doi.org/10.1038/s41592-020-0828-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Faktorová, Drahomíra and Nisbet, R. Ellen R. and Fernández Robledo, José A. and Casacuberta, Elena and Sudek, Lisa and Allen, Andrew E. and Ares, Manuel and Aresté, Cristina and Balestreri, Cecilia and Barbrook, Adrian C. and Beardslee, Patrick and Bender, Sara and Booth, David S. and Bouget, François-Yves and Bowler, Chris and Breglia, Susana A. and Brownlee, Colin and Burger, Gertraud and Cerutti, Heriberto and Cesaroni, Rachele and Chiurillo, Miguel A. and Clemente, Thomas and Coles, Duncan B. and Collier, Jackie L. and Cooney, Elizabeth C. and Coyne, Kathryn and Docampo, Roberto and Dupont, Christopher L. and Edgcomb, Virginia and Einarsson, Elin and Elustondo, Pía A. and Federici, Fernan and Freire-Beneitez, Veronica and Freyria, Nastasia J. and Fukuda, Kodai and García, Paulo A. and Girguis, Peter R. and Gomaa, Fatma and Gornik, Sebastian G. and Guo, Jian and Hampl, Vladimír and Hanawa, Yutaka and Haro-Contreras, Esteban R. and Hehenberger, Elisabeth and Highfield, Andrea and Hirakawa, Yoshihisa and Hopes, Amanda and Howe, Christopher J. and Hu, Ian and Ibañez, Jorge and Irwin, Nicholas A. T. and Ishii, Yuu and Janowicz, Natalia Ewa and Jones, Adam C. and Kachale, Ambar and Fujimura-Kamada, Konomi and Kaur, Binnypreet and Kaye, Jonathan Z. and Kazana, Eleanna and Keeling, Patrick J. and King, Nicole and Klobutcher, Lawrence A. and Lander, Noelia and Lassadi, Imen and Li, Zhuhong and Lin, Senjie and Lozano, Jean-Claude and Luan, Fulei and Maruyama, Shinichiro and Matute, Tamara and Miceli, Cristina and Minagawa, Jun and Moosburner, Mark and Najle, Sebastián R. and Nanjappa, Deepak and Nimmo, Isabel C. and Noble, Luke and Novák Vanclová, Anna M. G. and Nowacki, Mariusz and Nuñez, Isaac and Pain, Arnab and Piersanti, Angela and Pucciarelli, Sandra and Pyrih, Jan and Rest, Joshua S. and Rius, Mariana and Robertson, Deborah and Ruaud, Albane and Ruiz-Trillo, Iñaki and Sigg, Monika A. and Silver, Pamela A. and Slamovits, Claudio H. and Jason Smith, G. and Sprecher, Brittany N. and Stern, Rowena and Swart, Estienne C. and Tsaousis, Anastasios D. and Tsypin, Lev and Turkewitz, Aaron and Turnšek, Jernej and Valach, Matus and Vergé, Valérie and von Dassow, Peter and von der Haar, Tobias and Waller, Ross F. and Wang, Lu and Wen, Xiaoxue and Wheeler, Glen and Woods, April and Zhang, Huan and Mock, Thomas and Worden, Alexandra Z. and Lukeš, Julius},
  doi          = {10.1038/s41592-020-0828-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {551},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: genetic tool development in marine protists: emerging model organisms for experimental cell biology},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Multiplexed cre-dependent selection yields systemic AAVs for
targeting distinct brain cell types. <em>NMETH</em>, <em>17</em>(5),
541–550. (<a href="https://doi.org/10.1038/s41592-020-0799-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recombinant adeno-associated viruses (rAAVs) are efficient gene delivery vectors via intravenous delivery; however, natural serotypes display a finite set of tropisms. To expand their utility, we evolved AAV capsids to efficiently transduce specific cell types in adult mouse brains. Building upon our Cre-recombination-based AAV targeted evolution (CREATE) platform, we developed Multiplexed-CREATE (M-CREATE) to identify variants of interest in a given selection landscape through multiple positive and negative selection criteria. M-CREATE incorporates next-generation sequencing, synthetic library generation and a dedicated analysis pipeline. We have identified capsid variants that can transduce the central nervous system broadly, exhibit bias toward vascular cells and astrocytes, target neurons with greater specificity or cross the blood–brain barrier across diverse murine strains. Collectively, the M-CREATE methodology accelerates the discovery of capsids for use in neuroscience and gene-therapy applications. M-CREATE is an in vivo screening strategy for identifying recombinant AAVs with desired tropism. The approach involves both positive and negative selection and yields vectors with diversified cell-type tropism that can cross the blood–brain barrier in adult mice across strains when delivered intravenously.},
  archive      = {J_NMETH},
  author       = {Ravindra Kumar, Sripriya and Miles, Timothy F. and Chen, Xinhong and Brown, David and Dobreva, Tatyana and Huang, Qin and Ding, Xiaozhe and Luo, Yicheng and Einarsson, Pétur H. and Greenbaum, Alon and Jang, Min J. and Deverman, Benjamin E. and Gradinaru, Viviana},
  doi          = {10.1038/s41592-020-0799-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {541-550},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed cre-dependent selection yields systemic AAVs for targeting distinct brain cell types},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Three-dimensional nanoscopy of whole cells and tissues with
in situ point spread function retrieval. <em>NMETH</em>, <em>17</em>(5),
531–540. (<a href="https://doi.org/10.1038/s41592-020-0816-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy is a powerful tool for visualizing subcellular structures, interactions and protein functions in biological research. However, inhomogeneous refractive indices inside cells and tissues distort the fluorescent signal emitted from single-molecule probes, which rapidly degrades resolution with increasing depth. We propose a method that enables the construction of an in situ 3D response of single emitters directly from single-molecule blinking datasets, and therefore allows their locations to be pinpointed with precision that achieves the Cramér-Rao lower bound and uncompromised fidelity. We demonstrate this method, named in situ PSF retrieval (INSPR), across a range of cellular and tissue architectures, from mitochondrial networks and nuclear pores in mammalian cells to amyloid-β plaques and dendrites in brain tissues and elastic fibers in developing cartilage of mice. This advancement expands the routine applicability of super-resolution microscopy from selected cellular targets near coverslips to intra- and extracellular targets deep inside tissues. In situ point spread function (PSF) retrieval (INSPR) enables precise single-molecule localization in 3D single-molecule localization microscopy of whole cells and tissues. It directly determines PSF from a single-molecule blinking dataset, removing errors associated with sample-induced aberrations.},
  archive      = {J_NMETH},
  author       = {Xu, Fan and Ma, Donghan and MacPherson, Kathryn P. and Liu, Sheng and Bu, Ye and Wang, Yu and Tang, Yu and Bi, Cheng and Kwok, Tim and Chubykin, Alexander A. and Yin, Peng and Calve, Sarah and Landreth, Gary E. and Huang, Fang},
  doi          = {10.1038/s41592-020-0816-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {531-540},
  shortjournal = {Nat. Meth.},
  title        = {Three-dimensional nanoscopy of whole cells and tissues with in situ point spread function retrieval},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-molecule displacement mapping unveils nanoscale
heterogeneities in intracellular diffusivity. <em>NMETH</em>,
<em>17</em>(5), 524–530. (<a
href="https://doi.org/10.1038/s41592-020-0793-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intracellular diffusion underlies vital cellular processes. However, it remains difficult to elucidate how an unbound protein diffuses inside the cell with good spatial resolution and sensitivity. Here we introduce single-molecule displacement/diffusivity mapping (SMdM), a super-resolution strategy that enables the nanoscale mapping of intracellular diffusivity through local statistics of the instantaneous displacements of freely diffusing single molecules. We thus show that the diffusion of an average-sized protein in the mammalian cytoplasm and nucleus is spatially heterogeneous at the nanoscale, and that variations in local diffusivity correlate with the ultrastructure of the actin cytoskeleton and the organization of the genome, respectively. SMdM of differently charged proteins further unveils that the possession of positive, but not negative, net charges drastically impedes diffusion, and that the rate is determined by the specific subcellular environments. We thus unveil rich heterogeneities and charge effects in intracellular diffusion at the nanoscale. Single-molecule displacement/diffusivity mapping (SMdM) enables nanoscale mapping of freely diffusing molecules in mammalian cells and reveals the structural basis of variations in local diffusivity in both the cytoplasm and nucleus.},
  archive      = {J_NMETH},
  author       = {Xiang, Limin and Chen, Kun and Yan, Rui and Li, Wan and Xu, Ke},
  doi          = {10.1038/s41592-020-0793-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {524-530},
  shortjournal = {Nat. Meth.},
  title        = {Single-molecule displacement mapping unveils nanoscale heterogeneities in intracellular diffusivity},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Kethoxal-assisted single-stranded DNA sequencing captures
global transcription dynamics and enhancer activity in situ.
<em>NMETH</em>, <em>17</em>(5), 515–523. (<a
href="https://doi.org/10.1038/s41592-020-0797-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription is a highly dynamic process that generates single-stranded DNA (ssDNA) in the genome as ‘transcription bubbles’. Here we describe a kethoxal-assisted single-stranded DNA sequencing (KAS-seq) approach, based on the fast and specific reaction between N3-kethoxal and guanines in ssDNA. KAS-seq allows rapid (within 5 min), sensitive and genome-wide capture and mapping of ssDNA produced by transcriptionally active RNA polymerases or other processes in situ using as few as 1,000 cells. KAS-seq enables definition of a group of enhancers that are single-stranded and enrich unique sequence motifs. These enhancers are associated with specific transcription-factor binding and exhibit more enhancer–promoter interactions than typical enhancers do. Under conditions that inhibit protein condensation, KAS-seq uncovers a rapid release of RNA polymerase II (Pol II) from a group of promoters. KAS-seq thus facilitates fast and accurate analysis of transcription dynamics and enhancer activities simultaneously in both low-input and high-throughput manner. KAS-seq applies N3-kethoxal to label guanines along single-stranded DNA in live cells, enabling characterization of ssDNA-involved transcription dynamics with as little as 1,000 cells.},
  archive      = {J_NMETH},
  author       = {Wu, Tong and Lyu, Ruitu and You, Qiancheng and He, Chuan},
  doi          = {10.1038/s41592-020-0797-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {515-523},
  shortjournal = {Nat. Meth.},
  title        = {Kethoxal-assisted single-stranded DNA sequencing captures global transcription dynamics and enhancer activity in situ},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Three-photon head-mounted microscope for imaging deep
cortical layers in freely moving rats. <em>NMETH</em>, <em>17</em>(5),
509–513. (<a href="https://doi.org/10.1038/s41592-020-0817-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We designed a head-mounted three-photon microscope for imaging deep cortical layer neuronal activity in a freely moving rat. Delivery of high-energy excitation pulses at 1,320 nm required both a hollow-core fiber whose transmission properties did not change with fiber movement and dispersion compensation. These developments enabled imaging at &amp;gt;1.1 mm below the cortical surface and stable imaging of layer 5 neuronal activity for &amp;gt;1 h in freely moving rats performing a range of behaviors. A head-mounted three-photon microscope based on a custom-designed optical fiber and dispersion compensation enables imaging of activity from neuronal populations deep in the cortex of freely moving rats.},
  archive      = {J_NMETH},
  author       = {Klioutchnikov, Alexandr and Wallace, Damian J. and Frosz, Michael H. and Zeltner, Richard and Sawinski, Juergen and Pawlak, Verena and Voit, Kay-Michael and Russell, Philip St. J. and Kerr, Jason N. D.},
  doi          = {10.1038/s41592-020-0817-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {509-513},
  shortjournal = {Nat. Meth.},
  title        = {Three-photon head-mounted microscope for imaging deep cortical layers in freely moving rats},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Combining native and “omics” mass spectrometry to identify
endogenous ligands bound to membrane proteins. <em>NMETH</em>,
<em>17</em>(5), 505–508. (<a
href="https://doi.org/10.1038/s41592-020-0821-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ligands bound to protein assemblies provide critical information for function, yet are often difficult to capture and define. Here we develop a top-down method, ‘nativeomics’, unifying ‘omics’ (lipidomics, proteomics, metabolomics) analysis with native mass spectrometry to identify ligands bound to membrane protein assemblies. By maintaining the link between proteins and ligands, we define the lipidome/metabolome in contact with membrane porins and a mitochondrial translocator to discover potential regulators of protein function. ‘Nativeomics’ enables identification of ligands bound to membrane proteins through detection of intact protein–ligand assemblies followed by dissociation and identification of individual ligands within the same mass spectrometry experiment.},
  archive      = {J_NMETH},
  author       = {Gault, Joseph and Liko, Idlir and Landreh, Michael and Shutin, Denis and Bolla, Jani Reddy and Jefferies, Damien and Agasid, Mark and Yen, Hsin-Yung and Ladds, Marcus J. G. W. and Lane, David P. and Khalid, Syma and Mullen, Christopher and Remes, Philip M. and Huguet, Romain and McAlister, Graeme and Goodwin, Michael and Viner, Rosa and Syka, John E.P. and Robinson, Carol V.},
  doi          = {10.1038/s41592-020-0821-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {505-508},
  shortjournal = {Nat. Meth.},
  title        = {Combining native and ‘omics’ mass spectrometry to identify endogenous ligands bound to membrane proteins},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Meltome atlas—thermal proteome stability across the tree of
life. <em>NMETH</em>, <em>17</em>(5), 495–503. (<a
href="https://doi.org/10.1038/s41592-020-0801-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have used a mass spectrometry-based proteomic approach to compile an atlas of the thermal stability of 48,000 proteins across 13 species ranging from archaea to humans and covering melting temperatures of 30–90 °C. Protein sequence, composition and size affect thermal stability in prokaryotes and eukaryotic proteins show a nonlinear relationship between the degree of disordered protein structure and thermal stability. The data indicate that evolutionary conservation of protein complexes is reflected by similar thermal stability of their proteins, and we show examples in which genomic alterations can affect thermal stability. Proteins of the respiratory chain were found to be very stable in many organisms, and human mitochondria showed close to normal respiration at 46 °C. We also noted cell-type-specific effects that can affect protein stability or the efficacy of drugs. This meltome atlas broadly defines the proteome amenable to thermal profiling in biology and drug discovery and can be explored online at http://meltomeatlas.proteomics.wzw.tum.de:5003/ and http://www.proteomicsdb.org . The meltome atlas compiles the thermal stability of 48,000 proteins across 13 species ranging from archaea to humans, providing a resource for analyzing protein stability in the context of function and interactions.},
  archive      = {J_NMETH},
  author       = {Jarzab, Anna and Kurzawa, Nils and Hopf, Thomas and Moerch, Matthias and Zecha, Jana and Leijten, Niels and Bian, Yangyang and Musiol, Eva and Maschberger, Melanie and Stoehr, Gabriele and Becher, Isabelle and Daly, Charlotte and Samaras, Patroklos and Mergner, Julia and Spanier, Britta and Angelov, Angel and Werner, Thilo and Bantscheff, Marcus and Wilhelm, Mathias and Klingenspor, Martin and Lemeer, Simone and Liebl, Wolfgang and Hahne, Hannes and Savitski, Mikhail M. and Kuster, Bernhard},
  doi          = {10.1038/s41592-020-0801-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {495-503},
  shortjournal = {Nat. Meth.},
  title        = {Meltome atlas—thermal proteome stability across the tree of life},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Genetic tool development in marine protists: Emerging model
organisms for experimental cell biology. <em>NMETH</em>, <em>17</em>(5),
481–494. (<a href="https://doi.org/10.1038/s41592-020-0796-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diverse microbial ecosystems underpin life in the sea. Among these microbes are many unicellular eukaryotes that span the diversity of the eukaryotic tree of life. However, genetic tractability has been limited to a few species, which do not represent eukaryotic diversity or environmentally relevant taxa. Here, we report on the development of genetic tools in a range of protists primarily from marine environments. We present evidence for foreign DNA delivery and expression in 13 species never before transformed and for advancement of tools for eight other species, as well as potential reasons for why transformation of yet another 17 species tested was not achieved. Our resource in genetic manipulation will provide insights into the ancestral eukaryotic lifeforms, general eukaryote cell biology, protein diversification and the evolution of cellular pathways. This Resource describes genetic tools for microbial eukaryotes, providing a roadmap for developing genetically tractable organisms.},
  archive      = {J_NMETH},
  author       = {Faktorová, Drahomíra and Nisbet, R. Ellen R. and Fernández Robledo, José A. and Casacuberta, Elena and Sudek, Lisa and Allen, Andrew E. and Ares, Manuel and Aresté, Cristina and Balestreri, Cecilia and Barbrook, Adrian C. and Beardslee, Patrick and Bender, Sara and Booth, David S. and Bouget, François-Yves and Bowler, Chris and Breglia, Susana A. and Brownlee, Colin and Burger, Gertraud and Cerutti, Heriberto and Cesaroni, Rachele and Chiurillo, Miguel A. and Clemente, Thomas and Coles, Duncan B. and Collier, Jackie L. and Cooney, Elizabeth C. and Coyne, Kathryn and Docampo, Roberto and Dupont, Christopher L. and Edgcomb, Virginia and Einarsson, Elin and Elustondo, Pía A. and Federici, Fernan and Freire-Beneitez, Veronica and Freyria, Nastasia J. and Fukuda, Kodai and García, Paulo A. and Girguis, Peter R. and Gomaa, Fatma and Gornik, Sebastian G. and Guo, Jian and Hampl, Vladimír and Hanawa, Yutaka and Haro-Contreras, Esteban R. and Hehenberger, Elisabeth and Highfield, Andrea and Hirakawa, Yoshihisa and Hopes, Amanda and Howe, Christopher J. and Hu, Ian and Ibañez, Jorge and Irwin, Nicholas A. T. and Ishii, Yuu and Janowicz, Natalia Ewa and Jones, Adam C. and Kachale, Ambar and Fujimura-Kamada, Konomi and Kaur, Binnypreet and Kaye, Jonathan Z. and Kazana, Eleanna and Keeling, Patrick J. and King, Nicole and Klobutcher, Lawrence A. and Lander, Noelia and Lassadi, Imen and Li, Zhuhong and Lin, Senjie and Lozano, Jean-Claude and Luan, Fulei and Maruyama, Shinichiro and Matute, Tamara and Miceli, Cristina and Minagawa, Jun and Moosburner, Mark and Najle, Sebastián R. and Nanjappa, Deepak and Nimmo, Isabel C. and Noble, Luke and Novák Vanclová, Anna M. G. and Nowacki, Mariusz and Nuñez, Isaac and Pain, Arnab and Piersanti, Angela and Pucciarelli, Sandra and Pyrih, Jan and Rest, Joshua S. and Rius, Mariana and Robertson, Deborah and Ruaud, Albane and Ruiz-Trillo, Iñaki and Sigg, Monika A. and Silver, Pamela A. and Slamovits, Claudio H. and Jason Smith, G. and Sprecher, Brittany N. and Stern, Rowena and Swart, Estienne C. and Tsaousis, Anastasios D. and Tsypin, Lev and Turkewitz, Aaron and Turnšek, Jernej and Valach, Matus and Vergé, Valérie and von Dassow, Peter and von der Haar, Tobias and Waller, Ross F. and Wang, Lu and Wen, Xiaoxue and Wheeler, Glen and Woods, April and Zhang, Huan and Mock, Thomas and Worden, Alexandra Z. and Lukeš, Julius},
  doi          = {10.1038/s41592-020-0796-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {481-494},
  shortjournal = {Nat. Meth.},
  title        = {Genetic tool development in marine protists: Emerging model organisms for experimental cell biology},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Anti-CRISPR protein applications: Natural brakes for
CRISPR-cas technologies. <em>NMETH</em>, <em>17</em>(5), 471–479. (<a
href="https://doi.org/10.1038/s41592-020-0771-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clustered, regularly interspaced short palindromic repeats (CRISPR) and CRISPR-associated (Cas) genes, a diverse family of prokaryotic adaptive immune systems, have emerged as a biotechnological tool and therapeutic. The discovery of protein inhibitors of CRISPR-Cas systems, called anti-CRISPR (Acr) proteins, enables the development of more controllable and precise CRISPR-Cas tools. Here we discuss applications of Acr proteins for post-translational control of CRISPR-Cas systems in prokaryotic and mammalian cells, organisms and ecosystems. This Review highlights recent discoveries and applications of anti-CRISPR (Acr) proteins that enable the regulation of CRISPR-Cas technology.},
  archive      = {J_NMETH},
  author       = {Marino, Nicole D. and Pinilla-Redondo, Rafael and Csörgő, Bálint and Bondy-Denomy, Joseph},
  doi          = {10.1038/s41592-020-0771-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {471-479},
  shortjournal = {Nat. Meth.},
  title        = {Anti-CRISPR protein applications: Natural brakes for CRISPR-cas technologies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A genetic toolbox for marine protists. <em>NMETH</em>,
<em>17</em>(5), 469–470. (<a
href="https://doi.org/10.1038/s41592-020-0794-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A resource of detailed DNA delivery and expression protocols for marine protists will enable new studies to understand the fundamental and ancestral features of eukaryotic cells.},
  archive      = {J_NMETH},
  author       = {Brown, Matthew W. and Tice, Alexander K.},
  doi          = {10.1038/s41592-020-0794-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {469-470},
  shortjournal = {Nat. Meth.},
  title        = {A genetic toolbox for marine protists},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Coronavirus jolts labs to warp speed. <em>NMETH</em>,
<em>17</em>(5), 465–468. (<a
href="https://doi.org/10.1038/s41592-020-0827-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To thwart SARS-CoV-2, labs hurry to hone assays, swap findings and share resources.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0827-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {465-468},
  shortjournal = {Nat. Meth.},
  title        = {Coronavirus jolts labs to warp speed},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Charting the human cell landscape. <em>NMETH</em>,
<em>17</em>(5), 463. (<a
href="https://doi.org/10.1038/s41592-020-0834-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers construct a transcriptome-based single-cell human cell landscape for major human organs.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0834-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {463},
  shortjournal = {Nat. Meth.},
  title        = {Charting the human cell landscape},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). A comparative atlas of the brain. <em>NMETH</em>,
<em>17</em>(5), 462. (<a
href="https://doi.org/10.1038/s41592-020-0842-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0842-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {462},
  shortjournal = {Nat. Meth.},
  title        = {A comparative atlas of the brain},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020q). Mining bacteria for biosensors. <em>NMETH</em>,
<em>17</em>(5), 462. (<a
href="https://doi.org/10.1038/s41592-020-0843-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0843-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {462},
  shortjournal = {Nat. Meth.},
  title        = {Mining bacteria for biosensors},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). A pan-plant protein complex map. <em>NMETH</em>,
<em>17</em>(5), 462. (<a
href="https://doi.org/10.1038/s41592-020-0844-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0844-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {462},
  shortjournal = {Nat. Meth.},
  title        = {A pan-plant protein complex map},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Paired TCR discovery in high throughput. <em>NMETH</em>,
<em>17</em>(5), 462. (<a
href="https://doi.org/10.1038/s41592-020-0845-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-0845-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {462},
  shortjournal = {Nat. Meth.},
  title        = {Paired TCR discovery in high throughput},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020n). Mapping spatial diversity. <em>NMETH</em>, <em>17</em>(5),
461. (<a href="https://doi.org/10.1038/s41592-020-0838-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0838-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {461},
  shortjournal = {Nat. Meth.},
  title        = {Mapping spatial diversity},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020r). Noncanonical amino acids on display. <em>NMETH</em>,
<em>17</em>(5), 461. (<a
href="https://doi.org/10.1038/s41592-020-0839-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0839-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {461},
  shortjournal = {Nat. Meth.},
  title        = {Noncanonical amino acids on display},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Diverse human genomes. <em>NMETH</em>, <em>17</em>(5), 461.
(<a href="https://doi.org/10.1038/s41592-020-0840-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0840-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {461},
  shortjournal = {Nat. Meth.},
  title        = {Diverse human genomes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). Guiding cas13 for RNA knockdown. <em>NMETH</em>,
<em>17</em>(5), 461. (<a
href="https://doi.org/10.1038/s41592-020-0841-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0841-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {461},
  shortjournal = {Nat. Meth.},
  title        = {Guiding cas13 for RNA knockdown},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Human tumors in immunocompetent mice. <em>NMETH</em>,
<em>17</em>(5), 460. (<a
href="https://doi.org/10.1038/s41592-020-0830-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new platform allows researchers to study the tumor microenvironment in fully immunocompetent chimeric mice.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-020-0830-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {460},
  shortjournal = {Nat. Meth.},
  title        = {Human tumors in immunocompetent mice},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ac). Predicting brain-disorder risk genes. <em>NMETH</em>,
<em>17</em>(5), 459. (<a
href="https://doi.org/10.1038/s41592-020-0833-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {H-MAGMA leverages chromatin interaction data to interpret GWAS results for brain disorders.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0833-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {459},
  shortjournal = {Nat. Meth.},
  title        = {Predicting brain-disorder risk genes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020m). It’s a material world. <em>NMETH</em>, <em>17</em>(5), 458.
(<a href="https://doi.org/10.1038/s41592-020-0836-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The assembly of polymers can be genetically targeted to specific neurons or other cells to manipulate their properties.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0836-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {458},
  shortjournal = {Nat. Meth.},
  title        = {It’s a material world},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Directing evolution in cells. <em>NMETH</em>,
<em>17</em>(5), 457. (<a
href="https://doi.org/10.1038/s41592-020-0829-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A CRISPR–Cas9-based approach generates libraries of mutant proteins targeted to specific subcellular compartments for in situ directed evolution.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0829-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {457},
  shortjournal = {Nat. Meth.},
  title        = {Directing evolution in cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Carol robinson. <em>NMETH</em>, <em>17</em>(5), 453. (<a
href="https://doi.org/10.1038/s41592-020-0824-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying things small and large in one mass spec experiment, and why persistence matters.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0824-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {453},
  shortjournal = {Nat. Meth.},
  title        = {Carol robinson},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). All methods on deck. <em>NMETH</em>, <em>17</em>(5), 451.
(<a href="https://doi.org/10.1038/s41592-020-0846-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More basic research studies of marine microorganisms — supported by new methods, tools and resources — are needed to help inform policies to mitigate the impact of climate change.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0846-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {451},
  shortjournal = {Nat. Meth.},
  title        = {All methods on deck},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Modeling infectious epidemics. <em>NMETH</em>,
<em>17</em>(5), 455–456. (<a
href="https://doi.org/10.1038/s41592-020-0822-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {“Every day sadder and sadder news of its increase. In the City died this week 7496; and of them, 6102 of the plague. But it is feared that the true number of the dead this week is near 10,000 ....” —Samuel Pepys, 1665},
  archive      = {J_NMETH},
  author       = {Bjørnstad, Ottar N. and Shea, Katriona and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-020-0822-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {5},
  pages        = {455-456},
  shortjournal = {Nat. Meth.},
  title        = {Modeling infectious epidemics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Machine learning analysis of whole mouse brain vasculature.
<em>NMETH</em>, <em>17</em>(4), 442–449. (<a
href="https://doi.org/10.1038/s41592-020-0792-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissue clearing methods enable the imaging of biological specimens without sectioning. However, reliable and scalable analysis of large imaging datasets in three dimensions remains a challenge. Here we developed a deep learning-based framework to quantify and analyze brain vasculature, named Vessel Segmentation &amp;amp; Analysis Pipeline (VesSAP). Our pipeline uses a convolutional neural network (CNN) with a transfer learning approach for segmentation and achieves human-level accuracy. By using VesSAP, we analyzed the vascular features of whole C57BL/6J, CD1 and BALB/c mouse brains at the micrometer scale after registering them to the Allen mouse brain atlas. We report evidence of secondary intracranial collateral vascularization in CD1 mice and find reduced vascularization of the brainstem in comparison to the cerebrum. Thus, VesSAP enables unbiased and scalable quantifications of the angioarchitecture of cleared mouse brains and yields biological insights into the vascular function of the brain. VesSAP is a tissue clearing- and deep learning-based pipeline for comprehensively analyzing mouse vasculature, from large vessels to small capillaries.},
  archive      = {J_NMETH},
  author       = {Todorov, Mihail Ivilinov and Paetzold, Johannes Christian and Schoppe, Oliver and Tetteh, Giles and Shit, Suprosanna and Efremov, Velizar and Todorov-Völgyi, Katalin and Düring, Marco and Dichgans, Martin and Piraud, Marie and Menze, Bjoern and Ertürk, Ali},
  doi          = {10.1038/s41592-020-0792-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {442-449},
  shortjournal = {Nat. Meth.},
  title        = {Machine learning analysis of whole mouse brain vasculature},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A 6-nm ultra-photostable DNA FluoroCube for fluorescence
imaging. <em>NMETH</em>, <em>17</em>(4), 437–441. (<a
href="https://doi.org/10.1038/s41592-020-0782-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photobleaching limits extended imaging of fluorescent biological samples. We developed DNA-based ‘FluoroCubes’ that are similar in size to the green fluorescent protein, have single-point attachment to proteins, have a ~54-fold higher photobleaching lifetime and emit ~43-fold more photons than single organic dyes. We demonstrate that DNA FluoroCubes provide outstanding tools for single-molecule imaging, allowing the tracking of single motor proteins for &amp;gt;800 steps with nanometer precision. DNA-based FluoroCubes are ~6 nm small, monovalent probes that contain six organic dyes, which confer dramatic photostability relative to single dyes and offer more uniform signal compared to quantum dots for extended biological imaging.},
  archive      = {J_NMETH},
  author       = {Niekamp, Stefan and Stuurman, Nico and Vale, Ronald D.},
  doi          = {10.1038/s41592-020-0782-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {437-441},
  shortjournal = {Nat. Meth.},
  title        = {A 6-nm ultra-photostable DNA FluoroCube for fluorescence imaging},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). 3D ATAC-PALM: Super-resolution imaging of the accessible
genome. <em>NMETH</em>, <em>17</em>(4), 430–436. (<a
href="https://doi.org/10.1038/s41592-020-0775-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To image the accessible genome at nanometer scale in situ, we developed three-dimensional assay for transposase-accessible chromatin-photoactivated localization microscopy (3D ATAC-PALM) that integrates an assay for transposase-accessible chromatin with visualization, PALM super-resolution imaging and lattice light-sheet microscopy. Multiplexed with oligopaint DNA–fluorescence in situ hybridization (FISH), RNA–FISH and protein fluorescence, 3D ATAC-PALM connected microscopy and genomic data, revealing spatially segregated accessible chromatin domains (ACDs) that enclose active chromatin and transcribed genes. Using these methods to analyze genetically perturbed cells, we demonstrated that genome architectural protein CTCF prevents excessive clustering of accessible chromatin and decompacts ACDs. These results highlight 3D ATAC-PALM as a useful tool to probe the structure and organizing mechanism of the genome. 3D ATAC-PALM integrates ATAC with super-resolution imaging for nanoscale views of the accessible genome. When combined with FISH, protein fluorescence and genetic perturbation, the method enables investigation of accessible chromatin in situ.},
  archive      = {J_NMETH},
  author       = {Xie, Liangqi and Dong, Peng and Chen, Xingqi and Hsieh, Tsung-Han S. and Banala, Sambashiva and De Marzio, Margherita and English, Brian P. and Qi, Yifeng and Jung, Seol Kyoung and Kieffer-Kwon, Kyong-Rim and Legant, Wesley R. and Hansen, Anders S. and Schulmann, Anton and Casellas, Rafael and Zhang, Bin and Betzig, Eric and Lavis, Luke D. and Chang, Howard Y. and Tjian, Robert and Liu, Zhe},
  doi          = {10.1038/s41592-020-0775-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {430-436},
  shortjournal = {Nat. Meth.},
  title        = {3D ATAC-PALM: Super-resolution imaging of the accessible genome},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). RecV recombinase system for in vivo targeted optogenomic
modifications of single cells or cell populations. <em>NMETH</em>,
<em>17</em>(4), 422–429. (<a
href="https://doi.org/10.1038/s41592-020-0774-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain circuits comprise vast numbers of interconnected neurons with diverse molecular, anatomical and physiological properties. To allow targeting of individual neurons for structural and functional studies, we created light-inducible site-specific DNA recombinases based on Cre, Dre and Flp (RecVs). RecVs can induce genomic modifications by one-photon or two-photon light induction in vivo. They can produce targeted, sparse and strong labeling of individual neurons by modifying multiple loci within mouse and zebrafish genomes. In combination with other genetic strategies, they allow intersectional targeting of different neuronal classes. In the mouse cortex they enable sparse labeling and whole-brain morphological reconstructions of individual neurons. Furthermore, these enzymes allow single-cell two-photon targeted genetic modifications and can be used in combination with functional optical indicators with minimal interference. In summary, RecVs enable spatiotemporally precise optogenomic modifications that can facilitate detailed single-cell analysis of neural circuits by linking genetic identity, morphology, connectivity and function. Light-dependent variants of Cre, Dre and Flp enable targeted sparse or single-cell labeling in mouse and zebrafish.},
  archive      = {J_NMETH},
  author       = {Yao, Shenqin and Yuan, Peng and Ouellette, Ben and Zhou, Thomas and Mortrud, Marty and Balaram, Pooja and Chatterjee, Soumya and Wang, Yun and Daigle, Tanya L. and Tasic, Bosiljka and Kuang, Xiuli and Gong, Hui and Luo, Qingming and Zeng, Shaoqun and Curtright, Andrew and Dhaka, Ajay and Kahan, Anat and Gradinaru, Viviana and Chrapkiewicz, Radosław and Schnitzer, Mark and Zeng, Hongkui and Cetin, Ali},
  doi          = {10.1038/s41592-020-0774-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {422-429},
  shortjournal = {Nat. Meth.},
  title        = {RecV recombinase system for in vivo targeted optogenomic modifications of single cells or cell populations},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Cardelino: Computational integration of somatic clonal
substructure and single-cell transcriptomes. <em>NMETH</em>,
<em>17</em>(4), 414–421. (<a
href="https://doi.org/10.1038/s41592-020-0766-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bulk and single-cell DNA sequencing has enabled reconstructing clonal substructures of somatic tissues from frequency and cooccurrence patterns of somatic variants. However, approaches to characterize phenotypic variations between clones are not established. Here we present cardelino ( https://github.com/single-cell-genetics/cardelino ), a computational method for inferring the clonal tree configuration and the clone of origin of individual cells assayed using single-cell RNA-seq (scRNA-seq). Cardelino flexibly integrates information from imperfect clonal trees inferred based on bulk exome-seq data, and sparse variant alleles expressed in scRNA-seq data. We apply cardelino to a published cancer dataset and to newly generated matched scRNA-seq and exome-seq data from 32 human dermal fibroblast lines, identifying hundreds of differentially expressed genes between cells from different somatic clones. These genes are frequently enriched for cell cycle and proliferation pathways, indicating a role for cell division genes in somatic evolution in healthy skin. Cardelino leverages variant information from single-cell RNA-seq data for inferring clonal tree configuration and mapping cells to their clone of origin.},
  archive      = {J_NMETH},
  author       = {McCarthy, Davis J. and Rostom, Raghd and Huang, Yuanhua and Kunz, Daniel J. and Danecek, Petr and Bonder, Marc Jan and Hagai, Tzachi and Lyu, Ruqian and Wang, Wenyi and Gaffney, Daniel J. and Simons, Benjamin D. and Stegle, Oliver and Teichmann, Sarah A.},
  doi          = {10.1038/s41592-020-0766-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {414-421},
  shortjournal = {Nat. Meth.},
  title        = {Cardelino: Computational integration of somatic clonal substructure and single-cell transcriptomes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). TooManyCells identifies and visualizes relationships of
single-cell clades. <em>NMETH</em>, <em>17</em>(4), 405–413. (<a
href="https://doi.org/10.1038/s41592-020-0748-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying and visualizing transcriptionally similar cells is instrumental for accurate exploration of the cellular diversity revealed by single-cell transcriptomics. However, widely used clustering and visualization algorithms produce a fixed number of cell clusters. A fixed clustering ‘resolution’ hampers our ability to identify and visualize echelons of cell states. We developed TooManyCells, a suite of graph-based algorithms for efficient and unbiased identification and visualization of cell clades. TooManyCells introduces a visualization model built on a concept intentionally orthogonal to dimensionality-reduction methods. TooManyCells is also equipped with an efficient matrix-free divisive hierarchical spectral clustering different from prevalent single-resolution clustering methods. TooManyCells enables multiresolution and multifaceted exploration of single-cell clades. An advantage of this paradigm is the immediate detection of rare and common populations that outperforms popular clustering and visualization algorithms, as demonstrated using existing single-cell transcriptomic data sets and new data modeling drug-resistance acquisition in leukemic T cells. The TooManyCells approach to scRNA-seq data facilitates efficient and unbiased identification and visualization of cell clades and rare subpopulations. Application of TooManyCells to drug-resistant leukemia cells identifies a rare resistant-like subpopulation of treatment-naive cells.},
  archive      = {J_NMETH},
  author       = {Schwartz, Gregory W. and Zhou, Yeqiao and Petrovic, Jelena and Fasolino, Maria and Xu, Lanwei and Shaffer, Sydney M. and Pear, Warren S. and Vahedi, Golnaz and Faryabi, Robert B.},
  doi          = {10.1038/s41592-020-0748-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {405-413},
  shortjournal = {Nat. Meth.},
  title        = {TooManyCells identifies and visualizes relationships of single-cell clades},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). TMTpro reagents: A set of isobaric labeling mass tags
enables simultaneous proteome-wide measurements across 16 samples.
<em>NMETH</em>, <em>17</em>(4), 399–404. (<a
href="https://doi.org/10.1038/s41592-020-0781-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Isobaric labeling empowers proteome-wide expression measurements simultaneously across multiple samples. Here an expanded set of 16 isobaric reagents based on an isobutyl-proline immonium ion reporter structure (TMTpro) is presented. These reagents have similar characteristics to existing tandem mass tag reagents but with increased fragmentation efficiency and signal. In a proteome-scale example dataset, we compared eight common cell lines with and without Torin1 treatment with three replicates, quantifying more than 8,800 proteins (mean of 7.5 peptides per protein) per replicate with an analysis time of only 1.1 h per proteome. Finally, we modified the thermal stability assay to examine proteome-wide melting shifts after treatment with DMSO, 1 or 20 µM staurosporine with five replicates. This assay identified and dose-stratified staurosporine binding to 228 cellular kinases in just one, 18-h experiment. TMTpro reagents allow complex experimental designs—all with essentially no missing values across the 16 samples and no loss in quantitative integrity. A set of isobaric labeling reagents called TMTpro enables deep quantitative comparisons of proteome measurements across 16 samples.},
  archive      = {J_NMETH},
  author       = {Li, Jiaming and Van Vranken, Jonathan G. and Pontano Vaites, Laura and Schweppe, Devin K. and Huttlin, Edward L. and Etienne, Chris and Nandhikonda, Premchendar and Viner, Rosa and Robitaille, Aaron M. and Thompson, Andrew H. and Kuhn, Karsten and Pike, Ian and Bomgarden, Ryan D. and Rogers, John C. and Gygi, Steven P. and Paulo, Joao A.},
  doi          = {10.1038/s41592-020-0781-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {399-404},
  shortjournal = {Nat. Meth.},
  title        = {TMTpro reagents: A set of isobaric labeling mass tags enables simultaneous proteome-wide measurements across 16 samples},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Resolving heterogeneous macromolecular assemblies by
orbitrap-based single-particle charge detection mass spectrometry.
<em>NMETH</em>, <em>17</em>(4), 395–398. (<a
href="https://doi.org/10.1038/s41592-020-0770-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We demonstrate single-particle charge detection mass spectrometry on an Orbitrap for the analysis of megadalton biomolecular assemblies. We establish that the signal amplitudes of individual ions scale linearly with their charge, which can be used to resolve mixed ion populations, determine charge states and thus also determine the masses of individual ions. This enables the ultrasensitive analysis of heterogeneous protein assemblies including immunoglobulin oligomers, ribosomes, proteinaceous nanocontainers and genome-packed adeno-associated viruses. An Orbitrap-based method for single-particle charge detection mass spectrometry enables facile, ultrasensitive analysis of protein assemblies.},
  archive      = {J_NMETH},
  author       = {Wörner, Tobias P. and Snijder, Joost and Bennett, Antonette and Agbandje-McKenna, Mavis and Makarov, Alexander A. and Heck, Albert J. R.},
  doi          = {10.1038/s41592-020-0770-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {395-398},
  shortjournal = {Nat. Meth.},
  title        = {Resolving heterogeneous macromolecular assemblies by orbitrap-based single-particle charge detection mass spectrometry},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Multiplexed mass spectrometry of individual ions improves
measurement of proteoforms and their complexes. <em>NMETH</em>,
<em>17</em>(4), 391–394. (<a
href="https://doi.org/10.1038/s41592-020-0764-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An Orbitrap-based ion analysis procedure determines the direct charge for numerous individual protein ions to generate true mass spectra. This individual ion mass spectrometry (I2MS) method for charge detection enables the characterization of highly complicated mixtures of proteoforms and their complexes in both denatured and native modes of operation, revealing information not obtainable by typical measurements of ensembles of ions. Charge assignment for individual ions unlocks complex mixtures of proteoforms and their assemblies by native mass spectrometry.},
  archive      = {J_NMETH},
  author       = {Kafader, Jared O. and Melani, Rafael D. and Durbin, Kenneth R. and Ikwuagwu, Bon and Early, Bryan P. and Fellers, Ryan T. and Beu, Steven C. and Zabrouskov, Vlad and Makarov, Alexander A. and Maze, Joshua T. and Shinholt, Deven L. and Yip, Ping F. and Tullman-Ercek, Danielle and Senko, Michael W. and Compton, Philip D. and Kelleher, Neil L.},
  doi          = {10.1038/s41592-020-0764-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {391-394},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed mass spectrometry of individual ions improves measurement of proteoforms and their complexes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Purification and enrichment of specific chromatin loci.
<em>NMETH</em>, <em>17</em>(4), 380–389. (<a
href="https://doi.org/10.1038/s41592-020-0765-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how chromatin is regulated is essential to fully grasp genome biology, and establishing the locus-specific protein composition is a major step toward this goal. Here we explain why the isolation and analysis of a specific chromatin segment are technically challenging, independently of the method. We then describe the published strategies and discuss their advantages and limitations. We conclude by discussing why significant technology developments are required to unambiguously describe the composition of small single loci. This Perspective discusses the challenges, progress and future of proteome profiling at specific chromatin loci for elucidating genome biology.},
  archive      = {J_NMETH},
  author       = {Gauchier, Mathilde and van Mierlo, Guido and Vermeulen, Michiel and Déjardin, Jérôme},
  doi          = {10.1038/s41592-020-0765-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {380-389},
  shortjournal = {Nat. Meth.},
  title        = {Purification and enrichment of specific chromatin loci},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Visualizing the genome in high resolution challenges our
textbook understanding. <em>NMETH</em>, <em>17</em>(4), 371–379. (<a
href="https://doi.org/10.1038/s41592-020-0758-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The relationship between the 4D folding of the genome and its function is an outstanding question in biology. A range of methods that probe the folding of the genome in space and time with unprecedented resolution have been developed. These methods, including chromosome conformation capture and high-resolution light and electron microscopy, are shedding new light on genome architecture and function. Here, we review the emerging picture of genome organization revealed by super-resolution and live-cell imaging. We compare and contrast population-based chromosome conformation capture approaches and imaging-based approaches and highlight future challenges. This Perspective highlights how high-resolution imaging has informed our view and helped overturn the textbook understanding of 4D genome organization.},
  archive      = {J_NMETH},
  author       = {Lakadamyali, Melike and Cosma, Maria Pia},
  doi          = {10.1038/s41592-020-0758-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {371-379},
  shortjournal = {Nat. Meth.},
  title        = {Visualizing the genome in high resolution challenges our textbook understanding},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Cancer labs reach beyond exhausted t cells. <em>NMETH</em>,
<em>17</em>(4), 367–370. (<a
href="https://doi.org/10.1038/s41592-020-0800-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells race to the scene in cancer and infection. To tease out what is special in cancer, scientists widen their scope and tool sets.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0800-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {367-370},
  shortjournal = {Nat. Meth.},
  title        = {Cancer labs reach beyond exhausted t cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ab). Pinpointing RNA mutations. <em>NMETH</em>, <em>17</em>(4),
365. (<a href="https://doi.org/10.1038/s41592-020-0805-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Riboregulator-based sensors are designed to differentiate single-nucleotide RNA variants in live cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0805-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {365},
  shortjournal = {Nat. Meth.},
  title        = {Pinpointing RNA mutations},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). A photoswitch for modulating neuronal activity.
<em>NMETH</em>, <em>17</em>(4), 364. (<a
href="https://doi.org/10.1038/s41592-020-0811-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0811-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {364},
  shortjournal = {Nat. Meth.},
  title        = {A photoswitch for modulating neuronal activity},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020u). Selective volume illumination microscopy. <em>NMETH</em>,
<em>17</em>(4), 364. (<a
href="https://doi.org/10.1038/s41592-020-0812-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0812-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {364},
  shortjournal = {Nat. Meth.},
  title        = {Selective volume illumination microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). Guide RNA resource for drosophila. <em>NMETH</em>,
<em>17</em>(4), 364. (<a
href="https://doi.org/10.1038/s41592-020-0813-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0813-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {364},
  shortjournal = {Nat. Meth.},
  title        = {Guide RNA resource for drosophila},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Beware the unexpected in cre drivers. <em>NMETH</em>,
<em>17</em>(4), 364. (<a
href="https://doi.org/10.1038/s41592-020-0815-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0815-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {364},
  shortjournal = {Nat. Meth.},
  title        = {Beware the unexpected in cre drivers},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Deeper imaging with space gating. <em>NMETH</em>,
<em>17</em>(4), 363. (<a
href="https://doi.org/10.1038/s41592-020-0808-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0808-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {363},
  shortjournal = {Nat. Meth.},
  title        = {Deeper imaging with space gating},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020r). Smart anti-glycan reagents. <em>NMETH</em>, <em>17</em>(4),
363. (<a href="https://doi.org/10.1038/s41592-020-0809-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0809-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {363},
  shortjournal = {Nat. Meth.},
  title        = {Smart anti-glycan reagents},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020w). Microbial factory for crRNAs. <em>NMETH</em>,
<em>17</em>(4), 363. (<a
href="https://doi.org/10.1038/s41592-020-0810-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0810-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {363},
  shortjournal = {Nat. Meth.},
  title        = {Microbial factory for crRNAs},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020o). Quantitative proteomics of cancer cell lines.
<em>NMETH</em>, <em>17</em>(4), 363. (<a
href="https://doi.org/10.1038/s41592-020-0814-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0814-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {363},
  shortjournal = {Nat. Meth.},
  title        = {Quantitative proteomics of cancer cell lines},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Controlling mRNA localization with light. <em>NMETH</em>,
<em>17</em>(4), 362. (<a
href="https://doi.org/10.1038/s41592-020-0804-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An optogenetic approach allows researchers to probe how mRNA localization affects cellular processes.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0804-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {362},
  shortjournal = {Nat. Meth.},
  title        = {Controlling mRNA localization with light},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). Identifying the chromatin-associated proteome.
<em>NMETH</em>, <em>17</em>(4), 361. (<a
href="https://doi.org/10.1038/s41592-020-0803-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {ChromID identifies proteins associated with various chromatin modifications.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0803-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {361},
  shortjournal = {Nat. Meth.},
  title        = {Identifying the chromatin-associated proteome},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020w). Venomous organoids. <em>NMETH</em>, <em>17</em>(4), 360.
(<a href="https://doi.org/10.1038/s41592-020-0806-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Snake venom gland organoids provide a glimpse into the cell biology of the respective organ and serve as a platform for producing snake toxins.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0806-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {360},
  shortjournal = {Nat. Meth.},
  title        = {Venomous organoids},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020s). Integrating lineage tracing and single-cell analysis.
<em>NMETH</em>, <em>17</em>(4), 359. (<a
href="https://doi.org/10.1038/s41592-020-0802-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {LARRY unveils transcriptomic changes leading to fate determination in hematopoiesis.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0802-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {359},
  shortjournal = {Nat. Meth.},
  title        = {Integrating lineage tracing and single-cell analysis},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Science in the time of coronavirus. <em>NMETH</em>,
<em>17</em>(4), 355. (<a
href="https://doi.org/10.1038/s41592-020-0807-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the incredible challenges, the science research community is stepping up to address the coronavirus pandemic in multiple ways.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0807-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {355},
  shortjournal = {Nat. Meth.},
  title        = {Science in the time of coronavirus},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Ali ertürk. <em>NMETH</em>, <em>17</em>(4), 357. (<a
href="https://doi.org/10.1038/s41592-020-0798-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analyzing cleared tissue with a deep-learning pipeline, and why dreaming is good for science.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0798-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {4},
  pages        = {357},
  shortjournal = {Nat. Meth.},
  title        = {Ali ertürk},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Conversion of human fibroblasts to
angioblast-like progenitor cells. <em>NMETH</em>, <em>17</em>(3), 353.
(<a href="https://doi.org/10.1038/s41592-020-0745-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Kurian, Leo and Sancho-Martinez, Ignacio and Nivet, Emmanuel and Aguirre, Aitor and Moon, Krystal and Pendaries, Caroline and Volle-Challier, Cecile and Bono, Francoise and Herbert, Jean-Marc and Pulecio, Julian and Xia, Yun and Li, Mo and Montserrat, Nuria and Ruiz, Sergio and Dubova, Ilir and Rodriguez, Concepcion and Denli, Ahmet M. and Boscolo, Francesca S. and Thiagarajan, Rathi D. and Gage, Fred H. and Loring, Jeanne F. and Laurent, Louise C. and Belmonte, Juan Carlos Izpisua},
  doi          = {10.1038/s41592-020-0745-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {353},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Conversion of human fibroblasts to angioblast-like progenitor cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020s). Publisher correction: Isolating neurons on the basis of
function. <em>NMETH</em>, <em>17</em>(3), 353. (<a
href="https://doi.org/10.1038/s41592-020-0773-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0773-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {353},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Isolating neurons on the basis of function},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: SciPy 1.0: Fundamental algorithms for
scientific computing in python. <em>NMETH</em>, <em>17</em>(3), 352. (<a
href="https://doi.org/10.1038/s41592-020-0772-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Virtanen, Pauli and Gommers, Ralf and Oliphant, Travis E. and Haberland, Matt and Reddy, Tyler and Cournapeau, David and Burovski, Evgeni and Peterson, Pearu and Weckesser, Warren and Bright, Jonathan and van der Walt, Stéfan J. and Brett, Matthew and Wilson, Joshua and Millman, K. Jarrod and Mayorov, Nikolay and Nelson, Andrew R. J. and Jones, Eric and Kern, Robert and Larson, Eric and Carey, C J and Polat, İlhan and Feng, Yu and Moore, Eric W. and VanderPlas, Jake and Laxalde, Denis and Perktold, Josef and Cimrman, Robert and Henriksen, Ian and Quintero, E. A. and Harris, Charles R. and Archibald, Anne M. and Ribeiro, Antônio H. and Pedregosa, Fabian and van Mulbregt, Paul},
  doi          = {10.1038/s41592-020-0772-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {352},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: SciPy 1.0: fundamental algorithms for scientific computing in python},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A virtual reality system to analyze neural activity and
behavior in adult zebrafish. <em>NMETH</em>, <em>17</em>(3), 343–351.
(<a href="https://doi.org/10.1038/s41592-020-0759-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Virtual realities are powerful tools to analyze and manipulate interactions between animals and their environment and to enable measurements of neuronal activity during behavior. In many species, however, optical access to the brain and/or the behavioral repertoire are limited. We developed a high-resolution virtual reality for head-restrained adult zebrafish, which exhibit cognitive behaviors not shown by larvae. We noninvasively measured activity throughout the dorsal telencephalon by multiphoton calcium imaging. Fish in the virtual reality showed regular swimming patterns and were attracted to animations of conspecifics. Manipulations of visuo-motor feedback revealed neurons that responded selectively to the mismatch between the expected and the actual visual consequences of motor output. Such error signals were prominent in multiple telencephalic areas, consistent with models of predictive processing. A virtual reality system for adult zebrafish therefore provides opportunities to analyze neuronal processing mechanisms underlying higher brain functions including decision making, associative learning, and social interactions. Complex behaviors and the underlying neural activity in adult zebrafish can be accessed through a virtual reality system in combination with two-photon microscopy.},
  archive      = {J_NMETH},
  author       = {Huang, Kuo-Hua and Rupprecht, Peter and Frank, Thomas and Kawakami, Koichi and Bouwmeester, Tewis and Friedrich, Rainer W.},
  doi          = {10.1038/s41592-020-0759-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {343-351},
  shortjournal = {Nat. Meth.},
  title        = {A virtual reality system to analyze neural activity and behavior in adult zebrafish},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Cell-type-specific signaling networks in heterocellular
organoids. <em>NMETH</em>, <em>17</em>(3), 335–342. (<a
href="https://doi.org/10.1038/s41592-020-0737-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the widespread adoption of organoids as biomimetic tissue models, methods to comprehensively analyze cell-type-specific post-translational modification (PTM) signaling networks in organoids are absent. Here, we report multivariate single-cell analysis of such networks in organoids and organoid cocultures. Simultaneous analysis by mass cytometry of 28 PTMs in &amp;gt;1 million single cells derived from small intestinal organoids reveals cell-type- and cell-state-specific signaling networks in stem, Paneth, enteroendocrine, tuft and goblet cells, as well as enterocytes. Integrating single-cell PTM analysis with thiol-reactive organoid barcoding in situ (TOBis) enables high-throughput comparison of signaling networks between organoid cultures. Cell-type-specific PTM analysis of colorectal cancer organoid cocultures reveals that shApc, KrasG12D and Trp53R172H cell-autonomously mimic signaling states normally induced by stromal fibroblasts and macrophages. These results demonstrate how standard mass cytometry workflows can be modified to perform high-throughput multivariate cell-type-specific signaling analysis of healthy and cancerous organoids. Mass cytometry in combination with a thiol-reactive barcoding strategy allows analysis and comparison of cell-type-specific signaling networks in organoids.},
  archive      = {J_NMETH},
  author       = {Qin, Xiao and Sufi, Jahangir and Vlckova, Petra and Kyriakidou, Pelagia and Acton, Sophie E. and Li, Vivian S. W. and Nitz, Mark and Tape, Christopher J.},
  doi          = {10.1038/s41592-020-0737-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {335-342},
  shortjournal = {Nat. Meth.},
  title        = {Cell-type-specific signaling networks in heterocellular organoids},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Measurement of atom resolvability in cryo-EM maps with
q-scores. <em>NMETH</em>, <em>17</em>(3), 328–334. (<a
href="https://doi.org/10.1038/s41592-020-0731-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron microscopy (cryo-EM) maps are now at the point where resolvability of individual atoms can be achieved. However, resolvability is not necessarily uniform throughout the map. We introduce a quantitative parameter to characterize the resolvability of individual atoms in cryo-EM maps, the map Q-score. Q-scores can be calculated for atoms in proteins, nucleic acids, water, ligands and other solvent atoms, using models fitted to or derived from cryo-EM maps. Q-scores can also be averaged to represent larger features such as entire residues and nucleotides. Averaged over entire models, Q-scores correlate very well with the estimated resolution of cryo-EM maps for both protein and RNA. Assuming the models they are calculated from are well fitted to the map, Q-scores can be used as a measure of resolvability in cryo-EM maps at various scales, from entire macromolecules down to individual atoms. Q-score analysis of multiple cryo-EM maps of the same proteins derived from different laboratories confirms the reproducibility of structural features from side chains down to water and ion atoms. Q-scores provide a quantitative metric for resolvability in cryo-EM maps, and can be used at the atom, residue or macromolecule scale.},
  archive      = {J_NMETH},
  author       = {Pintilie, Grigore and Zhang, Kaiming and Su, Zhaoming and Li, Shanshan and Schmid, Michael F. and Chiu, Wah},
  doi          = {10.1038/s41592-020-0731-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {328-334},
  shortjournal = {Nat. Meth.},
  title        = {Measurement of atom resolvability in cryo-EM maps with Q-scores},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Long-range single-molecule mapping of chromatin
accessibility in eukaryotes. <em>NMETH</em>, <em>17</em>(3), 319–327.
(<a href="https://doi.org/10.1038/s41592-019-0730-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping open chromatin regions has emerged as a widely used tool for identifying active regulatory elements in eukaryotes. However, existing approaches, limited by reliance on DNA fragmentation and short-read sequencing, cannot provide information about large-scale chromatin states or reveal coordination between the states of distal regulatory elements. We have developed a method for profiling the accessibility of individual chromatin fibers, a single-molecule long-read accessible chromatin mapping sequencing assay (SMAC-seq), enabling the simultaneous, high-resolution, single-molecule assessment of chromatin states at multikilobase length scales. Our strategy is based on combining the preferential methylation of open chromatin regions by DNA methyltransferases with low sequence specificity, in this case EcoGII, an N6-methyladenosine (m6A) methyltransferase, and the ability of nanopore sequencing to directly read DNA modifications. We demonstrate that aggregate SMAC-seq signals match bulk-level accessibility measurements, observe single-molecule nucleosome and transcription factor protection footprints, and quantify the correlation between chromatin states of distal genomic elements. SMAC-seq combines long-read sequencing with open chromatin methylation by DNA methyltransferases to enable mapping of nucleosome position and chromatin accessibility.},
  archive      = {J_NMETH},
  author       = {Shipony, Zohar and Marinov, Georgi K. and Swaffer, Matthew P. and Sinnott-Armstrong, Nicholas A. and Skotheim, Jan M. and Kundaje, Anshul and Greenleaf, William J.},
  doi          = {10.1038/s41592-019-0730-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {319-327},
  shortjournal = {Nat. Meth.},
  title        = {Long-range single-molecule mapping of chromatin accessibility in eukaryotes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An optimized chemical-genetic method for cell-specific
metabolic labeling of RNA. <em>NMETH</em>, <em>17</em>(3), 311–318. (<a
href="https://doi.org/10.1038/s41592-019-0726-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissues and organs are composed of diverse cell types, which poses a major challenge for cell-type-specific profiling of gene expression. Current metabolic labeling methods rely on exogenous pyrimidine analogs that are only incorporated into RNA in cells expressing an exogenous enzyme. This approach assumes that off-target cells cannot incorporate these analogs. We disprove this assumption and identify and characterize the enzymatic pathways responsible for high background incorporation. We demonstrate that mammalian cells can incorporate uracil analogs and characterize the enzymatic pathways responsible for high background incorporation. To overcome these limitations, we developed a new small molecule–enzyme pair consisting of uridine/cytidine kinase 2 and 2′-azidouridine. We demonstrate that 2′-azidouridine is only incorporated in cells expressing uridine/cytidine kinase 2 and characterize selectivity mechanisms using molecular dynamics and X-ray crystallography. Furthermore, this pair can be used to purify and track RNA from specific cellular populations, making it ideal for high-resolution cell-specific RNA labeling. Overall, these results reveal new aspects of mammalian salvage pathways and serve as a new benchmark for designing, characterizing and evaluating methodologies for cell-specific labeling of biomolecules. Ubiquitous mammalian enzymes can scavenge uracil analogs, leading to non-specific background in cell-type-specific RNA labeling. This work reveals the enzymes involved and describes the uridine/cytidine kinase 2 and 2′-azidouridine pair as a highly specific and non-toxic alternative.},
  archive      = {J_NMETH},
  author       = {Nainar, Sarah and Cuthbert, Bonnie J. and Lim, Nathan M. and England, Whitney E. and Ke, Ke and Sophal, Kanika and Quechol, Robert and Mobley, David L. and Goulding, Celia W. and Spitale, Robert C.},
  doi          = {10.1038/s41592-019-0726-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {311-318},
  shortjournal = {Nat. Meth.},
  title        = {An optimized chemical-genetic method for cell-specific metabolic labeling of RNA},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Uncovering axes of variation among single-cell cancer
specimens. <em>NMETH</em>, <em>17</em>(3), 302–310. (<a
href="https://doi.org/10.1038/s41592-019-0689-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While several tools have been developed to map axes of variation among individual cells, no analogous approaches exist for identifying axes of variation among multicellular biospecimens profiled at single-cell resolution. For this purpose, we developed ‘phenotypic earth mover’s distance’ (PhEMD). PhEMD is a general method for embedding a ‘manifold of manifolds’, in which each datapoint in the higher-level manifold (of biospecimens) represents a collection of points that span a lower-level manifold (of cells). We apply PhEMD to a newly generated drug-screen dataset and demonstrate that PhEMD uncovers axes of cell subpopulational variation among a large set of perturbation conditions. Moreover, we show that PhEMD can be used to infer the phenotypes of biospecimens not directly profiled. Applied to clinical datasets, PhEMD generates a map of the patient-state space that highlights sources of patient-to-patient variation. PhEMD is scalable, compatible with leading batch-effect correction techniques and generalizable to multiple experimental designs. Phenotypic earth mover’s distance (PhEMD) facilitates the comparison of single-cell experimental conditions, each of which is a high-dimensional dataset, and identifies axes of variation among multicellular biospecimens.},
  archive      = {J_NMETH},
  author       = {Chen, William S. and Zivanovic, Nevena and van Dijk, David and Wolf, Guy and Bodenmiller, Bernd and Krishnaswamy, Smita},
  doi          = {10.1038/s41592-019-0689-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {302-310},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering axes of variation among single-cell cancer specimens},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Sketching algorithms for genomic data analysis and querying
in a secure enclave. <em>NMETH</em>, <em>17</em>(3), 295–301. (<a
href="https://doi.org/10.1038/s41592-020-0761-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS), especially on rare diseases, may necessitate exchange of sensitive genomic data between multiple institutions. Since genomic data sharing is often infeasible due to privacy concerns, cryptographic methods, such as secure multiparty computation (SMC) protocols, have been developed with the aim of offering privacy-preserving collaborative GWAS. Unfortunately, the computational overhead of these methods remain prohibitive for human-genome-scale data. Here we introduce SkSES (https://github.com/ndokmai/sgx-genome-variants-search), a hardware–software hybrid approach for privacy-preserving collaborative GWAS, which improves the running time of the most advanced cryptographic protocols by two orders of magnitude. The SkSES approach is based on trusted execution environments (TEEs) offered by current-generation microprocessors—in particular, Intel’s SGX. To overcome the severe memory limitation of the TEEs, SkSES employs novel ‘sketching’ algorithms that maintain essential statistical information on genomic variants in input VCF files. By additionally incorporating efficient data compression and population stratification reduction methods, SkSES identifies the top k genomic variants in a cohort quickly, accurately and in a privacy-preserving manner. The combination of Intel SGX platform with sketching algorithms enables efficient compaction of genomic data and the execution of secure GWAS in an untrusted cloud environment.},
  archive      = {J_NMETH},
  author       = {Kockan, Can and Zhu, Kaiyuan and Dokmai, Natnatee and Karpov, Nikolai and Kulekci, M. Oguzhan and Woodruff, David P. and Sahinalp, S. Cenk},
  doi          = {10.1038/s41592-020-0761-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {295-301},
  shortjournal = {Nat. Meth.},
  title        = {Sketching algorithms for genomic data analysis and querying in a secure enclave},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Rapid mesoscale volumetric imaging of neural activity with
synaptic resolution. <em>NMETH</em>, <em>17</em>(3), 291–294. (<a
href="https://doi.org/10.1038/s41592-020-0760-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging neurons and neural circuits over large volumes at high speed and subcellular resolution is a difficult task. Incorporating a Bessel focus module into a two-photon fluorescence mesoscope, we achieved rapid volumetric imaging of neural activity over the mesoscale with synaptic resolution. We applied the technology to calcium imaging of entire dendritic spans of neurons as well as neural ensembles within multiple cortical regions over two hemispheres of the awake mouse brain. Integrating a Bessel focus module into a two-photon fluorescence mesoscope enables high-speed volumetric imaging of neuronal activity in soma, dendrites and spines.},
  archive      = {J_NMETH},
  author       = {Lu, Rongwen and Liang, Yajie and Meng, Guanghan and Zhou, Pengcheng and Svoboda, Karel and Paninski, Liam and Ji, Na},
  doi          = {10.1038/s41592-020-0760-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {291-294},
  shortjournal = {Nat. Meth.},
  title        = {Rapid mesoscale volumetric imaging of neural activity with synaptic resolution},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Kilohertz two-photon fluorescence microscopy imaging of
neural activity in vivo. <em>NMETH</em>, <em>17</em>(3), 287–290. (<a
href="https://doi.org/10.1038/s41592-020-0762-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding information processing in the brain requires monitoring neuronal activity at high spatiotemporal resolution. Using an ultrafast two-photon fluorescence microscope empowered by all-optical laser scanning, we imaged neuronal activity in vivo at up to 3,000 frames per second and submicrometer spatial resolution. This imaging method enabled monitoring of both supra- and subthreshold electrical activity down to 345 μm below the brain surface in head-fixed awake mice. High-speed two-photon laser scanning microscopy using a passive laser scanner based on free-space angular-chirp-enhanced delay achieves frame rates suitable for voltage imaging in vivo in the mouse brain.},
  archive      = {J_NMETH},
  author       = {Wu, Jianglai and Liang, Yajie and Chen, Shuo and Hsu, Ching-Lung and Chavarha, Mariya and Evans, Stephen W. and Shi, Dongqing and Lin, Michael Z. and Tsia, Kevin K. and Ji, Na},
  doi          = {10.1038/s41592-020-0762-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {287-290},
  shortjournal = {Nat. Meth.},
  title        = {Kilohertz two-photon fluorescence microscopy imaging of neural activity in vivo},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Simultaneous multiplane imaging with reverberation
two-photon microscopy. <em>NMETH</em>, <em>17</em>(3), 283–286. (<a
href="https://doi.org/10.1038/s41592-019-0728-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiphoton microscopy has gained enormous popularity because of its unique capacity to provide high-resolution images from deep within scattering tissue. Here, we demonstrate video-rate multiplane imaging with two-photon microscopy by performing near-instantaneous axial scanning while maintaining three-dimensional micrometer-scale resolution. Our technique, termed reverberation microscopy, enables the monitoring of neuronal populations over large depth ranges and can be implemented as a simple add-on to a conventional design. Reverberation two-photon microscopy enables video-rate multiplane neuroimaging by performing near-instantaneous axial scanning over large depth ranges while maintaining 3D micrometer-scale resolution.},
  archive      = {J_NMETH},
  author       = {Beaulieu, Devin R. and Davison, Ian G. and Kılıç, Kıvılcım and Bifano, Thomas G. and Mertz, Jerome},
  doi          = {10.1038/s41592-019-0728-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {283-286},
  shortjournal = {Nat. Meth.},
  title        = {Simultaneous multiplane imaging with reverberation two-photon microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Chemogenetic control of nanobodies. <em>NMETH</em>,
<em>17</em>(3), 279–282. (<a
href="https://doi.org/10.1038/s41592-020-0746-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce an engineered nanobody whose affinity to green fluorescent protein (GFP) can be switched on and off with small molecules. By controlling the cellular localization of GFP fusion proteins, the engineered nanobody allows interrogation of their roles in basic biological processes, an approach that should be applicable to numerous previously described GFP fusions. We also outline how the binding affinities of other nanobodies can be controlled by small molecules. Engineered nanobody allows reversible control of activity in cells through the binding of small molecules.},
  archive      = {J_NMETH},
  author       = {Farrants, Helen and Tarnawski, Miroslaw and Müller, Thorsten G. and Otsuka, Shotaro and Hiblot, Julien and Koch, Birgit and Kueblbeck, Moritz and Kräusslich, Hans-Georg and Ellenberg, Jan and Johnsson, Kai},
  doi          = {10.1038/s41592-020-0746-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {279-282},
  shortjournal = {Nat. Meth.},
  title        = {Chemogenetic control of nanobodies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reply to: Spatial heterogeneity in molecular brightness.
<em>NMETH</em>, <em>17</em>(3), 276–278. (<a
href="https://doi.org/10.1038/s41592-020-0735-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Stoneman, Michael R. and Biener, Gabriel and Raicu, Valerică},
  doi          = {10.1038/s41592-020-0735-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {276-278},
  shortjournal = {Nat. Meth.},
  title        = {Reply to: Spatial heterogeneity in molecular brightness},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Spatial heterogeneity in molecular brightness.
<em>NMETH</em>, <em>17</em>(3), 273–275. (<a
href="https://doi.org/10.1038/s41592-020-0732-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Annibale, Paolo and Lohse, Martin J.},
  doi          = {10.1038/s41592-020-0732-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {273-275},
  shortjournal = {Nat. Meth.},
  title        = {Spatial heterogeneity in molecular brightness},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). SciPy 1.0: Fundamental algorithms for scientific computing
in python. <em>NMETH</em>, <em>17</em>(3), 261–272. (<a
href="https://doi.org/10.1038/s41592-019-0686-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SciPy is an open-source scientific computing library for the Python programming language. Since its initial release in 2001, SciPy has become a de facto standard for leveraging scientific algorithms in Python, with over 600 unique code contributors, thousands of dependent packages, over 100,000 dependent repositories and millions of downloads per year. In this work, we provide an overview of the capabilities and development practices of SciPy 1.0 and highlight some recent technical developments. This Perspective describes the development and capabilities of SciPy 1.0, an open source scientific computing library for the Python programming language.},
  archive      = {J_NMETH},
  author       = {Virtanen, Pauli and Gommers, Ralf and Oliphant, Travis E. and Haberland, Matt and Reddy, Tyler and Cournapeau, David and Burovski, Evgeni and Peterson, Pearu and Weckesser, Warren and Bright, Jonathan and van der Walt, Stéfan J. and Brett, Matthew and Wilson, Joshua and Millman, K. Jarrod and Mayorov, Nikolay and Nelson, Andrew R. J. and Jones, Eric and Kern, Robert and Larson, Eric and Carey, C J and Polat, İlhan and Feng, Yu and Moore, Eric W. and VanderPlas, Jake and Laxalde, Denis and Perktold, Josef and Cimrman, Robert and Henriksen, Ian and Quintero, E. A. and Harris, Charles R. and Archibald, Anne M. and Ribeiro, Antônio H. and Pedregosa, Fabian and van Mulbregt, Paul},
  doi          = {10.1038/s41592-019-0686-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {261-272},
  shortjournal = {Nat. Meth.},
  title        = {SciPy 1.0: Fundamental algorithms for scientific computing in python},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Precision control of intrabodies in live cells.
<em>NMETH</em>, <em>17</em>(3), 259–260. (<a
href="https://doi.org/10.1038/s41592-020-0767-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New methods from the Heo and Johnsson labs enable light- and drug-inducible control of intrabody binding in cells.},
  archive      = {J_NMETH},
  author       = {Marzilli, Alexander M. and McMahan, Jeffrey B. and Ngo, John T.},
  doi          = {10.1038/s41592-020-0767-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {259-260},
  shortjournal = {Nat. Meth.},
  title        = {Precision control of intrabodies in live cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Bench pressing with genomics benchmarkers. <em>NMETH</em>,
<em>17</em>(3), 255–258. (<a
href="https://doi.org/10.1038/s41592-020-0768-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Some -omics tools can be more accurate, sensitive or efficient than others. Yet benchmarking is no tell-all.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0768-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {255-258},
  shortjournal = {Nat. Meth.},
  title        = {Bench pressing with genomics benchmarkers},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020t). Learning the diversity of giant viruses. <em>NMETH</em>,
<em>17</em>(3), 253. (<a
href="https://doi.org/10.1038/s41592-020-0778-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers identify a broad phylogenetic diversity of nucleocytoplasmic large DNA viruses by mining global-scale metagenomics datasets.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0778-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {253},
  shortjournal = {Nat. Meth.},
  title        = {Learning the diversity of giant viruses},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ai). Single-cell structural variations. <em>NMETH</em>,
<em>17</em>(3), 252. (<a
href="https://doi.org/10.1038/s41592-020-0787-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0787-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {252},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell structural variations},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). A multiplexable miniscope. <em>NMETH</em>, <em>17</em>(3),
252. (<a href="https://doi.org/10.1038/s41592-020-0788-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0788-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {252},
  shortjournal = {Nat. Meth.},
  title        = {A multiplexable miniscope},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020w). Smart rotation for better SPIM. <em>NMETH</em>,
<em>17</em>(3), 252. (<a
href="https://doi.org/10.1038/s41592-020-0789-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0789-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {252},
  shortjournal = {Nat. Meth.},
  title        = {Smart rotation for better SPIM},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020t). Say goodbye to sCMOS noise. <em>NMETH</em>, <em>17</em>(3),
252. (<a href="https://doi.org/10.1038/s41592-020-0790-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0790-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {252},
  shortjournal = {Nat. Meth.},
  title        = {Say goodbye to sCMOS noise},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020p). Mid-IR optoacoustic microscopy. <em>NMETH</em>,
<em>17</em>(3), 251. (<a
href="https://doi.org/10.1038/s41592-020-0783-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0783-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {251},
  shortjournal = {Nat. Meth.},
  title        = {Mid-IR optoacoustic microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Biotin “painting” measures disorder. <em>NMETH</em>,
<em>17</em>(3), 251. (<a
href="https://doi.org/10.1038/s41592-020-0784-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0784-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {251},
  shortjournal = {Nat. Meth.},
  title        = {Biotin ‘painting’ measures disorder},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020o). High-throughput screening with transcriptomics.
<em>NMETH</em>, <em>17</em>(3), 251. (<a
href="https://doi.org/10.1038/s41592-020-0785-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0785-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {251},
  shortjournal = {Nat. Meth.},
  title        = {High-throughput screening with transcriptomics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). A detailed marmoset brain atlas. <em>NMETH</em>,
<em>17</em>(3), 251. (<a
href="https://doi.org/10.1038/s41592-020-0786-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0786-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {251},
  shortjournal = {Nat. Meth.},
  title        = {A detailed marmoset brain atlas},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020q). Inferring tumor subclonality. <em>NMETH</em>,
<em>17</em>(3), 250. (<a
href="https://doi.org/10.1038/s41592-020-0777-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A framework for evaluating tools that reconstruct tumor subclonal structure.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0777-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {250},
  shortjournal = {Nat. Meth.},
  title        = {Inferring tumor subclonality},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Deep learning 3D structures. <em>NMETH</em>,
<em>17</em>(3), 249. (<a
href="https://doi.org/10.1038/s41592-020-0779-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent developments in deep-learning-based methods improve protein structure prediction.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0779-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {249},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning 3D structures},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020q). Photoacoustics in a snap. <em>NMETH</em>, <em>17</em>(3),
248. (<a href="https://doi.org/10.1038/s41592-020-0780-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photoacoustic topography through an ergodic relay increases imaging throughput and opens a path toward wearable devices.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0780-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {248},
  shortjournal = {Nat. Meth.},
  title        = {Photoacoustics in a snap},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). CLEM takes a polar plunge. <em>NMETH</em>, <em>17</em>(3),
247. (<a href="https://doi.org/10.1038/s41592-020-0776-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combining cryogenic super-resolution microscopy and focused ion beam scanning electron microscopy allows nanoscale views of protein architecture in the context of cellular ultrastructure in whole cells.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0776-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {247},
  shortjournal = {Nat. Meth.},
  title        = {CLEM takes a polar plunge},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020q). Na ji. <em>NMETH</em>, <em>17</em>(3), 245. (<a
href="https://doi.org/10.1038/s41592-020-0769-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How joy and wide-ranging curiosity leads to neurobiology tools for new types of questions.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0769-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {245},
  shortjournal = {Nat. Meth.},
  title        = {Na ji},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Introducing matters arising. <em>NMETH</em>, <em>17</em>(3),
243. (<a href="https://doi.org/10.1038/s41592-020-0791-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discourse is fundamental to science. We now have a dedicated format for constructive comments on Nature Methods papers, called Matters Arising.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0791-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {243},
  shortjournal = {Nat. Meth.},
  title        = {Introducing matters arising},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Orchestrating single-cell analysis
with bioconductor. <em>NMETH</em>, <em>17</em>(2), 242. (<a
href="https://doi.org/10.1038/s41592-019-0700-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Amezquita, Robert A. and Lun, Aaron T. L. and Becht, Etienne and Carey, Vince J. and Carpp, Lindsay N. and Geistlinger, Ludwig and Marini, Federico and Rue-Albrecht, Kevin and Risso, Davide and Soneson, Charlotte and Waldron, Levi and Pagès, Hervé and Smith, Mike L. and Huber, Wolfgang and Morgan, Martin and Gottardo, Raphael and Hicks, Stephanie C.},
  doi          = {10.1038/s41592-019-0700-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {242},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Orchestrating single-cell analysis with bioconductor},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Directed evolution improves the
catalytic efficiency of TEV protease. <em>NMETH</em>, <em>17</em>(2),
242. (<a href="https://doi.org/10.1038/s41592-019-0729-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Sanchez, Mateo I. and Ting, Alice Y.},
  doi          = {10.1038/s41592-019-0729-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {242},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Directed evolution improves the catalytic efficiency of TEV protease},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Amplification-free single-cell
whole-genome sequencing gets a makeover. <em>NMETH</em>, <em>17</em>(2),
242. (<a href="https://doi.org/10.1038/s41592-020-0739-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Todorovic, Vesna},
  doi          = {10.1038/s41592-020-0739-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {242},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Amplification-free single-cell whole-genome sequencing gets a makeover},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: Nucleus segmentation across imaging
experiments: The 2018 data science bowl. <em>NMETH</em>, <em>17</em>(2),
241. (<a href="https://doi.org/10.1038/s41592-020-0733-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Caicedo, Juan C. and Goodman, Allen and Karhohs, Kyle W. and Cimini, Beth A. and Ackerman, Jeanelle and Haghighi, Marzieh and Heng, CherKeng and Becker, Tim and Doan, Minh and McQuin, Claire and Rohban, Mohammad and Singh, Shantanu and Carpenter, Anne E.},
  doi          = {10.1038/s41592-020-0733-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {241},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: nucleus segmentation across imaging experiments: the 2018 data science bowl},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Analysis of the human protein atlas
image classification competition. <em>NMETH</em>, <em>17</em>(2), 241.
(<a href="https://doi.org/10.1038/s41592-020-0734-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Ouyang, Wei and Winsnes, Casper F. and Hjelmare, Martin and Cesnik, Anthony J. and Åkesson, Lovisa and Xu, Hao and Sullivan, Devin P. and Dai, Shubin and Lan, Jun and Jinmo, Park and Galib, Shaikat M. and Henkel, Christof and Hwang, Kevin and Poplavskiy, Dmytro and Tunguz, Bojan and Wolfinger, Russel D. and Gu, Yinzheng and Li, Chuanpeng and Xie, Jinbin and Buslov, Dmitry and Fironov, Sergei and Kiselev, Alexander and Panchenko, Dmytro and Cao, Xuan and Wei, Runmin and Wu, Yuanhao and Zhu, Xun and Tseng, Kuan-Lun and Gao, Zhifeng and Ju, Cheng and Yi, Xiaohan and Zheng, Hongdong and Kappel, Constantin and Lundberg, Emma},
  doi          = {10.1038/s41592-020-0734-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {241},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Analysis of the human protein atlas image classification competition},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Template-free detection and
classification of membrane-bound complexes in cryo-electron tomograms.
<em>NMETH</em>, <em>17</em>(2), 240. (<a
href="https://doi.org/10.1038/s41592-020-0747-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Martinez-Sanchez, Antonio and Kochovski, Zdravko and Laugks, Ulrike and Meyer zum Alten Borgloh, Johannes and Chakraborty, Saikat and Pfeffer, Stefan and Baumeister, Wolfgang and Lučić, Vladan},
  doi          = {10.1038/s41592-020-0747-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {240},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Template-free detection and classification of membrane-bound complexes in cryo-electron tomograms},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Author correction: Template-free detection and
classification of membrane-bound complexes in cryo-electron tomograms.
<em>NMETH</em>, <em>17</em>(2), 240. (<a
href="https://doi.org/10.1038/s41592-020-0763-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Martinez-Sanchez, Antonio and Kochovski, Zdravko and Laugks, Ulrike and Meyer zum Alten Borgloh, Johannes and Chakraborty, Saikat and Pfeffer, Stefan and Baumeister, Wolfgang and Lučić, Vladan},
  doi          = {10.1038/s41592-020-0763-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {240},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Template-free detection and classification of membrane-bound complexes in cryo-electron tomograms},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An antibody for analysis of autophagy induction.
<em>NMETH</em>, <em>17</em>(2), 232–239. (<a
href="https://doi.org/10.1038/s41592-019-0661-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autophagy is a degradative program that maintains cellular homeostasis. Autophagy defects have been described in numerous diseases. However, analysis of autophagy rates can be challenging, particularly in rare cell populations or in vivo, due to limitations in currently available tools for measuring autophagy induction. Here, we describe a method to monitor autophagy by measuring phosphorylation of the protein ATG16L1. We developed and characterized a monoclonal antibody that can detect phospho-ATG16L1 endogenously in mammalian cells. Importantly, phospho-ATG16L1 is only present on newly forming autophagosomes. Therefore, its levels are not affected by prolonged stress or late-stage autophagy blocks, which can confound autophagy analysis. Moreover, we show that ATG16L1 phosphorylation is a conserved signaling pathway activated by numerous autophagy-inducing stressors. The described antibody is suitable for western blot, immunofluorescence and immunohistochemistry, and measured phospho-ATG16L1 levels directly correspond to autophagy rates. Taken together, this phospho-antibody represents an exciting tool to study autophagy induction. A new method of autophagy measurement is based on the detection of phospho-ATG16L1, a conserved early marker of autophagy. Sensitive detection can be achieved in multiple biological systems and assays with advantages over standard methods.},
  archive      = {J_NMETH},
  author       = {Tian, Wensheng and Alsaadi, Reham and Guo, Zhihao and Kalinina, Alena and Carrier, Micaël and Tremblay, Marie-Eve and Lacoste, Baptiste and Lagace, Diane and Russell, Ryan C.},
  doi          = {10.1038/s41592-019-0661-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {232-239},
  shortjournal = {Nat. Meth.},
  title        = {An antibody for analysis of autophagy induction},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Nanoscale subcellular architecture revealed by multicolor
three-dimensional salvaged fluorescence imaging. <em>NMETH</em>,
<em>17</em>(2), 225–231. (<a
href="https://doi.org/10.1038/s41592-019-0676-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combining the molecular specificity of fluorescent probes with three-dimensional imaging at nanoscale resolution is critical for investigating the spatial organization and interactions of cellular organelles and protein complexes. We present a 4Pi single-molecule switching super-resolution microscope that enables ratiometric multicolor imaging of mammalian cells at 5–10-nm localization precision in three dimensions using ‘salvaged fluorescence’. Imaging two or three fluorophores simultaneously, we show fluorescence images that resolve the highly convoluted Golgi apparatus and the close contacts between the endoplasmic reticulum and the plasma membrane, structures that have traditionally been the imaging realm of electron microscopy. The salvaged fluorescence approach is equally applicable in most single-objective microscopes. 4Pi single-molecule switching microscopy combined with ‘salvaged fluorescence’ enables improved ratiometric imaging that bypasses chromatic aberrations and allows for multicolor whole-cell imaging with sub-10-nm localization precision.},
  archive      = {J_NMETH},
  author       = {Zhang, Yongdeng and Schroeder, Lena K. and Lessard, Mark D. and Kidd, Phylicia and Chung, Jeeyun and Song, Yuanbin and Benedetti, Lorena and Li, Yiming and Ries, Jonas and Grimm, Jonathan B. and Lavis, Luke D. and De Camilli, Pietro and Rothman, James E. and Baddeley, David and Bewersdorf, Joerg},
  doi          = {10.1038/s41592-019-0676-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {225-231},
  shortjournal = {Nat. Meth.},
  title        = {Nanoscale subcellular architecture revealed by multicolor three-dimensional salvaged fluorescence imaging},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). MINFLUX nanoscopy delivers 3D multicolor nanometer
resolution in cells. <em>NMETH</em>, <em>17</em>(2), 217–224. (<a
href="https://doi.org/10.1038/s41592-019-0688-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ultimate goal of biological super-resolution fluorescence microscopy is to provide three-dimensional resolution at the size scale of a fluorescent marker. Here we show that by localizing individual switchable fluorophores with a probing donut-shaped excitation beam, MINFLUX nanoscopy can provide resolutions in the range of 1 to 3 nm for structures in fixed and living cells. This progress has been facilitated by approaching each fluorophore iteratively with the probing-donut minimum, making the resolution essentially uniform and isotropic over scalable fields of view. MINFLUX imaging of nuclear pore complexes of a mammalian cell shows that this true nanometer-scale resolution is obtained in three dimensions and in two color channels. Relying on fewer detected photons than standard camera-based localization, MINFLUX nanoscopy is poised to open a new chapter in the imaging of protein complexes and distributions in fixed and living cells. Advances in MINFLUX nanoscopy enable multicolor imaging over large fields of view, bringing true nanometer-scale fluorescence imaging to labeled structures in fixed and living cells.},
  archive      = {J_NMETH},
  author       = {Gwosch, Klaus C. and Pape, Jasmin K. and Balzarotti, Francisco and Hoess, Philipp and Ellenberg, Jan and Ries, Jonas and Hell, Stefan W.},
  doi          = {10.1038/s41592-019-0688-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {217-224},
  shortjournal = {Nat. Meth.},
  title        = {MINFLUX nanoscopy delivers 3D multicolor nanometer resolution in cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Template-free detection and classification of
membrane-bound complexes in cryo-electron tomograms. <em>NMETH</em>,
<em>17</em>(2), 209–216. (<a
href="https://doi.org/10.1038/s41592-019-0675-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With faithful sample preservation and direct imaging of fully hydrated biological material, cryo-electron tomography provides an accurate representation of molecular architecture of cells. However, detection and precise localization of macromolecular complexes within cellular environments is aggravated by the presence of many molecular species and molecular crowding. We developed a template-free image processing procedure for accurate tracing of complex networks of densities in cryo-electron tomograms, a comprehensive and automated detection of heterogeneous membrane-bound complexes and an unsupervised classification (PySeg). Applications to intact cells and isolated endoplasmic reticulum (ER) allowed us to detect and classify small protein complexes. This classification provided sufficiently homogeneous particle sets and initial references to allow subsequent de novo subtomogram averaging. Spatial distribution analysis showed that ER complexes have different localization patterns forming nanodomains. Therefore, this procedure allows a comprehensive detection and structural analysis of complexes in situ. A template-free image processing approach automatically detects and classifies membrane-bound protein complexes in cryo-electron tomograms of isolated endoplasmic reticulum and in intact cells.},
  archive      = {J_NMETH},
  author       = {Martinez-Sanchez, Antonio and Kochovski, Zdravko and Laugks, Ulrike and Meyer zum Alten Borgloh, Johannes and Chakraborty, Saikat and Pfeffer, Stefan and Baumeister, Wolfgang and Lučić, Vladan},
  doi          = {10.1038/s41592-019-0675-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {209-216},
  shortjournal = {Nat. Meth.},
  title        = {Template-free detection and classification of membrane-bound complexes in cryo-electron tomograms},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). In situ structure determination at nanometer resolution
using TYGRESS. <em>NMETH</em>, <em>17</em>(2), 201–208. (<a
href="https://doi.org/10.1038/s41592-019-0651-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The resolution of subtomogram averages calculated from cryo-electron tomograms (cryo-ET) of crowded cellular environments is often limited owing to signal loss in, and misalignment of, the subtomograms. By contrast, single-particle cryo-electron microscopy (SP-cryo-EM) routinely reaches near-atomic resolution of isolated complexes. We report a method called ‘tomography-guided 3D reconstruction of subcellular structures’ (TYGRESS) that is a hybrid of cryo-ET and SP-cryo-EM, and is able to achieve close-to-nanometer resolution of complexes inside crowded cellular environments. TYGRESS combines the advantages of SP-cryo-EM (images with good signal-to-noise ratio and contrast, as well as minimal radiation damage) and subtomogram averaging (three-dimensional alignment of macromolecules in a complex sample). Using TYGRESS, we determined the structure of the intact ciliary axoneme with up to resolution of 12 Å. These results reveal many structural details that were not visible by cryo-ET alone. TYGRESS is generally applicable to cellular complexes that are amenable to subtomogram averaging. The unique advantages of single-particle cryo-electron microscopy and cryo-electron tomography are combined in a method called TYGRESS, here applied to determine the structure of the intact ciliary axoneme at a resolution of 12 Å.},
  archive      = {J_NMETH},
  author       = {Song, Kangkang and Shang, Zhiguo and Fu, Xiaofeng and Lou, Xiaochu and Grigorieff, Nikolaus and Nicastro, Daniela},
  doi          = {10.1038/s41592-019-0651-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {201-208},
  shortjournal = {Nat. Meth.},
  title        = {In situ structure determination at nanometer resolution using TYGRESS},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Statistical analysis of spatial expression patterns for
spatially resolved transcriptomic studies. <em>NMETH</em>,
<em>17</em>(2), 193–200. (<a
href="https://doi.org/10.1038/s41592-019-0701-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying genes that display spatial expression patterns in spatially resolved transcriptomic studies is an important first step toward characterizing the spatial transcriptomic landscape of complex tissues. Here we present a statistical method, SPARK, for identifying spatial expression patterns of genes in data generated from various spatially resolved transcriptomic techniques. SPARK directly models spatial count data through generalized linear spatial models. It relies on recently developed statistical formulas for hypothesis testing, providing effective control of type I errors and yielding high statistical power. With a computationally efficient algorithm, which is based on penalized quasi-likelihood, SPARK is also scalable to datasets with tens of thousands of genes measured on tens of thousands of samples. Analyzing four published spatially resolved transcriptomic datasets using SPARK, we show it can be up to ten times more powerful than existing methods and disclose biological discoveries that otherwise cannot be revealed by existing approaches. A statistical method called SPARK for analyzing spatially resolved transcriptomic data can efficiently identify spatially expressed genes with effective control of type I errors and high statistical power.},
  archive      = {J_NMETH},
  author       = {Sun, Shiquan and Zhu, Jiaqiang and Zhou, Xiang},
  doi          = {10.1038/s41592-019-0701-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {193-200},
  shortjournal = {Nat. Meth.},
  title        = {Statistical analysis of spatial expression patterns for spatially resolved transcriptomic studies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deciphering interaction fingerprints from protein molecular
surfaces using geometric deep learning. <em>NMETH</em>, <em>17</em>(2),
184–192. (<a href="https://doi.org/10.1038/s41592-019-0666-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting interactions between proteins and other biomolecules solely based on structure remains a challenge in biology. A high-level representation of protein structure, the molecular surface, displays patterns of chemical and geometric features that fingerprint a protein’s modes of interactions with other biomolecules. We hypothesize that proteins participating in similar interactions may share common fingerprints, independent of their evolutionary history. Fingerprints may be difficult to grasp by visual analysis but could be learned from large-scale datasets. We present MaSIF (molecular surface interaction fingerprinting), a conceptual framework based on a geometric deep learning method to capture fingerprints that are important for specific biomolecular interactions. We showcase MaSIF with three prediction challenges: protein pocket-ligand prediction, protein–protein interaction site prediction and ultrafast scanning of protein surfaces for prediction of protein–protein complexes. We anticipate that our conceptual framework will lead to improvements in our understanding of protein function and design. MaSIF, a deep learning-based method, finds common patterns of chemical and geometric features on biomolecular surfaces for predicting protein–ligand and protein–protein interactions.},
  archive      = {J_NMETH},
  author       = {Gainza, P. and Sverrisson, F. and Monti, F. and Rodolà, E. and Boscaini, D. and Bronstein, M. M. and Correia, B. E.},
  doi          = {10.1038/s41592-019-0666-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {184-192},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering interaction fingerprints from protein molecular surfaces using geometric deep learning},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Biophysical prediction of protein–peptide interactions and
signaling networks using machine learning. <em>NMETH</em>,
<em>17</em>(2), 175–183. (<a
href="https://doi.org/10.1038/s41592-019-0687-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In mammalian cells, much of signal transduction is mediated by weak protein–protein interactions between globular peptide-binding domains (PBDs) and unstructured peptidic motifs in partner proteins. The number and diversity of these PBDs (over 1,800 are known), their low binding affinities and the sensitivity of binding properties to minor sequence variation represent a substantial challenge to experimental and computational analysis of PBD specificity and the networks PBDs create. Here, we introduce a bespoke machine-learning approach, hierarchical statistical mechanical modeling (HSM), capable of accurately predicting the affinities of PBD–peptide interactions across multiple protein families. By synthesizing biophysical priors within a modern machine-learning framework, HSM outperforms existing computational methods and high-throughput experimental assays. HSM models are interpretable in familiar biophysical terms at three spatial scales: the energetics of protein–peptide binding, the multidentate organization of protein–protein interactions and the global architecture of signaling networks. Protein–peptide interactions that underpin cell signaling are accurately predicted by wedding the strengths of machine learning with the interpretability of biophysical theory, facilitating detailed mechanistic analyses at the proteome scale.},
  archive      = {J_NMETH},
  author       = {Cunningham, Joseph M. and Koytiger, Grigoriy and Sorger, Peter K. and AlQuraishi, Mohammed},
  doi          = {10.1038/s41592-019-0687-1},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {175-183},
  shortjournal = {Nat. Meth.},
  title        = {Biophysical prediction of protein–peptide interactions and signaling networks using machine learning},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Directed evolution improves the catalytic efficiency of TEV
protease. <em>NMETH</em>, <em>17</em>(2), 167–174. (<a
href="https://doi.org/10.1038/s41592-019-0665-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tobacco etch virus protease (TEV) is one of the most widely used proteases in biotechnology because of its exquisite sequence specificity. A limitation, however, is its slow catalytic rate. We developed a generalizable yeast-based platform for directed evolution of protease catalytic properties. Protease activity is read out via proteolytic release of a membrane-anchored transcription factor, and we temporally regulate access to TEV’s cleavage substrate using a photosensory LOV domain. By gradually decreasing light exposure time, we enriched faster variants of TEV over multiple rounds of selection. Our TEV-S153N mutant (uTEV1Δ), when incorporated into the calcium integrator FLARE, improved the signal/background ratio by 27-fold, and enabled recording of neuronal activity in culture with 60-s temporal resolution. Given the widespread use of TEV in biotechnology, both our evolved TEV mutants and the directed-evolution platform used to generate them could be beneficial across a wide range of applications. An optogenetic strategy enables selection of proteases with improved catalytic rates. The developed TEV protease variants are well suited for biotechnology applications, including FLARE assays with substantially improved temporal resolution.},
  archive      = {J_NMETH},
  author       = {Sanchez, Mateo I. and Ting, Alice Y.},
  doi          = {10.1038/s41592-019-0665-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {167-174},
  shortjournal = {Nat. Meth.},
  title        = {Directed evolution improves the catalytic efficiency of TEV protease},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An adaptive excitation source for high-speed multiphoton
microscopy. <em>NMETH</em>, <em>17</em>(2), 163–166. (<a
href="https://doi.org/10.1038/s41592-019-0663-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical imaging is important for understanding brain function. However, established methods with high spatiotemporal resolution are limited by the potential for laser damage to living tissues. We describe an adaptive femtosecond excitation source that only illuminates the region of interest, which leads to a 30-fold reduction in the power requirement for two- or three-photon imaging of brain activity in awake mice for improved high-speed longitudinal neuroimaging. An adaptive excitation source enables two- and three-photon imaging of the awake mouse brain with high spatial and temporal resolution at 30-fold-reduced laser power relative to conventional approaches.},
  archive      = {J_NMETH},
  author       = {Li, Bo and Wu, Chunyan and Wang, Mengran and Charan, Kriti and Xu, Chris},
  doi          = {10.1038/s41592-019-0663-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {163-166},
  shortjournal = {Nat. Meth.},
  title        = {An adaptive excitation source for high-speed multiphoton microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). NicheNet: Modeling intercellular communication by linking
ligands to target genes. <em>NMETH</em>, <em>17</em>(2), 159–162. (<a
href="https://doi.org/10.1038/s41592-019-0667-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational methods that model how gene expression of a cell is influenced by interacting cells are lacking. We present NicheNet ( https://github.com/saeyslab/nichenetr ), a method that predicts ligand–target links between interacting cells by combining their expression data with prior knowledge on signaling and gene regulatory networks. We applied NicheNet to tumor and immune cell microenvironment data and demonstrate that NicheNet can infer active ligands and their gene regulatory effects on interacting cells. NicheNet uses expression data, in combination with a previous model built on known signaling and gene regulatory networks, to predict ligand–target links in cell-to-cell communications.},
  archive      = {J_NMETH},
  author       = {Browaeys, Robin and Saelens, Wouter and Saeys, Yvan},
  doi          = {10.1038/s41592-019-0667-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {159-162},
  shortjournal = {Nat. Meth.},
  title        = {NicheNet: Modeling intercellular communication by linking ligands to target genes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Fast and accurate long-read assembly with wtdbg2.
<em>NMETH</em>, <em>17</em>(2), 155–158. (<a
href="https://doi.org/10.1038/s41592-019-0669-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing long-read assemblers require thousands of central processing unit hours to assemble a human genome and are being outpaced by sequencing technologies in terms of both throughput and cost. We developed a long-read assembler wtdbg2 (https://github.com/ruanjue/wtdbg2) that is 2–17 times as fast as published tools while achieving comparable contiguity and accuracy. It paves the way for population-scale long-read assembly in future. Wtdbg2 assembles genomes with comparable contiguity and accuracy to existing tools using long-read sequencing data, and is several times faster, especially for large genomes.},
  archive      = {J_NMETH},
  author       = {Ruan, Jue and Li, Heng},
  doi          = {10.1038/s41592-019-0669-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {155-158},
  shortjournal = {Nat. Meth.},
  title        = {Fast and accurate long-read assembly with wtdbg2},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Benchmarking algorithms for gene regulatory network
inference from single-cell transcriptomic data. <em>NMETH</em>,
<em>17</em>(2), 147–154. (<a
href="https://doi.org/10.1038/s41592-019-0690-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a systematic evaluation of state-of-the-art algorithms for inferring gene regulatory networks from single-cell transcriptional data. As the ground truth for assessing accuracy, we use synthetic networks with predictable trajectories, literature-curated Boolean models and diverse transcriptional regulatory networks. We develop a strategy to simulate single-cell transcriptional data from synthetic and Boolean networks that avoids pitfalls of previously used methods. Furthermore, we collect networks from multiple experimental single-cell RNA-seq datasets. We develop an evaluation framework called BEELINE. We find that the area under the precision-recall curve and early precision of the algorithms are moderate. The methods are better in recovering interactions in synthetic networks than Boolean models. The algorithms with the best early precision values for Boolean models also perform well on experimental datasets. Techniques that do not require pseudotime-ordered cells are generally more accurate. Based on these results, we present recommendations to end users. BEELINE will aid the development of gene regulatory network inference algorithms. Comprehensive evaluation of algorithms for inferring gene regulatory networks using synthetic and experimental single-cell RNA-seq datasets finds heterogeneous performance and suggests recommendations to users.},
  archive      = {J_NMETH},
  author       = {Pratapa, Aditya and Jalihal, Amogh P. and Law, Jeffrey N. and Bharadwaj, Aditya and Murali, T. M.},
  doi          = {10.1038/s41592-019-0690-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {147-154},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Orchestrating single-cell analysis with bioconductor.
<em>NMETH</em>, <em>17</em>(2), 137–145. (<a
href="https://doi.org/10.1038/s41592-019-0654-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advancements have enabled the profiling of a large number of genome-wide features in individual cells. However, single-cell data present unique challenges that require the development of specialized methods and software infrastructure to successfully derive biological insights. The Bioconductor project has rapidly grown to meet these demands, hosting community-developed open-source software distributed as R packages. Featuring state-of-the-art computational methods, standardized data infrastructure and interactive data visualization tools, we present an overview and online book ( https://osca.bioconductor.org ) of single-cell methods for prospective users. This Perspective highlights open-source software for single-cell analysis released as part of the Bioconductor project, providing an overview for users and developers.},
  archive      = {J_NMETH},
  author       = {Amezquita, Robert A. and Lun, Aaron T. L. and Becht, Etienne and Carey, Vince J. and Carpp, Lindsay N. and Geistlinger, Ludwig and Marini, Federico and Rue-Albrecht, Kevin and Risso, Davide and Soneson, Charlotte and Waldron, Levi and Pagès, Hervé and Smith, Mike L. and Huber, Wolfgang and Morgan, Martin and Gottardo, Raphael and Hicks, Stephanie C.},
  doi          = {10.1038/s41592-019-0654-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {137-145},
  shortjournal = {Nat. Meth.},
  title        = {Orchestrating single-cell analysis with bioconductor},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020w). When microbiologists plunge into the ocean. <em>NMETH</em>,
<em>17</em>(2), 133–136. (<a
href="https://doi.org/10.1038/s41592-020-0736-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbiologists stake their claim: assessing climate change involves new ways of studying the ocean’s microbes.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0736-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {133-136},
  shortjournal = {Nat. Meth.},
  title        = {When microbiologists plunge into the ocean},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020o). Isolating neurons on the basis of function. <em>NMETH</em>,
<em>17</em>(2), 131. (<a
href="https://doi.org/10.1038/s41592-020-0744-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers have developed a strategy to collect and profile specific neuronal cell types on the basis of their physiological activity.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0744-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {131},
  shortjournal = {Nat. Meth.},
  title        = {Isolating neurons on the basis of function},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Asian genomes. <em>NMETH</em>, <em>17</em>(2), 130. (<a
href="https://doi.org/10.1038/s41592-020-0753-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0753-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {130},
  shortjournal = {Nat. Meth.},
  title        = {Asian genomes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020y). Oceans of genomes. <em>NMETH</em>, <em>17</em>(2), 130. (<a
href="https://doi.org/10.1038/s41592-020-0754-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0754-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {130},
  shortjournal = {Nat. Meth.},
  title        = {Oceans of genomes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020p). Reprogramming allosteric control in proteins.
<em>NMETH</em>, <em>17</em>(2), 130. (<a
href="https://doi.org/10.1038/s41592-020-0755-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0755-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {130},
  shortjournal = {Nat. Meth.},
  title        = {Reprogramming allosteric control in proteins},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). Encoding the glow. <em>NMETH</em>, <em>17</em>(2), 130. (<a
href="https://doi.org/10.1038/s41592-020-0756-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-020-0756-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {130},
  shortjournal = {Nat. Meth.},
  title        = {Encoding the glow},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ah). Sequencing BCR–antigen interactions. <em>NMETH</em>,
<em>17</em>(2), 129. (<a
href="https://doi.org/10.1038/s41592-020-0749-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0749-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {129},
  shortjournal = {Nat. Meth.},
  title        = {Sequencing BCR–antigen interactions},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020n). High-dimensional data visualization. <em>NMETH</em>,
<em>17</em>(2), 129. (<a
href="https://doi.org/10.1038/s41592-020-0750-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0750-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {129},
  shortjournal = {Nat. Meth.},
  title        = {High-dimensional data visualization},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A brain observatory. <em>NMETH</em>, <em>17</em>(2), 129.
(<a href="https://doi.org/10.1038/s41592-020-0751-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0751-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {129},
  shortjournal = {Nat. Meth.},
  title        = {A brain observatory},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020n). MS3-based cross-link search platform. <em>NMETH</em>,
<em>17</em>(2), 129. (<a
href="https://doi.org/10.1038/s41592-020-0752-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0752-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {129},
  shortjournal = {Nat. Meth.},
  title        = {MS3-based cross-link search platform},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020m). Mass spectrometry search tool (MASST). <em>NMETH</em>,
<em>17</em>(2), 128. (<a
href="https://doi.org/10.1038/s41592-020-0743-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Searching mass spectra against publicly available tandem MS databases just got easier.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-020-0743-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {128},
  shortjournal = {Nat. Meth.},
  title        = {Mass spectrometry search tool (MASST)},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). FAIR your data. <em>NMETH</em>, <em>17</em>(2), 127. (<a
href="https://doi.org/10.1038/s41592-020-0742-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A toolkit facilitating FAIRness assessment.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-020-0742-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {127},
  shortjournal = {Nat. Meth.},
  title        = {FAIR your data},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020ag). Sensing proteinogenic amino acids. <em>NMETH</em>,
<em>17</em>(2), 126. (<a
href="https://doi.org/10.1038/s41592-020-0741-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aerolysin nanopores offer a way to identify natural amino acids and their chemical modifications.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-020-0741-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {126},
  shortjournal = {Nat. Meth.},
  title        = {Sensing proteinogenic amino acids},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). Human embryogenesis in a dish. <em>NMETH</em>,
<em>17</em>(2), 125. (<a
href="https://doi.org/10.1038/s41592-020-0740-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A 3D culture system recapitulates human development prior to gastrulation and enables transcriptomic studies.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-020-0740-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {125},
  shortjournal = {Nat. Meth.},
  title        = {Human embryogenesis in a dish},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Galaxy external display applications: Closing a dataflow
interoperability loop. <em>NMETH</em>, <em>17</em>(2), 123–124. (<a
href="https://doi.org/10.1038/s41592-019-0727-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Blankenberg, Daniel and Chilton, John and Coraor, Nate},
  doi          = {10.1038/s41592-019-0727-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {123-124},
  shortjournal = {Nat. Meth.},
  title        = {Galaxy external display applications: Closing a dataflow interoperability loop},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Markov models — training and evaluation of hidden markov
models. <em>NMETH</em>, <em>17</em>(2), 121–122. (<a
href="https://doi.org/10.1038/s41592-019-0702-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {“With one eye you are looking at the outside world, while with the other you are looking within yourself.” —Amedeo Modigliani},
  archive      = {J_NMETH},
  author       = {Grewal, Jasleen K. and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-019-0702-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {121-122},
  shortjournal = {Nat. Meth.},
  title        = {Markov models — training and evaluation of hidden markov models},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Community reflections. <em>NMETH</em>, <em>17</em>(2), 117.
(<a href="https://doi.org/10.1038/s41592-020-0757-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This month, we take a look back on the first year of the online Springer Nature Protocols and Methods Community.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-020-0757-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {117},
  shortjournal = {Nat. Meth.},
  title        = {Community reflections},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020x). Xiang zhou. <em>NMETH</em>, <em>17</em>(2), 119. (<a
href="https://doi.org/10.1038/s41592-020-0738-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A love of math and biology comes together with a SPARK.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-020-0738-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {2},
  pages        = {119},
  shortjournal = {Nat. Meth.},
  title        = {Xiang zhou},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Publisher correction: Analysis of the human protein atlas
image classification competition. <em>NMETH</em>, <em>17</em>(1), 115.
(<a href="https://doi.org/10.1038/s41592-019-0699-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Ouyang, Wei and Winsnes, Casper F. and Hjelmare, Martin and Cesnik, Anthony J. and Åkesson, Lovisa and Xu, Hao and Sullivan, Devin P. and Dai, Shubin and Lan, Jun and Jinmo, Park and Galib, Shaikat M. and Henkel, Christof and Hwang, Kevin and Poplavskiy, Dmytro and Tunguz, Bojan and Wolfinger, Russel D. and Gu, Yinzheng and Li, Chuanpeng and Xie, Jinbin and Buslov, Dmitry and Fironov, Sergei and Kiselev, Alexander and Panchenko, Dmytro and Cao, Xuan and Wei, Runmin and Wu, Yuanhao and Zhu, Xun and Tseng, Kuan-Lun and Gao, Zhifeng and Ju, Cheng and Yi, Xiaohan and Zheng, Hongdong and Kappel, Constantin and Lundberg, Emma},
  doi          = {10.1038/s41592-019-0699-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {115},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Analysis of the human protein atlas image classification competition},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020s). Publisher correction: The author file: Sündüz keleş.
<em>NMETH</em>, <em>17</em>(1), 115. (<a
href="https://doi.org/10.1038/s41592-019-0725-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-019-0725-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {115},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: the author file: sündüz keleş},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Gas cluster ion beam SEM for imaging
of large tissue samples with 10 nm isotropic resolution. <em>NMETH</em>,
<em>17</em>(1), 114. (<a
href="https://doi.org/10.1038/s41592-019-0695-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Hayworth, Kenneth J. and Peale, David and Januszewski, Michał and Knott, Graham W. and Lu, Zhiyuan and Xu, C. Shan and Hess, Harald F.},
  doi          = {10.1038/s41592-019-0695-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {114},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Gas cluster ion beam SEM for imaging of large tissue samples with 10 nm isotropic resolution},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Nanopore native RNA sequencing of a human
poly(a) transcriptome. <em>NMETH</em>, <em>17</em>(1), 114. (<a
href="https://doi.org/10.1038/s41592-019-0697-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NMETH},
  author       = {Workman, Rachael E. and Tang, Alison D. and Tang, Paul S. and Jain, Miten and Tyson, John R. and Razaghi, Roham and Zuzarte, Philip C. and Gilpatrick, Timothy and Payne, Alexander and Quick, Joshua and Sadowski, Norah and Holmes, Nadine and de Jesus, Jaqueline Goes and Jones, Karen L. and Soulette, Cameron M. and Snutch, Terrance P. and Loman, Nicholas and Paten, Benedict and Loose, Matthew and Simpson, Jared T. and Olsen, Hugh E. and Brooks, Angela N. and Akeson, Mark and Timp, Winston},
  doi          = {10.1038/s41592-019-0697-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {114},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Nanopore native RNA sequencing of a human poly(A) transcriptome},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Simultaneous mesoscopic and two-photon imaging of neuronal
activity in cortical circuits. <em>NMETH</em>, <em>17</em>(1), 107–113.
(<a href="https://doi.org/10.1038/s41592-019-0625-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spontaneous and sensory-evoked activity propagates across varying spatial scales in the mammalian cortex, but technical challenges have limited conceptual links between the function of local neuronal circuits and brain-wide network dynamics. We present a method for simultaneous cellular-resolution two-photon calcium imaging of a local microcircuit and mesoscopic widefield calcium imaging of the entire cortical mantle in awake mice. Our multi-scale approach involves a microscope with an orthogonal axis design where the mesoscopic objective is oriented above the brain and the two-photon objective is oriented horizontally, with imaging performed through a microprism. We also introduce a viral transduction method for robust and widespread gene delivery in the mouse brain. These approaches allow us to identify the behavioral state-dependent functional connectivity of pyramidal neurons and vasoactive intestinal peptide-expressing interneurons with long-range cortical networks. Our imaging system provides a powerful strategy for investigating cortical architecture across a wide range of spatial scales. Simultaneous two-photon microscopic and one-photon mesoscopic imaging of calcium signals enables correlation of local cellular and brain-wide network activity.},
  archive      = {J_NMETH},
  author       = {Barson, Daniel and Hamodi, Ali S. and Shen, Xilin and Lur, Gyorgy and Constable, R. Todd and Cardin, Jessica A. and Crair, Michael C. and Higley, Michael J.},
  doi          = {10.1038/s41592-019-0625-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {107-113},
  shortjournal = {Nat. Meth.},
  title        = {Simultaneous mesoscopic and two-photon imaging of neuronal activity in cortical circuits},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Probabilistic cell typing enables fine mapping of closely
related cell types in situ. <em>NMETH</em>, <em>17</em>(1), 101–106. (<a
href="https://doi.org/10.1038/s41592-019-0631-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the function of a tissue requires knowing the spatial organization of its constituent cell types. In the cerebral cortex, single-cell RNA sequencing (scRNA-seq) has revealed the genome-wide expression patterns that define its many, closely related neuronal types, but cannot reveal their spatial arrangement. Here we introduce probabilistic cell typing by in situ sequencing (pciSeq), an approach that leverages previous scRNA-seq classification to identify cell types using multiplexed in situ RNA detection. We applied this method by mapping the inhibitory neurons of mouse hippocampal area CA1, for which ground truth is available from extensive previous work identifying their laminar organization. Our method identified these neuronal classes in a spatial arrangement matching ground truth, and further identified multiple classes of isocortical pyramidal cell in a pattern matching their known organization. This method will allow identifying the spatial organization of closely related cell types across the brain and other tissues. Probabilistic cell typing by in situ sequencing (pciSeq), leverages previous single-cell RNA sequencing classification and multiplexed in situ RNA detection to spatially map cell types accurately in the mouse hippocampus and isocortex.},
  archive      = {J_NMETH},
  author       = {Qian, Xiaoyan and Harris, Kenneth D. and Hauling, Thomas and Nicoloutsopoulos, Dimitris and Muñoz-Manchado, Ana B. and Skene, Nathan and Hjerling-Leffler, Jens and Nilsson, Mats},
  doi          = {10.1038/s41592-019-0631-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {101-106},
  shortjournal = {Nat. Meth.},
  title        = {Probabilistic cell typing enables fine mapping of closely related cell types in situ},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Isolating live cells after high-throughput, long-term,
time-lapse microscopy. <em>NMETH</em>, <em>17</em>(1), 93–100. (<a
href="https://doi.org/10.1038/s41592-019-0620-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell genetic screens can be incredibly powerful, but current high-throughput platforms do not track dynamic processes, and even for non-dynamic properties they struggle to separate mutants of interest from phenotypic outliers of the wild-type population. Here we introduce SIFT, single-cell isolation following time-lapse imaging, to address these limitations. After imaging and tracking individual bacteria for tens of consecutive generations under tightly controlled growth conditions, cells of interest are isolated and propagated for downstream analysis, free of contamination and without genetic or physiological perturbations. This platform can characterize tens of thousands of cell lineages per day, making it possible to accurately screen complex phenotypes without the need for barcoding or genetic modifications. We applied SIFT to identify a set of ultraprecise synthetic gene oscillators, with circuit variants spanning a 30-fold range of average periods. This revealed novel design principles in synthetic biology and demonstrated the power of SIFT to reliably screen diverse dynamic phenotypes. Single-cell isolation following time-lapse imaging (SIFT) enables high-throughput screening of complex and dynamic phenotypes from pooled bacterial libraries. SIFT was used to generate ultraprecise synthetic gene oscillators.},
  archive      = {J_NMETH},
  author       = {Luro, Scott and Potvin-Trottier, Laurent and Okumus, Burak and Paulsson, Johan},
  doi          = {10.1038/s41592-019-0620-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {93-100},
  shortjournal = {Nat. Meth.},
  title        = {Isolating live cells after high-throughput, long-term, time-lapse microscopy},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Time-resolved imaging-based CRISPRi screening.
<em>NMETH</em>, <em>17</em>(1), 86–92. (<a
href="https://doi.org/10.1038/s41592-019-0629-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our ability to connect genotypic variation to biologically important phenotypes has been seriously limited by the gap between live-cell microscopy and library-scale genomic engineering. Here, we show how in situ genotyping of a library of strains after time-lapse imaging in a microfluidic device overcomes this problem. We determine how 235 different CRISPR interference knockdowns impact the coordination of the replication and division cycles of Escherichia coli by monitoring the location of replication forks throughout on average &amp;gt;500 cell cycles per knockdown. Subsequent in situ genotyping allows us to map each phenotype distribution to a specific genetic perturbation to determine which genes are important for cell cycle control. The single-cell time-resolved assay allows us to determine the distribution of single-cell growth rates, cell division sizes and replication initiation volumes. The technology presented in this study enables genome-scale screens of most live-cell microscopy assays. DuMPLING (dynamic μ-fluidic microscopy phenotyping of a library before in situ genotyping) enables screening of dynamic phenotypes in strain libraries and was used here to study genes that coordinate replication and cell division in Escherichia coli.},
  archive      = {J_NMETH},
  author       = {Camsund, Daniel and Lawson, Michael J. and Larsson, Jimmy and Jones, Daniel and Zikrin, Spartak and Fange, David and Elf, Johan},
  doi          = {10.1038/s41592-019-0629-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {86-92},
  shortjournal = {Nat. Meth.},
  title        = {Time-resolved imaging-based CRISPRi screening},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Bottom-up structural proteomics: CryoEM of protein complexes
enriched from the cellular milieu. <em>NMETH</em>, <em>17</em>(1),
79–85. (<a href="https://doi.org/10.1038/s41592-019-0637-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {X-ray crystallography often requires non-native constructs involving mutations or truncations, and is challenged by membrane proteins and large multicomponent complexes. We present here a bottom-up endogenous structural proteomics approach whereby near-atomic-resolution cryo electron microscopy (cryoEM) maps are reconstructed ab initio from unidentified protein complexes enriched directly from the endogenous cellular milieu, followed by identification and atomic modeling of the proteins. The proteins in each complex are identified using cryoID, a program we developed to identify proteins in ab initio cryoEM maps. As a proof of principle, we applied this approach to the malaria-causing parasite Plasmodium falciparum, an organism that has resisted conventional structural-biology approaches, to obtain atomic models of multiple protein complexes implicated in intraerythrocytic survival of the parasite. Our approach is broadly applicable for determining structures of undiscovered protein complexes enriched directly from endogenous sources. An approach combining cryo-electron microscopy and mass spectrometry analysis of protein complexes enriched directly from cells enables structure determination of unknown complexes at atomic resolution.},
  archive      = {J_NMETH},
  author       = {Ho, Chi-Min and Li, Xiaorun and Lai, Mason and Terwilliger, Thomas C. and Beck, Josh R. and Wohlschlegel, James and Goldberg, Daniel E. and Fitzpatrick, Anthony W. P. and Zhou, Z. Hong},
  doi          = {10.1038/s41592-019-0637-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {79-85},
  shortjournal = {Nat. Meth.},
  title        = {Bottom-up structural proteomics: CryoEM of protein complexes enriched from the cellular milieu},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Time-resolved serial femtosecond crystallography at the
european XFEL. <em>NMETH</em>, <em>17</em>(1), 73–78. (<a
href="https://doi.org/10.1038/s41592-019-0628-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The European XFEL (EuXFEL) is a 3.4-km long X-ray source, which produces femtosecond, ultrabrilliant and spatially coherent X-ray pulses at megahertz (MHz) repetition rates. This X-ray source has been designed to enable the observation of ultrafast processes with near-atomic spatial resolution. Time-resolved crystallographic investigations on biological macromolecules belong to an important class of experiments that explore fundamental and functional structural displacements in these molecules. Due to the unusual MHz X-ray pulse structure at the EuXFEL, these experiments are challenging. Here, we demonstrate how a biological reaction can be followed on ultrafast timescales at the EuXFEL. We investigate the picosecond time range in the photocycle of photoactive yellow protein (PYP) with MHz X-ray pulse rates. We show that difference electron density maps of excellent quality can be obtained. The results connect the previously explored femtosecond PYP dynamics to timescales accessible at synchrotrons. This opens the door to a wide range of time-resolved studies at the EuXFEL. European XFEL allows investigation of the picosecond time range in the photocycle of photoactive yellow protein.},
  archive      = {J_NMETH},
  author       = {Pandey, Suraj and Bean, Richard and Sato, Tokushi and Poudyal, Ishwor and Bielecki, Johan and Cruz Villarreal, Jorvani and Yefanov, Oleksandr and Mariani, Valerio and White, Thomas A. and Kupitz, Christopher and Hunter, Mark and Abdellatif, Mohamed H. and Bajt, Saša and Bondar, Valerii and Echelmeier, Austin and Doppler, Diandra and Emons, Moritz and Frank, Matthias and Fromme, Raimund and Gevorkov, Yaroslav and Giovanetti, Gabriele and Jiang, Man and Kim, Daihyun and Kim, Yoonhee and Kirkwood, Henry and Klimovskaia, Anna and Knoska, Juraj and Koua, Faisal H. M. and Letrun, Romain and Lisova, Stella and Maia, Luis and Mazalova, Victoria and Meza, Domingo and Michelat, Thomas and Ourmazd, Abbas and Palmer, Guido and Ramilli, Marco and Schubert, Robin and Schwander, Peter and Silenzi, Alessandro and Sztuk-Dambietz, Jolanta and Tolstikova, Alexandra and Chapman, Henry N. and Ros, Alexandra and Barty, Anton and Fromme, Petra and Mancuso, Adrian P. and Schmidt, Marius},
  doi          = {10.1038/s41592-019-0628-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {73-78},
  shortjournal = {Nat. Meth.},
  title        = {Time-resolved serial femtosecond crystallography at the european XFEL},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Gas cluster ion beam SEM for imaging of large tissue
samples with 10 nm isotropic resolution. <em>NMETH</em>, <em>17</em>(1),
68–71. (<a href="https://doi.org/10.1038/s41592-019-0641-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We demonstrate gas cluster ion beam scanning electron microscopy (SEM), in which wide-area ion milling is performed on a series of thick tissue sections. This three-dimensional electron microscopy technique acquires datasets with &amp;lt;10 nm isotropic resolution of each section, and these can then be stitched together to span the sectioned volume. Incorporating gas cluster ion beam SEM into existing single-beam and multibeam SEM workflows should be straightforward, increasing reliability while improving z resolution by a factor of three or more. An alternative to focused ion beam scanning electron microscopy (FIB-SEM), gas cluster ion beam scanning electron microscopy (GCIB-SEM) is compatible with large tissue samples while achieving similar isotropic resolution.},
  archive      = {J_NMETH},
  author       = {Hayworth, Kenneth J. and Peale, David and Januszewski, Michał and Knott, Graham W. and Lu, Zhiyuan and Xu, C. Shan and Hess, Harald F.},
  doi          = {10.1038/s41592-019-0641-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {68-71},
  shortjournal = {Nat. Meth.},
  title        = {Gas cluster ion beam SEM for imaging of large tissue samples with 10 nm isotropic resolution},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A CMOS NMR needle for probing brain physiology with high
spatial and temporal resolution. <em>NMETH</em>, <em>17</em>(1), 64–67.
(<a href="https://doi.org/10.1038/s41592-019-0640-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Magnetic resonance imaging and spectroscopy are versatile methods for probing brain physiology, but their intrinsically low sensitivity limits the achievable spatial and temporal resolution. Here, we introduce a monolithically integrated NMR-on-a-chip needle that combines an ultra-sensitive 300 µm NMR coil with a complete NMR transceiver, enabling in vivo measurements of blood oxygenation and flow in nanoliter volumes at a sampling rate of 200 Hz. A miniaturized NMR-on-a-chip needle can be implanted into rodent brains and can measure blood flow and oxygenation changes in vivo in a small volume at an unprecedentedly high temporal resolution of a few milliseconds.},
  archive      = {J_NMETH},
  author       = {Handwerker, Jonas and Pérez-Rodas, Marlon and Beyerlein, Michael and Vincent, Franck and Beck, Armin and Freytag, Nicolas and Yu, Xin and Pohmann, Rolf and Anders, Jens and Scheffler, Klaus},
  doi          = {10.1038/s41592-019-0640-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {64-67},
  shortjournal = {Nat. Meth.},
  title        = {A CMOS NMR needle for probing brain physiology with high spatial and temporal resolution},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Localization microscopy at doubled precision with patterned
illumination. <em>NMETH</em>, <em>17</em>(1), 59–63. (<a
href="https://doi.org/10.1038/s41592-019-0657-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MINFLUX offers a breakthrough in single molecule localization precision, but is limited in field of view. Here we combine centroid estimation and illumination pattern induced photon count variations in a conventional widefield imaging setup to extract position information over a typical micrometer-sized field of view. We show a near two-fold improvement in precision over standard localization with the same photon count on DNA-origami nanostructures and tubulin in cells, using DNA-PAINT and STORM imaging. SIMFLUX combines elements of MINFLUX with structured illumination to double localization precision and improve resolution in localization microscopy. The approach was demonstrated on DNA origami and on cellular microtubules.},
  archive      = {J_NMETH},
  author       = {Cnossen, Jelmer and Hinsdale, Taylor and Thorsen, Rasmus Ø. and Siemons, Marijn and Schueder, Florian and Jungmann, Ralf and Smith, Carlas S. and Rieger, Bernd and Stallinga, Sjoerd},
  doi          = {10.1038/s41592-019-0657-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {59-63},
  shortjournal = {Nat. Meth.},
  title        = {Localization microscopy at doubled precision with patterned illumination},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). MEosEM withstands osmium staining and epon embedding for
super-resolution CLEM. <em>NMETH</em>, <em>17</em>(1), 55–58. (<a
href="https://doi.org/10.1038/s41592-019-0613-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Super-resolution correlative light and electron microscopy (SR-CLEM) is a powerful approach for imaging specific molecules at the nanoscale in the context of the cellular ultrastructure. Epon epoxy resin embedding offers advantages for SR-CLEM, including ultrastructural preservation and high quality sectioning. However, Epon embedding eliminates fluorescence from most fluorescent proteins. We describe a photocontrollable fluorescent protein, mEosEM, that can survive Epon embedding after osmium tetroxide (OsO4) treatment for improved SR-CLEM. mEosEM survives 1\% OsO4 treatment and Epon embedding for improved super-resolution correlative light and electron microscopy.},
  archive      = {J_NMETH},
  author       = {Fu, Zhifei and Peng, Dingming and Zhang, Mingshu and Xue, Fudong and Zhang, Rui and He, Wenting and Xu, Tao and Xu, Pingyong},
  doi          = {10.1038/s41592-019-0613-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {55-58},
  shortjournal = {Nat. Meth.},
  title        = {MEosEM withstands osmium staining and epon embedding for super-resolution CLEM},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Tailoring cryo-electron microscopy grids by
photo-micropatterning for in-cell structural studies. <em>NMETH</em>,
<em>17</em>(1), 50–54. (<a
href="https://doi.org/10.1038/s41592-019-0630-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially controlled cell adhesion on electron microscopy supports remains a bottleneck in specimen preparation for cellular cryo-electron tomography. Here, we describe contactless and mask-free photo-micropatterning of electron microscopy grids for site-specific deposition of extracellular matrix-related proteins. We attained refined cell positioning for micromachining by cryo-focused ion beam milling. Complex micropatterns generated predictable intracellular organization, allowing direct correlation between cell architecture and in-cell three-dimensional structural characterization of the underlying molecular machinery. Micropatterning of cryo-EM grids enables controlled adhesion of mammalian cells for cryo-ET-based structural studies. This approach leads to reproducible cellular morphology and improves focused ion beam thinning of cells for in-cell structural analyses.},
  archive      = {J_NMETH},
  author       = {Toro-Nahuelpan, Mauricio and Zagoriy, Ievgeniia and Senger, Fabrice and Blanchoin, Laurent and Théry, Manuel and Mahamid, Julia},
  doi          = {10.1038/s41592-019-0630-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {50-54},
  shortjournal = {Nat. Meth.},
  title        = {Tailoring cryo-electron microscopy grids by photo-micropatterning for in-cell structural studies},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Revealing dynamics of gene expression variability in cell
state space. <em>NMETH</em>, <em>17</em>(1), 45–49. (<a
href="https://doi.org/10.1038/s41592-019-0632-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To decipher cell state transitions from single-cell transcriptomes it is crucial to quantify weak expression of lineage-determining factors, which requires computational methods that are sensitive to the variability of weakly expressed genes. Here, I introduce VarID, a computational method that identifies locally homogenous neighborhoods in cell state space, permitting the quantification of local variability in gene expression. VarID delineates neighborhoods with differential gene expression variability and reveals pseudo-temporal dynamics of variability during differentiation. VarID is a computational method that quantifies the dynamics of transcriptional variability with the goal of identifying the role of highly variable genes, such as weakly expressed transcription factors, in cell differentiation or state transitions.},
  archive      = {J_NMETH},
  author       = {Grün, Dominic},
  doi          = {10.1038/s41592-019-0632-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {45-49},
  shortjournal = {Nat. Meth.},
  title        = {Revealing dynamics of gene expression variability in cell state space},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DIA-NN: Neural networks and interference correction enable
deep proteome coverage in high throughput. <em>NMETH</em>,
<em>17</em>(1), 41–44. (<a
href="https://doi.org/10.1038/s41592-019-0638-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an easy-to-use integrated software suite, DIA-NN, that exploits deep neural networks and new quantification and signal correction strategies for the processing of data-independent acquisition (DIA) proteomics experiments. DIA-NN improves the identification and quantification performance in conventional DIA proteomic applications, and is particularly beneficial for high-throughput applications, as it is fast and enables deep and confident proteome coverage when used in combination with fast chromatographic methods. A deep learning-based software tool, DIA-NN, enables deep proteome analysis from data generated using fast chromatographic approaches and data-independent acquisition mass spectrometry.},
  archive      = {J_NMETH},
  author       = {Demichev, Vadim and Messner, Christoph B. and Vernardis, Spyros I. and Lilley, Kathryn S. and Ralser, Markus},
  doi          = {10.1038/s41592-019-0638-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {41-44},
  shortjournal = {Nat. Meth.},
  title        = {DIA-NN: Neural networks and interference correction enable deep proteome coverage in high throughput},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). FreeHi-c simulates high-fidelity hi-c data for benchmarking
and data augmentation. <em>NMETH</em>, <em>17</em>(1), 37–40. (<a
href="https://doi.org/10.1038/s41592-019-0624-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to simulate high-throughput chromatin conformation (Hi-C) data is foundational for benchmarking Hi-C data analysis methods. Here we present a nonparametric strategy named FreeHi-C to simulate Hi-C data from the interacting genome fragments. Data from FreeHi-C exhibit high fidelity to biological Hi-C data. FreeHi-C boosts the precision and power of differential chromatin interaction detection through data augmentation under preserved false discovery rate control. FreeHi-C takes Hi-C sequencing data as input and simulates reads with random mutations and indels from the interacting fragment pairs. FreeHi-C-simulated replicates are used for benchmarking Hi-C analysis methods and enable data augmentation for differential chromatin interaction analysis.},
  archive      = {J_NMETH},
  author       = {Zheng, Ye and Keleş, Sündüz},
  doi          = {10.1038/s41592-019-0624-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {37-40},
  shortjournal = {Nat. Meth.},
  title        = {FreeHi-C simulates high-fidelity hi-C data for benchmarking and data augmentation},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Boost that metabolomic confidence. <em>NMETH</em>,
<em>17</em>(1), 33–36. (<a
href="https://doi.org/10.1038/s41592-019-0694-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Change is a constant in the burgeoning field of metabolomics. That includes data analysis tools and repositories.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-019-0694-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {33-36},
  shortjournal = {Nat. Meth.},
  title        = {Boost that metabolomic confidence},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Counting short tandem repeats. <em>NMETH</em>,
<em>17</em>(1), 31. (<a
href="https://doi.org/10.1038/s41592-019-0724-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method using CRISPR–Cas technology, nanopore sequencing and bioinformatics enables quantification of short tandem repeats with high accuracy.},
  archive      = {J_NMETH},
  author       = {Miura, Grant},
  doi          = {10.1038/s41592-019-0724-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {31},
  shortjournal = {Nat. Meth.},
  title        = {Counting short tandem repeats},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020af). Reconstituting selectable markers. <em>NMETH</em>,
<em>17</em>(1), 30. (<a
href="https://doi.org/10.1038/s41592-019-0713-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-019-0713-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Reconstituting selectable markers},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020z). Optically measuring barcodes. <em>NMETH</em>,
<em>17</em>(1), 30. (<a
href="https://doi.org/10.1038/s41592-019-0716-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-019-0716-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Optically measuring barcodes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). Genomic variation in non-tumor cells. <em>NMETH</em>,
<em>17</em>(1), 30. (<a
href="https://doi.org/10.1038/s41592-019-0717-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-019-0717-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Genomic variation in non-tumor cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). Finding exciting inputs. <em>NMETH</em>, <em>17</em>(1),
30. (<a href="https://doi.org/10.1038/s41592-019-0719-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-019-0719-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Finding exciting inputs},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020p). Optogenetic inhibition. <em>NMETH</em>, <em>17</em>(1), 29.
(<a href="https://doi.org/10.1038/s41592-019-0712-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-019-0712-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {Optogenetic inhibition},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020v). Small-molecule sensors. <em>NMETH</em>, <em>17</em>(1), 29.
(<a href="https://doi.org/10.1038/s41592-019-0714-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-019-0714-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {Small-molecule sensors},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020r). Primate post-implantation development in a dish.
<em>NMETH</em>, <em>17</em>(1), 29. (<a
href="https://doi.org/10.1038/s41592-019-0715-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-019-0715-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {Primate post-implantation development in a dish},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Aerolysin nanopores. <em>NMETH</em>, <em>17</em>(1), 29.
(<a href="https://doi.org/10.1038/s41592-019-0718-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-019-0718-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {Aerolysin nanopores},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Built by association. <em>NMETH</em>, <em>17</em>(1), 28.
(<a href="https://doi.org/10.1038/s41592-019-0723-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A protein co-regulation database facilitates proteome-wide functional annotations.},
  archive      = {J_NMETH},
  author       = {Lössl, Philip},
  doi          = {10.1038/s41592-019-0723-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {28},
  shortjournal = {Nat. Meth.},
  title        = {Built by association},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Amplification-free single-cell whole-genome sequencing gets
a makeover. <em>NMETH</em>, <em>17</em>(1), 27. (<a
href="https://doi.org/10.1038/s41592-019-0722-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Direct library preparation (DLP) for amplification-free single-cell whole-genome sequencing has been upgraded to a high-throughput, scalable platform for identification of clonal populations and their genomic features, integrating them with single-cell morphology.},
  archive      = {J_NMETH},
  author       = {Todorovic, Vesna},
  doi          = {10.1038/s41592-019-0722-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {27},
  shortjournal = {Nat. Meth.},
  title        = {Amplification-free single-cell whole-genome sequencing gets a makeover},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Be still, my beating heart. <em>NMETH</em>, <em>17</em>(1),
26. (<a href="https://doi.org/10.1038/s41592-019-0721-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An optical gating strategy allows day-long light-sheet imaging of the developing heart in zebrafish embryos.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-019-0721-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {26},
  shortjournal = {Nat. Meth.},
  title        = {Be still, my beating heart},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020t). RAMPing up voltage indicator imaging. <em>NMETH</em>,
<em>17</em>(1), 25. (<a
href="https://doi.org/10.1038/s41592-019-0720-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combining a voltage sensor with random-access microscopy allows imaging of neuronal activity in behaving mice.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-019-0720-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {25},
  shortjournal = {Nat. Meth.},
  title        = {RAMPing up voltage indicator imaging},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020s). Tools for metabolomics. <em>NMETH</em>, <em>17</em>(1), 24.
(<a href="https://doi.org/10.1038/s41592-019-0710-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-019-0710-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {24},
  shortjournal = {Nat. Meth.},
  title        = {Tools for metabolomics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020s). Putting on the squeeze. <em>NMETH</em>, <em>17</em>(1), 24.
(<a href="https://doi.org/10.1038/s41592-019-0711-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-019-0711-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {24},
  shortjournal = {Nat. Meth.},
  title        = {Putting on the squeeze},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020x). Smarter microscopes. <em>NMETH</em>, <em>17</em>(1), 23.
(<a href="https://doi.org/10.1038/s41592-019-0708-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-019-0708-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {23},
  shortjournal = {Nat. Meth.},
  title        = {Smarter microscopes},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020r). Informatics for genomics. <em>NMETH</em>, <em>17</em>(1),
23. (<a href="https://doi.org/10.1038/s41592-019-0709-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-019-0709-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {23},
  shortjournal = {Nat. Meth.},
  title        = {Informatics for genomics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Collaborative neuroscience. <em>NMETH</em>, <em>17</em>(1),
22. (<a href="https://doi.org/10.1038/s41592-019-0706-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Working together for a common goal propels neuroscience research.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-019-0706-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {22},
  shortjournal = {Nat. Meth.},
  title        = {Collaborative neuroscience},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Exploring microbial worlds. <em>NMETH</em>, <em>17</em>(1),
22. (<a href="https://doi.org/10.1038/s41592-019-0707-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-019-0707-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {22},
  shortjournal = {Nat. Meth.},
  title        = {Exploring microbial worlds},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020y). Structures in situ. <em>NMETH</em>, <em>17</em>(1), 21. (<a
href="https://doi.org/10.1038/s41592-019-0704-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-019-0704-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {21},
  shortjournal = {Nat. Meth.},
  title        = {Structures in situ},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Base editors beware. <em>NMETH</em>, <em>17</em>(1), 21.
(<a href="https://doi.org/10.1038/s41592-019-0705-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tools are needed to study off-target mutations introduced by base editors.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-019-0705-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {21},
  shortjournal = {Nat. Meth.},
  title        = {Base editors beware},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell biology: Beyond the sum of its parts.
<em>NMETH</em>, <em>17</em>(1), 17–20. (<a
href="https://doi.org/10.1038/s41592-019-0693-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The field of single-cell RNA sequencing (scRNA-seq) has been paired with genomics, epigenomics, spatial omics, proteomics and imaging to achieve multimodal measurements of individual cellular phenotypes and genotypes. In its purest form, single-cell multimodal omics involves the simultaneous detection of multiple traits in the same cell. More broadly, multimodal omics also encompasses comparative pairing and computational integration of measurements made across multiple distinct cells to reconstruct phenotypes. Here I highlight some of the biological insights gained from multimodal studies and discuss the challenges and opportunities in this emerging field.},
  archive      = {J_NMETH},
  author       = {Schier, Alexander F.},
  doi          = {10.1038/s41592-019-0693-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {17-20},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell biology: Beyond the sum of its parts},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Computational methods for single-cell omics across
modalities. <em>NMETH</em>, <em>17</em>(1), 14–17. (<a
href="https://doi.org/10.1038/s41592-019-0692-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell omics approaches provide high-resolution data on cellular phenotypes, developmental dynamics and communication networks in diverse tissues and conditions. Emerging technologies now measure different modalities of individual cells, such as genomes, epigenomes, transcriptomes and proteomes, in addition to spatial profiling. Combined with analytical approaches, these data open new avenues for accurate reconstruction of gene-regulatory and signaling networks driving cellular identity and function. Here we summarize computational methods for analysis and integration of single-cell omics data across different modalities and discuss their applications, challenges and future directions.},
  archive      = {J_NMETH},
  author       = {Efremova, Mirjana and Teichmann, Sarah A.},
  doi          = {10.1038/s41592-019-0692-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {14-17},
  shortjournal = {Nat. Meth.},
  title        = {Computational methods for single-cell omics across modalities},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell multimodal omics: The power of many.
<em>NMETH</em>, <em>17</em>(1), 11–14. (<a
href="https://doi.org/10.1038/s41592-019-0691-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in single-cell genomics technologies have enabled investigation of the gene regulation programs of multicellular organisms at unprecedented resolution and scale. Development of single-cell multimodal omics tools is another major step toward understanding the inner workings of biological systems.},
  archive      = {J_NMETH},
  author       = {Zhu, Chenxu and Preissl, Sebastian and Ren, Bing},
  doi          = {10.1038/s41592-019-0691-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {11-14},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell multimodal omics: The power of many},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). The secret life of cells. <em>NMETH</em>, <em>17</em>(1),
7–10. (<a href="https://doi.org/10.1038/s41592-019-0698-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Armed with a rapidly maturing toolbox for single-cell analysis, scientists are threading together multiple layers of omic data to assemble rich portraits of cellular identity and function.},
  archive      = {J_NMETH},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41592-019-0698-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {7-10},
  shortjournal = {Nat. Meth.},
  title        = {The secret life of cells},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). CLIJ: GPU-accelerated image processing for everyone.
<em>NMETH</em>, <em>17</em>(1), 5–6. (<a
href="https://doi.org/10.1038/s41592-019-0650-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Haase, Robert and Royer, Loic A. and Steinbach, Peter and Schmidt, Deborah and Dibrov, Alexandr and Schmidt, Uwe and Weigert, Martin and Maghelli, Nicola and Tomancak, Pavel and Jug, Florian and Myers, Eugene W.},
  doi          = {10.1038/s41592-019-0650-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {5-6},
  shortjournal = {Nat. Meth.},
  title        = {CLIJ: GPU-accelerated image processing for everyone},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Method of the year 2019: Single-cell multimodal omics.
<em>NMETH</em>, <em>17</em>(1), 1. (<a
href="https://doi.org/10.1038/s41592-019-0703-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multimodal omics measurement offers opportunities for gaining holistic views of cells one by one.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-019-0703-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year 2019: Single-cell multimodal omics},
  volume       = {17},
  year         = {2020},
}
</textarea>
</details></li>
</ul>

</body>
</html>
