<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng---200">NG - 200</h2>
<ul>
<li><details>
<summary>
(2020). Publisher correction: A genome-wide search for asthma
susceptibility loci in ethnically diverse populations. <em>NG</em>,
<em>52</em>(12), 1433. (<a
href="https://doi.org/10.1038/s41588-020-0650-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0650-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1433},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: A genome-wide search for asthma susceptibility loci in ethnically diverse populations},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Treating medical data as a durable
asset. <em>NG</em>, <em>52</em>(12), 1433. (<a
href="https://doi.org/10.1038/s41588-020-00734-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Telenti, Amalio and Jiang, Xiaoqian},
  doi          = {10.1038/s41588-020-00734-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1433},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Treating medical data as a durable asset},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Phased diploid genome assemblies and pan-genomes provide
insights into the genetic history of apple domestication. <em>NG</em>,
<em>52</em>(12), 1423–1432. (<a
href="https://doi.org/10.1038/s41588-020-00723-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Domestication of the apple was mainly driven by interspecific hybridization. In the present study, we report the haplotype-resolved genomes of the cultivated apple (Malus domestica cv. Gala) and its two major wild progenitors, M. sieversii and M. sylvestris. Substantial variations are identified between the two haplotypes of each genome. Inference of genome ancestry identifies ~23% of the Gala genome as of hybrid origin. Deep sequencing of 91 accessions identifies selective sweeps in cultivated apples that originated from either of the two progenitors and are associated with important domestication traits. Construction and analyses of apple pan-genomes uncover thousands of new genes, with hundreds of them being selected from one of the progenitors and largely fixed in cultivated apples, revealing that introgression of new genes/alleles is a hallmark of apple domestication through hybridization. Finally, transcriptome profiles of Gala fruits at 13 developmental stages unravel ~19% of genes displaying allele-specific expression, including many associated with fruit quality. Phased diploid genomes of the cultivated apple Malus domestica cv. Gala and its two major wild progenitors M. sieversii and M. sylvestris, as well as pan-genome analyses, provide insights into the genetic history of apple domestication.},
  archive      = {J_NG},
  author       = {Sun, Xuepeng and Jiao, Chen and Schwaninger, Heidi and Chao, C. Thomas and Ma, Yumin and Duan, Naibin and Khan, Awais and Ban, Seunghyun and Xu, Kenong and Cheng, Lailiang and Zhong, Gan-Yuan and Fei, Zhangjun},
  doi          = {10.1038/s41588-020-00723-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1423-1432},
  shortjournal = {Nat. Genet.},
  title        = {Phased diploid genome assemblies and pan-genomes provide insights into the genetic history of apple domestication},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Triticum population sequencing provides insights into wheat
adaptation. <em>NG</em>, <em>52</em>(12), 1412–1422. (<a
href="https://doi.org/10.1038/s41588-020-00722-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bread wheat expanded its habitat from a core area of the Fertile Crescent to global environments within ~10,000 years. The genetic mechanisms of this remarkable evolutionary success are not well understood. By whole-genome sequencing of populations from 25 subspecies within the genera Triticum and Aegilops, we identified composite introgression from wild populations contributing to a substantial portion (4–32%) of the bread wheat genome, which increased the genetic diversity of bread wheat and allowed its divergent adaptation. Meanwhile, convergent adaptation to human selection showed 2- to 16-fold enrichment relative to random expectation—a certain set of genes were repeatedly selected in Triticum species despite their drastic differences in ploidy levels and growing zones, indicating the important role of evolutionary constraints in shaping the adaptive landscape of bread wheat. These results showed the genetic necessities of wheat as a global crop and provided new perspectives on transferring adaptive success across species for crop improvement. Whole-genome sequencing of wheat populations from 25 subspecies within the genera Triticum and Aegilops provides insights into the role of evolutionary constraints in shaping the adaptive landscape of bread wheat.},
  archive      = {J_NG},
  author       = {Zhou, Yao and Zhao, Xuebo and Li, Yiwen and Xu, Jun and Bi, Aoyue and Kang, Lipeng and Xu, Daxing and Chen, Haofeng and Wang, Ying and Wang, Yuan-ge and Liu, Sanyang and Jiao, Chengzhi and Lu, Hongfeng and Wang, Jing and Yin, Changbin and Jiao, Yuling and Lu, Fei},
  doi          = {10.1038/s41588-020-00722-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1412-1422},
  shortjournal = {Nat. Genet.},
  title        = {Triticum population sequencing provides insights into wheat adaptation},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). MLL4-associated condensates counterbalance polycomb-mediated
nuclear mechanical stress in kabuki syndrome. <em>NG</em>,
<em>52</em>(12), 1397–1411. (<a
href="https://doi.org/10.1038/s41588-020-00724-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic elements required to tune gene expression are partitioned in active and repressive nuclear condensates. Chromatin compartments include transcriptional clusters whose dynamic establishment and functioning depend on multivalent interactions occurring among transcription factors, cofactors and basal transcriptional machinery. However, how chromatin players contribute to the assembly of transcriptional condensates is poorly understood. By interrogating the effect of KMT2D (also known as MLL4) haploinsufficiency in Kabuki syndrome, we found that mixed lineage leukemia 4 (MLL4) contributes to the assembly of transcriptional condensates through liquid–liquid phase separation. MLL4 loss of function impaired Polycomb-dependent chromatin compartmentalization, altering the nuclear architecture. By releasing the nuclear mechanical stress through inhibition of the mechanosensor ATR, we re-established the mechanosignaling of mesenchymal stem cells and their commitment towards chondrocytes both in vitro and in vivo. This study supports the notion that, in Kabuki syndrome, the haploinsufficiency of MLL4 causes an altered functional partitioning of chromatin, which determines the architecture and mechanical properties of the nucleus. MLL4 (KMT2D) loss of function, as found in Kabuki syndrome, affects the chromatin compartmentalization of Polycomb proteins and changes the nuclear architecture. Inhibition of ATR reestablishes mechanosignaling of mutant mesenchymal stem cells and their commitment to becoming chondrocytes.},
  archive      = {J_NG},
  author       = {Fasciani, Alessandra and D’Annunzio, Sarah and Poli, Vittoria and Fagnocchi, Luca and Beyes, Sven and Michelatti, Daniela and Corazza, Francesco and Antonelli, Laura and Gregoretti, Francesco and Oliva, Gennaro and Belli, Romina and Peroni, Daniele and Domenici, Enrico and Zambrano, Samuel and Intartaglia, Daniela and Settembre, Carmine and Conte, Ivan and Testi, Claudia and Vergyris, Panagiotis and Ruocco, Giancarlo and Zippo, Alessio},
  doi          = {10.1038/s41588-020-00724-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1397-1411},
  shortjournal = {Nat. Genet.},
  title        = {MLL4-associated condensates counterbalance polycomb-mediated nuclear mechanical stress in kabuki syndrome},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). BAHCC1 binds H3K27me3 via a conserved BAH module to mediate
gene silencing and oncogenesis. <em>NG</em>, <em>52</em>(12), 1384–1396.
(<a href="https://doi.org/10.1038/s41588-020-00729-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Trimethylated histone H3 lysine 27 (H3K27me3) regulates gene repression, cell-fate determination and differentiation. We report that a conserved bromo-adjacent homology (BAH) module of BAHCC1 (BAHCC1BAH) ‘recognizes’ H3K27me3 specifically and enforces silencing of H3K27me3-demarcated genes in mammalian cells. Biochemical, structural and integrated chromatin immunoprecipitation-sequencing-based analyses demonstrate that direct readout of H3K27me3 by BAHCC1 is achieved through a hydrophobic trimethyl-l-lysine-binding ‘cage’ formed by BAHCC1BAH, mediating colocalization of BAHCC1 and H3K27me3-marked genes. BAHCC1 is highly expressed in human acute leukemia and interacts with transcriptional corepressors. In leukemia, depletion of BAHCC1, or disruption of the BAHCC1BAH–H3K27me3 interaction, causes derepression of H3K27me3-targeted genes that are involved in tumor suppression and cell differentiation, leading to suppression of oncogenesis. In mice, introduction of a germline mutation at Bahcc1 to disrupt its H3K27me3 engagement causes partial postnatal lethality, supporting a role in development. This study identifies an H3K27me3-directed transduction pathway in mammals that relies on a conserved BAH ‘reader’. A BAH module in BAHCC1 recognizes H3K27me3 and is involved in gene silencing. BAHCC1 is highly expressed in acute leukemia cells and contributes to their proliferation.},
  archive      = {J_NG},
  author       = {Fan, Huitao and Lu, Jiuwei and Guo, Yiran and Li, Dongxu and Zhang, Zhi-Min and Tsai, Yi-Hsuan and Pi, Wen-Chieh and Ahn, Jeong Hyun and Gong, Weida and Xiang, Yu and Allison, David F. and Geng, Huimin and He, Shenghui and Diao, Yarui and Chen, Wei-Yi and Strahl, Brian D. and Cai, Ling and Song, Jikui and Wang, Gang Greg},
  doi          = {10.1038/s41588-020-00729-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1384-1396},
  shortjournal = {Nat. Genet.},
  title        = {BAHCC1 binds H3K27me3 via a conserved BAH module to mediate gene silencing and oncogenesis},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Integrated molecular drivers coordinate biological and
clinical states in melanoma. <em>NG</em>, <em>52</em>(12), 1373–1383.
(<a href="https://doi.org/10.1038/s41588-020-00739-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We performed harmonized molecular and clinical analysis on 1,048 melanomas and discovered markedly different global genomic properties among subtypes (BRAF, (N)RAS, NF1, triple wild-type (TWT)), subtype-specific preferences for secondary driver genes and active mutational processes previously unreported in melanoma. Secondary driver genes significantly enriched in specific subtypes reflected preferential dysregulation of additional pathways, such as induction of transforming growth factor-β signaling in BRAF melanomas and inactivation of the SWI/SNF complex in (N)RAS melanomas, and select co-mutation patterns coordinated selective response to immune checkpoint blockade. We also defined the mutational landscape of TWT melanomas and revealed enrichment of DNA-repair-defect signatures in this subtype, which were associated with transcriptional downregulation of key DNA-repair genes, and may revive previously discarded or currently unconsidered therapeutic modalities for genomically stratified melanoma patient subsets. Broadly, harmonized meta-analysis of melanoma whole exomes revealed distinct molecular drivers that may point to multiple opportunities for biological and therapeutic investigation. Harmonized analysis of 1,048 melanomas shows different global genomic properties among subtypes and identifies subtype-specific enrichment for secondary driver genes.},
  archive      = {J_NG},
  author       = {Conway, Jake R. and Dietlein, Felix and Taylor-Weiner, Amaro and AlDubayan, Saud and Vokes, Natalie and Keenan, Tanya and Reardon, Brendan and He, Meng Xiao and Margolis, Claire A. and Weirather, Jason L. and Haq, Rizwan and Schilling, Bastian and Stephen Hodi, F. and Schadendorf, Dirk and Liu, David and Van Allen, Eliezer M.},
  doi          = {10.1038/s41588-020-00739-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1373-1383},
  shortjournal = {Nat. Genet.},
  title        = {Integrated molecular drivers coordinate biological and clinical states in melanoma},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). CGAS-mediated induction of type i interferon due to inborn
errors of histone pre-mRNA processing. <em>NG</em>, <em>52</em>(12),
1364–1372. (<a
href="https://doi.org/10.1038/s41588-020-00737-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inappropriate stimulation or defective negative regulation of the type I interferon response can lead to autoinflammation. In genetically uncharacterized cases of the type I interferonopathy Aicardi–Goutières syndrome, we identified biallelic mutations in LSM11 and RNU7-1, which encode components of the replication-dependent histone pre-mRNA–processing complex. Mutations were associated with the misprocessing of canonical histone transcripts and a disturbance of linker histone stoichiometry. Additionally, we observed an altered distribution of nuclear cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS) and enhanced interferon signaling mediated by the cGAS–stimulator of interferon genes (STING) pathway in patient-derived fibroblasts. Finally, we established that chromatin without linker histone stimulates cyclic guanosine monophosphate–adenosine monophosphate (cGAMP) production in vitro more efficiently. We conclude that nuclear histones, as key constituents of chromatin, are essential in suppressing the immunogenicity of self-DNA. Mutations in LSM11 and RNU7-1, which encode components of the replication-dependent histone pre-mRNA–processing complex, cause an autoinflammatory syndrome due to enhanced interferon signaling mediated by the cGAS–STING pathway, showing an essential role for nuclear histones in suppressing the immunogenicity of self-DNA.},
  archive      = {J_NG},
  author       = {Uggenti, Carolina and Lepelley, Alice and Depp, Marine and Badrock, Andrew P. and Rodero, Mathieu P. and El-Daher, Marie-Thérèse and Rice, Gillian I. and Dhir, Somdutta and Wheeler, Ann P. and Dhir, Ashish and Albawardi, Waad and Frémond, Marie-Louise and Seabra, Luis and Doig, Jennifer and Blair, Natalie and Martin-Niclos, Maria José and Della Mina, Erika and Rubio-Roldán, Alejandro and García-Pérez, Jose L. and Sproul, Duncan and Rehwinkel, Jan and Hertzog, Jonny and Boland-Auge, Anne and Olaso, Robert and Deleuze, Jean-François and Baruteau, Julien and Brochard, Karine and Buckley, Jonathan and Cavallera, Vanessa and Cereda, Cristina and De Waele, Liesbeth M. H. and Dobbie, Angus and Doummar, Diane and Elmslie, Frances and Koch-Hogrebe, Margarete and Kumar, Ram and Lamb, Kate and Livingston, John H. and Majumdar, Anirban and Lorenço, Charles Marques and Orcesi, Simona and Peudenier, Sylviane and Rostasy, Kevin and Salmon, Caroline A. and Scott, Christiaan and Tonduti, Davide and Touati, Guy and Valente, Marialuisa and van der Linden, Hélio and Van Esch, Hilde and Vermelle, Marie and Webb, Kate and Jackson, Andrew P. and Reijns, Martin A. M. and Gilbert, Nick and Crow, Yanick J.},
  doi          = {10.1038/s41588-020-00737-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1364-1372},
  shortjournal = {Nat. Genet.},
  title        = {CGAS-mediated induction of type i interferon due to inborn errors of histone pre-mRNA processing},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Functionally informed fine-mapping and polygenic
localization of complex trait heritability. <em>NG</em>,
<em>52</em>(12), 1355–1363. (<a
href="https://doi.org/10.1038/s41588-020-00735-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fine-mapping aims to identify causal variants impacting complex traits. We propose PolyFun, a computationally scalable framework to improve fine-mapping accuracy by leveraging functional annotations across the entire genome—not just genome-wide-significant loci—to specify prior probabilities for fine-mapping methods such as SuSiE or FINEMAP. In simulations, PolyFun + SuSiE and PolyFun + FINEMAP were well calibrated and identified &amp;gt;20% more variants with a posterior causal probability &amp;gt;0.95 than identified in their nonfunctionally informed counterparts. In analyses of 49 UK Biobank traits (average n = 318,000), PolyFun + SuSiE identified 3,025 fine-mapped variant–trait pairs with posterior causal probability &amp;gt;0.95, a &amp;gt;32% improvement versus SuSiE. We used posterior mean per-SNP heritabilities from PolyFun + SuSiE to perform polygenic localization, constructing minimal sets of common SNPs causally explaining 50% of common SNP heritability; these sets ranged in size from 28 (hair color) to 3,400 (height) to 2 million (number of children). In conclusion, PolyFun prioritizes variants for functional follow-up and provides insights into complex trait architectures. PolyFun is a computationally scalable framework for functionally informed fine-mapping that makes full use of genome-wide data. It prioritizes more variants than previous methods when applied to 49 complex traits from UK Biobank.},
  archive      = {J_NG},
  author       = {Weissbrod, Omer and Hormozdiari, Farhad and Benner, Christian and Cui, Ran and Ulirsch, Jacob and Gazal, Steven and Schoech, Armin P. and van de Geijn, Bryce and Reshef, Yakir and Márquez-Luna, Carla and O’Connor, Luke and Pirinen, Matti and Finucane, Hilary K. and Price, Alkes L.},
  doi          = {10.1038/s41588-020-00735-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1355-1363},
  shortjournal = {Nat. Genet.},
  title        = {Functionally informed fine-mapping and polygenic localization of complex trait heritability},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Improving the trans-ancestry portability of polygenic risk
scores by prioritizing variants in predicted cell-type-specific
regulatory elements. <em>NG</em>, <em>52</em>(12), 1346–1354. (<a
href="https://doi.org/10.1038/s41588-020-00740-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Poor trans-ancestry portability of polygenic risk scores is a consequence of Eurocentric genetic studies and limited knowledge of shared causal variants. Leveraging regulatory annotations may improve portability by prioritizing functional over tagging variants. We constructed a resource of 707 cell-type-specific IMPACT regulatory annotations by aggregating 5,345 epigenetic datasets to predict binding patterns of 142 transcription factors across 245 cell types. We then partitioned the common SNP heritability of 111 genome-wide association study summary statistics of European (average n ≈ 189,000) and East Asian (average n ≈ 157,000) origin. IMPACT annotations captured consistent SNP heritability between populations, suggesting prioritization of shared functional variants. Variant prioritization using IMPACT resulted in increased trans-ancestry portability of polygenic risk scores from Europeans to East Asians across all 21 phenotypes analyzed (49.9% mean relative increase in R2). Our study identifies a crucial role for functional annotations such as IMPACT to improve the trans-ancestry portability of genetic data. A resource of cell-type-specific IMPACT regulatory annotations improves the trans-ancestry portability of polygenic risk scores by prioritizing variants enriched for trait heritability.},
  archive      = {J_NG},
  author       = {Amariuta, Tiffany and Ishigaki, Kazuyoshi and Sugishita, Hiroki and Ohta, Tazro and Koido, Masaru and Dey, Kushal K. and Matsuda, Koichi and Murakami, Yoshinori and Price, Alkes L. and Kawakami, Eiryo and Terao, Chikashi and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-020-00740-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1346-1354},
  shortjournal = {Nat. Genet.},
  title        = {Improving the trans-ancestry portability of polygenic risk scores by prioritizing variants in predicted cell-type-specific regulatory elements},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Common variants in signaling transcription-factor-binding
sites drive phenotypic variability in red blood cell traits.
<em>NG</em>, <em>52</em>(12), 1333–1345. (<a
href="https://doi.org/10.1038/s41588-020-00738-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies identify genomic variants associated with human traits and diseases. Most trait-associated variants are located within cell-type-specific enhancers, but the molecular mechanisms governing phenotypic variation are less well understood. Here, we show that many enhancer variants associated with red blood cell (RBC) traits map to enhancers that are co-bound by lineage-specific master transcription factors (MTFs) and signaling transcription factors (STFs) responsive to extracellular signals. The majority of enhancer variants reside on STF and not MTF motifs, perturbing DNA binding by various STFs (BMP/TGF-β-directed SMADs or WNT-induced TCFs) and affecting target gene expression. Analyses of engineered human blood cells and expression quantitative trait loci verify that disrupted STF binding leads to altered gene expression. Our results propose that the majority of the RBC-trait-associated variants that reside on transcription-factor-binding sequences fall in STF target sequences, suggesting that the phenotypic variation of RBC traits could stem from altered responsiveness to extracellular stimuli. Enhancer variants associated with red blood cell traits alter signaling transcription factor motifs, leading to changes in expression of genes that are upregulated during erythroid differentiation.},
  archive      = {J_NG},
  author       = {Choudhuri, Avik and Trompouki, Eirini and Abraham, Brian J. and Colli, Leandro M. and Kock, Kian Hong and Mallard, William and Yang, Min-Lee and Vinjamur, Divya S. and Ghamari, Alireza and Sporrij, Audrey and Hoi, Karen and Hummel, Barbara and Boatman, Sonja and Chan, Victoria and Tseng, Sierra and Nandakumar, Satish K. and Yang, Song and Lichtig, Asher and Superdock, Michael and Grimes, Seraj N. and Bowman, Teresa V. and Zhou, Yi and Takahashi, Shinichiro and Joehanes, Roby and Cantor, Alan B. and Bauer, Daniel E. and Ganesh, Santhi K. and Rinn, John and Albert, Paul S. and Bulyk, Martha L. and Chanock, Stephen J. and Young, Richard A. and Zon, Leonard I.},
  doi          = {10.1038/s41588-020-00738-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1333-1345},
  shortjournal = {Nat. Genet.},
  title        = {Common variants in signaling transcription-factor-binding sites drive phenotypic variability in red blood cell traits},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Discovery of rare variants associated with blood pressure
regulation through meta-analysis of 1.3 million individuals.
<em>NG</em>, <em>52</em>(12), 1314–1332. (<a
href="https://doi.org/10.1038/s41588-020-00713-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies of blood pressure (BP) to date have mainly analyzed common variants (minor allele frequency &amp;gt; 0.05). In a meta-analysis of up to ~1.3 million participants, we discovered 106 new BP-associated genomic regions and 87 rare (minor allele frequency ≤ 0.01) variant BP associations (P &amp;lt; 5 × 10−8), of which 32 were in new BP-associated loci and 55 were independent BP-associated single-nucleotide variants within known BP-associated regions. Average effects of rare variants (44% coding) were ~8 times larger than common variant effects and indicate potential candidate causal genes at new and known loci (for example, GATA5 and PLCB3). BP-associated variants (including rare and common) were enriched in regions of active chromatin in fetal tissues, potentially linking fetal development with BP regulation in later life. Multivariable Mendelian randomization suggested possible inverse effects of elevated systolic and diastolic BP on large artery stroke. Our study demonstrates the utility of rare-variant analyses for identifying candidate genes and the results highlight potential therapeutic targets. Meta-analyses in up to 1.3 million individuals identify 87 rare-variant associations with blood pressure traits. On average, rare variants exhibit effects ~8 times larger than the mean effects of common variants and implicate candidate causal genes at associated regions.},
  archive      = {J_NG},
  author       = {Surendran, Praveen and Feofanova, Elena V. and Lahrouchi, Najim and Ntalla, Ioanna and Karthikeyan, Savita and Cook, James and Chen, Lingyan and Mifsud, Borbala and Yao, Chen and Kraja, Aldi T. and Cartwright, James H. and Hellwege, Jacklyn N. and Giri, Ayush and Tragante, Vinicius and Thorleifsson, Gudmar and Liu, Dajiang J. and Prins, Bram P. and Stewart, Isobel D. and Cabrera, Claudia P. and Eales, James M. and Akbarov, Artur and Auer, Paul L. and Bielak, Lawrence F. and Bis, Joshua C. and Braithwaite, Vickie S. and Brody, Jennifer A. and Daw, E. Warwick and Warren, Helen R. and Drenos, Fotios and Nielsen, Sune Fallgaard and Faul, Jessica D. and Fauman, Eric B. and Fava, Cristiano and Ferreira, Teresa and Foley, Christopher N. and Franceschini, Nora and Gao, He and Giannakopoulou, Olga and Giulianini, Franco and Gudbjartsson, Daniel F. and Guo, Xiuqing and Harris, Sarah E. and Havulinna, Aki S. and Helgadottir, Anna and Huffman, Jennifer E. and Hwang, Shih-Jen and Kanoni, Stavroula and Kontto, Jukka and Larson, Martin G. and Li-Gao, Ruifang and Lindström, Jaana and Lotta, Luca A. and Lu, Yingchang and Luan, Jian’an and Mahajan, Anubha and Malerba, Giovanni and Masca, Nicholas G. D. and Mei, Hao and Menni, Cristina and Mook-Kanamori, Dennis O. and Mosen-Ansorena, David and Müller-Nurasyid, Martina and Paré, Guillaume and Paul, Dirk S. and Perola, Markus and Poveda, Alaitz and Rauramaa, Rainer and Richard, Melissa and Richardson, Tom G. and Sepúlveda, Nuno and Sim, Xueling and Smith, Albert V. and Smith, Jennifer A. and Staley, James R. and Stanáková, Alena and Sulem, Patrick and Thériault, Sébastien and Thorsteinsdottir, Unnur and Trompet, Stella and Varga, Tibor V. and Velez Edwards, Digna R. and Veronesi, Giovanni and Weiss, Stefan and Willems, Sara M. and Yao, Jie and Young, Robin and Yu, Bing and Zhang, Weihua and Zhao, Jing-Hua and Zhao, Wei and Evangelou, Evangelos and Aeschbacher, Stefanie and Asllanaj, Eralda and Blankenberg, Stefan and Bonnycastle, Lori L. and Bork-Jensen, Jette and Brandslund, Ivan and Braund, Peter S. and Burgess, Stephen and Cho, Kelly and Christensen, Cramer and Connell, John and Mutsert, Renée de and Dominiczak, Anna F. and Dörr, Marcus and Eiriksdottir, Gudny and Farmaki, Aliki-Eleni and Gaziano, J. Michael and Grarup, Niels and Grove, Megan L. and Hallmans, Göran and Hansen, Torben and Have, Christian T. and Heiss, Gerardo and Jørgensen, Marit E. and Jousilahti, Pekka and Kajantie, Eero and Kamat, Mihir and Käräjämäki, AnneMari and Karpe, Fredrik and Koistinen, Heikki A. and Kovesdy, Csaba P. and Kuulasmaa, Kari and Laatikainen, Tiina and Lannfelt, Lars and Lee, I-Te and Lee, Wen-Jane and Linneberg, Allan and Martin, Lisa W. and Moitry, Marie and Nadkarni, Girish and Neville, Matt J. and Palmer, Colin N. A. and Papanicolaou, George J. and Pedersen, Oluf and Peters, James and Poulter, Neil and Rasheed, Asif and Rasmussen, Katrine L. and Rayner, N. William and Mägi, Reedik and Renström, Frida and Rettig, Rainer and Rossouw, Jacques and Schreiner, Pamela J. and Sever, Peter S. and Sigurdsson, Emil L. and Skaaby, Tea and Sun, Yan V. and Sundstrom, Johan and Thorgeirsson, Gudmundur and Esko, Tõnu and Trabetti, Elisabetta and Tsao, Philip S. and Tuomi, Tiinamaija and Turner, Stephen T. and Tzoulaki, Ioanna and Vaartjes, Ilonca and Vergnaud, Anne-Claire and Willer, Cristen J. and Wilson, Peter W. F. and Witte, Daniel R. and Yonova-Doing, Ekaterina and Zhang, He and Aliya, Naheed and Almgren, Peter and Amouyel, Philippe and Asselbergs, Folkert W. and Barnes, Michael R. and Blakemore, Alexandra I. and Boehnke, Michael and Bots, Michiel L. and Bottinger, Erwin P. and Buring, Julie E. and Chambers, John C. and Chen, Yii-Der Ida and Chowdhury, Rajiv and Conen, David and Correa, Adolfo and Davey Smith, George and Boer, Rudolf A. de and Deary, Ian J. and Dedoussis, George and Deloukas, Panos and Di Angelantonio, Emanuele and Elliott, Paul and Felix, Stephan B. and Ferrières, Jean and Ford, Ian and Fornage, Myriam and Franks, Paul W. and Franks, Stephen and Frossard, Philippe and Gambaro, Giovanni and Gaunt, Tom R. and Groop, Leif and Gudnason, Vilmundur and Harris, Tamara B. and Hayward, Caroline and Hennig, Branwen J. and Herzig, Karl-Heinz and Ingelsson, Erik and Tuomilehto, Jaakko and Järvelin, Marjo-Riitta and Jukema, J. Wouter and Kardia, Sharon L. R. and Kee, Frank and Kooner, Jaspal S. and Kooperberg, Charles and Launer, Lenore J. and Lind, Lars and Loos, Ruth J. F. and Majumder, Abdulla al Shafi. and Laakso, Markku and McCarthy, Mark I. and Melander, Olle and Mohlke, Karen L. and Murray, Alison D. and Nordestgaard, Børge Grønne and Orho-Melander, Marju and Packard, Chris J. and Padmanabhan, Sandosh and Palmas, Walter and Polasek, Ozren and Porteous, David J. and Prentice, Andrew M. and Province, Michael A. and Relton, Caroline L. and Rice, Kenneth and Ridker, Paul M. and Rolandsson, Olov and Rosendaal, Frits R. and Rotter, Jerome I. and Rudan, Igor and Salomaa, Veikko and Samani, Nilesh J. and Sattar, Naveed and Sheu, Wayne H.-H. and Smith, Blair H. and Soranzo, Nicole and Spector, Timothy D. and Starr, John M. and Sebert, Sylvain and Taylor, Kent D. and Lakka, Timo A. and Timpson, Nicholas J. and Tobin, Martin D. and van der Harst, Pim and van der Meer, Peter and Ramachandran, Vasan S. and Verweij, Niek and Virtamo, Jarmo and Völker, Uwe and Weir, David R. and Zeggini, Eleftheria and Charchar, Fadi J. and Wareham, Nicholas J. and Langenberg, Claudia and Tomaszewski, Maciej and Butterworth, Adam S. and Caulfield, Mark J. and Danesh, John and Edwards, Todd L. and Holm, Hilma and Hung, Adriana M. and Lindgren, Cecilia M. and Liu, Chunyu and Manning, Alisa K. and Morris, Andrew P. and Morrison, Alanna C. and O’Donnell, Christopher J. and Psaty, Bruce M. and Saleheen, Danish and Stefansson, Kari and Boerwinkle, Eric and Chasman, Daniel I. and Levy, Daniel and Newton-Cheh, Christopher and Munroe, Patricia B. and Howson, Joanna M. M.},
  doi          = {10.1038/s41588-020-00713-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1314-1332},
  shortjournal = {Nat. Genet.},
  title        = {Discovery of rare variants associated with blood pressure regulation through meta-analysis of 1.3 million individuals},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genome-wide association study of intracranial aneurysms
identifies 17 risk loci and genetic overlap with clinical risk factors.
<em>NG</em>, <em>52</em>(12), 1303–1313. (<a
href="https://doi.org/10.1038/s41588-020-00725-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rupture of an intracranial aneurysm leads to subarachnoid hemorrhage, a severe type of stroke. To discover new risk loci and the genetic architecture of intracranial aneurysms, we performed a cross-ancestry, genome-wide association study in 10,754 cases and 306,882 controls of European and East Asian ancestry. We discovered 17 risk loci, 11 of which are new. We reveal a polygenic architecture and explain over half of the disease heritability. We show a high genetic correlation between ruptured and unruptured intracranial aneurysms. We also find a suggestive role for endothelial cells by using gene mapping and heritability enrichment. Drug-target enrichment shows pleiotropy between intracranial aneurysms and antiepileptic and sex hormone drugs, providing insights into intracranial aneurysm pathophysiology. Finally, genetic risks for smoking and high blood pressure, the two main clinical risk factors, play important roles in intracranial aneurysm risk, and drive most of the genetic correlation between intracranial aneurysms and other cerebrovascular traits. Cross-ancestry genome-wide association analyses in individuals of European and East Asian ancestry identify 11 new risk loci for intracranial aneurysms and highlight a polygenic architecture explaining a substantial fraction of disease heritability.},
  archive      = {J_NG},
  author       = {Bakker, Mark K. and van der Spek, Rick A. A. and van Rheenen, Wouter and Morel, Sandrine and Bourcier, Romain and Hostettler, Isabel C. and Alg, Varinder S. and van Eijk, Kristel R. and Koido, Masaru and Akiyama, Masato and Terao, Chikashi and Matsuda, Koichi and Walters, Robin G. and Lin, Kuang and Li, Liming and Millwood, Iona Y. and Chen, Zhengming and Rouleau, Guy A. and Zhou, Sirui and Rannikmäe, Kristiina and Sudlow, Cathie L. M. and Houlden, Henry and van den Berg, Leonard H. and Dina, Christian and Naggara, Olivier and Gentric, Jean-Christophe and Shotar, Eimad and Eugène, François and Desal, Hubert and Winsvold, Bendik S. and Børte, Sigrid and Johnsen, Marianne Bakke and Brumpton, Ben M. and Sandvei, Marie Søfteland and Willer, Cristen J. and Hveem, Kristian and Zwart, John-Anker and Verschuren, W. M. Monique and Friedrich, Christoph M. and Hirsch, Sven and Schilling, Sabine and Dauvillier, Jérôme and Martin, Olivier and Jones, Gregory T. and Bown, Matthew J. and Ko, Nerissa U. and Kim, Helen and Coleman, Jonathan R. I. and Breen, Gerome and Zaroff, Jonathan G. and Klijn, Catharina J. M. and Malik, Rainer and Dichgans, Martin and Sargurupremraj, Muralidharan and Tatlisumak, Turgut and Amouyel, Philippe and Debette, Stéphanie and Rinkel, Gabriel J. E. and Worrall, Bradford B. and Pera, Joanna and Slowik, Agnieszka and Gaál-Paavola, Emília I. and Niemelä, Mika and Jääskeläinen, Juha E. and von Und Zu Fraunberg, Mikael and Lindgren, Antti and Broderick, Joseph P. and Werring, David J. and Woo, Daniel and Redon, Richard and Bijlenga, Philippe and Kamatani, Yoichiro and Veldink, Jan H. and Ruigrok, Ynte M.},
  doi          = {10.1038/s41588-020-00725-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1303-1313},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study of intracranial aneurysms identifies 17 risk loci and genetic overlap with clinical risk factors},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Tissue-specific and interferon-inducible expression of
nonfunctional ACE2 through endogenous retroelement co-option.
<em>NG</em>, <em>52</em>(12), 1294–1302. (<a
href="https://doi.org/10.1038/s41588-020-00732-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Angiotensin-converting enzyme 2 (ACE2) is an entry receptor for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and a regulator of several physiological processes. ACE2 has recently been proposed to be interferon (IFN) inducible, suggesting that SARS-CoV-2 may exploit this phenomenon to enhance viral spread and questioning the efficacy of IFN treatment in coronavirus disease 2019. Using a recent de novo transcript assembly that captured previously unannotated transcripts, we describe a new isoform of ACE2, generated by co-option of intronic retroelements as promoter and alternative exon. The new transcript, termed MIRb-ACE2, exhibits specific expression patterns across the aerodigestive and gastrointestinal tracts and is highly responsive to IFN stimulation. In contrast, canonical ACE2 expression is unresponsive to IFN stimulation. Moreover, the MIRb-ACE2 translation product is a truncated, unstable ACE2 form, lacking domains required for SARS-CoV-2 binding and is therefore unlikely to contribute to or enhance viral infection. A truncated angiotensin-converting enzyme 2 (ACE2) isoform that lacks domains required for severe acute respiratory syndrome coronavirus 2 binding exhibits tissue-specific expression patterns and is responsive to interferon stimulation, in contrast to full-length ACE2, which is unresponsive to interferons.},
  archive      = {J_NG},
  author       = {Ng, Kevin W. and Attig, Jan and Bolland, William and Young, George R. and Major, Jack and Wrobel, Antoni G. and Gamblin, Steve and Wack, Andreas and Kassiotis, George},
  doi          = {10.1038/s41588-020-00732-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1294-1302},
  shortjournal = {Nat. Genet.},
  title        = {Tissue-specific and interferon-inducible expression of nonfunctional ACE2 through endogenous retroelement co-option},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Interferons and viruses induce a novel truncated ACE2
isoform and not the full-length SARS-CoV-2 receptor. <em>NG</em>,
<em>52</em>(12), 1283–1293. (<a
href="https://doi.org/10.1038/s41588-020-00731-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which causes COVID-19, utilizes angiotensin-converting enzyme 2 (ACE2) for entry into target cells. ACE2 has been proposed as an interferon-stimulated gene (ISG). Thus, interferon-induced variability in ACE2 expression levels could be important for susceptibility to COVID-19 or its outcomes. Here, we report the discovery of a novel, transcriptionally independent truncated isoform of ACE2, which we designate as deltaACE2 (dACE2). We demonstrate that dACE2, but not ACE2, is an ISG. In The Cancer Genome Atlas, the expression of dACE2 was enriched in squamous tumors of the respiratory, gastrointestinal and urogenital tracts. In vitro, dACE2, which lacks 356 amino-terminal amino acids, was non-functional in binding the SARS-CoV-2 spike protein and as a carboxypeptidase. Our results suggest that the ISG-type induction of dACE2 in IFN-high conditions created by treatments, an inflammatory tumor microenvironment or viral co-infections is unlikely to increase the cellular entry of SARS-CoV-2 and promote infection. dACE2 is a newly identified isoform of ACE2 that is unable to bind the SARS-CoV-2 spike protein. Truncated dACE2, but not full-length ACE2, is induced by interferons and viruses, thus suggesting that such conditions are unlikely to increase cellular entry of SARS-CoV-2.},
  archive      = {J_NG},
  author       = {Onabajo, Olusegun O. and Banday, A. Rouf and Stanifer, Megan L. and Yan, Wusheng and Obajemu, Adeola and Santer, Deanna M. and Florez-Vargas, Oscar and Piontkivska, Helen and Vargas, Joselin M. and Ring, Timothy J. and Kee, Carmon and Doldan, Patricio and Tyrrell, D. Lorne and Mendoza, Juan L. and Boulant, Steeve and Prokunina-Olsson, Ludmila},
  doi          = {10.1038/s41588-020-00731-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1283-1293},
  shortjournal = {Nat. Genet.},
  title        = {Interferons and viruses induce a novel truncated ACE2 isoform and not the full-length SARS-CoV-2 receptor},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reevaluating the roles of histone-modifying enzymes and
their associated chromatin modifications in transcriptional regulation.
<em>NG</em>, <em>52</em>(12), 1271–1281. (<a
href="https://doi.org/10.1038/s41588-020-00736-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Histone-modifying enzymes are implicated in the control of diverse DNA-templated processes including gene expression. Here, we outline historical and current thinking regarding the functions of histone modifications and their associated enzymes. One current viewpoint, based largely on correlative evidence, posits that histone modifications are instructive for transcriptional regulation and represent an epigenetic ‘code’. Recent studies have challenged this model and suggest that histone marks previously associated with active genes do not directly cause transcriptional activation. Additionally, many histone-modifying proteins possess non-catalytic functions that overshadow their enzymatic activities. Given that much remains unknown regarding the functions of these proteins, the field should be cautious in interpreting loss-of-function phenotypes and must consider both cellular and developmental context. In this Perspective, we focus on recent progress relating to the catalytic and non-catalytic functions of the Trithorax–COMPASS complexes, Polycomb repressive complexes and Clr4/Suv39 histone-modifying machineries. Recent progress relating to the catalytic and non-catalytic functions of histone modifying complexes warrants a fresh look at the role of histone modifications and the “histone code” model.},
  archive      = {J_NG},
  author       = {Morgan, Marc A. J. and Shilatifard, Ali},
  doi          = {10.1038/s41588-020-00736-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1271-1281},
  shortjournal = {Nat. Genet.},
  title        = {Reevaluating the roles of histone-modifying enzymes and their associated chromatin modifications in transcriptional regulation},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Loosening the grip on nuclear cGAS. <em>NG</em>,
<em>52</em>(12), 1269–1270. (<a
href="https://doi.org/10.1038/s41588-020-00746-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maintaining ignorance to self-nucleic acids can prevent inflammatory diseases such as Aicardi–Goutières syndrome (AGS). A new study finds that mutations in LSM11 and RNU7-1, which encode components of the histone messenger RNA–preprocessing complex, cause AGS by loosening the binding of cyclic GMP–AMP synthase (cGAS) to nucleosomes, thus enabling cGAS activation and induction of type I interferons.},
  archive      = {J_NG},
  author       = {Humphries, Fiachra and Fitzgerald, Katherine A.},
  doi          = {10.1038/s41588-020-00746-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1269-1270},
  shortjournal = {Nat. Genet.},
  title        = {Loosening the grip on nuclear cGAS},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). José luis gómez-skarmeta (1966–2020). <em>NG</em>,
<em>52</em>(12), 1267–1268. (<a
href="https://doi.org/10.1038/s41588-020-00743-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {On 16 September 2020, when the news of José Luis Gómez-Skarmeta’s untimely passing broke, the scientific community was left in dismay.},
  archive      = {J_NG},
  author       = {Bogdanovic, Ozren and Irimia, Manuel},
  doi          = {10.1038/s41588-020-00743-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1267-1268},
  shortjournal = {Nat. Genet.},
  title        = {José luis gómez-skarmeta (1966–2020)},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Complex genetic signatures in immune
cells underlie autoimmunity and inform therapy. <em>NG</em>,
<em>52</em>(11), 1266. (<a
href="https://doi.org/10.1038/s41588-020-00718-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Orrù, Valeria and Steri, Maristella and Sidore, Carlo and Marongiu, Michele and Serra, Valentina and Olla, Stefania and Sole, Gabriella and Lai, Sandra and Dei, Mariano and Mulas, Antonella and Virdis, Francesca and Piras, Maria Grazia and Lobina, Monia and Marongiu, Mara and Pitzalis, Maristella and Deidda, Francesca and Loizedda, Annalisa and Onano, Stefano and Zoledziewska, Magdalena and Sawcer, Stephen and Devoto, Marcella and Gorospe, Myriam and Abecasis, Gonçalo R. and Floris, Matteo and Pala, Mauro and Schlessinger, David and Fiorillo, Edoardo and Cucca, Francesco},
  doi          = {10.1038/s41588-020-00718-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1266},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Complex genetic signatures in immune cells underlie autoimmunity and inform therapy},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Author correction: An integrated multi-omics approach
identifies epigenetic alterations associated with alzheimer’s disease.
<em>NG</em>, <em>52</em>(11), 1266. (<a
href="https://doi.org/10.1038/s41588-020-00733-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Nativio, Raffaella and Lan, Yemin and Donahue, Greg and Sidoli, Simone and Berson, Amit and Srinivasan, Ananth R. and Shcherbakova, Oksana and Amlie-Wolf, Alexandre and Nie, Ji and Cui, Xiaolong and He, Chuan and Wang, Li-San and Garcia, Benjamin A. and Trojanowski, John Q. and Bonini, Nancy M. and Berger, Shelley L.},
  doi          = {10.1038/s41588-020-00733-7},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1266},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: An integrated multi-omics approach identifies epigenetic alterations associated with alzheimer’s disease},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: PGBD5 promotes site-specific oncogenic
mutations in human tumors. <em>NG</em>, <em>52</em>(11), 1265. (<a
href="https://doi.org/10.1038/s41588-020-00711-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Henssen, Anton G. and Koche, Richard and Zhuang, Jiali and Jiang, Eileen and Reed, Casie and Eisenberg, Amy and Still, Eric and MacArthur, Ian C. and Rodríguez-Fos, Elias and Gonzalez, Santiago and Puiggròs, Montserrat and Blackford, Andrew N. and Mason, Christopher E. and de Stanchina, Elisa and Gönen, Mithat and Emde, Anne-Katrin and Shah, Minita and Arora, Kanika and Reeves, Catherine and Socci, Nicholas D. and Perlman, Elizabeth and Antonescu, Cristina R. and Roberts, Charles W. M. and Steen, Hanno and Mullen, Elizabeth and Jackson, Stephen P. and Torrents, David and Weng, Zhiping and Armstrong, Scott A. and Kentsis, Alex},
  doi          = {10.1038/s41588-020-00711-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1265},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: PGBD5 promotes site-specific oncogenic mutations in human tumors},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: WNT signaling and AHCTF1 promote
oncogenic MYC expression through super-enhancer-mediated gene gating.
<em>NG</em>, <em>52</em>(11), 1265. (<a
href="https://doi.org/10.1038/s41588-020-00715-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Scholz, Barbara A. and Sumida, Noriyuki and de Lima, Carolina Diettrich Mallet and Chachoua, Ilyas and Martino, Mirco and Tzelepis, Ilias and Nikoshkov, Andrej and Zhao, Honglei and Mehmood, Rashid and Sifakis, Emmanouil G. and Bhartiya, Deeksha and Göndör, Anita and Ohlsson, Rolf},
  doi          = {10.1038/s41588-020-00715-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1265},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: WNT signaling and AHCTF1 promote oncogenic MYC expression through super-enhancer-mediated gene gating},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Assembly of whole-chromosome pseudomolecules for polyploid
plant genomes using outbred mapping populations. <em>NG</em>,
<em>52</em>(11), 1256–1264. (<a
href="https://doi.org/10.1038/s41588-020-00717-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in sequencing technologies, assembly of complex plant genomes remains elusive due to polyploidy and high repeat content. Here we report PolyGembler for grouping and ordering contigs into pseudomolecules by genetic linkage analysis. Our approach also provides an accurate method with which to detect and fix assembly errors. Using simulated data, we demonstrate that our approach is of high accuracy and outperforms three existing state-of-the-art genetic mapping tools. Particularly, our approach is more robust to the presence of missing genotype data and genotyping errors. We used our method to construct pseudomolecules for allotetraploid lawn grass utilizing PacBio long reads in combination with restriction site-associated DNA sequencing, and for diploid Ipomoea trifida and autotetraploid potato utilizing contigs assembled from Illumina reads in combination with genotype data generated by single-nucleotide polymorphism arrays and genotyping by sequencing, respectively. We resolved 13 assembly errors for a published I. trifida genome assembly and anchored eight unplaced scaffolds in the published potato genome. PolyGembler, a method for grouping and ordering contigs into complete pseudomolecules by combining long-read sequencing and genotype information from an outbred mapping population, improves the accuracy for assembly of polyploidy plant genomes.},
  archive      = {J_NG},
  author       = {Zhou, Chenxi and Olukolu, Bode and Gemenet, Dorcus C. and Wu, Shan and Gruneberg, Wolfgang and Cao, Minh Duc and Fei, Zhangjun and Zeng, Zhao-Bang and George, Andrew W. and Khan, Awais and Yencho, G. Craig and Coin, Lachlan J. M.},
  doi          = {10.1038/s41588-020-00717-7},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1256-1264},
  shortjournal = {Nat. Genet.},
  title        = {Assembly of whole-chromosome pseudomolecules for polyploid plant genomes using outbred mapping populations},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). CHESS enables quantitative comparison of chromatin contact
data and automatic feature extraction. <em>NG</em>, <em>52</em>(11),
1247–1255. (<a
href="https://doi.org/10.1038/s41588-020-00712-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamic changes in the three-dimensional (3D) organization of chromatin are associated with central biological processes, such as transcription, replication and development. Therefore, the comprehensive identification and quantification of these changes is fundamental to understanding of evolutionary and regulatory mechanisms. Here, we present Comparison of Hi-C Experiments using Structural Similarity (CHESS), an algorithm for the comparison of chromatin contact maps and automatic differential feature extraction. We demonstrate the robustness of CHESS to experimental variability and showcase its biological applications on (1) interspecies comparisons of syntenic regions in human and mouse models; (2) intraspecies identification of conformational changes in Zelda-depleted Drosophila embryos; (3) patient-specific aberrant chromatin conformation in a diffuse large B-cell lymphoma sample; and (4) the systematic identification of chromatin contact differences in high-resolution Capture-C data. In summary, CHESS is a computationally efficient method for the comparison and classification of changes in chromatin contact data. CHESS is an algorithm that compares chromatin contact maps and identifies differential features. It can analyze interspecies syntenic regions and three-dimensional changes caused by genetic perturbation.},
  archive      = {J_NG},
  author       = {Galan, Silvia and Machnik, Nick and Kruse, Kai and Díaz, Noelia and Marti-Renom, Marc A. and Vaquerizas, Juan M.},
  doi          = {10.1038/s41588-020-00712-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1247-1255},
  shortjournal = {Nat. Genet.},
  title        = {CHESS enables quantitative comparison of chromatin contact data and automatic feature extraction},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A unified framework for joint-tissue transcriptome-wide
association and mendelian randomization analysis. <em>NG</em>,
<em>52</em>(11), 1239–1246. (<a
href="https://doi.org/10.1038/s41588-020-0706-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here, we present a joint-tissue imputation (JTI) approach and a Mendelian randomization framework for causal inference, MR-JTI. JTI borrows information across transcriptomes of different tissues, leveraging shared genetic regulation, to improve prediction performance in a tissue-dependent manner. Notably, JTI includes the single-tissue imputation method PrediXcan as a special case and outperforms other single-tissue approaches (the Bayesian sparse linear mixed model and Dirichlet process regression). MR-JTI models variant-level heterogeneity (primarily due to horizontal pleiotropy, addressing a major challenge of transcriptome-wide association study interpretation) and performs causal inference with type I error control. We make explicit the connection between the genetic architecture of gene expression and of complex traits and the suitability of Mendelian randomization as a causal inference strategy for transcriptome-wide association studies. We provide a resource of imputation models generated from GTEx and PsychENCODE panels. Analysis of biobanks and meta-analysis data, and extensive simulations show substantially improved statistical power, replication and causal mapping rate for JTI relative to existing approaches. MR-JTI, a unified framework for joint-tissue imputation and Mendelian randomization, improves prediction performance in a tissue-dependent manner when applied to large-scale biobanks and meta-analysis data.},
  archive      = {J_NG},
  author       = {Zhou, Dan and Jiang, Yi and Zhong, Xue and Cox, Nancy J. and Liu, Chunyu and Gamazon, Eric R.},
  doi          = {10.1038/s41588-020-0706-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1239-1246},
  shortjournal = {Nat. Genet.},
  title        = {A unified framework for joint-tissue transcriptome-wide association and mendelian randomization analysis},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Highly interconnected enhancer communities control
lineage-determining genes in human mesenchymal stem cells. <em>NG</em>,
<em>52</em>(11), 1227–1238. (<a
href="https://doi.org/10.1038/s41588-020-0709-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adipocyte differentiation is driven by waves of transcriptional regulators that reprogram the enhancer landscape and change the wiring of the promoter interactome. Here, we use high-throughput chromosome conformation enhancer capture to interrogate the role of enhancer-to-enhancer interactions during differentiation of human mesenchymal stem cells. We find that enhancers form an elaborate network that is dynamic during differentiation and coupled with changes in enhancer activity. Transcription factors (TFs) at baited enhancers amplify TF binding at target enhancers, a phenomenon we term cross-interaction stabilization of TFs. Moreover, highly interconnected enhancers (HICE) act as integration hubs orchestrating differentiation by the formation of three-dimensional enhancer communities, inside which, HICE, and other enhancers, converge on phenotypically important gene promoters. Collectively, these results indicate that enhancer interactions play a key role in the regulation of enhancer function, and that HICE are important for both signal integration and compartmentalization of the genome. High-throughput chromosome conformation enhancer capture identifies dynamic enhancer networks that regulate differentiation of human mesenchymal stem cells. Transcription factors (TFs) at baited enhancers appear to stabilize TF binding at target enhancers.},
  archive      = {J_NG},
  author       = {Madsen, Jesper G. S. and Madsen, Maria S. and Rauch, Alexander and Traynor, Sofie and Van Hauwaert, Elvira L. and Haakonsson, Anders K. and Javierre, Biola M. and Hyldahl, Mette and Fraser, Peter and Mandrup, Susanne},
  doi          = {10.1038/s41588-020-0709-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1227-1238},
  shortjournal = {Nat. Genet.},
  title        = {Highly interconnected enhancer communities control lineage-determining genes in human mesenchymal stem cells},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Cancer therapy shapes the fitness landscape of clonal
hematopoiesis. <em>NG</em>, <em>52</em>(11), 1219–1226. (<a
href="https://doi.org/10.1038/s41588-020-00710-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acquired mutations are pervasive across normal tissues. However, understanding of the processes that drive transformation of certain clones to cancer is limited. Here we study this phenomenon in the context of clonal hematopoiesis (CH) and the development of therapy-related myeloid neoplasms (tMNs). We find that mutations are selected differentially based on exposures. Mutations in ASXL1 are enriched in current or former smokers, whereas cancer therapy with radiation, platinum and topoisomerase II inhibitors preferentially selects for mutations in DNA damage response genes (TP53, PPM1D, CHEK2). Sequential sampling provides definitive evidence that DNA damage response clones outcompete other clones when exposed to certain therapies. Among cases in which CH was previously detected, the CH mutation was present at tMN diagnosis. We identify the molecular characteristics of CH that increase risk of tMN. The increasing implementation of clinical sequencing at diagnosis provides an opportunity to identify patients at risk of tMN for prevention strategies. Environmental exposures shape patterns of selection for mutations in clonal hematopoiesis. Cancer therapies promote the growth of clones with mutations that are strongly enriched in treatment-related myeloid neoplasms.},
  archive      = {J_NG},
  author       = {Bolton, Kelly L. and Ptashkin, Ryan N. and Gao, Teng and Braunstein, Lior and Devlin, Sean M. and Kelly, Daniel and Patel, Minal and Berthon, Antonin and Syed, Aijazuddin and Yabe, Mariko and Coombs, Catherine C. and Caltabellotta, Nicole M. and Walsh, Mike and Offit, Kenneth and Stadler, Zsofia and Mandelker, Diana and Schulman, Jessica and Patel, Akshar and Philip, John and Bernard, Elsa and Gundem, Gunes and Ossa, Juan E. Arango and Levine, Max and Martinez, Juan S. Medina and Farnoud, Noushin and Glodzik, Dominik and Li, Sonya and Robson, Mark E. and Lee, Choonsik and Pharoah, Paul D. P. and Stopsack, Konrad H. and Spitzer, Barbara and Mantha, Simon and Fagin, James and Boucai, Laura and Gibson, Christopher J. and Ebert, Benjamin L. and Young, Andrew L. and Druley, Todd and Takahashi, Koichi and Gillis, Nancy and Ball, Markus and Padron, Eric and Hyman, David M. and Baselga, Jose and Norton, Larry and Gardos, Stuart and Klimek, Virginia M. and Scher, Howard and Bajorin, Dean and Paraiso, Eder and Benayed, Ryma and Arcila, Maria E. and Ladanyi, Marc and Solit, David B. and Berger, Michael F. and Tallman, Martin and Garcia-Closas, Montserrat and Chatterjee, Nilanjan and Diaz, Luis A. and Levine, Ross L. and Morton, Lindsay M. and Zehir, Ahmet and Papaemmanuil, Elli},
  doi          = {10.1038/s41588-020-00710-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1219-1226},
  shortjournal = {Nat. Genet.},
  title        = {Cancer therapy shapes the fitness landscape of clonal hematopoiesis},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Pan-cancer single-cell RNA-seq identifies recurring programs
of cellular heterogeneity. <em>NG</em>, <em>52</em>(11), 1208–1218. (<a
href="https://doi.org/10.1038/s41588-020-00726-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultured cell lines are the workhorse of cancer research, but the extent to which they recapitulate the heterogeneity observed among malignant cells in tumors is unclear. Here we used multiplexed single-cell RNA-seq to profile 198 cancer cell lines from 22 cancer types. We identified 12 expression programs that are recurrently heterogeneous within multiple cancer cell lines. These programs are associated with diverse biological processes, including cell cycle, senescence, stress and interferon responses, epithelial–mesenchymal transition and protein metabolism. Most of these programs recapitulate those recently identified as heterogeneous within human tumors. We prioritized specific cell lines as models of cellular heterogeneity and used them to study subpopulations of senescence-related cells, demonstrating their dynamics, regulation and unique drug sensitivities, which were predictive of clinical response. Our work describes the landscape of heterogeneity within diverse cancer cell lines and identifies recurrent patterns of heterogeneity that are shared between tumors and specific cell lines. Single-cell RNA-seq of a collection of 200 cancer cell lines finds common, recurrent heterogeneous expression programs, which also are found in patient samples and are linked to cell state and drug sensitivity.},
  archive      = {J_NG},
  author       = {Kinker, Gabriela S. and Greenwald, Alissa C. and Tal, Rotem and Orlova, Zhanna and Cuoco, Michael S. and McFarland, James M. and Warren, Allison and Rodman, Christopher and Roth, Jennifer A. and Bender, Samantha A. and Kumar, Bhavna and Rocco, James W. and Fernandes, Pedro A. C. M. and Mader, Christopher C. and Keren-Shaul, Hadas and Plotnikov, Alexander and Barr, Haim and Tsherniak, Aviad and Rozenblatt-Rosen, Orit and Krizhanovsky, Valery and Puram, Sidharth V. and Regev, Aviv and Tirosh, Itay},
  doi          = {10.1038/s41588-020-00726-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1208-1218},
  shortjournal = {Nat. Genet.},
  title        = {Pan-cancer single-cell RNA-seq identifies recurring programs of cellular heterogeneity},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Discovering functional evolutionary dependencies in human
cancers. <em>NG</em>, <em>52</em>(11), 1198–1207. (<a
href="https://doi.org/10.1038/s41588-020-0703-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer cells retain genomic alterations that provide a selective advantage. The prediction and validation of advantageous alterations are major challenges in cancer genomics. Moreover, it is crucial to understand how the coexistence of specific alterations alters response to genetic and therapeutic perturbations. In the present study, we inferred functional alterations and preferentially selected combinations of events in &amp;gt;9,000 human tumors. Using a Bayesian inference framework, we validated computational predictions with high-throughput readouts from genetic and pharmacological screenings on 2,000 cancer cell lines. Mutually exclusive and co-occurring cancer alterations reflected, respectively, functional redundancies able to rescue the phenotype of individual target inhibition, or synergistic interactions, increasing oncogene addiction. Among the top scoring dependencies, co-alteration of the phosphoinositide 3-kinase (PI3K) subunit PIK3CA and the nuclear factor NFE2L2 was a synergistic evolutionary trajectory in squamous cell carcinomas. By integrating computational, experimental and clinical evidence, we provide a framework to study the combinatorial functional effects of cancer genomic alterations. Computational analysis of over 9,000 cancer genomes, coupled with functional validation in cell lines, highlights combinations of mutations required for tumor progression. This integrated approach provides a framework to stratify patients on the basis of interdependent genetic aberrations.},
  archive      = {J_NG},
  author       = {Mina, Marco and Iyer, Arvind and Tavernari, Daniele and Raynaud, Franck and Ciriello, Giovanni},
  doi          = {10.1038/s41588-020-0703-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1198-1207},
  shortjournal = {Nat. Genet.},
  title        = {Discovering functional evolutionary dependencies in human cancers},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The mutational signature profile of known and suspected
human carcinogens in mice. <em>NG</em>, <em>52</em>(11), 1189–1197. (<a
href="https://doi.org/10.1038/s41588-020-0692-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epidemiological studies have identified many environmental agents that appear to significantly increase cancer risk in human populations. By analyzing tumor genomes from mice chronically exposed to 1 of 20 known or suspected human carcinogens, we reveal that most agents do not generate distinct mutational signatures or increase mutation burden, with most mutations, including driver mutations, resulting from tissue-specific endogenous processes. We identify signatures resulting from exposure to cobalt and vinylidene chloride and link distinct human signatures (SBS19 and SBS42) with 1,2,3-trichloropropane, a haloalkane and pollutant of drinking water, and find these and other signatures in human tumor genomes. We define the cross-species genomic landscape of tumors induced by an important compendium of agents with relevance to human health. A genomic analysis of tumors in mice caused by known or suspected carcinogens shows that most carcinogens do not generate distinct mutational signatures.},
  archive      = {J_NG},
  author       = {Riva, Laura and Pandiri, Arun R. and Li, Yun Rose and Droop, Alastair and Hewinson, James and Quail, Michael A. and Iyer, Vivek and Shepherd, Rebecca and Herbert, Ronald A. and Campbell, Peter J. and Sills, Robert C. and Alexandrov, Ludmil B. and Balmain, Allan and Adams, David J.},
  doi          = {10.1038/s41588-020-0692-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1189-1197},
  shortjournal = {Nat. Genet.},
  title        = {The mutational signature profile of known and suspected human carcinogens in mice},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Somatic mutation distributions in cancer genomes vary with
three-dimensional chromatin structure. <em>NG</em>, <em>52</em>(11),
1178–1188. (<a href="https://doi.org/10.1038/s41588-020-0708-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations in driver genes may ultimately lead to the development of cancer. Understanding how somatic mutations accumulate in cancer genomes and the underlying factors that generate somatic mutations is therefore crucial for developing novel therapeutic strategies. To understand the interplay between spatial genome organization and specific mutational processes, we studied 3,000 tumor–normal-pair whole-genome datasets from 42 different human cancer types. Our analyses reveal that the change in somatic mutational load in cancer genomes is co-localized with topologically-associating-domain boundaries. Domain boundaries constitute a better proxy to track mutational load change than replication timing measurements. We show that different mutational processes lead to distinct somatic mutation distributions where certain processes generate mutations in active domains, and others generate mutations in inactive domains. Overall, the interplay between three-dimensional genome organization and active mutational processes has a substantial influence on the large-scale mutation-rate variations observed in human cancers. Somatic mutational loads in cancer genomes change with chromatin domain boundaries. Different mutational processes lead to distinct somatic mutation distributions in active versus inactive domains, including during tumor evolution.},
  archive      = {J_NG},
  author       = {Akdemir, Kadir C. and Le, Victoria T. and Kim, Justin M. and Killcoyne, Sarah and King, Devin A. and Lin, Ya-Ping and Tian, Yanyan and Inoue, Akira and Amin, Samirkumar B. and Robinson, Frederick S. and Nimmakayalu, Manjunath and Herrera, Rafael E. and Lynn, Erica J. and Chan, Kin and Seth, Sahil and Klimczak, Leszek J. and Gerstung, Moritz and Gordenin, Dmitry A. and O’Brien, John and Li, Lei and Deribe, Yonathan Lissanu and Verhaak, Roel G. and Campbell, Peter J. and Fitzgerald, Rebecca and Morrison, Ashby J. and Dixon, Jesse R. and Andrew Futreal, P.},
  doi          = {10.1038/s41588-020-0708-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1178-1188},
  shortjournal = {Nat. Genet.},
  title        = {Somatic mutation distributions in cancer genomes vary with three-dimensional chromatin structure},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Population-specific and trans-ancestry genome-wide analyses
identify distinct and shared genetic risk loci for coronary artery
disease. <em>NG</em>, <em>52</em>(11), 1169–1177. (<a
href="https://doi.org/10.1038/s41588-020-0705-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To elucidate the genetics of coronary artery disease (CAD) in the Japanese population, we conducted a large-scale genome-wide association study of 168,228 individuals of Japanese ancestry (25,892 cases and 142,336 controls) with genotype imputation using a newly developed reference panel of Japanese haplotypes including 1,781 CAD cases and 2,636 controls. We detected eight new susceptibility loci and Japanese-specific rare variants contributing to disease severity and increased cardiovascular mortality. We then conducted a trans-ancestry meta-analysis and discovered 35 additional new loci. Using the meta-analysis results, we derived a polygenic risk score (PRS) for CAD, which outperformed those derived from either Japanese or European genome-wide association studies. The PRS prioritized risk factors among various clinical parameters and segregated individuals with increased risk of long-term cardiovascular mortality. Our data improve the clinical characterization of CAD genetics and suggest the utility of trans-ancestry meta-analysis for PRS derivation in non-European populations. Genome-wide association analyses in the Japanese population and trans-ancestry meta-analyses identify new risk loci for coronary artery disease. A polygenic risk score derived from these findings identifies individuals with increased risk of long-term cardiovascular mortality.},
  archive      = {J_NG},
  author       = {Koyama, Satoshi and Ito, Kaoru and Terao, Chikashi and Akiyama, Masato and Horikoshi, Momoko and Momozawa, Yukihide and Matsunaga, Hiroshi and Ieki, Hirotaka and Ozaki, Kouichi and Onouchi, Yoshihiro and Takahashi, Atsushi and Nomura, Seitaro and Morita, Hiroyuki and Akazawa, Hiroshi and Kim, Changhoon and Seo, Jeong-sun and Higasa, Koichiro and Iwasaki, Motoki and Yamaji, Taiki and Sawada, Norie and Tsugane, Shoichiro and Koyama, Teruhide and Ikezaki, Hiroaki and Takashima, Naoyuki and Tanaka, Keitaro and Arisawa, Kokichi and Kuriki, Kiyonori and Naito, Mariko and Wakai, Kenji and Suna, Shinichiro and Sakata, Yasuhiko and Sato, Hiroshi and Hori, Masatsugu and Sakata, Yasushi and Matsuda, Koichi and Murakami, Yoshinori and Aburatani, Hiroyuki and Kubo, Michiaki and Matsuda, Fumihiko and Kamatani, Yoichiro and Komuro, Issei},
  doi          = {10.1038/s41588-020-0705-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1169-1177},
  shortjournal = {Nat. Genet.},
  title        = {Population-specific and trans-ancestry genome-wide analyses identify distinct and shared genetic risk loci for coronary artery disease},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell epigenomic analyses implicate candidate causal
variants at inherited risk loci for alzheimer’s and parkinson’s
diseases. <em>NG</em>, <em>52</em>(11), 1158–1168. (<a
href="https://doi.org/10.1038/s41588-020-00721-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies of neurological diseases have identified thousands of variants associated with disease phenotypes. However, most of these variants do not alter coding sequences, making it difficult to assign their function. Here, we present a multi-omic epigenetic atlas of the adult human brain through profiling of single-cell chromatin accessibility landscapes and three-dimensional chromatin interactions of diverse adult brain regions across a cohort of cognitively healthy individuals. We developed a machine-learning classifier to integrate this multi-omic framework and predict dozens of functional SNPs for Alzheimer’s and Parkinson’s diseases, nominating target genes and cell types for previously orphaned loci from genome-wide association studies. Moreover, we dissected the complex inverted haplotype of the MAPT (encoding tau) Parkinson’s disease risk locus, identifying putative ectopic regulatory interactions in neurons that may mediate this disease association. This work expands understanding of inherited variation and provides a roadmap for the epigenomic dissection of causal regulatory variation in disease. Single-cell chromatin profiling of different brain regions identifies cell-type-specific regulatory elements, and helps to predict functional SNPs for Alzheimer’s and Parkinson’s diseases.},
  archive      = {J_NG},
  author       = {Corces, M. Ryan and Shcherbina, Anna and Kundu, Soumya and Gloudemans, Michael J. and Frésard, Laure and Granja, Jeffrey M. and Louie, Bryan H. and Eulalio, Tiffany and Shams, Shadi and Bagdatli, S. Tansu and Mumbach, Maxwell R. and Liu, Boxiang and Montine, Kathleen S. and Greenleaf, William J. and Kundaje, Anshul and Montgomery, Stephen B. and Chang, Howard Y. and Montine, Thomas J.},
  doi          = {10.1038/s41588-020-00721-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1158-1168},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell epigenomic analyses implicate candidate causal variants at inherited risk loci for alzheimer’s and parkinson’s diseases},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Regulation of single-cell genome organization into TADs and
chromatin nanodomains. <em>NG</em>, <em>52</em>(11), 1151–1157. (<a
href="https://doi.org/10.1038/s41588-020-00716-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genome folds into a hierarchy of three-dimensional structures within the nucleus. At the sub-megabase scale, chromosomes form topologically associating domains (TADs)1–4. However, how TADs fold in single cells is elusive. Here, we reveal TAD features inaccessible to cell population analysis by using super-resolution microscopy. TAD structures and physical insulation associated with their borders are variable between individual cells, yet chromatin intermingling is enriched within TADs compared to adjacent TADs in most cells. The spatial segregation of TADs is further exacerbated during cell differentiation. Favored interactions within TADs are regulated by cohesin and CTCF through distinct mechanisms: cohesin generates chromatin contacts and intermingling while CTCF prevents inter-TAD contacts. Furthermore, TADs are subdivided into discrete nanodomains, which persist in cells depleted of CTCF or cohesin, whereas disruption of nucleosome contacts alters their structural organization. Altogether, these results provide a physical basis for the folding of individual chromosomes at the nanoscale. Super-resolution microscopy identifies sub-topologically associating domain (TAD) nanodomains and intercellular heterogeneity in TAD conformation and insulation. Cohesin or CTCF depletion regulates distinct types of chromatin contacts at the TAD but not nanodomain level.},
  archive      = {J_NG},
  author       = {Szabo, Quentin and Donjon, Axelle and Jerković, Ivana and Papadopoulos, Giorgio L. and Cheutin, Thierry and Bonev, Boyan and Nora, Elphège P. and Bruneau, Benoit G. and Bantignies, Frédéric and Cavalli, Giacomo},
  doi          = {10.1038/s41588-020-00716-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1151-1157},
  shortjournal = {Nat. Genet.},
  title        = {Regulation of single-cell genome organization into TADs and chromatin nanodomains},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evidence for secondary-variant genetic burden and non-random
distribution across biological modules in a recessive ciliopathy.
<em>NG</em>, <em>52</em>(11), 1145–1150. (<a
href="https://doi.org/10.1038/s41588-020-0707-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The influence of genetic background on driver mutations is well established; however, the mechanisms by which the background interacts with Mendelian loci remain unclear. We performed a systematic secondary-variant burden analysis of two independent cohorts of patients with Bardet–Biedl syndrome (BBS) with known recessive biallelic pathogenic mutations in one of 17 BBS genes for each individual. We observed a significant enrichment of trans-acting rare nonsynonymous secondary variants in patients with BBS compared with either population controls or a cohort of individuals with a non-BBS diagnosis and recessive variants in the same gene set. Strikingly, we found a significant over-representation of secondary alleles in chaperonin-encoding genes—a finding corroborated by the observation of epistatic interactions involving this complex in vivo. These data indicate a complex genetic architecture for BBS that informs the biological properties of disease modules and presents a model for secondary-variant burden analysis in recessive disorders. Analysis of two independent cohorts of patients with Bardet–Biedl syndrome (BBS) with known recessive biallelic pathogenic mutations in one of 17 BBS genes shows an enrichment of rare nonsynonymous secondary variants in the same gene set, with significant over-representation of secondary alleles in chaperonin-encoding genes.},
  archive      = {J_NG},
  author       = {Kousi, Maria and Söylemez, Onuralp and Ozanturk, Aysegül and Mourtzi, Niki and Akle, Sebastian and Jungreis, Irwin and Muller, Jean and Cassa, Christopher A. and Brand, Harrison and Mokry, Jill Anne and Wolf, Maxim Y. and Sadeghpour, Azita and McFadden, Kelsey and Lewis, Richard A. and Talkowski, Michael E. and Dollfus, Hélène and Kellis, Manolis and Davis, Erica E. and Sunyaev, Shamil R. and Katsanis, Nicholas},
  doi          = {10.1038/s41588-020-0707-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1145-1150},
  shortjournal = {Nat. Genet.},
  title        = {Evidence for secondary-variant genetic burden and non-random distribution across biological modules in a recessive ciliopathy},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The critical roles of somatic mutations and environmental
tumor-promoting agents in cancer risk. <em>NG</em>, <em>52</em>(11),
1139–1143. (<a
href="https://doi.org/10.1038/s41588-020-00727-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer is driven by genomic mutations in ‘cancer driver’ genes, which have essential roles in tumor development. These mutations may be caused by exposure to mutagens in the environment or by endogenous DNA-replication errors in tissue stem cells. Recent observations of abundant mutations, including cancer driver mutations, in histologically normal human tissues suggest that mutations alone are not sufficient for tumor development, thus prompting the question of how single mutant cells give rise to neoplasia. In a concept supported by decades-old data from mouse tumor models, non-mutagenic tumor-promoting agents have been posited to activate the proliferation of dormant mutated cells, thus generating actively growing lesions, with the promotion stage as the rate-limiting step in tumor formation. Non-mutagenic promoting agents, either endogenous or environmental, may therefore have a more important role in human cancer etiology than previously thought. This Perspective explores the concept of tumor promotion and shows how carcinogenesis experiments performed decades ago in mice are remodeling the view of cancer initiation and prevention.},
  archive      = {J_NG},
  author       = {Balmain, Allan},
  doi          = {10.1038/s41588-020-00727-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1139-1143},
  shortjournal = {Nat. Genet.},
  title        = {The critical roles of somatic mutations and environmental tumor-promoting agents in cancer risk},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Are carcinogens direct mutagens? <em>NG</em>,
<em>52</em>(11), 1137–1138. (<a
href="https://doi.org/10.1038/s41588-020-00730-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In a classical view of carcinogenesis, carcinogens directly cause mutations. In this issue, Riva et al. explore the effects of 20 known or suspected human carcinogens and find discernible mutational signatures for only three of them.},
  archive      = {J_NG},
  author       = {Lopez-Bigas, Nuria and Gonzalez-Perez, Abel},
  doi          = {10.1038/s41588-020-00730-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1137-1138},
  shortjournal = {Nat. Genet.},
  title        = {Are carcinogens direct mutagens?},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evolutionary dependencies show paths to cancer development.
<em>NG</em>, <em>52</em>(11), 1135–1136. (<a
href="https://doi.org/10.1038/s41588-020-00728-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patterns of co-occurring and mutually exclusive mutations reveal synergistic interactions among cancer driver genes. A new study functionally confirms these interactions and builds the pairwise relationships into networks of pathway disruption that have better predictive power than specific mutations alone.},
  archive      = {J_NG},
  author       = {Taylor, Martin S.},
  doi          = {10.1038/s41588-020-00728-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1135-1136},
  shortjournal = {Nat. Genet.},
  title        = {Evolutionary dependencies show paths to cancer development},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Personal perspective: bootless. <em>NG</em>,
<em>52</em>(11), 1133–1134. (<a
href="https://doi.org/10.1038/s41588-020-00720-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Given an opportunity to chronicle the experience of traversing America as a Black man, I elect to convey a less frequently told vantage point. This is the perspective of the ‘accomplished’ bootless.},
  archive      = {J_NG},
  author       = {Ledet, Russell J.},
  doi          = {10.1038/s41588-020-00720-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1133-1134},
  shortjournal = {Nat. Genet.},
  title        = {Personal perspective: Bootless},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: A co-clinical approach identifies
mechanisms and potential therapies for androgen deprivation resistance
in prostate cancer. <em>NG</em>, <em>52</em>(10), 1132. (<a
href="https://doi.org/10.1038/s41588-020-0701-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Lunardi, Andrea and Ala, Ugo and Epping, Mirjam T. and Salmena, Leonardo and Clohessy, John G. and Webster, Kaitlyn A. and Wang, Guocan and Mazzucchelli, Roberta and Bianconi, Maristella and Stack, Edward C. and Lis, Rosina and Patnaik, Akash and Cantley, Lewis C. and Bubley, Glenn and Cordon-Cardo, Carlos and Gerald, William L. and Montironi, Rodolfo and Signoretti, Sabina and Loda, Massimo and Nardella, Caterina and Pandolfi, Pier Paolo},
  doi          = {10.1038/s41588-020-0701-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1132},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A co-clinical approach identifies mechanisms and potential therapies for androgen deprivation resistance in prostate cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Exploring the coronavirus pandemic with
the WashU virus genome browser. <em>NG</em>, <em>52</em>(10), 1132. (<a
href="https://doi.org/10.1038/s41588-020-00714-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Flynn, Jennifer A. and Purushotham, Deepak and Choudhary, Mayank N. K. and Zhuo, Xiaoyu and Fan, Changxu and Matt, Gavriel and Li, Daofeng and Wang, Ting},
  doi          = {10.1038/s41588-020-00714-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1132},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Exploring the coronavirus pandemic with the WashU virus genome browser},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Phenome-wide mendelian randomization mapping the influence
of the plasma proteome on complex diseases. <em>NG</em>,
<em>52</em>(10), 1122–1131. (<a
href="https://doi.org/10.1038/s41588-020-0682-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human proteome is a major source of therapeutic targets. Recent genetic association analyses of the plasma proteome enable systematic evaluation of the causal consequences of variation in plasma protein levels. Here we estimated the effects of 1,002 proteins on 225 phenotypes using two-sample Mendelian randomization (MR) and colocalization. Of 413 associations supported by evidence from MR, 130 (31.5%) were not supported by results of colocalization analyses, suggesting that genetic confounding due to linkage disequilibrium is widespread in naïve phenome-wide association studies of proteins. Combining MR and colocalization evidence in cis-only analyses, we identified 111 putatively causal effects between 65 proteins and 52 disease-related phenotypes ( https://www.epigraphdb.org/pqtl/ ). Evaluation of data from historic drug development programs showed that target-indication pairs with MR and colocalization support were more likely to be approved, evidencing the value of this approach in identifying and prioritizing potential therapeutic targets. Mendelian randomization (MR) and colocalization analyses are used to estimate causal effects of 1,002 plasma proteins on 225 phenotypes. Evidence from drug developmental programs shows that target-indication pairs with MR and colocalization support were more likely to be approved, highlighting the value of this approach for prioritizing therapeutic targets.},
  archive      = {J_NG},
  author       = {Zheng, Jie and Haberland, Valeriia and Baird, Denis and Walker, Venexia and Haycock, Philip C. and Hurle, Mark R. and Gutteridge, Alex and Erola, Pau and Liu, Yi and Luo, Shan and Robinson, Jamie and Richardson, Tom G. and Staley, James R. and Elsworth, Benjamin and Burgess, Stephen and Sun, Benjamin B. and Danesh, John and Runz, Heiko and Maranville, Joseph C. and Martin, Hannah M. and Yarmolinsky, James and Laurin, Charles and Holmes, Michael V. and Liu, Jimmy Z. and Estrada, Karol and Santos, Rita and McCarthy, Linda and Waterworth, Dawn and Nelson, Matthew R. and Smith, George Davey and Butterworth, Adam S. and Hemani, Gibran and Scott, Robert A. and Gaunt, Tom R.},
  doi          = {10.1038/s41588-020-0682-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1122-1131},
  shortjournal = {Nat. Genet.},
  title        = {Phenome-wide mendelian randomization mapping the influence of the plasma proteome on complex diseases},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Analysis of wild tomato introgression lines elucidates the
genetic basis of transcriptome and metabolome variation underlying fruit
traits and pathogen response. <em>NG</em>, <em>52</em>(10), 1111–1121.
(<a href="https://doi.org/10.1038/s41588-020-0690-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wild tomato species represent a rich gene pool for numerous desirable traits lost during domestication. Here, we exploited an introgression population representing wild desert-adapted species and a domesticated cultivar to establish the genetic basis of gene expression and chemical variation accompanying the transfer of wild-species-associated fruit traits. Transcriptome and metabolome analysis of 580 lines coupled to pathogen sensitivity assays resulted in the identification of genomic loci associated with levels of hundreds of transcripts and metabolites. These associations occurred in hotspots representing coordinated perturbation of metabolic pathways and ripening-related processes. Here, we identify components of the Solanum alkaloid pathway, as well as genes and metabolites involved in pathogen defense and linking fungal resistance with changes in the fruit ripening regulatory network. Our results outline a framework for understanding metabolism and pathogen resistance during tomato fruit ripening and provide insights into key fruit quality traits. A study of 580 tomato lines resulting from a cross between a wild desert-adapted species and a domesticated cultivar elucidates the genetic basis of gene expression and metabolite variation associated with fruit traits and defense against pathogens.},
  archive      = {J_NG},
  author       = {Szymański, Jędrzej and Bocobza, Samuel and Panda, Sayantan and Sonawane, Prashant and Cárdenas, Pablo D. and Lashbrooke, Justin and Kamble, Avinash and Shahaf, Nir and Meir, Sagit and Bovy, Arnaud and Beekwilder, Jules and Tikunov, Yury and Romero de la Fuente, Irene and Zamir, Dani and Rogachev, Ilana and Aharoni, Asaph},
  doi          = {10.1038/s41588-020-0690-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1111-1121},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of wild tomato introgression lines elucidates the genetic basis of transcriptome and metabolome variation underlying fruit traits and pathogen response},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The mosaic genome of indigenous african cattle as a unique
genetic resource for african pastoralism. <em>NG</em>, <em>52</em>(10),
1099–1110. (<a href="https://doi.org/10.1038/s41588-020-0694-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cattle pastoralism plays a central role in human livelihood in Africa. However, the genetic history of its success remains unknown. Here, through whole-genome sequence analysis of 172 indigenous African cattle from 16 breeds representative of the main cattle groups, we identify a major taurine × indicine cattle admixture event dated to circa 750–1,050 yr ago, which has shaped the genome of today’s cattle in the Horn of Africa. We identify 16 loci linked to African environmental adaptations across crossbred animals showing an excess of taurine or indicine ancestry. These include immune-, heat-tolerance- and reproduction-related genes. Moreover, we identify one highly divergent locus in African taurine cattle, which is putatively linked to trypanotolerance and present in crossbred cattle living in trypanosomosis-infested areas. Our findings indicate that a combination of past taurine and recent indicine admixture-derived genetic resources is at the root of the present success of African pastoralism. Whole-genome sequence analysis of 172 indigenous African cattle from 16 breeds identifies 16 loci linked to environmental adaptations among crossbred animals, including a highly divergent locus in African taurine cattle putatively linked to trypanotolerance.},
  archive      = {J_NG},
  author       = {Kim, Kwondo and Kwon, Taehyung and Dessie, Tadelle and Yoo, DongAhn and Mwai, Okeyo Ally and Jang, Jisung and Sung, Samsun and Lee, SaetByeol and Salim, Bashir and Jung, Jaehoon and Jeong, Heesu and Tarekegn, Getinet Mekuriaw and Tijjani, Abdulfatai and Lim, Dajeong and Cho, Seoae and Oh, Sung Jong and Lee, Hak-Kyo and Kim, Jaemin and Jeong, Choongwon and Kemp, Stephen and Hanotte, Olivier and Kim, Heebal},
  doi          = {10.1038/s41588-020-0694-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1099-1110},
  shortjournal = {Nat. Genet.},
  title        = {The mosaic genome of indigenous african cattle as a unique genetic resource for african pastoralism},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). NSD1-deposited H3K36me2 directs de novo methylation in the
mouse male germline and counteracts polycomb-associated silencing.
<em>NG</em>, <em>52</em>(10), 1088–1098. (<a
href="https://doi.org/10.1038/s41588-020-0689-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo DNA methylation (DNAme) in mammalian germ cells is dependent on DNMT3A and DNMT3L. However, oocytes and spermatozoa show distinct patterns of DNAme. In mouse oocytes, de novo DNAme requires the lysine methyltransferase (KMTase) SETD2, which deposits H3K36me3. We show here that SETD2 is dispensable for de novo DNAme in the male germline. Instead, the lysine methyltransferase NSD1, which broadly deposits H3K36me2 in euchromatic regions, plays a critical role in de novo DNAme in prospermatogonia, including at imprinted genes. However, males deficient in germline NSD1 show a more severe defect in spermatogenesis than Dnmt3l−/− males. Notably, unlike DNMT3L, NSD1 safeguards a subset of genes against H3K27me3-associated transcriptional silencing. In contrast, H3K36me2 in oocytes is predominantly dependent on SETD2 and coincides with H3K36me3. Furthermore, females with NSD1-deficient oocytes are fertile. Thus, the sexually dimorphic pattern of DNAme in mature mouse gametes is orchestrated by distinct profiles of H3K36 methylation. NSD1, which deposits H3K36me2, is a major regulator of DNA methylation in male but not in female gametogenesis. NSD1 safeguards against H3K27me3-associated transcriptional silencing.},
  archive      = {J_NG},
  author       = {Shirane, Kenjiro and Miura, Fumihito and Ito, Takashi and Lorincz, Matthew C.},
  doi          = {10.1038/s41588-020-0689-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1088-1098},
  shortjournal = {Nat. Genet.},
  title        = {NSD1-deposited H3K36me2 directs de novo methylation in the mouse male germline and counteracts polycomb-associated silencing},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Alteration of genome folding via contact domain boundary
insertion. <em>NG</em>, <em>52</em>(10), 1076–1087. (<a
href="https://doi.org/10.1038/s41588-020-0680-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal chromosomes are partitioned into contact domains. Pathogenic domain disruptions can result from chromosomal rearrangements or perturbation of architectural factors. However, such broad-scale alterations are insufficient to define the minimal requirements for domain formation. Moreover, to what extent domains can be engineered is just beginning to be explored. In an attempt to create contact domains, we inserted a 2-kb DNA sequence underlying a tissue-invariant domain boundary—containing a CTCF-binding site (CBS) and a transcription start site (TSS)—into 16 ectopic loci across 11 chromosomes, and characterized its architectural impact. Depending on local constraints, this fragment variably formed new domains, partitioned existing ones, altered compartmentalization and initiated contacts reflecting chromatin loop extrusion. Deletions of the CBS or the TSS individually or in combination within inserts revealed its distinct contributions to genome folding. Altogether, short DNA insertions can suffice to shape the spatial genome in a manner influenced by chromatin context. Insertion of a tissue-invariant chromatin domain boundary into 16 ectopic loci leads to various structural phenotypes, which depend on local chromatin features, CTCF binding and transcriptional status.},
  archive      = {J_NG},
  author       = {Zhang, Di and Huang, Peng and Sharma, Malini and Keller, Cheryl A. and Giardine, Belinda and Zhang, Haoyue and Gilgenast, Thomas G. and Phillips-Cremins, Jennifer E. and Hardison, Ross C. and Blobel, Gerd A.},
  doi          = {10.1038/s41588-020-0680-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1076-1087},
  shortjournal = {Nat. Genet.},
  title        = {Alteration of genome folding via contact domain boundary insertion},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Transcription imparts architecture, function and logic to
enhancer units. <em>NG</em>, <em>52</em>(10), 1067–1075. (<a
href="https://doi.org/10.1038/s41588-020-0686-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Distal enhancers play pivotal roles in development and disease yet remain one of the least understood regulatory elements. We used massively parallel reporter assays to perform functional comparisons of two leading enhancer models and find that gene-distal transcription start sites are robust predictors of active enhancers with higher resolution than histone modifications. We show that active enhancer units are precisely delineated by active transcription start sites, validate that these boundaries are sufficient for capturing enhancer function, and confirm that core promoter sequences are necessary for this activity. We assay adjacent enhancers and find that their joint activity is often driven by the stronger unit within the cluster. Finally, we validate these results through functional dissection of a distal enhancer cluster using CRISPR–Cas9 deletions. In summary, definition of high-resolution enhancer boundaries enables deconvolution of complex regulatory loci into modular units. A massively parallel reporter assay (eSTARR-seq) shows that gene-distal transcription start sites can delineate active enhancers with higher resolution than histone modifications.},
  archive      = {J_NG},
  author       = {Tippens, Nathaniel D. and Liang, Jin and Leung, Alden King-Yung and Wierbowski, Shayne D. and Ozer, Abdullah and Booth, James G. and Lis, John T. and Yu, Haiyuan},
  doi          = {10.1038/s41588-020-0686-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1067-1075},
  shortjournal = {Nat. Genet.},
  title        = {Transcription imparts architecture, function and logic to enhancer units},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evolutionary dynamics of neoantigens in growing tumors.
<em>NG</em>, <em>52</em>(10), 1057–1066. (<a
href="https://doi.org/10.1038/s41588-020-0687-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancers accumulate mutations that lead to neoantigens, novel peptides that elicit an immune response, and consequently undergo evolutionary selection. Here we establish how negative selection shapes the clonality of neoantigens in a growing cancer by constructing a mathematical model of neoantigen evolution. The model predicts that, without immune escape, tumor neoantigens are either clonal or at low frequency; hypermutated tumors can only establish after the evolution of immune escape. Moreover, the site frequency spectrum of somatic variants under negative selection appears more neutral as the strength of negative selection increases, which is consistent with classical neutral theory. These predictions are corroborated by the analysis of neoantigen frequencies and immune escape in exome and RNA sequencing data from 879 colon, stomach and endometrial cancers. Mathematical modeling of evolutionary dynamics of neoantigens and immune escape in growing tumors shows that strong negative selection for neoantigens inhibits tumor growth but also provides a strong selective pressure for the evolution of immune escape.},
  archive      = {J_NG},
  author       = {Lakatos, Eszter and Williams, Marc J. and Schenck, Ryan O. and Cross, William C. H. and Househam, Jacob and Zapata, Luis and Werner, Benjamin and Gatenbee, Chandler and Robertson-Tessi, Mark and Barnes, Chris P. and Anderson, Alexander R. A. and Sottoriva, Andrea and Graham, Trevor A.},
  doi          = {10.1038/s41588-020-0687-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1057-1066},
  shortjournal = {Nat. Genet.},
  title        = {Evolutionary dynamics of neoantigens in growing tumors},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Mutations disrupting neuritogenesis genes confer risk for
cerebral palsy. <em>NG</em>, <em>52</em>(10), 1046–1056. (<a
href="https://doi.org/10.1038/s41588-020-0695-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In addition to commonly associated environmental factors, genomic factors may cause cerebral palsy. We performed whole-exome sequencing of 250 parent–offspring trios, and observed enrichment of damaging de novo mutations in cerebral palsy cases. Eight genes had multiple damaging de novo mutations; of these, two (TUBA1A and CTNNB1) met genome-wide significance. We identified two novel monogenic etiologies, FBXO31 and RHOB, and showed that the RHOB mutation enhances active-state Rho effector binding while the FBXO31 mutation diminishes cyclin D levels. Candidate cerebral palsy risk genes overlapped with neurodevelopmental disorder genes. Network analyses identified enrichment of Rho GTPase, extracellular matrix, focal adhesion and cytoskeleton pathways. Cerebral palsy risk genes in enriched pathways were shown to regulate neuromotor function in a Drosophila reverse genetics screen. We estimate that 14% of cases could be attributed to an excess of damaging de novo or recessive variants. These findings provide evidence for genetically mediated dysregulation of early neuronal connectivity in cerebral palsy. Whole-exome sequencing of 250 parent–offspring trios identifies an enrichment of rare damaging de novo mutations in individuals with cerebral palsy and implicates genetically mediated dysregulation of early neuronal connectivity in the etiology of this disorder.},
  archive      = {J_NG},
  author       = {Jin, Sheng Chih and Lewis, Sara A. and Bakhtiari, Somayeh and Zeng, Xue and Sierant, Michael C. and Shetty, Sheetal and Nordlie, Sandra M. and Elie, Aureliane and Corbett, Mark A. and Norton, Bethany Y. and van Eyk, Clare L. and Haider, Shozeb and Guida, Brandon S. and Magee, Helen and Liu, James and Pastore, Stephen and Vincent, John B. and Brunstrom-Hernandez, Janice and Papavasileiou, Antigone and Fahey, Michael C. and Berry, Jesia G. and Harper, Kelly and Zhou, Chongchen and Zhang, Junhui and Li, Boyang and Zhao, Hongyu and Heim, Jennifer and Webber, Dani L. and Frank, Mahalia S. B. and Xia, Lei and Xu, Yiran and Zhu, Dengna and Zhang, Bohao and Sheth, Amar H. and Knight, James R. and Castaldi, Christopher and Tikhonova, Irina R. and López-Giráldez, Francesc and Keren, Boris and Whalen, Sandra and Buratti, Julien and Doummar, Diane and Cho, Megan and Retterer, Kyle and Millan, Francisca and Wang, Yangong and Waugh, Jeff L. and Rodan, Lance and Cohen, Julie S. and Fatemi, Ali and Lin, Angela E. and Phillips, John P. and Feyma, Timothy and MacLennan, Suzanna C. and Vaughan, Spencer and Crompton, Kylie E. and Reid, Susan M. and Reddihough, Dinah S. and Shang, Qing and Gao, Chao and Novak, Iona and Badawi, Nadia and Wilson, Yana A. and McIntyre, Sarah J. and Mane, Shrikant M. and Wang, Xiaoyang and Amor, David J. and Zarnescu, Daniela C. and Lu, Qiongshi and Xing, Qinghe and Zhu, Changlian and Bilguvar, Kaya and Padilla-Lopez, Sergio and Lifton, Richard P. and Gecz, Jozef and MacLennan, Alastair H. and Kruer, Michael C.},
  doi          = {10.1038/s41588-020-0695-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1046-1056},
  shortjournal = {Nat. Genet.},
  title        = {Mutations disrupting neuritogenesis genes confer risk for cerebral palsy},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Complex genetic signatures in immune cells underlie
autoimmunity and inform therapy. <em>NG</em>, <em>52</em>(10),
1036–1045. (<a href="https://doi.org/10.1038/s41588-020-0684-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report on the influence of ~22 million variants on 731 immune cell traits in a cohort of 3,757 Sardinians. We detected 122 significant (P &amp;lt; 1.28 × 10−11) independent association signals for 459 cell traits at 70 loci (53 of them novel) identifying several molecules and mechanisms involved in cell regulation. Furthermore, 53 signals at 36 loci overlapped with previously reported disease-associated signals, predominantly for autoimmune disorders, highlighting intermediate phenotypes in pathogenesis. Collectively, our findings illustrate complex genetic regulation of immune cells with highly selective effects on autoimmune disease risk at the cell-subtype level. These results identify drug-targetable pathways informing the design of more specific treatments for autoimmune diseases. An analysis of 3,757 Sardinian genomes identifies 122 association signals for 459 immune cell traits at 69 loci. Some variants are associated with autoimmune disorders.},
  archive      = {J_NG},
  author       = {Orrù, Valeria and Steri, Maristella and Sidore, Carlo and Marongiu, Michele and Serra, Valentina and Olla, Stefania and Sole, Gabriella and Lai, Sandra and Dei, Mariano and Mulas, Antonella and Virdis, Francesca and Piras, Maria Grazia and Lobina, Monia and Marongiu, Mara and Pitzalis, Maristella and Deidda, Francesca and Loizedda, Annalisa and Onano, Stefano and Zoledziewska, Magdalena and Sawcer, Stephen and Devoto, Marcella and Gorospe, Myriam and Abecasis, Gonçalo R. and Floris, Matteo and Pala, Mauro and Schlessinger, David and Fiorillo, Edoardo and Cucca, Francesco},
  doi          = {10.1038/s41588-020-0684-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1036-1045},
  shortjournal = {Nat. Genet.},
  title        = {Complex genetic signatures in immune cells underlie autoimmunity and inform therapy},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). An integrated multi-omics approach identifies epigenetic
alterations associated with alzheimer’s disease. <em>NG</em>,
<em>52</em>(10), 1024–1035. (<a
href="https://doi.org/10.1038/s41588-020-0696-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein aggregation is the hallmark of neurodegeneration, but the molecular mechanisms underlying late-onset Alzheimer’s disease (AD) are unclear. Here we integrated transcriptomic, proteomic and epigenomic analyses of postmortem human brains to identify molecular pathways involved in AD. RNA sequencing analysis revealed upregulation of transcription- and chromatin-related genes, including the histone acetyltransferases for H3K27ac and H3K9ac. An unbiased proteomic screening singled out H3K27ac and H3K9ac as the main enrichments specific to AD. In turn, epigenomic profiling revealed gains in the histone H3 modifications H3K27ac and H3K9ac linked to transcription, chromatin and disease pathways in AD. Increasing genome-wide H3K27ac and H3K9ac in a fly model of AD exacerbated amyloid-β42-driven neurodegeneration. Together, these findings suggest that AD involves a reconfiguration of the epigenome, wherein H3K27ac and H3K9ac affect disease pathways by dysregulating transcription- and chromatin–gene feedback loops. The identification of this process highlights potential epigenetic strategies for early-stage disease treatment. Multi-omic profiling of brain tissue from patients with Alzheimer’s disease (AD) identifies gains in H3K27ac and H3K9ac linked to transcription and disease pathways. Increasing H3K27ac and H3K9ac in a fly model of AD exacerbates neurodegeneration.},
  archive      = {J_NG},
  author       = {Nativio, Raffaella and Lan, Yemin and Donahue, Greg and Sidoli, Simone and Berson, Amit and Srinivasan, Ananth R. and Shcherbakova, Oksana and Amlie-Wolf, Alexandre and Nie, Ji and Cui, Xiaolong and He, Chuan and Wang, Li-San and Garcia, Benjamin A. and Trojanowski, John Q. and Bonini, Nancy M. and Berger, Shelley L.},
  doi          = {10.1038/s41588-020-0696-0},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1024-1035},
  shortjournal = {Nat. Genet.},
  title        = {An integrated multi-omics approach identifies epigenetic alterations associated with alzheimer’s disease},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Haplotype-resolved genome analyses of a heterozygous diploid
potato. <em>NG</em>, <em>52</em>(10), 1018–1023. (<a
href="https://doi.org/10.1038/s41588-020-0699-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Potato (Solanum tuberosum L.) is the most important tuber crop worldwide. Efforts are underway to transform the crop from a clonally propagated tetraploid into a seed-propagated, inbred-line-based hybrid, but this process requires a better understanding of potato genome. Here, we report the 1.67-Gb haplotype-resolved assembly of a diploid potato, RH89-039-16, using a combination of multiple sequencing strategies, including circular consensus sequencing. Comparison of the two haplotypes revealed ~2.1% intragenomic diversity, including 22,134 predicted deleterious mutations in 10,642 annotated genes. In 20,583 pairs of allelic genes, 16.6% and 30.8% exhibited differential expression and methylation between alleles, respectively. Deleterious mutations and differentially expressed alleles were dispersed throughout both haplotypes, complicating strategies to eradicate deleterious alleles or stack beneficial alleles via meiotic recombination. This study offers a holistic view of the genome organization of a clonally propagated diploid species and provides insights into technological evolution in resolving complex genomes. Haplotype-resolved assembly of a heterozygous diploid potato, RH89-039-16, and identification of deleterious mutations provide insights into the genome organization and breeding of a clonally propagated diploid species.},
  archive      = {J_NG},
  author       = {Zhou, Qian and Tang, Dié and Huang, Wu and Yang, Zhongmin and Zhang, Yu and Hamilton, John P. and Visser, Richard G. F. and Bachem, Christian W. B. and Robin Buell, C. and Zhang, Zhonghua and Zhang, Chunzhi and Huang, Sanwen},
  doi          = {10.1038/s41588-020-0699-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1018-1023},
  shortjournal = {Nat. Genet.},
  title        = {Haplotype-resolved genome analyses of a heterozygous diploid potato},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Chromatin binding of FOXA1 is promoted by LSD1-mediated
demethylation in prostate cancer. <em>NG</em>, <em>52</em>(10),
1011–1017. (<a href="https://doi.org/10.1038/s41588-020-0681-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {FOXA1 functions as a pioneer transcription factor by facilitating the access to chromatin for steroid hormone receptors, such as androgen receptor and estrogen receptor1–4, but mechanisms regulating its binding to chromatin remain elusive. LSD1 (KDM1A) acts as a transcriptional repressor by demethylating mono/dimethylated histone H3 lysine 4 (H3K4me1/2)5,6, but also acts as a steroid hormone receptor coactivator through mechanisms that are unclear. Here we show, in prostate cancer cells, that LSD1 associates with FOXA1 and active enhancer markers, and that LSD1 inhibition globally disrupts FOXA1 chromatin binding. Mechanistically, we demonstrate that LSD1 positively regulates FOXA1 binding by demethylating lysine 270, adjacent to the wing2 region of the FOXA1 DNA-binding domain. Acting through FOXA1, LSD1 inhibition broadly disrupted androgen-receptor binding and its transcriptional output, and dramatically decreased prostate cancer growth alone and in synergy with androgen-receptor antagonist treatment in vivo. These mechanistic insights suggest new therapeutic strategies in steroid-driven cancers. LSD1 promotes FOXA1 chromatin binding by demethylating lysine 270 of FOXA1 in prostate cancer cells. LSD1 inhibition decreases growth of prostate cancer cells.},
  archive      = {J_NG},
  author       = {Gao, Shuai and Chen, Sujun and Han, Dong and Wang, Zifeng and Li, Muqing and Han, Wanting and Besschetnova, Anna and Liu, Mingyu and Zhou, Feng and Barrett, David and Luong, My Phu and Owiredu, Jude and Liang, Yi and Ahmed, Musaddeque and Petricca, Jessica and Patalano, Susan and Macoska, Jill A. and Corey, Eva and Chen, Sen and Balk, Steven P. and He, Housheng Hansen and Cai, Changmeng},
  doi          = {10.1038/s41588-020-0681-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1011-1017},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin binding of FOXA1 is promoted by LSD1-mediated demethylation in prostate cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Treating medical data as a durable asset. <em>NG</em>,
<em>52</em>(10), 1005–1010. (<a
href="https://doi.org/10.1038/s41588-020-0698-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Access to medical data is central for conducting research on genomics. However, to tap these metadata (observable traits and phenotypes, diagnoses and medication, and labels), researchers must grapple with the complex and sensitive nature of the information. In this Perspective, we argue that, at this exciting time for genomics and artificial intelligence, several critical aspects of data generation, infrastructure and management are pillars of a modern data ecosystem. Many risks to privacy and many obstacles to medical research can be eliminated or mitigated by new secure data analytics. Finally, we discuss the potential consequences of medical data exiting the institutions and being managed by individuals. These shifts in data ownership have the potential for profound disruption and opportunity across many fields. This Perspective discusses important aspects of data generation, infrastructure and management that affect how the research community uses medical data, including genetic and genomic information.},
  archive      = {J_NG},
  author       = {Telenti, Amalio and Jiang, Xiaoqian},
  doi          = {10.1038/s41588-020-0698-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1005-1010},
  shortjournal = {Nat. Genet.},
  title        = {Treating medical data as a durable asset},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Setting new boundaries with transcription and CTCF.
<em>NG</em>, <em>52</em>(10), 1003–1004. (<a
href="https://doi.org/10.1038/s41588-020-0704-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How do boundary elements divide chromosomes into domains? A new study uses random genomic insertions to show how small genomic fragments can shape chromatin folding through the interplay of loop extrusion and compartmentalization. Spoiler: context matters.},
  archive      = {J_NG},
  author       = {Anderson, Erika C. and Nora, Elphège P.},
  doi          = {10.1038/s41588-020-0704-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1003-1004},
  shortjournal = {Nat. Genet.},
  title        = {Setting new boundaries with transcription and CTCF},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Targeting LSD1 and FOXA1 in prostate cancer. <em>NG</em>,
<em>52</em>(10), 1002–1003. (<a
href="https://doi.org/10.1038/s41588-020-0702-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {FOXA1 is a key pioneer factor in androgen-receptor activity but has been an elusive drug target. A new study shows that inhibition of the associated cofactor LSD1 modifies the methylation state of FOXA1, thus resulting in chromatin dissociation and tumor inhibition, even in models of treatment-resistant prostate cancer.},
  archive      = {J_NG},
  author       = {Omarjee, Soleilmane and Carroll, Jason S.},
  doi          = {10.1038/s41588-020-0702-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1002-1003},
  shortjournal = {Nat. Genet.},
  title        = {Targeting LSD1 and FOXA1 in prostate cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Exploring the structural distribution of genetic variation
in SARS-CoV-2 with the COVID-3D online resource. <em>NG</em>,
<em>52</em>(10), 999–1001. (<a
href="https://doi.org/10.1038/s41588-020-0693-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of the COVID-19 pandemic has spurred a global rush to uncover basic biological mechanisms to inform effective vaccine and drug development. Despite the novelty of the virus, global sequencing efforts have already identified genomic variation across isolates. To enable easy exploration and spatial visualization of the potential implications of SARS-CoV-2 mutations in infection, host immunity and drug development, we have developed COVID-3D ( http://biosig.unimelb.edu.au/covid3d/ ).},
  archive      = {J_NG},
  author       = {Portelli, Stephanie and Olshansky, Moshe and Rodrigues, Carlos H. M. and D’Souza, Elston N. and Myung, Yoochan and Silk, Michael and Alavi, Azadeh and Pires, Douglas E. V. and Ascher, David B.},
  doi          = {10.1038/s41588-020-0693-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {999-1001},
  shortjournal = {Nat. Genet.},
  title        = {Exploring the structural distribution of genetic variation in SARS-CoV-2 with the COVID-3D online resource},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The UCSC SARS-CoV-2 genome browser. <em>NG</em>,
<em>52</em>(10), 991–998. (<a
href="https://doi.org/10.1038/s41588-020-0700-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The UCSC SARS-CoV-2 Genome Browser ( https://genome.ucsc.edu/covid19.html ) is an adaptation of our popular genome-browser visualization tool for this virus, containing many annotation tracks and new features, including conservation with similar viruses, immune epitopes, RT–PCR and sequencing primers and CRISPR guides. We invite all investigators to contribute to this resource to accelerate research and development activities globally.},
  archive      = {J_NG},
  author       = {Fernandes, Jason D. and Hinrichs, Angie S. and Clawson, Hiram and Gonzalez, Jairo Navarro and Lee, Brian T. and Nassar, Luis R. and Raney, Brian J. and Rosenbloom, Kate R. and Nerli, Santrupti and Rao, Arjun A. and Schmelter, Daniel and Fyfe, Alastair and Maulding, Nathan and Zweig, Ann S. and Lowe, Todd M. and Ares, Manuel and Corbet-Detig, Russ and Kent, W. James and Haussler, David and Haeussler, Maximilian},
  doi          = {10.1038/s41588-020-0700-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {991-998},
  shortjournal = {Nat. Genet.},
  title        = {The UCSC SARS-CoV-2 genome browser},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Exploring the coronavirus pandemic with the WashU virus
genome browser. <em>NG</em>, <em>52</em>(10), 986–991. (<a
href="https://doi.org/10.1038/s41588-020-0697-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The WashU Virus Genome Browser is a web-based portal for efficient visualization of viral ‘omics’ data in the context of a variety of annotation tracks and host infection responses. The browser features both a phylogenetic-tree-based view and a genomic-coordinate, track-based view in which users can analyze the sequence features of viral genomes, sequence diversity among viral strains, genomic sites of diagnostic tests, predicted immunogenic epitopes and a continuously updated repository of publicly available genomic datasets.},
  archive      = {J_NG},
  author       = {Flynn, Jennifer A. and Purushotham, Deepak and Choudhary, Mayank N. K. and Zhuo, Xiaoyu and Fan, Changxu and Matt, Gavriel and Li, Daofeng and Wang, Ting},
  doi          = {10.1038/s41588-020-0697-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {986-991},
  shortjournal = {Nat. Genet.},
  title        = {Exploring the coronavirus pandemic with the WashU virus genome browser},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Stories in the DNA. <em>NG</em>, <em>52</em>(10), 985. (<a
href="https://doi.org/10.1038/s41588-020-00719-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic tools can help uncover evolutionary histories, migration patterns and admixture events of domesticated animals and their wild ancestors. The genetic window into the past can help shape breeding strategies and inform animal agricultural practices that should lead to a more resilient and sustainable future.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-00719-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {985},
  shortjournal = {Nat. Genet.},
  title        = {Stories in the DNA},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Aberrant ERG expression cooperates with
loss of PTEN to promote cancer progression in the prostate. <em>NG</em>,
<em>52</em>(9), 984. (<a
href="https://doi.org/10.1038/s41588-020-0688-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Carver, Brett S. and Tran, Jennifer and Gopalan, Anuradha and Chen, Zhenbang and Shaikh, Safa and Carracedo, Arkaitz and Alimonti, Andrea and Nardella, Caterina and Varmeh, Shohreh and Scardino, Peter T. and Cordon-Cardo, Carlos and Gerald, William and Pandolfi, Pier Paolo},
  doi          = {10.1038/s41588-020-0688-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {984},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Aberrant ERG expression cooperates with loss of PTEN to promote cancer progression in the prostate},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Dynamic incorporation of multiple in silico functional
annotations empowers rare variant association analysis of large
whole-genome sequencing studies at scale. <em>NG</em>, <em>52</em>(9),
969–983. (<a href="https://doi.org/10.1038/s41588-020-0676-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale whole-genome sequencing studies have enabled the analysis of rare variants (RVs) associated with complex phenotypes. Commonly used RV association tests have limited scope to leverage variant functions. We propose STAAR (variant-set test for association using annotation information), a scalable and powerful RV association test method that effectively incorporates both variant categories and multiple complementary annotations using a dynamic weighting scheme. For the latter, we introduce ‘annotation principal components’, multidimensional summaries of in silico variant annotations. STAAR accounts for population structure and relatedness and is scalable for analyzing very large cohort and biobank whole-genome sequencing studies of continuous and dichotomous traits. We applied STAAR to identify RVs associated with four lipid traits in 12,316 discovery and 17,822 replication samples from the Trans-Omics for Precision Medicine Program. We discovered and replicated new RV associations, including disruptive missense RVs of NPC1L1 and an intergenic region near APOC1P1 associated with low-density lipoprotein cholesterol. STAAR is a powerful rare variant association test that incorporates variant functional categories and complementary functional annotations using a dynamic weighting scheme based on annotation principal components. STAAR accounts for population structure and relatedness and is scalable for analyzing large whole-genome sequencing studies.},
  archive      = {J_NG},
  author       = {Li, Xihao and Li, Zilin and Zhou, Hufeng and Gaynor, Sheila M. and Liu, Yaowu and Chen, Han and Sun, Ryan and Dey, Rounak and Arnett, Donna K. and Aslibekyan, Stella and Ballantyne, Christie M. and Bielak, Lawrence F. and Blangero, John and Boerwinkle, Eric and Bowden, Donald W. and Broome, Jai G. and Conomos, Matthew P. and Correa, Adolfo and Cupples, L. Adrienne and Curran, Joanne E. and Freedman, Barry I. and Guo, Xiuqing and Hindy, George and Irvin, Marguerite R. and Kardia, Sharon L. R. and Kathiresan, Sekar and Khan, Alyna T. and Kooperberg, Charles L. and Laurie, Cathy C. and Liu, X. Shirley and Mahaney, Michael C. and Manichaikul, Ani W. and Martin, Lisa W. and Mathias, Rasika A. and McGarvey, Stephen T. and Mitchell, Braxton D. and Montasser, May E. and Moore, Jill E. and Morrison, Alanna C. and O’Connell, Jeffrey R. and Palmer, Nicholette D. and Pampana, Akhil and Peralta, Juan M. and Peyser, Patricia A. and Psaty, Bruce M. and Redline, Susan and Rice, Kenneth M. and Rich, Stephen S. and Smith, Jennifer A. and Tiwari, Hemant K. and Tsai, Michael Y. and Vasan, Ramachandran S. and Wang, Fei Fei and Weeks, Daniel E. and Weng, Zhiping and Wilson, James G. and Yanek, Lisa R. and Neale, Benjamin M. and Sunyaev, Shamil R. and Abecasis, Gonçalo R. and Rotter, Jerome I. and Willer, Cristen J. and Peloso, Gina M. and Natarajan, Pradeep and Lin, Xihong},
  doi          = {10.1038/s41588-020-0676-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {969-983},
  shortjournal = {Nat. Genet.},
  title        = {Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DNA mismatch repair promotes APOBEC3-mediated diffuse
hypermutation in human cancers. <em>NG</em>, <em>52</em>(9), 958–968.
(<a href="https://doi.org/10.1038/s41588-020-0674-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Certain mutagens, including the APOBEC3 (A3) cytosine deaminase enzymes, can create multiple genetic changes in a single event. Activity of A3s results in striking ‘mutation showers’ occurring near DNA breakpoints; however, less is known about the mechanisms underlying the majority of A3 mutations. We classified the diverse patterns of clustered mutagenesis in tumor genomes, which identified a new A3 pattern: nonrecurrent, diffuse hypermutation (omikli). This mechanism occurs independently of the known focal hypermutation (kataegis), and is associated with activity of the DNA mismatch-repair pathway, which can provide the single-stranded DNA substrate needed by A3, and contributes to a substantial proportion of A3 mutations genome wide. Because mismatch repair is directed towards early-replicating, gene-rich chromosomal domains, A3 mutagenesis has a high propensity to generate impactful mutations, which exceeds that of other common carcinogens such as tobacco smoke and ultraviolet exposure. Cells direct their DNA repair capacity towards more important genomic regions; thus, carcinogens that subvert DNA repair can be remarkably potent. A new statistical framework to classify mutagenesis clusters identifies a novel, diffuse hypermutation pattern, named omikli, that is induced by APOBEC3 and associated with mismatch-repair activity.},
  archive      = {J_NG},
  author       = {Mas-Ponte, David and Supek, Fran},
  doi          = {10.1038/s41588-020-0674-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {958-968},
  shortjournal = {Nat. Genet.},
  title        = {DNA mismatch repair promotes APOBEC3-mediated diffuse hypermutation in human cancers},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). European maize genomes highlight intraspecies variation in
repeat and gene content. <em>NG</em>, <em>52</em>(9), 950–957. (<a
href="https://doi.org/10.1038/s41588-020-0671-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The diversity of maize (Zea mays) is the backbone of modern heterotic patterns and hybrid breeding. Historically, US farmers exploited this variability to establish today’s highly productive Corn Belt inbred lines from blends of dent and flint germplasm pools. Here, we report de novo genome sequences of four European flint lines assembled to pseudomolecules with scaffold N50 ranging from 6.1 to 10.4 Mb. Comparative analyses with two US Corn Belt lines explains the pronounced differences between both germplasms. While overall syntenic order and consolidated gene annotations reveal only moderate pangenomic differences, whole-genome alignments delineating the core and dispensable genome, and the analysis of heterochromatic knobs and orthologous long terminal repeat retrotransposons unveil the dynamics of the maize genome. The high-quality genome sequences of the flint pool complement the maize pangenome and provide an important tool to study maize improvement at a genome scale and to enhance modern hybrid breeding. De novo genome assemblies of four European flint maize lines and comparison with two US Corn Belt genomes provide insights into the dynamics of intraspecies variation in repeat and gene content in maize genomes.},
  archive      = {J_NG},
  author       = {Haberer, Georg and Kamal, Nadia and Bauer, Eva and Gundlach, Heidrun and Fischer, Iris and Seidel, Michael A. and Spannagl, Manuel and Marcon, Caroline and Ruban, Alevtina and Urbany, Claude and Nemri, Adnane and Hochholdinger, Frank and Ouzunova, Milena and Houben, Andreas and Schön, Chris-Carolin and Mayer, Klaus F. X.},
  doi          = {10.1038/s41588-020-0671-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {950-957},
  shortjournal = {Nat. Genet.},
  title        = {European maize genomes highlight intraspecies variation in repeat and gene content},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genetic analyses support the contribution of mRNA
n6-methyladenosine (m6A) modification to human disease heritability.
<em>NG</em>, <em>52</em>(9), 939–949. (<a
href="https://doi.org/10.1038/s41588-020-0644-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {N6-methyladenosine (m6A) plays important roles in regulating messenger RNA processing. Despite rapid progress in this field, little is known about the genetic determinants of m6A modification and their role in common diseases. In this study, we mapped the quantitative trait loci (QTLs) of m6A peaks in 60 Yoruba (YRI) lymphoblastoid cell lines. We found that m6A QTLs are largely independent of expression and splicing QTLs and are enriched with binding sites of RNA-binding proteins, RNA structure-changing variants and transcriptional features. Joint analysis of the QTLs of m6A and related molecular traits suggests that the downstream effects of m6A are heterogeneous and context dependent. We identified proteins that mediate m6A effects on translation. Through integration with data from genome-wide association studies, we show that m6A QTLs contribute to the heritability of various immune and blood-related traits at levels comparable to splicing QTLs and roughly half of expression QTLs. By leveraging m6A QTLs in a transcriptome-wide association study framework, we identified putative risk genes of these traits. Quantitative trait locus (QTL) mapping of N6-methyladenosine (m6A) in human cells highlights the role of RNA-binding proteins, RNA secondary structure and context-dependent m6A effects. m6A QTLs are enriched in loci associated with immune and blood-related traits.},
  archive      = {J_NG},
  author       = {Zhang, Zijie and Luo, Kaixuan and Zou, Zhongyu and Qiu, Maguanyun and Tian, Jiakun and Sieh, Laura and Shi, Hailing and Zou, Yuxin and Wang, Gao and Morrison, Jean and Zhu, Allen C. and Qiao, Min and Li, Zhongshan and Stephens, Matthew and He, Xin and He, Chuan},
  doi          = {10.1038/s41588-020-0644-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {939-949},
  shortjournal = {Nat. Genet.},
  title        = {Genetic analyses support the contribution of mRNA n6-methyladenosine (m6A) modification to human disease heritability},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). RNA is essential for PRC2 chromatin occupancy and function
in human pluripotent stem cells. <em>NG</em>, <em>52</em>(9), 931–938.
(<a href="https://doi.org/10.1038/s41588-020-0662-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many chromatin-binding proteins and protein complexes that regulate transcription also bind RNA. One of these, Polycomb repressive complex 2 (PRC2), deposits the H3K27me3 mark of facultative heterochromatin and is required for stem cell differentiation. PRC2 binds RNAs broadly in vivo and in vitro. Yet, the biological importance of this RNA binding remains unsettled. Here, we tackle this question in human induced pluripotent stem cells by using multiple complementary approaches. Perturbation of RNA–PRC2 interaction by RNase A, by a chemical inhibitor of transcription or by an RNA-binding-defective mutant all disrupted PRC2 chromatin occupancy and localization genome wide. The physiological relevance of PRC2–RNA interactions is further underscored by a cardiomyocyte differentiation defect upon genetic disruption. We conclude that PRC2 requires RNA binding for chromatin localization in human pluripotent stem cells and in turn for defining cellular state. Perturbation of RNA–PRC2 interaction in human pluripotent stem cells disrupts PRC2 chromatin occupancy and localization genome wide. PRC2–RNA interactions contribute to cardiomyocyte differentiation.},
  archive      = {J_NG},
  author       = {Long, Yicheng and Hwang, Taeyoung and Gooding, Anne R. and Goodrich, Karen J. and Rinn, John L. and Cech, Thomas R.},
  doi          = {10.1038/s41588-020-0662-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {931-938},
  shortjournal = {Nat. Genet.},
  title        = {RNA is essential for PRC2 chromatin occupancy and function in human pluripotent stem cells},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Cross-species chromatin interactions drive transcriptional
rewiring in epstein–barr virus–positive gastric adenocarcinoma.
<em>NG</em>, <em>52</em>(9), 919–930. (<a
href="https://doi.org/10.1038/s41588-020-0665-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epstein–Barr virus (EBV) is associated with several human malignancies including 8–10% of gastric cancers (GCs). Genome-wide analysis of 3D chromatin topologies across GC lines, primary tissue and normal gastric samples revealed chromatin domains specific to EBV-positive GC, exhibiting heterochromatin-to-euchromatin transitions and long-range human–viral interactions with non-integrated EBV episomes. EBV infection in vitro suffices to remodel chromatin topology and function at EBV-interacting host genomic loci, converting H3K9me3+ heterochromatin to H3K4me1+/H3K27ac+ bivalency and unleashing latent enhancers to engage and activate nearby GC-related genes (for example TGFBR2 and MZT1). Higher-order epigenotypes of EBV-positive GC thus signify a novel oncogenic paradigm whereby non-integrative viral genomes can directly alter host epigenetic landscapes (‘enhancer infestation’), facilitating proto-oncogene activation and tumorigenesis. Genome-wide analysis of 3D chromatin topologies across gastric cancers suggests that Epstein–Barr virus infection may induce the epigenetic rewiring of EBV-positive tumors through human–viral chromatin interactions, a phenomenon termed ‘enhancer infestation’.},
  archive      = {J_NG},
  author       = {Okabe, Atsushi and Huang, Kie Kyon and Matsusaka, Keisuke and Fukuyo, Masaki and Xing, Manjie and Ong, Xuewen and Hoshii, Takayuki and Usui, Genki and Seki, Motoaki and Mano, Yasunobu and Rahmutulla, Bahityar and Kanda, Teru and Suzuki, Takayoshi and Rha, Sun Young and Ushiku, Tetsuo and Fukayama, Masashi and Tan, Patrick and Kaneda, Atsushi},
  doi          = {10.1038/s41588-020-0665-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {919-930},
  shortjournal = {Nat. Genet.},
  title        = {Cross-species chromatin interactions drive transcriptional rewiring in Epstein–Barr virus–positive gastric adenocarcinoma},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell transcriptomics identifies a distinct luminal
progenitor cell type in distal prostate invagination tips. <em>NG</em>,
<em>52</em>(9), 908–918. (<a
href="https://doi.org/10.1038/s41588-020-0642-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of prostate stem/progenitor cells and characterization of the prostate epithelial cell lineage hierarchy are critical for understanding prostate cancer initiation. Here, we characterized 35,129 cells from mouse prostates, and identified a unique luminal cell type (termed type C luminal cell (Luminal-C)) marked by Tacstd2, Ck4 and Psca expression. Luminal-C cells located at the distal prostate invagination tips (termed Dist-Luminal-C) exhibited greater capacity for organoid formation in vitro and prostate epithelial duct regeneration in vivo. Lineage tracing of Luminal-C cells indicated that Dist-Luminal-C cells reconstituted distal prostate luminal lineages through self-renewal and differentiation. Deletion of Pten in Dist-Luminal-C cells resulted in prostatic intraepithelial neoplasia. We further characterized 11,374 human prostate cells and confirmed the existence of h-Luminal-C cells. Our study provides insights into the prostate lineage hierarchy, identifies Dist-Luminal-C cells as the luminal progenitor cell population in invagination tips and suggests one of the potential cellular origins of prostate cancer. Single-cell RNA-seq and in vivo lineage tracing identify unique luminal progenitors in the mouse prostate that can contribute to regeneration and oncogenesis. Single-cell RNA-seq analysis of human prostate identifies a similar cell population.},
  archive      = {J_NG},
  author       = {Guo, Wangxin and Li, Lin and He, Juan and Liu, Zhuang and Han, Ming and Li, Fei and Xia, Xinyi and Zhang, Xiaoyu and Zhu, Yao and Wei, Yu and Li, Yunguang and Aji, Rebiguli and Dai, Hao and Wei, Hui and Li, Chunfeng and Chen, Yu and Chen, Luonan and Gao, Dong},
  doi          = {10.1038/s41588-020-0642-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {908-918},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell transcriptomics identifies a distinct luminal progenitor cell type in distal prostate invagination tips},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Subclonal reconstruction of tumors by using machine learning
and population genetics. <em>NG</em>, <em>52</em>(9), 898–907. (<a
href="https://doi.org/10.1038/s41588-020-0675-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most cancer genomic data are generated from bulk samples composed of mixtures of cancer subpopulations, as well as normal cells. Subclonal reconstruction methods based on machine learning aim to separate those subpopulations in a sample and infer their evolutionary history. However, current approaches are entirely data driven and agnostic to evolutionary theory. We demonstrate that systematic errors occur in the analysis if evolution is not accounted for, and this is exacerbated with multi-sampling of the same tumor. We present a novel approach for model-based tumor subclonal reconstruction, called MOBSTER, which combines machine learning with theoretical population genetics. Using public whole-genome sequencing data from 2,606 samples from different cohorts, new data and synthetic validation, we show that this method is more robust and accurate than current techniques in single-sample, multiregion and longitudinal data. This approach minimizes the confounding factors of nonevolutionary methods, thus leading to more accurate recovery of the evolutionary history of human cancers. MOBSTER is an approach for subclonal reconstruction of tumors from cancer genomics data on the basis of models that combine machine learning with evolutionary theory, thus leading to more accurate evolutionary histories of tumors.},
  archive      = {J_NG},
  author       = {Caravagna, Giulio and Heide, Timon and Williams, Marc J. and Zapata, Luis and Nichol, Daniel and Chkhaidze, Ketevan and Cross, William and Cresswell, George D. and Werner, Benjamin and Acar, Ahmet and Chesler, Louis and Barnes, Chris P. and Sanguinetti, Guido and Graham, Trevor A. and Sottoriva, Andrea},
  doi          = {10.1038/s41588-020-0675-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {898-907},
  shortjournal = {Nat. Genet.},
  title        = {Subclonal reconstruction of tumors by using machine learning and population genetics},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Extrachromosomal DNA is associated with oncogene
amplification and poor outcome across multiple cancers. <em>NG</em>,
<em>52</em>(9), 891–897. (<a
href="https://doi.org/10.1038/s41588-020-0678-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.},
  archive      = {J_NG},
  author       = {Kim, Hoon and Nguyen, Nam-Phuong and Turner, Kristen and Wu, Sihan and Gujar, Amit D. and Luebeck, Jens and Liu, Jihe and Deshpande, Viraj and Rajkumar, Utkrisht and Namburi, Sandeep and Amin, Samirkumar B. and Yi, Eunhee and Menghi, Francesca and Schulte, Johannes H. and Henssen, Anton G. and Chang, Howard Y. and Beck, Christine R. and Mischel, Paul S. and Bafna, Vineet and Verhaak, Roel G. W.},
  doi          = {10.1038/s41588-020-0678-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {891-897},
  shortjournal = {Nat. Genet.},
  title        = {Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). APOBEC3-dependent kataegis and TREX1-driven chromothripsis
during telomere crisis. <em>NG</em>, <em>52</em>(9), 884–890. (<a
href="https://doi.org/10.1038/s41588-020-0667-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromothripsis and kataegis are frequently observed in cancer and may arise from telomere crisis, a period of genome instability during tumorigenesis when depletion of the telomere reserve generates unstable dicentric chromosomes1–5. Here we examine the mechanism underlying chromothripsis and kataegis by using an in vitro telomere crisis model. We show that the cytoplasmic exonuclease TREX1, which promotes the resolution of dicentric chromosomes4, plays a prominent role in chromothriptic fragmentation. In the absence of TREX1, the genome alterations induced by telomere crisis primarily involve breakage–fusion–bridge cycles and simple genome rearrangements rather than chromothripsis. Furthermore, we show that the kataegis observed at chromothriptic breakpoints is the consequence of cytosine deamination by APOBEC3B. These data reveal that chromothripsis and kataegis arise from a combination of nucleolytic processing by TREX1 and cytosine editing by APOBEC3B. Nucleic acid processing by the cytoplasmic exonuclease TREX1 and cytosine editing by APOBEC3B drive chromothripsis and kataegis during telomere crisis.},
  archive      = {J_NG},
  author       = {Maciejowski, John and Chatzipli, Aikaterini and Dananberg, Alexandra and Chu, Kevan and Toufektchan, Eleonore and Klimczak, Leszek J. and Gordenin, Dmitry A. and Campbell, Peter J. and de Lange, Titia},
  doi          = {10.1038/s41588-020-0667-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {884-890},
  shortjournal = {Nat. Genet.},
  title        = {APOBEC3-dependent kataegis and TREX1-driven chromothripsis during telomere crisis},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Landscape of g-quadruplex DNA structural regions in breast
cancer. <em>NG</em>, <em>52</em>(9), 878–883. (<a
href="https://doi.org/10.1038/s41588-020-0672-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Response and resistance to anticancer therapies vary due to intertumor and intratumor heterogeneity1. Here, we map differentially enriched G-quadruplex (G4) DNA structure-forming regions (∆G4Rs) in 22 breast cancer patient-derived tumor xenograft (PDTX) models. ∆G4Rs are associated with the promoters of highly amplified genes showing high expression, and with somatic single-nucleotide variants. Differences in ΔG4R landscapes reveal seven transcription factor programs across PDTXs. ∆G4R abundance and locations stratify PDTXs into at least three G4-based subtypes. ∆G4Rs in most PDTXs (14 of 22) were found to associate with more than one breast cancer subtype, which we also call an integrative cluster (IC)2. This suggests the frequent coexistence of multiple breast cancer states within a PDTX model, the majority of which display aggressive triple-negative IC10 gene activity. Short-term cultures of PDTX models with increased ∆G4R levels are more sensitive to small molecules targeting G4 DNA. Thus, G4 landscapes reveal additional IC-related intratumor heterogeneity in PDTX biopsies, improving breast cancer stratification and potentially identifying new treatment strategies. Quantitative ChIP–seq analysis maps G-quadruplex (G4) DNA structures in breast cancer patient-derived tumor xenograft (PDTX) models. G4-based subtypes highlight additional tumor heterogeneity in the integrative cluster (IC) system.},
  archive      = {J_NG},
  author       = {Hänsel-Hertsch, Robert and Simeone, Angela and Shea, Abigail and Hui, Winnie W. I. and Zyner, Katherine G. and Marsico, Giovanni and Rueda, Oscar M. and Bruna, Alejandra and Martin, Alistair and Zhang, Xiaoyun and Adhikari, Santosh and Tannahill, David and Caldas, Carlos and Balasubramanian, Shankar},
  doi          = {10.1038/s41588-020-0672-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {878-883},
  shortjournal = {Nat. Genet.},
  title        = {Landscape of G-quadruplex DNA structural regions in breast cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). N6-methyladenosine co-transcriptionally directs the
demethylation of histone H3K9me2. <em>NG</em>, <em>52</em>(9), 870–877.
(<a href="https://doi.org/10.1038/s41588-020-0677-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A dynamic epigenome is critical for appropriate gene expression in development and health1–5. Central to this is the intricate process of transcription6–11, which integrates cellular signaling with chromatin changes, transcriptional machinery and modifications to messenger RNA, such as N6-methyladenosine (m6A), which is co-transcriptionally incorporated. The integration of these aspects of the dynamic epigenome, however, is not well understood mechanistically. Here we show that the repressive histone mark H3K9me2 is specifically removed by the induction of m6A-modified transcripts. We demonstrate that the methyltransferase METTL3/METTL14 regulates H3K9me2 modification. We observe a genome-wide correlation between m6A and occupancy by the H3K9me2 demethylase KDM3B, and we find that the m6A reader YTHDC1 physically interacts with and recruits KDM3B to m6A-associated chromatin regions, promoting H3K9me2 demethylation and gene expression. This study establishes a direct link between m6A and dynamic chromatin modification and provides mechanistic insight into the co-transcriptional interplay between RNA modifications and histone modifications. METTL3-induced deposition of N6-methyladenosine (m6A) in RNA correlates with removal of H3K9me2 genome wide. The m6A reader YTHDC1 recruits the H3K9me2 demethylase KDM3B to chromatin.},
  archive      = {J_NG},
  author       = {Li, Yuan and Xia, Linjian and Tan, Kaifen and Ye, Xidong and Zuo, Zhixiang and Li, Minchun and Xiao, Rui and Wang, Zihan and Liu, Xiaona and Deng, Mingqiang and Cui, Jinru and Yang, Mengtian and Luo, Qizhi and Liu, Sun and Cao, Xin and Zhu, Haoran and Liu, Tianqi and Hu, Jiaxin and Shi, Junfang and Xiao, Shan and Xia, Laixin},
  doi          = {10.1038/s41588-020-0677-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {870-877},
  shortjournal = {Nat. Genet.},
  title        = {N6-methyladenosine co-transcriptionally directs the demethylation of histone H3K9me2},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). RNA post-transcriptional modification speaks to chromatin.
<em>NG</em>, <em>52</em>(9), 868–869. (<a
href="https://doi.org/10.1038/s41588-020-0685-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The role of N6-methyladenosine (m6A) is still not fully understood. Two new studies advance understanding of this RNA modification. One shows that m6A modification of nascent messenger RNA promotes transcription by recruiting the histone H3 K9 demethylase KDM3B. Another study identifies genetic variants that affect m6A deposition and human disease.},
  archive      = {J_NG},
  author       = {Chan, Dalen and Batista, Pedro J.},
  doi          = {10.1038/s41588-020-0685-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {868-869},
  shortjournal = {Nat. Genet.},
  title        = {RNA post-transcriptional modification speaks to chromatin},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). RNA closing the polycomb circle. <em>NG</em>,
<em>52</em>(9), 866–867. (<a
href="https://doi.org/10.1038/s41588-020-0683-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Binding of RNA to the gene expression regulator Polycomb repressive complex 2 (PRC2) has been proposed to antagonize PRC2’s chromatin recruitment. A new study now shows that RNA is in fact critical for correct recruitment of PRC2 at its target genes in human pluripotent stem cells and suggests that interplay of PRC2 and RNA can fine-tune PRC2’s regulatory role.},
  archive      = {J_NG},
  author       = {Mocavini, Ivano and Di Croce, Luciano},
  doi          = {10.1038/s41588-020-0683-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {866-867},
  shortjournal = {Nat. Genet.},
  title        = {RNA closing the polycomb circle},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Crop genomes and beyond. <em>NG</em>, <em>52</em>(9), 865.
(<a href="https://doi.org/10.1038/s41588-020-0691-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increasing amounts of crop genomic resources, along with new technical achievements in genome analysis, can facilitate basic and translational research in agriculture, and expand the ability to meet the global challenge of food production and security.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0691-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {865},
  shortjournal = {Nat. Genet.},
  title        = {Crop genomes and beyond},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). High-definition likelihood inference of genetic correlations
across human complex traits. <em>NG</em>, <em>52</em>(8), 859–864. (<a
href="https://doi.org/10.1038/s41588-020-0653-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic correlation is a central parameter for understanding shared genetic architecture between complex traits. By using summary statistics from genome-wide association studies (GWAS), linkage disequilibrium score regression (LDSC) was developed for unbiased estimation of genetic correlations. Although easy to use, LDSC only partially utilizes LD information. By fully accounting for LD across the genome, we develop a high-definition likelihood (HDL) method to improve precision in genetic correlation estimation. Compared to LDSC, HDL reduces the variance of genetic correlation estimates by about 60%, equivalent to a 2.5-fold increase in sample size. We apply HDL and LDSC to estimate 435 genetic correlations among 30 behavioral and disease-related phenotypes measured in the UK Biobank (UKBB). In addition to 154 significant genetic correlations observed for both methods, HDL identified another 57 significant genetic correlations, compared to only another 2 significant genetic correlations identified by LDSC. HDL brings more power to genomic analyses and better reveals the underlying connections across human complex traits. The HDL method improves the precision in genetic correlation estimation over LD score regression when applied to GWAS summary statistics of complex traits from the UK Biobank.},
  archive      = {J_NG},
  author       = {Ning, Zheng and Pawitan, Yudi and Shen, Xia},
  doi          = {10.1038/s41588-020-0653-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {859-864},
  shortjournal = {Nat. Genet.},
  title        = {High-definition likelihood inference of genetic correlations across human complex traits},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Recurrent inversion toggling and great ape genome evolution.
<em>NG</em>, <em>52</em>(8), 849–858. (<a
href="https://doi.org/10.1038/s41588-020-0646-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inversions play an important role in disease and evolution but are difficult to characterize because their breakpoints map to large repeats. We increased by sixfold the number (n = 1,069) of previously reported great ape inversions by using single-cell DNA template strand and long-read sequencing. We find that the X chromosome is most enriched (2.5-fold) for inversions, on the basis of its size and duplication content. There is an excess of differentially expressed primate genes near the breakpoints of large (&amp;gt;100 kilobases (kb)) inversions but not smaller events. We show that when great ape lineage-specific duplications emerge, they preferentially (approximately 75%) occur in an inverted orientation compared to that at their ancestral locus. We construct megabase-pair scale haplotypes for individual chromosomes and identify 23 genomic regions that have recurrently toggled between a direct and an inverted state over 15 million years. The direct orientation is most frequently the derived state for human polymorphisms that predispose to recurrent copy number variants associated with neurodevelopmental disease. Genome-wide detection of inversions in great ape genomes by using long-read sequencing and single-cell DNA template strand sequencing (Strand-seq) expands the number of known ape inversions and identifies several regions that have recurrently toggled between a direct and an inverted state during primate evolution.},
  archive      = {J_NG},
  author       = {Porubsky, David and Sanders, Ashley D. and Höps, Wolfram and Hsieh, PingHsun and Sulovari, Arvis and Li, Ruiyang and Mercuri, Ludovica and Sorensen, Melanie and Murali, Shwetha C. and Gordon, David and Cantsilieris, Stuart and Pollen, Alex A. and Ventura, Mario and Antonacci, Francesca and Marschall, Tobias and Korbel, Jan O. and Eichler, Evan E.},
  doi          = {10.1038/s41588-020-0646-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {849-858},
  shortjournal = {Nat. Genet.},
  title        = {Recurrent inversion toggling and great ape genome evolution},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Cohesin promotes stochastic domain intermingling to ensure
proper regulation of boundary-proximal genes. <em>NG</em>,
<em>52</em>(8), 840–848. (<a
href="https://doi.org/10.1038/s41588-020-0647-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human genome can be segmented into topologically associating domains (TADs), which have been proposed to spatially sequester genes and regulatory elements through chromatin looping. Interactions between TADs have also been suggested, presumably because of variable boundary positions across individual cells. However, the nature, extent and consequence of these dynamic boundaries remain unclear. Here, we combine high-resolution imaging with Oligopaint technology to quantify the interaction frequencies across both weak and strong boundaries. We find that chromatin intermingling across population-defined boundaries is widespread but that the extent of permissibility is locus-specific. Cohesin depletion, which abolishes domain formation at the population level, does not induce ectopic interactions but instead reduces interactions across all boundaries tested. In contrast, WAPL or CTCF depletion increases inter-domain contacts in a cohesin-dependent manner. Reduced chromatin intermingling due to cohesin loss affects the topology and transcriptional bursting frequencies of genes near boundaries. We propose that cohesin occasionally bypasses boundaries to promote incorporation of boundary-proximal genes into neighboring domains. A combination of super-resolution microscopy and Oligopaint technology shows that TAD boundaries are variable at the single-cell level. Loss of cohesin, in contrast to WAPL or CTCF depletion, reduces interactions across boundaries and alters transcriptional bursting of genes near boundaries.},
  archive      = {J_NG},
  author       = {Luppino, Jennifer M. and Park, Daniel S. and Nguyen, Son C. and Lan, Yemin and Xu, Zhuxuan and Yunker, Rebecca and Joyce, Eric F.},
  doi          = {10.1038/s41588-020-0647-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {840-848},
  shortjournal = {Nat. Genet.},
  title        = {Cohesin promotes stochastic domain intermingling to ensure proper regulation of boundary-proximal genes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Imprecise DNMT1 activity coupled with neighbor-guided
correction enables robust yet flexible epigenetic inheritance.
<em>NG</em>, <em>52</em>(8), 828–839. (<a
href="https://doi.org/10.1038/s41588-020-0661-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The epigenome, including DNA methylation, is stably propagated during mitotic division. However, single-cell clonal expansion produces heterogeneous methylomes, thus raising the question of how the DNA methylome remains stable despite constant epigenetic drift. Here, we report that a clonal population of DNA (cytosine-5)-methyltransferase 1 (DNMT1)-only cells produces a heterogeneous methylome, which is robustly propagated on cell expansion and differentiation. Our data show that DNMT1 has imprecise maintenance activity and possibly possesses weak de novo activity, leading to spontaneous ‘epimutations’. However, these epimutations tend to be corrected through a neighbor-guided mechanism, which is likely to be enabled by the environment-sensitive de novo activity (‘tuner’) and maintenance activity (‘stabilizer’) of DNMT1. By generating base-resolution maps of de novo and maintenance activities, we find that H3K9me2/3-marked regions show enhanced de novo activity, and CpG islands have both poor maintenance and de novo activities. The imprecise epigenetic machinery coupled with neighbor-guided correction may be a fundamental mechanism underlying robust yet flexible epigenetic inheritance. Cells lacking TET proteins and DNMT3A and DNMT3B show that DNMT1 has imprecise maintenance activity and weak de novo activity leading to spontaneous ‘epimutations’. These epimutations can be corrected through a neighbor-guided mechanism.},
  archive      = {J_NG},
  author       = {Wang, Qiujun and Yu, Guang and Ming, Xuan and Xia, Weikun and Xu, Xiguang and Zhang, Yu and Zhang, Wenhao and Li, Yuanyuan and Huang, Chunyi and Xie, Hehuang and Zhu, Bing and Xie, Wei},
  doi          = {10.1038/s41588-020-0661-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {828-839},
  shortjournal = {Nat. Genet.},
  title        = {Imprecise DNMT1 activity coupled with neighbor-guided correction enables robust yet flexible epigenetic inheritance},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). TETs compete with DNMT3 activity in pluripotent cells at
thousands of methylated somatic enhancers. <em>NG</em>, <em>52</em>(8),
819–827. (<a href="https://doi.org/10.1038/s41588-020-0639-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian cells stably maintain high levels of DNA methylation despite expressing both positive (DNMT3A/B) and negative (TET1-3) regulators. Here, we analyzed the independent and combined effects of these regulators on the DNA methylation landscape using a panel of knockout human embryonic stem cell (ESC) lines. The greatest impact on global methylation levels was observed in DNMT3-deficient cells, including reproducible focal demethylation at thousands of normally methylated loci. Demethylation depends on TET expression and occurs only when both DNMT3s are absent. Dynamic loci are enriched for hydroxymethylcytosine and overlap with subsets of putative somatic enhancers that are methylated in ESCs and can be activated upon differentiation. We observe similar dynamics in mouse ESCs that were less frequent in epiblast stem cells (EpiSCs) and scarce in somatic tissues, suggesting a conserved pluripotency-linked mechanism. Taken together, our data reveal tightly regulated competition between DNMT3s and TETs at thousands of somatic regulatory sequences within pluripotent cells. Whole-genome bisulfite sequencing analysis of human embryonic stem cells shows that DNMT3 deficiency leads to global and local demethylation, which depends on TET activity. Dynamic loci overlap with putative somatic enhancers that are highly methylated in ESCs.},
  archive      = {J_NG},
  author       = {Charlton, Jocelyn and Jung, Eunmi J. and Mattei, Alexandra L. and Bailly, Nina and Liao, Jing and Martin, Eric J. and Giesselmann, Pay and Brändl, Björn and Stamenova, Elena K. and Müller, Franz-Josef and Kiskinis, Evangelos and Gnirke, Andreas and Smith, Zachary D. and Meissner, Alexander},
  doi          = {10.1038/s41588-020-0639-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {819-827},
  shortjournal = {Nat. Genet.},
  title        = {TETs compete with DNMT3 activity in pluripotent cells at thousands of methylated somatic enhancers},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Discovery of regulatory noncoding variants in individual
cancer genomes by using cis-x. <em>NG</em>, <em>52</em>(8), 811–818. (<a
href="https://doi.org/10.1038/s41588-020-0659-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed cis-X, a computational method for discovering regulatory noncoding variants in cancer by integrating whole-genome and transcriptome sequencing data from a single cancer sample. cis-X first finds aberrantly cis-activated genes that exhibit allele-specific expression accompanied by an elevated outlier expression. It then searches for causal noncoding variants that may introduce aberrant transcription factor binding motifs or enhancer hijacking by structural variations. Analysis of 13 T-lineage acute lymphoblastic leukemias identified a recurrent intronic variant predicted to cis-activate the TAL1 oncogene, a finding validated in vivo by chromatin immunoprecipitation sequencing of a patient-derived xenograft. Candidate oncogenes include the prolactin receptor PRLR activated by a focal deletion that removes a CTCF-insulated neighborhood boundary. cis-X may be applied to pediatric and adult solid tumors that are aneuploid and heterogeneous. In contrast to existing approaches, which require large sample cohorts, cis-X enables the discovery of regulatory noncoding variants in individual cancer genomes. A new computational method integrates whole-genome sequencing and transcriptomic data to identify regulatory noncoding variants in an individual cancer genome.},
  archive      = {J_NG},
  author       = {Liu, Yu and Li, Chunliang and Shen, Shuhong and Chen, Xiaolong and Szlachta, Karol and Edmonson, Michael N. and Shao, Ying and Ma, Xiaotu and Hyle, Judith and Wright, Shaela and Ju, Bensheng and Rusch, Michael C. and Liu, Yanling and Li, Benshang and Macias, Michael and Tian, Liqing and Easton, John and Qian, Maoxiang and Yang, Jun J. and Hu, Shaoyan and Look, A. Thomas and Zhang, Jinghui},
  doi          = {10.1038/s41588-020-0659-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {811-818},
  shortjournal = {Nat. Genet.},
  title        = {Discovery of regulatory noncoding variants in individual cancer genomes by using cis-X},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Analysis of ugandan cervical carcinomas identifies human
papillomavirus clade–specific epigenome and transcriptome landscapes.
<em>NG</em>, <em>52</em>(8), 800–810. (<a
href="https://doi.org/10.1038/s41588-020-0673-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cervical cancer is the most common cancer affecting sub-Saharan African women and is prevalent among HIV-positive (HIV+) individuals. No comprehensive profiling of cancer genomes, transcriptomes or epigenomes has been performed in this population thus far. We characterized 118 tumors from Ugandan patients, of whom 72 were HIV+, and performed extended mutation analysis on an additional 89 tumors. We detected human papillomavirus (HPV)-clade-specific differences in tumor DNA methylation, promoter- and enhancer-associated histone marks, gene expression and pathway dysregulation. Changes in histone modification at HPV integration events were correlated with upregulation of nearby genes and endogenous retroviruses. Genomic analysis of 118 cervical tumors from Ugandan individuals identifies HPV-clade-specific differences in tumor DNA methylation, regulatory-region-associated histone marks, gene expression and pathway dysregulation.},
  archive      = {J_NG},
  author       = {Gagliardi, Alessia and Porter, Vanessa L. and Zong, Zusheng and Bowlby, Reanne and Titmuss, Emma and Namirembe, Constance and Griner, Nicholas B. and Petrello, Hilary and Bowen, Jay and Chan, Simon K. and Culibrk, Luka and Darragh, Teresa M. and Stoler, Mark H. and Wright, Thomas C. and Gesuwan, Patee and Dyer, Maureen A. and Ma, Yussanne and Mungall, Karen L. and Jones, Steven J. M. and Nakisige, Carolyn and Novik, Karen and Orem, Jackson and Origa, Martin and Gastier-Foster, Julie M. and Yarchoan, Robert and Casper, Corey and Mills, Gordon B. and Rader, Janet S. and Ojesina, Akinyemi I. and Gerhard, Daniela S. and Mungall, Andrew J. and Marra, Marco A.},
  doi          = {10.1038/s41588-020-0673-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {800-810},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of ugandan cervical carcinomas identifies human papillomavirus clade–specific epigenome and transcriptome landscapes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Prostate cancer reactivates developmental epigenomic
programs during metastatic progression. <em>NG</em>, <em>52</em>(8),
790–799. (<a href="https://doi.org/10.1038/s41588-020-0664-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetic processes govern prostate cancer (PCa) biology, as evidenced by the dependency of PCa cells on the androgen receptor (AR), a prostate master transcription factor. We generated 268 epigenomic datasets spanning two state transitions—from normal prostate epithelium to localized PCa to metastases—in specimens derived from human tissue. We discovered that reprogrammed AR sites in metastatic PCa are not created de novo; rather, they are prepopulated by the transcription factors FOXA1 and HOXB13 in normal prostate epithelium. Reprogrammed regulatory elements commissioned in metastatic disease hijack latent developmental programs, accessing sites that are implicated in prostate organogenesis. Analysis of reactivated regulatory elements enabled the identification and functional validation of previously unknown metastasis-specific enhancers at HOXB13, FOXA1 and NKX3-1. Finally, we observed that prostate lineage-specific regulatory elements were strongly associated with PCa risk heritability and somatic mutation density. Examining prostate biology through an epigenomic lens is fundamental for understanding the mechanisms underlying tumor progression. Analyses of epigenomic datasets spanning transitions from normal prostate epithelium to localized prostate cancer to metastases show that latent developmental programs are reactivated in metastatic disease and that prostate lineage-specific regulatory elements are strongly enriched for prostate cancer risk heritability.},
  archive      = {J_NG},
  author       = {Pomerantz, Mark M. and Qiu, Xintao and Zhu, Yanyun and Takeda, David Y. and Pan, Wenting and Baca, Sylvan C. and Gusev, Alexander and Korthauer, Keegan D. and Severson, Tesa M. and Ha, Gavin and Viswanathan, Srinivas R. and Seo, Ji-Heui and Nguyen, Holly M. and Zhang, Baohui and Pasaniuc, Bogdan and Giambartolomei, Claudia and Alaiwi, Sarah A. and Bell, Connor A. and O’Connor, Edward P. and Chabot, Matthew S. and Stillman, David R. and Lis, Rosina and Font-Tello, Alba and Li, Lewyn and Cejas, Paloma and Bergman, Andries M. and Sanders, Joyce and van der Poel, Henk G. and Gayther, Simon A. and Lawrenson, Kate and Fonseca, Marcos A. S. and Reddy, Jessica and Corona, Rosario I. and Martovetsky, Gleb and Egan, Brian and Choueiri, Toni and Ellis, Leigh and Garraway, Isla P. and Lee, Gwo-Shu Mary and Corey, Eva and Long, Henry W. and Zwart, Wilbert and Freedman, Matthew L.},
  doi          = {10.1038/s41588-020-0664-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {790-799},
  shortjournal = {Nat. Genet.},
  title        = {Prostate cancer reactivates developmental epigenomic programs during metastatic progression},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The DNA methylation landscape of advanced prostate cancer.
<em>NG</em>, <em>52</em>(8), 778–789. (<a
href="https://doi.org/10.1038/s41588-020-0648-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although DNA methylation is a key regulator of gene expression, the comprehensive methylation landscape of metastatic cancer has never been defined. Through whole-genome bisulfite sequencing paired with deep whole-genome and transcriptome sequencing of 100 castration-resistant prostate metastases, we discovered alterations affecting driver genes that were detectable only with integrated whole-genome approaches. Notably, we observed that 22% of tumors exhibited a novel epigenomic subtype associated with hypermethylation and somatic mutations in TET2, DNMT3B, IDH1 and BRAF. We also identified intergenic regions where methylation is associated with RNA expression of the oncogenic driver genes AR, MYC and ERG. Finally, we showed that differential methylation during progression preferentially occurs at somatic mutational hotspots and putative regulatory regions. This study is a large integrated study of whole-genome, whole-methylome and whole-transcriptome sequencing in metastatic cancer that provides a comprehensive overview of the important regulatory role of methylation in metastatic castration-resistant prostate cancer. Whole-genome bisulfite sequencing along with whole-genome and transcriptome sequencing of 100 prostate cancer metastases identifies genomic regions that are differentially methylated during disease progression and a novel epigenomic subtype.},
  archive      = {J_NG},
  author       = {Zhao, Shuang G. and Chen, William S. and Li, Haolong and Foye, Adam and Zhang, Meng and Sjöström, Martin and Aggarwal, Rahul and Playdle, Denise and Liao, Arnold and Alumkal, Joshi J. and Das, Rajdeep and Chou, Jonathan and Hua, Junjie T. and Barnard, Travis J. and Bailey, Adina M. and Chow, Eric D. and Perry, Marc D. and Dang, Ha X. and Yang, Rendong and Moussavi-Baygi, Ruhollah and Zhang, Li and Alshalalfa, Mohammed and Laura Chang, S. and Houlahan, Kathleen E. and Shiah, Yu-Jia and Beer, Tomasz M. and Thomas, George and Chi, Kim N. and Gleave, Martin and Zoubeidi, Amina and Reiter, Robert E. and Rettig, Matthew B. and Witte, Owen and Yvonne Kim, M. and Fong, Lawrence and Spratt, Daniel E. and Morgan, Todd M. and Bose, Rohit and Huang, Franklin W. and Li, Hui and Chesner, Lisa and Shenoy, Tanushree and Goodarzi, Hani and Asangani, Irfan A. and Sandhu, Shahneen and Lang, Joshua M. and Mahajan, Nupam P. and Lara, Primo N. and Evans, Christopher P. and Febbo, Phillip and Batzoglou, Serafim and Knudsen, Karen E. and He, Housheng H. and Huang, Jiaoti and Zwart, Wilbert and Costello, Joseph F. and Luo, Jianhua and Tomlins, Scott A. and Wyatt, Alexander W. and Dehm, Scott M. and Ashworth, Alan and Gilbert, Luke A. and Boutros, Paul C. and Farh, Kyle and Chinnaiyan, Arul M. and Maher, Christopher A. and Small, Eric J. and Quigley, David A. and Feng, Felix Y.},
  doi          = {10.1038/s41588-020-0648-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {778-789},
  shortjournal = {Nat. Genet.},
  title        = {The DNA methylation landscape of advanced prostate cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genomic analyses implicate noncoding de novo variants in
congenital heart disease. <em>NG</em>, <em>52</em>(8), 769–777. (<a
href="https://doi.org/10.1038/s41588-020-0652-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A genetic etiology is identified for one-third of patients with congenital heart disease (CHD), with 8% of cases attributable to coding de novo variants (DNVs). To assess the contribution of noncoding DNVs to CHD, we compared genome sequences from 749 CHD probands and their parents with those from 1,611 unaffected trios. Neural network prediction of noncoding DNV transcriptional impact identified a burden of DNVs in individuals with CHD (n = 2,238 DNVs) compared to controls (n = 4,177; P = 8.7 × 10−4). Independent analyses of enhancers showed an excess of DNVs in associated genes (27 genes versus 3.7 expected, P = 1 × 10−5). We observed significant overlap between these transcription-based approaches (odds ratio (OR) = 2.5, 95% confidence interval (CI) 1.1–5.0, P = 5.4 × 10−3). CHD DNVs altered transcription levels in 5 of 31 enhancers assayed. Finally, we observed a DNV burden in RNA-binding-protein regulatory sites (OR = 1.13, 95% CI 1.1–1.2, P = 8.8 × 10−5). Our findings demonstrate an enrichment of potentially disruptive regulatory noncoding DNVs in a fraction of CHD at least as high as that observed for damaging coding DNVs. Computational analyses integrating whole-genome sequencing, cardiac epigenomic data and RNA-binding-protein data identify a role for noncoding de novo mutations in congenital heart disease.},
  archive      = {J_NG},
  author       = {Richter, Felix and Morton, Sarah U. and Kim, Seong Won and Kitaygorodsky, Alexander and Wasson, Lauren K. and Chen, Kathleen M. and Zhou, Jian and Qi, Hongjian and Patel, Nihir and DePalma, Steven R. and Parfenov, Michael and Homsy, Jason and Gorham, Joshua M. and Manheimer, Kathryn B. and Velinder, Matthew and Farrell, Andrew and Marth, Gabor and Schadt, Eric E. and Kaltman, Jonathan R. and Newburger, Jane W. and Giardini, Alessandro and Goldmuntz, Elizabeth and Brueckner, Martina and Kim, Richard and Porter, George A. and Bernstein, Daniel and Chung, Wendy K. and Srivastava, Deepak and Tristani-Firouzi, Martin and Troyanskaya, Olga G. and Dickel, Diane E. and Shen, Yufeng and Seidman, Jonathan G. and Seidman, Christine E. and Gelb, Bruce D.},
  doi          = {10.1038/s41588-020-0652-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {769-777},
  shortjournal = {Nat. Genet.},
  title        = {Genomic analyses implicate noncoding de novo variants in congenital heart disease},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Characterizing the ecological and evolutionary dynamics of
cancer. <em>NG</em>, <em>52</em>(8), 759–767. (<a
href="https://doi.org/10.1038/s41588-020-0668-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor initiation and progression are somatic evolutionary processes driven by the accumulation of genetic alterations, some of which confer selective fitness advantages to the host cell. This gene-centric model has shaped the field of cancer biology and advanced understanding of cancer pathophysiology. Importantly, however, each genotype encodes diverse phenotypic traits that permit acclimation to varied microenvironmental conditions. Epigenetic and transcriptional changes also contribute to the heritable phenotypic variation required for evolution. Additionally, interactions between cancer cells and surrounding stromal and immune cells through autonomous and non-autonomous signaling can influence competition for survival. Therefore, a mechanistic understanding of tumor progression must account for evolutionary and ecological dynamics. In this Perspective, we outline technological advances and model systems to characterize tumor progression through space and time. We discuss the importance of unifying experimentation with computational modeling and opportunities to inform cancer control. This Perspective discusses the need for mechanistic and quantitative understanding of the ecological and evolutionary dynamics of tumor growth to inform strategies for earlier detection, diagnosis and treatment of cancer.},
  archive      = {J_NG},
  author       = {Zahir, Nastaran and Sun, Ruping and Gallahan, Daniel and Gatenby, Robert A. and Curtis, Christina},
  doi          = {10.1038/s41588-020-0668-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {759-767},
  shortjournal = {Nat. Genet.},
  title        = {Characterizing the ecological and evolutionary dynamics of cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Guidelines for human gene nomenclature. <em>NG</em>,
<em>52</em>(8), 754–758. (<a
href="https://doi.org/10.1038/s41588-020-0669-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Standardized gene naming is crucial for effective communication about genes, and as genomics becomes increasingly important in health care, the need for a consistent language to refer to human genes becomes ever more essential. Here, we present the current HUGO Gene Nomenclature Committee (HGNC) guidelines for naming not only protein-coding genes but also RNA genes and pseudogenes, and we outline the changes in approach and ethos that have resulted from the discoveries of the past few decades.},
  archive      = {J_NG},
  author       = {Bruford, Elspeth A. and Braschi, Bryony and Denny, Paul and Jones, Tamsin E. M. and Seal, Ruth L. and Tweedie, Susan},
  doi          = {10.1038/s41588-020-0669-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {754-758},
  shortjournal = {Nat. Genet.},
  title        = {Guidelines for human gene nomenclature},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). TADs without borders. <em>NG</em>, <em>52</em>(8), 752–753.
(<a href="https://doi.org/10.1038/s41588-020-0670-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomes are highly organized in space and time. Compartments, topologically associating domains (TADs) and loops are three dimensional (3D) genome features that have been extensively studied. Among these three levels of organization, TADs have sparked the most debate. New microscopy data shed light on how TADs and their leaky borders contribute to gene regulation.},
  archive      = {J_NG},
  author       = {Farabella, Irene and Marti-Renom, Marc A.},
  doi          = {10.1038/s41588-020-0670-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {752-753},
  shortjournal = {Nat. Genet.},
  title        = {TADs without borders},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Naming human genes. <em>NG</em>, <em>52</em>(8), 751. (<a
href="https://doi.org/10.1038/s41588-020-0679-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene nomenclature can be complicated, and the official naming of genes requires rational standards to avoid confusion and to maximize clarity. The HUGO Gene Nomenclature Committee has released updated guidelines for the naming of human genes, and we encourage the community to adopt these recommendations.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0679-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {751},
  shortjournal = {Nat. Genet.},
  title        = {Naming human genes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Mendelian randomization accounting
for correlated and uncorrelated pleiotropic effects using genome-wide
summary statistics. <em>NG</em>, <em>52</em>(7), 750. (<a
href="https://doi.org/10.1038/s41588-020-0655-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Morrison, Jean and Knoblauch, Nicholas and Marcus, Joseph H. and Stephens, Matthew and He, Xin},
  doi          = {10.1038/s41588-020-0655-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {750},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Mendelian randomization accounting for correlated and uncorrelated pleiotropic effects using genome-wide summary statistics},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: A transcriptomic analysis of the phylum
nematoda. <em>NG</em>, <em>52</em>(7), 750. (<a
href="https://doi.org/10.1038/s41588-020-0658-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Parkinson, John and Mitreva, Makedonka and Whitton, Claire and Thomson, Marian and Daub, Jennifer and Martin, John and Schmid, Ralf and Hall, Neil and Barrell, Bart and Waterston, Robert H. and McCarter, James P. and Blaxter, Mark L.},
  doi          = {10.1038/s41588-020-0658-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {750},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A transcriptomic analysis of the phylum nematoda},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Cas9 activates the p53 pathway and
selects for p53-inactivating mutations. <em>NG</em>, <em>52</em>(7),
748–749. (<a href="https://doi.org/10.1038/s41588-020-0663-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Enache, Oana M. and Rendo, Veronica and Abdusamad, Mai and Lam, Daniel and Davison, Desiree and Pal, Sangita and Currimjee, Naomi and Hess, Julian and Pantel, Sasha and Nag, Anwesha and Thorner, Aaron R. and Doench, John G. and Vazquez, Francisca and Beroukhim, Rameen and Golub, Todd R. and Ben-David, Uri},
  doi          = {10.1038/s41588-020-0663-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {748-749},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Cas9 activates the p53 pathway and selects for p53-inactivating mutations},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Mendelian randomization accounting for correlated and
uncorrelated pleiotropic effects using genome-wide summary statistics.
<em>NG</em>, <em>52</em>(7), 740–747. (<a
href="https://doi.org/10.1038/s41588-020-0631-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mendelian randomization (MR) is a valuable tool for detecting causal effects by using genetic variant associations. Opportunities to apply MR are growing rapidly with the increasing number of genome-wide association studies (GWAS). However, existing MR methods rely on strong assumptions that are often violated, leading to false positives. Correlated horizontal pleiotropy, which arises when variants affect both traits through a heritable shared factor, remains a particularly challenging problem. We propose a new MR method, Causal Analysis Using Summary Effect estimates (CAUSE), that accounts for correlated and uncorrelated horizontal pleiotropic effects. We demonstrate, in simulations, that CAUSE avoids more false positives induced by correlated horizontal pleiotropy than other methods. Applied to traits studied in recent GWAS studies, we find that CAUSE detects causal relationships that have strong literature support and avoids identifying most unlikely relationships. Our results suggest that shared heritable factors are common and may lead to many false positives using alternative methods. CAUSE is a new Mendelian randomization method that accounts for correlated and uncorrelated horizontal pleiotropic effects. CAUSE is more robust to correlated pleiotropy than other methods and avoids identifying unlikely causal relationships.},
  archive      = {J_NG},
  author       = {Morrison, Jean and Knoblauch, Nicholas and Marcus, Joseph H. and Stephens, Matthew and He, Xin},
  doi          = {10.1038/s41588-020-0631-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {740-747},
  shortjournal = {Nat. Genet.},
  title        = {Mendelian randomization accounting for correlated and uncorrelated pleiotropic effects using genome-wide summary statistics},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The evolutionarily conserved piRNA-producing locus pi6 is
required for male mouse fertility. <em>NG</em>, <em>52</em>(7), 728–739.
(<a href="https://doi.org/10.1038/s41588-020-0657-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pachytene PIWI-interacting RNAs (piRNAs), which comprise &amp;gt;80% of small RNAs in the adult mouse testis, have been proposed to bind and regulate target RNAs like microRNAs, cleave targets like short interfering RNAs or lack biological function altogether. Although piRNA pathway protein mutants are male sterile, no biological function has been identified for any mammalian piRNA-producing locus. Here, we report that males lacking piRNAs from a conserved mouse pachytene piRNA locus on chromosome 6 (pi6) produce sperm with defects in capacitation and egg fertilization. Moreover, heterozygous embryos sired by pi6−/− fathers show reduced viability in utero. Molecular analyses suggest that pi6 piRNAs repress gene expression by cleaving messenger RNAs encoding proteins required for sperm function. pi6 also participates in a network of piRNA–piRNA precursor interactions that initiate piRNA production from a second piRNA locus on chromosome 10, as well as pi6 itself. Our data establish a direct role for pachytene piRNAs in spermiogenesis and embryo viability. Male mice lacking piRNAs from a conserved locus on chromosome 6 (pi6) produce sperm with defects in capacitation and egg fertilization. pi6 piRNAs repress mRNAs required for sperm function.},
  archive      = {J_NG},
  author       = {Wu, Pei-Hsuan and Fu, Yu and Cecchini, Katharine and Özata, Deniz M. and Arif, Amena and Yu, Tianxiong and Colpan, Cansu and Gainetdinov, Ildar and Weng, Zhiping and Zamore, Phillip D.},
  doi          = {10.1038/s41588-020-0657-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {728-739},
  shortjournal = {Nat. Genet.},
  title        = {The evolutionarily conserved piRNA-producing locus pi6 is required for male mouse fertility},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Selective mediator dependence of cell-type-specifying
transcription. <em>NG</em>, <em>52</em>(7), 719–727. (<a
href="https://doi.org/10.1038/s41588-020-0635-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Mediator complex directs signals from DNA-binding transcription factors to RNA polymerase II (Pol II). Despite this pivotal position, mechanistic understanding of Mediator in human cells remains incomplete. Here we quantified Mediator-controlled Pol II kinetics by coupling rapid subunit degradation with orthogonal experimental readouts. In agreement with a model of condensate-driven transcription initiation, large clusters of hypophosphorylated Pol II rapidly disassembled upon Mediator degradation. This was accompanied by a selective and pronounced disruption of cell-type-specifying transcriptional circuits, whose constituent genes featured exceptionally high rates of Pol II turnover. Notably, the transcriptional output of most other genes was largely unaffected by acute Mediator ablation. Maintenance of transcriptional activity at these genes was linked to an unexpected CDK9-dependent compensatory feedback loop that elevated Pol II pause release rates across the genome. Collectively, our work positions human Mediator as a globally acting coactivator that selectively safeguards the functionality of cell-type-specifying transcriptional networks. Analysis with alleles encoding pharmacologically degradable Mediator subunits shows that Mediator acts as a global coactivator that facilitates transcription globally but is acutely required for cell-type-specific gene regulatory circuits.},
  archive      = {J_NG},
  author       = {Jaeger, Martin G. and Schwalb, Björn and Mackowiak, Sebastian D. and Velychko, Taras and Hanzl, Alexander and Imrichova, Hana and Brand, Matthias and Agerer, Benedikt and Chorn, Someth and Nabet, Behnam and Ferguson, Fleur M. and Müller, André C. and Bergthaler, Andreas and Gray, Nathanael S. and Bradner, James E. and Bock, Christoph and Hnisz, Denes and Cramer, Patrick and Winter, Georg E.},
  doi          = {10.1038/s41588-020-0635-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {719-727},
  shortjournal = {Nat. Genet.},
  title        = {Selective mediator dependence of cell-type-specifying transcription},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell analysis of clonal maintenance of
transcriptional and epigenetic states in cancer cells. <em>NG</em>,
<em>52</em>(7), 709–718. (<a
href="https://doi.org/10.1038/s41588-020-0645-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Propagation of clonal regulatory programs contributes to cancer development. It is poorly understood how epigenetic mechanisms interact with genetic drivers to shape this process. Here, we combine single-cell analysis of transcription and DNA methylation with a Luria–Delbrück experimental design to demonstrate the existence of clonally stable epigenetic memory in multiple types of cancer cells. Longitudinal transcriptional and genetic analysis of clonal colon cancer cell populations reveals a slowly drifting spectrum of epithelial-to-mesenchymal transcriptional identities that is seemingly independent of genetic variation. DNA methylation landscapes correlate with these identities but also reflect an independent clock-like methylation loss process. Methylation variation can be explained as an effect of global trans-acting factors in most cases. However, for a specific class of promoters—in particular, cancer–testis antigens—de-repression is correlated with and probably driven by loss of methylation in cis. This study indicates how genetic sub-clonal structure in cancer cells can be diversified by epigenetic memory. Longitudinal single-cell analysis of transcription and DNA methylation dynamics in cancer cell lines suggests a clonally stable epigenetic memory. Colon cancer cells show a spectrum of epithelial-to-mesenchymal identities that seems independent of genetic variation.},
  archive      = {J_NG},
  author       = {Meir, Zohar and Mukamel, Zohar and Chomsky, Elad and Lifshitz, Aviezer and Tanay, Amos},
  doi          = {10.1038/s41588-020-0645-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {709-718},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell analysis of clonal maintenance of transcriptional and epigenetic states in cancer cells},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Multi-cancer analysis of clonality and the timing of
systemic spread in paired primary tumors and metastases. <em>NG</em>,
<em>52</em>(7), 701–708. (<a
href="https://doi.org/10.1038/s41588-020-0628-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metastasis is the primary cause of cancer-related deaths, but the natural history, clonal evolution and impact of treatment are poorly understood. We analyzed whole-exome sequencing (WES) data from 457 paired primary tumor and metastatic samples from 136 patients with breast, colorectal and lung cancer, including untreated (n = 99) and treated (n = 100) metastases. Treated metastases often harbored private ‘driver’ mutations, whereas untreated metastases did not, suggesting that treatment promotes clonal evolution. Polyclonal seeding was common in untreated lymph node metastases (n = 17 out of 29, 59%) and distant metastases (n = 20 out of 70, 29%), but less frequent in treated distant metastases (n = 9 out of 94, 10%). The low number of metastasis-private clonal mutations is consistent with early metastatic seeding, which we estimated occurred 2–4 years before diagnosis across these cancers. Furthermore, these data suggest that the natural course of metastasis is selectively relaxed relative to early tumorigenesis and that metastasis-private mutations are not drivers of cancer spread but instead associated with drug resistance. Analysis of whole-exome sequencing data from paired primary tumors and metastases from patients with breast, colorectal and lung cancer identifies clonal remodeling associated with therapy and few metastasis-private clonal mutations, consistent with early metastatic seeding.},
  archive      = {J_NG},
  author       = {Hu, Zheng and Li, Zan and Ma, Zhicheng and Curtis, Christina},
  doi          = {10.1038/s41588-020-0628-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {701-708},
  shortjournal = {Nat. Genet.},
  title        = {Multi-cancer analysis of clonality and the timing of systemic spread in paired primary tumors and metastases},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Lymph node metastases develop through a wider evolutionary
bottleneck than distant metastases. <em>NG</em>, <em>52</em>(7),
692–700. (<a href="https://doi.org/10.1038/s41588-020-0633-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic diversity among metastases is poorly understood but contains important information about disease evolution at secondary sites. Here we investigate inter- and intra-lesion heterogeneity for two types of metastases that associate with different clinical outcomes: lymph node and distant organ metastases in human colorectal cancer. We develop a rigorous mathematical framework for quantifying metastatic phylogenetic diversity. Distant metastases are typically monophyletic and genetically similar to each other. Lymph node metastases, in contrast, display high levels of inter-lesion diversity. We validate these findings by analyzing 317 multi-region biopsies from an independent cohort of 20 patients. We further demonstrate higher levels of intra-lesion heterogeneity in lymph node than in distant metastases. Our results show that fewer primary tumor lineages seed distant metastases than lymph node metastases, indicating that the two sites are subject to different levels of selection. Thus, lymph node and distant metastases develop through fundamentally different evolutionary mechanisms. Analysis of metastases using a mathematical framework and multi-region sampling data shows that lymph node metastases have higher levels of intratumor heterogeneity than distant metastases, and that these form via different evolutionary mechanisms.},
  archive      = {J_NG},
  author       = {Reiter, Johannes G. and Hung, Wei-Ting and Lee, I-Hsiu and Nagpal, Shriya and Giunta, Peter and Degner, Sebastian and Liu, Gang and Wassenaar, Emma C. E. and Jeck, William R. and Taylor, Martin S. and Farahani, Alexander A. and Marble, Hetal D. and Knott, Simon and Kranenburg, Onno and Lennerz, Jochen K. and Naxerova, Kamila},
  doi          = {10.1038/s41588-020-0633-2},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {692-700},
  shortjournal = {Nat. Genet.},
  title        = {Lymph node metastases develop through a wider evolutionary bottleneck than distant metastases},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Discovery of 318 new risk loci for type 2 diabetes and
related vascular outcomes among 1.4 million participants in a
multi-ancestry meta-analysis. <em>NG</em>, <em>52</em>(7), 680–691. (<a
href="https://doi.org/10.1038/s41588-020-0637-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We investigated type 2 diabetes (T2D) genetic susceptibility via multi-ancestry meta-analysis of 228,499 cases and 1,178,783 controls in the Million Veteran Program (MVP), DIAMANTE, Biobank Japan and other studies. We report 568 associations, including 286 autosomal, 7 X-chromosomal and 25 identified in ancestry-specific analyses that were previously unreported. Transcriptome-wide association analysis detected 3,568 T2D associations with genetically predicted gene expression in 687 novel genes; of these, 54 are known to interact with FDA-approved drugs. A polygenic risk score (PRS) was strongly associated with increased risk of T2D-related retinopathy and modestly associated with chronic kidney disease (CKD), peripheral artery disease (PAD) and neuropathy. We investigated the genetic etiology of T2D-related vascular outcomes in the MVP and observed statistical SNP–T2D interactions at 13 variants, including coronary heart disease (CHD), CKD, PAD and neuropathy. These findings may help to identify potential therapeutic targets for T2D and genomic pathways that link T2D to vascular outcomes. Genome-wide association meta-analyses among 1.4 million individuals identify 318 new risk loci for type 2 diabetes and provide insight into the contribution of these risk variants to diabetes-related vascular outcomes.},
  archive      = {J_NG},
  author       = {Vujkovic, Marijana and Keaton, Jacob M. and Lynch, Julie A. and Miller, Donald R. and Zhou, Jin and Tcheandjieu, Catherine and Huffman, Jennifer E. and Assimes, Themistocles L. and Lorenz, Kimberly and Zhu, Xiang and Hilliard, Austin T. and Judy, Renae L. and Huang, Jie and Lee, Kyung M. and Klarin, Derek and Pyarajan, Saiju and Danesh, John and Melander, Olle and Rasheed, Asif and Mallick, Nadeem H. and Hameed, Shahid and Qureshi, Irshad H. and Afzal, Muhammad Naeem and Malik, Uzma and Jalal, Anjum and Abbas, Shahid and Sheng, Xin and Gao, Long and Kaestner, Klaus H. and Susztak, Katalin and Sun, Yan V. and DuVall, Scott L. and Cho, Kelly and Lee, Jennifer S. and Gaziano, J. Michael and Phillips, Lawrence S. and Meigs, James B. and Reaven, Peter D. and Wilson, Peter W. and Edwards, Todd L. and Rader, Daniel J. and Damrauer, Scott M. and O’Donnell, Christopher J. and Tsao, Philip S. and Chang, Kyong-Mi and Voight, Benjamin F. and Saleheen, Danish},
  doi          = {10.1038/s41588-020-0637-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {680-691},
  shortjournal = {Nat. Genet.},
  title        = {Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Large-scale genome-wide association study in a japanese
population identifies novel susceptibility loci across different
diseases. <em>NG</em>, <em>52</em>(7), 669–679. (<a
href="https://doi.org/10.1038/s41588-020-0640-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The overwhelming majority of participants in current genetic studies are of European ancestry. To elucidate disease biology in the East Asian population, we conducted a genome-wide association study (GWAS) with 212,453 Japanese individuals across 42 diseases. We detected 320 independent signals in 276 loci for 27 diseases, with 25 novel loci (P &amp;lt; 9.58 × 10−9). East Asian–specific missense variants were identified as candidate causal variants for three novel loci, and we successfully replicated two of them by analyzing independent Japanese cohorts; p.R220W of ATG16L2 (associated with coronary artery disease) and p.V326A of POT1 (associated with lung cancer). We further investigated enrichment of heritability within 2,868 annotations of genome-wide transcription factor occupancy, and identified 378 significant enrichments across nine diseases (false discovery rate &amp;lt; 0.05) (for example, NKX3-1 for prostate cancer). This large-scale GWAS in a Japanese population provides insights into the etiology of complex diseases and highlights the importance of performing GWAS in non-European populations. Genome-wide analysis in 212,453 Japanese individuals identifies loci associated with 42 diseases. Comparative analysis with European populations identifies East Asian–specific associations.},
  archive      = {J_NG},
  author       = {Ishigaki, Kazuyoshi and Akiyama, Masato and Kanai, Masahiro and Takahashi, Atsushi and Kawakami, Eiryo and Sugishita, Hiroki and Sakaue, Saori and Matoba, Nana and Low, Siew-Kee and Okada, Yukinori and Terao, Chikashi and Amariuta, Tiffany and Gazal, Steven and Kochi, Yuta and Horikoshi, Momoko and Suzuki, Ken and Ito, Kaoru and Koyama, Satoshi and Ozaki, Kouichi and Niida, Shumpei and Sakata, Yasushi and Sakata, Yasuhiko and Kohno, Takashi and Shiraishi, Kouya and Momozawa, Yukihide and Hirata, Makoto and Matsuda, Koichi and Ikeda, Masashi and Iwata, Nakao and Ikegawa, Shiro and Kou, Ikuyo and Tanaka, Toshihiro and Nakagawa, Hidewaki and Suzuki, Akari and Hirota, Tomomitsu and Tamari, Mayumi and Chayama, Kazuaki and Miki, Daiki and Mori, Masaki and Nagayama, Satoshi and Daigo, Yataro and Miki, Yoshio and Katagiri, Toyomasa and Ogawa, Osamu and Obara, Wataru and Ito, Hidemi and Yoshida, Teruhiko and Imoto, Issei and Takahashi, Takashi and Tanikawa, Chizu and Suzuki, Takao and Sinozaki, Nobuaki and Minami, Shiro and Yamaguchi, Hiroki and Asai, Satoshi and Takahashi, Yasuo and Yamaji, Ken and Takahashi, Kazuhisa and Fujioka, Tomoaki and Takata, Ryo and Yanai, Hideki and Masumoto, Akihide and Koretsune, Yukihiro and Kutsumi, Hiromu and Higashiyama, Masahiko and Murayama, Shigeo and Minegishi, Naoko and Suzuki, Kichiya and Tanno, Kozo and Shimizu, Atsushi and Yamaji, Taiki and Iwasaki, Motoki and Sawada, Norie and Uemura, Hirokazu and Tanaka, Keitaro and Naito, Mariko and Sasaki, Makoto and Wakai, Kenji and Tsugane, Shoichiro and Yamamoto, Masayuki and Yamamoto, Kazuhiko and Murakami, Yoshinori and Nakamura, Yusuke and Raychaudhuri, Soumya and Inazawa, Johji and Yamauchi, Toshimasa and Kadowaki, Takashi and Kubo, Michiaki and Kamatani, Yoichiro},
  doi          = {10.1038/s41588-020-0640-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {669-679},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale genome-wide association study in a japanese population identifies novel susceptibility loci across different diseases},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Cas9 activates the p53 pathway and selects for
p53-inactivating mutations. <em>NG</em>, <em>52</em>(7), 662–668. (<a
href="https://doi.org/10.1038/s41588-020-0623-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cas9 is commonly introduced into cell lines to enable CRISPR–Cas9-mediated genome editing. Here, we studied the genetic and transcriptional consequences of Cas9 expression itself. Gene expression profiling of 165 pairs of human cancer cell lines and their Cas9-expressing derivatives revealed upregulation of the p53 pathway upon introduction of Cas9, specifically in wild-type TP53 (TP53-WT) cell lines. This was confirmed at the messenger RNA and protein levels. Moreover, elevated levels of DNA repair were observed in Cas9-expressing cell lines. Genetic characterization of 42 cell line pairs showed that introduction of Cas9 can lead to the emergence and expansion of p53-inactivating mutations. This was confirmed by competition experiments in isogenic TP53-WT and TP53-null (TP53−/−) cell lines. Lastly, Cas9 was less active in TP53-WT than in TP53-mutant cell lines, and Cas9-induced p53 pathway activation affected cellular sensitivity to both genetic and chemical perturbations. These findings may have broad implications for the proper use of CRISPR–Cas9-mediated genome editing. Cas9 expression induces DNA damage and activates the p53 pathway, and it can lead to the selection of cells with p53-inactivating mutations. Cas9 is less active in wild-type TP53 cell lines than in TP53-mutant cell lines.},
  archive      = {J_NG},
  author       = {Enache, Oana M. and Rendo, Veronica and Abdusamad, Mai and Lam, Daniel and Davison, Desiree and Pal, Sangita and Currimjee, Naomi and Hess, Julian and Pantel, Sasha and Nag, Anwesha and Thorner, Aaron R. and Doench, John G. and Vazquez, Francisca and Beroukhim, Rameen and Golub, Todd R. and Ben-David, Uri},
  doi          = {10.1038/s41588-020-0623-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {662-668},
  shortjournal = {Nat. Genet.},
  title        = {Cas9 activates the p53 pathway and selects for p53-inactivating mutations},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). CTCF is dispensable for immune cell transdifferentiation but
facilitates an acute inflammatory response. <em>NG</em>, <em>52</em>(7),
655–661. (<a href="https://doi.org/10.1038/s41588-020-0643-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Three-dimensional organization of the genome is important for transcriptional regulation1–7. In mammals, CTCF and the cohesin complex create submegabase structures with elevated internal chromatin contact frequencies, called topologically associating domains (TADs)8–12. Although TADs can contribute to transcriptional regulation, ablation of TAD organization by disrupting CTCF or the cohesin complex causes modest gene expression changes13–16. In contrast, CTCF is required for cell cycle regulation17, embryonic development and formation of various adult cell types18. To uncouple the role of CTCF in cell-state transitions and cell proliferation, we studied the effect of CTCF depletion during the conversion of human leukemic B cells into macrophages with minimal cell division. CTCF depletion disrupts TAD organization but not cell transdifferentiation. In contrast, CTCF depletion in induced macrophages impairs the full-blown upregulation of inflammatory genes after exposure to endotoxin. Our results demonstrate that CTCF-dependent genome topology is not strictly required for a functional cell-fate conversion but facilitates a rapid and efficient response to an external stimulus. CTCF is dispensable for transdifferentiation of B cells into induced macrophages despite widespread loss of topologically associating domains. CTCF depletion impairs upregulation of inflammatory genes after endotoxin exposure by destabilizing promoter–enhancer interactions.},
  archive      = {J_NG},
  author       = {Stik, Grégoire and Vidal, Enrique and Barrero, Mercedes and Cuartero, Sergi and Vila-Casadesús, Maria and Mendieta-Esteban, Julen and Tian, Tian V. and Choi, Jinmi and Berenguer, Clara and Abad, Amaya and Borsari, Beatrice and le Dily, François and Cramer, Patrick and Marti-Renom, Marc A. and Stadhouders, Ralph and Graf, Thomas},
  doi          = {10.1038/s41588-020-0643-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {655-661},
  shortjournal = {Nat. Genet.},
  title        = {CTCF is dispensable for immune cell transdifferentiation but facilitates an acute inflammatory response},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Privacy challenges and research opportunities for genomic
data sharing. <em>NG</em>, <em>52</em>(7), 646–654. (<a
href="https://doi.org/10.1038/s41588-020-0651-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The sharing of genomic data holds great promise in advancing precision medicine and providing personalized treatments and other types of interventions. However, these opportunities come with privacy concerns, and data misuse could potentially lead to privacy infringement for individuals and their blood relatives. With the rapid growth and increased availability of genomic datasets, understanding the current genome privacy landscape and identifying the challenges in developing effective privacy-protecting solutions are imperative. In this work, we provide an overview of major privacy threats identified by the research community and examine the privacy challenges in the context of emerging direct-to-consumer genetic-testing applications. We additionally present general privacy-protection techniques for genomic data sharing and their potential applications in direct-to-consumer genomic testing and forensic analyses. Finally, we discuss limitations in current privacy-protection methods, highlight possible mitigation strategies and suggest future research opportunities for advancing genomic data sharing. This review discusses major privacy threats relevant to collection and analysis of genomic data in the context of direct-to-consumer applications and provides suggestions for improving the design of privacy-protecting approaches.},
  archive      = {J_NG},
  author       = {Bonomi, Luca and Huang, Yingxiang and Ohno-Machado, Lucila},
  doi          = {10.1038/s41588-020-0651-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {646-654},
  shortjournal = {Nat. Genet.},
  title        = {Privacy challenges and research opportunities for genomic data sharing},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Pachytene piRNAs as beneficial regulators or a defense
system gone rogue. <em>NG</em>, <em>52</em>(7), 644–645. (<a
href="https://doi.org/10.1038/s41588-020-0656-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pachytene Piwi-interacting RNAs (piRNAs) are abundant small non-coding RNAs expressed in mammalian germ lines. A new study indicates that, among the diverse pool of piRNA sequences, a small number act as highly selective guides that induce cleavage of coding and non-coding transcripts, thus promoting piRNA generation and regulating gene expression.},
  archive      = {J_NG},
  author       = {Aravin, Alexei A.},
  doi          = {10.1038/s41588-020-0656-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {644-645},
  shortjournal = {Nat. Genet.},
  title        = {Pachytene piRNAs as beneficial regulators or a defense system gone rogue},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Navigating the path to distant metastasis. <em>NG</em>,
<em>52</em>(7), 642–643. (<a
href="https://doi.org/10.1038/s41588-020-0660-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How ‘difficult’ is it for somatic evolution to produce a cell that is capable of leaving the primary tumor and growing in a distant organ? In this issue, Reiter et al. assess genetic diversity across metastatic lesions and identify a tight selective bottleneck preceding distant metastasis.},
  archive      = {J_NG},
  author       = {Graham, Trevor A. and Shibata, Darryl},
  doi          = {10.1038/s41588-020-0660-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {642-643},
  shortjournal = {Nat. Genet.},
  title        = {Navigating the path to distant metastasis},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Racism and the status quo. <em>NG</em>, <em>52</em>(7), 641.
(<a href="https://doi.org/10.1038/s41588-020-0666-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When the status quo in science continues to perpetuate ingrained systems of discrimination, inequality and racism, the beneficiaries of these systems must not only reexamine their collective practices but also redress historical injustices through consequential, concrete actions that will lead to change. We cannot focus only on the first steps of listening and learning to build a better, fairer scientific community. We at Nature Genetics fully commit to pledging our energy and efforts toward long-term goals explicitly designed to fundamentally transform how science and its dissemination are conducted.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0666-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {641},
  shortjournal = {Nat. Genet.},
  title        = {Racism and the status quo},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Biallelic mutations in SORD cause a
common and potentially treatable hereditary neuropathy with implications
for diabetes. <em>NG</em>, <em>52</em>(6), 640. (<a
href="https://doi.org/10.1038/s41588-020-0649-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Cortese, Andrea and Zhu, Yi and Rebelo, Adriana P. and Negri, Sara and Courel, Steve and Abreu, Lisa and Bacon, Chelsea J. and Bai, Yunhong and Bis-Brewer, Dana M. and Bugiardini, Enrico and Buglo, Elena and Danzi, Matt C. and Feely, Shawna M. E. and Athanasiou-Fragkouli, Alkyoni and Haridy, Nourelhoda A. and Isasi, Rosario and Khan, Alaa and Laurà, Matilde and Magri, Stefania and Pipis, Menelaos and Pisciotta, Chiara and Powell, Eric and Rossor, Alexander M. and Saveri, Paola and Sowden, Janet E. and Tozza, Stefano and Vandrovcova, Jana and Dallman, Julia and Grignani, Elena and Marchioni, Enrico and Scherer, Steven S. and Tang, Beisha and Lin, Zhiqiang and Al-Ajmi, Abdullah and Schüle, Rebecca and Synofzik, Matthis and Maisonobe, Thierry and Stojkovic, Tanya and Auer-Grumbach, Michaela and Abdelhamed, Mohamed A. and Hamed, Sherifa A. and Zhang, Ruxu and Manganelli, Fiore and Santoro, Lucio and Taroni, Franco and Pareyson, Davide and Houlden, Henry and Herrmann, David N. and Reilly, Mary M. and Shy, Michael E. and Zhai, R. Grace and Zuchner, Stephan},
  doi          = {10.1038/s41588-020-0649-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {640},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Biallelic mutations in SORD cause a common and potentially treatable hereditary neuropathy with implications for diabetes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Scalable generalized linear mixed model for region-based
association tests in large biobanks and cohorts. <em>NG</em>,
<em>52</em>(6), 634–639. (<a
href="https://doi.org/10.1038/s41588-020-0621-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With very large sample sizes, biobanks provide an exciting opportunity to identify genetic components of complex traits. To analyze rare variants, region-based multiple-variant aggregate tests are commonly used to increase power for association tests. However, because of the substantial computational cost, existing region-based tests cannot analyze hundreds of thousands of samples while accounting for confounders such as population stratification and sample relatedness. Here we propose a scalable generalized mixed-model region-based association test, SAIGE-GENE, that is applicable to exome-wide and genome-wide region-based analysis for hundreds of thousands of samples and can account for unbalanced case–control ratios for binary traits. Through extensive simulation studies and analysis of the HUNT study with 69,716 Norwegian samples and the UK Biobank data with 408,910 White British samples, we show that SAIGE-GENE can efficiently analyze large-sample data (N &amp;gt; 400,000) with type I error rates well controlled. SAIGE-GENE is a scalable generalized mixed-model region-based association test that can analyze large datasets while accounting for sample relatedness and unbalanced case–control ratios for binary traits.},
  archive      = {J_NG},
  author       = {Zhou, Wei and Zhao, Zhangchen and Nielsen, Jonas B. and Fritsche, Lars G. and LeFaive, Jonathon and Gagliano Taliun, Sarah A. and Bi, Wenjian and Gabrielsen, Maiken E. and Daly, Mark J. and Neale, Benjamin M. and Hveem, Kristian and Abecasis, Goncalo R. and Willer, Cristen J. and Lee, Seunggeun},
  doi          = {10.1038/s41588-020-0621-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {634-639},
  shortjournal = {Nat. Genet.},
  title        = {Scalable generalized linear mixed model for region-based association tests in large biobanks and cohorts},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Quantifying genetic effects on disease mediated by assayed
gene expression levels. <em>NG</em>, <em>52</em>(6), 626–633. (<a
href="https://doi.org/10.1038/s41588-020-0625-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Disease variants identified by genome-wide association studies (GWAS) tend to overlap with expression quantitative trait loci (eQTLs), but it remains unclear whether this overlap is driven by gene expression levels ‘mediating’ genetic effects on disease. Here, we introduce a new method, mediated expression score regression (MESC), to estimate disease heritability mediated by the cis genetic component of gene expression levels. We applied MESC to GWAS summary statistics for 42 traits (average N = 323,000) and cis-eQTL summary statistics for 48 tissues from the Genotype-Tissue Expression (GTEx) consortium. Averaging across traits, only 11 ± 2% of heritability was mediated by assayed gene expression levels. Expression-mediated heritability was enriched in genes with evidence of selective constraint and genes with disease-appropriate annotations. Our results demonstrate that assayed bulk tissue eQTLs, although disease relevant, cannot explain the majority of disease heritability. Mediated expression score regression (MESC) is a new method that estimates disease heritability mediated by the cis genetic component of gene expression levels by using summary statistics from GWAS and eQTL studies.},
  archive      = {J_NG},
  author       = {Yao, Douglas W. and O’Connor, Luke J. and Price, Alkes L. and Gusev, Alexander},
  doi          = {10.1038/s41588-020-0625-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {626-633},
  shortjournal = {Nat. Genet.},
  title        = {Quantifying genetic effects on disease mediated by assayed gene expression levels},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Uncoupling histone H3K4 trimethylation from developmental
gene expression via an equilibrium of COMPASS, polycomb and DNA
methylation. <em>NG</em>, <em>52</em>(6), 615–625. (<a
href="https://doi.org/10.1038/s41588-020-0618-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COMPASS protein family catalyzes histone H3 Lys 4 (H3K4) methylation and its members are essential for regulating gene expression. MLL2/COMPASS methylates H3K4 on many developmental genes and bivalent clusters. To understand MLL2-dependent transcriptional regulation, we performed a CRISPR-based screen with an MLL2-dependent gene as a reporter in mouse embryonic stem cells. We found that MLL2 functions in gene expression by protecting developmental genes from repression via repelling PRC2 and DNA methylation machineries. Accordingly, repression in the absence of MLL2 is relieved by inhibition of PRC2 and DNA methyltransferases. Furthermore, DNA demethylation on such loci leads to reactivation of MLL2-dependent genes not only by removing DNA methylation but also by opening up previously CpG methylated regions for PRC2 recruitment, diluting PRC2 at Polycomb-repressed genes. These findings reveal how the context and function of these three epigenetic modifiers of chromatin can orchestrate transcriptional decisions and demonstrate that prevention of active repression by the context of the enzyme and not H3K4 trimethylation underlies transcriptional regulation on MLL2/COMPASS targets. A CRISPR screen for MLL2-dependent transcription identifies a compensatory repressive mechanism involving PRC2 and DNA methylation. PRC2 inactivation or DNA demethylation activates gene expression in the absence of H3K4me3.},
  archive      = {J_NG},
  author       = {Douillet, Delphine and Sze, Christie C. and Ryan, Caila and Piunti, Andrea and Shah, Avani P. and Ugarenko, Michal and Marshall, Stacy A. and Rendleman, Emily J. and Zha, Didi and Helmin, Kathryn A. and Zhao, Zibo and Cao, Kaixiang and Morgan, Marc A. and Singer, Benjamin D. and Bartom, Elizabeth T. and Smith, Edwin R. and Shilatifard, Ali},
  doi          = {10.1038/s41588-020-0618-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {615-625},
  shortjournal = {Nat. Genet.},
  title        = {Uncoupling histone H3K4 trimethylation from developmental gene expression via an equilibrium of COMPASS, polycomb and DNA methylation},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Spatial competition shapes the dynamic mutational landscape
of normal esophageal epithelium. <em>NG</em>, <em>52</em>(6), 604–614.
(<a href="https://doi.org/10.1038/s41588-020-0624-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During aging, progenitor cells acquire mutations, which may generate clones that colonize the surrounding tissue. By middle age, normal human tissues, including the esophageal epithelium (EE), become a patchwork of mutant clones. Despite their relevance for understanding aging and cancer, the processes that underpin mutational selection in normal tissues remain poorly understood. Here, we investigated this issue in the esophageal epithelium of mutagen-treated mice. Deep sequencing identified numerous mutant clones with multiple genes under positive selection, including Notch1, Notch2 and Trp53, which are also selected in human esophageal epithelium. Transgenic lineage tracing revealed strong clonal competition that evolved over time. Clone dynamics were consistent with a simple model in which the proliferative advantage conferred by positively selected mutations depends on the nature of the neighboring cells. When clones with similar competitive fitness collide, mutant cell fate reverts towards homeostasis, a constraint that explains how selection operates in normal-appearing epithelium. Deep sequencing and lineage tracing analysis of esophageal epithelium of mutagen-treated aging mice leads to a model in which the proliferative advantage of positively selected mutations depends on the competitive fitness of neighboring cells.},
  archive      = {J_NG},
  author       = {Colom, Bartomeu and Alcolea, Maria P. and Piedrafita, Gabriel and Hall, Michael W. J. and Wabik, Agnieszka and Dentro, Stefan C. and Fowler, Joanna C. and Herms, Albert and King, Charlotte and Ong, Swee Hoe and Sood, Roshan K. and Gerstung, Moritz and Martincorena, Inigo and Hall, Benjamin A. and Jones, Philip H.},
  doi          = {10.1038/s41588-020-0624-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {604-614},
  shortjournal = {Nat. Genet.},
  title        = {Spatial competition shapes the dynamic mutational landscape of normal esophageal epithelium},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Lineage-dependent gene expression programs influence the
immune landscape of colorectal cancer. <em>NG</em>, <em>52</em>(6),
594–603. (<a href="https://doi.org/10.1038/s41588-020-0636-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immunotherapy for metastatic colorectal cancer is effective only for mismatch repair-deficient tumors with high microsatellite instability that demonstrate immune infiltration, suggesting that tumor cells can determine their immune microenvironment. To understand this cross-talk, we analyzed the transcriptome of 91,103 unsorted single cells from 23 Korean and 6 Belgian patients. Cancer cells displayed transcriptional features reminiscent of normal differentiation programs, and genetic alterations that apparently fostered immunosuppressive microenvironments directed by regulatory T cells, myofibroblasts and myeloid cells. Intercellular network reconstruction supported the association between cancer cell signatures and specific stromal or immune cell populations. Our collective view of the cellular landscape and intercellular interactions in colorectal cancer provide mechanistic information for the design of efficient immuno-oncology treatment strategies. Single-cell RNA sequencing of colorectal tumors highlights associations between lineage-related gene expression signatures and specific stromal and immune cell populations.},
  archive      = {J_NG},
  author       = {Lee, Hae-Ock and Hong, Yourae and Etlioglu, Hakki Emre and Cho, Yong Beom and Pomella, Valentina and Van den Bosch, Ben and Vanhecke, Jasper and Verbandt, Sara and Hong, Hyekyung and Min, Jae-Woong and Kim, Nayoung and Eum, Hye Hyeon and Qian, Junbin and Boeckx, Bram and Lambrechts, Diether and Tsantoulis, Petros and De Hertogh, Gert and Chung, Woosung and Lee, Taeseob and An, Minae and Shin, Hyun-Tae and Joung, Je-Gun and Jung, Min-Hyeok and Ko, Gunhwan and Wirapati, Pratyaksha and Kim, Seok Hyung and Kim, Hee Cheol and Yun, Seong Hyeon and Tan, Iain Bee Huat and Ranjan, Bobby and Lee, Woo Yong and Kim, Tae-You and Choi, Jung Kyoon and Kim, Young-Joon and Prabhakar, Shyam and Tejpar, Sabine and Park, Woong-Yang},
  doi          = {10.1038/s41588-020-0636-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {594-603},
  shortjournal = {Nat. Genet.},
  title        = {Lineage-dependent gene expression programs influence the immune landscape of colorectal cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Unraveling tumor–immune heterogeneity in advanced ovarian
cancer uncovers immunogenic effect of chemotherapy. <em>NG</em>,
<em>52</em>(6), 582–593. (<a
href="https://doi.org/10.1038/s41588-020-0630-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In metastatic cancer, the degree of heterogeneity of the tumor microenvironment (TME) and its molecular underpinnings remain largely unstudied. To characterize the tumor–immune interface at baseline and during neoadjuvant chemotherapy (NACT) in high-grade serous ovarian cancer (HGSOC), we performed immunogenomic analysis of treatment-naive and paired samples from before and after treatment with chemotherapy. In treatment-naive HGSOC, we found that immune-cell-excluded and inflammatory microenvironments coexist within the same individuals and within the same tumor sites, indicating ubiquitous variability in immune cell infiltration. Analysis of TME cell composition, DNA copy number, mutations and gene expression showed that immune cell exclusion was associated with amplification of Myc target genes and increased expression of canonical Wnt signaling in treatment-naive HGSOC. Following NACT, increased natural killer (NK) cell infiltration and oligoclonal expansion of T cells were detected. We demonstrate that the tumor–immune microenvironment of advanced HGSOC is intrinsically heterogeneous and that chemotherapy induces local immune activation, suggesting that chemotherapy can potentiate the immunogenicity of immune-excluded HGSOC tumors. Immunogenomic analyses of advanced high-grade serous ovarian cancer samples before and after neoadjuvant chemotherapy show that the tumor–immune microenvironment is intrinsically heterogeneous and that chemotherapy induces local immune activation.},
  archive      = {J_NG},
  author       = {Jiménez-Sánchez, Alejandro and Cybulska, Paulina and Mager, Katherine LaVigne and Koplev, Simon and Cast, Oliver and Couturier, Dominique-Laurent and Memon, Danish and Selenica, Pier and Nikolovski, Ines and Mazaheri, Yousef and Bykov, Yonina and Geyer, Felipe C. and Macintyre, Geoff and Gavarró, Lena Morrill and Drews, Ruben M. and Gill, Michael B. and Papanastasiou, Anastasios D. and Sosa, Ramon E. and Soslow, Robert A. and Walther, Tyler and Shen, Ronglai and Chi, Dennis S. and Park, Kay J. and Hollmann, Travis and Reis-Filho, Jorge S. and Markowetz, Florian and Beltrao, Pedro and Vargas, Hebert Alberto and Zamarin, Dmitriy and Brenton, James D. and Snyder, Alexandra and Weigelt, Britta and Sala, Evis and Miller, Martin L.},
  doi          = {10.1038/s41588-020-0630-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {582-593},
  shortjournal = {Nat. Genet.},
  title        = {Unraveling tumor–immune heterogeneity in advanced ovarian cancer uncovers immunogenic effect of chemotherapy},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genome-wide association study identifies 32 novel breast
cancer susceptibility loci from overall and subtype-specific analyses.
<em>NG</em>, <em>52</em>(6), 572–581. (<a
href="https://doi.org/10.1038/s41588-020-0609-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breast cancer susceptibility variants frequently show heterogeneity in associations by tumor subtype1–3. To identify novel loci, we performed a genome-wide association study including 133,384 breast cancer cases and 113,789 controls, plus 18,908 BRCA1 mutation carriers (9,414 with breast cancer) of European ancestry, using both standard and novel methodologies that account for underlying tumor heterogeneity by estrogen receptor, progesterone receptor and human epidermal growth factor receptor 2 status and tumor grade. We identified 32 novel susceptibility loci (P &amp;lt; 5.0 × 10−8), 15 of which showed evidence for associations with at least one tumor feature (false discovery rate &amp;lt; 0.05). Five loci showed associations (P &amp;lt; 0.05) in opposite directions between luminal and non-luminal subtypes. In silico analyses showed that these five loci contained cell-specific enhancers that differed between normal luminal and basal mammary cells. The genetic correlations between five intrinsic-like subtypes ranged from 0.35 to 0.80. The proportion of genome-wide chip heritability explained by all known susceptibility loci was 54.2% for luminal A-like disease and 37.6% for triple-negative disease. The odds ratios of polygenic risk scores, which included 330 variants, for the highest 1% of quantiles compared with middle quantiles were 5.63 and 3.02 for luminal A-like and triple-negative disease, respectively. These findings provide an improved understanding of genetic predisposition to breast cancer subtypes and will inform the development of subtype-specific polygenic risk scores. Genome-wide analysis identifies 32 loci associated with breast cancer susceptibility, accounting for estrogen receptor, progesterone receptor and human epidermal growth factor receptor 2 status and tumor grade.},
  archive      = {J_NG},
  author       = {Zhang, Haoyu and Ahearn, Thomas U. and Lecarpentier, Julie and Barnes, Daniel and Beesley, Jonathan and Qi, Guanghao and Jiang, Xia and O’Mara, Tracy A. and Zhao, Ni and Bolla, Manjeet K. and Dunning, Alison M. and Dennis, Joe and Wang, Qin and Ful, Zumuruda Abu and Aittomäki, Kristiina and Andrulis, Irene L. and Anton-Culver, Hoda and Arndt, Volker and Aronson, Kristan J. and Arun, Banu K. and Auer, Paul L. and Azzollini, Jacopo and Barrowdale, Daniel and Becher, Heiko and Beckmann, Matthias W. and Behrens, Sabine and Benitez, Javier and Bermisheva, Marina and Bialkowska, Katarzyna and Blanco, Ana and Blomqvist, Carl and Bogdanova, Natalia V. and Bojesen, Stig E. and Bonanni, Bernardo and Bondavalli, Davide and Borg, Ake and Brauch, Hiltrud and Brenner, Hermann and Briceno, Ignacio and Broeks, Annegien and Brucker, Sara Y. and Brüning, Thomas and Burwinkel, Barbara and Buys, Saundra S. and Byers, Helen and Caldés, Trinidad and Caligo, Maria A. and Calvello, Mariarosaria and Campa, Daniele and Castelao, Jose E. and Chang-Claude, Jenny and Chanock, Stephen J. and Christiaens, Melissa and Christiansen, Hans and Chung, Wendy K. and Claes, Kathleen B. M. and Clarke, Christine L. and Cornelissen, Sten and Couch, Fergus J. and Cox, Angela and Cross, Simon S. and Czene, Kamila and Daly, Mary B. and Devilee, Peter and Diez, Orland and Domchek, Susan M. and Dörk, Thilo and Dwek, Miriam and Eccles, Diana M. and Ekici, Arif B. and Evans, D. Gareth and Fasching, Peter A. and Figueroa, Jonine and Foretova, Lenka and Fostira, Florentia and Friedman, Eitan and Frost, Debra and Gago-Dominguez, Manuela and Gapstur, Susan M. and Garber, Judy and García-Sáenz, José A. and Gaudet, Mia M. and Gayther, Simon A. and Giles, Graham G. and Godwin, Andrew K. and Goldberg, Mark S. and Goldgar, David E. and González-Neira, Anna and Greene, Mark H. and Gronwald, Jacek and Guénel, Pascal and Häberle, Lothar and Hahnen, Eric and Haiman, Christopher A. and Hake, Christopher R. and Hall, Per and Hamann, Ute and Harkness, Elaine F. and Heemskerk-Gerritsen, Bernadette A. M. and Hillemanns, Peter and Hogervorst, Frans B. L. and Holleczek, Bernd and Hollestelle, Antoinette and Hooning, Maartje J. and Hoover, Robert N. and Hopper, John L. and Howell, Anthony and Huebner, Hanna and Hulick, Peter J. and Imyanitov, Evgeny N. and Isaacs, Claudine and Izatt, Louise and Jager, Agnes and Jakimovska, Milena and Jakubowska, Anna and James, Paul and Janavicius, Ramunas and Janni, Wolfgang and John, Esther M. and Jones, Michael E. and Jung, Audrey and Kaaks, Rudolf and Kapoor, Pooja Middha and Karlan, Beth Y. and Keeman, Renske and Khan, Sofia and Khusnutdinova, Elza and Kitahara, Cari M. and Ko, Yon-Dschun and Konstantopoulou, Irene and Koppert, Linetta B. and Koutros, Stella and Kristensen, Vessela N. and Laenkholm, Anne-Vibeke and Lambrechts, Diether and Larsson, Susanna C. and Laurent-Puig, Pierre and Lazaro, Conxi and Lazarova, Emilija and Lejbkowicz, Flavio and Leslie, Goska and Lesueur, Fabienne and Lindblom, Annika and Lissowska, Jolanta and Lo, Wing-Yee and Loud, Jennifer T. and Lubinski, Jan and Lukomska, Alicja and MacInnis, Robert J. and Mannermaa, Arto and Manoochehri, Mehdi and Manoukian, Siranoush and Margolin, Sara and Martinez, Maria Elena and Matricardi, Laura and McGuffog, Lesley and McLean, Catriona and Mebirouk, Noura and Meindl, Alfons and Menon, Usha and Miller, Austin and Mingazheva, Elvira and Montagna, Marco and Mulligan, Anna Marie and Mulot, Claire and Muranen, Taru A. and Nathanson, Katherine L. and Neuhausen, Susan L. and Nevanlinna, Heli and Neven, Patrick and Newman, William G. and Nielsen, Finn C. and Nikitina-Zake, Liene and Nodora, Jesse and Offit, Kenneth and Olah, Edith and Olopade, Olufunmilayo I. and Olsson, Håkan and Orr, Nick and Papi, Laura and Papp, Janos and Park-Simon, Tjoung-Won and Parsons, Michael T. and Peissel, Bernard and Peixoto, Ana and Peshkin, Beth and Peterlongo, Paolo and Peto, Julian and Phillips, Kelly-Anne and Piedmonte, Marion and Plaseska-Karanfilska, Dijana and Prajzendanc, Karolina and Prentice, Ross and Prokofyeva, Darya and Rack, Brigitte and Radice, Paolo and Ramus, Susan J. and Rantala, Johanna and Rashid, Muhammad U. and Rennert, Gad and Rennert, Hedy S. and Risch, Harvey A. and Romero, Atocha and Rookus, Matti A. and Rübner, Matthias and Rüdiger, Thomas and Saloustros, Emmanouil and Sampson, Sarah and Sandler, Dale P. and Sawyer, Elinor J. and Scheuner, Maren T. and Schmutzler, Rita K. and Schneeweiss, Andreas and Schoemaker, Minouk J. and Schöttker, Ben and Schürmann, Peter and Senter, Leigha and Sharma, Priyanka and Sherman, Mark E. and Shu, Xiao-Ou and Singer, Christian F. and Smichkoska, Snezhana and Soucy, Penny and Southey, Melissa C. and Spinelli, John J. and Stone, Jennifer and Stoppa-Lyonnet, Dominique and Swerdlow, Anthony J. and Szabo, Csilla I. and Tamimi, Rulla M. and Tapper, William J. and Taylor, Jack A. and Teixeira, Manuel R. and Terry, MaryBeth and Thomassen, Mads and Thull, Darcy L. and Tischkowitz, Marc and Toland, Amanda E. and Tollenaar, Rob A. E. M. and Tomlinson, Ian and Torres, Diana and Troester, Melissa A. and Truong, Thérèse and Tung, Nadine and Untch, Michael and Vachon, Celine M. and van den Ouweland, Ans M. W. and van der Kolk, Lizet E. and van Veen, Elke M. and vanRensburg, Elizabeth J. and Vega, Ana and Wappenschmidt, Barbara and Weinberg, Clarice R. and Weitzel, Jeffrey N. and Wildiers, Hans and Winqvist, Robert and Wolk, Alicja and Yang, Xiaohong R. and Yannoukakos, Drakoulis and Zheng, Wei and Zorn, Kristin K. and Milne, Roger L. and Kraft, Peter and Simard, Jacques and Pharoah, Paul D. P. and Michailidou, Kyriaki and Antoniou, Antonis C. and Schmidt, Marjanka K. and Chenevix-Trench, Georgia and Easton, Douglas F. and Chatterjee, Nilanjan and García-Closas, Montserrat},
  doi          = {10.1038/s41588-020-0609-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {572-581},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study identifies 32 novel breast cancer susceptibility loci from overall and subtype-specific analyses},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genome-wide selection and genetic improvement during modern
maize breeding. <em>NG</em>, <em>52</em>(6), 565–571. (<a
href="https://doi.org/10.1038/s41588-020-0616-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since the development of single-hybrid maize breeding programs in the first half of the twentieth century1, maize yields have increased over sevenfold, and much of that increase can be attributed to tolerance of increased planting density2–4. To explore the genomic basis underlying the dramatic yield increase in maize, we conducted a comprehensive analysis of the genomic and phenotypic changes associated with modern maize breeding through chronological sampling of 350 elite inbred lines representing multiple eras of germplasm from both China and the United States. We document several convergent phenotypic changes in both countries. Using genome-wide association and selection scan methods, we identify 160 loci underlying adaptive agronomic phenotypes and more than 1,800 genomic regions representing the targets of selection during modern breeding. This work demonstrates the use of the breeding-era approach for identifying breeding signatures and lays the foundation for future genomics-enabled maize breeding. Genome-wide association analysis for morphological traits across 350 elite maize inbred lines in Chinese and US germplasm identifies loci and genomic regions representing the targets of selection during modern maize breeding.},
  archive      = {J_NG},
  author       = {Wang, Baobao and Lin, Zechuan and Li, Xin and Zhao, Yongping and Zhao, Binbin and Wu, Guangxia and Ma, Xiaojing and Wang, Hai and Xie, Yurong and Li, Quanquan and Song, Guangshu and Kong, Dexin and Zheng, Zhigang and Wei, Hongbin and Shen, Rongxin and Wu, Hong and Chen, Cuixia and Meng, Zhaodong and Wang, Tianyu and Li, Yu and Li, Xinhai and Chen, Yanhui and Lai, Jinsheng and Hufford, Matthew B. and Ross-Ibarra, Jeffrey and He, Hang and Wang, Haiyang},
  doi          = {10.1038/s41588-020-0616-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {565-571},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide selection and genetic improvement during modern maize breeding},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reply to: Evaluating two different models of peanut’s
origin. <em>NG</em>, <em>52</em>(6), 560–563. (<a
href="https://doi.org/10.1038/s41588-020-0627-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Zhuang, Weijian and Wang, Xiyin and Paterson, Andrew H. and Chen, Hua and Yang, Meng and Zhang, Chong and Sun, Pengchuan and Zheng, Yixiong and Wang, Lihui and Xie, Wenping and Chu, Wenting and Fu, Huiwen and Varshney, Rajeev K.},
  doi          = {10.1038/s41588-020-0627-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {560-563},
  shortjournal = {Nat. Genet.},
  title        = {Reply to: Evaluating two different models of peanut’s origin},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evaluating two different models of peanut’s origin.
<em>NG</em>, <em>52</em>(6), 557–559. (<a
href="https://doi.org/10.1038/s41588-020-0626-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Bertioli, David J. and Abernathy, Brian and Seijo, Guillermo and Clevenger, Josh and Cannon, Steven B.},
  doi          = {10.1038/s41588-020-0626-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {557-559},
  shortjournal = {Nat. Genet.},
  title        = {Evaluating two different models of peanut’s origin},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DNMT3A and TET2 mutations reshape hematopoiesis in opposing
ways. <em>NG</em>, <em>52</em>(6), 554–556. (<a
href="https://doi.org/10.1038/s41588-020-0641-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {TET2 and DNMT3A mutations lead to similar long-term outcomes in blood cancers despite the antagonistic biochemical functions of their encoded proteins. A new study highlights the opposing effects of TET2 and DNMT3A mutations in shaping the early erythroid or myeloid bias of hematopoietic progenitors.},
  archive      = {J_NG},
  author       = {López-Moyado, Isaac F. and Rao, Anjana},
  doi          = {10.1038/s41588-020-0641-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {554-556},
  shortjournal = {Nat. Genet.},
  title        = {DNMT3A and TET2 mutations reshape hematopoiesis in opposing ways},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Clonal competition in a confined space. <em>NG</em>,
<em>52</em>(6), 553–554. (<a
href="https://doi.org/10.1038/s41588-020-0638-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many normal tissues are populated by clonal expansions that compete for space. Colom et al. now show that mutant clones keep other mutant clones in check by effectively annulling one another’s advantages.},
  archive      = {J_NG},
  author       = {Naxerova, Kamila},
  doi          = {10.1038/s41588-020-0638-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {553-554},
  shortjournal = {Nat. Genet.},
  title        = {Clonal competition in a confined space},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Exploring and visualizing large-scale genetic associations
by using PheWeb. <em>NG</em>, <em>52</em>(6), 550–552. (<a
href="https://doi.org/10.1038/s41588-020-0622-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gagliano Taliun, Sarah A. and VandeHaar, Peter and Boughton, Andrew P. and Welch, Ryan P. and Taliun, Daniel and Schmidt, Ellen M. and Zhou, Wei and Nielsen, Jonas B. and Willer, Cristen J. and Lee, Seunggeun and Fritsche, Lars G. and Boehnke, Michael and Abecasis, Gonçalo R.},
  doi          = {10.1038/s41588-020-0622-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {550-552},
  shortjournal = {Nat. Genet.},
  title        = {Exploring and visualizing large-scale genetic associations by using PheWeb},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Going virtual. <em>NG</em>, <em>52</em>(6), 549. (<a
href="https://doi.org/10.1038/s41588-020-0654-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One of the many consequences of the global COVID-19 pandemic is the need for the scientific community to adapt to the cancellation of conferences and events because of travel restrictions and social-distancing guidelines. We have seen a very swift conversion to online meetings, which have allowed for this established form of science communication to continue and opened new avenues for innovation in the reporting of research and discussion of ideas.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0654-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {549},
  shortjournal = {Nat. Genet.},
  title        = {Going virtual},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Cdkn1a deletion improves stem cell
function and lifespan of mice with dysfunctional telomeres without
accelerating cancer formation. <em>NG</em>, <em>52</em>(5), 548. (<a
href="https://doi.org/10.1038/s41588-020-0593-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Choudhury, Aaheli Roy and Ju, Zhenyu and Djojosubroto, Meta W. and Schienke, Andrea and Lechel, Andre and Schaetzlein, Sonja and Jiang, Hong and Stepczynska, Anna and Wang, Chunfang and Buer, Jan and Lee, Han-Woong and von Zglinicki, Thomas and Ganser, Arnold and Schirmacher, Peter and Nakauchi, Hiromitsu and Rudolph, K. Lenhard},
  doi          = {10.1038/s41588-020-0593-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {548},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Cdkn1a deletion improves stem cell function and lifespan of mice with dysfunctional telomeres without accelerating cancer formation},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Liability threshold modeling of case–control status and
family history of disease increases association power. <em>NG</em>,
<em>52</em>(5), 541–547. (<a
href="https://doi.org/10.1038/s41588-020-0613-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Family history of disease can provide valuable information in case–control association studies, but it is currently unclear how to best combine case–control status and family history of disease. We developed an association method based on posterior mean genetic liabilities under a liability threshold model, conditional on case–control status and family history (LT-FH). Analyzing 12 diseases from the UK Biobank (average N = 350,000) we compared LT-FH to genome-wide association without using family history (GWAS) and a previous proxy-based method incorporating family history (GWAX). LT-FH was 63% (standard error (s.e.) 6%) more powerful than GWAS and 36% (s.e. 4%) more powerful than the trait-specific maximum of GWAS and GWAX, based on the number of independent genome-wide-significant loci across all diseases (for example, 690 loci for LT-FH versus 423 for GWAS); relative improvements were similar when applying BOLT-LMM to GWAS, GWAX and LT-FH phenotypes. Thus, LT-FH greatly increases association power when family history of disease is available. A new association method using both case–control status and family history (LT-FH) greatly increases association power in analyses of 12 diseases from the UK Biobank.},
  archive      = {J_NG},
  author       = {Hujoel, Margaux L. A. and Gazal, Steven and Loh, Po-Ru and Patterson, Nick and Price, Alkes L.},
  doi          = {10.1038/s41588-020-0613-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {541-547},
  shortjournal = {Nat. Genet.},
  title        = {Liability threshold modeling of case–control status and family history of disease increases association power},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Identifying genetic variants underlying phenotypic variation
in plants without complete genomes. <em>NG</em>, <em>52</em>(5),
534–540. (<a href="https://doi.org/10.1038/s41588-020-0612-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural variants and presence/absence polymorphisms are common in plant genomes, yet they are routinely overlooked in genome-wide association studies (GWAS). Here, we expand the type of genetic variants detected in GWAS to include major deletions, insertions and rearrangements. We first use raw sequencing data directly to derive short sequences, k-mers, that mark a broad range of polymorphisms independently of a reference genome. We then link k-mers associated with phenotypes to specific genomic regions. Using this approach, we reanalyzed 2,000 traits in Arabidopsis thaliana, tomato and maize populations. Associations identified with k-mers recapitulate those found with SNPs, but with stronger statistical support. Importantly, we discovered new associations with structural variants and with regions missing from reference genomes. Our results demonstrate the power of performing GWAS before linking sequence reads to specific genomic regions, which allows the detection of a wider range of genetic variants responsible for phenotypic variation. Application of a new k-mer-based genome-wide association approach to 2,000 phenotypes in Arabidopsis thaliana, tomato and maize detects new associations with structural variants and with regions missing from reference genomes.},
  archive      = {J_NG},
  author       = {Voichek, Yoav and Weigel, Detlef},
  doi          = {10.1038/s41588-020-0612-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {534-540},
  shortjournal = {Nat. Genet.},
  title        = {Identifying genetic variants underlying phenotypic variation in plants without complete genomes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genomic diversifications of five gossypium allopolyploid
species and their impact on cotton improvement. <em>NG</em>,
<em>52</em>(5), 525–533. (<a
href="https://doi.org/10.1038/s41588-020-0614-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyploidy is an evolutionary innovation for many animals and all flowering plants, but its impact on selection and domestication remains elusive. Here we analyze genome evolution and diversification for all five allopolyploid cotton species, including economically important Upland and Pima cottons. Although these polyploid genomes are conserved in gene content and synteny, they have diversified by subgenomic transposon exchanges that equilibrate genome size, evolutionary rate heterogeneities and positive selection between homoeologs within and among lineages. These differential evolutionary trajectories are accompanied by gene-family diversification and homoeolog expression divergence among polyploid lineages. Selection and domestication drive parallel gene expression similarities in fibers of two cultivated cottons, involving coexpression networks and N6-methyladenosine RNA modifications. Furthermore, polyploidy induces recombination suppression, which correlates with altered epigenetic landscapes and can be overcome by wild introgression. These genomic insights will empower efforts to manipulate genetic recombination and modify epigenetic landscapes and target genes for crop improvement. Sequencing and genomic diversification of five allopolyploid cotton species provide insights into polyploid genome evolution and epigenetic landscapes for cotton improvement.},
  archive      = {J_NG},
  author       = {Chen, Z. Jeffrey and Sreedasyam, Avinash and Ando, Atsumi and Song, Qingxin and De Santiago, Luis M. and Hulse-Kemp, Amanda M. and Ding, Mingquan and Ye, Wenxue and Kirkbride, Ryan C. and Jenkins, Jerry and Plott, Christopher and Lovell, John and Lin, Yu-Ming and Vaughn, Robert and Liu, Bo and Simpson, Sheron and Scheffler, Brian E. and Wen, Li and Saski, Christopher A. and Grover, Corrinne E. and Hu, Guanjing and Conover, Justin L. and Carlson, Joseph W. and Shu, Shengqiang and Boston, Lori B. and Williams, Melissa and Peterson, Daniel G. and McGee, Keith and Jones, Don C. and Wendel, Jonathan F. and Stelly, David M. and Grimwood, Jane and Schmutz, Jeremy},
  doi          = {10.1038/s41588-020-0614-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {525-533},
  shortjournal = {Nat. Genet.},
  title        = {Genomic diversifications of five gossypium allopolyploid species and their impact on cotton improvement},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genome sequence of gossypium herbaceum and genome updates of
gossypium arboreum and gossypium hirsutum provide insights into cotton
a-genome evolution. <em>NG</em>, <em>52</em>(5), 516–524. (<a
href="https://doi.org/10.1038/s41588-020-0607-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Upon assembling the first Gossypium herbaceum (A1) genome and substantially improving the existing Gossypium arboreum (A2) and Gossypium hirsutum ((AD)1) genomes, we showed that all existing A-genomes may have originated from a common ancestor, referred to here as A0, which was more phylogenetically related to A1 than A2. Further, allotetraploid formation was shown to have preceded the speciation of A1 and A2. Both A-genomes evolved independently, with no ancestor–progeny relationship. Gaussian probability density function analysis indicates that several long-terminal-repeat bursts that occurred from 5.7 million years ago to less than 0.61 million years ago contributed compellingly to A-genome size expansion, speciation and evolution. Abundant species-specific structural variations in genic regions changed the expression of many important genes, which may have led to fiber cell improvement in (AD)1. Our findings resolve existing controversial concepts surrounding A-genome origins and provide valuable genomic resources for cotton genetic improvement. Assembly of the first Gossypium herbaceum genome and improved Gossypium arboreum and Gossypium hirsutum genomes provide insights into the phylogenetic relationships and origin history of cotton A-genomes.},
  archive      = {J_NG},
  author       = {Huang, Gai and Wu, Zhiguo and Percy, Richard G. and Bai, Mingzhou and Li, Yang and Frelichowski, James E. and Hu, Jiang and Wang, Kun and Yu, John Z. and Zhu, Yuxian},
  doi          = {10.1038/s41588-020-0607-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {516-524},
  shortjournal = {Nat. Genet.},
  title        = {Genome sequence of gossypium herbaceum and genome updates of gossypium arboreum and gossypium hirsutum provide insights into cotton A-genome evolution},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Spt5-mediated enhancer transcription directly couples
enhancer activation with physical promoter interaction. <em>NG</em>,
<em>52</em>(5), 505–515. (<a
href="https://doi.org/10.1038/s41588-020-0605-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Active enhancers are frequently transcribed, yet the regulatory role of enhancer transcription remains debated. Here, we depleted the RNA polymerase II pausing and elongation factor Spt5 in activated mouse B cells and found that approximately 50% of enhancer–gene pairs showed co-regulated transcription, consistent with a potential functional requirement for enhancer transcription. In particular, Spt5 depletion led to loss of super-enhancer–promoter physical interaction and gene expression at the immunoglobulin heavy-chain locus (Igh), abrogating antibody class switch recombination. This defect correlated strictly with loss of enhancer transcription but did not affect acetylation of histone H3 at lysine 27, chromatin accessibility and occupancy of Mediator and cohesin at the enhancer. Strikingly, CRISPRa-mediated rescue of enhancer transcription in Spt5-depleted cells restored Igh gene expression. Our work suggests that Spt5-mediated enhancer transcription underlies the physical and functional interaction between a subset of active enhancers and their target promoters. Depletion of RNA polymerase II pausing and elongation factor Spt5 in B cells indicates that ~50% of enhancer–gene pairs show co-regulated transcription. CRISPRa-mediated rescue of enhancer transcription in Spt5-depleted cells restores Igh gene expression.},
  archive      = {J_NG},
  author       = {Fitz, Johanna and Neumann, Tobias and Steininger, Monika and Wiedemann, Eva-Maria and Garcia, Adriana Cantoran and Athanasiadis, Alexander and Schoeberl, Ursula E. and Pavri, Rushad},
  doi          = {10.1038/s41588-020-0605-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {505-515},
  shortjournal = {Nat. Genet.},
  title        = {Spt5-mediated enhancer transcription directly couples enhancer activation with physical promoter interaction},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genome-wide association meta-analyses combining multiple
risk phenotypes provide insights into the genetic architecture of
cutaneous melanoma susceptibility. <em>NG</em>, <em>52</em>(5), 494–504.
(<a href="https://doi.org/10.1038/s41588-020-0611-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most genetic susceptibility to cutaneous melanoma remains to be discovered. Meta-analysis genome-wide association study (GWAS) of 36,760 cases of melanoma (67% newly genotyped) and 375,188 controls identified 54 significant (P &amp;lt; 5 × 10−8) loci with 68 independent single nucleotide polymorphisms. Analysis of risk estimates across geographical regions and host factors suggests the acral melanoma subtype is uniquely unrelated to pigmentation. Combining this meta-analysis with GWAS of nevus count and hair color, and transcriptome association approaches, uncovered 31 potential secondary loci for a total of 85 cutaneous melanoma susceptibility loci. These findings provide insights into cutaneous melanoma genetic architecture, reinforcing the importance of nevogenesis, pigmentation and telomere maintenance, together with identifying potential new pathways for cutaneous melanoma pathogenesis. Meta-analysis of 36,760 cases and 375,188 controls identifies 54 loci associated with susceptibility to cutaneous melanoma. Further analysis combining nevus count and hair color GWAS results provide insights into the genetic architecture of melanoma.},
  archive      = {J_NG},
  author       = {Landi, Maria Teresa and Bishop, D. Timothy and MacGregor, Stuart and Machiela, Mitchell J. and Stratigos, Alexander J. and Ghiorzo, Paola and Brossard, Myriam and Calista, Donato and Choi, Jiyeon and Fargnoli, Maria Concetta and Zhang, Tongwu and Rodolfo, Monica and Trower, Adam J. and Menin, Chiara and Martinez, Jacobo and Hadjisavvas, Andreas and Song, Lei and Stefanaki, Irene and Scolyer, Richard and Yang, Rose and Goldstein, Alisa M. and Potrony, Miriam and Kypreou, Katerina P. and Pastorino, Lorenza and Queirolo, Paola and Pellegrini, Cristina and Cattaneo, Laura and Zawistowski, Matthew and Gimenez-Xavier, Pol and Rodriguez, Arantxa and Elefanti, Lisa and Manoukian, Siranoush and Rivoltini, Licia and Smith, Blair H. and Loizidou, Maria A. and Del Regno, Laura and Massi, Daniela and Mandala, Mario and Khosrotehrani, Kiarash and Akslen, Lars A. and Amos, Christopher I. and Andresen, Per A. and Avril, Marie-Françoise and Azizi, Esther and Soyer, H. Peter and Bataille, Veronique and Dalmasso, Bruna and Bowdler, Lisa M. and Burdon, Kathryn P. and Chen, Wei V. and Codd, Veryan and Craig, Jamie E. and Dębniak, Tadeusz and Falchi, Mario and Fang, Shenying and Friedman, Eitan and Simi, Sarah and Galan, Pilar and Garcia-Casado, Zaida and Gillanders, Elizabeth M. and Gordon, Scott and Green, Adele and Gruis, Nelleke A. and Hansson, Johan and Harland, Mark and Harris, Jessica and Helsing, Per and Henders, Anjali and Hočevar, Marko and Höiom, Veronica and Hunter, David and Ingvar, Christian and Kumar, Rajiv and Lang, Julie and Lathrop, G. Mark and Lee, Jeffrey E. and Li, Xin and Lubiński, Jan and Mackie, Rona M. and Malt, Maryrose and Malvehy, Josep and McAloney, Kerrie and Mohamdi, Hamida and Molven, Anders and Moses, Eric K. and Neale, Rachel E. and Novaković, Srdjan and Nyholt, Dale R. and Olsson, Håkan and Orr, Nicholas and Fritsche, Lars G. and Puig-Butille, Joan Anton and Qureshi, Abrar A. and Radford-Smith, Graham L. and Randerson-Moor, Juliette and Requena, Celia and Rowe, Casey and Samani, Nilesh J. and Sanna, Marianna and Schadendorf, Dirk and Schulze, Hans-Joachim and Simms, Lisa A. and Smithers, Mark and Song, Fengju and Swerdlow, Anthony J. and van der Stoep, Nienke and Kukutsch, Nicole A. and Visconti, Alessia and Wallace, Leanne and Ward, Sarah V. and Wheeler, Lawrie and Sturm, Richard A. and Hutchinson, Amy and Jones, Kristine and Malasky, Michael and Vogt, Aurelie and Zhou, Weiyin and Pooley, Karen A. and Elder, David E. and Han, Jiali and Hicks, Belynda and Hayward, Nicholas K. and Kanetsky, Peter A. and Brummett, Chad and Montgomery, Grant W. and Olsen, Catherine M. and Hayward, Caroline and Dunning, Alison M. and Martin, Nicholas G. and Evangelou, Evangelos and Mann, Graham J. and Long, Georgina and Pharoah, Paul D. P. and Easton, Douglas F. and Barrett, Jennifer H. and Cust, Anne E. and Abecasis, Goncalo and Duffy, David L. and Whiteman, David C. and Gogas, Helen and De Nicolo, Arcangela and Tucker, Margaret A. and Newton-Bishop, Julia A. and Peris, Ketty and Chanock, Stephen J. and Demenais, Florence and Brown, Kevin M. and Puig, Susana and Nagore, Eduardo and Shi, Jianxin and Iles, Mark M. and Law, Matthew H.},
  doi          = {10.1038/s41588-020-0611-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {494-504},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analyses combining multiple risk phenotypes provide insights into the genetic architecture of cutaneous melanoma susceptibility},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genetic identification of cell types underlying brain
complex traits yields insights into the etiology of parkinson’s disease.
<em>NG</em>, <em>52</em>(5), 482–493. (<a
href="https://doi.org/10.1038/s41588-020-0610-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have discovered hundreds of loci associated with complex brain disorders, but it remains unclear in which cell types these loci are active. Here we integrate genome-wide association study results with single-cell transcriptomic data from the entire mouse nervous system to systematically identify cell types underlying brain complex traits. We show that psychiatric disorders are predominantly associated with projecting excitatory and inhibitory neurons. Neurological diseases were associated with different cell types, which is consistent with other lines of evidence. Notably, Parkinson’s disease was genetically associated not only with cholinergic and monoaminergic neurons (which include dopaminergic neurons) but also with enteric neurons and oligodendrocytes. Using post-mortem brain transcriptomic data, we confirmed alterations in these cells, even at the earliest stages of disease progression. Our study provides an important framework for understanding the cellular basis of complex brain maladies, and reveals an unexpected role of oligodendrocytes in Parkinson’s disease. Integration of GWAS and single-cell transcriptomic data from the entire nervous system systematically identifies cell types underlying brain complex traits and provides insights into the etiology of Parkinson’s disease.},
  archive      = {J_NG},
  author       = {Bryois, Julien and Skene, Nathan G. and Hansen, Thomas Folkmann and Kogelman, Lisette J. A. and Watson, Hunna J. and Liu, Zijing and Brueggeman, Leo and Breen, Gerome and Bulik, Cynthia M. and Arenas, Ernest and Hjerling-Leffler, Jens and Sullivan, Patrick F.},
  doi          = {10.1038/s41588-020-0610-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {482-493},
  shortjournal = {Nat. Genet.},
  title        = {Genetic identification of cell types underlying brain complex traits yields insights into the etiology of parkinson’s disease},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Biallelic mutations in SORD cause a common and potentially
treatable hereditary neuropathy with implications for diabetes.
<em>NG</em>, <em>52</em>(5), 473–481. (<a
href="https://doi.org/10.1038/s41588-020-0615-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we report biallelic mutations in the sorbitol dehydrogenase gene (SORD) as the most frequent recessive form of hereditary neuropathy. We identified 45 individuals from 38 families across multiple ancestries carrying the nonsense c.757delG (p.Ala253GlnfsTer27) variant in SORD, in either a homozygous or compound heterozygous state. SORD is an enzyme that converts sorbitol into fructose in the two-step polyol pathway previously implicated in diabetic neuropathy. In patient-derived fibroblasts, we found a complete loss of SORD protein and increased intracellular sorbitol. Furthermore, the serum fasting sorbitol levels in patients were dramatically increased. In Drosophila, loss of SORD orthologs caused synaptic degeneration and progressive motor impairment. Reducing the polyol influx by treatment with aldose reductase inhibitors normalized intracellular sorbitol levels in patient-derived fibroblasts and in Drosophila, and also dramatically ameliorated motor and eye phenotypes. Together, these findings establish a novel and potentially treatable cause of neuropathy and may contribute to a better understanding of the pathophysiology of diabetes. Biallelic mutations in the sorbitol dehydrogenase gene SORD are identified as a common cause of hereditary neuropathy. Functional studies suggest that SORD deficiency may be treatable with aldose reductase inhibitors.},
  archive      = {J_NG},
  author       = {Cortese, Andrea and Zhu, Yi and Rebelo, Adriana P. and Negri, Sara and Courel, Steve and Abreu, Lisa and Bacon, Chelsea J. and Bai, Yunhong and Bis-Brewer, Dana M. and Bugiardini, Enrico and Buglo, Elena and Danzi, Matt C. and Feely, Shawna M. E. and Athanasiou-Fragkouli, Alkyoni and Haridy, Nourelhoda A. and Isasi, Rosario and Khan, Alaa and Laurà, Matilde and Magri, Stefania and Pipis, Menelaos and Pisciotta, Chiara and Powell, Eric and Rossor, Alexander M. and Saveri, Paola and Sowden, Janet E. and Tozza, Stefano and Vandrovcova, Jana and Dallman, Julia and Grignani, Elena and Marchioni, Enrico and Scherer, Steven S. and Tang, Beisha and Lin, Zhiqiang and Al-Ajmi, Abdullah and Schüle, Rebecca and Synofzik, Matthis and Maisonobe, Thierry and Stojkovic, Tanya and Auer-Grumbach, Michaela and Abdelhamed, Mohamed A. and Hamed, Sherifa A. and Zhang, Ruxu and Manganelli, Fiore and Santoro, Lucio and Taroni, Franco and Pareyson, Davide and Houlden, Henry and Herrmann, David N. and Reilly, Mary M. and Shy, Michael E. and Zhai, R. Grace and Zuchner, Stephan},
  doi          = {10.1038/s41588-020-0615-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {473-481},
  shortjournal = {Nat. Genet.},
  title        = {Biallelic mutations in SORD cause a common and potentially treatable hereditary neuropathy with implications for diabetes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Enhancer–promoter interactions and transcription.
<em>NG</em>, <em>52</em>(5), 470–471. (<a
href="https://doi.org/10.1038/s41588-020-0620-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study addresses whether transcription of enhancers and the resulting enhancer RNAs (eRNAs) play a role in mediating long-range interactions between enhancers and promoters. Studying the immunoglobulin heavy chain (Igh) locus, the authors find that transcription of the enhancers per se is required to establish but not maintain these interactions, and this mechanism may apply to a subset of other enhancer–promoter interactions.},
  archive      = {J_NG},
  author       = {Higgs, Douglas R.},
  doi          = {10.1038/s41588-020-0620-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {470-471},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer–promoter interactions and transcription},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Elevated sorbitol underlies a heritable neuropathy.
<em>NG</em>, <em>52</em>(5), 469–470. (<a
href="https://doi.org/10.1038/s41588-020-0619-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study identifies sorbitol dehydrogenase (SORD) deficiency as a slowly progressive hereditary motor axonopathy caused by a genetic defect in the second step of the polyol pathway, thus leading to elevated tissue and blood sorbitol. SORD deficiency is the most common recessive cause of neuropathy, for which therapeutic intervention with aldose reductase inhibitors may have potential.},
  archive      = {J_NG},
  author       = {Morava, Eva},
  doi          = {10.1038/s41588-020-0619-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {469-470},
  shortjournal = {Nat. Genet.},
  title        = {Elevated sorbitol underlies a heritable neuropathy},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Establishing the international genetic discrimination
observatory. <em>NG</em>, <em>52</em>(5), 466–468. (<a
href="https://doi.org/10.1038/s41588-020-0606-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic discrimination is one of the most pervasive challenges resulting from research and development in human genetics. To collaboratively study and prevent this ethical issue, we established an international Genetic Discrimination Observatory comprising a network of researchers and stakeholders from more than 19 jurisdictions.},
  archive      = {J_NG},
  author       = {Joly, Yann and Dalpé, Gratien and Dupras, Charles and Bévière-Boyer, Bénédicte and de Paor, Aisling and Dove, Edward S. and Granados Moreno, Palmira and Ho, Calvin W. L. and Ho, Chih-Hsing and Ó Cathaoir, Katharina and Kato, Kazuto and Kim, Hannah and Song, Lingqiao and Minssen, Timo and Nicolás, Pilar and Otlowski, Margaret and Prince, Anya E. R. and P. S. Nair, Athira and Van Hoyweghen, Ine and Voigt, Torsten H. and Yamasaki, Chisato and Bombard, Yvonne},
  doi          = {10.1038/s41588-020-0606-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {466-468},
  shortjournal = {Nat. Genet.},
  title        = {Establishing the international genetic discrimination observatory},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Tracking genetic discrimination. <em>NG</em>,
<em>52</em>(5), 465. (<a
href="https://doi.org/10.1038/s41588-020-0632-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The promise of personalized medicine lies in the tailored treatment of individual patients, a process requiring detailed phenotypic and genetic information. Although the widespread collection of such data can help to advance the implementation of precision healthcare, the genomic sequencing data being amassed also include private information that could potentially be used as a basis for genetic discrimination. It is important for the genetics community to be aware of these risks and to contribute to policies designed to monitor and mitigate threats to the equitable treatment of individuals or populations on the basis of genetics.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0632-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {465},
  shortjournal = {Nat. Genet.},
  title        = {Tracking genetic discrimination},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Human somatic cell mutagenesis creates
genetically tractable sarcomas. <em>NG</em>, <em>52</em>(4), 464. (<a
href="https://doi.org/10.1038/s41588-020-0589-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Molyneux, Sam D. and Waterhouse, Paul D. and Shelton, Dawne and Shao, Yang W. and Watling, Christopher M. and Tang, Qing-Lian and Harris, Isaac S. and Dickson, Brendan C. and Tharmapalan, Pirashaanthy and Sandve, Geir K. and Zhang, Xiaoyang and Bailey, Swneke D. and Berman, Hal and Wunder, Jay S. and Izsvák, Zsuzsanna and Lupien, Mathieu and Mak, Tak W. and Khokha, Rama},
  doi          = {10.1038/s41588-020-0589-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {464},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Human somatic cell mutagenesis creates genetically tractable sarcomas},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: Extrachromosomal circular DNA drives
oncogenic genome remodeling in neuroblastoma. <em>NG</em>,
<em>52</em>(4), 464. (<a
href="https://doi.org/10.1038/s41588-020-0598-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Koche, Richard P. and Rodriguez-Fos, Elias and Helmsauer, Konstantin and Burkert, Martin and MacArthur, Ian C. and Maag, Jesper and Chamorro, Rocio and Munoz-Perez, Natalia and Puiggròs, Montserrat and Garcia, Heathcliff Dorado and Bei, Yi and Röefzaad, Claudia and Bardinet, Victor and Szymansky, Annabell and Winkler, Annika and Thole, Theresa and Timme, Natalie and Kasack, Katharina and Fuchs, Steffen and Klironomos, Filippos and Thiessen, Nina and Blanc, Eric and Schmelz, Karin and Künkele, Annette and Hundsdörfer, Patrick and Rosswog, Carolina and Theissen, Jessica and Beule, Dieter and Deubzer, Hedwig and Sauer, Sascha and Toedling, Joern and Fischer, Matthias and Hertwig, Falk and Schwarz, Roland F. and Eggert, Angelika and Torrents, David and Schulte, Johannes H. and Henssen, Anton G.},
  doi          = {10.1038/s41588-020-0598-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {464},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Extrachromosomal circular DNA drives oncogenic genome remodeling in neuroblastoma},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Identification of erythroferrone as an
erythroid regulator of iron metabolism. <em>NG</em>, <em>52</em>(4),
463. (<a href="https://doi.org/10.1038/s41588-019-0548-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Kautz, Léon and Jung, Grace and Valore, Erika V. and Rivella, Stefano and Nemeth, Elizabeta and Ganz, Tomas},
  doi          = {10.1038/s41588-019-0548-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {463},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Identification of erythroferrone as an erythroid regulator of iron metabolism},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Transcription phenotypes of pancreatic
cancer are driven by genomic events during tumor evolution. <em>NG</em>,
<em>52</em>(4), 463. (<a
href="https://doi.org/10.1038/s41588-020-0588-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Chan-Seng-Yue, Michelle and Kim, Jaeseung C. and Wilson, Gavin W. and Ng, Karen and Figueroa, Eugenia Flores and O’Kane, Grainne M. and Connor, Ashton A. and Denroche, Robert E. and Grant, Robert C. and McLeod, Jessica and Wilson, Julie M. and Jang, Gun Ho and Zhang, Amy and Liang, Sheng-Ben and Borgida, Ayelet and Chadwick, Dianne and Kalimuthu, Sangeetha and Lungu, Ilinca and Bartlett, John M. S. and Krzyzanowski, Paul M. and Sandhu, Vandana and Tiriac, Hervé and Froeling, Fieke E. M. and Karasinska, Joanna M. and Topham, James T. and Renouf, Daniel J. and Schaeffer, David F. and Jones, Steven J. M. and Marra, Marco A. and Laskin, Janessa and Chetty, Runjan and Stein, Lincoln D. and Zogopoulos, George and Haibe-Kains, Benjamin and Campbell, Peter J. and Tuveson, David A. and Knox, Jennifer J. and Fischer, Sandra E. and Gallinger, Steven and Notta, Faiyaz},
  doi          = {10.1038/s41588-020-0588-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {463},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evaluating and improving heritability models using summary
statistics. <em>NG</em>, <em>52</em>(4), 458–462. (<a
href="https://doi.org/10.1038/s41588-020-0600-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is currently much debate regarding the best model for how heritability varies across the genome. The authors of GCTA recommend the GCTA-LDMS-I model, the authors of LD Score Regression recommend the Baseline LD model, and we have recommended the LDAK model. Here we provide a statistical framework for assessing heritability models using summary statistics from genome-wide association studies. Based on 31 studies of complex human traits (average sample size 136,000), we show that the Baseline LD model is more realistic than other existing heritability models, but that it can be improved by incorporating features from the LDAK model. Our framework also provides a method for estimating the selection-related parameter α from summary statistics. We find strong evidence (P &amp;lt; 1 × 10−6) of negative genome-wide selection for traits, including height, systolic blood pressure and college education, and that the impact of selection is stronger inside functional categories, such as coding SNPs and promoter regions. Assessing heritability models using summary statistics from genome-wide association studies of 31 human traits shows that the Baseline LD model is realistic and can be improved by incorporating features from the LDAK model.},
  archive      = {J_NG},
  author       = {Speed, Doug and Holmes, John and Balding, David J.},
  doi          = {10.1038/s41588-020-0600-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {458-462},
  shortjournal = {Nat. Genet.},
  title        = {Evaluating and improving heritability models using summary statistics},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A harmonized meta-knowledgebase of clinical interpretations
of somatic genomic variants in cancer. <em>NG</em>, <em>52</em>(4),
448–457. (<a href="https://doi.org/10.1038/s41588-020-0603-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precision oncology relies on accurate discovery and interpretation of genomic variants, enabling individualized diagnosis, prognosis and therapy selection. We found that six prominent somatic cancer variant knowledgebases were highly disparate in content, structure and supporting primary literature, impeding consensus when evaluating variants and their relevance in a clinical setting. We developed a framework for harmonizing variant interpretations to produce a meta-knowledgebase of 12,856 aggregate interpretations. We demonstrated large gains in overlap between resources across variants, diseases and drugs as a result of this harmonization. We subsequently demonstrated improved matching between a patient cohort and harmonized interpretations of potential clinical significance, observing an increase from an average of 33% per individual knowledgebase to 57% in aggregate. Our analyses illuminate the need for open, interoperable sharing of variant interpretation data. We also provide a freely available web interface (search.cancervariants.org) for exploring the harmonized interpretations from these six knowledgebases. This analysis presents a harmonized meta-knowledgebase to facilitate clinical interpretation of somatic genomic variants in cancer. This community-based project highlights the need for cooperative efforts to curate clinical interpretations of somatic variants for robust practice of precision oncology.},
  archive      = {J_NG},
  author       = {Wagner, Alex H. and Walsh, Brian and Mayfield, Georgia and Tamborero, David and Sonkin, Dmitriy and Krysiak, Kilannin and Deu-Pons, Jordi and Duren, Ryan P. and Gao, Jianjiong and McMurry, Julie and Patterson, Sara and del Vecchio Fitz, Catherine and Pitel, Beth A. and Sezerman, Ozman U. and Ellrott, Kyle and Warner, Jeremy L. and Rieke, Damian T. and Aittokallio, Tero and Cerami, Ethan and Ritter, Deborah I. and Schriml, Lynn M. and Freimuth, Robert R. and Haendel, Melissa and Raca, Gordana and Madhavan, Subha and Baudis, Michael and Beckmann, Jacques S. and Dienstmann, Rodrigo and Chakravarty, Debyani and Li, Xuan Shirley and Mockus, Susan and Elemento, Olivier and Schultz, Nikolaus and Lopez-Bigas, Nuria and Lawler, Mark and Goecks, Jeremy and Griffith, Malachi and Griffith, Obi L. and Margolin, Adam A.},
  doi          = {10.1038/s41588-020-0603-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {448-457},
  shortjournal = {Nat. Genet.},
  title        = {A harmonized meta-knowledgebase of clinical interpretations of somatic genomic variants in cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Minimal phenotyping yields genome-wide association signals
of low specificity for major depression. <em>NG</em>, <em>52</em>(4),
437–447. (<a href="https://doi.org/10.1038/s41588-020-0594-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Minimal phenotyping refers to the reliance on the use of a small number of self-reported items for disease case identification, increasingly used in genome-wide association studies (GWAS). Here we report differences in genetic architecture between depression defined by minimal phenotyping and strictly defined major depressive disorder (MDD): the former has a lower genotype-derived heritability that cannot be explained by inclusion of milder cases and a higher proportion of the genome contributing to this shared genetic liability with other conditions than for strictly defined MDD. GWAS based on minimal phenotyping definitions preferentially identifies loci that are not specific to MDD, and, although it generates highly predictive polygenic risk scores, the predictive power can be explained entirely by large sample sizes rather than by specificity for MDD. Our results show that reliance on results from minimal phenotyping may bias views of the genetic architecture of MDD and impede the ability to identify pathways specific to MDD. Genetic analyses of depression based on minimal phenotyping identify nonspecific genetic risk factors shared between major depressive disorder (MDD) and other psychiatric conditions, suggesting that this approach may have limited ability to identify pathways specific to MDD.},
  archive      = {J_NG},
  author       = {Cai, Na and Revez, Joana A. and Adams, Mark J. and Andlauer, Till F. M. and Breen, Gerome and Byrne, Enda M. and Clarke, Toni-Kim and Forstner, Andreas J. and Grabe, Hans J. and Hamilton, Steven P. and Levinson, Douglas F. and Lewis, Cathryn M. and Lewis, Glyn and Martin, Nicholas G. and Milaneschi, Yuri and Mors, Ole and Müller-Myhsok, Bertram and Penninx, Brenda W. J. H. and Perlis, Roy H. and Pistis, Giorgio and Potash, James B. and Preisig, Martin and Shi, Jianxin and Smoller, Jordan W. and Streit, Fabien and Tiemeier, Henning and Uher, Rudolf and Van der Auwera, Sandra and Viktorin, Alexander and Weissman, Myrna M. and Kendler, Kenneth S. and Flint, Jonathan},
  doi          = {10.1038/s41588-020-0594-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {437-447},
  shortjournal = {Nat. Genet.},
  title        = {Minimal phenotyping yields genome-wide association signals of low specificity for major depression},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Stepwise selection on homeologous PRR genes controlling
flowering and maturity during soybean domestication. <em>NG</em>,
<em>52</em>(4), 428–436. (<a
href="https://doi.org/10.1038/s41588-020-0604-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adaptive changes in plant phenology are often considered to be a feature of the so-called ‘domestication syndrome’ that distinguishes modern crops from their wild progenitors, but little detailed evidence supports this idea. In soybean, a major legume crop, flowering time variation is well characterized within domesticated germplasm and is critical for modern production, but its importance during domestication is unclear. Here, we identify sequential contributions of two homeologous pseudo-response-regulator genes, Tof12 and Tof11, to ancient flowering time adaptation, and demonstrate that they act via LHY homologs to promote expression of the legume-specific E1 gene and delay flowering under long photoperiods. We show that Tof12-dependent acceleration of maturity accompanied a reduction in dormancy and seed dispersal during soybean domestication, possibly predisposing the incipient crop to latitudinal expansion. Better understanding of this early phase of crop evolution will help to identify functional variation lost during domestication and exploit its potential for future crop improvement. Whole-genome resequencing and association analyses in 424 soybean accessions identify two homeologous genes that contributed to flowering time adaptation during soybean domestication.},
  archive      = {J_NG},
  author       = {Lu, Sijia and Dong, Lidong and Fang, Chao and Liu, Shulin and Kong, Lingping and Cheng, Qun and Chen, Liyu and Su, Tong and Nan, Haiyang and Zhang, Dan and Zhang, Lei and Wang, Zhijuan and Yang, Yongqing and Yu, Deyue and Liu, Xiaolei and Yang, Qingyong and Lin, Xiaoya and Tang, Yang and Zhao, Xiaohui and Yang, Xinquan and Tian, Changen and Xie, Qiguang and Li, Xia and Yuan, Xiaohui and Tian, Zhixi and Liu, Baohui and Weller, James L. and Kong, Fanjiang},
  doi          = {10.1038/s41588-020-0604-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {428-436},
  shortjournal = {Nat. Genet.},
  title        = {Stepwise selection on homeologous PRR genes controlling flowering and maturity during soybean domestication},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Gene network transitions in embryos depend upon interactions
between a pioneer transcription factor and core histones. <em>NG</em>,
<em>52</em>(4), 418–427. (<a
href="https://doi.org/10.1038/s41588-020-0591-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene network transitions in embryos and other fate-changing contexts involve combinations of transcription factors. A subset of fate-changing transcription factors act as pioneers; they scan and target nucleosomal DNA and initiate cooperative events that can open the local chromatin. However, a gap has remained in understanding how molecular interactions with the nucleosome contribute to the chromatin-opening phenomenon. Here we identified a short α-helical region, conserved among FOXA pioneer factors, that interacts with core histones and contributes to chromatin opening in vitro. The same domain is involved in chromatin opening in early mouse embryos for normal development. Thus, local opening of chromatin by interactions between pioneer factors and core histones promotes genetic programming. An α-helical region conserved among FOXA pioneer factors interacts with core histones and promotes chromatin opening in vitro. This region also promotes chromatin opening in early mouse embryos and is required for normal development.},
  archive      = {J_NG},
  author       = {Iwafuchi, Makiko and Cuesta, Isabel and Donahue, Greg and Takenaka, Naomi and Osipovich, Anna B. and Magnuson, Mark A. and Roder, Heinrich and Seeholzer, Steven H. and Santisteban, Pilar and Zaret, Kenneth S.},
  doi          = {10.1038/s41588-020-0591-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {418-427},
  shortjournal = {Nat. Genet.},
  title        = {Gene network transitions in embryos depend upon interactions between a pioneer transcription factor and core histones},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Using antagonistic pleiotropy to design a
chemotherapy-induced evolutionary trap to target drug resistance in
cancer. <em>NG</em>, <em>52</em>(4), 408–417. (<a
href="https://doi.org/10.1038/s41588-020-0590-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Local adaptation directs populations towards environment-specific fitness maxima through acquisition of positively selected traits. However, rapid environmental changes can identify hidden fitness trade-offs that turn adaptation into maladaptation, resulting in evolutionary traps. Cancer, a disease that is prone to drug resistance, is in principle susceptible to such traps. We therefore performed pooled CRISPR–Cas9 knockout screens in acute myeloid leukemia (AML) cells treated with various chemotherapies to map the drug-dependent genetic basis of fitness trade-offs, a concept known as antagonistic pleiotropy (AP). We identified a PRC2–NSD2/3-mediated MYC regulatory axis as a drug-induced AP pathway whose ability to confer resistance to bromodomain inhibition and sensitivity to BCL-2 inhibition templates an evolutionary trap. Across diverse AML cell-line and patient-derived xenograft models, we find that acquisition of resistance to bromodomain inhibition through this pathway exposes coincident hypersensitivity to BCL-2 inhibition. Thus, drug-induced AP can be leveraged to design evolutionary traps that selectively target drug resistance in cancer. CRISPR–Cas9 knockout screens in chemotherapy-treated acute myeloid leukemia cells help map the drug-dependent genetic basis of fitness trade-offs (antagonistic pleiotropy) for the design of evolutionary traps that target drug resistance in cancer.},
  archive      = {J_NG},
  author       = {Lin, Kevin H. and Rutter, Justine C. and Xie, Abigail and Pardieu, Bryann and Winn, Emily T. and Bello, Reinaldo Dal and Forget, Antoine and Itzykson, Raphael and Ahn, Yeong-Ran and Dai, Ziwei and Sobhan, Raiyan T. and Anderson, Gray R. and Singleton, Katherine R. and Decker, Amy E. and Winter, Peter S. and Locasale, Jason W. and Crawford, Lorin and Puissant, Alexandre and Wood, Kris C.},
  doi          = {10.1038/s41588-020-0590-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {408-417},
  shortjournal = {Nat. Genet.},
  title        = {Using antagonistic pleiotropy to design a chemotherapy-induced evolutionary trap to target drug resistance in cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Meta-analysis of 542,934 subjects of european ancestry
identifies new genes and mechanisms predisposing to refractive error and
myopia. <em>NG</em>, <em>52</em>(4), 401–407. (<a
href="https://doi.org/10.1038/s41588-020-0599-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Refractive errors, in particular myopia, are a leading cause of morbidity and disability worldwide. Genetic investigation can improve understanding of the molecular mechanisms that underlie abnormal eye development and impaired vision. We conducted a meta-analysis of genome-wide association studies (GWAS) that involved 542,934 European participants and identified 336 novel genetic loci associated with refractive error. Collectively, all associated genetic variants explain 18.4% of heritability and improve the accuracy of myopia prediction (area under the curve (AUC) = 0.75). Our results suggest that refractive error is genetically heterogeneous, driven by genes that participate in the development of every anatomical component of the eye. In addition, our analyses suggest that genetic factors controlling circadian rhythm and pigmentation are also involved in the development of myopia and refractive error. These results may enable the prediction of refractive error and the development of personalized myopia prevention strategies in the future. Meta-analysis of genome-wide association studies of 542,934 individuals identifies 336 novel loci associated with refractive error and implicates eye development, circadian rhythm and pigmentation pathways in controlling myopia.},
  archive      = {J_NG},
  author       = {Hysi, Pirro G. and Choquet, Hélène and Khawaja, Anthony P. and Wojciechowski, Robert and Tedja, Milly S. and Yin, Jie and Simcoe, Mark J. and Patasova, Karina and Mahroo, Omar A. and Thai, Khanh K. and Cumberland, Phillippa M. and Melles, Ronald B. and Verhoeven, Virginie J. M. and Vitart, Veronique and Segre, Ayellet and Stone, Richard A. and Wareham, Nick and Hewitt, Alex W. and Mackey, David A. and Klaver, Caroline C. W. and MacGregor, Stuart and Khaw, Peng T. and Foster, Paul J. and Guggenheim, Jeremy A. and Rahi, Jugnoo S. and Jorgenson, Eric and Hammond, Christopher J.},
  doi          = {10.1038/s41588-020-0599-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {401-407},
  shortjournal = {Nat. Genet.},
  title        = {Meta-analysis of 542,934 subjects of european ancestry identifies new genes and mechanisms predisposing to refractive error and myopia},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Three-dimensional chromatin landscapes in t cell acute
lymphoblastic leukemia. <em>NG</em>, <em>52</em>(4), 388–400. (<a
href="https://doi.org/10.1038/s41588-020-0602-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Differences in three-dimensional (3D) chromatin architecture can influence the integrity of topologically associating domains (TADs) and rewire specific enhancer–promoter interactions, impacting gene expression and leading to human disease. Here we investigate the 3D chromatin architecture in T cell acute lymphoblastic leukemia (T-ALL) by using primary human leukemia specimens and examine the dynamic responses of this architecture to pharmacological agents. Systematic integration of matched in situ Hi-C, RNA-seq and CTCF ChIP–seq datasets revealed widespread differences in intra-TAD chromatin interactions and TAD boundary insulation in T-ALL. Our studies identify and focus on a TAD ‘fusion’ event associated with absence of CTCF-mediated insulation, enabling direct interactions between the MYC promoter and a distal super-enhancer. Moreover, our data also demonstrate that small-molecule inhibitors targeting either oncogenic signal transduction or epigenetic regulation can alter specific 3D interactions found in leukemia. Overall, our study highlights the impact, complexity and dynamic nature of 3D chromatin architecture in human acute leukemia. Analysis of 3D chromatin architecture in T-ALL identifies differences in intra-TAD interactions and TAD boundary insulation. Inhibition of oncogenic signal transduction or epigenetic regulation can alter specific 3D interactions.},
  archive      = {J_NG},
  author       = {Kloetgen, Andreas and Thandapani, Palaniraja and Ntziachristos, Panagiotis and Ghebrechristos, Yohana and Nomikou, Sofia and Lazaris, Charalampos and Chen, Xufeng and Hu, Hai and Bakogianni, Sofia and Wang, Jingjing and Fu, Yi and Boccalatte, Francesco and Zhong, Hua and Paietta, Elisabeth and Trimarchi, Thomas and Zhu, Yixing and Van Vlierberghe, Pieter and Inghirami, Giorgio G. and Lionnet, Timothee and Aifantis, Iannis and Tsirigos, Aristotelis},
  doi          = {10.1038/s41588-020-0602-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {388-400},
  shortjournal = {Nat. Genet.},
  title        = {Three-dimensional chromatin landscapes in t cell acute lymphoblastic leukemia},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DNA methylation disruption reshapes the hematopoietic
differentiation landscape. <em>NG</em>, <em>52</em>(4), 378–387. (<a
href="https://doi.org/10.1038/s41588-020-0595-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations in genes involved in DNA methylation (DNAme; for example, TET2 and DNMT3A) are frequently observed in hematological malignancies1–3 and clonal hematopoiesis4,5. Applying single-cell sequencing to murine hematopoietic stem and progenitor cells, we observed that these mutations disrupt hematopoietic differentiation, causing opposite shifts in the frequencies of erythroid versus myelomonocytic progenitors following Tet2 or Dnmt3a loss. Notably, these shifts trace back to transcriptional priming skews in uncommitted hematopoietic stem cells. To reconcile genome-wide DNAme changes with specific erythroid versus myelomonocytic skews, we provide evidence in support of differential sensitivity of transcription factors due to biases in CpG enrichment in their binding motif. Single-cell transcriptomes with targeted genotyping showed similar skews in transcriptional priming of DNMT3A-mutated human clonal hematopoiesis bone marrow progenitors. These data show that DNAme shapes the topography of hematopoietic differentiation, and support a model in which genome-wide methylation changes are transduced to differentiation skews through biases in CpG enrichment of the transcription factor binding motif. Single-cell analysis of mouse hematopoietic stem cells shows that mutations in DNA methylation genes change the frequencies of erythroid versus myelomonocytic progenitors, owing to differential CpG enrichment in transcription factor binding motifs.},
  archive      = {J_NG},
  author       = {Izzo, Franco and Lee, Stanley C. and Poran, Asaf and Chaligne, Ronan and Gaiti, Federico and Gross, Baptiste and Murali, Rekha R. and Deochand, Sunil D. and Ang, Chelston and Jones, Philippa Wyndham and Nam, Anna S. and Kim, Kyu-Tae and Kothen-Hill, Steven and Schulman, Rafael C. and Ki, Michelle and Lhoumaud, Priscillia and Skok, Jane A. and Viny, Aaron D. and Levine, Ross L. and Kenigsberg, Ephraim and Abdel-Wahab, Omar and Landau, Dan A.},
  doi          = {10.1038/s41588-020-0595-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {378-387},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation disruption reshapes the hematopoietic differentiation landscape},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genomic characterization of human brain metastases
identifies drivers of metastatic lung adenocarcinoma. <em>NG</em>,
<em>52</em>(4), 371–377. (<a
href="https://doi.org/10.1038/s41588-020-0592-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain metastases from lung adenocarcinoma (BM-LUAD) frequently cause patient mortality. To identify genomic alterations that promote brain metastases, we performed whole-exome sequencing of 73 BM-LUAD cases. Using case-control analyses, we discovered candidate drivers of brain metastasis by identifying genes with more frequent copy-number aberrations in BM-LUAD compared to 503 primary LUADs. We identified three regions with significantly higher amplification frequencies in BM-LUAD, including MYC (12 versus 6%), YAP1 (7 versus 0.8%) and MMP13 (10 versus 0.6%), and significantly more frequent deletions in CDKN2A/B (27 versus 13%). We confirmed that the amplification frequencies of MYC, YAP1 and MMP13 were elevated in an independent cohort of 105 patients with BM-LUAD. Functional assessment in patient-derived xenograft mouse models validated the notion that MYC, YAP1 or MMP13 overexpression increased the incidence of brain metastasis. These results demonstrate that somatic alterations contribute to brain metastases and that genomic sequencing of a sufficient number of metastatic tumors can reveal previously unknown metastatic drivers. Whole-exome sequencing of human brain metastases from lung adenocarcinoma uncovers new drivers by comparison of somatic alteration frequencies in brain metastasis cases to those in primary lung adenocarcinomas.},
  archive      = {J_NG},
  author       = {Shih, David J. H. and Nayyar, Naema and Bihun, Ivanna and Dagogo-Jack, Ibiayi and Gill, Corey M. and Aquilanti, Elisa and Bertalan, Mia and Kaplan, Alexander and D’Andrea, Megan R. and Chukwueke, Ugonma and Ippen, Franziska Maria and Alvarez-Breckenridge, Christopher and Camarda, Nicholas D. and Lastrapes, Matthew and McCabe, Devin and Kuter, Ben and Kaufman, Benjamin and Strickland, Matthew R. and Martinez-Gutierrez, Juan Carlos and Nagabhushan, Deepika and De Sauvage, Magali and White, Michael D. and Castro, Brandyn A. and Hoang, Kaitlin and Kaneb, Andrew and Batchelor, Emily D. and Paek, Sun Ha and Park, Sun Hye and Martinez-Lage, Maria and Berghoff, Anna S. and Merrill, Parker and Gerstner, Elizabeth R. and Batchelor, Tracy T. and Frosch, Matthew P. and Frazier, Ryan P. and Borger, Darrell R. and Iafrate, A. John and Johnson, Bruce E. and Santagata, Sandro and Preusser, Matthias and Cahill, Daniel P. and Carter, Scott L. and Brastianos, Priscilla K.},
  doi          = {10.1038/s41588-020-0592-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {371-377},
  shortjournal = {Nat. Genet.},
  title        = {Genomic characterization of human brain metastases identifies drivers of metastatic lung adenocarcinoma},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Modeling the complex genetic architectures of brain disease.
<em>NG</em>, <em>52</em>(4), 363–369. (<a
href="https://doi.org/10.1038/s41588-020-0596-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic architecture of each individual comprises common and rare variants that, acting alone and in combination, confer risk of disease. The cell-type-specific and/or context-dependent functional consequences of the risk variants linked to brain disease must be resolved. Coupling human induced pluripotent stem cell (hiPSC)-based technology with CRISPR-based genome engineering facilitates precise isogenic comparisons of variants across genetic backgrounds. Although functional-validation studies are typically performed on one variant in isolation and in one cell type at a time, complex genetic diseases require multiplexed gene perturbations to interrogate combinations of genes and resolve physiologically relevant disease biology. Our aim is to discuss advances at the intersection of genomics, hiPSCs and CRISPR. A better understanding of the molecular mechanisms underlying disease risk will improve genetic diagnosis, drive phenotypic drug discovery and pave the way toward precision medicine. Combining genomic data with CRISPR, hiPSC and organoid technologies provides platforms to study the complex genetic architectures of brain disease. These studies could improve genetic diagnosis, drive drug discovery and move the field toward precision medicine.},
  archive      = {J_NG},
  author       = {Fernando, Michael B. and Ahfeldt, Tim and Brennand, Kristen J.},
  doi          = {10.1038/s41588-020-0596-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {363-369},
  shortjournal = {Nat. Genet.},
  title        = {Modeling the complex genetic architectures of brain disease},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Springing an evolutionary trap on cancer. <em>NG</em>,
<em>52</em>(4), 361–362. (<a
href="https://doi.org/10.1038/s41588-020-0608-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer cells have the extraordinary evolutionary potential to adapt and acquire resistance to most conventional and targeted therapies. In a new study, Lin et al., develop a systematic approach to identify combination therapies that produce cancer traps, in which evading the first drug makes the cancer vulnerable to the second.},
  archive      = {J_NG},
  author       = {Lin, Charles Y.},
  doi          = {10.1038/s41588-020-0608-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {361-362},
  shortjournal = {Nat. Genet.},
  title        = {Springing an evolutionary trap on cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Enhancing african orphan crops with genomics. <em>NG</em>,
<em>52</em>(4), 356–360. (<a
href="https://doi.org/10.1038/s41588-020-0601-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The African Orphan Crops Consortium promotes the strategic, genome-enabled improvement of under-researched crops for biodiversity-based, nutritious food solutions in Africa. The African Plant Breeding Academy empowers the continent’s plant breeders to apply advanced genetic approaches and shared genetic solutions to the task of tailoring the immense diversity of underutilized crops to the needs of Africa’s producers, processors and consumers.},
  archive      = {J_NG},
  author       = {Jamnadass, Ramni and Mumm, Rita H. and Hale, Iago and Hendre, Prasad and Muchugi, Alice and Dawson, Ian K. and Powell, Wayne and Graudal, Lars and Yana-Shapiro, Howard and Simons, Anthony J. and Van Deynze, Allen},
  doi          = {10.1038/s41588-020-0601-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {356-360},
  shortjournal = {Nat. Genet.},
  title        = {Enhancing african orphan crops with genomics},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Calling it what it is. <em>NG</em>, <em>52</em>(4), 355. (<a
href="https://doi.org/10.1038/s41588-020-0617-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In 2015, the World Health Organization (WHO) issued guidelines for the naming of new human infectious diseases. The current global outbreak of the SARS-CoV-2 virus underscores the need to be accurate with our language, particularly as it relates to pandemics.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0617-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {355},
  shortjournal = {Nat. Genet.},
  title        = {Calling it what it is},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Genome-wide meta-analysis identifies new
loci and functional pathways influencing alzheimer’s disease risk.
<em>NG</em>, <em>52</em>(3), 354. (<a
href="https://doi.org/10.1038/s41588-019-0573-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Jansen, Iris E. and Savage, Jeanne E. and Watanabe, Kyoko and Bryois, Julien and Williams, Dylan M. and Steinberg, Stacy and Sealock, Julia and Karlsson, Ida K. and Hägg, Sara and Athanasiu, Lavinia and Voyle, Nicola and Proitsi, Petroula and Witoelar, Aree and Stringer, Sven and Aarsland, Dag and Almdahl, Ina S. and Andersen, Fred and Bergh, Sverre and Bettella, Francesco and Bjornsson, Sigurbjorn and Brækhus, Anne and Bråthen, Geir and de Leeuw, Christiaan and Desikan, Rahul S. and Djurovic, Srdjan and Dumitrescu, Logan and Fladby, Tormod and Hohman, Timothy J. and Jonsson, Palmi V. and Kiddle, Steven J. and Rongve, Arvid and Saltvedt, Ingvild and Sando, Sigrid B. and Selbæk, Geir and Shoai, Maryam and Skene, Nathan G. and Snaedal, Jon and Stordal, Eystein and Ulstein, Ingun D. and Wang, Yunpeng and White, Linda R. and Hardy, John and Hjerling-Leffler, Jens and Sullivan, Patrick F. and van der Flier, Wiesje M. and Dobson, Richard and Davis, Lea K. and Stefansson, Hreinn and Stefansson, Kari and Pedersen, Nancy L. and Ripke, Stephan and Andreassen, Ole A. and Posthuma, Danielle},
  doi          = {10.1038/s41588-019-0573-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {354},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Genome-wide meta-analysis identifies new loci and functional pathways influencing alzheimer’s disease risk},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: ARID1A influences HDAC1/BRD4 activity,
intrinsic proliferative capacity and breast cancer treatment response.
<em>NG</em>, <em>52</em>(3), 354. (<a
href="https://doi.org/10.1038/s41588-020-0582-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Nagarajan, Sankari and Rao, Shalini V. and Sutton, Joseph and Cheeseman, Danya and Dunn, Shanade and Papachristou, Evangelia K. and Prada, Jose-Enrique Gonzalez and Couturier, Dominique-Laurent and Kumar, Sanjeev and Kishore, Kamal and Chilamakuri, Chandra Sekhar Reddy and Glont, Silvia-Elena and Goode, Emily Archer and Brodie, Cara and Guppy, Naomi and Natrajan, Rachael and Bruna, Alejandra and Caldas, Carlos and Russell, Alasdair and Siersbæk, Rasmus and Yusa, Kosuke and Chernukhin, Igor and Carroll, Jason S.},
  doi          = {10.1038/s41588-020-0582-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {354},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: ARID1A influences HDAC1/BRD4 activity, intrinsic proliferative capacity and breast cancer treatment response},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Postzygotic inactivating mutations of
RHOA cause a mosaic neuroectodermal syndrome. <em>NG</em>,
<em>52</em>(3), 353. (<a
href="https://doi.org/10.1038/s41588-019-0565-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Vabres, Pierre and Sorlin, Arthur and Kholmanskikh, Stanislav S. and Demeer, Bénédicte and St-Onge, Judith and Duffourd, Yannis and Kuentz, Paul and Courcet, Jean-Benoît and Carmignac, Virginie and Garret, Philippine and Bessis, Didier and Boute, Odile and Bron, Alain and Captier, Guillaume and Carmi, Esther and Devauchelle, Bernard and Geneviève, David and Gondry-Jouet, Catherine and Guibaud, Laurent and Lafon, Arnaud and Mathieu-Dramard, Michèle and Thevenon, Julien and Dobyns, William B. and Bernard, Geneviève and Polubothu, Satyamaanasa and Faravelli, Francesca and Kinsler, Veronica A. and Thauvin, Christel and Faivre, Laurence and Ross, M. Elizabeth and Rivière, Jean-Baptiste},
  doi          = {10.1038/s41588-019-0565-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {353},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Postzygotic inactivating mutations of RHOA cause a mosaic neuroectodermal syndrome},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: A global overview of pleiotropy and
genetic architecture in complex traits. <em>NG</em>, <em>52</em>(3),
353. (<a href="https://doi.org/10.1038/s41588-019-0571-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Watanabe, Kyoko and Stringer, Sven and Frei, Oleksandr and Mirkov, Maša Umićević and de Leeuw, Christiaan and Polderman, Tinca J. C. and van der Sluis, Sophie and Andreassen, Ole A. and Neale, Benjamin M. and Posthuma, Danielle},
  doi          = {10.1038/s41588-019-0571-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {353},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A global overview of pleiotropy and genetic architecture in complex traits},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Comprehensive molecular characterization of mitochondrial
genomes in human cancers. <em>NG</em>, <em>52</em>(3), 342–352. (<a
href="https://doi.org/10.1038/s41588-019-0557-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mitochondria are essential cellular organelles that play critical roles in cancer. Here, as part of the International Cancer Genome Consortium/The Cancer Genome Atlas Pan-Cancer Analysis of Whole Genomes Consortium, which aggregated whole-genome sequencing data from 2,658 cancers across 38 tumor types, we performed a multidimensional, integrated characterization of mitochondrial genomes and related RNA sequencing data. Our analysis presents the most definitive mutational landscape of mitochondrial genomes and identifies several hypermutated cases. Truncating mutations are markedly enriched in kidney, colorectal and thyroid cancers, suggesting oncogenic effects with the activation of signaling pathways. We find frequent somatic nuclear transfers of mitochondrial DNA, some of which disrupt therapeutic target genes. Mitochondrial copy number varies greatly within and across cancers and correlates with clinical variables. Co-expression analysis highlights the function of mitochondrial genes in oxidative phosphorylation, DNA repair and the cell cycle, and shows their connections with clinically actionable genes. Our study lays a foundation for translating mitochondrial biology into clinical applications. Analysis of mitochondrial genomes (mtDNA) by using whole-genome sequencing data from 2,658 cancer samples across 38 cancer types identifies hypermutated mtDNA cases, frequent somatic nuclear transfer of mtDNA and high variability of mtDNA copy number in many cancers.},
  archive      = {J_NG},
  author       = {Yuan, Yuan and Ju, Young Seok and Kim, Youngwook and Li, Jun and Wang, Yumeng and Yoon, Christopher J. and Yang, Yang and Martincorena, Inigo and Creighton, Chad J. and Weinstein, John N. and Xu, Yanxun and Han, Leng and Kim, Hyung-Lae and Nakagawa, Hidewaki and Park, Keunchil and Campbell, Peter J. and Liang, Han},
  doi          = {10.1038/s41588-019-0557-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {342-352},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive molecular characterization of mitochondrial genomes in human cancers},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Comprehensive analysis of chromothripsis in 2,658 human
cancers using whole-genome sequencing. <em>NG</em>, <em>52</em>(3),
331–341. (<a href="https://doi.org/10.1038/s41588-019-0576-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromothripsis is a mutational phenomenon characterized by massive, clustered genomic rearrangements that occurs in cancer and other diseases. Recent studies in selected cancer types have suggested that chromothripsis may be more common than initially inferred from low-resolution copy-number data. Here, as part of the Pan-Cancer Analysis of Whole Genomes (PCAWG) Consortium of the International Cancer Genome Consortium (ICGC) and The Cancer Genome Atlas (TCGA), we analyze patterns of chromothripsis across 2,658 tumors from 38 cancer types using whole-genome sequencing data. We find that chromothripsis events are pervasive across cancers, with a frequency of more than 50% in several cancer types. Whereas canonical chromothripsis profiles display oscillations between two copy-number states, a considerable fraction of events involve multiple chromosomes and additional structural alterations. In addition to non-homologous end joining, we detect signatures of replication-associated processes and templated insertions. Chromothripsis contributes to oncogene amplification and to inactivation of genes such as mismatch-repair-related genes. These findings show that chromothripsis is a major process that drives genome evolution in human cancer. Analysis of whole-genome sequencing data across 2,658 tumors spanning 38 cancer types shows that chromothripsis is pervasive, with a frequency of more than 50% in several cancer types, contributing to oncogene amplification, gene inactivation and cancer genome evolution.},
  archive      = {J_NG},
  author       = {Cortés-Ciriano, Isidro and Lee, Jake June-Koo and Xi, Ruibin and Jain, Dhawal and Jung, Youngsook L. and Yang, Lixing and Gordenin, Dmitry and Klimczak, Leszek J. and Zhang, Cheng-Zhong and Pellman, David S. and Park, Peter J.},
  doi          = {10.1038/s41588-019-0576-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {331-341},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive analysis of chromothripsis in 2,658 human cancers using whole-genome sequencing},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The landscape of viral associations in human cancers.
<em>NG</em>, <em>52</em>(3), 320–330. (<a
href="https://doi.org/10.1038/s41588-019-0558-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here, as part of the Pan-Cancer Analysis of Whole Genomes (PCAWG) Consortium, for which whole-genome and—for a subset—whole-transcriptome sequencing data from 2,658 cancers across 38 tumor types was aggregated, we systematically investigated potential viral pathogens using a consensus approach that integrated three independent pipelines. Viruses were detected in 382 genome and 68 transcriptome datasets. We found a high prevalence of known tumor-associated viruses such as Epstein–Barr virus (EBV), hepatitis B virus (HBV) and human papilloma virus (HPV; for example, HPV16 or HPV18). The study revealed significant exclusivity of HPV and driver mutations in head-and-neck cancer and the association of HPV with APOBEC mutational signatures, which suggests that impaired antiviral defense is a driving force in cervical, bladder and head-and-neck carcinoma. For HBV, HPV16, HPV18 and adeno-associated virus-2 (AAV2), viral integration was associated with local variations in genomic copy numbers. Integrations at the TERT promoter were associated with high telomerase expression evidently activating this tumor-driving process. High levels of endogenous retrovirus (ERV1) expression were linked to a worse survival outcome in patients with kidney cancer. Viral pathogen load in cancer genomes is estimated through analysis of sequencing data from 2,656 tumors across 35 cancer types using multiple pathogen-detection pipelines, identifying viruses in 382 genomic and 68 transcriptome datasets.},
  archive      = {J_NG},
  author       = {Zapatka, Marc and Borozan, Ivan and Brewer, Daniel S. and Iskar, Murat and Grundhoff, Adam and Alawi, Malik and Desai, Nikita and Sültmann, Holger and Moch, Holger and Cooper, Colin S. and Eils, Roland and Ferretti, Vincent and Lichter, Peter},
  doi          = {10.1038/s41588-019-0558-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {320-330},
  shortjournal = {Nat. Genet.},
  title        = {The landscape of viral associations in human cancers},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Pan-cancer analysis of whole genomes identifies driver
rearrangements promoted by LINE-1 retrotransposition. <em>NG</em>,
<em>52</em>(3), 306–319. (<a
href="https://doi.org/10.1038/s41588-019-0562-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {About half of all cancers have somatic integrations of retrotransposons. Here, to characterize their role in oncogenesis, we analyzed the patterns and mechanisms of somatic retrotransposition in 2,954 cancer genomes from 38 histological cancer subtypes within the framework of the Pan-Cancer Analysis of Whole Genomes (PCAWG) project. We identified 19,166 somatically acquired retrotransposition events, which affected 35% of samples and spanned a range of event types. Long interspersed nuclear element (LINE-1; L1 hereafter) insertions emerged as the first most frequent type of somatic structural variation in esophageal adenocarcinoma, and the second most frequent in head-and-neck and colorectal cancers. Aberrant L1 integrations can delete megabase-scale regions of a chromosome, which sometimes leads to the removal of tumor-suppressor genes, and can induce complex translocations and large-scale duplications. Somatic retrotranspositions can also initiate breakage–fusion–bridge cycles, leading to high-level amplification of oncogenes. These observations illuminate a relevant role of 22 L1 retrotransposition in remodeling the cancer genome, with potential implications for the development of human tumors. An analysis of 2,954 genomes from 38 cancer subtypes identified 19,166 retrotransposition events in 35% of samples. Aberrant LINE-1 retrotranspositions can lead to the deletion of tumor-suppressor genes as well as the amplification of oncogenes.},
  archive      = {J_NG},
  author       = {Rodriguez-Martin, Bernardo and Alvarez, Eva G. and Baez-Ortega, Adrian and Zamora, Jorge and Supek, Fran and Demeulemeester, Jonas and Santamarina, Martin and Ju, Young Seok and Temes, Javier and Garcia-Souto, Daniel and Detering, Harald and Li, Yilong and Rodriguez-Castro, Jorge and Dueso-Barroso, Ana and Bruzos, Alicia L. and Dentro, Stefan C. and Blanco, Miguel G. and Contino, Gianmarco and Ardeljan, Daniel and Tojo, Marta and Roberts, Nicola D. and Zumalave, Sonia and Edwards, Paul A. and Weischenfeldt, Joachim and Puiggròs, Montserrat and Chong, Zechen and Chen, Ken and Lee, Eunjung Alice and Wala, Jeremiah A. and Raine, Keiran M. and Butler, Adam and Waszak, Sebastian M. and Navarro, Fabio C. P. and Schumacher, Steven E. and Monlong, Jean and Maura, Francesco and Bolli, Niccolo and Bourque, Guillaume and Gerstein, Mark and Park, Peter J. and Wedge, David C. and Beroukhim, Rameen and Torrents, David and Korbel, Jan O. and Martincorena, Iñigo and Fitzgerald, Rebecca C. and Van Loo, Peter and Kazazian, Haig H. and Burns, Kathleen H. and Campbell, Peter J. and Tubio, Jose M. C.},
  doi          = {10.1038/s41588-019-0562-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {306-319},
  shortjournal = {Nat. Genet.},
  title        = {Pan-cancer analysis of whole genomes identifies driver rearrangements promoted by LINE-1 retrotransposition},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Disruption of chromatin folding domains by somatic genomic
rearrangements in human cancer. <em>NG</em>, <em>52</em>(3), 294–305.
(<a href="https://doi.org/10.1038/s41588-019-0564-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin is folded into successive layers to organize linear DNA. Genes within the same topologically associating domains (TADs) demonstrate similar expression and histone-modification profiles, and boundaries separating different domains have important roles in reinforcing the stability of these features. Indeed, domain disruptions in human cancers can lead to misregulation of gene expression. However, the frequency of domain disruptions in human cancers remains unclear. Here, as part of the Pan-Cancer Analysis of Whole Genomes (PCAWG) Consortium of the International Cancer Genome Consortium (ICGC) and The Cancer Genome Atlas (TCGA), which aggregated whole-genome sequencing data from 2,658 cancers across 38 tumor types, we analyzed 288,457 somatic structural variations (SVs) to understand the distributions and effects of SVs across TADs. Notably, SVs can lead to the fusion of discrete TADs, and complex rearrangements markedly change chromatin folding maps in the cancer genomes. Notably, only 14% of the boundary deletions resulted in a change in expression in nearby genes of more than twofold. A pan-cancer genomic analysis reports the effects of structural variations on chromatin domains (TADs). Most TAD disruptions do not result in appreciable changes in expression of nearby genes.},
  archive      = {J_NG},
  author       = {Akdemir, Kadir C. and Le, Victoria T. and Chandran, Sahaana and Li, Yilong and Verhaak, Roel G. and Beroukhim, Rameen and Campbell, Peter J. and Chin, Lynda and Dixon, Jesse R. and Futreal, P. Andrew},
  doi          = {10.1038/s41588-019-0564-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {294-305},
  shortjournal = {Nat. Genet.},
  title        = {Disruption of chromatin folding domains by somatic genomic rearrangements in human cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Interplay between whole-genome doubling and the accumulation
of deleterious alterations in cancer evolution. <em>NG</em>,
<em>52</em>(3), 283–293. (<a
href="https://doi.org/10.1038/s41588-020-0584-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome doubling (WGD) is a prevalent event in cancer, involving a doubling of the entire chromosome complement. However, despite its prevalence and prognostic relevance, the evolutionary selection pressures for WGD in cancer have not been investigated. Here, we combine evolutionary simulations with an analysis of cancer sequencing data to explore WGD during cancer evolution. Simulations suggest that WGD can be selected to mitigate the irreversible, ratchet-like, accumulation of deleterious somatic alterations, provided that they occur at a sufficiently high rate. Consistent with this, we observe an enrichment for WGD in tumor types with extensive loss of heterozygosity, including lung squamous cell carcinoma and triple-negative breast cancers, and we find evidence for negative selection against homozygous loss of essential genes before, but not after, WGD. Finally, we demonstrate that loss of heterozygosity and temporal dissection of mutations can be exploited to identify novel tumor suppressor genes and to obtain a deeper characterization of known cancer genes. Analysis of whole-genome doubling (WGD) by using cancer sequencing data combined with simulations of tumor evolution suggests that there is negative selection against homozygous loss of essential genes before WGD but not after.},
  archive      = {J_NG},
  author       = {López, Saioa and Lim, Emilia L. and Horswell, Stuart and Haase, Kerstin and Huebner, Ariana and Dietzen, Michelle and Mourikis, Thanos P. and Watkins, Thomas B. K. and Rowan, Andrew and Dewhurst, Sally M. and Birkbak, Nicolai J. and Wilson, Gareth A. and Van Loo, Peter and Jamal-Hanjani, Mariam and Swanton, Charles and McGranahan, Nicholas},
  doi          = {10.1038/s41588-020-0584-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {283-293},
  shortjournal = {Nat. Genet.},
  title        = {Interplay between whole-genome doubling and the accumulation of deleterious alterations in cancer evolution},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Lysine 4 of histone h3.3 is required for embryonic stem cell
differentiation, histone enrichment at regulatory regions and
transcription accuracy. <em>NG</em>, <em>52</em>(3), 273–282. (<a
href="https://doi.org/10.1038/s41588-020-0586-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations in enzymes that modify histone H3 at lysine 4 (H3K4) or lysine 36 (H3K36) have been linked to human disease, yet the role of these residues in mammals is unclear. We mutated K4 or K36 to alanine in the histone variant H3.3 and showed that the K4A mutation in mouse embryonic stem cells (ESCs) impaired differentiation and induced widespread gene expression changes. K4A resulted in substantial H3.3 depletion, especially at ESC promoters; it was accompanied by reduced remodeler binding and increased RNA polymerase II (Pol II) activity. Regulatory regions depleted of H3.3K4A showed histone modification alterations and changes in enhancer activity that correlated with gene expression. In contrast, the K36A mutation did not alter H3.3 deposition and affected gene expression at the later stages of differentiation. Thus, H3K4 is required for nucleosome deposition, histone turnover and chromatin remodeler binding at regulatory regions, where tight regulation of Pol II activity is necessary for proper ESC differentiation. Replacement of lysine 4 or 36 with alanine in histone H3.3 impairs ESC differentiation and induces widespread gene expression changes. Expression of H3.3K4A, but not H3.3K36A, leads to H3.3 depletion, reduces remodeler binding and increases RNA polymerase II activity.},
  archive      = {J_NG},
  author       = {Gehre, Maja and Bunina, Daria and Sidoli, Simone and Lübke, Marlena J. and Diaz, Nichole and Trovato, Matteo and Garcia, Benjamin A. and Zaugg, Judith B. and Noh, Kyung-Min},
  doi          = {10.1038/s41588-020-0586-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {273-282},
  shortjournal = {Nat. Genet.},
  title        = {Lysine 4 of histone h3.3 is required for embryonic stem cell differentiation, histone enrichment at regulatory regions and transcription accuracy},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Chromatin interaction analyses elucidate the roles of
PRC2-bound silencers in mouse development. <em>NG</em>, <em>52</em>(3),
264–272. (<a href="https://doi.org/10.1038/s41588-020-0581-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lineage-specific gene expression is modulated by a balance between transcriptional activation and repression during animal development. Knowledge about enhancer-centered transcriptional activation has advanced considerably, but silencers and their roles in normal development remain poorly understood. Here, we performed chromatin interaction analyses of Polycomb repressive complex 2 (PRC2), a key inducer of transcriptional gene silencing, to uncover silencers, their molecular identity and associated chromatin connectivity. Systematic analysis of cis-regulatory silencer elements reveals their chromatin features and gene-targeting specificity. Deletion of certain PRC2-bound silencers in mice results in transcriptional derepression of their interacting genes and pleiotropic developmental phenotypes, including embryonic lethality. While some PRC2-bound elements function as silencers in pluripotent cells, they can transition into active tissue-specific enhancers during development, highlighting their regulatory versatility. Our study characterizes the molecular profile of silencers and their associated chromatin architectures, and suggests the possibility of targeted reactivation of epigenetically silenced genes. Chromatin interaction analysis identifies PRC2-bound silencers in mESCs, which, when deleted in mice, can lead to developmental phenotypes. Silencers in pluripotent cells can transition into active tissue-specific enhancers during development.},
  archive      = {J_NG},
  author       = {Ngan, Chew Yee and Wong, Chee Hong and Tjong, Harianto and Wang, Wenbo and Goldfeder, Rachel L. and Choi, Cindy and He, Hao and Gong, Liang and Lin, Junyan and Urban, Barbara and Chow, Julianna and Li, Meihong and Lim, Joanne and Philip, Vivek and Murray, Stephen A. and Wang, Haoyi and Wei, Chia-Lin},
  doi          = {10.1038/s41588-020-0581-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {264-272},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin interaction analyses elucidate the roles of PRC2-bound silencers in mouse development},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Systematic identification of silencers in human cells.
<em>NG</em>, <em>52</em>(3), 254–263. (<a
href="https://doi.org/10.1038/s41588-020-0578-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The majority of the human genome does not encode proteins. Many of these noncoding regions contain important regulatory sequences that control gene expression. To date, most studies have focused on activators such as enhancers, but regions that repress gene expression—silencers—have not been systematically studied. We have developed a system that identifies silencer regions in a genome-wide fashion on the basis of silencer-mediated transcriptional repression of caspase 9. We found that silencers are widely distributed and may function in a tissue-specific fashion. These silencers harbor unique epigenetic signatures and are associated with specific transcription factors. Silencers also act at multiple genes, and at the level of chromosomal domains and long-range interactions. Deletion of silencer regions linked to the drug transporter genes ABCC2 and ABCG2 caused chemo-resistance. Overall, our study demonstrates that tissue-specific silencing is widespread throughout the human genome and probably contributes substantially to the regulation of gene expression and human biology. A genome-wide screen identifies silencer regions in human cells. Deletion of silencers linked to the transporter genes ABCC2 and ABCG2 causes their up-regulation and chemo-resistance.},
  archive      = {J_NG},
  author       = {Pang, Baoxu and Snyder, Michael P.},
  doi          = {10.1038/s41588-020-0578-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {254-263},
  shortjournal = {Nat. Genet.},
  title        = {Systematic identification of silencers in human cells},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Allele-specific expression changes dynamically during t cell
activation in HLA and other autoimmune loci. <em>NG</em>,
<em>52</em>(3), 247–253. (<a
href="https://doi.org/10.1038/s41588-020-0579-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies have revealed that autoimmune susceptibility variants are over-represented in memory CD4+ T cell regulatory elements1–3. Understanding how genetic variation affects gene expression in different T cell physiological states is essential for deciphering genetic mechanisms of autoimmunity4,5. Here, we characterized the dynamics of genetic regulatory effects at eight time points during memory CD4+ T cell activation with high-depth RNA-seq in healthy individuals. We discovered widespread, dynamic allele-specific expression across the genome, where the balance of alleles changes over time. These genes were enriched fourfold within autoimmune loci. We found pervasive dynamic regulatory effects within six HLA genes. HLA-DQB1 alleles had one of three distinct transcriptional regulatory programs. Using CRISPR–Cas9 genomic editing we demonstrated that a promoter variant is causal for T cell–specific control of HLA-DQB1 expression. Our study shows that genetic variation in cis-regulatory elements affects gene expression in a manner dependent on lymphocyte activation status, contributing to the interindividual complexity of immune responses. Deep mRNA sequencing at eight time points during memory CD4+ T cell activation identifies widespread dynamic allele-specific expression events that are enriched in HLA and other autoimmune disease loci.},
  archive      = {J_NG},
  author       = {Gutierrez-Arcelus, Maria and Baglaenko, Yuriy and Arora, Jatin and Hannes, Susan and Luo, Yang and Amariuta, Tiffany and Teslovich, Nikola and Rao, Deepak A. and Ermann, Joerg and Jonsson, A. Helena and Navarrete, Cristina and Rich, Stephen S. and Taylor, Kent D. and Rotter, Jerome I. and Gregersen, Peter K. and Esko, Tonu and Brenner, Michael B. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-020-0579-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {247-253},
  shortjournal = {Nat. Genet.},
  title        = {Allele-specific expression changes dynamically during t cell activation in HLA and other autoimmune loci},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Silencers in the spotlight. <em>NG</em>, <em>52</em>(3),
244–245. (<a href="https://doi.org/10.1038/s41588-020-0583-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efforts to systematically characterize silencers in animal genomes have been limited. Two studies in this issue of Nature Genetics bring silencers into the spotlight, identifying these elements genome wide in human and mouse cells, and providing insights into their function.},
  archive      = {J_NG},
  author       = {Della Rosa, Monica and Spivakov, Mikhail},
  doi          = {10.1038/s41588-020-0583-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {244-245},
  shortjournal = {Nat. Genet.},
  title        = {Silencers in the spotlight},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The GWAS diversity monitor tracks diversity by disease in
real time. <em>NG</em>, <em>52</em>(3), 242–243. (<a
href="https://doi.org/10.1038/s41588-020-0580-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Mills, Melinda C. and Rahal, Charles},
  doi          = {10.1038/s41588-020-0580-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {242-243},
  shortjournal = {Nat. Genet.},
  title        = {The GWAS diversity monitor tracks diversity by disease in real time},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Whole tumor genomes across cancers. <em>NG</em>,
<em>52</em>(3), 241. (<a
href="https://doi.org/10.1038/s41588-020-0597-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Pan-Cancer Analysis of Whole Genomes (PCAWG) Consortium project, led by the International Cancer Genome Consortium (ICGC) and The Cancer Genome Atlas (TCGA), coordinated the sequencing and analysis of 2,583 tumor whole genomes across 38 cancer types. This impressively large project, comprising many working groups focusing on various molecular or genetic features of cancer, has generated valuable data for the cancer research community that will continue to be mined for many years to come.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0597-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {241},
  shortjournal = {Nat. Genet.},
  title        = {Whole tumor genomes across cancers},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Transcription phenotypes of pancreatic cancer are driven by
genomic events during tumor evolution. <em>NG</em>, <em>52</em>(2),
231–240. (<a href="https://doi.org/10.1038/s41588-019-0566-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pancreatic adenocarcinoma presents as a spectrum of a highly aggressive disease in patients. The basis of this disease heterogeneity has proved difficult to resolve due to poor tumor cellularity and extensive genomic instability. To address this, a dataset of whole genomes and transcriptomes was generated from purified epithelium of primary and metastatic tumors. Transcriptome analysis demonstrated that molecular subtypes are a product of a gene expression continuum driven by a mixture of intratumoral subpopulations, which was confirmed by single-cell analysis. Integrated whole-genome analysis uncovered that molecular subtypes are linked to specific copy number aberrations in genes such as mutant KRAS and GATA6. By mapping tumor genetic histories, tetraploidization emerged as a key mutational process behind these events. Taken together, these data support the premise that the constellation of genomic aberrations in the tumor gives rise to the molecular subtype, and that disease heterogeneity is due to ongoing genomic instability during progression. Whole-genome sequencing, transcriptome sequencing and single-cell analysis of primary and metastatic pancreatic adenocarcinoma identify molecular subtypes and intratumor heterogeneity.},
  archive      = {J_NG},
  author       = {Chan-Seng-Yue, Michelle and Kim, Jaeseung C. and Wilson, Gavin W. and Ng, Karen and Figueroa, Eugenia Flores and O’Kane, Grainne M. and Connor, Ashton A. and Denroche, Robert E. and Grant, Robert C. and McLeod, Jessica and Wilson, Julie M. and Jang, Gun Ho and Zhang, Amy and Dodd, Anna and Liang, Sheng-Ben and Borgida, Ayelet and Chadwick, Dianne and Kalimuthu, Sangeetha and Lungu, Ilinca and Bartlett, John M. S. and Krzyzanowski, Paul M. and Sandhu, Vandana and Tiriac, Hervé and Froeling, Fieke E. M. and Karasinska, Joanna M. and Topham, James T. and Renouf, Daniel J. and Schaeffer, David F. and Jones, Steven J. M. and Marra, Marco A. and Laskin, Janessa and Chetty, Runjan and Stein, Lincoln D. and Zogopoulos, George and Haibe-Kains, Benjamin and Campbell, Peter J. and Tuveson, David A. and Knox, Jennifer J. and Fischer, Sandra E. and Gallinger, Steven and Notta, Faiyaz},
  doi          = {10.1038/s41588-019-0566-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {231-240},
  shortjournal = {Nat. Genet.},
  title        = {Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Early TP53 alterations engage environmental exposures to
promote gastric premalignancy in an integrative mouse model.
<em>NG</em>, <em>52</em>(2), 219–230. (<a
href="https://doi.org/10.1038/s41588-019-0574-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic alterations in cancer genes are being detected in normal and premalignant tissue, thus placing greater emphasis on gene–environment interactions that enable disease phenotypes. By combining early genetic alterations with disease-relevant exposures, we developed an integrative mouse model to study gastric premalignancy. Deletion of Trp53 in gastric cells confers a selective advantage and promotes the development of dysplasia in the setting of dietary carcinogens. Organoid derivation from dysplastic lesions facilitated genomic, transcriptional and functional evaluation of gastric premalignancy. Cell cycle regulators, most notably Cdkn2a, were upregulated by p53 inactivation in gastric premalignancy, serving as a barrier to disease progression. Co-deletion of Cdkn2a and Trp53 in dysplastic gastric organoids promoted cancer phenotypes but also induced replication stress, exposing a susceptibility to DNA damage response inhibitors. These findings demonstrate the utility of mouse models that integrate genomic alterations with relevant exposures and highlight the importance of gene–environment interactions in shaping the premalignant state. A mouse model that combines nitrosamide exposure with inducible, tissue-specific TP53 alterations is used to generate premalignant gastric organoids and provides insights into the development of gastric premalignancy.},
  archive      = {J_NG},
  author       = {Sethi, Nilay S. and Kikuchi, Osamu and Duronio, Gina N. and Stachler, Matthew D. and McFarland, James M. and Ferrer-Luna, Ruben and Zhang, Yanxi and Bao, Chunyang and Bronson, Roderick and Patil, Deepa and Sanchez-Vega, Francisco and Liu, Jie-Bin and Sicinska, Ewa and Lazaro, Jean-Bernard and Ligon, Keith L. and Beroukhim, Rameen and Bass, Adam J.},
  doi          = {10.1038/s41588-019-0574-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {219-230},
  shortjournal = {Nat. Genet.},
  title        = {Early TP53 alterations engage environmental exposures to promote gastric premalignancy in an integrative mouse model},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Identification of cancer driver genes based on nucleotide
context. <em>NG</em>, <em>52</em>(2), 208–218. (<a
href="https://doi.org/10.1038/s41588-019-0572-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer genomes contain large numbers of somatic mutations but few of these mutations drive tumor development. Current approaches either identify driver genes on the basis of mutational recurrence or approximate the functional consequences of nonsynonymous mutations by using bioinformatic scores. Passenger mutations are enriched in characteristic nucleotide contexts, whereas driver mutations occur in functional positions, which are not necessarily surrounded by a particular nucleotide context. We observed that mutations in contexts that deviate from the characteristic contexts around passenger mutations provide a signal in favor of driver genes. We therefore developed a method that combines this feature with the signals traditionally used for driver-gene identification. We applied our method to whole-exome sequencing data from 11,873 tumor–normal pairs and identified 460 driver genes that clustered into 21 cancer-related pathways. Our study provides a resource of driver genes across 28 tumor types with additional driver genes identified according to mutations in unusual nucleotide contexts. MutPanning is a new method to detect cancer driver genes that identifies genes with an excess of mutations in unusual nucleotide contexts. Applying this to whole-exome sequencing data from 11,873 tumor–normal pairs identifies 460 driver genes.},
  archive      = {J_NG},
  author       = {Dietlein, Felix and Weghorn, Donate and Taylor-Weiner, Amaro and Richters, André and Reardon, Brendan and Liu, David and Lander, Eric S. and Van Allen, Eliezer M. and Sunyaev, Shamil R.},
  doi          = {10.1038/s41588-019-0572-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {208-218},
  shortjournal = {Nat. Genet.},
  title        = {Identification of cancer driver genes based on nucleotide context},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). ARID1A determines luminal identity and therapeutic response
in estrogen-receptor-positive breast cancer. <em>NG</em>,
<em>52</em>(2), 198–207. (<a
href="https://doi.org/10.1038/s41588-019-0554-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations in ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, are the most common alterations of the SWI/SNF complex in estrogen-receptor-positive (ER+) breast cancer. We identify that ARID1A inactivating mutations are present at a high frequency in advanced endocrine-resistant ER+ breast cancer. An epigenome CRISPR–CAS9 knockout (KO) screen identifies ARID1A as the top candidate whose loss determines resistance to the ER degrader fulvestrant. ARID1A inactivation in cells and in patients leads to resistance to ER degraders by facilitating a switch from ER-dependent luminal cells to ER-independent basal-like cells. Cellular plasticity is mediated by loss of ARID1A-dependent SWI/SNF complex targeting to genomic sites of the luminal lineage-determining transcription factors including ER, forkhead box protein A1 (FOXA1) and GATA-binding factor 3 (GATA3). ARID1A also regulates genome-wide ER–FOXA1 chromatin interactions and ER-dependent transcription. Altogether, we uncover a critical role for ARID1A in maintaining luminal cell identity and endocrine therapeutic response in ER+ breast cancer. A CRISPR–CAS9 screen, analysis of patient data, and functional in vivo and in vitro experiments identify a critical role for ARID1A in determining breast luminal cell identity and endocrine therapeutic response in estrogen-receptor-positive breast cancer.},
  archive      = {J_NG},
  author       = {Xu, Guotai and Chhangawala, Sagar and Cocco, Emiliano and Razavi, Pedram and Cai, Yanyan and Otto, Jordan E and Ferrando, Lorenzo and Selenica, Pier and Ladewig, Erik and Chan, Carmen and Da Cruz Paula, Arnaud and Witkin, Matthew and Cheng, Yuanming and Park, Jane and Serna-Tamayo, Cristian and Zhao, HuiYong and Wu, Fan and Sallaku, Mirna and Qu, Xuan and Zhao, Alison and Collings, Clayton K and D’Avino, Andrew R. and Jhaveri, Komal and Koche, Richard and Levine, Ross L. and Reis-Filho, Jorge S. and Kadoch, Cigall and Scaltriti, Maurizio and Leslie, Christina S. and Baselga, José and Toska, Eneda},
  doi          = {10.1038/s41588-019-0554-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {198-207},
  shortjournal = {Nat. Genet.},
  title        = {ARID1A determines luminal identity and therapeutic response in estrogen-receptor-positive breast cancer},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). ARID1A influences HDAC1/BRD4 activity, intrinsic
proliferative capacity and breast cancer treatment response.
<em>NG</em>, <em>52</em>(2), 187–197. (<a
href="https://doi.org/10.1038/s41588-019-0541-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) screens to understand endocrine drug resistance, we discovered ARID1A and other SWI/SNF complex components as the factors most critically required for response to two classes of estrogen receptor-alpha (ER) antagonists. In this context, SWI/SNF-specific gene deletion resulted in drug resistance. Unexpectedly, ARID1A was also the top candidate in regard to response to the bromodomain and extraterminal domain inhibitor JQ1, but in the opposite direction, with loss of ARID1A sensitizing breast cancer cells to bromodomain and extraterminal domain inhibition. We show that ARID1A is a repressor that binds chromatin at ER cis-regulatory elements. However, ARID1A elicits repressive activity in an enhancer-specific, but forkhead box A1-dependent and active, ER-independent manner. Deletion of ARID1A resulted in loss of histone deacetylase 1 binding, increased histone 4 lysine acetylation and subsequent BRD4-driven transcription and growth. ARID1A mutations are more frequent in treatment-resistant disease, and our findings provide mechanistic insight into this process while revealing rational treatment strategies for these patients. ARID1A and other SWI/SNF components are critical for response to estrogen-receptor antagonists in breast cancer. SWI/SNF-specific gene knockouts lead to drug resistance and ARID1A mutations are more frequent in resistant tumors from patients.},
  archive      = {J_NG},
  author       = {Nagarajan, Sankari and Rao, Shalini V. and Sutton, Joseph and Cheeseman, Danya and Dunn, Shanade and Papachristou, Evangelia K. and Prada, Jose-Enrique Gonzalez and Couturier, Dominique-Laurent and Kumar, Sanjeev and Kishore, Kamal and Chilamakuri, Chandra Sekhar Reddy and Glont, Silvia-Elena and Archer Goode, Emily and Brodie, Cara and Guppy, Naomi and Natrajan, Rachael and Bruna, Alejandra and Caldas, Carlos and Russell, Alasdair and Siersbæk, Rasmus and Yusa, Kosuke and Chernukhin, Igor and Carroll, Jason S.},
  doi          = {10.1038/s41588-019-0541-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {187-197},
  shortjournal = {Nat. Genet.},
  title        = {ARID1A influences HDAC1/BRD4 activity, intrinsic proliferative capacity and breast cancer treatment response},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genomic landscape of lung adenocarcinoma in east asians.
<em>NG</em>, <em>52</em>(2), 177–186. (<a
href="https://doi.org/10.1038/s41588-019-0569-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lung cancer is the world’s leading cause of cancer death and shows strong ancestry disparities. By sequencing and assembling a large genomic and transcriptomic dataset of lung adenocarcinoma (LUAD) in individuals of East Asian ancestry (EAS; n = 305), we found that East Asian LUADs had more stable genomes characterized by fewer mutations and fewer copy number alterations than LUADs from individuals of European ancestry. This difference is much stronger in smokers as compared to nonsmokers. Transcriptomic clustering identified a new EAS-specific LUAD subgroup with a less complex genomic profile and upregulated immune-related genes, allowing the possibility of immunotherapy-based approaches. Integrative analysis across clinical and molecular features showed the importance of molecular phenotypes in patient prognostic stratification. EAS LUADs had better prediction accuracy than those of European ancestry, potentially due to their less complex genomic architecture. This study elucidated a comprehensive genomic landscape of EAS LUADs and highlighted important ancestry differences between the two cohorts. Genomic and transcriptomic analysis of lung adenocarcinoma (LUAD) in Asia indicates that Asian LUADs have fewer mutations, lower driver prevalence and fewer copy number alterations than European LUADs.},
  archive      = {J_NG},
  author       = {Chen, Jianbin and Yang, Hechuan and Teo, Audrey Su Min and Amer, Lidyana Bte and Sherbaf, Faranak Ghazi and Tan, Chu Quan and Alvarez, Jacob Josiah Santiago and Lu, Bingxin and Lim, Jia Qi and Takano, Angela and Nahar, Rahul and Lee, Yin Yeng and Phua, Cheryl Zi Jin and Chua, Khi Pin and Suteja, Lisda and Chen, Pauline Jieqi and Chang, Mei Mei and Koh, Tina Puay Theng and Ong, Boon-Hean and Anantham, Devanand and Hsu, Anne Ann Ling and Gogna, Apoorva and Too, Chow Wei and Aung, Zaw Win and Lee, Yi Fei and Wang, Lanying and Lim, Tony Kiat Hon and Wilm, Andreas and Choi, Poh Sum and Ng, Poh Yong and Toh, Chee Keong and Lim, Wan-Teck and Ma, Siming and Lim, Bing and Liu, Jin and Tam, Wai Leong and Skanderup, Anders Jacobsen and Yeong, Joe Poh Sheng and Tan, Eng-Huat and Creasy, Caretha L. and Tan, Daniel Shao Weng and Hillmer, Axel M. and Zhai, Weiwei},
  doi          = {10.1038/s41588-019-0569-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {177-186},
  shortjournal = {Nat. Genet.},
  title        = {Genomic landscape of lung adenocarcinoma in east asians},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genetic studies of urinary metabolites illuminate mechanisms
of detoxification and excretion in humans. <em>NG</em>, <em>52</em>(2),
167–176. (<a href="https://doi.org/10.1038/s41588-019-0567-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The kidneys integrate information from continuous systemic processes related to the absorption, distribution, metabolism and excretion (ADME) of metabolites. To identify underlying molecular mechanisms, we performed genome-wide association studies of the urinary concentrations of 1,172 metabolites among 1,627 patients with reduced kidney function. The 240 unique metabolite–locus associations (metabolite quantitative trait loci, mQTLs) that were identified and replicated highlight novel candidate substrates for transport proteins. The identified genes are enriched in ADME-relevant tissues and cell types, and they reveal novel candidates for biotransformation and detoxification reactions. Fine mapping of mQTLs and integration with single-cell gene expression permitted the prioritization of causal genes, functional variants and target cell types. The combination of mQTLs with genetic and health information from 450,000 UK Biobank participants illuminated metabolic mediators, and hence, novel urinary biomarkers of disease risk. This comprehensive resource of genetic targets and their substrates is informative for ADME processes in humans and is relevant to basic science, clinical medicine and pharmaceutical research. Genome-wide association analysis of 1,172 urinary metabolites identifies 240 metabolite–locus associations that when combined with UK Biobank data suggest novel metabolic mediators of disease and markers of disease risk.},
  archive      = {J_NG},
  author       = {Schlosser, Pascal and Li, Yong and Sekula, Peggy and Raffler, Johannes and Grundner-Culemann, Franziska and Pietzner, Maik and Cheng, Yurong and Wuttke, Matthias and Steinbrenner, Inga and Schultheiss, Ulla T. and Kotsis, Fruzsina and Kacprowski, Tim and Forer, Lukas and Hausknecht, Birgit and Ekici, Arif B. and Nauck, Matthias and Völker, Uwe and Walz, Gerd and Oefner, Peter J. and Kronenberg, Florian and Mohney, Robert P. and Köttgen, Michael and Suhre, Karsten and Eckardt, Kai-Uwe and Kastenmüller, Gabi and Köttgen, Anna},
  doi          = {10.1038/s41588-019-0567-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {167-176},
  shortjournal = {Nat. Genet.},
  title        = {Genetic studies of urinary metabolites illuminate mechanisms of detoxification and excretion in humans},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Multitrait analysis of glaucoma identifies new risk loci and
enables polygenic prediction of disease susceptibility and progression.
<em>NG</em>, <em>52</em>(2), 160–166. (<a
href="https://doi.org/10.1038/s41588-019-0556-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glaucoma, a disease characterized by progressive optic nerve degeneration, can be prevented through timely diagnosis and treatment. We characterize optic nerve photographs of 67,040 UK Biobank participants and use a multitrait genetic model to identify risk loci for glaucoma. A glaucoma polygenic risk score (PRS) enables effective risk stratification in unselected glaucoma cases and modifies penetrance of the MYOC variant encoding p.Gln368Ter, the most common glaucoma-associated myocilin variant. In the unselected glaucoma population, individuals in the top PRS decile reach an absolute risk for glaucoma 10 years earlier than the bottom decile and are at 15-fold increased risk of developing advanced glaucoma (top 10% versus remaining 90%, odds ratio = 4.20). The PRS predicts glaucoma progression in prospectively monitored, early manifest glaucoma cases (P = 0.004) and surgical intervention in advanced disease (P = 3.6 × 10−6). This glaucoma PRS will facilitate the development of a personalized approach for earlier treatment of high-risk individuals, with less intensive monitoring and treatment being possible for lower-risk groups. Multitrait genome-wide analysis of glaucoma and related phenotypes identifies new risk loci and enables development of a polygenic risk score to predict disease susceptibility and key clinical outcomes.},
  archive      = {J_NG},
  author       = {Craig, Jamie E. and Han, Xikun and Qassim, Ayub and Hassall, Mark and Cooke Bailey, Jessica N. and Kinzy, Tyler G. and Khawaja, Anthony P. and An, Jiyuan and Marshall, Henry and Gharahkhani, Puya and Igo, Robert P. and Graham, Stuart L. and Healey, Paul R. and Ong, Jue-Sheng and Zhou, Tiger and Siggs, Owen and Law, Matthew H. and Souzeau, Emmanuelle and Ridge, Bronwyn and Hysi, Pirro G. and Burdon, Kathryn P. and Mills, Richard A. and Landers, John and Ruddle, Jonathan B. and Agar, Ashish and Galanopoulos, Anna and White, Andrew J. R. and Willoughby, Colin E. and Andrew, Nicholas H. and Best, Stephen and Vincent, Andrea L. and Goldberg, Ivan and Radford-Smith, Graham and Martin, Nicholas G. and Montgomery, Grant W. and Vitart, Veronique and Hoehn, Rene and Wojciechowski, Robert and Jonas, Jost B. and Aung, Tin and Pasquale, Louis R. and Cree, Angela Jane and Sivaprasad, Sobha and Vallabh, Neeru A. and Viswanathan, Ananth C. and Pasutto, Francesca and Haines, Jonathan L. and Klaver, Caroline C. W. and van Duijn, Cornelia M. and Casson, Robert J. and Foster, Paul J. and Khaw, Peng Tee and Hammond, Christopher J. and Mackey, David A. and Mitchell, Paul and Lotery, Andrew J. and Wiggs, Janey L. and Hewitt, Alex W. and MacGregor, Stuart},
  doi          = {10.1038/s41588-019-0556-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {160-166},
  shortjournal = {Nat. Genet.},
  title        = {Multitrait analysis of glaucoma identifies new risk loci and enables polygenic prediction of disease susceptibility and progression},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A slipped-CAG DNA-binding small molecule induces
trinucleotide-repeat contractions in vivo. <em>NG</em>, <em>52</em>(2),
146–159. (<a href="https://doi.org/10.1038/s41588-019-0575-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In many repeat diseases, such as Huntington’s disease (HD), ongoing repeat expansions in affected tissues contribute to disease onset, progression and severity. Inducing contractions of expanded repeats by exogenous agents is not yet possible. Traditional approaches would target proteins driving repeat mutations. Here we report a compound, naphthyridine-azaquinolone (NA), that specifically binds slipped-CAG DNA intermediates of expansion mutations, a previously unsuspected target. NA efficiently induces repeat contractions in HD patient cells as well as en masse contractions in medium spiny neurons of HD mouse striatum. Contractions are specific for the expanded allele, independently of DNA replication, require transcription across the coding CTG strand and arise by blocking repair of CAG slip-outs. NA-induced contractions depend on active expansions driven by MutSβ. NA injections in HD mouse striatum reduce mutant HTT protein aggregates, a biomarker of HD pathogenesis and severity. Repeat-structure-specific DNA ligands are a novel avenue to contract expanded repeats. Naphthyridine-azaquinolone specifically binds slipped-CAG DNA intermediates, induces contractions of expanded repeats and reduces mutant HTT protein aggregates in cell and animal models of Huntington’s disease.},
  archive      = {J_NG},
  author       = {Nakamori, Masayuki and Panigrahi, Gagan B. and Lanni, Stella and Gall-Duncan, Terence and Hayakawa, Hideki and Tanaka, Hana and Luo, Jennifer and Otabe, Takahiro and Li, Jinxing and Sakata, Akihiro and Caron, Marie-Christine and Joshi, Niraj and Prasolava, Tanya and Chiang, Karen and Masson, Jean-Yves and Wold, Marc S. and Wang, Xiaoxiao and Lee, Marietta Y. W. T. and Huddleston, John and Munson, Katherine M. and Davidson, Scott and Layeghifard, Mehdi and Edward, Lisa-Monique and Gallon, Richard and Santibanez-Koref, Mauro and Murata, Asako and Takahashi, Masanori P. and Eichler, Evan E. and Shlien, Adam and Nakatani, Kazuhiko and Mochizuki, Hideki and Pearson, Christopher E.},
  doi          = {10.1038/s41588-019-0575-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {146-159},
  shortjournal = {Nat. Genet.},
  title        = {A slipped-CAG DNA-binding small molecule induces trinucleotide-repeat contractions in vivo},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Control of human hemoglobin switching by LIN28B-mediated
regulation of BCL11A translation. <em>NG</em>, <em>52</em>(2), 138–145.
(<a href="https://doi.org/10.1038/s41588-019-0568-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increased production of fetal hemoglobin (HbF) can ameliorate the severity of sickle cell disease and β-thalassemia1. BCL11A represses the genes encoding HbF and regulates human hemoglobin switching through variation in its expression during development2–7. However, the mechanisms underlying the developmental expression of BCL11A remain mysterious. Here we show that BCL11A is regulated at the level of messenger RNA (mRNA) translation during human hematopoietic development. Despite decreased BCL11A protein synthesis earlier in development, BCL11A mRNA continues to be associated with ribosomes. Through unbiased genomic and proteomic analyses, we demonstrate that the RNA-binding protein LIN28B, which is developmentally expressed in a pattern reciprocal to that of BCL11A, directly interacts with ribosomes and BCL11A mRNA. Furthermore, we show that BCL11A mRNA translation is suppressed by LIN28B through direct interactions, independently of its role in regulating let-7 microRNAs, and that BCL11A is the major target of LIN28B-mediated HbF induction. Our results reveal a previously unappreciated mechanism underlying human hemoglobin switching that illuminates new therapeutic opportunities. Experiments in developing human erythroid cells show that LIN28B controls hemoglobin switching by directly suppressing BCL11A translation, independently of its role in regulating let-7 microRNA biogenesis.},
  archive      = {J_NG},
  author       = {Basak, Anindita and Munschauer, Mathias and Lareau, Caleb A. and Montbleau, Kara E. and Ulirsch, Jacob C. and Hartigan, Christina R. and Schenone, Monica and Lian, John and Wang, Yaomei and Huang, Yumin and Wu, Xianfang and Gehrke, Lee and Rice, Charles M. and An, Xiuli and Christou, Helen A. and Mohandas, Narla and Carr, Steven A. and Chen, Jane-Jane and Orkin, Stuart H. and Lander, Eric S. and Sankaran, Vijay G.},
  doi          = {10.1038/s41588-019-0568-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {138-145},
  shortjournal = {Nat. Genet.},
  title        = {Control of human hemoglobin switching by LIN28B-mediated regulation of BCL11A translation},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A small molecule kicks repeat expansion into reverse.
<em>NG</em>, <em>52</em>(2), 136–137. (<a
href="https://doi.org/10.1038/s41588-020-0577-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Huntington’s disease is a severe progressive neurological disorder caused by a CAG-repeat expansion in the HTT gene. A small molecule shows therapeutic potential by inducing contraction of these expanded CAG repeats in cell and mouse models of the disease.},
  archive      = {J_NG},
  author       = {Flower, Michael D. and Tabrizi, Sarah J.},
  doi          = {10.1038/s41588-020-0577-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {136-137},
  shortjournal = {Nat. Genet.},
  title        = {A small molecule kicks repeat expansion into reverse},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Necessary voices. <em>NG</em>, <em>52</em>(2), 135. (<a
href="https://doi.org/10.1038/s41588-020-0585-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Given that it is based on heredity and has the ability to trace connections, genetics as a scientific discipline should be among those most attuned to diversity and global perspectives. Indigenous communities and scientists have much to contribute to genetics research, and they are making their voices heard. We celebrate these essential members of our larger genetics community, and we look forward to working together to learn from past examples, meet present challenges and support future opportunities.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-020-0585-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {135},
  shortjournal = {Nat. Genet.},
  title        = {Necessary voices},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Identifying cross-disease components of genetic risk across
hospital data in the UK biobank. <em>NG</em>, <em>52</em>(1), 126–134.
(<a href="https://doi.org/10.1038/s41588-019-0550-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic risk factors frequently affect multiple common human diseases, providing insight into shared pathophysiological pathways and opportunities for therapeutic development. However, systematic identification of genetic profiles of disease risk is limited by the availability of both comprehensive clinical data on population-scale cohorts and the lack of suitable statistical methodology that can handle the scale of and differential power inherent in multi-phenotype data. Here, we develop a disease-agnostic approach to cluster the genetic risk profiles for 3,025 genome-wide independent loci across 19,155 disease classification codes from 320,644 participants in the UK Biobank, representing a large and heterogeneous population. We identify 339 distinct disease association profiles and use multiple approaches to link clusters to the underlying biological pathways. We show how clusters can decompose the variance and covariance in risk for disease, thereby identifying underlying biological processes and their impact. We demonstrate the use of clusters in defining disease relationships and their potential in informing therapeutic strategies. A method to cluster genetic risk profiles applied to 3,025 loci across 19,155 disease codes from over 300,000 individuals in the UK Biobank identifies 339 distinct disease association profiles and links clusters to biological pathways.},
  archive      = {J_NG},
  author       = {Cortes, Adrian and Albers, Patrick K. and Dendrou, Calliope A. and Fugger, Lars and McVean, Gil},
  doi          = {10.1038/s41588-019-0550-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {126-134},
  shortjournal = {Nat. Genet.},
  title        = {Identifying cross-disease components of genetic risk across hospital data in the UK biobank},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Resequencing of 683 common bean genotypes identifies yield
component trait associations across a north–south cline. <em>NG</em>,
<em>52</em>(1), 118–125. (<a
href="https://doi.org/10.1038/s41588-019-0546-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted a large-scale genome-wide association study evaluation of 683 common bean accessions, including landraces and breeding lines, grown over 3 years and in four environments across China, ranging in latitude from 18.23° to 45.75° N, with different planting dates and abiotic or biotic stresses. A total of 505 loci were associated with yield components, of which seed size, flowering time and harvest maturity traits were stable across years and environments. Some loci aligned with candidate genes controlling these traits. Yield components were observed to have strong associations with a gene-rich region on the long arm of chromosome 1. Manipulation of seed size, through selection of seed length versus seed width and height, was deemed possible, providing a genome-based means to select for important yield components. This study shows that evaluation of large germplasm collections across north–south geographic clines is useful in the detection of marker associations that determine grain yield in pulses. Resequencing and genome-wide association analysis of 683 common bean accessions across different latitudes identifies 505 loci associated with yield components, of which seed size, flowering and harvest maturity traits are stable across environments.},
  archive      = {J_NG},
  author       = {Wu, Jing and Wang, Lanfen and Fu, Junjie and Chen, Jibao and Wei, Shuhong and Zhang, Shilong and Zhang, Jie and Tang, Yongsheng and Chen, Mingli and Zhu, Jifeng and Lei, Lei and Geng, Qinghe and Liu, Chunliang and Wu, Lei and Li, Xiaoming and Wang, Xiaoli and Wang, Qiang and Wang, Zhaoli and Xing, Shilai and Zhang, Haikuan and Blair, Matthew W. and Wang, Shumin},
  doi          = {10.1038/s41588-019-0546-0},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {118-125},
  shortjournal = {Nat. Genet.},
  title        = {Resequencing of 683 common bean genotypes identifies yield component trait associations across a north–south cline},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The indian cobra reference genome and transcriptome enables
comprehensive identification of venom toxins. <em>NG</em>,
<em>52</em>(1), 106–117. (<a
href="https://doi.org/10.1038/s41588-019-0559-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Snakebite envenoming is a serious and neglected tropical disease that kills ~100,000 people annually. High-quality, genome-enabled comprehensive characterization of toxin genes will facilitate development of effective humanized recombinant antivenom. We report a de novo near-chromosomal genome assembly of Naja naja, the Indian cobra, a highly venomous, medically important snake. Our assembly has a scaffold N50 of 223.35 Mb, with 19 scaffolds containing 95% of the genome. Of the 23,248 predicted protein-coding genes, 12,346 venom-gland-expressed genes constitute the ‘venom-ome’ and this included 139 genes from 33 toxin families. Among the 139 toxin genes were 19 ‘venom-ome-specific toxins’ (VSTs) that showed venom-gland-specific expression, and these probably encode the minimal core venom effector proteins. Synthetic venom reconstituted through recombinant VST expression will aid in the rapid development of safe and effective synthetic antivenom. Additionally, our genome could serve as a reference for snake genomes, support evolutionary studies and enable venom-driven drug discovery. Analysis of a near-chromosomal genome assembly and transcriptome profiling of the Indian cobra identifies genes expressed in the venom glands. These data should help develop a new antivenom.},
  archive      = {J_NG},
  author       = {Suryamohan, Kushal and Krishnankutty, Sajesh P. and Guillory, Joseph and Jevit, Matthew and Schröder, Markus S. and Wu, Meng and Kuriakose, Boney and Mathew, Oommen K. and Perumal, Rajadurai C. and Koludarov, Ivan and Goldstein, Leonard D. and Senger, Kate and Dixon, Mandumpala Davis and Velayutham, Dinesh and Vargas, Derek and Chaudhuri, Subhra and Muraleedharan, Megha and Goel, Ridhi and Chen, Ying-Jiun J. and Ratan, Aakrosh and Liu, Peter and Faherty, Brendan and de la Rosa, Guillermo and Shibata, Hiroki and Baca, Miriam and Sagolla, Meredith and Ziai, James and Wright, Gus A. and Vucic, Domagoj and Mohan, Sangeetha and Antony, Aju and Stinson, Jeremy and Kirkpatrick, Donald S. and Hannoush, Rami N. and Durinck, Steffen and Modrusan, Zora and Stawiski, Eric W. and Wiley, Kristen and Raudsepp, Terje and Kini, R. Manjunatha and Zachariah, Arun and Seshagiri, Somasekar},
  doi          = {10.1038/s41588-019-0559-8},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {106-117},
  shortjournal = {Nat. Genet.},
  title        = {The indian cobra reference genome and transcriptome enables comprehensive identification of venom toxins},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Epigenomic analysis of gastrulation identifies a unique
chromatin state for primed pluripotency. <em>NG</em>, <em>52</em>(1),
95–105. (<a href="https://doi.org/10.1038/s41588-019-0545-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Around implantation, the epiblast (Epi) transits from naïve to primed pluripotency, before giving rise to the three germ layers. How chromatin is reconfigured during this developmental window remains poorly understood. We performed a genome-wide investigation of chromatin landscapes during this period. We find that enhancers in ectoderm are already pre-accessible in embryonic day 6.5 (E6.5) Epi when cells enter a primed pluripotent state. Unexpectedly, strong trimethylation of histone H3 at lysine 4 (H3K4me3) emerges at developmental gene promoters in E6.5 Epi and positively correlates with H3K27me3, thus establishing bivalency. These genes also show enhanced spatial interactions. Both the strong bivalency and spatial clustering are virtually absent in preimplantation embryos and are markedly reduced in fate-committed lineages. Finally, we show that KMT2B is essential for establishing bivalent H3K4me3 at E6.5 but becomes partially dispensable later. Its deficiency leads to impaired activation of developmental genes and subsequent embryonic lethality. Thus, our data characterize lineage-specific chromatin reconfiguration and a unique chromatin state for primed pluripotency. Genome-wide maps of chromatin dynamics during mouse gastrulation highlight a unique bivalent signature at developmental genes in E6.5 epiblast cells. KMT2B is essential for bivalent H3K4me3 at E6.5.},
  archive      = {J_NG},
  author       = {Xiang, Yunlong and Zhang, Yu and Xu, Qianhua and Zhou, Chen and Liu, Bofeng and Du, Zhenhai and Zhang, Ke and Zhang, Bingjie and Wang, Xiaoxiao and Gayen, Srimonta and Liu, Ling and Wang, Yao and Li, Yuanyuan and Wang, Qiujun and Kalantry, Sundeep and Li, Lei and Xie, Wei},
  doi          = {10.1038/s41588-019-0545-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {95-105},
  shortjournal = {Nat. Genet.},
  title        = {Epigenomic analysis of gastrulation identifies a unique chromatin state for primed pluripotency},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). RNA isoform screens uncover the essentiality and
tumor-suppressor activity of ultraconserved poison exons. <em>NG</em>,
<em>52</em>(1), 84–94. (<a
href="https://doi.org/10.1038/s41588-019-0555-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While RNA-seq has enabled comprehensive quantification of alternative splicing, no correspondingly high-throughput assay exists for functionally interrogating individual isoforms. We describe pgFARM (paired guide RNAs for alternative exon removal), a CRISPR–Cas9-based method to manipulate isoforms independent of gene inactivation. This approach enabled rapid suppression of exon recognition in polyclonal settings to identify functional roles for individual exons, such as an SMNDC1 cassette exon that regulates pan-cancer intron retention. We generalized this method to a pooled screen to measure the functional relevance of ‘poison’ cassette exons, which disrupt their host genes’ reading frames yet are frequently ultraconserved. Many poison exons were essential for the growth of both cultured cells and lung adenocarcinoma xenografts, while a subset had clinically relevant tumor-suppressor activity. The essentiality and cancer relevance of poison exons are likely to contribute to their unusually high conservation and contrast with the dispensability of other ultraconserved elements for viability. pgFARM (paired guide RNAs for alternative exon removal) is a CRISPR–Cas9-based approach to manipulate alternative splicing and identify functional roles for individual exons, including poison exons with essential and tumor-suppressor roles.},
  archive      = {J_NG},
  author       = {Thomas, James D. and Polaski, Jacob T. and Feng, Qing and De Neef, Emma J. and Hoppe, Emma R. and McSharry, Maria V. and Pangallo, Joseph and Gabel, Austin M. and Belleville, Andrea E. and Watson, Jacqueline and Nkinsi, Naomi T. and Berger, Alice H. and Bradley, Robert K.},
  doi          = {10.1038/s41588-019-0555-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {84-94},
  shortjournal = {Nat. Genet.},
  title        = {RNA isoform screens uncover the essentiality and tumor-suppressor activity of ultraconserved poison exons},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genomic evidence supports a clonal diaspora model for
metastases of esophageal adenocarcinoma. <em>NG</em>, <em>52</em>(1),
74–83. (<a href="https://doi.org/10.1038/s41588-019-0551-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The poor outcomes in esophageal adenocarcinoma (EAC) prompted us to interrogate the pattern and timing of metastatic spread. Whole-genome sequencing and phylogenetic analysis of 388 samples across 18 individuals with EAC showed, in 90% of patients, that multiple subclones from the primary tumor spread very rapidly from the primary site to form multiple metastases, including lymph nodes and distant tissues—a mode of dissemination that we term ‘clonal diaspora’. Metastatic subclones at autopsy were present in tissue and blood samples from earlier time points. These findings have implications for our understanding and clinical evaluation of EAC. Whole-genome sequencing and phylogenetic analysis of nearly 400 esophageal adenocarcinoma samples suggest a model of rapid subclone spreading from the primary tumor to multiple metastatic sites, which the authors term ‘clonal diaspora’.},
  archive      = {J_NG},
  author       = {Noorani, Ayesha and Li, Xiaodun and Goddard, Martin and Crawte, Jason and Alexandrov, Ludmil B. and Secrier, Maria and Eldridge, Matthew D. and Bower, Lawrence and Weaver, Jamie and Lao-Sirieix, Pierre and Martincorena, Inigo and Debiram-Beecham, Irene and Grehan, Nicola and MacRae, Shona and Malhotra, Shalini and Miremadi, Ahmad and Thomas, Tabitha and Galbraith, Sarah and Petersen, Lorraine and Preston, Stephen D. and Gilligan, David and Hindmarsh, Andrew and Hardwick, Richard H. and Stratton, Michael R. and Wedge, David C. and Fitzgerald, Rebecca C.},
  doi          = {10.1038/s41588-019-0551-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {74-83},
  shortjournal = {Nat. Genet.},
  title        = {Genomic evidence supports a clonal diaspora model for metastases of esophageal adenocarcinoma},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Fine-mapping of 150 breast cancer risk regions identifies
191 likely target genes. <em>NG</em>, <em>52</em>(1), 56–73. (<a
href="https://doi.org/10.1038/s41588-019-0537-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have identified breast cancer risk variants in over 150 genomic regions, but the mechanisms underlying risk remain largely unknown. These regions were explored by combining association analysis with in silico genomic feature annotations. We defined 205 independent risk-associated signals with the set of credible causal variants in each one. In parallel, we used a Bayesian approach (PAINTOR) that combines genetic association, linkage disequilibrium and enriched genomic features to determine variants with high posterior probabilities of being causal. Potentially causal variants were significantly over-represented in active gene regulatory regions and transcription factor binding sites. We applied our INQUSIT pipeline for prioritizing genes as targets of those potentially causal variants, using gene expression (expression quantitative trait loci), chromatin interaction and functional annotations. Known cancer drivers, transcription factors and genes in the developmental, apoptosis, immune system and DNA integrity checkpoint gene ontology pathways were over-represented among the highest-confidence target genes. Fine-mapping of causal variants and integration of epigenetic and chromatin conformation data identify likely target genes for 150 breast cancer risk regions.},
  archive      = {J_NG},
  author       = {Fachal, Laura and Aschard, Hugues and Beesley, Jonathan and Barnes, Daniel R. and Allen, Jamie and Kar, Siddhartha and Pooley, Karen A. and Dennis, Joe and Michailidou, Kyriaki and Turman, Constance and Soucy, Penny and Lemaçon, Audrey and Lush, Michael and Tyrer, Jonathan P. and Ghoussaini, Maya and Moradi Marjaneh, Mahdi and Jiang, Xia and Agata, Simona and Aittomäki, Kristiina and Alonso, M. Rosario and Andrulis, Irene L. and Anton-Culver, Hoda and Antonenkova, Natalia N. and Arason, Adalgeir and Arndt, Volker and Aronson, Kristan J. and Arun, Banu K. and Auber, Bernd and Auer, Paul L. and Azzollini, Jacopo and Balmaña, Judith and Barkardottir, Rosa B. and Barrowdale, Daniel and Beeghly-Fadiel, Alicia and Benitez, Javier and Bermisheva, Marina and Białkowska, Katarzyna and Blanco, Amie M. and Blomqvist, Carl and Blot, William and Bogdanova, Natalia V. and Bojesen, Stig E. and Bolla, Manjeet K. and Bonanni, Bernardo and Borg, Ake and Bosse, Kristin and Brauch, Hiltrud and Brenner, Hermann and Briceno, Ignacio and Brock, Ian W. and Brooks-Wilson, Angela and Brüning, Thomas and Burwinkel, Barbara and Buys, Saundra S. and Cai, Qiuyin and Caldés, Trinidad and Caligo, Maria A. and Camp, Nicola J. and Campbell, Ian and Canzian, Federico and Carroll, Jason S. and Carter, Brian D. and Castelao, Jose E. and Chiquette, Jocelyne and Christiansen, Hans and Chung, Wendy K. and Claes, Kathleen B. M. and Clarke, Christine L. and Collée, J. Margriet and Cornelissen, Sten and Couch, Fergus J. and Cox, Angela and Cross, Simon S. and Cybulski, Cezary and Czene, Kamila and Daly, Mary B. and de la Hoya, Miguel and Devilee, Peter and Diez, Orland and Ding, Yuan Chun and Dite, Gillian S. and Domchek, Susan M. and Dörk, Thilo and dos-Santos-Silva, Isabel and Droit, Arnaud and Dubois, Stéphane and Dumont, Martine and Duran, Mercedes and Durcan, Lorraine and Dwek, Miriam and Eccles, Diana M. and Engel, Christoph and Eriksson, Mikael and Evans, D. Gareth and Fasching, Peter A. and Fletcher, Olivia and Floris, Giuseppe and Flyger, Henrik and Foretova, Lenka and Foulkes, William D. and Friedman, Eitan and Fritschi, Lin and Frost, Debra and Gabrielson, Marike and Gago-Dominguez, Manuela and Gambino, Gaetana and Ganz, Patricia A. and Gapstur, Susan M. and Garber, Judy and García-Sáenz, José A. and Gaudet, Mia M. and Georgoulias, Vassilios and Giles, Graham G. and Glendon, Gord and Godwin, Andrew K. and Goldberg, Mark S. and Goldgar, David E. and González-Neira, Anna and Tibiletti, Maria Grazia and Greene, Mark H. and Grip, Mervi and Gronwald, Jacek and Grundy, Anne and Guénel, Pascal and Hahnen, Eric and Haiman, Christopher A. and Håkansson, Niclas and Hall, Per and Hamann, Ute and Harrington, Patricia A. and Hartikainen, Jaana M. and Hartman, Mikael and He, Wei and Healey, Catherine S. and Heemskerk-Gerritsen, Bernadette A. M. and Heyworth, Jane and Hillemanns, Peter and Hogervorst, Frans B. L. and Hollestelle, Antoinette and Hooning, Maartje J. and Hopper, John L. and Howell, Anthony and Huang, Guanmengqian and Hulick, Peter J. and Imyanitov, Evgeny N. and Isaacs, Claudine and Iwasaki, Motoki and Jager, Agnes and Jakimovska, Milena and Jakubowska, Anna and James, Paul A. and Janavicius, Ramunas and Jankowitz, Rachel C. and John, Esther M. and Johnson, Nichola and Jones, Michael E. and Jukkola-Vuorinen, Arja and Jung, Audrey and Kaaks, Rudolf and Kang, Daehee and Kapoor, Pooja Middha and Karlan, Beth Y. and Keeman, Renske and Kerin, Michael J. and Khusnutdinova, Elza and Kiiski, Johanna I. and Kirk, Judy and Kitahara, Cari M. and Ko, Yon-Dschun and Konstantopoulou, Irene and Kosma, Veli-Matti and Koutros, Stella and Kubelka-Sabit, Katerina and Kwong, Ava and Kyriacou, Kyriacos and Laitman, Yael and Lambrechts, Diether and Lee, Eunjung and Leslie, Goska and Lester, Jenny and Lesueur, Fabienne and Lindblom, Annika and Lo, Wing-Yee and Long, Jirong and Lophatananon, Artitaya and Loud, Jennifer T. and Lubiński, Jan and MacInnis, Robert J. and Maishman, Tom and Makalic, Enes and Mannermaa, Arto and Manoochehri, Mehdi and Manoukian, Siranoush and Margolin, Sara and Martinez, Maria Elena and Matsuo, Keitaro and Maurer, Tabea and Mavroudis, Dimitrios and Mayes, Rebecca and McGuffog, Lesley and McLean, Catriona and Mebirouk, Noura and Meindl, Alfons and Miller, Austin and Miller, Nicola and Montagna, Marco and Moreno, Fernando and Muir, Kenneth and Mulligan, Anna Marie and Muñoz-Garzon, Victor M. and Muranen, Taru A. and Narod, Steven A. and Nassir, Rami and Nathanson, Katherine L. and Neuhausen, Susan L. and Nevanlinna, Heli and Neven, Patrick and Nielsen, Finn C. and Nikitina-Zake, Liene and Norman, Aaron and Offit, Kenneth and Olah, Edith and Olopade, Olufunmilayo I. and Olsson, Håkan and Orr, Nick and Osorio, Ana and Pankratz, V. Shane and Papp, Janos and Park, Sue K. and Park-Simon, Tjoung-Won and Parsons, Michael T. and Paul, James and Pedersen, Inge Sokilde and Peissel, Bernard and Peshkin, Beth and Peterlongo, Paolo and Peto, Julian and Plaseska-Karanfilska, Dijana and Prajzendanc, Karolina and Prentice, Ross and Presneau, Nadege and Prokofyeva, Darya and Pujana, Miquel Angel and Pylkäs, Katri and Radice, Paolo and Ramus, Susan J. and Rantala, Johanna and Rau-Murthy, Rohini and Rennert, Gad and Risch, Harvey A. and Robson, Mark and Romero, Atocha and Rossing, Maria and Saloustros, Emmanouil and Sánchez-Herrero, Estela and Sandler, Dale P. and Santamariña, Marta and Saunders, Christobel and Sawyer, Elinor J. and Scheuner, Maren T. and Schmidt, Daniel F. and Schmutzler, Rita K. and Schneeweiss, Andreas and Schoemaker, Minouk J. and Schöttker, Ben and Schürmann, Peter and Scott, Christopher and Scott, Rodney J. and Senter, Leigha and Seynaeve, Caroline M. and Shah, Mitul and Sharma, Priyanka and Shen, Chen-Yang and Shu, Xiao-Ou and Singer, Christian F. and Slavin, Thomas P. and Smichkoska, Snezhana and Southey, Melissa C. and Spinelli, John J. and Spurdle, Amanda B. and Stone, Jennifer and Stoppa-Lyonnet, Dominique and Sutter, Christian and Swerdlow, Anthony J. and Tamimi, Rulla M. and Tan, Yen Yen and Tapper, William J. and Taylor, Jack A. and Teixeira, Manuel R. and Tengström, Maria and Teo, Soo Hwang and Terry, Mary Beth and Teulé, Alex and Thomassen, Mads and Thull, Darcy L. and Tischkowitz, Marc and Toland, Amanda E. and Tollenaar, Rob A. E. M. and Tomlinson, Ian and Torres, Diana and Torres-Mejía, Gabriela and Troester, Melissa A. and Truong, Thérèse and Tung, Nadine and Tzardi, Maria and Ulmer, Hans-Ulrich and Vachon, Celine M. and van Asperen, Christi J. and van der Kolk, Lizet E. and van Rensburg, Elizabeth J. and Vega, Ana and Viel, Alessandra and Vijai, Joseph and Vogel, Maartje J. and Wang, Qin and Wappenschmidt, Barbara and Weinberg, Clarice R. and Weitzel, Jeffrey N. and Wendt, Camilla and Wildiers, Hans and Winqvist, Robert and Wolk, Alicja and Wu, Anna H. and Yannoukakos, Drakoulis and Zhang, Yan and Zheng, Wei and Hunter, David and Pharoah, Paul D. P. and Chang-Claude, Jenny and García-Closas, Montserrat and Schmidt, Marjanka K. and Milne, Roger L. and Kristensen, Vessela N. and French, Juliet D. and Edwards, Stacey L. and Antoniou, Antonis C. and Chenevix-Trench, Georgia and Simard, Jacques and Easton, Douglas F. and Kraft, Peter and Dunning, Alison M.},
  doi          = {10.1038/s41588-019-0537-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {56-73},
  shortjournal = {Nat. Genet.},
  title        = {Fine-mapping of 150 breast cancer risk regions identifies 191 likely target genes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). N6-methyladenosine regulates the stability of RNA: DNA
hybrids in human cells. <em>NG</em>, <em>52</em>(1), 48–55. (<a
href="https://doi.org/10.1038/s41588-019-0549-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {R-loops are nucleic acid structures formed by an RNA:DNA hybrid and unpaired single-stranded DNA that represent a source of genomic instability in mammalian cells1–4. Here we show that N6-methyladenosine (m6A) modification, contributing to different aspects of messenger RNA metabolism5,6, is detectable on the majority of RNA:DNA hybrids in human pluripotent stem cells. We demonstrate that m6A-containing R-loops accumulate during G2/M and are depleted at G0/G1 phases of the cell cycle, and that the m6A reader promoting mRNA degradation, YTHDF2 (ref. 7), interacts with R-loop-enriched loci in dividing cells. Consequently, YTHDF2 knockout leads to increased R-loop levels, cell growth retardation and accumulation of γH2AX, a marker for DNA double-strand breaks, in mammalian cells. Our results suggest that m6A regulates accumulation of R-loops, implying a role for this modification in safeguarding genomic stability. N6-methyladenosine (m6A) is prevalent at RNA:DNA hybrids in human pluripotent stem cells. The m6A reader YTHDF2 interacts with R-loop-enriched loci in dividing cells, and YTHDF2 loss leads to increased R-loop levels and accumulation of γH2AX.},
  archive      = {J_NG},
  author       = {Abakir, Abdulkadir and Giles, Tom C. and Cristini, Agnese and Foster, Jeremy M. and Dai, Nan and Starczak, Marta and Rubio-Roldan, Alejandro and Li, Miaomiao and Eleftheriou, Maria and Crutchley, James and Flatt, Luke and Young, Lorraine and Gaffney, Daniel J. and Denning, Chris and Dalhus, Bjørn and Emes, Richard D. and Gackowski, Daniel and Corrêa, Ivan R. and Garcia-Perez, Jose L. and Klungland, Arne and Gromak, Natalia and Ruzov, Alexey},
  doi          = {10.1038/s41588-019-0549-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {48-55},
  shortjournal = {Nat. Genet.},
  title        = {N6-methyladenosine regulates the stability of RNA: DNA hybrids in human cells},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Loss of ADAMTS19 causes progressive non-syndromic heart
valve disease. <em>NG</em>, <em>52</em>(1), 40–47. (<a
href="https://doi.org/10.1038/s41588-019-0536-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Valvular heart disease is observed in approximately 2% of the general population1. Although the initial observation is often localized (for example, to the aortic or mitral valve), disease manifestations are regularly observed in the other valves and patients frequently require surgery. Despite the high frequency of heart valve disease, only a handful of genes have so far been identified as the monogenic causes of disease2–7. Here we identify two consanguineous families, each with two affected family members presenting with progressive heart valve disease early in life. Whole-exome sequencing revealed homozygous, truncating nonsense alleles in ADAMTS19 in all four affected individuals. Homozygous knockout mice for Adamts19 show aortic valve dysfunction, recapitulating aspects of the human phenotype. Expression analysis using a lacZ reporter and single-cell RNA sequencing highlight Adamts19 as a novel marker for valvular interstitial cells; inference of gene regulatory networks in valvular interstitial cells positions Adamts19 in a highly discriminatory network driven by the transcription factor lymphoid enhancer-binding factor 1 downstream of the Wnt signaling pathway. Upregulation of endocardial Krüppel-like factor 2 in Adamts19 knockout mice precedes hemodynamic perturbation, showing that a tight balance in the Wnt–Adamts19–Klf2 axis is required for proper valve maturation and maintenance. Mutations in ADAMTS19 lead to progressive heart valve disease in humans. Analysis of mice lacking Adamts19 highlights the role of a Wnt–Adamts19–Klf2 axis in proper valve function.},
  archive      = {J_NG},
  author       = {Wünnemann, Florian and Ta-Shma, Asaf and Preuss, Christoph and Leclerc, Severine and van Vliet, Patrick Piet and Oneglia, Andrea and Thibeault, Maryse and Nordquist, Emily and Lincoln, Joy and Scharfenberg, Franka and Becker-Pauly, Christoph and Hofmann, Philipp and Hoff, Kirstin and Audain, Enrique and Kramer, Hans-Heiner and Makalowski, Wojciech and Nir, Amiram and Gerety, Sebastian S. and Hurles, Matthew and Comes, Johanna and Fournier, Anne and Osinska, Hanna and Robins, Jeffrey and Pucéat, Michel and Elpeleg, Orly and Hitz, Marc-Phillip and Andelfinger, Gregor},
  doi          = {10.1038/s41588-019-0536-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {40-47},
  shortjournal = {Nat. Genet.},
  title        = {Loss of ADAMTS19 causes progressive non-syndromic heart valve disease},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). GeVIR is a continuous gene-level metric that uses variant
distribution patterns to prioritize disease candidate genes.
<em>NG</em>, <em>52</em>(1), 35–39. (<a
href="https://doi.org/10.1038/s41588-019-0560-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With large-scale population sequencing projects gathering pace, there is a need for strategies that advance disease gene prioritization1,2. Metrics that provide information about a gene and its ability to tolerate protein-altering variation can aid in clinical interpretation of human genomes and can advance disease gene discovery1–4. Previous reported methods analyzed the total variant load in a gene1–4, but did not analyze the distribution pattern of variants within a gene. Using data from 138,632 exome and genome sequences2, we developed gene variation intolerance rank (GeVIR), a continuous gene-level metric for 19,361 genes that is able to prioritize both dominant and recessive Mendelian disease genes5, that outperforms missense constraint metrics3 and that is comparable—but complementary—to loss-of-function (LOF) constraint metrics2. GeVIR is also able to prioritize short genes, for which LOF constraint cannot be estimated with confidence2. The majority of the most intolerant genes identified here have no defined phenotype and are candidates for severe dominant disorders. GeVIR is a continuous gene-level metric that uses variant distribution patterns to prioritize both dominant and recessive Mendelian disease genes. GeVIR outperforms missense constraint metrics and complements loss-of-function constraint metrics.},
  archive      = {J_NG},
  author       = {Abramovs, Nikita and Brass, Andrew and Tassabehji, May},
  doi          = {10.1038/s41588-019-0560-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {35-39},
  shortjournal = {Nat. Genet.},
  title        = {GeVIR is a continuous gene-level metric that uses variant distribution patterns to prioritize disease candidate genes},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Extrachromosomal circular DNA drives oncogenic genome
remodeling in neuroblastoma. <em>NG</em>, <em>52</em>(1), 29–34. (<a
href="https://doi.org/10.1038/s41588-019-0547-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extrachromosomal circularization of DNA is an important genomic feature in cancer. However, the structure, composition and genome-wide frequency of extrachromosomal circular DNA have not yet been profiled extensively. Here, we combine genomic and transcriptomic approaches to describe the landscape of extrachromosomal circular DNA in neuroblastoma, a tumor arising in childhood from primitive cells of the sympathetic nervous system. Our analysis identifies and characterizes a wide catalog of somatically acquired and undescribed extrachromosomal circular DNAs. Moreover, we find that extrachromosomal circular DNAs are an unanticipated major source of somatic rearrangements, contributing to oncogenic remodeling through chimeric circularization and reintegration of circular DNA into the linear genome. Cancer-causing lesions can emerge out of circle-derived rearrangements and are associated with adverse clinical outcome. It is highly probable that circle-derived rearrangements represent an ongoing mutagenic process. Thus, extrachromosomal circular DNAs represent a multihit mutagenic process, with important functional and clinical implications for the origins of genomic remodeling in cancer. Combined genomic and transcriptomic approaches identify the landscape of extrachromosomal circular DNA in neuroblastoma and reveal that extrachromosomal circular DNA is a major source of somatic rearrangements.},
  archive      = {J_NG},
  author       = {Koche, Richard P. and Rodriguez-Fos, Elias and Helmsauer, Konstantin and Burkert, Martin and MacArthur, Ian C. and Maag, Jesper and Chamorro, Rocio and Munoz-Perez, Natalia and Puiggròs, Montserrat and Dorado Garcia, Heathcliff and Bei, Yi and Röefzaad, Claudia and Bardinet, Victor and Szymansky, Annabell and Winkler, Annika and Thole, Theresa and Timme, Natalie and Kasack, Katharina and Fuchs, Steffen and Klironomos, Filippos and Thiessen, Nina and Blanc, Eric and Schmelz, Karin and Künkele, Annette and Hundsdörfer, Patrick and Rosswog, Carolina and Theissen, Jessica and Beule, Dieter and Deubzer, Hedwig and Sauer, Sascha and Toedling, Joern and Fischer, Matthias and Hertwig, Falk and Schwarz, Roland F. and Eggert, Angelika and Torrents, David and Schulte, Johannes H. and Henssen, Anton G.},
  doi          = {10.1038/s41588-019-0547-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {29-34},
  shortjournal = {Nat. Genet.},
  title        = {Extrachromosomal circular DNA drives oncogenic genome remodeling in neuroblastoma},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). M6A RNA modification as a new player in r-loop regulation.
<em>NG</em>, <em>52</em>(1), 27–28. (<a
href="https://doi.org/10.1038/s41588-019-0563-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA:DNA hybrids that form across genomes control a wide range of biological processes. A new study shows that N6-methyladenosine (m6A) modification on the RNA moieties regulates the formation and genome integrity of these hybrids. This finding opens a new avenue of research on how RNA modifications (the ‘epitranscriptome’) can help control genome maintenance.},
  archive      = {J_NG},
  author       = {Marnef, Aline and Legube, Gaëlle},
  doi          = {10.1038/s41588-019-0563-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {27-28},
  shortjournal = {Nat. Genet.},
  title        = {M6A RNA modification as a new player in R-loop regulation},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A call for global action for rare diseases in africa.
<em>NG</em>, <em>52</em>(1), 21–26. (<a
href="https://doi.org/10.1038/s41588-019-0552-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The 11th International Conference on Rare Diseases and Orphan Drugs (ICORD), South Africa, included the Africa-Rare initiative launch and facilitated multi-stakeholder engagement in the challenges facing, and opportunities for, Africans living with rare diseases. The following ICORD Global Call to Action, developed in collaboration with the International Rare Diseases Research Consortium, synthesizes the outcomes of the deliberations and emphasizes the international collaborative efforts required to address the global effects of rare diseases on public health.},
  archive      = {J_NG},
  author       = {Baynam, Gareth S. and Groft, Stephen and van der Westhuizen, Francois H. and Gassman, Safiyya D. and du Plessis, Kelly and Coles, Emily P. and Selebatso, Eda and Selebatso, Moses and Gaobinelwe, Boikobo and Selebatso, Tebogo and Joel, Dipesalema and Llera, Virginia A. and Vorster, Barend C. and Wuebbels, Barbara and Djoudalbaye, Benjamin and Austin, Christopher P. and Kumuthini, Judit and Forman, John and Kaufmann, Petra and Chipeta, James and Gavhed, Désirée and Larsson, Annika and Stojiljkovic, Maja and Nordgren, Ann and Roldan, Emilio J. A. and Taruscio, Domenica and Wong-Rieger, Durhane and Nowak, Kristen and Bilkey, Gemma A. and Easteal, Simon and Bowdin, Sarah and Reichardt, Juergen K. V. and Beltran, Sergi and Kosaki, Kenjiro and van Karnebeek, Clara D. M. and Gong, Mengchun and Shuyang, Zhang and Mehrian-Shai, Ruty and Adams, David R. and Puri, Ratna D. and Zhang, Feng and Pachter, Nicholas and Muenke, Maximilian and Nellaker, Christoffer and Gahl, William A. and Cederroth, Helene and Broley, Stephanie and Schoonen, Maryke and Boycott, Kym M. and Posada, Manuel},
  doi          = {10.1038/s41588-019-0552-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {21-26},
  shortjournal = {Nat. Genet.},
  title        = {A call for global action for rare diseases in africa},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Returning incidental findings in african genomics research.
<em>NG</em>, <em>52</em>(1), 17–20. (<a
href="https://doi.org/10.1038/s41588-019-0542-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ancestral and geographical issues underlie the need to develop Africa-specific guidelines for the return of genomics research results in Africa. In this Commentary, we outline the challenges that will inform policies and practices in the future.},
  archive      = {J_NG},
  author       = {Wonkam, Ambroise and de Vries, Jantina},
  doi          = {10.1038/s41588-019-0542-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {17-20},
  shortjournal = {Nat. Genet.},
  title        = {Returning incidental findings in african genomics research},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). On the existence and functionality of topologically
associating domains. <em>NG</em>, <em>52</em>(1), 8–16. (<a
href="https://doi.org/10.1038/s41588-019-0561-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomes across a wide range of eukaryotic organisms fold into higher-order chromatin domains. Topologically associating domains (TADs) were originally discovered empirically in low-resolution Hi-C heat maps representing ensemble average interaction frequencies from millions of cells. Here, we discuss recent advances in high-resolution Hi-C, single-cell imaging experiments, and functional genetic studies, which provide an increasingly complex view of the genome’s hierarchical structure–function relationship. On the basis of these new findings, we update the definitions of distinct classes of chromatin domains according to emerging knowledge of their structural, mechanistic and functional properties. Recent advances in Hi-C, single-cell imaging and functional genetic studies warrant discussion on the functional relevance of topologically associating domains (TADs) and other classes of chromatin domains.},
  archive      = {J_NG},
  author       = {Beagan, Jonathan A. and Phillips-Cremins, Jennifer E.},
  doi          = {10.1038/s41588-019-0561-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {8-16},
  shortjournal = {Nat. Genet.},
  title        = {On the existence and functionality of topologically associating domains},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reply to: Revisiting the origin of octoploid strawberry.
<em>NG</em>, <em>52</em>(1), 5–7. (<a
href="https://doi.org/10.1038/s41588-019-0544-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Edger, Patrick P. and McKain, Michael R. and Yocca, Alan E. and Knapp, Steven J. and Qiao, Qin and Zhang, Ticao},
  doi          = {10.1038/s41588-019-0544-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {5-7},
  shortjournal = {Nat. Genet.},
  title        = {Reply to: Revisiting the origin of octoploid strawberry},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Revisiting the origin of octoploid strawberry. <em>NG</em>,
<em>52</em>(1), 2–4. (<a
href="https://doi.org/10.1038/s41588-019-0543-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Liston, Aaron and Wei, Na and Tennessen, Jacob A. and Li, Junmin and Dong, Ming and Ashman, Tia-Lynn},
  doi          = {10.1038/s41588-019-0543-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2-4},
  shortjournal = {Nat. Genet.},
  title        = {Revisiting the origin of octoploid strawberry},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Navigating 2020 and beyond. <em>NG</em>, <em>52</em>(1), 1.
(<a href="https://doi.org/10.1038/s41588-019-0570-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As we usher in a new year of a new decade and ponder what the future will bring for the genetics field, we wish to reflect on some specific areas related to diversity, privacy and genome editing that require attention and vigilance from the community.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-019-0570-0},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {Navigating 2020 and beyond},
  volume       = {52},
  year         = {2020},
}
</textarea>
</details></li>
</ul>

</body>
</html>
