<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng---351">NG - 351</h2>
<ul>
<li><details>
<summary>
(2023). Deep learning-based phenotype imputation on population-scale
biobank data increases genetic discoveries. <em>NG</em>,
<em>55</em>(12), 2269–2276. (<a
href="https://doi.org/10.1038/s41588-023-01558-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biobanks that collect deep phenotypic and genomic data across many individuals have emerged as a key resource in human genetics. However, phenotypes in biobanks are often missing across many individuals, limiting their utility. We propose AutoComplete, a deep learning-based imputation method to impute or ‘fill-in’ missing phenotypes in population-scale biobank datasets. When applied to collections of phenotypes measured across ~300,000 individuals from the UK Biobank, AutoComplete substantially improved imputation accuracy over existing methods. On three traits with notable amounts of missingness, we show that AutoComplete yields imputed phenotypes that are genetically similar to the originally observed phenotypes while increasing the effective sample size by about twofold on average. Further, genome-wide association analyses on the resulting imputed phenotypes led to a substantial increase in the number of associated loci. Our results demonstrate the utility of deep learning-based phenotype imputation to increase power for genetic discoveries in existing biobank datasets. AutoComplete is a deep learning-based method that imputes missing phenotypes in population-scale biobank datasets, increasing effective sample sizes and improving power for genetic discoveries in genome-wide association studies.},
  archive      = {J_NG},
  author       = {An, Ulzee and Pazokitoroudi, Ali and Alvarez, Marcus and Huang, Lianyun and Bacanu, Silviu and Schork, Andrew J. and Kendler, Kenneth and Pajukanta, Päivi and Flint, Jonathan and Zaitlen, Noah and Cai, Na and Dahl, Andy and Sankararaman, Sriram},
  doi          = {10.1038/s41588-023-01558-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2269-2276},
  shortjournal = {Nat. Genet.},
  title        = {Deep learning-based phenotype imputation on population-scale biobank data increases genetic discoveries},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping the dynamic genetic regulatory architecture of HLA
genes at single-cell resolution. <em>NG</em>, <em>55</em>(12),
2255–2268. (<a
href="https://doi.org/10.1038/s41588-023-01586-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human leukocyte antigen (HLA) locus plays a critical role in complex traits spanning autoimmune and infectious diseases, transplantation and cancer. While coding variation in HLA genes has been extensively documented, regulatory genetic variation modulating HLA expression levels has not been comprehensively investigated. Here we mapped expression quantitative trait loci (eQTLs) for classical HLA genes across 1,073 individuals and 1,131,414 single cells from three tissues. To mitigate technical confounding, we developed scHLApers, a pipeline to accurately quantify single-cell HLA expression using personalized reference genomes. We identified cell-type-specific cis-eQTLs for every classical HLA gene. Modeling eQTLs at single-cell resolution revealed that many eQTL effects are dynamic across cell states even within a cell type. HLA-DQ genes exhibit particularly cell-state-dependent effects within myeloid, B and T cells. For example, a T cell HLA-DQA1 eQTL ( rs3104371 ) is strongest in cytotoxic cells. Dynamic HLA regulation may underlie important interindividual variability in immune responses. scHLApers is an analysis pipeline that quantifies single-cell expression of HLA genes using a personalized genomic reference. Mapping of HLA expression quantitative trait loci at single-cell resolution identifies dynamic effects across cell states.},
  archive      = {J_NG},
  author       = {Kang, Joyce B. and Shen, Amber Z. and Gurajala, Saisriram and Nathan, Aparna and Rumker, Laurie and Aguiar, Vitor R. C. and Valencia, Cristian and Lagattuta, Kaitlyn A. and Zhang, Fan and Jonsson, Anna Helena and Yazar, Seyhan and Alquicira-Hernandez, Jose and Khalili, Hamed and Ananthakrishnan, Ashwin N. and Jagadeesh, Karthik and Dey, Kushal and Daly, Mark J. and Xavier, Ramnik J. and Donlin, Laura T. and Anolik, Jennifer H. and Powell, Joseph E. and Rao, Deepak A. and Brenner, Michael B. and Gutierrez-Arcelus, Maria and Luo, Yang and Sakaue, Saori and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-023-01586-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2255-2268},
  shortjournal = {Nat. Genet.},
  title        = {Mapping the dynamic genetic regulatory architecture of HLA genes at single-cell resolution},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pangenome analysis reveals genomic variations associated
with domestication traits in broomcorn millet. <em>NG</em>,
<em>55</em>(12), 2243–2254. (<a
href="https://doi.org/10.1038/s41588-023-01571-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Broomcorn millet (Panicum miliaceum L.) is an orphan crop with the potential to improve cereal production and quality, and ensure food security. Here we present the genetic variations, population structure and diversity of a diverse worldwide collection of 516 broomcorn millet genomes. Population analysis indicated that the domesticated broomcorn millet originated from its wild progenitor in China. We then constructed a graph-based pangenome of broomcorn millet based on long-read de novo genome assemblies of 32 representative accessions. Our analysis revealed that the structural variations were highly associated with transposable elements, which influenced gene expression when located in the coding or regulatory regions. We also identified 139 loci associated with 31 key domestication and agronomic traits, including candidate genes and superior haplotypes, such as LG1, for panicle architecture. Thus, the study’s findings provide foundational resources for developing genomics-assisted breeding programs in broomcorn millet. Population analysis of 516 wild and domesticated broomcorn millet genomes and a graph-based pangenome based on de novo assemblies of 32 representative accessions identify genomic variations associated with domestication traits.},
  archive      = {J_NG},
  author       = {Chen, Jinfeng and Liu, Yang and Liu, Minxuan and Guo, Wenlei and Wang, Yongqiang and He, Qiang and Chen, Weiyao and Liao, Yi and Zhang, Wei and Gao, Yuanzhu and Dong, Kongjun and Ren, Ruiyu and Yang, Tianyu and Zhang, Liyuan and Qi, Mingyu and Li, Zhiguang and Zhao, Min and Wang, Haigang and Wang, Junjie and Qiao, Zhijun and Li, Haiquan and Jiang, Yanmiao and Liu, Guoqing and Song, Xiaoqiang and Deng, Yarui and Li, Hai and Yan, Feng and Dong, Yang and Li, Qingquan and Li, Tao and Yang, Wenyao and Cui, Jianghui and Wang, Hongru and Zhou, Yongfeng and Zhang, Xiaoming and Jia, Guanqing and Lu, Ping and Zhi, Hui and Tang, Sha and Diao, Xianmin},
  doi          = {10.1038/s41588-023-01571-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2243-2254},
  shortjournal = {Nat. Genet.},
  title        = {Pangenome analysis reveals genomic variations associated with domestication traits in broomcorn millet},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A mutation rate model at the basepair resolution identifies
the mutagenic effect of polymerase III transcription. <em>NG</em>,
<em>55</em>(12), 2235–2242. (<a
href="https://doi.org/10.1038/s41588-023-01562-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo mutations occur at substantially different rates depending on genomic location, sequence context and DNA strand. The success of methods to estimate selection intensity, infer demographic history and map rare disease genes, depends strongly on assumptions about the local mutation rate. Here we present Roulette, a genome-wide mutation rate model at basepair resolution that incorporates known determinants of local mutation rate. Roulette is shown to be more accurate than existing models. We use Roulette to refine the estimates of population growth within Europe by incorporating the full range of human mutation rates. The analysis of significant deviations from the model predictions revealed a tenfold increase in mutation rate in nearly all genes transcribed by polymerase III (Pol III), suggesting a new mutagenic mechanism. We also detected an elevated mutation rate within transcription factor binding sites restricted to sites actively used in testis and residing in promoters. Roulette enables the estimation of germline mutation rates at basepair resolution from humans. Genes encoding small nuclear RNA showed significant deviations from the mutation rate predicted by Roulette, highlighting RNA polymerase III (Pol III)-dependent transcription as a potent source of mutations in the human genome.},
  archive      = {J_NG},
  author       = {Seplyarskiy, Vladimir and Koch, Evan M. and Lee, Daniel J. and Lichtman, Joshua S. and Luan, Harding H. and Sunyaev, Shamil R.},
  doi          = {10.1038/s41588-023-01562-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2235-2242},
  shortjournal = {Nat. Genet.},
  title        = {A mutation rate model at the basepair resolution identifies the mutagenic effect of polymerase III transcription},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Super-enhancer RNA m6A promotes local chromatin
accessibility and oncogene transcription in pancreatic ductal
adenocarcinoma. <em>NG</em>, <em>55</em>(12), 2224–2234. (<a
href="https://doi.org/10.1038/s41588-023-01568-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biological functions of noncoding RNA N6-methyladenosine (m6A) modification remain poorly understood. In the present study, we depict the landscape of super-enhancer RNA (seRNA) m6A modification in pancreatic ductal adenocarcinoma (PDAC) and reveal a regulatory axis of m6A seRNA, H3K4me3 modification, chromatin accessibility and oncogene transcription. We demonstrate the cofilin family protein CFL1, overexpressed in PDAC, as a METTL3 cofactor that helps seRNA m6A methylation formation. The increased seRNA m6As are recognized by the reader YTHDC2, which recruits H3K4 methyltransferase MLL1 to promote H3K4me3 modification cotranscriptionally. Super-enhancers with a high level of H3K4me3 augment chromatin accessibility and facilitate oncogene transcription. Collectively, these results shed light on a CFL1–METTL3–seRNA m6A–YTHDC2/MLL1 axis that plays a role in the epigenetic regulation of local chromatin state and gene expression, which strengthens our knowledge about the functions of super-enhancers and their transcripts. In pancreatic duct adenocarcinoma, super-enhancer RNAs (seRNAs) have higher N6-methyladenosine (m6A) levels than in adjacent normal tissue due to upregulation of the METTL3 cofactor CFL1. Aberrant m6A seRNAs promote oncogene expression via the YTHDC2–MLL1 complex.},
  archive      = {J_NG},
  author       = {Li, Rui and Zhao, Hongzhe and Huang, Xudong and Zhang, Jialiang and Bai, Ruihong and Zhuang, Lisha and Wen, Shujuan and Wu, Shaojia and Zhou, Quanbo and Li, Mei and Zeng, Lingxing and Zhang, Shaoping and Deng, Shuang and Su, Jiachun and Zuo, Zhixiang and Chen, Rufu and Lin, Dongxin and Zheng, Jian},
  doi          = {10.1038/s41588-023-01568-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2224-2234},
  shortjournal = {Nat. Genet.},
  title        = {Super-enhancer RNA m6A promotes local chromatin accessibility and oncogene transcription in pancreatic ductal adenocarcinoma},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Transcriptional and epigenetic regulators of human CD8+ t
cell function identified through orthogonal CRISPR screens. <em>NG</em>,
<em>55</em>(12), 2211–2223. (<a
href="https://doi.org/10.1038/s41588-023-01554-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clinical response to adoptive T cell therapies is associated with the transcriptional and epigenetic state of the cell product. Thus, discovery of regulators of T cell gene networks and their corresponding phenotypes has potential to improve T cell therapies. Here we developed pooled, epigenetic CRISPR screening approaches to systematically profile the effects of activating or repressing 120 transcriptional and epigenetic regulators on human CD8+ T cell state. We found that BATF3 overexpression promoted specific features of memory T cells and attenuated gene programs associated with cytotoxicity, regulatory T cell function, and exhaustion. Upon chronic antigen stimulation, BATF3 overexpression countered phenotypic and epigenetic signatures of T cell exhaustion. Moreover, BATF3 enhanced the potency of CAR T cells in both in vitro and in vivo tumor models and programmed a transcriptional profile that correlates with positive clinical response to adoptive T cell therapy. Finally, we performed CRISPR knockout screens that defined cofactors and downstream mediators of the BATF3 gene network. CRISPR activation/interference screens identify transcriptional regulators of human CD8+ T cells, including BATF3. BATF3 overexpression counteracts T cell exhaustion and enhances cancer immunotherapy in in vivo models.},
  archive      = {J_NG},
  author       = {McCutcheon, Sean R. and Swartz, Adam M. and Brown, Michael C. and Barrera, Alejandro and McRoberts Amador, Christian and Siklenka, Keith and Humayun, Lucas and ter Weele, Maria A. and Isaacs, James M. and Reddy, Timothy E. and Allen, Andrew S. and Nair, Smita K. and Antonia, Scott J. and Gersbach, Charles A.},
  doi          = {10.1038/s41588-023-01554-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2211-2223},
  shortjournal = {Nat. Genet.},
  title        = {Transcriptional and epigenetic regulators of human CD8+ t cell function identified through orthogonal CRISPR screens},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dynamic regulatory elements in single-cell multimodal data
implicate key immune cell states enriched for autoimmune disease
heritability. <em>NG</em>, <em>55</em>(12), 2200–2210. (<a
href="https://doi.org/10.1038/s41588-023-01577-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In autoimmune diseases such as rheumatoid arthritis, the immune system attacks the body’s own cells. Developing a precise understanding of the cell states where noncoding autoimmune risk variants impart causal mechanisms is critical to developing curative therapies. Here, to identify noncoding regions with accessible chromatin that associate with cell-state-defining gene expression patterns, we leveraged multimodal single-nucleus RNA and assay for transposase-accessible chromatin (ATAC) sequencing data across 28,674 cells from the inflamed synovial tissue of 12 donors. Specifically, we used a multivariate Poisson model to predict peak accessibility from single-nucleus RNA sequencing principal components. For 14 autoimmune diseases, we discovered that cell-state-dependent (‘dynamic’) chromatin accessibility peaks in immune cell types were enriched for heritability, compared with cell-state-invariant (‘cs-invariant’) peaks. These dynamic peaks marked regulatory elements associated with T peripheral helper, regulatory T, dendritic and STAT1+CXCL10+ myeloid cell states. We argue that dynamic regulatory elements can help identify precise cell states enriched for disease-critical genetic variation. Analysis of single-nucleus RNA sequencing and single-nucleus assay for transposase-accessible chromatin with sequencing data derived from synovium of patients with rheumatoid arthritis identifies regions with dynamic accessibility that correlate with cell states. Dynamic peaks are more strongly enriched for autoimmune disease heritability.},
  archive      = {J_NG},
  author       = {Gupta, Anika and Weinand, Kathryn and Nathan, Aparna and Sakaue, Saori and Zhang, Martin Jinye and Donlin, Laura and Wei, Kevin and Price, Alkes L. and Amariuta, Tiffany and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-023-01577-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2200-2210},
  shortjournal = {Nat. Genet.},
  title        = {Dynamic regulatory elements in single-cell multimodal data implicate key immune cell states enriched for autoimmune disease heritability},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Circular extrachromosomal DNA promotes tumor heterogeneity
in high-risk medulloblastoma. <em>NG</em>, <em>55</em>(12), 2189–2199.
(<a href="https://doi.org/10.1038/s41588-023-01551-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circular extrachromosomal DNA (ecDNA) in patient tumors is an important driver of oncogenic gene expression, evolution of drug resistance and poor patient outcomes. Applying computational methods for the detection and reconstruction of ecDNA across a retrospective cohort of 481 medulloblastoma tumors from 465 patients, we identify circular ecDNA in 82 patients (18%). Patients with ecDNA-positive medulloblastoma were more than twice as likely to relapse and three times as likely to die within 5 years of diagnosis. A subset of tumors harbored multiple ecDNA lineages, each containing distinct amplified oncogenes. Multimodal sequencing, imaging and CRISPR inhibition experiments in medulloblastoma models reveal intratumoral heterogeneity of ecDNA copy number per cell and frequent putative ‘enhancer rewiring’ events on ecDNA. This study reveals the frequency and diversity of ecDNA in medulloblastoma, stratified into molecular subgroups, and suggests copy number heterogeneity and enhancer rewiring as oncogenic features of ecDNA. Circular extrachromosomal DNA in high-risk medulloblastoma contributes to tumor heterogeneity and associates with relapse and survival. Enhancer rewiring events involving known oncogenes are frequent events, affecting transcription and proliferation.},
  archive      = {J_NG},
  author       = {Chapman, Owen S. and Luebeck, Jens and Sridhar, Sunita and Wong, Ivy Tsz-Lo and Dixit, Deobrat and Wang, Shanqing and Prasad, Gino and Rajkumar, Utkrisht and Pagadala, Meghana S. and Larson, Jon D. and He, Britney Jiayu and Hung, King L. and Lange, Joshua T. and Dehkordi, Siavash R. and Chandran, Sahaana and Adam, Miriam and Morgan, Ling and Wani, Sameena and Tiwari, Ashutosh and Guccione, Caitlin and Lin, Yingxi and Dutta, Aditi and Lo, Yan Yuen and Juarez, Edwin and Robinson, James T. and Korshunov, Andrey and Michaels, John-Edward A. and Cho, Yoon-Jae and Malicki, Denise M. and Coufal, Nicole G. and Levy, Michael L. and Hobbs, Charlotte and Scheuermann, Richard H. and Crawford, John R. and Pomeroy, Scott L. and Rich, Jeremy N. and Zhang, Xinlian and Chang, Howard Y. and Dixon, Jesse R. and Bagchi, Anindya and Deshpande, Aniruddha J. and Carter, Hannah and Fraenkel, Ernest and Mischel, Paul S. and Wechsler-Reya, Robert J. and Bafna, Vineet and Mesirov, Jill P. and Chavez, Lukas},
  doi          = {10.1038/s41588-023-01551-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2189-2199},
  shortjournal = {Nat. Genet.},
  title        = {Circular extrachromosomal DNA promotes tumor heterogeneity in high-risk medulloblastoma},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiomic analysis of cervical squamous cell carcinoma
identifies cellular ecosystems with biological and clinical relevance.
<em>NG</em>, <em>55</em>(12), 2175–2188. (<a
href="https://doi.org/10.1038/s41588-023-01570-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cervical squamous cell carcinoma (CSCC) exhibits a limited response to immune-checkpoint blockade. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and spatial proteomics, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in CSCC. Three tumor states (epithelial-cytokeratin, epithelial-immune (Epi-Imm) and epithelial senescence), recapitulating different stages of squamous differentiation, showed distinct tumor immune microenvironments. Bidirectional interactions between epithelial-cytokeratin malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment through transforming growth factor β pathway signaling mediated by FABP5. In Epi-Imm tumors, malignant cells interact with natural killer and T cells through interferon signaling. Preliminary analysis of samples from a cervical cancer clinical trial ( NCT04516616 ) demonstrated neoadjuvant chemotherapy induces a state transition to Epi-Imm, which correlates with pathological complete remission following treatment with immune-checkpoint blockade. These findings deepen the understanding of cellular state diversity in CSCC. Multiomic analysis of cervical squamous cell carcinoma identifies distinct cellular ecosystems characterized by immune cell infiltration, heterotypic signaling and patient outcomes.},
  archive      = {J_NG},
  author       = {Fan, Junpeng and Lu, Funian and Qin, Tianyu and Peng, Wenju and Zhuang, Xucui and Li, Yinuo and Hou, Xin and Fang, Zixuan and Yang, Yunyi and Guo, Ensong and Yang, Bin and Li, Xi and Fu, Yu and Kang, Xiaoyan and Wu, Zimeng and Han, Lili and Mills, Gordon B. and Ma, Xiangyi and Li, Kezhen and Wu, Peng and Ma, Ding and Chen, Gang and Sun, Chaoyang},
  doi          = {10.1038/s41588-023-01570-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2175-2188},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic analysis of cervical squamous cell carcinoma identifies cellular ecosystems with biological and clinical relevance},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Noncoding mutations cause super-enhancer retargeting
resulting in protein synthesis dysregulation during b cell lymphoma
progression. <em>NG</em>, <em>55</em>(12), 2160–2174. (<a
href="https://doi.org/10.1038/s41588-023-01561-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome sequencing of longitudinal tumor pairs representing transformation of follicular lymphoma to high-grade B cell lymphoma with MYC and BCL2 rearrangements (double-hit lymphoma) identified coding and noncoding genomic alterations acquired during lymphoma progression. Many of these transformation-associated alterations recurrently and focally occur at topologically associating domain resident regulatory DNA elements, including H3K4me3 promoter marks located within H3K27ac super-enhancer clusters in B cell non-Hodgkin lymphoma. One region found to undergo recurrent alteration upon transformation overlaps a super-enhancer affecting the expression of the PAX5/ZCCHC7 gene pair. ZCCHC7 encodes a subunit of the Trf4/5-Air1/2-Mtr4 polyadenylation-like complex and demonstrated copy number gain, chromosomal translocation and enhancer retargeting-mediated transcriptional upregulation upon lymphoma transformation. Consequently, lymphoma cells demonstrate nucleolar dysregulation via altered noncoding 5.8S ribosomal RNA processing. We find that a noncoding mutation acquired during lymphoma progression affects noncoding rRNA processing, thereby rewiring protein synthesis leading to oncogenic changes in the lymphoma proteome. Whole-genome analysis of paired follicular lymphoma and double-hit lymphoma shows that lymphoma progression is accompanied by enhanced somatic mutations targeting super-enhancer-embedded promoters.},
  archive      = {J_NG},
  author       = {Leeman-Neill, Rebecca J. and Song, Dong and Bizarro, Jonathan and Wacheul, Ludivine and Rothschild, Gerson and Singh, Sameer and Yang, Yang and Sarode, Aditya Y. and Gollapalli, Kishore and Wu, Lijing and Zhang, Wanwei and Chen, Yiyun and Lauring, Max C. and Whisenant, D. Eric and Bhavsar, Shweta and Lim, Junghyun and Swerdlow, Steven H. and Bhagat, Govind and Zhao, Qian and Berchowitz, Luke E. and Lafontaine, Denis L. J. and Wang, Jiguang and Basu, Uttiya},
  doi          = {10.1038/s41588-023-01561-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2160-2174},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding mutations cause super-enhancer retargeting resulting in protein synthesis dysregulation during b cell lymphoma progression},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetics and epidemiology of mutational barcode-defined
clonal hematopoiesis. <em>NG</em>, <em>55</em>(12), 2149–2159. (<a
href="https://doi.org/10.1038/s41588-023-01555-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clonal hematopoiesis (CH) arises when a substantial proportion of mature blood cells is derived from a single hematopoietic stem cell lineage. Using whole-genome sequencing of 45,510 Icelandic and 130,709 UK Biobank participants combined with a mutational barcode method, we identified 16,306 people with CH. Prevalence approaches 50% in elderly participants. Smoking demonstrates a dosage-dependent impact on risk of CH. CH associates with several smoking-related diseases. Contrary to published claims, we find no evidence that CH is associated with cardiovascular disease. We provide evidence that CH is driven by genes that are commonly mutated in myeloid neoplasia and implicate several new driver genes. The presence and nature of a driver mutation alters the risk profile for hematological disorders. Nevertheless, most CH cases have no known driver mutations. A CH genome-wide association study identified 25 loci, including 19 not implicated previously in CH. Splicing, protein and expression quantitative trait loci were identified for CD164 and TCL1A. A barcode-based approach applied to UK Biobank and an Icelandic cohort identifies drivers of clonal hematopoiesis (CH) and finds associations between CH and multiple diseases. Genome-wide association analyses identify 25 loci associated with CH susceptibility.},
  archive      = {J_NG},
  author       = {Stacey, Simon N. and Zink, Florian and Halldorsson, Gisli H. and Stefansdottir, Lilja and Gudjonsson, Sigurjon A. and Einarsson, Gudmundur and Hjörleifsson, Grimur and Eiriksdottir, Thjodbjorg and Helgadottir, Anna and Björnsdottir, Gyda and Thorgeirsson, Thorgeir E. and Olafsdottir, Thorunn A. and Jonsdottir, Ingileif and Gretarsdottir, Solveig and Tragante, Vinicius and Magnusson, Magnus K. and Jonsson, Hakon and Gudmundsson, Julius and Olafsson, Sigurgeir and Holm, Hilma and Gudbjartsson, Daniel F. and Sulem, Patrick and Helgason, Agnar and Thorsteinsdottir, Unnur and Tryggvadottir, Laufey and Rafnar, Thorunn and Melsted, Pall and Ulfarsson, Magnus Ö. and Vidarsson, Brynjar and Thorleifsson, Gudmar and Stefansson, Kari},
  doi          = {10.1038/s41588-023-01555-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2149-2159},
  shortjournal = {Nat. Genet.},
  title        = {Genetics and epidemiology of mutational barcode-defined clonal hematopoiesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Most large structural variants in cancer genomes can be
detected without long reads. <em>NG</em>, <em>55</em>(12), 2139–2148.
(<a href="https://doi.org/10.1038/s41588-023-01540-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Short-read sequencing is the workhorse of cancer genomics yet is thought to miss many structural variants (SVs), particularly large chromosomal alterations. To characterize missing SVs in short-read whole genomes, we analyzed ‘loose ends’—local violations of mass balance between adjacent DNA segments. In the landscape of loose ends across 1,330 high-purity cancer whole genomes, most large (&amp;gt;10-kb) clonal SVs were fully resolved by short reads in the 87% of the human genome where copy number could be reliably measured. Some loose ends represent neotelomeres, which we propose as a hallmark of the alternative lengthening of telomeres phenotype. These pan-cancer findings were confirmed by long-molecule profiles of 38 breast cancer and melanoma cases. Our results indicate that aberrant homologous recombination is unlikely to drive the majority of large cancer SVs. Furthermore, analysis of mass balance in short-read whole genome data provides a surprisingly complete picture of cancer chromosomal structure. JaBbA v1 pinpoints the ‘loose ends’ of large (&amp;gt;10-kb) unmapped structural variants in short-read DNA sequencing, suggesting that about 90% of cancer chromosomal alterations outside centromeres are resolvable with short reads and that long reads will primarily improve calling of smaller somatic variants.},
  archive      = {J_NG},
  author       = {Choo, Zi-Ning and Behr, Julie M. and Deshpande, Aditya and Hadi, Kevin and Yao, Xiaotong and Tian, Huasong and Takai, Kaori and Zakusilo, George and Rosiene, Joel and Da Cruz Paula, Arnaud and Weigelt, Britta and Setton, Jeremy and Riaz, Nadeem and Powell, Simon N. and Busam, Klaus and Shoushtari, Alexander N. and Ariyan, Charlotte and Reis-Filho, Jorge and de Lange, Titia and Imieliński, Marcin},
  doi          = {10.1038/s41588-023-01540-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2139-2148},
  shortjournal = {Nat. Genet.},
  title        = {Most large structural variants in cancer genomes can be detected without long reads},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). East asian-specific and cross-ancestry genome-wide
meta-analyses provide mechanistic insights into peptic ulcer disease.
<em>NG</em>, <em>55</em>(12), 2129–2138. (<a
href="https://doi.org/10.1038/s41588-023-01569-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peptic ulcer disease (PUD) refers to acid-induced injury of the digestive tract, occurring mainly in the stomach (gastric ulcer (GU)) or duodenum (duodenal ulcer (DU)). In the present study, we conducted a large-scale, cross-ancestry meta-analysis of PUD combining genome-wide association studies with Japanese and European studies (52,032 cases and 905,344 controls), and discovered 25 new loci highly concordant across ancestries. An examination of GU and DU genetic architecture demonstrated that GUs shared the same risk loci as DUs, although with smaller genetic effect sizes and higher polygenicity than DUs, indicating higher heterogeneity of GUs. Helicobacter pylori (HP)-stratified analysis found an HP-related host genetic locus. Integrative analyses using bulk and single-cell transcriptome profiles highlighted the genetic factors of PUD being enriched in the highly expressed genes in stomach tissues, especially in somatostatin-producing D cells. Our results provide genetic evidence that gastrointestinal cell differentiations and hormone regulations are critical in PUD etiology. Cross-ancestry genome-wide association meta-analyses identify new risk loci for peptic ulcer diseases and provide evidence that gastrointestinal cell differentiation and hormone regulation contribute to their etiology.},
  archive      = {J_NG},
  author       = {He, Yunye and Koido, Masaru and Sutoh, Yoichi and Shi, Mingyang and Otsuka-Yamasaki, Yayoi and Munter, Hans Markus and Morisaki, Takayuki and Nagai, Akiko and Murakami, Yoshinori and Tanikawa, Chizu and Hachiya, Tsuyoshi and Matsuda, Koichi and Shimizu, Atsushi and Kamatani, Yoichiro},
  doi          = {10.1038/s41588-023-01569-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2129-2138},
  shortjournal = {Nat. Genet.},
  title        = {East asian-specific and cross-ancestry genome-wide meta-analyses provide mechanistic insights into peptic ulcer disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Isoform-level transcriptome-wide association uncovers
genetic risk mechanisms for neuropsychiatric disorders in the human
brain. <em>NG</em>, <em>55</em>(12), 2117–2128. (<a
href="https://doi.org/10.1038/s41588-023-01560-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods integrating genetics with transcriptomic reference panels prioritize risk genes and mechanisms at only a fraction of trait-associated genetic loci, due in part to an overreliance on total gene expression as a molecular outcome measure. This challenge is particularly relevant for the brain, in which extensive splicing generates multiple distinct transcript-isoforms per gene. Due to complex correlation structures, isoform-level modeling from cis-window variants requires methodological innovation. Here we introduce isoTWAS, a multivariate, stepwise framework integrating genetics, isoform-level expression and phenotypic associations. Compared to gene-level methods, isoTWAS improves both isoform and gene expression prediction, yielding more testable genes, and increased power for discovery of trait associations within genome-wide association study loci across 15 neuropsychiatric traits. We illustrate multiple isoTWAS associations undetectable at the gene-level, prioritizing isoforms of AKT3, CUL3 and HSPD1 in schizophrenia and PCLO with multiple disorders. Results highlight the importance of incorporating isoform-level resolution within integrative approaches to increase discovery of trait associations, especially for brain-relevant traits. A multivariate framework for isoform-resolution transcriptome-wide association studies enables modeling of a greater number of genes, with the benefit of identifying isoform-specific associations with psychiatric traits not observed at the gene level.},
  archive      = {J_NG},
  author       = {Bhattacharya, Arjun and Vo, Daniel D. and Jops, Connor and Kim, Minsoo and Wen, Cindy and Hervoso, Jonatan L. and Pasaniuc, Bogdan and Gandal, Michael J.},
  doi          = {10.1038/s41588-023-01560-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2117-2128},
  shortjournal = {Nat. Genet.},
  title        = {Isoform-level transcriptome-wide association uncovers genetic risk mechanisms for neuropsychiatric disorders in the human brain},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A global view of aging and alzheimer’s
pathogenesis-associated cell population dynamics and molecular
signatures in human and mouse brains. <em>NG</em>, <em>55</em>(12),
2104–2116. (<a
href="https://doi.org/10.1038/s41588-023-01572-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conventional methods fall short in unraveling the dynamics of rare cell types related to aging and diseases. Here we introduce EasySci, an advanced single-cell combinatorial indexing strategy for exploring age-dependent cellular dynamics in the mammalian brain. Profiling approximately 1.5 million single-cell transcriptomes and 400,000 chromatin accessibility profiles across diverse mouse brains, we identified over 300 cell subtypes, uncovering their molecular characteristics and spatial locations. This comprehensive view elucidates rare cell types expanded or depleted upon aging. We also investigated cell-type-specific responses to genetic alterations linked to Alzheimer’s disease, identifying associated rare cell types. Additionally, by profiling 118,240 human brain single-cell transcriptomes, we discerned cell- and region-specific transcriptomic changes tied to Alzheimer’s pathogenesis. In conclusion, this research offers a valuable resource for probing cell-type-specific dynamics in both normal and pathological aging. Single-cell transcriptomes and single-cell chromatin accessibility profiles generated using EasySci provide a global view of aging and Alzheimer’s pathogenesis-associated cell population dynamics in human and mouse brains.},
  archive      = {J_NG},
  author       = {Sziraki, Andras and Lu, Ziyu and Lee, Jasper and Banyai, Gabor and Anderson, Sonya and Abdulraouf, Abdulraouf and Metzner, Eli and Liao, Andrew and Banfelder, Jason and Epstein, Alexander and Schaefer, Chloe and Xu, Zihan and Zhang, Zehao and Gan, Li and Nelson, Peter T. and Zhou, Wei and Cao, Junyue},
  doi          = {10.1038/s41588-023-01572-y},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2104-2116},
  shortjournal = {Nat. Genet.},
  title        = {A global view of aging and alzheimer’s pathogenesis-associated cell population dynamics and molecular signatures in human and mouse brains},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multi-ancestry genome-wide association study of cannabis use
disorder yields insight into disease biology and public health
implications. <em>NG</em>, <em>55</em>(12), 2094–2103. (<a
href="https://doi.org/10.1038/s41588-023-01563-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As recreational use of cannabis is being decriminalized in many places and medical use widely sanctioned, there are growing concerns about increases in cannabis use disorder (CanUD), which is associated with numerous medical comorbidities. Here we performed a genome-wide association study of CanUD in the Million Veteran Program (MVP), followed by meta-analysis in 1,054,365 individuals (ncases = 64,314) from four broad ancestries designated by the reference panel used for assignment (European n = 886,025, African n = 123,208, admixed American n = 38,289 and East Asian n = 6,843). Population-specific methods were applied to calculate single nucleotide polymorphism-based heritability within each ancestry. Statistically significant single nucleotide polymorphism-based heritability for CanUD was observed in all but the smallest population (East Asian). We discovered genome-wide significant loci unique to each ancestry: 22 in European, 2 each in African and East Asian, and 1 in admixed American ancestries. A genetically informed causal relationship analysis indicated a possible effect of genetic liability for CanUD on lung cancer risk, suggesting potential unanticipated future medical and psychiatric public health consequences that require further study to disentangle from other known risk factors such as cigarette smoking. Multi-ancestry genome-wide association meta-analyses identify risk loci for cannabis use disorder. Genomic structural equation modeling and genetic correlation analyses show overlap with several other traits, including impulsivity and psychopathology.},
  archive      = {J_NG},
  author       = {Levey, Daniel F. and Galimberti, Marco and Deak, Joseph D. and Wendt, Frank R. and Bhattacharya, Arjun and Koller, Dora and Harrington, Kelly M. and Quaden, Rachel and Johnson, Emma C. and Gupta, Priya and Biradar, Mahantesh and Lam, Max and Cooke, Megan and Rajagopal, Veera M. and Empke, Stefany L. L. and Zhou, Hang and Nunez, Yaira Z. and Kranzler, Henry R. and Edenberg, Howard J. and Agrawal, Arpana and Smoller, Jordan W. and Lencz, Todd and Hougaard, David M. and Børglum, Anders D. and Demontis, Ditte and Gaziano, J. Michael and Gandal, Michael J. and Polimanti, Renato and Stein, Murray B. and Gelernter, Joel},
  doi          = {10.1038/s41588-023-01563-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2094-2103},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genome-wide association study of cannabis use disorder yields insight into disease biology and public health implications},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Phenotype integration improves power and preserves
specificity in biobank-based genetic studies of major depressive
disorder. <em>NG</em>, <em>55</em>(12), 2082–2093. (<a
href="https://doi.org/10.1038/s41588-023-01559-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biobanks often contain several phenotypes relevant to diseases such as major depressive disorder (MDD), with partly distinct genetic architectures. Researchers face complex tradeoffs between shallow (large sample size, low specificity/sensitivity) and deep (small sample size, high specificity/sensitivity) phenotypes, and the optimal choices are often unclear. Here we propose to integrate these phenotypes to combine the benefits of each. We use phenotype imputation to integrate information across hundreds of MDD-relevant phenotypes, which significantly increases genome-wide association study (GWAS) power and polygenic risk score (PRS) prediction accuracy of the deepest available MDD phenotype in UK Biobank, LifetimeMDD. We demonstrate that imputation preserves specificity in its genetic architecture using a novel PRS-based pleiotropy metric. We further find that integration via summary statistics also enhances GWAS power and PRS predictions, but can introduce nonspecific genetic effects depending on input. Our work provides a simple and scalable approach to improve genetic studies in large biobanks by integrating shallow and deep phenotypes. Phenotype imputation increases the effective sample size of major depressive disorder cases in UK Biobank, enhancing study power and polygenic risk score (PRS) accuracy. A new pleiotropy metric enables assessment of PRS specificity and comparison among different PRS models.},
  archive      = {J_NG},
  author       = {Dahl, Andrew and Thompson, Michael and An, Ulzee and Krebs, Morten and Appadurai, Vivek and Border, Richard and Bacanu, Silviu-Alin and Werge, Thomas and Flint, Jonathan and Schork, Andrew J. and Sankararaman, Sriram and Kendler, Kenneth S. and Cai, Na},
  doi          = {10.1038/s41588-023-01559-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2082-2093},
  shortjournal = {Nat. Genet.},
  title        = {Phenotype integration improves power and preserves specificity in biobank-based genetic studies of major depressive disorder},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Primate-specific ZNF808 is essential for pancreatic
development in humans. <em>NG</em>, <em>55</em>(12), 2075–2081. (<a
href="https://doi.org/10.1038/s41588-023-01565-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying genes linked to extreme phenotypes in humans has the potential to highlight biological processes not shared with all other mammals. Here, we report the identification of homozygous loss-of-function variants in the primate-specific gene ZNF808 as a cause of pancreatic agenesis. ZNF808 is a member of the KRAB zinc finger protein family, a large and rapidly evolving group of epigenetic silencers which target transposable elements. We show that loss of ZNF808 in vitro results in aberrant activation of regulatory potential contained in the primate-specific transposable elements it represses during early pancreas development. This leads to inappropriate specification of cell fate with induction of genes associated with liver identity. Our results highlight the essential role of ZNF808 in pancreatic development in humans and the contribution of primate-specific regions of the human genome to congenital developmental disease. Loss-of-function mutations in primate-specific ZNF808 cause pancreatic agenesis. Mechanistically, the loss of ZNF808 leads to the activation of the MER11 family of transposable elements in a regulatory capacity that ultimately induces a liver-specific program of gene expression during pancreatic differentiation.},
  archive      = {J_NG},
  author       = {De Franco, Elisa and Owens, Nick D. L. and Montaser, Hossam and Wakeling, Matthew N. and Saarimäki-Vire, Jonna and Triantou, Athina and Ibrahim, Hazem and Balboa, Diego and Caswell, Richard C. and Jennings, Rachel E. and Kvist, Jouni A. and Johnson, Matthew B. and Muralidharan, Sachin and Ellard, Sian and Wright, Caroline F. and Maddirevula, Sateesh and Alkuraya, Fowzan S. and Hanley, Neil A. and Flanagan, Sarah E. and Otonkoski, Timo and Hattersley, Andrew T. and Imbeault, Michael},
  doi          = {10.1038/s41588-023-01565-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2075-2081},
  shortjournal = {Nat. Genet.},
  title        = {Primate-specific ZNF808 is essential for pancreatic development in humans},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Characterizing prostate cancer risk through multi-ancestry
genome-wide discovery of 187 novel risk variants. <em>NG</em>,
<em>55</em>(12), 2065–2074. (<a
href="https://doi.org/10.1038/s41588-023-01534-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The transferability and clinical value of genetic risk scores (GRSs) across populations remain limited due to an imbalance in genetic studies across ancestrally diverse populations. Here we conducted a multi-ancestry genome-wide association study of 156,319 prostate cancer cases and 788,443 controls of European, African, Asian and Hispanic men, reflecting a 57% increase in the number of non-European cases over previous prostate cancer genome-wide association studies. We identified 187 novel risk variants for prostate cancer, increasing the total number of risk variants to 451. An externally replicated multi-ancestry GRS was associated with risk that ranged from 1.8 (per standard deviation) in African ancestry men to 2.2 in European ancestry men. The GRS was associated with a greater risk of aggressive versus non-aggressive disease in men of African ancestry (P = 0.03). Our study presents novel prostate cancer susceptibility loci and a GRS with effective risk stratification across ancestry groups. A multi-ancestry genome-wide association study of prostate cancer performed in 156,319 cases and 788,443 controls identifies 187 novel risk variants associated with the disease. Genetic risk scores associated with overall risk, and risk of aggressive disease in men of African ancestry.},
  archive      = {J_NG},
  author       = {Wang, Anqi and Shen, Jiayi and Rodriguez, Alex A. and Saunders, Edward J. and Chen, Fei and Janivara, Rohini and Darst, Burcu F. and Sheng, Xin and Xu, Yili and Chou, Alisha J. and Benlloch, Sara and Dadaev, Tokhir and Brook, Mark N. and Plym, Anna and Sahimi, Ali and Hoffman, Thomas J. and Takahashi, Atushi and Matsuda, Koichi and Momozawa, Yukihide and Fujita, Masashi and Laisk, Triin and Figuerêdo, Jéssica and Muir, Kenneth and Ito, Shuji and Liu, Xiaoxi and Uchio, Yuji and Kubo, Michiaki and Kamatani, Yoichiro and Lophatananon, Artitaya and Wan, Peggy and Andrews, Caroline and Lori, Adriana and Choudhury, Parichoy P. and Schleutker, Johanna and Tammela, Teuvo L. J. and Sipeky, Csilla and Auvinen, Anssi and Giles, Graham G. and Southey, Melissa C. and MacInnis, Robert J. and Cybulski, Cezary and Wokolorczyk, Dominika and Lubinski, Jan and Rentsch, Christopher T. and Cho, Kelly and Mcmahon, Benjamin H. and Neal, David E. and Donovan, Jenny L. and Hamdy, Freddie C. and Martin, Richard M. and Nordestgaard, Borge G. and Nielsen, Sune F. and Weischer, Maren and Bojesen, Stig E. and Røder, Andreas and Stroomberg, Hein V. and Batra, Jyotsna and Chambers, Suzanne and Horvath, Lisa and Clements, Judith A. and Tilly, Wayne and Risbridger, Gail P. and Gronberg, Henrik and Aly, Markus and Szulkin, Robert and Eklund, Martin and Nordstrom, Tobias and Pashayan, Nora and Dunning, Alison M. and Ghoussaini, Maya and Travis, Ruth C. and Key, Tim J. and Riboli, Elio and Park, Jong Y. and Sellers, Thomas A. and Lin, Hui-Yi and Albanes, Demetrius and Weinstein, Stephanie and Cook, Michael B. and Mucci, Lorelei A. and Giovannucci, Edward and Lindstrom, Sara and Kraft, Peter and Hunter, David J. and Penney, Kathryn L. and Turman, Constance and Tangen, Catherine M. and Goodman, Phyllis J. and Thompson, Ian M. and Hamilton, Robert J. and Fleshner, Neil E. and Finelli, Antonio and Parent, Marie-Élise and Stanford, Janet L. and Ostrander, Elaine A. and Koutros, Stella and Beane Freeman, Laura E. and Stampfer, Meir and Wolk, Alicja and Håkansson, Niclas and Andriole, Gerald L. and Hoover, Robert N. and Machiela, Mitchell J. and Sørensen, Karina Dalsgaard and Borre, Michael and Blot, William J. and Zheng, Wei and Yeboah, Edward D. and Mensah, James E. and Lu, Yong-Jie and Zhang, Hong-Wei and Feng, Ninghan and Mao, Xueying and Wu, Yudong and Zhao, Shan-Chao and Sun, Zan and Thibodeau, Stephen N. and McDonnell, Shannon K. and Schaid, Daniel J. and West, Catharine M. L. and Barnett, Gill and Maier, Christiane and Schnoeller, Thomas and Luedeke, Manuel and Kibel, Adam S. and Drake, Bettina F. and Cussenot, Olivier and Cancel-Tassin, Geraldine and Menegaux, Florence and Truong, Thérèse and Koudou, Yves Akoli and John, Esther M. and Grindedal, Eli Marie and Maehle, Lovise and Khaw, Kay-Tee and Ingles, Sue A. and Stern, Mariana C. and Vega, Ana and Gómez-Caamaño, Antonio and Fachal, Laura and Rosenstein, Barry S. and Kerns, Sarah L. and Ostrer, Harry and Teixeira, Manuel R. and Paulo, Paula and Brandão, Andreia and Watya, Stephen and Lubwama, Alexander and Bensen, Jeannette T. and Butler, Ebonee N. and Mohler, James L. and Taylor, Jack A. and Kogevinas, Manolis and Dierssen-Sotos, Trinidad and Castaño-Vinyals, Gemma and Cannon-Albright, Lisa and Teerlink, Craig C. and Huff, Chad D. and Pilie, Patrick and Yu, Yao and Bohlender, Ryan J. and Gu, Jian and Strom, Sara S. and Multigner, Luc and Blanchet, Pascal and Brureau, Laurent and Kaneva, Radka and Slavov, Chavdar and Mitev, Vanio and Leach, Robin J. and Brenner, Hermann and Chen, Xuechen and Holleczek, Bernd and Schöttker, Ben and Klein, Eric A. and Hsing, Ann W. and Kittles, Rick A. and Murphy, Adam B. and Logothetis, Christopher J. and Kim, Jeri and Neuhausen, Susan L. and Steele, Linda and Ding, Yuan Chun and Isaacs, William B. and Nemesure, Barbara and Hennis, Anselm J. M. and Carpten, John and Pandha, Hardev and Michael, Agnieszka and De Ruyck, Kim and De Meerleer, Gert and Ost, Piet and Xu, Jianfeng and Razack, Azad and Lim, Jasmine and Teo, Soo-Hwang and Newcomb, Lisa F. and Lin, Daniel W. and Fowke, Jay H. and Neslund-Dudas, Christine M. and Rybicki, Benjamin A. and Gamulin, Marija and Lessel, Davor and Kulis, Tomislav and Usmani, Nawaid and Abraham, Aswin and Singhal, Sandeep and Parliament, Matthew and Claessens, Frank and Joniau, Steven and Van den Broeck, Thomas and Gago-Dominguez, Manuela and Castelao, Jose Esteban and Martinez, Maria Elena and Larkin, Samantha and Townsend, Paul A. and Aukim-Hastie, Claire and Bush, William S. and Aldrich, Melinda C. and Crawford, Dana C. and Srivastava, Shiv and Cullen, Jennifer and Petrovics, Gyorgy and Casey, Graham and Wang, Ying and Tettey, Yao and Lachance, Joseph and Tang, Wei and Biritwum, Richard B. and Adjei, Andrew A. and Tay, Evelyn and Truelove, Ann and Niwa, Shelley and Yamoah, Kosj and Govindasami, Koveela and Chokkalingam, Anand P. and Keaton, Jacob M. and Hellwege, Jacklyn N. and Clark, Peter E. and Jalloh, Mohamed and Gueye, Serigne M. and Niang, Lamine and Ogunbiyi, Olufemi and Shittu, Olayiwola and Amodu, Olukemi and Adebiyi, Akindele O. and Aisuodionoe-Shadrach, Oseremen I. and Ajibola, Hafees O. and Jamda, Mustapha A. and Oluwole, Olabode P. and Nwegbu, Maxwell and Adusei, Ben and Mante, Sunny and Darkwa-Abrahams, Afua and Diop, Halimatou and Gundell, Susan M. and Roobol, Monique J. and Jenster, Guido and van Schaik, Ron H. N. and Hu, Jennifer J. and Sanderson, Maureen and Kachuri, Linda and Varma, Rohit and McKean-Cowdin, Roberta and Torres, Mina and Preuss, Michael H. and Loos, Ruth J. F. and Zawistowski, Matthew and Zöllner, Sebastian and Lu, Zeyun and Van Den Eeden, Stephen K. and Easton, Douglas F. and Ambs, Stefan and Edwards, Todd L. and Mägi, Reedik and Rebbeck, Timothy R. and Fritsche, Lars and Chanock, Stephen J. and Berndt, Sonja I. and Wiklund, Fredrik and Nakagawa, Hidewaki and Witte, John S. and Gaziano, J. Michael and Justice, Amy C. and Mancuso, Nick and Terao, Chikashi and Eeles, Rosalind A. and Kote-Jarai, Zsofia and Madduri, Ravi K. and Conti, David V. and Haiman, Christopher A.},
  doi          = {10.1038/s41588-023-01534-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2065-2074},
  shortjournal = {Nat. Genet.},
  title        = {Characterizing prostate cancer risk through multi-ancestry genome-wide discovery of 187 novel risk variants},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Benchmarking of deep neural networks for predicting personal
gene expression from DNA sequence highlights shortcomings. <em>NG</em>,
<em>55</em>(12), 2060–2064. (<a
href="https://doi.org/10.1038/s41588-023-01524-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning methods have recently become the state of the art in a variety of regulatory genomic tasks1–6, including the prediction of gene expression from genomic DNA. As such, these methods promise to serve as important tools in interpreting the full spectrum of genetic variation observed in personal genomes. Previous evaluation strategies have assessed their predictions of gene expression across genomic regions; however, systematic benchmarking is lacking to assess their predictions across individuals, which would directly evaluate their utility as personal DNA interpreters. We used paired whole genome sequencing and gene expression from 839 individuals in the ROSMAP study7 to evaluate the ability of current methods to predict gene expression variation across individuals at varied loci. Our approach identifies a limitation of current methods to correctly predict the direction of variant effects. We show that this limitation stems from insufficiently learned sequence motif grammar and suggest new model training strategies to improve performance. Neural networks are a common machine learning architecture for predicting phenotype from genomic sequence. This analysis finds that they err in calling the variant direction of effect, with important implications for personalized predictions.},
  archive      = {J_NG},
  author       = {Sasse, Alexander and Ng, Bernard and Spiro, Anna E. and Tasaki, Shinya and Bennett, David A. and Gaiteri, Christopher and De Jager, Philip L. and Chikina, Maria and Mostafavi, Sara},
  doi          = {10.1038/s41588-023-01524-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2060-2064},
  shortjournal = {Nat. Genet.},
  title        = {Benchmarking of deep neural networks for predicting personal gene expression from DNA sequence highlights shortcomings},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Personal transcriptome variation is poorly explained by
current genomic deep learning models. <em>NG</em>, <em>55</em>(12),
2056–2059. (<a
href="https://doi.org/10.1038/s41588-023-01574-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic deep learning models can predict genome-wide epigenetic features and gene expression levels directly from DNA sequence. While current models perform well at predicting gene expression levels across genes in different cell types from the reference genome, their ability to explain expression variation between individuals due to cis-regulatory genetic variants remains largely unexplored. Here, we evaluate four state-of-the-art models on paired personal genome and transcriptome data and find limited performance when explaining variation in expression across individuals. In addition, models often fail to predict the correct direction of effect of cis-regulatory genetic variation on expression. A test of four genomic sequence-to-expression deep learning models (Enformer, Basenji2, ExPecto, Xpresso) finds that they often fail to predict the correct direction of effect of cis-regulatory genetic variation on gene expression.},
  archive      = {J_NG},
  author       = {Huang, Connie and Shuai, Richard W. and Baokar, Parth and Chung, Ryan and Rastogi, Ruchir and Kathail, Pooja and Ioannidis, Nilah M.},
  doi          = {10.1038/s41588-023-01574-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2056-2059},
  shortjournal = {Nat. Genet.},
  title        = {Personal transcriptome variation is poorly explained by current genomic deep learning models},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reply to: Revisiting the use of structural similarity index
in hi-c. <em>NG</em>, <em>55</em>(12), 2053–2055. (<a
href="https://doi.org/10.1038/s41588-023-01595-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Ing-Simmons, Elizabeth and Machnik, Nick and Vaquerizas, Juan M.},
  doi          = {10.1038/s41588-023-01595-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2053-2055},
  shortjournal = {Nat. Genet.},
  title        = {Reply to: Revisiting the use of structural similarity index in hi-C},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Revisiting the use of structural similarity index in
hi-c. <em>NG</em>, <em>55</em>(12), 2049–2052. (<a
href="https://doi.org/10.1038/s41588-023-01594-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lee, Hanjun and Blumberg, Bruce and Lawrence, Michael S. and Shioda, Toshihiro},
  doi          = {10.1038/s41588-023-01594-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2049-2052},
  shortjournal = {Nat. Genet.},
  title        = {Revisiting the use of structural similarity index in hi-C},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Complex synthetic lethality in cancer. <em>NG</em>,
<em>55</em>(12), 2039–2048. (<a
href="https://doi.org/10.1038/s41588-023-01557-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The concept of synthetic lethality has been widely applied to identify therapeutic targets in cancer, with varying degrees of success. The standard approach normally involves identifying genetic interactions between two genes, a driver and a target. In reality, however, most cancer synthetic lethal effects are likely complex and also polygenic, being influenced by the environment in addition to involving contributions from multiple genes. By acknowledging and delineating this complexity, we describe in this article how the success rate in cancer drug discovery and development could be improved. This Perspective article explores complex synthetic lethal relationships in cancer, which can involve several partners. Understanding this complexity presents challenges and opportunities for the development of therapeutics that target these interactions.},
  archive      = {J_NG},
  author       = {Ryan, Colm J. and Devakumar, Lovely Paul Solomon and Pettitt, Stephen J. and Lord, Christopher J.},
  doi          = {10.1038/s41588-023-01557-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2039-2048},
  shortjournal = {Nat. Genet.},
  title        = {Complex synthetic lethality in cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). M6A methylation of super-enhancer RNAs facilitates chromatin
accessibility. <em>NG</em>, <em>55</em>(12), 2037–2038. (<a
href="https://doi.org/10.1038/s41588-023-01567-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Previous studies reported an effect of N6-methyladenosine (m6A) of super-enhancer RNAs (seRNAs) on chromatin accessibility and gene transcription. We investigated seRNA m6A levels in pancreatic ductal adenocarcinoma (PDAC) and found that aberrantly increased m6A methylation promoted local chromatin accessibility, resulting in increased transcription of oncogenes acting in PDAC progression.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01567-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2037-2038},
  shortjournal = {Nat. Genet.},
  title        = {M6A methylation of super-enhancer RNAs facilitates chromatin accessibility},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integration of genetic data with dynamic gene regulatory
elements identifies autoimmune effector cell states. <em>NG</em>,
<em>55</em>(12), 2035–2036. (<a
href="https://doi.org/10.1038/s41588-023-01578-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combined analysis of genome-wide association studies and epigenetic data has identified certain immune cell types as drivers of autoimmune disease, but current methods have not been able to pinpoint key effector immune cell states. Using single-cell data from inflammatory tissues, we identified effector cell states embedded within inflammatory tissues — including T peripheral helper cells and tissue regulatory T cells — that capture disproportionate disease heritability.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01578-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2035-2036},
  shortjournal = {Nat. Genet.},
  title        = {Integration of genetic data with dynamic gene regulatory elements identifies autoimmune effector cell states},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A biology-aware mutation rate model for human germline.
<em>NG</em>, <em>55</em>(12), 2033–2034. (<a
href="https://doi.org/10.1038/s41588-023-01564-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a model to predict the chance of each possible de novo mutation in the human genome informed by recent insights into determinants of mutagenesis. Predictions were applied to refine demographic models, identify constrained genes, and uncover mutagenic effects of polymerase III transcription and transcription factor binding in testis.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01564-y},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2033-2034},
  shortjournal = {Nat. Genet.},
  title        = {A biology-aware mutation rate model for human germline},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A mass balance principle for finding loose ends in cancer
genomes. <em>NG</em>, <em>55</em>(12), 2031–2032. (<a
href="https://doi.org/10.1038/s41588-023-01539-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an analysis that shows that although nearly half of the human genome comprises repetitive sequences, recombination between homologous repeats has a minor role in cancer chromosomal evolution.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01539-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2031-2032},
  shortjournal = {Nat. Genet.},
  title        = {A mass balance principle for finding loose ends in cancer genomes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Is clonal hematopoiesis “mostly harmless”? <em>NG</em>,
<em>55</em>(12), 2029–2030. (<a
href="https://doi.org/10.1038/s41588-023-01556-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome sequencing data of individuals from the UK Biobank and Iceland and a somatic mutation barcoding strategy enabled detection of clonal hematopoiesis at scale. This comprehensive study provides insights into the epidemiology, somatic and germline genetics, and disease associations of clonal hematopoiesis.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01556-y},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2029-2030},
  shortjournal = {Nat. Genet.},
  title        = {Is clonal hematopoiesis ‘mostly harmless’?},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Aging- and alzheimer’s disease-related rare cell type
dynamics in the mammalian brain unveiled. <em>NG</em>, <em>55</em>(12),
2027–2028. (<a
href="https://doi.org/10.1038/s41588-023-01573-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {EasySci, a scalable single-cell profiling technique, uncovered over 300 mammalian brain cell states, revealing molecular features and dynamics of rare cell states linked to aging and Alzheimer’s disease. This work offers insights into cell states that expand (rare astrocytes and vascular leptomeningeal cells in the olfactory bulb, reactive microglia, and oligodendrocytes) or are depleted (neuronal progenitors, neuroblasts and committed oligodendrocyte precursors) during normal and pathological aging.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01573-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2027-2028},
  shortjournal = {Nat. Genet.},
  title        = {Aging- and alzheimer’s disease-related rare cell type dynamics in the mammalian brain unveiled},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Taming the HLA for single-cell genomics. <em>NG</em>,
<em>55</em>(12), 2025–2026. (<a
href="https://doi.org/10.1038/s41588-023-01590-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Variants in the HLA region on chromosome 6 are strongly associated with many immune-related diseases. A method to construct personalized HLA genomes from single-cell RNA sequencing data, coupled with single-cell HLA expression quantitative trait loci modeling, identifies how genetic variants influence HLA gene expression across cell states.},
  archive      = {J_NG},
  author       = {Kelly, Jennifer A. and Tessneer, Kandice L. and Gaffney, Patrick M.},
  doi          = {10.1038/s41588-023-01590-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2025-2026},
  shortjournal = {Nat. Genet.},
  title        = {Taming the HLA for single-cell genomics},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). New ingredients for old recipes. <em>NG</em>,
<em>55</em>(12), 2023–2024. (<a
href="https://doi.org/10.1038/s41588-023-01591-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The pancreas is an essential organ present in all vertebrates, and human pancreatic agenesis is an extremely rare disorder of largely unknown genetic determinants. A study now demonstrates that a primate-specific regulatory network controlled by the KRAB zinc-finger protein ZNF808 is essential for pancreas development.},
  archive      = {J_NG},
  author       = {Rosspopoff, Olga and Martins, Filipe and Trono, Didier},
  doi          = {10.1038/s41588-023-01591-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2023-2024},
  shortjournal = {Nat. Genet.},
  title        = {New ingredients for old recipes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Current approaches to genomic deep learning struggle to
fully capture human genetic variation. <em>NG</em>, <em>55</em>(12),
2021–2022. (<a
href="https://doi.org/10.1038/s41588-023-01517-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning shows promise for predicting gene expression levels from DNA sequences. However, recent studies show that current state-of-the-art models struggle to accurately characterize expression variation from personal genomes, limiting their usefulness in personalized medicine.},
  archive      = {J_NG},
  author       = {Tang, Ziqi and Toneyan, Shushan and Koo, Peter K.},
  doi          = {10.1038/s41588-023-01517-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2021-2022},
  shortjournal = {Nat. Genet.},
  title        = {Current approaches to genomic deep learning struggle to fully capture human genetic variation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Immunotherapy for ankylosing spondylitis. <em>NG</em>,
<em>55</em>(12), 2020. (<a
href="https://doi.org/10.1038/s41588-023-01612-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01612-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2020},
  shortjournal = {Nat. Genet.},
  title        = {Immunotherapy for ankylosing spondylitis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Applying compressed perturb-seq to genetic screens.
<em>NG</em>, <em>55</em>(12), 2020. (<a
href="https://doi.org/10.1038/s41588-023-01613-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01613-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2020},
  shortjournal = {Nat. Genet.},
  title        = {Applying compressed perturb-seq to genetic screens},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Epigenetic profiling of cancers using liquid biopsies.
<em>NG</em>, <em>55</em>(12), 2019. (<a
href="https://doi.org/10.1038/s41588-023-01614-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01614-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2019},
  shortjournal = {Nat. Genet.},
  title        = {Epigenetic profiling of cancers using liquid biopsies},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Epigenetic memory in 3D. <em>NG</em>, <em>55</em>(12),
2019. (<a href="https://doi.org/10.1038/s41588-023-01615-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-023-01615-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2019},
  shortjournal = {Nat. Genet.},
  title        = {Epigenetic memory in 3D},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A single-nucleus pan-cancer epigenetic and transcriptomic
landscape. <em>NG</em>, <em>55</em>(12), 2019. (<a
href="https://doi.org/10.1038/s41588-023-01616-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01616-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2019},
  shortjournal = {Nat. Genet.},
  title        = {A single-nucleus pan-cancer epigenetic and transcriptomic landscape},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Rewriting the mammalian genome, one locus at a time.
<em>NG</em>, <em>55</em>(12), 2018. (<a
href="https://doi.org/10.1038/s41588-023-01617-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-023-01617-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2018},
  shortjournal = {Nat. Genet.},
  title        = {Rewriting the mammalian genome, one locus at a time},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). EP400/TIP60 compensates for SWI/SNF loss. <em>NG</em>,
<em>55</em>(12), 2018. (<a
href="https://doi.org/10.1038/s41588-023-01618-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01618-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2018},
  shortjournal = {Nat. Genet.},
  title        = {EP400/TIP60 compensates for SWI/SNF loss},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). The myeloma microenvironment and response to immunotherapy.
<em>NG</em>, <em>55</em>(12), 2018. (<a
href="https://doi.org/10.1038/s41588-023-01619-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01619-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2018},
  shortjournal = {Nat. Genet.},
  title        = {The myeloma microenvironment and response to immunotherapy},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Jeanette erdmann (1965–2023). <em>NG</em>, <em>55</em>(12),
2016–2017. (<a
href="https://doi.org/10.1038/s41588-023-01599-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Aherrahrou, Redouane and Eschenhagen, Thomas and König, Inke R. and Samani, Nilesh J. and Schunkert, Heribert and Aherrahrou, Zouhair},
  doi          = {10.1038/s41588-023-01599-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2016-2017},
  shortjournal = {Nat. Genet.},
  title        = {Jeanette erdmann (1965–2023)},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). My bantu vision of genomic medicine in africa. <em>NG</em>,
<em>55</em>(12), 2013–2015. (<a
href="https://doi.org/10.1038/s41588-023-01566-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The preparedness of Africa for genomic medicine remains a matter of debate because this question is always evaluated from a technological standpoint. Yet the resilience of African researchers and the cultural values of equity and fairness are important assets to be considered in planning for the future of genomic medicine in Africa.},
  archive      = {J_NG},
  author       = {Lumaka, Aimé},
  doi          = {10.1038/s41588-023-01566-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2013-2015},
  shortjournal = {Nat. Genet.},
  title        = {My bantu vision of genomic medicine in africa},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Age-associated transcriptional stress due to accelerated
elongation and increased stalling of RNAPII. <em>NG</em>,
<em>55</em>(12), 2011–2012. (<a
href="https://doi.org/10.1038/s41588-023-01601-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Papadakis, Antonios and Gyenis, Akos and Pothof, Joris and H. J. Hoeijmakers, Jan and Papantonis, Argyris and Beyer, Andreas},
  doi          = {10.1038/s41588-023-01601-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {2011-2012},
  shortjournal = {Nat. Genet.},
  title        = {Age-associated transcriptional stress due to accelerated elongation and increased stalling of RNAPII},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Exome sequencing identifies breast
cancer susceptibility genes and defines the contribution of coding
variants to breast cancer risk. <em>NG</em>, <em>55</em>(11), 2009. (<a
href="https://doi.org/10.1038/s41588-023-01549-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Wilcox, Naomi and Dumont, Martine and González-Neira, Anna and Carvalho, Sara and Joly Beauparlant, Charles and Crotti, Marco and Luccarini, Craig and Soucy, Penny and Dubois, Stéphane and Nuñez-Torres, Rocio and Pita, Guillermo and Gardner, Eugene J. and Dennis, Joe and Alonso, M. Rosario and Álvarez, Nuria and Baynes, Caroline and Collin-Deschesnes, Annie Claude and Desjardins, Sylvie and Becher, Heiko and Behrens, Sabine and Bolla, Manjeet K. and Castelao, Jose E. and Chang-Claude, Jenny and Cornelissen, Sten and Dörk, Thilo and Engel, Christoph and Gago-Dominguez, Manuela and Guénel, Pascal and Hadjisavvas, Andreas and Hahnen, Eric and Hartman, Mikael and Herráez, Belén and Jung, Audrey and Keeman, Renske and Kiechle, Marion and Li, Jingmei and Loizidou, Maria A. and Lush, Michael and Michailidou, Kyriaki and Panayiotidis, Mihalis I. and Sim, Xueling and Teo, Soo Hwang and Tyrer, Jonathan P. and van der Kolk, Lizet E. and Wahlström, Cecilia and Wang, Qin and Perry, John R. B. and Benitez, Javier and Schmidt, Marjanka K. and Schmutzler, Rita K. and Pharoah, Paul D. P. and Droit, Arnaud and Dunning, Alison M. and Kvist, Anders and Devilee, Peter and Easton, Douglas F. and Simard, Jacques},
  doi          = {10.1038/s41588-023-01549-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {2009},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Exome sequencing identifies breast cancer susceptibility genes and defines the contribution of coding variants to breast cancer risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precise identification of cell states altered in disease
using healthy single-cell references. <em>NG</em>, <em>55</em>(11),
1998–2008. (<a
href="https://doi.org/10.1038/s41588-023-01523-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Joint analysis of single-cell genomics data from diseased tissues and a healthy reference can reveal altered cell states. We investigate whether integrated collections of data from healthy individuals (cell atlases) are suitable references for disease-state identification and whether matched control samples are needed to minimize false discoveries. We demonstrate that using a reference atlas for latent space learning followed by differential analysis against matched controls leads to improved identification of disease-associated cells, especially with multiple perturbed cell types. Additionally, when an atlas is available, reducing control sample numbers does not increase false discovery rates. Jointly analyzing data from a COVID-19 cohort and a blood cell atlas, we improve detection of infection-related cell states linked to distinct clinical severities. Similarly, we studied disease states in pulmonary fibrosis using a healthy lung atlas, characterizing two distinct aberrant basal states. Our analysis provides guidelines for designing disease cohort studies and optimizing cell atlas use. In single-cell studies, combining healthy reference atlases and designed control datasets allows more precise identification of disease-associated cell states.},
  archive      = {J_NG},
  author       = {Dann, Emma and Cujba, Ana-Maria and Oliver, Amanda J. and Meyer, Kerstin B. and Teichmann, Sarah A. and Marioni, John C.},
  doi          = {10.1038/s41588-023-01523-7},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1998-2008},
  shortjournal = {Nat. Genet.},
  title        = {Precise identification of cell states altered in disease using healthy single-cell references},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Regulatory controls of duplicated gene expression during
fiber development in allotetraploid cotton. <em>NG</em>,
<em>55</em>(11), 1987–1997. (<a
href="https://doi.org/10.1038/s41588-023-01530-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyploidy complicates transcriptional regulation and increases phenotypic diversity in organisms. The dynamics of genetic regulation of gene expression between coresident subgenomes in polyploids remains to be understood. Here we document the genetic regulation of fiber development in allotetraploid cotton Gossypium hirsutum by sequencing 376 genomes and 2,215 time-series transcriptomes. We characterize 1,258 genes comprising 36 genetic modules that control staged fiber development and uncover genetic components governing their partitioned expression relative to subgenomic duplicated genes (homoeologs). Only about 30% of fiber quality-related homoeologs show phenotypically favorable allele aggregation in cultivars, highlighting the potential for subgenome additivity in fiber improvement. We envision a genome-enabled breeding strategy, with particular attention to 48 favorable alleles related to fiber phenotypes that have been subjected to purifying selection during domestication. Our work delineates the dynamics of gene regulation during fiber development and highlights the potential of subgenomic coordination underpinning phenotypes in polyploid plants. Genome and transcriptome analyses of 376 Gossypium hirsutum accessions uncover the regulation of gene expression during fiber development in allotetraploid cotton and highlight the potential for fiber quality improvement.},
  archive      = {J_NG},
  author       = {You, Jiaqi and Liu, Zhenping and Qi, Zhengyang and Ma, Yizan and Sun, Mengling and Su, Ling and Niu, Hao and Peng, Yabing and Luo, Xuanxuan and Zhu, Mengmeng and Huang, Yuefan and Chang, Xing and Hu, Xiubao and Zhang, Yuqi and Pi, Ruizhen and Liu, Yuqi and Meng, Qingying and Li, Jianying and Zhang, Qinghua and Zhu, Longfu and Lin, Zhongxu and Min, Ling and Yuan, Daojun and Grover, Corrinne E. and Fang, David D. and Lindsey, Keith and Wendel, Jonathan F. and Tu, Lili and Zhang, Xianlong and Wang, Maojun},
  doi          = {10.1038/s41588-023-01530-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1987-1997},
  shortjournal = {Nat. Genet.},
  title        = {Regulatory controls of duplicated gene expression during fiber development in allotetraploid cotton},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chromosome-level genomes of three key allium crops and their
trait evolution. <em>NG</em>, <em>55</em>(11), 1976–1986. (<a
href="https://doi.org/10.1038/s41588-023-01546-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Allium crop breeding remains severely hindered due to the lack of high-quality reference genomes. Here we report high-quality chromosome-level genome assemblies for three key Allium crops (Welsh onion, garlic and onion), which are 11.17 Gb, 15.52 Gb and 15.78 Gb in size with the highest recorded contig N50 of 507.27 Mb, 109.82 Mb and 81.66 Mb, respectively. Beyond revealing the genome evolutionary process of Allium species, our pathogen infection experiments and comparative metabolomic and genomic analyses showed that genes encoding enzymes involved in the metabolic pathway of Allium-specific flavor compounds may have evolved from an ancient uncharacterized plant defense system widely existing in many plant lineages but extensively boosted in alliums. Using in situ hybridization and spatial RNA sequencing, we obtained an overview of cell-type categorization and gene expression changes associated with spongy mesophyll cell expansion during onion bulb formation, thus indicating the functional roles of bulb formation genes. Chromosome-level genome assemblies of three Allium crops (onion, garlic and Welsh onion) and spatial RNA sequencing provide insights into Allium trait evolution and gene expression patterns during onion bulb formation.},
  archive      = {J_NG},
  author       = {Hao, Fei and Liu, Xue and Zhou, Botong and Tian, Zunzhe and Zhou, Lina and Zong, Hang and Qi, Jiyan and He, Juan and Zhang, Yongting and Zeng, Peng and Li, Qiong and Wang, Kai and Xia, Keke and Guo, Xing and Li, Li and Shao, Wenwen and Zhang, Bohan and Li, Shengkang and Yang, Haifeng and Hui, Linchong and Chen, Wei and Peng, Lixin and Liu, Feipeng and Rong, Zi-Qiang and Peng, Yingmei and Zhu, Wenbo and McCallum, John A. and Li, Zhen and Xu, Xun and Yang, Hui and Macknight, Richard C. and Wang, Wen and Cai, Jing},
  doi          = {10.1038/s41588-023-01546-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1976-1986},
  shortjournal = {Nat. Genet.},
  title        = {Chromosome-level genomes of three key allium crops and their trait evolution},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pangenome analysis provides insight into the evolution of
the orange subfamily and a key gene for citric acid accumulation in
citrus fruits. <em>NG</em>, <em>55</em>(11), 1964–1975. (<a
href="https://doi.org/10.1038/s41588-023-01516-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The orange subfamily (Aurantioideae) contains several Citrus species cultivated worldwide, such as sweet orange and lemon. The origin of Citrus species has long been debated and less is known about the Aurantioideae. Here, we compiled the genome sequences of 314 accessions, de novo assembled the genomes of 12 species and constructed a graph-based pangenome for Aurantioideae. Our analysis indicates that the ancient Indian Plate is the ancestral area for Citrus-related genera and that South Central China is the primary center of origin of the Citrus genus. We found substantial variations in the sequence and expression of the PH4 gene in Citrus relative to Citrus-related genera. Gene editing and biochemical experiments demonstrate a central role for PH4 in the accumulation of citric acid in citrus fruits. This study provides insights into the origin and evolution of the orange subfamily and a regulatory mechanism underpinning the evolution of fruit taste. De novo genome assemblies of 12 species from Aurantioideae and pangenome analyses provide insights into the origin and evolution of the orange subfamily and the genetic basis of flavor in citrus fruits.},
  archive      = {J_NG},
  author       = {Huang, Yue and He, Jiaxian and Xu, Yuantao and Zheng, Weikang and Wang, Shaohua and Chen, Peng and Zeng, Bin and Yang, Shuizhi and Jiang, Xiaolin and Liu, Zishuang and Wang, Lun and Wang, Xia and Liu, Shengjun and Lu, Zhihao and Liu, Ziang and Yu, Huiwen and Yue, Jianqiang and Gao, Junyan and Zhou, Xianyan and Long, Chunrui and Zeng, Xiuli and Guo, Yong-Jie and Zhang, Wen-Fu and Xie, Zongzhou and Li, Chunlong and Ma, Zhaocheng and Jiao, Wenbiao and Zhang, Fei and Larkin, Robert M. and Krueger, Robert R. and Smith, Malcolm W. and Ming, Ray and Deng, Xiuxin and Xu, Qiang},
  doi          = {10.1038/s41588-023-01516-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1964-1975},
  shortjournal = {Nat. Genet.},
  title        = {Pangenome analysis provides insight into the evolution of the orange subfamily and a key gene for citric acid accumulation in citrus fruits},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-haplotype comparative genomics provides insights into
lineage-specific structural variation during cat evolution. <em>NG</em>,
<em>55</em>(11), 1953–1963. (<a
href="https://doi.org/10.1038/s41588-023-01548-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The role of structurally dynamic genomic regions in speciation is poorly understood due to challenges inherent in diploid genome assembly. Here we reconstructed the evolutionary dynamics of structural variation in five cat species by phasing the genomes of three interspecies F1 hybrids to generate near-gapless single-haplotype assemblies. We discerned that cat genomes have a paucity of segmental duplications relative to great apes, explaining their remarkable karyotypic stability. X chromosomes were hotspots of structural variation, including enrichment with inversions in a large recombination desert with characteristics of a supergene. The X-linked macrosatellite DXZ4 evolves more rapidly than 99.5% of the genome clarifying its role in felid hybrid incompatibility. Resolved sensory gene repertoires revealed functional copy number changes associated with ecomorphological adaptations, sociality and domestication. This study highlights the value of gapless genomes to reveal structural mechanisms underpinning karyotypic evolution, reproductive isolation and ecological niche adaptation. Single-haplotype genome assemblies from five cat species shed light on the dynamics of structural variations during felid radiation and resolve sensory gene repertoires associated with adaptation and domestication.},
  archive      = {J_NG},
  author       = {Bredemeyer, Kevin R. and Hillier, LaDeana and Harris, Andrew J. and Hughes, Graham M. and Foley, Nicole M. and Lawless, Colleen and Carroll, Rachel A. and Storer, Jessica M. and Batzer, Mark A. and Rice, Edward S. and Davis, Brian W. and Raudsepp, Terje and O’Brien, Stephen J. and Lyons, Leslie A. and Warren, Wesley C. and Murphy, William J.},
  doi          = {10.1038/s41588-023-01548-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1953-1963},
  shortjournal = {Nat. Genet.},
  title        = {Single-haplotype comparative genomics provides insights into lineage-specific structural variation during cat evolution},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cellular plasticity of the bone marrow niche promotes
hematopoietic stem cell regeneration. <em>NG</em>, <em>55</em>(11),
1941–1952. (<a
href="https://doi.org/10.1038/s41588-023-01528-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hematopoietic stem cells (HSCs) regenerate after myeloablation, a procedure that adversely disrupts the bone marrow and drives leptin receptor-expressing cells, a key niche component, to differentiate extensively into adipocytes. Regeneration of the bone marrow niche is associated with the resolution of adipocytes, but the mechanisms remain poorly understood. Using Plin1-creER knock-in mice, we followed the fate of adipocytes in the regenerating niche in vivo. We found that bone marrow adipocytes were highly dynamic and dedifferentiated to leptin receptor-expressing cells during regeneration after myeloablation. Bone marrow adipocytes could give rise to osteolineage cells after skeletal injury. The cellular fate of steady-state bone marrow adipocytes was also plastic. Deletion of adipose triglyceride lipase (Atgl) from bone marrow stromal cells, including adipocytes, obstructed adipocyte dedifferentiation and led to severely compromised regeneration of HSCs as well as impaired B lymphopoiesis after myeloablation, but not in the steady state. Thus, the regeneration of HSCs and their niche depends on the cellular plasticity of bone marrow adipocytes. Mouse lineage tracing in regenerating bone marrow after myeloablation shows a dynamic dedifferentiation of mature adipocytes into bone marrow stromal cells. Lipolysis disruption obstructs adipocyte dedifferentiation and hematopoietic stem cell regeneration.},
  archive      = {J_NG},
  author       = {Hirakawa, Hiroyuki and Gao, Longfei and Tavakol, Daniel Naveed and Vunjak-Novakovic, Gordana and Ding, Lei},
  doi          = {10.1038/s41588-023-01528-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1941-1952},
  shortjournal = {Nat. Genet.},
  title        = {Cellular plasticity of the bone marrow niche promotes hematopoietic stem cell regeneration},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Loss of phospholipase PLAAT3 causes a mixed lipodystrophic
and neurological syndrome due to impaired PPARγ signaling. <em>NG</em>,
<em>55</em>(11), 1929–1940. (<a
href="https://doi.org/10.1038/s41588-023-01535-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phospholipase A/acyltransferase 3 (PLAAT3) is a phospholipid-modifying enzyme predominantly expressed in neural and white adipose tissue (WAT). It is a potential drug target for metabolic syndrome, as Plaat3 deficiency in mice protects against diet-induced obesity. We identified seven patients from four unrelated consanguineous families, with homozygous loss-of-function variants in PLAAT3, who presented with a lipodystrophy syndrome with loss of fat varying from partial to generalized and associated with metabolic complications, as well as variable neurological features including demyelinating neuropathy and intellectual disability. Multi-omics analysis of mouse Plaat3−/− and patient-derived WAT showed enrichment of arachidonic acid-containing membrane phospholipids and a strong decrease in the signaling of peroxisome proliferator-activated receptor gamma (PPARγ), the master regulator of adipocyte differentiation. Accordingly, CRISPR–Cas9-mediated PLAAT3 inactivation in human adipose stem cells induced insulin resistance, altered adipocyte differentiation with decreased lipid droplet formation and reduced the expression of adipogenic and mature adipocyte markers, including PPARγ. These findings establish PLAAT3 deficiency as a hereditary lipodystrophy syndrome with neurological manifestations, caused by a PPARγ-dependent defect in WAT differentiation and function. Homozygous loss-of-function variants in phospholipase A/acyltransferase 3 (PLAAT3) underlie a new lipodystrophy syndrome. Functional studies link PLAAT3 loss with peroxisome proliferator-activated receptor gamma (PPARγ)-mediated defects in white adipose tissue differentiation and function.},
  archive      = {J_NG},
  author       = {Schuermans, Nika and El Chehadeh, Salima and Hemelsoet, Dimitri and Gautheron, Jérémie and Vantyghem, Marie-Christine and Nouioua, Sonia and Tazir, Meriem and Vigouroux, Corinne and Auclair, Martine and Bogaert, Elke and Dufour, Sara and Okawa, Fumiya and Hilbert, Pascale and Van Doninck, Nike and Taquet, Marie-Caroline and Rosseel, Toon and De Clercq, Griet and Debackere, Elke and Van Haverbeke, Carole and Cherif, Ferroudja Ramdane and Urtizberea, Jon Andoni and Chanson, Jean-Baptiste and Funalot, Benoit and Authier, François-Jérôme and Kaya, Sabine and Terryn, Wim and Callens, Steven and Depypere, Bernard and Van Dorpe, Jo and Poppe, Bruce and Impens, Francis and Mizushima, Noboru and Depienne, Christel and Jéru, Isabelle and Dermaut, Bart},
  doi          = {10.1038/s41588-023-01535-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1929-1940},
  shortjournal = {Nat. Genet.},
  title        = {Loss of phospholipase PLAAT3 causes a mixed lipodystrophic and neurological syndrome due to impaired PPARγ signaling},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Post-zygotic rescue of meiotic errors causes brain mosaicism
and focal epilepsy. <em>NG</em>, <em>55</em>(11), 1920–1928. (<a
href="https://doi.org/10.1038/s41588-023-01547-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mosaicism is a known cause of neurological disorders, including developmental brain malformations and epilepsy. Brain mosaicism is traditionally attributed to post-zygotic genetic alterations arising in fetal development. Here we describe post-zygotic rescue of meiotic errors as an alternate origin of brain mosaicism in patients with focal epilepsy who have mosaic chromosome 1q copy number gains. Genomic analysis showed evidence of an extra parentally derived chromosome 1q allele in the resected brain tissue from five of six patients. This copy number gain is observed only in patient brain tissue, but not in blood or buccal cells, and is strongly enriched in astrocytes. Astrocytes carrying chromosome 1q gains exhibit distinct gene expression signatures and hyaline inclusions, supporting a novel genetic association for astrocytic inclusions in epilepsy. Further, these data demonstrate an alternate mechanism of brain chromosomal mosaicism, with parentally derived copy number gain isolated to brain, reflecting rescue in other tissues during development. Mosaic copy number gains arising from an extra parentally derived chromosome 1q allele are found in brain tissue from five individuals with focal epilepsy. These copy number gains are strongly enriched in astrocytes, indicating somatic rescue in other tissues during development.},
  archive      = {J_NG},
  author       = {Miller, Katherine E. and Rivaldi, Adithe C. and Shinagawa, Noriyuki and Sran, Sahib and Navarro, Jason B. and Westfall, Jesse J. and Miller, Anthony R. and Roberts, Ryan D. and Akkari, Yassmine and Supinger, Rachel and Hester, Mark E. and Marhabaie, Mohammad and Gade, Meethila and Lu, Jinfeng and Rodziyevska, Olga and Bhattacharjee, Meenakshi B. and Von Allmen, Gretchen K. and Yang, Edward and Lidov, Hart G. W. and Harini, Chellamani and Shah, Manish N. and Leonard, Jeffrey and Pindrik, Jonathan and Shaikhouni, Ammar and Goldman, James E. and Pierson, Christopher R. and Thomas, Diana L. and Boué, Daniel R. and Ostendorf, Adam P. and Mardis, Elaine R. and Poduri, Annapurna and Koboldt, Daniel C. and Heinzen, Erin L. and Bedrosian, Tracy A.},
  doi          = {10.1038/s41588-023-01547-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1920-1928},
  shortjournal = {Nat. Genet.},
  title        = {Post-zygotic rescue of meiotic errors causes brain mosaicism and focal epilepsy},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mosaic chromosomal alterations in blood across ancestries
using whole-genome sequencing. <em>NG</em>, <em>55</em>(11), 1912–1919.
(<a href="https://doi.org/10.1038/s41588-023-01553-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Megabase-scale mosaic chromosomal alterations (mCAs) in blood are prognostic markers for a host of human diseases. Here, to gain a better understanding of mCA rates in genetically diverse populations, we analyzed whole-genome sequencing data from 67,390 individuals from the National Heart, Lung, and Blood Institute Trans-Omics for Precision Medicine program. We observed higher sensitivity with whole-genome sequencing data, compared with array-based data, in uncovering mCAs at low mutant cell fractions and found that individuals of European ancestry have the highest rates of autosomal mCAs and the lowest rates of chromosome X mCAs, compared with individuals of African or Hispanic ancestry. Although further studies in diverse populations will be needed to replicate our findings, we report three loci associated with loss of chromosome X, associations between autosomal mCAs and rare variants in DCPS, ADM17, PPP1R16B and TET2 and ancestry-specific variants in ATM and MPL with mCAs in cis. A method that allows for the detection of mosaic chromosomal alterations from blood whole-genome sequencing data highlights ancestry-specific differences in the distribution of common and rare germline susceptibility variants.},
  archive      = {J_NG},
  author       = {Jakubek, Yasminka A. and Zhou, Ying and Stilp, Adrienne and Bacon, Jason and Wong, Justin W. and Ozcan, Zuhal and Arnett, Donna and Barnes, Kathleen and Bis, Joshua C. and Boerwinkle, Eric and Brody, Jennifer A. and Carson, April P. and Chasman, Daniel I. and Chen, Jiawen and Cho, Michael and Conomos, Matthew P. and Cox, Nancy and Doyle, Margaret F. and Fornage, Myriam and Guo, Xiuqing and Kardia, Sharon L. R. and Lewis, Joshua P. and Loos, Ruth J. F. and Ma, Xiaolong and Machiela, Mitchell J. and Mack, Taralynn M. and Mathias, Rasika A. and Mitchell, Braxton D. and Mychaleckyj, Josyf C. and North, Kari and Pankratz, Nathan and Peyser, Patricia A. and Preuss, Michael H. and Psaty, Bruce and Raffield, Laura M. and Vasan, Ramachandran S. and Redline, Susan and Rich, Stephen S. and Rotter, Jerome I. and Silverman, Edwin K. and Smith, Jennifer A. and Smith, Aaron P. and Taub, Margaret and Taylor, Kent D. and Yun, Jeong and Li, Yun and Desai, Pinkal and Bick, Alexander G. and Reiner, Alexander P. and Scheet, Paul and Auer, Paul L.},
  doi          = {10.1038/s41588-023-01553-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1912-1919},
  shortjournal = {Nat. Genet.},
  title        = {Mosaic chromosomal alterations in blood across ancestries using whole-genome sequencing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A pan-tissue survey of mosaic chromosomal alterations in 948
individuals. <em>NG</em>, <em>55</em>(11), 1901–1911. (<a
href="https://doi.org/10.1038/s41588-023-01537-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic mutations accumulate in an organism’s body throughout its lifetime. While somatic single-nucleotide variants have been well characterized in the human body, the patterns and consequences of large chromosomal alterations in normal tissues remain largely unknown. Here, we present a pan-tissue survey of mosaic chromosomal alterations (mCAs) in 948 healthy individuals from the Genotype-Tissue Expression project, augmenting RNA-based allelic imbalance estimation with haplotype phasing. We found that approximately a quarter of the individuals carry a clonally-expanded mCA in at least one tissue, with incidence strongly correlated with age. The prevalence and genome-wide patterns of mCAs vary considerably across tissue types, suggesting tissue-specific mutagenic exposure and selection pressures. The mCA landscapes in normal adrenal and pituitary glands resemble those in tumors arising from these tissues, whereas the same is not true for the esophagus and skin. Together, our findings show a widespread age-dependent emergence of mCAs across normal human tissues with intricate connections to tumorigenesis. Analysis of GTEx RNA-seq samples identifies hundreds of mosaic chromosomal alterations (mCAs). Considerable inter-tissue variability and excess incidence of mCAs across malignancies suggest a complex relationship with tumorigenesis.},
  archive      = {J_NG},
  author       = {Gao, Teng and Kastriti, Maria Eleni and Ljungström, Viktor and Heinzel, Andreas and Tischler, Arthur S. and Oberbauer, Rainer and Loh, Po-Ru and Adameyko, Igor and Park, Peter J. and Kharchenko, Peter V.},
  doi          = {10.1038/s41588-023-01537-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1901-1911},
  shortjournal = {Nat. Genet.},
  title        = {A pan-tissue survey of mosaic chromosomal alterations in 948 individuals},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Effects of psoriasis and psoralen exposure on the somatic
mutation landscape of the skin. <em>NG</em>, <em>55</em>(11), 1892–1900.
(<a href="https://doi.org/10.1038/s41588-023-01545-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations are hypothesized to play a role in many non-neoplastic diseases. We performed whole-exome sequencing of 1,182 microbiopsies dissected from lesional and nonlesional epidermis from 111 patients with psoriasis to search for evidence that somatic mutations in keratinocytes may influence the disease process. Lesional skin remained highly polyclonal, showing no evidence of large-scale spread of clones carrying potentially pathogenic mutations. The mutation rate of keratinocytes was similarly only modestly affected by the disease. We found evidence of positive selection in previously reported driver genes NOTCH1, NOTCH2, TP53, FAT1 and PPM1D and also identified mutations in four genes (GXYLT1, CHEK2, ZFP36L2 and EEF1A1) that we hypothesize are selected for in squamous epithelium irrespective of disease status. Finally, we describe a mutational signature of psoralens—a class of chemicals previously found in some sunscreens and which are used as part of PUVA (psoralens and ultraviolet-A) photochemotherapy treatment for psoriasis. Analysis of the somatic mutations landscape of 111 patients with psoriasis vulgaris shows that the disease is unlikely to be driven by clonal expansions caused by somatic mutations in keratinocytes. A mutational footprint associated with psoralen treatment was observed and characterized.},
  archive      = {J_NG},
  author       = {Olafsson, Sigurgeir and Rodriguez, Elke and Lawson, Andrew R. J. and Abascal, Federico and Huber, Axel Rosendahl and Suembuel, Melike and Jones, Philip H. and Gerdes, Sascha and Martincorena, Iñigo and Weidinger, Stephan and Campbell, Peter J. and Anderson, Carl A.},
  doi          = {10.1038/s41588-023-01545-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1892-1900},
  shortjournal = {Nat. Genet.},
  title        = {Effects of psoriasis and psoralen exposure on the somatic mutation landscape of the skin},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integrative analyses highlight functional regulatory
variants associated with neuropsychiatric diseases. <em>NG</em>,
<em>55</em>(11), 1876–1891. (<a
href="https://doi.org/10.1038/s41588-023-01533-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Noncoding variants of presumed regulatory function contribute to the heritability of neuropsychiatric disease. A total of 2,221 noncoding variants connected to risk for ten neuropsychiatric disorders, including autism spectrum disorder, attention deficit hyperactivity disorder, bipolar disorder, borderline personality disorder, major depression, generalized anxiety disorder, panic disorder, post-traumatic stress disorder, obsessive-compulsive disorder and schizophrenia, were studied in developing human neural cells. Integrating epigenomic and transcriptomic data with massively parallel reporter assays identified differentially-active single-nucleotide variants (daSNVs) in specific neural cell types. Expression-gene mapping, network analyses and chromatin looping nominated candidate disease-relevant target genes modulated by these daSNVs. Follow-up integration of daSNV gene editing with clinical cohort analyses suggested that magnesium transport dysfunction may increase neuropsychiatric disease risk and indicated that common genetic pathomechanisms may mediate specific symptoms that are shared across multiple neuropsychiatric diseases. Epigenomic profiling and massively parallel reporter assays identify 892 functional differentially-active single-nucleotide variants (daSNVs) linked to ten neuropsychiatric diseases. CRISPRi and gene editing approaches show magnesium transport dysfunction as a common genetic pathomechanism.},
  archive      = {J_NG},
  author       = {Guo, Margaret G. and Reynolds, David L. and Ang, Cheen E. and Liu, Yingfei and Zhao, Yang and Donohue, Laura K. H. and Siprashvili, Zurab and Yang, Xue and Yoo, Yongjin and Mondal, Smarajit and Hong, Audrey and Kain, Jessica and Meservey, Lindsey and Fabo, Tania and Elfaki, Ibtihal and Kellman, Laura N. and Abell, Nathan S. and Pershad, Yash and Bayat, Vafa and Etminani, Payam and Holodniy, Mark and Geschwind, Daniel H. and Montgomery, Stephen B. and Duncan, Laramie E. and Urban, Alexander E. and Altman, Russ B. and Wernig, Marius and Khavari, Paul A.},
  doi          = {10.1038/s41588-023-01533-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1876-1891},
  shortjournal = {Nat. Genet.},
  title        = {Integrative analyses highlight functional regulatory variants associated with neuropsychiatric diseases},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Systematic differences in discovery of genetic effects on
gene expression and complex traits. <em>NG</em>, <em>55</em>(11),
1866–1875. (<a
href="https://doi.org/10.1038/s41588-023-01529-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most signals in genome-wide association studies (GWAS) of complex traits implicate noncoding genetic variants with putative gene regulatory effects. However, currently identified regulatory variants, notably expression quantitative trait loci (eQTLs), explain only a small fraction of GWAS signals. Here, we show that GWAS and cis-eQTL hits are systematically different: eQTLs cluster strongly near transcription start sites, whereas GWAS hits do not. Genes near GWAS hits are enriched in key functional annotations, are under strong selective constraint and have complex regulatory landscapes across different tissue/cell types, whereas genes near eQTLs are depleted of most functional annotations, show relaxed constraint, and have simpler regulatory landscapes. We describe a model to understand these observations, including how natural selection on complex traits hinders discovery of functionally relevant eQTLs. Our results imply that GWAS and eQTL studies are systematically biased toward different types of variant, and support the use of complementary functional approaches alongside the next generation of eQTL studies. This study seeks to explain the poor overlap of genome-wide association study and cis-expression quantitative trait locus variants using a model of differential selective constraint, suggesting that these two study types have biases towards different functional classes of variants.},
  archive      = {J_NG},
  author       = {Mostafavi, Hakhamanesh and Spence, Jeffrey P. and Naqvi, Sahin and Pritchard, Jonathan K.},
  doi          = {10.1038/s41588-023-01529-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1866-1875},
  shortjournal = {Nat. Genet.},
  title        = {Systematic differences in discovery of genetic effects on gene expression and complex traits},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Age-dependent topic modeling of comorbidities in UK biobank
identifies disease subtypes with differential genetic risk. <em>NG</em>,
<em>55</em>(11), 1854–1865. (<a
href="https://doi.org/10.1038/s41588-023-01522-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The analysis of longitudinal data from electronic health records (EHRs) has the potential to improve clinical diagnoses and enable personalized medicine, motivating efforts to identify disease subtypes from patient comorbidity information. Here we introduce an age-dependent topic modeling (ATM) method that provides a low-rank representation of longitudinal records of hundreds of distinct diseases in large EHR datasets. We applied ATM to 282,957 UK Biobank samples, identifying 52 diseases with heterogeneous comorbidity profiles; analyses of 211,908 All of Us samples produced concordant results. We defined subtypes of the 52 heterogeneous diseases based on their comorbidity profiles and compared genetic risk across disease subtypes using polygenic risk scores (PRSs), identifying 18 disease subtypes whose PRS differed significantly from other subtypes of the same disease. We further identified specific genetic variants with subtype-dependent effects on disease risk. In conclusion, ATM identifies disease subtypes with differential genome-wide and locus-specific genetic risk profiles. Age-dependent topic modeling provides a low-rank representation of longitudinal disease records and identifies diseases with heterogeneous comorbidity profiles, defining subtypes that exhibit distinct genome-wide and locus-specific association patterns.},
  archive      = {J_NG},
  author       = {Jiang, Xilin and Zhang, Martin Jinye and Zhang, Yidong and Durvasula, Arun and Inouye, Michael and Holmes, Chris and Price, Alkes L. and McVean, Gil},
  doi          = {10.1038/s41588-023-01522-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1854-1865},
  shortjournal = {Nat. Genet.},
  title        = {Age-dependent topic modeling of comorbidities in UK biobank identifies disease subtypes with differential genetic risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rare variants with large effects provide functional insights
into the pathology of migraine subtypes, with and without aura.
<em>NG</em>, <em>55</em>(11), 1843–1853. (<a
href="https://doi.org/10.1038/s41588-023-01538-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Migraine is a complex neurovascular disease with a range of severity and symptoms, yet mostly studied as one phenotype in genome-wide association studies (GWAS). Here we combine large GWAS datasets from six European populations to study the main migraine subtypes, migraine with aura (MA) and migraine without aura (MO). We identified four new MA-associated variants (in PRRT2, PALMD, ABO and LRRK2) and classified 13 MO-associated variants. Rare variants with large effects highlight three genes. A rare frameshift variant in brain-expressed PRRT2 confers large risk of MA and epilepsy, but not MO. A burden test of rare loss-of-function variants in SCN11A, encoding a neuron-expressed sodium channel with a key role in pain sensation, shows strong protection against migraine. Finally, a rare variant with cis-regulatory effects on KCNK5 confers large protection against migraine and brain aneurysms. Our findings offer new insights with therapeutic potential into the complex biology of migraine and its subtypes. Genome-wide association analyses of migraine and its subtypes identify new susceptibility loci, including rare variants with large effects implicating PRRT2, SCN11A and KCNK5.},
  archive      = {J_NG},
  author       = {Bjornsdottir, Gyda and Chalmer, Mona A. and Stefansdottir, Lilja and Skuladottir, Astros Th. and Einarsson, Gudmundur and Andresdottir, Margret and Beyter, Doruk and Ferkingstad, Egil and Gretarsdottir, Solveig and Halldorsson, Bjarni V. and Halldorsson, Gisli H. and Helgadottir, Anna and Helgason, Hannes and Hjorleifsson Eldjarn, Grimur and Jonasdottir, Adalbjorg and Jonasdottir, Aslaug and Jonsdottir, Ingileif and Knowlton, Kirk U. and Nadauld, Lincoln D. and Lund, Sigrun H. and Magnusson, Olafur Th. and Melsted, Pall and Moore, Kristjan H. S. and Oddsson, Asmundur and Olason, Pall I. and Sigurdsson, Asgeir and Stefansson, Olafur A. and Saemundsdottir, Jona and Sveinbjornsson, Gardar and Tragante, Vinicius and Unnsteinsdottir, Unnur and Walters, G. Bragi and Zink, Florian and Rødevand, Linn and Andreassen, Ole A. and Igland, Jannicke and Lie, Rolv T. and Haavik, Jan and Banasik, Karina and Brunak, Søren and Didriksen, Maria and T. Bruun, Mie and Erikstrup, Christian and Kogelman, Lisette J. A. and Nielsen, Kaspar R. and Sørensen, Erik and Pedersen, Ole B. and Ullum, Henrik and Masson, Gisli and Thorsteinsdottir, Unnur and Olesen, Jes and Ludvigsson, Petur and Thorarensen, Olafur and Bjornsdottir, Anna and Sigurdardottir, Gudrun R. and Sveinsson, Olafur A. and Ostrowski, Sisse R. and Holm, Hilma and Gudbjartsson, Daniel F. and Thorleifsson, Gudmar and Sulem, Patrick and Stefansson, Hreinn and Thorgeirsson, Thorgeir E. and Hansen, Thomas F. and Stefansson, Kari},
  doi          = {10.1038/s41588-023-01538-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1843-1853},
  shortjournal = {Nat. Genet.},
  title        = {Rare variants with large effects provide functional insights into the pathology of migraine subtypes, with and without aura},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide association meta-analysis identifies risk loci
for abdominal aortic aneurysm and highlights PCSK9 as a therapeutic
target. <em>NG</em>, <em>55</em>(11), 1831–1842. (<a
href="https://doi.org/10.1038/s41588-023-01510-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Abdominal aortic aneurysm (AAA) is a common disease with substantial heritability. In this study, we performed a genome-wide association meta-analysis from 14 discovery cohorts and uncovered 141 independent associations, including 97 previously unreported loci. A polygenic risk score derived from meta-analysis explained AAA risk beyond clinical risk factors. Genes at AAA risk loci indicate involvement of lipid metabolism, vascular development and remodeling, extracellular matrix dysregulation and inflammation as key mechanisms in AAA pathogenesis. These genes also indicate overlap between the development of AAA and other monogenic aortopathies, particularly via transforming growth factor β signaling. Motivated by the strong evidence for the role of lipid metabolism in AAA, we used Mendelian randomization to establish the central role of nonhigh-density lipoprotein cholesterol in AAA and identified the opportunity for repurposing of proprotein convertase, subtilisin/kexin-type 9 (PCSK9) inhibitors. This was supported by a study demonstrating that PCSK9 loss of function prevented the development of AAA in a preclinical mouse model. Genome-wide association meta-analysis of AAA identifies 121 independent risk loci and highlights potential therapeutic targets such as proprotein convertase, subtilisin/kexin-type 9 (PCSK9).},
  archive      = {J_NG},
  author       = {Roychowdhury, Tanmoy and Klarin, Derek and Levin, Michael G. and Spin, Joshua M. and Rhee, Yae Hyun and Deng, Alicia and Headley, Colwyn A. and Tsao, Noah L. and Gellatly, Corry and Zuber, Verena and Shen, Fred and Hornsby, Whitney E. and Laursen, Ina Holst and Verma, Shefali S. and Locke, Adam E. and Einarsson, Gudmundur and Thorleifsson, Gudmar and Graham, Sarah E. and Dikilitas, Ozan and Pattee, Jack W. and Judy, Renae L. and Pauls-Verges, Ferran and Nielsen, Jonas B. and Wolford, Brooke N. and Brumpton, Ben M. and Dilmé, Jaume and Peypoch, Olga and Juscafresa, Laura Calsina and Edwards, Todd L. and Li, Dadong and Banasik, Karina and Brunak, Søren and Jacobsen, Rikke L. and Garcia-Barrio, Minerva T. and Zhang, Jifeng and Rasmussen, Lars M. and Lee, Regent and Handa, Ashok and Wanhainen, Anders and Mani, Kevin and Lindholt, Jes S. and Obel, Lasse M. and Strauss, Ewa and Oszkinis, Grzegorz and Nelson, Christopher P. and Saxby, Katie L. and van Herwaarden, Joost A. and van der Laan, Sander W. and van Setten, Jessica and Camacho, Mercedes and Davis, Frank M. and Wasikowski, Rachael and Tsoi, Lam C. and Gudjonsson, Johann E. and Eliason, Jonathan L. and Coleman, Dawn M. and Henke, Peter K. and Ganesh, Santhi K. and Chen, Y. Eugene and Guan, Weihua and Pankow, James S. and Pankratz, Nathan and Pedersen, Ole B. and Erikstrup, Christian and Tang, Weihong and Hveem, Kristian and Gudbjartsson, Daniel and Gretarsdottir, Solveig and Thorsteinsdottir, Unnur and Holm, Hilma and Stefansson, Kari and Ferreira, Manuel A. and Baras, Aris and Kullo, Iftikhar J. and Ritchie, Marylyn D. and Christensen, Alex H. and Iversen, Kasper K. and Eldrup, Nikolaj and Sillesen, Henrik and Ostrowski, Sisse R. and Bundgaard, Henning and Ullum, Henrik and Burgess, Stephen and Gill, Dipender and Gallagher, Katherine and Sabater-Lleal, Maria and Surakka, Ida and Jones, Gregory T. and Bown, Matthew J. and Tsao, Philip S. and Willer, Cristen J. and Damrauer, Scott M.},
  doi          = {10.1038/s41588-023-01510-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1831-1842},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analysis identifies risk loci for abdominal aortic aneurysm and highlights PCSK9 as a therapeutic target},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An atlas of genetic determinants of forearm fracture.
<em>NG</em>, <em>55</em>(11), 1820–1830. (<a
href="https://doi.org/10.1038/s41588-023-01527-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Osteoporotic fracture is among the most common and costly of diseases. While reasonably heritable, its genetic determinants have remained elusive. Forearm fractures are the most common clinically recognized osteoporotic fractures with a relatively high heritability. To establish an atlas of the genetic determinants of forearm fractures, we performed genome-wide association analyses including 100,026 forearm fracture cases. We identified 43 loci, including 26 new fracture loci. Although most fracture loci associated with bone mineral density, we also identified loci that primarily regulate bone quality parameters. Functional studies of one such locus, at TAC4, revealed that Tac4–/– mice have reduced mechanical bone strength. The strongest forearm fracture signal, at WNT16, displayed remarkable bone-site-specificity with no association with hip fractures. Tall stature and low body mass index were identified as new causal risk factors for fractures. The insights from this atlas may improve fracture prediction and enable therapeutic development to prevent fractures. Genome-wide analyses identify 43 loci associated with forearm fracture, including some influencing bone quality parameters. Follow-up work shows that Tac4 knockout mice exhibit reduced mechanical bone strength with no effect on bone mineral density.},
  archive      = {J_NG},
  author       = {Nethander, Maria and Movérare-Skrtic, Sofia and Kämpe, Anders and Coward, Eivind and Reimann, Ene and Grahnemo, Louise and Borbély, Éva and Helyes, Zsuzsanna and Funck-Brentano, Thomas and Cohen-Solal, Martine and Tuukkanen, Juha and Koskela, Antti and Wu, Jianyao and Li, Lei and Lu, Tianyuan and Gabrielsen, Maiken E. and Mägi, Reedik and Hoff, Mari and Lerner, Ulf H. and Henning, Petra and Ullum, Henrik and Erikstrup, Christian and Brunak, Søren and Langhammer, Arnulf and Tuomi, Tiinamaija and Oddsson, Asmundur and Stefansson, Kari and Pettersson-Kymmer, Ulrika and Ostrowski, Sisse Rye and Pedersen, Ole Birger Vesterager and Styrkarsdottir, Unnur and Mäkitie, Outi and Hveem, Kristian and Richards, J. Brent and Ohlsson, Claes},
  doi          = {10.1038/s41588-023-01527-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1820-1830},
  shortjournal = {Nat. Genet.},
  title        = {An atlas of genetic determinants of forearm fracture},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide association study of placental weight identifies
distinct and shared genetic influences between placental and fetal
growth. <em>NG</em>, <em>55</em>(11), 1807–1819. (<a
href="https://doi.org/10.1038/s41588-023-01520-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A well-functioning placenta is essential for fetal and maternal health throughout pregnancy. Using placental weight as a proxy for placental growth, we report genome-wide association analyses in the fetal (n = 65,405), maternal (n = 61,228) and paternal (n = 52,392) genomes, yielding 40 independent association signals. Twenty-six signals are classified as fetal, four maternal and three fetal and maternal. A maternal parent-of-origin effect is seen near KCNQ1. Genetic correlation and colocalization analyses reveal overlap with birth weight genetics, but 12 loci are classified as predominantly or only affecting placental weight, with connections to placental development and morphology, and transport of antibodies and amino acids. Mendelian randomization analyses indicate that fetal genetically mediated higher placental weight is causally associated with preeclampsia risk and shorter gestational duration. Moreover, these analyses support the role of fetal insulin in regulating placental weight, providing a key link between fetal and placental growth. Genome-wide analyses using fetal and parental genotypes identify 40 independent associations influencing placental weight and highlight the role of the fetus in preeclampsia risk and placental growth regulation via insulin signaling.},
  archive      = {J_NG},
  author       = {Beaumont, Robin N. and Flatley, Christopher and Vaudel, Marc and Wu, Xiaoping and Chen, Jing and Moen, Gunn-Helen and Skotte, Line and Helgeland, Øyvind and Solé-Navais, Pol and Banasik, Karina and Albiñana, Clara and Ronkainen, Justiina and Fadista, João and Stinson, Sara Elizabeth and Trajanoska, Katerina and Wang, Carol A. and Westergaard, David and Srinivasan, Sundararajan and Sánchez-Soriano, Carlos and Bilbao, Jose Ramon and Allard, Catherine and Groleau, Marika and Kuulasmaa, Teemu and Leirer, Daniel J. and White, Frédérique and Jacques, Pierre-Étienne and Cheng, Haoxiang and Hao, Ke and Andreassen, Ole A. and Åsvold, Bjørn Olav and Atalay, Mustafa and Bhatta, Laxmi and Bouchard, Luigi and Brumpton, Ben Michael and Brunak, Søren and Bybjerg-Grauholm, Jonas and Ebbing, Cathrine and Elliott, Paul and Engelbrechtsen, Line and Erikstrup, Christian and Estarlich, Marisa and Franks, Stephen and Gaillard, Romy and Geller, Frank and Grove, Jakob and Hougaard, David M. and Kajantie, Eero and Morgen, Camilla S. and Nohr, Ellen A. and Nyegaard, Mette and Palmer, Colin N. A. and Pedersen, Ole Birger and Rivadeneira, Fernando and Sebert, Sylvain and Shields, Beverley M. and Stoltenberg, Camilla and Surakka, Ida and Thørner, Lise Wegner and Ullum, Henrik and Vaarasmaki, Marja and Vilhjalmsson, Bjarni J. and Willer, Cristen J. and Lakka, Timo A. and Gybel-Brask, Dorte and Bustamante, Mariona and Hansen, Torben and Pearson, Ewan R. and Reynolds, Rebecca M. and Ostrowski, Sisse R. and Pennell, Craig E. and Jaddoe, Vincent W. V. and Felix, Janine F. and Hattersley, Andrew T. and Melbye, Mads and Lawlor, Deborah A. and Hveem, Kristian and Werge, Thomas and Nielsen, Henriette Svarre and Magnus, Per and Evans, David M. and Jacobsson, Bo and Järvelin, Marjo-Riitta and Zhang, Ge and Hivert, Marie-France and Johansson, Stefan and Freathy, Rachel M. and Feenstra, Bjarke and Njølstad, Pål R.},
  doi          = {10.1038/s41588-023-01520-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1807-1819},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study of placental weight identifies distinct and shared genetic influences between placental and fetal growth},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reactivation of embryonic genetic programs in tissue
regeneration and disease. <em>NG</em>, <em>55</em>(11), 1792–1806. (<a
href="https://doi.org/10.1038/s41588-023-01526-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryonic genetic programs are reactivated in response to various types of tissue damage, providing cell plasticity for tissue regeneration or disease progression. In acute conditions, these programs remedy the damage and then halt to allow a return to homeostasis. In chronic situations, including inflammatory diseases, fibrosis and cancer, prolonged activation of embryonic programs leads to disease progression and tissue deterioration. Induction of progenitor identity and cell plasticity, for example, epithelial–mesenchymal plasticity, are critical outcomes of reactivated embryonic programs. In this Review, we describe molecular players governing reactivated embryonic genetic programs, their role during disease progression, their similarities and differences and lineage reversion in pathology and discuss associated therapeutics and drug-resistance mechanisms across many organs. We also discuss the diversity of reactivated programs in different disease contexts. A comprehensive overview of commonalities between development and disease will provide better understanding of the biology and therapeutic strategies. This Review discusses how embryonic transcriptional programs, such as epithelial–mesenchymal plasticity and stemness, may be harnessed in adult tissues to drive processes and diseases such as regeneration and cancer.},
  archive      = {J_NG},
  author       = {Fazilaty, Hassan and Basler, Konrad},
  doi          = {10.1038/s41588-023-01526-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1792-1806},
  shortjournal = {Nat. Genet.},
  title        = {Reactivation of embryonic genetic programs in tissue regeneration and disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Phospholipid modifier PLAAT3 links defective PPARγ-dependent
signaling to lipodystrophy. <em>NG</em>, <em>55</em>(11), 1790–1791. (<a
href="https://doi.org/10.1038/s41588-023-01536-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-exome and genome sequencing in consanguineous families with unsolved lipodystrophy identified biallelic pathogenic loss-of-function variants in the phospholipase gene PLAAT3. Multi-omics and functional analyses in human and mouse PLAAT3-deficient adipose tissue and adipose stem cells revealed an adipocyte differentiation defect that is mediated by an altered gene network downstream of the adipogenesis master regulator PPARγ.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01536-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1790-1791},
  shortjournal = {Nat. Genet.},
  title        = {Phospholipid modifier PLAAT3 links defective PPARγ-dependent signaling to lipodystrophy},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identifying genetic subtypes of disease from hospital
diagnosis records. <em>NG</em>, <em>55</em>(11), 1788–1789. (<a
href="https://doi.org/10.1038/s41588-023-01521-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a computational, age-dependent topic model to identify longitudinal comorbidity patterns from hospital diagnosis data. The inferred comorbidity patterns are robust across UK and US populations and identify disease subtypes with distinct genetic profiles.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01521-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1788-1789},
  shortjournal = {Nat. Genet.},
  title        = {Identifying genetic subtypes of disease from hospital diagnosis records},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Regulation of placental growth is a complex interplay of
maternal and fetal genomes. <em>NG</em>, <em>55</em>(11), 1786–1787. (<a
href="https://doi.org/10.1038/s41588-023-01519-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association analyses of placental weight identify 40 association signals, partially overlapping with birth weight genetics. We find parent-of-origin effects and connections to placental development and morphology, and transport of amino acids and antibodies. Mendelian randomization reveals a fetal contribution to preeclampsia and implicates fetal insulin in the regulation of placental growth.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01519-3},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1786-1787},
  shortjournal = {Nat. Genet.},
  title        = {Regulation of placental growth is a complex interplay of maternal and fetal genomes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Post-zygotic brain mosaicism as a result of partial
reversion of pre-zygotic aneuploidy. <em>NG</em>, <em>55</em>(11),
1784–1785. (<a
href="https://doi.org/10.1038/s41588-023-01552-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain somatic mosaicism is linked to several neurological disorders and is thought to arise post-zygotically. A study suggests that pre-zygotic aneuploidy followed by post-zygotic partial reversion leads to a recurrent form of brain mosaicism-related epilepsy.},
  archive      = {J_NG},
  author       = {Chung, Changuk and Yang, Xiaoxu and Gleeson, Joseph G.},
  doi          = {10.1038/s41588-023-01552-2},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1784-1785},
  shortjournal = {Nat. Genet.},
  title        = {Post-zygotic brain mosaicism as a result of partial reversion of pre-zygotic aneuploidy},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The power paradox of detecting disease-associated and
gene-expression-associated variants. <em>NG</em>, <em>55</em>(11),
1782–1783. (<a
href="https://doi.org/10.1038/s41588-023-01525-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms of many disease-associated variants are uncertain because of limited power to detect their modest effects on gene expression. This study finds that natural selection leads to preferential detection of disease-associated versus expression-associated variants.},
  archive      = {J_NG},
  author       = {Amariuta, Tiffany},
  doi          = {10.1038/s41588-023-01525-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1782-1783},
  shortjournal = {Nat. Genet.},
  title        = {The power paradox of detecting disease-associated and gene-expression-associated variants},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). UK biobank pharma proteomics resource. <em>NG</em>,
<em>55</em>(11), 1781. (<a
href="https://doi.org/10.1038/s41588-023-01575-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01575-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1781},
  shortjournal = {Nat. Genet.},
  title        = {UK biobank pharma proteomics resource},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Functional dissection of schizophrenia risk variants.
<em>NG</em>, <em>55</em>(11), 1781. (<a
href="https://doi.org/10.1038/s41588-023-01576-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01576-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1781},
  shortjournal = {Nat. Genet.},
  title        = {Functional dissection of schizophrenia risk variants},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Identifying regulators of γδ t cell activity. <em>NG</em>,
<em>55</em>(11), 1781. (<a
href="https://doi.org/10.1038/s41588-023-01579-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01579-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1781},
  shortjournal = {Nat. Genet.},
  title        = {Identifying regulators of γδ t cell activity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Acetyl-methyllysine is a dual-modification of histone h4.
<em>NG</em>, <em>55</em>(11), 1781. (<a
href="https://doi.org/10.1038/s41588-023-01580-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01580-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1781},
  shortjournal = {Nat. Genet.},
  title        = {Acetyl-methyllysine is a dual-modification of histone h4},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Publisher correction: Solanum americanum genome-assisted
discovery of immune receptors that detect potato late blight pathogen
effectors. <em>NG</em>, <em>55</em>(10), 1779. (<a
href="https://doi.org/10.1038/s41588-023-01550-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lin, Xiao and Jia, Yuxin and Heal, Robert and Prokchorchik, Maxim and Sindalovskaya, Maria and Olave-Achury, Andrea and Makechemu, Moffat and Fairhead, Sebastian and Noureen, Azka and Heo, Jung and Witek, Kamil and Smoker, Matthew and Taylor, Jodie and Shrestha, Ram-Krishna and Lee, Yoonyoung and Zhang, Chunzhi and Park, Soon Ju and Sohn, Kee Hoon and Huang, Sanwen and Jones, Jonathan D. G.},
  doi          = {10.1038/s41588-023-01550-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1779},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Solanum americanum genome-assisted discovery of immune receptors that detect potato late blight pathogen effectors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Multi-ancestry genome-wide association
analyses improve resolution of genes and pathways influencing lung
function and chronic obstructive pulmonary disease risk. <em>NG</em>,
<em>55</em>(10), 1778–1779. (<a
href="https://doi.org/10.1038/s41588-023-01531-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Shrine, Nick and Izquierdo, Abril G. and Chen, Jing and Packer, Richard and Hall, Robert J. and Guyatt, Anna L. and Batini, Chiara and Thompson, Rebecca J. and Pavuluri, Chandan and Malik, Vidhi and Hobbs, Brian D. and Moll, Matthew and Kim, Wonji and Tal-Singer, Ruth and Bakke, Per and Fawcett, Katherine A. and John, Catherine and Coley, Kayesha and Piga, Noemi Nicole and Pozarickij, Alfred and Lin, Kuang and Millwood, Iona Y. and Chen, Zhengming and Li, Liming and Wijnant, Sara R. A. and Lahousse, Lies and Brusselle, Guy and Uitterlinden, Andre G. and Manichaikul, Ani and Oelsner, Elizabeth C. and Rich, Stephen S. and Barr, R. Graham and Kerr, Shona M. and Vitart, Veronique and Brown, Michael R. and Wielscher, Matthias and Imboden, Medea and Jeong, Ayoung and Bartz, Traci M. and Gharib, Sina A. and Flexeder, Claudia and Karrasch, Stefan and Gieger, Christian and Peters, Annette and Stubbe, Beate and Hu, Xiaowei and Ortega, Victor E. and Meyers, Deborah A. and Bleecker, Eugene R. and Gabriel, Stacey B. and Gupta, Namrata and Smith, Albert Vernon and Luan, Jian’an and Zhao, Jing-Hua and Hansen, Ailin F. and Langhammer, Arnulf and Willer, Cristen and Bhatta, Laxmi and Porteous, David and Smith, Blair H. and Campbell, Archie and Sofer, Tamar and Lee, Jiwon and Daviglus, Martha L. and Yu, Bing and Lim, Elise and Xu, Hanfei and O’Connor, George T. and Thareja, Gaurav and Albagha, Omar M. E. and Suhre, Karsten and Granell, Raquel and Faquih, Tariq O. and Hiemstra, Pieter S. and Slats, Annelies M. and Mullin, Benjamin H. and Hui, Jennie and James, Alan and Beilby, John and Patasova, Karina and Hysi, Pirro and Koskela, Jukka T. and Wyss, Annah B. and Jin, Jianping and Sikdar, Sinjini and Lee, Mikyeong and May-Wilson, Sebastian and Pirastu, Nicola and Kentistou, Katherine A. and Joshi, Peter K. and Timmers, Paul R. H. J. and Williams, Alexander T. and Free, Robert C. and Wang, Xueyang and Morrison, John L. and Gilliland, Frank D. and Chen, Zhanghua and Wang, Carol A. and Foong, Rachel E. and Harris, Sarah E. and Taylor, Adele and Redmond, Paul and Cook, James P. and Mahajan, Anubha and Lind, Lars and Palviainen, Teemu and Lehtimäki, Terho and Raitakari, Olli T. and Kaprio, Jaakko and Rantanen, Taina and Pietiläinen, Kirsi H. and Cox, Simon R. and Pennell, Craig E. and Hall, Graham L. and Gauderman, W. James and Brightling, Chris and Wilson, James F. and Vasankari, Tuula and Laitinen, Tarja and Salomaa, Veikko and Mook-Kanamori, Dennis O. and Timpson, Nicholas J. and Zeggini, Eleftheria and Dupuis, Josée and Hayward, Caroline and Brumpton, Ben and Langenberg, Claudia and Weiss, Stefan and Homuth, Georg and Schmidt, Carsten Oliver and Probst-Hensch, Nicole and Jarvelin, Marjo-Riitta and Morrison, Alanna C. and Polasek, Ozren and Rudan, Igor and Lee, Joo-Hyeon and Sayers, Ian and Rawlins, Emma L. and Dudbridge, Frank and Silverman, Edwin K. and Strachan, David P. and Walters, Robin G. and Morris, Andrew P. and London, Stephanie J. and Cho, Michael H. and Wain, Louise V. and Hall, Ian P. and Tobin, Martin D.},
  doi          = {10.1038/s41588-023-01531-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1778-1779},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Multi-ancestry genome-wide association analyses improve resolution of genes and pathways influencing lung function and chronic obstructive pulmonary disease risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Multiparameter prediction of myeloid
neoplasia risk. <em>NG</em>, <em>55</em>(10), 1777. (<a
href="https://doi.org/10.1038/s41588-023-01532-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gu, Muxin and Kovilakam, Sruthi Cheloor and Dunn, William G. and Marando, Ludovica and Barcena, Clea and Mohorianu, Irina and Smith, Alexandra and Kar, Siddhartha P. and Fabre, Margarete A. and Gerstung, Moritz and Cargo, Catherine A. and Malcovati, Luca and Quiros, Pedro M. and Vassiliou, George S.},
  doi          = {10.1038/s41588-023-01532-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1777},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Multiparameter prediction of myeloid neoplasia risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Boosting the power of genome-wide association studies within
and across ancestries by using polygenic scores. <em>NG</em>,
<em>55</em>(10), 1769–1776. (<a
href="https://doi.org/10.1038/s41588-023-01500-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWASs) have been mostly conducted in populations of European ancestry, which currently limits the transferability of their findings to other populations. Here, we show, through theory, simulations and applications to real data, that adjustment of GWAS analyses for polygenic scores (PGSs) increases the statistical power for discovery across all ancestries. We applied this method to analyze seven traits available in three large biobanks with participants of East Asian ancestry (n = 340,000 in total) and report 139 additional associations across traits. We also present a two-stage meta-analysis strategy whereby, in contributing cohorts, a PGS-adjusted GWAS is rerun using PGSs derived from a first round of a standard meta-analysis. On average, across traits, this approach yields a 1.26-fold increase in the number of detected associations (range 1.07- to 1.76-fold increase). Altogether, our study demonstrates the value of using PGSs to increase the power of GWASs in underrepresented populations and promotes such an analytical strategy for future GWAS meta-analyses. Simulations and applications to real data show that adjustment of genome-wide association analyses for polygenic scores increases the statistical power for discovery across all ancestries, suggesting an analytical strategy for future studies in underrepresented populations.},
  archive      = {J_NG},
  author       = {Campos, Adrian I. and Namba, Shinichi and Lin, Shu-Chin and Nam, Kisung and Sidorenko, Julia and Wang, Huanwei and Kamatani, Yoichiro and Wang, Ling-Hua and Lee, Seunggeun and Lin, Yen-Feng and Feng, Yen-Chen Anne and Okada, Yukinori and Visscher, Peter M. and Yengo, Loic},
  doi          = {10.1038/s41588-023-01500-0},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1769-1776},
  shortjournal = {Nat. Genet.},
  title        = {Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A new method for multiancestry polygenic prediction improves
performance across diverse populations. <em>NG</em>, <em>55</em>(10),
1757–1768. (<a
href="https://doi.org/10.1038/s41588-023-01501-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polygenic risk scores (PRSs) increasingly predict complex traits; however, suboptimal performance in non-European populations raise concerns about clinical applications and health inequities. We developed CT-SLEB, a powerful and scalable method to calculate PRSs, using ancestry-specific genome-wide association study summary statistics from multiancestry training samples, integrating clumping and thresholding, empirical Bayes and superlearning. We evaluated CT-SLEB and nine alternative methods with large-scale simulated genome-wide association studies (~19 million common variants) and datasets from 23andMe, Inc., the Global Lipids Genetics Consortium, All of Us and UK Biobank, involving 5.1 million individuals of diverse ancestry, with 1.18 million individuals from four non-European populations across 13 complex traits. Results demonstrated that CT-SLEB significantly improves PRS performance in non-European populations compared with simple alternatives, with comparable or superior performance to a recent, computationally intensive method. Moreover, our simulation studies offered insights into sample size requirements and SNP density effects on multiancestry risk prediction. CT-SLEB, a powerful and scalable method, improves the performance of multiancestry polygenic prediction by generating polygenic risk scores based on GWAS summary statistics in diverse populations.},
  archive      = {J_NG},
  author       = {Zhang, Haoyu and Zhan, Jianan and Jin, Jin and Zhang, Jingning and Lu, Wenxuan and Zhao, Ruzhang and Ahearn, Thomas U. and Yu, Zhi and O’Connell, Jared and Jiang, Yunxuan and Chen, Tony and Okuhara, Dayne and Garcia-Closas, Montserrat and Lin, Xihong and Koelsch, Bertram L. and Chatterjee, Nilanjan},
  doi          = {10.1038/s41588-023-01501-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1757-1768},
  shortjournal = {Nat. Genet.},
  title        = {A new method for multiancestry polygenic prediction improves performance across diverse populations},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Structure and function of rice hybrid genomes reveal genetic
basis and optimal performance of heterosis. <em>NG</em>,
<em>55</em>(10), 1745–1756. (<a
href="https://doi.org/10.1038/s41588-023-01495-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exploitation of crop heterosis is crucial for increasing global agriculture production. However, the quantitative genomic analysis of heterosis was lacking, and there is currently no effective prediction tool to optimize cross-combinations. Here 2,839 rice hybrid cultivars and 9,839 segregation individuals were resequenced and phenotyped. Our findings demonstrated that indica–indica hybrid-improving breeding was a process that broadened genetic resources, pyramided breeding-favorable alleles through combinatorial selection and collaboratively improved both parents by eliminating the inferior alleles at negative dominant loci. Furthermore, we revealed that widespread genetic complementarity contributed to indica–japonica intersubspecific heterosis in yield traits, with dominance effect loci making a greater contribution to phenotypic variance than overdominance effect loci. On the basis of the comprehensive dataset, a genomic model applicable to diverse rice varieties was developed and optimized to predict the performance of hybrid combinations. Our data offer a valuable resource for advancing the understanding and facilitating the utilization of heterosis in rice. Resequencing of 2,839 hybrid rice cultivars and 9,839 F2 individuals from 18 elite crosses is used to characterize the genetics underlying a range of grain yield-related traits, providing insights into heterosis during breeding and a predictive model.},
  archive      = {J_NG},
  author       = {Gu, Zhoulin and Gong, Junyi and Zhu, Zhou and Li, Zhen and Feng, Qi and Wang, Changsheng and Zhao, Yan and Zhan, Qilin and Zhou, Congcong and Wang, Ahong and Huang, Tao and Zhang, Lei and Tian, Qilin and Fan, Danlin and Lu, Yiqi and Zhao, Qiang and Huang, Xuehui and Yang, Shihua and Han, Bin},
  doi          = {10.1038/s41588-023-01495-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1745-1756},
  shortjournal = {Nat. Genet.},
  title        = {Structure and function of rice hybrid genomes reveal genetic basis and optimal performance of heterosis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Functional characterization of alzheimer’s disease genetic
variants in microglia. <em>NG</em>, <em>55</em>(10), 1735–1744. (<a
href="https://doi.org/10.1038/s41588-023-01506-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Candidate cis-regulatory elements (cCREs) in microglia demonstrate the most substantial enrichment for Alzheimer’s disease (AD) heritability compared to other brain cell types. However, whether and how these genome-wide association studies (GWAS) variants contribute to AD remain elusive. Here we prioritize 308 previously unreported AD risk variants at 181 cCREs by integrating genetic information with microglia-specific 3D epigenome annotation. We further establish the link between functional variants and target genes by single-cell CRISPRi screening in microglia. In addition, we show that AD variants exhibit allelic imbalance on target gene expression. In particular, rs7922621 is the effective variant in controlling TSPAN14 expression among other nominated variants in the same cCRE and exerts multiple physiological effects including reduced cell surface ADAM10 and altered soluble TREM2 (sTREM2) shedding. Our work represents a systematic approach to prioritize and characterize AD-associated variants and provides a roadmap for advancing genetic association to experimentally validated cell-type-specific phenotypes and mechanisms. Genetic fine-mapping and CRISPRi screens identify functional variants and their target genes associated with Alzheimer’s disease in microglia. The variant rs7922621 modulates AD risk through control of TSPAN14 expression in this cell type.},
  archive      = {J_NG},
  author       = {Yang, Xiaoyu and Wen, Jia and Yang, Han and Jones, Ian R. and Zhu, Xiaodong and Liu, Weifang and Li, Bingkun and Clelland, Claire D. and Luo, Wenjie and Wong, Man Ying and Ren, Xingjie and Cui, Xiekui and Song, Michael and Liu, Hongjiang and Chen, Cady and Eng, Nicolas and Ravichandran, Mirunalini and Sun, Yang and Lee, David and Van Buren, Eric and Jiang, Min-Zhi and Chan, Candace S. Y. and Ye, Chun Jimmie and Perera, Rushika M. and Gan, Li and Li, Yun and Shen, Yin},
  doi          = {10.1038/s41588-023-01506-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1735-1744},
  shortjournal = {Nat. Genet.},
  title        = {Functional characterization of alzheimer’s disease genetic variants in microglia},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). APOBEC3B regulates r-loops and promotes
transcription-associated mutagenesis in cancer. <em>NG</em>,
<em>55</em>(10), 1721–1734. (<a
href="https://doi.org/10.1038/s41588-023-01504-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The single-stranded DNA cytosine-to-uracil deaminase APOBEC3B is an antiviral protein implicated in cancer. However, its substrates in cells are not fully delineated. Here APOBEC3B proteomics reveal interactions with a surprising number of R-loop factors. Biochemical experiments show APOBEC3B binding to R-loops in cells and in vitro. Genetic experiments demonstrate R-loop increases in cells lacking APOBEC3B and decreases in cells overexpressing APOBEC3B. Genome-wide analyses show major changes in the overall landscape of physiological and stimulus-induced R-loops with thousands of differentially altered regions, as well as binding of APOBEC3B to many of these sites. APOBEC3 mutagenesis impacts genes overexpressed in tumors and splice factor mutant tumors preferentially, and APOBEC3-attributed kataegis are enriched in RTCW motifs consistent with APOBEC3B deamination. Taken together with the fact that APOBEC3B binds single-stranded DNA and RNA and preferentially deaminates DNA, these results support a mechanism in which APOBEC3B regulates R-loops and contributes to R-loop mutagenesis in cancer. APOBEC3B interacts with R-loops and helps mediate their resolution in a deamination-dependent way. This association also renders R-loops susceptible to enhanced APOBEC3B-dependent mutagenesis.},
  archive      = {J_NG},
  author       = {McCann, Jennifer L. and Cristini, Agnese and Law, Emily K. and Lee, Seo Yun and Tellier, Michael and Carpenter, Michael A. and Beghè, Chiara and Kim, Jae Jin and Sanchez, Anthony and Jarvis, Matthew C. and Stefanovska, Bojana and Temiz, Nuri A. and Bergstrom, Erik N. and Salamango, Daniel J. and Brown, Margaret R. and Murphy, Shona and Alexandrov, Ludmil B. and Miller, Kyle M. and Gromak, Natalia and Harris, Reuben S.},
  doi          = {10.1038/s41588-023-01504-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1721-1734},
  shortjournal = {Nat. Genet.},
  title        = {APOBEC3B regulates R-loops and promotes transcription-associated mutagenesis in cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Systematic profiling of conditional pathway activation
identifies context-dependent synthetic lethalities. <em>NG</em>,
<em>55</em>(10), 1709–1720. (<a
href="https://doi.org/10.1038/s41588-023-01515-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The paradigm of cancer-targeted therapies has focused largely on inhibition of critical pathways in cancer. Conversely, conditional activation of signaling pathways as a new source of selective cancer vulnerabilities has not been deeply characterized. In this study, we sought to systematically identify context-specific gene-activation-induced lethalities in cancer. To this end, we developed a method for gain-of-function genetic perturbations simultaneously across ~500 barcoded cancer cell lines. Using this approach, we queried the pan-cancer vulnerability landscape upon activating ten key pathway nodes, revealing selective activation dependencies of MAPK and PI3K pathways associated with specific biomarkers. Notably, we discovered new pathway hyperactivation dependencies in subsets of APC-mutant colorectal cancers where further activation of the WNT pathway by APC knockdown or direct β-catenin overexpression led to robust antitumor effects in xenograft and patient-derived organoid models. Together, this study reveals a new class of conditional gene-activation dependencies in cancer. Gain-of-function perturbation screens across 488 barcoded cell lines identify context-specific activation lethalities. The authors show that cells with MAPK, PI3K and WNT pathway activation are vulnerable to mutations that lead to further activation, suggesting a new strategy for treating tumors driven by these oncogenic pathways.},
  archive      = {J_NG},
  author       = {Chang, Liang and Jung, Nancy Y. and Atari, Adel and Rodriguez, Diego J. and Kesar, Devishi and Song, Tian-Yu and Rees, Matthew G. and Ronan, Melissa and Li, Ruitong and Ruiz, Paloma and Chaturantabut, Saireudee and Ito, Takahiro and van Tienen, Laurens M. and Tseng, Yuen-Yi and Roth, Jennifer A. and Sellers, William R.},
  doi          = {10.1038/s41588-023-01515-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1709-1720},
  shortjournal = {Nat. Genet.},
  title        = {Systematic profiling of conditional pathway activation identifies context-dependent synthetic lethalities},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Molecular classification of hormone receptor-positive
HER2-negative breast cancer. <em>NG</em>, <em>55</em>(10), 1696–1708.
(<a href="https://doi.org/10.1038/s41588-023-01507-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hormone receptor-positive (HR+)/human epidermal growth factor receptor 2-negative (HER2−) breast cancer is the most prevalent type of breast cancer, in which endocrine therapy resistance and distant relapse remain unmet challenges. Accurate molecular classification is urgently required for guiding precision treatment. We established a large-scale multi-omics cohort of 579 patients with HR+/HER2− breast cancer and identified the following four molecular subtypes: canonical luminal, immunogenic, proliferative and receptor tyrosine kinase (RTK)-driven. Tumors of these four subtypes showed distinct biological and clinical features, suggesting subtype-specific therapeutic strategies. The RTK-driven subtype was characterized by the activation of the RTK pathways and associated with poor outcomes. The immunogenic subtype had enriched immune cells and could benefit from immune checkpoint therapy. In addition, we developed convolutional neural network models to discriminate these subtypes based on digital pathology for potential clinical translation. The molecular classification provides insights into molecular heterogeneity and highlights the potential for precision treatment of HR+/HER2− breast cancer. Multi-omic analysis of 579 patients with hormone receptor-positive (HR+)/human epidermal growth factor receptor 2-negative (HER2−) breast cancer identifies subgroups of the disease with distinct biological and clinical features.},
  archive      = {J_NG},
  author       = {Jin, Xi and Zhou, Yi-Fan and Ma, Ding and Zhao, Shen and Lin, Cai-Jin and Xiao, Yi and Fu, Tong and Liu, Cheng-Lin and Chen, Yi-Yu and Xiao, Wen-Xuan and Liu, Ya-Qing and Chen, Qing-Wang and Yu, Ying and Shi, Le-Ming and Shi, Jin-Xiu and Huang, Wei and Robertson, John F. R. and Jiang, Yi-Zhou and Shao, Zhi-Ming},
  doi          = {10.1038/s41588-023-01507-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1696-1708},
  shortjournal = {Nat. Genet.},
  title        = {Molecular classification of hormone receptor-positive HER2-negative breast cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mismatch repair deficiency is not sufficient to elicit tumor
immunogenicity. <em>NG</em>, <em>55</em>(10), 1686–1695. (<a
href="https://doi.org/10.1038/s41588-023-01499-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA mismatch repair deficiency (MMRd) is associated with a high tumor mutational burden (TMB) and sensitivity to immune checkpoint blockade (ICB) therapy. Nevertheless, most MMRd tumors do not durably respond to ICB and critical questions remain about immunosurveillance and TMB in these tumors. In the present study, we developed autochthonous mouse models of MMRd lung and colon cancer. Surprisingly, these models did not display increased T cell infiltration or ICB response, which we showed to be the result of substantial intratumor heterogeneity of mutations. Furthermore, we found that immunosurveillance shapes the clonal architecture but not the overall burden of neoantigens, and T cell responses against subclonal neoantigens are blunted. Finally, we showed that clonal, but not subclonal, neoantigen burden predicts ICB response in clinical trials of MMRd gastric and colorectal cancer. These results provide important context for understanding immune evasion in cancers with a high TMB and have major implications for therapies aimed at increasing TMB. Mouse models of lung and colorectal cancer with sporadic DNA mismatch repair deficiency clarify that the intratumor heterogeneity and clonal architecture rather than tumor mutational burden are powerful determinants of immunotherapy response.},
  archive      = {J_NG},
  author       = {Westcott, Peter M. K. and Muyas, Francesc and Hauck, Haley and Smith, Olivia C. and Sacks, Nathan J. and Ely, Zackery A. and Jaeger, Alex M. and Rideout, William M. and Zhang, Daniel and Bhutkar, Arjun and Beytagh, Mary C. and Canner, David A. and Jaramillo, Grissel C. and Bronson, Roderick T. and Naranjo, Santiago and Jin, Abbey and Patten, J. J. and Cruz, Amanda M. and Shanahan, Sean-Luc and Cortes-Ciriano, Isidro and Jacks, Tyler},
  doi          = {10.1038/s41588-023-01499-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1686-1695},
  shortjournal = {Nat. Genet.},
  title        = {Mismatch repair deficiency is not sufficient to elicit tumor immunogenicity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mutation rates and fitness consequences of mosaic
chromosomal alterations in blood. <em>NG</em>, <em>55</em>(10),
1677–1685. (<a
href="https://doi.org/10.1038/s41588-023-01490-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mosaic chromosomal alterations (mCAs) are common in cancers and can arise decades before diagnosis. A quantitative understanding of the rate at which these events occur, and their functional consequences, could improve cancer risk prediction and our understanding of somatic evolution. Using mCA clone size estimates from the blood of approximately 500,000 UK Biobank participants, we estimate mutation rates and fitness consequences of acquired gain, loss and copy-neutral loss of heterozygosity events. Most mCAs have moderate to high fitness effects but occur at a low rate, being more than tenfold less common than equivalently fit single-nucleotide variants. Notable exceptions are mosaic loss of X and Y, which we estimate have roughly 1,000-fold higher mutation rates than autosomal mCAs. Although the way in which most mCAs increase in prevalence with age is consistent with constant growth rates, some mCAs exhibit different behavior, suggesting that their fitness may depend on inherited variants, extrinsic factors or distributions of fitness effects. A survey of the fitness effects conferred by mosaic chromosomal alterations (mCAs) in UK Biobank shows that most mCAs—despite being relatively infrequent—are associated with increased fitness. Mosaic loss of the sex chromosomes was more common but these events afforded only small fitness gains.},
  archive      = {J_NG},
  author       = {Watson, Caroline J. and Blundell, Jamie R.},
  doi          = {10.1038/s41588-023-01490-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1677-1685},
  shortjournal = {Nat. Genet.},
  title        = {Mutation rates and fitness consequences of mosaic chromosomal alterations in blood},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multitissue H3K27ac profiling of GTEx samples links
epigenomic variation to disease. <em>NG</em>, <em>55</em>(10),
1665–1676. (<a
href="https://doi.org/10.1038/s41588-023-01509-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variants associated with complex traits are primarily noncoding, and their effects on gene-regulatory activity remain largely uncharacterized. To address this, we profile epigenomic variation of histone mark H3K27ac across 387 brain, heart, muscle and lung samples from Genotype-Tissue Expression (GTEx). We annotate 282 k active regulatory elements (AREs) with tissue-specific activity patterns. We identify 2,436 sex-biased AREs and 5,397 genetically influenced AREs associated with 130 k genetic variants (haQTLs) across tissues. We integrate genetic and epigenomic variation to provide mechanistic insights for disease-associated loci from 55 genome-wide association studies (GWAS), by revealing candidate tissues of action, driver SNPs and impacted AREs. Lastly, we build ARE–gene linking scores based on genetics (gLink scores) and demonstrate their unique ability to prioritize SNP–ARE–gene circuits. Overall, our epigenomic datasets, computational integration and mechanistic predictions provide valuable resources and important insights for understanding the molecular basis of human diseases/traits such as schizophrenia. Analysis of H3K27ac genome-wide maps across 387 brain, heart, muscle and lung samples, along with eQTL and genome-wide association studies (GWAS) data integration, identifies tissue-specific genetically influenced active regulatory regions and target genes that potentially mediate disease etiology.},
  archive      = {J_NG},
  author       = {Hou, Lei and Xiong, Xushen and Park, Yongjin and Boix, Carles and James, Benjamin and Sun, Na and He, Liang and Patel, Aman and Zhang, Zhizhuo and Molinie, Benoit and Van Wittenberghe, Nicholas and Steelman, Scott and Nusbaum, Chad and Aguet, François and Ardlie, Kristin G. and Kellis, Manolis},
  doi          = {10.1038/s41588-023-01509-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1665-1676},
  shortjournal = {Nat. Genet.},
  title        = {Multitissue H3K27ac profiling of GTEx samples links epigenomic variation to disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multi-ancestry genome-wide study identifies effector genes
and druggable pathways for coronary artery calcification. <em>NG</em>,
<em>55</em>(10), 1651–1664. (<a
href="https://doi.org/10.1038/s41588-023-01518-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coronary artery calcification (CAC), a measure of subclinical atherosclerosis, predicts future symptomatic coronary artery disease (CAD). Identifying genetic risk factors for CAC may point to new therapeutic avenues for prevention. Currently, there are only four known risk loci for CAC identified from genome-wide association studies (GWAS) in the general population. Here we conducted the largest multi-ancestry GWAS meta-analysis of CAC to date, which comprised 26,909 individuals of European ancestry and 8,867 individuals of African ancestry. We identified 11 independent risk loci, of which eight were new for CAC and five had not been reported for CAD. These new CAC loci are related to bone mineralization, phosphate catabolism and hormone metabolic pathways. Several new loci harbor candidate causal genes supported by multiple lines of functional evidence and are regulators of smooth muscle cell-mediated calcification ex vivo and in vitro. Together, these findings help refine the genetic architecture of CAC and extend our understanding of the biological and potential druggable pathways underlying CAC. Multi-ancestry genome-wide association meta-analysis identifies new risk loci for CAC. Functional evidence implicates candidate causal genes as regulators of smooth muscle cell-mediated calcification.},
  archive      = {J_NG},
  author       = {Kavousi, Maryam and Bos, Maxime M. and Barnes, Hanna J. and Lino Cardenas, Christian L. and Wong, Doris and Lu, Haojie and Hodonsky, Chani J. and Landsmeer, Lennart P. L. and Turner, Adam W. and Kho, Minjung and Hasbani, Natalie R. and de Vries, Paul S. and Bowden, Donald W. and Chopade, Sandesh and Deelen, Joris and Benavente, Ernest Diez and Guo, Xiuqing and Hofer, Edith and Hwang, Shih-Jen and Lutz, Sharon M. and Lyytikäinen, Leo-Pekka and Slenders, Lotte and Smith, Albert V. and Stanislawski, Maggie A. and van Setten, Jessica and Wong, Quenna and Yanek, Lisa R. and Becker, Diane M. and Beekman, Marian and Budoff, Matthew J. and Feitosa, Mary F. and Finan, Chris and Hilliard, Austin T. and Kardia, Sharon L. R. and Kovacic, Jason C. and Kral, Brian G. and Langefeld, Carl D. and Launer, Lenore J. and Malik, Shaista and Hoesein, Firdaus A. A. Mohamed and Mokry, Michal and Schmidt, Reinhold and Smith, Jennifer A. and Taylor, Kent D. and Terry, James G. and van der Grond, Jeroen and van Meurs, Joyce and Vliegenthart, Rozemarijn and Xu, Jianzhao and Young, Kendra A. and Zilhão, Nuno R. and Zweiker, Robert and Assimes, Themistocles L. and Becker, Lewis C. and Bos, Daniel and Carr, J. Jeffrey and Cupples, L. Adrienne and de Kleijn, Dominique P. v. and de Winther, Menno and den Ruijter, Hester M. and Fornage, Myriam and Freedman, Barry I. and Gudnason, Vilmundur and Hingorani, Aroon D. and Hokanson, John E. and Ikram, M. Arfan and Išgum, Ivana and Jacobs, David R. and Kähönen, Mika and Lange, Leslie A. and Lehtimäki, Terho and Pasterkamp, Gerard and Raitakari, Olli T. and Schmidt, Helena and Slagboom, P. Eline and Uitterlinden, André G. and Vernooij, Meike W. and Bis, Joshua C. and Franceschini, Nora and Psaty, Bruce M. and Post, Wendy S. and Rotter, Jerome I. and Björkegren, Johan L. M. and O’Donnell, Christopher J. and Bielak, Lawrence F. and Peyser, Patricia A. and Malhotra, Rajeev and van der Laan, Sander W. and Miller, Clint L.},
  doi          = {10.1038/s41588-023-01518-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1651-1664},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genome-wide study identifies effector genes and druggable pathways for coronary artery calcification},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide association meta-analysis identifies 17 loci
associated with nonalcoholic fatty liver disease. <em>NG</em>,
<em>55</em>(10), 1640–1650. (<a
href="https://doi.org/10.1038/s41588-023-01497-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nonalcoholic fatty liver disease (NAFLD) is common and partially heritable and has no effective treatments. We carried out a genome-wide association study (GWAS) meta-analysis of imaging (n = 66,814) and diagnostic code (3,584 cases versus 621,081 controls) measured NAFLD across diverse ancestries. We identified NAFLD-associated variants at torsin family 1 member B (TOR1B), fat mass and obesity associated (FTO), cordon-bleu WH2 repeat protein like 1 (COBLL1)/growth factor receptor-bound protein 14 (GRB14), insulin receptor (INSR), sterol regulatory element-binding transcription factor 1 (SREBF1) and patatin-like phospholipase domain-containing protein 2 (PNPLA2), as well as validated NAFLD-associated variants at patatin-like phospholipase domain-containing protein 3 (PNPLA3), transmembrane 6 superfamily 2 (TM6SF2), apolipoprotein E (APOE), glucokinase regulator (GCKR), tribbles homolog 1 (TRIB1), glycerol-3-phosphate acyltransferase (GPAM), mitochondrial amidoxime-reducing component 1 (MARC1), microsomal triglyceride transfer protein large subunit (MTTP), alcohol dehydrogenase 1B (ADH1B), transmembrane channel like 4 (TMC4)/membrane-bound O-acyltransferase domain containing 7 (MBOAT7) and receptor-type tyrosine-protein phosphatase δ (PTPRD). Implicated genes highlight mitochondrial, cholesterol and de novo lipogenesis as causally contributing to NAFLD predisposition. Phenome-wide association study (PheWAS) analyses suggest at least seven subtypes of NAFLD. Individuals in the top 10% and 1% of genetic risk have a 2.5-fold to 6-fold increased risk of NAFLD, cirrhosis and hepatocellular carcinoma. These genetic variants identify subtypes of NAFLD, improve estimates of disease risk and can guide the development of targeted therapeutics. Genome-wide association meta-analysis across individuals of diverse ancestries identifies risk loci for nonalcoholic fatty liver disease. The associated variants implicate plausible biological pathways and improve estimates of disease risk.},
  archive      = {J_NG},
  author       = {Chen, Yanhua and Du, Xiaomeng and Kuppa, Annapurna and Feitosa, Mary F. and Bielak, Lawrence F. and O’Connell, Jeffrey R. and Musani, Solomon K. and Guo, Xiuqing and Kahali, Bratati and Chen, Vincent L. and Smith, Albert V. and Ryan, Kathleen A. and Eirksdottir, Gudny and Allison, Matthew A. and Bowden, Donald W. and Budoff, Matthew J. and Carr, John Jeffrey and Chen, Yii-Der I. and Taylor, Kent D. and Oliveri, Antonino and Correa, Adolfo and Crudup, Breland F. and Kardia, Sharon L. R. and Mosley, Thomas H. and Norris, Jill M. and Terry, James G. and Rotter, Jerome I. and Wagenknecht, Lynne E. and Halligan, Brian D. and Young, Kendra A. and Hokanson, John E. and Washko, George R. and Gudnason, Vilmundur and Province, Michael A. and Peyser, Patricia A. and Palmer, Nicholette D. and Speliotes, Elizabeth K.},
  doi          = {10.1038/s41588-023-01497-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1640-1650},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analysis identifies 17 loci associated with nonalcoholic fatty liver disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Molecular basis for maternal inheritance of human
mitochondrial DNA. <em>NG</em>, <em>55</em>(10), 1632–1639. (<a
href="https://doi.org/10.1038/s41588-023-01505-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Uniparental inheritance of mitochondrial DNA (mtDNA) is an evolutionary trait found in nearly all eukaryotes. In many species, including humans, the sperm mitochondria are introduced to the oocyte during fertilization1,2. The mechanisms hypothesized to prevent paternal mtDNA transmission include ubiquitination of the sperm mitochondria and mitophagy3,4. However, the causative mechanisms of paternal mtDNA elimination have not been defined5,6. We found that mitochondria in human spermatozoa are devoid of intact mtDNA and lack mitochondrial transcription factor A (TFAM)—the major nucleoid protein required to protect, maintain and transcribe mtDNA. During spermatogenesis, sperm cells express an isoform of TFAM, which retains the mitochondrial presequence, ordinarily removed upon mitochondrial import. Phosphorylation of this presequence prevents mitochondrial import and directs TFAM to the spermatozoon nucleus. TFAM relocalization from the mitochondria of spermatogonia to the spermatozoa nucleus directly correlates with the elimination of mtDNA, thereby explaining maternal inheritance in this species. The mitochondrial transcription factor A is excluded from the mitochondria in spermatozoa by virtue of phosphorylation of the mitochondrial presequence. This is associated with transport to the nucleus and loss of mitochondrial DNA (mtDNA) from the mitochondria, providing a mechanistic basis for uniparental inheritance of mtDNA in humans.},
  archive      = {J_NG},
  author       = {Lee, William and Zamudio-Ochoa, Angelica and Buchel, Gina and Podlesniy, Petar and Marti Gutierrez, Nuria and Puigròs, Margalida and Calderon, Anna and Tang, Hsin-Yao and Li, Li and Mikhalchenko, Aleksei and Koski, Amy and Trullas, Ramon and Mitalipov, Shoukhrat and Temiakov, Dmitry},
  doi          = {10.1038/s41588-023-01505-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1632-1639},
  shortjournal = {Nat. Genet.},
  title        = {Molecular basis for maternal inheritance of human mitochondrial DNA},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Somatic SLC30A1 mutations altering zinc transporter ZnT1
cause aldosterone-producing adenomas and primary aldosteronism.
<em>NG</em>, <em>55</em>(10), 1623–1631. (<a
href="https://doi.org/10.1038/s41588-023-01498-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Primary aldosteronism (PA) is the most common form of endocrine hypertension and is characterized by inappropriately elevated aldosterone production via a renin-independent mechanism. Driver somatic mutations for aldosterone excess have been found in approximately 90% of aldosterone-producing adenomas (APAs). Other causes of lateralized adrenal PA include aldosterone-producing nodules (APNs). Using next-generation sequencing, we identified recurrent in-frame deletions in SLC30A1 in four APAs and one APN (p.L51_A57del, n = 3; p.L49_L55del, n = 2). SLC30A1 encodes the ubiquitous zinc efflux transporter ZnT1 (zinc transporter 1). The identified SLC30A1 variants are situated close to the zinc-binding site (His43 and Asp47) in transmembrane domain II and probably cause abnormal ion transport. Cases of PA with SLC30A1 mutations showed male dominance and demonstrated increased aldosterone and 18-oxocortisol concentrations. Functional studies of the SLC30A151_57del variant in a doxycycline-inducible adrenal cell system revealed pathological Na+ influx. An aberrant Na+ current led to depolarization of the resting membrane potential and, thus, to the opening of voltage-gated calcium (Ca2+) channels. This resulted in an increase in cytosolic Ca2+ activity, which stimulated CYP11B2 mRNA expression and aldosterone production. Collectively, these data implicate zinc transporter alterations as a dominant driver of aldosterone excess in PA. Somatic SLC30A1 mutations altering the zinc efflux transporter ZnT1 cause primary aldosteronism. These mutations result in membrane depolarization and opening of voltage-gated calcium channels, stimulating CYP11B2 expression and aldosterone production.},
  archive      = {J_NG},
  author       = {Rege, Juilee and Bandulik, Sascha and Nanba, Kazutaka and Kosmann, Carla and Blinder, Amy R. and Plain, Allein and Vats, Pankaj and Kumar-Sinha, Chandan and Lerario, Antonio M. and Else, Tobias and Yamazaki, Yuto and Satoh, Fumitoshi and Sasano, Hironobu and Giordano, Thomas J. and Williams, Tracy Ann and Reincke, Martin and Turcu, Adina F. and Udager, Aaron M. and Warth, Richard and Rainey, William E.},
  doi          = {10.1038/s41588-023-01498-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1623-1631},
  shortjournal = {Nat. Genet.},
  title        = {Somatic SLC30A1 mutations altering zinc transporter ZnT1 cause aldosterone-producing adenomas and primary aldosteronism},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Enhancing polygenic risk prediction in diverse populations:
Opportunities and challenges. <em>NG</em>, <em>55</em>(10), 1621–1622.
(<a href="https://doi.org/10.1038/s41588-023-01502-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polygenic risk scores (PRSs) are increasingly able to predict complex traits; however, they perform suboptimally in populations not of European ancestry. We present CT-SLEB, a powerful method that enables the calculation of PRSs from multi-ancestry samples and provides insights into the opportunities and challenges of enhancing polygenic risk prediction across populations of diverse ancestry.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01502-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1621-1622},
  shortjournal = {Nat. Genet.},
  title        = {Enhancing polygenic risk prediction in diverse populations: Opportunities and challenges},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rice heterosis: Quantitatively characterized and optimized
hybrid breeding. <em>NG</em>, <em>55</em>(10), 1619–1620. (<a
href="https://doi.org/10.1038/s41588-023-01496-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We re-sequenced and phenotyped 2,839 rice hybrid cultivars and 9,839 F2 individuals from elite hybrids. Based on the dataset, the genetic improvement during rice hybrid breeding was investigated, and the genetic basis underlying strong heterosis was quantitatively evaluated. Furthermore, a genomic selection model was constructed to optimize heterotic combinations.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01496-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1619-1620},
  shortjournal = {Nat. Genet.},
  title        = {Rice heterosis: Quantitatively characterized and optimized hybrid breeding},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Quantifying fitness effects and mutation rates of mCAs in
blood. <em>NG</em>, <em>55</em>(10), 1617–1618. (<a
href="https://doi.org/10.1038/s41588-023-01491-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using data from the UK Biobank, we reveal the roles of selection and mutation in shaping the genetic diversity of mosaic chromosomal alterations in healthy blood.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01491-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1617-1618},
  shortjournal = {Nat. Genet.},
  title        = {Quantifying fitness effects and mutation rates of mCAs in blood},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Linking non-coding variants to function in microglia in
alzheimer’s disease. <em>NG</em>, <em>55</em>(10), 1615–1616. (<a
href="https://doi.org/10.1038/s41588-023-01503-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies have associated thousands of non-coding variants with Alzheimer’s disease (AD), yet the functions of these variants remain elusive. We conducted cell-type-specific genetic fine mapping of AD variants and performed extensive functional characterization to unravel the causal variants that contribute to transcriptional regulation and AD­related phenotypes in microglia.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01503-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1615-1616},
  shortjournal = {Nat. Genet.},
  title        = {Linking non-coding variants to function in microglia in alzheimer’s disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Hyperactivation of oncogenic driver pathways as a precision
therapeutic strategy. <em>NG</em>, <em>55</em>(10), 1613–1614. (<a
href="https://doi.org/10.1038/s41588-023-01493-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many precision cancer therapies function by inhibiting oncogenic signaling pathways. A new study describes the counterintuitive finding that forced hyperactivation of the same pathways can also enable selective tumor targeting.},
  archive      = {J_NG},
  author       = {Wood, Kris C.},
  doi          = {10.1038/s41588-023-01493-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1613-1614},
  shortjournal = {Nat. Genet.},
  title        = {Hyperactivation of oncogenic driver pathways as a precision therapeutic strategy},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tumor heterogeneity impairs immunogenicity in mismatch
repair deficient tumors. <em>NG</em>, <em>55</em>(10), 1610–1612. (<a
href="https://doi.org/10.1038/s41588-023-01492-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA mismatch repair deficiency (MMRd) is associated with elevated tumor mutational burden (TMB) and exceptional immunotherapy responses, yet some patients experience no clinical benefit. Recent work proposes that high intra-tumoral heterogeneity can offset immunogenicity in sporadic MMRd, suggesting a potential mechanism of immunotherapy failure.},
  archive      = {J_NG},
  author       = {Reading, James L. and Caswell, Deborah R. and Swanton, Charles},
  doi          = {10.1038/s41588-023-01492-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1610-1612},
  shortjournal = {Nat. Genet.},
  title        = {Tumor heterogeneity impairs immunogenicity in mismatch repair deficient tumors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Neuronal chromatin meta-domains. <em>NG</em>,
<em>55</em>(10), 1609. (<a
href="https://doi.org/10.1038/s41588-023-01541-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01541-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1609},
  shortjournal = {Nat. Genet.},
  title        = {Neuronal chromatin meta-domains},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Saving endangered species with genomic prediction.
<em>NG</em>, <em>55</em>(10), 1609. (<a
href="https://doi.org/10.1038/s41588-023-01542-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01542-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1609},
  shortjournal = {Nat. Genet.},
  title        = {Saving endangered species with genomic prediction},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Protein structural alignment using deep learning.
<em>NG</em>, <em>55</em>(10), 1609. (<a
href="https://doi.org/10.1038/s41588-023-01543-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01543-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1609},
  shortjournal = {Nat. Genet.},
  title        = {Protein structural alignment using deep learning},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Calling SNVs in single cells. <em>NG</em>, <em>55</em>(10),
1609. (<a href="https://doi.org/10.1038/s41588-023-01544-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01544-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1609},
  shortjournal = {Nat. Genet.},
  title        = {Calling SNVs in single cells},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: HIC2 controls developmental hemoglobin
switching by repressing BCL11A transcription. <em>NG</em>,
<em>55</em>(9), 1608. (<a
href="https://doi.org/10.1038/s41588-023-01488-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Huang, Peng and Peslak, Scott A. and Ren, Ren and Khandros, Eugene and Qin, Kunhua and Keller, Cheryl A. and Giardine, Belinda and Bell, Henry W. and Lan, Xianjiang and Sharma, Malini and Horton, John R. and Abdulmalik, Osheiza and Chou, Stella T. and Shi, Junwei and Crossley, Merlin and Hardison, Ross C. and Cheng, Xiaodong and Blobel, Gerd A.},
  doi          = {10.1038/s41588-023-01488-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1608},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: HIC2 controls developmental hemoglobin switching by repressing BCL11A transcription},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Somatic mutations in facial skin from
countries of contrasting skin cancer risk. <em>NG</em>, <em>55</em>(9),
1608. (<a href="https://doi.org/10.1038/s41588-023-01508-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {King, Charlotte and Fowler, Joanna C. and Abnizova, Irina and Sood, Roshan K. and Hall, Michael W. J. and Szeverényi, Ildikó and Tham, Muly and Huang, Jingxiang and Young, Stephanie Ming and Hall, Benjamin A. and Birgitte Lane, E. and Jones, Philip H.},
  doi          = {10.1038/s41588-023-01508-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1608},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Somatic mutations in facial skin from countries of contrasting skin cancer risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). PhenoScore quantifies phenotypic variation for rare genetic
diseases by combining facial analysis with other clinical features using
a machine-learning framework. <em>NG</em>, <em>55</em>(9), 1598–1607.
(<a href="https://doi.org/10.1038/s41588-023-01469-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several molecular and phenotypic algorithms exist that establish genotype–phenotype correlations, including facial recognition tools. However, no unified framework that investigates both facial data and other phenotypic data directly from individuals exists. We developed PhenoScore: an open-source, artificial intelligence-based phenomics framework, combining facial recognition technology with Human Phenotype Ontology data analysis to quantify phenotypic similarity. Here we show PhenoScore’s ability to recognize distinct phenotypic entities by establishing recognizable phenotypes for 37 of 40 investigated syndromes against clinical features observed in individuals with other neurodevelopmental disorders and show it is an improvement on existing approaches. PhenoScore provides predictions for individuals with variants of unknown significance and enables sophisticated genotype–phenotype studies by testing hypotheses on possible phenotypic (sub)groups. PhenoScore confirmed previously known phenotypic subgroups caused by variants in the same gene for SATB1, SETBP1 and DEAF1 and provides objective clinical evidence for two distinct ADNP-related phenotypes, already established functionally. PhenoScore is an open-source machine-learning tool that combines facial image recognition with Human Phenotype Ontology for genetic syndrome identification without genomic data, with applications to subgroup analysis and variants of unknown significance classification.},
  archive      = {J_NG},
  author       = {Dingemans, Alexander J. M. and Hinne, Max and Truijen, Kim M. G. and Goltstein, Lia and van Reeuwijk, Jeroen and de Leeuw, Nicole and Schuurs-Hoeijmakers, Janneke and Pfundt, Rolph and Diets, Illja J. and den Hoed, Joery and de Boer, Elke and Coenen-van der Spek, Jet and Jansen, Sandra and van Bon, Bregje W. and Jonis, Noraly and Ockeloen, Charlotte W. and Vulto-van Silfhout, Anneke T. and Kleefstra, Tjitske and Koolen, David A. and Campeau, Philippe M. and Palmer, Elizabeth E. and Van Esch, Hilde and Lyon, Gholson J. and Alkuraya, Fowzan S. and Rauch, Anita and Marom, Ronit and Baralle, Diana and van der Sluijs, Pleuntje J. and Santen, Gijs W. E. and Kooy, R. Frank and van Gerven, Marcel A. J. and Vissers, Lisenka E. L. M. and de Vries, Bert B. A.},
  doi          = {10.1038/s41588-023-01469-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1598-1607},
  shortjournal = {Nat. Genet.},
  title        = {PhenoScore quantifies phenotypic variation for rare genetic diseases by combining facial analysis with other clinical features using a machine-learning framework},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GATK-gCNV enables the discovery of rare copy number variants
from exome sequencing data. <em>NG</em>, <em>55</em>(9), 1589–1597. (<a
href="https://doi.org/10.1038/s41588-023-01449-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Copy number variants (CNVs) are major contributors to genetic diversity and disease. While standardized methods, such as the genome analysis toolkit (GATK), exist for detecting short variants, technical challenges have confounded uniform large-scale CNV analyses from whole-exome sequencing (WES) data. Given the profound impact of rare and de novo coding CNVs on genome organization and human disease, we developed GATK-gCNV, a flexible algorithm to discover rare CNVs from sequencing read-depth information, complete with open-source distribution via GATK. We benchmarked GATK-gCNV in 7,962 exomes from individuals in quartet families with matched genome sequencing and microarray data, finding up to 95% recall of rare coding CNVs at a resolution of more than two exons. We used GATK-gCNV to generate a reference catalog of rare coding CNVs in WES data from 197,306 individuals in the UK Biobank, and observed strong correlations between per-gene CNV rates and measures of mutational constraint, as well as rare CNV associations with multiple traits. In summary, GATK-gCNV is a tunable approach for sensitive and specific CNV discovery in WES data, with broad applications. GATK-gCNV uses a probabilistic model and inference framework to discover rare copy number variants (CNVs) from sequencing read-depth information. This algorithm is used to generate a reference catalog of rare coding CNVs in exome sequencing data from UK Biobank.},
  archive      = {J_NG},
  author       = {Babadi, Mehrtash and Fu, Jack M. and Lee, Samuel K. and Smirnov, Andrey N. and Gauthier, Laura D. and Walker, Mark and Benjamin, David I. and Zhao, Xuefang and Karczewski, Konrad J. and Wong, Isaac and Collins, Ryan L. and Sanchis-Juan, Alba and Brand, Harrison and Banks, Eric and Talkowski, Michael E.},
  doi          = {10.1038/s41588-023-01449-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1589-1597},
  shortjournal = {Nat. Genet.},
  title        = {GATK-gCNV enables the discovery of rare copy number variants from exome sequencing data},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Solanum americanum genome-assisted discovery of immune
receptors that detect potato late blight pathogen effectors.
<em>NG</em>, <em>55</em>(9), 1579–1588. (<a
href="https://doi.org/10.1038/s41588-023-01486-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Potato (Solanum tuberosum) and tomato (Solanum lycopersicon) crops suffer severe losses to late blight caused by the oomycete pathogen Phytophthora infestans. Solanum americanum, a relative of potato and tomato, is globally distributed and most accessions are highly blight resistant. We generated high-quality reference genomes of four S. americanum accessions, resequenced 52 accessions, and defined a pan-NLRome of S. americanum immune receptor genes. We further screened for variation in recognition of 315P. infestans RXLR effectors in 52 S. americanum accessions. Using these genomic and phenotypic data, we cloned three NLR-encoding genes, Rpi-amr4, R02860 and R04373, that recognize cognate P. infestans RXLR effectors PITG_22825 (AVRamr4), PITG_02860 and PITG_04373. These genomic resources and methodologies will support efforts to engineer potatoes with durable late blight resistance and can be applied to diseases of other crops. High-quality genome assemblies of four Solanum Americanum accessions lead to the identification of three NLR-encoding genes, Rpi-amr4, R02860 and R04373, that recognize potato late blight pathogen Phytophthora infestans effectors.},
  archive      = {J_NG},
  author       = {Lin, Xiao and Jia, Yuxin and Heal, Robert and Prokchorchik, Maxim and Sindalovskaya, Maria and Olave-Achury, Andrea and Makechemu, Moffat and Fairhead, Sebastian and Noureen, Azka and Heo, Jung and Witek, Kamil and Smoker, Matthew and Taylor, Jodie and Shrestha, Ram-Krishna and Lee, Yoonyoung and Zhang, Chunzhi and Park, Soon Ju and Sohn, Kee Hoon and Huang, Sanwen and Jones, Jonathan D. G.},
  doi          = {10.1038/s41588-023-01486-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1579-1588},
  shortjournal = {Nat. Genet.},
  title        = {Solanum americanum genome-assisted discovery of immune receptors that detect potato late blight pathogen effectors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Symmetric inheritance of parental histones governs epigenome
maintenance and embryonic stem cell identity. <em>NG</em>,
<em>55</em>(9), 1567–1578. (<a
href="https://doi.org/10.1038/s41588-023-01476-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modified parental histones are segregated symmetrically to daughter DNA strands during replication and can be inherited through mitosis. How this may sustain the epigenome and cell identity remains unknown. Here we show that transmission of histone-based information during DNA replication maintains epigenome fidelity and embryonic stem cell plasticity. Asymmetric segregation of parental histones H3–H4 in MCM2-2A mutants compromised mitotic inheritance of histone modifications and globally altered the epigenome. This included widespread spurious deposition of repressive modifications, suggesting elevated epigenetic noise. Moreover, H3K9me3 loss at repeats caused derepression and H3K27me3 redistribution across bivalent promoters correlated with misexpression of developmental genes. MCM2-2A mutation challenged dynamic transitions in cellular states across the cell cycle, enhancing naïve pluripotency and reducing lineage priming in G1. Furthermore, developmental competence was diminished, correlating with impaired exit from pluripotency. Collectively, this argues that epigenetic inheritance of histone modifications maintains a correctly balanced and dynamic chromatin landscape able to support mammalian cell differentiation. Asymmetric segregation of parental histones H3 and H4 in MCM2-mutant embryonic stem cells impacts mitotic inheritance of histone modifications and genome regulation. MCM2-2A mutation perturbs exit from pluripotency and differentiation.},
  archive      = {J_NG},
  author       = {Wenger, Alice and Biran, Alva and Alcaraz, Nicolas and Redó-Riveiro, Alba and Sell, Annika Charlotte and Krautz, Robert and Flury, Valentin and Reverón-Gómez, Nazaret and Solis-Mezarino, Victor and Völker-Albert, Moritz and Imhof, Axel and Andersson, Robin and Brickman, Joshua M. and Groth, Anja},
  doi          = {10.1038/s41588-023-01476-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1567-1578},
  shortjournal = {Nat. Genet.},
  title        = {Symmetric inheritance of parental histones governs epigenome maintenance and embryonic stem cell identity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Symmetric inheritance of parental histones contributes to
safeguarding the fate of mouse embryonic stem cells during
differentiation. <em>NG</em>, <em>55</em>(9), 1555–1566. (<a
href="https://doi.org/10.1038/s41588-023-01477-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Parental histones, the carriers of posttranslational modifications, are deposited evenly onto the replicating DNA of sister chromatids in a process dependent on the Mcm2 subunit of DNA helicase and the Pole3 subunit of leading-strand DNA polymerase. The biological significance of parental histone propagation remains unclear. Here we show that Mcm2-mutated or Pole3-deleted mouse embryonic stem cells (ESCs) display aberrant histone landscapes and impaired neural differentiation. Mutation of the Mcm2 histone-binding domain causes defects in pre-implantation development and embryonic lethality. ESCs with biased parental histone transfer exhibit increased epigenetic heterogeneity, showing altered histone variant H3.3 and H3K27me3 patterning at genomic sites regulating differentiation genes. Our results indicate that the lagging strand pattern of H3.3 leads to the redistribution of H3K27me3 in Mcm2-2A ESCs. We demonstrate that symmetric parental histone deposition to sister chromatids contributes to cellular differentiation and development. Mcm2 mutation or Pole3 deletion in mouse embryonic stem cells leads to asymmetric parental histone distribution and impaired neural differentiation. Mutation of the Mcm2 histone-binding domain causes defects in pre-implantation development and embryonic lethality.},
  archive      = {J_NG},
  author       = {Wen, Qing and Zhou, Jiaqi and Tian, Congcong and Li, Xinran and Song, Guibing and Gao, Yuan and Sun, Yaping and Ma, Chiyuan and Yao, Sitong and Liang, Xiaoyan and Kang, Xing and Wang, Nan and Yao, Yuan and Wang, Hongbao and Liang, Xiaohuan and Tang, Jialin and Offer, Steven M. and Lei, Xiaohua and Yu, Chuanhe and Liu, Xiangyu and Liu, Zichuan and Wang, Zhiquan and Gan, Haiyun},
  doi          = {10.1038/s41588-023-01477-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1555-1566},
  shortjournal = {Nat. Genet.},
  title        = {Symmetric inheritance of parental histones contributes to safeguarding the fate of mouse embryonic stem cells during differentiation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). In vivo screening characterizes chromatin factor functions
during normal and malignant hematopoiesis. <em>NG</em>, <em>55</em>(9),
1542–1554. (<a
href="https://doi.org/10.1038/s41588-023-01471-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular differentiation requires extensive alterations in chromatin structure and function, which is elicited by the coordinated action of chromatin and transcription factors. By contrast with transcription factors, the roles of chromatin factors in differentiation have not been systematically characterized. Here, we combine bulk ex vivo and single-cell in vivo CRISPR screens to characterize the role of chromatin factor families in hematopoiesis. We uncover marked lineage specificities for 142 chromatin factors, revealing functional diversity among related chromatin factors (i.e. barrier-to-autointegration factor subcomplexes) as well as shared roles for unrelated repressive complexes that restrain excessive myeloid differentiation. Using epigenetic profiling, we identify functional interactions between lineage-determining transcription factors and several chromatin factors that explain their lineage dependencies. Studying chromatin factor functions in leukemia, we show that leukemia cells engage homeostatic chromatin factor functions to block differentiation, generating specific chromatin factor–transcription factor interactions that might be therapeutically targeted. Together, our work elucidates the lineage-determining properties of chromatin factors across normal and malignant hematopoiesis. Bulk ex vivo and single-cell in vivo CRISPR knockout screens are used to characterize 680 chromatin factors during mouse hematopoiesis, highlighting lineage-specific and normal and leukemia-specific functions.},
  archive      = {J_NG},
  author       = {Lara-Astiaso, David and Goñi-Salaverri, Ainhoa and Mendieta-Esteban, Julen and Narayan, Nisha and Del Valle, Cynthia and Gross, Torsten and Giotopoulos, George and Beinortas, Tumas and Navarro-Alonso, Mar and Aguado-Alvaro, Laura Pilar and Zazpe, Jon and Marchese, Francesco and Torrea, Natalia and Calvo, Isabel A. and Lopez, Cecile K. and Alignani, Diego and Lopez, Aitziber and Saez, Borja and Taylor-King, Jake P. and Prosper, Felipe and Fortelny, Nikolaus and Huntly, Brian J. P.},
  doi          = {10.1038/s41588-023-01471-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1542-1554},
  shortjournal = {Nat. Genet.},
  title        = {In vivo screening characterizes chromatin factor functions during normal and malignant hematopoiesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell multi-omics identifies chronic inflammation as a
driver of TP53-mutant leukemic evolution. <em>NG</em>, <em>55</em>(9),
1531–1541. (<a
href="https://doi.org/10.1038/s41588-023-01480-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the genetic and nongenetic determinants of tumor protein 53 (TP53)-mutation-driven clonal evolution and subsequent transformation is a crucial step toward the design of rational therapeutic strategies. Here we carry out allelic resolution single-cell multi-omic analysis of hematopoietic stem/progenitor cells (HSPCs) from patients with a myeloproliferative neoplasm who transform to TP53-mutant secondary acute myeloid leukemia (sAML). All patients showed dominant TP53 ‘multihit’ HSPC clones at transformation, with a leukemia stem cell transcriptional signature strongly predictive of adverse outcomes in independent cohorts, across both TP53-mutant and wild-type (WT) AML. Through analysis of serial samples, antecedent TP53-heterozygous clones and in vivo perturbations, we demonstrate a hitherto unrecognized effect of chronic inflammation, which suppressed TP53 WT HSPCs while enhancing the fitness advantage of TP53-mutant cells and promoted genetic evolution. Our findings will facilitate the development of risk-stratification, early detection and treatment strategies for TP53-mutant leukemia, and are of broad relevance to other cancer types. Single-cell multi-omic analyses show that chronic inflammation contributes to myeloproliferative neoplasm transformation to secondary acute myeloid leukemia by enhancing tumor protein 53 (TP53) mutant cell fitness and genetic evolution.},
  archive      = {J_NG},
  author       = {Rodriguez-Meira, Alba and Norfo, Ruggiero and Wen, Sean and Chédeville, Agathe L. and Rahman, Haseeb and O’Sullivan, Jennifer and Wang, Guanlin and Louka, Eleni and Kretzschmar, Warren W. and Paterson, Aimee and Brierley, Charlotte and Martin, Jean-Edouard and Demeule, Caroline and Bashton, Matthew and Sousos, Nikolaos and Moralli, Daniela and Subha Meem, Lamia and Carrelha, Joana and Wu, Bishan and Hamblin, Angela and Guermouche, Helene and Pasquier, Florence and Marzac, Christophe and Girodon, François and Vainchenker, William and Drummond, Mark and Harrison, Claire and Chapman, J. Ross and Plo, Isabelle and Jacobsen, Sten Eirik W. and Psaila, Bethan and Thongjuea, Supat and Antony-Debré, Iléana and Mead, Adam J.},
  doi          = {10.1038/s41588-023-01480-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1531-1541},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-omics identifies chronic inflammation as a driver of TP53-mutant leukemic evolution},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Multiparameter prediction of myeloid neoplasia risk.
<em>NG</em>, <em>55</em>(9), 1523–1530. (<a
href="https://doi.org/10.1038/s41588-023-01472-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The myeloid neoplasms encompass acute myeloid leukemia, myelodysplastic syndromes and myeloproliferative neoplasms. Most cases arise from the shared ancestor of clonal hematopoiesis (CH). Here we analyze data from 454,340 UK Biobank participants, of whom 1,808 developed a myeloid neoplasm 0–15 years after recruitment. We describe the differences in CH mutational landscapes and hematology/biochemistry test parameters among individuals that later develop myeloid neoplasms (pre-MN) versus controls, finding that disease-specific changes are detectable years before diagnosis. By analyzing differences between ‘pre-MN’ and controls, we develop and validate Cox regression models quantifying the risk of progression to each myeloid neoplasm subtype. We construct ‘MN-predict’, a web application that generates time-dependent predictions with the input of basic blood tests and genetic data. Our study demonstrates that many individuals that develop myeloid neoplasms can be identified years in advance and provides a framework for disease-specific prognostication that will be of substantial use to researchers and physicians. An analysis of UK Biobank participants shows that the risk of developing different types of myeloid neoplasms can be inferred years before diagnosis. The authors integrate somatic gene mutations with blood test parameters into a predictive model, which could guide future strategies for early detection and prevention of these diseases.},
  archive      = {J_NG},
  author       = {Gu, Muxin and Kovilakam, Sruthi Cheloor and Dunn, William G. and Marando, Ludovica and Barcena, Clea and Mohorianu, Irina and Smith, Alexandra and Kar, Siddhartha P. and Fabre, Margarete A. and Gerstung, Moritz and Cargo, Catherine A. and Malcovati, Luca and Quiros, Pedro M. and Vassiliou, George S.},
  doi          = {10.1038/s41588-023-01472-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1523-1530},
  shortjournal = {Nat. Genet.},
  title        = {Multiparameter prediction of myeloid neoplasia risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide prediction of disease variant effects with a
deep protein language model. <em>NG</em>, <em>55</em>(9), 1512–1522. (<a
href="https://doi.org/10.1038/s41588-023-01465-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the effects of coding variants is a major challenge. While recent deep-learning models have improved variant effect prediction accuracy, they cannot analyze all coding variants due to dependency on close homologs or software limitations. Here we developed a workflow using ESM1b, a 650-million-parameter protein language model, to predict all ~450 million possible missense variant effects in the human genome, and made all predictions available on a web portal. ESM1b outperformed existing methods in classifying ~150,000 ClinVar/HGMD missense variants as pathogenic or benign and predicting measurements across 28 deep mutational scan datasets. We further annotated ~2 million variants as damaging only in specific protein isoforms, demonstrating the importance of considering all isoforms when predicting variant effects. Our approach also generalizes to more complex coding variants such as in-frame indels and stop-gains. Together, these results establish protein language models as an effective, accurate and general approach to predicting variant effects. A modified framework leveraging a protein language model (ESM1b) is used to predict all possible 450 million missense variant effects in the human genome and shows potential for generalizing to more complex genetic variations such as indels and stop-gains.},
  archive      = {J_NG},
  author       = {Brandes, Nadav and Goldman, Grant and Wang, Charlotte H. and Ye, Chun Jimmie and Ntranos, Vasilis},
  doi          = {10.1038/s41588-023-01465-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1512-1522},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide prediction of disease variant effects with a deep protein language model},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Modeling tissue co-regulation estimates tissue-specific
contributions to disease. <em>NG</em>, <em>55</em>(9), 1503–1511. (<a
href="https://doi.org/10.1038/s41588-023-01474-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrative analyses of genome-wide association studies and gene expression data have implicated many disease-critical tissues. However, co-regulation of genetic effects on gene expression across tissues impedes distinguishing biologically causal tissues from tagging tissues. In the present study, we introduce tissue co-regulation score regression (TCSC), which disentangles causal tissues from tagging tissues by regressing gene–disease association statistics (from transcriptome-wide association studies) on tissue co-regulation scores, reflecting correlations of predicted gene expression across genes and tissues. We applied TCSC to 78 diseases/traits (average n = 302,000) and gene expression prediction models for 48 GTEx tissues. TCSC identified 21 causal tissue–trait pairs at a 5% false discovery rate (FDR), including well-established findings, biologically plausible new findings (for example, aorta artery and glaucoma) and increased specificity of known tissue–trait associations (for example, subcutaneous adipose, but not visceral adipose, and high-density lipoprotein). TCSC also identified 17 causal tissue–trait covariance pairs at 5% FDR. In conclusion, TCSC is a precise method for distinguishing causal tissues from tagging tissues. Tissue co-regulation score regression (TCSC) infers causal tissues and partitions trait heritability into tissue-specific components using a transcriptome-wide association study framework. Applying TCSC to 78 complex traits and diseases identifies biologically plausible tissue–trait relationships.},
  archive      = {J_NG},
  author       = {Amariuta, Tiffany and Siewert-Rocks, Katherine and Price, Alkes L.},
  doi          = {10.1038/s41588-023-01474-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1503-1511},
  shortjournal = {Nat. Genet.},
  title        = {Modeling tissue co-regulation estimates tissue-specific contributions to disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Extremely sparse models of linkage disequilibrium in
ancestrally diverse association studies. <em>NG</em>, <em>55</em>(9),
1494–1502. (<a
href="https://doi.org/10.1038/s41588-023-01487-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Linkage disequilibrium (LD) is the correlation among nearby genetic variants. In genetic association studies, LD is often modeled using large correlation matrices, but this approach is inefficient, especially in ancestrally diverse studies. In the present study, we introduce LD graphical models (LDGMs), which are an extremely sparse and efficient representation of LD. LDGMs are derived from genome-wide genealogies; statistical relationships among alleles in the LDGM correspond to genealogical relationships among haplotypes. We published LDGMs and ancestry-specific LDGM precision matrices for 18 million common variants (minor allele frequency &amp;gt;1%) in five ancestry groups, validated their accuracy and demonstrated order-of-magnitude improvements in runtime for commonly used LD matrix computations. We implemented an extremely fast multiancestry polygenic prediction method, BLUPx-ldgm, which performs better than a similar method based on the reference LD correlation matrix. LDGMs will enable sophisticated methods that scale to ancestrally diverse genetic association data across millions of variants and individuals. Linkage disequilibrium graphical models (LDGMs) derived from genome-wide genealogies provide an efficient representation of LD, yielding large improvements in runtime for LD matrix computations. LDGMs will enable methods that scale to millions of variants and individuals.},
  archive      = {J_NG},
  author       = {Salehi Nowbandegani, Pouria and Wohns, Anthony Wilder and Ballard, Jenna L. and Lander, Eric S. and Bloemendal, Alex and Neale, Benjamin M. and O’Connor, Luke J.},
  doi          = {10.1038/s41588-023-01487-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1494-1502},
  shortjournal = {Nat. Genet.},
  title        = {Extremely sparse models of linkage disequilibrium in ancestrally diverse association studies},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetic insights into human cortical organization and
development through genome-wide analyses of 2,347 neuroimaging
phenotypes. <em>NG</em>, <em>55</em>(9), 1483–1493. (<a
href="https://doi.org/10.1038/s41588-023-01475-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our understanding of the genetics of the human cerebral cortex is limited both in terms of the diversity and the anatomical granularity of brain structural phenotypes. Here we conducted a genome-wide association meta-analysis of 13 structural and diffusion magnetic resonance imaging-derived cortical phenotypes, measured globally and at 180 bilaterally averaged regions in 36,663 individuals and identified 4,349 experiment-wide significant loci. These phenotypes include cortical thickness, surface area, gray matter volume, measures of folding, neurite density and water diffusion. We identified four genetic latent structures and causal relationships between surface area and some measures of cortical folding. These latent structures partly relate to different underlying gene expression trajectories during development and are enriched for different cell types. We also identified differential enrichment for neurodevelopmental and constrained genes and demonstrate that common genetic variants associated with cortical expansion are associated with cephalic disorders. Finally, we identified complex interphenotype and inter-regional genetic relationships among the 13 phenotypes, reflecting the developmental differences among them. Together, these analyses identify distinct genetic organizational principles of the cortex and their correlates with neurodevelopment. Genome-wide association analyses of magnetic resonance imaging data describe the genetic architecture of 13 cortical phenotypes at both global and regional levels, implicating neurodevelopmental and constrained genes.},
  archive      = {J_NG},
  author       = {Warrier, Varun and Stauffer, Eva-Maria and Huang, Qin Qin and Wigdor, Emilie M. and Slob, Eric A. W. and Seidlitz, Jakob and Ronan, Lisa and Valk, Sofie L. and Mallard, Travis T. and Grotzinger, Andrew D. and Romero-Garcia, Rafael and Baron-Cohen, Simon and Geschwind, Daniel H. and Lancaster, Madeline A. and Murray, Graham K. and Gandal, Michael J. and Alexander-Bloch, Aaron and Won, Hyejung and Martin, Hilary C. and Bullmore, Edward T. and Bethlehem, Richard A. I.},
  doi          = {10.1038/s41588-023-01475-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1483-1493},
  shortjournal = {Nat. Genet.},
  title        = {Genetic insights into human cortical organization and development through genome-wide analyses of 2,347 neuroimaging phenotypes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GWAS meta-analysis of over 29,000 people with epilepsy
identifies 26 risk loci and subtype-specific genetic architecture.
<em>NG</em>, <em>55</em>(9), 1471–1482. (<a
href="https://doi.org/10.1038/s41588-023-01485-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epilepsy is a highly heritable disorder affecting over 50 million people worldwide, of which about one-third are resistant to current treatments. Here we report a multi-ancestry genome-wide association study including 29,944 cases, stratified into three broad categories and seven subtypes of epilepsy, and 52,538 controls. We identify 26 genome-wide significant loci, 19 of which are specific to genetic generalized epilepsy (GGE). We implicate 29 likely causal genes underlying these 26 loci. SNP-based heritability analyses show that common variants explain between 39.6% and 90% of genetic risk for GGE and its subtypes. Subtype analysis revealed markedly different genetic architectures between focal and generalized epilepsies. Gene-set analyses of GGE signals implicate synaptic processes in both excitatory and inhibitory neurons in the brain. Prioritized candidate genes overlap with monogenic epilepsy genes and with targets of current antiseizure medications. Finally, we leverage our results to identify alternate drugs with predicted efficacy if repurposed for epilepsy treatment. Genome-wide association meta-analyses identify 26 risk loci for epilepsy, including 19 loci specific to genetic generalized epilepsy. Prioritized candidate genes implicate synaptic processes and overlap with targets of antiseizure medications.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01485-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1471-1482},
  shortjournal = {Nat. Genet.},
  title        = {GWAS meta-analysis of over 29,000 people with epilepsy identifies 26 risk loci and subtype-specific genetic architecture},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide analysis of a model-derived binge eating
disorder phenotype identifies risk loci and implicates iron metabolism.
<em>NG</em>, <em>55</em>(9), 1462–1470. (<a
href="https://doi.org/10.1038/s41588-023-01464-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Binge eating disorder (BED) is the most common eating disorder, yet its genetic architecture remains largely unknown. Studying BED is challenging because it is often comorbid with obesity, a common and highly polygenic trait, and it is underdiagnosed in biobank data sets. To address this limitation, we apply a supervised machine-learning approach (using 822 cases of individuals diagnosed with BED) to estimate the probability of each individual having BED based on electronic medical records from the Million Veteran Program. We perform a genome-wide association study of individuals of African (n = 77,574) and European (n = 285,138) ancestry while controlling for body mass index to identify three independent loci near the HFE, MCHR2 and LRP11 genes and suggest APOE as a risk gene for BED. We identify shared heritability between BED and several neuropsychiatric traits, and implicate iron metabolism in the pathophysiology of BED. Overall, our findings provide insights into the genetics underlying BED and suggest directions for future translational research. Genome-wide association analysis of a binge eating disorder phenotype derived from a supervised machine-learning model applied to electronic medical records identifies three risk loci for this disorder and implicates iron metabolism in its etiology.},
  archive      = {J_NG},
  author       = {Burstein, David and Griffen, Trevor C. and Therrien, Karen and Bendl, Jaroslav and Venkatesh, Sanan and Dong, Pengfei and Modabbernia, Amirhossein and Zeng, Biao and Mathur, Deepika and Hoffman, Gabriel and Sysko, Robyn and Hildebrandt, Tom and Voloudakis, Georgios and Roussos, Panos},
  doi          = {10.1038/s41588-023-01464-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1462-1470},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analysis of a model-derived binge eating disorder phenotype identifies risk loci and implicates iron metabolism},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GWAS of random glucose in 476,326 individuals provide
insights into diabetes pathophysiology, complications and treatment
stratification. <em>NG</em>, <em>55</em>(9), 1448–1461. (<a
href="https://doi.org/10.1038/s41588-023-01462-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conventional measurements of fasting and postprandial blood glucose levels investigated in genome-wide association studies (GWAS) cannot capture the effects of DNA variability on ‘around the clock’ glucoregulatory processes. Here we show that GWAS meta-analysis of glucose measurements under nonstandardized conditions (random glucose (RG)) in 476,326 individuals of diverse ancestries and without diabetes enables locus discovery and innovative pathophysiological observations. We discovered 120 RG loci represented by 150 distinct signals, including 13 with sex-dimorphic effects, two cross-ancestry and seven rare frequency signals. Of these, 44 loci are new for glycemic traits. Regulatory, glycosylation and metagenomic annotations highlight ileum and colon tissues, indicating an underappreciated role of the gastrointestinal tract in controlling blood glucose. Functional follow-up and molecular dynamics simulations of lower frequency coding variants in glucagon-like peptide-1 receptor (GLP1R), a type 2 diabetes treatment target, reveal that optimal selection of GLP-1R agonist therapy will benefit from tailored genetic stratification. We also provide evidence from Mendelian randomization that lung function is modulated by blood glucose and that pulmonary dysfunction is a diabetes complication. Our investigation yields new insights into the biology of glucose regulation, diabetes complications and pathways for treatment stratification. Genome-wide association analyses of blood glucose measurements under nonstandardized conditions provide insights into the biology of glucose regulation, diabetes complications and pathways for treatment stratification.},
  archive      = {J_NG},
  author       = {Lagou, Vasiliki and Jiang, Longda and Ulrich, Anna and Zudina, Liudmila and González, Karla Sofia Gutiérrez and Balkhiyarova, Zhanna and Faggian, Alessia and Maina, Jared G. and Chen, Shiqian and Todorov, Petar V. and Sharapov, Sodbo and David, Alessia and Marullo, Letizia and Mägi, Reedik and Rujan, Roxana-Maria and Ahlqvist, Emma and Thorleifsson, Gudmar and Gao, Ηe and Εvangelou, Εvangelos and Benyamin, Beben and Scott, Robert A. and Isaacs, Aaron and Zhao, Jing Hua and Willems, Sara M. and Johnson, Toby and Gieger, Christian and Grallert, Harald and Meisinger, Christa and Müller-Nurasyid, Martina and Strawbridge, Rona J. and Goel, Anuj and Rybin, Denis and Albrecht, Eva and Jackson, Anne U. and Stringham, Heather M. and Corrêa, Ivan R. and Farber-Eger, Eric and Steinthorsdottir, Valgerdur and Uitterlinden, André G. and Munroe, Patricia B. and Brown, Morris J. and Schmidberger, Julian and Holmen, Oddgeir and Thorand, Barbara and Hveem, Kristian and Wilsgaard, Tom and Mohlke, Karen L. and Wang, Zhe and Shmeliov, Aleksey and den Hoed, Marcel and Loos, Ruth J. F. and Kratzer, Wolfgang and Haenle, Mark and Koenig, Wolfgang and Boehm, Bernhard O. and Tan, Tricia M. and Tomas, Alejandra and Salem, Victoria and Barroso, Inês and Tuomilehto, Jaakko and Boehnke, Michael and Florez, Jose C. and Hamsten, Anders and Watkins, Hugh and Njølstad, Inger and Wichmann, H.-Erich and Caulfield, Mark J. and Khaw, Kay-Tee and van Duijn, Cornelia M. and Hofman, Albert and Wareham, Nicholas J. and Langenberg, Claudia and Whitfield, John B. and Martin, Nicholas G. and Montgomery, Grant and Scapoli, Chiara and Tzoulaki, Ioanna and Elliott, Paul and Thorsteinsdottir, Unnur and Stefansson, Kari and Brittain, Evan L. and McCarthy, Mark I. and Froguel, Philippe and Sexton, Patrick M. and Wootten, Denise and Groop, Leif and Dupuis, Josée and Meigs, James B. and Deganutti, Giuseppe and Demirkan, Ayse and Pers, Tune H. and Reynolds, Christopher A. and Aulchenko, Yurii S. and Kaakinen, Marika A. and Jones, Ben and Prokopenko, Inga},
  doi          = {10.1038/s41588-023-01462-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1448-1461},
  shortjournal = {Nat. Genet.},
  title        = {GWAS of random glucose in 476,326 individuals provide insights into diabetes pathophysiology, complications and treatment stratification},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Somatic mutations in facial skin from countries of
contrasting skin cancer risk. <em>NG</em>, <em>55</em>(9), 1440–1447.
(<a href="https://doi.org/10.1038/s41588-023-01468-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The incidence of keratinocyte cancer (basal cell and squamous cell carcinomas of the skin) is 17-fold lower in Singapore than the UK1–3, despite Singapore receiving 2–3 times more ultraviolet (UV) radiation4,5. Aging skin contains somatic mutant clones from which such cancers develop6,7. We hypothesized that differences in keratinocyte cancer incidence may be reflected in the normal skin mutational landscape. Here we show that, compared to Singapore, aging facial skin from populations in the UK has a fourfold greater mutational burden, a predominant UV mutational signature, increased copy number aberrations and increased mutant TP53 selection. These features are shared by keratinocyte cancers from high-incidence and low-incidence populations8–13. In Singaporean skin, most mutations result from cell-intrinsic processes; mutant NOTCH1 and NOTCH2 are more strongly selected than in the UK. Aging skin in a high-incidence country has multiple features convergent with cancer that are not found in a low-risk country. These differences may reflect germline variation in UV-protective genes. A comparison of somatic mutations in skin from individuals from the UK and Singapore suggests that the difference in cancer incidence between the two countries is due to markedly different mutational spectra and patterns of selection.},
  archive      = {J_NG},
  author       = {King, Charlotte and Fowler, Joanna C. and Abnizova, Irina and Sood, Roshan K. and Hall, Michael W. J. and Szeverényi, Ildikó and Tham, Muly and Huang, Jingxiang and Young, Stephanie Ming and Hall, Benjamin A. and Birgitte Lane, E. and Jones, Philip H.},
  doi          = {10.1038/s41588-023-01468-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1440-1447},
  shortjournal = {Nat. Genet.},
  title        = {Somatic mutations in facial skin from countries of contrasting skin cancer risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Exome sequencing identifies breast cancer susceptibility
genes and defines the contribution of coding variants to breast cancer
risk. <em>NG</em>, <em>55</em>(9), 1435–1439. (<a
href="https://doi.org/10.1038/s41588-023-01466-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Linkage and candidate gene studies have identified several breast cancer susceptibility genes, but the overall contribution of coding variation to breast cancer is unclear. To evaluate the role of rare coding variants more comprehensively, we performed a meta-analysis across three large whole-exome sequencing datasets, containing 26,368 female cases and 217,673 female controls. Burden tests were performed for protein-truncating and rare missense variants in 15,616 and 18,601 genes, respectively. Associations between protein-truncating variants and breast cancer were identified for the following six genes at exome-wide significance (P &amp;lt; 2.5 × 10−6): the five known susceptibility genes ATM, BRCA1, BRCA2, CHEK2 and PALB2, together with MAP3K1. Associations were also observed for LZTR1, ATRIP and BARD1 with P &amp;lt; 1 × 10−4. Associations between predicted deleterious rare missense or protein-truncating variants and breast cancer were additionally identified for CDKN2A at exome-wide significance. The overall contribution of coding variants in genes beyond the previously known genes is estimated to be small. Meta-analysis of three large whole-exome sequencing datasets highlights protein-truncating and rare missense variants associated with breast cancer susceptibility.},
  archive      = {J_NG},
  author       = {Wilcox, Naomi and Dumont, Martine and González-Neira, Anna and Carvalho, Sara and Joly Beauparlant, Charles and Crotti, Marco and Luccarini, Craig and Soucy, Penny and Dubois, Stéphane and Nuñez-Torres, Rocio and Pita, Guillermo and Gardner, Eugene J. and Dennis, Joe and Alonso, M. Rosario and Álvarez, Nuria and Baynes, Caroline and Collin-Deschesnes, Annie Claude and Desjardins, Sylvie and Becher, Heiko and Behrens, Sabine and Bolla, Manjeet K. and Castelao, Jose E. and Chang-Claude, Jenny and Cornelissen, Sten and Dörk, Thilo and Engel, Christoph and Gago-Dominguez, Manuela and Guénel, Pascal and Hadjisavvas, Andreas and Hahnen, Eric and Hartman, Mikael and Herráez, Belén and Jung, Audrey and Keeman, Renske and Kiechle, Marion and Li, Jingmei and Loizidou, Maria A. and Lush, Michael and Michailidou, Kyriaki and Panayiotidis, Mihalis I. and Sim, Xueling and Teo, Soo Hwang and Tyrer, Jonathan P. and van der Kolk, Lizet E. and Wahlström, Cecilia and Wang, Qin and Perry, John R. B. and Benitez, Javier and Schmidt, Marjanka K. and Schmutzler, Rita K. and Pharoah, Paul D. P. and Droit, Arnaud and Dunning, Alison M. and Kvist, Anders and Devilee, Peter and Easton, Douglas F. and Simard, Jacques},
  doi          = {10.1038/s41588-023-01466-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1435-1439},
  shortjournal = {Nat. Genet.},
  title        = {Exome sequencing identifies breast cancer susceptibility genes and defines the contribution of coding variants to breast cancer risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Inflammation drives pressure on TP53 mutant clones in
myeloproliferative neoplasms. <em>NG</em>, <em>55</em>(9), 1432–1434.
(<a href="https://doi.org/10.1038/s41588-023-01479-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transformation of a myeloproliferative neoplasm to a secondary acute myeloid leukemia is rare but devastating. Single-cell, multi-omic characterization of hematopoietic stem and progenitor cells now shows the role of inflammation in transformation driven by mutations in TP53, with effects on the mutant clone but also non-mutant counterparts.},
  archive      = {J_NG},
  author       = {Benabid, Adam and Schneider, Rebekka K.},
  doi          = {10.1038/s41588-023-01479-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1432-1434},
  shortjournal = {Nat. Genet.},
  title        = {Inflammation drives pressure on TP53 mutant clones in myeloproliferative neoplasms},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Systematic perturbation of chromatin factors identifies
regulatory networks during healthy and malignant hematopoiesis.
<em>NG</em>, <em>55</em>(9), 1430–1431. (<a
href="https://doi.org/10.1038/s41588-023-01478-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue of Nature Genetics, Lara-Astiaso et al. systematically characterized the functional roles of several chromatin factors in hematopoiesis by combining functional CRISPR screens with single-cell transcriptomics and chromatin accessibility profiling, revealing lineage biases and relationships with important transcription factors.},
  archive      = {J_NG},
  author       = {Solé-Boldo, Llorenç and Haas, Simon},
  doi          = {10.1038/s41588-023-01478-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1430-1431},
  shortjournal = {Nat. Genet.},
  title        = {Systematic perturbation of chromatin factors identifies regulatory networks during healthy and malignant hematopoiesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Our ancestry dictates clonal architecture and skin cancer
susceptibility. <em>NG</em>, <em>55</em>(9), 1428–1429. (<a
href="https://doi.org/10.1038/s41588-023-01467-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Incidence of keratinocyte skin cancer varies markedly between populations living in different areas of the world. A detailed analysis of somatic mutations in the normal skin of individuals from the UK and Singapore reveals different patterns of clonal mutational landscapes that could contribute to differential risk.},
  archive      = {J_NG},
  author       = {De Dominici, Marco and DeGregori, James},
  doi          = {10.1038/s41588-023-01467-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1428-1429},
  shortjournal = {Nat. Genet.},
  title        = {Our ancestry dictates clonal architecture and skin cancer susceptibility},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Advancing variant effect prediction using protein language
models. <em>NG</em>, <em>55</em>(9), 1426–1427. (<a
href="https://doi.org/10.1038/s41588-023-01470-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A novel pipeline that expands the utility of the protein language model ESM1b has provided variant effect predictions for more than 40,000 protein isoforms. This strategy outperformed several state-of-the-art methods over multiple benchmarks.},
  archive      = {J_NG},
  author       = {Livesey, Benjamin J. and Marsh, Joseph A.},
  doi          = {10.1038/s41588-023-01470-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1426-1427},
  shortjournal = {Nat. Genet.},
  title        = {Advancing variant effect prediction using protein language models},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine learning drives genetic discovery for binge eating
disorder. <em>NG</em>, <em>55</em>(9), 1424–1425. (<a
href="https://doi.org/10.1038/s41588-023-01473-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying genetic risk factors for binge-eating disorder (BED) is vital to understand its etiology and develop effective prevention and intervention strategies. To overcome under-reporting of clinical BED diagnosis, a new study uses machine learning to identify genetic variants associated with quantitative BED risk scores and finds evidence for a pathological role of heme metabolism.},
  archive      = {J_NG},
  author       = {Thorp, Jackson G. and Gerring, Zachary F. and Derks, Eske M.},
  doi          = {10.1038/s41588-023-01473-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1424-1425},
  shortjournal = {Nat. Genet.},
  title        = {Machine learning drives genetic discovery for binge eating disorder},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A fork in the road to differentiation. <em>NG</em>,
<em>55</em>(9), 1422–1423. (<a
href="https://doi.org/10.1038/s41588-023-01489-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The symmetric inheritance of histone modifications by the nascent chromatin fibers during DNA replication is essential for proper developmental progression. Two new studies using mouse embryonic stem cells further illuminate the role of histone inheritance in early cell fate decisions.},
  archive      = {J_NG},
  author       = {Altamirano-Pacheco, Luis and Navarro, Pablo},
  doi          = {10.1038/s41588-023-01489-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1422-1423},
  shortjournal = {Nat. Genet.},
  title        = {A fork in the road to differentiation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). The complex relationship between cell transcriptomic state
and phenotype. <em>NG</em>, <em>55</em>(9), 1421. (<a
href="https://doi.org/10.1038/s41588-023-01511-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01511-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1421},
  shortjournal = {Nat. Genet.},
  title        = {The complex relationship between cell transcriptomic state and phenotype},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Genomics of the oldest domesticated wheat. <em>NG</em>,
<em>55</em>(9), 1421. (<a
href="https://doi.org/10.1038/s41588-023-01512-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01512-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1421},
  shortjournal = {Nat. Genet.},
  title        = {Genomics of the oldest domesticated wheat},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). VNTRs and disease risk. <em>NG</em>, <em>55</em>(9), 1421.
(<a href="https://doi.org/10.1038/s41588-023-01513-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01513-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1421},
  shortjournal = {Nat. Genet.},
  title        = {VNTRs and disease risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Alu elements confer enhancer–promoter specificity.
<em>NG</em>, <em>55</em>(9), 1421. (<a
href="https://doi.org/10.1038/s41588-023-01514-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-023-01514-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1421},
  shortjournal = {Nat. Genet.},
  title        = {Alu elements confer enhancer–promoter specificity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Studying the genetics of participation using footprints left
on the ascertained genotypes. <em>NG</em>, <em>55</em>(8), 1413–1420.
(<a href="https://doi.org/10.1038/s41588-023-01439-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The trait of participating in a genetic study probably has a genetic component. Identifying this component is difficult as we cannot compare genetic information of participants with nonparticipants directly, the latter being unavailable. Here, we show that alleles that are more common in participants than nonparticipants would be further enriched in genetic segments shared by two related participants. Genome-wide analysis was performed by comparing allele frequencies in shared and not-shared genetic segments of first-degree relative pairs of the UK Biobank. In nonoverlapping samples, a polygenic score constructed from that analysis is significantly associated with educational attainment, body mass index and being invited to a dietary study. The estimated correlation between the genetic components underlying participation in UK Biobank and educational attainment is estimated to be 36.6%—substantial but far from total. Taking participation behaviour into account would improve the analyses of the study data, including those of health traits. A new methodology to study participation genetics via analyzing shared and unshared genotypes is applied to the UK Biobank, suggesting that participating in genetic studies is a complex trait with a meaningful genetic component.},
  archive      = {J_NG},
  author       = {Benonisdottir, Stefania and Kong, Augustine},
  doi          = {10.1038/s41588-023-01439-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1413-1420},
  shortjournal = {Nat. Genet.},
  title        = {Studying the genetics of participation using footprints left on the ascertained genotypes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Landscape of mSWI/SNF chromatin remodeling complex
perturbations in neurodevelopmental disorders. <em>NG</em>,
<em>55</em>(8), 1400–1412. (<a
href="https://doi.org/10.1038/s41588-023-01451-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA sequencing-based studies of neurodevelopmental disorders (NDDs) have identified a wide range of genetic determinants. However, a comprehensive analysis of these data, in aggregate, has not to date been performed. Here, we find that genes encoding the mammalian SWI/SNF (mSWI/SNF or BAF) family of ATP-dependent chromatin remodeling protein complexes harbor the greatest number of de novo missense and protein-truncating variants among nuclear protein complexes. Non-truncating NDD-associated protein variants predominantly disrupt the cBAF subcomplex and cluster in four key structural regions associated with high disease severity, including mSWI/SNF-nucleosome interfaces, the ATPase-core ARID-armadillo repeat (ARM) module insertion site, the Arp module and DNA-binding domains. Although over 70% of the residues perturbed in NDDs overlap with those mutated in cancer, ~60% of amino acid changes are NDD-specific. These findings provide a foundation to functionally group variants and link complex aberrancies to phenotypic severity, serving as a resource for the chromatin, clinical genetics and neurodevelopment communities. Genes encoding members of mammalian SWI/SNF (BAF) complexes are frequently mutated in individuals with neurodevelopmental disorders (NDDs). Mutant NDD residues include some unique to NDD and those shared with human cancers, impacting key structural hubs.},
  archive      = {J_NG},
  author       = {Valencia, Alfredo M. and Sankar, Akshay and van der Sluijs, Pleuntje J. and Satterstrom, F. Kyle and Fu, Jack and Talkowski, Michael E. and Vergano, Samantha A. Schrier and Santen, Gijs W. E. and Kadoch, Cigall},
  doi          = {10.1038/s41588-023-01451-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1400-1412},
  shortjournal = {Nat. Genet.},
  title        = {Landscape of mSWI/SNF chromatin remodeling complex perturbations in neurodevelopmental disorders},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Telomere-to-telomere assemblies of 142 strains characterize
the genome structural landscape in saccharomyces cerevisiae.
<em>NG</em>, <em>55</em>(8), 1390–1399. (<a
href="https://doi.org/10.1038/s41588-023-01459-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pangenomes provide access to an accurate representation of the genetic diversity of species, both in terms of sequence polymorphisms and structural variants (SVs). Here we generated the Saccharomyces cerevisiae Reference Assembly Panel (ScRAP) comprising reference-quality genomes for 142 strains representing the species’ phylogenetic and ecological diversity. The ScRAP includes phased haplotype assemblies for several heterozygous diploid and polyploid isolates. We identified circa (ca.) 4,800 nonredundant SVs that provide a broad view of the genomic diversity, including the dynamics of telomere length and transposable elements. We uncovered frequent cases of complex aneuploidies where large chromosomes underwent large deletions and translocations. We found that SVs can impact gene expression near the breakpoints and substantially contribute to gene repertoire evolution. We also discovered that horizontally acquired regions insert at chromosome ends and can generate new telomeres. Overall, the ScRAP demonstrates the benefit of a pangenome in understanding genome evolution at population scale. Saccharomyces cerevisiae Reference Assembly Panel (ScRAP) comprising telomere-to-telomere assemblies of 142 strains representing phylogenetic and ecological diversity of the species characterizes its genome structural landscape.},
  archive      = {J_NG},
  author       = {O’Donnell, Samuel and Yue, Jia-Xing and Saada, Omar Abou and Agier, Nicolas and Caradec, Claudia and Cokelaer, Thomas and De Chiara, Matteo and Delmas, Stéphane and Dutreux, Fabien and Fournier, Téo and Friedrich, Anne and Kornobis, Etienne and Li, Jing and Miao, Zepu and Tattini, Lorenzo and Schacherer, Joseph and Liti, Gianni and Fischer, Gilles},
  doi          = {10.1038/s41588-023-01459-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1390-1399},
  shortjournal = {Nat. Genet.},
  title        = {Telomere-to-telomere assemblies of 142 strains characterize the genome structural landscape in saccharomyces cerevisiae},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Suppressing a phosphohydrolase of cytokinin nucleotide
enhances grain yield in rice. <em>NG</em>, <em>55</em>(8), 1381–1389.
(<a href="https://doi.org/10.1038/s41588-023-01454-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One-step and two-step pathways are proposed to synthesize cytokinin in plants. The one-step pathway is mediated by LONELY GUY (LOG) proteins. However, the enzyme for the two-step pathway remains to be identified. Here, we show that quantitative trait locus GY3 may boost grain yield by more than 20% through manipulating a two-step pathway. Locus GY3 encodes a LOG protein that acts as a 5′-ribonucleotide phosphohydrolase by excessively consuming the cytokinin precursors, which contrasts with the activity of canonical LOG members as phosphoribohydrolases in a one-step pathway. The residue S41 of GY3 is crucial for the dephosphorylation of iPRMP to produce iPR. A solo-LTR insertion within the promoter of GY3 suppressed its expression and resulted in a higher content of active cytokinins in young panicles. Introgression of GY302428 increased grain yield per plot by 7.4% to 16.3% in all investigated indica backgrounds, which demonstrates the great value of GY302428 in indica rice production. GY3 encodes a LOG-like protein that acts as a 5′-ribonucleotide phosphohydrolase in cytokinin synthesis. A Gypsy solo-LTR insertion downregulates GY3 expression and shows the potential to enhance grain yield in rice.},
  archive      = {J_NG},
  author       = {Wu, Bi and Meng, Jianghu and Liu, Hongbo and Mao, Donghai and Yin, Huanran and Zhang, Zhanyi and Zhou, Xiangchun and Zhang, Bo and Sherif, Ahmed and Liu, Haiyang and Li, Xianghua and Xiao, Jinghua and Yan, Wenhao and Wang, Lei and Li, Xingwang and Chen, Wei and Xie, Weibo and Yin, Ping and Zhang, Qifa and Xing, Yongzhong},
  doi          = {10.1038/s41588-023-01454-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1381-1389},
  shortjournal = {Nat. Genet.},
  title        = {Suppressing a phosphohydrolase of cytokinin nucleotide enhances grain yield in rice},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). RNA polymerase II dynamics shape enhancer–promoter
interactions. <em>NG</em>, <em>55</em>(8), 1370–1380. (<a
href="https://doi.org/10.1038/s41588-023-01442-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How enhancers control target gene expression over long genomic distances remains an important unsolved problem. Here we investigated enhancer–promoter communication by integrating data from nucleosome-resolution genomic contact maps, nascent transcription and perturbations affecting either RNA polymerase II (Pol II) dynamics or the activity of thousands of candidate enhancers. Integration of new Micro-C experiments with published CRISPRi data demonstrated that enhancers spend more time in close proximity to their target promoters in functional enhancer–promoter pairs compared to nonfunctional pairs, which can be attributed in part to factors unrelated to genomic position. Manipulation of the transcription cycle demonstrated a key role for Pol II in enhancer–promoter interactions. Notably, promoter-proximal paused Pol II itself partially stabilized interactions. We propose an updated model in which elements of transcriptional dynamics shape the duration or frequency of interactions to facilitate enhancer–promoter communication. Perturbation of RNA polymerase II (Pol II) via chemical inhibition or dTAG-induced degradation and analysis using Micro-C and run-on sequencing show that enhancer–promoter contacts are dependent on transcription and stabilized by paused RNA Pol II.},
  archive      = {J_NG},
  author       = {Barshad, Gilad and Lewis, James J. and Chivu, Alexandra G. and Abuhashem, Abderhman and Krietenstein, Nils and Rice, Edward J. and Ma, Yitian and Wang, Zhong and Rando, Oliver J. and Hadjantonakis, Anna-Katerina and Danko, Charles G.},
  doi          = {10.1038/s41588-023-01442-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1370-1380},
  shortjournal = {Nat. Genet.},
  title        = {RNA polymerase II dynamics shape enhancer–promoter interactions},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). G4access identifies g-quadruplexes and their associations
with open chromatin and imprinting control regions. <em>NG</em>,
<em>55</em>(8), 1359–1369. (<a
href="https://doi.org/10.1038/s41588-023-01437-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metazoan promoters are enriched in secondary DNA structure-forming motifs, such as G-quadruplexes (G4s). Here we describe ‘G4access’, an approach to isolate and sequence G4s associated with open chromatin via nuclease digestion. G4access is antibody- and crosslinking-independent and enriches for computationally predicted G4s (pG4s), most of which are confirmed in vitro. Using G4access in human and mouse cells, we identify cell-type-specific G4 enrichment correlated with nucleosome exclusion and promoter transcription. G4access allows measurement of variations in G4 repertoire usage following G4 ligand treatment, HDAC and G4 helicases inhibitors. Applying G4access to cells from reciprocal hybrid mouse crosses suggests a role for G4s in the control of active imprinting regions. Consistently, we also observed that G4access peaks are unmethylated, while methylation at pG4s correlates with nucleosome repositioning on DNA. Overall, our study provides a new tool for studying G4s in cellular dynamics and highlights their association with open chromatin, transcription and their antagonism to DNA methylation. G4access is a nuclease-based method for identifying DNA G-quadruplex forming regions genome-wide in open chromatin using sequencing. Application across a range of cell types and species highlights associations of G-quadruplexes with various epigenetic and regulatory features.},
  archive      = {J_NG},
  author       = {Esnault, Cyril and Magat, Talha and Zine El Aabidine, Amal and Garcia-Oliver, Encar and Cucchiarini, Anne and Bouchouika, Soumya and Lleres, David and Goerke, Lutz and Luo, Yu and Verga, Daniela and Lacroix, Laurent and Feil, Robert and Spicuglia, Salvatore and Mergny, Jean-Louis and Andrau, Jean-Christophe},
  doi          = {10.1038/s41588-023-01437-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1359-1369},
  shortjournal = {Nat. Genet.},
  title        = {G4access identifies G-quadruplexes and their associations with open chromatin and imprinting control regions},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cohesin maintains replication timing to suppress DNA damage
on cancer genes. <em>NG</em>, <em>55</em>(8), 1347–1358. (<a
href="https://doi.org/10.1038/s41588-023-01458-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cohesin loss-of-function mutations are frequently observed in tumors, but the mechanism underlying its role in tumorigenesis is unclear. Here, we found that depletion of RAD21, a core subunit of cohesin, leads to massive genome-wide DNA breaks and 147 translocation hotspot genes, co-mutated with cohesin in multiple cancers. Increased DNA damages are independent of RAD21-loss-induced transcription alteration and loop anchor disruption. However, damage-induced chromosomal translocations coincide with the asymmetrically distributed Okazaki fragments of DNA replication, suggesting that RAD21 depletion causes replication stresses evidenced by the slower replication speed and increased stalled forks. Mechanistically, approximately 30% of the human genome exhibits an earlier replication timing after RAD21 depletion, caused by the early initiation of &amp;gt;900 extra dormant origins. Correspondingly, most translocation hotspot genes lie in timing-altered regions. Therefore, we conclude that cohesin dysfunction causes replication stresses induced by excessive DNA replication initiation, resulting in gross DNA damages that may promote tumorigenesis. Depletion of cohesin subunit RAD21 increases DNA breaks genome-wide, leading to translocations at cancer-mutated hotspot genes. RAD21 depletion leads to dormant origin firing and induces irregular replication timing.},
  archive      = {J_NG},
  author       = {Wu, Jinchun and Liu, Yang and Zhangding, Zhengrong and Liu, Xuhao and Ai, Chen and Gan, Tingting and Liang, Haoxin and Guo, Yuefeng and Chen, Mohan and Liu, Yiyang and Yin, Jianhang and Zhang, Weiwei and Hu, Jiazhi},
  doi          = {10.1038/s41588-023-01458-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1347-1358},
  shortjournal = {Nat. Genet.},
  title        = {Cohesin maintains replication timing to suppress DNA damage on cancer genes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dynamic network-guided CRISPRi screen identifies
CTCF-loop-constrained nonlinear enhancer gene regulatory activity during
cell state transitions. <em>NG</em>, <em>55</em>(8), 1336–1346. (<a
href="https://doi.org/10.1038/s41588-023-01450-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive enhancer discovery is challenging because most enhancers, especially those contributing to complex diseases, have weak effects on gene expression. Our gene regulatory network modeling identified that nonlinear enhancer gene regulation during cell state transitions can be leveraged to improve the sensitivity of enhancer discovery. Using human embryonic stem cell definitive endoderm differentiation as a dynamic transition system, we conducted a mid-transition CRISPRi-based enhancer screen. We discovered a comprehensive set of enhancers for each of the core endoderm-specifying transcription factors. Many enhancers had strong effects mid-transition but weak effects post-transition, consistent with the nonlinear temporal responses to enhancer perturbation predicted by the modeling. Integrating three-dimensional genomic information, we were able to develop a CTCF-loop-constrained Interaction Activity model that can better predict functional enhancers compared to models that rely on Hi-C-based enhancer–promoter contact frequency. Our study provides generalizable strategies for sensitive and systematic enhancer discovery in both normal and pathological cell state transitions. CRISPRi-based screen identifies enhancers that nonlinearly regulate lineage-specifying transcription factors during definitive endoderm differentiation. A new CTCF-loop-constrained Interaction Activity (CIA) model is better at predicting functional enhancers than previous Hi-C-based enhancer–promoter contact frequency models.},
  archive      = {J_NG},
  author       = {Luo, Renhe and Yan, Jielin and Oh, Jin Woo and Xi, Wang and Shigaki, Dustin and Wong, Wilfred and Cho, Hyein S. and Murphy, Dylan and Cutler, Ronald and Rosen, Bess P. and Pulecio, Julian and Yang, Dapeng and Glenn, Rachel A. and Chen, Tingxu and Li, Qing V. and Vierbuchen, Thomas and Sidoli, Simone and Apostolou, Effie and Huangfu, Danwei and Beer, Michael A.},
  doi          = {10.1038/s41588-023-01450-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1336-1346},
  shortjournal = {Nat. Genet.},
  title        = {Dynamic network-guided CRISPRi screen identifies CTCF-loop-constrained nonlinear enhancer gene regulatory activity during cell state transitions},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Crosstalk between RNA m6A and DNA methylation regulates
transposable element chromatin activation and cell fate in human
pluripotent stem cells. <em>NG</em>, <em>55</em>(8), 1324–1335. (<a
href="https://doi.org/10.1038/s41588-023-01452-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposable elements (TEs) are parasitic DNA sequences accounting for over half of the human genome. Tight control of the repression and activation states of TEs is critical for genome integrity, development, immunity and diseases, including cancer. However, precisely how this regulation is achieved remains unclear. Here we develop a targeted proteomic proximity labeling approach to capture TE-associated proteins in human embryonic stem cells (hESCs). We find that the RNA N6-methyladenosine (m6A) reader, YTHDC2, occupies genomic loci of the primate-specific TE, LTR7/HERV-H, specifically through its interaction with m6A-modified HERV-H RNAs. Unexpectedly, YTHDC2 recruits the DNA 5-methylcytosine (5mC)-demethylase, TET1, to remove 5mC from LTR7/HERV-H and prevent epigenetic silencing. Functionally, the YTHDC2/LTR7 axis inhibits neural differentiation of hESCs. Our results reveal both an underappreciated crosstalk between RNA m6A and DNA 5mC, the most abundant regulatory modifications of RNA and DNA in eukaryotes, and the fact that in hESCs this interplay controls TE activity and cell fate. The N6-methyladenosine reader YTHDC2 recruits TET1 to remove 5-methylcytosine from LTR7/HERV-H loci in human embryonic stem cells. The YTHDC2/LTR7 pathway inhibits neural fate commitment.},
  archive      = {J_NG},
  author       = {Sun, Tongyu and Xu, Yueyuan and Xiang, Yu and Ou, Jianhong and Soderblom, Erik J. and Diao, Yarui},
  doi          = {10.1038/s41588-023-01452-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1324-1335},
  shortjournal = {Nat. Genet.},
  title        = {Crosstalk between RNA m6A and DNA methylation regulates transposable element chromatin activation and cell fate in human pluripotent stem cells},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SF3B1 hotspot mutations confer sensitivity to PARP
inhibition by eliciting a defective replication stress response.
<em>NG</em>, <em>55</em>(8), 1311–1323. (<a
href="https://doi.org/10.1038/s41588-023-01460-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SF3B1 hotspot mutations are associated with a poor prognosis in several tumor types and lead to global disruption of canonical splicing. Through synthetic lethal drug screens, we identify that SF3B1 mutant (SF3B1MUT) cells are selectively sensitive to poly (ADP-ribose) polymerase inhibitors (PARPi), independent of hotspot mutation and tumor site. SF3B1MUT cells display a defective response to PARPi-induced replication stress that occurs via downregulation of the cyclin-dependent kinase 2 interacting protein (CINP), leading to increased replication fork origin firing and loss of phosphorylated CHK1 (pCHK1; S317) induction. This results in subsequent failure to resolve DNA replication intermediates and G2/M cell cycle arrest. These defects are rescued through CINP overexpression, or further targeted by a combination of ataxia-telangiectasia mutated and PARP inhibition. In vivo, PARPi produce profound antitumor effects in multiple SF3B1MUT cancer models and eliminate distant metastases. These data provide the rationale for testing the clinical efficacy of PARPi in a biomarker-driven, homologous recombination proficient, patient population. SF3B1 mutations confer sensitivity to poly (ADP-ribose) polymerase inhibitors (PARPi). Mechanistically, this is independent of homologous recombination repair and instead relies on a defective replication stress response due to a reduction of the cyclin-dependent kinase 2 interacting protein (CINP). PARPi treatment of SF3B1 mutant (SF3B1MUT) tumors leads to replication stress induced by increased fork origin firing and culminates in cell cycle stalling.},
  archive      = {J_NG},
  author       = {Bland, Philip and Saville, Harry and Wai, Patty T. and Curnow, Lucinda and Muirhead, Gareth and Nieminuszczy, Jadwiga and Ravindran, Nivedita and John, Marie Beatrix and Hedayat, Somaieh and Barker, Holly E. and Wright, James and Yu, Lu and Mavrommati, Ioanna and Read, Abigail and Peck, Barrie and Allen, Mark and Gazinska, Patrycja and Pemberton, Helen N. and Gulati, Aditi and Nash, Sarah and Noor, Farzana and Guppy, Naomi and Roxanis, Ioannis and Pratt, Guy and Oldreive, Ceri and Stankovic, Tatjana and Barlow, Samantha and Kalirai, Helen and Coupland, Sarah E. and Broderick, Ronan and Alsafadi, Samar and Houy, Alexandre and Stern, Marc-Henri and Pettit, Stephen and Choudhary, Jyoti S. and Haider, Syed and Niedzwiedz, Wojciech and Lord, Christopher J. and Natrajan, Rachael},
  doi          = {10.1038/s41588-023-01460-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1311-1323},
  shortjournal = {Nat. Genet.},
  title        = {SF3B1 hotspot mutations confer sensitivity to PARP inhibition by eliciting a defective replication stress response},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-molecule genome-wide mutation profiles of cell-free
DNA for non-invasive detection of cancer. <em>NG</em>, <em>55</em>(8),
1301–1310. (<a
href="https://doi.org/10.1038/s41588-023-01446-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations are a hallmark of tumorigenesis and may be useful for non-invasive diagnosis of cancer. We analyzed whole-genome sequencing data from 2,511 individuals in the Pan-Cancer Analysis of Whole Genomes (PCAWG) study as well as 489 individuals from four prospective cohorts and found distinct regional mutation type-specific frequencies in tissue and cell-free DNA from patients with cancer that were associated with replication timing and other chromatin features. A machine-learning model using genome-wide mutational profiles combined with other features and followed by CT imaging detected &amp;gt;90% of patients with lung cancer, including those with stage I and II disease. The fixed model was validated in an independent cohort, detected patients with cancer earlier than standard approaches and could be used to monitor response to therapy. This approach lays the groundwork for non-invasive cancer detection using genome-wide mutation features that may facilitate cancer screening and monitoring. Somatic mutations are identified from circulating cell-free DNA using a single-molecule-based lowpass whole-genome sequencing method. The regional distribution of mutations can identify a tumor-specific mutational profile in patients with cancer and can be used to monitor patients through treatment.},
  archive      = {J_NG},
  author       = {Bruhm, Daniel C. and Mathios, Dimitrios and Foda, Zachariah H. and Annapragada, Akshaya V. and Medina, Jamie E. and Adleff, Vilmos and Chiao, Elaine Jiayuee and Ferreira, Leonardo and Cristiano, Stephen and White, James R. and Mazzilli, Sarah A. and Billatos, Ehab and Spira, Avrum and Zaidi, Ali H. and Mueller, Jeffrey and Kim, Amy K. and Anagnostou, Valsamo and Phallen, Jillian and Scharpf, Robert B. and Velculescu, Victor E.},
  doi          = {10.1038/s41588-023-01446-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1301-1310},
  shortjournal = {Nat. Genet.},
  title        = {Single-molecule genome-wide mutation profiles of cell-free DNA for non-invasive detection of cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integrated single-cell chromatin and transcriptomic analyses
of human scalp identify gene-regulatory programs and critical cell types
for hair and skin diseases. <em>NG</em>, <em>55</em>(8), 1288–1300. (<a
href="https://doi.org/10.1038/s41588-023-01445-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have identified many loci associated with hair and skin disease, but identification of causal variants requires deciphering of gene-regulatory networks in relevant cell types. We generated matched single-cell chromatin profiles and transcriptomes from scalp tissue from healthy controls and patients with alopecia areata, identifying diverse cell types of the hair follicle niche. By interrogating these datasets at multiple levels of cellular resolution, we infer 50–100% more enhancer–gene links than previous approaches and show that aggregate enhancer accessibility for highly regulated genes predicts expression. We use these gene-regulatory maps to prioritize cell types, genes and causal variants implicated in the pathobiology of androgenetic alopecia (AGA), eczema and other complex traits. AGA genome-wide association studies signals are enriched in dermal papilla regulatory regions, supporting the role of these cells as drivers of AGA pathogenesis. Finally, we train machine learning models to nominate single-nucleotide polymorphisms that affect gene expression through disruption of transcription factor binding, predicting candidate functional single-nucleotide polymorphism for AGA and eczema. Single-cell RNA- and ATAC-seq analysis of human scalp biopsies characterizes the regulatory programs of heterogeneous cell types. Integration with genome-wide association studies of hair and skin diseases implicates critical cell types, transcription factors and candidate causal variants.},
  archive      = {J_NG},
  author       = {Ober-Reynolds, Benjamin and Wang, Chen and Ko, Justin M. and Rios, Eon J. and Aasi, Sumaira Z. and Davis, Mark M. and Oro, Anthony E. and Greenleaf, William J.},
  doi          = {10.1038/s41588-023-01445-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1288-1300},
  shortjournal = {Nat. Genet.},
  title        = {Integrated single-cell chromatin and transcriptomic analyses of human scalp identify gene-regulatory programs and critical cell types for hair and skin diseases},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Converging evidence from exome sequencing and common
variants implicates target genes for osteoporosis. <em>NG</em>,
<em>55</em>(8), 1277–1287. (<a
href="https://doi.org/10.1038/s41588-023-01444-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, we leveraged the combined evidence of rare coding variants and common alleles to identify therapeutic targets for osteoporosis. We undertook a large-scale multiancestry exome-wide association study for estimated bone mineral density, which showed that the burden of rare coding alleles in 19 genes was associated with estimated bone mineral density (P &amp;lt; 3.6 × 10–7). These genes were highly enriched for a set of known causal genes for osteoporosis (65-fold; P = 2.5 × 10–5). Exome-wide significant genes had 96-fold increased odds of being the top ranked effector gene at a given GWAS locus (P = 1.8 × 10–10). By integrating proteomics Mendelian randomization evidence, we prioritized CD109 (cluster of differentiation 109) as a gene for which heterozygous loss of function is associated with higher bone density. CRISPR–Cas9 editing of CD109 in SaOS-2 osteoblast-like cell lines showed that partial CD109 knockdown led to increased mineralization. This study demonstrates that the convergence of common and rare variants, proteomics and CRISPR can highlight new bone biology to guide therapeutic development. Analysis of exome sequencing data identifies a burden of rare coding variants in 19 genes associated with bone mineral density. Integrated analyses show convergence of common- and rare-variant signals and highlight likely effector genes influencing osteoporosis risk.},
  archive      = {J_NG},
  author       = {Zhou, Sirui and Sosina, Olukayode A. and Bovijn, Jonas and Laurent, Laetitia and Sharma, Vasundhara and Akbari, Parsa and Forgetta, Vincenzo and Jiang, Lai and Kosmicki, Jack A. and Banerjee, Nilanjana and Morris, John A. and Oerton, Erin and Jones, Marcus and LeBlanc, Michelle G. and Idone, Vincent and Overton, John D. and Reid, Jeffrey G. and Cantor, Michael and Abecasis, Goncalo R. and Goltzman, David and Greenwood, Celia M. T. and Langenberg, Claudia and Baras, Aris and Economides, Aris N. and Ferreira, Manuel A. R. and Hatsell, Sarah and Ohlsson, Claes and Richards, J. Brent and Lotta, Luca A.},
  doi          = {10.1038/s41588-023-01444-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1277-1287},
  shortjournal = {Nat. Genet.},
  title        = {Converging evidence from exome sequencing and common variants implicates target genes for osteoporosis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Leveraging polygenic enrichments of gene features to predict
genes underlying complex traits and diseases. <em>NG</em>,
<em>55</em>(8), 1267–1276. (<a
href="https://doi.org/10.1038/s41588-023-01443-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWASs) are a valuable tool for understanding the biology of complex human traits and diseases, but associated variants rarely point directly to causal genes. In the present study, we introduce a new method, polygenic priority score (PoPS), that learns trait-relevant gene features, such as cell-type-specific expression, to prioritize genes at GWAS loci. Using a large evaluation set of genes with fine-mapped coding variants, we show that PoPS and the closest gene individually outperform other gene prioritization methods, but observe the best overall performance by combining PoPS with orthogonal methods. Using this combined approach, we prioritize 10,642 unique gene–trait pairs across 113 complex traits and diseases with high precision, finding not only well-established gene–trait relationships but nominating new genes at unresolved loci, such as LGR4 for estimated glomerular filtration rate and CCR7 for deep vein thrombosis. Overall, we demonstrate that PoPS provides a powerful addition to the gene prioritization toolbox. Polygenic Priority Score (PoPS) prioritizes candidate effector genes at complex trait loci by integrating genome-wide association summary statistics with other data types. Combining PoPS with methods that leverage local genetic signals further improves the performance.},
  archive      = {J_NG},
  author       = {Weeks, Elle M. and Ulirsch, Jacob C. and Cheng, Nathan Y. and Trippe, Brian L. and Fine, Rebecca S. and Miao, Jenkai and Patwardhan, Tejal A. and Kanai, Masahiro and Nasser, Joseph and Fulco, Charles P. and Tashman, Katherine C. and Aguet, Francois and Li, Taibo and Ordovas-Montanes, Jose and Smillie, Christopher S. and Biton, Moshe and Shalek, Alex K. and Ananthakrishnan, Ashwin N. and Xavier, Ramnik J. and Regev, Aviv and Gupta, Rajat M. and Lage, Kasper and Ardlie, Kristin G. and Hirschhorn, Joel N. and Lander, Eric S. and Engreitz, Jesse M. and Finucane, Hilary K.},
  doi          = {10.1038/s41588-023-01443-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1267-1276},
  shortjournal = {Nat. Genet.},
  title        = {Leveraging polygenic enrichments of gene features to predict genes underlying complex traits and diseases},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SF3B1-mutant cells succumb to replication stress under PARP
inhibition. <em>NG</em>, <em>55</em>(8), 1265–1266. (<a
href="https://doi.org/10.1038/s41588-023-01461-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across multiple cancer types, hotspot mutations in SF3B1 confer selective sensitivity to multiple clinically available PARP inhibitors. This sensitivity is due to reduced levels of CINP specifically in SF3B1-mutant cells, which leads to a loss of the canonical replication stress response after challenge with PARP inhibitors.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01461-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1265-1266},
  shortjournal = {Nat. Genet.},
  title        = {SF3B1-mutant cells succumb to replication stress under PARP inhibition},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identifying critical cell types and gene regulatory pathways
for hair and skin disease. <em>NG</em>, <em>55</em>(8), 1263–1264. (<a
href="https://doi.org/10.1038/s41588-023-01447-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Matched single-cell transcriptomic and epigenomic profiles obtained from human scalp reveal gene regulatory maps of diverse cell types in the hair follicle niche. Integrating these data with existing genome-wide association studies enabled prioritization of cell types, genes, and causal variants implicated in the pathobiology of androgenetic alopecia, eczema, and other complex traits.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01447-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1263-1264},
  shortjournal = {Nat. Genet.},
  title        = {Identifying critical cell types and gene regulatory pathways for hair and skin disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide mutation profiles from cell-free DNA for early
cancer detection. <em>NG</em>, <em>55</em>(8), 1261–1262. (<a
href="https://doi.org/10.1038/s41588-023-01448-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Through whole-genome sequencing of single molecules of circulating cell-free DNA, we found that tumor-derived mutations in cancer genomes are associated with regions of late replication timing and other chromatin features. These genome-wide analyses identified altered regional mutation profiles in people with cancer that distinguished them from people without cancer and reflected tumor burden during therapy.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01448-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1261-1262},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide mutation profiles from cell-free DNA for early cancer detection},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Regulation of transposable elements and stem cell fate by
crosstalk between RNA and DNA methylation. <em>NG</em>, <em>55</em>(8),
1259–1260. (<a
href="https://doi.org/10.1038/s41588-023-01453-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How the chromatin states of transposable elements (TEs) are controlled in development and disease is unclear. We present CARGO-BioID, a CRISPR-based proteomic approach to identify TE-associated proteins, and reveal an interplay between RNA N6-methyladenosine (m6A) and DNA methylation that is crucial for regulating TE activation and human embryonic stem cell (hESC) fate.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01453-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1259-1260},
  shortjournal = {Nat. Genet.},
  title        = {Regulation of transposable elements and stem cell fate by crosstalk between RNA and DNA methylation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). RNA pol II enters the ring of cohesin-mediated loop
extrusion. <em>NG</em>, <em>55</em>(8), 1256–1258. (<a
href="https://doi.org/10.1038/s41588-023-01463-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The three-dimensional organization of mammalian chromosomes can regulate transcription. Whether transcription itself influences genome structure has remained a source of debate. Using sensitive genome-wide readouts, two recent studies describe the involvement of transcription on genome architecture by different mechanisms.},
  archive      = {J_NG},
  author       = {Noordermeer, Daan},
  doi          = {10.1038/s41588-023-01463-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1256-1258},
  shortjournal = {Nat. Genet.},
  title        = {RNA pol II enters the ring of cohesin-mediated loop extrusion},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Leveraging family relatedness to detect participation bias
in genetic studies. <em>NG</em>, <em>55</em>(8), 1254–1255. (<a
href="https://doi.org/10.1038/s41588-023-01440-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying the ways that a study sample is not representative is essential for maximizing the generalizability of findings to the population. A new method proposes discerning non-representativeness in large-scale genetic studies by comparing the genotypes of closely related participants.},
  archive      = {J_NG},
  author       = {Adams, Mark J.},
  doi          = {10.1038/s41588-023-01440-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1254-1255},
  shortjournal = {Nat. Genet.},
  title        = {Leveraging family relatedness to detect participation bias in genetic studies},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Spatial multiomics of the human heart. <em>NG</em>,
<em>55</em>(8), 1253. (<a
href="https://doi.org/10.1038/s41588-023-01481-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01481-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1253},
  shortjournal = {Nat. Genet.},
  title        = {Spatial multiomics of the human heart},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Genetic insights into depression. <em>NG</em>,
<em>55</em>(8), 1253. (<a
href="https://doi.org/10.1038/s41588-023-01482-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01482-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1253},
  shortjournal = {Nat. Genet.},
  title        = {Genetic insights into depression},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). The “kiss and kick” model of gene activation. <em>NG</em>,
<em>55</em>(8), 1253. (<a
href="https://doi.org/10.1038/s41588-023-01483-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-023-01483-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1253},
  shortjournal = {Nat. Genet.},
  title        = {The ‘kiss and kick’ model of gene activation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Drug treatment and phenotypic diversity. <em>NG</em>,
<em>55</em>(8), 1253. (<a
href="https://doi.org/10.1038/s41588-023-01484-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01484-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1253},
  shortjournal = {Nat. Genet.},
  title        = {Drug treatment and phenotypic diversity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Publisher correction: Thymidine nucleotide metabolism
controls human telomere length. <em>NG</em>, <em>55</em>(7), 1251. (<a
href="https://doi.org/10.1038/s41588-023-01418-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Mannherz, William and Agarwal, Suneet},
  doi          = {10.1038/s41588-023-01418-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1251},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Thymidine nucleotide metabolism controls human telomere length},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Publisher correction: Deciphering DNA variant-associated
aberrant splicing with the aid of RNA sequencing. <em>NG</em>,
<em>55</em>(7), 1251. (<a
href="https://doi.org/10.1038/s41588-023-01441-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Zhang, Bin and Gao, Xin},
  doi          = {10.1038/s41588-023-01441-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1251},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Genetic effects on the timing of
parturition and links to fetal birth weight. <em>NG</em>,
<em>55</em>(7), 1250. (<a
href="https://doi.org/10.1038/s41588-023-01412-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Solé-Navais, Pol and Flatley, Christopher and Steinthorsdottir, Valgerdur and Vaudel, Marc and Juodakis, Julius and Chen, Jing and Laisk, Triin and LaBella, Abigail L. and Westergaard, David and Bacelis, Jonas and Brumpton, Ben and Skotte, Line and Borges, Maria C. and Helgeland, Øyvind and Mahajan, Anubha and Wielscher, Matthias and Lin, Frederick and Briggs, Catherine and Wang, Carol A. and Moen, Gunn-Helen and Beaumont, Robin N. and Bradfield, Jonathan P. and Abraham, Abin and Thorleifsson, Gudmar and Gabrielsen, Maiken E. and Ostrowski, Sisse R. and Modzelewska, Dominika and Nohr, Ellen A. and Hypponen, Elina and Srivastava, Amit and Talbot, Octavious and Allard, Catherine and Williams, Scott M. and Menon, Ramkumar and Shields, Beverley M. and Sveinbjornsson, Gardar and Xu, Huan and Melbye, Mads and Lowe, William and Bouchard, Luigi and Oken, Emily and Pedersen, Ole B. and Gudbjartsson, Daniel F. and Erikstrup, Christian and Sørensen, Erik and Lie, Rolv T. and Teramo, Kari and Hallman, Mikko and Juliusdottir, Thorhildur and Hakonarson, Hakon and Ullum, Henrik and Hattersley, Andrew T. and Sletner, Line and Merialdi, Mario and Rifas-Shiman, Sheryl L. and Steingrimsdottir, Thora and Scholtens, Denise and Power, Christine and West, Jane and Nyegaard, Mette and Capra, John A. and Skogholt, Anne H. and Magnus, Per and Andreassen, Ole A. and Thorsteinsdottir, Unnur and Grant, Struan F. A. and Qvigstad, Elisabeth and Pennell, Craig E. and Hivert, Marie-France and Hayes, Geoffrey M. and Jarvelin, Marjo-Riitta and McCarthy, Mark I. and Lawlor, Deborah A. and Nielsen, Henriette S. and Mägi, Reedik and Rokas, Antonis and Hveem, Kristian and Stefansson, Kari and Feenstra, Bjarke and Njolstad, Pål and Muglia, Louis J. and Freathy, Rachel M. and Johansson, Stefan and Zhang, Ge and Jacobsson, Bo},
  doi          = {10.1038/s41588-023-01412-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1250},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Genetic effects on the timing of parturition and links to fetal birth weight},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accurate rare variant phasing of whole-genome and
whole-exome sequencing data in the UK biobank. <em>NG</em>,
<em>55</em>(7), 1243–1249. (<a
href="https://doi.org/10.1038/s41588-023-01415-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phasing involves distinguishing the two parentally inherited copies of each chromosome into haplotypes. Here, we introduce SHAPEIT5, a new phasing method that quickly and accurately processes large sequencing datasets and applied it to UK Biobank (UKB) whole-genome and whole-exome sequencing data. We demonstrate that SHAPEIT5 phases rare variants with low switch error rates of below 5% for variants present in just 1 sample out of 100,000. Furthermore, we outline a method for phasing singletons, which, although less precise, constitutes an important step towards future developments. We then demonstrate that the use of UKB as a reference panel improves the accuracy of genotype imputation, which is even more pronounced when phased with SHAPEIT5 compared with other methods. Finally, we screen the UKB data for loss-of-function compound heterozygous events and identify 549 genes where both gene copies are knocked out. These genes complement current knowledge of gene essentiality in the human genome. SHAPEIT5, a phasing method that accurately processes large sequencing datasets, was applied on the UK Biobank whole-genome and whole-exome sequencing data to generate reference panels of haplotypes that boost imputation accuracy and enable the detection of compound heterozygous loss-of-function events for 549 genes.},
  archive      = {J_NG},
  author       = {Hofmeister, Robin J. and Ribeiro, Diogo M. and Rubinacci, Simone and Delaneau, Olivier},
  doi          = {10.1038/s41588-023-01415-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1243-1249},
  shortjournal = {Nat. Genet.},
  title        = {Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK biobank},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A graph-based genome and pan-genome variation of the model
plant setaria. <em>NG</em>, <em>55</em>(7), 1232–1242. (<a
href="https://doi.org/10.1038/s41588-023-01423-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Setaria italica (foxtail millet), a founder crop of East Asian agriculture, is a model plant for C4 photosynthesis and developing approaches to adaptive breeding across multiple climates. Here we established the Setaria pan-genome by assembling 110 representative genomes from a worldwide collection. The pan-genome is composed of 73,528 gene families, of which 23.8%, 42.9%, 29.4% and 3.9% are core, soft core, dispensable and private genes, respectively; 202,884 nonredundant structural variants were also detected. The characterization of pan-genomic variants suggests their importance during foxtail millet domestication and improvement, as exemplified by the identification of the yield gene SiGW3, where a 366-bp presence/absence promoter variant accompanies gene expression variation. We developed a graph-based genome and performed large-scale genetic studies for 68 traits across 13 environments, identifying potential genes for millet improvement at different geographic sites. These can be used in marker-assisted breeding, genomic selection and genome editing to accelerate crop improvement under different climatic conditions. Setaria pan-genome constructed using genome assemblies of 110 representative accessions and variation analyses provides insights into foxtail millet domestication and the genetic basis for crop improvement.},
  archive      = {J_NG},
  author       = {He, Qiang and Tang, Sha and Zhi, Hui and Chen, Jinfeng and Zhang, Jun and Liang, Hongkai and Alam, Ornob and Li, Hongbo and Zhang, Hui and Xing, Lihe and Li, Xukai and Zhang, Wei and Wang, Hailong and Shi, Junpeng and Du, Huilong and Wu, Hongpo and Wang, Liwei and Yang, Ping and Xing, Lu and Yan, Hongshan and Song, Zhongqiang and Liu, Jinrong and Wang, Haigang and Tian, Xiang and Qiao, Zhijun and Feng, Guojun and Guo, Ruifeng and Zhu, Wenjuan and Ren, Yuemei and Hao, Hongbo and Li, Mingzhe and Zhang, Aiying and Guo, Erhu and Yan, Feng and Li, Qingquan and Liu, Yanli and Tian, Bohong and Zhao, Xiaoqin and Jia, Ruiling and Feng, Baili and Zhang, Jiewei and Wei, Jianhua and Lai, Jinsheng and Jia, Guanqing and Purugganan, Michael and Diao, Xianmin},
  doi          = {10.1038/s41588-023-01423-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1232-1242},
  shortjournal = {Nat. Genet.},
  title        = {A graph-based genome and pan-genome variation of the model plant setaria},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A complete telomere-to-telomere assembly of the maize
genome. <em>NG</em>, <em>55</em>(7), 1221–1231. (<a
href="https://doi.org/10.1038/s41588-023-01419-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A complete telomere-to-telomere (T2T) finished genome has been the long pursuit of genomic research. Through generating deep coverage ultralong Oxford Nanopore Technology (ONT) and PacBio HiFi reads, we report here a complete genome assembly of maize with each chromosome entirely traversed in a single contig. The 2,178.6 Mb T2T Mo17 genome with a base accuracy of over 99.99% unveiled the structural features of all repetitive regions of the genome. There were several super-long simple-sequence-repeat arrays having consecutive thymine–adenine–guanine (TAG) tri-nucleotide repeats up to 235 kb. The assembly of the entire nucleolar organizer region of the 26.8 Mb array with 2,974 45S rDNA copies revealed the enormously complex patterns of rDNA duplications and transposon insertions. Additionally, complete assemblies of all ten centromeres enabled us to precisely dissect the repeat compositions of both CentC-rich and CentC-poor centromeres. The complete Mo17 genome represents a major step forward in understanding the complexity of the highly recalcitrant repetitive regions of higher plant genomes. A complete telomere-to-telomere genome assembly of the maize Mo17 inbred line uncovers structural features of the highly complex maize genome.},
  archive      = {J_NG},
  author       = {Chen, Jian and Wang, Zijian and Tan, Kaiwen and Huang, Wei and Shi, Junpeng and Li, Tong and Hu, Jiang and Wang, Kai and Wang, Chao and Xin, Beibei and Zhao, Haiming and Song, Weibin and Hufford, Matthew B. and Schnable, James C. and Jin, Weiwei and Lai, Jinsheng},
  doi          = {10.1038/s41588-023-01419-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1221-1231},
  shortjournal = {Nat. Genet.},
  title        = {A complete telomere-to-telomere assembly of the maize genome},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Potent and uniform fetal hemoglobin induction via base
editing. <em>NG</em>, <em>55</em>(7), 1210–1220. (<a
href="https://doi.org/10.1038/s41588-023-01434-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inducing fetal hemoglobin (HbF) in red blood cells can alleviate β-thalassemia and sickle cell disease. We compared five strategies in CD34+ hematopoietic stem and progenitor cells, using either Cas9 nuclease or adenine base editors. The most potent modification was adenine base editor generation of γ-globin –175A&amp;gt;G. Homozygous –175A&amp;gt;G edited erythroid colonies expressed 81 ± 7% HbF versus 17 ± 11% in unedited controls, whereas HbF levels were lower and more variable for two Cas9 strategies targeting a BCL11A binding motif in the γ-globin promoter or a BCL11A erythroid enhancer. The –175A&amp;gt;G base edit also induced HbF more potently than a Cas9 approach in red blood cells generated after transplantation of CD34+ hematopoietic stem and progenitor cells into mice. Our data suggest a strategy for potent, uniform induction of HbF and provide insights into γ-globin gene regulation. More generally, we demonstrate that diverse indels generated by Cas9 can cause unexpected phenotypic variation that can be circumvented by base editing. A comparison of fetal hemoglobin gene editing strategies using human sickle cell disease donor cells and in vivo transplantation finds that adenine base editing of the –175A&amp;gt;G site in the γ-globin gene promoters results in durable and potent expression.},
  archive      = {J_NG},
  author       = {Mayuranathan, Thiyagaraj and Newby, Gregory A. and Feng, Ruopeng and Yao, Yu and Mayberry, Kalin D. and Lazzarotto, Cicera R. and Li, Yichao and Levine, Rachel M. and Nimmagadda, Nikitha and Dempsey, Erin and Kang, Guolian and Porter, Shaina N. and Doerfler, Phillip A. and Zhang, Jingjing and Jang, Yoonjeong and Chen, Jingjing and Bell, Henry W. and Crossley, Merlin and Bhoopalan, Senthil Velan and Sharma, Akshay and Tisdale, John F. and Pruett-Miller, Shondra M. and Cheng, Yong and Tsai, Shengdar Q. and Liu, David R. and Weiss, Mitchell J. and Yen, Jonathan S.},
  doi          = {10.1038/s41588-023-01434-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1210-1220},
  shortjournal = {Nat. Genet.},
  title        = {Potent and uniform fetal hemoglobin induction via base editing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell multi-omics of mitochondrial DNA disorders
reveals dynamics of purifying selection across human immune cells.
<em>NG</em>, <em>55</em>(7), 1198–1209. (<a
href="https://doi.org/10.1038/s41588-023-01433-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pathogenic mutations in mitochondrial DNA (mtDNA) compromise cellular metabolism, contributing to cellular heterogeneity and disease. Diverse mutations are associated with diverse clinical phenotypes, suggesting distinct organ- and cell-type-specific metabolic vulnerabilities. Here we establish a multi-omics approach to quantify deletions in mtDNA alongside cell state features in single cells derived from six patients across the phenotypic spectrum of single large-scale mtDNA deletions (SLSMDs). By profiling 206,663 cells, we reveal the dynamics of pathogenic mtDNA deletion heteroplasmy consistent with purifying selection and distinct metabolic vulnerabilities across T-cell states in vivo and validate these observations in vitro. By extending analyses to hematopoietic and erythroid progenitors, we reveal mtDNA dynamics and cell-type-specific gene regulatory adaptations, demonstrating the context-dependence of perturbing mitochondrial genomic integrity. Collectively, we report pathogenic mtDNA heteroplasmy dynamics of individual blood and immune cells across lineages, demonstrating the power of single-cell multi-omics for revealing fundamental properties of mitochondrial genetics. Single-cell analyses of immune cells from patients with pathogenic, single large-scale mitochondrial DNA (mtDNA) deletions including Pearson syndrome describe heteroplasmy dynamics consistent with purifying selection, as well as T-cell state-specific regulatory mechanisms and metabolic vulnerabilities.},
  archive      = {J_NG},
  author       = {Lareau, Caleb A. and Dubois, Sonia M. and Buquicchio, Frank A. and Hsieh, Yu-Hsin and Garg, Kopal and Kautz, Pauline and Nitsch, Lena and Praktiknjo, Samantha D. and Maschmeyer, Patrick and Verboon, Jeffrey M. and Gutierrez, Jacob C. and Yin, Yajie and Fiskin, Evgenij and Luo, Wendy and Mimitou, Eleni P. and Muus, Christoph and Malhotra, Rhea and Parikh, Sumit and Fleming, Mark D. and Oevermann, Lena and Schulte, Johannes and Eckert, Cornelia and Kundaje, Anshul and Smibert, Peter and Vardhana, Santosha A. and Satpathy, Ansuman T. and Regev, Aviv and Sankaran, Vijay G. and Agarwal, Suneet and Ludwig, Leif S.},
  doi          = {10.1038/s41588-023-01433-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1198-1209},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-omics of mitochondrial DNA disorders reveals dynamics of purifying selection across human immune cells},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Transcriptomic classes of BCR-ABL1 lymphoblastic leukemia.
<em>NG</em>, <em>55</em>(7), 1186–1197. (<a
href="https://doi.org/10.1038/s41588-023-01429-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In BCR-ABL1 lymphoblastic leukemia, treatment heterogeneity to tyrosine kinase inhibitors (TKIs), especially in the absence of kinase domain mutations in BCR-ABL1, is poorly understood. Through deep molecular profiling, we uncovered three transcriptomic subtypes of BCR-ABL1 lymphoblastic leukemia, each representing a maturation arrest at a stage of B-cell progenitor differentiation. An earlier arrest was associated with lineage promiscuity, treatment refractoriness and poor patient outcomes. A later arrest was associated with lineage fidelity, durable leukemia remissions and improved patient outcomes. Each maturation arrest was marked by specific genomic events that control different transition points in B-cell development. Interestingly, these events were absent in BCR-ABL1+ preleukemic stem cells isolated from patients regardless of subtype, which supports that transcriptomic phenotypes are determined downstream of the leukemia-initialing event. Overall, our data indicate that treatment response and TKI efficacy are unexpected outcomes of the differentiation stage at which this leukemia transforms. Transcriptomic analysis of BCR-ABL1 lymphoblastic leukemia identifies three subgroups, each associated with a maturation arrest at a specific stage of B-cell progenitor differentiation and distinct genetic and clinical features.},
  archive      = {J_NG},
  author       = {Kim, Jaeseung C. and Chan-Seng-Yue, Michelle and Ge, Sabrina and Zeng, Andy G. X. and Ng, Karen and Gan, Olga I. and Garcia-Prat, Laura and Flores-Figueroa, Eugenia and Woo, Tristan and Zhang, Amy Xin Wei and Arruda, Andrea and Chithambaram, Shivapriya and Dobson, Stephanie M. and Khoo, Amanda and Khan, Shahbaz and Ibrahimova, Narmin and George, Ann and Tierens, Anne and Hitzler, Johann and Kislinger, Thomas and Dick, John E. and McPherson, John D. and Minden, Mark D. and Notta, Faiyaz},
  doi          = {10.1038/s41588-023-01429-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1186-1197},
  shortjournal = {Nat. Genet.},
  title        = {Transcriptomic classes of BCR-ABL1 lymphoblastic leukemia},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatiotemporal transcriptomic maps of whole mouse embryos at
the onset of organogenesis. <em>NG</em>, <em>55</em>(7), 1176–1185. (<a
href="https://doi.org/10.1038/s41588-023-01435-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatiotemporal orchestration of gene expression is required for proper embryonic development. The use of single-cell technologies has begun to provide improved resolution of early regulatory dynamics, including detailed molecular definitions of most cell states during mouse embryogenesis. Here we used Slide-seq to build spatial transcriptomic maps of complete embryonic day (E) 8.5 and E9.0, and partial E9.5 embryos. To support their utility, we developed sc3D, a tool for reconstructing and exploring three-dimensional ‘virtual embryos’, which enables the quantitative investigation of regionalized gene expression patterns. Our measurements along the main embryonic axes of the developing neural tube revealed several previously unannotated genes with distinct spatial patterns. We also characterized the conflicting transcriptional identity of ‘ectopic’ neural tubes that emerge in Tbx6 mutant embryos. Taken together, we present an experimental and computational framework for the spatiotemporal investigation of whole embryonic structures and mutant phenotypes. Slide-seq profiling of mouse embryos at the onset of organogenesis (embryonic days 8.5–9.5) coupled with a new three-dimensional reconstruction and visualization tool (sc3D) provides high-resolution maps of spatiotemporal gene expression dynamics.},
  archive      = {J_NG},
  author       = {Sampath Kumar, Abhishek and Tian, Luyi and Bolondi, Adriano and Hernández, Amèlia Aragonés and Stickels, Robert and Kretzmer, Helene and Murray, Evan and Wittler, Lars and Walther, Maria and Barakat, Gabriel and Haut, Leah and Elkabetz, Yechiel and Macosko, Evan Z. and Guignard, Léo and Chen, Fei and Meissner, Alexander},
  doi          = {10.1038/s41588-023-01435-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1176-1185},
  shortjournal = {Nat. Genet.},
  title        = {Spatiotemporal transcriptomic maps of whole mouse embryos at the onset of organogenesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sequential and directional insulation by conserved CTCF
sites underlies the hox timer in stembryos. <em>NG</em>, <em>55</em>(7),
1164–1175. (<a
href="https://doi.org/10.1038/s41588-023-01426-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During development, Hox genes are temporally activated according to their relative positions on their clusters, contributing to the proper identities of structures along the rostrocaudal axis. To understand the mechanism underlying this Hox timer, we used mouse embryonic stem cell-derived stembryos. Following Wnt signaling, the process involves transcriptional initiation at the anterior part of the cluster and a concomitant loading of cohesin complexes enriched on the transcribed DNA segments, that is, with an asymmetric distribution favoring the anterior part of the cluster. Chromatin extrusion then occurs with successively more posterior CTCF sites acting as transient insulators, thus generating a progressive time delay in the activation of more posterior-located genes due to long-range contacts with a flanking topologically associating domain. Mutant stembryos support this model and reveal that the presence of evolutionary conserved and regularly spaced intergenic CTCF sites controls the precision and the pace of this temporal mechanism. Temporal activation of Hox genes in mouse pseudo-embryos in vitro initiates at the anterior part of the cluster and is accompanied by asymmetric loading of cohesin. Posterior CTCF sites then successively act as transient insulators, regulating the timed transcription of more posterior-located genes.},
  archive      = {J_NG},
  author       = {Rekaik, Hocine and Lopez-Delisle, Lucille and Hintermann, Aurélie and Mascrez, Bénédicte and Bochaton, Célia and Mayran, Alexandre and Duboule, Denis},
  doi          = {10.1038/s41588-023-01426-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1164-1175},
  shortjournal = {Nat. Genet.},
  title        = {Sequential and directional insulation by conserved CTCF sites underlies the hox timer in stembryos},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Noncoding variants alter GATA2 expression in rhombomere 4
motor neurons and cause dominant hereditary congenital facial paresis.
<em>NG</em>, <em>55</em>(7), 1149–1163. (<a
href="https://doi.org/10.1038/s41588-023-01424-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hereditary congenital facial paresis type 1 (HCFP1) is an autosomal dominant disorder of absent or limited facial movement that maps to chromosome 3q21-q22 and is hypothesized to result from facial branchial motor neuron (FBMN) maldevelopment. In the present study, we report that HCFP1 results from heterozygous duplications within a neuron-specific GATA2 regulatory region that includes two enhancers and one silencer, and from noncoding single-nucleotide variants (SNVs) within the silencer. Some SNVs impair binding of NR2F1 to the silencer in vitro and in vivo and attenuate in vivo enhancer reporter expression in FBMNs. Gata2 and its effector Gata3 are essential for inner-ear efferent neuron (IEE) but not FBMN development. A humanized HCFP1 mouse model extends Gata2 expression, favors the formation of IEEs over FBMNs and is rescued by conditional loss of Gata3. These findings highlight the importance of temporal gene regulation in development and of noncoding variation in rare mendelian disease. GATA2 regulatory mutations are associated with hereditary congenital facial paresis in humans. A genetically engineered mouse model recapitulates the human phenotype, showing altered neuron-specific Gata2 expression and a bias in formation of inner-ear efferent neurons over facial branchial motor neurons.},
  archive      = {J_NG},
  author       = {Tenney, Alan P. and Di Gioia, Silvio Alessandro and Webb, Bryn D. and Chan, Wai-Man and de Boer, Elke and Garnai, Sarah J. and Barry, Brenda J. and Ray, Tammy and Kosicki, Michael and Robson, Caroline D. and Zhang, Zhongyang and Collins, Thomas E. and Gelber, Alon and Pratt, Brandon M. and Fujiwara, Yuko and Varshney, Arushi and Lek, Monkol and Warburton, Peter E. and Van Ryzin, Carol and Lehky, Tanya J. and Zalewski, Christopher and King, Kelly A. and Brewer, Carmen C. and Thurm, Audrey and Snow, Joseph and Facio, Flavia M. and Narisu, Narisu and Bonnycastle, Lori L. and Swift, Amy and Chines, Peter S. and Bell, Jessica L. and Mohan, Suresh and Whitman, Mary C. and Staffieri, Sandra E. and Elder, James E. and Demer, Joseph L. and Torres, Alcy and Rachid, Elza and Al-Haddad, Christiane and Boustany, Rose-Mary and Mackey, David A. and Brady, Angela F. and Fenollar-Cortés, María and Fradin, Melanie and Kleefstra, Tjitske and Padberg, George W. and Raskin, Salmo and Sato, Mario Teruo and Orkin, Stuart H. and Parker, Stephen C. J. and Hadlock, Tessa A. and Vissers, Lisenka E. L. M. and van Bokhoven, Hans and Jabs, Ethylin Wang and Collins, Francis S. and Pennacchio, Len A. and Manoli, Irini and Engle, Elizabeth C.},
  doi          = {10.1038/s41588-023-01424-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1149-1163},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding variants alter GATA2 expression in rhombomere 4 motor neurons and cause dominant hereditary congenital facial paresis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rare coding variants in CHRNB2 reduce the likelihood of
smoking. <em>NG</em>, <em>55</em>(7), 1138–1148. (<a
href="https://doi.org/10.1038/s41588-023-01417-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human genetic studies of smoking behavior have been thus far largely limited to common variants. Studying rare coding variants has the potential to identify drug targets. We performed an exome-wide association study of smoking phenotypes in up to 749,459 individuals and discovered a protective association in CHRNB2, encoding the β2 subunit of the α4β2 nicotine acetylcholine receptor. Rare predicted loss-of-function and likely deleterious missense variants in CHRNB2 in aggregate were associated with a 35% decreased odds for smoking heavily (odds ratio (OR) = 0.65, confidence interval (CI) = 0.56–0.76, P = 1.9 × 10−8). An independent common variant association in the protective direction ( rs2072659 ; OR = 0.96; CI = 0.94–0.98; P = 5.3 × 10−6) was also evident, suggesting an allelic series. Our findings in humans align with decades-old experimental observations in mice that β2 loss abolishes nicotine-mediated neuronal responses and attenuates nicotine self-administration. Our genetic discovery will inspire future drug designs targeting CHRNB2 in the brain for the treatment of nicotine addiction. An exome-wide association study of six smoking phenotypes in up to 749,459 individuals identifies associations of rare coding variants in CHRNB2 that may reduce the likelihood of smoking.},
  archive      = {J_NG},
  author       = {Rajagopal, Veera M. and Watanabe, Kyoko and Mbatchou, Joelle and Ayer, Ariane and Quon, Peter and Sharma, Deepika and Kessler, Michael D. and Praveen, Kavita and Gelfman, Sahar and Parikshak, Neelroop and Otto, Jacqueline M. and Bao, Suying and Chim, Shek Man and Pavlopoulos, Elias and Avbersek, Andreja and Kapoor, Manav and Chen, Esteban and Jones, Marcus B. and Leblanc, Michelle and Emberson, Jonathan and Collins, Rory and Torres, Jason and Morales, Pablo Kuri and Tapia-Conyer, Roberto and Alegre, Jesus and Berumen, Jaime and Shuldiner, Alan R. and Balasubramanian, Suganthi and Abecasis, Gonçalo R. and Kang, Hyun M. and Marchini, Jonathan and Stahl, Eli A. and Jorgenson, Eric and Sanchez, Robert and Liedtke, Wolfgang and Anderson, Matthew and Cantor, Michael and Lederer, David and Baras, Aris and Coppola, Giovanni},
  doi          = {10.1038/s41588-023-01417-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1138-1148},
  shortjournal = {Nat. Genet.},
  title        = {Rare coding variants in CHRNB2 reduce the likelihood of smoking},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cross-ancestry genome-wide association meta-analyses of
hippocampal and subfield volumes. <em>NG</em>, <em>55</em>(7),
1126–1137. (<a
href="https://doi.org/10.1038/s41588-023-01425-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The hippocampus is critical for memory and cognition and neuropsychiatric disorders, and its subfields differ in architecture and function. Genome-wide association studies on hippocampal and subfield volumes are mainly conducted in European populations; however, other ancestral populations are under-represented. Here we conduct cross-ancestry genome-wide association meta-analyses in 65,791 individuals for hippocampal volume and 38,977 for subfield volumes, including 7,009 individuals of East Asian ancestry. We identify 339 variant–trait associations at P &amp;lt; 1.13 × 10−9 for 44 hippocampal traits, including 23 new associations. Common genetic variants have similar effects on hippocampal traits across ancestries, although ancestry-specific associations exist. Cross-ancestry analysis improves the fine-mapping precision and the prediction performance of polygenic scores in under-represented populations. These genetic variants are enriched for Wnt signaling and neuron differentiation and affect cognition, emotion and neuropsychiatric disorders. These findings may provide insight into the genetic architectures of hippocampal and subfield volumes. Genome-wide association meta-analyses in populations of East Asian and European ancestries identify variant–trait associations for 44 hippocampal traits and provide insight into the genetic architectures of hippocampal and subfield volumes.},
  archive      = {J_NG},
  author       = {Liu, Nana and Zhang, Longjiang and Tian, Tian and Cheng, Jingliang and Zhang, Bing and Qiu, Shijun and Geng, Zuojun and Cui, Guangbin and Zhang, Quan and Liao, Weihua and Yu, Yongqiang and Zhang, Hui and Gao, Bo and Xu, Xiaojun and Han, Tong and Yao, Zhenwei and Qin, Wen and Liu, Feng and Liang, Meng and Xu, Qiang and Fu, Jilian and Xu, Jiayuan and Zhu, Wenzhen and Zhang, Peng and Li, Wei and Shi, Dapeng and Wang, Caihong and Lui, Su and Yan, Zhihan and Chen, Feng and Li, Jiance and Zhang, Jing and Wang, Dawei and Shen, Wen and Miao, Yanwei and Xian, Junfang and Gao, Jia-Hong and Zhang, Xiaochu and Li, Mulin Jun and Xu, Kai and Zuo, Xi-Nian and Wang, Meiyun and Ye, Zhaoxiang and Yu, Chunshui},
  doi          = {10.1038/s41588-023-01425-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1126-1137},
  shortjournal = {Nat. Genet.},
  title        = {Cross-ancestry genome-wide association meta-analyses of hippocampal and subfield volumes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Large-scale multitrait genome-wide association analyses
identify hundreds of glaucoma risk loci. <em>NG</em>, <em>55</em>(7),
1116–1125. (<a
href="https://doi.org/10.1038/s41588-023-01428-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glaucoma, a leading cause of irreversible blindness, is a highly heritable human disease. Previous genome-wide association studies have identified over 100 loci for the most common form, primary open-angle glaucoma. Two key glaucoma-associated traits also show high heritability: intraocular pressure and optic nerve head excavation damage quantified as the vertical cup-to-disc ratio. Here, since much of glaucoma heritability remains unexplained, we conducted a large-scale multitrait genome-wide association study in participants of European ancestry combining primary open-angle glaucoma and its two associated traits (total sample size over 600,000) to substantially improve genetic discovery power (263 loci). We further increased our power by then employing a multiancestry approach, which increased the number of independent risk loci to 312, with the vast majority replicating in a large independent cohort from 23andMe, Inc. (total sample size over 2.8 million; 296 loci replicated at P &amp;lt; 0.05, 240 after Bonferroni correction). Leveraging multiomics datasets, we identified many potential druggable genes, including neuro-protection targets likely to act via the optic nerve, a key advance for glaucoma because all existing drugs only target intraocular pressure. We further used Mendelian randomization and genetic correlation-based approaches to identify novel links to other complex traits, including immune-related diseases such as multiple sclerosis and systemic lupus erythematosus. Multitrait genome-wide association analyses identify hundreds of risk loci for primary open-angle glaucoma. Integration with other data types highlights potential new drug targets, including proteins likely to act via the optic nerve.},
  archive      = {J_NG},
  author       = {Han, Xikun and Gharahkhani, Puya and Hamel, Andrew R. and Ong, Jue Sheng and Rentería, Miguel E. and Mehta, Puja and Dong, Xianjun and Pasutto, Francesca and Hammond, Christopher and Young, Terri L. and Hysi, Pirro and Lotery, Andrew J. and Jorgenson, Eric and Choquet, Hélène and Hauser, Michael and Cooke Bailey, Jessica N. and Nakazawa, Toru and Akiyama, Masato and Shiga, Yukihiro and Fuller, Zachary L. and Wang, Xin and Hewitt, Alex W. and Craig, Jamie E. and Pasquale, Louis R. and Mackey, David A. and Wiggs, Janey L. and Khawaja, Anthony P. and Segrè, Ayellet V. and MacGregor, Stuart},
  doi          = {10.1038/s41588-023-01428-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1116-1125},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale multitrait genome-wide association analyses identify hundreds of glaucoma risk loci},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide association study of thoracic aortic aneurysm
and dissection in the million veteran program. <em>NG</em>,
<em>55</em>(7), 1106–1115. (<a
href="https://doi.org/10.1038/s41588-023-01420-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The current understanding of the genetic determinants of thoracic aortic aneurysms and dissections (TAAD) has largely been informed through studies of rare, Mendelian forms of disease. Here, we conducted a genome-wide association study (GWAS) of TAAD, testing ~25 million DNA sequence variants in 8,626 participants with and 453,043 participants without TAAD in the Million Veteran Program, with replication in an independent sample of 4,459 individuals with and 512,463 without TAAD from six cohorts. We identified 21 TAAD risk loci, 17 of which have not been previously reported. We leverage multiple downstream analytic methods to identify causal TAAD risk genes and cell types and provide human genetic evidence that TAAD is a non-atherosclerotic aortic disorder distinct from other forms of vascular disease. Our results demonstrate that the genetic architecture of TAAD mirrors that of other complex traits and that it is not solely inherited through protein-altering variants of large effect size. A genome-wide association study of thoracic aortic aneurysms and dissections (TAAD) in the Million Veteran Program identifies 17 new risk loci and demonstrates that the genetic architecture of TAAD mirrors that of other complex traits.},
  archive      = {J_NG},
  author       = {Klarin, Derek and Devineni, Poornima and Sendamarai, Anoop K. and Angueira, Anthony R. and Graham, Sarah E. and Shen, Ying H. and Levin, Michael G. and Pirruccello, James P. and Surakka, Ida and Karnam, Purushotham R. and Roychowdhury, Tanmoy and Li, Yanming and Wang, Minxian and Aragam, Krishna G. and Paruchuri, Kaavya and Zuber, Verena and Shakt, Gabrielle E. and Tsao, Noah L. and Judy, Renae L. and Vy, Ha My T. and Verma, Shefali S. and Rader, Daniel J. and Do, Ron and Bavaria, Joseph E. and Nadkarni, Girish N. and Ritchie, Marylyn D. and Burgess, Stephen and Guo, Dong-chuan and Ellinor, Patrick T. and LeMaire, Scott A. and Milewicz, Dianna M. and Willer, Cristen J. and Natarajan, Pradeep and Tsao, Philip S. and Pyarajan, Saiju and Damrauer, Scott M.},
  doi          = {10.1038/s41588-023-01420-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1106-1115},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study of thoracic aortic aneurysm and dissection in the million veteran program},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide association analyses define pathogenic signaling
pathways and prioritize drug targets for IgA nephropathy. <em>NG</em>,
<em>55</em>(7), 1091–1105. (<a
href="https://doi.org/10.1038/s41588-023-01422-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {IgA nephropathy (IgAN) is a progressive form of kidney disease defined by glomerular deposition of IgA. Here we performed a genome-wide association study of 10,146 kidney-biopsy-diagnosed IgAN cases and 28,751 controls across 17 international cohorts. We defined 30 genome-wide significant risk loci explaining 11% of disease risk. A total of 16 loci were new, including TNFSF4/TNFSF18, REL, CD28, PF4V1, LY86, LYN, ANXA3, TNFSF8/TNFSF15, REEP3, ZMIZ1, OVOL1/RELA, ETS1, IGH, IRF8, TNFRSF13B and FCAR. The risk loci were enriched in gene orthologs causing abnormal IgA levels when genetically manipulated in mice. We also observed a positive genetic correlation between IgAN and serum IgA levels. High polygenic score for IgAN was associated with earlier onset of kidney failure. In a comprehensive functional annotation analysis of candidate causal genes, we observed convergence of biological candidates on a common set of inflammatory signaling pathways and cytokine ligand–receptor pairs, prioritizing potential new drug targets. Genome-wide association analyses identify 30 risk loci for IgA nephropathy. Functional annotations of putative causal genes converge on inflammatory signaling pathways and cytokine ligand–receptor pairs, prioritizing potential new drug targets.},
  archive      = {J_NG},
  author       = {Kiryluk, Krzysztof and Sanchez-Rodriguez, Elena and Zhou, Xu-Jie and Zanoni, Francesca and Liu, Lili and Mladkova, Nikol and Khan, Atlas and Marasa, Maddalena and Zhang, Jun Y. and Balderes, Olivia and Sanna-Cherchi, Simone and Bomback, Andrew S. and Canetta, Pietro A. and Appel, Gerald B. and Radhakrishnan, Jai and Trimarchi, Hernan and Sprangers, Ben and Cattran, Daniel C. and Reich, Heather and Pei, York and Ravani, Pietro and Galesic, Kresimir and Maixnerova, Dita and Tesar, Vladimir and Stengel, Benedicte and Metzger, Marie and Canaud, Guillaume and Maillard, Nicolas and Berthoux, Francois and Berthelot, Laureline and Pillebout, Evangeline and Monteiro, Renato and Nelson, Raoul and Wyatt, Robert J. and Smoyer, William and Mahan, John and Samhar, Al-Akash and Hidalgo, Guillermo and Quiroga, Alejandro and Weng, Patricia and Sreedharan, Raji and Selewski, David and Davis, Keefe and Kallash, Mahmoud and Vasylyeva, Tetyana L. and Rheault, Michelle and Chishti, Aftab and Ranch, Daniel and Wenderfer, Scott E. and Samsonov, Dmitry and Claes, Donna J. and Akchurin, Oleh and Goumenos, Dimitrios and Stangou, Maria and Nagy, Judit and Kovacs, Tibor and Fiaccadori, Enrico and Amoroso, Antonio and Barlassina, Cristina and Cusi, Daniele and Del Vecchio, Lucia and Battaglia, Giovanni Giorgio and Bodria, Monica and Boer, Emanuela and Bono, Luisa and Boscutti, Giuliano and Caridi, Gianluca and Lugani, Francesca and Ghiggeri, GianMarco and Coppo, Rosanna and Peruzzi, Licia and Esposito, Vittoria and Esposito, Ciro and Feriozzi, Sandro and Polci, Rosaria and Frasca, Giovanni and Galliani, Marco and Garozzo, Maurizio and Mitrotti, Adele and Gesualdo, Loreto and Granata, Simona and Zaza, Gianluigi and Londrino, Francesco and Magistroni, Riccardo and Pisani, Isabella and Magnano, Andrea and Marcantoni, Carmelita and Messa, Piergiorgio and Mignani, Renzo and Pani, Antonello and Ponticelli, Claudio and Roccatello, Dario and Salvadori, Maurizio and Salvi, Erica and Santoro, Domenico and Gembillo, Guido and Savoldi, Silvana and Spotti, Donatella and Zamboli, Pasquale and Izzi, Claudia and Alberici, Federico and Delbarba, Elisa and Florczak, Michał and Krata, Natalia and Mucha, Krzysztof and Pączek, Leszek and Niemczyk, Stanisław and Moszczuk, Barbara and Pańczyk-Tomaszewska, Malgorzata and Mizerska-Wasiak, Malgorzata and Perkowska-Ptasińska, Agnieszka and Bączkowska, Teresa and Durlik, Magdalena and Pawlaczyk, Krzysztof and Sikora, Przemyslaw and Zaniew, Marcin and Kaminska, Dorota and Krajewska, Magdalena and Kuzmiuk-Glembin, Izabella and Heleniak, Zbigniew and Bullo-Piontecka, Barbara and Liberek, Tomasz and Dębska-Slizien, Alicja and Hryszko, Tomasz and Materna-Kiryluk, Anna and Miklaszewska, Monika and Szczepańska, Maria and Dyga, Katarzyna and Machura, Edyta and Siniewicz-Luzeńczyk, Katarzyna and Pawlak-Bratkowska, Monika and Tkaczyk, Marcin and Runowski, Dariusz and Kwella, Norbert and Drożdż, Dorota and Habura, Ireneusz and Kronenberg, Florian and Prikhodina, Larisa and van Heel, David and Fontaine, Bertrand and Cotsapas, Chris and Wijmenga, Cisca and Franke, Andre and Annese, Vito and Gregersen, Peter K. and Parameswaran, Sreeja and Weirauch, Matthew and Kottyan, Leah and Harley, John B. and Suzuki, Hitoshi and Narita, Ichiei and Goto, Shin and Lee, Hajeong and Kim, Dong Ki and Kim, Yon Su and Park, Jin-Ho and Cho, BeLong and Choi, Murim and Van Wijk, Ans and Huerta, Ana and Ars, Elisabet and Ballarin, Jose and Lundberg, Sigrid and Vogt, Bruno and Mani, Laila-Yasmin and Caliskan, Yasar and Barratt, Jonathan and Abeygunaratne, Thilini and Kalra, Philip A. and Gale, Daniel P. and Panzer, Ulf and Rauen, Thomas and Floege, Jürgen and Schlosser, Pascal and Ekici, Arif B. and Eckardt, Kai-Uwe and Chen, Nan and Xie, Jingyuan and Lifton, Richard P. and Loos, Ruth J. F. and Kenny, Eimear E. and Ionita-Laza, Iuliana and Köttgen, Anna and Julian, Bruce A. and Novak, Jan and Scolari, Francesco and Zhang, Hong and Gharavi, Ali G.},
  doi          = {10.1038/s41588-023-01422-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1091-1105},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses define pathogenic signaling pathways and prioritize drug targets for IgA nephropathy},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Imputation of low-coverage sequencing data from 150,119 UK
biobank genomes. <em>NG</em>, <em>55</em>(7), 1088–1090. (<a
href="https://doi.org/10.1038/s41588-023-01438-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The release of 150,119 UK Biobank sequences represents an unprecedented opportunity as a reference panel to impute low-coverage whole-genome sequencing data with high accuracy but current methods cannot cope with the size of the data. Here we introduce GLIMPSE2, a low-coverage whole-genome sequencing imputation method that scales sublinearly in both the number of samples and markers, achieving efficient whole-genome imputation from the UK Biobank reference panel while retaining high accuracy for ancient and modern genomes, particularly at rare variants and for very low-coverage samples. GLIMPSE2 is an improved method using sparse models for accurate, efficient and cost-effective genotype imputation in low-coverage whole-genome sequencing data.},
  archive      = {J_NG},
  author       = {Rubinacci, Simone and Hofmeister, Robin J. and Sousa da Mota, Bárbara and Delaneau, Olivier},
  doi          = {10.1038/s41588-023-01438-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1088-1090},
  shortjournal = {Nat. Genet.},
  title        = {Imputation of low-coverage sequencing data from 150,119 UK biobank genomes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cross-ancestry genetic discovery for hippocampal volumetric
traits. <em>NG</em>, <em>55</em>(7), 1086–1087. (<a
href="https://doi.org/10.1038/s41588-023-01427-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cross-ancestry genome-wide association meta-analyses of neuroimaging genetics data from European and East Asian populations identified 339 genetic variant–hippocampal volumetric trait associations (23 new). Further cross-ancestry analyses revealed similar genetic effects on hippocampal volumetric traits between ancestries, with improved fine-mapping precision and predictive accuracy of polygenic scores in the under-represented East Asian population.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01427-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1086-1087},
  shortjournal = {Nat. Genet.},
  title        = {Cross-ancestry genetic discovery for hippocampal volumetric traits},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell multiomic analyses sheds light on mitochondrial
mutational selection. <em>NG</em>, <em>55</em>(7), 1083–1085. (<a
href="https://doi.org/10.1038/s41588-023-01436-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single large-scale mitochondrial DNA deletions cause a spectrum of disease with differing severities and tissue expressivity. Quantification of single-cell deletion levels with multi-modal cellular readouts provides insights into hematopoietic cell fate and disease manifestation.},
  archive      = {J_NG},
  author       = {Pickett, Sarah J. and Hudson, Gavin and Greaves, Laura C.},
  doi          = {10.1038/s41588-023-01436-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1083-1085},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multiomic analyses sheds light on mitochondrial mutational selection},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Improved pathogenicity prediction using primate genomics.
<em>NG</em>, <em>55</em>(7), 1082. (<a
href="https://doi.org/10.1038/s41588-023-01455-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01455-2},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1082},
  shortjournal = {Nat. Genet.},
  title        = {Improved pathogenicity prediction using primate genomics},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Temperature-induced RNA recoding in octopus. <em>NG</em>,
<em>55</em>(7), 1082. (<a
href="https://doi.org/10.1038/s41588-023-01456-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-023-01456-1},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1082},
  shortjournal = {Nat. Genet.},
  title        = {Temperature-induced RNA recoding in octopus},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). The evolving cancer genome. <em>NG</em>, <em>55</em>(7),
1082. (<a href="https://doi.org/10.1038/s41588-023-01457-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01457-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1082},
  shortjournal = {Nat. Genet.},
  title        = {The evolving cancer genome},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Problems with dystopian representations in genetic futurism.
<em>NG</em>, <em>55</em>(7), 1081. (<a
href="https://doi.org/10.1038/s41588-023-01416-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Rueda, Jon},
  doi          = {10.1038/s41588-023-01416-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1081},
  shortjournal = {Nat. Genet.},
  title        = {Problems with dystopian representations in genetic futurism},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Pan-cancer analysis of whole genomes
identifies driver rearrangements promoted by LINE-1 retrotransposition.
<em>NG</em>, <em>55</em>(6), 1080. (<a
href="https://doi.org/10.1038/s41588-023-01319-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Rodriguez-Martin, Bernardo and Alvarez, Eva G. and Baez-Ortega, Adrian and Zamora, Jorge and Supek, Fran and Demeulemeester, Jonas and Santamarina, Martin and Ju, Young Seok and Temes, Javier and Garcia-Souto, Daniel and Detering, Harald and Li, Yilong and Rodriguez-Castro, Jorge and Dueso-Barroso, Ana and Bruzos, Alicia L. and Dentro, Stefan C. and Blanco, Miguel G. and Contino, Gianmarco and Ardeljan, Daniel and Tojo, Marta and Roberts, Nicola D. and Zumalave, Sonia and Edwards, Paul A. and Weischenfeldt, Joachim and Puiggròs, Montserrat and Chong, Zechen and Chen, Ken and Lee, Eunjung Alice and Wala, Jeremiah A. and Raine, Keiran M. and Butler, Adam and Waszak, Sebastian M. and Navarro, Fabio C. P. and Schumacher, Steven E. and Monlong, Jean and Maura, Francesco and Bolli, Niccolo and Bourque, Guillaume and Gerstein, Mark and Park, Peter J. and Wedge, David C. and Beroukhim, Rameen and Torrents, David and Korbel, Jan O. and Martincorena, Iñigo and Fitzgerald, Rebecca C. and Van Loo, Peter and Kazazian, Haig H. and Burns, Kathleen H. and Campbell, Peter J. and Tubio, Jose M. C.},
  doi          = {10.1038/s41588-023-01319-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1080},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Pan-cancer analysis of whole genomes identifies driver rearrangements promoted by LINE-1 retrotransposition},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Disruption of chromatin folding domains
by somatic genomic rearrangements in human cancer. <em>NG</em>,
<em>55</em>(6), 1079. (<a
href="https://doi.org/10.1038/s41588-023-01318-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Akdemir, Kadir C. and Le, Victoria T. and Chandran, Sahaana and Li, Yilong and Verhaak, Roel G. and Beroukhim, Rameen and Campbell, Peter J. and Chin, Lynda and Dixon, Jesse R. and Futreal, P. Andrew},
  doi          = {10.1038/s41588-023-01318-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1079},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Disruption of chromatin folding domains by somatic genomic rearrangements in human cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Comprehensive molecular characterization
of mitochondrial genomes in human cancers. <em>NG</em>, <em>55</em>(6),
1078. (<a href="https://doi.org/10.1038/s41588-023-01317-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Yuan, Yuan and Ju, Young Seok and Kim, Youngwook and Li, Jun and Wang, Yumeng and Yoon, Christopher J. and Yang, Yang and Martincorena, Inigo and Creighton, Chad J. and Weinstein, John N. and Xu, Yanxun and Han, Leng and Kim, Hyung-Lae and Nakagawa, Hidewaki and Park, Keunchil and Campbell, Peter J. and Liang, Han},
  doi          = {10.1038/s41588-023-01317-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1078},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Comprehensive molecular characterization of mitochondrial genomes in human cancers},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: The landscape of viral associations in
human cancers. <em>NG</em>, <em>55</em>(6), 1077. (<a
href="https://doi.org/10.1038/s41588-023-01316-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Zapatka, Marc and Borozan, Ivan and Brewer, Daniel S. and Iskar, Murat and Grundhoff, Adam and Alawi, Malik and Desai, Nikita and Sültmann, Holger and Moch, Holger and Cooper, Colin S. and Eils, Roland and Ferretti, Vincent and Lichter, Peter},
  doi          = {10.1038/s41588-023-01316-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1077},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The landscape of viral associations in human cancers},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Comprehensive analysis of chromothripsis
in 2,658 human cancers using whole-genome sequencing. <em>NG</em>,
<em>55</em>(6), 1076. (<a
href="https://doi.org/10.1038/s41588-023-01315-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Cortés-Ciriano, Isidro and Lee, Jake June-Koo and Xi, Ruibin and Jain, Dhawal and Jung, Youngsook L. and Yang, Lixing and Gordenin, Dmitry and Klimczak, Leszek J. and Zhang, Cheng-Zhong and Pellman, David S. and Park, Peter J.},
  doi          = {10.1038/s41588-023-01315-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1076},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Comprehensive analysis of chromothripsis in 2,658 human cancers using whole-genome sequencing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping interindividual dynamics of innate immune response
at single-cell resolution. <em>NG</em>, <em>55</em>(6), 1066–1075. (<a
href="https://doi.org/10.1038/s41588-023-01421-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common genetic variants across individuals modulate the cellular response to pathogens and are implicated in diverse immune pathologies, yet how they dynamically alter the response upon infection is not well understood. Here, we triggered antiviral responses in human fibroblasts from 68 healthy donors, and profiled tens of thousands of cells using single-cell RNA-sequencing. We developed GASPACHO (GAuSsian Processes for Association mapping leveraging Cell HeterOgeneity), a statistical approach designed to identify nonlinear dynamic genetic effects across transcriptional trajectories of cells. This approach identified 1,275 expression quantitative trait loci (local false discovery rate 10%) that manifested during the responses, many of which were colocalized with susceptibility loci identified by genome-wide association studies of infectious and autoimmune diseases, including the OAS1 splicing quantitative trait locus in a COVID-19 susceptibility locus. In summary, our analytical approach provides a unique framework for delineation of the genetic variants that shape a wide spectrum of transcriptional responses at single-cell resolution. GASPACHO is a statistical method that identifies nonlinear dynamic genetic effects using single-cell RNA-seq data. Analysis of an antiviral response in human fibroblasts identifies 1,275 expression QTLs, many of which colocalize with risk loci for autoimmune and infectious diseases.},
  archive      = {J_NG},
  author       = {Kumasaka, Natsuhiko and Rostom, Raghd and Huang, Ni and Polanski, Krzysztof and Meyer, Kerstin B. and Patel, Sharad and Boyd, Rachel and Gomez, Celine and Barnett, Sam N. and Panousis, Nikolaos I. and Schwartzentruber, Jeremy and Ghoussaini, Maya and Lyons, Paul A. and Calero-Nieto, Fernando J. and Göttgens, Berthold and Barnes, Josephine L. and Worlock, Kaylee B. and Yoshida, Masahiro and Nikolić, Marko Z. and Stephenson, Emily and Reynolds, Gary and Haniffa, Muzlifah and Marioni, John C. and Stegle, Oliver and Hagai, Tzachi and Teichmann, Sarah A.},
  doi          = {10.1038/s41588-023-01421-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1066-1075},
  shortjournal = {Nat. Genet.},
  title        = {Mapping interindividual dynamics of innate immune response at single-cell resolution},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CARMA is a new bayesian model for fine-mapping in
genome-wide association meta-analyses. <em>NG</em>, <em>55</em>(6),
1057–1065. (<a
href="https://doi.org/10.1038/s41588-023-01392-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fine-mapping is commonly used to identify putative causal variants at genome-wide significant loci. Here we propose a Bayesian model for fine-mapping that has several advantages over existing methods, including flexible specification of the prior distribution of effect sizes, joint modeling of summary statistics and functional annotations and accounting for discrepancies between summary statistics and external linkage disequilibrium in meta-analyses. Using simulations, we compare performance with commonly used fine-mapping methods and show that the proposed model has higher power and lower false discovery rate (FDR) when including functional annotations, and higher power, lower FDR and higher coverage for credible sets in meta-analyses. We further illustrate our approach by applying it to a meta-analysis of Alzheimer’s disease genome-wide association studies where we prioritize putatively causal variants and genes. Causal robust mapping method in meta-analysis (CARMA) studies incorporates flexible prior distributions, joint modeling of summary statistics and functional annotations and outlier detection for improved causal variant fine-mapping in genome-wide association meta-analyses.},
  archive      = {J_NG},
  author       = {Yang, Zikun and Wang, Chen and Liu, Linxi and Khan, Atlas and Lee, Annie and Vardarajan, Badri and Mayeux, Richard and Kiryluk, Krzysztof and Ionita-Laza, Iuliana},
  doi          = {10.1038/s41588-023-01392-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1057-1065},
  shortjournal = {Nat. Genet.},
  title        = {CARMA is a new bayesian model for fine-mapping in genome-wide association meta-analyses},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Region capture micro-c reveals coalescence of enhancers and
promoters into nested microcompartments. <em>NG</em>, <em>55</em>(6),
1048–1056. (<a
href="https://doi.org/10.1038/s41588-023-01391-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although enhancers are central regulators of mammalian gene expression, the mechanisms underlying enhancer–promoter (E-P) interactions remain unclear. Chromosome conformation capture (3C) methods effectively capture large-scale three-dimensional (3D) genome structure but struggle to achieve the depth necessary to resolve fine-scale E-P interactions. Here, we develop Region Capture Micro-C (RCMC) by combining micrococcal nuclease (MNase)-based 3C with a tiling region-capture approach and generate the deepest 3D genome maps reported with only modest sequencing. By applying RCMC in mouse embryonic stem cells and reaching the genome-wide equivalent of ~317 billion unique contacts, RCMC reveals previously unresolvable patterns of highly nested and focal 3D interactions, which we term microcompartments. Microcompartments frequently connect enhancers and promoters, and although loss of loop extrusion and inhibition of transcription disrupts some microcompartments, most are largely unaffected. We therefore propose that many E-P interactions form through a compartmentalization mechanism, which may partially explain why acute cohesin depletion only modestly affects global gene expression. Region Capture Micro-C (RCMC) combines MNase-based 3C with a tiling region-capture method. Profiling mouse embryonic stem cells with RCMC identifies nested microcompartments, which connect enhancers and promoters.},
  archive      = {J_NG},
  author       = {Goel, Viraat Y. and Huseyin, Miles K. and Hansen, Anders S.},
  doi          = {10.1038/s41588-023-01391-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1048-1056},
  shortjournal = {Nat. Genet.},
  title        = {Region capture micro-C reveals coalescence of enhancers and promoters into nested microcompartments},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Triplication of the interferon receptor locus contributes to
hallmarks of down syndrome in a mouse model. <em>NG</em>,
<em>55</em>(6), 1034–1047. (<a
href="https://doi.org/10.1038/s41588-023-01399-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Down syndrome (DS), the genetic condition caused by trisomy 21, is characterized by variable cognitive impairment, immune dysregulation, dysmorphogenesis and increased prevalence of diverse co-occurring conditions. The mechanisms by which trisomy 21 causes these effects remain largely unknown. We demonstrate that triplication of the interferon receptor (IFNR) gene cluster on chromosome 21 is necessary for multiple phenotypes in a mouse model of DS. Whole-blood transcriptome analysis demonstrated that IFNR overexpression associates with chronic interferon hyperactivity and inflammation in people with DS. To define the contribution of this locus to DS phenotypes, we used genome editing to correct its copy number in a mouse model of DS, which normalized antiviral responses, prevented heart malformations, ameliorated developmental delays, improved cognition and attenuated craniofacial anomalies. Triplication of the Ifnr locus modulates hallmarks of DS in mice, suggesting that trisomy 21 elicits an interferonopathy potentially amenable to therapeutic intervention. A mouse model of Down syndrome (DS) highlights the importance of triplication of the IFNR gene cluster for a variety of DS-associated traits. Copy number correction resulted in amelioration of multiple phenotypes associated with the condition.},
  archive      = {J_NG},
  author       = {Waugh, Katherine A. and Minter, Ross and Baxter, Jessica and Chi, Congwu and Galbraith, Matthew D. and Tuttle, Kathryn D. and Eduthan, Neetha P. and Kinning, Kohl T. and Andrysik, Zdenek and Araya, Paula and Dougherty, Hannah and Dunn, Lauren N. and Ludwig, Michael and Schade, Kyndal A. and Tracy, Dayna and Smith, Keith P. and Granrath, Ross E. and Busquet, Nicolas and Khanal, Santosh and Anderson, Ryan D. and Cox, Liza L. and Estrada, Belinda Enriquez and Rachubinski, Angela L. and Lyford, Hannah R. and Britton, Eleanor C. and Fantauzzo, Katherine A. and Orlicky, David J. and Matsuda, Jennifer L. and Song, Kunhua and Cox, Timothy C. and Sullivan, Kelly D. and Espinosa, Joaquin M.},
  doi          = {10.1038/s41588-023-01399-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1034-1047},
  shortjournal = {Nat. Genet.},
  title        = {Triplication of the interferon receptor locus contributes to hallmarks of down syndrome in a mouse model},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Clonal evolution during metastatic spread in high-risk
neuroblastoma. <em>NG</em>, <em>55</em>(6), 1022–1033. (<a
href="https://doi.org/10.1038/s41588-023-01395-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patients with high-risk neuroblastoma generally present with widely metastatic disease and often relapse despite intensive therapy. As most studies to date focused on diagnosis-relapse pairs, our understanding of the genetic and clonal dynamics of metastatic spread and disease progression remain limited. Here, using genomic profiling of 470 sequential and spatially separated samples from 283 patients, we characterize subtype-specific genetic evolutionary trajectories from diagnosis through progression and end-stage metastatic disease. Clonal tracing timed disease initiation to embryogenesis. Continuous acquisition of structural variants at disease-defining loci (MYCN, TERT, MDM2-CDK4) followed by convergent evolution of mutations targeting shared pathways emerged as the predominant feature of progression. At diagnosis metastatic clones were already established at distant sites where they could stay dormant, only to cause relapses years later and spread via metastasis-to-metastasis and polyclonal seeding after therapy. Genomic and transcriptomic analysis of 470 mostly high-risk neuroblastomas collected from 283 patients delineates subtype-specific evolutionary patterns and progression-related convergent evolution and describes the clonal dynamics of metastases.},
  archive      = {J_NG},
  author       = {Gundem, Gunes and Levine, Max F. and Roberts, Stephen S. and Cheung, Irene Y. and Medina-Martínez, Juan S. and Feng, Yi and Arango-Ossa, Juan E. and Chadoutaud, Loic and Rita, Mathieu and Asimomitis, Georgios and Zhou, Joe and You, Daoqi and Bouvier, Nancy and Spitzer, Barbara and Solit, David B. and Dela Cruz, Filemon and LaQuaglia, Michael P. and Kushner, Brian H. and Modak, Shakeel and Shukla, Neerav and Iacobuzio-Donahue, Christine A. and Kung, Andrew L. and Cheung, Nai-Kong V. and Papaemmanuil, Elli},
  doi          = {10.1038/s41588-023-01395-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1022-1033},
  shortjournal = {Nat. Genet.},
  title        = {Clonal evolution during metastatic spread in high-risk neuroblastoma},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Somatic mutations of CADM1 in aldosterone-producing adenomas
and gap junction-dependent regulation of aldosterone production.
<em>NG</em>, <em>55</em>(6), 1009–1021. (<a
href="https://doi.org/10.1038/s41588-023-01403-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aldosterone-producing adenomas (APAs) are the commonest curable cause of hypertension. Most have gain-of-function somatic mutations of ion channels or transporters. Herein we report the discovery, replication and phenotype of mutations in the neuronal cell adhesion gene CADM1. Independent whole exome sequencing of 40 and 81 APAs found intramembranous p.Val380Asp or p.Gly379Asp variants in two patients whose hypertension and periodic primary aldosteronism were cured by adrenalectomy. Replication identified two more APAs with each variant (total, n = 6). The most upregulated gene (10- to 25-fold) in human adrenocortical H295R cells transduced with the mutations (compared to wildtype) was CYP11B2 (aldosterone synthase), and biological rhythms were the most differentially expressed process. CADM1 knockdown or mutation inhibited gap junction (GJ)-permeable dye transfer. GJ blockade by Gap27 increased CYP11B2 similarly to CADM1 mutation. Human adrenal zona glomerulosa (ZG) expression of GJA1 (the main GJ protein) was patchy, and annular GJs (sequelae of GJ communication) were less prominent in CYP11B2-positive micronodules than adjacent ZG. Somatic mutations of CADM1 cause reversible hypertension and reveal a role for GJ communication in suppressing physiological aldosterone production. Recurrent somatic mutations altering residues in the transmembrane domain of CADM1 are identified in aldosterone-producing adenomas. Follow-up studies implicate a role of gap junction communication in regulating aldosterone production.},
  archive      = {J_NG},
  author       = {Wu, Xilin and Azizan, Elena A. B. and Goodchild, Emily and Garg, Sumedha and Hagiyama, Man and Cabrera, Claudia P. and Fernandes-Rosa, Fabio L. and Boulkroun, Sheerazed and Kuan, Jyn Ling and Tiang, Zenia and David, Alessia and Murakami, Masanori and Mein, Charles A. and Wozniak, Eva and Zhao, Wanfeng and Marker, Alison and Buss, Folma and Saleeb, Rebecca S. and Salsbury, Jackie and Tezuka, Yuta and Satoh, Fumitoshi and Oki, Kenji and Udager, Aaron M. and Cohen, Debbie L. and Wachtel, Heather and King, Peter J. and Drake, William M. and Gurnell, Mark and Ceral, Jiri and Ryska, Ales and Mustangin, Muaatamarulain and Wong, Yin Ping and Tan, Geok Chin and Solar, Miroslav and Reincke, Martin and Rainey, William E. and Foo, Roger S. and Takaoka, Yutaka and Murray, Sandra A. and Zennaro, Maria-Christina and Beuschlein, Felix and Ito, Akihiko and Brown, Morris J.},
  doi          = {10.1038/s41588-023-01403-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1009-1021},
  shortjournal = {Nat. Genet.},
  title        = {Somatic mutations of CADM1 in aldosterone-producing adenomas and gap junction-dependent regulation of aldosterone production},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetic studies of paired metabolomes reveal enzymatic and
transport processes at the interface of plasma and urine. <em>NG</em>,
<em>55</em>(6), 995–1008. (<a
href="https://doi.org/10.1038/s41588-023-01409-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The kidneys operate at the interface of plasma and urine by clearing molecular waste products while retaining valuable solutes. Genetic studies of paired plasma and urine metabolomes may identify underlying processes. We conducted genome-wide studies of 1,916 plasma and urine metabolites and detected 1,299 significant associations. Associations with 40% of implicated metabolites would have been missed by studying plasma alone. We detected urine-specific findings that provide information about metabolite reabsorption in the kidney, such as aquaporin (AQP)-7-mediated glycerol transport, and different metabolomic footprints of kidney-expressed proteins in plasma and urine that are consistent with their localization and function, including the transporters NaDC3 (SLC13A3) and ASBT (SLC10A2). Shared genetic determinants of 7,073 metabolite–disease combinations represent a resource to better understand metabolic diseases and revealed connections of dipeptidase 1 with circulating digestive enzymes and with hypertension. Extending genetic studies of the metabolome beyond plasma yields unique insights into processes at the interface of body compartments. Genome-wide studies of 1,916 plasma and urine metabolites from 5,023 participants of the German Chronic Kidney Disease study provide insights into systemic and kidney-specific enzymatic and transport processes.},
  archive      = {J_NG},
  author       = {Schlosser, Pascal and Scherer, Nora and Grundner-Culemann, Franziska and Monteiro-Martins, Sara and Haug, Stefan and Steinbrenner, Inga and Uluvar, Burulça and Wuttke, Matthias and Cheng, Yurong and Ekici, Arif B. and Gyimesi, Gergely and Karoly, Edward D. and Kotsis, Fruzsina and Mielke, Johanna and Gomez, Maria F. and Yu, Bing and Grams, Morgan E. and Coresh, Josef and Boerwinkle, Eric and Köttgen, Michael and Kronenberg, Florian and Meiselbach, Heike and Mohney, Robert P. and Akilesh, Shreeram and Schmidts, Miriam and Hediger, Matthias A. and Schultheiss, Ulla T. and Eckardt, Kai-Uwe and Oefner, Peter J. and Sekula, Peggy and Li, Yong and Köttgen, Anna},
  doi          = {10.1038/s41588-023-01409-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {995-1008},
  shortjournal = {Nat. Genet.},
  title        = {Genetic studies of paired metabolomes reveal enzymatic and transport processes at the interface of plasma and urine},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integrating genetics with single-cell multiomic measurements
across disease states identifies mechanisms of beta cell dysfunction in
type 2 diabetes. <em>NG</em>, <em>55</em>(6), 984–994. (<a
href="https://doi.org/10.1038/s41588-023-01397-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dysfunctional pancreatic islet beta cells are a hallmark of type 2 diabetes (T2D), but a comprehensive understanding of the underlying mechanisms, including gene dysregulation, is lacking. Here we integrate information from measurements of chromatin accessibility, gene expression and function in single beta cells with genetic association data to nominate disease-causal gene regulatory changes in T2D. Using machine learning on chromatin accessibility data from 34 nondiabetic, pre-T2D and T2D donors, we identify two transcriptionally and functionally distinct beta cell subtypes that undergo an abundance shift during T2D progression. Subtype-defining accessible chromatin is enriched for T2D risk variants, suggesting a causal contribution of subtype identity to T2D. Both beta cell subtypes exhibit activation of a stress-response transcriptional program and functional impairment in T2D, which is probably induced by the T2D-associated metabolic environment. Our findings demonstrate the power of multimodal single-cell measurements combined with machine learning for characterizing mechanisms of complex diseases. Single-cell multiomic and functional characterization of human pancreatic islets identifies two beta cell subtypes correlated with type 2 diabetes progression that exhibit distinct gene regulatory programs and electrophysiological phenotypes.},
  archive      = {J_NG},
  author       = {Wang, Gaowei and Chiou, Joshua and Zeng, Chun and Miller, Michael and Matta, Ileana and Han, Jee Yun and Kadakia, Nikita and Okino, Mei-Lin and Beebe, Elisha and Mallick, Medhavi and Camunas-Soler, Joan and dos Santos, Theodore and Dai, Xiao-Qing and Ellis, Cara and Hang, Yan and Kim, Seung K. and MacDonald, Patrick E. and Kandeel, Fouad R. and Preissl, Sebastian and Gaulton, Kyle J. and Sander, Maike},
  doi          = {10.1038/s41588-023-01397-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {984-994},
  shortjournal = {Nat. Genet.},
  title        = {Integrating genetics with single-cell multiomic measurements across disease states identifies mechanisms of beta cell dysfunction in type 2 diabetes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide association study and functional
characterization identifies candidate genes for insulin-stimulated
glucose uptake. <em>NG</em>, <em>55</em>(6), 973–983. (<a
href="https://doi.org/10.1038/s41588-023-01408-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Distinct tissue-specific mechanisms mediate insulin action in fasting and postprandial states. Previous genetic studies have largely focused on insulin resistance in the fasting state, where hepatic insulin action dominates. Here we studied genetic variants influencing insulin levels measured 2 h after a glucose challenge in &amp;gt;55,000 participants from three ancestry groups. We identified ten new loci (P &amp;lt; 5 × 10−8) not previously associated with postchallenge insulin resistance, eight of which were shown to share their genetic architecture with type 2 diabetes in colocalization analyses. We investigated candidate genes at a subset of associated loci in cultured cells and identified nine candidate genes newly implicated in the expression or trafficking of GLUT4, the key glucose transporter in postprandial glucose uptake in muscle and fat. By focusing on postprandial insulin resistance, we highlighted the mechanisms of action at type 2 diabetes loci that are not adequately captured by studies of fasting glycemic traits. Genome-wide association analyses of two oral glucose tolerance test-derived measures of postprandial insulin resistance discover ten new loci. Functional characterization identifies nine candidate genes implicated in the regulation of GLUT4.},
  archive      = {J_NG},
  author       = {Williamson, Alice and Norris, Dougall M. and Yin, Xianyong and Broadaway, K. Alaine and Moxley, Anne H. and Vadlamudi, Swarooparani and Wilson, Emma P. and Jackson, Anne U. and Ahuja, Vasudha and Andersen, Mette K. and Arzumanyan, Zorayr and Bonnycastle, Lori L. and Bornstein, Stefan R. and Bretschneider, Maxi P. and Buchanan, Thomas A. and Chang, Yi-Cheng and Chuang, Lee-Ming and Chung, Ren-Hua and Clausen, Tine D. and Damm, Peter and Delgado, Graciela E. and de Mello, Vanessa D. and Dupuis, Josée and Dwivedi, Om P. and Erdos, Michael R. and Silva, Lilian Fernandes and Frayling, Timothy M. and Gieger, Christian and Goodarzi, Mark O. and Guo, Xiuqing and Gustafsson, Stefan and Hakaste, Liisa and Hammar, Ulf and Hatem, Gad and Herrmann, Sandra and Højlund, Kurt and Horn, Katrin and Hsueh, Willa A. and Hung, Yi-Jen and Hwu, Chii-Min and Jonsson, Anna and Kårhus, Line L. and Kleber, Marcus E. and Kovacs, Peter and Lakka, Timo A. and Lauzon, Marie and Lee, I-Te and Lindgren, Cecilia M. and Lindström, Jaana and Linneberg, Allan and Liu, Ching-Ti and Luan, Jian’an and Aly, Dina Mansour and Mathiesen, Elisabeth and Moissl, Angela P. and Morris, Andrew P. and Narisu, Narisu and Perakakis, Nikolaos and Peters, Annette and Prasad, Rashmi B. and Rodionov, Roman N. and Roll, Kathryn and Rundsten, Carsten F. and Sarnowski, Chloé and Savonen, Kai and Scholz, Markus and Sharma, Sapna and Stinson, Sara E. and Suleman, Sufyan and Tan, Jingyi and Taylor, Kent D. and Uusitupa, Matti and Vistisen, Dorte and Witte, Daniel R. and Walther, Romy and Wu, Peitao and Xiang, Anny H. and Zethelius, Björn and Ahlqvist, Emma and Bergman, Richard N. and Chen, Yii-Der Ida and Collins, Francis S. and Fall, Tove and Florez, Jose C. and Fritsche, Andreas and Grallert, Harald and Groop, Leif and Hansen, Torben and Koistinen, Heikki A. and Komulainen, Pirjo and Laakso, Markku and Lind, Lars and Loeffler, Markus and März, Winfried and Meigs, James B. and Raffel, Leslie J. and Rauramaa, Rainer and Rotter, Jerome I. and Schwarz, Peter E. H. and Stumvoll, Michael and Sundström, Johan and Tönjes, Anke and Tuomi, Tiinamaija and Tuomilehto, Jaakko and Wagner, Robert and Barroso, Inês and Walker, Mark and Grarup, Niels and Boehnke, Michael and Wareham, Nicholas J. and Mohlke, Karen L. and Wheeler, Eleanor and O’Rahilly, Stephen and Fazakerley, Daniel J. and Langenberg, Claudia},
  doi          = {10.1038/s41588-023-01408-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {973-983},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study and functional characterization identifies candidate genes for insulin-stimulated glucose uptake},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide association meta-analysis of spontaneous
coronary artery dissection identifies risk variants and genes related to
artery integrity and tissue-mediated coagulation. <em>NG</em>,
<em>55</em>(6), 964–972. (<a
href="https://doi.org/10.1038/s41588-023-01410-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spontaneous coronary artery dissection (SCAD) is an understudied cause of myocardial infarction primarily affecting women. It is not known to what extent SCAD is genetically distinct from other cardiovascular diseases, including atherosclerotic coronary artery disease (CAD). Here we present a genome-wide association meta-analysis (1,917 cases and 9,292 controls) identifying 16 risk loci for SCAD. Integrative functional annotations prioritized genes that are likely to be regulated in vascular smooth muscle cells and artery fibroblasts and implicated in extracellular matrix biology. One locus containing the tissue factor gene F3, which is involved in blood coagulation cascade initiation, appears to be specific for SCAD risk. Several associated variants have diametrically opposite associations with CAD, suggesting that shared biological processes contribute to both diseases, but through different mechanisms. We also infer a causal role for high blood pressure in SCAD. Our findings provide novel pathophysiological insights involving arterial integrity and tissue-mediated coagulation in SCAD and set the stage for future specific therapeutics and preventions. Genome-wide association analyses identify risk loci for spontaneous coronary artery dissection and implicate arterial integrity and tissue-mediated coagulation in the disease etiology. Several risk variants show opposite effects on coronary artery disease risk.},
  archive      = {J_NG},
  author       = {Adlam, David and Berrandou, Takiy-Eddine and Georges, Adrien and Nelson, Christopher P. and Giannoulatou, Eleni and Henry, Joséphine and Ma, Lijiang and Blencowe, Montgomery and Turley, Tamiel N. and Yang, Min-Lee and Chopade, Sandesh and Finan, Chris and Braund, Peter S. and Sadeg-Sayoud, Ines and Iismaa, Siiri E. and Kosel, Matthew L. and Zhou, Xiang and Hamby, Stephen E. and Cheng, Jenny and Liu, Lu and Tarr, Ingrid and Muller, David W. M. and d’Escamard, Valentina and King, Annette and Brunham, Liam R. and Baranowska-Clarke, Ania A. and Debette, Stéphanie and Amouyel, Philippe and Olin, Jeffrey W. and Patil, Snehal and Hesselson, Stephanie E. and Junday, Keerat and Kanoni, Stavroula and Aragam, Krishna G. and Butterworth, Adam S. and Tweet, Marysia S. and Gulati, Rajiv and Combaret, Nicolas and Kadian-Dodov, Daniella and Kalman, Jonathan M. and Fatkin, Diane and Hingorani, Aroon D. and Saw, Jacqueline and Webb, Tom R. and Hayes, Sharonne N. and Yang, Xia and Ganesh, Santhi K. and Olson, Timothy M. and Kovacic, Jason C. and Graham, Robert M. and Samani, Nilesh J. and Bouatia-Naji, Nabila},
  doi          = {10.1038/s41588-023-01410-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {964-972},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analysis of spontaneous coronary artery dissection identifies risk variants and genes related to artery integrity and tissue-mediated coagulation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Gene expression in african americans, puerto ricans and
mexican americans reveals ancestry-specific patterns of genetic
architecture. <em>NG</em>, <em>55</em>(6), 952–963. (<a
href="https://doi.org/10.1038/s41588-023-01377-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We explored ancestry-related differences in the genetic architecture of whole-blood gene expression using whole-genome and RNA sequencing data from 2,733 African Americans, Puerto Ricans and Mexican Americans. We found that heritability of gene expression significantly increased with greater proportions of African genetic ancestry and decreased with higher proportions of Indigenous American ancestry, reflecting the relationship between heterozygosity and genetic variance. Among heritable protein-coding genes, the prevalence of ancestry-specific expression quantitative trait loci (anc-eQTLs) was 30% in African ancestry and 8% for Indigenous American ancestry segments. Most anc-eQTLs (89%) were driven by population differences in allele frequency. Transcriptome-wide association analyses of multi-ancestry summary statistics for 28 traits identified 79% more gene–trait associations using transcriptome prediction models trained in our admixed population than models trained using data from the Genotype-Tissue Expression project. Our study highlights the importance of measuring gene expression across large and ancestrally diverse populations for enabling new discoveries and reducing disparities. Analyses of whole-genome and RNA sequencing data from 2,733 African American, Puerto Rican and Mexican American individuals reveal ancestry-specific patterns in the genetic architecture of whole-blood gene expression.},
  archive      = {J_NG},
  author       = {Kachuri, Linda and Mak, Angel C. Y. and Hu, Donglei and Eng, Celeste and Huntsman, Scott and Elhawary, Jennifer R. and Gupta, Namrata and Gabriel, Stacey and Xiao, Shujie and Keys, Kevin L. and Oni-Orisan, Akinyemi and Rodríguez-Santana, José R. and LeNoir, Michael A. and Borrell, Luisa N. and Zaitlen, Noah A. and Williams, L. Keoki and Gignoux, Christopher R. and Burchard, Esteban González and Ziv, Elad},
  doi          = {10.1038/s41588-023-01377-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {952-963},
  shortjournal = {Nat. Genet.},
  title        = {Gene expression in african americans, puerto ricans and mexican americans reveals ancestry-specific patterns of genetic architecture},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mobile element variation contributes to population-specific
genome diversification, gene regulation and disease risk. <em>NG</em>,
<em>55</em>(6), 939–951. (<a
href="https://doi.org/10.1038/s41588-023-01390-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mobile genetic elements (MEs) are heritable mutagens that recursively generate structural variants (SVs). ME variants (MEVs) are difficult to genotype and integrate in statistical genetics, obscuring their impact on genome diversification and traits. We developed a tool that accurately genotypes MEVs using short-read whole-genome sequencing (WGS) and applied it to global human populations. We find unexpected population-specific MEV differences, including an Alu insertion distribution distinguishing Japanese from other populations. Integrating MEVs with expression quantitative trait loci (eQTL) maps shows that MEV classes regulate tissue-specific gene expression by shared mechanisms, including creating or attenuating enhancers and recruiting post-transcriptional regulators, supporting class-wide interpretability. MEVs more often associate with gene expression changes than SNVs, thus plausibly impacting traits. Performing genome-wide association study (GWAS) with MEVs pinpoints potential causes of disease risk, including a LINE-1 insertion associated with keloid and fasciitis. This work implicates MEVs as drivers of human divergence and disease risk. MEGAnE is a new tool to detect and genotype mobile element variants (MEVs) from short-read whole-genome sequencing datasets. Genetic analyses implicate MEVs as population-specific drivers of gene expression variation and disease risk.},
  archive      = {J_NG},
  author       = {Kojima, Shohei and Koyama, Satoshi and Ka, Mirei and Saito, Yuka and Parrish, Erica H. and Endo, Mikiko and Takata, Sadaaki and Mizukoshi, Misaki and Hikino, Keiko and Takeda, Atsushi and Gelinas, Asami F. and Heaton, Steven M. and Koide, Rie and Kamada, Anselmo J. and Noguchi, Michiya and Hamada, Michiaki and Kamatani, Yoichiro and Murakawa, Yasuhiro and Ishigaki, Kazuyoshi and Nakamura, Yukio and Ito, Kaoru and Terao, Chikashi and Momozawa, Yukihide and Parrish, Nicholas F.},
  doi          = {10.1038/s41588-023-01390-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {939-951},
  shortjournal = {Nat. Genet.},
  title        = {Mobile element variation contributes to population-specific genome diversification, gene regulation and disease risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The impact of rare protein coding genetic variation on adult
cognitive function. <em>NG</em>, <em>55</em>(6), 927–938. (<a
href="https://doi.org/10.1038/s41588-023-01398-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Compelling evidence suggests that human cognitive function is strongly influenced by genetics. Here, we conduct a large-scale exome study to examine whether rare protein-coding variants impact cognitive function in the adult population (n = 485,930). We identify eight genes (ADGRB2, KDM5B, GIGYF1, ANKRD12, SLC8A1, RC3H2, CACNA1A and BCAS3) that are associated with adult cognitive function through rare coding variants with large effects. Rare genetic architecture for cognitive function partially overlaps with that of neurodevelopmental disorders. In the case of KDM5B we show how the genetic dosage of one of these genes may determine the variability of cognitive, behavioral and molecular traits in mice and humans. We further provide evidence that rare and common variants overlap in association signals and contribute additively to cognitive function. Our study introduces the relevance of rare coding variants for cognitive function and unveils high-impact monogenic contributions to how cognitive function is distributed in the normal adult population. Analysis of rare coding variants in the UK Biobank identifies eight genes associated with adult cognitive function, including KDM5B. Rare and common variant signals overlap and contribute additively to the phenotype.},
  archive      = {J_NG},
  author       = {Chen, Chia-Yen and Tian, Ruoyu and Ge, Tian and Lam, Max and Sanchez-Andrade, Gabriela and Singh, Tarjinder and Urpa, Lea and Liu, Jimmy Z. and Sanderson, Mark and Rowley, Christine and Ironfield, Holly and Fang, Terry and Daly, Mark and Palotie, Aarno and Tsai, Ellen A. and Huang, Hailiang and Hurles, Matthew E. and Gerety, Sebastian S. and Lencz, Todd and Runz, Heiko},
  doi          = {10.1038/s41588-023-01398-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {927-938},
  shortjournal = {Nat. Genet.},
  title        = {The impact of rare protein coding genetic variation on adult cognitive function},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The wheat stem rust resistance gene sr43 encodes an unusual
protein kinase. <em>NG</em>, <em>55</em>(6), 921–926. (<a
href="https://doi.org/10.1038/s41588-023-01402-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To safeguard bread wheat against pests and diseases, breeders have introduced over 200 resistance genes into its genome, thus nearly doubling the number of designated resistance genes in the wheat gene pool1. Isolating these genes facilitates their fast-tracking in breeding programs and incorporation into polygene stacks for more durable resistance. We cloned the stem rust resistance gene Sr43, which was crossed into bread wheat from the wild grass Thinopyrum elongatum2,3. Sr43 encodes an active protein kinase fused to two domains of unknown function. The gene, which is unique to the Triticeae, appears to have arisen through a gene fusion event 6.7 to 11.6 million years ago. Transgenic expression of Sr43 in wheat conferred high levels of resistance to a wide range of isolates of the pathogen causing stem rust, highlighting the potential value of Sr43 in resistance breeding and engineering. The resistance gene Sr43, which was crossed into bread wheat from the wild grass Thinopyrum elongatum, encodes an unusual protein kinase fusion protein that confers wheat stem rust resistance.},
  archive      = {J_NG},
  author       = {Yu, Guotai and Matny, Oadi and Gourdoupis, Spyridon and Rayapuram, Naganand and Aljedaani, Fatimah R. and Wang, Yan L. and Nürnberger, Thorsten and Johnson, Ryan and Crean, Emma E. and Saur, Isabel M.-L. and Gardener, Catherine and Yue, Yajuan and Kangara, Ngonidzashe and Steuernagel, Burkhard and Hayta, Sadiye and Smedley, Mark and Harwood, Wendy and Patpour, Mehran and Wu, Shuangye and Poland, Jesse and Jones, Jonathan D. G. and Reuber, T. Lynne and Ronen, Moshe and Sharon, Amir and Rouse, Matthew N. and Xu, Steven and Holušová, Kateřina and Bartoš, Jan and Molnár, István and Karafiátová, Miroslava and Hirt, Heribert and Blilou, Ikram and Jaremko, Łukasz and Doležel, Jaroslav and Steffenson, Brian J. and Wulff, Brande B. H.},
  doi          = {10.1038/s41588-023-01402-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {921-926},
  shortjournal = {Nat. Genet.},
  title        = {The wheat stem rust resistance gene sr43 encodes an unusual protein kinase},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An unusual tandem kinase fusion protein confers leaf rust
resistance in wheat. <em>NG</em>, <em>55</em>(6), 914–920. (<a
href="https://doi.org/10.1038/s41588-023-01401-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The introgression of chromosome segments from wild relatives is an established strategy to enrich crop germplasm with disease-resistance genes1. Here we use mutagenesis and transcriptome sequencing to clone the leaf rust resistance gene Lr9, which was introduced into bread wheat from the wild grass species Aegilops umbellulata2. We established that Lr9 encodes an unusual tandem kinase fusion protein. Long-read sequencing of a wheat Lr9 introgression line and the putative Ae. umbellulata Lr9 donor enabled us to assemble the ~28.4-Mb Lr9 translocation and to identify the translocation breakpoint. We likewise cloned Lr58, which was reportedly introgressed from Aegilops triuncialis3, but has an identical coding sequence compared to Lr9. Cytogenetic and haplotype analyses corroborate that the two genes originate from the same translocation event. Our work sheds light on the emerging role of kinase fusion proteins in wheat disease resistance, expanding the repertoire of disease-resistance genes for breeding. The resistance gene Lr9, which was introduced into bread wheat from the wild grass species Aegilops umbellulata, encodes an unusual tandem kinase fusion protein that confers wheat leaf rust resistance.},
  archive      = {J_NG},
  author       = {Wang, Yajun and Abrouk, Michael and Gourdoupis, Spyridon and Koo, Dal-Hoe and Karafiátová, Miroslava and Molnár, István and Holušová, Kateřina and Doležel, Jaroslav and Athiyannan, Naveenkumar and Cavalet-Giorsa, Emile and Jaremko, Łukasz and Poland, Jesse and Krattinger, Simon G.},
  doi          = {10.1038/s41588-023-01401-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {914-920},
  shortjournal = {Nat. Genet.},
  title        = {An unusual tandem kinase fusion protein confers leaf rust resistance in wheat},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetic and biological insights into spontaneous coronary
artery dissection. <em>NG</em>, <em>55</em>(6), 912–913. (<a
href="https://doi.org/10.1038/s41588-023-01413-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Meta-analysis of genome-wide association studies of spontaneous coronary artery dissection (SCAD), an important cause of myocardial infarction, identified 11 risk loci that involve genes related to artery integrity and tissue-mediated coagulation. Evidence supports SCAD as a genetically distinct condition from atherosclerotic coronary artery disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01413-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {912-913},
  shortjournal = {Nat. Genet.},
  title        = {Genetic and biological insights into spontaneous coronary artery dissection},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genes with immune functions modulate key traits associated
with down syndrome in mice. <em>NG</em>, <em>55</em>(6), 910–911. (<a
href="https://doi.org/10.1038/s41588-023-01400-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An extra copy of a chromosome region that includes four genes encoding interferon receptors contributes to immune dysregulation, heart malformations, developmental delays, cognitive deficits and craniofacial abnormalities in a mouse model of Down syndrome.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01400-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {910-911},
  shortjournal = {Nat. Genet.},
  title        = {Genes with immune functions modulate key traits associated with down syndrome in mice},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pathogen perception and deception in plant immunity by
kinase fusion proteins. <em>NG</em>, <em>55</em>(6), 908–909. (<a
href="https://doi.org/10.1038/s41588-023-01396-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two studies describe kinase fusion proteins (KFPs) that regulate the perception and deception of wheat pathogens. These highlight the emergence of KFPs as plant immune regulators and emphasize the importance of crop wild relatives as a reservoir for resistance breeding and global food security.},
  archive      = {J_NG},
  author       = {Fahima, Tzion and Coaker, Gitta},
  doi          = {10.1038/s41588-023-01396-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {908-909},
  shortjournal = {Nat. Genet.},
  title        = {Pathogen perception and deception in plant immunity by kinase fusion proteins},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dynamic measures of insulin action identify genetic
determinants of dysglycemia. <em>NG</em>, <em>55</em>(6), 905–907. (<a
href="https://doi.org/10.1038/s41588-023-01346-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS) of type 2 diabetes have identified few loci connected to muscle biology. A GWAS of post-glucose-challenge measures of insulin has now identified loci related to insulin resistance and GLUT4 regulation in skeletal muscle, illustrating the benefit of studying dynamic glycemic measures.},
  archive      = {J_NG},
  author       = {Udler, Miriam S.},
  doi          = {10.1038/s41588-023-01346-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {905-907},
  shortjournal = {Nat. Genet.},
  title        = {Dynamic measures of insulin action identify genetic determinants of dysglycemia},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Transposable element evolution in mammals. <em>NG</em>,
<em>55</em>(6), 904. (<a
href="https://doi.org/10.1038/s41588-023-01430-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-023-01430-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {904},
  shortjournal = {Nat. Genet.},
  title        = {Transposable element evolution in mammals},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Simulating scRNA-seq for benchmarks. <em>NG</em>,
<em>55</em>(6), 904. (<a
href="https://doi.org/10.1038/s41588-023-01431-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01431-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {904},
  shortjournal = {Nat. Genet.},
  title        = {Simulating scRNA-seq for benchmarks},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Single-cell CRISPR screen for GWAS loci. <em>NG</em>,
<em>55</em>(6), 904. (<a
href="https://doi.org/10.1038/s41588-023-01432-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01432-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {904},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell CRISPR screen for GWAS loci},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tracking genetic variants in the biomedical literature using
LitVar 2.0. <em>NG</em>, <em>55</em>(6), 901–903. (<a
href="https://doi.org/10.1038/s41588-023-01414-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Keeping up with the latest variant-related research is vital for genomic medicine. Here we present LitVar 2.0, a significantly improved web-based system to accurately search for genetic variants in the unstructured literature. LitVar 2.0 provides a unified search of full text and supplementary data, and improved variant recognition accuracy.},
  archive      = {J_NG},
  author       = {Allot, Alexis and Wei, Chih-Hsuan and Phan, Lon and Hefferon, Timothy and Landrum, Melissa and Rehm, Heidi L. and Lu, Zhiyong},
  doi          = {10.1038/s41588-023-01414-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {901-903},
  shortjournal = {Nat. Genet.},
  title        = {Tracking genetic variants in the biomedical literature using LitVar 2.0},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Including multiracial individuals is crucial for race,
ethnicity and ancestry frameworks in genetics and genomics. <em>NG</em>,
<em>55</em>(6), 895–900. (<a
href="https://doi.org/10.1038/s41588-023-01394-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current ontologies of race, ethnicity and genetic ancestry rely on categorization, but have limitations — as exemplified by multiracial individuals. We argue that including these individuals will foster inclusion by better capturing complex identities, with equity benefits for the full human population.},
  archive      = {J_NG},
  author       = {Martschenko, Daphne O. and Wand, Hannah and Young, Jennifer L. and Wojcik, Genevieve L.},
  doi          = {10.1038/s41588-023-01394-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {895-900},
  shortjournal = {Nat. Genet.},
  title        = {Including multiracial individuals is crucial for race, ethnicity and ancestry frameworks in genetics and genomics},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Publisher correction: Comprehensive molecular
characterization of mitochondrial genomes in human cancers. <em>NG</em>,
<em>55</em>(5), 893. (<a
href="https://doi.org/10.1038/s41588-020-0587-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Yuan, Yuan and Ju, Young Seok and Kim, Youngwook and Li, Jun and Wang, Yumeng and Yoon, Christopher J. and Yang, Yang and Martincorena, Inigo and Creighton, Chad J. and Weinstein, John N. and Xu, Yanxun and Han, Leng and Kim, Hyung-Lae and Nakagawa, Hidewaki and Park, Keunchil and Campbell, Peter J. and Liang, Han},
  doi          = {10.1038/s41588-020-0587-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {893},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Comprehensive molecular characterization of mitochondrial genomes in human cancers},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Publisher correction: Comprehensive analysis of
chromothripsis in 2,658 human cancers using whole-genome sequencing.
<em>NG</em>, <em>55</em>(5), 893. (<a
href="https://doi.org/10.1038/s41588-020-0634-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Cortés-Ciriano, Isidro and Lee, Jake June-Koo and Xi, Ruibin and Jain, Dhawal and Jung, Youngsook L. and Yang, Lixing and Gordenin, Dmitry and Klimczak, Leszek J. and Zhang, Cheng-Zhong and Pellman, David S. and Park, Peter J.},
  doi          = {10.1038/s41588-020-0634-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {893},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Comprehensive analysis of chromothripsis in 2,658 human cancers using whole-genome sequencing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Author correction: Comprehensive molecular characterization
of mitochondrial genomes in human cancers. <em>NG</em>, <em>55</em>(5),
892. (<a href="https://doi.org/10.1038/s41588-020-0629-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NG},
  author       = {Yuan, Yuan and Ju, Young Seok and Kim, Youngwook and Li, Jun and Wang, Yumeng and Yoon, Christopher J. and Yang, Yang and Martincorena, Inigo and Creighton, Chad J. and Weinstein, John N. and Xu, Yanxun and Han, Leng and Kim, Hyung-Lae and Nakagawa, Hidewaki and Park, Keunchil and Campbell, Peter J. and Liang, Han},
  doi          = {10.1038/s41588-020-0629-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {892},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Comprehensive molecular characterization of mitochondrial genomes in human cancers},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Bipotent transitional liver progenitor
cells contribute to liver regeneration. <em>NG</em>, <em>55</em>(5),
892. (<a href="https://doi.org/10.1038/s41588-023-01393-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Pu, Wenjuan and Zhu, Huan and Zhang, Mingjun and Pikiolek, Monika and Ercan, Caner and Li, Jie and Huang, Xiuzhen and Han, Ximeng and Zhang, Zhenqian and Lv, Zan and Li, Yan and Liu, Kuo and He, Lingjuan and Liu, Xiuxiu and Heim, Markus H. and Terracciano, Luigi M. and Tchorz, Jan S. and Zhou, Bin},
  doi          = {10.1038/s41588-023-01393-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {892},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Bipotent transitional liver progenitor cells contribute to liver regeneration},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: The stress kinase MKK7 couples oncogenic
stress to p53 stability and tumor suppression. <em>NG</em>,
<em>55</em>(5), 891. (<a
href="https://doi.org/10.1038/s41588-023-01387-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Schramek, Daniel and Kotsinas, Athanassios and Meixner, Arabella and Wada, Teiji and Elling, Ulrich and Pospisilik, J. Andrew and Neely, G. Gregory and Zwick, Ralf-Harun and Sigl, Verena and Forni, Guido and Serrano, Manuel and Gorgoulis, Vassilis G. and Penninger, Josef M.},
  doi          = {10.1038/s41588-023-01387-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {891},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The stress kinase MKK7 couples oncogenic stress to p53 stability and tumor suppression},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Parallel sequencing of extrachromosomal circular DNAs and
transcriptomes in single cancer cells. <em>NG</em>, <em>55</em>(5),
880–890. (<a href="https://doi.org/10.1038/s41588-023-01386-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extrachromosomal DNAs (ecDNAs) are common in cancer, but many questions about their origin, structural dynamics and impact on intratumor heterogeneity are still unresolved. Here we describe single-cell extrachromosomal circular DNA and transcriptome sequencing (scEC&amp;amp;T-seq), a method for parallel sequencing of circular DNAs and full-length mRNA from single cells. By applying scEC&amp;amp;T-seq to cancer cells, we describe intercellular differences in ecDNA content while investigating their structural heterogeneity and transcriptional impact. Oncogene-containing ecDNAs were clonally present in cancer cells and drove intercellular oncogene expression differences. In contrast, other small circular DNAs were exclusive to individual cells, indicating differences in their selection and propagation. Intercellular differences in ecDNA structure pointed to circular recombination as a mechanism of ecDNA evolution. These results demonstrate scEC&amp;amp;T-seq as an approach to systematically characterize both small and large circular DNA in cancer cells, which will facilitate the analysis of these DNA elements in cancer and beyond. scEC&amp;amp;T-seq profiles extrachromosomal circular DNA and full-length mRNA from single human cancer cells, and may be used to interrogate heterogeneity in both cell lines and primary tumor samples.},
  archive      = {J_NG},
  author       = {Chamorro González, Rocío and Conrad, Thomas and Stöber, Maja C. and Xu, Robin and Giurgiu, Mădălina and Rodriguez-Fos, Elias and Kasack, Katharina and Brückner, Lotte and van Leen, Eric and Helmsauer, Konstantin and Dorado Garcia, Heathcliff and Stefanova, Maria E. and Hung, King L. and Bei, Yi and Schmelz, Karin and Lodrini, Marco and Mundlos, Stefan and Chang, Howard Y. and Deubzer, Hedwig E. and Sauer, Sascha and Eggert, Angelika and Schulte, Johannes H. and Schwarz, Roland F. and Haase, Kerstin and Koche, Richard P. and Henssen, Anton G.},
  doi          = {10.1038/s41588-023-01386-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {880-890},
  shortjournal = {Nat. Genet.},
  title        = {Parallel sequencing of extrachromosomal circular DNAs and transcriptomes in single cancer cells},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single duplex DNA sequencing with CODEC detects mutations
with high sensitivity. <em>NG</em>, <em>55</em>(5), 871–879. (<a
href="https://doi.org/10.1038/s41588-023-01376-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detecting mutations from single DNA molecules is crucial in many fields but challenging. Next-generation sequencing (NGS) affords tremendous throughput but cannot directly sequence double-stranded DNA molecules (‘single duplexes’) to discern the true mutations on both strands. Here we present Concatenating Original Duplex for Error Correction (CODEC), which confers single duplex resolution to NGS. CODEC affords 1,000-fold higher accuracy than NGS, using up to 100-fold fewer reads than duplex sequencing. CODEC revealed mutation frequencies of 2.72 × 10−8 in sperm of a 39-year-old individual, and somatic mutations acquired with age in blood cells. CODEC detected genome-wide, clonal hematopoiesis mutations from single DNA molecules, single mutated duplexes from tumor genomes and liquid biopsies, microsatellite instability with 10-fold greater sensitivity and mutational signatures, and specific tumor mutations with up to 100-fold fewer reads. CODEC enables more precise genetic testing and reveals biologically significant mutations, which are commonly obscured by NGS errors. Concatenating Original Duplex for Error Correction (CODEC) is a method that concatenates both strands of each DNA duplex to enable highly sensitive mutation detection in a range of analytes with fewer reads and lower error rates than current methods.},
  archive      = {J_NG},
  author       = {Bae, Jin H. and Liu, Ruolin and Roberts, Eugenia and Nguyen, Erica and Tabrizi, Shervin and Rhoades, Justin and Blewett, Timothy and Xiong, Kan and Gydush, Gregory and Shea, Douglas and An, Zhenyi and Patel, Sahil and Cheng, Ju and Sridhar, Sainetra and Liu, Mei Hong and Lassen, Emilie and Skytte, Anne-Bine and Grońska-Pęski, Marta and Shoag, Jonathan E. and Evrony, Gilad D. and Parsons, Heather A. and Mayer, Erica L. and Makrigiorgos, G. Mike and Golub, Todd R. and Adalsteinsson, Viktor A.},
  doi          = {10.1038/s41588-023-01376-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {871-879},
  shortjournal = {Nat. Genet.},
  title        = {Single duplex DNA sequencing with CODEC detects mutations with high sensitivity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Aberrant splicing prediction across human tissues.
<em>NG</em>, <em>55</em>(5), 861–870. (<a
href="https://doi.org/10.1038/s41588-023-01373-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant splicing is a major cause of genetic disorders but its direct detection in transcriptomes is limited to clinically accessible tissues such as skin or body fluids. While DNA-based machine learning models can prioritize rare variants for affecting splicing, their performance in predicting tissue-specific aberrant splicing remains unassessed. Here we generated an aberrant splicing benchmark dataset, spanning over 8.8 million rare variants in 49 human tissues from the Genotype-Tissue Expression (GTEx) dataset. At 20% recall, state-of-the-art DNA-based models achieve maximum 12% precision. By mapping and quantifying tissue-specific splice site usage transcriptome-wide and modeling isoform competition, we increased precision by threefold at the same recall. Integrating RNA-sequencing data of clinically accessible tissues into our model, AbSplice, brought precision to 60%. These results, replicated in two independent cohorts, substantially contribute to noncoding loss-of-function variant identification and to genetic diagnostics design and analytics. AbSplice predicts aberrant splicing for 50 human tissues by integrating sequence-based deep learning models, DNA variation and RNA-seq obtained from accessible tissues.},
  archive      = {J_NG},
  author       = {Wagner, Nils and Çelik, Muhammed H. and Hölzlwimmer, Florian R. and Mertes, Christian and Prokisch, Holger and Yépez, Vicente A. and Gagneur, Julien},
  doi          = {10.1038/s41588-023-01373-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {861-870},
  shortjournal = {Nat. Genet.},
  title        = {Aberrant splicing prediction across human tissues},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Super-pangenome analyses highlight genomic diversity and
structural variation across wild and cultivated tomato species.
<em>NG</em>, <em>55</em>(5), 852–860. (<a
href="https://doi.org/10.1038/s41588-023-01340-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Effective utilization of wild relatives is key to overcoming challenges in genetic improvement of cultivated tomato, which has a narrow genetic basis; however, current efforts to decipher high-quality genomes for tomato wild species are insufficient. Here, we report chromosome-scale tomato genomes from nine wild species and two cultivated accessions, representative of Solanum section Lycopersicon, the tomato clade. Together with two previously released genomes, we elucidate the phylogeny of Lycopersicon and construct a section-wide gene repertoire. We reveal the landscape of structural variants and provide entry to the genomic diversity among tomato wild relatives, enabling the discovery of a wild tomato gene with the potential to increase yields of modern cultivated tomatoes. Construction of a graph-based genome enables structural-variant-based genome-wide association studies, identifying numerous signals associated with tomato flavor-related traits and fruit metabolites. The tomato super-pangenome resources will expedite biological studies and breeding of this globally important crop. A tomato super-pangenome constructed using chromosome-scale genomes of nine wild species and two cultivated accessions highlights genomic diversity and structural variation across wild and cultivated tomatoes.},
  archive      = {J_NG},
  author       = {Li, Ning and He, Qiang and Wang, Juan and Wang, Baike and Zhao, Jiantao and Huang, Shaoyong and Yang, Tao and Tang, Yaping and Yang, Shengbao and Aisimutuola, Patiguli and Xu, Ruiqiang and Hu, Jiahui and Jia, Chunping and Ma, Kai and Li, Zhiqiang and Jiang, Fangling and Gao, Jie and Lan, Haiyan and Zhou, Yongfeng and Zhang, Xinyan and Huang, Sanwen and Fei, Zhangjun and Wang, Huan and Li, Hongbo and Yu, Qinghui},
  doi          = {10.1038/s41588-023-01340-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {852-860},
  shortjournal = {Nat. Genet.},
  title        = {Super-pangenome analyses highlight genomic diversity and structural variation across wild and cultivated tomato species},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Precise modulation of transcription factor levels identifies
features underlying dosage sensitivity. <em>NG</em>, <em>55</em>(5),
841–851. (<a href="https://doi.org/10.1038/s41588-023-01366-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional regulation exhibits extensive robustness, but human genetics indicates sensitivity to transcription factor (TF) dosage. Reconciling such observations requires quantitative studies of TF dosage effects at trait-relevant ranges, largely lacking so far. TFs play central roles in both normal-range and disease-associated variation in craniofacial morphology; we therefore developed an approach to precisely modulate TF levels in human facial progenitor cells and applied it to SOX9, a TF associated with craniofacial variation and disease (Pierre Robin sequence (PRS)). Most SOX9-dependent regulatory elements (REs) are buffered against small decreases in SOX9 dosage, but REs directly and primarily regulated by SOX9 show heightened sensitivity to SOX9 dosage; these RE responses partially predict gene expression responses. Sensitive REs and genes preferentially affect functional chondrogenesis and PRS-like craniofacial shape variation. We propose that such REs and genes underlie the sensitivity of specific phenotypes to TF dosage, while buffering of other genes leads to robust, nonlinear dosage-to-phenotype relationships. SOX9 titration in neural crest cells identifies regulatory elements and genes with sensitive or buffered responses. Sensitive genes are enriched for craniofacial disorder genes phenocopying SOX9, suggesting differential sensitivity contributes to phenotypic specificity.},
  archive      = {J_NG},
  author       = {Naqvi, Sahin and Kim, Seungsoo and Hoskens, Hanne and Matthews, Harold S. and Spritz, Richard A. and Klein, Ophir D. and Hallgrímsson, Benedikt and Swigut, Tomek and Claes, Peter and Pritchard, Jonathan K. and Wysocka, Joanna},
  doi          = {10.1038/s41588-023-01366-2},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {841-851},
  shortjournal = {Nat. Genet.},
  title        = {Precise modulation of transcription factor levels identifies features underlying dosage sensitivity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Enhancer–promoter contact formation requires RNAPII and
antagonizes loop extrusion. <em>NG</em>, <em>55</em>(5), 832–840. (<a
href="https://doi.org/10.1038/s41588-023-01364-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Homotypic chromatin interactions and loop extrusion are thought to be the two main drivers of mammalian chromosome folding. Here we tested the role of RNA polymerase II (RNAPII) across different scales of interphase chromatin organization in a cellular system allowing for its rapid, auxin-mediated degradation. We combined Micro-C and computational modeling to characterize subsets of loops differentially gained or lost upon RNAPII depletion. Gained loops, extrusion of which was antagonized by RNAPII, almost invariably formed by engaging new or rewired CTCF anchors. Lost loops selectively affected contacts between enhancers and promoters anchored by RNAPII, explaining the repression of most genes. Surprisingly, promoter–promoter interactions remained essentially unaffected by polymerase depletion, and cohesin occupancy was sustained. Together, our findings reconcile the role of RNAPII in transcription with its direct involvement in setting-up regulatory three-dimensional chromatin contacts genome wide, while also revealing an impact on cohesin loop extrusion. High-resolution Micro-C is applied to characterize the effect of RNA polymerase II (RNAPII) loss on chromosome looping, finding that the formation of enhancer–promoter, but not promoter–promoter, loops are dependent on RNAPII binding to their anchors.},
  archive      = {J_NG},
  author       = {Zhang, Shu and Übelmesser, Nadine and Barbieri, Mariano and Papantonis, Argyris},
  doi          = {10.1038/s41588-023-01364-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {832-840},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer–promoter contact formation requires RNAPII and antagonizes loop extrusion},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetic immune escape landscape in primary and metastatic
cancer. <em>NG</em>, <em>55</em>(5), 820–831. (<a
href="https://doi.org/10.1038/s41588-023-01367-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies have characterized the immune escape landscape across primary tumors. However, whether late-stage metastatic tumors present differences in genetic immune escape (GIE) prevalence and dynamics remains unclear. We performed a pan-cancer characterization of GIE prevalence across six immune escape pathways in 6,319 uniformly processed tumor samples. To address the complexity of the HLA-I locus in the germline and in tumors, we developed LILAC, an open-source integrative framework. One in four tumors harbors GIE alterations, with high mechanistic and frequency variability across cancer types. GIE prevalence is generally consistent between primary and metastatic tumors. We reveal that GIE alterations are selected for in tumor evolution and focal loss of heterozygosity of HLA-I tends to eliminate the HLA allele, presenting the largest neoepitope repertoire. Finally, high mutational burden tumors showed a tendency toward focal loss of heterozygosity of HLA-I as the immune evasion mechanism, whereas, in hypermutated tumors, other immune evasion strategies prevail. A pan-cancer analysis of primary and metastatic tumors highlights the diversity of genetic immune escape mechanisms established during tumor evolution. The authors also present LILAC, a tool to characterize the HLA-I locus from whole-genome sequencing data.},
  archive      = {J_NG},
  author       = {Martínez-Jiménez, Francisco and Priestley, Peter and Shale, Charles and Baber, Jonathan and Rozemuller, Erik and Cuppen, Edwin},
  doi          = {10.1038/s41588-023-01367-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {820-831},
  shortjournal = {Nat. Genet.},
  title        = {Genetic immune escape landscape in primary and metastatic cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genomic and transcriptomic analysis of checkpoint blockade
response in advanced non-small cell lung cancer. <em>NG</em>,
<em>55</em>(5), 807–819. (<a
href="https://doi.org/10.1038/s41588-023-01355-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Anti-PD-1/PD-L1 agents have transformed the treatment landscape of advanced non-small cell lung cancer (NSCLC). To expand our understanding of the molecular features underlying response to checkpoint inhibitors in NSCLC, we describe here the first joint analysis of the Stand Up To Cancer-Mark Foundation cohort, a resource of whole exome and/or RNA sequencing from 393 patients with NSCLC treated with anti-PD-(L)1 therapy, along with matched clinical response annotation. We identify a number of associations between molecular features and outcome, including (1) favorable (for example, ATM altered) and unfavorable (for example, TERT amplified) genomic subgroups, (2) a prominent association between expression of inducible components of the immunoproteasome and response and (3) a dedifferentiated tumor-intrinsic subtype with enhanced response to checkpoint blockade. Taken together, results from this cohort demonstrate the complexity of biological determinants underlying immunotherapy outcomes and reinforce the discovery potential of integrative analysis within large, well-curated, cancer-specific cohorts. Genomic and transcriptomic analysis of 393 non-small cell lung cancer patients treated with checkpoint inhibitors identifies molecular features associated with response.},
  archive      = {J_NG},
  author       = {Ravi, Arvind and Hellmann, Matthew D. and Arniella, Monica B. and Holton, Mark and Freeman, Samuel S. and Naranbhai, Vivek and Stewart, Chip and Leshchiner, Ignaty and Kim, Jaegil and Akiyama, Yo and Griffin, Aaron T. and Vokes, Natalie I. and Sakhi, Mustafa and Kamesan, Vashine and Rizvi, Hira and Ricciuti, Biagio and Forde, Patrick M. and Anagnostou, Valsamo and Riess, Jonathan W. and Gibbons, Don L. and Pennell, Nathan A. and Velcheti, Vamsidhar and Digumarthy, Subba R. and Mino-Kenudson, Mari and Califano, Andrea and Heymach, John V. and Herbst, Roy S. and Brahmer, Julie R. and Schalper, Kurt A. and Velculescu, Victor E. and Henick, Brian S. and Rizvi, Naiyer and Jänne, Pasi A. and Awad, Mark M. and Chow, Andrew and Greenbaum, Benjamin D. and Luksza, Marta and Shaw, Alice T. and Wolchok, Jedd and Hacohen, Nir and Getz, Gad and Gainor, Justin F.},
  doi          = {10.1038/s41588-023-01355-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {807-819},
  shortjournal = {Nat. Genet.},
  title        = {Genomic and transcriptomic analysis of checkpoint blockade response in advanced non-small cell lung cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetic architecture of the inflammatory bowel diseases
across east asian and european ancestries. <em>NG</em>, <em>55</em>(5),
796–806. (<a href="https://doi.org/10.1038/s41588-023-01384-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inflammatory bowel diseases (IBDs) are chronic disorders of the gastrointestinal tract with the following two subtypes: Crohn’s disease (CD) and ulcerative colitis (UC). To date, most IBD genetic associations were derived from individuals of European (EUR) ancestries. Here we report the largest IBD study of individuals of East Asian (EAS) ancestries, including 14,393 cases and 15,456 controls. We found 80 IBD loci in EAS alone and 320 when meta-analyzed with ~370,000 EUR individuals (~30,000 cases), among which 81 are new. EAS-enriched coding variants implicate many new IBD genes, including ADAP1 and GIT2. Although IBD genetic effects are generally consistent across ancestries, genetics underlying CD appears more ancestry dependent than UC, driven by allele frequency (NOD2) and effect (TNFSF15). We extended the IBD polygenic risk score (PRS) by incorporating both ancestries, greatly improving its accuracy and highlighting the importance of diversity for the equitable deployment of PRS. Genome-wide association analyses across individuals of East Asian and European ancestries identify new risk loci for inflammatory bowel diseases. A polygenic risk score derived from the combined datasets shows improved prediction accuracy.},
  archive      = {J_NG},
  author       = {Liu, Zhanju and Liu, Ruize and Gao, Han and Jung, Seulgi and Gao, Xiang and Sun, Ruicong and Liu, Xiaoming and Kim, Yongjae and Lee, Ho-Su and Kawai, Yosuke and Nagasaki, Masao and Umeno, Junji and Tokunaga, Katsushi and Kinouchi, Yoshitaka and Masamune, Atsushi and Shi, Wenzhao and Shen, Chengguo and Guo, Zhenglin and Yuan, Kai and Zhu, Shu and Li, Dalin and Liu, Jianjun and Ge, Tian and Cho, Judy and Daly, Mark J. and McGovern, Dermot P. B. and Ye, Byong Duk and Song, Kyuyoung and Kakuta, Yoichi and Li, Mingsong and Huang, Hailiang},
  doi          = {10.1038/s41588-023-01384-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {796-806},
  shortjournal = {Nat. Genet.},
  title        = {Genetic architecture of the inflammatory bowel diseases across east asian and european ancestries},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Inference of chronic obstructive pulmonary disease with deep
learning on raw spirograms identifies new genetic loci and improves risk
models. <em>NG</em>, <em>55</em>(5), 787–795. (<a
href="https://doi.org/10.1038/s41588-023-01372-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chronic obstructive pulmonary disease (COPD), the third leading cause of death worldwide, is highly heritable. While COPD is clinically defined by applying thresholds to summary measures of lung function, a quantitative liability score has more power to identify genetic signals. Here we train a deep convolutional neural network on noisy self-reported and International Classification of Diseases labels to predict COPD case–control status from high-dimensional raw spirograms and use the model’s predictions as a liability score. The machine-learning-based (ML-based) liability score accurately discriminates COPD cases and controls, and predicts COPD-related hospitalization without any domain-specific knowledge. Moreover, the ML-based liability score is associated with overall survival and exacerbation events. A genome-wide association study on the ML-based liability score replicates existing COPD and lung function loci and also identifies 67 new loci. Lastly, our method provides a general framework to use ML methods and medical-record-based labels that does not require domain knowledge or expert curation to improve disease prediction and genomic discovery for drug design. A deep convolutional neural network calculates liability scores for chronic obstructive pulmonary disease (COPD) from raw spirogram traces and noisy medical-record-based labels in the UK Biobank. Genome-wide analyses using these scores replicate known loci for lung function and identify 67 new disease loci.},
  archive      = {J_NG},
  author       = {Cosentino, Justin and Behsaz, Babak and Alipanahi, Babak and McCaw, Zachary R. and Hill, Davin and Schwantes-An, Tae-Hwi and Lai, Dongbing and Carroll, Andrew and Hobbs, Brian D. and Cho, Michael H. and McLean, Cory Y. and Hormozdiari, Farhad},
  doi          = {10.1038/s41588-023-01372-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {787-795},
  shortjournal = {Nat. Genet.},
  title        = {Inference of chronic obstructive pulmonary disease with deep learning on raw spirograms identifies new genetic loci and improves risk models},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetics of myocardial interstitial fibrosis in the human
heart and association with disease. <em>NG</em>, <em>55</em>(5),
777–786. (<a href="https://doi.org/10.1038/s41588-023-01371-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Myocardial interstitial fibrosis is associated with cardiovascular disease and adverse prognosis. Here, to investigate the biological pathways that underlie fibrosis in the human heart, we developed a machine learning model to measure native myocardial T1 time, a marker of myocardial fibrosis, in 41,505 UK Biobank participants who underwent cardiac magnetic resonance imaging. Greater T1 time was associated with diabetes mellitus, renal disease, aortic stenosis, cardiomyopathy, heart failure, atrial fibrillation, conduction disease and rheumatoid arthritis. Genome-wide association analysis identified 11 independent loci associated with T1 time. The identified loci implicated genes involved in glucose transport (SLC2A12), iron homeostasis (HFE, TMPRSS6), tissue repair (ADAMTSL1, VEGFC), oxidative stress (SOD2), cardiac hypertrophy (MYH7B) and calcium signaling (CAMK2D). Using a transforming growth factor β1-mediated cardiac fibroblast activation assay, we found that 9 of the 11 loci consisted of genes that exhibited temporal changes in expression or open chromatin conformation supporting their biological relevance to myofibroblast cell state acquisition. By harnessing machine learning to perform large-scale quantification of myocardial interstitial fibrosis using cardiac imaging, we validate associations between cardiac fibrosis and disease, and identify new biologically relevant pathways underlying fibrosis. Genome-wide association analyses identify 11 loci associated with native myocardial T1 time, a marker of interstitial fibrosis, providing insights into the pathways involved in myocardial fibrosis and myofibroblast cell state acquisition.},
  archive      = {J_NG},
  author       = {Nauffal, Victor and Di Achille, Paolo and Klarqvist, Marcus D. R. and Cunningham, Jonathan W. and Hill, Matthew C. and Pirruccello, James P. and Weng, Lu-Chen and Morrill, Valerie N. and Choi, Seung Hoan and Khurshid, Shaan and Friedman, Samuel F. and Nekoui, Mahan and Roselli, Carolina and Ng, Kenney and Philippakis, Anthony A. and Batra, Puneet and Ellinor, Patrick T. and Lubitz, Steven A.},
  doi          = {10.1038/s41588-023-01371-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {777-786},
  shortjournal = {Nat. Genet.},
  title        = {Genetics of myocardial interstitial fibrosis in the human heart and association with disease},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biobank-scale inference of ancestral recombination graphs
enables genealogical analysis of complex traits. <em>NG</em>,
<em>55</em>(5), 768–776. (<a
href="https://doi.org/10.1038/s41588-023-01379-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide genealogies compactly represent the evolutionary history of a set of genomes and inferring them from genetic data has the potential to facilitate a wide range of analyses. We introduce a method, ARG-Needle, for accurately inferring biobank-scale genealogies from sequencing or genotyping array data, as well as strategies to utilize genealogies to perform association and other complex trait analyses. We use these methods to build genome-wide genealogies using genotyping data for 337,464 UK Biobank individuals and test for association across seven complex traits. Genealogy-based association detects more rare and ultra-rare signals (N = 134, frequency range 0.0007−0.1%) than genotype imputation using ~65,000 sequenced haplotypes (N = 64). In a subset of 138,039 exome sequencing samples, these associations strongly tag (average r = 0.72) underlying sequencing variants enriched (4.8×) for loss-of-function variation. These results demonstrate that inferred genome-wide genealogies may be leveraged in the analysis of complex traits, complementing approaches that require the availability of large, population-specific sequencing panels. ARG-Needle is a method to infer genome-wide genealogies from large-scale genotyping data that can be used in association analyses. Applied to UK Biobank data, genealogy-based testing finds more trait associations than using imputed genotypes.},
  archive      = {J_NG},
  author       = {Zhang, Brian C. and Biddanda, Arjun and Gunnarsson, Árni Freyr and Cooper, Fergus and Palamara, Pier Francesco},
  doi          = {10.1038/s41588-023-01379-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {768-776},
  shortjournal = {Nat. Genet.},
  title        = {Biobank-scale inference of ancestral recombination graphs enables genealogical analysis of complex traits},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell analyses and host genetics highlight the role of
innate immune cells in COVID-19 severity. <em>NG</em>, <em>55</em>(5),
753–767. (<a href="https://doi.org/10.1038/s41588-023-01375-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mechanisms underpinning the dysfunctional immune response in severe acute respiratory syndrome coronavirus 2 infection are elusive. We analyzed single-cell transcriptomes and T and B cell receptors (BCR) of &amp;gt;895,000 peripheral blood mononuclear cells from 73 coronavirus disease 2019 (COVID-19) patients and 75 healthy controls of Japanese ancestry with host genetic data. COVID-19 patients showed a low fraction of nonclassical monocytes (ncMono). We report downregulated cell transitions from classical monocytes to ncMono in COVID-19 with reduced CXCL10 expression in ncMono in severe disease. Cell–cell communication analysis inferred decreased cellular interactions involving ncMono in severe COVID-19. Clonal expansions of BCR were evident in the plasmablasts of patients. Putative disease genes identified by COVID-19 genome-wide association study showed cell type-specific expressions in monocytes and dendritic cells. A COVID-19-associated risk variant at the IFNAR2 locus (rs13050728) had context-specific and monocyte-specific expression quantitative trait loci effects. Our study highlights biological and host genetic involvement of innate immune cells in COVID-19 severity. Peripheral blood mononuclear cells from 73 Japanese patients with coronavirus disease 2019 (COVID-19) and 75 healthy controls were analyzed using single-cell transcriptomics. Combining these data with genotyping data highlights the interplay between host genetics and the immune response in modulating disease severity.},
  archive      = {J_NG},
  author       = {Edahiro, Ryuya and Shirai, Yuya and Takeshima, Yusuke and Sakakibara, Shuhei and Yamaguchi, Yuta and Murakami, Teruaki and Morita, Takayoshi and Kato, Yasuhiro and Liu, Yu-Chen and Motooka, Daisuke and Naito, Yoko and Takuwa, Ayako and Sugihara, Fuminori and Tanaka, Kentaro and Wing, James B. and Sonehara, Kyuto and Tomofuji, Yoshihiko and Namkoong, Ho and Tanaka, Hiromu and Lee, Ho and Fukunaga, Koichi and Hirata, Haruhiko and Takeda, Yoshito and Okuzaki, Daisuke and Kumanogoh, Atsushi and Okada, Yukinori},
  doi          = {10.1038/s41588-023-01375-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {753-767},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell analyses and host genetics highlight the role of innate immune cells in COVID-19 severity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Maximum likelihood pandemic-scale phylogenetics.
<em>NG</em>, <em>55</em>(5), 746–752. (<a
href="https://doi.org/10.1038/s41588-023-01368-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phylogenetics has a crucial role in genomic epidemiology. Enabled by unparalleled volumes of genome sequence data generated to study and help contain the COVID-19 pandemic, phylogenetic analyses of SARS-CoV-2 genomes have shed light on the virus’s origins, spread, and the emergence and reproductive success of new variants. However, most phylogenetic approaches, including maximum likelihood and Bayesian methods, cannot scale to the size of the datasets from the current pandemic. We present ‘MAximum Parsimonious Likelihood Estimation’ (MAPLE), an approach for likelihood-based phylogenetic analysis of epidemiological genomic datasets at unprecedented scales. MAPLE infers SARS-CoV-2 phylogenies more accurately than existing maximum likelihood approaches while running up to thousands of times faster, and requiring at least 100 times less memory on large datasets. This extends the reach of genomic epidemiology, allowing the continued use of accurate phylogenetic, phylogeographic and phylodynamic analyses on datasets of millions of genomes. ‘MAximum Parsimonious Likelihood Estimation’ (MAPLE) is a maximum likelihood-based approach for inference of phylogenetic trees from very large datasets of similar sequences incorporating a sparse alignment representation and parsimony-based approximations, offering higher accuracy and reduced computational requirements.},
  archive      = {J_NG},
  author       = {De Maio, Nicola and Kalaghatgi, Prabhav and Turakhia, Yatish and Corbett-Detig, Russell and Minh, Bui Quang and Goldman, Nick},
  doi          = {10.1038/s41588-023-01368-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {746-752},
  shortjournal = {Nat. Genet.},
  title        = {Maximum likelihood pandemic-scale phylogenetics},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tomato super-pangenome highlights the potential use of wild
relatives in tomato breeding. <em>NG</em>, <em>55</em>(5), 744–745. (<a
href="https://doi.org/10.1038/s41588-023-01341-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome assembly of nine wild species and two domesticated accessions of tomato generated a super-pangenome for the tomato clade. Comparative analyses revealed the landscape of structural variations in wild and cultivated tomatoes and led to the discovery of a wild tomato gene that has the potential for yield increase in modern breeding.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01341-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {744-745},
  shortjournal = {Nat. Genet.},
  title        = {Tomato super-pangenome highlights the potential use of wild relatives in tomato breeding},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Innate immune cell genetic risk factors are linked to
COVID-19 severity. <em>NG</em>, <em>55</em>(5), 742–743. (<a
href="https://doi.org/10.1038/s41588-023-01378-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA-sequencing analysis combined with host genetic data for a Japanese population reveals the dysfunction of innate immune cells, particularly non-classical monocytes, in individuals with severe COVID-19, as well as enrichment of host genetic risk factors for severe COVID-19 in monocytes and dendritic cells.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01378-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {742-743},
  shortjournal = {Nat. Genet.},
  title        = {Innate immune cell genetic risk factors are linked to COVID-19 severity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Disentangling extrachromosomal circular DNA heterogeneity in
single cells with scEC&amp;t-seq. <em>NG</em>, <em>55</em>(5), 740–741.
(<a href="https://doi.org/10.1038/s41588-023-01385-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce scEC&amp;amp;T-seq, a new single-cell sequencing method that enables parallel profiling of extrachromosomal circular DNA and mRNAs in single cells. Using scEC&amp;amp;T-seq, we characterized all types of circular DNA elements in single human cancer cells and profiled the intercellular heterogeneity and structural dynamics of cancer-specific extrachromosomal DNA.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01385-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {740-741},
  shortjournal = {Nat. Genet.},
  title        = {Disentangling extrachromosomal circular DNA heterogeneity in single cells with scEC&amp;T-seq},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep learning model improves COPD risk prediction and gene
discovery. <em>NG</em>, <em>55</em>(5), 738–739. (<a
href="https://doi.org/10.1038/s41588-023-01388-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Liability scores for chronic obstructive pulmonary disease obtained from our deep learning model improve genetic association discovery and risk prediction. We trained our model using full spirograms and noisy medical record labels obtained from self-reporting and hospital diagnostic codes, and demonstrated that the machine-learning-based phenotyping approach can be generalized to diseases that lack expert-defined annotations.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01388-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {738-739},
  shortjournal = {Nat. Genet.},
  title        = {Deep learning model improves COPD risk prediction and gene discovery},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine learning to examine the genetic underpinnings of
cardiac fibrosis at scale. <em>NG</em>, <em>55</em>(5), 736–737. (<a
href="https://doi.org/10.1038/s41588-023-01369-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a machine learning model to quantify cardiac fibrosis (which is associated with cardiovascular disease) using cardiac MRI data from 41,505 UK Biobank participants. In the subsequent large-scale GWAS of cardiac fibrosis, we identified 11 independent genomic loci, 9 of which were implicated in in vitro cardiac fibroblast activation.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01369-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {736-737},
  shortjournal = {Nat. Genet.},
  title        = {Machine learning to examine the genetic underpinnings of cardiac fibrosis at scale},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Inferring phylogenies from pandemic-scale genome datasets.
<em>NG</em>, <em>55</em>(5), 734–735. (<a
href="https://doi.org/10.1038/s41588-023-01370-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reconstructing phylogenetic trees from large collections of genome sequences is a computationally challenging task. We developed MAPLE, a method for performing phylogenetic inference on large numbers of closely related genomes, which might be useful when studying the evolution and spread of SARS-CoV-2 and of infectious pathogens in future pandemics.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01370-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {734-735},
  shortjournal = {Nat. Genet.},
  title        = {Inferring phylogenies from pandemic-scale genome datasets},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Deciphering DNA variant-associated aberrant splicing with
the aid of RNA sequencing. <em>NG</em>, <em>55</em>(5), 732–733. (<a
href="https://doi.org/10.1038/s41588-023-01363-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant RNA splicing events resulting from DNA variations are common causes of genetic disorders. Two studies published in Nature Genetics independently describe methods to decipher DNA-variant-associated aberrant splicing using high-throughput RNA sequencing data.},
  archive      = {J_NG},
  author       = {Zhang, Bin and Gao, Xin},
  doi          = {10.1038/s41588-023-01363-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {732-733},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Using enormous genealogies to map causal variants in space
and time. <em>NG</em>, <em>55</em>(5), 730–731. (<a
href="https://doi.org/10.1038/s41588-023-01389-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new method infers huge gene trees and tests the tree branches for phenotypic associations. This improves power to map the effects of rare variants that are missing from genotype arrays and imputation panels.},
  archive      = {J_NG},
  author       = {Harris, Kelley},
  doi          = {10.1038/s41588-023-01389-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {730-731},
  shortjournal = {Nat. Genet.},
  title        = {Using enormous genealogies to map causal variants in space and time},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A systematic analysis of immune escape signals in the
primary and metastatic tumor genome. <em>NG</em>, <em>55</em>(5),
728–729. (<a href="https://doi.org/10.1038/s41588-023-01374-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumors develop mechanisms to escape immune destruction. A systematic analysis of large genome sequencing datasets shows that one in four tumors develop genetic immune escape and its prevalence is remarkably similar between primary and metastatic tumors, suggesting that immune escape is an early event during tumor evolution.},
  archive      = {J_NG},
  author       = {Van den Eynden, Jimmy},
  doi          = {10.1038/s41588-023-01374-2},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {728-729},
  shortjournal = {Nat. Genet.},
  title        = {A systematic analysis of immune escape signals in the primary and metastatic tumor genome},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Quantifying tRNA abundance by sequencing. <em>NG</em>,
<em>55</em>(5), 727. (<a
href="https://doi.org/10.1038/s41588-023-01404-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01404-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {727},
  shortjournal = {Nat. Genet.},
  title        = {Quantifying tRNA abundance by sequencing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Shared genetics of substance use disorders. <em>NG</em>,
<em>55</em>(5), 727. (<a
href="https://doi.org/10.1038/s41588-023-01405-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01405-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {727},
  shortjournal = {Nat. Genet.},
  title        = {Shared genetics of substance use disorders},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Linking TCL1A to clonal expansions in blood. <em>NG</em>,
<em>55</em>(5), 727. (<a
href="https://doi.org/10.1038/s41588-023-01406-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01406-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {727},
  shortjournal = {Nat. Genet.},
  title        = {Linking TCL1A to clonal expansions in blood},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Transcription factor affinities in native chromatin.
<em>NG</em>, <em>55</em>(5), 727. (<a
href="https://doi.org/10.1038/s41588-023-01407-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01407-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {727},
  shortjournal = {Nat. Genet.},
  title        = {Transcription factor affinities in native chromatin},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Annotating and prioritizing human non-coding variants with
RegulomeDB v.2. <em>NG</em>, <em>55</em>(5), 724–726. (<a
href="https://doi.org/10.1038/s41588-023-01365-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Dong, Shengcheng and Zhao, Nanxiang and Spragins, Emma and Kagda, Meenakshi S. and Li, Mingjie and Assis, Pedro and Jolanki, Otto and Luo, Yunhai and Cherry, J. Michael and Boyle, Alan P. and Hitz, Benjamin C.},
  doi          = {10.1038/s41588-023-01365-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {724-726},
  shortjournal = {Nat. Genet.},
  title        = {Annotating and prioritizing human non-coding variants with RegulomeDB v.2},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Code deposition is unskippable. <em>NG</em>, <em>55</em>(5),
723. (<a href="https://doi.org/10.1038/s41588-023-01411-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As computational analysis becomes ever-more ubiquitous for researchers, the deposition of the underlying code is now an expected part of publication. Shortcomings in code sharing can lead to delays in peer review and publication, as well as reproducibility issues that are easily avoided with author preparation.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01411-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {723},
  shortjournal = {Nat. Genet.},
  title        = {Code deposition is unskippable},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Genome-wide analyses of ADHD identify 27
risk loci, refine the genetic architecture and implicate several
cognitive domains. <em>NG</em>, <em>55</em>(4), 730. (<a
href="https://doi.org/10.1038/s41588-023-01350-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Demontis, Ditte and Walters, G. Bragi and Athanasiadis, Georgios and Walters, Raymond and Therrien, Karen and Nielsen, Trine Tollerup and Farajzadeh, Leila and Voloudakis, Georgios and Bendl, Jaroslav and Zeng, Biau and Zhang, Wen and Grove, Jakob and Als, Thomas D. and Duan, Jinjie and Satterstrom, F. Kyle and Bybjerg-Grauholm, Jonas and Bækved-Hansen, Marie and Gudmundsson, Olafur O. and Magnusson, Sigurdur H. and Baldursson, Gisli and Davidsdottir, Katrin and Haraldsdottir, Gyda S. and Agerbo, Esben and Hoffman, Gabriel E. and Dalsgaard, Søren and Martin, Jonna and Ribasés, Marta and Boomsma, Dorret I. and Soler Artigas, Maria and Roth Mota, Nina and Howrigan, Daniel and Medland, Sarah E. and Zayats, Tetyana and Rajagopal, Veera M. and Nordentoft, Merete and Mors, Ole and Hougaard, David M. and Mortensen, Preben Bo and Daly, Mark J. and Faraone, Stephen V. and Stefansson, Hreinn and Roussos, Panos and Franke, Barbara and Werge, Thomas and Neale, Benjamin M. and Stefansson, Kari and Børglum, Anders D.},
  doi          = {10.1038/s41588-023-01350-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {730},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Distinct chromatin signatures in the arabidopsis male
gametophyte. <em>NG</em>, <em>55</em>(4), 706–720. (<a
href="https://doi.org/10.1038/s41588-023-01329-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetic reprogramming in the germline contributes to the erasure of epigenetic inheritance across generations in mammals but remains poorly characterized in plants. Here we profiled histone modifications throughout Arabidopsis male germline development. We find that the sperm cell has widespread apparent chromatin bivalency, which is established by the acquisition of H3K27me3 or H3K4me3 at pre-existing H3K4me3 or H3K27me3 regions, respectively. These bivalent domains are associated with a distinct transcriptional status. Somatic H3K27me3 is generally reduced in sperm, while dramatic loss of H3K27me3 is observed at only ~700 developmental genes. The incorporation of the histone variant H3.10 facilitates the establishment of sperm chromatin identity without a strong impact on resetting of somatic H3K27me3. Vegetative nuclei harbor thousands of specific H3K27me3 domains at repressed genes, while pollination-related genes are highly expressed and marked by gene body H3K4me3. Our work highlights putative chromatin bivalency and restricted resetting of H3K27me3 at developmental regulators as key features in plant pluripotent sperm. Arabidopsis sperm cells exhibit widespread chromatin bivalency. Incorporation of histone variant H3.10 regulates sperm chromatin identity. In contrast, in vegetative cells, repressed genes are marked by H3K27me3, while pollination-related genes are active and marked by broad H3K4me3 domains.},
  archive      = {J_NG},
  author       = {Zhu, Danling and Wen, Yi and Yao, Wanyue and Zheng, Haiyan and Zhou, Sixian and Zhang, Qiqi and Qu, Li-Jia and Chen, Xi and Wu, Zhe},
  doi          = {10.1038/s41588-023-01329-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {706-720},
  shortjournal = {Nat. Genet.},
  title        = {Distinct chromatin signatures in the arabidopsis male gametophyte},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). MLL3/MLL4 methyltransferase activities control early
embryonic development and embryonic stem cell differentiation in a
lineage-selective manner. <em>NG</em>, <em>55</em>(4), 693–705. (<a
href="https://doi.org/10.1038/s41588-023-01356-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {H3K4me1 methyltransferases MLL3 (KMT2C) and MLL4 (KMT2D) are critical for enhancer activation, cell differentiation and development. However, roles of MLL3/4 enzymatic activities and MLL3/4-mediated enhancer H3K4me1 in these processes remain unclear. Here we report that constitutive elimination of both MLL3 and MLL4 enzymatic activities prevents initiation of gastrulation and leads to early embryonic lethality in mice. However, selective elimination of MLL3/4 enzymatic activities in embryonic, but not extraembryonic, lineages leaves gastrulation largely intact. Consistent with this, embryonic stem cells (ESCs) lacking MLL3/4 enzymatic activities can differentiate toward the three embryonic germ layers but show aberrant differentiation to extraembryonic endoderm (ExEn) and trophectoderm. The failure in ExEn differentiation can be attributed to markedly reduced enhancer-binding of the lineage-determining transcription factor GATA6. Furthermore, we show that MLL3/4-catalyzed H3K4me1 is largely dispensable for enhancer activation during ESC differentiation. Together, our findings suggest a lineage-selective, but enhancer activation-independent, role of MLL3/4 methyltransferase activities in early embryonic development and ESC differentiation. Disruption of MLL3/4 enzymatic activities prevents gastrulation and leads to early embryonic lethality in mice. This is largely due to defects in extraembryonic lineages, which compromise developmental progression.},
  archive      = {J_NG},
  author       = {Xie, Guojia and Lee, Ji-Eun and Senft, Anna D. and Park, Young-Kwon and Jang, Younghoon and Chakraborty, Shreeta and Thompson, Joyce J. and McKernan, Kaitlin and Liu, Chengyu and Macfarlan, Todd S. and Rocha, Pedro P. and Peng, Weiqun and Ge, Kai},
  doi          = {10.1038/s41588-023-01356-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {693-705},
  shortjournal = {Nat. Genet.},
  title        = {MLL3/MLL4 methyltransferase activities control early embryonic development and embryonic stem cell differentiation in a lineage-selective manner},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Acetylation of histone H2B marks active enhancers and
predicts CBP/p300 target genes. <em>NG</em>, <em>55</em>(4), 679–692.
(<a href="https://doi.org/10.1038/s41588-023-01348-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin features are widely used for genome-scale mapping of enhancers. However, discriminating active enhancers from other cis-regulatory elements, predicting enhancer strength and identifying their target genes is challenging. Here we establish histone H2B N-terminus multisite lysine acetylation (H2BNTac) as a signature of active enhancers. H2BNTac prominently marks candidate active enhancers and a subset of promoters and discriminates them from ubiquitously active promoters. Two mechanisms underlie the distinct H2BNTac specificity: (1) unlike H3K27ac, H2BNTac is specifically catalyzed by CBP/p300; (2) H2A–H2B, but not H3–H4, are rapidly exchanged through transcription-induced nucleosome remodeling. H2BNTac-positive candidate enhancers show a high validation rate in orthogonal enhancer activity assays and a vast majority of endogenously active enhancers are marked by H2BNTac and H3K27ac. Notably, H2BNTac intensity predicts enhancer strength and outperforms current state-of-the-art models in predicting CBP/p300 target genes. These findings have broad implications for generating fine-grained enhancer maps and modeling CBP/p300-dependent gene regulation. Histone H2B N-terminus multisite lysine acetylation (H2BNTac) is identified as a signature of active enhancers. H2BNTac-positive putative enhancers are validated using orthogonal enhancer activity assays.},
  archive      = {J_NG},
  author       = {Narita, Takeo and Higashijima, Yoshiki and Kilic, Sinan and Liebner, Tim and Walter, Jonas and Choudhary, Chunaram},
  doi          = {10.1038/s41588-023-01348-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {679-692},
  shortjournal = {Nat. Genet.},
  title        = {Acetylation of histone H2B marks active enhancers and predicts CBP/p300 target genes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dual genetic tracing reveals a unique fibroblast
subpopulation modulating cardiac fibrosis. <em>NG</em>, <em>55</em>(4),
665–678. (<a href="https://doi.org/10.1038/s41588-023-01337-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After severe heart injury, fibroblasts are activated and proliferate excessively to form scarring, leading to decreased cardiac function and eventually heart failure. It is unknown, however, whether cardiac fibroblasts are heterogeneous with respect to their degree of activation, proliferation and function during cardiac fibrosis. Here, using dual recombinase-mediated genetic lineage tracing, we find that endocardium-derived fibroblasts preferentially proliferate and expand in response to pressure overload. Fibroblast-specific proliferation tracing revealed highly regional expansion of activated fibroblasts after injury, whose pattern mirrors that of endocardium-derived fibroblast distribution in the heart. Specific ablation of endocardium-derived fibroblasts alleviates cardiac fibrosis and reduces the decline of heart function after pressure overload injury. Mechanistically, Wnt signaling promotes activation and expansion of endocardium-derived fibroblasts during cardiac remodeling. Our study identifies endocardium-derived fibroblasts as a key fibroblast subpopulation accounting for severe cardiac fibrosis after pressure overload injury and as a potential therapeutic target against cardiac fibrosis. Genetic lineage tracing in mice shows that endocardium-derived fibroblasts preferentially proliferate in response to pressure overload. Ablation of these cells alleviates cardiac fibrosis and reduces functional decline after pressure overload injury.},
  archive      = {J_NG},
  author       = {Han, Maoying and Liu, Zixin and Liu, Lei and Huang, Xiuzhen and Wang, Haixiao and Pu, Wenjuan and Wang, Enci and Liu, Xiuxiu and Li, Yan and He, Lingjuan and Li, Xufeng and Wu, Jiayu and Qiu, Lin and Shen, Ruling and Wang, Qing-Dong and Ji, Yong and Ardehali, Reza and Shu, Qiang and Lui, Kathy O. and Wang, Lixin and Zhou, Bin},
  doi          = {10.1038/s41588-023-01337-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {665-678},
  shortjournal = {Nat. Genet.},
  title        = {Dual genetic tracing reveals a unique fibroblast subpopulation modulating cardiac fibrosis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Bipotent transitional liver progenitor cells contribute to
liver regeneration. <em>NG</em>, <em>55</em>(4), 651–664. (<a
href="https://doi.org/10.1038/s41588-023-01335-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Following severe liver injury, when hepatocyte-mediated regeneration is impaired, biliary epithelial cells (BECs) can transdifferentiate into functional hepatocytes. However, the subset of BECs with such facultative tissue stem cell potential, as well as the mechanisms enabling transdifferentiation, remains elusive. Here we identify a transitional liver progenitor cell (TLPC), which originates from BECs and differentiates into hepatocytes during regeneration from severe liver injury. By applying a dual genetic lineage tracing approach, we specifically labeled TLPCs and found that they are bipotent, as they either differentiate into hepatocytes or re-adopt BEC fate. Mechanistically, Notch and Wnt/β-catenin signaling orchestrate BEC-to-TLPC and TLPC-to-hepatocyte conversions, respectively. Together, our study provides functional and mechanistic insights into transdifferentiation-assisted liver regeneration. Transitional liver progenitor cells (TLPCs), which derive from biliary epithelial cells (BECs), differentiate into hepatocytes after serious liver damage. Notch and WNT/β-catenin signaling regulate BEC-to-TLPC and TLPC-to-hepatocyte conversions, respectively.},
  archive      = {J_NG},
  author       = {Pu, Wenjuan and Zhu, Huan and Zhang, Mingjun and Pikiolek, Monika and Ercan, Caner and Li, Jie and Huang, Xiuzhen and Han, Ximeng and Zhang, Zhenqian and Lv, Zan and Li, Yan and Liu, Kuo and He, Lingjuan and Liu, Xiuxiu and Heim, Markus H. and Terracciano, Luigi M. and Tchorz, Jan S. and Zhou, Bin},
  doi          = {10.1038/s41588-023-01335-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {651-664},
  shortjournal = {Nat. Genet.},
  title        = {Bipotent transitional liver progenitor cells contribute to liver regeneration},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cellular states are coupled to genomic and viral
heterogeneity in HPV-related oropharyngeal carcinoma. <em>NG</em>,
<em>55</em>(4), 640–650. (<a
href="https://doi.org/10.1038/s41588-023-01357-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Head and neck squamous cell carcinoma (HNSCC) includes a subset of cancers driven by human papillomavirus (HPV). Here we use single-cell RNA-seq to profile both HPV-positive and HPV-negative oropharyngeal tumors, uncovering a high level of cellular diversity within and between tumors. First, we detect diverse chromosomal aberrations within individual tumors, suggesting genomic instability and enabling the identification of malignant cells even at pathologically negative margins. Second, we uncover diversity with respect to HNSCC subtypes and other cellular states such as the cell cycle, senescence and epithelial-mesenchymal transitions. Third, we find heterogeneity in viral gene expression within HPV-positive tumors. HPV expression is lost or repressed in a subset of cells, which are associated with a decrease in HPV-associated cell cycle phenotypes, decreased response to treatment, increased invasion and poor prognosis. These findings suggest that HPV expression diversity must be considered during diagnosis and treatment of HPV-positive tumors, with important prognostic ramifications. Single-cell analysis of head and neck squamous cell carcinoma (HNSCC), specifically from human papillomavirus (HPV)-positive and HPV-negative oropharynx tumors, reveals high levels of inter- and intratumoral heterogeneity. Patterns of HPV gene expression were divergent within HPV-positive tumors, with corresponding functional effects on treatment resistance.},
  archive      = {J_NG},
  author       = {Puram, Sidharth V. and Mints, Michael and Pal, Ananya and Qi, Zongtai and Reeb, Ashley and Gelev, Kyla and Barrett, Thomas F. and Gerndt, Sophie and Liu, Ping and Parikh, Anuraag S. and Ramadan, Salma and Law, Travis and Mroz, Edmund A. and Rocco, James W. and Adkins, Doug and Thorstad, Wade L. and Gay, Hiram A. and Ding, Li and Paniello, Randal C. and Pipkorn, Patrik and Jackson, Ryan S. and Wang, Xiaowei and Mazul, Angela and Chernock, Rebecca and Zevallos, Jose P. and Silva-Fisher, Jessica and Tirosh, Itay},
  doi          = {10.1038/s41588-023-01357-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {640-650},
  shortjournal = {Nat. Genet.},
  title        = {Cellular states are coupled to genomic and viral heterogeneity in HPV-related oropharyngeal carcinoma},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pan-cancer analysis identifies tumor-specific antigens
derived from transposable elements. <em>NG</em>, <em>55</em>(4),
631–639. (<a href="https://doi.org/10.1038/s41588-023-01349-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryptic promoters within transposable elements (TEs) can be transcriptionally reactivated in tumors to create new TE-chimeric transcripts, which can produce immunogenic antigens. We performed a comprehensive screen for these TE exaptation events in 33 TCGA tumor types, 30 GTEx adult tissues and 675 cancer cell lines, and identified 1,068 TE-exapted candidates with the potential to generate shared tumor-specific TE-chimeric antigens (TS-TEAs). Whole-lysate and HLA-pulldown mass spectrometry data confirmed that TS-TEAs are presented on the surface of cancer cells. In addition, we highlight tumor-specific membrane proteins transcribed from TE promoters that constitute aberrant epitopes on the extracellular surface of cancer cells. Altogether, we showcase the high pan-cancer prevalence of TS-TEAs and atypical membrane proteins that could potentially be therapeutically exploited and targeted. Analysis of multiple tumor types, cancer cell lines and adult tissues identifies tumor-specific transposable-element-chimeric transcripts. Mass spectrometry data confirm that many are translated and subsequently located on the extracellular surface of cancer cells, highlighting potential immunogenic therapies.},
  archive      = {J_NG},
  author       = {Shah, Nakul M. and Jang, H. Josh and Liang, Yonghao and Maeng, Ju Heon and Tzeng, Shin-Cheng and Wu, Angela and Basri, Noah L. and Qu, Xuan and Fan, Changxu and Li, Amy and Katz, Benjamin and Li, Daofeng and Xing, Xiaoyun and Evans, Bradley S. and Wang, Ting},
  doi          = {10.1038/s41588-023-01349-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {631-639},
  shortjournal = {Nat. Genet.},
  title        = {Pan-cancer analysis identifies tumor-specific antigens derived from transposable elements},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Neuroblastoma arises in early fetal development and its
evolutionary duration predicts outcome. <em>NG</em>, <em>55</em>(4),
619–630. (<a href="https://doi.org/10.1038/s41588-023-01332-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuroblastoma, the most frequent solid tumor in infants, shows very diverse outcomes from spontaneous regression to fatal disease. When these different tumors originate and how they evolve are not known. Here we quantify the somatic evolution of neuroblastoma by deep whole-genome sequencing, molecular clock analysis and population-genetic modeling in a comprehensive cohort covering all subtypes. We find that tumors across the entire clinical spectrum begin to develop via aberrant mitoses as early as the first trimester of pregnancy. Neuroblastomas with favorable prognosis expand clonally after short evolution, whereas aggressive neuroblastomas show prolonged evolution during which they acquire telomere maintenance mechanisms. The initial aneuploidization events condition subsequent evolution, with aggressive neuroblastoma exhibiting early genomic instability. We find in the discovery cohort (n = 100), and validate in an independent cohort (n = 86), that the duration of evolution is an accurate predictor of outcome. Thus, insight into neuroblastoma evolution may prospectively guide treatment decisions. Somatic evolutionary analysis of neuroblastoma, a pediatric tumor, proposes a common fetal time of origin. Notably, high-risk tumors exhibit early genomic instability and prolonged evolution, and this evolutionary duration predicts clinical outcomes.},
  archive      = {J_NG},
  author       = {Körber, Verena and Stainczyk, Sabine A. and Kurilov, Roma and Henrich, Kai-Oliver and Hero, Barbara and Brors, Benedikt and Westermann, Frank and Höfer, Thomas},
  doi          = {10.1038/s41588-023-01332-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {619-630},
  shortjournal = {Nat. Genet.},
  title        = {Neuroblastoma arises in early fetal development and its evolutionary duration predicts outcome},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiomic analysis of malignant pleural mesothelioma
identifies molecular axes and specialized tumor profiles driving
intertumor heterogeneity. <em>NG</em>, <em>55</em>(4), 607–618. (<a
href="https://doi.org/10.1038/s41588-023-01321-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Malignant pleural mesothelioma (MPM) is an aggressive cancer with rising incidence and challenging clinical management. Through a large series of whole-genome sequencing data, integrated with transcriptomic and epigenomic data using multiomics factor analysis, we demonstrate that the current World Health Organization classification only accounts for up to 10% of interpatient molecular differences. Instead, the MESOMICS project paves the way for a morphomolecular classification of MPM based on four dimensions: ploidy, tumor cell morphology, adaptive immune response and CpG island methylator profile. We show that these four dimensions are complementary, capture major interpatient molecular differences and are delimited by extreme phenotypes that—in the case of the interdependent tumor cell morphology and adapted immune response—reflect tumor specialization. These findings unearth the interplay between MPM functional biology and its genomic history, and provide insights into the variations observed in the clinical behavior of patients with MPM. Genomic, transcriptomic and epigenetic analyses of malignant pleural mesothelioma identify molecular axes and specialized tumor profiles underlying intertumoral heterogeneity.},
  archive      = {J_NG},
  author       = {Mangiante, Lise and Alcala, Nicolas and Sexton-Oates, Alexandra and Di Genova, Alex and Gonzalez-Perez, Abel and Khandekar, Azhar and Bergstrom, Erik N. and Kim, Jaehee and Liu, Xiran and Blazquez-Encinas, Ricardo and Giacobi, Colin and Le Stang, Nolwenn and Boyault, Sandrine and Cuenin, Cyrille and Tabone-Eglinger, Severine and Damiola, Francesca and Voegele, Catherine and Ardin, Maude and Michallet, Marie-Cecile and Soudade, Lorraine and Delhomme, Tiffany M. and Poret, Arnaud and Brevet, Marie and Copin, Marie-Christine and Giusiano-Courcambeck, Sophie and Damotte, Diane and Girard, Cecile and Hofman, Veronique and Hofman, Paul and Mouroux, Jérôme and Cohen, Charlotte and Lacomme, Stephanie and Mazieres, Julien and de Montpreville, Vincent Thomas and Perrin, Corinne and Planchard, Gaetane and Rousseau, Nathalie and Rouquette, Isabelle and Sagan, Christine and Scherpereel, Arnaud and Thivolet, Francoise and Vignaud, Jean-Michel and Jean, Didier and Ilg, Anabelle Gilg Soit and Olaso, Robert and Meyer, Vincent and Boland-Auge, Anne and Deleuze, Jean-Francois and Altmuller, Janine and Nuernberg, Peter and Ibáñez-Costa, Alejandro and Castaño, Justo P. and Lantuejoul, Sylvie and Ghantous, Akram and Maussion, Charles and Courtiol, Pierre and Hernandez-Vargas, Hector and Caux, Christophe and Girard, Nicolas and Lopez-Bigas, Nuria and Alexandrov, Ludmil B. and Galateau-Salle, Françoise and Foll, Matthieu and Fernandez-Cuesta, Lynnette},
  doi          = {10.1038/s41588-023-01321-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {607-618},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic analysis of malignant pleural mesothelioma identifies molecular axes and specialized tumor profiles driving intertumor heterogeneity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Preneoplastic stromal cells promote BRCA1-mediated breast
tumorigenesis. <em>NG</em>, <em>55</em>(4), 595–606. (<a
href="https://doi.org/10.1038/s41588-023-01298-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Women with germline BRCA1 mutations (BRCA1+/mut) have increased risk for hereditary breast cancer. Cancer initiation in BRCA1+/mut is associated with premalignant changes in breast epithelium; however, the role of the epithelium-associated stromal niche during BRCA1-driven tumor initiation remains unclear. Here we show that the premalignant stromal niche promotes epithelial proliferation and mutant BRCA1-driven tumorigenesis in trans. Using single-cell RNA sequencing analysis of human preneoplastic BRCA1+/mut and noncarrier breast tissues, we show distinct changes in epithelial homeostasis including increased proliferation and expansion of basal-luminal intermediate progenitor cells. Additionally, BRCA1+/mut stromal cells show increased expression of pro-proliferative paracrine signals. In particular, we identify pre-cancer-associated fibroblasts (pre-CAFs) that produce protumorigenic factors including matrix metalloproteinase 3 (MMP3), which promotes BRCA1-driven tumorigenesis in vivo. Together, our findings demonstrate that precancerous stroma in BRCA1+/mut may elevate breast cancer risk through the promotion of epithelial proliferation and an accumulation of luminal progenitor cells with altered differentiation. Premalignant stromal cells from women with germline BRCA1 mutations exhibit increased expression of secreted factors regulating epithelial homeostasis in a paracrine fashion. These secreted factors, such as MMP3, promote premalignant epithelial changes including elevated proliferation and altered differentiation of a subpopulation of luminal progenitor cells.},
  archive      = {J_NG},
  author       = {Nee, Kevin and Ma, Dennis and Nguyen, Quy H. and Pein, Maren and Pervolarakis, Nicholas and Insua-Rodríguez, Jacob and Gong, Yanwen and Hernandez, Grace and Alshetaiwi, Hamad and Williams, Justice and Rauf, Maha and Dave, Kushal Rajiv and Boyapati, Keerti and Hasnain, Aliza and Calderon, Christian and Markaryan, Anush and Edwards, Robert and Lin, Erin and Parajuli, Ritesh and Zhou, Peijie and Nie, Qing and Shalabi, Sundus and LaBarge, Mark A. and Kessenbrock, Kai},
  doi          = {10.1038/s41588-023-01298-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {595-606},
  shortjournal = {Nat. Genet.},
  title        = {Preneoplastic stromal cells promote BRCA1-mediated breast tumorigenesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiancestry genomic and transcriptomic analysis of gastric
cancer. <em>NG</em>, <em>55</em>(4), 581–594. (<a
href="https://doi.org/10.1038/s41588-023-01333-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gastric cancer is among the most common malignancies worldwide, characterized by geographical, epidemiological and histological heterogeneity. Here, we report an extensive, multiancestral landscape of driver events in gastric cancer, involving 1,335 cases. Seventy-seven significantly mutated genes (SMGs) were identified, including ARHGAP5 and TRIM49C. We also identified subtype-specific drivers, including PIGR and SOX9, which were enriched in the diffuse subtype of the disease. SMGs also varied according to Epstein–Barr virus infection status and ancestry. Non-protein-truncating CDH1 mutations, which are characterized by in-frame splicing alterations, targeted localized extracellular domains and uniquely occurred in sporadic diffuse-type cases. In patients with gastric cancer with East Asian ancestry, our data suggested a link between alcohol consumption or metabolism and the development of RHOA mutations. Moreover, mutations with potential roles in immune evasion were identified. Overall, these data provide comprehensive insights into the molecular landscape of gastric cancer across various subtypes and ancestries. A cross-ancestry genomic and transcriptomic cohort of gastric cancer highlights significantly mutated genes and mutational signatures, some of which are ancestry-specific.},
  archive      = {J_NG},
  author       = {Totoki, Yasushi and Saito-Adachi, Mihoko and Shiraishi, Yuichi and Komura, Daisuke and Nakamura, Hiromi and Suzuki, Akihiro and Tatsuno, Kenji and Rokutan, Hirofumi and Hama, Natsuko and Yamamoto, Shogo and Ono, Hanako and Arai, Yasuhito and Hosoda, Fumie and Katoh, Hiroto and Chiba, Kenichi and Iida, Naoko and Nagae, Genta and Ueda, Hiroki and Shihang, Chen and Sekine, Shigeki and Abe, Hiroyuki and Nomura, Sachiyo and Matsuura, Tetsuya and Sakai, Eiji and Ohshima, Takashi and Rino, Yasushi and Yeoh, Khay Guan and So, Jimmy and Sanghvi, Kaushal and Soong, Richie and Fukagawa, Akihiko and Yachida, Shinichi and Kato, Mamoru and Seto, Yasuyuki and Ushiku, Tetsuo and Nakajima, Atsushi and Katai, Hitoshi and Tan, Patrick and Ishikawa, Shumpei and Aburatani, Hiroyuki and Shibata, Tatsuhiro},
  doi          = {10.1038/s41588-023-01333-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {581-594},
  shortjournal = {Nat. Genet.},
  title        = {Multiancestry genomic and transcriptomic analysis of gastric cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Thymidine nucleotide metabolism controls human telomere
length. <em>NG</em>, <em>55</em>(4), 568–580. (<a
href="https://doi.org/10.1038/s41588-023-01339-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Telomere length in humans is associated with lifespan and severe diseases, yet the genetic determinants of telomere length remain incompletely defined. Here we performed genome-wide CRISPR–Cas9 functional telomere length screening and identified thymidine (dT) nucleotide metabolism as a limiting factor in human telomere maintenance. Targeted genetic disruption using CRISPR–Cas9 revealed multiple telomere length control points across the thymidine nucleotide metabolism pathway: decreasing dT nucleotide salvage via deletion of the gene encoding nuclear thymidine kinase (TK1) or de novo production by knockout of the thymidylate synthase gene (TYMS) decreased telomere length, whereas inactivation of the deoxynucleoside triphosphohydrolase-encoding gene SAMHD1 lengthened telomeres. Remarkably, supplementation with dT alone drove robust telomere elongation by telomerase in cells, and thymidine triphosphate stimulated telomerase activity in a substrate-independent manner in vitro. In induced pluripotent stem cells derived from patients with genetic telomere biology disorders, dT supplementation or inhibition of SAMHD1 promoted telomere restoration. Our results demonstrate a critical role of thymidine metabolism in controlling human telomerase and telomere length, which may be therapeutically actionable in patients with fatal degenerative diseases. Genome-wide CRISPR screening identifies thymidine nucleotide metabolism as a key regulator of human telomere length. Thymidine supplementation promotes telomere elongation in cells derived from patients with telomere biology disorders.},
  archive      = {J_NG},
  author       = {Mannherz, William and Agarwal, Suneet},
  doi          = {10.1038/s41588-023-01339-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {568-580},
  shortjournal = {Nat. Genet.},
  title        = {Thymidine nucleotide metabolism controls human telomere length},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Genetic effects on the timing of parturition and links to
fetal birth weight. <em>NG</em>, <em>55</em>(4), 559–567. (<a
href="https://doi.org/10.1038/s41588-023-01343-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The timing of parturition is crucial for neonatal survival and infant health. Yet, its genetic basis remains largely unresolved. We present a maternal genome-wide meta-analysis of gestational duration (n = 195,555), identifying 22 associated loci (24 independent variants) and an enrichment in genes differentially expressed during labor. A meta-analysis of preterm delivery (18,797 cases, 260,246 controls) revealed seven associated loci and large genetic similarities with gestational duration. Analysis of the parental transmitted and nontransmitted alleles (n = 136,833) shows that 15 of the gestational duration genetic variants act through the maternal genome, whereas 7 act both through the maternal and fetal genomes and 2 act only via the fetal genome. Finally, the maternal effects on gestational duration show signs of antagonistic pleiotropy with the fetal effects on birth weight: maternal alleles that increase gestational duration have negative fetal effects on birth weight. The present study provides insights into the genetic effects on the timing of parturition and the complex maternal–fetal relationship between gestational duration and birth weight. Maternal genome-wide analyses identify variants associated with gestational duration and preterm delivery. Maternal alleles positively associated with gestational duration exhibit negative fetal effects on birth weight, likely reflecting antagonistic pleiotropy.},
  archive      = {J_NG},
  author       = {Solé-Navais, Pol and Flatley, Christopher and Steinthorsdottir, Valgerdur and Vaudel, Marc and Juodakis, Julius and Chen, Jing and Laisk, Triin and LaBella, Abigail L. and Westergaard, David and Bacelis, Jonas and Brumpton, Ben and Skotte, Line and Borges, Maria C. and Helgeland, Øyvind and Mahajan, Anubha and Wielscher, Matthias and Lin, Frederick and Briggs, Catherine and Wang, Carol A. and Moen, Gunn-Helen and Beaumont, Robin N. and Bradfield, Jonathan P. and Abraham, Abin and Thorleifsson, Gudmar and Gabrielsen, Maiken E. and Ostrowski, Sisse R. and Modzelewska, Dominika and Nohr, Ellen A. and Hypponen, Elina and Srivastava, Amit and Talbot, Octavious and Allard, Catherine and Williams, Scott M. and Menon, Ramkumar and Shields, Beverley M. and Sveinbjornsson, Gardar and Xu, Huan and Melbye, Mads and Lowe, William and Bouchard, Luigi and Oken, Emily and Pedersen, Ole B. and Gudbjartsson, Daniel F. and Erikstrup, Christian and Sørensen, Erik and Lie, Rolv T. and Teramo, Kari and Hallman, Mikko and Juliusdottir, Thorhildur and Hakonarson, Hakon and Ullum, Henrik and Hattersley, Andrew T. and Sletner, Line and Merialdi, Mario and Rifas-Shiman, Sheryl L. and Steingrimsdottir, Thora and Scholtens, Denise and Power, Christine and West, Jane and Nyegaard, Mette and Capra, John A. and Skogholt, Anne H. and Magnus, Per and Andreassen, Ole A. and Thorsteinsdottir, Unnur and Grant, Struan F. A. and Qvigstad, Elisabeth and Pennell, Craig E. and Hivert, Marie-France and Hayes, Geoffrey M. and Jarvelin, Marjo-Riitta and McCarthy, Mark I. and Lawlor, Deborah A. and Nielsen, Henriette S. and Mägi, Reedik and Rokas, Antonis and Hveem, Kristian and Stefansson, Kari and Feenstra, Bjarke and Njolstad, Pål and Muglia, Louis J. and Freathy, Rachel M. and Johansson, Stefan and Zhang, Ge and Jacobsson, Bo},
  doi          = {10.1038/s41588-023-01343-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {559-567},
  shortjournal = {Nat. Genet.},
  title        = {Genetic effects on the timing of parturition and links to fetal birth weight},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Causal effects on complex traits are similar for common
variants across segments of different continental ancestries within
admixed individuals. <em>NG</em>, <em>55</em>(4), 549–558. (<a
href="https://doi.org/10.1038/s41588-023-01338-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Individuals of admixed ancestries (for example, African Americans) inherit a mosaic of ancestry segments (local ancestry) originating from multiple continental ancestral populations. This offers the unique opportunity of investigating the similarity of genetic effects on traits across ancestries within the same population. Here we introduce an approach to estimate correlation of causal genetic effects (radmix) across local ancestries and analyze 38 complex traits in African-European admixed individuals (N = 53,001) to observe very high correlations (meta-analysis radmix = 0.95, 95% credible interval 0.93–0.97), much higher than correlation of causal effects across continental ancestries. We replicate our results using regression-based methods from marginal genome-wide association study summary statistics. We also report realistic scenarios where regression-based methods yield inflated heterogeneity-by-ancestry due to ancestry-specific tagging of causal effects, and/or polygenicity. Our results motivate genetic analyses that assume minimal heterogeneity in causal effects by ancestry, with implications for the inclusion of ancestry-diverse individuals in studies. This analysis of individuals of admixed genetic ancestries suggests that complex trait causal variant effect sizes are, by and large, similar across ancestries, and discusses the implications for the study of these and other diverse populations.},
  archive      = {J_NG},
  author       = {Hou, Kangcheng and Ding, Yi and Xu, Ziqi and Wu, Yue and Bhattacharya, Arjun and Mester, Rachel and Belbin, Gillian M. and Buyske, Steve and Conti, David V. and Darst, Burcu F. and Fornage, Myriam and Gignoux, Chris and Guo, Xiuqing and Haiman, Christopher and Kenny, Eimear E. and Kim, Michelle and Kooperberg, Charles and Lange, Leslie and Manichaikul, Ani and North, Kari E. and Peters, Ulrike and Rasmussen-Torvik, Laura J. and Rich, Stephen S. and Rotter, Jerome I. and Wheeler, Heather E. and Wojcik, Genevieve L. and Zhou, Ying and Sankararaman, Sriram and Pasaniuc, Bogdan},
  doi          = {10.1038/s41588-023-01338-6},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {549-558},
  shortjournal = {Nat. Genet.},
  title        = {Causal effects on complex traits are similar for common variants across segments of different continental ancestries within admixed individuals},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Adjusting for common variant polygenic scores improves yield
in rare variant association analyses. <em>NG</em>, <em>55</em>(4),
544–548. (<a href="https://doi.org/10.1038/s41588-023-01342-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the emergence of large-scale sequencing data, methods for improving power in rare variant association tests are needed. Here we show that adjusting for common variant polygenic scores improves yield in gene-based rare variant association tests across 65 quantitative traits in the UK Biobank (up to 20% increase at α = 2.6 × 10−6), without marked increases in false-positive rates or genomic inflation. Benefits were seen for various models, with the largest improvements seen for efficient sparse mixed-effects models. Our results illustrate how polygenic score adjustment can efficiently improve power in rare variant association discovery. Adjusting for common variant polygenic scores improves yield in gene-based rare variant association tests for quantitative traits, particularly when using sparse mixed models or simple linear models as an alternative to dense mixed-model approaches.},
  archive      = {J_NG},
  author       = {Jurgens, Sean J. and Pirruccello, James P. and Choi, Seung Hoan and Morrill, Valerie N. and Chaffin, Mark and Lubitz, Steven A. and Lunetta, Kathryn L. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-023-01342-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {544-548},
  shortjournal = {Nat. Genet.},
  title        = {Adjusting for common variant polygenic scores improves yield in rare variant association analyses},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). New genetic loci associated with gestational duration show a
complex link with birth weight. <em>NG</em>, <em>55</em>(4), 542–543.
(<a href="https://doi.org/10.1038/s41588-023-01344-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A GWAS meta-analysis, combined with tracing the parental transmitted and non-transmitted alleles in parent–offspring pairs, enabled us to distinguish the effects of maternal and fetal genomes on gestational duration and their links with birth weight. The identified genes are more likely to be differentially expressed during labor and show signs of antagonistic pleiotropy with fetal genome effects on birth weight.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01344-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {542-543},
  shortjournal = {Nat. Genet.},
  title        = {New genetic loci associated with gestational duration show a complex link with birth weight},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Lifting the curtain on molecular differences between
malignant pleural mesotheliomas. <em>NG</em>, <em>55</em>(4), 540–541.
(<a href="https://doi.org/10.1038/s41588-023-01322-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce molecular and cellular criteria — based on morphology, ploidy, CpG island methylation and immune infiltration — that improve the characterization of malignant pleural mesothelioma. These criteria reveal adaptation strategies that are adopted by tumor cells and offer new possibilities for classification and clinical management.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01322-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {540-541},
  shortjournal = {Nat. Genet.},
  title        = {Lifting the curtain on molecular differences between malignant pleural mesotheliomas},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chimeric transcripts of transposable elements and genes are
a source of tumor-specific antigens. <em>NG</em>, <em>55</em>(4),
538–539. (<a href="https://doi.org/10.1038/s41588-023-01361-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposable elements (TEs) are transcriptionally activated in cancer and can generate chimeric TE–gene transcripts that are tumor specific. Our pan-cancer study reveals that these transcripts are a source of tumor-specific antigens that are shared across tumor types, bind to HLA and are presented on the extracellular surface of cancer cells appended to membrane-bound proteins.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01361-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {538-539},
  shortjournal = {Nat. Genet.},
  title        = {Chimeric transcripts of transposable elements and genes are a source of tumor-specific antigens},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Prevalent putative chromatin bivalency and partial resetting
of H3K27me3 in plant sperm. <em>NG</em>, <em>55</em>(4), 536–537. (<a
href="https://doi.org/10.1038/s41588-023-01345-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How histone modifications are reprogrammed through germline development in plants is poorly understood. We found that H3K27me3 and H3K4me3 are extensively reprogrammed throughout the development of Arabidopsis male gametophyte. This reprogramming leads to widespread chromatin bivalency and selective removal of H3K27me3 marks from key developmental regulators in sperm.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01345-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {536-537},
  shortjournal = {Nat. Genet.},
  title        = {Prevalent putative chromatin bivalency and partial resetting of H3K27me3 in plant sperm},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Unexpected heterogeneity in oropharyngeal squamous cell
tumors. <em>NG</em>, <em>55</em>(4), 534–535. (<a
href="https://doi.org/10.1038/s41588-023-01360-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study uses single-cell RNA sequencing to profile human papillomavirus (HPV)-positive and -negative oropharyngeal squamous cell carcinoma, revealing considerable diversity within and between tumors. Within HPV-positive tumors, subsets of malignant cells are found with undetectable HPV expression and decreased HPV-related phenotypes, which may influence prognosis and response to therapy.},
  archive      = {J_NG},
  author       = {Peace, David J. and Izumchenko, Evgeny and Sidransky, David},
  doi          = {10.1038/s41588-023-01360-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {534-535},
  shortjournal = {Nat. Genet.},
  title        = {Unexpected heterogeneity in oropharyngeal squamous cell tumors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Nucleotide metabolism regulates human telomere length via
telomerase activation. <em>NG</em>, <em>55</em>(4), 532–533. (<a
href="https://doi.org/10.1038/s41588-023-01359-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Telomere length is an important determinant of cellular aging and disease risk, but the genetics of telomere length control in humans is unclear. A genome-wide CRISPR screen has now identified a central role for thymidine nucleotide metabolism in the regulation of telomere length, which has implications for the diagnosis and treatment of disease.},
  archive      = {J_NG},
  author       = {Bryan, Tracy M.},
  doi          = {10.1038/s41588-023-01359-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {532-533},
  shortjournal = {Nat. Genet.},
  title        = {Nucleotide metabolism regulates human telomere length via telomerase activation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mathematical modeling of neuroblastoma associates
evolutionary patterns with outcomes. <em>NG</em>, <em>55</em>(4),
530–531. (<a href="https://doi.org/10.1038/s41588-023-01358-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study deciphers the origin and evolution of childhood neuroblastoma using genome sequencing data, mathematical models and statistical inference, showing how neuroblastoma evolution is an accurate predictor of outcome.},
  archive      = {J_NG},
  author       = {Caravagna, Giulio},
  doi          = {10.1038/s41588-023-01358-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {530-531},
  shortjournal = {Nat. Genet.},
  title        = {Mathematical modeling of neuroblastoma associates evolutionary patterns with outcomes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Estimation of cross-ancestry genetic correlations within
ancestry tracts of admixed samples. <em>NG</em>, <em>55</em>(4),
527–529. (<a href="https://doi.org/10.1038/s41588-023-01325-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantifying whether different populations share similar effect sizes of common causal variants is vital to understand the genetic basis of disease and build better prediction models. A new study proposes a method leveraging admixture to estimate the correlation of causal genetic variants and finds they are largely similar across ancestry backgrounds.},
  archive      = {J_NG},
  author       = {Atkinson, Elizabeth G.},
  doi          = {10.1038/s41588-023-01325-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {527-529},
  shortjournal = {Nat. Genet.},
  title        = {Estimation of cross-ancestry genetic correlations within ancestry tracts of admixed samples},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Catching active enhancers via H2B n-terminal acetylation.
<em>NG</em>, <em>55</em>(4), 525–526. (<a
href="https://doi.org/10.1038/s41588-023-01347-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Specific chromatin features, especially histone H3 lysine 27 acetylation, are widely used to identify active enhancers, yet current methods are imprecise. New work suggests that histone H2B N terminus multisite lysine acetylation (H2BNTac) is a notable signature of active enhancers and could substantially improve enhancer prediction.},
  archive      = {J_NG},
  author       = {Huang, Chang and Helin, Kristian},
  doi          = {10.1038/s41588-023-01347-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {525-526},
  shortjournal = {Nat. Genet.},
  title        = {Catching active enhancers via H2B N-terminal acetylation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Rare disease gene mining. <em>NG</em>, <em>55</em>(4), 524.
(<a href="https://doi.org/10.1038/s41588-023-01380-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01380-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {524},
  shortjournal = {Nat. Genet.},
  title        = {Rare disease gene mining},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Mutation rates across species. <em>NG</em>, <em>55</em>(4),
524. (<a href="https://doi.org/10.1038/s41588-023-01381-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01381-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {524},
  shortjournal = {Nat. Genet.},
  title        = {Mutation rates across species},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Whole-genome doubling perturbs the 3D genome. <em>NG</em>,
<em>55</em>(4), 524. (<a
href="https://doi.org/10.1038/s41588-023-01382-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01382-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {524},
  shortjournal = {Nat. Genet.},
  title        = {Whole-genome doubling perturbs the 3D genome},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Redefining teleost phylogeny. <em>NG</em>, <em>55</em>(4),
524. (<a href="https://doi.org/10.1038/s41588-023-01383-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01383-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {524},
  shortjournal = {Nat. Genet.},
  title        = {Redefining teleost phylogeny},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Charles david allis (1951–2023). <em>NG</em>,
<em>55</em>(4), 522–523. (<a
href="https://doi.org/10.1038/s41588-023-01331-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Chi, Ping and Lewis, Peter W. and Lu, Chao and Lu, Janice and Ruthenburg, Alexander J. and Sabari, Benjamin R. and Shechter, David and Wan, Liling and Wang, Gang Greg},
  doi          = {10.1038/s41588-023-01331-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {522-523},
  shortjournal = {Nat. Genet.},
  title        = {Charles david allis (1951–2023)},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The allis code. <em>NG</em>, <em>55</em>(4), 521. (<a
href="https://doi.org/10.1038/s41588-023-01362-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {David Allis (1951–2023) was a leading figure in the field of chromatin biology. He inspired many generations of scientists both through his work and his own personal example as a mentor and colleague. His influential ‘histone code’ theory remains an important guiding principle to study and understand gene regulation.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01362-6},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {521},
  shortjournal = {Nat. Genet.},
  title        = {The allis code},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Discovery of 42 genome-wide significant
loci associated with dyslexia. <em>NG</em>, <em>55</em>(3), 520. (<a
href="https://doi.org/10.1038/s41588-023-01336-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Doust, Catherine and Fontanillas, Pierre and Eising, Else and Gordon, Scott D. and Wang, Zhengjun and Alagöz, Gökberk and Molz, Barbara and Pourcain, Beate St and Francks, Clyde and Marioni, Riccardo E. and Zhao, Jingjing and Paracchini, Silvia and Talcott, Joel B. and Monaco, Anthony P. and Stein, John F. and Gruen, Jeffrey R. and Olson, Richard K. and Willcutt, Erik G. and DeFries, John C. and Pennington, Bruce F. and Smith, Shelley D. and Wright, Margaret J. and Martin, Nicholas G. and Auton, Adam and Bates, Timothy C. and Fisher, Simon E. and Luciano, Michelle},
  doi          = {10.1038/s41588-023-01336-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {520},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Discovery of 42 genome-wide significant loci associated with dyslexia},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Deciphering colorectal cancer genetics
through multi-omic analysis of 100,204 cases and 154,587 controls of
european and east asian ancestries. <em>NG</em>, <em>55</em>(3),
519–520. (<a href="https://doi.org/10.1038/s41588-023-01334-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fernandez-Rozadilla, Ceres and Timofeeva, Maria and Chen, Zhishan and Law, Philip and Thomas, Minta and Schmit, Stephanie and Díez-Obrero, Virginia and Hsu, Li and Fernandez-Tajes, Juan and Palles, Claire and Sherwood, Kitty and Briggs, Sarah and Svinti, Victoria and Donnelly, Kevin and Farrington, Susan and Blackmur, James and Vaughan-Shaw, Peter and Shu, Xiao-ou and Long, Jirong and Cai, Qiuyin and Guo, Xingyi and Lu, Yingchang and Broderick, Peter and Studd, James and Huyghe, Jeroen and Harrison, Tabitha and Conti, David and Dampier, Christopher and Devall, Mathew and Schumacher, Fredrick and Melas, Marilena and Rennert, Gad and Obón-Santacana, Mireia and Martín-Sánchez, Vicente and Moratalla-Navarro, Ferran and Oh, Jae Hwan and Kim, Jeongseon and Jee, Sun Ha and Jung, Keum Ji and Kweon, Sun-Seog and Shin, Min-Ho and Shin, Aesun and Ahn, Yoon-Ok and Kim, Dong-Hyun and Oze, Isao and Wen, Wanqing and Matsuo, Keitaro and Matsuda, Koichi and Tanikawa, Chizu and Ren, Zefang and Gao, Yu-Tang and Jia, Wei-Hua and Hopper, John and Jenkins, Mark and Win, Aung Ko and Pai, Rish and Figueiredo, Jane and Haile, Robert and Gallinger, Steven and Woods, Michael and Newcomb, Polly and Duggan, David and Cheadle, Jeremy and Kaplan, Richard and Maughan, Timothy and Kerr, Rachel and Kerr, David and Kirac, Iva and Böhm, Jan and Mecklin, Lukka-Pekka and Jousilahti, Pekka and Knekt, Paul and Aaltonen, Lauri and Rissanen, Harri and Pukkala, Eero and Eriksson, Johan and Cajuso, Tatiana and Hänninen, Ulrika and Kondelin, Johanna and Palin, Kimmo and Tanskanen, Tomas and Renkonen-Sinisalo, Laura and Zanke, Brent and Männistö, Satu and Albanes, Demetrius and Weinstein, Stephanie and Ruiz-Narvaez, Edward and Palmer, Julie and Buchanan, Daniel and Platz, Elizabeth and Visvanathan, Kala and Ulrich, Cornelia and Siegel, Erin and Brezina, Stefanie and Gsur, Andrea and Campbell, Peter and Chang-Claude, Jenny and Hoffmeister, Michael and Brenner, Hermann and Slattery, Martha and Potter, John and Tsilidis, Konstantinos and Schulze, Matthias and Gunter, Marc and Murphy, Neil and Castells, Antoni and Castellví-Bel, Sergi and Moreira, Leticia and Arndt, Volker and Shcherbina, Anna and Stern, Mariana and Pardamean, Bens and Bishop, Timothy and Giles, Graham and Southey, Melissa and Idos, Gregory and McDonnell, Kevin and Abu-Ful, Zomoroda and Greenson, Joel and Shulman, Katerina and Lejbkowicz, Flavio and Offit, Kenneth and Su, Yu-Ru and Steinfelder, Robert and Keku, Temitope and van Guelpen, Bethany and Hudson, Thomas and Hampel, Heather and Pearlman, Rachel and Berndt, Sonja and Hayes, Richard and Martinez, Marie Elena and Thomas, Sushma and Corley, Douglas and Pharoah, Paul and Larsson, Susanna and Yen, Yun and Lenz, Heinz-Josef and White, Emily and Li, Li and Doheny, Kimberly and Pugh, Elizabeth and Shelford, Tameka and Chan, Andrew and Cruz-Correa, Marcia and Lindblom, Annika and Hunter, David and Joshi, Amit and Schafmayer, Clemens and Scacheri, Peter and Kundaje, Anshul and Nickerson, Deborah and Schoen, Robert and Hampe, Jochen and Stadler, Zsofia and Vodicka, Pavel and Vodickova, Ludmila and Vymetalkova, Veronika and Papadopoulos, Nickolas and Edlund, Chistopher and Gauderman, William and Thomas, Duncan and Shibata, David and Toland, Amanda and Markowitz, Sanford and Kim, Andre and Chanock, Stephen and van Duijnhoven, Franzel and Feskens, Edith and Sakoda, Lori and Gago-Dominguez, Manuela and Wolk, Alicja and Naccarati, Alessio and Pardini, Barbara and FitzGerald, Liesel and Lee, Soo Chin and Ogino, Shuji and Bien, Stephanie and Kooperberg, Charles and Li, Christopher and Lin, Yi and Prentice, Ross and Qu, Conghui and Bézieau, Stéphane and Tangen, Catherine and Mardis, Elaine and Yamaji, Taiki and Sawada, Norie and Iwasaki, Motoki and Haiman, Christopher and Le Marchand, Loic and Wu, Anna and Qu, Chenxu and McNeil, Caroline and Coetzee, Gerhard and Hayward, Caroline and Deary, Ian and Harris, Sarah and Theodoratou, Evropi and Reid, Stuart and Walker, Marion and Ooi, Li Yin and Moreno, Victor and Casey, Graham and Gruber, Stephen and Tomlinson, Ian and Zheng, Wei and Dunlop, Malcolm and Houlston, Richard and Peters, Ulrike},
  doi          = {10.1038/s41588-023-01334-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {519-520},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Deciphering colorectal cancer genetics through multi-omic analysis of 100,204 cases and 154,587 controls of european and east asian ancestries},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pangenomic analysis identifies structural variation
associated with heat tolerance in pearl millet. <em>NG</em>,
<em>55</em>(3), 507–518. (<a
href="https://doi.org/10.1038/s41588-023-01302-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pearl millet is an important cereal crop worldwide and shows superior heat tolerance. Here, we developed a graph-based pan-genome by assembling ten chromosomal genomes with one existing assembly adapted to different climates worldwide and captured 424,085 genomic structural variations (SVs). Comparative genomics and transcriptomics analyses revealed the expansion of the RWP-RK transcription factor family and the involvement of endoplasmic reticulum (ER)-related genes in heat tolerance. The overexpression of one RWP-RK gene led to enhanced plant heat tolerance and transactivated ER-related genes quickly, supporting the important roles of RWP-RK transcription factors and ER system in heat tolerance. Furthermore, we found that some SVs affected the gene expression associated with heat tolerance and SVs surrounding ER-related genes shaped adaptation to heat tolerance during domestication in the population. Our study provides a comprehensive genomic resource revealing insights into heat tolerance and laying a foundation for generating more robust crops under the changing climate. A graph-based pan-genome constructed using de novo genome assemblies of ten pearl millet accessions adapted to different climates worldwide identifies structural variations and their contribution to heat tolerance in pearl millet.},
  archive      = {J_NG},
  author       = {Yan, Haidong and Sun, Min and Zhang, Zhongren and Jin, Yarong and Zhang, Ailing and Lin, Chuang and Wu, Bingchao and He, Min and Xu, Bin and Wang, Jing and Qin, Peng and Mendieta, John Pablo and Nie, Gang and Wang, Jianping and Jones, Chris S. and Feng, Guangyan and Srivastava, Rakesh K. and Zhang, Xinquan and Bombarely, Aureliano and Luo, Dan and Jin, Long and Peng, Yuanying and Wang, Xiaoshan and Ji, Yang and Tian, Shilin and Huang, Linkai},
  doi          = {10.1038/s41588-023-01302-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {507-518},
  shortjournal = {Nat. Genet.},
  title        = {Pangenomic analysis identifies structural variation associated with heat tolerance in pearl millet},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome assembly and genetic dissection of a prominent
drought-resistant maize germplasm. <em>NG</em>, <em>55</em>(3), 496–506.
(<a href="https://doi.org/10.1038/s41588-023-01297-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the context of climate change, drought is one of the most limiting factors that influence crop production. Maize, as a major crop, is highly vulnerable to water deficit, which causes significant yield loss. Thus, identification and utilization of drought-resistant germplasm are crucial for the genetic improvement of the trait. Here we report on a high-quality genome assembly of a prominent drought-resistant genotype, CIMBL55. Genomic and genetic variation analyses revealed that 65 favorable alleles of 108 previously identified drought-resistant candidate genes were found in CIMBL55, which may constitute the genetic basis for its excellent drought resistance. Notably, ZmRtn16, encoding a reticulon-like protein, was found to contribute to drought resistance by facilitating the vacuole H+-ATPase activity, which highlights the role of vacuole proton pumps in maize drought resistance. The assembled CIMBL55 genome provided a basis for genetic dissection and improvement of plant drought resistance, in support of global food security. High-quality genome assembly of a prominent drought-resistant maize germplasm CIMBL55 and genetic variation analyses provide a resource for genetic dissection and result in the improvement of maize drought resistance.},
  archive      = {J_NG},
  author       = {Tian, Tian and Wang, Shuhui and Yang, Shiping and Yang, Zhirui and Liu, Shengxue and Wang, Yijie and Gao, Huajian and Zhang, Shuaisong and Yang, Xiaohong and Jiang, Caifu and Qin, Feng},
  doi          = {10.1038/s41588-023-01297-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {496-506},
  shortjournal = {Nat. Genet.},
  title        = {Genome assembly and genetic dissection of a prominent drought-resistant maize germplasm},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Transcription of MERVL retrotransposons is required for
preimplantation embryo development. <em>NG</em>, <em>55</em>(3),
484–495. (<a href="https://doi.org/10.1038/s41588-023-01324-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zygotic genome activation (ZGA) is a critical postfertilization step that promotes totipotency and allows different cell fates to emerge in the developing embryo. MERVL (murine endogenous retrovirus-L) is transiently upregulated at the two-cell stage during ZGA. Although MERVL expression is widely used as a marker of totipotency, the role of this retrotransposon in mouse embryogenesis remains elusive. Here, we show that full-length MERVL transcripts, but not encoded retroviral proteins, are essential for accurate regulation of the host transcriptome and chromatin state during preimplantation development. Both knockdown and CRISPRi-based repression of MERVL result in embryonic lethality due to defects in differentiation and genomic stability. Furthermore, transcriptome and epigenome analysis revealed that loss of MERVL transcripts led to retention of an accessible chromatin state at, and aberrant expression of, a subset of two-cell-specific genes. Taken together, our results suggest a model in which an endogenous retrovirus plays a key role in regulating host cell fate potential. MERVL transcripts, but not MERVL-encoded proteins, are essential for mouse preimplantation development. Depletion of MERVL RNA perturbs early lineage specification and leads to increased chromatin accessibility at, and transcription of, two-cell-specific genes.},
  archive      = {J_NG},
  author       = {Sakashita, Akihiko and Kitano, Tomohiro and Ishizu, Hirotsugu and Guo, Youjia and Masuda, Harumi and Ariura, Masaru and Murano, Kensaku and Siomi, Haruhiko},
  doi          = {10.1038/s41588-023-01324-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {484-495},
  shortjournal = {Nat. Genet.},
  title        = {Transcription of MERVL retrotransposons is required for preimplantation embryo development},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pharmacological disruption of mSWI/SNF complex activity
restricts SARS-CoV-2 infection. <em>NG</em>, <em>55</em>(3), 471–483.
(<a href="https://doi.org/10.1038/s41588-023-01307-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of host determinants of coronavirus infection informs mechanisms of viral pathogenesis and can provide new drug targets. Here we demonstrate that mammalian SWItch/Sucrose Non-Fermentable (mSWI/SNF) chromatin remodeling complexes, specifically canonical BRG1/BRM-associated factor (cBAF) complexes, promote severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection and represent host-directed therapeutic targets. The catalytic activity of SMARCA4 is required for mSWI/SNF-driven chromatin accessibility at the ACE2 locus, ACE2 expression and virus susceptibility. The transcription factors HNF1A/B interact with and recruit mSWI/SNF complexes to ACE2 enhancers, which contain high HNF1A motif density. Notably, small-molecule mSWI/SNF ATPase inhibitors or degraders abrogate angiotensin-converting enzyme 2 (ACE2) expression and confer resistance to SARS-CoV-2 variants and a remdesivir-resistant virus in three cell lines and three primary human cell types, including airway epithelial cells, by up to 5 logs. These data highlight the role of mSWI/SNF complex activities in conferring SARS-CoV-2 susceptibility and identify a potential class of broad-acting antivirals to combat emerging coronaviruses and drug-resistant variants. The canonical BRG1/BRM-associated factor (cBAF) complex is recruited by HNF1A/B to angiotensin-converting enzyme 2 (ACE2) enhancers, promoting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection. Inhibition of the catalytic activity of SMARCA4 precludes ACE2 expression and reduces susceptibility to SARS-CoV-1 and major SARS-CoV-2 variants.},
  archive      = {J_NG},
  author       = {Wei, Jin and Patil, Ajinkya and Collings, Clayton K. and Alfajaro, Mia Madel and Liang, Yu and Cai, Wesley L. and Strine, Madison S. and Filler, Renata B. and DeWeirdt, Peter C. and Hanna, Ruth E. and Menasche, Bridget L. and Ökten, Arya and Peña-Hernández, Mario A. and Klein, Jon and McNamara, Andrew and Rosales, Romel and McGovern, Briana L. and Luis Rodriguez, M. and García-Sastre, Adolfo and White, Kris M. and Qin, Yiren and Doench, John G. and Yan, Qin and Iwasaki, Akiko and Zwaka, Thomas P. and Qi, Jun and Kadoch, Cigall and Wilen, Craig B.},
  doi          = {10.1038/s41588-023-01307-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {471-483},
  shortjournal = {Nat. Genet.},
  title        = {Pharmacological disruption of mSWI/SNF complex activity restricts SARS-CoV-2 infection},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetics of sexually dimorphic adipose distribution in
humans. <em>NG</em>, <em>55</em>(3), 461–470. (<a
href="https://doi.org/10.1038/s41588-023-01306-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obesity-associated morbidity is exacerbated by abdominal obesity, which can be measured as the waist-to-hip ratio adjusted for the body mass index (WHRadjBMI). Here we identify genes associated with obesity and WHRadjBMI and characterize allele-sensitive enhancers that are predicted to regulate WHRadjBMI genes in women. We found that several waist-to-hip ratio-associated variants map within primate-specific Alu retrotransposons harboring a DNA motif associated with adipocyte differentiation. This suggests that a genetic component of adipose distribution in humans may involve co-option of retrotransposons as adipose enhancers. We evaluated the role of the strongest female WHRadjBMI-associated gene, SNX10, in adipose biology. We determined that it is required for human adipocyte differentiation and function and participates in diet-induced adipose expansion in female mice, but not males. Our data identify genes and regulatory mechanisms that underlie female-specific adipose distribution and mediate metabolic dysfunction in women. The identification of genes and enhancers associated with waist-to-hip ratio adjusted for body mass index (WHRadjBMI) in women highlights variants within Alu retrotransposons. The WHRadjBMI-linked gene SNX10 plays a role in human adipocyte differentiation and diet-induced adipose expansion in female mice.},
  archive      = {J_NG},
  author       = {Hansen, Grace T. and Sobreira, Débora R. and Weber, Zachary T. and Thornburg, Alexis G. and Aneas, Ivy and Zhang, Li and Sakabe, Noboru J. and Joslin, Amelia C. and Haddad, Gabriela A. and Strobel, Sophie M. and Laber, Samantha and Sultana, Farhath and Sahebdel, Faezeh and Khan, Kohinoor and Li, Yang I. and Claussnitzer, Melina and Ye, Liang and Battaglino, Ricardo A. and Nóbrega, Marcelo A.},
  doi          = {10.1038/s41588-023-01306-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {461-470},
  shortjournal = {Nat. Genet.},
  title        = {Genetics of sexually dimorphic adipose distribution in humans},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Immune selection determines tumor antigenicity and
influences response to checkpoint inhibitors. <em>NG</em>,
<em>55</em>(3), 451–460. (<a
href="https://doi.org/10.1038/s41588-023-01313-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In cancer, evolutionary forces select for clones that evade the immune system. Here we analyzed &amp;gt;10,000 primary tumors and 356 immune-checkpoint-treated metastases using immune dN/dS, the ratio of nonsynonymous to synonymous mutations in the immunopeptidome, to measure immune selection in cohorts and individuals. We classified tumors as immune edited when antigenic mutations were removed by negative selection and immune escaped when antigenicity was covered up by aberrant immune modulation. Only in immune-edited tumors was immune predation linked to CD8 T cell infiltration. Immune-escaped metastases experienced the best response to immunotherapy, whereas immune-edited patients did not benefit, suggesting a preexisting resistance mechanism. Similarly, in a longitudinal cohort, nivolumab treatment removes neoantigens exclusively in the immunopeptidome of nonimmune-edited patients, the group with the best overall survival response. Our work uses dN/dS to differentiate between immune-edited and immune-escaped tumors, measuring potential antigenicity and ultimately helping predict response to treatment. Immune dN/dS is the ratio of nonsynonymous to synonymous mutations in the MHC-bound immunopeptidome. Application of immune dN/dS to cancer datasets shows that it distinguishes immune evasion and escape and predicts response to immunotherapies.},
  archive      = {J_NG},
  author       = {Zapata, Luis and Caravagna, Giulio and Williams, Marc J. and Lakatos, Eszter and AbdulJabbar, Khalid and Werner, Benjamin and Chowell, Diego and James, Chela and Gourmet, Lucie and Milite, Salvatore and Acar, Ahmet and Riaz, Nadeem and Chan, Timothy A. and Graham, Trevor A. and Sottoriva, Andrea},
  doi          = {10.1038/s41588-023-01313-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {451-460},
  shortjournal = {Nat. Genet.},
  title        = {Immune selection determines tumor antigenicity and influences response to checkpoint inhibitors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiomic analysis of homologous recombination-deficient
end-stage high-grade serous ovarian cancer. <em>NG</em>, <em>55</em>(3),
437–450. (<a href="https://doi.org/10.1038/s41588-023-01320-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-grade serous ovarian cancer (HGSC) is frequently characterized by homologous recombination (HR) DNA repair deficiency and, while most such tumors are sensitive to initial treatment, acquired resistance is common. We undertook a multiomics approach to interrogate molecular diversity in end-stage disease, using multiple autopsy samples collected from 15 women with HR-deficient HGSC. Patients had polyclonal disease, and several resistance mechanisms were identified within most patients, including reversion mutations and HR restoration by other means. We also observed frequent whole-genome duplication and global changes in immune composition with evidence of immune escape. This analysis highlights diverse evolutionary changes within HGSC that evade therapy and ultimately overwhelm individual patients. Multiple samples collected at autopsy from women with homologous recombination-deficient high-grade serous ovarian cancer highlight substantial inter- and intratumoral heterogeneity. A plethora of resistance mechanisms were identified within and between patients.},
  archive      = {J_NG},
  author       = {Burdett, Nikki L. and Willis, Madelynne O. and Alsop, Kathryn and Hunt, Allison L. and Pandey, Ahwan and Hamilton, Phineas T. and Abulez, Tamara and Liu, Xuan and Hoang, Therese and Craig, Stuart and Fereday, Sian and Hendley, Joy and Garsed, Dale W. and Milne, Katy and Kalaria, Shreena and Marshall, Ashley and Hood, Brian L. and Wilson, Katlin N. and Conrads, Kelly A. and Pishas, Kathleen I. and Ananda, Sumitra and Scott, Clare L. and Antill, Yoland and McNally, Orla and Mileshkin, Linda and Hamilton, Anne and Au-Yeung, George and Devereux, Lisa and Thorne, Heather and Bild, Andrea and Bateman, Nicholas W. and Maxwell, G. Larry and Chang, Jeffrey T. and Conrads, Thomas P. and Nelson, Brad H. and Bowtell, David D. L. and Christie, Elizabeth L.},
  doi          = {10.1038/s41588-023-01320-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {437-450},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic analysis of homologous recombination-deficient end-stage high-grade serous ovarian cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The genetic basis of endometriosis and comorbidity with
other pain and inflammatory conditions. <em>NG</em>, <em>55</em>(3),
423–436. (<a href="https://doi.org/10.1038/s41588-023-01323-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Endometriosis is a common condition associated with debilitating pelvic pain and infertility. A genome-wide association study meta-analysis, including 60,674 cases and 701,926 controls of European and East Asian descent, identified 42 genome-wide significant loci comprising 49 distinct association signals. Effect sizes were largest for stage 3/4 disease, driven by ovarian endometriosis. Identified signals explained up to 5.01% of disease variance and regulated expression or methylation of genes in endometrium and blood, many of which were associated with pain perception/maintenance (SRP14/BMF, GDAP1, MLLT10, BSN and NGF). We observed significant genetic correlations between endometriosis and 11 pain conditions, including migraine, back and multisite chronic pain (MCP), as well as inflammatory conditions, including asthma and osteoarthritis. Multitrait genetic analyses identified substantial sharing of variants associated with endometriosis and MCP/migraine. Targeted investigations of genetically regulated mechanisms shared between endometriosis and other pain conditions are needed to aid the development of new treatments and facilitate early symptomatic intervention. Meta-analyses of genome-wide association studies for endometriosis identify 49 distinct association signals. Fine-mapping of causal variants explores functional effects across various tissues. Genetic correlations between endometriosis and other pain conditions are also highlighted.},
  archive      = {J_NG},
  author       = {Rahmioglu, Nilufer and Mortlock, Sally and Ghiasi, Marzieh and Møller, Peter L. and Stefansdottir, Lilja and Galarneau, Geneviève and Turman, Constance and Danning, Rebecca and Law, Matthew H. and Sapkota, Yadav and Christofidou, Paraskevi and Skarp, Sini and Giri, Ayush and Banasik, Karina and Krassowski, Michal and Lepamets, Maarja and Marciniak, Błażej and Nõukas, Margit and Perro, Danielle and Sliz, Eeva and Sobalska-Kwapis, Marta and Thorleifsson, Gudmar and Topbas-Selcuki, Nura F. and Vitonis, Allison and Westergaard, David and Arnadottir, Ragnheidur and Burgdorf, Kristoffer S. and Campbell, Archie and Cheuk, Cecilia S. K. and Clementi, Caterina and Cook, James and De Vivo, Immaculata and DiVasta, Amy and Dorien, O. and Donoghue, Jacqueline F. and Edwards, Todd and Fontanillas, Pierre and Fung, Jenny N. and Geirsson, Reynir T. and Girling, Jane E. and Harkki, Paivi and Harris, Holly R. and Healey, Martin and Heikinheimo, Oskari and Holdsworth-Carson, Sarah and Hostettler, Isabel C. and Houlden, Henry and Houshdaran, Sahar and Irwin, Juan C. and Jarvelin, Marjo-Riitta and Kamatani, Yoichiro and Kennedy, Stephen H. and Kepka, Ewa and Kettunen, Johannes and Kubo, Michiaki and Kulig, Bartosz and Kurra, Venla and Laivuori, Hannele and Laufer, Marc R. and Lindgren, Cecilia M. and MacGregor, Stuart and Mangino, Massimo and Martin, Nicholas G. and Matalliotaki, Charoula and Matalliotakis, Michail and Murray, Alison D. and Ndungu, Anne and Nezhat, Camran and Olsen, Catherine M. and Opoku-Anane, Jessica and Padmanabhan, Sandosh and Paranjpe, Manish and Peters, Maire and Polak, Grzegorz and Porteous, David J. and Rabban, Joseph and Rexrode, Kathyrn M. and Romanowicz, Hanna and Saare, Merli and Saavalainen, Liisu and Schork, Andrew J. and Sen, Sushmita and Shafrir, Amy L. and Siewierska-Górska, Anna and Słomka, Marcin and Smith, Blair H. and Smolarz, Beata and Szaflik, Tomasz and Szyłło, Krzysztof and Takahashi, Atsushi and Terry, Kathryn L. and Tomassetti, Carla and Treloar, Susan A. and Vanhie, Arne and Vincent, Katy and Vo, Kim C. and Werring, David J. and Zeggini, Eleftheria and Zervou, Maria I. and Adachi, Sosuke and Buring, Julie E. and Ridker, Paul M. and D’Hooghe, Thomas and Goulielmos, George N. and Hapangama, Dharani K. and Hayward, Caroline and Horne, Andrew W. and Low, Siew-Kee and Martikainen, Hannu and Chasman, Daniel I. and Rogers, Peter A. W. and Saunders, Philippa T. and Sirota, Marina and Spector, Tim and Strapagiel, Dominik and Tung, Joyce Y. and Whiteman, David C. and Giudice, Linda C. and Velez-Edwards, Digna R. and Uimari, Outi and Kraft, Peter and Salumets, Andres and Nyholt, Dale R. and Mägi, Reedik and Stefansson, Kari and Becker, Christian M. and Yurttas-Beim, Piraye and Steinthorsdottir, Valgerdur and Nyegaard, Mette and Missmer, Stacey A. and Montgomery, Grant W. and Morris, Andrew P. and Zondervan, Krina T.},
  doi          = {10.1038/s41588-023-01323-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {423-436},
  shortjournal = {Nat. Genet.},
  title        = {The genetic basis of endometriosis and comorbidity with other pain and inflammatory conditions},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Multi-ancestry genome-wide association analyses improve
resolution of genes and pathways influencing lung function and chronic
obstructive pulmonary disease risk. <em>NG</em>, <em>55</em>(3),
410–422. (<a href="https://doi.org/10.1038/s41588-023-01314-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lung-function impairment underlies chronic obstructive pulmonary disease (COPD) and predicts mortality. In the largest multi-ancestry genome-wide association meta-analysis of lung function to date, comprising 588,452 participants, we identified 1,020 independent association signals implicating 559 genes supported by ≥2 criteria from a systematic variant-to-gene mapping framework. These genes were enriched in 29 pathways. Individual variants showed heterogeneity across ancestries, age and smoking groups, and collectively as a genetic risk score showed strong association with COPD across ancestry groups. We undertook phenome-wide association studies for selected associated variants as well as trait and pathway-specific genetic risk scores to infer possible consequences of intervening in pathways underlying lung function. We highlight new putative causal variants, genes, proteins and pathways, including those targeted by existing drugs. These findings bring us closer to understanding the mechanisms underlying lung function and COPD, and should inform functional genomics experiments and potentially future COPD therapies. Multi-ancestry genome-wide association analyses and systematic variant-to-gene mapping strategies implicate new genes and pathways influencing lung function and chronic obstructive pulmonary disease risk.},
  archive      = {J_NG},
  author       = {Shrine, Nick and Izquierdo, Abril G. and Chen, Jing and Packer, Richard and Hall, Robert J. and Guyatt, Anna L. and Batini, Chiara and Thompson, Rebecca J. and Pavuluri, Chandan and Malik, Vidhi and Hobbs, Brian D. and Moll, Matthew and Kim, Wonji and Tal-Singer, Ruth and Bakke, Per and Fawcett, Katherine A. and John, Catherine and Coley, Kayesha and Piga, Noemi Nicole and Pozarickij, Alfred and Lin, Kuang and Millwood, Iona Y. and Chen, Zhengming and Li, Liming and Wijnant, Sara R. A. and Lahousse, Lies and Brusselle, Guy and Uitterlinden, Andre G. and Manichaikul, Ani and Oelsner, Elizabeth C. and Rich, Stephen S. and Barr, R. Graham and Kerr, Shona M. and Vitart, Veronique and Brown, Michael R. and Wielscher, Matthias and Imboden, Medea and Jeong, Ayoung and Bartz, Traci M. and Gharib, Sina A. and Flexeder, Claudia and Karrasch, Stefan and Gieger, Christian and Peters, Annette and Stubbe, Beate and Hu, Xiaowei and Ortega, Victor E. and Meyers, Deborah A. and Bleecker, Eugene R. and Gabriel, Stacey B. and Gupta, Namrata and Smith, Albert Vernon and Luan, Jian’an and Zhao, Jing-Hua and Hansen, Ailin F. and Langhammer, Arnulf and Willer, Cristen and Bhatta, Laxmi and Porteous, David and Smith, Blair H. and Campbell, Archie and Sofer, Tamar and Lee, Jiwon and Daviglus, Martha L. and Yu, Bing and Lim, Elise and Xu, Hanfei and O’Connor, George T. and Thareja, Gaurav and Albagha, Omar M. E. and Suhre, Karsten and Granell, Raquel and Faquih, Tariq O. and Hiemstra, Pieter S. and Slats, Annelies M. and Mullin, Benjamin H. and Hui, Jennie and James, Alan and Beilby, John and Patasova, Karina and Hysi, Pirro and Koskela, Jukka T. and Wyss, Annah B. and Jin, Jianping and Sikdar, Sinjini and Lee, Mikyeong and May-Wilson, Sebastian and Pirastu, Nicola and Kentistou, Katherine A. and Joshi, Peter K. and Timmers, Paul R. H. J. and Williams, Alexander T. and Free, Robert C. and Wang, Xueyang and Morrison, John L. and Gilliland, Frank D. and Chen, Zhanghua and Wang, Carol A. and Foong, Rachel E. and Harris, Sarah E. and Taylor, Adele and Redmond, Paul and Cook, James P. and Mahajan, Anubha and Lind, Lars and Palviainen, Teemu and Lehtimäki, Terho and Raitakari, Olli T. and Kaprio, Jaakko and Rantanen, Taina and Pietiläinen, Kirsi H. and Cox, Simon R. and Pennell, Craig E. and Hall, Graham L. and Gauderman, W. James and Brightling, Chris and Wilson, James F. and Vasankari, Tuula and Laitinen, Tarja and Salomaa, Veikko and Mook-Kanamori, Dennis O. and Timpson, Nicholas J. and Zeggini, Eleftheria and Dupuis, Josée and Hayward, Caroline and Brumpton, Ben and Langenberg, Claudia and Weiss, Stefan and Homuth, Georg and Schmidt, Carsten Oliver and Probst-Hensch, Nicole and Jarvelin, Marjo-Riitta and Morrison, Alanna C. and Polasek, Ozren and Rudan, Igor and Lee, Joo-Hyeon and Sayers, Ian and Rawlins, Emma L. and Dudbridge, Frank and Silverman, Edwin K. and Strachan, David P. and Walters, Robin G. and Morris, Andrew P. and London, Stephanie J. and Cho, Michael H. and Wain, Louise V. and Hall, Ian P. and Tobin, Martin D.},
  doi          = {10.1038/s41588-023-01314-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {410-422},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genome-wide association analyses improve resolution of genes and pathways influencing lung function and chronic obstructive pulmonary disease risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide meta-analysis identifies 93 risk loci and
enables risk prediction equivalent to monogenic forms of venous
thromboembolism. <em>NG</em>, <em>55</em>(3), 399–409. (<a
href="https://doi.org/10.1038/s41588-022-01286-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report a genome-wide association study of venous thromboembolism (VTE) incorporating 81,190 cases and 1,419,671 controls sampled from six cohorts. We identify 93 risk loci, of which 62 are previously unreported. Many of the identified risk loci are at genes encoding proteins with functions converging on the coagulation cascade or platelet function. A VTE polygenic risk score (PRS) enabled effective identification of both high- and low-risk individuals. Individuals within the top 0.1% of PRS distribution had a VTE risk similar to homozygous or compound heterozygous carriers of the variants G20210A (c.*97 G &amp;gt; A) in F2 and p.R534Q in F5. We also document that F2 and F5 mutation carriers in the bottom 10% of the PRS distribution had a risk similar to that of the general population. We further show that PRS improved individual risk prediction beyond that of genetic and clinical risk factors. We investigated the extent to which venous and arterial thrombosis share clinical risk factors using Mendelian randomization, finding that some risk factors for arterial thrombosis were directionally concordant with VTE risk (for example, body mass index and smoking) whereas others were discordant (for example, systolic blood pressure and triglyceride levels). Genome-wide association analyses identify 93 risk loci for venous thromboembolism (VTE). A polygenic score derived from these results identifies individuals at increased VTE risk equivalent to monogenic forms of the disease.},
  archive      = {J_NG},
  author       = {Ghouse, Jonas and Tragante, Vinicius and Ahlberg, Gustav and Rand, Søren A. and Jespersen, Jakob B. and Leinøe, Eva Birgitte and Vissing, Christoffer Rasmus and Trudsø, Linea and Jonsdottir, Ingileif and Banasik, Karina and Brunak, Søren and Ostrowski, Sisse R. and Pedersen, Ole B. and Sørensen, Erik and Erikstrup, Christian and Bruun, Mie Topholm and Nielsen, Kaspar Rene and Køber, Lars and Christensen, Alex H. and Iversen, Kasper and Jones, David and Knowlton, Kirk U. and Nadauld, Lincoln and Halldorsson, Gisli H. and Ferkingstad, Egil and Olafsson, Isleifur and Gretarsdottir, Solveig and Onundarson, Pall T. and Sulem, Patrick and Thorsteinsdottir, Unnur and Thorgeirsson, Gudmundur and Gudbjartsson, Daniel F. and Stefansson, Kari and Holm, Hilma and Olesen, Morten Salling and Bundgaard, Henning},
  doi          = {10.1038/s41588-022-01286-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {399-409},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide meta-analysis identifies 93 risk loci and enables risk prediction equivalent to monogenic forms of venous thromboembolism},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Network expansion of genetic associations defines a
pleiotropy map of human cell biology. <em>NG</em>, <em>55</em>(3),
389–398. (<a href="https://doi.org/10.1038/s41588-023-01327-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interacting proteins tend to have similar functions, influencing the same organismal traits. Interaction networks can be used to expand the list of candidate trait-associated genes from genome-wide association studies. Here, we performed network-based expansion of trait-associated genes for 1,002 human traits showing that this recovers known disease genes or drug targets. The similarity of network expansion scores identifies groups of traits likely to share an underlying genetic and biological process. We identified 73 pleiotropic gene modules linked to multiple traits, enriched in genes involved in processes such as protein ubiquitination and RNA processing. In contrast to gene deletion studies, pleiotropy as defined here captures specifically multicellular-related processes. We show examples of modules linked to human diseases enriched in genes with known pathogenic variants that can be used to map targets of approved drugs for repurposing. Finally, we illustrate the use of network expansion scores to study genes at inflammatory bowel disease genome-wide association study loci, and implicate inflammatory bowel disease-relevant genes with strong functional and genetic support. Network expansion of trait-associated genes for 1,002 human phenotypes identifies pleiotropic gene modules and biological processes associated with multiple traits, and potential opportunities for drug repurposing and development.},
  archive      = {J_NG},
  author       = {Barrio-Hernandez, Inigo and Schwartzentruber, Jeremy and Shrivastava, Anjali and del-Toro, Noemi and Gonzalez, Asier and Zhang, Qian and Mountjoy, Edward and Suveges, Daniel and Ochoa, David and Ghoussaini, Maya and Bradley, Glyn and Hermjakob, Henning and Orchard, Sandra and Dunham, Ian and Anderson, Carl A. and Porras, Pablo and Beltrao, Pedro},
  doi          = {10.1038/s41588-023-01327-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {389-398},
  shortjournal = {Nat. Genet.},
  title        = {Network expansion of genetic associations defines a pleiotropy map of human cell biology},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Brain expression quantitative trait locus and network
analyses reveal downstream effects and putative drivers for
brain-related diseases. <em>NG</em>, <em>55</em>(3), 377–388. (<a
href="https://doi.org/10.1038/s41588-023-01300-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of therapeutic targets from genome-wide association studies (GWAS) requires insights into downstream functional consequences. We harmonized 8,613 RNA-sequencing samples from 14 brain datasets to create the MetaBrain resource and performed cis- and trans-expression quantitative trait locus (eQTL) meta-analyses in multiple brain region- and ancestry-specific datasets (n ≤ 2,759). Many of the 16,169 cortex cis-eQTLs were tissue-dependent when compared with blood cis-eQTLs. We inferred brain cell types for 3,549 cis-eQTLs by interaction analysis. We prioritized 186 cis-eQTLs for 31 brain-related traits using Mendelian randomization and co-localization including 40 cis-eQTLs with an inferred cell type, such as a neuron-specific cis-eQTL (CYP24A1) for multiple sclerosis. We further describe 737 trans-eQTLs for 526 unique variants and 108 unique genes. We used brain-specific gene-co-regulation networks to link GWAS loci and prioritize additional genes for five central nervous system diseases. This study represents a valuable resource for post-GWAS research on central nervous system diseases. Brain expression quantitative trait locus (eQTL) and network analyses encompassing 8,613 RNA-sequencing samples from 14 eQTL datasets reveal downstream effects and putative drivers for brain-related diseases.},
  archive      = {J_NG},
  author       = {de Klein, Niek and Tsai, Ellen A. and Vochteloo, Martijn and Baird, Denis and Huang, Yunfeng and Chen, Chia-Yen and van Dam, Sipko and Oelen, Roy and Deelen, Patrick and Bakker, Olivier B. and El Garwany, Omar and Ouyang, Zhengyu and Marshall, Eric E. and Zavodszky, Maria I. and van Rheenen, Wouter and Bakker, Mark K. and Veldink, Jan and Gaunt, Tom R. and Runz, Heiko and Franke, Lude and Westra, Harm-Jan},
  doi          = {10.1038/s41588-023-01300-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {377-388},
  shortjournal = {Nat. Genet.},
  title        = {Brain expression quantitative trait locus and network analyses reveal downstream effects and putative drivers for brain-related diseases},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Schizophrenia risk conferred by rare protein-truncating
variants is conserved across diverse human populations. <em>NG</em>,
<em>55</em>(3), 369–376. (<a
href="https://doi.org/10.1038/s41588-023-01305-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Schizophrenia (SCZ) is a chronic mental illness and among the most debilitating conditions encountered in medical practice. A recent landmark SCZ study of the protein-coding regions of the genome identified a causal role for ten genes and a concentration of rare variant signals in evolutionarily constrained genes1. This recent study—and most other large-scale human genetics studies—was mainly composed of individuals of European (EUR) ancestry, and the generalizability of the findings in non-EUR populations remains unclear. To address this gap, we designed a custom sequencing panel of 161 genes selected based on the current knowledge of SCZ genetics and sequenced a new cohort of 11,580 SCZ cases and 10,555 controls of diverse ancestries. Replicating earlier work, we found that cases carried a significantly higher burden of rare protein-truncating variants (PTVs) among evolutionarily constrained genes (odds ratio = 1.48; P = 5.4 × 10−6). In meta-analyses with existing datasets totaling up to 35,828 cases and 107,877 controls, this excess burden was largely consistent across five ancestral populations. Two genes (SRRM2 and AKAP11) were newly implicated as SCZ risk genes, and one gene (PCLO) was identified as shared by individuals with SCZ and those with autism. Overall, our results lend robust support to the rare allelic spectrum of the genetic architecture of SCZ being conserved across diverse human populations. Targeted sequencing finds a higher burden of rare protein-truncating variants in constrained genes among schizophrenia cases of diverse ancestries. Meta-analyses with existing datasets show that this excess burden is consistent across five ancestral populations.},
  archive      = {J_NG},
  author       = {Liu, Dongjing and Meyer, Dara and Fennessy, Brian and Feng, Claudia and Cheng, Esther and Johnson, Jessica S. and Park, You Jeong and Rieder, Marysia-Kolbe and Ascolillo, Steven and de Pins, Agathe and Dobbyn, Amanda and Lebovitch, Dannielle and Moya, Emily and Nguyen, Tan-Hoang and Wilkins, Lillian and Hassan, Arsalan and Burdick, Katherine E. and Buxbaum, Joseph D. and Domenici, Enrico and Frangou, Sophia and Hartmann, Annette M. and Laurent-Levinson, Claudine and Malhotra, Dheeraj and Pato, Carlos N. and Pato, Michele T. and Ressler, Kerry and Roussos, Panos and Rujescu, Dan and Arango, Celso and Bertolino, Alessandro and Blasi, Giuseppe and Bocchio-Chiavetto, Luisella and Campion, Dominique and Carr, Vaughan and Fullerton, Janice M. and Gennarelli, Massimo and González-Peñas, Javier and Levinson, Douglas F. and Mowry, Bryan and Nimgaokar, Vishwajit L. and Pergola, Giulio and Rampino, Antonio and Cervilla, Jorge A. and Rivera, Margarita and Schwab, Sibylle G. and Wildenauer, Dieter B. and Daly, Mark and Neale, Benjamin and Singh, Tarjinder and O’Donovan, Michael C. and Owen, Michael J. and Walters, James T. and Ayub, Muhammad and Malhotra, Anil K. and Lencz, Todd and Sullivan, Patrick F. and Sklar, Pamela and Stahl, Eli A. and Huckins, Laura M. and Charney, Alexander W.},
  doi          = {10.1038/s41588-023-01305-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {369-376},
  shortjournal = {Nat. Genet.},
  title        = {Schizophrenia risk conferred by rare protein-truncating variants is conserved across diverse human populations},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mouse embryonic development requires transposable element
expression. <em>NG</em>, <em>55</em>(3), 367–368. (<a
href="https://doi.org/10.1038/s41588-023-01326-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The expression of murine endogenous retrovirus-L (MERVL) is transiently upregulated at the two-cell stage in mouse embryos, coinciding with zygotic genome activation and the acquisition of totipotency; however, its role in embryogenesis remains elusive. We show that nuclear expression of MERVL is required for accurate regulation of the host transcriptome and chromatin state during preimplantation development.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01326-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {367-368},
  shortjournal = {Nat. Genet.},
  title        = {Mouse embryonic development requires transposable element expression},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A pleiotropy map of cell biology covering 1,002 human
traits. <em>NG</em>, <em>55</em>(3), 365–366. (<a
href="https://doi.org/10.1038/s41588-023-01328-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interacting proteins commonly perform similar functions. A protein interaction network analysis of genes linked to 1,002 human traits identifies molecular mechanisms that are shared across traits and so-called pleiotropic biological processes, whose disruption might affect many aspects of human biology.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01328-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {365-366},
  shortjournal = {Nat. Genet.},
  title        = {A pleiotropy map of cell biology covering 1,002 human traits},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Meta-analysis of harmonized brain transcriptomics data
prioritizes therapeutic target genes. <em>NG</em>, <em>55</em>(3),
363–364. (<a href="https://doi.org/10.1038/s41588-023-01301-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A meta-analysis of harmonized human brain RNA-seq datasets creates expression quantitative trait locus (eQTL) maps for multiple ancestries and brain regions, predicts cell-type-dependent eQTLs and produces gene networks. This prioritizes genes for multiple brain-related diseases, serving as a promising step toward the identification of central nervous system (CNS) drug targets.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01301-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {363-364},
  shortjournal = {Nat. Genet.},
  title        = {Meta-analysis of harmonized brain transcriptomics data prioritizes therapeutic target genes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A medley of resistance in ovarian cancers. <em>NG</em>,
<em>55</em>(3), 361–362. (<a
href="https://doi.org/10.1038/s41588-023-01303-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multi-omic profiling of lesions at autopsy reveals a plethora of resistance mechanisms present within individual patients with ovarian cancer. This highlights the extreme challenge faced in treating end-stage disease and underscores the need for new methods of early detection and intervention.},
  archive      = {J_NG},
  author       = {Hernando, Barbara and Macintyre, Geoff},
  doi          = {10.1038/s41588-023-01303-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {361-362},
  shortjournal = {Nat. Genet.},
  title        = {A medley of resistance in ovarian cancers},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identifying individuals at extreme risk of venous
thromboembolism using polygenic risk scores. <em>NG</em>,
<em>55</em>(3), 358–360. (<a
href="https://doi.org/10.1038/s41588-023-01330-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current risk assessment and treatment strategies for venous thromboembolism (VTE) consider genetic factors only in a limited way. New work shows a more pervasive role of common variants in VTE risk, inspiring genetic predictors that surpass and complement individual clinical risk factors and monogenic thrombophilia testing.},
  archive      = {J_NG},
  author       = {Chong, Michael and Paré, Guillaume},
  doi          = {10.1038/s41588-023-01330-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {358-360},
  shortjournal = {Nat. Genet.},
  title        = {Identifying individuals at extreme risk of venous thromboembolism using polygenic risk scores},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Genome-scale characterization of transcription factors.
<em>NG</em>, <em>55</em>(3), 357. (<a
href="https://doi.org/10.1038/s41588-023-01351-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01351-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {357},
  shortjournal = {Nat. Genet.},
  title        = {Genome-scale characterization of transcription factors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Sequencing genetic and epigenetic bases. <em>NG</em>,
<em>55</em>(3), 357. (<a
href="https://doi.org/10.1038/s41588-023-01352-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-023-01352-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {357},
  shortjournal = {Nat. Genet.},
  title        = {Sequencing genetic and epigenetic bases},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Engineering epigenetic inheritance. <em>NG</em>,
<em>55</em>(3), 357. (<a
href="https://doi.org/10.1038/s41588-023-01353-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01353-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {357},
  shortjournal = {Nat. Genet.},
  title        = {Engineering epigenetic inheritance},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Rare-variant genetic architecture. <em>NG</em>,
<em>55</em>(3), 327. (<a
href="https://doi.org/10.1038/s41588-023-01354-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01354-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {327},
  shortjournal = {Nat. Genet.},
  title        = {Rare-variant genetic architecture},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Tumor aneuploidy predicts survival
following immunotherapy across multiple cancers. <em>NG</em>,
<em>55</em>(2), 355. (<a
href="https://doi.org/10.1038/s41588-022-01288-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Spurr, Liam F. and Weichselbaum, Ralph R. and Pitroda, Sean P.},
  doi          = {10.1038/s41588-022-01288-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {355},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Tumor aneuploidy predicts survival following immunotherapy across multiple cancers},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: PHYTOCHROME-INTERACTING FACTORs trigger
environmentally responsive chromatin dynamics in plants. <em>NG</em>,
<em>55</em>(2), 355. (<a
href="https://doi.org/10.1038/s41588-023-01299-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Willige, Björn C. and Zander, Mark and Yoo, Chan Yul and Phan, Amy and Garza, Renee M. and Wanamaker, Shelly A. and He, Yupeng and Nery, Joseph R. and Chen, Huaming and Chen, Meng and Ecker, Joseph R. and Chory, Joanne},
  doi          = {10.1038/s41588-023-01299-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {355},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: PHYTOCHROME-INTERACTING FACTORs trigger environmentally responsive chromatin dynamics in plants},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: De novo genome assembly and analyses of
12 founder inbred lines provide insights into maize heterosis.
<em>NG</em>, <em>55</em>(2), 355. (<a
href="https://doi.org/10.1038/s41588-023-01308-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Wang, Baobao and Hou, Mei and Shi, Junpeng and Ku, Lixia and Song, Wei and Li, Chunhui and Ning, Qiang and Li, Xin and Li, Changyu and Zhao, Binbin and Zhang, Ruyang and Xu, Hua and Bai, Zhijing and Xia, Zhanchao and Wang, Hai and Kong, Dexin and Wei, Hongbin and Jing, Yifeng and Dai, Zhouyan and Wang, Hu Hailing and Zhu, Xinyu and Sun, Xuan and Wang, Shuaishuai and Yao, Wen and Hou, Gege and Qi, Zhi and Dai, He and Li, Xuming and Zheng, Hongkun and Zhang, Zuxin and Li, Yu and Wang, Tianyu and Jiang, Taijiao and Wan, Zhaoman and Chen, Yanhui and Zhao, Jiuran and Lai, Jinsheng and Wang, Haiyang},
  doi          = {10.1038/s41588-023-01308-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {355},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: De novo genome assembly and analyses of 12 founder inbred lines provide insights into maize heterosis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A single-cell massively parallel reporter assay detects
cell-type-specific gene regulation. <em>NG</em>, <em>55</em>(2),
346–354. (<a href="https://doi.org/10.1038/s41588-022-01278-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Massively parallel reporter gene assays are key tools in regulatory genomics but cannot be used to identify cell-type-specific regulatory elements without performing assays serially across different cell types. To address this problem, we developed a single-cell massively parallel reporter assay (scMPRA) to measure the activity of libraries of cis-regulatory sequences (CRSs) across multiple cell types simultaneously. We assayed a library of core promoters in a mixture of HEK293 and K562 cells and showed that scMPRA is a reproducible, highly parallel, single-cell reporter gene assay that detects cell-type-specific cis-regulatory activity. We then measured a library of promoter variants across multiple cell types in live mouse retinas and showed that subtle genetic variants can produce cell-type-specific effects on cis-regulatory activity. We anticipate that scMPRA will be widely applicable for studying the role of CRSs across diverse cell types. A single-cell massively parallel reporter assay is used to compare cis-regulatory sequence activities in cell line models and mouse retinal tissue ex vivo, identifying cell state- and cell-type-specific effects of sequence variation.},
  archive      = {J_NG},
  author       = {Zhao, Siqi and Hong, Clarice K. Y. and Myers, Connie A. and Granas, David M. and White, Michael A. and Corbo, Joseph C. and Cohen, Barak A.},
  doi          = {10.1038/s41588-022-01278-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {346-354},
  shortjournal = {Nat. Genet.},
  title        = {A single-cell massively parallel reporter assay detects cell-type-specific gene regulation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell sortChIC identifies hierarchical chromatin
dynamics during hematopoiesis. <em>NG</em>, <em>55</em>(2), 333–345. (<a
href="https://doi.org/10.1038/s41588-022-01260-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Post-translational histone modifications modulate chromatin activity to affect gene expression. How chromatin states underlie lineage choice in single cells is relatively unexplored. We develop sort-assisted single-cell chromatin immunocleavage (sortChIC) and map active (H3K4me1 and H3K4me3) and repressive (H3K27me3 and H3K9me3) histone modifications in the mouse bone marrow. During differentiation, hematopoietic stem and progenitor cells (HSPCs) acquire active chromatin states mediated by cell-type-specifying transcription factors, which are unique for each lineage. By contrast, most alterations in repressive marks during differentiation occur independent of the final cell type. Chromatin trajectory analysis shows that lineage choice at the chromatin level occurs at the progenitor stage. Joint profiling of H3K4me1 and H3K9me3 demonstrates that cell types within the myeloid lineage have distinct active chromatin but share similar myeloid-specific heterochromatin states. This implies a hierarchical regulation of chromatin during hematopoiesis: heterochromatin dynamics distinguish differentiation trajectories and lineages, while euchromatin dynamics reflect cell types within lineages. Sort-assisted single-cell chromatin immunocleavage (sortChIC) combines single-cell histone modification profiling with fluorescence-activated cell sorting (FACS), enabling the study of rare cell populations. H3K4me1/H3K4me3, H3K9me3 and H3K27me3 profiling of blood suggest a model of lineage-shared repressive and cell type-specific active chromatin.},
  archive      = {J_NG},
  author       = {Zeller, Peter and Yeung, Jake and Viñas Gaza, Helena and de Barbanson, Buys Anton and Bhardwaj, Vivek and Florescu, Maria and van der Linden, Reinier and van Oudenaarden, Alexander},
  doi          = {10.1038/s41588-022-01260-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {333-345},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell sortChIC identifies hierarchical chromatin dynamics during hematopoiesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SpliceVault predicts the precise nature of
variant-associated mis-splicing. <em>NG</em>, <em>55</em>(2), 324–332.
(<a href="https://doi.org/10.1038/s41588-022-01293-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Even for essential splice-site variants that are almost guaranteed to alter mRNA splicing, no current method can reliably predict whether exon-skipping, cryptic activation or multiple events will result, greatly complicating clinical interpretation of pathogenicity. Strikingly, ranking the four most common unannotated splicing events across 335,663 reference RNA-sequencing (RNA-seq) samples (300K-RNA Top-4) predicts the nature of variant-associated mis-splicing with 92% sensitivity. The 300K-RNA Top-4 events correctly identify 96% of exon-skipping events and 86% of cryptic splice sites for 140 clinical cases subject to RNA testing, showing higher sensitivity and positive predictive value than SpliceAI. Notably, RNA re-analyses showed we had missed 300K-RNA Top-4 events for several clinical cases tested before the development of this empirical predictive method. Simply, mis-splicing events that happen around a splice site in RNA-seq data are those most likely to be activated by a splice-site variant. The SpliceVault web portal allows users easy access to 300K-RNA for informed splice-site variant interpretation and classification. Re-analysis of published RNA-sequencing samples finds that unannotated splicing events predict, with high sensitivity, the activation of exon skipping and cryptic splicing by splice-site variants.},
  archive      = {J_NG},
  author       = {Dawes, Ruebena and Bournazos, Adam M. and Bryen, Samantha J. and Bommireddipalli, Shobhana and Marchant, Rhett G. and Joshi, Himanshu and Cooper, Sandra T.},
  doi          = {10.1038/s41588-022-01293-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {324-332},
  shortjournal = {Nat. Genet.},
  title        = {SpliceVault predicts the precise nature of variant-associated mis-splicing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). De novo genome assembly and analyses of 12 founder inbred
lines provide insights into maize heterosis. <em>NG</em>,
<em>55</em>(2), 312–323. (<a
href="https://doi.org/10.1038/s41588-022-01283-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hybrid maize displays superior heterosis and contributes over 30% of total worldwide cereal production. However, the molecular mechanisms of heterosis remain obscure. Here we show that structural variants (SVs) between the parental lines have a predominant role underpinning maize heterosis. De novo assembly and analyses of 12 maize founder inbred lines (FILs) reveal abundant genetic variations among these FILs and, through expression quantitative trait loci and association analyses, we identify several SVs contributing to genomic and phenotypic differentiations of various heterotic groups. Using a set of 91 diallel-cross F1 hybrids, we found strong positive correlations between better-parent heterosis of the F1 hybrids and the numbers of SVs between the parental lines, providing concrete genomic support for a prevalent role of genetic complementation underlying heterosis. Further, we document evidence that SVs in both ZAR1 and ZmACO2 contribute to yield heterosis in an overdominance fashion. Our results should promote genomics-based breeding of hybrid maize. De novo genome assembly and analyses of 12 maize FILs provide insights into genomic and phenotypic differentiation of various heterotic groups and the molecular basis of heterosis in maize.},
  archive      = {J_NG},
  author       = {Wang, Baobao and Hou, Mei and Shi, Junpeng and Ku, Lixia and Song, Wei and Li, Chunhui and Ning, Qiang and Li, Xin and Li, Changyu and Zhao, Binbin and Zhang, Ruyang and Xu, Hua and Bai, Zhijing and Xia, Zhanchao and Wang, Hai and Kong, Dexin and Wei, Hongbin and Jing, Yifeng and Dai, Zhouyan and Wang, Hu Hailing and Zhu, Xinyu and Sun, Xuan and Wang, Shuaishuai and Yao, Wen and Hou, Gege and Qi, Zhi and Dai, He and Li, Xuming and Zheng, Hongkun and Zhang, Zuxin and Li, Yu and Wang, Tianyu and Jiang, Taijiao and Wan, Zhaoman and Chen, Yanhui and Zhao, Jiuran and Lai, Jinsheng and Wang, Haiyang},
  doi          = {10.1038/s41588-022-01283-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {312-323},
  shortjournal = {Nat. Genet.},
  title        = {De novo genome assembly and analyses of 12 founder inbred lines provide insights into maize heterosis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A high-quality ixodes scapularis genome advances tick
science. <em>NG</em>, <em>55</em>(2), 301–311. (<a
href="https://doi.org/10.1038/s41588-022-01275-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ixodes spp. and related ticks transmit prevalent infections, although knowledge of their biology and development of anti-tick measures have been hindered by the lack of a high-quality genome. In the present study, we present the assembly of a 2.23-Gb Ixodes scapularis genome by sequencing two haplotypes within one individual, complemented by chromosome-level scaffolding and full-length RNA isoform sequencing, yielding a fully reannotated genome featuring thousands of new protein-coding genes and various RNA species. Analyses of the repetitive DNA identified transposable elements, whereas the examination of tick-associated bacterial sequences yielded an improved Rickettsia buchneri genome. We demonstrate how the Ixodes genome advances tick science by contributing to new annotations, gene models and epigenetic functions, expansion of gene families, development of in-depth proteome catalogs and deciphering of genetic variations in wild ticks. Overall, we report critical genetic resources and biological insights impacting our understanding of tick biology and future interventions against tick-transmitted infections. A high-quality Ixodes scapularis genome contributes to improved annotations, expansion of gene families, development of proteome catalogs and the deciphering of genetic variation in wild ticks.},
  archive      = {J_NG},
  author       = {De, Sandip and Kingan, Sarah B. and Kitsou, Chrysoula and Portik, Daniel M. and Foor, Shelby D. and Frederick, Julia C. and Rana, Vipin S. and Paulat, Nicole S. and Ray, David A. and Wang, Yan and Glenn, Travis C. and Pal, Utpal},
  doi          = {10.1038/s41588-022-01275-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {301-311},
  shortjournal = {Nat. Genet.},
  title        = {A high-quality ixodes scapularis genome advances tick science},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multi-ancestry transcriptome-wide association analyses yield
insights into tobacco use biology and drug repurposing. <em>NG</em>,
<em>55</em>(2), 291–300. (<a
href="https://doi.org/10.1038/s41588-022-01282-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most transcriptome-wide association studies (TWASs) so far focus on European ancestry and lack diversity. To overcome this limitation, we aggregated genome-wide association study (GWAS) summary statistics, whole-genome sequences and expression quantitative trait locus (eQTL) data from diverse ancestries. We developed a new approach, TESLA (multi-ancestry integrative study using an optimal linear combination of association statistics), to integrate an eQTL dataset with a multi-ancestry GWAS. By exploiting shared phenotypic effects between ancestries and accommodating potential effect heterogeneities, TESLA improves power over other TWAS methods. When applied to tobacco use phenotypes, TESLA identified 273 new genes, up to 55% more compared with alternative TWAS methods. These hits and subsequent fine mapping using TESLA point to target genes with biological relevance. In silico drug-repurposing analyses highlight several drugs with known efficacy, including dextromethorphan and galantamine, and new drugs such as muscle relaxants that may be repurposed for treating nicotine addiction. A multi-ancestry transcriptome-wide association study using an optimal linear combination of association statistics provides insights into tobacco use biology and suggests opportunities for drug repurposing.},
  archive      = {J_NG},
  author       = {Chen, Fang and Wang, Xingyan and Jang, Seon-Kyeong and Quach, Bryan C. and Weissenkampen, J. Dylan and Khunsriraksakul, Chachrit and Yang, Lina and Sauteraud, Renan and Albert, Christine M. and Allred, Nicholette D. D. and Arnett, Donna K. and Ashley-Koch, Allison E. and Barnes, Kathleen C. and Barr, R. Graham and Becker, Diane M. and Bielak, Lawrence F. and Bis, Joshua C. and Blangero, John and Boorgula, Meher Preethi and Chasman, Daniel I. and Chavan, Sameer and Chen, Yii-Der I. and Chuang, Lee-Ming and Correa, Adolfo and Curran, Joanne E. and David, Sean P. and Fuentes, Lisa de las and Deka, Ranjan and Duggirala, Ravindranath and Faul, Jessica D. and Garrett, Melanie E. and Gharib, Sina A. and Guo, Xiuqing and Hall, Michael E. and Hawley, Nicola L. and He, Jiang and Hobbs, Brian D. and Hokanson, John E. and Hsiung, Chao A. and Hwang, Shih-Jen and Hyde, Thomas M. and Irvin, Marguerite R. and Jaffe, Andrew E. and Johnson, Eric O. and Kaplan, Robert and Kardia, Sharon L. R. and Kaufman, Joel D. and Kelly, Tanika N. and Kleinman, Joel E. and Kooperberg, Charles and Lee, I-Te and Levy, Daniel and Lutz, Sharon M. and Manichaikul, Ani W. and Martin, Lisa W. and Marx, Olivia and McGarvey, Stephen T. and Minster, Ryan L. and Moll, Matthew and Moussa, Karine A. and Naseri, Take and North, Kari E. and Oelsner, Elizabeth C. and Peralta, Juan M. and Peyser, Patricia A. and Psaty, Bruce M. and Rafaels, Nicholas and Raffield, Laura M. and Reupena, Muagututi’a Sefuiva and Rich, Stephen S. and Rotter, Jerome I. and Schwartz, David A. and Shadyab, Aladdin H. and Sheu, Wayne H-H. and Sims, Mario and Smith, Jennifer A. and Sun, Xiao and Taylor, Kent D. and Telen, Marilyn J. and Watson, Harold and Weeks, Daniel E. and Weir, David R. and Yanek, Lisa R. and Young, Kendra A. and Young, Kristin L. and Zhao, Wei and Hancock, Dana B. and Jiang, Bibo and Vrieze, Scott and Liu, Dajiang J.},
  doi          = {10.1038/s41588-022-01282-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {291-300},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry transcriptome-wide association analyses yield insights into tobacco use biology and drug repurposing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Enhancer–promoter interactions can bypass CTCF-mediated
boundaries and contribute to phenotypic robustness. <em>NG</em>,
<em>55</em>(2), 280–290. (<a
href="https://doi.org/10.1038/s41588-022-01295-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How enhancers activate their distal target promoters remains incompletely understood. Here we dissect how CTCF-mediated loops facilitate and restrict such regulatory interactions. Using an allelic series of mouse mutants, we show that CTCF is neither required for the interaction of the Sox2 gene with distal enhancers, nor for its expression. Insertion of various combinations of CTCF motifs, between Sox2 and its distal enhancers, generated boundaries with varying degrees of insulation that directly correlated with reduced transcriptional output. However, in both epiblast and neural tissues, enhancer contacts and transcriptional induction could not be fully abolished, and insertions failed to disrupt implantation and neurogenesis. In contrast, Sox2 expression was undetectable in the anterior foregut of mutants carrying the strongest boundaries, and these animals fully phenocopied loss of SOX2 in this tissue. We propose that enhancer clusters with a high density of regulatory activity can better overcome physical barriers to maintain faithful gene expression and phenotypic robustness. Genetic manipulation of the Sox2 locus in mice shows that gene activation by distal enhancers does not require CTCF-mediated loops and can occur across ectopic CTCF-mediated boundaries. The ability to bypass CTCF boundaries varies with their insulation strength and the tissue-specific enhancers responsible for activation.},
  archive      = {J_NG},
  author       = {Chakraborty, Shreeta and Kopitchinski, Nina and Zuo, Zhenyu and Eraso, Ariel and Awasthi, Parirokh and Chari, Raj and Mitra, Apratim and Tobias, Ian C. and Moorthy, Sakthi D. and Dale, Ryan K. and Mitchell, Jennifer A. and Petros, Timothy J. and Rocha, Pedro P.},
  doi          = {10.1038/s41588-022-01295-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280-290},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer–promoter interactions can bypass CTCF-mediated boundaries and contribute to phenotypic robustness},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide RNA polymerase stalling shapes the transcriptome
during aging. <em>NG</em>, <em>55</em>(2), 268–279. (<a
href="https://doi.org/10.1038/s41588-022-01279-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression profiling has identified numerous processes altered in aging, but how these changes arise is largely unknown. Here we combined nascent RNA sequencing and RNA polymerase II chromatin immunoprecipitation followed by sequencing to elucidate the underlying mechanisms triggering gene expression changes in wild-type aged mice. We found that in 2-year-old liver, 40% of elongating RNA polymerases are stalled, lowering productive transcription and skewing transcriptional output in a gene-length-dependent fashion. We demonstrate that this transcriptional stress is caused by endogenous DNA damage and explains the majority of gene expression changes in aging in most mainly postmitotic organs, specifically affecting aging hallmark pathways such as nutrient sensing, autophagy, proteostasis, energy metabolism, immune function and cellular stress resilience. Age-related transcriptional stress is evolutionary conserved from nematodes to humans. Thus, accumulation of stochastic endogenous DNA damage during aging deteriorates basal transcription, which establishes the age-related transcriptome and causes dysfunction of key aging hallmark pathways, disclosing how DNA damage functionally underlies major aspects of normal aging. In aging mouse livers, 40% of elongating RNA polymerases are stalled, biasing transcriptional output dependent on gene length. This transcriptional stress appears to be caused by endogenous DNA damage.},
  archive      = {J_NG},
  author       = {Gyenis, Akos and Chang, Jiang and Demmers, Joris J. P. G. and Bruens, Serena T. and Barnhoorn, Sander and Brandt, Renata M. C. and Baar, Marjolein P. and Raseta, Marko and Derks, Kasper W. J. and Hoeijmakers, Jan H. J. and Pothof, Joris},
  doi          = {10.1038/s41588-022-01279-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {268-279},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide RNA polymerase stalling shapes the transcriptome during aging},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell transcriptomic analysis of endometriosis.
<em>NG</em>, <em>55</em>(2), 255–267. (<a
href="https://doi.org/10.1038/s41588-022-01254-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Endometriosis is a common condition in women that causes chronic pain and infertility and is associated with an elevated risk of ovarian cancer. We profiled transcriptomes of &amp;gt;370,000 individual cells from endometriomas (n = 8), endometriosis (n = 28), eutopic endometrium (n = 10), unaffected ovary (n = 4) and endometriosis-free peritoneum (n = 4), generating a cellular atlas of endometrial-type epithelial cells, stromal cells and microenvironmental cell populations across tissue sites. Cellular and molecular signatures of endometrial-type epithelium and stroma differed across tissue types, suggesting a role for cellular restructuring and transcriptional reprogramming in the disease. Epithelium, stroma and proximal mesothelial cells of endometriomas showed dysregulation of pro-inflammatory pathways and upregulation of complement proteins. Somatic ARID1A mutation in epithelial cells was associated with upregulation of pro-angiogenic and pro-lymphangiogenic factors and remodeling of the endothelial cell compartment, with enrichment of lymphatic endothelial cells. Finally, signatures of ciliated epithelial cells were enriched in ovarian cancers, reinforcing epidemiologic associations between these two diseases. A single-cell transcriptomic analysis of endometriosis, endometriomas, eutopic endometrial samples and uninvolved ovary tissues highlights cell populations characteristic of these tissue types. Transcriptional and cellular heterogeneity across tissues suggests novel therapeutic targets and biomarkers for this disease.},
  archive      = {J_NG},
  author       = {Fonseca, Marcos A. S. and Haro, Marcela and Wright, Kelly N. and Lin, Xianzhi and Abbasi, Forough and Sun, Jennifer and Hernandez, Lourdes and Orr, Natasha L. and Hong, Jooyoon and Choi-Kuaea, Yunhee and Maluf, Horacio M. and Balzer, Bonnie L. and Fishburn, Aaron and Hickey, Ryan and Cass, Ilana and Goodridge, Helen S. and Truong, Mireille and Wang, Yemin and Pisarska, Margareta D. and Dinh, Huy Q. and EL-Naggar, Amal and Huntsman, David G. and Anglesio, Michael S. and Goodman, Marc T. and Medeiros, Fabiola and Siedhoff, Matthew and Lawrenson, Kate},
  doi          = {10.1038/s41588-022-01254-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {255-267},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell transcriptomic analysis of endometriosis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). APOBEC mutagenesis is a common process in normal human small
intestine. <em>NG</em>, <em>55</em>(2), 246–254. (<a
href="https://doi.org/10.1038/s41588-022-01296-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {APOBEC mutational signatures SBS2 and SBS13 are common in many human cancer types. However, there is an incomplete understanding of its stimulus, when it occurs in the progression from normal to cancer cell and the APOBEC enzymes responsible. Here we whole-genome sequenced 342 microdissected normal epithelial crypts from the small intestines of 39 individuals and found that SBS2/SBS13 mutations were present in 17% of crypts, more frequent than most other normal tissues. Crypts with SBS2/SBS13 often had immediate crypt neighbors without SBS2/SBS13, suggesting that the underlying cause of SBS2/SBS13 is cell-intrinsic. APOBEC mutagenesis occurred in an episodic manner throughout the human lifespan, including in young children. APOBEC1 mRNA levels were very high in the small intestine epithelium, but low in the large intestine epithelium and other tissues. The results suggest that the high levels of SBS2/SBS13 in the small intestine are collateral damage from APOBEC1 fulfilling its physiological function of editing APOB mRNA. Whole-genome sequencing of healthy human epithelial crypts from the small intestines of 39 individuals highlights APOBEC enzymes as a common contributor to the overall mutational burden in this tissue.},
  archive      = {J_NG},
  author       = {Wang, Yichen and Robinson, Philip S. and Coorens, Tim H. H. and Moore, Luiza and Lee-Six, Henry and Noorani, Ayesha and Sanders, Mathijs A. and Jung, Hyunchul and Katainen, Riku and Heuschkel, Robert and Brunton-Sim, Roxanne and Weston, Robyn and Read, Debbie and Nobbs, Beverley and Fitzgerald, Rebecca C. and Saeb-Parsy, Kourosh and Martincorena, Iñigo and Campbell, Peter J. and Rushbrook, Simon and Zilbauer, Matthias and Buczacki, Simon James Alexander and Stratton, Michael R.},
  doi          = {10.1038/s41588-022-01296-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {246-254},
  shortjournal = {Nat. Genet.},
  title        = {APOBEC mutagenesis is a common process in normal human small intestine},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Notch1 mutations drive clonal expansion in normal esophageal
epithelium but impair tumor growth. <em>NG</em>, <em>55</em>(2),
232–245. (<a href="https://doi.org/10.1038/s41588-022-01280-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {NOTCH1 mutant clones occupy the majority of normal human esophagus by middle age but are comparatively rare in esophageal cancers, suggesting NOTCH1 mutations drive clonal expansion but impede carcinogenesis. Here we test this hypothesis. Sequencing NOTCH1 mutant clones in aging human esophagus reveals frequent biallelic mutations that block NOTCH1 signaling. In mouse esophagus, heterozygous Notch1 mutation confers a competitive advantage over wild-type cells, an effect enhanced by loss of the second allele. Widespread Notch1 loss alters transcription but has minimal effects on the epithelial structure and cell dynamics. In a carcinogenesis model, Notch1 mutations were less prevalent in tumors than normal epithelium. Deletion of Notch1 reduced tumor growth, an effect recapitulated by anti-NOTCH1 antibody treatment. Notch1 null tumors showed reduced proliferation. We conclude that Notch1 mutations in normal epithelium are beneficial as wild-type Notch1 favors tumor expansion. NOTCH1 blockade may have therapeutic potential in preventing esophageal squamous cancer. Notch1 mutations have opposing effects on clonal growth in normal and tumor cells of the mouse esophagus. In a mouse model of squamous esophageal tumorigenesis, Notch1 blockade reduced premalignant tumor growth, suggesting that it might be an effective prevention strategy for the disease.},
  archive      = {J_NG},
  author       = {Abby, Emilie and Dentro, Stefan C. and Hall, Michael W. J. and Fowler, Joanna C. and Ong, Swee Hoe and Sood, Roshan and Herms, Albert and Piedrafita, Gabriel and Abnizova, Irina and Siebel, Christian W. and Gerstung, Moritz and Hall, Benjamin A. and Jones, Philip H.},
  doi          = {10.1038/s41588-022-01280-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {232-245},
  shortjournal = {Nat. Genet.},
  title        = {Notch1 mutations drive clonal expansion in normal esophageal epithelium but impair tumor growth},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). MHC II immunogenicity shapes the neoepitope landscape in
human tumors. <em>NG</em>, <em>55</em>(2), 221–231. (<a
href="https://doi.org/10.1038/s41588-022-01273-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in predicting physical peptide-major histocompatibility complex I (pMHC I) binding, it remains challenging to identify functionally immunogenic neoepitopes, especially for MHC II. By using the results of &amp;gt;36,000 immunogenicity assay, we developed a method to identify pMHC whose structural alignment facilitates T cell reaction. Our method predicted neoepitopes for MHC II and MHC I that were responsive to checkpoint blockade when applied to &amp;gt;1,200 samples of various tumor types. To investigate selection by spontaneous immunity at the single epitope level, we analyzed the frequency spectrum of &amp;gt;25 million mutations in &amp;gt;9,000 treatment-naive tumors with &amp;gt;100 immune phenotypes. MHC II immunogenicity specifically lowered variant frequencies in tumors under high immune pressure, particularly with high TCR clonality and MHC II expression. A similar trend was shown for MHC I neoepitopes, but only in particular tissue types. In summary, we report immune selection imposed by MHC II-restricted natural or therapeutic T cell reactivity. DeepNeo identifies major histocompatibility complex (MHC) I or MHC II neoepitopes that are immunogenically compatible with the T cell repertoire. It can predict neoepitopes most likely to be depleted through spontaneous immunity or through immune checkpoint blockade from untreated and immunotherapy-treated tumor datasets.},
  archive      = {J_NG},
  author       = {Kim, Jeong Yeon and Cha, Hongui and Kim, Kyeonghui and Sung, Changhwan and An, Jinhyeon and Bang, Hyoeun and Kim, Hyungjoo and Yang, Jin Ok and Chang, Suhwan and Shin, Incheol and Noh, Seung-Jae and Shin, Inkyung and Cho, Dae-Yeon and Lee, Se-Hoon and Choi, Jung Kyoon},
  doi          = {10.1038/s41588-022-01273-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {221-231},
  shortjournal = {Nat. Genet.},
  title        = {MHC II immunogenicity shapes the neoepitope landscape in human tumors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Comprehensive multi-omic profiling of somatic mutations in
malformations of cortical development. <em>NG</em>, <em>55</em>(2),
209–220. (<a href="https://doi.org/10.1038/s41588-022-01276-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Malformations of cortical development (MCD) are neurological conditions involving focal disruptions of cortical architecture and cellular organization that arise during embryogenesis, largely from somatic mosaic mutations, and cause intractable epilepsy. Identifying the genetic causes of MCD has been a challenge, as mutations remain at low allelic fractions in brain tissue resected to treat condition-related epilepsy. Here we report a genetic landscape from 283 brain resections, identifying 69 mutated genes through intensive profiling of somatic mutations, combining whole-exome and targeted-amplicon sequencing with functional validation including in utero electroporation of mice and single-nucleus RNA sequencing. Genotype–phenotype correlation analysis elucidated specific MCD gene sets associated with distinct pathophysiological and clinical phenotypes. The unique single-cell level spatiotemporal expression patterns of mutated genes in control and patient brains indicate critical roles in excitatory neurogenic pools during brain development and in promoting neuronal hyperexcitability after birth. This Brain Somatic Mosaicism Network analysis of 283 cases of malformations of cortical development identifies 69 candidate and known genes in 76 patients. Single-nucleus RNA sequencing and mouse modeling implicate radial glia and daughter excitatory neurons.},
  archive      = {J_NG},
  author       = {Chung, Changuk and Yang, Xiaoxu and Bae, Taejeong and Vong, Keng Ioi and Mittal, Swapnil and Donkels, Catharina and Westley Phillips, H. and Li, Zhen and Marsh, Ashley P. L. and Breuss, Martin W. and Ball, Laurel L. and Garcia, Camila Araújo Bernardino and George, Renee D. and Gu, Jing and Xu, Mingchu and Barrows, Chelsea and James, Kiely N. and Stanley, Valentina and Nidhiry, Anna S. and Khoury, Sami and Howe, Gabrielle and Riley, Emily and Xu, Xin and Copeland, Brett and Wang, Yifan and Kim, Se Hoon and Kang, Hoon-Chul and Schulze-Bonhage, Andreas and Haas, Carola A. and Urbach, Horst and Prinz, Marco and Limbrick, David D. and Gurnett, Christina A. and Smyth, Matthew D. and Sattar, Shifteh and Nespeca, Mark and Gonda, David D. and Imai, Katsumi and Takahashi, Yukitoshi and Chen, Hsin-Hung and Tsai, Jin-Wu and Conti, Valerio and Guerrini, Renzo and Devinsky, Orrin and Silva, Wilson A. and Machado, Helio R. and Mathern, Gary W. and Abyzov, Alexej and Baldassari, Sara and Baulac, Stéphanie and Gleeson, Joseph G.},
  doi          = {10.1038/s41588-022-01276-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {209-220},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive multi-omic profiling of somatic mutations in malformations of cortical development},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Genome-wide analyses of ADHD identify 27 risk loci, refine
the genetic architecture and implicate several cognitive domains.
<em>NG</em>, <em>55</em>(2), 198–208. (<a
href="https://doi.org/10.1038/s41588-022-01285-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attention-deficit hyperactivity disorder (ADHD) is a prevalent neurodevelopmental disorder with a major genetic component. Here, we present a genome-wide association study meta-analysis of ADHD comprising 38,691 individuals with ADHD and 186,843 controls. We identified 27 genome-wide significant loci, highlighting 76 potential risk genes enriched among genes expressed particularly in early brain development. Overall, ADHD genetic risk was associated with several brain-specific neuronal subtypes and midbrain dopaminergic neurons. In exome-sequencing data from 17,896 individuals, we identified an increased load of rare protein-truncating variants in ADHD for a set of risk genes enriched with probable causal common variants, potentially implicating SORCS3 in ADHD by both common and rare variants. Bivariate Gaussian mixture modeling estimated that 84–98% of ADHD-influencing variants are shared with other psychiatric disorders. In addition, common-variant ADHD risk was associated with impaired complex cognition such as verbal reasoning and a range of executive functions, including attention. Genome-wide analyses identify 27 loci associated with attention-deficit hyperactivity disorder and provide insights into its genetic architecture in relation to other psychiatric disorders and cognitive traits.},
  archive      = {J_NG},
  author       = {Demontis, Ditte and Walters, G. Bragi and Athanasiadis, Georgios and Walters, Raymond and Therrien, Karen and Nielsen, Trine Tollerup and Farajzadeh, Leila and Voloudakis, Georgios and Bendl, Jaroslav and Zeng, Biau and Zhang, Wen and Grove, Jakob and Als, Thomas D. and Duan, Jinjie and Satterstrom, F. Kyle and Bybjerg-Grauholm, Jonas and Bækved-Hansen, Marie and Gudmundsson, Olafur O. and Magnusson, Sigurdur H. and Baldursson, Gisli and Davidsdottir, Katrin and Haraldsdottir, Gyda S. and Agerbo, Esben and Hoffman, Gabriel E. and Dalsgaard, Søren and Martin, Joanna and Ribasés, Marta and Boomsma, Dorret I. and Soler Artigas, Maria and Roth Mota, Nina and Howrigan, Daniel and Medland, Sarah E. and Zayats, Tetyana and Rajagopal, Veera M. and Nordentoft, Merete and Mors, Ole and Hougaard, David M. and Mortensen, Preben Bo and Daly, Mark J. and Faraone, Stephen V. and Stefansson, Hreinn and Roussos, Panos and Franke, Barbara and Werge, Thomas and Neale, Benjamin M. and Stefansson, Kari and Børglum, Anders D.},
  doi          = {10.1038/s41588-022-01285-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {198-208},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cross-ancestry genome-wide analysis of atrial fibrillation
unveils disease biology and enables cardioembolic risk prediction.
<em>NG</em>, <em>55</em>(2), 187–197. (<a
href="https://doi.org/10.1038/s41588-022-01284-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atrial fibrillation (AF) is a common cardiac arrhythmia resulting in increased risk of stroke. Despite highly heritable etiology, our understanding of the genetic architecture of AF remains incomplete. Here we performed a genome-wide association study in the Japanese population comprising 9,826 cases among 150,272 individuals and identified East Asian-specific rare variants associated with AF. A cross-ancestry meta-analysis of &amp;gt;1 million individuals, including 77,690 cases, identified 35 new susceptibility loci. Transcriptome-wide association analysis identified IL6R as a putative causal gene, suggesting the involvement of immune responses. Integrative analysis with ChIP-seq data and functional assessment using human induced pluripotent stem cell-derived cardiomyocytes demonstrated ERRg as having a key role in the transcriptional regulation of AF-associated genes. A polygenic risk score derived from the cross-ancestry meta-analysis predicted increased risks of cardiovascular and stroke mortalities and segregated individuals with cardioembolic stroke in undiagnosed AF patients. Our results provide new biological and clinical insights into AF genetics and suggest their potential for clinical applications. Genome-wide analyses in BioBank Japan and populations of European ancestry identify new risk loci for atrial fibrillation. A polygenic risk score constructed from the cross-ancestry meta-analysis is associated with increased risk of long-term cardiovascular mortality.},
  archive      = {J_NG},
  author       = {Miyazawa, Kazuo and Ito, Kaoru and Ito, Masamichi and Zou, Zhaonan and Kubota, Masayuki and Nomura, Seitaro and Matsunaga, Hiroshi and Koyama, Satoshi and Ieki, Hirotaka and Akiyama, Masato and Koike, Yoshinao and Kurosawa, Ryo and Yoshida, Hiroki and Ozaki, Kouichi and Onouchi, Yoshihiro and Takahashi, Atsushi and Matsuda, Koichi and Murakami, Yoshinori and Aburatani, Hiroyuki and Kubo, Michiaki and Momozawa, Yukihide and Terao, Chikashi and Oki, Shinya and Akazawa, Hiroshi and Kamatani, Yoichiro and Komuro, Issei},
  doi          = {10.1038/s41588-022-01284-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {187-197},
  shortjournal = {Nat. Genet.},
  title        = {Cross-ancestry genome-wide analysis of atrial fibrillation unveils disease biology and enables cardioembolic risk prediction},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The singapore national precision medicine strategy.
<em>NG</em>, <em>55</em>(2), 178–186. (<a
href="https://doi.org/10.1038/s41588-022-01274-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precision medicine promises to transform healthcare for groups and individuals through early disease detection, refining diagnoses and tailoring treatments. Analysis of large-scale genomic–phenotypic databases is a critical enabler of precision medicine. Although Asia is home to 60% of the world’s population, many Asian ancestries are under-represented in existing databases, leading to missed opportunities for new discoveries, particularly for diseases most relevant for these populations. The Singapore National Precision Medicine initiative is a whole-of-government 10-year initiative aiming to generate precision medicine data of up to one million individuals, integrating genomic, lifestyle, health, social and environmental data. Beyond technologies, routine adoption of precision medicine in clinical practice requires social, ethical, legal and regulatory barriers to be addressed. Identifying driver use cases in which precision medicine results in standardized changes to clinical workflows or improvements in population health, coupled with health economic analysis to demonstrate value-based healthcare, is a vital prerequisite for responsible health system adoption. This Perspective article discusses Singapore’s efforts to implement a National Precision Medicine Strategy through the integration of genomic, clinical and lifestyle data of up to one million Singaporean individuals.},
  archive      = {J_NG},
  author       = {Wong, Eleanor and Bertin, Nicolas and Hebrard, Maxime and Tirado-Magallanes, Roberto and Bellis, Claire and Lim, Weng Khong and Chua, Chee Yong and Tong, Philomena Mei Lin and Chua, Raymond and Mak, Kenneth and Lim, Tit Meng and Cheong, Wei Yang and Thien, Kwee Eng and Goh, Khean Teik and Chai, Jin-Fang and Lee, Jimmy and Sung, Joseph Jao-Yiu and Wong, Tien Yin and Chin, Calvin Woon Loong and Gluckman, Peter D. and Goh, Liuh Ling and Ban, Kenneth Hon Kim and Tan, Tin Wee and Sim, Xueling and Cheng, Ching-Yu and Davila, Sonia and Karnani, Neerja and Leong, Khai Pang and Liu, Jianjun and Prabhakar, Shyam and Maurer-Stroh, Sebastian and Verma, Chandra Shekhar and Krishnaswamy, Pavitra and Goh, Rick Siow Mong and Chia, Irenaeus and Ho, Clarissa and Low, Doreen and Virabhak, Suchin and Yong, Jacklyn and Zheng, Weiling and Seow, Shih Wee and Seck, Yee Kwang and Koh, Mingshi and Chambers, John C. and Tai, E. Shyong and Tan, Patrick},
  doi          = {10.1038/s41588-022-01274-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {178-186},
  shortjournal = {Nat. Genet.},
  title        = {The singapore national precision medicine strategy},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mutational signatures in human small intestinal crypts
indicate APOBEC mutagenesis. <em>NG</em>, <em>55</em>(2), 176–177. (<a
href="https://doi.org/10.1038/s41588-023-01304-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using laser-capture microdissection and whole-genome sequencing of individual crypts, we characterized the landscape of somatic mutations in human small intestinal epithelium. Mutational signatures of APOBEC mutagenesis were found frequently and are probably due to the activity of APOBEC1, which is expressed at high levels in the small intestine.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-023-01304-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {176-177},
  shortjournal = {Nat. Genet.},
  title        = {Mutational signatures in human small intestinal crypts indicate APOBEC mutagenesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sox2 expression can be regulated across boundaries generated
by CTCF–cohesin loops. <em>NG</em>, <em>55</em>(2), 174–175. (<a
href="https://doi.org/10.1038/s41588-022-01294-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using a series of mouse mutants, we found that the Sox2 promoter does not require CTCF–cohesin loops to interact with distal enhancers. Surprisingly, mice with varying numbers of CTCF motifs in different positions showed that some distal enhancers can bypass boundaries that are created by CTCF–cohesin loops to ensure robust Sox2 expression.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01294-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {174-175},
  shortjournal = {Nat. Genet.},
  title        = {Sox2 expression can be regulated across boundaries generated by CTCF–cohesin loops},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A scalable technology for measuring cell-type-specific
activity of cis-regulatory sequences. <em>NG</em>, <em>55</em>(2),
172–173. (<a href="https://doi.org/10.1038/s41588-022-01277-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A high-throughput reporter gene assay for use in living tissues has been developed. The assay allows investigators to quantify, in parallel, the activities of reporter genes in each of the cell types that constitute a tissue.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01277-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {172-173},
  shortjournal = {Nat. Genet.},
  title        = {A scalable technology for measuring cell-type-specific activity of cis-regulatory sequences},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SortChIC enables detailed chromatin analysis of rare cell
types. <em>NG</em>, <em>55</em>(2), 170–171. (<a
href="https://doi.org/10.1038/s41588-022-01259-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods of chromatin analysis focus mainly on the most abundant cell types in a sample. We present a workflow that combines enrichment of rare cell types with high-resolution mapping of histone modifications, which enables us to study chromatin dynamics in rare stem and progenitor cell populations.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01259-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {170-171},
  shortjournal = {Nat. Genet.},
  title        = {SortChIC enables detailed chromatin analysis of rare cell types},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dissecting endometriosis by single-cell transcriptomic and
genomic analysis. <em>NG</em>, <em>55</em>(2), 168–169. (<a
href="https://doi.org/10.1038/s41588-022-01255-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Endometriosis affects around 10% of individuals born with a uterus, yet we know remarkably little about its underlying biology. Our single-cell transcriptional profiling of endometrial-type epithelial and stromal cells is shedding light on the cells and processes that contribute to endometriosis, which opens up new avenues for diagnostics and therapeutics.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01255-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {168-169},
  shortjournal = {Nat. Genet.},
  title        = {Dissecting endometriosis by single-cell transcriptomic and genomic analysis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Linking GWAS to gene regulation. <em>NG</em>,
<em>55</em>(2), 167. (<a
href="https://doi.org/10.1038/s41588-023-01309-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-023-01309-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {167},
  shortjournal = {Nat. Genet.},
  title        = {Linking GWAS to gene regulation},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Sarcoma predisposition. <em>NG</em>, <em>55</em>(2), 167.
(<a href="https://doi.org/10.1038/s41588-023-01310-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-023-01310-4},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {167},
  shortjournal = {Nat. Genet.},
  title        = {Sarcoma predisposition},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Human cell-type methylomes. <em>NG</em>, <em>55</em>(2),
167. (<a href="https://doi.org/10.1038/s41588-023-01311-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-023-01311-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {167},
  shortjournal = {Nat. Genet.},
  title        = {Human cell-type methylomes},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Linking viruses with neurodegeneration. <em>NG</em>,
<em>55</em>(2), 167. (<a
href="https://doi.org/10.1038/s41588-023-01312-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-023-01312-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {167},
  shortjournal = {Nat. Genet.},
  title        = {Linking viruses with neurodegeneration},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Publisher correction: Reference component analysis of
single-cell transcriptomes elucidates cellular heterogeneity in human
colorectal tumors. <em>NG</em>, <em>55</em>(1), 166. (<a
href="https://doi.org/10.1038/s41588-022-01281-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Huipeng and Courtois, Elise T. and Sengupta, Debarka and Tan, Yuliana and Chen, Kok Hao and Goh, Jolene Jie Lin and Kong, Say Li and Chua, Clarinda and Hon, Lim Kiat and Tan, Wah Siew and Wong, Mark and Choi, Paul Jongjoon and Wee, Lawrence J. K. and Hillmer, Axel M. and Tan, Iain Beehuat and Robson, Paul and Prabhakar, Shyam},
  doi          = {10.1038/s41588-022-01281-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {166},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Reference component analysis of single-cell transcriptomes elucidates cellular heterogeneity in human colorectal tumors},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Mapping the temporal and spatial dynamics
of the human endometrium in vivo and in vitro. <em>NG</em>,
<em>55</em>(1), 165. (<a
href="https://doi.org/10.1038/s41588-022-01287-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Garcia-Alonso, Luz and Handfield, Louis-François and Roberts, Kenny and Nikolakopoulou, Konstantina and Fernando, Ridma C. and Gardner, Lucy and Woodhams, Benjamin and Arutyunyan, Anna and Polanski, Krzysztof and Hoo, Regina and Sancho-Serra, Carmen and Li, Tong and Kwakwa, Kwasi and Tuck, Elizabeth and Lorenzi, Valentina and Massalha, Hassan and Prete, Martin and Kleshchevnikov, Vitalii and Tarkowska, Aleksandra and Porter, Tarryn and Mazzeo, Cecilia Icoresi and van Dongen, Stijn and Dabrowska, Monika and Vaskivskyi, Vasyl and Mahbubani, Krishnaa T. and Park, Jong-eun and Jimenez-Linan, Mercedes and Campos, Lia and Kiselev, Vladimir Yu. and Lindskog, Cecilia and Ayuk, Paul and Prigmore, Elena and Stratton, Michael R. and Saeb-Parsy, Kourosh and Moffett, Ashley and Moore, Luiza and Bayraktar, Omer A. and Teichmann, Sarah A. and Turco, Margherita Y. and Vento-Tormo, Roser},
  doi          = {10.1038/s41588-022-01287-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {165},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Mapping the temporal and spatial dynamics of the human endometrium in vivo and in vitro},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Powerful, scalable and resource-efficient meta-analysis of
rare variant associations in large whole genome sequencing studies.
<em>NG</em>, <em>55</em>(1), 154–164. (<a
href="https://doi.org/10.1038/s41588-022-01225-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Meta-analysis of whole genome sequencing/whole exome sequencing (WGS/WES) studies provides an attractive solution to the problem of collecting large sample sizes for discovering rare variants associated with complex phenotypes. Existing rare variant meta-analysis approaches are not scalable to biobank-scale WGS data. Here we present MetaSTAAR, a powerful and resource-efficient rare variant meta-analysis framework for large-scale WGS/WES studies. MetaSTAAR accounts for relatedness and population structure, can analyze both quantitative and dichotomous traits and boosts the power of rare variant tests by incorporating multiple variant functional annotations. Through meta-analysis of four lipid traits in 30,138 ancestrally diverse samples from 14 studies of the Trans Omics for Precision Medicine (TOPMed) Program, we show that MetaSTAAR performs rare variant meta-analysis at scale and produces results comparable to using pooled data. Additionally, we identified several conditionally significant rare variant associations with lipid traits. We further demonstrate that MetaSTAAR is scalable to biobank-scale cohorts through meta-analysis of TOPMed WGS data and UK Biobank WES data of ~200,000 samples. MetaSTAAR enables functionally informed rare variant association analysis in biobank-scale cohorts using an efficient, sparse matrix approach for summary statistic storage.},
  archive      = {J_NG},
  author       = {Li, Xihao and Quick, Corbin and Zhou, Hufeng and Gaynor, Sheila M. and Liu, Yaowu and Chen, Han and Selvaraj, Margaret Sunitha and Sun, Ryan and Dey, Rounak and Arnett, Donna K. and Bielak, Lawrence F. and Bis, Joshua C. and Blangero, John and Boerwinkle, Eric and Bowden, Donald W. and Brody, Jennifer A. and Cade, Brian E. and Correa, Adolfo and Cupples, L. Adrienne and Curran, Joanne E. and de Vries, Paul S. and Duggirala, Ravindranath and Freedman, Barry I. and Göring, Harald H. H. and Guo, Xiuqing and Haessler, Jeffrey and Kalyani, Rita R. and Kooperberg, Charles and Kral, Brian G. and Lange, Leslie A. and Manichaikul, Ani and Martin, Lisa W. and McGarvey, Stephen T. and Mitchell, Braxton D. and Montasser, May E. and Morrison, Alanna C. and Naseri, Take and O’Connell, Jeffrey R. and Palmer, Nicholette D. and Peyser, Patricia A. and Psaty, Bruce M. and Raffield, Laura M. and Redline, Susan and Reiner, Alexander P. and Reupena, Muagututi’a Sefuiva and Rice, Kenneth M. and Rich, Stephen S. and Sitlani, Colleen M. and Smith, Jennifer A. and Taylor, Kent D. and Vasan, Ramachandran S. and Willer, Cristen J. and Wilson, James G. and Yanek, Lisa R. and Zhao, Wei and Rotter, Jerome I. and Natarajan, Pradeep and Peloso, Gina M. and Li, Zilin and Lin, Xihong},
  doi          = {10.1038/s41588-022-01225-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {154-164},
  shortjournal = {Nat. Genet.},
  title        = {Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A multi-omics integrative network map of maize. <em>NG</em>,
<em>55</em>(1), 144–153. (<a
href="https://doi.org/10.1038/s41588-022-01262-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Networks are powerful tools to uncover functional roles of genes in phenotypic variation at a system-wide scale. Here, we constructed a maize network map that contains the genomic, transcriptomic, translatomic and proteomic networks across maize development. This map comprises over 2.8 million edges in more than 1,400 functional subnetworks, demonstrating an extensive network divergence of duplicated genes. We applied this map to identify factors regulating flowering time and identified 2,651 genes enriched in eight subnetworks. We validated the functions of 20 genes, including 18 with previously unknown connections to flowering time in maize. Furthermore, we uncovered a flowering pathway involving histone modification. The multi-omics integrative network map illustrates the principles of how molecular networks connect different types of genes and potential pathways to map a genome-wide functional landscape in maize, which should be applicable in a wide range of species. An integrative network map of maize (Zea mays L.) that contains genomic, transcriptomic, translatomic and proteomic networks illustrates the landscape of molecular interactions of different functional elements and potential pathway modules in maize.},
  archive      = {J_NG},
  author       = {Han, Linqian and Zhong, Wanshun and Qian, Jia and Jin, Minliang and Tian, Peng and Zhu, Wanchao and Zhang, Hongwei and Sun, Yonghao and Feng, Jia-Wu and Liu, Xiangguo and Chen, Guo and Farid, Babar and Li, Ruonan and Xiong, Zimo and Tian, Zhihui and Li, Juan and Luo, Zi and Du, Dengxiang and Chen, Sijia and Jin, Qixiao and Li, Jiaxin and Li, Zhao and Liang, Yan and Jin, Xiaomeng and Peng, Yong and Zheng, Chang and Ye, Xinnan and Yin, Yuejia and Chen, Hong and Li, Weifu and Chen, Ling-Ling and Li, Qing and Yan, Jianbing and Yang, Fang and Li, Lin},
  doi          = {10.1038/s41588-022-01262-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {144-153},
  shortjournal = {Nat. Genet.},
  title        = {A multi-omics integrative network map of maize},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dynamics of DNA hydroxymethylation and methylation during
mouse embryonic and germline development. <em>NG</em>, <em>55</em>(1),
130–143. (<a href="https://doi.org/10.1038/s41588-022-01258-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In mammals, DNA 5-hydroxymethylcytosine (5hmC) is involved in methylation reprogramming during early embryonic development. Yet, to what extent 5hmC participates in genome-wide methylation reprogramming remains largely unknown. Here, we characterize the 5hmC landscapes in mouse early embryos and germ cells with parental allele specificity. DNA hydroxymethylation was most strongly correlated with DNA demethylation as compared with de novo or maintenance methylation in zygotes, while 5hmC was targeted to particular de novo methylated sites in postimplantation epiblasts. Surprisingly, DNA replication was also required for 5hmC generation, especially in the female pronucleus. More strikingly, aberrant nuclear localization of Dnmt1/Uhrf1 in mouse zygotes due to maternal deficiency of Nlrp14 led to defects in DNA-replication-coupled passive demethylation and impaired 5hmC deposition, revealing the divergency between genome-wide 5-methylcytosine (5mC) maintenance and Tet-mediated oxidation. In summary, our work provides insights and a valuable resource for the study of epigenetic regulation in early embryo development. DNA 5-hydroxymethylcytosine (5hmC) is mapped in mouse preimplantation embryos, postimplantation epiblasts, primordial germ cells (PGCs) and gametes. Nlrp14 maternal depletion perturbs DNA-replication-coupled passive demethylation and impairs 5hmC deposition.},
  archive      = {J_NG},
  author       = {Yan, Rui and Cheng, Xin and Gu, Chan and Xu, Yanhong and Long, Xin and Zhai, Jinglei and Sun, Fengyuan and Qian, Jingjing and Du, Yarui and Wang, Hongmei and Guo, Fan},
  doi          = {10.1038/s41588-022-01258-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {130-143},
  shortjournal = {Nat. Genet.},
  title        = {Dynamics of DNA hydroxymethylation and methylation during mouse embryonic and germline development},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dietary stress remodels the genetic architecture of lifespan
variation in outbred drosophila. <em>NG</em>, <em>55</em>(1), 123–129.
(<a href="https://doi.org/10.1038/s41588-022-01246-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionary theory suggests that lifespan-reducing alleles should be purged from the gene pool, and yet decades of genome-wide association and model organism studies have shown that they persist. One potential explanation is that alleles that regulate lifespan do so only in certain environmental contexts. We exposed outbred Drosophila to control and high-sugar diets and genotyped more than 10,000 adult flies to track allele frequency changes over the course of a single adult lifespan. We identified thousands of lifespan-associated alleles associated with early versus late-life trade-offs, late-onset effects and genotype-by-environment interactions. Remarkably, a third of lifespan-associated genetic variation had environmentally dependent effects on lifespan. We find that lifespan-reducing alleles are often recently derived, have stronger effects on a high-sugar diet and show signatures of selection in wild Drosophila populations, consistent with the evolutionary mismatch hypothesis. Our results provide insight into the highly polygenic and context-dependent genetic architecture of lifespan variation and the evolutionary processes that shape this key trait. An analysis of the effects of dietary stress in outbred Drosophila shows that lifespan has a polygenic architecture and is subject to environmental influence, suggesting that this context dependency is important for complex trait variation and evolution.},
  archive      = {J_NG},
  author       = {Pallares, Luisa F. and Lea, Amanda J. and Han, Clair and Filippova, Elena V. and Andolfatto, Peter and Ayroles, Julien F.},
  doi          = {10.1038/s41588-022-01246-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {123-129},
  shortjournal = {Nat. Genet.},
  title        = {Dietary stress remodels the genetic architecture of lifespan variation in outbred drosophila},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). DNA methylation QTL mapping across diverse human tissues
provides molecular links between genetic variation and complex traits.
<em>NG</em>, <em>55</em>(1), 112–122. (<a
href="https://doi.org/10.1038/s41588-022-01248-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies of DNA methylation (DNAm) in solid human tissues are relatively scarce; tissue-specific characterization of DNAm is needed to understand its role in gene regulation and its relevance to complex traits. We generated array-based DNAm profiles for 987 human samples from the Genotype-Tissue Expression (GTEx) project, representing 9 tissue types and 424 subjects. We characterized methylome and transcriptome correlations (eQTMs), genetic regulation in cis (mQTLs and eQTLs) across tissues and e/mQTLs links to complex traits. We identified mQTLs for 286,152 CpG sites, many of which (&amp;gt;5%) show tissue specificity, and mQTL colocalizations with 2,254 distinct GWAS hits across 83 traits. For 91% of these loci, a candidate gene link was identified by integration of functional maps, including eQTMs, and/or eQTL colocalization, but only 33% of loci involved an eQTL and mQTL present in the same tissue type. With this DNAm-focused integrative analysis, we contribute to the understanding of molecular regulatory mechanisms in human tissues and their impact on complex traits. As part of the enhanced GTEx (eGTEx) project, 987 human samples from 9 tissue types and 424 donors are assayed using DNA methylation microarrays. Colocalization of GWAS variants, eQTLs and mQTLs shows diverse links between genetic variation, molecular phenotypes and complex traits.},
  archive      = {J_NG},
  author       = {Oliva, Meritxell and Demanelis, Kathryn and Lu, Yihao and Chernoff, Meytal and Jasmine, Farzana and Ahsan, Habibul and Kibriya, Muhammad G. and Chen, Lin S. and Pierce, Brandon L.},
  doi          = {10.1038/s41588-022-01248-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {112-122},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation QTL mapping across diverse human tissues provides molecular links between genetic variation and complex traits},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spurious transcription causing innate immune responses is
prevented by 5-hydroxymethylcytosine. <em>NG</em>, <em>55</em>(1),
100–111. (<a href="https://doi.org/10.1038/s41588-022-01252-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generation of functional transcripts requires transcriptional initiation at regular start sites, avoiding production of aberrant and potentially hazardous aberrant RNAs. The mechanisms maintaining transcriptional fidelity and the impact of spurious transcripts on cellular physiology and organ function have not been fully elucidated. Here we show that TET3, which successively oxidizes 5-methylcytosine to 5-hydroxymethylcytosine (5hmC) and other derivatives, prevents aberrant intragenic entry of RNA polymerase II pSer5 into highly expressed genes of airway smooth muscle cells, assuring faithful transcriptional initiation at canonical start sites. Loss of TET3-dependent 5hmC production in SMCs results in accumulation of spurious transcripts, which stimulate the endosomal nucleic-acid-sensing TLR7/8 signaling pathway, thereby provoking massive inflammation and airway remodeling resembling human bronchial asthma. Furthermore, we found that 5hmC levels are substantially lower in human asthma airways compared with control samples. Suppression of spurious transcription might be important to prevent chronic inflammation in asthma. Smooth muscle cell-specific knockout of Tet3 in mice leads to loss of intragenic 5-hydroxymethylcytosine, accumulation of spurious transcripts and TLR7/8-mediated lung inflammation resembling asthma in human lung samples.},
  archive      = {J_NG},
  author       = {Wu, Fan and Li, Xiang and Looso, Mario and Liu, Hang and Ding, Dong and Günther, Stefan and Kuenne, Carsten and Liu, Shuya and Weissmann, Norbert and Boettger, Thomas and Atzberger, Ann and Kolahian, Saeed and Renz, Harald and Offermanns, Stefan and Gärtner, Ulrich and Potente, Michael and Zhou, Yonggang and Yuan, Xuejun and Braun, Thomas},
  doi          = {10.1038/s41588-022-01252-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {100-111},
  shortjournal = {Nat. Genet.},
  title        = {Spurious transcription causing innate immune responses is prevented by 5-hydroxymethylcytosine},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Deciphering colorectal cancer genetics through multi-omic
analysis of 100,204 cases and 154,587 controls of european and east
asian ancestries. <em>NG</em>, <em>55</em>(1), 89–99. (<a
href="https://doi.org/10.1038/s41588-022-01222-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Colorectal cancer (CRC) is a leading cause of mortality worldwide. We conducted a genome-wide association study meta-analysis of 100,204 CRC cases and 154,587 controls of European and east Asian ancestry, identifying 205 independent risk associations, of which 50 were unreported. We performed integrative genomic, transcriptomic and methylomic analyses across large bowel mucosa and other tissues. Transcriptome- and methylome-wide association studies revealed an additional 53 risk associations. We identified 155 high-confidence effector genes functionally linked to CRC risk, many of which had no previously established role in CRC. These have multiple different functions and specifically indicate that variation in normal colorectal homeostasis, proliferation, cell adhesion, migration, immunity and microbial interactions determines CRC risk. Crosstissue analyses indicated that over a third of effector genes most probably act outside the colonic mucosa. Our findings provide insights into colorectal oncogenesis and highlight potential targets across tissues for new CRC treatment and chemoprevention strategies. A multi-ancestry genome-wide association study meta-analysis, combined with transcriptome- and methylome-wide association analyses, identifies risk loci associated with colorectal cancer. Credible effector genes and their target tissues are also highlighted, showing that over a third probably act outside the colonic mucosa.},
  archive      = {J_NG},
  author       = {Fernandez-Rozadilla, Ceres and Timofeeva, Maria and Chen, Zhishan and Law, Philip and Thomas, Minta and Schmit, Stephanie and Díez-Obrero, Virginia and Hsu, Li and Fernandez-Tajes, Juan and Palles, Claire and Sherwood, Kitty and Briggs, Sarah and Svinti, Victoria and Donnelly, Kevin and Farrington, Susan and Blackmur, James and Vaughan-Shaw, Peter and Shu, Xiao-ou and Long, Jirong and Cai, Qiuyin and Guo, Xingyi and Lu, Yingchang and Broderick, Peter and Studd, James and Huyghe, Jeroen and Harrison, Tabitha and Conti, David and Dampier, Christopher and Devall, Mathew and Schumacher, Fredrick and Melas, Marilena and Rennert, Gad and Obón-Santacana, Mireia and Martín-Sánchez, Vicente and Moratalla-Navarro, Ferran and Oh, Jae Hwan and Kim, Jeongseon and Jee, Sun Ha and Jung, Keum Ji and Kweon, Sun-Seog and Shin, Min-Ho and Shin, Aesun and Ahn, Yoon-Ok and Kim, Dong-Hyun and Oze, Isao and Wen, Wanqing and Matsuo, Keitaro and Matsuda, Koichi and Tanikawa, Chizu and Ren, Zefang and Gao, Yu-Tang and Jia, Wei-Hua and Hopper, John and Jenkins, Mark and Win, Aung Ko and Pai, Rish and Figueiredo, Jane and Haile, Robert and Gallinger, Steven and Woods, Michael and Newcomb, Polly and Duggan, David and Cheadle, Jeremy and Kaplan, Richard and Maughan, Timothy and Kerr, Rachel and Kerr, David and Kirac, Iva and Böhm, Jan and Mecklin, Lukka-Pekka and Jousilahti, Pekka and Knekt, Paul and Aaltonen, Lauri and Rissanen, Harri and Pukkala, Eero and Eriksson, Johan and Cajuso, Tatiana and Hänninen, Ulrika and Kondelin, Johanna and Palin, Kimmo and Tanskanen, Tomas and Renkonen-Sinisalo, Laura and Zanke, Brent and Männistö, Satu and Albanes, Demetrius and Weinstein, Stephanie and Ruiz-Narvaez, Edward and Palmer, Julie and Buchanan, Daniel and Platz, Elizabeth and Visvanathan, Kala and Ulrich, Cornelia and Siegel, Erin and Brezina, Stefanie and Gsur, Andrea and Campbell, Peter and Chang-Claude, Jenny and Hoffmeister, Michael and Brenner, Hermann and Slattery, Martha and Potter, John and Tsilidis, Konstantinos and Schulze, Matthias and Gunter, Marc and Murphy, Neil and Castells, Antoni and Castellví-Bel, Sergi and Moreira, Leticia and Arndt, Volker and Shcherbina, Anna and Stern, Mariana and Pardamean, Bens and Bishop, Timothy and Giles, Graham and Southey, Melissa and Idos, Gregory and McDonnell, Kevin and Abu-Ful, Zomoroda and Greenson, Joel and Shulman, Katerina and Lejbkowicz, Flavio and Offit, Kenneth and Su, Yu-Ru and Steinfelder, Robert and Keku, Temitope and van Guelpen, Bethany and Hudson, Thomas and Hampel, Heather and Pearlman, Rachel and Berndt, Sonja and Hayes, Richard and Martinez, Marie Elena and Thomas, Sushma and Corley, Douglas and Pharoah, Paul and Larsson, Susanna and Yen, Yun and Lenz, Heinz-Josef and White, Emily and Li, Li and Doheny, Kimberly and Pugh, Elizabeth and Shelford, Tameka and Chan, Andrew and Cruz-Correa, Marcia and Lindblom, Annika and Hunter, David and Joshi, Amit and Schafmayer, Clemens and Scacheri, Peter and Kundaje, Anshul and Nickerson, Deborah and Schoen, Robert and Hampe, Jochen and Stadler, Zsofia and Vodicka, Pavel and Vodickova, Ludmila and Vymetalkova, Veronika and Papadopoulos, Nickolas and Edlund, Chistopher and Gauderman, William and Thomas, Duncan and Shibata, David and Toland, Amanda and Markowitz, Sanford and Kim, Andre and Chanock, Stephen and van Duijnhoven, Franzel and Feskens, Edith and Sakoda, Lori and Gago-Dominguez, Manuela and Wolk, Alicja and Naccarati, Alessio and Pardini, Barbara and FitzGerald, Liesel and Lee, Soo Chin and Ogino, Shuji and Bien, Stephanie and Kooperberg, Charles and Li, Christopher and Lin, Yi and Prentice, Ross and Qu, Conghui and Bézieau, Stéphane and Tangen, Catherine and Mardis, Elaine and Yamaji, Taiki and Sawada, Norie and Iwasaki, Motoki and Haiman, Christopher and Le Marchand, Loic and Wu, Anna and Qu, Chenxu and McNeil, Caroline and Coetzee, Gerhard and Hayward, Caroline and Deary, Ian and Harris, Sarah and Theodoratou, Evropi and Reid, Stuart and Walker, Marion and Ooi, Li Yin and Moreno, Victor and Casey, Graham and Gruber, Stephen and Tomlinson, Ian and Zheng, Wei and Dunlop, Malcolm and Houlston, Richard and Peters, Ulrike},
  doi          = {10.1038/s41588-022-01222-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {89-99},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering colorectal cancer genetics through multi-omic analysis of 100,204 cases and 154,587 controls of european and east asian ancestries},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SpiceMix enables integrative single-cell spatial modeling of
cell identity. <em>NG</em>, <em>55</em>(1), 78–88. (<a
href="https://doi.org/10.1038/s41588-022-01256-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics can reveal spatially resolved gene expression of diverse cells in complex tissues. However, the development of computational methods that can use the unique properties of spatial transcriptome data to unveil cell identities remains a challenge. Here we introduce SpiceMix, an interpretable method based on probabilistic, latent variable modeling for joint analysis of spatial information and gene expression from spatial transcriptome data. Both simulation and real data evaluations demonstrate that SpiceMix markedly improves on the inference of cell types and their spatial patterns compared with existing approaches. By applying to spatial transcriptome data of brain regions in human and mouse acquired by seqFISH+, STARmap and Visium, we show that SpiceMix can enhance the inference of complex cell identities, reveal interpretable spatial metagenes and uncover differentiation trajectories. SpiceMix is a generalizable analysis framework for spatial transcriptome data to investigate cell-type composition and spatial organization of cells in complex tissues. SpiceMix uses latent variable modeling to infer cell identities by jointly analyzing intrinsic and spatial factors, allowing the identification of spatially variable metagenes and refined cell subtypes.},
  archive      = {J_NG},
  author       = {Chidester, Benjamin and Zhou, Tianming and Alam, Shahul and Ma, Jian},
  doi          = {10.1038/s41588-022-01256-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {78-88},
  shortjournal = {Nat. Genet.},
  title        = {SpiceMix enables integrative single-cell spatial modeling of cell identity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A spatially resolved atlas of the human lung characterizes a
gland-associated immune niche. <em>NG</em>, <em>55</em>(1), 66–77. (<a
href="https://doi.org/10.1038/s41588-022-01243-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell transcriptomics has allowed unprecedented resolution of cell types/states in the human lung, but their spatial context is less well defined. To (re)define tissue architecture of lung and airways, we profiled five proximal-to-distal locations of healthy human lungs in depth using multi-omic single cell/nuclei and spatial transcriptomics (queryable at lungcellatlas.org ). Using computational data integration and analysis, we extend beyond the suspension cell paradigm and discover macro and micro-anatomical tissue compartments including previously unannotated cell types in the epithelial, vascular, stromal and nerve bundle micro-environments. We identify and implicate peribronchial fibroblasts in lung disease. Importantly, we discover and validate a survival niche for IgA plasma cells in the airway submucosal glands (SMG). We show that gland epithelial cells recruit B cells and IgA plasma cells, and promote longevity and antibody secretion locally through expression of CCL28, APRIL and IL-6. This new ‘gland-associated immune niche’ has implications for respiratory health. Multi-omics profiling of 45 human lung samples highlights 80 different cell types along the proximal to distal axis of the lung with certain cell types showing enrichment for disease-associated genes. An immune niche for IgA-expressing plasma cells within airway submucosal glands (SMG) is also identified.},
  archive      = {J_NG},
  author       = {Madissoon, Elo and Oliver, Amanda J. and Kleshchevnikov, Vitalii and Wilbrey-Clark, Anna and Polanski, Krzysztof and Richoz, Nathan and Ribeiro Orsi, Ana and Mamanova, Lira and Bolt, Liam and Elmentaite, Rasa and Pett, J. Patrick and Huang, Ni and Xu, Chuan and He, Peng and Dabrowska, Monika and Pritchard, Sophie and Tuck, Liz and Prigmore, Elena and Perera, Shani and Knights, Andrew and Oszlanczi, Agnes and Hunter, Adam and Vieira, Sara F. and Patel, Minal and Lindeboom, Rik G. H. and Campos, Lia S. and Matsuo, Kazuhiko and Nakayama, Takashi and Yoshida, Masahiro and Worlock, Kaylee B. and Nikolić, Marko Z. and Georgakopoulos, Nikitas and Mahbubani, Krishnaa T. and Saeb-Parsy, Kourosh and Bayraktar, Omer Ali and Clatworthy, Menna R. and Stegle, Oliver and Kumasaka, Natsuhiko and Teichmann, Sarah A. and Meyer, Kerstin B.},
  doi          = {10.1038/s41588-022-01243-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {66-77},
  shortjournal = {Nat. Genet.},
  title        = {A spatially resolved atlas of the human lung characterizes a gland-associated immune niche},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A genome-wide CRISPR screen identifies CALCOCO2 as a
regulator of beta cell function influencing type 2 diabetes risk.
<em>NG</em>, <em>55</em>(1), 54–65. (<a
href="https://doi.org/10.1038/s41588-022-01261-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of the genes and processes mediating genetic association signals for complex diseases represents a major challenge. As many of the genetic signals for type 2 diabetes (T2D) exert their effects through pancreatic islet-cell dysfunction, we performed a genome-wide pooled CRISPR loss-of-function screen in a human pancreatic beta cell line. We assessed the regulation of insulin content as a disease-relevant readout of beta cell function and identified 580 genes influencing this phenotype. Integration with genetic and genomic data provided experimental support for 20 candidate T2D effector transcripts including the autophagy receptor CALCOCO2. Loss of CALCOCO2 was associated with distorted mitochondria, less proinsulin-containing immature granules and accumulation of autophagosomes upon inhibition of late-stage autophagy. Carriers of T2D-associated variants at the CALCOCO2 locus further displayed altered insulin secretion. Our study highlights how cellular screens can augment existing multi-omic efforts to support mechanistic understanding and provide evidence for causal effects at genome-wide association studies loci. A genome-wide CRISPR knockout screen in the human EndoC-βH1 pancreatic beta cell line identifies 580 regulators of intracellular insulin content. Loss of CALCOCO2 perturbs insulin granule homeostasis in pancreatic beta cells.},
  archive      = {J_NG},
  author       = {Rottner, Antje K. and Ye, Yingying and Navarro-Guerrero, Elena and Rajesh, Varsha and Pollner, Alina and Bevacqua, Romina J. and Yang, Jing and Spigelman, Aliya F. and Baronio, Roberta and Bautista, Austin and Thomsen, Soren K. and Lyon, James and Nawaz, Sameena and Smith, Nancy and Wesolowska-Andersen, Agata and Fox, Jocelyn E. Manning and Sun, Han and Kim, Seung K. and Ebner, Daniel and MacDonald, Patrick E. and Gloyn, Anna L.},
  doi          = {10.1038/s41588-022-01261-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {54-65},
  shortjournal = {Nat. Genet.},
  title        = {A genome-wide CRISPR screen identifies CALCOCO2 as a regulator of beta cell function influencing type 2 diabetes risk},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genomic atlas of the plasma metabolome prioritizes
metabolites implicated in human diseases. <em>NG</em>, <em>55</em>(1),
44–53. (<a href="https://doi.org/10.1038/s41588-022-01270-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metabolic processes can influence disease risk and provide therapeutic targets. By conducting genome-wide association studies of 1,091 blood metabolites and 309 metabolite ratios, we identified associations with 690 metabolites at 248 loci and associations with 143 metabolite ratios at 69 loci. Integrating metabolite-gene and gene expression information identified 94 effector genes for 109 metabolites and 48 metabolite ratios. Using Mendelian randomization (MR), we identified 22 metabolites and 20 metabolite ratios having estimated causal effect on 12 traits and diseases, including orotate for estimated bone mineral density, α-hydroxyisovalerate for body mass index and ergothioneine for inflammatory bowel disease and asthma. We further measured the orotate level in a separate cohort and demonstrated that, consistent with MR, orotate levels were positively associated with incident hip fractures. This study provides a valuable resource describing the genetic architecture of metabolites and delivers insights into their roles in common diseases, thereby offering opportunities for therapeutic targets. Genome-wide association studies comprising 1,091 metabolites and 309 metabolite ratios in 8,299 individuals from the Canadian Longitudinal Study on Aging provide insights into the genetic architecture of metabolites and their role in human diseases.},
  archive      = {J_NG},
  author       = {Chen, Yiheng and Lu, Tianyuan and Pettersson-Kymmer, Ulrika and Stewart, Isobel D. and Butler-Laporte, Guillaume and Nakanishi, Tomoko and Cerani, Agustin and Liang, Kevin Y. H. and Yoshiji, Satoshi and Willett, Julian Daniel Sunday and Su, Chen-Yang and Raina, Parminder and Greenwood, Celia M. T. and Farjoun, Yossi and Forgetta, Vincenzo and Langenberg, Claudia and Zhou, Sirui and Ohlsson, Claes and Richards, J. Brent},
  doi          = {10.1038/s41588-022-01270-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {44-53},
  shortjournal = {Nat. Genet.},
  title        = {Genomic atlas of the plasma metabolome prioritizes metabolites implicated in human diseases},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Human genetic diversity alters off-target outcomes of
therapeutic gene editing. <em>NG</em>, <em>55</em>(1), 34–43. (<a
href="https://doi.org/10.1038/s41588-022-01257-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR gene editing holds great promise to modify DNA sequences in somatic cells to treat disease. However, standard computational and biochemical methods to predict off-target potential focus on reference genomes. We developed an efficient tool called CRISPRme that considers single-nucleotide polymorphism (SNP) and indel genetic variants to nominate and prioritize off-target sites. We tested the software with a BCL11A enhancer targeting guide RNA (gRNA) showing promise in clinical trials for sickle cell disease and β-thalassemia and found that the top candidate off-target is produced by an allele common in African-ancestry populations (MAF 4.5%) that introduces a protospacer adjacent motif (PAM) sequence. We validated that SpCas9 generates strictly allele-specific indels and pericentric inversions in CD34+ hematopoietic stem and progenitor cells (HSPCs), although high-fidelity Cas9 mitigates this off-target. This report illustrates how genetic variants should be considered as modifiers of gene editing outcomes. We expect that variant-aware off-target assessment will become integral to therapeutic genome editing evaluation and provide a powerful approach for comprehensive off-target nomination. CRISPRme is an off-target nomination tool that accounts for human genetic diversity. Ancestry-dependent allele-specific off-target edits can occur with therapies currently in clinical trials, highlighting the importance of genetic variation-aware assessment.},
  archive      = {J_NG},
  author       = {Cancellieri, Samuele and Zeng, Jing and Lin, Linda Yingqi and Tognon, Manuel and Nguyen, My Anh and Lin, Jiecong and Bombieri, Nicola and Maitland, Stacy A. and Ciuculescu, Marioara-Felicia and Katta, Varun and Tsai, Shengdar Q. and Armant, Myriam and Wolfe, Scot A. and Giugno, Rosalba and Bauer, Daniel E. and Pinello, Luca},
  doi          = {10.1038/s41588-022-01257-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {34-43},
  shortjournal = {Nat. Genet.},
  title        = {Human genetic diversity alters off-target outcomes of therapeutic gene editing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SARS-CoV-2 diagnostic testing rates determine the
sensitivity of genomic surveillance programs. <em>NG</em>,
<em>55</em>(1), 26–33. (<a
href="https://doi.org/10.1038/s41588-022-01267-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first step in SARS-CoV-2 genomic surveillance is testing to identify people who are infected. However, global testing rates are falling as we emerge from the acute health emergency and remain low in many low- and middle-income countries (mean = 27 tests per 100,000 people per day). We simulated COVID-19 epidemics in a prototypical low- and middle-income country to investigate how testing rates, sampling strategies and sequencing proportions jointly impact surveillance outcomes, and showed that low testing rates and spatiotemporal biases delay time to detection of new variants by weeks to months and can lead to unreliable estimates of variant prevalence, even when the proportion of samples sequenced is increased. Accordingly, investments in wider access to diagnostics to support testing rates of approximately 100 tests per 100,000 people per day could enable more timely detection of new variants and reliable estimates of variant prevalence. The performance of global SARS-CoV-2 genomic surveillance programs is fundamentally limited by access to diagnostic testing. The PATAT model is used to simulate SARS-CoV-2 epidemics in low- and middle-income countries, finding that diagnostic testing rates and proportions of viruses sequenced underpin timely and accurate novel variant virus detection.},
  archive      = {J_NG},
  author       = {Han, Alvin X. and Toporowski, Amy and Sacks, Jilian A. and Perkins, Mark D. and Briand, Sylvie and van Kerkhove, Maria and Hannay, Emma and Carmona, Sergio and Rodriguez, Bill and Parker, Edyth and Nichols, Brooke E. and Russell, Colin A.},
  doi          = {10.1038/s41588-022-01267-w},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {26-33},
  shortjournal = {Nat. Genet.},
  title        = {SARS-CoV-2 diagnostic testing rates determine the sensitivity of genomic surveillance programs},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multimodal single-cell and whole-genome sequencing of small,
frozen clinical specimens. <em>NG</em>, <em>55</em>(1), 19–25. (<a
href="https://doi.org/10.1038/s41588-022-01268-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell genomics enables dissection of tumor heterogeneity and molecular underpinnings of drug response at an unprecedented resolution1–11. However, broad clinical application of these methods remains challenging, due to several practical and preanalytical challenges that are incompatible with typical clinical care workflows, namely the need for relatively large, fresh tissue inputs. In the present study, we show that multimodal, single-nucleus (sn)RNA/T cell receptor (TCR) sequencing, spatial transcriptomics and whole-genome sequencing (WGS) are feasible from small, frozen tissues that approximate routinely collected clinical specimens (for example, core needle biopsies). Compared with data from sample-matched fresh tissue, we find a similar quality in the biological outputs of snRNA/TCR-seq data, while reducing artifactual signals and compositional biases introduced by fresh tissue processing. Profiling sequentially collected melanoma samples from a patient treated in the KEYNOTE-001 trial12, we resolved cellular, genomic, spatial and clonotype dynamics that represent molecular patterns of heterogeneous intralesional evolution during anti-programmed cell death protein 1 therapy. To demonstrate applicability to banked biospecimens of rare diseases13, we generated a single-cell atlas of uveal melanoma liver metastasis with matched WGS data. These results show that single-cell genomics from archival, clinical specimens is feasible and provides a framework for translating these methods more broadly to the clinical arena. High-quality multimodal single-cell, bulk and spatial genomics data are prepared from low-input, frozen needle biopsy specimens collected during routine clinical procedures.},
  archive      = {J_NG},
  author       = {Wang, Yiping and Fan, Joy Linyue and Melms, Johannes C. and Amin, Amit Dipak and Georgis, Yohanna and Barrera, Irving and Ho, Patricia and Tagore, Somnath and Abril-Rodríguez, Gabriel and He, Siyu and Jin, Yinuo and Biermann, Jana and Hofree, Matan and Caprio, Lindsay and Berhe, Simon and Khan, Shaheer A. and Henick, Brian S. and Ribas, Antoni and Macosko, Evan Z. and Chen, Fei and Taylor, Alison M. and Schwartz, Gary K. and Carvajal, Richard D. and Azizi, Elham and Izar, Benjamin},
  doi          = {10.1038/s41588-022-01268-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {19-25},
  shortjournal = {Nat. Genet.},
  title        = {Multimodal single-cell and whole-genome sequencing of small, frozen clinical specimens},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reply to: Genotype by sex interactions in ankylosing
spondylitis. <em>NG</em>, <em>55</em>(1), 17–18. (<a
href="https://doi.org/10.1038/s41588-022-01251-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Bernabeu, Elena and Rawlik, Konrad and Canela-Xandri, Oriol and Talenti, Andrea and Prendergast, James and Tenesa, Albert},
  doi          = {10.1038/s41588-022-01251-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {17-18},
  shortjournal = {Nat. Genet.},
  title        = {Reply to: Genotype by sex interactions in ankylosing spondylitis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genotype by sex interactions in ankylosing spondylitis.
<em>NG</em>, <em>55</em>(1), 14–16. (<a
href="https://doi.org/10.1038/s41588-022-01250-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Zhixiu and McRae, Allan F. and Wang, Geng and Ellis, Jonathan J. and Whyte, Jessica and Kenna, Tony J. and Brown, Matthew A. and Evans, David M.},
  doi          = {10.1038/s41588-022-01250-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {14-16},
  shortjournal = {Nat. Genet.},
  title        = {Genotype by sex interactions in ankylosing spondylitis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). DNA methylation provides molecular links underlying complex
traits. <em>NG</em>, <em>55</em>(1), 12–13. (<a
href="https://doi.org/10.1038/s41588-022-01249-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We profiled human DNA methylation for 987 GTEx samples across nine tissues and characterized how genetic regulation of the methylome, compared with the transcriptome, contributes to GWAS phenotypes. This resource contributes to our understanding of molecular regulatory mechanisms in human tissues and their effects on complex traits.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01249-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {12-13},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation provides molecular links underlying complex traits},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial atlas reveals insight into lung immunity.
<em>NG</em>, <em>55</em>(1), 10–11. (<a
href="https://doi.org/10.1038/s41588-022-01244-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cell types of the lung enable gas exchange and protect against infection. Our spatial atlas of the human lung and airways revealed 11 new cell types and mapped their anatomical locations. In particular, we defined the gland-associated immune niche (GAIN), which is involved in fighting respiratory infections.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01244-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {10-11},
  shortjournal = {Nat. Genet.},
  title        = {Spatial atlas reveals insight into lung immunity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Diet modulates genetic effects on longevity. <em>NG</em>,
<em>55</em>(1), 8–9. (<a
href="https://doi.org/10.1038/s41588-022-01247-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is well known that dietary composition affects lifespan, but whether the effects of diet are mediated through interactions with genetics is unknown. By careful tracking of genome-wide allele frequency in Drosophila, we identify hundreds of loci that affect longevity only in the context of a high-sugar diet.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01247-0},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {8-9},
  shortjournal = {Nat. Genet.},
  title        = {Diet modulates genetic effects on longevity},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accounting for diversity in the design of CRISPR-based
therapeutic genome editing. <em>NG</em>, <em>55</em>(1), 6–7. (<a
href="https://doi.org/10.1038/s41588-022-01272-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR cell and gene therapy have been designed largely with respect to a single reference human genome. A new study reveals how human genetic diversity could lead to off-target effects and presents a new tool to identify these risks.},
  archive      = {J_NG},
  author       = {Saha, Krishanu},
  doi          = {10.1038/s41588-022-01272-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {6-7},
  shortjournal = {Nat. Genet.},
  title        = {Accounting for diversity in the design of CRISPR-based therapeutic genome editing},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identifying type 2 diabetes risk genes by β-cell CRISPR
screening. <em>NG</em>, <em>55</em>(1), 4–5. (<a
href="https://doi.org/10.1038/s41588-022-01269-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A major challenge in human genetics is the prioritization of causal genes in common complex diseases. A genome-wide CRISPR screen for intracellular insulin content in a human β-cell line has now identified a new candidate gene for type 2 diabetes, demonstrating the utility of this screening approach in β-cells.},
  archive      = {J_NG},
  author       = {Wagner, Bridget K.},
  doi          = {10.1038/s41588-022-01269-8},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {4-5},
  shortjournal = {Nat. Genet.},
  title        = {Identifying type 2 diabetes risk genes by β-cell CRISPR screening},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). 5-hydroxymethylcytosine stabilizes transcription by
preventing aberrant initiation in gene bodies. <em>NG</em>,
<em>55</em>(1), 2–3. (<a
href="https://doi.org/10.1038/s41588-022-01253-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {5-hydroxymethylcytosine (5hmC) accumulates in transcribed gene regions (called ‘gene bodies’) and near enhancers, but its biological role has remained mysterious. A new study demonstrates that 5hmC serves to counteract inappropriate, spurious intragenic transcription in airway smooth muscle cells and by doing so, this DNA base functions in the prevention of chronic inflammation in the lung and an asthma-like phenotype.},
  archive      = {J_NG},
  author       = {Pfeifer, Gerd P.},
  doi          = {10.1038/s41588-022-01253-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2-3},
  shortjournal = {Nat. Genet.},
  title        = {5-hydroxymethylcytosine stabilizes transcription by preventing aberrant initiation in gene bodies},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). NR5A2 activates the zygotic genome. <em>NG</em>,
<em>55</em>(1), 1. (<a
href="https://doi.org/10.1038/s41588-022-01289-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-022-01289-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {NR5A2 activates the zygotic genome},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Genetics of clonal hematopoiesis. <em>NG</em>,
<em>55</em>(1), 1. (<a
href="https://doi.org/10.1038/s41588-022-01290-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-022-01290-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {Genetics of clonal hematopoiesis},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Mapping dog behavior. <em>NG</em>, <em>55</em>(1), 1. (<a
href="https://doi.org/10.1038/s41588-022-01291-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-022-01291-w},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {Mapping dog behavior},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Fanconi anemia-associated signature in cancer. <em>NG</em>,
<em>55</em>(1), 1. (<a
href="https://doi.org/10.1038/s41588-022-01292-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-022-01292-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {Fanconi anemia-associated signature in cancer},
  volume       = {55},
  year         = {2023},
}
</textarea>
</details></li>
</ul>

</body>
</html>
