<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth---31">NMETH - 31</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: A comprehensive human embryo reference
tool using single-cell RNA-sequencing data. <em>NMETH</em>,
<em>22</em>(3), 636. (<a
href="https://doi.org/10.1038/s41592-025-02601-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zhao, Cheng and Plaza Reyes, Alvaro and Schell, John Paul and Weltner, Jere and Ortega, Nicolás M. and Zheng, Yi and Björklund, Åsa K. and Baqué-Vidal, Laura and Sokka, Joonas and Trokovic, Ras and Cox, Brian and Rossant, Janet and Fu, Jianping and Petropoulos, Sophie and Lanner, Fredrik},
  doi          = {10.1038/s41592-025-02601-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {636},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A comprehensive human embryo reference tool using single-cell RNA-sequencing data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Resolving tissue complexity by
multimodal spatial omics modeling with MISO. <em>NMETH</em>,
<em>22</em>(3), 635. (<a
href="https://doi.org/10.1038/s41592-025-02600-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Coleman, Kyle and Schroeder, Amelia and Loth, Melanie and Zhang, Daiwei and Park, Jeong Hwan and Sung, Ji-Youn and Blank, Niklas and Cowan, Alexis J. and Qian, Xuyu and Chen, Jianfeng and Jiang, Jiahui and Yan, Hanying and Samarah, Laith Z. and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Rabinowitz, Joshua D. and Deng, Yanxiang and Lee, Edward B. and Lazar, Alexander and Gao, Jianjun and Furth, Emma E. and Hwang, Tae Hyun and Wang, Linghua and Thaiss, Christoph A. and Hu, Jian and Li, Mingyao},
  doi          = {10.1038/s41592-025-02600-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {635},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Resolving tissue complexity by multimodal spatial omics modeling with MISO},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Arkitekt: Streaming analysis and
real-time workflows for microscopy. <em>NMETH</em>, <em>22</em>(3), 634.
(<a href="https://doi.org/10.1038/s41592-025-02597-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Roos, Johannes and Bancelin, Stéphane and Delaire, Tom and Wilhelmi, Alexander and Levet, Florian and Engelhardt, Maren and Viasnoff, Virgile and Galland, Rémi and Nägerl, U. Valentin and Sibarita, Jean-Baptiste},
  doi          = {10.1038/s41592-025-02597-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {634},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: arkitekt: streaming analysis and real-time workflows for microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A genome-wide atlas of human cell morphology.
<em>NMETH</em>, <em>22</em>(3), 621–633. (<a
href="https://doi.org/10.1038/s41592-024-02537-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key challenge of the modern genomics era is developing empirical data-driven representations of gene function. Here we present the first unbiased morphology-based genome-wide perturbation atlas in human cells, containing three genome-wide genotype–phenotype maps comprising CRISPR–Cas9-based knockouts of &amp;gt;20,000 genes in &amp;gt;30 million cells. Our optical pooled cell profiling platform (PERISCOPE) combines a destainable high-dimensional phenotyping panel (based on Cell Painting) with optical sequencing of molecular barcodes and a scalable open-source analysis pipeline to facilitate massively parallel screening of pooled perturbation libraries. This perturbation atlas comprises high-dimensional phenotypic profiles of individual cells with sufficient resolution to cluster thousands of human genes, reconstruct known pathways and protein–protein interaction networks, interrogate subcellular processes and identify culture media-specific responses. Using this atlas, we identify the poorly characterized disease-associated TMEM251/LYSET as a Golgi-resident transmembrane protein essential for mannose-6-phosphate-dependent trafficking of lysosomal enzymes. In sum, this perturbation atlas and screening platform represents a rich and accessible resource for connecting genes to cellular functions at scale. An optical pooled cell profiling platform (PERISCOPE) based on Cell Painting and optical sequencing of molecular barcodes was used to develop the first unbiased genome-wide morphology-based perturbation atlas in human cells.},
  archive      = {J_NMETH},
  author       = {Ramezani, Meraj and Weisbart, Erin and Bauman, Julia and Singh, Avtar and Yong, John and Lozada, Maria and Way, Gregory P. and Kavari, Sanam L. and Diaz, Celeste and Leardini, Eddy and Jetley, Gunjan and Pagnotta, Jenlu and Haghighi, Marzieh and Batista, Thiago M. and Pérez-Schindler, Joaquín and Claussnitzer, Melina and Singh, Shantanu and Cimini, Beth A. and Blainey, Paul C. and Carpenter, Anne E. and Jan, Calvin H. and Neal, James T.},
  doi          = {10.1038/s41592-024-02537-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {621-633},
  shortjournal = {Nat. Meth.},
  title        = {A genome-wide atlas of human cell morphology},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MARBLE: Interpretable representations of neural population
dynamics using geometric deep learning. <em>NMETH</em>, <em>22</em>(3),
612–620. (<a href="https://doi.org/10.1038/s41592-024-02582-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dynamics of neuron populations commonly evolve on low-dimensional manifolds. Thus, we need methods that learn the dynamical processes over neural manifolds to infer interpretable and consistent latent representations. We introduce a representation learning method, MARBLE, which decomposes on-manifold dynamics into local flow fields and maps them into a common latent space using unsupervised geometric deep learning. In simulated nonlinear dynamical systems, recurrent neural networks and experimental single-neuron recordings from primates and rodents, we discover emergent low-dimensional latent representations that parametrize high-dimensional neural dynamics during gain modulation, decision-making and changes in the internal state. These representations are consistent across neural networks and animals, enabling the robust comparison of cognitive computations. Extensive benchmarking demonstrates state-of-the-art within- and across-animal decoding accuracy of MARBLE compared to current representation learning approaches, with minimal user input. Our results suggest that a manifold structure provides a powerful inductive bias to develop decoding algorithms and assimilate data across experiments. MARBLE uses geometric deep learning to map dynamics such as neural activity into a latent representation, which can then be used to decode the neural activity or compare it across systems.},
  archive      = {J_NMETH},
  author       = {Gosztolai, Adam and Peach, Robert L. and Arnaudon, Alexis and Barahona, Mauricio and Vandergheynst, Pierre},
  doi          = {10.1038/s41592-024-02582-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {612-620},
  shortjournal = {Nat. Meth.},
  title        = {MARBLE: Interpretable representations of neural population dynamics using geometric deep learning},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A deep learning pipeline for three-dimensional brain-wide
mapping of local neuronal ensembles in teravoxel light-sheet microscopy.
<em>NMETH</em>, <em>22</em>(3), 600–611. (<a
href="https://doi.org/10.1038/s41592-024-02583-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Teravoxel-scale, cellular-resolution images of cleared rodent brains acquired with light-sheet fluorescence microscopy have transformed the way we study the brain. Realizing the potential of this technology requires computational pipelines that generalize across experimental protocols and map neuronal activity at the laminar and subpopulation-specific levels, beyond atlas-defined regions. Here, we present artficial intelligence-based cartography of ensembles (ACE), an end-to-end pipeline that employs three-dimensional deep learning segmentation models and advanced cluster-wise statistical algorithms, to enable unbiased mapping of local neuronal activity and connectivity. Validation against state-of-the-art segmentation and detection methods on unseen datasets demonstrated ACE’s high generalizability and performance. Applying ACE in two distinct neurobiological contexts, we discovered subregional effects missed by existing atlas-based analyses and showcase ACE’s ability to reveal localized or laminar neuronal activity brain-wide. Our open-source pipeline enables whole-brain mapping of neuronal ensembles at a high level of precision across a wide range of neuroscientific applications. The ACE pipeline utilized deep learning and advanced statistics for mapping neural activity at a granular level that is independent of atlas-defined regions.},
  archive      = {J_NMETH},
  author       = {Attarpour, Ahmadreza and Osmann, Jonas and Rinaldi, Anthony and Qi, Tianbo and Lal, Neeraj and Patel, Shruti and Rozak, Matthew and Yu, Fengqing and Cho, Newton and Squair, Jordan and McLaurin, JoAnne and Raffiee, Misha and Deisseroth, Karl and Courtine, Gregoire and Ye, Li and Stefanovic, Bojana and Goubran, Maged},
  doi          = {10.1038/s41592-024-02583-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {600-611},
  shortjournal = {Nat. Meth.},
  title        = {A deep learning pipeline for three-dimensional brain-wide mapping of local neuronal ensembles in teravoxel light-sheet microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cellpose3: One-click image restoration for improved cellular
segmentation. <em>NMETH</em>, <em>22</em>(3), 592–599. (<a
href="https://doi.org/10.1038/s41592-025-02595-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generalist methods for cellular segmentation have good out-of-the-box performance on a variety of image types; however, existing methods struggle for images that are degraded by noise, blurring or undersampling, all of which are common in microscopy. We focused the development of Cellpose3 on addressing these cases and here we demonstrate substantial out-of-the-box gains in segmentation and image quality for noisy, blurry and undersampled images. Unlike previous approaches that train models to restore pixel values, we trained Cellpose3 to output images that are well segmented by a generalist segmentation model, while maintaining perceptual similarity to the target images. Furthermore, we trained the restoration models on a large, varied collection of datasets, thus ensuring good generalization to user images. We provide these tools as ‘one-click’ buttons inside the graphical interface of Cellpose as well as in the Cellpose API. Cellpose3 employs deep-learning-based approaches for image restoration to improve cellular segmentation and shows strong generalized performance even on images degraded by noise, blurring or undersampling.},
  archive      = {J_NMETH},
  author       = {Stringer, Carsen and Pachitariu, Marius},
  doi          = {10.1038/s41592-025-02595-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {592-599},
  shortjournal = {Nat. Meth.},
  title        = {Cellpose3: One-click image restoration for improved cellular segmentation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Segment anything for microscopy. <em>NMETH</em>,
<em>22</em>(3), 579–591. (<a
href="https://doi.org/10.1038/s41592-024-02580-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate segmentation of objects in microscopy images remains a bottleneck for many researchers despite the number of tools developed for this purpose. Here, we present Segment Anything for Microscopy (μSAM), a tool for segmentation and tracking in multidimensional microscopy data. It is based on Segment Anything, a vision foundation model for image segmentation. We extend it by fine-tuning generalist models for light and electron microscopy that clearly improve segmentation quality for a wide range of imaging conditions. We also implement interactive and automatic segmentation in a napari plugin that can speed up diverse segmentation tasks and provides a unified solution for microscopy annotation across different microscopy modalities. Our work constitutes the application of vision foundation models in microscopy, laying the groundwork for solving image analysis tasks in this domain with a small set of powerful deep learning models. Segment Anything for Microscopy (μSAM) builds on the vision foundation model Segment Anything for high-quality image segmentation over a wide range of imaging conditions including light and electron microscopy.},
  archive      = {J_NMETH},
  author       = {Archit, Anwai and Freckmann, Luca and Nair, Sushmita and Khalid, Nabeel and Hilt, Paul and Rajashekar, Vikas and Freitag, Marei and Teuber, Carolin and Buckley, Genevieve and von Haaren, Sebastian and Gupta, Sagnik and Dengel, Andreas and Ahmed, Sheraz and Pape, Constantin},
  doi          = {10.1038/s41592-024-02580-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {579-591},
  shortjournal = {Nat. Meth.},
  title        = {Segment anything for microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Imaging of cellular dynamics from a whole organism to
subcellular scale with self-driving, multiscale microscopy.
<em>NMETH</em>, <em>22</em>(3), 569–578. (<a
href="https://doi.org/10.1038/s41592-025-02598-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most biological processes, from development to pathogenesis, span multiple time and length scales. While light-sheet fluorescence microscopy has become a fast and efficient method for imaging organisms, cells and subcellular dynamics, simultaneous observations across all these scales have remained challenging. Moreover, continuous high-resolution imaging inside living organisms has mostly been limited to a few hours, as regions of interest quickly move out of view due to sample movement and growth. Here, we present a self-driving, multiresolution light-sheet microscope platform controlled by custom Python-based software, to simultaneously observe and quantify subcellular dynamics in the context of entire organisms in vitro and in vivo over hours of imaging. We apply the platform to the study of developmental processes, cancer invasion and metastasis, and we provide quantitative multiscale analysis of immune–cancer cell interactions in zebrafish xenografts. A self-driving multiresolution light-sheet microscope enables the simultaneous observation and quantification of cellular and subcellular dynamics in the context of intact and developing organisms over many hours of imaging.},
  archive      = {J_NMETH},
  author       = {Daetwyler, Stephan and Mazloom-Farsibaf, Hanieh and Zhou, Felix Y. and Segal, Dagan and Sapoznik, Etai and Chen, Bingying and Westcott, Jill M. and Brekken, Rolf A. and Danuser, Gaudenz and Fiolka, Reto},
  doi          = {10.1038/s41592-025-02598-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {569-578},
  shortjournal = {Nat. Meth.},
  title        = {Imaging of cellular dynamics from a whole organism to subcellular scale with self-driving, multiscale microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oblique line scan illumination enables expansive, accurate
and sensitive single-protein measurements in solution and in living
cells. <em>NMETH</em>, <em>22</em>(3), 559–568. (<a
href="https://doi.org/10.1038/s41592-025-02594-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An ideal tool for the study of cellular biology would enable the measure of molecular activity nondestructively within living cells. Single-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells but have historically been limited by a necessary tradeoff between spatiotemporal resolution and throughput. Here we address these limitations using oblique line scan (OLS), a robust single-objective light-sheet-based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a large field of view. We show that OLS can be used to capture protein motion up to 14 μm2 s−1 in living cells. We further extend the utility of OLS with in-solution SMT for single-molecule measurement of ligand–protein interactions and disruption of protein–protein interactions using purified proteins. We illustrate the versatility of OLS by showcasing two-color SMT, STORM and single-molecule fluorescence recovery after photobleaching. OLS paves the way for robust, high-throughput, single-molecule investigations of protein function required for basic research, drug screening and systems biology studies. Oblique line scan microscopy achieves nanoscale spatial and sub-millisecond temporal resolution across a large field of view, enabling improved and robust single-molecule biophysical measurements and single-molecule tracking in both cells and solution.},
  archive      = {J_NMETH},
  author       = {Driouchi, Amine and Bretan, Mason and Davis, Brynmor J. and Heckert, Alec and Seeger, Markus and Silva, Maité Bradley and Forrest, William S. R. and Hsiung, Jessica and Tan, Jiongyi and Yang, Hongli and McSwiggen, David T. and Song, Linda and Sule, Askhay and Abaie, Behnam and Chen, Hanzhe and Chhun, Bryant and Conroy, Brianna and Elliott, Liam A. and Gonzalez, Eric and Ilkov, Fedor and Isaacs, Joshua and Labaria, George and Lagana, Michelle and Larsen, DeLaine D. and Margolin, Brian and Nguyen, Mai K. and Park, Eugene and Rine, Jeremy and Tang, Yangzhong and Vana, Martin and Wilkey, Andrew and Zhang, Zhengjian and Basham, Stephen and Ho, Jaclyn J. and Johnson, Stephanie and Klammer, Aaron A. and Lin, Kevin and Darzacq, Xavier and Betzig, Eric and Berman, Russell T. and Anderson, Daniel J.},
  doi          = {10.1038/s41592-025-02594-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {559-568},
  shortjournal = {Nat. Meth.},
  title        = {Oblique line scan illumination enables expansive, accurate and sensitive single-protein measurements in solution and in living cells},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super-photostable organic dye for long-term live-cell
single-protein imaging. <em>NMETH</em>, <em>22</em>(3), 550–558. (<a
href="https://doi.org/10.1038/s41592-024-02584-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organic dyes play a crucial role in live-cell imaging because of their advantageous properties, such as photostability and high brightness. Here we introduce a super-photostable and bright organic dye, Phoenix Fluor 555 (PF555), which exhibits an order-of-magnitude longer photobleaching lifetime than conventional organic dyes without the requirement of any anti-photobleaching additives. PF555 is an asymmetric cyanine structure in which, on one side, the indole in the conventional Cyanine-3 is substituted with 3-oxo-quinoline. PF555 provides a powerful tool for long-term live-cell single-molecule imaging, as demonstrated by the imaging of the dynamic single-molecule interactions of the epidermal growth factor receptor with clathrin-coated structures on the plasma membrane of a live cell under physiological conditions. Phoenix Fluor 555 (PF555) is a bright dye with an exceptional, order-of-magnitude longer photobleaching lifetime than conventional organic dyes that enables extended live-cell single-molecule imaging without anti-photobleaching additives.},
  archive      = {J_NMETH},
  author       = {Kim, Do-Hyeon and Triet, Hong Minh and Lee, Sun Hyeok and Jazani, Sina and Jang, Seongjae and Abedi, Syed Ali Abbas and Liu, Xiaogang and Seo, Jongcheol and Ha, Taekjip and Chang, Young-Tae and Ryu, Sung Ho},
  doi          = {10.1038/s41592-024-02584-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {550-558},
  shortjournal = {Nat. Meth.},
  title        = {Super-photostable organic dye for long-term live-cell single-protein imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A temperature-inducible protein module for control of
mammalian cell fate. <em>NMETH</em>, <em>22</em>(3), 539–549. (<a
href="https://doi.org/10.1038/s41592-024-02572-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inducible protein switches are currently limited for use in tissues and organisms because common inducers cannot be controlled with precision in space and time in optically dense settings. Here, we introduce a protein that can be reversibly toggled with a small change in temperature, a stimulus that is both penetrant and dynamic. This protein, called Melt (Membrane localization using temperature) oligomerizes and translocates to the plasma membrane when temperature is lowered. We generated a library of Melt variants with switching temperatures ranging from 30 °C to 40 °C, including two that operate at and above 37 °C. Melt was a highly modular actuator of cell function, permitting thermal control over diverse processes including signaling, proteolysis, nuclear shuttling, cytoskeletal rearrangements and cell death. Finally, Melt permitted thermal control of cell death in a mouse model of human cancer. Melt represents a versatile thermogenetic module for straightforward, non-invasive and spatiotemporally defined control of mammalian cells with broad potential for biotechnology and biomedicine. The Melt (Membrane localization using temperature) protein translocates to the plasma membrane upon temperature shift. Melt variants with a range of switching temperatures enable straightforward thermogenetic control of diverse cellular processes.},
  archive      = {J_NMETH},
  author       = {Benman, William and Huang, Zikang and Iyengar, Pavan and Wilde, Delaney and Mumford, Thomas R. and Bugaj, Lukasz J.},
  doi          = {10.1038/s41592-024-02572-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {539-549},
  shortjournal = {Nat. Meth.},
  title        = {A temperature-inducible protein module for control of mammalian cell fate},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Resolving tissue complexity by multimodal spatial omics
modeling with MISO. <em>NMETH</em>, <em>22</em>(3), 530–538. (<a
href="https://doi.org/10.1038/s41592-024-02574-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial molecular profiling has provided biomedical researchers valuable opportunities to better understand the relationship between cellular localization and tissue function. Effectively modeling multimodal spatial omics data is crucial for understanding tissue complexity and underlying biology. Furthermore, improvements in spatial resolution have led to the advent of technologies that can generate spatial molecular data with subcellular resolution, requiring the development of computationally efficient methods that can handle the resulting large-scale datasets. MISO (MultI-modal Spatial Omics) is a versatile algorithm for feature extraction and clustering, capable of integrating multiple modalities from diverse spatial omics experiments with high spatial resolution. Its effectiveness is demonstrated across various datasets, encompassing gene expression, protein expression, epigenetics, metabolomics and tissue histology modalities. MISO outperforms existing methods in identifying biologically relevant spatial domains, representing a substantial advancement in multimodal spatial omics analysis. Moreover, MISO’s computational efficiency ensures its scalability to handle large-scale datasets generated by subcellular resolution spatial omics technologies. MISO (MultI-modal Spatial Omics) integrates two or more spatial omics modalities, despite differences in data quality and spatial resolution for improved feature extraction and clustering to reveal biologically meaningful tissue organization.},
  archive      = {J_NMETH},
  author       = {Coleman, Kyle and Schroeder, Amelia and Loth, Melanie and Zhang, Daiwei and Park, Jeong Hwan and Sung, Ji-Youn and Blank, Niklas and Cowan, Alexis J. and Qian, Xuyu and Chen, Jianfeng and Jiang, Jiahui and Yan, Hanying and Samarah, Laith Z. and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Rabinowitz, Joshua D. and Deng, Yanxiang and Lee, Edward B. and Lazar, Alexander and Gao, Jianjun and Furth, Emma E. and Hwang, Tae Hyun and Wang, Linghua and Thaiss, Christoph A. and Hu, Jian and Li, Mingyao},
  doi          = {10.1038/s41592-024-02574-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {530-538},
  shortjournal = {Nat. Meth.},
  title        = {Resolving tissue complexity by multimodal spatial omics modeling with MISO},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiplexed spatial mapping of chromatin features,
transcriptome and proteins in tissues. <em>NMETH</em>, <em>22</em>(3),
520–529. (<a href="https://doi.org/10.1038/s41592-024-02576-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The phenotypic and functional states of cells are modulated by a complex interactive molecular hierarchy of multiple omics layers, involving the genome, epigenome, transcriptome, proteome and metabolome. Spatial omics approaches have enabled the study of these layers in tissue context but are often limited to one or two modalities, offering an incomplete view of cellular identity. Here we present spatial-Mux-seq, a multimodal spatial technology that allows simultaneous profiling of five different modalities: two histone modifications, chromatin accessibility, whole transcriptome and a panel of proteins at tissue scale and cellular level in a spatially resolved manner. We applied this technology to mouse embryos and mouse brains, generating detailed multimodal tissue maps that identified more cell types and states compared to unimodal data. This analysis uncovered spatiotemporal relationships among histone modifications, chromatin accessibility, gene expression and protein levels during neuron differentiation, and revealed a radial glia niche with spatially dynamic epigenetic signals. Collectively, the spatial multi-omics approach heralds a new era for characterizing tissue and cellular heterogeneity that single-modality studies alone could not reveal. Spatial-Mux-seq offers a multimodal spatial platform capable of profiling multiple molecular modalities, including the transcriptome, chromatin accessibility, histone modifications and targeted proteins.},
  archive      = {J_NMETH},
  author       = {Guo, Pengfei and Mao, Liran and Chen, Yufan and Lee, Chin Nien and Cardilla, Angelysia and Li, Mingyao and Bartosovic, Marek and Deng, Yanxiang},
  doi          = {10.1038/s41592-024-02576-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {520-529},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed spatial mapping of chromatin features, transcriptome and proteins in tissues},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Challenging the astral mass analyzer to quantify up to 5,300
proteins per single cell at unseen accuracy to uncover cellular
heterogeneity. <em>NMETH</em>, <em>22</em>(3), 510–519. (<a
href="https://doi.org/10.1038/s41592-024-02559-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite significant advancements in sample preparation, instrumentation and data analysis, single-cell proteomics is currently limited by proteomic depth and quantitative performance. Here we demonstrate highly improved depth of proteome coverage as well as accuracy and precision for quantification of ultra-low input amounts. Using a tailored library, we identify up to 7,400 protein groups from as little as 250 pg of HeLa cell peptides at a throughput of 50 samples per day. Using a two-proteome mix, we check for optimal parameters of quantification and show that fold change differences of 2 can still be successfully determined at single-cell-level inputs. Eventually, we apply our workflow to A549 cells, yielding a proteome coverage ranging from 1,801 to a maximum of &amp;gt;5,300 protein groups from a single cell depending on cell size and search strategy used, which allows for the study of dependencies between cell size and cell cycle phase. Additionally, our workflow enables us to distinguish between in vitro analogs of two human blastocyst lineages: naive human pluripotent stem cells (epiblast) and trophectoderm-like cells. Our data harmoniously align with transcriptomic data, indicating that single-cell proteomics possesses the capability to identify biologically relevant differences within the blastocyst. An ultra-low-input single-cell proteomic workflow is optimized to yield maximum proteome coverage with high accuracy and precision on the Orbitrap Astral mass spectrometer.},
  archive      = {J_NMETH},
  author       = {Bubis, Julia A. and Arrey, Tabiwang N. and Damoc, Eugen and Delanghe, Bernard and Slovakova, Jana and Sommer, Theresa M. and Kagawa, Harunobu and Pichler, Peter and Rivron, Nicolas and Mechtler, Karl and Matzinger, Manuel},
  doi          = {10.1038/s41592-024-02559-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {510-519},
  shortjournal = {Nat. Meth.},
  title        = {Challenging the astral mass analyzer to quantify up to 5,300 proteins per single cell at unseen accuracy to uncover cellular heterogeneity},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhanced sensitivity and scalability with a chip-tip
workflow enables deep single-cell proteomics. <em>NMETH</em>,
<em>22</em>(3), 499–509. (<a
href="https://doi.org/10.1038/s41592-024-02558-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell proteomics (SCP) promises to revolutionize biomedicine by providing an unparalleled view of the proteome in individual cells. Here, we present a high-sensitivity SCP workflow named Chip-Tip, identifying &amp;gt;5,000 proteins in individual HeLa cells. It also facilitated direct detection of post-translational modifications in single cells, making the need for specific post-translational modification-enrichment unnecessary. Our study demonstrates the feasibility of processing up to 120 label-free SCP samples per day. An optimized tissue dissociation buffer enabled effective single-cell disaggregation of drug-treated cancer cell spheroids, refining overall SCP analysis. Analyzing nondirected human-induced pluripotent stem cell differentiation, we consistently quantified stem cell markers OCT4 and SOX2 in human-induced pluripotent stem cells and lineage markers such as GATA4 (endoderm), HAND1 (mesoderm) and MAP2 (ectoderm) in different embryoid body cells. Our workflow sets a benchmark in SCP for sensitivity and throughput, with broad applications in basic biology and biomedicine for identification of cell type-specific markers and therapeutic targets. Chip-Tip is a label-free quantification-based single-cell proteomics workflow for deep single-cell proteomics, which identifies over 5,000 proteins and 40,000 peptides in single HeLa cells.},
  archive      = {J_NMETH},
  author       = {Ye, Zilu and Sabatier, Pierre and van der Hoeven, Leander and Lechner, Maico Y. and Phlairaharn, Teeradon and Guzman, Ulises H. and Liu, Zhen and Huang, Haoran and Huang, Min and Li, Xiangjun and Hartlmayr, David and Izaguirre, Fabiana and Seth, Anjali and Joshi, Hiren J. and Rodin, Sergey and Grinnemo, Karl-Henrik and Hørning, Ole B. and Bekker-Jensen, Dorte B. and Bache, Nicolai and Olsen, Jesper V.},
  doi          = {10.1038/s41592-024-02558-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {499-509},
  shortjournal = {Nat. Meth.},
  title        = {Enhanced sensitivity and scalability with a chip-tip workflow enables deep single-cell proteomics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-resolution, noninvasive single-cell lineage tracing in
mice and humans based on DNA methylation epimutations. <em>NMETH</em>,
<em>22</em>(3), 488–498. (<a
href="https://doi.org/10.1038/s41592-024-02567-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vivo lineage tracing holds great potential to reveal fundamental principles of tissue development and homeostasis. However, current lineage tracing in humans relies on extremely rare somatic mutations, which has limited temporal resolution and lineage accuracy. Here, we developed a generic lineage-tracing tool based on frequent epimutations on DNA methylation, enabled by our computational method MethylTree. Using single-cell genome-wide DNA methylation datasets with known lineage and phenotypic labels, MethylTree reconstructed lineage histories at nearly 100% accuracy across different cell types, developmental stages, and species. We demonstrated the epimutation-based single-cell multi-omic lineage tracing in mouse and human blood, where MethylTree recapitulated the differentiation hierarchy in hematopoiesis. Applying MethylTree to human embryos, we revealed early fate commitment at the four-cell stage. In native mouse blood, we identified ~250 clones of hematopoietic stem cells. MethylTree opens the door for high-resolution, noninvasive and multi-omic lineage tracing in humans and beyond. This work presents a computational tool MethylTree to infer cell lineages based on epimutations on DNA methylation.},
  archive      = {J_NMETH},
  author       = {Chen, Mengyang and Fu, Ruijiang and Chen, Yiqian and Li, Li and Wang, Shou-Wen},
  doi          = {10.1038/s41592-024-02567-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {488-498},
  shortjournal = {Nat. Meth.},
  title        = {High-resolution, noninvasive single-cell lineage tracing in mice and humans based on DNA methylation epimutations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scalable co-sequencing of RNA and DNA from individual
nuclei. <em>NMETH</em>, <em>22</em>(3), 477–487. (<a
href="https://doi.org/10.1038/s41592-024-02579-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ideal technology for directly investigating the relationship between genotype and phenotype would analyze both RNA and DNA genome-wide and with single-cell resolution; however, existing tools lack the throughput required for comprehensive analysis of complex tumors and tissues. We introduce a highly scalable method for jointly profiling DNA and expression following nucleosome depletion (DEFND-seq). In DEFND-seq, nuclei are nucleosome-depleted, tagmented and separated into individual droplets for messenger RNA and genomic DNA barcoding. Once nuclei have been depleted of nucleosomes, subsequent steps can be performed using the widely available 10x Genomics droplet microfluidic technology and commercial kits. We demonstrate the production of high-complexity mRNA and gDNA sequencing libraries from thousands of individual nuclei from cell lines, fresh and archived surgical specimens for associating gene expression with both copy number and single-nucleotide variants. Building on a nucleosome-depletion strategy, DEFND-seq utilizes a droplet microfluidic platform to enable high-throughput co-profiling of DNA and RNA in single cells.},
  archive      = {J_NMETH},
  author       = {Olsen, Timothy R. and Talla, Pranay and Sagatelian, Romella K. and Furnari, Julia and Bruce, Jeffrey N. and Canoll, Peter and Zha, Shan and Sims, Peter A.},
  doi          = {10.1038/s41592-024-02579-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {477-487},
  shortjournal = {Nat. Meth.},
  title        = {Scalable co-sequencing of RNA and DNA from individual nuclei},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DeepPrep: An accelerated, scalable and robust pipeline for
neuroimaging preprocessing empowered by deep learning. <em>NMETH</em>,
<em>22</em>(3), 473–476. (<a
href="https://doi.org/10.1038/s41592-025-02599-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuroimaging has entered the era of big data. However, the advancement of preprocessing pipelines falls behind the rapid expansion of data volume, causing substantial computational challenges. Here we present DeepPrep, a pipeline empowered by deep learning and a workflow manager. Evaluated on over 55,000 scans, DeepPrep demonstrates tenfold acceleration, scalability and robustness compared to the state-of-the-art pipeline, thereby meeting the scalability requirements of neuroimaging. DeepPrep is a preprocessing pipeline for functional and structural MRI data from humans. Deep learning-based modules and an efficient workflow allow DeepPrep to handle large datasets.},
  archive      = {J_NMETH},
  author       = {Ren, Jianxun and An, Ning and Lin, Cong and Zhang, Youjia and Sun, Zhenyu and Zhang, Wei and Li, Shiyi and Guo, Ning and Cui, Weigang and Hu, Qingyu and Wang, Weiwei and Wu, Xuehai and Wang, Yinyan and Jiang, Tao and Satterthwaite, Theodore D. and Wang, Danhong and Liu, Hesheng},
  doi          = {10.1038/s41592-025-02599-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {473-476},
  shortjournal = {Nat. Meth.},
  title        = {DeepPrep: An accelerated, scalable and robust pipeline for neuroimaging preprocessing empowered by deep learning},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid and sensitive protein complex alignment with
foldseek-multimer. <em>NMETH</em>, <em>22</em>(3), 469–472. (<a
href="https://doi.org/10.1038/s41592-025-02593-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in computational structure prediction will vastly augment the hundreds of thousands of currently available protein complex structures. Translating these into discoveries requires aligning them, which is computationally prohibitive. Foldseek-Multimer computes complex alignments from compatible chain-to-chain alignments, identified by efficiently clustering their superposition vectors. Foldseek-Multimer is 3–4 orders of magnitudes faster than the gold standard, while producing comparable alignments; this allows it to compare billions of complex pairs in 11 h. Foldseek-Multimer is open-source software available at GitHub via https://github.com/steineggerlab/foldseek/ , https://search.foldseek.com/search/ and the BFMD database. Foldseek-Multimer offers a fast strategy for complex-to-complex alignment to quickly identify compatible sets of chain-to-chain alignments by their superpositions. It can compare billions of complex pairs in 11 h.},
  archive      = {J_NMETH},
  author       = {Kim, Woosub and Mirdita, Milot and Levy Karin, Eli and Gilchrist, Cameron L. M. and Schweke, Hugo and Söding, Johannes and Levy, Emmanuel D. and Steinegger, Martin},
  doi          = {10.1038/s41592-025-02593-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {469-472},
  shortjournal = {Nat. Meth.},
  title        = {Rapid and sensitive protein complex alignment with foldseek-multimer},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Interpreting and comparing neural activity across systems by
geometric deep learning. <em>NMETH</em>, <em>22</em>(3), 467–468. (<a
href="https://doi.org/10.1038/s41592-024-02581-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The MARBLE method addresses a critical challenge in neural population recordings: inferring expressive and interpretable latent representations that are comparable across experiments and animals. It achieves this by explicitly leveraging the low-dimensional structure of neural states through geometric deep learning to learn the dynamical flow fields in neural activity.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02581-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {467-468},
  shortjournal = {Nat. Meth.},
  title        = {Interpreting and comparing neural activity across systems by geometric deep learning},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precision control of cellular functions with a
temperature-sensitive protein. <em>NMETH</em>, <em>22</em>(3), 465–466.
(<a href="https://doi.org/10.1038/s41592-024-02573-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Temperature-sensitive proteins would enable the remote control of cellular functions deep within tissues, although few such proteins have been characterized. Melt is a protein that reversibly clusters and translocates to the membrane in response to small temperature changes, enabling flexible regulation of cellular processes in vitro and in vivo.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02573-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {465-466},
  shortjournal = {Nat. Meth.},
  title        = {Precision control of cellular functions with a temperature-sensitive protein},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MethylTree: Exploring epimutations for accurate and
non-invasive lineage tracing. <em>NMETH</em>, <em>22</em>(3), 463–464.
(<a href="https://doi.org/10.1038/s41592-024-02568-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Non-invasive lineage tracing in humans relies on rare somatic mutations, which have limited throughput and temporal resolution. We developed a computational method, ‘MethylTree’, which uses epimutations on DNA methylation to accurately infer lineages across cell types, developmental stages and species, providing a superior alternative for non-invasive lineage tracing in humans and other organisms.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02568-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {463-464},
  shortjournal = {Nat. Meth.},
  title        = {MethylTree: Exploring epimutations for accurate and non-invasive lineage tracing},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The next generation of in situ multi-omics. <em>NMETH</em>,
<em>22</em>(3), 461–462. (<a
href="https://doi.org/10.1038/s41592-024-02571-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial-Mux-seq adds to a growing portfolio of innovative spatial omics technologies and enables simultaneous profiling of up to five omic modalities in situ.},
  archive      = {J_NMETH},
  author       = {Salla, Maren and Penkert, Klara and Ludwig, Leif S.},
  doi          = {10.1038/s41592-024-02571-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {461-462},
  shortjournal = {Nat. Meth.},
  title        = {The next generation of in situ multi-omics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Entering the era of deep single-cell proteomics.
<em>NMETH</em>, <em>22</em>(3), 459–460. (<a
href="https://doi.org/10.1038/s41592-025-02620-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in single-cell proteomics allow quantification of half of the expressed proteome in an individual cell.},
  archive      = {J_NMETH},
  author       = {Sinn, Ludwig R. and Demichev, Vadim},
  doi          = {10.1038/s41592-025-02620-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {459-460},
  shortjournal = {Nat. Meth.},
  title        = {Entering the era of deep single-cell proteomics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vector choices, vector surprises. <em>NMETH</em>,
<em>22</em>(3), 454–458. (<a
href="https://doi.org/10.1038/s41592-025-02609-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vectors can be the ultimate vehicle for transporting material into cells. Except when they’re not so ultimate.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02609-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {454-458},
  shortjournal = {Nat. Meth.},
  title        = {Vector choices, vector surprises},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural biology at the plasma membrane. <em>NMETH</em>,
<em>22</em>(3), 453. (<a
href="https://doi.org/10.1038/s41592-025-02640-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-025-02640-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {453},
  shortjournal = {Nat. Meth.},
  title        = {Structural biology at the plasma membrane},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Strolling down rats’ memory lane. <em>NMETH</em>,
<em>22</em>(3), 453. (<a
href="https://doi.org/10.1038/s41592-025-02641-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-025-02641-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {453},
  shortjournal = {Nat. Meth.},
  title        = {Strolling down rats’ memory lane},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chemical space exploration with quantum computing.
<em>NMETH</em>, <em>22</em>(3), 452. (<a
href="https://doi.org/10.1038/s41592-025-02638-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02638-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {452},
  shortjournal = {Nat. Meth.},
  title        = {Chemical space exploration with quantum computing},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optimal transport for single-cell genomics. <em>NMETH</em>,
<em>22</em>(3), 452. (<a
href="https://doi.org/10.1038/s41592-025-02639-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-025-02639-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {452},
  shortjournal = {Nat. Meth.},
  title        = {Optimal transport for single-cell genomics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A look back at embryo models. <em>NMETH</em>,
<em>22</em>(3), 449–450. (<a
href="https://doi.org/10.1038/s41592-025-02642-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In 2023, Nature Methods chose methods to model development as our Method of the Year. Here, we catch up on what has happened in this field since.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02642-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {449-450},
  shortjournal = {Nat. Meth.},
  title        = {A look back at embryo models},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
