<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng---28">NG - 28</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: A multilineage screen identifies
actionable synthetic lethal interactions in human cancers. <em>NG</em>,
<em>57</em>(2), 480. (<a
href="https://doi.org/10.1038/s41588-025-02090-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fong, Samson H. and Kuenzi, Brent M. and Mattson, Nicole M. and Lee, John and Sanchez, Kyle and Bojorquez-Gomez, Ana and Ford, Kyle and Munson, Brenton P. and Licon, Katherine and Bergendahl, Sarah and Shen, John Paul and Kreisberg, Jason F. and Mali, Prashant and Hager, Jeffrey H. and White, Michael A. and Ideker, Trey},
  doi          = {10.1038/s41588-025-02090-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {480},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A multilineage screen identifies actionable synthetic lethal interactions in human cancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The ZmCPK39–ZmDi19–ZmPR10 immune module
regulates quantitative resistance to multiple foliar diseases in maize.
<em>NG</em>, <em>57</em>(2), 480. (<a
href="https://doi.org/10.1038/s41588-025-02091-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Zhu, Mang and Zhong, Tao and Xu, Ling and Guo, Chenyu and Zhang, Xiaohui and Liu, Yulin and Zhang, Yan and Li, Yancong and Xie, Zhijian and Liu, Tingting and Jiang, Fuyan and Fan, Xingming and Balint-Kurti, Peter and Xu, Mingliang},
  doi          = {10.1038/s41588-025-02091-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {480},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The ZmCPK39–ZmDi19–ZmPR10 immune module regulates quantitative resistance to multiple foliar diseases in maize},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synchronized long-read genome, methylome, epigenome and
transcriptome profiling resolve a mendelian condition. <em>NG</em>,
<em>57</em>(2), 469–479. (<a
href="https://doi.org/10.1038/s41588-024-02067-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resolving the molecular basis of a Mendelian condition remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion–deletion and structural variant calling and diploid de novo genome assembly. This permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network participant with a chromosome X;13-balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1 and RB1) previously associated with single-gene Mendelian conditions. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four ‘omes’ to resolve. These included fusion transcript formation, enhancer adoption, transcriptional readthrough silencing and inappropriate X-chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes. Simultaneous profiling of the genome, methylome, epigenome and transcriptome using single-molecule chromatin fiber sequencing and multiplexed arrays isoform sequencing identifies the genetic and molecular basis of an undiagnosed Mendelian disease case with an X;13-balanced translocation.},
  archive      = {J_NG},
  author       = {Vollger, Mitchell R. and Korlach, Jonas and Eldred, Kiara C. and Swanson, Elliott and Underwood, Jason G. and Bohaczuk, Stephanie C. and Mao, Yizi and Cheng, Yong-Han H. and Ranchalis, Jane and Blue, Elizabeth E. and Schwarze, Ulrike and Munson, Katherine M. and Saunders, Christopher T. and Wenger, Aaron M. and Allworth, Aimee and Chanprasert, Sirisak and Duerden, Brittney L. and Glass, Ian and Horike-Pyne, Martha and Kim, Michelle and Leppig, Kathleen A. and McLaughlin, Ian J. and Ogawa, Jessica and Rosenthal, Elisabeth A. and Sheppeard, Sam and Sherman, Stephanie M. and Strohbehn, Samuel and Yuen, Amy L. and Stacey, Andrew W. and Reh, Thomas A. and Byers, Peter H. and Bamshad, Michael J. and Hisama, Fuki M. and Jarvik, Gail P. and Sancak, Yasemin and Dipple, Katrina M. and Stergachis, Andrew B.},
  doi          = {10.1038/s41588-024-02067-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {469-479},
  shortjournal = {Nat. Genet.},
  title        = {Synchronized long-read genome, methylome, epigenome and transcriptome profiling resolve a mendelian condition},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A scalable variational inference approach for increased
mixed-model association power. <em>NG</em>, <em>57</em>(2), 461–468. (<a
href="https://doi.org/10.1038/s41588-024-02044-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid growth of modern biobanks is creating new opportunities for large-scale genome-wide association studies (GWASs) and the analysis of complex traits. However, performing GWASs on millions of samples often leads to trade-offs between computational efficiency and statistical power, reducing the benefits of large-scale data collection efforts. We developed Quickdraws, a method that increases association power in quantitative and binary traits without sacrificing computational efficiency, leveraging a spike-and-slab prior on variant effects, stochastic variational inference and graphics processing unit acceleration. We applied Quickdraws to 79 quantitative and 50 binary traits in 405,088 UK Biobank samples, identifying 4.97% and 3.25% more associations than REGENIE and 22.71% and 7.07% more than FastGWA. Quickdraws had costs comparable to REGENIE, FastGWA and SAIGE on the UK Biobank Research Analysis Platform service, while being substantially faster than BOLT-LMM. These results highlight the promise of leveraging machine learning techniques for scalable GWASs without sacrificing power or robustness. Quickdraws is a mixed-model association tool with a noninfinitesimal prior for analyzing binary and quantitative traits, using a scalable variational inference that allows analysis of biobank-scale cohorts.},
  archive      = {J_NG},
  author       = {Loya, Hrushikesh and Kalantzis, Georgios and Cooper, Fergus and Palamara, Pier Francesco},
  doi          = {10.1038/s41588-024-02044-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {461-468},
  shortjournal = {Nat. Genet.},
  title        = {A scalable variational inference approach for increased mixed-model association power},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transcript-specific enrichment enables profiling of rare
cell states via single-cell RNA sequencing. <em>NG</em>, <em>57</em>(2),
451–460. (<a href="https://doi.org/10.1038/s41588-024-02036-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell genomics technologies have accelerated our understanding of cell-state heterogeneity in diverse contexts. Although single-cell RNA sequencing identifies rare populations that express specific marker transcript combinations, traditional flow sorting requires cell surface markers with high-fidelity antibodies, limiting our ability to interrogate these populations. In addition, many single-cell studies require the isolation of nuclei from tissue, eliminating the ability to enrich learned rare cell states based on extranuclear protein markers. In the present report, we addressed these limitations by developing Programmable Enrichment via RNA FlowFISH by sequencing (PERFF-seq), a scalable assay that enables scRNA-seq profiling of subpopulations defined by the abundance of specific RNA transcripts. Across immune populations (n = 184,126 cells) and fresh-frozen and formalin-fixed, paraffin-embedded brain tissue (n = 33,145 nuclei), we demonstrated that programmable sorting logic via RNA-based cytometry can isolate rare cell populations and uncover phenotypic heterogeneity via downstream, high-throughput, single-cell genomics analyses. Programmable Enrichment via RNA FlowFISH by sequencing (PERFF-seq) isolates rare cells based on RNA marker transcripts for single-cell RNA sequencing profiling of complex tissues, with applicability to a broad variety of samples and cell types.},
  archive      = {J_NG},
  author       = {Abay, Tsion and Stickels, Robert R. and Takizawa, Meril T. and Nalbant, Benan N. and Hsieh, Yu-Hsin and Hwang, Sidney and Snopkowski, Catherine and Yu, Kenny Kwok Hei and Abou-Mrad, Zaki and Tabar, Viviane and Howitt, Brooke E. and Ludwig, Leif S. and Chaligné, Ronan and Satpathy, Ansuman T. and Lareau, Caleb A.},
  doi          = {10.1038/s41588-024-02036-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {451-460},
  shortjournal = {Nat. Genet.},
  title        = {Transcript-specific enrichment enables profiling of rare cell states via single-cell RNA sequencing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A barley pan-transcriptome reveals layers of
genotype-dependent transcriptional complexity. <em>NG</em>,
<em>57</em>(2), 441–450. (<a
href="https://doi.org/10.1038/s41588-024-02069-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A pan-transcriptome describes the transcriptional and post-transcriptional consequences of genome diversity from multiple individuals within a species. We developed a barley pan-transcriptome using 20 inbred genotypes representing domesticated barley diversity by generating and analyzing short- and long-read RNA-sequencing datasets from multiple tissues. To overcome single reference bias in transcript quantification, we constructed genotype-specific reference transcript datasets (RTDs) and integrated these into a linear pan-genome framework to create a pan-RTD, allowing transcript categorization as core, shell or cloud. Focusing on the core (expressed in all genotypes), we observed significant transcript abundance variation among tissues and between genotypes driven partly by RNA processing, gene copy number, structural rearrangements and conservation of promotor motifs. Network analyses revealed conserved co-expression module::tissue correlations and frequent functional diversification. To complement the pan-transcriptome, we constructed a comprehensive cultivar (cv.) Morex gene-expression atlas and illustrate how these combined datasets can be used to guide biological inquiry. A long- and short-read barley pan-transcriptome assembled from 20 diverse barley genotypes offers insights into genotype- and tissue-dependent gene expression and function.},
  archive      = {J_NG},
  author       = {Guo, Wenbin and Schreiber, Miriam and Marosi, Vanda B. and Bagnaresi, Paolo and Jørgensen, Morten Egevang and Braune, Katarzyna B. and Chalmers, Ken and Chapman, Brett and Dang, Viet and Dockter, Christoph and Fiebig, Anne and Fincher, Geoffrey B. and Fricano, Agostino and Fuller, John and Haaning, Allison and Haberer, Georg and Himmelbach, Axel and Jayakodi, Murukarthick and Jia, Yong and Kamal, Nadia and Langridge, Peter and Li, Chengdao and Lu, Qiongxian and Lux, Thomas and Mascher, Martin and Mayer, Klaus F. X. and McCallum, Nicola and Milne, Linda and Muehlbauer, Gary J. and Nielsen, Martin T. S. and Padmarasu, Sudharsan and Pedas, Pai Rosager and Pillen, Klaus and Pozniak, Curtis and Rasmussen, Magnus W. and Sato, Kazuhiro and Schmutzer, Thomas and Scholz, Uwe and Schüler, Danuta and Šimková, Hana and Skadhauge, Birgitte and Stein, Nils and Thomsen, Nina W. and Voss, Cynthia and Wang, Penghao and Wonneberger, Ronja and Zhang, Xiao-Qi and Zhang, Guoping and Cattivelli, Luigi and Spannagl, Manuel and Bayer, Micha and Simpson, Craig and Zhang, Runxuan and Waugh, Robbie},
  doi          = {10.1038/s41588-024-02069-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {441-450},
  shortjournal = {Nat. Genet.},
  title        = {A barley pan-transcriptome reveals layers of genotype-dependent transcriptional complexity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The NAT1–bHLH110–CER1/CER1L module regulates heat stress
tolerance in rice. <em>NG</em>, <em>57</em>(2), 427–440. (<a
href="https://doi.org/10.1038/s41588-024-02065-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rice production is facing substantial threats from global warming associated with extreme temperatures. Here we report that modifying a heat stress-induced negative regulator, a negative regulator of thermotolerance 1 (NAT1), increases wax deposition and enhances thermotolerance in rice. We demonstrated that the C2H2 family transcription factor NAT1 directly inhibits bHLH110 expression, and bHLH110 directly promotes the expression of wax biosynthetic genes CER1/CER1L under heat stress conditions. In situ hybridization revealed that both NAT1 and bHLH110 are predominantly expressed in epidermal layers. By using gene-editing technology, we successfully mutated NAT1 to eliminate its inhibitory effects on wax biosynthesis and improved thermotolerance without yield penalty under normal temperature conditions. Field trials further confirmed the potential of NAT1-edited rice to increase seed-setting rate and grain yield. Therefore, our findings shed light on the regulatory mechanisms governing wax biosynthesis under heat stress conditions in rice and provide a strategy to enhance heat resilience through the modification of NAT1. Negative regulator of thermotolerance 1 (NAT1) is identified as a negative regulator of thermotolerance in rice through the NAT1–bHLH110–CER1/CER1L module. Modifying NAT1 by targeted gene editing increases wax deposition and enhances thermotolerance in rice.},
  archive      = {J_NG},
  author       = {Lu, Hai-Ping and Liu, Xue-Huan and Wang, Mei-Jing and Zhu, Qiao-Yun and Lyu, Yu-Shu and Xu, Jian-Hang and Liu, Jian-Xiang},
  doi          = {10.1038/s41588-024-02065-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {427-440},
  shortjournal = {Nat. Genet.},
  title        = {The NAT1–bHLH110–CER1/CER1L module regulates heat stress tolerance in rice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Human subcutaneous and visceral adipocyte atlases uncover
classical and nonclassical adipocytes and depot-specific patterns.
<em>NG</em>, <em>57</em>(2), 413–426. (<a
href="https://doi.org/10.1038/s41588-024-02048-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human adipose depots are functionally distinct. Yet, recent single-nucleus RNA sequencing (snRNA-seq) analyses largely uncovered overlapping or similar cell-type landscapes. We hypothesized that adipocyte subtypes, differentiation trajectories and/or intercellular communication patterns could illuminate this depot similarity–difference gap. For this, we performed snRNA-seq of human subcutaneous or visceral adipose tissues (five or ten samples, respectively). Of 27,665 adipocyte nuclei in both depots, most were ‘classical’, namely enriched in lipid metabolism pathways. However, we also observed ‘nonclassical’ adipocyte subtypes, enriched in immune-related, extracellular matrix deposition (fibrosis), vascularization or angiogenesis or ribosomal and mitochondrial processes. Pseudo-temporal analysis showed a developmental trajectory from adipose progenitor cells to classical adipocytes via nonclassical adipocytes, suggesting that the classical state stems from loss, rather than gain, of specialized functions. Last, intercellular communication routes were consistent with the different inflammatory tone of the two depots. Jointly, these findings provide a high-resolution view into the contribution of cellular composition, differentiation and intercellular communication patterns to human fat depot differences. Single-nucleus RNA sequencing of human visceral and subcutaneous adipose tissues is used to identify adipocyte subpopulations and explore their developmental trajectories and interactions.},
  archive      = {J_NG},
  author       = {Lazarescu, Or and Ziv-Agam, Maya and Haim, Yulia and Hekselman, Idan and Jubran, Juman and Shneyour, Ariel and Muallem, Habib and Zemer, Alon and Rosengarten-Levin, Marina and Kitsberg, Daniel and Levin, Liron and Liberty, Idit F. and Yoel, Uri and Dukhno, Oleg and Adam, Miriam and Braune, Julia and Müller, Claudia and Raulien, Nora and Gericke, Martin and Körner, Antje and Murphy, Rinki and Blüher, Matthias and Habib, Naomi and Rudich, Assaf and Yeger-Lotem, Esti},
  doi          = {10.1038/s41588-024-02048-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {413-426},
  shortjournal = {Nat. Genet.},
  title        = {Human subcutaneous and visceral adipocyte atlases uncover classical and nonclassical adipocytes and depot-specific patterns},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oncofetal reprogramming drives phenotypic plasticity in
WNT-dependent colorectal cancer. <em>NG</em>, <em>57</em>(2), 402–412.
(<a href="https://doi.org/10.1038/s41588-024-02058-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeting cancer stem cells (CSCs) is crucial for effective cancer treatment, yet resistance mechanisms to LGR5+ CSC depletion in WNT-driven colorectal cancer (CRC) remain elusive. In the present study, we revealed that mutant intestinal stem cells (SCs) depart from their canonical identity, traversing a dynamic phenotypic spectrum. This enhanced plasticity is initiated by oncofetal (OnF) reprogramming, driven by YAP and AP-1, with subsequent AP-1 hyperactivation promoting lineage infidelity. The retinoid X receptor serves as a gatekeeper of OnF reprogramming and its deregulation after adenomatous polyposis coli (APC) loss of function establishes an OnF ‘memory’ sustained by YAP and AP-1. Notably, the clinical significance of OnF and LGR5+ states in isolation is constrained by their functional redundancy. Although the canonical LGR5+ state is sensitive to the FOLFIRI regimen, an active OnF program correlates with resistance, supporting its role in driving drug-tolerant states. Targeting this program in combination with the current standard of care is pivotal for achieving effective and durable CRC treatment. Oncofetal (OnF) reprogramming, driven by YAP and AP-1, induces phenotypic plasticity and therapy resistance in WNT-dependent colorectal cancer (CRC). Targeting the OnF state in combination with chemotherapy substantially attenuates tumor growth in mouse models and patient-derived CRC tumoroids.},
  archive      = {J_NG},
  author       = {Mzoughi, Slim and Schwarz, Megan and Wang, Xuedi and Demircioglu, Deniz and Ulukaya, Gulay and Mohammed, Kevin and Zorgati, Habiba and Torre, Denis and Tomalin, Lewis E. and Di Tullio, Federico and Company, Carlos and Dramaretska, Yuliia and Leushacke, Marc and Giotti, Bruno and Lannagan, Tamsin RM and Lozano-Ojalvo, Daniel and Karras, Panagiotis and Vermeulen, Peter B. and Hasson, Dan and Sebra, Robert and Tsankov, Alexander M. and Sansom, Owen J. and Marine, Jean-Christophe and Barker, Nick and Gargiulo, Gaetano and Guccione, Ernesto},
  doi          = {10.1038/s41588-024-02058-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {402-412},
  shortjournal = {Nat. Genet.},
  title        = {Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural polymorphism and diversity of human segmental
duplications. <em>NG</em>, <em>57</em>(2), 390–401. (<a
href="https://doi.org/10.1038/s41588-024-02051-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Segmental duplications (SDs) contribute significantly to human disease, evolution and diversity but have been difficult to resolve at the sequence level. We present a population genetics survey of SDs by analyzing 170 human genome assemblies (from 85 samples representing 38 Africans and 47 non-Africans) in which the majority of autosomal SDs are fully resolved using long-read sequence assembly. Excluding the acrocentric short arms and sex chromosomes, we identify 173.2 Mb of duplicated sequence (47.4 Mb not present in the telomere-to-telomere reference) distinguishing fixed from structurally polymorphic events. We find that intrachromosomal SDs are among the most variable, with rare events mapping near their progenitor sequences. African genomes harbor significantly more intrachromosomal SDs and are more likely to have recently duplicated gene families with higher copy numbers than non-African samples. Comparison to a resource of 563 million full-length isoform sequencing reads identifies 201 novel, potentially protein-coding genes corresponding to these copy number polymorphic SDs. Analysis of 170 human genomes assembled using long-read sequencing provides a map of structural variation within regions of segmental duplication and identifies novel candidate protein-coding genes supported by full-length Iso-Seq reads.},
  archive      = {J_NG},
  author       = {Jeong, Hyeonsoo and Dishuck, Philip C. and Yoo, DongAhn and Harvey, William T. and Munson, Katherine M. and Lewis, Alexandra P. and Kordosky, Jennifer and Garcia, Gage H. and Yilmaz, Feyza and Hallast, Pille and Lee, Charles and Pastinen, Tomi and Eichler, Evan E.},
  doi          = {10.1038/s41588-024-02051-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {390-401},
  shortjournal = {Nat. Genet.},
  title        = {Structural polymorphism and diversity of human segmental duplications},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fine-scale population structure and widespread conservation
of genetic effect sizes between human groups across traits. <em>NG</em>,
<em>57</em>(2), 379–389. (<a
href="https://doi.org/10.1038/s41588-024-02035-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding genetic differences between populations is essential for avoiding confounding in genome-wide association studies and improving polygenic score (PGS) portability. We developed a statistical pipeline to infer fine-scale Ancestry Components and applied it to UK Biobank data. Ancestry Components identify population structure not captured by widely used principal components, improving stratification correction for geographically correlated traits. To estimate the similarity of genetic effect sizes between groups, we developed ANCHOR, which estimates changes in the predictive power of an existing PGS in distinct local ancestry segments. ANCHOR infers highly similar (estimated correlation 0.98 ± 0.07) effect sizes between UK Biobank participants of African and European ancestry for 47 of 53 quantitative phenotypes, suggesting that gene–environment and gene–gene interactions do not play major roles in poor cross-ancestry PGS transferability for these traits in the United Kingdom, and providing optimism that shared causal mutations operate similarly in different populations. This study introduces the concept of Ancestry Components and shows that they can offer improved population stratification correction for geographically correlated traits. By using ancestry-aware polygenic score construction in admixed individuals, the authors find that effect sizes are conserved across ancestry groups.},
  archive      = {J_NG},
  author       = {Hu, Sile and Ferreira, Lino A. F. and Shi, Sinan and Hellenthal, Garrett and Marchini, Jonathan and Lawson, Daniel J. and Myers, Simon R.},
  doi          = {10.1038/s41588-024-02035-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {379-389},
  shortjournal = {Nat. Genet.},
  title        = {Fine-scale population structure and widespread conservation of genetic effect sizes between human groups across traits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multi-omic quantitative trait loci link tandem repeat size
variation to gene regulation in human brain. <em>NG</em>,
<em>57</em>(2), 369–378. (<a
href="https://doi.org/10.1038/s41588-024-02057-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tandem repeat (TR) size variation is implicated in ~50 neurological disorders, yet its impact on gene regulation in the human brain remains largely unknown. In the present study, we quantified the impact of TR size variation on brain gene regulation across distinct molecular phenotypes, based on 4,412 multi-omics samples from 1,597 donors, including 1,586 newly sequenced ones. We identified ~2.2 million TR molecular quantitative trait loci (TR-xQTLs), linking ~139,000 unique TRs to nearby molecular phenotypes, including many known disease-risk TRs, such as the G2C4 expansion in C9orf72 associated with amyotrophic lateral sclerosis. Fine-mapping revealed ~18,700 TRs as potential causal variants. Our in vitro experiments further confirmed the causal and independent regulatory effects of three TRs. Additional colocalization analysis indicated the potential causal role of TR variation in brain-related phenotypes, highlighted by a 3ʹ-UTR TR in NUDT14 linked to cortical surface area and a TG repeat in PLEKHA1, associated with Alzheimer’s disease. Mapping of multi-omic molecular quantitative trait loci associated with tandem repeat size variation in up to 4,412 human brain samples from 1,597 donors offers insights into how these variants affect gene regulation and mediate disease risk.},
  archive      = {J_NG},
  author       = {Cui, Ya and Arnold, Frederick J. and Li, Jason Sheng and Wu, Jie and Wang, Dan and Philippe, Julien and Colwin, Michael R. and Michels, Sebastian and Chen, Chaorong and Sallam, Tamer and Thompson, Leslie M. and La Spada, Albert R. and Li, Wei},
  doi          = {10.1038/s41588-024-02057-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {369-378},
  shortjournal = {Nat. Genet.},
  title        = {Multi-omic quantitative trait loci link tandem repeat size variation to gene regulation in human brain},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cell state-dependent allelic effects and contextual
mendelian randomization analysis for human brain phenotypes.
<em>NG</em>, <em>57</em>(2), 358–368. (<a
href="https://doi.org/10.1038/s41588-024-02050-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression quantitative trait loci are widely used to infer relationships between genes and central nervous system (CNS) phenotypes; however, the effect of brain disease on these inferences is unclear. Using 2,348,438 single-nuclei profiles from 391 disease-case and control brains, we report 13,939 genes whose expression correlated with genetic variation, of which 16.7–40.8% (depending on cell type) showed disease-dependent allelic effects. Across 501 colocalizations for 30 CNS traits, 23.6% had a disease dependency, even after adjusting for disease status. To estimate the unconfounded effect of genes on outcomes, we repeated the analysis using nondiseased brains (n = 183) and reported an additional 91 colocalizations not present in the larger mixed disease and control dataset, demonstrating enhanced interpretation of disease-associated variants. Principled implementation of single-cell Mendelian randomization in control-only brains identified 140 putatively causal gene–trait associations, of which 11 were replicated in the UK Biobank, prioritizing candidate peripheral biomarkers predictive of CNS outcomes. Analysis of single nuclei RNA-sequencing data across eight major brain cell types identifies putative causal associations between cell-type-specific expression and brain phenotypes.},
  archive      = {J_NG},
  author       = {Haglund, Alexander and Zuber, Verena and Abouzeid, Maya and Yang, Yifei and Ko, Jeong Hun and Wiemann, Liv and Otero-Jimenez, Maria and Muhammed, Louwai and Feleke, Rahel and Nott, Alexi and Mills, James D. and Laaniste, Liisi and Gveric, Djordje O. and Clode, Daniel and Babtie, Ann C. and Pagni, Susanna and Bellampalli, Ravishankara and Somani, Alyma and McDade, Karina and Anink, Jasper J. and Mesarosova, Lucia and Fancy, Nurun and Willumsen, Nanet and Smith, Amy and Jackson, Johanna and Alegre-Abarrategui, Javier and Aronica, Eleonora and Matthews, Paul M. and Thom, Maria and Sisodiya, Sanjay M. and Srivastava, Prashant K. and Malhotra, Dheeraj and Bryois, Julien and Bottolo, Leonardo and Johnson, Michael R.},
  doi          = {10.1038/s41588-024-02050-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {358-368},
  shortjournal = {Nat. Genet.},
  title        = {Cell state-dependent allelic effects and contextual mendelian randomization analysis for human brain phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Integrative proteogenomic analysis identifies COL6A3-derived
endotrophin as a mediator of the effect of obesity on coronary artery
disease. <em>NG</em>, <em>57</em>(2), 345–357. (<a
href="https://doi.org/10.1038/s41588-024-02052-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obesity strongly increases the risk of cardiometabolic diseases, yet the underlying mediators of this relationship are not fully understood. Given that obesity strongly influences circulating protein levels, we investigated proteins mediating the effects of obesity on coronary artery disease, stroke and type 2 diabetes. By integrating two-step proteome-wide Mendelian randomization, colocalization, epigenomics and single-cell RNA sequencing, we identified five mediators and prioritized collagen type VI α3 (COL6A3). COL6A3 levels were strongly increased by body mass index and increased coronary artery disease risk. Notably, the carboxyl terminus product of COL6A3, endotrophin, drove this effect. COL6A3 was highly expressed in disease-relevant cell types and tissues. Finally, we found that body fat reduction could reduce plasma levels of COL6A3-derived endotrophin, indicating a tractable way to modify endotrophin levels. In summary, we provide actionable insights into how circulating proteins mediate the effects of obesity on cardiometabolic diseases and prioritize endotrophin as a potential therapeutic target. Two-step Mendelian randomization, combined with multiple layers of omics evidence, implicates COL6A3-derived endotrophin as a mediator of coronary artery disease risk in the context of obesity.},
  archive      = {J_NG},
  author       = {Yoshiji, Satoshi and Lu, Tianyuan and Butler-Laporte, Guillaume and Carrasco-Zanini-Sanchez, Julia and Su, Chen-Yang and Chen, Yiheng and Liang, Kevin and Willett, Julian Daniel Sunday and Wang, Shidong and Adra, Darin and Ilboudo, Yann and Sasako, Takayoshi and Koyama, Satoshi and Nakao, Tetsushi and Forgetta, Vincenzo and Farjoun, Yossi and Zeberg, Hugo and Zhou, Sirui and Marks-Hultström, Michael and Machiela, Mitchell J. and Kaalia, Rama and Dashti, Hesam and Claussnitzer, Melina and Flannick, Jason and Wareham, Nicholas J. and Mooser, Vincent and Timpson, Nicholas J. and Langenberg, Claudia and Richards, J. Brent},
  doi          = {10.1038/s41588-024-02052-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {345-357},
  shortjournal = {Nat. Genet.},
  title        = {Integrative proteogenomic analysis identifies COL6A3-derived endotrophin as a mediator of the effect of obesity on coronary artery disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide association study of prostate-specific antigen
levels in 392,522 men identifies new loci and improves prediction across
ancestry groups. <em>NG</em>, <em>57</em>(2), 334–344. (<a
href="https://doi.org/10.1038/s41588-024-02068-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted a multiancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry, 58,236 African ancestry, 23,546 Hispanic/Latino and 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (P ≤ 5 × 10−8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n = 95,768). Meta-analyzing discovery and replication (n = 392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our genome-wide polygenic risk scores ranged from 11.6% to 16.6% for European ancestry, 5.5% to 9.5% for African ancestry, 13.5% to 18.2% for Hispanic/Latino and 8.6% to 15.3% for Asian ancestry and decreased with increasing age. Midlife genetically adjusted PSA levels were more strongly associated with overall and aggressive prostate cancer than unadjusted PSA levels. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, offering potential to personalize prostate cancer screening. This study shows how including different ancestry groups in a genome-wide association study for prostate-specific antigen levels can improve prostate cancer risk prediction, with implications for population screening.},
  archive      = {J_NG},
  author       = {Hoffmann, Thomas J. and Graff, Rebecca E. and Madduri, Ravi K. and Rodriguez, Alex A. and Cario, Clinton L. and Feng, Karen and Jiang, Yu and Wang, Anqi and Klein, Robert J. and Pierce, Brandon L. and Eggener, Scott and Tong, Lin and Blot, William and Long, Jirong and Goss, Louisa B. and Darst, Burcu F. and Rebbeck, Timothy and Lachance, Joseph and Andrews, Caroline and Adebiyi, Akindele O. and Adusei, Ben and Aisuodionoe-Shadrach, Oseremen I. and Fernandez, Pedro W. and Jalloh, Mohamed and Janivara, Rohini and Chen, Wenlong C. and Mensah, James E. and Agalliu, Ilir and Berndt, Sonja I. and Shelley, John P. and Schaffer, Kerry and Machiela, Mitchell J. and Freedman, Neal D. and Huang, Wen-Yi and Li, Shengchao A. and Goodman, Phyllis J. and Till, Cathee and Thompson, Ian and Lilja, Hans and Ranatunga, Dilrini K. and Presti, Joseph and Van Den Eeden, Stephen K. and Chanock, Stephen J. and Mosley, Jonathan D. and Conti, David V. and Haiman, Christopher A. and Justice, Amy C. and Kachuri, Linda and Witte, John S.},
  doi          = {10.1038/s41588-024-02068-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {334-344},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves prediction across ancestry groups},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Prioritizing effector genes at trait-associated loci using
multimodal evidence. <em>NG</em>, <em>57</em>(2), 323–333. (<a
href="https://doi.org/10.1038/s41588-025-02084-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus-trait associations remains challenging. Here we present the FLAMES (fine-mapped locus assessment model of effector genes) framework, which predicts the most likely effector gene in a locus. FLAMES creates machine learning predictions from biological data linking single-nucleotide polymorphisms to genes, and then evaluates these scores together with gene-centric evidence of convergence of the GWAS signal in functional networks. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication and domain knowledge of molecular traits. We demonstrate that combining single-nucleotide-polymorphism-based and convergence-based modalities outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life. FLAMES is a machine learning approach combining variant fine-mapping, SNP-to-gene annotations and convergence-based gene prioritization scores to identify candidate effector genes at genome-wide associated loci with high accuracy.},
  archive      = {J_NG},
  author       = {Schipper, Marijn and de Leeuw, Christiaan A. and Maciel, Bernardo A. P. C. and Wightman, Douglas P. and Hubers, Nikki and Boomsma, Dorret I. and O’Donovan, Michael C. and Posthuma, Danielle},
  doi          = {10.1038/s41588-025-02084-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {323-333},
  shortjournal = {Nat. Genet.},
  title        = {Prioritizing effector genes at trait-associated loci using multimodal evidence},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In vivo CRISPR–Cas9 genome editing in mice identifies
genetic modifiers of somatic CAG repeat instability in huntington’s
disease. <em>NG</em>, <em>57</em>(2), 314–322. (<a
href="https://doi.org/10.1038/s41588-024-02054-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Huntington’s disease, one of more than 50 inherited repeat expansion disorders1, is a dominantly inherited neurodegenerative disease caused by a CAG expansion in HTT2. Inherited CAG repeat length is the primary determinant of age of onset, with human genetic studies underscoring that the disease is driven by the CAG length-dependent propensity of the repeat to further expand in the brain3–9. Routes to slowing somatic CAG expansion, therefore, hold promise for disease-modifying therapies. Several DNA repair genes, notably in the mismatch repair pathway, modify somatic expansion in Huntington’s disease mouse models10. To identify novel modifiers of somatic expansion, we used CRISPR–Cas9 editing in Huntington’s disease knock-in mice to enable in vivo screening of expansion-modifier candidates at scale. This included testing of Huntington’s disease onset modifier genes emerging from human genome-wide association studies as well as interactions between modifier genes, providing insight into pathways underlying CAG expansion and potential therapeutic targets. A novel in vivo screening strategy identifies new modifiers of somatic CAG repeat expansion that contribute to age of onset in Huntington’s disease.},
  archive      = {J_NG},
  author       = {Mouro Pinto, Ricardo and Murtha, Ryan and Azevedo, António and Douglas, Cameron and Kovalenko, Marina and Ulloa, Jessica and Crescenti, Steven and Burch, Zoe and Oliver, Esaria and Kesavan, Maheswaran and Shibata, Shota and Vitalo, Antonia and Mota-Silva, Eduarda and Riggs, Marion J. and Correia, Kevin and Elezi, Emanuela and Demelo, Brigitte and Carroll, Jeffrey B. and Gillis, Tammy and Gusella, James F. and MacDonald, Marcy E. and Wheeler, Vanessa C.},
  doi          = {10.1038/s41588-024-02054-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {314-322},
  shortjournal = {Nat. Genet.},
  title        = {In vivo CRISPR–Cas9 genome editing in mice identifies genetic modifiers of somatic CAG repeat instability in huntington’s disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutagenesis and analysis of contrasting wheat lines do not
support a role for PFT in fusarium head blight resistance. <em>NG</em>,
<em>57</em>(2), 310–313. (<a
href="https://doi.org/10.1038/s41588-024-02056-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ambiguity about whether the histidine-rich calcium-binding protein-coding gene (HisR) or the pore-forming toxin-like gene (PFT) or both are responsible for Fusarium head blight (FHB) resistance conferred by the Fhb1 quantitative trait locus hinders progress toward elucidating Fhb1 resistance mechanisms. Here, with a series of developed lines with or without PFT but all possessing HisR and five HisR-carrying PFT mutant lines created via gene editing, we show that PFT does not confer FHB resistance and that the HisR resistance effect does not require PFT in the tested conditions. We also show that PFT mutations are not associated with morphological and phenological characteristics that often affect FHB severity. Analysis of wheat lines generated through introgression or gene editing provides evidence that Fhb1-mediated resistance to Fusarium head blight is independent of the pore-forming toxin gene PFT.},
  archive      = {J_NG},
  author       = {Shi, Jinxing and Kong, Zhongxin and Zhong, Jinkun and Zhang, Xiaoqiu and Luo, Xianmin and Wang, Heyu and Xu, Boyang and Wang, Xin and Cheng, Rui and Yuan, Yang and Li, Na and Xie, Quan and Li, Guoqiang and Jia, Haiyan and Ma, Zhengqiang},
  doi          = {10.1038/s41588-024-02056-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {310-313},
  shortjournal = {Nat. Genet.},
  title        = {Mutagenesis and analysis of contrasting wheat lines do not support a role for PFT in fusarium head blight resistance},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Functional innovation through new genes as a general
evolutionary process. <em>NG</em>, <em>57</em>(2), 295–309. (<a
href="https://doi.org/10.1038/s41588-024-02059-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the past decade, our understanding of how new genes originate in diverse organisms has advanced substantially, and more than a dozen molecular mechanisms for generating initial gene structures were identified, in addition to gene duplication. These new genes have been found to integrate into and modify pre-existing gene networks primarily through mutation and selection, revealing new patterns and rules with stable origination rates across various organisms. This progress has challenged the prevailing belief that new proteins evolve from pre-existing genes, as new genes may arise de novo from noncoding DNA sequences in many organisms, with high rates observed in flowering plants. New genes have important roles in phenotypic and functional evolution across diverse biological processes and structures, with detectable fitness effects of sexual conflict genes that can shape species divergence. Such knowledge of new genes can be of translational value in agriculture and medicine. This Review discusses the various molecular mechanisms underlying the generation of new genes and highlights their important functions and phenotypes with an emphasis on the evolutionary forces underlying natural selection and sexual conflict.},
  archive      = {J_NG},
  author       = {Xia, Shengqian and Chen, Jianhai and Arsala, Deanna and Emerson, J. J. and Long, Manyuan},
  doi          = {10.1038/s41588-024-02059-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {295-309},
  shortjournal = {Nat. Genet.},
  title        = {Functional innovation through new genes as a general evolutionary process},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Overcoming challenges associated with broad sharing of human
genomic data. <em>NG</em>, <em>57</em>(2), 287–294. (<a
href="https://doi.org/10.1038/s41588-024-02049-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since the Human Genome Project, the consensus position in genomics has been that data should be shared widely to achieve the greatest societal benefit. This position relies on imprecise definitions of the concept of ‘broad data sharing’. Accordingly, the implementation of data sharing varies among landmark genomic studies. In this Perspective, we identify definitions of broad that have been used interchangeably, despite their distinct implications. We further offer a framework with clarified concepts for genomic data sharing and probe six examples in genomics that produced public data. Finally, we articulate three challenges. First, we explore the need to reinterpret the limits of general research use data. Second, we consider the governance of public data deposition from extant samples. Third, we ask whether, in light of changing concepts of broad, participants should be encouraged to share their status as participants publicly or not. Each of these challenges is followed with recommendations. This Perspective discusses the definitions of ‘broad’ that have been used in the context of sharing of human genomic data and proposes a clarified and reformed terminology for describing genomic data sharing more precisely.},
  archive      = {J_NG},
  author       = {LoTempio, Jonathan E. and Moreno, Jonathan D.},
  doi          = {10.1038/s41588-024-02049-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {287-294},
  shortjournal = {Nat. Genet.},
  title        = {Overcoming challenges associated with broad sharing of human genomic data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oncofetal reprogramming induces phenotypic heterogeneity in
colorectal cancer. <em>NG</em>, <em>57</em>(2), 285–286. (<a
href="https://doi.org/10.1038/s41588-025-02092-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotypic plasticity of cancer cells is increasingly recognized as a mechanism of tumor escape from targeted therapies. Yet, the phenotypic heterogeneity of colorectal cancer remains poorly explored. Our study identifies oncofetal reprogramming of neoplastic stem cells, driven by the transcription factors YAP and AP-1, as a critical driver of phenotypic heterogeneity, lineage plasticity and therapy resistance. Targeting the oncofetal program enhances the efficacy and durability of current treatments.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02092-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {285-286},
  shortjournal = {Nat. Genet.},
  title        = {Oncofetal reprogramming induces phenotypic heterogeneity in colorectal cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unveiling heterogeneity in rare cells by combining RNA-based
sorting and scRNA-seq. <em>NG</em>, <em>57</em>(2), 283–284. (<a
href="https://doi.org/10.1038/s41588-024-02073-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study describes the development of PERFF-seq, a method to ‘FISH’ out rare cells using RNA markers, helping to solve the challenge in single-cell biology of studying cells that make up less than 1% of a sample.},
  archive      = {J_NG},
  author       = {Wahlsten, Madison and Shaffer, Sydney M.},
  doi          = {10.1038/s41588-024-02073-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {283-284},
  shortjournal = {Nat. Genet.},
  title        = {Unveiling heterogeneity in rare cells by combining RNA-based sorting and scRNA-seq},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Defining genes and pathways that modify huntingtin CAG
repeat somatic instability in vivo. <em>NG</em>, <em>57</em>(2),
281–282. (<a href="https://doi.org/10.1038/s41588-024-02055-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A novel in vivo CRISPR screening platform identifies genetic modifiers of huntingtin CAG repeat somatic instability. These modifiers include known and novel genes that are promising therapeutic targets for Huntington’s disease.},
  archive      = {J_NG},
  author       = {Sathitloetsakun, Suphinya and Heiman, Myriam},
  doi          = {10.1038/s41588-024-02055-4},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {281-282},
  shortjournal = {Nat. Genet.},
  title        = {Defining genes and pathways that modify huntingtin CAG repeat somatic instability in vivo},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA loop extrusion is asymmetric but can switch direction.
<em>NG</em>, <em>57</em>(2), 280. (<a
href="https://doi.org/10.1038/s41588-025-02102-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02102-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {DNA loop extrusion is asymmetric but can switch direction},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering extrachromosomal DNA. <em>NG</em>,
<em>57</em>(2), 280. (<a
href="https://doi.org/10.1038/s41588-025-02103-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02103-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {Engineering extrachromosomal DNA},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Germline mutations increasing pediatric cancer risk.
<em>NG</em>, <em>57</em>(2), 280. (<a
href="https://doi.org/10.1038/s41588-025-02104-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02104-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {Germline mutations increasing pediatric cancer risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A model of heritable genome editing for complex traits.
<em>NG</em>, <em>57</em>(2), 280. (<a
href="https://doi.org/10.1038/s41588-025-02105-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-025-02105-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {A model of heritable genome editing for complex traits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Introducing the global alliance for spatial technologies
(GESTALT). <em>NG</em>, <em>57</em>(2), 275–279. (<a
href="https://doi.org/10.1038/s41588-024-02066-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This Comment outlines the creation of the Global Alliance for Spatial Technologies (GESTALT), a collaborative initiative aimed at fostering the growth and standardization of spatial tissue profiling technologies. It explores the need for GESTALT, its community-driven structure and its goals, spanning from the immediate to the long term.},
  archive      = {J_NG},
  author       = {Plummer, Jasmine T. and Vlachos, Ioannis S. and Martelotto, Luciano G.},
  doi          = {10.1038/s41588-024-02066-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {275-279},
  shortjournal = {Nat. Genet.},
  title        = {Introducing the global alliance for spatial technologies (GESTALT)},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
