<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>nature_all</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h1 id="nature">NATURE</h1>
<h2 id="natcs---11">NATCS - 11</h2>
<ul>
<li><details>
<summary>
(2025). Efficient crystal structure prediction based on the symmetry
principle. <em>NATCS</em>, <em>5</em>(3), 255–267. (<a
href="https://doi.org/10.1038/s43588-025-00775-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crystal structure prediction (CSP) is an evolving field aimed at discerning crystal structures with minimal prior information. Despite the success of various CSP algorithms, their practical applicability remains circumscribed, particularly for large and complex systems. Here, to address this challenge, we show an evolutionary structure generator within the MAGUS (Machine Learning and Graph Theory Assisted Universal Structure Searcher) framework, inspired by the symmetry principle. This generator extracts both global and local features of explored crystal structures using group and graph theory. By integrating an on-the-fly space group miner and fragment reorganizer, augmented by symmetry-kept mutation, our approach generates higher-quality initial structures, reducing the computational costs of CSP tasks. Benchmarking tests show up to fourfold performance improvements. The method also proves valid in complex phosphorus allotrope systems. Furthermore, we apply our approach to the diamond–silicon (111)-(7 × 7) surface system, identifying up to 42 metastable structures within an 18 meV Å−2 energy range, demonstrating the efficacy of our approach in navigating challenging search spaces. This study presents a symmetry principle-biased crystal structure prediction scheme within the MAGUS framework, achieving up to a fourfold performance improvement compared with state-of-the-art methods.},
  archive      = {J_NATCS},
  author       = {Han, Yu and Ding, Chi and Wang, Junjie and Gao, Hao and Shi, Jiuyang and Yu, Shaobo and Jia, Qiuhan and Pan, Shuning and Sun, Jian},
  doi          = {10.1038/s43588-025-00775-z},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {255-267},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient crystal structure prediction based on the symmetry principle},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Harnessing large language models for data-scarce learning of
polymer properties. <em>NATCS</em>, <em>5</em>(3), 245–254. (<a
href="https://doi.org/10.1038/s43588-025-00768-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) bear promise as a fast and accurate material modeling paradigm for evaluation, analysis and design. Their vast number of trainable parameters necessitates a wealth of data to achieve accuracy and mitigate overfitting. However, experimental measurements are often limited and costly to obtain in sufficient quantities for fine-tuning. To this end, here we present a physics-based training pipeline that tackles the pathology of data scarcity. The core enabler is a physics-based modeling framework that generates a multitude of synthetic data to align the LLM to a physically consistent initial state before fine-tuning. Our framework features a two-phase training strategy: utilizing the large-in-amount but less accurate synthetic data for supervised pretraining, and fine-tuning the phase-1 model with limited experimental data. We empirically demonstrate that supervised pretraining is vital to obtaining accurate fine-tuned LLMs, via the lens of learning polymer flammability metrics where cone calorimeter data are sparse. A physics-based training pipeline is developed to help tackle the challenges of data scarcity. The framework aligns large language models to a physically consistent initial state that is fine-tuned for learning polymer properties.},
  archive      = {J_NATCS},
  author       = {Liu, Ning and Jafarzadeh, Siavash and Lattimer, Brian Y. and Ni, Shuna and Lua, Jim and Yu, Yue},
  doi          = {10.1038/s43588-025-00768-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {245-254},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing large language models for data-scarce learning of polymer properties},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decoupled peak property learning for efficient and
interpretable electronic circular dichroism spectrum prediction.
<em>NATCS</em>, <em>5</em>(3), 234–244. (<a
href="https://doi.org/10.1038/s43588-024-00757-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic circular dichroism (ECD) spectra contain key information about molecular chirality by discriminating the absolute configurations of chiral molecules, which is crucial in asymmetric organic synthesis and the drug industry. However, existing predictive approaches lack the consideration of ECD spectra owing to the data scarcity and the limited interpretability to achieve trustworthy prediction. Here we establish a large-scale dataset for chiral molecular ECD spectra and propose ECDFormer for accurate and interpretable ECD spectrum prediction. ECDFormer decomposes ECD spectra into peak entities, uses the QFormer architecture to learn peak properties and renders peaks into spectra. Compared with spectrum sequence prediction methods, our decoupled peak prediction approach substantially enhances both accuracy and efficiency, improving the peak symbol accuracy from 37.3% to 72.7% and decreasing the time cost from an average of 4.6 central processing unit hours to 1.5 s. Moreover, ECDFormer demonstrated its ability to capture molecular orbital information directly from spectral data using the explainable peak-decoupling approach. Furthermore, ECDFormer proved to be equally proficient at predicting various types of spectrum, including infrared and mass spectroscopies, highlighting its substantial generalization capabilities. A large-scale electronic circular dichroism spectrum dataset is proposed and the ECDFormer framework is developed to achieve accurate and interpretable ECD spectrum prediction for natural products.},
  archive      = {J_NATCS},
  author       = {Li, Hao and Long, Da and Yuan, Li and Wang, Yu and Tian, Yonghong and Wang, Xinchang and Mo, Fanyang},
  doi          = {10.1038/s43588-024-00757-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {234-244},
  shortjournal = {Nat. Comput. Sci},
  title        = {Decoupled peak property learning for efficient and interpretable electronic circular dichroism spectrum prediction},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying associations between socio-spatial factors and
cognitive development in the ABCD cohort. <em>NATCS</em>, <em>5</em>(3),
221–233. (<a href="https://doi.org/10.1038/s43588-025-00774-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the mounting demand for generative population models, their limited generalizability to underrepresented demographic groups hinders widespread adoption in real-world applications. Here we propose a diversity-aware population modeling framework that can guide targeted strategies in public health and education, by estimating subgroup-level effects and stratifying predictions to capture sociodemographic variability. We leverage Bayesian multilevel regression and post-stratification to systematically quantify inter-individual differences in the relationship between socioeconomic status and cognitive development. Post-stratification enhanced the interpretability of model predictions across underrepresented groups by incorporating US Census data to gain additional insights into smaller subgroups in the Adolescent Brain Cognitive Development Study. This ensured that predictions were not skewed by overly heterogeneous or homogeneous representations. Our analyses underscore the importance of combining Bayesian multilevel modeling with post-stratification to validate reliability and provide a more holistic explanation of sociodemographic disparities in our diversity-aware population modeling framework. The study proposes a diversity-aware population modeling framework that can guide targeted strategies in public health, using Bayesian multilevel regression and post-stratification to quantify sociodemographic disparities in cognitive development.},
  archive      = {J_NATCS},
  author       = {Osayande, Nicole and Marotta, Justin and Aggarwal, Shambhavi and Kopal, Jakub and Holmes, Avram and Yip, Sarah W. and Bzdok, Danilo},
  doi          = {10.1038/s43588-025-00774-0},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {221-233},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying associations between socio-spatial factors and cognitive development in the ABCD cohort},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leveraging pharmacovigilance data to predict
population-scale toxicity profiles of checkpoint inhibitor
immunotherapy. <em>NATCS</em>, <em>5</em>(3), 207–220. (<a
href="https://doi.org/10.1038/s43588-024-00748-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immune checkpoint inhibitor (ICI) therapies have made considerable advances in cancer immunotherapy, but the complex and diverse spectrum of ICI-induced toxicities poses substantial challenges to treatment outcomes and computational analysis. Here we introduce DySPred, a dynamic graph convolutional network-based deep learning framework, to map and predict the toxicity profiles of ICIs at the population level by leveraging large-scale real-world pharmacovigilance data. DySPred accurately predicts toxicity risks across diverse demographic cohorts and cancer types, demonstrating resilience in small-sample scenarios and revealing toxicity trends over time. Furthermore, DySPred consistently aligns the toxicity-safety profiles of small-molecule antineoplastic agents with their drug-induced transcriptional alterations. Our study provides a versatile methodology for population-level profiling of ICI-induced toxicities, enabling proactive toxicity monitoring and timely tailoring of treatment and intervention strategies in the advancement of cancer immunotherapy. A deep learning framework is proposed with real-world pharmacovigilance data to predict population-scale toxicity profiles of checkpoint inhibitor immunotherapy, enabling proactive toxicity monitoring and timely tailoring of treatment.},
  archive      = {J_NATCS},
  author       = {Yan, Dongxue and Bao, Siqi and Zhang, Zicheng and Sun, Jie and Zhou, Meng},
  doi          = {10.1038/s43588-024-00748-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {207-220},
  shortjournal = {Nat. Comput. Sci},
  title        = {Leveraging pharmacovigilance data to predict population-scale toxicity profiles of checkpoint inhibitor immunotherapy},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive prediction and analysis of human protein
essentiality based on a pretrained large language model. <em>NATCS</em>,
<em>5</em>(3), 196–206. (<a
href="https://doi.org/10.1038/s43588-024-00733-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human essential proteins (HEPs) are indispensable for individual viability and development. However, experimental methods to identify HEPs are often costly, time consuming and labor intensive. In addition, existing computational methods predict HEPs only at the cell line level, but HEPs vary across living human, cell line and animal models. Here we develop a sequence-based deep learning model, Protein Importance Calculator (PIC), by fine-tuning a pretrained protein language model. PIC not only substantially outperforms existing methods for predicting HEPs but also provides comprehensive prediction results across three levels: human, cell line and mouse. Furthermore, we define the protein essential score, derived from PIC, to quantify human protein essentiality and validate its effectiveness by a series of biological analyses. We also demonstrate the biomedical value of the protein essential score by identifying potential prognostic biomarkers for breast cancer and quantifying the essentiality of 617,462 human microproteins. This study introduces the Protein Importance Calculator (PIC), a deep learning model designed to predict human essential proteins (HEPs) crucial for survival and development. Unlike conventional methods, PIC offers a comprehensive assessment of HEPs across three levels: humans, cell lines and mice.},
  archive      = {J_NATCS},
  author       = {Kang, Boming and Fan, Rui and Cui, Chunmei and Cui, Qinghua},
  doi          = {10.1038/s43588-024-00733-1},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {196-206},
  shortjournal = {Nat. Comput. Sci},
  title        = {Comprehensive prediction and analysis of human protein essentiality based on a pretrained large language model},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diversity-aware population modeling. <em>NATCS</em>,
<em>5</em>(3), 194–195. (<a
href="https://doi.org/10.1038/s43588-025-00787-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We propose a diversity-aware population modeling framework using Bayesian multilevel regression and post-stratification to quantify sociodemographic disparities in cognitive development. Our approach improved subgroup estimates, guiding targeted public health strategies and addressing biases in traditional models to support more equitable decision-making.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00787-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {194-195},
  shortjournal = {Nat. Comput. Sci},
  title        = {Diversity-aware population modeling},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosting crystal structure prediction via symmetry.
<em>NATCS</em>, <em>5</em>(3), 192–193. (<a
href="https://doi.org/10.1038/s43588-025-00779-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting stable crystal structures for complex systems that involve multiple elements or a large number of atoms presents a formidable challenge in computational materials science. A recent study presents an efficient crystal-structure search method for this task, utilizing symmetry and graph theory.},
  archive      = {J_NATCS},
  author       = {Wang, Yanchao},
  doi          = {10.1038/s43588-025-00779-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {192-193},
  shortjournal = {Nat. Comput. Sci},
  title        = {Boosting crystal structure prediction via symmetry},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An organelle-agnostic image analysis tool. <em>NATCS</em>,
<em>5</em>(3), 191. (<a
href="https://doi.org/10.1038/s43588-025-00785-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Badri, Michelle},
  doi          = {10.1038/s43588-025-00785-x},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {191},
  shortjournal = {Nat. Comput. Sci},
  title        = {An organelle-agnostic image analysis tool},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On the unknowable limits to prediction. <em>NATCS</em>,
<em>5</em>(3), 188–190. (<a
href="https://doi.org/10.1038/s43588-025-00776-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Yan, Jiani and Rahal, Charles},
  doi          = {10.1038/s43588-025-00776-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {188-190},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the unknowable limits to prediction},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Celebrating a pioneer in bioinformatics. <em>NATCS</em>,
<em>5</em>(3), 187. (<a
href="https://doi.org/10.1038/s43588-025-00784-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In honor of the 100th birthday of Margaret Dayhoff, we spotlight her footprint in the field of bioinformatics.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00784-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {187},
  shortjournal = {Nat. Comput. Sci},
  title        = {Celebrating a pioneer in bioinformatics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="natee---24">NATEE - 24</h2>
<ul>
<li><details>
<summary>
(2025). Glasswing butterfly (mechanitis messenoides).
<em>NATEE</em>, <em>9</em>(3), 526. (<a
href="https://doi.org/10.1038/s41559-025-02655-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Joana Meier is fascinated by the diversity and mimetic colour patterns of glasswing butterflies.},
  archive      = {J_NATEE},
  author       = {Meier, Joana Isabel},
  doi          = {10.1038/s41559-025-02655-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {526},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Glasswing butterfly (Mechanitis messenoides)},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Publisher correction: Evolution of sexual size dimorphism
in tetrapods is driven by varying patterns of sex-specific selection on
size. <em>NATEE</em>, <em>9</em>(3), 525. (<a
href="https://doi.org/10.1038/s41559-025-02635-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Slavenko, Alex and Cooper, Natalie and Meiri, Shai and Murali, Gopal and Pincheira-Donoso, Daniel and Thomas, Gavin H.},
  doi          = {10.1038/s41559-025-02635-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {525},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Evolution of sexual size dimorphism in tetrapods is driven by varying patterns of sex-specific selection on size},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The current state, opportunities and challenges for
upscaling private investment in biodiversity in europe. <em>NATEE</em>,
<em>9</em>(3), 515–524. (<a
href="https://doi.org/10.1038/s41559-024-02632-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {European countries have committed to ambitious upscaling of privately funded nature conservation. We review the status and drivers of biodiversity finance in Europe. By implementing semistructured interviews with 25 biodiversity finance key informants and three focus groups across Europe, we explore opportunities and challenges for upscaling private investment in nature. Opportunities arise from macroeconomic and regulatory changes, along with various technological and financial innovations and growing professional experience. However, persistent barriers to upscaling include the ongoing lack of highly profitable investment opportunities and the multitude of risks facing investors, including political, ecological and reputational risks influencing supply and demand of investment opportunities. Public policy plays the foundational role in creating and hindering these mechanisms. Public policy can create nature markets and investment opportunities, meanwhile agricultural subsidies and poor coordination between public funding sources undermine the supply of return-seeking investment opportunities. Investors demand derisking investments from uncertainties; in part caused by political uncertainty. These markets require profound state intervention to enable upscaling whilst achieving positive ecological outcomes; private investment will probably not upscale without major public policy change and public investment. Semistructured interviews and focus groups with experts in biodiversity finance in Europe identify opportunities and barriers to upscaling private investment in biodiversity, and emphasize that public policy is the key to enabling progression.},
  archive      = {J_NATEE},
  author       = {zu Ermgassen, Sophus O.S.E. and Hawkins, Isobel and Lundhede, Thomas and Liu, Qian and Thorsen, Bo Jellesmark and Bull, Joseph W.},
  doi          = {10.1038/s41559-024-02632-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {515-524},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The current state, opportunities and challenges for upscaling private investment in biodiversity in europe},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Asymmetric sensitivity of boreal forest resilience to forest
gain and loss. <em>NATEE</em>, <em>9</em>(3), 505–514. (<a
href="https://doi.org/10.1038/s41559-024-02631-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Forest gains and losses may have unequal effects on forest resilience, particularly given their distinct temporal dynamics. Here, we quantify the sensitivities of boreal forest resilience to forest cover gain and loss using a resilience indicator derived from the temporal autocorrelation (TAC) of the kernel normalized difference vegetation index from 2000 to 2020. Our findings unveil pronounced asymmetric sensitivities, with stronger sensitivity to forest loss (−4.26 ± 0.14 × 10−3; TAC increase per 1% forest cover loss) than to forest gain (−1.65 ± 0.12 × 10−3; TAC decrease per 1% forest cover gain). Locally, ~73% of the boreal forest exhibits negative sensitivity, indicating enhanced resilience with forest cover gain and vice versa, especially in intact forests compared to managed ones. This sensitivity is affected by various trajectories in forest cover change, stemming primarily from temporal asynchrony in the recovery rates of various ecosystem functions. The observed asymmetry underscores the importance of prioritizing forest conservation over reactive management strategies following losses, ultimately contributing to more sustainable forest management practices. Boreal forests have been experiencing both gains and losses in recent decades. Here, the authors show that boreal forest resilience is more sensitive to forest cover losses than to gains, indicating that functional declines due to forest loss outpace improvements following forest recovery.},
  archive      = {J_NATEE},
  author       = {Liu, Xiaoye and Wang, Dashan and Chen, Anping and Zeng, Zhenzhong},
  doi          = {10.1038/s41559-024-02631-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {505-514},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Asymmetric sensitivity of boreal forest resilience to forest gain and loss},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multispecies migratory connectivity indicates
hemispheric-scale risk to bird populations from global change.
<em>NATEE</em>, <em>9</em>(3), 491–504. (<a
href="https://doi.org/10.1038/s41559-024-02575-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global agreements to reduce the extinction risk of migratory species depend critically on intersecting migratory connectivity—the linking of individuals between regions in different seasons—and spatial patterns of environmental change. Here we integrate movement data from &amp;gt;329,000 migratory birds of 112 species to develop a parameter representing exposure to global change: multispecies migratory connectivity. We then combine exposure with projected climate and land-cover changes as a measure of hazard and species conservation assessment scores as a metric of vulnerability to estimate the relative risk of migratory bird population declines across the Western Hemisphere. Multispecies migratory connectivity (exposure) is the strongest driver of risk relative to hazard and vulnerability, indicating the importance of synthesizing connectivity across species to comprehensively assess risk. Connections between breeding regions in Canada and non-breeding regions in South America are at the greatest risk, which underscores the particular susceptibility of long-distance migrants. Over half (54%) of the connections categorized as very high risk include breeding regions in the eastern United States. This three-part framework serves as an ecological risk assessment designed specifically for migratory species, providing both decision support for global biodiversity conservation and opportunities for intergovernmental collaboration to sustain migratory bird populations year-round. On the basis of data from &amp;gt;329,000 migratory birds, this study presents multispecies migratory connectivity as a parameter representing exposure to global change and shows that connections between breeding regions in Canada and non-breeding regions in South America are at greatest risk from global change.},
  archive      = {J_NATEE},
  author       = {Saunders, Sarah P. and DeLuca, William V. and Bateman, Brooke L. and Deppe, Jill L. and Grand, Joanna and Knight, Erika J. and Meehan, Timothy D. and Michel, Nicole L. and Seavy, Nathaniel E. and Smith, Melanie A. and Taylor, Lotem and Witko, Chad J. and Wilsey, Chad B.},
  doi          = {10.1038/s41559-024-02575-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {491-504},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Multispecies migratory connectivity indicates hemispheric-scale risk to bird populations from global change},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reduced evolutionary constraint accompanies ongoing
radiation in deep-sea anglerfishes. <em>NATEE</em>, <em>9</em>(3),
474–490. (<a href="https://doi.org/10.1038/s41559-024-02586-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Colonization of a novel habitat is often followed by phenotypic diversification in the wake of ecological opportunity. However, some habitats should be inherently more constraining than others if the challenges of that environment offer few evolutionary solutions. We examined this push-and-pull on macroevolutionary diversification following habitat transitions in the anglerfishes (Lophiiformes). We constructed a phylogeny with extensive sampling (1,092 loci and ~38% of species), combined with three-dimensional phenotypic data from museum specimens. We used these datasets to examine the tempo and mode of phenotypic diversification. The deep-sea pelagic anglerfishes originated from a benthic ancestor and shortly after experienced rapid lineage diversification rates. This transition incurred shifts towards larger jaws, smaller eyes and a more laterally compressed body plan. Despite these directional trends, this lineage still evolved high phenotypic disparity in body, skull and jaw shapes. In particular, bathypelagic anglerfishes show high variability in body elongation, while benthic anglerfishes are constrained around optimal shapes. Within this radiation, phenotypic evolution was concentrated among recently diverged lineages, notably those that deviated from the archetypical globose body plan. Taken together, these results demonstrate that spectacular evolutionary radiations can unfold even within environments with few ecological resources and demanding physiological challenges. Based on phylogenomic and geometric morphometric analyses of 132 anglerfish species, the authors infer a Cretaceous origin of the clade and show that bathypelagic anglerfish are undergoing rapid phenotypic diversification despite inhabiting a relatively homogeneous deep-sea habitat.},
  archive      = {J_NATEE},
  author       = {Miller, Elizabeth Christina and Faucher, Rose and Hart, Pamela B. and Rincón-Sandoval, Melissa and Santaquiteria, Aintzane and White, William T. and Baldwin, Carole C. and Miya, Masaki and Betancur-R, Ricardo and Tornabene, Luke and Evans, Kory and Arcila, Dahiana},
  doi          = {10.1038/s41559-024-02586-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {474-490},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reduced evolutionary constraint accompanies ongoing radiation in deep-sea anglerfishes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Evolution of sexual size dimorphism in tetrapods is driven
by varying patterns of sex-specific selection on size. <em>NATEE</em>,
<em>9</em>(3), 464–473. (<a
href="https://doi.org/10.1038/s41559-024-02600-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sexual size dimorphism (SSD) is highly prevalent in nature. Several hypotheses aim to explain its evolution including sexual selection, differential equilibrium and ecological niche divergence. Disentangling the causal mechanism behind the evolution of SSD is challenging, as selection arising from multiple pressures on fitness may act simultaneously to generate observed patterns. Here, we use phylogenetic comparative methods to study the evolution of SSD across tetrapods globally. We estimate directional changes in body size evolution, and compare the number, phylogenetic position and magnitude of size changes between sexes. We find evidence that directional changes in size associated with SSD are typically more common in males—even in lineages where females are larger. However, underlying mechanisms differ among lineages—whereas SSD in amphibians becomes more male-biased with greater increases in male size and mammalian SSD becomes more female-biased with greater decreases in male size. Thus, differing mechanisms of directional body size evolution across sexes are essential to explain observed SSD patterns. A phylogenetic comparative analysis of male and female body size across tetrapods globally shows that directional change in size is usually greater in males but reveals different underlying mechanisms among lineages.},
  archive      = {J_NATEE},
  author       = {Slavenko, Alex and Cooper, Natalie and Meiri, Shai and Murali, Gopal and Pincheira-Donoso, Daniel and Thomas, Gavin H.},
  doi          = {10.1038/s41559-024-02600-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {464-473},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolution of sexual size dimorphism in tetrapods is driven by varying patterns of sex-specific selection on size},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolutionary divergence between homologous x–y chromosome
genes shapes sex-biased biology. <em>NATEE</em>, <em>9</em>(3), 448–463.
(<a href="https://doi.org/10.1038/s41559-024-02627-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sex chromosomes are a fundamental aspect of sex-biased biology, but the extent to which homologous X–Y gene pairs (‘the gametologs’) contribute to sex-biased phenotypes remains hotly debated. Although these genes tend to exhibit large sex differences in expression throughout the body (XX females can express both X members, and XY males can express one X and one Y member), there is conflicting evidence regarding the degree of functional divergence between the X and Y members. Here we develop and apply co-expression fingerprint analysis to characterize functional divergence between the X and Y members of 17 gametolog gene pairs across &amp;gt;40 human tissues. Gametolog pairs exhibit functional divergence between the sexes that is driven by divergence between the X versus Y members (assayed in males), and this within-pair divergence is greatest among pairs with evolutionarily distant X and Y members. These patterns reflect that X versus Y gametologs show coordinated patterns of asymmetric coupling with large sets of autosomal genes, which are enriched for functional pathways and gene sets implicated in sex-biased biology and disease. Our findings suggest that the X versus Y gametologs have diverged in function and prioritize specific gametolog pairs for future targeted experimental studies. Based on co-expression fingerprint analysis between homologous genes in the X and Y chromosomes (gametolog pairs) across &amp;gt;40 human tissues, the authors suggest that X and Y gametologs have diverged in function.},
  archive      = {J_NATEE},
  author       = {DeCasien, Alex R. and Tsai, Kathryn and Liu, Siyuan and Thomas, Adam and Raznahan, Armin},
  doi          = {10.1038/s41559-024-02627-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {448-463},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolutionary divergence between homologous X–Y chromosome genes shapes sex-biased biology},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying disturbance effects on ecosystem services in a
changing climate. <em>NATEE</em>, <em>9</em>(3), 436–447. (<a
href="https://doi.org/10.1038/s41559-024-02626-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Disturbances, such as hurricanes, fires, droughts and pest outbreaks, can cause major changes in ecosystem conditions that threaten Nature’s contributions to people (ecosystem services). Climate change is intensifying disturbances, posing risks to ecosystem services. To assess those risks, we develop a flexible, functional trait-based approach to quantify ecological, ecosystem service and economic impacts from disturbance regimes. Our broadly applicable approach integrates knowledge from disturbance ecology and ecosystem service valuation, and we highlight the pitfalls of using either perspective in isolation. We demonstrate our approach by quantifying impacts to timber and recreational enjoyment from extreme windstorms in a midlatitude forest. While we predict large potential losses to these services under historical and future disturbance regimes, common ecological metrics of compositional and biomass stability are inadequate for predicting these impacts. We then provide a roadmap for applying our approach across different social-ecological systems, illustrating the approach for crop pollination, flood hazard mitigation and cultural values from coral reefs—which all face intensifying disturbances. This study highlights and provides tools to address the pressing need to consider disturbances in future ecosystem service assessments. A generalizable, functional-trait-based approach for quantifying the effects of disturbances to ecosystem services and economic outcomes, including under climate change, highlights the need for incorporating disturbances in ecosystem services assessments.},
  archive      = {J_NATEE},
  author       = {Dee, Laura E. and Miller, Steve J. and Helmstedt, Kate J. and Boersma, Kate S. and Polasky, Stephen and Reich, Peter B.},
  doi          = {10.1038/s41559-024-02626-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {436-447},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Quantifying disturbance effects on ecosystem services in a changing climate},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The cost of recovering australia’s threatened species.
<em>NATEE</em>, <em>9</em>(3), 425–435. (<a
href="https://doi.org/10.1038/s41559-024-02617-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accounting for the cost of repairing the degradation of Earth’s biosphere is critical to guide conservation and sustainable development decisions. Yet the costs of repairing nature through the recovery of a continental suite of threatened species across their range have never been calculated. We estimated the cost of in situ recovery of nationally listed terrestrial and freshwater threatened species (n = 1,657) across the megadiverse continent of Australia by combining the spatially explicit costs of all strategies required to address species-specific threats. Individual species recovery required up to 12 strategies (mean 2.3), predominantly habitat retention and restoration, and the management of fire and invasive species. The estimated costs of maximizing threatened species recovery across Australia varied from AU$0–$12,626 per ha, depending on the species, threats and context of each location. The total cost of implementing all strategies to recover threatened species in their in situ habitat across Australia summed to an estimated AU$583 billion per year, with management of invasive weeds making up 81% of the total cost. This figure, at 25% of Australia’s GDP, does not represent a realistic biodiversity conservation budget, but needs to be accounted for when weighing up decisions that lead to further costly degradation of Australia’s natural heritage. An estimated AU$583 billion per year cost of in situ recovery of terrestrial and freshwater species in Australia, including through habitat restoration and retention and management of invasive species, is not an expected conservation budget, but exemplifies the severe cost of nature declines.},
  archive      = {J_NATEE},
  author       = {Reside, April E. and Carwardine, Josie and Ward, Michelle and Yong, Chuanji and Venegas Li, Ruben and Rogers, Andrew and Wintle, Brendan A. and Silcock, Jennifer and Woinarski, John and Lintermans, Mark and Taylor, Gary and Pintor, Anna F. V. and Watson, James E. M.},
  doi          = {10.1038/s41559-024-02617-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {425-435},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The cost of recovering australia’s threatened species},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ambitions in national plans do not yet match bold
international protection and restoration commitments. <em>NATEE</em>,
<em>9</em>(3), 417–424. (<a
href="https://doi.org/10.1038/s41559-025-02636-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Almost 200 nations have made bold commitments to halt biodiversity loss as signatories to the Kunming-Montreal Global Biodiversity Framework (GBF). The effective achievement of the GBF relies on domestic targets and actions, reflected in National Biodiversity Strategies and Action Plans (NBSAPs). NBSAPS are an integral feature of the Convention on Biological Diversity (CBD) framework and signatory nations were requested to submit revised NBSAPs before the 16th Conference of the Parties (COP-16) incorporating the GBF goals and targets. Here we review NBSAPs of the 36 nations that submitted before COP-16 and assess their commitments to implementing target 2 (the 30% restoration target) and target 3 (the 30 × 30 protection target). By first breaking these targets into their constituent elements and assessing the detailed wording of each NBSAP we discover that no nation has created a plan that meets all the requirements—and overall ambitions—of these two targets. With 5 years remaining until the intended realization of the GBF, countries will need to increase both their ambition and action if the biodiversity crisis of the Earth is to be abated. Analysis of the 36 National Biodiversity Strategies and Action Plans that were submitted to the Convention on Biological Diversity by 21 October 2024 reveal that no nation has yet created a plan that meets all elements of targets 2 and 3 of the Kunming-Montreal Global Biodiversity Framework.},
  archive      = {J_NATEE},
  author       = {Bell-James, Justine and Watson, James E. M.},
  doi          = {10.1038/s41559-025-02636-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {417-424},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ambitions in national plans do not yet match bold international protection and restoration commitments},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Collective dynamical regimes predict invasion success and
impacts in microbial communities. <em>NATEE</em>, <em>9</em>(3),
406–416. (<a href="https://doi.org/10.1038/s41559-024-02618-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The outcomes of ecological invasions may depend on either characteristics of the invading species or attributes of the resident community. Here we use a combination of experiments and theory to show that the interplay between dynamics, interaction strength and diversity determine the invasion outcome in microbial communities. We find that the communities with fluctuating species abundances are more invasible and diverse than stable communities, leading to a positive diversity–invasibility relationship among communities assembled in the same environment. As predicted by theory, increasing interspecies interaction strength and species pool size leads to a decrease of invasion probability in our experiment. Our results show a positive correspondence between invasibility and survival fraction of resident species across all conditions. Communities composed of strongly interacting species can exhibit an emergent priority effect in which invader species are less likely to colonize than species in the original pool. However, if an invasion is successful, its ecological effects on the resident community are greater when interspecies interactions are strong. Our findings provide a unified perspective on the diversity–invasibility debate by showing that invasibility and invasion effect are emergent properties of interacting species, which can be predicted by simple community-level features. Predicting invasion outcomes in microbial communities is challenging. Here the authors demonstrate that these outcomes in both theoretical and experimental microbial communities can be predicted based on community dynamics, diversity and species interactions.},
  archive      = {J_NATEE},
  author       = {Hu, Jiliang and Barbier, Matthieu and Bunin, Guy and Gore, Jeff},
  doi          = {10.1038/s41559-024-02618-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {406-416},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Collective dynamical regimes predict invasion success and impacts in microbial communities},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pre-exposure of abundant species to disturbance improves
resilience in microbial metacommunities. <em>NATEE</em>, <em>9</em>(3),
395–405. (<a href="https://doi.org/10.1038/s41559-024-02624-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding factors influencing community resilience to disturbance is critical for mitigating harm at various scales, including harm from medication to gut microbiota and harm from human activity to global biodiversity, yet there is a lack of data from large-scale controlled experiments. Factors expected to boost resilience include prior exposure to the same disturbance and dispersal from undisturbed patches. Here we set up an in vitro system to test the effect of disturbance pre-exposure and dispersal represented by community mixing. We performed a serial passage experiment on a 23-species bacterial model community, varying pre-exposure history and dispersal rate between three metacommunity patches subjected to different levels of disturbance by the antibiotic streptomycin. As expected, pre-exposure caused evolution of resistance, which prevented decrease in species abundance. The more abundant the pre-exposed species had been in the undisturbed community, the less the entire community changed. Pre-exposure of the most dominant species also decreased abundance change in off-target species. In the absence of pre-exposure, increasing dispersal rates caused increasing spread of the disturbance across the metacommunity. However, pre-exposure kept the metacommunity close to the undisturbed state regardless of dispersal rate. Our findings demonstrate that pre-exposure is an important modifier of ecological resilience in a metacommunity setting. Community resilience to stress is affected by factors such as pre-exposure to the same stress and intercommunity dispersal. The authors show that pre-exposing the most dominant members of a 23-species bacterial community to different levels of antibiotic stress leads to rapid evolution of resistance that improves metacommunity resilience even under high levels of dispersal.},
  archive      = {J_NATEE},
  author       = {Cairns, Johannes and Hogle, Shane and Alitupa, Elizaveta and Mustonen, Ville and Hiltunen, Teppo},
  doi          = {10.1038/s41559-024-02624-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {395-405},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pre-exposure of abundant species to disturbance improves resilience in microbial metacommunities},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). No winter halt in below-ground wood growth of four
angiosperm deciduous tree species. <em>NATEE</em>, <em>9</em>(3),
386–394. (<a href="https://doi.org/10.1038/s41559-024-02602-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the temperate zone, deciduous trees exhibit clear above-ground seasonality, marked by a halt in wood growth that represents the completion of wood formation in autumn and reactivation in spring. However, the growth seasonality of below-ground woody organs, such as coarse roots, has been largely overlooked. Here we use tree monitoring data and pot experiments involving saplings to examine the late-season xylem development of stem and coarse roots with leaf phenology in four common deciduous tree species in Western Europe. Coarse-roots wood growth continued throughout the winter whereas stem wood growth halted in autumn, regardless of the tree species, experimental setting or location. Our results do not indicate a clear temperature constraint on below-ground wood growth, even during prolonged periods with soil temperatures lower than 3 °C. The continuous differentiation of xylem root cells in autumn and winter suggests that the non-growing season does not exist sensu stricto for all woody organs of angiosperm deciduous tree species of the temperate zone. Our findings hold implications for understanding tree functioning, in particular the seasonal wood formation, the environmental controls of tree growth and the carbon reserves dynamics. As wood growth in deciduous tree stems halts during winter, it has been assumed that wood growth in coarse roots follows the same pattern. This study on the growth of stem and coarse roots of four European tree species challenges the assumption of winter halt in below-ground wood growth of temperate deciduous trees.},
  archive      = {J_NATEE},
  author       = {Marchand, Lorène J. and Gričar, Jožica and Zuccarini, Paolo and Dox, Inge and Mariën, Bertold and Verlinden, Melanie and Heinecke, Thilo and Prislan, Peter and Marie, Guillaume and Lange, Holger and Van den Bulcke, Jan and Penuelas, Josep and Fonti, Patrick and Campioli, Matteo},
  doi          = {10.1038/s41559-024-02602-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {386-394},
  shortjournal = {Nat. Ecol. Evol},
  title        = {No winter halt in below-ground wood growth of four angiosperm deciduous tree species},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Generative AI as a tool to accelerate the field of ecology.
<em>NATEE</em>, <em>9</em>(3), 378–385. (<a
href="https://doi.org/10.1038/s41559-024-02623-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of generative artificial intelligence (AI) models specializing in the generation of new data with the statistical patterns and properties of the data upon which the models were trained has profoundly influenced a range of academic disciplines, industry and public discourse. Combined with the vast amounts of diverse data now available to ecologists, from genetic sequences to remotely sensed animal tracks, generative AI presents enormous potential applications within ecology. Here we draw upon a range of fields to discuss unique potential applications in which generative AI could accelerate the field of ecology, including augmenting data-scarce datasets, extending observations of ecological patterns and increasing the accessibility of ecological data. We also highlight key challenges, risks and considerations when using generative AI within ecology, such as privacy risks, model biases and environmental effects. Ultimately, the future of generative AI in ecology lies in the development of robust interdisciplinary collaborations between ecologists and computer scientists. Such partnerships will be important for embedding ecological knowledge within AI, leading to more ecologically meaningful and relevant models. This will be critical for leveraging the power of generative AI to drive ecological insights into species across the globe. This Progress discusses potential applications of artificial intelligence models that generate new data and how they can be used to advance ecology research.},
  archive      = {J_NATEE},
  author       = {Rafiq, Kasim and Beery, Sara and Palmer, Meredith S. and Harchaoui, Zaid and Abrahms, Briana},
  doi          = {10.1038/s41559-024-02623-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {378-385},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Generative AI as a tool to accelerate the field of ecology},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Insights into the human sex chromosomes from divergence
between the x–y gametologues. <em>NATEE</em>, <em>9</em>(3), 376–377.
(<a href="https://doi.org/10.1038/s41559-024-02628-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study introduces a method to systematically compare the functions of homologous (shared) genes between the X and Y chromosomes (gametologues) across the human body. The authors discover differences that are linked to the evolutionary histories of these genes and could contribute to sex-biased biology and disease.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-024-02628-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {376-377},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Insights into the human sex chromosomes from divergence between the X–Y gametologues},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pre-exposure to disturbance enhances species resilience and
protects from change. <em>NATEE</em>, <em>9</em>(3), 374–375. (<a
href="https://doi.org/10.1038/s41559-024-02625-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In a serial passage experiment with a 23-species model microbial community, we found that higher dispersal led to stronger spread of the effects of antibiotic disturbance across a metacommunity. This effect was cancelled by protecting the species from the disturbance through adaptation.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-024-02625-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {374-375},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pre-exposure to disturbance enhances species resilience and protects from change},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accounting for the true costs of recovery of threatened
species. <em>NATEE</em>, <em>9</em>(3), 372–373. (<a
href="https://doi.org/10.1038/s41559-024-02621-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New comprehensive datasets of threats to species, management actions and costs of these actions reveal the total costs of fully recovering all of Australia’s threatened species. At AUD $583 billion per year (25% of Australia’s GDP for 2023), the cost of reversing two centuries of declines showcases the recovery challenge ahead and the value of avoiding further degradation.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-024-02621-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {372-373},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Accounting for the true costs of recovery of threatened species},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Effects of extreme events on nature’s benefits to people.
<em>NATEE</em>, <em>9</em>(3), 370–371. (<a
href="https://doi.org/10.1038/s41559-024-02620-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A mathematical framework integrates the effect of disturbances on ecosystem services under climate change, and offers a vital tool to incorporate changing disturbance regimes into risk-sensitive decision making.},
  archive      = {J_NATEE},
  author       = {Runting, Rebecca K. and Wells, Jessie A.},
  doi          = {10.1038/s41559-024-02620-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {370-371},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Effects of extreme events on nature’s benefits to people},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). No refuge for amazon birds. <em>NATEE</em>, <em>9</em>(3),
369. (<a href="https://doi.org/10.1038/s41559-025-02660-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-025-02660-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {369},
  shortjournal = {Nat. Ecol. Evol},
  title        = {No refuge for amazon birds},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ancient history of MX antiviral proteins. <em>NATEE</em>,
<em>9</em>(3), 368. (<a
href="https://doi.org/10.1038/s41559-025-02659-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-025-02659-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {368},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ancient history of MX antiviral proteins},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Indigenous communities share how to live with wildlife.
<em>NATEE</em>, <em>9</em>(3), 367. (<a
href="https://doi.org/10.1038/s41559-025-02657-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-025-02657-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {367},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Indigenous communities share how to live with wildlife},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mortality in european lynxes. <em>NATEE</em>, <em>9</em>(3),
366. (<a href="https://doi.org/10.1038/s41559-025-02656-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Andriuzzi, Walter},
  doi          = {10.1038/s41559-025-02656-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {366},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mortality in european lynxes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep cuts. <em>NATEE</em>, <em>9</em>(3), 365. (<a
href="https://doi.org/10.1038/s41559-025-02666-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sweeping job losses and freezes to science funding in the USA have created a time of immense unease for researchers and are likely to result in costs to global health and innovation, and for the planet.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-025-02666-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {365},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deep cuts},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="natmi---18">NATMI - 18</h2>
<ul>
<li><details>
<summary>
(2025). Benchmarking AI-powered docking methods from the perspective
of virtual screening. <em>NATMI</em>, <em>7</em>(3), 509–520. (<a
href="https://doi.org/10.1038/s42256-025-00993-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recently, many artificial intelligence (AI)-powered protein–ligand docking and scoring methods have been developed, demonstrating impressive speed and accuracy. However, these methods often neglected the physical plausibility of the docked complexes and their efficacy in virtual screening (VS) projects. Therefore, we conducted a comprehensive benchmark analysis of four AI-powered and four physics-based docking tools and two AI-enhanced rescoring methods. We initially constructed the TrueDecoy set, a dataset on which the redocking experiments revealed that KarmaDock and CarsiDock surpassed all physics-based tools in docking accuracy, whereas all physics-based tools notably outperformed AI-based methods in structural rationality. The low physical plausibility of docked structures generated by the top AI method, CarsiDock, mainly stems from insufficient intermolecular validity. The VS results on the TrueDecoy set highlight the effectiveness of RTMScore as a rescore function, and Glide-based methods achieved the highest enrichment factors among all docking tools. Furthermore, we created the RandomDecoy set, a dataset that more closely resembles real-world VS scenarios, where AI-based tools obviously outperformed Glide. Additionally, we found that the employed ligand-based postprocessing methods had a weak or even negative impact on optimizing the conformations of docked complexes and enhancing VS performance. Finally, we proposed a hierarchical VS strategy that could efficiently and accurately enrich active molecules in large-scale VS projects. Artificial intelligence (AI)-based docking and scoring methods demonstrate considerable potential for virtual drug screening. Gu et al. go further by assessing the structural rationality of AI-predicted complex conformations from various sources.},
  archive      = {J_NATMI},
  author       = {Gu, Shukai and Shen, Chao and Zhang, Xujun and Sun, Huiyong and Cai, Heng and Luo, Hao and Zhao, Huifeng and Liu, Bo and Du, Hongyan and Zhao, Yihao and Fu, Chenggong and Zhai, Silong and Deng, Yafeng and Liu, Huanxiang and Hou, Tingjun and Kang, Yu},
  doi          = {10.1038/s42256-025-00993-0},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {509-520},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Benchmarking AI-powered docking methods from the perspective of virtual screening},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards a more inductive world for drug repurposing
approaches. <em>NATMI</em>, <em>7</em>(3), 495–508. (<a
href="https://doi.org/10.1038/s42256-025-00987-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug–target interaction (DTI) prediction is a challenging albeit essential task in drug repurposing. Learning on graph models has drawn special attention as they can substantially reduce drug repurposing costs and time commitment. However, many current approaches require high-demand additional information besides DTIs that complicates their evaluation process and usability. Additionally, structural differences in the learning architecture of current models hinder their fair benchmarking. In this work, we first perform an in-depth evaluation of current DTI datasets and prediction models through a robust benchmarking process and show that DTI methods based on transductive models lack generalization and lead to inflated performance when traditionally evaluated, making them unsuitable for drug repurposing. We then propose a biologically driven strategy for negative-edge subsampling and uncovered previously unknown interactions via in vitro validation, missed by traditional subsampling. Finally, we provide a toolbox from all generated resources, crucial for fair benchmarking and robust model design. The authors address the challenge of predicting drug–target interactions, which is crucial for drug repurposing, by introducing a robust benchmarking framework. Using a biologically driven strategy, they uncover previously unknown interactions.},
  archive      = {J_NATMI},
  author       = {de la Fuente, Jesus and Serrano, Guillermo and Veleiro, Uxía and Casals, Mikel and Vera, Laura and Pizurica, Marija and Gómez-Cebrián, Nuria and Puchades-Carrasco, Leonor and Pineda-Lucena, Antonio and Ochoa, Idoia and Vicent, Silve and Gevaert, Olivier and Hernaez, Mikel},
  doi          = {10.1038/s42256-025-00987-y},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {495-508},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards a more inductive world for drug repurposing approaches},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Teaching robots to build simulations of themselves.
<em>NATMI</em>, <em>7</em>(3), 484–494. (<a
href="https://doi.org/10.1038/s42256-025-01006-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of vision catalysed a pivotal evolutionary advancement, enabling organisms not only to perceive but also to interact intelligently with their environment. This transformation is mirrored by the evolution of robotic systems, where the ability to leverage vision to simulate and predict their own dynamics marks a leap towards autonomy and self-awareness. Humans utilize vision to record experiences and internally simulate potential actions. For example, we can imagine that, if we stand up and raise our arms, the body will form a ‘T’ shape without physical movement. Similarly, simulation allows robots to plan and predict the outcomes of potential actions without execution. Here we introduce a self-supervised learning framework to enable robots to model and predict their morphology, kinematics and motor control using only brief raw video data, eliminating the need for extensive real-world data collection and kinematic priors. By observing their own movements, akin to humans watching their reflection in a mirror, robots learn an ability to simulate themselves and predict their spatial motion for various tasks. Our results demonstrate that this self-learned simulation not only enables accurate motion planning but also allows the robot to detect abnormalities and recover from damage. Motion planning for a robot generally requires full knowledge of its structure. Here Hu and colleagues present a method for inferring the structure of a robot from visual information.},
  archive      = {J_NATMI},
  author       = {Hu, Yuhang and Lin, Jiong and Lipson, Hod},
  doi          = {10.1038/s42256-025-01006-w},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {484-494},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Teaching robots to build simulations of themselves},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards unveiling sensitive and decisive patterns in
explainable AI with a case study in geometric deep learning.
<em>NATMI</em>, <em>7</em>(3), 471–483. (<a
href="https://doi.org/10.1038/s42256-025-00998-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The interpretability of machine learning models has gained increasing attention, particularly in scientific domains where high precision and accountability are crucial. This research focuses on distinguishing between two critical data patterns—sensitive patterns (model related) and decisive patterns (task related)—which are commonly used as model interpretations but often lead to confusion. Specifically, this study compares the effectiveness of two main streams of interpretation methods: post-hoc methods and self-interpretable methods, in detecting these patterns. Recently, geometric deep learning (GDL) has shown superior predictive performance in various scientific applications, creating an urgent need for principled interpretation methods. Here, therefore, we conduct our study using several representative GDL applications as case studies. We evaluate 13 interpretation methods applied to 3 major GDL backbone models, using 4 scientific datasets to assess how well these methods identify sensitive and decisive patterns. Our findings indicate that post-hoc methods tend to provide interpretations better aligned with sensitive patterns, whereas certain self-interpretable methods exhibit strong and stable performance in detecting decisive patterns. Moreover, our study offers valuable insights into improving the reliability of these interpretation methods. For example, ensembling post-hoc interpretations from multiple models trained on the same task can effectively uncover the task’s decisive patterns. Interpreting decisions made by machine learning systems remains difficult. Here Zhu et al. test interpretability methods on their ability to identify model-related and task-related patterns.},
  archive      = {J_NATMI},
  author       = {Zhu, Jiajun and Miao, Siqi and Ying, Rex and Li, Pan},
  doi          = {10.1038/s42256-025-00998-9},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {471-483},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Towards unveiling sensitive and decisive patterns in explainable AI with a case study in geometric deep learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Categorizing robots by performance fitness into the tree of
robots. <em>NATMI</em>, <em>7</em>(3), 459–470. (<a
href="https://doi.org/10.1038/s42256-025-00995-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Robots are typically classified based on specific morphological features, like their kinematic structure. However, a complex interplay between morphology and intelligence shapes how well a robot performs processes. Just as delicate surgical procedures demand high dexterity and tactile precision, manual warehouse or construction work requires strength and endurance. These process requirements necessitate robot systems that provide a level of performance fitting the process. In this work, we introduce the tree of robots as a taxonomy to bridge the gap between morphological classification and process-based performance. It classifies robots based on their fitness to perform, for example, physical interaction processes. Using 11 industrial manipulators, we constructed the first part of the tree of robots based on a carefully deduced set of metrics reflecting fundamental robot capabilities for various industrial physical interaction processes. Through significance analysis, we identified substantial differences between the systems, grouping them via an expectation-maximization algorithm to create a fitness-based robot classification that is open for contributions and accessible. It is challenging to compare how well robots perform a task, as the evaluation depends on the process and skills required. It is proposed to group robots into a taxonomy based on their performance on a set of embodied skill benchmarks.},
  archive      = {J_NATMI},
  author       = {Kirschner, Robin Jeanne and Karacan, Kübra and Melone, Alessandro and Haddadin, Sami},
  doi          = {10.1038/s42256-025-00995-y},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {459-470},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Categorizing robots by performance fitness into the tree of robots},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep lead optimization enveloped in protein pocket and its
application in designing potent and selective ligands targeting LTK
protein. <em>NATMI</em>, <em>7</em>(3), 448–458. (<a
href="https://doi.org/10.1038/s42256-025-00997-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optimizing the chemical structure of promising drug candidates through systematic modifications to improve potency and physiochemical properties is a vital step in the drug discovery pipeline. In contrast to the well-established de novo generation schemes, computational methods specifically tailored for lead optimization remain largely underexplored. Prior models are often limited to addressing specific subtasks, such as generating two-dimensional molecular structures, while neglecting crucial protein–ligand interactions in three-dimensional space. To overcome these challenges, we propose Delete (Deep lead optimization enveloped in protein pocket), a one-stop solution for lead optimization by combining generative artificial intelligence and structure-based approaches. Our model can handle all subtasks of lead optimization through a unified deleting (masking) strategy, and it accounts for intricate pocket–ligand interactions through an equivariant network design. Statistical assessments and retrospective studies across individual subtasks demonstrate that Delete has an outstanding ability to craft molecules with superior protein-binding energy and reasonable drug-likeness using given fragments or atoms. Subsequently, we utilize Delete to design inhibitors targeting the previously identified LTK protein. Among the ligands designed by Delete, CA-B-1 is successfully validated as a potent (1.36 nM) and selective inhibitor by in vitro and in vivo experiments. This work represents a successful implementation of the powerful structure-based lead optimization model, Delete, for rapid and controllable rational drug design. Chen et al. present a deep learning-based lead optimization model that combines generative artificial intelligence with structure-based approaches. The method is successfully applied to the design of drug-like molecules targeting the recently identified LTK protein target with high potency and selectivity.},
  archive      = {J_NATMI},
  author       = {Chen, Shicheng and Zhang, Odin and Jiang, Chenran and Zhao, Huifeng and Zhang, Xujun and Chen, Mengting and Liu, Yun and Su, Qun and Wu, Zhenxing and Wang, Xinyue and Qu, Wanglin and Ye, Yuanyi and Chai, Xin and Wang, Ning and Wang, Tianyue and An, Yuan and Wu, Guanlin and Yang, Qianqian and Chen, Jiean and Xie, Wei and Lin, Haitao and Li, Dan and Hsieh, Chang-Yu and Huang, Yong and Kang, Yu and Hou, Tingjun and Pan, Peichen},
  doi          = {10.1038/s42256-025-00997-w},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {448-458},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Deep lead optimization enveloped in protein pocket and its application in designing potent and selective ligands targeting LTK protein},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large language models for scientific discovery in molecular
property prediction. <em>NATMI</em>, <em>7</em>(3), 437–447. (<a
href="https://doi.org/10.1038/s42256-025-00994-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) are a form of artificial intelligence system encapsulating vast knowledge in the form of natural language. These systems are adept at numerous complex tasks including creative writing, storytelling, translation, question-answering, summarization and computer code generation. Although LLMs have seen initial applications in natural sciences, their potential for driving scientific discovery remains largely unexplored. In this work, we introduce LLM4SD, a framework designed to harness LLMs for driving scientific discovery in molecular property prediction by synthesizing knowledge from literature and inferring knowledge from scientific data. LLMs synthesize knowledge by extracting established information from scientific literature, such as molecular weight being key to predicting solubility. For inference, LLMs identify patterns in molecular data, particularly in Simplified Molecular Input Line Entry System-encoded structures, such as halogen-containing molecules being more likely to cross the blood–brain barrier. This information is presented as interpretable knowledge, enabling the transformation of molecules into feature vectors. By using these features with interpretable models such as random forest, LLM4SD can outperform the current state of the art across a range of benchmark tasks for predicting molecular properties. We foresee it providing interpretable and potentially new insights, aiding scientific discovery in molecular property prediction. Zheng et al. developed LLM4SD, a framework using large language models to predict molecular properties. The method leverages the ability of large language models to synthesize knowledge from literature and to reason about scientific data with domain expertise.},
  archive      = {J_NATMI},
  author       = {Zheng, Yizhen and Koh, Huan Yee and Ju, Jiaxin and Nguyen, Anh T. N. and May, Lauren T. and Webb, Geoffrey I. and Pan, Shirui},
  doi          = {10.1038/s42256-025-00994-z},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {437-447},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Large language models for scientific discovery in molecular property prediction},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Data-driven federated learning in drug discovery with
knowledge distillation. <em>NATMI</em>, <em>7</em>(3), 423–436. (<a
href="https://doi.org/10.1038/s42256-025-00991-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A main challenge for artificial intelligence in scientific research is ensuring access to sufficient, high-quality data for the development of impactful models. Despite the abundance of public data, the most valuable knowledge often remains embedded within confidential corporate data silos. Although industries are increasingly open to sharing non-competitive insights, such collaboration is often constrained by the confidentiality of the underlying data. Federated learning makes it possible to share knowledge without compromising data privacy, but it has notable limitations. Here, we introduce FLuID (federated learning using information distillation), a data-centric application of federated distillation tailored to drug discovery aiming to preserve data privacy. We validate FLuID in two experiments, first involving public data simulating a virtual consortium and second in a real-world research collaboration between eight pharmaceutical companies. Although the alignment of the models with the partner specific domain remains challenging, the data-driven nature of FLuID offers several avenues to mitigate domain shift. FLuID fosters knowledge sharing among pharmaceutical organizations, paving the way for a new generation of models with enhanced performance and an expanded applicability domain in biological activity predictions. FLuID enables privacy-preserving knowledge sharing in drug discovery using knowledge distillation. The results show that the approach expands applicability domains and fosters collaboration across organizations without compromising data privacy or security.},
  archive      = {J_NATMI},
  author       = {Hanser, Thierry and Ahlberg, Ernst and Amberg, Alexander and Anger, Lennart T. and Barber, Chris and Brennan, Richard J. and Brigo, Alessandro and Delaunois, Annie and Glowienke, Susanne and Greene, Nigel and Johnston, Laura and Kuhn, Daniel and Kuhnke, Lara and Marchaland, Jean-François and Muster, Wolfgang and Plante, Jeffrey and Rippmann, Friedrich and Sabnis, Yogesh and Schmidt, Friedemann and van Deursen, Ruud and Werner, Stéphane and White, Angela and Wichard, Joerg and Yukawa, Tomoya},
  doi          = {10.1038/s42256-025-00991-2},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {423-436},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Data-driven federated learning in drug discovery with knowledge distillation},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Explainable AI reveals clever hans effects in unsupervised
learning models. <em>NATMI</em>, <em>7</em>(3), 412–422. (<a
href="https://doi.org/10.1038/s42256-025-01000-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unsupervised learning has become an essential building block of artifical intelligence systems. The representations it produces, for example, in foundation models, are critical to a wide variety of downstream applications. It is therefore important to carefully examine unsupervised models to ensure not only that they produce accurate predictions on the available data but also that these accurate predictions do not arise from a Clever Hans (CH) effect. Here, using specially developed explainable artifical intelligence techniques and applying them to popular representation learning and anomaly detection models for image data, we show that CH effects are widespread in unsupervised learning. In particular, through use cases on medical and industrial inspection data, we demonstrate that CH effects systematically lead to significant performance loss of downstream models under plausible dataset shifts or reweighting of different data subgroups. Our empirical findings are enriched by theoretical insights, which point to inductive biases in the unsupervised learning machine as a primary source of CH effects. Overall, our work sheds light on unexplored risks associated with practical applications of unsupervised learning and suggests ways to systematically mitigate CH effects, thereby making unsupervised learning more robust. Building on recent explainable AI techniques, this Article highlights the pervasiveness of Clever Hans effects in unsupervised learning and the substantial risks associated with these effects in terms of the prediction accuracy on new data.},
  archive      = {J_NATMI},
  author       = {Kauffmann, Jacob and Dippel, Jonas and Ruff, Lukas and Samek, Wojciech and Müller, Klaus-Robert and Montavon, Grégoire},
  doi          = {10.1038/s42256-025-01000-2},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {412-422},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Explainable AI reveals clever hans effects in unsupervised learning models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large language models that replace human participants can
harmfully misportray and flatten identity groups. <em>NATMI</em>,
<em>7</em>(3), 400–411. (<a
href="https://doi.org/10.1038/s42256-025-00986-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) are increasing in capability and popularity, propelling their application in new domains—including as replacements for human participants in computational social science, user testing, annotation tasks and so on. In many settings, researchers seek to distribute their surveys to a sample of participants that are representative of the underlying human population of interest. This means that to be a suitable replacement, LLMs will need to be able to capture the influence of positionality (that is, the relevance of social identities like gender and race). However, we show that there are two inherent limitations in the way current LLMs are trained that prevent this. We argue analytically for why LLMs are likely to both misportray and flatten the representations of demographic groups, and then empirically show this on four LLMs through a series of human studies with 3,200 participants across 16 demographic identities. We also discuss a third limitation about how identity prompts can essentialize identities. Throughout, we connect each limitation to a pernicious history of epistemic injustice against the value of lived experiences that explains why replacement is harmful for marginalized demographic groups. Overall, we urge caution in use cases in which LLMs are intended to replace human participants whose identities are relevant to the task at hand. At the same time, in cases where the benefits of LLM replacement are determined to outweigh the harms (for example, engaging human participants may cause them harm, or the goal is to supplement rather than fully replace), we empirically demonstrate that our inference-time techniques reduce—but do not remove—these harms. Large language models are being considered to simulate responses from participants of different backgrounds in computational social science experiments. Here it is shown that this practice can misportray and flatten demographic groups in distinctively harmful ways.},
  archive      = {J_NATMI},
  author       = {Wang, Angelina and Morgenstern, Jamie and Dickerson, John P.},
  doi          = {10.1038/s42256-025-00986-z},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {400-411},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Large language models that replace human participants can harmfully misportray and flatten identity groups},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synergy-based robotic quadruped leveraging passivity for
natural intelligence and behavioural diversity. <em>NATMI</em>,
<em>7</em>(3), 386–399. (<a
href="https://doi.org/10.1038/s42256-025-00988-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quadrupedal animals show remarkable capabilities in traversing diverse terrains and display a range of behaviours and gait patterns. Achieving similar performance by exploiting the natural dynamics of the system is a key goal for robotics researchers. Here we show a bioinspired approach to the design of quadrupeds that seeks to exploit the body and the passive properties of the robot while maintaining active controllability on the system through minimal actuation. Utilizing an end-to-end computational design pipeline, neuromechanical couplings recorded in biological quadrupeds are translated into motor synergies, allowing minimal actuation to control the full structure via multijoint compliant mechanical couplings. Using this approach, we develop PAWS, a passive automata with synergies. By leveraging the principles of motor synergies, the design incorporates variable stiffness, anatomical insights and self-organization to simplify control while maximizing its capabilities. The resulting synergy-based quadruped requires only four actuators and exhibits emergent, animal-like dynamical responses, including passive robustness to environmental perturbations and a wide range of actuated behaviours. The finding contributes to the development of machine physical intelligence and provides robots with more efficient and natural-looking robotic locomotion by combining synergistic actuation, compliant body properties and embodied compensatory strategies. Stella, Achkar and colleagues present a bio-inspired quadruped robot that uses passive dynamics, motor synergies, variable stiffness and self-organization to achieve robust, adaptive, animal-like movement with just four actuators.},
  archive      = {J_NATMI},
  author       = {Stella, Francesco and Achkar, Mickaël M. and Della Santina, Cosimo and Hughes, Josie},
  doi          = {10.1038/s42256-025-00988-x},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {386-399},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Synergy-based robotic quadruped leveraging passivity for natural intelligence and behavioural diversity},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantum circuit optimization with AlphaTensor.
<em>NATMI</em>, <em>7</em>(3), 374–385. (<a
href="https://doi.org/10.1038/s42256-025-01001-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key challenge in realizing fault-tolerant quantum computers is circuit optimization. Focusing on the most expensive gates in fault-tolerant quantum computation (namely, the T gates), we address the problem of T-count optimization, that is, minimizing the number of T gates needed to implement a given circuit. To achieve this, we develop AlphaTensor-Quantum, a method based on deep reinforcement learning that exploits the relationship between optimizing the T-count and tensor decomposition. Unlike existing methods for T-count optimization, AlphaTensor-Quantum can incorporate domain-specific knowledge about quantum computation and leverage gadgets, which substantially reduces the T-count of the optimized circuits. AlphaTensor-Quantum outperforms the existing methods for T-count optimization on a set of arithmetic benchmarks (even when compared without using gadgets). Remarkably, it discovers an efficient algorithm akin to Karatsuba’s method for multiplication in finite fields. AlphaTensor-Quantum also finds the best human-designed solutions for relevant arithmetic computations used in Shor’s algorithm and for quantum chemistry simulation, thus demonstrating that it can save hundreds of hours of research by optimizing relevant quantum circuits in a fully automated way. Ruiz and colleagues introduce AlphaTensor-Quantum, a deep reinforcement learning method for optimizing quantum circuits. It outperforms existing methods and is capable of finding the best human-designed solutions for relevant quantum computations in a fully automated way.},
  archive      = {J_NATMI},
  author       = {Ruiz, Francisco J. R. and Laakkonen, Tuomas and Bausch, Johannes and Balog, Matej and Barekatain, Mohammadamin and Heras, Francisco J. H. and Novikov, Alexander and Fitzpatrick, Nathan and Romera-Paredes, Bernardino and van de Wetering, John and Fawzi, Alhussein and Meichanetzidis, Konstantinos and Kohli, Pushmeet},
  doi          = {10.1038/s42256-025-01001-1},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {374-385},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Quantum circuit optimization with AlphaTensor},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fast, scale-adaptive and uncertainty-aware downscaling of
earth system model fields with generative machine learning.
<em>NATMI</em>, <em>7</em>(3), 363–373. (<a
href="https://doi.org/10.1038/s42256-025-00980-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate and high-resolution Earth system model (ESM) simulations are essential to assess the ecological and socioeconomic impacts of anthropogenic climate change, but are computationally too expensive to be run at sufficiently high spatial resolution. Recent machine learning approaches have shown promising results in downscaling ESM simulations, outperforming state-of-the-art statistical approaches. However, existing methods require computationally costly retraining for each ESM and extrapolate poorly to climates unseen during training. We address these shortcomings by learning a consistency model that efficiently and accurately downscales arbitrary ESM simulations without retraining in a zero-shot manner. Our approach yields probabilistic downscaled fields at a resolution only limited by the observational reference data. We show that the consistency model outperforms state-of-the-art diffusion models at a fraction of the computational cost and maintains high controllability on the downscaling task. Further, our method generalizes to climate states unseen during training without explicitly formulated physical constraints. A generative machine learning approach is proposed to improve the resolution of Earth system models in an efficient, adaptive and uncertainty-aware manner.},
  archive      = {J_NATMI},
  author       = {Hess, Philipp and Aich, Michael and Pan, Baoxiang and Boers, Niklas},
  doi          = {10.1038/s42256-025-00980-5},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {363-373},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Fast, scale-adaptive and uncertainty-aware downscaling of earth system model fields with generative machine learning},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transformers and genome language models. <em>NATMI</em>,
<em>7</em>(3), 346–362. (<a
href="https://doi.org/10.1038/s42256-025-01007-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models based on the transformer deep learning architecture have revolutionized natural language processing. Motivated by the analogy between human language and the genome’s biological code, researchers have begun to develop genome language models (gLMs) based on transformers and related architectures. This Review explores the use of transformers and language models in genomics. We survey open questions in genomics amenable to the use of gLMs, and motivate the use of gLMs and the transformer architecture for these problems. We discuss the potential of gLMs for modelling the genome using unsupervised pretraining tasks, specifically focusing on the power of zero- and few-shot learning. We explore the strengths and limitations of the transformer architecture, as well as the strengths and limitations of current gLMs more broadly. Additionally, we contemplate the future of genomic modelling beyond the transformer architecture, based on current trends in research. This Review serves as a guide for computational biologists and computer scientists interested in transformers and language models for genomic data. Micaela Consens et al. discuss and review the recent rise of transformer-based and large language models in genomics. They also highlight promising directions for genome language models beyond the transformer architecture.},
  archive      = {J_NATMI},
  author       = {Consens, Micaela E. and Dufault, Cameron and Wainberg, Michael and Forster, Duncan and Karimzadeh, Mehran and Goodarzi, Hani and Theis, Fabian J. and Moses, Alan and Wang, Bo},
  doi          = {10.1038/s42256-025-01007-9},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {346-362},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Transformers and genome language models},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Materiality and risk in the age of pervasive AI sensors.
<em>NATMI</em>, <em>7</em>(3), 334–345. (<a
href="https://doi.org/10.1038/s42256-025-01017-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence (AI) systems connected to sensor-laden devices are becoming pervasive, which has notable implications for a range of AI risks, including to privacy, the environment, autonomy and more. There is therefore a growing need for increased accountability around the responsible development and deployment of these technologies. Here we highlight the dimensions of risk associated with AI systems that arise from the material affordances of sensors and their underlying calculative models. We propose a sensor-sensitive framework for diagnosing these risks, complementing existing approaches such as the US National Institute of Standards and Technology AI Risk Management Framework and the European Union AI Act, and discuss its implementation. We conclude by advocating for increased attention to the materiality of algorithmic systems, and of on-device AI sensors in particular, and highlight the need for development of a sensor design paradigm that empowers users and communities and leads to a future of increased fairness, accountability and transparency. Sloane and colleagues review emerging new dimensions of risks associated with materiality and AI algorithms run on pervasive sensors.},
  archive      = {J_NATMI},
  author       = {Sloane, Mona and Moss, Emanuel and Kennedy, Susan and Stewart, Matthew and Warden, Pete and Plancher, Brian and Reddi, Vijay Janapa},
  doi          = {10.1038/s42256-025-01017-7},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {334-345},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Materiality and risk in the age of pervasive AI sensors},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). From data chaos to precision medicine. <em>NATMI</em>,
<em>7</em>(3), 332–333. (<a
href="https://doi.org/10.1038/s42256-025-01015-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More than 30 years have passed since the advent of omics technologies revolutionized biological and medical research. Research now highlights the unique opportunity to integrate and decode complex biological mechanisms for health and diseases with machine learning.},
  archive      = {J_NATMI},
  author       = {Schönhuth, Alexander},
  doi          = {10.1038/s42256-025-01015-9},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {332-333},
  shortjournal = {Nat. Mach. Intell.},
  title        = {From data chaos to precision medicine},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bridging the gap between machine confidence and human
perceptions. <em>NATMI</em>, <em>7</em>(3), 330–331. (<a
href="https://doi.org/10.1038/s42256-025-01013-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Users often overestimate the accuracy of large language models (LLMs). A new approach examines user perceptions and finds that aligning LLM explanations with the models’ internal confidence improves user perception.},
  archive      = {J_NATMI},
  author       = {Yin, Ming},
  doi          = {10.1038/s42256-025-01013-x},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {330-331},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Bridging the gap between machine confidence and human perceptions},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transparency (in training data) is what we want.
<em>NATMI</em>, <em>7</em>(3), 329. (<a
href="https://doi.org/10.1038/s42256-025-01023-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As more powerful generative AI tools appear on the market, legal debates about the use of copyrighted content to develop such tools are intensifying. To resolve these issues, transparency regarding which copyrighted data have been used and where in the AI training pipeline needs to be a starting point.},
  archive      = {J_NATMI},
  doi          = {10.1038/s42256-025-01023-9},
  journal      = {Nature Machine Intelligence},
  month        = {3},
  number       = {3},
  pages        = {329},
  shortjournal = {Nat. Mach. Intell.},
  title        = {Transparency (in training data) is what we want},
  volume       = {7},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="nature---65">NATURE - 65</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Observation of nagaoka polarons in a
fermi–hubbard quantum simulator. <em>NATURE</em>, <em>639</em>(8056),
E23. (<a href="https://doi.org/10.1038/s41586-025-08819-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Lebrat, Martin and Xu, Muqing and Kendrick, Lev Haldar and Kale, Anant and Gang, Youqi and Seetharaman, Pranav and Morera, Ivan and Khatami, Ehsan and Demler, Eugene and Greiner, Markus},
  doi          = {10.1038/s41586-025-08819-0},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {E23},
  shortjournal = {Nature},
  title        = {Author correction: Observation of nagaoka polarons in a Fermi–Hubbard quantum simulator},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Classification with a disordered dopant
atom network in silicon. <em>NATURE</em>, <em>639</em>(8056), E22. (<a
href="https://doi.org/10.1038/s41586-025-08803-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Chen, Tao and van Gelder, Jeroen and van de Ven, Bram and Amitonov, Sergey V. and de Wilde, Bram and Ruiz Euler, Hans-Christian and Broersma, Hajo and Bobbert, Peter A. and Zwanenburg, Floris A. and van der Wiel, Wilfred G.},
  doi          = {10.1038/s41586-025-08803-8},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {E22},
  shortjournal = {Nature},
  title        = {Author correction: Classification with a disordered dopant atom network in silicon},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Transforming a head direction signal into
a goal-oriented steering command. <em>NATURE</em>, <em>639</em>(8056),
E21. (<a href="https://doi.org/10.1038/s41586-024-08245-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Westeinde, Elena A. and Kellogg, Emily and Dawson, Paul M. and Lu, Jenny and Hamburg, Lydia and Midler, Benjamin and Druckmann, Shaul and Wilson, Rachel I.},
  doi          = {10.1038/s41586-024-08245-8},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {E21},
  shortjournal = {Nature},
  title        = {Author correction: Transforming a head direction signal into a goal-oriented steering command},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Warming and cooling catalyse widespread
temporal turnover in biodiversity. <em>NATURE</em>, <em>639</em>(8056),
E20. (<a href="https://doi.org/10.1038/s41586-025-08857-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATURE},
  author       = {Pinsky, Malin L. and Hillebrand, Helmut and Chase, Jonathan M. and Antão, Laura H. and Hirt, Myriam R. and Brose, Ulrich and Burrows, Michael T. and Gauzens, Benoit and Rosenbaum, Benjamin and Blowes, Shane A.},
  doi          = {10.1038/s41586-025-08857-8},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {E20},
  shortjournal = {Nature},
  title        = {Author correction: Warming and cooling catalyse widespread temporal turnover in biodiversity},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reviving the biodiversity around an ancient palace.
<em>NATURE</em>, <em>639</em>(8056), 1110. (<a
href="https://doi.org/10.1038/d41586-025-00921-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Juan Ramón Fernández Cardenete tends the Alhambra’s many water features and gardens. Juan Ramón Fernández Cardenete tends the Alhambra’s many water features and gardens.},
  archive      = {J_NATURE},
  author       = {Mellone, Ugo},
  doi          = {10.1038/d41586-025-00921-7},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1110},
  shortjournal = {Nature},
  title        = {Reviving the biodiversity around an ancient palace},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). RNA function follows form – why is it so hard to predict?
<em>NATURE</em>, <em>639</em>(8056), 1106–1108. (<a
href="https://doi.org/10.1038/d41586-025-00920-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AlphaFold’s highly accurate structural models transformed protein biology,but RNA lags behind. AlphaFold’s highly accurate structural models transformed protein biology,but RNA lags behind.},
  archive      = {J_NATURE},
  author       = {Kwon, Diana},
  doi          = {10.1038/d41586-025-00920-8},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1106-1108},
  shortjournal = {Nature},
  title        = {RNA function follows form – why is it so hard to predict?},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Shaking up earthquake safety in nepal with a folk song.
<em>NATURE</em>, <em>639</em>(8056), 1103–1105. (<a
href="https://doi.org/10.1038/d41586-025-00919-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Geophysicist Shiba Subedi combines science and songwriting to raise awareness and debunk earthquake myths. Geophysicist Shiba Subedi combines science and songwriting to raise awareness and debunk earthquake myths.},
  archive      = {J_NATURE},
  author       = {Bolakhe, Saugat},
  doi          = {10.1038/d41586-025-00919-1},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1103-1105},
  shortjournal = {Nature},
  title        = {Shaking up earthquake safety in nepal with a folk song},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structure and mechanism of the zorya anti-phage defence
system. <em>NATURE</em>, <em>639</em>(8056), 1093–1101. (<a
href="https://doi.org/10.1038/s41586-024-08493-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zorya is a recently identified and widely distributed bacterial immune system that protects bacteria from viral (phage) infections. Three Zorya subtypes have been identified, each containing predicted membrane-embedded ZorA–ZorB (ZorAB) complexes paired with soluble subunits that differ among Zorya subtypes, notably ZorC and ZorD in type I Zorya systems1,2. Here we investigate the molecular basis of Zorya defence using cryo-electron microscopy, mutagenesis, fluorescence microscopy, proteomics and functional studies. We present cryo-electron microscopy structures of ZorAB and show that it shares stoichiometry and features of other 5:2 inner membrane ion-driven rotary motors. The ZorA5B2 complex contains a dimeric ZorB peptidoglycan-binding domain and a pentameric α-helical coiled-coil tail made of ZorA that projects approximately 70 nm into the cytoplasm. We also characterize the structure and function of the soluble Zorya components ZorC and ZorD, finding that they have DNA-binding and nuclease activity, respectively. Comprehensive functional and mutational analyses demonstrate that all Zorya components work in concert to protect bacterial cells against invading phages. We provide evidence that ZorAB operates as a proton-driven motor that becomes activated after sensing of phage invasion. Subsequently, ZorAB transfers the phage invasion signal through the ZorA cytoplasmic tail to recruit and activate the soluble ZorC and ZorD effectors, which facilitate the degradation of the phage DNA. In summary, our study elucidates the foundational mechanisms of Zorya function as an anti-phage defence system. Zorya uses a membrane-anchored H+-driven rotary motor ZorAB to sense phage invasion and to recruit the intracellular effectors ZorC and ZorD that execute anti-phage defence.},
  archive      = {J_NATURE},
  author       = {Hu, Haidai and Popp, Philipp F. and Hughes, Thomas C. D. and Roa-Eguiara, Aritz and Rutbeek, Nicole R. and Martin, Freddie J. O. and Hendriks, Ivo Alexander and Payne, Leighton J. and Yan, Yumeng and Humolli, Dorentina and Klein-Sousa, Victor and Songailiene, Inga and Wang, Yong and Nielsen, Michael Lund and Berry, Richard M. and Harms, Alexander and Erhardt, Marc and Jackson, Simon A. and Taylor, Nicholas M. I.},
  doi          = {10.1038/s41586-024-08493-8},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1093-1101},
  shortjournal = {Nature},
  title        = {Structure and mechanism of the zorya anti-phage defence system},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mechanism for local attenuation of DNA replication at
double-strand breaks. <em>NATURE</em>, <em>639</em>(8056), 1084–1092.
(<a href="https://doi.org/10.1038/s41586-024-08557-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA double-strand breaks (DSBs) disrupt the continuity of the genome, with consequences for malignant transformation. Massive DNA damage can elicit a cellular checkpoint response that prevents cell proliferation1,2. However, how highly aggressive cancer cells, which can tolerate widespread DNA damage, respond to DSBs alongside continuous chromosome duplication is unknown. Here we show that DSBs induce a local genome maintenance mechanism that inhibits replication initiation in DSB-containing topologically associating domains (TADs) without affecting DNA synthesis at other genomic locations. This process is facilitated by mediators of replication and DSBs (MRDs). In normal and cancer cells, MRDs include the TIMELESS–TIPIN complex and the WEE1 kinase, which actively dislodges the TIMELESS–TIPIN complex from replication origins adjacent to DSBs and prevents initiation of DNA synthesis at DSB-containing TADs. Dysregulation of MRDs, or disruption of 3D chromatin architecture by dissolving TADs, results in inadvertent replication in damaged chromatin and increased DNA damage in cancer cells. We propose that the intact MRD cascade precedes DSB repair to prevent genomic instability, which is otherwise observed when replication is forced, or when genome architecture is challenged, in the presence of DSBs3–5. These observations reveal a previously unknown vulnerability in the DNA replication machinery that may be exploited to therapeutically target cancer cells. DNA double-strand breaks induce local genome maintenance and inhibition of replication initiation at nearby topologically associating domains without affecting global DNA synthesis.},
  archive      = {J_NATURE},
  author       = {Sebastian, Robin and Sun, Eric G. and Fedkenheuer, Michael and Fu, Haiqing and Jung, SeolKyoung and Thakur, Bhushan L. and Redon, Christophe E. and Pegoraro, Gianluca and Tran, Andy D. and Gross, Jacob M. and Mosavarpour, Sara and Kusi, Nana Afua and Ray, Anagh and Dhall, Anjali and Pongor, Lorinc S. and Casellas, Rafael and Aladjem, Mirit I.},
  doi          = {10.1038/s41586-024-08557-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1084-1092},
  shortjournal = {Nature},
  title        = {Mechanism for local attenuation of DNA replication at double-strand breaks},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A metagenomic “dark matter” enzyme catalyses oxidative
cellulose conversion. <em>NATURE</em>, <em>639</em>(8056), 1076–1083.
(<a href="https://doi.org/10.1038/s41586-024-08553-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The breakdown of cellulose is one of the most important reactions in nature1,2 and is central to biomass conversion to fuels and chemicals3. However, the microfibrillar organization of cellulose and its complex interactions with other components of the plant cell wall poses a major challenge for enzymatic conversion4. Here, by mining the metagenomic ‘dark matter’ (unclassified DNA with unknown function) of a microbial community specialized in lignocellulose degradation, we discovered a metalloenzyme that oxidatively cleaves cellulose. This metalloenzyme acts on cellulose through an exo-type mechanism with C1 regioselectivity, resulting exclusively in cellobionic acid as a product. The crystal structure reveals a catalytic copper buried in a compact jelly-roll scaffold that features a flattened cellulose binding site. This metalloenzyme exhibits a homodimeric configuration that enables in situ hydrogen peroxide generation by one subunit while the other is productively interacting with cellulose. The secretome of an engineered strain of the fungus Trichoderma reesei expressing this metalloenzyme boosted the glucose release from pretreated lignocellulosic biomass under industrially relevant conditions, demonstrating its biotechnological potential. This discovery modifies the current understanding of bacterial redox enzymatic systems devoted to overcoming biomass recalcitrance5–7. Furthermore, it enables the conversion of agro-industrial residues into value-added bioproducts, thereby contributing to the transition to a sustainable and bio-based economy. A metalloenzyme capable of oxidatively cleaving cellulose, found in a microbial community specialized in lignocellulose degradation, could enable sustainable biofuel production.},
  archive      = {J_NATURE},
  author       = {Santos, Clelton A. and Morais, Mariana A. B. and Mandelli, Fernanda and Lima, Evandro A. and Miyamoto, Renan Y. and Higasi, Paula M. R. and Araujo, Evandro A. and Paixão, Douglas A. A. and Junior, Joaquim M. and Motta, Maria L. and Streit, Rodrigo S. A. and Morão, Luana G. and Silva, Claudio B. C. and Wolf, Lucia D. and Terrasan, Cesar R. F. and Bulka, Nathalia R. and Diogo, Jose A. and Fuzita, Felipe J. and Colombari, Felippe M. and Santos, Camila R. and Rodrigues, Priscila T. and Silva, Daiane B. and Grisel, Sacha and Bernardes, Juliana S. and Terrapon, Nicolas and Lombard, Vincent and Filho, Antonio J. C. and Henrissat, Bernard and Bissaro, Bastien and Berrin, Jean-Guy and Persinoti, Gabriela F. and Murakami, Mario T.},
  doi          = {10.1038/s41586-024-08553-z},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1076-1083},
  shortjournal = {Nature},
  title        = {A metagenomic ‘dark matter’ enzyme catalyses oxidative cellulose conversion},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Plasmodium blood stage development requires the chromatin
remodeller Snf2L. <em>NATURE</em>, <em>639</em>(8056), 1069–1075. (<a
href="https://doi.org/10.1038/s41586-025-08595-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The complex life cycle of the malaria parasite Plasmodium falciparum involves several major differentiation stages, each requiring strict control of gene expression. Fundamental changes in chromatin structure and epigenetic modifications during life cycle progression suggest a central role for these mechanisms in regulating the transcriptional program of malaria parasite development1–6. P. falciparum chromatin is distinct from other eukaryotes, with an extraordinarily high AT content (&amp;gt;80%)7 and highly divergent histones resulting in atypical DNA packaging properties8. Moreover, the chromatin remodellers that are critical for shaping chromatin structure are not conserved and are unexplored in P. falciparum. Here we identify P. falciparum Snf2L (PfSnf2L, encoded by PF3D7_1104200) as an ISWI-related ATPase that actively repositions P. falciparum nucleosomes in vitro. Our results demonstrate that PfSnf2L is essential, regulating both asexual development and sexual differentiation. PfSnf2L globally controls just-in-time transcription by spatiotemporally determining nucleosome positioning at the promoters of stage-specific genes. The unique sequence and functional properties of PfSnf2L led to the identification of an inhibitor that specifically kills P. falciparum and phenocopies the loss of correct gene expression timing. The inhibitor represents a new class of antimalarial transmission-blocking drugs, inhibiting gametocyte formation. Plasmodium falciparum Snf2L is an ISWI-related ATPase that actively repositions P. falciparum nucleosomes in vivo.},
  archive      = {J_NATURE},
  author       = {Watzlowik, Maria Theresia and Silberhorn, Elisabeth and Das, Sujaan and Singhal, Ritwik and Venugopal, Kannan and Holzinger, Simon and Stokes, Barbara and Schadt, Ella and Sollelis, Lauriane and Bonnell, Victoria A. and Gow, Matthew and Klingl, Andreas and Marti, Matthias and Llinás, Manuel and Meissner, Markus and Längst, Gernot},
  doi          = {10.1038/s41586-025-08595-x},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1069-1075},
  shortjournal = {Nature},
  title        = {Plasmodium blood stage development requires the chromatin remodeller Snf2L},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). GABAergic neuron-to-glioma synapses in diffuse midline
gliomas. <em>NATURE</em>, <em>639</em>(8056), 1060–1068. (<a
href="https://doi.org/10.1038/s41586-024-08579-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-grade gliomas (HGGs) are the leading cause of brain cancer-related death. HGGs include clinically, anatomically and molecularly distinct subtypes that stratify into diffuse midline gliomas (DMGs), such as H3K27M-altered diffuse intrinsic pontine glioma, and hemispheric HGGs, such as IDH wild-type glioblastoma. Neuronal activity drives glioma progression through paracrine signalling1,2 and neuron-to-glioma synapses3–6. Glutamatergic AMPA receptor-dependent synapses between neurons and glioma cells have been demonstrated in paediatric3 and adult4 high-grade gliomas, and early work has suggested heterogeneous glioma GABAergic responses7. However, neuron-to-glioma synapses mediated by neurotransmitters other than glutamate remain understudied. Using whole-cell patch-clamp electrophysiology, in vivo optogenetics and patient-derived orthotopic xenograft models, we identified functional, tumour-promoting GABAergic neuron-to-glioma synapses mediated by GABAA receptors in DMGs. GABAergic input has a depolarizing effect on DMG cells due to NKCC1 chloride transporter function and consequently elevated intracellular chloride concentration in DMG malignant cells. As membrane depolarization increases glioma proliferation3,6, we found that the activity of GABAergic interneurons promotes DMG proliferation in vivo. The benzodiazepine lorazepam enhances GABA-mediated signalling, increases glioma proliferation and growth, and shortens survival in DMG patient-derived orthotopic xenograft models. By contrast, only minimal&amp;nbsp;depolarizing GABAergic currents were found in hemispheric HGGs and lorazepam did not influence the growth rate of hemispheric glioblastoma xenografts. Together, these findings uncover growth-promoting GABAergic synaptic communication between GABAergic neurons and H3K27M-altered DMG cells, underscoring a tumour subtype-specific mechanism of brain cancer neurophysiology. Functional, tumour-promoting GABAergic neuron-to-glioma synapses in diffuse midline gliomas are identified.},
  archive      = {J_NATURE},
  author       = {Barron, Tara and Yalçın, Belgin and Su, Minhui and Byun, Youkyeong Gloria and Gavish, Avishai and Shamardani, Kiarash and Xu, Haojun and Ni, Lijun and Soni, Neeraj and Mehta, Vilina and Maleki Jahan, Samin and Kim, Yoon Seok and Taylor, Kathryn R. and Keough, Michael B. and Quezada, Michael A. and Geraghty, Anna C. and Mancusi, Rebecca and Vo, Linh Thuy and Castañeda, Enrique Herrera and Woo, Pamelyn J. and Petritsch, Claudia K. and Vogel, Hannes and Kaila, Kai and Monje, Michelle},
  doi          = {10.1038/s41586-024-08579-3},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1060-1068},
  shortjournal = {Nature},
  title        = {GABAergic neuron-to-glioma synapses in diffuse midline gliomas},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Clonal driver neoantigen loss under EGFR TKI and immune
selection pressures. <em>NATURE</em>, <em>639</em>(8056), 1052–1059. (<a
href="https://doi.org/10.1038/s41586-025-08586-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies. We show the evolution of a case of EGFR mutant lung cancer treated with a combination of erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine targeting somatic mutations, including EGFR exon 19 deletion.},
  archive      = {J_NATURE},
  author       = {Al Bakir, Maise and Reading, James L. and Gamble, Samuel and Rosenthal, Rachel and Uddin, Imran and Rowan, Andrew and Przewrocka, Joanna and Rogers, Amber and Wong, Yien Ning Sophia and Bentzen, Amalie K. and Veeriah, Selvaraju and Ward, Sophia and Garnett, Aaron T. and Kalavakur, Paula and Martínez-Ruiz, Carlos and Puttick, Clare and Huebner, Ariana and Cook, Daniel E. and Moore, David A. and Abbosh, Chris and Hiley, Crispin T. and Naceur-Lombardelli, Cristina and Watkins, Thomas B. K. and Petkovic, Marina and Schwarz, Roland F. and Gálvez-Cancino, Felipe and Litchfield, Kevin and Meldgaard, Peter and Sorensen, Boe Sandahl and Madsen, Line Bille and Jäger, Dirk and Forster, Martin D. and Arkenau, Tobias and Domingo-Vila, Clara and Tree, Timothy I. M. and Kadivar, Mohammad and Hadrup, Sine Reker and Chain, Benny and Quezada, Sergio A. and McGranahan, Nicholas and Swanton, Charles},
  doi          = {10.1038/s41586-025-08586-y},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1052-1059},
  shortjournal = {Nature},
  title        = {Clonal driver neoantigen loss under EGFR TKI and immune selection pressures},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). RNA neoantigen vaccines prime long-lived CD8+ t cells in
pancreatic cancer. <em>NATURE</em>, <em>639</em>(8056), 1042–1051. (<a
href="https://doi.org/10.1038/s41586-024-08508-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fundamental challenge for cancer vaccines is to generate long-lived functional T cells that are specific for tumour antigens. Here we find that mRNA–lipoplex vaccines against somatic mutation-derived neoantigens may solve this challenge in pancreatic ductal adenocarcinoma (PDAC), a lethal cancer with few mutations. At an extended 3.2-year median follow-up from a phase 1 trial of surgery, atezolizumab (PD-L1 inhibitory antibody), autogene cevumeran1 (individualized neoantigen vaccine with backbone-optimized uridine mRNA–lipoplex nanoparticles) and modified (m) FOLFIRINOX (chemotherapy) in patients with PDAC, we find that responders with vaccine-induced T cells (n = 8) have prolonged recurrence-free survival (RFS; median not reached) compared with non-responders without vaccine-induced T cells (n = 8; median RFS 13.4 months; P  =  0.007). In responders, autogene cevumeran induces CD8+ T cell clones with an average estimated lifespan of 7.7 years (range 1.5 to roughly 100 years), with approximately 20% of clones having latent multi-decade lifespans that may outlive hosts. Eighty-six percent&amp;nbsp;of clones per patient persist at substantial frequencies approximately 3 years post-vaccination, including clones with high avidity to PDAC neoepitopes. Using PhenoTrack, a novel computational strategy to trace single T cell phenotypes, we uncover that vaccine-induced clones are undetectable in pre-vaccination tissues, and assume a cytotoxic, tissue-resident memory-like T cell state up to three years post-vaccination with preserved neoantigen-specific effector function. Two responders recurred and evidenced fewer vaccine-induced T cells. Furthermore, recurrent PDACs were pruned of vaccine-targeted cancer clones. Thus, in PDAC, autogene cevumeran induces de novo CD8+ T cells with multiyear longevity, substantial magnitude and durable effector functions that may delay PDAC recurrence. Adjuvant mRNA–lipoplex neoantigen vaccines may thus solve a pivotal obstacle for cancer vaccination. In a phase 1 trial, patients with pancreatic ductal adenocarcinoma who were treated with surgery and bespoke neoantigen&amp;nbsp;mRNA vaccines combined with anti-PD-L1 and chemotherapy exhibited marked long-lived&amp;nbsp;persistence of neoantigen-specific CD8+ T cell clones, which correlated with prolonged recurrence-free survival at a 3.2-year follow-up.},
  archive      = {J_NATURE},
  author       = {Sethna, Zachary and Guasp, Pablo and Reiche, Charlotte and Milighetti, Martina and Ceglia, Nicholas and Patterson, Erin and Lihm, Jayon and Payne, George and Lyudovyk, Olga and Rojas, Luis A. and Pang, Nan and Ohmoto, Akihiro and Amisaki, Masataka and Zebboudj, Abderezak and Odgerel, Zagaa and Bruno, Emmanuel M. and Zhang, Siqi Linsey and Cheng, Charlotte and Elhanati, Yuval and Derhovanessian, Evelyna and Manning, Luisa and Müller, Felicitas and Rhee, Ina and Yadav, Mahesh and Merghoub, Taha and Wolchok, Jedd D. and Basturk, Olca and Gönen, Mithat and Epstein, Andrew S. and Momtaz, Parisa and Park, Wungki and Sugarman, Ryan and Varghese, Anna M. and Won, Elizabeth and Desai, Avni and Wei, Alice C. and D’Angelica, Michael I. and Kingham, T. Peter and Soares, Kevin C. and Jarnagin, William R. and Drebin, Jeffrey and O’Reilly, Eileen M. and Mellman, Ira and Sahin, Ugur and Türeci, Özlem and Greenbaum, Benjamin D. and Balachandran, Vinod P.},
  doi          = {10.1038/s41586-024-08508-4},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1042-1051},
  shortjournal = {Nature},
  title        = {RNA neoantigen vaccines prime long-lived CD8+ t cells in pancreatic cancer},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cell-autonomous innate immunity by proteasome-derived
defence peptides. <em>NATURE</em>, <em>639</em>(8056), 1032–1041. (<a
href="https://doi.org/10.1038/s41586-025-08615-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For decades, antigen presentation on major histocompatibility complex class I&amp;nbsp;for T cell-mediated immunity has been considered the primary function of proteasome-derived peptides1,2. However, whether the products of proteasomal degradation play additional parts in mounting immune responses remains unknown. Antimicrobial peptides serve as a first line of defence against invading pathogens before the adaptive immune system responds. Although the protective function of antimicrobial peptides across numerous tissues is well established, the cellular mechanisms underlying their generation are not fully understood. Here we uncover a role for proteasomes in the constitutive and bacterial-induced generation of defence peptides that impede bacterial growth both in vitro and in vivo by disrupting bacterial membranes. In silico prediction of proteome-wide proteasomal cleavage identified hundreds of thousands of potential proteasome-derived defence peptides with cationic properties that may be generated en route to degradation to act as a first line of defence. Furthermore, bacterial infection induces changes in proteasome composition and function, including PSME3 recruitment and increased tryptic-like cleavage, enhancing antimicrobial activity. Beyond providing mechanistic insights into the role of proteasomes in cell-autonomous innate immunity, our study suggests that proteasome-cleaved peptides may have previously overlooked functions downstream of degradation. From a translational standpoint, identifying proteasome-derived defence peptides could provide an untapped source of natural antibiotics for biotechnological applications and therapeutic interventions in infectious diseases and immunocompromised conditions. Proteasomal degradation of cellular proteins&amp;nbsp;generate defence peptides constitutively and in response to bacterial infection.&amp;nbsp;Such peptides might provide a source of natural antibiotics that could lead to biotechnology applications and therapeutic interventions.},
  archive      = {J_NATURE},
  author       = {Goldberg, Karin and Lobov, Arseniy and Antonello, Paola and Shmueli, Merav D. and Yakir, Idan and Weizman, Tal and Ulman, Adi and Sheban, Daoud and Laser, Einav and Kramer, Matthias P. and Shteinvil, Ronen and Chen, Guoyun and Ibraheem, Angham and Sysoeva, Vera and Fishbain-Yoskovitz, Vered and Mohapatra, Gayatree and Abramov, Anat and Shimshi, Sandy and Ogneva, Kseniia and Nandy, Madhurima and Amidror, Sivan and Bootz-Maoz, Hadar and Kuo, Shanny H. and Dezorella, Nili and Kacen, Assaf and Javitt, Aaron and Lau, Gee W. and Yissachar, Nissan and Hayouka, Zvi and Merbl, Yifat},
  doi          = {10.1038/s41586-025-08615-w},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1032-1041},
  shortjournal = {Nature},
  title        = {Cell-autonomous innate immunity by proteasome-derived defence peptides},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Differential protection against SARS-CoV-2 reinfection pre-
and post-omicron. <em>NATURE</em>, <em>639</em>(8056), 1024–1031. (<a
href="https://doi.org/10.1038/s41586-024-08511-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has rapidly evolved over short timescales, leading to the emergence of more transmissible variants such as Alpha and Delta1–3. The arrival of the Omicron variant marked a major shift, introducing numerous extra mutations in the spike gene compared with earlier variants1,2. These evolutionary changes have raised concerns regarding their potential impact on immune evasion, disease severity and the effectiveness of vaccines and treatments1,3. In this epidemiological study, we identified two distinct patterns in the protective effect of natural infection against reinfection in the Omicron versus pre-Omicron eras. Before Omicron, natural infection provided strong and durable protection against reinfection, with minimal waning over time. However, during the Omicron era, protection was robust only for those recently infected, declining rapidly over time and diminishing within a year. These results demonstrate that SARS-CoV-2 immune protection is shaped by a dynamic interaction between host immunity and viral evolution, leading to contrasting reinfection patterns before and after Omicron’s first wave. This shift in patterns suggests a change in evolutionary pressures, with intrinsic transmissibility driving adaptation pre-Omicron and immune escape becoming dominant post-Omicron, underscoring the need for periodic vaccine updates to sustain immunity. Two distinct patterns in the protective effect of natural infection against reinfection in the Omicron variant versus pre-Omicron eras show that SARS-CoV-2 immune protection is shaped by dynamic interaction between host immunity and viral evolution.},
  archive      = {J_NATURE},
  author       = {Chemaitelly, Hiam and Ayoub, Houssein H. and Coyle, Peter and Tang, Patrick and Hasan, Mohammad R. and Yassine, Hadi M. and Al Thani, Asmaa A. and Al-Kanaani, Zaina and Al-Kuwari, Einas and Jeremijenko, Andrew and Kaleeckal, Anvar Hassan and Latif, Ali Nizar and Shaik, Riyazuddin Mohammad and Abdul-Rahim, Hanan F. and Nasrallah, Gheyath K. and Al-Kuwari, Mohamed Ghaith and Butt, Adeel A. and Al-Romaihi, Hamad Eid and Al-Thani, Mohamed H. and Al-Khal, Abdullatif and Bertollini, Roberto and Abu-Raddad, Laith J.},
  doi          = {10.1038/s41586-024-08511-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1024-1031},
  shortjournal = {Nature},
  title        = {Differential protection against SARS-CoV-2 reinfection pre- and post-omicron},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Clonal candida auris and ESKAPE pathogens on the skin of
residents of nursing homes. <em>NATURE</em>, <em>639</em>(8056),
1016–1023. (<a
href="https://doi.org/10.1038/s41586-025-08608-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antimicrobial resistance is a public health threat associated with increased morbidity, mortality and financial burden in nursing homes and other healthcare settings1. Residents of nursing homes are at increased risk of pathogen colonization and infection owing to antimicrobial-resistant bacteria and fungi. Nursing homes act as reservoirs, amplifiers and disseminators of antimicrobial resistance in healthcare networks and across geographical regions2. Here we investigate the genomic epidemiology of the emerging, multidrug-resistant human fungal pathogen Candida auris in a ventilator-capable nursing home. Coupling&amp;nbsp;strain-resolved metagenomics with isolate sequencing, we report skin&amp;nbsp;colonization and clonal spread of C. auris on the skin of nursing home residents and&amp;nbsp;throughout a metropolitan region. We also report that&amp;nbsp;most Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Entobacter species (ESKAPE) pathogens and other high-priority pathogens (including Escherichia coli, Providencia stuartii, Proteus mirabilis and Morganella morganii) are shared in a nursing home. Integrating microbiome and clinical microbiology data, we detect carbapenemase genes at multiple skin sites on residents identified as carriers of these genes. We analyse publicly available shotgun metagenomic samples (stool and skin) collected from residents with varying medical conditions living in seven other nursing homes and provide additional evidence of previously unappreciated bacterial strain sharing. Taken together, our data suggest that skin is a reservoir for colonization by C. auris and ESKAPE pathogens and their associated antimicrobial-resistance genes. Analyses of strain-resolved metagenomics with isolate sequencing data of skin samples from residents at nursing homes suggest that skin is a reservoir for Candida auris and other multidrug-resistant bacterial&amp;nbsp;species.},
  archive      = {J_NATURE},
  author       = {Proctor, Diana M. and Sansom, Sarah E. and Deming, Clay and Conlan, Sean and Blaustein, Ryan A. and Atkins, Thomas K. and Dangana, Thelma and Fukuda, Christine and Thotapalli, Lahari and Kong, Heidi H. and Lin, Michael Y. and Hayden, Mary K. and Segre, Julia A.},
  doi          = {10.1038/s41586-025-08608-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1016-1023},
  shortjournal = {Nature},
  title        = {Clonal candida auris and ESKAPE pathogens on the skin of residents of nursing homes},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A crucial role for the cortical amygdala in shaping social
encounters. <em>NATURE</em>, <em>639</em>(8056), 1006–1015. (<a
href="https://doi.org/10.1038/s41586-024-08540-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aggression is an evolutionarily conserved behaviour that controls social hierarchies and protects valuable resources. In mice, aggressive behaviour can be broken down into an appetitive phase, which involves approach and investigation, and a consummatory phase, which involves biting, kicking and wrestling1. Here, by performing an unsupervised weighted correlation network analysis on whole-brain FOS expression in mice, we identify a cluster of brain regions, including hypothalamic and amygdalar subregions and olfactory cortical regions, that are highly co-activated in male but not in female aggressors. The posterolateral cortical amygdala (COApl)—an extended olfactory structure—was found to be a hub region, on the basis of the number and strength of correlations with other regions in the cluster. Our data also show that oestrogen receptor 1 (Esr1)-expressing cells in the COApl (COAplEsr1) exhibit increased activity during attack behaviour and during bouts of investigation that precede an attack, in male mice only. Chemogenetic or optogenetic inhibition of COAplEsr1 cells in male aggressors reduces aggression and increases pro-social investigation without affecting social reward and reinforcement behaviour. We further show that COAplEsr1 projections to the ventromedial hypothalamus and central amygdala are necessary for these behaviours. Collectively, these data suggest that, in aggressive males, COAplEsr1 cells respond specifically to social stimuli, thereby enhancing their salience and promoting attack behaviour. The posterolateral cortical amygdala and other connected brain regions have a key role in mediating the transition from investigative to aggressive behaviour in male mice.},
  archive      = {J_NATURE},
  author       = {Aubry, Antonio V. and Durand-de Cuttoli, Romain and Karpman, Elizabeth and Fisher-Foye, Rachel L. and Parise, Lyonna F. and Cathomas, Flurin and Burnett, C. Joseph and Yang, Yewon and Yuan, Chongzhen and LaBanca, Alexa R. and Chan, Kenny L. and Winston, Kion T. and Lin, Hsiao-yun and Dackour, Farah and Tavallaei, Arman A. and Alvarez, Johana and Nishioka, Tadaaki and Morishita, Hirofumi and Froemke, Robert C. and Li, Long and Russo, Scott J.},
  doi          = {10.1038/s41586-024-08540-4},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {1006-1015},
  shortjournal = {Nature},
  title        = {A crucial role for the cortical amygdala in shaping social encounters},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Left–right-alternating theta sweeps in
entorhinal–hippocampal maps of space. <em>NATURE</em>,
<em>639</em>(8056), 995–1005. (<a
href="https://doi.org/10.1038/s41586-024-08527-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Place cells in the hippocampus and grid cells in the entorhinal cortex are elements of a neural map of self position1–5. For these cells to benefit navigation, their representation must be dynamically related to the surrounding locations2. A candidate mechanism for linking places along an animal’s path has been described for place cells, in which the sequence of spikes in each cycle of the hippocampal theta oscillation encodes a trajectory from the animal’s current location towards upcoming locations6–8. In mazes that bifurcate, such trajectories alternately traverse the two upcoming arms when the animal approaches the choice point9,10, raising the possibility that the trajectories express available forward paths encoded on previous trials10. However, to bridge the animal’s path with the wider environment, beyond places previously or subsequently visited, an experience-independent spatial sampling mechanism might be required. Here we show in freely moving rats that in individual theta cycles, ensembles of grid cells and place cells encode a position signal that sweeps linearly outwards from the animal’s location into the ambient environment, with sweep direction alternating stereotypically between left and right across successive theta cycles. These sweeps are accompanied by, and aligned with, a similarly alternating directional signal in a discrete population of parasubiculum cells that have putative connections to grid cells via conjunctive grid × direction cells. Sweeps extend into never-visited locations that&amp;nbsp;are inaccessible to the animal. Sweeps persist during REM sleep. The sweep directions can be explained by an algorithm that maximizes the cumulative coverage of the surrounding manifold space. The sustained and unconditional expression of theta-patterned left–right-alternating sweeps in the entorhinal–hippocampal positioning system provides an efficient ‘look around’ mechanism for sampling locations beyond the travelled path. A study in rats proposes a mechanism for how the brain maps the surrounding environment, including places it has never seen, by alternating left and right forward sweeps in successive theta cycles.},
  archive      = {J_NATURE},
  author       = {Vollan, Abraham Z. and Gardner, Richard J. and Moser, May-Britt and Moser, Edvard I.},
  doi          = {10.1038/s41586-024-08527-1},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {995-1005},
  shortjournal = {Nature},
  title        = {Left–right-alternating theta sweeps in entorhinal–hippocampal maps of space},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Glycocalyx dysregulation impairs blood–brain barrier in
ageing and disease. <em>NATURE</em>, <em>639</em>(8056), 985–994. (<a
href="https://doi.org/10.1038/s41586-025-08589-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The blood–brain barrier (BBB) is highly specialized to protect the brain from harmful circulating factors in the blood and maintain brain homeostasis1,2. The brain endothelial glycocalyx layer, a carbohydrate-rich meshwork composed primarily of proteoglycans, glycoproteins and glycolipids that coats the BBB lumen, is a key structural component of the BBB3,4. This layer forms the first interface between the blood and brain vasculature, yet little is known about its composition and roles in supporting BBB function in homeostatic and diseased states. Here we find that the brain endothelial glycocalyx is highly dysregulated during ageing and neurodegenerative disease. We identify significant perturbation in an underexplored class of densely O-glycosylated proteins known as mucin-domain glycoproteins. We demonstrate that ageing- and disease-associated aberrations in brain endothelial mucin-domain glycoproteins lead to dysregulated BBB function and, in severe cases, brain haemorrhaging in mice. Finally, we demonstrate that we can improve BBB function and reduce neuroinflammation and cognitive deficits in aged mice by restoring core 1 mucin-type O-glycans to the brain endothelium using adeno-associated viruses. Cumulatively, our findings provide a detailed compositional and structural mapping of the ageing brain endothelial glycocalyx layer and reveal important consequences of ageing- and disease-associated glycocalyx dysregulation on BBB integrity and brain health. Disruption of mucin-domain glycoprotein expression and function in the endothelial glycocalyx are associated with ageing and Alzheimer’s disease, leading to dysregulated blood–brain barrier function.},
  archive      = {J_NATURE},
  author       = {Shi, Sophia M. and Suh, Ryan J. and Shon, D. Judy and Garcia, Francisco J. and Buff, Josephine K. and Atkins, Micaiah and Li, Lulin and Lu, Nannan and Sun, Bryan and Luo, Jian and To, Ning-Sum and Cheung, Tom H. and McNerney, M. Windy and Heiman, Myriam and Bertozzi, Carolyn R. and Wyss-Coray, Tony},
  doi          = {10.1038/s41586-025-08589-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {985-994},
  shortjournal = {Nature},
  title        = {Glycocalyx dysregulation impairs blood–brain barrier in ageing and disease},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Measuring racial educational disparities over time amongst
top achievers. <em>NATURE</em>, <em>639</em>(8056), 976–984. (<a
href="https://doi.org/10.1038/s41586-024-08536-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Educational disparities remain a key contributor to increasing social and wealth inequalities. To address this, researchers and policymakers have focused on average differences between racial groups or differences among students who are falling behind1. This focus potentially leads to educational triage, diverting resources away from high-achieving students, including those from racial minorities2,3. Here we focus on the ‘racial excellence gap’—the difference in the likelihood that students from racial minorities (Black and Hispanic) reach the highest levels of academic achievement compared with their non-minority (white and Asian) peers. There is a shortage of evidence that systematically measures the magnitude of the excellence gap and how it evolves4,5. Using longitudinal, statewide, administrative data, we document eight facts regarding the excellence gap from third grade (typically&amp;nbsp;ages 8–9) to high school (typically&amp;nbsp;ages 14–18), link the stability of excellence gaps and student backgrounds, and assess the efficacy of public policies. We show that excellence gaps in maths and reading are evident by the third grade and grow slightly over time, especially for female students. About one third of the gap is explained by a student’s socioeconomic status, and about one tenth is explained by the school environment. Top-achieving racial minority students are also less likely to persist in excellence as they progress through school. Moreover, state accountability policies that direct additional resources to reduce non-race-based inequality6 had minimal effects on the racial excellence gaps. Documenting these patterns is an important step towards eliminating excellence gaps and removing the ‘racial glass ceiling’. Data from Michigan, USA, systematically document that comparatively fewer students from racial minorities excel in maths and reading, and that this gap grows as students progress through school.},
  archive      = {J_NATURE},
  author       = {Karna, Uditi and Lee, Min Sok and List, John A. and Simon, Andrew and Uchida, Haruka},
  doi          = {10.1038/s41586-024-08536-0},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {976-984},
  shortjournal = {Nature},
  title        = {Measuring racial educational disparities over time amongst top achievers},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genomics yields biological and phenotypic insights into
bipolar disorder. <em>NATURE</em>, <em>639</em>(8056), 968–975. (<a
href="https://doi.org/10.1038/s41586-024-08468-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bipolar disorder is a leading contributor to the global burden of disease1. Despite high heritability (60–80%), the majority of the underlying genetic determinants remain unknown2. We analysed data from participants of European, East Asian, African American and Latino ancestries (n = 158,036 cases with bipolar disorder, 2.8 million controls), combining clinical, community and self-reported samples. We identified 298 genome-wide significant loci in the multi-ancestry meta-analysis, a fourfold increase over previous findings3, and identified an ancestry-specific association in the East Asian cohort. Integrating results from fine-mapping and other variant-to-gene mapping approaches identified 36 credible genes in the aetiology of bipolar disorder. Genes prioritized through fine-mapping were enriched for ultra-rare damaging missense and protein-truncating variations in cases with bipolar disorder4, highlighting convergence of common and rare variant signals. We report differences in the genetic architecture of bipolar disorder&amp;nbsp;depending on the source of patient ascertainment and on bipolar disorder subtype (type I or type II). Several analyses implicate specific cell types in the pathophysiology of bipolar disorder, including GABAergic interneurons and medium spiny neurons. Together, these analyses provide additional insights into the genetic architecture and biological underpinnings of bipolar disorder. Using multi-ancestry genome-wide association study and fine-mapping, 298 loci and 36 credible genes are identified in the aetiology of bipolar disorder.},
  archive      = {J_NATURE},
  author       = {O’Connell, Kevin S. and Koromina, Maria and van der Veen, Tracey and Boltz, Toni and David, Friederike S. and Yang, Jessica Mei Kay and Lin, Keng-Han and Wang, Xin and Coleman, Jonathan R. I. and Mitchell, Brittany L. and McGrouther, Caroline C. and Rangan, Aaditya V. and Lind, Penelope A. and Koch, Elise and Harder, Arvid and Parker, Nadine and Bendl, Jaroslav and Adorjan, Kristina and Agerbo, Esben and Albani, Diego and Alemany, Silvia and Alliey-Rodriguez, Ney and Als, Thomas D. and Andlauer, Till F. M. and Antoniou, Anastasia and Ask, Helga and Bass, Nicholas and Bauer, Michael and Beins, Eva C. and Bigdeli, Tim B. and Pedersen, Carsten Bøcker and Boks, Marco P. and Børte, Sigrid and Bosch, Rosa and Brum, Murielle and Brumpton, Ben M. and Brunkhorst-Kanaan, Nathalie and Budde, Monika and Bybjerg-Grauholm, Jonas and Byerley, William and Cabana-Domínguez, Judit and Cairns, Murray J. and Carpiniello, Bernardo and Casas, Miquel and Cervantes, Pablo and Chatzinakos, Chris and Chen, Hsi-Chung and Clarence, Tereza and Clarke, Toni-Kim and Claus, Isabelle and Coombes, Brandon and Corfield, Elizabeth C. and Cruceanu, Cristiana and Cuellar-Barboza, Alfredo and Czerski, Piotr M. and Dafnas, Konstantinos and Dale, Anders M. and Dalkner, Nina and Degenhardt, Franziska and DePaulo, J. Raymond and Djurovic, Srdjan and Drange, Ole Kristian and Escott-Price, Valentina and Fanous, Ayman H. and Fellendorf, Frederike T. and Ferrier, I. Nicol and Forty, Liz and Frank, Josef and Frei, Oleksandr and Freimer, Nelson B. and Fullard, John F. and Garnham, Julie and Gizer, Ian R. and Gordon, Scott D. and Gordon-Smith, Katherine and Greenwood, Tiffany A. and Grove, Jakob and Guzman-Parra, José and Ha, Tae Hyon and Hahn, Tim and Haraldsson, Magnus and Hautzinger, Martin and Havdahl, Alexandra and Heilbronner, Urs and Hellgren, Dennis and Herms, Stefan and Hickie, Ian B. and Hoffmann, Per and Holmans, Peter A. and Huang, Ming-Chyi and Ikeda, Masashi and Jamain, Stéphane and Johnson, Jessica S. and Jonsson, Lina and Kalman, Janos L. and Kamatani, Yoichiro and Kennedy, James L. and Kim, Euitae and Kim, Jaeyoung and Kittel-Schneider, Sarah and Knowles, James A. and Kogevinas, Manolis and Kranz, Thorsten M. and Krebs, Kristi and Kushner, Steven A. and Lavebratt, Catharina and Lawrence, Jacob and Leber, Markus and Lee, Heon-Jeong and Liao, Calwing and Lucae, Susanne and Lundberg, Martin and MacIntyre, Donald J. and Maier, Wolfgang and Maihofer, Adam X. and Malaspina, Dolores and Manchia, Mirko and Maratou, Eirini and Martinsson, Lina and Mattheisen, Manuel and McGregor, Nathaniel W. and McInnis, Melvin G. and McKay, James D. and Medeiros, Helena and Meyer-Lindenberg, Andreas and Millischer, Vincent and Morris, Derek W. and Moutsatsou, Paraskevi and Mühleisen, Thomas W. and O’Donovan, Claire and Olsen, Catherine M. and Panagiotaropoulou, Georgia and Papiol, Sergi and Pardiñas, Antonio F. and Park, Hye Youn and Perry, Amy and Pfennig, Andrea and Pisanu, Claudia and Potash, James B. and Quested, Digby and Rapaport, Mark H. and Regeer, Eline J. and Rice, John P. and Rivera, Margarita and Schulte, Eva C. and Senner, Fanny and Shadrin, Alexey and Shilling, Paul D. and Sigurdsson, Engilbert and Sindermann, Lisa and Sirignano, Lea and Siskind, Dan and Slaney, Claire and Sloofman, Laura G. and Smeland, Olav B. and Smith, Daniel J. and Sobell, Janet L. and Soler Artigas, Maria and Stein, Dan J. and Stein, Frederike and Su, Mei-Hsin and Sung, Heejong and Świątkowska, Beata and Terao, Chikashi and Tesfaye, Markos and Tesli, Martin and Thorgeirsson, Thorgeir E. and Thorp, Jackson G. and Toma, Claudio and Tondo, Leonardo and Tooney, Paul A. and Tsai, Shih-Jen and Tsermpini, Evangelia Eirini and Vawter, Marquis P. and Vedder, Helmut and Vreeker, Annabel and Walters, James T. R. and Winsvold, Bendik S. and Witt, Stephanie H. and Won, Hong-Hee and Ye, Robert and Young, Allan H. and Zandi, Peter P. and Zillich, Lea and Adolfsson, Rolf and Alda, Martin and Alfredsson, Lars and Backlund, Lena and Baune, Bernhard T. and Bellivier, Frank and Bengesser, Susanne and Berrettini, Wade H. and Biernacka, Joanna M. and Boehnke, Michael and Børglum, Anders D. and Breen, Gerome and Carr, Vaughan J. and Catts, Stanley and Cichon, Sven and Corvin, Aiden and Craddock, Nicholas and Dannlowski, Udo and Dikeos, Dimitris and Etain, Bruno and Ferentinos, Panagiotis and Frye, Mark and Fullerton, Janice M. and Gawlik, Micha and Gershon, Elliot S. and Goes, Fernando S. and Green, Melissa J. and Grigoroiu-Serbanescu, Maria and Hauser, Joanna and Henskens, Frans A. and Hjerling-Leffler, Jens and Hougaard, David M. and Hveem, Kristian and Iwata, Nakao and Jones, Ian and Jones, Lisa A. and Kahn, René S. and Kelsoe, John R. and Kircher, Tilo and Kirov, George and Kuo, Po-Hsiu and Landén, Mikael and Leboyer, Marion and Li, Qingqin S. and Lissowska, Jolanta and Lochner, Christine and Loughland, Carmel and Luykx, Jurjen J. and Martin, Nicholas G. and Mathews, Carol A. and Mayoral, Fermin and McElroy, Susan L. and McIntosh, Andrew M. and McMahon, Francis J. and Medland, Sarah E. and Melle, Ingrid and Milani, Lili and Mitchell, Philip B. and Morken, Gunnar and Mors, Ole and Mortensen, Preben Bo and Müller-Myhsok, Bertram and Myers, Richard M. and Myung, Woojae and Neale, Benjamin M. and Nievergelt, Caroline M. and Nordentoft, Merete and Nöthen, Markus M. and Nurnberger, John I. and O’Donovan, Michael C. and Oedegaard, Ketil J. and Olsson, Tomas and Owen, Michael J. and Paciga, Sara A. and Pantelis, Christos and Pato, Carlos N. and Pato, Michele T. and Patrinos, George P. and Pawlak, Joanna M. and Ramos-Quiroga, Josep Antoni and Reif, Andreas and Reininghaus, Eva Z. and Ribasés, Marta and Rietschel, Marcella and Ripke, Stephan and Rouleau, Guy A. and Roussos, Panos and Saito, Takeo and Schall, Ulrich and Schalling, Martin and Schofield, Peter R. and Schulze, Thomas G. and Scott, Laura J. and Scott, Rodney J. and Serretti, Alessandro and Smoller, Jordan W. and Squassina, Alessio and Stahl, Eli A. and Stefansson, Hreinn and Stefansson, Kari and Stordal, Eystein and Streit, Fabian and Sullivan, Patrick F. and Turecki, Gustavo and Vaaler, Arne E. and Vieta, Eduard and Vincent, John B. and Waldman, Irwin D. and Weickert, Cynthia S. and Weickert, Thomas W. and Werge, Thomas and Whiteman, David C. and Zwart, John-Anker and Edenberg, Howard J. and McQuillin, Andrew and Forstner, Andreas J. and Mullins, Niamh and Di Florio, Arianna and Ophoff, Roel A. and Andreassen, Ole A.},
  doi          = {10.1038/s41586-024-08468-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {968-975},
  shortjournal = {Nature},
  title        = {Genomics yields biological and phenotypic insights into bipolar disorder},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolutionary lability of a key innovation spurs rapid
diversification. <em>NATURE</em>, <em>639</em>(8056), 962–967. (<a
href="https://doi.org/10.1038/s41586-025-08612-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rates of lineage diversification vary considerably across the tree of life, often as&amp;nbsp;a result of evolutionary innovations1–5. Although the ability to produce new traits can vary between clades and may drive ecological transitions6–9, the impact of differences in the pace at which innovations evolve at macroevolutionary scales has been overlooked. Complex teeth are one innovation that contributed to the evolutionary success of major vertebrate lineages10–12. Here we show that evolutionary lability of tooth complexity, but not complexity itself, spurs rapid diversification across ray-finned fishes. Speciation rates are five times higher when transitions between simple and complex teeth occur rapidly. We find that African cichlids are unique among all fishes; they are dominated by lineages that transition between simple and complex teeth at unparalleled rates. This innovation interacted with the ecological versatility of complex teeth to spur rapid adaptive radiations in lakes Malawi, Victoria and Barombi Mbo. The marked effect on diversification stems from the tight association of tooth complexity with microhabitat and diet. Our results show that phylogenetic variation in how innovations evolve can have a stronger effect on patterns of diversification than the innovation itself. Investigating the impact of innovations from this new perspective will probably implicate more traits in causing heterogeneous diversification rates across the tree of life. A study shows that the rapid diversification of cichlids in African lakes is driven by their ability to evolve between having simple or complex teeth.},
  archive      = {J_NATURE},
  author       = {Peoples, Nick and Burns, Michael D. and Mihalitsis, Michalis and Wainwright, Peter C.},
  doi          = {10.1038/s41586-025-08612-z},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {962-967},
  shortjournal = {Nature},
  title        = {Evolutionary lability of a key innovation spurs rapid diversification},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vulnerability of amphibians to global warming.
<em>NATURE</em>, <em>639</em>(8056), 954–961. (<a
href="https://doi.org/10.1038/s41586-025-08665-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amphibians are the most threatened vertebrates, yet their resilience to rising temperatures remains poorly understood1,2. This is primarily because knowledge of thermal tolerance is taxonomically and geographically biased3, compromising global climate vulnerability assessments. Here we&amp;nbsp;used a phylogenetically informed data-imputation approach to predict the heat tolerance of 60% of amphibian species and assessed their vulnerability to daily temperature variations in thermal refugia. We found that 104 out of 5,203 species (2%) are currently exposed to overheating events in shaded terrestrial conditions. Despite accounting for heat-tolerance plasticity, a 4 °C global temperature increase would create a step change in impact severity, pushing 7.5% of species beyond their physiological limits. In the Southern Hemisphere, tropical species encounter disproportionally more overheating events, while non-tropical species are more susceptible in the Northern Hemisphere. These findings challenge evidence for a general latitudinal gradient in overheating risk4–6 and underscore the importance of considering climatic variability in vulnerability assessments. We provide conservative estimates assuming access to cool shaded microenvironments. Thus, the impacts of global warming will probably exceed our projections. Our microclimate-explicit analyses demonstrate that vegetation and water bodies are critical in buffering amphibians during heat waves. Immediate action is needed to preserve and manage these microhabitat features. A 4 °C global temperature increase&amp;nbsp;would push 7.5% of amphibian species beyond their physiological limits.},
  archive      = {J_NATURE},
  author       = {Pottier, Patrice and Kearney, Michael R. and Wu, Nicholas C. and Gunderson, Alex R. and Rej, Julie E. and Rivera-Villanueva, A. Nayelli and Pollo, Pietro and Burke, Samantha and Drobniak, Szymon M. and Nakagawa, Shinichi},
  doi          = {10.1038/s41586-025-08665-0},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {954-961},
  shortjournal = {Nature},
  title        = {Vulnerability of amphibians to global warming},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sliding and healing of frictional interfaces that appear
stationary. <em>NATURE</em>, <em>639</em>(8056), 947–953. (<a
href="https://doi.org/10.1038/s41586-025-08673-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Frictional interfaces are found in systems ranging from biological joints to earthquake faults. When and how these interfaces slide is a fundamental problem in geosciences and engineering1–20. It is believed that there exists a threshold shear force, called static friction, below which the interface is stationary4,10, despite many studies suggesting that this concept is outdated1,21–28. By contrast, rate-and-state friction formulations1,26,27 predict that interfaces are always sliding29, but this feature is often considered an artefact that calls for modifications30. Here we show that nominally stationary interfaces subjected to constant shear and normal loads, with a driving force that is notably below the classically defined static friction for which creep is known to occur9,27–29, are sliding, but with diminishingly small rates down to 10−12 m s−1. Our precise measurements directly at the interface are enabled by digital image correlation18,31,32. This behaviour contradicts classical models of friction but confirms the prediction of rate-and-state friction1,26,27. The diminishing slip rates of nominally stationary interfaces reflect interface healing, which would manifest itself in higher peak friction in subsequent slip events15,27,33, such as earthquakes and landslides, substantially modifying their nucleation and propagation and hence their hazard3,12,13,34. Digital image correlation measurements show that nominally stationary interfaces subjected to constant shear and normal loads are sliding at extremely small rates, confirming the predictions of rate-and-state friction formulations.},
  archive      = {J_NATURE},
  author       = {Sirorattanakul, Krittanon and Larochelle, Stacy and Rubino, Vito and Lapusta, Nadia and Rosakis, Ares J.},
  doi          = {10.1038/s41586-025-08673-0},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {947-953},
  shortjournal = {Nature},
  title        = {Sliding and healing of frictional interfaces that appear stationary},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Record sea surface temperature jump in 2023–2024 unlikely
but not unexpected. <em>NATURE</em>, <em>639</em>(8056), 942–946. (<a
href="https://doi.org/10.1038/s41586-025-08674-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global ocean surface temperatures were at record levels for more than a year from April 2023&amp;nbsp;onwards, exceeding the previous record in 2015–2016 by 0.25 °C on average between April 2023 and March 20241. The nearly global extent and unprecedented intensity of this event prompted questions about how exceptional it was and whether climate models can represent such record-shattering jumps in surface ocean temperatures2. Here we construct observation-based synthetic time series to show that a jump in global sea surface temperatures that breaks the previous record by at least 0.25 °C is a 1-in-512-year event under the current long-term warming trend (1-in-205-year to 1-in-1,185-year event; 95% confidence interval). Without a global warming trend, such an event would have been practically impossible. Using 270 simulations from a wide range of fully coupled climate models, we show that these models successfully simulate such record-shattering jumps in global ocean surface temperatures, underpinning the models’ usefulness in understanding the characteristics, drivers and consequences of such events. These model simulations suggest that the record-shattering jump in surface ocean temperatures in 2023–2024 was an extreme event after which surface ocean temperatures are expected to revert to the expected long-term warming trend. Observations and climate models suggest that the global sea surface temperature jump in 2023–2024 was not unexpected and would have been&amp;nbsp;nearly impossible without anthropogenic warming.},
  archive      = {J_NATURE},
  author       = {Terhaar, Jens and Burger, Friedrich A. and Vogt, Linus and Frölicher, Thomas L. and Stocker, Thomas F.},
  doi          = {10.1038/s41586-025-08674-z},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {942-946},
  shortjournal = {Nature},
  title        = {Record sea surface temperature jump in 2023–2024 unlikely but not unexpected},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Adaptive locomotion of active solids. <em>NATURE</em>,
<em>639</em>(8056), 935–941. (<a
href="https://doi.org/10.1038/s41586-025-08646-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Active systems composed of energy-generating microscopic constituents are a promising platform to create autonomous functional materials1–16 that can, for example, locomote through complex and unpredictable environments. Yet coaxing these energy sources into useful mechanical work has proved challenging. Here we engineer active solids based on centimetre-scale building blocks that perform adaptive locomotion. These prototypes exhibit a non-variational form of elasticity characterized by odd moduli8,12,17, whose magnitude we predict from microscopics using coarse-grained theories and which we validate experimentally. When interacting with an external environment, these active solids spontaneously undergo limit cycles of shape changes, which naturally lead to locomotion such as rolling and crawling. The robustness of the locomotion is rooted in an emergent feedback loop between the active solid and the environment, which is mediated by elastic deformations and stresses. As a result, our active solids are able to accelerate, adjust their gaits and locomote through a variety of terrains with a similar performance to more complex control strategies implemented by neural networks. Our work establishes active solids as a bridge between materials and robots and suggests decentralized strategies to control the nonlinear dynamics of biological systems8,18–22, soft materials5,6,9,11,12,23–25 and driven nanomechanical devices7,26–30. The development of active solids based on centimetre-scale building blocks incorporating odd elasticity shows that they can spontaneously undergo limit cycles of shape changes, leading to adaptive locomotion such as rolling and crawling.},
  archive      = {J_NATURE},
  author       = {Veenstra, Jonas and Scheibner, Colin and Brandenbourger, Martin and Binysh, Jack and Souslov, Anton and Vitelli, Vincenzo and Coulais, Corentin},
  doi          = {10.1038/s41586-025-08646-3},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {935-941},
  shortjournal = {Nature},
  title        = {Adaptive locomotion of active solids},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An ultra-broadband photonic-chip-based parametric amplifier.
<em>NATURE</em>, <em>639</em>(8056), 928–934. (<a
href="https://doi.org/10.1038/s41586-025-08666-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical amplification, crucial for modern communication, primarily relies on erbium-doped fibre amplifiers (EDFAs)1,2. Yet, EDFAs only cover a portion of the low-loss spectrum of optical fibres. This has motivated the development of amplifiers operating beyond the erbium gain window. Pioneering work on optical parametric amplifiers (OPAs)3,4 using intrinsic third-order optical nonlinearity has led to demonstrations of increased channel capacity. OPAs offer high gain, can reach the 3-dB quantum limit for phase-preserving amplifiers and exhibit unidirectional operation. However, power requirements for highly nonlinear fibres3,5–8 or bulk waveguides9,10 have impeded their adoption. By contrast, OPAs based on integrated photonic circuits offer the advantages of substantially increased mode confinement and optical nonlinearity but have been limited in bandwidth11,12. We overcome this challenge by using low-loss gallium phosphide-on-silicon dioxide13–15 photonic integrated circuits (PICs) and attain up to 35 dB of parametric gain with waveguides only a few centimetres long in a compact footprint of 0.25 square millimetres. Fibre-to-fibre net gain exceeding 10 dB across an ultra-broad bandwidth of approximately 140 nm (that is, 17 THz) is achieved, with a threefold increase in the gain window compared with C-band EDFAs. We further demonstrate a high dynamic range for input signals, spanning six orders of magnitude, while maintaining a low noise figure. We exploit these performance characteristics to amplify coherent communication signals. This marks, to our knowledge, the first ultra-broadband, high-gain, continuous-wave amplification in a photonic chip, opening up new capabilities for next-generation integrated photonics. An optical parametric amplifier based on integrated photonic circuits fabricated using low-loss gallium phosphide-on-silicon dioxide demonstrates improved bandwidth and gain performance over state-of-the-art erbium-doped fibre amplifiers while maintaining a low noise figure.},
  archive      = {J_NATURE},
  author       = {Kuznetsov, Nikolai and Nardi, Alberto and Riemensberger, Johann and Davydova, Alisa and Churaev, Mikhail and Seidler, Paul and Kippenberg, Tobias J.},
  doi          = {10.1038/s41586-025-08666-z},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {928-934},
  shortjournal = {Nature},
  title        = {An ultra-broadband photonic-chip-based parametric amplifier},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Down-converted photon pairs in a high-q silicon nitride
microresonator. <em>NATURE</em>, <em>639</em>(8056), 922–927. (<a
href="https://doi.org/10.1038/s41586-025-08662-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Entangled photon pairs from spontaneous parametric down-conversion (SPDC)1 are central to many quantum applications2–6. SPDC is typically performed in non-centrosymmetric systems7 with an inherent second-order nonlinearity (χ(2))8–10. We demonstrate strong narrowband SPDC with an on-chip rate of 0.8 million pairs per second in Si3N4. Si3N4 is the pre-eminent material for photonic integration and also exhibits the lowest waveguide loss (which is essential for integrated quantum circuits). However, being amorphous, silicon nitride lacks an intrinsic χ(2), which limits its role in photonic quantum devices. We enabled SPDC in Si3N4 by combining strong light-field enhancement inside a high optical Q-factor microcavity with an optically induced space-charge field. We present narrowband photon pairs with a high spectral brightness. The quantum nature of the down-converted photon pairs is verified through coincidence measurements. This light source, based on Si3N4 integrated photonics technology, unlocks new avenues for quantum systems on a chip. Spontaneous parametric down-conversion was used to generate narrowband photon pairs with a high spectral brightness in a high-Q silicon nitride microresonator.},
  archive      = {J_NATURE},
  author       = {Li, Bohan and Yuan, Zhiquan and Williams, James and Jin, Warren and Beckert, Adrian and Xie, Tian and Guo, Joel and Feshali, Avi and Paniccia, Mario and Faraon, Andrei and Bowers, John and Marandi, Alireza and Vahala, Kerry},
  doi          = {10.1038/s41586-025-08662-3},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {922-927},
  shortjournal = {Nature},
  title        = {Down-converted photon pairs in a high-Q silicon nitride microresonator},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hyperbolic phonon-polariton electroluminescence in 2D
heterostructures. <em>NATURE</em>, <em>639</em>(8056), 915–921. (<a
href="https://doi.org/10.1038/s41586-025-08686-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phonon polaritons are quasiparticles resulting from the coherent coupling of photons with optical phonons in polar dielectrics1. Owing to their exceptional ability to confine electric fields to deep-subwavelength scales with low loss, they are uniquely poised to enable a suite of applications beyond the reach of conventional photonics, such as subdiffraction imaging2 and near-field energy transfer3–5. The conventional approach to exciting phonon polaritons through optical methods, however, involves costly light sources along with near-field schemes6,7, and generally leads to low excitation efficiency owing to substantial momentum mismatch between phonon polaritons and free-space photons. Here we demonstrate that under proper conditions, phonon polaritons can be excited all-electrically by drifting charge carriers. Specifically, in hexagonal boron nitride (hBN)/graphene heterostructures, by electrically driving charge carriers in ultrahigh-mobility graphene out of equilibrium, we observe bright electroluminescence of hBN’s hyperbolic phonon polaritons (HPhPs) at mid-infrared frequencies, which shows a temperature and carrier density dependence distinct from black-body thermal emission. Moreover, the carrier density dependence of the HPhP electroluminescence spectra reveals that HPhP electroluminescence can arise from both interband transition and intraband Cherenkov radiation8 of charge carriers in graphene. The HPhP electroluminescence offers avenues for realizing electrically pumped mid-infrared and terahertz phonon-polariton light sources. All-electrical excitation of the hyperbolic phonon polaritons in hexagonal boron nitride by drifting charge carriers in nearby graphene results in electroluminescence at mid-infrared frequencies.},
  archive      = {J_NATURE},
  author       = {Guo, Qiushi and Esin, Iliya and Li, Cheng and Chen, Chen and Han, Guanyu and Liu, Song and Edgar, James H. and Zhou, Selina and Demler, Eugene and Refael, Gil and Xia, Fengnian},
  doi          = {10.1038/s41586-025-08686-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {915-921},
  shortjournal = {Nature},
  title        = {Hyperbolic phonon-polariton electroluminescence in 2D heterostructures},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electroluminescence and energy transfer mediated by
hyperbolic polaritons. <em>NATURE</em>, <em>639</em>(8056), 909–914. (<a
href="https://doi.org/10.1038/s41586-025-08627-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Under high electrical current, some materials can emit electromagnetic radiation beyond incandescence. This phenomenon, referred to as electroluminescence, leads to the efficient emission of visible photons and is the basis of domestic lighting devices (for example, light-emitting diodes)1,2. In principle, electroluminescence can lead to mid-infrared emission of confined light–matter excitations called phonon polaritons3,4, resulting from the coupling of photons with crystal lattice vibrations (optical phonons). In particular, phonon polaritons arising in the van der Waals crystal hexagonal boron nitride (hBN) present hyperbolic dispersion, which enhances light–matter coupling5,6. For this reason, electroluminescence of hyperbolic phonon polaritons (HPhPs) has been proposed as an explanation for the peculiar radiative energy transfer within hBN-encapsulated graphene transistors7,8. However, as HPhPs are locally confined, they are inaccessible in the far field, and as such, any hint of electroluminescence has been based on indirect electronic signatures and has yet to be confirmed by direct observation. Here we demonstrate far-field mid-infrared (wavelength approximately 6.5 μm) electroluminescence of HPhPs excited by strongly biased high-mobility graphene within a van der Waals heterostructure, and we quantify the associated radiative energy transfer through the material. The presence of HPhPs is revealed by far-field mid-infrared spectroscopy owing to their elastic scattering at discontinuities in the heterostructure. The resulting radiative flux is quantified by mid-infrared pyrometry of the substrate receiving the energy. This radiative energy transfer is also shown to be reduced in hBN with nanoscale inhomogeneities, demonstrating the central role of the electromagnetic environment in this process. Far-field mid-infrared spectroscopy reveals both the electroluminescence of hyperbolic phonon polaritons of hexagonal boron nitride excited by strongly biased graphene, and the associated radiative energy transfer through the material.},
  archive      = {J_NATURE},
  author       = {Abou-Hamdan, Loubnan and Schmitt, Aurélien and Bretel, Rémi and Rossetti, Sylvio and Tharrault, Marin and Mele, David and Pierret, Aurélie and Rosticher, Michael and Taniguchi, Takashi and Watanabe, Kenji and Maestre, Camille and Journet, Catherine and Toury, Bérangère and Garnier, Vincent and Steyer, Philippe and Edgar, James H. and Janzen, Eli and Berroir, Jean-Marc and Fève, Gwendal and Ménard, Gerbold and Plaçais, Bernard and Voisin, Christophe and Hugonin, Jean-Paul and Bailly, Elise and Vest, Benjamin and Greffet, Jean-Jacques and Bouchon, Patrick and De Wilde, Yannick and Baudin, Emmanuel},
  doi          = {10.1038/s41586-025-08627-6},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {909-914},
  shortjournal = {Nature},
  title        = {Electroluminescence and energy transfer mediated by hyperbolic polaritons},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vertical structure of an exoplanet’s atmospheric jet stream.
<em>NATURE</em>, <em>639</em>(8056), 902–908. (<a
href="https://doi.org/10.1038/s41586-025-08664-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultra-hot Jupiters, an extreme class of planets not found in our Solar System, provide a unique window into atmospheric processes. The extreme temperature contrasts between their day and night sides pose a fundamental climate puzzle: how is energy distributed? To address this, we must observe the three-dimensional structure of these atmospheres, particularly their vertical circulation patterns that can serve as a testbed for advanced global circulation models, for example, in ref. 1. Here we show a notable shift in atmospheric circulation in an ultra-hot Jupiter: a unilateral flow from the hot star-facing side to the cooler space-facing side of the planet sits below an equatorial super-rotational jet stream. By resolving the vertical structure of atmospheric dynamics, we move beyond integrated global snapshots of the atmosphere, enabling more accurate identification of flow patterns and allowing for a more nuanced comparison to models. Global circulation models based on first principles struggle to replicate the observed circulation pattern2 underscoring a critical gap between theoretical understanding of atmospheric flows and observational evidence. This work serves as a testbed to develop more comprehensive models applicable beyond our Solar System as we prepare for the next generation of giant telescopes. Ultra-hot Jupiters provide a unique window into atmospheric processes and this in-depth study enables integrated global snapshots of the atmosphere and more accurate identification of flow patterns, thus allowing for better comparison to models.},
  archive      = {J_NATURE},
  author       = {Seidel, Julia V. and Prinoth, Bibiana and Pino, Lorenzo and dos Santos, Leonardo A. and Chakraborty, Hritam and Parmentier, Vivien and Sedaghati, Elyar and Wardenier, Joost P. and Farret Jentink, Casper and Zapatero Osorio, Maria Rosa and Allart, Romain and Ehrenreich, David and Lendl, Monika and Roccetti, Giulia and Damasceno, Yuri and Bourrier, Vincent and Lillo-Box, Jorge and Hoeijmakers, H. Jens and Pallé, Enric and Santos, Nuno and Suárez Mascareño, Alejandro and Sousa, Sergio G. and Tabernero, Hugo M. and Pepe, Francesco A.},
  doi          = {10.1038/s41586-025-08664-1},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {902-908},
  shortjournal = {Nature},
  title        = {Vertical structure of an exoplanet’s atmospheric jet stream},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Witnessing the onset of reionization through lyman-α
emission at redshift 13. <em>NATURE</em>, <em>639</em>(8056), 897–901.
(<a href="https://doi.org/10.1038/s41586-025-08779-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cosmic reionization began when ultraviolet (UV) radiation produced in the first galaxies began illuminating the cold, neutral gas that filled the primordial Universe1,2. Recent James Webb Space Telescope (JWST) observations have shown that surprisingly UV-bright galaxies were in place beyond redshift z = 14, when the Universe was less than 300 Myr old3–5. Smooth turnovers of their UV continua have been interpreted as damping-wing absorption of Lyman-α (Ly-α), the principal hydrogen transition6–9. However, spectral signatures encoding crucial properties of these sources, such as their emergent radiation field, largely remain elusive. Here we report spectroscopy from the JWST Advanced Deep Extragalactic Survey (JADES10) of a galaxy at redshift z = 13.0 that reveals a singular, bright emission line unambiguously identified as Ly-α, as well as a smooth turnover. We observe an equivalent width of EWLy-α &amp;gt; 40 Å (rest frame), previously only seen at z &amp;lt; 9&amp;nbsp;where the intervening intergalactic medium becomes increasingly ionized11. Together with an extremely blue UV continuum, the unexpected Ly-α emission indicates that the galaxy is a prolific producer and leaker of ionizing photons. This suggests that massive, hot stars or an active galactic nucleus have created an early reionized region to prevent complete extinction of Ly-α, thus shedding new light on the nature of the earliest galaxies and the onset of reionization only 330 Myr after the Big Bang. Spectroscopy from the JWST Advanced Deep Extragalactic Survey of a galaxy at redshift 13 shows a singular, bright emission line identified as Lyman-α, suggesting the onset of reionization only 330 Myr after the Big Bang.},
  archive      = {J_NATURE},
  author       = {Witstok, Joris and Jakobsen, Peter and Maiolino, Roberto and Helton, Jakob M. and Johnson, Benjamin D. and Robertson, Brant E. and Tacchella, Sandro and Cameron, Alex J. and Smit, Renske and Bunker, Andrew J. and Saxena, Aayush and Sun, Fengwu and Alberts, Stacey and Arribas, Santiago and Baker, William M. and Bhatawdekar, Rachana and Boyett, Kristan and Cargile, Phillip A. and Carniani, Stefano and Charlot, Stéphane and Chevallard, Jacopo and Curti, Mirko and Curtis-Lake, Emma and D’Eugenio, Francesco and Eisenstein, Daniel J. and Hainline, Kevin N. and Jones, Gareth C. and Kumari, Nimisha and Maseda, Michael V. and Pérez-González, Pablo G. and Rinaldi, Pierluigi and Scholtz, Jan and Übler, Hannah and Williams, Christina C. and Willmer, Christopher N. A. and Willott, Chris and Zhu, Yongda},
  doi          = {10.1038/s41586-025-08779-5},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {897-901},
  shortjournal = {Nature},
  title        = {Witnessing the onset of reionization through lyman-α emission at redshift 13},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal generative AI for medical image interpretation.
<em>NATURE</em>, <em>639</em>(8056), 888–896. (<a
href="https://doi.org/10.1038/s41586-025-08675-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurately interpreting medical images and generating insightful narrative reports is indispensable for patient care but places heavy burdens on clinical experts. Advances in artificial intelligence (AI), especially in an area that we refer to as multimodal generative medical image interpretation (GenMI), create opportunities to automate parts of this complex process. In this Perspective, we synthesize progress and challenges in developing AI systems for generation of medical reports from images. We focus extensively on radiology as a domain with enormous reporting needs and research efforts. In addition to analysing the strengths and applications of new models for medical report generation, we advocate for a novel paradigm to deploy GenMI in a manner that empowers clinicians and their patients. Initial research suggests that GenMI could one day match human expert performance in generating reports across disciplines, such as radiology, pathology and dermatology. However, formidable obstacles remain in validating model accuracy, ensuring transparency and eliciting nuanced impressions. If carefully implemented, GenMI could meaningfully assist clinicians in improving quality of care, enhancing medical education, reducing workloads, expanding specialty access and providing real-time expertise. Overall, we highlight opportunities alongside key challenges for developing multimodal generative AI that complements human experts for reliable medical report writing. This Perspective describes how recent advances in artificial intelligence could be used to automate medical image interpretation&amp;nbsp;to complement human expertise and empower physicians and patients.},
  archive      = {J_NATURE},
  author       = {Rao, Vishwanatha M. and Hla, Michael and Moor, Michael and Adithan, Subathra and Kwak, Stephen and Topol, Eric J. and Rajpurkar, Pranav},
  doi          = {10.1038/s41586-025-08675-y},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {888-896},
  shortjournal = {Nature},
  title        = {Multimodal generative AI for medical image interpretation},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The other climate crisis. <em>NATURE</em>,
<em>639</em>(8056), 877–887. (<a
href="https://doi.org/10.1038/s41586-025-08680-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As Earth warms, regional climate signals are accumulating. Some signals, for example, land warming more than the ocean and the Arctic warming the most, were expected and successfully predicted. Underlying this success was the application of physical laws under the assumption that large and small spatial scales are well separated. This established what we call the standard approach, climate science’s dominant paradigm. With additional warming, however, discrepancies between real-world signals and expectations based on this standard approach are piling up, especially at regional scales. At the same time, disruptive computational approaches are advancing new&amp;nbsp;paradigms. Philosophers of science characterize situations where accumulating discrepancies (anomalies) and disruptions lead to a loss of confidence in the dominant paradigm as a ‘crisis’. Here we articulate what we consider to be the dominant paradigm, or standard approach, and the discrepancies and disruptions that have emerged in recent years. The policy implications of a purported crisis are discussed, as well as paths forward, crisis or no crisis. These paths include using signals to test assumptions and processes driving a warming Earth&amp;nbsp;for the first time, developing testable hypotheses, and revitalizing conceptual thinking by filling gaps across climate-system components and spatial scales. The standard approach of&amp;nbsp;climate science is showing signs of a crisis owing to the emergence of discrepancies and disruptions in recent years; this Perspective discusses the policy implications and the paths forward.},
  archive      = {J_NATURE},
  author       = {Shaw, Tiffany A. and Stevens, Bjorn},
  doi          = {10.1038/s41586-025-08680-1},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {877-887},
  shortjournal = {Nature},
  title        = {The other climate crisis},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fossilized dinosaur cells that defied the ravages of time —
20 years since a key discovery. <em>NATURE</em>, <em>639</em>(8056),
875–876. (<a href="https://doi.org/10.1038/d41586-025-00733-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The 2005 finding of cells and blood vessels in dinosaur bone launched a systematic search for fossil remnants of biomolecules, creating innovations in methods and applications. The 2005 finding of cells and blood vessels in dinosaur bone launched a systematic search for fossil remnants of biomolecules, creating innovations in methods and applications.},
  archive      = {J_NATURE},
  author       = {Wiemann, Jasmina},
  doi          = {10.1038/d41586-025-00733-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {875-876},
  shortjournal = {Nature},
  title        = {Fossilized dinosaur cells that defied the ravages of time — 20 years since a key discovery},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Why is my cello howling? <em>NATURE</em>,
<em>639</em>(8056), 875. (<a
href="https://doi.org/10.1038/d41586-025-00914-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A musician’s query as to the origin of a warped ‘wolf’ tone, and a review of a book by physicist Marie Curie, in our weekly dip into Nature’s archive. Snippets from Nature’s past.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-00914-6},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {875},
  shortjournal = {Nature},
  title        = {Why is my cello howling?},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nursing-home residents’ skin is a source of transmitted
harmful and drug-resistant microbes. <em>NATURE</em>,
<em>639</em>(8056), 873–875. (<a
href="https://doi.org/10.1038/d41586-025-00306-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic sequencing has revealed that individual strains of life-threatening, antibiotic-resistant bacteria and Candida auris fungi reside on human skin and can be commonly transmitted between residents of nursing homes. How harmful antibiotic-resistant bacteria and Candida auris fungi spread.},
  archive      = {J_NATURE},
  author       = {O’Meara, Teresa R.},
  doi          = {10.1038/d41586-025-00306-w},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {873-875},
  shortjournal = {Nature},
  title        = {Nursing-home residents’ skin is a source of transmitted harmful and drug-resistant microbes},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Protein waste turned into antibiotics as a defence strategy
of human cells. <em>NATURE</em>, <em>639</em>(8056), 872–873. (<a
href="https://doi.org/10.1038/d41586-025-00514-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As well as degrading proteins, the proteosome surprisingly also functions in human defence against bacterial infection. By changing how proteins are cleaved, this system generates antibiotic peptides on demand. Proteasome cleavage changes during infection to generate antibiotics.},
  archive      = {J_NATURE},
  author       = {Clausen, Tim},
  doi          = {10.1038/d41586-025-00514-4},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {872-873},
  shortjournal = {Nature},
  title        = {Protein waste turned into antibiotics as a defence strategy of human cells},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A lighthouse galaxy shines unexpectedly through the fog of
the cosmic dawn. <em>NATURE</em>, <em>639</em>(8056), 870–872. (<a
href="https://doi.org/10.1038/d41586-025-00899-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultraviolet light from a galaxy observed when the Universe was just 330 million years old has intriguing implications for understanding how the first generations of stars and black holes were formed. Ancient UV light illuminates how galaxies and black holes first formed.},
  archive      = {J_NATURE},
  author       = {Trenti, Michele},
  doi          = {10.1038/d41586-025-00899-2},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {870-872},
  shortjournal = {Nature},
  title        = {A lighthouse galaxy shines unexpectedly through the fog of the cosmic dawn},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolutionary flexibility to gain or lose tooth complexity
sparks fish diversification. <em>NATURE</em>, <em>639</em>(8056),
869–870. (<a href="https://doi.org/10.1038/d41586-025-00541-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Why are some species more likely to diversify than others? For the largest group of vertebrates, ray-finned fishes, the ability to evolve by transitioning back and forth between simple and complex teeth fuelled their diversification. Evolutionary insights about how ray-finned fishes diversified.},
  archive      = {J_NATURE},
  author       = {Ngoepe, Nare},
  doi          = {10.1038/d41586-025-00541-1},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {869-870},
  shortjournal = {Nature},
  title        = {Evolutionary flexibility to gain or lose tooth complexity sparks fish diversification},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How to get more women into mining. <em>NATURE</em>,
<em>639</em>(8056), 867. (<a
href="https://doi.org/10.1038/d41586-025-00925-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Huang, Linqi and Li, Xibing and Dong, Longjun and Yu, Haoxuan},
  doi          = {10.1038/d41586-025-00925-3},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {867},
  shortjournal = {Nature},
  title        = {How to get more women into mining},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Give grants to female scientists in war zones.
<em>NATURE</em>, <em>639</em>(8056), 867. (<a
href="https://doi.org/10.1038/d41586-025-00926-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Tsybuliak, Natalia and Suchikova, Yana},
  doi          = {10.1038/d41586-025-00926-2},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {867},
  shortjournal = {Nature},
  title        = {Give grants to female scientists in war zones},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lesotho matters. <em>NATURE</em>, <em>639</em>(8056), 867.
(<a href="https://doi.org/10.1038/d41586-025-00927-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Mokoena-Mokhali, Nthabiseng and Senyane, Liteboho and Stewart, Brian and Mitchell, Peter},
  doi          = {10.1038/d41586-025-00927-1},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {867},
  shortjournal = {Nature},
  title        = {Lesotho matters},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lessons from portugal on effects of cutting research
funding. <em>NATURE</em>, <em>639</em>(8056), 867. (<a
href="https://doi.org/10.1038/d41586-025-00928-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Letter to the Editor},
  archive      = {J_NATURE},
  author       = {Veldhoen, Marc and Barata, João T. and Figueiredo, Luisa M. and Azzalin, Claus M.},
  doi          = {10.1038/d41586-025-00928-0},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {867},
  shortjournal = {Nature},
  title        = {Lessons from portugal on effects of cutting research funding},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Microbes can capture carbon and degrade plastic — why aren’t
we using them more? <em>NATURE</em>, <em>639</em>(8056), 864–866. (<a
href="https://doi.org/10.1038/d41586-025-00875-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interventions involving bacteria or fungi could help to sequester greenhouse gases, create more sustainable products and clean up pollution — in ways that are economically viable and safe. Interventions involving bacteria or fungi could help to sequester greenhouse gases, create more sustainable products and clean up pollution — in ways that are economically viable and safe.},
  archive      = {J_NATURE},
  author       = {Rappuoli, Rino and Nguyen, Nguyen K. and Bloom, David E. and Brooke, Charles and Burckhardt, Rachel M. and Dangour, Alan D. and Egamberdieva, Dilfuza and Gronvall, Gigi K. and Lawley, Trevor D. and Lennon, Jay T. and Morhard, Ryan and Mukhopadhyay, Aindrila and Peixoto, Raquel and Silver, Pamela A. and Stein, Lisa Y.},
  doi          = {10.1038/d41586-025-00875-w},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {864-866},
  shortjournal = {Nature},
  title        = {Microbes can capture carbon and degrade plastic — why aren’t we using them more?},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Glaciers are not just blocks of ice — plans to save them
mustn’t overlook their hidden life. <em>NATURE</em>, <em>639</em>(8056),
861–863. (<a href="https://doi.org/10.1038/d41586-025-00897-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As glaciers begin to disappear, technological fixes to slow or halt ice melt are emerging. But regulations are urgently required before these fixes are used widely. As glaciers begin to disappear, technological fixes to slow or halt ice melt are emerging. But regulations are urgently required before these fixes are used widely.},
  archive      = {J_NATURE},
  author       = {Battin, Tom J. and van Wijngaarden, Albert and Sattler, Birgit and Anesio, Alexandre M. and Cook, Joseph and Huss, Matthias and Edwards, Arwyn},
  doi          = {10.1038/d41586-025-00897-4},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {861-863},
  shortjournal = {Nature},
  title        = {Glaciers are not just blocks of ice — plans to save them mustn’t overlook their hidden life},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The expanding universe — do ongoing tensions leave room for
new physics? <em>NATURE</em>, <em>639</em>(8056), 858–860. (<a
href="https://doi.org/10.1038/d41586-025-00896-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One century after Edwin Hubble revealed his astonishing discovery of a cosmos beyond the Milky Way, the most precise measurements still can’t agree on how fast galaxies are moving. One century after Edwin Hubble revealed his astonishing discovery of a cosmos beyond the Milky Way, the most precise measurements still can’t agree on how fast galaxies are moving.},
  archive      = {J_NATURE},
  author       = {Freedman, Wendy},
  doi          = {10.1038/d41586-025-00896-5},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {858-860},
  shortjournal = {Nature},
  title        = {The expanding universe — do ongoing tensions leave room for new physics?},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fasting for weight loss is all the rage: What are the health
benefits? <em>NATURE</em>, <em>639</em>(8056), 855–857. (<a
href="https://doi.org/10.1038/d41586-025-00895-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Intermittent fasting has gained a following, in part because of tantalizing hints that it can boost cognition, fend off cancer and even slow ageing. Intermittent fasting has gained a following, in part because of tantalizing hints that it can boost cognition, fend off cancer and even slow ageing.},
  archive      = {J_NATURE},
  author       = {Fleming, Nic},
  doi          = {10.1038/d41586-025-00895-6},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {855-857},
  shortjournal = {Nature},
  title        = {Fasting for weight loss is all the rage: What are the health benefits?},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). AI is transforming peer review — and many scientists are
worried. <em>NATURE</em>, <em>639</em>(8056), 852–854. (<a
href="https://doi.org/10.1038/d41586-025-00894-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence software is increasingly involved in reviewing papers, provoking interest and unease. Artificial intelligence software is increasingly involved in reviewing papers, provoking interest and unease.},
  archive      = {J_NATURE},
  author       = {Naddaf, Miryam},
  doi          = {10.1038/d41586-025-00894-7},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {852-854},
  shortjournal = {Nature},
  title        = {AI is transforming peer review — and many scientists are worried},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Immune cells “bandage” wounds with bacteria-trapping goo.
<em>NATURE</em>, <em>639</em>(8056), 850. (<a
href="https://doi.org/10.1038/d41586-025-00796-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells called neutrophils form gooey rings around sites where skin is punctured, study in mice shows. Cells called neutrophils form gooey rings around sites where skin is punctured, study in mice shows.},
  archive      = {J_NATURE},
  author       = {Naddaf, Miryam},
  doi          = {10.1038/d41586-025-00796-8},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {850},
  shortjournal = {Nature},
  title        = {Immune cells ‘bandage’ wounds with bacteria-trapping goo},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Is dark energy getting weaker? Fresh data bolster shock
finding. <em>NATURE</em>, <em>639</em>(8056), 849. (<a
href="https://doi.org/10.1038/d41586-025-00837-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Physicists had long assumed that the elusive force has constant strength. But the latest results from a project to map the Universe’s expansion challenge this idea. Physicists had long assumed that the elusive force has constant strength. But the latest results from a project to map the Universe’s expansion challenge this idea.},
  archive      = {J_NATURE},
  author       = {Castelvecchi, Davide},
  doi          = {10.1038/d41586-025-00837-2},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {849},
  shortjournal = {Nature},
  title        = {Is dark energy getting weaker? fresh data bolster shock finding},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Man survives with titanium heart for 100 days — a world
first. <em>NATURE</em>, <em>639</em>(8056), 848–849. (<a
href="https://doi.org/10.1038/d41586-025-00782-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The device, to be tested in more people, could be used as a temporary measure for those waiting for a donor organ. The device, to be tested in more people, could be used as a temporary measure for those waiting for a donor organ.},
  archive      = {J_NATURE},
  author       = {Mallapaty, Smriti},
  doi          = {10.1038/d41586-025-00782-0},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {848-849},
  shortjournal = {Nature},
  title        = {Man survives with titanium heart for 100 days — a world first},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Catchy, clear, concise: Three-part phrases boost research
paper citations. <em>NATURE</em>, <em>639</em>(8056), 847–848. (<a
href="https://doi.org/10.1038/d41586-025-00771-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Memorable ‘tripartite’ phrases in titles make studies more likely to be read and cited. Memorable ‘tripartite’ phrases in titles make studies more likely to be read and cited.},
  archive      = {J_NATURE},
  author       = {Singh Chawla, Dalmeet},
  doi          = {10.1038/d41586-025-00771-3},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {847-848},
  shortjournal = {Nature},
  title        = {Catchy, clear, concise: Three-part phrases boost research paper citations},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Measles is surging in the US: How bad could it get?
<em>NATURE</em>, <em>639</em>(8056), 846–847. (<a
href="https://doi.org/10.1038/d41586-025-00786-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers flag groups that should get an extra dose of measles vaccine as case numbers continue to rise in Texas and neighbouring states. Researchers flag groups that should get an extra dose of measles vaccine as case numbers continue to rise in Texas and neighbouring states.},
  archive      = {J_NATURE},
  author       = {Nowogrodzki, Julian},
  doi          = {10.1038/d41586-025-00786-w},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {846-847},
  shortjournal = {Nature},
  title        = {Measles is surging in the US: How bad could it get?},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). NIH has cut one mRNA-vaccine grant. Will more follow?
<em>NATURE</em>, <em>639</em>(8056), 844–845. (<a
href="https://doi.org/10.1038/d41586-025-00828-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concern rises among researchers after agency officials ask for a list of projects funded to study the jabs. Concern rises among researchers after agency officials ask for a list of projects funded to study the jabs.},
  archive      = {J_NATURE},
  author       = {Kozlov, Max},
  doi          = {10.1038/d41586-025-00828-3},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {844-845},
  shortjournal = {Nature},
  title        = {NIH has cut one mRNA-vaccine grant. will more follow?},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Light pollution threatens fleet of world-class telescopes in
atacama desert. <em>NATURE</em>, <em>639</em>(8056), 843–844. (<a
href="https://doi.org/10.1038/d41586-025-00397-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effects of a proposed green-energy facility in Chile could be devastating for some of the most powerful instruments available to astronomers. The effects of a proposed green-energy facility in Chile could be devastating for some of the most powerful instruments available to astronomers.},
  archive      = {J_NATURE},
  author       = {Basilio, Humberto},
  doi          = {10.1038/d41586-025-00397-5},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {843-844},
  shortjournal = {Nature},
  title        = {Light pollution threatens fleet of world-class telescopes in atacama desert},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Iguanas reached fiji by floating 8,000 kilometres across the
sea. <em>NATURE</em>, <em>639</em>(8056), 841. (<a
href="https://doi.org/10.1038/d41586-025-00767-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic analysis suggests that the ancestors of lizards on Fiji today rafted from North America some 30 million years ago. Genomic analysis suggests that the ancestors of lizards on Fiji today rafted from North America some 30 million years ago.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-00767-z},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {841},
  shortjournal = {Nature},
  title        = {Iguanas reached fiji by floating 8,000 kilometres across the sea},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Food-industry waste finds a second life as bioplastic.
<em>NATURE</em>, <em>639</em>(8056), 841. (<a
href="https://doi.org/10.1038/d41586-025-00768-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The protein keratin, the principle ingredient of wool and feathers, can be repurposed as strong, flexible plastic. The protein keratin, the principle ingredient of wool and feathers, can be repurposed as strong, flexible plastic.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-00768-y},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {841},
  shortjournal = {Nature},
  title        = {Food-industry waste finds a second life as bioplastic},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ship-pollution cuts have an electrifying effect: Less
lightning at sea. <em>NATURE</em>, <em>639</em>(8056), 841. (<a
href="https://doi.org/10.1038/d41586-025-00769-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A crackdown on sulfur in ship fuel is linked to a steep drop in thunderclouds in the Indian Ocean and South China Sea. A crackdown on sulfur in ship fuel is linked to a steep drop in thunderclouds in the Indian Ocean and South China Sea.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-00769-x},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {841},
  shortjournal = {Nature},
  title        = {Ship-pollution cuts have an electrifying effect: Less lightning at sea},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Drugs that target “zombie” cells soothe an aching back.
<em>NATURE</em>, <em>639</em>(8056), 841. (<a
href="https://doi.org/10.1038/d41586-025-00770-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research on mice show promise for agents that attack senescent cells, which contribute to the degradation of spinal discs. Research on mice show promise for agents that attack senescent cells, which contribute to the degradation of spinal discs.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-00770-4},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {841},
  shortjournal = {Nature},
  title        = {Drugs that target ‘zombie’ cells soothe an aching back},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Empower families to lead the design of their ageing loved
ones’ health care. <em>NATURE</em>, <em>639</em>(8056), 840. (<a
href="https://doi.org/10.1038/d41586-025-00893-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rather than assuming that relatives of older people will step up to fill systemic gaps in health care, communities need research and policies that align with families’ existing routines. Rather than assuming that relatives of older people will step up to fill systemic gaps in health care, communities need research and policies that align with families’ existing routines.},
  archive      = {J_NATURE},
  author       = {Ali, Shahmir H.},
  doi          = {10.1038/d41586-025-00893-8},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {840},
  shortjournal = {Nature},
  title        = {Empower families to lead the design of their ageing loved ones’ health care},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Climate change will send home insurance spiralling. Here’s
how to control costs. <em>NATURE</em>, <em>639</em>(8056), 839. (<a
href="https://doi.org/10.1038/d41586-025-00892-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Insurers rely on sophisticated catastrophe modelling to estimate risk exposure — and as climate risks are rising, so are the costs for protection. Insurers rely on sophisticated catastrophe modelling to estimate risk exposure — and as climate risks are rising, so are the costs for protection.},
  archive      = {J_NATURE},
  author       = {St. George, Scott},
  doi          = {10.1038/d41586-025-00892-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {839},
  shortjournal = {Nature},
  title        = {Climate change will send home insurance spiralling. here’s how to control costs},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Track gender ratios in research to keep countries,
institutions and publishers accountable. <em>NATURE</em>,
<em>639</em>(8056), 838. (<a
href="https://doi.org/10.1038/d41586-025-00891-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Index data reveal how countries and fields differ in gender equity in research. Nature Index data reveal how countries and fields differ in gender equity in research.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-00891-w},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {838},
  shortjournal = {Nature},
  title        = {Track gender ratios in research to keep countries, institutions and publishers accountable},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What CERN does next matters for science and for
international cooperation. <em>NATURE</em>, <em>639</em>(8056), 837–838.
(<a href="https://doi.org/10.1038/d41586-025-00787-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The world’s largest particle-physics laboratory is approaching a pivotal moment in its history. The world’s largest particle-physics laboratory is approaching a pivotal moment in its history.},
  archive      = {J_NATURE},
  doi          = {10.1038/d41586-025-00787-9},
  journal      = {Nature},
  month        = {3},
  number       = {8056},
  pages        = {837-838},
  shortjournal = {Nature},
  title        = {What CERN does next matters for science and for international cooperation},
  volume       = {639},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="nbt---34">NBT - 34</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(3), 452. (<a
href="https://doi.org/10.1038/s41587-025-02588-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02588-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {452},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 2H24 biotech job picture. <em>NBT</em>, <em>43</em>(3), 451.
(<a href="https://doi.org/10.1038/s41587-025-02586-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A semiannual snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-025-02586-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {451},
  shortjournal = {Nature Biotech.},
  title        = {2H24 biotech job picture},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An open letter to graduate students and other
procrastinators: It’s time to write. <em>NBT</em>, <em>43</em>(3),
447–450. (<a href="https://doi.org/10.1038/s41587-025-02584-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hazelett, Dennis J.},
  doi          = {10.1038/s41587-025-02584-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {447-450},
  shortjournal = {Nature Biotech.},
  title        = {An open letter to graduate students and other procrastinators: It’s time to write},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Identification of clinically relevant t
cell receptors for personalized t cell therapy using combinatorial
algorithms. <em>NBT</em>, <em>43</em>(3), 445. (<a
href="https://doi.org/10.1038/s41587-024-02318-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Pétremand, Rémy and Chiffelle, Johanna and Bobisse, Sara and Perez, Marta A. S. and Schmidt, Julien and Arnaud, Marion and Barras, David and Lozano-Rabella, Maria and Genolet, Raphael and Sauvage, Christophe and Saugy, Damien and Michel, Alexandra and Huguenin-Bergenat, Anne-Laure and Capt, Charlotte and Moore, Jonathan S. and De Vito, Claudio and Labidi-Galy, S. Intidhar and Kandalaft, Lana E. and Dangaj Laniti, Denarda and Bassani-Sternberg, Michal and Oliveira, Giacomo and Wu, Catherine J. and Coukos, George and Zoete, Vincent and Harari, Alexandre},
  doi          = {10.1038/s41587-024-02318-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {445},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Computational scoring and experimental
evaluation of enzymes generated by neural networks. <em>NBT</em>,
<em>43</em>(3), 444. (<a
href="https://doi.org/10.1038/s41587-024-02275-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Johnson, Sean R. and Fu, Xiaozhi and Viknander, Sandra and Goldin, Clara and Monaco, Sarah and Zelezniak, Aleksej and Yang, Kevin K.},
  doi          = {10.1038/s41587-024-02275-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {444},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Computational scoring and experimental evaluation of enzymes generated by neural networks},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Bioinstructive implantable scaffolds for
rapid in vivo manufacture and release of CAR-t cells. <em>NBT</em>,
<em>43</em>(3), 443. (<a
href="https://doi.org/10.1038/s41587-025-02572-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Agarwalla, Pritha and Ogunnaike, Edikan A. and Ahn, Sarah and Froehlich, Kristen A. and Jansson, Anton and Ligler, Frances S. and Dotti, Gianpietro and Brudno, Yevgeny},
  doi          = {10.1038/s41587-025-02572-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {443},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Bioinstructive implantable scaffolds for rapid in vivo manufacture and release of CAR-T cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis and benchmarking of small and large genomic
variants across tandem repeats. <em>NBT</em>, <em>43</em>(3), 431–442.
(<a href="https://doi.org/10.1038/s41587-024-02225-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tandem repeats (TRs) are highly polymorphic in the human genome, have thousands of associated molecular traits and are linked to over 60 disease phenotypes. However, they are often excluded from at-scale studies because of challenges with variant calling and representation, as well as a lack of a genome-wide standard. Here, to promote the development of TR methods, we created a catalog of TR regions and explored TR properties across 86 haplotype-resolved long-read human assemblies. We curated variants from the Genome in a Bottle (GIAB) HG002 individual to create a TR dataset to benchmark existing and future TR analysis methods. We also present an improved variant comparison method that handles variants greater than 4 bp in length and varying allelic representation. The 8.1% of the genome covered by the TR catalog holds ~24.9% of variants per individual, including 124,728 small and 17,988 large variants for the GIAB HG002 ‘truth-set’ TR benchmark. We demonstrate the utility of this pipeline across short-read and long-read technologies. A truth-set catalog of human tandem repeats allows benchmarking of variant analysis tools.},
  archive      = {J_NBT},
  author       = {English, Adam C. and Dolzhenko, Egor and Ziaei Jam, Helyaneh and McKenzie, Sean K. and Olson, Nathan D. and De Coster, Wouter and Park, Jonghun and Gu, Bida and Wagner, Justin and Eberle, Michael A. and Gymrek, Melissa and Chaisson, Mark J. P. and Zook, Justin M. and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41587-024-02225-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {431-442},
  shortjournal = {Nature Biotech.},
  title        = {Analysis and benchmarking of small and large genomic variants across tandem repeats},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CRISPRi-based circuits to control gene expression in plants.
<em>NBT</em>, <em>43</em>(3), 416–430. (<a
href="https://doi.org/10.1038/s41587-024-02236-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The construction of synthetic gene circuits in plants has been limited by a lack of orthogonal and modular parts. Here, we implement a CRISPR (clustered regularly interspaced short palindromic repeats) interference (CRISPRi)-based reversible gene circuit platform in plants. We create a toolkit of engineered repressible promoters of different strengths and construct NOT and NOR gates in Arabidopsis thaliana protoplasts. We determine the optimal processing system to express single guide RNAs from RNA Pol II promoters to introduce NOR gate programmability for interfacing with host regulatory sequences. The performance of a NOR gate in stably transformed Arabidopsis plants demonstrates the system’s programmability and reversibility in a complex multicellular organism. Furthermore, cross-species activity of CRISPRi-based logic gates is shown in Physcomitrium patens, Triticum aestivum and Brassica napus protoplasts. Layering multiple NOR gates together creates OR, NIMPLY and AND logic functions, highlighting the modularity of our system. Our CRISPRi circuits are orthogonal, compact, reversible, programmable and modular and provide a platform for sophisticated spatiotemporal control of gene expression in plants. Programmable and reversible CRISPRi-based genetic circuits function in a variety of plants.},
  archive      = {J_NBT},
  author       = {Khan, Muhammad Adil and Herring, Gabrielle and Zhu, Jia Yuan and Oliva, Marina and Fourie, Elliott and Johnston, Benjamin and Zhang, Zhining and Potter, Jarred and Pineda, Luke and Pflueger, Jahnvi and Swain, Tessa and Pflueger, Christian and Lloyd, James P. B. and Secco, David and Small, Ian and Kidd, Brendan N. and Lister, Ryan},
  doi          = {10.1038/s41587-024-02236-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {416-430},
  shortjournal = {Nature Biotech.},
  title        = {CRISPRi-based circuits to control gene expression in plants},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decrypting the molecular basis of cellular drug phenotypes
by dose-resolved expression proteomics. <em>NBT</em>, <em>43</em>(3),
406–415. (<a href="https://doi.org/10.1038/s41587-024-02218-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteomics is making important contributions to drug discovery, from target deconvolution to mechanism of action (MoA) elucidation and the identification of biomarkers of drug response. Here we introduce decryptE, a proteome-wide approach that measures the full dose–response characteristics of drug-induced protein expression changes that informs cellular drug MoA. Assaying 144 clinical drugs and research compounds against 8,000 proteins resulted in more than 1 million dose–response curves that can be interactively explored online in ProteomicsDB and a custom-built Shiny App. Analysis of the collective data provided molecular explanations for known phenotypic drug effects and uncovered new aspects of the MoA of human medicines. We found that histone deacetylase inhibitors potently and strongly down-regulated the T cell receptor complex resulting in impaired human T cell activation in vitro and ex vivo. This offers a rational explanation for the efficacy of histone deacetylase inhibitors in certain lymphomas and autoimmune diseases and explains their poor performance in treating solid tumors. Proteome-wide measurements of dose–response curves for 144 drugs reveal drug mechanisms of action.},
  archive      = {J_NBT},
  author       = {Eckert, Stephan and Berner, Nicola and Kramer, Karl and Schneider, Annika and Müller, Julian and Lechner, Severin and Brajkovic, Sarah and Sakhteman, Amirhossein and Graetz, Christian and Fackler, Jonas and Dudek, Michael and Pfaffl, Michael W. and Knolle, Percy and Wilhelm, Stephanie and Kuster, Bernhard},
  doi          = {10.1038/s41587-024-02218-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {406-415},
  shortjournal = {Nature Biotech.},
  title        = {Decrypting the molecular basis of cellular drug phenotypes by dose-resolved expression proteomics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Computational scoring and experimental evaluation of
enzymes generated by neural networks. <em>NBT</em>, <em>43</em>(3),
396–405. (<a href="https://doi.org/10.1038/s41587-024-02214-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, generative protein sequence models have been developed to sample novel sequences. However, predicting whether generated proteins will fold and function remains challenging. We evaluate a set of 20 diverse computational metrics to assess the quality of enzyme sequences produced by three contrasting generative models: ancestral sequence reconstruction, a generative adversarial network and a protein language model. Focusing on two enzyme families, we expressed and purified over 500 natural and generated sequences with 70–90% identity to the most similar natural sequences to benchmark computational metrics for predicting in vitro enzyme activity. Over three rounds of experiments, we developed a computational filter that improved the rate of experimental success by 50–150%. The proposed metrics and models will drive protein engineering research by serving as a benchmark for generative protein sequence models and helping to select active variants for experimental testing. Metrics that predict the success of folding and the activity of designed protein sequences are developed and experimentally validated.},
  archive      = {J_NBT},
  author       = {Johnson, Sean R. and Fu, Xiaozhi and Viknander, Sandra and Goldin, Clara and Monaco, Sarah and Zelezniak, Aleksej and Yang, Kevin K.},
  doi          = {10.1038/s41587-024-02214-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {396-405},
  shortjournal = {Nature Biotech.},
  title        = {Computational scoring and experimental evaluation of enzymes generated by neural networks},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping variant effects on anti-tumor hallmarks of primary
human t cells with base-editing screens. <em>NBT</em>, <em>43</em>(3),
384–395. (<a href="https://doi.org/10.1038/s41587-024-02235-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nucleotide variants (SNVs) in key T cell genes can drive clinical pathologies and could be repurposed to improve cellular cancer immunotherapies. Here, we perform massively parallel base-editing screens to generate thousands of variants at gene loci annotated with known or potential clinical relevance. We discover a broad landscape of putative gain-of-function (GOF) and loss-of-function (LOF) mutations, including in PIK3CD and the gene encoding its regulatory subunit, PIK3R1, LCK, SOS1, AKT1 and RHOA. Base editing of PIK3CD and PIK3R1 variants in T cells with an engineered T cell receptor specific to a melanoma epitope or in different generations of CD19 chimeric antigen receptor (CAR) T cells demonstrates that discovered GOF variants, but not LOF or silent mutation controls, enhanced signaling, cytokine production and lysis of cognate melanoma and leukemia cell models, respectively. Additionally, we show that generations of CD19 CAR T cells engineered with PIK3CD GOF mutations demonstrate enhanced antigen-specific signaling, cytokine production and leukemia cell killing, including when benchmarked against other recent strategies. Base-editing screens of primary T cells reveal genetic variants beneficial to immunotherapies.},
  archive      = {J_NBT},
  author       = {Walsh, Zachary H. and Shah, Parin and Kothapalli, Neeharika and Shah, Shivem B. and Nikolenyi, Gergo and Brodtman, D. Zack and Leuzzi, Giuseppe and Rogava, Meri and Mu, Michael and Ho, Patricia and Abuzaid, Sinan and Vasan, Neil and AlQuraishi, Mohammed and Milner, Joshua D. and Ciccia, Alberto and Melms, Johannes C. and Izar, Benjamin},
  doi          = {10.1038/s41587-024-02235-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {384-395},
  shortjournal = {Nature Biotech.},
  title        = {Mapping variant effects on anti-tumor hallmarks of primary human t cells with base-editing screens},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered CRISPR-cas12a for higher-order combinatorial
chromatin perturbations. <em>NBT</em>, <em>43</em>(3), 369–383. (<a
href="https://doi.org/10.1038/s41587-024-02224-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to double-stranded DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays. However, current CRISPRi pooled screening approaches are limited to targeting one to three genomic sites per cell. We engineer an Acidaminococcus Cas12a (AsCas12a) variant, multiplexed transcriptional interference AsCas12a (multiAsCas12a), that incorporates R1226A, a mutation that stabilizes the ribonucleoprotein–DNA complex via DNA nicking. The multiAsCas12a-KRAB fusion improves CRISPRi activity over DNase-dead AsCas12a-KRAB fusions, often rescuing the activities of lentivirally delivered CRISPR RNAs (crRNA) that are inactive when used with the latter. multiAsCas12a-KRAB supports CRISPRi using 6-plex crRNA arrays in high-throughput pooled screens. Using multiAsCas12a-KRAB, we discover enhancer elements and dissect the combinatorial function of cis-regulatory elements in human cells. These results instantiate a group testing framework for efficiently surveying numerous combinations of chromatin perturbations for biological discovery and engineering. An engineered Cas12a enables higher-order combinatorial functional genomic screens using CRISPR interference.},
  archive      = {J_NBT},
  author       = {Hsiung, C. C.-S. and Wilson, C. M. and Sambold, N. A. and Dai, R. and Chen, Q. and Teyssier, N. and Misiukiewicz, S. and Arab, A. and O’Loughlin, T. and Cofsky, J. C. and Shi, J. and Gilbert, L. A.},
  doi          = {10.1038/s41587-024-02224-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {369-383},
  shortjournal = {Nature Biotech.},
  title        = {Engineered CRISPR-cas12a for higher-order combinatorial chromatin perturbations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bidirectional epigenetic editing reveals hierarchies in gene
regulation. <em>NBT</em>, <em>43</em>(3), 355–368. (<a
href="https://doi.org/10.1038/s41587-024-02213-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR perturbation methods are limited in their ability to study non-coding elements and genetic interactions. In this study, we developed a system for bidirectional epigenetic editing, called CRISPRai, in which we apply activating (CRISPRa) and repressive (CRISPRi) perturbations to two loci simultaneously in the same cell. We developed CRISPRai Perturb-seq by coupling dual perturbation gRNA detection with single-cell RNA sequencing, enabling study of pooled perturbations in a mixed single-cell population. We applied this platform to study the genetic interaction between two hematopoietic lineage transcription factors, SPI1 and GATA1, and discovered novel characteristics of their co-regulation on downstream target genes, including differences in SPI1 and GATA1 occupancy at genes that are regulated through different modes. We also studied the regulatory landscape of IL2 (interleukin-2) in Jurkat T cells, primary T cells and chimeric antigen receptor (CAR) T cells and elucidated mechanisms of enhancer-mediated IL2 gene regulation. CRISPRai facilitates investigation of context-specific genetic interactions, provides new insights into gene regulation and will enable exploration of non-coding disease-associated variants. CRISPRai simultaneously activates and represses two genes in single cells.},
  archive      = {J_NBT},
  author       = {Pacalin, Naomi M. and Steinhart, Zachary and Shi, Quanming and Belk, Julia A. and Dorovskyi, Dmytro and Kraft, Katerina and Parker, Kevin R. and Shy, Brian R. and Marson, Alexander and Chang, Howard Y.},
  doi          = {10.1038/s41587-024-02213-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {355-368},
  shortjournal = {Nature Biotech.},
  title        = {Bidirectional epigenetic editing reveals hierarchies in gene regulation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Self-pigmenting textiles grown from cellulose-producing
bacteria with engineered tyrosinase expression. <em>NBT</em>,
<em>43</em>(3), 345–354. (<a
href="https://doi.org/10.1038/s41587-024-02194-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Environmental concerns are driving interest in postpetroleum synthetic textiles produced from microbial and fungal sources. Bacterial cellulose (BC) is a promising sustainable leather alternative, on account of its material properties, low infrastructure needs and biodegradability. However, for alternative textiles like BC to be fully sustainable, alternative ways to dye textiles need to be developed alongside alternative production methods. To address this, we genetically engineer Komagataeibacter rhaeticus to create a bacterial strain that grows self-pigmenting BC. Melanin biosynthesis in the bacteria from recombinant tyrosinase expression achieves dark black coloration robust to material use. Melanated BC production can be scaled up for the construction of prototype fashion products, and we illustrate the potential of combining engineered self-pigmentation with tools from synthetic biology, through the optogenetic patterning of gene expression in cellulose-producing bacteria. With this study, we demonstrate that combining genetic engineering with current and future methods of textile biofabrication has the potential to create a new class of textiles. Genetic engineering of material-producing bacteria creates pigmented products.},
  archive      = {J_NBT},
  author       = {Walker, Kenneth T. and Li, Ivy S. and Keane, Jennifer and Goosens, Vivianne J. and Song, Wenzhe and Lee, Koon-Yang and Ellis, Tom},
  doi          = {10.1038/s41587-024-02194-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {345-354},
  shortjournal = {Nature Biotech.},
  title        = {Self-pigmenting textiles grown from cellulose-producing bacteria with engineered tyrosinase expression},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Generation of allogeneic CAR-NKT cells from hematopoietic
stem and progenitor cells using a clinically guided culture method.
<em>NBT</em>, <em>43</em>(3), 329–344. (<a
href="https://doi.org/10.1038/s41587-024-02226-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer immunotherapy with autologous chimeric antigen receptor (CAR) T cells faces challenges in manufacturing and patient selection that could be avoided by using ‘off-the-shelf’ products, such as allogeneic CAR natural killer T (AlloCAR-NKT) cells. Previously, we reported a system for differentiating human hematopoietic stem and progenitor cells into AlloCAR-NKT cells, but the use of three-dimensional culture and xenogeneic feeders precluded its clinical application. Here we describe a clinically guided method to differentiate and expand IL-15-enhanced AlloCAR-NKT cells with high yield and purity. We generated AlloCAR-NKT cells targeting seven cancers and, in a multiple myeloma model, demonstrated their antitumor efficacy, expansion and persistence. The cells also selectively depleted immunosuppressive cells in the tumor microenviroment and antagonized tumor immune evasion via triple targeting of CAR, TCR and NK receptors. They exhibited a stable hypoimmunogenic phenotype associated with epigenetic and signaling regulation and did not induce detectable graft versus host disease or cytokine release syndrome. These properties of AlloCAR-NKT cells support their potential for clinical translation. Off-the-shelf CAR-NKT cells for cancer immunotherapy are advanced toward clinical translation.},
  archive      = {J_NBT},
  author       = {Li, Yan-Ruide and Zhou, Yang and Yu, Jiaji and Kim, Yu Jeong and Li, Miao and Lee, Derek and Zhou, Kuangyi and Chen, Yuning and Zhu, Yichen and Wang, Yu-Chen and Li, Zhe and Yu, Yanqi and Dunn, Zachary Spencer and Guo, Wenbin and Cen, Xinjian and Husman, Tiffany and Bajpai, Aarushi and Kramer, Adam and Wilson, Matthew and Fang, Ying and Huang, Jie and Li, Shuo and Zhou, Yonggang and Zhang, Yuchong and Hahn, Zoe and Zhu, Enbo and Ma, Feiyang and Pan, Calvin and Lusis, Aldons J. and Zhou, Jin J. and Seet, Christopher S. and Kohn, Donald B. and Wang, Pin and Zhou, Xianghong Jasmine and Pellegrini, Matteo and Puliafito, Benjamin R. and Larson, Sarah M. and Yang, Lili},
  doi          = {10.1038/s41587-024-02226-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {329-344},
  shortjournal = {Nature Biotech.},
  title        = {Generation of allogeneic CAR-NKT cells from hematopoietic stem and progenitor cells using a clinically guided culture method},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Identification of clinically relevant t cell receptors for
personalized t cell therapy using combinatorial algorithms.
<em>NBT</em>, <em>43</em>(3), 323–328. (<a
href="https://doi.org/10.1038/s41587-024-02232-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central challenge in developing personalized cancer cell immunotherapy is the identification of tumor-reactive T cell receptors (TCRs). By exploiting the distinct transcriptomic profile of tumor-reactive T cells relative to bystander cells, we build and benchmark TRTpred, an antigen-agnostic in silico predictor of tumor-reactive TCRs. We integrate TRTpred with an avidity predictor to derive a combinatorial algorithm of clinically relevant TCRs for personalized T cell therapy and benchmark it in patient-derived xenografts. Tumor-reactive T cell receptors are predicted based on single-cell RNA sequencing and single-cell T cell receptor sequencing.},
  archive      = {J_NBT},
  author       = {Pétremand, Rémy and Chiffelle, Johanna and Bobisse, Sara and Perez, Marta A. S. and Schmidt, Julien and Arnaud, Marion and Barras, David and Lozano-Rabella, Maria and Genolet, Raphael and Sauvage, Christophe and Saugy, Damien and Michel, Alexandra and Huguenin-Bergenat, Anne-Laure and Capt, Charlotte and Moore, Jonathan S. and De Vito, Claudio and Labidi-Galy, S. Intidhar and Kandalaft, Lana E. and Dangaj Laniti, Denarda and Bassani-Sternberg, Michal and Oliveira, Giacomo and Wu, Catherine J. and Coukos, George and Zoete, Vincent and Harari, Alexandre},
  doi          = {10.1038/s41587-024-02232-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {323-328},
  shortjournal = {Nature Biotech.},
  title        = {Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward single-molecule protein sequencing using nanopores.
<em>NBT</em>, <em>43</em>(3), 312–322. (<a
href="https://doi.org/10.1038/s41587-025-02587-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past three decades, biological nanopore sequencing has grown from a research curiosity to a mature technology to sequence nucleic acids at the single-molecule level. Now, recent achievements suggest that nanopores might be able to sequence proteins soon. In this Perspective, we analyze the different approaches that have been proposed to measure proteins and peptides using nanopores. We predict that, more likely than not, nanopores will be capable of identifying full-length proteins at the single-molecule level and with single-amino acid resolution, paving the way to single-molecule protein sequencing. This would allow several applications in proteomics that are at present challenging, including measuring the heterogeneity of post-translational modifications, quantifying low-abundance proteins and characterizing protein splicing. Maglia and colleagues discuss advances in nanopore technology en route to single-molecule protein sequencing},
  archive      = {J_NBT},
  author       = {Lu, Chunzhe and Bonini, Andrea and Viel, Jakob H. and Maglia, Giovanni},
  doi          = {10.1038/s41587-025-02587-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {312-322},
  shortjournal = {Nature Biotech.},
  title        = {Toward single-molecule protein sequencing using nanopores},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing programmable gene expression in plants using
CRISPRi-based boolean gates. <em>NBT</em>, <em>43</em>(3), 310–311. (<a
href="https://doi.org/10.1038/s41587-024-02273-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To advance the toolset for controlling plant gene expression, we developed a CRISPR interference-based platform for the construction of synthetic Boolean logic gates that is functional in multiple plant species. These genetic circuits are programmable and reversible in nature, which will enable spatiotemporal control of plant responses to dynamic cues.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02273-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {310-311},
  shortjournal = {Nature Biotech.},
  title        = {Advancing programmable gene expression in plants using CRISPRi-based boolean gates},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Assessing the laboratory performance of AI-generated
enzymes. <em>NBT</em>, <em>43</em>(3), 308–309. (<a
href="https://doi.org/10.1038/s41587-024-02239-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A set of 20 computational metrics was evaluated to determine whether they could predict the functionality of synthetic enzyme sequences produced by generative protein models, resulting in the development of a computational filter, COMPSS, that increased experimental success rates by 50–150%, tested in over 500 natural and AI-generated enzymes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02239-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {308-309},
  shortjournal = {Nature Biotech.},
  title        = {Assessing the laboratory performance of AI-generated enzymes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetically encoding colors and images into bioengineered
microbial materials. <em>NBT</em>, <em>43</em>(3), 306–307. (<a
href="https://doi.org/10.1038/s41587-024-02200-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using synthetic biology, we engineered a cellulose-producing bacterium that can produce eumelanin and respond to light, so that it is possible to grow a microbial leather material that is colored black or contains projected black patterns.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02200-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {306-307},
  shortjournal = {Nature Biotech.},
  title        = {Genetically encoding colors and images into bioengineered microbial materials},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decoding gene regulation with CRISPR perturbations.
<em>NBT</em>, <em>43</em>(3), 304–305. (<a
href="https://doi.org/10.1038/s41587-024-02222-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two CRISPR tools for combinatorial genetic perturbations reveal gene regulatory networks.},
  archive      = {J_NBT},
  author       = {Oberlin, Stefan and McManus, Michael T.},
  doi          = {10.1038/s41587-024-02222-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {304-305},
  shortjournal = {Nature Biotech.},
  title        = {Decoding gene regulation with CRISPR perturbations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A milestone method to make natural killer t cells.
<em>NBT</em>, <em>43</em>(3), 302–303. (<a
href="https://doi.org/10.1038/s41587-024-02243-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A differentiation protocol to produce off-the-shelf natural killer T cells may enable clinical application.},
  archive      = {J_NBT},
  author       = {Metelitsa, Leonid S.},
  doi          = {10.1038/s41587-024-02243-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {302-303},
  shortjournal = {Nature Biotech.},
  title        = {A milestone method to make natural killer t cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Intranasal bacteria transport obesity drugs to the brain.
<em>NBT</em>, <em>43</em>(3), 301. (<a
href="https://doi.org/10.1038/s41587-025-02598-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02598-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Intranasal bacteria transport obesity drugs to the brain},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Fluorescence imaging of RNA and protein in deep tissue.
<em>NBT</em>, <em>43</em>(3), 301. (<a
href="https://doi.org/10.1038/s41587-025-02604-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02604-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Fluorescence imaging of RNA and protein in deep tissue},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025c). Microbial metalloenzyme boosts cellulose breakdown.
<em>NBT</em>, <em>43</em>(3), 301. (<a
href="https://doi.org/10.1038/s41587-025-02615-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02615-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Microbial metalloenzyme boosts cellulose breakdown},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Epigenetics. <em>NBT</em>, <em>43</em>(3), 300. (<a
href="https://doi.org/10.1038/s41587-025-02582-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to epigenetic modification and analysis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02582-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {300},
  shortjournal = {Nature Biotech.},
  title        = {Epigenetics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The puzzle of biologics manufacturing platform patents.
<em>NBT</em>, <em>43</em>(3), 295–299. (<a
href="https://doi.org/10.1038/s41587-025-02579-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The predominance of manufacturing process patents asserted in litigation by originator biologics companies against would-be biosimilar entrants has resulted in a number of Congressional and administrative agency proposals that could increase scrutiny and limit enforceability.},
  archive      = {J_NBT},
  author       = {Jefferson, Osmat Azzam and Price, W. Nicholson and Tu, S. Sean and Vishnubhakat, Saurabh and Rai, Arti K.},
  doi          = {10.1038/s41587-025-02579-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {295-299},
  shortjournal = {Nature Biotech.},
  title        = {The puzzle of biologics manufacturing platform patents},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>,
<em>43</em>(3), 294. (<a
href="https://doi.org/10.1038/s41587-025-02599-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02599-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {294},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trapped in a lake, founder spots a bioplastic opportunity.
<em>NBT</em>, <em>43</em>(3), 293. (<a
href="https://doi.org/10.1038/s41587-025-02605-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-025-02605-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {293},
  shortjournal = {Nature Biotech.},
  title        = {Trapped in a lake, founder spots a bioplastic opportunity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tough times for CRISPR startups. <em>NBT</em>,
<em>43</em>(3), 292. (<a
href="https://doi.org/10.1038/s41587-025-02609-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02609-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {292},
  shortjournal = {Nature Biotech.},
  title        = {Tough times for CRISPR startups},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Baby genomics: Newborn sequencing starts to fulfill its
promise. <em>NBT</em>, <em>43</em>(3), 290–292. (<a
href="https://doi.org/10.1038/s41587-025-02595-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome sequencing in newborns is delivering insights into severe genetic diseases not afforded by traditional tests, but integrating genomics into newborn screening remains a logistics and cost challenge.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-025-02595-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {290-292},
  shortjournal = {Nature Biotech.},
  title        = {Baby genomics: Newborn sequencing starts to fulfill its promise},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Philanthropic giants join forces over antimicrobial
resistance. <em>NBT</em>, <em>43</em>(3), 289. (<a
href="https://doi.org/10.1038/s41587-025-02620-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02620-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {289},
  shortjournal = {Nature Biotech.},
  title        = {Philanthropic giants join forces over antimicrobial resistance},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). First inhaled lentiviral gene therapy enters cystic fibrosis
trial. <em>NBT</em>, <em>43</em>(3), 288. (<a
href="https://doi.org/10.1038/s41587-025-02616-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02616-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {288},
  shortjournal = {Nature Biotech.},
  title        = {First inhaled lentiviral gene therapy enters cystic fibrosis trial},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bite-sized solutions. <em>NBT</em>, <em>43</em>(3), 285–286.
(<a href="https://doi.org/10.1038/s41587-025-02603-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study found high amounts of microplastics in the human brain. This could spur funding and technological advances for plastic degradation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02603-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {285-286},
  shortjournal = {Nature Biotech.},
  title        = {Bite-sized solutions},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="nchem---27">NCHEM - 27</h2>
<ul>
<li><details>
<summary>
(2025). The phosphate of life. <em>NCHEM</em>, <em>17</em>(3), 460.
(<a href="https://doi.org/10.1038/s41557-025-01758-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elena De Vita and Rebecca Page reflect on the unique properties of phosphate, an essential building block with versatile functions in living systems. Modulating protein phosphorylation is an effective therapeutic strategy, with emerging approaches highlighting the continuous development in this area of drug discovery.},
  archive      = {J_NCHEM},
  author       = {De Vita, Elena and Page, Rebecca},
  doi          = {10.1038/s41557-025-01758-3},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {460},
  shortjournal = {Nat. Chem.},
  title        = {The phosphate of life},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Protocells by spontaneous reaction of
cysteine with short-chain thioesters. <em>NCHEM</em>, <em>17</em>(3),
459. (<a href="https://doi.org/10.1038/s41557-025-01736-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NCHEM},
  author       = {Cho, Christy J. and An, Taeyang and Lai, Yei-Chen and Vázquez-Salazar, Alberto and Fracassi, Alessandro and Brea, Roberto J. and Chen, Irene A. and Devaraj, Neal K.},
  doi          = {10.1038/s41557-025-01736-9},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {459},
  shortjournal = {Nat. Chem.},
  title        = {Publisher correction: Protocells by spontaneous reaction of cysteine with short-chain thioesters},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Engineering a DNA polymerase for
modifying large RNA at specific positions. <em>NCHEM</em>,
<em>17</em>(3), 458. (<a
href="https://doi.org/10.1038/s41557-025-01752-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NCHEM},
  author       = {Chen, Dian and Han, Zhanghui and Liang, Xiaoge and Liu, Yu},
  doi          = {10.1038/s41557-025-01752-9},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {458},
  shortjournal = {Nat. Chem.},
  title        = {Author correction: Engineering a DNA polymerase for modifying large RNA at specific positions},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Discovery of the selenium-containing
antioxidant ovoselenol derived from convergent evolution.
<em>NCHEM</em>, <em>17</em>(3), 457. (<a
href="https://doi.org/10.1038/s41557-025-01739-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NCHEM},
  author       = {Kayrouz, Chase M. and Ireland, Kendra A. and Ying, Vanessa Y. and Davis, Katherine M. and Seyedsayamdost, Mohammad R.},
  doi          = {10.1038/s41557-025-01739-6},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {457},
  shortjournal = {Nat. Chem.},
  title        = {Author correction: Discovery of the selenium-containing antioxidant ovoselenol derived from convergent evolution},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Cytoskeleton-functionalized synthetic
cells with life-like mechanical features and regulated membrane
dynamicity. <em>NCHEM</em>, <em>17</em>(3), 457. (<a
href="https://doi.org/10.1038/s41557-025-01740-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NCHEM},
  author       = {Novosedlik, Sebastian and Reichel, Felix and van Veldhuisen, Thijs and Li, Yudong and Wu, Hanglong and Janssen, Henk and Guck, Jochen and van Hest, Jan},
  doi          = {10.1038/s41557-025-01740-z},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {457},
  shortjournal = {Nat. Chem.},
  title        = {Author correction: Cytoskeleton-functionalized synthetic cells with life-like mechanical features and regulated membrane dynamicity},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synthetic biomolecular condensates enhance translation from
a target mRNA in living cells. <em>NCHEM</em>, <em>17</em>(3), 448–456.
(<a href="https://doi.org/10.1038/s41557-024-01706-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biomolecular condensates composed of proteins and RNA are one approach by which cells regulate post-transcriptional gene expression. Their formation typically involves the phase separation of intrinsically disordered proteins with a target mRNA, sequestering the mRNA into a liquid condensate. This sequestration regulates gene expression by modulating translation or facilitating RNA processing. Here we engineer synthetic condensates using a fusion of an RNA-binding protein, the human Pumilio2 homology domain (Pum2), and a synthetic intrinsically disordered protein, an elastin-like polypeptide (ELP), that can bind and sequester a target mRNA transcript. In protocells, sequestration of a target mRNA largely limits its translation. Conversely, in Escherichia coli, sequestration of the same target mRNA increases its translation. We characterize the Pum2–ELP condensate system using microscopy, biophysical and biochemical assays, and RNA sequencing. This approach enables the modulation of cell function via the formation of synthetic biomolecular condensates that regulate the expression of a target protein. Formation of biomolecular condensates composed of proteins and RNA facilitates the regulation of gene expression by modulating translation or facilitating RNA processing. Now, synthetic ribonucleoprotein granules created with engineered intrinsically disordered proteins selectively sequester mRNA and enhance protein translation in cells. These highly liquid-like condensates exchange biomolecules across the cell and facilitate target mRNA and ribosome partitioning.},
  archive      = {J_NCHEM},
  author       = {Shapiro, Daniel Mark and Deshpande, Sonal and Eghtesadi, Seyed Ali and Zhong, Miranda and Fontes, Cassio Mendes and Fiflis, David and Rohm, Dahlia and Min, Junseon and Kaur, Taranpreet and Peng, Joanna and Ney, Max and Su, Jonathan and Dai, Yifan and Asokan, Aravind and Gersbach, Charles A. and Chilkoti, Ashutosh},
  doi          = {10.1038/s41557-024-01706-7},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {448-456},
  shortjournal = {Nat. Chem.},
  title        = {Synthetic biomolecular condensates enhance translation from a target mRNA in living cells},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vibrational weak and strong coupling modify a chemical
reaction via cavity-mediated radiative energy transfer. <em>NCHEM</em>,
<em>17</em>(3), 439–447. (<a
href="https://doi.org/10.1038/s41557-024-01723-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Controlling reaction outcomes through external influences is a central goal in chemistry. Vibrational coupling between molecular vibrations and cavity modes is rapidly emerging as a distinct strategy compared with conventional thermochemical and photochemical methods; however, insight into the fundamental mechanisms remains limited. Here we investigate how vibrational weak and strong coupling in plasmonic nanocavities modifies the thermal dehydration of copper sulfate pentahydrate. We demonstrate that light–matter coupling reduces the onset temperature for dehydration by up to 14 °C, and we attribute this effect to enhanced radiative energy transport that is mediated by resonant electromagnetic modes, eliminating temperature gradients in the coupled system. Our findings provide direct evidence of localized energy transfer leading to modified chemical behaviour in specific regions of high optical density of states. This work establishes a mechanism for modifying thermally driven chemical processes using optical cavities, with implications for the development of catalytic systems that exploit these tailored interactions to achieve targeted reaction control. Vibrational weak and strong light–matter coupling in infrared nanocavities modifies chemical processes. Now it has been shown that this coupling can control thermally driven reactions through enhanced radiative energy transport.},
  archive      = {J_NCHEM},
  author       = {Brawley, Zachary T. and Pannir-Sivajothi, Sindhana and Yim, Ju Eun and Poh, Yong Rui and Yuen-Zhou, Joel and Sheldon, Matthew},
  doi          = {10.1038/s41557-024-01723-6},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {439-447},
  shortjournal = {Nat. Chem.},
  title        = {Vibrational weak and strong coupling modify a chemical reaction via cavity-mediated radiative energy transfer},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational design of highly signalling-active membrane
receptors through solvent-mediated allosteric networks. <em>NCHEM</em>,
<em>17</em>(3), 429–438. (<a
href="https://doi.org/10.1038/s41557-024-01719-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein catalysis and allostery require the atomic-level orchestration and motion of residues and ligand, solvent and protein effector molecules. However, the ability to design protein activity through precise protein–solvent cooperative interactions has not yet been demonstrated. Here we report the design of 14 membrane receptors that catalyse G protein nucleotide exchange through diverse engineered allosteric pathways mediated by cooperative networks of intraprotein, protein–ligand and –solvent molecule interactions. Consistent with predictions, the designed protein activities correlated well with the level of plasticity of the networks at flexible transmembrane helical interfaces. Several designs displayed considerably enhanced thermostability and activity compared with related natural receptors. The most stable and active variant crystallized in an unforeseen signalling-active conformation, in excellent agreement with the design models. The allosteric network topologies of the best designs bear limited similarity to those of natural receptors and reveal an allosteric interaction space larger than previously inferred from natural proteins. The approach should prove useful for engineering proteins with novel complex protein binding, catalytic and signalling activities. Solvent-mediated interaction networks control a wide range of protein functions, but their design has been neglected owing to a lack of accurate computational tools. Now it is shown that allosteric signalling membrane proteins can be engineered through such networks, revealing a broader space of designable protein interactions and functions.},
  archive      = {J_NCHEM},
  author       = {Chen, K.-Y. M. and Lai, J. K. and Rudden, L. S. P. and Wang, J. and Russell, A. M. and Conners, K. and Rutter, M. E. and Condon, B. and Tung, F. and Kodandapani, L. and Chau, B. and Zhao, X. and Benach, J. and Baker, K. and Hembre, E. J. and Barth, P.},
  doi          = {10.1038/s41557-024-01719-2},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {429-438},
  shortjournal = {Nat. Chem.},
  title        = {Computational design of highly signalling-active membrane receptors through solvent-mediated allosteric networks},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synthesis of pillar-layered metal–organic frameworks with
variable backbones through sequence control. <em>NCHEM</em>,
<em>17</em>(3), 421–428. (<a
href="https://doi.org/10.1038/s41557-024-01717-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The properties and functions of metal–organic frameworks (MOFs) can be tailored by tuning their structure, including their shape, porosity and topology. However, the design and synthesis of complex structures in a predictable manner remains challenging. Here we report the preparation of a series of isomeric pillar-layered MOFs, and we show that their three-dimensional topology can be controlled by altering the layer stacking. This enables variability on the backbone structure, as well as diverse spatial arrangements of pillars and the partitioning of pore space into several kinds of cages packing in distinct sequences. These sequence-controlled MOFs (SC-MOF-1–6) showcase ultrahigh benzene capture capacities at low-pressure and high volumetric and gravimetric uptake performances in high-pressure methane storage. We provide the construction principles of the SC-MOFs and predict nearly 2,000 possible SC-networks with sophisticated composition sequences at the atomic level by using a Python script. The tailoring of reticular materials is key for enhancing the complexity and diversity of their structure and function. Now, a series of isomeric pillar-layered metal–organic frameworks with tunable topologies have been prepared through altering the layer stacking, which enables variability on the backbone structure, pillar spatial arrangements and pore structure.},
  archive      = {J_NCHEM},
  author       = {Yuan, Jingjing and Yang, Ming and Yang, Bin and Chen, Shuting and Liu, Zhiqiang and Pang, Qingqing and Wan, Mingyu and Zheng, Anmin and Tu, Binbin},
  doi          = {10.1038/s41557-024-01717-4},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {421-428},
  shortjournal = {Nat. Chem.},
  title        = {Synthesis of pillar-layered metal–organic frameworks with variable backbones through sequence control},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient and modular synthesis of ibogaine and related
alkaloids. <em>NCHEM</em>, <em>17</em>(3), 412–420. (<a
href="https://doi.org/10.1038/s41557-024-01714-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Anecdotal reports and preliminary clinical trials suggest that the psychoactive alkaloid ibogaine and its active metabolite noribogaine have powerful anti-addictive properties, producing long-lasting therapeutic effects across a range of substance use disorders and co-occurring neuropsychiatric diseases such as depression and post-traumatic stress disorder. Here we report a gram-scale, seven-step synthesis of ibogaine from pyridine. Key features of this strategy enabled the synthesis of three additional iboga alkaloids, as well as an enantioselective total synthesis of (+)-ibogaine and the construction of four analogues. Biological testing revealed that the unnatural enantiomer of ibogaine does not produce ibogaine-like effects on cortical neuron growth, while (−)-10-fluoroibogamine exhibits exceptional psychoplastogenic properties and is a potent modulator of the serotonin transporter. This work provides a platform for accessing iboga alkaloids and congeners for further biological study. Preliminary clinical trials suggest that ibogaine and its active metabolite noribogaine have powerful anti-addictive properties, Now, a strategy for the scalable, asymmetric total synthesis of ibogaine has been developed that also provides access to iboga analogues. Biological testing identified a psychoplastogenic iboga analogue that is a potent modulator of the serotonin transporter.},
  archive      = {J_NCHEM},
  author       = {Iyer, Rishab N. and Favela, David and Domokos, Andras and Zhang, Guoliang and Avanes, Arabo A. and Carter, Samuel J. and Basargin, Andrian G. and Davis, Alexis R. and Tantillo, Dean J. and Olson, David E.},
  doi          = {10.1038/s41557-024-01714-7},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {412-420},
  shortjournal = {Nat. Chem.},
  title        = {Efficient and modular synthesis of ibogaine and related alkaloids},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The mechanism of amyloid fibril growth from φ-value
analysis. <em>NCHEM</em>, <em>17</em>(3), 403–411. (<a
href="https://doi.org/10.1038/s41557-024-01712-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amyloid fibrils are highly stable misfolded protein assemblies that play an important role in several neurodegenerative and systemic diseases. Although structural information of the amyloid state is now abundant, mechanistic details about the misfolding process remain elusive. Inspired by the Φ-value analysis of protein folding, we combined experiments and molecular simulations to resolve amino-acid contacts and determine the structure of the transition-state ensemble—the rate-limiting step—for fibril elongation of PI3K-SH3 amyloid fibrils. The ensemble was validated experimentally by Tanford β analysis and computationally by free energy calculations. Although protein folding proceeds on funnel-shaped landscapes, here we find that the energy landscape for the misfolding reaction consists of a large ‘golf course’ region, defined by a single energy barrier and transition state, accessing a sharply funnelled region. Thus, misfolding occurs by rare, successful monomer–fibril end collisions interspersed by numerous unsuccessful binding attempts. Taken together, these insights provide a quantitative and highly resolved description of a protein misfolding reaction. Amyloid fibrils grow through the recruitment of soluble monomer to the fibril end that propagates the fibril structure. Here the transition state, the rate-limiting conformation, of such a reaction has been characterized by Φ-value analysis. An energy landscape model has been developed and fibril growth rates predicted from first principles.},
  archive      = {J_NCHEM},
  author       = {Larsen, Jacob Aunstrup and Barclay, Abigail and Vettore, Nicola and Klausen, Louise K. and Mangels, Lena N. and Coden, Alberto and Schmit, Jeremy D. and Lindorff-Larsen, Kresten and Buell, Alexander K.},
  doi          = {10.1038/s41557-024-01712-9},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {403-411},
  shortjournal = {Nat. Chem.},
  title        = {The mechanism of amyloid fibril growth from Φ-value analysis},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enantioselective synthesis of 2-substituted
bicyclo[1.1.1]pentanes via sequential asymmetric imine addition of
bicyclo[1.1.0]butanes and skeletal editing. <em>NCHEM</em>,
<em>17</em>(3), 393–402. (<a
href="https://doi.org/10.1038/s41557-024-01710-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The substitution of an aromatic ring with a C(sp3)-rich bicyclic hydrocarbon, known as bioisosteric replacement, plays a crucial role in modern drug discovery. Substituted bicyclo[1.1.1]pentanes (BCPs) are particularly noteworthy owing to their uniquely three-dimensional stereochemical complexity. 1,3-Difunctionalized BCPs have been widely used as bioisosteres for para-substituted phenyl rings, and they have been incorporated into numerous lead pharmaceutical candidates. 2-Substituted BCPs (substituted at the bridge position) can function as alternatives to ortho- or meta-substituted arene rings; however, the general and efficient construction of these scaffolds remains challenging, particularly if performed in an enantioselective manner. Here we present an approach for synthesizing enantioenriched 2-substituted BCPs by a nitrogen-atom insertion-and-deletion strategy, involving a chiral Brønsted acid-catalytic enantioselective cycloaddition of bicyclo[1.1.0]butanes with imines and nitrogen deletion of resulting aza-bicyclo[2.1.1]hexanes (aza-BCHs) with generally good enantiopurity retention. Mechanistic experiments verify the radical pathway. Chiral BCPs have been readily incorporated into medicinally relevant molecules, and a drug analogue has been successfully prepared enantioselectively. Substituted bicyclo[1.1.1]pentanes (BCPs) are widely used as bioisosteres for para-substituted phenyl rings, providing improved pharmacological profiles for drug candidates, but strategies for the preparation of chiral BCPs remain limited. Now a route to chiral bridge-substituted BCPs has been developed via a nitrogen-atom insertion-and-deletion strategy, enabling a practical avenue towards chiral BCP bioisosteres of lomitapide.},
  archive      = {J_NCHEM},
  author       = {Che, Jin-Teng and Ding, Wei-Yi and Zhang, Hong-Bo and Wang, Yong-Bin and Xiang, Shao-Hua and Tan, Bin},
  doi          = {10.1038/s41557-024-01710-x},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {393-402},
  shortjournal = {Nat. Chem.},
  title        = {Enantioselective synthesis of 2-substituted bicyclo[1.1.1]pentanes via sequential asymmetric imine addition of bicyclo[1.1.0]butanes and skeletal editing},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Engineering a DNA polymerase for modifying large RNA at
specific positions. <em>NCHEM</em>, <em>17</em>(3), 382–392. (<a
href="https://doi.org/10.1038/s41557-024-01707-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The synthesis of large RNA with precise modifications at specific positions is in high demand for both basic research and therapeutic applications, but efficient methods are limited. Engineered DNA polymerases have recently emerged as attractive tools for RNA labelling, offering distinct advantages over conventional RNA polymerases. Here, through semi-rational designs, we engineered a DNA polymerase variant and used it to precisely incorporate a diverse range of modifications, including base modifications, 2′-ribose modifications and backbone modifications, into desired positions within RNA. We achieved efficiencies exceeding 85% in the majority of modification cases, demonstrating success in introducing 2′-O-methyl, phosphorothioate, N4-acetylcytidine and a fluorophore to specific sites in eGFP and Firefly luciferase messenger RNA. Our mRNA products with N4-acetylcytidine, 2′-O-methyl and/or phosphorothioate have demonstrated the ability to enhance stability and affect protein production. This method presents a promising tool for the comprehensive functionalization of RNA, enabling the introduction of plentiful modifications irrespective of RNA lengths and sequences. The demand for large, position-specific modified RNA molecules is high across diverse fields. Now a DNA polymerase has been engineered to enable the efficient and flexible synthesis of such molecules using a pause–restart strategy. This methodology can be implemented in both liquid and hybrid solid–liquid phases.},
  archive      = {J_NCHEM},
  author       = {Chen, Dian and Han, Zhanghui and Liang, Xiaoge and Liu, Yu},
  doi          = {10.1038/s41557-024-01707-6},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {382-392},
  shortjournal = {Nat. Chem.},
  title        = {Engineering a DNA polymerase for modifying large RNA at specific positions},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Amphoteric chalcogen-bonding and halogen-bonding rotaxanes
for anion or cation recognition. <em>NCHEM</em>, <em>17</em>(3),
373–381. (<a href="https://doi.org/10.1038/s41557-025-01742-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ever-increasing demand in the development of host molecules for the recognition of charged species is stimulated by their fundamental roles in numerous biological and environmental processes. Here, capitalizing on the inherent amphoteric nature of anisotropically polarized tellurium or iodine atoms, we demonstrate a proof of concept in charged guest recognition, where the same neutral host structure binds both cations or anions solely through its chalcogen or halogen donor atoms. Through extensive 1H nuclear magnetic resonance titration experiments and computational density functional theory studies, a library of chalcogen-bonding (ChB) and halogen-bonding (XB) mechanically interlocked [2]rotaxane molecules, including seminal examples of all-ChB and mixed ChB/XB [2]rotaxanes, are shown to function as either Lewis-acidic or Lewis-basic multidentate hosts for selective halide anion and metal cation binding. Notably, the exploitation of the inherent amphoteric character of an atom for the strategic purpose of either cation or anion recognition constitutes the inception of a previously unexplored area of supramolecular host–guest chemistry. The importance of charged species in numerous biological and environmental processes has stimulated the development of host molecules for their selective recognition. Now anisotropically polarized halogen- and chalcogen-bonding [2]rotaxanes are demonstrated to exhibit dual Lewis-acidic and Lewis-basic amphoteric properties for anion or cation recognition via the same donor atoms.},
  archive      = {J_NCHEM},
  author       = {Tse, Yuen Cheong and Docker, Andrew and Marques, Igor and Félix, Vítor and Beer, Paul D.},
  doi          = {10.1038/s41557-025-01742-x},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {373-381},
  shortjournal = {Nat. Chem.},
  title        = {Amphoteric chalcogen-bonding and halogen-bonding rotaxanes for anion or cation recognition},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Light-activated hypervalent iodine agents enable diverse
aliphatic c–h functionalization. <em>NCHEM</em>, <em>17</em>(3),
365–372. (<a href="https://doi.org/10.1038/s41557-025-01749-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The functionalization of aliphatic C–H bonds is a crucial step in the synthesis and transformation of complex molecules relevant to medicinal, agricultural and materials chemistry. As such, there is substantial interest in the development of general synthetic platforms that enable the efficient diversification of aliphatic C–H bonds. Here we report a hypervalent iodine reagent that releases a potent hydrogen atom abstractor for C–H activation under mild photochemical conditions. Using this reagent, we demonstrate selective (N-phenyltetrazole)thiolation of aliphatic C–H bonds for a broad scope of substrates. The synthetic utility of the thiolated products is showcased through various derivatizations. Simply by altering the radical trapping agent, our method can directly transform C–H bonds into diverse functionalities, including C–S, C–Cl, C–Br, C–I, C–O, C–N, C–C and C=C bonds. Aliphatic C–H functionalization is a valuable tool in organic synthesis. Now a hypervalent iodine reagent has been shown to release a potent hydrogen atom abstractor for C–H activation under mild photochemical conditions. This enables the transformation of C–H bonds into diverse functional groups with tunable control over the site selectivity.},
  archive      = {J_NCHEM},
  author       = {Lu, Zhipeng and Putziger, John and Lin, Song},
  doi          = {10.1038/s41557-025-01749-4},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {365-372},
  shortjournal = {Nat. Chem.},
  title        = {Light-activated hypervalent iodine agents enable diverse aliphatic C–H functionalization},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Cytoskeleton-functionalized synthetic cells with life-like
mechanical features and regulated membrane dynamicity. <em>NCHEM</em>,
<em>17</em>(3), 356–364. (<a
href="https://doi.org/10.1038/s41557-024-01697-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cytoskeleton is a crucial determinant of mammalian cell structure and function, providing mechanical resilience, supporting the cell membrane and orchestrating essential processes such as cell division and motility. Because of its fundamental role in living cells, developing a reconstituted or artificial cytoskeleton is of major interest. Here we present an approach to construct an artificial cytoskeleton that imparts mechanical support and regulates membrane dynamics. Our system involves amylose-based coacervates stabilized by a terpolymer membrane, with a cytoskeleton formed from polydiacetylene fibrils. The fibrils bundle due to interactions with the positively charged amylose derivative, forming micrometre-sized structures mimicking a cytoskeleton. Given the intricate interplay between cellular structure and function, the design and integration of this artificial cytoskeleton represent a crucial advancement, paving the way for the development of artificial cell platforms exhibiting enhanced life-like behaviour. The cytoskeleton is vital for living organisms. Now an artificial cytoskeleton within amylose-based coacervates stabilized by a terpolymer membrane has been developed using polydiacetylene fibrils to mimic natural cytoskeletal functions. The integration of this artificial cytoskeleton marks an important step in creating artificial cell platforms with enhanced life-like behaviour.},
  archive      = {J_NCHEM},
  author       = {Novosedlik, Sebastian and Reichel, Felix and van Veldhuisen, Thijs and Li, Yudong and Wu, Hanglong and Janssen, Henk and Guck, Jochen and van Hest, Jan},
  doi          = {10.1038/s41557-024-01697-5},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {356-364},
  shortjournal = {Nat. Chem.},
  title        = {Cytoskeleton-functionalized synthetic cells with life-like mechanical features and regulated membrane dynamicity},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Direct stereoselective c(sp3)–h alkylation of saturated
heterocycles using olefins. <em>NCHEM</em>, <em>17</em>(3), 344–355. (<a
href="https://doi.org/10.1038/s41557-025-01747-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite cross-coupling strategies that enable the functionalization of aromatic heterocycles, the enantioselective C(sp3)–H alkylation of readily available saturated hydrocarbons to construct C(sp3)–C(sp3) bonds remains a formidable challenge. Here we describe a nickel-catalysed enantioselective C(sp3)–H alkylation of saturated heterocycles using olefins, providing an efficient strategy for the stereoselective construction of C(sp3)–C(sp3) bonds. Using readily available and stable olefins and simple saturated nitrogen and oxygen heterocycles as prochiral nucleophiles, the coupling reactions proceed under mild conditions and exhibit broad scope and high functional group tolerance. Furthermore, the enantio- and diastereoselective C(sp3)–H alkylation of saturated hydrocarbons with alkenyl boronates has been achieved, enabling the synthesis of versatile alkyl boronates containing 1,2-adjacent C(sp3) stereocentres. Application of this approach to the late-stage modification of natural products and drugs, as well as to the enantioselective synthesis of a range of chiral building blocks and natural products, is demonstrated. The enantioselective C(sp3)–H alkylation of saturated hydrocarbons to construct C(sp3)–C(sp3) bonds is challenging. Now a nickel-catalysed enantioselective C(sp3)–H alkylation of saturated heterocycles using olefins has been developed. The enantio- and diastereoselective C(sp3)–H alkylation of saturated hydrocarbons with alkenyl boronates has also been achieved to give alkyl boronates containing 1,2-adjacent C(sp3) stereocentres.},
  archive      = {J_NCHEM},
  author       = {Zhou, Zhijun and Ke, Yang and Miao, Rui and Hu, Fen and Wang, Xiaoqin and Ping, Yuanyuan and Xu, Sheng and Kong, Wangqing},
  doi          = {10.1038/s41557-025-01747-6},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {344-355},
  shortjournal = {Nat. Chem.},
  title        = {Direct stereoselective C(sp3)–H alkylation of saturated heterocycles using olefins},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Isotopic labelling of water reveals the hydrogen transfer
route in electrochemical CO2 reduction. <em>NCHEM</em>, <em>17</em>(3),
334–343. (<a href="https://doi.org/10.1038/s41557-024-01721-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the hydrogenation pathway in electrochemical CO2 reduction is important for controlling product selectivity. The Eley–Rideal mechanism involving proton-coupled electron transfer directly from solvent water is often considered to be the primary hydrogen transfer route. However, in principle, hydrogenation can also occur via the Langmuir–Hinshelwood mechanism using surface-adsorbed *H. Here, by performing CO2 reduction with Cu in H2O–D2O mixtures, we present evidence that the Langmuir–Hinshelwood mechanism is probably the dominant hydrogenation route. From this, we estimate the extent to which each mechanism contributes towards the formation of six important CO2 reduction products. Through computational simulations, we find that the formation of C–H bonds and O–H bonds is governed by the Langmuir–Hinshelwood and Eley–Rideal mechanism, respectively. We also show that promoting the Eley–Rideal pathway could be crucial towards selective multicarbon product formation and suppressing hydrogen evolution. These findings introduce important considerations for the theoretical modelling of CO2 reduction pathways and electrocatalyst design. CO2 electroreduction to higher-value carbons can occur through adsorbed hydrogen or through proton-coupled electron transfer from water. Understanding the impact of each route on product selectivity is challenging. Now H/D isotopic labelling reveals the contribution of each mechanism towards product formation and shows that adsorbed hydrogen dominates the reaction.},
  archive      = {J_NCHEM},
  author       = {Zhang, Jiguang and Zhang, Chengyi and Wang, Meng and Mao, Yu and Wu, Bo and Yang, Qin and Wang, Bingqing and Mi, Ziyu and Zhang, Mingsheng and Ling, Ning and Leow, Wan Ru and Wang, Ziyun and Lum, Yanwei},
  doi          = {10.1038/s41557-024-01721-8},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {334-343},
  shortjournal = {Nat. Chem.},
  title        = {Isotopic labelling of water reveals the hydrogen transfer route in electrochemical CO2 reduction},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Interface flexibility controls the nucleation and growth of
supramolecular networks. <em>NCHEM</em>, <em>17</em>(3), 325–333. (<a
href="https://doi.org/10.1038/s41557-025-01741-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Supramolecular networks are abundantly present in nature and, like crystalline materials, often develop from an initial nucleation site, followed by growth based on directional interactions between components. Traditionally, the binding strength and directionality of interactions is thought to dictate nucleation and crystal growth, whereas structural flexibility favours defects. Usually, macromonomers present multiple binding sites with relative intramolecular flexibility, but the effects of such flexibility on regulating network formation have been given little attention. Here we introduce the concept of ‘interface flexibility’ and demonstrate its critical importance in the nucleation and growth of supramolecular networks. As a model system, we use trisymmetric DNA-based macromonomers, which organize into hexagonal networks through weak π–π interactions at their tips. The directional nature and low spatial tolerance of π–π interactions mean that small shifts in orientation have a large effect on effective valency. We show that too much interface flexibility disrupts network formation, regardless of affinity. Tuning the interface flexibility greatly expands the available design space for synthetic supramolecular materials. Supramolecular ordered networks are formed through directional interactions of uniform macromonomer building blocks. Now it has been shown that, rather than intermolecular affinity, the flexibility of the binding interface (‘interface flexibility’) dominates the mechanism of self-assembly. This study provides an intuitive understanding of the role of interface flexibility in supramolecular self-assembly.},
  archive      = {J_NCHEM},
  author       = {Caroprese, Vincenzo and Tekin, Cem and Cencen, Veronika and Mosayebi, Majid and Asmari, Navid and Liverpool, Tanniemola B. and Woolfson, Derek N. and Fantner, Georg E. and Bastings, Maartje M. C.},
  doi          = {10.1038/s41557-025-01741-y},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {325-333},
  shortjournal = {Nat. Chem.},
  title        = {Interface flexibility controls the nucleation and growth of supramolecular networks},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Progress and pitfalls in designing heterogeneous catalysts
with molecular precision. <em>NCHEM</em>, <em>17</em>(3), 318–324. (<a
href="https://doi.org/10.1038/s41557-024-01731-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atomically dispersed heterogeneous catalysts offer the opportunity to design active sites with molecular precision. A recent proliferation of synthetic strategies and advanced characterization tools has propelled broad interest in leveraging these catalysts in diverse areas of chemistry beyond thermal heterogeneous catalysis, including but not limited to organic synthesis, electrochemistry, photochemistry and environmental chemistry. This Perspective aims to arm researchers in this area with the methodological framework and fundamental principles to accurately assess the degree of uniformity of heterogeneous catalyst active sites, and to link the synthesis, structure and function of atomically dispersed catalysts to one another. A paedagogical discussion of catalyst structural dynamics and active-site-specific characterization is supplemented with recent case studies that illustrate these fundamentals. This analysis shows how progress can be guided by requiring that the characterization of active sites is connected to their kinetic behaviour in the form of turnover rates and by leveraging the thermokinetic factors that control active site structures and their dynamics. Atomically dispersed heterogeneous catalysts impact diverse areas of chemistry through the design of active sites with molecular precision, but this vision is not fully realized. This Perspective presents the fundamental principles and methodological framework to guide progress in the field.},
  archive      = {J_NCHEM},
  author       = {Bates, Jason S.},
  doi          = {10.1038/s41557-024-01731-6},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {318-324},
  shortjournal = {Nat. Chem.},
  title        = {Progress and pitfalls in designing heterogeneous catalysts with molecular precision},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rotaxanes with amphoteric σ-hole donors for cation or anion
recognition. <em>NCHEM</em>, <em>17</em>(3), 316–317. (<a
href="https://doi.org/10.1038/s41557-025-01743-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The amphoteric character of chalcogen bonding (ChB) and halogen bonding (XB) donor atoms for anion or cation recognition is demonstrated using interlocked host molecules. A family of neutral tri- and tetradentate ChB and/or XB [2]rotaxanes exhibit Lewis acidic halide anion and Lewis basic metal cation binding at the same chalcogen or halogen donor site.},
  archive      = {J_NCHEM},
  doi          = {10.1038/s41557-025-01743-w},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {316-317},
  shortjournal = {Nat. Chem.},
  title        = {Rotaxanes with amphoteric σ-hole donors for cation or anion recognition},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Photomediated aliphatic c–h functionalization.
<em>NCHEM</em>, <em>17</em>(3), 314–315. (<a
href="https://doi.org/10.1038/s41557-025-01755-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The C–H functionalization of inert alkanes has long been one of the most challenging reactions in organic synthesis. Now, the use of hypervalent iodine reagents has enabled the diverse functionalization of various aliphatic C–H bonds under blue light irradiation, achieving a high level of reactivity and selectivity.},
  archive      = {J_NCHEM},
  author       = {Nguyen, Tin V. T. and Waser, Jerome},
  doi          = {10.1038/s41557-025-01755-6},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {314-315},
  shortjournal = {Nat. Chem.},
  title        = {Photomediated aliphatic C–H functionalization},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A programmable biomimetic cytoskeleton. <em>NCHEM</em>,
<em>17</em>(3), 311–313. (<a
href="https://doi.org/10.1038/s41557-025-01757-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cytoskeleton provides internal organization, resistance and other essential functionalities to cells; but complex regulation makes natural components challenging to engineer for synthetic systems. Now, designer polymers are shown to assemble as a multifunctional, biomimetic cytoskeleton in artificial cells.},
  archive      = {J_NCHEM},
  author       = {Niederholtmeyer, Henrike},
  doi          = {10.1038/s41557-025-01757-4},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {311-313},
  shortjournal = {Nat. Chem.},
  title        = {A programmable biomimetic cytoskeleton},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stereoselective alkylation of saturated heterocycles.
<em>NCHEM</em>, <em>17</em>(3), 309–310. (<a
href="https://doi.org/10.1038/s41557-025-01754-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enantioselective C(sp3)–H alkylation of easily accessible saturated heterocycles is challenging. Now, a nickel-catalysed enantioselective C(sp3)–H alkylation of saturated nitrogen and oxygen heterocycles with olefins has been developed, offering an efficient strategy for the stereoselective formation of C(sp3)–C(sp3) bonds.},
  archive      = {J_NCHEM},
  author       = {Li, Yan and Lu, Xi},
  doi          = {10.1038/s41557-025-01754-7},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {309-310},
  shortjournal = {Nat. Chem.},
  title        = {Stereoselective alkylation of saturated heterocycles},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hydrogen transfer pathway controls selectivity in
electrocatalytic CO2 reduction. <em>NCHEM</em>, <em>17</em>(3), 307–308.
(<a href="https://doi.org/10.1038/s41557-025-01761-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The electrocatalytic reduction of CO2 involves electron/proton transfers, with hydrogenation of intermediates occurring via surface-bound hydrogen or hydrogen originating from water. Now, isotope-labelling studies have elucidated the relative contributions of both pathways on copper electrocatalysts, offering new perspectives on achieving selectivity control.},
  archive      = {J_NCHEM},
  author       = {Wan, Chengzhang and Ager, Joel W. and Huang, Yu},
  doi          = {10.1038/s41557-025-01761-8},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {307-308},
  shortjournal = {Nat. Chem.},
  title        = {Hydrogen transfer pathway controls selectivity in electrocatalytic CO2 reduction},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Flexible interaction patches lead to building blocks with
fluctuating valency. <em>NCHEM</em>, <em>17</em>(3), 305–306. (<a
href="https://doi.org/10.1038/s41557-025-01756-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patchy particles have directional interactions that enable self-assembly into materials with precisely tailored microstructures. The patches are usually rigid, but a study now shows that flexible patches can fluctuate between an on- and off-state, which dramatically affects the assembly process.},
  archive      = {J_NCHEM},
  author       = {Stevens, Tine C. M. and Moerman, Pepijn G.},
  doi          = {10.1038/s41557-025-01756-5},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {305-306},
  shortjournal = {Nat. Chem.},
  title        = {Flexible interaction patches lead to building blocks with fluctuating valency},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Preparing and publishing a paper. <em>NCHEM</em>,
<em>17</em>(3), 303–304. (<a
href="https://doi.org/10.1038/s41557-025-01759-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There are many steps to preparing a research article for publication, from generating the figures and writing the draft, to responding to reviewers. Shira Joudan explains how their group approaches this task, specifically during the preparation of the research group’s first paper.},
  archive      = {J_NCHEM},
  author       = {Joudan, Shira},
  doi          = {10.1038/s41557-025-01759-2},
  journal      = {Nature Chemistry},
  month        = {3},
  number       = {3},
  pages        = {303-304},
  shortjournal = {Nat. Chem.},
  title        = {Preparing and publishing a paper},
  volume       = {17},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="ng---38">NG - 38</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Functional innovation through new
genes as a general evolutionary process. <em>NG</em>, <em>57</em>(3),
764. (<a href="https://doi.org/10.1038/s41588-025-02116-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Xia, Shengqian and Chen, Jianhai and Arsala, Deanna and Emerson, J. J. and Long, Manyuan},
  doi          = {10.1038/s41588-025-02116-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {764},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Functional innovation through new genes as a general evolutionary process},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The NAT1–bHLH110–CER1/CER1L module
regulates heat stress tolerance in rice. <em>NG</em>, <em>57</em>(3),
763. (<a href="https://doi.org/10.1038/s41588-025-02106-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lu, Hai-Ping and Liu, Xue-Huan and Wang, Mei-Jing and Zhu, Qiao-Yun and Lyu, Yu-Shu and Xu, Jian-Hang and Liu, Jian-Xiang},
  doi          = {10.1038/s41588-025-02106-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The NAT1–bHLH110–CER1/CER1L module regulates heat stress tolerance in rice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Pangenome graphs and their applications
in biodiversity genomics. <em>NG</em>, <em>57</em>(3), 763. (<a
href="https://doi.org/10.1038/s41588-025-02132-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Secomandi, Simona and Gallo, Guido Roberto and Rossi, Riccardo and Rodríguez Fernandes, Carlos and Jarvis, Erich D. and Bonisoli-Alquati, Andrea and Gianfranceschi, Luca and Formenti, Giulio},
  doi          = {10.1038/s41588-025-02132-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Pangenome graphs and their applications in biodiversity genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Origin and de novo domestication of sweet orange.
<em>NG</em>, <em>57</em>(3), 754–762. (<a
href="https://doi.org/10.1038/s41588-025-02122-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sweet orange is cultivated worldwide but suffers from various devastating diseases because of its monogenetic background. The elucidation of the origin of a crop facilitates the domestication of new crops that may better cope with new challenges. Here we collected and sequenced 226 citrus accessions and assembled telomere-to-telomere phased diploid genomes of sweet orange and sour orange. On the basis of a high-resolution haplotype-resolved genome analysis, we inferred that sweet orange originated from a sour orange × mandarin cross and confirmed this model using artificial hybridization experiments. We identified defense-related metabolites that potently inhibited the growth of multiple industrially important pathogenic bacteria. We introduced diversity to sweet orange, which showed wide segregation in fruit flavor and disease resistance and produced canker-resistant sweet orange by selecting defense-related metabolites. Our findings elucidate the origin of sweet orange and de novo domesticated disease-resistant sweet oranges, illuminating a strategy for the rapid domestication of perennial crops. Genomic analyses of Citrus species including haplotype-resolved genomes of Citrus sinensis and Citrus aurantium highlight the origin of sweet orange and provide a strategy for de novo domestication of perennial crops.},
  archive      = {J_NG},
  author       = {Liu, Shengjun and Xu, Yuantao and Yang, Kun and Huang, Yue and Lu, Zhihao and Chen, Shulin and Gao, Xiang and Xiao, Gongao and Chen, Peng and Zeng, Xiuli and Wang, Lun and Zheng, Weikang and Liu, Zishuang and Liao, Guanglian and He, Fa and Liu, Junjie and Wan, Pengfei and Ding, Fang and Ye, Junli and Jiao, Wenbiao and Chai, Lijun and Pan, Zhiyong and Zhang, Fei and Lin, Zongcheng and Zan, Yanjun and Guo, Wenwu and Larkin, Robert M. and Xie, Zongzhou and Wang, Xia and Deng, Xiuxin and Xu, Qiang},
  doi          = {10.1038/s41588-025-02122-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {754-762},
  shortjournal = {Nat. Genet.},
  title        = {Origin and de novo domestication of sweet orange},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super pangenome of vitis empowers identification of downy
mildew resistance genes for grapevine improvement. <em>NG</em>,
<em>57</em>(3), 741–753. (<a
href="https://doi.org/10.1038/s41588-025-02111-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Grapevine (Vitis) is one of the oldest domesticated fruit crops with great cultural and economic importance. Here we assembled and annotated haplotype-resolved genomes of 72 global Vitis accessions including 25 wild and 47 cultivated grapevines, among which genomes for 60 grapevines are newly released. Haplotype-aware phylogenomics disentangled the mysterious hybridization history of grapevines, revealing the enormous genetic diversity of the Vitis genus. Pangenomic analysis reveals that European cultivars, more susceptible to the destructive disease downy mildew (DM), have a smaller repertoire of resistance genes in the NLR family encoding the TIR-NBARC-LRR domain. Through extensive structural variation (SV) characterization, phenotyping, DM-infection transcriptome profiling of 113 Vitis accessions, and SV–expression quantitative trait loci analysis, we have identified over 63 SVs and their relevant genes significantly associated with DM resistance, exemplified by a lysine histidine transporter, VvLHT8. This haplotype-resolved super pangenome of the Vitis genus will accelerate breeding and enrich our understanding of the evolution and biology of grapevines. Super pangenome of the Vitis genus assembled from haplotype-resolved genomes of 72 global Vitis accessions characterizes the landscape of structural variants (SVs) and identifies SV–expression quantitative trait loci for downy mildew resistance.},
  archive      = {J_NG},
  author       = {Guo, Li and Wang, Xiangfeng and Ayhan, Dilay Hazal and Rhaman, Mohammad Saidur and Yan, Ming and Jiang, Jianfu and Wang, Dongyue and Zheng, Wei and Mei, Junjie and Ji, Wei and Jiao, Jian and Chen, Shaoying and Sun, Jie and Yi, Shu and Meng, Dian and Wang, Jing and Bhuiyan, Mohammad Nasim and Qin, Guochen and Guo, Linling and Yang, Qingxian and Zhang, Xuenan and Sun, Haisheng and Liu, Chonghuai and Deng, Xing Wang and Ye, Wenxiu},
  doi          = {10.1038/s41588-025-02111-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {741-753},
  shortjournal = {Nat. Genet.},
  title        = {Super pangenome of vitis empowers identification of downy mildew resistance genes for grapevine improvement},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The marchantia polymorpha pangenome reveals ancient
mechanisms of plant adaptation to the environment. <em>NG</em>,
<em>57</em>(3), 729–740. (<a
href="https://doi.org/10.1038/s41588-024-02071-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant adaptation to terrestrial life started 450 million years ago and has played a major role in the evolution of life on Earth. The genetic mechanisms allowing this adaptation to a diversity of terrestrial constraints have been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the model bryophyte Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, a genome–environment association study and a pangenome. We identified adaptive features, such as peroxidases or nucleotide-binding and leucine-rich repeats (NLRs), also observed in flowering plants, likely inherited from the first land plants. The M. polymorpha pangenome also harbors lineage-specific accessory genes absent from seed plants. We conclude that different land plant lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage-specific polymorphisms and gene family evolution. Pangenome analyses of 133 wild accessions of the model bryophyte Marchantia polymorpha identify adaptive features and provide insights into the mechanisms of plant adaptation to the terrestrial environment.},
  archive      = {J_NG},
  author       = {Beaulieu, Chloé and Libourel, Cyril and Mbadinga Zamar, Duchesse Lacourt and El Mahboubi, Karima and Hoey, David J. and Greiff, George R. L. and Keller, Jean and Girou, Camille and San Clemente, Helene and Diop, Issa and Amblard, Emilie and Castel, Baptiste and Théron, Anthony and Cauet, Stéphane and Rodde, Nathalie and Zachgo, Sabine and Halpape, Wiebke and Meierhenrich, Anja and Laker, Bianca and Bräutigam, Andrea and Szovenyi, Peter and Cheng, Shifeng and Tanizawa, Yasuhiro and Aziz, Simon and Leebens-Mack, James H. and Schmutz, Jeremy and Webber, Jenell and Grimwood, Jane and Jacquet, Christophe and Dunand, Christophe and Nelson, Jessica M. and Roux, Fabrice and Philippe, Hervé and Schornack, Sebastian and Bonhomme, Maxime and Delaux, Pierre-Marc},
  doi          = {10.1038/s41588-024-02071-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {729-740},
  shortjournal = {Nat. Genet.},
  title        = {The marchantia polymorpha pangenome reveals ancient mechanisms of plant adaptation to the environment},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Functional analysis of cancer-associated germline risk
variants. <em>NG</em>, <em>57</em>(3), 718–728. (<a
href="https://doi.org/10.1038/s41588-024-02070-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nucleotide variants (SNVs) in regulatory DNA are linked to inherited cancer risk. Massively parallel reporter assays of 4,041 SNVs linked to 13 neoplasms comprising &amp;gt;90% of human malignancies were performed in pertinent primary human cell types and then integrated with matching chromatin accessibility, DNA looping and expression quantitative trait loci data to nominate 380 potentially regulatory SNVs and their putative target genes. The latter highlighted specific protein networks in lifetime cancer risk, including mitochondrial translation, DNA damage repair and Rho GTPase activity. A CRISPR knockout screen demonstrated that a subset of germline putative risk genes also enables the growth of established cancers. Editing one SNV, rs10411210 , showed that its risk allele increases rhophilin RHPN2 expression and stimulus-responsive RhoA activation, indicating that individual SNVs may upregulate cancer-linked pathways. These functional data are a resource for variant prioritization efforts and further interrogation of the mechanisms underlying inherited risk for cancer. Analysis of 4,041 single-nucleotide variants (SNVs) linked to 13 cancers performed in primary human cell types identifies 380 potentially regulatory SNVs and their putative target genes. Editing one SNV, rs10411210 , revealed that the risk allele increases RHPN2 expression and stimulus-responsive RhoA activation.},
  archive      = {J_NG},
  author       = {Kellman, Laura N. and Neela, Poornima H. and Srinivasan, Suhas and Siprashvili, Zurab and Shanderson, Ronald L. and Hong, Audrey W. and Rao, Deepti and Porter, Douglas F. and Reynolds, David L. and Meyers, Robin M. and Guo, Margaret G. and Yang, Xue and Zhao, Yang and Wozniak, Glenn G. and Donohue, Laura K. H. and Shenoy, Rajani and Ko, Lisa A. and Nguyen, Duy T. and Mondal, Smarajit and Garcia, Omar S. and Elcavage, Lara E. and Elfaki, Ibtihal and Abell, Nathan S. and Tao, Shiying and Lopez, Christopher M. and Montgomery, Stephen B. and Khavari, Paul A.},
  doi          = {10.1038/s41588-024-02070-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {718-728},
  shortjournal = {Nat. Genet.},
  title        = {Functional analysis of cancer-associated germline risk variants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying cell divisions along evolutionary lineages in
cancer. <em>NG</em>, <em>57</em>(3), 706–717. (<a
href="https://doi.org/10.1038/s41588-025-02078-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell division drives somatic evolution but is challenging to quantify. We developed a framework to count cell divisions with DNA replication-related mutations in polyguanine homopolymers. Analyzing 505 samples from 37 patients, we studied the milestones of colorectal cancer evolution. Primary tumors diversify at ~250 divisions from the founder cell, while distant metastasis divergence occurs significantly later, at ~500 divisions. Notably, distant but not lymph node metastases originate from primary tumor regions that have undergone surplus divisions, tying subclonal expansion to metastatic capacity. Then, we analyzed a cohort of 73 multifocal lung cancers and showed that the cell division burden of the tumors’ common ancestor distinguishes independent primary tumors from intrapulmonary metastases and correlates with patient survival. In lung cancer too, metastatic capacity is tied to more extensive proliferation. The cell division history of human cancers is easily accessible using our simple framework and contains valuable biological and clinical information. This work presents a framework for estimating cell division numbers using DNA replication-associated polyguanine tract mutations, with applications for understanding tumor natural histories and origins.},
  archive      = {J_NG},
  author       = {Blohmer, Martin and Cheek, David M. and Hung, Wei-Ting and Kessler, Maria and Chatzidimitriou, Foivos and Wang, Jiahe and Hung, William and Lee, I-Hsiu and Gorelick, Alexander N. and Wassenaar, Emma CE and Yang, Ching-Yeuh and Yeh, Yi-Chen and Ho, Hsiang-Ling and Speiser, Dorothee and Karsten, Maria M. and Lanuti, Michael and Pai, Sara I. and Kranenburg, Onno and Lennerz, Jochen K. and Chou, Teh-Ying and Kloor, Matthias and Naxerova, Kamila},
  doi          = {10.1038/s41588-025-02078-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {706-717},
  shortjournal = {Nat. Genet.},
  title        = {Quantifying cell divisions along evolutionary lineages in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ImmuneLENS characterizes systemic immune dysregulation in
aging and cancer. <em>NG</em>, <em>57</em>(3), 694–705. (<a
href="https://doi.org/10.1038/s41588-025-02086-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recognition and elimination of pathogens and cancer cells depend on the adaptive immune system. Thus, accurate quantification of immune subsets is vital for precision medicine. We present immune lymphocyte estimation from nucleotide sequencing (ImmuneLENS), which estimates T cell and B cell fractions, class switching and clonotype diversity from whole-genome sequencing data at depths as low as 5× coverage. By applying ImmuneLENS to the 100,000 Genomes Project, we identify genes enriched with somatic mutations in T cell-rich tumors, significant sex-based differences in circulating T cell fraction and demonstrated that the circulating T cell fraction in patients with cancer is significantly lower than in healthy individuals. Low circulating B cell fraction was linked to increased cancer incidence. Finally, circulating T cell abundance was more prognostic of 5-year cancer survival than infiltrating T cells. Immune lymphocyte estimation from nucleotide sequencing (ImmuneLENS) infers B cell and T cell fractions from whole-genome sequencing data. Applied to the 100,000 Genomes Project datasets, circulating T cell fraction provides sex-dependent and prognostic insights in patients.},
  archive      = {J_NG},
  author       = {Bentham, Robert and Jones, Thomas P. and Black, James R. M. and Martinez-Ruiz, Carlos and Dietzen, Michelle and Litovchenko, Maria and Thol, Kerstin and Watkins, Thomas B. K. and Bailey, Chris and Pich, Oriol and Zhang, Zhihui and Van Loo, Peter and Swanton, Charles and McGranahan, Nicholas},
  doi          = {10.1038/s41588-025-02086-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {694-705},
  shortjournal = {Nat. Genet.},
  title        = {ImmuneLENS characterizes systemic immune dysregulation in aging and cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Alterations in PD-l1 succinylation shape anti-tumor immune
responses in melanoma. <em>NG</em>, <em>57</em>(3), 680–693. (<a
href="https://doi.org/10.1038/s41588-025-02077-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumors undergo metabolic reprogramming to meet the energetic, synthetic and redox demands essential for malignancy, often characterized by increased glycolysis and lactate production. However, the role of mitochondrial metabolism in tumor immunity remains unclear. The present study integrates spatial transcriptomics, bulk transcriptomics and proteomics, revealing a strong link between the metabolite succinyl-CoA and tumor immunity as well as the efficacy of anti-programmed cell death protein-1 (PD-1) therapy in patients with melanoma. Elevated succinyl-CoA levels, through α-ketoglutarate or succinate supplementation, enhanced T cell-mediated tumor elimination, both in vitro and in vivo. Mechanistically, succinylation of the ligand of PD-1 (PD-L1) at lysine 129 led to its degradation. Increased carnitine palmitoyltransferase 1A (CPT1A), identified as a succinyltransferase for PD-L1, boosted anti-tumor activity. Preclinically, bezafibrate, a hyperlipidemia drug, upregulated CPT1A and synergized with CTLA-4 monoclonal antibody to inhibit tumor growth. Clinically, higher PD-L1 and lower CPT1A levels in tumors correlated with better anti-PD-1 therapy responses, suggesting potential biomarkers for prediction of treatment efficacy. Succinylation of PD-L1 by carnitine palmitoyltransferase 1A (CPT1A) in melanoma leads to its degradation and enhanced T cell-dependent killing in vitro. Increasing CPT1A levels synergizes with anti-CTLA-4 treatment to suppress tumor growth in a mouse melanoma model.},
  archive      = {J_NG},
  author       = {Liang, Long and Kuang, Xinwei and He, Yi and Zhu, Lin and Lau, Poyee and Li, Xin and Luo, Dingan and Gong, Lan and Zhou, Wenbin and Zhang, Fanglin and Liang, Xiaowei and Li, Zhuofeng and Hu, Bin and Liu, Dandan and Ding, Tao and Li, Hui and Zhao, Shuang and Su, Juan and Hung, Mien-Chie and Liu, Jing and Liu, Hong and Chen, Xiang},
  doi          = {10.1038/s41588-025-02077-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {680-693},
  shortjournal = {Nat. Genet.},
  title        = {Alterations in PD-l1 succinylation shape anti-tumor immune responses in melanoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Active repression of cell fate plasticity by PROX1
safeguards hepatocyte identity and prevents liver tumorigenesis.
<em>NG</em>, <em>57</em>(3), 668–679. (<a
href="https://doi.org/10.1038/s41588-025-02081-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell fate plasticity enables development, yet unlocked plasticity is a cancer hallmark. While transcription master regulators induce lineage-specific genes to restrict plasticity, it remains unclear whether plasticity is actively suppressed by lineage-specific repressors. Here we computationally predict so-called safeguard repressors for 18 cell types that block phenotypic plasticity lifelong. We validated hepatocyte-specific candidates using reprogramming, revealing that prospero homeobox protein 1 (PROX1) enhanced hepatocyte identity by direct repression of alternative fate master regulators. In mice, Prox1 was required for efficient hepatocyte regeneration after injury and was sufficient to prevent liver tumorigenesis. In line with patient data, Prox1 depletion caused hepatocyte fate loss in vivo and enabled the transition of hepatocellular carcinoma to cholangiocarcinoma. Conversely, overexpression promoted cholangiocarcinoma to hepatocellular carcinoma transdifferentiation. Our findings provide evidence for PROX1 as a hepatocyte-specific safeguard and support a model where cell-type-specific repressors actively suppress plasticity throughout life to safeguard lineage identity and thus prevent disease. This study identifies candidate safeguard repressor transcription factors that repress alternate lineages in mature cell types and provides functional evidence that Prox1 performs such a function in hepatocytes during reprogramming, regeneration and in cancer.},
  archive      = {J_NG},
  author       = {Lim, Bryce and Kamal, Aryan and Gomez Ramos, Borja and Adrian Segarra, Juan M. and Ibarra, Ignacio L. and Dignas, Lennart and Kindinger, Tim and Volz, Kai and Rahbari, Mohammad and Rahbari, Nuh and Poisel, Eric and Kafetzopoulou, Kanela and Böse, Lio and Breinig, Marco and Heide, Danijela and Gallage, Suchira and Barragan Avila, Jose E. and Wiethoff, Hendrik and Berest, Ivan and Schnabellehner, Sarah and Schneider, Martin and Becker, Jonas and Helm, Dominic and Grimm, Dirk and Mäkinen, Taija and Tschaharganeh, Darjus F. and Heikenwalder, Mathias and Zaugg, Judith B. and Mall, Moritz},
  doi          = {10.1038/s41588-025-02081-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {668-679},
  shortjournal = {Nat. Genet.},
  title        = {Active repression of cell fate plasticity by PROX1 safeguards hepatocyte identity and prevents liver tumorigenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis of multi-condition single-cell data with latent
embedding multivariate regression. <em>NG</em>, <em>57</em>(3), 659–667.
(<a href="https://doi.org/10.1038/s41588-024-01996-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying gene expression differences in heterogeneous tissues across conditions is a fundamental biological task, enabled by multi-condition single-cell RNA sequencing (RNA-seq). Current data analysis approaches divide the constituent cells into clusters meant to represent cell types, but such discrete categorization tends to be an unsatisfactory model of the underlying biology. Here, we introduce latent embedding multivariate regression (LEMUR), a model that operates without, or before, commitment to discrete categorization. LEMUR (1) integrates data from different conditions, (2) predicts each cell’s gene expression changes as a function of the conditions and its position in latent space and (3) for each gene, identifies a compact neighborhood of cells with consistent differential expression. We apply LEMUR to cancer, zebrafish development and spatial gradients in Alzheimer’s disease, demonstrating its broad applicability. Latent embedding multivariate regression models multi-condition single-cell RNA-seq using a continuous latent space, enabling data integration, per-cell gene expression prediction and clustering-free differential expression analysis.},
  archive      = {J_NG},
  author       = {Ahlmann-Eltze, Constantin and Huber, Wolfgang},
  doi          = {10.1038/s41588-024-01996-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {659-667},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of multi-condition single-cell data with latent embedding multivariate regression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial transcriptomics identifies molecular niche
dysregulation associated with distal lung remodeling in pulmonary
fibrosis. <em>NG</em>, <em>57</em>(3), 647–658. (<a
href="https://doi.org/10.1038/s41588-025-02080-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale changes in the structure and cellular makeup of the distal lung are a hallmark of pulmonary fibrosis (PF), but the spatial contexts that contribute to disease pathogenesis have remained uncertain. Using image-based spatial transcriptomics, we analyzed the gene expression of 1.6 million cells from 35 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell types, established the cellular and molecular basis of classical PF histopathologic features and identified a diversity of distinct molecularly defined spatial niches in control and PF lungs. Using machine learning and trajectory analysis to segment and rank airspaces on a gradient of remodeling severity, we identified compositional and molecular changes associated with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially resolved view of PF and establish methods that could be applied to other spatial transcriptomic studies. Xenium spatial transcriptomic profiling of pulmonary fibrosis characterizes cell composition dynamics and histopathological features associated with the disease.},
  archive      = {J_NG},
  author       = {Vannan, Annika and Lyu, Ruqian and Williams, Arianna L. and Negretti, Nicholas M. and Mee, Evan D. and Hirsh, Joseph and Hirsh, Samuel and Hadad, Niran and Nichols, David S. and Calvi, Carla L. and Taylor, Chase J. and Polosukhin, Vasiliy. V. and Serezani, Ana P. M. and McCall, A. Scott and Gokey, Jason J. and Shim, Heejung and Ware, Lorraine B. and Bacchetta, Matthew J. and Shaver, Ciara M. and Blackwell, Timothy S. and Walia, Rajat and Sucre, Jennifer M. S. and Kropski, Jonathan A. and McCarthy, Davis J. and Banovich, Nicholas E.},
  doi          = {10.1038/s41588-025-02080-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {647-658},
  shortjournal = {Nat. Genet.},
  title        = {Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Plasma proteome variation and its genetic determinants in
children and adolescents. <em>NG</em>, <em>57</em>(3), 635–646. (<a
href="https://doi.org/10.1038/s41588-025-02089-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our current understanding of the determinants of plasma proteome variation during pediatric development remains incomplete. Here, we show that genetic variants, age, sex and body mass index significantly influence this variation. Using a streamlined and highly quantitative mass spectrometry-based proteomics workflow, we analyzed plasma from 2,147 children and adolescents, identifying 1,216 proteins after quality control. Notably, the levels of 70% of these were associated with at least one of the aforementioned factors, with protein levels also being predictive. Quantitative trait loci (QTLs) regulated at least one-third of the proteins; between a few percent and up to 30-fold. Together with excellent replication in an additional 1,000 children and 558 adults, this reveals substantial genetic effects on plasma protein levels, persisting from childhood into adulthood. Through Mendelian randomization and colocalization analyses, we identified 41 causal genes for 33 cardiometabolic traits, emphasizing the value of protein QTLs in drug target identification and disease understanding. This mass spectrometry-based proteomic study profiles the plasma proteome in 2,147 children and adolescents and reveals its association with age, sex, puberty, body mass index and genetics.},
  archive      = {J_NG},
  author       = {Niu, Lili and Stinson, Sara Elizabeth and Holm, Louise Aas and Lund, Morten Asp Vonsild and Fonvig, Cilius Esmann and Cobuccio, Leonardo and Meisner, Jonas and Juel, Helene Bæk and Fadista, Joao and Thiele, Maja and Krag, Aleksander and Holm, Jens-Christian and Rasmussen, Simon and Hansen, Torben and Mann, Matthias},
  doi          = {10.1038/s41588-025-02089-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {635-646},
  shortjournal = {Nat. Genet.},
  title        = {Plasma proteome variation and its genetic determinants in children and adolescents},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Whole-genome sequencing analysis identifies rare,
large-effect noncoding variants and regulatory regions associated with
circulating protein levels. <em>NG</em>, <em>57</em>(3), 626–634. (<a
href="https://doi.org/10.1038/s41588-025-02095-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The contribution of rare noncoding genetic variation to common phenotypes is largely unknown, as a result of a historical lack of population-scale whole-genome sequencing data and the difficulty of categorizing noncoding variants into functionally similar groups. To begin addressing these challenges, we performed a cis association analysis using whole-genome sequencing data, consisting of 1.1 billion variants, 123 million noncoding aggregate-based tests and 2,907 circulating protein levels in ~50,000 UK Biobank participants. We identified 604 independent rare noncoding single-variant associations with circulating protein levels. Unlike protein-coding variation, rare noncoding genetic variation was almost as likely to increase or decrease protein levels. Rare noncoding aggregate testing identified 357 conditionally independent associated regions. Of these, 74 (21%) were not detectable by single-variant testing alone. Our findings have important implications for the identification, and role, of rare noncoding genetic variation associated with common human phenotypes, including the importance of testing aggregates of noncoding variants. Rare variant association analysis of plasma proteins using whole-genome sequencing data in 54,306 individuals in the UK Biobank demonstrates that combining both single-variant and aggregate testing will be needed for noncoding variant analysis.},
  archive      = {J_NG},
  author       = {Hawkes, Gareth and Chundru, Kartik and Jackson, Leigh and Patel, Kashyap A. and Murray, Anna and Wood, Andrew R. and Wright, Caroline F. and Weedon, Michael N. and Frayling, Timothy M. and Beaumont, Robin N.},
  doi          = {10.1038/s41588-025-02095-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {626-634},
  shortjournal = {Nat. Genet.},
  title        = {Whole-genome sequencing analysis identifies rare, large-effect noncoding variants and regulatory regions associated with circulating protein levels},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The contribution of genetic determinants of blood gene
expression and splicing to molecular phenotypes and health outcomes.
<em>NG</em>, <em>57</em>(3), 616–625. (<a
href="https://doi.org/10.1038/s41588-025-02096-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biological mechanisms through which most nonprotein-coding genetic variants affect disease risk are unknown. To investigate gene-regulatory mechanisms, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA sequencing in 4,732 participants and integrated protein, metabolite and lipid data from the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Colocalization analyses revealed 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access resource on the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans ( https://IntervalRNA.org.uk ). Integrative analyses of whole-blood gene expression and splicing QTLs with protein, metabolite and lipid QTLs from the same individuals shed light on the shared genetic architecture of molecular traits and health outcomes.},
  archive      = {J_NG},
  author       = {Tokolyi, Alex and Persyn, Elodie and Nath, Artika P. and Burnham, Katie L. and Marten, Jonathan and Vanderstichele, Thomas and Tardaguila, Manuel and Stacey, David and Farr, Ben and Iyer, Vivek and Jiang, Xilin and Lambert, Samuel A. and Noell, Guillaume and Quail, Michael A. and Rajan, Diana and Ritchie, Scott C. and Sun, Benjamin B. and Thurston, Scott A. J. and Xu, Yu and Whelan, Christopher D. and Runz, Heiko and Petrovski, Slavé and Gaffney, Daniel J. and Roberts, David J. and Di Angelantonio, Emanuele and Peters, James E. and Soranzo, Nicole and Danesh, John and Butterworth, Adam S. and Inouye, Michael and Davenport, Emma E. and Paul, Dirk S.},
  doi          = {10.1038/s41588-025-02096-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {616-625},
  shortjournal = {Nat. Genet.},
  title        = {The contribution of genetic determinants of blood gene expression and splicing to molecular phenotypes and health outcomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-read RNA sequencing atlas of human microglia isoforms
elucidates disease-associated genetic regulation of splicing.
<em>NG</em>, <em>57</em>(3), 604–615. (<a
href="https://doi.org/10.1038/s41588-025-02099-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microglia, the innate immune cells of the central nervous system, have been genetically implicated in multiple neurodegenerative diseases. Mapping the genetics of gene expression in human microglia has identified several loci associated with disease-associated genetic variants in microglia-specific regulatory elements. However, identifying genetic effects on splicing is challenging because of the use of short sequencing reads. Here, we present the isoform-centric microglia genomic atlas (isoMiGA), which leverages long-read RNA sequencing to identify 35,879 novel microglia isoforms. We show that these isoforms are involved in stimulation response and brain region specificity. We then quantified the expression of both known and novel isoforms in a multi-ancestry meta-analysis of 555 human microglia short-read RNA sequencing samples from 391 donors, and found associations with genetic risk loci in Alzheimer’s and Parkinson’s disease. We nominate several loci that may act through complex changes in isoform and splice-site usage. This isoform-centric microglia genomic atlas includes 35,879 novel human microglia isoforms identified by long-read RNA sequencing. A multi-ancestry quantitative trait locus meta-analysis of known and novel isoforms in 555 samples from 391 donors finds associations with genetic risk loci in Alzheimer’s and Parkinson’s diseases.},
  archive      = {J_NG},
  author       = {Humphrey, Jack and Brophy, Erica and Kosoy, Roman and Zeng, Biao and Coccia, Elena and Mattei, Daniele and Ravi, Ashvin and Naito, Tatsuhiko and Efthymiou, Anastasia G. and Navarro, Elisa and De Sanctis, Claudia and Flores-Almazan, Victoria and Muller, Benjamin Z. and Snijders, Gijsje J. L. J. and Allan, Amanda and Münch, Alexandra and Kitata, Reta Birhanu and Kleopoulos, Steven P. and Argyriou, Stathis and Malakates, Periklis and Psychogyiou, Konstantina and Shao, Zhiping and Francoeur, Nancy and Tsai, Chia-Feng and Gritsenko, Marina A. and Monroe, Matthew E. and Paurus, Vanessa L. and Weitz, Karl K. and Shi, Tujin and Sebra, Robert and Liu, Tao and de Witte, Lot D. and Goate, Alison M. and Bennett, David A. and Haroutunian, Vahram and Hoffman, Gabriel E. and Fullard, John F. and Roussos, Panos and Raj, Towfique},
  doi          = {10.1038/s41588-025-02099-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {604-615},
  shortjournal = {Nat. Genet.},
  title        = {Long-read RNA sequencing atlas of human microglia isoforms elucidates disease-associated genetic regulation of splicing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiomic single-cell profiling identifies critical
regulators of postnatal brain. <em>NG</em>, <em>57</em>(3), 591–603. (<a
href="https://doi.org/10.1038/s41588-025-02083-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human brain development spans from embryogenesis to adulthood, with dynamic gene expression controlled by cell-type-specific cis-regulatory element activity and three-dimensional genome organization. To advance our understanding of postnatal brain development, we simultaneously profiled gene expression and chromatin accessibility in 101,924 single nuclei from four brain regions across ten donors, covering five key postnatal stages from infancy to late adulthood. Using this dataset and chromosome conformation capture data, we constructed enhancer-based gene regulatory networks to identify cell-type-specific regulators of brain development and interpret genome-wide association study loci for ten main brain disorders. Our analysis connected 2,318 cell-specific loci to 1,149 unique genes, representing 41% of loci linked to the investigated traits, and highlighted 55 genes influencing several disease phenotypes. Pseudotime analysis revealed distinct stages of postnatal oligodendrogenesis and their regulatory programs. These findings provide a comprehensive dataset of cell-type-specific gene regulation at critical timepoints in postnatal brain development. Simultaneous profiling of RNA and chromatin accessibility in single nuclei isolated from human postnatal brain regions from infancy to late adulthood identifies key cellular regulators and nominates target genes and mechanisms for brain-related diseases and disorders.},
  archive      = {J_NG},
  author       = {Clarence, Tereza and Bendl, Jaroslav and Cao, Xuan and Wang, Xinyi and Zheng, Shiwei and Hoffman, Gabriel E. and Kozlenkov, Alexey and Hong, Aram and Iskhakova, Marina and Jaiswal, Manoj K. and Murphy, Sarah and Yu, Alexander and Haroutunian, Vahram and Dracheva, Stella and Akbarian, Schahram and Fullard, John F. and Yuan, Guo-Cheng and Lee, Donghoon and Roussos, Panos},
  doi          = {10.1038/s41588-025-02083-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {591-603},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic single-cell profiling identifies critical regulators of postnatal brain},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparison of the multivariate genetic architecture of eight
major psychiatric disorders across sex. <em>NG</em>, <em>57</em>(3),
583–590. (<a href="https://doi.org/10.1038/s41588-025-02093-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Differences in the patterning of genetic sharing between groups of individuals may arise from biological pathways, social mechanisms, phenotyping and ascertainment. We expand genomic structural equation modeling to allow for testing genomic structural invariance (GSI), that is, the formal comparison of multivariate genetic architecture across groups. We apply GSI to compare the autosomal multivariate genetic architecture of eight psychiatric disorders spanning three factors (psychotic, neurodevelopmental and internalizing) between cisgender males and females. We find that the genetic factor structure is largely similar across sex, permitting meaningful comparisons of associations at the level of the factors. However, in females, problematic alcohol use and posttraumatic stress disorder loaded more strongly on the internalizing factor, while the neurodevelopmental disorder factor exhibited weaker genetic correlations with the other factors. Four phenotypes (educational attainment, insomnia, smoking and deprivation) showed significant, albeit small, sex-differentiated associations with the psychotic factor. As genome-wide association study samples grow and diversify, GSI will become increasingly valuable for comparing multivariate genetic architecture across groups. Genomic structural invariance, an extension of genomic structural equation modeling, identifies similarities and differences between males and females in the genetic architectures of eight psychiatric disorders spanning three broad factors.},
  archive      = {J_NG},
  author       = {Schwaba, Ted and Mallard, Travis T. and Maihofer, Adam X. and Rhemtulla, Mijke and Lee, Phil H. and Smoller, Jordan W. and Davis, Lea K. and Nivard, Michel G. and Grotzinger, Andrew D. and Tucker-Drob, Elliot M.},
  doi          = {10.1038/s41588-025-02093-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {583-590},
  shortjournal = {Nat. Genet.},
  title        = {Comparison of the multivariate genetic architecture of eight major psychiatric disorders across sex},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparative analysis of the mexico city prospective study
and the UK biobank identifies ancestry-specific effects on clonal
hematopoiesis. <em>NG</em>, <em>57</em>(3), 572–582. (<a
href="https://doi.org/10.1038/s41588-025-02085-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of genetic ancestry on the development of clonal hematopoiesis (CH) remains largely unexplored. Here, we compared CH in 136,401 participants from the Mexico City Prospective Study (MCPS) to 416,118 individuals from the UK Biobank (UKB) and observed CH to be significantly less common in MCPS compared to UKB (adjusted odds ratio = 0.59, 95% confidence interval (CI) = [0.57, 0.61], P = 7.31 × 10−185). Among MCPS participants, CH frequency was positively correlated with the percentage of European ancestry (adjusted beta = 0.84, 95% CI = [0.66, 1.03], P = 7.35 × 10−19). Genome-wide and exome-wide association analyses in MCPS identified ancestry-specific variants in the TCL1B locus with opposing effects on DNMT3A-CH versus non-DNMT3A-CH. Meta-analysis of MCPS and UKB identified five novel loci associated with CH, including polymorphisms at PARP11/CCND2, MEIS1 and MYCN. Our CH study, the largest in a non-European population to date, demonstrates the power of cross-ancestry comparisons to derive novel insights into CH pathogenesis. This paper compares the frequency and genetic basis of clonal hematopoiesis in 136,401 admixed American participants from the Mexico City Prospective Cohort with 416,118 largely European ancestry individuals from the UK Biobank cohort.},
  archive      = {J_NG},
  author       = {Wen, Sean and Kuri-Morales, Pablo and Hu, Fengyuan and Nag, Abhishek and Tachmazidou, Ioanna and Deevi, Sri V. V. and Taiy, Haeyam and Smith, Katherine R. and Loesch, Douglas P. and Burren, Oliver S. and Dhindsa, Ryan S. and Wasilewski, Sebastian and Alegre-Díaz, Jesus and Berumen, Jaime and Emberson, Jonathan and Torres, Jason M. and Collins, Rory and Carss, Keren and Wang, Quanli and Petrovski, Slavé and Tapia-Conyer, Roberto and Fabre, Margarete A. and Harper, Andrew R. and Vassiliou, George S. and Mitchell, Jonathan},
  doi          = {10.1038/s41588-025-02085-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {572-582},
  shortjournal = {Nat. Genet.},
  title        = {Comparative analysis of the mexico city prospective study and the UK biobank identifies ancestry-specific effects on clonal hematopoiesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evaluation of polygenic scores for hypertrophic
cardiomyopathy in the general population and across clinical settings.
<em>NG</em>, <em>57</em>(3), 563–571. (<a
href="https://doi.org/10.1038/s41588-025-02094-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertrophic cardiomyopathy (HCM) is an important cause of morbidity and mortality, with pathogenic variants found in about a third of cases. Large-scale genome-wide association studies (GWAS) demonstrate that common genetic variation contributes to HCM risk. Here we derive polygenic scores (PGS) from HCM GWAS and genetically correlated traits and test their performance in the UK Biobank, 100,000 Genomes Project, and clinical cohorts. We show that higher PGS significantly increases the risk of HCM in the general population, particularly among pathogenic variant carriers, where HCM penetrance differs 10-fold between those in the highest and lowest PGS quintiles. Among relatives of HCM probands, PGS stratifies risks of developing HCM and adverse outcomes. Finally, among HCM cases, PGS strongly predicts the risk of adverse outcomes and death. These findings support the broad utility of PGS across clinical settings, enabling tailored screening and surveillance and stratification of risk of adverse outcomes. Polygenic scores derived from genome-wide association analyses of hypertrophic cardiomyopathy and genetically correlated left ventricular traits effectively stratify risk of disease and adverse outcomes in the general population and across clinical cohorts.},
  archive      = {J_NG},
  author       = {Zheng, Sean L. and Jurgens, Sean J. and McGurk, Kathryn A. and Xu, Xiao and Grace, Chris and Theotokis, Pantazis I. and Buchan, Rachel J. and Francis, Catherine and de Marvao, Antonio and Curran, Lara and Bai, Wenjia and Pua, Chee Jian and Tang, Hak Chiaw and Jorda, Paloma and van Slegtenhorst, Marjon A. and Verhagen, Judith M. A. and Harper, Andrew R. and Ormondroyd, Elizabeth and Chin, Calvin W. L. and Pantazis, Antonis and Baksi, John and Halliday, Brian P. and Matthews, Paul and Pinto, Yigal M. and Walsh, Roddy and Amin, Ahmad S. and Wilde, Arthur A. M. and Cook, Stuart A. and Prasad, Sanjay K. and Barton, Paul J. R. and O’Regan, Declan P. and Lumbers, R. T. and Goel, Anuj and Tadros, Rafik and Michels, Michelle and Watkins, Hugh and Bezzina, Connie R. and Ware, James S.},
  doi          = {10.1038/s41588-025-02094-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {563-571},
  shortjournal = {Nat. Genet.},
  title        = {Evaluation of polygenic scores for hypertrophic cardiomyopathy in the general population and across clinical settings},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sequencing in over 50,000 cases identifies coding and
structural variation underlying atrial fibrillation risk. <em>NG</em>,
<em>57</em>(3), 548–562. (<a
href="https://doi.org/10.1038/s41588-025-02074-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atrial fibrillation (AF) is a prevalent and morbid abnormality of the heart rhythm with a strong genetic component. Here, we meta-analyzed genome and exome sequencing data from 36 studies that included 52,416 AF cases and 277,762 controls. In burden tests of rare coding variation, we identified novel associations between AF and the genes MYBPC3, LMNA, PKP2, FAM189A2 and KDM5B. We further identified associations between AF and rare structural variants owing to deletions in CTNNA3 and duplications of GATA4. We broadly replicated our findings in independent samples from MyCode, deCODE and UK Biobank. Finally, we found that CRISPR knockout of KDM5B in stem-cell-derived atrial cardiomyocytes led to a shortening of the action potential duration and widespread transcriptomic dysregulation of genes relevant to atrial homeostasis and conduction. Our results highlight the contribution of rare coding and structural variants to AF, including genetic links between AF and cardiomyopathies, and expand our understanding of the rare variant architecture for this common arrhythmia. Analyses of genome and exome sequencing data identify rare coding and structural variants associated with atrial fibrillation and highlight genetic links between atrial fibrillation and cardiomyopathies.},
  archive      = {J_NG},
  author       = {Choi, Seung Hoan and Jurgens, Sean J. and Xiao, Ling and Hill, Matthew C. and Haggerty, Christopher M. and Sveinbjörnsson, Garðar and Morrill, Valerie N. and Marston, Nicholas A. and Weng, Lu-Chen and Pirruccello, James P. and Arnar, David O. and Gudbjartsson, Daniel Fannar and Mantineo, Helene and von Falkenhausen, Aenne S. and Natale, Andrea and Tveit, Arnljot and Geelhoed, Bastiaan and Roselli, Carolina and Van Wagoner, David R. and Darbar, Dawood and Haase, Doreen and Soliman, Elsayed Z. and Davogustto, Giovanni E. and Jun, Goo and Calkins, Hugh and Anderson, Jeffrey L. and Brody, Jennifer A. and Halford, Jennifer L. and Barnard, John and Hokanson, John E. and Smith, Jonathan D. and Bis, Joshua C. and Young, Kendra and Johnson, Linda S. B. and Risch, Lorenz and Gula, Lorne J. and Kwee, Lydia Coulter and Chaffin, Mark D. and Kühne, Michael and Preuss, Michael and Gupta, Namrata and Nafissi, Navid A. and Smith, Nicholas L. and Nilsson, Peter M. and van der Harst, Pim and Wells, Quinn S. and Judy, Renae L. and Schnabel, Renate B. and Johnson, Renee and Smit, Roelof A. J. and Gabriel, Stacey and Knight, Stacey and Furukawa, Tetsushi and Blackwell, Thomas W. and Nauffal, Victor and Wang, Xin and Min, Yuan-I and Yoneda, Zachary T. and Laksman, Zachary W. M. and Bezzina, Connie R. and Alonso, Alvaro and Psaty, Bruce M. and Albert, Christine M. and Arking, Dan E. and Roden, Dan M. and Chasman, Daniel I. and Rader, Daniel J. and Conen, David and McManus, David D. and Fatkin, Diane and Benjamin, Emelia J. and Boerwinkle, Eric and Marcus, Gregory M. and Christophersen, Ingrid E. and Smith, J. Gustav and Roberts, Jason D. and Raffield, Laura M. and Shoemaker, M. Benjamin and Cho, Michael H. and Cutler, Michael J. and Rienstra, Michiel and Chung, Mina K. and S. Olesen, Morten and Sinner, Moritz F. and Sotoodehnia, Nona and Kirchhof, Paulus and Loos, Ruth J. F. and Nazarian, Saman and Mohanty, Sanghamitra and Damrauer, Scott M. and Kaab, Stefan and Heckbert, Susan R. and Redline, Susan and Shah, Svati H. and Tanaka, Toshihiro and Ebana, Yusuke and Holm, Hilma and Stefansson, Kari and Ruff, Christian T. and Sabatine, Marc S. and Lunetta, Kathryn L. and Lubitz, Steven A. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-025-02074-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {548-562},
  shortjournal = {Nat. Genet.},
  title        = {Sequencing in over 50,000 cases identifies coding and structural variation underlying atrial fibrillation risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Meta-analysis of genome-wide associations and polygenic risk
prediction for atrial fibrillation in more than 180,000 cases.
<em>NG</em>, <em>57</em>(3), 539–547. (<a
href="https://doi.org/10.1038/s41588-024-02072-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atrial fibrillation (AF) is the most common heart rhythm abnormality and is a leading cause of heart failure and stroke. This large-scale meta-analysis of genome-wide association studies increased the power to detect single-nucleotide variant associations and found more than 350 AF-associated genetic loci. We identified candidate genes related to muscle contractility, cardiac muscle development and cell–cell communication at 139 loci. Furthermore, we assayed chromatin accessibility using assay for transposase-accessible chromatin with sequencing and histone H3 lysine 4 trimethylation in stem cell-derived atrial cardiomyocytes. We observed a marked increase in chromatin accessibility for our sentinel variants and prioritized genes in atrial cardiomyocytes. Finally, a polygenic risk score (PRS) based on our updated effect estimates improved AF risk prediction compared to the CHARGE-AF clinical risk score and a previously reported PRS for AF. The doubling of known risk loci will facilitate a greater understanding of the pathways underlying AF. Genome-wide association meta-analyses identify more than 350 loci associated with atrial fibrillation. A polygenic score derived from these results improves atrial fibrillation risk prediction compared to published clinical and polygenic scores.},
  archive      = {J_NG},
  author       = {Roselli, Carolina and Surakka, Ida and Olesen, Morten S. and Sveinbjornsson, Gardar and Marston, Nicholas A. and Choi, Seung Hoan and Holm, Hilma and Chaffin, Mark and Gudbjartsson, Daniel and Hill, Matthew C. and Aegisdottir, Hildur and Albert, Christine M. and Alonso, Alvaro and Anderson, Christopher D. and Arking, Dan E. and Arnar, David O. and Barnard, John and Benjamin, Emelia J. and Braunwald, Eugene and Brumpton, Ben and Campbell, Archie and Chami, Nathalie and Chasman, Daniel I. and Cho, Kelly and Choi, Eue-Keun and Christophersen, Ingrid E. and Chung, Mina K. and Conen, David and Crijns, Harry J. and Cutler, Michael J. and Czuba, Tomasz and Damrauer, Scott M. and Dichgans, Martin and Dörr, Marcus and Dudink, Elton and Duong, ThuyVy and Erikstrup, Christian and Esko, Tõnu and Fatkin, Diane and Faul, Jessica D. and Ferreira, Manuel and Freitag, Daniel F. and Ganesh, Santhi K. and Gaziano, J. Michael and Geelhoed, Bastiaan and Ghouse, Jonas and Gieger, Christian and Giulianini, Franco and Graham, Sarah E. and Gudnason, Vilmundur and Guo, Xiuqing and Haggerty, Christopher and Hayward, Caroline and Heckbert, Susan R. and Hveem, Kristian and Ito, Kaoru and Johnson, Renee and Jukema, J. Wouter and Jurgens, Sean J. and Kääb, Stefan and Kane, John P. and Kany, Shinwan and Kardia, Sharon L. R. and Kavousi, Maryam and Khurshid, Shaan and Kamanu, Frederick K. and Kirchhof, Paulus and Kleber, Marcus E. and Knight, Stacey and Komuro, Issei and Krieger, Jose E. and Launer, Lenore J. and Li, Dadong and Lin, Honghuang and Lin, Henry J. and Loos, Ruth J. F. and Lotta, Luca and Lubitz, Steven A. and Lunetta, Kathryn L. and Macfarlane, Peter W. and Magnusson, Patrik K. E. and Malik, Rainer and Mantineo, Helene and Marcus, Gregory M. and März, Winfried and McManus, David D. and Melander, Olle and Melloni, Giorgio E. M. and Meyre, Pascal B. and Miyazawa, Kazuo and Mohanty, Sanghamitra and Monfort, Laia M. and Müller-Nurasyid, Martina and Nafissi, Navid A. and Natale, Andrea and Nazarian, Saman and Ostrowski, Sisse R. and Pak, Hui-Nam and Pang, Shichao and Pedersen, Ole B. and Pedersen, Nancy L. and Pereira, Alexandre C. and Pirruccello, James P. and Preuss, Michael and Psaty, Bruce M. and Pullinger, Clive R. and Rader, Daniel J. and Rämö, Joel T. and Ridker, Paul M. and Rienstra, Michiel and Risch, Lorenz and Roden, Dan M. and Rotter, Jerome I. and Sabatine, Marc S. and Schunkert, Heribert and Shah, Svati H. and Shim, Jaemin and Shoemaker, M. Benjamin and Simonson, Bridget and Sinner, Moritz F. and Smit, Roelof A. J. and Smith, Jennifer A. and Smith, Nicholas L. and Smith, J. Gustav and Soliman, Elsayed Z. and Sørensen, Erik and Sotoodehnia, Nona and Strbian, Daniel and Stricker, Bruno H. and Teder-Laving, Maris and Sun, Yan V. and Thériault, Sébastien and Thorolfsdottir, Rosa B. and Thorsteinsdottir, Unnur and Tveit, Arnljot and van der Harst, Pim and van Meurs, Joyce and Wang, Biqi and Weiss, Stefan and Wells, Quinn S. and Weng, Lu-Chen and Wilson, Peter W. and Xiao, Ling and Yang, Pil-Sung and Yao, Jie and Yoneda, Zachary T. and Zeller, Tanja and Zeng, Lingyao and Zhao, Wei and Zhou, Xiang and Zöllner, Sebastian and Ruff, Christian T. and Bundgaard, Henning and Willer, Cristen and Stefansson, Kari and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-024-02072-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {539-547},
  shortjournal = {Nat. Genet.},
  title        = {Meta-analysis of genome-wide associations and polygenic risk prediction for atrial fibrillation in more than 180,000 cases},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large-scale genome-wide association analyses identify novel
genetic loci and mechanisms in hypertrophic cardiomyopathy. <em>NG</em>,
<em>57</em>(3), 530–538. (<a
href="https://doi.org/10.1038/s41588-025-02087-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertrophic cardiomyopathy (HCM) is an important cause of morbidity and mortality with both monogenic and polygenic components. Here, we report results from a large genome-wide association study and multitrait analysis including 5,900 HCM cases, 68,359 controls and 36,083 UK Biobank participants with cardiac magnetic resonance imaging. We identified 70 loci (50 novel) associated with HCM and 62 loci (20 novel) associated with relevant left ventricular traits. Among the prioritized genes in the HCM loci, we identify a novel HCM disease gene, SVIL, which encodes the actin-binding protein supervillin, showing that rare truncating SVIL variants confer a roughly tenfold increased risk of HCM. Mendelian randomization analyses support a causal role of increased left ventricular contractility in both obstructive and nonobstructive forms of HCM, suggesting common disease mechanisms and anticipating shared response to therapy. Taken together, these findings increase our understanding of the genetic basis of HCM, with potential implications for disease management. Genome-wide and multitrait analyses identify novel loci associated with hypertrophic cardiomyopathy and relevant left ventricular traits. Gene-level burden analyses show that rare truncating SVIL variants are associated with high risk of hypertrophic cardiomyopathy.},
  archive      = {J_NG},
  author       = {Tadros, Rafik and Zheng, Sean L. and Grace, Christopher and Jordà, Paloma and Francis, Catherine and West, Dominique M. and Jurgens, Sean J. and Thomson, Kate L. and Harper, Andrew R. and Ormondroyd, Elizabeth and Xu, Xiao and Theotokis, Pantazis I. and Buchan, Rachel J. and McGurk, Kathryn A. and Mazzarotto, Francesco and Boschi, Beatrice and Pelo, Elisabetta and Lee, Michael and Noseda, Michela and Varnava, Amanda and Vermeer, Alexa M. C. and Walsh, Roddy and Amin, Ahmad S. and van Slegtenhorst, Marjon A. and Roslin, Nicole M. and Strug, Lisa J. and Salvi, Erika and Lanzani, Chiara and de Marvao, Antonio and Roberts, Jason D. and Tremblay-Gravel, Maxime and Giraldeau, Genevieve and Cadrin-Tourigny, Julia and L’Allier, Philippe L. and Garceau, Patrick and Talajic, Mario and Gagliano Taliun, Sarah A. and Pinto, Yigal M. and Rakowski, Harry and Pantazis, Antonis and Bai, Wenjia and Baksi, John and Halliday, Brian P. and Prasad, Sanjay K. and Barton, Paul J. R. and O’Regan, Declan P. and Cook, Stuart A. and de Boer, Rudolf A. and Christiaans, Imke and Michels, Michelle and Kramer, Christopher M. and Ho, Carolyn Y. and Neubauer, Stefan and Matthews, Paul M. and Wilde, Arthur A. M. and Tardif, Jean-Claude and Olivotto, Iacopo and Adler, Arnon and Goel, Anuj and Ware, James S. and Bezzina, Connie R. and Watkins, Hugh},
  doi          = {10.1038/s41588-025-02087-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {530-538},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale genome-wide association analyses identify novel genetic loci and mechanisms in hypertrophic cardiomyopathy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The transcriptomic architecture of common cancers reflects
synthetic lethal interactions. <em>NG</em>, <em>57</em>(3), 522–529. (<a
href="https://doi.org/10.1038/s41588-025-02108-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from &amp;gt;9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection. Tumor cells upregulate compensatory buffering genes following tumor suppressor loss. These genes may represent new synthetic lethal partners that could be harnessed therapeutically.},
  archive      = {J_NG},
  author       = {Haider, Syed and Brough, Rachel and Madera, Santiago and Iacovacci, Jacopo and Gulati, Aditi and Wicks, Andrew and Alexander, John and Pettitt, Stephen J. and Tutt, Andrew N. J. and Lord, Christopher J.},
  doi          = {10.1038/s41588-025-02108-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {522-529},
  shortjournal = {Nat. Genet.},
  title        = {The transcriptomic architecture of common cancers reflects synthetic lethal interactions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer-independent somatic mutation of the wild-type NF1
allele in normal tissues in neurofibromatosis type 1. <em>NG</em>,
<em>57</em>(3), 515–521. (<a
href="https://doi.org/10.1038/s41588-025-02097-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer predisposition syndromes mediated by recessive cancer genes generate tumors via somatic variants (second hits) in the unaffected allele. Second hits may or may not be sufficient for neoplastic transformation. Here we performed whole-genome and whole-exome sequencing on 479 tissue biopsies from a child with neurofibromatosis type 1, a multisystem cancer-predisposing syndrome mediated by constitutive monoallelic NF1 inactivation. We identified multiple independent NF1 driver variants in histologically normal tissues, but not in 610 biopsies from two nonpredisposed children. We corroborated this finding using targeted duplex sequencing, including a further nine adults with the same syndrome. Overall, truncating NF1 mutations were under positive selection in normal tissues from individuals with neurofibromatosis type 1. We demonstrate that normal tissues in neurofibromatosis type 1 commonly harbor second hits in NF1, the extent and pattern of which may underpin the syndrome’s cancer phenotype. Histologically normal tissues of individuals with a germline NF1 mutation exhibited multiple second NF1 hits, unrelated to their tumors, that conferred a selective advantage.},
  archive      = {J_NG},
  author       = {Oliver, Thomas R. W. and Lawson, Andrew R. J. and Lee-Six, Henry and Tollit, Anna and Jung, Hyunchul and Hooks, Yvette and Sanghvi, Rashesh and Young, Matthew D. and Butler, Timothy M. and Nicola, Pantelis A. and Treger, Taryn D. and Lensing, Stefanie V. and Burke, G. A. Amos and Aquilina, Kristian and Löbel, Ulrike and Cortes-Ciriano, Isidro and Hargrave, Darren and Jorgensen, Mette and Jessop, Flora A. and Coorens, Tim H. H. and Flanagan, Adrienne M. and Allinson, Kieren and Martincorena, Inigo and Jacques, Thomas S. and Behjati, Sam},
  doi          = {10.1038/s41588-025-02097-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {515-521},
  shortjournal = {Nat. Genet.},
  title        = {Cancer-independent somatic mutation of the wild-type NF1 allele in normal tissues in neurofibromatosis type 1},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genomics of rare diseases in the greater middle east.
<em>NG</em>, <em>57</em>(3), 505–514. (<a
href="https://doi.org/10.1038/s41588-025-02075-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Greater Middle East (GME) represents a concentrated region of unparalleled genetic diversity, characterized by an abundance of distinct alleles, founder mutations and extensive autozygosity driven by high consanguinity rates. These genetic hallmarks present a unique, yet vastly untapped resource for genomic research on Mendelian diseases. Despite this immense potential, the GME continues to face substantial challenges in comprehensive data collection and analysis. This Perspective highlights the region’s unique position as a natural laboratory for genetic discovery and explores the challenges that have stifled progress thus far. Importantly, we propose strategic solutions, advocating for an all-inclusive research approach. With targeted investment and focused efforts, the latent genetic wealth in the GME can be transformed into a global hub for genomic research that will redefine and advance our understanding of the human genome. The Greater Middle East is home to a rich genetic diversity that remains largely untapped in genomic research. This Perspective explores its potential for rare disease research, obstacles limiting progress and strategies to drive genomic discoveries.},
  archive      = {J_NG},
  author       = {Chekroun, Ikram and Shenbagam, Shruti and Almarri, Mohamed A. and Mokrab, Younes and Uddin, Mohammed and Alkhnbashi, Omer S. and Zaki, Maha S. and Najmabadi, Hossein and Kahrizi, Kimia and Fakhro, Khalid A. and Almontashiri, Naif A. M. and Ali, Fahad R. and Özbek, Uğur and Reversade, Bruno and Alkuraya, Fowzan S. and Alsheikh-Ali, Alawi and Abou Tayoun, Ahmad N.},
  doi          = {10.1038/s41588-025-02075-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {505-514},
  shortjournal = {Nat. Genet.},
  title        = {Genomics of rare diseases in the greater middle east},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The african animal breeding network as a pathway towards
genetic improvement of livestock. <em>NG</em>, <em>57</em>(3), 498–504.
(<a href="https://doi.org/10.1038/s41588-025-02079-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transforming Africa’s agricultural production and food systems is an imperative to achieve the United Nations Sustainable Development Goals and deliver on the Africa Union’s 2063 vision, ‘the Africa we want’. Transforming livestock systems through genetic improvement will sustainably increase productivity and proffer an inclusive socioeconomic development of farming communities. The African Animal Breeding Network (AABNet) is a platform of highly knowledgeable geneticists, animal breeders and professionals willing to provide information, training, advice and support across the continent. It will leverage available human resources among its members, facilitate partnerships and investment and develop infrastructure for innovative livestock genetic improvement in Africa. This Perspective introduces the African Animal Breeding Network as a platform to develop infrastructure for innovative livestock genetic improvement in Africa and discusses the challenges in implementing successful animal breeding programs.},
  archive      = {J_NG},
  author       = {Djikeng, Appolinaire and Olori, Victor E. and Houaga, Isidore and Aggrey, Samuel E. and Mwai, Okeyo and Ibeagha-Awemu, Eveline M. and Mrode, Raphael and Chagunda, Mizeck G. G. and Tiambo, Christian K. and Rekaya, Romdhane and Nash, Oyenkanmi and Nziku, Zabron and Opoola, Oluyinka and Ntanganedzeni, Mapholi and Ekine-Dzivenu, Chinyere and Kahi, Alexander and Okeno, Tobias and Hickey, John M. and Enyew, Negussie and Rege, Edward J. O.},
  doi          = {10.1038/s41588-025-02079-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {498-504},
  shortjournal = {Nat. Genet.},
  title        = {The african animal breeding network as a pathway towards genetic improvement of livestock},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unraveling the role of succinyl-CoA in tumor immunity.
<em>NG</em>, <em>57</em>(3), 496–497. (<a
href="https://doi.org/10.1038/s41588-025-02076-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study highlights the crucial role of succinyl-CoA in tumor immunity by inducing the succinylation of PD-L1. We identified the succinyltransferase CPT1A as a potential therapeutic target. CPT1A could be a valuable biomarker for evaluating the efficacy of anti-PD-1 therapy and suggests a promising treatment strategy.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02076-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {496-497},
  shortjournal = {Nat. Genet.},
  title        = {Unraveling the role of succinyl-CoA in tumor immunity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Safeguard repressor locks hepatocyte identity and blocks
liver cancer. <em>NG</em>, <em>57</em>(3), 494–495. (<a
href="https://doi.org/10.1038/s41588-025-02082-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {PROX1 acts as a molecular ‘cell fate guardian’ in hepatocytes, preserving liver cell identity throughout life by actively repressing unwanted plasticity. PROX1 loss impairs liver regeneration and fuels tumor formation, whereas PROX1 activation halts cancer formation and progression, revealing unexpected therapeutic potential.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02082-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {494-495},
  shortjournal = {Nat. Genet.},
  title        = {Safeguard repressor locks hepatocyte identity and blocks liver cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Novel mRNA isoforms in human microglia refine genetic
associations with neurodegeneration. <em>NG</em>, <em>57</em>(3),
492–493. (<a href="https://doi.org/10.1038/s41588-025-02112-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read RNA sequencing of human microglia expands the catalog of mRNA isoforms in this important cell type and assists with the interpretation of genetic associations with Alzheimer’s disease and Parkinson’s disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02112-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {492-493},
  shortjournal = {Nat. Genet.},
  title        = {Novel mRNA isoforms in human microglia refine genetic associations with neurodegeneration},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Polyguanine microsatellites are robust replication clocks in
cancer. <em>NG</em>, <em>57</em>(3), 489–491. (<a
href="https://doi.org/10.1038/s41588-025-02098-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer phylogenetic trees describe the evolutionary relationship between primary tumors and metastatic sites. A study now shows that mutations in guanine homopolymer microsatellites represent accurate molecular clocks, revealing the number of cell divisions that have occurred during cancer development and progression.},
  archive      = {J_NG},
  author       = {Gejman, Ron S. and Izar, Benjamin},
  doi          = {10.1038/s41588-025-02098-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {489-491},
  shortjournal = {Nat. Genet.},
  title        = {Polyguanine microsatellites are robust replication clocks in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mass spectrometry-based mapping of plasma protein QTLs in
children and adolescents. <em>NG</em>, <em>57</em>(3), 487–488. (<a
href="https://doi.org/10.1038/s41588-025-02088-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectrometry-based proteomics demonstrated its increasing analytical power and unique strengths in measuring genome-wide plasma proteomes and identifying protein quantitative trait loci (pQTLs), as reported in a study of over 3,000 Danish children and adolescents.},
  archive      = {J_NG},
  author       = {Liu, Yansheng},
  doi          = {10.1038/s41588-025-02088-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {487-488},
  shortjournal = {Nat. Genet.},
  title        = {Mass spectrometry-based mapping of plasma protein QTLs in children and adolescents},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell mapping of HTT somatic repeat expansions.
<em>NG</em>, <em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02142-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02142-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell mapping of HTT somatic repeat expansions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput screening identifies modulators of gene
bursting. <em>NG</em>, <em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02143-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02143-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {High-throughput screening identifies modulators of gene bursting},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding immune escape in lung cancer. <em>NG</em>,
<em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02144-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02144-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Understanding immune escape in lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hyperactive histone genes fuel cancer. <em>NG</em>,
<em>57</em>(3), 486. (<a
href="https://doi.org/10.1038/s41588-025-02145-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02145-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Hyperactive histone genes fuel cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The federated european genome–phenome archive as a global
network for sharing human genomics data. <em>NG</em>, <em>57</em>(3),
481–485. (<a href="https://doi.org/10.1038/s41588-025-02101-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this era of rapidly expanding human genomics in research and healthcare, efficient data reuse is essential to maximize benefits for society. In response, the Federated European Genome–Phenome Archive (FEGA) was launched in 2022, and as of 2024, the FEGA network was composed of seven national nodes. Here we describe the complexities, challenges and achievements of FEGA, unravelling the dynamic interplay of regulatory frameworks, technical challenges and the shared vision of advancing genomic research.},
  archive      = {J_NG},
  author       = {D’Altri, Teresa and Freeberg, Mallory Ann and Curwin, Amy J. and Alonso, Ana and Freitas, Ana T. and Capella-Gutierrez, Salvador and Gadelha, Luiz and Hagwall, Anna and Hovig, Eivind and Kerry, Giselle and Kirli, Koray and Kochel, Krzysztof and Kohlbacher, Oliver and Korbel, Jan O. and Leinonen, Jaakko and Marciniak, Blazej and Oliveira, Jorge S. and Petersen, Kjell and Silva, Mário J. and Stegle, Oliver and Valencia, Alfonso and Viklund, Johan and Guigo, Roderic and Parkinson, Helen and Navarro, Arcadi and Rambla, Jordi and Keane, Thomas M.},
  doi          = {10.1038/s41588-025-02101-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {481-485},
  shortjournal = {Nat. Genet.},
  title        = {The federated european Genome–Phenome archive as a global network for sharing human genomics data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="nmat---28">NMAT - 28</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Switchable long-distance propagation
of chiral magnonic edge states. <em>NMAT</em>, <em>24</em>(3), 472. (<a
href="https://doi.org/10.1038/s41563-025-02132-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMAT},
  author       = {Zhang, Yuelin and Qiu, Lei and Chen, Jilei and Wu, Shizhe and Wang, Hanchen and Malik, Iftikhar Ahmed and Cai, Miming and Wu, Mei and Gao, Peng and Hua, Chensong and Yu, Weichao and Xiao, Jiang and Jiang, Yong and Yu, Haiming and Shen, Ka and Zhang, Jinxing},
  doi          = {10.1038/s41563-025-02132-x},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {472},
  shortjournal = {Nat. Mater.},
  title        = {Publisher correction: Switchable long-distance propagation of chiral magnonic edge states},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Pentagonal two-dimensional lattices.
<em>NMAT</em>, <em>24</em>(3), 471. (<a
href="https://doi.org/10.1038/s41563-024-02045-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMAT},
  author       = {Heine, Thomas},
  doi          = {10.1038/s41563-024-02045-1},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {471},
  shortjournal = {Nat. Mater.},
  title        = {Author correction: Pentagonal two-dimensional lattices},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stretch-induced endogenous electric fields drive directed
collective cell migration in vivo. <em>NMAT</em>, <em>24</em>(3),
462–470. (<a href="https://doi.org/10.1038/s41563-024-02060-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directed collective cell migration is essential for morphogenesis, and chemical, electrical, mechanical and topological features have been shown to guide cell migration in vitro. Here we provide in vivo evidence showing that endogenous electric fields drive the directed collective cell migration of an embryonic stem cell population—the cephalic neural crest of Xenopus laevis. We demonstrate that the voltage-sensitive phosphatase 1 is a key component of the molecular mechanism, enabling neural crest cells to specifically transduce electric fields into a directional cue in vivo. Finally, we propose that endogenous electric fields are mechanically established by the convergent extension movements of the ectoderm, which generate a membrane tension gradient that opens stretch-activated ion channels. Overall, these findings establish a role for electrotaxis in tissue morphogenesis, highlighting the functions of endogenous bioelectrical stimuli in non-neural contexts. Electric fields guide collective cell migration in developing embryos of Xenopus laevis via a voltage-sensitive phosphatase.},
  archive      = {J_NMAT},
  author       = {Ferreira, Fernando and Moreira, Sofia and Zhao, Min and Barriga, Elias H.},
  doi          = {10.1038/s41563-024-02060-2},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {462-470},
  shortjournal = {Nat. Mater.},
  title        = {Stretch-induced endogenous electric fields drive directed collective cell migration in vivo},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Topological linking determines elasticity in limited valence
networks. <em>NMAT</em>, <em>24</em>(3), 454–461. (<a
href="https://doi.org/10.1038/s41563-024-02091-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the relationship between the microscopic structure and topology of a material and its macroscopic properties is a fundamental challenge across a wide range of systems. Here we investigate the viscoelasticity of DNA nanostar hydrogels—a model system for physical networks with limited valence—by coupling rheology measurements, confocal imaging and molecular dynamics simulations. We discover that these networks display a large degree of interpenetration and that loops within the network are topologically linked, forming a percolating network-within-network structure. Below the overlapping concentration, the fraction of branching points and the pore size determine the high-frequency elasticity of these physical gels. At higher concentrations, we discover that this elastic response is dictated by the abundance of topological links between looped motifs in the gel. Our findings highlight the emergence of ‘topological elasticity’ as a previously overlooked mechanism in generic network-forming liquids and gels and inform the design of topologically controllable material behaviours. Experiments and simulations of DNA nanostar hydrogels reveal that microscopic topology determines macroscale elasticity in amorphous networks.},
  archive      = {J_NMAT},
  author       = {Palombo, Giorgia and Weir, Simon and Michieletto, Davide and Gutiérrez Fosado, Yair Augusto},
  doi          = {10.1038/s41563-024-02091-9},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {454-461},
  shortjournal = {Nat. Mater.},
  title        = {Topological linking determines elasticity in limited valence networks},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Organic solar cells with 20.82. <em>NMAT</em>,
<em>24</em>(3), 444–453. (<a
href="https://doi.org/10.1038/s41563-024-02062-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Printing of large-area solar panels necessitates advanced organic solar cells with thick active layers. However, increasing the active layer thickness typically leads to a marked drop in the power conversion efficiency. Here we developed an organic semiconductor regulator, called AT-β2O, to tune the crystallization sequence of the components in active layers. When adding AT-β2O in the donor (D18-Cl) and acceptor (N3) blend, N3 crystallizes behind D18-Cl, and this phenomenon is different from the co-crystallization observed in binary D18-Cl:N3 blends. This manipulation of crystallization dynamics is favourable to form bulk-heterojunction-gradient vertical phase separation in the active layer accompanied by the high crystallinity of the acceptor and balanced charge carrier mobilities in thick films. The resultant single-junction organic solar cells exhibited a certified power conversion efficiency of over 20%, as well as demonstrated exceptional adaptability across the active layer thicknesses (100–400 nm) and remarkable universality. Such breakthroughs enable large-area modules with a certified power conversion efficiency of 18.04%. An organic regulator that can tune the crystallization sequence of active layer components has been described, achieving a certified efficiency of over 20% in single-junction organic solar cells, demonstrating remarkable tolerance for active layer thickness of 100–400 nm.},
  archive      = {J_NMAT},
  author       = {Chen, Haiyang and Huang, Yuting and Zhang, Rui and Mou, Hongyu and Ding, Junyuan and Zhou, Jiadong and Wang, Zukun and Li, Hongxiang and Chen, Weijie and Zhu, Juan and Cheng, Qinrong and Gu, Hao and Wu, Xiaoxiao and Zhang, Tianjiao and Wang, Yingyi and Zhu, Haiming and Xie, Zengqi and Gao, Feng and Li, Yaowen and Li, Yongfang},
  doi          = {10.1038/s41563-024-02062-0},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {444-453},
  shortjournal = {Nat. Mater.},
  title        = {Organic solar cells with 20.82% efficiency and high tolerance of active layer thickness through crystallization sequence manipulation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Non-fullerene acceptors with high crystallinity and
photoluminescence quantum yield enable &gt;20. <em>NMAT</em>,
<em>24</em>(3), 433–443. (<a
href="https://doi.org/10.1038/s41563-024-02087-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rational design of non-fullerene acceptors (NFAs) with both high crystallinity and photoluminescence quantum yield (PLQY) is of crucial importance for achieving high-efficiency and low-energy-loss organic solar cells (OSCs). However, increasing the crystallinity of an NFA tends to decrease its PLQY, which results in a high non-radiative energy loss in OSCs. Here we demonstrate that the crystallinity and PLQY of NFAs can be fine-tuned by asymmetrically adapting the branching position of alkyl chains on the thiophene unit of the L8-BO acceptor. It was found that L8-BO-C4, with 2-butyloctyl on one side and 4-butyldecyl on the other side, can simultaneously achieve high crystallinity and PLQY. A high efficiency of 20.42% (certified as 20.1%) with an open-circuit voltage of 0.894 V and a fill factor of 81.6% is achieved for the single-junction OSC. This work reveals how important the strategy of shifting the alkyl chain branching position is in developing high-performance NFAs for efficient OSCs. Design strategies for non-fullerene acceptors are important for achieving high-efficiency organic solar cells. Here the authors design asymmetrically branched alkyl chains on the thiophene unit of the L8-BO acceptor to achieve high crystallinity and photoluminescence quantum yield, yielding over 20% efficiency in single-junction organic solar cells.},
  archive      = {J_NMAT},
  author       = {Li, Chao and Song, Jiali and Lai, Hanjian and Zhang, Huotian and Zhou, Rongkun and Xu, Jinqiu and Huang, Haodong and Liu, Liming and Gao, Jiaxin and Li, Yuxuan and Jee, Min Hun and Zheng, Zilong and Liu, Sha and Yan, Jun and Chen, Xian-Kai and Tang, Zheng and Zhang, Chen and Woo, Han Young and He, Feng and Gao, Feng and Yan, He and Sun, Yanming},
  doi          = {10.1038/s41563-024-02087-5},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {433-443},
  shortjournal = {Nat. Mater.},
  title        = {Non-fullerene acceptors with high crystallinity and photoluminescence quantum yield enable &gt;20% efficiency organic solar cells},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electroreduction-driven distorted nanotwins activate pure cu
for efficient hydrogen evolution. <em>NMAT</em>, <em>24</em>(3),
424–432. (<a href="https://doi.org/10.1038/s41563-024-02098-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precious metals such as Pt are favoured as catalysts for the hydrogen evolution reaction (HER) due to their excellent catalytic activity. However, the scarcity and high cost of precious metals have prompted researchers to explore cheaper alternatives such as Cu. Nevertheless, Cu shows poor catalytic performance due to weak binding with intermediates. Here the catalytic activity of pure Cu is activated via electroreduction-driven modification of the local structure, achieving a HER catalytic performance superior to commercial Pt/C catalysts for working current densities greater than 100 mA cm−2 in acid electrolyte. Activation involved two steps. First, polycrystalline Cu2O nanoparticles were prepared via pulsed laser ablation, resulting in grain boundaries within the Cu2O particles as observed using electron microscopy. Next, the Cu2O particles were electroreduced to pure Cu, inducing the formation of distorted nanotwins and edge dislocations. These local structures induce high lattice strain and decrease the Cu coordination number, enhancing the interaction between Cu and intermediates—as calculated using density functional theory—leading to the excellent catalytic activity and durability of the catalyst. Our observations show that low-cost pure Cu can be a promising HER catalyst for large-scale industrial applications. Low-cost Cu catalysts for the hydrogen evolution reaction (HER) can transform industrial water electrolysis, but pure Cu typically exhibits a negligible HER. Here, combining pulsed laser ablation and subsequent electroreduction, Cu nanotwins form that enable the HER at an overpotential of 301 mV, with 125 h of stable operation at a current density of 500 mA cm−2.},
  archive      = {J_NMAT},
  author       = {Li, Zhe and Wang, Yueshuai and Liu, Hui and Feng, Yi and Du, Xiwen and Xie, Zhiheng and Zhou, Jihan and Liu, Yang and Song, Yun and Wang, Fei and Sui, Manling and Lu, Yue and Fang, Fang and Sun, Dalin},
  doi          = {10.1038/s41563-024-02098-2},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {424-432},
  shortjournal = {Nat. Mater.},
  title        = {Electroreduction-driven distorted nanotwins activate pure cu for efficient hydrogen evolution},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Revitalizing interphase in all-solid-state li metal
batteries by electrophile reduction. <em>NMAT</em>, <em>24</em>(3),
414–423. (<a href="https://doi.org/10.1038/s41563-024-02064-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {All-solid-state lithium metal batteries promise high levels of safety and energy density, but their practical realization is limited by low Li reversibility, limited cell loading and demand for high-temperature and high-pressure operation, stemming from solid-state electrolyte (SSE) low-voltage reduction and high-voltage decomposition, and from lithium dendrite growth. Here we concurrently address these challenges by reporting that a family of reductive electrophiles gain electrons and cations from metal–nucleophile materials (here a Li sulfide SSE) upon contact to undergo electrochemical reduction and form interphase layers (named solid reductive-electrophile interphase) on material surfaces. The solid reductive-electrophile interphase is electron blocking and lithiophobic, prevents SSE reduction, suppresses Li dendrites and supports high-voltage cathodes. Consequently, a reductive-electrophile-treated SSE exhibits high critical capacity and Li reversibility at the anode, and enables Li(1% Mg)/SSE/LiNi0.8Co0.15Al0.05O2 all-solid-state lithium metal batteries to achieve a high coulombic efficiency (&amp;gt;99.9%), long cycle life (~10,000 h) and high loading (&amp;gt;7 mAh cm−2) at 30 °C and 2.5 MPa. This concept also extends to cathodes of other materials (for example, metal oxides), boosting the high-nickel cathode’s cycle life and expanding the operational voltage up to 4.5 V. Such solid reductive-electrophile interphase tailoring of material surfaces holds promise to accelerate all-solid-state lithium metal battery commercialization and offer solutions for a wide range of materials. Solid-state electrolyte reduction and Li dendrite growth limit the stability of all-solid-state Li metal batteries. Here the authors show that reductive electrophiles gain electrons and metal cations from metal–nucleophile materials on contact, allowing the electrochemical formation of a dense, electron-blocking film that improves the stability of both the anode and high-voltage cathode.},
  archive      = {J_NMAT},
  author       = {Zhang, Weiran and Wang, Zeyi and Wan, Hongli and Li, Ai-Min and Liu, Yijie and Liou, Sz-Chian and Zhang, Kai and Ren, Yuxun and Jayawardana, Chamithri and Lucht, Brett L. and Wang, Chunsheng},
  doi          = {10.1038/s41563-024-02064-y},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {414-423},
  shortjournal = {Nat. Mater.},
  title        = {Revitalizing interphase in all-solid-state li metal batteries by electrophile reduction},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Correlated spin-wave generation and domain-wall oscillation
in a topologically textured magnetic film. <em>NMAT</em>,
<em>24</em>(3), 406–413. (<a
href="https://doi.org/10.1038/s41563-024-02085-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spin waves, or magnons, are essential for next-generation energy-efficient spintronics and magnonics. Yet, visualizing spin-wave dynamics at nanoscale and microwave frequencies remains a formidable challenge due to the lack of spin-sensitive, time-resolved microscopy. Here we report a breakthrough in imaging dipole-exchange spin waves in a ferromagnetic film owing to the development of laser-free ultrafast Lorentz electron microscopy, which is equipped with a microwave-mediated electron pulser for high spatiotemporal resolution. Using topological spin textures, we captured the emission, propagation, reflection and interference of spin waves from spin anti-vortices under radio-frequency excitations. Remarkably, we show that spin-wave generation is closely tied to the oscillatory motion of specific magnetic domain walls, providing the missing link between wave emission and wall dynamics near magnetic singularities. This work opens new possibilities in magnonics, offering a nanoscopic view of spin dynamics via transmission electron microscopy and enabling controlled excitation via radio-frequency fields for exploring non-equilibrium states in magnetic and multiferroic systems. A laser-free ultrafast Lorentz electron microscope has been developed, integrating a microwave-based electron pulser to achieve high spatiotemporal imaging of spin-wave dynamics in a topologically textured thin-film permalloy.},
  archive      = {J_NMAT},
  author       = {Liu, Chuhang and Ai, Fangzhou and Reisbick, Spencer and Zong, Alfred and Pofelski, Alexandre and Han, Myung-Geun and Camino, Fernando and Jing, Chunguang and Lomakin, Vitaliy and Zhu, Yimei},
  doi          = {10.1038/s41563-024-02085-7},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {406-413},
  shortjournal = {Nat. Mater.},
  title        = {Correlated spin-wave generation and domain-wall oscillation in a topologically textured magnetic film},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Double-sided van der waals epitaxy of topological insulators
across an atomically thin membrane. <em>NMAT</em>, <em>24</em>(3),
399–405. (<a href="https://doi.org/10.1038/s41563-024-02079-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atomically thin van der Waals (vdW) films provide a material platform for the epitaxial growth of quantum heterostructures. However, unlike the remote epitaxial growth of three-dimensional bulk crystals, the growth of two-dimensional material heterostructures across atomic layers has been limited due to the weak vdW interaction. Here we report the double-sided epitaxy of vdW layered materials through atomic membranes. We grow vdW topological insulators Sb2Te3 and Bi2Se3 by molecular-beam epitaxy on both surfaces of atomically thin graphene or hexagonal boron nitride, which serve as suspended two-dimensional vdW substrate layers. Both homo- and hetero-double-sided vdW topological insulator tunnel junctions are fabricated, with the atomically thin hexagonal boron nitride acting as a crystal-momentum-conserving tunnelling barrier with abrupt and epitaxial interfaces. By performing field-angle-dependent magneto-tunnelling spectroscopy on these devices, we reveal the energy–momentum–spin resonance of massless Dirac electrons tunnelling between helical Landau levels developed in the topological surface states at the interfaces. Double-sided epitaxy of van der Waals materials through atomic membranes is demonstrated, enabling electrons to resonantly tunnel between aligned topological insulator surfaces with the conservation of energy, momentum and spin helicity.},
  archive      = {J_NMAT},
  author       = {Park, Joon Young and Shin, Young Jae and Shin, Jeacheol and Kim, Jehyun and Jo, Janghyun and Yoo, Hyobin and Haei, Danial and Hyun, Chohee and Yun, Jiyoung and Huber, Robert M. and Gupta, Arijit and Watanabe, Kenji and Taniguchi, Takashi and Park, Wan Kyu and Shin, Hyeon Suk and Kim, Miyoung and Kim, Dohun and Yi, Gyu-Chul and Kim, Philip},
  doi          = {10.1038/s41563-024-02079-5},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {399-405},
  shortjournal = {Nat. Mater.},
  title        = {Double-sided van der waals epitaxy of topological insulators across an atomically thin membrane},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Magnetically confined surface and bulk excitons in a layered
antiferromagnet. <em>NMAT</em>, <em>24</em>(3), 391–398. (<a
href="https://doi.org/10.1038/s41563-025-02129-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery of two-dimensional van der Waals magnets has greatly expanded our ability to create and control nanoscale quantum phases. A unique capability emerges when a two-dimensional magnet is also a semiconductor that features tightly bound excitons with large oscillator strengths that fundamentally determine the optical response and are tunable with magnetic fields. Here we report a previously unidentified type of optical excitation—a magnetic surface exciton—enabled by the antiferromagnetic spin correlations that confine excitons to the surface of CrSBr. Magnetic surface excitons exhibit stronger Coulomb attraction, leading to a higher binding energy than excitons confined in bulk layers, and profoundly alter the optical response of few-layer crystals. Distinct magnetic confinement of surface and bulk excitons is established by layer- and temperature-dependent exciton reflection spectroscopy and corroborated by ab initio many-body perturbation theory calculations. By quenching interlayer excitonic interactions, the antiferromagnetic order of CrSBr strictly confines the bound electron–hole pairs within the same layer, regardless of the total number of layers. Our work unveils unique confined excitons in a layered antiferromagnet, highlighting magnetic interactions as a vital approach for nanoscale quantum confinement, from few layers to the bulk limit. The emergence of magnetically confined surface excitons enabled by antiferromagnetic spin correlations is reported, which leads to the confinement of excitons to the surface of layered antiferromagnet CrSBr.},
  archive      = {J_NMAT},
  author       = {Shao, Yinming and Dirnberger, Florian and Qiu, Siyuan and Acharya, Swagata and Terres, Sophia and Telford, Evan J. and Pashov, Dimitar and Kim, Brian S. Y. and Ruta, Francesco L. and Chica, Daniel G. and Dismukes, Avalon H. and Ziebel, Michael E. and Wang, Yiping and Choe, Jeongheon and Bae, Youn Jue and Millis, Andrew J. and Katsnelson, Mikhail I. and Mosina, Kseniia and Sofer, Zdenek and Huber, Rupert and Zhu, Xiaoyang and Roy, Xavier and van Schilfgaarde, Mark and Chernikov, Alexey and Basov, D. N.},
  doi          = {10.1038/s41563-025-02129-6},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {391-398},
  shortjournal = {Nat. Mater.},
  title        = {Magnetically confined surface and bulk excitons in a layered antiferromagnet},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Controlling coulomb correlations and fine structure of
quasi-one-dimensional excitons by magnetic order. <em>NMAT</em>,
<em>24</em>(3), 384–390. (<a
href="https://doi.org/10.1038/s41563-025-02120-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many surprising properties of quantum materials result from Coulomb correlations defining electronic quasiparticles and their interaction chains. In van der Waals layered crystals, enhanced correlations have been tailored in reduced dimensions, enabling excitons with giant binding energies and emergent phases including ferroelectric, ferromagnetic and multiferroic orders. Yet, correlation design has primarily relied on structural engineering. Here we present quantitative experiment–theory proof that excitonic correlations can be switched through magnetic order. By probing internal Rydberg-like transitions of excitons in the magnetic semiconductor CrSBr, we reveal their binding energy and a dramatic anisotropy of their quasi-one-dimensional orbitals manifesting in strong fine-structure splitting. We switch the internal structure from strongly bound, monolayer-localized states to weakly bound, interlayer-delocalized states by pushing the system from antiferromagnetic to paramagnetic phases. Our analysis connects this transition to the exciton’s spin-controlled effective quantum confinement, supported by the exciton’s dynamics. In future applications, excitons or even condensates may be interfaced with spintronics; extrinsically switchable Coulomb correlations could shape phase transitions on demand. The antiferromagnetic-to-paramagnetic phase transition in a two-dimensional semiconducting magnet, CrSBr, induces an exciton confinement transition from a strongly bound quasi-one-dimensional state to a weakly bound interlayer-delocalized state.},
  archive      = {J_NMAT},
  author       = {Liebich, M. and Florian, M. and Nilforoushan, N. and Mooshammer, F. and Koulouklidis, A. D. and Wittmann, L. and Mosina, K. and Sofer, Z. and Dirnberger, F. and Kira, M. and Huber, R.},
  doi          = {10.1038/s41563-025-02120-1},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {384-390},
  shortjournal = {Nat. Mater.},
  title        = {Controlling coulomb correlations and fine structure of quasi-one-dimensional excitons by magnetic order},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Graphene rolls with tunable chirality. <em>NMAT</em>,
<em>24</em>(3), 377–383. (<a
href="https://doi.org/10.1038/s41563-025-02127-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Creating chirality in achiral graphene and other two-dimensional materials has attracted broad scientific interest due to their potential application in advanced optics, electronics and spintronics. However, investigations into their optical activities and related chiro-electronic properties are constrained by experimental challenges, particularly in the precise control over the chirality of these materials. Here a universal wax-aided immersion method is developed to yield graphene rolls with controllable chiral angles, and the method can be generalized in other two-dimensional materials for high-yield fabrication. The left-handed and right-handed rolls exhibit optical activity and excellent spin selectivity effects with a spin polarization over 90% at room temperature. The discovery of tunable chirality-induced spin selectivity in tailored roll-shaped allotropes, achievable only through precise control of chirality, distinguishes itself from other carbon materials or existing chiral materials. Our Dirac fermion model shows that the electrons moving predominately along one side of the chiral roll develop a preferred spin polarization, and the rolling-chirality-induced spin selectivity is a result of this finite spin selectivity effect. Our method opens up opportunities for endowing achiral two-dimensional materials with tunable chirality, and may enable the emergence of quantum behaviours and room-temperature spintronic technologies. A wax-aided immersion methodology is developed to yield graphene rolls with tunable chiral angles; these graphene rolls exhibit promising chiral electronic properties beyond those of other carbon allotropes.},
  archive      = {J_NMAT},
  author       = {Zhang, Enbing and Ding, Shuaishuai and Li, Xiaopeng and Ma, Xiangyun and Gao, Xiaoqing and Liu, Lei and Shen, Yongtao and Cheng, Shiyu and Mi, Wenbo and Zhou, Yunlong and Feng, Guangyuan and Song, Yaru and Li, Xiaojuan and Xue, Yunjia and Xin, Kaiyao and Zeng, Xin and Jiang, Qinyuan and Zhang, Rufan and Liu, Xinfeng and Wei, Zhongming and Zeng, Qingdao and Wang, Bin and Li, Qifeng and Liu, Ji and Yan, Jing and Lei, Shengbin and Yang, Yanlian and Hu, Wenping},
  doi          = {10.1038/s41563-025-02127-8},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {377-383},
  shortjournal = {Nat. Mater.},
  title        = {Graphene rolls with tunable chirality},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Interlayer reconstruction phase transition in van der waals
materials. <em>NMAT</em>, <em>24</em>(3), 369–376. (<a
href="https://doi.org/10.1038/s41563-024-02082-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Van der Waals materials display rich structural polymorphs with distinct physical properties. An atomistic understanding of the phase-transition dynamics, propagation pathway and associated evolution of physical properties is essential for capturing their potential in practical technologies. However, direct visualization of the rapid phase-transition process is fundamentally challenging due to the inherent trade-offs among atomic resolution, field of view and imaging frame rate. Here we exploit a controllable current-driven phase transition and utilize in situ scanning transmission electron microscopy to visualize dynamic atomic rearrangements during the 2H-α to 2H-β transition in layered In2Se3. We identify a unique intralayer-splitting (unzipping) and interlayer-reconstruction (zipping) pathway, driven by an energy-cascading mechanism through which bond formation across the van der Waals gap facilitates bond cleavage in the covalent layers. We also observe current-direction-dependent asymmetric phase-transition propagation and attribute it to a temperature profile induced by the Peltier effect at the heterophase interface. These findings provide insights that are essential for designing tailored structural phase transitions in advanced technologies. Current-driven dynamic atomic rearrangements in layered In2Se3 are visualized. The authors identify an intralayer ‘unzipping’ and interlayer ‘zipping’ phase-transition pathway in which bond formation across the van der Waals gaps drives bond cleavage within covalent layers.},
  archive      = {J_NMAT},
  author       = {Zhang, Junwei and Wang, Laiyuan and Lü, Jingtao and Wang, Zhe and Wu, Huan and Zhu, Guilin and Wang, Nan and Xue, Fei and Zeng, Xue and Zhu, Liu and Hu, Yang and Deng, Xia and Guan, Chaoshuai and Yang, Chen and Lin, Zhaoyang and Wang, Peiqi and Zhou, Boxuan and Lü, Jing and Zhu, Wenguang and Zhang, Xixiang and Huang, Yu and Huang, Wei and Peng, Yong and Duan, Xiangfeng},
  doi          = {10.1038/s41563-024-02082-w},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {369-376},
  shortjournal = {Nat. Mater.},
  title        = {Interlayer reconstruction phase transition in van der waals materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Activating deformation twinning in cubic boron nitride.
<em>NMAT</em>, <em>24</em>(3), 361–368. (<a
href="https://doi.org/10.1038/s41563-024-02111-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deformation twinning, a phenomenon primarily documented within metallic systems, has remained essentially unexplored in covalent materials due to the formidable challenges posed by their inherent extreme hardness and brittleness. Here, by employing a five-degree-of-freedom nano-manipulation stage inside a transmission electron microscope, we reveal a loading-specific twinning criterion for cubic boron nitride and successfully activate extensive deformation twinning with substantial improvements in mechanical properties in &amp;lt;100&amp;gt;-oriented cubic boron nitride submicrometre pillars at room temperature. Beyond cubic boron nitride, this criterion is also proven widely applicable across a spectrum of covalent materials. Investigations on the twinning dynamics at the atomic level in cubic boron nitride suggest a continuous-transition-mediated pathway. These findings substantially advance our comprehension of twinning mechanisms in covalent face-centred cubic materials, and herald a promising avenue for microstructural engineering aimed at enhancing the strength and toughness of these materials in their applications. Deformation twinning, a key deformation mechanism that is rarely explored in superhard materials, is shown to be activated in cubic boron nitride and other cubic covalent materials under a loading-specific twinning criterion.},
  archive      = {J_NMAT},
  author       = {Bu, Yeqiang and Su, Zhengping and Huang, Junquan and Tong, Ke and Li, Penghui and Wang, Chong and Jin, Tianye and Zhao, Song and Zhao, Zhisheng and Soldatov, Alexander and Wang, Yanbin and Xu, Bo and Liu, Zhongyuan and Nie, Anmin and Wang, Hongtao and Yang, Wei and Tian, Yongjun},
  doi          = {10.1038/s41563-024-02111-8},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {361-368},
  shortjournal = {Nat. Mater.},
  title        = {Activating deformation twinning in cubic boron nitride},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Origins of elasticity in molecular materials. <em>NMAT</em>,
<em>24</em>(3), 356–360. (<a
href="https://doi.org/10.1038/s41563-025-02133-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elasticity is ubiquitous and produces a spontaneously reversible response to applied stress1. Despite the utility and importance of this property in regard to scientific and engineering applications, the atomic-scale location of the force that returns an object to its original shape remains elusive in molecular crystals. Here we use a series of density functional theory calculations to locate precisely where the energy is stored when single crystals of three molecular materials are placed under elastic stress. We show for each material that different intermolecular interactions are responsible for the restoring force under both expansive and compressive strain. These findings provide insight into the elastic behaviour of crystalline materials that is needed for more efficient design of flexible technologies and future smart devices. Elasticity is ubiquitous in everyday life, but the molecular origin of the restoring force remains elusive. Here the authors use a series of density functional theory calculations to understand how interaction energies change as a result of the bending of molecular crystals.},
  archive      = {J_NMAT},
  author       = {Thompson, Amy J. and Chong, Bowie S. K. and Kenny, Elise P. and Evans, Jack D. and Powell, Joshua A. and Spackman, Mark A. and McMurtrie, John C. and Powell, Benjamin J. and Clegg, Jack K.},
  doi          = {10.1038/s41563-025-02133-w},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {356-360},
  shortjournal = {Nat. Mater.},
  title        = {Origins of elasticity in molecular materials},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cavity-enhanced energy transport in molecular systems.
<em>NMAT</em>, <em>24</em>(3), 344–355. (<a
href="https://doi.org/10.1038/s41563-024-01962-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecules are the building blocks of all of nature’s functional components, serving as the machinery that captures, stores and releases energy or converts it into useful work. However, molecules interact with each other over extremely short distances, which hinders the spread of energy across molecular systems. Conversely, photons are inert, but they are fast and can traverse large distances very efficiently. Using optical resonators, these distinct entities can be mixed with each other, opening a path to new architectures that benefit from both the active nature of molecules and the long-range transport obtained by the coupling with light. In this Review, we present the physics underlying the enhancement of energy transfer and energy transport in molecular systems, and highlight the experimental and theoretical advances in this field over the past decade. Finally, we identify several key questions and theoretical challenges that remain to be resolved via future research. The authors discuss the physics underlying the enhancement of energy transfer and energy transport in molecular systems and identify key questions and theoretical challenges for future research.},
  archive      = {J_NMAT},
  author       = {Sandik, Gal and Feist, Johannes and García-Vidal, Francisco J. and Schwartz, Tal},
  doi          = {10.1038/s41563-024-01962-5},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {344-355},
  shortjournal = {Nat. Mater.},
  title        = {Cavity-enhanced energy transport in molecular systems},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tensile strain and low coordination activate pure copper for
hydrogen evolution. <em>NMAT</em>, <em>24</em>(3), 342–343. (<a
href="https://doi.org/10.1038/s41563-024-02099-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Copper is typically used as a current collector rather than a catalyst for water electrolysis owing to its weak hydrogen adsorption capability. Now, the electroreduction-driven induction of strong and stable tensile strain in low-coordination copper nanoparticles is shown to increase the copper–hydrogen interaction and, thus, the catalytic activity for hydrogen evolution.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-024-02099-1},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {342-343},
  shortjournal = {Nat. Mater.},
  title        = {Tensile strain and low coordination activate pure copper for hydrogen evolution},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). When stars make loopy networks. <em>NMAT</em>,
<em>24</em>(3), 340–341. (<a
href="https://doi.org/10.1038/s41563-025-02157-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The high-frequency elastic response reveals interpenetrated and polycatenated structures in DNA nanostar network materials.},
  archive      = {J_NMAT},
  author       = {Del Gado, Emanuela},
  doi          = {10.1038/s41563-025-02157-2},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {340-341},
  shortjournal = {Nat. Mater.},
  title        = {When stars make loopy networks},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Asymmetric side-chains work. <em>NMAT</em>, <em>24</em>(3),
338–339. (<a href="https://doi.org/10.1038/s41563-024-02102-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new series of non-fullerene acceptors with asymmetric branched alkyl chains are developed to achieve more than 20% efficiency organic solar cells.},
  archive      = {J_NMAT},
  author       = {Neher, Dieter},
  doi          = {10.1038/s41563-024-02102-9},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {338-339},
  shortjournal = {Nat. Mater.},
  title        = {Asymmetric side-chains work},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Organic photovoltaics surpass the 20. <em>NMAT</em>,
<em>24</em>(3), 336–337. (<a
href="https://doi.org/10.1038/s41563-024-02076-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crystallization dynamics manipulation leads to vertically separated donor and acceptor phases in thick films, improving charge mobility and device efficiency.},
  archive      = {J_NMAT},
  author       = {Furlan, Francesco and Gasparini, Nicola},
  doi          = {10.1038/s41563-024-02076-8},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {336-337},
  shortjournal = {Nat. Mater.},
  title        = {Organic photovoltaics surpass the 20% efficiency milestone},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Double-sided van der waals epitaxy across an atomic layer.
<em>NMAT</em>, <em>24</em>(3), 334–335. (<a
href="https://doi.org/10.1038/s41563-025-02131-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Double-sided van der Waals epitaxy across an ultrathin membrane provides a new method for growing a vertical junction with an atomically sharp interface, enabling the direct detection of selection rules for resonant tunnelling between topological surface states.},
  archive      = {J_NMAT},
  author       = {Feng, Xiao and Wang, Yayu},
  doi          = {10.1038/s41563-025-02131-y},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {334-335},
  shortjournal = {Nat. Mater.},
  title        = {Double-sided van der waals epitaxy across an atomic layer},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Magnetic order as a tuning knob for coulomb correlation.
<em>NMAT</em>, <em>24</em>(3), 332–333. (<a
href="https://doi.org/10.1038/s41563-025-02122-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antiferromagnetic order blocks interlayer hopping of electron–hole pairs in a two-dimensional magnetic semiconductor, leading to the formation of a type of optical excitation — magnetic surface excitons — with quasi-one-dimensional quantum confinement.},
  archive      = {J_NMAT},
  author       = {Duan, Jiahua and Zhou, Yixi},
  doi          = {10.1038/s41563-025-02122-z},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {332-333},
  shortjournal = {Nat. Mater.},
  title        = {Magnetic order as a tuning knob for coulomb correlation},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rolling carbon on a rock. <em>NMAT</em>, <em>24</em>(3),
330–331. (<a href="https://doi.org/10.1038/s41563-025-02151-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method is reported to create chiral rolls from two-dimensional atomic layers such as graphene with controlled rolling angles, which show optical activity and spin-selective transport dependent on the chiral lattice structures.},
  archive      = {J_NMAT},
  author       = {Yamamoto, Hiroshi M.},
  doi          = {10.1038/s41563-025-02151-8},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {330-331},
  shortjournal = {Nat. Mater.},
  title        = {Rolling carbon on a rock},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Layers split and zip for phase transition. <em>NMAT</em>,
<em>24</em>(3), 328–329. (<a
href="https://doi.org/10.1038/s41563-025-02148-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In situ high-resolution electron microscopy reveals that an electrically controlled metal-to-semiconductor phase transition in In2Se3 operates by intralayer splitting and interlayer zipping of atomic planes.},
  archive      = {J_NMAT},
  author       = {Funni, Stephen D. and Cha, Judy J.},
  doi          = {10.1038/s41563-025-02148-3},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {328-329},
  shortjournal = {Nat. Mater.},
  title        = {Layers split and zip for phase transition},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Reflections on progress in organic photovoltaics.
<em>NMAT</em>, <em>24</em>(3), 326–327. (<a
href="https://doi.org/10.1038/s41563-025-02161-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Jenny Nelson, a professor at Imperial College London (Department of Physics), talks to Nature Materials about recent research advances in organic photovoltaics.},
  archive      = {J_NMAT},
  author       = {Fan, Wei},
  doi          = {10.1038/s41563-025-02161-6},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {326-327},
  shortjournal = {Nat. Mater.},
  title        = {Reflections on progress in organic photovoltaics},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Lab-to-fab progress for organic photovoltaics.
<em>NMAT</em>, <em>24</em>(3), 324–325. (<a
href="https://doi.org/10.1038/s41563-025-02160-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Derya Baran, an associate professor at King Abdullah University of Science and Technology (Department of Materials Science and Engineering), talks to Nature Materials about the progress of laboratory-to-fabrication for organic photovoltaics},
  archive      = {J_NMAT},
  author       = {Fan, Wei},
  doi          = {10.1038/s41563-025-02160-7},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {324-325},
  shortjournal = {Nat. Mater.},
  title        = {Lab-to-fab progress for organic photovoltaics},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The role of non-fullerene acceptors continues.
<em>NMAT</em>, <em>24</em>(3), 323. (<a
href="https://doi.org/10.1038/s41563-025-02177-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Non-fullerene acceptors help organic solar cells achieve high performance, transforming organic photovoltaics into a useful technology.},
  archive      = {J_NMAT},
  doi          = {10.1038/s41563-025-02177-y},
  journal      = {Nature Materials},
  month        = {3},
  number       = {3},
  pages        = {323},
  shortjournal = {Nat. Mater.},
  title        = {The role of non-fullerene acceptors continues},
  volume       = {24},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="nmeth---31">NMETH - 31</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: A comprehensive human embryo reference
tool using single-cell RNA-sequencing data. <em>NMETH</em>,
<em>22</em>(3), 636. (<a
href="https://doi.org/10.1038/s41592-025-02601-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zhao, Cheng and Plaza Reyes, Alvaro and Schell, John Paul and Weltner, Jere and Ortega, Nicolás M. and Zheng, Yi and Björklund, Åsa K. and Baqué-Vidal, Laura and Sokka, Joonas and Trokovic, Ras and Cox, Brian and Rossant, Janet and Fu, Jianping and Petropoulos, Sophie and Lanner, Fredrik},
  doi          = {10.1038/s41592-025-02601-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {636},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A comprehensive human embryo reference tool using single-cell RNA-sequencing data},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Resolving tissue complexity by
multimodal spatial omics modeling with MISO. <em>NMETH</em>,
<em>22</em>(3), 635. (<a
href="https://doi.org/10.1038/s41592-025-02600-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Coleman, Kyle and Schroeder, Amelia and Loth, Melanie and Zhang, Daiwei and Park, Jeong Hwan and Sung, Ji-Youn and Blank, Niklas and Cowan, Alexis J. and Qian, Xuyu and Chen, Jianfeng and Jiang, Jiahui and Yan, Hanying and Samarah, Laith Z. and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Rabinowitz, Joshua D. and Deng, Yanxiang and Lee, Edward B. and Lazar, Alexander and Gao, Jianjun and Furth, Emma E. and Hwang, Tae Hyun and Wang, Linghua and Thaiss, Christoph A. and Hu, Jian and Li, Mingyao},
  doi          = {10.1038/s41592-025-02600-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {635},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Resolving tissue complexity by multimodal spatial omics modeling with MISO},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Arkitekt: Streaming analysis and
real-time workflows for microscopy. <em>NMETH</em>, <em>22</em>(3), 634.
(<a href="https://doi.org/10.1038/s41592-025-02597-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Roos, Johannes and Bancelin, Stéphane and Delaire, Tom and Wilhelmi, Alexander and Levet, Florian and Engelhardt, Maren and Viasnoff, Virgile and Galland, Rémi and Nägerl, U. Valentin and Sibarita, Jean-Baptiste},
  doi          = {10.1038/s41592-025-02597-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {634},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: arkitekt: streaming analysis and real-time workflows for microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A genome-wide atlas of human cell morphology.
<em>NMETH</em>, <em>22</em>(3), 621–633. (<a
href="https://doi.org/10.1038/s41592-024-02537-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key challenge of the modern genomics era is developing empirical data-driven representations of gene function. Here we present the first unbiased morphology-based genome-wide perturbation atlas in human cells, containing three genome-wide genotype–phenotype maps comprising CRISPR–Cas9-based knockouts of &amp;gt;20,000 genes in &amp;gt;30 million cells. Our optical pooled cell profiling platform (PERISCOPE) combines a destainable high-dimensional phenotyping panel (based on Cell Painting) with optical sequencing of molecular barcodes and a scalable open-source analysis pipeline to facilitate massively parallel screening of pooled perturbation libraries. This perturbation atlas comprises high-dimensional phenotypic profiles of individual cells with sufficient resolution to cluster thousands of human genes, reconstruct known pathways and protein–protein interaction networks, interrogate subcellular processes and identify culture media-specific responses. Using this atlas, we identify the poorly characterized disease-associated TMEM251/LYSET as a Golgi-resident transmembrane protein essential for mannose-6-phosphate-dependent trafficking of lysosomal enzymes. In sum, this perturbation atlas and screening platform represents a rich and accessible resource for connecting genes to cellular functions at scale. An optical pooled cell profiling platform (PERISCOPE) based on Cell Painting and optical sequencing of molecular barcodes was used to develop the first unbiased genome-wide morphology-based perturbation atlas in human cells.},
  archive      = {J_NMETH},
  author       = {Ramezani, Meraj and Weisbart, Erin and Bauman, Julia and Singh, Avtar and Yong, John and Lozada, Maria and Way, Gregory P. and Kavari, Sanam L. and Diaz, Celeste and Leardini, Eddy and Jetley, Gunjan and Pagnotta, Jenlu and Haghighi, Marzieh and Batista, Thiago M. and Pérez-Schindler, Joaquín and Claussnitzer, Melina and Singh, Shantanu and Cimini, Beth A. and Blainey, Paul C. and Carpenter, Anne E. and Jan, Calvin H. and Neal, James T.},
  doi          = {10.1038/s41592-024-02537-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {621-633},
  shortjournal = {Nat. Meth.},
  title        = {A genome-wide atlas of human cell morphology},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MARBLE: Interpretable representations of neural population
dynamics using geometric deep learning. <em>NMETH</em>, <em>22</em>(3),
612–620. (<a href="https://doi.org/10.1038/s41592-024-02582-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dynamics of neuron populations commonly evolve on low-dimensional manifolds. Thus, we need methods that learn the dynamical processes over neural manifolds to infer interpretable and consistent latent representations. We introduce a representation learning method, MARBLE, which decomposes on-manifold dynamics into local flow fields and maps them into a common latent space using unsupervised geometric deep learning. In simulated nonlinear dynamical systems, recurrent neural networks and experimental single-neuron recordings from primates and rodents, we discover emergent low-dimensional latent representations that parametrize high-dimensional neural dynamics during gain modulation, decision-making and changes in the internal state. These representations are consistent across neural networks and animals, enabling the robust comparison of cognitive computations. Extensive benchmarking demonstrates state-of-the-art within- and across-animal decoding accuracy of MARBLE compared to current representation learning approaches, with minimal user input. Our results suggest that a manifold structure provides a powerful inductive bias to develop decoding algorithms and assimilate data across experiments. MARBLE uses geometric deep learning to map dynamics such as neural activity into a latent representation, which can then be used to decode the neural activity or compare it across systems.},
  archive      = {J_NMETH},
  author       = {Gosztolai, Adam and Peach, Robert L. and Arnaudon, Alexis and Barahona, Mauricio and Vandergheynst, Pierre},
  doi          = {10.1038/s41592-024-02582-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {612-620},
  shortjournal = {Nat. Meth.},
  title        = {MARBLE: Interpretable representations of neural population dynamics using geometric deep learning},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A deep learning pipeline for three-dimensional brain-wide
mapping of local neuronal ensembles in teravoxel light-sheet microscopy.
<em>NMETH</em>, <em>22</em>(3), 600–611. (<a
href="https://doi.org/10.1038/s41592-024-02583-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Teravoxel-scale, cellular-resolution images of cleared rodent brains acquired with light-sheet fluorescence microscopy have transformed the way we study the brain. Realizing the potential of this technology requires computational pipelines that generalize across experimental protocols and map neuronal activity at the laminar and subpopulation-specific levels, beyond atlas-defined regions. Here, we present artficial intelligence-based cartography of ensembles (ACE), an end-to-end pipeline that employs three-dimensional deep learning segmentation models and advanced cluster-wise statistical algorithms, to enable unbiased mapping of local neuronal activity and connectivity. Validation against state-of-the-art segmentation and detection methods on unseen datasets demonstrated ACE’s high generalizability and performance. Applying ACE in two distinct neurobiological contexts, we discovered subregional effects missed by existing atlas-based analyses and showcase ACE’s ability to reveal localized or laminar neuronal activity brain-wide. Our open-source pipeline enables whole-brain mapping of neuronal ensembles at a high level of precision across a wide range of neuroscientific applications. The ACE pipeline utilized deep learning and advanced statistics for mapping neural activity at a granular level that is independent of atlas-defined regions.},
  archive      = {J_NMETH},
  author       = {Attarpour, Ahmadreza and Osmann, Jonas and Rinaldi, Anthony and Qi, Tianbo and Lal, Neeraj and Patel, Shruti and Rozak, Matthew and Yu, Fengqing and Cho, Newton and Squair, Jordan and McLaurin, JoAnne and Raffiee, Misha and Deisseroth, Karl and Courtine, Gregoire and Ye, Li and Stefanovic, Bojana and Goubran, Maged},
  doi          = {10.1038/s41592-024-02583-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {600-611},
  shortjournal = {Nat. Meth.},
  title        = {A deep learning pipeline for three-dimensional brain-wide mapping of local neuronal ensembles in teravoxel light-sheet microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cellpose3: One-click image restoration for improved cellular
segmentation. <em>NMETH</em>, <em>22</em>(3), 592–599. (<a
href="https://doi.org/10.1038/s41592-025-02595-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generalist methods for cellular segmentation have good out-of-the-box performance on a variety of image types; however, existing methods struggle for images that are degraded by noise, blurring or undersampling, all of which are common in microscopy. We focused the development of Cellpose3 on addressing these cases and here we demonstrate substantial out-of-the-box gains in segmentation and image quality for noisy, blurry and undersampled images. Unlike previous approaches that train models to restore pixel values, we trained Cellpose3 to output images that are well segmented by a generalist segmentation model, while maintaining perceptual similarity to the target images. Furthermore, we trained the restoration models on a large, varied collection of datasets, thus ensuring good generalization to user images. We provide these tools as ‘one-click’ buttons inside the graphical interface of Cellpose as well as in the Cellpose API. Cellpose3 employs deep-learning-based approaches for image restoration to improve cellular segmentation and shows strong generalized performance even on images degraded by noise, blurring or undersampling.},
  archive      = {J_NMETH},
  author       = {Stringer, Carsen and Pachitariu, Marius},
  doi          = {10.1038/s41592-025-02595-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {592-599},
  shortjournal = {Nat. Meth.},
  title        = {Cellpose3: One-click image restoration for improved cellular segmentation},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Segment anything for microscopy. <em>NMETH</em>,
<em>22</em>(3), 579–591. (<a
href="https://doi.org/10.1038/s41592-024-02580-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate segmentation of objects in microscopy images remains a bottleneck for many researchers despite the number of tools developed for this purpose. Here, we present Segment Anything for Microscopy (μSAM), a tool for segmentation and tracking in multidimensional microscopy data. It is based on Segment Anything, a vision foundation model for image segmentation. We extend it by fine-tuning generalist models for light and electron microscopy that clearly improve segmentation quality for a wide range of imaging conditions. We also implement interactive and automatic segmentation in a napari plugin that can speed up diverse segmentation tasks and provides a unified solution for microscopy annotation across different microscopy modalities. Our work constitutes the application of vision foundation models in microscopy, laying the groundwork for solving image analysis tasks in this domain with a small set of powerful deep learning models. Segment Anything for Microscopy (μSAM) builds on the vision foundation model Segment Anything for high-quality image segmentation over a wide range of imaging conditions including light and electron microscopy.},
  archive      = {J_NMETH},
  author       = {Archit, Anwai and Freckmann, Luca and Nair, Sushmita and Khalid, Nabeel and Hilt, Paul and Rajashekar, Vikas and Freitag, Marei and Teuber, Carolin and Buckley, Genevieve and von Haaren, Sebastian and Gupta, Sagnik and Dengel, Andreas and Ahmed, Sheraz and Pape, Constantin},
  doi          = {10.1038/s41592-024-02580-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {579-591},
  shortjournal = {Nat. Meth.},
  title        = {Segment anything for microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Imaging of cellular dynamics from a whole organism to
subcellular scale with self-driving, multiscale microscopy.
<em>NMETH</em>, <em>22</em>(3), 569–578. (<a
href="https://doi.org/10.1038/s41592-025-02598-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most biological processes, from development to pathogenesis, span multiple time and length scales. While light-sheet fluorescence microscopy has become a fast and efficient method for imaging organisms, cells and subcellular dynamics, simultaneous observations across all these scales have remained challenging. Moreover, continuous high-resolution imaging inside living organisms has mostly been limited to a few hours, as regions of interest quickly move out of view due to sample movement and growth. Here, we present a self-driving, multiresolution light-sheet microscope platform controlled by custom Python-based software, to simultaneously observe and quantify subcellular dynamics in the context of entire organisms in vitro and in vivo over hours of imaging. We apply the platform to the study of developmental processes, cancer invasion and metastasis, and we provide quantitative multiscale analysis of immune–cancer cell interactions in zebrafish xenografts. A self-driving multiresolution light-sheet microscope enables the simultaneous observation and quantification of cellular and subcellular dynamics in the context of intact and developing organisms over many hours of imaging.},
  archive      = {J_NMETH},
  author       = {Daetwyler, Stephan and Mazloom-Farsibaf, Hanieh and Zhou, Felix Y. and Segal, Dagan and Sapoznik, Etai and Chen, Bingying and Westcott, Jill M. and Brekken, Rolf A. and Danuser, Gaudenz and Fiolka, Reto},
  doi          = {10.1038/s41592-025-02598-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {569-578},
  shortjournal = {Nat. Meth.},
  title        = {Imaging of cellular dynamics from a whole organism to subcellular scale with self-driving, multiscale microscopy},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oblique line scan illumination enables expansive, accurate
and sensitive single-protein measurements in solution and in living
cells. <em>NMETH</em>, <em>22</em>(3), 559–568. (<a
href="https://doi.org/10.1038/s41592-025-02594-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An ideal tool for the study of cellular biology would enable the measure of molecular activity nondestructively within living cells. Single-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells but have historically been limited by a necessary tradeoff between spatiotemporal resolution and throughput. Here we address these limitations using oblique line scan (OLS), a robust single-objective light-sheet-based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a large field of view. We show that OLS can be used to capture protein motion up to 14 μm2 s−1 in living cells. We further extend the utility of OLS with in-solution SMT for single-molecule measurement of ligand–protein interactions and disruption of protein–protein interactions using purified proteins. We illustrate the versatility of OLS by showcasing two-color SMT, STORM and single-molecule fluorescence recovery after photobleaching. OLS paves the way for robust, high-throughput, single-molecule investigations of protein function required for basic research, drug screening and systems biology studies. Oblique line scan microscopy achieves nanoscale spatial and sub-millisecond temporal resolution across a large field of view, enabling improved and robust single-molecule biophysical measurements and single-molecule tracking in both cells and solution.},
  archive      = {J_NMETH},
  author       = {Driouchi, Amine and Bretan, Mason and Davis, Brynmor J. and Heckert, Alec and Seeger, Markus and Silva, Maité Bradley and Forrest, William S. R. and Hsiung, Jessica and Tan, Jiongyi and Yang, Hongli and McSwiggen, David T. and Song, Linda and Sule, Askhay and Abaie, Behnam and Chen, Hanzhe and Chhun, Bryant and Conroy, Brianna and Elliott, Liam A. and Gonzalez, Eric and Ilkov, Fedor and Isaacs, Joshua and Labaria, George and Lagana, Michelle and Larsen, DeLaine D. and Margolin, Brian and Nguyen, Mai K. and Park, Eugene and Rine, Jeremy and Tang, Yangzhong and Vana, Martin and Wilkey, Andrew and Zhang, Zhengjian and Basham, Stephen and Ho, Jaclyn J. and Johnson, Stephanie and Klammer, Aaron A. and Lin, Kevin and Darzacq, Xavier and Betzig, Eric and Berman, Russell T. and Anderson, Daniel J.},
  doi          = {10.1038/s41592-025-02594-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {559-568},
  shortjournal = {Nat. Meth.},
  title        = {Oblique line scan illumination enables expansive, accurate and sensitive single-protein measurements in solution and in living cells},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super-photostable organic dye for long-term live-cell
single-protein imaging. <em>NMETH</em>, <em>22</em>(3), 550–558. (<a
href="https://doi.org/10.1038/s41592-024-02584-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organic dyes play a crucial role in live-cell imaging because of their advantageous properties, such as photostability and high brightness. Here we introduce a super-photostable and bright organic dye, Phoenix Fluor 555 (PF555), which exhibits an order-of-magnitude longer photobleaching lifetime than conventional organic dyes without the requirement of any anti-photobleaching additives. PF555 is an asymmetric cyanine structure in which, on one side, the indole in the conventional Cyanine-3 is substituted with 3-oxo-quinoline. PF555 provides a powerful tool for long-term live-cell single-molecule imaging, as demonstrated by the imaging of the dynamic single-molecule interactions of the epidermal growth factor receptor with clathrin-coated structures on the plasma membrane of a live cell under physiological conditions. Phoenix Fluor 555 (PF555) is a bright dye with an exceptional, order-of-magnitude longer photobleaching lifetime than conventional organic dyes that enables extended live-cell single-molecule imaging without anti-photobleaching additives.},
  archive      = {J_NMETH},
  author       = {Kim, Do-Hyeon and Triet, Hong Minh and Lee, Sun Hyeok and Jazani, Sina and Jang, Seongjae and Abedi, Syed Ali Abbas and Liu, Xiaogang and Seo, Jongcheol and Ha, Taekjip and Chang, Young-Tae and Ryu, Sung Ho},
  doi          = {10.1038/s41592-024-02584-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {550-558},
  shortjournal = {Nat. Meth.},
  title        = {Super-photostable organic dye for long-term live-cell single-protein imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A temperature-inducible protein module for control of
mammalian cell fate. <em>NMETH</em>, <em>22</em>(3), 539–549. (<a
href="https://doi.org/10.1038/s41592-024-02572-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inducible protein switches are currently limited for use in tissues and organisms because common inducers cannot be controlled with precision in space and time in optically dense settings. Here, we introduce a protein that can be reversibly toggled with a small change in temperature, a stimulus that is both penetrant and dynamic. This protein, called Melt (Membrane localization using temperature) oligomerizes and translocates to the plasma membrane when temperature is lowered. We generated a library of Melt variants with switching temperatures ranging from 30 °C to 40 °C, including two that operate at and above 37 °C. Melt was a highly modular actuator of cell function, permitting thermal control over diverse processes including signaling, proteolysis, nuclear shuttling, cytoskeletal rearrangements and cell death. Finally, Melt permitted thermal control of cell death in a mouse model of human cancer. Melt represents a versatile thermogenetic module for straightforward, non-invasive and spatiotemporally defined control of mammalian cells with broad potential for biotechnology and biomedicine. The Melt (Membrane localization using temperature) protein translocates to the plasma membrane upon temperature shift. Melt variants with a range of switching temperatures enable straightforward thermogenetic control of diverse cellular processes.},
  archive      = {J_NMETH},
  author       = {Benman, William and Huang, Zikang and Iyengar, Pavan and Wilde, Delaney and Mumford, Thomas R. and Bugaj, Lukasz J.},
  doi          = {10.1038/s41592-024-02572-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {539-549},
  shortjournal = {Nat. Meth.},
  title        = {A temperature-inducible protein module for control of mammalian cell fate},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Resolving tissue complexity by multimodal spatial omics
modeling with MISO. <em>NMETH</em>, <em>22</em>(3), 530–538. (<a
href="https://doi.org/10.1038/s41592-024-02574-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial molecular profiling has provided biomedical researchers valuable opportunities to better understand the relationship between cellular localization and tissue function. Effectively modeling multimodal spatial omics data is crucial for understanding tissue complexity and underlying biology. Furthermore, improvements in spatial resolution have led to the advent of technologies that can generate spatial molecular data with subcellular resolution, requiring the development of computationally efficient methods that can handle the resulting large-scale datasets. MISO (MultI-modal Spatial Omics) is a versatile algorithm for feature extraction and clustering, capable of integrating multiple modalities from diverse spatial omics experiments with high spatial resolution. Its effectiveness is demonstrated across various datasets, encompassing gene expression, protein expression, epigenetics, metabolomics and tissue histology modalities. MISO outperforms existing methods in identifying biologically relevant spatial domains, representing a substantial advancement in multimodal spatial omics analysis. Moreover, MISO’s computational efficiency ensures its scalability to handle large-scale datasets generated by subcellular resolution spatial omics technologies. MISO (MultI-modal Spatial Omics) integrates two or more spatial omics modalities, despite differences in data quality and spatial resolution for improved feature extraction and clustering to reveal biologically meaningful tissue organization.},
  archive      = {J_NMETH},
  author       = {Coleman, Kyle and Schroeder, Amelia and Loth, Melanie and Zhang, Daiwei and Park, Jeong Hwan and Sung, Ji-Youn and Blank, Niklas and Cowan, Alexis J. and Qian, Xuyu and Chen, Jianfeng and Jiang, Jiahui and Yan, Hanying and Samarah, Laith Z. and Clemenceau, Jean R. and Jang, Inyeop and Kim, Minji and Barnfather, Isabel and Rabinowitz, Joshua D. and Deng, Yanxiang and Lee, Edward B. and Lazar, Alexander and Gao, Jianjun and Furth, Emma E. and Hwang, Tae Hyun and Wang, Linghua and Thaiss, Christoph A. and Hu, Jian and Li, Mingyao},
  doi          = {10.1038/s41592-024-02574-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {530-538},
  shortjournal = {Nat. Meth.},
  title        = {Resolving tissue complexity by multimodal spatial omics modeling with MISO},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiplexed spatial mapping of chromatin features,
transcriptome and proteins in tissues. <em>NMETH</em>, <em>22</em>(3),
520–529. (<a href="https://doi.org/10.1038/s41592-024-02576-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The phenotypic and functional states of cells are modulated by a complex interactive molecular hierarchy of multiple omics layers, involving the genome, epigenome, transcriptome, proteome and metabolome. Spatial omics approaches have enabled the study of these layers in tissue context but are often limited to one or two modalities, offering an incomplete view of cellular identity. Here we present spatial-Mux-seq, a multimodal spatial technology that allows simultaneous profiling of five different modalities: two histone modifications, chromatin accessibility, whole transcriptome and a panel of proteins at tissue scale and cellular level in a spatially resolved manner. We applied this technology to mouse embryos and mouse brains, generating detailed multimodal tissue maps that identified more cell types and states compared to unimodal data. This analysis uncovered spatiotemporal relationships among histone modifications, chromatin accessibility, gene expression and protein levels during neuron differentiation, and revealed a radial glia niche with spatially dynamic epigenetic signals. Collectively, the spatial multi-omics approach heralds a new era for characterizing tissue and cellular heterogeneity that single-modality studies alone could not reveal. Spatial-Mux-seq offers a multimodal spatial platform capable of profiling multiple molecular modalities, including the transcriptome, chromatin accessibility, histone modifications and targeted proteins.},
  archive      = {J_NMETH},
  author       = {Guo, Pengfei and Mao, Liran and Chen, Yufan and Lee, Chin Nien and Cardilla, Angelysia and Li, Mingyao and Bartosovic, Marek and Deng, Yanxiang},
  doi          = {10.1038/s41592-024-02576-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {520-529},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed spatial mapping of chromatin features, transcriptome and proteins in tissues},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Challenging the astral mass analyzer to quantify up to 5,300
proteins per single cell at unseen accuracy to uncover cellular
heterogeneity. <em>NMETH</em>, <em>22</em>(3), 510–519. (<a
href="https://doi.org/10.1038/s41592-024-02559-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite significant advancements in sample preparation, instrumentation and data analysis, single-cell proteomics is currently limited by proteomic depth and quantitative performance. Here we demonstrate highly improved depth of proteome coverage as well as accuracy and precision for quantification of ultra-low input amounts. Using a tailored library, we identify up to 7,400 protein groups from as little as 250 pg of HeLa cell peptides at a throughput of 50 samples per day. Using a two-proteome mix, we check for optimal parameters of quantification and show that fold change differences of 2 can still be successfully determined at single-cell-level inputs. Eventually, we apply our workflow to A549 cells, yielding a proteome coverage ranging from 1,801 to a maximum of &amp;gt;5,300 protein groups from a single cell depending on cell size and search strategy used, which allows for the study of dependencies between cell size and cell cycle phase. Additionally, our workflow enables us to distinguish between in vitro analogs of two human blastocyst lineages: naive human pluripotent stem cells (epiblast) and trophectoderm-like cells. Our data harmoniously align with transcriptomic data, indicating that single-cell proteomics possesses the capability to identify biologically relevant differences within the blastocyst. An ultra-low-input single-cell proteomic workflow is optimized to yield maximum proteome coverage with high accuracy and precision on the Orbitrap Astral mass spectrometer.},
  archive      = {J_NMETH},
  author       = {Bubis, Julia A. and Arrey, Tabiwang N. and Damoc, Eugen and Delanghe, Bernard and Slovakova, Jana and Sommer, Theresa M. and Kagawa, Harunobu and Pichler, Peter and Rivron, Nicolas and Mechtler, Karl and Matzinger, Manuel},
  doi          = {10.1038/s41592-024-02559-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {510-519},
  shortjournal = {Nat. Meth.},
  title        = {Challenging the astral mass analyzer to quantify up to 5,300 proteins per single cell at unseen accuracy to uncover cellular heterogeneity},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhanced sensitivity and scalability with a chip-tip
workflow enables deep single-cell proteomics. <em>NMETH</em>,
<em>22</em>(3), 499–509. (<a
href="https://doi.org/10.1038/s41592-024-02558-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell proteomics (SCP) promises to revolutionize biomedicine by providing an unparalleled view of the proteome in individual cells. Here, we present a high-sensitivity SCP workflow named Chip-Tip, identifying &amp;gt;5,000 proteins in individual HeLa cells. It also facilitated direct detection of post-translational modifications in single cells, making the need for specific post-translational modification-enrichment unnecessary. Our study demonstrates the feasibility of processing up to 120 label-free SCP samples per day. An optimized tissue dissociation buffer enabled effective single-cell disaggregation of drug-treated cancer cell spheroids, refining overall SCP analysis. Analyzing nondirected human-induced pluripotent stem cell differentiation, we consistently quantified stem cell markers OCT4 and SOX2 in human-induced pluripotent stem cells and lineage markers such as GATA4 (endoderm), HAND1 (mesoderm) and MAP2 (ectoderm) in different embryoid body cells. Our workflow sets a benchmark in SCP for sensitivity and throughput, with broad applications in basic biology and biomedicine for identification of cell type-specific markers and therapeutic targets. Chip-Tip is a label-free quantification-based single-cell proteomics workflow for deep single-cell proteomics, which identifies over 5,000 proteins and 40,000 peptides in single HeLa cells.},
  archive      = {J_NMETH},
  author       = {Ye, Zilu and Sabatier, Pierre and van der Hoeven, Leander and Lechner, Maico Y. and Phlairaharn, Teeradon and Guzman, Ulises H. and Liu, Zhen and Huang, Haoran and Huang, Min and Li, Xiangjun and Hartlmayr, David and Izaguirre, Fabiana and Seth, Anjali and Joshi, Hiren J. and Rodin, Sergey and Grinnemo, Karl-Henrik and Hørning, Ole B. and Bekker-Jensen, Dorte B. and Bache, Nicolai and Olsen, Jesper V.},
  doi          = {10.1038/s41592-024-02558-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {499-509},
  shortjournal = {Nat. Meth.},
  title        = {Enhanced sensitivity and scalability with a chip-tip workflow enables deep single-cell proteomics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-resolution, noninvasive single-cell lineage tracing in
mice and humans based on DNA methylation epimutations. <em>NMETH</em>,
<em>22</em>(3), 488–498. (<a
href="https://doi.org/10.1038/s41592-024-02567-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vivo lineage tracing holds great potential to reveal fundamental principles of tissue development and homeostasis. However, current lineage tracing in humans relies on extremely rare somatic mutations, which has limited temporal resolution and lineage accuracy. Here, we developed a generic lineage-tracing tool based on frequent epimutations on DNA methylation, enabled by our computational method MethylTree. Using single-cell genome-wide DNA methylation datasets with known lineage and phenotypic labels, MethylTree reconstructed lineage histories at nearly 100% accuracy across different cell types, developmental stages, and species. We demonstrated the epimutation-based single-cell multi-omic lineage tracing in mouse and human blood, where MethylTree recapitulated the differentiation hierarchy in hematopoiesis. Applying MethylTree to human embryos, we revealed early fate commitment at the four-cell stage. In native mouse blood, we identified ~250 clones of hematopoietic stem cells. MethylTree opens the door for high-resolution, noninvasive and multi-omic lineage tracing in humans and beyond. This work presents a computational tool MethylTree to infer cell lineages based on epimutations on DNA methylation.},
  archive      = {J_NMETH},
  author       = {Chen, Mengyang and Fu, Ruijiang and Chen, Yiqian and Li, Li and Wang, Shou-Wen},
  doi          = {10.1038/s41592-024-02567-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {488-498},
  shortjournal = {Nat. Meth.},
  title        = {High-resolution, noninvasive single-cell lineage tracing in mice and humans based on DNA methylation epimutations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scalable co-sequencing of RNA and DNA from individual
nuclei. <em>NMETH</em>, <em>22</em>(3), 477–487. (<a
href="https://doi.org/10.1038/s41592-024-02579-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ideal technology for directly investigating the relationship between genotype and phenotype would analyze both RNA and DNA genome-wide and with single-cell resolution; however, existing tools lack the throughput required for comprehensive analysis of complex tumors and tissues. We introduce a highly scalable method for jointly profiling DNA and expression following nucleosome depletion (DEFND-seq). In DEFND-seq, nuclei are nucleosome-depleted, tagmented and separated into individual droplets for messenger RNA and genomic DNA barcoding. Once nuclei have been depleted of nucleosomes, subsequent steps can be performed using the widely available 10x Genomics droplet microfluidic technology and commercial kits. We demonstrate the production of high-complexity mRNA and gDNA sequencing libraries from thousands of individual nuclei from cell lines, fresh and archived surgical specimens for associating gene expression with both copy number and single-nucleotide variants. Building on a nucleosome-depletion strategy, DEFND-seq utilizes a droplet microfluidic platform to enable high-throughput co-profiling of DNA and RNA in single cells.},
  archive      = {J_NMETH},
  author       = {Olsen, Timothy R. and Talla, Pranay and Sagatelian, Romella K. and Furnari, Julia and Bruce, Jeffrey N. and Canoll, Peter and Zha, Shan and Sims, Peter A.},
  doi          = {10.1038/s41592-024-02579-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {477-487},
  shortjournal = {Nat. Meth.},
  title        = {Scalable co-sequencing of RNA and DNA from individual nuclei},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DeepPrep: An accelerated, scalable and robust pipeline for
neuroimaging preprocessing empowered by deep learning. <em>NMETH</em>,
<em>22</em>(3), 473–476. (<a
href="https://doi.org/10.1038/s41592-025-02599-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuroimaging has entered the era of big data. However, the advancement of preprocessing pipelines falls behind the rapid expansion of data volume, causing substantial computational challenges. Here we present DeepPrep, a pipeline empowered by deep learning and a workflow manager. Evaluated on over 55,000 scans, DeepPrep demonstrates tenfold acceleration, scalability and robustness compared to the state-of-the-art pipeline, thereby meeting the scalability requirements of neuroimaging. DeepPrep is a preprocessing pipeline for functional and structural MRI data from humans. Deep learning-based modules and an efficient workflow allow DeepPrep to handle large datasets.},
  archive      = {J_NMETH},
  author       = {Ren, Jianxun and An, Ning and Lin, Cong and Zhang, Youjia and Sun, Zhenyu and Zhang, Wei and Li, Shiyi and Guo, Ning and Cui, Weigang and Hu, Qingyu and Wang, Weiwei and Wu, Xuehai and Wang, Yinyan and Jiang, Tao and Satterthwaite, Theodore D. and Wang, Danhong and Liu, Hesheng},
  doi          = {10.1038/s41592-025-02599-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {473-476},
  shortjournal = {Nat. Meth.},
  title        = {DeepPrep: An accelerated, scalable and robust pipeline for neuroimaging preprocessing empowered by deep learning},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid and sensitive protein complex alignment with
foldseek-multimer. <em>NMETH</em>, <em>22</em>(3), 469–472. (<a
href="https://doi.org/10.1038/s41592-025-02593-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in computational structure prediction will vastly augment the hundreds of thousands of currently available protein complex structures. Translating these into discoveries requires aligning them, which is computationally prohibitive. Foldseek-Multimer computes complex alignments from compatible chain-to-chain alignments, identified by efficiently clustering their superposition vectors. Foldseek-Multimer is 3–4 orders of magnitudes faster than the gold standard, while producing comparable alignments; this allows it to compare billions of complex pairs in 11 h. Foldseek-Multimer is open-source software available at GitHub via https://github.com/steineggerlab/foldseek/ , https://search.foldseek.com/search/ and the BFMD database. Foldseek-Multimer offers a fast strategy for complex-to-complex alignment to quickly identify compatible sets of chain-to-chain alignments by their superpositions. It can compare billions of complex pairs in 11 h.},
  archive      = {J_NMETH},
  author       = {Kim, Woosub and Mirdita, Milot and Levy Karin, Eli and Gilchrist, Cameron L. M. and Schweke, Hugo and Söding, Johannes and Levy, Emmanuel D. and Steinegger, Martin},
  doi          = {10.1038/s41592-025-02593-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {469-472},
  shortjournal = {Nat. Meth.},
  title        = {Rapid and sensitive protein complex alignment with foldseek-multimer},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Interpreting and comparing neural activity across systems by
geometric deep learning. <em>NMETH</em>, <em>22</em>(3), 467–468. (<a
href="https://doi.org/10.1038/s41592-024-02581-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The MARBLE method addresses a critical challenge in neural population recordings: inferring expressive and interpretable latent representations that are comparable across experiments and animals. It achieves this by explicitly leveraging the low-dimensional structure of neural states through geometric deep learning to learn the dynamical flow fields in neural activity.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02581-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {467-468},
  shortjournal = {Nat. Meth.},
  title        = {Interpreting and comparing neural activity across systems by geometric deep learning},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precision control of cellular functions with a
temperature-sensitive protein. <em>NMETH</em>, <em>22</em>(3), 465–466.
(<a href="https://doi.org/10.1038/s41592-024-02573-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Temperature-sensitive proteins would enable the remote control of cellular functions deep within tissues, although few such proteins have been characterized. Melt is a protein that reversibly clusters and translocates to the membrane in response to small temperature changes, enabling flexible regulation of cellular processes in vitro and in vivo.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02573-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {465-466},
  shortjournal = {Nat. Meth.},
  title        = {Precision control of cellular functions with a temperature-sensitive protein},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MethylTree: Exploring epimutations for accurate and
non-invasive lineage tracing. <em>NMETH</em>, <em>22</em>(3), 463–464.
(<a href="https://doi.org/10.1038/s41592-024-02568-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Non-invasive lineage tracing in humans relies on rare somatic mutations, which have limited throughput and temporal resolution. We developed a computational method, ‘MethylTree’, which uses epimutations on DNA methylation to accurately infer lineages across cell types, developmental stages and species, providing a superior alternative for non-invasive lineage tracing in humans and other organisms.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02568-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {463-464},
  shortjournal = {Nat. Meth.},
  title        = {MethylTree: Exploring epimutations for accurate and non-invasive lineage tracing},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The next generation of in situ multi-omics. <em>NMETH</em>,
<em>22</em>(3), 461–462. (<a
href="https://doi.org/10.1038/s41592-024-02571-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial-Mux-seq adds to a growing portfolio of innovative spatial omics technologies and enables simultaneous profiling of up to five omic modalities in situ.},
  archive      = {J_NMETH},
  author       = {Salla, Maren and Penkert, Klara and Ludwig, Leif S.},
  doi          = {10.1038/s41592-024-02571-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {461-462},
  shortjournal = {Nat. Meth.},
  title        = {The next generation of in situ multi-omics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Entering the era of deep single-cell proteomics.
<em>NMETH</em>, <em>22</em>(3), 459–460. (<a
href="https://doi.org/10.1038/s41592-025-02620-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in single-cell proteomics allow quantification of half of the expressed proteome in an individual cell.},
  archive      = {J_NMETH},
  author       = {Sinn, Ludwig R. and Demichev, Vadim},
  doi          = {10.1038/s41592-025-02620-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {459-460},
  shortjournal = {Nat. Meth.},
  title        = {Entering the era of deep single-cell proteomics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vector choices, vector surprises. <em>NMETH</em>,
<em>22</em>(3), 454–458. (<a
href="https://doi.org/10.1038/s41592-025-02609-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vectors can be the ultimate vehicle for transporting material into cells. Except when they’re not so ultimate.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02609-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {454-458},
  shortjournal = {Nat. Meth.},
  title        = {Vector choices, vector surprises},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural biology at the plasma membrane. <em>NMETH</em>,
<em>22</em>(3), 453. (<a
href="https://doi.org/10.1038/s41592-025-02640-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-025-02640-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {453},
  shortjournal = {Nat. Meth.},
  title        = {Structural biology at the plasma membrane},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Strolling down rats’ memory lane. <em>NMETH</em>,
<em>22</em>(3), 453. (<a
href="https://doi.org/10.1038/s41592-025-02641-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-025-02641-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {453},
  shortjournal = {Nat. Meth.},
  title        = {Strolling down rats’ memory lane},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chemical space exploration with quantum computing.
<em>NMETH</em>, <em>22</em>(3), 452. (<a
href="https://doi.org/10.1038/s41592-025-02638-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02638-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {452},
  shortjournal = {Nat. Meth.},
  title        = {Chemical space exploration with quantum computing},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optimal transport for single-cell genomics. <em>NMETH</em>,
<em>22</em>(3), 452. (<a
href="https://doi.org/10.1038/s41592-025-02639-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-025-02639-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {452},
  shortjournal = {Nat. Meth.},
  title        = {Optimal transport for single-cell genomics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A look back at embryo models. <em>NMETH</em>,
<em>22</em>(3), 449–450. (<a
href="https://doi.org/10.1038/s41592-025-02642-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In 2023, Nature Methods chose methods to model development as our Method of the Year. Here, we catch up on what has happened in this field since.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02642-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {449-450},
  shortjournal = {Nat. Meth.},
  title        = {A look back at embryo models},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>
<h2 id="nphys---39">NPHYS - 39</h2>
<ul>
<li><details>
<summary>
(2025). Never judge a book by its cover. <em>NPHYS</em>,
<em>21</em>(3), 487. (<a
href="https://doi.org/10.1038/s41567-025-02794-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The SI brochure has described the global measurement system for more than 50 years, and yet it has kept a low profile. Richard Brown leafs through its history.},
  archive      = {J_NPHYS},
  author       = {Brown, Richard J. C.},
  doi          = {10.1038/s41567-025-02794-z},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {487},
  shortjournal = {Nat. Phys.},
  title        = {Never judge a book by its cover},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Real-space imaging of phase transitions
in bridged artificial kagome spin ice. <em>NPHYS</em>, <em>21</em>(3),
486. (<a href="https://doi.org/10.1038/s41567-024-02747-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Hofhuis, Kevin and Skjærvø, Sandra Helen and Parchenko, Sergii and Arava, Hanu and Luo, Zhaochu and Kleibert, Armin and Derlet, Peter Michael and Heyderman, Laura Jane},
  doi          = {10.1038/s41567-024-02747-y},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Phys.},
  title        = {Author correction: Real-space imaging of phase transitions in bridged artificial kagome spin ice},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Heavy-tailed neuronal connectivity
arises from hebbian self-organization. <em>NPHYS</em>, <em>21</em>(3),
486. (<a href="https://doi.org/10.1038/s41567-024-02748-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Lynn, Christopher W. and Holmes, Caroline M. and Palmer, Stephanie E.},
  doi          = {10.1038/s41567-024-02748-x},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Phys.},
  title        = {Publisher correction: Heavy-tailed neuronal connectivity arises from hebbian self-organization},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Publisher correction: Unified percolation scenario for the
α and β processes in simple glass formers. <em>NPHYS</em>,
<em>21</em>(3), 486. (<a
href="https://doi.org/10.1038/s41567-025-02860-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Gao, Liang and Yu, Hai-Bin and Schrøder, Thomas B. and Dyre, Jeppe C.},
  doi          = {10.1038/s41567-025-02860-6},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Phys.},
  title        = {Publisher correction: Unified percolation scenario for the α and β processes in simple glass formers},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Constraints on the location of the liquid–liquid critical
point in water. <em>NPHYS</em>, <em>21</em>(3), 480–485. (<a
href="https://doi.org/10.1038/s41567-024-02761-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The fascinating hypothesis that supercooled water may segregate into two distinct liquid phases, each with unique structures and densities, was first posited in 1992. This idea, initially based on numerical analyses with the ST2 water-like empirical potential, challenged the conventional understanding of water’s phase behaviour at the time and has since intrigued the scientific community. Over the past three decades, advancements in computational modelling—particularly through the advent of data-driven many-body potentials rigorously derived from first principles and augmented by the efficiency of neural networks—have greatly enhanced the accuracy of molecular simulations, enabling the exploration of the phase behaviour of water with unprecedented realism. Our study leverages these computational advances to probe the elusive liquid–liquid transition in supercooled water. Microsecond-long simulations with chemical accuracy, conducted over several years, provide compelling evidence that water indeed exists in two discernibly distinct liquid states at low temperature and high pressure. By pinpointing a realistic estimate for the location of the liquid–liquid critical point at ~198 K and ~1,250 atm, our study not only advances the current understanding of water’s anomalous behaviour but also establishes a basis for experimental validation. Importantly, our simulations indicate that the liquid–liquid critical point falls within temperature and pressure ranges that could potentially be experimentally probed in water nanodroplets, opening up the possibility for direct measurements. A liquid–liquid transition in supercooled water has long been predicted. State-of-the-art simulations now precisely confine the temperature and pressure ranges for this transition, which are found to be within experimental reach.},
  archive      = {J_NPHYS},
  author       = {Sciortino, F. and Zhai, Y. and Bore, S. L. and Paesani, F.},
  doi          = {10.1038/s41567-024-02761-0},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {480-485},
  shortjournal = {Nat. Phys.},
  title        = {Constraints on the location of the liquid–liquid critical point in water},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Unified percolation scenario for the α and β processes in
simple glass formers. <em>NPHYS</em>, <em>21</em>(3), 471–479. (<a
href="https://doi.org/10.1038/s41567-024-02762-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Given the vast differences in interaction details, describing the dynamics of structurally disordered materials in a unified theoretical framework presents a fundamental challenge to condensed-matter physics and materials science. Here we numerically investigate a double-percolation scenario for the two most important relaxation processes of supercooled liquids and glasses, the so-called α and β relaxations. For several simple glass formers, we find that when monitoring the dynamic shear modulus as temperature is lowered from the liquid state, percolation of immobile particles takes place at the temperature locating the α process. Mirroring this, upon continued cooling into the glass state, the mobile-particle percolation transition pinpoints a β process whenever the latter is well separated from the main (α) process. For two-dimensional systems under the same conditions, percolation of mobile and immobile particles occurs nearly simultaneously, and no β relaxation can be identified. Our findings suggest that a general description of glassy dynamics should be based on a percolation perspective. A unified description of the dynamics of structurally disordered materials is challenging. Simulations of model systems now show that percolation theory provides a framework unifying the two most prominent relaxation processes in supercooled liquids and glasses.},
  archive      = {J_NPHYS},
  author       = {Gao, Liang and Yu, Hai-Bin and Schrøder, Thomas B. and Dyre, Jeppe C.},
  doi          = {10.1038/s41567-024-02762-z},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {471-479},
  shortjournal = {Nat. Phys.},
  title        = {Unified percolation scenario for the α and β processes in simple glass formers},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Thermal triggering for multi-state switching of polar
topologies. <em>NPHYS</em>, <em>21</em>(3), 464–470. (<a
href="https://doi.org/10.1038/s41567-024-02729-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Particle-like topological structures such as polar skyrmions in ferroelectrics have the potential for application in high-density information storage. Since the polar topologies arise from a complicated competitive energy balance, such non-trivial topological states are difficult to manipulate by applying non-persistent external stimuli, such as bias or strain. Thus, a flexible strategy for manipulating topological polar states is needed to realize ultrahigh-density topological devices. Here we demonstrate that thermal excitation can simultaneously regulate the competition of elastic, electrostatic, polarization gradient and Landau energies to trigger polar topological state switching. By designing the temperature evolution pathways, the individual states that are believed to be unstable or intermediate can now be switched and stabilized. Therefore, our strategy expands the diversity of polar topologies in a single superlattice system. Furthermore, we demonstrate the laser-based thermal local switching of polar solitons ranging from several hundred nanometres to a few topologies. These findings will advance the design of polar topology-based ultrahigh-density storage. Stable manipulation of polar skyrmions is challenging because of the underlying competitive energy scales. Now thermal excitation has been demonstrated to be an effective way to control such topological states.},
  archive      = {J_NPHYS},
  author       = {Tong, Peiran and Zhou, Linming and Du, Kai and Zhang, Meng and Sun, Yuting and Sun, Tulai and Wu, Yongjun and Liu, Yong and Guo, Haizhong and Hong, Zijian and Xie, Yanwu and Tian, He and Zhang, Ze},
  doi          = {10.1038/s41567-024-02729-0},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {464-470},
  shortjournal = {Nat. Phys.},
  title        = {Thermal triggering for multi-state switching of polar topologies},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Floquet–bloch manipulation of the dirac gap in a topological
antiferromagnet. <em>NPHYS</em>, <em>21</em>(3), 458–463. (<a
href="https://doi.org/10.1038/s41567-024-02769-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Floquet–Bloch manipulation, achieved by driving a material periodically with a laser pulse, is a method that enables the engineering of electronic and magnetic phases in solids by effectively modifying the structure of their electronic bands. However, the application of Floquet–Bloch manipulation in topological magnetic systems, particularly those with inherent disorder, remains largely unexplored. Here we realize Floquet–Bloch manipulation of the Dirac surface-state mass of the topological antiferromagnet MnBi2Te4. Using time- and angle-resolved photoemission spectroscopy, we show that opposite helicities of mid-infrared circularly polarized light result in substantially different Dirac mass gaps in the antiferromagnetic phase, despite the equilibrium Dirac cone being massless. We explain our findings in terms of a Dirac fermion with a random mass. Our results underscore Floquet–Bloch manipulation as a powerful tool for controlling topology, even in the presence of disorder, and for uncovering properties of materials that may elude conventional probes. Periodic laser light can modify the electronic properties of solids and offers a path to create new material phases. In a topological antiferromagnet, periodic driving with opposite light helicities is now shown to produce distinct Dirac mass gaps.},
  archive      = {J_NPHYS},
  author       = {Bielinski, Nina and Chari, Rajas and May-Mann, Julian and Kim, Soyeun and Zwettler, Jack and Deng, Yujun and Aishwarya, Anuva and Roychowdhury, Subhajit and Shekhar, Chandra and Hashimoto, Makoto and Lu, Donghui and Yan, Jiaqiang and Felser, Claudia and Madhavan, Vidya and Shen, Zhi-Xun and Hughes, Taylor L. and Mahmood, Fahad},
  doi          = {10.1038/s41567-024-02769-6},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {458-463},
  shortjournal = {Nat. Phys.},
  title        = {Floquet–Bloch manipulation of the dirac gap in a topological antiferromagnet},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Time-hidden magnetic order in a multi-orbital mott
insulator. <em>NPHYS</em>, <em>21</em>(3), 451–457. (<a
href="https://doi.org/10.1038/s41567-024-02752-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photo-excited quantum materials can be driven into thermally inaccessible metastable states that exhibit structural, charge, spin, topological and superconducting orders. Metastable states typically emerge on timescales set by the intrinsic electronic and phononic energy scales, ranging from femtoseconds to picoseconds, and can persist for weeks. Therefore, studies have primarily focused on ultrafast or quasi-static limits, leaving the intermediate time window less explored. Here we reveal a metastable state with broken glide-plane symmetry in photo-doped Ca2RuO4 using time-resolved optical second-harmonic generation and birefringence measurements. We find that the metastable state appears long after intralayer antiferromagnetic order has melted and photo-carriers have recombined. Its properties are distinct from all known states in the equilibrium phase diagram and are consistent with intralayer ferromagnetic order. Furthermore, model Hamiltonian calculations reveal that a non-thermal trajectory to this state can be accessed via photo-doping. Our results expand the search space for out-of-equilibrium electronic matter to metastable states emerging at intermediate timescales. Searches for metastable states with properties not found in thermal equilibrium have been restricted to either ultrafast or slow timescales. A metastable state in an intermediate time window has now been identified in a photo-doped Mott insulator.},
  archive      = {J_NPHYS},
  author       = {Li, Xinwei and Esin, Iliya and Han, Youngjoon and Liu, Yincheng and Zhao, Hengdi and Ning, Honglie and Barrett, Cora and Shan, Jun-Yi and Seyler, Kyle and Cao, Gang and Refael, Gil and Hsieh, David},
  doi          = {10.1038/s41567-024-02752-1},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {451-457},
  shortjournal = {Nat. Phys.},
  title        = {Time-hidden magnetic order in a multi-orbital mott insulator},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nematicity and orbital depairing in superconducting bernal
bilayer graphene. <em>NPHYS</em>, <em>21</em>(3), 444–450. (<a
href="https://doi.org/10.1038/s41567-024-02776-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Superconductivity is a common feature of graphite allotropes, having been observed in Bernal bilayers, rhombohedral trilayers and a wide variety of angle-misaligned multilayers. Despite notable differences in the electronic structure of these systems, supporting the graphite on a WSe2 substrate has been consistently observed to expand the range of the superconductivity in terms of carrier density and temperature. Here we report the observation of two distinct superconducting states in Bernal bilayer graphene with strong proximity-induced Ising spin–orbit coupling. Our quantum oscillation measurements show that, although the normal state of the first superconducting phase is consistent with the single-particle band structure, the second emerges from a nematic normal state with broken rotational symmetry. Both superconductors are robust to in-plane magnetic fields, but neither reach fields expected for spin–valley-locked Ising superconductors. The Fermi surface geometry of the first superconducting phase suggests that the superconductivity is limited by orbital depairing arising from the imperfect layer polarization of the electron wavefunctions. Finally, an analysis of transport and thermodynamic compressibility measurements in the second superconducting phase shows that the proximity to isospin phase boundaries, observed in other rhombohedral graphene allotropes, is probably coincidental, thus constraining theories of the pairing mechanisms in these systems. Two regions of superconductivity are observed in the phase diagram of Bernal-stacked bilayer graphene. Spin–orbit coupling induced by the substrate and orbital moments are shown to be important in describing their properties.},
  archive      = {J_NPHYS},
  author       = {Holleis, Ludwig and Patterson, Caitlin L. and Zhang, Yiran and Vituri, Yaar and Yoo, Heun Mo and Zhou, Haoxin and Taniguchi, Takashi and Watanabe, Kenji and Berg, Erez and Nadj-Perge, Stevan and Young, Andrea F.},
  doi          = {10.1038/s41567-024-02776-7},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {444-450},
  shortjournal = {Nat. Phys.},
  title        = {Nematicity and orbital depairing in superconducting bernal bilayer graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An electronic microemulsion phase emerging from a quantum
crystal-to-liquid transition. <em>NPHYS</em>, <em>21</em>(3), 437–443.
(<a href="https://doi.org/10.1038/s41567-024-02759-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strongly interacting electronic systems often exhibit a complicated phase diagram that results from the competition between different quantum ground states. One feature of these phase diagrams is the emergence of microemulsion phases, where regions of different phases self-organize across multiple length scales. The experimental characterization of these microemulsions can pose considerable challenges, as the long-range Coulomb interaction microscopically mingles with the competing states. Here we observe the signatures of the microemulsion between an electronic Wigner crystal and an electron liquid in a MoSe2 monolayer using cryogenic reflectance and magneto-optical spectroscopy. We find that the transition into this microemulsion state is marked by anomalies in exciton reflectance, spin susceptibility and umklapp scattering, establishing it as a distinct phase of electronic matter. Competition between different possible ground states of strongly correlated electron systems can lead to the emergence of mixed states called microemulsions. Now this phenomenon is reported at the melting transition of a Wigner crystal.},
  archive      = {J_NPHYS},
  author       = {Sung, Jiho and Wang, Jue and Esterlis, Ilya and Volkov, Pavel A. and Scuri, Giovanni and Zhou, You and Brutschea, Elise and Taniguchi, Takashi and Watanabe, Kenji and Yang, Yubo and Morales, Miguel A. and Zhang, Shiwei and Millis, Andrew J. and Lukin, Mikhail D. and Kim, Philip and Demler, Eugene and Park, Hongkun},
  doi          = {10.1038/s41567-024-02759-8},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {437-443},
  shortjournal = {Nat. Phys.},
  title        = {An electronic microemulsion phase emerging from a quantum crystal-to-liquid transition},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pressure-enhanced splitting of density wave transitions in
La3Ni2O7–δ. <em>NPHYS</em>, <em>21</em>(3), 430–436. (<a
href="https://doi.org/10.1038/s41567-024-02754-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The observation of superconductivity in La3Ni2O7–δ under pressure, following the suppression of a high-temperature density wave state, has attracted considerable attention. The nature of this density wave order was not clearly identified. Here we probe the magnetic response of the zero-pressure phase of La3Ni2O7–δ as hydrostatic pressure is applied, and find that the apparent single density wave transition at zero applied pressure splits into two. The comparison of our muon-spin rotation and relaxation experiments with dipole-field numerical analysis reveals the magnetic structure’s compatibility with a stripe-type arrangement of Ni moments, characterized by alternating lines of magnetic moments and non-magnetic stripes at ambient pressure. When pressure is applied, the magnetic ordering temperature increases, whereas the unidentified density wave transition temperature falls. Our findings reveal that the ground state of the La3Ni2O7–δ system is characterized by the coexistence of two distinct orders—a magnetically ordered spin density wave and a lower-temperature ordering that is most probably a charge density wave—with a notable pressure-enhanced separation between them. The density wave transition in a superconducting nickelate is shown to split when hydrostatic pressure is applied, indicating that it is composed of both a spin density wave and another form of ordered state.},
  archive      = {J_NPHYS},
  author       = {Khasanov, Rustem and Hicken, Thomas J. and Gawryluk, Dariusz J. and Sazgari, Vahid and Plokhikh, Igor and Sorel, Loïc Pierre and Bartkowiak, Marek and Bötzel, Steffen and Lechermann, Frank and Eremin, Ilya M. and Luetkens, Hubertus and Guguchia, Zurab},
  doi          = {10.1038/s41567-024-02754-z},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {430-436},
  shortjournal = {Nat. Phys.},
  title        = {Pressure-enhanced splitting of density wave transitions in La3Ni2O7–δ},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Coulomb interactions and migrating dirac cones imaged by
local quantum oscillations in twisted graphene. <em>NPHYS</em>,
<em>21</em>(3), 421–429. (<a
href="https://doi.org/10.1038/s41567-025-02786-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Flat-band moiré graphene systems are a quintessential platform for investigating correlated phases of matter. Various interaction-driven ground states have been proposed, but despite extensive experimental effort, there has been little direct evidence that distinguishes between various phases, in particular near the charge neutrality point. Here we probe the fine details of the density of states and the effects of Coulomb interactions in alternating-twist trilayer graphene by imaging the local thermodynamic quantum oscillations with a nanoscale scanning superconducting quantum interference device. We find that the charging self-energy due to occupied electronic states is most important in explaining the high-carrier-density physics. At half-filling of the conduction flat band, we observe ferromagnetic-driven symmetry breaking, suggesting that it is the most robust mechanism in the hierarchy of phase transitions. Near charge neutrality, where exchange energy dominates over charging self-energy, we find a nematic semimetal ground state, which is theoretically favoured over gapped states in the presence of heterostrain. In this semimetallic phase, the flat-band Dirac cones migrate towards the mini-Brillouin zone centre, spontaneously breaking the threefold rotational symmetry. Our low-field local quantum oscillation technique can be used to explore the ground states of many strongly interacting van der Waals systems. Our understanding of the phase diagram of twisted graphene structures is incomplete. Now, twisted trilayer graphene is examined using a technique that locally images quantum oscillations and shows that a nematic semimetal is favoured at low density.},
  archive      = {J_NPHYS},
  author       = {Bocarsly, Matan and Roy, Indranil and Bhardwaj, Vishal and Uzan, Matan and Ledwith, Patrick and Shavit, Gal and Banu, Nasrin and Zhou, Yaozhang and Myasoedov, Yuri and Watanabe, Kenji and Taniguchi, Takashi and Oreg, Yuval and Parker, Daniel E. and Ronen, Yuval and Zeldov, Eli},
  doi          = {10.1038/s41567-025-02786-z},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {421-429},
  shortjournal = {Nat. Phys.},
  title        = {Coulomb interactions and migrating dirac cones imaged by local quantum oscillations in twisted graphene},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diffraction minima resolve point scatterers at few
hundredths of the wavelength. <em>NPHYS</em>, <em>21</em>(3), 412–420.
(<a href="https://doi.org/10.1038/s41567-024-02760-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resolving two or more constantly scattering identical point sources using freely propagating waves is limited by diffraction. Here we show that, by illuminating with a diffraction minimum, a given number of point scatterers can be resolved at distances of small fractions of the wavelength. Specifically, we identify an 8 nm distance, which corresponds to 1/80 of the employed 640 nm wavelength, between two constantly emitting fluorescent molecules in the focal plane of an optical microscope. We also measure 22 nm side length for a quadratic array of four molecules. Moreover, we show that the measurement precision improves with decreasing distance and with increased scatterer density. This work opens up the prospect of resolving individual scatterers in clusters that are far smaller than the wavelength. The central maximum of a diffraction pattern is commonly used to resolve point-like scatterers. Now spatial resolution far below the diffraction limit is achieved by using the scattering minima.},
  archive      = {J_NPHYS},
  author       = {Hensel, Thomas A. and Wirth, Jan O. and Schwarz, Ole L. and Hell, Stefan W.},
  doi          = {10.1038/s41567-024-02760-1},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {412-420},
  shortjournal = {Nat. Phys.},
  title        = {Diffraction minima resolve point scatterers at few hundredths of the wavelength},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Non-markovian dynamics of a superconducting qubit in a
phononic bandgap. <em>NPHYS</em>, <em>21</em>(3), 406–411. (<a
href="https://doi.org/10.1038/s41567-024-02740-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reducing decoherence in quantum computers rapidly decreases the overhead needed to construct a logical qubit from physical qubits. In solid-state systems, a class of defects known as two-level systems is a major source of decoherence. Currently, superconducting qubit experiments reduce dissipation due to the two-level systems by using large device dimensions. However, this approach only provides partial protection and results in a trade-off between qubit size and dissipation. In this work, we instead engineer the interactions between a qubit and the surrounding two-level systems using phononics. We fabricate a superconducting qubit on a phononic-bandgap metamaterial that suppresses phonon emission mediated by the two-level systems. The phonon-engineered bath of two-level systems shows increased lifetime and affects the thermalization dynamics of the qubit. Within the phononic bandgap, we observe the emergence of a non-Markovian qubit behaviour. Combined with qubit miniaturization, our approach could substantially extend the qubit relaxation times. Defects known as two-level systems are a major source of noise for superconducting qubits. Adding a phononic crystal is now shown to extend the lifetime of these two-level systems, which could lead to improved qubit coherence.},
  archive      = {J_NPHYS},
  author       = {Odeh, Mutasem and Godeneli, Kadircan and Li, Eric and Tangirala, Rohin and Zhou, Haoxin and Zhang, Xueyue and Zhang, Zi-Huai and Sipahigil, Alp},
  doi          = {10.1038/s41567-024-02740-5},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {406-411},
  shortjournal = {Nat. Phys.},
  title        = {Non-markovian dynamics of a superconducting qubit in a phononic bandgap},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Optical readout of a superconducting qubit using a
piezo-optomechanical transducer. <em>NPHYS</em>, <em>21</em>(3),
401–405. (<a href="https://doi.org/10.1038/s41567-024-02742-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Superconducting quantum processors have made important progress in size and computing potential. However, the practical cryogenic limitations of operating large numbers of superconducting qubits are becoming a bottleneck for further scaling. Due to the low thermal conductivity and the dense optical multiplexing capacity of telecommunications fibre, converting qubit signal processing to the optical domain using microwave-to-optics transduction would substantially relax the strain on cryogenic space and thermal budgets. Here we demonstrate optical readout of a superconducting transmon qubit through an optical fibre connected via a coaxial cable to a fully integrated piezo-optomechanical transducer. Using a demolition readout technique, we achieve a single-shot readout fidelity of 81%. Our results illustrate the benefits of piezo-optomechanical transduction for low-dissipation operation of large quantum processors. Superconducting qubits are measured using microwaves, posing constraints on its size and thermal budgets. The electro-optic transceiver presented here can be used to perform optical readout without affecting qubit performance.},
  archive      = {J_NPHYS},
  author       = {van Thiel, T. C. and Weaver, M. J. and Berto, F. and Duivestein, P. and Lemang, M. and Schuurman, K. L. and Žemlička, M. and Hijazi, F. and Bernasconi, A. C. and Ferrer, C. and Cataldo, E. and Lachman, E. and Field, M. and Mohan, Y. and de Vries, F. K. and Bultink, C. C. and van Oven, J. C. and Mutus, J. Y. and Stockill, R. and Gröblacher, S.},
  doi          = {10.1038/s41567-024-02742-3},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {401-405},
  shortjournal = {Nat. Phys.},
  title        = {Optical readout of a superconducting qubit using a piezo-optomechanical transducer},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). All-optical superconducting qubit readout. <em>NPHYS</em>,
<em>21</em>(3), 393–400. (<a
href="https://doi.org/10.1038/s41567-024-02741-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid development of superconducting quantum hardware is expected to run into substantial restrictions on scalability because error correction in a cryogenic environment has stringent input–output requirements. Classical data centres rely on fibre-optic interconnects to remove similar networking bottlenecks. In the same spirit, ultracold electro-optic links have been proposed and used to generate qubit control signals, or to replace cryogenic readout electronics. So far, these approaches have suffered from either low efficiency, low bandwidth or additional noise. Here we realize radio-over-fibre qubit readout at millikelvin temperatures. We use one device to simultaneously perform upconversion and downconversion between microwave and optical frequencies and so do not require any active or passive cryogenic microwave equipment. We demonstrate all-optical single-shot readout in a circulator-free readout scheme. Importantly, we do not observe any direct radiation impact on the qubit state, despite the absence of shielding elements. This compatibility between superconducting circuits and telecom-wavelength light is not only a prerequisite to establish modular quantum networks, but it is also relevant for multiplexed readout of superconducting photon detectors and classical superconducting logic. Microwaves are usually used to interact with superconducting qubits, but optical photons can be processed at room temperature. The electro-optical transceiver presented here allows all-optical readout of a qubit without affecting its performance.},
  archive      = {J_NPHYS},
  author       = {Arnold, Georg and Werner, Thomas and Sahu, Rishabh and Kapoor, Lucky N. and Qiu, Liu and Fink, Johannes M.},
  doi          = {10.1038/s41567-024-02741-4},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {393-400},
  shortjournal = {Nat. Phys.},
  title        = {All-optical superconducting qubit readout},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Stirring the false vacuum via interacting quantized bubbles
on a 5,564-qubit quantum annealer. <em>NPHYS</em>, <em>21</em>(3),
386–392. (<a href="https://doi.org/10.1038/s41567-024-02765-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {False vacuum decay—the transition from a metastable quantum state to a true vacuum state—plays an important role in quantum field theory and non-equilibrium phenomena such as phase transitions and dynamical metastability. The non-perturbative nature of false vacuum decay and the limited experimental access to this process make it challenging to study, leaving several open questions regarding how true vacuum bubbles form, move and interact. Here we observe quantized bubble formation in real time, a key feature of false vacuum decay dynamics, using a quantum annealer with 5,564 superconducting flux qubits. We develop an effective model that captures both initial bubble creation and subsequent interactions, and remains accurate under dissipation. The annealer reveals coherent scaling laws in the driven many-body dynamics for more than 1,000 intrinsic qubit time units. This work provides a method for investigating false vacuum dynamics of large quantum systems in quantum annealers. Bubble formation is a signal of false vacuum decay, in which a system transitions from a local energy minimum to a true vacuum. Now, simulations on a quantum annealer show how interactions between bubbles drive the long-time dynamics of this process.},
  archive      = {J_NPHYS},
  author       = {Vodeb, Jaka and Desaules, Jean-Yves and Hallam, Andrew and Rava, Andrea and Humar, Gregor and Willsch, Dennis and Jin, Fengping and Willsch, Madita and Michielsen, Kristel and Papić, Zlatko},
  doi          = {10.1038/s41567-024-02765-w},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {386-392},
  shortjournal = {Nat. Phys.},
  title        = {Stirring the false vacuum via interacting quantized bubbles on a 5,564-qubit quantum annealer},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wideband electric field quantum sensing via motional raman
transitions. <em>NPHYS</em>, <em>21</em>(3), 380–385. (<a
href="https://doi.org/10.1038/s41567-024-02753-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultrasensitive detection of the frequency, phase and amplitude of radiofrequency electric fields is crucial for applications in radio communication, cosmology, dark matter searches and high-fidelity qubit control. Quantum harmonic oscillator systems, especially trapped ions, offer high-sensitivity electric field sensing with nanometre spatial resolution but are typically restricted to narrow frequency ranges centred around the motional frequency of the trapped-ion oscillator or the frequency of an optical transition in the ion. Here we present a procedure that enables precise electric field detection over an expanded frequency range. Specifically, we use motional Raman transitions in a single trapped ion to achieve sensitivity across a frequency range over 800 times larger than previous approaches. We show that the method is compatible with both quantum amplification via squeezing and measurement in the Fock basis, allowing a demonstration of performance 3.4(2.0) dB below the standard quantum limit. The approach can be extended to other quantum harmonic oscillator systems, such as superconducting qubit–resonator systems. The narrow frequency response of quantum harmonic oscillators typically limits their application in radiofrequency electric field detection. Now motional Raman transitions in trapped ions are shown to enable wideband, high-precision radiofrequency field sensing.},
  archive      = {J_NPHYS},
  author       = {Wu, Hao and Mitts, Grant D. and Ho, Clayton Z. C. and Rabinowitz, Joshua A. and Hudson, Eric R.},
  doi          = {10.1038/s41567-024-02753-0},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {380-385},
  shortjournal = {Nat. Phys.},
  title        = {Wideband electric field quantum sensing via motional raman transitions},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Observation of a finite-energy phase transition in a
one-dimensional quantum simulator. <em>NPHYS</em>, <em>21</em>(3),
374–379. (<a href="https://doi.org/10.1038/s41567-024-02751-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Equilibrium phase transitions in many-body systems have been predicted and observed in two and three spatial dimensions but have long been thought not to exist in one-dimensional systems. It was suggested that a phase transition in one dimension can occur in the presence of long-range interactions. However, an experimental realization has so far not been achieved due to the requirement to both realize interactions over sufficiently long distances and to prepare equilibrium states. Here we demonstrate a finite-energy phase transition in one dimension by implementing a long-range interacting model in a trapped-ion quantum simulator. We show that finite-energy states can be generated by time-evolving initial product states and letting them thermalize under the dynamics of a many-body Hamiltonian. By preparing initial states with different energies, we study the finite-energy phase diagram of a long-range interacting quantum system. We observe a ferromagnetic equilibrium phase transition as well as a crossover from a low-energy polarized paramagnet to a high-energy unpolarized paramagnet, in agreement with numerical simulations. Our work presents a scheme for preparing finite-energy states in quantum simulation platforms, enabling access to phases at finite energy density. Long-range interactions have been predicted to enable a phase transition in one-dimensional systems. An experiment now validates this hypothesis in a trapped-ion quantum simulator by observing a finite-energy phase transition in one dimension.},
  archive      = {J_NPHYS},
  author       = {Schuckert, Alexander and Katz, Or and Feng, Lei and Crane, Eleanor and De, Arinjoy and Hafezi, Mohammad and Gorshkov, Alexey V. and Monroe, Christopher},
  doi          = {10.1038/s41567-024-02751-2},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {374-379},
  shortjournal = {Nat. Phys.},
  title        = {Observation of a finite-energy phase transition in a one-dimensional quantum simulator},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A many-body quantum register for a spin qubit.
<em>NPHYS</em>, <em>21</em>(3), 368–373. (<a
href="https://doi.org/10.1038/s41567-024-02746-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum networks require quantum nodes with coherent optical interfaces and several stationary qubits. In terms of optical properties, semiconductor quantum dots are highly compelling, but their adoption as quantum nodes has been impaired by the lack of auxiliary qubits. Here we demonstrate that the dense, always-present, nuclear spin ensemble surrounding a gallium arsenide quantum dot can be used as a functional quantum register. We prepared 13,000 host nuclear spins in a single many-body dark state that acts as a logical state of the register. A second logical state is defined as a single nuclear-magnon excitation, enabling controlled quantum-state transfer between an electron spin qubit in the quantum dot and the nuclear magnonic register. Using SWAP gates, we implemented a full write–store–retrieve-read-out protocol with 68.6(4)% raw overall fidelity and a storage time of 130(16) μs, which could be extended to 20 ms or beyond using dynamical decoupling techniques. Our work establishes how many-body physics can add functionality to quantum devices, in this case transforming quantum dots into multi-qubit quantum nodes with deterministic registers. The thousands of nuclear spins surrounding gallium arsenide quantum dots can interface with electron spin qubits and photons. With quantum engineering, this nuclear spin ensemble becomes a robust register for quantum information storage.},
  archive      = {J_NPHYS},
  author       = {Appel, Martin Hayhurst and Ghorbal, Alexander and Shofer, Noah and Zaporski, Leon and Manna, Santanu and da Silva, Saimon Filipe Covre and Haeusler, Urs and Le Gall, Claire and Rastelli, Armando and Gangloff, Dorian A. and Atatüre, Mete},
  doi          = {10.1038/s41567-024-02746-z},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {368-373},
  shortjournal = {Nat. Phys.},
  title        = {A many-body quantum register for a spin qubit},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Schrödinger cat states of a nuclear spin qudit in silicon.
<em>NPHYS</em>, <em>21</em>(3), 362–367. (<a
href="https://doi.org/10.1038/s41567-024-02745-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-dimensional quantum systems are a valuable resource for quantum information processing. They can be used to encode error-correctable logical qubits, which has been demonstrated using continuous-variable states in microwave cavities or the motional modes of trapped ions. For example, high-dimensional systems can be used to realize ‘Schrödinger cat’ states, which are superpositions of widely displaced coherent states that can be used to illustrate quantum effects at large scales. Recent proposals have suggested encoding qubits in high-spin atomic nuclei, which are finite-dimensional systems that can host hardware-efficient versions of continuous-variable codes. Here we demonstrate the creation and manipulation of Schrödinger cat states using the spin-7/2 nucleus of an antimony atom embedded in a silicon nanoelectronic device. We use a multi-frequency control scheme to produce spin rotations that preserve the symmetry of the qudit, and we constitute logical Pauli operations for qubits encoded in the Schrödinger cat states. Our work demonstrates the ability to prepare and control non-classical resource states, which is a prerequisite for applications in quantum information processing and quantum error correction, using our scalable, manufacturable semiconductor platform. A large nuclear spin has been successfully placed in a Schrödinger cat state, a superposition of its two most widely separated spin coherent states. This can be used as an error-correctable qubit.},
  archive      = {J_NPHYS},
  author       = {Yu, Xi and Wilhelm, Benjamin and Holmes, Danielle and Vaartjes, Arjen and Schwienbacher, Daniel and Nurizzo, Martin and Kringhøj, Anders and Blankenstein, Mark R. van and Jakob, Alexander M. and Gupta, Pragati and Hudson, Fay E. and Itoh, Kohei M. and Murray, Riley J. and Blume-Kohout, Robin and Ladd, Thaddeus D. and Anand, Namit and Dzurak, Andrew S. and Sanders, Barry C. and Jamieson, David N. and Morello, Andrea},
  doi          = {10.1038/s41567-024-02745-0},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {362-367},
  shortjournal = {Nat. Phys.},
  title        = {Schrödinger cat states of a nuclear spin qudit in silicon},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Topology shapes dynamics of higher-order networks.
<em>NPHYS</em>, <em>21</em>(3), 353–361. (<a
href="https://doi.org/10.1038/s41567-024-02757-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Higher-order networks capture the many-body interactions present in complex systems, shedding light on the interplay between topology and dynamics. The theory of higher-order topological dynamics, which combines higher-order interactions with discrete topology and nonlinear dynamics, has the potential to enhance our understanding of complex systems, such as the brain and the climate, and to advance the development of next-generation AI algorithms. This theoretical framework, which goes beyond traditional node-centric descriptions, encodes the dynamics of a network through topological signals—variables assigned not only to nodes but also to edges, triangles and other higher-order cells. Recent findings show that topological signals lead to the emergence of distinct types of dynamical state and collective phenomena, including topological and Dirac synchronization, pattern formation and triadic percolation. These results offer insights into how topology shapes dynamics, how dynamics learns topology and how topology evolves dynamically. This Perspective primarily aims to guide physicists, mathematicians, computer scientists and network scientists through the emerging field of higher-order topological dynamics, while also outlining future research challenges. Higher-order interactions reveal new aspects of the interplay between topology and dynamics in complex systems. This Perspective describes the emerging field of higher-order topological dynamics and discusses the open research questions in the area.},
  archive      = {J_NPHYS},
  author       = {Millán, Ana P. and Sun, Hanlin and Giambagli, Lorenzo and Muolo, Riccardo and Carletti, Timoteo and Torres, Joaquín J. and Radicchi, Filippo and Kurths, Jürgen and Bianconi, Ginestra},
  doi          = {10.1038/s41567-024-02757-w},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {353-361},
  shortjournal = {Nat. Phys.},
  title        = {Topology shapes dynamics of higher-order networks},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A simulation of false vacuum decay using a quantum annealer.
<em>NPHYS</em>, <em>21</em>(3), 351–352. (<a
href="https://doi.org/10.1038/s41567-024-02767-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {False vacuum decay is a process of fundamental importance in quantum field theory. Here, a 5,564-qubit quantum annealer is used to simulate the dynamics of false vacuum decay and observe the formation of bubbles of true vacuum. This approach could provide insight into the role of phase transitions in the early Universe.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-024-02767-8},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {351-352},
  shortjournal = {Nat. Phys.},
  title        = {A simulation of false vacuum decay using a quantum annealer},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Maple stir-up. <em>NPHYS</em>, <em>21</em>(3), 350. (<a
href="https://doi.org/10.1038/s41567-025-02842-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Verberck, Bart},
  doi          = {10.1038/s41567-025-02842-8},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {350},
  shortjournal = {Nat. Phys.},
  title        = {Maple stir-up},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The magnetic cartography of sea turtles. <em>NPHYS</em>,
<em>21</em>(3), 349. (<a
href="https://doi.org/10.1038/s41567-025-02841-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Benini, Leonardo},
  doi          = {10.1038/s41567-025-02841-9},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {349},
  shortjournal = {Nat. Phys.},
  title        = {The magnetic cartography of sea turtles},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The way the bubbles ride. <em>NPHYS</em>, <em>21</em>(3),
348. (<a href="https://doi.org/10.1038/s41567-025-02834-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Dubrovina, Elizaveta},
  doi          = {10.1038/s41567-025-02834-8},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {348},
  shortjournal = {Nat. Phys.},
  title        = {The way the bubbles ride},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Glass is a matter of time. <em>NPHYS</em>, <em>21</em>(3),
346–347. (<a href="https://doi.org/10.1038/s41567-025-02788-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the behaviour of viscous liquids close to the glass transition is a century-old problem. The microscopic underpinnings of their mechanical response have now been made clearer by a unified percolation description, in both two and three dimensions.},
  archive      = {J_NPHYS},
  author       = {Coslovich, Daniele},
  doi          = {10.1038/s41567-025-02788-x},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {346-347},
  shortjournal = {Nat. Phys.},
  title        = {Glass is a matter of time},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Murmurations of electric dipoles. <em>NPHYS</em>,
<em>21</em>(3), 344–345. (<a
href="https://doi.org/10.1038/s41567-024-02777-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stable and metastable electrical dipole patterns have been imaged and manipulated using in situ heating and cooling in ferroelectric superlattices.},
  archive      = {J_NPHYS},
  author       = {Gregg, J. M.},
  doi          = {10.1038/s41567-024-02777-6},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {344-345},
  shortjournal = {Nat. Phys.},
  title        = {Murmurations of electric dipoles},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gap-opening time periodic perturbations. <em>NPHYS</em>,
<em>21</em>(3), 342–343. (<a
href="https://doi.org/10.1038/s41567-025-02805-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An intrinsic magnetic topological insulator with gapped surface states would be very useful for both fundamental and application prospects. Now, the use of the time-periodic drive of a laser makes these states not only possible, but also tunable.},
  archive      = {J_NPHYS},
  author       = {Ojeda-Aristizabal, Claudia},
  doi          = {10.1038/s41567-025-02805-z},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {342-343},
  shortjournal = {Nat. Phys.},
  title        = {Gap-opening time periodic perturbations},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electronic melt. <em>NPHYS</em>, <em>21</em>(3), 340–341.
(<a href="https://doi.org/10.1038/s41567-024-02779-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electrons at extremely low density and temperature can crystallize into a solid known as a Wigner crystal. Optical spectroscopy now reveals how these crystals melt at higher densities via an intermediate phase, where crystalline and liquid regions coexist.},
  archive      = {J_NPHYS},
  author       = {Yoon, Yoseob},
  doi          = {10.1038/s41567-024-02779-4},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {340-341},
  shortjournal = {Nat. Phys.},
  title        = {Electronic melt},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Magnetic order sets the stage. <em>NPHYS</em>,
<em>21</em>(3), 338–339. (<a
href="https://doi.org/10.1038/s41567-024-02772-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-temperature superconductivity has been found in a nickelate material under pressure, but the underlying state is unclear. Muons now show that the material hosts distinct phases that evolve differently with increasing pressure.},
  archive      = {J_NPHYS},
  author       = {Smidman, Michael},
  doi          = {10.1038/s41567-024-02772-x},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {338-339},
  shortjournal = {Nat. Phys.},
  title        = {Magnetic order sets the stage},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Better qubits through phononic engineering. <em>NPHYS</em>,
<em>21</em>(3), 336–337. (<a
href="https://doi.org/10.1038/s41567-024-02775-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Controlling qubit–phonon interactions is crucial for solid-state quantum devices. Two recent studies demonstrate that phononic bandgap engineering can alter these interactions, leading to enhanced qubit coherence and scalability.},
  archive      = {J_NPHYS},
  author       = {Zou, Chang-Ling and Sun, Luyan},
  doi          = {10.1038/s41567-024-02775-8},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {336-337},
  shortjournal = {Nat. Phys.},
  title        = {Better qubits through phononic engineering},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A central spin coordinates a nuclear crowd. <em>NPHYS</em>,
<em>21</em>(3), 334–335. (<a
href="https://doi.org/10.1038/s41567-025-02790-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To act as nodes in quantum networks, solid-state quantum dots must combine their high performance as quantum emitters with a quantum memory. Now, nuclear spins around a gallium arsenide quantum dot have been used as a many-body quantum memory register.},
  archive      = {J_NPHYS},
  author       = {Lei, Xia and Zhai, Liang},
  doi          = {10.1038/s41567-025-02790-3},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {334-335},
  shortjournal = {Nat. Phys.},
  title        = {A central spin coordinates a nuclear crowd},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Worlds beyond imagination. <em>NPHYS</em>, <em>21</em>(3),
333. (<a href="https://doi.org/10.1038/s41567-025-02820-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Brogi, Matteo},
  doi          = {10.1038/s41567-025-02820-0},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {333},
  shortjournal = {Nat. Phys.},
  title        = {Worlds beyond imagination},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The lasting legacy of poincaré. <em>NPHYS</em>,
<em>21</em>(3), 332. (<a
href="https://doi.org/10.1038/s41567-025-02818-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NPHYS},
  author       = {Buchanan, Mark},
  doi          = {10.1038/s41567-025-02818-8},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {332},
  shortjournal = {Nat. Phys.},
  title        = {The lasting legacy of poincaré},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Different facets of unconventional magnetism.
<em>NPHYS</em>, <em>21</em>(3), 329–331. (<a
href="https://doi.org/10.1038/s41567-024-02750-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in classifying magnets according to spin-group symmetry have expanded the possibilities of unconventional magnetism. Unconventional magnets — such as collinear spin-split antiferromagnets, also known as altermagnets, noncollinear spin-split antiferromagnets and anomalous-Hall antiferromagnets — combine the advantages of ferromagnetism and antiferromagnetism.},
  archive      = {J_NPHYS},
  author       = {Liu, Qihang and Dai, Xi and Blügel, Stefan},
  doi          = {10.1038/s41567-024-02750-3},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {329-331},
  shortjournal = {Nat. Phys.},
  title        = {Different facets of unconventional magnetism},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Physics should be for everyone. <em>NPHYS</em>,
<em>21</em>(3), 328. (<a
href="https://doi.org/10.1038/s41567-025-02795-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Physics is amazing and transformational. But for far too long the people most likely to benefit from its opportunity and wonder have been too narrowly defined.},
  archive      = {J_NPHYS},
  author       = {Bakewell, Sarah},
  doi          = {10.1038/s41567-025-02795-y},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {328},
  shortjournal = {Nat. Phys.},
  title        = {Physics should be for everyone},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Progress to parity. <em>NPHYS</em>, <em>21</em>(3), 327. (<a
href="https://doi.org/10.1038/s41567-025-02852-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Following International Women’s Day, we draw inspiration from trailblazing women physicists from history and ask how the physics community can accelerate the closing of the gender gap.},
  archive      = {J_NPHYS},
  doi          = {10.1038/s41567-025-02852-6},
  journal      = {Nature Physics},
  month        = {3},
  number       = {3},
  pages        = {327},
  shortjournal = {Nat. Phys.},
  title        = {Progress to parity},
  volume       = {21},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
