<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt---404">NBT - 404</h2>
<ul>
<li><details>
<summary>
(2022b). Chicken from cells is safe to eat, says FDA. <em>NBT</em>,
<em>41</em>(1), 8. (<a
href="https://doi.org/10.1038/d41587-022-00018-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The regulator&#39;s nod is a first for lab-grown meat in the United States. The regulator&#39;s nod is a first for lab-grown meat in the United States.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/d41587-022-00018-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {8},
  shortjournal = {Nature Biotech.},
  title        = {Chicken from cells is safe to eat, says FDA},
  volume       = {41},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). People. <em>NBT</em>, <em>40</em>(12), 1880. (<a
href="https://doi.org/10.1038/s41587-022-01606-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01606-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1880},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). How to grow (almost) anything: A hybrid distance learning
model for global laboratory-based synthetic biology education.
<em>NBT</em>, <em>40</em>(12), 1874–1879. (<a
href="https://doi.org/10.1038/s41587-022-01601-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A pilot program for synthetic biology education via a scalable distributed network model of distance-based laboratory learning can be accessible globally across disciplines and backgrounds.},
  archive      = {J_NBT},
  author       = {Perry, Eyal and Weber, Jessica and Pataranutaporn, Pat and Volf, Verena and Gonzalez, Laura Maria and Nejad, Sara and Angleton, Carolyn and Chen, Jia-En and Gabo, Ananda and Jammalamadaka, Mani Sai Suryateja and Kuru, Erkin and Fortuna, Patrick and Rico, Andres and Sulich, Karolina and Wawrzyniak, Dominika and Jacobson, Joseph and Church, George and Kong, David},
  doi          = {10.1038/s41587-022-01601-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1874-1879},
  shortjournal = {Nature Biotech.},
  title        = {How to grow (almost) anything: A hybrid distance learning model for global laboratory-based synthetic biology education},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Engineering the amoeba dictyostelium
discoideum for biosynthesis of a cannabinoid precursor and other
polyketides. <em>NBT</em>, <em>40</em>(12), 1873. (<a
href="https://doi.org/10.1038/s41587-022-01607-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Reimer, Christin and Kufs, Johann E. and Rautschek, Julia and Regestein, Lars and Valiante, Vito and Hillmann, Falk},
  doi          = {10.1038/s41587-022-01607-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1873},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Engineering the amoeba dictyostelium discoideum for biosynthesis of a cannabinoid precursor and other polyketides},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Synthetic memory circuits for stable cell reprogramming in
plants. <em>NBT</em>, <em>40</em>(12), 1862–1872. (<a
href="https://doi.org/10.1038/s41587-022-01383-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant biotechnology predominantly relies on a restricted set of genetic parts with limited capability to customize spatiotemporal and conditional expression patterns. Synthetic gene circuits have the potential to integrate multiple customizable input signals through a processing unit constructed from biological parts to produce a predictable and programmable output. Here we present a suite of functional recombinase-based gene circuits for use in plants. We first established a range of key gene circuit components compatible with plant cell functionality. We then used these to develop a range of operational logic gates using the identify function (activation) and negation function (repression) in Arabidopsis protoplasts and in vivo, demonstrating their utility for programmable manipulation of transcriptional activity in a complex multicellular organism. Specifically, using recombinases and plant control elements, we activated transgenes in YES, OR and AND gates and repressed them in NOT, NOR and NAND gates; we also implemented the A NIMPLY B gate that combines activation and repression. Through use of genetic recombination, these circuits create stable long-term changes in expression and recording of past stimuli. This highly compact programmable gene circuit platform provides new capabilities for engineering sophisticated transcriptional programs and previously unrealized traits into plants. Transcriptional activity in plants is controlled with a programmable gene circuit.},
  archive      = {J_NBT},
  author       = {Lloyd, James P. B. and Ly, Florence and Gong, Patrick and Pflueger, Jahnvi and Swain, Tessa and Pflueger, Christian and Fourie, Elliott and Khan, Muhammad Adil and Kidd, Brendan N. and Lister, Ryan},
  doi          = {10.1038/s41587-022-01383-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1862-1872},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic memory circuits for stable cell reprogramming in plants},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Rapid biosensor development using plant hormone receptors as
reprogrammable scaffolds. <em>NBT</em>, <em>40</em>(12), 1855–1861. (<a
href="https://doi.org/10.1038/s41587-022-01364-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A general method to generate biosensors for user-defined molecules could provide detection tools for a wide range of biological applications. Here, we describe an approach for the rapid engineering of biosensors using PYR1 (Pyrabactin Resistance 1), a plant abscisic acid (ABA) receptor with a malleable ligand-binding pocket and a requirement for ligand-induced heterodimerization, which facilitates the construction of sense–response functions. We applied this platform to evolve 21 sensors with nanomolar to micromolar sensitivities for a range of small molecules, including structurally diverse natural and synthetic cannabinoids and several organophosphates. X-ray crystallography analysis revealed the mechanistic basis for new ligand recognition by an evolved cannabinoid receptor. We demonstrate that PYR1-derived receptors are readily ported to various ligand-responsive outputs, including enzyme-linked immunosorbent assay (ELISA)-like assays, luminescence by protein-fragment complementation and transcriptional circuits, all with picomolar to nanomolar sensitivity. PYR1 provides a scaffold for rapidly evolving new biosensors for diverse sense–response applications. Biosensors are developed from malleable hormone receptors.},
  archive      = {J_NBT},
  author       = {Beltrán, Jesús and Steiner, Paul J. and Bedewitz, Matthew and Wei, Shuang and Peterson, Francis C. and Li, Zongbo and Hughes, Brigid E. and Hartley, Zachary and Robertson, Nicholas R. and Medina-Cucurella, Angélica V. and Baumer, Zachary T. and Leonard, Alison C. and Park, Sang-Youl and Volkman, Brian F. and Nusinow, Dmitri A. and Zhong, Wenwan and Wheeldon, Ian and Cutler, Sean R. and Whitehead, Timothy A.},
  doi          = {10.1038/s41587-022-01364-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1855-1861},
  shortjournal = {Nature Biotech.},
  title        = {Rapid biosensor development using plant hormone receptors as reprogrammable scaffolds},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The trispecific DARPin ensovibep inhibits diverse SARS-CoV-2
variants. <em>NBT</em>, <em>40</em>(12), 1845–1854. (<a
href="https://doi.org/10.1038/s41587-022-01382-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants with potential resistance to existing drugs emphasizes the need for new therapeutic modalities with broad variant activity. Here we show that ensovibep, a trispecific DARPin (designed ankyrin repeat protein) clinical candidate, can engage the three units of the spike protein trimer of SARS-CoV-2 and inhibit ACE2 binding with high potency, as revealed by cryo-electron microscopy analysis. The cooperative binding together with the complementarity of the three DARPin modules enable ensovibep to inhibit frequent SARS-CoV-2 variants, including Omicron sublineages BA.1 and BA.2. In Roborovski dwarf hamsters infected with SARS-CoV-2, ensovibep reduced fatality similarly to a standard-of-care monoclonal antibody (mAb) cocktail. When used as a single agent in viral passaging experiments in vitro, ensovibep reduced the emergence of escape mutations in a similar fashion to the same mAb cocktail. These results support further clinical evaluation of ensovibep as a broad variant alternative to existing targeted therapies for Coronavirus Disease 2019 (COVID-19). Multiple variants of SARS-CoV-2 are inhibited by a trispecific DARPin.},
  archive      = {J_NBT},
  author       = {Rothenberger, Sylvia and Hurdiss, Daniel L. and Walser, Marcel and Malvezzi, Francesca and Mayor, Jennifer and Ryter, Sarah and Moreno, Hector and Liechti, Nicole and Bosshart, Andreas and Iss, Chloé and Calabro, Valérie and Cornelius, Andreas and Hospodarsch, Tanja and Neculcea, Alexandra and Looser, Thamar and Schlegel, Anja and Fontaine, Simon and Villemagne, Denis and Paladino, Maria and Schiegg, Dieter and Mangold, Susanne and Reichen, Christian and Radom, Filip and Kaufmann, Yvonne and Schaible, Doris and Schlegel, Iris and Zitt, Christof and Sigrist, Gabriel and Straumann, Marcel and Wolter, Julia and Comby, Marco and Sacarcelik, Feyza and Drulyte, Ieva and Lyoo, Heyrhyoung and Wang, Chunyan and Li, Wentao and Du, Wenjuan and Binz, H. Kaspar and Herrup, Rachel and Lusvarghi, Sabrina and Neerukonda, Sabari Nath and Vassell, Russell and Wang, Wei and Adler, Julia M. and Eschke, Kathrin and Nascimento, Mariana and Abdelgawad, Azza and Gruber, Achim D. and Bushe, Judith and Kershaw, Olivia and Knutson, Charles G. and Balavenkatraman, Kamal K. and Ramanathan, Krishnan and Wyler, Emanuel and Teixeira Alves, Luiz Gustavo and Lewis, Seth and Watson, Randall and Haeuptle, Micha A. and Zürcher, Alexander and Dawson, Keith M. and Steiner, Daniel and Weiss, Carol D. and Amstutz, Patrick and van Kuppeveld, Frank J. M. and Stumpp, Michael T. and Bosch, Berend-Jan and Engler, Olivier and Trimpert, Jakob},
  doi          = {10.1038/s41587-022-01382-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1845-1854},
  shortjournal = {Nature Biotech.},
  title        = {The trispecific DARPin ensovibep inhibits diverse SARS-CoV-2 variants},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ras-mutant cancers are sensitive to small molecule
inhibition of v-type ATPases in mice. <em>NBT</em>, <em>40</em>(12),
1834–1844. (<a
href="https://doi.org/10.1038/s41587-022-01386-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations in Ras family proteins are implicated in 33\% of human cancers, but direct pharmacological inhibition of Ras mutants remains challenging. As an alternative to direct inhibition, we screened for sensitivities in Ras-mutant cells and discovered 249C as a Ras-mutant selective cytotoxic agent with nanomolar potency against a spectrum of Ras-mutant cancers. 249C binds to vacuolar (V)-ATPase with nanomolar affinity and inhibits its activity, preventing lysosomal acidification and inhibiting autophagy and macropinocytosis pathways that several Ras-driven cancers rely on for survival. Unexpectedly, potency of 249C varies with the identity of the Ras driver mutation, with the highest potency for KRASG13D and G12V both in vitro and in vivo, highlighting a mutant-specific dependence on macropinocytosis and lysosomal pH. Indeed, 249C potently inhibits tumor growth without adverse side effects in mouse xenografts of KRAS-driven lung and colon cancers. A comparison of isogenic SW48 xenografts with different KRAS mutations confirmed that KRASG13D/+ (followed by G12V/+) mutations are especially sensitive to 249C treatment. These data establish proof-of-concept for targeting V-ATPase in cancers driven by specific KRAS mutations such as KRASG13D and G12V. Cancers with common KRAS mutations can be treated with an inhibitor of lysosomal acidification in mice.},
  archive      = {J_NBT},
  author       = {Tolani, Bhairavi and Celli, Anna and Yao, Yanmin and Tan, Yong Zi and Fetter, Richard and Liem, Christina R. and de Smith, Adam J. and Vasanthakumar, Thamiya and Bisignano, Paola and Cotton, Adam D. and Seiple, Ian B. and Rubinstein, John L. and Jost, Marco and Weissman, Jonathan S.},
  doi          = {10.1038/s41587-022-01386-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1834-1844},
  shortjournal = {Nature Biotech.},
  title        = {Ras-mutant cancers are sensitive to small molecule inhibition of V-type ATPases in mice},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A multiplex implantable microdevice assay identifies
synergistic combinations of cancer immunotherapies and conventional
drugs. <em>NBT</em>, <em>40</em>(12), 1823–1833. (<a
href="https://doi.org/10.1038/s41587-022-01379-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Systematically identifying synergistic combinations of targeted agents and immunotherapies for cancer treatments remains difficult. In this study, we integrated high-throughput and high-content techniques—an implantable microdevice to administer multiple drugs into different sites in tumors at nanodoses and multiplexed imaging of tumor microenvironmental states—to investigate the tumor cell and immunological response signatures to different treatment regimens. Using a mouse model of breast cancer, we identified effective combinations from among numerous agents within days. In vivo studies in three immunocompetent mammary carcinoma models demonstrated that the predicted combinations synergistically increased therapeutic efficacy. We identified at least five promising treatment strategies, of which the panobinostat, venetoclax and anti-CD40 triple therapy was the most effective in inducing complete tumor remission across models. Successful drug combinations increased spatial association of cancer stem cells with dendritic cells during immunogenic cell death, suggesting this as an important mechanism of action in long-term breast cancer control. Testing of combinations of cancer immunotherapies and conventional drugs yields promising leads.},
  archive      = {J_NBT},
  author       = {Tatarova, Zuzana and Blumberg, Dylan C. and Korkola, James E. and Heiser, Laura M. and Muschler, John L. and Schedin, Pepper J. and Ahn, Sebastian W. and Mills, Gordon B. and Coussens, Lisa M. and Jonas, Oliver and Gray, Joe W.},
  doi          = {10.1038/s41587-022-01379-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1823-1833},
  shortjournal = {Nature Biotech.},
  title        = {A multiplex implantable microdevice assay identifies synergistic combinations of cancer immunotherapies and conventional drugs},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Viral variant-resolved wastewater surveillance of SARS-CoV-2
at national scale. <em>NBT</em>, <em>40</em>(12), 1814–1822. (<a
href="https://doi.org/10.1038/s41587-022-01387-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SARS-CoV-2 surveillance by wastewater-based epidemiology is poised to provide a complementary approach to sequencing individual cases. However, robust quantification of variants and de novo detection of emerging variants remains challenging for existing strategies. We deep sequenced 3,413 wastewater samples representing 94 municipal catchments, covering &amp;gt;59\% of the population of Austria, from December 2020 to February 2022. Our system of variant quantification in sewage pipeline designed for robustness (termed VaQuERo) enabled us to deduce the spatiotemporal abundance of predefined variants from complex wastewater samples. These results were validated against epidemiological records of &amp;gt;311,000 individual cases. Furthermore, we describe elevated viral genetic diversity during the Delta variant period, provide a framework to predict emerging variants and measure the reproductive advantage of variants of concern by calculating variant-specific reproduction numbers from wastewater. Together, this study demonstrates the power of national-scale WBE to support public health and promises particular value for countries without extensive individual monitoring. Wastewater surveillance of SARS-CoV-2 at the national scale tracks emerging variants.},
  archive      = {J_NBT},
  author       = {Amman, Fabian and Markt, Rudolf and Endler, Lukas and Hupfauf, Sebastian and Agerer, Benedikt and Schedl, Anna and Richter, Lukas and Zechmeister, Melanie and Bicher, Martin and Heiler, Georg and Triska, Petr and Thornton, Matthew and Penz, Thomas and Senekowitsch, Martin and Laine, Jan and Keszei, Zsofia and Klimek, Peter and Nägele, Fabiana and Mayr, Markus and Daleiden, Beatrice and Steinlechner, Martin and Niederstätter, Harald and Heidinger, Petra and Rauch, Wolfgang and Scheffknecht, Christoph and Vogl, Gunther and Weichlinger, Günther and Wagner, Andreas Otto and Slipko, Katarzyna and Masseron, Amandine and Radu, Elena and Allerberger, Franz and Popper, Niki and Bock, Christoph and Schmid, Daniela and Oberacher, Herbert and Kreuzinger, Norbert and Insam, Heribert and Bergthaler, Andreas},
  doi          = {10.1038/s41587-022-01387-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1814-1822},
  shortjournal = {Nature Biotech.},
  title        = {Viral variant-resolved wastewater surveillance of SARS-CoV-2 at national scale},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Frequent aneuploidy in primary human t cells after
CRISPR–Cas9 cleavage. <em>NBT</em>, <em>40</em>(12), 1807–1813. (<a
href="https://doi.org/10.1038/s41587-022-01377-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple clinical trials of allogeneic T cell therapy use site-specific nucleases to disrupt T cell receptor (TCR) and other genes1–6. In this study, using single-cell RNA sequencing, we investigated genome editing outcomes in primary human T cells transfected with CRISPR–Cas9 and guide RNAs targeting genes for TCR chains and programmed cell death protein 1. Four days after transfection, we found a loss of chromosome 14, harboring the TCRα locus, in up to 9\% of the cells and a chromosome 14 gain in up to 1.4\% of the cells. Chromosome 7, harboring the TCRβ locus, was truncated in 9.9\% of the cells. Aberrations were validated using fluorescence in situ hybridization and digital droplet PCR. Aneuploidy was associated with reduced proliferation, induced p53 activation and cell death. However, at 11 days after transfection, 0.9\% of T cells still had a chromosome 14 loss. Aneuploidy and chromosomal truncations are, thus, frequent outcomes of CRISPR–Cas9 cleavage that should be monitored and minimized in clinical protocols. Aneuploidy and chromosomal truncations are frequent outcomes of CRISPR–Cas9 editing in human T cells.},
  archive      = {J_NBT},
  author       = {Nahmad, Alessio David and Reuveni, Eli and Goldschmidt, Ella and Tenne, Tamar and Liberman, Meytal and Horovitz-Fried, Miriam and Khosravi, Rami and Kobo, Hila and Reinstein, Eyal and Madi, Asaf and Ben-David, Uri and Barzel, Adi},
  doi          = {10.1038/s41587-022-01377-0},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1807-1813},
  shortjournal = {Nature Biotech.},
  title        = {Frequent aneuploidy in primary human t cells after CRISPR–Cas9 cleavage},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High-plex imaging of RNA and proteins at subcellular
resolution in fixed tissue by spatial molecular imaging. <em>NBT</em>,
<em>40</em>(12), 1794–1806. (<a
href="https://doi.org/10.1038/s41587-022-01483-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resolving the spatial distribution of RNA and protein in tissues at subcellular resolution is a challenge in the field of spatial biology. We describe spatial molecular imaging, a system that measures RNAs and proteins in intact biological samples at subcellular resolution by performing multiple cycles of nucleic acid hybridization of fluorescent molecular barcodes. We demonstrate that spatial molecular imaging has high sensitivity (one or two copies per cell) and very low error rate (0.0092 false calls per cell) and background (~0.04 counts per cell). The imaging system generates three-dimensional, super-resolution localization of analytes at ~2 million cells per sample. Cell segmentation is morphology based using antibodies, compatible with formalin-fixed, paraffin-embedded samples. We measured multiomic data (980 RNAs and 108 proteins) at subcellular resolution in formalin-fixed, paraffin-embedded tissues (nonsmall cell lung and breast cancer) and identified &amp;gt;18 distinct cell types, ten unique tumor microenvironments and 100 pairwise ligand–receptor interactions. Data on &amp;gt;800,000 single cells and ~260 million transcripts can be accessed at http://nanostring.com/CosMx-dataset . Hundreds of RNAs and proteins are imaged in fixed tissue at subcellular resolution.},
  archive      = {J_NBT},
  author       = {He, Shanshan and Bhatt, Ruchir and Brown, Carl and Brown, Emily A. and Buhr, Derek L. and Chantranuvatana, Kan and Danaher, Patrick and Dunaway, Dwayne and Garrison, Ryan G. and Geiss, Gary and Gregory, Mark T. and Hoang, Margaret L. and Khafizov, Rustem and Killingbeck, Emily E. and Kim, Dae and Kim, Tae Kyung and Kim, Youngmi and Klock, Andrew and Korukonda, Mithra and Kutchma, Alecksandr and Lewis, Zachary R. and Liang, Yan and Nelson, Jeffrey S. and Ong, Giang T. and Perillo, Evan P. and Phan, Joseph C. and Phan-Everson, Tien and Piazza, Erin and Rane, Tushar and Reitz, Zachary and Rhodes, Michael and Rosenbloom, Alyssa and Ross, David and Sato, Hiromi and Wardhani, Aster W. and Williams-Wietzikoski, Corey A. and Wu, Lidan and Beechem, Joseph M.},
  doi          = {10.1038/s41587-022-01483-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1794-1806},
  shortjournal = {Nature Biotech.},
  title        = {High-plex imaging of RNA and proteins at subcellular resolution in fixed tissue by spatial molecular imaging},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High-throughput total RNA sequencing in single cells using
VASA-seq. <em>NBT</em>, <em>40</em>(12), 1780–1793. (<a
href="https://doi.org/10.1038/s41587-022-01361-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most methods for single-cell transcriptome sequencing amplify the termini of polyadenylated transcripts, capturing only a small fraction of the total cellular transcriptome. This precludes the detection of many long non-coding, short non-coding and non-polyadenylated protein-coding transcripts and hinders alternative splicing analysis. We, therefore, developed VASA-seq to detect the total transcriptome in single cells, which is enabled by fragmenting and tailing all RNA molecules subsequent to cell lysis. The method is compatible with both plate-based formats and droplet microfluidics. We applied VASA-seq to more than 30,000 single cells in the developing mouse embryo during gastrulation and early organogenesis. Analyzing the dynamics of the total single-cell transcriptome, we discovered cell type markers, many based on non-coding RNA, and performed in vivo cell cycle analysis via detection of non-polyadenylated histone genes. RNA velocity characterization was improved, accurately retracing blood maturation trajectories. Moreover, our VASA-seq data provide a comprehensive analysis of alternative splicing during mammalian development, which highlighted substantial rearrangements during blood development and heart morphogenesis. VASA-seq sequences total RNA in single cells, revealing non-coding expression and splicing patterns.},
  archive      = {J_NBT},
  author       = {Salmen, Fredrik and De Jonghe, Joachim and Kaminski, Tomasz S. and Alemany, Anna and Parada, Guillermo E. and Verity-Legg, Joe and Yanagida, Ayaka and Kohler, Timo N. and Battich, Nicholas and van den Brekel, Floris and Ellermann, Anna L. and Arias, Alfonso Martinez and Nichols, Jennifer and Hemberg, Martin and Hollfelder, Florian and van Oudenaarden, Alexander},
  doi          = {10.1038/s41587-022-01361-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1780-1793},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput total RNA sequencing in single cells using VASA-seq},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhancing untargeted metabolomics using metadata-based
source annotation. <em>NBT</em>, <em>40</em>(12), 1774–1779. (<a
href="https://doi.org/10.1038/s41587-022-01368-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human untargeted metabolomics studies annotate only ~10\% of molecular features. We introduce reference-data-driven analysis to match metabolomics tandem mass spectrometry (MS/MS) data against metadata-annotated source data as a pseudo-MS/MS reference library. Applying this approach to food source data, we show that it increases MS/MS spectral usage 5.1-fold over conventional structural MS/MS library matches and allows empirical assessment of dietary patterns from untargeted data. Metabolomics is improved by using a reference library of both known and unknown molecules.},
  archive      = {J_NBT},
  author       = {Gauglitz, Julia M. and West, Kiana A. and Bittremieux, Wout and Williams, Candace L. and Weldon, Kelly C. and Panitchpakdi, Morgan and Di Ottavio, Francesca and Aceves, Christine M. and Brown, Elizabeth and Sikora, Nicole C. and Jarmusch, Alan K. and Martino, Cameron and Tripathi, Anupriya and Meehan, Michael J. and Dorrestein, Kathleen and Shaffer, Justin P. and Coras, Roxana and Vargas, Fernando and Goldasich, Lindsay DeRight and Schwartz, Tara and Bryant, MacKenzie and Humphrey, Gregory and Johnson, Abigail J. and Spengler, Katharina and Belda-Ferre, Pedro and Diaz, Edgar and McDonald, Daniel and Zhu, Qiyun and Elijah, Emmanuel O. and Wang, Mingxun and Marotz, Clarisse and Sprecher, Kate E. and Vargas-Robles, Daniela and Withrow, Dana and Ackermann, Gail and Herrera, Lourdes and Bradford, Barry J. and Marques, Lucas Maciel Mauriz and Amaral, Juliano Geraldo and Silva, Rodrigo Moreira and Veras, Flavio Protasio and Cunha, Thiago Mattar and Oliveira, Rene Donizeti Ribeiro and Louzada-Junior, Paulo and Mills, Robert H. and Piotrowski, Paulina K. and Servetas, Stephanie L. and Da Silva, Sandra M. and Jones, Christina M. and Lin, Nancy J. and Lippa, Katrice A. and Jackson, Scott A. and Daouk, Rima Kaddurah and Galasko, Douglas and Dulai, Parambir S. and Kalashnikova, Tatyana I. and Wittenberg, Curt and Terkeltaub, Robert and Doty, Megan M. and Kim, Jae H. and Rhee, Kyung E. and Beauchamp-Walters, Julia and Wright, Kenneth P. and Dominguez-Bello, Maria Gloria and Manary, Mark and Oliveira, Michelli F. and Boland, Brigid S. and Lopes, Norberto Peporine and Guma, Monica and Swafford, Austin D. and Dutton, Rachel J. and Knight, Rob and Dorrestein, Pieter C.},
  doi          = {10.1038/s41587-022-01368-1},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1774-1779},
  shortjournal = {Nature Biotech.},
  title        = {Enhancing untargeted metabolomics using metadata-based source annotation},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Plant hormone sensors as scaffolds for biosensor design.
<em>NBT</em>, <em>40</em>(12), 1772–1773. (<a
href="https://doi.org/10.1038/s41587-022-01373-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most plant hormone sensors, including the abscisic acid receptor PYR1, function through chemically induced dimerization. Using computationally designed libraries of PYR1, we created high-affinity receptors for 21 structurally diverse ligands, setting the stage for large-scale small-molecule biosensor development},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01373-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1772-1773},
  shortjournal = {Nature Biotech.},
  title        = {Plant hormone sensors as scaffolds for biosensor design},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identifying drug combinations that enhance treatment
responses mediated by the tumor microenvironment. <em>NBT</em>,
<em>40</em>(12), 1770–1771. (<a
href="https://doi.org/10.1038/s41587-022-01380-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug combinations predicted to increase tumor cell death directly and by creating strong antitumor tumor microenvironments were identified by computational analysis of local responses to combinations of anticancer drugs delivered inside a tumor. Such predicted drug combinations were highly effective when administered systemically in mouse models of breast cancer.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01380-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1770-1771},
  shortjournal = {Nature Biotech.},
  title        = {Identifying drug combinations that enhance treatment responses mediated by the tumor microenvironment},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Wastewater is a robust proxy for monitoring circulating
SARS-CoV-2 variants. <em>NBT</em>, <em>40</em>(12), 1768–1769. (<a
href="https://doi.org/10.1038/s41587-022-01388-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic surveillance is important for pandemic preparedness. By deep sequencing of wastewater samples, we monitored the genetic dynamics of circulating SARS-CoV-2 virus variants in Austria from December 2020 to February 2022. We show how wastewater-based epidemiology enables robust quantification of viral spatiotemporal dynamics as well as the detection of emergent new variants.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01388-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1768-1769},
  shortjournal = {Nature Biotech.},
  title        = {Wastewater is a robust proxy for monitoring circulating SARS-CoV-2 variants},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Profiling the total single-cell transciptome using droplet
microfluidics. <em>NBT</em>, <em>40</em>(12), 1766–1767. (<a
href="https://doi.org/10.1038/s41587-022-01370-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current high-throughput single-cell methods detect only a small part of the transcriptome. The workflow presented here integrates molecular analysis and droplet microfluidics to derive total transcriptomic atlases that encompass alternative splicing and non-coding transcripts in large numbers of single cells. The utility of this method is demonstrated by analysis of mouse gastrulation and early organogenesis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01370-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1766-1767},
  shortjournal = {Nature Biotech.},
  title        = {Profiling the total single-cell transciptome using droplet microfluidics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genotoxins from gut bacteria. <em>NBT</em>, <em>40</em>(12),
1765. (<a href="https://doi.org/10.1038/s41587-022-01605-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Leake, Isobel},
  doi          = {10.1038/s41587-022-01605-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1765},
  shortjournal = {Nature Biotech.},
  title        = {Genotoxins from gut bacteria},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-molecule peptide sequencing on semiconductor chips.
<em>NBT</em>, <em>40</em>(12), 1765. (<a
href="https://doi.org/10.1038/s41587-022-01614-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Otto, Grant},
  doi          = {10.1038/s41587-022-01614-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1765},
  shortjournal = {Nature Biotech.},
  title        = {Single-molecule peptide sequencing on semiconductor chips},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Imaging epigenomics in the brain. <em>NBT</em>,
<em>40</em>(12), 1765. (<a
href="https://doi.org/10.1038/s41587-022-01615-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Wrighton, Katharine H.},
  doi          = {10.1038/s41587-022-01615-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1765},
  shortjournal = {Nature Biotech.},
  title        = {Imaging epigenomics in the brain},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Biomedical imaging. <em>NBT</em>, <em>40</em>(12), 1764. (<a
href="https://doi.org/10.1038/s41587-022-01608-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods and devices for improved imaging in the biomedical field.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01608-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1764},
  shortjournal = {Nature Biotech.},
  title        = {Biomedical imaging},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The humira patent thicket, the noerr–pennington doctrine and
antitrust’s patent problem. <em>NBT</em>, <em>40</em>(12), 1761–1763.
(<a href="https://doi.org/10.1038/s41587-022-01583-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent federal appellate court decision will make challenging future pharmaceutical ‘patent thickets’ an uphill battle.},
  archive      = {J_NBT},
  author       = {Knox, Ryan and Curfman, Gregory},
  doi          = {10.1038/s41587-022-01583-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1761-1763},
  shortjournal = {Nature Biotech.},
  title        = {The humira patent thicket, the Noerr–Pennington doctrine and antitrust’s patent problem},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Biopharmaceutical benchmarks 2022. <em>NBT</em>,
<em>40</em>(12), 1722–1760. (<a
href="https://doi.org/10.1038/s41587-022-01582-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monoclonal antibodies as a group continue to lead biopharmaceuticals in numbers of approvals and sales, although COVID-19 vaccines shot to the top of the list of highest-grossing individual products.},
  archive      = {J_NBT},
  author       = {Walsh, Gary and Walsh, Eithne},
  doi          = {10.1038/s41587-022-01582-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1722-1760},
  shortjournal = {Nature Biotech.},
  title        = {Biopharmaceutical benchmarks 2022},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improving combination drug trials using “definitive
screening designs.” <em>NBT</em>, <em>40</em>(12), 1720–1721. (<a
href="https://doi.org/10.1038/s41587-022-01521-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Dodds, Michael and Roberts, James and Finrow, Brian},
  doi          = {10.1038/s41587-022-01521-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1720-1721},
  shortjournal = {Nature Biotech.},
  title        = {Improving combination drug trials using ‘definitive screening designs’},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Long-term preservation of liver grafts brings
“off-the-shelf” organs closer. <em>NBT</em>, <em>40</em>(12), 1717–1719.
(<a href="https://doi.org/10.1038/s41587-022-01523-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Martins, Paulo N. and Ghinolfi, Davide},
  doi          = {10.1038/s41587-022-01523-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1717-1719},
  shortjournal = {Nature Biotech.},
  title        = {Long-term preservation of liver grafts brings ‘off-the-shelf’ organs closer},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Innovating manufacturing technology in emerging economies.
<em>NBT</em>, <em>40</em>(12), 1714–1716. (<a
href="https://doi.org/10.1038/s41587-022-01499-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Rathore, Anurag and Shereef, Faheem},
  doi          = {10.1038/s41587-022-01499-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1714-1716},
  shortjournal = {Nature Biotech.},
  title        = {Innovating manufacturing technology in emerging economies},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(12), 1708. (<a
href="https://doi.org/10.1038/s41587-022-01609-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01609-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1708},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pompe disease treated before birth. <em>NBT</em>,
<em>40</em>(12), 1707. (<a
href="https://doi.org/10.1038/s41587-022-01621-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01621-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1707},
  shortjournal = {Nature Biotech.},
  title        = {Pompe disease treated before birth},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Big mRNA players focus on flu vaccines. <em>NBT</em>,
<em>40</em>(12), 1706. (<a
href="https://doi.org/10.1038/s41587-022-01620-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01620-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1706},
  shortjournal = {Nature Biotech.},
  title        = {Big mRNA players focus on flu vaccines},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Parkinson’s disease drug hunters think outside the
α-synuclein box. <em>NBT</em>, <em>40</em>(12), 1705–1707. (<a
href="https://doi.org/10.1038/s41587-022-01610-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent trial failures have called into question the central role of α-synuclein in Parkinson’s disease pathology. Although many continue to target α-synuclein aggregates in the brain, others are paying attention to the endolysosomal pathway, mitochondria and even glucose metabolism.},
  archive      = {J_NBT},
  author       = {Johnson, Ben},
  doi          = {10.1038/s41587-022-01610-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1705-1707},
  shortjournal = {Nature Biotech.},
  title        = {Parkinson’s disease drug hunters think outside the α-synuclein box},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Noma: From grasshopper brews to age-old “garum.”
<em>NBT</em>, <em>40</em>(12), 1704. (<a
href="https://doi.org/10.1038/s41587-022-01622-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-022-01622-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1704},
  shortjournal = {Nature Biotech.},
  title        = {Noma: From grasshopper brews to age-old ‘garum’},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Alzheimer’s space heats up. <em>NBT</em>, <em>40</em>(12),
1703. (<a href="https://doi.org/10.1038/s41587-022-01619-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01619-1},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1703},
  shortjournal = {Nature Biotech.},
  title        = {Alzheimer’s space heats up},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). TYK2-blocking agent showcases power of atypical kinase.
<em>NBT</em>, <em>40</em>(12), 1701–1704. (<a
href="https://doi.org/10.1038/s41587-022-01602-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immunomodulatory drug treats psoriasis by a new mechanism: the selective deactivation of TYK2 through its catalytically defective domain, the pseudokinase.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01602-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1701-1704},
  shortjournal = {Nature Biotech.},
  title        = {TYK2-blocking agent showcases power of atypical kinase},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Global health equity: The next waiver. <em>NBT</em>,
<em>40</em>(12), 1699. (<a
href="https://doi.org/10.1038/s41587-022-01623-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Putting life science in the service of the global South requires reconstitution of the biotech model, not patent waivers},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01623-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1699},
  shortjournal = {Nature Biotech.},
  title        = {Global health equity: The next waiver},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The lysosomal degraders. <em>NBT</em>, <em>40</em>(12),
1709–1713. (<a
href="https://doi.org/10.1038/s41587-022-01594-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by PROTACs, new degraders are emerging that harness endocytosis and autophagy to send a wide array of targets to the lysosome for destruction.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-022-01594-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1709-1713},
  shortjournal = {Nature Biotech.},
  title        = {The lysosomal degraders},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). People. <em>NBT</em>, <em>40</em>(11), 1698. (<a
href="https://doi.org/10.1038/s41587-022-01550-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01550-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1698},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Third-quarter biotech job picture. <em>NBT</em>,
<em>40</em>(11), 1697. (<a
href="https://doi.org/10.1038/s41587-022-01542-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-022-01542-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1697},
  shortjournal = {Nature Biotech.},
  title        = {Third-quarter biotech job picture},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mind the gap: Closing the growing chasm between academia and
industry. <em>NBT</em>, <em>40</em>(11), 1693–1696. (<a
href="https://doi.org/10.1038/s41587-022-01543-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Academia and the pharmaceutical industry must unite to offer more comprehensive opportunities for those wanting to stay in research in the field of drug development.},
  archive      = {J_NBT},
  author       = {Spicer, Alexander J. and Colcomb, Pierre-Albert and Kraft, Ann},
  doi          = {10.1038/s41587-022-01543-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1693-1696},
  shortjournal = {Nature Biotech.},
  title        = {Mind the gap: Closing the growing chasm between academia and industry},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Retraction note: Rescue of the spinal muscular atrophy
phenotype in a mouse model by early postnatal delivery of SMN.
<em>NBT</em>, <em>40</em>(11), 1692. (<a
href="https://doi.org/10.1038/s41587-022-01497-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Foust, Kevin D. and Wang, Xueyong and McGovern, Vicki L. and Braun, Lyndsey and Bevan, Adam K. and Haidet, Amanda M. and Le, Thanh T. and Morales, Pablo R. and Rich, Mark M. and Burghes, Arthur H. M. and Kaspar, Brian K.},
  doi          = {10.1038/s41587-022-01497-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1692},
  shortjournal = {Nature Biotech.},
  title        = {Retraction note: Rescue of the spinal muscular atrophy phenotype in a mouse model by early postnatal delivery of SMN},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Publisher correction: Single-sequence protein structure
prediction using a language model and deep learning. <em>NBT</em>,
<em>40</em>(11), 1692. (<a
href="https://doi.org/10.1038/s41587-022-01556-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chowdhury, Ratul and Bouatta, Nazim and Biswas, Surojit and Floristean, Christina and Kharkar, Anant and Roy, Koushik and Rochereau, Charlotte and Ahdritz, Gustaf and Zhang, Joanna and Church, George M. and Sorger, Peter K. and AlQuraishi, Mohammed},
  doi          = {10.1038/s41587-022-01556-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1692},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Single-sequence protein structure prediction using a language model and deep learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Massively parallel phenotyping of coding
variants in cancer with perturb-seq. <em>NBT</em>, <em>40</em>(11),
1691. (<a href="https://doi.org/10.1038/s41587-022-01495-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ursu, Oana and Neal, James T. and Shea, Emily and Thakore, Pratiksha I. and Jerby-Arnon, Livnat and Nguyen, Lan and Dionne, Danielle and Diaz, Celeste and Bauman, Julia and Mosaad, Mariam Mounir and Fagre, Christian and Lo, April and McSharry, Maria and Giacomelli, Andrew O. and Ly, Seav Huong and Rozenblatt-Rosen, Orit and Hahn, William C. and Aguirre, Andrew J. and Berger, Alice H. and Regev, Aviv and Boehm, Jesse S.},
  doi          = {10.1038/s41587-022-01495-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1691},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Massively parallel phenotyping of coding variants in cancer with perturb-seq},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Hypoimmunogenic derivatives of induced
pluripotent stem cells evade immune rejection in fully immunocompetent
allogeneic recipients. <em>NBT</em>, <em>40</em>(11), 1690. (<a
href="https://doi.org/10.1038/s41587-022-01426-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Deuse, Tobias and Hu, Xiaomeng and Gravina, Alessia and Wang, Dong and Tediashvili, Grigol and De, Chandrav and Thayer, William O. and Wahl, Angela and Garcia, J. Victor and Reichenspurner, Hermann and Davis, Mark M. and Lanier, Lewis L. and Schrepfer, Sonja},
  doi          = {10.1038/s41587-022-01426-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1690},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Hypoimmunogenic derivatives of induced pluripotent stem cells evade immune rejection in fully immunocompetent allogeneic recipients},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Rapid, scalable assessment of SARS-CoV-2 cellular immunity
by whole-blood PCR. <em>NBT</em>, <em>40</em>(11), 1680–1689. (<a
href="https://doi.org/10.1038/s41587-022-01347-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fast, high-throughput methods for measuring the level and duration of protective immune responses to SARS-CoV-2 are needed to anticipate the risk of breakthrough infections. Here we report the development of two quantitative PCR assays for SARS-CoV-2-specific T cell activation. The assays are rapid, internally normalized and probe-based: qTACT requires RNA extraction and dqTACT avoids sample preparation steps. Both assays rely on the quantification of CXCL10 messenger RNA, a chemokine whose expression is strongly correlated with activation of antigen-specific T cells. On restimulation of whole-blood cells with SARS-CoV-2 viral antigens, viral-specific T cells secrete IFN-γ, which stimulates monocytes to produce CXCL10. CXCL10 mRNA can thus serve as a proxy to quantify cellular immunity. Our assays may allow large-scale monitoring of the magnitude and duration of functional T cell immunity to SARS-CoV-2, thus helping to prioritize revaccination strategies in vulnerable populations. The T cell response to SARS-CoV-2 is detected by a PCR assay on whole blood.},
  archive      = {J_NBT},
  author       = {Schwarz, Megan and Torre, Denis and Lozano-Ojalvo, Daniel and Tan, Anthony T. and Tabaglio, Tommaso and Mzoughi, Slim and Sanchez-Tarjuelo, Rodrigo and Le Bert, Nina and Lim, Joey Ming Er and Hatem, Sandra and Tuballes, Kevin and Camara, Carmen and Lopez-Granados, Eduardo and Paz-Artal, Estela and Correa-Rocha, Rafael and Ortiz, Alberto and Lopez-Hoyos, Marcos and Portoles, Jose and Cervera, Isabel and Gonzalez-Perez, Maria and Bodega-Mayor, Irene and Conde, Patricia and Oteo-Iglesias, Jesús and Borobia, Alberto M. and Carcas, Antonio J. and Frías, Jesús and Belda-Iniesta, Cristóbal and Ho, Jessica S. Y. and Nunez, Kemuel and Hekmaty, Saboor and Mohammed, Kevin and Marsiglia, William M. and Carreño, Juan Manuel and Dar, Arvin C. and Berin, Cecilia and Nicoletti, Giuseppe and Della Noce, Isabella and Colombo, Lorenzo and Lapucci, Cristina and Santoro, Graziano and Ferrari, Maurizio and Nie, Kai and Patel, Manishkumar and Barcessat, Vanessa and Gnjatic, Sacha and Harris, Jocelyn and Sebra, Robert and Merad, Miriam and Krammer, Florian and Kim-schulze, Seunghee and Marazzi, Ivan and Bertoletti, Antonio and Ochando, Jordi and Guccione, Ernesto},
  doi          = {10.1038/s41587-022-01347-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1680-1689},
  shortjournal = {Nature Biotech.},
  title        = {Rapid, scalable assessment of SARS-CoV-2 cellular immunity by whole-blood PCR},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A red light–responsive photoswitch for deep tissue
optogenetics. <em>NBT</em>, <em>40</em>(11), 1672–1679. (<a
href="https://doi.org/10.1038/s41587-022-01351-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Red light penetrates deep into mammalian tissues and has low phototoxicity, but few optogenetic tools that use red light have been developed. Here we present MagRed, a red light–activatable photoswitch that consists of a red light–absorbing bacterial phytochrome incorporating a mammalian endogenous chromophore, biliverdin and a photo-state-specific binder that we developed using Affibody library selection. Red light illumination triggers the binding of the two components of MagRed and the assembly of split-proteins fused to them. Using MagRed, we developed a red light–activatable Cre recombinase, which enables light-activatable DNA recombination deep in mammalian tissues. We also created red light–inducible transcriptional regulators based on CRISPR–Cas9 that enable an up to 378-fold activation (average, 135-fold induction) of multiple endogenous target genes. MagRed will facilitate optogenetic applications deep in mammalian organisms in a variety of biological research areas. A photoswitch based on a bacterial phytochrome enables optogenetic manipulations using red light.},
  archive      = {J_NBT},
  author       = {Kuwasaki, Yuto and Suzuki, Kazushi and Yu, Gaigai and Yamamoto, Shota and Otabe, Takahiro and Kakihara, Yuki and Nishiwaki, Michiru and Miyake, Keita and Fushimi, Keiji and Bekdash, Ramsey and Shimizu, Yoshihiro and Narikawa, Rei and Nakajima, Takahiro and Yazawa, Masayuki and Sato, Moritoshi},
  doi          = {10.1038/s41587-022-01351-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1672-1679},
  shortjournal = {Nature Biotech.},
  title        = {A red light–responsive photoswitch for deep tissue optogenetics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep tissue multi-photon imaging using adaptive optics with
direct focus sensing and shaping. <em>NBT</em>, <em>40</em>(11),
1663–1671. (<a
href="https://doi.org/10.1038/s41587-022-01343-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-resolution optical imaging deep in tissues is challenging because of optical aberrations and scattering of light caused by the complex structure of living matter. Here we present an adaptive optics three-photon microscope based on analog lock-in phase detection for focus sensing and shaping (ALPHA-FSS). ALPHA-FSS accurately measures and effectively compensates for both aberrations and scattering induced by specimens and recovers subcellular resolution at depth. A conjugate adaptive optics configuration with remote focusing enables in vivo imaging of fine neuronal structures in the mouse cortex through the intact skull up to a depth of 750 µm below the pia, enabling near-non-invasive high-resolution microscopy in cortex. Functional calcium imaging with high sensitivity and high-precision laser-mediated microsurgery through the intact skull were also demonstrated. Moreover, we achieved in vivo high-resolution imaging of the deep cortex and subcortical hippocampus up to 1.1 mm below the pia within the intact brain. ALPHA-FSS enables high-resolution imaging deep in the mouse brain.},
  archive      = {J_NBT},
  author       = {Qin, Zhongya and She, Zhentao and Chen, Congping and Wu, Wanjie and Lau, Jackie K. Y. and Ip, Nancy Y. and Qu, Jianan Y.},
  doi          = {10.1038/s41587-022-01343-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1663-1671},
  shortjournal = {Nature Biotech.},
  title        = {Deep tissue multi-photon imaging using adaptive optics with direct focus sensing and shaping},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatiotemporal multiplexed immunofluorescence imaging of
living cells and tissues with bioorthogonal cycling of fluorescent
probes. <em>NBT</em>, <em>40</em>(11), 1654–1662. (<a
href="https://doi.org/10.1038/s41587-022-01339-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells in complex organisms undergo frequent functional changes, but few methods allow comprehensive longitudinal profiling of living cells. Here we introduce scission-accelerated fluorophore exchange (SAFE), a method for multiplexed temporospatial imaging of living cells with immunofluorescence. SAFE uses a rapid bioorthogonal click chemistry to remove immunofluorescent signals from the surface of labeled cells, cycling the nanomolar-concentration reagents in seconds and enabling multiple rounds of staining of the same samples. It is non-toxic and functional in both dispersed cells and intact living tissues. We demonstrate multiparameter (n ≥ 14), non-disruptive imaging of murine peripheral blood mononuclear and bone marrow cells to profile cellular differentiation. We also show longitudinal multiplexed imaging of bone marrow progenitor cells as they develop into neutrophils over 6 days and real-time multiplexed cycling of living mouse hepatic tissues. We anticipate that SAFE will find broad utility for investigating physiologic dynamics in living systems. Live cells and tissues are imaged over long time periods using a fast, non-toxic click chemistry.},
  archive      = {J_NBT},
  author       = {Ko, Jina and Wilkovitsch, Martin and Oh, Juhyun and Kohler, Rainer H. and Bolli, Evangelia and Pittet, Mikael J. and Vinegoni, Claudio and Sykes, David B. and Mikula, Hannes and Weissleder, Ralph and Carlson, Jonathan C. T.},
  doi          = {10.1038/s41587-022-01339-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1654-1662},
  shortjournal = {Nature Biotech.},
  title        = {Spatiotemporal multiplexed immunofluorescence imaging of living cells and tissues with bioorthogonal cycling of fluorescent probes},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Variant to function mapping at single-cell resolution
through network propagation. <em>NBT</em>, <em>40</em>(11), 1644–1653.
(<a href="https://doi.org/10.1038/s41587-022-01341-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies in combination with single-cell genomic atlases can provide insights into the mechanisms of disease-causal genetic variation. However, identification of disease-relevant or trait-relevant cell types, states and trajectories is often hampered by sparsity and noise, particularly in the analysis of single-cell epigenomic data. To overcome these challenges, we present SCAVENGE, a computational algorithm that uses network propagation to map causal variants to their relevant cellular context at single-cell resolution. We demonstrate how SCAVENGE can help identify key biological mechanisms underlying human genetic variation, applying the method to blood traits at distinct stages of human hematopoiesis, to monocyte subsets that increase the risk for severe Coronavirus Disease 2019 (COVID-19) and to intermediate lymphocyte developmental states that predispose to acute leukemia. Our approach not only provides a framework for enabling variant-to-function insights at single-cell resolution but also suggests a more general strategy for maximizing the inferences that can be made using single-cell genomic data. SCAVENGE identifies causal variants from single-cell epigenomic data.},
  archive      = {J_NBT},
  author       = {Yu, Fulong and Cato, Liam D. and Weng, Chen and Liggett, L. Alexander and Jeon, Soyoung and Xu, Keren and Chiang, Charleston W. K. and Wiemels, Joseph L. and Weissman, Jonathan S. and de Smith, Adam J. and Sankaran, Vijay G.},
  doi          = {10.1038/s41587-022-01341-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1644-1653},
  shortjournal = {Nature Biotech.},
  title        = {Variant to function mapping at single-cell resolution through network propagation},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide mapping of somatic mutation rates uncovers
drivers of cancer. <em>NBT</em>, <em>40</em>(11), 1634–1643. (<a
href="https://doi.org/10.1038/s41587-022-01353-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of cancer driver mutations that confer a proliferative advantage is central to understanding cancer; however, searches have often been limited to protein-coding sequences and specific non-coding elements (for example, promoters) because of the challenge of modeling the highly variable somatic mutation rates observed across tumor genomes. Here we present Dig, a method to search for driver elements and mutations anywhere in the genome. We use deep neural networks to map cancer-specific mutation rates genome-wide at kilobase-scale resolution. These estimates are then refined to search for evidence of driver mutations under positive selection throughout the genome by comparing observed to expected mutation counts. We mapped mutation rates for 37 cancer types and applied these maps to identify putative drivers within intronic cryptic splice regions, 5′ untranslated regions and infrequently mutated genes. Our high-resolution mutation rate maps, available for web-based exploration, are a resource to enable driver discovery genome-wide. Cancer driver mutations are identified by predicting neutral mutation rates across the entire genome.},
  archive      = {J_NBT},
  author       = {Sherman, Maxwell A. and Yaari, Adam U. and Priebe, Oliver and Dietlein, Felix and Loh, Po-Ru and Berger, Bonnie},
  doi          = {10.1038/s41587-022-01353-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1634-1643},
  shortjournal = {Nature Biotech.},
  title        = {Genome-wide mapping of somatic mutation rates uncovers drivers of cancer},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Estimation of tumor cell total mRNA expression in 15 cancer
types predicts disease progression. <em>NBT</em>, <em>40</em>(11),
1624–1633. (<a
href="https://doi.org/10.1038/s41587-022-01342-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing studies have suggested that total mRNA content correlates with tumor phenotypes. Technical and analytical challenges, however, have so far impeded at-scale pan-cancer examination of total mRNA content. Here we present a method to quantify tumor-specific total mRNA expression (TmS) from bulk sequencing data, taking into account tumor transcript proportion, purity and ploidy, which are estimated through transcriptomic/genomic deconvolution. We estimate and validate TmS in 6,590 patient tumors across 15 cancer types, identifying significant inter-tumor variability. Across cancers, high TmS is associated with increased risk of disease progression and death. TmS is influenced by cancer-specific patterns of gene alteration and intra-tumor genetic heterogeneity as well as by pan-cancer trends in metabolic dysregulation. Taken together, our results indicate that measuring cell-type-specific total mRNA expression in tumor cells predicts tumor phenotypes and clinical outcomes. Total mRNA expression in cancer cells is a marker for disease progression and death.},
  archive      = {J_NBT},
  author       = {Cao, Shaolong and Wang, Jennifer R. and Ji, Shuangxi and Yang, Peng and Dai, Yaoyi and Guo, Shuai and Montierth, Matthew D. and Shen, John Paul and Zhao, Xiao and Chen, Jingxiao and Lee, Jaewon James and Guerrero, Paola A. and Spetsieris, Nicholas and Engedal, Nikolai and Taavitsainen, Sinja and Yu, Kaixian and Livingstone, Julie and Bhandari, Vinayak and Hubert, Shawna M. and Daw, Najat C. and Futreal, P. Andrew and Efstathiou, Eleni and Lim, Bora and Viale, Andrea and Zhang, Jianjun and Nykter, Matti and Czerniak, Bogdan A. and Brown, Powel H. and Swanton, Charles and Msaouel, Pavlos and Maitra, Anirban and Kopetz, Scott and Campbell, Peter and Speed, Terence P. and Boutros, Paul C. and Zhu, Hongtu and Urbanucci, Alfonso and Demeulemeester, Jonas and Van Loo, Peter and Wang, Wenyi},
  doi          = {10.1038/s41587-022-01342-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1624-1633},
  shortjournal = {Nature Biotech.},
  title        = {Estimation of tumor cell total mRNA expression in 15 cancer types predicts disease progression},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Single-sequence protein structure prediction using a
language model and deep learning. <em>NBT</em>, <em>40</em>(11),
1617–1623. (<a
href="https://doi.org/10.1038/s41587-022-01432-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AlphaFold2 and related computational systems predict protein structure using deep learning and co-evolutionary relationships encoded in multiple sequence alignments (MSAs). Despite high prediction accuracy achieved by these systems, challenges remain in (1) prediction of orphan and rapidly evolving proteins for which an MSA cannot be generated; (2) rapid exploration of designed structures; and (3) understanding the rules governing spontaneous polypeptide folding in solution. Here we report development of an end-to-end differentiable recurrent geometric network (RGN) that uses a protein language model (AminoBERT) to learn latent structural information from unaligned proteins. A linked geometric module compactly represents Cα backbone geometry in a translationally and rotationally invariant way. On average, RGN2 outperforms AlphaFold2 and RoseTTAFold on orphan proteins and classes of designed proteins while achieving up to a 106-fold reduction in compute time. These findings demonstrate the practical and theoretical strengths of protein language models relative to MSAs in structure prediction. RGN2 predicts a protein’s structure from its sequence without a multiple sequence alignment.},
  archive      = {J_NBT},
  author       = {Chowdhury, Ratul and Bouatta, Nazim and Biswas, Surojit and Floristean, Christina and Kharkar, Anant and Roy, Koushik and Rochereau, Charlotte and Ahdritz, Gustaf and Zhang, Joanna and Church, George M. and Sorger, Peter K. and AlQuraishi, Mohammed},
  doi          = {10.1038/s41587-022-01432-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1617-1623},
  shortjournal = {Nature Biotech.},
  title        = {Single-sequence protein structure prediction using a language model and deep learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Transplantation of a human liver following 3 days of ex situ
normothermic preservation. <em>NBT</em>, <em>40</em>(11), 1610–1616. (<a
href="https://doi.org/10.1038/s41587-022-01354-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current organ preservation methods provide a narrow window (usually &amp;lt;12 hours) to assess, transport and implant donor grafts for human transplantation. Here we report the transplantation of a human liver discarded by all centers, which could be preserved for several days using ex situ normothermic machine perfusion. The transplanted liver exhibited normal function, with minimal reperfusion injury and the need for only a minimal immunosuppressive regimen. The patient rapidly recovered a normal quality of life without any signs of liver damage, such as rejection or injury to the bile ducts, according to a 1-year follow up. This inaugural clinical success opens new horizons in clinical research and promises an extended time window of up to 10 days for assessment of viability of donor organs as well as converting an urgent and highly demanding surgery into an elective procedure. A human liver is transplanted successfully after preservation for 3 days via machine perfusion.},
  archive      = {J_NBT},
  author       = {Clavien, Pierre-Alain and Dutkowski, Philipp and Mueller, Matteo and Eshmuminov, Dilmurodjon and Bautista Borrego, Lucia and Weber, Achim and Muellhaupt, Beat and Sousa Da Silva, Richard X. and Burg, Brian R. and Rudolf von Rohr, Philipp and Schuler, Martin J. and Becker, Dustin and Hefti, Max and Tibbitt, Mark W.},
  doi          = {10.1038/s41587-022-01354-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1610-1616},
  shortjournal = {Nature Biotech.},
  title        = {Transplantation of a human liver following 3 days of ex situ normothermic preservation},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Directed evolution and selection of biostable l-DNA aptamers
with a mirror-image DNA polymerase. <em>NBT</em>, <em>40</em>(11),
1601–1609. (<a
href="https://doi.org/10.1038/s41587-022-01337-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mirror-image aptamers made from chirally inverted nucleic acids are nuclease-resistant and exceptionally biostable, opening up opportunities for unique applications. However, the directed evolution and selection of mirror-image aptamers directly from large randomized l-DNA libraries has,﻿ to our knowledge, not been demonstrated previously. Here, we developed a ‘mirror-image selection’ scheme for the directed evolution and selection of biostable l-DNA aptamers with a mirror-image DNA polymerase. We performed iterative rounds of enrichment and mirror-image polymerase chain reaction (PCR) amplification of l-DNA sequences that bind native human thrombin, in conjunction with denaturing gradient gel electrophoresis (DGGE) to isolate individual aptamers and l-DNA sequencing-by-synthesis to determine their sequences. Based on the selected l-DNA aptamers, we designed biostable thrombin sensors and inhibitors, which remained functional in physiologically relevant nuclease-rich environments, even in the presence of human serum that rapidly degraded d-DNA aptamers. Mirror-image selection of biostable l-DNA aptamers directly from large randomized l-DNA libraries greatly expands the range of biomolecules that can be targeted, broadening their applications as biostable sensors, therapeutics and basic research tools. Mirror-image PCR and l-DNA sequencing-by-synthesis enable the directed evolution and selection of functional l-DNA aptamers.},
  archive      = {J_NBT},
  author       = {Chen, Ji and Chen, Mengyin and Zhu, Ting F.},
  doi          = {10.1038/s41587-022-01337-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1601-1609},
  shortjournal = {Nature Biotech.},
  title        = {Directed evolution and selection of biostable l-DNA aptamers with a mirror-image DNA polymerase},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Unlocking the promise of mRNA therapeutics. <em>NBT</em>,
<em>40</em>(11), 1586–1600. (<a
href="https://doi.org/10.1038/s41587-022-01491-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The extraordinary success of mRNA vaccines against coronavirus disease 2019 (COVID-19) has renewed interest in mRNA as a means of delivering therapeutic proteins. Early clinical trials of mRNA therapeutics include studies of paracrine vascular endothelial growth factor (VEGF) mRNA for heart failure and of CRISPR–Cas9 mRNA for a congenital liver-specific storage disease. However, a series of challenges remains to be addressed before mRNA can be established as a general therapeutic modality with broad relevance to both rare and common diseases. An array of new technologies is being developed to surmount these challenges, including approaches to optimize mRNA cargos, lipid carriers with inherent tissue tropism and in vivo percutaneous delivery systems. The judicious integration of these advances may unlock the promise of biologically targeted mRNA therapeutics, beyond vaccines and other immunostimulatory agents, for the treatment of diverse clinical indications. mRNA therapeutics face more challenges than mRNA vaccines, but solutions are on the horizon.},
  archive      = {J_NBT},
  author       = {Rohner, Eduarde and Yang, Ran and Foo, Kylie S. and Goedel, Alexander and Chien, Kenneth R.},
  doi          = {10.1038/s41587-022-01491-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1586-1600},
  shortjournal = {Nature Biotech.},
  title        = {Unlocking the promise of mRNA therapeutics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Measuring SARS-CoV-2 t cell immunity with a scalable
qPCR-based assay. <em>NBT</em>, <em>40</em>(11), 1584–1585. (<a
href="https://doi.org/10.1038/s41587-022-01358-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed two quantitative PCR-based assays to detect SARS-CoV-2-specific T cell immunity: qTACT and dqTACT. The assays quantify CXCL10 mRNA, after incubation of whole blood with viral peptides, as a proxy of an antigen-specific T cell response, and will allow population-level monitoring of cellular immunity to SARS-CoV-2.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01358-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1584-1585},
  shortjournal = {Nature Biotech.},
  title        = {Measuring SARS-CoV-2 t cell immunity with a scalable qPCR-based assay},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards the non-invasive imaging of brain networks and
functions at high resolution. <em>NBT</em>, <em>40</em>(11), 1582–1583.
(<a href="https://doi.org/10.1038/s41587-022-01344-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study describes a microscope combining three-photon excitation and adaptive optics capable of high-resolution in vivo imaging of fine neuronal structures in the mouse cortex through the intact skull. The authors demonstrate the use of this platform to guide precise laser-mediated microsurgery and for accurate and sensitive functional calcium imaging.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01344-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1582-1583},
  shortjournal = {Nature Biotech.},
  title        = {Towards the non-invasive imaging of brain networks and functions at high resolution},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Visualization of living cells and tissues in many colors.
<em>NBT</em>, <em>40</em>(11), 1580–1581. (<a
href="https://doi.org/10.1038/s41587-022-01348-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing technologies can map the intricate spatial distribution of biomolecules and cell types within tissues, but not in specimens that remain alive and intact. This study introduces scission-accelerated fluorophore exchange (SAFE) bioorthogonal imaging tools that enable living cells and tissues to be deeply and serially profiled, revealing their biological dynamics across both space and time.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01348-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1580-1581},
  shortjournal = {Nature Biotech.},
  title        = {Visualization of living cells and tissues in many colors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). First liver transplantation using a graft maintained ex vivo
for several days. <em>NBT</em>, <em>40</em>(11), 1578–1579. (<a
href="https://doi.org/10.1038/s41587-022-01355-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our multidisciplinary team used novel perfusion technology to successfully keep an injured liver graft discarded by all other centers alive for several days, allowing proper graft evaluation and repair. The graft was successfully transplanted into a sick patient, who recovered quickly and enjoyed a normal life at one-year follow up.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01355-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1578-1579},
  shortjournal = {Nature Biotech.},
  title        = {First liver transplantation using a graft maintained ex vivo for several days},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A language model beats alphafold2 on orphans. <em>NBT</em>,
<em>40</em>(11), 1576–1577. (<a
href="https://doi.org/10.1038/s41587-022-01466-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein structure prediction with a language model improves accuracy for orphan and designed proteins.},
  archive      = {J_NBT},
  author       = {Michaud, Jennifer M. and Madani, Ali and Fraser, James S.},
  doi          = {10.1038/s41587-022-01466-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1576-1577},
  shortjournal = {Nature Biotech.},
  title        = {A language model beats alphafold2 on orphans},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Uncovering the global RNA virome. <em>NBT</em>,
<em>40</em>(11), 1575. (<a
href="https://doi.org/10.1038/s41587-022-01571-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Duarte, João H.},
  doi          = {10.1038/s41587-022-01571-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1575},
  shortjournal = {Nature Biotech.},
  title        = {Uncovering the global RNA virome},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). CAR-t cells tackle autoimmunity. <em>NBT</em>,
<em>40</em>(11), 1575. (<a
href="https://doi.org/10.1038/s41587-022-01576-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Duarte, João H.},
  doi          = {10.1038/s41587-022-01576-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1575},
  shortjournal = {Nature Biotech.},
  title        = {CAR-T cells tackle autoimmunity},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial transcriptomics with light barcodes in intact
tissues. <em>NBT</em>, <em>40</em>(11), 1575. (<a
href="https://doi.org/10.1038/s41587-022-01577-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Despang, Alexandra},
  doi          = {10.1038/s41587-022-01577-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1575},
  shortjournal = {Nature Biotech.},
  title        = {Spatial transcriptomics with light barcodes in intact tissues},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Organelle-like scaffolds in bacteria for synthetic biology.
<em>NBT</em>, <em>40</em>(11), 1574. (<a
href="https://doi.org/10.1038/s41587-022-01569-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Barnes, Natalie G.},
  doi          = {10.1038/s41587-022-01569-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1574},
  shortjournal = {Nature Biotech.},
  title        = {Organelle-like scaffolds in bacteria for synthetic biology},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell RNA-seq relates GWAS variants to disease risk.
<em>NBT</em>, <em>40</em>(11), 1574. (<a
href="https://doi.org/10.1038/s41587-022-01570-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Dörr, Anne},
  doi          = {10.1038/s41587-022-01570-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1574},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell RNA-seq relates GWAS variants to disease risk},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Metabolomics. <em>NBT</em>, <em>40</em>(11), 1573. (<a
href="https://doi.org/10.1038/s41587-022-01553-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to employing metabolomic data to diagnose disease states and determine the likelihood that a patient will respond to certain treatments.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01553-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1573},
  shortjournal = {Nature Biotech.},
  title        = {Metabolomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Has the PTAB made a difference in drug settlements and
generic entry? <em>NBT</em>, <em>40</em>(11), 1569–1572. (<a
href="https://doi.org/10.1038/s41587-022-01526-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Legal barriers are preventing generic drug makers from being able to utilize the Patent Trial and Appeal Board for its intended purpose — to serve as a faster and less cumbersome alternative to district court litigation.},
  archive      = {J_NBT},
  author       = {Hovenkamp, Erik and Lemus, Jorge and Rai, Arti and Vishnubhakat, Saurabh},
  doi          = {10.1038/s41587-022-01526-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1569-1572},
  shortjournal = {Nature Biotech.},
  title        = {Has the PTAB made a difference in drug settlements and generic entry?},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The coronavirus standards working group’s roadmap for
improved population testing. <em>NBT</em>, <em>40</em>(11), 1563–1568.
(<a href="https://doi.org/10.1038/s41587-022-01538-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Mercer, Tim and Almond, Neil and Crone, Michael A. and Chain, Patrick S. G. and Deshpande, Alina and Eveleigh, Deepa and Freemont, Paul and Fuchs, Sebastien and Garlick, Russell and Huggett, Jim and Kammel, Martin and Li, Po-E and Milavec, Mojca and Marlowe, Elizabeth M. and O’Sullivan, Denise M. and Page, Mark and Pestano, Gary A. and Suliman, Sara and Simen, Birgitte and Sninsky, John J. and Sopchak, Lynne and Tato, Cristina M. and Vallone, Peter M. and Vandesompele, Jo and White, Thomas J. and Zeichhardt, Heinz and Salit, Marc},
  doi          = {10.1038/s41587-022-01538-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1563-1568},
  shortjournal = {Nature Biotech.},
  title        = {The coronavirus standards working group’s roadmap for improved population testing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Nature biotechnology’s academic spinouts 2021. <em>NBT</em>,
<em>40</em>(11), 1551–1562. (<a
href="https://doi.org/10.1038/s41587-022-01530-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology’s annual survey highlights academic startups that are, among other things, designing circular RNA therapeutics, tackling cancer with arenaviruses, creating psychedelics without the trip, editing genes and cells in vivo, harnessing the power of autoantibodies and editing the epigenome.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael and Garber, Ken and Landhuis, Esther and DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01530-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1551-1562},
  shortjournal = {Nature Biotech.},
  title        = {Nature biotechnology’s academic spinouts 2021},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Pharma backs off biotech acquisitions. <em>NBT</em>,
<em>40</em>(11), 1546–1550. (<a
href="https://doi.org/10.1038/s41587-022-01529-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rising costs and an uncertain global economy have left many potential buyers sitting on their hands — or partnering instead.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-022-01529-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1546-1550},
  shortjournal = {Nature Biotech.},
  title        = {Pharma backs off biotech acquisitions},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). 3Q22 — more malaise in the markets. <em>NBT</em>,
<em>40</em>(11), 1542–1545. (<a
href="https://doi.org/10.1038/s41587-022-01541-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01541-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1542-1545},
  shortjournal = {Nature Biotech.},
  title        = {3Q22 — more malaise in the markets},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Drug pipeline 3Q22 — rare disease and alzheimer’s
treatments. <em>NBT</em>, <em>40</em>(11), 1539–1541. (<a
href="https://doi.org/10.1038/s41587-022-01545-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01545-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1539-1541},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 3Q22 — rare disease and alzheimer’s treatments},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(11), 1538. (<a
href="https://doi.org/10.1038/s41587-022-01549-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01549-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1538},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Elite greek warrior myth debunked. <em>NBT</em>,
<em>40</em>(11), 1537. (<a
href="https://doi.org/10.1038/s41587-022-01579-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-022-01579-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1537},
  shortjournal = {Nature Biotech.},
  title        = {Elite greek warrior myth debunked},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). UK biobank’s reimaging colossus. <em>NBT</em>,
<em>40</em>(11), 1536. (<a
href="https://doi.org/10.1038/s41587-022-01575-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01575-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1536},
  shortjournal = {Nature Biotech.},
  title        = {UK biobank’s reimaging colossus},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Nobel laureate bertozzi’s companies. <em>NBT</em>,
<em>40</em>(11), 1535. (<a
href="https://doi.org/10.1038/s41587-022-01573-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01573-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1535},
  shortjournal = {Nature Biotech.},
  title        = {Nobel laureate bertozzi’s companies},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Cow-less milk: The rising tide of animal-free dairy
attracts big players. <em>NBT</em>, <em>40</em>(11), 1534–1536. (<a
href="https://doi.org/10.1038/s41587-022-01548-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dairy alternative producers are not just squeezing plants to make proteins, they are using fermentation to replicate components of conventional milk.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-022-01548-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1534-1536},
  shortjournal = {Nature Biotech.},
  title        = {Cow-less milk: The rising tide of animal-free dairy attracts big players},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). First allo t cell gets european nod. <em>NBT</em>,
<em>40</em>(11), 1534. (<a
href="https://doi.org/10.1038/s41587-022-01572-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01572-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1534},
  shortjournal = {Nature Biotech.},
  title        = {First allo t cell gets european nod},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). For hemophilia and thalassemia, a new era of “one-and-done”
gene therapies has arrived. <em>NBT</em>, <em>40</em>(11), 1531–1533.
(<a href="https://doi.org/10.1038/s41587-022-01555-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Industry celebrates two newly approved genetic medicines, but patients with these blood disorders may face a long wait, and many will never access them because of their high cost.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-022-01555-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1531-1533},
  shortjournal = {Nature Biotech.},
  title        = {For hemophilia and thalassemia, a new era of ‘one-and-done’ gene therapies has arrived},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). People. <em>NBT</em>, <em>40</em>(10), 1530. (<a
href="https://doi.org/10.1038/s41587-022-01500-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01500-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1530},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrating medicine, engineering and business to educate
early-stage researchers in cardiovascular device development.
<em>NBT</em>, <em>40</em>(10), 1528–1529. (<a
href="https://doi.org/10.1038/s41587-022-01498-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Developing a device development educational core will further enhance national, international and multidisciplinary innovation and collaboration in the biomedical device domain.},
  archive      = {J_NBT},
  author       = {Vizgan, Gabriel and Hill-Whilton, Zachary and Gillespie, Colleen and Cobos, Daniel and Johnson, LeAnn and Dib, Nabil and Gold-von Simson, Gabrielle},
  doi          = {10.1038/s41587-022-01498-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1528-1529},
  shortjournal = {Nature Biotech.},
  title        = {Integrating medicine, engineering and business to educate early-stage researchers in cardiovascular device development},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prediction of protein–ligand binding affinity from
sequencing data with interpretable machine learning. <em>NBT</em>,
<em>40</em>(10), 1520–1527. (<a
href="https://doi.org/10.1038/s41587-022-01307-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein–ligand interactions are increasingly profiled at high throughput using affinity selection and massively parallel sequencing. However, these assays do not provide the biophysical parameters that most rigorously quantify molecular interactions. Here we describe a flexible machine learning method, called ProBound, that accurately defines sequence recognition in terms of equilibrium binding constants or kinetic rates. This is achieved using a multi-layered maximum-likelihood framework that models both the molecular interactions and the data generation process. We show that ProBound quantifies transcription factor (TF) behavior with models that predict binding affinity over a range exceeding that of previous resources; captures the impact of DNA modifications and conformational flexibility of multi-TF complexes; and infers specificity directly from in vivo data such as ChIP-seq without peak calling. When coupled with an assay called KD-seq, it determines the absolute affinity of protein–ligand interactions. We also apply ProBound to profile the kinetics of kinase–substrate interactions. ProBound opens new avenues for decoding biological networks and rationally engineering protein–ligand interactions. Protein–ligand binding affinity is predicted quantitatively from sequencing data.},
  archive      = {J_NBT},
  author       = {Rube, H. Tomas and Rastogi, Chaitanya and Feng, Siqian and Kribelbauer, Judith F. and Li, Allyson and Becerra, Basheer and Melo, Lucas A. N. and Do, Bach Viet and Li, Xiaoting and Adam, Hammaad H. and Shah, Neel H. and Mann, Richard S. and Bussemaker, Harmen J.},
  doi          = {10.1038/s41587-022-01307-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1520-1527},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of protein–ligand binding affinity from sequencing data with interpretable machine learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Logic-gated antibody pairs that selectively act on cells
co-expressing two antigens. <em>NBT</em>, <em>40</em>(10), 1509–1519.
(<a href="https://doi.org/10.1038/s41587-022-01384-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of therapeutic monoclonal antibodies is constrained because single antigen targets often do not provide sufficient selectivity to distinguish diseased from healthy tissues. We present HexElect®, an approach to enhance the functional selectivity of therapeutic antibodies by making their activity dependent on clustering after binding to two different antigens expressed on the same target cell. lmmunoglobulin G (lgG)-mediated clustering of membrane receptors naturally occurs on cell surfaces to trigger complement- or cell-mediated effector functions or to initiate intracellular signaling. We engineer the Fc domains of two different lgG antibodies to suppress their individual homo-oligomerization while promoting their pairwise hetero-oligomerization after binding co-expressed antigens. We show that recruitment of complement component C1q to these hetero-oligomers leads to clustering-dependent activation of effector functions such as complement mediated killing of target cells or activation of cell surface receptors. HexElect allows selective antibody activity on target cells expressing unique, potentially unexplored combinations of surface antigens. Antibody pairs are made to act selectively on cells co-expressing two targets by engineering their Fc domains.},
  archive      = {J_NBT},
  author       = {Oostindie, Simone C. and Rinaldi, Derek A. and Zom, Gijs G. and Wester, Michael J. and Paulet, Desiree and Al-Tamimi, Kusai and van der Meijden, Els and Scheick, Jennifer R. and Wilpshaar, Tessa and de Jong, Bart and Hoff-van den Broek, Marloes and Grattan, Rachel M. and Oosterhoff, Janita J. and Vignau, Julie and Verploegen, Sandra and Boross, Peter and Beurskens, Frank J. and Lidke, Diane S. and Schuurman, Janine and de Jong, Rob N.},
  doi          = {10.1038/s41587-022-01384-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1509-1519},
  shortjournal = {Nature Biotech.},
  title        = {Logic-gated antibody pairs that selectively act on cells co-expressing two antigens},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Expanding RNAi therapeutics to extrahepatic tissues with
lipophilic conjugates. <em>NBT</em>, <em>40</em>(10), 1500–1508. (<a
href="https://doi.org/10.1038/s41587-022-01334-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Therapeutics based on short interfering RNAs (siRNAs) delivered to hepatocytes have been approved, but new delivery solutions are needed to target additional organs. Here we show that conjugation of 2′-O-hexadecyl (C16) to siRNAs enables safe, potent and durable silencing in the central nervous system (CNS), eye and lung in rodents and non-human primates with broad cell type specificity. We show that intrathecally or intracerebroventricularly delivered C16-siRNAs were active across CNS regions and cell types, with sustained RNA interference (RNAi) activity for at least 3 months. Similarly, intravitreal administration to the eye or intranasal administration to the lung resulted in a potent and durable knockdown. The preclinical efficacy of an siRNA targeting the amyloid precursor protein was evaluated through intracerebroventricular dosing in a mouse model of Alzheimer’s disease, resulting in amelioration of physiological and behavioral deficits. Altogether, C16 conjugation of siRNAs has the potential for safe therapeutic silencing of target genes outside the liver with infrequent dosing. Lipophilic siRNA conjugates exert therapeutic activity in the mouse CNS.},
  archive      = {J_NBT},
  author       = {Brown, Kirk M. and Nair, Jayaprakash K. and Janas, Maja M. and Anglero-Rodriguez, Yesseinia I. and Dang, Lan T. H. and Peng, Haiyan and Theile, Christopher S. and Castellanos-Rizaldos, Elena and Brown, Christopher and Foster, Donald and Kurz, Jeffrey and Allen, Jeffrey and Maganti, Rajanikanth and Li, Jing and Matsuda, Shigeo and Stricos, Matthew and Chickering, Tyler and Jung, Michelle and Wassarman, Kelly and Rollins, Jeff and Woods, Lauren and Kelin, Alex and Guenther, Dale C. and Mobley, Melissa W. and Petrulis, John and McDougall, Robin and Racie, Timothy and Bombardier, Jessica and Cha, Diana and Agarwal, Saket and Johnson, Lei and Jiang, Yongfeng and Lentini, Scott and Gilbert, Jason and Nguyen, Tuyen and Chigas, Samantha and LeBlanc, Sarah and Poreci, Urjana and Kasper, Anne and Rogers, Arlin B. and Chong, Saeho and Davis, Wendell and Sutherland, Jessica E. and Castoreno, Adam and Milstein, Stuart and Schlegel, Mark K. and Zlatev, Ivan and Charisse, Klaus and Keating, Mark and Manoharan, Muthiah and Fitzgerald, Kevin and Wu, Jing-Tao and Maier, Martin A. and Jadhav, Vasant},
  doi          = {10.1038/s41587-022-01334-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1500-1508},
  shortjournal = {Nature Biotech.},
  title        = {Expanding RNAi therapeutics to extrahepatic tissues with lipophilic conjugates},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identifying synergistic high-order 3D chromatin
conformations from genome-scale nanopore concatemer sequencing.
<em>NBT</em>, <em>40</em>(10), 1488–1499. (<a
href="https://doi.org/10.1038/s41587-022-01289-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-order three-dimensional (3D) interactions between more than two genomic loci are common in human chromatin, but their role in gene regulation is unclear. Previous high-order 3D chromatin assays either measure distant interactions across the genome or proximal interactions at selected targets. To address this gap, we developed Pore-C, which combines chromatin conformation capture with nanopore sequencing of concatemers to profile proximal high-order chromatin contacts at the genome scale. We also developed the statistical method Chromunity to identify sets of genomic loci with frequencies of high-order contacts significantly higher than background (‘synergies’). Applying these methods to human cell lines, we found that synergies were enriched in enhancers and promoters in active chromatin and in highly transcribed and lineage-defining genes. In prostate cancer cells, these included binding sites of androgen-driven transcription factors and the promoters of androgen-regulated genes. Concatemers of high-order contacts in highly expressed genes were demethylated relative to pairwise contacts at the same loci. Synergies in breast cancer cells were associated with tyfonas, a class of complex DNA amplicons. These results rigorously link genome-wide high-order 3D interactions to lineage-defining transcriptional programs and establish Pore-C and Chromunity as scalable approaches to assess high-order genome structure. High-order chromatin contacts are identified using a combination of 3C, nanopore sequencing and robust statistical analysis.},
  archive      = {J_NBT},
  author       = {Deshpande, Aditya S. and Ulahannan, Netha and Pendleton, Matthew and Dai, Xiaoguang and Ly, Lynn and Behr, Julie M. and Schwenk, Stefan and Liao, Will and Augello, Michael A. and Tyer, Carly and Rughani, Priyesh and Kudman, Sarah and Tian, Huasong and Otis, Hannah G. and Adney, Emily and Wilkes, David and Mosquera, Juan Miguel and Barbieri, Christopher E. and Melnick, Ari and Stoddart, David and Turner, Daniel J. and Juul, Sissel and Harrington, Eoghan and Imieliński, Marcin},
  doi          = {10.1038/s41587-022-01289-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1488-1499},
  shortjournal = {Nature Biotech.},
  title        = {Identifying synergistic high-order 3D chromatin conformations from genome-scale nanopore concatemer sequencing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Comparison and imputation-aided integration of five
commercial platforms for targeted DNA methylome analysis. <em>NBT</em>,
<em>40</em>(10), 1478–1487. (<a
href="https://doi.org/10.1038/s41587-022-01336-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted bisulfite sequencing (TBS) has become the method of choice for the cost-effective, targeted analysis of the human methylome at base-pair resolution. In this study, we benchmarked five commercially available TBS platforms—three hybridization capture-based (Agilent, Roche and Illumina) and two reduced-representation-based (Diagenode and NuGen)—across 11 samples. Two samples were also compared with whole-genome DNA methylation sequencing with the Illumina and Oxford Nanopore platforms. We assessed workflow complexity, on/off-target performance, coverage, accuracy and reproducibility. Although all platforms produced robust and reproducible data, major differences in the number and identity of the CpG sites covered make it difficult to compare datasets generated on different platforms. To overcome this limitation, we applied imputation and show that it improves interoperability from an average of 10.35\% (0.8 million) to 97\% (7.6 million) common CpG sites. Our study provides guidance on which TBS platform to use for different methylome features and offers an imputation-based harmonization solution that allows comparative, integrative analysis. Differences in CpG coverage of targeted bisulfite sequencing methods can be overcome using imputation.},
  archive      = {J_NBT},
  author       = {Tanić, Miljana and Moghul, Ismail and Rodney, Simon and Dhami, Pawan and Vaikkinen, Heli and Ambrose, John and Barrett, James and Feber, Andrew and Beck, Stephan},
  doi          = {10.1038/s41587-022-01336-9},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1478-1487},
  shortjournal = {Nature Biotech.},
  title        = {Comparison and imputation-aided integration of five commercial platforms for targeted DNA methylome analysis},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DIALOGUE maps multicellular programs in tissue from
single-cell or spatial transcriptomics data. <em>NBT</em>,
<em>40</em>(10), 1467–1477. (<a
href="https://doi.org/10.1038/s41587-022-01288-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering the functional interactions of cells in tissues remains a major challenge. Here we describe DIALOGUE, a method to systematically uncover multicellular programs (MCPs)—combinations of coordinated cellular programs in different cell types that form higher-order functional units at the tissue level—from either spatial data or single-cell data obtained without spatial information. Tested on spatial datasets from the mouse hypothalamus, cerebellum, visual cortex and neocortex, DIALOGUE identified MCPs associated with animal behavior and recovered spatial properties when tested on unseen data while outperforming other methods and metrics. In spatial data from human lung cancer, DIALOGUE identified MCPs marking immune activation and tissue remodeling. Applied to single-cell RNA sequencing data across individuals or regions, DIALOGUE uncovered MCPs marking Alzheimer’s disease, ulcerative colitis and resistance to cancer immunotherapy. These programs were predictive of disease outcome and predisposition in independent cohorts and included risk genes from genome-wide association studies. DIALOGUE enables the analysis of multicellular regulation in health and disease. Coordinated gene programs spanning multiple different cell types are identified in healthy and diseased tissues.},
  archive      = {J_NBT},
  author       = {Jerby-Arnon, Livnat and Regev, Aviv},
  doi          = {10.1038/s41587-022-01288-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1467-1477},
  shortjournal = {Nature Biotech.},
  title        = {DIALOGUE maps multicellular programs in tissue from single-cell or spatial transcriptomics data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-omics single-cell data integration and regulatory
inference with graph-linked embedding. <em>NBT</em>, <em>40</em>(10),
1458–1466. (<a
href="https://doi.org/10.1038/s41587-022-01284-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the emergence of experimental methods for simultaneous measurement of multiple omics modalities in single cells, most single-cell datasets include only one modality. A major obstacle in integrating omics data from multiple modalities is that different omics layers typically have distinct feature spaces. Here, we propose a computational framework called GLUE (graph-linked unified embedding), which bridges the gap by modeling regulatory interactions across omics layers explicitly. Systematic benchmarking demonstrated that GLUE is more accurate, robust and scalable than state-of-the-art tools for heterogeneous single-cell multi-omics data. We applied GLUE to various challenging tasks, including triple-omics integration, integrative regulatory inference and multi-omics human cell atlas construction over millions of cells, where GLUE was able to correct previous annotations. GLUE features a modular design that can be flexibly extended and enhanced for new analysis tasks. The full package is available online at https://github.com/gao-lab/GLUE . Different single-cell data modalities are integrated at atlas-scale by modeling regulatory interactions.},
  archive      = {J_NBT},
  author       = {Cao, Zhi-Jie and Gao, Ge},
  doi          = {10.1038/s41587-022-01284-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1458-1466},
  shortjournal = {Nature Biotech.},
  title        = {Multi-omics single-cell data integration and regulatory inference with graph-linked embedding},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Scalable single-cell RNA sequencing from full transcripts
with smart-seq3xpress. <em>NBT</em>, <em>40</em>(10), 1452–1457. (<a
href="https://doi.org/10.1038/s41587-022-01311-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current single-cell RNA sequencing (scRNA-seq) methods with high cellular throughputs sacrifice full-transcript coverage and often sensitivity. Here we describe Smart-seq3xpress, which miniaturizes and streamlines the Smart-seq3 protocol to substantially reduce reagent use and increase cellular throughput. Smart-seq3xpress analysis of peripheral blood mononuclear cells resulted in a granular atlas complete with common and rare cell types. Compared with droplet-based single-cell RNA sequencing that sequences RNA ends, the additional full-transcript coverage revealed cell-type-associated isoform variation. Smart-seq3xpress enables single-cell RNA sequencing from full transcripts at high throughput.},
  archive      = {J_NBT},
  author       = {Hagemann-Jensen, Michael and Ziegenhain, Christoph and Sandberg, Rickard},
  doi          = {10.1038/s41587-022-01311-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1452-1457},
  shortjournal = {Nature Biotech.},
  title        = {Scalable single-cell RNA sequencing from full transcripts with smart-seq3xpress},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fast and highly sensitive full-length single-cell RNA
sequencing using FLASH-seq. <em>NBT</em>, <em>40</em>(10), 1447–1451.
(<a href="https://doi.org/10.1038/s41587-022-01312-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present FLASH-seq (FS), a full-length single-cell RNA sequencing (scRNA-seq) method with increased sensitivity and reduced hands-on time compared to Smart-seq3. The entire FS protocol can be performed in ~4.5 hours, is simple to automate and can be easily miniaturized to decrease resource consumption. The FS protocol can also use unique molecular identifiers (UMIs) for molecule counting while displaying reduced strand-invasion artifacts. FS will be especially useful for characterizing gene expression at high resolution across multiple samples. FLASH-seq speeds up high-sensitivity scRNA-seq while sparing resources.},
  archive      = {J_NBT},
  author       = {Hahaut, Vincent and Pavlinic, Dinko and Carbone, Walter and Schuierer, Sven and Balmer, Pierre and Quinodoz, Mathieu and Renner, Magdalena and Roma, Guglielmo and Cowan, Cameron S. and Picelli, Simone},
  doi          = {10.1038/s41587-022-01312-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1447-1451},
  shortjournal = {Nature Biotech.},
  title        = {Fast and highly sensitive full-length single-cell RNA sequencing using FLASH-seq},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reply to: Assessing the efficiency of verily’s automated
process for production and release of male wolbachia-infected
mosquitoes. <em>NBT</em>, <em>40</em>(10), 1443–1446. (<a
href="https://doi.org/10.1038/s41587-022-01325-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Crawford, Jacob E. and Hopkins, Kaycie C. and Buchman, Anna and Zha, Tiantian and Howell, Paul and Kakani, Evdoxia and Ohm, Johanna R. and Snoad, Nigel and Upson, Linus and Holeman, Jodi and Massaro, Peter and Dobson, Stephen L. and Mulligan, F. Stephen and White, Bradley J.},
  doi          = {10.1038/s41587-022-01325-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1443-1446},
  shortjournal = {Nature Biotech.},
  title        = {Reply to: Assessing the efficiency of verily’s automated process for production and release of male wolbachia-infected mosquitoes},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Assessing the efficiency of verily’s automated process for
production and release of male wolbachia-infected mosquitoes.
<em>NBT</em>, <em>40</em>(10), 1441–1442. (<a
href="https://doi.org/10.1038/s41587-022-01324-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bouyer, Jérémy and Maiga, Hamidou and Vreysen, Marc J. B.},
  doi          = {10.1038/s41587-022-01324-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1441-1442},
  shortjournal = {Nature Biotech.},
  title        = {Assessing the efficiency of verily’s automated process for production and release of male wolbachia-infected mosquitoes},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Lipophilic siRNA conjugates yield durable silencing in
extrahepatic tissues. <em>NBT</em>, <em>40</em>(10), 1439–1440. (<a
href="https://doi.org/10.1038/s41587-022-01335-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA interference (RNAi) therapeutics can silence disease-causing gene transcripts, but extrahepatic delivery has been challenging. Conjugating short interfering RNAs (siRNAs) to a lipophilic alkyl chain enabled safe delivery and long-term mRNA silencing in the brain, eye and lung in animal models, thereby opening new applications for RNAi therapeutics.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01335-w},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1439-1440},
  shortjournal = {Nature Biotech.},
  title        = {Lipophilic siRNA conjugates yield durable silencing in extrahepatic tissues},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Team architecture in 3D genomic interactions revealed
through nanopore sequencing. <em>NBT</em>, <em>40</em>(10), 1437–1438.
(<a href="https://doi.org/10.1038/s41587-022-01290-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Are major decisions in the cell made through pairs of interacting loci (enhancers and promoters) or larger teams of cooperating regulatory elements? A new genome-wide assay and algorithm answers this question and provides a scalable technology to link new dimensions of genome structure with cellular function.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01290-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1437-1438},
  shortjournal = {Nature Biotech.},
  title        = {Team architecture in 3D genomic interactions revealed through nanopore sequencing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Forum: Teichmann and regev. <em>NBT</em>, <em>40</em>(10),
1436. (<a href="https://doi.org/10.1038/s41587-022-01490-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01490-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1436},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Teichmann and regev},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Precision cancer targeting with antibody pairs.
<em>NBT</em>, <em>40</em>(10), 1434–1435. (<a
href="https://doi.org/10.1038/s41587-022-01401-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Logic-gated monoclonal antibodies have been designed to improve safety and efficacy.},
  archive      = {J_NBT},
  author       = {Lawrence, Robert T. and Senter, Peter D.},
  doi          = {10.1038/s41587-022-01401-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1434-1435},
  shortjournal = {Nature Biotech.},
  title        = {Precision cancer targeting with antibody pairs},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). De novo protein design with a language model. <em>NBT</em>,
<em>40</em>(10), 1433. (<a
href="https://doi.org/10.1038/s41587-022-01518-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01518-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1433},
  shortjournal = {Nature Biotech.},
  title        = {De novo protein design with a language model},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Visualizing protein nanostructures in intact brain.
<em>NBT</em>, <em>40</em>(10), 1432. (<a
href="https://doi.org/10.1038/s41587-022-01511-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Barnes, Natalie G.},
  doi          = {10.1038/s41587-022-01511-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1432},
  shortjournal = {Nature Biotech.},
  title        = {Visualizing protein nanostructures in intact brain},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Single-cell RNA-seq keeps cells alive. <em>NBT</em>,
<em>40</em>(10), 1432. (<a
href="https://doi.org/10.1038/s41587-022-01515-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01515-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1432},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell RNA-seq keeps cells alive},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Optogenetics. <em>NBT</em>, <em>40</em>(10), 1431. (<a
href="https://doi.org/10.1038/s41587-022-01501-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods and devices for optogenetic control of cells.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01501-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1431},
  shortjournal = {Nature Biotech.},
  title        = {Optogenetics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). What the COVID-19 pandemic revealed about intellectual
property. <em>NBT</em>, <em>40</em>(10), 1428–1430. (<a
href="https://doi.org/10.1038/s41587-022-01485-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COVID-19 pandemic dispelled some myths underlying intellectual property policy and revealed how stakeholders can develop policies to accelerate development and ensure access using existing tools and experimenting with open science.},
  archive      = {J_NBT},
  author       = {Gold, E. Richard},
  doi          = {10.1038/s41587-022-01485-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1428-1430},
  shortjournal = {Nature Biotech.},
  title        = {What the COVID-19 pandemic revealed about intellectual property},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Public biotech in 2021 — the numbers. <em>NBT</em>,
<em>40</em>(10), 1427. (<a
href="https://doi.org/10.1038/s41587-022-01496-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura and Lähteenmäki, Riku},
  doi          = {10.1038/s41587-022-01496-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Public biotech in 2021 — the numbers},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(10), 1426. (<a
href="https://doi.org/10.1038/s41587-022-01503-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01503-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1426},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Baltic bacteria sink CO2 into smoothies. <em>NBT</em>,
<em>40</em>(10), 1425. (<a
href="https://doi.org/10.1038/s41587-022-01507-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Balakrishnan, Vijay Shankar},
  doi          = {10.1038/s41587-022-01507-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Baltic bacteria sink CO2 into smoothies},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). AI predicts cancer relapse from slides. <em>NBT</em>,
<em>40</em>(10), 1423. (<a
href="https://doi.org/10.1038/s41587-022-01514-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01514-9},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1423},
  shortjournal = {Nature Biotech.},
  title        = {AI predicts cancer relapse from slides},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Precision nutrition to boost cancer treatments.
<em>NBT</em>, <em>40</em>(10), 1422–1424. (<a
href="https://doi.org/10.1038/s41587-022-01502-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers are designing diets that may improve patient responses to cancer therapy by using machine learning and genotyping to uncover tumors’ nutritional vulnerabilities.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-022-01502-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1422-1424},
  shortjournal = {Nature Biotech.},
  title        = {Precision nutrition to boost cancer treatments},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cancer moonshot aims at liquid biopsies. <em>NBT</em>,
<em>40</em>(10), 1422. (<a
href="https://doi.org/10.1038/s41587-022-01513-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01513-w},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1422},
  shortjournal = {Nature Biotech.},
  title        = {Cancer moonshot aims at liquid biopsies},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CAR-ts made entirely within the body. <em>NBT</em>,
<em>40</em>(10), 1421. (<a
href="https://doi.org/10.1038/s41587-022-01512-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01512-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1421},
  shortjournal = {Nature Biotech.},
  title        = {CAR-ts made entirely within the body},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Repairing the eye. <em>NBT</em>, <em>40</em>(10),
1419–1421. (<a
href="https://doi.org/10.1038/s41587-022-01506-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are pursuing interventions to edit, regenerate and reprogram eye tissue to restore lost vision.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-022-01506-9},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1419-1421},
  shortjournal = {Nature Biotech.},
  title        = {Repairing the eye},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). People. <em>NBT</em>, <em>40</em>(9), 1418. (<a
href="https://doi.org/10.1038/s41587-022-01461-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01461-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1418},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). How researchers can join the race to develop new ways of
making meat. <em>NBT</em>, <em>40</em>(9), 1414–1417. (<a
href="https://doi.org/10.1038/s41587-022-01454-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The sustainable protein sector, which promises to create a better food system for the planet, is part of a growing industry with a lot to offer to young researchers.},
  archive      = {J_NBT},
  author       = {Kell, Seren L.},
  doi          = {10.1038/s41587-022-01454-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1414-1417},
  shortjournal = {Nature Biotech.},
  title        = {How researchers can join the race to develop new ways of making meat},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Author correction: An engineered prime editor with enhanced
editing efficiency in plants. <em>NBT</em>, <em>40</em>(9), 1412. (<a
href="https://doi.org/10.1038/s41587-022-01308-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zong, Yuan and Liu, Yijing and Xue, Chenxiao and Li, Boshu and Li, Xiangyang and Wang, Yanpeng and Li, Ji and Liu, Guanwen and Huang, Xingxu and Cao, Xiaofeng and Gao, Caixia},
  doi          = {10.1038/s41587-022-01308-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1412},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: An engineered prime editor with enhanced editing efficiency in plants},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Author correction: A highly photostable and bright green
fluorescent protein. <em>NBT</em>, <em>40</em>(9), 1412. (<a
href="https://doi.org/10.1038/s41587-022-01469-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hirano, Masahiko and Ando, Ryoko and Shimozono, Satoshi and Sugiyama, Mayu and Takeda, Noriyo and Kurokawa, Hiroshi and Deguchi, Ryusaku and Endo, Kazuki and Haga, Kei and Takai-Todaka, Reiko and Inaura, Shunsuke and Matsumura, Yuta and Hama, Hiroshi and Okada, Yasushi and Fujiwara, Takahiro and Morimoto, Takuya and Katayama, Kazuhiko and Miyawaki, Atsushi},
  doi          = {10.1038/s41587-022-01469-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1412},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: A highly photostable and bright green fluorescent protein},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Targeting a gene regulatory element enhances rice grain
yield by decoupling panicle number and size. <em>NBT</em>,
<em>40</em>(9), 1403–1411. (<a
href="https://doi.org/10.1038/s41587-022-01281-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crop genetic improvement requires balancing complex tradeoffs caused by gene pleiotropy and linkage drags, as exemplified by IPA1 (Ideal Plant Architecture 1), a typical pleiotropic gene in rice that increases grains per panicle but reduces tillers. In this study, we identified a 54-base pair cis-regulatory region in IPA1 via a tiling-deletion-based CRISPR–Cas9 screen that, when deleted, resolves the tradeoff between grains per panicle and tiller number, leading to substantially enhanced grain yield per plant. Mechanistic studies revealed that the deleted fragment is a target site for the transcription factor An-1 to repress IPA1 expression in panicles and roots. Targeting gene regulatory regions should help dissect tradeoff effects and provide a rich source of targets for breeding complementary beneficial traits. Rice yield is boosted by overcoming a trait tradeoff.},
  archive      = {J_NBT},
  author       = {Song, Xiaoguang and Meng, Xiangbing and Guo, Hongyan and Cheng, Qiao and Jing, Yanhui and Chen, Mingjiang and Liu, Guifu and Wang, Bing and Wang, Yonghong and Li, Jiayang and Yu, Hong},
  doi          = {10.1038/s41587-022-01281-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1403-1411},
  shortjournal = {Nature Biotech.},
  title        = {Targeting a gene regulatory element enhances rice grain yield by decoupling panicle number and size},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). An engineered prime editor with enhanced editing efficiency
in plants. <em>NBT</em>, <em>40</em>(9), 1394–1402. (<a
href="https://doi.org/10.1038/s41587-022-01254-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing is a versatile genome-editing technology, but it suffers from low editing efficiency. In the present study, we introduce optimized prime editors with substantially improved editing efficiency. We engineered the Moloney–murine leukemia virus reverse transcriptase by removing its ribonuclease H domain and incorporated a viral nucleocapsid protein with nucleic acid chaperone activity. Each modification independently improved prime editing efficiency by ~1.8–3.4-fold in plant cells. When combined in our engineered plant prime editor (ePPE), the two modifications synergistically enhanced the efficiency of base substitutions, deletions and insertions at various endogenous sites by on average 5.8-fold compared with the original PPE in cell culture. No significant increase in byproducts or off-target editing was observed. We used the ePPE to generate rice plants tolerant to sulfonylurea and imidazolinone herbicides, observing an editing frequency of 11.3\% compared with 2.1\% using PPE. We also combined ePPE with the previously reported dual-prime editing guide (peg) RNAs and engineered pegRNAs to further increase efficiency. Prime editing in plants is improved fivefold through molecular engineering.},
  archive      = {J_NBT},
  author       = {Zong, Yuan and Liu, Yijing and Xue, Chenxiao and Li, Boshu and Li, Xiangyang and Wang, Yanpeng and Li, Ji and Liu, Guanwen and Huang, Xingxu and Cao, Xiaofeng and Gao, Caixia},
  doi          = {10.1038/s41587-022-01254-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1394-1402},
  shortjournal = {Nature Biotech.},
  title        = {An engineered prime editor with enhanced editing efficiency in plants},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A split prime editor with untethered reverse transcriptase
and circular RNA template. <em>NBT</em>, <em>40</em>(9), 1388–1393. (<a
href="https://doi.org/10.1038/s41587-022-01255-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Delivery and optimization of prime editors (PEs) have been hampered by their large size and complexity. Although split versions of genome-editing tools can reduce construct size, they require special engineering to tether the binding and catalytic domains. Here we report a split PE (sPE) in which the Cas9 nickase (nCas9) remains untethered from the reverse transcriptase (RT). The sPE showed similar efficiencies in installing precise edits as the parental unsplit PE3 and no increase in insertion–deletion (indel) byproducts. Delivery of sPE to the mouse liver with hydrodynamic injection to modify β-catenin drove tumor formation with similar efficiency as PE3. Delivery with two adeno-associated virus (AAV) vectors corrected the disease-causing mutation in a mouse model of type I tyrosinemia. Similarly, prime editing guide RNAs (pegRNAs) can be split into a single guide RNA (sgRNA) and a circular RNA RT template to increase flexibility and stability. Compared to previous sPEs, ours lacks inteins, protein–protein affinity modules and nuclease-sensitive pegRNA extensions, which increase construct complexity and might reduce efficiency. Our modular system will facilitate the delivery and optimization of PEs. A split prime editor simplifies delivery.},
  archive      = {J_NBT},
  author       = {Liu, Bin and Dong, Xiaolong and Cheng, Haoyang and Zheng, Chunwei and Chen, Zexiang and Rodríguez, Tomás C. and Liang, Shun-Qing and Xue, Wen and Sontheimer, Erik J.},
  doi          = {10.1038/s41587-022-01255-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1388-1393},
  shortjournal = {Nature Biotech.},
  title        = {A split prime editor with untethered reverse transcriptase and circular RNA template},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CRISPR-free base editors with enhanced activity and expanded
targeting scope in mitochondrial and nuclear DNA. <em>NBT</em>,
<em>40</em>(9), 1378–1387. (<a
href="https://doi.org/10.1038/s41587-022-01256-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The all-protein cytosine base editor DdCBE uses TALE proteins and a double-stranded DNA-specific cytidine deaminase (DddA) to mediate targeted C•G-to-T•A editing. To improve editing efficiency and overcome the strict TC sequence-context constraint of DddA, we used phage-assisted non-continuous and continuous evolution to evolve DddA variants with improved activity and expanded targeting scope. Compared to canonical DdCBEs, base editors with evolved DddA6 improved mitochondrial DNA (mtDNA) editing efficiencies at TC by 3.3-fold on average. DdCBEs containing evolved DddA11 offered a broadened HC (H = A, C or T) sequence compatibility for both mitochondrial and nuclear base editing, increasing average editing efficiencies at AC and CC targets from less than 10\% for canonical DdCBE to 15–30\% and up to 50\% in cell populations sorted to express both halves of DdCBE. We used these evolved DdCBEs to efficiently install disease-associated mtDNA mutations in human cells at non-TC target sites. DddA6 and DddA11 substantially increase the effectiveness and applicability of all-protein base editing. Continuously evolved double-stranded DNA deaminases increase the efficiency and targeting scope of all-protein base editors.},
  archive      = {J_NBT},
  author       = {Mok, Beverly Y. and Kotrys, Anna V. and Raguram, Aditya and Huang, Tony P. and Mootha, Vamsi K. and Liu, David R.},
  doi          = {10.1038/s41587-022-01256-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1378-1387},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR-free base editors with enhanced activity and expanded targeting scope in mitochondrial and nuclear DNA},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Generation of a live attenuated influenza a vaccine by
proteolysis targeting. <em>NBT</em>, <em>40</em>(9), 1370–1377. (<a
href="https://doi.org/10.1038/s41587-022-01381-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The usefulness of live attenuated virus vaccines has been limited by suboptimal immunogenicity, safety concerns or cumbersome manufacturing processes and techniques. Here we describe the generation of a live attenuated influenza A virus vaccine using proteolysis-targeting chimeric (PROTAC) technology to degrade viral proteins via the endogenous ubiquitin–proteasome system of host cells. We engineered the genome of influenza A viruses in stable cell lines engineered for virus production to introduce a conditionally removable proteasome-targeting domain, generating fully infective PROTAC viruses that were live attenuated by the host protein degradation machinery upon infection. In mouse and ferret models, PROTAC viruses were highly attenuated and able to elicit robust and broad humoral, mucosal and cellular immunity against homologous and heterologous virus challenges. PROTAC-mediated attenuation of viruses may be broadly applicable for generating live attenuated vaccines. Influenza virus is attenuated for vaccine production using PROTAC degradation technology.},
  archive      = {J_NBT},
  author       = {Si, Longlong and Shen, Quan and Li, Jing and Chen, Li and Shen, Jinying and Xiao, Xue and Bai, Haiqing and Feng, Tang and Ye, Adam Yongxin and Li, Le and Zhang, Chunhe and Li, Zhen and Wang, Ping and Oh, Crystal Yuri and Nurani, Atiq and Niu, Siwen and Zhang, Chengxin and Wei, Xiaoqiong and Yuan, Wanqiong and Liao, Hao and Huang, Xiaojie and Wang, Ning and Tian, Wen-xia and Tian, Hongwei and Li, Li and Liu, Xiaoheng and Plebani, Roberto},
  doi          = {10.1038/s41587-022-01381-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1370-1377},
  shortjournal = {Nature Biotech.},
  title        = {Generation of a live attenuated influenza a vaccine by proteolysis targeting},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DestVI identifies continuums of cell types in spatial
transcriptomics data. <em>NBT</em>, <em>40</em>(9), 1360–1369. (<a
href="https://doi.org/10.1038/s41587-022-01272-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most spatial transcriptomics technologies are limited by their resolution, with spot sizes larger than that of a single cell. Although joint analysis with single-cell RNA sequencing can alleviate this problem, current methods are limited to assessing discrete cell types, revealing the proportion of cell types inside each spot. To identify continuous variation of the transcriptome within cells of the same type, we developed Deconvolution of Spatial Transcriptomics profiles using Variational Inference (DestVI). Using simulations, we demonstrate that DestVI outperforms existing methods for estimating gene expression for every cell type inside every spot. Applied to a study of infected lymph nodes and of a mouse tumor model, DestVI provides high-resolution, accurate spatial characterization of the cellular organization of these tissues and identifies cell-type-specific changes in gene expression between different tissue regions or between conditions. DestVI is available as part of the open-source software package scvi-tools ( https://scvi-tools.org ). DestVI models continuous cell states in spatial transcriptomics data.},
  archive      = {J_NBT},
  author       = {Lopez, Romain and Li, Baoguo and Keren-Shaul, Hadas and Boyeau, Pierre and Kedmi, Merav and Pilzer, David and Jelinski, Adam and Yofe, Ido and David, Eyal and Wagner, Allon and Ergen, Can and Addadi, Yoseph and Golani, Ofra and Ronchese, Franca and Jordan, Michael I. and Amit, Ido and Yosef, Nir},
  doi          = {10.1038/s41587-022-01272-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1360-1369},
  shortjournal = {Nature Biotech.},
  title        = {DestVI identifies continuums of cell types in spatial transcriptomics data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatially informed cell-type deconvolution for spatial
transcriptomics. <em>NBT</em>, <em>40</em>(9), 1349–1359. (<a
href="https://doi.org/10.1038/s41587-022-01273-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many spatially resolved transcriptomic technologies do not have single-cell resolution but measure the average gene expression for each spot from a mixture of cells of potentially heterogeneous cell types. Here, we introduce a deconvolution method, conditional autoregressive-based deconvolution (CARD), that combines cell-type-specific expression information from single-cell RNA sequencing (scRNA-seq) with correlation in cell-type composition across tissue locations. Modeling spatial correlation allows us to borrow the cell-type composition information across locations, improving accuracy of deconvolution even with a mismatched scRNA-seq reference. CARD can also impute cell-type compositions and gene expression levels at unmeasured tissue locations to enable the construction of a refined spatial tissue map with a resolution arbitrarily higher than that measured in the original study and can perform deconvolution without an scRNA-seq reference. Applications to four datasets, including a pancreatic cancer dataset, identified multiple cell types and molecular markers with distinct spatial localization that define the progression, heterogeneity and compartmentalization of pancreatic cancer. Analysis of spatial transcriptomics is improved by correlating cell-type compositions across tissue locations.},
  archive      = {J_NBT},
  author       = {Ma, Ying and Zhou, Xiang},
  doi          = {10.1038/s41587-022-01273-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1349-1359},
  shortjournal = {Nature Biotech.},
  title        = {Spatially informed cell-type deconvolution for spatial transcriptomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A genome and gene catalog of glacier microbiomes.
<em>NBT</em>, <em>40</em>(9), 1341–1348. (<a
href="https://doi.org/10.1038/s41587-022-01367-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glaciers represent a unique inventory of microbial genetic diversity and a record of evolution. The Tibetan Plateau contains the largest area of low-latitude glaciers and is particularly vulnerable to global warming. By sequencing 85 metagenomes and 883 cultured isolates from 21 Tibetan glaciers covering snow, ice and cryoconite habitats, we present a specialized glacier microbial genome and gene catalog to archive glacial genomic and functional diversity. This comprehensive Tibetan Glacier Genome and Gene (TG2G) catalog includes 883 genomes and 2,358 metagenome-assembled genomes, which represent 968 candidate species spanning 30 phyla. The catalog also contains over 25 million non-redundant protein-encoding genes, the utility of which is demonstrated by the exploration of secondary metabolite biosynthetic potentials, virulence factor identification and global glacier metagenome comparison. The TG2G catalog is a valuable resource that enables enhanced understanding of the structure and functions of Tibetan glacial microbiomes. The bacterial life in glaciers is comprehensively catalogued and analyzed.},
  archive      = {J_NBT},
  author       = {Liu, Yongqin and Ji, Mukan and Yu, Tao and Zaugg, Julian and Anesio, Alexandre M. and Zhang, Zhihao and Hu, Songnian and Hugenholtz, Philip and Liu, Keshao and Liu, Pengfei and Chen, Yuying and Luo, Yingfeng and Yao, Tandong},
  doi          = {10.1038/s41587-022-01367-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1341-1348},
  shortjournal = {Nature Biotech.},
  title        = {A genome and gene catalog of glacier microbiomes},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Thermodynamically coupled biosensors for detecting
neutralizing antibodies against SARS-CoV-2 variants. <em>NBT</em>,
<em>40</em>(9), 1336–1340. (<a
href="https://doi.org/10.1038/s41587-022-01280-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We designed a protein biosensor that uses thermodynamic coupling for sensitive and rapid detection of neutralizing antibodies against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants in serum. The biosensor is a switchable, caged luciferase–receptor-binding domain (RBD) construct that detects serum-antibody interference with the binding of virus RBD to angiotensin-converting enzyme 2 (ACE-2) as a proxy for neutralization. Our coupling approach does not require target modification and can better distinguish sample-to-sample differences in analyte binding affinity and abundance than traditional competition-based assays. SARS-CoV-2 neutralizing antibodies are readily detected with a biosensor.},
  archive      = {J_NBT},
  author       = {Zhang, Jason Z. and Yeh, Hsien-Wei and Walls, Alexandra C. and Wicky, Basile I. M. and Sprouse, Kaitlin R. and VanBlargan, Laura A. and Treger, Rebecca and Quijano-Rubio, Alfredo and Pham, Minh N. and Kraft, John C. and Haydon, Ian C. and Yang, Wei and DeWitt, Michelle and Bowen, John E. and Chow, Cameron M. and Carter, Lauren and Ravichandran, Rashmi and Wener, Mark H. and Stewart, Lance and Veesler, David and Diamond, Michael S. and Greninger, Alexander L. and Koelle, David M. and Baker, David},
  doi          = {10.1038/s41587-022-01280-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1336-1340},
  shortjournal = {Nature Biotech.},
  title        = {Thermodynamically coupled biosensors for detecting neutralizing antibodies against SARS-CoV-2 variants},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Haplotype-resolved assembly of diploid genomes without
parental data. <em>NBT</em>, <em>40</em>(9), 1332–1335. (<a
href="https://doi.org/10.1038/s41587-022-01261-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Routine haplotype-resolved genome assembly from single samples remains an unresolved problem. Here we describe an algorithm that combines PacBio HiFi reads and Hi-C chromatin interaction data to produce a haplotype-resolved assembly without the sequencing of parents. Applied to human and other vertebrate samples, our algorithm consistently outperforms existing single-sample assembly pipelines and generates assemblies of similar quality to the best pedigree-based assemblies. Haplotype-resolved genome assemblies are generated by combining HiFi reads with Hi-C long-range interactions.},
  archive      = {J_NBT},
  author       = {Cheng, Haoyu and Jarvis, Erich D. and Fedrigo, Olivier and Koepfli, Klaus-Peter and Urban, Lara and Gemmell, Neil J. and Li, Heng},
  doi          = {10.1038/s41587-022-01261-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1332-1335},
  shortjournal = {Nature Biotech.},
  title        = {Haplotype-resolved assembly of diploid genomes without parental data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Archiving the genomic and genetic resources of glaciers.
<em>NBT</em>, <em>40</em>(9), 1330–1331. (<a
href="https://doi.org/10.1038/s41587-022-01378-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Tibetan glacier genome and gene catalog includes 3,241 metagenome-assembled or cultured genomes spanning 30 phyla and over 25 million genes encoding non-redundant proteins. This catalog provides a wealth of resources for archiving glacier microbial diversity and bioprospecting for bioactive compounds, as well as a platform for global comparison of glacier microbiomes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01378-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1330-1331},
  shortjournal = {Nature Biotech.},
  title        = {Archiving the genomic and genetic resources of glaciers},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A new route to vaccines using PROTACs. <em>NBT</em>,
<em>40</em>(9), 1328–1329. (<a
href="https://doi.org/10.1038/s41587-022-01406-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An influenza vaccine is created by attenuating the live virus through targeted proteolysis.},
  archive      = {J_NBT},
  author       = {Gilbertson, Brad and Subbarao, Kanta},
  doi          = {10.1038/s41587-022-01406-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1328-1329},
  shortjournal = {Nature Biotech.},
  title        = {A new route to vaccines using PROTACs},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Phage therapy suppresses gut inflammation in IBD.
<em>NBT</em>, <em>40</em>(9), 1327. (<a
href="https://doi.org/10.1038/s41587-022-01477-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01477-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1327},
  shortjournal = {Nature Biotech.},
  title        = {Phage therapy suppresses gut inflammation in IBD},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Large-scale in situ CRISPR screens in mice. <em>NBT</em>,
<em>40</em>(9), 1327. (<a
href="https://doi.org/10.1038/s41587-022-01478-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01478-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1327},
  shortjournal = {Nature Biotech.},
  title        = {Large-scale in situ CRISPR screens in mice},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Synthetic mouse embryos. <em>NBT</em>, <em>40</em>(9),
1327. (<a href="https://doi.org/10.1038/s41587-022-01479-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01479-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1327},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic mouse embryos},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Lipid nanoparticle drug delivery. <em>NBT</em>,
<em>40</em>(9), 1326. (<a
href="https://doi.org/10.1038/s41587-022-01462-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to making and using lipid coatings for the delivery of therapeutic agents.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01462-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1326},
  shortjournal = {Nature Biotech.},
  title        = {Lipid nanoparticle drug delivery},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Brand-name market exclusivity for nebulizer therapy to treat
asthma and COPD. <em>NBT</em>, <em>40</em>(9), 1319–1325. (<a
href="https://doi.org/10.1038/s41587-022-01451-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Manufacturers of brand-name nebulizer solutions have employed a variety of strategies to preserve market dominance over their products but have been less successful than manufacturers of brand-name inhalers in preventing generic competition.},
  archive      = {J_NBT},
  author       = {Feldman, William B. and Bloomfield, Doni and Beall, Reed F. and Kesselheim, Aaron S.},
  doi          = {10.1038/s41587-022-01451-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1319-1325},
  shortjournal = {Nature Biotech.},
  title        = {Brand-name market exclusivity for nebulizer therapy to treat asthma and COPD},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Arthur d. Riggs 1939–2022. <em>NBT</em>, <em>40</em>(9),
1317–1318. (<a
href="https://doi.org/10.1038/s41587-022-01453-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A world-renowned diabetes expert and molecular biologist, Arthur (Art) Riggs pioneered the lifesaving recombinant protein and antibody drugs on which the biotech industry was built.},
  archive      = {J_NBT},
  author       = {Pfeifer, Gerd P. and Singer-Sam, Judith and Itakura, Keiichi},
  doi          = {10.1038/s41587-022-01453-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1317-1318},
  shortjournal = {Nature Biotech.},
  title        = {Arthur d. riggs 1939–2022},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). HSV-1’s contribution as a vector for gene therapy.
<em>NBT</em>, <em>40</em>(9), 1316. (<a
href="https://doi.org/10.1038/s41587-022-01449-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Epstein, Alberto L.},
  doi          = {10.1038/s41587-022-01449-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1316},
  shortjournal = {Nature Biotech.},
  title        = {HSV-1’s contribution as a vector for gene therapy},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(9), 1309. (<a
href="https://doi.org/10.1038/s41587-022-01458-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01458-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1309},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Fungi solution for field fires. <em>NBT</em>,
<em>40</em>(9), 1308. (<a
href="https://doi.org/10.1038/s41587-022-01460-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Balakrishnan, Vijay Shankar},
  doi          = {10.1038/s41587-022-01460-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1308},
  shortjournal = {Nature Biotech.},
  title        = {Fungi solution for field fires},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Monkeypox response relies on three vaccine suppliers.
<em>NBT</em>, <em>40</em>(9), 1306–1307. (<a
href="https://doi.org/10.1038/s41587-022-01463-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biodefense stockpiles are helping to curb the monkeypox outbreak, but the vaccines are based on old technology with significant side effects. Are other vaccines on the horizon?},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-022-01463-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1306-1307},
  shortjournal = {Nature Biotech.},
  title        = {Monkeypox response relies on three vaccine suppliers},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Itch receptor OSMR attracts industry. <em>NBT</em>,
<em>40</em>(9), 1306. (<a
href="https://doi.org/10.1038/s41587-022-01481-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01481-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1306},
  shortjournal = {Nature Biotech.},
  title        = {Itch receptor OSMR attracts industry},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). WHO tools up for equitable genomics. <em>NBT</em>,
<em>40</em>(9), 1305. (<a
href="https://doi.org/10.1038/s41587-022-01482-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01482-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1305},
  shortjournal = {Nature Biotech.},
  title        = {WHO tools up for equitable genomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Lab-made antibody stops malaria. <em>NBT</em>,
<em>40</em>(9), 1304. (<a
href="https://doi.org/10.1038/s41587-022-01480-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01480-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1304},
  shortjournal = {Nature Biotech.},
  title        = {Lab-made antibody stops malaria},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Machine learning powers biobank-driven drug discovery.
<em>NBT</em>, <em>40</em>(9), 1303–1305. (<a
href="https://doi.org/10.1038/s41587-022-01457-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug hunters are moving into the clinic with human-first ‘no-hypothesis’ target discovery, applying the full force of machine learning to massive collections of human omics data.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-022-01457-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1303-1305},
  shortjournal = {Nature Biotech.},
  title        = {Machine learning powers biobank-driven drug discovery},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). One pangenome to bind them all. <em>NBT</em>,
<em>40</em>(9), 1301. (<a
href="https://doi.org/10.1038/s41587-022-01484-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The pangenome provides a first glimpse of the scope of human genetic diversity. But its routine adoption into research and clinical practice faces several challenges.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01484-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1301},
  shortjournal = {Nature Biotech.},
  title        = {One pangenome to bind them all},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Decentralized investor communities gain traction in biotech.
<em>NBT</em>, <em>40</em>(9), 1310–1315. (<a
href="https://doi.org/10.1038/s41587-022-01459-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite its checkered reputation, cryptocurrency is being considered by a handful of academics as a means to funnel funds into translational research. Is the trend likely to spread? Laura DeFrancesco and Ariel Klevecz investigate.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura and Klevecz, Ariel},
  doi          = {10.1038/s41587-022-01459-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {9},
  pages        = {1310-1315},
  shortjournal = {Nature Biotech.},
  title        = {Decentralized investor communities gain traction in biotech},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). People. <em>NBT</em>, <em>40</em>(8), 1300. (<a
href="https://doi.org/10.1038/s41587-022-01422-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01422-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1300},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Second-quarter biotech job picture. <em>NBT</em>,
<em>40</em>(8), 1299. (<a
href="https://doi.org/10.1038/s41587-022-01421-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-022-01421-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1299},
  shortjournal = {Nature Biotech.},
  title        = {Second-quarter biotech job picture},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Can hackathons unlock a new talent pool from the developing
world? <em>NBT</em>, <em>40</em>(8), 1297–1298. (<a
href="https://doi.org/10.1038/s41587-022-01415-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hackathons not only advance science itself but also help to educate young biotechnologists and artificial intelligence enthusiasts and to build capacity in digital biotechnology in low-income settings.},
  archive      = {J_NBT},
  author       = {Jenkins, Timothy P. and Lopez Carranza, Nicolas and Bray, Amy and Beguir, Karim and Laustsen, Andreas H.},
  doi          = {10.1038/s41587-022-01415-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1297-1298},
  shortjournal = {Nature Biotech.},
  title        = {Can hackathons unlock a new talent pool from the developing world?},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Integrative spatial analysis of cell
morphologies and transcriptional states with MUSE. <em>NBT</em>,
<em>40</em>(8), 1295. (<a
href="https://doi.org/10.1038/s41587-022-01340-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bao, Feng and Deng, Yue and Wan, Sen and Shen, Susan Q. and Wang, Bo and Dai, Qionghai and Altschuler, Steven J. and Wu, Lani F.},
  doi          = {10.1038/s41587-022-01340-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1295},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Integrative spatial analysis of cell morphologies and transcriptional states with MUSE},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: LAMP-seq enables sensitive, multiplexed
COVID-19 diagnostics using molecular barcoding. <em>NBT</em>,
<em>40</em>(8), 1295. (<a
href="https://doi.org/10.1038/s41587-022-01428-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ludwig, Kerstin U. and Schmithausen, Ricarda M. and Li, David and Jacobs, Max L. and Hollstein, Ronja and Blumenstock, Katja and Liebing, Jana and Słabicki, Mikołaj and Ben-Shmuel, Amir and Israeli, Ofir and Weiss, Shay and Ebert, Thomas S. and Paran, Nir and Rüdiger, Wibke and Wilbring, Gero and Feldman, David and Lippke, Bärbel and Ishorst, Nina and Hochfeld, Lara M. and Beins, Eva C. and Kaltheuner, Ines H. and Schmitz, Maximilian and Wöhler, Aliona and Döhla, Manuel and Sib, Esther and Jentzsch, Marius and Moench, Eva-Maria C. and Borrajo, Jacob D. and Strecker, Jonathan and Reinhardt, Julia and Cleary, Brian and Geyer, Matthias and Hölzel, Michael and Macrae, Rhiannon and Nöthen, Markus M. and Hoffmann, Per and Exner, Martin and Regev, Aviv and Zhang, Feng and Schmid-Burgk, Jonathan L.},
  doi          = {10.1038/s41587-022-01428-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1295},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: LAMP-seq enables sensitive, multiplexed COVID-19 diagnostics using molecular barcoding},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fludarabine increases nuclease-free AAV- and
CRISPR/Cas9-mediated homologous recombination in mice. <em>NBT</em>,
<em>40</em>(8), 1285–1294. (<a
href="https://doi.org/10.1038/s41587-022-01240-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Homologous recombination (HR)-based gene therapy using adeno-associated viruses (AAV-HR) without nucleases has several advantages over classic gene therapy, especially the potential for permanent transgene expression. However, the low efficiency of AAV-HR remains a major limitation. Here, we tested a series of small-molecule compounds and found that ribonucleotide reductase (RNR) inhibitors substantially enhance AAV-HR efficiency in mouse and human liver cell lines approximately threefold. Short-term administration of the RNR inhibitor fludarabine increased the in vivo efficiency of both non-nuclease- and CRISPR/Cas9-mediated AAV-HR two- to sevenfold in the murine liver, without causing overt toxicity. Fludarabine administration induced transient DNA damage signaling in both proliferating and quiescent hepatocytes. Notably, the majority of AAV-HR events occurred in non-proliferating hepatocytes in both fludarabine-treated and control mice, suggesting that the induction of transient DNA repair signaling in non-dividing hepatocytes was responsible for enhancing AAV-HR efficiency in mice. These results suggest that use of a clinically approved RNR inhibitor can potentiate AAV-HR-based genome-editing therapeutics. Fludarabine-induced DNA damage pathways enhance the in vivo efficiency of homologous recombination-based gene editing.},
  archive      = {J_NBT},
  author       = {Tsuji, Shinnosuke and Stephens, Calvin J. and Bortolussi, Giulia and Zhang, Feijie and Baj, Gabriele and Jang, Hagoon and de Alencastro, Gustavo and Muro, Andrés F. and Pekrun, Katja and Kay, Mark A.},
  doi          = {10.1038/s41587-022-01240-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1285-1294},
  shortjournal = {Nature Biotech.},
  title        = {Fludarabine increases nuclease-free AAV- and CRISPR/Cas9-mediated homologous recombination in mice},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accurate detection of tumor-specific gene fusions reveals
strongly immunogenic personal neo-antigens. <em>NBT</em>,
<em>40</em>(8), 1276–1284. (<a
href="https://doi.org/10.1038/s41587-022-01247-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer-associated gene fusions are a potential source for highly immunogenic neoantigens, but the lack of computational tools for accurate, sensitive identification of personal gene fusions has limited their targeting in personalized cancer immunotherapy. Here we present EasyFuse, a machine learning computational pipeline for detecting cancer-specific gene fusions in transcriptome data obtained from human cancer samples. EasyFuse predicts personal gene fusions with high precision and sensitivity, outperforming previously described tools. By testing immunogenicity with autologous blood lymphocytes from patients with cancer, we detected pre-established CD4+ and CD8+ T cell responses for 10 of 21 (48\%) and for 1 of 30 (3\%) identified gene fusions, respectively. The high frequency of T cell responses detected in patients with cancer supports the relevance of individual gene fusions as neoantigens that might be targeted in personalized immunotherapies, especially for tumors with low mutation burden. EasyFuse detects gene fusions in cancer transcriptomes for personalized immunotherapy.},
  archive      = {J_NBT},
  author       = {Weber, David and Ibn-Salem, Jonas and Sorn, Patrick and Suchan, Martin and Holtsträter, Christoph and Lahrmann, Urs and Vogler, Isabel and Schmoldt, Kathrin and Lang, Franziska and Schrörs, Barbara and Löwer, Martin and Sahin, Ugur},
  doi          = {10.1038/s41587-022-01247-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1276-1284},
  shortjournal = {Nature Biotech.},
  title        = {Accurate detection of tumor-specific gene fusions reveals strongly immunogenic personal neo-antigens},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficient discovery of SARS-CoV-2-neutralizing antibodies
via b cell receptor sequencing and ligand blocking. <em>NBT</em>,
<em>40</em>(8), 1270–1275. (<a
href="https://doi.org/10.1038/s41587-022-01232-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although several monoclonal antibodies (mAbs) targeting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) have been approved for coronavirus disease 2019 (COVID-19) therapy, development was generally inefficient, with lead generation often requiring the production and testing of numerous antibody candidates. Here, we report that the integration of target–ligand blocking with a previously described B cell receptor-sequencing approach (linking B cell receptor to antigen specificity through sequencing (LIBRA-seq)) enables the rapid and efficient identification of multiple neutralizing mAbs that prevent the binding of SARS-CoV-2 spike (S) protein to angiotensin-converting enzyme 2 (ACE2). The combination of target–ligand blocking and high-throughput antibody sequencing promises to increase the throughput of programs aimed at discovering new neutralizing antibodies. B cell receptor sequencing with ligand blocking speeds up neutralizing antibody discovery.},
  archive      = {J_NBT},
  author       = {Shiakolas, Andrea R. and Kramer, Kevin J. and Johnson, Nicole V. and Wall, Steven C. and Suryadevara, Naveenchandra and Wrapp, Daniel and Periasamy, Sivakumar and Pilewski, Kelsey A. and Raju, Nagarajan and Nargi, Rachel and Sutton, Rachel E. and Walker, Lauren M. and Setliff, Ian and Crowe, James E. and Bukreyev, Alexander and Carnahan, Robert H. and McLellan, Jason S. and Georgiev, Ivelin S.},
  doi          = {10.1038/s41587-022-01232-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1270-1275},
  shortjournal = {Nature Biotech.},
  title        = {Efficient discovery of SARS-CoV-2-neutralizing antibodies via b cell receptor sequencing and ligand blocking},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A programmable encapsulation system improves delivery of
therapeutic bacteria in mice. <em>NBT</em>, <em>40</em>(8), 1259–1269.
(<a href="https://doi.org/10.1038/s41587-022-01244-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Living bacteria therapies have been proposed as an alternative approach to treating a broad array of cancers. In this study, we developed a genetically encoded microbial encapsulation system with tunable and dynamic expression of surface capsular polysaccharides that enhances systemic delivery. Based on a small RNA screen of capsular biosynthesis pathways, we constructed inducible synthetic gene circuits that regulate bacterial encapsulation in Escherichia coli Nissle 1917. These bacteria are capable of temporarily evading immune attack, whereas subsequent loss of encapsulation results in effective clearance in vivo. This dynamic delivery strategy enabled a ten-fold increase in maximum tolerated dose of bacteria and improved anti-tumor efficacy in murine models of cancer. Furthermore, in situ encapsulation increased the fraction of microbial translocation among mouse tumors, leading to efficacy in distal tumors. The programmable encapsulation system promises to enhance the therapeutic utility of living engineered bacteria for cancer. Transient capsule induction allows engineered bacteria to evade initial immune surveillance in a colorectal cancer model.},
  archive      = {J_NBT},
  author       = {Harimoto, Tetsuhiro and Hahn, Jaeseung and Chen, Yu-Yu and Im, Jongwon and Zhang, Joanna and Hou, Nicholas and Li, Fangda and Coker, Courtney and Gray, Kelsey and Harr, Nicole and Chowdhury, Sreyan and Pu, Kelly and Nimura, Clare and Arpaia, Nicholas and Leong, Kam W. and Danino, Tal},
  doi          = {10.1038/s41587-022-01244-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1259-1269},
  shortjournal = {Nature Biotech.},
  title        = {A programmable encapsulation system improves delivery of therapeutic bacteria in mice},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bioinstructive implantable scaffolds for rapid in vivo
manufacture and release of CAR-t cells. <em>NBT</em>, <em>40</em>(8),
1250–1258. (<a
href="https://doi.org/10.1038/s41587-022-01245-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite their clinical success, chimeric antigen receptor (CAR)-T cell therapies for B cell malignancies are limited by lengthy, costly and labor-intensive ex vivo manufacturing procedures that might lead to cell products with heterogeneous composition. Here we describe an implantable Multifunctional Alginate Scaffold for T Cell Engineering and Release (MASTER) that streamlines in vivo CAR-T cell manufacturing and reduces processing time to a single day. When seeded with human peripheral blood mononuclear cells and CD19-encoding retroviral particles, MASTER provides the appropriate interface for viral vector-mediated gene transfer and, after subcutaneous implantation, mediates the release of functional CAR-T cells in mice. We further demonstrate that in vivo-generated CAR-T cells enter the bloodstream and control distal tumor growth in a mouse xenograft model of lymphoma, showing greater persistence than conventional CAR-T cells. MASTER promises to transform CAR-T cell therapy by fast-tracking manufacture and potentially reducing the complexity and resources needed for provision of this type of therapy. Implantable scaffolds rapidly generate and release anti-tumor CAR-T cells in mice.},
  archive      = {J_NBT},
  author       = {Agarwalla, Pritha and Ogunnaike, Edikan A. and Ahn, Sarah and Froehlich, Kristen A. and Jansson, Anton and Ligler, Frances S. and Dotti, Gianpietro and Brudno, Yevgeny},
  doi          = {10.1038/s41587-022-01245-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1250-1258},
  shortjournal = {Nature Biotech.},
  title        = {Bioinstructive implantable scaffolds for rapid in vivo manufacture and release of CAR-T cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). In vivo engineered b cells secrete high titers of broadly
neutralizing anti-HIV antibodies in mice. <em>NBT</em>, <em>40</em>(8),
1241–1249. (<a
href="https://doi.org/10.1038/s41587-022-01328-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transplantation of B cells engineered ex vivo to secrete broadly neutralizing antibodies (bNAbs) has shown efficacy in disease models. However, clinical translation of this approach would require specialized medical centers, technically demanding protocols and major histocompatibility complex compatibility of donor cells and recipients. Here we report in vivo B cell engineering using two adeno-associated viral vectors, with one coding for Staphylococcus aureus Cas9 (saCas9) and the other for 3BNC117, an anti-HIV bNAb. After intravenously injecting the vectors into mice, we observe successful editing of B cells leading to memory retention and bNAb secretion at neutralizing titers of up to 6.8 µg ml−1. We observed minimal clustered regularly interspaced palindromic repeats (CRISPR)–Cas9 off-target cleavage as detected by unbiased CHANGE-sequencing analysis, whereas on-target cleavage in undesired tissues is reduced by expressing saCas9 from a B cell-specific promoter. In vivo B cell engineering to express therapeutic antibodies is a safe, potent and scalable method, which may be applicable not only to infectious diseases but also in the treatment of noncommunicable conditions, such as cancer and autoimmune disease. B cells are engineered in vivo to secrete anti-HIV antibodies},
  archive      = {J_NBT},
  author       = {Nahmad, Alessio D. and Lazzarotto, Cicera R. and Zelikson, Natalie and Kustin, Talia and Tenuta, Mary and Huang, Deli and Reuveni, Inbal and Nataf, Daniel and Raviv, Yuval and Horovitz-Fried, Miriam and Dotan, Iris and Carmi, Yaron and Rosin-Arbesfeld, Rina and Nemazee, David and Voss, James E. and Stern, Adi and Tsai, Shengdar Q. and Barzel, Adi},
  doi          = {10.1038/s41587-022-01328-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1241-1249},
  shortjournal = {Nature Biotech.},
  title        = {In vivo engineered b cells secrete high titers of broadly neutralizing anti-HIV antibodies in mice},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep visual proteomics defines single-cell identity and
heterogeneity. <em>NBT</em>, <em>40</em>(8), 1231–1240. (<a
href="https://doi.org/10.1038/s41587-022-01302-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the availabilty of imaging-based and mass-spectrometry-based methods for spatial proteomics, a key challenge remains connecting images with single-cell-resolution protein abundance measurements. Here, we introduce Deep Visual Proteomics (DVP), which combines artificial-intelligence-driven image analysis of cellular phenotypes with automated single-cell or single-nucleus laser microdissection and ultra-high-sensitivity mass spectrometry. DVP links protein abundance to complex cellular or subcellular phenotypes while preserving spatial context. By individually excising nuclei from cell culture, we classified distinct cell states with proteomic profiles defined by known and uncharacterized proteins. In an archived primary melanoma tissue, DVP identified spatially resolved proteome changes as normal melanocytes transition to fully invasive melanoma, revealing pathways that change in a spatial manner as cancer progresses, such as mRNA splicing dysregulation in metastatic vertical growth that coincides with reduced interferon signaling and antigen presentation. The ability of DVP to retain precise spatial proteomic information in the tissue context has implications for the molecular profiling of clinical samples. Deep Visual Proteomics combines machine learning, automated image analysis and single-cell proteomics.},
  archive      = {J_NBT},
  author       = {Mund, Andreas and Coscia, Fabian and Kriston, András and Hollandi, Réka and Kovács, Ferenc and Brunner, Andreas-David and Migh, Ede and Schweizer, Lisa and Santos, Alberto and Bzorek, Michael and Naimy, Soraya and Rahbek-Gjerdrum, Lise Mette and Dyring-Andersen, Beatrice and Bulkescher, Jutta and Lukas, Claudia and Eckert, Mark Adam and Lengyel, Ernst and Gnann, Christian and Lundberg, Emma and Horvath, Peter and Mann, Matthias},
  doi          = {10.1038/s41587-022-01302-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1231-1240},
  shortjournal = {Nature Biotech.},
  title        = {Deep visual proteomics defines single-cell identity and heterogeneity},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Characterizing cellular heterogeneity in chromatin state
with scCUT&amp;tag-pro. <em>NBT</em>, <em>40</em>(8), 1220–1230. (<a
href="https://doi.org/10.1038/s41587-022-01250-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technologies that profile chromatin modifications at single-cell resolution offer enormous promise for functional genomic characterization, but the sparsity of the measurements and integrating multiple binding maps represent substantial challenges. Here we introduce single-cell (sc)CUT&amp;amp;Tag-pro, a multimodal assay for profiling protein–DNA interactions coupled with the abundance of surface proteins in single cells. In addition, we introduce single-cell ChromHMM, which integrates data from multiple experiments to infer and annotate chromatin states based on combinatorial histone modification patterns. We apply these tools to perform an integrated analysis across nine different molecular modalities in circulating human immune cells. We demonstrate how these two approaches can characterize dynamic changes in the function of individual genomic elements across both discrete cell states and continuous developmental trajectories, nominate associated motifs and regulators that establish chromatin states and identify extensive and cell-type-specific regulatory priming. Finally, we demonstrate how our integrated reference can serve as a scaffold to map and improve the interpretation of additional scCUT&amp;amp;Tag datasets. scCUT&amp;amp;Tag-pro measures a histone modification and surface proteins in single cells.},
  archive      = {J_NBT},
  author       = {Zhang, Bingjie and Srivastava, Avi and Mimitou, Eleni and Stuart, Tim and Raimondi, Ivan and Hao, Yuhan and Smibert, Peter and Satija, Rahul},
  doi          = {10.1038/s41587-022-01250-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1220-1230},
  shortjournal = {Nature Biotech.},
  title        = {Characterizing cellular heterogeneity in chromatin state with scCUT&amp;Tag-pro},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). M6A RNA modifications are measured at single-base resolution
across the mammalian transcriptome. <em>NBT</em>, <em>40</em>(8),
1210–1219. (<a
href="https://doi.org/10.1038/s41587-022-01243-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Functional studies of the RNA N6-methyladenosine (m6A) modification have been limited by an inability to map individual m6A-modified sites in whole transcriptomes. To enable such studies, here, we introduce m6A-selective allyl chemical labeling and sequencing (m6A-SAC-seq), a method for quantitative, whole-transcriptome mapping of m6A at single-nucleotide resolution. The method requires only ~30 ng of poly(A) or rRNA-depleted RNA. We mapped m6A modification stoichiometries in RNA from cell lines and during in vitro monocytopoiesis from human hematopoietic stem and progenitor cells (HSPCs). We identified numerous cell-state-specific m6A sites whose methylation status was highly dynamic during cell differentiation. We observed changes of m6A stoichiometry as well as expression levels of transcripts encoding or regulated by key transcriptional factors (TFs) critical for HSPC differentiation. m6A-SAC-seq is a quantitative method to dissect the dynamics and functional roles of m6A sites in diverse biological processes using limited input RNA. m6A-SAC-seq uses chemical labeling to quantify m6A at single-base resolution in the mammalian transcriptome.},
  archive      = {J_NBT},
  author       = {Hu, Lulu and Liu, Shun and Peng, Yong and Ge, Ruiqi and Su, Rui and Senevirathne, Chamara and Harada, Bryan T. and Dai, Qing and Wei, Jiangbo and Zhang, Lisheng and Hao, Ziyang and Luo, Liangzhi and Wang, Huanyu and Wang, Yuru and Luo, Minkui and Chen, Mengjie and Chen, Jianjun and He, Chuan},
  doi          = {10.1038/s41587-022-01243-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1210-1219},
  shortjournal = {Nature Biotech.},
  title        = {M6A RNA modifications are measured at single-base resolution across the mammalian transcriptome},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Integrative spatial analysis of cell morphologies and
transcriptional states with MUSE. <em>NBT</em>, <em>40</em>(8),
1200–1209. (<a
href="https://doi.org/10.1038/s41587-022-01251-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics enables the simultaneous measurement of morphological features and transcriptional profiles of the same cells or regions in tissues. Here we present multi-modal structured embedding (MUSE), an approach to characterize cells and tissue regions by integrating morphological and spatially resolved transcriptional data. We demonstrate that MUSE can discover tissue subpopulations missed by either modality as well as compensate for modality-specific noise. We apply MUSE to diverse datasets containing spatial transcriptomics (seqFISH+, STARmap or Visium) and imaging (hematoxylin and eosin or fluorescence microscopy) modalities. MUSE identified biologically meaningful tissue subpopulations and stereotyped spatial patterning in healthy brain cortex and intestinal tissues. In diseased tissues, MUSE revealed gene biomarkers for proximity to tumor region and heterogeneity of amyloid precursor protein processing across Alzheimer brain regions. MUSE enables the integration of multi-modal data to provide insights into the states, functions and organization of cells in complex biological tissues. Deep learning analysis of microscopy and spatial transcriptomics data characterizes cell types and states.},
  archive      = {J_NBT},
  author       = {Bao, Feng and Deng, Yue and Wan, Sen and Shen, Susan Q. and Wang, Bo and Dai, Qionghai and Altschuler, Steven J. and Wu, Lani F.},
  doi          = {10.1038/s41587-022-01251-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1200-1209},
  shortjournal = {Nature Biotech.},
  title        = {Integrative spatial analysis of cell morphologies and transcriptional states with MUSE},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial charting of single-cell transcriptomes in tissues.
<em>NBT</em>, <em>40</em>(8), 1190–1199. (<a
href="https://doi.org/10.1038/s41587-022-01233-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing methods can profile the transcriptomes of single cells but cannot preserve spatial information. Conversely, spatial transcriptomics assays can profile spatial regions in tissue sections, but do not have single-cell resolution. Here, we developed a computational method called CellTrek that combines these two datasets to achieve single-cell spatial mapping through coembedding and metric learning approaches. We benchmarked CellTrek using simulation and in situ hybridization datasets, which demonstrated its accuracy and robustness. We then applied CellTrek to existing mouse brain and kidney datasets and showed that CellTrek can detect topological patterns of different cell types and cell states. We performed single-cell RNA sequencing and spatial transcriptomics experiments on two ductal carcinoma in situ tissues and applied CellTrek to identify tumor subclones that were restricted to different ducts, and specific T cell states adjacent to the tumor areas. Our data show that CellTrek can accurately map single cells in diverse tissue types to resolve their spatial organization. CellTrek maps single cells to their spatial coordinates in tissues.},
  archive      = {J_NBT},
  author       = {Wei, Runmin and He, Siyuan and Bai, Shanshan and Sei, Emi and Hu, Min and Thompson, Alastair and Chen, Ken and Krishnamurthy, Savitri and Navin, Nicholas E.},
  doi          = {10.1038/s41587-022-01233-1},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1190-1199},
  shortjournal = {Nature Biotech.},
  title        = {Spatial charting of single-cell transcriptomes in tissues},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Expanding the spectrum of cancer targets by predicting
individual gene fusions. <em>NBT</em>, <em>40</em>(8), 1188–1189. (<a
href="https://doi.org/10.1038/s41587-022-01267-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed EasyFuse, a computational machine learning pipeline that detects cancer-specific gene fusions with superior performance over existing tools. Individual gene fusions exhibit a high frequency of pre-established CD4+ and CD8+ T-cell responses and thus represent a previously untapped source of neo-antigens that can be exploited for personalized immunotherapies.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01267-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1188-1189},
  shortjournal = {Nature Biotech.},
  title        = {Expanding the spectrum of cancer targets by predicting individual gene fusions},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial characterization of single tumor cells by
proteomics. <em>NBT</em>, <em>40</em>(8), 1186–1187. (<a
href="https://doi.org/10.1038/s41587-022-01321-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By quantifying thousands of proteins in tumor cells in an unbiased manner, Deep Visual Proteomics uncovers mechanisms that drive tumor evolution and reveals new therapeutic targets. The method incorporates advanced microscopy, artificial intelligence and ultra-high-sensitivity proteomics to characterize individual cell identities.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01321-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1186-1187},
  shortjournal = {Nature Biotech.},
  title        = {Spatial characterization of single tumor cells by proteomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Forum: Bertozzi and deshaies. <em>NBT</em>, <em>40</em>(8),
1185. (<a href="https://doi.org/10.1038/s41587-022-01436-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01436-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1185},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bertozzi and deshaies},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CRISPR comes a-knock-in to reprogram antibodies in vivo.
<em>NBT</em>, <em>40</em>(8), 1183–1184. (<a
href="https://doi.org/10.1038/s41587-022-01299-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reprogramming of antibody responses in mice is achieved via adeno-associated virus delivery of SaCas9, single guide RNA (sgRNA) and a repair template targeting immunoglobulin genes.},
  archive      = {J_NBT},
  author       = {Hartweger, Harald and Nussenzweig, Michel C.},
  doi          = {10.1038/s41587-022-01299-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1183-1184},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR comes a-knock-in to reprogram antibodies in vivo},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Capturing toxic amyloid in biofluids. <em>NBT</em>,
<em>40</em>(8), 1182. (<a
href="https://doi.org/10.1038/s41587-022-01437-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01437-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1182},
  shortjournal = {Nature Biotech.},
  title        = {Capturing toxic amyloid in biofluids},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). A DNA typewriter in living cells. <em>NBT</em>,
<em>40</em>(8), 1182. (<a
href="https://doi.org/10.1038/s41587-022-01438-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01438-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1182},
  shortjournal = {Nature Biotech.},
  title        = {A DNA typewriter in living cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A beating biofabricated heart. <em>NBT</em>,
<em>40</em>(8), 1182. (<a
href="https://doi.org/10.1038/s41587-022-01439-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kotsiliti, Eleni},
  doi          = {10.1038/s41587-022-01439-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1182},
  shortjournal = {Nature Biotech.},
  title        = {A beating biofabricated heart},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Epigenomics. <em>NBT</em>, <em>40</em>(8), 1181. (<a
href="https://doi.org/10.1038/s41587-022-01433-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to epigenomic analyses, modification and engineering.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01433-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1181},
  shortjournal = {Nature Biotech.},
  title        = {Epigenomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Patents benefit patients and patent reform would spur
diagnostic and therapeutic development. <em>NBT</em>, <em>40</em>(8),
1178–1180. (<a
href="https://doi.org/10.1038/s41587-022-01405-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patients need the patent system to spur the investment necessary for the continued development of life-saving diagnostics and therapies.},
  archive      = {J_NBT},
  author       = {Coruzzi, Laura A.},
  doi          = {10.1038/s41587-022-01405-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1178-1180},
  shortjournal = {Nature Biotech.},
  title        = {Patents benefit patients and patent reform would spur diagnostic and therapeutic development},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The BioInnovation institute as a catalyst for european life
science startup creation. <em>NBT</em>, <em>40</em>(8), 1175–1177. (<a
href="https://doi.org/10.1038/s41587-022-01416-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Nielsen, Jens and Herrgård, Markus J. and Soni, Bobby},
  doi          = {10.1038/s41587-022-01416-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1175-1177},
  shortjournal = {Nature Biotech.},
  title        = {The BioInnovation institute as a catalyst for european life science startup creation},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The digital phenotype of vaccination. <em>NBT</em>,
<em>40</em>(8), 1174–1175. (<a
href="https://doi.org/10.1038/s41587-022-01417-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Quer, Giorgio and Topol, Eric J. and Steinhubl, Steven R.},
  doi          = {10.1038/s41587-022-01417-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1174-1175},
  shortjournal = {Nature Biotech.},
  title        = {The digital phenotype of vaccination},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Fighting fibrosis. <em>NBT</em>, <em>40</em>(8), 1169–1173.
(<a href="https://doi.org/10.1038/s41587-022-01412-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Improving target selectivity and defining cell phenotypes may improve the odds of finding therapies for fibrosis.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-022-01412-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1169-1173},
  shortjournal = {Nature Biotech.},
  title        = {Fighting fibrosis},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). 2Q22 — slip sliding away. <em>NBT</em>, <em>40</em>(8),
1166–1168. (<a
href="https://doi.org/10.1038/s41587-022-01414-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01414-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1166-1168},
  shortjournal = {Nature Biotech.},
  title        = {2Q22 — slip sliding away},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Drug pipeline 2Q22 — a downturn in approvals. <em>NBT</em>,
<em>40</em>(8), 1164–1165. (<a
href="https://doi.org/10.1038/s41587-022-01413-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01413-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1164-1165},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 2Q22 — a downturn in approvals},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Cellulose shoes made by bacteria. <em>NBT</em>,
<em>40</em>(8), 1163. (<a
href="https://doi.org/10.1038/s41587-022-01431-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-022-01431-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1163},
  shortjournal = {Nature Biotech.},
  title        = {Cellulose shoes made by bacteria},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Vertex snaps up diabetes stem cell rival. <em>NBT</em>,
<em>40</em>(8), 1161. (<a
href="https://doi.org/10.1038/s41587-022-01443-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01443-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1161},
  shortjournal = {Nature Biotech.},
  title        = {Vertex snaps up diabetes stem cell rival},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). MRNA printers kick-start personalized medicines for all.
<em>NBT</em>, <em>40</em>(8), 1160–1162. (<a
href="https://doi.org/10.1038/s41587-022-01430-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {mRNA printers will bring low-cost vaccines and made-to-order treatments for a range of different diseases.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-022-01430-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1160-1162},
  shortjournal = {Nature Biotech.},
  title        = {MRNA printers kick-start personalized medicines for all},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CureVac sues BioNTech over mRNA technology. <em>NBT</em>,
<em>40</em>(8), 1160. (<a
href="https://doi.org/10.1038/s41587-022-01442-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01442-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1160},
  shortjournal = {Nature Biotech.},
  title        = {CureVac sues BioNTech over mRNA technology},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Verve takes base editors into humans. <em>NBT</em>,
<em>40</em>(8), 1159. (<a
href="https://doi.org/10.1038/s41587-022-01445-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01445-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1159},
  shortjournal = {Nature Biotech.},
  title        = {Verve takes base editors into humans},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). HER3-addicted tumors: How biotechs are closing in.
<em>NBT</em>, <em>40</em>(8), 1157–1159. (<a
href="https://doi.org/10.1038/s41587-022-01425-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The elusive HER3, known to drive aggressive cancers, may finally be toppled by once-abandoned antibody drugs.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01425-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1157-1159},
  shortjournal = {Nature Biotech.},
  title        = {HER3-addicted tumors: How biotechs are closing in},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Increase vigilance against cyberattacks. <em>NBT</em>,
<em>40</em>(8), 1155. (<a
href="https://doi.org/10.1038/s41587-022-01446-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biotech sector must devote more resources to cybersecurity — especially those companies that are manufacturers of essential medical products.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01446-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1155},
  shortjournal = {Nature Biotech.},
  title        = {Increase vigilance against cyberattacks},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). People. <em>NBT</em>, <em>40</em>(7), 1154. (<a
href="https://doi.org/10.1038/s41587-022-01391-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01391-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1154},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). What digitalization in biology r&amp;d means for biotech
companies and life scientists. <em>NBT</em>, <em>40</em>(7), 1151–1153.
(<a href="https://doi.org/10.1038/s41587-022-01309-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The digitalization of R&amp;amp;D can potentially transform the life sciences, but it cannot succeed without experts collaborating to develop a vision of what a better working environment can look like.},
  archive      = {J_NBT},
  author       = {Wong, Sybil and Pan, Mingke and Shaw, Allen and Gershater, Markus},
  doi          = {10.1038/s41587-022-01309-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1151-1153},
  shortjournal = {Nature Biotech.},
  title        = {What digitalization in biology R&amp;D means for biotech companies and life scientists},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Biotech patenting 2019. <em>NBT</em>,
<em>40</em>(7), 1150. (<a
href="https://doi.org/10.1038/s41587-022-01399-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-022-01399-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1150},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Biotech patenting 2019},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Publisher correction: Biotech patenting 2020. <em>NBT</em>,
<em>40</em>(7), 1150. (<a
href="https://doi.org/10.1038/s41587-022-01400-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-022-01400-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1150},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Biotech patenting 2020},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computationally designed dual-color MRI reporters for
noninvasive imaging of transgene expression. <em>NBT</em>,
<em>40</em>(7), 1143–1149. (<a
href="https://doi.org/10.1038/s41587-021-01162-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging of gene-expression patterns in live animals is difficult to achieve with fluorescent proteins because tissues are opaque to visible light. Imaging of transgene expression with magnetic resonance imaging (MRI), which penetrates to deep tissues, has been limited by single reporter visualization capabilities. Moreover, the low-throughput capacity of MRI limits large-scale mutagenesis strategies to improve existing reporters. Here we develop an MRI system, called GeneREFORM, comprising orthogonal reporters for two-color imaging of transgene expression in deep tissues. Starting from two promiscuous deoxyribonucleoside kinases, we computationally designed highly active, orthogonal enzymes (‘reporter genes’) that specifically phosphorylate two MRI-detectable synthetic deoxyribonucleosides (‘reporter probes’). Systemically administered reporter probes exclusively accumulate in cells expressing the designed reporter genes, and their distribution is displayed as pseudo-colored MRI maps based on dynamic proton exchange for noninvasive visualization of transgene expression. We envision that future extensions of GeneREFORM will pave the way to multiplexed deep-tissue mapping of gene expression in live animals. MRI detection of transgene expression in animals is expanded to two colors.},
  archive      = {J_NBT},
  author       = {Allouche-Arnon, Hyla and Khersonsky, Olga and Tirukoti, Nishanth D. and Peleg, Yoav and Dym, Orly and Albeck, Shira and Brandis, Alexander and Mehlman, Tevie and Avram, Liat and Harris, Talia and Yadav, Nirbhay N. and Fleishman, Sarel J. and Bar-Shir, Amnon},
  doi          = {10.1038/s41587-021-01162-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1143-1149},
  shortjournal = {Nature Biotech.},
  title        = {Computationally designed dual-color MRI reporters for noninvasive imaging of transgene expression},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A highly photostable and bright green fluorescent protein.
<em>NBT</em>, <em>40</em>(7), 1132–1142. (<a
href="https://doi.org/10.1038/s41587-022-01278-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The low photostability of fluorescent proteins is a limiting factor in many applications of fluorescence microscopy. Here we present StayGold, a green fluorescent protein (GFP) derived from the jellyfish Cytaeis uchidae. StayGold is over one order of magnitude more photostable than any currently available fluorescent protein and has a cellular brightness similar to mNeonGreen. We used StayGold to image the dynamics of the endoplasmic reticulum (ER) with high spatiotemporal resolution over several minutes using structured illumination microscopy (SIM) and observed substantially less photobleaching than with a GFP variant optimized for stability in the ER. Using StayGold fusions and SIM, we also imaged the dynamics of mitochondrial fusion and fission and mapped the viral spike proteins in fixed cells infected with severe acute respiratory syndrome coronavirus 2. As StayGold is a dimer, we created a tandem dimer version that allowed us to observe the dynamics of microtubules and the excitatory post-synaptic density in neurons. StayGold will substantially reduce the limitations imposed by photobleaching, especially in live cell or volumetric imaging. StayGold is over one order of magnitude more photostable than current fluorescent proteins},
  archive      = {J_NBT},
  author       = {Hirano, Masahiko and Ando, Ryoko and Shimozono, Satoshi and Sugiyama, Mayu and Takeda, Noriyo and Kurokawa, Hiroshi and Deguchi, Ryusaku and Endo, Kazuki and Haga, Kei and Takai-Todaka, Reiko and Inaura, Shunsuke and Matsumura, Yuta and Hama, Hiroshi and Okada, Yasushi and Fujiwara, Takahiro and Morimoto, Takuya and Katayama, Kazuhiko and Miyawaki, Atsushi},
  doi          = {10.1038/s41587-022-01278-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1132-1142},
  shortjournal = {Nature Biotech.},
  title        = {A highly photostable and bright green fluorescent protein},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Designing sensitive viral diagnostics with machine learning.
<em>NBT</em>, <em>40</em>(7), 1123–1131. (<a
href="https://doi.org/10.1038/s41587-022-01213-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Design of nucleic acid-based viral diagnostics typically follows heuristic rules and, to contend with viral variation, focuses on a genome’s conserved regions. A design process could, instead, directly optimize diagnostic effectiveness using a learned model of sensitivity for targets and their variants. Toward that goal, we screen 19,209 diagnostic–target pairs, concentrated on CRISPR-based diagnostics, and train a deep neural network to accurately predict diagnostic readout. We join this model with combinatorial optimization to maximize sensitivity over the full spectrum of a virus’s genomic variation. We introduce Activity-informed Design with All-inclusive Patrolling of Targets (ADAPT), a system for automated design, and use it to design diagnostics for 1,933 vertebrate-infecting viral species within 2 hours for most species and within 24 hours for all but three. We experimentally show that ADAPT’s designs are sensitive and specific to the lineage level and permit lower limits of detection, across a virus’s variation, than the outputs of standard design techniques. Our strategy could facilitate a proactive resource of assays for detecting pathogens. Viral diagnostics with maximum sensitivity are designed using machine learning and combinatorial optimization.},
  archive      = {J_NBT},
  author       = {Metsky, Hayden C. and Welch, Nicole L. and Pillai, Priya P. and Haradhvala, Nicholas J. and Rumker, Laurie and Mantena, Sreekar and Zhang, Yibin B. and Yang, David K. and Ackerman, Cheri M. and Weller, Juliane and Blainey, Paul C. and Myhrvold, Cameron and Mitzenmacher, Michael and Sabeti, Pardis C.},
  doi          = {10.1038/s41587-022-01213-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1123-1131},
  shortjournal = {Nature Biotech.},
  title        = {Designing sensitive viral diagnostics with machine learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning protein fitness models from evolutionary and
assay-labeled data. <em>NBT</em>, <em>40</em>(7), 1114–1122. (<a
href="https://doi.org/10.1038/s41587-021-01146-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning-based models of protein fitness typically learn from either unlabeled, evolutionarily related sequences or variant sequences with experimentally measured labels. For regimes where only limited experimental data are available, recent work has suggested methods for combining both sources of information. Toward that goal, we propose a simple combination approach that is competitive with, and on average outperforms more sophisticated methods. Our approach uses ridge regression on site-specific amino acid features combined with one probability density feature from modeling the evolutionary data. Within this approach, we find that a variational autoencoder-based probability density model showed the best overall performance, although any evolutionary density model can be used. Moreover, our analysis highlights the importance of systematic evaluations and sufficient baselines. A simple machine learning algorithm combines evolutionary and experimental data for improved protein fitness prediction.},
  archive      = {J_NBT},
  author       = {Hsu, Chloe and Nisonoff, Hunter and Fannjiang, Clara and Listgarten, Jennifer},
  doi          = {10.1038/s41587-021-01146-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1114-1122},
  shortjournal = {Nature Biotech.},
  title        = {Learning protein fitness models from evolutionary and assay-labeled data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Synthetic introns enable splicing factor mutation-dependent
targeting of cancer cells. <em>NBT</em>, <em>40</em>(7), 1103–1113. (<a
href="https://doi.org/10.1038/s41587-022-01224-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many cancers carry recurrent, change-of-function mutations affecting RNA splicing factors. Here, we describe a method to harness this abnormal splicing activity to drive splicing factor mutation-dependent gene expression to selectively eliminate tumor cells. We engineered synthetic introns that were efficiently spliced in cancer cells bearing SF3B1 mutations, but unspliced in otherwise isogenic wild-type cells, to yield mutation-dependent protein production. A massively parallel screen of 8,878 introns delineated ideal intronic size and mapped elements underlying mutation-dependent splicing. Synthetic introns enabled mutation-dependent expression of herpes simplex virus–thymidine kinase (HSV–TK) and subsequent ganciclovir (GCV)-mediated killing of SF3B1-mutant leukemia, breast cancer, uveal melanoma and pancreatic cancer cells in vitro, while leaving wild-type cells unaffected. Delivery of synthetic intron-containing HSV–TK constructs to leukemia, breast cancer and uveal melanoma cells and GCV treatment in vivo significantly suppressed the growth of these otherwise lethal xenografts and improved mouse host survival. Synthetic introns provide a means to exploit tumor-specific changes in RNA splicing for cancer gene therapy. Synthetic introns tailored for specific splice-factor mutations enable targeted cancer gene therapy.},
  archive      = {J_NBT},
  author       = {North, Khrystyna and Benbarche, Salima and Liu, Bo and Pangallo, Joseph and Chen, Sisi and Stahl, Maximilian and Bewersdorf, Jan Philipp and Stanley, Robert F. and Erickson, Caroline and Cho, Hana and Pineda, Jose Mario Bello and Thomas, James D. and Polaski, Jacob T. and Belleville, Andrea E. and Gabel, Austin M. and Udy, Dylan B. and Humbert, Olivier and Kiem, Hans-Peter and Abdel-Wahab, Omar and Bradley, Robert K.},
  doi          = {10.1038/s41587-022-01224-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1103-1113},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic introns enable splicing factor mutation-dependent targeting of cancer cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Endogenous ADAR-mediated RNA editing in non-human primates
using stereopure chemically modified oligonucleotides. <em>NBT</em>,
<em>40</em>(7), 1093–1102. (<a
href="https://doi.org/10.1038/s41587-022-01225-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technologies that recruit and direct the activity of endogenous RNA-editing enzymes to specific cellular RNAs have therapeutic potential, but translating them from cell culture into animal models has been challenging. Here we describe short, chemically modified oligonucleotides called AIMers that direct efficient and specific A-to-I editing of endogenous transcripts by endogenous adenosine deaminases acting on RNA (ADAR) enzymes, including the ubiquitously and constitutively expressed ADAR1 p110 isoform. We show that fully chemically modified AIMers with chimeric backbones containing stereopure phosphorothioate and nitrogen-containing linkages based on phosphoryl guanidine enhanced potency and editing efficiency 100-fold compared with those with uniformly phosphorothioate-modified backbones in vitro. In vivo, AIMers targeted to hepatocytes with N-acetylgalactosamine achieve up to 50\% editing with no bystander editing of the endogenous ACTB transcript in non-human primate liver, with editing persisting for at least one month. These results support further investigation of the therapeutic potential of stereopure AIMers. RNA in non-human primate liver is efficiently edited using short stereopure oligonucleotides.},
  archive      = {J_NBT},
  author       = {Monian, Prashant and Shivalila, Chikdu and Lu, Genliang and Shimizu, Mamoru and Boulay, David and Bussow, Karley and Byrne, Michael and Bezigian, Adam and Chatterjee, Arindom and Chew, David and Desai, Jigar and Favaloro, Frank and Godfrey, Jack and Hoss, Andrew and Iwamoto, Naoki and Kawamoto, Tomomi and Kumarasamy, Jayakanthan and Lamattina, Anthony and Lindsey, Amber and Liu, Fangjun and Looby, Richard and Marappan, Subramanian and Metterville, Jake and Murphy, Ronelle and Rossi, Jeff and Pu, Tom and Bhattarai, Bijay and Standley, Stephany and Tripathi, Snehlata and Yang, Hailin and Yin, Yuan and Yu, Hui and Zhou, Cong and Apponi, Luciano H. and Kandasamy, Pachamuthu and Vargeese, Chandra},
  doi          = {10.1038/s41587-022-01225-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1093-1102},
  shortjournal = {Nature Biotech.},
  title        = {Endogenous ADAR-mediated RNA editing in non-human primates using stereopure chemically modified oligonucleotides},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-nuclei isoform RNA sequencing unlocks barcoded exon
connectivity in frozen brain tissue. <em>NBT</em>, <em>40</em>(7),
1082–1092. (<a
href="https://doi.org/10.1038/s41587-022-01231-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nuclei RNA sequencing characterizes cell types at the gene level. However, compared to single-cell approaches, many single-nuclei cDNAs are purely intronic, lack barcodes and hinder the study of isoforms. Here we present single-nuclei isoform RNA sequencing (SnISOr-Seq). Using microfluidics, PCR-based artifact removal, target enrichment and long-read sequencing, SnISOr-Seq increased barcoded, exon-spanning long reads 7.5-fold compared to naive long-read single-nuclei sequencing. We applied SnISOr-Seq to adult human frontal cortex and found that exons associated with autism exhibit coordinated and highly cell-type-specific inclusion. We found two distinct combination patterns: those distinguishing neural cell types, enriched in TSS-exon, exon-polyadenylation-site and non-adjacent exon pairs, and those with multiple configurations within one cell type, enriched in adjacent exon pairs. Finally, we observed that human-specific exons are almost as tightly coordinated as conserved exons, implying that coordination can be rapidly established during evolution. SnISOr-Seq enables cell-type-specific long-read isoform analysis in human brain and in any frozen or hard-to-dissociate sample. Complete RNA isoforms are captured by a new single-nuclei sequencing method.},
  archive      = {J_NBT},
  author       = {Hardwick, Simon A. and Hu, Wen and Joglekar, Anoushka and Fan, Li and Collier, Paul G. and Foord, Careen and Balacco, Jennifer and Lanjewar, Samantha and Sampson, Maureen McGuirk and Koopmans, Frank and Prjibelski, Andrey D. and Mikheenko, Alla and Belchikov, Natan and Jarroux, Julien and Lucas, Anne Bergstrom and Palkovits, Miklós and Luo, Wenjie and Milner, Teresa A. and Ndhlovu, Lishomwa C. and Smit, August B. and Trojanowski, John Q. and Lee, Virginia M. Y. and Fedrigo, Olivier and Sloan, Steven A. and Tombácz, Dóra and Ross, M. Elizabeth and Jarvis, Erich and Boldogkői, Zsolt and Gan, Li and Tilgner, Hagen U.},
  doi          = {10.1038/s41587-022-01231-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1082-1092},
  shortjournal = {Nature Biotech.},
  title        = {Single-nuclei isoform RNA sequencing unlocks barcoded exon connectivity in frozen brain tissue},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multiplex de bruijn graphs enable genome assembly from long,
high-fidelity reads. <em>NBT</em>, <em>40</em>(7), 1075–1081. (<a
href="https://doi.org/10.1038/s41587-022-01220-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although most existing genome assemblers are based on de Bruijn graphs, the construction of these graphs for large genomes and large k-mer sizes has remained elusive. This algorithmic challenge has become particularly pressing with the emergence of long, high-fidelity (HiFi) reads that have been recently used to generate a semi-manual telomere-to-telomere assembly of the human genome. To enable automated assemblies of long, HiFi reads, we present the La Jolla Assembler (LJA), a fast algorithm using the Bloom filter, sparse de Bruijn graphs and disjointig generation. LJA reduces the error rate in HiFi reads by three orders of magnitude, constructs the de Bruijn graph for large genomes and large k-mer sizes and transforms it into a multiplex de Bruijn graph with varying k-mer sizes. Compared to state-of-the-art assemblers, our algorithm not only achieves five-fold fewer misassemblies but also generates more contiguous assemblies. We demonstrate the utility of LJA via the automated assembly of a human genome that completely assembled six chromosomes. A multiplex de Bruijn graph algorithm allows high-accuracy genome assembly from long, high-fidelity reads.},
  archive      = {J_NBT},
  author       = {Bankevich, Anton and Bzikadze, Andrey V. and Kolmogorov, Mikhail and Antipov, Dmitry and Pevzner, Pavel A.},
  doi          = {10.1038/s41587-022-01220-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1075-1081},
  shortjournal = {Nature Biotech.},
  title        = {Multiplex de bruijn graphs enable genome assembly from long, high-fidelity reads},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CoSpar identifies early cell fate biases from single-cell
transcriptomic and lineage information. <em>NBT</em>, <em>40</em>(7),
1066–1074. (<a
href="https://doi.org/10.1038/s41587-022-01209-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A goal of single-cell genome-wide profiling is to reconstruct dynamic transitions during cell differentiation, disease onset and drug response. Single-cell assays have recently been integrated with lineage tracing, a set of methods that identify cells of common ancestry to establish bona fide dynamic relationships between cell states. These integrated methods have revealed unappreciated cell dynamics, but their analysis faces recurrent challenges arising from noisy, dispersed lineage data. In this study, we developed coherent, sparse optimization (CoSpar) as a robust computational approach to infer cell dynamics from single-cell transcriptomics integrated with lineage tracing. Built on assumptions of coherence and sparsity of transition maps, CoSpar is robust to severe downsampling and dispersion of lineage data, which enables simpler experimental designs and requires less calibration. In datasets representing hematopoiesis, reprogramming and directed differentiation, CoSpar identifies early fate biases not previously detected, predicting transcription factors and receptors implicated in fate choice. Documentation and detailed examples for common experimental designs are available at https://cospar.readthedocs.io/ . A computational algorithm integrates lineage tracing with single-cell RNA sequencing and improves early cell fate prediction.},
  archive      = {J_NBT},
  author       = {Wang, Shou-Wen and Herriges, Michael J. and Hurley, Kilian and Kotton, Darrell N. and Klein, Allon M.},
  doi          = {10.1038/s41587-022-01209-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1066-1074},
  shortjournal = {Nature Biotech.},
  title        = {CoSpar identifies early cell fate biases from single-cell transcriptomic and lineage information},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A comparison of experimental assays and analytical methods
for genome-wide identification of active enhancers. <em>NBT</em>,
<em>40</em>(7), 1056–1065. (<a
href="https://doi.org/10.1038/s41587-022-01211-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mounting evidence supports the idea that transcriptional patterns serve as more specific identifiers of active enhancers than histone marks; however, the optimal strategy to identify active enhancers both experimentally and computationally has not been determined. Here, we compared 13 genome-wide RNA sequencing (RNA-seq) assays in K562 cells and show that nuclear run-on followed by cap-selection assay (GRO/PRO-cap) has advantages in enhancer RNA detection and active enhancer identification. We also introduce a tool, peak identifier for nascent transcript starts (PINTS), to identify active promoters and enhancers genome wide and pinpoint the precise location of 5′ transcription start sites. Finally, we compiled a comprehensive enhancer candidate compendium based on the detected enhancer RNA (eRNA) transcription start sites (TSSs) available in 120 cell and tissue types, which can be accessed at https://pints.yulab.org . With knowledge of the best available assays and pipelines, this large-scale annotation of candidate enhancers will pave the way for selection and characterization of their functions in a time- and labor-efficient manner. Experimental and computational methods to detect enhancer RNAs are benchmarked.},
  archive      = {J_NBT},
  author       = {Yao, Li and Liang, Jin and Ozer, Abdullah and Leung, Alden King-Yung and Lis, John T. and Yu, Haiyuan},
  doi          = {10.1038/s41587-022-01211-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1056-1065},
  shortjournal = {Nature Biotech.},
  title        = {A comparison of experimental assays and analytical methods for genome-wide identification of active enhancers},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Functional, metabolic and transcriptional maturation of
human pancreatic islets derived from stem cells. <em>NBT</em>,
<em>40</em>(7), 1042–1055. (<a
href="https://doi.org/10.1038/s41587-022-01219-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transplantation of pancreatic islet cells derived from human pluripotent stem cells is a promising treatment for diabetes. Despite progress in the generation of stem-cell-derived islets (SC-islets), no detailed characterization of their functional properties has been conducted. Here, we generated functionally mature SC-islets using an optimized protocol and benchmarked them comprehensively against primary adult islets. Biphasic glucose-stimulated insulin secretion developed during in vitro maturation, associated with cytoarchitectural reorganization and the increasing presence of alpha cells. Electrophysiology, signaling and exocytosis of SC-islets were similar to those of adult islets. Glucose-responsive insulin secretion was achieved despite differences in glycolytic and mitochondrial glucose metabolism. Single-cell transcriptomics of SC-islets in vitro and throughout 6 months of engraftment in mice revealed a continuous maturation trajectory culminating in a transcriptional landscape closely resembling that of primary islets. Our thorough evaluation of SC-islet maturation highlights their advanced degree of functionality and supports their use in further efforts to understand and combat diabetes. Pancreatic islets derived from stem cells are benchmarked against primary cells.},
  archive      = {J_NBT},
  author       = {Balboa, Diego and Barsby, Tom and Lithovius, Väinö and Saarimäki-Vire, Jonna and Omar-Hmeadi, Muhmmad and Dyachok, Oleg and Montaser, Hossam and Lund, Per-Eric and Yang, Mingyu and Ibrahim, Hazem and Näätänen, Anna and Chandra, Vikash and Vihinen, Helena and Jokitalo, Eija and Kvist, Jouni and Ustinov, Jarkko and Nieminen, Anni I. and Kuuluvainen, Emilia and Hietakangas, Ville and Katajisto, Pekka and Lau, Joey and Carlsson, Per-Ola and Barg, Sebastian and Tengholm, Anders and Otonkoski, Timo},
  doi          = {10.1038/s41587-022-01219-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1042-1055},
  shortjournal = {Nature Biotech.},
  title        = {Functional, metabolic and transcriptional maturation of human pancreatic islets derived from stem cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accelerated identification of disease-causing variants with
ultra-rapid nanopore genome sequencing. <em>NBT</em>, <em>40</em>(7),
1035–1041. (<a
href="https://doi.org/10.1038/s41587-022-01221-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome sequencing (WGS) can identify variants that cause genetic disease, but the time required for sequencing and analysis has been a barrier to its use in acutely ill patients. In the present study, we develop an approach for ultra-rapid nanopore WGS that combines an optimized sample preparation protocol, distributing sequencing over 48 flow cells, near real-time base calling and alignment, accelerated variant calling and fast variant filtration for efficient manual review. Application to two example clinical cases identified a candidate variant in &amp;lt;8 h from sample preparation to variant identification. We show that this framework provides accurate variant calls and efficient prioritization, and accelerates diagnostic clinical genome sequencing twofold compared with previous approaches. A streamlined sequencing process enables identification of disease-causing variants in the clinic within 8 hours.},
  archive      = {J_NBT},
  author       = {Goenka, Sneha D. and Gorzynski, John E. and Shafin, Kishwar and Fisk, Dianna G. and Pesout, Trevor and Jensen, Tanner D. and Monlong, Jean and Chang, Pi-Chuan and Baid, Gunjan and Bernstein, Jonathan A. and Christle, Jeffrey W. and Dalton, Karen P. and Garalde, Daniel R. and Grove, Megan E. and Guillory, Joseph and Kolesnikov, Alexey and Nattestad, Maria and Ruzhnikov, Maura R. Z. and Samadi, Mehrzad and Sethia, Ankit and Spiteri, Elizabeth and Wright, Christopher J. and Xiong, Katherine and Zhu, Tong and Jain, Miten and Sedlazeck, Fritz J. and Carroll, Andrew and Paten, Benedict and Ashley, Euan A.},
  doi          = {10.1038/s41587-022-01221-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1035-1041},
  shortjournal = {Nature Biotech.},
  title        = {Accelerated identification of disease-causing variants with ultra-rapid nanopore genome sequencing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mitochondrial variant enrichment from high-throughput
single-cell RNA sequencing resolves clonal populations. <em>NBT</em>,
<em>40</em>(7), 1030–1034. (<a
href="https://doi.org/10.1038/s41587-022-01210-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The combination of single-cell transcriptomics with mitochondrial DNA variant detection can be used to establish lineage relationships in primary human cells, but current methods are not scalable to interrogate complex tissues. Here, we combine common 3′ single-cell RNA-sequencing protocols with mitochondrial transcriptome enrichment to increase coverage by more than 50-fold, enabling high-confidence mutation detection. The method successfully identifies skewed immune-cell expansions in primary human clonal hematopoiesis. Clonal dynamics are inferred from mitochondrial variants in primary human cells.},
  archive      = {J_NBT},
  author       = {Miller, Tyler E. and Lareau, Caleb A. and Verga, Julia A. and DePasquale, Erica A. K. and Liu, Vincent and Ssozi, Daniel and Sandor, Katalin and Yin, Yajie and Ludwig, Leif S. and El Farran, Chadi A. and Morgan, Duncan M. and Satpathy, Ansuman T. and Griffin, Gabriel K. and Lane, Andrew A. and Love, J. Christopher and Bernstein, Bradley E. and Sankaran, Vijay G. and van Galen, Peter},
  doi          = {10.1038/s41587-022-01210-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1030-1034},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial variant enrichment from high-throughput single-cell RNA sequencing resolves clonal populations},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fast nanopore sequencing data analysis with SLOW5.
<em>NBT</em>, <em>40</em>(7), 1026–1029. (<a
href="https://doi.org/10.1038/s41587-021-01147-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanopore sequencing depends on the FAST5 file format, which does not allow efficient parallel analysis. Here we introduce SLOW5, an alternative format engineered for efficient parallelization and acceleration of nanopore data analysis. Using the example of DNA methylation profiling of a human genome, analysis runtime is reduced from more than two weeks to approximately 10.5 h on a typical high-performance computer. SLOW5 is approximately 25\% smaller than FAST5 and delivers consistent improvements on different computer architectures. Nanopore sequencing data are rapidly analyzed with parallel data access.},
  archive      = {J_NBT},
  author       = {Gamaarachchi, Hasindu and Samarakoon, Hiruna and Jenner, Sasha P. and Ferguson, James M. and Amos, Timothy G. and Hammond, Jillian M. and Saadat, Hassaan and Smith, Martin A. and Parameswaran, Sri and Deveson, Ira W.},
  doi          = {10.1038/s41587-021-01147-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1026-1029},
  shortjournal = {Nature Biotech.},
  title        = {Fast nanopore sequencing data analysis with SLOW5},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). SignalP 6.0 predicts all five types of signal peptides using
protein language models. <em>NBT</em>, <em>40</em>(7), 1023–1025. (<a
href="https://doi.org/10.1038/s41587-021-01156-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Signal peptides (SPs) are short amino acid sequences that control protein secretion and translocation in all living organisms. SPs can be predicted from sequence data, but existing algorithms are unable to detect all known types of SPs. We introduce SignalP 6.0, a machine learning model that detects all five SP types and is applicable to metagenomic data. A new version of SignalP predicts all types of signal peptides.},
  archive      = {J_NBT},
  author       = {Teufel, Felix and Almagro Armenteros, José Juan and Johansen, Alexander Rosenberg and Gíslason, Magnús Halldór and Pihl, Silas Irby and Tsirigos, Konstantinos D. and Winther, Ole and Brunak, Søren and von Heijne, Gunnar and Nielsen, Henrik},
  doi          = {10.1038/s41587-021-01156-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1023-1025},
  shortjournal = {Nature Biotech.},
  title        = {SignalP 6.0 predicts all five types of signal peptides using protein language models},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Smartphone apps in the COVID-19 pandemic. <em>NBT</em>,
<em>40</em>(7), 1013–1022. (<a
href="https://doi.org/10.1038/s41587-022-01350-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {At the beginning of the COVID-19 pandemic, analog tools such as nasopharyngeal swabs for PCR tests were center stage and the major prevention tactics of masking and physical distancing were a throwback to the 1918 influenza pandemic. Overall, there has been scant regard for digital tools, particularly those based on smartphone apps, which is surprising given the ubiquity of smartphones across the globe. Smartphone apps, given accessibility in the time of physical distancing, were widely used for tracking, tracing and educating the public about COVID-19. Despite limitations, such as concerns around data privacy, data security, digital health illiteracy and structural inequities, there is ample evidence that apps are beneficial for understanding outbreak epidemiology, individual screening and contact tracing. While there were successes and failures in each category, outbreak epidemiology and individual screening were substantially enhanced by the reach of smartphone apps and accessory wearables. Continued use of apps within the digital infrastructure promises to provide an important tool for rigorous investigation of outcomes both in the ongoing outbreak and in future epidemics. An overview of apps for outbreak epidemiology, individual screening and contact tracing during the COVID-19 pandemic.},
  archive      = {J_NBT},
  author       = {Pandit, Jay A. and Radin, Jennifer M. and Quer, Giorgio and Topol, Eric J.},
  doi          = {10.1038/s41587-022-01350-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1013-1022},
  shortjournal = {Nature Biotech.},
  title        = {Smartphone apps in the COVID-19 pandemic},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sustainable bioimaging using a fluorescent protein with
unprecedented photostability. <em>NBT</em>, <em>40</em>(7), 1011–1012.
(<a href="https://doi.org/10.1038/s41587-022-01305-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The jellyfish-derived green fluorescent protein StayGold is bright and hardly fades, contributing to improving spatiotemporal resolution and dramatically extending the observation period. To fully benefit from the rich photon budget, we tried some unusual illumination modalities for sustainable, quantitative live-cell or volumetric imaging.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01305-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1011-1012},
  shortjournal = {Nature Biotech.},
  title        = {Sustainable bioimaging using a fluorescent protein with unprecedented photostability},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Synthetic introns enable highly specific targeting of cancer
cells. <em>NBT</em>, <em>40</em>(7), 1009–1010. (<a
href="https://doi.org/10.1038/s41587-022-01235-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new method uses synthetic introns to express therapeutic proteins selectively in cells bearing cancer-initiating mutations affecting RNA splicing factors, while healthy cells remain unaffected. This approach enabled the eradication of human leukemia, breast cancer and uveal melanoma cells in mouse models and significantly prolonged host survival.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01235-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1009-1010},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic introns enable highly specific targeting of cancer cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sizing up beta cells made from stem cells. <em>NBT</em>,
<em>40</em>(7), 1006–1008. (<a
href="https://doi.org/10.1038/s41587-022-01271-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A thorough study of lab-made pancreatic beta cells reveals their differences from primary cells.},
  archive      = {J_NBT},
  author       = {Nguyen-Ngoc, Kim-Vy and Wortham, Matthew and Sander, Maike},
  doi          = {10.1038/s41587-022-01271-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1006-1008},
  shortjournal = {Nature Biotech.},
  title        = {Sizing up beta cells made from stem cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Immuno-oncology checkpoint inhibitors. <em>NBT</em>,
<em>40</em>(7), 1005. (<a
href="https://doi.org/10.1038/s41587-022-01392-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to checkpoint inhibitors and their use in treating cancers.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01392-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1005},
  shortjournal = {Nature Biotech.},
  title        = {Immuno-oncology checkpoint inhibitors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The COVID-19 vaccine patent race. <em>NBT</em>,
<em>40</em>(7), 1001–1004. (<a
href="https://doi.org/10.1038/s41587-022-01376-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The seemingly simple modular technology for making new mRNA vaccines could make vaccine developers a victim of their own success.},
  archive      = {J_NBT},
  author       = {Storz, Ulrich},
  doi          = {10.1038/s41587-022-01376-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1001-1004},
  shortjournal = {Nature Biotech.},
  title        = {The COVID-19 vaccine patent race},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). PRO-ACTive sharing of clinical data. <em>NBT</em>,
<em>40</em>(7), 999–1000. (<a
href="https://doi.org/10.1038/s41587-022-01395-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zach, Neta and Leitner, Melanie L.},
  doi          = {10.1038/s41587-022-01395-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {999-1000},
  shortjournal = {Nature Biotech.},
  title        = {PRO-ACTive sharing of clinical data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Standardized annotation of translated open reading frames.
<em>NBT</em>, <em>40</em>(7), 994–999. (<a
href="https://doi.org/10.1038/s41587-022-01369-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Mudge, Jonathan M. and Ruiz-Orera, Jorge and Prensner, John R. and Brunet, Marie A. and Calvet, Ferriol and Jungreis, Irwin and Gonzalez, Jose Manuel and Magrane, Michele and Martinez, Thomas F. and Schulz, Jana Felicitas and Yang, Yucheng T. and Albà, M. Mar and Aspden, Julie L. and Baranov, Pavel V. and Bazzini, Ariel A. and Bruford, Elspeth and Martin, Maria Jesus and Calviello, Lorenzo and Carvunis, Anne-Ruxandra and Chen, Jin and Couso, Juan Pablo and Deutsch, Eric W. and Flicek, Paul and Frankish, Adam and Gerstein, Mark and Hubner, Norbert and Ingolia, Nicholas T. and Kellis, Manolis and Menschaert, Gerben and Moritz, Robert L. and Ohler, Uwe and Roucou, Xavier and Saghatelian, Alan and Weissman, Jonathan S. and van Heesch, Sebastiaan},
  doi          = {10.1038/s41587-022-01369-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {994-999},
  shortjournal = {Nature Biotech.},
  title        = {Standardized annotation of translated open reading frames},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(7), 989. (<a
href="https://doi.org/10.1038/s41587-022-01394-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01394-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {989},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). 100,000 genomes — in africa, for africa. <em>NBT</em>,
<em>40</em>(7), 988. (<a
href="https://doi.org/10.1038/s41587-022-01404-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01404-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {988},
  shortjournal = {Nature Biotech.},
  title        = {100,000 genomes — in africa, for africa},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). IL-2 upgrades show promise at ASCO. <em>NBT</em>,
<em>40</em>(7), 986–988. (<a
href="https://doi.org/10.1038/s41587-022-01390-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies pursue souped-up IL-2 candidates, undeterred by Nektar’s setback.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01390-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {986-988},
  shortjournal = {Nature Biotech.},
  title        = {IL-2 upgrades show promise at ASCO},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). PD-1 blockade works alone. <em>NBT</em>, <em>40</em>(7),
986. (<a href="https://doi.org/10.1038/s41587-022-01403-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01403-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {986},
  shortjournal = {Nature Biotech.},
  title        = {PD-1 blockade works alone},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bluebird’s CALD gene therapy poised for approval.
<em>NBT</em>, <em>40</em>(7), 985. (<a
href="https://doi.org/10.1038/s41587-022-01402-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01402-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {985},
  shortjournal = {Nature Biotech.},
  title        = {Bluebird’s CALD gene therapy poised for approval},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). RNA drugs lower lipoprotein(a) and genetically driven
cholesterol. <em>NBT</em>, <em>40</em>(7), 983–985. (<a
href="https://doi.org/10.1038/s41587-022-01396-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are deploying siRNA and antisense oligonucleotides to tackle dangerously high cholesterol driven by genetics, betting that a wider population will benefit.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-022-01396-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {983-985},
  shortjournal = {Nature Biotech.},
  title        = {RNA drugs lower lipoprotein(a) and genetically driven cholesterol},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Revisiting checkpoint blockade. <em>NBT</em>,
<em>40</em>(7), 981. (<a
href="https://doi.org/10.1038/s41587-022-01407-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After over a decade of experience with immune checkpoint inhibitors in oncology, more effort needs to be spent unraveling why some patients respond — and why the majority do not — and integrating knowledge about biomarkers into patient selection in trials.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01407-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {981},
  shortjournal = {Nature Biotech.},
  title        = {Revisiting checkpoint blockade},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Diagnostics to take your breath away. <em>NBT</em>,
<em>40</em>(7), 990–993. (<a
href="https://doi.org/10.1038/s41587-022-01385-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After decades of research and development, devices for detecting infectious agents in breath are finally maturing, with SARS-CoV-2 galvanizing progress. Carrie Arnold reports.},
  archive      = {J_NBT},
  author       = {Arnold, Carrie},
  doi          = {10.1038/s41587-022-01385-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {7},
  pages        = {990-993},
  shortjournal = {Nature Biotech.},
  title        = {Diagnostics to take your breath away},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). People. <em>NBT</em>, <em>40</em>(6), 980. (<a
href="https://doi.org/10.1038/s41587-022-01356-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01356-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {980},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A virtual innovation bootcamp to remotely connect and
empower students to solve COVID-19-related medical problems.
<em>NBT</em>, <em>40</em>(6), 976–979. (<a
href="https://doi.org/10.1038/s41587-022-01352-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An intensive program supplements student education and collaboration while narrowing the disconnect between rising healthcare needs and trainees with the technical expertise to tackle pandemic-related challenges.},
  archive      = {J_NBT},
  author       = {Russo, Mario V. and Appukutty, Abhinav J. and Shah, Aadit P. and Mohan, Harsha K. and Daniel, Andy G. S. and Pack, Andrew and Xie, Richard},
  doi          = {10.1038/s41587-022-01352-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {976-979},
  shortjournal = {Nature Biotech.},
  title        = {A virtual innovation bootcamp to remotely connect and empower students to solve COVID-19-related medical problems},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Cell types of origin of the cell-free
transcriptome. <em>NBT</em>, <em>40</em>(6), 974. (<a
href="https://doi.org/10.1038/s41587-022-01293-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Vorperian, Sevahn K. and Moufarrej, Mira N. and Quake, Stephen R.},
  doi          = {10.1038/s41587-022-01293-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {974},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Cell types of origin of the cell-free transcriptome},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Development of spirulina for the
manufacture and oral delivery of protein therapeutics. <em>NBT</em>,
<em>40</em>(6), 974. (<a
href="https://doi.org/10.1038/s41587-022-01323-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Jester, Benjamin W. and Zhao, Hui and Gewe, Mesfin and Adame, Thomas and Perruzza, Lisa and Bolick, David T. and Agosti, Jan and Khuong, Nhi and Kuestner, Rolf and Gamble, Caitlin and Cruickshank, Kendra and Ferrara, Jeremy and Lim, Rachelle and Paddock, Troy and Brady, Colin and Ertel, Stacey and Zhang, Miaohua and Pollock, Alex and Lee, Jamie and Xiong, Jian and Tasch, Michael and Saveria, Tracy and Doughty, David and Marshall, Jacob and Carrieri, Damian and Goetsch, Lauren and Dang, Jason and Sanjaya, Nathaniel and Fletcher, David and Martinez, Anissa and Kadis, Bryce and Sigmar, Kristjan and Afreen, Esha and Nguyen, Tammy and Randolph, Amanda and Taber, Alexandria and Krzeszowski, Ashley and Robinett, Brittney and Volkin, David B. and Grassi, Fabio and Guerrant, Richard and Takeuchi, Ryo and Finrow, Brian and Behnke, Craig and Roberts, James},
  doi          = {10.1038/s41587-022-01323-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {974},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Development of spirulina for the manufacture and oral delivery of protein therapeutics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Imaging of innate immunity activation in vivo with a
redox-tuned PET reporter. <em>NBT</em>, <em>40</em>(6), 965–973. (<a
href="https://doi.org/10.1038/s41587-021-01169-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-redox-potential reactive oxygen species and reactive nitrogen species (ROS/RNS), generated by NADPH oxidase-2 (NOX2), myeloperoxidase (MPO) and related enzymes, are key effector molecules of innate immunity. High-redox-potential radicals are difficult to distinguish by imaging from less potent ROS/RNS functioning as background biological signaling molecules. Here we present 4-[18F]fluoro-1-naphthol ([18F]4FN), a redox-tuned radiopharmaceutical that selectively binds proteins and cells when oxidized by products of human MPO plus H2O2, but not H2O2 alone, and can be detected using positron emission tomography (PET). Activating HL-60 neutrophil-like human cells with phorbol ester (PMA) caused [18F]4FN retention five-fold over unstimulated cells. An MPO-specific inhibitor (4-ABAH) blocked cellular retention by more than 95\%. [18F]4FN PET/CT imaging discriminated inflammatory foci in vivo in three murine models of activated innate immunity: endotoxin-induced toxic shock, PMA-induced contact dermatitis and lipopolysaccharide-induced ankle arthritis. 4-ABAH and Cybb−/− (Nox2−/−) gene deletion strongly abrogated [18F]4FN retention in vivo. Thus, [18F]4FN shows promise as a robust reporter of innate immunity activation by PET/CT. Inflammation is imaged in mice with a PET reporter of high-energy reactive oxygen and nitrogen species.},
  archive      = {J_NBT},
  author       = {Pisaneschi, Federica and Gammon, Seth T. and Paolillo, Vincenzo and Qureshy, Sarah A. and Piwnica-Worms, David},
  doi          = {10.1038/s41587-021-01169-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {965-973},
  shortjournal = {Nature Biotech.},
  title        = {Imaging of innate immunity activation in vivo with a redox-tuned PET reporter},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Development of spirulina for the manufacture and oral
delivery of protein therapeutics. <em>NBT</em>, <em>40</em>(6), 956–964.
(<a href="https://doi.org/10.1038/s41587-022-01249-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of the edible photosynthetic cyanobacterium Arthrospira platensis (spirulina) as a biomanufacturing platform has been limited by a lack of genetic tools. Here we report genetic engineering methods for stable, high-level expression of bioactive proteins in spirulina, including large-scale, indoor cultivation and downstream processing methods. Following targeted integration of exogenous genes into the spirulina chromosome (chr), encoded protein biopharmaceuticals can represent as much as 15\% of total biomass, require no purification before oral delivery and are stable without refrigeration and protected during gastric transit when encapsulated within dry spirulina. Oral delivery of a spirulina-expressed antibody targeting campylobacter—a major cause of infant mortality in the developing world—prevents disease in mice, and a phase 1 clinical trial demonstrated safety for human administration. Spirulina provides an advantageous system for the manufacture of orally delivered therapeutic proteins by combining the safety of a food-based production host with the accessible genetic manipulation and high productivity of microbial platforms. Spirulina is used to manufacture a therapeutic antibody against campylobacter.},
  archive      = {J_NBT},
  author       = {Jester, Benjamin W. and Zhao, Hui and Gewe, Mesfin and Adame, Thomas and Perruzza, Lisa and Bolick, David T. and Agosti, Jan and Khuong, Nhi and Kuestner, Rolf and Gamble, Caitlin and Cruickshank, Kendra and Ferrara, Jeremy and Lim, Rachelle and Paddock, Troy and Brady, Colin and Ertel, Stacey and Zhang, Miaohua and Pollock, Alex and Lee, Jamie and Xiong, Jian and Tasch, Michael and Saveria, Tracy and Doughty, David and Marshall, Jacob and Carrieri, Damian and Goetsch, Lauren and Dang, Jason and Sanjaya, Nathaniel and Fletcher, David and Martinez, Anissa and Kadis, Bryce and Sigmar, Kristjan and Afreen, Esha and Nguyen, Tammy and Randolph, Amanda and Taber, Alexandria and Krzeszowski, Ashley and Robinett, Brittney and Volkin, David B. and Grassi, Fabio and Guerrant, Richard and Takeuchi, Ryo and Finrow, Brian and Behnke, Craig and Roberts, James},
  doi          = {10.1038/s41587-022-01249-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {956-964},
  shortjournal = {Nature Biotech.},
  title        = {Development of spirulina for the manufacture and oral delivery of protein therapeutics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Engineered circular ADAR-recruiting RNAs increase the
efficiency and fidelity of RNA editing in vitro and in vivo.
<em>NBT</em>, <em>40</em>(6), 946–955. (<a
href="https://doi.org/10.1038/s41587-021-01180-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for programmed RNA editing using endogenous ADAR enzymes and engineered ADAR-recruiting RNAs (arRNAs) suffer from low efficiency and bystander off-target editing. Here, we describe LEAPER 2.0, an updated version of LEAPER that uses covalently closed circular arRNAs, termed circ-arRNAs. We demonstrate on average ~3.1-fold higher editing efficiency than their linear counterparts when expressed in cells or delivered as in vitro-transcribed circular RNA oligonucleotides. To lower off-target editing we deleted pairings of uridines with off-target adenosines, which almost completely eliminated bystander off-target adenosine editing. Engineered circ-arRNAs enhanced the efficiency and fidelity of editing endogenous CTNNB1 and mutant TP53 transcripts in cell culture. Delivery of circ-arRNAs using adeno-associated virus in a mouse model of Hurler syndrome corrected the pathogenic point mutation and restored α-L-iduronidase catalytic activity, lowering glycosaminoglycan accumulation in the liver. LEAPER 2.0 provides a new design of arRNA that enables more precise, efficient RNA editing with broad applicability for therapy and basic research. Circular ADAR-recruiting RNAs improve RNA editing in vitro and in a mouse disease model.},
  archive      = {J_NBT},
  author       = {Yi, Zongyi and Qu, Liang and Tang, Huixian and Liu, Zhiheng and Liu, Ying and Tian, Feng and Wang, Chunhui and Zhang, Xiaoxue and Feng, Ziqi and Yu, Ying and Yuan, Pengfei and Yi, Zexuan and Zhao, Yanxia and Wei, Wensheng},
  doi          = {10.1038/s41587-021-01180-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {946-955},
  shortjournal = {Nature Biotech.},
  title        = {Engineered circular ADAR-recruiting RNAs increase the efficiency and fidelity of RNA editing in vitro and in vivo},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficient in vitro and in vivo RNA editing via recruitment
of endogenous ADARs using circular guide RNAs. <em>NBT</em>,
<em>40</em>(6), 938–945. (<a
href="https://doi.org/10.1038/s41587-021-01171-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recruiting endogenous adenosine deaminases using exogenous guide RNAs to edit cellular RNAs is a promising therapeutic strategy, but editing efficiency and durability remain low using current guide RNA designs. In this study, we engineered circular ADAR-recruiting guide RNAs (cadRNAs) to enable more efficient programmable adenosine-to-inosine RNA editing without requiring co-delivery of any exogenous proteins. Using these cadRNAs, we observed robust and durable RNA editing across multiple sites and cell lines, in both untranslated and coding regions of RNAs, and high transcriptome-wide specificity. Additionally, we increased transcript-level specificity for the target adenosine by incorporating interspersed loops in the antisense domains, reducing bystander editing. In vivo delivery of cadRNAs via adeno-associated viruses enabled 53\% RNA editing of the mPCSK9 transcript in C57BL/6J mice livers and 12\% UAG-to-UGG RNA correction of the amber nonsense mutation in the IDUA-W392X mouse model of mucopolysaccharidosis type I-Hurler syndrome. cadRNAs enable efficient programmable RNA editing in vivo with diverse protein modulation and gene therapeutic applications. Circular guide RNAs boost the efficiency of RNA editing with endogenous ADARs.},
  archive      = {J_NBT},
  author       = {Katrekar, Dhruva and Yen, James and Xiang, Yichen and Saha, Anushka and Meluzzi, Dario and Savva, Yiannis and Mali, Prashant},
  doi          = {10.1038/s41587-021-01171-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {938-945},
  shortjournal = {Nature Biotech.},
  title        = {Efficient in vitro and in vivo RNA editing via recruitment of endogenous ADARs using circular guide RNAs},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Using deep learning to annotate the protein universe.
<em>NBT</em>, <em>40</em>(6), 932–937. (<a
href="https://doi.org/10.1038/s41587-021-01179-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the relationship between amino acid sequence and protein function is a long-standing challenge with far-reaching scientific and translational implications. State-of-the-art alignment-based techniques cannot predict function for one-third of microbial protein sequences, hampering our ability to exploit data from diverse organisms. Here, we train deep learning models to accurately predict functional annotations for unaligned amino acid sequences across rigorous benchmark assessments built from the 17,929 families of the protein families database Pfam. The models infer known patterns of evolutionary substitutions and learn representations that accurately cluster sequences from unseen families. Combining deep models with existing methods significantly improves remote homology detection, suggesting that the deep models learn complementary information. This approach extends the coverage of Pfam by &amp;gt;9.5\%, exceeding additions made over the last decade, and predicts function for 360 human reference proteome proteins with no previous Pfam annotation. These results suggest that deep learning models will be a core component of future protein annotation tools. A deep learning model predicts protein functional annotations for unaligned amino acid sequences.},
  archive      = {J_NBT},
  author       = {Bileschi, Maxwell L. and Belanger, David and Bryant, Drew H. and Sanderson, Theo and Carter, Brandon and Sculley, D. and Bateman, Alex and DePristo, Mark A. and Colwell, Lucy J.},
  doi          = {10.1038/s41587-021-01179-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {932-937},
  shortjournal = {Nature Biotech.},
  title        = {Using deep learning to annotate the protein universe},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identification of antimicrobial peptides from the human gut
microbiome using deep learning. <em>NBT</em>, <em>40</em>(6), 921–931.
(<a href="https://doi.org/10.1038/s41587-022-01226-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human gut microbiome encodes a large variety of antimicrobial peptides (AMPs), but the short lengths of AMPs pose a challenge for computational prediction. Here we combined multiple natural language processing neural network models, including LSTM, Attention and BERT, to form a unified pipeline for candidate AMP identification from human gut microbiome data. Of 2,349 sequences identified as candidate AMPs, 216 were chemically synthesized, with 181 showing antimicrobial activity (a positive rate of &amp;gt;83\%). Most of these peptides have less than 40\% sequence homology to AMPs in the training set. Further characterization of the 11 most potent AMPs showed high efficacy against antibiotic-resistant, Gram-negative pathogens and demonstrated significant efficacy in lowering bacterial load by more than tenfold against a mouse model of bacterial lung infection. Our study showcases the potential of machine learning approaches for mining functional peptides from metagenome data and accelerating the discovery of promising AMP candidate molecules for in-depth investigations. Antimicrobial peptides are identified from metagenomics data using deep learning.},
  archive      = {J_NBT},
  author       = {Ma, Yue and Guo, Zhengyan and Xia, Binbin and Zhang, Yuwei and Liu, Xiaolin and Yu, Ying and Tang, Na and Tong, Xiaomei and Wang, Min and Ye, Xin and Feng, Jie and Chen, Yihua and Wang, Jun},
  doi          = {10.1038/s41587-022-01226-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {921-931},
  shortjournal = {Nature Biotech.},
  title        = {Identification of antimicrobial peptides from the human gut microbiome using deep learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A humanized mouse model of chronic COVID-19. <em>NBT</em>,
<em>40</em>(6), 906–920. (<a
href="https://doi.org/10.1038/s41587-021-01155-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coronavirus disease 2019 (COVID-19) is an infectious disease that can present as an uncontrolled, hyperactive immune response, causing severe immunological injury. Existing rodent models do not recapitulate the sustained immunopathology of patients with severe disease. Here we describe a humanized mouse model of COVID-19 that uses adeno-associated virus to deliver human ACE2 to the lungs of humanized MISTRG6 mice. This model recapitulates innate and adaptive human immune responses to severe acute respiratory syndrome coronavirus 2 infection up to 28 days after infection, with key features of chronic COVID-19, including weight loss, persistent viral RNA, lung pathology with fibrosis, a human inflammatory macrophage response, a persistent interferon-stimulated gene signature and T cell lymphopenia. We used this model to study two therapeutics on immunopathology, patient-derived antibodies and steroids and found that the same inflammatory macrophages crucial to containing early infection later drove immunopathology. This model will enable evaluation of COVID-19 disease mechanisms and treatments. A mouse model of chronic COVID-19 facilitates the study of disease mechanisms and therapies.},
  archive      = {J_NBT},
  author       = {Sefik, Esen and Israelow, Benjamin and Mirza, Haris and Zhao, Jun and Qu, Rihao and Kaffe, Eleanna and Song, Eric and Halene, Stephanie and Meffre, Eric and Kluger, Yuval and Nussenzweig, Michel and Wilen, Craig B. and Iwasaki, Akiko and Flavell, Richard A.},
  doi          = {10.1038/s41587-021-01155-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {906-920},
  shortjournal = {Nature Biotech.},
  title        = {A humanized mouse model of chronic COVID-19},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Massively parallel phenotyping of coding variants in cancer
with perturb-seq. <em>NBT</em>, <em>40</em>(6), 896–905. (<a
href="https://doi.org/10.1038/s41587-021-01160-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome sequencing studies have identified millions of somatic variants in cancer, but it remains challenging to predict the phenotypic impact of most. Experimental approaches to distinguish impactful variants often use phenotypic assays that report on predefined gene-specific functional effects in bulk cell populations. Here, we develop an approach to functionally assess variant impact in single cells by pooled Perturb-seq. We measured the impact of 200 TP53 and KRAS variants on RNA profiles in over 300,000 single lung cancer cells, and used the profiles to categorize variants into phenotypic subsets to distinguish gain-of-function, loss-of-function and dominant negative variants, which we validated by comparison with orthogonal assays. We discovered that KRAS variants did not merely fit into discrete functional categories, but spanned a continuum of gain-of-function phenotypes, and that their functional impact could not have been predicted solely by their frequency in patient cohorts. Our work provides a scalable, gene-agnostic method for coding variant impact phenotyping, with potential applications in multiple disease settings. The functional impact of somatic mutations in cancer genes is determined by pooled Perturb-seq.},
  archive      = {J_NBT},
  author       = {Ursu, Oana and Neal, James T. and Shea, Emily and Thakore, Pratiksha I. and Jerby-Arnon, Livnat and Nguyen, Lan and Dionne, Danielle and Diaz, Celeste and Bauman, Julia and Mosaad, Mariam Mounir and Fagre, Christian and Lo, April and McSharry, Maria and Giacomelli, Andrew O. and Ly, Seav Huong and Rozenblatt-Rosen, Orit and Hahn, William C. and Aguirre, Andrew J. and Berger, Alice H. and Regev, Aviv and Boehm, Jesse S.},
  doi          = {10.1038/s41587-021-01160-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {896-905},
  shortjournal = {Nature Biotech.},
  title        = {Massively parallel phenotyping of coding variants in cancer with perturb-seq},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Saturation variant interpretation using CRISPR prime
editing. <em>NBT</em>, <em>40</em>(6), 885–895. (<a
href="https://doi.org/10.1038/s41587-021-01201-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-throughput functional characterization of genetic variants in their endogenous locus has so far been possible only with methods that rely on homology-directed repair, which are limited by low editing efficiencies. Here, we adapted CRISPR prime editing for high-throughput variant classification and combined it with a strategy that allows for haploidization of any locus, which simplifies variant interpretation. We demonstrate the utility of saturation prime editing (SPE) by applying it to the NPC intracellular cholesterol transporter 1 gene (NPC1), mutations in which cause the lysosomal storage disorder Niemann–Pick disease type C. Our data suggest that NPC1 is very sensitive to genetic perturbation, with 410 of 706 assayed missense mutations being classified as deleterious, and that the derived function score of variants is reflective of diverse molecular defects. We further applied our approach to the BRCA2 gene, demonstrating that SPE is translatable to other genes with an appropriate cellular assay. In sum, we show that SPE allows for efficient, accurate functional characterization of genetic variants. Genetic variants of uncertain significance are characterized with a prime editing method.},
  archive      = {J_NBT},
  author       = {Erwood, Steven and Bily, Teija M. I. and Lequyer, Jason and Yan, Joyce and Gulati, Nitya and Brewer, Reid A. and Zhou, Liangchi and Pelletier, Laurence and Ivakine, Evgueni A. and Cohn, Ronald D.},
  doi          = {10.1038/s41587-021-01201-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {885-895},
  shortjournal = {Nature Biotech.},
  title        = {Saturation variant interpretation using CRISPR prime editing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High-throughput functional evaluation of human
cancer-associated mutations using base editors. <em>NBT</em>,
<em>40</em>(6), 874–884. (<a
href="https://doi.org/10.1038/s41587-022-01276-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive phenotypic characterization of the many mutations found in cancer tissues is one of the biggest challenges in cancer genomics. In this study, we evaluated the functional effects of 29,060 cancer-related transition mutations that result in protein variants on the survival and proliferation of non-tumorigenic lung cells using cytosine and adenine base editors and single guide RNA (sgRNA) libraries. By monitoring base editing efficiencies and outcomes using surrogate target sequences paired with sgRNA-encoding sequences on the lentiviral delivery construct, we identified sgRNAs that induced a single primary protein variant per sgRNA, enabling linking those mutations to the cellular phenotypes caused by base editing. The functions of the vast majority of the protein variants (28,458 variants, 98\%) were classified as neutral or likely neutral; only 18 (0.06\%) and 157 (0.5\%) variants caused outgrowing and likely outgrowing phenotypes, respectively. We expect that our approach can be extended to more variants of unknown significance and other tumor types. Cancer-associated variants of unknown significance are identified using base editing.},
  archive      = {J_NBT},
  author       = {Kim, Younggwang and Lee, Seungho and Cho, Soohyuk and Park, Jinman and Chae, Dongwoo and Park, Taeyoung and Minna, John D. and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-022-01276-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {874-884},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput functional evaluation of human cancer-associated mutations using base editors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Base editing sensor libraries for high-throughput
engineering and functional analysis of cancer-associated single
nucleotide variants. <em>NBT</em>, <em>40</em>(6), 862–873. (<a
href="https://doi.org/10.1038/s41587-021-01172-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Base editing can be applied to characterize single nucleotide variants of unknown function, yet defining effective combinations of single guide RNAs (sgRNAs) and base editors remains challenging. Here, we describe modular base-editing-activity ‘sensors’ that link sgRNAs and cognate target sites in cis and use them to systematically measure the editing efficiency and precision of thousands of sgRNAs paired with functionally distinct base editors. By quantifying sensor editing across &amp;gt;200,000 editor-sgRNA combinations, we provide a comprehensive resource of sgRNAs for introducing and interrogating cancer-associated single nucleotide variants in multiple model systems. We demonstrate that sensor-validated tools streamline production of in vivo cancer models and that integrating sensor modules in pooled sgRNA libraries can aid interpretation of high-throughput base editing screens. Using this approach, we identify several previously uncharacterized mutant TP53 alleles as drivers of cancer cell proliferation and in vivo tumor development. We anticipate that the framework described here will facilitate the functional interrogation of cancer variants in cell and animal models. Improved base editing libraries enable high-throughput functional analysis of single-nucleotide variants in cancer.},
  archive      = {J_NBT},
  author       = {Sánchez-Rivera, Francisco J. and Diaz, Bianca J. and Kastenhuber, Edward R. and Schmidt, Henri and Katti, Alyna and Kennedy, Margaret and Tem, Vincent and Ho, Yu-Jui and Leibold, Josef and Paffenholz, Stella V. and Barriga, Francisco M. and Chu, Kevan and Goswami, Sukanya and Wuest, Alexandra N. and Simon, Janelle M. and Tsanov, Kaloyan M. and Chakravarty, Debyani and Zhang, Hongxin and Leslie, Christina S. and Lowe, Scott W. and Dow, Lukas E.},
  doi          = {10.1038/s41587-021-01172-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {862-873},
  shortjournal = {Nature Biotech.},
  title        = {Base editing sensor libraries for high-throughput engineering and functional analysis of cancer-associated single nucleotide variants},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Cell types of origin of the cell-free transcriptome.
<em>NBT</em>, <em>40</em>(6), 855–861. (<a
href="https://doi.org/10.1038/s41587-021-01188-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell-free RNA from liquid biopsies can be analyzed to determine disease tissue of origin. We extend this concept to identify cell types of origin using the Tabula Sapiens transcriptomic cell atlas as well as individual tissue transcriptomic cell atlases in combination with the Human Protein Atlas RNA consensus dataset. We define cell type signature scores, which allow the inference of cell types that contribute to cell-free RNA for a variety of diseases. Cell types affected by various diseases are inferred from cell-free RNA.},
  archive      = {J_NBT},
  author       = {Vorperian, Sevahn K. and Moufarrej, Mira N. and Quake, Stephen R.},
  doi          = {10.1038/s41587-021-01188-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {855-861},
  shortjournal = {Nature Biotech.},
  title        = {Cell types of origin of the cell-free transcriptome},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The clinical progress of mRNA vaccines and immunotherapies.
<em>NBT</em>, <em>40</em>(6), 840–854. (<a
href="https://doi.org/10.1038/s41587-022-01294-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergency use authorizations (EUAs) of two mRNA-based severe acute respiratory syndrome coronavirus (SARS-CoV)-2 vaccines approximately 11 months after publication of the viral sequence highlights the transformative potential of this nucleic acid technology. Most clinical applications of mRNA to date have focused on vaccines for infectious disease and cancer for which low doses, low protein expression and local delivery can be effective because of the inherent immunostimulatory properties of some mRNA species and formulations. In addition, work on mRNA-encoded protein or cellular immunotherapies has also begun, for which minimal immune stimulation, high protein expression in target cells and tissues, and the need for repeated administration have led to additional manufacturing and formulation challenges for clinical translation. Building on this momentum, the past year has seen clinical progress with second-generation coronavirus disease 2019 (COVID-19) vaccines, Omicron-specific boosters and vaccines against seasonal influenza, Epstein–Barr virus, human immunodeficiency virus (HIV) and cancer. Here we review the clinical progress of mRNA therapy as well as provide an overview and future outlook of the transformative technology behind these mRNA-based drugs. Anderson and colleagues discuss the progress and challenges of using mRNA for vaccines and immunotherapy.},
  archive      = {J_NBT},
  author       = {Barbier, Ann J. and Jiang, Allen Yujie and Zhang, Peng and Wooster, Richard and Anderson, Daniel G.},
  doi          = {10.1038/s41587-022-01294-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {840-854},
  shortjournal = {Nature Biotech.},
  title        = {The clinical progress of mRNA vaccines and immunotherapies},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identification of antimicrobial peptides from the human gut
microbiome using deep learning. <em>NBT</em>, <em>40</em>(6), 838–839.
(<a href="https://doi.org/10.1038/s41587-022-01230-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using a combination of metagenomic big data and deep learning tools, small proteins that inhibit pathogens — and could be further developed into novel antibiotics — are mined en masse. Such methods could greatly improve the throughput of drug discovery and translational usage of the microbiome.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01230-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {838-839},
  shortjournal = {Nature Biotech.},
  title        = {Identification of antimicrobial peptides from the human gut microbiome using deep learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Forum: Phillips-cremins and pombo. <em>NBT</em>,
<em>40</em>(6), 837. (<a
href="https://doi.org/10.1038/s41587-022-01365-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01365-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {837},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Phillips-cremins and pombo},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pathogenic or benign? <em>NBT</em>, <em>40</em>(6), 834–836.
(<a href="https://doi.org/10.1038/s41587-022-01333-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR base- and prime-editing pooled screens reveal the function of genetic variants at unprecedented resolution.},
  archive      = {J_NBT},
  author       = {Du, Peter P. and Liu, Katherine and Bassik, Michael C. and Hess, Gaelen T.},
  doi          = {10.1038/s41587-022-01333-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {834-836},
  shortjournal = {Nature Biotech.},
  title        = {Pathogenic or benign?},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antibiotic discovery with machine learning. <em>NBT</em>,
<em>40</em>(6), 833–834. (<a
href="https://doi.org/10.1038/s41587-022-01327-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence finds candidate peptide antibiotics in the human gut microbiome.},
  archive      = {J_NBT},
  author       = {de la Fuente-Nunez, Cesar},
  doi          = {10.1038/s41587-022-01327-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {833-834},
  shortjournal = {Nature Biotech.},
  title        = {Antibiotic discovery with machine learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CRISPR technology. <em>NBT</em>, <em>40</em>(6), 832. (<a
href="https://doi.org/10.1038/s41587-022-01359-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to CRISPR methods and compositions for gene editing and therapeutic use.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01359-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {832},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR technology},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Vascepa patents under legal scrutiny. <em>NBT</em>,
<em>40</em>(6), 829–831. (<a
href="https://doi.org/10.1038/s41587-022-01345-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Even if Amarin prevails in its patent infringement lawsuit, its key patents on Vascepa are likely to be ruled invalid in ongoing litigation, opening the way for generic formulations.},
  archive      = {J_NBT},
  author       = {Curfman, Gregory and Cole, Justin},
  doi          = {10.1038/s41587-022-01345-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {829-831},
  shortjournal = {Nature Biotech.},
  title        = {Vascepa patents under legal scrutiny},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The worldview national ranking of health biotech sectors.
<em>NBT</em>, <em>40</em>(6), 821–828. (<a
href="https://doi.org/10.1038/s41587-022-01349-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A survey of national R&amp;amp;D-driven health biotech sectors ranks Switzerland, Sweden and the United States as leading centers for R&amp;amp;D-driven biotech. John Hodgson and Deanna Schreiber-Gregory report.},
  archive      = {J_NBT},
  author       = {Hodgson, John and Schreiber-Gregory, Deanna},
  doi          = {10.1038/s41587-022-01349-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {821-828},
  shortjournal = {Nature Biotech.},
  title        = {The worldview national ranking of health biotech sectors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The GA4GH phenopacket schema defines a computable
representation of clinical data. <em>NBT</em>, <em>40</em>(6), 817–820.
(<a href="https://doi.org/10.1038/s41587-022-01357-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Jacobsen, Julius O. B. and Baudis, Michael and Baynam, Gareth S. and Beckmann, Jacques S. and Beltran, Sergi and Buske, Orion J. and Callahan, Tiffany J. and Chute, Christopher G. and Courtot, Mélanie and Danis, Daniel and Elemento, Olivier and Essenwanger, Andrea and Freimuth, Robert R. and Gargano, Michael A. and Groza, Tudor and Hamosh, Ada and Harris, Nomi L. and Kaliyaperumal, Rajaram and Lloyd, Kevin C. Kent and Khalifa, Aly and Krawitz, Peter M. and Köhler, Sebastian and Laraway, Brian J. and Lehväslaiho, Heikki and Matalonga, Leslie and McMurry, Julie A. and Metke-Jimenez, Alejandro and Mungall, Christopher J. and Munoz-Torres, Monica C. and Ogishima, Soichi and Papakonstantinou, Anastasios and Piscia, Davide and Pontikos, Nikolas and Queralt-Rosinach, Núria and Roos, Marco and Sass, Julian and Schofield, Paul N. and Seelow, Dominik and Siapos, Anastasios and Smedley, Damian and Smith, Lindsay D. and Steinhaus, Robin and Sundaramurthi, Jagadish Chandrabose and Swietlik, Emilia M. and Thun, Sylvia and Vasilevsky, Nicole A. and Wagner, Alex H. and Warner, Jeremy L. and Weiland, Claus and Haendel, Melissa A. and Robinson, Peter N.},
  doi          = {10.1038/s41587-022-01357-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {817-820},
  shortjournal = {Nature Biotech.},
  title        = {The GA4GH phenopacket schema defines a computable representation of clinical data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An open invitation to the understudied proteins initiative.
<em>NBT</em>, <em>40</em>(6), 815–817. (<a
href="https://doi.org/10.1038/s41587-022-01316-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kustatscher, Georg and Collins, Tom and Gingras, Anne-Claude and Guo, Tiannan and Hermjakob, Henning and Ideker, Trey and Lilley, Kathryn S. and Lundberg, Emma and Marcotte, Edward M. and Ralser, Markus and Rappsilber, Juri},
  doi          = {10.1038/s41587-022-01316-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {815-817},
  shortjournal = {Nature Biotech.},
  title        = {An open invitation to the understudied proteins initiative},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Biotech patenting 2021. <em>NBT</em>, <em>40</em>(6),
813–814. (<a href="https://doi.org/10.1038/s41587-022-01338-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-022-01338-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {813-814},
  shortjournal = {Nature Biotech.},
  title        = {Biotech patenting 2021},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Egg without the chicken. <em>NBT</em>, <em>40</em>(6), 812.
(<a href="https://doi.org/10.1038/s41587-022-01366-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-022-01366-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {812},
  shortjournal = {Nature Biotech.},
  title        = {Egg without the chicken},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). AI’s plastic recycling. <em>NBT</em>, <em>40</em>(6), 811.
(<a href="https://doi.org/10.1038/s41587-022-01372-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01372-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {811},
  shortjournal = {Nature Biotech.},
  title        = {AI’s plastic recycling},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Topical herpesvirus gene therapy enters final lap.
<em>NBT</em>, <em>40</em>(6), 809–811. (<a
href="https://doi.org/10.1038/s41587-022-01362-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first herpesvirus replacement gene therapy for epidermolysis bullosa, a blistering skin condition, approaches the clinic.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-022-01362-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {809-811},
  shortjournal = {Nature Biotech.},
  title        = {Topical herpesvirus gene therapy enters final lap},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). FDA okays first cardiac myosin inhibitor. <em>NBT</em>,
<em>40</em>(6), 809. (<a
href="https://doi.org/10.1038/s41587-022-01374-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01374-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {809},
  shortjournal = {Nature Biotech.},
  title        = {FDA okays first cardiac myosin inhibitor},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(6), 808. (<a
href="https://doi.org/10.1038/s41587-022-01360-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01360-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {808},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Caribou’s first CRISPR CAR-t impresses. <em>NBT</em>,
<em>40</em>(6), 807. (<a
href="https://doi.org/10.1038/s41587-022-01371-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01371-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {807},
  shortjournal = {Nature Biotech.},
  title        = {Caribou’s first CRISPR CAR-T impresses},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Unconventional γδ t cells “the new black” in cancer
therapy. <em>NBT</em>, <em>40</em>(6), 805–808. (<a
href="https://doi.org/10.1038/s41587-022-01363-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new era of cancer immunotherapy beckons as γδ T cell trials enter final stage.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01363-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {805-808},
  shortjournal = {Nature Biotech.},
  title        = {Unconventional γδ t cells ‘the new black’ in cancer therapy},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Knock-in on CRISPR’s door. <em>NBT</em>, <em>40</em>(6),
803. (<a href="https://doi.org/10.1038/s41587-022-01375-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interest is growing in genome-editing tools that can insert large chunks of DNA into the genome — and avoid the double-strand breaks associated with CRISPR–Cas9 genotoxicity.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01375-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {803},
  shortjournal = {Nature Biotech.},
  title        = {Knock-in on CRISPR’s door},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). People. <em>NBT</em>, <em>40</em>(5), 802. (<a
href="https://doi.org/10.1038/s41587-022-01315-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01315-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {802},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). First-quarter biotech job picture. <em>NBT</em>,
<em>40</em>(5), 801. (<a
href="https://doi.org/10.1038/s41587-022-01313-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-022-01313-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {801},
  shortjournal = {Nature Biotech.},
  title        = {First-quarter biotech job picture},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Engineered promoters enable constant gene
expression at any copy number in bacteria. <em>NBT</em>, <em>40</em>(5),
799. (<a href="https://doi.org/10.1038/s41587-022-01300-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Segall-Shapiro, Thomas H. and Sontag, Eduardo D. and Voigt, Christopher A.},
  doi          = {10.1038/s41587-022-01300-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {799},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Engineered promoters enable constant gene expression at any copy number in bacteria},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Publisher correction: Diabetes cell therapies take evasive
action. <em>NBT</em>, <em>40</em>(5), 799. (<a
href="https://doi.org/10.1038/s41587-022-01322-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01322-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {799},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Diabetes cell therapies take evasive action},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Publisher correction: Bio-leather gears up to wow fashion
industry. <em>NBT</em>, <em>40</em>(5), 799. (<a
href="https://doi.org/10.1038/s41587-022-01329-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-022-01329-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {799},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Bio-leather gears up to wow fashion industry},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A fluorescent sensor for spatiotemporally resolved imaging
of endocannabinoid dynamics in vivo. <em>NBT</em>, <em>40</em>(5),
787–798. (<a href="https://doi.org/10.1038/s41587-021-01074-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Endocannabinoids (eCBs) are retrograde neuromodulators with important functions in a wide range of physiological processes, but their in vivo dynamics remain largely uncharacterized. Here we developed a genetically encoded eCB sensor called GRABeCB2.0. GRABeCB2.0 consists of a circular-permutated EGFP and the human CB1 cannabinoid receptor, providing cell membrane trafficking, second-resolution kinetics with high specificity for eCBs, and shows a robust fluorescence response at physiological eCB concentrations. Using GRABeCB2.0, we monitored evoked and spontaneous changes in eCB dynamics in cultured neurons and acute brain slices. We observed spontaneous compartmentalized eCB transients in cultured neurons and eCB transients from single axonal boutons in acute brain slices, suggesting constrained, localized eCB signaling. When GRABeCB2.0 was expressed in the mouse brain, we observed foot shock-elicited and running-triggered eCB signaling in the basolateral amygdala and hippocampus, respectively. In a mouse model of epilepsy, we observed a spreading wave of eCB release that followed a Ca2+ wave through the hippocampus. GRABeCB2.0 is a robust probe for eCB release in vivo. A genetically encoded sensor reveals the dynamics of endocannabinoid signaling.},
  archive      = {J_NBT},
  author       = {Dong, Ao and He, Kaikai and Dudok, Barna and Farrell, Jordan S. and Guan, Wuqiang and Liput, Daniel J. and Puhl, Henry L. and Cai, Ruyi and Wang, Huan and Duan, Jiali and Albarran, Eddy and Ding, Jun and Lovinger, David M. and Li, Bo and Soltesz, Ivan and Li, Yulong},
  doi          = {10.1038/s41587-021-01074-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {787-798},
  shortjournal = {Nature Biotech.},
  title        = {A fluorescent sensor for spatiotemporally resolved imaging of endocannabinoid dynamics in vivo},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Optogenetic control of RNA function and metabolism using
engineered light-switchable RNA-binding proteins. <em>NBT</em>,
<em>40</em>(5), 779–786. (<a
href="https://doi.org/10.1038/s41587-021-01112-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-binding proteins (RBPs) play an essential role in regulating the function of RNAs in a cellular context, but our ability to control RBP activity in time and space is limited. Here, we describe the engineering of LicV, a photoswitchable RBP that binds to a specific RNA sequence in response to blue light irradiation. When fused to various RNA effectors, LicV allows for optogenetic control of RNA localization, splicing, translation and stability in cell culture. Furthermore, LicV-assisted CRISPR–Cas systems allow for efficient and tunable photoswitchable regulation of transcription and genomic locus labeling. These data demonstrate that the photoswitchable RBP LicV can serve as a programmable scaffold for the spatiotemporal control of synthetic RNA effectors. Photoswitchable RNA-binding proteins enable the spatiotemporal control of RNA function.},
  archive      = {J_NBT},
  author       = {Liu, Renmei and Yang, Jing and Yao, Jing and Zhao, Zhou and He, Wei and Su, Ni and Zhang, Zeyi and Zhang, Chenxia and Zhang, Zhuo and Cai, Haibo and Zhu, Linyong and Zhao, Yuzheng and Quan, Shu and Chen, Xianjun and Yang, Yi},
  doi          = {10.1038/s41587-021-01112-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {779-786},
  shortjournal = {Nature Biotech.},
  title        = {Optogenetic control of RNA function and metabolism using engineered light-switchable RNA-binding proteins},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conformation-locking antibodies for the discovery and
characterization of KRAS inhibitors. <em>NBT</em>, <em>40</em>(5),
769–778. (<a href="https://doi.org/10.1038/s41587-021-01126-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Small molecules that stabilize inactive protein conformations are an underutilized strategy for drugging dynamic or otherwise intractable proteins. To facilitate the discovery and characterization of such inhibitors, we created a screening platform to identify conformation-locking antibodies for molecular probes (CLAMPs) that distinguish and induce rare protein conformational states. Applying the approach to KRAS, we discovered CLAMPs that recognize the open conformation of KRASG12C stabilized by covalent inhibitors. One CLAMP enables the visualization of KRASG12C covalent modification in vivo and can be used to investigate response heterogeneity to KRASG12C inhibitors in patient tumors. A second CLAMP enhances the affinity of weak ligands binding to the KRASG12C switch II region (SWII) by stabilizing a specific conformation of KRASG12C, thereby enabling the discovery of such ligands that could serve as leads for the development of drugs in a high-throughput screen. We show that combining the complementary properties of antibodies and small molecules facilitates the study and drugging of dynamic proteins. Antibodies that lock KRAS mutants in inactive conformations facilitate drug discovery.},
  archive      = {J_NBT},
  author       = {Davies, Christopher W. and Oh, Angela J. and Mroue, Rana and Steffek, Micah and Bruning, John M. and Xiao, Yang and Feng, Siyu and Jayakar, Sangeeta and Chan, Emily and Arumugam, Vidhyalakshmi and Uribe, Sean Carlo and Drummond, Jake and Frommlet, Alexandra and Lu, Cheng and Franke, Yvonne and Merchant, Mark and Koeppen, Hartmut and Quinn, John G. and Malhotra, Sushant and Do, Steve and Gazzard, Lewis and Purkey, Hans E. and Rudolph, Joachim and Mulvihill, Melinda M. and Koerber, James T. and Wang, Weiru and Evangelista, Marie},
  doi          = {10.1038/s41587-021-01126-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {769-778},
  shortjournal = {Nature Biotech.},
  title        = {Conformation-locking antibodies for the discovery and characterization of KRAS inhibitors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CLUSTER guide RNAs enable precise and efficient RNA editing
with endogenous ADAR enzymes in vivo. <em>NBT</em>, <em>40</em>(5),
759–768. (<a href="https://doi.org/10.1038/s41587-021-01105-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA base editing represents a promising alternative to genome editing. Recent approaches harness the endogenous RNA-editing enzyme adenosine deaminase acting on RNA (ADAR) to circumvent problems caused by ectopic expression of engineered editing enzymes, but suffer from sequence restriction, lack of efficiency and bystander editing. Here we present in silico-optimized CLUSTER guide RNAs that bind their target messenger RNAs in a multivalent fashion, achieve editing with high precision and efficiency and enable targeting of sequences that were not accessible using previous gRNA designs. CLUSTER gRNAs can be genetically encoded and delivered using viruses, and are active in a wide range of cell lines. In cell culture, CLUSTER gRNAs achieve on-target editing of endogenous transcripts with yields of up to 45\% without bystander editing. In vivo, CLUSTER gRNAs delivered to mouse liver by hydrodynamic tail vein injection edited reporter constructs at rates of up to 10\%. The CLUSTER approach opens avenues for drug development in the field of RNA base editing. Optimized guide RNAs improve RNA editing with endogenous enzymes.},
  archive      = {J_NBT},
  author       = {Reautschnig, Philipp and Wahn, Nicolai and Wettengel, Jacqueline and Schulz, Annika E. and Latifi, Ngadhnjim and Vogel, Paul and Kang, Tae-Won and Pfeiffer, Laura S. and Zarges, Christine and Naumann, Ulrike and Zender, Lars and Li, Jin Billy and Stafforst, Thorsten},
  doi          = {10.1038/s41587-021-01105-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {759-768},
  shortjournal = {Nature Biotech.},
  title        = {CLUSTER guide RNAs enable precise and efficient RNA editing with endogenous ADAR enzymes in vivo},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Engineering the amoeba dictyostelium discoideum for
biosynthesis of a cannabinoid precursor and other polyketides.
<em>NBT</em>, <em>40</em>(5), 751–758. (<a
href="https://doi.org/10.1038/s41587-021-01143-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aromatic polyketides are natural polyphenolic compounds with a broad spectrum of pharmacological activities. Production of those metabolites in the model organisms Escherichia coli and Saccharomyces cerevisiae has been limited by the extensive cellular engineering needed for the coordinated biosynthesis of polyketides and their precursors. In contrast, the amoeba Dictyostelium discoideum is a native producer of secondary metabolites and harbors a wide, but largely unexplored, repertoire of genes for the biosynthesis of polyketides and terpenoids. Here we present D. discoideum as an advantageous chassis for the production of aromatic polyketides. By expressing its native and cognate plant polyketide synthase genes in D. discoideum, we demonstrate production of phlorocaprophenone, methyl-olivetol, resveratrol and olivetolic acid (OA), which is the central intermediate in the biosynthesis of cannabinoids. To facilitate OA synthesis, we further engineered an amoeba/plant inter-kingdom hybrid enzyme that produced OA from primary metabolites in two enzymatic steps, providing a shortcut in a synthetic cannabinoid pathway using the D. discoideum host system. Polyketides, including a cannabinoid precursor, are produced in a slime mold.},
  archive      = {J_NBT},
  author       = {Reimer, Christin and Kufs, Johann E. and Rautschek, Julia and Regestein, Lars and Valiante, Vito and Hillmann, Falk},
  doi          = {10.1038/s41587-021-01143-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {751-758},
  shortjournal = {Nature Biotech.},
  title        = {Engineering the amoeba dictyostelium discoideum for biosynthesis of a cannabinoid precursor and other polyketides},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Partitioning RNAs by length improves transcriptome
reconstruction from short-read RNA-seq data. <em>NBT</em>,
<em>40</em>(5), 741–750. (<a
href="https://doi.org/10.1038/s41587-021-01136-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accuracy of methods for assembling transcripts from short-read RNA sequencing data is limited by the lack of long-range information. Here we introduce Ladder-seq, an approach that separates transcripts according to their lengths before sequencing and uses the additional information to improve the quantification and assembly of transcripts. Using simulated data, we show that a kallisto algorithm extended to process Ladder-seq data quantifies transcripts of complex genes with substantially higher accuracy than conventional kallisto. For reference-based assembly, a tailored scheme based on the StringTie2 algorithm reconstructs a single transcript with 30.8\% higher precision than its conventional counterpart and is more than 30\% more sensitive for complex genes. For de novo assembly, a similar scheme based on the Trinity algorithm correctly assembles 78\% more transcripts than conventional Trinity while improving precision by 78\%. In experimental data, Ladder-seq reveals 40\% more genes harboring isoform switches compared to conventional RNA sequencing and unveils widespread changes in isoform usage upon m6A depletion by Mettl14 knockout. The quality of RNA sequencing transcriptomes is improved when mRNAs are separated by length.},
  archive      = {J_NBT},
  author       = {Ringeling, Francisca Rojas and Chakraborty, Shounak and Vissers, Caroline and Reiman, Derek and Patel, Akshay M. and Lee, Ki-Heon and Hong, Ari and Park, Chan-Woo and Reska, Tim and Gagneur, Julien and Chang, Hyeshik and Spletter, Maria L. and Yoon, Ki-Jun and Ming, Guo-li and Song, Hongjun and Canzar, Stefan},
  doi          = {10.1038/s41587-021-01136-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {741-750},
  shortjournal = {Nature Biotech.},
  title        = {Partitioning RNAs by length improves transcriptome reconstruction from short-read RNA-seq data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Programmable deletion, replacement, integration and
inversion of large DNA sequences with twin prime editing. <em>NBT</em>,
<em>40</em>(5), 731–740. (<a
href="https://doi.org/10.1038/s41587-021-01133-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The targeted deletion, replacement, integration or inversion of genomic sequences could be used to study or treat human genetic diseases, but existing methods typically require double-strand DNA breaks (DSBs) that lead to undesired consequences, including uncontrolled indel mixtures and chromosomal abnormalities. Here we describe twin prime editing (twinPE), a DSB-independent method that uses a prime editor protein and two prime editing guide RNAs (pegRNAs) for the programmable replacement or excision of DNA sequences at endogenous human genomic sites. The two pegRNAs template the synthesis of complementary DNA flaps on opposing strands of genomic DNA, which replace the endogenous DNA sequence between the prime-editor-induced nick sites. When combined with a site-specific serine recombinase, twinPE enabled targeted integration of gene-sized DNA plasmids (&amp;gt;5,000 bp) and targeted sequence inversions of 40 kb in human cells. TwinPE expands the capabilities of precision gene editing and might synergize with other tools for the correction or complementation of large or complex human pathogenic alleles. Prime editing of large DNA sequences is achieved with two pegRNAs and site-specific recombinases.},
  archive      = {J_NBT},
  author       = {Anzalone, Andrew V. and Gao, Xin D. and Podracky, Christopher J. and Nelson, Andrew T. and Koblan, Luke W. and Raguram, Aditya and Levy, Jonathan M. and Mercer, Jaron A. M. and Liu, David R.},
  doi          = {10.1038/s41587-021-01133-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {731-740},
  shortjournal = {Nature Biotech.},
  title        = {Programmable deletion, replacement, integration and inversion of large DNA sequences with twin prime editing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fluctuating methylation clocks for cell lineage tracing at
high temporal resolution in human tissues. <em>NBT</em>, <em>40</em>(5),
720–730. (<a href="https://doi.org/10.1038/s41587-021-01109-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular clocks that record cell ancestry mutate too slowly to measure the short-timescale dynamics of cell renewal in adult tissues. Here, we show that fluctuating DNA methylation marks can be used as clocks in cells where ongoing methylation and demethylation cause repeated ‘flip–flops’ between methylated and unmethylated states. We identify endogenous fluctuating CpG (fCpG) sites using standard methylation arrays and develop a mathematical model to quantitatively measure human adult stem cell dynamics from these data. Small intestinal crypts were inferred to contain slightly more stem cells than the colon, with slower stem cell replacement in the small intestine. Germline APC mutation increased the number of replacements per crypt. In blood, we measured rapid expansion of acute leukemia and slower growth of chronic disease. Thus, the patterns of human somatic cell birth and death are measurable with fluctuating methylation clocks (FMCs). Lineage tracing of human stem cells is achieved by measuring fluctuating DNA methylation.},
  archive      = {J_NBT},
  author       = {Gabbutt, Calum and Schenck, Ryan O. and Weisenberger, Daniel J. and Kimberley, Christopher and Berner, Alison and Househam, Jacob and Lakatos, Eszter and Robertson-Tessi, Mark and Martin, Isabel and Patel, Roshani and Clark, Susan K. and Latchford, Andrew and Barnes, Chris P. and Leedham, Simon J. and Anderson, Alexander R. A. and Graham, Trevor A. and Shibata, Darryl},
  doi          = {10.1038/s41587-021-01109-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {720-730},
  shortjournal = {Nature Biotech.},
  title        = {Fluctuating methylation clocks for cell lineage tracing at high temporal resolution in human tissues},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Generating lineage-resolved, complete metagenome-assembled
genomes from complex microbial communities. <em>NBT</em>,
<em>40</em>(5), 711–719. (<a
href="https://doi.org/10.1038/s41587-021-01130-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbial communities might include distinct lineages of closely related organisms that complicate metagenomic assembly and prevent the generation of complete metagenome-assembled genomes (MAGs). Here we show that deep sequencing using long (HiFi) reads combined with Hi-C binning can address this challenge even for complex microbial communities. Using existing methods, we sequenced the sheep fecal metagenome and identified 428 MAGs with more than 90\% completeness, including 44 MAGs in single circular contigs. To resolve closely related strains (lineages), we developed MAGPhase, which separates lineages of related organisms by discriminating variant haplotypes across hundreds of kilobases of genomic sequence. MAGPhase identified 220 lineage-resolved MAGs in our dataset. The ability to resolve closely related microbes in complex microbial communities improves the identification of biosynthetic gene clusters and the precision of assigning mobile genetic elements to host genomes. We identified 1,400 complete and 350 partial biosynthetic gene clusters, most of which are novel, as well as 424 (298) potential host–viral (host–plasmid) associations using Hi-C data. Metagenome sequencing can now distinguish closely related microbes using long reads and haplotype phasing.},
  archive      = {J_NBT},
  author       = {Bickhart, Derek M. and Kolmogorov, Mikhail and Tseng, Elizabeth and Portik, Daniel M. and Korobeynikov, Anton and Tolstoganov, Ivan and Uritskiy, Gherman and Liachko, Ivan and Sullivan, Shawn T. and Shin, Sung Bong and Zorea, Alvah and Andreu, Victòria Pascal and Panke-Buisse, Kevin and Medema, Marnix H. and Mizrahi, Itzhak and Pevzner, Pavel A. and Smith, Timothy P. L.},
  doi          = {10.1038/s41587-021-01130-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {711-719},
  shortjournal = {Nature Biotech.},
  title        = {Generating lineage-resolved, complete metagenome-assembled genomes from complex microbial communities},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ScJoint integrates atlas-scale single-cell RNA-seq and
ATAC-seq data with transfer learning. <em>NBT</em>, <em>40</em>(5),
703–710. (<a href="https://doi.org/10.1038/s41587-021-01161-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell multiomics data continues to grow at an unprecedented pace. Although several methods have demonstrated promising results in integrating several data modalities from the same tissue, the complexity and scale of data compositions present in cell atlases still pose a challenge. Here, we present scJoint, a transfer learning method to integrate atlas-scale, heterogeneous collections of scRNA-seq and scATAC-seq data. scJoint leverages information from annotated scRNA-seq data in a semisupervised framework and uses a neural network to simultaneously train labeled and unlabeled data, allowing label transfer and joint visualization in an integrative framework. Using atlas data as well as multimodal datasets generated with ASAP-seq and CITE-seq, we demonstrate that scJoint is computationally efficient and consistently achieves substantially higher cell-type label accuracy than existing methods while providing meaningful joint visualizations. Thus, scJoint overcomes the heterogeneity of different data modalities to enable a more comprehensive understanding of cellular phenotypes. Integration of data from single-cell RNA-seq and ATAC-seq is achieved with transfer learning.},
  archive      = {J_NBT},
  author       = {Lin, Yingxin and Wu, Tung-Yu and Wan, Sheng and Yang, Jean Y. H. and Wong, Wing H. and Wang, Y. X. Rachel},
  doi          = {10.1038/s41587-021-01161-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {703-710},
  shortjournal = {Nature Biotech.},
  title        = {ScJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A knowledge graph to interpret clinical proteomics data.
<em>NBT</em>, <em>40</em>(5), 692–702. (<a
href="https://doi.org/10.1038/s41587-021-01145-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Implementing precision medicine hinges on the integration of omics data, such as proteomics, into the clinical decision-making process, but the quantity and diversity of biomedical data, and the spread of clinically relevant knowledge across multiple biomedical databases and publications, pose a challenge to data integration. Here we present the Clinical Knowledge Graph (CKG), an open-source platform currently comprising close to 20 million nodes and 220 million relationships that represent relevant experimental data, public databases and literature. The graph structure provides a flexible data model that is easily extendable to new nodes and relationships as new databases become available. The CKG incorporates statistical and machine learning algorithms that accelerate the analysis and interpretation of typical proteomics workflows. Using a set of proof-of-concept biomarker studies, we show how the CKG might augment and enrich proteomics data and help inform clinical decision-making. A knowledge graph platform integrates proteomics with other omics data and biomedical databases.},
  archive      = {J_NBT},
  author       = {Santos, Alberto and Colaço, Ana R. and Nielsen, Annelaura B. and Niu, Lili and Strauss, Maximilian and Geyer, Philipp E. and Coscia, Fabian and Albrechtsen, Nicolai J. Wewer and Mundt, Filip and Jensen, Lars Juhl and Mann, Matthias},
  doi          = {10.1038/s41587-021-01145-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {692-702},
  shortjournal = {Nature Biotech.},
  title        = {A knowledge graph to interpret clinical proteomics data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multiscale PHATE identifies multimodal signatures of
COVID-19. <em>NBT</em>, <em>40</em>(5), 681–691. (<a
href="https://doi.org/10.1038/s41587-021-01186-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As the biomedical community produces datasets that are increasingly complex and high dimensional, there is a need for more sophisticated computational tools to extract biological insights. We present Multiscale PHATE, a method that sweeps through all levels of data granularity to learn abstracted biological features directly predictive of disease outcome. Built on a coarse-graining process called diffusion condensation, Multiscale PHATE learns a data topology that can be analyzed at coarse resolutions for high-level summarizations of data and at fine resolutions for detailed representations of subsets. We apply Multiscale PHATE to a coronavirus disease 2019 (COVID-19) dataset with 54 million cells from 168 hospitalized patients and find that patients who die show CD16hiCD66blo neutrophil and IFN-γ+ granzyme B+ Th17 cell responses. We also show that population groupings from Multiscale PHATE directly fed into a classifier predict disease outcome more accurately than naive featurizations of the data. Multiscale PHATE is broadly generalizable to different data types, including flow cytometry, single-cell RNA sequencing (scRNA-seq), single-cell sequencing assay for transposase-accessible chromatin (scATAC-seq), and clinical variables. Disease signatures in high-dimensional biomedical data are detected with a visualization algorithm.},
  archive      = {J_NBT},
  author       = {Kuchroo, Manik and Huang, Jessie and Wong, Patrick and Grenier, Jean-Christophe and Shung, Dennis and Tong, Alexander and Lucas, Carolina and Klein, Jon and Burkhardt, Daniel B. and Gigante, Scott and Godavarthi, Abhinav and Rieck, Bastian and Israelow, Benjamin and Simonov, Michael and Mao, Tianyang and Oh, Ji Eun and Silva, Julio and Takahashi, Takehiro and Odio, Camila D. and Casanovas-Massana, Arnau and Fournier, John and Farhadian, Shelli and Dela Cruz, Charles S. and Ko, Albert I. and Hirn, Matthew J. and Wilson, F. Perry and Hussin, Julie G. and Wolf, Guy and Iwasaki, Akiko and Krishnaswamy, Smita},
  doi          = {10.1038/s41587-021-01186-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {681-691},
  shortjournal = {Nature Biotech.},
  title        = {Multiscale PHATE identifies multimodal signatures of COVID-19},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Curated variation benchmarks for challenging medically
relevant autosomal genes. <em>NBT</em>, <em>40</em>(5), 672–680. (<a
href="https://doi.org/10.1038/s41587-021-01158-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The repetitive nature and complexity of some medically relevant genes poses a challenge for their accurate analysis in a clinical setting. The Genome in a Bottle Consortium has provided variant benchmark sets, but these exclude nearly 400 medically relevant genes due to their repetitiveness or polymorphic complexity. Here, we characterize 273 of these 395 challenging autosomal genes using a haplotype-resolved whole-genome assembly. This curated benchmark reports over 17,000 single-nucleotide variations, 3,600 insertions and deletions and 200 structural variations each for human genome reference GRCh37 and GRCh38 across HG002. We show that false duplications in either GRCh37 or GRCh38 result in reference-specific, missed variants for short- and long-read technologies in medically relevant genes, including CBS, CRYAA and KCNE1. When masking these false duplications, variant recall can improve from 8\% to 100\%. Forming benchmarks from a haplotype-resolved whole-genome assembly may become a prototype for future benchmarks covering the whole genome. Variant detection in problematic genes is facilitated with a curated benchmark.},
  archive      = {J_NBT},
  author       = {Wagner, Justin and Olson, Nathan D. and Harris, Lindsay and McDaniel, Jennifer and Cheng, Haoyu and Fungtammasan, Arkarachai and Hwang, Yih-Chii and Gupta, Richa and Wenger, Aaron M. and Rowell, William J. and Khan, Ziad M. and Farek, Jesse and Zhu, Yiming and Pisupati, Aishwarya and Mahmoud, Medhat and Xiao, Chunlin and Yoo, Byunggil and Sahraeian, Sayed Mohammad Ebrahim and Miller, Danny E. and Jáspez, David and Lorenzo-Salazar, José M. and Muñoz-Barrera, Adrián and Rubio-Rodríguez, Luis A. and Flores, Carlos and Narzisi, Giuseppe and Evani, Uday Shanker and Clarke, Wayne E. and Lee, Joyce and Mason, Christopher E. and Lincoln, Stephen E. and Miga, Karen H. and Ebbert, Mark T. W. and Shumate, Alaina and Li, Heng and Chin, Chen-Shan and Zook, Justin M. and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41587-021-01158-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {672-680},
  shortjournal = {Nature Biotech.},
  title        = {Curated variation benchmarks for challenging medically relevant autosomal genes},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cell2location maps fine-grained cell types in spatial
transcriptomics. <em>NBT</em>, <em>40</em>(5), 661–671. (<a
href="https://doi.org/10.1038/s41587-021-01139-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomic technologies promise to resolve cellular wiring diagrams of tissues in health and disease, but comprehensive mapping of cell types in situ remains a challenge. Here we present сell2location, a Bayesian model that can resolve fine-grained cell types in spatial transcriptomic data and create comprehensive cellular maps of diverse tissues. Cell2location accounts for technical sources of variation and borrows statistical strength across locations, thereby enabling the integration of single-cell and spatial transcriptomics with higher sensitivity and resolution than existing tools. We assessed cell2location in three different tissues and show improved mapping of fine-grained cell types. In the mouse brain, we discovered fine regional astrocyte subtypes across the thalamus and hypothalamus. In the human lymph node, we spatially mapped a rare pre-germinal center B cell population. In the human gut, we resolved fine immune cell populations in lymphoid follicles. Collectively, our results present сell2location as a versatile analysis tool for mapping tissue architectures in a comprehensive manner. A Bayesian model maps the location of cell types in tissues with higher sensitivity.},
  archive      = {J_NBT},
  author       = {Kleshchevnikov, Vitalii and Shmatko, Artem and Dann, Emma and Aivazidis, Alexander and King, Hamish W. and Li, Tong and Elmentaite, Rasa and Lomakin, Artem and Kedlian, Veronika and Gayoso, Adam and Jain, Mika Sarkin and Park, Jun Sung and Ramona, Lauma and Tuck, Elizabeth and Arutyunyan, Anna and Vento-Tormo, Roser and Gerstung, Moritz and James, Louisa and Stegle, Oliver and Bayraktar, Omer Ali},
  doi          = {10.1038/s41587-021-01139-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {661-671},
  shortjournal = {Nature Biotech.},
  title        = {Cell2location maps fine-grained cell types in spatial transcriptomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sensitive identification of neoantigens and cognate TCRs in
human solid tumors. <em>NBT</em>, <em>40</em>(5), 656–660. (<a
href="https://doi.org/10.1038/s41587-021-01072-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of patient-specific tumor antigens is complicated by the low frequency of T cells specific for each tumor antigen. Here we describe NeoScreen, a method that enables the sensitive identification of rare tumor (neo)antigens and of cognate T cell receptors (TCRs) expressed by tumor-infiltrating lymphocytes. T cells transduced with tumor antigen-specific TCRs identified by NeoScreen mediate regression of established tumors in patient-derived xenograft mice. NeoScreen helps identify rare tumor antigens for personalized cancer vaccines and T cell therapies.},
  archive      = {J_NBT},
  author       = {Arnaud, Marion and Chiffelle, Johanna and Genolet, Raphael and Navarro Rodrigo, Blanca and Perez, Marta A. S. and Huber, Florian and Magnin, Morgane and Nguyen-Ngoc, Tu and Guillaume, Philippe and Baumgaertner, Petra and Chong, Chloe and Stevenson, Brian J. and Gfeller, David and Irving, Melita and Speiser, Daniel E. and Schmidt, Julien and Zoete, Vincent and Kandalaft, Lana E. and Bassani-Sternberg, Michal and Bobisse, Sara and Coukos, George and Harari, Alexandre},
  doi          = {10.1038/s41587-021-01072-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {656-660},
  shortjournal = {Nature Biotech.},
  title        = {Sensitive identification of neoantigens and cognate TCRs in human solid tumors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chimeric antigen receptors. <em>NBT</em>, <em>40</em>(5),
654. (<a href="https://doi.org/10.1038/s41587-022-01320-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods of making and using chimeric antigen receptor (CAR)-T cells to treat cancer and other diseases.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01320-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {654},
  shortjournal = {Nature Biotech.},
  title        = {Chimeric antigen receptors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Relieving patent-eligibility barriers in biotech with a
preparation or treatment method. <em>NBT</em>, <em>40</em>(5), 651–653.
(<a href="https://doi.org/10.1038/s41587-022-01301-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of decisions related to patent eligibility by the US Court of Appeals for the Federal Circuit indicates a way to avoid rejection or invalidation of eligibility.},
  archive      = {J_NBT},
  author       = {Wang, Jir-You and Tsai, Yi-Fang and Wang, Tien-Hsiang and Wang, Shyh-Jen},
  doi          = {10.1038/s41587-022-01301-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {651-653},
  shortjournal = {Nature Biotech.},
  title        = {Relieving patent-eligibility barriers in biotech with a preparation or treatment method},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reflections on alnylam. <em>NBT</em>, <em>40</em>(5),
641–650. (<a href="https://doi.org/10.1038/s41587-022-01304-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recollections from an extraordinary 19-year journey guiding a tiny startup with an unproven therapeutic modality into a mature drug company with marketed products and &amp;gt;1,600 employees in nearly 20 countries provide lessons for those seeking to create the culture and values that are core to biotech success.},
  archive      = {J_NBT},
  author       = {Maraganore, John},
  doi          = {10.1038/s41587-022-01304-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {641-650},
  shortjournal = {Nature Biotech.},
  title        = {Reflections on alnylam},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). 1Q22 — blood in the water. <em>NBT</em>, <em>40</em>(5),
633–635. (<a href="https://doi.org/10.1038/s41587-022-01314-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01314-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {633-635},
  shortjournal = {Nature Biotech.},
  title        = {1Q22 — blood in the water},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Drug pipeline 1Q22 — a raft of new modalities … and
clinical blowups. <em>NBT</em>, <em>40</em>(5), 631–632. (<a
href="https://doi.org/10.1038/s41587-022-01310-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01310-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {631-632},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 1Q22 — a raft of new modalities … and clinical blowups},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(5), 630. (<a
href="https://doi.org/10.1038/s41587-022-01317-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01317-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {630},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Climate-friendly couture. <em>NBT</em>, <em>40</em>(5),
629. (<a href="https://doi.org/10.1038/s41587-022-01319-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-022-01319-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {629},
  shortjournal = {Nature Biotech.},
  title        = {Climate-friendly couture},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fungal genomes scoured for drugs. <em>NBT</em>,
<em>40</em>(5), 628. (<a
href="https://doi.org/10.1038/s41587-022-01330-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01330-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {628},
  shortjournal = {Nature Biotech.},
  title        = {Fungal genomes scoured for drugs},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Startup tackles oncogene-rich ecDNA. <em>NBT</em>,
<em>40</em>(5), 627. (<a
href="https://doi.org/10.1038/s41587-022-01332-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01332-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {627},
  shortjournal = {Nature Biotech.},
  title        = {Startup tackles oncogene-rich ecDNA},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Cosmetics: When biotech is better than nature.
<em>NBT</em>, <em>40</em>(5), 626–628. (<a
href="https://doi.org/10.1038/s41587-022-01318-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biotech companies find success with sustainably sourced ingredients for the beauty market},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-022-01318-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {626-628},
  shortjournal = {Nature Biotech.},
  title        = {Cosmetics: When biotech is better than nature},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). FDA approves anti-LAG3 checkpoint. <em>NBT</em>,
<em>40</em>(5), 625. (<a
href="https://doi.org/10.1038/s41587-022-01331-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01331-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {625},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves anti-LAG3 checkpoint},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Base editing marches on the clinic. <em>NBT</em>,
<em>40</em>(5), 623–625. (<a
href="https://doi.org/10.1038/s41587-022-01326-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This year could see the first trials of therapies employing base editing that introduce precise single-nucleotide changes into patient genomes.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-022-01326-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {623-625},
  shortjournal = {Nature Biotech.},
  title        = {Base editing marches on the clinic},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Gene therapy at the crossroads. <em>NBT</em>,
<em>40</em>(5), 621. (<a
href="https://doi.org/10.1038/s41587-022-01346-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two upcoming regulatory decisions represent a tipping point for commercial gene therapy, with implications for work on existing viral vectors and the pursuit of new ones.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01346-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {621},
  shortjournal = {Nature Biotech.},
  title        = {Gene therapy at the crossroads},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Forum: CRISPR screening roundtable with stegmaier and
doench. <em>NBT</em>, <em>40</em>(5), 655. (<a
href="https://doi.org/10.1038/s41587-022-01303-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01303-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {5},
  pages        = {655},
  shortjournal = {Nature Biotech.},
  title        = {Forum: CRISPR screening roundtable with stegmaier and doench},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Baby’s first genome. <em>NBT</em>, <em>40</em>(5), 636–640.
(<a href="https://doi.org/10.1038/s41587-022-01306-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome sequencing may be the fastest way to diagnose rare complex diseases, but should it be incorporated into healthy newborn screening?},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-022-01306-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {5},
  pages        = {636-640},
  shortjournal = {Nature Biotech.},
  title        = {Baby’s first genome},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). People. <em>NBT</em>, <em>40</em>(4), 620. (<a
href="https://doi.org/10.1038/s41587-022-01283-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01283-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {620},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Helping others enhances graduate student wellness and mental
health. <em>NBT</em>, <em>40</em>(4), 618–619. (<a
href="https://doi.org/10.1038/s41587-022-01275-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Actively engaging students in community-based educational outreach activities improves their mental health and will hopefully promote their retention and success in graduate school and beyond.},
  archive      = {J_NBT},
  author       = {Hermanstyne, Tracey O. and Johnson, Lauren and Wylie, Kristine M. and Skeath, James B.},
  doi          = {10.1038/s41587-022-01275-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {618-619},
  shortjournal = {Nature Biotech.},
  title        = {Helping others enhances graduate student wellness and mental health},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sparse deconvolution improves the resolution of live-cell
super-resolution fluorescence microscopy. <em>NBT</em>, <em>40</em>(4),
606–617. (<a href="https://doi.org/10.1038/s41587-021-01092-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A main determinant of the spatial resolution of live-cell super-resolution (SR) microscopes is the maximum photon flux that can be collected. To further increase the effective resolution for a given photon flux, we take advantage of a priori knowledge about the sparsity and continuity of biological structures to develop a deconvolution algorithm that increases the resolution of SR microscopes nearly twofold. Our method, sparse structured illumination microscopy (Sparse-SIM), achieves ~60-nm resolution at a frame rate of up to 564 Hz, allowing it to resolve intricate structures, including small vesicular fusion pores, ring-shaped nuclear pores formed by nucleoporins and relative movements of inner and outer mitochondrial membranes in live cells. Sparse deconvolution can also be used to increase the three-dimensional resolution of spinning-disc confocal-based SIM, even at low signal-to-noise ratios, which allows four-color, three-dimensional live-cell SR imaging at ~90-nm resolution. Overall, sparse deconvolution will be useful to increase the spatiotemporal resolution of live-cell fluorescence microscopy. The resolution of fluorescence microscopy is increased by incorporating prior information into deconvolution algorithms.},
  archive      = {J_NBT},
  author       = {Zhao, Weisong and Zhao, Shiqun and Li, Liuju and Huang, Xiaoshuai and Xing, Shijia and Zhang, Yulin and Qiu, Guohua and Han, Zhenqian and Shang, Yingxu and Sun, De-en and Shan, Chunyan and Wu, Runlong and Gu, Lusheng and Zhang, Shuwen and Chen, Riwang and Xiao, Jian and Mo, Yanquan and Wang, Jianyong and Ji, Wei and Chen, Xing and Ding, Baoquan and Liu, Yanmei and Mao, Heng and Song, Bao-Liang and Tan, Jiubin and Liu, Jian and Li, Haoyu and Chen, Liangyi},
  doi          = {10.1038/s41587-021-01092-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {606-617},
  shortjournal = {Nature Biotech.},
  title        = {Sparse deconvolution improves the resolution of live-cell super-resolution fluorescence microscopy},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetically encoded photo-switchable molecular sensors for
optoacoustic and super-resolution imaging. <em>NBT</em>, <em>40</em>(4),
598–605. (<a href="https://doi.org/10.1038/s41587-021-01100-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reversibly photo-switchable proteins are essential for many super-resolution fluorescence microscopic and optoacoustic imaging methods. However, they have yet to be used as sensors that measure the distribution of specific analytes at the nanoscale or in the tissues of live animals. Here we constructed the prototype of a photo-switchable Ca2+ sensor based on GCaMP5G that can be switched with 405/488-nm light and describe its molecular mechanisms at the structural level, including the importance of the interaction of the core barrel structure of the fluorescent protein with the Ca2+ receptor moiety. We demonstrate super-resolution imaging of Ca2+ concentration in cultured cells and optoacoustic Ca2+ imaging in implanted tumor cells in mice under controlled Ca2+ conditions. Finally, we show the generalizability of the concept by constructing examples of photo-switching maltose and dopamine sensors based on periplasmatic binding protein and G-protein-coupled receptor-based sensors. Calcium and other analytes can be imaged at super-resolution and in vivo with photo-switchable sensors.},
  archive      = {J_NBT},
  author       = {Mishra, Kanuj and Fuenzalida-Werner, Juan Pablo and Pennacchietti, Francesca and Janowski, Robert and Chmyrov, Andriy and Huang, Yuanhui and Zakian, Christian and Klemm, Uwe and Testa, Ilaria and Niessing, Dierk and Ntziachristos, Vasilis and Stiel, Andre C.},
  doi          = {10.1038/s41587-021-01100-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {598-605},
  shortjournal = {Nature Biotech.},
  title        = {Genetically encoded photo-switchable molecular sensors for optoacoustic and super-resolution imaging},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Inferring gene expression from cell-free DNA fragmentation
profiles. <em>NBT</em>, <em>40</em>(4), 585–597. (<a
href="https://doi.org/10.1038/s41587-022-01222-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Profiling of circulating tumor DNA (ctDNA) in the bloodstream shows promise for noninvasive cancer detection. Chromatin fragmentation features have previously been explored to infer gene expression profiles from cell-free DNA (cfDNA), but current fragmentomic methods require high concentrations of tumor-derived DNA and provide limited resolution. Here we describe promoter fragmentation entropy as an epigenomic cfDNA feature that predicts RNA expression levels at individual genes. We developed ‘epigenetic expression inference from cell-free DNA-sequencing’ (EPIC-seq), a method that uses targeted sequencing of promoters of genes of interest. Profiling 329 blood samples from 201 patients with cancer and 87 healthy adults, we demonstrate classification of subtypes of lung carcinoma and diffuse large B cell lymphoma. Applying EPIC-seq to serial blood samples from patients treated with PD-(L)1 immune-checkpoint inhibitors, we show that gene expression profiles inferred by EPIC-seq are correlated with clinical response. Our results indicate that EPIC-seq could enable noninvasive, high-throughput tissue-of-origin characterization with diagnostic, prognostic and therapeutic potential. EPIC-seq predicts expression of individual genes from cell-free DNA.},
  archive      = {J_NBT},
  author       = {Esfahani, Mohammad Shahrokh and Hamilton, Emily G. and Mehrmohamadi, Mahya and Nabet, Barzin Y. and Alig, Stefan K. and King, Daniel A. and Steen, Chloé B. and Macaulay, Charles W. and Schultz, Andre and Nesselbush, Monica C. and Soo, Joanne and Schroers-Martin, Joseph G. and Chen, Binbin and Binkley, Michael S. and Stehr, Henning and Chabon, Jacob J. and Sworder, Brian J. and Hui, Angela B-Y and Frank, Matthew J. and Moding, Everett J. and Liu, Chih Long and Newman, Aaron M. and Isbell, James M. and Rudin, Charles M. and Li, Bob T. and Kurtz, David M. and Diehn, Maximilian and Alizadeh, Ash A.},
  doi          = {10.1038/s41587-022-01222-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {585-597},
  shortjournal = {Nature Biotech.},
  title        = {Inferring gene expression from cell-free DNA fragmentation profiles},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Personalized phosphoproteomics identifies functional
signaling. <em>NBT</em>, <em>40</em>(4), 576–584. (<a
href="https://doi.org/10.1038/s41587-021-01099-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein phosphorylation dynamically integrates environmental and cellular information to control biological processes. Identifying functional phosphorylation amongst the thousands of phosphosites regulated by a perturbation at a global scale is a major challenge. Here we introduce ‘personalized phosphoproteomics’, a combination of experimental and computational analyses to link signaling with biological function by utilizing human phenotypic variance. We measure individual subject phosphoproteome responses to interventions with corresponding phenotypes measured in parallel. Applying this approach to investigate how exercise potentiates insulin signaling in human skeletal muscle, we identify both known and previously unidentified phosphosites on proteins involved in glucose metabolism. This includes a cooperative relationship between mTOR and AMPK whereby the former directly phosphorylates the latter on S377, for which we find a role in metabolic regulation. These results establish personalized phosphoproteomics as a general approach for investigating the signal transduction underlying complex biology. Functionally relevant phosphorylation sites are detected by integrating phosphoproteomic and phenotypic data.},
  archive      = {J_NBT},
  author       = {Needham, Elise J. and Hingst, Janne R. and Parker, Benjamin L. and Morrison, Kaitlin R. and Yang, Guang and Onslev, Johan and Kristensen, Jonas M. and Højlund, Kurt and Ling, Naomi X. Y. and Oakhill, Jonathan S. and Richter, Erik A. and Kiens, Bente and Petersen, Janni and Pehmøller, Christian and James, David E. and Wojtaszewski, Jørgen F. P. and Humphrey, Sean J.},
  doi          = {10.1038/s41587-021-01099-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {576-584},
  shortjournal = {Nature Biotech.},
  title        = {Personalized phosphoproteomics identifies functional signaling},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep distributed computing to reconstruct extremely large
lineage trees. <em>NBT</em>, <em>40</em>(4), 566–575. (<a
href="https://doi.org/10.1038/s41587-021-01111-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phylogeny estimation (the reconstruction of evolutionary trees) has recently been applied to CRISPR-based cell lineage tracing, allowing the developmental history of an individual tissue or organism to be inferred from a large number of mutated sequences in somatic cells. However, current computational methods are not able to construct phylogenetic trees from extremely large numbers of input sequences. Here, we present a deep distributed computing framework to comprehensively trace accurate large lineages (FRACTAL) that substantially enhances the scalability of current lineage estimation software tools. FRACTAL first reconstructs only an upstream lineage of the input sequences and recursively iterates the same produce for its downstream lineages using independent computing nodes. We demonstrate the utility of FRACTAL by reconstructing lineages from &amp;gt;235 million simulated sequences and from &amp;gt;16 million cells from a simulated experiment with a CRISPR system that accumulates mutations during cell proliferation. We also successfully applied FRACTAL to evolutionary tree reconstructions and to an experiment using error-prone PCR (EP-PCR) for large-scale sequence diversification. Cell lineage tracing is scaled up to hundreds of millions of simulated sequences with distributed computing.},
  archive      = {J_NBT},
  author       = {Konno, Naoki and Kijima, Yusuke and Watano, Keito and Ishiguro, Soh and Ono, Keiichiro and Tanaka, Mamoru and Mori, Hideto and Masuyama, Nanami and Pratt, Dexter and Ideker, Trey and Iwasaki, Wataru and Yachie, Nozomu},
  doi          = {10.1038/s41587-021-01111-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {566-575},
  shortjournal = {Nature Biotech.},
  title        = {Deep distributed computing to reconstruct extremely large lineage trees},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Whole-cell segmentation of tissue images with human-level
performance using large-scale data annotation and deep learning.
<em>NBT</em>, <em>40</em>(4), 555–565. (<a
href="https://doi.org/10.1038/s41587-021-01094-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A principal challenge in the analysis of tissue imaging data is cell segmentation—the task of identifying the precise boundary of every cell in an image. To address this problem we constructed TissueNet, a dataset for training segmentation models that contains more than 1 million manually labeled cells, an order of magnitude more than all previously published segmentation training datasets. We used TissueNet to train Mesmer, a deep-learning-enabled segmentation algorithm. We demonstrated that Mesmer is more accurate than previous methods, generalizes to the full diversity of tissue types and imaging platforms in TissueNet, and achieves human-level performance. Mesmer enabled the automated extraction of key cellular features, such as subcellular localization of protein signal, which was challenging with previous approaches. We then adapted Mesmer to harness cell lineage information in highly multiplexed datasets and used this enhanced version to quantify cell morphology changes during human gestation. All code, data and models are released as a community resource. Deep learning algorithms perform as well as humans in identifying cells in tissue images.},
  archive      = {J_NBT},
  author       = {Greenwald, Noah F. and Miller, Geneva and Moen, Erick and Kong, Alex and Kagel, Adam and Dougherty, Thomas and Fullaway, Christine Camacho and McIntosh, Brianna J. and Leow, Ke Xuan and Schwartz, Morgan Sarah and Pavelchek, Cole and Cui, Sunny and Camplisson, Isabella and Bar-Tal, Omer and Singh, Jaiveer and Fong, Mara and Chaudhry, Gautam and Abraham, Zion and Moseley, Jackson and Warshawsky, Shiri and Soon, Erin and Greenbaum, Shirley and Risom, Tyler and Hollmann, Travis and Bendall, Sean C. and Keren, Leeat and Graf, William and Angelo, Michael and Van Valen, David},
  doi          = {10.1038/s41587-021-01094-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {555-565},
  shortjournal = {Nature Biotech.},
  title        = {Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Locus-specific expression of transposable elements in single
cells with CELLO-seq. <em>NBT</em>, <em>40</em>(4), 546–554. (<a
href="https://doi.org/10.1038/s41587-021-01093-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposable elements (TEs) regulate diverse biological processes, from early development to cancer. Expression of young TEs is difficult to measure with next-generation, single-cell sequencing technologies because their highly repetitive nature means that short complementary DNA reads cannot be unambiguously mapped to a specific locus. Single CELl LOng-read RNA-sequencing (CELLO-seq) combines long-read single cell RNA-sequencing with computational analyses to measure TE expression at unique loci. We used CELLO-seq to assess the widespread expression of TEs in two-cell mouse blastomeres as well as in human induced pluripotent stem cells. Across both species, old and young TEs showed evidence of locus-specific expression with simulations demonstrating that only a small number of very young elements in the mouse could not be mapped back to the reference with high confidence. Exploring the relationship between the expression of individual elements and putative regulators revealed large heterogeneity, with TEs within a class showing different patterns of correlation and suggesting distinct regulatory mechanisms. Expression of transposable elements is mapped to unique genomic loci with CELLO-seq.},
  archive      = {J_NBT},
  author       = {Berrens, Rebecca V. and Yang, Andrian and Laumer, Christopher E. and Lun, Aaron T. L. and Bieberich, Florian and Law, Cheuk-Ting and Lan, Guocheng and Imaz, Maria and Bowness, Joseph S. and Brockdorff, Neil and Gaffney, Daniel J. and Marioni, John C.},
  doi          = {10.1038/s41587-021-01093-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {546-554},
  shortjournal = {Nature Biotech.},
  title        = {Locus-specific expression of transposable elements in single cells with CELLO-seq},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). RNA-responsive elements for eukaryotic translational
control. <em>NBT</em>, <em>40</em>(4), 539–545. (<a
href="https://doi.org/10.1038/s41587-021-01068-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to control translation of endogenous or exogenous RNAs in eukaryotic cells would facilitate a variety of biotechnological applications. Current strategies are limited by low fold changes in transgene output and the size of trigger RNAs (trRNAs). Here we introduce eukaryotic toehold switches (eToeholds) as modular riboregulators. eToeholds contain internal ribosome entry site sequences and form inhibitory loops in the absence of a specific trRNA. When the trRNA is present, eToeholds anneal to it, disrupting the inhibitory loops and allowing translation. Through optimization of RNA annealing, we achieved up to 16-fold induction of transgene expression in mammalian cells. We demonstrate that eToeholds can discriminate among viral infection status, presence or absence of gene expression and cell types based on the presence of exogenous or endogenous RNA transcripts. The translation of transgenes is regulated using RNA sensors.},
  archive      = {J_NBT},
  author       = {Zhao, Evan M. and Mao, Angelo S. and de Puig, Helena and Zhang, Kehan and Tippens, Nathaniel D. and Tan, Xiao and Ran, F. Ann and Han, Isaac and Nguyen, Peter Q. and Chory, Emma J. and Hua, Tiffany Y. and Ramesh, Pradeep and Thompson, David B. and Oh, Crystal Yuri and Zigon, Eric S. and English, Max A. and Collins, James J.},
  doi          = {10.1038/s41587-021-01068-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {539-545},
  shortjournal = {Nature Biotech.},
  title        = {RNA-responsive elements for eukaryotic translational control},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identifying phenotype-associated subpopulations by
integrating bulk and single-cell sequencing data. <em>NBT</em>,
<em>40</em>(4), 527–538. (<a
href="https://doi.org/10.1038/s41587-021-01091-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bulk and single cell measurements are integrated to identify phenotype-associated subpopulations of cells. Single-cell RNA sequencing (scRNA-seq) distinguishes cell types, states and lineages within the context of heterogeneous tissues. However, current single-cell data cannot directly link cell clusters with specific phenotypes. Here we present Scissor, a method that identifies cell subpopulations from single-cell data that are associated with a given phenotype. Scissor integrates phenotype-associated bulk expression data and single-cell data by first quantifying the similarity between each single cell and each bulk sample. It then optimizes a regression model on the correlation matrix with the sample phenotype to identify relevant subpopulations. Applied to a lung cancer scRNA-seq dataset, Scissor identified subsets of cells associated with worse survival and with TP53 mutations. In melanoma, Scissor discerned a T cell subpopulation with low PDCD1/CTLA4 and high TCF7 expression associated with an immunotherapy response. Beyond cancer, Scissor was effective in interpreting facioscapulohumeral muscular dystrophy and Alzheimer’s disease datasets. Scissor identifies biologically and clinically relevant cell subpopulations from single-cell assays by leveraging phenotype and bulk-omics datasets.},
  archive      = {J_NBT},
  author       = {Sun, Duanchen and Guan, Xiangnan and Moran, Amy E. and Wu, Ling-Yun and Qian, David Z. and Schedin, Pepper and Dai, Mu-Shui and Danilov, Alexey V. and Alumkal, Joshi J. and Adey, Andrew C. and Spellman, Paul T. and Xia, Zheng},
  doi          = {10.1038/s41587-021-01091-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {527-538},
  shortjournal = {Nature Biotech.},
  title        = {Identifying phenotype-associated subpopulations by integrating bulk and single-cell sequencing data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Robust decomposition of cell type mixtures in spatial
transcriptomics. <em>NBT</em>, <em>40</em>(4), 517–526. (<a
href="https://doi.org/10.1038/s41587-021-00830-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A limitation of spatial transcriptomics technologies is that individual measurements may contain contributions from multiple cells, hindering the discovery of cell-type-specific spatial patterns of localization and expression. Here, we develop robust cell type decomposition (RCTD), a computational method that leverages cell type profiles learned from single-cell RNA-seq to decompose cell type mixtures while correcting for differences across sequencing technologies. We demonstrate the ability of RCTD to detect mixtures and identify cell types on simulated datasets. Furthermore, RCTD accurately reproduces known cell type and subtype localization patterns in Slide-seq and Visium datasets of the mouse brain. Finally, we show how RCTD’s recovery of cell type localization enables the discovery of genes within a cell type whose expression depends on spatial environment. Spatial mapping of cell types with RCTD enables the spatial components of cellular identity to be defined, uncovering new principles of cellular organization in biological tissue. RCTD is publicly available as an open-source R package at https://github.com/dmcable/RCTD . Cell type mapping in spatial transcriptomics is enabled by accounting for compositional mixtures and differences in sequencing technologies.},
  archive      = {J_NBT},
  author       = {Cable, Dylan M. and Murray, Evan and Zou, Luli S. and Goeva, Aleksandrina and Macosko, Evan Z. and Chen, Fei and Irizarry, Rafael A.},
  doi          = {10.1038/s41587-021-00830-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {517-526},
  shortjournal = {Nature Biotech.},
  title        = {Robust decomposition of cell type mixtures in spatial transcriptomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fast and accurate metagenotyping of the human gut microbiome
with GT-pro. <em>NBT</em>, <em>40</em>(4), 507–516. (<a
href="https://doi.org/10.1038/s41587-021-01102-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single nucleotide polymorphisms (SNPs) in metagenomics are used to quantify population structure, track strains and identify genetic determinants of microbial phenotypes. However, existing alignment-based approaches for metagenomic SNP detection require high-performance computing and enough read coverage to distinguish SNPs from sequencing errors. To address these issues, we developed the GenoTyper for Prokaryotes (GT-Pro), a suite of methods to catalog SNPs from genomes and use unique k-mers to rapidly genotype these SNPs from metagenomes. Compared to methods that use read alignment, GT-Pro is more accurate and two orders of magnitude faster. Using high-quality genomes, we constructed a catalog of 104 million SNPs in 909 human gut species and used unique k-mers targeting this catalog to characterize the global population structure of gut microbes from 7,459 samples. GT-Pro enables fast and memory-efficient metagenotyping of millions of SNPs on a personal computer. Alignment-free SNP calling from metagenomes reduces computational time by two orders of magnitude.},
  archive      = {J_NBT},
  author       = {Shi, Zhou Jason and Dimitrov, Boris and Zhao, Chunyu and Nayfach, Stephen and Pollard, Katherine S.},
  doi          = {10.1038/s41587-021-01102-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {507-516},
  shortjournal = {Nature Biotech.},
  title        = {Fast and accurate metagenotyping of the human gut microbiome with GT-pro},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improved prediction of immune checkpoint blockade efficacy
across multiple cancer types. <em>NBT</em>, <em>40</em>(4), 499–506. (<a
href="https://doi.org/10.1038/s41587-021-01070-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Only a fraction of patients with cancer respond to immune checkpoint blockade (ICB) treatment, but current decision-making procedures have limited accuracy. In this study, we developed a machine learning model to predict ICB response by integrating genomic, molecular, demographic and clinical data from a comprehensively curated cohort (MSK-IMPACT) with 1,479 patients treated with ICB across 16 different cancer types. In a retrospective analysis, the model achieved high sensitivity and specificity in predicting clinical response to immunotherapy and predicted both overall survival and progression-free survival in the test data across different cancer types. Our model significantly outperformed predictions based on tumor mutational burden, which was recently approved by the U.S. Food and Drug Administration for this purpose1. Additionally, the model provides quantitative assessments of the model features that are most salient for the predictions. We anticipate that this approach will substantially improve clinical decision-making in immunotherapy and inform future interventions. A combination of genomic and clinical features improves predictions of response to immune checkpoint blockade.},
  archive      = {J_NBT},
  author       = {Chowell, Diego and Yoo, Seong-Keun and Valero, Cristina and Pastore, Alessandro and Krishna, Chirag and Lee, Mark and Hoen, Douglas and Shi, Hongyu and Kelly, Daniel W. and Patel, Neal and Makarov, Vladimir and Ma, Xiaoxiao and Vuong, Lynda and Sabio, Erich Y. and Weiss, Kate and Kuo, Fengshen and Lenz, Tobias L. and Samstein, Robert M. and Riaz, Nadeem and Adusumilli, Prasad S. and Balachandran, Vinod P. and Plitas, George and Ari Hakimi, A. and Abdel-Wahab, Omar and Shoushtari, Alexander N. and Postow, Michael A. and Motzer, Robert J. and Ladanyi, Marc and Zehir, Ahmet and Berger, Michael F. and Gönen, Mithat and Morris, Luc G. T. and Weinhold, Nils and Chan, Timothy A.},
  doi          = {10.1038/s41587-021-01070-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {499-506},
  shortjournal = {Nature Biotech.},
  title        = {Improved prediction of immune checkpoint blockade efficacy across multiple cancer types},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). T cells targeted to TdT kill leukemic lymphoblasts while
sparing normal lymphocytes. <em>NBT</em>, <em>40</em>(4), 488–498. (<a
href="https://doi.org/10.1038/s41587-021-01089-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike chimeric antigen receptors, T-cell receptors (TCRs) can recognize intracellular targets presented on human leukocyte antigen (HLA) molecules. Here we demonstrate that T cells expressing TCRs specific for peptides from the intracellular lymphoid-specific enzyme terminal deoxynucleotidyl transferase (TdT), presented in the context of HLA-A*02:01, specifically eliminate primary acute lymphoblastic leukemia (ALL) cells of T- and B-cell origin in vitro and in three mouse models of disseminated B-ALL. By contrast, the treatment spares normal peripheral T- and B-cell repertoires and normal myeloid cells in vitro, and in vivo in humanized mice. TdT is an attractive cancer target as it is highly and homogeneously expressed in 80–94\% of B- and T-ALLs, but only transiently expressed during normal lymphoid differentiation, limiting on-target toxicity of TdT-specific T cells. TCR-modified T cells targeting TdT may be a promising immunotherapy for B-ALL and T-ALL that preserves normal lymphocytes. Engineered T cells kill leukemic cells with little off-target toxicity.},
  archive      = {J_NBT},
  author       = {Ali, Muhammad and Giannakopoulou, Eirini and Li, Yingqian and Lehander, Madeleine and Virding Culleton, Stina and Yang, Weiwen and Knetter, Cathrine and Odabasi, Mete Can and Bollineni, Ravi Chand and Yang, Xinbo and Foldvari, Zsofia and Böschen, Maxi-Lu and Taraldsrud, Eli and Strønen, Erlend and Toebes, Mireille and Hillen, Amy and Mazzi, Stefania and de Ru, Arnoud H. and Janssen, George M. C. and Kolstad, Arne and Tjønnfjord, Geir Erland and Lie, Benedicte A. and Griffioen, Marieke and Lehmann, Sören and Osnes, Liv Toril and Buechner, Jochen and Garcia, K. Christopher and Schumacher, Ton N. and van Veelen, Peter A. and Leisegang, Matthias and Jacobsen, Sten Eirik W. and Woll, Petter and Olweus, Johanna},
  doi          = {10.1038/s41587-021-01089-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {488-498},
  shortjournal = {Nature Biotech.},
  title        = {T cells targeted to TdT kill leukemic lymphoblasts while sparing normal lymphocytes},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Remote smartphone monitoring of parkinson’s disease and
individual response to therapy. <em>NBT</em>, <em>40</em>(4), 480–487.
(<a href="https://doi.org/10.1038/s41587-021-00974-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Remote health assessments that gather real-world data (RWD) outside clinic settings require a clear understanding of appropriate methods for data collection, quality assessment, analysis and interpretation. Here we examine the performance and limitations of smartphones in collecting RWD in the remote mPower observational study of Parkinson’s disease (PD). Within the first 6 months of study commencement, 960 participants had enrolled and performed at least five self-administered active PD symptom assessments (speeded tapping, gait/balance, phonation or memory). Task performance, especially speeded tapping, was predictive of self-reported PD status (area under the receiver operating characteristic curve (AUC) = 0.8) and correlated with in-clinic evaluation of disease severity (r = 0.71; P &amp;lt; 1.8 × 10−6) when compared with motor Movement Disorder Society-Unified Parkinson’s Disease Rating Scale (MDS-UPDRS). Although remote assessment requires careful consideration for accurate interpretation of RWD, our results support the use of smartphones and wearables in objective and personalized disease assessments. Smartphone sensors that monitor disease symptoms enable remote assessment of Parkinson’s patients.},
  archive      = {J_NBT},
  author       = {Omberg, Larsson and Chaibub Neto, Elias and Perumal, Thanneer M. and Pratap, Abhishek and Tediarjo, Aryton and Adams, Jamie and Bloem, Bastiaan R. and Bot, Brian M. and Elson, Molly and Goldman, Samuel M. and Kellen, Michael R. and Kieburtz, Karl and Klein, Arno and Little, Max A. and Schneider, Ruth and Suver, Christine and Tarolli, Christopher and Tanner, Caroline M. and Trister, Andrew D. and Wilbanks, John and Dorsey, E. Ray and Mangravite, Lara M.},
  doi          = {10.1038/s41587-021-00974-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {480-487},
  shortjournal = {Nature Biotech.},
  title        = {Remote smartphone monitoring of parkinson’s disease and individual response to therapy},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Super-resolved spatial transcriptomics by deep data fusion.
<em>NBT</em>, <em>40</em>(4), 476–479. (<a
href="https://doi.org/10.1038/s41587-021-01075-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for spatial transcriptomics are limited by low spatial resolution. Here we introduce a method that integrates spatial gene expression data with histological image data from the same tissue section to infer higher-resolution expression maps. Using a deep generative model, our method characterizes the transcriptome of micrometer-scale anatomical features and can predict spatial gene expression from histology images alone. The low resolution of spatial transcriptomics is substantially improved by including histology images.},
  archive      = {J_NBT},
  author       = {Bergenstråhle, Ludvig and He, Bryan and Bergenstråhle, Joseph and Abalo, Xesús and Mirzazadeh, Reza and Thrane, Kim and Ji, Andrew L. and Andersson, Alma and Larsson, Ludvig and Stakenborg, Nathalie and Boeckxstaens, Guy and Khavari, Paul and Zou, James and Lundeberg, Joakim and Maaskola, Jonas},
  doi          = {10.1038/s41587-021-01075-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {476-479},
  shortjournal = {Nature Biotech.},
  title        = {Super-resolved spatial transcriptomics by deep data fusion},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). First rounders: Jay keasling. <em>NBT</em>, <em>40</em>(4),
475. (<a href="https://doi.org/10.1038/s41587-022-01292-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01292-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {475},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Jay keasling},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhanced cancer detection from cell-free DNA. <em>NBT</em>,
<em>40</em>(4), 473–474. (<a
href="https://doi.org/10.1038/s41587-021-01207-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new assay for cell-free DNA fragmentation improves prediction of gene expression in liquid biopsies.},
  archive      = {J_NBT},
  author       = {Jiang, Peiyong and Lo, Y. M. Dennis},
  doi          = {10.1038/s41587-021-01207-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {473-474},
  shortjournal = {Nature Biotech.},
  title        = {Enhanced cancer detection from cell-free DNA},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Organoids and organs on a chip. <em>NBT</em>,
<em>40</em>(4), 472. (<a
href="https://doi.org/10.1038/s41587-022-01287-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the manufacture and use of organoids and organs on a chip.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01287-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {472},
  shortjournal = {Nature Biotech.},
  title        = {Organoids and organs on a chip},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). European patent protection for medical uses of known
products and drug repurposing. <em>NBT</em>, <em>40</em>(4), 465–471.
(<a href="https://doi.org/10.1038/s41587-022-01269-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obtaining European patent protection for medical uses of known products is not a key factor limiting repurposing activity.},
  archive      = {J_NBT},
  author       = {Aboy, Mateo and Liddell, Kathleen and Jordan, Matthew and Crespo, Cristina and Liddicoat, Johnathon},
  doi          = {10.1038/s41587-022-01269-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {465-471},
  shortjournal = {Nature Biotech.},
  title        = {European patent protection for medical uses of known products and drug repurposing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Who will oversee the ethical limits of human embryo
research? <em>NBT</em>, <em>40</em>(4), 463–464. (<a
href="https://doi.org/10.1038/s41587-022-01274-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Adashi, Eli Y. and Cohen, I. Glenn},
  doi          = {10.1038/s41587-022-01274-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {463-464},
  shortjournal = {Nature Biotech.},
  title        = {Who will oversee the ethical limits of human embryo research?},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Differentiated dollars. <em>NBT</em>, <em>40</em>(4),
458–462. (<a href="https://doi.org/10.1038/s41587-022-01260-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Building your startup with venture philanthropy investment.},
  archive      = {J_NBT},
  author       = {Alvarez, Daniel L. and Lo, Andrew W.},
  doi          = {10.1038/s41587-022-01260-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {458-462},
  shortjournal = {Nature Biotech.},
  title        = {Differentiated dollars},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(4), 449. (<a
href="https://doi.org/10.1038/s41587-022-01279-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01279-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {449},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CRISPR beef cattle get FDA green light. <em>NBT</em>,
<em>40</em>(4), 448. (<a
href="https://doi.org/10.1038/s41587-022-01297-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01297-z},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {448},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR beef cattle get FDA green light},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Pyrukynd ushers in new era for rare cellular metabolic
disease. <em>NBT</em>, <em>40</em>(4), 446–448. (<a
href="https://doi.org/10.1038/s41587-022-01286-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The approval of the first oral pill to treat a rare anemia heralds the arrival of a new class of agents that could shake up enzyme replacement therapies for metabolic disease.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-022-01286-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {446-448},
  shortjournal = {Nature Biotech.},
  title        = {Pyrukynd ushers in new era for rare cellular metabolic disease},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Glyco-immune checkpoints for cancer reach clinic.
<em>NBT</em>, <em>40</em>(4), 446. (<a
href="https://doi.org/10.1038/s41587-022-01295-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01295-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {446},
  shortjournal = {Nature Biotech.},
  title        = {Glyco-immune checkpoints for cancer reach clinic},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). In antarctica, diesel is for dinner. <em>NBT</em>,
<em>40</em>(4), 444. (<a
href="https://doi.org/10.1038/s41587-022-01285-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-022-01285-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {444},
  shortjournal = {Nature Biotech.},
  title        = {In antarctica, diesel is for dinner},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). COVID vaccine IP waiver agreed. <em>NBT</em>,
<em>40</em>(4), 443. (<a
href="https://doi.org/10.1038/s41587-022-01298-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01298-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {443},
  shortjournal = {Nature Biotech.},
  title        = {COVID vaccine IP waiver agreed},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Licensing for profit and for good. <em>NBT</em>,
<em>40</em>(4), 439. (<a
href="https://doi.org/10.1038/s41587-022-01296-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Broad Institute’s enlightened licensing approach to CRISPR–Cas9 intellectual property stands out in the otherwise regrettable spat for patent rights over the foundational technology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01296-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {439},
  shortjournal = {Nature Biotech.},
  title        = {Licensing for profit and for good},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Innovators take cover as market bubble bursts.
<em>NBT</em>, <em>40</em>(4), 450–457. (<a
href="https://doi.org/10.1038/s41587-022-01277-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Can biotech’s cash cushion soften the blow as public markets tumble?},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-022-01277-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {4},
  pages        = {450-457},
  shortjournal = {Nature Biotech.},
  title        = {Innovators take cover as market bubble bursts},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Broad defeats berkeley CRISPR patent. <em>NBT</em>,
<em>40</em>(4), 445. (<a
href="https://doi.org/10.1038/d41587-022-00004-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A US decision to award a set of key patents related to CRISPR–Cas9 gene editing to the Broad Institute could spell the end for a long-running dispute over inventorship with the University of California and the University of Vienna. A US decision to award a set of key patents related to CRISPR–Cas9 gene editing to the Broad Institute could spell the end for a long-running dispute over inventorship with the University of California and the University of Vienna.},
  archive      = {J_NBT},
  author       = {Shaffer, Catherine},
  doi          = {10.1038/d41587-022-00004-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {4},
  pages        = {445},
  shortjournal = {Nature Biotech.},
  title        = {Broad defeats berkeley CRISPR patent},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). First soluble TCR therapy opens “new universe” of cancer
targets. <em>NBT</em>, <em>40</em>(4), 441–444. (<a
href="https://doi.org/10.1038/s41587-022-01282-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cell receptor bispecific agents provide off-the-shelf therapies for targeting intracellular tumor antigens.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01282-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {4},
  pages        = {441-444},
  shortjournal = {Nature Biotech.},
  title        = {First soluble TCR therapy opens ‘new universe’ of cancer targets},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). People. <em>NBT</em>, <em>40</em>(3), 438. (<a
href="https://doi.org/10.1038/s41587-022-01259-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01259-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {438},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrating innovation as a core objective in medical
training. <em>NBT</em>, <em>40</em>(3), 434–437. (<a
href="https://doi.org/10.1038/s41587-022-01253-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As innovations in the biotechnology sector continue to proliferate, the traditional education of medical students, residents and fellows will need to change to incorporate innovation as a core tenet of training.},
  archive      = {J_NBT},
  author       = {Boms, Okechi and Shi, Zhuo and Mallipeddi, Nathan and Chung, Janice J. and Marks, William H. and Whitehead, David C. and Succi, Marc D.},
  doi          = {10.1038/s41587-022-01253-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {434-437},
  shortjournal = {Nature Biotech.},
  title        = {Integrating innovation as a core objective in medical training},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Engineered pegRNAs improve prime editing
efficiency. <em>NBT</em>, <em>40</em>(3), 432. (<a
href="https://doi.org/10.1038/s41587-021-01175-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Nelson, James W. and Randolph, Peyton B. and Shen, Simon P. and Everette, Kelcee A. and Chen, Peter J. and Anzalone, Andrew V. and An, Meirui and Newby, Gregory A. and Chen, Jonathan C. and Hsu, Alvin and Liu, David R.},
  doi          = {10.1038/s41587-021-01175-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {432},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Engineered pegRNAs improve prime editing efficiency},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Multiscale and integrative single-cell
hi-c analysis with higashi. <em>NBT</em>, <em>40</em>(3), 432. (<a
href="https://doi.org/10.1038/s41587-022-01263-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zhang, Ruochi and Zhou, Tianming and Ma, Jian},
  doi          = {10.1038/s41587-022-01263-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {432},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Multiscale and integrative single-cell hi-C analysis with higashi},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Population genomic analysis of aegilops tauschii identifies
targets for bread wheat improvement. <em>NBT</em>, <em>40</em>(3),
422–431. (<a href="https://doi.org/10.1038/s41587-021-01058-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aegilops tauschii, the diploid wild progenitor of the D subgenome of bread wheat, is a reservoir of genetic diversity for improving bread wheat performance and environmental resilience. Here we sequenced 242 Ae. tauschii accessions and compared them to the wheat D subgenome to characterize genomic diversity. We found that a rare lineage of Ae. tauschii geographically restricted to present-day Georgia contributed to the wheat D subgenome in the independent hybridizations that gave rise to modern bread wheat. Through k-mer-based association mapping, we identified discrete genomic regions with candidate genes for disease and pest resistance and demonstrated their functional transfer into wheat by transgenesis and wide crossing, including the generation of a library of hexaploids incorporating diverse Ae. tauschii genomes. Exploiting the genomic diversity of the Ae. tauschii ancestral diploid genome permits rapid trait discovery and functional genetic validation in a hexaploid background amenable to breeding. Exploring the diversity of the wild progenitor of the wheat D subgenome allows for rapid trait discovery.},
  archive      = {J_NBT},
  author       = {Gaurav, Kumar and Arora, Sanu and Silva, Paula and Sánchez-Martín, Javier and Horsnell, Richard and Gao, Liangliang and Brar, Gurcharn S. and Widrig, Victoria and John Raupp, W. and Singh, Narinder and Wu, Shuangye and Kale, Sandip M. and Chinoy, Catherine and Nicholson, Paul and Quiroz-Chávez, Jesús and Simmonds, James and Hayta, Sadiye and Smedley, Mark A. and Harwood, Wendy and Pearce, Suzannah and Gilbert, David and Kangara, Ngonidzashe and Gardener, Catherine and Forner-Martínez, Macarena and Liu, Jiaqian and Yu, Guotai and Boden, Scott A. and Pascucci, Attilio and Ghosh, Sreya and Hafeez, Amber N. and O’Hara, Tom and Waites, Joshua and Cheema, Jitender and Steuernagel, Burkhard and Patpour, Mehran and Justesen, Annemarie Fejer and Liu, Shuyu and Rudd, Jackie C. and Avni, Raz and Sharon, Amir and Steiner, Barbara and Kirana, Rizky Pasthika and Buerstmayr, Hermann and Mehrabi, Ali A. and Nasyrova, Firuza Y. and Chayut, Noam and Matny, Oadi and Steffenson, Brian J. and Sandhu, Nitika and Chhuneja, Parveen and Lagudah, Evans and Elkot, Ahmed F. and Tyrrell, Simon and Bian, Xingdong and Davey, Robert P. and Simonsen, Martin and Schauser, Leif and Tiwari, Vijay K. and Randy Kutcher, H. and Hucl, Pierre and Li, Aili and Liu, Deng-Cai and Mao, Long and Xu, Steven and Brown-Guedira, Gina and Faris, Justin and Dvorak, Jan and Luo, Ming-Cheng and Krasileva, Ksenia and Lux, Thomas and Artmeier, Susanne and Mayer, Klaus F. X. and Uauy, Cristobal and Mascher, Martin and Bentley, Alison R. and Keller, Beat and Poland, Jesse and Wulff, Brande B. H.},
  doi          = {10.1038/s41587-021-01058-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {422-431},
  shortjournal = {Nature Biotech.},
  title        = {Population genomic analysis of aegilops tauschii identifies targets for bread wheat improvement},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High-confidence structural annotation of metabolites absent
from spectral libraries. <em>NBT</em>, <em>40</em>(3), 411–421. (<a
href="https://doi.org/10.1038/s41587-021-01045-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Untargeted metabolomics experiments rely on spectral libraries for structure annotation, but, typically, only a small fraction of spectra can be matched. Previous in silico methods search in structure databases but cannot distinguish between correct and incorrect annotations. Here we introduce the COSMIC workflow that combines in silico structure database generation and annotation with a confidence score consisting of kernel density P value estimation and a support vector machine with enforced directionality of features. On diverse datasets, COSMIC annotates a substantial number of hits at low false discovery rates and outperforms spectral library search. To demonstrate that COSMIC can annotate structures never reported before, we annotated 12 natural bile acids. The annotation of nine structures was confirmed by manual evaluation and two structures using synthetic standards. In human samples, we annotated and manually validated 315 molecular structures currently absent from the Human Metabolome Database. Application of COSMIC to data from 17,400 metabolomics experiments led to 1,715 high-confidence structural annotations that were absent from spectral libraries. COSMIC outperforms spectral library search for metabolite annotation and annotates previously unseen structures.},
  archive      = {J_NBT},
  author       = {Hoffmann, Martin A. and Nothias, Louis-Félix and Ludwig, Marcus and Fleischauer, Markus and Gentry, Emily C. and Witting, Michael and Dorrestein, Pieter C. and Dührkop, Kai and Böcker, Sebastian},
  doi          = {10.1038/s41587-021-01045-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {411-421},
  shortjournal = {Nature Biotech.},
  title        = {High-confidence structural annotation of metabolites absent from spectral libraries},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Engineered pegRNAs improve prime editing efficiency.
<em>NBT</em>, <em>40</em>(3), 402–410. (<a
href="https://doi.org/10.1038/s41587-021-01039-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing enables the installation of virtually any combination of point mutations, small insertions or small deletions in the DNA of living cells. A prime editing guide RNA (pegRNA) directs the prime editor protein to the targeted locus and also encodes the desired edit. Here we show that degradation of the 3′ region of the pegRNA that contains the reverse transcriptase template and the primer binding site can poison the activity of prime editing systems, impeding editing efficiency. We incorporated structured RNA motifs to the 3′ terminus of pegRNAs that enhance their stability and prevent degradation of the 3′ extension. The resulting engineered pegRNAs (epegRNAs) improve prime editing efficiency 3–4-fold in HeLa, U2OS and K562 cells and in primary human fibroblasts without increasing off-target editing activity. We optimized the choice of 3′ structural motif and developed pegLIT, a computational tool to identify non-interfering nucleotide linkers between pegRNAs and 3′ motifs. Finally, we showed that epegRNAs enhance the efficiency of the installation or correction of disease-relevant mutations. Stabilizing pegRNAs with 3′ RNA structures increases prime editing efficiency.},
  archive      = {J_NBT},
  author       = {Nelson, James W. and Randolph, Peyton B. and Shen, Simon P. and Everette, Kelcee A. and Chen, Peter J. and Anzalone, Andrew V. and An, Meirui and Newby, Gregory A. and Chen, Jonathan C. and Hsu, Alvin and Liu, David R.},
  doi          = {10.1038/s41587-021-01039-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {402-410},
  shortjournal = {Nature Biotech.},
  title        = {Engineered pegRNAs improve prime editing efficiency},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Functional single-cell genomics of human cytomegalovirus
infection. <em>NBT</em>, <em>40</em>(3), 391–401. (<a
href="https://doi.org/10.1038/s41587-021-01059-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how viral and host factors interact and how perturbations impact infection is the basis for designing antiviral interventions. Here we define the functional contribution of each viral and host factor involved in human cytomegalovirus infection in primary human fibroblasts through pooled CRISPR interference and nuclease screening. To determine how genetic perturbation of critical host and viral factors alters the timing, course and progression of infection, we applied Perturb-seq to record the transcriptomes of tens of thousands of CRISPR-modified single cells and found that, normally, most cells follow a stereotypical transcriptional trajectory. Perturbing critical host factors does not change the stereotypical transcriptional trajectory per se but can stall, delay or accelerate progression along the trajectory, allowing one to pinpoint the stage of infection at which host factors act. Conversely, perturbation of viral factors can create distinct, abortive trajectories. Our results reveal the roles of host and viral factors and provide a roadmap for the dissection of host–pathogen interactions. Pooled and single-cell CRISPR screens provide a detailed map of human cytomegalovirus infection.},
  archive      = {J_NBT},
  author       = {Hein, Marco Y. and Weissman, Jonathan S.},
  doi          = {10.1038/s41587-021-01059-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {391-401},
  shortjournal = {Nature Biotech.},
  title        = {Functional single-cell genomics of human cytomegalovirus infection},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High-throughput functional characterization of protein
phosphorylation sites in yeast. <em>NBT</em>, <em>40</em>(3), 382–390.
(<a href="https://doi.org/10.1038/s41587-021-01051-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phosphorylation is a critical post-translational modification involved in the regulation of almost all cellular processes. However, fewer than 5\% of thousands of recently discovered phosphosites have been functionally annotated. In this study, we devised a chemical genetic approach to study the functional relevance of phosphosites in Saccharomyces cerevisiae. We generated 474 yeast strains with mutations in specific phosphosites that were screened for fitness in 102 conditions, along with a gene deletion library. Of these phosphosites, 42\% exhibited growth phenotypes, suggesting that these are more likely functional. We inferred their function based on the similarity of their growth profiles with that of gene deletions and validated a subset by thermal proteome profiling and lipidomics. A high fraction exhibited phenotypes not seen in the corresponding gene deletion, suggestive of a gain-of-function effect. For phosphosites conserved in humans, the severity of the yeast phenotypes is indicative of their human functional relevance. This high-throughput approach allows for functionally characterizing individual phosphosites at scale. Phenotypic profiling of a phospho-deficient mutant library informs functional annotations of phosphosites.},
  archive      = {J_NBT},
  author       = {Viéitez, Cristina and Busby, Bede P. and Ochoa, David and Mateus, André and Memon, Danish and Galardini, Marco and Yildiz, Umut and Trovato, Matteo and Jawed, Areeb and Geiger, Alexander G. and Oborská-Oplová, Michaela and Potel, Clement M. and Vonesch, Sibylle C. and Szu Tu, Chelsea and Shahraz, Mohammed and Stein, Frank and Steinmetz, Lars M. and Panse, Vikram G. and Noh, Kyung-Min and Savitski, Mikhail M. and Typas, Athanasios and Beltrao, Pedro},
  doi          = {10.1038/s41587-021-01051-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {382-390},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput functional characterization of protein phosphorylation sites in yeast},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell profiling of proteins and chromatin
accessibility using PHAGE-ATAC. <em>NBT</em>, <em>40</em>(3), 374–381.
(<a href="https://doi.org/10.1038/s41587-021-01065-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multimodal measurements of single-cell profiles are proving increasingly useful for characterizing cell states and regulatory mechanisms. In the present study, we developed PHAGE-ATAC (Assay for Transposase-Accessible Chromatin), a massively parallel droplet-based method that uses phage displaying, engineered, camelid single-domain antibodies (‘nanobodies’) for simultaneous single-cell measurements of protein levels and chromatin accessibility profiles, and mitochondrial DNA-based clonal tracing. We use PHAGE-ATAC for multimodal analysis in primary human immune cells, sample multiplexing, intracellular protein analysis and the detection of SARS-CoV-2 spike protein in human cell populations. Finally, we construct a synthetic high-complexity phage library for selection of antigen-specific nanobodies that bind cells of particular molecular profiles, opening an avenue for protein detection, cell characterization and screening with single-cell genomics. Protein epitopes and chromatin accessibility are measured in single cells using phage display.},
  archive      = {J_NBT},
  author       = {Fiskin, Evgenij and Lareau, Caleb A. and Ludwig, Leif S. and Eraslan, Gökcen and Liu, Feimei and Ring, Aaron M. and Xavier, Ramnik J. and Regev, Aviv},
  doi          = {10.1038/s41587-021-01065-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {374-381},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell profiling of proteins and chromatin accessibility using PHAGE-ATAC},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Highly-multiplexed volumetric mapping with raman dye imaging
and tissue clearing. <em>NBT</em>, <em>40</em>(3), 364–373. (<a
href="https://doi.org/10.1038/s41587-021-01041-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping the localization of multiple proteins in their native three-dimensional (3D) context would be useful across many areas of biomedicine, but multiplexed fluorescence imaging has limited intrinsic multiplexing capability, and most methods for increasing multiplexity can only be applied to thin samples (&amp;lt;100 µm). Here, we harness the narrow spectrum of Raman spectroscopy and introduce Raman dye imaging and tissue clearing (RADIANT), an optical method that is capable of imaging multiple targets in thick samples in one shot. We expanded the range of suitable bioorthogonal Raman dyes and developed a tissue-clearing strategy for them (Raman 3D imaging of solvent-cleared organs (rDISCO)). We applied RADIANT to image up to 11 targets in millimeter-thick brain slices, extending the imaging depth 10- to 100-fold compared to prior multiplexed protein imaging methods. We showcased the utility of RADIANT in extracting systems information, including region-specific correlation networks and their topology in cerebellum development. RADIANT will facilitate the exploration of the intricate 3D protein interactions in complex systems. Simultaneous, Raman-based imaging of more than 10 proteins in thick samples.},
  archive      = {J_NBT},
  author       = {Shi, Lixue and Wei, Mian and Miao, Yupeng and Qian, Naixin and Shi, Lingyan and Singer, Ruth A. and Benninger, Richard K. P. and Min, Wei},
  doi          = {10.1038/s41587-021-01041-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {364-373},
  shortjournal = {Nature Biotech.},
  title        = {Highly-multiplexed volumetric mapping with raman dye imaging and tissue clearing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Co-varying neighborhood analysis identifies cell populations
associated with phenotypes of interest from single-cell transcriptomics.
<em>NBT</em>, <em>40</em>(3), 355–363. (<a
href="https://doi.org/10.1038/s41587-021-01066-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As single-cell datasets grow in sample size, there is a critical need to characterize cell states that vary across samples and associate with sample attributes, such as clinical phenotypes. Current statistical approaches typically map cells to clusters and then assess differences in cluster abundance. Here we present co-varying neighborhood analysis (CNA), an unbiased method to identify associated cell populations with greater flexibility than cluster-based approaches. CNA characterizes dominant axes of variation across samples by identifying groups of small regions in transcriptional space—termed neighborhoods—that co-vary in abundance across samples, suggesting shared function or regulation. CNA performs statistical testing for associations between any sample-level attribute and the abundances of these co-varying neighborhood groups. Simulations show that CNA enables more sensitive and accurate identification of disease-associated cell states than a cluster-based approach. When applied to published datasets, CNA captures a Notch activation signature in rheumatoid arthritis, identifies monocyte populations expanded in sepsis and identifies a novel T cell population associated with progression to active tuberculosis. Inter-sample variability reveals disease-associated cell subpopulations in single-cell RNA sequencing.},
  archive      = {J_NBT},
  author       = {Reshef, Yakir A. and Rumker, Laurie and Kang, Joyce B. and Nathan, Aparna and Korsunsky, Ilya and Asgari, Samira and Murray, Megan B. and Moody, D. Branch and Raychaudhuri, Soumya},
  doi          = {10.1038/s41587-021-01066-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {355-363},
  shortjournal = {Nature Biotech.},
  title        = {Co-varying neighborhood analysis identifies cell populations associated with phenotypes of interest from single-cell transcriptomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cell segmentation in imaging-based spatial transcriptomics.
<em>NBT</em>, <em>40</em>(3), 345–354. (<a
href="https://doi.org/10.1038/s41587-021-01044-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule spatial transcriptomics protocols based on in situ sequencing or multiplexed RNA fluorescent hybridization can reveal detailed tissue organization. However, distinguishing the boundaries of individual cells in such data is challenging and can hamper downstream analysis. Current methods generally approximate cells positions using nuclei stains. We describe a segmentation method, Baysor, that optimizes two-dimensional (2D) or three-dimensional (3D) cell boundaries considering joint likelihood of transcriptional composition and cell morphology. While Baysor can take into account segmentation based on co-stains, it can also perform segmentation based on the detected transcripts alone. To evaluate performance, we extend multiplexed error-robust fluorescence in situ hybridization (MERFISH) to incorporate immunostaining of cell boundaries. Using this and other benchmarks, we show that Baysor segmentation can, in some cases, nearly double the number of cells compared to existing tools while reducing segmentation artifacts. We demonstrate that Baysor performs well on data acquired using five different protocols, making it a useful general tool for analysis of imaging-based spatial transcriptomics. Baysor enables cell segmentation based on transcripts detected by multiplexed FISH or in situ sequencing.},
  archive      = {J_NBT},
  author       = {Petukhov, Viktor and Xu, Rosalind J. and Soldatov, Ruslan A. and Cadinu, Paolo and Khodosevich, Konstantin and Moffitt, Jeffrey R. and Kharchenko, Peter V.},
  doi          = {10.1038/s41587-021-01044-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {345-354},
  shortjournal = {Nature Biotech.},
  title        = {Cell segmentation in imaging-based spatial transcriptomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Carbon-negative production of acetone and isopropanol by gas
fermentation at industrial pilot scale. <em>NBT</em>, <em>40</em>(3),
335–344. (<a href="https://doi.org/10.1038/s41587-021-01195-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many industrial chemicals that are produced from fossil resources could be manufactured more sustainably through fermentation. Here we describe the development of a carbon-negative fermentation route to producing the industrially important chemicals acetone and isopropanol from abundant, low-cost waste gas feedstocks, such as industrial emissions and syngas. Using a combinatorial pathway library approach, we first mined a historical industrial strain collection for superior enzymes that we used to engineer the autotrophic acetogen Clostridium autoethanogenum. Next, we used omics analysis, kinetic modeling and cell-free prototyping to optimize flux. Finally, we scaled-up our optimized strains for continuous production at rates of up to ~3 g/L/h and ~90\% selectivity. Life cycle analysis confirmed a negative carbon footprint for the products. Unlike traditional production processes, which result in release of greenhouse gases, our process fixes carbon. These results show that engineered acetogens enable sustainable, high-efficiency, high-selectivity chemicals production. We expect that our approach can be readily adapted to a wide range of commodity chemicals. Two industrial chemicals are sustainably produced at large scale by microbial gas fermentation.},
  archive      = {J_NBT},
  author       = {Liew, Fungmin Eric and Nogle, Robert and Abdalla, Tanus and Rasor, Blake J. and Canter, Christina and Jensen, Rasmus O. and Wang, Lan and Strutz, Jonathan and Chirania, Payal and De Tissera, Sashini and Mueller, Alexander P. and Ruan, Zhenhua and Gao, Allan and Tran, Loan and Engle, Nancy L. and Bromley, Jason C. and Daniell, James and Conrado, Robert and Tschaplinski, Timothy J. and Giannone, Richard J. and Hettich, Robert L. and Karim, Ashty S. and Simpson, Séan D. and Brown, Steven D. and Leang, Ching and Jewett, Michael C. and Köpke, Michael},
  doi          = {10.1038/s41587-021-01195-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {335-344},
  shortjournal = {Nature Biotech.},
  title        = {Carbon-negative production of acetone and isopropanol by gas fermentation at industrial pilot scale},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Direct targeting of amplified gene loci for proapoptotic
anticancer therapy. <em>NBT</em>, <em>40</em>(3), 325–334. (<a
href="https://doi.org/10.1038/s41587-021-01057-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene amplification drives oncogenesis in a broad spectrum of cancers. A number of drugs have been developed to inhibit the protein products of amplified driver genes, but their clinical efficacy is often hampered by drug resistance. Here, we introduce a therapeutic strategy for targeting cancer-associated gene amplifications by activating the DNA damage response with triplex-forming oligonucleotides (TFOs), which drive the induction of apoptosis in tumors, whereas cells without amplifications process lower levels of DNA damage. Focusing on cancers driven by HER2 amplification, we find that TFOs targeting HER2 induce copy number-dependent DNA double-strand breaks (DSBs) and activate p53-independent apoptosis in HER2-positive cancer cells and human tumor xenografts via a mechanism that is independent of HER2 cellular function. This strategy has demonstrated in vivo efficacy comparable to that of current precision medicines and provided a feasible alternative to combat drug resistance in HER2-positive breast and ovarian cancer models. These findings offer a general strategy for targeting tumors with amplified genomic loci. Amplified loci in cancer genomes are directly targeted with triplex-forming oligonucleotides.},
  archive      = {J_NBT},
  author       = {Kaushik Tiwari, Meetu and Colon-Rios, Daniel A. and Tumu, Hemanta C. Rao and Liu, Yanfeng and Quijano, Elias and Krysztofiak, Adam and Chan, Cynthia and Song, Eric and Braddock, Demetrios T. and Suh, Hee-Won and Saltzman, W. Mark and Rogers, Faye A.},
  doi          = {10.1038/s41587-021-01057-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {325-334},
  shortjournal = {Nature Biotech.},
  title        = {Direct targeting of amplified gene loci for proapoptotic anticancer therapy},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pre-activated antiviral innate immunity in the upper airways
controls early SARS-CoV-2 infection in children. <em>NBT</em>,
<em>40</em>(3), 319–324. (<a
href="https://doi.org/10.1038/s41587-021-01037-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Children have reduced severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection rates and a substantially lower risk for developing severe coronavirus disease 2019 compared with adults. However, the molecular mechanisms underlying protection in younger age groups remain unknown. Here we characterize the single-cell transcriptional landscape in the upper airways of SARS-CoV-2-negative (n = 18) and age-matched SARS-CoV-2-positive (n = 24) children and corresponding samples from adults (n = 44), covering an age range of 4 weeks to 77 years. Children displayed higher basal expression of relevant pattern recognition receptors such as MDA5 (IFIH1) and RIG-I (DDX58) in upper airway epithelial cells, macrophages and dendritic cells, resulting in stronger innate antiviral responses upon SARS-CoV-2 infection than in adults. We further detected distinct immune cell subpopulations including KLRC1 (NKG2A)+ cytotoxic T cells and a CD8+ T cell population with a memory phenotype occurring predominantly in children. Our study provides evidence that the airway immune cells of children are primed for virus sensing, resulting in a stronger early innate antiviral response to SARS-CoV-2 infection than in adults. Single-cell sequencing reveals pre-activated immunity as important for milder COVID-19 symptoms in children.},
  archive      = {J_NBT},
  author       = {Loske, J. and Röhmel, J. and Lukassen, S. and Stricker, S. and Magalhães, V. G. and Liebig, J. and Chua, R. L. and Thürmann, L. and Messingschlager, M. and Seegebarth, A. and Timmermann, B. and Klages, S. and Ralser, M. and Sawitzki, B. and Sander, L. E. and Corman, V. M. and Conrad, C. and Laudi, S. and Binder, M. and Trump, S. and Eils, R. and Mall, M. A. and Lehmann, I.},
  doi          = {10.1038/s41587-021-01037-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {319-324},
  shortjournal = {Nature Biotech.},
  title        = {Pre-activated antiviral innate immunity in the upper airways controls early SARS-CoV-2 infection in children},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial components of molecular tissue biology.
<em>NBT</em>, <em>40</em>(3), 308–318. (<a
href="https://doi.org/10.1038/s41587-021-01182-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods for profiling RNA and protein expression in a spatially resolved manner are rapidly evolving, making it possible to comprehensively characterize cells and tissues in health and disease. To maximize the biological insights obtained using these techniques, it is critical to both clearly articulate the key biological questions in spatial analysis of tissues and develop the requisite computational tools to address them. Developers of analytical tools need to decide on the intrinsic molecular features of each cell that need to be considered, and how cell shape and morphological features are incorporated into the analysis. Also, optimal ways to compare different tissue samples at various length scales are still being sought. Grouping these biological problems and related computational algorithms into classes across length scales, thus characterizing common issues that need to be addressed, will facilitate further progress in spatial transcriptomics and proteomics. Regev, Theis and colleagues outline the challenges and concepts of the analysis of spatial transcriptomics data.},
  archive      = {J_NBT},
  author       = {Palla, Giovanni and Fischer, David S. and Regev, Aviv and Theis, Fabian J.},
  doi          = {10.1038/s41587-021-01182-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {308-318},
  shortjournal = {Nature Biotech.},
  title        = {Spatial components of molecular tissue biology},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sustainable manufacturing with synthetic biology.
<em>NBT</em>, <em>40</em>(3), 304–307. (<a
href="https://doi.org/10.1038/s41587-022-01248-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Producing commodity chemicals in bacteria that live on industrial air pollution captures more greenhouse gases than it emits.},
  archive      = {J_NBT},
  author       = {Scown, Corinne D. and Keasling, Jay D.},
  doi          = {10.1038/s41587-022-01248-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {304-307},
  shortjournal = {Nature Biotech.},
  title        = {Sustainable manufacturing with synthetic biology},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell sequencing. <em>NBT</em>, <em>40</em>(3), 303.
(<a href="https://doi.org/10.1038/s41587-022-01262-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods, apparatus and algorithms for single-cell sequencing.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01262-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {303},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell sequencing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Contractual solutions to overcome drug scarcity during
pandemics and epidemics. <em>NBT</em>, <em>40</em>(3), 301–302. (<a
href="https://doi.org/10.1038/s41587-022-01241-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Licensing provisions that obligate recipients of government funding to share relevant technology and know-how for scarce drugs during pandemics and epidemics can reduce shortages and overcome obstacles that intellectual property rights present.},
  archive      = {J_NBT},
  author       = {Kumar, Sapna and Rutschman, Ana Santos},
  doi          = {10.1038/s41587-022-01241-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301-302},
  shortjournal = {Nature Biotech.},
  title        = {Contractual solutions to overcome drug scarcity during pandemics and epidemics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). So you want to start a biotech company. <em>NBT</em>,
<em>40</em>(3), 296–300. (<a
href="https://doi.org/10.1038/s41587-022-01239-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Commercializing research is fraught with pitfalls, but a thoughtful checklist can ensure you set off on the right path and give your fledgling business the best chance of success.},
  archive      = {J_NBT},
  author       = {Chitale, Sadhana and Lawler, Colm and Klausner, Arthur},
  doi          = {10.1038/s41587-022-01239-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {296-300},
  shortjournal = {Nature Biotech.},
  title        = {So you want to start a biotech company},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(3), 290. (<a
href="https://doi.org/10.1038/s41587-022-01257-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01257-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {290},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bilateral china–brazil ties over agbiotech. <em>NBT</em>,
<em>40</em>(3), 289. (<a
href="https://doi.org/10.1038/s41587-022-01265-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01265-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {289},
  shortjournal = {Nature Biotech.},
  title        = {Bilateral China–Brazil ties over agbiotech},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Bio-leather gears up to wow fashion industry. <em>NBT</em>,
<em>40</em>(3), 287–289. (<a
href="https://doi.org/10.1038/s41587-022-01258-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fashion houses are starting to experiment with bioreactor-made leather, an animal-friendly option that also aims to help save the planet.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-022-01258-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {287-289},
  shortjournal = {Nature Biotech.},
  title        = {Bio-leather gears up to wow fashion industry},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). How to grow cement. <em>NBT</em>, <em>40</em>(3), 286. (<a
href="https://doi.org/10.1038/s41587-022-01264-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-022-01264-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {286},
  shortjournal = {Nature Biotech.},
  title        = {How to grow cement},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DARPins score against COVID-19. <em>NBT</em>,
<em>40</em>(3), 285. (<a
href="https://doi.org/10.1038/s41587-022-01266-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01266-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {285},
  shortjournal = {Nature Biotech.},
  title        = {DARPins score against COVID-19},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). MRNA made in africa. <em>NBT</em>, <em>40</em>(3), 284. (<a
href="https://doi.org/10.1038/s41587-022-01268-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01268-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {284},
  shortjournal = {Nature Biotech.},
  title        = {MRNA made in africa},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fine-tuning epigenome editors. <em>NBT</em>, <em>40</em>(3),
281. (<a href="https://doi.org/10.1038/s41587-022-01270-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For the raft of new ventures developing epigenome editors, a compelling niche may be diseases of haploinsufficiency or genome imprinting that require exquisite control of gene expression.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01270-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {281},
  shortjournal = {Nature Biotech.},
  title        = {Fine-tuning epigenome editors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Diabetes cell therapies take evasive action. <em>NBT</em>,
<em>40</em>(3), 291–295. (<a
href="https://doi.org/10.1038/s41587-022-01246-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protective encapsulating devices and gene-editing technologies could obviate the need for antirejection drugs in stem-cell-derived therapies for diabetes.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01246-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {291-295},
  shortjournal = {Nature Biotech.},
  title        = {Diabetes cell therapies take evasive action},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). TRNA therapeutics burst onto startup scene. <em>NBT</em>,
<em>40</em>(3), 283–286. (<a
href="https://doi.org/10.1038/s41587-022-01252-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies advance tRNA therapeutics to overcome mutant stoppages in protein synthesis shared by thousands of genetic diseases and cancers.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-022-01252-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {283-286},
  shortjournal = {Nature Biotech.},
  title        = {TRNA therapeutics burst onto startup scene},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). People. <em>NBT</em>, <em>40</em>(2), 280. (<a
href="https://doi.org/10.1038/s41587-022-01217-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01217-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Fourth-quarter biotech job picture. <em>NBT</em>,
<em>40</em>(2), 279. (<a
href="https://doi.org/10.1038/s41587-022-01214-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-022-01214-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {279},
  shortjournal = {Nature Biotech.},
  title        = {Fourth-quarter biotech job picture},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Assessing workforce needs for the emerging CAR-t cell
therapy industry. <em>NBT</em>, <em>40</em>(2), 275–278. (<a
href="https://doi.org/10.1038/s41587-022-01212-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The complicated and labor-intensive manufacturing of novel cell and gene therapies requires efforts to improve workforce development programs for the nascent cell therapy industry broadly, and for cell manufacturing more specifically.},
  archive      = {J_NBT},
  author       = {Ho, Linda D. and Robbins, Hadassah L. and Levine, Aaron D.},
  doi          = {10.1038/s41587-022-01212-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {275-278},
  shortjournal = {Nature Biotech.},
  title        = {Assessing workforce needs for the emerging CAR-T cell therapy industry},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: CRISPR prime editing with
ribonucleoprotein complexes in zebrafish and primary human cells.
<em>NBT</em>, <em>40</em>(2), 273. (<a
href="https://doi.org/10.1038/s41587-021-00939-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Petri, Karl and Zhang, Weiting and Ma, Junyan and Schmidts, Andrea and Lee, Hyunho and Horng, Joy E. and Kim, Daniel Y. and Kurt, Ibrahim C. and Clement, Kendell and Hsu, Jonathan Y. and Pinello, Luca and Maus, Marcela V. and Joung, J. Keith and Yeh, Jing-Ruey Joanna},
  doi          = {10.1038/s41587-021-00939-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {273},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: CRISPR prime editing with ribonucleoprotein complexes in zebrafish and primary human cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A small and highly sensitive red/far-red optogenetic switch
for applications in mammals. <em>NBT</em>, <em>40</em>(2), 262–272. (<a
href="https://doi.org/10.1038/s41587-021-01036-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optogenetic technologies have transformed our ability to precisely control biological processes in time and space. Yet, current eukaryotic optogenetic systems are limited by large or complex optogenetic modules, long illumination times, low tissue penetration or slow activation and deactivation kinetics. Here, we report a red/far-red light-mediated and miniaturized Δphytochrome A (ΔPhyA)-based photoswitch (REDMAP) system based on the plant photoreceptor PhyA, which rapidly binds the shuttle protein far-red elongated hypocotyl 1 (FHY1) under illumination with 660-nm light with dissociation occurring at 730 nm. We demonstrate multiple applications of REDMAP, including dynamic on/off control of the endogenous Ras/Erk mitogen-activated protein kinase (MAPK) cascade and control of epigenetic remodeling using a REDMAP-mediated CRISPR–nuclease-deactivated Cas9 (CRISPR–dCas9) (REDMAPcas) system in mice. We also demonstrate the utility of REDMAP tools for in vivo applications by activating the expression of transgenes delivered by adeno-associated viruses (AAVs) or incorporated into cells in microcapsules implanted into mice, rats and rabbits illuminated by light-emitting diodes (LEDs). Further, we controlled glucose homeostasis in type 1 diabetic (T1D) mice and rats using REDMAP to trigger insulin expression. REDMAP is a compact and sensitive tool for the precise spatiotemporal control of biological activities in animals with applications in basic biology and potentially therapy. Highly sensitive photoswitchable dimerizer enables the control of biological processes with red light in mammals.},
  archive      = {J_NBT},
  author       = {Zhou, Yang and Kong, Deqiang and Wang, Xinyi and Yu, Guiling and Wu, Xin and Guan, Ningzi and Weber, Wilfried and Ye, Haifeng},
  doi          = {10.1038/s41587-021-01036-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {262-272},
  shortjournal = {Nature Biotech.},
  title        = {A small and highly sensitive red/far-red optogenetic switch for applications in mammals},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Multiscale and integrative single-cell hi-c analysis with
higashi. <em>NBT</em>, <em>40</em>(2), 254–261. (<a
href="https://doi.org/10.1038/s41587-021-01034-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell Hi-C (scHi-C) can identify cell-to-cell variability of three-dimensional (3D) chromatin organization, but the sparseness of measured interactions poses an analysis challenge. Here we report Higashi, an algorithm based on hypergraph representation learning that can incorporate the latent correlations among single cells to enhance overall imputation of contact maps. Higashi outperforms existing methods for embedding and imputation of scHi-C data and is able to identify multiscale 3D genome features in single cells, such as compartmentalization and TAD-like domain boundaries, allowing refined delineation of their cell-to-cell variability. Moreover, Higashi can incorporate epigenomic signals jointly profiled in the same cell into the hypergraph representation learning framework, as compared to separate analysis of two modalities, leading to improved embeddings for single-nucleus methyl-3C data. In an scHi-C dataset from human prefrontal cortex, Higashi identifies connections between 3D genome features and cell-type-specific gene regulation. Higashi can also potentially be extended to analyze single-cell multiway chromatin interactions and other multimodal single-cell omics data. Single-cell Hi-C analysis with Higashi identifies cell-to-cell variability of 3D genome organization.},
  archive      = {J_NBT},
  author       = {Zhang, Ruochi and Zhou, Tianming and Ma, Jian},
  doi          = {10.1038/s41587-021-01034-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {254-261},
  shortjournal = {Nature Biotech.},
  title        = {Multiscale and integrative single-cell hi-C analysis with higashi},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Differential abundance testing on single-cell data using
k-nearest neighbor graphs. <em>NBT</em>, <em>40</em>(2), 245–253. (<a
href="https://doi.org/10.1038/s41587-021-01033-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current computational workflows for comparative analyses of single-cell datasets typically use discrete clusters as input when testing for differential abundance among experimental conditions. However, clusters do not always provide the appropriate resolution and cannot capture continuous trajectories. Here we present Milo, a scalable statistical framework that performs differential abundance testing by assigning cells to partially overlapping neighborhoods on a k-nearest neighbor graph. Using simulations and single-cell RNA sequencing (scRNA-seq) data, we show that Milo can identify perturbations that are obscured by discretizing cells into clusters, that it maintains false discovery rate control across batch effects and that it outperforms alternative differential abundance testing strategies. Milo identifies the decline of a fate-biased epithelial precursor in the aging mouse thymus and identifies perturbations to multiple lineages in human cirrhotic liver. As Milo is based on a cell–cell similarity structure, it might also be applicable to single-cell data other than scRNA-seq. Milo is provided as an open-source R software package at https://github.com/MarioniLab/miloR . Milo identifies differentially abundant populations of cells in scRNA-seq data without clustering.},
  archive      = {J_NBT},
  author       = {Dann, Emma and Henderson, Neil C. and Teichmann, Sarah A. and Morgan, Michael D. and Marioni, John C.},
  doi          = {10.1038/s41587-021-01033-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {245-253},
  shortjournal = {Nature Biotech.},
  title        = {Differential abundance testing on single-cell data using k-nearest neighbor graphs},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chromatin velocity reveals epigenetic dynamics by
single-cell profiling of heterochromatin and euchromatin. <em>NBT</em>,
<em>40</em>(2), 235–244. (<a
href="https://doi.org/10.1038/s41587-021-01031-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent efforts have succeeded in surveying open chromatin at the single-cell level, but high-throughput, single-cell assessment of heterochromatin and its underlying genomic determinants remains challenging. We engineered a hybrid transposase including the chromodomain (CD) of the heterochromatin protein-1α (HP-1α), which is involved in heterochromatin assembly and maintenance through its binding to trimethylation of the lysine 9 on histone 3 (H3K9me3), and developed a single-cell method, single-cell genome and epigenome by transposases sequencing (scGET-seq), that, unlike single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq), comprehensively probes both open and closed chromatin and concomitantly records the underlying genomic sequences. We tested scGET-seq in cancer-derived organoids and human-derived xenograft (PDX) models and identified genetic events and plasticity-driven mechanisms contributing to cancer drug resistance. Next, building upon the differential enrichment of closed and open chromatin, we devised a method, Chromatin Velocity, that identifies the trajectories of epigenetic modifications at the single-cell level. Chromatin Velocity uncovered paths of epigenetic reorganization during stem cell reprogramming and identified key transcription factors driving these developmental processes. scGET-seq reveals the dynamics of genomic and epigenetic landscapes underlying any cellular processes. Single-cell mapping of heterochromatin and euchromatin using chromatin velocity defines trajectories of epigenetic modifications.},
  archive      = {J_NBT},
  author       = {Tedesco, Martina and Giannese, Francesca and Lazarević, Dejan and Giansanti, Valentina and Rosano, Dalia and Monzani, Silvia and Catalano, Irene and Grassi, Elena and Zanella, Eugenia R. and Botrugno, Oronza A. and Morelli, Leonardo and Panina Bordignon, Paola and Caravagna, Giulio and Bertotti, Andrea and Martino, Gianvito and Aldrighetti, Luca and Pasqualato, Sebastiano and Trusolino, Livio and Cittaro, Davide and Tonon, Giovanni},
  doi          = {10.1038/s41587-021-01031-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {235-244},
  shortjournal = {Nature Biotech.},
  title        = {Chromatin velocity reveals epigenetic dynamics by single-cell profiling of heterochromatin and euchromatin},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deletion and replacement of long genomic sequences using
prime editing. <em>NBT</em>, <em>40</em>(2), 227–234. (<a
href="https://doi.org/10.1038/s41587-021-01026-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic insertions, duplications and insertion/deletions (indels), which account for ~14\% of human pathogenic mutations, cannot be accurately or efficiently corrected by current gene-editing methods, especially those that involve larger alterations (&amp;gt;100 base pairs (bp)). Here, we optimize prime editing (PE) tools for creating precise genomic deletions and direct the replacement of a genomic fragment ranging from ~1 kilobases (kb) to ~10 kb with a desired sequence (up to 60 bp) in the absence of an exogenous DNA template. By conjugating Cas9 nuclease to reverse transcriptase (PE-Cas9) and combining it with two PE guide RNAs (pegRNAs) targeting complementary DNA strands, we achieve precise and specific deletion and repair of target sequences via using this PE-Cas9-based deletion and repair (PEDAR) method. PEDAR outperformed other genome-editing methods in a reporter system and at endogenous loci, efficiently creating large and precise genomic alterations. In a mouse model of tyrosinemia, PEDAR removed a 1.38-kb pathogenic insertion within the Fah gene and precisely repaired the deletion junction to restore FAH expression in liver. Prime editing is expanded to deletions and replacements of genomic sequences of up to 10 kb.},
  archive      = {J_NBT},
  author       = {Jiang, Tingting and Zhang, Xiao-Ou and Weng, Zhiping and Xue, Wen},
  doi          = {10.1038/s41587-021-01026-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {227-234},
  shortjournal = {Nature Biotech.},
  title        = {Deletion and replacement of long genomic sequences using prime editing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Precise genomic deletions using paired prime editing.
<em>NBT</em>, <em>40</em>(2), 218–226. (<a
href="https://doi.org/10.1038/s41587-021-01025-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods to delete genomic sequences are based on clustered regularly interspaced short palindromic repeats (CRISPR)–Cas9 and pairs of single-guide RNAs (sgRNAs), but can be inefficient and imprecise, with errors including small indels as well as unintended large deletions and more complex rearrangements. In the present study, we describe a prime editing-based method, PRIME-Del, which induces a deletion using a pair of prime editing sgRNAs (pegRNAs) that target opposite DNA strands, programming not only the sites that are nicked but also the outcome of the repair. PRIME-Del achieves markedly higher precision than CRISPR–Cas9 and sgRNA pairs in programming deletions up to 10 kb, with 1–30\% editing efficiency. PRIME-Del can also be used to couple genomic deletions with short insertions, enabling deletions with junctions that do not fall at protospacer-adjacent motif sites. Finally, extended expression of prime editing components can substantially enhance efficiency without compromising precision. We anticipate that PRIME-Del will be broadly useful for precise, flexible programming of genomic deletions, epitope tagging and, potentially, programming genomic rearrangements. Paired prime editing is used to precisely delete genomic sequences up to 10 kb.},
  archive      = {J_NBT},
  author       = {Choi, Junhong and Chen, Wei and Suiter, Chase C. and Lee, Choli and Chardon, Florence M. and Yang, Wei and Leith, Anh and Daza, Riza M. and Martin, Beth and Shendure, Jay},
  doi          = {10.1038/s41587-021-01025-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {218-226},
  shortjournal = {Nature Biotech.},
  title        = {Precise genomic deletions using paired prime editing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Unannotated proteins expand the MHC-i-restricted
immunopeptidome in cancer. <em>NBT</em>, <em>40</em>(2), 209–217. (<a
href="https://doi.org/10.1038/s41587-021-01021-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor-associated epitopes presented on MHC-I that can activate the immune system against cancer cells are typically identified from annotated protein-coding regions of the genome, but whether peptides originating from novel or unannotated open reading frames (nuORFs) can contribute to antitumor immune responses remains unclear. Here we show that peptides originating from nuORFs detected by ribosome profiling of malignant and healthy samples can be displayed on MHC-I of cancer cells, acting as additional sources of cancer antigens. We constructed a high-confidence database of translated nuORFs across tissues (nuORFdb) and used it to detect 3,555 translated nuORFs from MHC-I immunopeptidome mass spectrometry analysis, including peptides that result from somatic mutations in nuORFs of cancer samples as well as tumor-specific nuORFs translated in melanoma, chronic lymphocytic leukemia and glioblastoma. NuORFs are an unexplored pool of MHC-I-presented, tumor-specific peptides with potential as immunotherapy targets. New tumor epitopes are discovered by ribosome profiling and immunopeptidome mass spectrometry.},
  archive      = {J_NBT},
  author       = {Ouspenskaia, Tamara and Law, Travis and Clauser, Karl R. and Klaeger, Susan and Sarkizova, Siranush and Aguet, François and Li, Bo and Christian, Elena and Knisbacher, Binyamin A. and Le, Phuong M. and Hartigan, Christina R. and Keshishian, Hasmik and Apffel, Annie and Oliveira, Giacomo and Zhang, Wandi and Chen, Sarah and Chow, Yuen Ting and Ji, Zhe and Jungreis, Irwin and Shukla, Sachet A. and Justesen, Sune and Bachireddy, Pavan and Kellis, Manolis and Getz, Gad and Hacohen, Nir and Keskin, Derin B. and Carr, Steven A. and Wu, Catherine J. and Regev, Aviv},
  doi          = {10.1038/s41587-021-01021-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {209-217},
  shortjournal = {Nature Biotech.},
  title        = {Unannotated proteins expand the MHC-I-restricted immunopeptidome in cancer},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Wireless closed-loop optogenetics across the entire
dorsoventral spinal cord in mice. <em>NBT</em>, <em>40</em>(2), 198–208.
(<a href="https://doi.org/10.1038/s41587-021-01019-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optoelectronic systems can exert precise control over targeted neurons and pathways throughout the brain in untethered animals, but similar technologies for the spinal cord are not well established. In the present study, we describe a system for ultrafast, wireless, closed-loop manipulation of targeted neurons and pathways across the entire dorsoventral spinal cord in untethered mice. We developed a soft stretchable carrier, integrating microscale light-emitting diodes (micro-LEDs), that conforms to the dura mater of the spinal cord. A coating of silicone–phosphor matrix over the micro-LEDs provides mechanical protection and light conversion for compatibility with a large library of opsins. A lightweight, head-mounted, wireless platform powers the micro-LEDs and performs low-latency, on-chip processing of sensed physiological signals to control photostimulation in a closed loop. We use the device to reveal the role of various neuronal subtypes, sensory pathways and supraspinal projections in the control of locomotion in healthy and spinal-cord injured mice. Optogenetics is applied to the entire mouse spinal cord.},
  archive      = {J_NBT},
  author       = {Kathe, Claudia and Michoud, Frédéric and Schönle, Philipp and Rowald, Andreas and Brun, Noé and Ravier, Jimmy and Furfaro, Ivan and Paggi, Valentina and Kim, Kyungjin and Soloukey, Sadaf and Asboth, Leonie and Hutson, Thomas H. and Jelescu, Ileana and Philippides, Antoine and Alwahab, Noaf and Gandar, Jérôme and Huber, Daniel and De Zeeuw, Chris I. and Barraud, Quentin and Huang, Qiuting and Lacour, Stéphanie P. and Courtine, Grégoire},
  doi          = {10.1038/s41587-021-01019-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {198-208},
  shortjournal = {Nature Biotech.},
  title        = {Wireless closed-loop optogenetics across the entire dorsoventral spinal cord in mice},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Compact RNA editors with small cas13 proteins. <em>NBT</em>,
<em>40</em>(2), 194–197. (<a
href="https://doi.org/10.1038/s41587-021-01030-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas13 systems have been developed for precise RNA editing, and can potentially be used therapeutically when temporary changes are desirable or when DNA editing is challenging. We have identified and characterized an ultrasmall family of Cas13b proteins—Cas13bt—that can mediate mammalian transcript knockdown. We have engineered compact variants of REPAIR and RESCUE RNA editors by functionalizing Cas13bt with adenosine and cytosine deaminase domains, and demonstrated packaging of the editors within a single adeno-associated virus. Compact Cas13 proteins are used to design RNA editors that fit into a single adeno-associated virus.},
  archive      = {J_NBT},
  author       = {Kannan, Soumya and Altae-Tran, Han and Jin, Xin and Madigan, Victoria J. and Oshiro, Rachel and Makarova, Kira S. and Koonin, Eugene V. and Zhang, Feng},
  doi          = {10.1038/s41587-021-01030-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {194-197},
  shortjournal = {Nature Biotech.},
  title        = {Compact RNA editors with small cas13 proteins},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). CRISPR prime editing with ribonucleoprotein complexes in
zebrafish and primary human cells. <em>NBT</em>, <em>40</em>(2),
189–193. (<a href="https://doi.org/10.1038/s41587-021-00901-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editors have been delivered using DNA or RNA vectors. Here we demonstrate prime editing with purified ribonucleoprotein complexes. We introduced somatic mutations in zebrafish embryos with frequencies as high as 30\% and demonstrate germline transmission. We also observed unintended insertions, deletions and prime editing guide RNA (pegRNA) scaffold incorporations. In HEK293T and primary human T cells, prime editing with purified ribonucleoprotein complexes introduced desired edits with frequencies of up to 21 and 7.5\%, respectively. Prime editors are delivered as ribonucleoproteins to zebrafish embryos and human primary cells.},
  archive      = {J_NBT},
  author       = {Petri, Karl and Zhang, Weiting and Ma, Junyan and Schmidts, Andrea and Lee, Hyunho and Horng, Joy E. and Kim, Daniel Y. and Kurt, Ibrahim C. and Clement, Kendell and Hsu, Jonathan Y. and Pinello, Luca and Maus, Marcela V. and Joung, J. Keith and Yeh, Jing-Ruey Joanna},
  doi          = {10.1038/s41587-021-00901-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {189-193},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR prime editing with ribonucleoprotein complexes in zebrafish and primary human cells},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identification of tumor antigens with immunopeptidomics.
<em>NBT</em>, <em>40</em>(2), 175–188. (<a
href="https://doi.org/10.1038/s41587-021-01038-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of actionable tumor antigens is indispensable for the development of several cancer immunotherapies, including T cell receptor–transduced T cells and patient-specific mRNA or peptide vaccines. Most known tumor antigens have been identified through extensive molecular characterization and are considered canonical if they derive from protein-coding regions of the genome. By eluting human leukocyte antigen-bound peptides from tumors and subjecting these to mass spectrometry analysis, the peptides can be identified by matching the resulting spectra against reference databases. Recently, mass-spectrometry-based immunopeptidomics has enabled the discovery of noncanonical antigens—antigens derived from sequences outside protein-coding regions or generated by noncanonical antigen-processing mechanisms. Coupled with transcriptomics and ribosome profiling, this method enables the identification of thousands of noncanonical peptides, of which a substantial fraction may be detected exclusively in tumors. Spectral matching against the immense noncanonical reference may generate false positives. However, sensitive mass spectrometry, analytical validation and advanced bioinformatics solutions are expected to uncover the full landscape of presented antigens and clinically relevant targets. Chong et al. review how the integration of mass spectrometry with proteogenomic approaches can identify noncanonical antigens.},
  archive      = {J_NBT},
  author       = {Chong, Chloe and Coukos, George and Bassani-Sternberg, Michal},
  doi          = {10.1038/s41587-021-01038-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {175-188},
  shortjournal = {Nature Biotech.},
  title        = {Identification of tumor antigens with immunopeptidomics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). First rounders: Ingmar hoerr. <em>NBT</em>, <em>40</em>(2),
174. (<a href="https://doi.org/10.1038/s41587-022-01234-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01234-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {174},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Ingmar hoerr},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Back and forth with nanopore peptide sequencing.
<em>NBT</em>, <em>40</em>(2), 172–173. (<a
href="https://doi.org/10.1038/s41587-021-01205-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tagging peptides with DNA allows repeated reads via a helicase in a nanopore with reduced error rates.},
  archive      = {J_NBT},
  author       = {Wanunu, Meni},
  doi          = {10.1038/s41587-021-01205-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {172-173},
  shortjournal = {Nature Biotech.},
  title        = {Back and forth with nanopore peptide sequencing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Dreaming ideal protein structures. <em>NBT</em>,
<em>40</em>(2), 171–172. (<a
href="https://doi.org/10.1038/s41587-021-01196-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A deep learning algorithm for protein structure prediction is used in reverse for de novo protein design.},
  archive      = {J_NBT},
  author       = {Ferruz, Noelia and Höcker, Birte},
  doi          = {10.1038/s41587-021-01196-9},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {171-172},
  shortjournal = {Nature Biotech.},
  title        = {Dreaming ideal protein structures},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antivirals. <em>NBT</em>, <em>40</em>(2), 170. (<a
href="https://doi.org/10.1038/s41587-022-01218-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the synthesis, composition and uses of novel antiviral agents.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01218-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {170},
  shortjournal = {Nature Biotech.},
  title        = {Antivirals},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Recent orange and purple book legislation suggests a need to
bridge drug and biologic patent regimes. <em>NBT</em>, <em>40</em>(2),
167–169. (<a href="https://doi.org/10.1038/s41587-021-01204-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Additional changes could make the registries of drug patents published by the FDA, called the Orange and Purple Books, more useful and reduce barriers to effective competition from generic and biosimilar drugs.},
  archive      = {J_NBT},
  author       = {Walsh, Bryan S. and Darrow, Jonathan J. and Kesselheim, Aaron S.},
  doi          = {10.1038/s41587-021-01204-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {167-169},
  shortjournal = {Nature Biotech.},
  title        = {Recent orange and purple book legislation suggests a need to bridge drug and biologic patent regimes},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A python library for probabilistic analysis of single-cell
omics data. <em>NBT</em>, <em>40</em>(2), 163–166. (<a
href="https://doi.org/10.1038/s41587-021-01206-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Gayoso, Adam and Lopez, Romain and Xing, Galen and Boyeau, Pierre and Valiollah Pour Amiri, Valeh and Hong, Justin and Wu, Katherine and Jayasuriya, Michael and Mehlman, Edouard and Langevin, Maxime and Liu, Yining and Samaran, Jules and Misrachi, Gabriel and Nazaret, Achille and Clivio, Oscar and Xu, Chenling and Ashuach, Tal and Gabitto, Mariano and Lotfollahi, Mohammad and Svensson, Valentine and da Veiga Beltrame, Eduardo and Kleshchevnikov, Vitalii and Talavera-López, Carlos and Pachter, Lior and Theis, Fabian J. and Streets, Aaron and Jordan, Michael I. and Regier, Jeffrey and Yosef, Nir},
  doi          = {10.1038/s41587-021-01206-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {163-166},
  shortjournal = {Nature Biotech.},
  title        = {A python library for probabilistic analysis of single-cell omics data},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). 2021 — the bubble’s hangover. <em>NBT</em>, <em>40</em>(2),
151–153. (<a href="https://doi.org/10.1038/s41587-022-01215-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01215-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {151-153},
  shortjournal = {Nature Biotech.},
  title        = {2021 — the bubble’s hangover},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Drug pipeline 4Q21. <em>NBT</em>, <em>40</em>(2), 149–150.
(<a href="https://doi.org/10.1038/s41587-022-01216-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-022-01216-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {149-150},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 4Q21},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(2), 148. (<a
href="https://doi.org/10.1038/s41587-022-01223-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01223-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {148},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chroma to pioneer epigenetic editors. <em>NBT</em>,
<em>40</em>(2), 146. (<a
href="https://doi.org/10.1038/s41587-022-01236-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01236-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {146},
  shortjournal = {Nature Biotech.},
  title        = {Chroma to pioneer epigenetic editors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). First pig-to-human heart. <em>NBT</em>, <em>40</em>(2), 145.
(<a href="https://doi.org/10.1038/s41587-022-01238-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01238-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {145},
  shortjournal = {Nature Biotech.},
  title        = {First pig-to-human heart},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Fashion’s microbial dyeing machines. <em>NBT</em>,
<em>40</em>(2), 143. (<a
href="https://doi.org/10.1038/s41587-022-01237-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-022-01237-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {143},
  shortjournal = {Nature Biotech.},
  title        = {Fashion’s microbial dyeing machines},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The post-theranos world. <em>NBT</em>, <em>40</em>(2), 139.
(<a href="https://doi.org/10.1038/s41587-022-01242-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in technology and changes to healthcare during the pandemic may finally realize the vision of patient-centric blood testing espoused by disgraced Theranos CEO Elizabeth Holmes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-022-01242-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {139},
  shortjournal = {Nature Biotech.},
  title        = {The post-theranos world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Fresh from the biotech pipeline: Too much, too fast?
<em>NBT</em>, <em>40</em>(2), 155–162. (<a
href="https://doi.org/10.1038/s41587-022-01208-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {2021 witnessed regulators’ continued push to accelerate approvals and adjust to COVID-19, but has the pendulum swung too far in drug makers’ favor?},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-022-01208-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {155-162},
  shortjournal = {Nature Biotech.},
  title        = {Fresh from the biotech pipeline: Too much, too fast?},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). African coronavirus surveillance network provides early
warning for world. <em>NBT</em>, <em>40</em>(2), 147–148. (<a
href="https://doi.org/10.1038/d41587-022-00003-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As Omicron spread across the globe, South African labs were the first to detect it and flag it to the world. As Omicron spread across the globe, South African labs were the first to detect it and flag it to the world.},
  archive      = {J_NBT},
  author       = {Adepoju, Paul},
  doi          = {10.1038/d41587-022-00003-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {147-148},
  shortjournal = {Nature Biotech.},
  title        = {African coronavirus surveillance network provides early warning for world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Rejuvenation by controlled reprogramming is the latest
gambit in anti-aging. <em>NBT</em>, <em>40</em>(2), 144–146. (<a
href="https://doi.org/10.1038/d41587-022-00002-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Startups are betting that partial reprogramming with Yamanaka factors may lead to age reversal, but hurdles remain. Startups are betting that partial reprogramming with Yamanaka factors may lead to age reversal, but hurdles remain.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/d41587-022-00002-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {144-146},
  shortjournal = {Nature Biotech.},
  title        = {Rejuvenation by controlled reprogramming is the latest gambit in anti-aging},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). COVID-19 vaccine makers chase variant-ready vaccines.
<em>NBT</em>, <em>40</em>(2), 141–143. (<a
href="https://doi.org/10.1038/d41587-022-00001-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Omicron is prompting vaccine makers to look beyond SARS-CoV-2 spike protein and test whether T-cell immunity or multivalency can combat new variants Omicron is prompting vaccine makers to look beyond SARS-CoV-2 spike protein and test whether T-cell immunity or multivalency can combat new variants},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-022-00001-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {141-143},
  shortjournal = {Nature Biotech.},
  title        = {COVID-19 vaccine makers chase variant-ready vaccines},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022l). People. <em>NBT</em>, <em>40</em>(1), 138. (<a
href="https://doi.org/10.1038/s41587-021-01178-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01178-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {138},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Virtual meetings promise to eliminate geographical and
administrative barriers and increase accessibility, diversity and
inclusivity. <em>NBT</em>, <em>40</em>(1), 133–137. (<a
href="https://doi.org/10.1038/s41587-021-01176-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conference organizers must adopt a novel, comprehensive approach to ensure increased accessibility, diversity and inclusivity of post-pandemic conferences.},
  archive      = {J_NBT},
  author       = {Wu, Juncheng and Rajesh, Anushka and Huang, Yu-Ning and Chhugani, Karishma and Acharya, Rajesh and Peng, Kerui and Johnson, Ruth D. and Fiscutean, Andrada and Robles-Espinoza, Carla Daniela and De La Vega, Francisco M. and Bao, Riyue and Mangul, Serghei},
  doi          = {10.1038/s41587-021-01176-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {133-137},
  shortjournal = {Nature Biotech.},
  title        = {Virtual meetings promise to eliminate geographical and administrative barriers and increase accessibility, diversity and inclusivity},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Publisher correction: Top 20 translational researchers of
2020. <em>NBT</em>, <em>40</em>(1), 131. (<a
href="https://doi.org/10.1038/s41587-021-01183-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-021-01183-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {131},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Top 20 translational researchers of 2020},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Publisher correction: Sickle cell gene therapies approach
watershed. <em>NBT</em>, <em>40</em>(1), 131. (<a
href="https://doi.org/10.1038/s41587-021-01193-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-01193-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {131},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Sickle cell gene therapies approach watershed},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Publisher correction: Treg engineers take aim at
autoimmunity. <em>NBT</em>, <em>40</em>(1), 131. (<a
href="https://doi.org/10.1038/s41587-021-01200-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-01200-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {131},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Treg engineers take aim at autoimmunity},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mapping single-cell data to reference atlases by transfer
learning. <em>NBT</em>, <em>40</em>(1), 121–130. (<a
href="https://doi.org/10.1038/s41587-021-01001-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large single-cell atlases are now routinely generated to serve as references for analysis of smaller-scale studies. Yet learning from reference data is complicated by batch effects between datasets, limited availability of computational resources and sharing restrictions on raw data. Here we introduce a deep learning strategy for mapping query datasets on top of a reference called single-cell architectural surgery (scArches). scArches uses transfer learning and parameter optimization to enable efficient, decentralized, iterative reference building and contextualization of new datasets with existing references without sharing raw data. Using examples from mouse brain, pancreas, immune and whole-organism atlases, we show that scArches preserves biological state information while removing batch effects, despite using four orders of magnitude fewer parameters than de novo integration. scArches generalizes to multimodal reference mapping, allowing imputation of missing modalities. Finally, scArches retains coronavirus disease 2019 (COVID-19) disease variation when mapping to a healthy reference, enabling the discovery of disease-specific cell states. scArches will facilitate collaborative projects by enabling iterative construction, updating, sharing and efficient use of reference atlases. Single-cell data are readily integrated with cell atlases using scArches.},
  archive      = {J_NBT},
  author       = {Lotfollahi, Mohammad and Naghipourfar, Mohsen and Luecken, Malte D. and Khajavi, Matin and Büttner, Maren and Wagenstetter, Marco and Avsec, Žiga and Gayoso, Adam and Yosef, Nir and Interlandi, Marta and Rybakov, Sergei and Misharin, Alexander V. and Theis, Fabian J.},
  doi          = {10.1038/s41587-021-01001-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {121-130},
  shortjournal = {Nature Biotech.},
  title        = {Mapping single-cell data to reference atlases by transfer learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrated analysis of plasma and single immune cells
uncovers metabolic changes in individuals with COVID-19. <em>NBT</em>,
<em>40</em>(1), 110–120. (<a
href="https://doi.org/10.1038/s41587-021-01020-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A better understanding of the metabolic alterations in immune cells during severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection may elucidate the wide diversity of clinical symptoms experienced by individuals with coronavirus disease 2019 (COVID-19). Here, we report the metabolic changes associated with the peripheral immune response of 198 individuals with COVID-19 through an integrated analysis of plasma metabolite and protein levels as well as single-cell multiomics analyses from serial blood draws collected during the first week after clinical diagnosis. We document the emergence of rare but metabolically dominant T cell subpopulations and find that increasing disease severity correlates with a bifurcation of monocytes into two metabolically distinct subsets. This integrated analysis reveals a robust interplay between plasma metabolites and cell-type-specific metabolic reprogramming networks that is associated with disease severity and could predict survival. The immune cells of individuals with COVID-19 show metabolic changes.},
  archive      = {J_NBT},
  author       = {Lee, Jihoon W. and Su, Yapeng and Baloni, Priyanka and Chen, Daniel and Pavlovitch-Bedzyk, Ana Jimena and Yuan, Dan and Duvvuri, Venkata R. and Ng, Rachel H. and Choi, Jongchan and Xie, Jingyi and Zhang, Rongyu and Murray, Kim and Kornilov, Sergey and Smith, Brett and Magis, Andrew T. and Hoon, Dave S. B. and Hadlock, Jennifer J. and Goldman, Jason D. and Price, Nathan D. and Gottardo, Raphael and Davis, Mark M. and Hood, Leroy and Greenberg, Philip D. and Heath, James R.},
  doi          = {10.1038/s41587-021-01020-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {110-120},
  shortjournal = {Nature Biotech.},
  title        = {Integrated analysis of plasma and single immune cells uncovers metabolic changes in individuals with COVID-19},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Oral delivery of systemic monoclonal antibodies, peptides
and small molecules using gastric auto-injectors. <em>NBT</em>,
<em>40</em>(1), 103–109. (<a
href="https://doi.org/10.1038/s41587-021-01024-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oral administration provides a simple and non-invasive approach for drug delivery. However, due to poor absorption and swift enzymatic degradation in the gastrointestinal tract, a wide range of molecules must be parenterally injected to attain required doses and pharmacokinetics. Here we present an orally dosed liquid auto-injector capable of delivering up to 4-mg doses of a bioavailable drug with the rapid pharmacokinetics of an injection, reaching an absolute bioavailability of up to 80\% and a maximum plasma drug concentration within 30 min after dosing. This approach improves dosing efficiencies and pharmacokinetics an order of magnitude over our previously designed injector capsules and up to two orders of magnitude over clinically available and preclinical chemical permeation enhancement technologies. We administered the capsules to swine for delivery of clinically relevant doses of four commonly injected medications, including adalimumab, a GLP-1 analog, recombinant human insulin and epinephrine. These multi-day dosing experiments and oral administration in awake animal models support the translational potential of the system. Biologics are delivered by a pill that pricks the stomach wall.},
  archive      = {J_NBT},
  author       = {Abramson, Alex and Frederiksen, Morten Revsgaard and Vegge, Andreas and Jensen, Brian and Poulsen, Mette and Mouridsen, Brian and Jespersen, Mikkel Oliver and Kirk, Rikke Kaae and Windum, Jesper and Hubálek, František and Water, Jorrit J. and Fels, Johannes and Gunnarsson, Stefán B. and Bohr, Adam and Straarup, Ellen Marie and Ley, Mikkel Wennemoes Hvitfeld and Lu, Xiaoya and Wainer, Jacob and Collins, Joy and Tamang, Siddartha and Ishida, Keiko and Hayward, Alison and Herskind, Peter and Buckley, Stephen T. and Roxhed, Niclas and Langer, Robert and Rahbek, Ulrik and Traverso, Giovanni},
  doi          = {10.1038/s41587-021-01024-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {103-109},
  shortjournal = {Nature Biotech.},
  title        = {Oral delivery of systemic monoclonal antibodies, peptides and small molecules using gastric auto-injectors},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficient CRISPR editing with a hypercompact cas12f1 and
engineered guide RNAs delivered by adeno-associated virus. <em>NBT</em>,
<em>40</em>(1), 94–102. (<a
href="https://doi.org/10.1038/s41587-021-01009-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene therapy would benefit from a miniature CRISPR system that fits into the small adeno-associated virus (AAV) genome and has high cleavage activity and specificity in eukaryotic cells. One of the most compact CRISPR-associated nucleases yet discovered is the archaeal Un1Cas12f1. However, Un1Cas12f1 and its variants have very low activity in eukaryotic cells. In the present study, we redesigned the natural guide RNA of Un1Cas12f1 at five sites: the 5′ terminus of the trans-activating CRISPR RNA (tracrRNA), the tracrRNA–crRNA complementary region, a penta(uridinylate) sequence, the 3′ terminus of the crRNA and a disordered stem 2 region in the tracrRNA. These optimizations synergistically increased the average indel frequency by 867-fold. The optimized Un1Cas12f1 system enabled efficient, specific genome editing in human cells when delivered by plasmid vectors, PCR amplicons and AAV. As Un1Cas12f1 cleaves outside the protospacer, it can be used to create large deletions efficiently. The engineered Un1Cas12f1 system showed efficiency comparable to that of SpCas9 and specificity similar to that of AsCas12a. A miniature CRISPR system may enable genome editing using single AAV vectors.},
  archive      = {J_NBT},
  author       = {Kim, Do Yon and Lee, Jeong Mi and Moon, Su Bin and Chin, Hyun Jung and Park, Seyeon and Lim, Youjung and Kim, Daesik and Koo, Taeyoung and Ko, Jeong-Heon and Kim, Yong-Sam},
  doi          = {10.1038/s41587-021-01009-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {94-102},
  shortjournal = {Nature Biotech.},
  title        = {Efficient CRISPR editing with a hypercompact cas12f1 and engineered guide RNAs delivered by adeno-associated virus},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identifying transcriptional programs underlying cancer drug
response with TraCe-seq. <em>NBT</em>, <em>40</em>(1), 86–93. (<a
href="https://doi.org/10.1038/s41587-021-01005-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic and non-genetic heterogeneity within cancer cell populations represent major challenges to anticancer therapies. We currently lack robust methods to determine how preexisting and adaptive features affect cellular responses to therapies. Here, by conducting clonal fitness mapping and transcriptional characterization using expressed barcodes and single-cell RNA sequencing (scRNA-seq), we have developed tracking differential clonal response by scRNA-seq (TraCe-seq). TraCe-seq is a method that captures at clonal resolution the origin, fate and differential early adaptive transcriptional programs of cells in a complex population in response to distinct treatments. We used TraCe-seq to benchmark how next-generation dual epidermal growth factor receptor (EGFR) inhibitor–degraders compare to standard EGFR kinase inhibitors in EGFR-mutant lung cancer cells. We identified a loss of antigrowth activity associated with targeted degradation of EGFR protein and an essential role of the endoplasmic reticulum (ER) protein processing pathway in anti-EGFR therapeutic efficacy. Our results suggest that targeted degradation is not always superior to enzymatic inhibition and establish TraCe-seq as an approach to study how preexisting transcriptional programs affect treatment responses. Transcriptional mechanisms of drug response are identified with scRNA-seq and molecular barcoding.},
  archive      = {J_NBT},
  author       = {Chang, Matthew T. and Shanahan, Frances and Nguyen, Thi Thu Thao and Staben, Steven T. and Gazzard, Lewis and Yamazoe, Sayumi and Wertz, Ingrid E. and Piskol, Robert and Yang, Yeqing Angela and Modrusan, Zora and Haley, Benjamin and Evangelista, Marie and Malek, Shiva and Foster, Scott A. and Ye, Xin},
  doi          = {10.1038/s41587-021-01005-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {86-93},
  shortjournal = {Nature Biotech.},
  title        = {Identifying transcriptional programs underlying cancer drug response with TraCe-seq},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integration of spatial and single-cell transcriptomic data
elucidates mouse organogenesis. <em>NBT</em>, <em>40</em>(1), 74–85. (<a
href="https://doi.org/10.1038/s41587-021-01006-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular profiling of single cells has advanced our knowledge of the molecular basis of development. However, current approaches mostly rely on dissociating cells from tissues, thereby losing the crucial spatial context of regulatory processes. Here, we apply an image-based single-cell transcriptomics method, sequential fluorescence in situ hybridization (seqFISH), to detect mRNAs for 387 target genes in tissue sections of mouse embryos at the 8–12 somite stage. By integrating spatial context and multiplexed transcriptional measurements with two single-cell transcriptome atlases, we characterize cell types across the embryo and demonstrate that spatially resolved expression of genes not profiled by seqFISH can be imputed. We use this high-resolution spatial map to characterize fundamental steps in the patterning of the midbrain–hindbrain boundary (MHB) and the developing gut tube. We uncover axes of cell differentiation that are not apparent from single-cell RNA-sequencing (scRNA-seq) data, such as early dorsal–ventral separation of esophageal and tracheal progenitor populations in the gut tube. Our method provides an approach for studying cell fate decisions in complex tissues and development. Improved integration of spatial and single-cell transcriptomic data provides insights into mouse development.},
  archive      = {J_NBT},
  author       = {Lohoff, T. and Ghazanfar, S. and Missarova, A. and Koulena, N. and Pierson, N. and Griffiths, J. A. and Bardot, E. S. and Eng, C.-H. L. and Tyser, R. C. V. and Argelaguet, R. and Guibentif, C. and Srinivas, S. and Briscoe, J. and Simons, B. D. and Hadjantonakis, A.-K. and Göttgens, B. and Reik, W. and Nichols, J. and Cai, L. and Marioni, J. C.},
  doi          = {10.1038/s41587-021-01006-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {74-85},
  shortjournal = {Nature Biotech.},
  title        = {Integration of spatial and single-cell transcriptomic data elucidates mouse organogenesis},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell measurement of higher-order 3D genome
organization with scSPRITE. <em>NBT</em>, <em>40</em>(1), 64–73. (<a
href="https://doi.org/10.1038/s41587-021-00998-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although three-dimensional (3D) genome organization is central to many aspects of nuclear function, it has been difficult to measure at the single-cell level. To address this, we developed ‘single-cell split-pool recognition of interactions by tag extension’ (scSPRITE). scSPRITE uses split-and-pool barcoding to tag DNA fragments in the same nucleus and their 3D spatial arrangement. Because scSPRITE measures multiway DNA contacts, it generates higher-resolution maps within an individual cell than can be achieved by proximity ligation. We applied scSPRITE to thousands of mouse embryonic stem cells and detected known genome structures, including chromosome territories, active and inactive compartments, and topologically associating domains (TADs) as well as long-range inter-chromosomal structures organized around various nuclear bodies. We observe that these structures exhibit different levels of heterogeneity across the population, with TADs representing dynamic units of genome organization across cells. We expect that scSPRITE will be a critical tool for studying genome structure within heterogeneous populations. Long-range genome interactions are measured in thousands of single cells.},
  archive      = {J_NBT},
  author       = {Arrastia, Mary V. and Jachowicz, Joanna W. and Ollikainen, Noah and Curtis, Matthew S. and Lai, Charlotte and Quinodoz, Sofia A. and Selck, David A. and Ismagilov, Rustem F. and Guttman, Mitchell},
  doi          = {10.1038/s41587-021-00998-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {64-73},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell measurement of higher-order 3D genome organization with scSPRITE},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrating t cell receptor sequences and transcriptional
profiles by clonotype neighbor graph analysis (CoNGA). <em>NBT</em>,
<em>40</em>(1), 54–63. (<a
href="https://doi.org/10.1038/s41587-021-00989-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Links between T cell clonotypes, as defined by T cell receptor (TCR) sequences, and phenotype, as reflected in gene expression (GEX) profiles, surface protein expression and peptide:major histocompatibility complex binding, can reveal functional relationships beyond the features shared by clonally related cells. Here we present clonotype neighbor graph analysis (CoNGA), a graph theoretic approach that identifies correlations between GEX profile and TCR sequence through statistical analysis of GEX and TCR similarity graphs. Using CoNGA, we uncovered associations between TCR sequence and GEX profiles that include a previously undescribed ‘natural lymphocyte’ population of human circulating CD8+ T cells and a set of TCR sequence determinants of differentiation in thymocytes. These examples show that CoNGA might help elucidate complex relationships between TCR sequence and T cell phenotype in large, heterogeneous, single-cell datasets. Integrating T cell sequences and gene expression profiles uncovers functional subsets in single-cell datasets.},
  archive      = {J_NBT},
  author       = {Schattgen, Stefan A. and Guion, Kate and Crawford, Jeremy Chase and Souquette, Aisha and Barrio, Alvaro Martinez and Stubbington, Michael J. T. and Thomas, Paul G. and Bradley, Philip},
  doi          = {10.1038/s41587-021-00989-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {54-63},
  shortjournal = {Nature Biotech.},
  title        = {Integrating t cell receptor sequences and transcriptional profiles by clonotype neighbor graph analysis (CoNGA)},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Purification of multiplex oligonucleotide libraries by
synthesis and selection. <em>NBT</em>, <em>40</em>(1), 47–53. (<a
href="https://doi.org/10.1038/s41587-021-00988-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex oligonucleotide (oligo) libraries are essential materials for diverse applications in synthetic biology, pharmaceutical production, nanotechnology and DNA-based data storage. However, the error rates in synthesizing complex oligo libraries can be substantial, leading to increment in cost and labor for the applications. As most synthesis errors arise from faulty insertions and deletions, we developed a length-based method with single-base resolution for purification of complex libraries containing oligos of identical or different lengths. Our method—purification of multiplex oligonucleotide libraries by synthesis and selection—can be performed either step-by-step manually or using a next-generation sequencer. When applied to a digital data-encoded library containing oligos of identical length, the method increased the purity of full-length oligos from 83\% to 97\%. We also show that libraries encoding the complementarity-determining region H3 with three different lengths (with an empirically achieved diversity &amp;gt;106) can be simultaneously purified in one pot, increasing the in-frame oligo fraction from 49.6\% to 83.5\%. Accurate oligonucleotide libraries are produced by synthesis and selection.},
  archive      = {J_NBT},
  author       = {Choi, Hansol and Choi, Yeongjae and Choi, Jaewon and Lee, Amos Chungwon and Yeom, Huiran and Hyun, Jinwoo and Ryu, Taehoon and Kwon, Sunghoon},
  doi          = {10.1038/s41587-021-00988-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {47-53},
  shortjournal = {Nature Biotech.},
  title        = {Purification of multiplex oligonucleotide libraries by synthesis and selection},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multiplexed direct detection of barcoded protein reporters
on a nanopore array. <em>NBT</em>, <em>40</em>(1), 42–46. (<a
href="https://doi.org/10.1038/s41587-021-01002-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detection of specific proteins using nanopores is currently challenging. To address this challenge, we developed a collection of over twenty nanopore-addressable protein tags engineered as reporters (NanoporeTERs, or NTERs). NTERs are constructed with a secretion tag, folded domain and a nanopore-targeting C-terminal tail in which arbitrary peptide barcodes can be encoded. We demonstrate simultaneous detection of up to nine NTERs expressed in bacterial or human cells using MinION nanopore sensor arrays. Reporter proteins are easily multiplexed using protein barcodes measured with nanopores.},
  archive      = {J_NBT},
  author       = {Cardozo, Nicolas and Zhang, Karen and Doroschak, Kathryn and Nguyen, Aerilynn and Siddiqui, Zoheb and Bogard, Nicholas and Strauss, Karin and Ceze, Luis and Nivala, Jeff},
  doi          = {10.1038/s41587-021-01002-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {42-46},
  shortjournal = {Nature Biotech.},
  title        = {Multiplexed direct detection of barcoded protein reporters on a nanopore array},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell immunology of SARS-CoV-2 infection.
<em>NBT</em>, <em>40</em>(1), 30–41. (<a
href="https://doi.org/10.1038/s41587-021-01131-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gaining a better understanding of the immune cell subsets and molecular factors associated with protective or pathological immunity against severe acute respiratory syndrome coronavirus (SARS-CoV)-2 could aid the development of vaccines and therapeutics for coronavirus disease 2019 (COVID-19). Single-cell technologies, such as flow cytometry, mass cytometry, single-cell transcriptomics and single-cell multi-omic profiling, offer considerable promise in dissecting the heterogeneity of immune responses among individual cells and uncovering the molecular mechanisms of COVID-19 pathogenesis. Single-cell immune-profiling studies reported to date have identified innate and adaptive immune cell subsets that correlate with COVID-19 disease severity, as well as immunological factors and pathways of potential relevance to the development of vaccines and treatments for COVID-19. For facilitation of integrative studies and meta-analyses into the immunology of SARS-CoV-2 infection, we provide standardized, download-ready versions of 21 published single-cell sequencing datasets (over 3.2 million cells in total) as well as an interactive visualization portal for data exploration. This Review provides an overview of existing studies using single-cell technologies to provide insights over the immune responses and molecular mechanisms at work in COVID-19.},
  archive      = {J_NBT},
  author       = {Tian, Yuan and Carpp, Lindsay N. and Miller, Helen E. R. and Zager, Michael and Newell, Evan W. and Gottardo, Raphael},
  doi          = {10.1038/s41587-021-01131-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {30-41},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell immunology of SARS-CoV-2 infection},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting disease variants using biodiversity and machine
learning. <em>NBT</em>, <em>40</em>(1), 27–28. (<a
href="https://doi.org/10.1038/s41587-021-01187-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An algorithm that uses evolutionary data to predict disease variants makes a case for embracing computational evidence for clinical interpretation of genetic variation.},
  archive      = {J_NBT},
  author       = {Arnedo-Pac, Claudia and Lopez-Bigas, Nuria and Muiños, Ferran},
  doi          = {10.1038/s41587-021-01187-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {27-28},
  shortjournal = {Nature Biotech.},
  title        = {Predicting disease variants using biodiversity and machine learning},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cancer diagnostics. <em>NBT</em>, <em>40</em>(1), 26. (<a
href="https://doi.org/10.1038/s41587-021-01181-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the detection, monitoring and treatment of cancer.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01181-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {26},
  shortjournal = {Nature Biotech.},
  title        = {Cancer diagnostics},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The characteristics of patents impacting availability of
biosimilars. <em>NBT</em>, <em>40</em>(1), 22–25. (<a
href="https://doi.org/10.1038/s41587-021-01170-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The large number of biologic drug patents in the United States has contributed to delays in biosimilar availability.},
  archive      = {J_NBT},
  author       = {Van de Wiele, Victor L. and Beall, Reed F. and Kesselheim, Aaron S. and Sarpatwari, Ameet},
  doi          = {10.1038/s41587-021-01170-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {22-25},
  shortjournal = {Nature Biotech.},
  title        = {The characteristics of patents impacting availability of biosimilars},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Expanding global access to genetic therapies. <em>NBT</em>,
<em>40</em>(1), 20–21. (<a
href="https://doi.org/10.1038/s41587-021-01191-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is time to rethink intellectual property and pricing practices that prevent global access to genetic therapies.},
  archive      = {J_NBT},
  author       = {Muigai, Anne W. T.},
  doi          = {10.1038/s41587-021-01191-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {20-21},
  shortjournal = {Nature Biotech.},
  title        = {Expanding global access to genetic therapies},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The DxConnect virtual biobank connects diagnostic
researchers to clinical samples. <em>NBT</em>, <em>40</em>(1), 18–19.
(<a href="https://doi.org/10.1038/s41587-021-01168-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ongarello, Stefano and Fernández Suárez, Marta and Betsou, Fay},
  doi          = {10.1038/s41587-021-01168-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {18-19},
  shortjournal = {Nature Biotech.},
  title        = {The DxConnect virtual biobank connects diagnostic researchers to clinical samples},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Constructive principles for gene editing oversight.
<em>NBT</em>, <em>40</em>(1), 17–18. (<a
href="https://doi.org/10.1038/s41587-021-01189-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {May, Mike and Giddings, L. Val and DeLisi, Charles and Drell, Daniel and Patrinos, Ari and Hirsch, Stanley and Roberts, Richard J.},
  doi          = {10.1038/s41587-021-01189-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {17-18},
  shortjournal = {Nature Biotech.},
  title        = {Constructive principles for gene editing oversight},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). Biotech news from around the world. <em>NBT</em>,
<em>40</em>(1), 11. (<a
href="https://doi.org/10.1038/s41587-021-01184-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01184-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {11},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Synbio salvages alcohol-free beer. <em>NBT</em>,
<em>40</em>(1), 8. (<a
href="https://doi.org/10.1038/s41587-021-01202-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-021-01202-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nature Biotech.},
  title        = {Synbio salvages alcohol-free beer},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Little black carbon-capture dress. <em>NBT</em>,
<em>40</em>(1), 7. (<a
href="https://doi.org/10.1038/s41587-021-01199-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01199-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {7},
  shortjournal = {Nature Biotech.},
  title        = {Little black carbon-capture dress},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). FDA go-ahead for myasthenia gravis agent. <em>NBT</em>,
<em>40</em>(1), 4. (<a
href="https://doi.org/10.1038/s41587-021-01198-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01198-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nature Biotech.},
  title        = {FDA go-ahead for myasthenia gravis agent},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Insulin — the new battleground for drug pricing.
<em>NBT</em>, <em>40</em>(1), 1. (<a
href="https://doi.org/10.1038/s41587-021-01203-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Outrage over the cost of insulin is driving drug-pricing reform. Industry must do more to support patients dealing with spiraling out-of-pocket costs for biotech medicines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01203-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nature Biotech.},
  title        = {Insulin — the new battleground for drug pricing},
  volume       = {40},
  year         = {2022},
}
</textarea>
</details></li>
</ul>

</body>
</html>
