<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATCS_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs---201">NATCS - 201</h2>
<ul>
<li><details>
<summary>
(2022b). Publisher correction: Defining shapes of two-dimensional
crystals with undefinable edge energies. <em>NATCS</em>, <em>2</em>(12),
866. (<a href="https://doi.org/10.1038/s43588-022-00385-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Wang, Luqing and Shirodkar, Sharmila N. and Zhang, Zhuhua and Yakobson, Boris I.},
  doi          = {10.1038/s43588-022-00385-z},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {866},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Defining shapes of two-dimensional crystals with undefinable edge energies},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Unconstrained generation of synthetic antibody–antigen
structures to guide machine learning methodology for antibody
specificity prediction. <em>NATCS</em>, <em>2</em>(12), 845–865. (<a
href="https://doi.org/10.1038/s43588-022-00372-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) is a key technology for accurate prediction of antibody–antigen binding. Two orthogonal problems hinder the application of ML to antibody-specificity prediction and the benchmarking thereof: the lack of a unified ML formalization of immunological antibody-specificity prediction problems and the unavailability of large-scale synthetic datasets to benchmark real-world relevant ML methods and dataset design. Here we developed the Absolut! software suite that enables parameter-based unconstrained generation of synthetic lattice-based three-dimensional antibody–antigen-binding structures with ground-truth access to conformational paratope, epitope and affinity. We formalized common immunological antibody-specificity prediction problems as ML tasks and confirmed that for both sequence- and structure-based tasks, accuracy-based rankings of ML methods trained on experimental data hold for ML methods trained on Absolut!-generated data. The Absolut! framework has the potential to enable real-world relevant development and benchmarking of ML strategies for biotherapeutics design. The Absolut! framework can generate synthetic three-dimensional antibody–antigen structures to assist machine learning and dataset construction for antibody design. Most importantly, the relative machine learning performance learnt on Absolut! datasets is shown to transfer to experimental datasets.},
  archive      = {J_NATCS},
  author       = {Robert, Philippe A. and Akbar, Rahmad and Frank, Robert and Pavlović, Milena and Widrich, Michael and Snapkov, Igor and Slabodkin, Andrei and Chernigovskaya, Maria and Scheffer, Lonneke and Smorodina, Eva and Rawat, Puneet and Mehta, Brij Bhushan and Vu, Mai Ha and Mathisen, Ingvild Frøberg and Prósz, Aurél and Abram, Krzysztof and Olar, Alex and Miho, Enkelejda and Haug, Dag Trygve Tryslew and Lund-Johansen, Fridtjof and Hochreiter, Sepp and Haff, Ingrid Hobæk and Klambauer, Günter and Sandve, Geir Kjetil and Greiff, Victor},
  doi          = {10.1038/s43588-022-00372-4},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {845-865},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unconstrained generation of synthetic antibody–antigen structures to guide machine learning methodology for antibody specificity prediction},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Dimensionally consistent learning with buckingham pi.
<em>NATCS</em>, <em>2</em>(12), 834–844. (<a
href="https://doi.org/10.1038/s43588-022-00355-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the absence of governing equations, dimensional analysis is a robust technique for extracting insights and finding symmetries in physical systems. Given measurement variables and parameters, the Buckingham Pi theorem provides a procedure for finding a set of dimensionless groups that spans the solution space, although this set is not unique. We propose an automated approach using the symmetric and self-similar structure of available measurement data to discover the dimensionless groups that best collapse these data to a lower dimensional space according to an optimal fit. We develop three data-driven techniques that use the Buckingham Pi theorem as a constraint: (1) a constrained optimization problem with a non-parametric input–output fitting function, (2) a deep learning algorithm (BuckiNet) that projects the input parameter space to a lower dimension in the first layer and (3) a technique based on sparse identification of nonlinear dynamics to discover dimensionless equations whose coefficients parameterize the dynamics. We explore the accuracy, robustness and computational complexity of these methods and show that they successfully identify dimensionless groups in three example problems: a bead on a rotating hoop, a laminar boundary layer and Rayleigh–Bénard convection. Three machine learning methods are developed for discovering physically meaningful dimensionless groups and scaling parameters from data, with the Buckingham Pi theorem as a constraint.},
  archive      = {J_NATCS},
  author       = {Bakarji, Joseph and Callaham, Jared and Brunton, Steven L. and Kutz, J. Nathan},
  doi          = {10.1038/s43588-022-00355-5},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {834-844},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dimensionally consistent learning with buckingham pi},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Discovering and forecasting extreme events via active
learning in neural operators. <em>NATCS</em>, <em>2</em>(12), 823–833.
(<a href="https://doi.org/10.1038/s43588-022-00376-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extreme events in society and nature, such as pandemic spikes, rogue waves or structural failures, can have catastrophic consequences. Characterizing extremes is difficult, as they occur rarely, arise from seemingly benign conditions, and belong to complex and often unknown infinite-dimensional systems. Such challenges render attempts at characterizing them moot. We address each of these difficulties by combining output-weighted training schemes in Bayesian experimental design (BED) with an ensemble of deep neural operators. This model-agnostic framework pairs a BED scheme that actively selects data for quantifying extreme events with an ensemble of deep neural operators that approximate infinite-dimensional nonlinear operators. We show that not only does this framework outperform Gaussian processes, but that (1) shallow ensembles of just two members perform best; (2) extremes are uncovered regardless of the state of the initial data (that is, with or without extremes); (3) our method eliminates ‘double-descent’ phenomena; (4) the use of batches of suboptimal acquisition samples compared to step-by-step global optima does not hinder BED performance; and (5) Monte Carlo acquisition outperforms standard optimizers in high dimensions. Together, these conclusions form a scalable artificial intelligence (AI)-assisted experimental infrastructure that can efficiently infer and pinpoint critical situations across many domains, from physical to societal systems. This study presents a model-agnostic framework that pairs deep neural operators and Bayesian experimental design for the accurate prediction of extreme events, such as rogue waves, pandemic spikes and structural ship failures.},
  archive      = {J_NATCS},
  author       = {Pickering, Ethan and Guth, Stephen and Karniadakis, George Em and Sapsis, Themistoklis P.},
  doi          = {10.1038/s43588-022-00376-0},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {823-833},
  shortjournal = {Nat. Comput. Sci},
  title        = {Discovering and forecasting extreme events via active learning in neural operators},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fast multi-source nanophotonic simulations using augmented
partial factorization. <em>NATCS</em>, <em>2</em>(12), 815–822. (<a
href="https://doi.org/10.1038/s43588-022-00370-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerical solutions of Maxwell’s equations are indispensable for nanophotonics and electromagnetics but are constrained when it comes to large systems, especially multi-channel ones such as disordered media, aperiodic metasurfaces and densely packed photonic circuits where the many inputs require many large-scale simulations. Conventionally, before extracting the quantities of interest, Maxwell’s equations are first solved on every element of a discretization basis set that contains much more information than is typically needed. Furthermore, such simulations are often performed one input at a time, which can be slow and repetitive. Here we propose to bypass the full-basis solutions and directly compute the quantities of interest while also eliminating the repetition over inputs. We do so by augmenting the Maxwell operator with all the input source profiles and all the output projection profiles, followed by a single partial factorization that yields the entire generalized scattering matrix via the Schur complement, with no approximation beyond discretization. This method applies to any linear partial differential equation. Benchmarks show that this approach is 1,000–30,000,000 times faster than existing methods for two-dimensional systems with about 10,000,000 variables. As examples, we demonstrate simulations of entangled photon backscattering from disorder and high-numerical-aperture metalenses that are thousands of wavelengths wide. Partial differential equations are typically solved on every element of a discretization basis before extracting the desired information, and each input requires one solution. In this study, a strategy is proposed to directly compute the quantities of interest, bypassing full-basis solutions and avoiding repetition over inputs.},
  archive      = {J_NATCS},
  author       = {Lin, Ho-Chun and Wang, Zeyu and Hsu, Chia Wei},
  doi          = {10.1038/s43588-022-00370-6},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {815-822},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fast multi-source nanophotonic simulations using augmented partial factorization},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-sequence protein structure prediction using
supervised transformer protein language models. <em>NATCS</em>,
<em>2</em>(12), 804–814. (<a
href="https://doi.org/10.1038/s43588-022-00373-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Significant progress has been made in protein structure prediction in recent years. However, it remains challenging for AlphaFold2 and other deep learning-based methods to predict protein structure with single-sequence input. Here we introduce trRosettaX-Single, an automated algorithm for single-sequence protein structure prediction. It incorporates the sequence embedding from a supervised transformer protein language model into a multi-scale network enhanced by knowledge distillation to predict inter-residue two-dimensional geometry, which is then used to reconstruct three-dimensional structures via energy minimization. Benchmark tests show that trRosettaX-Single outperforms AlphaFold2 and RoseTTAFold on orphan proteins and works well on human-designed proteins (with an average template modeling score (TM-score) of 0.79). An experimental test shows that the full trRosettaX-Single pipeline is two times faster than AlphaFold2, using much fewer computing resources (&amp;lt;10\%). On 2,000 designed proteins from network hallucination, trRosettaX-Single generates structure models with high confidence. As a demonstration, trRosettaX-Single is applied to missense mutation analysis. These data suggest that trRosettaX-Single may find potential applications in protein design and related studies. In this study, a supervised protein language model is proposed to predict protein structure from a single sequence. It achieves state-of-the-art accuracy on orphan proteins and is competitive with other methods on human-designed proteins.},
  archive      = {J_NATCS},
  author       = {Wang, Wenkai and Peng, Zhenling and Yang, Jianyi},
  doi          = {10.1038/s43588-022-00373-3},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {804-814},
  shortjournal = {Nat. Comput. Sci},
  title        = {Single-sequence protein structure prediction using supervised transformer protein language models},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Symphonizing pileup and full-alignment for deep
learning-based long-read variant calling. <em>NATCS</em>,
<em>2</em>(12), 797–803. (<a
href="https://doi.org/10.1038/s43588-022-00387-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning-based variant callers are becoming the standard and have achieved superior single nucleotide polymorphisms calling performance using long reads. Here we present Clair3, which leverages two major method categories: pileup calling handles most variant candidates with speed, and full-alignment tackles complicated candidates to maximize precision and recall. Clair3 runs faster than any of the other state-of-the-art variant callers and demonstrates improved performance, especially at lower coverage. Leveraging both the simple pileup input and full-alignment input, small variant calling using noisy long reads has improved speed and accuracy.},
  archive      = {J_NATCS},
  author       = {Zheng, Zhenxian and Li, Shumin and Su, Junhao and Leung, Amy Wing-Sze and Lam, Tak-Wah and Luo, Ruibang},
  doi          = {10.1038/s43588-022-00387-x},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {797-803},
  shortjournal = {Nat. Comput. Sci},
  title        = {Symphonizing pileup and full-alignment for deep learning-based long-read variant calling},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mathematical modeling of cancer immunotherapy for
personalized clinical translation. <em>NATCS</em>, <em>2</em>(12),
785–796. (<a href="https://doi.org/10.1038/s43588-022-00377-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Encouraging advances are being made in cancer immunotherapy modeling, especially in the key areas of developing personalized treatment strategies based on individual patient parameters, predicting treatment outcomes and optimizing immunotherapy synergy when used in combination with other treatment approaches. Here we present a focused review of the most recent mathematical modeling work on cancer immunotherapy with a focus on clinical translatability. It can be seen that this field is transitioning from pure basic science to applications that can make impactful differences in patients’ lives. We discuss how researchers are integrating experimental and clinical data to fully inform models so that they can be applied for clinical predictions, and present the challenges that remain to be overcome if widespread clinical adaptation is to be realized. Lastly, we discuss the most promising future applications and areas that are expected to be the focus of extensive upcoming modeling studies. Immunotherapy has begun to make a transformative impact on oncology practice, and mathematical modeling has been used to provide quantitative insights into this field. This Review discusses how models are being designed for direct clinical integration to improve the success rate of immunotherapy.},
  archive      = {J_NATCS},
  author       = {Butner, Joseph D. and Dogra, Prashant and Chung, Caroline and Pasqualini, Renata and Arap, Wadih and Lowengrub, John and Cristini, Vittorio and Wang, Zhihui},
  doi          = {10.1038/s43588-022-00377-z},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {785-796},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mathematical modeling of cancer immunotherapy for personalized clinical translation},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The meaning emerging from combining words can be detected in
space but not time. <em>NATCS</em>, <em>2</em>(12), 783–784. (<a
href="https://doi.org/10.1038/s43588-022-00361-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We used computational models built using neural networks to predict what brain areas process the new meaning that emerges when words are combined. The brain activity evoked by this composed meaning was detected only with some brain recording modalities, a finding that might have consequences for brain–computer interfaces.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00361-7},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {783-784},
  shortjournal = {Nat. Comput. Sci},
  title        = {The meaning emerging from combining words can be detected in space but not time},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Simulations that capture antigen–antibody complexity.
<em>NATCS</em>, <em>2</em>(12), 781–782. (<a
href="https://doi.org/10.1038/s43588-022-00379-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antigen–antibody prediction remains a complex computational challenge. Simulations with the new Absolut! package provide novel insights into the models and datasets tackling this problem.},
  archive      = {J_NATCS},
  author       = {Meysman, Pieter},
  doi          = {10.1038/s43588-022-00379-x},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {781-782},
  shortjournal = {Nat. Comput. Sci},
  title        = {Simulations that capture antigen–antibody complexity},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Synthesizing domain science with machine learning.
<em>NATCS</em>, <em>2</em>(12), 779–780. (<a
href="https://doi.org/10.1038/s43588-022-00358-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand whether or not the design of machine learning systems can integrate domain expertise, a recent work proposes methodologies to synthesize domain science with machine learning, which shows added benefits.},
  archive      = {J_NATCS},
  author       = {del Rosario, Zachary and del Rosario, Mason},
  doi          = {10.1038/s43588-022-00358-2},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {779-780},
  shortjournal = {Nat. Comput. Sci},
  title        = {Synthesizing domain science with machine learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficient simulators for multi-source nanophotonics.
<em>NATCS</em>, <em>2</em>(12), 777–778. (<a
href="https://doi.org/10.1038/s43588-022-00381-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method for making large-scale nanophotonic simulations more computationally efficient is proposed, enabling a wide range of studies to be less time- and memory-intensive.},
  archive      = {J_NATCS},
  author       = {Liu, Haitao},
  doi          = {10.1038/s43588-022-00381-3},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {777-778},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient simulators for multi-source nanophotonics},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting protein structure from single sequences.
<em>NATCS</em>, <em>2</em>(12), 775–776. (<a
href="https://doi.org/10.1038/s43588-022-00378-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As artificial intelligence begins to profoundly impact structural biology, one of the next challenges is to predict protein structures from individual sequences alone. A deep learning model addresses this challenge by representing single sequences with protein language models and distilling knowledge from multi-sequence structure predictors.},
  archive      = {J_NATCS},
  author       = {Shen, Yang},
  doi          = {10.1038/s43588-022-00378-y},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {775-776},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting protein structure from single sequences},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Artificial agents that negotiate and reach agreements.
<em>NATCS</em>, <em>2</em>(12), 774. (<a
href="https://doi.org/10.1038/s43588-022-00386-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Omelchenko, Iryna},
  doi          = {10.1038/s43588-022-00386-y},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {774},
  shortjournal = {Nat. Comput. Sci},
  title        = {Artificial agents that negotiate and reach agreements},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Seamless sharing and peer review of code. <em>NATCS</em>,
<em>2</em>(12), 773. (<a
href="https://doi.org/10.1038/s43588-022-00388-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent changes to our submission system, including a better integration with the Code Ocean platform, make the code peer review process more effortless for authors and referees.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00388-w},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {773},
  shortjournal = {Nat. Comput. Sci},
  title        = {Seamless sharing and peer review of code},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Computational design of quantum defects
in two-dimensional materials. <em>NATCS</em>, <em>2</em>(11), 772. (<a
href="https://doi.org/10.1038/s43588-022-00346-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Ping, Yuan and Smart, Tyler J.},
  doi          = {10.1038/s43588-022-00346-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {772},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Computational design of quantum defects in two-dimensional materials},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Forced and spontaneous symmetry breaking
in cell polarization. <em>NATCS</em>, <em>2</em>(11), 771. (<a
href="https://doi.org/10.1038/s43588-022-00345-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Miller, Pearson W. and Fortunato, Daniel and Muratov, Cyrill and Greengard, Leslie and Shvartsman, Stanislav},
  doi          = {10.1038/s43588-022-00345-7},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {771},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Forced and spontaneous symmetry breaking in cell polarization},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pathway trajectory analysis with tensor imputation reveals
drug-induced single-cell transcriptomic landscape. <em>NATCS</em>,
<em>2</em>(11), 758–770. (<a
href="https://doi.org/10.1038/s43588-022-00352-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide identification of single-cell transcriptomic responses of drugs in various human cells is a challenging issue in medical and pharmaceutical research. Here we present a computational method, tensor-based imputation of gene-expression data at the single-cell level (TIGERS), which reveals the drug-induced single-cell transcriptomic landscape. With this algorithm, we predict missing drug-induced single-cell gene-expression data with tensor imputation, and identify trajectories of regulated pathways considering intercellular heterogeneity. Tensor imputation outperformed existing imputation methods for data completion, and provided cell-type-specific transcriptomic responses for unobserved drugs. For example, TIGERS correctly predicted the cell-type-specific expression of maker genes for pancreatic islets. Pathway trajectory analysis of the imputed gene-expression profiles of all combinations of drugs and human cells identified single-cell-specific drug activities and pathway trajectories that reflect drug-induced changes in pathway regulation. The proposed method is expected to expand our understanding of the single-cell mechanisms of drugs at the pathway level. A method is proposed to predict drug-induced single-cell gene-expression profiles with tensor imputation, detect cell type-specific maker genes, and identify the trajectories of regulated pathways considering intercellular heterogeneity.},
  archive      = {J_NATCS},
  author       = {Iwata, Michio and Mutsumine, Hiroaki and Nakayama, Yusuke and Suita, Naomasa and Yamanishi, Yoshihiro},
  doi          = {10.1038/s43588-022-00352-8},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {758-770},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pathway trajectory analysis with tensor imputation reveals drug-induced single-cell transcriptomic landscape},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Combining computational controls with natural text reveals
aspects of meaning composition. <em>NATCS</em>, <em>2</em>(11), 745–757.
(<a href="https://doi.org/10.1038/s43588-022-00354-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To study a core component of human intelligence—our ability to combine the meaning of words—neuroscientists have looked to linguistics. However, linguistic theories are insufficient to account for all brain responses reflecting linguistic composition. In contrast, we adopt a data-driven approach to study the composed meaning of words beyond their individual meaning, which we term ‘supra-word meaning’. We construct a computational representation for supra-word meaning and study its brain basis through brain recordings from two complementary imaging modalities. Using functional magnetic resonance imaging, we reveal that hubs that are thought to process lexical meaning also maintain supra-word meaning, suggesting a common substrate for lexical and combinatorial semantics. Surprisingly, we cannot detect supra-word meaning in magnetoencephalography, which suggests that composed meaning might be maintained through a different neural mechanism than the synchronized firing of pyramidal cells. This sensitivity difference has implications for past neuroimaging results and future wearable neurotechnology. A neural network-based language model of supra-word meaning, that is, the combined meaning of words in a sentence, is proposed. Analysis of functional magnetic resonance imaging and magnetoencephalography data helps identify the regions of the brain responsible for understanding this meaning.},
  archive      = {J_NATCS},
  author       = {Toneva, Mariya and Mitchell, Tom M. and Wehbe, Leila},
  doi          = {10.1038/s43588-022-00354-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {745-757},
  shortjournal = {Nat. Comput. Sci},
  title        = {Combining computational controls with natural text reveals aspects of meaning composition},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A structural optimization algorithm with stochastic forces
and stresses. <em>NATCS</em>, <em>2</em>(11), 736–744. (<a
href="https://doi.org/10.1038/s43588-022-00350-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We propose an algorithm for optimizations in which the gradients contain stochastic noise. This arises, for example, in structural optimizations when computations of forces and stresses rely on methods involving Monte Carlo sampling, such as quantum Monte Carlo or neural network states, or are performed on quantum devices that have intrinsic noise. Our proposed algorithm is based on the combination of two ingredients: an update rule derived from the steepest-descent method, and a staged scheduling of the targeted statistical error and step size, with position averaging. We compare it with commonly applied algorithms, including some of the latest machine learning optimization methods, and show that the algorithm consistently performs efficiently and robustly under realistic conditions. Applying this algorithm, we achieve full-degree optimizations in solids using ab initio many-body computations, by auxiliary-field quantum Monte Carlo with plane waves and pseudopotentials. A potential metastable structure in Si is discovered using density-functional calculations with synthetic noisy forces. Energy gradients from accurate post-density-functional-theory methods often involve statistical uncertainties. This study proposes an algorithm for structural optimizations when using gradients originating from such methods.},
  archive      = {J_NATCS},
  author       = {Chen, Siyuan and Zhang, Shiwei},
  doi          = {10.1038/s43588-022-00350-w},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {736-744},
  shortjournal = {Nat. Comput. Sci},
  title        = {A structural optimization algorithm with stochastic forces and stresses},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Defining shapes of two-dimensional crystals with
undefinable edge energies. <em>NATCS</em>, <em>2</em>(11), 729–735. (<a
href="https://doi.org/10.1038/s43588-022-00347-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The equilibrium shape of crystals is a fundamental property of both aesthetic appeal and practical importance: the shape and its facets control the catalytic, light-emitting, sensing, magnetic and plasmonic behaviors. It is also a visible macro-manifestation of the underlying atomic-scale forces and chemical makeup, most conspicuous in two-dimensional (2D) materials of keen current interest. If the crystal surface/edge energy is known for different directions, its shape can be obtained by the geometric Wulff construction, a tenet of crystal physics; however, if symmetry is lacking, the crystal edge energy cannot be defined or calculated and thus its shape becomes elusive, presenting an insurmountable problem for theory. Here we show how one can proceed with auxiliary edge energies towards a constructive prediction, through well-planned computations, of a unique crystal shape. We demonstrate it for challenging materials such as SnSe, which is of C2v symmetry, and even AgNO2 of C1, which has no symmetry at all. The indeterminacy of edge or surface energy — unknowable for low-symmetry crystals — is avoided by an algebraic system complemented with closure equations, which enables computing algorithms to predict the equilibrium shape of any crystal.},
  archive      = {J_NATCS},
  author       = {Wang, Luqing and Shirodkar, Sharmila N. and Zhang, Zhuhua and Yakobson, Boris I.},
  doi          = {10.1038/s43588-022-00347-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {729-735},
  shortjournal = {Nat. Comput. Sci},
  title        = {Defining shapes of two-dimensional crystals with undefinable edge energies},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A universal graph deep learning interatomic potential for
the periodic table. <em>NATCS</em>, <em>2</em>(11), 718–728. (<a
href="https://doi.org/10.1038/s43588-022-00349-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interatomic potentials (IAPs), which describe the potential energy surface of atoms, are a fundamental input for atomistic simulations. However, existing IAPs are either fitted to narrow chemistries or too inaccurate for general applications. Here we report a universal IAP for materials based on graph neural networks with three-body interactions (M3GNet). The M3GNet IAP was trained on the massive database of structural relaxations performed by the Materials Project over the past ten years and has broad applications in structural relaxation, dynamic simulations and property prediction of materials across diverse chemical spaces. About 1.8 million materials from a screening of 31 million hypothetical crystal structures were identified to be potentially stable against existing Materials Project crystals based on M3GNet energies. Of the top 2,000 materials with the lowest energies above the convex hull, 1,578 were verified to be stable using density functional theory calculations. These results demonstrate a machine learning-accelerated pathway to the discovery of synthesizable materials with exceptional properties. A machine learning interatomic potential model is designed and trained on diverse crystal data comprising 89 elements, enabling materials discovery across a vast chemical space without retraining.},
  archive      = {J_NATCS},
  author       = {Chen, Chi and Ong, Shyue Ping},
  doi          = {10.1038/s43588-022-00349-3},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {718-728},
  shortjournal = {Nat. Comput. Sci},
  title        = {A universal graph deep learning interatomic potential for the periodic table},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Experimental quantum adversarial learning with programmable
superconducting qubits. <em>NATCS</em>, <em>2</em>(11), 711–717. (<a
href="https://doi.org/10.1038/s43588-022-00351-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum computing promises to enhance machine learning and artificial intelligence. However, recent theoretical works show that, similar to traditional classifiers based on deep classical neural networks, quantum classifiers would suffer from adversarial perturbations as well. Here we report an experimental demonstration of quantum adversarial learning with programmable superconducting qubits. We train quantum classifiers, which are built on variational quantum circuits consisting of ten transmon qubits featuring average lifetimes of 150 μs, and average fidelities of simultaneous single- and two-qubit gates above 99.94\% and 99.4\%, respectively, with both real-life images (for example, medical magnetic resonance imaging scans) and quantum data. We demonstrate that these well-trained classifiers (with testing accuracy up to 99\%) can be practically deceived by small adversarial perturbations, whereas an adversarial training process would substantially enhance their robustness to such perturbations. The vulnerability of quantum machine learning models against adversarial noises, together with a defense strategy way out of this dilemma, is demonstrated experimentally with a programmable superconducting quantum processor.},
  archive      = {J_NATCS},
  author       = {Ren, Wenhui and Li, Weikang and Xu, Shibo and Wang, Ke and Jiang, Wenjie and Jin, Feitong and Zhu, Xuhao and Chen, Jiachen and Song, Zixuan and Zhang, Pengfei and Dong, Hang and Zhang, Xu and Deng, Jinfeng and Gao, Yu and Zhang, Chuanyu and Wu, Yaozu and Zhang, Bing and Guo, Qiujiang and Li, Hekang and Wang, Zhen and Biamonte, Jacob and Song, Chao and Deng, Dong-Ling and Wang, H.},
  doi          = {10.1038/s43588-022-00351-9},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {711-717},
  shortjournal = {Nat. Comput. Sci},
  title        = {Experimental quantum adversarial learning with programmable superconducting qubits},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cumulative reciprocity can sustain cooperation in repeated
social interactions. <em>NATCS</em>, <em>2</em>(11), 709–710. (<a
href="https://doi.org/10.1038/s43588-022-00335-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In social interactions, individuals are often tempted to ‘free ride’ (benefit without paying) on other group members’ contributions. A new mathematical framework suggests a strategy based on cumulative reciprocity that can help to sustain mutual cooperation.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00335-9},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {709-710},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cumulative reciprocity can sustain cooperation in repeated social interactions},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell-specific drug activities are revealed by a
tensor imputation algorithm. <em>NATCS</em>, <em>2</em>(11), 707–708.
(<a href="https://doi.org/10.1038/s43588-022-00353-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a computational method to reveal the drug-induced single-cell transcriptomic landscape. This algorithm enabled us to impute unknown drug-induced single-cell gene expression profiles using tensor imputation, predict cell type-specific drug efficacy, detect cell-type-specific marker genes, and identify the trajectories of regulated biological pathways while considering intercellular heterogeneity.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00353-7},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {707-708},
  shortjournal = {Nat. Comput. Sci},
  title        = {Single-cell-specific drug activities are revealed by a tensor imputation algorithm},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting the shape of crystals with “unknowable” surface
energies. <em>NATCS</em>, <em>2</em>(11), 705–706. (<a
href="https://doi.org/10.1038/s43588-022-00356-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The surface energy cannot be assigned to each direction in low-symmetry crystals, making it impossible to predict their shapes by any known methods. Now, combining incomputable energies in an algebraic system, complemented by closure equations, it is possible to predict the equilibrium shape of any crystal.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00356-4},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {705-706},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting the shape of crystals with ‘unknowable’ surface energies},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Expanding materials science with universal many-body graph
neural networks. <em>NATCS</em>, <em>2</em>(11), 703–704. (<a
href="https://doi.org/10.1038/s43588-022-00360-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A universal interatomic potential for the periodic table has been developed by combining graph neural networks with three-body interactions. This M3GNet potential can perform structural relaxations, dynamic simulations and property predictions for materials across a diverse chemical space.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00360-8},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {703-704},
  shortjournal = {Nat. Comput. Sci},
  title        = {Expanding materials science with universal many-body graph neural networks},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Understanding the combined meaning of words. <em>NATCS</em>,
<em>2</em>(11), 701–702. (<a
href="https://doi.org/10.1038/s43588-022-00338-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent work uses a language model to gain insight into how the human brain understands the combined meaning of words in a sentence, and uncovers parts of the brain that contribute to this understanding.},
  archive      = {J_NATCS},
  author       = {Erk, Katrin},
  doi          = {10.1038/s43588-022-00338-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {701-702},
  shortjournal = {Nat. Comput. Sci},
  title        = {Understanding the combined meaning of words},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Robust quantum classifiers via NISQ adversarial learning.
<em>NATCS</em>, <em>2</em>(11), 699–700. (<a
href="https://doi.org/10.1038/s43588-022-00359-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The vulnerability of quantum machine learning is demonstrated on a superconducting quantum computer, together with a defense strategy based on noisy intermediate-scale quantum (NISQ) adversarial learning.},
  archive      = {J_NATCS},
  author       = {Banchi, Leonardo},
  doi          = {10.1038/s43588-022-00359-1},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {699-700},
  shortjournal = {Nat. Comput. Sci},
  title        = {Robust quantum classifiers via NISQ adversarial learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Learning about few-shot concept learning. <em>NATCS</em>,
<em>2</em>(11), 698. (<a
href="https://doi.org/10.1038/s43588-022-00367-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00367-1},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {698},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning about few-shot concept learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Computational models of social learning. <em>NATCS</em>,
<em>2</em>(11), 697. (<a
href="https://doi.org/10.1038/s43588-022-00364-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00364-4},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {697},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational models of social learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Reducing the network dimensionality. <em>NATCS</em>,
<em>2</em>(11), 696. (<a
href="https://doi.org/10.1038/s43588-022-00362-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Omelchenko, Iryna},
  doi          = {10.1038/s43588-022-00362-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {696},
  shortjournal = {Nat. Comput. Sci},
  title        = {Reducing the network dimensionality},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). On the cost–accuracy trade-off in electrolytes.
<em>NATCS</em>, <em>2</em>(11), 695. (<a
href="https://doi.org/10.1038/s43588-022-00366-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00366-2},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {695},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the cost–accuracy trade-off in electrolytes},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Making heat capacity prediction more reliable.
<em>NATCS</em>, <em>2</em>(11), 694. (<a
href="https://doi.org/10.1038/s43588-022-00363-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Jin, Sumin},
  doi          = {10.1038/s43588-022-00363-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {694},
  shortjournal = {Nat. Comput. Sci},
  title        = {Making heat capacity prediction more reliable},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Active learning for alloy design. <em>NATCS</em>,
<em>2</em>(11), 693. (<a
href="https://doi.org/10.1038/s43588-022-00365-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00365-3},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {693},
  shortjournal = {Nat. Comput. Sci},
  title        = {Active learning for alloy design},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Modeling excited-state molecular dynamics beyond the
born–oppenheimer regime. <em>NATCS</em>, <em>2</em>(11), 689–692. (<a
href="https://doi.org/10.1038/s43588-022-00357-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nonadiabatic molecular dynamics is the method of choice for modeling a wide range of excited-state phenomena. Although much progress has been made in improving the usability and efficiency of ground-state calculations, there are still challenges in translating this advance to the excited state.},
  archive      = {J_NATCS},
  author       = {Nelson, Tammie R. and Fernandez-Alberti, Sebastian and Tretiak, Sergei},
  doi          = {10.1038/s43588-022-00357-3},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {689-692},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling excited-state molecular dynamics beyond the Born–Oppenheimer regime},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pioneering quantum information science. <em>NATCS</em>,
<em>2</em>(11), 687–688. (<a
href="https://doi.org/10.1038/s43588-022-00368-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We discuss the research recognized by the most recent Nobel Prize in Physics, which has had important implications for quantum technologies.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00368-0},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {687-688},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pioneering quantum information science},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evolution of cooperation through cumulative reciprocity.
<em>NATCS</em>, <em>2</em>(10), 677–686. (<a
href="https://doi.org/10.1038/s43588-022-00334-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reciprocity is a simple principle for cooperation that explains many of the patterns of how humans seek and receive help from each other. To capture reciprocity, traditional models often assume that individuals use simple strategies with restricted memory. These memory-1 strategies are mathematically convenient, but they miss important aspects of human reciprocity, where defections can have lasting effects. Here we instead propose a strategy of cumulative reciprocity. Cumulative reciprocators count the imbalance of cooperation across their previous interactions with their opponent. They cooperate as long as this imbalance is sufficiently small. Using analytical and computational methods, we show that this strategy can sustain cooperation in the presence of errors, that it enforces fair outcomes and that it evolves in hostile environments. Using an economic experiment, we confirm that cumulative reciprocity is more predictive of human behaviour than several classical strategies. The basic principle of cumulative reciprocity is versatile and can be extended to a range of social dilemmas. A strategy for cooperation in repeated games, called cumulative reciprocity, is proposed. This strategy is robust with respect to errors, enforces fair outcomes, and evolves in environments that are usually hostile to cooperation.},
  archive      = {J_NATCS},
  author       = {Li, Juan and Zhao, Xiaowei and Li, Bing and Rossetti, Charlotte S. L. and Hilbe, Christian and Xia, Haoxiang},
  doi          = {10.1038/s43588-022-00334-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {677-686},
  shortjournal = {Nat. Comput. Sci},
  title        = {Evolution of cooperation through cumulative reciprocity},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Homeostatic coordination and up-regulation of neural
activity by activity-dependent myelination. <em>NATCS</em>,
<em>2</em>(10), 665–676. (<a
href="https://doi.org/10.1038/s43588-022-00315-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Activity-dependent myelination (ADM) is a fundamental dimension of brain plasticity through which myelin changes as a function of neural activity. Mediated by structural changes in glia, ADM notably regulates axonal conduction velocity. Yet, it remains unclear how neural activity impacts myelination to orchestrate the timing of neural signalling, and how ADM shapes neural activity. We developed a model of spiking neurons enhanced with neuron–oligodendrocyte feedback and examined the relationship between ADM and neural activity. We found that ADM implements a homeostatic gain control mechanism that enhances neural firing rates and correlations through the temporal coordination of action potentials as axon lengths increase. Stimuli engage ADM plasticity to trigger bidirectional and reversible changes in conduction delays, as may occur during learning. Furthermore, ADM was found to enhance information transmission under various types of time-varying stimuli. These results highlight the role of ADM in shaping neural activity and communication. Conduction of neural impulses along axons in the brain is sped up by a substance called myelin, which changes during development and learning. This study reveals that myelin remodelling coordinates and optimizes neuronal communication.},
  archive      = {J_NATCS},
  author       = {Talidou, Afroditi and Frankland, Paul W. and Mabbott, Donald and Lefebvre, Jérémie},
  doi          = {10.1038/s43588-022-00315-z},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {665-676},
  shortjournal = {Nat. Comput. Sci},
  title        = {Homeostatic coordination and up-regulation of neural activity by activity-dependent myelination},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Demand-driven design of bicycle infrastructure networks for
improved urban bikeability. <em>NATCS</em>, <em>2</em>(10), 655–664. (<a
href="https://doi.org/10.1038/s43588-022-00318-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cycling is crucial for sustainable urban transportation. Promoting cycling critically relies on sufficiently developed infrastructure; however, designing efficient bike path networks constitutes a complex problem that requires balancing multiple constraints. Here we propose a framework for generating efficient bike path networks, explicitly taking into account cyclists’ demand distribution and route choices based on safety preferences. By reversing the network formation, we iteratively remove bike paths from an initially complete bike path network and continually update cyclists’ route choices to create a sequence of networks adapted to the cycling demand. We illustrate the applicability of this demand-driven approach for two cities. A comparison of the resulting bike path networks with those created for homogenized demand enables us to quantify the importance of the demand distribution for network planning. The proposed framework may thus enable quantitative evaluation of the structure of current and planned cycling networks, and support the demand-driven design of efficient infrastructures. Designing efficient bike path networks requires balancing multiple constraints. In this study, a demand-driven inverse percolation approach is proposed to generate families of efficient bike path networks taking into account cyclist demand and safety preferences.},
  archive      = {J_NATCS},
  author       = {Steinacker, Christoph and Storch, David-Maximilian and Timme, Marc and Schröder, Malte},
  doi          = {10.1038/s43588-022-00318-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {655-664},
  shortjournal = {Nat. Comput. Sci},
  title        = {Demand-driven design of bicycle infrastructure networks for improved urban bikeability},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fast kinetic simulator for relativistic matter.
<em>NATCS</em>, <em>2</em>(10), 641–654. (<a
href="https://doi.org/10.1038/s43588-022-00333-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Relativistic kinetic theory is ubiquitous to several fields of modern physics, finding application at large scales in systems in astrophysical contexts, all of the way down to subnuclear scales and into the realm of quark–gluon plasmas. This motivates the quest for powerful and efficient computational methods that are able to accurately study fluid dynamics in the relativistic regime as well as the transition to beyond hydrodynamics—in principle all of the way down to ballistic regimes. We present a family of relativistic lattice kinetic schemes for the efficient simulation of relativistic flows in both strongly (fluid) and weakly (rarefied gas) interacting regimes. The method can deal with both massless and massive particles, thereby encompassing ultra- and mildly relativistic regimes alike. The computational performance of the method for the simulation of relativistic flows across the aforementioned regimes is discussed in detail, along with prospects of future applications. A family of lattice kinetic schemes is introduced for the simulation of relativistic flows. Taking advantage of GPU acceleration, the scheme allows one to efficiently probe both strongly and weakly interacting regimes, for massive and massless particles.},
  archive      = {J_NATCS},
  author       = {Ambruş, V. E. and Bazzanini, L. and Gabbana, A. and Simeoni, D. and Succi, S. and Tripiccione, R.},
  doi          = {10.1038/s43588-022-00333-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {641-654},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fast kinetic simulator for relativistic matter},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chiral topographic instability in shrinking spheres.
<em>NATCS</em>, <em>2</em>(10), 632–640. (<a
href="https://doi.org/10.1038/s43588-022-00332-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many biological structures exhibit intriguing morphological patterns adapted to environmental cues, which contribute to their important biological functions and also inspire material designs. Here, we report a chiral wrinkling topography in shrinking core–shell spheres, as observed in excessively dehydrated passion fruit and experimentally demonstrated in silicon core–shells under air extraction. Upon shrinkage deformation, the surface initially buckles into a buckyball pattern (periodic hexagons and pentagons) and then transforms into a chiral mode. The neighbouring chiral cellular patterns can further interact with each other, resulting in secondary symmetry breaking and the formation of two types of topological network. We develop a core–shell model and derive a universal scaling law to understand the underlying morphoelastic mechanism and to effectively describe and predict such chiral symmetry breaking far beyond the critical instability threshold. Moreover, we show experimentally that the chiral characteristic adapted to local perturbation can be harnessed to effectively and stably grasp small-sized objects of various shapes and made of different stiff and soft materials. Our results not only reveal chiral instability topographies, providing fundamental insights into the surface morphogenesis of the deformed core–shell spheres that are ubiquitous in the real world, but also demonstrate potential applications of adaptive grasping based on delicate chiral localization. This study reports a chiral instability topography in highly deformed core–shell spheres. A core–shell model and a scaling law are developed to understand its morphoelastic mechanism, which helps the design of a nature-inspired smart topographic gripper based on chiral localization.},
  archive      = {J_NATCS},
  author       = {Xu, Fan and Huang, Yangchao and Zhao, Shichen and Feng, Xi-Qiao},
  doi          = {10.1038/s43588-022-00332-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {632-640},
  shortjournal = {Nat. Comput. Sci},
  title        = {Chiral topographic instability in shrinking spheres},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Designing efficient urban bike path networks that meet the
needs of cyclists. <em>NATCS</em>, <em>2</em>(10), 630–631. (<a
href="https://doi.org/10.1038/s43588-022-00324-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Designing efficient bike path networks requires balancing multiple opposing constraints such as cost and safety. An adaptive demand-driven inverse percolation approach is proposed to generate efficient network structures by explicitly taking into account the demands of cyclists and their route choice behavior based on safety preferences.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00324-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {630-631},
  shortjournal = {Nat. Comput. Sci},
  title        = {Designing efficient urban bike path networks that meet the needs of cyclists},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The role of ADM in brain function. <em>NATCS</em>,
<em>2</em>(10), 628–629. (<a
href="https://doi.org/10.1038/s43588-022-00320-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In a recent study a phenomenological model was used to study the effects of activity-dependent myelination (ADM) on network activity and information transmission in the brain. The model explores how the conduction velocity of an axon — and thus the overall transmission delay — varies as a function of neural activity.},
  archive      = {J_NATCS},
  author       = {Chacron, Maurice J.},
  doi          = {10.1038/s43588-022-00320-2},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {628-629},
  shortjournal = {Nat. Comput. Sci},
  title        = {The role of ADM in brain function},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Simulating fluids, gases and everything in between.
<em>NATCS</em>, <em>2</em>(10), 626–627. (<a
href="https://doi.org/10.1038/s43588-022-00337-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The simulation of relativistic flows that can transit from a fluid-like to a gas-like substance poses challenges for computational methods. A lattice kinetic scheme is proposed to simulate such flows, which allows a computational probe of both strongly and weakly interacting regimes.},
  archive      = {J_NATCS},
  author       = {Romatschke, Paul},
  doi          = {10.1038/s43588-022-00337-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {626-627},
  shortjournal = {Nat. Comput. Sci},
  title        = {Simulating fluids, gases and everything in between},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bio-inspired adaptive grasper by chiral wrinkling.
<em>NATCS</em>, <em>2</em>(10), 624–625. (<a
href="https://doi.org/10.1038/s43588-022-00336-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The modeling of non-linear morphological changes in biological systems is a challenging task. Motivated by the observation of exotic pattern formation processes on fruit surfaces, a chiral wrinkling topology is disclosed as a mechanical structural instability, which is then exploited for the design of enhanced adaptive graspers.},
  archive      = {J_NATCS},
  author       = {Dal Corso, Francesco},
  doi          = {10.1038/s43588-022-00336-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {624-625},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bio-inspired adaptive grasper by chiral wrinkling},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Making sense of bacterial toxin evolution. <em>NATCS</em>,
<em>2</em>(10), 623. (<a
href="https://doi.org/10.1038/s43588-022-00343-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00343-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {623},
  shortjournal = {Nat. Comput. Sci},
  title        = {Making sense of bacterial toxin evolution},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022m). Nowcasting food security. <em>NATCS</em>, <em>2</em>(10),
622. (<a href="https://doi.org/10.1038/s43588-022-00342-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00342-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {622},
  shortjournal = {Nat. Comput. Sci},
  title        = {Nowcasting food security},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Keeping the power grid stable. <em>NATCS</em>,
<em>2</em>(10), 621. (<a
href="https://doi.org/10.1038/s43588-022-00341-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Omelchenko, Iryna},
  doi          = {10.1038/s43588-022-00341-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {621},
  shortjournal = {Nat. Comput. Sci},
  title        = {Keeping the power grid stable},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Classical–quantum advantage boundary for enzymes.
<em>NATCS</em>, <em>2</em>(10), 620. (<a
href="https://doi.org/10.1038/s43588-022-00339-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00339-5},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {620},
  shortjournal = {Nat. Comput. Sci},
  title        = {Classical–quantum advantage boundary for enzymes},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Machine learning proved efficient. <em>NATCS</em>,
<em>2</em>(10), 619. (<a
href="https://doi.org/10.1038/s43588-022-00344-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00344-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {619},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning proved efficient},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Rational design of mechanical metamaterials. <em>NATCS</em>,
<em>2</em>(10), 618. (<a
href="https://doi.org/10.1038/s43588-022-00340-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Geddes, Harry},
  doi          = {10.1038/s43588-022-00340-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {618},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rational design of mechanical metamaterials},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Dos and don’ts in a cover letter. <em>NATCS</em>,
<em>2</em>(10), 617. (<a
href="https://doi.org/10.1038/s43588-022-00348-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How to write an effective cover letter.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00348-4},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {617},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dos and don’ts in a cover letter},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identifying patterns in amyotrophic lateral sclerosis
progression from sparse longitudinal data. <em>NATCS</em>,
<em>2</em>(9), 605–616. (<a
href="https://doi.org/10.1038/s43588-022-00299-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The clinical presentation of amyotrophic lateral sclerosis (ALS), a fatal neurodegenerative disease, varies widely across patients, making it challenging to determine if potential therapeutics slow progression. We sought to determine whether there were common patterns of disease progression that could aid in the design and analysis of clinical trials. We developed an approach based on a mixture of Gaussian processes to identify clusters of patients sharing similar disease progression patterns, modeling their average trajectories and the variability in each cluster. We show that ALS progression is frequently nonlinear, with periods of stable disease preceded or followed by rapid decline. We also show that our approach can be extended to Alzheimer’s and Parkinson’s diseases. Our results advance the characterization of disease progression of ALS and provide a flexible modeling approach that can be applied to other progressive diseases. A machine learning approach is presented to identify dominant patterns in disease progression in amyotrophic lateral sclerosis (ALS), Alzheimer’s disease and Parkinson’s disease. The nonlinearity of ALS progression has important clinical implications.},
  archive      = {J_NATCS},
  author       = {Ramamoorthy, Divya and Severson, Kristen and Ghosh, Soumya and Sachs, Karen and Glass, Jonathan D. and Fournier, Christina N. and Herrington, Todd M. and Berry, James D. and Ng, Kenney and Fraenkel, Ernest},
  doi          = {10.1038/s43588-022-00299-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {605-616},
  shortjournal = {Nat. Comput. Sci},
  title        = {Identifying patterns in amyotrophic lateral sclerosis progression from sparse longitudinal data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Micro-level social structures and the success of COVID-19
national policies. <em>NATCS</em>, <em>2</em>(9), 595–604. (<a
href="https://doi.org/10.1038/s43588-022-00314-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Similar policies in response to the COVID-19 pandemic have resulted in different success rates. Although many factors are responsible for the variances in policy success, our study shows that the micro-level structure of person-to-person interactions—measured by the average household size and in-person social contact rate—can be an important explanatory factor. To create an explainable model, we propose a network transformation algorithm to create a simple and computationally efficient scaled network based on these micro-level parameters, as well as incorporate national-level policy data in the network dynamic for SEIR simulations. The model was validated during the early stages of the COVID-19 pandemic, which demonstrated that it can reproduce the dynamic ordinal ranking and trend of infected cases of various European countries that are sufficiently similar in terms of some socio-cultural factors. We also performed several counterfactual analyses to illustrate how policy-based scenario analysis can be performed rapidly and easily with these explainable models. A dynamic network model using data from ten European countries indicates that differences in micro-level social interactions can explain a substantial part of variations in the success of national pandemic policies.},
  archive      = {J_NATCS},
  author       = {Cao, Qingtao and Heydari, Babak},
  doi          = {10.1038/s43588-022-00314-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {595-604},
  shortjournal = {Nat. Comput. Sci},
  title        = {Micro-level social structures and the success of COVID-19 national policies},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Quantifying the information in noisy epidemic curves.
<em>NATCS</em>, <em>2</em>(9), 584–594. (<a
href="https://doi.org/10.1038/s43588-022-00313-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reliably estimating the dynamics of transmissible diseases from noisy surveillance data is an enduring problem in modern epidemiology. Key parameters are often inferred from incident time series, with the aim of informing policy-makers on the growth rate of outbreaks or testing hypotheses about the effectiveness of public health interventions. However, the reliability of these inferences depends critically on reporting errors and latencies innate to the time series. Here, we develop an analytical framework to quantify the uncertainty induced by under-reporting and delays in reporting infections, as well as a metric for ranking surveillance data informativeness. We apply this metric to two primary data sources for inferring the instantaneous reproduction number: epidemic case and death curves. We find that the assumption of death curves as more reliable, commonly made for acute infectious diseases such as COVID-19 and influenza, is not obvious and possibly untrue in many settings. Our framework clarifies and quantifies how actionable information about pathogen transmissibility is lost due to surveillance limitations. A framework for measuring how noise in different outbreak data limits the reliability of estimates of epidemic spread is developed and used to show that death time series are rarely better than case data for inferring COVID-19 transmissibility.},
  archive      = {J_NATCS},
  author       = {Parag, Kris V. and Donnelly, Christl A. and Zarebski, Alexander E.},
  doi          = {10.1038/s43588-022-00313-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {584-594},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying the information in noisy epidemic curves},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fast intratumor heterogeneity inference from single-cell
sequencing data. <em>NATCS</em>, <em>2</em>(9), 577–583. (<a
href="https://doi.org/10.1038/s43588-022-00298-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce HUNTRESS, a computational method for mutational intratumor heterogeneity inference from noisy genotype matrices derived from single-cell sequencing data, the running time of which is linear with the number of cells and quadratic with the number of mutations. We prove that, under reasonable conditions, HUNTRESS computes the true progression history of a tumor with high probability. On simulated and real tumor sequencing data, HUNTRESS is demonstrated to be faster than available alternatives with comparable or better accuracy. Additionally, the progression histories of tumors inferred by HUNTRESS on real single-cell sequencing datasets agree with the best known evolution scenarios for the associated tumors. A computational method is introduced for mutational intratumor heterogeneity inference from noisy genotype matrices derived from single-cell sequencing data. The proposed method is shown to be accurate and faster than available alternatives.},
  archive      = {J_NATCS},
  author       = {Kızılkale, Can and Rashidi Mehrabadi, Farid and Sadeqi Azer, Erfan and Pérez-Guijarro, Eva and Marie, Kerrie L. and Lee, Maxwell P. and Day, Chi-Ping and Merlino, Glenn and Ergün, Funda and Buluç, Aydın and Sahinalp, S. Cenk and Malikić, Salem},
  doi          = {10.1038/s43588-022-00298-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {577-583},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fast intratumor heterogeneity inference from single-cell sequencing data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Challenges and opportunities in quantum machine learning.
<em>NATCS</em>, <em>2</em>(9), 567–576. (<a
href="https://doi.org/10.1038/s43588-022-00311-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {At the intersection of machine learning and quantum computing, quantum machine learning has the potential of accelerating data analysis, especially for quantum data, with applications for quantum materials, biochemistry and high-energy physics. Nevertheless, challenges remain regarding the trainability of quantum machine learning models. Here we review current methods and applications for quantum machine learning. We highlight differences between quantum and classical machine learning, with a focus on quantum neural networks and quantum deep learning. Finally, we discuss opportunities for quantum advantage with quantum machine learning. Quantum machine learning has become an essential tool to process and analyze the increased amount of quantum data. Despite recent progress, there are still many challenges to be addressed and myriad future avenues of research.},
  archive      = {J_NATCS},
  author       = {Cerezo, M. and Verdon, Guillaume and Huang, Hsin-Yuan and Cincio, Lukasz and Coles, Patrick J.},
  doi          = {10.1038/s43588-022-00311-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {567-576},
  shortjournal = {Nat. Comput. Sci},
  title        = {Challenges and opportunities in quantum machine learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Machine learning approach finds nonlinear patterns of
neurodegenerative disease progression. <em>NATCS</em>, <em>2</em>(9),
565–566. (<a href="https://doi.org/10.1038/s43588-022-00300-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a machine learning method that consistently and accurately identified dominant patterns of disease progression in amyotrophic later sclerosis (ALS), Alzheimer’s disease and Parkinson’s disease. Of note, the model was able to identify nonlinear progression trajectories in ALS, a finding that has clinical implications for patient stratification and clinical trial design.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00300-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {565-566},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning approach finds nonlinear patterns of neurodegenerative disease progression},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Calculating effective carrier lifetimes in semiconductor
crystals under realistic conditions. <em>NATCS</em>, <em>2</em>(9),
563–564. (<a href="https://doi.org/10.1038/s43588-022-00301-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A systematic procedure is reported for calculating effective carrier lifetimes in semiconductor crystals from first-principles calculations. Consideration of three major recombination mechanisms and the use of realistic carrier and defect densities is key in resolving the discrepancy between experimental measurements and lifetimes calculated from nonadiabatic molecular dynamics simulations.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00301-5},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {563-564},
  shortjournal = {Nat. Comput. Sci},
  title        = {Calculating effective carrier lifetimes in semiconductor crystals under realistic conditions},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mixing patterns and the spread of pandemics. <em>NATCS</em>,
<em>2</em>(9), 561–562. (<a
href="https://doi.org/10.1038/s43588-022-00312-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrating social mixing data into epidemic models can help policy makers better understand epidemic spread. However, empirical mixing data might not be immediately available in most populations. In a recent work, a network model methodology is proposed to construct micro-level social mixing structure when empirical data are not available.},
  archive      = {J_NATCS},
  author       = {Leung, Kathy and Jia, Jayson S. and Wu, Joseph T.},
  doi          = {10.1038/s43588-022-00312-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {561-562},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mixing patterns and the spread of pandemics},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Getting the most out of noisy surveillance data.
<em>NATCS</em>, <em>2</em>(9), 559–560. (<a
href="https://doi.org/10.1038/s43588-022-00319-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes a metric to quantify how much information different types of epidemiological surveillance data, such as case counts and death counts, convey about the real-time transmission of an epidemic.},
  archive      = {J_NATCS},
  author       = {McGough, Lauren},
  doi          = {10.1038/s43588-022-00319-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {559-560},
  shortjournal = {Nat. Comput. Sci},
  title        = {Getting the most out of noisy surveillance data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). DNA clustering made more efficient. <em>NATCS</em>,
<em>2</em>(9), 558. (<a
href="https://doi.org/10.1038/s43588-022-00330-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00330-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {558},
  shortjournal = {Nat. Comput. Sci},
  title        = {DNA clustering made more efficient},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Escaping the local catalytic activity space. <em>NATCS</em>,
<em>2</em>(9), 557. (<a
href="https://doi.org/10.1038/s43588-022-00326-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Kuttner, Christian},
  doi          = {10.1038/s43588-022-00326-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {557},
  shortjournal = {Nat. Comput. Sci},
  title        = {Escaping the local catalytic activity space},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Accelerated mass spectra analysis. <em>NATCS</em>,
<em>2</em>(9), 556. (<a
href="https://doi.org/10.1038/s43588-022-00327-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00327-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {556},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerated mass spectra analysis},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Controlling patterns of synchrony. <em>NATCS</em>,
<em>2</em>(9), 555. (<a
href="https://doi.org/10.1038/s43588-022-00328-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Omelchenko, Iryna},
  doi          = {10.1038/s43588-022-00328-8},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {555},
  shortjournal = {Nat. Comput. Sci},
  title        = {Controlling patterns of synchrony},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Mathematically probing membrane fluctuations.
<em>NATCS</em>, <em>2</em>(9), 554. (<a
href="https://doi.org/10.1038/s43588-022-00329-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00329-7},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {554},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mathematically probing membrane fluctuations},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). How to change harmful beliefs. <em>NATCS</em>,
<em>2</em>(9), 553. (<a
href="https://doi.org/10.1038/s43588-022-00331-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00331-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {553},
  shortjournal = {Nat. Comput. Sci},
  title        = {How to change harmful beliefs},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Making complex systems computable. <em>NATCS</em>,
<em>2</em>(9), 550–552. (<a
href="https://doi.org/10.1038/s43588-022-00316-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr John Wettlaufer, A. M. Bateman Professor of Geophysics, Mathematics, and Physics at Yale University, research professor at the Nordic Institute for Theoretical Physics, and a member of the Nobel Committee for Physics, discusses the contributions from the laureates of the 2021 Nobel Prize in Physics, his insights into complex system modeling, and his personal experience serving as a Nobel Committee member.},
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00316-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {550-552},
  shortjournal = {Nat. Comput. Sci},
  title        = {Making complex systems computable},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Quantum mechanics guided by simplicity. <em>NATCS</em>,
<em>2</em>(9), 548–549. (<a
href="https://doi.org/10.1038/s43588-022-00285-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Arieh Warshel, distinguished professor of chemistry at the University of Southern California and 2013 Nobel laureate in chemistry, discusses with Nature Computational Science past and current research, his Nobel Prize, and the benefits and challenges of using computational modeling in his work.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00285-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {548-549},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantum mechanics guided by simplicity},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022n). The universe’s expansion in the eyes of computers.
<em>NATCS</em>, <em>2</em>(9), 545–547. (<a
href="https://doi.org/10.1038/s43588-022-00303-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Saul Perlmutter, a professor of physics at the University of California, Berkeley, a senior scientist at Lawrence Berkeley National Laboratory (LBNL), and a 2011 Nobel laureate in physics, discusses the discovery of the accelerating expansion of the Universe and the essential role of computing in this field of research.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00303-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {545-547},
  shortjournal = {Nat. Comput. Sci},
  title        = {The universe’s expansion in the eyes of computers},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Curiosity-driven method development. <em>NATCS</em>,
<em>2</em>(9), 542–544. (<a
href="https://doi.org/10.1038/s43588-022-00317-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Martin Head-Gordon, Distinguished Professor of Chemistry at the University of California, Berkeley, talks with Nature Computational Science about his research on electronic structure theory, quantum chemistry software development, applications in renewable energy, as well as his time working with John Pople, who was recognized by the 1998 Nobel Prize in Chemistry.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00317-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {542-544},
  shortjournal = {Nat. Comput. Sci},
  title        = {Curiosity-driven method development},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A collective effort for building DFT. <em>NATCS</em>,
<em>2</em>(9), 539–541. (<a
href="https://doi.org/10.1038/s43588-022-00294-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Lu Sham, Distinguished Professor Emeritus of Physics at the University of California, San Diego, talks with Nature Computational Science about his current research, the density functional theory (DFT) work that was recognized by the 1998 Nobel Prize in Chemistry — awarded to his co-author Dr Walter Kohn — and where he thinks the field is heading.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00294-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {539-541},
  shortjournal = {Nat. Comput. Sci},
  title        = {A collective effort for building DFT},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The roadmap of climate models. <em>NATCS</em>,
<em>2</em>(9), 536–538. (<a
href="https://doi.org/10.1038/s43588-022-00322-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The 2021 Nobel Prize in Physics recognized the importance of climate modeling and its role in explaining anthropogenic effects on climate change and global warming. To further understand our Earth’s climates, computational models pose new challenges to account for various complexities.},
  archive      = {J_NATCS},
  author       = {Latif, Mojib},
  doi          = {10.1038/s43588-022-00322-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {536-538},
  shortjournal = {Nat. Comput. Sci},
  title        = {The roadmap of climate models},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computation empowers CRISPR discovery and technology.
<em>NATCS</em>, <em>2</em>(9), 533–535. (<a
href="https://doi.org/10.1038/s43588-022-00321-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR has revolutionized biomedical and bioengineering research as a programmable genome engineering technology. Computational tools have been integral throughout the discovery of CRISPR biology and the development of CRISPR-based technologies that hold promise to solve many challenges ahead.},
  archive      = {J_NATCS},
  author       = {Shang, Stephen and Cai, Xiangmeng S. and Qi, Lei S.},
  doi          = {10.1038/s43588-022-00321-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {533-535},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computation empowers CRISPR discovery and technology},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bridging the gap between density functional theory and
quantum materials. <em>NATCS</em>, <em>2</em>(9), 529–532. (<a
href="https://doi.org/10.1038/s43588-022-00323-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The focus on quantum materials has raised questions on the fitness of density functional theory for the description of the basic physics of such strongly correlated systems. Recent studies point to another possibility: the perceived limitations are often not a failure of the density functional theory per se, but rather a failure to break symmetry.},
  archive      = {J_NATCS},
  author       = {Zunger, Alex},
  doi          = {10.1038/s43588-022-00323-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {529-532},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bridging the gap between density functional theory and quantum materials},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A computational perspective on the nobel prize.
<em>NATCS</em>, <em>2</em>(9), 527–528. (<a
href="https://doi.org/10.1038/s43588-022-00325-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The contributions of the computational science community to Nobel Prizes in chemistry and physics take center stage in this month’s Focus issue.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00325-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {527-528},
  shortjournal = {Nat. Comput. Sci},
  title        = {A computational perspective on the nobel prize},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Publisher correction: Rotamer-free protein sequence design
based on deep learning and self-consistency. <em>NATCS</em>,
<em>2</em>(8), 526. (<a
href="https://doi.org/10.1038/s43588-022-00305-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Liu, Yufeng and Zhang, Lu and Wang, Weilun and Zhu, Min and Wang, Chenchen and Li, Fudong and Zhang, Jiahai and Li, Houqiang and Chen, Quan and Liu, Haiyan},
  doi          = {10.1038/s43588-022-00305-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {526},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Rotamer-free protein sequence design based on deep learning and self-consistency},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Disentangling the flow of signals between populations of
neurons. <em>NATCS</em>, <em>2</em>(8), 512–525. (<a
href="https://doi.org/10.1038/s43588-022-00282-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technological advances now allow us to record from large populations of neurons across multiple brain areas. These recordings may illuminate how communication between areas contributes to brain function, yet a substantial barrier remains: how do we disentangle the concurrent, bidirectional flow of signals between populations of neurons? We propose here a dimensionality reduction framework, delayed latents across groups (DLAG), that disentangles signals relayed in each direction, identifies how these signals are represented by each population and characterizes how they evolve within and across trials. We demonstrate that DLAG performs well on synthetic datasets similar in scale to current neurophysiological recordings. Then we study simultaneously recorded populations in primate visual areas V1 and V2, where DLAG reveals signatures of bidirectional yet selective communication. Our framework lays a foundation for dissecting the intricate flow of signals across populations of neurons, and how this signalling contributes to cortical computation. A dimensionality reduction framework, delayed latents across groups (DLAG), is proposed for disentangling the concurrent flow of signals between populations of neurons. DLAG reveals bidirectional communication between visual cortical areas.},
  archive      = {J_NATCS},
  author       = {Gokcen, Evren and Jasper, Anna I. and Semedo, João D. and Zandvakili, Amin and Kohn, Adam and Machens, Christian K. and Yu, Byron M.},
  doi          = {10.1038/s43588-022-00282-5},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {512-525},
  shortjournal = {Nat. Comput. Sci},
  title        = {Disentangling the flow of signals between populations of neurons},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Forced and spontaneous symmetry breaking in cell
polarization. <em>NATCS</em>, <em>2</em>(8), 504–511. (<a
href="https://doi.org/10.1038/s43588-022-00295-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How does breaking the symmetry of an equation alter the symmetry of its solutions? Here, we systematically examine how reducing underlying symmetries from spherical to axisymmetric influences the dynamics of an archetypal model of cell polarization, a key process of biological spatial self-organization. Cell polarization is characterized by nonlinear and non-local dynamics, but we overcome the theory challenges these traits pose by introducing a broadly applicable numerical scheme allowing us to efficiently study continuum models in a wide range of geometries. Guided by numerical results, we discover a dynamical hierarchy of timescales that allows us to reduce relaxation to a purely geometric problem of area-preserving geodesic curvature flow. Through application of variational results, we analytically construct steady states on a number of biologically relevant shapes. In doing so, we reveal non-trivial solutions for symmetry breaking. A spectrally accurate numerical method for solving partial differential equations (PDEs) on non-uniformly curved surfaces is developed. The method is applied to a PDE model of cell polarization to show that geometric effects allow the existence of unexpected multidomain solutions.},
  archive      = {J_NATCS},
  author       = {Miller, Pearson W. and Fortunato, Daniel and Muratov, Cyrill and Greengard, Leslie and Shvartsman, Stanislav},
  doi          = {10.1038/s43588-022-00295-0},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {504-511},
  shortjournal = {Nat. Comput. Sci},
  title        = {Forced and spontaneous symmetry breaking in cell polarization},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The effect of co-location on human communication networks.
<em>NATCS</em>, <em>2</em>(8), 494–503. (<a
href="https://doi.org/10.1038/s43588-022-00296-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to rewire ties in communication networks is vital for large-scale human cooperation and the spread of new ideas. We show that lack of researcher co-location during the COVID-19 lockdown caused the loss of more than 4,800 weak ties—ties between distant parts of the social system that enable the flow of novel information—over 18 months in the email network of a large North American university. Furthermore, we find that the reintroduction of partial co-location through a hybrid work mode led to a partial regeneration of weak ties. We quantify the effect of co-location in forming ties through a model based on physical proximity, which is able to reproduce all empirical observations. Results indicate that employees who are not co-located are less likely to form ties, weakening the spread of information in the workplace. Such findings could contribute to a better understanding of the spatiotemporal dynamics of human communication networks and help organizations that are moving towards the implementation of hybrid work policies to evaluate the minimum amount of in-person interaction necessary for a productive work environment. This study suggests that a lack of co-location hinders the formation of ‘weak ties’—which are crucial for information spread—in communication networks on the basis of an analysis of an email network of more than 2,800 university researchers.},
  archive      = {J_NATCS},
  author       = {Carmody, Daniel and Mazzarello, Martina and Santi, Paolo and Harris, Trevor and Lehmann, Sune and Abbiasov, Timur and Dunbar, Robin and Ratti, Carlo},
  doi          = {10.1038/s43588-022-00296-z},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {494-503},
  shortjournal = {Nat. Comput. Sci},
  title        = {The effect of co-location on human communication networks},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Effective lifetime of non-equilibrium carriers in
semiconductors from non-adiabatic molecular dynamics simulations.
<em>NATCS</em>, <em>2</em>(8), 486–493. (<a
href="https://doi.org/10.1038/s43588-022-00297-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The lifetimes of non-equilibrium charge carriers in semiconductors calculated using non-adiabatic molecular dynamics often differ from experimental results by orders of magnitude. By revisiting the definition of carrier lifetime, we report a systematic procedure for calculating the effective carrier lifetime in semiconductor crystals under realistic conditions. The consideration of all recombination mechanisms and the use of appropriate carrier and defect densities are crucial to bridging the gap between modeling and measurements. Our calculated effective carrier lifetime of CH3NH3PbI3 agrees with experiments, and is limited by band-to-band radiative recombination and Shockley–Read–Hall defect-assisted non-radiative recombination, whereas the band-to-band non-radiative recombination is found to be negligible. The procedure is further validated by application to the compound semiconductors CdTe and GaAs, and thus can be applied in carrier lifetime simulations in other material systems. A systematic framework is introduced to calculate the effective carrier lifetime in semiconductor crystals under realistic conditions that are comparable with experiments. It helps explain the discrepancy between the calculated and experimental lifetimes in hybrid perovskites.},
  archive      = {J_NATCS},
  author       = {Wang, Shanshan and Huang, Menglin and Wu, Yu-Ning and Chu, Weibin and Zhao, Jin and Walsh, Aron and Gong, Xin-Gao and Wei, Su-Huai and Chen, Shiyou},
  doi          = {10.1038/s43588-022-00297-y},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {486-493},
  shortjournal = {Nat. Comput. Sci},
  title        = {Effective lifetime of non-equilibrium carriers in semiconductors from non-adiabatic molecular dynamics simulations},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computational challenges for multimodal astrophysics.
<em>NATCS</em>, <em>2</em>(8), 479–485. (<a
href="https://doi.org/10.1038/s43588-022-00288-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the coming decades, we will face major computational challenges, when the improved sensitivity of third-generation gravitational wave detectors will be such that they will be able to detect a high number (of the order of 7 × 104 per year) of multi-messenger events from binary neutron star mergers, similar to GW 170817. In this Perspective, we discuss the application of multimodal artificial intelligence techniques for multi-messenger astrophysics, fusing the information from different signal emissions. Multi-messenger astronomy offers promises for exploring Universe events in distance. Nevertheless, there are numerous computational challenges when analyzing the massive heterogeneous messenger data from various detectors, creating research opportunities to the community, such as developing multimodal machine learning.},
  archive      = {J_NATCS},
  author       = {Cuoco, Elena and Patricelli, Barbara and Iess, Alberto and Morawski, Filip},
  doi          = {10.1038/s43588-022-00288-z},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {479-485},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational challenges for multimodal astrophysics},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Machine learning reveals how complex molecules bind to
catalyst surfaces. <em>NATCS</em>, <em>2</em>(8), 477–478. (<a
href="https://doi.org/10.1038/s43588-022-00287-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A machine learning method is developed and used to predict the adsorption configurations and energies of complex molecules at the surfaces of transition metals and alloys. This method will be useful for investigating complex reaction networks at complex catalyst materials to understand and improve the performance of heterogeneous catalysts.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00287-0},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {477-478},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning reveals how complex molecules bind to catalyst surfaces},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Untangling network information flow. <em>NATCS</em>,
<em>2</em>(8), 475–476. (<a
href="https://doi.org/10.1038/s43588-022-00284-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining how information flows throughout a network of interconnected components is a challenging task in many scientific domains. A framework is presented to deconstruct the flow of signals that are transmitted across any two areas (such as brain areas) and define how each area represents these signals.},
  archive      = {J_NATCS},
  author       = {Recanatesi, Stefano},
  doi          = {10.1038/s43588-022-00284-3},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {475-476},
  shortjournal = {Nat. Comput. Sci},
  title        = {Untangling network information flow},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Geometry and symmetry-breaking in cell polarity.
<em>NATCS</em>, <em>2</em>(8), 473–474. (<a
href="https://doi.org/10.1038/s43588-022-00304-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An algorithm is presented for the simulation of reaction–diffusion systems on complex geometries, providing insight on how the interplay of cell geometry and biochemistry can control polarity in living cells.},
  archive      = {J_NATCS},
  author       = {Di Talia, Stefano},
  doi          = {10.1038/s43588-022-00304-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {473-474},
  shortjournal = {Nat. Comput. Sci},
  title        = {Geometry and symmetry-breaking in cell polarity},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Model supports storied social network theory.
<em>NATCS</em>, <em>2</em>(8), 471–472. (<a
href="https://doi.org/10.1038/s43588-022-00302-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new Bayesian analysis of remote work data supports one of the oldest theories in social networks, with fresh implications for the future of work environments.},
  archive      = {J_NATCS},
  author       = {Meluso, John},
  doi          = {10.1038/s43588-022-00302-4},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {471-472},
  shortjournal = {Nat. Comput. Sci},
  title        = {Model supports storied social network theory},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). Network science to study the origins of life.
<em>NATCS</em>, <em>2</em>(8), 470. (<a
href="https://doi.org/10.1038/s43588-022-00308-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00308-y},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {470},
  shortjournal = {Nat. Comput. Sci},
  title        = {Network science to study the origins of life},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). The value of quantum effects in reaction rates.
<em>NATCS</em>, <em>2</em>(8), 469. (<a
href="https://doi.org/10.1038/s43588-022-00310-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00310-4},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {469},
  shortjournal = {Nat. Comput. Sci},
  title        = {The value of quantum effects in reaction rates},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Improving segregation prediction. <em>NATCS</em>,
<em>2</em>(8), 468. (<a
href="https://doi.org/10.1038/s43588-022-00309-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00309-x},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {468},
  shortjournal = {Nat. Comput. Sci},
  title        = {Improving segregation prediction},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Bringing the pulsatile release system to light.
<em>NATCS</em>, <em>2</em>(8), 467. (<a
href="https://doi.org/10.1038/s43588-022-00307-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00307-z},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {467},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bringing the pulsatile release system to light},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computing optimal carbon dioxide removal portfolios.
<em>NATCS</em>, <em>2</em>(8), 465–466. (<a
href="https://doi.org/10.1038/s43588-022-00286-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Tan, Raymond R. and Aviso, Kathleen B. and Foo, Dominic C. Y. and Migo-Sumagang, Maria Victoria and Nair, Purusothmn Nair S. Bhasker and Short, Michael},
  doi          = {10.1038/s43588-022-00286-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {465-466},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computing optimal carbon dioxide removal portfolios},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Turing patterns, 70 years later. <em>NATCS</em>,
<em>2</em>(8), 463–464. (<a
href="https://doi.org/10.1038/s43588-022-00306-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {On the 70th anniversary of Alan Turing’s seminal paper on morphogenesis, we look back at the history of the paper and its many applications.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00306-0},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {463-464},
  shortjournal = {Nat. Comput. Sci},
  title        = {Turing patterns, 70 years later},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Rotamer-free protein sequence design based on deep learning
and self-consistency. <em>NATCS</em>, <em>2</em>(7), 451–462. (<a
href="https://doi.org/10.1038/s43588-022-00273-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several previously proposed deep learning methods to design amino acid sequences that autonomously fold into a given protein backbone yielded promising results in computational tests but did not outperform conventional energy function-based methods in wet experiments. Here we present the ABACUS-R method, which uses an encoder–decoder network trained using a multitask learning strategy to predict the sidechain type of a central residue from its three-dimensional local environment, which includes, besides other features, the types but not the conformations of the surrounding sidechains. This eliminates the need to reconstruct and optimize sidechain structures, and drastically simplifies the sequence design process. Thus iteratively applying the encoder–decoder to different central residues is able to produce self-consistent overall sequences for a target backbone. Results of wet experiments, including five structures solved by X-ray crystallography, show that ABACUS-R outperforms state-of-the-art energy function-based methods in success rate and design precision. A deep learning method for protein sequence design on given backbones, ABACUS-R, is proposed in this study. ABACUS-R shows an improved performance when compared with conventional energy function-based methods in wet experiments.},
  archive      = {J_NATCS},
  author       = {Liu, Yufeng and Zhang, Lu and Wang, Weilun and Zhu, Min and Wang, Chenchen and Li, Fudong and Zhang, Jiahai and Li, Houqiang and Chen, Quan and Liu, Haiyan},
  doi          = {10.1038/s43588-022-00273-6},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {451-462},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rotamer-free protein sequence design based on deep learning and self-consistency},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting binding motifs of complex adsorbates using
machine learning with a physics-inspired graph representation.
<em>NATCS</em>, <em>2</em>(7), 443–450. (<a
href="https://doi.org/10.1038/s43588-022-00280-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational screening in heterogeneous catalysis relies increasingly on machine learning models for predicting key input parameters due to the high cost of computing these directly using first-principles methods. This becomes especially relevant when considering complex materials spaces such as alloys, or complex reaction mechanisms with adsorbates that may exhibit bi- or higher-dentate adsorption motifs. Here we present a data-efficient approach to the prediction of binding motifs and associated adsorption enthalpies of complex adsorbates at transition metals and their alloys based on a customized Wasserstein Weisfeiler–Lehman graph kernel and Gaussian process regression. The model shows good predictive performance, not only for the elemental transition metals on which it was trained, but also for an alloy based on these transition metals. Furthermore, incorporation of minimal new training data allows for predicting an out-of-domain transition metal. We believe the model may be useful in active learning approaches, for which we present an ensemble uncertainty estimation approach. A machine learning method that can predict how complex molecules bind to the surfaces of transition metals and alloys is developed, which will be useful for catalyst screening, and for understanding and improving catalyst performance.},
  archive      = {J_NATCS},
  author       = {Xu, Wenbin and Reuter, Karsten and Andersen, Mie},
  doi          = {10.1038/s43588-022-00280-7},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {443-450},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting binding motifs of complex adsorbates using machine learning with a physics-inspired graph representation},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automated discovery of fundamental variables hidden in
experimental data. <em>NATCS</em>, <em>2</em>(7), 433–442. (<a
href="https://doi.org/10.1038/s43588-022-00281-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {All physical laws are described as mathematical relationships between state variables. These variables give a complete and non-redundant description of the relevant system. However, despite the prevalence of computing power and artificial intelligence, the process of identifying the hidden state variables themselves has resisted automation. Most data-driven methods for modelling physical phenomena still rely on the assumption that the relevant state variables are already known. A longstanding question is whether it is possible to identify state variables from only high-dimensional observational data. Here we propose a principle for determining how many state variables an observed system is likely to have, and what these variables might be. We demonstrate the effectiveness of this approach using video recordings of a variety of physical dynamical systems, ranging from elastic double pendulums to fire flames. Without any prior knowledge of the underlying physics, our algorithm discovers the intrinsic dimension of the observed dynamics and identifies candidate sets of state variables. The determination of state variables to describe physical systems is a challenging task. A data-driven approach is proposed to automatically identify state variables for unknown systems from high-dimensional observational data.},
  archive      = {J_NATCS},
  author       = {Chen, Boyuan and Huang, Kuang and Raghupathi, Sunand and Chandratreya, Ishaan and Du, Qiang and Lipson, Hod},
  doi          = {10.1038/s43588-022-00281-6},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {433-442},
  shortjournal = {Nat. Comput. Sci},
  title        = {Automated discovery of fundamental variables hidden in experimental data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Quantum embedding theories to simulate condensed systems on
quantum computers. <em>NATCS</em>, <em>2</em>(7), 424–432. (<a
href="https://doi.org/10.1038/s43588-022-00279-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum computers hold promise to improve the efficiency of quantum simulations of materials and to enable the investigation of systems and properties that are more complex than tractable at present on classical architectures. Here, we discuss computational frameworks to carry out electronic structure calculations of solids on noisy intermediate-scale quantum computers using embedding theories, and we give examples for a specific class of materials, that is, solid materials hosting spin defects. These are promising systems to build future quantum technologies, such as quantum computers, quantum sensors and quantum communication devices. Although quantum simulations on quantum architectures are in their infancy, promising results for realistic systems appear to be within reach. Quantum embedding theory promises the simulation of realistic materials in quantum computers. In this Perspective, challenges and opportunities of applying different embedding frameworks to calculate solid materials properties are discussed, with a focus on electronic structures of spin defects.},
  archive      = {J_NATCS},
  author       = {Vorwerk, Christian and Sheng, Nan and Govoni, Marco and Huang, Benchen and Galli, Giulia},
  doi          = {10.1038/s43588-022-00279-0},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {424-432},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantum embedding theories to simulate condensed systems on quantum computers},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Graph embedding enables cell identification from spatial
transcripts. <em>NATCS</em>, <em>2</em>(7), 422–423. (<a
href="https://doi.org/10.1038/s43588-022-00272-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A cell clustering model for spatial transcripts that uses cell embedding obtained by graph neural networks can be applied to datasets from multiple platforms for cell type or subpopulation identification and further analysis of the spatial microenvironment.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00272-7},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {422-423},
  shortjournal = {Nat. Comput. Sci},
  title        = {Graph embedding enables cell identification from spatial transcripts},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ActiveSVM selects minimal gene sets from gene expression
data. <em>NATCS</em>, <em>2</em>(7), 420–421. (<a
href="https://doi.org/10.1038/s43588-022-00267-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by active learning approaches, we have developed a computational method that selects minimal gene sets capable of reliably identifying cell-types and transcriptional states in large sets of single-cell RNA-sequencing data. As the procedure focuses computational resources on poorly classified cells, active support vector machine (ActiveSVM) scales to data sets with over one million cells.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00267-4},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {420-421},
  shortjournal = {Nat. Comput. Sci},
  title        = {ActiveSVM selects minimal gene sets from gene expression data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improving the efficiency of ab initio electronic-structure
calculations by deep learning. <em>NATCS</em>, <em>2</em>(7), 418–419.
(<a href="https://doi.org/10.1038/s43588-022-00270-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A deep neural network method is developed to learn the density functional theory (DFT) Hamiltonian as a function of atomic structure. This approach provides a solution to the accuracy–efficiency dilemma of DFT and opens opportunities to investigate large-scale materials, such as twisted van der Waals materials.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00270-9},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {418-419},
  shortjournal = {Nat. Comput. Sci},
  title        = {Improving the efficiency of ab initio electronic-structure calculations by deep learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Protein sequence design by deep learning. <em>NATCS</em>,
<em>2</em>(7), 416–417. (<a
href="https://doi.org/10.1038/s43588-022-00274-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The design of protein sequences that can precisely fold into pre-specified 3D structures is a challenging task. A recently proposed deep-learning algorithm improves such designs when compared with traditional, physics-based protein design approaches.},
  archive      = {J_NATCS},
  author       = {Wang, Jue},
  doi          = {10.1038/s43588-022-00274-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {416-417},
  shortjournal = {Nat. Comput. Sci},
  title        = {Protein sequence design by deep learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning state variables for physical systems.
<em>NATCS</em>, <em>2</em>(7), 414–415. (<a
href="https://doi.org/10.1038/s43588-022-00283-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The problem of automatically determining state variables for physical systems is challenging, but essential in the modeling process of almost all scientific and engineering processes. A deep neural network-based approach is proposed to find state variables for systems whose data are given as video frames.},
  archive      = {J_NATCS},
  author       = {Kramer, Boris},
  doi          = {10.1038/s43588-022-00283-4},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {414-415},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning state variables for physical systems},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Modeling and combating misinformation spread.
<em>NATCS</em>, <em>2</em>(7), 413. (<a
href="https://doi.org/10.1038/s43588-022-00290-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00290-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {413},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling and combating misinformation spread},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Effects of mutations on SARS-CoV-2 fitness. <em>NATCS</em>,
<em>2</em>(7), 412. (<a
href="https://doi.org/10.1038/s43588-022-00289-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00289-y},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {412},
  shortjournal = {Nat. Comput. Sci},
  title        = {Effects of mutations on SARS-CoV-2 fitness},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). A move in the light direction. <em>NATCS</em>,
<em>2</em>(7), 411. (<a
href="https://doi.org/10.1038/s43588-022-00291-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00291-4},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {411},
  shortjournal = {Nat. Comput. Sci},
  title        = {A move in the light direction},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Quantum experiment design via interpretable DL.
<em>NATCS</em>, <em>2</em>(7), 410. (<a
href="https://doi.org/10.1038/s43588-022-00292-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00292-3},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {410},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantum experiment design via interpretable DL},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mathematics, the queen of sciences. <em>NATCS</em>,
<em>2</em>(7), 409. (<a
href="https://doi.org/10.1038/s43588-022-00293-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We highlight how this year’s awardees from some of the most important prizes in mathematics have had an impact in the computational science community.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00293-2},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {409},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mathematics, the queen of sciences},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cell clustering for spatial transcriptomics data with graph
neural networks. <em>NATCS</em>, <em>2</em>(6), 399–408. (<a
href="https://doi.org/10.1038/s43588-022-00266-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics data can provide high-throughput gene expression profiling and the spatial structure of tissues simultaneously. Most studies have relied on only the gene expression information but cannot utilize the spatial information efficiently. Taking advantage of spatial transcriptomics and graph neural networks, we introduce cell clustering for spatial transcriptomics data with graph neural networks, an unsupervised cell clustering method based on graph convolutional networks to improve ab initio cell clustering and discovery of cell subtypes based on curated cell category annotation. On the basis of its application to five in vitro and in vivo spatial datasets, we show that cell clustering for spatial transcriptomics outperforms other spatial clustering approaches on spatial transcriptomics datasets and can clearly identify all four cell cycle phases from multiplexed error-robust fluorescence in situ hybridization data of cultured cells. From enhanced sequential fluorescence in situ hybridization data of brain, cell clustering for spatial transcriptomics finds functional cell subtypes with different micro-environments, which are all validated experimentally, inspiring biological hypotheses about the underlying interactions among the cell state, cell type and micro-environment. A graph neural network-based cell clustering model for spatial transcripts obtains cell embeddings from global cell interactions across tissue samples and identifies cell types and subpopulations.},
  archive      = {J_NATCS},
  author       = {Li, Jiachen and Chen, Siheng and Pan, Xiaoyong and Yuan, Ye and Shen, Hong-Bin},
  doi          = {10.1038/s43588-022-00266-5},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {399-408},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cell clustering for spatial transcriptomics data with graph neural networks},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Minimal gene set discovery in single-cell mRNA-seq datasets
with ActiveSVM. <em>NATCS</em>, <em>2</em>(6), 387–398. (<a
href="https://doi.org/10.1038/s43588-022-00263-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing costs currently prohibit the application of single-cell mRNA-seq to many biological and clinical analyses. Targeted single-cell mRNA-sequencing reduces sequencing costs by profiling reduced gene sets that capture biological information with a minimal number of genes. Here we introduce an active learning method that identifies minimal but highly informative gene sets that enable the identification of cell types, physiological states and genetic perturbations in single-cell data using a small number of genes. Our active feature selection procedure generates minimal gene sets from single-cell data by employing an active support vector machine (ActiveSVM) classifier. We demonstrate that ActiveSVM feature selection identifies gene sets that enable ~90\% cell-type classification accuracy across, for example, cell atlas and disease-characterization datasets. The discovery of small but highly informative gene sets should enable reductions in the number of measurements necessary for application of single-cell mRNA-seq to clinical tests, therapeutic discovery and genetic screens. ActiveSVM is introduced as a feature selection procedure for discovering minimal gene sets in large single-cell mRNA-seq datasets.},
  archive      = {J_NATCS},
  author       = {Chen, Xiaoqiao and Chen, Sisi and Thomson, Matt},
  doi          = {10.1038/s43588-022-00263-8},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {387-398},
  shortjournal = {Nat. Comput. Sci},
  title        = {Minimal gene set discovery in single-cell mRNA-seq datasets with ActiveSVM},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Generative aptamer discovery using RaptGen. <em>NATCS</em>,
<em>2</em>(6), 378–386. (<a
href="https://doi.org/10.1038/s43588-022-00249-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nucleic acid aptamers are generated by an in vitro molecular evolution method known as systematic evolution of ligands by exponential enrichment (SELEX). Various candidates are limited by actual sequencing data from an experiment. Here we developed RaptGen, which is a variational autoencoder for in silico aptamer generation. RaptGen exploits a profile hidden Markov model decoder to represent motif sequences effectively. We showed that RaptGen embedded simulation sequence data into low-dimensional latent space on the basis of motif information. We also performed sequence embedding using two independent SELEX datasets. RaptGen successfully generated aptamers from the latent space even though they were not included in high-throughput sequencing. RaptGen could also generate a truncated aptamer with a short learning model. We demonstrated that RaptGen could be applied to activity-guided aptamer generation according to Bayesian optimization. We concluded that a generative method by RaptGen and latent representation are useful for aptamer discovery. A probabilistic generative model for aptamers called RaptGen is introduced, which accelerates the process of aptamer development by generating new aptamer sequences.},
  archive      = {J_NATCS},
  author       = {Iwano, Natsuki and Adachi, Tatsuo and Aoki, Kazuteru and Nakamura, Yoshikazu and Hamada, Michiaki},
  doi          = {10.1038/s43588-022-00249-6},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {378-386},
  shortjournal = {Nat. Comput. Sci},
  title        = {Generative aptamer discovery using RaptGen},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep-learning density functional theory hamiltonian for
efficient ab initio electronic-structure calculation. <em>NATCS</em>,
<em>2</em>(6), 367–377. (<a
href="https://doi.org/10.1038/s43588-022-00265-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The marriage of density functional theory (DFT) and deep-learning methods has the potential to revolutionize modern computational materials science. Here we develop a deep neural network approach to represent the DFT Hamiltonian (DeepH) of crystalline materials, aiming to bypass the computationally demanding self-consistent field iterations of DFT and substantially improve the efficiency of ab initio electronic-structure calculations. A general framework is proposed to deal with the large dimensionality and gauge (or rotation) covariance of the DFT Hamiltonian matrix by virtue of locality, and this is realized by a message-passing neural network for deep learning. High accuracy, high efficiency and good transferability of the DeepH method are generally demonstrated for various kinds of material system and physical property. The method provides a solution to the accuracy–efficiency dilemma of DFT and opens opportunities to explore large-scale material systems, as evidenced by a promising application in the study of twisted van der Waals materials. A deep neural network method is developed to learn the mapping function from atomic structure to density functional theory (DFT) Hamiltonian, which helps address the accuracy–efficiency dilemma of DFT and is useful for studying large-scale materials.},
  archive      = {J_NATCS},
  author       = {Li, He and Wang, Zun and Zou, Nianlong and Ye, Meng and Xu, Runzhang and Gong, Xiaoxun and Duan, Wenhui and Xu, Yong},
  doi          = {10.1038/s43588-022-00265-6},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {367-377},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep-learning density functional theory hamiltonian for efficient ab initio electronic-structure calculation},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhancing computational fluid dynamics with machine
learning. <em>NATCS</em>, <em>2</em>(6), 358–366. (<a
href="https://doi.org/10.1038/s43588-022-00264-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning is rapidly becoming a core technology for scientific computing, with numerous opportunities to advance the field of computational fluid dynamics. Here we highlight some of the areas of highest potential impact, including to accelerate direct numerical simulations, to improve turbulence closure modeling and to develop enhanced reduced-order models. We also discuss emerging areas of machine learning that are promising for computational fluid dynamics, as well as some potential limitations that should be taken into account. Machine learning has been used to accelerate the simulation of fluid dynamics. However, despite the recent developments in this field, there are still challenges to be addressed by the community, a fact that creates research opportunities.},
  archive      = {J_NATCS},
  author       = {Vinuesa, Ricardo and Brunton, Steven L.},
  doi          = {10.1038/s43588-022-00264-7},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {358-366},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing computational fluid dynamics with machine learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). AI-powered aptamer generation. <em>NATCS</em>,
<em>2</em>(6), 356–357. (<a
href="https://doi.org/10.1038/s43588-022-00253-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aptamers are expected to be next-generation drugs, but identifying candidate aptamers is a challenging task given the large search space. Now, an artificial intelligence (AI)-powered tool called RaptGen is proposed for improving the successful identification of aptamer sequences.},
  archive      = {J_NATCS},
  author       = {Khabbazian, Majid and Jabbari, Hosna},
  doi          = {10.1038/s43588-022-00253-w},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {356-357},
  shortjournal = {Nat. Comput. Sci},
  title        = {AI-powered aptamer generation},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Graphing cell relations in spatial transcriptomics.
<em>NATCS</em>, <em>2</em>(6), 354–355. (<a
href="https://doi.org/10.1038/s43588-022-00269-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A graph neural network-based tool is introduced to perform unsupervised cell clustering using spatially resolved transcriptomics data that can uncover cell identities, interactions, and spatial organization in tissues and organs.},
  archive      = {J_NATCS},
  author       = {Zhou, Xin},
  doi          = {10.1038/s43588-022-00269-2},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {354-355},
  shortjournal = {Nat. Comput. Sci},
  title        = {Graphing cell relations in spatial transcriptomics},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022l). Neuromechanical modeling of adult fly. <em>NATCS</em>,
<em>2</em>(6), 353. (<a
href="https://doi.org/10.1038/s43588-022-00276-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00276-3},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {353},
  shortjournal = {Nat. Comput. Sci},
  title        = {Neuromechanical modeling of adult fly},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Cellular scale modeling of cytoskeletal assemblies.
<em>NATCS</em>, <em>2</em>(6), 352. (<a
href="https://doi.org/10.1038/s43588-022-00275-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00275-4},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {352},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cellular scale modeling of cytoskeletal assemblies},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022l). Stabilizing quantum GANs. <em>NATCS</em>, <em>2</em>(6),
351. (<a href="https://doi.org/10.1038/s43588-022-00277-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00277-2},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {351},
  shortjournal = {Nat. Comput. Sci},
  title        = {Stabilizing quantum GANs},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022m). Theoretical progress toward sustainable batteries.
<em>NATCS</em>, <em>2</em>(6), 350. (<a
href="https://doi.org/10.1038/s43588-022-00278-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00278-1},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {350},
  shortjournal = {Nat. Comput. Sci},
  title        = {Theoretical progress toward sustainable batteries},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Digging deeper into the gut microbiome. <em>NATCS</em>,
<em>2</em>(6), 348–349. (<a
href="https://doi.org/10.1038/s43588-022-00262-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Sean Gibbons, assistant professor at the Institute for Systems Biology and a Washington Research Foundation Distinguished Investigator, discusses with Nature Computational Science how he uses computational science to gain insights into the gut microbiome and to address the major challenges of this field, as well as his advice to young LGBTQIA+ scientists.},
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00262-9},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {348-349},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digging deeper into the gut microbiome},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Gender inequality infiltrates the in silico modeling world.
<em>NATCS</em>, <em>2</em>(6), 346–347. (<a
href="https://doi.org/10.1038/s43588-022-00268-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gender inequality has been the unspoken truth, rampant for centuries. Although a deep-rooted cultural mindset, the inequality has reverse-translated from society into the way we study and practice science, and more currently, into the computational modeling world.},
  archive      = {J_NATCS},
  author       = {Chellappa, Anirudh S. and Sahoo, Madhulika and Sahoo, Swagatika},
  doi          = {10.1038/s43588-022-00268-3},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {346-347},
  shortjournal = {Nat. Comput. Sci},
  title        = {Gender inequality infiltrates the in silico modeling world},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enough is enough. <em>NATCS</em>, <em>2</em>(6), 345. (<a
href="https://doi.org/10.1038/s43588-022-00271-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data science studies have provided evidence that regulating gun ownership can save lives. However, US lawmakers still fail to follow the science.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00271-8},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {345},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enough is enough},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Autonomous inference of complex
network dynamics from incomplete and noisy data. <em>NATCS</em>,
<em>2</em>(5), 343. (<a
href="https://doi.org/10.1038/s43588-022-00255-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Gao, Ting-Ting and Yan, Gang},
  doi          = {10.1038/s43588-022-00255-8},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {343},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Autonomous inference of complex network dynamics from incomplete and noisy data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Dynamic forecasting of severe acute
graft-versus- host disease after transplantation. <em>NATCS</em>,
<em>2</em>(5), 342. (<a
href="https://doi.org/10.1038/s43588-022-00256-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Liu, Xueou and Cao, Yigeng and Guo, Ye and Gong, Xiaowen and Feng, Yahui and Wang, Yao and Wang, Mingyang and Cui, Mengxuan and Guo, Wenwen and Zhang, Luyang and Zhao, Ningning and Song, Xiaoqiang and Zheng, Xuetong and Chen, Xia and Shen, Qiujin and Zhang, Song and Song, Zhen and Li, Linfeng and Feng, Sizhou and Han, Mingzhe and Zhu, Xiaofan and Jiang, Erlie and Chen, Junren},
  doi          = {10.1038/s43588-022-00256-7},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {342},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Dynamic forecasting of severe acute graft-versus- host disease after transplantation},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Solving the electronic schrödinger equation for multiple
nuclear geometries with weight-sharing deep neural networks.
<em>NATCS</em>, <em>2</em>(5), 331–341. (<a
href="https://doi.org/10.1038/s43588-022-00228-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Schrödinger equation describes the quantum-mechanical behaviour of particles, making it the most fundamental equation in chemistry. A solution for a given molecule allows computation of any of its properties. Finding accurate solutions for many different molecules and geometries is thus crucial to the discovery of new materials such as drugs or catalysts. Despite its importance, the Schrödinger equation is notoriously difficult to solve even for single molecules, as established methods scale exponentially with the number of particles. Combining Monte Carlo techniques with unsupervised optimization of neural networks was recently discovered as a promising approach to overcome this curse of dimensionality, but the corresponding methods do not exploit synergies that arise when considering multiple geometries. Here we show that sharing the vast majority of weights across neural network models for different geometries substantially accelerates optimization. Furthermore, weight-sharing yields pretrained models that require only a small number of additional optimization steps to obtain high-accuracy solutions for new geometries. Weight-sharing is used to accelerate and to effectively pretrain neural network-based variational Monte Carlo methods when solving the electronic Schrödinger equation for multiple geometries.},
  archive      = {J_NATCS},
  author       = {Scherbela, Michael and Reisenhofer, Rafael and Gerard, Leon and Marquetand, Philipp and Grohs, Philipp},
  doi          = {10.1038/s43588-022-00228-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {331-341},
  shortjournal = {Nat. Comput. Sci},
  title        = {Solving the electronic schrödinger equation for multiple nuclear geometries with weight-sharing deep neural networks},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Adversarial domain translation networks for integrating
large-scale atlas-level single-cell datasets. <em>NATCS</em>,
<em>2</em>(5), 317–330. (<a
href="https://doi.org/10.1038/s43588-022-00251-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid emergence of large-scale atlas-level single-cell RNA-seq datasets presents remarkable opportunities for broad and deep biological investigations through integrative analyses. However, harmonizing such datasets requires integration approaches to be not only computationally scalable, but also capable of preserving a wide range of fine-grained cell populations. We have created Portal, a unified framework of adversarial domain translation to learn harmonized representations of datasets. When compared to other state-of-the-art methods, Portal achieves better performance for preserving biological variation during integration, while achieving the integration of millions of cells, in minutes, with low memory consumption. We show that Portal is widely applicable to integrating datasets across different samples, platforms and data types. We also apply Portal to the integration of cross-species datasets with limited shared information among them, elucidating biological insights into the similarities and divergences in the spermatogenesis process among mouse, macaque and human. An adversarial domain translation framework is presented for scalable integration of single-cell atlases across samples, technical platforms, data modalities and species.},
  archive      = {J_NATCS},
  author       = {Zhao, Jia and Wang, Gefei and Ming, Jingsi and Lin, Zhixiang and Wang, Yang and Wu, Angela Ruohao and Yang, Can},
  doi          = {10.1038/s43588-022-00251-y},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {317-330},
  shortjournal = {Nat. Comput. Sci},
  title        = {Adversarial domain translation networks for integrating large-scale atlas-level single-cell datasets},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Large-scale microbiome data integration enables robust
biomarker identification. <em>NATCS</em>, <em>2</em>(5), 307–316. (<a
href="https://doi.org/10.1038/s43588-022-00247-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The close association between gut microbiota dysbiosis and human diseases is being increasingly recognized. However, contradictory results are frequently reported, as confounding effects exist. The lack of unbiased data integration methods is also impeding the discovery of disease-associated microbial biomarkers from different cohorts. Here we propose an algorithm, NetMoss, for assessing shifts of microbial network modules to identify robust biomarkers associated with various diseases. Compared to previous approaches, the NetMoss method shows better performance in removing batch effects. Through comprehensive evaluations on both simulated and real datasets, we demonstrate that NetMoss has great advantages in the identification of disease-related biomarkers. Based on analysis of pandisease microbiota studies, there is a high prevalence of multidisease-related bacteria in global populations. We believe that large-scale data integration will help in understanding the role of the microbiome from a more comprehensive perspective and that accurate biomarker identification will greatly promote microbiome-based medical diagnosis. An efficient algorithm is proposed to integrate large-scale microbiome datasets. This unbiased data integration method enables the identification of robust biomarkers associated with various diseases through assessing shifts of microbial network modules.},
  archive      = {J_NATCS},
  author       = {Xiao, Liwen and Zhang, Fengyi and Zhao, Fangqing},
  doi          = {10.1038/s43588-022-00247-8},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {307-316},
  shortjournal = {Nat. Comput. Sci},
  title        = {Large-scale microbiome data integration enables robust biomarker identification},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). GPU-accelerated connectome discovery at scale.
<em>NATCS</em>, <em>2</em>(5), 298–306. (<a
href="https://doi.org/10.1038/s43588-022-00250-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diffusion magnetic resonance imaging and tractography enable the estimation of anatomical connectivity in the human brain, in vivo. Yet, without ground-truth validation, different tractography algorithms can yield widely varying connectivity estimates. Although streamline pruning techniques mitigate this challenge, slow compute times preclude their use in big-data applications. We present ‘Regularized, Accelerated, Linear Fascicle Evaluation’ (ReAl-LiFE), a GPU-based implementation of a state-of-the-art streamline pruning algorithm (LiFE), which achieves &amp;gt;100× speedups over previous CPU-based implementations. Leveraging these speedups, we overcome key limitations with LiFE’s algorithm to generate sparser and more accurate connectomes. We showcase ReAl-LiFE’s ability to estimate connections with superlative test–retest reliability, while outperforming competing approaches. Moreover, we predicted inter-individual variations in multiple cognitive scores with ReAl-LiFE connectome features. We propose ReAl-LiFE as a timely tool, surpassing the state of the art, for accurate discovery of individualized brain connectomes at scale. Finally, our GPU-accelerated implementation of a popular non-negative least-squares optimization algorithm is widely applicable to many real-world problems. Accurate structural brain connectivity estimation is key to uncovering brain–behavior relationships. ReAl-LiFE, a GPU-accelerated approach, is applied for fast and reliable evaluation of individualized brain connectomes at scale.},
  archive      = {J_NATCS},
  author       = {Sreenivasan, Varsha and Kumar, Sawan and Pestilli, Franco and Talukdar, Partha and Sridharan, Devarajan},
  doi          = {10.1038/s43588-022-00250-z},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {298-306},
  shortjournal = {Nat. Comput. Sci},
  title        = {GPU-accelerated connectome discovery at scale},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Using metric learning to identify the lab-of-origin of
engineered DNA. <em>NATCS</em>, <em>2</em>(5), 296–297. (<a
href="https://doi.org/10.1038/s43588-022-00240-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining the origin of engineered DNA can help to foster responsible innovation within the biotechnology community. A convolutional neural network approach that learns distances between engineered DNA sequences and various labs that could have created them is used to accurately predict the lab-of-origin.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00240-1},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {296-297},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using metric learning to identify the lab-of-origin of engineered DNA},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Universal descriptors to predict interactions of inorganic
nanoparticles with proteins. <em>NATCS</em>, <em>2</em>(5), 294–295. (<a
href="https://doi.org/10.1038/s43588-022-00230-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biomimetic nanoparticles can form complexes with proteins. Structural descriptors have been identified to predict nanoparticle–protein complex formation and their interaction sites. These descriptors include geometrical and graph-theoretical molecular features that are universally applicable to all nanoscale macromolecules of both organic and inorganic chemistries.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00230-3},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {294-295},
  shortjournal = {Nat. Comput. Sci},
  title        = {Universal descriptors to predict interactions of inorganic nanoparticles with proteins},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accelerating quantum molecular simulations. <em>NATCS</em>,
<em>2</em>(5), 292–293. (<a
href="https://doi.org/10.1038/s43588-022-00237-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Variational Monte Carlo is one of the most accurate methods to solve the many-electron Schrödinger equation, but suffers from high computational cost. A recent study uses a weight-sharing technique to accelerate the neural network-based variational Monte Carlo method, allowing accurate and effective simulations of molecules.},
  archive      = {J_NATCS},
  author       = {Tran, Huan},
  doi          = {10.1038/s43588-022-00237-w},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {292-293},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating quantum molecular simulations},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhancing biomarkers with co-abundance. <em>NATCS</em>,
<em>2</em>(5), 290–291. (<a
href="https://doi.org/10.1038/s43588-022-00254-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of robust and generalizable biomarkers based on microbial abundance data is a challenging task. An algorithm shows an enhanced classification performance by quantifying shifts in microbial co-abundances.},
  archive      = {J_NATCS},
  author       = {Lahti, Leo},
  doi          = {10.1038/s43588-022-00254-9},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {290-291},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing biomarkers with co-abundance},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficiently pruning brain connectomes. <em>NATCS</em>,
<em>2</em>(5), 288–289. (<a
href="https://doi.org/10.1038/s43588-022-00252-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing the brain’s connectome at multiple scales is essential for unraveling fundamental principles of cortical information processing and how it impacts behavior. A GPU-based implementation for connectome pruning is proposed, achieving greater than 100-fold speedups over previous CPU-based implementations.},
  archive      = {J_NATCS},
  author       = {Zuo, Xi-Nian},
  doi          = {10.1038/s43588-022-00252-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {288-289},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficiently pruning brain connectomes},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). AI for resource-constrained medical fields. <em>NATCS</em>,
<em>2</em>(5), 287. (<a
href="https://doi.org/10.1038/s43588-022-00257-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00257-6},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {287},
  shortjournal = {Nat. Comput. Sci},
  title        = {AI for resource-constrained medical fields},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Impact of aspirin on cancer incidence. <em>NATCS</em>,
<em>2</em>(5), 286. (<a
href="https://doi.org/10.1038/s43588-022-00259-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00259-4},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {286},
  shortjournal = {Nat. Comput. Sci},
  title        = {Impact of aspirin on cancer incidence},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Expanding the capability of autonomous microscopy.
<em>NATCS</em>, <em>2</em>(5), 285. (<a
href="https://doi.org/10.1038/s43588-022-00260-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00260-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {285},
  shortjournal = {Nat. Comput. Sci},
  title        = {Expanding the capability of autonomous microscopy},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Neural-network wavefunctions. <em>NATCS</em>,
<em>2</em>(5), 284. (<a
href="https://doi.org/10.1038/s43588-022-00258-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00258-5},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {284},
  shortjournal = {Nat. Comput. Sci},
  title        = {Neural-network wavefunctions},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Fighting hate speech and misinformation online.
<em>NATCS</em>, <em>2</em>(5), 281–283. (<a
href="https://doi.org/10.1038/s43588-022-00238-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Srijan Kumar, assistant professor at Georgia Institute of Technology and a Forbes 30 Under 30 honoree in science, discusses with Nature Computational Science how he uses machine learning and data science to identify and mitigate malicious activities on online platforms, including misinformation and anti-Asian hate speech.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00238-9},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {281-283},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fighting hate speech and misinformation online},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). From mechanistic insight towards clinical implementation
using normative modeling. <em>NATCS</em>, <em>2</em>(5), 278–280. (<a
href="https://doi.org/10.1038/s43588-022-00248-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Normative modeling is considered one of the most promising avenues towards personalized medicine. The integration of multimodal, mechanistic and lifespan modeling will play an essential role, but significant challenges need to be overcome before this promise can be turned into reality.},
  archive      = {J_NATCS},
  author       = {Rieg, Thilo and Schwarz, Emanuel},
  doi          = {10.1038/s43588-022-00248-7},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {278-280},
  shortjournal = {Nat. Comput. Sci},
  title        = {From mechanistic insight towards clinical implementation using normative modeling},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cracking the code review process. <em>NATCS</em>,
<em>2</em>(5), 277. (<a
href="https://doi.org/10.1038/s43588-022-00261-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What does it entail to perform a code review for Nature Computational Science?},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00261-w},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {277},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cracking the code review process},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Progressive assembly of multi-domain protein structures from
cryo-EM density maps. <em>NATCS</em>, <em>2</em>(4), 265–275. (<a
href="https://doi.org/10.1038/s43588-022-00232-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Progress in cryo-electron microscopy has provided the potential for large-size protein structure determination. However, the success rate for solving multi-domain proteins remains low because of the difficulty in modelling inter-domain orientations. Here we developed domain enhanced modeling using cryo-electron microscopy (DEMO-EM), an automatic method to assemble multi-domain structures from cryo-electron microscopy maps through a progressive structural refinement procedure combining rigid-body domain fitting and flexible assembly simulations with deep-neural-network inter-domain distance profiles. The method was tested on a large-scale benchmark set of proteins containing up to 12 continuous and discontinuous domains with medium- to low-resolution density maps, where DEMO-EM produced models with correct inter-domain orientations (template modeling score (TM-score) &amp;gt;0.5) for 97\% of cases and outperformed state-of-the-art methods. DEMO-EM was applied to the severe acute respiratory syndrome coronavirus 2 genome and generated models with average TM-score and root-mean-square deviation of 0.97 and 1.3 Å, respectively, with respect to the deposited structures. These results demonstrate an efficient pipeline that enables automated and reliable large-scale multi-domain protein structure modelling from cryo-electron microscopy maps. A protocol is developed to construct multi-domain protein structures from cryo-electron microscopy density maps. The results demonstrate the effectiveness of deep-learning-guided inter-domain structure assembly and refinement simulations.},
  archive      = {J_NATCS},
  author       = {Zhou, Xiaogen and Li, Yang and Zhang, Chengxin and Zheng, Wei and Zhang, Guijun and Zhang, Yang},
  doi          = {10.1038/s43588-022-00232-1},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {265-275},
  shortjournal = {Nat. Comput. Sci},
  title        = {Progressive assembly of multi-domain protein structures from cryo-EM density maps},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improving lab-of-origin prediction of genetically engineered
plasmids via deep metric learning. <em>NATCS</em>, <em>2</em>(4),
253–264. (<a href="https://doi.org/10.1038/s43588-022-00234-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome engineering is undergoing unprecedented development and is now becoming widely available. Genetic engineering attribution can make sequence–lab associations and assist forensic experts in ensuring responsible biotechnology innovation and reducing misuse of engineered DNA sequences. Here we propose a method based on metric learning to rank the most likely labs of origin while simultaneously generating embeddings for plasmid sequences and labs. These embeddings can be used to perform various downstream tasks, such as clustering DNA sequences and labs, as well as using them as features in machine learning models. Our approach employs a circular shift augmentation method and can correctly rank the lab of origin 90\% of the time within its top-10 predictions. We also demonstrate that we can perform few-shot learning and obtain 76\% top-10 accuracy using only 10\% of the sequences. Finally, our approach can also extract key signatures in plasmid sequences for particular labs, allowing for an interpretable examination of the model’s outputs. Associating biotechnology to its lab of origin is a challenging task. A deep learning approach is proposed to find distances between engineered plasmids, which allows the ranking of their most probable labs of origin with high accuracy.},
  archive      = {J_NATCS},
  author       = {Soares, Igor M. and Camargo, Fernando H. F. and Marques, Adriano and Crook, Oliver M.},
  doi          = {10.1038/s43588-022-00234-z},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {253-264},
  shortjournal = {Nat. Comput. Sci},
  title        = {Improving lab-of-origin prediction of genetically engineered plasmids via deep metric learning},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Unifying structural descriptors for biological and
bioinspired nanoscale complexes. <em>NATCS</em>, <em>2</em>(4), 243–252.
(<a href="https://doi.org/10.1038/s43588-022-00229-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biomimetic nanoparticles are known to serve as nanoscale adjuvants, enzyme mimics and amyloid fibrillation inhibitors. Their further development requires better understanding of their interactions with proteins. The abundant knowledge about protein–protein interactions can serve as a guide for designing protein–nanoparticle assemblies, but the chemical and biological inputs used in computational packages for protein–protein interactions are not applicable to inorganic nanoparticles. Analysing chemical, geometrical and graph-theoretical descriptors for protein complexes, we found that geometrical and graph-theoretical descriptors are uniformly applicable to biological and inorganic nanostructures and can predict interaction sites in protein pairs with accuracy &amp;gt;80\% and classification probability ~90\%. We extended the machine-learning algorithms trained on protein–protein interactions to inorganic nanoparticles and found a nearly exact match between experimental and predicted interaction sites with proteins. These findings can be extended to other organic and inorganic nanoparticles to predict their assemblies with biomolecules and other chemical structures forming lock-and-key complexes. Unified structural descriptors of geometrical and graph-theoretical features are developed, allowing knowledge about protein lock-and-key complexes to be utilized to predict the formation of and interaction sites in protein–nanoparticle pairs.},
  archive      = {J_NATCS},
  author       = {Cha, Minjeong and Emre, Emine Sumeyra Turali and Xiao, Xiongye and Kim, Ji-Young and Bogdan, Paul and VanEpps, J. Scott and Violi, Angela and Kotov, Nicholas A.},
  doi          = {10.1038/s43588-022-00229-w},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {243-252},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unifying structural descriptors for biological and bioinspired nanoscale complexes},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards practical and robust DNA-based data archiving using
the yin–yang codec system. <em>NATCS</em>, <em>2</em>(4), 234–242. (<a
href="https://doi.org/10.1038/s43588-022-00231-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA is a promising data storage medium due to its remarkable durability and space-efficient storage. Early bit-to-base transcoding schemes have primarily pursued information density, at the expense of introducing biocompatibility challenges or decoding failure. Here we propose a robust transcoding algorithm named the yin–yang codec, using two rules to encode two binary bits into one nucleotide, to generate DNA sequences that are highly compatible with synthesis and sequencing technologies. We encoded two representative file formats and stored them in vitro as 200 nt oligo pools and in vivo as a ~54 kbps DNA fragment in yeast cells. Sequencing results show that the yin–yang codec exhibits high robustness and reliability for a wide variety of data types, with an average recovery rate of 99.9\% above 104 molecule copies and an achieved recovery rate of 87.53\% at ≤102 copies. Additionally, the in vivo storage demonstration achieved an experimentally measured physical density close to the theoretical maximum. The yin–yang codec transcoding algorithm is proposed to improve the practicality and robustness of DNA data storage.},
  archive      = {J_NATCS},
  author       = {Ping, Zhi and Chen, Shihong and Zhou, Guangyu and Huang, Xiaoluo and Zhu, Sha Joe and Zhang, Haoling and Lee, Henry H. and Lan, Zhaojun and Cui, Jie and Chen, Tai and Zhang, Wenwei and Yang, Huanming and Xu, Xun and Church, George M. and Shen, Yue},
  doi          = {10.1038/s43588-022-00231-2},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {234-242},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards practical and robust DNA-based data archiving using the yin–yang codec system},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Optimizing social and economic activity while containing
SARS-CoV-2 transmission using DAEDALUS. <em>NATCS</em>, <em>2</em>(4),
223–233. (<a href="https://doi.org/10.1038/s43588-022-00233-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To study the trade-off between economic, social and health outcomes in the management of a pandemic, DAEDALUS integrates a dynamic epidemiological model of SARS-CoV-2 transmission with a multi-sector economic model, reflecting sectoral heterogeneity in transmission and complex supply chains. The model identifies mitigation strategies that optimize economic production while constraining infections so that hospital capacity is not exceeded but allowing essential services, including much of the education sector, to remain active. The model differentiates closures by economic sector, keeping those sectors open that contribute little to transmission but much to economic output and those that produce essential services as intermediate or final consumption products. In an illustrative application to 63 sectors in the United Kingdom, the model achieves an economic gain of between £161 billion (24\%) and £193 billion (29\%) compared to a blanket lockdown of non-essential activities over six months. Although it has been designed for SARS-CoV-2, DAEDALUS is sufficiently flexible to be applicable to pandemics with different epidemiological characteristics. Smart pandemic mitigation strategies are proposed to strategically close higher-risk economic sectors, while allowing dozens of other economic sectors to continue. This would enable schools to remain open and keep hospitalizations within capacity.},
  archive      = {J_NATCS},
  author       = {Haw, David J. and Forchini, Giovanni and Doohan, Patrick and Christen, Paula and Pianella, Matteo and Johnson, Robert and Bajaj, Sumali and Hogan, Alexandra B. and Winskill, Peter and Miraldo, Marisa and White, Peter J. and Ghani, Azra C. and Ferguson, Neil M. and Smith, Peter C. and Hauck, Katharina D.},
  doi          = {10.1038/s43588-022-00233-0},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {223-233},
  shortjournal = {Nat. Comput. Sci},
  title        = {Optimizing social and economic activity while containing SARS-CoV-2 transmission using DAEDALUS},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antibodies as programmable, bipedal walkers. <em>NATCS</em>,
<em>2</em>(4), 221–222. (<a
href="https://doi.org/10.1038/s43588-022-00222-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stochastic modeling of antibody binding dynamics on patterned antigen substrates suggests the separation distance between adjacent antigens could be a control mechanism for the directed bipedal migration of bound antibodies.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00222-3},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {221-222},
  shortjournal = {Nat. Comput. Sci},
  title        = {Antibodies as programmable, bipedal walkers},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The yin–yang codec for archival DNA storage. <em>NATCS</em>,
<em>2</em>(4), 219–220. (<a
href="https://doi.org/10.1038/s43588-022-00235-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A robust and reliable codec is the backbone for any digital DNA storage. A recent work introduces a codec based on ancient Chinese philosophy, yin–yang, that outperforms other codecs in terms of reliability and physical information density.},
  archive      = {J_NATCS},
  author       = {Gupta, Manish K.},
  doi          = {10.1038/s43588-022-00235-y},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {219-220},
  shortjournal = {Nat. Comput. Sci},
  title        = {The yin–yang codec for archival DNA storage},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Smart lockdowns to control COVID-19. <em>NATCS</em>,
<em>2</em>(4), 217–218. (<a
href="https://doi.org/10.1038/s43588-022-00239-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A dynamic model of SARS-CoV-2 transmission is integrated with a 63-sector economic model to identify control strategies for optimizing economic production while keeping schools and universities operational, and for constraining infections such that emergency hospital capacity is not exceeded.},
  archive      = {J_NATCS},
  author       = {Goenka, Aditya and Liu, Lin},
  doi          = {10.1038/s43588-022-00239-8},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {217-218},
  shortjournal = {Nat. Comput. Sci},
  title        = {Smart lockdowns to control COVID-19},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Gauging urban development with neural networks.
<em>NATCS</em>, <em>2</em>(4), 216. (<a
href="https://doi.org/10.1038/s43588-022-00244-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00244-x},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {216},
  shortjournal = {Nat. Comput. Sci},
  title        = {Gauging urban development with neural networks},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Modeling plant circadian clocks. <em>NATCS</em>,
<em>2</em>(4), 215. (<a
href="https://doi.org/10.1038/s43588-022-00243-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00243-y},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {215},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling plant circadian clocks},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Making light work of hydride transfer. <em>NATCS</em>,
<em>2</em>(4), 214. (<a
href="https://doi.org/10.1038/s43588-022-00241-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00241-0},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {214},
  shortjournal = {Nat. Comput. Sci},
  title        = {Making light work of hydride transfer},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Quantum monte carlo guided by quantum computing.
<em>NATCS</em>, <em>2</em>(4), 213. (<a
href="https://doi.org/10.1038/s43588-022-00242-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00242-z},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {213},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantum monte carlo guided by quantum computing},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A key player in high-performance computing. <em>NATCS</em>,
<em>2</em>(4), 211–212. (<a
href="https://doi.org/10.1038/s43588-022-00245-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Jack Dongarra, Distinguished Professor at the University of Tennessee and recipient of the 2021 A. M. Turing Award, spoke with Nature Computational Science about his contributions to high-performance computing (HPC) and his insights into the future of this field.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00245-w},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {211-212},
  shortjournal = {Nat. Comput. Sci},
  title        = {A key player in high-performance computing},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Why the search for a privacy-preserving data sharing
mechanism is failing. <em>NATCS</em>, <em>2</em>(4), 208–210. (<a
href="https://doi.org/10.1038/s43588-022-00236-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapidly growing demand to share data more openly creates a need for secure and privacy-preserving sharing technologies. However, there are multiple challenges associated with the development of a universal privacy-preserving data sharing mechanism, and existing solutions still fall short of their promises.},
  archive      = {J_NATCS},
  author       = {Stadler, Theresa and Troncoso, Carmela},
  doi          = {10.1038/s43588-022-00236-x},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {208-210},
  shortjournal = {Nat. Comput. Sci},
  title        = {Why the search for a privacy-preserving data sharing mechanism is failing},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). And the turing award goes to…. <em>NATCS</em>,
<em>2</em>(4), 207. (<a
href="https://doi.org/10.1038/s43588-022-00246-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The 2021 A. M. Turing Award celebrates Jack Dongarra’s contributions in high-performance computing, which have had a significant impact in computational science.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00246-9},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {207},
  shortjournal = {Nat. Comput. Sci},
  title        = {And the turing award goes to…},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Opportunities for neuromorphic
computing algorithms and applications. <em>NATCS</em>, <em>2</em>(3),
205. (<a href="https://doi.org/10.1038/s43588-022-00223-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Schuman, Catherine D. and Kulkarni, Shruti R. and Parsa, Maryam and Mitchell, J. Parker and Date, Prasanna and Kay, Bill},
  doi          = {10.1038/s43588-022-00223-2},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {205},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Opportunities for neuromorphic computing algorithms and applications},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A flexible bayesian framework for unbiased estimation of
timescales. <em>NATCS</em>, <em>2</em>(3), 193–204. (<a
href="https://doi.org/10.1038/s43588-022-00214-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Timescales characterize the pace of change for many dynamic processes in nature. They are usually estimated by fitting the exponential decay of data autocorrelation in the time or frequency domain. Here we show that this standard procedure often fails to recover the correct timescales due to a statistical bias arising from the finite sample size. We develop an alternative approach to estimate timescales by fitting the sample autocorrelation or power spectrum with a generative model based on a mixture of Ornstein–Uhlenbeck processes using adaptive approximate Bayesian computations. Our method accounts for finite sample size and noise in data and returns a posterior distribution of timescales that quantifies the estimation uncertainty and can be used for model selection. We demonstrate the accuracy of our method on synthetic data and illustrate its application to recordings from the primate cortex. We provide a customizable Python package that implements our framework via different generative models suitable for diverse applications. A Bayesian method is presented for unbiased estimation of timescales from different types of experimental data; the method quantifies the estimation uncertainty and allows for comparing the alternative hypotheses on the underlying dynamics.},
  archive      = {J_NATCS},
  author       = {Zeraati, Roxana and Engel, Tatiana A. and Levina, Anna},
  doi          = {10.1038/s43588-022-00214-3},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {193-204},
  shortjournal = {Nat. Comput. Sci},
  title        = {A flexible bayesian framework for unbiased estimation of timescales},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Stochastic modeling of antibody binding predicts
programmable migration on antigen patterns. <em>NATCS</em>,
<em>2</em>(3), 179–192. (<a
href="https://doi.org/10.1038/s43588-022-00218-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Viruses and bacteria commonly exhibit spatial repetition of the surface molecules that directly interface with the host immune system. However, the complex interaction of patterned surfaces with immune molecules containing multiple binding domains is poorly understood. We developed a pipeline for constructing mechanistic models of antibody interactions with patterned antigen substrates. Our framework relies on immobilized DNA origami nanostructures decorated with precisely placed antigens. The results revealed that antigen spacing is a spatial control parameter that can be tuned to influence the antibody residence time and migration speed. The model predicts that gradients in antigen spacing can drive persistent, directed antibody migration in the direction of more stable spacing. These results depict antibody–antigen interactions as a computational system where antigen geometry constrains and potentially directs the antibody movement. We propose that this form of molecular programmability could be exploited during the co-evolution of pathogens and immune systems or in the design of molecular machines. A modeling pipeline for the stochastic binding behavior of antibodies on patterned antigen substrates predicts programmable walking behavior that can be manipulated and directed through pattern geometry.},
  archive      = {J_NATCS},
  author       = {Hoffecker, Ian T. and Shaw, Alan and Sorokina, Viktoria and Smyrlaki, Ioanna and Högberg, Björn},
  doi          = {10.1038/s43588-022-00218-z},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {179-192},
  shortjournal = {Nat. Comput. Sci},
  title        = {Stochastic modeling of antibody binding predicts programmable migration on antigen patterns},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Physics-informed recurrent neural network for time dynamics
in optical resonances. <em>NATCS</em>, <em>2</em>(3), 169–178. (<a
href="https://doi.org/10.1038/s43588-022-00215-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resonance structures and features are ubiquitous in optical science. However, capturing their time dynamics in real-world scenarios suffers from long data acquisition time and low analysis accuracy due to slow convergence and limited time windows. Here we report a physics-informed recurrent neural network to forecast the time-domain response of optical resonances and infer corresponding resonance frequencies by acquiring a fraction of the sequence as input. The model is trained in a two-step multi-fidelity framework for high-accuracy forecast, using first a large amount of low-fidelity physical-model-generated synthetic data and then a small set of high-fidelity application-specific data. Through simulations and experiments, we demonstrate that the model is applicable to a wide range of resonances, including dielectric metasurfaces, graphene plasmonics and ultra-strongly coupled Landau polaritons, where our model captures small signal features and learns physical quantities. The demonstrated machine-learning algorithm can help to accelerate the exploration of physical phenomena and device design under resonance-enhanced light–matter interaction. Cascaded gated-recurrent-unit networks trained through a physics-informed multi-fidelity approach can accurately forecast long time sequences and capture their dynamics in a wide range of optical resonance structures and features.},
  archive      = {J_NATCS},
  author       = {Tang, Yingheng and Fan, Jichao and Li, Xinwei and Ma, Jianzhu and Qi, Minghao and Yu, Cunxi and Gao, Weilu},
  doi          = {10.1038/s43588-022-00215-2},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {169-178},
  shortjournal = {Nat. Comput. Sci},
  title        = {Physics-informed recurrent neural network for time dynamics in optical resonances},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Autonomous inference of complex network dynamics from
incomplete and noisy data. <em>NATCS</em>, <em>2</em>(3), 160–168. (<a
href="https://doi.org/10.1038/s43588-022-00217-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The availability of empirical data that capture the structure and behaviour of complex networked systems has been greatly increased in recent years; however, a versatile computational toolbox for unveiling a complex system’s nodal and interaction dynamics from data remains elusive. Here we develop a two-phase approach for the autonomous inference of complex network dynamics, and its effectiveness is demonstrated by the tests of inferring neuronal, genetic, social and coupled oscillator dynamics on various synthetic and real networks. Importantly, the approach is robust to incompleteness and noises, including low resolution, observational and dynamical noises, missing and spurious links, and dynamical heterogeneity. We apply the two-phase approach to infer the early spreading dynamics of influenza A flu on the worldwide airline network, and the inferred dynamical equation can also capture the spread of severe acute respiratory syndrome and coronavirus disease 2019. These findings together offer an avenue to discover the hidden microscopic mechanisms of a broad array of real networked systems. The authors propose a two-phase approach to solve the inverse problem of inferring dynamical principles of complex systems from incomplete and noisy data, and apply it to infer the spreading dynamics of H1N1, SARS, and COVID-19.},
  archive      = {J_NATCS},
  author       = {Gao, Ting-Ting and Yan, Gang},
  doi          = {10.1038/s43588-022-00217-0},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {160-168},
  shortjournal = {Nat. Comput. Sci},
  title        = {Autonomous inference of complex network dynamics from incomplete and noisy data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Dynamic forecasting of severe acute graft-versus-host
disease after transplantation. <em>NATCS</em>, <em>2</em>(3), 153–159.
(<a href="https://doi.org/10.1038/s43588-022-00213-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Forecasting of severe acute graft-versus-host disease (aGVHD) after transplantation is a challenging ‘large p, small n’ problem that suffers from nonuniform data sampling. We propose a dynamic probabilistic algorithm, daGOAT, that accommodates sampling heterogeneity, integrates multidimensional clinical data and continuously updates the daily risk score for severe aGVHD onset within a two-week moving window. In the studied cohorts, the cross-validated area under the receiver operator characteristic curve (AUROC) of daGOAT rose steadily after transplantation and peaked at ≥0.78 in both the adult and pediatric cohorts, outperforming the two-biomarker MAGIC score, three-biomarker Ann Arbor score, peri-transplantation features-based models and XGBoost. Simulation experiments indicated that the daGOAT algorithm is well suited for short time-series scenarios where the underlying process for event generation is smooth, multidimensional and where there are frequent and irregular data missing. daGOAT’s broader utility was demonstrated by performance testing on a remotely different task, that is, prediction of imminent human postural change based on smartphone inertial sensor time-series data. A dynamic probabilistic algorithm that integrates many variables over time for forecasting severe acute graft-versus-host disease is proposed to improve healthcare decisions for individual patients on a case-by-case basis.},
  archive      = {J_NATCS},
  author       = {Liu, Xueou and Cao, Yigeng and Guo, Ye and Gong, Xiaowen and Feng, Yahui and Wang, Yao and Wang, Mingyang and Cui, Mengxuan and Guo, Wenwen and Zhang, Luyang and Zhao, Ningning and Song, Xiaoqiang and Zheng, Xuetong and Chen, Xia and Shen, Qiujin and Zhang, Song and Song, Zhen and Li, Linfeng and Feng, Sizhou and Han, Mingzhe and Zhu, Xiaofan and Jiang, Erlie and Chen, Junren},
  doi          = {10.1038/s43588-022-00213-4},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {153-159},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dynamic forecasting of severe acute graft-versus-host disease after transplantation},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accurate assembly of multi-end RNA-seq data with scallop2.
<em>NATCS</em>, <em>2</em>(3), 148–152. (<a
href="https://doi.org/10.1038/s43588-022-00216-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern RNA-sequencing (RNA-seq) protocols can produce multi-end data, where multiple reads originating from the same transcript are attached to the same barcode. The long-range information in the multi-end reads is beneficial in phasing complicated spliced isoforms, but assembly algorithms that leverage such information are lacking. Here we introduce Scallop2, a reference-based assembler optimized for multi-end RNA-seq data. The algorithmic core of Scallop2 consists of three steps: (1) using an algorithm to ‘bridge’ multi-end reads into single-end phasing paths in the context of a splice graph, (2) employing a method to refine erroneous splice graphs by utilizing multi-end reads that fail to bridge and (3) piping the refined splice graph and bridged phasing paths into an algorithm that integrates multiple phase-preserving decompositions. Tested on 561 cells in two Smart-seq3 datasets and on ten Illumina paired-end RNA-seq samples, Scallop2 substantially improves the assembly accuracy compared with two popular assemblers (StringTie2 and Scallop). Scallop2 enables a more accurate assembly of transcripts at both single-cell resolution and sample level through a suite of algorithms that leverage the multi-end and paired-end information in Smart-seq3 and Illumina RNA-seq data.},
  archive      = {J_NATCS},
  author       = {Zhang, Qimin and Shi, Qian and Shao, Mingfu},
  doi          = {10.1038/s43588-022-00216-1},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {148-152},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accurate assembly of multi-end RNA-seq data with scallop2},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards accurate predictions of NMR chemical shifts in
liquid solutions. <em>NATCS</em>, <em>2</em>(3), 146–147. (<a
href="https://doi.org/10.1038/s43588-022-00204-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fully automated, high-throughput computational framework accurately predicts stable species in liquid solutions by computing the nuclear magnetic resonance chemical shifts. Data collected from the framework can provide fingerprints to guide the rational design of liquid solutions with optimal properties.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00204-5},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {146-147},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards accurate predictions of NMR chemical shifts in liquid solutions},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Post-transplant dynamic risk prediction. <em>NATCS</em>,
<em>2</em>(3), 144–145. (<a
href="https://doi.org/10.1038/s43588-022-00220-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the risk of acute graft-versus-host-disease after transplantation is challenging due to the presence of multimodal data and continuous evolution of disease states. A dynamic probabilistic algorithm has recently been proposed to address these challenges.},
  archive      = {J_NATCS},
  author       = {Gupta, Vibhuti},
  doi          = {10.1038/s43588-022-00220-5},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {144-145},
  shortjournal = {Nat. Comput. Sci},
  title        = {Post-transplant dynamic risk prediction},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bridge over troubled transcripts. <em>NATCS</em>,
<em>2</em>(3), 142–143. (<a
href="https://doi.org/10.1038/s43588-022-00219-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scallop2, a tool that enables accurate reference-guided transcriptome assembly, capitalizes on recent developments in short-read sequencing protocols for single-cell RNA sequencing by leveraging multi-end and paired-end information.},
  archive      = {J_NATCS},
  author       = {Parada, Guillermo E. and Hemberg, Martin},
  doi          = {10.1038/s43588-022-00219-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {142-143},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bridge over troubled transcripts},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Correlates of protection via modeling. <em>NATCS</em>,
<em>2</em>(3), 140–141. (<a
href="https://doi.org/10.1038/s43588-022-00221-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes a mathematical model of SARS-CoV-2 to help identify mechanistic correlates of protection, which can be used to assist in determining vaccine efficacy.},
  archive      = {J_NATCS},
  author       = {Schiffer, Joshua T.},
  doi          = {10.1038/s43588-022-00221-4},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {140-141},
  shortjournal = {Nat. Comput. Sci},
  title        = {Correlates of protection via modeling},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Machine learning for spectral properties. <em>NATCS</em>,
<em>2</em>(3), 139. (<a
href="https://doi.org/10.1038/s43588-022-00225-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00225-0},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {139},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning for spectral properties},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022l). Theoretical insights into single-atom catalysts.
<em>NATCS</em>, <em>2</em>(3), 138. (<a
href="https://doi.org/10.1038/s43588-022-00226-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00226-z},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {138},
  shortjournal = {Nat. Comput. Sci},
  title        = {Theoretical insights into single-atom catalysts},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Cultural diversity through the lenses of ecology.
<em>NATCS</em>, <em>2</em>(3), 137. (<a
href="https://doi.org/10.1038/s43588-022-00224-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00224-1},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {137},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cultural diversity through the lenses of ecology},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computational chemistry for all. <em>NATCS</em>,
<em>2</em>(3), 134–136. (<a
href="https://doi.org/10.1038/s43588-022-00209-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Laura Gagliardi, professor at the University of Chicago and director of the Chicago Center for Theoretical Chemistry, shares with Nature Computational Science her research trajectory and projects, insights into the synergy between experimental and computational chemistry, and her advice for women and young scientists.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin and Pan, Jie},
  doi          = {10.1038/s43588-022-00209-0},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {134-136},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational chemistry for all},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Acing the peer review process. <em>NATCS</em>,
<em>2</em>(3), 133. (<a
href="https://doi.org/10.1038/s43588-022-00227-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide some guidelines on how to write a constructive review for Nature Computational Science.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00227-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {133},
  shortjournal = {Nat. Comput. Sci},
  title        = {Acing the peer review process},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Balancing efficient analysis and storage
of quantitative genomics data with the d4 format and d4tools.
<em>NATCS</em>, <em>2</em>(2), 132. (<a
href="https://doi.org/10.1038/s43588-022-00211-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Hou, Hao and Pedersen, Brent and Quinlan, Aaron},
  doi          = {10.1038/s43588-022-00211-6},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {132},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Balancing efficient analysis and storage of quantitative genomics data with the d4 format and d4tools},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Modeling how antibody responses may determine the efficacy
of COVID-19 vaccines. <em>NATCS</em>, <em>2</em>(2), 123–131. (<a
href="https://doi.org/10.1038/s43588-022-00198-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the efficacy of COVID-19 vaccines would aid vaccine development and usage strategies, which is of importance given their limited supplies. Here we develop a multiscale mathematical model that proposes mechanistic links between COVID-19 vaccine efficacies and the neutralizing antibody (NAb) responses they elicit. We hypothesized that the collection of all NAbs would constitute a shape space and that responses of individuals are random samples from this space. We constructed the shape space by analyzing reported in vitro dose–response curves of ~80 NAbs. Sampling NAb subsets from the space, we recapitulated the responses of convalescent patients. We assumed that vaccination would elicit similar NAb responses. We developed a model of within-host SARS-CoV-2 dynamics, applied it to virtual patient populations and, invoking the NAb responses above, predicted vaccine efficacies. Our predictions quantitatively captured the efficacies from clinical trials. Our study thus suggests plausible mechanistic underpinnings of COVID-19 vaccines and generates testable hypotheses for establishing them. A multiscale model is presented to quantitatively predict COVID-19 vaccine efficacies by describing the generation, activity and diversity of neutralizing antibodies.},
  archive      = {J_NATCS},
  author       = {Padmanabhan, Pranesh and Desikan, Rajat and Dixit, Narendra M.},
  doi          = {10.1038/s43588-022-00198-0},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {123-131},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling how antibody responses may determine the efficacy of COVID-19 vaccines},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An automated framework for high-throughput predictions of
NMR chemical shifts within liquid solutions. <em>NATCS</em>,
<em>2</em>(2), 112–122. (<a
href="https://doi.org/10.1038/s43588-022-00200-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying stable speciation in multi-component liquid solutions is fundamentally important to areas from electrochemistry to organic chemistry and biomolecular systems. Here we introduce a fully automated, high-throughput computational framework for the accurate prediction of stable species in liquid solutions by computing the nuclear magnetic resonance (NMR) chemical shifts. The framework automatically extracts and categorizes hundreds of thousands of atomic clusters from classical molecular dynamics simulations, identifies the most stable species in solution and calculates their NMR chemical shifts via density functional theory calculations. Additionally, the framework creates a database of computed chemical shifts for liquid solutions across a wide chemical and parameter space. We compare our computational results to experimental measurements for magnesium bis(trifluoromethanesulfonyl)imide Mg(TFSI)2 salt in dimethoxyethane solvent. Our analysis of the Mg2+ solvation structural evolutions reveals key factors that influence the accuracy of NMR chemical shift predictions in liquid solutions. Furthermore, we show how the framework reduces the performance of over 300 13C and 600 1H density functional theory chemical shift predictions to a single submission procedure. A fully automated, high-throughput computational framework is presented to predict stable species in liquid solutions. This framework combines density functional theory with classical molecular dynamics simulations to compute the NMR chemical shifts.},
  archive      = {J_NATCS},
  author       = {Atwi, Rasha and Chen, Ying and Han, Kee Sung and Mueller, Karl T. and Murugesan, Vijayakumar and Rajput, Nav Nidhi},
  doi          = {10.1038/s43588-022-00200-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {112-122},
  shortjournal = {Nat. Comput. Sci},
  title        = {An automated framework for high-throughput predictions of NMR chemical shifts within liquid solutions},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Diffraction-limited molecular cluster quantification with
bayesian nonparametrics. <em>NATCS</em>, <em>2</em>(2), 102–111. (<a
href="https://doi.org/10.1038/s43588-022-00197-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Life’s fundamental processes involve multiple molecules operating in close proximity within cells. To probe the molecular composition of such small (diffraction-limited) regions, experiments often report on the total fluorescence intensity emitted from labeled molecules within. Methods exist to enumerate total fluorophore numbers (for example, step counting by photobleaching); however, these methods cannot treat photophysical dynamics nor learn their associated kinetic rates. Here we propose a method to simultaneously enumerate fluorophores and determine their photophysical properties. Although our focus here is on photophysical dynamics, such dynamics can also serve as a proxy for other types of dynamics such as the kinetics of assembly and disassembly of clusters. As the number of active fluorescent molecules at any given time is unknown, we rely on Bayesian nonparametrics to derive our kinetic estimates. We provide a versatile framework for enumerating up to 100 fluorophores from brightness time traces, benchmarked on synthetic and real datasets. The authors demonstrate a robust and rigorous framework that can enumerate up to 100 fluorescent labels in a diffraction limited spot using Bayesian nonparametrics.},
  archive      = {J_NATCS},
  author       = {Bryan IV, J. Shepard and Sgouralis, Ioannis and Pressé, Steve},
  doi          = {10.1038/s43588-022-00197-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {102-111},
  shortjournal = {Nat. Comput. Sci},
  title        = {Diffraction-limited molecular cluster quantification with bayesian nonparametrics},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A laboratory and simulation platform to integrate individual
life history traits and population dynamics. <em>NATCS</em>,
<em>2</em>(2), 90–101. (<a
href="https://doi.org/10.1038/s43588-022-00190-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding populations is important as they are a fundamental level of biological organization. Individual traits such as aging and lifespan interact in complex ways to determine birth and death, and thereby influence population dynamics; however, we lack a deep understanding of the relationships between individual traits and population dynamics. To address this challenge, we established a laboratory population using the model organism Caenorhabditis elegans and an individual-based computational simulation informed by measurements of real worms. The simulation realistically models individual worms and the behavior of the laboratory population. To elucidate the role of aging in population dynamics, we analyzed old age as a cause of death and showed, using computer simulations, that it was influenced by maximum lifespan, rate of adult culling and progeny number/food stability. Notably, populations displayed a tipping point for aging as the primary cause of adult death. Our work establishes a conceptual framework that could be used for better understanding why certain animals die of old age in the wild. To help determine how life history traits of individuals result in emergent properties of a population, laboratory studies of Caenorhabditis elegans were combined with an individual-based simulation, pointing out to potential factors that influence old age as a cause of death.},
  archive      = {J_NATCS},
  author       = {Scharf, Andrea and Mitteldorf, Josh and Armstead, Brinda and Schneider, Daniel and Jin, He and Kocsisova, Zuzana and Tan, Chieh-Hsiang and Sanchez, Francesca and Brady, Brian and Ram, Natasha and DiAntonio, Gabriel B. and Wilson, Andrea M. and Kornfeld, Kerry},
  doi          = {10.1038/s43588-022-00190-8},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {90-101},
  shortjournal = {Nat. Comput. Sci},
  title        = {A laboratory and simulation platform to integrate individual life history traits and population dynamics},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Network cartographs for interpretable visualizations.
<em>NATCS</em>, <em>2</em>(2), 84–89. (<a
href="https://doi.org/10.1038/s43588-022-00199-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Networks offer an intuitive visual representation of complex systems. Important network characteristics can often be recognized by eye and, in turn, patterns that stand out visually often have a meaningful interpretation. In conventional network layout algorithms, however, the precise determinants of a node’s position within a layout are difficult to decipher and to control. Here we propose an approach for directly encoding arbitrary structural or functional network characteristics into node positions. We introduce a series of two- and three-dimensional layouts, benchmark their efficiency for model networks, and demonstrate their power for elucidating structure-to-function relationships in large-scale biological networks. Networks offer a powerful visual representation of complex systems. This study introduces network visualizations that are easy to interpret and can help explore large datasets, such as the map of all molecular interactions in the cell.},
  archive      = {J_NATCS},
  author       = {Hütter, Christiane V. R. and Sin, Celine and Müller, Felix and Menche, Jörg},
  doi          = {10.1038/s43588-022-00199-z},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {84-89},
  shortjournal = {Nat. Comput. Sci},
  title        = {Network cartographs for interpretable visualizations},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accelerating minimap2 for long-read sequencing applications
on modern CPUs. <em>NATCS</em>, <em>2</em>(2), 78–83. (<a
href="https://doi.org/10.1038/s43588-022-00201-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing is now routinely used at scale for genomics and transcriptomics applications. Mapping long reads or a draft genome assembly to a reference sequence is often one of the most time-consuming steps in these applications. Here we present techniques to accelerate minimap2, a widely used software for this task. We present multiple optimizations using single-instruction multiple-data parallelization, efficient cache utilization and a learned index data structure to accelerate the three main computational modules of minimap2: seeding, chaining and pairwise sequence alignment. These optimizations result in an up to 1.8-fold reduction of end-to-end mapping time of minimap2 while maintaining identical output. mm2-fast is an accelerated version of minimap2, a popular software for long-read data analysis. mm2-fast introduces high-performance parallel computing techniques to reduce the overall runtime of minimap2.},
  archive      = {J_NATCS},
  author       = {Kalikar, Saurabh and Jain, Chirag and Vasimuddin, Md and Misra, Sanchit},
  doi          = {10.1038/s43588-022-00201-8},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {78-83},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating minimap2 for long-read sequencing applications on modern CPUs},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cartographs enable interpretation of complex network
visualizations. <em>NATCS</em>, <em>2</em>(2), 76–77. (<a
href="https://doi.org/10.1038/s43588-022-00203-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Networks offer a powerful visual representation of complex systems. Cartographs introduce a diverse set of network layouts for highlighting and visually inspecting chosen characteristics of a network. The resulting visualizations are interpretable and can be used to explore complex datasets, such as large-scale biological networks.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00203-6},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {76-77},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cartographs enable interpretation of complex network visualizations},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Counting biomolecules with bayesian inference.
<em>NATCS</em>, <em>2</em>(2), 74–75. (<a
href="https://doi.org/10.1038/s43588-022-00202-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A combination of Bayesian inference, physics modeling, and Markov chain Monte Carlo sampling allows for accurate inference of biomolecule numbers and their photophysical state in cellular clusters.},
  archive      = {J_NATCS},
  author       = {Masson, Jean-Baptiste},
  doi          = {10.1038/s43588-022-00202-7},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {74-75},
  shortjournal = {Nat. Comput. Sci},
  title        = {Counting biomolecules with bayesian inference},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The art of mechanistic modeling in biology. <em>NATCS</em>,
<em>2</em>(2), 72–73. (<a
href="https://doi.org/10.1038/s43588-021-00187-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A combination of experimental evaluation with computational modeling sheds light on how aging and limited lifespan influence population dynamics.},
  archive      = {J_NATCS},
  author       = {Kalirad, Ata and Sommer, Ralf J.},
  doi          = {10.1038/s43588-021-00187-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {72-73},
  shortjournal = {Nat. Comput. Sci},
  title        = {The art of mechanistic modeling in biology},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Revisiting signal analysis in the big data era.
<em>NATCS</em>, <em>2</em>(2), 70–71. (<a
href="https://doi.org/10.1038/s43588-022-00210-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fast and accurate time–frequency analysis is challenging for many applications, especially in the current big data era. A recent work introduces a fast continuous wavelet transform that effectively boosts the analysis speed without sacrificing the resolution of the result.},
  archive      = {J_NATCS},
  author       = {Srivastava, Madhur},
  doi          = {10.1038/s43588-022-00210-7},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {70-71},
  shortjournal = {Nat. Comput. Sci},
  title        = {Revisiting signal analysis in the big data era},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards quantum computing of turbulence. <em>NATCS</em>,
<em>2</em>(2), 68–69. (<a
href="https://doi.org/10.1038/s43588-022-00189-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Precise numerical simulations of turbulent flows in practical applications are still challenging. A recent quantum-inspired computational method improves the way to account for the interscale correlations in turbulence, and further sheds light on the development of quantum computing algorithms for efficient turbulence simulations.},
  archive      = {J_NATCS},
  author       = {Fukagata, Koji},
  doi          = {10.1038/s43588-022-00189-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {68-69},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards quantum computing of turbulence},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Co-occurrence and similarity revisited. <em>NATCS</em>,
<em>2</em>(2), 67. (<a
href="https://doi.org/10.1038/s43588-022-00206-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00206-3},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {67},
  shortjournal = {Nat. Comput. Sci},
  title        = {Co-occurrence and similarity revisited},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022n). Theory and experimentation to assess reactivity.
<em>NATCS</em>, <em>2</em>(2), 66. (<a
href="https://doi.org/10.1038/s43588-022-00207-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00207-2},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {66},
  shortjournal = {Nat. Comput. Sci},
  title        = {Theory and experimentation to assess reactivity},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Modeling the building blocks of life. <em>NATCS</em>,
<em>2</em>(2), 65. (<a
href="https://doi.org/10.1038/s43588-022-00205-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00205-4},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {65},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling the building blocks of life},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). Reinstating functionality of complex systems.
<em>NATCS</em>, <em>2</em>(2), 64. (<a
href="https://doi.org/10.1038/s43588-022-00208-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00208-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {64},
  shortjournal = {Nat. Comput. Sci},
  title        = {Reinstating functionality of complex systems},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Crypto and technology for the people. <em>NATCS</em>,
<em>2</em>(2), 62–63. (<a
href="https://doi.org/10.1038/s43588-022-00196-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Seny Kamara, associate professor at Brown University, talks to Nature Computational Science about his current research focus on the intersection between social responsibility and cryptography/technology, as well as about the need for multidisciplinary work in this arena.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00196-2},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {62-63},
  shortjournal = {Nat. Comput. Sci},
  title        = {Crypto and technology for the people},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cover runners-up of 2021. <em>NATCS</em>, <em>2</em>(2), 61.
(<a href="https://doi.org/10.1038/s43588-022-00212-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We highlight some of our favorite cover suggestions, submitted by authors last year, that were not ultimately selected for the journal.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00212-5},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {61},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cover runners-up of 2021},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Publisher correction: A rigid body framework for
multicellular modeling. <em>NATCS</em>, <em>2</em>(1), 59. (<a
href="https://doi.org/10.1038/s43588-021-00188-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Brown, Phillip J. and Green, J. Edward F. and Binder, Benjamin J. and Osborne, James M.},
  doi          = {10.1038/s43588-021-00188-8},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {59},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: A rigid body framework for multicellular modeling},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The fast continuous wavelet transformation (fCWT) for
real-time, high-quality, noise-resistant time–frequency analysis.
<em>NATCS</em>, <em>2</em>(1), 47–58. (<a
href="https://doi.org/10.1038/s43588-021-00183-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The spectral analysis of signals is currently either dominated by the speed–accuracy trade-off or ignores a signal’s often non-stationary character. Here we introduce an open-source algorithm to calculate the fast continuous wavelet transform (fCWT). The parallel environment of fCWT separates scale-independent and scale-dependent operations, while utilizing optimized fast Fourier transforms that exploit downsampled wavelets. fCWT is benchmarked for speed against eight competitive algorithms, tested on noise resistance and validated on synthetic electroencephalography and in vivo extracellular local field potential data. fCWT is shown to have the accuracy of CWT, to have 100 times higher spectral resolution than algorithms equal in speed, to be 122 times and 34 times faster than the reference and fastest state-of-the-art implementations and we demonstrate its real-time performance, as confirmed by the real-time analysis ratio. fCWT provides an improved balance between speed and accuracy, which enables real-time, wide-band, high-quality, time–frequency analysis of non-stationary noisy signals. The authors present an open-source framework that enables fast and accurate time–frequency analysis of signals and demonstrate it on real-world applications, such as signals from the brain–computer interface.},
  archive      = {J_NATCS},
  author       = {Arts, Lukas P. A. and van den Broek, Egon. L.},
  doi          = {10.1038/s43588-021-00183-z},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {47-58},
  shortjournal = {Nat. Comput. Sci},
  title        = {The fast continuous wavelet transformation (fCWT) for real-time, high-quality, noise-resistant time–frequency analysis},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A deep manifold-regularized learning model for improving
phenotype prediction from multi-modal data. <em>NATCS</em>,
<em>2</em>(1), 38–46. (<a
href="https://doi.org/10.1038/s43588-021-00185-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The phenotypes of complex biological systems are fundamentally driven by various multi-scale mechanisms. Multi-modal data, such as single-cell multi-omics data, enable a deeper understanding of underlying complex mechanisms across scales for phenotypes. We have developed an interpretable regularized learning model, deepManReg, to predict phenotypes from multi-modal data. First, deepManReg employs deep neural networks to learn cross-modal manifolds and then to align multi-modal features onto a common latent space. Second, deepManReg uses cross-modal manifolds as a feature graph to regularize the classifiers for improving phenotype predictions and also for prioritizing the multi-modal features and cross-modal interactions for the phenotypes. We apply deepManReg to (1) an image dataset of handwritten digits with multi-features and (2) single-cell multi-modal data (Patch-seq data) including transcriptomics and electrophysiology for neuronal cells in the mouse brain. We show that deepManReg improves phenotype prediction in both datasets, and also prioritizes genes and electrophysiological features for the phenotypes of neuronal cells. deepManReg uses deep neural networks to map various data types onto a topological space (manifolds) and unfold unseen data connections, thus improving prediction of phenotypes from multi-modal data.},
  archive      = {J_NATCS},
  author       = {Nguyen, Nam D. and Huang, Jiawei and Wang, Daifeng},
  doi          = {10.1038/s43588-021-00185-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {38-46},
  shortjournal = {Nat. Comput. Sci},
  title        = {A deep manifold-regularized learning model for improving phenotype prediction from multi-modal data},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A quantum-inspired approach to exploit turbulence
structures. <em>NATCS</em>, <em>2</em>(1), 30–37. (<a
href="https://doi.org/10.1038/s43588-021-00181-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding turbulence is key to our comprehension of many natural and technological flow processes. At the heart of this phenomenon lies its intricate multiscale nature, describing the coupling between different-sized eddies in space and time. Here we analyze the structure of turbulent flows by quantifying correlations between different length scales using methods inspired from quantum many-body physics. We present the results for interscale correlations of two paradigmatic flow examples, and use these insights along with tensor network theory to design a structure-resolving algorithm for simulating turbulent flows. With this algorithm, we find that the incompressible Navier–Stokes equations can be accurately solved even when reducing the number of parameters required to represent the velocity field by more than one order of magnitude compared to direct numerical simulation. Our quantum-inspired approach provides a pathway towards conducting computational fluid dynamics on quantum computers. Tensor networks exploit the structure of turbulence to offer a compressed description of flows, which leads to efficient fluid simulation algorithms that can be implemented on both classical and quantum computers.},
  archive      = {J_NATCS},
  author       = {Gourianov, Nikita and Lubasch, Michael and Dolgov, Sergey and van den Berg, Quincy Y. and Babaee, Hessam and Givi, Peyman and Kiffner, Martin and Jaksch, Dieter},
  doi          = {10.1038/s43588-021-00181-1},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {30-37},
  shortjournal = {Nat. Comput. Sci},
  title        = {A quantum-inspired approach to exploit turbulence structures},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficient sampling of high-dimensional free energy
landscapes using adaptive reinforced dynamics. <em>NATCS</em>,
<em>2</em>(1), 20–29. (<a
href="https://doi.org/10.1038/s43588-021-00173-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enhanced sampling methods such as metadynamics and umbrella sampling have become essential tools for exploring the configuration space of molecules and materials. At the same time, they have long faced a number of issues such as the inefficiency when dealing with a large number of collective variables (CVs) or systems with high free energy barriers. Here we show that, with clustering and adaptive tuning techniques, the reinforced dynamics (RiD) scheme can be used to efficiently explore the configuration space and free energy landscapes with a large number of CVs or systems with high free energy barriers. We illustrate this by studying various representative and challenging examples. First we demonstrate the efficiency of adaptive RiD compared with other methods and construct the nine-dimensional (9D) free energy landscape of a peptoid trimer, which has energy barriers of more than 8 kcal mol−1. We then study the folding of the protein chignolin using 18 CVs. In this case, both the folding and unfolding rates are observed to be 4.30 μs−1. Finally, we propose a protein structure refinement protocol based on RiD. This protocol allows us to efficiently employ more than 100 CVs for exploring the landscape of protein structures and it gives rise to an overall improvement of 14.6 units over the initial global distance test–high accuracy (GDT-HA) score. The authors have developed an adaptive reinforced dynamics approach, which improves the efficiency when exploring the configurational space and free energy landscape of large biomolecules, such as proteins.},
  archive      = {J_NATCS},
  author       = {Wang, Dongdong and Wang, Yanze and Chang, Junhan and Zhang, Linfeng and Wang, Han and E., Weinan},
  doi          = {10.1038/s43588-021-00173-1},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {20-29},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient sampling of high-dimensional free energy landscapes using adaptive reinforced dynamics},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Opportunities for neuromorphic computing algorithms and
applications. <em>NATCS</em>, <em>2</em>(1), 10–19. (<a
href="https://doi.org/10.1038/s43588-021-00184-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuromorphic computing technologies will be important for the future of computing, but much of the work in neuromorphic computing has focused on hardware development. Here, we review recent results in neuromorphic computing algorithms and applications. We highlight characteristics of neuromorphic computing technologies that make them attractive for the future of computing and we discuss opportunities for future development of algorithms and applications on these systems. There is still a wide variety of challenges that restrict the rapid growth of neuromorphic algorithmic and application development. Addressing these challenges is essential for the research community to be able to effectively use neuromorphic computers in the future.},
  archive      = {J_NATCS},
  author       = {Schuman, Catherine D. and Kulkarni, Shruti R. and Parsa, Maryam and Mitchell, J. Parker and Date, Prasanna and Kay, Bill},
  doi          = {10.1038/s43588-021-00184-y},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {10-19},
  shortjournal = {Nat. Comput. Sci},
  title        = {Opportunities for neuromorphic computing algorithms and applications},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Interpretable multi-modal data integration. <em>NATCS</em>,
<em>2</em>(1), 8–9. (<a
href="https://doi.org/10.1038/s43588-021-00186-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrating multi-modal features is challenging due to the differences in the underlying distributions of each data type and the nonlinear associations across modalities. The deepManReg model improves the identification and interpretability of associations between modalities defining complex phenotypes.},
  archive      = {J_NATCS},
  author       = {Osorio, Daniel},
  doi          = {10.1038/s43588-021-00186-w},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {8-9},
  shortjournal = {Nat. Comput. Sci},
  title        = {Interpretable multi-modal data integration},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The curse of dimensionality loses its power. <em>NATCS</em>,
<em>2</em>(1), 6–7. (<a
href="https://doi.org/10.1038/s43588-021-00182-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An adaptive and computationally efficient machine-learning-based biasing technique for rare-event sampling is introduced, allowing an effective generation of high-dimensional free energy surfaces associated with complex processes, such as protein folding.},
  archive      = {J_NATCS},
  author       = {Tuckerman, Mark E.},
  doi          = {10.1038/s43588-021-00182-0},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {6-7},
  shortjournal = {Nat. Comput. Sci},
  title        = {The curse of dimensionality loses its power},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Effect of spatial sampling on phylogeography.
<em>NATCS</em>, <em>2</em>(1), 5. (<a
href="https://doi.org/10.1038/s43588-022-00194-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-022-00194-4},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {5},
  shortjournal = {Nat. Comput. Sci},
  title        = {Effect of spatial sampling on phylogeography},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Predicting oxidation states iteratively. <em>NATCS</em>,
<em>2</em>(1), 4. (<a
href="https://doi.org/10.1038/s43588-022-00192-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-022-00192-6},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting oxidation states iteratively},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Mining literature for perovskite solar cells.
<em>NATCS</em>, <em>2</em>(1), 3. (<a
href="https://doi.org/10.1038/s43588-022-00193-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-022-00193-5},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {3},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mining literature for perovskite solar cells},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022l). Machine learning to guide mathematicians. <em>NATCS</em>,
<em>2</em>(1), 2. (<a
href="https://doi.org/10.1038/s43588-022-00191-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-022-00191-7},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning to guide mathematicians},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A look back at 2021. <em>NATCS</em>, <em>2</em>(1), 1. (<a
href="https://doi.org/10.1038/s43588-022-00195-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {On our one-year anniversary at Nature Computational Science, we discuss some of the research that we published over the course of the last year.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-022-00195-3},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Comput. Sci},
  title        = {A look back at 2021},
  volume       = {2},
  year         = {2022},
}
</textarea>
</details></li>
</ul>

</body>
</html>
