<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth---354">NMETH - 354</h2>
<ul>
<li><details>
<summary>
(2022a). Author correction: Unsupervised discovery of tissue
architecture in multiplexed imaging. <em>NMETH</em>, <em>19</em>(12),
1662. (<a href="https://doi.org/10.1038/s41592-022-01717-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kim, Junbum and Rustam, Samir and Mosquera, Juan Miguel and Randell, Scott H. and Shaykhiev, Renat and Rendeiro, André F. and Elemento, Olivier},
  doi          = {10.1038/s41592-022-01717-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1662},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Unsupervised discovery of tissue architecture in multiplexed imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Unsupervised discovery of tissue architecture in
multiplexed imaging. <em>NMETH</em>, <em>19</em>(12), 1653–1661. (<a
href="https://doi.org/10.1038/s41592-022-01657-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed imaging and spatial transcriptomics enable highly resolved spatial characterization of cellular phenotypes, but still largely depend on laborious manual annotation to understand higher-order patterns of tissue organization. As a result, higher-order patterns of tissue organization are poorly understood and not systematically connected to disease pathology or clinical outcomes. To address this gap, we developed an approach called UTAG to identify and quantify microanatomical tissue structures in multiplexed images without human intervention. Our method combines information on cellular phenotypes with the physical proximity of cells to accurately identify organ-specific microanatomical domains in healthy and diseased tissue. We apply our method to various types of images across healthy and disease states to show that it can consistently detect higher-level architectures in human tissues, quantify structural differences between healthy and diseased tissue, and reveal tissue organization patterns at the organ scale. Unsupervised discovery of tissue architecture with graphs (UTAG) combines information on cellular morphology and protein expression with the physical proximity of cells to identify architectural domains from highly multiplexed imaging data.},
  archive      = {J_NMETH},
  author       = {Kim, Junbum and Rustam, Samir and Mosquera, Juan Miguel and Randell, Scott H. and Shaykhiev, Renat and Rendeiro, André F. and Elemento, Olivier},
  doi          = {10.1038/s41592-022-01657-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1653-1661},
  shortjournal = {Nat. Meth.},
  title        = {Unsupervised discovery of tissue architecture in multiplexed imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Capturing the start point of the virus–cell interaction with
high-speed 3D single-virus tracking. <em>NMETH</em>, <em>19</em>(12),
1642–1652. (<a
href="https://doi.org/10.1038/s41592-022-01672-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The early stages of the virus–cell interaction have long evaded observation by existing microscopy methods due to the rapid diffusion of virions in the extracellular space and the large three-dimensional cellular structures involved. Here we present an active-feedback single-particle tracking method with simultaneous volumetric imaging of the live cell environment called 3D-TrIm to address this knowledge gap. 3D-TrIm captures the extracellular phase of the infectious cycle in what we believe is unprecedented detail. We report what are, to our knowledge, previously unobserved phenomena in the early stages of the virus–cell interaction, including skimming contact events at the millisecond timescale, orders of magnitude change in diffusion coefficient upon binding and cylindrical and linear diffusion modes along cellular protrusions. Finally, we demonstrate how this method can move single-particle tracking from simple monolayer culture toward more tissue-like conditions by tracking single virions in tightly packed epithelial cells. This multiresolution method presents opportunities for capturing fast, three-dimensional processes in biological systems. 3D-TrIm combines real-time, high-speed, single-virion tracking with volumetric imaging of cellular microenvironments.},
  archive      = {J_NMETH},
  author       = {Johnson, Courtney and Exell, Jack and Lin, Yuxin and Aguilar, Jonathan and Welsher, Kevin D.},
  doi          = {10.1038/s41592-022-01672-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1642-1652},
  shortjournal = {Nat. Meth.},
  title        = {Capturing the start point of the virus–cell interaction with high-speed 3D single-virus tracking},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cellpose 2.0: How to train your own model. <em>NMETH</em>,
<em>19</em>(12), 1634–1641. (<a
href="https://doi.org/10.1038/s41592-022-01663-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pretrained neural network models for biological segmentation can provide good out-of-the-box results for many image types. However, such models do not allow users to adapt the segmentation style to their specific needs and can perform suboptimally for test images that are very different from the training images. Here we introduce Cellpose 2.0, a new package that includes an ensemble of diverse pretrained models as well as a human-in-the-loop pipeline for rapid prototyping of new custom models. We show that models pretrained on the Cellpose dataset can be fine-tuned with only 500–1,000 user-annotated regions of interest (ROI) to perform nearly as well as models trained on entire datasets with up to 200,000 ROI. A human-in-the-loop approach further reduced the required user annotation to 100–200 ROI, while maintaining high-quality segmentations. We provide software tools such as an annotation graphical user interface, a model zoo and a human-in-the-loop pipeline to facilitate the adoption of Cellpose 2.0. Cellpose 2.0 improves cell segmentation by offering pretrained models that can be fine-tuned using a human-in-the-loop training pipeline and fewer than 1,000 user-annotated regions of interest.},
  archive      = {J_NMETH},
  author       = {Pachitariu, Marius and Stringer, Carsen},
  doi          = {10.1038/s41592-022-01663-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1634-1641},
  shortjournal = {Nat. Meth.},
  title        = {Cellpose 2.0: How to train your own model},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Image-seq: Spatially resolved single-cell sequencing guided
by in situ and in vivo imaging. <em>NMETH</em>, <em>19</em>(12),
1622–1633. (<a
href="https://doi.org/10.1038/s41592-022-01673-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissue function depends on cellular organization. While the properties of individual cells are increasingly being deciphered using powerful single-cell sequencing technologies, understanding their spatial organization and temporal evolution remains a major challenge. Here, we present Image-seq, a technology that provides single-cell transcriptional data on cells that are isolated from specific spatial locations under image guidance, thus preserving the spatial information of the target cells. It is compatible with in situ and in vivo imaging and can document the temporal and dynamic history of the cells being analyzed. Cell samples are isolated from intact tissue and processed with state-of-the-art library preparation protocols. The technique therefore combines spatial information with highly sensitive RNA sequencing readouts from individual, intact cells. We have used both high-throughput, droplet-based sequencing as well as SMARTseq-v4 library preparation to demonstrate its application to bone marrow and leukemia biology. We discovered that DPP4 is a highly upregulated gene during early progression of acute myeloid leukemia and that it marks a more proliferative subpopulation that is confined to specific bone marrow microenvironments. Furthermore, the ability of Image-seq to isolate viable, intact cells should make it compatible with a range of downstream single-cell analysis tools including multi-omics protocols. Image-seq isolates cells from specific tissue locations under image guidance for analysis by single-cell RNA sequencing. The technique can be combined with in vivo imaging to document the temporal and dynamic history of the cells prior to sequencing.},
  archive      = {J_NMETH},
  author       = {Haase, Christa and Gustafsson, Karin and Mei, Shenglin and Yeh, Shu-Chi and Richter, Dmitry and Milosevic, Jelena and Turcotte, Raphaël and Kharchenko, Peter V. and Sykes, David B. and Scadden, David T. and Lin, Charles P.},
  doi          = {10.1038/s41592-022-01673-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1622-1633},
  shortjournal = {Nat. Meth.},
  title        = {Image-seq: Spatially resolved single-cell sequencing guided by in situ and in vivo imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chemically stable fluorescent proteins for advanced
microscopy. <em>NMETH</em>, <em>19</em>(12), 1612–1621. (<a
href="https://doi.org/10.1038/s41592-022-01660-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report the rational engineering of a remarkably stable yellow fluorescent protein (YFP), ‘hyperfolder YFP’ (hfYFP), that withstands chaotropic conditions that denature most biological structures within seconds, including superfolder green fluorescent protein (GFP). hfYFP contains no cysteines, is chloride insensitive and tolerates aldehyde and osmium tetroxide fixation better than common fluorescent proteins, enabling its use in expansion and electron microscopies. We solved crystal structures of hfYFP (to 1.7-Å resolution), a monomeric variant, monomeric hyperfolder YFP (1.6 Å) and an mGreenLantern mutant (1.2 Å), and then rationally engineered highly stable 405-nm-excitable GFPs, large Stokes shift (LSS) monomeric GFP (LSSmGFP) and LSSA12 from these structures. Lastly, we directly exploited the chemical stability of hfYFP and LSSmGFP by devising a fluorescence-assisted protein purification strategy enabling all steps of denaturing affinity chromatography to be visualized using ultraviolet or blue light. hfYFP and LSSmGFP represent a new generation of robustly stable fluorescent proteins developed for advanced biotechnological applications. The engineered hyperfolder YFP (hfYFP) and variants offer unprecedented chemical and thermal stability, making them versatile probes for microscopy as well as for challenging applications like correlative light and electron microscopy and expansion microscopy.},
  archive      = {J_NMETH},
  author       = {Campbell, Benjamin C. and Paez-Segala, Maria G. and Looger, Loren L. and Petsko, Gregory A. and Liu, Ce Feng},
  doi          = {10.1038/s41592-022-01660-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1612-1621},
  shortjournal = {Nat. Meth.},
  title        = {Chemically stable fluorescent proteins for advanced microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A framework for detecting noncoding rare-variant
associations of large-scale whole-genome sequencing studies.
<em>NMETH</em>, <em>19</em>(12), 1599–1611. (<a
href="https://doi.org/10.1038/s41592-022-01640-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale whole-genome sequencing studies have enabled analysis of noncoding rare-variant (RV) associations with complex human diseases and traits. Variant-set analysis is a powerful approach to study RV association. However, existing methods have limited ability in analyzing the noncoding genome. We propose a computationally efficient and robust noncoding RV association detection framework, STAARpipeline, to automatically annotate a whole-genome sequencing study and perform flexible noncoding RV association analysis, including gene-centric analysis and fixed window-based and dynamic window-based non-gene-centric analysis by incorporating variant functional annotations. In gene-centric analysis, STAARpipeline uses STAAR to group noncoding variants based on functional categories of genes and incorporate multiple functional annotations. In non-gene-centric analysis, STAARpipeline uses SCANG-STAAR to incorporate dynamic window sizes and multiple functional annotations. We apply STAARpipeline to identify noncoding RV sets associated with four lipid traits in 21,015 discovery samples from the Trans-Omics for Precision Medicine (TOPMed) program and replicate several of them in an additional 9,123 TOPMed samples. We also analyze five non-lipid TOPMed traits. STAARpipeline is a comprehensive framework for flexible and scalable rare-variant association analysis using whole-genome sequencing data and annotation information.},
  archive      = {J_NMETH},
  author       = {Li, Zilin and Li, Xihao and Zhou, Hufeng and Gaynor, Sheila M. and Selvaraj, Margaret Sunitha and Arapoglou, Theodore and Quick, Corbin and Liu, Yaowu and Chen, Han and Sun, Ryan and Dey, Rounak and Arnett, Donna K. and Auer, Paul L. and Bielak, Lawrence F. and Bis, Joshua C. and Blackwell, Thomas W. and Blangero, John and Boerwinkle, Eric and Bowden, Donald W. and Brody, Jennifer A. and Cade, Brian E. and Conomos, Matthew P. and Correa, Adolfo and Cupples, L. Adrienne and Curran, Joanne E. and de Vries, Paul S. and Duggirala, Ravindranath and Franceschini, Nora and Freedman, Barry I. and Göring, Harald H. H. and Guo, Xiuqing and Kalyani, Rita R. and Kooperberg, Charles and Kral, Brian G. and Lange, Leslie A. and Lin, Bridget M. and Manichaikul, Ani and Manning, Alisa K. and Martin, Lisa W. and Mathias, Rasika A. and Meigs, James B. and Mitchell, Braxton D. and Montasser, May E. and Morrison, Alanna C. and Naseri, Take and O’Connell, Jeffrey R. and Palmer, Nicholette D. and Peyser, Patricia A. and Psaty, Bruce M. and Raffield, Laura M. and Redline, Susan and Reiner, Alexander P. and Reupena, Muagututi’a Sefuiva and Rice, Kenneth M. and Rich, Stephen S. and Smith, Jennifer A. and Taylor, Kent D. and Taub, Margaret A. and Vasan, Ramachandran S. and Weeks, Daniel E. and Wilson, James G. and Yanek, Lisa R. and Zhao, Wei and Rotter, Jerome I. and Willer, Cristen J. and Natarajan, Pradeep and Peloso, Gina M. and Lin, Xihong},
  doi          = {10.1038/s41592-022-01640-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1599-1611},
  shortjournal = {Nat. Meth.},
  title        = {A framework for detecting noncoding rare-variant associations of large-scale whole-genome sequencing studies},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Detection of m6A from direct RNA sequencing using a multiple
instance learning framework. <em>NMETH</em>, <em>19</em>(12), 1590–1598.
(<a href="https://doi.org/10.1038/s41592-022-01666-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA modifications such as m6A methylation form an additional layer of complexity in the transcriptome. Nanopore direct RNA sequencing can capture this information in the raw current signal for each RNA molecule, enabling the detection of RNA modifications using supervised machine learning. However, experimental approaches provide only site-level training data, whereas the modification status for each single RNA molecule is missing. Here we present m6Anet, a neural-network-based method that leverages the multiple instance learning framework to specifically handle missing read-level modification labels in site-level training data. m6Anet outperforms existing computational methods, shows similar accuracy as experimental approaches, and generalizes with high accuracy to different cell lines and species without retraining model parameters. In addition, we demonstrate that m6Anet captures the underlying read-level stoichiometry, which can be used to approximate differences in modification rates. Overall, m6Anet offers a tool to capture the transcriptome-wide identification and quantification of m6A from a single run of direct RNA sequencing. This work presents m6Anet, which implements a neural-network-based multiple instance learning model to detect m6A modifications from direct RNA sequencing data.},
  archive      = {J_NMETH},
  author       = {Hendra, Christopher and Pratanwanich, Ploy N. and Wan, Yuk Kei and Goh, W. S. Sho and Thiery, Alexandre and Göke, Jonathan},
  doi          = {10.1038/s41592-022-01666-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1590-1598},
  shortjournal = {Nat. Meth.},
  title        = {Detection of m6A from direct RNA sequencing using a multiple instance learning framework},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Quantification of extracellular proteins, protein complexes
and mRNAs in single cells by proximity sequencing. <em>NMETH</em>,
<em>19</em>(12), 1578–1589. (<a
href="https://doi.org/10.1038/s41592-022-01684-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present proximity sequencing (Prox-seq) for simultaneous measurement of proteins, protein complexes and mRNAs in thousands of single cells. Prox-seq combines proximity ligation assay with single-cell sequencing to measure proteins and their complexes from all pairwise combinations of targeted proteins, providing quadratically scaled multiplexing. We validate Prox-seq and analyze a mixture of T cells and B cells to show that it accurately identifies these cell types and detects well-known protein complexes. Next, by studying human peripheral blood mononuclear cells, we discover that naïve CD8+ T cells display the protein complex CD8–CD9. Finally, we study protein interactions during Toll-like receptor (TLR) signaling in human macrophages. We observe the formation of signal-specific protein complexes, find CD36 co-receptor activity and additive signal integration under lipopolysaccharide (TLR4) and Pam2CSK4 (TLR2) stimulation, and show that quantification of protein complexes identifies signaling inputs received by macrophages. Prox-seq provides access to an untapped measurement modality for single-cell phenotyping and can discover uncharacterized protein interactions in different cell types. This work presents Prox-seq that couples sequencing and proximity ligation assay to simultaneously measure extracellular proteins, protein–protein interactions and mRNA in single cells.},
  archive      = {J_NMETH},
  author       = {Vistain, Luke and Van Phan, Hoang and Keisham, Bijentimala and Jordi, Christian and Chen, Mengjie and Reddy, Sai T. and Tay, Savaş},
  doi          = {10.1038/s41592-022-01684-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1578-1589},
  shortjournal = {Nat. Meth.},
  title        = {Quantification of extracellular proteins, protein complexes and mRNAs in single cells by proximity sequencing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A large-scale neural network training framework for
generalized estimation of single-trial population dynamics.
<em>NMETH</em>, <em>19</em>(12), 1572–1577. (<a
href="https://doi.org/10.1038/s41592-022-01675-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Achieving state-of-the-art performance with deep neural population dynamics models requires extensive hyperparameter tuning for each dataset. AutoLFADS is a model-tuning framework that automatically produces high-performing autoencoding models on data from a variety of brain areas and tasks, without behavioral or task information. We demonstrate its broad applicability on several rhesus macaque datasets: from motor cortex during free-paced reaching, somatosensory cortex during reaching with perturbations, and dorsomedial frontal cortex during a cognitive timing task. AutoLFADS models neural population activity via a deep learning-based approach with automated hyperparameter optimization.},
  archive      = {J_NMETH},
  author       = {Keshtkaran, Mohammad Reza and Sedler, Andrew R. and Chowdhury, Raeed H. and Tandon, Raghav and Basrai, Diya and Nguyen, Sarah L. and Sohn, Hansem and Jazayeri, Mehrdad and Miller, Lee E. and Pandarinath, Chethan},
  doi          = {10.1038/s41592-022-01675-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1572-1577},
  shortjournal = {Nat. Meth.},
  title        = {A large-scale neural network training framework for generalized estimation of single-trial population dynamics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). TemplateFlow: FAIR-sharing of multi-scale, multi-species
brain models. <em>NMETH</em>, <em>19</em>(12), 1568–1571. (<a
href="https://doi.org/10.1038/s41592-022-01681-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reference anatomies of the brain (‘templates’) and corresponding atlases are the foundation for reporting standardized neuroimaging results. Currently, there is no registry of templates and atlases; therefore, the redistribution of these resources occurs either bundled within existing software or in ad hoc ways such as downloads from institutional sites and general-purpose data repositories. We introduce TemplateFlow as a publicly available framework for human and non-human brain models. The framework combines an open database with software for access, management, and vetting, allowing scientists to share their resources under FAIR—findable, accessible, interoperable, and reusable—principles. TemplateFlow enables multifaceted insights into brains across species, and supports multiverse analyses testing whether results generalize across standard references, scales, and in the long term, species. TemplateFlow is a repository for human and other brain templates and atlases, which operates under the FAIR principles.},
  archive      = {J_NMETH},
  author       = {Ciric, Rastko and Thompson, William H. and Lorenz, Romy and Goncalves, Mathias and MacNicol, Eilidh E. and Markiewicz, Christopher J. and Halchenko, Yaroslav O. and Ghosh, Satrajit S. and Gorgolewski, Krzysztof J. and Poldrack, Russell A. and Esteban, Oscar},
  doi          = {10.1038/s41592-022-01681-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1568-1571},
  shortjournal = {Nat. Meth.},
  title        = {TemplateFlow: FAIR-sharing of multi-scale, multi-species brain models},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). RS-FISH: Precise, interactive, fast, and scalable FISH spot
detection. <em>NMETH</em>, <em>19</em>(12), 1563–1567. (<a
href="https://doi.org/10.1038/s41592-022-01669-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescent in-situ hybridization (FISH)-based methods extract spatially resolved genetic and epigenetic information from biological samples by detecting fluorescent spots in microscopy images, an often challenging task. We present Radial Symmetry-FISH (RS-FISH), an accurate, fast, and user-friendly software for spot detection in two- and three-dimensional images. RS-FISH offers interactive parameter tuning and readily scales to large datasets and image volumes of cleared or expanded samples using distributed processing on workstations, clusters, or the cloud. RS-FISH maintains high detection accuracy and low localization error across a wide range of signal-to-noise ratios, a key feature for single-molecule FISH, spatial transcriptomics, or spatial genomics applications. RS-FISH is a user-friendly software for accurate spot detection that is applicable to smFISH experiments, spatial transcriptomics, and spatial genomics. The approach enables fast spot detection in even very large volumetric datasets.},
  archive      = {J_NMETH},
  author       = {Bahry, Ella and Breimann, Laura and Zouinkhi, Marwan and Epstein, Leo and Kolyvanov, Klim and Mamrak, Nicholas and King, Benjamin and Long, Xi and Harrington, Kyle I. S. and Lionnet, Timothée and Preibisch, Stephan},
  doi          = {10.1038/s41592-022-01669-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1563-1567},
  shortjournal = {Nat. Meth.},
  title        = {RS-FISH: Precise, interactive, fast, and scalable FISH spot detection},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An improved imaging system that corrects MS2-induced RNA
destabilization. <em>NMETH</em>, <em>19</em>(12), 1558–1562. (<a
href="https://doi.org/10.1038/s41592-022-01658-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The MS2 and MS2-coat protein (MS2-MCP) imaging system is widely used to study messenger RNA (mRNA) spatial distribution in living cells. Here, we report that the MS2-MCP system destabilizes some tagged mRNAs by activating the nonsense-mediated mRNA decay pathway. We introduce an improved version, which counteracts this effect by increasing the efficiency of translation termination of the tagged mRNAs. Improved versions were developed for both yeast and mammalian systems. An improved version of the MS2-MCP system for imaging RNA dynamics involves tethering translation termination factors to tagged mRNAs to bypass destabilization caused by NMD machinery.},
  archive      = {J_NMETH},
  author       = {Li, Weihan and Maekiniemi, Anna and Sato, Hanae and Osman, Christof and Singer, Robert H.},
  doi          = {10.1038/s41592-022-01658-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1558-1562},
  shortjournal = {Nat. Meth.},
  title        = {An improved imaging system that corrects MS2-induced RNA destabilization},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High-dimensional gene expression and morphology profiles of
cells across 28,000 genetic and chemical perturbations. <em>NMETH</em>,
<em>19</em>(12), 1550–1557. (<a
href="https://doi.org/10.1038/s41592-022-01667-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells can be perturbed by various chemical and genetic treatments and the impact on gene expression and morphology can be measured via transcriptomic profiling and image-based assays, respectively. The patterns observed in these high-dimensional profile data can power a dozen applications in drug discovery and basic biology research, but both types of profiles are rarely available for large-scale experiments. Here, we provide a collection of four datasets with both gene expression and morphological profile data useful for developing and testing multimodal methodologies. Roughly a thousand features are measured for each of the two data types, across more than 28,000 chemical and genetic perturbations. We define biological problems that use the shared and complementary information in these two data modalities, provide baseline analysis and evaluation metrics for multi-omic applications, and make the data resource publicly available ( https://broad.io/rosetta/ ). This Resource presents and analyzes four datasets containing both gene expression and morphological profile data for cells subjected to hundreds to thousands of chemical or genetic perturbations and highlights their complementary nature.},
  archive      = {J_NMETH},
  author       = {Haghighi, Marzieh and Caicedo, Juan C. and Cimini, Beth A. and Carpenter, Anne E. and Singh, Shantanu},
  doi          = {10.1038/s41592-022-01667-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1550-1557},
  shortjournal = {Nat. Meth.},
  title        = {High-dimensional gene expression and morphology profiles of cells across 28,000 genetic and chemical perturbations},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Practical considerations for quantitative light sheet
fluorescence microscopy. <em>NMETH</em>, <em>19</em>(12), 1538–1549. (<a
href="https://doi.org/10.1038/s41592-022-01632-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence microscopy has evolved from a purely observational tool to a platform for quantitative, hypothesis-driven research. As such, the demand for faster and less phototoxic imaging modalities has spurred a rapid growth in light sheet fluorescence microscopy (LSFM). By restricting the excitation to a thin plane, LSFM reduces the overall light dose to a specimen while simultaneously improving image contrast. However, the defining characteristics of light sheet microscopes subsequently warrant unique considerations in their use for quantitative experiments. In this Perspective, we outline many of the pitfalls in LSFM that can compromise analysis and confound interpretation. Moreover, we offer guidance in addressing these caveats when possible. In doing so, we hope to provide a useful resource for life scientists seeking to adopt LSFM to quantitatively address complex biological hypotheses. This Perspective describes common pitfalls that can occur when using light sheet microscopy and offers guidance for improved quantitative imaging with these instruments.},
  archive      = {J_NMETH},
  author       = {Hobson, Chad M. and Guo, Min and Vishwasrao, Harshad D. and Wu, Yicong and Shroff, Hari and Chew, Teng-Leong},
  doi          = {10.1038/s41592-022-01632-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1538-1549},
  shortjournal = {Nat. Meth.},
  title        = {Practical considerations for quantitative light sheet fluorescence microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A cellular segmentation algorithm with fast customization.
<em>NMETH</em>, <em>19</em>(12), 1536–1537. (<a
href="https://doi.org/10.1038/s41592-022-01664-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common cellular segmentation models based on machine learning perform suboptimally for test images that differ greatly from training images. Cellpose 2.0 allows biologists to quickly train state-of-the-art segmentation models on their own imaging data. This was previously only possible using large, annotated datasets and required expert machine learning knowledge.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01664-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1536-1537},
  shortjournal = {Nat. Meth.},
  title        = {A cellular segmentation algorithm with fast customization},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Robust fluorescent proteins for high-resolution microscopy
and biochemical techniques. <em>NMETH</em>, <em>19</em>(12), 1534–1535.
(<a href="https://doi.org/10.1038/s41592-022-01661-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hyperfolder yellow fluorescent protein (hfYFP) and its variants are fluorescent proteins with high chemical and thermal stability. They resist aggregation, withstand diverse chemical challenges and show promise in expansion and electron microscopies. The chloride resistance and uncanny stability in guanidinium of hfYFP enable fluorescence-guided protein purification under denaturing conditions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01661-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1534-1535},
  shortjournal = {Nat. Meth.},
  title        = {Robust fluorescent proteins for high-resolution microscopy and biochemical techniques},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). STAARpipeline: An all-in-one rare-variant tool for
biobank-scale whole-genome sequencing data. <em>NMETH</em>,
<em>19</em>(12), 1532–1533. (<a
href="https://doi.org/10.1038/s41592-022-01641-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detecting rare-variant associations in the noncoding genome is challenging. We present a scalable, flexible and streamlined rare-variant association analysis framework for biobank-scale whole-genome sequencing data, including gene-centric and non-gene-centric analyses by incorporating multiple variant functional annotations using various coding and noncoding units, conditional analysis, result summary and visualization.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01641-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1532-1533},
  shortjournal = {Nat. Meth.},
  title        = {STAARpipeline: An all-in-one rare-variant tool for biobank-scale whole-genome sequencing data},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). M6Anet identifies n6-methyladenosine from individual direct
RNA sequencing reads. <em>NMETH</em>, <em>19</em>(12), 1530–1531. (<a
href="https://doi.org/10.1038/s41592-022-01668-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By modeling the probability of N6-methyladenosine (m6A) RNA modifications for individual reads from direct RNA sequencing, m6Anet achieves high classification accuracy and takes a step towards transcriptome-wide maps of m6A modifications at single-base, single-molecule resolution.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01668-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1530-1531},
  shortjournal = {Nat. Meth.},
  title        = {M6Anet identifies n6-methyladenosine from individual direct RNA sequencing reads},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Proximity sequencing for single-cell measurement of protein
complexes. <em>NMETH</em>, <em>19</em>(12), 1528–1529. (<a
href="https://doi.org/10.1038/s41592-022-01686-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to measure protein complexes in single cells is currently limited to a very small number of targets. Combining a proximity ligation assay with single-cell sequencing creates the ability to measure hundreds of extracellular protein complexes and thousands of mRNAs in individual cells.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01686-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1528-1529},
  shortjournal = {Nat. Meth.},
  title        = {Proximity sequencing for single-cell measurement of protein complexes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Harnessing the multiverse of neuroimaging standard
references. <em>NMETH</em>, <em>19</em>(12), 1526–1527. (<a
href="https://doi.org/10.1038/s41592-022-01682-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a FAIR (findable, accessible, interoperable, reusable) framework for researchers to share spatially standardized brain models. TemplateFlow enables the implementation of more reliable data processing pipelines by maximizing the accessibility of such models. It equips neuroimaging researchers with a foundational tool to bridge gaps between populations and species in neuroscience research.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01682-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1526-1527},
  shortjournal = {Nat. Meth.},
  title        = {Harnessing the multiverse of neuroimaging standard references},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Next-generation single virus tracking. <em>NMETH</em>,
<em>19</em>(12), 1524–1525. (<a
href="https://doi.org/10.1038/s41592-022-01670-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexing real-time single virus tracking with imaging paves the way for detailed information on virus–host interactions, offering a potential paradigm shift.},
  archive      = {J_NMETH},
  author       = {Padilla-Parra, Sergi},
  doi          = {10.1038/s41592-022-01670-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1524-1525},
  shortjournal = {Nat. Meth.},
  title        = {Next-generation single virus tracking},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Eavesdropping on extracellular vesicles. <em>NMETH</em>,
<em>19</em>(12), 1518–1523. (<a
href="https://doi.org/10.1038/s41592-022-01698-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A rapidly evolving toolbox is helping researchers to get a handle on the biological and functional diversity of these ubiquitous — but still somewhat enigmatic — cell-secreted nanoparticles},
  archive      = {J_NMETH},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41592-022-01698-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1518-1523},
  shortjournal = {Nat. Meth.},
  title        = {Eavesdropping on extracellular vesicles},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Single-molecule protein sequencing on a semiconductor
device. <em>NMETH</em>, <em>19</em>(12), 1517. (<a
href="https://doi.org/10.1038/s41592-022-01719-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01719-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1517},
  shortjournal = {Nat. Meth.},
  title        = {Single-molecule protein sequencing on a semiconductor device},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Neural networks built with biomolecules. <em>NMETH</em>,
<em>19</em>(12), 1517. (<a
href="https://doi.org/10.1038/s41592-022-01720-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01720-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1517},
  shortjournal = {Nat. Meth.},
  title        = {Neural networks built with biomolecules},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Spatially resolved epigenomics. <em>NMETH</em>,
<em>19</em>(12), 1516. (<a
href="https://doi.org/10.1038/s41592-022-01718-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An approach combining in situ tagmentation and transcription with MERFISH enables spatial profiling of the epigenome in tissues with single-cell resolution.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01718-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1516},
  shortjournal = {Nat. Meth.},
  title        = {Spatially resolved epigenomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022r). Researchers who reach far beyond their disabilities.
<em>NMETH</em>, <em>19</em>(12), 1512. (<a
href="https://doi.org/10.1038/s41592-022-01699-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists with visible and invisible disabilities take on adversity, helping themselves and others.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01699-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1512},
  shortjournal = {Nat. Meth.},
  title        = {Researchers who reach far beyond their disabilities},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Year in review 2022. <em>NMETH</em>, <em>19</em>(12), 1511.
(<a href="https://doi.org/10.1038/s41592-022-01721-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As 2022 draws to a close, we highlight some of our favorite papers that we published this year in Nature Methods.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01721-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1511},
  shortjournal = {Nat. Meth.},
  title        = {Year in review 2022},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Regression modeling of time-to-event data with censoring.
<em>NMETH</em>, <em>19</em>(12), 1513–1515. (<a
href="https://doi.org/10.1038/s41592-022-01689-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {If you sit on the sofa for your entire life, you’re running a higher risk of getting heart disease and cancer. —Alex Honnold, American rock climber},
  archive      = {J_NMETH},
  author       = {Dey, Tanujit and Lipsitz, Stuart R. and Cooper, Zara and Trinh, Quoc-Dien and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-022-01689-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {12},
  pages        = {1513-1515},
  shortjournal = {Nat. Meth.},
  title        = {Regression modeling of time-to-event data with censoring},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Estimation of skeletal kinematics in freely moving rodents.
<em>NMETH</em>, <em>19</em>(11), 1500–1509. (<a
href="https://doi.org/10.1038/s41592-022-01634-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Forming a complete picture of the relationship between neural activity and skeletal kinematics requires quantification of skeletal joint biomechanics during free behavior; however, without detailed knowledge of the underlying skeletal motion, inferring limb kinematics using surface-tracking approaches is difficult, especially for animals where the relationship between the surface and underlying skeleton changes during motion. Here we developed a videography-based method enabling detailed three-dimensional kinematic quantification of an anatomically defined skeleton in untethered freely behaving rats and mice. This skeleton-based model was constrained using anatomical principles and joint motion limits and provided skeletal pose estimates for a range of body sizes, even when limbs were occluded. Model-inferred limb positions and joint kinematics during gait and gap-crossing behaviors were verified by direct measurement of either limb placement or limb kinematics using inertial measurement units. Together we show that complex decision-making behaviors can be accurately reconstructed at the level of skeletal kinematics using our anatomically constrained model. Pose estimation in combination with an anatomically constrained model allows inferring skeletal kinematics in rodents.},
  archive      = {J_NMETH},
  author       = {Monsees, Arne and Voit, Kay-Michael and Wallace, Damian J. and Sawinski, Juergen and Charyasz, Edyta and Scheffler, Klaus and Macke, Jakob H. and Kerr, Jason N. D.},
  doi          = {10.1038/s41592-022-01634-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1500-1509},
  shortjournal = {Nat. Meth.},
  title        = {Estimation of skeletal kinematics in freely moving rodents},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CODA: Quantitative 3D reconstruction of large tissues at
cellular resolution. <em>NMETH</em>, <em>19</em>(11), 1490–1499. (<a
href="https://doi.org/10.1038/s41592-022-01650-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central challenge in biology is obtaining high-content, high-resolution information while analyzing tissue samples at volumes relevant to disease progression. We address this here with CODA, a method to reconstruct exceptionally large (up to multicentimeter cubed) tissues at subcellular resolution using serially sectioned hematoxylin and eosin-stained tissue sections. Here we demonstrate CODA’s ability to reconstruct three-dimensional (3D) distinct microanatomical structures in pancreas, skin, lung and liver tissues. CODA allows creation of readily quantifiable tissue volumes amenable to biological research. As a testbed, we assess the microanatomy of the human pancreas during tumorigenesis within the branching pancreatic ductal system, labeling ten distinct structures to examine heterogeneity and structural transformation during neoplastic progression. We show that pancreatic precancerous lesions develop into distinct 3D morphological phenotypes and that pancreatic cancer tends to spread far from the bulk tumor along collagen fibers that are highly aligned to the 3D curves of ductal, lobular, vascular and neural structures. Thus, CODA establishes a means to transform broadly the structural study of human diseases through exploration of exhaustively labeled 3D microarchitecture. CODA: a method for 3D reconstruction of large serially sectioned tissues.},
  archive      = {J_NMETH},
  author       = {Kiemen, Ashley L. and Braxton, Alicia M. and Grahn, Mia P. and Han, Kyu Sang and Babu, Jaanvi Mahesh and Reichel, Rebecca and Jiang, Ann C. and Kim, Bridgette and Hsu, Jocelyn and Amoa, Falone and Reddy, Sashank and Hong, Seung-Mo and Cornish, Toby C. and Thompson, Elizabeth D. and Huang, Peng and Wood, Laura D. and Hruban, Ralph H. and Wirtz, Denis and Wu, Pei-Hsun},
  doi          = {10.1038/s41592-022-01650-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1490-1499},
  shortjournal = {Nat. Meth.},
  title        = {CODA: Quantitative 3D reconstruction of large tissues at cellular resolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Netie: Inferring the evolution of neoantigen–t cell
interactions in tumors. <em>NMETH</em>, <em>19</em>(11), 1480–1489. (<a
href="https://doi.org/10.1038/s41592-022-01644-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neoantigens are the key targets of antitumor immune responses from cytotoxic T cells and play a critical role in affecting tumor progressions and immunotherapy treatment responses. However, little is known about how the interaction between neoantigens and T cells ultimately affects the evolution of cancerous masses. Here, we develop a hierarchical Bayesian model, named neoantigen–T cell interaction estimation (netie) to infer the history of neoantigen-CD8+ T cell interactions in tumors. Netie was systematically validated and applied to examine the molecular patterns of 3,219 tumors, compiled from a panel of 18 cancer types. We showed that tumors with an increase in immune selection pressure over time are associated with T cells that have an activation-related expression signature. We also identified a subset of exhausted cytotoxic T cells postimmunotherapy associated with tumor clones that newly arise after treatment. These analyses demonstrate how netie enables the interrogation of the relationship between individual neoantigen repertoires and the tumor molecular profiles. We found that a T cell inflammation gene expression profile (TIGEP) is more predictive of patient outcomes in the tumors with an increase in immune pressure over time, which reveals a curious synergy between T cells and neoantigen distributions. Overall, we provide a new tool that is capable of revealing the imprints left by neoantigens during each tumor’s developmental process and of predicting how tumors will progress under further pressure of the host’s immune system. Netie, a hierarchical Bayesian model to infer the neoantigen evolution and immune selection pressure during tumor progression.},
  archive      = {J_NMETH},
  author       = {Lu, Tianshi and Park, Seongoh and Han, Yi and Wang, Yunguan and Hubert, Shawna Marie and Futreal, P. Andy and Wistuba, Ignacio and Heymach, John V. and Reuben, Alexandre and Zhang, Jianjun and Wang, Tao},
  doi          = {10.1038/s41592-022-01644-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1480-1489},
  shortjournal = {Nat. Meth.},
  title        = {Netie: Inferring the evolution of neoantigen–T cell interactions in tumors},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Neuromaps: Structural and functional interpretation of brain
maps. <em>NMETH</em>, <em>19</em>(11), 1472–1479. (<a
href="https://doi.org/10.1038/s41592-022-01625-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging technologies are increasingly used to generate high-resolution reference maps of brain structure and function. Comparing experimentally generated maps to these reference maps facilitates cross-disciplinary scientific discovery. Although recent data sharing initiatives increase the accessibility of brain maps, data are often shared in disparate coordinate systems, precluding systematic and accurate comparisons. Here we introduce neuromaps, a toolbox for accessing, transforming and analyzing structural and functional brain annotations. We implement functionalities for generating high-quality transformations between four standard coordinate systems. The toolbox includes curated reference maps and biological ontologies of the human brain, such as molecular, microstructural, electrophysiological, developmental and functional ontologies. Robust quantitative assessment of map-to-map similarity is enabled via a suite of spatial autocorrelation-preserving null models. neuromaps combines open-access data with transparent functionality for standardizing and comparing brain maps, providing a systematic workflow for comprehensive structural and functional annotation enrichment analysis of the human brain. neuromaps is a toolbox for accessing, transforming and comparing human neuroimaging data.},
  archive      = {J_NMETH},
  author       = {Markello, Ross D. and Hansen, Justine Y. and Liu, Zhen-Qi and Bazinet, Vincent and Shafiei, Golia and Suárez, Laura E. and Blostein, Nadia and Seidlitz, Jakob and Baillet, Sylvain and Satterthwaite, Theodore D. and Chakravarty, M. Mallar and Raznahan, Armin and Misic, Bratislav},
  doi          = {10.1038/s41592-022-01625-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1472-1479},
  shortjournal = {Nat. Meth.},
  title        = {Neuromaps: Structural and functional interpretation of brain maps},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sensitive genetically encoded sensors for population and
subcellular imaging of cAMP in vivo. <em>NMETH</em>, <em>19</em>(11),
1461–1471. (<a
href="https://doi.org/10.1038/s41592-022-01646-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cyclic adenosine monophosphate (cAMP) signaling integrates information from diverse G-protein-coupled receptors, such as neuromodulator receptors, to regulate pivotal biological processes in a cellular-specific and subcellular-specific manner. However, in vivo cellular-resolution imaging of cAMP dynamics remains challenging. Here, we screen existing genetically encoded cAMP sensors and further develop the best performer to derive three improved variants, called cAMPFIREs. Compared with their parental sensor, these sensors exhibit up to 10-fold increased sensitivity to cAMP and a cytosolic distribution. cAMPFIREs are compatible with both ratiometric and fluorescence lifetime imaging and can detect cAMP dynamics elicited by norepinephrine at physiologically relevant, nanomolar concentrations. Imaging of cAMPFIREs in awake mice reveals tonic levels of cAMP in cortical neurons that are associated with wakefulness, modulated by opioids, and differentially regulated across subcellular compartments. Furthermore, enforced locomotion elicits neuron-specific, bidirectional cAMP dynamics. cAMPFIREs also function in Drosophila. Overall, cAMPFIREs may have broad applicability for studying intracellular signaling in vivo. cAMPFIREs are genetically encoded cAMP sensors that are suitable for in vivo imaging of cAMP signaling, as demonstrated in Drosophila larvae and behaving mice.},
  archive      = {J_NMETH},
  author       = {Massengill, Crystian I. and Bayless-Edwards, Landon and Ceballos, Cesar C. and Cebul, Elizabeth R. and Cahill, James and Bharadwaj, Arpita and Wilson, Evan and Qin, Maozhen and Whorton, Matthew R. and Baconguis, Isabelle and Ye, Bing and Mao, Tianyi and Zhong, Haining},
  doi          = {10.1038/s41592-022-01646-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1461-1471},
  shortjournal = {Nat. Meth.},
  title        = {Sensitive genetically encoded sensors for population and subcellular imaging of cAMP in vivo},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Geometric engineering of organoid culture for enhanced
organogenesis in a dish. <em>NMETH</em>, <em>19</em>(11), 1449–1460. (<a
href="https://doi.org/10.1038/s41592-022-01643-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here, we introduce a facile, scalable engineering approach to enable long-term development and maturation of organoids. We have redesigned the configuration of conventional organoid culture to develop a platform that converts single injections of stem cell suspensions to radial arrays of organoids that can be maintained for extended periods without the need for passaging. Using this system, we demonstrate accelerated production of intestinal organoids with significantly enhanced structural and functional maturity, and their continuous development for over 4 weeks. Furthermore, we present a patient-derived organoid model of inflammatory bowel disease (IBD) and its interrogation using single-cell RNA sequencing to demonstrate its ability to reproduce key pathological features of IBD. Finally, we describe the extension of our approach to engineer vascularized, perfusable human enteroids, which can be used to model innate immune responses in IBD. This work provides an immediately deployable platform technology toward engineering more realistic organ-like structures in a dish. A scalable platform with geometrical reconfiguration of culture systems for long-term growth and maturation of organoids.},
  archive      = {J_NMETH},
  author       = {Park, Sunghee Estelle and Kang, Shawn and Paek, Jungwook and Georgescu, Andrei and Chang, Jeehan and Yi, Alex Yoon and Wilkins, Benjamin J. and Karakasheva, Tatiana A. and Hamilton, Kathryn E. and Huh, Dan Dongeun},
  doi          = {10.1038/s41592-022-01643-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1449-1460},
  shortjournal = {Nat. Meth.},
  title        = {Geometric engineering of organoid culture for enhanced organogenesis in a dish},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Omnipose: A high-precision morphology-independent solution
for bacterial cell segmentation. <em>NMETH</em>, <em>19</em>(11),
1438–1448. (<a
href="https://doi.org/10.1038/s41592-022-01639-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in microscopy hold great promise for allowing quantitative and precise measurement of morphological and molecular phenomena at the single-cell level in bacteria; however, the potential of this approach is ultimately limited by the availability of methods to faithfully segment cells independent of their morphological or optical characteristics. Here, we present Omnipose, a deep neural network image-segmentation algorithm. Unique network outputs such as the gradient of the distance field allow Omnipose to accurately segment cells on which current algorithms, including its predecessor, Cellpose, produce errors. We show that Omnipose achieves unprecedented segmentation performance on mixed bacterial cultures, antibiotic-treated cells and cells of elongated or branched morphology. Furthermore, the benefits of Omnipose extend to non-bacterial subjects, varied imaging modalities and three-dimensional objects. Finally, we demonstrate the utility of Omnipose in the characterization of extreme morphological phenotypes that arise during interbacterial antagonism. Our results distinguish Omnipose as a powerful tool for characterizing diverse and arbitrarily shaped cell types from imaging data. Omnipose is a deep neural network algorithm for image segmentation that improves upon existing approaches by solving the challenging problem of accurately segmenting morphologically diverse cells from images acquired with any modality.},
  archive      = {J_NMETH},
  author       = {Cutler, Kevin J. and Stringer, Carsen and Lo, Teresa W. and Rappez, Luca and Stroustrup, Nicholas and Brook Peterson, S. and Wiggins, Paul A. and Mougous, Joseph D.},
  doi          = {10.1038/s41592-022-01639-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1438-1448},
  shortjournal = {Nat. Meth.},
  title        = {Omnipose: A high-precision morphology-independent solution for bacterial cell segmentation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Incorporating the image formation process into deep learning
improves network performance. <em>NMETH</em>, <em>19</em>(11),
1427–1437. (<a
href="https://doi.org/10.1038/s41592-022-01652-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Richardson–Lucy network (RLN), a fast and lightweight deep learning method for three-dimensional fluorescence microscopy deconvolution. RLN combines the traditional Richardson–Lucy iteration with a fully convolutional network structure, establishing a connection to the image formation process and thereby improving network performance. Containing only roughly 16,000 parameters, RLN enables four- to 50-fold faster processing than purely data-driven networks with many more parameters. By visual and quantitative analysis, we show that RLN provides better deconvolution, better generalizability and fewer artifacts than other networks, especially along the axial dimension. RLN outperforms classic Richardson–Lucy deconvolution on volumes contaminated with severe out of focus fluorescence or noise and provides four- to sixfold faster reconstructions of large, cleared-tissue datasets than classic multi-view pipelines. We demonstrate RLN’s performance on cells, tissues and embryos imaged with widefield-, light-sheet-, confocal- and super-resolution microscopy. Richardson–Lucy Network (RLN) combines the traditional Richardson–Lucy iteration with deep learning for improved deconvolution. RLN is more generalizable, offers fewer artifacts and requires less computing time than alternative approaches.},
  archive      = {J_NMETH},
  author       = {Li, Yue and Su, Yijun and Guo, Min and Han, Xiaofei and Liu, Jiamin and Vishwasrao, Harshad D. and Li, Xuesong and Christensen, Ryan and Sengupta, Titas and Moyle, Mark W. and Rey-Suarez, Ivan and Chen, Jiji and Upadhyaya, Arpita and Usdin, Ted B. and Colón-Ramos, Daniel Alfonso and Liu, Huafeng and Wu, Yicong and Shroff, Hari},
  doi          = {10.1038/s41592-022-01652-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1427-1437},
  shortjournal = {Nat. Meth.},
  title        = {Incorporating the image formation process into deep learning improves network performance},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Resolution doubling in light-sheet microscopy via oblique
plane structured illumination. <em>NMETH</em>, <em>19</em>(11),
1419–1426. (<a
href="https://doi.org/10.1038/s41592-022-01635-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structured illumination microscopy (SIM) doubles the spatial resolution of a fluorescence microscope without requiring high laser powers or specialized fluorophores. However, the excitation of out-of-focus fluorescence can accelerate photobleaching and phototoxicity. In contrast, light-sheet fluorescence microscopy (LSFM) largely avoids exciting out-of-focus fluorescence, thereby enabling volumetric imaging with low photobleaching and intrinsic optical sectioning. Combining SIM with LSFM would enable gentle three-dimensional (3D) imaging at doubled resolution. However, multiple orientations of the illumination pattern, which are needed for isotropic resolution doubling in SIM, are challenging to implement in a light-sheet format. Here we show that multidirectional structured illumination can be implemented in oblique plane microscopy, an LSFM technique that uses a single objective for excitation and detection, in a straightforward manner. We demonstrate isotropic lateral resolution below 150 nm, combined with lower phototoxicity compared to traditional SIM systems and volumetric acquisition speed exceeding 1 Hz. The longstanding goal of combining the optical sectioning of light-sheet illumination and the resolving power of multidirectional structured illumination microscopy is realized using an oblique plane microscope for improved fast 3D subcellular imaging.},
  archive      = {J_NMETH},
  author       = {Chen, Bingying and Chang, Bo-Jui and Roudot, Philippe and Zhou, Felix and Sapoznik, Etai and Marlar-Pavey, Madeleine and Hayes, James B. and Brown, Peter T. and Zeng, Chih-Wei and Lambert, Talley and Friedman, Jonathan R. and Zhang, Chun-Li and Burnette, Dylan T. and Shepherd, Douglas P. and Dean, Kevin M. and Fiolka, Reto P.},
  doi          = {10.1038/s41592-022-01635-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1419-1426},
  shortjournal = {Nat. Meth.},
  title        = {Resolution doubling in light-sheet microscopy via oblique plane structured illumination},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Annotation of spatially resolved single-cell data with
STELLAR. <em>NMETH</em>, <em>19</em>(11), 1411–1418. (<a
href="https://doi.org/10.1038/s41592-022-01651-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate cell-type annotation from spatially resolved single cells is crucial to understand functional spatial biology that is the basis of tissue organization. However, current computational methods for annotating spatially resolved single-cell data are typically based on techniques established for dissociated single-cell technologies and thus do not take spatial organization into account. Here we present STELLAR, a geometric deep learning method for cell-type discovery and identification in spatially resolved single-cell datasets. STELLAR automatically assigns cells to cell types present in the annotated reference dataset and discovers novel cell types and cell states. STELLAR transfers annotations across different dissection regions, different tissues and different donors, and learns cell representations that capture higher-order tissue structures. We successfully applied STELLAR to CODEX multiplexed fluorescent microscopy data and multiplexed RNA imaging datasets. Within the Human BioMolecular Atlas Program, STELLAR has annotated 2.6 million spatially resolved single cells with dramatic time savings. STELLAR (spatial cell learning) is a geometric deep learning model that works with spatially resolved single-cell datasets to both assign cell types in unannotated datasets based on a reference dataset and discover new cell types.},
  archive      = {J_NMETH},
  author       = {Brbić, Maria and Cao, Kaidi and Hickey, John W. and Tan, Yuqi and Snyder, Michael P. and Nolan, Garry P. and Leskovec, Jure},
  doi          = {10.1038/s41592-022-01651-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1411-1418},
  shortjournal = {Nat. Meth.},
  title        = {Annotation of spatially resolved single-cell data with STELLAR},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ClampFISH 2.0 enables rapid, scalable amplified RNA
detection in situ. <em>NMETH</em>, <em>19</em>(11), 1403–1410. (<a
href="https://doi.org/10.1038/s41592-022-01653-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA labeling in situ has enormous potential to visualize transcripts and quantify their levels in single cells, but it remains challenging to produce high levels of signal while also enabling multiplexed detection of multiple RNA species simultaneously. Here, we describe clampFISH 2.0, a method that uses an inverted padlock design to efficiently detect many RNA species and exponentially amplify their signals at once, while also reducing the time and cost compared with the prior clampFISH method. We leverage the increased throughput afforded by multiplexed signal amplification and sequential detection to detect 10 different RNA species in more than 1 million cells. We also show that clampFISH 2.0 works in tissue sections. We expect that the advantages offered by clampFISH 2.0 will enable many applications in spatial transcriptomics. ClampFISH 2.0 enables highly specific multiplexed signal amplification in RNA FISH. The approach was used to detect 10 RNA species that ranged in abundance in more than 1 million cells and is also applicable to tissue sections.},
  archive      = {J_NMETH},
  author       = {Dardani, Ian and Emert, Benjamin L. and Goyal, Yogesh and Jiang, Connie L. and Kaur, Amanpreet and Lee, Jasmine and Rouhanifard, Sara H. and Alicea, Gretchen M. and Fane, Mitchell E. and Xiao, Min and Herlyn, Meenhard and Weeraratna, Ashani T. and Raj, Arjun},
  doi          = {10.1038/s41592-022-01653-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1403-1410},
  shortjournal = {Nat. Meth.},
  title        = {ClampFISH 2.0 enables rapid, scalable amplified RNA detection in situ},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Light-seq: Light-directed in situ barcoding of biomolecules
in fixed cells and tissues for spatially indexed sequencing.
<em>NMETH</em>, <em>19</em>(11), 1393–1402. (<a
href="https://doi.org/10.1038/s41592-022-01604-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Light-Seq, an approach for multiplexed spatial indexing of intact biological samples using light-directed DNA barcoding in fixed cells and tissues followed by ex situ sequencing. Light-Seq combines spatially targeted, rapid photocrosslinking of DNA barcodes onto complementary DNAs in situ with a one-step DNA stitching reaction to create pooled, spatially indexed sequencing libraries. This light-directed barcoding enables in situ selection of multiple cell populations in intact fixed tissue samples for full-transcriptome sequencing based on location, morphology or protein stains, without cellular dissociation. Applying Light-Seq to mouse retinal sections, we recovered thousands of differentially enriched transcripts from three cellular layers and discovered biomarkers for a very rare neuronal subtype, dopaminergic amacrine cells, from only four to eight individual cells per section. Light-Seq provides an accessible workflow to combine in situ imaging and protein staining with next generation sequencing of the same cells, leaving the sample intact for further analysis post-sequencing. Light-Seq uses light-directed DNA barcoding in fixed cells and tissues for multiplexed spatial indexing and subsequent next generation sequencing. This approach blends spatial and omics information to enable analysis of rare cell types in complex tissues.},
  archive      = {J_NMETH},
  author       = {Kishi, Jocelyn Y. and Liu, Ninning and West, Emma R. and Sheng, Kuanwei and Jordanides, Jack J. and Serrata, Matthew and Cepko, Constance L. and Saka, Sinem K. and Yin, Peng},
  doi          = {10.1038/s41592-022-01604-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1393-1402},
  shortjournal = {Nat. Meth.},
  title        = {Light-seq: Light-directed in situ barcoding of biomolecules in fixed cells and tissues for spatially indexed sequencing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Profiling RNA at chromatin targets in situ by
antibody-targeted tagmentation. <em>NMETH</em>, <em>19</em>(11),
1383–1392. (<a
href="https://doi.org/10.1038/s41592-022-01618-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whereas techniques to map chromatin-bound proteins are well developed, mapping chromatin-associated RNAs remains a challenge. Here, we describe Reverse Transcribe and Tagment (RT&amp;amp;Tag), in which RNAs associated with a chromatin epitope are targeted by an antibody followed by a protein A-Tn5 transposome. Localized reverse transcription generates RNA/cDNA hybrids that are subsequently tagmented by Tn5 transposases for downstream sequencing. We demonstrate the utility of RT&amp;amp;Tag in Drosophila cells for capturing the noncoding RNA roX2 with the dosage compensation complex and maturing transcripts associated with silencing histone modifications. We also show that RT&amp;amp;Tag can detect N6-methyladenosine-modified mRNAs, and show that genes producing methylated transcripts are characterized by extensive promoter pausing of RNA polymerase II. The high efficiency of in situ antibody tethering and tagmentation makes RT&amp;amp;Tag especially suitable for rapid low-cost profiling of chromatin-associated RNAs. This work presents RT&amp;amp;Tag, which captures RNA molecules in proximity to a protein of interest in intact nuclei.},
  archive      = {J_NMETH},
  author       = {Khyzha, Nadiya and Henikoff, Steven and Ahmad, Kami},
  doi          = {10.1038/s41592-022-01618-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1383-1392},
  shortjournal = {Nat. Meth.},
  title        = {Profiling RNA at chromatin targets in situ by antibody-targeted tagmentation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improved AlphaFold modeling with implicit experimental
information. <em>NMETH</em>, <em>19</em>(11), 1376–1382. (<a
href="https://doi.org/10.1038/s41592-022-01645-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine-learning prediction algorithms such as AlphaFold and RoseTTAFold can create remarkably accurate protein models, but these models usually have some regions that are predicted with low confidence or poor accuracy. We hypothesized that by implicitly including new experimental information such as a density map, a greater portion of a model could be predicted accurately, and that this might synergistically improve parts of the model that were not fully addressed by either machine learning or experiment alone. An iterative procedure was developed in which AlphaFold models are automatically rebuilt on the basis of experimental density maps and the rebuilt models are used as templates in new AlphaFold predictions. We show that including experimental information improves prediction beyond the improvement obtained with simple rebuilding guided by the experimental data. This procedure for AlphaFold modeling with density has been incorporated into an automated procedure for interpretation of crystallographic and electron cryo-microscopy maps. This paper presents an iterative procedure where AlphaFold models are automatically rebuilt on the basis of experimental density maps and the rebuilt models are used as templates in new AlphaFold predictions.},
  archive      = {J_NMETH},
  author       = {Terwilliger, Thomas C. and Poon, Billy K. and Afonine, Pavel V. and Schlicksup, Christopher J. and Croll, Tristan I. and Millán, Claudia and Richardson, Jane. S. and Read, Randy J. and Adams, Paul D.},
  doi          = {10.1038/s41592-022-01645-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1376-1382},
  shortjournal = {Nat. Meth.},
  title        = {Improved AlphaFold modeling with implicit experimental information},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A multi-purpose, regenerable, proteome-scale, human
phosphoserine resource for phosphoproteomics. <em>NMETH</em>,
<em>19</em>(11), 1371–1375. (<a
href="https://doi.org/10.1038/s41592-022-01638-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass-spectrometry-based phosphoproteomics has become indispensable for understanding cellular signaling in complex biological systems. Despite the central role of protein phosphorylation, the field still lacks inexpensive, regenerable, and diverse phosphopeptides with ground-truth phosphorylation positions. Here, we present Iterative Synthetically Phosphorylated Isomers (iSPI), a proteome-scale library of human-derived phosphoserine-containing phosphopeptides that is inexpensive, regenerable, and diverse, with precisely known positions of phosphorylation. We demonstrate possible uses of iSPI, including use as a phosphopeptide standard, a tool to evaluate and optimize phosphorylation-site localization algorithms, and a benchmark to compare performance across data analysis pipelines. We also present AScorePro, an updated version of the AScore algorithm specifically optimized for phosphorylation-site localization in higher energy fragmentation spectra, and the FLR viewer, a web tool for phosphorylation-site localization, to enable community use of the iSPI resource. iSPI and its associated data constitute a useful, multi-purpose resource for the phosphoproteomics community. Iterative Synthetically Phosphorylated Isomers (iSPI) is a proteome-scale library of human-derived phosphoserine-containing phosphopeptides with precisely known positions of phosphorylation. This multi-purpose resource is available for optimization, standardization, and benchmarking of key steps in phosphoproteomics workflows.},
  archive      = {J_NMETH},
  author       = {Gassaway, Brandon M. and Li, Jiaming and Rad, Ramin and Mintseris, Julian and Mohler, Kyle and Levy, Tyler and Aguiar, Mike and Beausoleil, Sean A. and Paulo, Joao A. and Rinehart, Jesse and Huttlin, Edward L. and Gygi, Steven P.},
  doi          = {10.1038/s41592-022-01638-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1371-1375},
  shortjournal = {Nat. Meth.},
  title        = {A multi-purpose, regenerable, proteome-scale, human phosphoserine resource for phosphoproteomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). SyConn2: Dense synaptic connectivity inference for volume
electron microscopy. <em>NMETH</em>, <em>19</em>(11), 1367–1370. (<a
href="https://doi.org/10.1038/s41592-022-01624-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to acquire ever larger datasets of brain tissue using volume electron microscopy leads to an increasing demand for the automated extraction of connectomic information. We introduce SyConn2, an open-source connectome analysis toolkit, which works with both on-site high-performance compute environments and rentable cloud computing clusters. SyConn2 was tested on connectomic datasets with more than 10 million synapses, provides a web-based visualization interface and makes these data amenable to complex anatomical and neuronal connectivity queries. SyConn2 is a machine learning-based framework for inferring and analyzing the connectomes contained in a volume electron microscopy dataset of brain tissue, for example from the zebra finch.},
  archive      = {J_NMETH},
  author       = {Schubert, Philipp J. and Dorkenwald, Sven and Januszewski, Michał and Klimesch, Jonathan and Svara, Fabian and Mancu, Andrei and Ahmad, Hashir and Fee, Michale S. and Jain, Viren and Kornfeld, Joergen},
  doi          = {10.1038/s41592-022-01624-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1367-1370},
  shortjournal = {Nat. Meth.},
  title        = {SyConn2: Dense synaptic connectivity inference for volume electron microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automated synapse-level reconstruction of neural circuits in
the larval zebrafish brain. <em>NMETH</em>, <em>19</em>(11), 1357–1366.
(<a href="https://doi.org/10.1038/s41592-022-01621-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dense reconstruction of synaptic connectivity requires high-resolution electron microscopy images of entire brains and tools to efficiently trace neuronal wires across the volume. To generate such a resource, we sectioned and imaged a larval zebrafish brain by serial block-face electron microscopy at a voxel size of 14 × 14 × 25 nm3. We segmented the resulting dataset with the flood-filling network algorithm, automated the detection of chemical synapses and validated the results by comparisons to transmission electron microscopic images and light-microscopic reconstructions. Neurons and their connections are stored in the form of a queryable and expandable digital address book. We reconstructed a network of 208 neurons involved in visual motion processing, most of them located in the pretectum, which had been functionally characterized in the same specimen by two-photon calcium imaging. Moreover, we mapped all 407 presynaptic and postsynaptic partners of two superficial interneurons in the tectum. The resource developed here serves as a foundation for synaptic-resolution circuit analyses in the zebrafish nervous system. This Resource presents a serial block-face EM dataset of the whole larval zebrafish brain, including automated segmentation of neurons, detection of synapses and reconstruction of circuitry for visual motion processing.},
  archive      = {J_NMETH},
  author       = {Svara, Fabian and Förster, Dominique and Kubo, Fumi and Januszewski, Michał and dal Maschio, Marco and Schubert, Philipp J. and Kornfeld, Jörgen and Wanner, Adrian A. and Laurell, Eva and Denk, Winfried and Baier, Herwig},
  doi          = {10.1038/s41592-022-01621-0},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1357-1366},
  shortjournal = {Nat. Meth.},
  title        = {Automated synapse-level reconstruction of neural circuits in the larval zebrafish brain},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Doubling the resolution of light-sheet fluorescence
microscopy. <em>NMETH</em>, <em>19</em>(11), 1355–1356. (<a
href="https://doi.org/10.1038/s41592-022-01636-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A combination of light-sheet fluorescence microscopy (LSFM) with structured illumination doubles resolving power over LSFM alone. We show a practical implementation using a single objective for illumination and fluorescence detection and demonstrate its use for rapid volumetric imaging.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01636-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1355-1356},
  shortjournal = {Nat. Meth.},
  title        = {Doubling the resolution of light-sheet fluorescence microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Light-seq: From microscopy to transcriptomics and back.
<em>NMETH</em>, <em>19</em>(11), 1353–1354. (<a
href="https://doi.org/10.1038/s41592-022-01608-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light-Seq combines high resolution imaging with next generation sequencing of selected cell populations in fixed biological samples. Specifically, microscopically analyzed cells can be subjected to RNA expression profiling while keeping the sample intact for further assays, enabling cellular phenotypes and states to be assessed in the context of the original tissue.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01608-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1353-1354},
  shortjournal = {Nat. Meth.},
  title        = {Light-seq: From microscopy to transcriptomics and back},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An in situ method for tagging chromatin-associated RNAs.
<em>NMETH</em>, <em>19</em>(11), 1351–1352. (<a
href="https://doi.org/10.1038/s41592-022-01619-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA comprises a substantial fraction of eukaryotic chromatin, but techniques to identify and map RNAs are cumbersome. We adapted existing tagmentation-based profiling techniques to enable chromatin-associated RNAs to be profiled in a simple workflow, enhancing the capability to identify regulatory RNAs.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01619-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1351-1352},
  shortjournal = {Nat. Meth.},
  title        = {An in situ method for tagging chromatin-associated RNAs},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The data science future of neuroscience theory.
<em>NMETH</em>, <em>19</em>(11), 1349–1350. (<a
href="https://doi.org/10.1038/s41592-022-01630-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An approach for integrating the wealth of heterogeneous brain data — from gene expression and neurotransmitter receptor density to structure and function — allows neuroscientists to easily place their data within the broader neuroscientific context.},
  archive      = {J_NMETH},
  author       = {Voytek, Bradley},
  doi          = {10.1038/s41592-022-01630-z},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1349-1350},
  shortjournal = {Nat. Meth.},
  title        = {The data science future of neuroscience theory},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Redesigning hydrogel geometry for enhanced organoids.
<em>NMETH</em>, <em>19</em>(11), 1347–1348. (<a
href="https://doi.org/10.1038/s41592-022-01656-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {OCTOPUS is a bioengineered device for high-fidelity organoid cultures long term.},
  archive      = {J_NMETH},
  author       = {Hillion, Killian and Mahe, Maxime M.},
  doi          = {10.1038/s41592-022-01656-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1347-1348},
  shortjournal = {Nat. Meth.},
  title        = {Redesigning hydrogel geometry for enhanced organoids},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mapping of the zebrafish brain takes shape. <em>NMETH</em>,
<em>19</em>(11), 1345–1346. (<a
href="https://doi.org/10.1038/s41592-022-01637-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The generation of a whole larval zebrafish brain electron microscopy volume in tandem with automated tools lays the groundwork for producing the first vertebrate brain connectome.},
  archive      = {J_NMETH},
  author       = {Brooks, Paul and Champion, Andrew and Costa, Marta},
  doi          = {10.1038/s41592-022-01637-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1345-1346},
  shortjournal = {Nat. Meth.},
  title        = {Mapping of the zebrafish brain takes shape},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022p). Patch-seq takes neuroscience to a multimodal place.
<em>NMETH</em>, <em>19</em>(11), 1340–1344. (<a
href="https://doi.org/10.1038/s41592-022-01662-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patch-seq delivers morphology, electrophysiology and transcriptomic data to those with skill, patience and persistence.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01662-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1340-1344},
  shortjournal = {Nat. Meth.},
  title        = {Patch-seq takes neuroscience to a multimodal place},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A diamond microscope. <em>NMETH</em>, <em>19</em>(11),
1339. (<a href="https://doi.org/10.1038/s41592-022-01679-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01679-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1339},
  shortjournal = {Nat. Meth.},
  title        = {A diamond microscope},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Choosing the best conditional degron tags. <em>NMETH</em>,
<em>19</em>(11), 1339. (<a
href="https://doi.org/10.1038/s41592-022-01680-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01680-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1339},
  shortjournal = {Nat. Meth.},
  title        = {Choosing the best conditional degron tags},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Imaging-based spatial transcriptomics goes electric.
<em>NMETH</em>, <em>19</em>(11), 1338. (<a
href="https://doi.org/10.1038/s41592-022-01678-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers use electric fields to transfer RNA from a tissue sample onto a surface for subsequent fluorescence in situ hybridization-based profiling of transcriptomes at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01678-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1338},
  shortjournal = {Nat. Meth.},
  title        = {Imaging-based spatial transcriptomics goes electric},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The EQIPD framework for rigor in the design, conduct,
analysis and documentation of animal experiments. <em>NMETH</em>,
<em>19</em>(11), 1334–1337. (<a
href="https://doi.org/10.1038/s41592-022-01615-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The EQIPD framework for rigor in animal experiments aims to unify current recommendations based on evidence behind their rationale and was prospectively tested for feasibility in multicenter animal experiments.},
  archive      = {J_NMETH},
  author       = {Vollert, Jan and Macleod, Malcolm and Dirnagl, Ulrich and Kas, Martien J. and Michel, Martin C. and Potschka, Heidrun and Riedel, Gernot and Wever, Kimberley E. and Würbel, Hanno and Steckler, Thomas and Rice, Andrew S. C.},
  doi          = {10.1038/s41592-022-01615-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1334-1337},
  shortjournal = {Nat. Meth.},
  title        = {The EQIPD framework for rigor in the design, conduct, analysis and documentation of animal experiments},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ShareLoc — an open platform for sharing localization
microscopy data. <em>NMETH</em>, <em>19</em>(11), 1331–1333. (<a
href="https://doi.org/10.1038/s41592-022-01659-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Ouyang, Wei and Bai, Jiachuan and Singh, Manish Kumar and Leterrier, Christophe and Barthelemy, Paul and Barnett, Samuel F. H. and Klein, Teresa and Sauer, Markus and Kanchanawong, Pakorn and Bourg, Nicolas and Cohen, Mickael M. and Lelandais, Benoît and Zimmer, Christophe},
  doi          = {10.1038/s41592-022-01659-0},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1331-1333},
  shortjournal = {Nat. Meth.},
  title        = {ShareLoc — an open platform for sharing localization microscopy data},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BioImageIT: Open-source framework for integration of image
data management with analysis. <em>NMETH</em>, <em>19</em>(11),
1328–1330. (<a
href="https://doi.org/10.1038/s41592-022-01642-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Prigent, Sylvain and Valades-Cruz, Cesar Augusto and Leconte, Ludovic and Maury, Léo and Salamero, Jean and Kervrann, Charles},
  doi          = {10.1038/s41592-022-01642-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1328-1330},
  shortjournal = {Nat. Meth.},
  title        = {BioImageIT: Open-source framework for integration of image data management with analysis},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). PyImageJ: A library for integrating ImageJ and python.
<em>NMETH</em>, <em>19</em>(11), 1326–1327. (<a
href="https://doi.org/10.1038/s41592-022-01655-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Rueden, Curtis T. and Hiner, Mark C. and Evans, Edward L. and Pinkert, Michael A. and Lucas, Alice M. and Carpenter, Anne E. and Cimini, Beth A. and Eliceiri, Kevin W.},
  doi          = {10.1038/s41592-022-01655-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1326-1327},
  shortjournal = {Nat. Meth.},
  title        = {PyImageJ: A library for integrating ImageJ and python},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The red harvester ant. <em>NMETH</em>, <em>19</em>(11),
1324–1325. (<a
href="https://doi.org/10.1038/s41592-022-01671-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Harvester ants live in desert grasslands and eat seeds. Colonies manage water stress by regulating foraging using olfactory interactions between outgoing and returning foragers. A long-term study in New Mexico shows how this collective behavior is evolving in drought conditions.},
  archive      = {J_NMETH},
  author       = {Gordon, Deborah M.},
  doi          = {10.1038/s41592-022-01671-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1324-1325},
  shortjournal = {Nat. Meth.},
  title        = {The red harvester ant},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Athlete–scientists like to sweat. <em>NMETH</em>,
<em>19</em>(11), 1323. (<a
href="https://doi.org/10.1038/s41592-022-01665-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Some scientists find ways to balance their love of athleticism with research.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01665-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1323},
  shortjournal = {Nat. Meth.},
  title        = {Athlete–scientists like to sweat},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The beauty of imaging. <em>NMETH</em>, <em>19</em>(11),
1321. (<a href="https://doi.org/10.1038/s41592-022-01677-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This month we continue our yearly celebration of the beauty of imaging.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01677-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1321},
  shortjournal = {Nat. Meth.},
  title        = {The beauty of imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). De novo construction of t cell compartment in humanized mice
engrafted with iPSC-derived thymus organoids. <em>NMETH</em>,
<em>19</em>(10), 1306–1319. (<a
href="https://doi.org/10.1038/s41592-022-01583-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hematopoietic humanized (hu) mice are powerful tools for modeling the action of human immune system and are widely used for preclinical studies and drug discovery. However, generating a functional human T cell compartment in hu mice remains challenging, primarily due to the species-related differences between human and mouse thymus. While engrafting human fetal thymic tissues can support robust T cell development in hu mice, tissue scarcity and ethical concerns limit their wide use. Here, we describe the tissue engineering of human thymus organoids from inducible pluripotent stem cells (iPSC-thymus) that can support the de novo generation of a diverse population of functional human T cells. T cells of iPSC-thymus-engrafted hu mice could mediate both cellular and humoral immune responses, including mounting robust proinflammatory responses on T cell receptor engagement, inhibiting allogeneic tumor graft growth and facilitating efficient Ig class switching. Our findings indicate that hu mice engrafted with iPSC-thymus can serve as a new animal model to study human T cell-mediated immunity and accelerate the translation of findings from animal studies into the clinic. Engraftment of human thymic organoids supports de novo development of a functional human T cell compartment in a humanized mouse model.},
  archive      = {J_NMETH},
  author       = {Zeleniak, Ann and Wiegand, Connor and Liu, Wen and McCormick, Catherine and K., Ravikumar and Alavi, Amir and Guan, Haonan and Bertera, Suzanne and Lakomy, Robert and Tajima, Asako and Cohen, Henry and Wong, Stephanie and Balikani, Lame and Mizerak, Benjamin and Bar-Joseph, Ziv and Trucco, Massimo and Banerjee, Ipsita and Fan, Yong},
  doi          = {10.1038/s41592-022-01583-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1306-1319},
  shortjournal = {Nat. Meth.},
  title        = {De novo construction of t cell compartment in humanized mice engrafted with iPSC-derived thymus organoids},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A bead-based method for high-throughput mapping of the
sequence- and force-dependence of t cell activation. <em>NMETH</em>,
<em>19</em>(10), 1295–1305. (<a
href="https://doi.org/10.1038/s41592-022-01592-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adaptive immunity relies on T lymphocytes that use αβ T cell receptors (TCRs) to discriminate among peptides presented by major histocompatibility complex molecules (pMHCs). Identifying pMHCs capable of inducing robust T cell responses will not only enable a deeper understanding of the mechanisms governing immune responses but could also have broad applications in diagnosis and treatment. T cell recognition of sparse antigenic pMHCs in vivo relies on biomechanical forces. However, in vitro screening methods test potential pMHCs without force and often at high (nonphysiological) pMHC densities and thus fail to predict potent agonists in vivo. Here, we present a technology termed BATTLES (biomechanically assisted T cell triggering for large-scale exogenous-pMHC screening) that uses biomechanical force to initiate T cell triggering for peptides and cells in parallel. BATTLES displays candidate pMHCs on spectrally encoded beads composed of a thermo-responsive polymer capable of applying shear loads to T cells, facilitating exploration of the force- and sequence-dependent landscape of T cell responses. BATTLES can be used to explore basic T cell mechanobiology and T cell-based immunotherapies. BATTLES enables high-throughput analysis of TCR–pMHC interactions in a physiological force-dependent manner.},
  archive      = {J_NMETH},
  author       = {Feng, Yinnian and Zhao, Xiang and White, Adam K. and Garcia, K. Christopher and Fordyce, Polly M.},
  doi          = {10.1038/s41592-022-01592-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1295-1305},
  shortjournal = {Nat. Meth.},
  title        = {A bead-based method for high-throughput mapping of the sequence- and force-dependence of t cell activation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A fluorescent sensor for real-time measurement of
extracellular oxytocin dynamics in the brain. <em>NMETH</em>,
<em>19</em>(10), 1286–1294. (<a
href="https://doi.org/10.1038/s41592-022-01597-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oxytocin (OT), a hypothalamic neuropeptide that acts as a neuromodulator in the brain, orchestrates a variety of animal behaviors. However, the relationship between brain OT dynamics and complex animal behaviors remains largely elusive, partly because of the lack of a suitable technique for its real-time recording in vivo. Here, we describe MTRIAOT, a G-protein-coupled receptor-based green fluorescent OT sensor that has a large dynamic range, suitable affinity, ligand specificity for OT orthologs, minimal effects on downstream signaling and long-term fluorescence stability. By combining viral gene delivery and fiber photometry-mediated fluorescence measurements, we demonstrate the utility of MTRIAOT for real-time detection of brain OT dynamics in living mice. MTRIAOT-mediated measurements indicate variability of OT dynamics depending on the behavioral context and physical condition of an animal. MTRIAOT will likely enable the analysis of OT dynamics in a variety of physiological and pathological processes. A fluorescent sensor for oxytocin called MTRIAOT has been developed. The sensor’s capabilities are demonstrated in fiber photometry measurements in freely behaving mice.},
  archive      = {J_NMETH},
  author       = {Ino, Daisuke and Tanaka, Yudai and Hibino, Hiroshi and Nishiyama, Masaaki},
  doi          = {10.1038/s41592-022-01597-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1286-1294},
  shortjournal = {Nat. Meth.},
  title        = {A fluorescent sensor for real-time measurement of extracellular oxytocin dynamics in the brain},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cell region fingerprints enable highly precise single-cell
tracking and lineage reconstruction. <em>NMETH</em>, <em>19</em>(10),
1276–1285. (<a
href="https://doi.org/10.1038/s41592-022-01603-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimental studies of cell growth, inheritance and their associated processes by microscopy require accurate single-cell observations of sufficient duration to reconstruct the genealogy. However, cell tracking—assigning identical cells on consecutive images to a track—is often challenging, resulting in laborious manual verification. Here, we propose fingerprints to identify problematic assignments rapidly. A fingerprint distance compares the structural information contained in the low frequencies of a Fourier transform to measure the similarity between cells in two consecutive images. We show that fingerprints are broadly applicable across cell types and image modalities, provided the image has sufficient structural information. Our tracker (TracX) uses fingerprints to reject unlikely assignments, thereby increasing tracking performance on published and newly generated long-term data sets. For Saccharomyces cerevisiae, we propose a comprehensive model for cell size control at the single-cell and population level centered on the Whi5 regulator, demonstrating how precise tracking can help uncover previously undescribed single-cell biology. TracX improves the accuracy of single-cell tracking by using a fingerprinting approach to measure the similarity between cells in two consecutive images. The approach is applicable across modalities and enables biological discovery.},
  archive      = {J_NMETH},
  author       = {Cuny, Andreas P. and Ponti, Aaron and Kündig, Tomas and Rudolf, Fabian and Stelling, Jörg},
  doi          = {10.1038/s41592-022-01603-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1276-1285},
  shortjournal = {Nat. Meth.},
  title        = {Cell region fingerprints enable highly precise single-cell tracking and lineage reconstruction},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Event-triggered STED imaging. <em>NMETH</em>,
<em>19</em>(10), 1268–1275. (<a
href="https://doi.org/10.1038/s41592-022-01588-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monitoring the proteins and lipids that mediate all cellular processes requires imaging methods with increased spatial and temporal resolution. STED (stimulated emission depletion) nanoscopy enables fast imaging of nanoscale structures in living cells but is limited by photobleaching. Here, we present event-triggered STED, an automated multiscale method capable of rapidly initiating two-dimensional (2D) and 3D STED imaging after detecting cellular events such as protein recruitment, vesicle trafficking and second messengers activity using biosensors. STED is applied in the vicinity of detected events to maximize the temporal resolution. We imaged synaptic vesicle dynamics at up to 24 Hz, 40 ms after local calcium activity; endocytosis and exocytosis events at up to 11 Hz, 40 ms after local protein recruitment or pH changes; and the interaction between endosomal vesicles at up to 3 Hz, 70 ms after approaching one another. Event-triggered STED extends the capabilities of live nanoscale imaging, enabling novel biological observations in real time. Event-triggered STED is an automated approach that can initiate 2D or 3D STED imaging of specific regions in biological samples after detection of an event of interest. This approach can help maximize observations in live cell imaging and enable discovery.},
  archive      = {J_NMETH},
  author       = {Alvelid, Jonatan and Damenti, Martina and Sgattoni, Chiara and Testa, Ilaria},
  doi          = {10.1038/s41592-022-01588-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1268-1275},
  shortjournal = {Nat. Meth.},
  title        = {Event-triggered STED imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Event-driven acquisition for content-enriched microscopy.
<em>NMETH</em>, <em>19</em>(10), 1262–1267. (<a
href="https://doi.org/10.1038/s41592-022-01589-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A common goal of fluorescence microscopy is to collect data on specific biological events. Yet, the event-specific content that can be collected from a sample is limited, especially for rare or stochastic processes. This is due in part to photobleaching and phototoxicity, which constrain imaging speed and duration. We developed an event-driven acquisition framework, in which neural-network-based recognition of specific biological events triggers real-time control in an instant structured illumination microscope. Our setup adapts acquisitions on-the-fly by switching between a slow imaging rate while detecting the onset of events, and a fast imaging rate during their progression. Thus, we capture mitochondrial and bacterial divisions at imaging rates that match their dynamic timescales, while extending overall imaging durations. Because event-driven acquisition allows the microscope to respond specifically to complex biological events, it acquires data enriched in relevant content. Event-driven acquisition uses neural-network-based recognition of specific biological events to trigger switching between slow and fast super-resolution imaging, enriching the capture of interesting events with high spatiotemporal resolution.},
  archive      = {J_NMETH},
  author       = {Mahecic, Dora and Stepp, Willi L. and Zhang, Chen and Griffié, Juliette and Weigert, Martin and Manley, Suliana},
  doi          = {10.1038/s41592-022-01589-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1262-1267},
  shortjournal = {Nat. Meth.},
  title        = {Event-driven acquisition for content-enriched microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BIONIC: Biological network integration using convolutions.
<em>NMETH</em>, <em>19</em>(10), 1250–1261. (<a
href="https://doi.org/10.1038/s41592-022-01616-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biological networks constructed from varied data can be used to map cellular function, but each data type has limitations. Network integration promises to address these limitations by combining and automatically weighting input information to obtain a more accurate and comprehensive representation of the underlying biology. We developed a deep learning-based network integration algorithm that incorporates a graph convolutional network framework. Our method, BIONIC (Biological Network Integration using Convolutions), learns features that contain substantially more functional information compared to existing approaches. BIONIC has unsupervised and semisupervised learning modes, making use of available gene function annotations. BIONIC is scalable in both size and quantity of the input networks, making it feasible to integrate numerous networks on the scale of the human genome. To demonstrate the use of BIONIC in identifying new biology, we predicted and experimentally validated essential gene chemical–genetic interactions from nonessential gene profiles in yeast. BIONIC uses a deep learning framework for accurate and scalable integration of biological networks.},
  archive      = {J_NMETH},
  author       = {Forster, Duncan T. and Li, Sheena C. and Yashiroda, Yoko and Yoshimura, Mami and Li, Zhijian and Isuhuaylas, Luis Alberto Vega and Itto-Nakama, Kaori and Yamanaka, Daisuke and Ohya, Yoshikazu and Osada, Hiroyuki and Wang, Bo and Bader, Gary D. and Boone, Charles},
  doi          = {10.1038/s41592-022-01616-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1250-1261},
  shortjournal = {Nat. Meth.},
  title        = {BIONIC: Biological network integration using convolutions},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ISSAAC-seq enables sensitive and flexible multimodal
profiling of chromatin accessibility and gene expression in single
cells. <em>NMETH</em>, <em>19</em>(10), 1243–1249. (<a
href="https://doi.org/10.1038/s41592-022-01601-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Joint profiling of chromatin accessibility and gene expression from the same single cell provides critical information about cell types in a tissue and cell states during a dynamic process. Here, we develop in situ sequencing hetero RNA–DNA-hybrid after assay for transposase-accessible chromatin-sequencing (ISSAAC-seq), a highly sensitive and flexible single-cell multi-omics method to interrogate chromatin accessibility and gene expression from the same single nucleus. We demonstrated that ISSAAC-seq is sensitive and provides high quality data with orders of magnitude more features than existing methods. Using the joint profiles from over 10,000 nuclei from the mouse cerebral cortex, we uncovered major and rare cell types and cell-type specific regulatory elements and identified heterogeneity at the chromatin level within established cell types defined by gene expression. Finally, we revealed distinct dynamics and relationships of gene expression and chromatin accessibility during an oligodendrocyte maturation trajectory. ISSAAC-seq offers a sensitive and modular tool for joint profiling gene expression and chromatin accessibility in single cells.},
  archive      = {J_NMETH},
  author       = {Xu, Wei and Yang, Weilong and Zhang, Yunlong and Chen, Yawen and Hong, Ni and Zhang, Qian and Wang, Xuefei and Hu, Yukun and Song, Kun and Jin, Wenfei and Chen, Xi},
  doi          = {10.1038/s41592-022-01601-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1243-1249},
  shortjournal = {Nat. Meth.},
  title        = {ISSAAC-seq enables sensitive and flexible multimodal profiling of chromatin accessibility and gene expression in single cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). RNA secondary structure packages evaluated and improved by
high-throughput experiments. <em>NMETH</em>, <em>19</em>(10), 1234–1242.
(<a href="https://doi.org/10.1038/s41592-022-01605-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the popularity of computer-aided study and design of RNA molecules, little is known about the accuracy of commonly used structure modeling packages in tasks sensitive to ensemble properties of RNA. Here, we demonstrate that the EternaBench dataset, a set of more than 20,000 synthetic RNA constructs designed on the RNA design platform Eterna, provides incisive discriminative power in evaluating current packages in ensemble-oriented structure prediction tasks. We find that CONTRAfold and RNAsoft, packages with parameters derived through statistical learning, achieve consistently higher accuracy than more widely used packages in their standard settings, which derive parameters primarily from thermodynamic experiments. We hypothesized that training a multitask model with the varied data types in EternaBench might improve inference on ensemble-based prediction tasks. Indeed, the resulting model, named EternaFold, demonstrated improved performance that generalizes to diverse external datasets including complete messenger RNAs, viral genomes probed in human cells and synthetic designs modeling mRNA vaccines. The EternaBench dataset of synthetic RNA constructs was used to directly compare RNA secondary structure prediction software packages on ensemble-oriented prediction tasks and used to train the EternaFold model for improved performance.},
  archive      = {J_NMETH},
  author       = {Wayment-Steele, Hannah K. and Kladwang, Wipapat and Strom, Alexandra I. and Lee, Jeehyung and Treuille, Adrien and Becka, Alex and Das, Rhiju},
  doi          = {10.1038/s41592-022-01605-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1234-1242},
  shortjournal = {Nat. Meth.},
  title        = {RNA secondary structure packages evaluated and improved by high-throughput experiments},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). SVision: A deep learning approach to resolve complex
structural variants. <em>NMETH</em>, <em>19</em>(10), 1230–1233. (<a
href="https://doi.org/10.1038/s41592-022-01609-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex structural variants (CSVs) encompass multiple breakpoints and are often missed or misinterpreted. We developed SVision, a deep-learning-based multi-object-recognition framework, to automatically detect and characterize CSVs from long-read sequencing data. SVision outperforms current callers at identifying the internal structure of complex events and has revealed 80 high-quality CSVs with 25 distinct structures from an individual genome. SVision directly detects CSVs without matching known structures, allowing sensitive detection of both common and previously uncharacterized complex rearrangements. SVision is a deep-learning-based method that can sensitively and accurately detect and characterize complex structural variants using long-read sequencing data.},
  archive      = {J_NMETH},
  author       = {Lin, Jiadong and Wang, Songbo and Audano, Peter A. and Meng, Deyu and Flores, Jacob I. and Kosters, Walter and Yang, Xiaofei and Jia, Peng and Marschall, Tobias and Beck, Christine R. and Ye, Kai},
  doi          = {10.1038/s41592-022-01609-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1230-1233},
  shortjournal = {Nat. Meth.},
  title        = {SVision: A deep learning approach to resolve complex structural variants},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Analysis of the human protein atlas weakly supervised
single-cell classification competition. <em>NMETH</em>, <em>19</em>(10),
1221–1229. (<a
href="https://doi.org/10.1038/s41592-022-01606-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While spatial proteomics by fluorescence imaging has quickly become an essential discovery tool for researchers, fast and scalable methods to classify and embed single-cell protein distributions in such images are lacking. Here, we present the design and analysis of the results from the competition Human Protein Atlas – Single-Cell Classification hosted on the Kaggle platform. This represents a crowd-sourced competition to develop machine learning models trained on limited annotations to label single-cell protein patterns in fluorescent images. The particular challenges of this competition include class imbalance, weak labels and multi-label classification, prompting competitors to apply a wide range of approaches in their solutions. The winning models serve as the first subcellular omics tools that can annotate single-cell locations, extract single-cell features and capture cellular dynamics. A machine learning competition results in tools for labeling protein patterns of single cells in images with population labels. The winners improve the state of the art and provide strategies to deal with weak classification challenges.},
  archive      = {J_NMETH},
  author       = {Le, Trang and Winsnes, Casper F. and Axelsson, Ulrika and Xu, Hao and Mohanakrishnan Kaimal, Jayasankar and Mahdessian, Diana and Dai, Shubin and Makarov, Ilya S. and Ostankovich, Vladislav and Xu, Yang and Benhamou, Eric and Henkel, Christof and Solovyev, Roman A. and Banić, Nikola and Bošnjak, Vito and Bošnjak, Ana and Miličević, Andrija and Ouyang, Wei and Lundberg, Emma},
  doi          = {10.1038/s41592-022-01606-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1221-1229},
  shortjournal = {Nat. Meth.},
  title        = {Analysis of the human protein atlas weakly supervised single-cell classification competition},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Best practice standards for circular RNA research.
<em>NMETH</em>, <em>19</em>(10), 1208–1220. (<a
href="https://doi.org/10.1038/s41592-022-01487-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circular RNAs (circRNAs) are formed in all domains of life and via different mechanisms. There has been an explosion in the number of circRNA papers in recent years; however, as a relatively young field, circRNA biology has an urgent need for common experimental standards for isolating, analyzing, expressing and depleting circRNAs. Here we propose a set of guidelines for circRNA studies based on the authors’ experience. This Perspective will specifically address the major class of circRNAs in Eukarya that are generated by a spliceosome-catalyzed back-splicing event. We hope that the implementation of best practice principles for circRNA research will help move the field forward and allow a better functional understanding of this fascinating group of RNAs. This Perspective focuses on experimental techniques developed for circRNA studies, and provides practical guidelines for performing circRNA research.},
  archive      = {J_NMETH},
  author       = {Nielsen, Anne F. and Bindereif, Albrecht and Bozzoni, Irene and Hanan, Mor and Hansen, Thomas B. and Irimia, Manuel and Kadener, Sebastian and Kristensen, Lasse S. and Legnini, Ivano and Morlando, Mariangela and Jarlstad Olesen, Morten T. and Pasterkamp, R. Jeroen and Preibisch, Stephan and Rajewsky, Nikolaus and Suenkel, Christin and Kjems, Jørgen},
  doi          = {10.1038/s41592-022-01487-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1208-1220},
  shortjournal = {Nat. Meth.},
  title        = {Best practice standards for circular RNA research},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Advances and opportunities in RNA structure experimental
determination and computational modeling. <em>NMETH</em>,
<em>19</em>(10), 1193–1207. (<a
href="https://doi.org/10.1038/s41592-022-01623-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Beyond transferring genetic information, RNAs are molecules with diverse functions that include catalyzing biochemical reactions and regulating gene expression. Most of these activities depend on RNAs’ specific structures. Therefore, accurately determining RNA structure is integral to advancing our understanding of RNA functions. Here, we summarize the state-of-the-art experimental and computational technologies developed to evaluate RNA secondary and tertiary structures. We also highlight how the rapid increase of experimental data facilitates the integrative modeling approaches for better resolving RNA structures. Finally, we provide our thoughts on the latest advances and challenges in RNA structure determination methods, as well as on future directions for both experimental approaches and artificial intelligence-based computational tools to model RNA structure. Ultimately, we hope the technological advances will deepen our understanding of RNA biology and facilitate RNA structure-based biomedical research such as designing specific RNA structures for therapeutics and deploying RNA-targeting small-molecule drugs. This Review summarizes recent methodological advances in experimental and computational tools developed in studying RNA structures, which provides a bridge for communication between both experimentalists and computational experts.},
  archive      = {J_NMETH},
  author       = {Zhang, Jinsong and Fei, Yuhan and Sun, Lei and Zhang, Qiangfeng Cliff},
  doi          = {10.1038/s41592-022-01623-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1193-1207},
  shortjournal = {Nat. Meth.},
  title        = {Advances and opportunities in RNA structure experimental determination and computational modeling},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A tissue-engineered artificial human thymus from human iPSCs
to study t cell immunity. <em>NMETH</em>, <em>19</em>(10), 1191–1192.
(<a href="https://doi.org/10.1038/s41592-022-01584-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A tissue engineering method using a 3D scaffolding enables the generation of an artificial human thymus from inducible pluripotent stem cells (iPSCs). The artificial thymus can be used to study human T cell development in hematopoietic humanized mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01584-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1191-1192},
  shortjournal = {Nat. Meth.},
  title        = {A tissue-engineered artificial human thymus from human iPSCs to study t cell immunity},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BATTLES: High-throughput screening of antigen recognition
under force. <em>NMETH</em>, <em>19</em>(10), 1189–1190. (<a
href="https://doi.org/10.1038/s41592-022-01593-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vivo, forces applied to molecular interactions between T cells and antigen-presenting cells are essential for specific foreign antigen recognition. A new technology, BATTLES, applies force to thousands of T cells interacting with tens of candidate antigens to identify antigens capable of efficient T cell activation. The method improves throughput over current methods that profile force-dependent interactions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01593-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1189-1190},
  shortjournal = {Nat. Meth.},
  title        = {BATTLES: High-throughput screening of antigen recognition under force},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Lighting up oxytocin dynamics in the brain with MTRIAOT.
<em>NMETH</em>, <em>19</em>(10), 1187–1188. (<a
href="https://doi.org/10.1038/s41592-022-01598-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A genetically encoded green fluorescent sensor for oxytocin, MTRIAOT, offers an opportunity to perform real-time recording of brain oxytocin dynamics in living animals.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01598-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1187-1188},
  shortjournal = {Nat. Meth.},
  title        = {Lighting up oxytocin dynamics in the brain with MTRIAOT},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BIONIC: Discovering new biology through deep learning-based
network integration. <em>NMETH</em>, <em>19</em>(10), 1185–1186. (<a
href="https://doi.org/10.1038/s41592-022-01617-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {BIONIC (Biological Network Integration using Convolutions) is a scalable deep learning network integration approach that learns and combines diverse data representations across a range of biological network types to consolidate knowledge of gene function. BIONIC outperforms existing integration approaches by capturing biological information more comprehensively and with greater accuracy than previously possible.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01617-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1185-1186},
  shortjournal = {Nat. Meth.},
  title        = {BIONIC: Discovering new biology through deep learning-based network integration},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sensitive, flexible and modular single-cell multi-omics
profiling with ISSAAC-seq. <em>NMETH</em>, <em>19</em>(10), 1183–1184.
(<a href="https://doi.org/10.1038/s41592-022-01600-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Joint profiling of multiple modalities in the same cell is challenging. We developed a method with a modular design to enable the simultaneous detection of chromatin accessibility and the transcriptome within single cells with flexible throughput.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01600-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1183-1184},
  shortjournal = {Nat. Meth.},
  title        = {Sensitive, flexible and modular single-cell multi-omics profiling with ISSAAC-seq},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Learning RNA structure prediction from crowd-designed RNAs.
<em>NMETH</em>, <em>19</em>(10), 1181–1182. (<a
href="https://doi.org/10.1038/s41592-022-01607-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA molecules designed by citizen scientists and probed in high-throughput experiments highlighted discrepancies among RNA folding algorithms in their ability to predict RNA structure ensembles. These datasets were used to train a new algorithm that demonstrated improved performance in a collection of independent datasets, including viral genomic RNAs and mRNAs probed in cells.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01607-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1181-1182},
  shortjournal = {Nat. Meth.},
  title        = {Learning RNA structure prediction from crowd-designed RNAs},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Obtaining cryo-EM structures by scanning transmission
electron microscopy. <em>NMETH</em>, <em>19</em>(10), 1179–1180. (<a
href="https://doi.org/10.1038/s41592-022-01587-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scanning transmission electron microscopy (STEM) techniques reveal atomic-resolution details of organic and inorganic materials. The application of STEM to biological vitrified specimens under low-dose cryogenic imaging conditions demonstrates that STEM also achieves near-atomic-resolution 3D structures of biological macromolecules.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01587-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1179-1180},
  shortjournal = {Nat. Meth.},
  title        = {Obtaining cryo-EM structures by scanning transmission electron microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improved tracking via cell region fingerprints.
<em>NMETH</em>, <em>19</em>(10), 1177–1178. (<a
href="https://doi.org/10.1038/s41592-022-01525-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {TracX shows that fast proofreading of track assignments leads to improved cell tracking performance.},
  archive      = {J_NMETH},
  author       = {Cimini, Beth A.},
  doi          = {10.1038/s41592-022-01525-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1177-1178},
  shortjournal = {Nat. Meth.},
  title        = {Improved tracking via cell region fingerprints},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Microscopes are coming for your job. <em>NMETH</em>,
<em>19</em>(10), 1175–1176. (<a
href="https://doi.org/10.1038/s41592-022-01566-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in microscopy, computer vision and open source software are converging to usher in a new era of microscopes that control themselves.},
  archive      = {J_NMETH},
  author       = {Pinkard, Henry and Waller, Laura},
  doi          = {10.1038/s41592-022-01566-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1175-1176},
  shortjournal = {Nat. Meth.},
  title        = {Microscopes are coming for your job},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022s). Some roads ahead for noncoding RNAs. <em>NMETH</em>,
<em>19</em>(10), 1171–1174. (<a
href="https://doi.org/10.1038/s41592-022-01628-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers roll up their sleeves for the work ahead in the noncoding RNA field.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01628-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1171-1174},
  shortjournal = {Nat. Meth.},
  title        = {Some roads ahead for noncoding RNAs},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). How noncoding RNAs began to leave the junkyard.
<em>NMETH</em>, <em>19</em>(10), 1167–1170. (<a
href="https://doi.org/10.1038/s41592-022-01627-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The research community focused on noncoding RNAs keeps growing. Skepticism about the field has some history.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01627-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1167-1170},
  shortjournal = {Nat. Meth.},
  title        = {How noncoding RNAs began to leave the junkyard},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022q). Sequencing single cells without killing. <em>NMETH</em>,
<em>19</em>(10), 1166. (<a
href="https://doi.org/10.1038/s41592-022-01648-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01648-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1166},
  shortjournal = {Nat. Meth.},
  title        = {Sequencing single cells without killing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). The human immune interactome. <em>NMETH</em>,
<em>19</em>(10), 1166. (<a
href="https://doi.org/10.1038/s41592-022-01649-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01649-2},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1166},
  shortjournal = {Nat. Meth.},
  title        = {The human immune interactome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A closer look at FluoroCubes. <em>NMETH</em>,
<em>19</em>(10), 1165. (<a
href="https://doi.org/10.1038/s41592-022-01647-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers explore the unique and puzzling photostability of DNA FluoroCubes. Moreover, they improve the probes’ performance and highlight their diverse applicability.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01647-4},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1165},
  shortjournal = {Nat. Meth.},
  title        = {A closer look at FluoroCubes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Advances in nanopore direct RNA sequencing. <em>NMETH</em>,
<em>19</em>(10), 1160–1164. (<a
href="https://doi.org/10.1038/s41592-022-01633-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanopore direct RNA sequencing (DRS) reads continuous native RNA strands. Early adopters have used this technology to document nucleotide modifications and 3′ polyadenosine tails on RNA strands without added chemistry steps. Individual strands ranging in length from 70 to 26,000 nucleotides have been sequenced. In our opinion, broader acceptance of nanopore DRS by molecular biologists and cell biologists will be accelerated by higher basecall accuracy and lower RNA input requirements.},
  archive      = {J_NMETH},
  author       = {Jain, Miten and Abu-Shumays, Robin and Olsen, Hugh E. and Akeson, Mark},
  doi          = {10.1038/s41592-022-01633-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1160-1164},
  shortjournal = {Nat. Meth.},
  title        = {Advances in nanopore direct RNA sequencing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Regulatory non-coding RNAs: Everything is possible, but what
is important? <em>NMETH</em>, <em>19</em>(10), 1156–1159. (<a
href="https://doi.org/10.1038/s41592-022-01629-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, the number of annotated noncoding RNAs (ncRNAs) and RNA-binding proteins (RBPs) has increased dramatically. The wide range of RBPs identified highlights the enormous potential for RNA in virtually all aspects of cell biology, from transcriptional regulation to metabolic control. Yet, there is a growing gap between what is possible and what has been demonstrated to be functionally important. Here we highlight recent methodological developments in the study of RNA–protein interactions, discuss the challenges and opportunities for exploring their functional roles, and provide our perspectives on what is needed to bridge the gap in this rapidly expanding field.},
  archive      = {J_NMETH},
  author       = {Guo, Jimmy K. and Guttman, Mitchell},
  doi          = {10.1038/s41592-022-01629-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1156-1159},
  shortjournal = {Nat. Meth.},
  title        = {Regulatory non-coding RNAs: Everything is possible, but what is important?},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards higher-resolution and in vivo understanding of
lncRNA biogenesis and function. <em>NMETH</em>, <em>19</em>(10),
1152–1155. (<a
href="https://doi.org/10.1038/s41592-022-01626-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent studies have revealed multifaceted roles of long noncoding RNAs (lncRNAs) in gene regulation, accompanying an increased understanding of lncRNA processing, localization, interacting macromolecules and structural modules. Here, progress and recently developed technological advances for understanding lncRNA biogenesis, modes of action and cellular phenotypes are highlighted, and challenges and opportunities towards higher-resolution and in vivo studies in this field are discussed.},
  archive      = {J_NMETH},
  author       = {Chen, Ling-Ling},
  doi          = {10.1038/s41592-022-01626-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1152-1155},
  shortjournal = {Nat. Meth.},
  title        = {Towards higher-resolution and in vivo understanding of lncRNA biogenesis and function},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The turquoise killifish. <em>NMETH</em>, <em>19</em>(10),
1150–1151. (<a
href="https://doi.org/10.1038/s41592-022-01631-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A naturally short-lived fish opens the gates for rapid exploration of vertebrate aging.},
  archive      = {J_NMETH},
  author       = {Harel, Itamar},
  doi          = {10.1038/s41592-022-01631-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1150-1151},
  shortjournal = {Nat. Meth.},
  title        = {The turquoise killifish},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Cross-disciplinary ways to connect and blend.
<em>NMETH</em>, <em>19</em>(10), 1149. (<a
href="https://doi.org/10.1038/s41592-022-01622-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Jumping across silos is a tough sport. But practitioners find joy in leaving comfort zones.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01622-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1149},
  shortjournal = {Nat. Meth.},
  title        = {Cross-disciplinary ways to connect and blend},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Decoding noncoding RNAs. <em>NMETH</em>, <em>19</em>(10),
1147–1148. (<a
href="https://doi.org/10.1038/s41592-022-01654-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research interest in noncoding RNAs and their biological implications in a variety of cellular contexts has been growing. In this issue, we present a series of pieces discussing recent method advances and future directions for deciphering the regulatory roles of noncoding RNAs.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01654-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1147-1148},
  shortjournal = {Nat. Meth.},
  title        = {Decoding noncoding RNAs},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antibody stabilization for thermally accelerated deep
immunostaining. <em>NMETH</em>, <em>19</em>(9), 1137–1146. (<a
href="https://doi.org/10.1038/s41592-022-01569-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antibodies have diverse applications due to their high reaction specificities but are sensitive to denaturation when a higher working temperature is required. We have developed a simple, highly scalable and generalizable chemical approach for stabilizing off-the-shelf antibodies against thermal and chemical denaturation. We demonstrate that the stabilized antibodies (termed SPEARs) can withstand up to 4 weeks of continuous heating at 55 °C and harsh denaturants, and apply our method to 33 tested antibodies. SPEARs enable flexible applications of thermocycling and denaturants to dynamically modulate their binding kinetics, reaction equilibrium, macromolecular diffusivity and aggregation propensity. In particular, we show that SPEARs permit the use of a thermally facilitated three-dimensional immunolabeling strategy (termed ThICK staining), achieving whole mouse brain immunolabeling within 72 h, as well as nearly fourfold deeper penetration with threefold less antibodies in human brain tissue. With faster deep-tissue immunolabeling and broad compatibility with tissue processing and clearing methods without the need for any specialized equipment, we anticipate the wide applicability of ThICK staining with SPEARs for deep immunostaining. Thermostable antibodies called SPEARs enable rapid immunostaining with improved tissue penetration.},
  archive      = {J_NMETH},
  author       = {Lai, Hei Ming and Tang, Yumi and Lau, Zachary Y. H. and Campbell, Robert A. A. and Yau, Juno C. N. and Chan, Caleb C. Y. and Chan, Danny C. W. and Wong, Tin Yan and Wong, Harriet K. T. and Yan, Leo Y. C. and Wu, William K. K. and Wong, Sunny H. and Kwok, Ka-Wai and Wing, Yun-Kwok and Lam, Henry H. N. and Ng, Ho-Keung and Mrsic-Flogel, Thomas D. and Mok, Vincent C. T. and Chan, Jason Y. K. and Ko, Ho},
  doi          = {10.1038/s41592-022-01569-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1137-1146},
  shortjournal = {Nat. Meth.},
  title        = {Antibody stabilization for thermally accelerated deep immunostaining},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-particle cryo-EM structures from iDPC–STEM at
near-atomic resolution. <em>NMETH</em>, <em>19</em>(9), 1126–1136. (<a
href="https://doi.org/10.1038/s41592-022-01586-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In electron cryomicroscopy (cryo-EM), molecular images of vitrified biological samples are obtained by conventional transmission microscopy (CTEM) using large underfocuses and subsequently computationally combined into a high-resolution three-dimensional structure. Here, we apply scanning transmission electron microscopy (STEM) using the integrated differential phase contrast mode also known as iDPC–STEM to two cryo-EM test specimens, keyhole limpet hemocyanin (KLH) and tobacco mosaic virus (TMV). The micrographs show complete contrast transfer to high resolution and enable the cryo-EM structure determination for KLH at 6.5 Å resolution, as well as for TMV at 3.5 Å resolution using single-particle reconstruction methods, which share identical features with maps obtained by CTEM of a previously acquired same-sized TMV data set. These data show that STEM imaging in general, and in particular the iDPC–STEM approach, can be applied to vitrified single-particle specimens to determine near-atomic resolution cryo-EM structures of biological macromolecules. This paper explores the use of scanning transmission electron microscopy (STEM) to vitrified biological samples for biomolecular structure elucidation. Integrated differential phase contrast (iDPC)–STEM imaging of keyhole limpet hemocyanin and tobacco mosaic virus enabled cryo-EM structure determination at 6.5 and 3.5 Å resolution, respectively.},
  archive      = {J_NMETH},
  author       = {Lazić, Ivan and Wirix, Maarten and Leidl, Max Leo and de Haas, Felix and Mann, Daniel and Beckers, Maximilian and Pechnikova, Evgeniya V. and Müller-Caspary, Knut and Egoavil, Ricardo and Bosch, Eric G. T. and Sachse, Carsten},
  doi          = {10.1038/s41592-022-01586-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1126-1136},
  shortjournal = {Nat. Meth.},
  title        = {Single-particle cryo-EM structures from iDPC–STEM at near-atomic resolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Residue-wise local quality estimation for protein models
from cryo-EM maps. <em>NMETH</em>, <em>19</em>(9), 1116–1125. (<a
href="https://doi.org/10.1038/s41592-022-01574-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An increasing number of protein structures are being determined by cryogenic electron microscopy (cryo-EM). Although the resolution of determined cryo-EM density maps is improving in general, there are still many cases where amino acids of a protein are assigned with different levels of confidence. Here we developed a method that identifies potential misassignment of residues in the map, including residue shifts along an otherwise correct main-chain trace. The score, named DAQ, computes the likelihood that the local density corresponds to different amino acids, atoms, and secondary structures, estimated via deep learning, and assesses the consistency of the amino acid assignment in the protein structure model with that likelihood. When DAQ was applied to different versions of model structures in the Protein Data Bank that were derived from the same density maps, a clear improvement in the DAQ score was observed in the newer versions of the models. DAQ also found potential misassignment errors in a substantial number of deposited protein structure models built into cryo-EM maps. The DAQ score assesses the consistency of amino acid assignment in protein structure models with local density from cryo-EM maps. The method complements existing quality metrics and is a versatile tool for highlighting problematic regions of model structures.},
  archive      = {J_NMETH},
  author       = {Terashi, Genki and Wang, Xiao and Maddhuri Venkata Subramaniya, Sai Raghavendra and Tesmer, John J. G. and Kihara, Daisuke},
  doi          = {10.1038/s41592-022-01574-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1116-1125},
  shortjournal = {Nat. Meth.},
  title        = {Residue-wise local quality estimation for protein models from cryo-EM maps},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). US-align: Universal structure alignments of proteins,
nucleic acids, and macromolecular complexes. <em>NMETH</em>,
<em>19</em>(9), 1109–1115. (<a
href="https://doi.org/10.1038/s41592-022-01585-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structure comparison and alignment are of fundamental importance in structural biology studies. We developed the first universal platform, US-align, to uniformly align monomer and complex structures of different macromolecules—proteins, RNAs and DNAs. The pipeline is built on a uniform TM-score objective function coupled with a heuristic alignment searching algorithm. Large-scale benchmarks demonstrated consistent advantages of US-align over state-of-the-art methods in pairwise and multiple structure alignments of different molecules. Detailed analyses showed that the main advantage of US-align lies in the extensive optimization of the unified objective function powered by efficient heuristic search iterations, which substantially improve the accuracy and speed of the structural alignment process. Meanwhile, the universal protocol fusing different molecular and structural types helps facilitate the heterogeneous oligomer structure comparison and template-based protein–protein and protein–RNA/DNA docking. US-align is a universal protocol for monomeric and oligomeric structural alignments of protein, RNA and DNA molecules, built on the coupling of a uniform TM-score objective function and the heuristic iterative searching algorithm.},
  archive      = {J_NMETH},
  author       = {Zhang, Chengxin and Shine, Morgan and Pyle, Anna Marie and Zhang, Yang},
  doi          = {10.1038/s41592-022-01585-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1109-1115},
  shortjournal = {Nat. Meth.},
  title        = {US-align: Universal structure alignments of proteins, nucleic acids, and macromolecular complexes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). MIRA: Joint regulatory modeling of multimodal expression and
chromatin accessibility in single cells. <em>NMETH</em>, <em>19</em>(9),
1097–1108. (<a
href="https://doi.org/10.1038/s41592-022-01595-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rigorously comparing gene expression and chromatin accessibility in the same single cells could illuminate the logic of how coupling or decoupling of these mechanisms regulates fate commitment. Here we present MIRA, probabilistic multimodal models for integrated regulatory analysis, a comprehensive methodology that systematically contrasts transcription and accessibility to infer the regulatory circuitry driving cells along cell state trajectories. MIRA leverages topic modeling of cell states and regulatory potential modeling of individual gene loci. MIRA thereby represents cell states in an efficient and interpretable latent space, infers high-fidelity cell state trees, determines key regulators of fate decisions at branch points and exposes the variable influence of local accessibility on transcription at distinct loci. Applied to epidermal differentiation and embryonic brain development from two different multimodal platforms, MIRA revealed that early developmental genes were tightly regulated by local chromatin landscape whereas terminal fate genes were titrated without requiring extensive chromatin remodeling. MIRA facilitates accurate inference of cell state trees and regulatory mechanisms driving cell fate decisions using single-cell multimodal data profiling gene expression and chromatin accessibility.},
  archive      = {J_NMETH},
  author       = {Lynch, Allen W. and Theodoris, Christina V. and Long, Henry W. and Brown, Myles and Liu, X. Shirley and Meyer, Clifford A.},
  doi          = {10.1038/s41592-022-01595-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1097-1108},
  shortjournal = {Nat. Meth.},
  title        = {MIRA: Joint regulatory modeling of multimodal expression and chromatin accessibility in single cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ScBasset: Sequence-based modeling of single-cell ATAC-seq
using convolutional neural networks. <em>NMETH</em>, <em>19</em>(9),
1088–1096. (<a
href="https://doi.org/10.1038/s41592-022-01562-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell assay for transposase-accessible chromatin using sequencing (scATAC) shows great promise for studying cellular heterogeneity in epigenetic landscapes, but there remain important challenges in the analysis of scATAC data due to the inherent high dimensionality and sparsity. Here we introduce scBasset, a sequence-based convolutional neural network method to model scATAC data. We show that by leveraging the DNA sequence information underlying accessibility peaks and the expressiveness of a neural network model, scBasset achieves state-of-the-art performance across a variety of tasks on scATAC and single-cell multiome datasets, including cell clustering, scATAC profile denoising, data integration across assays and transcription factor activity inference. Using a sequence-based deep neural network, scBasset facilitates various tasks of single-cell ATAC-seq analysis in a unified framework.},
  archive      = {J_NMETH},
  author       = {Yuan, Han and Kelley, David R.},
  doi          = {10.1038/s41592-022-01562-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1088-1096},
  shortjournal = {Nat. Meth.},
  title        = {ScBasset: Sequence-based modeling of single-cell ATAC-seq using convolutional neural networks},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cell type-specific inference of differential expression in
spatial transcriptomics. <em>NMETH</em>, <em>19</em>(9), 1076–1087. (<a
href="https://doi.org/10.1038/s41592-022-01575-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central problem in spatial transcriptomics is detecting differentially expressed (DE) genes within cell types across tissue context. Challenges to learning DE include changing cell type composition across space and measurement pixels detecting transcripts from multiple cell types. Here, we introduce a statistical method, cell type-specific inference of differential expression (C-SIDE), that identifies cell type-specific DE in spatial transcriptomics, accounting for localization of other cell types. We model gene expression as an additive mixture across cell types of log-linear cell type-specific expression functions. C-SIDE’s framework applies to many contexts: DE due to pathology, anatomical regions, cell-to-cell interactions and cellular microenvironment. Furthermore, C-SIDE enables statistical inference across multiple/replicates. Simulations and validation experiments on Slide-seq, MERFISH and Visium datasets demonstrate that C-SIDE accurately identifies DE with valid uncertainty quantification. Last, we apply C-SIDE to identify plaque-dependent immune activity in Alzheimer’s disease and cellular interactions between tumor and immune cells. We distribute C-SIDE within the R package https://github.com/dmcable/spacexr . C-SIDE facilitates accurate cell type-specific differential expression analysis for multiple spatially resolved transcriptomics technologies by cell type mixture modeling.},
  archive      = {J_NMETH},
  author       = {Cable, Dylan M. and Murray, Evan and Shanmugam, Vignesh and Zhang, Simon and Zou, Luli S. and Diao, Michael and Chen, Haiqi and Macosko, Evan Z. and Irizarry, Rafael A. and Chen, Fei},
  doi          = {10.1038/s41592-022-01575-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1076-1087},
  shortjournal = {Nat. Meth.},
  title        = {Cell type-specific inference of differential expression in spatial transcriptomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DNA-PAINT MINFLUX nanoscopy. <em>NMETH</em>, <em>19</em>(9),
1072–1075. (<a
href="https://doi.org/10.1038/s41592-022-01577-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MINimal fluorescence photon FLUXes (MINFLUX) nanoscopy, providing photon-efficient fluorophore localizations, has brought about three-dimensional resolution at nanometer scales. However, by using an intrinsic on–off switching process for single fluorophore separation, initial MINFLUX implementations have been limited to two color channels. Here we show that MINFLUX can be effectively combined with sequentially multiplexed DNA-based labeling (DNA-PAINT), expanding MINFLUX nanoscopy to multiple molecular targets. Our method is exemplified with three-color recordings of mitochondria in human cells. A systematic exploration of MINFLUX nanoscopy with DNA-PAINT labeling leads to improved nanoscopy in fixed cells and MINFLUX imaging with increased multiplexing, as exemplified by three-color imaging of mitochondria in mammalian cells.},
  archive      = {J_NMETH},
  author       = {Ostersehlt, Lynn M. and Jans, Daniel C. and Wittek, Anna and Keller-Findeisen, Jan and Inamdar, Kaushik and Sahl, Steffen J. and Hell, Stefan W. and Jakobs, Stefan},
  doi          = {10.1038/s41592-022-01577-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1072-1075},
  shortjournal = {Nat. Meth.},
  title        = {DNA-PAINT MINFLUX nanoscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Challenges and opportunities for the next generation of
cardiovascular tissue engineering. <em>NMETH</em>, <em>19</em>(9),
1064–1071. (<a
href="https://doi.org/10.1038/s41592-022-01591-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered cardiac tissues derived from human induced pluripotent stem cells offer unique opportunities for patient-specific disease modeling, drug discovery and cardiac repair. Since the first engineered hearts were introduced over two decades ago, human induced pluripotent stem cell-based three-dimensional cardiac organoids and heart-on-a-chip systems have now become mainstays in basic cardiovascular research as valuable platforms for investigating fundamental human pathophysiology and development. However, major obstacles remain to be addressed before the field can truly advance toward commercial and clinical translation. Here we provide a snapshot of the state-of-the-art methods in cardiac tissue engineering, with a focus on in vitro models of the human heart. Looking ahead, we discuss major challenges and opportunities in the field and suggest strategies for enabling broad acceptance of engineered cardiac tissues as models of cardiac pathophysiology and testbeds for the development of therapies. In this Perspective, technologies and challenges in the cardiac tissue engineering field are discussed and strategies to overcome these challenges are proposed.},
  archive      = {J_NMETH},
  author       = {Cho, Sangkyun and Discher, Dennis E. and Leong, Kam W. and Vunjak-Novakovic, Gordana and Wu, Joseph C.},
  doi          = {10.1038/s41592-022-01591-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1064-1071},
  shortjournal = {Nat. Meth.},
  title        = {Challenges and opportunities for the next generation of cardiovascular tissue engineering},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Advances in measuring cancer cell metabolism with
subcellular resolution. <em>NMETH</em>, <em>19</em>(9), 1048–1063. (<a
href="https://doi.org/10.1038/s41592-022-01572-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing metabolism in cancer is crucial for understanding tumor biology and for developing potential therapies. Although most metabolic investigations analyze averaged metabolite levels from all cell compartments, subcellular metabolomics can provide more detailed insight into the biochemical processes associated with the disease. Methodological limitations have historically prevented the wider application of subcellular metabolomics in cancer research. Recently, however, ways to distinguish and identify metabolic pathways within organelles have been developed, including state-of-the-art methods to monitor metabolism in situ (such as mass spectrometry-based imaging, Raman spectroscopy and fluorescence microscopy), to isolate key organelles via new approaches and to use tailored isotope-tracing strategies. Herein, we examine the advantages and limitations of these developments and look to the future of this field of research. This Review covers advances in methods for studying metabolism at the subcellular level and how they have influenced our understanding of cancer biology.},
  archive      = {J_NMETH},
  author       = {Ruiz-Rodado, Victor and Lita, Adrian and Larion, Mioara},
  doi          = {10.1038/s41592-022-01572-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1048-1063},
  shortjournal = {Nat. Meth.},
  title        = {Advances in measuring cancer cell metabolism with subcellular resolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A statistical method to uncover gene expression changes in
spatial transcriptomics. <em>NMETH</em>, <em>19</em>(9), 1046–1047. (<a
href="https://doi.org/10.1038/s41592-022-01576-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell type-specific inference of differential expression (C-SIDE) is a statistical model that identifies which genes (within a determined cell type) are differentially expressed on the basis of spatial position, pathological changes or cell–cell interactions. C-SIDE facilitates differential expression analysis in spatial transcriptomics by jointly modeling cell type mixtures and spatially varying gene expression.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01576-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1046-1047},
  shortjournal = {Nat. Meth.},
  title        = {A statistical method to uncover gene expression changes in spatial transcriptomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). PROBER: Overcoming the high noise of DNA–protein
interactions studies. <em>NMETH</em>, <em>19</em>(9), 1044–1045. (<a
href="https://doi.org/10.1038/s41592-022-01553-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {PROBER is a fast and sensitive episome-based method to identify sequence-specific DNA-binding proteins from living cells using proximity proteomics. This method quantifies steady-state and inducible association of transcription factors and corresponding chromatin regulators to specific DNA sequences as well as binding quantitative trait loci present as a result of single nucleotide variants.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01553-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1044-1045},
  shortjournal = {Nat. Meth.},
  title        = {PROBER: Overcoming the high noise of DNA–protein interactions studies},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). How regulatory sequences learn cell representations.
<em>NMETH</em>, <em>19</em>(9), 1041–1043. (<a
href="https://doi.org/10.1038/s41592-022-01570-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New computational method uses convolutional neural networks for cis-regulatory sequence analysis to analyze and cluster scATAC-seq data.},
  archive      = {J_NMETH},
  author       = {Aerts, Stein},
  doi          = {10.1038/s41592-022-01570-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1041-1043},
  shortjournal = {Nat. Meth.},
  title        = {How regulatory sequences learn cell representations},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Diving deeper into the proteome. <em>NMETH</em>,
<em>19</em>(9), 1036–1040. (<a
href="https://doi.org/10.1038/s41592-022-01599-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As new technology enables researchers to find and characterize less-common post-translational modifications that drive gene expression and cellular metabolism, the movement to catalog the entire human proteome gains momentum},
  archive      = {J_NMETH},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41592-022-01599-9},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1036-1040},
  shortjournal = {Nat. Meth.},
  title        = {Diving deeper into the proteome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Nanopores distinguish RNA modifications. <em>NMETH</em>,
<em>19</em>(9), 1035. (<a
href="https://doi.org/10.1038/s41592-022-01614-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Different nucleotides and derivatives can be read out by MspA-PBA, an engineered nanopore.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01614-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1035},
  shortjournal = {Nat. Meth.},
  title        = {Nanopores distinguish RNA modifications},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Neuroscience data analysis in the cloud. <em>NMETH</em>,
<em>19</em>(9), 1034. (<a
href="https://doi.org/10.1038/s41592-022-01613-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The NeuroCAAS platform simplifies data analysis in the neuroscience space for users and enhances reproducibility.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01613-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1034},
  shortjournal = {Nat. Meth.},
  title        = {Neuroscience data analysis in the cloud},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Ex utero synthetic embryos. <em>NMETH</em>, <em>19</em>(9),
1033. (<a href="https://doi.org/10.1038/s41592-022-01612-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method to develop gastrulating embryoids from stem cells ex utero.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01612-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1033},
  shortjournal = {Nat. Meth.},
  title        = {Ex utero synthetic embryos},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Sensitive protein analysis with plexDIA. <em>NMETH</em>,
<em>19</em>(9), 1032. (<a
href="https://doi.org/10.1038/s41592-022-01611-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {plexDIA increases throughput of proteomics analysis with low sample amounts.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01611-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1032},
  shortjournal = {Nat. Meth.},
  title        = {Sensitive protein analysis with plexDIA},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022v). Tracking the order of transcriptional events in living
cells. <em>NMETH</em>, <em>19</em>(9), 1031. (<a
href="https://doi.org/10.1038/s41592-022-01610-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers describe Retro-Cascorder, a tool that employs prokaryotic retrons and CRISPR integrases to write transcriptional events into the genome of Escherichia coli.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01610-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1031},
  shortjournal = {Nat. Meth.},
  title        = {Tracking the order of transcriptional events in living cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CAD we share? Publishing reproducible microscope hardware.
<em>NMETH</em>, <em>19</em>(9), 1026–1030. (<a
href="https://doi.org/10.1038/s41592-022-01484-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we discuss barriers to reproducibility in regard to microscopes and related hardware, along with best practices for sharing novel designs created using computer-aided design (CAD). We hope to start a fruitful community discussion on how instrument development, especially in microscopy, can become more open and reproducible, ultimately leading to better, more trustworthy science.},
  archive      = {J_NMETH},
  author       = {Diederich, Benedict and Müllenbroich, Caroline and Vladimirov, Nikita and Bowman, Richard and Stirling, Julian and Reynaud, Emmanuel G. and Andreev, Andrey},
  doi          = {10.1038/s41592-022-01484-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1026-1030},
  shortjournal = {Nat. Meth.},
  title        = {CAD we share? publishing reproducible microscope hardware},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Open microscopy in the life sciences: Quo vadis?
<em>NMETH</em>, <em>19</em>(9), 1020–1025. (<a
href="https://doi.org/10.1038/s41592-022-01602-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light microscopy enables researchers to observe cellular mechanisms with high spatial and temporal resolution. However, the increasing complexity of current imaging technologies, coupled with financial constraints of potential users, hampers the general accessibility and potential reach of cutting-edge microscopy. Open microscopy can address this issue by making well-designed and well-documented hardware and software solutions openly available to a broad audience. In this Comment, we provide a definition of open microscopy and present recent projects in the field. We discuss current and future challenges of open microscopy and their implications for funders, policymakers, researchers and scientists. We believe that open microscopy requires a holistic approach. Sample preparation, designing and building of hardware components, writing software, data acquisition and data interpretation must go hand in hand to enable interdisciplinary and reproducible science to the benefit of society.},
  archive      = {J_NMETH},
  author       = {Hohlbein, Johannes and Diederich, Benedict and Marsikova, Barbora and Reynaud, Emmanuel G. and Holden, Séamus and Jahr, Wiebke and Haase, Robert and Prakash, Kirti},
  doi          = {10.1038/s41592-022-01602-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1020-1025},
  shortjournal = {Nat. Meth.},
  title        = {Open microscopy in the life sciences: Quo vadis?},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). VDJdb in the pandemic era: A compendium of t cell receptors
specific for SARS-CoV-2. <em>NMETH</em>, <em>19</em>(9), 1017–1019. (<a
href="https://doi.org/10.1038/s41592-022-01578-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Goncharov, Mikhail and Bagaev, Dmitry and Shcherbinin, Dmitrii and Zvyagin, Ivan and Bolotin, Dmitry and Thomas, Paul G. and Minervina, Anastasia A. and Pogorelyy, Mikhail V. and Ladell, Kristin and McLaren, James E. and Price, David A. and Nguyen, Thi H. O. and Rowntree, Louise C. and Clemens, E. Bridie and Kedzierska, Katherine and Dolton, Garry and Rius, Cristina Rafael and Sewell, Andrew and Samir, Jerome and Luciani, Fabio and Zornikova, Ksenia V. and Khmelevskaya, Alexandra A. and Sheetikov, Saveliy A. and Efimov, Grigory A. and Chudakov, Dmitry and Shugay, Mikhail},
  doi          = {10.1038/s41592-022-01578-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1017-1019},
  shortjournal = {Nat. Meth.},
  title        = {VDJdb in the pandemic era: A compendium of t cell receptors specific for SARS-CoV-2},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The crustacean parhyale. <em>NMETH</em>, <em>19</em>(9),
1015–1016. (<a
href="https://doi.org/10.1038/s41592-022-01596-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Parhyale hawaiensis comes from tropical intertidal shores and mangroves. In research, it is used to explore topics ranging from embryonic development and regeneration, to tidal rhythms and environmental pollution.},
  archive      = {J_NMETH},
  author       = {Averof, Michalis},
  doi          = {10.1038/s41592-022-01596-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1015-1016},
  shortjournal = {Nat. Meth.},
  title        = {The crustacean parhyale},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Hi, new PI. <em>NMETH</em>, <em>19</em>(9), 1014. (<a
href="https://doi.org/10.1038/s41592-022-01594-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Your name is on the door to your new lab. Life is getting exciting and turbulent.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01594-0},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1014},
  shortjournal = {Nat. Meth.},
  title        = {Hi, new PI},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conferences then, now, and moving forward. <em>NMETH</em>,
<em>19</em>(9), 1013. (<a
href="https://doi.org/10.1038/s41592-022-01620-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As pandemic restrictions ease and we at Nature Methods begin to travel again, we muse about the highs and lows of in-person and virtual meetings and imagine the future of scientific conferences.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01620-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1013},
  shortjournal = {Nat. Meth.},
  title        = {Conferences then, now, and moving forward},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Functional ultrasound localization microscopy reveals
brain-wide neurovascular activity on a microscopic scale.
<em>NMETH</em>, <em>19</em>(8), 1004–1012. (<a
href="https://doi.org/10.1038/s41592-022-01549-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The advent of neuroimaging has increased our understanding of brain function. While most brain-wide functional imaging modalities exploit neurovascular coupling to map brain activity at millimeter resolutions, the recording of functional responses at microscopic scale in mammals remains the privilege of invasive electrophysiological or optical approaches, but is mostly restricted to either the cortical surface or the vicinity of implanted sensors. Ultrasound localization microscopy (ULM) has achieved transcranial imaging of cerebrovascular flow, up to micrometre scales, by localizing intravenously injected microbubbles; however, the long acquisition time required to detect microbubbles within microscopic vessels has so far restricted ULM application mainly to microvasculature structural imaging. Here we show how ULM can be modified to quantify functional hyperemia dynamically during brain activation reaching a 6.5-µm spatial and 1-s temporal resolution in deep regions of the rat brain. Functional ultrasound localization microscopy monitors cerebrovascular blood flow by detecting the flow of injected microbubbles, providing access to brain activity at high spatiotemporal resolution.},
  archive      = {J_NMETH},
  author       = {Renaudin, Noémi and Demené, Charlie and Dizeux, Alexandre and Ialy-Radio, Nathalie and Pezet, Sophie and Tanter, Mickael},
  doi          = {10.1038/s41592-022-01549-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1004-1012},
  shortjournal = {Nat. Meth.},
  title        = {Functional ultrasound localization microscopy reveals brain-wide neurovascular activity on a microscopic scale},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Self-supervised deep learning encodes high-resolution
features of protein subcellular localization. <em>NMETH</em>,
<em>19</em>(8), 995–1003. (<a
href="https://doi.org/10.1038/s41592-022-01541-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Explaining the diversity and complexity of protein localization is essential to fully understand cellular architecture. Here we present cytoself, a deep-learning approach for fully self-supervised protein localization profiling and clustering. Cytoself leverages a self-supervised training scheme that does not require preexisting knowledge, categories or annotations. Training cytoself on images of 1,311 endogenously labeled proteins from the OpenCell database reveals a highly resolved protein localization atlas that recapitulates major scales of cellular organization, from coarse classes, such as nuclear and cytoplasmic, to the subtle localization signatures of individual protein complexes. We quantitatively validate cytoself’s ability to cluster proteins into organelles and protein complexes, showing that cytoself outperforms previous self-supervised approaches. Moreover, to better understand the inner workings of our model, we dissect the emergent features from which our clustering is derived, interpret them in the context of the fluorescence images, and analyze the performance contributions of each component of our approach. Cytoself is a self-supervised deep learning-based approach for profiling and clustering protein localization from fluorescence images. Cytoself outperforms established approaches and can accurately predict protein subcellular localization.},
  archive      = {J_NMETH},
  author       = {Kobayashi, Hirofumi and Cheveralls, Keith C. and Leonetti, Manuel D. and Royer, Loic A.},
  doi          = {10.1038/s41592-022-01541-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {995-1003},
  shortjournal = {Nat. Meth.},
  title        = {Self-supervised deep learning encodes high-resolution features of protein subcellular localization},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Photoswitching fingerprint analysis bypasses the 10-nm
resolution barrier. <em>NMETH</em>, <em>19</em>(8), 986–994. (<a
href="https://doi.org/10.1038/s41592-022-01548-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in super-resolution microscopy have demonstrated single-molecule localization precisions of a few nanometers. However, translation of such high localization precisions into sub-10-nm spatial resolution in biological samples remains challenging. Here we show that resonance energy transfer between fluorophores separated by less than 10 nm results in accelerated fluorescence blinking and consequently lower localization probabilities impeding sub-10-nm fluorescence imaging. We demonstrate that time-resolved fluorescence detection in combination with photoswitching fingerprint analysis can be used to determine the number and distance even of spatially unresolvable fluorophores in the sub-10-nm range. In combination with genetic code expansion with unnatural amino acids and bioorthogonal click labeling with small fluorophores, photoswitching fingerprint analysis can be used advantageously to reveal information about the number of fluorophores present and their distances in the sub-10-nm range in cells. Energy transfer between fluorophores is shown to impede SMLM at sub-10-nm spatial resolution. Time-resolved detection and photoswitching fingerprinting analysis are used to determine the number and separation of closely spaced fluorophores.},
  archive      = {J_NMETH},
  author       = {Helmerich, Dominic A. and Beliu, Gerti and Taban, Danush and Meub, Mara and Streit, Marcel and Kuhlemann, Alexander and Doose, Sören and Sauer, Markus},
  doi          = {10.1038/s41592-022-01548-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {986-994},
  shortjournal = {Nat. Meth.},
  title        = {Photoswitching fingerprint analysis bypasses the 10-nm resolution barrier},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Directed evolution of adeno-associated virus for efficient
gene delivery to microglia. <em>NMETH</em>, <em>19</em>(8), 976–985. (<a
href="https://doi.org/10.1038/s41592-022-01547-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As the resident immune cells in the central nervous system (CNS), microglia orchestrate immune responses and dynamically sculpt neural circuits in the CNS. Microglial dysfunction and mutations of microglia-specific genes have been implicated in many diseases of the CNS. Developing effective and safe vehicles for transgene delivery into microglia will facilitate the studies of microglia biology and microglia-associated disease mechanisms. Here, we report the discovery of adeno-associated virus (AAV) variants that mediate efficient in vitro and in vivo microglial transduction via directed evolution of the AAV capsid protein. These AAV-cMG and AAV-MG variants are capable of delivering various genetic payloads into microglia with high efficiency, and enable sufficient transgene expression to support fluorescent labeling, Ca2+ and neurotransmitter imaging and genome editing in microglia in vivo. Furthermore, single-cell RNA sequencing shows that the AAV-MG variants mediate in vivo transgene delivery without inducing microglia immune activation. These AAV variants should facilitate the use of various genetically encoded sensors and effectors in the study of microglia-related biology. Recombinant adeno-associated virus tools for enhanced microglial transduction in mice are reported. These viruses can be used to express functional reporters or genome editing tools with high microglial specificity, with the help of microglia-specific Cre lines.},
  archive      = {J_NMETH},
  author       = {Lin, Rui and Zhou, Youtong and Yan, Ting and Wang, Ruiyu and Li, Heng and Wu, Zhaofa and Zhang, Xinshuang and Zhou, Xiangyu and Zhao, Fei and Zhang, Li and Li, Yulong and Luo, Minmin},
  doi          = {10.1038/s41592-022-01547-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {976-985},
  shortjournal = {Nat. Meth.},
  title        = {Directed evolution of adeno-associated virus for efficient gene delivery to microglia},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Real age prediction from the transcriptome with RAPToR.
<em>NMETH</em>, <em>19</em>(8), 969–975. (<a
href="https://doi.org/10.1038/s41592-022-01540-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptomic data is often affected by uncontrolled variation among samples that can obscure and confound the effects of interest. This variation is frequently due to unintended differences in developmental stages between samples. The transcriptome itself can be used to estimate developmental progression, but existing methods require many samples and do not estimate a specimen’s real age. Here we present real-age prediction from transcriptome staging on reference (RAPToR), a computational method that precisely estimates the real age of a sample from its transcriptome, exploiting existing time-series data as reference. RAPToR works with whole animal, dissected tissue and single-cell data for the most common animal models, humans and even for non-model organisms lacking reference data. We show that RAPToR can be used to remove age as a confounding factor and allow recovery of a signal of interest in differential expression analysis. RAPToR will be especially useful in large-scale single-organism profiling because it eliminates the need for accurate staging or synchronisation before profiling. Real age prediction from transcriptome staging on reference (RAPToR) precisely estimates the real age of a specimen on the basis of transcriptomic data. RAPToR is broadly applicable and can be used to remove age as a confounding variable.},
  archive      = {J_NMETH},
  author       = {Bulteau, Romain and Francesconi, Mirko},
  doi          = {10.1038/s41592-022-01540-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {969-975},
  shortjournal = {Nat. Meth.},
  title        = {Real age prediction from the transcriptome with RAPToR},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). PROBER identifies proteins associated with programmable
sequence-specific DNA in living cells. <em>NMETH</em>, <em>19</em>(8),
959–968. (<a href="https://doi.org/10.1038/s41592-022-01552-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA–protein interactions mediate physiologic gene regulation and may be altered by DNA variants linked to polygenic disease. To enhance the speed and signal-to-noise ratio (SNR) in the identification and quantification of proteins associated with specific DNA sequences in living cells, we developed proximal biotinylation by episomal recruitment (PROBER). PROBER uses high-copy episomes to amplify SNR, and proximity proteomics (BioID) to identify the transcription factors and additional gene regulators associated with short DNA sequences of interest. PROBER quantified both constitutive and inducible association of transcription factors and corresponding chromatin regulators to target DNA sequences and binding quantitative trait loci due to single-nucleotide variants. PROBER identified alterations in regulator associations due to cancer hotspot mutations in the hTERT promoter, indicating that these mutations increase promoter association with specific gene activators. PROBER provides an approach to rapidly identify proteins associated with specific DNA sequences and their variants in living cells. PROBER offers a method to identify proteins associated with the DNA sequence of interest and quantify differential binding caused by sequence variants in living cells.},
  archive      = {J_NMETH},
  author       = {Mondal, Smarajit and Ramanathan, Muthukumar and Miao, Weili and Meyers, Robin M. and Rao, Deepti and Lopez-Pajares, Vanessa and Siprashvili, Zurab and Reynolds, David L. and Porter, Douglas F. and Ferguson, Ian and Neela, Poornima and Zhao, Yang and Meservey, Lindsey M. and Guo, Margaret and Yang, Yen-Yu and Li, Lin and Wang, Yinsheng and Khavari, Paul A.},
  doi          = {10.1038/s41592-022-01552-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {959-968},
  shortjournal = {Nat. Meth.},
  title        = {PROBER identifies proteins associated with programmable sequence-specific DNA in living cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sprod for de-noising spatially resolved transcriptomics data
based on position and image information. <em>NMETH</em>, <em>19</em>(8),
950–958. (<a href="https://doi.org/10.1038/s41592-022-01560-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved transcriptomics (SRT) provide gene expression close to, or even superior to, single-cell resolution while retaining the physical locations of sequencing and often also providing matched pathology images. However, SRT expression data suffer from high noise levels, due to the shallow coverage in each sequencing unit and the extra experimental steps required to preserve the locations of sequencing. Fortunately, such noise can be removed by leveraging information from the physical locations of sequencing, and the tissue organization reflected in corresponding pathology images. In this work, we developed Sprod, based on latent graph learning of matched location and imaging data, to impute accurate SRT gene expression. We validated Sprod comprehensively and demonstrated its advantages over previous methods for removing drop-outs in single-cell RNA-sequencing data. We showed that, after imputation by Sprod, differential expression analyses, pathway enrichment and cell-to-cell interaction inferences are more accurate. Overall, we envision de-noising by Sprod to become a key first step towards empowering SRT technologies for biomedical discoveries. Sprod accurately denoises spatially resolved transcriptomics data and improves downstream analysis results.},
  archive      = {J_NMETH},
  author       = {Wang, Yunguan and Song, Bing and Wang, Shidan and Chen, Mingyi and Xie, Yang and Xiao, Guanghua and Wang, Li and Wang, Tao},
  doi          = {10.1038/s41592-022-01560-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {950-958},
  shortjournal = {Nat. Meth.},
  title        = {Sprod for de-noising spatially resolved transcriptomics data based on position and image information},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrative genome modeling platform reveals essentiality of
rare contact events in 3D genome organizations. <em>NMETH</em>,
<em>19</em>(8), 938–949. (<a
href="https://doi.org/10.1038/s41592-022-01527-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A multitude of sequencing-based and microscopy technologies provide the means to unravel the relationship between the three-dimensional organization of genomes and key regulatory processes of genome function. Here, we develop a multimodal data integration approach to produce populations of single-cell genome structures that are highly predictive for nuclear locations of genes and nuclear bodies, local chromatin compaction and spatial segregation of functionally related chromatin. We demonstrate that multimodal data integration can compensate for systematic errors in some of the data and can greatly increase accuracy and coverage of genome structure models. We also show that alternative combinations of different orthogonal data sources can converge to models with similar predictive power. Moreover, our study reveals the key contributions of low-frequency (‘rare’) interchromosomal contacts to accurately predicting the global nuclear architecture, including the positioning of genes and chromosomes. Overall, our results highlight the benefits of multimodal data integration for genome structure analysis, available through the Integrative Genome Modeling software package. The Integrative Genome Modeling platform is a tool for population-based three-dimensional genome structure modeling and analysis by integrating various experimental data sources.},
  archive      = {J_NMETH},
  author       = {Boninsegna, Lorenzo and Yildirim, Asli and Polles, Guido and Zhan, Yuxiang and Quinodoz, Sofia A. and Finn, Elizabeth H. and Guttman, Mitchell and Zhou, Xianghong Jasmine and Alber, Frank},
  doi          = {10.1038/s41592-022-01527-x},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {938-949},
  shortjournal = {Nat. Meth.},
  title        = {Integrative genome modeling platform reveals essentiality of rare contact events in 3D genome organizations},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Inferring biological age from the transcriptome with RAPToR.
<em>NMETH</em>, <em>19</em>(8), 936–937. (<a
href="https://doi.org/10.1038/s41592-022-01542-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RAPToR (real age prediction from transcriptome staging on reference) is a new, broadly applicable method that can precisely estimate the age of a sample from a reference transcriptome time series.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01542-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {936-937},
  shortjournal = {Nat. Meth.},
  title        = {Inferring biological age from the transcriptome with RAPToR},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reconstructing whole genome structures using a multi-modal
data integration approach. <em>NMETH</em>, <em>19</em>(8), 934–935. (<a
href="https://doi.org/10.1038/s41592-022-01528-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Integrative Genome Modeling (IGM) platform incorporates information from multiple, complementary experimental data sources to accurately simulate whole diploid genome structures. We show that such structures have high predictive power and give access to a large variety of structural observables for the characterization of the gene microenvironment.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01528-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {934-935},
  shortjournal = {Nat. Meth.},
  title        = {Reconstructing whole genome structures using a multi-modal data integration approach},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fluorophores’ talk turns them dark. <em>NMETH</em>,
<em>19</em>(8), 932–933. (<a
href="https://doi.org/10.1038/s41592-022-01565-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dipole–dipole crosstalk between fluorophores separated by a distance of less than 10 nm induces changes in their photophysics, which adds a challenge to localization microscopy in the sub-10-nm regime.},
  archive      = {J_NMETH},
  author       = {Almahayni, Karim and Spiekermann, Malte and Möckl, Leonhard},
  doi          = {10.1038/s41592-022-01565-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {932-933},
  shortjournal = {Nat. Meth.},
  title        = {Fluorophores’ talk turns them dark},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022w). Trawling the ocean virome. <em>NMETH</em>, <em>19</em>(8),
928–931. (<a href="https://doi.org/10.1038/s41592-022-01568-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Even without a stint on an ocean-faring vessel, scientists can trawl through data to explore marine viruses and address new puzzles and cultural shifts.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01568-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {928-931},
  shortjournal = {Nat. Meth.},
  title        = {Trawling the ocean virome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022aa). Why the ocean virome matters. <em>NMETH</em>,
<em>19</em>(8), 924–927. (<a
href="https://doi.org/10.1038/s41592-022-01567-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expeditions are delivering data wealth about our planet’s oceans, including its microbes. Some labs are now diving deep into the ocean virome.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01567-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {924-927},
  shortjournal = {Nat. Meth.},
  title        = {Why the ocean virome matters},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Light-gated potassium channels from nature. <em>NMETH</em>,
<em>19</em>(8), 923. (<a
href="https://doi.org/10.1038/s41592-022-01582-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers have discovered two naturally occurring channelrhodopsins for potassium ion transport that can be used in optogenetic applications.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01582-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {923},
  shortjournal = {Nat. Meth.},
  title        = {Light-gated potassium channels from nature},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022u). Towards a full picture of the total transcriptome.
<em>NMETH</em>, <em>19</em>(8), 922. (<a
href="https://doi.org/10.1038/s41592-022-01581-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {VASA-seq offers a single cell sequencing tool for capturing full-length coverage of coding sequences and supplementing non-coding RNA molecules.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01581-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {922},
  shortjournal = {Nat. Meth.},
  title        = {Towards a full picture of the total transcriptome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Vascular subtypes on a dish. <em>NMETH</em>,
<em>19</em>(8), 921. (<a
href="https://doi.org/10.1038/s41592-022-01580-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stem cell-derived endothelial cell subtypes enable study of viral tropism.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01580-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {921},
  shortjournal = {Nat. Meth.},
  title        = {Vascular subtypes on a dish},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Dissecting genetic effects. <em>NMETH</em>, <em>19</em>(8),
920. (<a href="https://doi.org/10.1038/s41592-022-01579-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imputing missing parental genotypes sharpens estimates of direct genetic effects.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01579-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {920},
  shortjournal = {Nat. Meth.},
  title        = {Dissecting genetic effects},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Why it is important to study human–monkey embryonic chimeras
in a dish. <em>NMETH</em>, <em>19</em>(8), 914–919. (<a
href="https://doi.org/10.1038/s41592-022-01571-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study of human–animal chimeras is fraught with technical and ethical challenges. In this Comment, we discuss the importance and future of human–monkey chimera research within the context of current scientific and regulatory obstacles.},
  archive      = {J_NMETH},
  author       = {De Los Angeles, Alejandro and Regenberg, Alan and Mascetti, Victoria and Benvenisty, Nissim and Church, George and Deng, Hongkui and Izpisua Belmonte, Juan Carlos and Ji, Weizhi and Koplin, Julian and Loh, Yuin-Han and Niu, Yuyu and Pei, Duanqing and Pera, Martin and Pho, Nam and Pinzon-Arteaga, Carlos and Saitou, Mitinori and Silva, Jose C. R. and Tao, Tan and Trounson, Alan and Warrier, Tushar and Zambidis, Elias T.},
  doi          = {10.1038/s41592-022-01571-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {914-919},
  shortjournal = {Nat. Meth.},
  title        = {Why it is important to study human–monkey embryonic chimeras in a dish},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Nucleome browser: An integrative and multimodal data
navigation platform for 4D nucleome. <em>NMETH</em>, <em>19</em>(8),
911–913. (<a href="https://doi.org/10.1038/s41592-022-01559-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zhu, Xiaopeng and Zhang, Yang and Wang, Yuchuan and Tian, Dechao and Belmont, Andrew S. and Swedlow, Jason R. and Ma, Jian},
  doi          = {10.1038/s41592-022-01559-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {911-913},
  shortjournal = {Nat. Meth.},
  title        = {Nucleome browser: An integrative and multimodal data navigation platform for 4D nucleome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Exploring genomic data coupled with 3D chromatin structures
using the WashU epigenome browser. <em>NMETH</em>, <em>19</em>(8),
909–910. (<a href="https://doi.org/10.1038/s41592-022-01550-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Li, Daofeng and Harrison, Jessica K. and Purushotham, Deepak and Wang, Ting},
  doi          = {10.1038/s41592-022-01550-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {909-910},
  shortjournal = {Nat. Meth.},
  title        = {Exploring genomic data coupled with 3D chromatin structures using the WashU epigenome browser},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Survival analysis—time-to-event data and censoring.
<em>NMETH</em>, <em>19</em>(8), 906–908. (<a
href="https://doi.org/10.1038/s41592-022-01563-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Love’s the only engine of survival. —Leonard Cohen},
  archive      = {J_NMETH},
  author       = {Dey, Tanujit and Lipsitz, Stuart R. and Cooper, Zara and Trinh, Quoc-Dien and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-022-01563-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {906-908},
  shortjournal = {Nat. Meth.},
  title        = {Survival analysis—time-to-event data and censoring},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Tardigrades. <em>NMETH</em>, <em>19</em>(8), 904–905. (<a
href="https://doi.org/10.1038/s41592-022-01573-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tardigrades are everywhere. They’re tiny — usually under a millimeter long — and they’re mostly transparent, so they’re easy to miss. But you probably walk by them every day. We’ve been grooming them as emerging models for studying how body forms evolve and how biological materials can survive extreme conditions.},
  archive      = {J_NMETH},
  author       = {Goldstein, Bob},
  doi          = {10.1038/s41592-022-01573-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {904-905},
  shortjournal = {Nat. Meth.},
  title        = {Tardigrades},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022y). When labs welcome under-represented groups. <em>NMETH</em>,
<em>19</em>(8), 903. (<a
href="https://doi.org/10.1038/s41592-022-01564-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To diversify science, some labs open summer doors wide to reach out to under-represented groups.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01564-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {903},
  shortjournal = {Nat. Meth.},
  title        = {When labs welcome under-represented groups},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Creatures galore. <em>NMETH</em>, <em>19</em>(8), 901. (<a
href="https://doi.org/10.1038/s41592-022-01590-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue, we are delighted to launch a new regular feature: the Creature Column.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01590-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {901},
  shortjournal = {Nat. Meth.},
  title        = {Creatures galore},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: A complete, telomere-to-telomere
human genome sequence presents new opportunities for evolutionary
genomics. <em>NMETH</em>, <em>19</em>(7), 899. (<a
href="https://doi.org/10.1038/s41592-022-01551-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mao, Yafei and Zhang, Guojie},
  doi          = {10.1038/s41592-022-01551-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {899},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: A complete, telomere-to-telomere human genome sequence presents new opportunities for evolutionary genomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Augmenting and directing long-range
CRISPR-mediated activation in human cells. <em>NMETH</em>,
<em>19</em>(7), 899. (<a
href="https://doi.org/10.1038/s41592-022-01561-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tak, Y. Esther and Horng, Joy E. and Perry, Nicholas T. and Schultz, Hayley T. and Iyer, Sowmya and Yao, Qiuming and Zou, Luli S. and Aryee, Martin J. and Pinello, Luca and Joung, J. Keith},
  doi          = {10.1038/s41592-022-01561-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {899},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Augmenting and directing long-range CRISPR-mediated activation in human cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multiplexed bioluminescence microscopy via phasor analysis.
<em>NMETH</em>, <em>19</em>(7), 893–898. (<a
href="https://doi.org/10.1038/s41592-022-01529-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioluminescence imaging with luciferase–luciferin pairs is a well-established technique for visualizing biological processes across tissues and whole organisms. Applications at the microscale, by contrast, have been hindered by a lack of detection platforms and easily resolved probes. We addressed this limitation by combining bioluminescence with phasor analysis, a method commonly used to distinguish spectrally similar fluorophores. We built a camera-based microscope equipped with special optical filters to directly assign phasor locations to unique luciferase–luciferin pairs. Six bioluminescent reporters were easily resolved in live cells, and the readouts were quantitative and instantaneous. Multiplexed imaging was also performed over extended time periods. Bioluminescent phasor further provided direct measures of resonance energy transfer in single cells, setting the stage for dynamic measures of cellular and molecular features. The merger of bioluminescence with phasor analysis fills a long-standing void in imaging capabilities, and will bolster future efforts to visualize biological events in real time and over multiple length scales. The combination of engineered probes and spectral phasor analysis overcomes long-standing challenges associated with bioluminescence detection at the microscale, enabling multiplexed, real-time imaging of cellular features without the need for excitation light.},
  archive      = {J_NMETH},
  author       = {Yao, Zi and Brennan, Caroline K. and Scipioni, Lorenzo and Chen, Hongtao and Ng, Kevin K. and Tedeschi, Giulia and Parag-Sharma, Kshitij and Amelio, Antonio L. and Gratton, Enrico and Digman, Michelle A. and Prescher, Jennifer A.},
  doi          = {10.1038/s41592-022-01529-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {893-898},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed bioluminescence microscopy via phasor analysis},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automated high-speed 3D imaging of organoid cultures with
multi-scale phenotypic quantification. <em>NMETH</em>, <em>19</em>(7),
881–892. (<a href="https://doi.org/10.1038/s41592-022-01508-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current imaging approaches limit the ability to perform multi-scale characterization of three-dimensional (3D) organotypic cultures (organoids) in large numbers. Here, we present an automated multi-scale 3D imaging platform synergizing high-density organoid cultures with rapid and live 3D single-objective light-sheet imaging. It is composed of disposable microfabricated organoid culture chips, termed JeWells, with embedded optical components and a laser beam-steering unit coupled to a commercial inverted microscope. It permits streamlining organoid culture and high-content 3D imaging on a single user-friendly instrument with minimal manipulations and a throughput of 300 organoids per hour. We demonstrate that the large number of 3D stacks that can be collected via our platform allows training deep learning-based algorithms to quantify morphogenetic organizations of organoids at multi-scales, ranging from the subcellular scale to the whole organoid level. We validated the versatility and robustness of our approach on intestine, hepatic, neuroectoderm organoids and oncospheres. A method for high-content 3D imaging of organoids.},
  archive      = {J_NMETH},
  author       = {Beghin, Anne and Grenci, Gianluca and Sahni, Geetika and Guo, Su and Rajendiran, Harini and Delaire, Tom and Mohamad Raffi, Saburnisha Binte and Blanc, Damien and de Mets, Richard and Ong, Hui Ting and Galindo, Xareni and Monet, Anais and Acharya, Vidhyalakshmi and Racine, Victor and Levet, Florian and Galland, Remi and Sibarita, Jean-Baptiste and Viasnoff, Virgile},
  doi          = {10.1038/s41592-022-01508-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {881-892},
  shortjournal = {Nat. Meth.},
  title        = {Automated high-speed 3D imaging of organoid cultures with multi-scale phenotypic quantification},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A general approach for engineering RTKs optically controlled
with far-red light. <em>NMETH</em>, <em>19</em>(7), 871–880. (<a
href="https://doi.org/10.1038/s41592-022-01517-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Regulation of receptor tyrosine kinase (RTK) activity is necessary for studying cell signaling pathways in health and disease. We developed a generalized approach for engineering RTKs optically controlled with far-red light. We targeted the bacterial phytochrome DrBphP to the cell surface and allowed its light-induced conformational changes to be transmitted across the plasma membrane via transmembrane helices to intracellular RTK domains. Systematic optimization of these constructs has resulted in optically regulated epidermal growth factor receptor, HER2, TrkA, TrkB, FGFR1, IR1, cKIT and cMet, named eDrRTKs. eDrRTKs induced downstream signaling in mammalian cells in tens of seconds. The ability to activate eDrRTKs with far-red light enabled spectral multiplexing with fluorescent probes operating in a shorter spectral range, allowing for all-optical assays. We validated eDrTrkB performance in mice and found that minimally invasive stimulation in the neocortex with penetrating via skull far-red light-induced neural activity, early immediate gene expression and affected sleep patterns. A suite of eight optogenetic receptor tyrosine kinases controlled by conformational changes in bacterial phytochrome induced by far-red light is introduced. These optogenetic tools perform well in cells and in the brain of behaving mice.},
  archive      = {J_NMETH},
  author       = {Leopold, Anna V. and Thankachan, Stephen and Yang, Chun and Gerashchenko, Dmitry and Verkhusha, Vladislav V.},
  doi          = {10.1038/s41592-022-01517-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {871-880},
  shortjournal = {Nat. Meth.},
  title        = {A general approach for engineering RTKs optically controlled with far-red light},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). MSNovelist: De novo structure generation from mass spectra.
<em>NMETH</em>, <em>19</em>(7), 865–870. (<a
href="https://doi.org/10.1038/s41592-022-01486-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for structure elucidation of small molecules rely on finding similarity with spectra of known compounds, but do not predict structures de novo for unknown compound classes. We present MSNovelist, which combines fingerprint prediction with an encoder–decoder neural network to generate structures de novo solely from tandem mass spectrometry (MS2) spectra. In an evaluation with 3,863 MS2 spectra from the Global Natural Product Social Molecular Networking site, MSNovelist predicted 25\% of structures correctly on first rank, retrieved 45\% of structures overall and reproduced 61\% of correct database annotations, without having ever seen the structure in the training phase. Similarly, for the CASMI 2016 challenge, MSNovelist correctly predicted 26\% and retrieved 57\% of structures, recovering 64\% of correct database annotations. Finally, we illustrate the application of MSNovelist in a bryophyte MS2 dataset, in which de novo structure prediction substantially outscored the best database candidate for seven spectra. MSNovelist is ideally suited to complement library-based annotation in the case of poorly represented analyte classes and novel compounds. MSNovelist combines fingerprint prediction with an encoder–decoder neural network for de novo structure generation of small molecules from mass spectra.},
  archive      = {J_NMETH},
  author       = {Stravs, Michael A. and Dührkop, Kai and Böcker, Sebastian and Zamboni, Nicola},
  doi          = {10.1038/s41592-022-01486-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {865-870},
  shortjournal = {Nat. Meth.},
  title        = {MSNovelist: De novo structure generation from mass spectra},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cyclic immonium ion of lactyllysine reveals widespread
lactylation in the human proteome. <em>NMETH</em>, <em>19</em>(7),
854–864. (<a href="https://doi.org/10.1038/s41592-022-01523-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lactylation was initially discovered on human histones. Given its nascence, its occurrence on nonhistone proteins and downstream functional consequences remain elusive. Here we report a cyclic immonium ion of lactyllysine formed during tandem mass spectrometry that enables confident protein lactylation assignment. We validated the sensitivity and specificity of this ion for lactylation through affinity-enriched lactylproteome analysis and large-scale informatic assessment of nonlactylated spectral libraries. With this diagnostic ion-based strategy, we confidently determined new lactylation, unveiling a wide landscape beyond histones from not only the enriched lactylproteome but also existing unenriched human proteome resources. Specifically, by mining the public human Meltome Atlas, we found that lactylation is common on glycolytic enzymes and conserved on ALDOA. We also discovered prevalent lactylation on DHRS7 in the draft of the human tissue proteome. We partially demonstrated the functional importance of lactylation: site-specific engineering of lactylation into ALDOA caused enzyme inhibition, suggesting a lactylation-dependent feedback loop in glycolysis. This article reports the cyclic immonium ion as a diagnostic fragment ion for lysine lactylation. The approach was used for identifying lactylation in various enriched and unenriched proteome databases, demonstrating prevalence of lactylation beyond histones.},
  archive      = {J_NMETH},
  author       = {Wan, Ning and Wang, Nian and Yu, Siqin and Zhang, Hanqing and Tang, Shuo and Wang, Dexiang and Lu, Wenjie and Li, Huanhuan and Delafield, Daniel G. and Kong, Ying and Wang, Xinmiao and Shao, Chang and Lv, Langlang and Wang, Guangji and Tan, Renxiang and Wang, Nanxi and Hao, Haiping and Ye, Hui},
  doi          = {10.1038/s41592-022-01523-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {854-864},
  shortjournal = {Nat. Meth.},
  title        = {Cyclic immonium ion of lactyllysine reveals widespread lactylation in the human proteome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Emu: Species-level microbial community profiling of
full-length 16S rRNA oxford nanopore sequencing data. <em>NMETH</em>,
<em>19</em>(7), 845–853. (<a
href="https://doi.org/10.1038/s41592-022-01520-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {16S ribosomal RNA-based analysis is the established standard for elucidating the composition of microbial communities. While short-read 16S rRNA analyses are largely confined to genus-level resolution at best, given that only a portion of the gene is sequenced, full-length 16S rRNA gene amplicon sequences have the potential to provide species-level accuracy. However, existing taxonomic identification algorithms are not optimized for the increased read length and error rate often observed in long-read data. Here we present Emu, an approach that uses an expectation–maximization algorithm to generate taxonomic abundance profiles from full-length 16S rRNA reads. Results produced from simulated datasets and mock communities show that Emu is capable of accurate microbial community profiling while obtaining fewer false positives and false negatives than alternative methods. Additionally, we illustrate a real-world application of Emu by comparing clinical sample composition estimates generated by an established whole-genome shotgun sequencing workflow with those returned by full-length 16S rRNA gene sequences processed with Emu. Emu accurately estimates microbial abundance using full-length Nanopore 16S rRNA gene sequencing data.},
  archive      = {J_NMETH},
  author       = {Curry, Kristen D. and Wang, Qi and Nute, Michael G. and Tyshaieva, Alona and Reeves, Elizabeth and Soriano, Sirena and Wu, Qinglong and Graeber, Enid and Finzer, Patrick and Mendling, Werner and Savidge, Tor and Villapol, Sonia and Dilthey, Alexander and Treangen, Todd J.},
  doi          = {10.1038/s41592-022-01520-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {845-853},
  shortjournal = {Nat. Meth.},
  title        = {Emu: Species-level microbial community profiling of full-length 16S rRNA oxford nanopore sequencing data},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Direct identification of a-to-i editing sites with nanopore
native RNA sequencing. <em>NMETH</em>, <em>19</em>(7), 833–844. (<a
href="https://doi.org/10.1038/s41592-022-01513-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inosine is a prevalent RNA modification in animals and is formed when an adenosine is deaminated by the ADAR family of enzymes. Traditionally, inosines are identified indirectly as variants from Illumina RNA-sequencing data because they are interpreted as guanosines by cellular machineries. However, this indirect method performs poorly in protein-coding regions where exons are typically short, in non-model organisms with sparsely annotated single-nucleotide polymorphisms, or in disease contexts where unknown DNA mutations are pervasive. Here, we show that Oxford Nanopore direct RNA sequencing can be used to identify inosine-containing sites in native transcriptomes with high accuracy. We trained convolutional neural network models to distinguish inosine from adenosine and guanosine, and to estimate the modification rate at each editing site. Furthermore, we demonstrated their utility on the transcriptomes of human, mouse and Xenopus. Our approach expands the toolkit for studying adenosine-to-inosine editing and can be further extended to investigate other RNA modifications. This work combines nanopore native RNA sequencing with machine learning models for identifying inosine-containing sites in transcriptomes.},
  archive      = {J_NMETH},
  author       = {Nguyen, Tram Anh and Heng, Jia Wei Joel and Kaewsapsak, Pornchai and Kok, Eng Piew Louis and Stanojević, Dominik and Liu, Hao and Cardilla, Angelysia and Praditya, Albert and Yi, Zirong and Lin, Mingwan and Aw, Jong Ghut Ashley and Ho, Yin Ying and Peh, Kai Lay Esther and Wang, Yuanming and Zhong, Qixing and Heraud-Farlow, Jacki and Xue, Shifeng and Reversade, Bruno and Walkley, Carl and Ho, Ying Swan and Šikić, Mile and Wan, Yue and Tan, Meng How},
  doi          = {10.1038/s41592-022-01513-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {833-844},
  shortjournal = {Nat. Meth.},
  title        = {Direct identification of A-to-I editing sites with nanopore native RNA sequencing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). TrackMate 7: Integrating state-of-the-art segmentation
algorithms into tracking pipelines. <em>NMETH</em>, <em>19</em>(7),
829–832. (<a href="https://doi.org/10.1038/s41592-022-01507-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {TrackMate is an automated tracking software used to analyze bioimages and is distributed as a Fiji plugin. Here, we introduce a new version of TrackMate. TrackMate 7 is built to address the broad spectrum of modern challenges researchers face by integrating state-of-the-art segmentation algorithms into tracking pipelines. We illustrate qualitatively and quantitatively that these new capabilities function effectively across a wide range of bio-imaging experiments. TrackMate 7 combines the benefits of machine and deep learning-based image segmentation with accurate object tracking to enable improved 2D and 3D tracking of diverse objects in biological research.},
  archive      = {J_NMETH},
  author       = {Ershov, Dmitry and Phan, Minh-Son and Pylvänäinen, Joanna W. and Rigaud, Stéphane U. and Le Blanc, Laure and Charles-Orszag, Arthur and Conway, James R. W. and Laine, Romain F. and Roy, Nathan H. and Bonazzi, Daria and Duménil, Guillaume and Jacquemet, Guillaume and Tinevez, Jean-Yves},
  doi          = {10.1038/s41592-022-01507-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {829-832},
  shortjournal = {Nat. Meth.},
  title        = {TrackMate 7: Integrating state-of-the-art segmentation algorithms into tracking pipelines},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Histidine phosphorylation in human cells; a needle or
phantom in the haystack? <em>NMETH</em>, <em>19</em>(7), 827–828. (<a
href="https://doi.org/10.1038/s41592-022-01524-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It has been suggested that in mammalian cells histidine residues in proteins may become as frequently phosphorylated as serine, threonine and tyrosine, and may play a key role in mammalian signaling. Here we applied a robust workflow that earlier allowed us to detect histidine phosphorylation in bacteria unambiguously, to probe for histidine phosphorylation in four human cell lines. Initially, seemingly hundreds of protein histidine phosphorylations were picked up in all studied human cell lines. However, careful examination of the data, and several control experiments, led us to the conclusion that &amp;gt;99\% of these initially assigned pHis sites were not genuine, and should be site localized to neighboring Ser/Thr residues. Nevertheless, our methods are selective enough to detect just a handful of genuine pHis sites in mammalian cells, representing well-known enzymatic intermediates. Consequently, we do not find any evidence in our data supporting that protein histidine phosphorylation plays a role in mammalian signaling. Extensive analyses of mammalian phosphoproteomics datasets show that protein histidine phosphorylation in human cells may not be as prevalent as previously thought.},
  archive      = {J_NMETH},
  author       = {Leijten, Niels M. and Heck, Albert J. R. and Lemeer, Simone},
  doi          = {10.1038/s41592-022-01524-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {827-828},
  shortjournal = {Nat. Meth.},
  title        = {Histidine phosphorylation in human cells; a needle or phantom in the haystack?},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Oxford nanopore r10.4 long-read sequencing enables the
generation of near-finished bacterial genomes from pure cultures and
metagenomes without short-read or reference polishing. <em>NMETH</em>,
<em>19</em>(7), 823–826. (<a
href="https://doi.org/10.1038/s41592-022-01539-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read Oxford Nanopore sequencing has democratized microbial genome sequencing and enables the recovery of highly contiguous microbial genomes from isolates or metagenomes. However, to obtain near-finished genomes it has been necessary to include short-read polishing to correct insertions and deletions derived from homopolymer regions. Here, we show that Oxford Nanopore R10.4 can be used to generate near-finished microbial genomes from isolates or metagenomes without short-read or reference polishing. This study demonstrates the feasibility of generating near-finished microbial genomes using only Oxford Nanopore R10.4 data from pure cultures or metagenomes.},
  archive      = {J_NMETH},
  author       = {Sereika, Mantas and Kirkegaard, Rasmus Hansen and Karst, Søren Michael and Michaelsen, Thomas Yssing and Sørensen, Emil Aarre and Wollenberg, Rasmus Dam and Albertsen, Mads},
  doi          = {10.1038/s41592-022-01539-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {823-826},
  shortjournal = {Nat. Meth.},
  title        = {Oxford nanopore r10.4 long-read sequencing enables the generation of near-finished bacterial genomes from pure cultures and metagenomes without short-read or reference polishing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ranking reprogramming factors for cell differentiation.
<em>NMETH</em>, <em>19</em>(7), 812–822. (<a
href="https://doi.org/10.1038/s41592-022-01522-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factor over-expression is a proven method for reprogramming cells to a desired cell type for regenerative medicine and therapeutic discovery. However, a general method for the identification of reprogramming factors to create an arbitrary cell type is an open problem. Here we examine the success rate of methods and data for differentiation by testing the ability of nine computational methods (CellNet, GarNet, EBseq, AME, DREME, HOMER, KMAC, diffTF and DeepAccess) to discover and rank candidate factors for eight target cell types with known reprogramming solutions. We compare methods that use gene expression, biological networks and chromatin accessibility data, and comprehensively test parameter and preprocessing of input data to optimize performance. We find the best factor identification methods can identify an average of 50–60\% of reprogramming factors within the top ten candidates, and methods that use chromatin accessibility perform the best. Among the chromatin accessibility methods, complex methods DeepAccess and diffTF have higher correlation with the ranked significance of transcription factor candidates within reprogramming protocols for differentiation. We provide evidence that AME and diffTF are optimal methods for transcription factor recovery that will allow for systematic prioritization of transcription factor candidates to aid in the design of new reprogramming protocols. A comparison of nine computational methods for identification of reprogramming factors for cell differentiation.},
  archive      = {J_NMETH},
  author       = {Hammelman, Jennifer and Patel, Tulsi and Closser, Michael and Wichterle, Hynek and Gifford, David},
  doi          = {10.1038/s41592-022-01522-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {812-822},
  shortjournal = {Nat. Meth.},
  title        = {Ranking reprogramming factors for cell differentiation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mass spectrometry-based draft of the mouse proteome.
<em>NMETH</em>, <em>19</em>(7), 803–811. (<a
href="https://doi.org/10.1038/s41592-022-01526-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The laboratory mouse ranks among the most important experimental systems for biomedical research and molecular reference maps of such models are essential informational tools. Here, we present a quantitative draft of the mouse proteome and phosphoproteome constructed from 41 healthy tissues and several lines of analyses exemplify which insights can be gleaned from the data. For instance, tissue- and cell-type resolved profiles provide protein evidence for the expression of 17,000 genes, thousands of isoforms and 50,000 phosphorylation sites in vivo. Proteogenomic comparison of mouse, human and Arabidopsis reveal common and distinct mechanisms of gene expression regulation and, despite many similarities, numerous differentially abundant orthologs that likely serve species-specific functions. We leverage the mouse proteome by integrating phenotypic drug (n &amp;gt; 400) and radiation response data with the proteomes of 66 pancreatic ductal adenocarcinoma (PDAC) cell lines to reveal molecular markers for sensitivity and resistance. This unique atlas complements other molecular resources for the mouse and can be explored online via ProteomicsDB and PACiFIC. This work presents a quantitative draft of the mouse proteome and phosphoproteome constructed from 41 healthy tissues covering 15 major anatomical systems and 66 cell lines.},
  archive      = {J_NMETH},
  author       = {Giansanti, Piero and Samaras, Patroklos and Bian, Yangyang and Meng, Chen and Coluccio, Andrea and Frejno, Martin and Jakubowsky, Hannah and Dobiasch, Sophie and Hazarika, Rashmi R. and Rechenberger, Julia and Calzada-Wack, Julia and Krumm, Johannes and Mueller, Sebastian and Lee, Chien-Yun and Wimberger, Nicole and Lautenbacher, Ludwig and Hassan, Zonera and Chang, Yun-Chien and Falcomatà, Chiara and Bayer, Florian P. and Bärthel, Stefanie and Schmidt, Tobias and Rad, Roland and Combs, Stephanie E. and The, Matthew and Johannes, Frank and Saur, Dieter and de Angelis, Martin Hrabe and Wilhelm, Mathias and Schneider, Günter and Kuster, Bernhard},
  doi          = {10.1038/s41592-022-01526-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {803-811},
  shortjournal = {Nat. Meth.},
  title        = {Mass spectrometry-based draft of the mouse proteome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spectral phasor analysis enables multiplexed microscopy with
bioluminescent probes. <em>NMETH</em>, <em>19</em>(7), 801–802. (<a
href="https://doi.org/10.1038/s41592-022-01530-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioluminescent phasor is a new technology for multiplexed, excitation-free imaging at the microscale using luciferase–luciferin pairs. This platform can readily unmix the broad, overlapping emission spectra of bioluminescent reporters, making possible the dynamic tracking of cellular and molecular features over prolonged time periods.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01530-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {801-802},
  shortjournal = {Nat. Meth.},
  title        = {Spectral phasor analysis enables multiplexed microscopy with bioluminescent probes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). All-in-one approach to culture, image and quantify large
numbers of organoids at all scales. <em>NMETH</em>, <em>19</em>(7),
799–800. (<a href="https://doi.org/10.1038/s41592-022-01509-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a streamlined approach coupling microfabricated cell culture substrates, 3D single-objective light sheet imaging and artificial intelligence quantifications to characterize the variability of morphologies of small organoids. Arrayed organoids can be imaged in 3D at around 100 organoids per hour.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01509-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {799-800},
  shortjournal = {Nat. Meth.},
  title        = {All-in-one approach to culture, image and quantify large numbers of organoids at all scales},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep learning identifies a-to-i RNA edits using nanopore
sequencing data. <em>NMETH</em>, <em>19</em>(7), 797–798. (<a
href="https://doi.org/10.1038/s41592-022-01514-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adenosine-to-inosine RNA editing is a common post-transcriptional modification, but can be challenging to identify correctly from Illumina data. We show that Oxford Nanopore RNA sequencing, combined with deep learning models, can be used to accurately detect inosine-containing sites in native transcriptomes and to estimate the modification rate of each site.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01514-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {797-798},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning identifies A-to-I RNA edits using nanopore sequencing data},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Some assembly required. <em>NMETH</em>, <em>19</em>(7),
795–796. (<a href="https://doi.org/10.1038/s41592-022-01521-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo structure generation from tandem mass spectrometry data will advance annotation of unknown metabolomic signals.},
  archive      = {J_NMETH},
  author       = {Broeckling, Corey D.},
  doi          = {10.1038/s41592-022-01521-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {795-796},
  shortjournal = {Nat. Meth.},
  title        = {Some assembly required},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Uncovering ubiquitous protein lactylation. <em>NMETH</em>,
<em>19</em>(7), 793–794. (<a
href="https://doi.org/10.1038/s41592-022-01536-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A diagnostic fragment ion in tandem mass spectrometry enables confident protein lactylation assignment and the discovery of broad lysine modification beyond histones.},
  archive      = {J_NMETH},
  author       = {Wu, Xiaofeng and Tao, W. Andy},
  doi          = {10.1038/s41592-022-01536-w},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {793-794},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering ubiquitous protein lactylation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The adult mouse proteome. <em>NMETH</em>, <em>19</em>(7),
792–793. (<a href="https://doi.org/10.1038/s41592-022-01546-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evidence for at least one protein product from 80\% of all mouse genes is reported in a comprehensive proteomic analysis of 41 adult mouse tissues. Comparison of tissue profiles between mouse and human suggests that the fundamental biology of this important model organism is even more different from our own than we thought.},
  archive      = {J_NMETH},
  author       = {Foster, Leonard J.},
  doi          = {10.1038/s41592-022-01546-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {792-793},
  shortjournal = {Nat. Meth.},
  title        = {The adult mouse proteome},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022v). The making of microscope camera standards. <em>NMETH</em>,
<em>19</em>(7), 788–791. (<a
href="https://doi.org/10.1038/s41592-022-01545-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cameras are a crucial part of microscopes and are also built into many kinds of instruments. To make their output comparable takes standards.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01545-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {788-791},
  shortjournal = {Nat. Meth.},
  title        = {The making of microscope camera standards},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Imaging standards to ease reproducibility and the everyday.
<em>NMETH</em>, <em>19</em>(7), 784–788. (<a
href="https://doi.org/10.1038/s41592-022-01544-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging and microscopy technology advances in leaps and bounds. To address accumulated pain points, academics and companies are making headway on standards.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01544-w},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {784-788},
  shortjournal = {Nat. Meth.},
  title        = {Imaging standards to ease reproducibility and the everyday},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). Predicting and designing enhancers. <em>NMETH</em>,
<em>19</em>(7), 783. (<a
href="https://doi.org/10.1038/s41592-022-01557-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning decodes the hidden rules of enhancers.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01557-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {783},
  shortjournal = {Nat. Meth.},
  title        = {Predicting and designing enhancers},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022n). Recording protein–DNA interactions in bacteria.
<em>NMETH</em>, <em>19</em>(7), 782. (<a
href="https://doi.org/10.1038/s41592-022-01556-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers repurpose cytosine deaminase toxin DddA to measure DNA–protein interaction sites in bacteria.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01556-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {782},
  shortjournal = {Nat. Meth.},
  title        = {Recording protein–DNA interactions in bacteria},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). FlipFlop of the t cell immune system. <em>NMETH</em>,
<em>19</em>(7), 781. (<a
href="https://doi.org/10.1038/s41592-022-01555-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {FlipFlop mice harbor functionally reversed T cells due to a switch in CD4 and CD8 expression.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01555-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {781},
  shortjournal = {Nat. Meth.},
  title        = {FlipFlop of the t cell immune system},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Spatial proteomics with subcellular resolution.
<em>NMETH</em>, <em>19</em>(7), 780. (<a
href="https://doi.org/10.1038/s41592-022-01554-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep Visual Proteomics combines the power of deep-learning-based image analysis with microdissection and ultrasensitive mass spectrometry to provide insights into the spatial proteome.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01554-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {780},
  shortjournal = {Nat. Meth.},
  title        = {Spatial proteomics with subcellular resolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Understudied proteins: Opportunities and challenges for
functional proteomics. <em>NMETH</em>, <em>19</em>(7), 774–779. (<a
href="https://doi.org/10.1038/s41592-022-01454-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most research aiming at understanding the molecular foundations of life and disease has focused on a limited set of increasingly well-known proteins while the biological functions of many others remain poorly understood. We propose to form the Understudied Protein Initiative with the objective of reducing the annotation gap by systematically associating uncharacterized proteins with proteins of known function, thereby laying the groundwork for future detailed mechanistic studies.},
  archive      = {J_NMETH},
  author       = {Kustatscher, Georg and Collins, Tom and Gingras, Anne-Claude and Guo, Tiannan and Hermjakob, Henning and Ideker, Trey and Lilley, Kathryn S. and Lundberg, Emma and Marcotte, Edward M. and Ralser, Markus and Rappsilber, Juri},
  doi          = {10.1038/s41592-022-01454-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {774-779},
  shortjournal = {Nat. Meth.},
  title        = {Understudied proteins: Opportunities and challenges for functional proteomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022k). Lab retreats for stepping away and moving closer.
<em>NMETH</em>, <em>19</em>(7), 773. (<a
href="https://doi.org/10.1038/s41592-022-01543-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A blend of science, food and fun can empower well-being and new connections.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01543-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {773},
  shortjournal = {Nat. Meth.},
  title        = {Lab retreats for stepping away and moving closer},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). What makes a nature methods paper. <em>NMETH</em>,
<em>19</em>(7), 771–772. (<a
href="https://doi.org/10.1038/s41592-022-01558-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We explain what our editorial team looks for when considering a methods paper for publication.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01558-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {771-772},
  shortjournal = {Nat. Meth.},
  title        = {What makes a nature methods paper},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Automated segmentation and tracking of
mitochondria in live-cell time-lapse images. <em>NMETH</em>,
<em>19</em>(6), 770. (<a
href="https://doi.org/10.1038/s41592-022-01506-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Lefebvre, Austin E. Y. T. and Ma, Dennis and Kessenbrock, Kai and Lawson, Devon A. and Digman, Michelle A.},
  doi          = {10.1038/s41592-022-01506-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {770},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Automated segmentation and tracking of mitochondria in live-cell time-lapse images},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Searching thousands of genomes to
classify somatic and novel structural variants using STIX.
<em>NMETH</em>, <em>19</em>(6), 770. (<a
href="https://doi.org/10.1038/s41592-022-01538-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Chowdhury, Murad and Pedersen, Brent S. and Sedlazeck, Fritz J. and Quinlan, Aaron R. and Layer, Ryan M.},
  doi          = {10.1038/s41592-022-01538-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {770},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Searching thousands of genomes to classify somatic and novel structural variants using STIX},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identification of cell types in multiplexed in situ images
by combining protein expression and spatial information using CELESTA.
<em>NMETH</em>, <em>19</em>(6), 759–769. (<a
href="https://doi.org/10.1038/s41592-022-01498-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in multiplexed in situ imaging are revealing important insights in spatial biology. However, cell type identification remains a major challenge in imaging analysis, with most existing methods involving substantial manual assessment and subjective decisions for thousands of cells. We developed an unsupervised machine learning algorithm, CELESTA, which identifies the cell type of each cell, individually, using the cell’s marker expression profile and, when needed, its spatial information. We demonstrate the performance of CELESTA on multiplexed immunofluorescence images of colorectal cancer and head and neck squamous cell carcinoma (HNSCC). Using the cell types identified by CELESTA, we identify tissue architecture associated with lymph node metastasis in HNSCC, and validate our findings in an independent cohort. By coupling our spatial analysis with single-cell RNA-sequencing data on proximal sections of the same specimens, we identify cell–cell crosstalk associated with lymph node metastasis, demonstrating the power of CELESTA to facilitate identification of clinically relevant interactions. CELESTA identifies cell types in multiplexed imaging datasets based on the expression profiles of cells and their spatial information.},
  archive      = {J_NMETH},
  author       = {Zhang, Weiruo and Li, Irene and Reticker-Flynn, Nathan E. and Good, Zinaida and Chang, Serena and Samusik, Nikolay and Saumyaa, Saumyaa and Li, Yuanyuan and Zhou, Xin and Liang, Rachel and Kong, Christina S. and Le, Quynh-Thu and Gentles, Andrew J. and Sunwoo, John B. and Nolan, Garry P. and Engleman, Edgar G. and Plevritis, Sylvia K.},
  doi          = {10.1038/s41592-022-01498-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {759-769},
  shortjournal = {Nat. Meth.},
  title        = {Identification of cell types in multiplexed in situ images by combining protein expression and spatial information using CELESTA},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Label-free nanofluidic scattering microscopy of size and
mass of single diffusing molecules and nanoparticles. <em>NMETH</em>,
<em>19</em>(6), 751–758. (<a
href="https://doi.org/10.1038/s41592-022-01491-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Label-free characterization of single biomolecules aims to complement fluorescence microscopy in situations where labeling compromises data interpretation, is technically challenging or even impossible. However, existing methods require the investigated species to bind to a surface to be visible, thereby leaving a large fraction of analytes undetected. Here, we present nanofluidic scattering microscopy (NSM), which overcomes these limitations by enabling label-free, real-time imaging of single biomolecules diffusing inside a nanofluidic channel. NSM facilitates accurate determination of molecular weight from the measured optical contrast and of the hydrodynamic radius from the measured diffusivity, from which information about the conformational state can be inferred. Furthermore, we demonstrate its applicability to the analysis of a complex biofluid, using conditioned cell culture medium containing extracellular vesicles as an example. We foresee the application of NSM to monitor conformational changes, aggregation and interactions of single biomolecules, and to analyze single-cell secretomes. Nanofluidic scattering microscopy enables label-free, quantitative measurements of the molecular weight and hydrodynamic radius of biological molecules and nanoparticles freely diffusing inside a nanofluidic channel.},
  archive      = {J_NMETH},
  author       = {Špačková, Barbora and Klein Moberg, Henrik and Fritzsche, Joachim and Tenghamn, Johan and Sjösten, Gustaf and Šípová-Jungová, Hana and Albinsson, David and Lubart, Quentin and van Leeuwen, Daniel and Westerlund, Fredrik and Midtvedt, Daniel and Esbjörner, Elin K. and Käll, Mikael and Volpe, Giovanni and Langhammer, Christoph},
  doi          = {10.1038/s41592-022-01491-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {751-758},
  shortjournal = {Nat. Meth.},
  title        = {Label-free nanofluidic scattering microscopy of size and mass of single diffusing molecules and nanoparticles},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-domain near-infrared protein provides a scaffold for
antigen-dependent fluorescent nanobodies. <em>NMETH</em>,
<em>19</em>(6), 740–750. (<a
href="https://doi.org/10.1038/s41592-022-01467-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Small near-infrared (NIR) fluorescent proteins (FPs) are much needed as protein tags for imaging applications. We developed a 17 kDa NIR FP, called miRFP670nano3, which brightly fluoresces in mammalian cells and enables deep-brain imaging. By exploring miRFP670nano3 as an internal tag, we engineered 32 kDa NIR fluorescent nanobodies, termed NIR-Fbs, whose stability and fluorescence strongly depend on the presence of specific intracellular antigens. NIR-Fbs allowed background-free visualization of endogenous proteins, detection of viral antigens, labeling of cells expressing target molecules and identification of double-positive cell populations with bispecific NIR-Fbs against two antigens. Applying NIR-Fbs as destabilizing fusion partners, we developed molecular tools for directed degradation of targeted proteins, controllable protein expression and modulation of enzymatic activities. Altogether, NIR-Fbs enable the detection and manipulation of a variety of cellular processes based on the intracellular protein profile. miRFP670nano3 offers improved near-infrared imaging and was used to develop fluorescent nanobodies whose stability and fluorescence strongly depend on antigen binding, with broad implications for detecting and manipulating cellular targets.},
  archive      = {J_NMETH},
  author       = {Oliinyk, Olena S. and Baloban, Mikhail and Clark, Charles L. and Carey, Erin and Pletnev, Sergei and Nimmerjahn, Axel and Verkhusha, Vladislav V.},
  doi          = {10.1038/s41592-022-01467-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {740-750},
  shortjournal = {Nat. Meth.},
  title        = {Single-domain near-infrared protein provides a scaffold for antigen-dependent fluorescent nanobodies},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ScanNet: An interpretable geometric deep learning model for
structure-based protein binding site prediction. <em>NMETH</em>,
<em>19</em>(6), 730–739. (<a
href="https://doi.org/10.1038/s41592-022-01490-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting the functional sites of a protein from its structure, such as the binding sites of small molecules, other proteins or antibodies, sheds light on its function in vivo. Currently, two classes of methods prevail: machine learning models built on top of handcrafted features and comparative modeling. They are, respectively, limited by the expressivity of the handcrafted features and the availability of similar proteins. Here, we introduce ScanNet, an end-to-end, interpretable geometric deep learning model that learns features directly from 3D structures. ScanNet builds representations of atoms and amino acids based on the spatio-chemical arrangement of their neighbors. We train ScanNet for detecting protein–protein and protein–antibody binding sites, demonstrate its accuracy—including for unseen protein folds—and interpret the filters learned. Finally, we predict epitopes of the SARS-CoV-2 spike protein, validating known antigenic regions and predicting previously uncharacterized ones. Overall, ScanNet is a versatile, powerful and interpretable model suitable for functional site prediction tasks. A webserver for ScanNet is available from http://bioinfo3d.cs.tau.ac.il/ScanNet/ . ScanNet is an end-to-end, interpretable geometric deep learning model that learns spatio-chemical and atomic features directly from protein 3D structures and can be employed for functional site prediction tasks.},
  archive      = {J_NMETH},
  author       = {Tubiana, Jérôme and Schneidman-Duhovny, Dina and Wolfson, Haim J.},
  doi          = {10.1038/s41592-022-01490-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {730-739},
  shortjournal = {Nat. Meth.},
  title        = {ScanNet: An interpretable geometric deep learning model for structure-based protein binding site prediction},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ab initio phasing macromolecular structures using
electron-counted MicroED data. <em>NMETH</em>, <em>19</em>(6), 724–729.
(<a href="https://doi.org/10.1038/s41592-022-01485-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structures of two globular proteins were determined ab initio using microcrystal electron diffraction (MicroED) data that were collected on a direct electron detector in counting mode. Microcrystals were identified using a scanning electron microscope (SEM) and thinned with a focused ion beam (FIB) to produce crystalline lamellae of ideal thickness. Continuous-rotation data were collected using an ultra-low exposure rate to enable electron counting in diffraction. For the first sample, triclinic lysozyme extending to a resolution of 0.87 Å, an ideal helical fragment of only three alanine residues provided initial phases. These phases were improved using density modification, allowing the entire atomic structure to be built automatically. A similar approach was successful on a second macromolecular sample, proteinase K, which is much larger and diffracted to a resolution of 1.5 Å. These results demonstrate that macromolecules can be determined to sub-ångström resolution by MicroED and that ab initio phasing can be successfully applied to counting data. This article reports sub- and near-atomic structures of triclinic lysozyme and serine protease proteinase K, respectively, providing first demonstrations of ab initio phasing using electron counted MicroED data to solve macromolecular structures.},
  archive      = {J_NMETH},
  author       = {Martynowycz, Michael W. and Clabbers, Max T. B. and Hattne, Johan and Gonen, Tamir},
  doi          = {10.1038/s41592-022-01485-4},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {724-729},
  shortjournal = {Nat. Meth.},
  title        = {Ab initio phasing macromolecular structures using electron-counted MicroED data},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DiMeLo-seq: A long-read, single-molecule method for mapping
protein–DNA interactions genome wide. <em>NMETH</em>, <em>19</em>(6),
711–723. (<a href="https://doi.org/10.1038/s41592-022-01475-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies of genome regulation routinely use high-throughput DNA sequencing approaches to determine where specific proteins interact with DNA, and they rely on DNA amplification and short-read sequencing, limiting their quantitative application in complex genomic regions. To address these limitations, we developed directed methylation with long-read sequencing (DiMeLo-seq), which uses antibody-tethered enzymes to methylate DNA near a target protein’s binding sites in situ. These exogenous methylation marks are then detected simultaneously with endogenous CpG methylation on unamplified DNA using long-read, single-molecule sequencing technologies. We optimized and benchmarked DiMeLo-seq by mapping chromatin-binding proteins and histone modifications across the human genome. Furthermore, we identified where centromere protein A localizes within highly repetitive regions that were unmappable with short sequencing reads, and we estimated the density of centromere protein A molecules along single chromatin fibers. DiMeLo-seq is a versatile method that provides multimodal, genome-wide information for investigating protein–DNA interactions. DiMeLo-seq uses native long-read sequencing to examine protein–DNA interactions by mapping exogenous methylation marks generated by a nonspecific DNA methyltransferase, as well as profile endogenous CpG methylation simultaneously.},
  archive      = {J_NMETH},
  author       = {Altemose, Nicolas and Maslan, Annie and Smith, Owen K. and Sundararajan, Kousik and Brown, Rachel R. and Mishra, Reet and Detweiler, Angela M. and Neff, Norma and Miga, Karen H. and Straight, Aaron F. and Streets, Aaron},
  doi          = {10.1038/s41592-022-01475-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {711-723},
  shortjournal = {Nat. Meth.},
  title        = {DiMeLo-seq: A long-read, single-molecule method for mapping protein–DNA interactions genome wide},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Long-read mapping to repetitive reference sequences using
winnowmap2. <em>NMETH</em>, <em>19</em>(6), 705–710. (<a
href="https://doi.org/10.1038/s41592-022-01457-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Approximately 5–10\% of the human genome remains inaccessible due to the presence of repetitive sequences such as segmental duplications and tandem repeat arrays. We show that existing long-read mappers often yield incorrect alignments and variant calls within long, near-identical repeats, as they remain vulnerable to allelic bias. In the presence of a nonreference allele within a repeat, a read sampled from that region could be mapped to an incorrect repeat copy. To address this limitation, we developed a new long-read mapping method, Winnowmap2, by using minimal confidently alignable substrings. Winnowmap2 computes each read mapping through a collection of confident subalignments. This approach is more tolerant of structural variation and more sensitive to paralog-specific variants within repeats. Our experiments highlight that Winnowmap2 successfully addresses the issue of allelic bias, enabling more accurate downstream variant calls in repetitive sequences. Winnowmap2 enables better long-read mapping and more accurate variant calling in repetitive regions of the genome.},
  archive      = {J_NMETH},
  author       = {Jain, Chirag and Rhie, Arang and Hansen, Nancy F. and Koren, Sergey and Phillippy, Adam M.},
  doi          = {10.1038/s41592-022-01457-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {705-710},
  shortjournal = {Nat. Meth.},
  title        = {Long-read mapping to repetitive reference sequences using winnowmap2},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Merfin: Improved variant filtering, assembly evaluation and
polishing via k-mer validation. <em>NMETH</em>, <em>19</em>(6), 696–704.
(<a href="https://doi.org/10.1038/s41592-022-01445-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Variant calling has been widely used for genotyping and for improving the consensus accuracy of long-read assemblies. Variant calls are commonly hard-filtered with user-defined cutoffs. However, it is impossible to define a single set of optimal cutoffs, as the calls heavily depend on the quality of the reads, the variant caller of choice and the quality of the unpolished assembly. Here, we introduce Merfin, a k-mer based variant-filtering algorithm for improved accuracy in genotyping and genome assembly polishing. Merfin evaluates each variant based on the expected k-mer multiplicity in the reads, independently of the quality of the read alignment and variant caller’s internal score. Merfin increased the precision of genotyped calls in several benchmarks, improved consensus accuracy and reduced frameshift errors when applied to human and nonhuman assemblies built from Pacific Biosciences HiFi and continuous long reads or Oxford Nanopore reads, including the first complete human genome. Moreover, we introduce assembly quality and completeness metrics that account for the expected genomic copy numbers. Merfin is a computational tool for variant filtering that improves accuracy in genotyping and genome assembly polishing.},
  archive      = {J_NMETH},
  author       = {Formenti, Giulio and Rhie, Arang and Walenz, Brian P. and Thibaud-Nissen, Françoise and Shafin, Kishwar and Koren, Sergey and Myers, Eugene W. and Jarvis, Erich D. and Phillippy, Adam M.},
  doi          = {10.1038/s41592-022-01445-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {696-704},
  shortjournal = {Nat. Meth.},
  title        = {Merfin: Improved variant filtering, assembly evaluation and polishing via k-mer validation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chasing perfection: Validation and polishing strategies for
telomere-to-telomere genome assemblies. <em>NMETH</em>, <em>19</em>(6),
687–695. (<a href="https://doi.org/10.1038/s41592-022-01440-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in long-read sequencing technologies and genome assembly methods have enabled the recent completion of the first telomere-to-telomere human genome assembly, which resolves complex segmental duplications and large tandem repeats, including centromeric satellite arrays in a complete hydatidiform mole (CHM13). Although derived from highly accurate sequences, evaluation revealed evidence of small errors and structural misassemblies in the initial draft assembly. To correct these errors, we designed a new repeat-aware polishing strategy that made accurate assembly corrections in large repeats without overcorrection, ultimately fixing 51\% of the existing errors and improving the assembly quality value from 70.2 to 73.9 measured from PacBio high-fidelity and Illumina k-mers. By comparing our results to standard automated polishing tools, we outline common polishing errors and offer practical suggestions for genome projects with limited resources. We also show how sequencing biases in both high-fidelity and Oxford Nanopore Technologies reads cause signature assembly errors that can be corrected with a diverse panel of sequencing technologies. The work describes the validation and polishing strategies developed by the telomere-to-telomere consortium for evaluating and improving the first complete human genome assembly.},
  archive      = {J_NMETH},
  author       = {Mc Cartney, Ann M. and Shafin, Kishwar and Alonge, Michael and Bzikadze, Andrey V. and Formenti, Giulio and Fungtammasan, Arkarachai and Howe, Kerstin and Jain, Chirag and Koren, Sergey and Logsdon, Glennis A. and Miga, Karen H. and Mikheenko, Alla and Paten, Benedict and Shumate, Alaina and Soto, Daniela C. and Sović, Ivan and Wood, Jonathan M. D. and Zook, Justin M. and Phillippy, Adam M. and Rhie, Arang},
  doi          = {10.1038/s41592-022-01440-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {687-695},
  shortjournal = {Nat. Meth.},
  title        = {Chasing perfection: Validation and polishing strategies for telomere-to-telomere genome assemblies},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ASLPrep: A platform for processing of arterial spin labeled
MRI and quantification of regional brain perfusion. <em>NMETH</em>,
<em>19</em>(6), 683–686. (<a
href="https://doi.org/10.1038/s41592-022-01458-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Arterial spin labeled (ASL) magnetic resonance imaging (MRI) is the primary method for noninvasively measuring regional brain perfusion in humans. We introduce ASLPrep, a suite of software pipelines that ensure the reproducible and generalizable processing of ASL MRI data. ASLPrep is a software suite for reproducible processing of arterial spin labeled magnetic resonance imaging data.},
  archive      = {J_NMETH},
  author       = {Adebimpe, Azeez and Bertolero, Maxwell and Dolui, Sudipto and Cieslak, Matthew and Murtha, Kristin and Baller, Erica B. and Boeve, Bradley and Boxer, Adam and Butler, Ellyn R. and Cook, Phil and Colcombe, Stan and Covitz, Sydney and Davatzikos, Christos and Davila, Diego G. and Elliott, Mark A. and Flounders, Matthew W. and Franco, Alexandre R. and Gur, Raquel E. and Gur, Ruben C. and Jaber, Basma and McMillian, Corey and Milham, Michael and Mutsaerts, Henk J. M. M. and Oathes, Desmond J. and Olm, Christopher A. and Phillips, Jeffrey S. and Tackett, Will and Roalf, David R. and Rosen, Howard and Tapera, Tinashe M. and Tisdall, M. Dylan and Zhou, Dale and Esteban, Oscar and Poldrack, Russell A. and Detre, John A. and Satterthwaite, Theodore D.},
  doi          = {10.1038/s41592-022-01458-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {683-686},
  shortjournal = {Nat. Meth.},
  title        = {ASLPrep: A platform for processing of arterial spin labeled MRI and quantification of regional brain perfusion},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ColabFold: Making protein folding accessible to all.
<em>NMETH</em>, <em>19</em>(6), 679–682. (<a
href="https://doi.org/10.1038/s41592-022-01488-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {ColabFold offers accelerated prediction of protein structures and complexes by combining the fast homology search of MMseqs2 with AlphaFold2 or RoseTTAFold. ColabFold’s 40−60-fold faster search and optimized model utilization enables prediction of close to 1,000 structures per day on a server with one graphics processing unit. Coupled with Google Colaboratory, ColabFold becomes a free and accessible platform for protein folding. ColabFold is open-source software available at https://github.com/sokrypton/ColabFold and its novel environmental databases are available at https://colabfold.mmseqs.com . ColabFold is a free and accessible platform for protein folding that provides accelerated prediction of protein structures and complexes using AlphaFold2 or RoseTTAFold.},
  archive      = {J_NMETH},
  author       = {Mirdita, Milot and Schütze, Konstantin and Moriwaki, Yoshitaka and Heo, Lim and Ovchinnikov, Sergey and Steinegger, Martin},
  doi          = {10.1038/s41592-022-01488-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {679-682},
  shortjournal = {Nat. Meth.},
  title        = {ColabFold: Making protein folding accessible to all},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A learned embedding for efficient joint analysis of millions
of mass spectra. <em>NMETH</em>, <em>19</em>(6), 675–678. (<a
href="https://doi.org/10.1038/s41592-022-01496-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational methods that aim to exploit publicly available mass spectrometry repositories rely primarily on unsupervised clustering of spectra. Here we trained a deep neural network in a supervised fashion on the basis of previous assignments of peptides to spectra. The network, called ‘GLEAMS’, learns to embed spectra in a low-dimensional space in which spectra generated by the same peptide are close to one another. We applied GLEAMS for large-scale spectrum clustering, detecting groups of unidentified, proximal spectra representing the same peptide. We used these clusters to explore the dark proteome of repeatedly observed yet consistently unidentified mass spectra. GLEAMS, a deep learning-based algorithm, embeds mass spectra such that spectra related to the same peptide are close to each other, enabling unknown spectra to be identified on a massive scale.},
  archive      = {J_NMETH},
  author       = {Bittremieux, Wout and May, Damon H. and Bilmes, Jeffrey and Noble, William Stafford},
  doi          = {10.1038/s41592-022-01496-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {675-678},
  shortjournal = {Nat. Meth.},
  title        = {A learned embedding for efficient joint analysis of millions of mass spectra},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Metagenome assembly of high-fidelity long reads with
hifiasm-meta. <em>NMETH</em>, <em>19</em>(6), 671–674. (<a
href="https://doi.org/10.1038/s41592-022-01478-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo assembly of metagenome samples is a common approach to the study of microbial communities. Current metagenome assemblers developed for short sequence reads or noisy long reads were not optimized for accurate long reads. We thus developed hifiasm-meta, a metagenome assembler that exploits the high accuracy of recent data. Evaluated on seven empirical datasets, hifiasm-meta reconstructed tens to hundreds of complete circular bacterial genomes per dataset, consistently outperforming other metagenome assemblers. This work presents hifiasm-meta, a software tool designed for assembling metagenomes using high-accuracy long-read data.},
  archive      = {J_NMETH},
  author       = {Feng, Xiaowen and Cheng, Haoyu and Portik, Daniel and Li, Heng},
  doi          = {10.1038/s41592-022-01478-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {671-674},
  shortjournal = {Nat. Meth.},
  title        = {Metagenome assembly of high-fidelity long reads with hifiasm-meta},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Benchmarking spatial and single-cell transcriptomics
integration methods for transcript distribution prediction and cell type
deconvolution. <em>NMETH</em>, <em>19</em>(6), 662–670. (<a
href="https://doi.org/10.1038/s41592-022-01480-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics approaches have substantially advanced our capacity to detect the spatial distribution of RNA transcripts in tissues, yet it remains challenging to characterize whole-transcriptome-level data for single cells in space. Addressing this need, researchers have developed integration methods to combine spatial transcriptomic data with single-cell RNA-seq data to predict the spatial distribution of undetected transcripts and/or perform cell type deconvolution of spots in histological sections. However, to date, no independent studies have comparatively analyzed these integration methods to benchmark their performance. Here we present benchmarking of 16 integration methods using 45 paired datasets (comprising both spatial transcriptomics and scRNA-seq data) and 32 simulated datasets. We found that Tangram, gimVI, and SpaGE outperformed other integration methods for predicting the spatial distribution of RNA transcripts, whereas Cell2location, SpatialDWLS, and RCTD are the top-performing methods for the cell type deconvolution of spots. We provide a benchmark pipeline to help researchers select optimal integration methods to process their datasets. This work presents a comprehensive benchmarking analysis of computational methods that integrates spatial and single-cell transcriptomics data for transcript distribution prediction and cell type deconvolution.},
  archive      = {J_NMETH},
  author       = {Li, Bin and Zhang, Wen and Guo, Chuang and Xu, Hao and Li, Longfei and Fang, Minghao and Hu, Yinlei and Zhang, Xinye and Yao, Xinfeng and Tang, Meifang and Liu, Ke and Zhao, Xuetong and Lin, Jun and Cheng, Linzhao and Chen, Falai and Xue, Tian and Qu, Kun},
  doi          = {10.1038/s41592-022-01480-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {662-670},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ScanNet uncovers binding motifs in protein structures with
deep learning. <em>NMETH</em>, <em>19</em>(6), 660–661. (<a
href="https://doi.org/10.1038/s41592-022-01492-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining the functional properties of a protein from its structure is challenging. This study presents an interpretable deep learning model that directly learns function-bearing structural motifs from raw data, allowing accurate mapping of protein binding sites and antibody epitopes onto a protein structure.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01492-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {660-661},
  shortjournal = {Nat. Meth.},
  title        = {ScanNet uncovers binding motifs in protein structures with deep learning},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A neural network for large-scale clustering of peptide mass
spectra. <em>NMETH</em>, <em>19</em>(6), 658–659. (<a
href="https://doi.org/10.1038/s41592-022-01497-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Repository-scale analysis of hundreds of millions to billions of mass spectra is a challenging endeavor due to the complexity and volume of associated data. A deep neural network embedding method is presented that enables large-scale investigation of repeatedly observed yet consistently unidentified mass spectra.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01497-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {658-659},
  shortjournal = {Nat. Meth.},
  title        = {A neural network for large-scale clustering of peptide mass spectra},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Benchmarking spatial and single-cell transcriptomics
integration methods. <em>NMETH</em>, <em>19</em>(6), 656–657. (<a
href="https://doi.org/10.1038/s41592-022-01481-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tangram, gimVI and SpaGE outperformed other integration methods for predicting the spatial distributions of RNA transcripts, while Cell2location, SpatialDWLS and RCTD were the top-performing methods for the cell type deconvolution of spots in histological sections.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01481-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {656-657},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking spatial and single-cell transcriptomics integration methods},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). New near-infrared fluorescent probes and tools.
<em>NMETH</em>, <em>19</em>(6), 654–655. (<a
href="https://doi.org/10.1038/s41592-022-01425-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A novel bright near-infrared fluorescent protein inserted into a nanobody enables visualization of native proteins inside living cells and specific manipulation of cell function, including Boolean protein-based operators.},
  archive      = {J_NMETH},
  author       = {Gadella, Theodorus W. J.},
  doi          = {10.1038/s41592-022-01425-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {654-655},
  shortjournal = {Nat. Meth.},
  title        = {New near-infrared fluorescent probes and tools},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Electron counting takes microED to the next level.
<em>NMETH</em>, <em>19</em>(6), 652–653. (<a
href="https://doi.org/10.1038/s41592-022-01518-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New data collection methods enable ab initio protein structure determination by micro-electron diffraction for the first time.},
  archive      = {J_NMETH},
  author       = {Corbett, Kevin D. and Herzik, Mark A.},
  doi          = {10.1038/s41592-022-01518-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {652-653},
  shortjournal = {Nat. Meth.},
  title        = {Electron counting takes microED to the next level},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mapping beads on strings. <em>NMETH</em>, <em>19</em>(6),
651–652. (<a href="https://doi.org/10.1038/s41592-022-01519-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DiMeLo-seq leverages immunotethered DNA methyltransferases with long-read sequencing to map the locations of chromatin proteins in their natural context.},
  archive      = {J_NMETH},
  author       = {Ahmad, Kami},
  doi          = {10.1038/s41592-022-01519-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {651-652},
  shortjournal = {Nat. Meth.},
  title        = {Mapping beads on strings},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Polishing high-quality genome assemblies. <em>NMETH</em>,
<em>19</em>(6), 649–650. (<a
href="https://doi.org/10.1038/s41592-022-01515-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A repeat-aware pipeline corrects structural assembly errors and polishes assembly sequences in large repeats without overcorrection.},
  archive      = {J_NMETH},
  author       = {Fang, Li and Wang, Kai},
  doi          = {10.1038/s41592-022-01515-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {649-650},
  shortjournal = {Nat. Meth.},
  title        = {Polishing high-quality genome assemblies},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022o). Modeling the early embryo. <em>NMETH</em>, <em>19</em>(6),
644–648. (<a href="https://doi.org/10.1038/s41592-022-01516-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Widespread adoption of new embryo models hinges on how faithfully they mirror real embryos.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01516-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {644-648},
  shortjournal = {Nat. Meth.},
  title        = {Modeling the early embryo},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Modeling condensate formation in silico. <em>NMETH</em>,
<em>19</em>(6), 643. (<a
href="https://doi.org/10.1038/s41592-022-01535-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A coarse-grained model enables simulating droplet formation with low-complexity RNA sequences.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01535-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {643},
  shortjournal = {Nat. Meth.},
  title        = {Modeling condensate formation in silico},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). A fluorescent protein with staying power. <em>NMETH</em>,
<em>19</em>(6), 642. (<a
href="https://doi.org/10.1038/s41592-022-01534-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A newly described fluorescent protein, StayGold, is bright and extremely photostable, enabling extended time-lapse imaging.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01534-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {642},
  shortjournal = {Nat. Meth.},
  title        = {A fluorescent protein with staying power},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Modeling multi-organ systems on a chip. <em>NMETH</em>,
<em>19</em>(6), 641. (<a
href="https://doi.org/10.1038/s41592-022-01533-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To recapitulate some of the complexities of the human physiology, engineered tissues can be linked by vascular flow in a multi-organ chip.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01533-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {641},
  shortjournal = {Nat. Meth.},
  title        = {Modeling multi-organ systems on a chip},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Base editing in mitochondrial DNA. <em>NMETH</em>,
<em>19</em>(6), 640. (<a
href="https://doi.org/10.1038/s41592-022-01532-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {TALE-fused deaminases enable C-to-T or A-to-G base editing in human mitochondria.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01532-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {640},
  shortjournal = {Nat. Meth.},
  title        = {Base editing in mitochondrial DNA},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Reversing the clock on human somatic cells. <em>NMETH</em>,
<em>19</em>(6), 639. (<a
href="https://doi.org/10.1038/s41592-022-01531-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A chemical cocktail reprograms human somatic cells into pluripotent stem cells.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01531-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {639},
  shortjournal = {Nat. Meth.},
  title        = {Reversing the clock on human somatic cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A complete, telomere-to-telomere human genome sequence
presents new opportunities for evolutionary genomics. <em>NMETH</em>,
<em>19</em>(6), 635–638. (<a
href="https://doi.org/10.1038/s41592-022-01512-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The release of the first telomere-to-telomere (T2T) human genome sequence marks a milestone for human genomics research and holds promise of complete genomes for evolutionary genomic studies. Here we describe the advances that this new human genome assembly represents and explore the potential insights that the complete genome sequence could bring to evolutionary genomics. We also discuss the potential challenges to be faced in applying this new sequencing strategy to a broad spectrum of extant species.},
  archive      = {J_NMETH},
  author       = {Mao, Yafei and Zhang, Guojie},
  doi          = {10.1038/s41592-022-01512-4},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {635-638},
  shortjournal = {Nat. Meth.},
  title        = {A complete, telomere-to-telomere human genome sequence presents new opportunities for evolutionary genomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Best practices for reporting throughput in biomedical
research. <em>NMETH</em>, <em>19</em>(6), 633–634. (<a
href="https://doi.org/10.1038/s41592-022-01483-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Herbig, Maik and Isozaki, Akihiro and Di Carlo, Dino and Guck, Jochen and Nitta, Nao and Damoiseaux, Robert and Kamikawaji, Shogo and Suyama, Eigo and Shintaku, Hirofumi and Wu, Angela Ruohao and Nikaido, Itoshi and Goda, Keisuke},
  doi          = {10.1038/s41592-022-01483-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {633-634},
  shortjournal = {Nat. Meth.},
  title        = {Best practices for reporting throughput in biomedical research},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Everyday ups and downs for LGBT+ scientists.
<em>NMETH</em>, <em>19</em>(6), 632. (<a
href="https://doi.org/10.1038/s41592-022-01510-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How to gauge academic culture and celebrate being an LGBT+ scientist.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01510-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {632},
  shortjournal = {Nat. Meth.},
  title        = {Everyday ups and downs for LGBT+ scientists},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Completing human genomes. <em>NMETH</em>, <em>19</em>(6),
629. (<a href="https://doi.org/10.1038/s41592-022-01537-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Methods is pleased to publish several papers presenting methods developed by members of the Telomere-to-Telomere (T2T) Consortium, which facilitated the generation and analysis of the first complete human genome.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01537-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {629},
  shortjournal = {Nat. Meth.},
  title        = {Completing human genomes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022t). Tamir gonen. <em>NMETH</em>, <em>19</em>(6), 631. (<a
href="https://doi.org/10.1038/s41592-022-01511-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MicroED for structure determination without a model and with heavy metal guitar riffs.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01511-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {6},
  pages        = {631},
  shortjournal = {Nat. Meth.},
  title        = {Tamir gonen},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Museum of spatial transcriptomics.
<em>NMETH</em>, <em>19</em>(5), 628. (<a
href="https://doi.org/10.1038/s41592-022-01494-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Moses, Lambda and Pachter, Lior},
  doi          = {10.1038/s41592-022-01494-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {628},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Museum of spatial transcriptomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Publisher correction: SLEAP: A deep learning system for
multi-animal pose tracking. <em>NMETH</em>, <em>19</em>(5), 628. (<a
href="https://doi.org/10.1038/s41592-022-01495-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Pereira, Talmo D. and Tabris, Nathaniel and Matsliah, Arie and Turner, David M. and Li, Junyu and Ravindranath, Shruthi and Papadoyannis, Eleni S. and Normand, Edna and Deutsch, David S. and Wang, Z. Yan and McKenzie-Smith, Grace C. and Mitelut, Catalin C. and Castro, Marielisa Diez and D’Uva, John and Kislin, Mikhail and Sanes, Dan H. and Kocher, Sarah D. and Wang, Samuel S.-H. and Falkner, Annegret L. and Shaevitz, Joshua W. and Murthy, Mala},
  doi          = {10.1038/s41592-022-01495-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {628},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: SLEAP: a deep learning system for multi-animal pose tracking},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Publisher correction: The SpliZ generalizes “percent
spliced in” to reveal regulated splicing at single-cell resolution.
<em>NMETH</em>, <em>19</em>(5), 628. (<a
href="https://doi.org/10.1038/s41592-022-01500-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Olivieri, Julia Eve and Dehghannasiri, Roozbeh and Salzman, Julia},
  doi          = {10.1038/s41592-022-01500-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {628},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: The SpliZ generalizes ‘percent spliced in’ to reveal regulated splicing at single-cell resolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). NeuroMechFly, a neuromechanical model of adult drosophila
melanogaster. <em>NMETH</em>, <em>19</em>(5), 620–627. (<a
href="https://doi.org/10.1038/s41592-022-01466-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal behavior emerges from an interaction between neural network dynamics, musculoskeletal properties and the physical environment. Accessing and understanding the interplay between these elements requires the development of integrative and morphologically realistic neuromechanical simulations. Here we present NeuroMechFly, a data-driven model of the widely studied organism, Drosophila melanogaster. NeuroMechFly combines four independent computational modules: a physics-based simulation environment, a biomechanical exoskeleton, muscle models and neural network controllers. To enable use cases, we first define the minimum degrees of freedom of the leg from real three-dimensional kinematic measurements during walking and grooming. Then, we show how, by replaying these behaviors in the simulator, one can predict otherwise unmeasured torques and contact forces. Finally, we leverage NeuroMechFly’s full neuromechanical capacity to discover neural networks and muscle parameters that drive locomotor gaits optimized for speed and stability. Thus, NeuroMechFly can increase our understanding of how behaviors emerge from interactions between complex neuromechanical systems and their physical surroundings. NeuroMechFly enables simulations of adult Drosophila melanogaster. The platform combines a biomechanical representation of the fly body, models of the muscles, a neural controller and a physics-based simulation of the environment.},
  archive      = {J_NMETH},
  author       = {Lobato-Rios, Victor and Ramalingasetty, Shravan Tata and Özdil, Pembe Gizem and Arreguit, Jonathan and Ijspeert, Auke Jan and Ramdya, Pavan},
  doi          = {10.1038/s41592-022-01466-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {620-627},
  shortjournal = {Nat. Meth.},
  title        = {NeuroMechFly, a neuromechanical model of adult drosophila melanogaster},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A hybrid open-top light-sheet microscope for versatile
multi-scale imaging of cleared tissues. <em>NMETH</em>, <em>19</em>(5),
613–619. (<a href="https://doi.org/10.1038/s41592-022-01468-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light-sheet microscopy has emerged as the preferred means for high-throughput volumetric imaging of cleared tissues. However, there is a need for a flexible system that can address imaging applications with varied requirements in terms of resolution, sample size, tissue-clearing protocol, and transparent sample-holder material. Here, we present a ‘hybrid’ system that combines a unique non-orthogonal dual-objective and conventional (orthogonal) open-top light-sheet (OTLS) architecture for versatile multi-scale volumetric imaging. We demonstrate efficient screening and targeted sub-micrometer imaging of sparse axons within an intact, cleared mouse brain. The same system enables high-throughput automated imaging of multiple specimens, as spotlighted by a quantitative multi-scale analysis of brain metastases. Compared with existing academic and commercial light-sheet microscopy systems, our hybrid OTLS system provides a unique combination of versatility and performance necessary to satisfy the diverse requirements of a growing number of cleared-tissue imaging applications. A ‘hybrid’ open-top light-sheet microscope is described, which can be used for broad multi-scale volumetric imaging of one or more large tissues, cleared with diverse protocols, and conveniently mounted on an array of sample holders.},
  archive      = {J_NMETH},
  author       = {Glaser, Adam K. and Bishop, Kevin W. and Barner, Lindsey A. and Susaki, Etsuo A. and Kubota, Shimpei I. and Gao, Gan and Serafin, Robert B. and Balaram, Pooja and Turschak, Emily and Nicovich, Philip R. and Lai, Hoyin and Lucas, Luciano A. G. and Yi, Yating and Nichols, Eva K. and Huang, Hongyi and Reder, Nicholas P. and Wilson, Jasmine J. and Sivakumar, Ramya and Shamskhou, Elya and Stoltzfus, Caleb R. and Wei, Xing and Hempton, Andrew K. and Pende, Marko and Murawala, Prayag and Dodt, Hans-Ulrich and Imaizumi, Takato and Shendure, Jay and Beliveau, Brian J. and Gerner, Michael Y. and Xin, Li and Zhao, Hu and True, Lawrence D. and Reid, R. Clay and Chandrashekar, Jayaram and Ueda, Hiroki R. and Svoboda, Karel and Liu, Jonathan T. C.},
  doi          = {10.1038/s41592-022-01468-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {613-619},
  shortjournal = {Nat. Meth.},
  title        = {A hybrid open-top light-sheet microscope for versatile multi-scale imaging of cleared tissues},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Optimal precision and accuracy in 4Pi-STORM using dynamic
spline PSF models. <em>NMETH</em>, <em>19</em>(5), 603–612. (<a
href="https://doi.org/10.1038/s41592-022-01465-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coherent fluorescence imaging with two objective lenses (4Pi detection) enables single-molecule localization microscopy with sub-10 nm spatial resolution in three dimensions. Despite its outstanding sensitivity, wider application of this technique has been hindered by complex instrumentation and the challenging nature of the data analysis. Here we report the development of a 4Pi-STORM microscope, which obtains optimal resolution and accuracy by modeling the 4Pi point spread function (PSF) dynamically while also using a simpler optical design. Dynamic spline PSF models incorporate fluctuations in the modulation phase of the experimentally determined PSF, capturing the temporal evolution of the optical system. Our method reaches the theoretical limits for precision and minimizes phase-wrapping artifacts by making full use of the information content of the data. 4Pi-STORM achieves a near-isotropic three-dimensional localization precision of 2–3 nm, and we demonstrate its capabilities by investigating protein and nucleic acid organization in primary neurons and mammalian mitochondria. A dynamic model of the 4Pi point spread function enables localization microscopy with exceptional three-dimensional resolution and a simpler optical design. 4Pi-STORM images of neurons and mitochondria reveal new details of nanoscale protein and nucleic acid organization.},
  archive      = {J_NMETH},
  author       = {Bates, Mark and Keller-Findeisen, Jan and Przybylski, Adrian and Hüper, Andreas and Stephan, Till and Ilgen, Peter and Cereceda Delgado, Angel R. and D’Este, Elisa and Egner, Alexander and Jakobs, Stefan and Sahl, Steffen J. and Hell, Stefan W.},
  doi          = {10.1038/s41592-022-01465-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {603-612},
  shortjournal = {Nat. Meth.},
  title        = {Optimal precision and accuracy in 4Pi-STORM using dynamic spline PSF models},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Correction of multiple-blinking artifacts in photoactivated
localization microscopy. <em>NMETH</em>, <em>19</em>(5), 594–602. (<a
href="https://doi.org/10.1038/s41592-022-01463-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photoactivated localization microscopy (PALM) produces an array of localization coordinates by means of photoactivatable fluorescent proteins. However, observations are subject to fluorophore multiple blinking and each protein is included in the dataset an unknown number of times at different positions, due to localization error. This causes artificial clustering to be observed in the data. We present a ‘model-based correction’ (MBC) workflow using calibration-free estimation of blinking dynamics and model-based clustering to produce a corrected set of localization coordinates representing the true underlying fluorophore locations with enhanced localization precision, outperforming the state of the art. The corrected data can be reliably tested for spatial randomness or analyzed by other clustering approaches, and descriptors such as the absolute number of fluorophores per cluster are now quantifiable, which we validate with simulated data and experimental data with known ground truth. Using MBC, we confirm that the adapter protein, the linker for activation of T cells, is clustered at the T cell immunological synapse. A model-based correction (MBC) algorithm offers fast and accurate correction of multiple-blinking artifacts in PALM data. MBC outperforms other algorithms in both speed and accuracy and improves quantitative downstream image analysis.},
  archive      = {J_NMETH},
  author       = {Jensen, Louis G. and Hoh, Tjun Yee and Williamson, David J. and Griffié, Juliette and Sage, Daniel and Rubin-Delanchy, Patrick and Owen, Dylan M.},
  doi          = {10.1038/s41592-022-01463-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {594-602},
  shortjournal = {Nat. Meth.},
  title        = {Correction of multiple-blinking artifacts in photoactivated localization microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Precision size and refractive index analysis of weakly
scattering nanoparticles in polydispersions. <em>NMETH</em>,
<em>19</em>(5), 586–593. (<a
href="https://doi.org/10.1038/s41592-022-01460-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterization of the size and material properties of particles in liquid suspensions is in very high demand, for example, in the analysis of colloidal samples or of bodily fluids such as urine or blood plasma. However, existing methods are limited in their ability to decipher the constituents of realistic samples. Here we introduce iNTA as a new method that combines interferometric detection of scattering with nanoparticle tracking analysis to reach unprecedented sensitivity and precision in determining the size and refractive index distributions of nanoparticles in suspensions. After benchmarking iNTA with samples of colloidal gold, we present its remarkable ability to resolve the constituents of various multicomponent and polydisperse samples of known origin. Furthermore, we showcase the method by elucidating the refractive index and size distributions of extracellular vesicles from Leishmania parasites and human urine. The current performance of iNTA already enables advances in several important applications, but we also discuss possible improvements. iNTA combines interferometric detection of scattering with nanoparticle tracking analysis for determining the size and refractive index distributions of nanoparticles in suspension with high sensitivity and precision.},
  archive      = {J_NMETH},
  author       = {Kashkanova, Anna D. and Blessing, Martin and Gemeinhardt, André and Soulat, Didier and Sandoghdar, Vahid},
  doi          = {10.1038/s41592-022-01460-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {586-593},
  shortjournal = {Nat. Meth.},
  title        = {Precision size and refractive index analysis of weakly scattering nanoparticles in polydispersions},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sub-3-å cryo-EM structure of RNA enabled by engineered
homomeric self-assembly. <em>NMETH</em>, <em>19</em>(5), 576–585. (<a
href="https://doi.org/10.1038/s41592-022-01455-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-resolution structural studies are essential for understanding the folding and function of diverse RNAs. Herein, we present a nanoarchitectural engineering strategy for efficient structural determination of RNA-only structures using single-particle cryogenic electron microscopy (cryo-EM). This strategy—ROCK (RNA oligomerization-enabled cryo-EM via installing kissing loops)—involves installing kissing-loop sequences onto the functionally nonessential stems of RNAs for homomeric self-assembly into closed rings with multiplied molecular weights and mitigated structural flexibility. ROCK enables cryo-EM reconstruction of the Tetrahymena group I intron at 2.98-Å resolution overall (2.85 Å for the core), allowing de novo model building of the complete RNA, including the previously unknown peripheral domains. ROCK is further applied to two smaller RNAs—the Azoarcus group I intron and the FMN riboswitch, revealing the conformational change of the former and the bound ligand in the latter. ROCK holds promise to greatly facilitate the use of cryo-EM in RNA structural studies. ROCK (RNA oligomerization-enabled cryo-EM via installing kissing loops) enables improved single-particle cryo-EM of RNAs. ROCK was used to generate high-quality structures of three diverse RNAs, including the Tetrahymena group I intron.},
  archive      = {J_NMETH},
  author       = {Liu, Di and Thélot, François A. and Piccirilli, Joseph A. and Liao, Maofu and Yin, Peng},
  doi          = {10.1038/s41592-022-01455-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {576-585},
  shortjournal = {Nat. Meth.},
  title        = {Sub-3-Å cryo-EM structure of RNA enabled by engineered homomeric self-assembly},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Alignment and integration of spatial transcriptomics data.
<em>NMETH</em>, <em>19</em>(5), 567–575. (<a
href="https://doi.org/10.1038/s41592-022-01459-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics (ST) measures mRNA expression across thousands of spots from a tissue slice while recording the two-dimensional (2D) coordinates of each spot. We introduce probabilistic alignment of ST experiments (PASTE), a method to align and integrate ST data from multiple adjacent tissue slices. PASTE computes pairwise alignments of slices using an optimal transport formulation that models both transcriptional similarity and physical distances between spots. PASTE further combines pairwise alignments to construct a stacked 3D alignment of a tissue. Alternatively, PASTE can integrate multiple ST slices into a single consensus slice. We show that PASTE accurately aligns spots across adjacent slices in both simulated and real ST data, demonstrating the advantages of using both transcriptional similarity and spatial information. We further show that the PASTE integrated slice improves the identification of cell types and differentially expressed genes compared with existing approaches that either analyze single ST slices or ignore spatial information. PASTE aligns and integrates spatial transcriptomics data generated from adjacent tissue slices by leveraging their transcriptomic similarity and spatial coordinates, which ultimately increases the power for downstream analysis.},
  archive      = {J_NMETH},
  author       = {Zeira, Ron and Land, Max and Strzalkowski, Alexander and Raphael, Benjamin J.},
  doi          = {10.1038/s41592-022-01459-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {567-575},
  shortjournal = {Nat. Meth.},
  title        = {Alignment and integration of spatial transcriptomics data},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Molecular spikes: A gold standard for single-cell RNA
counting. <em>NMETH</em>, <em>19</em>(5), 560–566. (<a
href="https://doi.org/10.1038/s41592-022-01446-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell sequencing methods rely on molecule-counting strategies to account for amplification biases, yet no experimental strategy to evaluate counting performance exists. Here, we introduce molecular spikes—RNA spike-ins containing built-in unique molecular identifiers (UMIs) that we use to identify critical experimental and computational conditions for accurate RNA counting in single-cell RNA-sequencing (scRNA-seq). Using molecular spikes, we uncovered impaired RNA counting in methods that were not informative for cellular RNA abundances due to inflated UMI counts. We further leverage molecular spikes to improve estimates of total endogenous RNA amounts in cells, and introduce a strategy to correct experiments with impaired RNA counting. The molecular spikes and the accompanying R package UMIcountR ( https://github.com/cziegenhain/UMIcountR ) will improve the validation of new methods, better estimate and adjust for cellular mRNA amounts and enable more indepth characterization of RNA counting in scRNA-seq. This work presents an RNA spike-in that can be used to improve RNA counting in single-cell RNA-sequencing (scRNA-seq) analysis, as well as to report the performance of scRNA-seq methods.},
  archive      = {J_NMETH},
  author       = {Ziegenhain, Christoph and Hendriks, Gert-Jan and Hagemann-Jensen, Michael and Sandberg, Rickard},
  doi          = {10.1038/s41592-022-01446-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {560-566},
  shortjournal = {Nat. Meth.},
  title        = {Molecular spikes: A gold standard for single-cell RNA counting},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fluorogenic DNA-PAINT for faster, low-background
super-resolution imaging. <em>NMETH</em>, <em>19</em>(5), 554–559. (<a
href="https://doi.org/10.1038/s41592-022-01464-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA-based points accumulation for imaging in nanoscale topography (DNA-PAINT) is a powerful super-resolution microscopy method that can acquire high-fidelity images at nanometer resolution. It suffers, however, from high background and slow imaging speed, both of which can be attributed to the presence of unbound fluorophores in solution. Here we present two-color fluorogenic DNA-PAINT, which uses improved imager probe and docking strand designs to solve these problems. These self-quenching single-stranded DNA probes are conjugated with a fluorophore and quencher at the terminals, which permits an increase in fluorescence by up to 57-fold upon binding and unquenching. In addition, the engineering of base pair mismatches between the fluorogenic imager probes and docking strands allowed us to achieve both high fluorogenicity and the fast binding kinetics required for fast imaging. We demonstrate a 26-fold increase in imaging speed over regular DNA-PAINT and show that our new implementation enables three-dimensional super-resolution DNA-PAINT imaging without optical sectioning. Two-color fluorogenic DNA-PAINT introduces self-quenching, kinetics-optimized probe designs. This approach improves imaging speed 26-fold and eliminates the need for optical sectioning.},
  archive      = {J_NMETH},
  author       = {Chung, Kenny K. H. and Zhang, Zhao and Kidd, Phylicia and Zhang, Yongdeng and Williams, Nathan D. and Rollins, Bennett and Yang, Yang and Lin, Chenxiang and Baddeley, David and Bewersdorf, Joerg},
  doi          = {10.1038/s41592-022-01464-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {554-559},
  shortjournal = {Nat. Meth.},
  title        = {Fluorogenic DNA-PAINT for faster, low-background super-resolution imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). NEAT-seq: Simultaneous profiling of intra-nuclear proteins,
chromatin accessibility and gene expression in single cells.
<em>NMETH</em>, <em>19</em>(5), 547–553. (<a
href="https://doi.org/10.1038/s41592-022-01461-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this work, we describe NEAT-seq (sequencing of nuclear protein epitope abundance, chromatin accessibility and the transcriptome in single cells), enabling interrogation of regulatory mechanisms spanning the central dogma. We apply this technique to profile CD4 memory T cells using a panel of master transcription factors (TFs) that drive T cell subsets and identify examples of TFs with regulatory activity gated by transcription, translation and regulation of chromatin binding. We also link a noncoding genome-wide association study single-nucleotide polymorphism (SNP) within a GATA motif to a putative target gene, using NEAT-seq data to internally validate SNP impact on GATA3 regulation. NEAT-seq offers a tri-omics tool for profiling nuclear protein abundance along with ATAC-seq and RNA-seq in single cells.},
  archive      = {J_NMETH},
  author       = {Chen, Amy F. and Parks, Benjamin and Kathiria, Arwa S. and Ober-Reynolds, Benjamin and Goronzy, Jorg J. and Greenleaf, William J.},
  doi          = {10.1038/s41592-022-01461-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {547-553},
  shortjournal = {Nat. Meth.},
  title        = {NEAT-seq: Simultaneous profiling of intra-nuclear proteins, chromatin accessibility and gene expression in single cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Museum of spatial transcriptomics. <em>NMETH</em>,
<em>19</em>(5), 534–546. (<a
href="https://doi.org/10.1038/s41592-022-01409-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The function of many biological systems, such as embryos, liver lobules, intestinal villi, and tumors, depends on the spatial organization of their cells. In the past decade, high-throughput technologies have been developed to quantify gene expression in space, and computational methods have been developed that leverage spatial gene expression data to identify genes with spatial patterns and to delineate neighborhoods within tissues. To comprehensively document spatial gene expression technologies and data-analysis methods, we present a curated review of literature on spatial transcriptomics dating back to 1987, along with a thorough analysis of trends in the field, such as usage of experimental techniques, species, tissues studied, and computational approaches used. Our Review places current methods in a historical context, and we derive insights about the field that can guide current research strategies. A companion supplement offers a more detailed look at the technologies and methods analyzed: https://pachterlab.github.io/LP_2021/ . This work presents an overview of the evolution of spatial transcriptomics and highlights recent efforts in method developments in this space.},
  archive      = {J_NMETH},
  author       = {Moses, Lambda and Pachter, Lior},
  doi          = {10.1038/s41592-022-01409-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {534-546},
  shortjournal = {Nat. Meth.},
  title        = {Museum of spatial transcriptomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). NeuroMechFly: An integrative simulation testbed for studying
drosophila behavioral control. <em>NMETH</em>, <em>19</em>(5), 532–533.
(<a href="https://doi.org/10.1038/s41592-022-01411-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuromechanical simulations enable the study of how interactions between organisms and their physical surroundings give rise to behavior. NeuroMechFly is an open-source neuromechanical model of adult Drosophila, with data-driven morphological biorealism that enables a synergistic cross-talk between computational and experimental neuroscience.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01411-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {532-533},
  shortjournal = {Nat. Meth.},
  title        = {NeuroMechFly: An integrative simulation testbed for studying drosophila behavioral control},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Novel spike-ins for single-cell sequencing enable
ground-truth RNA counting. <em>NMETH</em>, <em>19</em>(5), 530–531. (<a
href="https://doi.org/10.1038/s41592-022-01456-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Counting of RNA molecules using unique molecular identifiers (UMIs) is ubiquitous in single-cell sequencing. Here, we introduce molecular spikes, a new type of RNA spike-ins with in-built UMIs. These versatile molecular spikes have many uses in experimental and computational method development and routine biological applications.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01456-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {530-531},
  shortjournal = {Nat. Meth.},
  title        = {Novel spike-ins for single-cell sequencing enable ground-truth RNA counting},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Imaging cleared tissues made easy. <em>NMETH</em>,
<em>19</em>(5), 527–529. (<a
href="https://doi.org/10.1038/s41592-022-01424-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A flexible open-top light-sheet microscope has been developed that can perform deep three-dimensional imaging on all clearing protocols with low and high optical resolution.},
  archive      = {J_NMETH},
  author       = {Kanatani, Shigeaki and Uhlén, Per},
  doi          = {10.1038/s41592-022-01424-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {527-529},
  shortjournal = {Nat. Meth.},
  title        = {Imaging cleared tissues made easy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). 3D reconstruction of spatial expression. <em>NMETH</em>,
<em>19</em>(5), 526–527. (<a
href="https://doi.org/10.1038/s41592-022-01476-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A novel approach to probabilistically align adjacent multiple tissue slices from spatially resolved transcriptomics data provides unprecedented depth for the investigation of tissue architecture and paves the way for new developments in 3D spatial analytics.},
  archive      = {J_NMETH},
  author       = {Lin, Yingxin and Yang, Jean Y. H.},
  doi          = {10.1038/s41592-022-01476-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {526-527},
  shortjournal = {Nat. Meth.},
  title        = {3D reconstruction of spatial expression},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Conservation genomics in practice. <em>NMETH</em>,
<em>19</em>(5), 522–525. (<a
href="https://doi.org/10.1038/s41592-022-01477-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Securing biodiversity takes many measures, including the involvement of Indigenous peoples.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01477-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {522-525},
  shortjournal = {Nat. Meth.},
  title        = {Conservation genomics in practice},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Reporting t cell proliferation. <em>NMETH</em>,
<em>19</em>(5), 521. (<a
href="https://doi.org/10.1038/s41592-022-01505-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A genetic tracing approach follows the division history of cells in vivo.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01505-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {521},
  shortjournal = {Nat. Meth.},
  title        = {Reporting t cell proliferation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Quantifying radiation damage. <em>NMETH</em>,
<em>19</em>(5), 520. (<a
href="https://doi.org/10.1038/s41592-022-01504-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bnet provides a measure of radiation damage suffered by protein crystals during X-ray crystallography.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01504-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {520},
  shortjournal = {Nat. Meth.},
  title        = {Quantifying radiation damage},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Cell-free DNA cues for gene expression. <em>NMETH</em>,
<em>19</em>(5), 519. (<a
href="https://doi.org/10.1038/s41592-022-01503-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell-free DNA ‘fragmentomics’ empowers cancer detection and classification.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01503-5},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {519},
  shortjournal = {Nat. Meth.},
  title        = {Cell-free DNA cues for gene expression},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Two-photon imaging in freely behaving mice. <em>NMETH</em>,
<em>19</em>(5), 518. (<a
href="https://doi.org/10.1038/s41592-022-01502-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The MINI2P is a two-photon miniscope that enables robust calcium imaging during mouse behavior.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01502-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {518},
  shortjournal = {Nat. Meth.},
  title        = {Two-photon imaging in freely behaving mice},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022s). Spatial CRISPR screens in tumors. <em>NMETH</em>,
<em>19</em>(5), 517. (<a
href="https://doi.org/10.1038/s41592-022-01501-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Perturb-map combines multiplex imaging and spatial transcriptomics to dissect complex biological functions from CRISPR screens in tumor tissues.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01501-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {517},
  shortjournal = {Nat. Meth.},
  title        = {Spatial CRISPR screens in tumors},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Viv: Multiscale visualization of high-resolution multiplexed
bioimaging data on the web. <em>NMETH</em>, <em>19</em>(5), 515–516. (<a
href="https://doi.org/10.1038/s41592-022-01482-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Manz, Trevor and Gold, Ilan and Patterson, Nathan Heath and McCallum, Chuck and Keller, Mark S. and Herr, Bruce W. and Börner, Katy and Spraggins, Jeffrey M. and Gehlenborg, Nils},
  doi          = {10.1038/s41592-022-01482-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {515-516},
  shortjournal = {Nat. Meth.},
  title        = {Viv: Multiscale visualization of high-resolution multiplexed bioimaging data on the web},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The simularium viewer: An interactive online tool for
sharing spatiotemporal biological models. <em>NMETH</em>,
<em>19</em>(5), 513–515. (<a
href="https://doi.org/10.1038/s41592-022-01442-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Lyons, Blair and Isaac, Eric and Choi, Na Hyung and Do, Thao P. and Domingus, Justin and Iwasa, Janet and Leonard, Andrew and Riel-Mehan, Megan and Rodgers, Emily and Schaefbauer, Lisa and Toloudis, Daniel and Waltner, Olivia and Wilhelm, Lyndsay and Johnson, Graham T.},
  doi          = {10.1038/s41592-022-01442-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {513-515},
  shortjournal = {Nat. Meth.},
  title        = {The simularium viewer: An interactive online tool for sharing spatiotemporal biological models},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An online notebook resource for reproducible inference,
analysis and publication of gene regulatory networks. <em>NMETH</em>,
<em>19</em>(5), 511–513. (<a
href="https://doi.org/10.1038/s41592-022-01479-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Ben Guebila, Marouen and Weighill, Deborah and Lopes-Ramos, Camila M. and Burkholz, Rebekka and Pop, Romana T. and Palepu, Kalyan and Shapoval, Mia and Fagny, Maud and Schlauch, Daniel and Glass, Kimberly and Altenbuchinger, Michael and Kuijjer, Marieke L. and Platig, John and Quackenbush, John},
  doi          = {10.1038/s41592-022-01479-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {511-513},
  shortjournal = {Nat. Meth.},
  title        = {An online notebook resource for reproducible inference, analysis and publication of gene regulatory networks},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022x). Vahid sandoghdar. <em>NMETH</em>, <em>19</em>(5), 509. (<a
href="https://doi.org/10.1038/s41592-022-01493-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {From a physicist who explores biology’s unknowns, greater precision by linking two methods.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01493-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {509},
  shortjournal = {Nat. Meth.},
  title        = {Vahid sandoghdar},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). New happenings at nature methods. <em>NMETH</em>,
<em>19</em>(5), 507. (<a
href="https://doi.org/10.1038/s41592-022-01499-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It’s been a busy start to 2022 in the editorial offices of Nature Methods. We summarize several developments here: new content types, editorial initiatives, travel and outreach, and recent Focus issues.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01499-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {507},
  shortjournal = {Nat. Meth.},
  title        = {New happenings at nature methods},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022z). When peace arrives. <em>NMETH</em>, <em>19</em>(5), 510.
(<a href="https://doi.org/10.1038/s41592-022-01489-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists in Ukraine share their current reality and post-war plans.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01489-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {5},
  pages        = {510},
  shortjournal = {Nat. Meth.},
  title        = {When peace arrives},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Author correction: A genetically encoded sensor for in vivo
imaging of orexin neuropeptides. <em>NMETH</em>, <em>19</em>(4), 505.
(<a href="https://doi.org/10.1038/s41592-022-01449-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Duffet, Loïc and Kosar, Seher and Panniello, Mariangela and Viberti, Bianca and Bracey, Edward and Zych, Anna D. and Radoux-Mergault, Arthur and Zhou, Xuehan and Dernic, Jan and Ravotto, Luca and Tsai, Yuan-Chen and Figueiredo, Marta and Tyagarajan, Shiva K. and Weber, Bruno and Stoeber, Miriam and Gogolla, Nadine and Schmidt, Markus H. and Adamantidis, Antoine R. and Fellin, Tommaso and Burdakov, Denis and Patriarchi, Tommaso},
  doi          = {10.1038/s41592-022-01449-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {505},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A genetically encoded sensor for in vivo imaging of orexin neuropeptides},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Fast and flexible analysis of linked
microbiome data with mako. <em>NMETH</em>, <em>19</em>(4), 505. (<a
href="https://doi.org/10.1038/s41592-022-01462-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Röttjers, Lisa and Faust, Karoline},
  doi          = {10.1038/s41592-022-01462-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {505},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Fast and flexible analysis of linked microbiome data with mako},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-animal pose estimation, identification and tracking
with DeepLabCut. <em>NMETH</em>, <em>19</em>(4), 496–504. (<a
href="https://doi.org/10.1038/s41592-022-01443-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimating the pose of multiple animals is a challenging computer vision problem: frequent interactions cause occlusions and complicate the association of detected keypoints to the correct individuals, as well as having highly similar looking animals that interact more closely than in typical multi-human scenarios. To take up this challenge, we build on DeepLabCut, an open-source pose estimation toolbox, and provide high-performance animal assembly and tracking—features required for multi-animal scenarios. Furthermore, we integrate the ability to predict an animal’s identity to assist tracking (in case of occlusions). We illustrate the power of this framework with four datasets varying in complexity, which we release to serve as a benchmark for future algorithm development. DeepLabCut is extended to enable multi-animal pose estimation, animal identification and tracking, thereby enabling the analysis of social behaviors.},
  archive      = {J_NMETH},
  author       = {Lauer, Jessy and Zhou, Mu and Ye, Shaokai and Menegas, William and Schneider, Steffen and Nath, Tanmay and Rahman, Mohammed Mostafizur and Di Santo, Valentina and Soberanes, Daniel and Feng, Guoping and Murthy, Venkatesh N. and Lauder, George and Dulac, Catherine and Mathis, Mackenzie Weygandt and Mathis, Alexander},
  doi          = {10.1038/s41592-022-01443-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {496-504},
  shortjournal = {Nat. Meth.},
  title        = {Multi-animal pose estimation, identification and tracking with DeepLabCut},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). SLEAP: A deep learning system for multi-animal pose
tracking. <em>NMETH</em>, <em>19</em>(4), 486–495. (<a
href="https://doi.org/10.1038/s41592-022-01426-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The desire to understand how the brain generates and patterns behavior has driven rapid methodological innovation in tools to quantify natural animal behavior. While advances in deep learning and computer vision have enabled markerless pose estimation in individual animals, extending these to multiple animals presents unique challenges for studies of social behaviors or animals in their natural environments. Here we present Social LEAP Estimates Animal Poses (SLEAP), a machine learning system for multi-animal pose tracking. This system enables versatile workflows for data labeling, model training and inference on previously unseen data. SLEAP features an accessible graphical user interface, a standardized data model, a reproducible configuration system, over 30 model architectures, two approaches to part grouping and two approaches to identity tracking. We applied SLEAP to seven datasets across flies, bees, mice and gerbils to systematically evaluate each approach and architecture, and we compare it with other existing approaches. SLEAP achieves greater accuracy and speeds of more than 800 frames per second, with latencies of less than 3.5 ms at full 1,024 × 1,024 image resolution. This makes SLEAP usable for real-time applications, which we demonstrate by controlling the behavior of one animal on the basis of the tracking and detection of social interactions with another animal. SLEAP is a versatile deep learning-based multi-animal pose-tracking tool designed to work on videos of diverse animals, including during social behavior.},
  archive      = {J_NMETH},
  author       = {Pereira, Talmo D. and Tabris, Nathaniel and Matsliah, Arie and Turner, David M. and Li, Junyu and Ravindranath, Shruthi and Papadoyannis, Eleni S. and Normand, Edna and Deutsch, David S. and Wang, Z. Yan and McKenzie-Smith, Grace C. and Mitelut, Catalin C. and Castro, Marielisa Diez and D’Uva, John and Kislin, Mikhail and Sanes, Dan H. and Kocher, Sarah D. and Wang, Samuel S.-H. and Falkner, Annegret L. and Shaevitz, Joshua W. and Murthy, Mala},
  doi          = {10.1038/s41592-022-01426-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {486-495},
  shortjournal = {Nat. Meth.},
  title        = {SLEAP: A deep learning system for multi-animal pose tracking},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). HYBRiD: Hydrogel-reinforced DISCO for clearing mammalian
bodies. <em>NMETH</em>, <em>19</em>(4), 479–485. (<a
href="https://doi.org/10.1038/s41592-022-01427-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recent development of solvent- and polymer-based brain-clearing techniques has advanced our ability to visualize the mammalian nervous system in three dimensions. However, it remains challenging to image the mammalian body en bloc. Here we developed HYBRiD (hydrogel-based reinforcement of three-dimensional imaging solvent-cleared organs (DISCO)), by recombining components of organic- and polymer-based clearing pipelines. We achieved high transparency and protein retention, as well as compatibility with direct fluorescent imaging and immunostaining in cleared mammalian bodies. Using parvalbumin- and somatostatin-Cre models, we demonstrated the utility of HYBRiD for whole-body imaging of genetically encoded fluorescent reporters without antibody enhancement of signals in newborn and juvenile mice. Using K18-hACE2 transgenic mice, HYBRiD enabled perfusion-free clearing and visualization of SARS-CoV-2 infection in a whole mouse chest, revealing macroscopic and microscopic features of viral pathology in the same sample. HYBRiD offers a simple and universal solution to visualize large heterogeneous body parts or entire animals for basic and translational research. HYBRiD is a tissue-clearing approach that combines elements of solvent-based and aqueous, hydrogel-based methods, thereby achieving high transparency while maintaining fluorescence.},
  archive      = {J_NMETH},
  author       = {Nudell, Victoria and Wang, Yu and Pang, Zhengyuan and Lal, Neeraj K. and Huang, Min and Shaabani, Namir and Kanim, Wesam and Teijaro, John and Maximov, Anton and Ye, Li},
  doi          = {10.1038/s41592-022-01427-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {479-485},
  shortjournal = {Nat. Meth.},
  title        = {HYBRiD: Hydrogel-reinforced DISCO for clearing mammalian bodies},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Detecting and correcting false transients in calcium
imaging. <em>NMETH</em>, <em>19</em>(4), 470–478. (<a
href="https://doi.org/10.1038/s41592-022-01422-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Population recordings of calcium activity are a major source of insight into neural function. Large datasets require automated processing, but this can introduce errors that are difficult to detect. Here we show that popular time course-estimation algorithms often contain substantial misattribution errors affecting 10–20\% of transients. Misattribution, in which fluorescence is ascribed to the wrong cell, arises when overlapping cells and processes are imperfectly defined or not identified. To diagnose misattribution, we develop metrics and visualization tools for evaluating large datasets. To correct time courses, we introduce a robust estimator that explicitly accounts for contaminating signals. In one hippocampal dataset, removing contamination reduced the number of place cells by 15\%, and 19\% of place fields shifted by over 10 cm. Our methods are compatible with other cell-finding techniques, empowering users to diagnose and correct a potentially widespread problem that could alter scientific conclusions. SEUDO is a tool for detecting and correcting errors introduced by automated processing of calcium imaging data.},
  archive      = {J_NMETH},
  author       = {Gauthier, Jeffrey L. and Koay, Sue Ann and Nieh, Edward H. and Tank, David W. and Pillow, Jonathan W. and Charles, Adam S.},
  doi          = {10.1038/s41592-022-01422-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {470-478},
  shortjournal = {Nat. Meth.},
  title        = {Detecting and correcting false transients in calcium imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DaXi—high-resolution, large imaging volume and multi-view
single-objective light-sheet microscopy. <em>NMETH</em>, <em>19</em>(4),
461–469. (<a href="https://doi.org/10.1038/s41592-022-01417-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The promise of single-objective light-sheet microscopy is to combine the convenience of standard single-objective microscopes with the speed, coverage, resolution and gentleness of light-sheet microscopes. We present DaXi, a single-objective light-sheet microscope design based on oblique plane illumination that achieves: (1) a wider field of view and high-resolution imaging via a custom remote focusing objective; (2) fast volumetric imaging over larger volumes without compromising image quality or necessitating tiled acquisition; (3) fuller image coverage for large samples via multi-view imaging and (4) higher throughput multi-well imaging via remote coverslip placement. Our instrument achieves a resolution of 450 nm laterally and 2 μm axially over an imaging volume of 3,000 × 800 × 300 μm. We demonstrate the speed, field of view, resolution and versatility of our instrument by imaging various systems, including Drosophila egg chamber development, zebrafish whole-brain activity and zebrafish embryonic development – up to nine embryos at a time. The DaXi single-objective light-sheet microscope achieves fast, high-quality imaging of large volumes. DaXi’s design allows increased scanning range without sacrificing imaging speed or quality, multiview imaging and versatile sample mounting.},
  archive      = {J_NMETH},
  author       = {Yang, Bin and Lange, Merlin and Millett-Sikking, Alfred and Zhao, Xiang and Bragantini, Jordão and VijayKumar, Shruthi and Kamb, Mason and Gómez-Sjöberg, Rafael and Solak, Ahmet Can and Wang, Wanpeng and Kobayashi, Hirofumi and McCarroll, Matthew N. and Whitehead, Lachlan W. and Fiolka, Reto P. and Kornberg, Thomas B. and York, Andrew G. and Royer, Loic A.},
  doi          = {10.1038/s41592-022-01417-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {461-469},
  shortjournal = {Nat. Meth.},
  title        = {DaXi—high-resolution, large imaging volume and multi-view single-objective light-sheet microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antigen identification and high-throughput interaction
mapping by reprogramming viral entry. <em>NMETH</em>, <em>19</em>(4),
449–460. (<a href="https://doi.org/10.1038/s41592-022-01436-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering immune recognition is critical for understanding a broad range of diseases and for the development of effective vaccines and immunotherapies. Efforts to do so are limited by a lack of technologies capable of simultaneously capturing the complexity of adaptive immunoreceptor repertoires and the landscape of potential antigens. To address this, we present receptor–antigen pairing by targeted retroviruses, which combines viral pseudotyping and molecular engineering approaches to enable one-pot library-on-library interaction screens by displaying antigens on the surface of lentiviruses and encoding their identity in the viral genome. Antigen-specific viral infection of cell lines expressing human T or B cell receptors allows readout of both antigen and receptor identities via single-cell sequencing. The resulting system is modular, scalable and compatible with any cell type. These techniques provide a suite of tools for targeted viral entry, molecular engineering and interaction screens with broad potential applications. RAPTR, a lentivirus surface display-based library-on-library screening method to decipher antigen and immunoreceptor identities},
  archive      = {J_NMETH},
  author       = {Dobson, Connor S. and Reich, Anna N. and Gaglione, Stephanie and Smith, Blake E. and Kim, Ellen J. and Dong, Jiayi and Ronsard, Larance and Okonkwo, Vintus and Lingwood, Daniel and Dougan, Michael and Dougan, Stephanie K. and Birnbaum, Michael E.},
  doi          = {10.1038/s41592-022-01436-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {449-460},
  shortjournal = {Nat. Meth.},
  title        = {Antigen identification and high-throughput interaction mapping by reprogramming viral entry},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Searching thousands of genomes to classify somatic and
novel structural variants using STIX. <em>NMETH</em>, <em>19</em>(4),
445–448. (<a href="https://doi.org/10.1038/s41592-022-01423-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural variants are associated with cancers and developmental disorders, but challenges with estimating population frequency remain a barrier to prioritizing mutations over inherited variants. In particular, variability in variant calling heuristics and filtering limits the use of current structural variant catalogs. We present STIX, a method that, instead of relying on variant calls, indexes and searches the raw alignments from thousands of samples to enable more comprehensive allele frequency estimation. This work describes a strategy for estimating the population frequency of structural variations by searching the raw alignments of large population sequencing samples using the STIX framework.},
  archive      = {J_NMETH},
  author       = {Chowdhury, Murad and Pedersen, Brent S. and Sedlazeck, Fritz J. and Quinlan, Aaron R. and Layer, Ryan M.},
  doi          = {10.1038/s41592-022-01423-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {445-448},
  shortjournal = {Nat. Meth.},
  title        = {Searching thousands of genomes to classify somatic and novel structural variants using STIX},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CoLoRd: Compressing long reads. <em>NMETH</em>,
<em>19</em>(4), 441–444. (<a
href="https://doi.org/10.1038/s41592-022-01432-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cost of maintaining exabytes of data produced by sequencing experiments every year has become a major issue in today’s genomic research. In spite of the increasing popularity of third-generation sequencing, the existing algorithms for compressing long reads exhibit a minor advantage over the general-purpose gzip. We present CoLoRd, an algorithm able to reduce the size of third-generation sequencing data by an order of magnitude without affecting the accuracy of downstream analyses. CoLoRd achieves high compression rates for long-read sequencing data without affecting downstream analyses.},
  archive      = {J_NMETH},
  author       = {Kokot, Marek and Gudyś, Adam and Li, Heng and Deorowicz, Sebastian},
  doi          = {10.1038/s41592-022-01432-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {441-444},
  shortjournal = {Nat. Meth.},
  title        = {CoLoRd: Compressing long reads},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Critical assessment of metagenome interpretation: The second
round of challenges. <em>NMETH</em>, <em>19</em>(4), 429–440. (<a
href="https://doi.org/10.1038/s41592-022-01431-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evaluating metagenomic software is key for optimizing metagenome interpretation and focus of the Initiative for the Critical Assessment of Metagenome Interpretation (CAMI). The CAMI II challenge engaged the community to assess methods on realistic and complex datasets with long- and short-read sequences, created computationally from around 1,700 new and known genomes, as well as 600 new plasmids and viruses. Here we analyze 5,002 results by 76 program versions. Substantial improvements were seen in assembly, some due to long-read data. Related strains still were challenging for assembly and genome recovery through binning, as was assembly quality for the latter. Profilers markedly matured, with taxon profilers and binners excelling at higher bacterial ranks, but underperforming for viruses and Archaea. Clinical pathogen detection results revealed a need to improve reproducibility. Runtime and memory usage analyses identified efficient programs, including top performers with other metrics. The results identify challenges and guide researchers in selecting methods for analyses. This study presents the results of the second round of the Critical Assessment of Metagenome Interpretation challenges (CAMI II), which is a community-driven effort for comprehensively benchmarking tools for metagenomics data analysis.},
  archive      = {J_NMETH},
  author       = {Meyer, Fernando and Fritz, Adrian and Deng, Zhi-Luo and Koslicki, David and Lesker, Till Robin and Gurevich, Alexey and Robertson, Gary and Alser, Mohammed and Antipov, Dmitry and Beghini, Francesco and Bertrand, Denis and Brito, Jaqueline J. and Brown, C. Titus and Buchmann, Jan and Buluç, Aydin and Chen, Bo and Chikhi, Rayan and Clausen, Philip T. L. C. and Cristian, Alexandru and Dabrowski, Piotr Wojciech and Darling, Aaron E. and Egan, Rob and Eskin, Eleazar and Georganas, Evangelos and Goltsman, Eugene and Gray, Melissa A. and Hansen, Lars Hestbjerg and Hofmeyr, Steven and Huang, Pingqin and Irber, Luiz and Jia, Huijue and Jørgensen, Tue Sparholt and Kieser, Silas D. and Klemetsen, Terje and Kola, Axel and Kolmogorov, Mikhail and Korobeynikov, Anton and Kwan, Jason and LaPierre, Nathan and Lemaitre, Claire and Li, Chenhao and Limasset, Antoine and Malcher-Miranda, Fabio and Mangul, Serghei and Marcelino, Vanessa R. and Marchet, Camille and Marijon, Pierre and Meleshko, Dmitry and Mende, Daniel R. and Milanese, Alessio and Nagarajan, Niranjan and Nissen, Jakob and Nurk, Sergey and Oliker, Leonid and Paoli, Lucas and Peterlongo, Pierre and Piro, Vitor C. and Porter, Jacob S. and Rasmussen, Simon and Rees, Evan R. and Reinert, Knut and Renard, Bernhard and Robertsen, Espen Mikal and Rosen, Gail L. and Ruscheweyh, Hans-Joachim and Sarwal, Varuni and Segata, Nicola and Seiler, Enrico and Shi, Lizhen and Sun, Fengzhu and Sunagawa, Shinichi and Sørensen, Søren Johannes and Thomas, Ashleigh and Tong, Chengxuan and Trajkovski, Mirko and Tremblay, Julien and Uritskiy, Gherman and Vicedomini, Riccardo and Wang, Zhengyang and Wang, Ziye and Wang, Zhong and Warren, Andrew and Willassen, Nils Peder and Yelick, Katherine and You, Ronghui and Zeller, Georg and Zhao, Zhengqiao and Zhu, Shanfeng and Zhu, Jie and Garrido-Oter, Ruben and Gastmeier, Petra and Hacquard, Stephane and Häußler, Susanne and Khaledi, Ariane and Maechler, Friederike and Mesny, Fantin and Radutoiu, Simona and Schulze-Lefert, Paul and Smit, Nathiana and Strowig, Till and Bremges, Andreas and Sczyrba, Alexander and McHardy, Alice Carolyn},
  doi          = {10.1038/s41592-022-01431-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {429-440},
  shortjournal = {Nat. Meth.},
  title        = {Critical assessment of metagenome interpretation: The second round of challenges},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Human organoid models to study SARS-CoV-2 infection.
<em>NMETH</em>, <em>19</em>(4), 418–428. (<a
href="https://doi.org/10.1038/s41592-022-01453-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is one of the deadliest pandemics in history. SARS-CoV-2 not only infects the respiratory tract, but also causes damage to many organs. Organoids, which can self-renew and recapitulate the various physiology of different organs, serve as powerful platforms to model COVID-19. In this Perspective, we overview the current effort to apply both human pluripotent stem cell-derived organoids and adult organoids to study SARS-CoV-2 tropism, host response and immune cell-mediated host damage, and perform drug discovery and vaccine development. We summarize the technologies used in organoid-based COVID-19 research, discuss the remaining challenges and provide future perspectives in the application of organoid models to study SARS-CoV-2 and future emerging viruses. This Perspective discusses the use of organoid technology, its current limitations and unmet needs in the context of SARS-CoV-2 research.},
  archive      = {J_NMETH},
  author       = {Han, Yuling and Yang, Liuliu and Lacko, Lauretta A. and Chen, Shuibing},
  doi          = {10.1038/s41592-022-01453-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {418-428},
  shortjournal = {Nat. Meth.},
  title        = {Human organoid models to study SARS-CoV-2 infection},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). HYBRiD: A simple method for clearing and visualizing
mammalian bodies. <em>NMETH</em>, <em>19</em>(4), 416–417. (<a
href="https://doi.org/10.1038/s41592-022-01410-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A simple and affordable technique passively clears and images whole mammalian bodies or large tissues. This technique is compatible with the use of endogeneous fluoresent proteins, without the loss of signal associated with other existing methods for whole-animal clearing.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01410-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {416-417},
  shortjournal = {Nat. Meth.},
  title        = {HYBRiD: A simple method for clearing and visualizing mammalian bodies},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DaXi: Pushing the limits of single-objective light-sheet
microscopy. <em>NMETH</em>, <em>19</em>(4), 414–415. (<a
href="https://doi.org/10.1038/s41592-022-01420-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new single-objective light-sheet microscope has been developed that uses novel optics and imaging protocols to increase resolution without compromising imaging speed and volume.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01420-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {414-415},
  shortjournal = {Nat. Meth.},
  title        = {DaXi: Pushing the limits of single-objective light-sheet microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CAMI II: Identifying best practices and issues for
metagenomics software. <em>NMETH</em>, <em>19</em>(4), 412–413. (<a
href="https://doi.org/10.1038/s41592-022-01419-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By providing challenges to the metagenomics community based on complex and realistic metagenome benchmark datasets, CAMI — the community-driven initiative for the Critical Assessment of Metagenome Interpretation — has created a comprehensive assessment of the performance of metagenomics software for common analyses. As part of its second contest, CAMI II, it evaluates ~5,000 submissions from 76 software programs and their different versions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01419-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {412-413},
  shortjournal = {Nat. Meth.},
  title        = {CAMI II: Identifying best practices and issues for metagenomics software},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Tracking together: Estimating social poses. <em>NMETH</em>,
<em>19</em>(4), 410–411. (<a
href="https://doi.org/10.1038/s41592-022-01452-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two new toolkits that leverage deep-learning approaches can track the positions of multiple animals and estimate poses in different experimental paradigms.},
  archive      = {J_NMETH},
  author       = {Agezo, Sena and Berman, Gordon J.},
  doi          = {10.1038/s41592-022-01452-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {410-411},
  shortjournal = {Nat. Meth.},
  title        = {Tracking together: Estimating social poses},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Engineered retroviruses map ligand–receptor interactions.
<em>NMETH</em>, <em>19</em>(4), 408–410. (<a
href="https://doi.org/10.1038/s41592-022-01437-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered viral entry combined with single-cell sequencing technology makes it possible to identify specific ligand–receptor interactions in a high-throughput manner.},
  archive      = {J_NMETH},
  author       = {Wang, Yuqian and Wang, Zhe and Li, Guideng},
  doi          = {10.1038/s41592-022-01437-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {408-410},
  shortjournal = {Nat. Meth.},
  title        = {Engineered retroviruses map ligand–receptor interactions},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022l). Lessons from the global south’s fight against COVID-19.
<em>NMETH</em>, <em>19</em>(4), 403–407. (<a
href="https://doi.org/10.1038/s41592-022-01439-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Against the backdrop of its many battles with infectious disease, the Global South applies networking and creative grit to face COVID-19.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01439-w},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {403-407},
  shortjournal = {Nat. Meth.},
  title        = {Lessons from the global south’s fight against COVID-19},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022p). Sequencing RNA isoforms in brain tissue. <em>NMETH</em>,
<em>19</em>(4), 402. (<a
href="https://doi.org/10.1038/s41592-022-01473-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nuclei isoform RNA sequencing enables cell-type specific isoform analysis in frozen brain samples.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01473-8},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {402},
  shortjournal = {Nat. Meth.},
  title        = {Sequencing RNA isoforms in brain tissue},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Enhanced protein isoform characterization. <em>NMETH</em>,
<em>19</em>(4), 401. (<a
href="https://doi.org/10.1038/s41592-022-01472-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read proteogenomics helps delineate isoform diversity in full-length proteins.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01472-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {401},
  shortjournal = {Nat. Meth.},
  title        = {Enhanced protein isoform characterization},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Human genealogical histories. <em>NMETH</em>,
<em>19</em>(4), 400. (<a
href="https://doi.org/10.1038/s41592-022-01471-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionary trees link ancient and modern human genomes.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01471-w},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {400},
  shortjournal = {Nat. Meth.},
  title        = {Human genealogical histories},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). In vivo traction force microscopy. <em>NMETH</em>,
<em>19</em>(4), 399. (<a
href="https://doi.org/10.1038/s41592-022-01470-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryogram allows stress measurements during migration of the lateral line primordium in zebrafish embryos.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01470-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {399},
  shortjournal = {Nat. Meth.},
  title        = {In vivo traction force microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Image in vivo, skip the lenses. <em>NMETH</em>,
<em>19</em>(4), 398. (<a
href="https://doi.org/10.1038/s41592-022-01469-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A lensless microscope captures 3D fluorescence images of complex biological tissues.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01469-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {398},
  shortjournal = {Nat. Meth.},
  title        = {Image in vivo, skip the lenses},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Methods for fighting emerging pathogens. <em>NMETH</em>,
<em>19</em>(4), 395–397. (<a
href="https://doi.org/10.1038/s41592-022-01441-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We asked scientists around the globe for their views on the basic research methods, tools and resources needed to fight future emerging pathogenic threats.},
  archive      = {J_NMETH},
  author       = {Alcantara, Luiz Carlos Junior and Amenga-Etego, Lucas and Andersson, Robin and Bhaumik, Moumita and Choi, Young Ki and Decaluwe, Hélène and Geoghegan, Jemma and Haagmans, Bart L. and López, Susana and Mukhtar, Maowia M. and Nelwan, Erni and Rahal, Elias A. and Sato, Kei and Sklan, Ella H. and Fang, Yvonne Su Chuan},
  doi          = {10.1038/s41592-022-01441-2},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {395-397},
  shortjournal = {Nat. Meth.},
  title        = {Methods for fighting emerging pathogens},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Animal models in SARS-CoV-2 research. <em>NMETH</em>,
<em>19</em>(4), 392–394. (<a
href="https://doi.org/10.1038/s41592-022-01447-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This Comment discusses the main animal models that have had a key role in our understanding of the immune and viral dynamics of SARS-CoV-2.},
  archive      = {J_NMETH},
  author       = {Chu, Hin and Chan, Jasper Fuk-Woo and Yuen, Kwok-Yung},
  doi          = {10.1038/s41592-022-01447-w},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {392-394},
  shortjournal = {Nat. Meth.},
  title        = {Animal models in SARS-CoV-2 research},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Response under pressure: Deploying emerging technologies to
understand b-cell-mediated immunity in COVID-19. <em>NMETH</em>,
<em>19</em>(4), 387–391. (<a
href="https://doi.org/10.1038/s41592-022-01450-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Critical technological advances have enabled the rapid investigations into the immune responses elicited by SARS-CoV-2, the pathogen responsible for the COVID-19 pandemic. We discuss the cutting-edge methods used to deconvolve the B-cell responses against this virus and the impact they have had in the ongoing public health crisis.},
  archive      = {J_NMETH},
  author       = {Woodruff, Matthew C. and Nguyen, Doan C. and Faliti, Caterina E. and Saini, Ankur Singh and Lee, F. Eun-Hyung and Sanz, Ignacio},
  doi          = {10.1038/s41592-022-01450-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {387-391},
  shortjournal = {Nat. Meth.},
  title        = {Response under pressure: Deploying emerging technologies to understand B-cell-mediated immunity in COVID-19},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Advances in glycoscience to understand viral infection and
colonization. <em>NMETH</em>, <em>19</em>(4), 384–387. (<a
href="https://doi.org/10.1038/s41592-022-01451-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interactions between carbohydrates and the proteins that bind them (lectins) are often some of the first between a host cell and a viral invader. With its highly glycosylated spike protein, SARS-CoV-2 is no exception. Interrogating glycosylation is vital to understand viral infection, yet it has been a challenge. Improvement in methods ranging from mass spectrometry to glycan arrays and modeling simulations are yielding atomic-level information about the glycans that decorate viruses and host cells alike.},
  archive      = {J_NMETH},
  author       = {Dugan, Amanda E. and Peiffer, Amanda L. and Kiessling, Laura L.},
  doi          = {10.1038/s41592-022-01451-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {384-387},
  shortjournal = {Nat. Meth.},
  title        = {Advances in glycoscience to understand viral infection and colonization},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fighting SARS-CoV-2 with structural biology methods.
<em>NMETH</em>, <em>19</em>(4), 381–383. (<a
href="https://doi.org/10.1038/s41592-022-01448-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-resolution structural information is critical for rapid development of vaccines and therapeutics against emerging human pathogens. Structural biology methods have been at the forefront of research on SARS-CoV-2 since the beginning of the COVID-19 pandemic. These technologies will continue to be powerful tools to fend off future public health threats.},
  archive      = {J_NMETH},
  author       = {Zhang, Jun and Chen, Bing},
  doi          = {10.1038/s41592-022-01448-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {381-383},
  shortjournal = {Nat. Meth.},
  title        = {Fighting SARS-CoV-2 with structural biology methods},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Unlocking capacities of genomics for the COVID-19 response
and future pandemics. <em>NMETH</em>, <em>19</em>(4), 374–380. (<a
href="https://doi.org/10.1038/s41592-022-01444-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During the COVID-19 pandemic, genomics and bioinformatics have emerged as essential public health tools. The genomic data acquired using these methods have supported the global health response, facilitated the development of testing methods and allowed the timely tracking of novel SARS-CoV-2 variants. Yet the virtually unlimited potential for rapid generation and analysis of genomic data is also coupled with unique technical, scientific and organizational challenges. Here, we discuss the application of genomic and computational methods for efficient data-driven COVID-19 response, the advantages of the democratization of viral sequencing around the world and the challenges associated with viral genome data collection and processing.},
  archive      = {J_NMETH},
  author       = {Knyazev, Sergey and Chhugani, Karishma and Sarwal, Varuni and Ayyala, Ram and Singh, Harman and Karthikeyan, Smruthi and Deshpande, Dhrithi and Baykal, Pelin Icer and Comarova, Zoia and Lu, Angela and Porozov, Yuri and Vasylyeva, Tetyana I. and Wertheim, Joel O. and Tierney, Braden T. and Chiu, Charles Y. and Sun, Ren and Wu, Aiping and Abedalthagafi, Malak S. and Pak, Victoria M. and Nagaraj, Shivashankar H. and Smith, Adam L. and Skums, Pavel and Pasaniuc, Bogdan and Komissarov, Andrey and Mason, Christopher E. and Bortz, Eric and Lemey, Philippe and Kondrashov, Fyodor and Beerenwinkel, Niko and Lam, Tommy Tsan-Yuk and Wu, Nicholas C. and Zelikovsky, Alex and Knight, Rob and Crandall, Keith A. and Mangul, Serghei},
  doi          = {10.1038/s41592-022-01444-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {374-380},
  shortjournal = {Nat. Meth.},
  title        = {Unlocking capacities of genomics for the COVID-19 response and future pandemics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022m). Mackenzie weygandt mathis. <em>NMETH</em>, <em>19</em>(4),
373. (<a href="https://doi.org/10.1038/s41592-022-01438-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Building a sustainable open source toolbox to track social behavior and how to get in the zone.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01438-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {373},
  shortjournal = {Nat. Meth.},
  title        = {Mackenzie weygandt mathis},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Methods lead the way. <em>NMETH</em>, <em>19</em>(4),
371–372. (<a href="https://doi.org/10.1038/s41592-022-01474-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The COVID-19 pandemic has highlighted the importance of methodological advancements in basic biological research. We believe that method development will continue to propel both fundamental and applied studies on SARS-CoV-2 and other pathogens.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01474-7},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {371-372},
  shortjournal = {Nat. Meth.},
  title        = {Methods lead the way},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: An analytical framework for interpretable
and generalizable single-cell data analysis. <em>NMETH</em>,
<em>19</em>(3), 370. (<a
href="https://doi.org/10.1038/s41592-022-01421-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zhou, Jian and Troyanskaya, Olga G.},
  doi          = {10.1038/s41592-022-01421-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {370},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: An analytical framework for interpretable and generalizable single-cell data analysis},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Isotropic super-resolution light-sheet microscopy of dynamic
intracellular structures at subsecond timescales. <em>NMETH</em>,
<em>19</em>(3), 359–369. (<a
href="https://doi.org/10.1038/s41592-022-01395-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-term visualization of the dynamic interactions between intracellular structures throughout the three-dimensional space of whole live cells is essential to better understand their functions, but this task remains challenging due to the limitations of existing three-dimensional fluorescence microscopy techniques, such as an insufficient axial resolution, low volumetric imaging rate and photobleaching. Here, we present the combination of a progressive deep-learning super-resolution strategy with a double-ring-modulated selective plane illumination microscopy design capable of visualizing the dynamics of intracellular structures in live cells for hours at an isotropic spatial resolution of roughly 100 nm in three dimensions at speeds up to roughly 17 Hz. Using this approach, we reveal the complex spatial relationships and interactions between endoplasmic reticulum (ER) and mitochondria throughout live cells, providing new insights into ER-mediated mitochondrial division. We also examined the motion of Drp1 oligomers involved in mitochondrial fission and revealed the dynamic interactions between Drp1 and mitochondria in three dimensions. Combining a double-ring modulated SPIM with reduced side lobes and a sectionalized deep-learning based super-resolution algorithm enables fast, high-resolution, volumetric imaging of organelle interactions and dynamics in live cells.},
  archive      = {J_NMETH},
  author       = {Zhao, Yuxuan and Zhang, Meng and Zhang, Wenting and Zhou, Yao and Chen, Longbiao and Liu, Qing and Wang, Peng and Chen, Rong and Duan, Xinxin and Chen, Feifan and Deng, Huan and Wei, Yunfei and Fei, Peng and Zhang, Yu-Hui},
  doi          = {10.1038/s41592-022-01395-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {359-369},
  shortjournal = {Nat. Meth.},
  title        = {Isotropic super-resolution light-sheet microscopy of dynamic intracellular structures at subsecond timescales},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Targeted multicolor in vivo imaging over 1,000 nm enabled by
nonamethine cyanines. <em>NMETH</em>, <em>19</em>(3), 353–358. (<a
href="https://doi.org/10.1038/s41592-022-01394-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent progress has shown that using wavelengths between 1,000 and 2,000 nm, referred to as the shortwave-infrared or near-infrared (NIR)-II range, can enable high-resolution in vivo imaging at depths not possible with conventional optical wavelengths. However, few bioconjugatable probes of the type that have proven invaluable for multiplexed imaging in the visible and NIR range are available for imaging these wavelengths. Using rational design, we have generated persulfonated indocyanine dyes with absorbance maxima at 872 and 1,072 nm through catechol-ring and aryl-ring fusion, respectively, onto the nonamethine scaffold. Multiplexed two-color and three-color in vivo imaging using monoclonal antibody and dextran conjugates in several tumor models illustrate the benefits of concurrent labeling of the tumor and healthy surrounding tissue and lymphatics. These efforts are enabled by complementary advances in a custom-built NIR/shortwave-infrared imaging setup and software package for multicolor real-time imaging. Rational design was used to develop a suite of red-shifted, bioconjugatable heptamethine cyanine dyes for multiplexed in vivo imaging in the shortwave-infrared/near-infrared-II region.},
  archive      = {J_NMETH},
  author       = {Bandi, Venu G. and Luciano, Michael P. and Saccomano, Mara and Patel, Nimit L. and Bischof, Thomas S. and Lingg, Jakob G. P. and Tsrunchev, Peter T. and Nix, Meredith N. and Ruehle, Bastian and Sanders, Chelsea and Riffle, Lisa and Robinson, Christina M. and Difilippantonio, Simone and Kalen, Joseph D. and Resch-Genger, Ute and Ivanic, Joseph and Bruns, Oliver T. and Schnermann, Martin J.},
  doi          = {10.1038/s41592-022-01394-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {353-358},
  shortjournal = {Nat. Meth.},
  title        = {Targeted multicolor in vivo imaging over 1,000 nm enabled by nonamethine cyanines},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Global profiling of phosphorylation-dependent changes in
cysteine reactivity. <em>NMETH</em>, <em>19</em>(3), 341–352. (<a
href="https://doi.org/10.1038/s41592-022-01398-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteomics has revealed that the ~20,000 human genes engender a far greater number of proteins, or proteoforms, that are diversified in large part by post-translational modifications (PTMs). How such PTMs affect protein structure and function is an active area of research but remains technically challenging to assess on a proteome-wide scale. Here, we describe a chemical proteomic method to quantitatively relate serine/threonine phosphorylation to changes in the reactivity of cysteine residues, a parameter that can affect the potential for cysteines to be post-translationally modified or engaged by covalent drugs. Leveraging the extensive high-stoichiometry phosphorylation occurring in mitotic cells, we discover numerous cysteines that exhibit phosphorylation-dependent changes in reactivity on diverse proteins enriched in cell cycle regulatory pathways. The discovery of bidirectional changes in cysteine reactivity often occurring in proximity to serine/threonine phosphorylation events points to the broad impact of phosphorylation on the chemical reactivity of proteins and the future potential to create small-molecule probes that differentially target proteoforms with PTMs. This article describes a chemical proteomic approach to quantitatively relate serine/threonine phosphorylation to changes in the reactivity of cysteine residues, thereby affecting their potential to be post-translationally modified and/or targeted by electrophilic small molecules.},
  archive      = {J_NMETH},
  author       = {Kemper, Esther K. and Zhang, Yuanjin and Dix, Melissa M. and Cravatt, Benjamin F.},
  doi          = {10.1038/s41592-022-01398-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {341-352},
  shortjournal = {Nat. Meth.},
  title        = {Global profiling of phosphorylation-dependent changes in cysteine reactivity},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficient targeted insertion of large DNA fragments without
DNA donors. <em>NMETH</em>, <em>19</em>(3), 331–340. (<a
href="https://doi.org/10.1038/s41592-022-01399-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted insertion of large DNA fragments holds great potential for treating genetic diseases. Prime editors can effectively insert short fragments (~44 bp) but not large ones. Here we developed GRAND editing to precisely insert large DNA fragments without DNA donors. In contrast to prime editors, which require reverse transcription templates hybridizing with the target sequence, GRAND editing employs a pair of prime editing guide RNAs, with reverse transcription templates nonhomologous to the target site but complementary to each other. This strategy exhibited an efficiency of up to 63.0\% of a 150-bp insertion with minor by-products and 28.4\% of a 250-bp insertion. It allowed insertions up to ~1 kb, although the efficiency remains low for fragments larger than 400 bp. We confirmed efficient insertion in multiple genomic loci of several cell lines and non-dividing cells, which expands the scope of genome editing to enable donor-free insertion of large DNA sequences. GRAND editing employs a pair of prime editing guide RNAs to enable efficient DNA insertion up to 250 bp without the need of a DNA donor.},
  archive      = {J_NMETH},
  author       = {Wang, Jinlin and He, Zhou and Wang, Guoquan and Zhang, Ruiwen and Duan, Junyi and Gao, Pan and Lei, Xinlin and Qiu, Houyuan and Zhang, Chuanping and Zhang, Ying and Yin, Hao},
  doi          = {10.1038/s41592-022-01399-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {331-340},
  shortjournal = {Nat. Meth.},
  title        = {Efficient targeted insertion of large DNA fragments without DNA donors},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deterministic scRNA-seq captures variation in intestinal
crypt and organoid composition. <em>NMETH</em>, <em>19</em>(3), 323–330.
(<a href="https://doi.org/10.1038/s41592-021-01391-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing (scRNA-seq) approaches have transformed our ability to resolve cellular properties across systems, but are currently tailored toward large cell inputs (&amp;gt;1,000 cells). This renders them inefficient and costly when processing small, individual tissue samples, a problem that tends to be resolved by loading bulk samples, yielding confounded mosaic cell population read-outs. Here, we developed a deterministic, mRNA-capture bead and cell co-encapsulation dropleting system, DisCo, aimed at processing low-input samples (&amp;lt;500 cells). We demonstrate that DisCo enables precise particle and cell positioning and droplet sorting control through combined machine-vision and multilayer microfluidics, enabling continuous processing of low-input single-cell suspensions at high capture efficiency (&amp;gt;70\%) and at speeds up to 350 cells per hour. To underscore DisCo’s unique capabilities, we analyzed 31 individual intestinal organoids at varying developmental stages. This revealed extensive organoid heterogeneity, identifying distinct subtypes including a regenerative fetal-like Ly6a+ stem cell population that persists as symmetrical cysts, or spheroids, even under differentiation conditions, and an uncharacterized ‘gobloid’ subtype consisting predominantly of precursor and mature (Muc2+) goblet cells. To complement this dataset and to demonstrate DisCo’s capacity to process low-input, in vivo-derived tissues, we also analyzed individual mouse intestinal crypts. This revealed the existence of crypts with a compositional similarity to spheroids, which consisted predominantly of regenerative stem cells, suggesting the existence of regenerating crypts in the homeostatic intestine. These findings demonstrate the unique power of DisCo in providing high-resolution snapshots of cellular heterogeneity in small, individual tissues. DisCo is a deterministic droplet microfluidics tool for single-cell analysis on low cell input samples, which is demonstrated to profile individual intestinal organoids and in vivo-derived small tissues.},
  archive      = {J_NMETH},
  author       = {Bues, Johannes and Biočanin, Marjan and Pezoldt, Joern and Dainese, Riccardo and Chrisnandy, Antonius and Rezakhani, Saba and Saelens, Wouter and Gardeux, Vincent and Gupta, Revant and Sarkis, Rita and Russeil, Julie and Saeys, Yvan and Amstad, Esther and Claassen, Manfred and Lutolf, Matthias P. and Deplancke, Bart},
  doi          = {10.1038/s41592-021-01391-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {323-330},
  shortjournal = {Nat. Meth.},
  title        = {Deterministic scRNA-seq captures variation in intestinal crypt and organoid composition},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Alevin-fry unlocks rapid, accurate and memory-frugal
quantification of single-cell RNA-seq data. <em>NMETH</em>,
<em>19</em>(3), 316–322. (<a
href="https://doi.org/10.1038/s41592-022-01408-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid growth of high-throughput single-cell and single-nucleus RNA-sequencing (scRNA-seq and snRNA-seq) technologies has produced a wealth of data over the past few years. The size, volume and distinctive characteristics of these data necessitate the development of new computational methods to accurately and efficiently quantify sc/snRNA-seq data into count matrices that constitute the input to downstream analyses. We introduce the alevin-fry framework for quantifying sc/snRNA-seq data. In addition to being faster and more memory frugal than other accurate quantification approaches, alevin-fry ameliorates the memory scalability and false-positive expression issues that are exhibited by other lightweight tools. We demonstrate how alevin-fry can be effectively used to quantify sc/snRNA-seq data, and also how the spliced and unspliced molecule quantification required as input for RNA velocity analyses can be seamlessly extracted from the same preprocessed data used to generate normal gene expression count matrices. Alevin-fry accurately quantifies single-cell and single-nucleus RNA-seq data with high speed and memory efficiency.},
  archive      = {J_NMETH},
  author       = {He, Dongze and Zakeri, Mohsen and Sarkar, Hirak and Soneson, Charlotte and Srivastava, Avi and Patro, Rob},
  doi          = {10.1038/s41592-022-01408-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {316-322},
  shortjournal = {Nat. Meth.},
  title        = {Alevin-fry unlocks rapid, accurate and memory-frugal quantification of single-cell RNA-seq data},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). MCMICRO: A scalable, modular image-processing pipeline for
multiplexed tissue imaging. <em>NMETH</em>, <em>19</em>(3), 311–315. (<a
href="https://doi.org/10.1038/s41592-021-01308-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Highly multiplexed tissue imaging makes detailed molecular analysis of single cells possible in a preserved spatial context. However, reproducible analysis of large multichannel images poses a substantial computational challenge. Here, we describe a modular and open-source computational pipeline, MCMICRO, for performing the sequential steps needed to transform whole-slide images into single-cell data. We demonstrate the use of MCMICRO on tissue and tumor images acquired using multiple imaging platforms, thereby providing a solid foundation for the continued development of tissue imaging software. MCMICRO is a modular and open-source computational pipeline for transforming highly multiplexed whole-slide images of tissues into single-cell data. MCMICRO is versatile and can be used with CODEX, mxIF, CyCIF, mIHC and H&amp;amp;E staining data.},
  archive      = {J_NMETH},
  author       = {Schapiro, Denis and Sokolov, Artem and Yapp, Clarence and Chen, Yu-An and Muhlich, Jeremy L. and Hess, Joshua and Creason, Allison L. and Nirmal, Ajit J. and Baker, Gregory J. and Nariya, Maulik K. and Lin, Jia-Ren and Maliga, Zoltan and Jacobson, Connor A. and Hodgman, Matthew W. and Ruokonen, Juha and Farhi, Samouil L. and Abbondanza, Domenic and McKinley, Eliot T. and Persson, Daniel and Betts, Courtney and Sivagnanam, Shamilene and Regev, Aviv and Goecks, Jeremy and Coffey, Robert J. and Coussens, Lisa M. and Santagata, Sandro and Sorger, Peter K.},
  doi          = {10.1038/s41592-021-01308-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {311-315},
  shortjournal = {Nat. Meth.},
  title        = {MCMICRO: A scalable, modular image-processing pipeline for multiplexed tissue imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). The SpliZ generalizes “percent spliced in” to reveal
regulated splicing at single-cell resolution. <em>NMETH</em>,
<em>19</em>(3), 307–310. (<a
href="https://doi.org/10.1038/s41592-022-01400-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detecting single-cell-regulated splicing from droplet-based technologies is challenging. Here, we introduce the splicing Z score (SpliZ), an annotation-free statistical method to detect regulated splicing in single-cell RNA sequencing. We applied the SpliZ to human lung cells, discovering hundreds of genes with cell-type-specific splicing patterns including ones with potential implications for basic and translational biology. The Splicing Z Score (SpliZ) provides single-cell-level quantification of alternative splicing with improved statistical power.},
  archive      = {J_NMETH},
  author       = {Olivieri, Julia Eve and Dehghannasiri, Roozbeh and Salzman, Julia},
  doi          = {10.1038/s41592-022-01400-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {307-310},
  shortjournal = {Nat. Meth.},
  title        = {The SpliZ generalizes ‘percent spliced in’ to reveal regulated splicing at single-cell resolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A pan-tissue DNA methylation atlas enables in silico
decomposition of human tissue methylomes at cell-type resolution.
<em>NMETH</em>, <em>19</em>(3), 296–306. (<a
href="https://doi.org/10.1038/s41592-022-01412-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bulk-tissue DNA methylomes represent an average over many different cell types, hampering our understanding of cell-type-specific contributions to disease development. As single-cell methylomics is not scalable to large cohorts of individuals, cost-effective computational solutions are needed, yet current methods are limited to tissues such as blood. Here we leverage the high-resolution nature of tissue-specific single-cell RNA-sequencing datasets to construct a DNA methylation atlas defined for 13 solid tissue types and 40 cell types. We comprehensively validate this atlas in independent bulk and single-nucleus DNA methylation datasets. We demonstrate that it correctly predicts the cell of origin of diverse cancer types and discovers new prognostic associations in olfactory neuroblastoma and stage 2 melanoma. In brain, the atlas predicts a neuronal origin for schizophrenia, with neuron-specific differential DNA methylation enriched for corresponding genome-wide association study risk loci. In summary, the DNA methylation atlas enables the decomposition of 13 different human tissue types at a high cellular resolution, paving the way for an improved interpretation of epigenetic data. This resource presents an in silico generated DNA methylation atlas that can be used for cell-type deconvolution of human tissues.},
  archive      = {J_NMETH},
  author       = {Zhu, Tianyu and Liu, Jacklyn and Beck, Stephan and Pan, Sun and Capper, David and Lechner, Matt and Thirlwell, Chrissie and Breeze, Charles E. and Teschendorff, Andrew E.},
  doi          = {10.1038/s41592-022-01412-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {296-306},
  shortjournal = {Nat. Meth.},
  title        = {A pan-tissue DNA methylation atlas enables in silico decomposition of human tissue methylomes at cell-type resolution},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial mapping of protein composition and tissue
organization: A primer for multiplexed antibody-based imaging.
<em>NMETH</em>, <em>19</em>(3), 284–295. (<a
href="https://doi.org/10.1038/s41592-021-01316-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissues and organs are composed of distinct cell types that must operate in concert to perform physiological functions. Efforts to create high-dimensional biomarker catalogs of these cells have been largely based on single-cell sequencing approaches, which lack the spatial context required to understand critical cellular communication and correlated structural organization. To probe in situ biology with sufficient depth, several multiplexed protein imaging methods have been recently developed. Though these technologies differ in strategy and mode of immunolabeling and detection tags, they commonly utilize antibodies directed against protein biomarkers to provide detailed spatial and functional maps of complex tissues. As these promising antibody-based multiplexing approaches become more widely adopted, new frameworks and considerations are critical for training future users, generating molecular tools, validating antibody panels, and harmonizing datasets. In this Perspective, we provide essential resources, key considerations for obtaining robust and reproducible imaging data, and specialized knowledge from domain experts and technology developers. This Perspective offers guidance for robust and reproducible antibody-based highly multiplexed tissue imaging.},
  archive      = {J_NMETH},
  author       = {Hickey, John W. and Neumann, Elizabeth K. and Radtke, Andrea J. and Camarillo, Jeannie M. and Beuschel, Rebecca T. and Albanese, Alexandre and McDonough, Elizabeth and Hatler, Julia and Wiblin, Anne E. and Fisher, Jeremy and Croteau, Josh and Small, Eliza C. and Sood, Anup and Caprioli, Richard M. and Angelo, R. Michael and Nolan, Garry P. and Chung, Kwanghun and Hewitt, Stephen M. and Germain, Ronald N. and Spraggins, Jeffrey M. and Lundberg, Emma and Snyder, Michael P. and Kelleher, Neil L. and Saka, Sinem K.},
  doi          = {10.1038/s41592-021-01316-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {284-295},
  shortjournal = {Nat. Meth.},
  title        = {Spatial mapping of protein composition and tissue organization: A primer for multiplexed antibody-based imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Uncovering an overlooked consequence of phosphorylation:
Change in cysteine reactivity. <em>NMETH</em>, <em>19</em>(3), 281–283.
(<a href="https://doi.org/10.1038/s41592-022-01414-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global profiling of changes in the reactivity of cysteine residues in response to phosphorylation during mitosis identifies cysteine residues as potential regulatory and drug binding sites on proteins.},
  archive      = {J_NMETH},
  author       = {Lakemeyer, Markus and Bogyo, Matthew},
  doi          = {10.1038/s41592-022-01414-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {281-283},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering an overlooked consequence of phosphorylation: Change in cysteine reactivity},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell, single-organoid phenotypic landscapes.
<em>NMETH</em>, <em>19</em>(3), 280–281. (<a
href="https://doi.org/10.1038/s41592-022-01416-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A droplet microfluidic system enables deterministic capture and subsequent sequencing and analysis of single-cell transcriptomes from organoids and other small, individual tissue samples.},
  archive      = {J_NMETH},
  author       = {Camp, J. Gray and Lukonin, Ilya},
  doi          = {10.1038/s41592-022-01416-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {280-281},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell, single-organoid phenotypic landscapes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022u). The chemistry of microbiome–host togetherness.
<em>NMETH</em>, <em>19</em>(3), 274–279. (<a
href="https://doi.org/10.1038/s41592-022-01413-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Labs are stacking techniques to disentangle the chemical influences of microbiomes.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01413-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {274-279},
  shortjournal = {Nat. Meth.},
  title        = {The chemistry of microbiome–host togetherness},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Viral discovery at a global scale. <em>NMETH</em>,
<em>19</em>(3), 273. (<a
href="https://doi.org/10.1038/s41592-022-01430-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultra-high-throughput sequence alignment enables identification of &amp;gt;130,000 novel RNA viruses.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01430-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {273},
  shortjournal = {Nat. Meth.},
  title        = {Viral discovery at a global scale},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Unraveling immune cell behavior. <em>NMETH</em>,
<em>19</em>(3), 272. (<a
href="https://doi.org/10.1038/s41592-022-01435-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Live imaging enables temporal tracking of immune cell behavior.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01435-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {272},
  shortjournal = {Nat. Meth.},
  title        = {Unraveling immune cell behavior},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022r). Simulating a whole cell. <em>NMETH</em>, <em>19</em>(3),
271. (<a href="https://doi.org/10.1038/s41592-022-01429-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive modeling of cellular networks of cells with a minimal genome.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01429-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {271},
  shortjournal = {Nat. Meth.},
  title        = {Simulating a whole cell},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022l). Reassessing 6mA in eukaryotes. <em>NMETH</em>,
<em>19</em>(3), 270. (<a
href="https://doi.org/10.1038/s41592-022-01434-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A metagenomic approach to identifying the sources of N6-methyldeoxyadenine (6mA) in eukaryotic DNA found no evidence of high 6mA abundances in sampled eukaryotes.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01434-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {270},
  shortjournal = {Nat. Meth.},
  title        = {Reassessing 6mA in eukaryotes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022i). Potent optogenetics. <em>NMETH</em>, <em>19</em>(3), 269.
(<a href="https://doi.org/10.1038/s41592-022-01433-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two groups report improved variants of the channelrhodopsins ChRmine and ChroME, respectively.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01433-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {269},
  shortjournal = {Nat. Meth.},
  title        = {Potent optogenetics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). MITI minimum information guidelines for highly multiplexed
tissue images. <em>NMETH</em>, <em>19</em>(3), 262–267. (<a
href="https://doi.org/10.1038/s41592-022-01415-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The imminent release of tissue atlases combining multichannel microscopy with single-cell sequencing and other omics data from normal and diseased specimens creates an urgent need for data and metadata standards to guide data deposition, curation and release. We describe a Minimum Information about Highly Multiplexed Tissue Imaging (MITI) standard that applies best practices developed for genomics and for other microscopy data to highly multiplexed tissue images and traditional histology.},
  archive      = {J_NMETH},
  author       = {Schapiro, Denis and Yapp, Clarence and Sokolov, Artem and Reynolds, Sheila M. and Chen, Yu-An and Sudar, Damir and Xie, Yubin and Muhlich, Jeremy and Arias-Camison, Raquel and Arena, Sarah and Taylor, Adam J. and Nikolov, Milen and Tyler, Madison and Lin, Jia-Ren and Burlingame, Erik A. and Chang, Young H. and Farhi, Samouil L. and Thorsson, Vésteinn and Venkatamohan, Nithya and Drewes, Julia L. and Pe’er, Dana and Gutman, David A. and Herrmann, Markus D. and Gehlenborg, Nils and Bankhead, Peter and Roland, Joseph T. and Herndon, John M. and Snyder, Michael P. and Angelo, Michael and Nolan, Garry and Swedlow, Jason R. and Schultz, Nikolaus and Merrick, Daniel T. and Mazzili, Sarah A. and Cerami, Ethan and Rodig, Scott J. and Santagata, Sandro and Sorger, Peter K.},
  doi          = {10.1038/s41592-022-01415-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {262-267},
  shortjournal = {Nat. Meth.},
  title        = {MITI minimum information guidelines for highly multiplexed tissue images},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022q). Peter sorger. <em>NMETH</em>, <em>19</em>(3), 261. (<a
href="https://doi.org/10.1038/s41592-022-01418-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hiking his way to a well-engineered translational future for tissue-based imaging.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01418-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {261},
  shortjournal = {Nat. Meth.},
  title        = {Peter sorger},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Catching up with multiplexed tissue imaging. <em>NMETH</em>,
<em>19</em>(3), 259. (<a
href="https://doi.org/10.1038/s41592-022-01428-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Highly multiplexed tissue imaging continues to show its power for biomedical discovery. In this issue, we publish tools and guidance for implementing this class of methods and reporting subsequent results.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01428-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {259},
  shortjournal = {Nat. Meth.},
  title        = {Catching up with multiplexed tissue imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Single-cell chromatin state analysis with
signac. <em>NMETH</em>, <em>19</em>(2), 257. (<a
href="https://doi.org/10.1038/s41592-022-01393-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Stuart, Tim and Srivastava, Avi and Madad, Shaista and Lareau, Caleb A. and Satija, Rahul},
  doi          = {10.1038/s41592-022-01393-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {257},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Single-cell chromatin state analysis with signac},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Fast optical recording of neuronal
activity by three-dimensional custom-access serial holography.
<em>NMETH</em>, <em>19</em>(2), 257. (<a
href="https://doi.org/10.1038/s41592-022-01401-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Akemann, Walther and Wolf, Sébastien and Villette, Vincent and Mathieu, Benjamin and Tangara, Astou and Fodor, Jozsua and Ventalon, Cathie and Léger, Jean-François and Dieudonné, Stéphane and Bourdieu, Laurent},
  doi          = {10.1038/s41592-022-01401-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {257},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Fast optical recording of neuronal activity by three-dimensional custom-access serial holography},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: QUAREP-LiMi: A community endeavor to
advance quality assessment and reproducibility in light microscopy.
<em>NMETH</em>, <em>19</em>(2), 256. (<a
href="https://doi.org/10.1038/s41592-021-01387-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Boehm, Ulrike and Nelson, Glyn and Brown, Claire M. and Bagley, Steve and Bajcsy, Peter and Bischof, Johanna and Dauphin, Aurelien and Dobbie, Ian M. and Eriksson, John E. and Faklaris, Orestis and Fernandez-Rodriguez, Julia and Ferrand, Alexia and Gelman, Laurent and Gheisari, Ali and Hartmann, Hella and Kukat, Christian and Laude, Alex and Mitkovski, Miso and Munck, Sebastian and North, Alison J. and Rasse, Tobias M. and Resch-Genger, Ute and Schuetz, Lucas C. and Seitz, Arne and Strambio-De-Castillia, Caterina and Swedlow, Jason R. and Nitschke, Roland},
  doi          = {10.1038/s41592-021-01387-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {256},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: QUAREP-LiMi: a community endeavor to advance quality assessment and reproducibility in light microscopy},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Micro-meta app: An interactive tool for
collecting microscopy metadata based on community specifications.
<em>NMETH</em>, <em>19</em>(2), 256. (<a
href="https://doi.org/10.1038/s41592-021-01388-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Rigano, Alessandro and Ehmsen, Shannon and Öztürk, Serkan Utku and Ryan, Joel and Balashov, Alexander and Hammer, Mathias and Kirli, Koray and Boehm, Ulrike and Brown, Claire M. and Bellve, Karl and Chambers, James J. and Cosolo, Andrea and Coleman, Robert A. and Faklaris, Orestis and Fogarty, Kevin E. and Guilbert, Thomas and Hamacher, Anna B. and Itano, Michelle S. and Keeley, Daniel P. and Kunis, Susanne and Lacoste, Judith and Laude, Alex and Ma, Willa Y. and Marcello, Marco and Montero-Llopis, Paula and Nelson, Glyn and Nitschke, Roland and Pimentel, Jaime A. and Weidtkamp-Peters, Stefanie and Park, Peter J. and Alver, Burak H. and Grunwald, David and Strambio-De-Castillia, Caterina},
  doi          = {10.1038/s41592-021-01388-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {256},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: micro-meta app: an interactive tool for collecting microscopy metadata based on community specifications},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Programmable RNA editing with compact
CRISPR–Cas13 systems from uncultivated microbes. <em>NMETH</em>,
<em>19</em>(2), 255. (<a
href="https://doi.org/10.1038/s41592-021-01379-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Xu, Chunlong and Zhou, Yingsi and Xiao, Qingquan and He, Bingbing and Geng, Guannan and Wang, Zikang and Cao, Birong and Dong, Xue and Bai, Weiya and Wang, Yifan and Wang, Xiang and Zhou, Dongming and Yuan, Tanglong and Huo, Xiaona and Lai, Jinsheng and Yang, Hui},
  doi          = {10.1038/s41592-021-01379-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {255},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Programmable RNA editing with compact CRISPR–Cas13 systems from uncultivated microbes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Towards community-driven metadata
standards for light microscopy: Tiered specifications extending the OME
model. <em>NMETH</em>, <em>19</em>(2), 255. (<a
href="https://doi.org/10.1038/s41592-021-01386-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hammer, Mathias and Huisman, Maximiliaan and Rigano, Alessandro and Boehm, Ulrike and Chambers, James J. and Gaudreault, Nathalie and North, Alison J. and Pimentel, Jaime A. and Sudar, Damir and Bajcsy, Peter and Brown, Claire M. and Corbett, Alexander D. and Faklaris, Orestis and Lacoste, Judith and Laude, Alex and Nelson, Glyn and Nitschke, Roland and Farzam, Farzin and Smith, Carlas S. and Grunwald, David and Strambio-De-Castillia, Caterina},
  doi          = {10.1038/s41592-021-01386-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {255},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: towards community-driven metadata standards for light microscopy: tiered specifications extending the OME model},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). VascuViz: A multimodality and multiscale imaging and
visualization pipeline for vascular systems biology. <em>NMETH</em>,
<em>19</em>(2), 242–254. (<a
href="https://doi.org/10.1038/s41592-021-01363-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in imaging, image-based vascular systems biology has remained challenging because blood vessel data are often available only from a single modality or at a given spatial scale, and cross-modality data are difficult to integrate. Therefore, there is an exigent need for a multimodality pipeline that enables ex vivo vascular imaging with magnetic resonance imaging, computed tomography and optical microscopy of the same sample, while permitting imaging with complementary contrast mechanisms from the whole-organ to endothelial cell spatial scales. To achieve this, we developed ‘VascuViz’—an easy-to-use method for simultaneous three-dimensional imaging and visualization of the vascular microenvironment using magnetic resonance imaging, computed tomography and optical microscopy in the same intact, unsectioned tissue. The VascuViz workflow permits multimodal imaging with a single labeling step using commercial reagents and is compatible with diverse tissue types and protocols. VascuViz’s interdisciplinary utility in conjunction with new data visualization approaches opens up new vistas in image-based vascular systems biology. VascuViz represents a versatile workflow for multimodal imaging of the vasculature in ex vivo tissue samples across length and resolution scales, paving the way for improved and novel image-based vascular systems biology applications.},
  archive      = {J_NMETH},
  author       = {Bhargava, Akanksha and Monteagudo, Benjamin and Kushwaha, Priyanka and Senarathna, Janaka and Ren, Yunke and Riddle, Ryan C. and Aggarwal, Manisha and Pathak, Arvind P.},
  doi          = {10.1038/s41592-021-01363-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {242-254},
  shortjournal = {Nat. Meth.},
  title        = {VascuViz: A multimodality and multiscale imaging and visualization pipeline for vascular systems biology},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A genetically encoded sensor for in vivo imaging of orexin
neuropeptides. <em>NMETH</em>, <em>19</em>(2), 231–241. (<a
href="https://doi.org/10.1038/s41592-021-01390-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Orexins (also called hypocretins) are hypothalamic neuropeptides that carry out essential functions in the central nervous system; however, little is known about their release and range of action in vivo owing to the limited resolution of current detection technologies. Here we developed a genetically encoded orexin sensor (OxLight1) based on the engineering of circularly permutated green fluorescent protein into the human type-2 orexin receptor. In mice OxLight1 detects optogenetically evoked release of endogenous orexins in vivo with high sensitivity. Photometry recordings of OxLight1 in mice show rapid orexin release associated with spontaneous running behavior, acute stress and sleep-to-wake transitions in different brain areas. Moreover, two-photon imaging of OxLight1 reveals orexin release in layer 2/3 of the mouse somatosensory cortex during emergence from anesthesia. Thus, OxLight1 enables sensitive and direct optical detection of orexin neuropeptides with high spatiotemporal resolution in living animals. OxLight1 is a genetically encoded sensor for the orexin neuropeptides. It has been applied in fiber photometry recordings and two-photon imaging in mice during a variety of behaviors.},
  archive      = {J_NMETH},
  author       = {Duffet, Loïc and Kosar, Seher and Panniello, Mariangela and Viberti, Bianca and Bracey, Edward and Zych, Anna D. and Radoux-Mergault, Arthur and Zhou, Xuehan and Dernic, Jan and Ravotto, Luca and Tsai, Yuan-Chen and Figueiredo, Marta and Tyagarajan, Shiva K. and Weber, Bruno and Stoeber, Miriam and Gogolla, Nadine and Schmidt, Markus H. and Adamantidis, Antoine R. and Fellin, Tommaso and Burdakov, Denis and Patriarchi, Tommaso},
  doi          = {10.1038/s41592-021-01390-2},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {231-241},
  shortjournal = {Nat. Meth.},
  title        = {A genetically encoded sensor for in vivo imaging of orexin neuropeptides},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatially resolved isotope tracing reveals tissue metabolic
activity. <em>NMETH</em>, <em>19</em>(2), 223–230. (<a
href="https://doi.org/10.1038/s41592-021-01378-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Isotope tracing has helped to determine the metabolic activities of organs. Methods to probe metabolic heterogeneity within organs are less developed. We couple stable-isotope-labeled nutrient infusion to matrix-assisted laser desorption ionization imaging mass spectrometry (iso-imaging) to quantitate metabolic activity in mammalian tissues in a spatially resolved manner. In the kidney, we visualize gluconeogenic flux and glycolytic flux in the cortex and medulla, respectively. Tricarboxylic acid cycle substrate usage differs across kidney regions; glutamine and citrate are used preferentially in the cortex and fatty acids are used in the medulla. In the brain, we observe spatial gradations in carbon inputs to the tricarboxylic acid cycle and glutamate under a ketogenic diet. In a carbohydrate-rich diet, glucose predominates throughout but in a ketogenic diet, 3-hydroxybutyrate contributes most strongly in the hippocampus and least in the midbrain. Brain nitrogen sources also vary spatially; branched-chain amino acids contribute most in the midbrain, whereas ammonia contributes in the thalamus. Thus, iso-imaging can reveal the spatial organization of metabolic activity. Iso-imaging integrates stable-isotope infusions with imaging mass spectrometry to enable quantitative analysis of metabolic activity in mammalian tissues with spatial resolution. IsoScope software facilitates analysis of iso-imaging data.},
  archive      = {J_NMETH},
  author       = {Wang, Lin and Xing, Xi and Zeng, Xianfeng and Jackson, S. RaElle and TeSlaa, Tara and Al-Dalahmah, Osama and Samarah, Laith Z. and Goodwin, Katharine and Yang, Lifeng and McReynolds, Melanie R. and Li, Xiaoxuan and Wolff, Jeremy J. and Rabinowitz, Joshua D. and Davidson, Shawn M.},
  doi          = {10.1038/s41592-021-01378-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {223-230},
  shortjournal = {Nat. Meth.},
  title        = {Spatially resolved isotope tracing reveals tissue metabolic activity},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Visualizing the native cellular organization by coupling
cryofixation with expansion microscopy (cryo-ExM). <em>NMETH</em>,
<em>19</em>(2), 216–222. (<a
href="https://doi.org/10.1038/s41592-021-01356-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryofixation has proven to be the gold standard for efficient preservation of native cell ultrastructure compared to chemical fixation, but this approach is not widely used in fluorescence microscopy owing to implementation challenges. Here, we develop Cryo-ExM, a method that preserves native cellular organization by coupling cryofixation with expansion microscopy. This method bypasses artifacts associated with chemical fixation and its simplicity will contribute to its widespread use in super-resolution microscopy. Cryo-ExM combines the expansion microscopy for super-resolution imaging with cryofixation for ultrastructure preservation. Cryo-ExM outperforms established fixation methods on a range of sensitive subcellular structures.},
  archive      = {J_NMETH},
  author       = {Laporte, Marine H. and Klena, Nikolai and Hamel, Virginie and Guichard, Paul},
  doi          = {10.1038/s41592-021-01356-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {216-222},
  shortjournal = {Nat. Meth.},
  title        = {Visualizing the native cellular organization by coupling cryofixation with expansion microscopy (Cryo-ExM)},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). NTR 2.0: A rationally engineered prodrug-converting enzyme
with substantially enhanced efficacy for targeted cell ablation.
<em>NMETH</em>, <em>19</em>(2), 205–215. (<a
href="https://doi.org/10.1038/s41592-021-01364-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transgenic expression of bacterial nitroreductase (NTR) enzymes sensitizes eukaryotic cells to prodrugs such as metronidazole (MTZ), enabling selective cell-ablation paradigms that have expanded studies of cell function and regeneration in vertebrates. However, first-generation NTRs required confoundingly toxic prodrug treatments to achieve effective cell ablation, and some cell types have proven resistant. Here we used rational engineering and cross-species screening to develop an NTR variant, NTR 2.0, which exhibits ~100-fold improvement in MTZ-mediated cell-specific ablation efficacy, eliminating the need for near-toxic prodrug treatment regimens. NTR 2.0 therefore enables sustained cell-loss paradigms and ablation of previously resistant cell types. These properties permit enhanced interrogations of cell function, extended challenges to the regenerative capacities of discrete stem cell niches, and novel modeling of chronic degenerative diseases. Accordingly, we have created a series of bipartite transgenic reporter/effector resources to facilitate dissemination of NTR 2.0 to the research community. An engineered bacterial nitroreductase, NTR 2.0, improves chemically induced cell ablation, facilitating novel sustained ablation paradigms for testing the effects of chronic inflammation on regeneration, and modeling degenerative disease.},
  archive      = {J_NMETH},
  author       = {Sharrock, Abigail V. and Mulligan, Timothy S. and Hall, Kelsi R. and Williams, Elsie M. and White, David T. and Zhang, Liyun and Emmerich, Kevin and Matthews, Frazer and Nimmagadda, Saumya and Washington, Selena and Le, Katherine D. and Meir-Levi, Danielle and Cox, Olivia L. and Saxena, Meera T. and Calof, Anne L. and Lopez-Burks, Martha E. and Lander, Arthur D. and Ding, Ding and Ji, Hongkai and Ackerley, David F. and Mumm, Jeff S.},
  doi          = {10.1038/s41592-021-01364-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {205-215},
  shortjournal = {Nat. Meth.},
  title        = {NTR 2.0: A rationally engineered prodrug-converting enzyme with substantially enhanced efficacy for targeted cell ablation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CR-i-TASSER: Assemble protein structures from cryo-EM
density maps using deep convolutional neural networks. <em>NMETH</em>,
<em>19</em>(2), 195–204. (<a
href="https://doi.org/10.1038/s41592-021-01389-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron microscopy (cryo-EM) has become a leading approach for protein structure determination, but it remains challenging to accurately model atomic structures with cryo-EM density maps. We propose a hybrid method, CR-I-TASSER (cryo-EM iterative threading assembly refinement), which integrates deep neural-network learning with I-TASSER assembly simulations for automated cryo-EM structure determination. The method is benchmarked on 778 proteins with simulated and experimental density maps, where CR-I-TASSER constructs models with a correct fold (template modeling (TM) score &amp;gt;0.5) for 643 targets that is 64\% higher than the best of some other de novo and refinement-based approaches on high-resolution data samples. Detailed data analyses showed that the main advantage of CR-I-TASSER lies in the deep learning-based Cα position prediction, which significantly improves the threading template quality and therefore boosts the accuracy of final models through optimized fragment assembly simulations. These results demonstrate a new avenue to determine cryo-EM protein structures with high accuracy and robustness covering various target types and density map resolutions. CR-I-TASSER integrates deep neural-network learning with I-TASSER assembly simulations for automated cryo-EM structure determination.},
  archive      = {J_NMETH},
  author       = {Zhang, Xi and Zhang, Biao and Freddolino, Peter L. and Zhang, Yang},
  doi          = {10.1038/s41592-021-01389-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {195-204},
  shortjournal = {Nat. Meth.},
  title        = {CR-I-TASSER: Assemble protein structures from cryo-EM density maps using deep convolutional neural networks},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reprogramming the piRNA pathway for multiplexed and
transgenerational gene silencing in c. elegans. <em>NMETH</em>,
<em>19</em>(2), 187–194. (<a
href="https://doi.org/10.1038/s41592-021-01369-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-guide RNAs can target exogenous CRISPR–Cas proteins to unique DNA locations, enabling genetic tools that are efficient, specific and scalable. Here we show that short synthetic guide Piwi-interacting RNAs (piRNAs) (21-nucleotide sg-piRNAs) expressed from extrachromosomal transgenes can, analogously, reprogram the endogenous piRNA pathway for gene-specific silencing in the hermaphrodite germline, sperm and embryos of Caenorhabditis elegans. piRNA-mediated interference (‘piRNAi’) is more efficient than RNAi and can be multiplexed, and auxin-mediated degradation of the piRNA-specific Argonaute PRG-1 allows conditional gene silencing. Target-specific silencing results in decreased messenger RNA levels, amplification of secondary small interfering RNAs and repressive chromatin modifications. Short (300 base pairs) piRNAi transgenes amplified from arrayed oligonucleotide pools also induce silencing, potentially making piRNAi highly scalable. We show that piRNAi can induce transgenerational epigenetic silencing of two endogenous genes (him-5 and him-8). Silencing is inherited for four to six generations after target-specific sg-piRNAs are lost, whereas depleting PRG-1 leads to essentially permanent epigenetic silencing. This work presents a piRNA-mediated interference for multiplexed gene silencing in Caenorhabditis elegans.},
  archive      = {J_NMETH},
  author       = {Priyadarshini, Monika and Ni, Julie Zhouli and Vargas-Velazquez, Amhed M. and Gu, Sam Guoping and Frøkjær-Jensen, Christian},
  doi          = {10.1038/s41592-021-01369-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {187-194},
  shortjournal = {Nat. Meth.},
  title        = {Reprogramming the piRNA pathway for multiplexed and transgenerational gene silencing in c. elegans},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identifying temporal and spatial patterns of variation from
multimodal data using MEFISTO. <em>NMETH</em>, <em>19</em>(2), 179–186.
(<a href="https://doi.org/10.1038/s41592-021-01343-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Factor analysis is a widely used method for dimensionality reduction in genome biology, with applications from personalized health to single-cell biology. Existing factor analysis models assume independence of the observed samples, an assumption that fails in spatio-temporal profiling studies. Here we present MEFISTO, a flexible and versatile toolbox for modeling high-dimensional data when spatial or temporal dependencies between the samples are known. MEFISTO maintains the established benefits of factor analysis for multimodal data, but enables the performance of spatio-temporally informed dimensionality reduction, interpolation, and separation of smooth from non-smooth patterns of variation. Moreover, MEFISTO can integrate multiple related datasets by simultaneously identifying and aligning the underlying patterns of variation in a data-driven manner. To illustrate MEFISTO, we apply the model to different datasets with spatial or temporal resolution, including an evolutionary atlas of organ development, a longitudinal microbiome study, a single-cell multi-omics atlas of mouse gastrulation and spatially resolved transcriptomics. MEFISTO models bulk and single-cell multi-omics data with temporal or spatial dependencies for interpretable pattern discovery and integration.},
  archive      = {J_NMETH},
  author       = {Velten, Britta and Braunger, Jana M. and Argelaguet, Ricard and Arnol, Damien and Wirbel, Jakob and Bredikhin, Danila and Zeller, Georg and Stegle, Oliver},
  doi          = {10.1038/s41592-021-01343-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {179-186},
  shortjournal = {Nat. Meth.},
  title        = {Identifying temporal and spatial patterns of variation from multimodal data using MEFISTO},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Squidpy: A scalable framework for spatial omics analysis.
<em>NMETH</em>, <em>19</em>(2), 171–178. (<a
href="https://doi.org/10.1038/s41592-021-01358-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics data are advancing the study of tissue organization and cellular communication at an unprecedented scale. Flexible tools are required to store, integrate and visualize the large diversity of spatial omics data. Here, we present Squidpy, a Python framework that brings together tools from omics and image analysis to enable scalable description of spatial molecular data, such as transcriptome or multivariate proteins. Squidpy provides efficient infrastructure and numerous analysis methods that allow to efficiently store, manipulate and interactively visualize spatial omics data. Squidpy is extensible and can be interfaced with a variety of already existing libraries for the scalable analysis of spatial omics data. Squidpy enables comprehensive analysis and visualization of spatial omics data and image with high efficiency.},
  archive      = {J_NMETH},
  author       = {Palla, Giovanni and Spitzer, Hannah and Klein, Michal and Fischer, David and Schaar, Anna Christina and Kuemmerle, Louis Benedikt and Rybakov, Sergei and Ibarra, Ignacio L. and Holmberg, Olle and Virshup, Isaac and Lotfollahi, Mohammad and Richter, Sabrina and Theis, Fabian J.},
  doi          = {10.1038/s41592-021-01358-2},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {171-178},
  shortjournal = {Nat. Meth.},
  title        = {Squidpy: A scalable framework for spatial omics analysis},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CellRank for directed single-cell fate mapping.
<em>NMETH</em>, <em>19</em>(2), 159–170. (<a
href="https://doi.org/10.1038/s41592-021-01346-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational trajectory inference enables the reconstruction of cell state dynamics from single-cell RNA sequencing experiments. However, trajectory inference requires that the direction of a biological process is known, largely limiting its application to differentiating systems in normal development. Here, we present CellRank ( https://cellrank.org ) for single-cell fate mapping in diverse scenarios, including regeneration, reprogramming and disease, for which direction is unknown. Our approach combines the robustness of trajectory inference with directional information from RNA velocity, taking into account the gradual and stochastic nature of cellular fate decisions, as well as uncertainty in velocity vectors. On pancreas development data, CellRank automatically detects initial, intermediate and terminal populations, predicts fate potentials and visualizes continuous gene expression trends along individual lineages. Applied to lineage-traced cellular reprogramming data, predicted fate probabilities correctly recover reprogramming outcomes. CellRank also predicts a new dedifferentiation trajectory during postinjury lung regeneration, including previously unknown intermediate cell states, which we confirm experimentally. CellRank infers directed cell state transitions and cell fates incorporating RNA velocity information into a graph based Markov process.},
  archive      = {J_NMETH},
  author       = {Lange, Marius and Bergen, Volker and Klein, Michal and Setty, Manu and Reuter, Bernhard and Bakhti, Mostafa and Lickert, Heiko and Ansari, Meshal and Schniering, Janine and Schiller, Herbert B. and Pe’er, Dana and Theis, Fabian J.},
  doi          = {10.1038/s41592-021-01346-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {159-170},
  shortjournal = {Nat. Meth.},
  title        = {CellRank for directed single-cell fate mapping},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Caveat fluorophore: An insiders’ guide to small-molecule
fluorescent labels. <em>NMETH</em>, <em>19</em>(2), 149–158. (<a
href="https://doi.org/10.1038/s41592-021-01338-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The last three decades have brought a revolution in fluorescence microscopy. The development of new microscopes, fluorescent labels and analysis techniques has pushed the frontiers of biological imaging forward, moving from fixed to live cells, from diffraction-limited to super-resolution imaging and from simple cell culture systems to experiments in vivo. The large and ever-evolving collection of tools can be daunting for biologists, who must invest substantial time and effort in adopting new technologies to answer their specific questions. This is particularly relevant when working with small-molecule fluorescent labels, where users must navigate the jargon, idiosyncrasies and caveats of chemistry. Here, we present an overview of chemical dyes used in biology and provide frank advice from a chemist’s perspective. This Perspective offers an overview of organic dyes commonly used as fluorescent labels and gives a chemist’s insight into their benefits and peculiarities.},
  archive      = {J_NMETH},
  author       = {Grimm, Jonathan B. and Lavis, Luke D.},
  doi          = {10.1038/s41592-021-01338-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {149-158},
  shortjournal = {Nat. Meth.},
  title        = {Caveat fluorophore: An insiders’ guide to small-molecule fluorescent labels},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Expansion microscopy opens the door to exploring more
challenges. <em>NMETH</em>, <em>19</em>(2), 147–148. (<a
href="https://doi.org/10.1038/s41592-022-01396-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryofixation-based ultrastructure-expansion microscopy (cryo-ExM) bypasses artifacts caused by chemical fixation and establishes more-native preservation of biological samples.},
  archive      = {J_NMETH},
  author       = {Gao, Mengfei},
  doi          = {10.1038/s41592-022-01396-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {147-148},
  shortjournal = {Nat. Meth.},
  title        = {Expansion microscopy opens the door to exploring more challenges},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). How single-cell multi-omics builds relationships.
<em>NMETH</em>, <em>19</em>(2), 142–146. (<a
href="https://doi.org/10.1038/s41592-022-01392-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Labs reaping information from single cells balance desirables with headaches such as data sparsity, cost and protocol surprises.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01392-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {142-146},
  shortjournal = {Nat. Meth.},
  title        = {How single-cell multi-omics builds relationships},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Epigenetic clock of cell dynamics. <em>NMETH</em>,
<em>19</em>(2), 141. (<a
href="https://doi.org/10.1038/s41592-022-01406-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clonal evolution of stem cells unveiled by modeling DNA methylation fluctuation.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01406-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {141},
  shortjournal = {Nat. Meth.},
  title        = {Epigenetic clock of cell dynamics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Jellyfish enter neuroscience research. <em>NMETH</em>,
<em>19</em>(2), 140. (<a
href="https://doi.org/10.1038/s41592-022-01404-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Jellyfish are established as a model system for neuroscience.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01404-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {140},
  shortjournal = {Nat. Meth.},
  title        = {Jellyfish enter neuroscience research},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022t). Spatially resolved DNA sequencing. <em>NMETH</em>,
<em>19</em>(2), 139. (<a
href="https://doi.org/10.1038/s41592-022-01405-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Slide-DNA-seq enables profiling of spatially resolved mutations and clonal populations in intact tumor samples.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01405-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {139},
  shortjournal = {Nat. Meth.},
  title        = {Spatially resolved DNA sequencing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Tumor microenvironments in 3D. <em>NMETH</em>,
<em>19</em>(2), 138. (<a
href="https://doi.org/10.1038/s41592-022-01402-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An imaging mass cytometry method reconstructs tissue microenvironments in 3D.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01402-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {138},
  shortjournal = {Nat. Meth.},
  title        = {Tumor microenvironments in 3D},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Three views are better than one. <em>NMETH</em>,
<em>19</em>(2), 137. (<a
href="https://doi.org/10.1038/s41592-022-01403-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers push the limits of confocal microscopy by combining multiple views, super-resolution and deep learning.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01403-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {137},
  shortjournal = {Nat. Meth.},
  title        = {Three views are better than one},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). GNPS dashboard: Collaborative exploration of mass
spectrometry data in the web browser. <em>NMETH</em>, <em>19</em>(2),
134–136. (<a href="https://doi.org/10.1038/s41592-021-01339-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Petras, Daniel and Phelan, Vanessa V. and Acharya, Deepa and Allen, Andrew E. and Aron, Allegra T. and Bandeira, Nuno and Bowen, Benjamin P. and Belle-Oudry, Deirdre and Boecker, Simon and Cummings, Dale A. and Deutsch, Jessica M. and Fahy, Eoin and Garg, Neha and Gregor, Rachel and Handelsman, Jo and Navarro-Hoyos, Mirtha and Jarmusch, Alan K. and Jarmusch, Scott A. and Louie, Katherine and Maloney, Katherine N. and Marty, Michael T. and Meijler, Michael M. and Mizrahi, Itzhak and Neve, Rachel L. and Northen, Trent R. and Molina-Santiago, Carlos and Panitchpakdi, Morgan and Pullman, Benjamin and Puri, Aaron W. and Schmid, Robin and Subramaniam, Shankar and Thukral, Monica and Vasquez-Castro, Felipe and Dorrestein, Pieter C. and Wang, Mingxun},
  doi          = {10.1038/s41592-021-01339-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {134-136},
  shortjournal = {Nat. Meth.},
  title        = {GNPS dashboard: Collaborative exploration of mass spectrometry data in the web browser},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Christian frøkjær-jensen. <em>NMETH</em>, <em>19</em>(2),
133. (<a href="https://doi.org/10.1038/s41592-022-01397-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A bioengineer and world traveler with a systems approach to gene silencing.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01397-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {133},
  shortjournal = {Nat. Meth.},
  title        = {Christian frøkjær-jensen},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Registered reports at nature methods. <em>NMETH</em>,
<em>19</em>(2), 131. (<a
href="https://doi.org/10.1038/s41592-022-01407-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Methods is introducing a new article format: Registered Reports. We encourage all authors interested in submitting comparative analyses of the performance of established, related tools or methods to familiarize themselves with this alternative approach to peer review.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01407-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {131},
  shortjournal = {Nat. Meth.},
  title        = {Registered reports at nature methods},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Community evaluation of glycoproteomics
informatics solutions reveals high-performance search strategies for
serum glycopeptide analysis. <em>NMETH</em>, <em>19</em>(1), 130. (<a
href="https://doi.org/10.1038/s41592-021-01368-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Kawahara, Rebeca and Chernykh, Anastasia and Alagesan, Kathirvel and Bern, Marshall and Cao, Weiqian and Chalkley, Robert J. and Cheng, Kai and Choo, Matthew S. and Edwards, Nathan and Goldman, Radoslav and Hoffmann, Marcus and Hu, Yingwei and Huang, Yifan and Kim, Jin Young and Kletter, Doron and Liquet, Benoit and Liu, Mingqi and Mechref, Yehia and Meng, Bo and Neelamegham, Sriram and Nguyen-Khuong, Terry and Nilsson, Jonas and Pap, Adam and Park, Gun Wook and Parker, Benjamin L. and Pegg, Cassandra L. and Penninger, Josef M. and Phung, Toan K. and Pioch, Markus and Rapp, Erdmann and Sakalli, Enes and Sanda, Miloslav and Schulz, Benjamin L. and Scott, Nichollas E. and Sofronov, Georgy and Stadlmann, Johannes and Vakhrushev, Sergey Y. and Woo, Christina M. and Wu, Hung-Yi and Yang, Pengyuan and Ying, Wantao and Zhang, Hui and Zhang, Yong and Zhao, Jingfu and Zaia, Joseph and Haslam, Stuart M. and Palmisano, Giuseppe and Yoo, Jong Shin and Larson, Göran and Khoo, Kai-Hooi and Medzihradszky, Katalin F. and Kolarich, Daniel and Packer, Nicolle H. and Thaysen-Andersen, Morten},
  doi          = {10.1038/s41592-021-01368-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {130},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Community evaluation of glycoproteomics informatics solutions reveals high-performance search strategies for serum glycopeptide analysis},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Deep learning improves macromolecule
identification in 3D cellular cryo-electron tomograms. <em>NMETH</em>,
<em>19</em>(1), 129. (<a
href="https://doi.org/10.1038/s41592-021-01349-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Moebel, Emmanuel and Martinez-Sanchez, Antonio and Lamm, Lorenz and Righetto, Ricardo D. and Wietrzynski, Wojciech and Albert, Sahradha and Larivière, Damien and Fourmentin, Eric and Pfeffer, Stefan and Ortiz, Julio and Baumeister, Wolfgang and Peng, Tingying and Engel, Benjamin D. and Kervrann, Charles},
  doi          = {10.1038/s41592-021-01349-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {129},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Deep learning improves macromolecule identification in 3D cellular cryo-electron tomograms},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: A 4D single-cell protein atlas of
transcription factors delineates spatiotemporal patterning during
embryogenesis. <em>NMETH</em>, <em>19</em>(1), 129. (<a
href="https://doi.org/10.1038/s41592-021-01366-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Ma, Xuehua and Zhao, Zhiguang and Xiao, Long and Xu, Weina and Kou, Yahui and Zhang, Yanping and Wu, Gang and Wang, Yangyang and Du, Zhuo},
  doi          = {10.1038/s41592-021-01366-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {129},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: A 4D single-cell protein atlas of transcription factors delineates spatiotemporal patterning during embryogenesis},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). FlyWire: Online community for whole-brain connectomics.
<em>NMETH</em>, <em>19</em>(1), 119–128. (<a
href="https://doi.org/10.1038/s41592-021-01330-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Due to advances in automated image acquisition and analysis, whole-brain connectomes with 100,000 or more neurons are on the horizon. Proofreading of whole-brain automated reconstructions will require many person-years of effort, due to the huge volumes of data involved. Here we present FlyWire, an online community for proofreading neural circuits in a Drosophila melanogaster brain and explain how its computational and social structures are organized to scale up to whole-brain connectomics. Browser-based three-dimensional interactive segmentation by collaborative editing of a spatially chunked supervoxel graph makes it possible to distribute proofreading to individuals located virtually anywhere in the world. Information in the edit history is programmatically accessible for a variety of uses such as estimating proofreading accuracy or building incentive systems. An open community accelerates proofreading by recruiting more participants and accelerates scientific discovery by requiring information sharing. We demonstrate how FlyWire enables circuit analysis by reconstructing and analyzing the connectome of mechanosensory neurons. FlyWire is an online community and a platform for proofreading electron microscopy-based connectome data of the Drosophila brain.},
  archive      = {J_NMETH},
  author       = {Dorkenwald, Sven and McKellar, Claire E. and Macrina, Thomas and Kemnitz, Nico and Lee, Kisuk and Lu, Ran and Wu, Jingpeng and Popovych, Sergiy and Mitchell, Eric and Nehoran, Barak and Jia, Zhen and Bae, J. Alexander and Mu, Shang and Ih, Dodam and Castro, Manuel and Ogedengbe, Oluwaseun and Halageri, Akhilesh and Kuehner, Kai and Sterling, Amy R. and Ashwood, Zoe and Zung, Jonathan and Brittain, Derrick and Collman, Forrest and Schneider-Mizell, Casey and Jordan, Chris and Silversmith, William and Baker, Christa and Deutsch, David and Encarnacion-Rivera, Lucas and Kumar, Sandeep and Burke, Austin and Bland, Doug and Gager, Jay and Hebditch, James and Koolman, Selden and Moore, Merlin and Morejohn, Sarah and Silverman, Ben and Willie, Kyle and Willie, Ryan and Yu, Szi-chieh and Murthy, Mala and Seung, H. Sebastian},
  doi          = {10.1038/s41592-021-01330-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {119-128},
  shortjournal = {Nat. Meth.},
  title        = {FlyWire: Online community for whole-brain connectomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cross-modal coherent registration of whole mouse brains.
<em>NMETH</em>, <em>19</em>(1), 111–118. (<a
href="https://doi.org/10.1038/s41592-021-01334-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent whole-brain mapping projects are collecting large-scale three-dimensional images using modalities such as serial two-photon tomography, fluorescence micro-optical sectioning tomography, light-sheet fluorescence microscopy, volumetric imaging with synchronous on-the-fly scan and readout or magnetic resonance imaging. Registration of these multi-dimensional whole-brain images onto a standard atlas is essential for characterizing neuron types and constructing brain wiring diagrams. However, cross-modal image registration is challenging due to intrinsic variations of brain anatomy and artifacts resulting from different sample preparation methods and imaging modalities. We introduce a cross-modal registration method, mBrainAligner, which uses coherent landmark mapping and deep neural networks to align whole mouse brain images to the standard Allen Common Coordinate Framework atlas. We build a brain atlas for the fluorescence micro-optical sectioning tomography modality to facilitate single-cell mapping, and used our method to generate a whole-brain map of three-dimensional single-neuron morphology and neuron cell types. mBrainAligner is a cross-modal registration platform for whole mouse brains imaged with different modalities. In addition, a fluorescence micro-optical sectioning tomography-based mouse brain atlas has been generated.},
  archive      = {J_NMETH},
  author       = {Qu, Lei and Li, Yuanyuan and Xie, Peng and Liu, Lijuan and Wang, Yimin and Wu, Jun and Liu, Yu and Wang, Tao and Li, Longfei and Guo, Kaixuan and Wan, Wan and Ouyang, Lei and Xiong, Feng and Kolstad, Anna C. and Wu, Zhuhao and Xu, Fang and Zheng, Yefeng and Gong, Hui and Luo, Qingming and Bi, Guoqiang and Dong, Hongwei and Hawrylycz, Michael and Zeng, Hongkui and Peng, Hanchuan},
  doi          = {10.1038/s41592-021-01334-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {111-118},
  shortjournal = {Nat. Meth.},
  title        = {Cross-modal coherent registration of whole mouse brains},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Fast optical recording of neuronal activity by
three-dimensional custom-access serial holography. <em>NMETH</em>,
<em>19</em>(1), 100–110. (<a
href="https://doi.org/10.1038/s41592-021-01329-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical recording of neuronal activity in three-dimensional (3D) brain circuits at cellular and millisecond resolution in vivo is essential for probing information flow in the brain. While random-access multiphoton microscopy permits fast optical access to neuronal targets in three dimensions, the method is challenged by motion artifacts when recording from behaving animals. Therefore, we developed three-dimensional custom-access serial holography (3D-CASH). Built on a fast acousto-optic light modulator, 3D-CASH performs serial sampling at 40 kHz from neurons at freely selectable 3D locations. Motion artifacts are eliminated by targeting each neuron with a size-optimized pattern of excitation light covering the cell body and its anticipated displacement field. Spike rates inferred from GCaMP6f recordings in visual cortex of awake mice tracked the phase of a moving bar stimulus with higher spike correlation between intra compared to interlaminar neuron pairs. 3D-CASH offers access to the millisecond correlation structure of in vivo neuronal activity in 3D microcircuits. 3D-CASH is a random-access microscopy approach that avoids in vivo motion artifacts by sampling each targeted neuron with a holographically shaped grid of illumination spots. The technology allows recording neuronal activity in the mouse cortex at a throughput of 20,000 neurons per second.},
  archive      = {J_NMETH},
  author       = {Akemann, Walther and Wolf, Sébastien and Villette, Vincent and Mathieu, Benjamin and Tangara, Astou and Fodor, Jozsua and Ventalon, Cathie and Léger, Jean-François and Dieudonné, Stéphane and Bourdieu, Laurent},
  doi          = {10.1038/s41592-021-01329-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {100-110},
  shortjournal = {Nat. Meth.},
  title        = {Fast optical recording of neuronal activity by three-dimensional custom-access serial holography},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Lineage recording in human cerebral organoids.
<em>NMETH</em>, <em>19</em>(1), 90–99. (<a
href="https://doi.org/10.1038/s41592-021-01344-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Induced pluripotent stem cell (iPSC)-derived organoids provide models to study human organ development. Single-cell transcriptomics enable highly resolved descriptions of cell states within these systems; however, approaches are needed to directly measure lineage relationships. Here we establish iTracer, a lineage recorder that combines reporter barcodes with inducible CRISPR–Cas9 scarring and is compatible with single-cell and spatial transcriptomics. We apply iTracer to explore clonality and lineage dynamics during cerebral organoid development and identify a time window of fate restriction as well as variation in neurogenic dynamics between progenitor neuron families. We also establish long-term four-dimensional light-sheet microscopy for spatial lineage recording in cerebral organoids and confirm regional clonality in the developing neuroepithelium. We incorporate gene perturbation (iTracer-perturb) and assess the effect of mosaic TSC2 mutations on cerebral organoid development. Our data shed light on how lineages and fates are established during cerebral organoid formation. More broadly, our techniques can be adapted in any iPSC-derived culture system to dissect lineage alterations during normal or perturbed development. A dual-channel recording system for high-resolution lineage tracing.},
  archive      = {J_NMETH},
  author       = {He, Zhisong and Maynard, Ashley and Jain, Akanksha and Gerber, Tobias and Petri, Rebecca and Lin, Hsiu-Chuan and Santel, Malgorzata and Ly, Kevin and Dupré, Jean-Samuel and Sidow, Leila and Sanchis Calleja, Fatima and Jansen, Sophie M. J. and Riesenberg, Stephan and Camp, J. Gray and Treutlein, Barbara},
  doi          = {10.1038/s41592-021-01344-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {90-99},
  shortjournal = {Nat. Meth.},
  title        = {Lineage recording in human cerebral organoids},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Rapid, efficient and activation-neutral gene editing of
polyclonal primary human resting CD4+ t cells allows complex functional
analyses. <em>NMETH</em>, <em>19</em>(1), 81–89. (<a
href="https://doi.org/10.1038/s41592-021-01328-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CD4+ T cells are central mediators of adaptive and innate immune responses and constitute a major reservoir for human immunodeficiency virus (HIV) in vivo. Detailed investigations of resting human CD4+ T cells have been precluded by the absence of efficient approaches for genetic manipulation limiting our understanding of HIV replication and restricting efforts to find a cure. Here we report a method for rapid, efficient, activation-neutral gene editing of resting, polyclonal human CD4+ T cells using optimized cell cultivation and nucleofection conditions of Cas9–guide RNA ribonucleoprotein complexes. Up to six genes, including HIV dependency and restriction factors, were knocked out individually or simultaneously and functionally characterized. Moreover, we demonstrate the knock in of double-stranded DNA donor templates into different endogenous loci, enabling the study of the physiological interplay of cellular and viral components at single-cell resolution. Together, this technique allows improved molecular and functional characterizations of HIV biology and general immune functions in resting CD4+ T cells. A nucleofection-based method to enable efficient gene editing of primary human resting CD4+ T cells.},
  archive      = {J_NMETH},
  author       = {Albanese, Manuel and Ruhle, Adrian and Mittermaier, Jennifer and Mejías-Pérez, Ernesto and Gapp, Madeleine and Linder, Andreas and Schmacke, Niklas A. and Hofmann, Katharina and Hennrich, Alexandru A. and Levy, David N. and Humpe, Andreas and Conzelmann, Karl-Klaus and Hornung, Veit and Fackler, Oliver T. and Keppler, Oliver T.},
  doi          = {10.1038/s41592-021-01328-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {81-89},
  shortjournal = {Nat. Meth.},
  title        = {Rapid, efficient and activation-neutral gene editing of polyclonal primary human resting CD4+ t cells allows complex functional analyses},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Monitoring protein conformational changes using fluorescent
nanoantennas. <em>NMETH</em>, <em>19</em>(1), 71–80. (<a
href="https://doi.org/10.1038/s41592-021-01355-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the relationship between protein structural dynamics and function is crucial for both basic research and biotechnology. However, methods for studying the fast dynamics of structural changes are limited. Here, we introduce fluorescent nanoantennas as a spectroscopic technique to sense and report protein conformational changes through noncovalent dye-protein interactions. Using experiments and molecular simulations, we detect and characterize five distinct conformational states of intestinal alkaline phosphatase, including the transient enzyme–substrate complex. We also explored the universality of the nanoantenna strategy with another model protein, Protein G and its interaction with antibodies, and demonstrated a rapid screening strategy to identify efficient nanoantennas. These versatile nanoantennas can be used with diverse dyes to monitor small and large conformational changes, suggesting that they could be used to characterize diverse protein movements or in high-throughput screening applications. Fluorescent nanoantennas represent a versatile detection strategy for monitoring fast, large- and small-scale protein dynamics.},
  archive      = {J_NMETH},
  author       = {Harroun, Scott G. and Lauzon, Dominic and Ebert, Maximilian C. C. J. C. and Desrosiers, Arnaud and Wang, Xiaomeng and Vallée-Bélisle, Alexis},
  doi          = {10.1038/s41592-021-01355-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {71-80},
  shortjournal = {Nat. Meth.},
  title        = {Monitoring protein conformational changes using fluorescent nanoantennas},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Engineered HaloTag variants for fluorescence lifetime
multiplexing. <em>NMETH</em>, <em>19</em>(1), 65–70. (<a
href="https://doi.org/10.1038/s41592-021-01341-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Self-labeling protein tags such as HaloTag are powerful tools that can label fusion proteins with synthetic fluorophores for use in fluorescence microscopy. Here we introduce HaloTag variants with either increased or decreased brightness and fluorescence lifetime compared with HaloTag7 when labeled with rhodamines. Combining these HaloTag variants enabled live-cell fluorescence lifetime multiplexing of three cellular targets in one spectral channel using a single fluorophore and the generation of a fluorescence lifetime-based biosensor. Additionally, the brightest HaloTag variant showed up to 40\% higher brightness in live-cell imaging applications. HaloTag variants offer distinct brightness and fluorescence lifetimes compared with HaloTag7 when labeled with rhodamines. These variants were used for multiplexed imaging with a single fluorophore and to create lifetime-based cell cycle indicators.},
  archive      = {J_NMETH},
  author       = {Frei, Michelle S. and Tarnawski, Miroslaw and Roberti, M. Julia and Koch, Birgit and Hiblot, Julien and Johnsson, Kai},
  doi          = {10.1038/s41592-021-01341-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {65-70},
  shortjournal = {Nat. Meth.},
  title        = {Engineered HaloTag variants for fluorescence lifetime multiplexing},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Systematic molecular evolution enables robust biomolecule
discovery. <em>NMETH</em>, <em>19</em>(1), 55–64. (<a
href="https://doi.org/10.1038/s41592-021-01348-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolution occurs when selective pressures from the environment shape inherited variation over time. Within the laboratory, evolution is commonly used to engineer proteins and RNA, but experimental constraints have limited the ability to reproducibly and reliably explore factors such as population diversity, the timing of environmental changes and chance on outcomes. We developed a robotic system termed phage- and robotics-assisted near-continuous evolution (PRANCE) to comprehensively explore biomolecular evolution by performing phage-assisted continuous evolution in high-throughput. PRANCE implements an automated feedback control system that adjusts the stringency of selection in response to real-time measurements of each molecular activity. In evolving three distinct types of biomolecule, we find that evolution is reproducibly altered by both random chance and the historical pattern of environmental changes. This work improves the reliability of protein engineering and enables the systematic analysis of the historical, environmental and random factors governing biomolecular evolution. Phage and robotics-assisted near-continuous evolution enables phage-assisted continuous evolution in high throughput, allowing for improved exploration of sequence space and insight into how variables affect evolution outcomes.},
  archive      = {J_NMETH},
  author       = {DeBenedictis, Erika A. and Chory, Emma J. and Gretton, Dana W. and Wang, Brian and Golas, Stefan and Esvelt, Kevin M.},
  doi          = {10.1038/s41592-021-01348-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {55-64},
  shortjournal = {Nat. Meth.},
  title        = {Systematic molecular evolution enables robust biomolecule discovery},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Fast and flexible analysis of linked microbiome data with
mako. <em>NMETH</em>, <em>19</em>(1), 51–54. (<a
href="https://doi.org/10.1038/s41592-021-01335-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mako is a software tool that converts microbiome data and networks into a graph database and visualizes query results, thus allowing users without programming knowledge to carry out network-based queries. Mako is accompanied by a database compiled from 60 microbiome studies that is easily extended with the user’s own data. We illustrate mako’s strengths by enumerating association partners linked to propionate production and comparing frequencies of different network motifs across habitat types. This work describes mako, a tool for efficiently constructing and querying network databases with large-scale microbiome data.},
  archive      = {J_NMETH},
  author       = {Röttjers, Lisa and Faust, Karoline},
  doi          = {10.1038/s41592-021-01335-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {51-54},
  shortjournal = {Nat. Meth.},
  title        = {Fast and flexible analysis of linked microbiome data with mako},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Benchmarking atlas-level data integration in single-cell
genomics. <em>NMETH</em>, <em>19</em>(1), 41–50. (<a
href="https://doi.org/10.1038/s41592-021-01336-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell atlases often include samples that span locations, laboratories and conditions, leading to complex, nested batch effects in data. Thus, joint analysis of atlas datasets requires reliable data integration. To guide integration method choice, we benchmarked 68 method and preprocessing combinations on 85 batches of gene expression, chromatin accessibility and simulation data from 23 publications, altogether representing &amp;gt;1.2 million cells distributed in 13 atlas-level integration tasks. We evaluated methods according to scalability, usability and their ability to remove batch effects while retaining biological variation using 14 evaluation metrics. We show that highly variable gene selection improves the performance of data integration methods, whereas scaling pushes methods to prioritize batch removal over conservation of biological variation. Overall, scANVI, Scanorama, scVI and scGen perform well, particularly on complex integration tasks, while single-cell ATAC-sequencing integration performance is strongly affected by choice of feature space. Our freely available Python module and benchmarking pipeline can identify optimal data integration methods for new data, benchmark new methods and improve method development. This benchmarking study compares 16 methods for integrating complex single-cell RNA and ATAC datasets and provides a guide to method choice.},
  archive      = {J_NMETH},
  author       = {Luecken, Malte D. and Büttner, M. and Chaichoompu, K. and Danese, A. and Interlandi, M. and Mueller, M. F. and Strobl, D. C. and Zappia, L. and Dugas, M. and Colomé-Tatché, M. and Theis, Fabian J.},
  doi          = {10.1038/s41592-021-01336-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {41-50},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking atlas-level data integration in single-cell genomics},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). New dual-channel system records lineage in high definition.
<em>NMETH</em>, <em>19</em>(1), 38–39. (<a
href="https://doi.org/10.1038/s41592-021-01340-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Introducing iTracer, a system for long-term measurement of lineage dynamics at high resolution.},
  archive      = {J_NMETH},
  author       = {Jindal, Kunal and VanHorn, Sadie and Morris, Samantha A.},
  doi          = {10.1038/s41592-021-01340-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {38-39},
  shortjournal = {Nat. Meth.},
  title        = {New dual-channel system records lineage in high definition},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Editing naive CD4+ t cells. <em>NMETH</em>, <em>19</em>(1),
36–37. (<a href="https://doi.org/10.1038/s41592-021-01332-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As last we can edit the immune system’s sleeping giants, as CRISPR tools advance into the world of naive CD4+ T cells.},
  archive      = {J_NMETH},
  author       = {Papadopoulos, Andrea Olga and Ndhlovu, Zaza Mtine},
  doi          = {10.1038/s41592-021-01332-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {36-37},
  shortjournal = {Nat. Meth.},
  title        = {Editing naive CD4+ t cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022m). Recapitulating miRNA biogenesis in cells. <em>NMETH</em>,
<em>19</em>(1), 35. (<a
href="https://doi.org/10.1038/s41592-021-01385-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MapToCleave enables the large-scale study of miRNA precursor processing in living cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01385-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {35},
  shortjournal = {Nat. Meth.},
  title        = {Recapitulating miRNA biogenesis in cells},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Resolving human gastrulation. <em>NMETH</em>,
<em>19</em>(1), 34. (<a
href="https://doi.org/10.1038/s41592-021-01384-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first single-cell transcriptome of the gastrulating human embryo.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01384-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {34},
  shortjournal = {Nat. Meth.},
  title        = {Resolving human gastrulation},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Estimating selection in healthy tissues. <em>NMETH</em>,
<em>19</em>(1), 33. (<a
href="https://doi.org/10.1038/s41592-021-01383-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Frequencies of passenger mutations hint at unobserved driver mutations.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01383-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {33},
  shortjournal = {Nat. Meth.},
  title        = {Estimating selection in healthy tissues},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Flexibility in mesoscopic imaging. <em>NMETH</em>,
<em>19</em>(1), 32. (<a
href="https://doi.org/10.1038/s41592-021-01382-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two microscopy approaches bring flexibility to mesoscopic imaging in the brain, allowing independent imaging in multiple regions simultaneously.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01382-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {32},
  shortjournal = {Nat. Meth.},
  title        = {Flexibility in mesoscopic imaging},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Reading proteins with nanopores. <em>NMETH</em>,
<em>19</em>(1), 31. (<a
href="https://doi.org/10.1038/s41592-021-01381-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two studies use nanopores for single-protein fingerprinting and make headway toward single-protein sequencing.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01381-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {31},
  shortjournal = {Nat. Meth.},
  title        = {Reading proteins with nanopores},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Imaging without the labels. <em>NMETH</em>, <em>19</em>(1),
30. (<a href="https://doi.org/10.1038/s41592-021-01376-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in label-free microscopy are expanding experimental observations, from biophysics to cell biology and beyond.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-021-01376-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Imaging without the labels},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Greater detail in the nucleus. <em>NMETH</em>,
<em>19</em>(1), 30. (<a
href="https://doi.org/10.1038/s41592-021-01377-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Probing spatially organized DNA and its interacting elements in single cells will deepen our understanding of cell-type-specific gene regulation.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01377-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nat. Meth.},
  title        = {Greater detail in the nucleus},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Genomics beyond complete genomes. <em>NMETH</em>,
<em>19</em>(1), 29. (<a
href="https://doi.org/10.1038/s41592-021-01374-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Method development is indispensable for obtaining complete genomes, and deciphering them.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-021-01374-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {Genomics beyond complete genomes},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A dynamic direction for cryo-EM. <em>NMETH</em>,
<em>19</em>(1), 29. (<a
href="https://doi.org/10.1038/s41592-021-01375-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emerging algorithms are extracting information about macromolecular motions from cryo-EM data.},
  archive      = {J_NMETH},
  author       = {Doerr, Allison},
  doi          = {10.1038/s41592-021-01375-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nat. Meth.},
  title        = {A dynamic direction for cryo-EM},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Deconstructing subcellular composition. <em>NMETH</em>,
<em>19</em>(1), 28. (<a
href="https://doi.org/10.1038/s41592-021-01372-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-throughput and ultrasensitive mass spectrometry–based approaches are gaining ground in subcellular compositional analysis.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-021-01372-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {28},
  shortjournal = {Nat. Meth.},
  title        = {Deconstructing subcellular composition},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Neuroscience goes viral. <em>NMETH</em>, <em>19</em>(1),
28. (<a href="https://doi.org/10.1038/s41592-021-01373-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Viral tools are a mainstay in mammalian neuroscience, but there is room for improvement.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-021-01373-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {28},
  shortjournal = {Nat. Meth.},
  title        = {Neuroscience goes viral},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Tracing cell relationships. <em>NMETH</em>, <em>19</em>(1),
27. (<a href="https://doi.org/10.1038/s41592-021-01370-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lineage tracing methods help unravel complexities of cell state and differentiation programs.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-021-01370-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {27},
  shortjournal = {Nat. Meth.},
  title        = {Tracing cell relationships},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022o). Seeking more nucleases. <em>NMETH</em>, <em>19</em>(1), 27.
(<a href="https://doi.org/10.1038/s41592-021-01371-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The diverse microbial world provides unprecedented potential for exploring genome editing tools.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-021-01371-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {27},
  shortjournal = {Nat. Meth.},
  title        = {Seeking more nucleases},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Structural biology is solved — now what? <em>NMETH</em>,
<em>19</em>(1), 24–26. (<a
href="https://doi.org/10.1038/s41592-021-01357-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The splendid computational success of AlphaFold and RoseTTAFold in solving the 60-year-old problem of protein folding raises an obvious question: what new avenues should structural biology explore? We propose a strong pivot toward the goal of reading mechanism and function directly from the amino acid sequence. This ambitious goal will require new data analytical tools and an extensive database of the atomic-level structural trajectories traced out on energy landscapes as proteins perform their function.},
  archive      = {J_NMETH},
  author       = {Ourmazd, Abbas and Moffat, Keith and Lattman, Eaton Edward},
  doi          = {10.1038/s41592-021-01357-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {24-26},
  shortjournal = {Nat. Meth.},
  title        = {Structural biology is solved — now what?},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A paradigm shift in structural biology. <em>NMETH</em>,
<em>19</em>(1), 20–23. (<a
href="https://doi.org/10.1038/s41592-021-01361-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The release of protein structure predictions from AlphaFold will increase the number of protein structural models by almost three orders of magnitude. Structural biology and bioinformatics will never be the same, and the need for incisive experimental approaches will be greater than ever. Combining these advances in structure prediction with recent advances in cryo-electron microscopy suggests a new paradigm for structural biology.},
  archive      = {J_NMETH},
  author       = {Subramaniam, Sriram and Kleywegt, Gerard J.},
  doi          = {10.1038/s41592-021-01361-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {20-23},
  shortjournal = {Nat. Meth.},
  title        = {A paradigm shift in structural biology},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The impact of AlphaFold2 one year on. <em>NMETH</em>,
<em>19</em>(1), 15–20. (<a
href="https://doi.org/10.1038/s41592-021-01365-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The greatly improved prediction of protein 3D structure from sequence achieved by the second version of AlphaFold in 2020 has already had a huge impact on biological research, but challenges remain; the protein folding problem cannot be considered solved. We expect fierce competition to improve the method even further and new applications of machine learning to help illuminate proteomes and their many interactions.},
  archive      = {J_NMETH},
  author       = {Jones, David T. and Thornton, Janet M.},
  doi          = {10.1038/s41592-021-01365-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {15-20},
  shortjournal = {Nat. Meth.},
  title        = {The impact of AlphaFold2 one year on},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep learning and protein structure modeling.
<em>NMETH</em>, <em>19</em>(1), 13–14. (<a
href="https://doi.org/10.1038/s41592-021-01360-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning has transformed protein structure modeling. Here we relate AlphaFold and RoseTTAFold to classical physically based approaches to protein structure prediction, and discuss the many areas of structural biology that are likely to be affected by further advances in deep learning.},
  archive      = {J_NMETH},
  author       = {Baek, Minkyung and Baker, David},
  doi          = {10.1038/s41592-021-01360-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {13-14},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning and protein structure modeling},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Protein structure predictions to atomic accuracy with
AlphaFold. <em>NMETH</em>, <em>19</em>(1), 11–12. (<a
href="https://doi.org/10.1038/s41592-021-01362-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AlphaFold is a neural-network-based approach to predicting protein structures with high accuracy. We describe how it works in general terms and discuss some anticipated impacts on the field of structural biology.},
  archive      = {J_NMETH},
  author       = {Jumper, John and Hassabis, Demis},
  doi          = {10.1038/s41592-021-01362-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {11-12},
  shortjournal = {Nat. Meth.},
  title        = {Protein structure predictions to atomic accuracy with AlphaFold},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022n). Method of the year: Protein structure prediction.
<em>NMETH</em>, <em>19</em>(1), 5–10. (<a
href="https://doi.org/10.1038/s41592-021-01359-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Methods has named protein structure prediction the Method of the Year 2021.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01359-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {5-10},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year: Protein structure prediction},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022j). Karoline faust. <em>NMETH</em>, <em>19</em>(1), 3. (<a
href="https://doi.org/10.1038/s41592-021-01367-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A tool to make microbial network-oriented deep dives easier, and caring about equality.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-021-01367-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {3},
  shortjournal = {Nat. Meth.},
  title        = {Karoline faust},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Method of the year 2021: Protein structure prediction.
<em>NMETH</em>, <em>19</em>(1), 1. (<a
href="https://doi.org/10.1038/s41592-021-01380-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep Learning based approaches for protein structure prediction have sent shock waves through the structural biology community. We anticipate far-reaching and long-lasting impact.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-021-01380-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year 2021: Protein structure prediction},
  volume       = {19},
  year         = {2022},
}
</textarea>
</details></li>
</ul>

</body>
</html>
