<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATEE_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natee---327">NATEE - 327</h2>
<ul>
<li><details>
<summary>
(2022). Evidence for the cooking of fish 780,000 years ago at gesher
benot ya’aqov, israel. <em>NATEE</em>, <em>6</em>(12), 2016–2028. (<a
href="https://doi.org/10.1038/s41559-022-01910-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although cooking is regarded as a key element in the evolutionary success of the genus Homo, impacting various biological and social aspects, when intentional cooking first began remains unknown. The early Middle Pleistocene site of Gesher Benot Ya’aqov, Israel (marine isotope stages 18–20; ~0.78 million years ago), has preserved evidence of hearth-related hominin activities and large numbers of freshwater fish remains (&amp;gt;40,000). A taphonomic study and isotopic analyses revealed significant differences between the characteristics of the fish bone assemblages recovered in eight sequential archaeological horizons of Area B (Layer II-6 levels 1–7) and natural fish bone assemblages (identified in Area A). Gesher Benot Ya’aqov archaeological horizons II-6 L1–7 exhibited low fish species richness, with a clear preference for two species of large Cyprinidae (Luciobarbus longiceps and Carasobarbus canis) and the almost total absence of fish bones in contrast to the richness of pharyngeal teeth (&amp;gt;95%). Most of the pharyngeal teeth recovered in archaeological horizons II-6 L1–7 were spatially associated with ‘phantom’ hearths (clusters of burnt flint microartifacts). Size–strain analysis using X-ray powder diffraction provided evidence that these teeth had been exposed to low temperature (&amp;lt;500 °C), suggesting, together with the archaeological and taphonomic data, that the fish from the archaeological horizons of Area B had been cooked and consumed on site. This is the earliest evidence of cooking by hominins. Taphonomic and stable isotopic analysis of fish bone assemblages at the early Middle Pleistocene site of Gesher Benot Ya&#39;aqov in Israel suggest that fish were cooked before consumption by hominins.},
  archive      = {J_NATEE},
  author       = {Zohar, Irit and Alperson-Afil, Nira and Goren-Inbar, Naama and Prévost, Marion and Tütken, Thomas and Sisma-Ventura, Guy and Hershkovitz, Israel and Najorka, Jens},
  doi          = {10.1038/s41559-022-01910-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2016-2028},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evidence for the cooking of fish 780,000 years ago at gesher benot ya’aqov, israel},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Admixture has obscured signals of historical hard sweeps in
humans. <em>NATEE</em>, <em>6</em>(12), 2003–2015. (<a
href="https://doi.org/10.1038/s41559-022-01914-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The role of natural selection in shaping biological diversity is an area of intense interest in modern biology. To date, studies of positive selection have primarily relied on genomic datasets from contemporary populations, which are susceptible to confounding factors associated with complex and often unknown aspects of population history. In particular, admixture between diverged populations can distort or hide prior selection events in modern genomes, though this process is not explicitly accounted for in most selection studies despite its apparent ubiquity in humans and other species. Through analyses of ancient and modern human genomes, we show that previously reported Holocene-era admixture has masked more than 50 historic hard sweeps in modern European genomes. Our results imply that this canonical mode of selection has probably been underappreciated in the evolutionary history of humans and suggest that our current understanding of the tempo and mode of selection in natural populations may be inaccurate. Through analyses of ancient and modern human genomes, the authors show that previously reported Holocene-era admixture has masked more than 50 historic hard sweeps in modern European genomes.},
  archive      = {J_NATEE},
  author       = {Souilmi, Yassine and Tobler, Raymond and Johar, Angad and Williams, Matthew and Grey, Shane T. and Schmidt, Joshua and Teixeira, João C. and Rohrlach, Adam and Tuke, Jonathan and Johnson, Olivia and Gower, Graham and Turney, Chris and Cox, Murray and Cooper, Alan and Huber, Christian D.},
  doi          = {10.1038/s41559-022-01914-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2003-2015},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Admixture has obscured signals of historical hard sweeps in humans},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mito-nuclear selection induces a trade-off between species
ecological dominance and evolutionary lifespan. <em>NATEE</em>,
<em>6</em>(12), 1992–2002. (<a
href="https://doi.org/10.1038/s41559-022-01901-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mitochondrial and nuclear genomes must be co-adapted to ensure proper cellular respiration and energy production. Mito-nuclear incompatibility reduces individual fitness and induces hybrid infertility, which can drive reproductive barriers and speciation. Here, we develop a birth–death model for evolution in spatially extended populations under selection for mito-nuclear co-adaptation. Mating is constrained by physical and genetic proximity, and offspring inherit nuclear genomes from both parents, with recombination. The model predicts macroscopic patterns including a community’s species diversity, species abundance distribution, speciation and extinction rates, as well as intraspecific and interspecific genetic variation. We explore how these long-term outcomes depend upon the parameters of reproduction: individual fitness governed by mito-nuclear compatibility, constraints on mating compatibility and ecological carrying capacity. We find that strong selection for mito-nuclear compatibility reduces the equilibrium number of species after a radiation, increasing species’ abundances and simultaneously increasing both speciation and extinction rates. The negative correlation between species diversity and diversification rates in our model agrees with the broad empirical pattern of lower diversity and higher speciation/extinction rates in temperate regions, compared to the tropics. We conclude that these empirical patterns may be caused in part by latitudinal variation in metabolic demands and corresponding variation in selection for mito-nuclear function. A theoretical model of a community of species under selection for compatibility between nuclear and mitochondrial genomes predicts patterns of species diversity, abundance, speciation and extinction, and also suggests a link between metabolic demands and latitudinal variation in diversity.},
  archive      = {J_NATEE},
  author       = {Princepe, Débora and de Aguiar, Marcus A. M. and Plotkin, Joshua B.},
  doi          = {10.1038/s41559-022-01901-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1992-2002},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mito-nuclear selection induces a trade-off between species ecological dominance and evolutionary lifespan},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Within-patient evolution of plasmid-mediated antimicrobial
resistance. <em>NATEE</em>, <em>6</em>(12), 1980–1991. (<a
href="https://doi.org/10.1038/s41559-022-01908-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antimicrobial resistance (AMR) in bacteria is a major threat to public health; one of the key elements in the spread and evolution of AMR in clinical pathogens is the transfer of conjugative plasmids. The drivers of AMR evolution have been studied extensively in vitro but the evolution of plasmid-mediated AMR in vivo remains poorly explored. Here, we tracked the evolution of the clinically relevant plasmid pOXA-48, which confers resistance to the last-resort antibiotics carbapenems, in a large collection of enterobacterial clones isolated from the gut of hospitalized patients. Combining genomic and experimental approaches, we first characterized plasmid diversity and the genotypic and phenotypic effects of multiple plasmid mutations on a common genetic background. Second, using cutting-edge genomic editing in wild-type multidrug-resistant enterobacteria, we dissected three cases of within-patient plasmid-mediated AMR evolution. Our results revealed compensatory evolution of plasmid-associated fitness cost and the evolution of enhanced plasmid-mediated AMR in bacteria evolving in the gut of hospitalized patients. Crucially, we observed that the evolution of pOXA-48-mediated AMR in vivo involves a pivotal trade-off between resistance levels and bacterial fitness. This study highlights the need to develop new evolution-informed approaches to tackle plasmid-mediated AMR dissemination. A combination of analysis of plasmid diversity in the gut of hospitalized patients with experimental evolution shows that the evolution of plasmid-mediated antibiotic resistance involves a trade-off between antibiotic resistance levels and bacterial fitness.},
  archive      = {J_NATEE},
  author       = {DelaFuente, Javier and Toribio-Celestino, Laura and Santos-Lopez, Alfonso and León-Sampedro, Ricardo and Alonso-del Valle, Aida and Costas, Coloma and Hernández-García, Marta and Cui, Lun and Rodríguez-Beltrán, Jerónimo and Bikard, David and Cantón, Rafael and San Millan, Alvaro},
  doi          = {10.1038/s41559-022-01908-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1980-1991},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Within-patient evolution of plasmid-mediated antimicrobial resistance},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chromosomal inversion polymorphisms shape the genomic
landscape of deer mice. <em>NATEE</em>, <em>6</em>(12), 1965–1979. (<a
href="https://doi.org/10.1038/s41559-022-01890-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosomal inversions are an important form of structural variation that can affect recombination, chromosome structure and fitness. However, because inversions can be challenging to detect, the prevalence and hence the significance of inversions segregating within species remains largely unknown, especially in natural populations of mammals. Here, by combining population-genomic and long-read sequencing analyses in a single, widespread species of deer mouse (Peromyscus maniculatus), we identified 21 polymorphic inversions that are large (1.5–43.8 Mb) and cause near-complete suppression of recombination when heterozygous (0–0.03 cM Mb−1). We found that inversion breakpoints frequently occur in centromeric and telomeric regions and are often flanked by long inverted repeats (0.5–50 kb), suggesting that they probably arose via ectopic recombination. By genotyping inversions in populations across the species’ range, we found that the inversions are often widespread and do not harbour deleterious mutational loads, and many are likely to be maintained as polymorphisms by divergent selection. Comparisons of forest and prairie ecotypes of deer mice revealed 13 inversions that contribute to differentiation between populations, of which five exhibit significant associations with traits implicated in local adaptation. Taken together, these results show that inversion polymorphisms have a significant impact on recombination, genome structure and genetic diversity in deer mice and likely facilitate local adaptation across the widespread range of this species. The identification of 21 large inversion polymorphisms in populations of deer mice shows that they are widespread, important for patterns of recombination and likely to be involved in local adaptation.},
  archive      = {J_NATEE},
  author       = {Harringmeyer, Olivia S. and Hoekstra, Hopi E.},
  doi          = {10.1038/s41559-022-01890-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1965-1979},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Chromosomal inversion polymorphisms shape the genomic landscape of deer mice},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Climatic similarity and genomic background shape the extent
of parallel adaptation in timema stick insects. <em>NATEE</em>,
<em>6</em>(12), 1952–1964. (<a
href="https://doi.org/10.1038/s41559-022-01909-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolution can repeat itself, resulting in parallel adaptations in independent lineages occupying similar environments. Moreover, parallel evolution sometimes, but not always, uses the same genes. Two main hypotheses have been put forth to explain the probability and extent of parallel evolution. First, parallel evolution is more likely when shared ecologies result in similar patterns of natural selection in different taxa. Second, parallelism is more likely when genomes are similar because of shared standing variation and similar mutational effects in closely related genomes. Here we combine ecological, genomic, experimental and phenotypic data with Bayesian modelling and randomization tests to quantify the degree of parallelism and its relationship with ecology and genetics. Our results show that the extent to which genomic regions associated with climate are parallel among species of Timema stick insects is shaped collectively by shared ecology and genomic background. Specifically, the extent of genomic parallelism decays with divergence in climatic conditions (that is, habitat or ecological similarity) and genomic similarity. Moreover, we find that climate-associated loci are likely subject to selection in a field experiment, overlap with genetic regions associated with cuticular hydrocarbon traits and are not strongly shaped by introgression between species. Our findings shed light on when evolution is most expected to repeat itself. Multiple factors contribute to the independent evolution of the same adaptations in nature. This study quantifies parallel evolution among species of Timema stick insects and shows that the degree of parallelism is determined by shared ecology and genomic background.},
  archive      = {J_NATEE},
  author       = {Chaturvedi, Samridhi and Gompert, Zachariah and Feder, Jeffrey L. and Osborne, Owen G. and Muschick, Moritz and Riesch, Rüdiger and Soria-Carrasco, Víctor and Nosil, Patrik},
  doi          = {10.1038/s41559-022-01909-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1952-1964},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Climatic similarity and genomic background shape the extent of parallel adaptation in timema stick insects},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Epigenetic divergence during early stages of speciation in
an african crater lake cichlid fish. <em>NATEE</em>, <em>6</em>(12),
1940–1951. (<a
href="https://doi.org/10.1038/s41559-022-01894-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetic variation can alter transcription and promote phenotypic divergence between populations facing different environmental challenges. Here, we assess the epigenetic basis of diversification during the early stages of speciation. Specifically, we focus on the extent and functional relevance of DNA methylome divergence in the very young radiation of Astatotilapia calliptera in crater Lake Masoko, southern Tanzania. Our study focuses on two lake ecomorphs that diverged approximately 1,000 years ago and a population in the nearby river from which they separated approximately 10,000 years ago. The two lake ecomorphs show no fixed genetic differentiation, yet are characterized by different morphologies, depth preferences and diets. We report extensive genome-wide methylome divergence between the two lake ecomorphs, and between the lake and river populations, linked to key biological processes and associated with altered transcriptional activity of ecologically relevant genes. Such genes differing between lake ecomorphs include those involved in steroid metabolism, hemoglobin composition and erythropoiesis, consistent with their divergent habitat occupancy. Using a common-garden experiment, we found that global methylation profiles are often rapidly remodeled across generations but ecomorph-specific differences can be inherited. Collectively, our study suggests an epigenetic contribution to the early stages of vertebrate speciation. Analysis of the DNA methylomes of two ecomorphs of Astatotilapia calliptera from a single lake, which diverged about 1,000 years ago plus a third riverine ecomorph, from which they likely separated about 10,000 years ago, shows epigenetic differences associated with altered transcriptional activity of ecologically relevant genes, despite low levels of genetic divergence.},
  archive      = {J_NATEE},
  author       = {Vernaz, Grégoire and Hudson, Alan G. and Santos, M. Emília and Fischer, Bettina and Carruthers, Madeleine and Shechonge, Asilatu H. and Gabagambi, Nestory P. and Tyers, Alexandra M. and Ngatunga, Benjamin P. and Malinsky, Milan and Durbin, Richard and Turner, George F. and Genner, Martin J. and Miska, Eric A.},
  doi          = {10.1038/s41559-022-01894-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1940-1951},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Epigenetic divergence during early stages of speciation in an african crater lake cichlid fish},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An ancestral wnt–brachyury feedback loop in axial patterning
and recruitment of mesoderm-determining target genes. <em>NATEE</em>,
<em>6</em>(12), 1921–1939. (<a
href="https://doi.org/10.1038/s41559-022-01905-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factors are crucial drivers of cellular differentiation during animal development and often share ancient evolutionary origins. The T-box transcription factor Brachyury plays a pivotal role as an early mesoderm determinant and neural repressor in vertebrates; yet, the ancestral function and key evolutionary transitions of the role of this transcription factor remain obscure. Here, we present a genome-wide target-gene screen using chromatin immunoprecipitation sequencing in the sea anemone Nematostella vectensis, an early branching non-bilaterian, and the sea urchin Strongylocentrotus purpuratus, a representative of the sister lineage of chordates. Our analysis reveals an ancestral gene regulatory feedback loop connecting Brachyury, FoxA and canonical Wnt signalling involved in axial patterning that predates the cnidarian–bilaterian split about 700 million years ago. Surprisingly, we also found that part of the gene regulatory network controlling the fate of neuromesodermal progenitors in vertebrates was already present in the common ancestor of cnidarians and bilaterians. However, while several endodermal and neuronal Brachyury target genes are ancestrally shared, hardly any of the key mesodermal downstream targets in vertebrates are found in the sea anemone or the sea urchin. Our study suggests that a limited number of target genes involved in mesoderm formation were newly acquired in the vertebrate lineage, leading to a dramatic shift in the function of this ancestral developmental regulator. Brachyury is an early mesoderm determinant and neural repressor in vertebrates. Comparative Brachyury target screens between a sea anemone and a sea urchin reveal an ancestral gene regulatory feedback loop involved in axial patterning, with conserved endodermal and neuronal, but not mesodermal, targets.},
  archive      = {J_NATEE},
  author       = {Schwaiger, Michaela and Andrikou, Carmen and Dnyansagar, Rohit and Murguia, Patricio Ferrer and Paganos, Periklis and Voronov, Danila and Zimmermann, Bob and Lebedeva, Tatiana and Schmidt, Heiko A. and Genikhovich, Grigory and Benvenuto, Giovanna and Arnone, Maria Ina and Technau, Ulrich},
  doi          = {10.1038/s41559-022-01905-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1921-1939},
  shortjournal = {Nat. Ecol. Evol},
  title        = {An ancestral Wnt–Brachyury feedback loop in axial patterning and recruitment of mesoderm-determining target genes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Recent reconfiguration of an ancient developmental gene
regulatory network in heliocidaris sea urchins. <em>NATEE</em>,
<em>6</em>(12), 1907–1920. (<a
href="https://doi.org/10.1038/s41559-022-01906-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Changes in developmental gene regulatory networks (dGRNs) underlie much of the diversity of life, but the evolutionary mechanisms that operate on regulatory interactions remain poorly understood. Closely related species with extreme phenotypic divergence provide a valuable window into the genetic and molecular basis for changes in dGRNs and their relationship to adaptive changes in organismal traits. Here we analyse genomes, epigenomes and transcriptomes during early development in two Heliocidaris sea urchin species that exhibit highly divergent life histories and in an outgroup species. Positive selection and chromatin accessibility modifications within putative regulatory elements are enriched on the branch leading to the derived life history, particularly near dGRN genes. Single-cell transcriptomes reveal a dramatic delay in cell fate specification in the derived state, which also has far fewer open chromatin regions, especially near conserved cell fate specification genes. Experimentally perturbing key transcription factors reveals profound evolutionary changes to early embryonic patterning events, disrupting regulatory interactions previously conserved for ~225 million years. These results demonstrate that natural selection can rapidly reshape developmental gene expression on a broad scale when selective regimes abruptly change. More broadly, even highly conserved dGRNs and patterning mechanisms in the early embryo remain evolvable under appropriate ecological circumstances. Analysis of genomes, epigenomes and transcriptomes of developing sea urchins with divergent life histories shows how natural selection can reshape developmental gene regulatory networks.},
  archive      = {J_NATEE},
  author       = {Davidson, Phillip L. and Guo, Haobing and Swart, Jane S. and Massri, Abdull J. and Edgar, Allison and Wang, Lingyu and Berrio, Alejandro and Devens, Hannah R. and Koop, Demian and Cisternas, Paula and Zhang, He and Zhang, Yaolei and Byrne, Maria and Fan, Guangyi and Wray, Gregory A.},
  doi          = {10.1038/s41559-022-01906-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1907-1920},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Recent reconfiguration of an ancient developmental gene regulatory network in heliocidaris sea urchins},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ancient homomorphy of molluscan sex chromosomes sustained by
reversible sex-biased genes and sex determiner translocation.
<em>NATEE</em>, <em>6</em>(12), 1891–1906. (<a
href="https://doi.org/10.1038/s41559-022-01898-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Contrary to classic theory prediction, sex-chromosome homomorphy is prevalent in the animal kingdom but it is unclear how ancient homomorphic sex chromosomes avoid chromosome-scale degeneration. Molluscs constitute the second largest, Precambrian-originated animal phylum and have ancient, uncharacterized homomorphic sex chromosomes. Here, we profile eight genomes of the bivalve mollusc family of Pectinidae in a phylogenetic context and show 350 million years sex-chromosome homomorphy, which is the oldest known sex-chromosome homomorphy in the animal kingdom, far exceeding the ages of well-known heteromorphic sex chromosomes such as 130–200 million years in mammals, birds and flies. The long-term undifferentiation of molluscan sex chromosomes is potentially sustained by the unexpected intertwined regulation of reversible sex-biased genes, together with the lack of sexual dimorphism and occasional sex chromosome turnover. The pleiotropic constraint of regulation of reversible sex-biased genes is widely present in ancient homomorphic sex chromosomes and might be resolved in heteromorphic sex chromosomes through gene duplication followed by subfunctionalization. The evolutionary dynamics of sex chromosomes suggest a mechanism for ‘inheritance’ turnover of sex-determining genes that is mediated by translocation of a sex-determining enhancer. On the basis of these findings, we propose an evolutionary model for the long-term preservation of homomorphic sex chromosomes. Analysis of genomes in a phylogenetic context reveals a 350-million-year-old homomorphic sex chromosome in molluscs, probably maintained by regulation of reversible sex-biased genes and sex chromosome turnover.},
  archive      = {J_NATEE},
  author       = {Han, Wentao and Liu, Liangjie and Wang, Jing and Wei, Huilan and Li, Yuli and Zhang, Lijing and Guo, Zhenyi and Li, Yajuan and Liu, Tian and Zeng, Qifan and Xing, Qiang and Shu, Ya and Wang, Tong and Yang, Yaxin and Zhang, Meiwei and Li, Ruojiao and Yu, Jiachen and Pu, Zhongqi and Lv, Jia and Lian, Shanshan and Hu, Jingjie and Hu, Xiaoli and Bao, Zhenmin and Bao, Lisui and Zhang, Lingling and Wang, Shi},
  doi          = {10.1038/s41559-022-01898-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1891-1906},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ancient homomorphy of molluscan sex chromosomes sustained by reversible sex-biased genes and sex determiner translocation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Coupled abiotic-biotic cycling of nitrous oxide in tropical
peatlands. <em>NATEE</em>, <em>6</em>(12), 1881–1890. (<a
href="https://doi.org/10.1038/s41559-022-01892-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atmospheric nitrous oxide (N2O) is a potent greenhouse gas thought to be mainly derived from microbial metabolism as part of the denitrification pathway. Here we report that in unexplored peat soils of Central and South America, N2O production can be driven by abiotic reactions (≤98%) highly competitive to their enzymatic counterparts. Extracted soil iron positively correlated with in situ abiotic N2O production determined by isotopic tracers. Moreover, we found that microbial N2O reduction accompanied abiotic production, essentially closing a coupled abiotic-biotic N2O cycle. Anaerobic N2O consumption occurred ubiquitously (pH 6.4–3.7), with proportions of diverse clade II N2O reducers increasing with consumption rates. Our findings show that denitrification in tropical peat soils is not a purely biological process but rather a ‘mosaic’ of abiotic and biotic reduction reactions. We predict that hydrological and temperature fluctuations differentially affect abiotic and biotic drivers and further contribute to the high N2O flux variation in the region. There are many open questions about biogeochemical function in peatlands. Here, the authors investigate the nitrogen cycle of tropical peatlands, finding that a surprisingly high fraction of nitrous oxide production is abiotic and that denitrification is a coupled abiotic-biotic process.},
  archive      = {J_NATEE},
  author       = {Buessecker, Steffen and Sarno, Analissa F. and Reynolds, Mark C. and Chavan, Ramani and Park, Jin and Fontánez Ortiz, Marc and Pérez-Castillo, Ana G. and Panduro Pisco, Grober and Urquiza-Muñoz, José David and Reis, Leonardo P. and Ferreira-Ferreira, Jefferson and Furtunato Maia, Jair M. and Holbert, Keith E. and Penton, C. Ryan and Hall, Sharon J. and Gandhi, Hasand and Boëchat, Iola G. and Gücker, Björn and Ostrom, Nathaniel E. and Cadillo-Quiroz, Hinsby},
  doi          = {10.1038/s41559-022-01892-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1881-1890},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Coupled abiotic-biotic cycling of nitrous oxide in tropical peatlands},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Plankton response to global warming is characterized by
non-uniform shifts in assemblage composition since the last ice age.
<em>NATEE</em>, <em>6</em>(12), 1871–1880. (<a
href="https://doi.org/10.1038/s41559-022-01888-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biodiversity is expected to change in response to future global warming. However, it is difficult to predict how species will track the ongoing climate change. Here we use the fossil record of planktonic foraminifera to assess how biodiversity responded to climate change with a magnitude comparable to future anthropogenic warming. We compiled time series of planktonic foraminifera assemblages, covering the time from the last ice age across the deglaciation to the current warm period. Planktonic foraminifera assemblages shifted immediately when temperature began to rise at the end of the last ice age and continued to change until approximately 5,000 years ago, even though global temperature remained relatively stable during the last 11,000 years. The biotic response was largest in the mid latitudes and dominated by range expansion, which resulted in the emergence of new assemblages without analogues in the glacial ocean. Our results indicate that the plankton response to global warming was spatially heterogeneous and did not track temperature change uniformly over the past 24,000 years. Climate change led to the establishment of new assemblages and possibly new ecological interactions, which suggests that current anthropogenic warming may lead to new, different plankton community composition. Analysing a compilation of planktonic foraminifera assemblage time series covering the past 24,000 years, from the last ice age to the current warm period, the authors find that responses to warming were highly heterogeneous leading to the emergence of novel assemblages and possibly new ecological interactions.},
  archive      = {J_NATEE},
  author       = {Strack, Tonke and Jonkers, Lukas and C. Rillo, Marina and Hillebrand, Helmut and Kucera, Michal},
  doi          = {10.1038/s41559-022-01888-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1871-1880},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Plankton response to global warming is characterized by non-uniform shifts in assemblage composition since the last ice age},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Abiotic conditions shape spatial and temporal morphological
variation in north american birds. <em>NATEE</em>, <em>6</em>(12),
1860–1870. (<a
href="https://doi.org/10.1038/s41559-022-01893-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantifying environment–morphology relationships is important not only for understanding the fundamental processes driving phenotypic diversity within and among species but also for predicting how species will respond to ongoing global change. Despite a clear set of expectations motivated by ecological theory, broad evidence in support of generalizable effects of abiotic conditions on spatial and temporal intraspecific morphological variation has been limited. Using standardized data from &amp;gt;250,000 captures of 105 landbird species, we assessed intraspecific shifts in the morphology of adult male birds since 1989 while simultaneously measuring spatial morphological gradients across the North American continent. We found strong spatial and temporal trends in average body size, with warmer temperatures associated with smaller body sizes both at more equatorial latitudes and in more recent years. The magnitude of these thermal effects varied both across and within species, with results suggesting it is the warmest, rather than the coldest, temperatures that drive both spatial and temporal trends. Stronger responses to spatial—rather than temporal—variation in temperature suggest that morphological change may not be keeping up with the pace of climate change. Additionally, as elevation increases, we found that body size declines as relative wing length increases, probably due to the benefits that longer wings confer for flight in thin air environments. Our results provide support for both existing and new large-scale ecomorphological ‘rules’ and highlight how the response of functional trade-offs to abiotic variation drives morphological change. Analysis of a dataset of the morphology of more than 250,000 adult birds of 105 species over a 30 year period across North America reveals changes in body size and relative wing length over time and with relation to latitude, elevation and temperature variation.},
  archive      = {J_NATEE},
  author       = {Youngflesh, Casey and Saracco, James F. and Siegel, Rodney B. and Tingley, Morgan W.},
  doi          = {10.1038/s41559-022-01893-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1860-1870},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Abiotic conditions shape spatial and temporal morphological variation in north american birds},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Citizen science plant observations encode global trait
patterns. <em>NATEE</em>, <em>6</em>(12), 1850–1859. (<a
href="https://doi.org/10.1038/s41559-022-01904-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global maps of plant functional traits are essential for studying the dynamics of the terrestrial biosphere, yet the spatial distribution of trait measurements remains sparse. With the increasing popularity of species identification apps, citizen scientists contribute to growing vegetation data collections. The question emerges whether such opportunistic citizen science data can help map plant functional traits globally. Here we show that we can map global trait patterns by complementing vascular plant observations from the global citizen science project iNaturalist with measurements from the plant trait database TRY. We evaluate these maps using sPlotOpen, a global collection of vegetation plot data. Our results show high correlations between the iNaturalist- and sPlotOpen-based maps of up to 0.69 (r) and higher correlations than to previously published trait maps. As citizen science data collections continue to grow, we can expect them to play a significant role in further improving maps of plant functional traits. Comparing global vegetation trait patterns derived from citizen science data versus those from scientific survey plots, the authors reveal high correlations between the two approaches and improvements over previously published trait maps.},
  archive      = {J_NATEE},
  author       = {Wolf, Sophie and Mahecha, Miguel D. and Sabatini, Francesco Maria and Wirth, Christian and Bruelheide, Helge and Kattge, Jens and Moreno Martínez, Álvaro and Mora, Karin and Kattenborn, Teja},
  doi          = {10.1038/s41559-022-01904-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1850-1859},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Citizen science plant observations encode global trait patterns},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Humid tropical vertebrates are at lower risk of extinction
and population decline in forests with higher structural integrity.
<em>NATEE</em>, <em>6</em>(12), 1840–1849. (<a
href="https://doi.org/10.1038/s41559-022-01915-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reducing deforestation underpins global biodiversity conservation efforts. However, this focus on retaining forest cover overlooks the multitude of anthropogenic pressures that can degrade forest quality and imperil biodiversity. We use remotely sensed indices of tropical rainforest structural condition and associated human pressures to quantify the relative importance of forest cover, structural condition and integrity (the cumulative effect of condition and pressures) on vertebrate species extinction risk and population trends across the global humid tropics. We found that tropical rainforests of high integrity (structurally intact and under low pressures) were associated with lower likelihood of species being threatened and having declining populations, compared with forest cover alone (without consideration of condition and pressures). Further, species were more likely to be threatened or have declining populations if their geographic ranges contained high proportions of degraded forest than if their ranges contained lower proportions of forest cover but of high quality. Our work suggests that biodiversity conservation policies to preserve forest integrity are now urgently required alongside ongoing efforts to halt deforestation in the hyperdiverse humid tropics. Not all forest cover is of equal quality. Here, the authors ask whether forest cover or forest structural complexity influences extinction risk in tropical rainforest vertebrates, finding that forest structural conditions are more important than cover alone in terms of buffering species against extinction and population declines.},
  archive      = {J_NATEE},
  author       = {Pillay, Rajeev and Watson, James E. M. and Hansen, Andrew J. and Jantz, Patrick A. and Aragon-Osejo, Jose and Armenteras, Dolors and Atkinson, Scott C. and Burns, Patrick and Ervin, Jamison and Goetz, Scott J. and González-del-Pliego, Pamela and Robinson, Nathaniel P. and Supples, Christina and Virnig, Anne L. S. and Williams, Brooke A. and Venter, Oscar},
  doi          = {10.1038/s41559-022-01915-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1840-1849},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Humid tropical vertebrates are at lower risk of extinction and population decline in forests with higher structural integrity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Reply to: Existing methods are effective at measuring
natural selection on gene expression. <em>NATEE</em>, <em>6</em>(12),
1838–1839. (<a
href="https://doi.org/10.1038/s41559-022-01916-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Price, Peter D. and Palmer Droguett, Daniela H. and Taylor, Jessica A. and Kim, Dong W. and Place, Elsie S. and Rogers, Thea F. and Mank, Judith E. and Cooney, Christopher R. and Wright, Alison E.},
  doi          = {10.1038/s41559-022-01916-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1838-1839},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Existing methods are effective at measuring natural selection on gene expression},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Existing methods are effective at measuring natural
selection on gene expression. <em>NATEE</em>, <em>6</em>(12), 1836–1837.
(<a href="https://doi.org/10.1038/s41559-022-01889-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Fraser, Hunter B.},
  doi          = {10.1038/s41559-022-01889-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1836-1837},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Existing methods are effective at measuring natural selection on gene expression},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reply to: Phylogenetic affiliation of mitochondria with
alpha-II and rickettsiales is an artefact. <em>NATEE</em>,
<em>6</em>(12), 1832–1835. (<a
href="https://doi.org/10.1038/s41559-022-01896-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Fan, Lu and Wu, Dingfeng and Goremykin, Vadim and Trost, Katharina and Knopp, Michael and Zhang, Chuanlun and Martin, William F. and Zhu, Ruixin},
  doi          = {10.1038/s41559-022-01896-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1832-1835},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Phylogenetic affiliation of mitochondria with alpha-II and rickettsiales is an artefact},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Phylogenetic affiliation of mitochondria with alpha-II and
rickettsiales is an artefact. <em>NATEE</em>, <em>6</em>(12), 1829–1831.
(<a href="https://doi.org/10.1038/s41559-022-01871-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Martijn, Joran and Vosseberg, Julian and Guy, Lionel and Offre, Pierre and Ettema, Thijs J. G.},
  doi          = {10.1038/s41559-022-01871-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1829-1831},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Phylogenetic affiliation of mitochondria with alpha-II and rickettsiales is an artefact},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improving quantitative synthesis to achieve generality in
ecology. <em>NATEE</em>, <em>6</em>(12), 1818–1828. (<a
href="https://doi.org/10.1038/s41559-022-01891-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Synthesis of primary ecological data is often assumed to achieve a notion of ‘generality’, through the quantification of overall effect sizes and consistency among studies, and has become a dominant research approach in ecology. Unfortunately, ecologists rarely define either the generality of their findings, their estimand (the target of estimation) or the population of interest. Given that generality is fundamental to science, and the urgent need for scientific understanding to curb global scale ecological breakdown, loose usage of the term ‘generality’ is problematic. In other disciplines, generality is defined as comprising both generalizability—extending an inference about an estimand from the sample to the population—and transferability—the validity of estimand predictions in a different sampling unit or population. We review current practice in ecological synthesis and demonstrate that, when researchers fail to define the assumptions underpinning generalizations and transfers of effect sizes, generality often misses its target. We provide guidance for communicating nuanced inferences and maximizing the impact of syntheses both within and beyond academia. We propose pathways to generality applicable to ecological syntheses, including the development of quantitative and qualitative criteria with which to license the transfer of estimands from both primary and synthetic studies. Ecological syntheses are often assumed to identify generalities in effects, but this concept is rarely defined. Here, the authors review current practice in ecological synthesis and propose pathways to achieving generality.},
  archive      = {J_NATEE},
  author       = {Spake, Rebecca and O’Dea, Rose E. and Nakagawa, Shinichi and Doncaster, C. Patrick and Ryo, Masahiro and Callaghan, Corey T. and Bullock, James M.},
  doi          = {10.1038/s41559-022-01891-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1818-1828},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Improving quantitative synthesis to achieve generality in ecology},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Safeguarding nutrients from coral reefs under climate
change. <em>NATEE</em>, <em>6</em>(12), 1808–1817. (<a
href="https://doi.org/10.1038/s41559-022-01878-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The sustainability of coral reef fisheries is jeopardized by complex and interacting socio-ecological stressors that undermine their contribution to food and nutrition security. Climate change has emerged as one of the key stressors threatening coral reefs and their fish-associated services. How fish nutrient concentrations respond to warming oceans remains unclear but these responses are probably affected by both direct (metabolism and trophodynamics) and indirect (habitat and species range shifts) effects. Climate-driven coral habitat loss can cause changes in fish abundance and biomass, revealing potential winners and losers among major fisheries targets that can be predicted using ecological indicators and biological traits. A critical next step is to extend research focused on the quantity of available food (fish biomass) to also consider its nutritional quality, which is relevant to progress in the fields of food security and malnutrition. Biological traits are robust predictors of fish nutrient content and thus potentially indicate how climate-driven changes are expected to impact nutrient availability within future food webs on coral reefs. Here, we outline future research priorities and an anticipatory framework towards sustainable reef fisheries contributing to nutrition-sensitive food systems in a warming ocean. This Perspective discusses potential effects of ocean warming on human nutrition provision from coral reef fish, ranging from altered species compositions of fish populations through to changed fish nutrient profiles resulting from altered metabolism, microbiome composition and trophic interactions.},
  archive      = {J_NATEE},
  author       = {Mellin, Camille and Hicks, Christina C. and Fordham, Damien A. and Golden, Christopher D. and Kjellevold, Marian and MacNeil, M. Aaron and Maire, Eva and Mangubhai, Sangeeta and Mouillot, David and Nash, Kirsty L. and Omukoto, Johnstone O. and Robinson, James P. W. and Stuart-Smith, Rick D. and Zamborain-Mason, Jessica and Edgar, Graham J. and Graham, Nicholas A. J.},
  doi          = {10.1038/s41559-022-01878-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1808-1817},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Safeguarding nutrients from coral reefs under climate change},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Plasmid evolution in the clinic. <em>NATEE</em>,
<em>6</em>(12), 1806–1807. (<a
href="https://doi.org/10.1038/s41559-022-01907-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plasmids are well known for transferring antibiotic resistance genes between bacteria. A study in the clinic shows that evolutionary dynamics within the gut microbiomes of hospitalized patients lead to rapid adaptive changes, balancing the level of resistance that plasmids provide against the fitness costs that they impose on bacteria.},
  archive      = {J_NATEE},
  author       = {Wright, Rosanna C. T. and Brockhurst, Michael A.},
  doi          = {10.1038/s41559-022-01907-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1806-1807},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Plasmid evolution in the clinic},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Smaller birds with warmer temperatures. <em>NATEE</em>,
<em>6</em>(12), 1804–1805. (<a
href="https://doi.org/10.1038/s41559-022-01897-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Measurements of individual birds of 105 species across North America over almost twenty years reveal intraspecific trends of smaller body sizes towards the equator and of decreasing body size as average temperatures increase.},
  archive      = {J_NATEE},
  author       = {Hughes, Emma C. and Slavenko, Alex},
  doi          = {10.1038/s41559-022-01897-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1804-1805},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Smaller birds with warmer temperatures},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Statistical power from the people. <em>NATEE</em>,
<em>6</em>(12), 1802–1803. (<a
href="https://doi.org/10.1038/s41559-022-01902-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A global comparison of plant trait patterns calculated using citizen science observations versus those calculated using traditional scientific data reveals remarkable congruence between the two approaches.},
  archive      = {J_NATEE},
  author       = {Moles, Angela T. and Xirocostas, Zoe A.},
  doi          = {10.1038/s41559-022-01902-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1802-1803},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Statistical power from the people},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The expanding value of long-term studies of individuals in
the wild. <em>NATEE</em>, <em>6</em>(12), 1799–1801. (<a
href="https://doi.org/10.1038/s41559-022-01940-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past seventy-five years, long-term population studies of individual organisms in their natural environments have been influential in illuminating how ecological and evolutionary processes operate, and the extent of variation and temporal change in these processes. As these studies have matured, the incorporation of new technologies has generated an ever-broadening perspective, from molecular and genomic to landscape-level analyses facilitated by remote-sensing.},
  archive      = {J_NATEE},
  author       = {Sheldon, Ben C. and Kruuk, Loeske E. B. and Alberts, Susan C.},
  doi          = {10.1038/s41559-022-01940-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1799-1801},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The expanding value of long-term studies of individuals in the wild},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Renegotiating identities in international academic careers.
<em>NATEE</em>, <em>6</em>(12), 1796–1798. (<a
href="https://doi.org/10.1038/s41559-022-01895-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many academics move countries in pursuit of career opportunities. With every move, personal identities are renegotiated as people shift between belonging to majority and minority groups in different contexts. Institutes should consider people’s dynamic and intersectional identities in their diversity, equity and inclusion practices.},
  archive      = {J_NATEE},
  author       = {Echeverri, Alejandra and Guzman, Laura Melissa and Heredia, Sylvia and Ocampo-Peñuela, Natalia and Umaña, Maria Natalia},
  doi          = {10.1038/s41559-022-01895-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1796-1798},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Renegotiating identities in international academic careers},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Introducing african cheetahs to india is an ill-advised
conservation attempt. <em>NATEE</em>, <em>6</em>(12), 1794–1795. (<a
href="https://doi.org/10.1038/s41559-022-01922-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Gopalaswamy, Arjun M. and Khalatbari, Leili and Chellam, Ravi and Mills, Michael G. L. and Vanak, Abi Tamim and Thuo, David and Karanth, K. Ullas and Broekhuis, Femke},
  doi          = {10.1038/s41559-022-01922-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1794-1795},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Introducing african cheetahs to india is an ill-advised conservation attempt},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Record-breaking fires in the brazilian amazon associated
with uncontrolled deforestation. <em>NATEE</em>, <em>6</em>(12),
1792–1793. (<a
href="https://doi.org/10.1038/s41559-022-01945-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Mataveli, Guilherme and de Oliveira, Gabriel and Silva-Junior, Celso H. L. and Stark, Scott C. and Carvalho, Nathália and Anderson, Liana O. and Gatti, Luciana V. and Aragão, Luiz E. O. C.},
  doi          = {10.1038/s41559-022-01945-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1792-1793},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Record-breaking fires in the brazilian amazon associated with uncontrolled deforestation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Time at last for biodiversity. <em>NATEE</em>,
<em>6</em>(12), 1791. (<a
href="https://doi.org/10.1038/s41559-022-01954-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The delayed UN Biodiversity COP15 follows closely on the heels of the Climate COP27. We look at what comparisons can valuably be made between the two summits.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01954-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1791},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Time at last for biodiversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Ecosystem productivity affected the
spatiotemporal disappearance of neanderthals in iberia. <em>NATEE</em>,
<em>6</em>(11), 1789. (<a
href="https://doi.org/10.1038/s41559-022-01917-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Vidal-Cordasco, M. and Ocio, D. and Hickler, T. and Marín-Arroyo, A. B.},
  doi          = {10.1038/s41559-022-01917-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1789},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Ecosystem productivity affected the spatiotemporal disappearance of neanderthals in iberia},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Widespread extinction debts and
colonization credits in united states breeding bird communities.
<em>NATEE</em>, <em>6</em>(11), 1788. (<a
href="https://doi.org/10.1038/s41559-022-01912-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Haddou, Yacob and Mancy, Rebecca and Matthiopoulos, Jason and Spatharis, Sofie and Dominoni, Davide M.},
  doi          = {10.1038/s41559-022-01912-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1788},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Widespread extinction debts and colonization credits in united states breeding bird communities},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Species traits and reduced habitat
suitability limit efficacy of climate change refugia in streams.
<em>NATEE</em>, <em>6</em>(11), 1777–1787. (<a
href="https://doi.org/10.1038/s41559-022-01874-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Troia, Matthew J. and Kaz, Anna L. and Niemeyer, J. Cameron and Giam, Xingli},
  doi          = {10.1038/s41559-022-01874-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1777-1787},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Species traits and reduced habitat suitability limit efficacy of climate change refugia in streams},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Patterns and consequences of age-linked change in local
relatedness in animal societies. <em>NATEE</em>, <em>6</em>(11),
1766–1776. (<a
href="https://doi.org/10.1038/s41559-022-01872-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ultimate payoff of behaviours depends not only on their direct impact on an individual, but also on the impact on their relatives. Local relatedness—the average relatedness of an individual to their social environment—therefore has profound effects on social and life history evolution. Recent work has begun to show that local relatedness has the potential to change systematically over an individual’s lifetime, a process called kinship dynamics. However, it is unclear how general these kinship dynamics are, whether they are predictable in real systems and their effects on behaviour and life history evolution. In this study, we combine modelling with data from real systems to explore the extent and impact of kinship dynamics. We use data from seven group-living mammals with diverse social and mating systems to demonstrate not only that kinship dynamics occur in animal systems, but also that the direction and magnitude of kinship dynamics can be accurately predicted using a simple model. We use a theoretical model to demonstrate that kinship dynamics can profoundly affect lifetime patterns of behaviour and can drive sex differences in helping and harming behaviour across the lifespan in social species. Taken together, this work demonstrates that kinship dynamics are likely to be a fundamental dimension of social evolution, especially when considering age-linked changes and sex differences in behaviour and life history. A model of how local relatedness changes with age fits empirical patterns from seven group-living mammal species and reveals that patterns differ between the sexes and the potential behavioural consequences of these changes.},
  archive      = {J_NATEE},
  author       = {Ellis, Samuel and Johnstone, Rufus A. and Cant, Michael A. and Franks, Daniel W. and Weiss, Michael N. and Alberts, Susan C. and Balcomb, Kenneth C. and Benton, Claire H. and Brent, Lauren J. N. and Crockford, Catherine and Davidian, Eve and Delahay, Richard J. and Ellifrit, David K. and Höner, Oliver P. and Meniri, Magali and McDonald, Robbie A. and Nichols, Hazel J. and Thompson, Faye J. and Vigilant, Linda and Wittig, Roman M. and Croft, Darren P.},
  doi          = {10.1038/s41559-022-01872-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1766-1776},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Patterns and consequences of age-linked change in local relatedness in animal societies},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Canalized gene expression during development mediates caste
differentiation in ants. <em>NATEE</em>, <em>6</em>(11), 1753–1765. (<a
href="https://doi.org/10.1038/s41559-022-01884-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ant colonies are higher-level organisms consisting of specialized reproductive and non-reproductive individuals that differentiate early in development, similar to germ–soma segregation in bilateral Metazoa. Analogous to diverging cell lines, developmental differentiation of individual ants has often been considered in epigenetic terms but the sets of genes that determine caste phenotypes throughout larval and pupal development remain unknown. Here, we reconstruct the individual developmental trajectories of two ant species, Monomorium pharaonis and Acromyrmex echinatior, after obtaining &amp;gt;1,400 whole-genome transcriptomes. Using a new backward prediction algorithm, we show that caste phenotypes can be accurately predicted by genome-wide transcriptome profiling. We find that caste differentiation is increasingly canalized from early development onwards, particularly in germline individuals (gynes/queens) and that the juvenile hormone signalling pathway plays a key role in this process by regulating body mass divergence between castes. We quantified gene-specific canalization levels and found that canalized genes with gyne/queen-biased expression were enriched for ovary and wing functions while canalized genes with worker-biased expression were enriched in brain and behavioural functions. Suppression in gyne larvae of Freja, a highly canalized gyne-biased ovary gene, disturbed pupal development by inducing non-adaptive intermediate phenotypes between gynes and workers. Our results are consistent with natural selection actively maintaining canalized caste phenotypes while securing robustness in the life cycle ontogeny of ant colonies. Using individual transcriptomes of two ant species, the authors show that caste differentiation is canalized from early development and identify key regulatory genes for the development of ant caste phenotypes.},
  archive      = {J_NATEE},
  author       = {Qiu, Bitao and Dai, Xueqin and Li, Panyi and Larsen, Rasmus Stenbak and Li, Ruyan and Price, Alivia Lee and Ding, Guo and Texada, Michael James and Zhang, Xiafang and Zuo, Dashuang and Gao, Qionghua and Jiang, Wei and Wen, Tinggang and Pontieri, Luigi and Guo, Chunxue and Rewitz, Kim and Li, Qiye and Liu, Weiwei and Boomsma, Jacobus J. and Zhang, Guojie},
  doi          = {10.1038/s41559-022-01884-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1753-1765},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Canalized gene expression during development mediates caste differentiation in ants},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The structure of genotype-phenotype maps makes fitness
landscapes navigable. <em>NATEE</em>, <em>6</em>(11), 1742–1752. (<a
href="https://doi.org/10.1038/s41559-022-01867-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fitness landscapes are often described in terms of ‘peaks’ and ‘valleys’, indicating an intuitive low-dimensional landscape of the kind encountered in everyday experience. The space of genotypes, however, is extremely high dimensional, which results in counter-intuitive structural properties of genotype-phenotype maps. Here we show that these properties, such as the presence of pervasive neutral networks, make fitness landscapes navigable. For three biologically realistic genotype-phenotype map models—RNA secondary structure, protein tertiary structure and protein complexes—we find that, even under random fitness assignment, fitness maxima can be reached from almost any other phenotype without passing through fitness valleys. This in turn indicates that true fitness valleys are very rare. By considering evolutionary simulations between pairs of real examples of functional RNA sequences, we show that accessible paths are also likely to be used under evolutionary dynamics. Our findings have broad implications for the prediction of natural evolutionary outcomes and for directed evolution. In realistic high-dimensional fitness landscapes of RNA secondary structure, protein tertiary structure and protein complexes, fitness peaks can be easily reached from any starting point without traversing fitness valleys.},
  archive      = {J_NATEE},
  author       = {Greenbury, Sam F. and Louis, Ard A. and Ahnert, Sebastian E.},
  doi          = {10.1038/s41559-022-01867-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1742-1752},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The structure of genotype-phenotype maps makes fitness landscapes navigable},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Industrial energy development decouples ungulate migration
from the green wave. <em>NATEE</em>, <em>6</em>(11), 1733–1741. (<a
href="https://doi.org/10.1038/s41559-022-01887-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to freely move across the landscape to track the emergence of nutritious spring green-up (termed ‘green-wave surfing’) is key to the foraging strategy of migratory ungulates. Across the vast landscapes traversed by many migratory herds, habitats are being altered by development with unknown consequences for surfing. Using a unique long-term tracking dataset, we found that when energy development occurs within mule deer (Odocoileus hemionus) migration corridors, migrating animals become decoupled from the green wave. During the early phases of a coalbed natural gas development, deer synchronized their movements with peak green-up. But faced with increasing disturbance as development expanded, deer altered their movements by holding up at the edge of the gas field and letting the green wave pass them by. Development often modified only a small portion of the migration corridor but had far-reaching effects on behaviour before and after migrating deer encountered it, thus reducing surfing along the entire route by 38.65% over the 14-year study period. Our study suggests that industrial development within migratory corridors can change the behaviour of migrating ungulates and diminish the benefits of migration. Such disruptions to migratory behaviour present a common mechanism whereby corridors become unprofitable and could ultimately be lost on highly developed landscapes. Analysing a long-term tracking dataset of migrating mule deer, the authors show that the expansion of natural gas energy infrastructure over 14 years along a migratory corridor changes deer behaviour and reduces by more than 38% their ability to keep pace with spring vegetation green-up.},
  archive      = {J_NATEE},
  author       = {Aikens, Ellen O. and Wyckoff, Teal B. and Sawyer, Hall and Kauffman, Matthew J.},
  doi          = {10.1038/s41559-022-01887-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1733-1741},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Industrial energy development decouples ungulate migration from the green wave},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Naturalized alien floras still carry the legacy of european
colonialism. <em>NATEE</em>, <em>6</em>(11), 1723–1732. (<a
href="https://doi.org/10.1038/s41559-022-01865-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The redistribution of alien species across the globe accelerated with the start of European colonialism. European powers were responsible for the deliberate and accidental transportation, introduction and establishment of alien species throughout their occupied territories and the metropolitan state. Here, we show that these activities left a lasting imprint on the global distribution of alien plants. Specifically, we investigated how four European empires (British, Spanish, Portuguese and Dutch) structured current alien floras worldwide. We found that compositional similarity is higher than expected among regions that once were occupied by the same empire. Further, we provide strong evidence that floristic similarity between regions occupied by the same empire increases with the time a region was occupied. Network analysis suggests that historically more economically or strategically important regions have more similar alien floras across regions occupied by an empire. Overall, we find that European colonial history is still detectable in alien floras worldwide. Compiling data on floral introductions and European colonial history of regions worldwide, the authors find that compositional similarity of floras is higher than expected among regions once occupied by the same empire and similarity increases with the length of time the region was occupied by that empire.},
  archive      = {J_NATEE},
  author       = {Lenzner, Bernd and Latombe, Guillaume and Schertler, Anna and Seebens, Hanno and Yang, Qiang and Winter, Marten and Weigelt, Patrick and van Kleunen, Mark and Pyšek, Petr and Pergl, Jan and Kreft, Holger and Dawson, Wayne and Dullinger, Stefan and Essl, Franz},
  doi          = {10.1038/s41559-022-01865-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1723-1732},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Naturalized alien floras still carry the legacy of european colonialism},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The likely extinction of hundreds of palm species threatens
their contributions to people and ecosystems. <em>NATEE</em>,
<em>6</em>(11), 1710–1722. (<a
href="https://doi.org/10.1038/s41559-022-01858-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protecting nature’s contributions to people requires accelerating extinction risk assessment and better integrating evolutionary, functional and used diversity with conservation planning. Here, we report machine learning extinction risk predictions for 1,381 palm species (Arecaceae), a plant family of high socio-economic and ecological importance. We integrate these predictions with published assessments for 508 species (covering 75% of all palm species) and we identify top-priority regions for palm conservation on the basis of their proportion of threatened evolutionarily distinct, functionally distinct and used species. Finally, we explore palm use resilience to identify non-threatened species that could potentially serve as substitutes for threatened used species by providing similar products. We estimate that over a thousand palms (56%) are probably threatened, including 185 species with documented uses. Some regions (New Guinea, Vanuatu and Vietnam) emerge as top ten priorities for conservation only after incorporating machine learning extinction risk predictions. Potential substitutes are identified for 91% of the threatened used species and regional use resilience increases with total palm richness. However, 16 threatened used species lack potential substitutes and 30 regions lack substitutes for at least one of their threatened used palm species. Overall, we show that hundreds of species of this keystone family face extinction, some of them probably irreplaceable, at least locally. This highlights the need for urgent actions to avoid major repercussions on palm-associated ecosystem processes and human livelihoods in the coming decades. Machine learning is used to predict extinction risk for 1,381 palm species, allowing identification of priority regions for palm conservation and cases where substitution with non-threatened species could provide products for human use.},
  archive      = {J_NATEE},
  author       = {Bellot, S. and Lu, Y. and Antonelli, A. and Baker, W. J. and Dransfield, J. and Forest, F. and Kissling, W. D. and Leitch, I. J. and Nic Lughadha, E. and Ondo, I. and Pironon, S. and Walker, B. E. and Cámara-Leret, R. and Bachman, S. P.},
  doi          = {10.1038/s41559-022-01858-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1710-1722},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The likely extinction of hundreds of palm species threatens their contributions to people and ecosystems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Fish community structure and dynamics are insufficient to
mediate coral resilience. <em>NATEE</em>, <em>6</em>(11), 1700–1709. (<a
href="https://doi.org/10.1038/s41559-022-01882-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coral reefs are being impacted by myriad stressors leading to drastic changes to their structure and function. Fishes play essential roles in driving ecosystem processes on coral reefs but the extent to which these processes are emergent at temporal or ecosystem scales or otherwise masked by other drivers (for example, climatic events and crown-of-thorns starfish outbreaks) is poorly understood. Using time series data on fish community composition and coral and macroalgae percentage cover between 2006 and 2017 from 57 sites around Mo’orea, Polynesia, we found that fish community diversity predicts temporal stability in fish biomass but did not translate to temporal stability of coral cover. Furthermore, we found limited evidence of directional influence of fish on coral dynamics at temporal and ecosystem scales and no evidence that fish mediate coral recovery rate from disturbance. Our findings suggest that coral reef fisheries management will benefit from maintaining fish diversity but that this level of management is unlikely to strongly mediate coral loss or recovery over time. A comparison of fish community data with reef coral and macroalgae cover at several sites around Polynesia over 11 years and spanning disturbance events suggests that fish community diversity has only minimal influence on coral dynamics, including recovery from disturbance.},
  archive      = {J_NATEE},
  author       = {Cline, Timothy J. and Allgeier, Jacob E.},
  doi          = {10.1038/s41559-022-01882-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1700-1709},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fish community structure and dynamics are insufficient to mediate coral resilience},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Isotopic evidence that aestivation allows malaria mosquitoes
to persist through the dry season in the sahel. <em>NATEE</em>,
<em>6</em>(11), 1687–1699. (<a
href="https://doi.org/10.1038/s41559-022-01886-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data suggest that the malaria vector mosquito Anopheles coluzzii persists during the dry season in the Sahel through a dormancy mechanism known as aestivation; however, the contribution of aestivation compared with alternative strategies such as migration is unknown. Here we marked larval Anopheles mosquitoes in two Sahelian villages in Mali using deuterium (2H) to assess the contribution of aestivation to persistence of mosquitoes through the seven-month dry season. After an initial enrichment period, 33% of An. coluzzii mosquitoes were strongly marked. Seven months following enrichment, multiple analysis methods supported the ongoing presence of marked mosquitoes, compatible with the prediction that the fraction of marked mosquitoes should remain stable throughout the dry season if local aestivation is occurring. The results suggest that aestivation is a major persistence mechanism of An. coluzzii in the Sahel, contributing at least 20% of the adults at the onset of rains. This persistence strategy could influence mosquito control and malaria elimination campaigns. In the Sahel region of West Africa, An. coluzzii mosquitoes appear to survive the dry season locally, but the relative contribution of this subpopulation to the persistence of the species in the Sahel has remained unknown. Here the authors use stable isotope tracking to determine the fraction of mosquitoes that undergo aestivation, a state of dormancy that allows them to persist through the dry season and maintain yearly malaria transmission.},
  archive      = {J_NATEE},
  author       = {Faiman, Roy and Yaro, Alpha S. and Dao, Adama and Sanogo, Zana L. and Diallo, Moussa and Samake, Djibril and Yossi, Ousmane and Veru, Laura M. and Graber, Leland C. and Conte, Abigail R. and Kouam, Cedric and Krajacich, Benjamin J. and Lehmann, Tovi},
  doi          = {10.1038/s41559-022-01886-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1687-1699},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Isotopic evidence that aestivation allows malaria mosquitoes to persist through the dry season in the sahel},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). No evidence for long-range male sex pheromones in two
malaria mosquitoes. <em>NATEE</em>, <em>6</em>(11), 1676–1686. (<a
href="https://doi.org/10.1038/s41559-022-01869-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cues involved in mate seeking and recognition prevent hybridization and can be involved in speciation processes. In malaria mosquitoes, females of the two sibling species Anopheles gambiae s.s. and An. coluzzii mate in monospecific male swarms and hybrids are rare. Long-range sex pheromones driving this behaviour have been debated in literature but so far, no study has proven their existence or their absence. Here, we attempted to bring to light their existence. To put all the odds in our favour, we used different chemical ecology methods such as behavioural and electrophysiological assays as well chemical analyses, and we worked with mosquitoes at their optimal physiological mating state that is with swarming males during their natural swarming windows. Despite all our efforts, our results support the absence of long-range sex pheromones involved in swarm detection and recognition by females. We briefly discuss the implications of this finding in ecology, evolution and for control strategies. A series of behavioural, electrophysiological and chemical assays are used to attempt to detect long-range sex pheromones involved in species-specific male swarm recognition by Anopheles female mosquitoes, but no evidence is found.},
  archive      = {J_NATEE},
  author       = {Poda, Serge Bèwadéyir and Buatois, Bruno and Lapeyre, Benoit and Dormont, Laurent and Diabaté, Abdoulaye and Gnankiné, Olivier and Dabiré, Roch K. and Roux, Olivier},
  doi          = {10.1038/s41559-022-01869-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1676-1686},
  shortjournal = {Nat. Ecol. Evol},
  title        = {No evidence for long-range male sex pheromones in two malaria mosquitoes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Consistent stabilizing effects of plant diversity across
spatial scales and climatic gradients. <em>NATEE</em>, <em>6</em>(11),
1669–1675. (<a
href="https://doi.org/10.1038/s41559-022-01868-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biodiversity has widely been documented to enhance local community stability but whether such stabilizing effects of biodiversity extend to broader scales remains elusive. Here, we investigated the relationships between biodiversity and community stability in natural plant communities from quadrat (1 m2) to plot (400 m2) and regional (5−214 km2) scales and across broad climatic conditions, using an extensive plant community dataset from the National Ecological Observatory Network. We found that plant diversity provided consistent stabilizing effects on total community abundance across three nested spatial scales and climatic gradients. The strength of the stabilizing effects of biodiversity increased modestly with spatial scale and decreased as precipitation seasonality increased. Our findings illustrate the generality of diversity–stability theory across scales and climatic gradients, which provides a robust framework for understanding ecosystem responses to biodiversity and climate changes. Analysing &amp;gt;6,000 plant species from plots across the US National Ecological Observatory Network (NEON), the authors show that plant diversity consistently stabilizes community abundance across spatial scales and broad ecoclimatic domains, with the strength of the stabilizing effect increasing with scale.},
  archive      = {J_NATEE},
  author       = {Liang, Maowei and Baiser, Benjamin and Hallett, Lauren M. and Hautier, Yann and Jiang, Lin and Loreau, Michel and Record, Sydne and Sokol, Eric R. and Zarnetske, Phoebe L. and Wang, Shaopeng},
  doi          = {10.1038/s41559-022-01868-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1669-1675},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Consistent stabilizing effects of plant diversity across spatial scales and climatic gradients},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Dual ancestries and ecologies of the late glacial
palaeolithic in britain. <em>NATEE</em>, <em>6</em>(11), 1658–1668. (<a
href="https://doi.org/10.1038/s41559-022-01883-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic investigations of Upper Palaeolithic Europe have revealed a complex and transformative history of human population movements and ancestries, with evidence of several instances of genetic change across the European continent in the period following the Last Glacial Maximum (LGM). Concurrent with these genetic shifts, the post-LGM period is characterized by a series of significant climatic changes, population expansions and cultural diversification. Britain lies at the extreme northwest corner of post-LGM expansion and its earliest Late Glacial human occupation remains unclear. Here we present genetic data from Palaeolithic human individuals in the United Kingdom and the oldest human DNA thus far obtained from Britain or Ireland. We determine that a Late Upper Palaeolithic individual from Gough&#39;s Cave probably traced all its ancestry to Magdalenian-associated individuals closely related to those from sites such as El Mirón Cave, Spain, and Troisième Caverne in Goyet, Belgium. However, an individual from Kendrick&#39;s Cave shows no evidence of having ancestry related to the Gough’s Cave individual. Instead, the Kendrick’s Cave individual traces its ancestry to groups who expanded across Europe during the Late Glacial and are represented at sites such as Villabruna, Italy. Furthermore, the individuals differ not only in their genetic ancestry profiles but also in their mortuary practices and their diets and ecologies, as evidenced through stable isotope analyses. This finding mirrors patterns of dual genetic ancestry and admixture previously detected in Iberia but may suggest a more drastic genetic turnover in northwestern Europe than in the southwest. The authors report genetic, archaeological and stable isotopic data from two late Palaeolithic individuals in Britain, from Gough&#39;s Cave and Kendrick&#39;s Cave. The individuals differ not only in their ancestry but also their diets, ecologies and mortuary practices, revealing diverse origins and lifeways among inhabitants of late Pleistocene Britain.},
  archive      = {J_NATEE},
  author       = {Charlton, Sophy and Brace, Selina and Hajdinjak, Mateja and Kearney, Rebecca and Booth, Thomas and Reade, Hazel and Tripp, Jennifer A. and Sayle, Kerry L. and Grimm, Sonja B. and Bello, Silvia M. and Walker, Elizabeth A. and Gilardet, Alexandre and East, Philip and Glocke, Isabelle and Larson, Greger and Higham, Tom and Stringer, Chris and Skoglund, Pontus and Barnes, Ian and Stevens, Rhiannon E.},
  doi          = {10.1038/s41559-022-01883-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1658-1668},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dual ancestries and ecologies of the late glacial palaeolithic in britain},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Ecosystem productivity affected the spatiotemporal
disappearance of neanderthals in iberia. <em>NATEE</em>, <em>6</em>(11),
1644–1657. (<a
href="https://doi.org/10.1038/s41559-022-01861-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What role did fluctuations play in biomass availability for secondary consumers in the disappearance of Neanderthals and the survival of modern humans? To answer this, we quantify the effects of stadial and interstadial conditions on ecosystem productivity and human spatiotemporal distribution patterns during the Middle to Upper Palaeolithic transition (50,000–30,000 calibrated years before the present) in Iberia. First, we used summed probability distribution, optimal linear estimation and Bayesian age modelling to reconstruct an updated timescale for the transition. Next, we executed a generalized dynamic vegetation model to estimate the net primary productivity. Finally, we developed a macroecological model validated with present-day observations to calculate herbivore abundance. The results indicate that, in the Eurosiberian region, the disappearance of Neanderthal groups was contemporaneous with a significant decrease in the available biomass for secondary consumers, and the arrival of the first Homo sapiens populations coincided with an increase in herbivore carrying capacity. During stadials, the Mediterranean region had the most stable conditions and the highest biomass of medium and medium–large herbivores. These outcomes support an ecological cause for the hiatus between the Mousterian and Aurignacian technocomplexes in Northern Iberia and the longer persistence of Neanderthals in southern latitudes. Quantifying changing climatic effects on ecosystem productivity and human spatiotemporal distributions during the Middle to Upper Palaeolithic transition in Iberia, the authors find evidence that the hiatus between Neanderthal and modern human cultural complexes in North Atlantic Iberia and the longer persistence of Neanderthals in southern latitudes had an ecological cause.},
  archive      = {J_NATEE},
  author       = {Vidal-Cordasco, M. and Ocio, D. and Hickler, T. and Marín-Arroyo, A. B.},
  doi          = {10.1038/s41559-022-01861-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1644-1657},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecosystem productivity affected the spatiotemporal disappearance of neanderthals in iberia},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Divergent evolutionary trajectories of bryophytes and
tracheophytes from a complex common ancestor of land plants.
<em>NATEE</em>, <em>6</em>(11), 1634–1643. (<a
href="https://doi.org/10.1038/s41559-022-01885-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The origin of plants and their colonization of land fundamentally transformed the terrestrial environment. Here we elucidate the basis of this formative episode in Earth history through patterns of lineage, gene and genome evolution. We use new fossil calibrations, a relative clade age calibration (informed by horizontal gene transfer) and new phylogenomic methods for mapping gene family origins. Distinct rooting strategies resolve tracheophytes (vascular plants) and bryophytes (non-vascular plants) as monophyletic sister groups that diverged during the Cambrian, 515–494 million years ago. The embryophyte stem is characterized by a burst of gene innovation, while bryophytes subsequently experienced an equally dramatic episode of reductive genome evolution in which they lost genes associated with the elaboration of vasculature and the stomatal complex. Overall, our analyses reveal that extant tracheophytes and bryophytes are both highly derived from a more complex ancestral land plant. Understanding the origin of land plants requires tracing character evolution across a diversity of modern lineages. Fossil calibrations, a relative clade age calibration (informed by horizontal gene transfer) and new phylogenomic methods for mapping gene family origins resolve tracheophytes (vascular plants) and bryophytes (non-vascular plants) as monophyletic sister groups that diverged during the Cambrian, 515–494 million years ago. The early evolution of both groups, but particularly that of bryophytes, was characterized by major gene content change.},
  archive      = {J_NATEE},
  author       = {Harris, Brogan J. and Clark, James W. and Schrempf, Dominik and Szöllősi, Gergely J. and Donoghue, Philip C. J. and Hetherington, Alistair M. and Williams, Tom A.},
  doi          = {10.1038/s41559-022-01885-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1634-1643},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Divergent evolutionary trajectories of bryophytes and tracheophytes from a complex common ancestor of land plants},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). No evidence that mandatory open data policies increase error
correction. <em>NATEE</em>, <em>6</em>(11), 1630–1633. (<a
href="https://doi.org/10.1038/s41559-022-01879-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using a database of open data policies for 199 journals in ecology and evolution, we found no detectable link between data sharing requirements and article retractions or corrections. Despite the potential for open data to facilitate error detection, poorly archived datasets, the absence of open code and the stigma associated with correcting or retracting articles probably stymie error correction. Requiring code alongside data and destigmatizing error correction among authors and journal editors could increase the effectiveness of open data policies at helping science self-correct. An analysis of 199 journals in ecology and evolution finds no link between policies mandating data sharing and the rate of article retraction or correction. The authors position this finding in a broader discussion of open data and code.},
  archive      = {J_NATEE},
  author       = {Berberi, Ilias and Roche, Dominique G.},
  doi          = {10.1038/s41559-022-01879-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1630-1633},
  shortjournal = {Nat. Ecol. Evol},
  title        = {No evidence that mandatory open data policies increase error correction},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reply to “reduction in grain pollen indicates population
decline, but not necessarily black death mortality.” <em>NATEE</em>,
<em>6</em>(11), 1628–1629. (<a
href="https://doi.org/10.1038/s41559-022-01863-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Izdebski, A. and Bauch, M. and Guzowski, P. and Mensing, S. and Mercuri, A. M. and Newfield, T. and Piovesan, G. and Sadori, L. and Vignola, C. and Masi, A.},
  doi          = {10.1038/s41559-022-01863-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1628-1629},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to ‘Reduction in grain pollen indicates population decline, but not necessarily black death mortality’},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reduction in grain pollen indicates population decline, but
not necessarily black death mortality. <em>NATEE</em>, <em>6</em>(11),
1626–1627. (<a
href="https://doi.org/10.1038/s41559-022-01862-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Curtis, Daniel R.},
  doi          = {10.1038/s41559-022-01862-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1626-1627},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reduction in grain pollen indicates population decline, but not necessarily black death mortality},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The influence of social cues on timing of animal migrations.
<em>NATEE</em>, <em>6</em>(11), 1617–1625. (<a
href="https://doi.org/10.1038/s41559-022-01866-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal migration plays a central role in many ecological and evolutionary processes, yet migratory populations worldwide are increasingly threatened. Adjusting migration timing to match ecosystem phenology is key to survival in dynamic and changing ecosystems, especially in an era of human-induced rapid environmental change. Social cues are increasingly recognized as major components of migratory behaviour, yet a comprehensive understanding of how social cues influence the timing of animal migrations remains elusive. Here, we introduce a framework for assessing the role that social cues, ranging from explicit (for example, active cueing) to implicit (for example, competition), play in animals’ temporal migration decisions across a range of scales. By applying this theoretical lens to a systematic review of published literature, we show that a broad range of social cues frequently mediate migration timing at a range of temporal scales and across highly diverse migratory taxa. We further highlight that while rarely documented, several social cue mechanisms (for example, social learning and density dependency) play important adaptive roles in matching migration timing with ecosystem dynamics. Thus, social cues play a fundamental role in migration timing, with potentially widespread ecological consequences and implications for the conservation of migratory species. Furthermore, our analysis establishes a theoretical basis on which to evaluate future findings on the role of both conspecific and interspecific social cues in this intersection of behavioural ecology and global change biology. Social cues are known to be involved in the timing of animal migrations. In this Review, the authors outline a framework for understanding the roles of different cues across temporal scales, and how these match with ecosystem dynamics.},
  archive      = {J_NATEE},
  author       = {Oestreich, William K. and Aiu, Kanoe M. and Crowder, Larry B. and McKenna, Megan F. and Berdahl, Andrew M. and Abrahms, Briana},
  doi          = {10.1038/s41559-022-01866-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1617-1625},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The influence of social cues on timing of animal migrations},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bridging landscape ecology and urban science to respond to
the rising threat of mosquito-borne diseases. <em>NATEE</em>,
<em>6</em>(11), 1601–1616. (<a
href="https://doi.org/10.1038/s41559-022-01876-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prevalence of diseases borne by mosquitoes, particularly in the genus Aedes, is rising worldwide. This has been attributed, in part, to the dramatic rates of contemporary urbanization. While Aedes-borne disease risk varies within and between cities, few investigations use urban science-based approaches to examine how city structure and function contribute to vector or pathogen introduction and maintenance. Here, we integrate theories from complex adaptive systems, landscape ecology and urban geography to develop an urban systems framework for understanding Aedes-borne diseases. The framework establishes that cities comprise hierarchically structured patches of different land uses and characteristics. Properties of the patches (that is, composition) determine localized disease risk, while configuration and connectivity drive emergent patterns of pathogen spread. Complexity is added by incorporating individual and collective human social structures, considering how feedbacks among social actors and with the landscape drive risk and transmission. We discuss how these concepts apply to case studies of Aedes-borne disease from around the world. Ultimately, the framework strengthens existing theoretical and mixed qualitative–quantitative approaches, and advances considerations of how interventions including urban planning (for example, piped water provisioning) and emerging vector control strategies (for example, Wolbachia-infected mosquitoes) can be implemented to prevent and control the rising threat of Aedes-borne diseases. In this Perspective, the authors demonstrate how concepts and models from landscape ecology and complex adaptive systems science can be used to explore the dynamics of mosquito-borne diseases in urban environments.},
  archive      = {J_NATEE},
  author       = {Kache, Pallavi A. and Santos-Vega, Mauricio and Stewart-Ibarra, Anna M. and Cook, Elizabeth M. and Seto, Karen C. and Diuk-Wasser, Maria A.},
  doi          = {10.1038/s41559-022-01876-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1601-1616},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Bridging landscape ecology and urban science to respond to the rising threat of mosquito-borne diseases},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Life finds a way. <em>NATEE</em>, <em>6</em>(11), 1599–1600.
(<a href="https://doi.org/10.1038/s41559-022-01877-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fitness landscapes were described almost a century ago as smooth surfaces with peaks and valleys that are difficult to navigate. Now, more realistic high-dimensional genotype–phenotype maps show that fitness maxima can be reached from almost any other phenotype while avoiding fitness valleys, which are very rare.},
  archive      = {J_NATEE},
  author       = {Aguirre, Jacobo},
  doi          = {10.1038/s41559-022-01877-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1599-1600},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Life finds a way},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Colonialism shaped today’s biodiversity. <em>NATEE</em>,
<em>6</em>(11), 1597–1598. (<a
href="https://doi.org/10.1038/s41559-022-01903-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effects of the redistribution of flora and fauna by European empires are still visible in global biodiversity today and can be traced through the distribution of introduced species. Attempts to solve today’s biodiversity crisis necessitates grappling these colonial legacies head on.},
  archive      = {J_NATEE},
  author       = {Raja, Nussaïbah B.},
  doi          = {10.1038/s41559-022-01903-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1597-1598},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Colonialism shaped today’s biodiversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ecoinformatics for conservation biology. <em>NATEE</em>,
<em>6</em>(11), 1595–1596. (<a
href="https://doi.org/10.1038/s41559-022-01870-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Harnessing big data and machine learning provides an assessment of the extinction risks of palm species worldwide, and illustrates an integrative conservation planning approach that incorporates evolutionary and ecological distinctiveness as well as human use.},
  archive      = {J_NATEE},
  author       = {Neves, Danilo M.},
  doi          = {10.1038/s41559-022-01870-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1595-1596},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecoinformatics for conservation biology},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Resolving malaria’s dry-season dilemma. <em>NATEE</em>,
<em>6</em>(11), 1593–1594. (<a
href="https://doi.org/10.1038/s41559-022-01832-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An innovative isotopic labelling strategy shows that malaria mosquitoes in the West-African Sahel region survive in dormancy over the prolonged dry season. These results have implications for efforts to suppress malaria transmission in Africa.},
  archive      = {J_NATEE},
  author       = {Armbruster, Peter A.},
  doi          = {10.1038/s41559-022-01832-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1593-1594},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Resolving malaria’s dry-season dilemma},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The view at the end of the palaeolithic world.
<em>NATEE</em>, <em>6</em>(11), 1591–1592. (<a
href="https://doi.org/10.1038/s41559-022-01899-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two Palaeolithic genomes from Britain provide the oldest currently available genetic data from the region and appear to map on to wider European patterns of genetic ancestry and associated archaeology. However, with sparse samples and wide temporal gaps between them, it might be premature to draw wider conclusions about the consistency of these patterns.},
  archive      = {J_NATEE},
  author       = {Conneller, Chantal},
  doi          = {10.1038/s41559-022-01899-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1591-1592},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The view at the end of the palaeolithic world},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Life through the lens of metabolism. <em>NATEE</em>,
<em>6</em>(11), 1590. (<a
href="https://doi.org/10.1038/s41559-022-01880-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Eme, Laura and Stairs, Courtney W.},
  doi          = {10.1038/s41559-022-01880-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1590},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Life through the lens of metabolism},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Kamoya kimeu. <em>NATEE</em>, <em>6</em>(11), 1588–1589. (<a
href="https://doi.org/10.1038/s41559-022-01900-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expert field palaeontologist who made many key discoveries about early human evolution in East Africa.},
  archive      = {J_NATEE},
  author       = {Leakey, Louise N. and Foley, Robert A.},
  doi          = {10.1038/s41559-022-01900-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1588-1589},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Kamoya kimeu},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mosquito matters. <em>NATEE</em>, <em>6</em>(11), 1587. (<a
href="https://doi.org/10.1038/s41559-022-01938-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Three studies of disease-carrying mosquitoes in this issue illustrate the need for both interdisciplinary approaches and more research into fundamental biology.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01938-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1587},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mosquito matters},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Merging morphological and genetic evidence to assess
hybridization in western eurasian late pleistocene hominins.
<em>NATEE</em>, <em>6</em>(10), 1573–1585. (<a
href="https://doi.org/10.1038/s41559-022-01875-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Previous scientific consensus saw human evolution as defined by adaptive differences (behavioural and/or biological) and the emergence of Homo sapiens as the ultimate replacement of non-modern groups by a modern, adaptively more competitive group. However, recent research has shown that the process underlying our origins was considerably more complex. While archaeological and fossil evidence suggests that behavioural complexity may not be confined to the modern human lineage, recent palaeogenomic work shows that gene flow between distinct lineages (for example, Neanderthals, Denisovans, early H. sapiens) occurred repeatedly in the late Pleistocene, probably contributing elements to our genetic make-up that might have been crucial to our success as a diverse, adaptable species. Following these advances, the prevailing human origins model has shifted from one of near-complete replacement to a more nuanced view of partial replacement with considerable reticulation. Here we provide a brief introduction to the current genetic evidence for hybridization among hominins, its prevalence in, and effects on, comparative mammal groups, and especially how it manifests in the skull. We then explore the degree to which cranial variation seen in the fossil record of late Pleistocene hominins from Western Eurasia corresponds with our current genetic and comparative data. We are especially interested in understanding the degree to which skeletal data can reflect admixture. Our findings indicate some correspondence between these different lines of evidence, flag individual fossils as possibly admixed, and suggest that different cranial regions may preserve hybridization signals differentially. We urge further studies of the phenotype to expand our ability to detect the ways in which migration, interaction and genetic exchange have shaped the human past, beyond what is currently visible with the lens of ancient DNA. Genetic evidence suggests that Neanderthals and Homo sapiens repeatedly interbred in Europe. Here the authors generate predictions for how this might manifest in the late Pleistocene fossil record of Europe, focusing on the skull, and analyse the available evidence.},
  archive      = {J_NATEE},
  author       = {Harvati, K. and Ackermann, R. R.},
  doi          = {10.1038/s41559-022-01875-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1573-1585},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Merging morphological and genetic evidence to assess hybridization in western eurasian late pleistocene hominins},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An early origin of iron–sulfur cluster biosynthesis
machineries before earth oxygenation. <em>NATEE</em>, <em>6</em>(10),
1564–1572. (<a
href="https://doi.org/10.1038/s41559-022-01857-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Iron–sulfur (Fe–S) clusters are ubiquitous cofactors essential for life. It is largely thought that the emergence of oxygenic photosynthesis and progressive oxygenation of the atmosphere led to the origin of multiprotein machineries (ISC, NIF and SUF) assisting Fe–S cluster synthesis in the presence of oxidative stress and shortage of bioavailable iron. However, previous analyses have left unclear the origin and evolution of these systems. Here, we combine exhaustive homology searches with genomic context analysis and phylogeny to precisely identify Fe–S cluster biogenesis systems in over 10,000 archaeal and bacterial genomes. We highlight the existence of two additional and clearly distinct ‘minimal’ Fe–S cluster assembly machineries, MIS (minimal iron–sulfur) and SMS (SUF-like minimal system), which we infer in the last universal common ancestor (LUCA) and we experimentally validate SMS as a bona fide Fe–S cluster biogenesis system. These ancestral systems were kept in archaea whereas they went through stepwise complexification in bacteria to incorporate additional functions for higher Fe–S cluster synthesis efficiency leading to SUF, ISC and NIF. Horizontal gene transfers and losses then shaped the current distribution of these systems, driving ecological adaptations such as the emergence of aerobic lifestyles in archaea. Our results show that dedicated machineries were in place early in evolution to assist Fe–S cluster biogenesis and that their origin is not directly linked to Earth oxygenation. Iron–sulfur (Fe–S) clusters are cofactors essential for life. Combining large-scale phylogenomic analyses with biochemical validation, the authors identify two ancestral minimal Fe–S cluster biogenesis systems and show that they originated before Earth oxygenation.},
  archive      = {J_NATEE},
  author       = {Garcia, Pierre Simon and D’Angelo, Francesca and Ollagnier de Choudens, Sandrine and Dussouchaud, Macha and Bouveret, Emmanuelle and Gribaldo, Simonetta and Barras, Frédéric},
  doi          = {10.1038/s41559-022-01857-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1564-1572},
  shortjournal = {Nat. Ecol. Evol},
  title        = {An early origin of iron–sulfur cluster biosynthesis machineries before earth oxygenation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Footprint beds record holocene decline in large mammal
diversity on the irish sea coast of britain. <em>NATEE</em>,
<em>6</em>(10), 1553–1563. (<a
href="https://doi.org/10.1038/s41559-022-01856-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-term monitoring along the Irish Sea coast of Britain at Formby has identified hundreds of animal and human footprints in 31 discrete sediment beds. A new programme of radiocarbon dating shows that the Formby footprints span at least 8,000 years of the Holocene Epoch from the Mesolithic period to Medieval times. In a landscape largely devoid of conventional archaeology and faunal records, we show how species data from the footprint stratigraphy document long-term change in both large mammal diversity and human behaviour. The footprint beds record shifting community structure in the native fauna through an era of profound global change. As sea levels rose rapidly in the Early Holocene, men, women and children formed part of rich Mesolithic intertidal ecosystems from ~9,000 to 6,000 cal bp, with aurochs, red deer, roe deer, wild boar, beaver, wolf and lynx. Doggerland was reclaimed by the sea in this period. In the agriculture-based societies that followed, after 5,500 cal bp human footprints dominate the Neolithic period and later beds, alongside a striking fall in large mammal species richness. Stacked footprint beds can form multimillennial records of ecosystem change with precise geographical context that cannot be retrieved from site-based fossil bone assemblages. The authors analyse hundreds of animal and human footprints spanning at least 8,000 years at Formby on the Irish Sea coast of Britain. In the absence of conventional faunal records, the footprints document long-term changes in large mammal diversity and human activity during the Holocene.},
  archive      = {J_NATEE},
  author       = {Burns, Alison and Woodward, Jamie and Conneller, Chantal and Reimer, Paula},
  doi          = {10.1038/s41559-022-01856-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1553-1563},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Footprint beds record holocene decline in large mammal diversity on the irish sea coast of britain},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evolution of stickleback spines through independent
cis-regulatory changes at HOXDB. <em>NATEE</em>, <em>6</em>(10),
1537–1552. (<a
href="https://doi.org/10.1038/s41559-022-01855-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the mechanisms leading to new traits or additional features in organisms is a fundamental goal of evolutionary biology. We show that HOXDB regulatory changes have been used repeatedly in different fish genera to alter the length and number of the prominent dorsal spines used to classify stickleback species. In Gasterosteus aculeatus (typically ‘three-spine sticklebacks’), a variant HOXDB allele is genetically linked to shortening an existing spine and adding an additional spine. In Apeltes quadracus (typically ‘four-spine sticklebacks’), a variant HOXDB allele is associated with lengthening a spine and adding an additional spine in natural populations. The variant alleles alter the same non-coding enhancer region in the HOXDB locus but do so by diverse mechanisms, including single-nucleotide polymorphisms, deletions and transposable element insertions. The independent regulatory changes are linked to anterior expansion or contraction of HOXDB expression. We propose that associated changes in spine lengths and numbers are partial identity transformations in a repeating skeletal series that forms major defensive structures in fish. Our findings support the long-standing hypothesis that natural Hox gene variation underlies key patterning changes in wild populations and illustrate how different mutational mechanisms affecting the same region may produce opposite gene expression changes with similar phenotypic outcomes. Differences in a non-coding enhancer region of the HOXDB locus underlie the differences between stickleback species in dorsal spine length and number. These differences include single-nucleotide polymorphisms, deletions and transposable element insertions.},
  archive      = {J_NATEE},
  author       = {Wucherpfennig, Julia I. and Howes, Timothy R. and Au, Jessica N. and Au, Eric H. and Roberts Kingman, Garrett A. and Brady, Shannon D. and Herbert, Amy L. and Reimchen, Thomas E. and Bell, Michael A. and Lowe, Craig B. and Dalziel, Anne C. and Kingsley, David M.},
  doi          = {10.1038/s41559-022-01855-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1537-1552},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolution of stickleback spines through independent cis-regulatory changes at HOXDB},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Estimating comparable distances to tipping points across
mutualistic systems by scaled recovery rates. <em>NATEE</em>,
<em>6</em>(10), 1524–1536. (<a
href="https://doi.org/10.1038/s41559-022-01850-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutualistic systems can experience abrupt and irreversible regime shifts caused by local or global stressors. Despite decades of efforts to understand ecosystem dynamics and determine whether a tipping point could occur, there are no current approaches to estimate distances (in state/parameter space) to tipping points and compare the distances across various mutualistic systems. Here we develop a general dimension-reduction approach that simultaneously compresses the natural control and state parameters of high-dimensional complex systems and introduces a scaling factor for recovery rates. Our theoretical framework places various systems with entirely different dynamical parameters, network structure and state perturbations on the same scale. More importantly, it compares distances to tipping points across different systems on the basis of data on abundance and topology. By applying the method to 54 real-world mutualistic networks, our analytical results unveil the network characteristics and system parameters that control a system’s resilience. We contribute to the ongoing efforts in developing a general framework for mapping and predicting distance to tipping points of ecological and potentially other systems. The authors develop a theoretical method to reduce the complex dynamics of a mutualistic system to a single dynamical equation, which is then used to estimate and compare the distances to a potential tipping point across systems with scaled recovery rates.},
  archive      = {J_NATEE},
  author       = {Zhang, Huixin and Wang, Qi and Zhang, Weidong and Havlin, Shlomo and Gao, Jianxi},
  doi          = {10.1038/s41559-022-01850-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1524-1536},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Estimating comparable distances to tipping points across mutualistic systems by scaled recovery rates},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Greater bee diversity is needed to maintain crop pollination
over time. <em>NATEE</em>, <em>6</em>(10), 1516–1523. (<a
href="https://doi.org/10.1038/s41559-022-01847-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The current biodiversity crisis underscores the need to understand how biodiversity loss affects ecosystem function in real-world ecosystems. At any one place and time, a few highly abundant species often provide the majority of function, suggesting that function could be maintained with relatively little biodiversity. However, biodiversity may be critical to ecosystem function at longer timescales if different species are needed to provide function at different times. Here we show that the number of wild bee species needed to maintain a threshold level of crop pollination increased steeply with the timescale examined: two to three times as many bee species were needed over a growing season compared to on a single day and twice as many species were needed over six years compared to during a single year. Our results demonstrate the importance of pollinator biodiversity to maintaining pollination services across time and thus to stable agricultural output. The timescale over which ecosystem functions rely on biodiversity is not well characterized. Analysing pollination of two different crops by wild bees, the authors show that a greater number of species are needed to provide the same threshold pollination service as temporal scale increases.},
  archive      = {J_NATEE},
  author       = {Lemanski, Natalie J. and Williams, Neal M. and Winfree, Rachael},
  doi          = {10.1038/s41559-022-01847-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1516-1523},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Greater bee diversity is needed to maintain crop pollination over time},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Asymmetrical dose responses shape the evolutionary trade-off
between antifungal resistance and nutrient use. <em>NATEE</em>,
<em>6</em>(10), 1501–1515. (<a
href="https://doi.org/10.1038/s41559-022-01846-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antimicrobial resistance is an emerging threat for public health. The success of resistance mutations depends on the trade-off between the benefits and costs they incur. This trade-off is largely unknown and uncharacterized for antifungals. Here, we systematically measure the effect of all amino acid substitutions in the yeast cytosine deaminase Fcy1, the target of the antifungal 5-fluorocytosine (5-FC, flucytosine). We identify over 900 missense mutations granting resistance to 5-FC, a large fraction of which appear to act through destabilization of the protein. The relationship between 5-FC resistance and growth sustained by cytosine deamination is characterized by a sharp trade-off, such that small gains in resistance universally lead to large losses in canonical enzyme function. We show that this steep relationship can be explained by differences in the dose–response functions of 5-FC and cytosine. Finally, we observe the same trade-off shape for the orthologue of FCY1 in Cryptoccocus neoformans, a human pathogen. Our results provide a powerful resource and platform for interpreting drug target variants in fungal pathogens as well as unprecedented insights into resistance–function trade-offs. Studying all amino acid substitutions in the yeast cytosine deaminase Fcy1, the target of the antifungal 5-FC, the authors show a sharp trade-off between 5-FC resistance and growth sustained by cytosine deamination.},
  archive      = {J_NATEE},
  author       = {Després, Philippe C. and Cisneros, Angel F. and Alexander, Emilie M. M. and Sonigara, Ria and Gagné-Thivierge, Cynthia and Dubé, Alexandre K. and Landry, Christian R.},
  doi          = {10.1038/s41559-022-01846-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1501-1515},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Asymmetrical dose responses shape the evolutionary trade-off between antifungal resistance and nutrient use},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evolution of fungal phenotypic disparity. <em>NATEE</em>,
<em>6</em>(10), 1489–1500. (<a
href="https://doi.org/10.1038/s41559-022-01844-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organismal-grade multicellularity has been achieved only in animals, plants and fungi. All three kingdoms manifest phenotypically disparate body plans but their evolution has only been considered in detail for animals. Here we tested the general relevance of hypotheses on the evolutionary assembly of animal body plans by characterizing the evolution of fungal phenotypic variety (disparity). The distribution of living fungal form is defined by four distinct morphotypes: flagellated; zygomycetous; sac-bearing; and club-bearing. The discontinuity between morphotypes is a consequence of extinction, indicating that a complete record of fungal disparity would present a more homogeneous distribution of form. Fungal disparity expands episodically through time, punctuated by a sharp increase associated with the emergence of multicellular body plans. Simulations show these temporal trends to be non-random and at least partially shaped by hierarchical contingency. These trends are decoupled from changes in gene number, genome size and taxonomic diversity. Only differences in organismal complexity, characterized as the number of traits that constitute an organism, exhibit a meaningful relationship with fungal disparity. Both animals and fungi exhibit episodic increases in disparity through time, resulting in distributions of form made discontinuous by extinction. These congruences suggest a common mode of multicellular body plan evolution. Analyses of phenotypic variety in Fungi show that fungal body plans diversified episodically over time and appear distinct because of the extinction of intermediate forms, similar to what has been described in animals.},
  archive      = {J_NATEE},
  author       = {Smith, Thomas J. and Donoghue, Philip C. J.},
  doi          = {10.1038/s41559-022-01844-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1489-1500},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolution of fungal phenotypic disparity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Competition and coevolution drive the evolution and the
diversification of CRISPR immunity. <em>NATEE</em>, <em>6</em>(10),
1480–1488. (<a
href="https://doi.org/10.1038/s41559-022-01841-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The diversity of resistance challenges the ability of pathogens to spread and to exploit host populations. Yet, how this host diversity evolves over time remains unclear because it depends on the interplay between intraspecific competition among host genotypes and coevolution with pathogens. Here we study experimentally the effect of coevolving phage populations on the diversification of bacterial CRISPR immunity across space and time. We demonstrate that the negative-frequency-dependent selection generated by coevolution is a powerful force that maintains host resistance diversity and selects for new resistance mutations in the host. We also find that host evolution is driven by asymmetries in competitive abilities among different host genotypes. Even if the fittest host genotypes are targeted preferentially by the evolving phages, they often escape extinctions through the acquisition of new CRISPR immunity. Together, these fluctuating selective pressures maintain diversity, but not by preserving the pre-existing host composition. Instead, we repeatedly observe the introduction of new resistance genotypes stemming from the fittest hosts in each population. These results highlight the importance of competition on the transient dynamics of host–pathogen coevolution. This experimental study shows how the interplay between bacteria-phage coevolution and competition among bacterial genotypes results in increased diversity of bacterial CRISPR immunity.},
  archive      = {J_NATEE},
  author       = {Guillemet, Martin and Chabas, Hélène and Nicot, Antoine and Gatchich, François and Ortega-Abboud, Enrique and Buus, Cornelia and Hindhede, Lotte and Rousseau, Geneviève M. and Bataillon, Thomas and Moineau, Sylvain and Gandon, Sylvain},
  doi          = {10.1038/s41559-022-01841-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1480-1488},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Competition and coevolution drive the evolution and the diversification of CRISPR immunity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The gut microbiota affects the social network of honeybees.
<em>NATEE</em>, <em>6</em>(10), 1471–1479. (<a
href="https://doi.org/10.1038/s41559-022-01840-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The gut microbiota influences animal neurodevelopment and behaviour but has not previously been documented to affect group-level properties of social organisms. Here, we use honeybees to probe the effect of the gut microbiota on host social behaviour. We found that the microbiota increased the rate and specialization of head-to-head interactions between bees. Microbiota colonization was associated with higher abundances of one-third of the metabolites detected in the brain, including amino acids with roles in synaptic transmission and brain energetic function. Some of these metabolites were significant predictors of the number of social interactions. Microbiota colonization also affected brain transcriptional processes related to amino acid metabolism and epigenetic modifications in a brain region involved in sensory perception. These results demonstrate that the gut microbiota modulates the emergent colony social network of honeybees and suggest changes in chromatin accessibility and amino acid biosynthesis as underlying processes. Tracking the behaviour of normal versus microbiome-free honeybees in experimental colonies, the authors show that gut microbiota colonization was associated with an increase in the rate and specificity of social interactions among bees and higher abundances of brain metabolites linked to these interactions.},
  archive      = {J_NATEE},
  author       = {Liberti, Joanito and Kay, Tomas and Quinn, Andrew and Kesner, Lucie and Frank, Erik T. and Cabirol, Amélie and Richardson, Thomas O. and Engel, Philipp and Keller, Laurent},
  doi          = {10.1038/s41559-022-01840-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1471-1479},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The gut microbiota affects the social network of honeybees},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Global patterns and rates of habitat transitions across the
eukaryotic tree of life. <em>NATEE</em>, <em>6</em>(10), 1458–1470. (<a
href="https://doi.org/10.1038/s41559-022-01838-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The successful colonization of new habitats has played a fundamental role during the evolution of life. Salinity is one of the strongest barriers for organisms to cross, which has resulted in the evolution of distinct marine and non-marine (including both freshwater and soil) communities. Although microbes represent by far the vast majority of eukaryote diversity, the role of the salt barrier in shaping the diversity across the eukaryotic tree is poorly known. Traditional views suggest rare and ancient marine/non-marine transitions but this view is being challenged by the discovery of several recently transitioned lineages. Here, we investigate habitat evolution across the tree of eukaryotes using a unique set of taxon-rich phylogenies inferred from a combination of long-read and short-read environmental metabarcoding data spanning the ribosomal DNA operon. Our results show that, overall, marine and non-marine microbial communities are phylogenetically distinct but transitions have occurred in both directions in almost all major eukaryotic lineages, with hundreds of transition events detected. Some groups have experienced relatively high rates of transitions, most notably fungi for which crossing the salt barrier has probably been an important aspect of their successful diversification. At the deepest phylogenetic levels, ancestral habitat reconstruction analyses suggest that eukaryotes may have first evolved in non-marine habitats and that the two largest known eukaryotic assemblages (TSAR and Amorphea) arose in different habitats. Overall, our findings indicate that the salt barrier has played an important role during eukaryote evolution and provide a global perspective on habitat transitions in this domain of life. Eukaryotic phylogenies inferred from metabarcoding show that marine and non-marine microbial eukaryotes are in general phylogenetically distinct, but that transitions across the salt barrier have occurred hundreds of times, and in all lineages, and are particularly important for evolutionary diversification in fungi.},
  archive      = {J_NATEE},
  author       = {Jamy, Mahwash and Biwer, Charlie and Vaulot, Daniel and Obiol, Aleix and Jing, Hongmei and Peura, Sari and Massana, Ramon and Burki, Fabien},
  doi          = {10.1038/s41559-022-01838-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1458-1470},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Global patterns and rates of habitat transitions across the eukaryotic tree of life},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Adaptive phenotypic plasticity is under stabilizing
selection in daphnia. <em>NATEE</em>, <em>6</em>(10), 1449–1457. (<a
href="https://doi.org/10.1038/s41559-022-01837-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The adaptive nature of phenotypic plasticity is widely documented. However, little is known about the evolutionary forces that shape genetic variation of plasticity within populations. Whether genetic variation in plasticity is driven by stabilizing or diversifying selection and whether the strength of such forces remains constant through time, remain open questions. Here, we address this issue by assessing the evolutionary forces that shape genetic variation in antipredator developmental plasticity of Daphnia pulex. Antipredator plasticity in D. pulex is characterized by the growth of a pedestal and spikes in the dorsal head region upon exposure to predator cue. We characterized genetic variation in plasticity using a method that describes the entire dorsal shape amongst &amp;gt;100 D. pulex strains recently derived from the wild. We observed the strongest reduction in genetic variation in dorsal areas where plastic responses were greatest, consistent with stabilizing selection. We compared mutational variation (Vm) to standing variation (Vg) and found that Vg/Vm is lowest in areas of greatest plasticity, again consistent with stabilizing selection. Our results suggest that stabilizing selection operates directly on phenotypic plasticity in Daphnia and provide a rare glimpse into the evolution of fitness-related traits in natural populations. Analysis of genetic variation underlying an antipredator morphological defence in Daphnia pulex shows that stabilizing selection operates on this plastic trait.},
  archive      = {J_NATEE},
  author       = {Becker, Dörthe and Barnard-Kubow, Karen and Porter, Robert and Edwards, Austin and Voss, Erin and Beckerman, Andrew P. and Bergland, Alan O.},
  doi          = {10.1038/s41559-022-01837-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1449-1457},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Adaptive phenotypic plasticity is under stabilizing selection in daphnia},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mass spectrometry of short peptides reveals common features
of metazoan peptidergic neurons. <em>NATEE</em>, <em>6</em>(10),
1438–1448. (<a
href="https://doi.org/10.1038/s41559-022-01835-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolutionary origins of neurons remain unknown. Although recent genome data of extant early-branching animals have shown that neural genes existed in the common ancestor of animals, the physiological and genetic properties of neurons in the early evolutionary phase are still unclear. Here, we performed a mass spectrometry-based comprehensive survey of short peptides from early-branching lineages Cnidaria, Porifera and Ctenophora. We identified a number of mature ctenophore neuropeptides that are expressed in neurons associated with sensory, muscular and digestive systems. The ctenophore peptides are stored in vesicles in cell bodies and neurites, suggesting volume transmission similar to that of cnidarian and bilaterian peptidergic systems. A comparison of genetic characteristics revealed that the peptide-expressing cells of Cnidaria and Ctenophora express the vast majority of genes that have pivotal roles in maturation, secretion and degradation of neuropeptides in Bilateria. Functional analysis of neuropeptides and prediction of receptors with machine learning demonstrated peptide regulation of a wide range of target effector cells, including cells of muscular systems. The striking parallels between the peptidergic neuronal properties of Cnidaria and Bilateria and those of Ctenophora, the most basal neuron-bearing animals, suggest a common evolutionary origin of metazoan peptidergic nervous systems. Biochemical identification of neuropeptides in Cnidaria and Ctenophora, followed by analyses of their expression, suggests that peptidergic neurons were present at early stages of nervous system evolution.},
  archive      = {J_NATEE},
  author       = {Hayakawa, Eisuke and Guzman, Christine and Horiguchi, Osamu and Kawano, Chihiro and Shiraishi, Akira and Mohri, Kurato and Lin, Mei-Fang and Nakamura, Ryotaro and Nakamura, Ryo and Kawai, Erina and Komoto, Shinya and Jokura, Kei and Shiba, Kogiku and Shigenobu, Shuji and Satake, Honoo and Inaba, Kazuo and Watanabe, Hiroshi},
  doi          = {10.1038/s41559-022-01835-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1438-1448},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mass spectrometry of short peptides reveals common features of metazoan peptidergic neurons},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Co-limitation towards lower latitudes shapes global forest
diversity gradients. <em>NATEE</em>, <em>6</em>(10), 1423–1437. (<a
href="https://doi.org/10.1038/s41559-022-01831-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The latitudinal diversity gradient (LDG) is one of the most recognized global patterns of species richness exhibited across a wide range of taxa. Numerous hypotheses have been proposed in the past two centuries to explain LDG, but rigorous tests of the drivers of LDGs have been limited by a lack of high-quality global species richness data. Here we produce a high-resolution (0.025° × 0.025°) map of local tree species richness using a global forest inventory database with individual tree information and local biophysical characteristics from ~1.3 million sample plots. We then quantify drivers of local tree species richness patterns across latitudes. Generally, annual mean temperature was a dominant predictor of tree species richness, which is most consistent with the metabolic theory of biodiversity (MTB). However, MTB underestimated LDG in the tropics, where high species richness was also moderated by topographic, soil and anthropogenic factors operating at local scales. Given that local landscape variables operate synergistically with bioclimatic factors in shaping the global LDG pattern, we suggest that MTB be extended to account for co-limitation by subordinate drivers. Examining drivers of the latitudinal biodiversity gradient in a global database of local tree species richness, the authors show that co-limitation by multiple environmental and anthropogenic factors causes steeper increases in richness with latitude in tropical versus temperate and boreal zones.},
  archive      = {J_NATEE},
  author       = {Liang, Jingjing and Gamarra, Javier G. P. and Picard, Nicolas and Zhou, Mo and Pijanowski, Bryan and Jacobs, Douglass F. and Reich, Peter B. and Crowther, Thomas W. and Nabuurs, Gert-Jan and de-Miguel, Sergio and Fang, Jingyun and Woodall, Christopher W. and Svenning, Jens-Christian and Jucker, Tommaso and Bastin, Jean-Francois and Wiser, Susan K. and Slik, Ferry and Hérault, Bruno and Alberti, Giorgio and Keppel, Gunnar and Hengeveld, Geerten M. and Ibisch, Pierre L. and Silva, Carlos A. and ter Steege, Hans and Peri, Pablo L. and Coomes, David A. and Searle, Eric B. and von Gadow, Klaus and Jaroszewicz, Bogdan and Abbasi, Akane O. and Abegg, Meinrad and Yao, Yves C. Adou and Aguirre-Gutiérrez, Jesús and Zambrano, Angelica M. Almeyda and Altman, Jan and Alvarez-Dávila, Esteban and Álvarez-González, Juan Gabriel and Alves, Luciana F. and Amani, Bienvenu H. K. and Amani, Christian A. and Ammer, Christian and Ilondea, Bhely Angoboy and Antón-Fernández, Clara and Avitabile, Valerio and Aymard, Gerardo A. and Azihou, Akomian F. and Baard, Johan A. and Baker, Timothy R. and Balazy, Radomir and Bastian, Meredith L. and Batumike, Rodrigue and Bauters, Marijn and Beeckman, Hans and Benu, Nithanel Mikael Hendrik and Bitariho, Robert and Boeckx, Pascal and Bogaert, Jan and Bongers, Frans and Bouriaud, Olivier and Brancalion, Pedro H. S. and Brandl, Susanne and Brearley, Francis Q. and Briseno-Reyes, Jaime and Broadbent, Eben N. and Bruelheide, Helge and Bulte, Erwin and Catlin, Ann Christine and Cazzolla Gatti, Roberto and César, Ricardo G. and Chen, Han Y. H. and Chisholm, Chelsea and Cienciala, Emil and Colletta, Gabriel D. and Corral-Rivas, José Javier and Cuchietti, Anibal and Cuni-Sanchez, Aida and Dar, Javid A. and Dayanandan, Selvadurai and de Haulleville, Thales and Decuyper, Mathieu and Delabye, Sylvain and Derroire, Géraldine and DeVries, Ben and Diisi, John and Do, Tran Van and Dolezal, Jiri and Dourdain, Aurélie and Durrheim, Graham P. and Obiang, Nestor Laurier Engone and Ewango, Corneille E. N. and Eyre, Teresa J. and Fayle, Tom M. and Feunang, Lethicia Flavine N. and Finér, Leena and Fischer, Markus and Fridman, Jonas and Frizzera, Lorenzo and de Gasper, André L. and Gianelle, Damiano and Glick, Henry B. and Gonzalez-Elizondo, Maria Socorro and Gorenstein, Lev and Habonayo, Richard and Hardy, Olivier J. and Harris, David J. and Hector, Andrew and Hemp, Andreas and Herold, Martin and Hillers, Annika and Hubau, Wannes and Ibanez, Thomas and Imai, Nobuo and Imani, Gerard and Jagodzinski, Andrzej M. and Janecek, Stepan and Johannsen, Vivian Kvist and Joly, Carlos A. and Jumbam, Blaise and Kabelong, Banoho L. P. R. and Kahsay, Goytom Abraha and Karminov, Viktor and Kartawinata, Kuswata and Kassi, Justin N. and Kearsley, Elizabeth and Kennard, Deborah K. and Kepfer-Rojas, Sebastian and Khan, Mohammed Latif and Kigomo, John N. and Kim, Hyun Seok and Klauberg, Carine and Klomberg, Yannick and Korjus, Henn and Kothandaraman, Subashree and Kraxner, Florian and Kumar, Amit and Kuswandi, Relawan and Lang, Mait and Lawes, Michael J. and Leite, Rodrigo V. and Lentner, Geoffrey and Lewis, Simon L. and Libalah, Moses B. and Lisingo, Janvier and López-Serrano, Pablito Marcelo and Lu, Huicui and Lukina, Natalia V. and Lykke, Anne Mette and Maicher, Vincent and Maitner, Brian S. and Marcon, Eric and Marshall, Andrew R. and Martin, Emanuel H. and Martynenko, Olga and Mbayu, Faustin M. and Mbuvi, Musingo T. E. and Meave, Jorge A. and Merow, Cory and Miscicki, Stanislaw and Moreno, Vanessa S. and Morera, Albert and Mukul, Sharif A. and Müller, Jörg C. and Murdjoko, Agustinus and Nava-Miranda, Maria Guadalupe and Ndive, Litonga Elias and Neldner, Victor J. and Nevenic, Radovan V. and Nforbelie, Louis N. and Ngoh, Michael L. and N’Guessan, Anny E. and Ngugi, Michael R. and Ngute, Alain S. K. and Njila, Emile Narcisse N. and Nyako, Melanie C. and Ochuodho, Thomas O. and Oleksyn, Jacek and Paquette, Alain and Parfenova, Elena I. and Park, Minjee and Parren, Marc and Parthasarathy, Narayanaswamy and Pfautsch, Sebastian and Phillips, Oliver L. and Piedade, Maria T. F. and Piotto, Daniel and Pollastrini, Martina and Poorter, Lourens and Poulsen, John R. and Poulsen, Axel Dalberg and Pretzsch, Hans and Rodeghiero, Mirco and Rolim, Samir G. and Rovero, Francesco and Rutishauser, Ervan and Sagheb-Talebi, Khosro and Saikia, Purabi and Sainge, Moses Nsanyi and Salas-Eljatib, Christian and Salis, Antonello and Schall, Peter and Schepaschenko, Dmitry and Scherer-Lorenzen, Michael and Schmid, Bernhard and Schöngart, Jochen and Šebeň, Vladimír and Sellan, Giacomo and Selvi, Federico and Serra-Diaz, Josep M. and Sheil, Douglas and Shvidenko, Anatoly Z. and Sist, Plinio and Souza, Alexandre F. and Stereńczak, Krzysztof J. and Sullivan, Martin J. P. and Sundarapandian, Somaiah and Svoboda, Miroslav and Swaine, Mike D. and Targhetta, Natalia and Tchebakova, Nadja and Trethowan, Liam A. and Tropek, Robert and Mukendi, John Tshibamba and Umunay, Peter Mbanda and Usoltsev, Vladimir A. and Vaglio Laurin, Gaia and Valentini, Riccardo and Valladares, Fernando and van der Plas, Fons and Vega-Nieva, Daniel José and Verbeeck, Hans and Viana, Helder and Vibrans, Alexander C. and Vieira, Simone A. and Vleminckx, Jason and Waite, Catherine E. and Wang, Hua-Feng and Wasingya, Eric Katembo and Wekesa, Chemuku and Westerlund, Bertil and Wittmann, Florian and Wortel, Verginia and Zawiła-Niedźwiecki, Tomasz and Zhang, Chunyu and Zhao, Xiuhai and Zhu, Jun and Zhu, Xiao and Zhu, Zhi-Xin and Zo-Bi, Irie C. and Hui, Cang},
  doi          = {10.1038/s41559-022-01831-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1423-1437},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Co-limitation towards lower latitudes shapes global forest diversity gradients},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ecological and evolutionary dynamics of multi-strain RNA
viruses. <em>NATEE</em>, <em>6</em>(10), 1414–1422. (<a
href="https://doi.org/10.1038/s41559-022-01860-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Potential interactions among co-circulating viral strains in host populations are often overlooked in the study of virus transmission. However, these interactions probably shape transmission dynamics by influencing host immune responses or altering the relative fitness among co-circulating strains. In this Review, we describe multi-strain dynamics from ecological and evolutionary perspectives, outline scales in which multi-strain dynamics occur and summarize important immunological, phylogenetic and mathematical modelling approaches used to quantify interactions among strains. We also discuss how host–pathogen interactions influence the co-circulation of pathogens. Finally, we highlight outstanding questions and knowledge gaps in the current theory and study of ecological and evolutionary dynamics of multi-strain viruses. Many viruses evolve quickly, leading to the coexistence of multiple strains within the same host and population. In this Review, the authors synthesize ecological and evolutionary approaches to studying the dynamics of multi-strain RNA virus infections and suggest opportunities for future work.},
  archive      = {J_NATEE},
  author       = {Makau, Dennis N. and Lycett, Samantha and Michalska-Smith, Matthew and Paploski, Igor A. D. and Cheeran, Maxim C.-J. and Craft, Meggan E. and Kao, Rowland R. and Schroeder, Declan C. and Doeschl-Wilson, Andrea and VanderWaal, Kimberly},
  doi          = {10.1038/s41559-022-01860-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1414-1422},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecological and evolutionary dynamics of multi-strain RNA viruses},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CRISPR evolves among the winners. <em>NATEE</em>,
<em>6</em>(10), 1412–1413. (<a
href="https://doi.org/10.1038/s41559-022-01843-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coevolutionary warfare between bacteria and phage results in the diversification of anti-phage CRISPR arrays among the most successful bacterial competitors},
  archive      = {J_NATEE},
  author       = {Saha, Saheli and Pande, Samay},
  doi          = {10.1038/s41559-022-01843-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1412-1413},
  shortjournal = {Nat. Ecol. Evol},
  title        = {CRISPR evolves among the winners},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Neuropeptides at the origin of neurons. <em>NATEE</em>,
<em>6</em>(10), 1410–1411. (<a
href="https://doi.org/10.1038/s41559-022-01828-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cnidarians and ctenophores have morphologically simpler nervous systems than those of bilaterians. Discovery and characterization of neuropeptides in a comb jelly and a sea anemone support a common origin of animal peptidergic neurons from digestive cells that could sense their environment.},
  archive      = {J_NATEE},
  author       = {Sachkova, Maria Y.},
  doi          = {10.1038/s41559-022-01828-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1410-1411},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Neuropeptides at the origin of neurons},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Insects welcome foreign genes. <em>NATEE</em>,
<em>6</em>(10), 1409. (<a
href="https://doi.org/10.1038/s41559-022-01873-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-022-01873-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1409},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Insects welcome foreign genes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Making sense of animal senses. <em>NATEE</em>,
<em>6</em>(10), 1408. (<a
href="https://doi.org/10.1038/s41559-022-01851-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Stuart-Fox, Devi},
  doi          = {10.1038/s41559-022-01851-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1408},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Making sense of animal senses},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Coral conservation in a warming world must harness
evolutionary adaptation. <em>NATEE</em>, <em>6</em>(10), 1405–1407. (<a
href="https://doi.org/10.1038/s41559-022-01854-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To facilitate evolutionary adaptation to climate change, we must protect networks of coral reefs that span a range of environmental conditions — not just apparent ‘refugia’.},
  archive      = {J_NATEE},
  author       = {Colton, Madhavi A. and McManus, Lisa C. and Schindler, Daniel E. and Mumby, Peter J. and Palumbi, Stephen R. and Webster, Michael M. and Essington, Timothy E. and Fox, Helen E. and Forrest, Daniel L. and Schill, Steven R. and Pollock, F. Joseph and DeFilippo, Lukas B. and Tekwa, E. W. and Walsworth, Timothy E. and Pinsky, Malin L.},
  doi          = {10.1038/s41559-022-01854-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1405-1407},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Coral conservation in a warming world must harness evolutionary adaptation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The european green deal misses europe’s subterranean
biodiversity hotspots. <em>NATEE</em>, <em>6</em>(10), 1403–1404. (<a
href="https://doi.org/10.1038/s41559-022-01859-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Fišer, Cene and Borko, Špela and Delić, Teo and Kos, Anja and Premate, Ester and Zagmajster, Maja and Zakšek, Valerija and Altermatt, Florian},
  doi          = {10.1038/s41559-022-01859-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1403-1404},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The european green deal misses europe’s subterranean biodiversity hotspots},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A complex story of groundwater abstraction and ecological
threats to the doñana national park world heritage site. <em>NATEE</em>,
<em>6</em>(10), 1401–1402. (<a
href="https://doi.org/10.1038/s41559-022-01836-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Acreman, Mike and Salathe, Tobias},
  doi          = {10.1038/s41559-022-01836-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1401-1402},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A complex story of groundwater abstraction and ecological threats to the doñana national park world heritage site},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Silent spring at sixty. <em>NATEE</em>, <em>6</em>(10),
1399–1400. (<a
href="https://doi.org/10.1038/s41559-022-01911-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rachel Carson’s book has had lasting impacts on the global regulation of chemicals harmful to life. Six decades since its publication, however, novel threats to wildlife and human health are still emerging.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01911-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1399-1400},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Silent spring at sixty},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Publisher correction: Replicated radiation of a plant clade
along a cloud forest archipelago. <em>NATEE</em>, <em>6</em>(9), 1398.
(<a href="https://doi.org/10.1038/s41559-022-01864-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Donoghue, Michael J. and Eaton, Deren A. R. and Maya-Lastra, Carlos A. and Landis, Michael J. and Sweeney, Patrick W. and Olson, Mark E. and Cacho, N. Ivalú and Moeglein, Morgan K. and Gardner, Jordan R. and Heaphy, Nora M. and Castorena, Matiss and Rivas, Alí Segovia and Clement, Wendy L. and Edwards, Erika J.},
  doi          = {10.1038/s41559-022-01864-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1398},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Replicated radiation of a plant clade along a cloud forest archipelago},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Quantifying research waste in ecology. <em>NATEE</em>,
<em>6</em>(9), 1390–1397. (<a
href="https://doi.org/10.1038/s41559-022-01820-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research inefficiencies can generate huge waste: evidence from biomedical research has shown that most research is avoidably wasted and steps have been taken to tackle this costly problem. Although other scientific fields could also benefit from identifying and quantifying waste and acting to reduce it, no other estimates of research waste are available. Given that ecological issues interweave most of the United Nations Sustainable Development Goals, we argue that tackling research waste in ecology should be prioritized. Our study leads the way. We estimate components of waste in ecological research based on a literature review and a meta-analysis. Shockingly, our results suggest only 11–18% of conducted ecological research reaches its full informative value. All actors within the research system—including academic institutions, policymakers, funders and publishers—have a duty towards science, the environment, study organisms and the public, to urgently act and reduce this considerable yet preventable loss. We discuss potential ways forward and call for two major actions: (1) further research into waste in ecology (and beyond); (2) focused development and implementation of solutions to reduce unused potential of ecological research. Using a literature review and meta-analysis, the authors quantify the proportion of ecological research that is wasted because of poor study design and implementation, or because the work remains unpublished.},
  archive      = {J_NATEE},
  author       = {Purgar, Marija and Klanjscek, Tin and Culina, Antica},
  doi          = {10.1038/s41559-022-01820-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1390-1397},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Quantifying research waste in ecology},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Neuron numbers link innovativeness with both absolute and
relative brain size in birds. <em>NATEE</em>, <em>6</em>(9), 1381–1389.
(<a href="https://doi.org/10.1038/s41559-022-01815-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A longstanding issue in biology is whether the intelligence of animals can be predicted by absolute or relative brain size. However, progress has been hampered by an insufficient understanding of how neuron numbers shape internal brain organization and cognitive performance. On the basis of estimations of neuron numbers for 111 bird species, we show here that the number of neurons in the pallial telencephalon is positively associated with a major expression of intelligence: innovation propensity. The number of pallial neurons, in turn, is greater in brains that are larger in both absolute and relative terms and positively covaries with longer post-hatching development periods. Thus, our analyses show that neuron numbers link cognitive performance to both absolute and relative brain size through developmental adjustments. These findings help unify neuro-anatomical measures at multiple levels, reconciling contradictory views over the biological significance of brain expansion. The results also highlight the value of a life history perspective to advance our understanding of the evolutionary bases of the connections between brain and cognition. Using estimation data on neuron numbers in 111 bird species across 24 families, the authors show that number of neurons is positively associated with innovation propensity and encephalization.},
  archive      = {J_NATEE},
  author       = {Sol, Daniel and Olkowicz, Seweryn and Sayol, Ferran and Kocourek, Martin and Zhang, Yicheng and Marhounová, Lucie and Osadnik, Christin and Corssmit, Eva and Garcia-Porta, Joan and Martin, Thomas E. and Lefebvre, Louis and Němec, Pavel},
  doi          = {10.1038/s41559-022-01815-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1381-1389},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Neuron numbers link innovativeness with both absolute and relative brain size in birds},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The roles of balancing selection and recombination in the
evolution of rattlesnake venom. <em>NATEE</em>, <em>6</em>(9),
1367–1380. (<a
href="https://doi.org/10.1038/s41559-022-01829-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The origin of snake venom involved duplication and recruitment of non-venom genes into venom systems. Several studies have predicted that directional positive selection has governed this process. Venom composition varies substantially across snake species and venom phenotypes are locally adapted to prey, leading to coevolutionary interactions between predator and prey. Venom origins and contemporary snake venom evolution may therefore be driven by fundamentally different selection regimes, yet investigations of population-level patterns of selection have been limited. Here, we use whole-genome data from 68 rattlesnakes to test hypotheses about the factors that drive genomic diversity and differentiation in major venom gene regions. We show that selection has resulted in long-term maintenance of genetic diversity within and between species in multiple venom gene families. Our findings are inconsistent with a dominant role of directional positive selection and instead support a role of long-term balancing selection in shaping venom evolution. We also detect rapid decay of linkage disequilibrium due to high recombination rates in venom regions, suggesting that venom genes have reduced selective interference with nearby loci, including other venom paralogues. Our results provide an example of long-term balancing selection that drives trans-species polymorphism and help to explain how snake venom keeps pace with prey resistance. Analysing whole-genome sequences from 68 rattlesnakes, the authors show a role of long-term balancing selection in maintaining diversity of multiple venom gene families and find reduced selective interference of venom genes with neighbouring loci.},
  archive      = {J_NATEE},
  author       = {Schield, Drew R. and Perry, Blair W. and Adams, Richard H. and Holding, Matthew L. and Nikolakis, Zachary L. and Gopalan, Siddharth S. and Smith, Cara F. and Parker, Joshua M. and Meik, Jesse M. and DeGiorgio, Michael and Mackessy, Stephen P. and Castoe, Todd A.},
  doi          = {10.1038/s41559-022-01829-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1367-1380},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The roles of balancing selection and recombination in the evolution of rattlesnake venom},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Comparative genome anatomy reveals evolutionary insights
into a unique amphitriploid fish. <em>NATEE</em>, <em>6</em>(9),
1354–1366. (<a
href="https://doi.org/10.1038/s41559-022-01813-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Triploids are rare in nature because of difficulties in meiotic and gametogenic processes, especially in vertebrates. The Carassius complex of cyprinid teleosts contains sexual tetraploid crucian carp/goldfish (C. auratus) and unisexual hexaploid gibel carp/Prussian carp (C. gibelio) lineages, providing a valuable model for studying the evolution and maintenance mechanism of unisexual polyploids in vertebrates. Here we sequence the genomes of the two species and assemble their haplotypes, which contain two subgenomes (A and B), to the chromosome level. Sequencing coverage analysis reveals that C. gibelio is an amphitriploid (AAABBB) with two triploid sets of chromosomes; each set is derived from a different ancestor. Resequencing data from different strains of C. gibelio show that unisexual reproduction has been maintained for over 0.82 million years. Comparative genomics show intensive expansion and alterations of meiotic cell cycle-related genes and an oocyte-specific histone variant. Cytological assays indicate that C. gibelio produces unreduced oocytes by an alternative ameiotic pathway; however, sporadic homologous recombination and a high rate of gene conversion also exist in C. gibelio. These genomic changes might have facilitated purging deleterious mutations and maintaining genome stability in this unisexual amphitriploid fish. Overall, the current results provide novel insights into the evolutionary mechanisms of the reproductive success in unisexual polyploid vertebrates. Genome sequencing and haplotype assembly of two cyprinid teleosts, a sexual tetraploid and an unisexual hexaploid, reveal insights into the evolutionary mechanisms underpinning the reproductive success of unisexual polyploid vertebrates.},
  archive      = {J_NATEE},
  author       = {Wang, Yang and Li, Xi-Yin and Xu, Wen-Jie and Wang, Kun and Wu, Bin and Xu, Meng and Chen, Yan and Miao, Li-Jun and Wang, Zhong-Wei and Li, Zhi and Zhang, Xiao-Juan and Yin, Zhan and Zhou, Bo-Tong and Yang, Yu-Lan and Zhu, Cheng-Long and Hu, Ming-Liang and Zheng, Jiang-Ming and Feng, Chen-Guang and Qiu, Qiang and Tian, Le-Tian and Lu, Meng and Peng, Fang and Lu, Wei-Jia and Tong, Jin-Feng and Tong, Jin-Gou and Fu, Bei-De and Yu, Peng and Ding, Miao and Gan, Rui-Hai and Zhang, Qin-Qin and Jian, Jian-Bo and Zhang, Chi and He, Wei-Ming and Yang, Wei and Zhao, Zi-Cheng and Zhang, Qian-Qian and Gao, Qiang and Xu, Jun-Yang and Bai, Ming-Zhou and Zhang, Ya-Ping and Yang, Huan-Ming and Fang, Xiao-Dong and Wang, Wen and Zhou, Li and Gui, Jian-Fang},
  doi          = {10.1038/s41559-022-01813-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1354-1366},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Comparative genome anatomy reveals evolutionary insights into a unique amphitriploid fish},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Copy number changes in co-expressed odorant receptor genes
enable selection for sensory differences in drosophilid species.
<em>NATEE</em>, <em>6</em>(9), 1343–1353. (<a
href="https://doi.org/10.1038/s41559-022-01830-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite numerous examples of chemoreceptor gene family expansions and contractions, how these relate to modifications in the sensory neuron populations in which they are expressed remains unclear. Drosophila melanogaster’s odorant receptor (Or) family is ideal for addressing this question because most Ors are expressed in distinct olfactory sensory neuron (OSN) types. Between-species changes in Or copy number may therefore indicate increases or reductions in the number of OSN populations. Here we investigated the Or67a subfamily, which exhibits copy number variation in D. melanogaster and its closest relatives: D. simulans, D. sechellia and D. mauritiana. These species’ common ancestor had three Or67a paralogues that had already diverged adaptively. Following speciation, two Or67a paralogues were lost independently in D. melanogaster and D. sechellia, with ongoing positive selection shaping the intact genes. Unexpectedly, the functionally diverged Or67a paralogues in D. simulans are co-expressed in a single neuron population, which projects to a glomerulus homologous to that innervated by Or67a neurons in D. melanogaster. Thus, while sensory pathway neuroanatomy is conserved, independent selection on co-expressed receptors has contributed to species-specific peripheral coding. This work reveals a type of adaptive change largely overlooked for olfactory evolution, raising the possibility that similar processes influence other cases of insect Or co-expression. Studying the odorant receptor subfamily Or67a in Drosophila melanogaster and closely related species, the authors show that independent selection on co-expressed receptors has contributed to species-specific peripheral coding of olfactory information.},
  archive      = {J_NATEE},
  author       = {Auer, Thomas O. and Álvarez-Ocaña, Raquel and Cruchet, Steeve and Benton, Richard and Arguello, J. Roman},
  doi          = {10.1038/s41559-022-01830-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1343-1353},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Copy number changes in co-expressed odorant receptor genes enable selection for sensory differences in drosophilid species},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic evidence that a sexually selected trait captures
genome-wide variation and facilitates the purging of genetic load.
<em>NATEE</em>, <em>6</em>(9), 1330–1342. (<a
href="https://doi.org/10.1038/s41559-022-01816-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolution of costly traits such as deer antlers and peacock trains, which drove the formation of Darwinian sexual selection theory, has been suggested to both reflect and affect patterns of genetic variance across the genome, but direct tests are missing. Here, we used an evolve and resequence approach to reveal patterns of genome-wide diversity associated with the expression of a sexually selected weapon that is dimorphic among males of the bulb mite, Rhizoglyphus robini. Populations selected for the weapon showed reduced genome-wide diversity compared to populations selected against the weapon, particularly in terms of the number of segregating non-synonymous positions, indicating enhanced purifying selection. This increased purifying selection reduced inbreeding depression, but outbred female fitness did not improve, possibly because any benefits were offset by increased sexual antagonism. Most single nucleotide polymorphisms (SNPs) that consistently diverged in response to selection were initially rare and overrepresented in exons, and enriched in regions under balancing or relaxed selection, suggesting they are probably moderately deleterious variants. These diverged SNPs were scattered across the genome, further demonstrating that selection for or against the weapon and the associated changes to the mating system can both capture and influence genome-wide variation. Populations of a bulb mite that were experimentally selected for a male weapon showed reduced diversity across the genome, indicating that strong sexual selection increases the strength of purifying selection.},
  archive      = {J_NATEE},
  author       = {Parrett, Jonathan M. and Chmielewski, Sebastian and Aydogdu, Eylem and Łukasiewicz, Aleksandra and Rombauts, Stephane and Szubert-Kruszyńska, Agnieszka and Babik, Wiesław and Konczal, Mateusz and Radwan, Jacek},
  doi          = {10.1038/s41559-022-01816-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1330-1342},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genomic evidence that a sexually selected trait captures genome-wide variation and facilitates the purging of genetic load},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Replicated radiation of a plant clade along a cloud forest
archipelago. <em>NATEE</em>, <em>6</em>(9), 1318–1329. (<a
href="https://doi.org/10.1038/s41559-022-01823-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Replicated radiations, in which sets of similar forms evolve repeatedly within different regions, can provide powerful insights into parallel evolution and the assembly of functional diversity within communities. Several cases have been described in animals, but in plants we lack well-documented cases of replicated radiation that combine comprehensive phylogenetic and biogeographic analyses, the delimitation of geographic areas within which a set of ‘ecomorphs’ evolved independently and the identification of potential underlying mechanisms. Here we document the repeated evolution of a set of leaf ecomorphs in a group of neotropical plants. The Oreinotinus lineage within the angiosperm clade Viburnum spread from Mexico to Argentina through disjunct cloud forest environments. In 9 of 11 areas of endemism, species with similar sets of leaf forms evolved in parallel. We reject gene-flow-mediated evolution of similar leaves and show, instead, that species with disparate leaf forms differ in their climatic niches, supporting ecological adaptation as the driver of parallelism. Our identification of a case of replicated radiation in plants sets the stage for comparative analyses of such phenomena across the tree of life. Several cases of replicated radiations (in which sets of similar forms evolve repeatedly within different regions) have been described in animals. Here the authors provide a well-documented example in plants, specifically the Oreinotinus lineage within the angiosperm clade Viburnum in its spread from Mexico to Argentina through disjunct cloud forest environments.},
  archive      = {J_NATEE},
  author       = {Donoghue, Michael J. and Eaton, Deren A. R. and Maya-Lastra, Carlos A. and Landis, Michael J. and Sweeney, Patrick W. and Olson, Mark E. and Cacho, N. Ivalú and Moeglein, Morgan K. and Gardner, Jordan R. and Heaphy, Nora M. and Castorena, Matiss and Rivas, Alí Segovia and Clement, Wendy L. and Edwards, Erika J.},
  doi          = {10.1038/s41559-022-01823-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1318-1329},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Replicated radiation of a plant clade along a cloud forest archipelago},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The role of mitochondrial energetics in the origin and
diversification of eukaryotes. <em>NATEE</em>, <em>6</em>(9), 1307–1317.
(<a href="https://doi.org/10.1038/s41559-022-01833-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The origin of eukaryotic cell size and complexity is often thought to have required an energy excess supplied by mitochondria. Recent observations show energy demands to scale continuously with cell volume, suggesting that eukaryotes do not have higher energetic capacity. However, respiratory membrane area scales superlinearly with the cell surface area. Furthermore, the consequences of the contrasting genomic architectures between prokaryotes and eukaryotes have not been precisely quantified. Here, we investigated (1) the factors that affect the volumes at which prokaryotes become surface area-constrained, (2) the amount of energy divested to DNA due to contrasting genomic architectures and (3) the costs and benefits of respiring symbionts. Our analyses suggest that prokaryotes are not surface area-constrained at volumes of 100‒103 µm3, the genomic architecture of extant eukaryotes is only slightly advantageous at genomes sizes of 106‒107 base pairs and a larger host cell may have derived a greater advantage (lower cost) from harbouring ATP-producing symbionts. This suggests that eukaryotes first evolved without the need for mitochondria since these ranges hypothetically encompass the last eukaryotic common ancestor and its relatives. Our analyses also show that larger and faster-dividing prokaryotes would have a shortage of respiratory membrane area and divest more energy into DNA. Thus, we argue that although mitochondria may not have been required by the first eukaryotes, eukaryote diversification was ultimately dependent on mitochondria. Analysing the energetic constraints on prokaryotic cell size, the energetic implications of eukaryotic genome architecture, and the presence of endosymbionts, the authors suggest that mitochondria were not required for the initial origin of eukaryotes, but did facilitate their subsequent diversification and expansion.},
  archive      = {J_NATEE},
  author       = {Schavemaker, Paul E. and Muñoz-Gómez, Sergio A.},
  doi          = {10.1038/s41559-022-01833-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1307-1317},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The role of mitochondrial energetics in the origin and diversification of eukaryotes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Increasing climatic decoupling of bird abundances and
distributions. <em>NATEE</em>, <em>6</em>(9), 1299–1306. (<a
href="https://doi.org/10.1038/s41559-022-01814-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Species abundances and distributions are changing in response to changing climate and other anthropogenic drivers but how this translates into how well species can match their optimal climate conditions as they change is not well understood. Using a continental-scale 30-year time series, we quantified temporal trends in climate matching of North American bird species and tested whether geographical variation in rates of climate and land use change and/or species traits could underlie variation in trends among species. Overall, we found that species abundances and distributions are becoming more decoupled from climate as it changes through time. Species differences in climate matching trends were related to their ecological traits, particularly habitat specialization, but not to average rates of climate and land use change within the species’ ranges. Climatic decoupling through time was particularly prominent for birds that were declining in abundance and occupancy, including threatened species. While we could not discern whether climate decoupling causes or is caused by the negative population trends, higher climatic decoupling in declining species could lead to a feedback as birds experience increasing exposure to suboptimal climatic conditions. Using a 30-year dataset of North American bird species, the authors show that species’ abundances and distributions have become more decoupled from climate over time and that this is associated with ecological traits; the effect is particularly strong in threatened species.},
  archive      = {J_NATEE},
  author       = {Viana, Duarte S. and Chase, Jonathan M.},
  doi          = {10.1038/s41559-022-01814-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1299-1306},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increasing climatic decoupling of bird abundances and distributions},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evolutionary history of grazing and resources determine
herbivore exclusion effects on plant diversity. <em>NATEE</em>,
<em>6</em>(9), 1290–1298. (<a
href="https://doi.org/10.1038/s41559-022-01809-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ecological models predict that the effects of mammalian herbivore exclusion on plant diversity depend on resource availability and plant exposure to ungulate grazing over evolutionary time. Using an experiment replicated in 57 grasslands on six continents, with contrasting evolutionary history of grazing, we tested how resources (mean annual precipitation and soil nutrients) determine herbivore exclusion effects on plant diversity, richness and evenness. Here we show that at sites with a long history of ungulate grazing, herbivore exclusion reduced plant diversity by reducing both richness and evenness and the responses of richness and diversity to herbivore exclusion decreased with mean annual precipitation. At sites with a short history of grazing, the effects of herbivore exclusion were not related to precipitation but differed for native and exotic plant richness. Thus, plant species’ evolutionary history of grazing continues to shape the response of the world’s grasslands to changing mammalian herbivory. A NutNet experiment in 57 grasslands across six continents shows that when herbivores are excluded from grasslands with a long coevolutionary history of grazing plant diversity is reduced, while in grasslands without a long grazing history the evolutionary history of the plant species regulates the response of plant diversity.},
  archive      = {J_NATEE},
  author       = {Price, Jodi N. and Sitters, Judith and Ohlert, Timothy and Tognetti, Pedro M. and Brown, Cynthia S. and Seabloom, Eric W. and Borer, Elizabeth T. and Prober, Suzanne M. and Bakker, Elisabeth S. and MacDougall, Andrew S. and Yahdjian, Laura and Gruner, Daniel S. and Olde Venterink, Harry and Barrio, Isabel C. and Graff, Pamela and Bagchi, Sumanta and Arnillas, Carlos Alberto and Bakker, Jonathan D. and Blumenthal, Dana M. and Boughton, Elizabeth H. and Brudvig, Lars A. and Bugalho, Miguel N. and Cadotte, Marc W. and Caldeira, Maria C. and Dickman, Chris R. and Donohue, Ian and Grégory, Sonnier and Hautier, Yann and Jónsdóttir, Ingibjörg S. and Lannes, Luciola S. and McCulley, Rebecca L. and Moore, Joslin L. and Power, Sally A. and Risch, Anita C. and Schütz, Martin and Standish, Rachel and Stevens, Carly J. and Veen, G. F. and Virtanen, Risto and Wardle, Glenda M.},
  doi          = {10.1038/s41559-022-01809-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1290-1298},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolutionary history of grazing and resources determine herbivore exclusion effects on plant diversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Human pressure drives biodiversity–multifunctionality
relationships in large neotropical wetlands. <em>NATEE</em>,
<em>6</em>(9), 1279–1289. (<a
href="https://doi.org/10.1038/s41559-022-01827-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many studies have shown that biodiversity regulates multiple ecological functions that are needed to maintain the productivity of a variety of ecosystem types. What is unknown is how human activities may alter the ‘multifunctionality’ of ecosystems through both direct impacts on ecosystems and indirect effects mediated by the loss of multifaceted biodiversity. Using an extensive database of 72 lakes spanning four large Neotropical wetlands in Brazil, we demonstrate that species richness and functional diversity across multiple larger (fish and macrophytes) and smaller (microcrustaceans, rotifers, protists and phytoplankton) groups of aquatic organisms are positively associated with ecosystem multifunctionality. Whereas the positive association between smaller organisms and multifunctionality broke down with increasing human pressure, this positive relationship was maintained for larger organisms despite the increase in human pressure. Human pressure impacted multifunctionality both directly and indirectly through reducing species richness and functional diversity of multiple organismal groups. These findings provide further empirical evidence about the importance of aquatic biodiversity for maintaining wetland multifunctionality. Despite the key role of biodiversity, human pressure reduces the diversity of multiple groups of aquatic organisms, eroding their positive impacts on a suite of ecological functions that sustain wetlands. Analysis of the species richness and functional diversity among species across 72 lakes finds that both variables are positively associated with ecosystem multifunctionality, but that—for smaller organisms only—these positive relationships break down with increasing human pressure.},
  archive      = {J_NATEE},
  author       = {Moi, Dieison A. and Lansac-Tôha, Fernando M. and Romero, Gustavo Q. and Sobral-Souza, Thadeu and Cardinale, Bradley J. and Kratina, Pavel and Perkins, Daniel M. and Teixeira de Mello, Franco and Jeppesen, Erik and Heino, Jani and Lansac-Tôha, Fábio A. and Velho, Luiz F. M. and Mormul, Roger P.},
  doi          = {10.1038/s41559-022-01827-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1279-1289},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Human pressure drives biodiversity–multifunctionality relationships in large neotropical wetlands},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhanced leaf turnover and nitrogen recycling sustain CO2
fertilization effect on tree-ring growth. <em>NATEE</em>, <em>6</em>(9),
1271–1278. (<a
href="https://doi.org/10.1038/s41559-022-01811-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whether increased photosynthates under elevated atmospheric CO2 could translate into sustained biomass accumulation in forest trees remains uncertain. Here we demonstrate how tree radial growth is closely linked to litterfall dynamics, which enhances nitrogen recycling to support a sustained effect of CO2 fertilization on tree-ring growth. Our ten-year observations in two alpine treeline forests indicated that annual (or seasonal) stem radial increments generally had a positive relationship with the previous year’s (or season’s) litterfall and its associated nitrogen return and resorption. Annual tree-ring width, annual litterfall and annual nitrogen return and resorption all showed an increasing trend during 2007–2017, and most of the variations were explained by elevated atmospheric CO2 rather than climate change. Similar patterns were found in the longer time series of tree-ring width index from 1986–2017. The regional representativeness of our observed patterns was confirmed by the literature data of six other tree species at 11 treeline sites over the Tibetan Plateau. Enhanced nitrogen recycling through increased litterfall under elevated atmospheric CO2 supports a general increasing trend of tree-ring growth in recent decades, especially in cold and nitrogen-poor environments. A ten-year dataset from the Tibetan Plateau shows a general increase in tree-ring growth that is largely explained by enhanced nitrogen recycling through increased litterfall under elevated atmospheric CO2.},
  archive      = {J_NATEE},
  author       = {Guo, Ying and Zhang, Lin and Yang, Liu and Shen, Wei and Pan, Yude and Wright, Ian J. and Luo, Yiqi and Luo, Tianxiang},
  doi          = {10.1038/s41559-022-01811-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1271-1278},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Enhanced leaf turnover and nitrogen recycling sustain CO2 fertilization effect on tree-ring growth},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A global horizon scan of issues impacting marine and coastal
biodiversity conservation. <em>NATEE</em>, <em>6</em>(9), 1262–1270. (<a
href="https://doi.org/10.1038/s41559-022-01812-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biodiversity of marine and coastal habitats is experiencing unprecedented change. While there are well-known drivers of these changes, such as overexploitation, climate change and pollution, there are also relatively unknown emerging issues that are poorly understood or recognized that have potentially positive or negative impacts on marine and coastal ecosystems. In this inaugural Marine and Coastal Horizon Scan, we brought together 30 scientists, policymakers and practitioners with transdisciplinary expertise in marine and coastal systems to identify new issues that are likely to have a significant impact on the functioning and conservation of marine and coastal biodiversity over the next 5–10 years. Based on a modified Delphi voting process, the final 15 issues presented were distilled from a list of 75 submitted by participants at the start of the process. These issues are grouped into three categories: ecosystem impacts, for example the impact of wildfires and the effect of poleward migration on equatorial biodiversity; resource exploitation, including an increase in the trade of fish swim bladders and increased exploitation of marine collagens; and new technologies, such as soft robotics and new biodegradable products. Our early identification of these issues and their potential impacts on marine and coastal biodiversity will support scientists, conservationists, resource managers and policymakers to address the challenges facing marine ecosystems. A panel of scientists, policymakers and practitioners have used an iterative voting process to collate a list of 15 priority emerging issues likely to affect marine and coastal biodiversity over the next 5–10 years.},
  archive      = {J_NATEE},
  author       = {Herbert-Read, James E. and Thornton, Ann and Amon, Diva J. and Birchenough, Silvana N. R. and Côté, Isabelle M. and Dias, Maria P. and Godley, Brendan J. and Keith, Sally A. and McKinley, Emma and Peck, Lloyd S. and Calado, Ricardo and Defeo, Omar and Degraer, Steven and Johnston, Emma L. and Kaartokallio, Hermanni and Macreadie, Peter I. and Metaxas, Anna and Muthumbi, Agnes W. N. and Obura, David O. and Paterson, David M. and Piola, Alberto R. and Richardson, Anthony J. and Schloss, Irene R. and Snelgrove, Paul V. R. and Stewart, Bryce D. and Thompson, Paul M. and Watson, Gordon J. and Worthington, Thomas A. and Yasuhara, Moriaki and Sutherland, William J.},
  doi          = {10.1038/s41559-022-01812-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1262-1270},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A global horizon scan of issues impacting marine and coastal biodiversity conservation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Phenotypic plasticity promotes species coexistence.
<em>NATEE</em>, <em>6</em>(9), 1256–1261. (<a
href="https://doi.org/10.1038/s41559-022-01826-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ecological explanations for species coexistence assume that species’ traits, and therefore the differences between species, are fixed on short timescales. However, species’ traits are not fixed, but can instead change rapidly as a consequence of phenotypic plasticity. Here we use a combined experimental–theoretical approach to demonstrate that plasticity in response to interspecific competition between two aquatic plants allows for species coexistence where competitive exclusion is otherwise predicted to occur. Our results show that rapid trait changes in response to a shift in the competitive environment can promote coexistence in a way that is not captured by common measures of niche differentiation. The authors use a theoretical model along with competition experiments between two aquatic plant species to show that phenotypic plasticity affects the outcome of competition.},
  archive      = {J_NATEE},
  author       = {Hess, Cyrill and Levine, Jonathan M. and Turcotte, Martin M. and Hart, Simon P.},
  doi          = {10.1038/s41559-022-01826-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1256-1261},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Phenotypic plasticity promotes species coexistence},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Closing the energetics gap. <em>NATEE</em>, <em>6</em>(9),
1254–1255. (<a
href="https://doi.org/10.1038/s41559-022-01839-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A modelling study suggests that the proposed energetic barrier between prokaryotes and eukaryotes may not be relevant to the complexity gap between the two domains. The energetic advantage of early mitochondria was probably small, and eukaryotes likely emerged without the help of an endosymbiont.},
  archive      = {J_NATEE},
  author       = {Zachar, István},
  doi          = {10.1038/s41559-022-01839-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1254-1255},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Closing the energetics gap},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Struggling to keep pace. <em>NATEE</em>, <em>6</em>(9),
1252–1253. (<a
href="https://doi.org/10.1038/s41559-022-01825-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Distribution data from the North American Breeding Bird Survey shows an increasing mismatch between species distribution and climate.},
  archive      = {J_NATEE},
  author       = {Eyres, Alison},
  doi          = {10.1038/s41559-022-01825-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1252-1253},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Struggling to keep pace},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Humans pressure wetland multifunctionality. <em>NATEE</em>,
<em>6</em>(9), 1250–1251. (<a
href="https://doi.org/10.1038/s41559-022-01842-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A large dataset of aquatic biodiversity across multiple trophic levels from several wetlands in Brazil reveals that biodiversity–multifunctionality relationships break down with human pressures.},
  archive      = {J_NATEE},
  author       = {Pillay, Rajeev},
  doi          = {10.1038/s41559-022-01842-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1250-1251},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Humans pressure wetland multifunctionality},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Interpreting intertidal invasions. <em>NATEE</em>,
<em>6</em>(9), 1249. (<a
href="https://doi.org/10.1038/s41559-022-01852-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-022-01852-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1249},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Interpreting intertidal invasions},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022h). Tiny arms, big head. <em>NATEE</em>, <em>6</em>(9), 1248.
(<a href="https://doi.org/10.1038/s41559-022-01848-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-022-01848-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1248},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Tiny arms, big head},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Biodegradable sensors are ready to transform autonomous
ecological monitoring. <em>NATEE</em>, <em>6</em>(9), 1245–1247. (<a
href="https://doi.org/10.1038/s41559-022-01824-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent breakthroughs have led to the development of biodegradable sensors which, after collecting data, break down into byproducts that are harmless to their surroundings. Using these sensors to collect ecological data on vast scales and in fine resolution could transform our management and understanding of natural ecosystems.},
  archive      = {J_NATEE},
  author       = {Sethi, Sarab S. and Kovac, Mirko and Wiesemüller, Fabian and Miriyev, Aslan and Boutry, Clementine M.},
  doi          = {10.1038/s41559-022-01824-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1245-1247},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biodegradable sensors are ready to transform autonomous ecological monitoring},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Don’t dilute the term nature positive. <em>NATEE</em>,
<em>6</em>(9), 1243–1244. (<a
href="https://doi.org/10.1038/s41559-022-01845-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Positive is an aspirational term that is increasingly being used by businesses, governments and NGOs, but there is a danger that its meaning is being diluted away from measurable overall net gain in biodiversity towards merely any action that benefits nature, argues E.J. Milner-Gulland.},
  archive      = {J_NATEE},
  author       = {Milner-Gulland, E. J.},
  doi          = {10.1038/s41559-022-01845-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1243-1244},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Don’t dilute the term nature positive},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Eukaryotes were shaped by oxygen. <em>NATEE</em>,
<em>6</em>(9), 1242. (<a
href="https://doi.org/10.1038/s41559-022-01819-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Speijer, Dave},
  doi          = {10.1038/s41559-022-01819-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1242},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Eukaryotes were shaped by oxygen},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Wasted opportunities. <em>NATEE</em>, <em>6</em>(9), 1241.
(<a href="https://doi.org/10.1038/s41559-022-01881-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most ecological research does not reach its full potential, for reasons that range from poor design to publication bias to insufficient reporting. There are several straightforward steps that researchers, institutions, funders and publishers can take to cut the amount of wasted research.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01881-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1241},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Wasted opportunities},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: A new wave of marine fish invasions
through the panama and suez canals. <em>NATEE</em>, <em>6</em>(8), 1239.
(<a href="https://doi.org/10.1038/s41559-022-01849-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Castellanos-Galindo, Gustavo A. and Robertson, D. Ross and Sharpe, Diana M. T. and Torchin, Mark E.},
  doi          = {10.1038/s41559-022-01849-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1239},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: A new wave of marine fish invasions through the panama and suez canals},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ageing red deer alter their spatial behaviour and become
less social. <em>NATEE</em>, <em>6</em>(8), 1231–1238. (<a
href="https://doi.org/10.1038/s41559-022-01817-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Social relationships are important to many aspects of animals’ lives, and an individual’s connections may change over the course of their lifespan. Currently, it is unclear whether social connectedness declines within individuals as they age, and what the underlying mechanisms might be, so the role of age in structuring animal social systems remains unresolved, particularly in non-primates. Here we describe senescent declines in social connectedness using 46 years of data in a wild, individually monitored population of a long-lived mammal (European red deer, Cervus elaphus). Applying a series of spatial and social network analyses, we demonstrate that these declines occur because of within-individual changes in social behaviour, with correlated changes in spatial behaviour (smaller home ranges and movements to lower-density, lower-quality areas). These findings demonstrate that within-individual socio-spatial behavioural changes can lead older animals in fission–fusion societies to become less socially connected, shedding light on the ecological and evolutionary processes structuring wild animal populations. Using 46 years of individually monitored data for European red deer, the authors show that older individuals become less socially connected, with correlated changes to their spatial behaviour.},
  archive      = {J_NATEE},
  author       = {Albery, Gregory F. and Clutton-Brock, Tim H. and Morris, Alison and Morris, Sean and Pemberton, Josephine M. and Nussey, Daniel H. and Firth, Josh A.},
  doi          = {10.1038/s41559-022-01817-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1231-1238},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ageing red deer alter their spatial behaviour and become less social},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Vision and vocal communication guide three-dimensional
spatial coordination of zebra finches during wind-tunnel flights.
<em>NATEE</em>, <em>6</em>(8), 1221–1230. (<a
href="https://doi.org/10.1038/s41559-022-01800-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal collective motion is a natural phenomenon readily observable in various taxa. Although theoretical models can predict the macroscopic pattern of group movements based on the relative spatial position of group members, it is poorly understood how group members exchange directional information, which enables the spatial coordination between individuals during collective motion. To test if vocalizations emitted during flocking flight are used by birds to transmit directional information between group members, we recorded vocal behaviour, head orientation and spatial position of each individual in a small flock of zebra finches (Taeniopygia guttata) flying in a wind tunnel. We found that the finches can use both visual and acoustic cues for three-dimensional flock coordination. When visual information is insufficient, birds can increasingly exploit active vocal communication to avoid collisions with flock mates. Our study furthers the mechanistic understanding of collective motion in birds and highlights the impact interindividual vocal interactions can have on group performances in these animals. Zebra finches flying in a wind tunnel use both vocal and visual communication to orientate themselves within the flock, and are able to enhance their use of one form of communication over another depending on circumstance.},
  archive      = {J_NATEE},
  author       = {Arnold, Fabian and Staniszewski, Michael S. and Pelzl, Lisa and Ramenda, Claudia and Gahr, Manfred and Hoffmann, Susanne},
  doi          = {10.1038/s41559-022-01800-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1221-1230},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Vision and vocal communication guide three-dimensional spatial coordination of zebra finches during wind-tunnel flights},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prolonged morphological expansion of spiny-rayed fishes
following the end-cretaceous. <em>NATEE</em>, <em>6</em>(8), 1211–1220.
(<a href="https://doi.org/10.1038/s41559-022-01801-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spiny-rayed fishes (Acanthomorpha) dominate modern marine habitats and account for more than a quarter of all living vertebrate species. Previous time-calibrated phylogenies and patterns from the fossil record explain this dominance by correlating the origin of major acanthomorph lineages with the Cretaceous–Palaeogene mass extinction. Here we infer a time-calibrated phylogeny using ultraconserved elements that samples 91.4% of all acanthomorph families and investigate patterns of body shape disparity. Our results show that acanthomorph lineages steadily accumulated throughout the Cenozoic and underwent a significant expansion of among-clade morphological disparity several million years after the end-Cretaceous. These acanthomorph lineages radiated into and diversified within distinct regions of morphospace that characterize iconic lineages, including fast-swimming open-ocean predators, laterally compressed reef fishes, bottom-dwelling flatfishes, seahorses and pufferfishes. The evolutionary success of spiny-rayed fishes is the culmination of multiple species-rich and phenotypically disparate lineages independently diversifying across the globe under a wide range of ecological conditions. The authors construct a time-calibrated phylogeny spanning &amp;gt;90% of spiny-rayed fishes to explore patterns of body shape disparity within acanthomorphs. They find a trend of steady accumulation of lineages from the Cenozoic, with an increase in morphological disparity following the Cretaceous–Palaeogene event, facilitating the radiation of diverse morphotypes that characterize acanthomorphs’ widespread ecological success today.},
  archive      = {J_NATEE},
  author       = {Ghezelayagh, Ava and Harrington, Richard C. and Burress, Edward D. and Campbell, Matthew A. and Buckner, Janet C. and Chakrabarty, Prosanta and Glass, Jessica R. and McCraney, W. Tyler and Unmack, Peter J. and Thacker, Christine E. and Alfaro, Michael E. and Friedman, Sarah T. and Ludt, William B. and Cowman, Peter F. and Friedman, Matt and Price, Samantha A. and Dornburg, Alex and Faircloth, Brant C. and Wainwright, Peter C. and Near, Thomas J.},
  doi          = {10.1038/s41559-022-01801-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1211-1220},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Prolonged morphological expansion of spiny-rayed fishes following the end-cretaceous},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Phylogenomics reveals the origin of mammal lice out of
afrotheria. <em>NATEE</em>, <em>6</em>(8), 1205–1210. (<a
href="https://doi.org/10.1038/s41559-022-01803-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammals host a wide diversity of parasites. Lice, comprising more than 5,000 species, are one group of ectoparasites whose major lineages have a somewhat patchwork distribution across the major groups of mammals. Here we explored patterns in the diversification of mammalian lice by reconstructing a higher-level phylogeny of these lice, leveraging whole genome sequence reads to assemble single-copy orthologue genes across the genome. The evolutionary tree of lice indicated that three of the major lineages of placental mammal lice had a single common ancestor. Comparisons of this parasite phylogeny with that for their mammalian hosts indicated that the common ancestor of elephants, elephant shrews and hyraxes (that is, Afrotheria) was the ancestral host of this group of lice. Other groups of placental mammals obtained their lice via host-switching out of these Afrotherian ancestors. In addition, reconstructions of the ancestral host group (bird versus mammal) for all parasitic lice supported an avian ancestral host, indicating that the ancestor of Afrotheria acquired these parasites via host-switching from an ancient avian host. These results shed new light on the long-standing question of why the major groups of parasitic lice are not uniformly distributed across mammals and reveal the origins of mammalian lice. Mammals host a diversity of parasites including lice. Using cophylogenetics and phylogenetic comparative methods, the authors show that the main lineages of placental mammal lice had a single common ancestor and find that all parasitic lice had an avian ancestral host.},
  archive      = {J_NATEE},
  author       = {Johnson, Kevin P. and Matthee, Conrad and Doña, Jorge},
  doi          = {10.1038/s41559-022-01803-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1205-1210},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Phylogenomics reveals the origin of mammal lice out of afrotheria},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A single-cell transcriptomic atlas tracking the neural basis
of division of labour in an ant superorganism. <em>NATEE</em>,
<em>6</em>(8), 1191–1204. (<a
href="https://doi.org/10.1038/s41559-022-01784-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ant colonies with permanent division of labour between castes and highly distinct roles of the sexes have been conceptualized to be superorganisms, but the cellular and molecular mechanisms that mediate caste/sex-specific behavioural specialization have remained obscure. Here we characterized the brain cell repertoire of queens, gynes (virgin queens), workers and males of Monomorium pharaonis by obtaining 206,367 single-nucleus transcriptomes. In contrast to Drosophila, the mushroom body Kenyon cells are abundant in ants and display a high diversity with most subtypes being enriched in worker brains, the evolutionarily derived caste. Male brains are as specialized as worker brains but with opposite trends in cell composition with higher abundances of all optic lobe neuronal subtypes, while the composition of gyne and queen brains remained generalized, reminiscent of solitary ancestors. Role differentiation from virgin gynes to inseminated queens induces abundance changes in roughly 35% of cell types, indicating active neurogenesis and/or programmed cell death during this transition. We also identified insemination-induced cell changes probably associated with the longevity and fecundity of the reproductive caste, including increases of ensheathing glia and a population of dopamine-regulated Dh31-expressing neurons. We conclude that permanent caste differentiation and extreme sex-differentiation induced major changes in the neural circuitry of ants. Using single-cell transcriptomics, the authors generate a brain cell atlas for the pharaoh ant including individuals of different sexes and castes and show changes in cell composition underlying division of labour and reproductive specialization.},
  archive      = {J_NATEE},
  author       = {Li, Qiye and Wang, Mingyue and Zhang, Pei and Liu, Yang and Guo, Qunfei and Zhu, Yuanzhen and Wen, Tinggang and Dai, Xueqin and Zhang, Xiafang and Nagel, Manuel and Dethlefsen, Bjarke Hamberg and Xie, Nianxia and Zhao, Jie and Jiang, Wei and Han, Lei and Wu, Liang and Zhong, Wenjiang and Wang, Zhifeng and Wei, Xiaoyu and Dai, Wei and Liu, Longqi and Xu, Xun and Lu, Haorong and Yang, Huanming and Wang, Jian and Boomsma, Jacobus J. and Liu, Chuanyu and Zhang, Guojie and Liu, Weiwei},
  doi          = {10.1038/s41559-022-01784-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1191-1204},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A single-cell transcriptomic atlas tracking the neural basis of division of labour in an ant superorganism},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A single gene integrates sex and hormone regulators into
sexual attractiveness. <em>NATEE</em>, <em>6</em>(8), 1180–1190. (<a
href="https://doi.org/10.1038/s41559-022-01808-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sex differentiation and hormones are essential for the development of sexual signals in animals, and the regulation of sexual signals involves complex gene networks. However, it is unknown whether a core gene is able to connect the upstream regulators for controlling sexual signal outputs and behavioural consequences. Here, we identify a single gene that integrates both sex differentiation and hormone signalling with sexual attractiveness in an insect model. CYP4PC1 in the German cockroach, Blattella germanica, controls the rate-limiting step in producing female-specific contact sex pheromone (CSP) that stimulates male courtship. As revealed by behavioural, biochemical, molecular, genetic and bioinformatic approaches, in sexually mature females, CYP4PC1 expression and CSP production are coordinately induced by sex differentiation genes and juvenile hormone (JH) signalling. In adult males, direct inhibition of CYP4PC1 expression by doublesexM binding in gene promoter and lack of the gonadotropic hormone JH prevent CSP production, thus avoiding male–male attraction. By manipulating the upstream regulators, we show that wild-type males prefer to court cockroaches with higher CYP4PC1 expression and CSP production in a dose-dependent manner, regardless of their sex. These findings shed light on how sex-specific and high sexual attractiveness is conferred in insects. A multidisciplinary approach, including genetics and behavioural assays, identifies a single gene, CYP4PC1, which integrates both sex differentiation and hormone signalling with sexual attractiveness in the German cockroach.},
  archive      = {J_NATEE},
  author       = {Chen, Nan and Liu, Yong-Jun and Fan, Yong-Liang and Pei, Xiao-Jin and Yang, Yang and Liao, Ming-Tao and Zhong, Jiru and Li, Na and Liu, Tong-Xian and Wang, Guirong and Pan, Yufeng and Schal, Coby and Li, Sheng},
  doi          = {10.1038/s41559-022-01808-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1180-1190},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A single gene integrates sex and hormone regulators into sexual attractiveness},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Gene regulation in escherichia coli is commonly selected for
both high plasticity and low noise. <em>NATEE</em>, <em>6</em>(8),
1165–1179. (<a
href="https://doi.org/10.1038/s41559-022-01783-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacteria often respond to dynamically changing environments by regulating gene expression. Despite this regulation being critically important for growth and survival, little is known about how selection shapes gene regulation in natural populations. To better understand the role natural selection plays in shaping bacterial gene regulation, here we compare differences in the regulatory behaviour of naturally segregating promoter variants from Escherichia coli (which have been subject to natural selection) to randomly mutated promoter variants (which have never been exposed to natural selection). We quantify gene expression phenotypes (expression level, plasticity and noise) for hundreds of promoter variants across multiple environments and show that segregating promoter variants are enriched for mutations with minimal effects on expression level. In many promoters, we infer that there is strong selection to maintain high levels of plasticity, and direct selection to decrease or increase cell-to-cell variability in expression. Taken together, these results expand our knowledge of how gene regulation is affected by natural selection and highlight the power of comparing naturally segregating polymorphisms to de novo random mutations to quantify the action of selection. Comparing the regulatory behaviour of naturally segregating promoter variants to randomly mutated promoters, the authors demonstrate both stabilizing and directional selection that reduce variation in phenotype.},
  archive      = {J_NATEE},
  author       = {Vlková, Markéta and Silander, Olin K.},
  doi          = {10.1038/s41559-022-01783-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1165-1179},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Gene regulation in escherichia coli is commonly selected for both high plasticity and low noise},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Low protein expression enhances phenotypic evolvability by
intensifying selection on folding stability. <em>NATEE</em>,
<em>6</em>(8), 1155–1164. (<a
href="https://doi.org/10.1038/s41559-022-01797-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein abundance affects the evolution of protein genotypes, but we do not know how it affects the evolution of protein phenotypes. Here we investigate the role of protein abundance in the evolvability of green fluorescent protein (GFP) towards the novel phenotype of cyan fluorescence. We evolve GFP in E. coli through multiple cycles of mutation and selection and show that low GFP expression facilitates the evolution of cyan fluorescence. A computational model whose predictions we test experimentally helps explain why: lowly expressed proteins are under stronger selection for proper folding, which facilitates their evolvability on short evolutionary time scales. The reason is that high fluorescence can be achieved by either few proteins that fold well or by many proteins that fold less well. In other words, we observe a synergy between a protein’s scarcity and its stability. Because many proteins meet the essential requirements for this scarcity–stability synergy, it may be a widespread mechanism by which low expression helps proteins evolve new phenotypes and functions. Directed evolution shows that low expression of the green fluorescent protein facilitates the evolution of cyan fluorescence in E. coli, which can be explained by synergy between the protein’s scarcity and its stability.},
  archive      = {J_NATEE},
  author       = {Karve, Shraddha and Dasmeh, Pouria and Zheng, Jia and Wagner, Andreas},
  doi          = {10.1038/s41559-022-01797-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1155-1164},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Low protein expression enhances phenotypic evolvability by intensifying selection on folding stability},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Agricultural management and pesticide use reduce the
functioning of beneficial plant symbionts. <em>NATEE</em>,
<em>6</em>(8), 1145–1154. (<a
href="https://doi.org/10.1038/s41559-022-01799-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phosphorus (P) acquisition is key for plant growth. Arbuscular mycorrhizal fungi (AMF) help plants acquire P from soil. Understanding which factors drive AMF-supported nutrient uptake is essential to develop more sustainable agroecosystems. Here we collected soils from 150 cereal fields and 60 non-cropped grassland sites across a 3,000 km trans-European gradient. In a greenhouse experiment, we tested the ability of AMF in these soils to forage for the radioisotope 33P from a hyphal compartment. AMF communities in grassland soils were much more efficient in acquiring 33P and transferred 64% more 33P to plants compared with AMF in cropland soils. Fungicide application best explained hyphal 33P transfer in cropland soils. The use of fungicides and subsequent decline in AMF richness in croplands reduced 33P uptake by 43%. Our results suggest that land-use intensity and fungicide use are major deterrents to the functioning and natural nutrient uptake capacity of AMF in agroecosystems. Combining field data and greenhouse experiments, the authors show how agricultural management practices like fungicide applications can affect the degree to which arbuscular mycorrhizal fungi in the soil provision phosphorus to plants.},
  archive      = {J_NATEE},
  author       = {Edlinger, Anna and Garland, Gina and Hartman, Kyle and Banerjee, Samiran and Degrune, Florine and García-Palacios, Pablo and Hallin, Sara and Valzano-Held, Alain and Herzog, Chantal and Jansa, Jan and Kost, Elena and Maestre, Fernando T. and Pescador, David Sánchez and Philippot, Laurent and Rillig, Matthias C. and Romdhane, Sana and Saghaï, Aurélien and Spor, Ayme and Frossard, Emmanuel and van der Heijden, Marcel G. A.},
  doi          = {10.1038/s41559-022-01799-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1145-1154},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Agricultural management and pesticide use reduce the functioning of beneficial plant symbionts},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The coral pathogen vibrio coralliilyticus kills
non-pathogenic holobiont competitors by triggering prophage induction.
<em>NATEE</em>, <em>6</em>(8), 1132–1144. (<a
href="https://doi.org/10.1038/s41559-022-01795-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The coral reef microbiome is central to reef health and resilience. Competitive interactions between opportunistic coral pathogens and other commensal microbes affect the health of coral. Despite great advances over the years in sequencing-based microbial profiling of healthy and diseased coral, the molecular mechanism underlying colonization competition has been much less explored. In this study, by examining the culturable bacteria inhabiting the gastric cavity of healthy Galaxea fascicularis, a scleractinian coral, we found that temperate phages played a major role in mediating colonization competition in the coral microbiota. Specifically, the non-toxigenic Vibrio sp. inhabiting the healthy coral had a much higher colonization capacity than the coral pathogen Vibrio coralliilyticus, yet this advantage was diminished by the latter killing the former. Pathogen-encoded LodAB, which produces hydrogen peroxide, triggers the lytic cycle of prophage in the non-toxicogenic Vibrio sp. Importantly, V. coralliilyticus could outcompete other coral symbiotic bacteria (for example, Endozoicomonas sp.) through LodAB-dependent prophage induction. Overall, we reveal that LodAB can be used by pathogens as an important weapon to gain a competitive advantage over lysogenic competitors when colonizing corals. Competition among species in the coral microbiome has important outcomes in terms of coral health, but little is known about the mechanisms allowing pathogens to gain a competitive advantage. Here the authors show that the pathogen Vibrio coralliilyticus outcompetes commensal bacteria by inducing prophages.},
  archive      = {J_NATEE},
  author       = {Wang, Weiquan and Tang, Kaihao and Wang, Pengxia and Zeng, Zhenshun and Xu, Tao and Zhan, Waner and Liu, Tianlang and Wang, Yan and Wang, Xiaoxue},
  doi          = {10.1038/s41559-022-01795-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1132-1144},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The coral pathogen vibrio coralliilyticus kills non-pathogenic holobiont competitors by triggering prophage induction},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Increasing calcium scarcity along afrotropical forest
succession. <em>NATEE</em>, <em>6</em>(8), 1122–1131. (<a
href="https://doi.org/10.1038/s41559-022-01810-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Secondary forests constitute an increasingly important component of tropical forests worldwide. Although cycling of essential nutrients affects recovery trajectories of secondary forests, the effect of nutrient limitation on forest regrowth is poorly constrained. Here we use three lines of evidence from secondary forest succession sequences in central Africa to identify potential nutrient limitation in regrowing forests. First, we show that atmospheric phosphorus supply exceeds demand along forest succession, whereas forests rely on soil stocks to meet their base cation demands. Second, soil nutrient metrics indicate that available phosphorus increases along the succession, whereas available cations decrease. Finally, fine root, foliar and litter stoichiometry show that tissue calcium concentrations decline relative to those of nitrogen and phosphorus during succession. Taken together, these observations suggest that calcium becomes an increasingly scarce resource in central African forests during secondary succession. Furthermore, ecosystem calcium storage shifts from soil to woody biomass over succession, making it a vulnerable nutrient in the wake of land-use change scenarios that involve woody biomass export. Our results thus call for a broadened focus on elements other than nitrogen and phosphorus regarding tropical forest biogeochemical cycles and identify calcium as a scarce and potentially limiting nutrient in an increasingly disturbed and dynamic tropical forest landscape. Biogeochemical analysis of a chronosequence of secondary forest succession in lowland Central Africa suggests that calcium becomes an increasingly scarce and potentially limiting resource with stand age and ecosystem calcium storage shifts from soil to woody biomass.},
  archive      = {J_NATEE},
  author       = {Bauters, Marijn and Janssens, Ivan A. and Wasner, Daniel and Doetterl, Sebastian and Vermeir, Pieter and Griepentrog, Marco and Drake, Travis W. and Six, Johan and Barthel, Matti and Baumgartner, Simon and Van Oost, Kristof and Makelele, Isaac A. and Ewango, Corneille and Verheyen, Kris and Boeckx, Pascal},
  doi          = {10.1038/s41559-022-01810-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1122-1131},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increasing calcium scarcity along afrotropical forest succession},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Meta-analysis shows that plant mixtures increase soil
phosphorus availability and plant productivity in diverse ecosystems.
<em>NATEE</em>, <em>6</em>(8), 1112–1121. (<a
href="https://doi.org/10.1038/s41559-022-01794-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soil phosphorus (P) availability is critical to plant productivity in many terrestrial ecosystems. How soil P availability responds to changes in plant diversity remains uncertain, despite the global crisis of rapid biodiversity loss. Our meta-analysis based on 180 studies across various ecosystems (croplands, grasslands, forests and pot experiments) shows that, on average, soil total P, phosphatase activity and available P are 6.8%, 8.5% and 4.6%, respectively, higher in species mixtures than in monocultures. The mixture effect on phosphatase activity becomes more positive with increasing species and functional group richness, with more pronounced increases in the rhizosphere than in the bulk soil. The mixture effects on soil-available P in the bulk soil do not change, but with increasing species or functional group richness these effects in the rhizosphere soil shift from positive to negative. Nonetheless, enhanced soil phosphatase activity stimulated available P in diverse species mixtures, offsetting increased plant uptake effects that decrease soil-available P. Moreover, the enhancement effects of species richness on soil phosphatase activity are positively associated with increased plant productivity. Our findings highlight that preserving plant diversity could increase soil phosphatase activity and P availability, which sustain the current and future productivity of terrestrial ecosystems. In a meta-analysis of studies comparing monocultures to plant species mixtures, the authors show that mixtures increase three different variables related to the phosphorus cycle: soil total phosphorus, phosphatase activity and available phosphorus.},
  archive      = {J_NATEE},
  author       = {Chen, Xinli and Chen, Han Y. H. and Chang, Scott X.},
  doi          = {10.1038/s41559-022-01794-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1112-1121},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Meta-analysis shows that plant mixtures increase soil phosphorus availability and plant productivity in diverse ecosystems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chaos is not rare in natural ecosystems. <em>NATEE</em>,
<em>6</em>(8), 1105–1111. (<a
href="https://doi.org/10.1038/s41559-022-01787-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chaotic dynamics are thought to be rare in natural populations but this may be due to methodological and data limitations, rather than the inherent stability of ecosystems. Following extensive simulation testing, we applied multiple chaos detection methods to a global database of 172 population time series and found evidence for chaos in &amp;gt;30%. In contrast, fitting traditional one-dimensional models identified &amp;lt;10% as chaotic. Chaos was most prevalent among plankton and insects and least among birds and mammals. Lyapunov exponents declined with generation time and scaled as the −1/6 power of body mass among chaotic populations. These results demonstrate that chaos is not rare in natural populations, indicating that there may be intrinsic limits to ecological forecasting and cautioning against the use of steady-state approaches to conservation and management. Previous meta-analyses have found that few ecological time series are actually chaotic; here, the authors use recently developed chaos detection methods and a larger number of time series to show that 30% of studied time series are characterized as chaotic.},
  archive      = {J_NATEE},
  author       = {Rogers, Tanya L. and Johnson, Bethany J. and Munch, Stephan B.},
  doi          = {10.1038/s41559-022-01787-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1105-1111},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Chaos is not rare in natural ecosystems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A crown-group cnidarian from the ediacaran of charnwood
forest, UK. <em>NATEE</em>, <em>6</em>(8), 1095–1104. (<a
href="https://doi.org/10.1038/s41559-022-01807-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cnidarians are a disparate and ancient phylum, encompassing corals and jellyfish, and occupy both the pelagic and benthic realms. They have a rich fossil record from the Phanerozoic eon lending insight into the early history of the group but, although cnidarians diverged from other animals in the Precambrian period, their record from the Ediacaran period (635–542 million years ago) is controversial. Here, we describe a new fossil cnidarian—Auroralumina attenboroughii gen. et sp. nov.—from the Ediacaran of Charnwood Forest (557–562 million years ago) that shows two bifurcating polyps enclosed in a rigid, polyhedral, organic skeleton with evidence of simple, densely packed tentacles. Auroralumina displays a suite of characters allying it to early medusozoans but shows others more typical of Anthozoa. Phylogenetic analyses recover Auroralumina as a stem-group medusozoan and, therefore, the oldest crown-group cnidarian. Auroralumina demonstrates both the establishment of the crown group of an animal phylum and the fixation of its body plan tens of millions of years before the Cambrian diversification of animal life. A new fossil cnidarian, Auroralumina attenboroughi, from the Ediacaran of Charnwood Forest, UK, described as showing mosaic anthozoan and medusozoan characters, is the oldest yet-known crown-group cnidarian.},
  archive      = {J_NATEE},
  author       = {Dunn, F. S. and Kenchington, C. G. and Parry, L. A. and Clark, J. W. and Kendall, R. S. and Wilby, P. R.},
  doi          = {10.1038/s41559-022-01807-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1095-1104},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A crown-group cnidarian from the ediacaran of charnwood forest, UK},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reply to: Modelling hominin evolution requires accurate
hominin data. <em>NATEE</em>, <em>6</em>(8), 1092–1094. (<a
href="https://doi.org/10.1038/s41559-022-01792-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Püschel, Hans P. and Bertrand, Ornella C. and Reilly, Joseph E. O’ and Bobe, René and Püschel, Thomas A.},
  doi          = {10.1038/s41559-022-01792-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1092-1094},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Modelling hominin evolution requires accurate hominin data},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Modelling hominin evolution requires accurate hominin data.
<em>NATEE</em>, <em>6</em>(8), 1090–1091. (<a
href="https://doi.org/10.1038/s41559-022-01791-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Mongle, Carrie S. and Pugh, Kelsey D. and Strait, David S. and Grine, Frederick E.},
  doi          = {10.1038/s41559-022-01791-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1090-1091},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Modelling hominin evolution requires accurate hominin data},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Population genetics of clonally transmissible cancers.
<em>NATEE</em>, <em>6</em>(8), 1077–1089. (<a
href="https://doi.org/10.1038/s41559-022-01790-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Populations of cancer cells are subject to the same core evolutionary processes as asexually reproducing, unicellular organisms. Transmissible cancers are particularly striking examples of these processes. These unusual cancers are clonal lineages that can spread through populations via physical transfer of living cancer cells from one host individual to another, and they have achieved long-term success in the colonization of at least eight different host species. Population genetic theory provides a useful framework for understanding the shift from a multicellular sexual animal into a unicellular asexual clone and its long-term effects on the genomes of these cancers. In this Review, we consider recent findings from transmissible cancer research with the goals of developing an evolutionarily informed perspective on transmissible cancers, examining possible implications for their long-term fate and identifying areas for future research on these exceptional lineages. Transmissible cancers are governed by the same evolutionary processes as asexually reproducing, unicellular organisms. This Review discusses population genetics processes that determine the evolution of clonally transmissible cancers.},
  archive      = {J_NATEE},
  author       = {Ní Leathlobhair, Máire and Lenski, Richard E.},
  doi          = {10.1038/s41559-022-01790-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1077-1089},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Population genetics of clonally transmissible cancers},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Dryland mechanisms could widely control ecosystem
functioning in a drier and warmer world. <em>NATEE</em>, <em>6</em>(8),
1064–1076. (<a
href="https://doi.org/10.1038/s41559-022-01779-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Responses of terrestrial ecosystems to climate change have been explored in many regions worldwide. While continued drying and warming may alter process rates and deteriorate the state and performance of ecosystems, it could also lead to more fundamental changes in the mechanisms governing ecosystem functioning. Here we argue that climate change will induce unprecedented shifts in these mechanisms in historically wetter climatic zones, towards mechanisms currently prevalent in dry regions, which we refer to as ‘dryland mechanisms’. We discuss 12 dryland mechanisms affecting multiple processes of ecosystem functioning, including vegetation development, water flow, energy budget, carbon and nutrient cycling, plant production and organic matter decomposition. We then examine mostly rare examples of the operation of these mechanisms in non-dryland regions where they have been considered irrelevant at present. Current and future climate trends could force microclimatic conditions across thresholds and lead to the emergence of dryland mechanisms and their increasing control over ecosystem functioning in many biomes on Earth. In drylands, there are unique mechanisms that influence multiple ecosystem processes. In this Perspective, the authors identify these dryland mechanisms and show that they could become more important in non-dryland regions or areas that will become drier in the future.},
  archive      = {J_NATEE},
  author       = {Grünzweig, José M. and De Boeck, Hans J. and Rey, Ana and Santos, Maria J. and Adam, Ori and Bahn, Michael and Belnap, Jayne and Deckmyn, Gaby and Dekker, Stefan C. and Flores, Omar and Gliksman, Daniel and Helman, David and Hultine, Kevin R. and Liu, Lingli and Meron, Ehud and Michael, Yaron and Sheffer, Efrat and Throop, Heather L. and Tzuk, Omer and Yakir, Dan},
  doi          = {10.1038/s41559-022-01779-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1064-1076},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dryland mechanisms could widely control ecosystem functioning in a drier and warmer world},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Social senescence in red deer. <em>NATEE</em>,
<em>6</em>(8), 1062–1063. (<a
href="https://doi.org/10.1038/s41559-022-01766-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Longitudinal data spanning 43 years from a wild ungulate population reveal changes in social connectedness as individuals age, and suggest that these changes may in part be driven by changes in spatial behaviour.},
  archive      = {J_NATEE},
  author       = {Siracusa, Erin R.},
  doi          = {10.1038/s41559-022-01766-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1062-1063},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Social senescence in red deer},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Complementary brains. <em>NATEE</em>, <em>6</em>(8),
1060–1061. (<a
href="https://doi.org/10.1038/s41559-022-01805-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pharaoh ants live in highly organized colonies with elaborate social structure. An atlas of the brain cells of the different sexes and social groups of this ant reveals cell compositions tailored to the tasks performed by each group.},
  archive      = {J_NATEE},
  author       = {Sieriebriennikov, Bogdan},
  doi          = {10.1038/s41559-022-01805-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1060-1061},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Complementary brains},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The sex of scents. <em>NATEE</em>, <em>6</em>(8), 1058–1059.
(<a href="https://doi.org/10.1038/s41559-022-01818-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mating in insects relies on pheromone production in just one of the sexes. A multidisciplinary study on the German cockroach identifies a gene that connects sex differentiation factors with the production of sexual pheromones in females only.},
  archive      = {J_NATEE},
  author       = {Belles, Xavier},
  doi          = {10.1038/s41559-022-01818-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1058-1059},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The sex of scents},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Beyond nitrogen and phosphorus. <em>NATEE</em>,
<em>6</em>(8), 1056–1057. (<a
href="https://doi.org/10.1038/s41559-022-01788-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An experiment in secondary forests in the Democratic Republic of the Congo finds that calcium, an overlooked soil nutrient, is scarcer than phosphorus, and represents a potentially greater limitation on tropical forest growth.},
  archive      = {J_NATEE},
  author       = {Vallicrosa, Helena},
  doi          = {10.1038/s41559-022-01788-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1056-1057},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Beyond nitrogen and phosphorus},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Microbes of tiny invertebrates. <em>NATEE</em>,
<em>6</em>(8), 1055. (<a
href="https://doi.org/10.1038/s41559-022-01821-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-022-01821-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1055},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Microbes of tiny invertebrates},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An inordinate fondness for wasps. <em>NATEE</em>,
<em>6</em>(8), 1054. (<a
href="https://doi.org/10.1038/s41559-022-01822-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Gadagkar, Raghavendra},
  doi          = {10.1038/s41559-022-01822-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1054},
  shortjournal = {Nat. Ecol. Evol},
  title        = {An inordinate fondness for wasps},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Paul mellars (1939–2022). <em>NATEE</em>, <em>6</em>(8),
1052–1053. (<a
href="https://doi.org/10.1038/s41559-022-01804-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Archaeologist who emphasized the importance of chronology in understanding Palaeolithic Europe, and laid the framework for the archaeology of modern human origins.},
  archive      = {J_NATEE},
  author       = {Foley, Robert},
  doi          = {10.1038/s41559-022-01804-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1052-1053},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Paul mellars (1939–2022)},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Only a human rights-based approach will address biodiversity
loss. <em>NATEE</em>, <em>6</em>(8), 1050–1051. (<a
href="https://doi.org/10.1038/s41559-022-01796-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Josefa Cariño Tauli is an Ibaloi-Kankanaey Igorot Indigenous person from the Cordillera Region in the Northern Philippines. She is currently the policy co-coordinator for the Global Youth Biodiversity Network, the international coordination platform for youth participation in the UN Convention on Biological Diversity (CBD).},
  archive      = {J_NATEE},
  author       = {Tauli, Josefa Cariño},
  doi          = {10.1038/s41559-022-01796-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1050-1051},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Only a human rights-based approach will address biodiversity loss},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Policy responses to the ukraine crisis threaten european
biodiversity. <em>NATEE</em>, <em>6</em>(8), 1048–1049. (<a
href="https://doi.org/10.1038/s41559-022-01786-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Strange, Niels and Geldmann, Jonas and Burgess, Neil D. and Bull, Joseph William},
  doi          = {10.1038/s41559-022-01786-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1048-1049},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Policy responses to the ukraine crisis threaten european biodiversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Broader values for better biodiversity outcomes.
<em>NATEE</em>, <em>6</em>(8), 1047. (<a
href="https://doi.org/10.1038/s41559-022-01853-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recently released IPBES Values Assessment explores the myriad ways in which nature can and should be valued. Policymakers now need to diversify their view of the relationship between nature and people.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01853-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1047},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Broader values for better biodiversity outcomes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Publisher correction: Talk the talk and walk the walk.
<em>NATEE</em>, <em>6</em>(7), 1046. (<a
href="https://doi.org/10.1038/s41559-022-01781-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach and Tommy, Kimberleigh},
  doi          = {10.1038/s41559-022-01781-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1046},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Talk the talk and walk the walk},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Detecting signatures of selection on gene expression.
<em>NATEE</em>, <em>6</em>(7), 1035–1045. (<a
href="https://doi.org/10.1038/s41559-022-01761-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A substantial amount of phenotypic diversity results from changes in gene expression levels and patterns. Understanding how the transcriptome evolves is therefore a key priority in identifying mechanisms of adaptive change. However, in contrast to powerful models of sequence evolution, we lack a consensus model of gene expression evolution. Furthermore, recent work has shown that many of the comparative approaches used to study gene expression are subject to biases that can lead to false signatures of selection. Here we first outline the main approaches for describing expression evolution and their inherent biases. Next, we bridge the gap between the fields of phylogenetic comparative methods and transcriptomics to reinforce the main pitfalls of inferring selection on expression patterns and use simulation studies to show that shifts in tissue composition can heavily bias inferences of selection. We close by highlighting the multi-dimensional nature of transcriptional variation and identifying major unanswered questions in disentangling how selection acts on the transcriptome. This paper examines the main approaches for studying gene expression evolution, tests their inherent biases and discusses open questions about the evolution of the transcriptome.},
  archive      = {J_NATEE},
  author       = {Price, Peter D. and Palmer Droguett, Daniela H. and Taylor, Jessica A. and Kim, Dong Won and Place, Elsie S. and Rogers, Thea F. and Mank, Judith E. and Cooney, Christopher R. and Wright, Alison E.},
  doi          = {10.1038/s41559-022-01761-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1035-1045},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Detecting signatures of selection on gene expression},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ancient genomes from the last three millennia support
multiple human dispersals into wallacea. <em>NATEE</em>, <em>6</em>(7),
1024–1034. (<a
href="https://doi.org/10.1038/s41559-022-01775-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Previous research indicates that human genetic diversity in Wallacea—islands in present-day Eastern Indonesia and Timor-Leste that were never part of the Sunda or Sahul continental shelves—has been shaped by complex interactions between migrating Austronesian farmers and indigenous hunter–gatherer communities. Yet, inferences based on present-day groups proved insufficient to disentangle this region’s demographic movements and admixture timings. Here, we investigate the spatio-temporal patterns of variation in Wallacea based on genome-wide data from 16 ancient individuals (2600–250 years BP) from the North Moluccas, Sulawesi and East Nusa Tenggara. While ancestry in the northern islands primarily reflects contact between Austronesian- and Papuan-related groups, ancestry in the southern islands reveals additional contributions from Mainland Southeast Asia that seem to predate the arrival of Austronesians. Admixture time estimates further support multiple and/or continuous admixture involving Papuan- and Asian-related groups throughout Wallacea. Our results clarify previously debated times of admixture and suggest that the Neolithic dispersals into Island Southeast Asia are associated with the spread of multiple genetic ancestries. Genome-wide data of 16 ancient human individuals from islands of the North Moluccas, Sulawesi and East Nusa Tenggara reveal diverse sources of admixture and support a scenario of multiple dispersals into Wallacea.},
  archive      = {J_NATEE},
  author       = {Oliveira, Sandra and Nägele, Kathrin and Carlhoff, Selina and Pugach, Irina and Koesbardiati, Toetik and Hübner, Alexander and Meyer, Matthias and Oktaviana, Adhi Agus and Takenaka, Masami and Katagiri, Chiaki and Murti, Delta Bayu and Putri, Rizky Sugianto and Mahirta and Petchey, Fiona and Higham, Thomas and Higham, Charles F. W. and O’Connor, Sue and Hawkins, Stuart and Kinaston, Rebecca and Bellwood, Peter and Ono, Rintaro and Powell, Adam and Krause, Johannes and Posth, Cosimo and Stoneking, Mark},
  doi          = {10.1038/s41559-022-01775-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1024-1034},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ancient genomes from the last three millennia support multiple human dispersals into wallacea},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A phylogenetic and proteomic reconstruction of eukaryotic
chromatin evolution. <em>NATEE</em>, <em>6</em>(7), 1007–1023. (<a
href="https://doi.org/10.1038/s41559-022-01771-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Histones and associated chromatin proteins have essential functions in eukaryotic genome organization and regulation. Despite this fundamental role in eukaryotic cell biology, we lack a phylogenetically comprehensive understanding of chromatin evolution. Here, we combine comparative proteomics and genomics analysis of chromatin in eukaryotes and archaea. Proteomics uncovers the existence of histone post-translational modifications in archaea. However, archaeal histone modifications are scarce, in contrast with the highly conserved and abundant marks we identify across eukaryotes. Phylogenetic analysis reveals that chromatin-associated catalytic functions (for example, methyltransferases) have pre-eukaryotic origins, whereas histone mark readers and chaperones are eukaryotic innovations. We show that further chromatin evolution is characterized by expansion of readers, including capture by transposable elements and viruses. Overall, our study infers detailed evolutionary history of eukaryotic chromatin: from its archaeal roots, through the emergence of nucleosome-based regulation in the eukaryotic ancestor, to the diversification of chromatin regulators and their hijacking by genomic parasites. Comparative genomics and proteomics of archaea and eukaryotes are used to explore the evolutionary history of eukaryotic chromatin, including modifications, catalytic functions and relationship with genomic parasites.},
  archive      = {J_NATEE},
  author       = {Grau-Bové, Xavier and Navarrete, Cristina and Chiva, Cristina and Pribasnig, Thomas and Antó, Meritxell and Torruella, Guifré and Galindo, Luis Javier and Lang, Bernd Franz and Moreira, David and López-Garcia, Purificación and Ruiz-Trillo, Iñaki and Schleper, Christa and Sabidó, Eduard and Sebé-Pedrós, Arnau},
  doi          = {10.1038/s41559-022-01771-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1007-1023},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A phylogenetic and proteomic reconstruction of eukaryotic chromatin evolution},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evolutionary causes and consequences of ungulate migration.
<em>NATEE</em>, <em>6</em>(7), 998–1006. (<a
href="https://doi.org/10.1038/s41559-022-01749-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ungulate migrations are crucial for maintaining abundant populations and functional ecosystems. However, little is known about how or why migratory behaviour evolved in ungulates. To investigate the evolutionary origins of ungulate migration, we employed phylogenetic path analysis using a comprehensive species-level phylogeny of mammals. We found that 95 of 207 extant ungulate species are at least partially migratory, with migratory behaviour originating independently in 17 lineages. The evolution of migratory behaviour is associated with reliance on grass forage and living at higher latitudes wherein seasonal resource waves are most prevalent. Indeed, originations coincide with mid-Miocene cooling and the subsequent rise of C4 grasslands. Also, evolving migratory behaviour supported the evolution of larger bodies, allowing ungulates to exploit new ecological space. Reconstructions of migratory behaviour further revealed that seven of ten recently extinct species were probably migratory, suggesting that contemporary migrations are important models for understanding the ecology of the past. The authors examine present and past drivers of ungulate migratory behaviour, finding that current migratory ungulates are larger, more grass-dependent and live at higher latitudes on average than non-migrants, and that migration probably emerged after the rise of C4 grasslands and increased seasonality towards the poles.},
  archive      = {J_NATEE},
  author       = {Abraham, Joel O. and Upham, Nathan S. and Damian-Serrano, Alejandro and Jesmer, Brett R.},
  doi          = {10.1038/s41559-022-01749-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {998-1006},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolutionary causes and consequences of ungulate migration},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Compensation for wind drift during raptor migration improves
with age through mortality selection. <em>NATEE</em>, <em>6</em>(7),
989–997. (<a href="https://doi.org/10.1038/s41559-022-01776-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, billions of flying and swimming migrants negotiate the challenging displacement imposed by travelling through a flowing medium. However, little is known about how the ability to cope with drift improves through life and what mechanisms drive its development. We examined 3,140 days of migration by 90 GPS-tagged raptorial black kites (Milvus migrans) aged 1–27 years to show that the ability to compensate for lateral drift develops gradually through many more years than previously appreciated. Drift negotiation was under strong selective pressure, with inferior navigators subject to increased mortality. This progressively selected for adults able to compensate for current cross flows and for previously accumulated drift in a flexible, context-dependent and risk-dependent manner. Displacements accumulated en route carried over to shape the wintering distribution of the population. For many migrants, migratory journeys by younger individuals represent concentrated episodes of trait selection that shape adult populations and mediate their adaptation to climate change. Using data from 90 GPS-tagged black kites, the authors show that the ability to negotiate lateral drift is strongly selected over the course of the birds’ lifetimes, thereby shaping the abilities of the adult population.},
  archive      = {J_NATEE},
  author       = {Sergio, Fabrizio and Barbosa, Jomar M. and Tanferna, Alessandro and Silva, Rafa and Blas, Julio and Hiraldo, Fernando},
  doi          = {10.1038/s41559-022-01776-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {989-997},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Compensation for wind drift during raptor migration improves with age through mortality selection},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Polymicrobial infections can select against pseudomonas
aeruginosa mutators because of quorum-sensing trade-offs.
<em>NATEE</em>, <em>6</em>(7), 979–988. (<a
href="https://doi.org/10.1038/s41559-022-01768-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacteria with increased mutation rates (mutators) are common in chronic infections and are associated with poorer clinical outcomes, especially in the case of Pseudomonas aeruginosa infecting cystic fibrosis (CF) patients. There is, however, considerable between-patient variation in both P. aeruginosa mutator frequency and the composition of co-infecting pathogen communities. We investigated whether community context might affect selection of mutators. Using an in vitro CF model community, we show that P. aeruginosa mutators were favoured in the absence of other species but not in their presence. This was because there were trade-offs between adaptation to the biotic and abiotic environments (for example, loss of quorum sensing and associated toxin production was beneficial in the latter but not the former in our in vitro model community) limiting the evolvability advantage of an elevated mutation rate. Consistent with a role of co-infecting pathogens selecting against P. aeruginosa mutators in vivo, we show that the mutation frequency of P. aeruginosa population was negatively correlated with the frequency and diversity of co-infecting bacteria in CF infections. Our results suggest that co-infecting taxa can select against P. aeruginosa mutators, which may have potentially beneficial clinical consequences. Using in vitro model communities and in vivo cystic fibrosis sputum samples, the authors show that interspecific competition undergoing in certain microbial communities can constrain the fixation of loss-of-function mutations in Pseudomonas aeruginosa quorum sensing, limiting the fixation of mutators.},
  archive      = {J_NATEE},
  author       = {Luján, Adela M. and Paterson, Steve and Hesse, Elze and Sommer, Lea M. and Marvig, Rasmus L. and Sharma, M. D. and Alseth, Ellinor O. and Ciofu, Oana and Smania, Andrea M. and Molin, Søren and Johansen, Helle Krogh and Buckling, Angus},
  doi          = {10.1038/s41559-022-01768-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {979-988},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Polymicrobial infections can select against pseudomonas aeruginosa mutators because of quorum-sensing trade-offs},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The role of zinc in the adaptive evolution of polar
phytoplankton. <em>NATEE</em>, <em>6</em>(7), 965–978. (<a
href="https://doi.org/10.1038/s41559-022-01750-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zinc is an essential trace metal for oceanic primary producers with the highest concentrations in polar oceans. However, its role in the biological functioning and adaptive evolution of polar phytoplankton remains enigmatic. Here, we have applied a combination of evolutionary genomics, quantitative proteomics, co-expression analyses and cellular physiology to suggest that model polar phytoplankton species have a higher demand for zinc because of elevated cellular levels of zinc-binding proteins. We propose that adaptive expansion of regulatory zinc-finger protein families, co-expanded and co-expressed zinc-binding proteins families involved in photosynthesis and growth in these microalgal species and their natural communities were identified to be responsible for the higher zinc demand. The expression of their encoding genes in eukaryotic phytoplankton metatranscriptomes from pole-to-pole was identified to correlate not only with dissolved zinc concentrations in the upper ocean but also with temperature, suggesting that environmental conditions of polar oceans are responsible for an increased demand of zinc. These results suggest that zinc plays an important role in supporting photosynthetic growth in eukaryotic polar phytoplankton and that this has been critical for algal colonization of low-temperature polar oceans. Polar microalgae have high zinc demand. Here, the authors use quantitative proteomics and transcriptomics of polar and non-polar model algae combined with cellular physiology to show that zinc plays an important role in supporting photosynthetic growth in eukaryotic polar phytoplankton.},
  archive      = {J_NATEE},
  author       = {Ye, Naihao and Han, Wentao and Toseland, Andrew and Wang, Yitao and Fan, Xiao and Xu, Dong and van Oosterhout, Cock and Grigoriev, Igor V. and Tagliabue, Alessandro and Zhang, Jian and Zhang, Yan and Ma, Jian and Qiu, Huan and Li, Youxun and Zhang, Xiaowen and Mock, Thomas},
  doi          = {10.1038/s41559-022-01750-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {965-978},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The role of zinc in the adaptive evolution of polar phytoplankton},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Synchrony and idiosyncrasy in the gut microbiome of wild
baboons. <em>NATEE</em>, <em>6</em>(7), 955–964. (<a
href="https://doi.org/10.1038/s41559-022-01773-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human gut microbial dynamics are highly individualized, making it challenging to link microbiota to health and to design universal microbiome therapies. This individuality is typically attributed to variation in host genetics, diets, environments and medications but it could also emerge from fundamental ecological forces that shape microbiota more generally. Here, we leverage extensive gut microbial time series from wild baboons—hosts who experience little interindividual dietary and environmental heterogeneity—to test whether gut microbial dynamics are synchronized across hosts or largely idiosyncratic. Despite their shared lifestyles, baboon microbiota were only weakly synchronized. The strongest synchrony occurred among baboons living in the same social group, probably because group members range over the same habitat and simultaneously encounter the same sources of food and water. However, this synchrony was modest compared to each host’s personalized dynamics. In support, host-specific factors, especially host identity, explained, on average, more than three times the deviance in longitudinal dynamics compared to factors shared with social group members and ten times the deviance of factors shared across the host population. These results contribute to mounting evidence that highly idiosyncratic gut microbiomes are not an artefact of modern human environments and that synchronizing forces in the gut microbiome (for example, shared environments, diets and microbial dispersal) are not strong enough to overwhelm key drivers of microbiome personalization, such as host genetics, priority effects, horizontal gene transfer and functional redundancy. Analysing gut microbial time series from wild baboons, the authors show that microbiome dynamics are rarely synchronized across hosts in shared environments but are highly individualized even within the same social groups.},
  archive      = {J_NATEE},
  author       = {Björk, Johannes R. and Dasari, Mauna R. and Roche, Kim and Grieneisen, Laura and Gould, Trevor J. and Grenier, Jean-Christophe and Yotova, Vania and Gottel, Neil and Jansen, David and Gesquiere, Laurence R. and Gordon, Jacob B. and Learn, Niki H. and Wango, Tim L. and Mututua, Raphael S. and Kinyua Warutere, J. and Siodi, Long’ida and Mukherjee, Sayan and Barreiro, Luis B. and Alberts, Susan C. and Gilbert, Jack A. and Tung, Jenny and Blekhman, Ran and Archie, Elizabeth A.},
  doi          = {10.1038/s41559-022-01773-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {955-964},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Synchrony and idiosyncrasy in the gut microbiome of wild baboons},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Shoaling guppies evade predation but have deadlier
parasites. <em>NATEE</em>, <em>6</em>(7), 945–954. (<a
href="https://doi.org/10.1038/s41559-022-01772-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Parasites exploit hosts to replicate and transmit, but overexploitation kills both host and parasite. Predators may shift this cost–benefit balance by consuming infected hosts or changing host behaviour, but the strength of these effects remains unclear. Here we use field and lab data on Trinidadian guppies and their Gyrodactylus spp. parasites to show how differential predation pressure influences parasite virulence and transmission. We use an experimentally demonstrated virulence–transmission trade-off to parametrize a mathematical model in which host shoaling (as a means of anti-predator defence), increases contact rates and selects for higher virulence. Then we validate model predictions by collecting parasites from wild, Trinidadian populations; parasites from high-predation populations were more virulent in common gardens than those from low-predation populations. Broadly, our results indicate that reduced social contact selects against parasite virulence. Social living has costs and benefits; here the authors use field studies, experiments and models to show that non-consumptive predation pressure in Trinidadian guppy shoals increases parasite transmission and selects for higher virulence.},
  archive      = {J_NATEE},
  author       = {Walsman, Jason C. and Janecka, Mary J. and Clark, David R. and Kramp, Rachael D. and Rovenolt, Faith and Patrick, Regina and Mohammed, Ryan S. and Konczal, Mateusz and Cressler, Clayton E. and Stephenson, Jessica F.},
  doi          = {10.1038/s41559-022-01772-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {945-954},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Shoaling guppies evade predation but have deadlier parasites},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A polar bear paleogenome reveals extensive ancient gene flow
from polar bears into brown bears. <em>NATEE</em>, <em>6</em>(7),
936–944. (<a href="https://doi.org/10.1038/s41559-022-01753-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polar bears (Ursus maritimus) and brown bears (Ursus arctos) are sister species possessing distinct physiological and behavioural adaptations that evolved over the last 500,000 years. However, comparative and population genomics analyses have revealed that several extant and extinct brown bear populations have relatively recent polar bear ancestry, probably as the result of geographically localized instances of gene flow from polar bears into brown bears. Here, we generate and analyse an approximate 20X paleogenome from an approximately 100,000-year-old polar bear that reveals a massive prehistoric admixture event, which is evident in the genomes of all living brown bears. This ancient admixture event was not visible from genomic data derived from living polar bears. Like more recent events, this massive admixture event mainly involved unidirectional gene flow from polar bears into brown bears and occurred as climate changes caused overlap in the ranges of the two species. These findings highlight the complex reticulate paths that evolution can take within a regime of radically shifting climate. A paleogenome from an approximately 100,000-year-old polar bear shows massive prehistoric, and largely unidirectional, gene flow from polar bears into brown bears at a time of climate change-induced overlap in the ranges of the two species. This admixture event cannot be detected using genomic data from living polar bears.},
  archive      = {J_NATEE},
  author       = {Wang, Ming-Shan and Murray, Gemma G. R. and Mann, Daniel and Groves, Pamela and Vershinina, Alisa O. and Supple, Megan A. and Kapp, Joshua D. and Corbett-Detig, Russell and Crump, Sarah E. and Stirling, Ian and Laidre, Kristin L. and Kunz, Michael and Dalén, Love and Green, Richard E. and Shapiro, Beth},
  doi          = {10.1038/s41559-022-01753-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {936-944},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A polar bear paleogenome reveals extensive ancient gene flow from polar bears into brown bears},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide analyses of introgression between two sympatric
asian oak species. <em>NATEE</em>, <em>6</em>(7), 924–935. (<a
href="https://doi.org/10.1038/s41559-022-01754-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Introgression can be an important source of new alleles for adaption under rapidly changing environments, perhaps even more important than standing variation. Though introgression has been extensively studied in many plants and animals, key questions on the underlying mechanisms of introgression still remain unanswered. In particular, we are yet to determine the genomic distribution of introgressed regions along the genome; whether the extent and patterns of introgression are influenced by ecological factors; and when and how introgression contributes to adaptation. Here, we generated high-quality genomic resources for two sympatric widespread Asian oak species, Quercus acutissima and Q. variabilis, sampled in multiple forests to study introgression between them. We show that introgressed regions are broadly distributed across the genome. Introgression was affected by genetic divergence between pairs of populations and by the similarity of the environments in which they live—populations occupying similar ecological sites tended to share the same introgressed regions. Introgressed genomic footprints of adaptation were preferentially located in regions with suppressed recombination rate. Introgression probably confers adaptation in these oak populations by introducing allelic variation in cis-regulatory elements, in particular through transposable element insertions, thereby altering the regulation of genes related to stress. Our results provide new avenues of research for uncovering mechanisms of adaptation due to hybridization in sympatric species. Introgression is an important source of genetic variation. Analysing genomes of two sympatric widespread Asian oak species, the authors find introgression across the genome and signatures of adaptive introgression in regions with suppressed recombination rate.},
  archive      = {J_NATEE},
  author       = {Fu, Ruirui and Zhu, Yuxiang and Liu, Ying and Feng, Yu and Lu, Rui-Sen and Li, Yao and Li, Pan and Kremer, Antoine and Lascoux, Martin and Chen, Jun},
  doi          = {10.1038/s41559-022-01754-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {924-935},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genome-wide analyses of introgression between two sympatric asian oak species},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Allorecognition genes drive reproductive isolation in
podospora anserina. <em>NATEE</em>, <em>6</em>(7), 910–923. (<a
href="https://doi.org/10.1038/s41559-022-01734-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Allorecognition, the capacity to discriminate self from conspecific non-self, is a ubiquitous organismal feature typically governed by genes evolving under balancing selection. Here, we show that in the fungus Podospora anserina, allorecognition loci controlling vegetative incompatibility (het genes), define two reproductively isolated groups through pleiotropic effects on sexual compatibility. These two groups emerge from the antagonistic interactions of the unlinked loci het-r (encoding a NOD-like receptor) and het-v (encoding a methyltransferase and an MLKL/HeLo domain protein). Using a combination of genetic and ecological data, supported by simulations, we provide a concrete and molecularly defined example whereby the origin and coexistence of reproductively isolated groups in sympatry is driven by pleiotropic genes under balancing selection. Genetic, ecological and simulation data demonstrate that the origin and coexistence of reproductively isolated sympatric groups in a fungus is driven by pleiotropic vegetative incompatibility genes under balancing selection.},
  archive      = {J_NATEE},
  author       = {Ament-Velásquez, S. Lorena and Vogan, Aaron A. and Granger-Farbos, Alexandra and Bastiaans, Eric and Martinossi-Allibert, Ivain and Saupe, Sven J. and de Groot, Suzette and Lascoux, Martin and Debets, Alfons J. M. and Clavé, Corinne and Johannesson, Hanna},
  doi          = {10.1038/s41559-022-01734-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {910-923},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Allorecognition genes drive reproductive isolation in podospora anserina},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Phylotype diversity within soil fungal functional groups
drives ecosystem stability. <em>NATEE</em>, <em>6</em>(7), 900–909. (<a
href="https://doi.org/10.1038/s41559-022-01756-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soil fungi are fundamental to plant productivity, yet their influence on the temporal stability of global terrestrial ecosystems, and their capacity to buffer plant productivity against extreme drought events, remain uncertain. Here we combined three independent global field surveys of soil fungi with a satellite-derived temporal assessment of plant productivity, and report that phylotype richness within particular fungal functional groups drives the stability of terrestrial ecosystems. The richness of fungal decomposers was consistently and positively associated with ecosystem stability worldwide, while the opposite pattern was found for the richness of fungal plant pathogens, particularly in grasslands. We further demonstrated that the richness of soil decomposers was consistently positively linked with higher resistance of plant productivity in response to extreme drought events, while that of fungal plant pathogens showed a general negative relationship with plant productivity resilience/resistance patterns. Together, our work provides evidence supporting the critical role of soil fungal diversity to secure stable plant production over time in global ecosystems, and to buffer against extreme climate events. The authors link fungal diversity to the stability of terrestrial ecosystem productivity across three global datasets, finding that richness of decomposers and mycorrhizae are positively associated with stability while the richness of plant pathogens is negatively related to stability.},
  archive      = {J_NATEE},
  author       = {Liu, Shengen and García-Palacios, Pablo and Tedersoo, Leho and Guirado, Emilio and van der Heijden, Marcel G. A. and Wagg, Cameron and Chen, Dima and Wang, Qingkui and Wang, Juntao and Singh, Brajesh K. and Delgado-Baquerizo, Manuel},
  doi          = {10.1038/s41559-022-01756-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {900-909},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Phylotype diversity within soil fungal functional groups drives ecosystem stability},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhanced habitat loss of the himalayan endemic flora driven
by warming-forced upslope tree expansion. <em>NATEE</em>, <em>6</em>(7),
890–899. (<a href="https://doi.org/10.1038/s41559-022-01774-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-elevation trees cannot always reach the thermal treeline, the potential upper range limit set by growing-season temperature. But delineation of the realized upper range limit of trees and quantification of the drivers, which lead to trees being absent from the treeline, is lacking. Here, we used 30 m resolution satellite tree-cover data, validated by more than 0.7 million visual interpretations from Google Earth images, to map the realized range limit of trees along the Himalaya which harbours one of the world’s richest alpine endemic flora. The realized range limit of trees is ~800 m higher in the eastern Himalaya than in the western and central Himalaya. Trees had reached their thermal treeline positions in more than 80% of the cases over eastern Himalaya but are absent from the treeline position in western and central Himalaya, due to anthropogenic disturbance and/or premonsoon drought. By combining projections of the deviation of trees from the treeline position due to regional environmental stresses with warming-induced treeline shift, we predict that trees will migrate upslope by ~140 m by the end of the twenty-first century in the eastern Himalaya. This shift will cause the endemic flora to lose at least ~20% of its current habitats, highlighting the necessity to reassess the effectiveness of current conservation networks and policies over the Himalaya. A high-resolution map of the realized range limit of high-elevation trees across the Himalayas shows that trees are absent from the thermal treeline, determined by growing-season temperature, across the western and central Himalayas, as a result of human disturbance and/or premonsoon drought.},
  archive      = {J_NATEE},
  author       = {Wang, Xiaoyi and Wang, Tao and Xu, Jinfeng and Shen, Zehao and Yang, Yongping and Chen, Anping and Wang, Shaopeng and Liang, Eryuan and Piao, Shilong},
  doi          = {10.1038/s41559-022-01774-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {890-899},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Enhanced habitat loss of the himalayan endemic flora driven by warming-forced upslope tree expansion},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Functional susceptibility of tropical forests to climate
change. <em>NATEE</em>, <em>6</em>(7), 878–889. (<a
href="https://doi.org/10.1038/s41559-022-01747-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tropical forests are some of the most biodiverse ecosystems in the world, yet their functioning is threatened by anthropogenic disturbances and climate change. Global actions to conserve tropical forests could be enhanced by having local knowledge on the forestsʼ functional diversity and functional redundancy as proxies for their capacity to respond to global environmental change. Here we create estimates of plant functional diversity and redundancy across the tropics by combining a dataset of 16 morphological, chemical and photosynthetic plant traits sampled from 2,461 individual trees from 74 sites distributed across four continents together with local climate data for the past half century. Our findings suggest a strong link between climate and functional diversity and redundancy with the three trait groups responding similarly across the tropics and climate gradient. We show that drier tropical forests are overall less functionally diverse than wetter forests and that functional redundancy declines with increasing soil water and vapour pressure deficits. Areas with high functional diversity and high functional redundancy tend to better maintain ecosystem functioning, such as aboveground biomass, after extreme weather events. Our predictions suggest that the lower functional diversity and lower functional redundancy of drier tropical forests, in comparison with wetter forests, may leave them more at risk of shifting towards alternative states in face of further declines in water availability across tropical regions. A dataset of 16 plant traits sampled from 2,461 individual trees from 74 tropical forest sites around the world is used to show a strong link between climate and plant functional diversity and redundancy, with drier tropical forests likely being less able to respond to declines in water availability.},
  archive      = {J_NATEE},
  author       = {Aguirre‐Gutiérrez, Jesús and Berenguer, Erika and Oliveras Menor, Imma and Bauman, David and Corral-Rivas, Jose Javier and Nava-Miranda, Maria Guadalupe and Both, Sabine and Ndong, Josué Edzang and Ondo, Fidèle Evouna and Bengone, Natacha N’ssi and Mihinhou, Vianet and Dalling, James W. and Heineman, Katherine and Figueiredo, Axa and González-M, Roy and Norden, Natalia and Hurtado-M, Ana Belén and González, Diego and Salgado-Negret, Beatriz and Reis, Simone Matias and Moraes de Seixas, Marina Maria and Farfan-Rios, William and Shenkin, Alexander and Riutta, Terhi and Girardin, Cécile A. J. and Moore, Sam and Abernethy, Kate and Asner, Gregory P. and Bentley, Lisa Patrick and Burslem, David F.R.P. and Cernusak, Lucas A. and Enquist, Brian J. and Ewers, Robert M. and Ferreira, Joice and Jeffery, Kathryn J. and Joly, Carlos A. and Marimon-Junior, Ben Hur and Martin, Roberta E. and Morandi, Paulo S. and Phillips, Oliver L. and Bennett, Amy C. and Lewis, Simon L. and Quesada, Carlos A. and Marimon, Beatriz Schwantes and Kissling, W. Daniel and Silman, Miles and Teh, Yit Arn and White, Lee J. T. and Salinas, Norma and Coomes, David A. and Barlow, Jos and Adu-Bredu, Stephen and Malhi, Yadvinder},
  doi          = {10.1038/s41559-022-01747-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {878-889},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Functional susceptibility of tropical forests to climate change},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sugars dominate the seagrass rhizosphere. <em>NATEE</em>,
<em>6</em>(7), 866–877. (<a
href="https://doi.org/10.1038/s41559-022-01740-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Seagrasses are among the most efficient sinks of carbon dioxide on Earth. While carbon sequestration in terrestrial plants is linked to the microorganisms living in their soils, the interactions of seagrasses with their rhizospheres are poorly understood. Here, we show that the seagrass, Posidonia oceanica excretes sugars, mainly sucrose, into its rhizosphere. These sugars accumulate to µM concentrations—nearly 80 times higher than previously observed in marine environments. This finding is unexpected as sugars are readily consumed by microorganisms. Our experiments indicated that under low oxygen conditions, phenolic compounds from P. oceanica inhibited microbial consumption of sucrose. Analyses of the rhizosphere community revealed that many microbes had the genes for degrading sucrose but these were only expressed by a few taxa that also expressed genes for degrading phenolics. Given that we observed high sucrose concentrations underneath three other species of marine plants, we predict that the presence of plant-produced phenolics under low oxygen conditions allows the accumulation of labile molecules across aquatic rhizospheres. Seagrass meadows are important carbon sinks. Here, the authors show that organic carbon in the form of simple sugars can accumulate at high concentrations in seagrass rhizospheres because plant phenolic compounds inhibit their consumption by microorganisms.},
  archive      = {J_NATEE},
  author       = {Sogin, E. Maggie and Michellod, Dolma and Gruber-Vodicka, Harald R. and Bourceau, Patric and Geier, Benedikt and Meier, Dimitri V. and Seidel, Michael and Ahmerkamp, Soeren and Schorn, Sina and D’Angelo, Grace and Procaccini, Gabriele and Dubilier, Nicole and Liebeke, Manuel},
  doi          = {10.1038/s41559-022-01740-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {866-877},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sugars dominate the seagrass rhizosphere},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ecological modelling approaches for predicting emergent
properties in microbial communities. <em>NATEE</em>, <em>6</em>(7),
855–865. (<a href="https://doi.org/10.1038/s41559-022-01746-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent studies have brought forward the critical role of emergent properties in shaping microbial communities and the ecosystems of which they are a part. Emergent properties—patterns or functions that cannot be deduced linearly from the properties of the constituent parts—underlie important ecological characteristics such as resilience, niche expansion and spatial self-organization. While it is clear that emergent properties are a consequence of interactions within the community, their non-linear nature makes mathematical modelling imperative for establishing the quantitative link between community structure and function. As the need for conservation and rational modulation of microbial ecosystems is increasingly apparent, so is the consideration of the benefits and limitations of the approaches to model emergent properties. Here we review ecosystem modelling approaches from the viewpoint of emergent properties. We consider the scope, advantages and limitations of Lotka–Volterra, consumer–resource, trait-based, individual-based and genome-scale metabolic models. Future efforts in this research area would benefit from capitalizing on the complementarity between these approaches towards enabling rational modulation of complex microbial ecosystems. As extremely large microbial community datasets have proliferated, numerous approaches to studying their ecological properties have emerged. In this Review, the authors offer a guide to five ecological modelling techniques used to study complex microbial communities.},
  archive      = {J_NATEE},
  author       = {van den Berg, Naomi Iris and Machado, Daniel and Santos, Sophia and Rocha, Isabel and Chacón, Jeremy and Harcombe, William and Mitri, Sara and Patil, Kiran R.},
  doi          = {10.1038/s41559-022-01746-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {855-865},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecological modelling approaches for predicting emergent properties in microbial communities},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ungulates on the move. <em>NATEE</em>, <em>6</em>(7),
853–854. (<a href="https://doi.org/10.1038/s41559-022-01757-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Examining the evolutionary history of ungulate migration shows that this behaviour has evolved multiple times in response to grassland expansion and increased seasonality of resources.},
  archive      = {J_NATEE},
  author       = {Tucker, Marlee A.},
  doi          = {10.1038/s41559-022-01757-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {853-854},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ungulates on the move},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Polar algae flaunt their zinc assets. <em>NATEE</em>,
<em>6</em>(7), 851–852. (<a
href="https://doi.org/10.1038/s41559-022-01721-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-genome sequencing and comparative omics analyses highlight recent and parallel paths to adaptive evolution involving expansions in zinc-binding proteins in the genomes of diverse cold-adapted algae.},
  archive      = {J_NATEE},
  author       = {Blaby-Haas, Crysten E.},
  doi          = {10.1038/s41559-022-01721-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {851-852},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Polar algae flaunt their zinc assets},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Personalized microbiomes in social baboons. <em>NATEE</em>,
<em>6</em>(7), 849–850. (<a
href="https://doi.org/10.1038/s41559-022-01769-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Longitudinal data on gut microbiomes of wild baboons show that microbial communities are highly individualized, despite shared diet and environment within primate social groups.},
  archive      = {J_NATEE},
  author       = {Videvall, Elin},
  doi          = {10.1038/s41559-022-01769-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {849-850},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Personalized microbiomes in social baboons},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A sweet spot in marine ecosystems. <em>NATEE</em>,
<em>6</em>(7), 847–848. (<a
href="https://doi.org/10.1038/s41559-022-01764-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of sediment and pore water from three seagrass meadows around the world reveals an unexpected accumulation of labile carbon in the marine rhizosphere, explained by sediment chemistry.},
  archive      = {J_NATEE},
  author       = {Hu, Yuntao and Northen, Trent R.},
  doi          = {10.1038/s41559-022-01764-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {847-848},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A sweet spot in marine ecosystems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Wrangling rabies. <em>NATEE</em>, <em>6</em>(7), 846. (<a
href="https://doi.org/10.1038/s41559-022-01802-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-022-01802-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {846},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Wrangling rabies},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Phylogenies at a cellular scale. <em>NATEE</em>,
<em>6</em>(7), 845. (<a
href="https://doi.org/10.1038/s41559-022-01798-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Goymer, Patrick},
  doi          = {10.1038/s41559-022-01798-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {845},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Phylogenies at a cellular scale},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Lockdown fires. <em>NATEE</em>, <em>6</em>(7), 844. (<a
href="https://doi.org/10.1038/s41559-022-01793-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-022-01793-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {844},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Lockdown fires},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Salt-stressed frogs. <em>NATEE</em>, <em>6</em>(7), 843.
(<a href="https://doi.org/10.1038/s41559-022-01789-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-022-01789-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {843},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Salt-stressed frogs},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Healthy indigenous harvests. <em>NATEE</em>, <em>6</em>(7),
842. (<a href="https://doi.org/10.1038/s41559-022-01785-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-022-01785-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {842},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Healthy indigenous harvests},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conservation’s hot ticket. <em>NATEE</em>, <em>6</em>(7),
840–841. (<a href="https://doi.org/10.1038/s41559-022-01752-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Mustin, Karen},
  doi          = {10.1038/s41559-022-01752-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {840-841},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Conservation’s hot ticket},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A collaborative agenda for archaeology and fire science.
<em>NATEE</em>, <em>6</em>(7), 835–839. (<a
href="https://doi.org/10.1038/s41559-022-01759-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Humans have influenced global fire activity for millennia and will continue to do so into the future. Given the long-term interaction between humans and fire, we propose a collaborative research agenda linking archaeology and fire science that emphasizes the socioecological histories and consequences of anthropogenic fire in the development of fire management strategies today.},
  archive      = {J_NATEE},
  author       = {Snitker, Grant and Roos, Christopher I. and Sullivan, Alan P. and Maezumi, S. Yoshi and Bird, Douglas W. and Coughlan, Michael R. and Derr, Kelly M. and Gassaway, Linn and Klimaszewski-Patterson, Anna and Loehman, Rachel A.},
  doi          = {10.1038/s41559-022-01759-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {835-839},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A collaborative agenda for archaeology and fire science},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A fundamental limit to the search for the oldest fossils.
<em>NATEE</em>, <em>6</em>(7), 832–834. (<a
href="https://doi.org/10.1038/s41559-022-01777-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Life emerged from and amidst non-living phenomena that already possessed some of the hallmarks now used four billion years later to recognize fossil organisms. It may be next to impossible to distinguish the earliest signs of life against this background. What can we still learn from fossil-like materials on the early Earth and elsewhere?},
  archive      = {J_NATEE},
  author       = {McMahon, Sean and Jordan, Seán F.},
  doi          = {10.1038/s41559-022-01777-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {832-834},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A fundamental limit to the search for the oldest fossils},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Opportunities and risks of publishing academic talks online.
<em>NATEE</em>, <em>6</em>(7), 830–831. (<a
href="https://doi.org/10.1038/s41559-022-01755-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {van der Wal, Jessica E. M. and Spottiswoode, Claire N. and Sugasawa, Shoko},
  doi          = {10.1038/s41559-022-01755-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {830-831},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Opportunities and risks of publishing academic talks online},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Stop ignoring map uncertainty in biodiversity science and
conservation policy. <em>NATEE</em>, <em>6</em>(7), 828–829. (<a
href="https://doi.org/10.1038/s41559-022-01778-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Jansen, Jan and Woolley, Skipton N. C. and Dunstan, Piers K. and Foster, Scott D. and Hill, Nicole A. and Haward, Marcus and Johnson, Craig R.},
  doi          = {10.1038/s41559-022-01778-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {828-829},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Stop ignoring map uncertainty in biodiversity science and conservation policy},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mendelian inheritance. <em>NATEE</em>, <em>6</em>(7), 827.
(<a href="https://doi.org/10.1038/s41559-022-01834-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The bicentenary of the birth of Mendel is an opportunity to explore the origins of genetics and to confront some of its difficult subsequent history.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01834-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {827},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mendelian inheritance},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Publisher correction: Healing the land and the academy.
<em>NATEE</em>, <em>6</em>(6), 825. (<a
href="https://doi.org/10.1038/s41559-022-01782-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa and Grenz, Jennifer},
  doi          = {10.1038/s41559-022-01782-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {825},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Healing the land and the academy},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Publisher correction: Diversity in nature and academia.
<em>NATEE</em>, <em>6</em>(6), 824. (<a
href="https://doi.org/10.1038/s41559-022-01780-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera and Gordon, Swanne},
  doi          = {10.1038/s41559-022-01780-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {824},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Diversity in nature and academia},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Field experiments underestimate
aboveground biomass response to drought. <em>NATEE</em>, <em>6</em>(6),
823. (<a href="https://doi.org/10.1038/s41559-022-01767-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Kröel-Dulay, György and Mojzes, Andrea and Szitár, Katalin and Bahn, Michael and Batáry, Péter and Beier, Claus and Bilton, Mark and De Boeck, Hans J. and Dukes, Jeffrey S. and Estiarte, Marc and Holub, Petr and Jentsch, Anke and Schmidt, Inger Kappel and Kreyling, Juergen and Reinsch, Sabine and Larsen, Klaus Steenberg and Sternberg, Marcelo and Tielbörger, Katja and Tietema, Albert and Vicca, Sara and Peñuelas, Josep},
  doi          = {10.1038/s41559-022-01767-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {823},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Field experiments underestimate aboveground biomass response to drought},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Emergence and intensification of dairying in the caucasus
and eurasian steppes. <em>NATEE</em>, <em>6</em>(6), 813–822. (<a
href="https://doi.org/10.1038/s41559-022-01701-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Archaeological and archaeogenetic evidence points to the Pontic–Caspian steppe zone between the Caucasus and the Black Sea as the crucible from which the earliest steppe pastoralist societies arose and spread, ultimately influencing populations from Europe to Inner Asia. However, little is known about their economic foundations and the factors that may have contributed to their extensive mobility. Here, we investigate dietary proteins within the dental calculus proteomes of 45 individuals spanning the Neolithic to Greco-Roman periods in the Pontic–Caspian Steppe and neighbouring South Caucasus, Oka–Volga–Don and East Urals regions. We find that sheep dairying accompanies the earliest forms of Eneolithic pastoralism in the North Caucasus. During the fourth millennium bc, Maykop and early Yamnaya populations also focused dairying exclusively on sheep while reserving cattle for traction and other purposes. We observe a breakdown in livestock specialization and an economic diversification of dairy herds coinciding with aridification during the subsequent late Yamnaya and North Caucasus Culture phases, followed by severe climate deterioration during the Catacomb and Lola periods. The need for additional pastures to support these herds may have driven the heightened mobility of the Middle and Late Bronze Age periods. Following a hiatus of more than 500 years, the North Caucasian steppe was repopulated by Early Iron Age societies with a broad mobile dairy economy, including a new focus on horse milking. Milk proteins from the North Caucasus and Eurasian steppe support the initial development of sheep dairying during the Eneolithic, followed by subsequent intensification and husbandry of different dairy animals during the Middle Bronze Age and later periods.},
  archive      = {J_NATEE},
  author       = {Scott, Ashley and Reinhold, Sabine and Hermes, Taylor and Kalmykov, Alexey A. and Belinskiy, Andrey and Buzhilova, Alexandra and Berezina, Natalia and Kantorovich, Anatoliy R. and Maslov, Vladimir E. and Guliyev, Farhad and Lyonnet, Bertille and Gasimov, Parviz and Jalilov, Bakhtiyar and Eminli, Jeyhun and Iskandarov, Emil and Hammer, Emily and Nugent, Selin E. and Hagan, Richard and Majander, Kerttu and Onkamo, Päivi and Nordqvist, Kerkko and Shishlina, Natalia and Kaverzneva, Elena and Korolev, Arkadiy I. and Khokhlov, Aleksandr A. and Smolyaninov, Roman V. and Sharapova, Svetlana V. and Krause, Rüdiger and Karapetian, Marina and Stolarczyk, Eliza and Krause, Johannes and Hansen, Svend and Haak, Wolfgang and Warinner, Christina},
  doi          = {10.1038/s41559-022-01701-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {813-822},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Emergence and intensification of dairying in the caucasus and eurasian steppes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Frontier lapita interaction with resident papuan populations
set the stage for initial peopling of the pacific. <em>NATEE</em>,
<em>6</em>(6), 802–812. (<a
href="https://doi.org/10.1038/s41559-022-01735-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The initial peopling of the remote Pacific islands was one of the greatest migrations in human history, beginning three millennia ago by Lapita cultural groups. The spread of Lapita out of an ancestral Asian homeland is a dominant narrative in the origins of Pacific peoples, and although Island New Guinea has long been recognized as a springboard for the peopling of Oceania, the role of Indigenous populations in this remarkable phase of exploration remains largely untested. Here, we report the earliest evidence for Lapita-introduced animals, turtle bone technology and repeated obsidian import in southern New Guinea 3,480–3,060 years ago, synchronous with the establishment of the earliest known Lapita settlements 700 km away. Our findings precede sustained Lapita migrations and pottery introductions by several centuries, occur alongside Indigenous technologies and suggest continued multicultural influences on population diversity despite language replacement. Our work shows that initial Lapita expansion throughout Island New Guinea was more expansive than previously considered, with Indigenous contact influencing migration pathways and island-hopping strategies that culminated in rapid and purposeful Pacific-wide settlement. Later Lapita dispersals through New Guinea were facilitated by earlier contact with Indigenous populations and profoundly influenced the region as a global centre of cultural and linguistic diversity. Excavation in Island New Guinea reveals features associated with the Pacific Lapita cultural complex as well as sustained local cultural traditions from 3,480–3,060 years ago, contemporary with the earliest known Lapita settlements 700 km away. This supports New Guinea as a springboard for Lapita dispersal throughout the Pacific and illuminates their origins.},
  archive      = {J_NATEE},
  author       = {Shaw, Ben and Hawkins, Stuart and Becerra-Valdivia, Lorena and Turney, Chris S. M. and Coxe, Simon and Kewibu, Vincent and Haro, Jemina and Miamba, Kenneth and Leclerc, Mathieu and Spriggs, Matthew and Privat, Karen and Haberle, Simon and Hopf, Felicitas and Hull, Emily and Pengilley, Alana and Brown, Samantha and Marjo, Christopher E. and Jacobsen, Geraldine},
  doi          = {10.1038/s41559-022-01735-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {802-812},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Frontier lapita interaction with resident papuan populations set the stage for initial peopling of the pacific},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Urban-adapted mammal species have more known pathogens.
<em>NATEE</em>, <em>6</em>(6), 794–801. (<a
href="https://doi.org/10.1038/s41559-022-01723-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The world is rapidly urbanizing, inviting mounting concern that urban environments will experience increased zoonotic disease risk. Urban animals could have more frequent contact with humans, therefore transmitting more zoonotic parasites; however, this relationship is complicated by sampling bias and phenotypic confounders. Here we test whether urban mammal species host more zoonotic parasites, investigating the underlying drivers alongside a suite of phenotypic, taxonomic and geographic predictors. We found that urban-adapted mammals have more documented parasites and more zoonotic parasites: despite comprising only 6% of investigated species, urban mammals provided 39% of known host–parasite combinations. However, contrary to predictions, much of the observed effect was attributable to parasite discovery and research effort rather than to urban adaptation status, and urban-adapted species in fact hosted fewer zoonotic parasites than expected on the basis of their total parasite richness. We conclude that extended historical contact with humans has had a limited impact on zoonotic parasite richness in urban-adapted mammals; instead, their greater observed zoonotic richness probably reflects sampling bias arising from proximity to humans, supporting a near-universal conflation between zoonotic risk, research effort and synanthropy. These findings underscore the need to resolve the mechanisms linking anthropogenic change, sampling bias and observed wildlife disease dynamics. After compiling literature data on mammal parasites across urban and non-urban areas, the authors show that mammals in urban areas have more parasites overall without disproportionately more zoonotic ones, as is commonly thought.},
  archive      = {J_NATEE},
  author       = {Albery, Gregory F. and Carlson, Colin J. and Cohen, Lily E. and Eskew, Evan A. and Gibb, Rory and Ryan, Sadie J. and Sweeny, Amy R. and Becker, Daniel J.},
  doi          = {10.1038/s41559-022-01723-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {794-801},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Urban-adapted mammal species have more known pathogens},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Effects of epistasis and recombination between
vaccine-escape and virulence alleles on the dynamics of pathogen
adaptation. <em>NATEE</em>, <em>6</em>(6), 786–793. (<a
href="https://doi.org/10.1038/s41559-022-01709-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pathogen adaptation to public health interventions such as vaccination may take tortuous routes and involve multiple mutations at different locations in the pathogen genome, acting on distinct phenotypic traits. Yet how these multi-locus adaptations jointly evolve is poorly understood. Here we consider the joint evolution of two adaptations: pathogen escape from the vaccine-induced immune response and adjustments to pathogen virulence affecting transmission or clearance. We elucidate the role played by epistasis and recombination, with an emphasis on the different protective effects of vaccination. We show that vaccines blocking infection, reducing transmission and/or increasing clearance generate positive epistasis between the vaccine-escape and virulence alleles, favouring strains that carry both mutations, whereas vaccines reducing virulence mortality generate negative epistasis, favouring strains that carry either mutation but not both. High rates of recombination can affect these predictions. If epistasis is positive, frequent recombination can prevent the transient build-up of more virulent escape strains. If epistasis is negative, frequent recombination between loci can create an evolutionary bistability, favouring whichever adaptation is more accessible. Our work provides a timely alternative to the variant-centred perspective on pathogen adaptation and captures the effect of different types of vaccine on the interference between multiple adaptive mutations. Modelling shows that vaccines that reduce infection or hasten infection clearance generate positive epistasis between vaccine-escape and virulence alleles, whereas vaccines that reduce virulence generate negative epistasis. High rates of recombination also affect how selection acts on both alleles.},
  archive      = {J_NATEE},
  author       = {McLeod, David V. and Gandon, Sylvain},
  doi          = {10.1038/s41559-022-01709-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {786-793},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Effects of epistasis and recombination between vaccine-escape and virulence alleles on the dynamics of pathogen adaptation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Decomposing phenotypic skew and its effects on the predicted
response to strong selection. <em>NATEE</em>, <em>6</em>(6), 774–785.
(<a href="https://doi.org/10.1038/s41559-022-01694-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The major frameworks for predicting evolutionary change assume that a phenotype’s underlying genetic and environmental components are normally distributed. However, the predictions of these frameworks may no longer hold if distributions are skewed. Despite this, phenotypic skew has never been decomposed, meaning the fundamental assumptions of quantitative genetics remain untested. Here we demonstrate that the substantial phenotypic skew in the body size of juvenile blue tits (Cyanistes caeruleus) is driven by environmental factors. Although skew had little impact on our predictions of selection response in this case, our results highlight the impact of skew on the estimation of inheritance and selection. Specifically, the nonlinear parent–offspring regressions induced by skew, alongside selective disappearance, can strongly bias estimates of heritability. The ubiquity of skew and strong directional selection on juvenile body size imply that heritability is commonly overestimated, which may in part explain the discrepancy between predicted and observed trait evolution. Using data from blue tits, the authors show that the distribution of juvenile body size is skewed due to environmental factors, and that although it does not affect selection estimates in this case, skew has the potential to overestimate heritability and explain discrepancies in predicted trait evolution.},
  archive      = {J_NATEE},
  author       = {Pick, Joel L. and Lemon, Hannah E. and Thomson, Caroline E. and Hadfield, Jarrod D.},
  doi          = {10.1038/s41559-022-01694-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {774-785},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Decomposing phenotypic skew and its effects on the predicted response to strong selection},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Gene loss and compensatory evolution promotes the emergence
of morphological novelties in budding yeast. <em>NATEE</em>,
<em>6</em>(6), 763–773. (<a
href="https://doi.org/10.1038/s41559-022-01730-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deleterious mutations are generally considered to be irrelevant for morphological evolution. However, they could be compensated by conditionally beneficial mutations, thereby providing access to new adaptive paths. Here we use high-dimensional phenotyping of laboratory-evolved budding yeast lineages to demonstrate that new cellular morphologies emerge exceptionally rapidly as a by-product of gene loss and subsequent compensatory evolution. Unexpectedly, the capacities for invasive growth, multicellular aggregation and biofilm formation also spontaneously evolve in response to gene loss. These multicellular phenotypes can be achieved by diverse mutational routes and without reactivating the canonical regulatory pathways. These ecologically and clinically relevant traits originate as pleiotropic side effects of compensatory evolution and have no obvious utility in the laboratory environment. The extent of morphological diversity in the evolved lineages is comparable to that of natural yeast isolates with diverse genetic backgrounds and lifestyles. Finally, we show that both the initial gene loss and subsequent compensatory mutations contribute to new morphologies, with their synergistic effects underlying specific morphological changes. We conclude that compensatory evolution is a previously unrecognized source of morphological diversity and phenotypic novelties. The role of compensatory mutations in phenotypic evolution is unclear. Here the authors use experimental evolution to show that gene loss followed by compensatory evolution contributes to the rapid emergence of new cellular and multicellular morphologies in the budding yeast.},
  archive      = {J_NATEE},
  author       = {Farkas, Zoltán and Kovács, Károly and Sarkadi, Zsuzsa and Kalapis, Dorottya and Fekete, Gergely and Birtyik, Fanni and Ayaydin, Ferhan and Molnár, Csaba and Horváth, Péter and Pál, Csaba and Papp, Balázs},
  doi          = {10.1038/s41559-022-01730-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {763-773},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Gene loss and compensatory evolution promotes the emergence of morphological novelties in budding yeast},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cophylogeny and convergence shape holobiont evolution in
sponge–microbe symbioses. <em>NATEE</em>, <em>6</em>(6), 750–762. (<a
href="https://doi.org/10.1038/s41559-022-01712-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Symbiotic microbial communities of sponges serve critical functions that have shaped the evolution of reef ecosystems since their origins. Symbiont abundance varies tremendously among sponges, with many species classified as either low microbial abundance (LMA) or high microbial abundance (HMA), but the evolutionary dynamics of these symbiotic states remain unknown. This study examines the LMA/HMA dichotomy across an exhaustive sampling of Caribbean sponge biodiversity and predicts that the LMA symbiotic state is the ancestral state among sponges. Conversely, HMA symbioses, consisting of more specialized microorganisms, have evolved multiple times by recruiting similar assemblages, mostly since the rise of scleractinian-dominated reefs. Additionally, HMA symbioses show stronger signals of phylosymbiosis and cophylogeny, consistent with stronger co-evolutionary interaction in these complex holobionts. These results indicate that HMA holobionts are characterized by increased endemism, metabolic dependence and chemical defences. The selective forces driving these patterns may include the concurrent increase in dissolved organic matter in reef ecosystems or the diversification of spongivorous fishes. Surveying sponge biodiversity across the Caribbean, the authors show that low microbial abundance is the ancestral symbiotic state, whereas high microbial abundance has evolved multiple times and exhibits increased endemism, metabolic dependence and chemical defences.},
  archive      = {J_NATEE},
  author       = {Sabrina Pankey, M. and Plachetzki, David C. and Macartney, Keir J. and Gastaldi, Marianela and Slattery, Marc and Gochfeld, Deborah J. and Lesser, Michael P.},
  doi          = {10.1038/s41559-022-01712-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {750-762},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cophylogeny and convergence shape holobiont evolution in sponge–microbe symbioses},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evolution of coastal forests based on a full set of mangrove
genomes. <em>NATEE</em>, <em>6</em>(6), 738–749. (<a
href="https://doi.org/10.1038/s41559-022-01744-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic studies are now poised to explore whole communities of species. The ~70 species of woody plants that anchor the coastal ecosystems of the tropics, collectively referred to as mangroves, are particularly suited to this exploration. In this study, we de novo sequenced the whole genomes of 32 mangroves, which we combined with other sequences of 30 additional species, comprising almost all mangroves globally. These community-wide genomic data will be valuable for ecology, evolution and biodiversity research. While the data revealed 27 independent origins of mangroves, the total phylogeny shows only modest increases in species number, even in coastal areas of active speciation, suggesting that mangrove extinction is common. A possible explanation for common extinction is the frequent sea-level rises and falls (SLRs and SLFs) documented in the geological record. Indeed, near-extinctions of species with extremely small population size (N) often happened during periods of rapid SLR, as revealed by the genome-wide heterozygosity of almost all mangroves. Reduction in N has possibly been further compounded by population fragmentation and the subsequent accumulation of deleterious mutations, thus pushing mangroves even closer to extinction. Crucially, the impact of the next SLR will be exacerbated by human encroachment into these mangrove habitats, potentially altering the ecosystems of tropical coasts irreversibly. Analysis of 48 newly generated genomes, including 32 mangrove species, sheds light on patterns of speciation and extinction of these woody, coastal plants.},
  archive      = {J_NATEE},
  author       = {He, Ziwen and Feng, Xiao and Chen, Qipian and Li, Liangwei and Li, Sen and Han, Kai and Guo, Zixiao and Wang, Jiayan and Liu, Min and Shi, Chengcheng and Xu, Shaohua and Shao, Shao and Liu, Xin and Mao, Xiaomeng and Xie, Wei and Wang, Xinfeng and Zhang, Rufan and Li, Guohong and Wu, Weihong and Zheng, Zheng and Zhong, Cairong and Duke, Norman C. and Boufford, David E. and Fan, Guangyi and Wu, Chung-I and Ricklefs, Robert E. and Shi, Suhua},
  doi          = {10.1038/s41559-022-01744-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {738-749},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolution of coastal forests based on a full set of mangrove genomes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Privately protected areas increase global protected area
coverage and connectivity. <em>NATEE</em>, <em>6</em>(6), 730–737. (<a
href="https://doi.org/10.1038/s41559-022-01715-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Privately protected areas (PPAs) are increasing in number and extent. Yet, we know little about their contribution to conservation and how this compares to other forms of protected area (PA). We address this gap by assessing the contribution of 17,561 PPAs to the coverage, complementarity and connectivity of existing PA networks in 15 countries across 5 continents. We find that PPAs (1) are three times more likely to be in biomes with &amp;lt;10% of their area protected than are other PA governance types and twice as likely to be in areas with the greatest human disturbance; (2) that they protect a further 1.2% of key biodiversity areas; (3) that they account for 3.4% of land under protection; and (4) that they increase PA network connectivity by 7.05%. Our results demonstrate the unique and significant contributions that PPAs can make to the conservation estate and that PPAs deserve more attention, recognition and resources for better design and implementation. Exploring more than 17,000 privately protected areas in 15 countries across 5 continents, the authors identify the contribution of this kind of protected area to global conservation efforts and identify the roles they can play to achieving new biodiversity conservation targets.},
  archive      = {J_NATEE},
  author       = {Palfrey, Rachel and Oldekop, Johan A. and Holmes, George},
  doi          = {10.1038/s41559-022-01715-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {730-737},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Privately protected areas increase global protected area coverage and connectivity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Extinction, coextinction and colonization dynamics in
plant–hummingbird networks under climate change. <em>NATEE</em>,
<em>6</em>(6), 720–729. (<a
href="https://doi.org/10.1038/s41559-022-01693-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Climate-driven range shifts may cause local extinctions, while the accompanying loss of biotic interactions may trigger secondary coextinctions. At the same time, climate change may facilitate colonizations from regional source pools, balancing out local species loss. At present, how these extinction–coextinction–colonization dynamics affect biological communities under climate change is poorly understood. Using 84 communities of interacting plants and hummingbirds, we simulated patterns in climate-driven extinctions, coextinctions and colonizations under future climate change scenarios. Our simulations showed clear geographic discrepancies in the communities’ vulnerability to climate change. Andean communities were the least affected by future climate change, as they experienced few climate-driven extinctions and coextinctions while having the highest colonization potential. In North America and lowland South America, communities had many climate-driven extinctions and few colonization events. Meanwhile, the pattern of coextinction was highly dependent on the configuration of networks formed by interacting hummingbirds and plants. Notably, North American communities experienced proportionally fewer coextinctions than other regions because climate-driven extinctions here primarily affected species with peripheral network roles. Moreover, coextinctions generally decreased in communities where species have few overlapping interactions, that is, communities with more complementary specialized and modular networks. Together, these results highlight that we should not expect colonizations to adequately balance out local extinctions in the most vulnerable ecoregions. Using 84 plant–hummingbird networks from across the Americas, the authors show that the balance between climate change-induced extinction, coextinction and colonization varies both with geography and network structure.},
  archive      = {J_NATEE},
  author       = {Sonne, Jesper and Maruyama, Pietro K. and Martín González, Ana M. and Rahbek, Carsten and Bascompte, Jordi and Dalsgaard, Bo},
  doi          = {10.1038/s41559-022-01693-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {720-729},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Extinction, coextinction and colonization dynamics in plant–hummingbird networks under climate change},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Forest degradation drives widespread avian habitat and
population declines. <em>NATEE</em>, <em>6</em>(6), 709–719. (<a
href="https://doi.org/10.1038/s41559-022-01737-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In many regions of the world, forest management has reduced old forest and simplified forest structure and composition. We hypothesized that such forest degradation has resulted in long-term habitat loss for forest-associated bird species of eastern Canada (130,017 km2) which, in turn, has caused bird-population declines. Despite little change in overall forest cover, we found substantial reductions in old forest as a result of frequent clear-cutting and a broad-scale transformation to intensified forestry. Back-cast species distribution models revealed that breeding habitat loss occurred for 66% of the 54 most common species from 1985 to 2020 and was strongly associated with reduction in old age classes. Using a long-term, independent dataset, we found that habitat amount predicted population size for 94% of species, and habitat loss was associated with population declines for old-forest species. Forest degradation may therefore be a primary cause of biodiversity decline in managed forest landscapes. A study in eastern Canada finds that forest-management strategies that lead to simplified forest structure and composition have resulted in loss of breeding habitat and associated population losses for many bird species.},
  archive      = {J_NATEE},
  author       = {Betts, Matthew G. and Yang, Zhiqiang and Hadley, Adam S. and Smith, Adam C. and Rousseau, Josée S. and Northrup, Joseph M. and Nocera, Joseph J. and Gorelick, Noel and Gerber, Brian D.},
  doi          = {10.1038/s41559-022-01737-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {709-719},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Forest degradation drives widespread avian habitat and population declines},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Biological trade-offs underpin coral reef ecosystem
functioning. <em>NATEE</em>, <em>6</em>(6), 701–708. (<a
href="https://doi.org/10.1038/s41559-022-01710-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human impact increasingly alters global ecosystems, often reducing biodiversity and disrupting the provision of essential ecosystem services to humanity. Therefore, preserving ecosystem functioning is a critical challenge of the twenty-first century. Coral reefs are declining worldwide due to the pervasive effects of climate change and intensive fishing, and although research on coral reef ecosystem functioning has gained momentum, most studies rely on simplified proxies, such as fish biomass. This lack of quantitative assessments of multiple process-based ecosystem functions hinders local and regional conservation efforts. Here we combine global coral reef fish community surveys and bioenergetic models to quantify five key ecosystem functions mediated by coral reef fishes. We show that functions exhibit critical trade-offs driven by varying community structures, such that no community can maximize all functions. Furthermore, functions are locally dominated by few species, but the identity of dominant species substantially varies at the global scale. In fact, half of the 1,110 species in our dataset are functionally dominant in at least one location. Our results reinforce the need for a nuanced, locally tailored approach to coral reef conservation that considers multiple ecological functions beyond the effect of standing stock biomass. Integrating bioenergetic models and global coral reef fish community surveys, the authors show that there are functional trade-offs, meaning that no community can maximize all functions, and that dominant species underpin local functions, but their identity varies geographically.},
  archive      = {J_NATEE},
  author       = {Schiettekatte, Nina M. D. and Brandl, Simon J. and Casey, Jordan M. and Graham, Nicholas A. J. and Barneche, Diego R. and Burkepile, Deron E. and Allgeier, Jacob E. and Arias-Gonzaléz, Jesús E. and Edgar, Graham J. and Ferreira, Carlos E. L. and Floeter, Sergio R. and Friedlander, Alan M. and Green, Alison L. and Kulbicki, Michel and Letourneur, Yves and Luiz, Osmar J. and Mercière, Alexandre and Morat, Fabien and Munsterman, Katrina S. and Rezende, Enrico L. and Rodríguez‐Zaragoza, Fabian A. and Stuart-Smith, Rick D. and Vigliola, Laurent and Villéger, Sébastien and Parravicini, Valeriano},
  doi          = {10.1038/s41559-022-01710-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {701-708},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biological trade-offs underpin coral reef ecosystem functioning},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Complexity–stability trade-off in empirical microbial
ecosystems. <em>NATEE</em>, <em>6</em>(6), 693–700. (<a
href="https://doi.org/10.1038/s41559-022-01745-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {May’s stability theory, which holds that large ecosystems can be stable up to a critical level of complexity, a product of the number of resident species and the intensity of their interactions, has been a central paradigm in theoretical ecology. So far, however, empirically demonstrating this theory in real ecological systems has been a long-standing challenge with inconsistent results. Especially, it is unknown whether this theory is pertinent in the rich and complex communities of natural microbiomes, mainly due to the challenge of reliably reconstructing such large ecological interaction networks. Here we introduce a computational framework for estimating an ecosystem’s complexity without relying on a priori knowledge of its underlying interaction network. By applying this method to human-associated microbial communities from different body sites and sponge-associated microbial communities from different geographical locations, we found that in both cases the communities display a pronounced trade-off between the number of species and their effective connectance. These results suggest that natural microbiomes are shaped by stability constraints, which limit their complexity. After developing a method to infer network complexity without knowledge of underlying species interactions, the authors show that, in several types of microbial communities, an ecosystem’s stability constrains its complexity.},
  archive      = {J_NATEE},
  author       = {Yonatan, Yogev and Amit, Guy and Friedman, Jonathan and Bashan, Amir},
  doi          = {10.1038/s41559-022-01745-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {693-700},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Complexity–stability trade-off in empirical microbial ecosystems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Anthropogenic disruptions to longstanding patterns of
trophic-size structure in vertebrates. <em>NATEE</em>, <em>6</em>(6),
684–692. (<a href="https://doi.org/10.1038/s41559-022-01726-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diet and body mass are inextricably linked in vertebrates: while herbivores and carnivores have converged on much larger sizes, invertivores and omnivores are, on average, much smaller, leading to a roughly U-shaped relationship between body size and trophic guild. Although this U-shaped trophic-size structure is well documented in extant terrestrial mammals, whether this pattern manifests across diverse vertebrate clades and biomes is unknown. Moreover, emergence of the U-shape over geological time and future persistence are unknown. Here we compiled a comprehensive dataset of diet and body size spanning several vertebrate classes and show that the U-shaped pattern is taxonomically and biogeographically universal in modern vertebrate groups, except for marine mammals and seabirds. We further found that, for terrestrial mammals, this U-shape emerged by the Palaeocene and has thus persisted for at least 66 million years. Yet disruption of this fundamental trophic-size structure in mammals appears likely in the next century, based on projected extinctions. Actions to prevent declines in the largest animals will sustain the functioning of Earth’s wild ecosystems and biomass energy distributions that have persisted through deep time. Analysis of diet and body size in terrestrial and aquatic vertebrates shows that a U-shaped relationship between body size and trophic guild prevails across extant vertebrates with the exception of marine mammals and seabirds. Analysis of fossil data shows that, for terrestrial mammals, this pattern has persisted for at least 66 million years, despite anthropogenic perturbance, which may have greater effects in the next centuries.},
  archive      = {J_NATEE},
  author       = {Cooke, Rob and Gearty, William and Chapman, Abbie S. A. and Dunic, Jillian and Edgar, Graham J. and Lefcheck, Jonathan S. and Rilov, Gil and McClain, Craig R. and Stuart-Smith, Rick D. and Kathleen Lyons, S. and Bates, Amanda E.},
  doi          = {10.1038/s41559-022-01726-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {684-692},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Anthropogenic disruptions to longstanding patterns of trophic-size structure in vertebrates},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Potential negative effects of ocean afforestation on
offshore ecosystems. <em>NATEE</em>, <em>6</em>(6), 675–683. (<a
href="https://doi.org/10.1038/s41559-022-01722-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our scientific understanding of climate change makes clear the necessity for both emission reduction and carbon dioxide removal (CDR). The ocean with its large surface area, great depths and long coastlines is central to developing CDR approaches commensurate with the scale needed to limit warming to below 2 °C. Many proposed marine CDR approaches rely on spatial upscaling along with enhancement and/or acceleration of the rates of naturally occurring processes. One such approach is ‘ocean afforestation’, which involves offshore transport and concurrent growth of nearshore macroalgae (seaweed), followed by their export into the deep ocean. The purposeful occupation for months of open ocean waters by macroalgae, which do not naturally occur there, will probably affect offshore ecosystems through a range of biological threats, including altered ocean chemistry and changed microbial physiology and ecology. Here, we present model simulations of ocean afforestation and link these to lessons from other examples of offshore dispersal, including rafting plastic debris, and discuss the ramifications for offshore ecosystems. We explore what additional metrics are required to assess the ecological implications of this proposed CDR. In our opinion, these ecological metrics must have equal weight to CDR capacity in the development of initial trials, pilot studies and potential licensing. Ocean afforestation is a proposed method for large-scale carbon dioxide removal, involving exporting rafts of nearshore macroalgae to the open ocean for long-term occupation and then sinking. In this Perspective, the authors caution that this approach has multiple potential ramifications for ocean chemistry and ecology.},
  archive      = {J_NATEE},
  author       = {Boyd, Philip W. and Bach, Lennart T. and Hurd, Catriona L. and Paine, Ellie and Raven, John A. and Tamsitt, Veronica},
  doi          = {10.1038/s41559-022-01722-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {675-683},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Potential negative effects of ocean afforestation on offshore ecosystems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Adapting to vaccination. <em>NATEE</em>, <em>6</em>(6),
673–674. (<a href="https://doi.org/10.1038/s41559-022-01748-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Simultaneous evolution of vaccine-induced immune escape and virulence leads to different evolutionary end points, depending on the type of vaccine-induced protection.},
  archive      = {J_NATEE},
  author       = {Bernhauerová, Veronika},
  doi          = {10.1038/s41559-022-01748-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {673-674},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Adapting to vaccination},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Weaklings make trouble. <em>NATEE</em>, <em>6</em>(6),
671–672. (<a href="https://doi.org/10.1038/s41559-022-01739-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of phenotypic skew — the asymmetrical distributions of traits — explores how it can bias estimates of inheritance and selection, and how to correct for those biases.},
  archive      = {J_NATEE},
  author       = {Houle, David},
  doi          = {10.1038/s41559-022-01739-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {671-672},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Weaklings make trouble},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Functional trade-offs in fish communities. <em>NATEE</em>,
<em>6</em>(6), 669–670. (<a
href="https://doi.org/10.1038/s41559-022-01706-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combining pantropical fish community surveys with bioenergetic models has revealed the global distribution of reef-fish ecosystem functions, and that trade-offs linked to demographic and trophic structure prevent any community from maximizing all functions simultaneously.},
  archive      = {J_NATEE},
  author       = {McLean, Matthew},
  doi          = {10.1038/s41559-022-01706-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {669-670},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Functional trade-offs in fish communities},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). How diverse ecosystems remain stable. <em>NATEE</em>,
<em>6</em>(6), 667–668. (<a
href="https://doi.org/10.1038/s41559-022-01758-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A computational method that negates the need to directly measure species interactions provides evidence in support of classic theory, stating that microbial communities with higher diversity remain stable as long they have low complexity and weaker interactions.},
  archive      = {J_NATEE},
  author       = {Goyal, Akshit},
  doi          = {10.1038/s41559-022-01758-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {667-668},
  shortjournal = {Nat. Ecol. Evol},
  title        = {How diverse ecosystems remain stable},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Multicellularity gets real. <em>NATEE</em>, <em>6</em>(6),
666. (<a href="https://doi.org/10.1038/s41559-022-01765-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Goymer, Patrick},
  doi          = {10.1038/s41559-022-01765-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {666},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Multicellularity gets real},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Adapting to city life. <em>NATEE</em>, <em>6</em>(6), 665.
(<a href="https://doi.org/10.1038/s41559-022-01760-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-022-01760-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {665},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Adapting to city life},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Signs of ageing. <em>NATEE</em>, <em>6</em>(6), 664. (<a
href="https://doi.org/10.1038/s41559-022-01751-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-022-01751-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {664},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Signs of ageing},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Playing god with evolution. <em>NATEE</em>, <em>6</em>(6),
662–663. (<a href="https://doi.org/10.1038/s41559-022-01736-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {West, Stuart},
  doi          = {10.1038/s41559-022-01736-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {662-663},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Playing god with evolution},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Matthew j. G. Gage (1967–2022). <em>NATEE</em>,
<em>6</em>(6), 660–661. (<a
href="https://doi.org/10.1038/s41559-022-01762-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researcher who studied fundamental questions about sexual selection, and an inspiring and kind colleague and friend.},
  archive      = {J_NATEE},
  author       = {Chapman, Tracey and Stockley, Paula},
  doi          = {10.1038/s41559-022-01762-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {660-661},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Matthew j. g. gage (1967–2022)},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Isaiah odhiambo nengo (1961–2022). <em>NATEE</em>,
<em>6</em>(6), 659. (<a
href="https://doi.org/10.1038/s41559-022-01741-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Primate palaeontologist and passionate advocate for diversity in human origins research.},
  archive      = {J_NATEE},
  author       = {Rowan, John and Princehouse, Patricia and Kinyanjui, Rahab N. and Uno, Kevin T.},
  doi          = {10.1038/s41559-022-01741-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {659},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Isaiah odhiambo nengo (1961–2022)},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Making forest data fair and open. <em>NATEE</em>,
<em>6</em>(6), 656–658. (<a
href="https://doi.org/10.1038/s41559-022-01738-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data on tropical forests are in high demand. But ground forest measurements are hard to sustain and the people who make them are extremely disadvantaged compared to those who use them. We propose a new approach to forest data that focuses on the needs of data originators, and ensures users and funders contribute properly.},
  archive      = {J_NATEE},
  author       = {de Lima, Renato A. F. and Phillips, Oliver L. and Duque, Alvaro and Tello, J. Sebastian and Davies, Stuart J. and de Oliveira, Alexandre Adalardo and Muller, Sandra and Honorio Coronado, Euridice N. and Vilanova, Emilio and Cuni-Sanchez, Aida and Baker, Timothy R. and Ryan, Casey M. and Malizia, Agustina and Lewis, Simon L. and ter Steege, Hans and Ferreira, Joice and Marimon, Beatriz Schwantes and Luu, Hong Truong and Imani, Gerard and Arroyo, Luzmila and Blundo, Cecilia and Kenfack, David and Sainge, Moses N. and Sonké, Bonaventure and Vásquez, Rodolfo},
  doi          = {10.1038/s41559-022-01738-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {656-658},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Making forest data fair and open},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Groundwater extraction poses extreme threat to doñana world
heritage site. <em>NATEE</em>, <em>6</em>(6), 654–655. (<a
href="https://doi.org/10.1038/s41559-022-01763-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Camacho, Carlos and Negro, Juan J. and Elmberg, Johan and Fox, Anthony D. and Nagy, Szabolcs and Pain, Deborah J. and Green, Andy J.},
  doi          = {10.1038/s41559-022-01763-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {654-655},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Groundwater extraction poses extreme threat to doñana world heritage site},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Old forests are not replaceable. <em>NATEE</em>,
<em>6</em>(6), 653. (<a
href="https://doi.org/10.1038/s41559-022-01806-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Governments around the world are too slow and too weak in their commitments to stop deforestation. And promises of restoration will not make up for the loss of old forests.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01806-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {653},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Old forests are not replaceable},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sociality predicts orangutan vocal phenotype.
<em>NATEE</em>, <em>6</em>(5), 644–652. (<a
href="https://doi.org/10.1038/s41559-022-01689-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In humans, individuals’ social setting determines which and how language is acquired. Social seclusion experiments show that sociality also guides vocal development in songbirds and marmoset monkeys, but absence of similar great ape data has been interpreted as support to saltational notions for language origin, even if such laboratorial protocols are unethical with great apes. Here we characterize the repertoire entropy of orangutan individuals and show that in the wild, different degrees of sociality across populations are associated with different ‘vocal personalities’ in the form of distinct regimes of alarm call variants. In high-density populations, individuals are vocally more original and acoustically unpredictable but new call variants are short lived, whereas individuals in low-density populations are more conformative and acoustically consistent but also exhibit more complex call repertoires. Findings provide non-invasive evidence that sociality predicts vocal phenotype in a wild great ape. They prove false hypotheses that discredit great apes as having hardwired vocal development programmes and non-plastic vocal behaviour. Social settings mould vocal output in hominids besides humans. Analysis of wild orangutan calls demonstrates that different degrees of sociality across populations are associated with different ‘vocal personalities’.},
  archive      = {J_NATEE},
  author       = {Lameira, Adriano R. and Santamaría-Bonfil, Guillermo and Galeone, Deborah and Gamba, Marco and Hardus, Madeleine E. and Knott, Cheryl D. and Morrogh-Bernard, Helen and Nowak, Matthew G. and Campbell-Smith, Gail and Wich, Serge A.},
  doi          = {10.1038/s41559-022-01689-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {644-652},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sociality predicts orangutan vocal phenotype},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic signatures of high-altitude adaptation and
chromosomal polymorphism in geladas. <em>NATEE</em>, <em>6</em>(5),
630–643. (<a href="https://doi.org/10.1038/s41559-022-01703-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Primates have adapted to numerous environments and lifestyles but very few species are native to high elevations. Here we investigated high-altitude adaptations in the gelada (Theropithecus gelada), a monkey endemic to the Ethiopian Plateau. We examined genome-wide variation in conjunction with measurements of haematological and morphological traits. Our new gelada reference genome is highly intact and assembled at chromosome-length levels. Unexpectedly, we identified a chromosomal polymorphism in geladas that could potentially contribute to reproductive barriers between populations. Compared with baboons at low altitude, we found that high-altitude geladas exhibit significantly expanded chest circumferences, potentially allowing for greater lung surface area for increased oxygen diffusion. We identified gelada-specific amino acid substitutions in the alpha-chain subunit of adult haemoglobin but found that gelada haemoglobin does not exhibit markedly altered oxygenation properties compared with lowland primates. We also found that geladas at high altitude do not exhibit elevated blood haemoglobin concentrations, in contrast to the normal acclimatization response to hypoxia in lowland primates. The absence of altitude-related polycythaemia suggests that geladas are able to sustain adequate tissue-oxygen delivery despite environmental hypoxia. Finally, we identified numerous genes and genomic regions exhibiting accelerated rates of evolution, as well as gene families exhibiting expansions in the gelada lineage, potentially reflecting altitude-related selection. Our findings lend insight into putative mechanisms of high-altitude adaptation while suggesting promising avenues for functional hypoxia research. Analysing a newly assembled genome of the gelada, an endemic Ethiopian monkey that lives in high altitudes, the authors identify a novel karyotype and genomic elements associated with high-altitude adaptation.},
  archive      = {J_NATEE},
  author       = {Chiou, Kenneth L. and Janiak, Mareike C. and Schneider-Crease, India A. and Sen, Sharmi and Ayele, Ferehiwot and Chuma, Idrissa S. and Knauf, Sascha and Lemma, Alemayehu and Signore, Anthony V. and D’Ippolito, Anthony M. and Abebe, Belayneh and Haile, Abebaw Azanaw and Kebede, Fanuel and Fashing, Peter J. and Nguyen, Nga and McCann, Colleen and Houck, Marlys L. and Wall, Jeffrey D. and Burrell, Andrew S. and Bergey, Christina M. and Rogers, Jeffrey and Phillips-Conroy, Jane E. and Jolly, Clifford J. and Melin, Amanda D. and Storz, Jay F. and Lu, Amy and Beehner, Jacinta C. and Bergman, Thore J. and Snyder-Mackler, Noah},
  doi          = {10.1038/s41559-022-01703-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {630-643},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genomic signatures of high-altitude adaptation and chromosomal polymorphism in geladas},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Latitudinal gradients in avian colourfulness.
<em>NATEE</em>, <em>6</em>(5), 622–629. (<a
href="https://doi.org/10.1038/s41559-022-01714-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It has long been suggested that tropical species are generally more colourful than temperate species, but whether latitudinal gradients in organismal colourfulness exist remains controversial. Here we quantify global latitudinal trends in colourfulness (within-individual colour diversity) by collating and analysing a photographic dataset of whole-body plumage reflectance information for &amp;gt;4,500 species of passerine birds. We show that male and female birds of tropical passerine species are generally more colourful than their temperate counterparts, both on average and in the extreme. We also show that these geographic gradients can be explained in part by the effects of several latitude-related factors related to classic hypotheses for climatic and ecological determinants of organismal colourfulness. Taken together, our results reveal that species’ colourfulness peaks in the tropics for passerine birds, confirming the existence of a long-suspected yet hitherto elusive trend in the distribution of global biodiversity. After generating a dataset on plumage colouration for over 4,500 bird species, the authors show that tropical species are more colourful than temperate species, confirming a long-held but difficult-to-prove belief.},
  archive      = {J_NATEE},
  author       = {Cooney, Christopher R. and He, Yichen and Varley, Zoë K. and Nouri, Lara O. and Moody, Christopher J. A. and Jardine, Michael D. and Liker, András and Székely, Tamás and Thomas, Gavin H.},
  doi          = {10.1038/s41559-022-01714-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {622-629},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Latitudinal gradients in avian colourfulness},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Snake-like limb loss in a carboniferous amniote.
<em>NATEE</em>, <em>6</em>(5), 614–621. (<a
href="https://doi.org/10.1038/s41559-022-01698-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Among living tetrapods, many lineages have converged on a snake-like body plan, where extreme axial elongation is accompanied by reduction or loss of paired limbs. However, when and how this adaptive body plan first evolved in amniotes remains poorly understood. Here, we provide insights into this question by reporting on a new taxon of molgophid recumbirostran, Nagini mazonense gen. et sp. nov., from the Francis Creek Shale (309–307 million years ago) of Illinois, United States, that exhibits extreme axial elongation and corresponding limb reduction. The molgophid lacks entirely the forelimb and pectoral girdle, thus representing the earliest occurrence of complete loss of a limb in a taxon recovered phylogenetically within amniotes. This forelimb-first limb reduction is consistent with the pattern of limb reduction that is seen in modern snakes and contrasts with the hindlimb-first reduction process found in many other tetrapod groups. Our findings suggest that a snake-like limb-reduction mechanism may be operating more broadly across the amniote tree. A new taxon of molgophid recumbirostran from the Carboniferous of Illinois, Nagini mazonense, suggests that the forelimb-first pattern of limb reduction that characterizes modern snakes also occurs early on in amniote evolution.},
  archive      = {J_NATEE},
  author       = {Mann, Arjan and Pardo, Jason D. and Maddin, Hillary C.},
  doi          = {10.1038/s41559-022-01698-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {614-621},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Snake-like limb loss in a carboniferous amniote},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Embryonic muscle splitting patterns reveal homologies of
amniote forelimb muscles. <em>NATEE</em>, <em>6</em>(5), 604–613. (<a
href="https://doi.org/10.1038/s41559-022-01699-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Limb muscles are remarkably complex and evolutionarily labile. Although their anatomy is of great interest for studies of the evolution of form and function, their homologies among major amniote clades have remained obscure. Studies of adult musculature are inconclusive owing to the highly derived morphology of modern amniote limbs but correspondences become increasingly evident earlier in ontogeny. We followed the embryonic development of forelimb musculature in representatives of six major amniote clades and found, contrary to current consensus, that these early splitting patterns are highly conserved across Amniota. Muscle mass cleavage patterns and topology are highly conserved in reptiles including birds, irrespective of their skeletal modifications: the avian flight apparatus results from slight early topological modifications that are exaggerated during ontogeny. Therian mammals, while conservative in their cleavage patterns, depart drastically from the ancestral amniote musculoskeletal organization in terms of topology. These topological changes occur through extension, translocation and displacement of muscle groups later in development. Overall, the simplicity underlying the apparent complexity of forelimb muscle development allows us to resolve conflicting hypotheses about homology and to trace the history of each individual forelimb muscle throughout the amniote radiations. The evolutionary history of amniote limb muscles remains unclear. Here the authors study the embryonic development of forelimb musculature in six species of amniotes and show that early splitting patterns of embryonic muscle masses are highly conserved across Amniota.},
  archive      = {J_NATEE},
  author       = {Smith-Paredes, Daniel and Vergara-Cereghino, Miccaella E. and Lord, Arianna and Moses, Malcolm M. and Behringer, Richard R. and Bhullar, Bhart-Anjan S.},
  doi          = {10.1038/s41559-022-01699-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {604-613},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Embryonic muscle splitting patterns reveal homologies of amniote forelimb muscles},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Co-evolution of interacting proteins through non-contacting
and non-specific mutations. <em>NATEE</em>, <em>6</em>(5), 590–603. (<a
href="https://doi.org/10.1038/s41559-022-01688-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteins often accumulate neutral mutations that do not affect current functions but can profoundly influence future mutational possibilities and functions. Understanding such hidden potential has major implications for protein design and evolutionary forecasting but has been limited by a lack of systematic efforts to identify potentiating mutations. Here, through the comprehensive analysis of a bacterial toxin–antitoxin system, we identified all possible single substitutions in the toxin that enable it to tolerate otherwise interface-disrupting mutations in its antitoxin. Strikingly, the majority of enabling mutations in the toxin do not contact and promote tolerance non-specifically to many different antitoxin mutations, despite covariation in homologues occurring primarily between specific pairs of contacting residues across the interface. In addition, the enabling mutations we identified expand future mutational paths that both maintain old toxin–antitoxin interactions and form new ones. These non-specific mutations are missed by widely used covariation and machine learning methods. Identifying such enabling mutations will be critical for ensuring continued binding of therapeutically relevant proteins, such as antibodies, aimed at evolving targets. Enabling mutations do not affect current protein function but affect future mutational possibilities. The authors explore the nature of enabling mutations in a bacterial toxin–antitoxin system.},
  archive      = {J_NATEE},
  author       = {Ding, David and Green, Anna G. and Wang, Boyuan and Lite, Thuy-Lan Vo and Weinstein, Eli N. and Marks, Debora S. and Laub, Michael T.},
  doi          = {10.1038/s41559-022-01688-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {590-603},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Co-evolution of interacting proteins through non-contacting and non-specific mutations},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Selection drives convergent gene expression changes during
transitions to co-sexuality in haploid sexual systems. <em>NATEE</em>,
<em>6</em>(5), 579–589. (<a
href="https://doi.org/10.1038/s41559-022-01692-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Co-sexuality has evolved repeatedly from unisexual (dioicous) ancestors across a wide range of taxa. However, the molecular changes underpinning this important transition remain unknown, particularly in organisms with haploid sexual systems such as bryophytes, red algae and brown algae. Here we explore four independent events of emergence of co-sexuality from unisexual ancestors in brown algal clades to examine the nature, evolution and degree of convergence of gene expression changes that accompany the breakdown of dioicy. The amounts of male versus female phenotypic differences in dioicous species were not correlated with the extent of sex-biased gene expression, in stark contrast to what is observed in animals. Although sex-biased genes exhibited a high turnover rate during brown alga diversification, some of their predicted functions were conserved across species. Transitions to co-sexuality consistently involved adaptive gene expression shifts and rapid sequence evolution, particularly for male-biased genes. Gene expression in co-sexual species was more similar to that in females rather than males of related dioicous species, suggesting that co-sexuality may have arisen from ancestral females. Finally, extensive convergent gene expression changes, driven by selection, were associated with the transition to co-sexuality. Together, our observations provide insights on how co-sexual systems arise from ancestral, haploid UV sexual systems. Studying four examples of the transition to co-sexuality in brown algae, the authors show extensive convergent changes in gene expression driven by selection, and greater similarity of co-sexual gene expression profiles to those of ancestral females than to those of ancestral males.},
  archive      = {J_NATEE},
  author       = {Cossard, Guillaume G. and Godfroy, Olivier and Nehr, Zofia and Cruaud, Corinne and Cock, J. Mark and Lipinska, Agnieszka P. and Coelho, Susana M.},
  doi          = {10.1038/s41559-022-01692-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {579-589},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Selection drives convergent gene expression changes during transitions to co-sexuality in haploid sexual systems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Innate immune pathways act synergistically to constrain RNA
virus evolution in drosophila melanogaster. <em>NATEE</em>,
<em>6</em>(5), 565–578. (<a
href="https://doi.org/10.1038/s41559-022-01697-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Host–pathogen interactions impose recurrent selective pressures that lead to constant adaptation and counter-adaptation in both competing species. Here, we sought to study this evolutionary arms-race and assessed the impact of the innate immune system on viral population diversity and evolution, using Drosophila melanogaster as model host and its natural pathogen Drosophila C virus (DCV). We isogenized eight fly genotypes generating animals defective for RNAi, Imd and Toll innate immune pathways as well as pathogen-sensing and gut renewal pathways. Wild-type or mutant flies were then orally infected with DCV and the virus was serially passaged ten times via reinfection in naive flies. Viral population diversity was studied after each viral passage by high-throughput sequencing and infection phenotypes were assessed at the beginning and at the end of the evolution experiment. We found that the absence of any of the various immune pathways studied increased viral genetic diversity while attenuating virulence. Strikingly, these effects were observed in a range of host factors described as having mainly antiviral or antibacterial functions. Together, our results indicate that the innate immune system as a whole and not specific antiviral defence pathways in isolation, generally constrains viral diversity and evolution. The innate immune system affects viral evolution. Here, the authors use experimental evolution to ask how different immune pathways impact viral genetic diversity and virulence using Drosophila melanogaster and its natural pathogen Drosophila C virus.},
  archive      = {J_NATEE},
  author       = {Mongelli, Vanesa and Lequime, Sebastian and Kousathanas, Athanasios and Gausson, Valérie and Blanc, Hervé and Nigg, Jared and Quintana-Murci, Lluis and Elena, Santiago F. and Saleh, Maria-Carla},
  doi          = {10.1038/s41559-022-01697-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {565-578},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Innate immune pathways act synergistically to constrain RNA virus evolution in drosophila melanogaster},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Intra- and interpopulation transposition of mobile genetic
elements driven by antibiotic selection. <em>NATEE</em>, <em>6</em>(5),
555–564. (<a href="https://doi.org/10.1038/s41559-022-01705-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The spread of genes encoding antibiotic resistance is often mediated by horizontal gene transfer (HGT). Many of these genes are associated with transposons, a type of mobile genetic element that can translocate between the chromosome and plasmids. It is widely accepted that the translocation of antibiotic resistance genes onto plasmids potentiates their spread by HGT. However, it is unclear how this process is modulated by environmental factors, especially antibiotic treatment. To address this issue, we asked whether antibiotic exposure would select for the transposition of resistance genes from chromosomes onto plasmids and, if so, whether antibiotic concentration could tune the distribution of resistance genes between chromosomes and plasmids. We addressed these questions by analysing the transposition dynamics of synthetic and natural transposons that encode resistance to different antibiotics. We found that stronger antibiotic selection leads to a higher fraction of cells carrying the resistance on plasmids because the increased copy number of resistance genes on multicopy plasmids leads to higher expression of those genes and thus higher cell survival when facing antibiotic selection. Once they have transposed to plasmids, antibiotic resistance genes are primed for rapid spread by HGT. Our results provide quantitative evidence for a mechanism by which antibiotic selection accelerates the spread of antibiotic resistance in microbial communities. Analysing the transposition dynamics of synthetic and natural transposons that encode resistance to different antibiotics, the authors show that stronger antibiotic selection leads to a higher fraction of cells carrying the resistance on plasmids.},
  archive      = {J_NATEE},
  author       = {Yao, Yi and Maddamsetti, Rohan and Weiss, Andrea and Ha, Yuanchi and Wang, Teng and Wang, Shangying and You, Lingchong},
  doi          = {10.1038/s41559-022-01705-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {555-564},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Intra- and interpopulation transposition of mobile genetic elements driven by antibiotic selection},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Changes in precipitation patterns can destabilize plant
species coexistence via changes in plant–soil feedback. <em>NATEE</em>,
<em>6</em>(5), 546–554. (<a
href="https://doi.org/10.1038/s41559-022-01700-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Climate change can alter species coexistence through changes in biotic interactions. By describing reciprocal interactions between plants and soil microbes, plant–soil feedback (PSF) has emerged as a powerful framework for predicting plant species coexistence and community dynamics, but little is known about how PSF will respond to changing climate conditions. Hence, the context dependency of PSF has recently gained attention. Water availability is a major driver of all biotic interactions, and it is expected that precipitation patterns will change with ongoing climate change. We tested how soil water content affects PSF by conducting a full factorial pairwise PSF experiment using eight plant species common to southeastern United States coastal prairies under three watering treatments. We found coexistence-stabilizing negative PSF at drier-than-average conditions shifted to coexistence-destabilizing positive PSF under wetter-than-average conditions. A simulation model parameterized with the experimental results supports the prediction that more positive PSF accelerates the erosion of diversity within communities while decreasing the predictability in plant community composition. Our results underline the importance of considering environmental context dependency of PSF in light of a rapidly changing climate. The authors use experiments and model simulations to show that among a community of eight grassland plant species, pairwise plant–soil feedbacks are more likely to be positive in high-precipitation conditions and negative in dry conditions.},
  archive      = {J_NATEE},
  author       = {Dudenhöffer, Jan-Hendrik and Luecke, Noah C. and Crawford, Kerri M.},
  doi          = {10.1038/s41559-022-01700-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {546-554},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Changes in precipitation patterns can destabilize plant species coexistence via changes in plant–soil feedback},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Field experiments underestimate aboveground biomass
response to drought. <em>NATEE</em>, <em>6</em>(5), 540–545. (<a
href="https://doi.org/10.1038/s41559-022-01685-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers use both experiments and observations to study the impacts of climate change on ecosystems, but results from these contrasting approaches have not been systematically compared for droughts. Using a meta-analysis and accounting for potential confounding factors, we demonstrate that aboveground biomass responded only about half as much to experimentally imposed drought events as to natural droughts. Our findings indicate that experimental results may underestimate climate change impacts and highlight the need to integrate results across approaches. In a meta-analysis comparing experimental versus observational studies of aboveground biomass responses to drought in grasslands, the authors show that effect sizes in experiments are 53% weaker than in observational studies, suggesting that experiments are underestimating drought responses.},
  archive      = {J_NATEE},
  author       = {Kröel-Dulay, György and Mojzes, Andrea and Szitár, Katalin and Bahn, Michael and Batáry, Péter and Beier, Claus and Bilton, Mark and De Boeck, Hans J. and Dukes, Jeffrey S. and Estiarte, Marc and Holub, Petr and Jentsch, Anke and Schmidt, Inger Kappel and Kreyling, Juergen and Reinsch, Sabine and Larsen, Klaus Steenberg and Sternberg, Marcelo and Tielbörger, Katja and Tietema, Albert and Vicca, Sara and Peñuelas, Josep},
  doi          = {10.1038/s41559-022-01685-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {540-545},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Field experiments underestimate aboveground biomass response to drought},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reply to: Logging elevated the probability of high-severity
fire in the 2019–20 australian forest fires. <em>NATEE</em>,
<em>6</em>(5), 536–539. (<a
href="https://doi.org/10.1038/s41559-022-01716-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bowman, David M. J. S. and Williamson, Grant J. and Gibson, Rebecca K. and Bradstock, Ross A. and Keenan, Rodney J.},
  doi          = {10.1038/s41559-022-01716-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {536-539},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Logging elevated the probability of high-severity fire in the 2019–20 australian forest fires},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Logging elevated the probability of high-severity fire in
the 2019–20 australian forest fires. <em>NATEE</em>, <em>6</em>(5),
533–535. (<a href="https://doi.org/10.1038/s41559-022-01717-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Lindenmayer, David B. and Zylstra, Phil and Kooyman, Robert and Taylor, Chris and Ward, Michelle and Watson, James E. M.},
  doi          = {10.1038/s41559-022-01717-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {533-535},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Logging elevated the probability of high-severity fire in the 2019–20 australian forest fires},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Eukaryogenesis and oxygen in earth history. <em>NATEE</em>,
<em>6</em>(5), 520–532. (<a
href="https://doi.org/10.1038/s41559-022-01733-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The endosymbiotic origin of mitochondria during eukaryogenesis has long been viewed as an adaptive response to the oxygenation of Earth’s surface environment, presuming a fundamentally aerobic lifestyle for the free-living bacterial ancestors of mitochondria. This oxygen-centric view has been robustly challenged by recent advances in the Earth and life sciences. While the permanent oxygenation of the atmosphere above trace concentrations is now thought to have occurred 2.2 billion years ago, large parts of the deep ocean remained anoxic until less than 0.5 billion years ago. Neither fossils nor molecular clocks correlate the origin of mitochondria, or eukaryogenesis more broadly, to either of these planetary redox transitions. Instead, mitochondria-bearing eukaryotes are consistently dated to between these two oxygenation events, during an interval of pervasive deep-sea anoxia and variable surface-water oxygenation. The discovery and cultivation of the Asgard archaea has reinforced metabolic evidence that eukaryogenesis was initially mediated by syntrophic H2 exchange between an archaeal host and an α-proteobacterial symbiont living under anoxia. Together, these results temporally, spatially and metabolically decouple the earliest stages of eukaryogenesis from the oxygen content of the surface ocean and atmosphere. Rather than reflecting the ancestral metabolic state, obligate aerobiosis in eukaryotes is most probably derived, having only become globally widespread over the past 1 billion years as atmospheric oxygen approached modern levels. For decades, the origin of mitochondria during eukaryogenesis has been viewed as a response to Earth’s oxygenation, but this has been challenged by more recent research. Here, the authors review recent literature, concluding that eukaryogenesis and the rise of oxygen were decoupled, and obligate aerosis in eukaryotes has only become widespread in the past 1 billion years},
  archive      = {J_NATEE},
  author       = {Mills, Daniel B. and Boyle, Richard A. and Daines, Stuart J. and Sperling, Erik A. and Pisani, Davide and Donoghue, Philip C. J. and Lenton, Timothy M.},
  doi          = {10.1038/s41559-022-01733-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {520-532},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Eukaryogenesis and oxygen in earth history},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrating remote sensing with ecology and evolution to
advance biodiversity conservation. <em>NATEE</em>, <em>6</em>(5),
506–519. (<a href="https://doi.org/10.1038/s41559-022-01702-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Remote sensing has transformed the monitoring of life on Earth by revealing spatial and temporal dimensions of biological diversity through structural, compositional and functional measurements of ecosystems. Yet, many aspects of Earth’s biodiversity are not directly quantified by reflected or emitted photons. Inclusive integration of remote sensing with field-based ecology and evolution is needed to fully understand and preserve Earth’s biodiversity. In this Perspective, we argue that multiple data types are necessary for almost all draft targets set by the Convention on Biological Diversity. We examine five key topics in biodiversity science that can be advanced by integrating remote sensing with in situ data collection from field sampling, experiments and laboratory studies to benefit conservation. Lowering the barriers for bringing these approaches together will require global-scale collaboration. This Perspective discusses how the latest advances in remote sensing can be used to answer basic ecological and evolutionary questions, as well as contribute to important biodiversity monitoring.},
  archive      = {J_NATEE},
  author       = {Cavender-Bares, Jeannine and Schneider, Fabian D. and Santos, Maria João and Armstrong, Amanda and Carnaval, Ana and Dahlin, Kyla M. and Fatoyinbo, Lola and Hurtt, George C. and Schimel, David and Townsend, Philip A. and Ustin, Susan L. and Wang, Zhihui and Wilson, Adam M.},
  doi          = {10.1038/s41559-022-01702-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {506-519},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Integrating remote sensing with ecology and evolution to advance biodiversity conservation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Social orangutans have varied vocal personalities.
<em>NATEE</em>, <em>6</em>(5), 504–505. (<a
href="https://doi.org/10.1038/s41559-022-01718-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Taking advantage of natural variation present in six populations of wild orangutans, a new study correlates population density with multiple facets of individuals’ vocal phenotype and demonstrates that sociality influences vocal plasticity in great apes.},
  archive      = {J_NATEE},
  author       = {Kalan, Ammie K.},
  doi          = {10.1038/s41559-022-01718-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {504-505},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Social orangutans have varied vocal personalities},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A colourful tropical world. <em>NATEE</em>, <em>6</em>(5),
502–503. (<a href="https://doi.org/10.1038/s41559-022-01696-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A large dataset quantifying variation in bird colourfulness lends support to a long-hypothesized latitudinal diversity gradient in animal colouration.},
  archive      = {J_NATEE},
  author       = {Diniz-Filho, José Alexandre Felizola},
  doi          = {10.1038/s41559-022-01696-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {502-503},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A colourful tropical world},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Characterizing the landscape of evolvability.
<em>NATEE</em>, <em>6</em>(5), 500–501. (<a
href="https://doi.org/10.1038/s41559-022-01731-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A framework to experimentally traverse the large space of functionally neutral variants in a toxin–antitoxin protein complex reveals insights on evolvability and entrenchment of molecular interactions.},
  archive      = {J_NATEE},
  author       = {Morcos, Faruck},
  doi          = {10.1038/s41559-022-01731-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {500-501},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Characterizing the landscape of evolvability},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The journey of bacterial genes. <em>NATEE</em>,
<em>6</em>(5), 498–499. (<a
href="https://doi.org/10.1038/s41559-022-01713-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of the dynamics of transposons that encode resistance to different antibiotics shows that the movement of genes under positive selection from the chromosome to mobile genetic elements such as plasmids can be beneficial in bacteria. Once integrated into plasmids, these genes can spread by horizontal gene transfer.},
  archive      = {J_NATEE},
  author       = {San Millan, Alvaro},
  doi          = {10.1038/s41559-022-01713-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {498-499},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The journey of bacterial genes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Water shifts the balance of coexistence. <em>NATEE</em>,
<em>6</em>(5), 496–497. (<a
href="https://doi.org/10.1038/s41559-022-01725-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimentally manipulating precipitation levels in a plant–soil feedback experiment reveals changes to the interactions between plants and soil microbes that render community dynamics less predictable under wetter conditions.},
  archive      = {J_NATEE},
  author       = {Ke, Po-Ju},
  doi          = {10.1038/s41559-022-01725-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {496-497},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Water shifts the balance of coexistence},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Finding food from phylogenies. <em>NATEE</em>,
<em>6</em>(5), 495. (<a
href="https://doi.org/10.1038/s41559-022-01727-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-022-01727-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {495},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Finding food from phylogenies},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Aquaculture effects in the wild. <em>NATEE</em>,
<em>6</em>(5), 494. (<a
href="https://doi.org/10.1038/s41559-022-01720-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-022-01720-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {494},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Aquaculture effects in the wild},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Jurassic dearc. <em>NATEE</em>, <em>6</em>(5), 493. (<a
href="https://doi.org/10.1038/s41559-022-01719-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-022-01719-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {493},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Jurassic dearc},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Fatty acid phenology. <em>NATEE</em>, <em>6</em>(5), 492.
(<a href="https://doi.org/10.1038/s41559-022-01711-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-022-01711-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {492},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fatty acid phenology},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Animal power. <em>NATEE</em>, <em>6</em>(5), 491. (<a
href="https://doi.org/10.1038/s41559-022-01732-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Shen, Sheng-Feng},
  doi          = {10.1038/s41559-022-01732-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {491},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Animal power},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Changing worlds of insects. <em>NATEE</em>, <em>6</em>(5),
489–490. (<a href="https://doi.org/10.1038/s41559-022-01729-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Roy, Helen E.},
  doi          = {10.1038/s41559-022-01729-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {489-490},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Changing worlds of insects},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Carbon parks could secure essential ecosystems for climate
stabilization. <em>NATEE</em>, <em>6</em>(5), 486–488. (<a
href="https://doi.org/10.1038/s41559-022-01695-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Achieving net-zero targets and climate stabilization will require better accounting for the immense amount of carbon naturally stored belowground. We propose ‘carbon parks’ as a conservation tool and financial instrument to protect and value carbon-rich ecosystems.},
  archive      = {J_NATEE},
  author       = {Loisel, Julie and Walenta, Jayme},
  doi          = {10.1038/s41559-022-01695-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {486-488},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Carbon parks could secure essential ecosystems for climate stabilization},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pantanal port licence would threaten the world’s largest
tropical wetland. <em>NATEE</em>, <em>6</em>(5), 484–485. (<a
href="https://doi.org/10.1038/s41559-022-01724-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Coelho-Junior, Marcondes G. and Diele-Viegas, Luisa M. and Calheiros, Debora F. and Silva Neto, Eduardo C. and Fearnside, Philip M. and Ferrante, Lucas},
  doi          = {10.1038/s41559-022-01724-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {484-485},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pantanal port licence would threaten the world’s largest tropical wetland},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improving peer review on many fronts. <em>NATEE</em>,
<em>6</em>(5), 483. (<a
href="https://doi.org/10.1038/s41559-022-01770-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is a range of approaches to improving peer review, and at this journal we encourage many of them.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01770-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {483},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Improving peer review on many fronts},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Natural selection for imprecise
vertical transmission in host–microbiota systems. <em>NATEE</em>,
<em>6</em>(4), 482. (<a
href="https://doi.org/10.1038/s41559-022-01728-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bruijning, Marjolein and Henry, Lucas P. and Forsberg, Simon K. G. and Metcalf, C. Jessica E. and Ayroles, Julien F.},
  doi          = {10.1038/s41559-022-01728-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {482},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Natural selection for imprecise vertical transmission in host–microbiota systems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Supergene origin and maintenance in atlantic cod.
<em>NATEE</em>, <em>6</em>(4), 469–481. (<a
href="https://doi.org/10.1038/s41559-022-01661-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Supergenes are sets of genes that are inherited as a single marker and encode complex phenotypes through their joint action. They are identified in an increasing number of organisms, yet their origins and evolution remain enigmatic. In Atlantic cod, four megabase-scale supergenes have been identified and linked to migratory lifestyle and environmental adaptations. Here we investigate the origin and maintenance of these four supergenes through analysis of whole-genome-sequencing data, including a new long-read-based genome assembly for a non-migratory Atlantic cod individual. We corroborate the finding that chromosomal inversions underlie all four supergenes, and we show that they originated at different times between 0.40 and 1.66 million years ago. We reveal gene flux between supergene haplotypes where migratory and stationary Atlantic cod co-occur and conclude that this gene flux is driven by gene conversion, on the basis of an increase in GC content in exchanged sites. Additionally, we find evidence for double crossover between supergene haplotypes, leading to the exchange of an ~275 kilobase fragment with genes potentially involved in adaptation to low salinity in the Baltic Sea. Our results suggest that supergenes can be maintained over long timescales in the same way as hybridizing species, through the selective purging of introduced genetic variation. Atlantic cod carries four supergenes linked to migratory lifestyle and environmental adaptations. Using whole-genome sequencing, the authors show that the genome inversions that underlie the supergenes originated at different times and show gene flux between supergene haplotypes.},
  archive      = {J_NATEE},
  author       = {Matschiner, Michael and Barth, Julia Maria Isis and Tørresen, Ole Kristian and Star, Bastiaan and Baalsrud, Helle Tessand and Brieuc, Marine Servane Ono and Pampoulie, Christophe and Bradbury, Ian and Jakobsen, Kjetill Sigurd and Jentoft, Sissel},
  doi          = {10.1038/s41559-022-01661-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {469-481},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Supergene origin and maintenance in atlantic cod},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic variation from an extinct species is retained in the
extant radiation following speciation reversal. <em>NATEE</em>,
<em>6</em>(4), 461–468. (<a
href="https://doi.org/10.1038/s41559-022-01665-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ecosystem degradation and biodiversity loss are major global challenges. When reproductive isolation between species is contingent on the interaction of intrinsic lineage traits with features of the environment, environmental change can weaken reproductive isolation and result in extinction through hybridization. By this process called speciation reversal, extinct species can leave traces in genomes of extant species through introgressive hybridization. Using historical and contemporary samples, we sequenced all four species of an Alpine whitefish radiation before and after anthropogenic lake eutrophication and the associated loss of one species through speciation reversal. Despite the extinction of this taxon, substantial fractions of its genome, including regions shaped by positive selection before eutrophication, persist within surviving species as a consequence of introgressive hybridization during eutrophication. Given the prevalence of environmental change, studying speciation reversal and its genomic consequences provides fundamental insights into evolutionary processes and informs biodiversity conservation. Genomic analysis of an Alpine whitefish radiation before and after human-driven lake eutrophication that led to the extinction of one species through hybridization shows that substantial parts of the genome of the extinct species persist within surviving species due to introgressive hybridization.},
  archive      = {J_NATEE},
  author       = {Frei, David and De-Kayne, Rishi and Selz, Oliver M. and Seehausen, Ole and Feulner, Philine G. D.},
  doi          = {10.1038/s41559-022-01665-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {461-468},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genomic variation from an extinct species is retained in the extant radiation following speciation reversal},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Domestication reprogrammed the budding yeast life cycle.
<em>NATEE</em>, <em>6</em>(4), 448–460. (<a
href="https://doi.org/10.1038/s41559-022-01671-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Domestication of plants and animals is the foundation for feeding the world human population but can profoundly alter the biology of the domesticated species. Here we investigated the effect of domestication on one of our prime model organisms, the yeast Saccharomyces cerevisiae, at a species-wide level. We tracked the capacity for sexual and asexual reproduction and the chronological life span across a global collection of 1,011 genome-sequenced yeast isolates and found a remarkable dichotomy between domesticated and wild strains. Domestication had systematically enhanced fermentative and reduced respiratory asexual growth, altered the tolerance to many stresses and abolished or impaired the sexual life cycle. The chronological life span remained largely unaffected by domestication and was instead dictated by clade-specific evolution. We traced the genetic origins of the yeast domestication syndrome using genome-wide association analysis and genetic engineering and disclosed causative effects of aneuploidy, gene presence/absence variations, copy number variations and single-nucleotide polymorphisms. Overall, we propose domestication to be the most dramatic event in budding yeast evolution, raising questions about how much domestication has distorted our understanding of the natural biology of this key model species. This study assesses the capacity for sexual and asexual reproduction and the chronological life span across 1,011 genome-sequenced budding yeast isolates and shows the remarkable impact of domestication on budding yeast evolution.},
  archive      = {J_NATEE},
  author       = {De Chiara, Matteo and Barré, Benjamin P. and Persson, Karl and Irizar, Agurtzane and Vischioni, Chiara and Khaiwal, Sakshi and Stenberg, Simon and Amadi, Onyetugo Chioma and Žun, Gašper and Doberšek, Katja and Taccioli, Cristian and Schacherer, Joseph and Petrovič, Uroš and Warringer, Jonas and Liti, Gianni},
  doi          = {10.1038/s41559-022-01671-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {448-460},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Domestication reprogrammed the budding yeast life cycle},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Population size mediates the contribution of high-rate and
large-benefit mutations to parallel evolution. <em>NATEE</em>,
<em>6</em>(4), 439–447. (<a
href="https://doi.org/10.1038/s41559-022-01669-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations with large fitness benefits and mutations occurring at high rates may both cause parallel evolution, but their contribution is predicted to depend on population size. Moreover, high-rate and large-benefit mutations may have different long-term adaptive consequences. We show that small and 100-fold larger bacterial populations evolve resistance to a β-lactam antibiotic by using similar numbers, but different types of mutations. Small populations frequently substitute similar high-rate structural variants and loss-of-function point mutations, including the deletion of a low-activity β-lactamase, and evolve modest resistance levels. Large populations more often use low-rate, large-benefit point mutations affecting the same targets, including mutations activating the β-lactamase and other gain-of-function mutations, leading to much higher resistance levels. Our results demonstrate the separation by clonal interference of mutation classes with divergent adaptive consequences, causing a shift from high-rate to large-benefit mutations with increases in population size. Population size influences mutation supply but may also influence what types of mutations eventually become fixed. Here, Schenk et al. show that small and large experimentally evolving bacteria populations predictably fix different types of mutations, with greater antibiotic resistance emerging in large populations.},
  archive      = {J_NATEE},
  author       = {Schenk, Martijn F. and Zwart, Mark P. and Hwang, Sungmin and Ruelens, Philip and Severing, Edouard and Krug, Joachim and de Visser, J. Arjan G. M.},
  doi          = {10.1038/s41559-022-01669-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {439-447},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Population size mediates the contribution of high-rate and large-benefit mutations to parallel evolution},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Environmental selection and epistasis in an empirical
phenotype–environment–fitness landscape. <em>NATEE</em>, <em>6</em>(4),
427–438. (<a href="https://doi.org/10.1038/s41559-022-01675-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fitness landscapes, mappings of genotype/phenotype to their effects on fitness, are invaluable concepts in evolutionary biochemistry. Although widely discussed, measurements of phenotype–fitness landscapes in proteins remain scarce. Here, we quantify all single mutational effects on fitness and phenotype (EC50) of VIM-2 β-lactamase across a 64-fold range of ampicillin concentrations. We then construct a phenotype–fitness landscape that takes variations in environmental selection pressure into account. We found that a simple, empirical landscape accurately models the ~39,000 mutational data points, suggesting that the evolution of VIM-2 can be predicted on the basis of the selection environment. Our landscape provides new quantitative knowledge on the evolution of the β-lactamases and proteins in general, particularly their evolutionary dynamics under subinhibitory antibiotic concentrations, as well as the mechanisms and environmental dependence of non-specific epistasis. Constructing a fitness landscape from the fitness of Escherichia coli carrying single point mutants in VIM-2 β-lactamase across environments with different ampicillin concentrations and the antibiotic resistance of each variant, the authors show that the evolution of β-lactamases can be predicted based on the selection pressure in the environment.},
  archive      = {J_NATEE},
  author       = {Chen, J. Z. and Fowler, D. M. and Tokuriki, N.},
  doi          = {10.1038/s41559-022-01675-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {427-438},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Environmental selection and epistasis in an empirical phenotype–environment–fitness landscape},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hibernation slows epigenetic ageing in yellow-bellied
marmots. <em>NATEE</em>, <em>6</em>(4), 418–426. (<a
href="https://doi.org/10.1038/s41559-022-01679-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Species that hibernate generally live longer than would be expected based solely on their body size. Hibernation is characterized by long periods of metabolic suppression (torpor) interspersed by short periods of increased metabolism (arousal). The torpor–arousal cycles occur multiple times during hibernation, and it has been suggested that processes controlling the transition between torpor and arousal states cause ageing suppression. Metabolic rate is also a known correlate of longevity; we thus proposed the ‘hibernation–ageing hypothesis’ whereby ageing is suspended during hibernation. We tested this hypothesis in a well-studied population of yellow-bellied marmots (Marmota flaviventer), which spend 7–8 months per year hibernating. We used two approaches to estimate epigenetic age: the epigenetic clock and the epigenetic pacemaker. Variation in epigenetic age of 149 samples collected throughout the life of 73 females was modelled using generalized additive mixed models (GAMM), where season (cyclic cubic spline) and chronological age (cubic spline) were fixed effects. As expected, the GAMM using epigenetic ages calculated from the epigenetic pacemaker was better able to detect nonlinear patterns in epigenetic ageing over time. We observed a logarithmic curve of epigenetic age with time, where the epigenetic age increased at a higher rate until females reached sexual maturity (two years old). With respect to circannual patterns, the epigenetic age increased during the active season and essentially stalled during the hibernation period. Taken together, our results are consistent with the hibernation–ageing hypothesis and may explain the enhanced longevity in hibernators. Species that hibernate generally have longer lifespans than expected based on their body size. The authors show epigenetic ageing patterns from a natural population of hibernating yellow-bellied marmots consistent with the hypothesis that ageing is suspended during hibernation.},
  archive      = {J_NATEE},
  author       = {Pinho, Gabriela M. and Martin, Julien G. A. and Farrell, Colin and Haghani, Amin and Zoller, Joseph A. and Zhang, Joshua and Snir, Sagi and Pellegrini, Matteo and Wayne, Robert K. and Blumstein, Daniel T. and Horvath, Steve},
  doi          = {10.1038/s41559-022-01679-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {418-426},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Hibernation slows epigenetic ageing in yellow-bellied marmots},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Experimental manipulation of microbiota reduces host thermal
tolerance and fitness under heat stress in a vertebrate ectotherm.
<em>NATEE</em>, <em>6</em>(4), 405–417. (<a
href="https://doi.org/10.1038/s41559-022-01686-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying factors that influence how ectothermic animals respond physiologically to changing temperatures is of high importance given current threats of global climate change. Host-associated microbial communities impact animal physiology and have been shown to influence host thermal tolerance in invertebrate systems. However, the role of commensal microbiota in the thermal tolerance of ectothermic vertebrates is unknown. Here we show that experimentally manipulating the tadpole microbiome through environmental water sterilization reduces the host’s acute thermal tolerance to both heat and cold, alters the thermal sensitivity of locomotor performance, and reduces animal survival under prolonged heat stress. We show that these tadpoles have reduced activities of mitochondrial enzymes and altered metabolic rates compared with tadpoles colonized with unmanipulated microbiota, which could underlie differences in thermal phenotypes. These results demonstrate a strong link between the microbiota of an ectothermic vertebrate and the host’s thermal tolerance, performance and fitness. It may therefore be important to consider host-associated microbial communities when predicting species’ responses to climate change. Tadpoles with experimentally manipulated microbiomes have reduced activity of mitochondrial enzymes and altered metabolic rates, resulting in altered thermal sensitivity of locomotion and reduced thermal tolerance and survival.},
  archive      = {J_NATEE},
  author       = {Fontaine, Samantha S. and Mineo, Patrick M. and Kohl, Kevin D.},
  doi          = {10.1038/s41559-022-01686-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {405-417},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Experimental manipulation of microbiota reduces host thermal tolerance and fitness under heat stress in a vertebrate ectotherm},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An earlier start of the thermal growing season enhances tree
growth in cold humid areas but not in dry areas. <em>NATEE</em>,
<em>6</em>(4), 397–404. (<a
href="https://doi.org/10.1038/s41559-022-01668-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Climatic warming alters the onset, duration and cessation of the vegetative season. While previous studies have shown a tight link between thermal conditions and leaf phenology, less is known about the impacts of phenological changes on tree growth. Here, we assessed the relationships between the start of the thermal growing season and tree growth across the extratropical Northern Hemisphere using 3,451 tree-ring chronologies and daily climatic data for 1948–2014. An earlier start of the thermal growing season promoted growth in regions with high ratios of precipitation to temperature but limited growth in cold–dry regions. Path analyses indicated that an earlier start of the thermal growing season enhanced growth primarily by alleviating thermal limitations on wood formation in boreal forests and by lengthening the period of growth in temperate and Mediterranean forests. Semi-arid and dry subalpine forests, however, did not benefit from an earlier onset of growth and a longer growing season, presumably due to associated water loss and/or more frequent early spring frosts. These emergent patterns of how climatic impacts on wood phenology affect tree growth at regional to hemispheric scales hint at how future phenological changes may affect the carbon sequestration capacity of extratropical forest ecosystems. The authors use tree-ring width data across the extratropical Northern Hemisphere to show that earlier growing season onsets lead to enhanced tree radial growth in cold humid but not in dry areas.},
  archive      = {J_NATEE},
  author       = {Gao, Shan and Liang, Eryuan and Liu, Ruishun and Babst, Flurin and Camarero, J. Julio and Fu, Yongshuo H. and Piao, Shilong and Rossi, Sergio and Shen, Miaogen and Wang, Tao and Peñuelas, Josep},
  doi          = {10.1038/s41559-022-01668-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {397-404},
  shortjournal = {Nat. Ecol. Evol},
  title        = {An earlier start of the thermal growing season enhances tree growth in cold humid areas but not in dry areas},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Commensal pseudomonas strains facilitate protective response
against pathogens in the host plant. <em>NATEE</em>, <em>6</em>(4),
383–396. (<a href="https://doi.org/10.1038/s41559-022-01673-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The community structure in the plant-associated microbiome depends collectively on host–microbe, microbe–microbe and host–microbe–microbe interactions. The ensemble of interactions between the host and microbial consortia may lead to outcomes that are not easily predicted from pairwise interactions. Plant–microbe–microbe interactions are important to plant health but could depend on both host and microbe strain variation. Here we study interactions between groups of naturally co-existing commensal and pathogenic Pseudomonas strains in the Arabidopsis thaliana phyllosphere. We find that commensal Pseudomonas prompt a host response that leads to selective inhibition of a specific pathogenic lineage, resulting in plant protection. The extent of protection depends on plant genotype, supporting that these effects are host-mediated. Strain-specific effects are also demonstrated by one individual Pseudomonas isolate eluding the plant protection provided by commensals. Our work highlights how within-species genetic differences in both hosts and microbes can affect host–microbe–microbe dynamics. The authors conduct competition experiments with multiple strains of Pseudomonas (some pathogenic and some commensal) in the phylosphere microbiome of Arabidopsis plants, showing that both the host and the commensal strains interact to inhibit the pathogenic strains.},
  archive      = {J_NATEE},
  author       = {Shalev, Or and Karasov, Talia L. and Lundberg, Derek S. and Ashkenazy, Haim and Pramoj Na Ayutthaya, Pratchaya and Weigel, Detlef},
  doi          = {10.1038/s41559-022-01673-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {383-396},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Commensal pseudomonas strains facilitate protective response against pathogens in the host plant},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Alternative stable states of the forest mycobiome are
maintained through positive feedbacks. <em>NATEE</em>, <em>6</em>(4),
375–382. (<a href="https://doi.org/10.1038/s41559-022-01663-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most trees on Earth form a symbiosis with either arbuscular mycorrhizal or ectomycorrhizal fungi. By forming common mycorrhizal networks, actively modifying the soil environment and other ecological mechanisms, these contrasting symbioses may generate positive feedbacks that favour their own mycorrhizal strategy (that is, the con-mycorrhizal strategy) at the expense of the alternative strategy. Positive con-mycorrhizal feedbacks set the stage for alternative stable states of forests and their fungi, where the presence of different forest mycorrhizal strategies is determined not only by external environmental conditions but also mycorrhiza-mediated feedbacks embedded within the forest ecosystem. Here, we test this hypothesis using thousands of US forest inventory sites to show that arbuscular and ectomycorrhizal tree recruitment and survival exhibit positive con-mycorrhizal density dependence. Data-driven simulations show that these positive feedbacks are sufficient in magnitude to generate and maintain alternative stable states of the forest mycobiome. Given the links between forest mycorrhizal strategy and carbon sequestration potential, the presence of mycorrhizal-mediated alternative stable states affects how we forecast forest composition, carbon sequestration and terrestrial climate feedbacks. Forests tend to be comprised of tree species that mostly associate with either arbuscular or ectomycorrhizal fungi. The authors show that positive feedbacks maintain this biomodal distribution of dominant mycorrhizal associations across US forest inventory plots.},
  archive      = {J_NATEE},
  author       = {Averill, Colin and Fortunel, Claire and Maynard, Daniel S. and van den Hoogen, Johan and Dietze, Michael C. and Bhatnagar, Jennifer M. and Crowther, Thomas W.},
  doi          = {10.1038/s41559-022-01663-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {375-382},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Alternative stable states of the forest mycobiome are maintained through positive feedbacks},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mycorrhizal dominance reduces local tree species diversity
across US forests. <em>NATEE</em>, <em>6</em>(4), 370–374. (<a
href="https://doi.org/10.1038/s41559-021-01634-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ectomycorrhizas and arbuscular mycorrhizas, the two most widespread plant–fungal symbioses, are thought to differentially influence tree species diversity, with positive plant–soil feedbacks favouring locally abundant ectomycorrhizal tree species and negative feedbacks promoting species coexistence and diversity in arbuscular mycorrhizal forests. While seedling recruitment studies and cross-biome patterns of plant diversity and mycorrhizal dominance support this hypothesis, it remains to be tested at the forest stand level over continental scales. Here, we analyse approximately 82,000 forest plots across the USA to show that both ectomycorrhizal-dominated and arbuscular mycorrhizal-dominated forests show relatively low tree diversity, while forests with a mixture of mycorrhizal strategies support a higher number of tree species. Our findings suggest that mycorrhizal dominance, rather than mycorrhizal type, shapes tree diversity in forests. Associations with mycorrhizal fungi can affect many aspects of plant ecology, including the structuring of plant communities at large spatial scales. Here, the authors use forest inventory plot data to show that tree species diversity is highest in plots with mixed presence of arbuscular and ectomycorrhizal fungi, while plots with dominance by either fungal type have lower diversity.},
  archive      = {J_NATEE},
  author       = {Carteron, Alexis and Vellend, Mark and Laliberté, Etienne},
  doi          = {10.1038/s41559-021-01634-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {370-374},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mycorrhizal dominance reduces local tree species diversity across US forests},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Environmental influences on human innovation and behavioural
diversity in southern africa 92–80 thousand years ago. <em>NATEE</em>,
<em>6</em>(4), 361–369. (<a
href="https://doi.org/10.1038/s41559-022-01667-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Africa’s Middle Stone Age preserves sporadic evidence for novel behaviours among early modern humans, prompting a range of questions about the influence of social and environmental factors on patterns of human behavioural evolution. Here we document a suite of novel adaptations dating approximately 92–80 thousand years before the present at the archaeological site Varsche Rivier 003 (VR003), located in southern Africa’s arid Succulent Karoo biome. Distinctive innovations include the production of ostrich eggshell artefacts, long-distance transportation of marine molluscs and systematic use of heat shatter in stone tool production, none of which occur in coeval assemblages at sites in more humid, well-studied regions immediately to the south. The appearance of these novelties at VR003 corresponds with a period of reduced regional wind strength and enhanced summer rainfall, and all of them disappear with increasing winter rainfall dominance after 80 thousand years before the present, following which a pattern of technological similarity emerges at sites throughout the broader region. The results indicate complex and environmentally contingent processes of innovation and cultural transmission in southern Africa during the Middle Stone Age. Evidence of ostrich eggshell artefacts, long-distance transport of marine shells and heat treatment of stone tools in a marginal southern African environment 92–80 thousand years ago suggests that innovation among early human groups was sensitive to ecological variation.},
  archive      = {J_NATEE},
  author       = {Mackay, Alex and Armitage, Simon J. and Niespolo, Elizabeth M. and Sharp, Warren D. and Stahlschmidt, Mareike C. and Blackwood, Alexander F. and Boyd, Kelsey C. and Chase, Brian M. and Lagle, Susan E. and Kaplan, Chester F. and Low, Marika A. and Martisius, Naomi L. and McNeill, Patricia J. and Moffat, Ian and O’Driscoll, Corey A. and Rudd, Rachel and Orton, Jayson and Steele, Teresa E.},
  doi          = {10.1038/s41559-022-01667-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {361-369},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Environmental influences on human innovation and behavioural diversity in southern africa 92–80 thousand years ago},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reply to: Conclusions of low extinction risk for most
species of reef-building corals are premature. <em>NATEE</em>,
<em>6</em>(4), 359–360. (<a
href="https://doi.org/10.1038/s41559-022-01660-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Dietzel, Andreas and Bode, Michael and Connolly, Sean R. and Hughes, Terry P.},
  doi          = {10.1038/s41559-022-01660-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {359-360},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Conclusions of low extinction risk for most species of reef-building corals are premature},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conclusions of low extinction risk for most species of
reef-building corals are premature. <em>NATEE</em>, <em>6</em>(4),
357–358. (<a href="https://doi.org/10.1038/s41559-022-01659-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Muir, Paul R. and Obura, David O. and Hoeksema, Bert W. and Sheppard, Charles and Pichon, Michel and Richards, Zoe T.},
  doi          = {10.1038/s41559-022-01659-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {357-358},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Conclusions of low extinction risk for most species of reef-building corals are premature},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evolution of cod supergenes. <em>NATEE</em>, <em>6</em>(4),
355–356. (<a href="https://doi.org/10.1038/s41559-022-01662-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Population genomic and phylogenomic analyses of Atlantic cod provide new insights into the origin and maintenance of supergenes and highlight the role of recombination and structural variants.},
  archive      = {J_NATEE},
  author       = {Tigano, Anna},
  doi          = {10.1038/s41559-022-01662-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {355-356},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolution of cod supergenes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Population size matters for mutations. <em>NATEE</em>,
<em>6</em>(4), 353–354. (<a
href="https://doi.org/10.1038/s41559-022-01670-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rate of fixation of different antibiotic-resistance mutation types demonstrates a trade-off shaped by population size.},
  archive      = {J_NATEE},
  author       = {Gordo, Isabel},
  doi          = {10.1038/s41559-022-01670-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {353-354},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Population size matters for mutations},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The biology of beauty sleep. <em>NATEE</em>, <em>6</em>(4),
351–352. (<a href="https://doi.org/10.1038/s41559-022-01683-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Age estimation based on DNA methylation levels suggests that hibernation arrests biological ageing in wild yellow-bellied marmots.},
  archive      = {J_NATEE},
  author       = {Anderson, Jordan A. and Tung, Jenny},
  doi          = {10.1038/s41559-022-01683-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {351-352},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The biology of beauty sleep},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Weeds fight back. <em>NATEE</em>, <em>6</em>(4), 350. (<a
href="https://doi.org/10.1038/s41559-022-01691-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-022-01691-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {350},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Weeds fight back},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Madagascan megafrugivores. <em>NATEE</em>, <em>6</em>(4),
349. (<a href="https://doi.org/10.1038/s41559-022-01690-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-022-01690-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {349},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Madagascan megafrugivores},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). Kunga line. <em>NATEE</em>, <em>6</em>(4), 348. (<a
href="https://doi.org/10.1038/s41559-022-01676-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-022-01676-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {348},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Kunga line},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Fungal traits. <em>NATEE</em>, <em>6</em>(4), 347. (<a
href="https://doi.org/10.1038/s41559-022-01678-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-022-01678-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {347},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fungal traits},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Indigenous land healing beyond hashtags. <em>NATEE</em>,
<em>6</em>(4), 346. (<a
href="https://doi.org/10.1038/s41559-022-01708-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Grenz, Jennifer},
  doi          = {10.1038/s41559-022-01708-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {346},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Indigenous land healing beyond hashtags},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). William g. Hill (1940–2021). <em>NATEE</em>, <em>6</em>(4),
344–345. (<a href="https://doi.org/10.1038/s41559-022-01687-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative geneticist who made fundamental contributions to understanding the nature of genetic variation.},
  archive      = {J_NATEE},
  author       = {Keightley, Peter D.},
  doi          = {10.1038/s41559-022-01687-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {344-345},
  shortjournal = {Nat. Ecol. Evol},
  title        = {William g. hill (1940–2021)},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Biodiversity conservation in afghanistan under the returned
taliban. <em>NATEE</em>, <em>6</em>(4), 342–343. (<a
href="https://doi.org/10.1038/s41559-021-01655-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Maheshwari, Aishwarya},
  doi          = {10.1038/s41559-021-01655-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {342-343},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biodiversity conservation in afghanistan under the returned taliban},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Seeing the forest for the trees. <em>NATEE</em>,
<em>6</em>(4), 341. (<a
href="https://doi.org/10.1038/s41559-022-01743-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ecologists are embracing the use of forestry data, offering the chance to move beyond decades of division between disciplines.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01743-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {341},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Seeing the forest for the trees},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Biodiversity targets will not be met
without debt and tax justice. <em>NATEE</em>, <em>6</em>(3), 340. (<a
href="https://doi.org/10.1038/s41559-022-01704-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Dempsey, Jessica and Irvine-Broque, Audrey and Bigger, Patrick and Christiansen, Jens and Muchhala, Bhumika and Nelson, Sara and Rojas-Marchini, Fernanda and Shapiro-Garza, Elizabeth and Schuldt, Andrew and DiSilvestro, Adriana},
  doi          = {10.1038/s41559-022-01704-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {340},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Biodiversity targets will not be met without debt and tax justice},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Plant-water sensitivity regulates wildfire vulnerability.
<em>NATEE</em>, <em>6</em>(3), 332–339. (<a
href="https://doi.org/10.1038/s41559-021-01654-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extreme wildfires extensively impact human health and the environment. Increasing vapour pressure deficit (VPD) has led to a chronic increase in wildfire area in the western United States, yet some regions have been more affected than others. Here we show that for the same increase in VPD, burned area increases more in regions where vegetation moisture shows greater sensitivity to water limitation (plant-water sensitivity; R2 = 0.71). This has led to rapid increases in human exposure to wildfire risk, both because the population living in areas with high plant-water sensitivity grew 50% faster during 1990–2010 than in other wildland–urban interfaces and because VPD has risen most rapidly in these vulnerable areas. As plant-water sensitivity is strongly linked to wildfire vulnerability, accounting for ecophysiological controls should improve wildfire forecasts. If recent trends in VPD and demographic shifts continue, human wildfire risk will probably continue to increase. The authors show that an ecosystem’s sensitivity to drought, measured as the amount of change in vegetation moisture content for a given change in background moisture, predicts the fire hazard in that location.},
  archive      = {J_NATEE},
  author       = {Rao, Krishna and Williams, A. Park and Diffenbaugh, Noah S. and Yebra, Marta and Konings, Alexandra G.},
  doi          = {10.1038/s41559-021-01654-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {332-339},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Plant-water sensitivity regulates wildfire vulnerability},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Widespread extinction debts and colonization credits in
united states breeding bird communities. <em>NATEE</em>, <em>6</em>(3),
324–331. (<a href="https://doi.org/10.1038/s41559-021-01653-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Species extinctions and colonizations in response to land cover change often occur with time lags rather than instantaneously, leading to extinction debts and colonization credits. These debts and credits can lead to erroneous predictions of future biodiversity. Recent attempts to measure debts and credits have been limited to small geographical areas and have not considered multiple land cover types, or the directionality of land cover change. Here we quantify the relative contribution of past and current landscapes on the current effective number of species of 2,880 US bird communities, explicitly measuring the response of biodiversity to increases and decreases in five land cover types. We find that the current effective number of species is still largely explained by the past landscape composition (legacy effect), depending on the type, magnitude and directionality of recent land cover change. This legacy effect leads to widespread extinction debts and colonization credits. Specifically, we reveal debts across 52% of the United States, particularly in recently urbanized areas, and colonization credits in the remaining 48%, which are primarily associated with grassland decrease. We conclude that biodiversity policy targets risk becoming rapidly obsolete unless past landscapes are considered and debts and credits accounted for. The authors use US Breeding Bird Survey data to quantify the losses and gains in bird species richness that have not yet been realized owing to ecological time lags following past land use changes.},
  archive      = {J_NATEE},
  author       = {Haddou, Yacob and Mancy, Rebecca and Matthiopoulos, Jason and Spatharis, Sofie and Dominoni, Davide M.},
  doi          = {10.1038/s41559-021-01653-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {324-331},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Widespread extinction debts and colonization credits in united states breeding bird communities},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Contrasting responses of woody and grassland ecosystems to
increased CO2 as water supply varies. <em>NATEE</em>, <em>6</em>(3),
315–323. (<a href="https://doi.org/10.1038/s41559-021-01642-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experiments show that elevated atmospheric CO2 (eCO2) often enhances plant photosynthesis and productivity, yet this effect varies substantially and may be climate sensitive. Understanding if, where and how water supply regulates CO2 enhancement is critical for projecting terrestrial responses to increasing atmospheric CO2 and climate change. Here, using data from 14 long-term ecosystem-scale CO2 experiments, we show that the eCO2 enhancement of annual aboveground net primary productivity is sensitive to annual precipitation and that this sensitivity differs between woody and grassland ecosystems. During wetter years, CO2 enhancement increases in woody ecosystems but declines in grass-dominated systems. Consistent with this difference, woody ecosystems can increase leaf area index in wetter years more effectively under eCO2 than can grassland ecosystems. Overall, and across different precipitation regimes, woody systems had markedly stronger CO2 enhancement (24%) than grasslands (13%). We developed an empirical relationship to quantify aboveground net primary productivity enhancement on the basis of changes in leaf area index, providing a new approach for evaluating eCO2 impacts on the productivity of terrestrial ecosystems. The extent to which excess carbon dioxide causes a ‘fertilization effect’ varies across biomes and as a function of water supply, finds a study examining data from 14 CO2 experimental sites.},
  archive      = {J_NATEE},
  author       = {Pan, Yude and Jackson, Robert B. and Hollinger, David Y. and Phillips, Oliver L. and Nowak, Robert S. and Norby, Richard J. and Oren, Ram and Reich, Peter B. and Lüscher, Andreas and Mueller, Kevin E. and Owensby, Clenton and Birdsey, Richard and Hom, John and Luo, Yiqi},
  doi          = {10.1038/s41559-021-01642-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {315-323},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Contrasting responses of woody and grassland ecosystems to increased CO2 as water supply varies},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ecological network complexity scales with area.
<em>NATEE</em>, <em>6</em>(3), 307–314. (<a
href="https://doi.org/10.1038/s41559-021-01644-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Larger geographical areas contain more species—an observation raised to a law in ecology. Less explored is whether biodiversity changes are accompanied by a modification of interaction networks. We use data from 32 spatial interaction networks from different ecosystems to analyse how network structure changes with area. We find that basic community structure descriptors (number of species, links and links per species) increase with area following a power law. Yet, the distribution of links per species varies little with area, indicating that the fundamental organization of interactions within networks is conserved. Our null model analyses suggest that the spatial scaling of network structure is determined by factors beyond species richness and the number of links. We demonstrate that biodiversity–area relationships can be extended from species counts to higher levels of network complexity. Therefore, the consequences of anthropogenic habitat destruction may extend from species loss to wider simplification of natural communities. Using 32 ecological networks (host–parasite, plant–pollinator, plant–herbivore and other food webs), the authors show that several network properties scale with the size of the sampling area, suggesting a new type of biodiversity–area relationship.},
  archive      = {J_NATEE},
  author       = {Galiana, Núria and Lurgi, Miguel and Bastazini, Vinicius A. G. and Bosch, Jordi and Cagnolo, Luciano and Cazelles, Kevin and Claramunt-López, Bernat and Emer, Carine and Fortin, Marie-Josée and Grass, Ingo and Hernández-Castellano, Carlos and Jauker, Frank and Leroux, Shawn J. and McCann, Kevin and McLeod, Anne M. and Montoya, Daniel and Mulder, Christian and Osorio-Canadas, Sergio and Reverté, Sara and Rodrigo, Anselm and Steffan-Dewenter, Ingolf and Traveset, Anna and Valverde, Sergi and Vázquez, Diego P. and Wood, Spencer A. and Gravel, Dominique and Roslin, Tomas and Thuiller, Wilfried and Montoya, José M.},
  doi          = {10.1038/s41559-021-01644-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {307-314},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecological network complexity scales with area},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Palaeoecological data indicates land-use changes across
europe linked to spatial heterogeneity in mortality during the black
death pandemic. <em>NATEE</em>, <em>6</em>(3), 297–306. (<a
href="https://doi.org/10.1038/s41559-021-01652-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Black Death (1347–1352 ce) is the most renowned pandemic in human history, believed by many to have killed half of Europe’s population. However, despite advances in ancient DNA research that conclusively identified the pandemic’s causative agent (bacterium Yersinia pestis), our knowledge of the Black Death remains limited, based primarily on qualitative remarks in medieval written sources available for some areas of Western Europe. Here, we remedy this situation by applying a pioneering new approach, ‘big data palaeoecology’, which, starting from palynological data, evaluates the scale of the Black Death’s mortality on a regional scale across Europe. We collected pollen data on landscape change from 261 radiocarbon-dated coring sites (lakes and wetlands) located across 19 modern-day European countries. We used two independent methods of analysis to evaluate whether the changes we see in the landscape at the time of the Black Death agree with the hypothesis that a large portion of the population, upwards of half, died within a few years in the 21 historical regions we studied. While we can confirm that the Black Death had a devastating impact in some regions, we found that it had negligible or no impact in others. These inter-regional differences in the Black Death’s mortality across Europe demonstrate the significance of cultural, ecological, economic, societal and climatic factors that mediated the dissemination and impact of the disease. The complex interplay of these factors, along with the historical ecology of plague, should be a focus of future research on historical pandemics. Historical accounts of the mortality outcomes of the Black Death plague pandemic are variable across Europe, with much higher death tolls suggested in some areas than others. Here the authors use a ‘big data palaeoecology’ approach to show that land use change following the pandemic was spatially variable across Europe, confirming heterogeneous responses with empirical data.},
  archive      = {J_NATEE},
  author       = {Izdebski, A. and Guzowski, P. and Poniat, R. and Masci, L. and Palli, J. and Vignola, C. and Bauch, M. and Cocozza, C. and Fernandes, R. and Ljungqvist, F. C. and Newfield, T. and Seim, A. and Abel-Schaad, D. and Alba-Sánchez, F. and Björkman, L. and Brauer, A. and Brown, A. and Czerwiński, S. and Ejarque, A. and Fiłoc, M. and Florenzano, A. and Fredh, E. D. and Fyfe, R. and Jasiunas, N. and Kołaczek, P. and Kouli, K. and Kozáková, R. and Kupryjanowicz, M. and Lagerås, P. and Lamentowicz, M. and Lindbladh, M. and López-Sáez, J. A. and Luelmo-Lautenschlaeger, R. and Marcisz, K. and Mazier, F. and Mensing, S. and Mercuri, A. M. and Milecka, K. and Miras, Y. and Noryśkiewicz, A. M. and Novenko, E. and Obremska, M. and Panajiotidis, S. and Papadopoulou, M. L. and Pędziszewska, A. and Pérez-Díaz, S. and Piovesan, G. and Pluskowski, A. and Pokorny, P. and Poska, A. and Reitalu, T. and Rösch, M. and Sadori, L. and Sá Ferreira, C. and Sebag, D. and Słowiński, M. and Stančikaitė, M. and Stivrins, N. and Tunno, I. and Veski, S. and Wacnik, A. and Masi, A.},
  doi          = {10.1038/s41559-021-01652-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {297-306},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Palaeoecological data indicates land-use changes across europe linked to spatial heterogeneity in mortality during the black death pandemic},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A titanosaurian sauropod with gondwanan affinities in the
latest cretaceous of europe. <em>NATEE</em>, <em>6</em>(3), 288–296. (<a
href="https://doi.org/10.1038/s41559-021-01651-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The origin of the last sauropod dinosaur communities in Europe and their evolution during the final 15 million years of the Cretaceous have become a complex phylogenetic and palaeobiogeographic puzzle characterized by the controversy on the alleged coexistence of immigrant, Gondwana-related taxa alongside relictual and insular clades. In this context, we describe a new titanosaurian sauropod dinosaur, Abditosaurus kuehnei gen. et sp. nov., from the Late Cretaceous (Maastrichtian) Tremp Group of Catalonia (Spain). Phylogenetic analyses recover Abditosaurus separately from other European titanosaurs, within a clade of otherwise South American and African saltasaurines. The affinity of the new taxon with southern landmasses is reinforced by spatiotemporal co-occurrence with Gondwanan titanosaurian oospecies in southern Europe. The large size and the lack of osteohistological features potentially related to insular dwarfism or size reduction support the idea that Abditosaurus belongs to an immigrant lineage, unequivocally distinct from some of the island dwarfs of the European archipelago. The arrival of the Abditosaurus lineage to the Ibero–Armorican Island is hypothesized to have occurred during the earliest Maastrichtian (70.6 Ma), probably as a result of a global and regional sea-level drop that reactivated ancient dispersal routes between Africa and Europe. The arrival of large-bodied titanosaurs to the European archipelago produced dramatic changes in its insular ecosystems and important evolutionary changes in its dinosaur faunas, especially with respect to the ‘island rule’ effect. The authors report a new genus and species of titanosaurian sauropod, Abditosaurus kuehnei, from the Late Cretaceous (ca. 70.5 Ma) of Spain. A. kuehnei groups phylogenetically with South American and African taxa, suggesting geographical connectivity between Africa and Europe at this time.},
  archive      = {J_NATEE},
  author       = {Vila, Bernat and Sellés, Albert and Moreno-Azanza, Miguel and Razzolini, Novella L. and Gil-Delgado, Alejandro and Canudo, José Ignacio and Galobart, Àngel},
  doi          = {10.1038/s41559-021-01651-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {288-296},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A titanosaurian sauropod with gondwanan affinities in the latest cretaceous of europe},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Dosage sensitivity and exon shuffling shape the landscape of
polymorphic duplicates in drosophila and humans. <em>NATEE</em>,
<em>6</em>(3), 273–287. (<a
href="https://doi.org/10.1038/s41559-021-01614-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite polymorphic duplicate genes&#39; importance for the early stages of duplicate gene evolution, they are less studied than old gene duplicates. Two essential questions thus remain poorly addressed: how does dosage sensitivity, imposed by stoichiometry in protein complexes or by X chromosome dosage compensation, affect the emergence of complete duplicate genes? Do introns facilitate intergenic and intragenic chimaerism as predicted by the theory of exon shuffling? Here, we analysed new data for Drosophila and public data for humans, to characterize polymorphic duplicate genes with respect to dosage, exon–intron structures and allele frequencies. We found that complete duplicate genes are under dosage constraint induced by protein stoichiometry but potentially tolerated by X chromosome dosage compensation. We also found that in the intron-rich human genome, gene fusions and intragenic duplications extensively use intronic breakpoints generating in-frame proteins, in accordance with the theory of exon shuffling. Finally, we found that only a small proportion of complete or partial duplicates are at high frequencies, indicating the deleterious nature of dosage or gene structural changes. Altogether, we demonstrate how mechanistic factors including dosage sensitivity and exon–intron structure shape the short-term functional consequences of gene duplication. Polymorphic duplicate genes are important early stages of duplicate gene evolution. Here, the authors characterize polymorphic gene duplicates with respect to dosage, exon–intron structures and allele frequencies for Drosophila and humans.},
  archive      = {J_NATEE},
  author       = {Zhang, Dan and Leng, Liang and Chen, Chunyan and Huang, Jiawei and Zhang, Yaqiong and Yuan, Hao and Ma, Chenyu and Chen, Hua and Zhang, Yong E.},
  doi          = {10.1038/s41559-021-01614-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {273-287},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dosage sensitivity and exon shuffling shape the landscape of polymorphic duplicates in drosophila and humans},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Morphological volatility precedes ecological innovation in
early echinoderms. <em>NATEE</em>, <em>6</em>(3), 263–272. (<a
href="https://doi.org/10.1038/s41559-021-01656-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Origins of higher taxonomic groups entail dramatic and nearly simultaneous changes in morphology and ecological function, limiting our ability to disentangle the drivers of evolutionary diversification. Here we phylogenetically compare the anatomy and life habits of Cambrian–Ordovician echinoderms to test which facet better facilitates future success. Rates of morphological evolution are faster and involve more volatile trait changes, allowing morphological disparity to accrue faster and earlier in the Cambrian. However, persistent life-habit evolution throughout the early Palaeozoic, combined with iterative functional convergence within adaptive strategies, results in major expansion of ecospace and functional diversity. The interactions between tempo, divergence and convergence demonstrate not only that anatomical novelty precedes ecological success, but also that ecological innovation is constrained, even during a phylum’s origin. Comparing anatomy and life habits for fossil echinoderms spanning the Cambrian to Ordovician periods, the authors show that anatomical form and functional diversity are decoupled, and ecological innovation is constrained.},
  archive      = {J_NATEE},
  author       = {Novack-Gottshall, Philip M. and Sultan, Ali and Smith, Nicholas S. and Purcell, Jack and Hanson, Kathryn E. and Lively, Rachel and Ranjha, Isa and Collins, Clayton and Parker, Runeshia and Sumrall, Colin D. and Deline, Bradley},
  doi          = {10.1038/s41559-021-01656-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {263-272},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Morphological volatility precedes ecological innovation in early echinoderms},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Site-and-branch-heterogeneous analyses of an expanded
dataset favour mitochondria as sister to known alphaproteobacteria.
<em>NATEE</em>, <em>6</em>(3), 253–262. (<a
href="https://doi.org/10.1038/s41559-021-01638-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining the phylogenetic origin of mitochondria is key to understanding the ancestral mitochondrial symbiosis and its role in eukaryogenesis. However, the precise evolutionary relationship between mitochondria and their closest bacterial relatives remains hotly debated. The reasons include pervasive phylogenetic artefacts as well as limited protein and taxon sampling. Here we developed a new model of protein evolution that accommodates both across-site and across-branch compositional heterogeneity. We applied this site-and-branch-heterogeneous model (MAM60 + GFmix) to a considerably expanded dataset that comprises 108 mitochondrial proteins of alphaproteobacterial origin, and novel metagenome-assembled genomes from microbial mats, microbialites and sediments. The MAM60 + GFmix model fits the data much better and agrees with analyses of compositionally homogenized datasets with conventional site-heterogenous models. The consilience of evidence thus suggests that mitochondria are sister to the Alphaproteobacteria to the exclusion of MarineProteo1 and Magnetococcia. We also show that the ancestral presence of the crista-developing mitochondrial contact site and cristae organizing system (a mitofilin-domain-containing Mic60 protein) in mitochondria and the Alphaproteobacteria only supports their close relationship. The evolutionary relationship between mitochondria and their closest bacterial relatives remains uncertain. Applying a new model of protein evolution to an extended dataset, the authors reconstruct the phylogenetic position of the mitochondria as sister to the Alphaproteobacteria.},
  archive      = {J_NATEE},
  author       = {Muñoz-Gómez, Sergio A. and Susko, Edward and Williamson, Kelsey and Eme, Laura and Slamovits, Claudio H. and Moreira, David and López-García, Purificación and Roger, Andrew J.},
  doi          = {10.1038/s41559-021-01638-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {253-262},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Site-and-branch-heterogeneous analyses of an expanded dataset favour mitochondria as sister to known alphaproteobacteria},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Titanosaur boom. <em>NATEE</em>, <em>6</em>(3), 251–252. (<a
href="https://doi.org/10.1038/s41559-022-01677-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An exceptionally large species of sauropod titanosaur from the Late Cretaceous of northern Spain provides insight into changing diversity dynamics of titanosaurs over time, and sheds light on faunal turnover and migration.},
  archive      = {J_NATEE},
  author       = {Díez Díaz, Verónica},
  doi          = {10.1038/s41559-022-01677-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {251-252},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Titanosaur boom},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Young duplicate genic DNA. <em>NATEE</em>, <em>6</em>(3),
249–250. (<a href="https://doi.org/10.1038/s41559-021-01639-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of population genomic and transcriptomic data of flies and humans shows that species-specific mutation biases and common selective forces have collectively shaped the early evolutionary phase of duplicated DNA segments that overlap with coding genes.},
  archive      = {J_NATEE},
  author       = {Liao, Ben-Yang},
  doi          = {10.1038/s41559-021-01639-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {249-250},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Young duplicate genic DNA},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Early echinoderms decouple form and function.
<em>NATEE</em>, <em>6</em>(3), 247–248. (<a
href="https://doi.org/10.1038/s41559-022-01664-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid morphological evolution in early echinoderms was later outpaced by increases in ecological diversification, indicating the phylum exhibited morphological volatility and ecological constraints at its origin.},
  archive      = {J_NATEE},
  author       = {Cole, Selina R.},
  doi          = {10.1038/s41559-022-01664-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {247-248},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Early echinoderms decouple form and function},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Freshwater stressors. <em>NATEE</em>, <em>6</em>(3), 246.
(<a href="https://doi.org/10.1038/s41559-022-01666-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-022-01666-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {246},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Freshwater stressors},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Density-dependent genes. <em>NATEE</em>, <em>6</em>(3),
245. (<a href="https://doi.org/10.1038/s41559-022-01658-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-022-01658-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {245},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Density-dependent genes},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Dams threaten big cats. <em>NATEE</em>, <em>6</em>(3), 244.
(<a href="https://doi.org/10.1038/s41559-021-01657-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-021-01657-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {244},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dams threaten big cats},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Thomas e. Lovejoy (1941–2021). <em>NATEE</em>,
<em>6</em>(3), 242–243. (<a
href="https://doi.org/10.1038/s41559-022-01672-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biodiversity advocate and Amazon expert},
  archive      = {J_NATEE},
  author       = {Laurance, William F.},
  doi          = {10.1038/s41559-022-01672-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {242-243},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Thomas e. lovejoy (1941–2021)},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Edward o. Wilson (1929–2021). <em>NATEE</em>, <em>6</em>(3),
240–241. (<a href="https://doi.org/10.1038/s41559-022-01680-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Naturalist, synthesizer and conservation champion.},
  archive      = {J_NATEE},
  author       = {Moreau, Corrie S. and Pierce, Naomi E.},
  doi          = {10.1038/s41559-022-01680-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {240-241},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Edward o. wilson (1929–2021)},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Biodiversity targets will not be met without debt and tax
justice. <em>NATEE</em>, <em>6</em>(3), 237–239. (<a
href="https://doi.org/10.1038/s41559-021-01619-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Approaches to financing biodiversity conservation tend to focus on funding gaps, but fail to address underlying political and economic drivers. We propose two strategies — tax reform and debt justice — to supercharge public financing for biodiversity and deflate harmful financial flows, while chipping away at the causes of state austerity.},
  archive      = {J_NATEE},
  author       = {Dempsey, Jessica and Irvine-Broque, Audrey and Bigger, Patrick and Christiansen, Jens and Muchhala, Bhumika and Nelson, Sara and Rojas-Marchini, Fernanda and Shapiro-Garza, Elizabeth and Schuldt, Andrew and DiSilvestro, Adriana},
  doi          = {10.1038/s41559-021-01619-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {237-239},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biodiversity targets will not be met without debt and tax justice},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Implementation of the COP26 declaration to halt forest loss
must safeguard and include indigenous people. <em>NATEE</em>,
<em>6</em>(3), 235–236. (<a
href="https://doi.org/10.1038/s41559-021-01650-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Haenssgen, Marco J. and Lechner, Alex M. and Rakotonarivo, Sarobidy and Leepreecha, Prasit and Sakboon, Mukdawan and Chu, Ta-Wei and Auclair, Elizabeth and Vlaev, Ivo},
  doi          = {10.1038/s41559-021-01650-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {235-236},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Implementation of the COP26 declaration to halt forest loss must safeguard and include indigenous people},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Brazilian budget cuts further threaten gender equality in
research. <em>NATEE</em>, <em>6</em>(3), 234. (<a
href="https://doi.org/10.1038/s41559-021-01640-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Hipólito, Juliana and Shirai, Leila Teruko and Diele-Viegas, Luisa Maria and Halinski, Rosana and Soares Pires, Carmen Sílvia and Fontes, Eliana Maria Gouveia},
  doi          = {10.1038/s41559-021-01640-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {234},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Brazilian budget cuts further threaten gender equality in research},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). We need to talk about mental health. <em>NATEE</em>,
<em>6</em>(3), 233. (<a
href="https://doi.org/10.1038/s41559-022-01707-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Environmental scientists can find themselves exposed to a range of mental health challenges, and we all need to support each other in addressing them.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01707-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {233},
  shortjournal = {Nat. Ecol. Evol},
  title        = {We need to talk about mental health},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Global predictors of language
endangerment and the future of linguistic diversity. <em>NATEE</em>,
<em>6</em>(2), 231. (<a
href="https://doi.org/10.1038/s41559-022-01684-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bromham, Lindell and Dinnage, Russell and Skirgård, Hedvig and Ritchie, Andrew and Cardillo, Marcel and Meakins, Felicity and Greenhill, Simon and Hua, Xia},
  doi          = {10.1038/s41559-022-01684-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {231},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Global predictors of language endangerment and the future of linguistic diversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Colonial history and global economics
distort our understanding of deep-time biodiversity. <em>NATEE</em>,
<em>6</em>(2), 230. (<a
href="https://doi.org/10.1038/s41559-022-01682-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Raja, Nussaïbah B. and Dunne, Emma M. and Matiwane, Aviwe and Khan, Tasnuva Ming and Nätscher, Paulina S. and Ghilardi, Aline M. and Chattopadhyay, Devapriya},
  doi          = {10.1038/s41559-022-01682-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {230},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Colonial history and global economics distort our understanding of deep-time biodiversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Complex marine microbial communities partition metabolism of
scarce resources over the diel cycle. <em>NATEE</em>, <em>6</em>(2),
218–229. (<a href="https://doi.org/10.1038/s41559-021-01606-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex assemblages of microbes in the surface ocean are responsible for approximately half of global carbon fixation. The persistence of high taxonomic diversity despite competition for a small suite of relatively homogeneously distributed nutrients, that is, ‘the paradox of the plankton’, represents a long-standing challenge for ecological theory. Here we find evidence consistent with temporal niche partitioning of nitrogen assimilation processes over a diel cycle in the North Pacific Subtropical Gyre. We jointly analysed transcript abundances, lipids and metabolites and discovered that a small number of diel archetypes can explain pervasive periodic dynamics. Metabolic pathway analysis of identified diel signals revealed asynchronous timing in the transcription of nitrogen uptake and assimilation genes among different microbial groups—cyanobacteria, heterotrophic bacteria and eukaryotes. This temporal niche partitioning of nitrogen uptake emerged despite synchronous transcription of photosynthesis and central carbon metabolism genes and associated macromolecular abundances. Temporal niche partitioning may be a mechanism by which microorganisms in the open ocean mitigate competition for scarce resources, supporting community coexistence. By integrating time series analyses of transcripts, lipids and metabolites, the authors show that microorganisms in the open ocean partition scarce resources temporally, with different microbial groups expressing nitrogen uptake and assimilation processes at different points throughout the diel cycle.},
  archive      = {J_NATEE},
  author       = {Muratore, Daniel and Boysen, Angela K. and Harke, Matthew J. and Becker, Kevin W. and Casey, John R. and Coesel, Sacha N. and Mende, Daniel R. and Wilson, Samuel T. and Aylward, Frank O. and Eppley, John M. and Vislova, Alice and Peng, Shengyun and Rodriguez-Gonzalez, Rogelio A. and Beckett, Stephen J. and Virginia Armbrust, E. and DeLong, Edward F. and Karl, David M. and White, Angelicque E. and Zehr, Jonathan P. and Van Mooy, Benjamin A. S. and Dyhrman, Sonya T. and Ingalls, Anitra E. and Weitz, Joshua S.},
  doi          = {10.1038/s41559-021-01606-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {218-229},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Complex marine microbial communities partition metabolism of scarce resources over the diel cycle},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial structure governs the mode of tumour evolution.
<em>NATEE</em>, <em>6</em>(2), 207–217. (<a
href="https://doi.org/10.1038/s41559-021-01615-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing the mode—the way, manner or pattern—of evolution in tumours is important for clinical forecasting and optimizing cancer treatment. Sequencing studies have inferred various modes, including branching, punctuated and neutral evolution, but it is unclear why a particular pattern predominates in any given tumour. Here we propose that tumour architecture is key to explaining the variety of observed genetic patterns. We examine this hypothesis using spatially explicit population genetics models and demonstrate that, within biologically relevant parameter ranges, different spatial structures can generate four tumour evolutionary modes: rapid clonal expansion, progressive diversification, branching evolution and effectively almost neutral evolution. Quantitative indices for describing and classifying these evolutionary modes are presented. Using these indices, we show that our model predictions are consistent with empirical observations for cancer types with corresponding spatial structures. The manner of cell dispersal and the range of cell–cell interactions are found to be essential factors in accurately characterizing, forecasting and controlling tumour evolution. Tumour evolution modelling indicates that different tumour spatial structures can determine different tumour evolutionary modes, which are regulated by cell dispersal and cell–cell interactions. Model predictions of four evolutionary modes are consistent with empirical observations of cancers with varying architectures.},
  archive      = {J_NATEE},
  author       = {Noble, Robert and Burri, Dominik and Le Sueur, Cécile and Lemant, Jeanne and Viossat, Yannick and Kather, Jakob Nikolas and Beerenwinkel, Niko},
  doi          = {10.1038/s41559-021-01615-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {207-217},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Spatial structure governs the mode of tumour evolution},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A masculinizing supergene underlies an exaggerated male
reproductive morph in a spider. <em>NATEE</em>, <em>6</em>(2), 195–206.
(<a href="https://doi.org/10.1038/s41559-021-01626-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In many species, individuals can develop into strikingly different morphs, which are determined by a simple Mendelian locus. How selection shapes loci that control complex phenotypic differences remains poorly understood. In the spider Oedothorax gibbosus, males develop either into a ‘hunched’ morph with conspicuous head structures or as a fast-developing ‘flat’ morph with a female-like appearance. We show that the hunched-determining allele contains a unique genomic fragment of approximately 3 megabases that is absent in the flat-determining allele. This fragment comprises dozens of genes that duplicated from genes found at the same as well as different chromosomes. All functional duplicates, including a duplicate of the key sexual differentiation regulatory gene doublesex, show male-specific expression, which illustrates their integrated role as a masculinizing supergene. Our findings demonstrate how extensive indel polymorphisms and duplications of regulatory genes may contribute to the evolution of co-adapted gene clusters, sex-limited reproductive morphs and the enigmatic evolution of exaggerated sexual traits in general. The hunchback spider exhibits male-specific polymorphism in head shape. Here, the authors show that this polymorphism is determined by a large insertion that comprises duplicated male-specific genes including a duplicate of a key sexual differentiation regulatory gene.},
  archive      = {J_NATEE},
  author       = {Hendrickx, Frederik and De Corte, Zoë and Sonet, Gontran and Van Belleghem, Steven M. and Köstlbacher, Stephan and Vangestel, Carl},
  doi          = {10.1038/s41559-021-01626-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {195-206},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A masculinizing supergene underlies an exaggerated male reproductive morph in a spider},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prochlorococcus have low global mutation rate and small
effective population size. <em>NATEE</em>, <em>6</em>(2), 183–194. (<a
href="https://doi.org/10.1038/s41559-021-01591-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prochlorococcus are the most abundant free-living photosynthetic carbon-fixing organisms in the ocean. Prochlorococcus show small genome sizes, low genomic G+C content, reduced DNA repair gene pool and fast evolutionary rates, which are typical features of endosymbiotic bacteria. Nevertheless, their evolutionary mechanisms are believed to be different. Evolution of endosymbiotic bacteria is dominated by genetic drift owing to repeated population bottlenecks, whereas Prochlorococcus are postulated to have extremely large effective population sizes (Ne) and thus drift has rarely been considered. However, accurately extrapolating Ne requires measuring an unbiased global mutation rate through mutation accumulation, which is challenging for Prochlorococcus. Here, we managed this experiment over 1,065 days using Prochlorococcus marinus AS9601, sequenced genomes of 141 mutant lines and determined its mutation rate to be 3.50 × 10−10 per site per generation. Extrapolating Ne additionally requires identifying population boundaries, which we defined using PopCOGenT and over 400 genomes related to AS9601. Accordingly, we calculated its Ne to be 1.68 × 107, which is only reasonably greater than that of endosymbiotic bacteria but surprisingly smaller than that of many free-living bacteria extrapolated using the same approach. Our results therefore suggest that genetic drift is a key driver of Prochlorococcus evolution. Based on an empirically estimated mutation rate, the authors show that the effective population size of Prochlorococcus, the most abundant carbon-fixing organisms in the ocean, is smaller than that of many free-living bacteria, suggesting an important role of drift in Prochlorococcus evolution.},
  archive      = {J_NATEE},
  author       = {Chen, Zhuoyu and Wang, Xiaojun and Song, Yu and Zeng, Qinglu and Zhang, Yao and Luo, Haiwei},
  doi          = {10.1038/s41559-021-01591-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {183-194},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Prochlorococcus have low global mutation rate and small effective population size},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hunting alters viral transmission and evolution in a large
carnivore. <em>NATEE</em>, <em>6</em>(2), 174–182. (<a
href="https://doi.org/10.1038/s41559-021-01635-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hunting can fundamentally alter wildlife population dynamics but the consequences of hunting on pathogen transmission and evolution remain poorly understood. Here, we present a study that leverages a unique landscape-scale quasi-experiment coupled with pathogen-transmission tracing, network simulation and phylodynamics to provide insights into how hunting shapes feline immunodeficiency virus (FIV) dynamics in puma (Puma concolor). We show that removing hunting pressure enhances the role of males in transmission, increases the viral population growth rate and increases the role of evolutionary forces on the pathogen compared to when hunting was reinstated. Changes in transmission observed with the removal of hunting could be linked to short-term social changes while the male puma population increased. These findings are supported through comparison with a region with stable hunting management over the same time period. This study shows that routine wildlife management can have impacts on pathogen transmission and evolution not previously considered. By conducting viral phylodynamic analysis on samples of puma feline immunodeficiency virus from regions with and without puma hunting, the authors show that stopping hunting disrupts male social structure and in turn influences viral dynamics.},
  archive      = {J_NATEE},
  author       = {Fountain-Jones, Nicholas M. and Kraberger, Simona and Gagne, Roderick B. and Gilbertson, Marie L. J. and Trumbo, Daryl R. and Charleston, Michael and Salerno, Patricia E. and Chris Funk, W. and Crooks, Kevin and Logan, Kenneth and Alldredge, Mathew and Dellicour, Simon and Baele, Guy and Didelot, Xavier and VandeWoude, Sue and Carver, Scott and Craft, Meggan E.},
  doi          = {10.1038/s41559-021-01635-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {174-182},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Hunting alters viral transmission and evolution in a large carnivore},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Global predictors of language endangerment and the future
of linguistic diversity. <em>NATEE</em>, <em>6</em>(2), 163–173. (<a
href="https://doi.org/10.1038/s41559-021-01604-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Language diversity is under threat. While each language is subject to specific social, demographic and political pressures, there may also be common threatening processes. We use an analysis of 6,511 spoken languages with 51 predictor variables spanning aspects of population, documentation, legal recognition, education policy, socioeconomic indicators and environmental features to show that, counter to common perception, contact with other languages per se is not a driver of language loss. However, greater road density, which may encourage population movement, is associated with increased endangerment. Higher average years of schooling is also associated with greater endangerment, evidence that formal education can contribute to loss of language diversity. Without intervention, language loss could triple within 40 years, with at least one language lost per month. To avoid the loss of over 1,500 languages by the end of the century, urgent investment is needed in language documentation, bilingual education programmes and other community-based programmes. Using a global analysis of 6,511 spoken languages with 51 predictor variables spanning aspects of population, documentation, legal recognition, education policy, socioeconomic indicators and environmental features, the authors identify predictors of current and future language endangerment and loss.},
  archive      = {J_NATEE},
  author       = {Bromham, Lindell and Dinnage, Russell and Skirgård, Hedvig and Ritchie, Andrew and Cardillo, Marcel and Meakins, Felicity and Greenhill, Simon and Hua, Xia},
  doi          = {10.1038/s41559-021-01604-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {163-173},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Global predictors of language endangerment and the future of linguistic diversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Radiocarbon dating from yuzhniy oleniy ostrov cemetery
reveals complex human responses to socio-ecological stress during the
8.2 ka cooling event. <em>NATEE</em>, <em>6</em>(2), 155–162. (<a
href="https://doi.org/10.1038/s41559-021-01628-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Yuzhniy Oleniy Ostrov in Karelia, northwest Russia, is one of the largest Early Holocene cemeteries in northern Eurasia, with 177 burials recovered in excavations in the 1930s; originally, more than 400 graves may have been present. A new radiocarbon dating programme, taking into account a correction for freshwater reservoir effects, suggests that the main use of the cemetery spanned only some 100–300 years, centring on ca. 8250 to 8000 cal bp. This coincides remarkably closely with the 8.2 ka cooling event, the most dramatic climatic downturn in the Holocene in the northern hemisphere, inviting an interpretation in terms of human response to a climate-driven environmental change. Rather than suggesting a simple deterministic relationship, we draw on a body of anthropological and archaeological theory to argue that the burial of the dead at this location served to demarcate and negotiate rights of access to a favoured locality with particularly rich and resilient fish and game stocks during a period of regional resource depression. This resulted in increased social stress in human communities that exceeded and subverted the ‘normal’ commitment of many hunter-gatherers to egalitarianism and widespread resource sharing, and gave rise to greater mortuary complexity. However, this seems to have lasted only for the duration of the climate downturn. Our results have implications for understanding the context of the emergence—and dissolution—of socio-economic inequality and territoriality under conditions of socio-ecological stress. New radiocarbon dates for the Mesolithic cemetery of Yuzhniy Oleniy Ostrov suggest that the main use of the site spanned only a few centuries, centring on a northern hemisphere climatic downturn event.},
  archive      = {J_NATEE},
  author       = {Schulting, Rick J. and Mannermaa, Kristiina and Tarasov, Pavel E. and Higham, Thomas and Ramsey, Christopher Bronk and Khartanovich, Valeri and Moiseyev, Vyacheslav and Gerasimov, Dmitriy and O’Shea, John and Weber, Andrzej},
  doi          = {10.1038/s41559-021-01628-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {155-162},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Radiocarbon dating from yuzhniy oleniy ostrov cemetery reveals complex human responses to socio-ecological stress during the 8.2 ka cooling event},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Colonial history and global economics distort our
understanding of deep-time biodiversity. <em>NATEE</em>, <em>6</em>(2),
145–154. (<a href="https://doi.org/10.1038/s41559-021-01608-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sampling biases in the fossil record distort estimates of past biodiversity. However, these biases not only reflect the geological and spatial aspects of the fossil record, but also the historical and current collation of fossil data. We demonstrate how the legacy of colonialism and socioeconomic factors, such as wealth, education and political stability, impact the global distribution of fossil data over the past 30 years. We find that a global power imbalance persists in palaeontology, with researchers in high- or upper-middle-income countries holding a monopoly over palaeontological knowledge production by contributing to 97% of fossil data. As a result, some countries or regions tend to be better sampled than others, ultimately leading to heterogeneous spatial sampling across the globe. This illustrates how efforts to mitigate sampling biases to obtain a truly representative view of past biodiversity are not disconnected from the aim of diversifying and decolonizing our discipline. Analysing the global distribution, source and authorship of fossil research over the past 30 years, the authors find that researchers in high- or upper-middle-income countries hold a monopoly over palaeontological knowledge production, contributing to 97% of fossil data in the Paleobiology Database, leading to disenfranchisement of researchers in lower-income countries and biased spatial sampling across the globe.},
  archive      = {J_NATEE},
  author       = {Raja, Nussaïbah B. and Dunne, Emma M. and Matiwane, Aviwe and Khan, Tasnuva Ming and Nätscher, Paulina S. and Ghilardi, Aline M. and Chattopadhyay, Devapriya},
  doi          = {10.1038/s41559-021-01608-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {145-154},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Colonial history and global economics distort our understanding of deep-time biodiversity},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). No pervasive relationship between species size and local
abundance trends. <em>NATEE</em>, <em>6</em>(2), 140–144. (<a
href="https://doi.org/10.1038/s41559-021-01624-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although there is some evidence that larger species could be more prone to population declines, the potential role of size traits in determining changes in community composition has been underexplored in global-scale analyses. Here, we combine a large cross-taxon assemblage time series database (BioTIME) with multiple trait databases to show that there is no clear correlation within communities between size traits and changes in abundance over time, suggesting that there is no consistent tendency for larger species to be doing proportionally better or worse than smaller species at local scales. Despite expectations that global anthropogenic pressures on species with communities may be size biased, this relationship has not been tested on a large scale. Here the authors use existing databases to show that larger species have not experienced more declines in abundance within their respective communities than small species.},
  archive      = {J_NATEE},
  author       = {Terry, J. Christopher D. and O’Sullivan, Jacob D. and Rossberg, Axel G.},
  doi          = {10.1038/s41559-021-01624-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {140-144},
  shortjournal = {Nat. Ecol. Evol},
  title        = {No pervasive relationship between species size and local abundance trends},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). When maths meets phytoplankton ecology. <em>NATEE</em>,
<em>6</em>(2), 138–139. (<a
href="https://doi.org/10.1038/s41559-021-01641-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Combined mutation rate estimation and reverse ecology sheds light on the forces shaping population size of Prochlorococcus, a major bacterial carbon sink.},
  archive      = {J_NATEE},
  author       = {Wielgoss, Sébastien},
  doi          = {10.1038/s41559-021-01641-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {138-139},
  shortjournal = {Nat. Ecol. Evol},
  title        = {When maths meets phytoplankton ecology},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hunting shapes wildlife disease transmission.
<em>NATEE</em>, <em>6</em>(2), 136–137. (<a
href="https://doi.org/10.1038/s41559-021-01633-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A landscape-level natural experiment in free-ranging pumas reveals how changes in hunting pressure alter viral evolution and infection dynamics through indirect effects on puma population size, demography and behaviour.},
  archive      = {J_NATEE},
  author       = {Kamath, Pauline L. and Prentice, Melanie B.},
  doi          = {10.1038/s41559-021-01633-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {136-137},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Hunting shapes wildlife disease transmission},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Smaller species are not better off. <em>NATEE</em>,
<em>6</em>(2), 134–135. (<a
href="https://doi.org/10.1038/s41559-021-01636-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Contrary to previous studies, an analysis of 7,000 plant and animal species shows that species size is unrelated to changes in their population abundance.},
  archive      = {J_NATEE},
  author       = {Leung, Brian},
  doi          = {10.1038/s41559-021-01636-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {134-135},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Smaller species are not better off},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Drivers of language loss. <em>NATEE</em>, <em>6</em>(2),
132–133. (<a href="https://doi.org/10.1038/s41559-021-01621-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A macroecological view suggests some global drivers of language endangerment and continuity, but a focus on individual languages will be important to stem the tide of language loss.},
  archive      = {J_NATEE},
  author       = {Bowern, Claire},
  doi          = {10.1038/s41559-021-01621-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {132-133},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Drivers of language loss},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Vulnerable reefs. <em>NATEE</em>, <em>6</em>(2), 131. (<a
href="https://doi.org/10.1038/s41559-021-01648-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Goymer, Patrick},
  doi          = {10.1038/s41559-021-01648-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {131},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Vulnerable reefs},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Amoebas innovate. <em>NATEE</em>, <em>6</em>(2), 130. (<a
href="https://doi.org/10.1038/s41559-021-01647-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-021-01647-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {130},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Amoebas innovate},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Hominins walk this way. <em>NATEE</em>, <em>6</em>(2), 129.
(<a href="https://doi.org/10.1038/s41559-021-01646-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-021-01646-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {129},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Hominins walk this way},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The rise of mammals. <em>NATEE</em>, <em>6</em>(2), 127–128.
(<a href="https://doi.org/10.1038/s41559-021-01643-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Simões, Tiago R.},
  doi          = {10.1038/s41559-021-01643-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {127-128},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The rise of mammals},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Include biodiversity representation indicators in area-based
conservation targets. <em>NATEE</em>, <em>6</em>(2), 123–126. (<a
href="https://doi.org/10.1038/s41559-021-01620-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in spatial biodiversity science and nationally available data have enabled the development of indicators that report on biodiversity outcomes, account for uneven global biodiversity between countries, and provide direct planning support. We urge their inclusion in the post-2020 global biodiversity framework.},
  archive      = {J_NATEE},
  author       = {Jetz, Walter and McGowan, Jennifer and Rinnan, D. Scott and Possingham, Hugh P. and Visconti, Piero and O’Donnell, Brian and Londoño-Murcia, Maria Cecilia},
  doi          = {10.1038/s41559-021-01620-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {123-126},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Include biodiversity representation indicators in area-based conservation targets},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Statistical inference, scale and noise in comparative
anthropology. <em>NATEE</em>, <em>6</em>(2), 122. (<a
href="https://doi.org/10.1038/s41559-021-01637-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Hamilton, Marcus J. and Tallavaara, Miikka},
  doi          = {10.1038/s41559-021-01637-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {122},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Statistical inference, scale and noise in comparative anthropology},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pending bill could devastate brazil’s serra do divisor
national park. <em>NATEE</em>, <em>6</em>(2), 120–121. (<a
href="https://doi.org/10.1038/s41559-021-01632-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Ruaro, Renata and Laurance, William F.},
  doi          = {10.1038/s41559-021-01632-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {120-121},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pending bill could devastate brazil’s serra do divisor national park},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conservation needs to integrate knowledge across scales.
<em>NATEE</em>, <em>6</em>(2), 118–119. (<a
href="https://doi.org/10.1038/s41559-021-01605-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Chaplin-Kramer, Rebecca and Brauman, Kate A. and Cavender-Bares, Jeannine and Díaz, Sandra and Duarte, Gabriela Teixeira and Enquist, Brian J. and Garibaldi, Lucas A. and Geldmann, Jonas and Halpern, Benjamin S. and Hertel, Thomas W. and Khoury, Colin K. and Krieger, Joana Madeira and Lavorel, Sandra and Mueller, Thomas and Neugarten, Rachel A. and Pinto-Ledezma, Jesús and Polasky, Stephen and Purvis, Andy and Reyes-García, Victoria and Roehrdanz, Patrick R. and Shannon, Lynne J. and Shaw, M. Rebecca and Strassburg, Bernardo B. N. and Tylianakis, Jason M. and Verburg, Peter H. and Visconti, Piero and Zafra-Calvo, Noelia},
  doi          = {10.1038/s41559-021-01605-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {118-119},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Conservation needs to integrate knowledge across scales},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Book-keeping. <em>NATEE</em>, <em>6</em>(2), 117. (<a
href="https://doi.org/10.1038/s41559-022-01681-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have recently rekindled our Books &amp;amp; Arts coverage.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01681-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {117},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Book-keeping},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Publisher correction: Collective behaviour can stabilize
ecosystems. <em>NATEE</em>, <em>6</em>(1), 116. (<a
href="https://doi.org/10.1038/s41559-021-01645-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Dalziel, Benjamin D. and Novak, Mark and Watson, James R. and Ellner, Stephen P.},
  doi          = {10.1038/s41559-021-01645-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {116},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Collective behaviour can stabilize ecosystems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Expression level is a major modifier of the fitness
landscape of a protein coding gene. <em>NATEE</em>, <em>6</em>(1),
103–115. (<a href="https://doi.org/10.1038/s41559-021-01578-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The phenotypic consequence of a genetic mutation depends on many factors including the expression level of a gene. However, a comprehensive quantification of this expression effect is still lacking, as is a further general mechanistic understanding of the effect. Here, we measured the fitness effect of almost all (&amp;gt;97.5%) single-nucleotide mutations in GFP, an exogenous gene with no physiological function, and URA3, a conditionally essential gene. Both genes were driven by two promoters whose expression levels differed by around tenfold. The resulting fitness landscapes revealed that the fitness effects of at least 42% of all single-nucleotide mutations within the genes were expression dependent. Although only a small fraction of variation in fitness effects among different mutations can be explained by biophysical properties of the protein and messenger RNA of the gene, our analyses revealed that the avoidance of stochastic molecular errors generally underlies the expression dependency of mutational effects and suggested protein misfolding as the most important type of molecular error among those examined. Our results therefore directly explained the slower evolution of highly expressed genes and highlighted cytotoxicity due to stochastic molecular errors as a non-negligible component for understanding the phenotypic consequence of mutations. Measuring fitness effects of mutations in the same gene expressed at two levels, the authors show expression dependency of fitness effects in at least 42% of all single-nucleotide mutations, probably driven by avoidance of stochastic molecular errors.},
  archive      = {J_NATEE},
  author       = {Wu, Zhuoxing and Cai, Xiujuan and Zhang, Xin and Liu, Yao and Tian, Guo-bao and Yang, Jian-Rong and Chen, Xiaoshu},
  doi          = {10.1038/s41559-021-01578-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {103-115},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Expression level is a major modifier of the fitness landscape of a protein coding gene},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial patterns of tumour growth impact clonal
diversification in a computational model and the TRACERx renal study.
<em>NATEE</em>, <em>6</em>(1), 88–102. (<a
href="https://doi.org/10.1038/s41559-021-01586-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic intra-tumour heterogeneity fuels clonal evolution, but our understanding of clinically relevant clonal dynamics remain limited. We investigated spatial and temporal features of clonal diversification in clear cell renal cell carcinoma through a combination of modelling and real tumour analysis. We observe that the mode of tumour growth, surface or volume, impacts the extent of subclonal diversification, enabling interpretation of clonal diversity in patient tumours. Specific patterns of proliferation and necrosis explain clonal expansion and emergence of parallel evolution and microdiversity in tumours. In silico time-course studies reveal the appearance of budding structures before detectable subclonal diversification. Intriguingly, we observe radiological evidence of budding structures in early-stage clear cell renal cell carcinoma, indicating that future clonal evolution may be predictable from imaging. Our findings offer a window into the temporal and spatial features of clinically relevant clonal evolution. A combined modelling and tumour analysis approach is used to study the temporal and spatial patterns of subclone evolution in the TRACERx renal study. Studying the tumour shape and spatial features of clonal diversity in early-stage tumours may allow the prediction of tumour progression and patterns of subclone diversification over time.},
  archive      = {J_NATEE},
  author       = {Fu, Xiao and Zhao, Yue and Lopez, Jose I. and Rowan, Andrew and Au, Lewis and Fendler, Annika and Hazell, Steve and Xu, Hang and Horswell, Stuart and Shepherd, Scott T. C. and Spencer, Charlotte E. and Spain, Lavinia and Byrne, Fiona and Stamp, Gordon and O’Brien, Tim and Nicol, David and Augustine, Marcellus and Chandra, Ashish and Rudman, Sarah and Toncheva, Antonia and Furness, Andrew J. S. and Pickering, Lisa and Kumar, Santosh and Koh, Dow-Mu and Messiou, Christina and Dafydd, Derfel ap and Orton, Matthew R. and Doran, Simon J. and Larkin, James and Swanton, Charles and Sahai, Erik and Litchfield, Kevin and Turajlic, Samra and Bates, Paul A.},
  doi          = {10.1038/s41559-021-01586-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {88-102},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Spatial patterns of tumour growth impact clonal diversification in a computational model and the TRACERx renal study},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Natural selection for imprecise vertical transmission in
host–microbiota systems. <em>NATEE</em>, <em>6</em>(1), 77–87. (<a
href="https://doi.org/10.1038/s41559-021-01593-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How and when the microbiome modulates host adaptation remains an evolutionary puzzle, despite evidence that the extended genetic repertoire of the microbiome can shape host phenotypes and fitness. One complicating factor is that the microbiome is often transmitted imperfectly across host generations, leading to questions about the degree to which the microbiome contributes to host adaptation. Here, using an evolutionary model, we demonstrate that decreasing vertical transmission fidelity can increase microbiome variation, and thus phenotypic variation, across hosts. When the most beneficial microbial genotypes change unpredictably from one generation to the next (for example, in variable environments), hosts can maximize fitness by increasing the microbiome variation among offspring, as this improves the chance of there being an offspring with the right microbial combination for the next generation. Imperfect vertical transmission can therefore be adaptive in varying environments. We characterize how selection on vertical transmission is shaped by environmental conditions, microbiome changes during host development and the contribution of other factors to trait variation. We illustrate how environmentally dependent microbial effects can favour intermediate transmission and set our results in the context of examples from natural systems. We also suggest research avenues to empirically test our predictions. Our model provides a basis to understand the evolutionary pathways that potentially led to the wide diversity of microbe transmission patterns found in nature. Vertical transmission is thought to favour beneficial host–microbe interactions, but these may also be context dependent. Here Bruijning et al. show with a model that variable environments can select for bet-hedging by hosts via imperfect vertical transmission of microbes.},
  archive      = {J_NATEE},
  author       = {Bruijning, Marjolein and Henry, Lucas P. and Forsberg, Simon K. G. and Metcalf, C. Jessica E. and Ayroles, Julien F.},
  doi          = {10.1038/s41559-021-01593-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {77-87},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Natural selection for imprecise vertical transmission in host–microbiota systems},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Gene family evolution underlies cell-type diversification in
the hypothalamus of teleosts. <em>NATEE</em>, <em>6</em>(1), 63–76. (<a
href="https://doi.org/10.1038/s41559-021-01580-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hundreds of cell types form the vertebrate brain but it is largely unknown how similar cellular repertoires are between or within species or how cell-type diversity evolves. To examine cell-type diversity across and within species, we performed single-cell RNA sequencing of ~130,000 hypothalamic cells from zebrafish (Danio rerio) and surface and cave morphs of Mexican tetra (Astyanax mexicanus). We found that over 75% of cell types were shared between zebrafish and Mexican tetra, which diverged from a common ancestor over 150 million years ago. Shared cell types displayed shifts in paralogue expression that were generated by subfunctionalization after genome duplication. Expression of terminal effector genes, such as neuropeptides, was more conserved than the expression of their associated transcriptional regulators. Species-specific cell types were enriched for the expression of species-specific genes and characterized by the neofunctionalization of expression patterns of members of recently expanded or contracted gene families. Comparisons between surface and cave morphs revealed differences in immune repertoires and transcriptional changes in neuropeptidergic cell types associated with genomic differences. The single-cell atlases presented here are a powerful resource to explore hypothalamic cell types and reveal how gene family evolution and shifts in paralogue expression contribute to cellular diversity. Single-cell transcriptomics of hypothalamic cells from zebrafish and surface and cave morphs of Mexican tetra shows conservation of cell types and that cellular novelty is associated with genetic novelty and the species-specific expression of paralogous genes.},
  archive      = {J_NATEE},
  author       = {Shafer, Maxwell E. R. and Sawh, Ahilya N. and Schier, Alexander F.},
  doi          = {10.1038/s41559-021-01580-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {63-76},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Gene family evolution underlies cell-type diversification in the hypothalamus of teleosts},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antigenic escape selects for the evolution of higher
pathogen transmission and virulence. <em>NATEE</em>, <em>6</em>(1),
51–62. (<a href="https://doi.org/10.1038/s41559-021-01603-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the propensity for complex and non-equilibrium dynamics in nature, eco-evolutionary analytical theory typically assumes that populations are at equilibria. In particular, pathogens often show antigenic escape from host immune defences, leading to repeated epidemics, fluctuating selection and diversification, but we do not understand how this impacts the evolution of virulence. We model the impact of antigenic drift and escape on the evolution of virulence in a generalized pathogen and apply a recently introduced oligomorphic methodology that captures the dynamics of the mean and variance of traits, to show analytically that these non-equilibrium dynamics select for the long-term persistence of more acute pathogens with higher virulence. Our analysis predicts both the timings and outcomes of antigenic shifts leading to repeated epidemics and predicts the increase in variation in both antigenicity and virulence before antigenic escape. There is considerable variation in the degree of antigenic escape that occurs across pathogens and our results may help to explain the difference in virulence between related pathogens including, potentially, human influenzas. Furthermore, it follows that these pathogens will have a lower R0, with clear implications for epidemic behaviour, endemic behaviour and control. More generally, our results show the importance of examining the evolutionary consequences of non-equilibrium dynamics. A model of pathogen evolution that allows for non-equilibrium dynamics finds that antigenic drift and escape select for the long-term persistence of more acute pathogens with higher virulence.},
  archive      = {J_NATEE},
  author       = {Sasaki, Akira and Lion, Sébastien and Boots, Mike},
  doi          = {10.1038/s41559-021-01603-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {51-62},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Antigenic escape selects for the evolution of higher pathogen transmission and virulence},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Climatic and soil factors explain the two-dimensional
spectrum of global plant trait variation. <em>NATEE</em>, <em>6</em>(1),
36–50. (<a href="https://doi.org/10.1038/s41559-021-01616-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant functional traits can predict community assembly and ecosystem functioning and are thus widely used in global models of vegetation dynamics and land–climate feedbacks. Still, we lack a global understanding of how land and climate affect plant traits. A previous global analysis of six traits observed two main axes of variation: (1) size variation at the organ and plant level and (2) leaf economics balancing leaf persistence against plant growth potential. The orthogonality of these two axes suggests they are differently influenced by environmental drivers. We find that these axes persist in a global dataset of 17 traits across more than 20,000 species. We find a dominant joint effect of climate and soil on trait variation. Additional independent climate effects are also observed across most traits, whereas independent soil effects are almost exclusively observed for economics traits. Variation in size traits correlates well with a latitudinal gradient related to water or energy limitation. In contrast, variation in economics traits is better explained by interactions of climate with soil fertility. These findings have the potential to improve our understanding of biodiversity patterns and our predictions of climate change impacts on biogeochemical cycles. The authors investigate the broad-scale climatological and soil properties that co-vary with major axes of plant functional traits. They find that variation in plant size is attributed to latitudinal gradients in water or energy limitation, while variation in leaf economics traits is attributed to both climate and soil fertility including their interaction.},
  archive      = {J_NATEE},
  author       = {Joswig, Julia S. and Wirth, Christian and Schuman, Meredith C. and Kattge, Jens and Reu, Björn and Wright, Ian J. and Sippel, Sebastian D. and Rüger, Nadja and Richter, Ronny and Schaepman, Michael E. and van Bodegom, Peter M. and Cornelissen, J. H. C. and Díaz, Sandra and Hattingh, Wesley N. and Kramer, Koen and Lens, Frederic and Niinemets, Ülo and Reich, Peter B. and Reichstein, Markus and Römermann, Christine and Schrodt, Franziska and Anand, Madhur and Bahn, Michael and Byun, Chaeho and Campetella, Giandiego and Cerabolini, Bruno E. L. and Craine, Joseph M. and Gonzalez-Melo, Andres and Gutiérrez, Alvaro G. and He, Tianhua and Higuchi, Pedro and Jactel, Hervé and Kraft, Nathan J. B. and Minden, Vanessa and Onipchenko, Vladimir and Peñuelas, Josep and Pillar, Valério D. and Sosinski, Ênio and Soudzilovskaia, Nadejda A. and Weiher, Evan and Mahecha, Miguel D.},
  doi          = {10.1038/s41559-021-01616-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {36-50},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Climatic and soil factors explain the two-dimensional spectrum of global plant trait variation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The earliest denisovans and their cultural adaptation.
<em>NATEE</em>, <em>6</em>(1), 28–35. (<a
href="https://doi.org/10.1038/s41559-021-01581-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since the initial identification of the Denisovans a decade ago, only a handful of their physical remains have been discovered. Here we analysed ~3,800 non-diagnostic bone fragments using collagen peptide mass fingerprinting to locate new hominin remains from Denisova Cave (Siberia, Russia). We identified five new hominin bones, four of which contained sufficient DNA for mitochondrial analysis. Three carry mitochondrial DNA of the Denisovan type and one was found to carry mtDNA of the Neanderthal type. The former come from the same archaeological layer near the base of the cave’s sequence and are the oldest securely dated evidence of Denisovans at 200 ka (thousand years ago) (205–192 ka at 68.2% or 217–187 ka at 95% probability). The stratigraphic context in which they were located contains a wealth of archaeological material in the form of lithics and faunal remains, allowing us to determine the material culture associated with these early hominins and explore their behavioural and environmental adaptations. The combination of bone collagen fingerprinting and genetic analyses has so far more-than-doubled the number of hominin bones at Denisova Cave and has expanded our understanding of Denisovan and Neanderthal interactions, as well as their archaeological signatures. Identifying Neanderthal and Denisovan bone fragments using collagen peptide mass fingerprinting and mitochondrial DNA analysis at Denisova Cave, the authors are able to date the earliest secure Denisovan presence at the cave to c. 200 ka. The stratigraphic association with lithics and faunal remains allows the authors to explore the behavioural and environmental adaptations of these elusive hominins.},
  archive      = {J_NATEE},
  author       = {Brown, Samantha and Massilani, Diyendo and Kozlikin, Maxim B. and Shunkov, Michael V. and Derevianko, Anatoly P. and Stoessel, Alexander and Jope-Street, Blair and Meyer, Matthias and Kelso, Janet and Pääbo, Svante and Higham, Thomas and Douka, Katerina},
  doi          = {10.1038/s41559-021-01581-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {28-35},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The earliest denisovans and their cultural adaptation},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatial models of tumour evolution. <em>NATEE</em>,
<em>6</em>(1), 26–27. (<a
href="https://doi.org/10.1038/s41559-021-01584-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatiotemporal modelling of tumours detects at least two distinct models of cancer evolution and reveals the influence of necrosis in enhancing the metastatic potential in both models.},
  archive      = {J_NATEE},
  author       = {Gaines, Dakim K. and Rathmell, W. Kimryn},
  doi          = {10.1038/s41559-021-01584-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {26-27},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Spatial models of tumour evolution},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antigenic evolution can drive virulence evolution.
<em>NATEE</em>, <em>6</em>(1), 24–25. (<a
href="https://doi.org/10.1038/s41559-021-01600-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Repeated epidemic waves, characteristic of an antigenically evolving pathogen, can drive selection for higher pathogen virulence},
  archive      = {J_NATEE},
  author       = {Mideo, Nicole and Kamiya, Tsukushi},
  doi          = {10.1038/s41559-021-01600-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {24-25},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Antigenic evolution can drive virulence evolution},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Environmental drivers of plant form and function.
<em>NATEE</em>, <em>6</em>(1), 22–23. (<a
href="https://doi.org/10.1038/s41559-021-01576-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A global synthesis of plant traits finds that climate and soil variables explain two key axes of trait variation, offering a new framework to understand how the environment shapes plant form and function.},
  archive      = {J_NATEE},
  author       = {Thomas, Haydn J. D.},
  doi          = {10.1038/s41559-021-01576-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {22-23},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Environmental drivers of plant form and function},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). The wilderness myth. <em>NATEE</em>, <em>6</em>(1), 21. (<a
href="https://doi.org/10.1038/s41559-021-01630-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-021-01630-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {21},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The wilderness myth},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). An ocean of benefits. <em>NATEE</em>, <em>6</em>(1), 20.
(<a href="https://doi.org/10.1038/s41559-021-01629-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-021-01629-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {20},
  shortjournal = {Nat. Ecol. Evol},
  title        = {An ocean of benefits},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022g). Rainforest origins. <em>NATEE</em>, <em>6</em>(1), 19. (<a
href="https://doi.org/10.1038/s41559-021-01618-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-021-01618-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {19},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Rainforest origins},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). New asian hominin. <em>NATEE</em>, <em>6</em>(1), 18. (<a
href="https://doi.org/10.1038/s41559-021-01617-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-021-01617-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {18},
  shortjournal = {Nat. Ecol. Evol},
  title        = {New asian hominin},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Falcon flyways. <em>NATEE</em>, <em>6</em>(1), 17. (<a
href="https://doi.org/10.1038/s41559-021-01627-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-021-01627-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {17},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Falcon flyways},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Long tail of land use. <em>NATEE</em>, <em>6</em>(1), 16.
(<a href="https://doi.org/10.1038/s41559-021-01622-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-021-01622-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {16},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Long tail of land use},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Deeply tangled evolution. <em>NATEE</em>, <em>6</em>(1),
15. (<a href="https://doi.org/10.1038/s41559-021-01631-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Jones, Emily},
  doi          = {10.1038/s41559-021-01631-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {15},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deeply tangled evolution},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Stable network cores. <em>NATEE</em>, <em>6</em>(1), 14.
(<a href="https://doi.org/10.1038/s41559-021-01625-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Jones, Emily},
  doi          = {10.1038/s41559-021-01625-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {14},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Stable network cores},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Alerts protect forests. <em>NATEE</em>, <em>6</em>(1), 13.
(<a href="https://doi.org/10.1038/s41559-021-01611-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-021-01611-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {13},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Alerts protect forests},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022f). Ocean blooms. <em>NATEE</em>, <em>6</em>(1), 12. (<a
href="https://doi.org/10.1038/s41559-021-01610-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-021-01610-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {12},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ocean blooms},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022d). Puzzling loss of y. <em>NATEE</em>, <em>6</em>(1), 11. (<a
href="https://doi.org/10.1038/s41559-021-01623-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-021-01623-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {11},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Puzzling loss of y},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022e). SARS-CoV-2 roots. <em>NATEE</em>, <em>6</em>(1), 10. (<a
href="https://doi.org/10.1038/s41559-021-01612-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-021-01612-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {10},
  shortjournal = {Nat. Ecol. Evol},
  title        = {SARS-CoV-2 roots},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Dinosaur dossier. <em>NATEE</em>, <em>6</em>(1), 9. (<a
href="https://doi.org/10.1038/s41559-021-01590-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-021-01590-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dinosaur dossier},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Promise and peril of ecological and evolutionary modelling
using cross-cultural datasets. <em>NATEE</em>, <em>6</em>(1), 6–8. (<a
href="https://doi.org/10.1038/s41559-021-01579-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comparative analysis using cross-cultural data has a long tradition in anthropology and applications by ecologically minded researchers have exploded in the past decade. Here we discuss problems and recommend solutions for use of cross-cultural datasets in ecology.},
  archive      = {J_NATEE},
  author       = {Bliege Bird, Rebecca and Codding, Brian F.},
  doi          = {10.1038/s41559-021-01579-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {6-8},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Promise and peril of ecological and evolutionary modelling using cross-cultural datasets},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrate geographic scales in equity, diversity and
inclusion. <em>NATEE</em>, <em>6</em>(1), 4–5. (<a
href="https://doi.org/10.1038/s41559-021-01609-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Khelifa, Rassim and Mahdjoub, Hayat},
  doi          = {10.1038/s41559-021-01609-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {4-5},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Integrate geographic scales in equity, diversity and inclusion},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The moral and legal imperative to return illegally exported
fossils. <em>NATEE</em>, <em>6</em>(1), 2–3. (<a
href="https://doi.org/10.1038/s41559-021-01588-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Cisneros, Juan C. and Ghilardi, Aline M. and Raja, Nussaïbah B. and Stewens, Paul P.},
  doi          = {10.1038/s41559-021-01588-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {2-3},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The moral and legal imperative to return illegally exported fossils},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The first five years. <em>NATEE</em>, <em>6</em>(1), 1. (<a
href="https://doi.org/10.1038/s41559-021-01649-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We celebrate our fifth anniversary with a look back at publications that illustrate the breadth of our scope.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-021-01649-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The first five years},
  volume       = {6},
  year         = {2022},
}
</textarea>
</details></li>
</ul>

</body>
</html>
