<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt---388">NBT - 388</h2>
<ul>
<li><details>
<summary>
(2020f). Forum: The t cell wars. <em>NBT</em>, <em>39</em>(1), 26.
(<a href="https://doi.org/10.1038/s41587-020-00783-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00783-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {26},
  shortjournal = {Nature Biotech.},
  title        = {Forum: The t cell wars},
  volume       = {39},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). People. <em>NBT</em>, <em>38</em>(12), 1484. (<a
href="https://doi.org/10.1038/s41587-020-00759-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00759-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1484},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The art of virtual mentoring in the twenty-first century for
STEM majors and beyond. <em>NBT</em>, <em>38</em>(12), 1477–1482. (<a
href="https://doi.org/10.1038/s41587-020-00758-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Social quarantine has severely impacted the training of new scientists, but the use of virtual platforms can supplement mentorship, making career development seamless.},
  archive      = {J_NBT},
  author       = {McReynolds, Melanie R. and Termini, Christina M. and Hinton, Antentor O. and Taylor, Brittany L. and Vue, Zer and Huang, Sunny C. and Roby, ReAnna S. and Shuler, Haysetta and Carter, Calvin S.},
  doi          = {10.1038/s41587-020-00758-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1477-1482},
  shortjournal = {Nature Biotech.},
  title        = {The art of virtual mentoring in the twenty-first century for STEM majors and beyond},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Integrating microarray-based spatial
transcriptomics and single-cell RNA-seq reveals tissue architecture in
pancreatic ductal adenocarcinomas. <em>NBT</em>, <em>38</em>(12), 1476.
(<a href="https://doi.org/10.1038/s41587-020-00776-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-019-0392-8.},
  archive      = {J_NBT},
  author       = {Moncada, Reuben and Barkley, Dalia and Wagner, Florian and Chiodin, Marta and Devlin, Joseph C. and Baron, Maayan and Hajdu, Cristina H. and Simeone, Diane M. and Yanai, Itai},
  doi          = {10.1038/s41587-020-00776-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1476},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Integrating microarray-based spatial transcriptomics and single-cell RNA-seq reveals tissue architecture in pancreatic ductal adenocarcinomas},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Automated design of thousands of nonrepetitive parts for
engineering stable genetic systems. <em>NBT</em>, <em>38</em>(12),
1466–1475. (<a href="https://doi.org/10.1038/s41587-020-0584-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered genetic systems are prone to failure when their genetic parts contain repetitive sequences. Designing many nonrepetitive genetic parts with desired functionalities remains a difficult challenge with high computational complexity. To overcome this challenge, we developed the Nonrepetitive Parts Calculator to rapidly generate thousands of highly nonrepetitive genetic parts from specified design constraints, including promoters, ribosome-binding sites and terminators. As a demonstration, we designed and experimentally characterized 4,350 nonrepetitive bacterial promoters with transcription rates that varied across a 820,000-fold range, and 1,722 highly nonrepetitive yeast promoters with transcription rates that varied across a 25,000-fold range. We applied machine learning to explain how specific interactions controlled the promoters’ transcription rates. We also show that using nonrepetitive genetic parts substantially reduces homologous recombination, resulting in greater genetic stability. An algorithm to design nonrepetitive genetic parts enables synthetic biologists to reliably engineer stable circuits.},
  archive      = {J_NBT},
  author       = {Hossain, Ayaan and Lopez, Eriberto and Halper, Sean M. and Cetnar, Daniel P. and Reis, Alexander C. and Strickland, Devin and Klavins, Eric and Salis, Howard M.},
  doi          = {10.1038/s41587-020-0584-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1466-1475},
  shortjournal = {Nature Biotech.},
  title        = {Automated design of thousands of nonrepetitive parts for engineering stable genetic systems},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Precise, predictable multi-nucleotide deletions in rice and
wheat using APOBEC–Cas9. <em>NBT</em>, <em>38</em>(12), 1460–1465. (<a
href="https://doi.org/10.1038/s41587-020-0566-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Short insertions and deletions can be produced in plant genomes using CRISPR–Cas editors, but reliable production of larger deletions in specific target sites has proven difficult to achieve. We report the development of a series of APOBEC–Cas9 fusion-induced deletion systems (AFIDs) that combine Cas9 with human APOBEC3A (A3A), uracil DNA-glucosidase and apurinic or apyrimidinic site lyase. In rice and wheat, AFID-3 generated deletions from 5′-deaminated C bases to the Cas9-cleavage site. Approximately one-third of deletions produced using AFID-3 in rice and wheat protoplasts (30.2\%) and regenerated plants (34.8\%) were predictable. We show that eAFID-3, in which the A3A in AFID-3 is replaced with truncated APOBEC3B (A3Bctd), produced more uniform deletions from the preferred TC motif to the double-strand break. AFIDs could be applied to study regulatory regions and protein domains to improve crop plants. A suite of APOBEC–Cas9 complexes for predictable and targeted deletions in plant genomes.},
  archive      = {J_NBT},
  author       = {Wang, Shengxing and Zong, Yuan and Lin, Qiupeng and Zhang, Huawei and Chai, Zhuangzhuang and Zhang, Dandan and Chen, Kunling and Qiu, Jin-Long and Gao, Caixia},
  doi          = {10.1038/s41587-020-0566-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1460-1465},
  shortjournal = {Nature Biotech.},
  title        = {Precise, predictable multi-nucleotide deletions in rice and wheat using APOBEC–Cas9},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Cell-free biosensors for rapid detection of water
contaminants. <em>NBT</em>, <em>38</em>(12), 1451–1459. (<a
href="https://doi.org/10.1038/s41587-020-0571-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lack of access to safe drinking water is a global problem, and methods to reliably and easily detect contaminants could be transformative. We report the development of a cell-free in vitro transcription system that uses RNA Output Sensors Activated by Ligand Induction (ROSALIND) to detect contaminants in water. A combination of highly processive RNA polymerases, allosteric protein transcription factors and synthetic DNA transcription templates regulates the synthesis of a fluorescence-activating RNA aptamer. The presence of a target contaminant induces the transcription of the aptamer, and a fluorescent signal is produced. We apply ROSALIND to detect a range of water contaminants, including antibiotics, small molecules and metals. We also show that adding RNA circuitry can invert responses, reduce crosstalk and improve sensitivity without protein engineering. The ROSALIND system can be freeze-dried for easy storage and distribution, and we apply it in the field to test municipal water supplies, demonstrating its potential use for monitoring water quality. Water contaminants are detected with freeze-dried, regulated in vitro transcription reactions.},
  archive      = {J_NBT},
  author       = {Jung, Jaeyoung K. and Alam, Khalid K. and Verosloff, Matthew S. and Capdevila, Daiana A. and Desmau, Morgane and Clauer, Phillip R. and Lee, Jeong Wook and Nguyen, Peter Q. and Pastén, Pablo A. and Matiasek, Sandrine J. and Gaillard, Jean-François and Giedroc, David P. and Collins, James J. and Lucks, Julius B.},
  doi          = {10.1038/s41587-020-0571-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1451-1459},
  shortjournal = {Nature Biotech.},
  title        = {Cell-free biosensors for rapid detection of water contaminants},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Metabolic engineering generates a transgene-free safety
switch for cell therapy. <em>NBT</em>, <em>38</em>(12), 1441–1450. (<a
href="https://doi.org/10.1038/s41587-020-0580-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Safeguard mechanisms can ameliorate the potential risks associated with cell therapies but currently rely on the introduction of transgenes. This limits their application owing to immunogenicity or transgene silencing. We aimed to create a control mechanism for human cells that is not mediated by a transgene. Using genome editing methods, we disrupt uridine monophosphate synthetase (UMPS) in the pyrimidine de novo synthesis pathway in cell lines, pluripotent cells and primary human T cells. We show that this makes proliferation dependent on external uridine and enables us to control cell growth by modulating the uridine supply, both in vitro and in vivo after transplantation in xenograft models. Additionally, disrupting this pathway creates resistance to 5-fluoroorotic acid, which enables positive selection of UMPS-knockout cells. We envision that this approach will add an additional level of safety to cell therapies and therefore enable the development of approaches with higher risks, especially those that are intended for limited treatment durations. A new safety switch allows for control of cell therapies by dietary uridine.},
  archive      = {J_NBT},
  author       = {Wiebking, Volker and Patterson, James O. and Martin, Renata and Chanda, Monica K. and Lee, Ciaran M. and Srifa, Waracharee and Bao, Gang and Porteus, Matthew H.},
  doi          = {10.1038/s41587-020-0580-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1441-1450},
  shortjournal = {Nature Biotech.},
  title        = {Metabolic engineering generates a transgene-free safety switch for cell therapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Programmable m6A modification of cellular RNAs with a
cas13-directed methyltransferase. <em>NBT</em>, <em>38</em>(12),
1431–1440. (<a href="https://doi.org/10.1038/s41587-020-0572-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {N6-Methyladenosine (m6A) is the most widespread internal messenger RNA modification in humans. Despite recent progress in understanding the biological roles of m6A, the inability to install m6A site specifically in individual transcripts has hampered efforts to elucidate causal relationships between the presence of a specific m6A and phenotypic outcomes. In the present study, we demonstrate that nucleus-localized dCas13 fusions with a truncated METTL3 methyltransferase domain and cytoplasm-localized fusions with a modified METTL3:METTL14 methyltransferase complex can direct site-specific m6A incorporation in distinct cellular compartments, with the former fusion protein having particularly low off-target activity. Independent cellular assays across multiple sites confirm that this targeted RNA methylation (TRM) system mediates efficient m6A installation in endogenous RNA transcripts with high specificity. Finally, we show that TRM can induce m6A-mediated changes to transcript abundance and alternative splicing. These findings establish TRM as a tool for targeted epitranscriptome engineering that can reveal the effect of individual m6A modifications and dissect their functional roles. The most abundant RNA modification in humans, m6A, can be installed at specified RNA sequences in cells, enabling functional studies.},
  archive      = {J_NBT},
  author       = {Wilson, Christopher and Chen, Peter J. and Miao, Zhuang and Liu, David R.},
  doi          = {10.1038/s41587-020-0572-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1431-1440},
  shortjournal = {Nature Biotech.},
  title        = {Programmable m6A modification of cellular RNAs with a cas13-directed methyltransferase},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Generation of human striatal organoids and cortico-striatal
assembloids from human pluripotent stem cells. <em>NBT</em>,
<em>38</em>(12), 1421–1430. (<a
href="https://doi.org/10.1038/s41587-020-00763-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cortico-striatal projections are critical components of forebrain circuitry that regulate motivated behaviors. To enable the study of the human cortico-striatal pathway and how its dysfunction leads to neuropsychiatric disease, we developed a method to convert human pluripotent stem cells into region-specific brain organoids that resemble the developing human striatum and include electrically active medium spiny neurons. We then assembled these organoids with cerebral cortical organoids in three-dimensional cultures to form cortico-striatal assembloids. Using viral tracing and functional assays in intact or sliced assembloids, we show that cortical neurons send axonal projections into striatal organoids and form synaptic connections. Medium spiny neurons mature electrophysiologically following assembly and display calcium activity after optogenetic stimulation of cortical neurons. Moreover, we derive cortico-striatal assembloids from patients with a neurodevelopmental disorder caused by a deletion on chromosome 22q13.3 and capture disease-associated defects in calcium activity, showing that this approach will allow investigation of the development and functional assembly of cortico-striatal connectivity using patient-derived cells. Organoids representing the human striatum are generated and used to build circuits in cortico-striatal assembloids.},
  archive      = {J_NBT},
  author       = {Miura, Yuki and Li, Min-Yin and Birey, Fikri and Ikeda, Kazuya and Revah, Omer and Thete, Mayuri Vijay and Park, Jin-Young and Puno, Alyssa and Lee, Samuel H. and Porteus, Matthew H. and Pașca, Sergiu P.},
  doi          = {10.1038/s41587-020-00763-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1421-1430},
  shortjournal = {Nature Biotech.},
  title        = {Generation of human striatal organoids and cortico-striatal assembloids from human pluripotent stem cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A dual-constriction biological nanopore resolves
homonucleotide sequences with high fidelity. <em>NBT</em>,
<em>38</em>(12), 1415–1420. (<a
href="https://doi.org/10.1038/s41587-020-0570-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule long-read DNA sequencing with biological nanopores is fast and high-throughput but suffers reduced accuracy in homonucleotide stretches. We now combine the CsgG nanopore with the 35-residue N-terminal region of its extracellular interaction partner CsgF to produce a dual-constriction pore with improved signal and base-calling accuracy for homopolymer regions. The electron cryo-microscopy structure of CsgG in complex with full-length CsgF shows that the 33 N-terminal residues of CsgF bind inside the β-barrel of the pore, forming a defined second constriction. In complexes of CsgG bound to a 35-residue CsgF constriction peptide, the second constriction is separated from the primary constriction by ~25 Å. We find that both constrictions contribute to electrical signal modulation during single-stranded DNA translocation. DNA sequencing using a prototype CsgG–CsgF protein pore with two constrictions improved single-read accuracy by 25 to 70\% in homopolymers up to 9 nucleotides long. Equipping a protein nanopore with a second constriction improves sequencing of homopolymer DNA stretches.},
  archive      = {J_NBT},
  author       = {Van der Verren, Sander E. and Van Gerven, Nani and Jonckheere, Wim and Hambley, Richard and Singh, Pratik and Kilgour, John and Jordan, Michael and Wallace, E. Jayne and Jayasinghe, Lakmal and Remaut, Han},
  doi          = {10.1038/s41587-020-0570-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1415-1420},
  shortjournal = {Nature Biotech.},
  title        = {A dual-constriction biological nanopore resolves homonucleotide sequences with high fidelity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Generalizing RNA velocity to transient cell states through
dynamical modeling. <em>NBT</em>, <em>38</em>(12), 1408–1414. (<a
href="https://doi.org/10.1038/s41587-020-0591-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA velocity has opened up new ways of studying cellular differentiation in single-cell RNA-sequencing data. It describes the rate of gene expression change for an individual gene at a given time point based on the ratio of its spliced and unspliced messenger RNA (mRNA). However, errors in velocity estimates arise if the central assumptions of a common splicing rate and the observation of the full splicing dynamics with steady-state mRNA levels are violated. Here we present scVelo, a method that overcomes these limitations by solving the full transcriptional dynamics of splicing kinetics using a likelihood-based dynamical model. This generalizes RNA velocity to systems with transient cell states, which are common in development and in response to perturbations. We apply scVelo to disentangling subpopulation kinetics in neurogenesis and pancreatic endocrinogenesis. We infer gene-specific rates of transcription, splicing and degradation, recover each cell’s position in the underlying differentiation processes and detect putative driver genes. scVelo will facilitate the study of lineage decisions and gene regulation. scVelo reconstructs transient cell states and differentiation pathways from single-cell RNA-sequencing data.},
  archive      = {J_NBT},
  author       = {Bergen, Volker and Lange, Marius and Peidli, Stefan and Wolf, F. Alexander and Theis, Fabian J.},
  doi          = {10.1038/s41587-020-0591-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1408-1414},
  shortjournal = {Nature Biotech.},
  title        = {Generalizing RNA velocity to transient cell states through dynamical modeling},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Targeted, efficient sequence insertion and replacement in
rice. <em>NBT</em>, <em>38</em>(12), 1402–1407. (<a
href="https://doi.org/10.1038/s41587-020-0581-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas9 methods have been applied to generate random insertions and deletions, large deletions, targeted insertions or replacements of short sequences, and precise base changes in plants1–7. However, versatile methods for targeted insertion or replacement of long sequences and genes, which are needed for functional genomics studies and trait improvement in crops, are few and largely depend on the use of selection markers8–11. Building on methods developed in mammalian cells12, we used chemically modified donor DNA and CRISPR–Cas9 to insert sequences of up to 2,049 base pairs (bp), including enhancers and promoters, into the rice genome at an efficiency of 25\%. We also report a method for gene replacement that relies on homology-directed repair, chemically modified donor DNA and the presence of tandem repeats at target sites, achieving replacement with up to 130-bp sequences at 6.1\% efficiency. Rice genes are replaced using chemically modified donor DNA and CRISPR targeting.},
  archive      = {J_NBT},
  author       = {Lu, Yuming and Tian, Yifu and Shen, Rundong and Yao, Qi and Wang, Mugui and Chen, Mei and Dong, Jinsong and Zhang, Tongen and Li, Feng and Lei, Mingguang and Zhu, Jian-Kang},
  doi          = {10.1038/s41587-020-0581-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1402-1407},
  shortjournal = {Nature Biotech.},
  title        = {Targeted, efficient sequence insertion and replacement in rice},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Generation of paternal haploids in wheat by genome editing
of the centromeric histone CENH3. <em>NBT</em>, <em>38</em>(12),
1397–1401. (<a href="https://doi.org/10.1038/s41587-020-0728-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New breeding technologies accelerate germplasm improvement and reduce the cost of goods in seed production1–3. Many such technologies could use in vivo paternal haploid induction (HI), which occurs when double fertilization precedes maternal (egg cell) genome loss. Engineering of the essential CENTROMERIC HISTONE (CENH3) gene induces paternal HI in Arabidopsis4–6. Despite conservation of CENH3 function across crops, CENH3-based HI has not been successful outside of the Arabidopsis model system7. Here we report a commercially operable paternal HI line in wheat with a ~7\% HI rate, identified by screening genome-edited TaCENH3α-heteroallelic combinations. Unlike in Arabidopsis, edited alleles exhibited reduced transmission in female gametophytes, and heterozygous genotypes triggered higher HI rates than homozygous combinations. These developments might pave the way for the deployment of CENH3 HI technology in diverse crops. Gene editing induces paternal haploid plants in hexaploid wheat.},
  archive      = {J_NBT},
  author       = {Lv, Jian and Yu, Kun and Wei, Juan and Gui, Huaping and Liu, Chunxia and Liang, Dawei and Wang, Yanli and Zhou, Hongju and Carlin, Ryan and Rich, Randy and Lu, Tiancong and Que, Qiudeng and Wang, Wen Chung and Zhang, Xingping and Kelliher, Tim},
  doi          = {10.1038/s41587-020-0728-4},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1397-1401},
  shortjournal = {Nature Biotech.},
  title        = {Generation of paternal haploids in wheat by genome editing of the centromeric histone CENH3},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Chimeric antigen receptors. <em>NBT</em>, <em>38</em>(12),
1395. (<a href="https://doi.org/10.1038/s41587-020-00766-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents for methods of making and using chimeric antigen receptor (CAR) T cells to treat cancer and other diseases.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00766-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1395},
  shortjournal = {Nature Biotech.},
  title        = {Chimeric antigen receptors},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The global chimeric antigen receptor t (CAR-t) cell therapy
patent landscape. <em>NBT</em>, <em>38</em>(12), 1387–1394. (<a
href="https://doi.org/10.1038/s41587-020-00749-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global patents in the field of CAR-T cell therapy show a changing landscape with fierce competition and intensive collaboration.},
  archive      = {J_NBT},
  author       = {Lyu, Liyang and Feng, Ye and Chen, Xin and Hu, Yuanjia},
  doi          = {10.1038/s41587-020-00749-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1387-1394},
  shortjournal = {Nature Biotech.},
  title        = {The global chimeric antigen receptor t (CAR-t) cell therapy patent landscape},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Sample sharing. <em>NBT</em>, <em>38</em>(12), 1386. (<a
href="https://doi.org/10.1038/s41587-020-00752-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kamens, Joanne},
  doi          = {10.1038/s41587-020-00752-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1386},
  shortjournal = {Nature Biotech.},
  title        = {Sample sharing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Guidelines for reporting single-cell RNA-seq experiments.
<em>NBT</em>, <em>38</em>(12), 1384–1386. (<a
href="https://doi.org/10.1038/s41587-020-00744-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Füllgrabe, Anja and George, Nancy and Green, Matthew and Nejad, Parisa and Aronow, Bruce and Fexova, Silvie Korena and Fischer, Clay and Freeberg, Mallory Ann and Huerta, Laura and Morrison, Norman and Scheuermann, Richard H. and Taylor, Deanne and Vasilevsky, Nicole and Clarke, Laura and Gehlenborg, Nils and Kent, Jim and Marioni, John and Teichmann, Sarah and Brazma, Alvis and Papatheodorou, Irene},
  doi          = {10.1038/s41587-020-00744-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1384-1386},
  shortjournal = {Nature Biotech.},
  title        = {Guidelines for reporting single-cell RNA-seq experiments},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Supporting US public health experts. <em>NBT</em>,
<em>38</em>(12), 1381–1384. (<a
href="https://doi.org/10.1038/s41587-020-00767-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Cohen, Ron and Francois, Cedric and Crowley, John and Hastings, Paul and King, Rachel and Love, Ted W. and Maraganore, John and McMurry-Heath, Michelle and Levin, Jeremy},
  doi          = {10.1038/s41587-020-00767-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1381-1384},
  shortjournal = {Nature Biotech.},
  title        = {Supporting US public health experts},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Increased cyber-biosecurity for DNA synthesis. <em>NBT</em>,
<em>38</em>(12), 1379–1381. (<a
href="https://doi.org/10.1038/s41587-020-00761-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Puzis, Rami and Farbiash, Dor and Brodt, Oleg and Elovici, Yuval and Greenbaum, Dov},
  doi          = {10.1038/s41587-020-00761-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1379-1381},
  shortjournal = {Nature Biotech.},
  title        = {Increased cyber-biosecurity for DNA synthesis},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). FDALabel for drug repurposing studies and beyond.
<em>NBT</em>, <em>38</em>(12), 1378–1379. (<a
href="https://doi.org/10.1038/s41587-020-00751-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Fang, Hong and Harris, Stephen and Liu, Zhichao and Thakkar, Shraddha and Yang, Junshuang and Ingle, Taylor and Xu, Joshua and Lesko, Lawrence and Rosario, Lilliam and Tong, Weida},
  doi          = {10.1038/s41587-020-00751-0},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1378-1379},
  shortjournal = {Nature Biotech.},
  title        = {FDALabel for drug repurposing studies and beyond},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Around the world in a month. <em>NBT</em>, <em>38</em>(12),
1370. (<a href="https://doi.org/10.1038/s41587-020-00764-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00764-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1370},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DARPins stack up as anti-COVID-19 agents. <em>NBT</em>,
<em>38</em>(12), 1369. (<a
href="https://doi.org/10.1038/s41587-020-00771-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00771-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1369},
  shortjournal = {Nature Biotech.},
  title        = {DARPins stack up as anti-COVID-19 agents},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A biotech insect repellent, safe enough to eat.
<em>NBT</em>, <em>38</em>(12), 1368–1369. (<a
href="https://doi.org/10.1038/s41587-020-00760-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The fledgling metabolic engineering industry expands its products to pest control.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-020-00760-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1368-1369},
  shortjournal = {Nature Biotech.},
  title        = {A biotech insect repellent, safe enough to eat},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Long live california’s stem cells. <em>NBT</em>,
<em>38</em>(12), 1368. (<a
href="https://doi.org/10.1038/s41587-020-00772-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00772-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1368},
  shortjournal = {Nature Biotech.},
  title        = {Long live california’s stem cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Microbial cocktails raise bar for c. Diff. treatments.
<em>NBT</em>, <em>38</em>(12), 1366–1367. (<a
href="https://doi.org/10.1038/s41587-020-00765-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Seres’s success with an industrially made bacterial mix in phase 3 trials against Clostridium difficile infection promises an alternative to fecal microbial transplantation in clinical practice.},
  archive      = {J_NBT},
  author       = {Ratner, Mark},
  doi          = {10.1038/s41587-020-00765-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1366-1367},
  shortjournal = {Nature Biotech.},
  title        = {Microbial cocktails raise bar for c. diff. treatments},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). First systemic CRISPR agent in humans. <em>NBT</em>,
<em>38</em>(12), 1364. (<a
href="https://doi.org/10.1038/s41587-020-00773-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00773-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1364},
  shortjournal = {Nature Biotech.},
  title        = {First systemic CRISPR agent in humans},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Specter of eye toxicity looms over BCMA-targeted therapy.
<em>NBT</em>, <em>38</em>(12), 1363–1365. (<a
href="https://doi.org/10.1038/s41587-020-00757-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first antibody–drug conjugate to treat multiple myeloma can spur reversible loss of vision, but that may not be the case for other anti-BCMA T-cell therapies or bispecific antibodies.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-00757-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1363-1365},
  shortjournal = {Nature Biotech.},
  title        = {Specter of eye toxicity looms over BCMA-targeted therapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The nucleotides that bind. <em>NBT</em>, <em>38</em>(12),
1361. (<a href="https://doi.org/10.1038/s41587-020-00782-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid progress is being made in our understanding of RNA–protein interactions, their role in disease and their influence on the action of oligonucleotide drugs.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00782-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1361},
  shortjournal = {Nature Biotech.},
  title        = {The nucleotides that bind},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). First rounders: Tillman gerngross. <em>NBT</em>,
<em>38</em>(12), 1396. (<a
href="https://doi.org/10.1038/s41587-020-00754-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00754-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1396},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Tillman gerngross},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Send in the senolytics. <em>NBT</em>, <em>38</em>(12),
1371–1377. (<a
href="https://doi.org/10.1038/s41587-020-00750-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite early failures in the clinic, the idea of anti-aging therapies that purge the body of dying cells is gaining traction with a raft of startups now focused on senescence.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-00750-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1371-1377},
  shortjournal = {Nature Biotech.},
  title        = {Send in the senolytics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). People. <em>NBT</em>, <em>38</em>(11), 1360. (<a
href="https://doi.org/10.1038/s41587-020-0727-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0727-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1360},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Third-quarter biotech job picture. <em>NBT</em>,
<em>38</em>(11), 1359. (<a
href="https://doi.org/10.1038/s41587-020-0723-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-020-0723-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1359},
  shortjournal = {Nature Biotech.},
  title        = {Third-quarter biotech job picture},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Modeling neural tube development by
differentiation of human embryonic stem cells in a microfluidic WNT
gradient. <em>NBT</em>, <em>38</em>(11), 1357. (<a
href="https://doi.org/10.1038/s41587-020-0590-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Rifes, Pedro and Isaksson, Marc and Rathore, Gaurav Singh and Aldrin-Kirk, Patrick and Møller, Oliver Knights and Barzaghi, Guido and Lee, Julie and Egerod, Kristoffer Lihme and Rausch, Dylan M. and Parmar, Malin and Pers, Tune H. and Laurell, Thomas and Kirkeby, Agnete},
  doi          = {10.1038/s41587-020-0590-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1357},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Modeling neural tube development by differentiation of human embryonic stem cells in a microfluidic WNT gradient},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Author correction: A robust benchmark for detection of
germline large deletions and insertions. <em>NBT</em>, <em>38</em>(11),
1357. (<a href="https://doi.org/10.1038/s41587-020-0640-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Zook, Justin M. and Hansen, Nancy F. and Olson, Nathan D. and Chapman, Lesley and Mullikin, James C. and Xiao, Chunlin and Sherry, Stephen and Koren, Sergey and Phillippy, Adam M. and Boutros, Paul C. and Sahraeian, Sayed Mohammad E. and Huang, Vincent and Rouette, Alexandre and Alexander, Noah and Mason, Christopher E. and Hajirasouliha, Iman and Ricketts, Camir and Lee, Joyce and Tearle, Rick and Fiddes, Ian T. and Barrio, Alvaro Martinez and Wala, Jeremiah and Carroll, Andrew and Ghaffari, Noushin and Rodriguez, Oscar L. and Bashir, Ali and Jackman, Shaun and Farrell, John J. and Wenger, Aaron M. and Alkan, Can and Soylev, Arda and Schatz, Michael C. and Garg, Shilpa and Church, George and Marschall, Tobias and Chen, Ken and Fan, Xian and English, Adam C. and Rosenfeld, Jeffrey A. and Zhou, Weichen and Mills, Ryan E. and Sage, Jay M. and Davis, Jennifer R. and Kaiser, Michael D. and Oliver, John S. and Catalano, Anthony P. and Chaisson, Mark J. P. and Spies, Noah and Sedlazeck, Fritz J. and Salit, Marc},
  doi          = {10.1038/s41587-020-0640-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1357},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: A robust benchmark for detection of germline large deletions and insertions},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Publisher correction: Antibody engineers seek optimal drug
targeting TIGIT checkpoint. <em>NBT</em>, <em>38</em>(11), 1357. (<a
href="https://doi.org/10.1038/s41587-020-0714-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-0714-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1357},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Antibody engineers seek optimal drug targeting TIGIT checkpoint},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Multiplexed droplet single-cell
RNA-sequencing using natural genetic variation. <em>NBT</em>,
<em>38</em>(11), 1356. (<a
href="https://doi.org/10.1038/s41587-020-0715-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Kang, Hyun Min and Subramaniam, Meena and Targ, Sasha and Nguyen, Michelle and Maliskova, Lenka and McCarthy, Elizabeth and Wan, Eunice and Wong, Simon and Byrnes, Lauren and Lanata, Cristina M. and Gate, Rachel E. and Mostafavi, Sara and Marson, Alexander and Zaitlen, Noah and Criswell, Lindsey A. and Ye, Chun Jimmie},
  doi          = {10.1038/s41587-020-0715-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1356},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Multiplexed droplet single-cell RNA-sequencing using natural genetic variation},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A robust benchmark for detection of germline large
deletions and insertions. <em>NBT</em>, <em>38</em>(11), 1347–1355. (<a
href="https://doi.org/10.1038/s41587-020-0538-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New technologies and analysis methods are enabling genomic structural variants (SVs) to be detected with ever-increasing accuracy, resolution and comprehensiveness. To help translate these methods to routine research and clinical practice, we developed a sequence-resolved benchmark set for identification of both false-negative and false-positive germline large insertions and deletions. To create this benchmark for a broadly consented son in a Personal Genome Project trio with broadly available cells and DNA, the Genome in a Bottle Consortium integrated 19 sequence-resolved variant calling methods from diverse technologies. The final benchmark set contains 12,745 isolated, sequence-resolved insertion (7,281) and deletion (5,464) calls ≥50 base pairs (bp). The Tier 1 benchmark regions, for which any extra calls are putative false positives, cover 2.51 Gbp and 5,262 insertions and 4,095 deletions supported by ≥1 diploid assembly. We demonstrate that the benchmark set reliably identifies false negatives and false positives in high-quality SV callsets from short-, linked- and long-read sequencing and optical mapping. Detection of structural variants in the human genome is facilitated by a benchmark set of large deletions and insertions.},
  archive      = {J_NBT},
  author       = {Zook, Justin M. and Hansen, Nancy F. and Olson, Nathan D. and Chapman, Lesley and Mullikin, James C. and Xiao, Chunlin and Sherry, Stephen and Koren, Sergey and Phillippy, Adam M. and Boutros, Paul C. and Sahraeian, Sayed Mohammad E. and Huang, Vincent and Rouette, Alexandre and Alexander, Noah and Mason, Christopher E. and Hajirasouliha, Iman and Ricketts, Camir and Lee, Joyce and Tearle, Rick and Fiddes, Ian T. and Barrio, Alvaro Martinez and Wala, Jeremiah and Carroll, Andrew and Ghaffari, Noushin and Rodriguez, Oscar L. and Bashir, Ali and Jackman, Shaun and Farrell, John J. and Wenger, Aaron M. and Alkan, Can and Soylev, Arda and Schatz, Michael C. and Garg, Shilpa and Church, George and Marschall, Tobias and Chen, Ken and Fan, Xian and English, Adam C. and Rosenfeld, Jeffrey A. and Zhou, Weichen and Mills, Ryan E. and Sage, Jay M. and Davis, Jennifer R. and Kaiser, Michael D. and Oliver, John S. and Catalano, Anthony P. and Chaisson, Mark J. P. and Spies, Noah and Sedlazeck, Fritz J. and Salit, Marc},
  doi          = {10.1038/s41587-020-0538-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1347-1355},
  shortjournal = {Nature Biotech.},
  title        = {A robust benchmark for detection of germline large deletions and insertions},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Rapid image deconvolution and multiview fusion for optical
microscopy. <em>NBT</em>, <em>38</em>(11), 1337–1346. (<a
href="https://doi.org/10.1038/s41587-020-0560-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The contrast and resolution of images obtained with optical microscopes can be improved by deconvolution and computational fusion of multiple views of the same sample, but these methods are computationally expensive for large datasets. Here we describe theoretical and practical advances in algorithm and software design that result in image processing times that are tenfold to several thousand fold faster than with previous methods. First, we show that an ‘unmatched back projector’ accelerates deconvolution relative to the classic Richardson–Lucy algorithm by at least tenfold. Second, three-dimensional image-based registration with a graphics processing unit enhances processing speed 10- to 100-fold over CPU processing. Third, deep learning can provide further acceleration, particularly for deconvolution with spatially varying point spread functions. We illustrate our methods from the subcellular to millimeter spatial scale on diverse samples, including single cells, embryos and cleared tissue. Finally, we show performance enhancement on recently developed microscopes that have improved spatial resolution, including dual-view cleared-tissue light-sheet microscopes and reflective lattice light-sheet microscopes. Microscopy datasets are processed orders-of-magnitude faster with improved algorithms and deep learning.},
  archive      = {J_NBT},
  author       = {Guo, Min and Li, Yue and Su, Yijun and Lambert, Talley and Nogare, Damian Dalle and Moyle, Mark W. and Duncan, Leighton H. and Ikegami, Richard and Santella, Anthony and Rey-Suarez, Ivan and Green, Daniel and Beiriger, Anastasia and Chen, Jiji and Vishwasrao, Harshad and Ganesan, Sundar and Prince, Victoria and Waters, Jennifer C. and Annunziata, Christina M. and Hafner, Markus and Mohler, William A. and Chitnis, Ajay B. and Upadhyaya, Arpita and Usdin, Ted B. and Bao, Zhirong and Colón-Ramos, Daniel and La Riviere, Patrick and Liu, Huafeng and Wu, Yicong and Shroff, Hari},
  doi          = {10.1038/s41587-020-0560-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1337-1346},
  shortjournal = {Nature Biotech.},
  title        = {Rapid image deconvolution and multiview fusion for optical microscopy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Prediction of the sequence-specific cleavage activity of
cas9 variants. <em>NBT</em>, <em>38</em>(11), 1328–1336. (<a
href="https://doi.org/10.1038/s41587-020-0537-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several Streptococcus pyogenes Cas9 (SpCas9) variants have been developed to improve an enzyme’s specificity or to alter or broaden its protospacer-adjacent motif (PAM) compatibility, but selecting the optimal variant for a given target sequence and application remains difficult. To build computational models to predict the sequence-specific activity of 13 SpCas9 variants, we first assessed their cleavage efficiency at 26,891 target sequences. We found that, of the 256 possible four-nucleotide NNNN sequences, 156 can be used as a PAM by at least one of the SpCas9 variants. For the high-fidelity variants, overall activity could be ranked as SpCas9 ≥ Sniper-Cas9 &amp;gt; eSpCas9(1.1) &amp;gt; SpCas9-HF1 &amp;gt; HypaCas9 ≈ xCas9 &amp;gt;&amp;gt; evoCas9, whereas their overall specificities could be ranked as evoCas9 &amp;gt;&amp;gt; HypaCas9 ≥ SpCas9-HF1 ≈ eSpCas9(1.1) &amp;gt; xCas9 &amp;gt; Sniper-Cas9 &amp;gt; SpCas9. Using these data, we developed 16 deep-learning-based computational models that accurately predict the activity of these variants at any target sequence. Deep-learning models predict the Cas9 variant with optimal activity and specificity for any target sequence.},
  archive      = {J_NBT},
  author       = {Kim, Nahye and Kim, Hui Kwon and Lee, Sungtae and Seo, Jung Hwa and Choi, Jae Woo and Park, Jinman and Min, Seonwoo and Yoon, Sungroh and Cho, Sung-Rae and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-020-0537-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1328-1336},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of the sequence-specific cleavage activity of cas9 variants},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). CHANGE-seq reveals genetic and epigenetic effects on
CRISPR–Cas9 genome-wide activity. <em>NBT</em>, <em>38</em>(11),
1317–1327. (<a href="https://doi.org/10.1038/s41587-020-0555-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods can illuminate the genome-wide activity of CRISPR–Cas9 nucleases, but are not easily scalable to the throughput needed to fully understand the principles that govern Cas9 specificity. Here we describe ‘circularization for high-throughput analysis of nuclease genome-wide effects by sequencing’ (CHANGE-seq), a scalable, automatable tagmentation-based method for measuring the genome-wide activity of Cas9 in vitro. We applied CHANGE-seq to 110 single guide RNA targets across 13 therapeutically relevant loci in human primary T cells and identified 201,934 off-target sites, enabling the training of a machine learning model to predict off-target activity. Comparing matched genome-wide off-target, chromatin modification and accessibility, and transcriptional data, we found that cellular off-target activity was two to four times more likely to occur near active promoters, enhancers and transcribed regions. Finally, CHANGE-seq analysis of six targets across eight individual genomes revealed that human single-nucleotide variation had significant effects on activity at ~15.2\% of off-target sites analyzed. CHANGE-seq is a simplified, sensitive and scalable approach to understanding the specificity of genome editors. The genome-wide activity of Cas9 is measured in unprecedented detail.},
  archive      = {J_NBT},
  author       = {Lazzarotto, Cicera R. and Malinin, Nikolay L. and Li, Yichao and Zhang, Ruochi and Yang, Yang and Lee, GaHyun and Cowley, Eleanor and He, Yanghua and Lan, Xin and Jividen, Kasey and Katta, Varun and Kolmakova, Natalia G. and Petersen, Christopher T. and Qi, Qian and Strelcov, Evgheni and Maragh, Samantha and Krenciute, Giedre and Ma, Jian and Cheng, Yong and Tsai, Shengdar Q.},
  doi          = {10.1038/s41587-020-0555-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1317-1327},
  shortjournal = {Nature Biotech.},
  title        = {CHANGE-seq reveals genetic and epigenetic effects on CRISPR–Cas9 genome-wide activity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Automated assembly of centromeres from ultra-long
error-prone reads. <em>NBT</em>, <em>38</em>(11), 1309–1316. (<a
href="https://doi.org/10.1038/s41587-020-0582-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Centromeric variation has been linked to cancer and infertility, but centromere sequences contain multiple tandem repeats and can only be assembled manually from long error-prone reads. Here we describe the centroFlye algorithm for centromere assembly using long error-prone reads, and apply it to assemble human centromeres on chromosomes 6 and X. Our analyses reveal putative breakpoints in the manual reconstruction of the human X centromere, demonstrate that human X chromosome is partitioned into repeat subfamilies and provide initial insights into centromere evolution. We anticipate that centroFlye could be applied to automatically close remaining multimegabase gaps in the reference human genome. CentroFlye resolves tandem repeats to assemble human centromeres from nanopore reads.},
  archive      = {J_NBT},
  author       = {Bzikadze, Andrey V. and Pevzner, Pavel A.},
  doi          = {10.1038/s41587-020-0582-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1309-1316},
  shortjournal = {Nature Biotech.},
  title        = {Automated assembly of centromeres from ultra-long error-prone reads},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Efficient gene editing of human long-term hematopoietic stem
cells validated by clonal tracking. <em>NBT</em>, <em>38</em>(11),
1298–1308. (<a href="https://doi.org/10.1038/s41587-020-0551-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted gene editing in hematopoietic stem cells (HSCs) is a promising treatment for several diseases. However, the limited efficiency of homology-directed repair (HDR) in HSCs and the unknown impact of the procedure on clonal composition and dynamics of transplantation have hampered clinical translation. Here, we apply a barcoding strategy to clonal tracking of edited cells (BAR-Seq) and show that editing activates p53, which substantially shrinks the HSC clonal repertoire in hematochimeric mice, although engrafted edited clones preserve multilineage and self-renewing capacity. Transient p53 inhibition restored polyclonal graft composition. We increased HDR efficiency by forcing cell-cycle progression and upregulating components of the HDR machinery through transient expression of the adenovirus 5 E4orf6/7 protein, which recruits the cell-cycle controller E2F on its target genes. Combined E4orf6/7 expression and p53 inhibition resulted in HDR editing efficiencies of up to 50\% in the long-term human graft, without perturbing repopulation and self-renewal of edited HSCs. This enhanced protocol should broaden applicability of HSC gene editing and pave its way to clinical translation. Transient p53 inhibition and induced cell-cycle progression increase clonal engraftment and homology-directed repair in hematopoietic stem cells.},
  archive      = {J_NBT},
  author       = {Ferrari, Samuele and Jacob, Aurelien and Beretta, Stefano and Unali, Giulia and Albano, Luisa and Vavassori, Valentina and Cittaro, Davide and Lazarevic, Dejan and Brombin, Chiara and Cugnata, Federica and Kajaste-Rudnitski, Anna and Merelli, Ivan and Genovese, Pietro and Naldini, Luigi},
  doi          = {10.1038/s41587-020-0551-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1298-1308},
  shortjournal = {Nature Biotech.},
  title        = {Efficient gene editing of human long-term hematopoietic stem cells validated by clonal tracking},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Directed remodeling of the mouse gut microbiome inhibits the
development of atherosclerosis. <em>NBT</em>, <em>38</em>(11),
1288–1297. (<a href="https://doi.org/10.1038/s41587-020-0549-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The gut microbiome is a malleable microbial community that can remodel in response to various factors, including diet, and contribute to the development of several chronic diseases, including atherosclerosis. We devised an in vitro screening protocol of the mouse gut microbiome to discover molecules that can selectively modify bacterial growth. This approach was used to identify cyclic d,l-α-peptides that remodeled the Western diet (WD) gut microbiome toward the low-fat-diet microbiome state. Daily oral administration of the peptides in WD-fed LDLr−/− mice reduced plasma total cholesterol levels and atherosclerotic plaques. Depletion of the microbiome with antibiotics abrogated these effects. Peptide treatment reprogrammed the microbiome transcriptome, suppressed the production of pro-inflammatory cytokines (including interleukin-6, tumor necrosis factor-α and interleukin-1β), rebalanced levels of short-chain fatty acids and bile acids, improved gut barrier integrity and increased intestinal T regulatory cells. Directed chemical manipulation provides an additional tool for deciphering the chemical biology of the gut microbiome and might advance microbiome-targeted therapeutics. Gut microbiome composition is remodeled using small peptides.},
  archive      = {J_NBT},
  author       = {Chen, Poshen B. and Black, Audrey S. and Sobel, Adam L. and Zhao, Yannan and Mukherjee, Purba and Molparia, Bhuvan and Moore, Nina E. and Aleman Muench, German R. and Wu, Jiejun and Chen, Weixuan and Pinto, Antonio F. M. and Maryanoff, Bruce E. and Saghatelian, Alan and Soroosh, Pejman and Torkamani, Ali and Leman, Luke J. and Ghadiri, M. Reza},
  doi          = {10.1038/s41587-020-0549-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1288-1297},
  shortjournal = {Nature Biotech.},
  title        = {Directed remodeling of the mouse gut microbiome inhibits the development of atherosclerosis},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A tissue-engineered uterus supports live births in rabbits.
<em>NBT</em>, <em>38</em>(11), 1280–1287. (<a
href="https://doi.org/10.1038/s41587-020-0547-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioengineered uterine tissue could provide a treatment option for women with uterine factor infertility. In large animal models, reconstruction of the uterus has been demonstrated only with xenogeneic tissue grafts. Here we use biodegradable polymer scaffolds seeded with autologous cells to restore uterine structure and function in rabbits. Rabbits underwent a subtotal uterine excision and were reconstructed with autologous cell-seeded constructs, with nonseeded scaffolds or by suturing. At 6 months postimplantation, only the cell-seeded engineered uteri developed native tissue-like structures, including organized luminal/glandular epithelium, stroma, vascularized mucosa and two-layered myometrium. Only rabbits with cell-seeded constructs had normal pregnancies (four in ten) in the reconstructed segment of the uterus and supported fetal development to term and live birth. With further development, this approach may provide a regenerative medicine solution to uterine factor infertility. The rabbit uterus is reconstructed using synthetic scaffolds seeded with cells.},
  archive      = {J_NBT},
  author       = {Magalhaes, Renata S. and Williams, J. Koudy and Yoo, Kyung W. and Yoo, James J. and Atala, Anthony},
  doi          = {10.1038/s41587-020-0547-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1280-1287},
  shortjournal = {Nature Biotech.},
  title        = {A tissue-engineered uterus supports live births in rabbits},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A GRF–GIF chimeric protein improves the regeneration
efficiency of transgenic plants. <em>NBT</em>, <em>38</em>(11),
1274–1279. (<a href="https://doi.org/10.1038/s41587-020-0703-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The potential of genome editing to improve the agronomic performance of crops is often limited by low plant regeneration efficiencies and few transformable genotypes. Here, we show that expression of a fusion protein combining wheat GROWTH-REGULATING FACTOR 4 (GRF4) and its cofactor GRF-INTERACTING FACTOR 1 (GIF1) substantially increases the efficiency and speed of regeneration in wheat, triticale and rice and increases the number of transformable wheat genotypes. GRF4–GIF1 transgenic plants were fertile and without obvious developmental defects. Moreover, GRF4–GIF1 induced efficient wheat regeneration in the absence of exogenous cytokinins, which facilitates selection of transgenic plants without selectable markers. We also combined GRF4–GIF1 with CRISPR–Cas9 genome editing and generated 30 edited wheat plants with disruptions in the gene Q (AP2L-A5). Finally, we show that a dicot GRF–GIF chimera improves regeneration efficiency in citrus, suggesting that this strategy can be applied to dicot crops. A method to increase plant regeneration efficiency extends gene editing to more species and genotypes.},
  archive      = {J_NBT},
  author       = {Debernardi, Juan M. and Tricoli, David M. and Ercoli, Maria F. and Hayta, Sadiye and Ronald, Pamela and Palatnik, Javier F. and Dubcovsky, Jorge},
  doi          = {10.1038/s41587-020-0703-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1274-1279},
  shortjournal = {Nature Biotech.},
  title        = {A GRF–GIF chimeric protein improves the regeneration efficiency of transgenic plants},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Modeling neural tube development by differentiation of
human embryonic stem cells in a microfluidic WNT gradient. <em>NBT</em>,
<em>38</em>(11), 1265–1273. (<a
href="https://doi.org/10.1038/s41587-020-0525-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study of brain development in humans is limited by the lack of tissue samples and suitable in vitro models. Here, we model early human neural tube development using human embryonic stem cells cultured in a microfluidic device. The approach, named microfluidic-controlled stem cell regionalization (MiSTR), exposes pluripotent stem cells to signaling gradients that mimic developmental patterning. Using a WNT-activating gradient, we generated a neural tissue exhibiting progressive caudalization from forebrain to midbrain to hindbrain, including formation of isthmic organizer characteristics. Single-cell transcriptomics revealed that rostro-caudal organization was already established at 24 h of differentiation, and that the first markers of a neural-specific transcription program emerged in the rostral cells at 48 h. The transcriptomic hallmarks of rostro-caudal organization recapitulated gene expression patterns of the early rostro-caudal neural plate in mouse embryos. Thus, MiSTR will facilitate research on the factors and processes underlying rostro-caudal neural tube patterning. Patterning of the human neural tube is modeled in a microfluidic device.},
  archive      = {J_NBT},
  author       = {Rifes, Pedro and Isaksson, Marc and Rathore, Gaurav Singh and Aldrin-Kirk, Patrick and Møller, Oliver Knights and Barzaghi, Guido and Lee, Julie and Egerod, Kristoffer Lihme and Rausch, Dylan M. and Parmar, Malin and Pers, Tune H. and Laurell, Thomas and Kirkeby, Agnete},
  doi          = {10.1038/s41587-020-0525-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1265-1273},
  shortjournal = {Nature Biotech.},
  title        = {Modeling neural tube development by differentiation of human embryonic stem cells in a microfluidic WNT gradient},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reply to: Lactate measurements in an integrated perfusion
machine for human livers. <em>NBT</em>, <em>38</em>(11), 1263–1264. (<a
href="https://doi.org/10.1038/s41587-020-0627-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Dutkowski, Philipp and Mueller, Matteo and Eshmuminov, Dilmurodjon and Bautista Borrego, Lucia and Becker, Dustin and Hefti, Max and Schuler, Martin J. and Rudolf von Rohr, Philipp and Clavien, Pierre-Alain},
  doi          = {10.1038/s41587-020-0627-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1263-1264},
  shortjournal = {Nature Biotech.},
  title        = {Reply to: Lactate measurements in an integrated perfusion machine for human livers},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Viability criteria assessment during liver machine
perfusion. <em>NBT</em>, <em>38</em>(11), 1260–1262. (<a
href="https://doi.org/10.1038/s41587-020-0720-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Brüggenwirth, Isabel M. A. and de Meijer, Vincent E. and Porte, Robert J. and Martins, Paulo N.},
  doi          = {10.1038/s41587-020-0720-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1260-1262},
  shortjournal = {Nature Biotech.},
  title        = {Viability criteria assessment during liver machine perfusion},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Lactate measurements in an integrated perfusion machine for
human livers. <em>NBT</em>, <em>38</em>(11), 1259. (<a
href="https://doi.org/10.1038/s41587-020-0626-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Stephenson, Barney T. F. and Afford, Simon C. and Mergental, Hynek and Mirza, Darius F.},
  doi          = {10.1038/s41587-020-0626-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1259},
  shortjournal = {Nature Biotech.},
  title        = {Lactate measurements in an integrated perfusion machine for human livers},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell analysis and microfluidics. <em>NBT</em>,
<em>38</em>(11), 1257. (<a
href="https://doi.org/10.1038/s41587-020-0729-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to microfluidic devices and methods for single-cell analysis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0729-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1257},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell analysis and microfluidics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A qualitative study of biosimilar manufacturer and regulator
perceptions on intellectual property and abbreviated approval pathways.
<em>NBT</em>, <em>38</em>(11), 1253–1256. (<a
href="https://doi.org/10.1038/s41587-020-0717-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Abbreviated regulatory approval pathways for biosimilars were created to accommodate intellectual property protection, foster competition and lower drug prices, but their success in achieving these goals has been mixed.},
  archive      = {J_NBT},
  author       = {Druedahl, Louise C. and Almarsdóttir, Anna Birna and Kälvemark Sporrong, Sofia and De Bruin, Marie Louise and Hoogland, Hans and Minssen, Timo and van de Weert, Marco and Kesselheim, Aaron S. and Sarpatwari, Ameet},
  doi          = {10.1038/s41587-020-0717-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1253-1256},
  shortjournal = {Nature Biotech.},
  title        = {A qualitative study of biosimilar manufacturer and regulator perceptions on intellectual property and abbreviated approval pathways},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). COVID-19 antibodies on trial. <em>NBT</em>,
<em>38</em>(11), 1242–1252. (<a
href="https://doi.org/10.1038/s41587-020-0732-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the first readouts of trials of antibodies against COVID-19 appearing and others coming thick and fast, Nature Biotechnology asked a group of experts to comment on the challenges and timelines for these products.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0732-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1242-1252},
  shortjournal = {Nature Biotech.},
  title        = {COVID-19 antibodies on trial},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A novel computational architecture for large-scale genomics.
<em>NBT</em>, <em>38</em>(11), 1239–1241. (<a
href="https://doi.org/10.1038/s41587-020-0699-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Becker, Matthias and Schultze, Hartmut and Bresniker, Kirk and Singhal, Sharad and Ulas, Thomas and Schultze, Joachim L.},
  doi          = {10.1038/s41587-020-0699-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1239-1241},
  shortjournal = {Nature Biotech.},
  title        = {A novel computational architecture for large-scale genomics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Animal-free alternatives and the antibody iceberg.
<em>NBT</em>, <em>38</em>(11), 1234–1239. (<a
href="https://doi.org/10.1038/s41587-020-0687-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Gray, Alison and Bradbury, Andrew R. M. and Knappik, Achim and Plückthun, Andreas and Borrebaeck, Carl A. K. and Dübel, Stefan},
  doi          = {10.1038/s41587-020-0687-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1234-1239},
  shortjournal = {Nature Biotech.},
  title        = {Animal-free alternatives and the antibody iceberg},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). 3Q20 — good times roll on. <em>NBT</em>, <em>38</em>(11),
1232–1233. (<a href="https://doi.org/10.1038/s41587-020-0722-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0722-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1232-1233},
  shortjournal = {Nature Biotech.},
  title        = {3Q20 — good times roll on},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Drug pipeline 3Q20. <em>NBT</em>, <em>38</em>(11),
1230–1231. (<a href="https://doi.org/10.1038/s41587-020-0721-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0721-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1230-1231},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 3Q20},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Around the world in a month. <em>NBT</em>, <em>38</em>(11),
1229. (<a href="https://doi.org/10.1038/s41587-020-0730-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0730-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1229},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Doudna’s scribe unveils. <em>NBT</em>, <em>38</em>(11),
1228. (<a href="https://doi.org/10.1038/s41587-020-0736-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0736-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1228},
  shortjournal = {Nature Biotech.},
  title        = {Doudna’s scribe unveils},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Illumina wants grail back. <em>NBT</em>, <em>38</em>(11),
1227. (<a href="https://doi.org/10.1038/s41587-020-0735-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0735-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1227},
  shortjournal = {Nature Biotech.},
  title        = {Illumina wants grail back},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Big pharma buys into exosomes for drug delivery.
<em>NBT</em>, <em>38</em>(11), 1226–1228. (<a
href="https://doi.org/10.1038/s41587-020-0725-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several pharmaceutical companies are placing big bets on exosomes and other extracellular vesicles as a means to deliver nucleic acid therapeutics.},
  archive      = {J_NBT},
  author       = {Zipkin, Mark},
  doi          = {10.1038/s41587-020-0725-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1226-1228},
  shortjournal = {Nature Biotech.},
  title        = {Big pharma buys into exosomes for drug delivery},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). First video game to treat disease gains FDA okay.
<em>NBT</em>, <em>38</em>(11), 1224–1225. (<a
href="https://doi.org/10.1038/s41587-020-0726-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prescription-only game is said to improve attention function in children with attention deficit hyperactivity disorder (ADHD).},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-020-0726-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1224-1225},
  shortjournal = {Nature Biotech.},
  title        = {First video game to treat disease gains FDA okay},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). 10x genomics sequences in situ. <em>NBT</em>,
<em>38</em>(11), 1222. (<a
href="https://doi.org/10.1038/s41587-020-0734-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0734-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1222},
  shortjournal = {Nature Biotech.},
  title        = {10x genomics sequences in situ},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). IRAK4 degrader to take on innate immunity. <em>NBT</em>,
<em>38</em>(11), 1221–1223. (<a
href="https://doi.org/10.1038/s41587-020-0724-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interest in anti-inflammatory IRAK4 signaling is ramping up, as small-molecule ‘degraders’ that bind both the kinase and E3 ubiquitin ligases prepare to take on traditional kinase inhibitors.},
  archive      = {J_NBT},
  author       = {Mullard, Asher},
  doi          = {10.1038/s41587-020-0724-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1221-1223},
  shortjournal = {Nature Biotech.},
  title        = {IRAK4 degrader to take on innate immunity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Preparing for the vaccine. <em>NBT</em>, <em>38</em>(11),
1219. (<a href="https://doi.org/10.1038/s41587-020-0743-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clear and consistent messaging on COVID-19 vaccine rollout will be key to building public trust and managing expectations.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0743-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1219},
  shortjournal = {Nature Biotech.},
  title        = {Preparing for the vaccine},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Forum: Bioengineered scaffolds repair the uterus in
rabbits. <em>NBT</em>, <em>38</em>(11), 1258. (<a
href="https://doi.org/10.1038/s41587-020-0618-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0618-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {11},
  pages        = {1258},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bioengineered scaffolds repair the uterus in rabbits},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). People. <em>NBT</em>, <em>38</em>(10), 1218. (<a
href="https://doi.org/10.1038/s41587-020-0698-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0698-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1218},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Advancing healthcare technology education and innovation in
academia. <em>NBT</em>, <em>38</em>(10), 1213–1217. (<a
href="https://doi.org/10.1038/s41587-020-0689-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A national network of medical technology incubators provides experiential training for the next generation of medical entrepreneurs and enables creation of innovative technologies for pressing clinical needs.},
  archive      = {J_NBT},
  author       = {Linderman, Stephen W. and Appukutty, Abhinav J. and Russo, Mario V. and Shah, Aadit P. and Javaherian, Kavon},
  doi          = {10.1038/s41587-020-0689-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1213-1217},
  shortjournal = {Nature Biotech.},
  title        = {Advancing healthcare technology education and innovation in academia},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: An engineered ScCas9 with broad PAM
range and high specificity and activity. <em>NBT</em>, <em>38</em>(10),
1212. (<a href="https://doi.org/10.1038/s41587-020-0604-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Chatterjee, Pranam and Jakimo, Noah and Lee, Jooyoung and Amrani, Nadia and Rodríguez, Tomás and Koseki, Sabrina R. T. and Tysinger, Emma and Qing, Rui and Hao, Shilei and Sontheimer, Erik J. and Jacobson, Joseph},
  doi          = {10.1038/s41587-020-0604-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1212},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: An engineered ScCas9 with broad PAM range and high specificity and activity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). Publisher correction: The making of the biohub.
<em>NBT</em>, <em>38</em>(10), 1212. (<a
href="https://doi.org/10.1038/s41587-020-0702-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0702-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1212},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: The making of the biohub},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Deciphering eukaryotic gene-regulatory
logic with 100 million random promoters. <em>NBT</em>, <em>38</em>(10),
1211. (<a href="https://doi.org/10.1038/s41587-020-0665-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {de Boer, Carl G. and Vaishnav, Eeshit Dhaval and Sadeh, Ronen and Abeyta, Esteban Luis and Friedman, Nir and Regev, Aviv},
  doi          = {10.1038/s41587-020-0665-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1211},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Deciphering eukaryotic gene-regulatory logic with 100 million random promoters},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Detection of SARS-CoV-2 in nasal swabs
using MALDI-MS. <em>NBT</em>, <em>38</em>(10), 1211. (<a
href="https://doi.org/10.1038/s41587-020-0701-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Nachtigall, Fabiane M. and Pereira, Alfredo and Trofymchuk, Oleksandra S. and Santos, Leonardo S.},
  doi          = {10.1038/s41587-020-0701-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1211},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Detection of SARS-CoV-2 in nasal swabs using MALDI-MS},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A genome resource for green millet setaria viridis enables
discovery of agronomically valuable loci. <em>NBT</em>, <em>38</em>(10),
1203–1210. (<a href="https://doi.org/10.1038/s41587-020-0681-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wild and weedy relatives of domesticated crops harbor genetic variants that can advance agricultural biotechnology. Here we provide a genome resource for the wild plant green millet (Setaria viridis), a model species for studies of C4 grasses, and use the resource to probe domestication genes in the close crop relative foxtail millet (Setaria italica). We produced a platinum-quality genome assembly of S. viridis and de novo assemblies for 598 wild accessions and exploited these assemblies to identify loci underlying three traits: response to climate, a ‘loss of shattering’ trait that permits mechanical harvest and leaf angle, a predictor of yield in many grass crops. With CRISPR–Cas9 genome editing, we validated Less Shattering1 (SvLes1) as a gene whose product controls seed shattering. In S. italica, this gene was rendered nonfunctional by a retrotransposon insertion in the domesticated loss-of-shattering allele SiLes1-TE (transposable element). This resource will enhance the utility of S. viridis for dissection of complex traits and biotechnological improvement of panicoid crops. Sequencing wild relatives of millet identifies genes that regulate yield and harvesting traits.},
  archive      = {J_NBT},
  author       = {Mamidi, Sujan and Healey, Adam and Huang, Pu and Grimwood, Jane and Jenkins, Jerry and Barry, Kerrie and Sreedasyam, Avinash and Shu, Shengqiang and Lovell, John T. and Feldman, Maximilian and Wu, Jinxia and Yu, Yunqing and Chen, Cindy and Johnson, Jenifer and Sakakibara, Hitoshi and Kiba, Takatoshi and Sakurai, Tetsuya and Tavares, Rachel and Nusinow, Dmitri A. and Baxter, Ivan and Schmutz, Jeremy and Brutnell, Thomas P. and Kellogg, Elizabeth A.},
  doi          = {10.1038/s41587-020-0681-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1203-1210},
  shortjournal = {Nature Biotech.},
  title        = {A genome resource for green millet setaria viridis enables discovery of agronomically valuable loci},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Analyzing the mycobacterium tuberculosis immune response by
t-cell receptor clustering with GLIPH2 and genome-wide antigen
screening. <em>NBT</em>, <em>38</em>(10), 1194–1202. (<a
href="https://doi.org/10.1038/s41587-020-0505-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CD4+ T cells are critical to fighting pathogens, but a comprehensive analysis of human T-cell specificities is hindered by the diversity of HLA alleles (&amp;gt;20,000) and the complexity of many pathogen genomes. We previously described GLIPH, an algorithm to cluster T-cell receptors (TCRs) that recognize the same epitope and to predict their HLA restriction, but this method loses efficiency and accuracy when &amp;gt;10,000 TCRs are analyzed. Here we describe an improved algorithm, GLIPH2, that can process millions of TCR sequences. We used GLIPH2 to analyze 19,044 unique TCRβ sequences from 58 individuals latently infected with Mycobacterium tuberculosis (Mtb) and to group them according to their specificity. To identify the epitopes targeted by clusters of Mtb-specific T cells, we carried out a screen of 3,724 distinct proteins covering 95\% of Mtb protein-coding genes using artificial antigen-presenting cells (aAPCs) and reporter T cells. We found that at least five PPE (Pro-Pro-Glu) proteins are targets for T-cell recognition in Mtb. The T-cell response to tuberculosis is examined by clustering T-cell receptor sequences to identify shared specificities, along with whole-genome antigen screening.},
  archive      = {J_NBT},
  author       = {Huang, Huang and Wang, Chunlin and Rubelt, Florian and Scriba, Thomas J. and Davis, Mark M.},
  doi          = {10.1038/s41587-020-0505-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1194-1202},
  shortjournal = {Nature Biotech.},
  title        = {Analyzing the mycobacterium tuberculosis immune response by T-cell receptor clustering with GLIPH2 and genome-wide antigen screening},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). GPSeq reveals the radial organization of chromatin in the
cell nucleus. <em>NBT</em>, <em>38</em>(10), 1184–1193. (<a
href="https://doi.org/10.1038/s41587-020-0519-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the exception of lamina-associated domains, the radial organization of chromatin in mammalian cells remains largely unexplored. Here we describe genomic loci positioning by sequencing (GPSeq), a genome-wide method for inferring distances to the nuclear lamina all along the nuclear radius. GPSeq relies on gradual restriction digestion of chromatin from the nuclear lamina toward the nucleus center, followed by sequencing of the generated cut sites. Using GPSeq, we mapped the radial organization of the human genome at 100-kb resolution, which revealed radial patterns of genomic and epigenomic features and gene expression, as well as A and B subcompartments. By combining radial information with chromosome contact frequencies measured by Hi-C, we substantially improved the accuracy of whole-genome structure modeling. Finally, we charted the radial topography of DNA double-strand breaks, germline variants and cancer mutations and found that they have distinctive radial arrangements in A and B subcompartments. We conclude that GPSeq can reveal fundamental aspects of genome architecture. The location of genetic and epigenetic elements in mammalian nuclei is measured by gradual DNA fragmentation.},
  archive      = {J_NBT},
  author       = {Girelli, Gabriele and Custodio, Joaquin and Kallas, Tomasz and Agostini, Federico and Wernersson, Erik and Spanjaard, Bastiaan and Mota, Ana and Kolbeinsdottir, Solrun and Gelali, Eleni and Crosetto, Nicola and Bienko, Magda},
  doi          = {10.1038/s41587-020-0519-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1184-1193},
  shortjournal = {Nature Biotech.},
  title        = {GPSeq reveals the radial organization of chromatin in the cell nucleus},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evaluation of SARS-CoV-2 serology assays reveals a range of
test performance. <em>NBT</em>, <em>38</em>(10), 1174–1183. (<a
href="https://doi.org/10.1038/s41587-020-0659-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Appropriate use and interpretation of serological tests for assessments of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) exposure, infection and potential immunity require accurate data on assay performance. We conducted a head-to-head evaluation of ten point-of-care-style lateral flow assays (LFAs) and two laboratory-based enzyme-linked immunosorbent assays to detect anti-SARS-CoV-2 IgM and IgG antibodies in 5-d time intervals from symptom onset and studied the specificity of each assay in pre-coronavirus disease 2019 specimens. The percent of seropositive individuals increased with time, peaking in the latest time interval tested (&amp;gt;20 d after symptom onset). Test specificity ranged from 84.3\% to 100.0\% and was predominantly affected by variability in IgM results. LFA specificity could be increased by considering weak bands as negative, but this decreased detection of antibodies (sensitivity) in a subset of SARS-CoV-2 real-time PCR-positive cases. Our results underline the importance of seropositivity threshold determination and reader training for reliable LFA deployment. Although there was no standout serological assay, four tests achieved more than 80\% positivity at later time points tested and more than 95\% specificity. Of 12 serology assays tested, four detect antibodies in more than 80\% of patients with COVID-19.},
  archive      = {J_NBT},
  author       = {Whitman, Jeffrey D. and Hiatt, Joseph and Mowery, Cody T. and Shy, Brian R. and Yu, Ruby and Yamamoto, Tori N. and Rathore, Ujjwal and Goldgof, Gregory M. and Whitty, Caroline and Woo, Jonathan M. and Gallman, Antonia E. and Miller, Tyler E. and Levine, Andrew G. and Nguyen, David N. and Bapat, Sagar P. and Balcerek, Joanna and Bylsma, Sophia A. and Lyons, Ana M. and Li, Stacy and Wong, Allison Wai-yi and Gillis-Buck, Eva Mae and Steinhart, Zachary B. and Lee, Youjin and Apathy, Ryan and Lipke, Mitchell J. and Smith, Jennifer Anne and Zheng, Tina and Boothby, Ian C. and Isaza, Erin and Chan, Jackie and Acenas, Dante D. and Lee, Jinwoo and Macrae, Trisha A. and Kyaw, Than S. and Wu, David and Ng, Dianna L. and Gu, Wei and York, Vanessa A. and Eskandarian, Haig Alexander and Callaway, Perri C. and Warrier, Lakshmi and Moreno, Mary E. and Levan, Justine and Torres, Leonel and Farrington, Lila A. and Loudermilk, Rita P. and Koshal, Kanishka and Zorn, Kelsey C. and Garcia-Beltran, Wilfredo F. and Yang, Diane and Astudillo, Michael G. and Bernstein, Bradley E. and Gelfand, Jeffrey A. and Ryan, Edward T. and Charles, Richelle C. and Iafrate, A. John and Lennerz, Jochen K. and Miller, Steve and Chiu, Charles Y. and Stramer, Susan L. and Wilson, Michael R. and Manglik, Aashish and Ye, Chun Jimmie and Krogan, Nevan J. and Anderson, Mark S. and Cyster, Jason G. and Ernst, Joel D. and Wu, Alan H. B. and Lynch, Kara L. and Bern, Caryn and Hsu, Patrick D. and Marson, Alexander},
  doi          = {10.1038/s41587-020-0659-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1174-1183},
  shortjournal = {Nature Biotech.},
  title        = {Evaluation of SARS-CoV-2 serology assays reveals a range of test performance},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Detection of SARS-CoV-2 in nasal swabs using MALDI-MS.
<em>NBT</em>, <em>38</em>(10), 1168–1173. (<a
href="https://doi.org/10.1038/s41587-020-0644-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detection of SARS-CoV-2 using RT–PCR and other advanced methods can achieve high accuracy. However, their application is limited in countries that lack sufficient resources to handle large-scale testing during the COVID-19 pandemic. Here, we describe a method to detect SARS-CoV-2 in nasal swabs using matrix-assisted laser desorption/ionization mass spectrometry (MALDI-MS) and machine learning analysis. This approach uses equipment and expertise commonly found in clinical laboratories in developing countries. We obtained mass spectra from a total of 362 samples (211 SARS-CoV-2-positive and 151 negative by RT–PCR) without prior sample preparation from three different laboratories. We tested two feature selection methods and six machine learning approaches to identify the top performing analysis approaches and determine the accuracy of SARS-CoV-2 detection. The support vector machine model provided the highest accuracy (93.9\%), with 7\% false positives and 5\% false negatives. Our results suggest that MALDI-MS and machine learning analysis can be used to reliably detect SARS-CoV-2 in nasal swab samples. SARS-CoV-2 is reliably detected in nasal swab samples using mass spectrometry and machine learning analysis.},
  archive      = {J_NBT},
  author       = {Nachtigall, Fabiane M. and Pereira, Alfredo and Trofymchuk, Oleksandra S. and Santos, Leonardo S.},
  doi          = {10.1038/s41587-020-0644-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1168-1173},
  shortjournal = {Nature Biotech.},
  title        = {Detection of SARS-CoV-2 in nasal swabs using MALDI-MS},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Measurement of SARS-CoV-2 RNA in wastewater tracks community
infection dynamics. <em>NBT</em>, <em>38</em>(10), 1164–1167. (<a
href="https://doi.org/10.1038/s41587-020-0684-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We measured severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) RNA concentrations in primary sewage sludge in the New Haven, Connecticut, USA, metropolitan area during the Coronavirus Disease 2019 (COVID-19) outbreak in Spring 2020. SARS-CoV-2 RNA was detected throughout the more than 10-week study and, when adjusted for time lags, tracked the rise and fall of cases seen in SARS-CoV-2 clinical test results and local COVID-19 hospital admissions. Relative to these indicators, SARS-CoV-2 RNA concentrations in sludge were 0–2 d ahead of SARS-CoV-2 positive test results by date of specimen collection, 0–2 d ahead of the percentage of positive tests by date of specimen collection, 1–4 d ahead of local hospital admissions and 6–8 d ahead of SARS-CoV-2 positive test results by reporting date. Our data show the utility of viral RNA monitoring in municipal wastewater for SARS-CoV-2 infection surveillance at a population-wide level. In communities facing a delay between specimen collection and the reporting of test results, immediate wastewater results can provide considerable advance notice of infection dynamics. Testing sewage for the novel coronavirus reveals epidemiological trends.},
  archive      = {J_NBT},
  author       = {Peccia, Jordan and Zulli, Alessandro and Brackney, Doug E. and Grubaugh, Nathan D. and Kaplan, Edward H. and Casanovas-Massana, Arnau and Ko, Albert I. and Malik, Amyn A. and Wang, Dennis and Wang, Mike and Warren, Joshua L. and Weinberger, Daniel M. and Arnold, Wyatt and Omer, Saad B.},
  doi          = {10.1038/s41587-020-0684-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1164-1167},
  shortjournal = {Nature Biotech.},
  title        = {Measurement of SARS-CoV-2 RNA in wastewater tracks community infection dynamics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A lipidome atlas in MS-DIAL 4. <em>NBT</em>,
<em>38</em>(10), 1159–1163. (<a
href="https://doi.org/10.1038/s41587-020-0531-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Mass Spectrometry-Data Independent Analysis software version 4 (MS-DIAL 4), a comprehensive lipidome atlas with retention time, collision cross-section and tandem mass spectrometry information. We formulated mass spectral fragmentations of lipids across 117 lipid subclasses and included ion mobility tandem mass spectrometry. Using human, murine, algal and plant biological samples, we annotated and semiquantified 8,051 lipids using MS-DIAL 4 with a 1–2\% estimated false discovery rate. MS-DIAL 4 helps standardize lipidomics data and discover lipid pathways. Mass spectral fragmentations of lipids across 117 lipid subclasses are presented in a lipidome atlas.},
  archive      = {J_NBT},
  author       = {Tsugawa, Hiroshi and Ikeda, Kazutaka and Takahashi, Mikiko and Satoh, Aya and Mori, Yoshifumi and Uchino, Haruki and Okahashi, Nobuyuki and Yamada, Yutaka and Tada, Ipputa and Bonini, Paolo and Higashi, Yasuhiro and Okazaki, Yozo and Zhou, Zhiwei and Zhu, Zheng-Jiang and Koelmel, Jeremy and Cajka, Tomas and Fiehn, Oliver and Saito, Kazuki and Arita, Masanori and Arita, Makoto},
  doi          = {10.1038/s41587-020-0531-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1159-1163},
  shortjournal = {Nature Biotech.},
  title        = {A lipidome atlas in MS-DIAL 4},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). An engineered ScCas9 with broad PAM range and high
specificity and activity. <em>NBT</em>, <em>38</em>(10), 1154–1158. (<a
href="https://doi.org/10.1038/s41587-020-0517-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR enzymes require a protospacer-adjacent motif (PAM) near the target cleavage site, constraining the sequences accessible for editing. In the present study, we combine protein motifs from several orthologs to engineer two variants of Streptococcus canis Cas9—Sc++ and a higher-fidelity mutant HiFi-Sc++—that have simultaneously broad 5′-NNG-3′ PAM compatibility, robust DNA-cleavage activity and minimal off-target activity. Sc++ and HiFi-Sc++ extend the use of CRISPR editing for diverse applications. A hybrid CRISPR–Cas9 variant is able to specifically and efficiently target a larger proportion of the genome.},
  archive      = {J_NBT},
  author       = {Chatterjee, Pranam and Jakimo, Noah and Lee, Jooyoung and Amrani, Nadia and Rodríguez, Tomás and Koseki, Sabrina R. T. and Tysinger, Emma and Qing, Rui and Hao, Shilei and Sontheimer, Erik J. and Jacobson, Joseph},
  doi          = {10.1038/s41587-020-0517-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1154-1158},
  shortjournal = {Nature Biotech.},
  title        = {An engineered ScCas9 with broad PAM range and high specificity and activity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). First rounders: Noubar afeyan. <em>NBT</em>,
<em>38</em>(10), 1153. (<a
href="https://doi.org/10.1038/s41587-020-0683-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0683-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1153},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Noubar afeyan},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Tracking COVID-19 with wastewater. <em>NBT</em>,
<em>38</em>(10), 1151–1153. (<a
href="https://doi.org/10.1038/s41587-020-0690-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wastewater testing captures the rise and fall of novel coronavirus cases in a mid-sized metropolitan region.},
  archive      = {J_NBT},
  author       = {Larsen, David A. and Wigginton, Krista R.},
  doi          = {10.1038/s41587-020-0690-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1151-1153},
  shortjournal = {Nature Biotech.},
  title        = {Tracking COVID-19 with wastewater},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Exosomes. <em>NBT</em>, <em>38</em>(10), 1150. (<a
href="https://doi.org/10.1038/s41587-020-0696-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to methods for the mass production of exosomes and engineering of exosomes for therapeutic use.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0696-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1150},
  shortjournal = {Nature Biotech.},
  title        = {Exosomes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Pledging intellectual property for COVID-19. <em>NBT</em>,
<em>38</em>(10), 1146–1149. (<a
href="https://doi.org/10.1038/s41587-020-0682-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Voluntary pledges to make intellectual property broadly available to address urgent public health crises can overcome administrative and legal hurdles faced by more elaborate legal arrangements such as patent pools and achieve greater acceptance than governmental compulsory licensing.},
  archive      = {J_NBT},
  author       = {Contreras, Jorge L. and Eisen, Michael and Ganz, Ariel and Lemley, Mark and Molloy, Jenny and Peters, Diane M. and Tietze, Frank},
  doi          = {10.1038/s41587-020-0682-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1146-1149},
  shortjournal = {Nature Biotech.},
  title        = {Pledging intellectual property for COVID-19},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020m). Whither COVID-19 vaccines? <em>NBT</em>, <em>38</em>(10),
1132–1145. (<a href="https://doi.org/10.1038/s41587-020-0697-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology convenes a group of experts to provide their insights into SARS-CoV-2 vaccines in development.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0697-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1132-1145},
  shortjournal = {Nature Biotech.},
  title        = {Whither COVID-19 vaccines?},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Artificial intelligence, drug repurposing and peer review.
<em>NBT</em>, <em>38</em>(10), 1127–1131. (<a
href="https://doi.org/10.1038/s41587-020-0686-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Can traditional computational analysis and machine learning help compensate for inadequate peer review of drug-repurposing papers in the context of an infodemic?},
  archive      = {J_NBT},
  author       = {Levin, Jeremy M. and Oprea, Tudor I. and Davidovich, Sagie and Clozel, Thomas and Overington, John P. and Vanhaelen, Quentin and Cantor, Charles R. and Bischof, Evelyne and Zhavoronkov, Alex},
  doi          = {10.1038/s41587-020-0686-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1127-1131},
  shortjournal = {Nature Biotech.},
  title        = {Artificial intelligence, drug repurposing and peer review},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Leveling up citizen science. <em>NBT</em>, <em>38</em>(10),
1124–1126. (<a href="https://doi.org/10.1038/s41587-020-0694-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Waldispühl, Jérôme and Szantner, Attila and Knight, Rob and Caisse, Sébastien and Pitchford, Randy},
  doi          = {10.1038/s41587-020-0694-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1124-1126},
  shortjournal = {Nature Biotech.},
  title        = {Leveling up citizen science},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). BIO’s plan to promote opportunity and access for the
underserved. <em>NBT</em>, <em>38</em>(10), 1123–1124. (<a
href="https://doi.org/10.1038/s41587-020-0688-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {McMurry-Heath, Michelle},
  doi          = {10.1038/s41587-020-0688-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1123-1124},
  shortjournal = {Nature Biotech.},
  title        = {BIO’s plan to promote opportunity and access for the underserved},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Top 20 translational researchers of 2019. <em>NBT</em>,
<em>38</em>(10), 1121–1122. (<a
href="https://doi.org/10.1038/s41587-020-0680-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-020-0680-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1121-1122},
  shortjournal = {Nature Biotech.},
  title        = {Top 20 translational researchers of 2019},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Ag giant enters vertical farming. <em>NBT</em>,
<em>38</em>(10), 1114. (<a
href="https://doi.org/10.1038/s41587-020-0708-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0708-8},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1114},
  shortjournal = {Nature Biotech.},
  title        = {Ag giant enters vertical farming},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Enzymatic DNA synthesis enters new phase. <em>NBT</em>,
<em>38</em>(10), 1113–1115. (<a
href="https://doi.org/10.1038/s41587-020-0695-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several startups are now pursuing the potential of enzymatic synthesis as a faster and more efficient route for synthesizing longer DNA sequences than is possible with traditional chemical means.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-020-0695-9},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1113-1115},
  shortjournal = {Nature Biotech.},
  title        = {Enzymatic DNA synthesis enters new phase},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Around the world in a month. <em>NBT</em>, <em>38</em>(10),
1112. (<a href="https://doi.org/10.1038/s41587-020-0691-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0691-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1112},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). First oral SMA drug. <em>NBT</em>, <em>38</em>(10), 1111.
(<a href="https://doi.org/10.1038/s41587-020-0706-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0706-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1111},
  shortjournal = {Nature Biotech.},
  title        = {First oral SMA drug},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Mist begins to clear for lung delivery of RNA. <em>NBT</em>,
<em>38</em>(10), 1110–1112. (<a
href="https://doi.org/10.1038/s41587-020-0692-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inhaled RNA is coming soon for countermeasures against respiratory pathogens, including COVID-19.},
  archive      = {J_NBT},
  author       = {Shaffer, Catherine},
  doi          = {10.1038/s41587-020-0692-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1110-1112},
  shortjournal = {Nature Biotech.},
  title        = {Mist begins to clear for lung delivery of RNA},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Swoop on orphan autoimmune player momenta. <em>NBT</em>,
<em>38</em>(10), 1109. (<a
href="https://doi.org/10.1038/s41587-020-0705-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0705-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1109},
  shortjournal = {Nature Biotech.},
  title        = {Swoop on orphan autoimmune player momenta},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A reprieve from hemophilia a, but for how long?
<em>NBT</em>, <em>38</em>(10), 1107–1109. (<a
href="https://doi.org/10.1038/s41587-020-0693-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The agency’s ‘no-go’ for BioMarin’s hemophilia A gene therapy came as a surprise, but reflects outstanding questions as to the durability of the treatment.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-020-0693-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1107-1109},
  shortjournal = {Nature Biotech.},
  title        = {A reprieve from hemophilia a, but for how long?},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Trumping science. <em>NBT</em>, <em>38</em>(10), 1105. (<a
href="https://doi.org/10.1038/s41587-020-0713-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scientific integrity of US government agencies is under attack from the Trump administration. Agency senior executives and industry leaders need to continue pushing back.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0713-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1105},
  shortjournal = {Nature Biotech.},
  title        = {Trumping science},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). The making of the biohub. <em>NBT</em>, <em>38</em>(10),
1116–1120. (<a href="https://doi.org/10.1038/s41587-020-0685-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Chan Zuckerberg Biohub is both an institute and an academic network devoted to accelerating research via cutting edge technologies and open science. Its focus on single-cell analysis and infectious disease has placed it front and center in the pandemic.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0685-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {10},
  pages        = {1116-1120},
  shortjournal = {Nature Biotech.},
  title        = {The making of the biohub},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). People. <em>NBT</em>, <em>38</em>(9), 1104. (<a
href="https://doi.org/10.1038/s41587-020-0668-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0668-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1104},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Opening doors for diverse talent in biotechnology with the
BIO i-corps experience. <em>NBT</em>, <em>38</em>(9), 1099–1102. (<a
href="https://doi.org/10.1038/s41587-020-0663-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new workshop brings together under-represented participants to gain experience working in diverse teams, identifying problems worth solving, and learning evidence-based entrepreneurship approaches.},
  archive      = {J_NBT},
  author       = {Maloy, Stanley and Pucher, Cathy and Sedam, Marc and Womack, Chad and Baxter, Susan},
  doi          = {10.1038/s41587-020-0663-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1099-1102},
  shortjournal = {Nature Biotech.},
  title        = {Opening doors for diverse talent in biotechnology with the BIO I-corps experience},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Author correction: A complete domain-to-species taxonomy
for bacteria and archaea. <em>NBT</em>, <em>38</em>(9), 1098. (<a
href="https://doi.org/10.1038/s41587-020-0539-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Parks, Donovan H. and Chuvochina, Maria and Chaumeil, Pierre-Alain and Rinke, Christian and Mussig, Aaron J. and Hugenholtz, Philip},
  doi          = {10.1038/s41587-020-0539-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1098},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: A complete domain-to-species taxonomy for bacteria and archaea},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Extending the small-molecule
similarity principle to all levels of biology with the chemical checker.
<em>NBT</em>, <em>38</em>(9), 1098. (<a
href="https://doi.org/10.1038/s41587-020-0564-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Duran-Frigola, Miquel and Pauls, Eduardo and Guitart-Pla, Oriol and Bertoni, Martino and Alcalde, Víctor and Amat, David and Juan-Blanco, Teresa and Aloy, Patrick},
  doi          = {10.1038/s41587-020-0564-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1098},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Extending the small-molecule similarity principle to all levels of biology with the chemical checker},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Histone-writer cancer drugs enter
center stage. <em>NBT</em>, <em>38</em>(9), 1098. (<a
href="https://doi.org/10.1038/s41587-020-0672-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-020-0672-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1098},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Histone-writer cancer drugs enter center stage},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Morphing electronics enable
neuromodulation in growing tissue. <em>NBT</em>, <em>38</em>(9), 1097.
(<a href="https://doi.org/10.1038/s41587-020-0533-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Liu, Yuxin and Li, Jinxing and Song, Shang and Kang, Jiheong and Tsao, Yuchi and Chen, Shucheng and Mottini, Vittorio and McConnell, Kelly and Xu, Wenhui and Zheng, Yu-Qing and Tok, Jeffrey B.-H. and George, Paul M. and Bao, Zhenan},
  doi          = {10.1038/s41587-020-0533-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1097},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Morphing electronics enable neuromodulation in growing tissue},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Author correction: A male-biased sex-distorter gene drive
for the human malaria vector anopheles gambiae. <em>NBT</em>,
<em>38</em>(9), 1097. (<a
href="https://doi.org/10.1038/s41587-020-0658-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Simoni, Alekos and Hammond, Andrew M. and Beaghton, Andrea K. and Galizi, Roberto and Taxiarchi, Chrysanthi and Kyrou, Kyros and Meacci, Dario and Gribble, Matthew and Morselli, Giulia and Burt, Austin and Nolan, Tony and Crisanti, Andrea},
  doi          = {10.1038/s41587-020-0658-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1097},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: A male-biased sex-distorter gene drive for the human malaria vector anopheles gambiae},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Extending the small-molecule similarity principle to all
levels of biology with the chemical checker. <em>NBT</em>,
<em>38</em>(9), 1087–1096. (<a
href="https://doi.org/10.1038/s41587-020-0502-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Small molecules are usually compared by their chemical structure, but there is no unified analytic framework for representing and comparing their biological activity. We present the Chemical Checker (CC), which provides processed, harmonized and integrated bioactivity data on ~800,000 small molecules. The CC divides data into five levels of increasing complexity, from the chemical properties of compounds to their clinical outcomes. In between, it includes targets, off-targets, networks and cell-level information, such as omics data, growth inhibition and morphology. Bioactivity data are expressed in a vector format, extending the concept of chemical similarity to similarity between bioactivity signatures. We show how CC signatures can aid drug discovery tasks, including target identification and library characterization. We also demonstrate the discovery of compounds that reverse and mimic biological signatures of disease models and genetic perturbations in cases that could not be addressed using chemical information alone. Overall, the CC signatures facilitate the conversion of bioactivity data to a format that is readily amenable to machine learning methods. The biological activities of &amp;gt;800,000 small molecules are represented within a uniform framework.},
  archive      = {J_NBT},
  author       = {Duran-Frigola, Miquel and Pauls, Eduardo and Guitart-Pla, Oriol and Bertoni, Martino and Alcalde, Víctor and Amat, David and Juan-Blanco, Teresa and Aloy, Patrick},
  doi          = {10.1038/s41587-020-0502-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1087-1096},
  shortjournal = {Nature Biotech.},
  title        = {Extending the small-molecule similarity principle to all levels of biology with the chemical checker},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A complete domain-to-species taxonomy for bacteria and
archaea. <em>NBT</em>, <em>38</em>(9), 1079–1086. (<a
href="https://doi.org/10.1038/s41587-020-0501-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Genome Taxonomy Database is a phylogenetically consistent, genome-based taxonomy that provides rank-normalized classifications for ~150,000 bacterial and archaeal genomes from domain to genus. However, almost 40\% of the genomes in the Genome Taxonomy Database lack a species name. We address this limitation by using commonly accepted average nucleotide identity criteria to set bounds on species and propose species clusters that encompass all publicly available bacterial and archaeal genomes. Unlike previous average nucleotide identity studies, we chose a single representative genome to serve as the effective nomenclatural ‘type’ defining each species. Of the 24,706 proposed species clusters, 8,792 are based on published names. We assigned placeholder names to the remaining 15,914 species clusters to provide names to the growing number of genomes from uncultivated species. This resource provides a complete domain-to-species taxonomic framework for bacterial and archaeal genomes, which will facilitate research on uncultivated species and improve communication of scientific results. A full species classification is built for all publicly available bacterial and archaeal genomes.},
  archive      = {J_NBT},
  author       = {Parks, Donovan H. and Chuvochina, Maria and Chaumeil, Pierre-Alain and Rinke, Christian and Mussig, Aaron J. and Hugenholtz, Philip},
  doi          = {10.1038/s41587-020-0501-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1079-1086},
  shortjournal = {Nature Biotech.},
  title        = {A complete domain-to-species taxonomy for bacteria and archaea},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A SARS-CoV-2 surrogate virus neutralization test based on
antibody-mediated blockage of ACE2–spike protein–protein interaction.
<em>NBT</em>, <em>38</em>(9), 1073–1078. (<a
href="https://doi.org/10.1038/s41587-020-0631-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A robust serological test to detect neutralizing antibodies to SARS-CoV-2 is urgently needed to determine not only the infection rate, herd immunity and predicted humoral protection, but also vaccine efficacy during clinical trials and after large-scale vaccination. The current gold standard is the conventional virus neutralization test requiring live pathogen and a biosafety level 3 laboratory. Here, we report a SARS-CoV-2 surrogate virus neutralization test that detects total immunodominant neutralizing antibodies targeting the viral spike (S) protein receptor-binding domain in an isotype- and species-independent manner. Our simple and rapid test is based on antibody-mediated blockage of the interaction between the angiotensin-converting enzyme 2 (ACE2) receptor protein and the receptor-binding domain. The test, which has been validated with two cohorts of patients with COVID-19 in two different countries, achieves 99.93\% specificity and 95–100\% sensitivity, and differentiates antibody responses to several human coronaviruses. The surrogate virus neutralization test does not require biosafety level 3 containment, making it broadly accessible to the wider community for both research and clinical applications. A blocking assay based on the recombinant receptor-binding domain of SARS-CoV-2 spike protein and human angiotensin-converting enzyme 2 receptor provides an alternative to conventional antibody neutralization assays requiring live virus.},
  archive      = {J_NBT},
  author       = {Tan, Chee Wah and Chia, Wan Ni and Qin, Xijian and Liu, Pei and Chen, Mark I.-C. and Tiu, Charles and Hu, Zhiliang and Chen, Vivian Chih-Wei and Young, Barnaby E. and Sia, Wan Rong and Tan, Yee-Joo and Foo, Randy and Yi, Yongxiang and Lye, David C. and Anderson, Danielle E. and Wang, Lin-Fa},
  doi          = {10.1038/s41587-020-0631-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1073-1078},
  shortjournal = {Nature Biotech.},
  title        = {A SARS-CoV-2 surrogate virus neutralization test based on antibody-mediated blockage of ACE2–spike protein–protein interaction},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Generation of pancreatic β cells from CD177+ anterior
definitive endoderm. <em>NBT</em>, <em>38</em>(9), 1061–1072. (<a
href="https://doi.org/10.1038/s41587-020-0492-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods for differentiating human pluripotent stem cells to pancreatic and liver lineages in vitro have been limited by the inability to identify and isolate distinct endodermal subpopulations specific to these two organs. Here we report that pancreatic and hepatic progenitors can be isolated using the surface markers CD177/NB1 glycoprotein and inducible T-cell costimulatory ligand CD275/ICOSL, respectively, from seemingly homogeneous definitive endoderm derived from human pluripotent stem cells. Anterior definitive endoderm (ADE) subpopulations identified by CD177 and CD275 show inverse activation of canonical and noncanonical WNT signaling. CD177+ ADE expresses and synthesizes the secreted WNT, NODAL and BMP antagonist CERBERUS1 and is specified toward the pancreatic fate. CD275+ ADE receives canonical Wnt signaling and is specified toward the liver fate. Isolated CD177+ ADE differentiates more homogeneously into pancreatic progenitors and into more functionally mature and glucose-responsive β-like cells in vitro compared with cells from unsorted differentiation cultures. Markers that distinguish pancreatic and hepatic progenitors improve stem-cell differentiation to pancreatic β cells.},
  archive      = {J_NBT},
  author       = {Mahaddalkar, Pallavi U. and Scheibner, Katharina and Pfluger, Sandra and Ansarullah and Sterr, Michael and Beckenbauer, Julia and Irmler, Martin and Beckers, Johannes and Knöbel, Sebastian and Lickert, Heiko},
  doi          = {10.1038/s41587-020-0492-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1061-1072},
  shortjournal = {Nature Biotech.},
  title        = {Generation of pancreatic β cells from CD177+ anterior definitive endoderm},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A male-biased sex-distorter gene drive for the human
malaria vector anopheles gambiae. <em>NBT</em>, <em>38</em>(9),
1054–1060. (<a href="https://doi.org/10.1038/s41587-020-0508-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Only female insects transmit diseases such as malaria, dengue and Zika; therefore, control methods that bias the sex ratio of insect offspring have long been sought. Genetic elements such as sex-chromosome drives can distort sex ratios to produce unisex populations that eventually collapse, but the underlying molecular mechanisms are unknown. We report a male-biased sex-distorter gene drive (SDGD) in the human malaria vector Anopheles gambiae. We induced super-Mendelian inheritance of the X-chromosome-shredding I-PpoI nuclease by coupling this to a CRISPR-based gene drive inserted into a conserved sequence of the doublesex (dsx) gene. In modeling of invasion dynamics, SDGD was predicted to have a quicker impact on female mosquito populations than previously developed gene drives targeting female fertility. The SDGD at the dsx locus led to a male-only population from a 2.5\% starting allelic frequency in 10–14 generations, with population collapse and no selection for resistance. Our results support the use of SDGD for malaria vector control. A sex-distorter gene drive causes population collapse in the malaria mosquito.},
  archive      = {J_NBT},
  author       = {Simoni, Alekos and Hammond, Andrew M. and Beaghton, Andrea K. and Galizi, Roberto and Taxiarchi, Chrysanthi and Kyrou, Kyros and Meacci, Dario and Gribble, Matthew and Morselli, Giulia and Burt, Austin and Nolan, Tony and Crisanti, Andrea},
  doi          = {10.1038/s41587-020-0508-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1054-1060},
  shortjournal = {Nature Biotech.},
  title        = {A male-biased sex-distorter gene drive for the human malaria vector anopheles gambiae},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Nanopore sequencing and the shasta toolkit enable efficient
de novo assembly of eleven human genomes. <em>NBT</em>, <em>38</em>(9),
1044–1053. (<a href="https://doi.org/10.1038/s41587-020-0503-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo assembly of a human genome using nanopore long-read sequences has been reported, but it used more than 150,000 CPU hours and weeks of wall-clock time. To enable rapid human genome assembly, we present Shasta, a de novo long-read assembler, and polishing algorithms named MarginPolish and HELEN. Using a single PromethION nanopore sequencer and our toolkit, we assembled 11 highly contiguous human genomes de novo in 9 d. We achieved roughly 63× coverage, 42-kb read N50 values and 6.5× coverage in reads &amp;gt;100 kb using three flow cells per sample. Shasta produced a complete haploid human genome assembly in under 6 h on a single commercial compute node. MarginPolish and HELEN polished haploid assemblies to more than 99.9\% identity (Phred quality score QV = 30) with nanopore reads alone. Addition of proximity-ligation sequencing enabled near chromosome-level scaffolds for all 11 genomes. We compare our assembly performance to existing methods for diploid, haploid and trio-binned human samples and report superior accuracy and speed. High contiguity human genomes can be assembled de novo in 6 h using nanopore long-read sequences and the Shasta toolkit.},
  archive      = {J_NBT},
  author       = {Shafin, Kishwar and Pesout, Trevor and Lorig-Roach, Ryan and Haukness, Marina and Olsen, Hugh E. and Bosworth, Colleen and Armstrong, Joel and Tigyi, Kristof and Maurer, Nicholas and Koren, Sergey and Sedlazeck, Fritz J. and Marschall, Tobias and Mayes, Simon and Costa, Vania and Zook, Justin M. and Liu, Kelvin J. and Kilburn, Duncan and Sorensen, Melanie and Munson, Katy M. and Vollger, Mitchell R. and Monlong, Jean and Garrison, Erik and Eichler, Evan E. and Salama, Sofie and Haussler, David and Green, Richard E. and Akeson, Mark and Phillippy, Adam and Miga, Karen H. and Carnevali, Paolo and Jain, Miten and Paten, Benedict},
  doi          = {10.1038/s41587-020-0503-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1044-1053},
  shortjournal = {Nature Biotech.},
  title        = {Nanopore sequencing and the shasta toolkit enable efficient de novo assembly of eleven human genomes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Sequence-specific prediction of the efficiencies of adenine
and cytosine base editors. <em>NBT</em>, <em>38</em>(9), 1037–1043. (<a
href="https://doi.org/10.1038/s41587-020-0573-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Base editors, including adenine base editors (ABEs)1 and cytosine base editors (CBEs)2,3, are widely used to induce point mutations. However, determining whether a specific nucleotide in its genomic context can be edited requires time-consuming experiments. Furthermore, when the editable window contains multiple target nucleotides, various genotypic products can be generated. To develop computational tools to predict base-editing efficiency and outcome product frequencies, we first evaluated the efficiencies of an ABE and a CBE and the outcome product frequencies at 13,504 and 14,157 target sequences, respectively, in human cells. We found that there were only modest asymmetric correlations between the activities of the base editors and Cas9 at the same targets. Using deep-learning-based computational modeling, we built tools to predict the efficiencies and outcome frequencies of ABE- and CBE-directed editing at any target sequence, with Pearson correlations ranging from 0.50 to 0.95. These tools and results will facilitate modeling and therapeutic correction of genetic diseases by base editing. The activity of adenine or cytosine base editors at specific target nucleotides is predicted computationally.},
  archive      = {J_NBT},
  author       = {Song, Myungjae and Kim, Hui Kwon and Lee, Sungtae and Kim, Younggwang and Seo, Sang-Yeon and Park, Jinman and Choi, Jae Woo and Jang, Hyewon and Shin, Jeong Hong and Min, Seonwoo and Quan, Zhejiu and Kim, Ji Hun and Kang, Hoon Chul and Yoon, Sungroh and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-020-0573-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1037-1043},
  shortjournal = {Nature Biotech.},
  title        = {Sequence-specific prediction of the efficiencies of adenine and cytosine base editors},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Morphing electronics enable neuromodulation in growing
tissue. <em>NBT</em>, <em>38</em>(9), 1031–1036. (<a
href="https://doi.org/10.1038/s41587-020-0495-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioelectronics for modulating the nervous system have shown promise in treating neurological diseases1–3. However, their fixed dimensions cannot accommodate rapid tissue growth4,5 and may impair development6. For infants, children and adolescents, once implanted devices are outgrown, additional surgeries are often needed for device replacement, leading to repeated interventions and complications6–8. Here, we address this limitation with morphing electronics, which adapt to in vivo nerve tissue growth with minimal mechanical constraint. We design and fabricate multilayered morphing electronics, consisting of viscoplastic electrodes and a strain sensor that eliminate the stress at the interface between the electronics and growing tissue. The ability of morphing electronics to self-heal during implantation surgery allows a reconfigurable and seamless neural interface. During the fastest growth period in rats, morphing electronics caused minimal damage to the rat nerve, which grows 2.4-fold in diameter, and allowed chronic electrical stimulation and monitoring for 2 months without disruption of functional behavior. Morphing electronics offers a path toward growth-adaptive pediatric electronic medicine. Viscoplastic electronic devices adapt as nerves enlarge in growing animals.},
  archive      = {J_NBT},
  author       = {Liu, Yuxin and Li, Jinxing and Song, Shang and Kang, Jiheong and Tsao, Yuchi and Chen, Shucheng and Mottini, Vittorio and McConnell, Kelly and Xu, Wenhui and Zheng, Yu-Qing and Tok, Jeffrey B.-H. and George, Paul M. and Bao, Zhenan},
  doi          = {10.1038/s41587-020-0495-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1031-1036},
  shortjournal = {Nature Biotech.},
  title        = {Morphing electronics enable neuromodulation in growing tissue},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Transcriptomics. <em>NBT</em>, <em>38</em>(9), 1030. (<a
href="https://doi.org/10.1038/s41587-020-0670-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to new biomarkers and methods of transcriptomic data analysis and integration.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0670-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1030},
  shortjournal = {Nature Biotech.},
  title        = {Transcriptomics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The FDA de novo medical device pathway, patents and
anticompetition. <em>NBT</em>, <em>38</em>(9), 1028–1029. (<a
href="https://doi.org/10.1038/s41587-020-0653-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The interaction between patents and FDA’s De Novo and 510(k) regulatory pathways has the potential to threaten follow-on innovation for medical devices.},
  archive      = {J_NBT},
  author       = {Sherkow, Jacob S. and Aboy, Mateo},
  doi          = {10.1038/s41587-020-0653-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1028-1029},
  shortjournal = {Nature Biotech.},
  title        = {The FDA de novo medical device pathway, patents and anticompetition},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Agtech infrastructure for pandemic preparedness.
<em>NBT</em>, <em>38</em>(9), 1025–1027. (<a
href="https://doi.org/10.1038/s41587-020-0654-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Webb, Steven R. and Twyman, Richard M. and Moloney, Maurice},
  doi          = {10.1038/s41587-020-0654-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1025-1027},
  shortjournal = {Nature Biotech.},
  title        = {Agtech infrastructure for pandemic preparedness},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The COVID-19 XPRIZE and the need for scalable, fast, and
widespread testing. <em>NBT</em>, <em>38</em>(9), 1021–1024. (<a
href="https://doi.org/10.1038/s41587-020-0655-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {MacKay, Matthew J. and Hooker, Anna C. and Afshinnekoo, Ebrahim and Salit, Marc and Kelly, Jason and Feldstein, Jonathan V. and Haft, Nick and Schenkel, Doug and Nambi, Subhalaxmi and Cai, Yizhi and Zhang, Feng and Church, George and Dai, Junbiao and Wang, Cliff L. and Levy, Shawn and Huber, Jeff and Ji, Hanlee P. and Kriegel, Alison and Wyllie, Anne L. and Mason, Christopher E.},
  doi          = {10.1038/s41587-020-0655-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1021-1024},
  shortjournal = {Nature Biotech.},
  title        = {The COVID-19 XPRIZE and the need for scalable, fast, and widespread testing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). California shows the way for biosecurity in commercial gene
synthesis. <em>NBT</em>, <em>38</em>(9), 1021. (<a
href="https://doi.org/10.1038/s41587-020-0667-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {West, Rachel and Gronvall, Gigi Kwik},
  doi          = {10.1038/s41587-020-0667-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1021},
  shortjournal = {Nature Biotech.},
  title        = {California shows the way for biosecurity in commercial gene synthesis},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Green light for deep brain stimulator incorporating
neurofeedback. <em>NBT</em>, <em>38</em>(9), 1014–1015. (<a
href="https://doi.org/10.1038/s41587-020-0664-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A deep brain stimulation device capable of fine-tuning output on the basis of patient neuronal activity has been approved for the first time for use in Parkinson’s and a range of other neurological disorders.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-020-0664-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1014-1015},
  shortjournal = {Nature Biotech.},
  title        = {Green light for deep brain stimulator incorporating neurofeedback},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Around the world in a month. <em>NBT</em>, <em>38</em>(9),
1013. (<a href="https://doi.org/10.1038/s41587-020-0669-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0669-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1013},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). FDA okays second CAR-t for kite. <em>NBT</em>,
<em>38</em>(9), 1012. (<a
href="https://doi.org/10.1038/s41587-020-0676-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0676-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1012},
  shortjournal = {Nature Biotech.},
  title        = {FDA okays second CAR-T for kite},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Massive data initiatives and AI provide testbed for
pandemic forecasting. <em>NBT</em>, <em>38</em>(9), 1010–1013. (<a
href="https://doi.org/10.1038/s41587-020-0671-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Initiatives to gather massive epidemiological datasets aim to cut through national COVID-19 stats in a bid to understand the new coronavirus and aid public health policymakers.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-020-0671-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1010-1013},
  shortjournal = {Nature Biotech.},
  title        = {Massive data initiatives and AI provide testbed for pandemic forecasting},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Roche buys into RET inhibitor showdown. <em>NBT</em>,
<em>38</em>(9), 1010. (<a
href="https://doi.org/10.1038/s41587-020-0675-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0675-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1010},
  shortjournal = {Nature Biotech.},
  title        = {Roche buys into RET inhibitor showdown},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Moderna loses key patent challenge. <em>NBT</em>,
<em>38</em>(9), 1009. (<a
href="https://doi.org/10.1038/s41587-020-0674-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0674-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1009},
  shortjournal = {Nature Biotech.},
  title        = {Moderna loses key patent challenge},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Antibody engineers seek optimal drug targeting TIGIT
checkpoint. <em>NBT</em>, <em>38</em>(9), 1007–1009. (<a
href="https://doi.org/10.1038/s41587-020-0666-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Four antibodies are already in trials to test efficacy against the emerging checkpoint molecule TIGIT, but debate swirls around the importance of Fc binding for drug action.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-0666-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1007-1009},
  shortjournal = {Nature Biotech.},
  title        = {Antibody engineers seek optimal drug targeting TIGIT checkpoint},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Thank you for sharing. <em>NBT</em>, <em>38</em>(9), 1005.
(<a href="https://doi.org/10.1038/s41587-020-0678-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The pandemic has highlighted long-standing, deep-rooted challenges to the sharing of biological samples. Greater attention is needed to mechanisms for incentivizing materials transfer.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0678-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1005},
  shortjournal = {Nature Biotech.},
  title        = {Thank you for sharing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A singular view of COVID-19. <em>NBT</em>, <em>38</em>(9),
1016–1020. (<a href="https://doi.org/10.1038/s41587-020-0657-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell analysis sheds light on immune response to COVID-19 infection, enables the rapid discovery of antibody leads, and points to ways to get ahead of future pandemics.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-020-0657-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {9},
  pages        = {1016-1020},
  shortjournal = {Nature Biotech.},
  title        = {A singular view of COVID-19},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). People. <em>NBT</em>, <em>38</em>(8), 1004. (<a
href="https://doi.org/10.1038/s41587-020-0633-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0633-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1004},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Second-quarter biotech job picture. <em>NBT</em>,
<em>38</em>(8), 1003. (<a
href="https://doi.org/10.1038/s41587-020-0622-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-020-0622-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1003},
  shortjournal = {Nature Biotech.},
  title        = {Second-quarter biotech job picture},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Genetic circuit design automation for
the gut resident species bacteroides thetaiotaomicron. <em>NBT</em>,
<em>38</em>(8), 1001. (<a
href="https://doi.org/10.1038/s41587-020-0545-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Taketani, Mao and Zhang, Jianbo and Zhang, Shuyi and Triassi, Alexander J. and Huang, Yu-Ja and Griffith, Linda G. and Voigt, Christopher A.},
  doi          = {10.1038/s41587-020-0545-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1001},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Genetic circuit design automation for the gut resident species bacteroides thetaiotaomicron},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Plants with genetically encoded
autoluminescence. <em>NBT</em>, <em>38</em>(8), 1001. (<a
href="https://doi.org/10.1038/s41587-020-0578-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Mitiouchkina, Tatiana and Mishin, Alexander S. and Somermeyer, Louisa Gonzalez and Markina, Nadezhda M. and Chepurnyh, Tatiana V. and Guglya, Elena B. and Karataeva, Tatiana A. and Palkina, Kseniia A. and Shakhova, Ekaterina S. and Fakhranurova, Liliia I. and Chekova, Sofia V. and Tsarkova, Aleksandra S. and Golubev, Yaroslav V. and Negrebetsky, Vadim V. and Dolgushin, Sergey A. and Shalaev, Pavel V. and Shlykov, Dmitry and Melnik, Olesya A. and Shipunova, Victoria O. and Deyev, Sergey M. and Bubyrev, Andrey I. and Pushin, Alexander S. and Choob, Vladimir V. and Dolgov, Sergey V. and Kondrashov, Fyodor A. and Yampolsky, Ilia V. and Sarkisyan, Karen S.},
  doi          = {10.1038/s41587-020-0578-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1001},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Plants with genetically encoded autoluminescence},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Publisher correction: The hothouse for protein design.
<em>NBT</em>, <em>38</em>(8), 1001. (<a
href="https://doi.org/10.1038/s41587-020-0641-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-020-0641-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1001},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: The hothouse for protein design},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Scalable and robust SARS-CoV-2 testing
in an academic center. <em>NBT</em>, <em>38</em>(8), 1000. (<a
href="https://doi.org/10.1038/s41587-020-0623-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Aitken, Jim and Ambrose, Karen and Barrell, Sam and Beale, Rupert and Bineva-Todd, Ganka and Biswas, Dhruva and Byrne, Richard and Caidan, Simon and Cherepanov, Peter and Churchward, Laura and Clark, Graham and Crawford, Margaret and Cubitt, Laura and Dearing, Vicky and Earl, Christopher and Edwards, Amelia and Ekin, Chris and Fidanis, Efthymios and Gaiba, Alessandra and Gamblin, Steve and Gandhi, Sonia and Goldman, Jacki and Goldstone, Robert and Grant, Paul R. and Greco, Maria and Heaney, Judith and Hindmarsh, Steve and Houlihan, Catherine F. and Howell, Michael and Hubank, Michael and Hughes, Deborah and Instrell, Rachael and Jackson, Deborah and Jamal-Hanjani, Mariam and Jiang, Ming and Johnson, Mark and Jones, Leigh and Kanu, Nnennaya and Kassiotis, George and Kirk, Stuart and Kjaer, Svend and Levett, Andrew and Levett, Lisa and Levi, Marcel and Lu, Wei-Ting and MacRae, James I. and Matthews, John and McCoy, Laura E. and Moore, Catherine and Moore, David and Nastouli, Eleni and Nicod, Jerome and Nightingale, Luke and Olsen, Jessica and O’Reilly, Nicola and Pabari, Amar and Papayannopoulos, Venizelos and Patel, Namita and Peat, Nigel and Pollitt, Marc and Ratcliffe, Peter and Reis e Sousa, Caetano and Rosa, Annachiara and Rosenthal, Rachel and Roustan, Chloe and Rowan, Andrew and Shin, Gee Yen and Snell, Daniel M. and Song, Ok-Ryul and Spyer, Moira J. and Strange, Amy and Swanton, Charles and Turner, James M. A. and Turner, Melanie and Wack, Andreas and Walker, Philip A. and Ward, Sophia and Wong, Wai Keong and Wright, Joshua and Wu, Mary},
  doi          = {10.1038/s41587-020-0623-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1000},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Scalable and robust SARS-CoV-2 testing in an academic center},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Efficient production of male
wolbachia-infected aedes aegypti mosquitoes enables large-scale
suppression of wild populations. <em>NBT</em>, <em>38</em>(8), 1000. (<a
href="https://doi.org/10.1038/s41587-020-0649-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Crawford, Jacob E. and Clarke, David W. and Criswell, Victor and Desnoyer, Mark and Cornel, Devon and Deegan, Brittany and Gong, Kyle and Hopkins, Kaycie C. and Howell, Paul and Hyde, Justin S. and Livni, Josh and Behling, Charlie and Benza, Renzo and Chen, Willa and Dobson, Karen L. and Eldershaw, Craig and Greeley, Daniel and Han, Yi and Hughes, Bridgette and Kakani, Evdoxia and Karbowski, Joe and Kitchell, Angus and Lee, Erika and Lin, Teresa and Liu, Jianyi and Lozano, Martin and MacDonald, Warren and Mains, James W. and Metlitz, Matty and Mitchell, Sara N. and Moore, David and Ohm, Johanna R. and Parkes, Kathleen and Porshnikoff, Alexandra and Robuck, Chris and Sheridan, Martin and Sobecki, Robert and Smith, Peter and Stevenson, Jessica and Sullivan, Jordan and Wasson, Brian and Weakley, Allison M. and Wilhelm, Mark and Won, Joshua and Yasunaga, Ari and Chan, William C. and Holeman, Jodi and Snoad, Nigel and Upson, Linus and Zha, Tiantian and Dobson, Stephen L. and Mulligan, F. Steven and Massaro, Peter and White, Bradley J.},
  doi          = {10.1038/s41587-020-0649-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1000},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Efficient production of male wolbachia-infected aedes aegypti mosquitoes enables large-scale suppression of wild populations},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Rapid discovery and evolution of orthogonal aminoacyl-tRNA
synthetase–tRNA pairs. <em>NBT</em>, <em>38</em>(8), 989–999. (<a
href="https://doi.org/10.1038/s41587-020-0479-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central challenge in expanding the genetic code of cells to incorporate noncanonical amino acids into proteins is the scalable discovery of aminoacyl-tRNA synthetase (aaRS)–tRNA pairs that are orthogonal in their aminoacylation specificity. Here we computationally identify candidate orthogonal tRNAs from millions of sequences and develop a rapid, scalable approach—named tRNA Extension (tREX)—to determine the in vivo aminoacylation status of tRNAs. Using tREX, we test 243 candidate tRNAs in Escherichia coli and identify 71 orthogonal tRNAs, covering 16 isoacceptor classes, and 23 functional orthogonal tRNA–cognate aaRS pairs. We discover five orthogonal pairs, including three highly active amber suppressors, and evolve new amino acid substrate specificities for two pairs. Finally, we use tREX to characterize a matrix of 64 orthogonal synthetase–orthogonal tRNA specificities. This work expands the number of orthogonal pairs available for genetic code expansion and provides a pipeline for the discovery of additional orthogonal pairs and a foundation for encoding the cellular synthesis of noncanonical biopolymers. Orthogonal aminoacyl-tRNA synthetase–tRNA pairs are discovered by analyzing millions of sequences.},
  archive      = {J_NBT},
  author       = {Cervettini, Daniele and Tang, Shan and Fried, Stephen D. and Willis, Julian C. W. and Funke, Louise F. H. and Colwell, Lucy J. and Chin, Jason W.},
  doi          = {10.1038/s41587-020-0479-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {989-999},
  shortjournal = {Nature Biotech.},
  title        = {Rapid discovery and evolution of orthogonal aminoacyl-tRNA synthetase–tRNA pairs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Sci-fate characterizes the dynamics of gene expression in
single cells. <em>NBT</em>, <em>38</em>(8), 980–988. (<a
href="https://doi.org/10.1038/s41587-020-0480-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression programs change over time, differentiation and development, and in response to stimuli. However, nearly all techniques for profiling gene expression in single cells do not directly capture transcriptional dynamics. In the present study, we present a method for combined single-cell combinatorial indexing and messenger RNA labeling (sci-fate), which uses combinatorial cell indexing and 4-thiouridine labeling of newly synthesized mRNA to concurrently profile the whole and newly synthesized transcriptome in each of many single cells. We used sci-fate to study the cortisol response in &amp;gt;6,000 single cultured cells. From these data, we quantified the dynamics of the cell cycle and glucocorticoid receptor activation, and explored their intersection. Finally, we developed software to infer and analyze cell-state transitions. We anticipate that sci-fate will be broadly applicable to quantitatively characterize transcriptional dynamics in diverse systems. Gene expression dynamics in single cells are tracked by labeling newly synthesized RNA in indexed cells.},
  archive      = {J_NBT},
  author       = {Cao, Junyue and Zhou, Wei and Steemers, Frank and Trapnell, Cole and Shendure, Jay},
  doi          = {10.1038/s41587-020-0480-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {980-988},
  shortjournal = {Nature Biotech.},
  title        = {Sci-fate characterizes the dynamics of gene expression in single cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). COVID-19 severity correlates with airway epithelium–immune
cell interactions identified by single-cell analysis. <em>NBT</em>,
<em>38</em>(8), 970–979. (<a
href="https://doi.org/10.1038/s41587-020-0602-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To investigate the immune response and mechanisms associated with severe coronavirus disease 2019 (COVID-19), we performed single-cell RNA sequencing on nasopharyngeal and bronchial samples from 19 clinically well-characterized patients with moderate or critical disease and from five healthy controls. We identified airway epithelial cell types and states vulnerable to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection. In patients with COVID-19, epithelial cells showed an average three-fold increase in expression of the SARS-CoV-2 entry receptor ACE2, which correlated with interferon signals by immune cells. Compared to moderate cases, critical cases exhibited stronger interactions between epithelial and immune cells, as indicated by ligand–receptor expression profiles, and activated immune cells, including inflammatory macrophages expressing CCL2, CCL3, CCL20, CXCL1, CXCL3, CXCL10, IL8, IL1B and TNF. The transcriptional differences in critical cases compared to moderate cases likely contribute to clinical observations of heightened inflammatory tissue damage, lung injury and respiratory failure. Our data suggest that pharmacologic inhibition of the CCR1 and/or CCR5 pathways might suppress immune hyperactivation in critical COVID-19. Single-cell analysis of COVID-19 patient samples identifies activated immune pathways that correlate with severe disease.},
  archive      = {J_NBT},
  author       = {Chua, Robert Lorenz and Lukassen, Soeren and Trump, Saskia and Hennig, Bianca P. and Wendisch, Daniel and Pott, Fabian and Debnath, Olivia and Thürmann, Loreen and Kurth, Florian and Völker, Maria Theresa and Kazmierski, Julia and Timmermann, Bernd and Twardziok, Sven and Schneider, Stefan and Machleidt, Felix and Müller-Redetzky, Holger and Maier, Melanie and Krannich, Alexander and Schmidt, Sein and Balzer, Felix and Liebig, Johannes and Loske, Jennifer and Suttorp, Norbert and Eils, Jürgen and Ishaque, Naveed and Liebert, Uwe Gerd and von Kalle, Christof and Hocke, Andreas and Witzenrath, Martin and Goffinet, Christine and Drosten, Christian and Laudi, Sven and Lehmann, Irina and Conrad, Christian and Sander, Leif-Erik and Eils, Roland},
  doi          = {10.1038/s41587-020-0602-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {970-979},
  shortjournal = {Nature Biotech.},
  title        = {COVID-19 severity correlates with airway epithelium–immune cell interactions identified by single-cell analysis},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Genetic circuit design automation for the gut resident
species bacteroides thetaiotaomicron. <em>NBT</em>, <em>38</em>(8),
962–969. (<a href="https://doi.org/10.1038/s41587-020-0468-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacteroides thetaiotaomicron is a human-associated bacterium that holds promise for delivery of therapies in the gut microbiome1. Therapeutic bacteria would benefit from the ability to turn on different programs of gene expression in response to conditions inside and outside of the gut; however, the availability of regulatory parts, and methods to combine them, have been limited in B. thetaiotaomicron2–5. We report implementation of Cello circuit design automation software6 for this species. First, we characterize a set of genome-integrated NOT/NOR gates based on single guide RNAs (CRISPR–dCas9) to inform a Bt user constraint file (UCF) for Cello. Then, logic circuits are designed to integrate sensors that respond to bile acid and anhydrotetracycline (aTc), including one created to distinguish between environments associated with bioproduction, the human gut, and after release. This circuit was found to be stable under laboratory conditions for at least 12 days and to function in bacteria associated with a primary colonic epithelial monolayer in an in vitro human gut model system. A platform for genetic circuit design in a human gut commensal bacterium brings microbiome therapies that respond to gastrointestinal signals a step closer.},
  archive      = {J_NBT},
  author       = {Taketani, Mao and Zhang, Jianbo and Zhang, Shuyi and Triassi, Alexander J. and Huang, Yu-Ja and Griffith, Linda G. and Voigt, Christopher A.},
  doi          = {10.1038/s41587-020-0468-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {962-969},
  shortjournal = {Nature Biotech.},
  title        = {Genetic circuit design automation for the gut resident species bacteroides thetaiotaomicron},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Combinatorial single-cell CRISPR screens by direct guide RNA
capture and targeted sequencing. <em>NBT</em>, <em>38</em>(8), 954–961.
(<a href="https://doi.org/10.1038/s41587-020-0470-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell CRISPR screens enable the exploration of mammalian gene function and genetic regulatory networks. However, use of this technology has been limited by reliance on indirect indexing of single-guide RNAs (sgRNAs). Here we present direct-capture Perturb-seq, a versatile screening approach in which expressed sgRNAs are sequenced alongside single-cell transcriptomes. Direct-capture Perturb-seq enables detection of multiple distinct sgRNA sequences from individual cells and thus allows pooled single-cell CRISPR screens to be easily paired with combinatorial perturbation libraries that contain dual-guide expression vectors. We demonstrate the utility of this approach for high-throughput investigations of genetic interactions and, leveraging this ability, dissect epistatic interactions between cholesterol biogenesis and DNA repair. Using direct capture Perturb-seq, we also show that targeting individual genes with multiple sgRNAs per cell improves efficacy of CRISPR interference and activation, facilitating the use of compact, highly active CRISPR libraries for single-cell screens. Last, we show that hybridization-based target enrichment permits sensitive, specific sequencing of informative transcripts from single-cell RNA-seq experiments. Single-cell CRISPR screens are readily multiplexed and scaled with an improved version of Perturb-seq.},
  archive      = {J_NBT},
  author       = {Replogle, Joseph M. and Norman, Thomas M. and Xu, Albert and Hussmann, Jeffrey A. and Chen, Jin and Cogan, J. Zachery and Meer, Elliott J. and Terry, Jessica M. and Riordan, Daniel P. and Srinivas, Niranjan and Fiddes, Ian T. and Arthur, Joseph G. and Alvarado, Luigi J. and Pfeiffer, Katherine A. and Mikkelsen, Tarjei S. and Weissman, Jonathan S. and Adamson, Britt},
  doi          = {10.1038/s41587-020-0470-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {954-961},
  shortjournal = {Nature Biotech.},
  title        = {Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Human chimeric antigen receptor macrophages for cancer
immunotherapy. <em>NBT</em>, <em>38</em>(8), 947–953. (<a
href="https://doi.org/10.1038/s41587-020-0462-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor (CAR) T cell therapy has shown promise in hematologic malignancies, but its application to solid tumors has been challenging1–4. Given the unique effector functions of macrophages and their capacity to penetrate tumors5, we genetically engineered human macrophages with CARs to direct their phagocytic activity against tumors. We found that a chimeric adenoviral vector overcame the inherent resistance of primary human macrophages to genetic manipulation and imparted a sustained pro-inflammatory (M1) phenotype. CAR macrophages (CAR-Ms) demonstrated antigen-specific phagocytosis and tumor clearance in vitro. In two solid tumor xenograft mouse models, a single infusion of human CAR-Ms decreased tumor burden and prolonged overall survival. Characterization of CAR-M activity showed that CAR-Ms expressed pro-inflammatory cytokines and chemokines, converted bystander M2 macrophages to M1, upregulated antigen presentation machinery, recruited and presented antigen to T cells and resisted the effects of immunosuppressive cytokines. In humanized mouse models, CAR-Ms were further shown to induce a pro-inflammatory tumor microenvironment and boost anti-tumor T cell activity. Primary macrophages engineered to express chimeric antigen receptors have anti-tumor activity in humanized mice.},
  archive      = {J_NBT},
  author       = {Klichinsky, Michael and Ruella, Marco and Shestova, Olga and Lu, Xueqing Maggie and Best, Andrew and Zeeman, Martha and Schmierer, Maggie and Gabrusiewicz, Konrad and Anderson, Nicholas R. and Petty, Nicholas E. and Cummins, Katherine D. and Shen, Feng and Shan, Xinhe and Veliz, Kimberly and Blouch, Kristin and Yashiro-Ohtani, Yumi and Kenderian, Saad S. and Kim, Miriam Y. and O’Connor, Roddy S. and Wallace, Stephen R. and Kozlowski, Miroslaw S. and Marchione, Dylan M. and Shestov, Maksim and Garcia, Benjamin A. and June, Carl H. and Gill, Saar},
  doi          = {10.1038/s41587-020-0462-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {947-953},
  shortjournal = {Nature Biotech.},
  title        = {Human chimeric antigen receptor macrophages for cancer immunotherapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Plants with genetically encoded autoluminescence.
<em>NBT</em>, <em>38</em>(8), 944–946. (<a
href="https://doi.org/10.1038/s41587-020-0500-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autoluminescent plants engineered to express a bacterial bioluminescence gene cluster in plastids have not been widely adopted because of low light output. We engineered tobacco plants with a fungal bioluminescence system that converts caffeic acid (present in all plants) into luciferin and report self-sustained luminescence that is visible to the naked eye. Our findings could underpin development of a suite of imaging tools for plants. Luminescence is engineered in whole plants, without an exogenous substrate, using a fungal gene cluster.},
  archive      = {J_NBT},
  author       = {Mitiouchkina, Tatiana and Mishin, Alexander S. and Somermeyer, Louisa Gonzalez and Markina, Nadezhda M. and Chepurnyh, Tatiana V. and Guglya, Elena B. and Karataeva, Tatiana A. and Palkina, Kseniia A. and Shakhova, Ekaterina S. and Fakhranurova, Liliia I. and Chekova, Sofia V. and Tsarkova, Aleksandra S. and Golubev, Yaroslav V. and Negrebetsky, Vadim V. and Dolgushin, Sergey A. and Shalaev, Pavel V. and Shlykov, Dmitry and Melnik, Olesya A. and Shipunova, Victoria O. and Deyev, Sergey M. and Bubyrev, Andrey I. and Pushin, Alexander S. and Choob, Vladimir V. and Dolgov, Sergey V. and Kondrashov, Fyodor A. and Yampolsky, Ilia V. and Sarkisyan, Karen S.},
  doi          = {10.1038/s41587-020-0500-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {944-946},
  shortjournal = {Nature Biotech.},
  title        = {Plants with genetically encoded autoluminescence},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Biomedical imaging. <em>NBT</em>, <em>38</em>(8), 942. (<a
href="https://doi.org/10.1038/s41587-020-0635-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to systems and methods of imaging.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0635-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {942},
  shortjournal = {Nature Biotech.},
  title        = {Biomedical imaging},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Federal judge invalidates icosapent ethyl patents — but on
the basis of a common statistical mistake. <em>NBT</em>, <em>38</em>(8),
939–941. (<a href="https://doi.org/10.1038/s41587-020-0616-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {If the decision is allowed to stand, it will have major implications for pharmaceutical patents and biomedical innovation.},
  archive      = {J_NBT},
  author       = {Curfman, Gregory and Bhatt, Deepak L. and Pencina, Michael},
  doi          = {10.1038/s41587-020-0616-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {939-941},
  shortjournal = {Nature Biotech.},
  title        = {Federal judge invalidates icosapent ethyl patents — but on the basis of a common statistical mistake},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Early experience with the FDA’s breakthrough devices
program. <em>NBT</em>, <em>38</em>(8), 933–938. (<a
href="https://doi.org/10.1038/s41587-020-0636-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Johnston, James L. and Dhruva, Sanket S. and Ross, Joseph S. and Rathi, Vinay K.},
  doi          = {10.1038/s41587-020-0636-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {933-938},
  shortjournal = {Nature Biotech.},
  title        = {Early experience with the FDA’s breakthrough devices program},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Pandemic preparedness in taiwan. <em>NBT</em>,
<em>38</em>(8), 932–933. (<a
href="https://doi.org/10.1038/s41587-020-0630-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Wu, Wei-Kai and Liou, Jyh-Ming and Hsu, Cheng-Chih and Lin, Yung-Hsiang and Wu, Ming-Shiang},
  doi          = {10.1038/s41587-020-0630-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {932-933},
  shortjournal = {Nature Biotech.},
  title        = {Pandemic preparedness in taiwan},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Scalable and robust SARS-CoV-2 testing in an academic
center. <em>NBT</em>, <em>38</em>(8), 927–931. (<a
href="https://doi.org/10.1038/s41587-020-0588-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Aitken, Jim and Ambrose, Karen and Barrell, Sam and Beale, Rupert and Bineva-Todd, Ganka and Biswas, Dhruva and Byrne, Richard and Caidan, Simon and Cherepanov, Peter and Churchward, Laura and Clark, Graham and Crawford, Margaret and Cubitt, Laura and Dearing, Vicky and Earl, Christopher and Edwards, Amelia and Ekin, Chris and Fidanis, Efthymios and Gaiba, Alessandra and Gamblin, Steve and Gandhi, Sonia and Goldman, Jacki and Goldstone, Robert and Grant, Paul R. and Greco, Maria and Heaney, Judith and Hindmarsh, Steve and Houlihan, Catherine F. and Howell, Michael and Hubank, Michael and Hughes, Deborah and Instrell, Rachael and Jackson, Deborah and Jamal-Hanjani, Mariam and Jiang, Ming and Johnson, Mark and Jones, Leigh and Kanu, Nnennaya and Kassiotis, George and Kirk, Stuart and Kjaer, Svend and Levett, Andrew and Levett, Lisa and Levi, Marcel and Lu, Wei-Ting and MacRae, James I. and Matthews, John and McCoy, Laura E. and Moore, Catherine and Moore, David and Nastouli, Eleni and Nicod, Jerome and Nightingale, Luke and Olsen, Jessica and O’Reilly, Nicola and Pabari, Amar and Papayannopoulos, Venizelos and Patel, Namita and Peat, Nigel and Pollitt, Marc and Ratcliffe, Peter and Reis e Sousa, Caetano and Rosa, Annachiara and Rosenthal, Rachel and Roustan, Chloe and Rowan, Andrew and Shin, Gee Yen and Snell, Daniel M. and Song, Ok-Ryul and Spyer, Moira J. and Strange, Amy and Swanton, Charles and Turner, James M. A. and Turner, Melanie and Wack, Andreas and Walker, Philip A. and Ward, Sophia and Wong, Wai Keong and Wright, Joshua and Wu, Mary},
  doi          = {10.1038/s41587-020-0588-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {927-931},
  shortjournal = {Nature Biotech.},
  title        = {Scalable and robust SARS-CoV-2 testing in an academic center},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Investing in racial diversity: A call to action to the
venture capital community. <em>NBT</em>, <em>38</em>(8), 925–927. (<a
href="https://doi.org/10.1038/s41587-020-0624-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Grant, Jackie},
  doi          = {10.1038/s41587-020-0624-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {925-927},
  shortjournal = {Nature Biotech.},
  title        = {Investing in racial diversity: A call to action to the venture capital community},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Surfacing anti-black science and building antiracist teams.
<em>NBT</em>, <em>38</em>(8), 923–925. (<a
href="https://doi.org/10.1038/s41587-020-0637-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Caicedo, H. Hugo},
  doi          = {10.1038/s41587-020-0637-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {923-925},
  shortjournal = {Nature Biotech.},
  title        = {Surfacing anti-black science and building antiracist teams},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Biotech patenting 2019. <em>NBT</em>, <em>38</em>(8),
921–922. (<a href="https://doi.org/10.1038/s41587-020-0615-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-020-0615-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {921-922},
  shortjournal = {Nature Biotech.},
  title        = {Biotech patenting 2019},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). 2Q20 — put on a happy face. <em>NBT</em>, <em>38</em>(8),
915–916. (<a href="https://doi.org/10.1038/s41587-020-0629-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0629-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {915-916},
  shortjournal = {Nature Biotech.},
  title        = {2Q20 — put on a happy face},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Drug pipeline 2Q20. <em>NBT</em>, <em>38</em>(8), 913–914.
(<a href="https://doi.org/10.1038/s41587-020-0625-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0625-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {913-914},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 2Q20},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Around the world in a month. <em>NBT</em>, <em>38</em>(8),
912. (<a href="https://doi.org/10.1038/s41587-020-0632-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0632-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {912},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Intercept’s NASH hopes dashed. <em>NBT</em>, <em>38</em>(8),
911. (<a href="https://doi.org/10.1038/s41587-020-0638-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0638-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {911},
  shortjournal = {Nature Biotech.},
  title        = {Intercept’s NASH hopes dashed},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). High-dose AAV gene therapy deaths. <em>NBT</em>,
<em>38</em>(8), 910. (<a
href="https://doi.org/10.1038/s41587-020-0642-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0642-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {910},
  shortjournal = {Nature Biotech.},
  title        = {High-dose AAV gene therapy deaths},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Histone-writer cancer drugs enter center stage.
<em>NBT</em>, <em>38</em>(8), 909–912. (<a
href="https://doi.org/10.1038/s41587-020-0621-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epizyme’s EZH2 inhibitor gains two approvals in quick succession — a milestone for this new class of epigenetic cancer drug that targets histone marks with greater precision than previous generations.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-020-0621-1},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {909-912},
  shortjournal = {Nature Biotech.},
  title        = {Histone-writer cancer drugs enter center stage},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). <em>NBT</em>, <em>38</em>(8), 909. (<a
href="https://doi.org/10.1038/s41587-020-0639-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0639-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {909},
  shortjournal = {Nature Biotech.},
  title        = {Antibacterials get $1 billion injection},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). COVID vaccine results pour in. <em>NBT</em>, <em>38</em>(8),
908. (<a href="https://doi.org/10.1038/s41587-020-0647-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0647-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {908},
  shortjournal = {Nature Biotech.},
  title        = {COVID vaccine results pour in},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Focus shifts to antibody cocktails for COVID-19 cytokine
storm. <em>NBT</em>, <em>38</em>(8), 905–908. (<a
href="https://doi.org/10.1038/s41587-020-0634-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emerging clinical trial data suggest that individual immunomodulatory drugs can dampen the hyperactive immune system in severe COVID-19, but polytherapy is the way forward.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-020-0634-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {905-908},
  shortjournal = {Nature Biotech.},
  title        = {Focus shifts to antibody cocktails for COVID-19 cytokine storm},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Weighing and communicating evidence. <em>NBT</em>,
<em>38</em>(8), 903. (<a
href="https://doi.org/10.1038/s41587-020-0650-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As preliminary research findings are shared more widely and at an increasing pace, action is needed to counter the spread of misinformation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0650-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {903},
  shortjournal = {Nature Biotech.},
  title        = {Weighing and communicating evidence},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). First rounders: Ted love. <em>NBT</em>, <em>38</em>(8),
943. (<a href="https://doi.org/10.1038/s41587-020-0617-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0617-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {943},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Ted love},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Forum: CRISPR roundtable with doudna and liu. <em>NBT</em>,
<em>38</em>(8), 943. (<a
href="https://doi.org/10.1038/s41587-020-0619-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0619-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {943},
  shortjournal = {Nature Biotech.},
  title        = {Forum: CRISPR roundtable with doudna and liu},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Watcher in the wastewater. <em>NBT</em>, <em>38</em>(8),
917–920. (<a href="https://doi.org/10.1038/s41587-020-0620-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research groups around the globe are looking to see whether urban wastewater monitoring can be integrated into surveillance systems for SARS-CoV-2 and other pathogens.},
  archive      = {J_NBT},
  author       = {Schmidt, Charles},
  doi          = {10.1038/s41587-020-0620-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {917-920},
  shortjournal = {Nature Biotech.},
  title        = {Watcher in the wastewater},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). People. <em>NBT</em>, <em>38</em>(7), 902. (<a
href="https://doi.org/10.1038/s41587-020-0594-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0594-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {902},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Phage-assisted evolution of an adenine
base editor with improved cas domain compatibility and activity.
<em>NBT</em>, <em>38</em>(7), 901. (<a
href="https://doi.org/10.1038/s41587-020-0562-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Richter, Michelle F. and Zhao, Kevin T. and Eton, Elliot and Lapinaite, Audrone and Newby, Gregory A. and Thuronyi, B W. and Wilson, Christopher and Koblan, Luke W. and Zeng, Jing and Bauer, Daniel E. and Doudna, Jennifer A. and Liu, David R.},
  doi          = {10.1038/s41587-020-0562-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {901},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Phage-assisted evolution of an adenine base editor with improved cas domain compatibility and activity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Base editors for simultaneous
introduction of c-to-t and a-to-g mutations. <em>NBT</em>,
<em>38</em>(7), 901. (<a
href="https://doi.org/10.1038/s41587-020-0585-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Sakata, Rina C. and Ishiguro, Soh and Mori, Hideto and Tanaka, Mamoru and Tatsuno, Kenji and Ueda, Hiroki and Yamamoto, Shogo and Seki, Motoaki and Masuyama, Nanami and Nishida, Keiji and Nishimasu, Hiroshi and Arakawa, Kazuharu and Kondo, Akihiko and Nureki, Osamu and Tomita, Masaru and Aburatani, Hiroyuki and Yachie, Nozomu},
  doi          = {10.1038/s41587-020-0585-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {901},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Base editors for simultaneous introduction of C-to-T and A-to-G mutations},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: Engineered CRISPR–Cas12a variants with
increased activities and improved targeting ranges for gene, epigenetic
and base editing. <em>NBT</em>, <em>38</em>(7), 901. (<a
href="https://doi.org/10.1038/s41587-020-0587-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Kleinstiver, Benjamin P. and Sousa, Alexander A. and Walton, Russell T. and Tak, Y. Esther and Hsu, Jonathan Y. and Clement, Kendell and Welch, Moira M. and Horng, Joy E. and Malagon-Lopez, Jose and Scarfò, Irene and Maus, Marcela V. and Pinello, Luca and Aryee, Martin J. and Joung, J. Keith},
  doi          = {10.1038/s41587-020-0587-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {901},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Engineered CRISPR–Cas12a variants with increased activities and improved targeting ranges for gene, epigenetic and base editing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Directed evolution of adenine base editors with increased
activity and therapeutic application. <em>NBT</em>, <em>38</em>(7),
892–900. (<a href="https://doi.org/10.1038/s41587-020-0491-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The foundational adenine base editors (for example, ABE7.10) enable programmable A•T to G•C point mutations but editing efficiencies can be low at challenging loci in primary human cells. Here we further evolve ABE7.10 using a library of adenosine deaminase variants to create ABE8s. At NGG protospacer adjacent motif (PAM) sites, ABE8s result in ~1.5× higher editing at protospacer positions A5–A7 and ~3.2× higher editing at positions A3–A4 and A8–A10 compared with ABE7.10. Non-NGG PAM variants have a ~4.2-fold overall higher on-target editing efficiency than ABE7.10. In human CD34+ cells, ABE8 can recreate a natural allele at the promoter of the γ-globin genes HBG1 and HBG2 with up to 60\% efficiency, causing persistence of fetal hemoglobin. In primary human T cells, ABE8s achieve 98–99\% target modification, which is maintained when multiplexed across three loci. Delivered as messenger RNA, ABE8s induce no significant levels of single guide RNA (sgRNA)-independent off-target adenine deamination in genomic DNA and very low levels of adenine deamination in cellular mRNA. Adenine base editors are evolved to be more efficient and more compatible with Cas9 variants.},
  archive      = {J_NBT},
  author       = {Gaudelli, Nicole M. and Lam, Dieter K. and Rees, Holly A. and Solá-Esteves, Noris M. and Barrera, Luis A. and Born, David A. and Edwards, Aaron and Gehrke, Jason M. and Lee, Seung-Joo and Liquori, Alexander J. and Murray, Ryan and Packer, Michael S. and Rinaldi, Conrad and Slaymaker, Ian M. and Yen, Jonathan and Young, Lauren E. and Ciaramella, Giuseppe},
  doi          = {10.1038/s41587-020-0491-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {892-900},
  shortjournal = {Nature Biotech.},
  title        = {Directed evolution of adenine base editors with increased activity and therapeutic application},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Phage-assisted evolution of an adenine base editor with
improved cas domain compatibility and activity. <em>NBT</em>,
<em>38</em>(7), 883–891. (<a
href="https://doi.org/10.1038/s41587-020-0453-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Applications of adenine base editors (ABEs) have been constrained by the limited compatibility of the deoxyadenosine deaminase component with Cas homologs other than SpCas9. We evolved the deaminase component of ABE7.10 using phage-assisted non-continuous and continuous evolution (PANCE and PACE), which resulted in ABE8e. ABE8e contains eight additional mutations that increase activity (kapp) 590-fold compared with that of ABE7.10. ABE8e offers substantially improved editing efficiencies when paired with a variety of Cas9 or Cas12 homologs. ABE8e is more processive than ABE7.10, which could benefit screening, disruption of regulatory regions and multiplex base editing applications. A modest increase in Cas9-dependent and -independent DNA off-target editing, and in transcriptome-wide RNA off-target editing can be ameliorated by the introduction of an additional mutation in the TadA-8e domain. Finally, we show that ABE8e can efficiently install natural mutations that upregulate fetal hemoglobin expression in the BCL11A enhancer or in the the HBG promoter in human cells, targets that were poorly edited with ABE7.10. ABE8e augments the effectiveness and applicability of adenine base editing. A continuously evolved adenine base editor is compatible with various Cas proteins and mediates efficient A•T-to-G•C base conversions at a wide variety of PAM sites.},
  archive      = {J_NBT},
  author       = {Richter, Michelle F. and Zhao, Kevin T. and Eton, Elliot and Lapinaite, Audrone and Newby, Gregory A. and Thuronyi, B W. and Wilson, Christopher and Koblan, Luke W. and Zeng, Jing and Bauer, Daniel E. and Doudna, Jennifer A. and Liu, David R.},
  doi          = {10.1038/s41587-020-0453-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {883-891},
  shortjournal = {Nature Biotech.},
  title        = {Phage-assisted evolution of an adenine base editor with improved cas domain compatibility and activity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Targeted, random mutagenesis of plant genes with dual
cytosine and adenine base editors. <em>NBT</em>, <em>38</em>(7),
875–882. (<a href="https://doi.org/10.1038/s41587-019-0393-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeted saturation mutagenesis of crop genes could be applied to produce genetic variants with improved agronomic performance. However, tools for directed evolution of plant genes, such as error-prone PCR or DNA shuffling, are limited1. We engineered five saturated targeted endogenous mutagenesis editors (STEMEs) that can generate de novo mutations and facilitate directed evolution of plant genes. In rice protoplasts, STEME-1 edited cytosine and adenine at the same target site with C &amp;gt; T efficiency up to 61.61\% and simultaneous C &amp;gt; T and A &amp;gt; G efficiency up to 15.10\%. STEME-NG, which incorporates the nickase Cas9-NG protospacer-adjacent motif variant, was used with 20 individual single guide RNAs in rice protoplasts to produce near-saturated mutagenesis (73.21\%) for a 56-amino-acid portion of the rice acetyl-coenzyme A carboxylase (OsACC). We also applied STEME-1 and STEME-NG for directed evolution of the OsACC gene in rice and obtained herbicide resistance mutations. This set of two STEMEs will accelerate trait development and should work in any plants amenable to CRISPR-based editing. Saturation mutagenesis using dual base editors improves the herbicide resistance of rice.},
  archive      = {J_NBT},
  author       = {Li, Chao and Zhang, Rui and Meng, Xiangbing and Chen, Sha and Zong, Yuan and Lu, Chunju and Qiu, Jin-Long and Chen, Yu-Hang and Li, Jiayang and Gao, Caixia},
  doi          = {10.1038/s41587-019-0393-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {875-882},
  shortjournal = {Nature Biotech.},
  title        = {Targeted, random mutagenesis of plant genes with dual cytosine and adenine base editors},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). CRISPR–Cas12-based detection of SARS-CoV-2. <em>NBT</em>,
<em>38</em>(7), 870–874. (<a
href="https://doi.org/10.1038/s41587-020-0513-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An outbreak of betacoronavirus severe acute respiratory syndrome (SARS)-CoV-2 began in Wuhan, China in December 2019. COVID-19, the disease associated with SARS-CoV-2 infection, rapidly spread to produce a global pandemic. We report development of a rapid (&amp;lt;40 min), easy-to-implement and accurate CRISPR–Cas12-based lateral flow assay for detection of SARS-CoV-2 from respiratory swab RNA extracts. We validated our method using contrived reference samples and clinical samples from patients in the United States, including 36 patients with COVID-19 infection and 42 patients with other viral respiratory infections. Our CRISPR-based DETECTR assay provides a visual and faster alternative to the US Centers for Disease Control and Prevention SARS-CoV-2 real-time RT–PCR assay, with 95\% positive predictive agreement and 100\% negative predictive agreement. SARS-CoV-2 in patient samples is detected in under an hour using a CRISPR-based lateral flow assay.},
  archive      = {J_NBT},
  author       = {Broughton, James P. and Deng, Xianding and Yu, Guixia and Fasching, Clare L. and Servellita, Venice and Singh, Jasmeet and Miao, Xin and Streithorst, Jessica A. and Granados, Andrea and Sotomayor-Gonzalez, Alicia and Zorn, Kelsey and Gopez, Allan and Hsu, Elaine and Gu, Wei and Miller, Steve and Pan, Chao-Yang and Guevara, Hugo and Wadford, Debra A. and Chen, Janice S. and Chiu, Charles Y.},
  doi          = {10.1038/s41587-020-0513-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {870-874},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR–Cas12-based detection of SARS-CoV-2},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Base editors for simultaneous introduction of c-to-t and
a-to-g mutations. <em>NBT</em>, <em>38</em>(7), 865–869. (<a
href="https://doi.org/10.1038/s41587-020-0509-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe base editors that combine both cytosine and adenine base-editing functions. A codon-optimized fusion of the cytosine deaminase PmCDA1, the adenosine deaminase TadA and a Cas9 nickase (Target-ACEmax) showed a high median simultaneous C-to-T and A-to-G editing activity at 47 genomic targets. On-target as well as DNA and RNA off-target activities of Target-ACEmax were similar to those of existing single-function base editors. Base editors that modify both adenosine and cytosine broaden the potential applications of base editing.},
  archive      = {J_NBT},
  author       = {Sakata, Rina C. and Ishiguro, Soh and Mori, Hideto and Tanaka, Mamoru and Tatsuno, Kenji and Ueda, Hiroki and Yamamoto, Shogo and Seki, Motoaki and Masuyama, Nanami and Nishida, Keiji and Nishimasu, Hiroshi and Arakawa, Kazuharu and Kondo, Akihiko and Nureki, Osamu and Tomita, Masaru and Aburatani, Hiroyuki and Yachie, Nozomu},
  doi          = {10.1038/s41587-020-0509-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {865-869},
  shortjournal = {Nature Biotech.},
  title        = {Base editors for simultaneous introduction of C-to-T and A-to-G mutations},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A dual-deaminase CRISPR base editor enables concurrent
adenine and cytosine editing. <em>NBT</em>, <em>38</em>(7), 861–864. (<a
href="https://doi.org/10.1038/s41587-020-0535-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing adenine and cytosine base editors induce only a single type of modification, limiting the range of DNA alterations that can be created. Here we describe a CRISPR–Cas9-based synchronous programmable adenine and cytosine editor (SPACE) that can concurrently introduce A-to-G and C-to-T substitutions with minimal RNA off-target edits. SPACE expands the range of possible DNA sequence alterations, broadening the research applications of CRISPR base editors. A base editor that concurrently modifies both adenine and cytosine broadens the potential applications of base editing.},
  archive      = {J_NBT},
  author       = {Grünewald, Julian and Zhou, Ronghao and Lareau, Caleb A. and Garcia, Sara P. and Iyer, Sowmya and Miller, Bret R. and Langner, Lukas M. and Hsu, Jonathan Y. and Aryee, Martin J. and Joung, J. Keith},
  doi          = {10.1038/s41587-020-0535-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {861-864},
  shortjournal = {Nature Biotech.},
  title        = {A dual-deaminase CRISPR base editor enables concurrent adenine and cytosine editing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Dual base editor catalyzes both cytosine and adenine base
conversions in human cells. <em>NBT</em>, <em>38</em>(7), 856–860. (<a
href="https://doi.org/10.1038/s41587-020-0527-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although base editors are useful tools for precise genome editing, current base editors can only convert either adenines or cytosines. We developed a dual adenine and cytosine base editor (A&amp;amp;C-BEmax) by fusing both deaminases with a Cas9 nickase to achieve C-to-T and A-to-G conversions at the same target site. Compared to single base editors, A&amp;amp;C-BEmax’s activity on adenines is slightly reduced, whereas activity on cytosines is higher and RNA off-target activity is substantially decreased. Base editors that modify both adenine and cytosine broaden the potential applications of base editing.},
  archive      = {J_NBT},
  author       = {Zhang, Xiaohui and Zhu, Biyun and Chen, Liang and Xie, Ling and Yu, Weishi and Wang, Ying and Li, Linxi and Yin, Shuming and Yang, Lei and Hu, Handan and Han, Honghui and Li, Yongmei and Wang, Liren and Chen, Geng and Ma, Xueyun and Geng, Hongquan and Huang, Wanfeng and Pang, Xiufeng and Yang, Zuozhen and Wu, Yuxuan and Siwko, Stefan and Kurita, Ryo and Nakamura, Yukio and Yang, Li and Liu, Mingyao and Li, Dali},
  doi          = {10.1038/s41587-020-0527-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {856-860},
  shortjournal = {Nature Biotech.},
  title        = {Dual base editor catalyzes both cytosine and adenine base conversions in human cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The delivery challenge: Fulfilling the promise of
therapeutic genome editing. <em>NBT</em>, <em>38</em>(7), 845–855. (<a
href="https://doi.org/10.1038/s41587-020-0565-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing has the potential to treat an extensive range of incurable monogenic and complex diseases. In particular, advances in sequence-specific nuclease technologies have dramatically accelerated the development of therapeutic genome editing strategies that are based on either the knockout of disease-causing genes or the repair of endogenous mutated genes. These technologies are progressing into human clinical trials. However, challenges remain before the therapeutic potential of genome editing can be fully realized. Delivery technologies that have serendipitously been developed over the past couple decades in the protein and nucleic acid delivery fields have been crucial to genome editing success to date, including adeno-associated viral and lentiviral vectors for gene therapy and lipid nanoparticle and other non-viral vectors for nucleic acid and protein delivery. However, the efficiency and tissue targeting capabilities of these vehicles must be further improved. In addition, the genome editing enzymes themselves need to be optimized, and challenges regarding their editing efficiency, specificity and immunogenicity must be addressed. Emerging protein engineering and synthetic chemistry approaches can offer solutions and enable the development of safe and efficacious clinical genome editing. As CRISPR therapies move into clinical testing, David Schaffer and colleagues review a raft of different delivery technologies being road tested to address cargo capacity limitations, maximize potency, minimize off-target effects and avoid immunogenicity.},
  archive      = {J_NBT},
  author       = {van Haasteren, Joost and Li, Jie and Scheideler, Olivia J. and Murthy, Niren and Schaffer, David V.},
  doi          = {10.1038/s41587-020-0565-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {845-855},
  shortjournal = {Nature Biotech.},
  title        = {The delivery challenge: Fulfilling the promise of therapeutic genome editing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genome editing with CRISPR–cas nucleases, base editors,
transposases and prime editors. <em>NBT</em>, <em>38</em>(7), 824–844.
(<a href="https://doi.org/10.1038/s41587-020-0561-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of new CRISPR–Cas genome editing tools continues to drive major advances in the life sciences. Four classes of CRISPR–Cas-derived genome editing agents—nucleases, base editors, transposases/recombinases and prime editors—are currently available for modifying genomes in experimental systems. Some of these agents have also moved rapidly into the clinic. Each tool comes with its own capabilities and limitations, and major efforts have broadened their editing capabilities, expanded their targeting scope and improved editing specificity. We analyze key considerations when choosing genome editing agents and identify opportunities for future improvements and applications in basic research and therapeutics. A growing arsenal of CRISPR-based tools enables increasingly sophisticated genome editing applications.},
  archive      = {J_NBT},
  author       = {Anzalone, Andrew V. and Koblan, Luke W. and Liu, David R.},
  doi          = {10.1038/s41587-020-0561-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {824-844},
  shortjournal = {Nature Biotech.},
  title        = {Genome editing with CRISPR–Cas nucleases, base editors, transposases and prime editors},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Design and analysis of CRISPR–cas experiments. <em>NBT</em>,
<em>38</em>(7), 813–823. (<a
href="https://doi.org/10.1038/s41587-020-0490-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A large and ever-expanding set of CRISPR–Cas systems now enables the rapid and flexible manipulation of genomes in both targeted and large-scale experiments. Numerous software tools and analytical methods have been developed for the design and analysis of CRISPR–Cas experiments, including resources to design optimal guide RNAs for various modes of manipulation and to analyze the results of such experiments. A major recent focus has been the development of comprehensive tools for use on data from large-scale CRISPR-based genetic screens. As this field continues to progress, a clear ongoing challenge is not only to innovate, but to actively maintain and improve existing tools so that researchers across disciplines can rely on a stable set of excellent computational resources for CRISPR–Cas experiments. Hanna and Doench review the computational methods and tools that have become indispensable for planning and analyzing CRISPR experiments.},
  archive      = {J_NBT},
  author       = {Hanna, Ruth E. and Doench, John G.},
  doi          = {10.1038/s41587-020-0490-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {813-823},
  shortjournal = {Nature Biotech.},
  title        = {Design and analysis of CRISPR–Cas experiments},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). First rounders: Katrine bosley. <em>NBT</em>,
<em>38</em>(7), 812. (<a
href="https://doi.org/10.1038/s41587-020-0598-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0598-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {812},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Katrine bosley},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Gene editing. <em>NBT</em>, <em>38</em>(7), 811. (<a
href="https://doi.org/10.1038/s41587-020-0599-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to CRISPR-based gene editing, disease treatment and methods for targeting nucleic acids.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0599-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {811},
  shortjournal = {Nature Biotech.},
  title        = {Gene editing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Microbiome therapeutics and patent protection. <em>NBT</em>,
<em>38</em>(7), 806–810. (<a
href="https://doi.org/10.1038/s41587-020-0579-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obtaining patent protection for microbiome therapeutics has been affected profoundly by the US Supreme Court decisions in Myriad and Mayo, which redefined the scope of which natural phenomena, including microbiome therapeutics, are patent eligible.},
  archive      = {J_NBT},
  author       = {FitzGerald, Mark J. and Spek, Erik J.},
  doi          = {10.1038/s41587-020-0579-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {806-810},
  shortjournal = {Nature Biotech.},
  title        = {Microbiome therapeutics and patent protection},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). The biopharmaceutical anomaly. <em>NBT</em>,
<em>38</em>(7), 798–805. (<a
href="https://doi.org/10.1038/s41587-020-0593-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Investment and funding has continued to flow into biotech, unlike most business sectors. But with healthcare and hospital budgets increasingly under pressure, do belt-tightening and consolidation lie ahead? Melanie Senior investigates, with additional reporting by Riku Lähteenmäki.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-020-0593-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {798-805},
  shortjournal = {Nature Biotech.},
  title        = {The biopharmaceutical anomaly},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Blueprint for a pop-up SARS-CoV-2 testing lab. <em>NBT</em>,
<em>38</em>(7), 791–797. (<a
href="https://doi.org/10.1038/s41587-020-0583-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0583-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {791-797},
  shortjournal = {Nature Biotech.},
  title        = {Blueprint for a pop-up SARS-CoV-2 testing lab},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Implications of antibody-dependent enhancement of infection
for SARS-CoV-2 countermeasures. <em>NBT</em>, <em>38</em>(7), 789–791.
(<a href="https://doi.org/10.1038/s41587-020-0577-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Eroshenko, Nikolai and Gill, Taylor and Keaveney, Marianna K. and Church, George M. and Trevejo, Jose M. and Rajaniemi, Hannu},
  doi          = {10.1038/s41587-020-0577-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {789-791},
  shortjournal = {Nature Biotech.},
  title        = {Implications of antibody-dependent enhancement of infection for SARS-CoV-2 countermeasures},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Regulatory centaurs. <em>NBT</em>, <em>38</em>(7), 788–789.
(<a href="https://doi.org/10.1038/s41587-020-0589-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Pitts, Peter J.},
  doi          = {10.1038/s41587-020-0589-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {788-789},
  shortjournal = {Nature Biotech.},
  title        = {Regulatory centaurs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). What you need to know when working with big pharma.
<em>NBT</em>, <em>38</em>(7), 785–787. (<a
href="https://doi.org/10.1038/s41587-020-0576-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ask any pharma executive whether their company is interested in academic collaborations and the answer is a resounding “yes.” So why do so few academic institutions have success stories to share?},
  archive      = {J_NBT},
  author       = {López, Juan Carlos},
  doi          = {10.1038/s41587-020-0576-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {785-787},
  shortjournal = {Nature Biotech.},
  title        = {What you need to know when working with big pharma},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). First NGS-based COVID-19 diagnostic. <em>NBT</em>,
<em>38</em>(7), 777. (<a
href="https://doi.org/10.1038/s41587-020-0608-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0608-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {777},
  shortjournal = {Nature Biotech.},
  title        = {First NGS-based COVID-19 diagnostic},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Bioconcrete presages new wave in environmentally friendly
construction. <em>NBT</em>, <em>38</em>(7), 776–778. (<a
href="https://doi.org/10.1038/s41587-020-0595-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Living cells used in construction to create concrete, bricks or tiles could also soak up CO2.},
  archive      = {J_NBT},
  author       = {Peplow, Mark},
  doi          = {10.1038/s41587-020-0595-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {776-778},
  shortjournal = {Nature Biotech.},
  title        = {Bioconcrete presages new wave in environmentally friendly construction},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Around the world in a month. <em>NBT</em>, <em>38</em>(7),
775. (<a href="https://doi.org/10.1038/s41587-020-0596-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0596-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {775},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Danish bispecifics meet ADCs. <em>NBT</em>, <em>38</em>(7),
774. (<a href="https://doi.org/10.1038/s41587-020-0607-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0607-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {774},
  shortjournal = {Nature Biotech.},
  title        = {Danish bispecifics meet ADCs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A COVID-19 model. <em>NBT</em>, <em>38</em>(7), 773. (<a
href="https://doi.org/10.1038/s41587-020-0606-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0606-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {773},
  shortjournal = {Nature Biotech.},
  title        = {A COVID-19 model},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). COVID-19 spurs wave of innovative diagnostics.
<em>NBT</em>, <em>38</em>(7), 769–772. (<a
href="https://doi.org/10.1038/s41587-020-0597-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coronavirus is instigating new thinking in diagnostics, such as face masks that can detect viruses on a wearer’s breath or paper-based microfluidics for pathogen identification in low-resource settings.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-020-0597-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {769-772},
  shortjournal = {Nature Biotech.},
  title        = {COVID-19 spurs wave of innovative diagnostics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The app credibility gap. <em>NBT</em>, <em>38</em>(7), 768.
(<a href="https://doi.org/10.1038/s41587-020-0610-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A slew of national COVID-19 exposure apps have failed because of poor public engagement and acceptance. Apps from the workplace may offer a partial solution if privacy can be guaranteed.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0610-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {768},
  shortjournal = {Nature Biotech.},
  title        = {The app credibility gap},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The clinical cutting edge. <em>NBT</em>, <em>38</em>(7),
767. (<a href="https://doi.org/10.1038/s41587-020-0612-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As CRISPR–Cas9 enters human testing, the blistering pace of progress in preclinical research is offering a bewildering array of options for therapeutic gene editing.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0612-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {767},
  shortjournal = {Nature Biotech.},
  title        = {The clinical cutting edge},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Focus on CRISPR tools and therapies. <em>NBT</em>,
<em>38</em>(7), 765. (<a
href="https://doi.org/10.1038/s41587-020-0611-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0611-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {765},
  shortjournal = {Nature Biotech.},
  title        = {Focus on CRISPR tools and therapies},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). The hothouse for protein design. <em>NBT</em>,
<em>38</em>(7), 779–784. (<a
href="https://doi.org/10.1038/s41587-020-0586-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The University of Washington’s Institute for Protein Design has become a hub to galvanize de novo protein engineering, citizen science and much more since its founding in 2012.},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-020-0586-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {7},
  pages        = {779-784},
  shortjournal = {Nature Biotech.},
  title        = {The hothouse for protein design},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). People. <em>NBT</em>, <em>38</em>(6), 764. (<a
href="https://doi.org/10.1038/s41587-020-0557-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0557-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {764},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Translating academic careers into industry healthcare
professions. <em>NBT</em>, <em>38</em>(6), 758–763. (<a
href="https://doi.org/10.1038/s41587-020-0552-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Though academic institutions have contributed enormously to the remarkable growth in the healthcare industry, a key limiting resource for this continued growth is the creation of career opportunities for doctoral graduates.},
  archive      = {J_NBT},
  author       = {Gehr, Sinje and Garner, Craig C. and Kleinhans, Karin N.},
  doi          = {10.1038/s41587-020-0552-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {758-763},
  shortjournal = {Nature Biotech.},
  title        = {Translating academic careers into industry healthcare professions},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: ChromID identifies the protein
interactome at chromatin marks. <em>NBT</em>, <em>38</em>(6), 757. (<a
href="https://doi.org/10.1038/s41587-020-0484-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Villaseñor, Rodrigo and Pfaendler, Ramon and Ambrosi, Christina and Butz, Stefan and Giuliani, Sara and Bryan, Elana and Sheahan, Thomas W. and Gable, Annika L. and Schmolka, Nina and Manzo, Massimiliano and Wirz, Joël and Feller, Christian and von Mering, Christian and Aebersold, Ruedi and Voigt, Philipp and Baubec, Tuncay},
  doi          = {10.1038/s41587-020-0484-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {757},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: ChromID identifies the protein interactome at chromatin marks},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Publisher correction: The extended polydimensional immunome
characterization (EPIC) web-based reference and discovery tool for
cytometry data. <em>NBT</em>, <em>38</em>(6), 757. (<a
href="https://doi.org/10.1038/s41587-020-0574-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Yeo, Joo Guan and Wasser, Martin and Kumar, Pavanish and Pan, Lu and Poh, Su Li and Ally, Fauziah and Arkachaisri, Thaschawee and Lim, Amanda Jin Mei and Leong, Jing Yao and Lai, Liyun and Yeo, Kee Thai and Lee, Elene Seck Choon and Chua, Camillus Jian Hui and Larbi, Anis and Nyunt, Ma Shwe Zin and Ng, Tze Pin and Chiesa, Sabrina and Gattorno, Marco and Martini, Alberto and Paleja, Bhairav Suryakant and Dutertre, Charles-Antoine and Chen, Jinmiao and Nay Yaung, Katherine and Tang, Swee Ping and Ng, Sue Kheng and Yung, Chee Fu and Tan, Angela Yun June and Lee, Shu Ying and Ginhoux, Florent and Albani, Salvatore},
  doi          = {10.1038/s41587-020-0574-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {757},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: The extended polydimensional immunome characterization (EPIC) web-based reference and discovery tool for cytometry data},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Systematic comparison of single-cell and
single-nucleus RNA-sequencing methods. <em>NBT</em>, <em>38</em>(6),
756. (<a href="https://doi.org/10.1038/s41587-020-0534-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Ding, Jiarui and Adiconis, Xian and Simmons, Sean K. and Kowalczyk, Monika S. and Hession, Cynthia C. and Marjanovic, Nemanja D. and Hughes, Travis K. and Wadsworth, Marc H. and Burks, Tyler and Nguyen, Lan T. and Kwon, John Y. H. and Barak, Boaz and Ge, William and Kedaigle, Amanda J. and Carroll, Shaina and Li, Shuqiang and Hacohen, Nir and Rozenblatt-Rosen, Orit and Shalek, Alex K. and Villani, Alexandra-Chloé and Regev, Aviv and Levin, Joshua Z.},
  doi          = {10.1038/s41587-020-0534-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {756},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Systematic comparison of single-cell and single-nucleus RNA-sequencing methods},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: High-throughput single-cell
activity-based screening and sequencing of antibodies using droplet
microfluidics. <em>NBT</em>, <em>38</em>(6), 756. (<a
href="https://doi.org/10.1038/s41587-020-0563-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Gérard, Annabelle and Woolfe, Adam and Mottet, Guillaume and Reichen, Marcel and Castrillon, Carlos and Menrath, Vera and Ellouze, Sami and Poitou, Adeline and Doineau, Raphaël and Briseno-Roa, Luis and Canales-Herrerias, Pablo and Mary, Pascaline and Rose, Gregory and Ortega, Charina and Delincé, Matthieu and Essono, Sosthene and Jia, Bin and Iannascoli, Bruno and Richard-Le Goff, Odile and Kumar, Roshan and Stewart, Samantha N. and Pousse, Yannick and Shen, Bingqing and Grosselin, Kevin and Saudemont, Baptiste and Sautel-Caillé, Antoine and Godina, Alexei and McNamara, Scott and Eyer, Klaus and Millot, Gaël A. and Baudry, Jean and England, Patrick and Nizak, Clément and Jensen, Allan and Griffiths, Andrew D. and Bruhns, Pierre and Brenan, Colin},
  doi          = {10.1038/s41587-020-0563-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {756},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: High-throughput single-cell activity-based screening and sequencing of antibodies using droplet microfluidics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Benchmarking single-cell RNA-sequencing protocols for cell
atlas projects. <em>NBT</em>, <em>38</em>(6), 747–755. (<a
href="https://doi.org/10.1038/s41587-020-0469-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing (scRNA-seq) is the leading technique for characterizing the transcriptomes of individual cells in a sample. The latest protocols are scalable to thousands of cells and are being used to compile cell atlases of tissues, organs and organisms. However, the protocols differ substantially with respect to their RNA capture efficiency, bias, scale and costs, and their relative advantages for different applications are unclear. In the present study, we generated benchmark datasets to systematically evaluate protocols in terms of their power to comprehensively describe cell types and states. We performed a multicenter study comparing 13 commonly used scRNA-seq and single-nucleus RNA-seq protocols applied to a heterogeneous reference sample resource. Comparative analysis revealed marked differences in protocol performance. The protocols differed in library complexity and their ability to detect cell-type markers, impacting their predictive value and suitability for integration into reference cell atlases. These results provide guidance both for individual researchers and for consortium projects such as the Human Cell Atlas. A multicenter study compares 13 commonly used single-cell RNA-seq protocols.},
  archive      = {J_NBT},
  author       = {Mereu, Elisabetta and Lafzi, Atefeh and Moutinho, Catia and Ziegenhain, Christoph and McCarthy, Davis J. and Álvarez-Varela, Adrián and Batlle, Eduard and Sagar and Grün, Dominic and Lau, Julia K. and Boutet, Stéphane C. and Sanada, Chad and Ooi, Aik and Jones, Robert C. and Kaihara, Kelly and Brampton, Chris and Talaga, Yasha and Sasagawa, Yohei and Tanaka, Kaori and Hayashi, Tetsutaro and Braeuning, Caroline and Fischer, Cornelius and Sauer, Sascha and Trefzer, Timo and Conrad, Christian and Adiconis, Xian and Nguyen, Lan T. and Regev, Aviv and Levin, Joshua Z. and Parekh, Swati and Janjic, Aleksandar and Wange, Lucas E. and Bagnoli, Johannes W. and Enard, Wolfgang and Gut, Marta and Sandberg, Rickard and Nikaido, Itoshi and Gut, Ivo and Stegle, Oliver and Heyn, Holger},
  doi          = {10.1038/s41587-020-0469-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {747-755},
  shortjournal = {Nature Biotech.},
  title        = {Benchmarking single-cell RNA-sequencing protocols for cell atlas projects},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Systematic comparison of single-cell and single-nucleus
RNA-sequencing methods. <em>NBT</em>, <em>38</em>(6), 737–746. (<a
href="https://doi.org/10.1038/s41587-020-0465-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scale and capabilities of single-cell RNA-sequencing methods have expanded rapidly in recent years, enabling major discoveries and large-scale cell mapping efforts. However, these methods have not been systematically and comprehensively benchmarked. Here, we directly compare seven methods for single-cell and/or single-nucleus profiling—selecting representative methods based on their usage and our expertise and resources to prepare libraries—including two low-throughput and five high-throughput methods. We tested the methods on three types of samples: cell lines, peripheral blood mononuclear cells and brain tissue, generating 36 libraries in six separate experiments in a single center. To directly compare the methods and avoid processing differences introduced by the existing pipelines, we developed scumi, a flexible computational pipeline that can be used with any single-cell RNA-sequencing method. We evaluated the methods for both basic performance, such as the structure and alignment of reads, sensitivity and extent of multiplets, and for their ability to recover known biological information in the samples. Seven methods for single-cell RNA sequencing are benchmarked on cell lines, primary cells and mouse cortex.},
  archive      = {J_NBT},
  author       = {Ding, Jiarui and Adiconis, Xian and Simmons, Sean K. and Kowalczyk, Monika S. and Hession, Cynthia C. and Marjanovic, Nemanja D. and Hughes, Travis K. and Wadsworth, Marc H. and Burks, Tyler and Nguyen, Lan T. and Kwon, John Y. H. and Barak, Boaz and Ge, William and Kedaigle, Amanda J. and Carroll, Shaina and Li, Shuqiang and Hacohen, Nir and Rozenblatt-Rosen, Orit and Shalek, Alex K. and Villani, Alexandra-Chloé and Regev, Aviv and Levin, Joshua Z.},
  doi          = {10.1038/s41587-020-0465-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {737-746},
  shortjournal = {Nature Biotech.},
  title        = {Systematic comparison of single-cell and single-nucleus RNA-sequencing methods},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). ChromID identifies the protein interactome at chromatin
marks. <em>NBT</em>, <em>38</em>(6), 728–736. (<a
href="https://doi.org/10.1038/s41587-020-0434-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromatin modifications regulate genome function by recruiting proteins to the genome. However, the protein composition at distinct chromatin modifications has yet to be fully characterized. In this study, we used natural protein domains as modular building blocks to develop engineered chromatin readers (eCRs) selective for DNA methylation and histone tri-methylation at H3K4, H3K9 and H3K27 residues. We first demonstrated their utility as selective chromatin binders in living cells by stably expressing eCRs in mouse embryonic stem cells and measuring their subnuclear localization, genomic distribution and histone-modification-binding preference. By fusing eCRs to the biotin ligase BASU, we established ChromID, a method for identifying the chromatin-dependent protein interactome on the basis of proximity biotinylation, and applied it to distinct chromatin modifications in mouse stem cells. Using a synthetic dual-modification reader, we also uncovered the protein composition at bivalently modified promoters marked by H3K4me3 and H3K27me3. These results highlight the ability of ChromID to obtain a detailed view of protein interaction networks on chromatin. The protein complexes associated with specific chromatin marks in living cells are identified using engineered binding proteins.},
  archive      = {J_NBT},
  author       = {Villaseñor, Rodrigo and Pfaendler, Ramon and Ambrosi, Christina and Butz, Stefan and Giuliani, Sara and Bryan, Elana and Sheahan, Thomas W. and Gable, Annika L. and Schmolka, Nina and Manzo, Massimiliano and Wirz, Joël and Feller, Christian and von Mering, Christian and Aebersold, Ruedi and Voigt, Philipp and Baubec, Tuncay},
  doi          = {10.1038/s41587-020-0434-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {728-736},
  shortjournal = {Nature Biotech.},
  title        = {ChromID identifies the protein interactome at chromatin marks},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Massively parallel cas13 screens reveal principles for guide
RNA design. <em>NBT</em>, <em>38</em>(6), 722–727. (<a
href="https://doi.org/10.1038/s41587-020-0456-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Type VI CRISPR enzymes are RNA-targeting proteins with nuclease activity that enable specific and robust target gene knockdown without altering the genome. To define rules for the design of Cas13d guide RNAs (gRNAs), we conducted massively parallel screens targeting messenger RNAs (mRNAs) of a green fluorescent protein transgene, and CD46, CD55 and CD71 cell-surface proteins in human cells. In total, we measured the activity of 24,460 gRNAs with and without mismatches relative to the target sequences. Knockdown efficacy is driven by gRNA-specific features and target site context. Single mismatches generally reduce knockdown to a modest degree, but spacer nucleotides 15–21 are largely intolerant of target site mismatches. We developed a computational model to identify optimal gRNAs and confirm their generalizability, testing 3,979 guides targeting mRNAs of 48 endogenous genes. We show that Cas13 can be used in forward transcriptomic pooled screens and, using our model, predict optimized Cas13 gRNAs for all protein-coding transcripts in the human genome. Design of optimal guide RNAs for Cas13d-mediated RNA knockdown is enabled by computational modeling of large-scale screening data.},
  archive      = {J_NBT},
  author       = {Wessels, Hans-Hermann and Méndez-Mancilla, Alejandro and Guo, Xinyi and Legut, Mateusz and Daniloski, Zharko and Sanjana, Neville E.},
  doi          = {10.1038/s41587-020-0456-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {722-727},
  shortjournal = {Nature Biotech.},
  title        = {Massively parallel cas13 screens reveal principles for guide RNA design},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). High-throughput single-cell activity-based screening and
sequencing of antibodies using droplet microfluidics. <em>NBT</em>,
<em>38</em>(6), 715–721. (<a
href="https://doi.org/10.1038/s41587-020-0466-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mining the antibody repertoire of plasma cells and plasmablasts could enable the discovery of useful antibodies for therapeutic or research purposes1. We present a method for high-throughput, single-cell screening of IgG-secreting primary cells to characterize antibody binding to soluble and membrane-bound antigens. CelliGO is a droplet microfluidics system that combines high-throughput screening for IgG activity, using fluorescence-based in-droplet single-cell bioassays2, with sequencing of paired antibody V genes, using in-droplet single-cell barcoded reverse transcription. We analyzed IgG repertoire diversity, clonal expansion and somatic hypermutation in cells from mice immunized with a vaccine target, a multifunctional enzyme or a membrane-bound cancer target. Immunization with these antigens yielded 100–1,000 IgG sequences per mouse. We generated 77 recombinant antibodies from the identified sequences and found that 93\% recognized the soluble antigen and 14\% the membrane antigen. The platform also allowed recovery of ~450–900 IgG sequences from ~2,200 IgG-secreting activated human memory B cells, activated ex vivo, demonstrating its versatility. Millions of primary IgG-secreting cells from mouse and human are characterized for activity and antibody sequence at the single-cell level.},
  archive      = {J_NBT},
  author       = {Gérard, Annabelle and Woolfe, Adam and Mottet, Guillaume and Reichen, Marcel and Castrillon, Carlos and Menrath, Vera and Ellouze, Sami and Poitou, Adeline and Doineau, Raphaël and Briseno-Roa, Luis and Canales-Herrerias, Pablo and Mary, Pascaline and Rose, Gregory and Ortega, Charina and Delincé, Matthieu and Essono, Sosthene and Jia, Bin and Iannascoli, Bruno and Richard-Le Goff, Odile and Kumar, Roshan and Stewart, Samantha N. and Pousse, Yannick and Shen, Bingqing and Grosselin, Kevin and Saudemont, Baptiste and Sautel-Caillé, Antoine and Godina, Alexei and McNamara, Scott and Eyer, Klaus and Millot, Gaël A. and Baudry, Jean and England, Patrick and Nizak, Clément and Jensen, Allan and Griffiths, Andrew D. and Bruhns, Pierre and Brenan, Colin},
  doi          = {10.1038/s41587-020-0466-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {715-721},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput single-cell activity-based screening and sequencing of antibodies using droplet microfluidics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell RNA counting at allele and isoform resolution
using smart-seq3. <em>NBT</em>, <em>38</em>(6), 708–714. (<a
href="https://doi.org/10.1038/s41587-020-0497-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale sequencing of RNA from individual cells can reveal patterns of gene, isoform and allelic expression across cell types and states1. However, current short-read single-cell RNA-sequencing methods have limited ability to count RNAs at allele and isoform resolution, and long-read sequencing techniques lack the depth required for large-scale applications across cells2,3. Here we introduce Smart-seq3, which combines full-length transcriptome coverage with a 5′ unique molecular identifier RNA counting strategy that enables in silico reconstruction of thousands of RNA molecules per cell. Of the counted and reconstructed molecules, 60\% could be directly assigned to allelic origin and 30–50\% to specific isoforms, and we identified substantial differences in isoform usage in different mouse strains and human cell types. Smart-seq3 greatly increased sensitivity compared to Smart-seq2, typically detecting thousands more transcripts per cell. We expect that Smart-seq3 will enable large-scale characterization of cell types and states across tissues and organisms. Smart-seq3 enables isoform- and allele-specific reconstruction of RNA molecules.},
  archive      = {J_NBT},
  author       = {Hagemann-Jensen, Michael and Ziegenhain, Christoph and Chen, Ping and Ramsköld, Daniel and Hendriks, Gert-Jan and Larsson, Anton J. M. and Faridani, Omid R. and Sandberg, Rickard},
  doi          = {10.1038/s41587-020-0497-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {708-714},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell RNA counting at allele and isoform resolution using smart-seq3},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Complete, closed bacterial genomes from microbiomes using
nanopore sequencing. <em>NBT</em>, <em>38</em>(6), 701–707. (<a
href="https://doi.org/10.1038/s41587-020-0422-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbial genomes can be assembled from short-read sequencing data, but the assembly contiguity of these metagenome-assembled genomes is constrained by repeat elements. Correct assignment of genomic positions of repeats is crucial for understanding the effect of genome structure on genome function. We applied nanopore sequencing and our workflow, named Lathe, which incorporates long-read assembly and short-read error correction, to assemble closed bacterial genomes from complex microbiomes. We validated our approach with a synthetic mixture of 12 bacterial species. Seven genomes were completely assembled into single contigs and three genomes were assembled into four or fewer contigs. Next, we used our methods to analyze metagenomics data from 13 human stool samples. We assembled 20 circular genomes, including genomes of Prevotella copri and a candidate Cibiobacter sp. Despite the decreased nucleotide accuracy compared with alternative sequencing and assembly approaches, our methods improved assembly contiguity, allowing for investigation of the role of repeat elements in microbial function and adaptation. Nanopore-metagenomics workflow enables assembly of complete, closed bacterial genomes from human stool samples.},
  archive      = {J_NBT},
  author       = {Moss, Eli L. and Maghini, Dylan G. and Bhatt, Ami S.},
  doi          = {10.1038/s41587-020-0422-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {701-707},
  shortjournal = {Nature Biotech.},
  title        = {Complete, closed bacterial genomes from microbiomes using nanopore sequencing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell RNA sequencing at isoform resolution.
<em>NBT</em>, <em>38</em>(6), 697–698. (<a
href="https://doi.org/10.1038/s41587-020-0553-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SMART-seq3 reconstructs the full-length transcripts of single cells with high sensitivity.},
  archive      = {J_NBT},
  author       = {Macosko, Evan Z.},
  doi          = {10.1038/s41587-020-0553-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {697-698},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell RNA sequencing at isoform resolution},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Coronaviruses. <em>NBT</em>, <em>38</em>(6), 695. (<a
href="https://doi.org/10.1038/s41587-020-0559-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to vaccines and methods of treatment of coronaviruses.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0559-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {695},
  shortjournal = {Nature Biotech.},
  title        = {Coronaviruses},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). 3D and 4D bioprinted human model patenting and the future of
drug development. <em>NBT</em>, <em>38</em>(6), 689–694. (<a
href="https://doi.org/10.1038/s41587-020-0540-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bioprinted 3D and 4D tissues and organs are expected to revolutionize the biomedical field, eliminating the need for laboratory animals, but little is known about the future impacts of these technologies on drug development.},
  archive      = {J_NBT},
  author       = {Mota, Fabio and Braga, Luiza and Rocha, Leonardo and Cabral, Bernardo},
  doi          = {10.1038/s41587-020-0540-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {689-694},
  shortjournal = {Nature Biotech.},
  title        = {3D and 4D bioprinted human model patenting and the future of drug development},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). PICRUSt2 for prediction of metagenome functions.
<em>NBT</em>, <em>38</em>(6), 685–688. (<a
href="https://doi.org/10.1038/s41587-020-0548-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Douglas, Gavin M. and Maffei, Vincent J. and Zaneveld, Jesse R. and Yurgel, Svetlana N. and Brown, James R. and Taylor, Christopher M. and Huttenhower, Curtis and Langille, Morgan G. I.},
  doi          = {10.1038/s41587-020-0548-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {685-688},
  shortjournal = {Nature Biotech.},
  title        = {PICRUSt2 for prediction of metagenome functions},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). The extended polydimensional immunome characterization
(EPIC) web-based reference and discovery tool for cytometry data.
<em>NBT</em>, <em>38</em>(6), 679–684. (<a
href="https://doi.org/10.1038/s41587-020-0532-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yeo, Joo Guan and Wasser, Martin and Kumar, Pavanish and Pan, Lu and Poh, Su Li and Ally, Fauziah and Arkachaisri, Thaschawee and Lim, Amanda Jin Mei and Leong, Jing Yao and Lai, Liyun and Yeo, Kee Thai and Lee, Elene Seck Choon and Chua, Camillus Jian Hui and Larbi, Anis and Nyunt, Ma Shwe Zin and Ng, Tze Pin and Chiesa, Sabrina and Gattorno, Marco and Martini, Alberto and Paleja, Bhairav Suryakant and Dutertre, Charles-Antoine and Chen, Jinmiao and Nay Yaung, Katherine and Tang, Swee Ping and Ng, Sue Kheng and Yung, Chee Fu and Tan, Angela Yun June and Lee, Shu Ying and Ginhoux, Florent and Albani, Salvatore},
  doi          = {10.1038/s41587-020-0532-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {679-684},
  shortjournal = {Nature Biotech.},
  title        = {The extended polydimensional immunome characterization (EPIC) web-based reference and discovery tool for cytometry data},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Visualizing and interpreting cancer genomics data via the
xena platform. <em>NBT</em>, <em>38</em>(6), 675–678. (<a
href="https://doi.org/10.1038/s41587-020-0546-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Goldman, Mary J. and Craft, Brian and Hastie, Mim and Repečka, Kristupas and McDade, Fran and Kamath, Akhil and Banerjee, Ayan and Luo, Yunhai and Rogers, Dave and Brooks, Angela N. and Zhu, Jingchun and Haussler, David},
  doi          = {10.1038/s41587-020-0546-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {675-678},
  shortjournal = {Nature Biotech.},
  title        = {Visualizing and interpreting cancer genomics data via the xena platform},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Diagnostics and the coronavirus: Don’t let the standards
slip. <em>NBT</em>, <em>38</em>(6), 673–674. (<a
href="https://doi.org/10.1038/s41587-020-0558-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Page, Mark and Almond, Neil and Rose, Nicola J. and Schneider, Christian K.},
  doi          = {10.1038/s41587-020-0558-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {673-674},
  shortjournal = {Nature Biotech.},
  title        = {Diagnostics and the coronavirus: Don’t let the standards slip},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Overcoming barriers to early disease intervention.
<em>NBT</em>, <em>38</em>(6), 669–673. (<a
href="https://doi.org/10.1038/s41587-020-0550-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Caicedo, H. Hugo and Hashimoto, Daniel A. and Caicedo, Julio C. and Pentland, Alex and Pisano, Gary P.},
  doi          = {10.1038/s41587-020-0550-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {669-673},
  shortjournal = {Nature Biotech.},
  title        = {Overcoming barriers to early disease intervention},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). FDA approves first RET inhibitor. <em>NBT</em>,
<em>38</em>(6), 662. (<a
href="https://doi.org/10.1038/s41587-020-0568-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0568-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {662},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves first RET inhibitor},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Dyno advances AI-powered gene therapy. <em>NBT</em>,
<em>38</em>(6), 661. (<a
href="https://doi.org/10.1038/s41587-020-0569-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0569-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {661},
  shortjournal = {Nature Biotech.},
  title        = {Dyno advances AI-powered gene therapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Around the world in a month. <em>NBT</em>, <em>38</em>(6),
658. (<a href="https://doi.org/10.1038/s41587-020-0556-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0556-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {658},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genomic search for COVID-19 severity clues. <em>NBT</em>,
<em>38</em>(6), 657. (<a
href="https://doi.org/10.1038/s41587-020-0567-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0567-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {657},
  shortjournal = {Nature Biotech.},
  title        = {Genomic search for COVID-19 severity clues},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The COVID-19 testing debacle. <em>NBT</em>, <em>38</em>(6),
653. (<a href="https://doi.org/10.1038/s41587-020-0575-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Insufficient SARS-CoV-2 testing has left many nations flying blind as they reopen their economies. Without adequate testing, new infections could outrun control measures.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0575-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {653},
  shortjournal = {Nature Biotech.},
  title        = {The COVID-19 testing debacle},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). First rounders: Alexis borisy. <em>NBT</em>,
<em>38</em>(6), 699. (<a
href="https://doi.org/10.1038/s41587-020-0542-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0542-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {699},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Alexis borisy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Forum: Worms and gut microbes team up to kill corn pest.
<em>NBT</em>, <em>38</em>(6), 699. (<a
href="https://doi.org/10.1038/s41587-020-0543-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0543-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {699},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Worms and gut microbes team up to kill corn pest},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Forum: First human trial of engineered t cells.
<em>NBT</em>, <em>38</em>(6), 699. (<a
href="https://doi.org/10.1038/s41587-020-0544-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0544-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {699},
  shortjournal = {Nature Biotech.},
  title        = {Forum: First human trial of engineered t cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Closing the recycling circle. <em>NBT</em>, <em>38</em>(6),
665–668. (<a href="https://doi.org/10.1038/s41587-020-0541-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Capturing the value back from plastic waste has been the holy grail of recyclers. Biotechnology is taking us closer to a solution.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0541-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {665-668},
  shortjournal = {Nature Biotech.},
  title        = {Closing the recycling circle},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Biotech companies prepare for COVID-19 downturn.
<em>NBT</em>, <em>38</em>(6), 663–664. (<a
href="https://doi.org/10.1038/d41587-020-00012-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biopharmaceutical companies are awash with more cash than at any time in the sector’s history. But they are scrambling to prepare for the drought of a COVID-19-related recession. Biopharmaceutical companies are awash with more cash than at any time in the sector’s history. But they are scrambling to prepare for the drought of a COVID-19-related recession.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/d41587-020-00012-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {663-664},
  shortjournal = {Nature Biotech.},
  title        = {Biotech companies prepare for COVID-19 downturn},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Drug researchers pursue new lines of attack against
COVID-19. <em>NBT</em>, <em>38</em>(6), 659–662. (<a
href="https://doi.org/10.1038/d41587-020-00013-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Learning which proteins in the body are co-opted by coronavirus may uncover new drug targets and enable a better understanding of the body’s responses to infection. Learning which proteins in the body are co-opted by coronavirus may uncover new drug targets and enable a better understanding of the body’s responses to infection.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/d41587-020-00013-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {659-662},
  shortjournal = {Nature Biotech.},
  title        = {Drug researchers pursue new lines of attack against COVID-19},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Convalescent serum lines up as first-choice treatment for
coronavirus. <em>NBT</em>, <em>38</em>(6), 655–658. (<a
href="https://doi.org/10.1038/d41587-020-00011-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antibodies from blood donated by people who recovered from the illness and hyper-immunoglobulins are becoming treatments of choice for COVID-19, with recombinant polyclonal antibody approaches to follow. Antibodies from blood donated by people who recovered from the illness and hyper-immunoglobulins are becoming treatments of choice for COVID-19, with recombinant polyclonal antibody approaches to follow.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-020-00011-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {6},
  pages        = {655-658},
  shortjournal = {Nature Biotech.},
  title        = {Convalescent serum lines up as first-choice treatment for coronavirus},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). People. <em>NBT</em>, <em>38</em>(5), 652. (<a
href="https://doi.org/10.1038/s41587-020-0522-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0522-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {652},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). First-quarter biotech job picture. <em>NBT</em>,
<em>38</em>(5), 651. (<a
href="https://doi.org/10.1038/s41587-020-0515-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-020-0515-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {651},
  shortjournal = {Nature Biotech.},
  title        = {First-quarter biotech job picture},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Publisher correction: Single-cell RNA-seq analysis software
providers scramble to offer solutions. <em>NBT</em>, <em>38</em>(5),
650. (<a href="https://doi.org/10.1038/s41587-020-0510-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-020-0510-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {650},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Single-cell RNA-seq analysis software providers scramble to offer solutions},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Author correction: Reply to: Fitness effects of
CRISPR/Cas9-targeting of long noncoding RNA genes. <em>NBT</em>,
<em>38</em>(5), 649. (<a
href="https://doi.org/10.1038/s41587-020-0475-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Liu, Ying and Liu, Zhiheng and Cao, Zhongzheng and Wei, Wensheng},
  doi          = {10.1038/s41587-020-0475-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {649},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: reply to: fitness effects of CRISPR/Cas9-targeting of long noncoding RNA genes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Engineering bacterial symbionts of
nematodes improves their biocontrol potential to counter the western
corn rootworm. <em>NBT</em>, <em>38</em>(5), 649. (<a
href="https://doi.org/10.1038/s41587-020-0478-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Machado, Ricardo A. R. and Thönen, Lisa and Arce, Carla C. M. and Theepan, Vanitha and Prada, Fausto and Wüthrich, Daniel and Robert, Christelle A. M. and Vogiatzaki, Evangelia and Shi, Yi-Ming and Schaeren, Olivier P. and Notter, Matheus and Bruggmann, Rémy and Hapfelmeier, Siegfried and Bode, Helge B. and Erb, Matthias},
  doi          = {10.1038/s41587-020-0478-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {649},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Engineering bacterial symbionts of nematodes improves their biocontrol potential to counter the western corn rootworm},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Publisher correction: Γδ t cells bring unconventional
cancer-targeting to the clinic — again. <em>NBT</em>, <em>38</em>(5),
649. (<a href="https://doi.org/10.1038/s41587-020-0526-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-020-0526-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {649},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: γδ t cells bring unconventional cancer-targeting to the clinic — again},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genetic interaction mapping and exon-resolution functional
genomics with a hybrid Cas9–Cas12a platform. <em>NBT</em>,
<em>38</em>(5), 638–648. (<a
href="https://doi.org/10.1038/s41587-020-0437-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Systematic mapping of genetic interactions (GIs) and interrogation of the functions of sizable genomic segments in mammalian cells represent important goals of biomedical research. To advance these goals, we present a CRISPR (clustered regularly interspaced short palindromic repeats)-based screening system for combinatorial genetic manipulation that employs coexpression of CRISPR-associated nucleases 9 and 12a (Cas9 and Cas12a) and machine-learning-optimized libraries of hybrid Cas9–Cas12a guide RNAs. This system, named Cas Hybrid for Multiplexed Editing and screening Applications (CHyMErA), outperforms genetic screens using Cas9 or Cas12a editing alone. Application of CHyMErA to the ablation of mammalian paralog gene pairs reveals extensive GIs and uncovers phenotypes normally masked by functional redundancy. Application of CHyMErA in a chemogenetic interaction screen identifies genes that impact cell growth in response to mTOR pathway inhibition. Moreover, by systematically targeting thousands of alternative splicing events, CHyMErA identifies exons underlying human cell line fitness. CHyMErA thus represents an effective screening approach for GI mapping and the functional analysis of sizable genomic regions, such as alternative exons. Combining Cas9 and Cas12a outperforms previous approaches for combinatorial screening in mammalian cells.},
  archive      = {J_NBT},
  author       = {Gonatopoulos-Pournatzis, Thomas and Aregger, Michael and Brown, Kevin R. and Farhangmehr, Shaghayegh and Braunschweig, Ulrich and Ward, Henry N. and Ha, Kevin C. H. and Weiss, Alexander and Billmann, Maximilian and Durbic, Tanja and Myers, Chad L. and Blencowe, Benjamin J. and Moffat, Jason},
  doi          = {10.1038/s41587-020-0437-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {638-648},
  shortjournal = {Nature Biotech.},
  title        = {Genetic interaction mapping and exon-resolution functional genomics with a hybrid Cas9–Cas12a platform},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Dissecting cellular crosstalk by sequencing physically
interacting cells. <em>NBT</em>, <em>38</em>(5), 629–637. (<a
href="https://doi.org/10.1038/s41587-020-0442-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crosstalk between neighboring cells underlies many biological processes, including cell signaling, proliferation and differentiation. Current single-cell genomic technologies profile each cell separately after tissue dissociation, losing information on cell–cell interactions. In the present study, we present an approach for sequencing physically interacting cells (PIC-seq), which combines cell sorting of physically interacting cells (PICs) with single-cell RNA-sequencing. Using computational modeling, PIC-seq systematically maps in situ cellular interactions and characterizes their molecular crosstalk. We apply PIC-seq to interrogate diverse interactions including immune–epithelial PICs in neonatal murine lungs. Focusing on interactions between T cells and dendritic cells (DCs) in vitro and in vivo, we map T cell–DC interaction preferences, and discover regulatory T cells as a major T cell subtype interacting with DCs in mouse draining lymph nodes. Analysis of T cell–DC pairs reveals an interaction-specific program between pathogen-presenting migratory DCs and T cells. PIC-seq provides a direct and broadly applicable technology to characterize intercellular interaction-specific pathways at high resolution. PIC-seq characterizes cellular crosstalk by sorting and sequencing physically interacting cells.},
  archive      = {J_NBT},
  author       = {Giladi, Amir and Cohen, Merav and Medaglia, Chiara and Baran, Yael and Li, Baoguo and Zada, Mor and Bost, Pierre and Blecher-Gonen, Ronnie and Salame, Tomer-Meir and Mayer, Johannes U. and David, Eyal and Ronchese, Franca and Tanay, Amos and Amit, Ido},
  doi          = {10.1038/s41587-020-0442-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {629-637},
  shortjournal = {Nature Biotech.},
  title        = {Dissecting cellular crosstalk by sequencing physically interacting cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evaluation and minimization of cas9-independent off-target
DNA editing by cytosine base editors. <em>NBT</em>, <em>38</em>(5),
620–628. (<a href="https://doi.org/10.1038/s41587-020-0414-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine base editors (CBEs) enable targeted C•G-to-T•A conversions in genomic DNA. Recent studies report that BE3, the original CBE, induces a low frequency of genome-wide Cas9-independent off-target C•G-to-T•A mutation in mouse embryos and in rice. Here we develop multiple rapid, cost-effective methods to screen the propensity of different CBEs to induce Cas9-independent deamination in Escherichia coli and in human cells. We use these assays to identify CBEs with reduced Cas9-independent deamination and validate via whole-genome sequencing that YE1, a narrowed-window CBE variant, displays background levels of Cas9-independent off-target editing. We engineered YE1 variants that retain the substrate-targeting scope of high-activity CBEs while maintaining minimal Cas9-independent off-target editing. The suite of CBEs characterized and engineered in this study collectively offer ~10–100-fold lower average Cas9-independent off-target DNA editing while maintaining robust on-target editing at most positions targetable by canonical CBEs, and thus are especially promising for applications in which off-target editing must be minimized. Methods to efficiently detect Cas9-independent cytosine base editor off-target activity enable the identification and development of variants with minimal off-target editing and robust on-target editing.},
  archive      = {J_NBT},
  author       = {Doman, Jordan L. and Raguram, Aditya and Newby, Gregory A. and Liu, David R.},
  doi          = {10.1038/s41587-020-0414-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {620-628},
  shortjournal = {Nature Biotech.},
  title        = {Evaluation and minimization of cas9-independent off-target DNA editing by cytosine base editors},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Massively parallel interrogation and mining of natively
paired human TCRαβ repertoires. <em>NBT</em>, <em>38</em>(5), 609–619.
(<a href="https://doi.org/10.1038/s41587-020-0438-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cells engineered to express antigen-specific T cell receptors (TCRs) are potent therapies for viral infections and cancer. However, efficient identification of clinical candidate TCRs is complicated by the size and complexity of T cell repertoires and the challenges of working with primary T cells. Here we present a high-throughput method to identify TCRs with high functional avidity from diverse human T cell repertoires. The approach used massively parallel microfluidics to generate libraries of natively paired, full-length TCRαβ clones, from millions of primary T cells, which were then expressed in Jurkat cells. The TCRαβ–Jurkat libraries enabled repeated screening and panning for antigen-reactive TCRs using peptide major histocompatibility complex binding and cellular activation. We captured more than 2.9 million natively paired TCRαβ clonotypes from six healthy human donors and identified rare (&amp;lt;0.001\% frequency) viral-antigen-reactive TCRs. We also mined a tumor-infiltrating lymphocyte sample from a patient with melanoma and identified several tumor-specific TCRs, which, after expression in primary T cells, led to tumor cell killing. T cell receptors are identified in large-scale screening of T cell repertoires cloned from primary human T cells.},
  archive      = {J_NBT},
  author       = {Spindler, Matthew J. and Nelson, Ayla L. and Wagner, Ellen K. and Oppermans, Natasha and Bridgeman, John S. and Heather, James M. and Adler, Adam S. and Asensio, Michael A. and Edgar, Robert C. and Lim, Yoong Wearn and Meyer, Everett H. and Hawkins, Robert E. and Cobbold, Mark and Johnson, David S.},
  doi          = {10.1038/s41587-020-0438-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {609-619},
  shortjournal = {Nature Biotech.},
  title        = {Massively parallel interrogation and mining of natively paired human TCRαβ repertoires},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Engineering bacterial symbionts of nematodes improves their
biocontrol potential to counter the western corn rootworm. <em>NBT</em>,
<em>38</em>(5), 600–608. (<a
href="https://doi.org/10.1038/s41587-020-0419-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The western corn rootworm (WCR) decimates maize crops worldwide. One potential way to control this pest is treatment with entomopathogenic nematodes (EPNs) that harbor bacterial symbionts that are pathogenic to insects. However, WCR larvae sequester benzoxazinoid secondary metabolites that are produced by maize and use them to increase their resistance to the nematodes and their symbionts. Here we report that experimental evolution and selection for bacterial symbionts that are resistant to benzoxazinoids improve the ability of a nematode–symbiont pair to kill WCR larvae. We isolated five Photorhabdus symbionts from different nematodes and increased their benzoxazinoid resistance through experimental evolution. Benzoxazinoid resistance evolved through multiple mechanisms, including a mutation in the aquaporin-like channel gene aqpZ. We reintroduced benzoxazinoid-resistant Photorhabdus strains into their original EPN hosts and identified one nematode–symbiont pair that was able to kill benzoxazinoid-sequestering WCR larvae more efficiently. Our results suggest that modification of bacterial symbionts might provide a generalizable strategy to improve biocontrol of agricultural pests. Nematode bacterial symbionts were evolved in vitro to resist insect-sequestered plant toxins to improve the insect-killing efficacy of their nematode host.},
  archive      = {J_NBT},
  author       = {Machado, Ricardo A. R. and Thönen, Lisa and Arce, Carla C. M. and Theepan, Vanitha and Prada, Fausto and Wüthrich, Daniel and Robert, Christelle A. M. and Vogiatzaki, Evangelia and Shi, Yi-Ming and Schaeren, Olivier P. and Notter, Matheus and Bruggmann, Rémy and Hapfelmeier, Siegfried and Bode, Helge B. and Erb, Matthias},
  doi          = {10.1038/s41587-020-0419-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {600-608},
  shortjournal = {Nature Biotech.},
  title        = {Engineering bacterial symbionts of nematodes improves their biocontrol potential to counter the western corn rootworm},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Multiplex digital spatial profiling of proteins and RNA in
fixed tissue. <em>NBT</em>, <em>38</em>(5), 586–599. (<a
href="https://doi.org/10.1038/s41587-020-0472-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Digital Spatial Profiling (DSP) is a method for highly multiplex spatial profiling of proteins or RNAs suitable for use on formalin-fixed, paraffin-embedded (FFPE) samples. The approach relies on (1) multiplexed readout of proteins or RNAs using oligonucleotide tags; (2) oligonucleotide tags attached to affinity reagents (antibodies or RNA probes) through a photocleavable (PC) linker; and (3) photocleaving light projected onto the tissue sample to release PC oligonucleotides in any spatial pattern across a region of interest (ROI) covering 1 to ~5,000 cells. DSP is capable of single-cell sensitivity within an ROI using the antibody readout, with RNA detection feasible down to ~600 individual mRNA transcripts. We show spatial profiling of up to 44 proteins and 96 genes (928 RNA probes) in lymphoid, colorectal tumor and autoimmune tissues by using the nCounter system and 1,412 genes (4,998 RNA probes) by using next-generation sequencing (NGS). DSP may be used to profile not only proteins and RNAs in biobanked samples but also immune markers in patient samples, with potential prognostic and predictive potential for clinical decision-making. A turnkey system allows for spatial profiling of proteins and RNA in fixed tissues, providing a window on cellular heterogeneity.},
  archive      = {J_NBT},
  author       = {Merritt, Christopher R. and Ong, Giang T. and Church, Sarah E. and Barker, Kristi and Danaher, Patrick and Geiss, Gary and Hoang, Margaret and Jung, Jaemyeong and Liang, Yan and McKay-Fleisch, Jill and Nguyen, Karen and Norgaard, Zach and Sorg, Kristina and Sprague, Isaac and Warren, Charles and Warren, Sarah and Webster, Philippa J. and Zhou, Zoey and Zollinger, Daniel R. and Dunaway, Dwayne L. and Mills, Gordon B. and Beechem, Joseph M.},
  doi          = {10.1038/s41587-020-0472-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {586-599},
  shortjournal = {Nature Biotech.},
  title        = {Multiplex digital spatial profiling of proteins and RNA in fixed tissue},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Prime genome editing in rice and wheat. <em>NBT</em>,
<em>38</em>(5), 582–585. (<a
href="https://doi.org/10.1038/s41587-020-0455-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editors, which are CRISPR–Cas9 nickase (H840A)–reverse transcriptase fusions programmed with prime editing guide RNAs (pegRNAs), can edit bases in mammalian cells without donor DNA or double-strand breaks. We adapted prime editors for use in plants through codon, promoter, and editing-condition optimization. The resulting suite of plant prime editors enable point mutations, insertions and deletions in rice and wheat protoplasts. Regenerated prime-edited rice plants were obtained at frequencies of up to 21.8\%. Prime editors are optimized for use in plants.},
  archive      = {J_NBT},
  author       = {Lin, Qiupeng and Zong, Yuan and Xue, Chenxiao and Wang, Shengxing and Jin, Shuai and Zhu, Zixu and Wang, Yanpeng and Anzalone, Andrew V. and Raguram, Aditya and Doman, Jordan L. and Liu, David R. and Gao, Caixia},
  doi          = {10.1038/s41587-020-0455-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {582-585},
  shortjournal = {Nature Biotech.},
  title        = {Prime genome editing in rice and wheat},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Superior field performance of waxy corn engineered using
CRISPR–Cas9. <em>NBT</em>, <em>38</em>(5), 579–581. (<a
href="https://doi.org/10.1038/s41587-020-0444-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created waxy corn hybrids by CRISPR–Cas9 editing of a waxy allele in 12 elite inbred maize lines, a process that was more than a year faster than conventional trait introgression using backcrossing and marker-assisted selection. Field trials at 25 locations showed that CRISPR-waxy hybrids were agronomically superior to introgressed hybrids, producing on average 5.5 bushels per acre higher yield. Gene-edited waxy corn lines have higher yields in field trials than hybrids produced by traditional trait introgression.},
  archive      = {J_NBT},
  author       = {Gao, Huirong and Gadlage, Mark J. and Lafitte, H. Renee and Lenderts, Brian and Yang, Meizhu and Schroder, Megan and Farrell, Jeffry and Snopek, Kay and Peterson, Dave and Feigenbutz, Lanie and Jones, Spencer and St Clair, Grace and Rahe, Melissa and Sanyour-Doyel, Nathalie and Peng, Chenna and Wang, Lijuan and Young, Joshua K. and Beatty, Mary and Dahlke, Brian and Hazebroek, Jan and Greene, Thomas W. and Cigan, A. Mark and Chilcoat, N. Doane and Meeley, R. Bob},
  doi          = {10.1038/s41587-020-0444-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {579-581},
  shortjournal = {Nature Biotech.},
  title        = {Superior field performance of waxy corn engineered using CRISPR–Cas9},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Reply to: Fitness effects of CRISPR/Cas9-targeting of long
noncoding RNA genes. <em>NBT</em>, <em>38</em>(5), 577–578. (<a
href="https://doi.org/10.1038/s41587-020-0431-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Liu, Ying and Liu, Zhiheng and Cao, Zhongzheng and Wei, Wensheng},
  doi          = {10.1038/s41587-020-0431-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {577-578},
  shortjournal = {Nature Biotech.},
  title        = {Reply to: Fitness effects of CRISPR/Cas9-targeting of long noncoding RNA genes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Fitness effects of CRISPR/Cas9-targeting of long noncoding
RNA genes. <em>NBT</em>, <em>38</em>(5), 573–576. (<a
href="https://doi.org/10.1038/s41587-020-0428-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Horlbeck, Max A. and Liu, S. John and Chang, Howard Y. and Lim, Daniel A. and Weissman, Jonathan S.},
  doi          = {10.1038/s41587-020-0428-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {573-576},
  shortjournal = {Nature Biotech.},
  title        = {Fitness effects of CRISPR/Cas9-targeting of long noncoding RNA genes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Viral contamination in biologic manufacture and implications
for emerging therapies. <em>NBT</em>, <em>38</em>(5), 563–572. (<a
href="https://doi.org/10.1038/s41587-020-0507-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recombinant protein therapeutics, vaccines, and plasma products have a long record of safety. However, the use of cell culture to produce recombinant proteins is still susceptible to contamination with viruses. These contaminations cost millions of dollars to recover from, can lead to patients not receiving therapies, and are very rare, which makes learning from past events difficult. A consortium of biotech companies, together with the Massachusetts Institute of Technology, has convened to collect data on these events. This industry-wide study provides insights into the most common viral contaminants, the source of those contaminants, the cell lines affected, corrective actions, as well as the impact of such events. These results have implications for the safe and effective production of not just current products, but also emerging cell and gene therapies which have shown much therapeutic promise. The Consortium on Adventitious Agent Contamination in Biomanufacturing (CAACB) provides a comprehensive and forward-looking overview of industry’s experience with viral contamination of cell cultures used to produce recombinant proteins.},
  archive      = {J_NBT},
  author       = {Barone, Paul W. and Wiebe, Michael E. and Leung, James C. and Hussein, Islam T. M. and Keumurian, Flora J. and Bouressa, James and Brussel, Audrey and Chen, Dayue and Chong, Ming and Dehghani, Houman and Gerentes, Lionel and Gilbert, James and Gold, Dan and Kiss, Robert and Kreil, Thomas R. and Labatut, René and Li, Yuling and Müllberg, Jürgen and Mallet, Laurent and Menzel, Christian and Moody, Mark and Monpoeho, Serge and Murphy, Marie and Plavsic, Mark and Roth, Nathan J. and Roush, David and Ruffing, Michael and Schicho, Richard and Snyder, Richard and Stark, Daniel and Zhang, Chun and Wolfrum, Jacqueline and Sinskey, Anthony J. and Springs, Stacy L.},
  doi          = {10.1038/s41587-020-0507-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {563-572},
  shortjournal = {Nature Biotech.},
  title        = {Viral contamination in biologic manufacture and implications for emerging therapies},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Diamonds in the doublets. <em>NBT</em>, <em>38</em>(5),
559–561. (<a href="https://doi.org/10.1038/s41587-020-0511-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interacting cells are characterized by sequencing FACS clusters.},
  archive      = {J_NBT},
  author       = {Bendall, Sean C.},
  doi          = {10.1038/s41587-020-0511-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {559-561},
  shortjournal = {Nature Biotech.},
  title        = {Diamonds in the doublets},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Epigenetic drugs. <em>NBT</em>, <em>38</em>(5), 558. (<a
href="https://doi.org/10.1038/s41587-020-0523-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents for pharmaceutical compounds comprising and treatments using histone deacetylase and DNA methyltransferase inhibitors.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0523-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {558},
  shortjournal = {Nature Biotech.},
  title        = {Epigenetic drugs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Adapting the ordre public and morality exclusion of european
patent law to accommodate emerging technologies. <em>NBT</em>,
<em>38</em>(5), 555–557. (<a
href="https://doi.org/10.1038/s41587-020-0504-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patent law’s existing public policy exclusion should be reinterpreted and a new method introduced for assessing the moral and public policy implications of commercializing emerging technologies.},
  archive      = {J_NBT},
  author       = {Pila, Justine},
  doi          = {10.1038/s41587-020-0504-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {555-557},
  shortjournal = {Nature Biotech.},
  title        = {Adapting the ordre public and morality exclusion of european patent law to accommodate emerging technologies},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Nature biotechnology’s academic spinouts of 2019.
<em>NBT</em>, <em>38</em>(5), 546–554. (<a
href="https://doi.org/10.1038/s41587-020-0506-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our annual survey highlights startups tackling intractable viruses with new vaccine design, engineering a reliable source of platelets, universalizing cell therapies, improving cancer screening, developing RNA-editing platforms and targeting protein–RNA interactions. Michael Eisenstein, Ken Garber, Caroline Seydel and Laura DeFrancesco report.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael and Garber, Ken and Seydel, Caroline and DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0506-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {546-554},
  shortjournal = {Nature Biotech.},
  title        = {Nature biotechnology’s academic spinouts of 2019},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Developing therapeutic monoclonal antibodies at pandemic
pace. <em>NBT</em>, <em>38</em>(5), 540–545. (<a
href="https://doi.org/10.1038/s41587-020-0512-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The time from discovery to proof-of-concept trials could be reduced to 5–6 months from a traditional timeline of 10–12 months.},
  archive      = {J_NBT},
  author       = {Kelley, Brian},
  doi          = {10.1038/s41587-020-0512-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {540-545},
  shortjournal = {Nature Biotech.},
  title        = {Developing therapeutic monoclonal antibodies at pandemic pace},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Access and benefit-sharing following the synthesis of
horsepox virus. <em>NBT</em>, <em>38</em>(5), 537–539. (<a
href="https://doi.org/10.1038/s41587-020-0518-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Rourke, Michelle F. and Phelan, Alexandra and Lawson, Charles},
  doi          = {10.1038/s41587-020-0518-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {537-539},
  shortjournal = {Nature Biotech.},
  title        = {Access and benefit-sharing following the synthesis of horsepox virus},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). In conversation with four startup CEOs. <em>NBT</em>,
<em>38</em>(5), 533–536. (<a
href="https://doi.org/10.1038/s41587-020-0498-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A cadre of biotech leaders discuss lessons learned from the bioentrepreneur experience.},
  archive      = {J_NBT},
  author       = {Huggett, Brady},
  doi          = {10.1038/s41587-020-0498-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {533-536},
  shortjournal = {Nature Biotech.},
  title        = {In conversation with four startup CEOs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). 1Q20 — biotech stocks nosedive. <em>NBT</em>,
<em>38</em>(5), 521–522. (<a
href="https://doi.org/10.1038/s41587-020-0516-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0516-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {521-522},
  shortjournal = {Nature Biotech.},
  title        = {1Q20 — biotech stocks nosedive},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Drug pipeline 1Q20. <em>NBT</em>, <em>38</em>(5), 519–520.
(<a href="https://doi.org/10.1038/s41587-020-0514-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0514-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {519-520},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 1Q20},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Around the world in a month. <em>NBT</em>, <em>38</em>(5),
518. (<a href="https://doi.org/10.1038/s41587-020-0524-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0524-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {518},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Drug-induced protein degradation heats up. <em>NBT</em>,
<em>38</em>(5), 514. (<a
href="https://doi.org/10.1038/s41587-020-0529-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0529-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {514},
  shortjournal = {Nature Biotech.},
  title        = {Drug-induced protein degradation heats up},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). First drug approved for neurofibromas is a MEK inhibitor.
<em>NBT</em>, <em>38</em>(5), 513. (<a
href="https://doi.org/10.1038/s41587-020-0530-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0530-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {513},
  shortjournal = {Nature Biotech.},
  title        = {First drug approved for neurofibromas is a MEK inhibitor},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Active machine learning helps drug hunters tackle biology.
<em>NBT</em>, <em>38</em>(5), 512–514. (<a
href="https://doi.org/10.1038/s41587-020-0521-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A growing cadre of startups is pursuing iterative cycles of machine learning, wet-lab experimentation and human feedback to accelerate target drug discovery.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-020-0521-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {512-514},
  shortjournal = {Nature Biotech.},
  title        = {Active machine learning helps drug hunters tackle biology},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Biggest COVID-19 trial tests repurposed drugs first.
<em>NBT</em>, <em>38</em>(5), 510. (<a
href="https://doi.org/10.1038/s41587-020-0528-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0528-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {510},
  shortjournal = {Nature Biotech.},
  title        = {Biggest COVID-19 trial tests repurposed drugs first},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Cancer-eating immune cells kitted out with CARs.
<em>NBT</em>, <em>38</em>(5), 509–511. (<a
href="https://doi.org/10.1038/s41587-020-0520-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immunotherapy startups look to engineer macrophages to fight solid tumors.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-0520-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {509-511},
  shortjournal = {Nature Biotech.},
  title        = {Cancer-eating immune cells kitted out with CARs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). All that’s fit to preprint. <em>NBT</em>, <em>38</em>(5),
507. (<a href="https://doi.org/10.1038/s41587-020-0536-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {COVID-19 has reinforced the importance of preprints as an indispensable means for rapid research dissemination.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0536-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {507},
  shortjournal = {Nature Biotech.},
  title        = {All that’s fit to preprint},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). The pandemic pipeline. <em>NBT</em>, <em>38</em>(5),
523–532. (<a href="https://doi.org/10.1038/d41587-020-00005-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are doing their best to accelerate experimental drugs and vaccines for COVID-19 through the pipeline. Each faces its own set of challenges, but all agree on the need for a radical rethink of the clinical development process for pandemics. Companies are doing their best to accelerate experimental drugs and vaccines for COVID-19 through the pipeline. Each faces its own set of challenges, but all agree on the need for a radical rethink of the clinical development process for pandemics.},
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/d41587-020-00005-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {5},
  pages        = {523-532},
  shortjournal = {Nature Biotech.},
  title        = {The pandemic pipeline},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Fast, portable tests come online to curb coronavirus
pandemic. <em>NBT</em>, <em>38</em>(5), 515–518. (<a
href="https://doi.org/10.1038/d41587-020-00010-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Testing kits delivered by courier and digital tools combine to battle the COVID-19 outbreak. Testing kits delivered by courier and digital tools combine to battle the COVID-19 outbreak.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-020-00010-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {5},
  pages        = {515-518},
  shortjournal = {Nature Biotech.},
  title        = {Fast, portable tests come online to curb coronavirus pandemic},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). People. <em>NBT</em>, <em>38</em>(4), 506. (<a
href="https://doi.org/10.1038/s41587-020-0489-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0489-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {506},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: MEMOTE for standardized genome-scale
metabolic model testing. <em>NBT</em>, <em>38</em>(4), 504. (<a
href="https://doi.org/10.1038/s41587-020-0477-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Lieven, Christian and Beber, Moritz E. and Olivier, Brett G. and Bergmann, Frank T. and Ataman, Meric and Babaei, Parizad and Bartell, Jennifer A. and Blank, Lars M. and Chauhan, Siddharth and Correia, Kevin and Diener, Christian and Dräger, Andreas and Ebert, Birgitta E. and Edirisinghe, Janaka N. and Faria, José P. and Feist, Adam M. and Fengos, Georgios and Fleming, Ronan M. T. and García-Jiménez, Beatriz and Hatzimanikatis, Vassily and van Helvoirt, Wout and Henry, Christopher S. and Hermjakob, Henning and Herrgård, Markus J. and Kaafarani, Ali and Kim, Hyun Uk and King, Zachary and Klamt, Steffen and Klipp, Edda and Koehorst, Jasper J. and König, Matthias and Lakshmanan, Meiyappan and Lee, Dong-Yup and Lee, Sang Yup and Lee, Sunjae and Lewis, Nathan E. and Liu, Filipe and Ma, Hongwu and Machado, Daniel and Mahadevan, Radhakrishnan and Maia, Paulo and Mardinoglu, Adil and Medlock, Gregory L. and Monk, Jonathan M. and Nielsen, Jens and Nielsen, Lars Keld and Nogales, Juan and Nookaew, Intawat and Palsson, Bernhard O. and Papin, Jason A. and Patil, Kiran R. and Poolman, Mark and Price, Nathan D. and Resendis-Antonio, Osbaldo and Richelle, Anne and Rocha, Isabel and Sánchez, Benjamín J. and Schaap, Peter J. and Sheriff, Rahuman S. Malik and Shoaie, Saeed and Sonnenschein, Nikolaus and Teusink, Bas and Vilaça, Paulo and Vik, Jon Olav and Wodke, Judith A. H. and Xavier, Joana C. and Yuan, Qianqian and Zakhartsev, Maksim and Zhang, Cheng},
  doi          = {10.1038/s41587-020-0477-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {504},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: MEMOTE for standardized genome-scale metabolic model testing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Author correction: A computationally designed chimeric
antigen receptor provides a small-molecule safety switch for t-cell
therapy. <em>NBT</em>, <em>38</em>(4), 503. (<a
href="https://doi.org/10.1038/s41587-020-0461-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Giordano-Attianese, Greta and Gainza, Pablo and Gray-Gaillard, Elise and Cribioli, Elisabetta and Shui, Sailan and Kim, Seonghoon and Kwak, Mi-Jeong and Vollers, Sabrina and Corria Osorio, Angel De Jesus and Reichenbach, Patrick and Bonet, Jaume and Oh, Byung-Ha and Irving, Melita and Coukos, George and Correia, Bruno E.},
  doi          = {10.1038/s41587-020-0461-z},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {503},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: A computationally designed chimeric antigen receptor provides a small-molecule safety switch for T-cell therapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Using GlycoDelete to produce proteins
lacking plant-specific n-glycan modification in seeds. <em>NBT</em>,
<em>38</em>(4), 503. (<a
href="https://doi.org/10.1038/s41587-020-0464-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Piron, Robin and Santens, Francis and De Paepe, Annelies and Depicker, Ann and Callewaert, Nico},
  doi          = {10.1038/s41587-020-0464-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {503},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Using GlycoDelete to produce proteins lacking plant-specific N-glycan modification in seeds},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Template plasmid integration in germline
genome-edited cattle. <em>NBT</em>, <em>38</em>(4), 503. (<a
href="https://doi.org/10.1038/s41587-020-0467-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Norris, Alexis L. and Lee, Stella S. and Greenlees, Kevin J. and Tadesse, Daniel A. and Miller, Mayumi F. and Lombardi, Heather A.},
  doi          = {10.1038/s41587-020-0467-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {503},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Template plasmid integration in germline genome-edited cattle},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reconstructing kinase network topologies from
phosphoproteomics data reveals cancer-associated rewiring. <em>NBT</em>,
<em>38</em>(4), 493–502. (<a
href="https://doi.org/10.1038/s41587-019-0391-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how oncogenic mutations rewire regulatory-protein networks is important for rationalizing the mechanisms of oncogenesis and for individualizing anticancer treatments. We report a chemical phosphoproteomics method to elucidate the topology of kinase-signaling networks in mammalian cells. We identified &amp;gt;6,000 protein phosphorylation sites that can be used to infer &amp;gt;1,500 kinase–kinase interactions and devised algorithms that can reconstruct kinase network topologies from these phosphoproteomics data. Application of our methods to primary acute myeloid leukemia and breast cancer tumors quantified the relationship between kinase expression and activity, and enabled the identification of hitherto unknown kinase network topologies associated with drug-resistant phenotypes or specific genetic mutations. Using orthogonal methods we validated that PIK3CA wild-type cells adopt MAPK-dependent circuitries in breast cancer cells and that the kinase TTK is important in acute myeloid leukemia. Our phosphoproteomic signatures of network circuitry can identify kinase topologies associated with both phenotypes and genotypes of cancer cells. Improved methods to reconstruct global kinase-signaling networks from phosphoproteomics data are applied in cancer cells.},
  archive      = {J_NBT},
  author       = {Hijazi, Maruan and Smith, Ryan and Rajeeve, Vinothini and Bessant, Conrad and Cutillas, Pedro R.},
  doi          = {10.1038/s41587-019-0391-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {493-502},
  shortjournal = {Nature Biotech.},
  title        = {Reconstructing kinase network topologies from phosphoproteomics data reveals cancer-associated rewiring},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Efficient production of male wolbachia-infected aedes
aegypti mosquitoes enables large-scale suppression of wild populations.
<em>NBT</em>, <em>38</em>(4), 482–492. (<a
href="https://doi.org/10.1038/s41587-020-0471-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The range of the mosquito Aedes aegypti continues to expand, putting more than two billion people at risk of arboviral infection. The sterile insect technique (SIT) has been used to successfully combat agricultural pests at large scale, but not mosquitoes, mainly because of challenges with consistent production and distribution of high-quality male mosquitoes. We describe automated processes to rear and release millions of competitive, sterile male Wolbachia-infected mosquitoes, and use of these males in a large-scale suppression trial in Fresno County, California. In 2018, we released 14.4 million males across three replicate neighborhoods encompassing 293 hectares. At peak mosquito season, the number of female mosquitoes was 95.5\% lower (95\% CI, 93.6–96.9) in release areas compared to non-release areas, with the most geographically isolated neighborhood reaching a 99\% reduction. This work demonstrates the high efficacy of mosquito SIT in an area ninefold larger than in previous similar trials, supporting the potential of this approach in public health and nuisance-mosquito eradication programs. Mosquitoes are nearly eradicated in three suburbs of California using accurately sorted sterile male mosquitoes.},
  archive      = {J_NBT},
  author       = {Crawford, Jacob E. and Clarke, David W. and Criswell, Victor and Desnoyer, Mark and Cornel, Devon and Deegan, Brittany and Gong, Kyle and Hopkins, Kaycie C. and Howell, Paul and Hyde, Justin S. and Livni, Josh and Behling, Charlie and Benza, Renzo and Chen, Willa and Dobson, Karen L. and Eldershaw, Craig and Greeley, Daniel and Han, Yi and Hughes, Bridgette and Kakani, Evdoxia and Karbowski, Joe and Kitchell, Angus and Lee, Erika and Lin, Teresa and Liu, Jianyi and Lozano, Martin and MacDonald, Warren and Mains, James W. and Metlitz, Matty and Mitchell, Sara N. and Moore, David and Ohm, Johanna R. and Parkes, Kathleen and Porshnikoff, Alexandra and Robuck, Chris and Sheridan, Martin and Sobecki, Robert and Smith, Peter and Stevenson, Jessica and Sullivan, Jordan and Wasson, Brian and Weakley, Allison M. and Wilhelm, Mark and Won, Joshua and Yasunaga, Ari and Chan, William C. and Holeman, Jodi and Snoad, Nigel and Upson, Linus and Zha, Tiantian and Dobson, Stephen L. and Mulligan, F. Steven and Massaro, Peter and White, Bradley J.},
  doi          = {10.1038/s41587-020-0471-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {482-492},
  shortjournal = {Nature Biotech.},
  title        = {Efficient production of male wolbachia-infected aedes aegypti mosquitoes enables large-scale suppression of wild populations},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Continuous evolution of SpCas9 variants compatible with
non-g PAMs. <em>NBT</em>, <em>38</em>(4), 471–481. (<a
href="https://doi.org/10.1038/s41587-020-0412-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The targeting scope of Streptococcus pyogenes Cas9 (SpCas9) and its engineered variants is largely restricted to protospacer-adjacent motif (PAM) sequences containing G bases. Here we report the evolution of three new SpCas9 variants that collectively recognize NRNH PAMs (where R is A or G and H is A, C or T) using phage-assisted non-continuous evolution, three new phage-assisted continuous evolution strategies for DNA binding and a secondary selection for DNA cleavage. The targeting capabilities of these evolved variants and SpCas9-NG were characterized in HEK293T cells using a library of 11,776 genomically integrated protospacer–sgRNA pairs containing all possible NNNN PAMs. The evolved variants mediated indel formation and base editing in human cells and enabled A•T-to-G•C base editing of a sickle cell anemia mutation using a previously inaccessible CACC PAM. These new evolved SpCas9 variants, together with previously reported variants, in principle enable targeting of most NR PAM sequences and substantially reduce the fraction of genomic sites that are inaccessible by Cas9-based methods. PAM sequences without G bases can be edited with SpCas9 variants that were continuously evolved in the laboratory.},
  archive      = {J_NBT},
  author       = {Miller, Shannon M. and Wang, Tina and Randolph, Peyton B. and Arbab, Mandana and Shen, Max W. and Huang, Tony P. and Matuszek, Zaneta and Newby, Gregory A. and Rees, Holly A. and Liu, David R.},
  doi          = {10.1038/s41587-020-0412-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {471-481},
  shortjournal = {Nature Biotech.},
  title        = {Continuous evolution of SpCas9 variants compatible with non-G PAMs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Targeting the cytoskeleton to direct pancreatic
differentiation of human pluripotent stem cells. <em>NBT</em>,
<em>38</em>(4), 460–470. (<a
href="https://doi.org/10.1038/s41587-020-0430-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generation of pancreatic β cells from human pluripotent stem cells (hPSCs) holds promise as a cell replacement therapy for diabetes. In this study, we establish a link between the state of the actin cytoskeleton and the expression of pancreatic transcription factors that drive pancreatic lineage specification. Bulk and single-cell RNA sequencing demonstrated that different degrees of actin polymerization biased cells toward various endodermal lineages and that conditions favoring a polymerized cytoskeleton strongly inhibited neurogenin 3-induced endocrine differentiation. Using latrunculin A to depolymerize the cytoskeleton during endocrine induction, we developed a two-dimensional differentiation protocol for generating human pluripotent stem-cell-derived β (SC-β) cells with improved in vitro and in vivo function. SC-β cells differentiated from four hPSC lines exhibited first- and second-phase dynamic glucose-stimulated insulin secretion. Transplantation of islet-sized aggregates of these cells rapidly reversed severe preexisting diabetes in mice at a rate close to that of human islets and maintained normoglycemia for at least 9 months. Generation of pancreatic β cells from stem cells is enhanced by manipulating the cytoskeleton.},
  archive      = {J_NBT},
  author       = {Hogrebe, Nathaniel J. and Augsornworawat, Punn and Maxwell, Kristina G. and Velazco-Cruz, Leonardo and Millman, Jeffrey R.},
  doi          = {10.1038/s41587-020-0430-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {460-470},
  shortjournal = {Nature Biotech.},
  title        = {Targeting the cytoskeleton to direct pancreatic differentiation of human pluripotent stem cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Interleukin-23 engineering improves CAR t cell function in
solid tumors. <em>NBT</em>, <em>38</em>(4), 448–459. (<a
href="https://doi.org/10.1038/s41587-019-0398-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytokines that stimulate T cell proliferation, such as interleukin (IL)-15, have been explored as a means of boosting the antitumor activity of chimeric antigen receptor (CAR) T cells. However, constitutive cytokine signaling in T cells and activation of bystander cells may cause toxicity. IL-23 is a two-subunit cytokine known to promote proliferation of memory T cells and T helper type 17 cells. We found that, upon T cell antigen receptor (TCR) stimulation, T cells upregulated the IL-23 receptor and the IL-23α p19 subunit, but not the p40 subunit. We engineered expression of the p40 subunit in T cells (p40-Td cells) and obtained selective proliferative activity in activated T cells via autocrine IL-23 signaling. In comparison to CAR T cells, p40-Td CAR T cells showed improved antitumor capacity in vitro, with increased granzyme B and decreased PD-1 expression. In two xenograft and two syngeneic solid tumor mouse models, p40-Td CAR T cells showed superior efficacy in comparison to CAR T cells and attenuated side effects in comparison to CAR T cells expressing IL-18 or IL-15. The efficacy of chimeric antigen receptor (CAR) T cells in solid tumor models is enhanced by interleukin-23 engineering.},
  archive      = {J_NBT},
  author       = {Ma, Xingcong and Shou, Peishun and Smith, Christof and Chen, Yuhui and Du, Hongwei and Sun, Chuang and Porterfield Kren, Nancy and Michaud, Daniel and Ahn, Sarah and Vincent, Benjamin and Savoldo, Barbara and Pylayeva-Gupta, Yuliya and Zhang, Shuqun and Dotti, Gianpietro and Xu, Yang},
  doi          = {10.1038/s41587-019-0398-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {448-459},
  shortjournal = {Nature Biotech.},
  title        = {Interleukin-23 engineering improves CAR t cell function in solid tumors},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An electroencephalographic signature predicts antidepressant
response in major depression. <em>NBT</em>, <em>38</em>(4), 439–447. (<a
href="https://doi.org/10.1038/s41587-019-0397-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antidepressants are widely prescribed, but their efficacy relative to placebo is modest, in part because the clinical diagnosis of major depression encompasses biologically heterogeneous conditions. Here, we sought to identify a neurobiological signature of response to antidepressant treatment as compared to placebo. We designed a latent-space machine-learning algorithm tailored for resting-state electroencephalography (EEG) and applied it to data from the largest imaging-coupled, placebo-controlled antidepressant study (n = 309). Symptom improvement was robustly predicted in a manner both specific for the antidepressant sertraline (versus placebo) and generalizable across different study sites and EEG equipment. This sertraline-predictive EEG signature generalized to two depression samples, wherein it reflected general antidepressant medication responsivity and related differentially to a repetitive transcranial magnetic stimulation treatment outcome. Furthermore, we found that the sertraline resting-state EEG signature indexed prefrontal neural responsivity, as measured by concurrent transcranial magnetic stimulation and EEG. Our findings advance the neurobiological understanding of antidepressant treatment through an EEG-tailored computational model and provide a clinical avenue for personalized treatment of depression. The efficacy of an antidepressant is predicted from an EEG signature.},
  archive      = {J_NBT},
  author       = {Wu, Wei and Zhang, Yu and Jiang, Jing and Lucas, Molly V. and Fonzo, Gregory A. and Rolle, Camarin E. and Cooper, Crystal and Chin-Fatt, Cherise and Krepel, Noralie and Cornelssen, Carena A. and Wright, Rachael and Toll, Russell T. and Trivedi, Hersh M. and Monuszko, Karen and Caudle, Trevor L. and Sarhadi, Kamron and Jha, Manish K. and Trombello, Joseph M. and Deckersbach, Thilo and Adams, Phil and McGrath, Patrick J. and Weissman, Myrna M. and Fava, Maurizio and Pizzagalli, Diego A. and Arns, Martijn and Trivedi, Madhukar H. and Etkin, Amit},
  doi          = {10.1038/s41587-019-0397-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {439-447},
  shortjournal = {Nature Biotech.},
  title        = {An electroencephalographic signature predicts antidepressant response in major depression},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Targeted nanopore sequencing with cas9-guided adapter
ligation. <em>NBT</em>, <em>38</em>(4), 433–438. (<a
href="https://doi.org/10.1038/s41587-020-0407-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent improvements in sequencing methods, there remains a need for assays that provide high sequencing depth and comprehensive variant detection. Current methods1–4 are limited by the loss of native modifications, short read length, high input requirements, low yield or long protocols. In the present study, we describe nanopore Cas9-targeted sequencing (nCATS), an enrichment strategy that uses targeted cleavage of chromosomal DNA with Cas9 to ligate adapters for nanopore sequencing. We show that nCATS can simultaneously assess haplotype-resolved single-nucleotide variants, structural variations and CpG methylation. We apply nCATS to four cell lines, to a cell-line-derived xenograft, and to normal and paired tumor/normal primary human breast tissue. Median sequencing coverage was 675× using a MinION flow cell and 34× using the smaller Flongle flow cell. The nCATS sequencing requires only ~3 μg of genomic DNA and can target a large number of loci in a single reaction. The method will facilitate the use of long-read sequencing in research and in the clinic. Point mutations, structural variants and DNA methylation at target loci are assessed by nanopore sequencing.},
  archive      = {J_NBT},
  author       = {Gilpatrick, Timothy and Lee, Isac and Graham, James E. and Raimondeau, Etienne and Bowen, Rebecca and Heron, Andrew and Downs, Bradley and Sukumar, Saraswati and Sedlazeck, Fritz J and Timp, Winston},
  doi          = {10.1038/s41587-020-0407-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {433-438},
  shortjournal = {Nature Biotech.},
  title        = {Targeted nanopore sequencing with cas9-guided adapter ligation},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). A computationally designed chimeric antigen receptor
provides a small-molecule safety switch for t-cell therapy.
<em>NBT</em>, <em>38</em>(4), 426–432. (<a
href="https://doi.org/10.1038/s41587-019-0403-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Approaches to increase the activity of chimeric antigen receptor (CAR)-T cells against solid tumors may also increase the risk of toxicity and other side effects. To improve the safety of CAR-T-cell therapy, we computationally designed a chemically disruptable heterodimer (CDH) based on the binding of two human proteins. The CDH self-assembles, can be disrupted by a small-molecule drug and has a high-affinity protein interface with minimal amino acid deviation from wild-type human proteins. We incorporated the CDH into a synthetic heterodimeric CAR, called STOP-CAR, that has an antigen-recognition chain and a CD3ζ- and CD28-containing endodomain signaling chain. We tested STOP-CAR-T cells specific for two antigens in vitro and in vivo and found similar antitumor activity compared to second-generation (2G) CAR-T cells. Timed administration of the small-molecule drug dynamically inactivated the activity of STOP-CAR-T cells. Our work highlights the potential for structure-based design to add controllable elements to synthetic cellular therapies. The activity of CAR-T cells is reversibly halted with a small-molecule drug.},
  archive      = {J_NBT},
  author       = {Giordano-Attianese, Greta and Gainza, Pablo and Gray-Gaillard, Elise and Cribioli, Elisabetta and Shui, Sailan and Kim, Seonghoon and Kwak, Mi-Jeong and Vollers, Sabrina and Corria Osorio, Angel De Jesus and Reichenbach, Patrick and Bonet, Jaume and Oh, Byung-Ha and Irving, Melita and Coukos, George and Correia, Bruno E.},
  doi          = {10.1038/s41587-019-0403-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {426-432},
  shortjournal = {Nature Biotech.},
  title        = {A computationally designed chimeric antigen receptor provides a small-molecule safety switch for T-cell therapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Antibody-mediated delivery of viral epitopes to tumors
harnesses CMV-specific t cells for cancer therapy. <em>NBT</em>,
<em>38</em>(4), 420–425. (<a
href="https://doi.org/10.1038/s41587-019-0404-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several cancer immunotherapy approaches, such as immune checkpoint blockade and adoptive T-cell therapy, boost T-cell activity against the tumor, but these strategies are not effective in the absence of T cells specific for displayed tumor antigens. Here we outline an immunotherapy in which endogenous T cells specific for a noncancer antigen are retargeted to attack tumors. The approach relies on the use of antibody–peptide epitope conjugates (APECs) to deliver suitable antigens to the tumor surface for presention by HLA-I. To retarget cytomegalovirus (CMV)-specific CD8+ T cells against tumors, we used APECs containing CMV-derived epitopes conjugated to tumor-targeting antibodies via metalloprotease-sensitive linkers. These APECs redirect pre-existing CMV immunity against tumor cells in vitro and in mouse cancer models. In vitro, APECs activated specifically CMV-reactive effector T cells whereas a bispecific T-cell engager activated both effector and regulatory T cells. Our approach may provide an effective alternative in cancers that are not amenable to checkpoint inhibitors or other immunotherapies. Anti-tumor activity of cytomegalovirus (CMV)-specific CD8 T cells is achieved by antibody-mediated delivery of CMV epitopes.},
  archive      = {J_NBT},
  author       = {Millar, David G. and Ramjiawan, Rakesh R. and Kawaguchi, Kosuke and Gupta, Nisha and Chen, Jiang and Zhang, Songfa and Nojiri, Takashi and Ho, William W. and Aoki, Shuichi and Jung, Keehoon and Chen, Ivy and Shi, Feng and Heather, James M. and Shigeta, Kohei and Morton, Laura T. and Sepulveda, Sean and Wan, Li and Joseph, Ricky and Minogue, Eleanor and Khatri, Ashok and Bardia, Aditya and Ellisen, Leif W. and Corcoran, Ryan B. and Hata, Aaron N. and Pai, Sara I. and Jain, Rakesh K. and Fukumura, Dai and Duda, Dan G. and Cobbold, Mark},
  doi          = {10.1038/s41587-019-0404-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {420-425},
  shortjournal = {Nature Biotech.},
  title        = {Antibody-mediated delivery of viral epitopes to tumors harnesses CMV-specific t cells for cancer therapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Forum: Repurposing viral immunity to fight cancer.
<em>NBT</em>, <em>38</em>(4), 419. (<a
href="https://doi.org/10.1038/s41587-020-0474-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0474-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {419},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Repurposing viral immunity to fight cancer},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Predicting antidepressant response by
electroencephalography. <em>NBT</em>, <em>38</em>(4), 417–419. (<a
href="https://doi.org/10.1038/s41587-020-0476-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patients who respond to an antidepressant are identified by retrospective analysis of their brain waves.},
  archive      = {J_NBT},
  author       = {Michel, Christoph M. and Pascual-Leone, Alvaro},
  doi          = {10.1038/s41587-020-0476-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {417-419},
  shortjournal = {Nature Biotech.},
  title        = {Predicting antidepressant response by electroencephalography},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Antimicrobials. <em>NBT</em>, <em>38</em>(4), 416. (<a
href="https://doi.org/10.1038/s41587-020-0488-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to new antimicrobial materials and methods of imparting antimicrobial activity to an article or product.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0488-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {416},
  shortjournal = {Nature Biotech.},
  title        = {Antimicrobials},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Europe’s biotech renaissance. <em>NBT</em>, <em>38</em>(4),
408–415. (<a href="https://doi.org/10.1038/s41587-020-0483-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {European biotech companies continued to raise unprecedented amounts of capital last year. Can they survive what are likely to be leaner times ahead?},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-020-0483-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {408-415},
  shortjournal = {Nature Biotech.},
  title        = {Europe’s biotech renaissance},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The case for biotech on mars. <em>NBT</em>, <em>38</em>(4),
401–407. (<a href="https://doi.org/10.1038/s41587-020-0485-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The stepwise application of biotechnology will be instrumental to addressing four key challenges of Martian settlement.},
  archive      = {J_NBT},
  author       = {Nangle, Shannon N. and Wolfson, Mikhail Y. and Hartsough, Lucas and Ma, Natalie J. and Mason, Christopher E. and Merighi, Massimo and Nathan, Vinitra and Silver, Pamela A. and Simon, Mark and Swett, Jacob and Thompson, David B. and Ziesack, Marika},
  doi          = {10.1038/s41587-020-0485-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {401-407},
  shortjournal = {Nature Biotech.},
  title        = {The case for biotech on mars},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). What would responsible remedial human germline editing look
like? <em>NBT</em>, <em>38</em>(4), 398–400. (<a
href="https://doi.org/10.1038/s41587-020-0482-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Adashi, Eli Y. and Cohen, I. Glenn},
  doi          = {10.1038/s41587-020-0482-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {398-400},
  shortjournal = {Nature Biotech.},
  title        = {What would responsible remedial human germline editing look like?},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Γδ t cells bring unconventional cancer-targeting to the
clinic — again. <em>NBT</em>, <em>38</em>(4), 389–391. (<a
href="https://doi.org/10.1038/s41587-020-0487-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A growing number of companies seek to unleash these unusual T cells against cancer.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-020-0487-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {389-391},
  shortjournal = {Nature Biotech.},
  title        = {γδ t cells bring unconventional cancer-targeting to the clinic — again},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Forty seven to gilead: “Eat me.” <em>NBT</em>,
<em>38</em>(4), 389. (<a
href="https://doi.org/10.1038/s41587-020-0496-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0496-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {389},
  shortjournal = {Nature Biotech.},
  title        = {Forty seven to gilead: “Eat me”},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020i). Around the world in a month. <em>NBT</em>, <em>38</em>(4),
388. (<a href="https://doi.org/10.1038/s41587-020-0486-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0486-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {388},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). IGF-1R drugs travel from cancer cradle to graves’.
<em>NBT</em>, <em>38</em>(4), 385–388. (<a
href="https://doi.org/10.1038/s41587-020-0481-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One of the most intensively investigated molecular targets in oncology proves its therapeutic worth for thyroid eye disease.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-0481-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {385-388},
  shortjournal = {Nature Biotech.},
  title        = {IGF-1R drugs travel from cancer cradle to graves’},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Second RNAi drug approved. <em>NBT</em>, <em>38</em>(4),
385. (<a href="https://doi.org/10.1038/s41587-020-0494-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0494-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {385},
  shortjournal = {Nature Biotech.},
  title        = {Second RNAi drug approved},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). First CRISPR therapy dosed. <em>NBT</em>, <em>38</em>(4),
382. (<a href="https://doi.org/10.1038/s41587-020-0493-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0493-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {382},
  shortjournal = {Nature Biotech.},
  title        = {First CRISPR therapy dosed},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Open for outbreaks. <em>NBT</em>, <em>38</em>(4), 377. (<a
href="https://doi.org/10.1038/s41587-020-0499-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the onslaught of the COVID-19 pandemic, the open sharing of materials, information and data is gaining increasing importance in biotech.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0499-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {377},
  shortjournal = {Nature Biotech.},
  title        = {Open for outbreaks},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Out of your skin. <em>NBT</em>, <em>38</em>(4), 392–397.
(<a href="https://doi.org/10.1038/s41587-020-0473-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long neglected compared with the gut microbiome, skin microbes are beginning to yield their secrets in health and disease, spurring a new cadre of biotech startups.},
  archive      = {J_NBT},
  author       = {Schmidt, Charlie},
  doi          = {10.1038/s41587-020-0473-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {4},
  pages        = {392-397},
  shortjournal = {Nature Biotech.},
  title        = {Out of your skin},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Coronavirus and the race to distribute reliable
diagnostics. <em>NBT</em>, <em>38</em>(4), 382–384. (<a
href="https://doi.org/10.1038/d41587-020-00002-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {International teams worked at speed to make tests for the virus available in record time. International teams worked at speed to make tests for the virus available in record time.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-020-00002-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {4},
  pages        = {382-384},
  shortjournal = {Nature Biotech.},
  title        = {Coronavirus and the race to distribute reliable diagnostics},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Coronavirus puts drug repurposing on the fast track.
<em>NBT</em>, <em>38</em>(4), 379–381. (<a
href="https://doi.org/10.1038/d41587-020-00003-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing antivirals and knowledge gained from the SARS and MERS outbreaks gain traction as the fastest route to fight the current coronavirus epidemic. Existing antivirals and knowledge gained from the SARS and MERS outbreaks gain traction as the fastest route to fight the current coronavirus epidemic.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/d41587-020-00003-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {4},
  pages        = {379-381},
  shortjournal = {Nature Biotech.},
  title        = {Coronavirus puts drug repurposing on the fast track},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). People. <em>NBT</em>, <em>38</em>(3), 376. (<a
href="https://doi.org/10.1038/s41587-020-0450-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0450-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {376},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Retraction note: Transient cytokine treatment induces acinar
cell reprogramming and regenerates functional beta cell mass in diabetic
mice. <em>NBT</em>, <em>38</em>(3), 374. (<a
href="https://doi.org/10.1038/s41587-020-0426-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This article has been retracted; see accompanying Retraction Note, which can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Baeyens, Luc and Lemper, Marie and Leuckx, Gunter and De Groef, Sofie and Bonfanti, Paola and Stangé, Geert and Shemer, Ruth and Nord, Christoffer and Scheel, David W. and Pan, Fong C. and Ahlgren, Ulf and Gu, Guoqiang and Stoffers, Doris A. and Dor, Yuval and Ferrer, Jorge and Gradwohl, Gerard and Wright, Christopher V. E. and Van de Casteele, Mark and German, Michael S. and Bouwens, Luc and Heimberg, Harry},
  doi          = {10.1038/s41587-020-0426-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {374},
  shortjournal = {Nature Biotech.},
  title        = {Retraction note: Transient cytokine treatment induces acinar cell reprogramming and regenerates functional beta cell mass in diabetic mice},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Full-length mRNA-seq from single-cell
levels of RNA and individual circulating tumor cells. <em>NBT</em>,
<em>38</em>(3), 374. (<a
href="https://doi.org/10.1038/s41587-020-0427-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Ramsköld, Daniel and Luo, Shujun and Wang, Yu-Chieh and Li, Robin and Deng, Qiaolin and Faridani, Omid R. and Daniels, Gregory A. and Khrebtukova, Irina and Loring, Jeanne F. and Laurent, Louise C. and Schroth, Gary P. and Sandberg, Rickard},
  doi          = {10.1038/s41587-020-0427-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {374},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Full-length mRNA-seq from single-cell levels of RNA and individual circulating tumor cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Publisher correction: OpenSWATH enables automated, targeted
analysis of data-independent acquisition MS data. <em>NBT</em>,
<em>38</em>(3), 374. (<a
href="https://doi.org/10.1038/s41587-020-0457-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Röst, Hannes L. and Rosenberger, George and Navarro, Pedro and Gillet, Ludovic and Miladinović, Saša M. and Schubert, Olga T. and Wolski, Witold and Collins, Ben C. and Malmström, Johan and Malmström, Lars and Aebersold, Ruedi},
  doi          = {10.1038/s41587-020-0457-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {374},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: OpenSWATH enables automated, targeted analysis of data-independent acquisition MS data},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The functional landscape of the human phosphoproteome.
<em>NBT</em>, <em>38</em>(3), 365–373. (<a
href="https://doi.org/10.1038/s41587-019-0344-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein phosphorylation is a key post-translational modification regulating protein function in almost all cellular processes. Although tens of thousands of phosphorylation sites have been identified in human cells, approaches to determine the functional importance of each phosphosite are lacking. Here, we manually curated 112 datasets of phospho-enriched proteins, generated from 104 different human cell types or tissues. We re-analyzed the 6,801 proteomics experiments that passed our quality control criteria, creating a reference phosphoproteome containing 119,809 human phosphosites. To prioritize functional sites, we used machine learning to identify 59 features indicative of proteomic, structural, regulatory or evolutionary relevance and integrate them into a single functional score. Our approach identifies regulatory phosphosites across different molecular mechanisms, processes and diseases, and reveals genetic susceptibilities at a genomic scale. Several regulatory phosphosites were experimentally validated, including identifying a role in neuronal differentiation for phosphosites in SMARCC2, a member of the SWI/SNF chromatin-remodeling complex. Phosphorylation sites are ranked for functional relevance using a comprehensive, high-quality human phosphoproteome.},
  archive      = {J_NBT},
  author       = {Ochoa, David and Jarnuczak, Andrew F. and Viéitez, Cristina and Gehre, Maja and Soucheray, Margaret and Mateus, André and Kleefeldt, Askar A. and Hill, Anthony and Garcia-Alonso, Luz and Stein, Frank and Krogan, Nevan J. and Savitski, Mikhail M. and Swaney, Danielle L. and Vizcaíno, Juan A. and Noh, Kyung-Min and Beltrao, Pedro},
  doi          = {10.1038/s41587-019-0344-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {365-373},
  shortjournal = {Nature Biotech.},
  title        = {The functional landscape of the human phosphoproteome},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Titrating gene expression using libraries of systematically
attenuated CRISPR guide RNAs. <em>NBT</em>, <em>38</em>(3), 355–364. (<a
href="https://doi.org/10.1038/s41587-019-0387-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A lack of tools to precisely control gene expression has limited our ability to evaluate relationships between expression levels and phenotypes. Here, we describe an approach to titrate expression of human genes using CRISPR interference and series of single-guide RNAs (sgRNAs) with systematically modulated activities. We used large-scale measurements across multiple cell models to characterize activities of sgRNAs containing mismatches to their target sites and derived rules governing mismatched sgRNA activity using deep learning. These rules enabled us to synthesize a compact sgRNA library to titrate expression of ~2,400 genes essential for robust cell growth and to construct an in silico sgRNA library spanning the human genome. Staging cells along a continuum of gene expression levels combined with single-cell RNA-seq readout revealed sharp transitions in cellular behaviors at gene-specific expression thresholds. Our work provides a general tool to control gene expression, with applications ranging from tuning biochemical pathways to identifying suppressors for diseases of dysregulated gene expression. Gene expression is precisely modulated with CRISPR interference and mismatched guide RNAs.},
  archive      = {J_NBT},
  author       = {Jost, Marco and Santos, Daniel A. and Saunders, Reuben A. and Horlbeck, Max A. and Hawkins, John S. and Scaria, Sonia M. and Norman, Thomas M. and Hussmann, Jeffrey A. and Liem, Christina R. and Gross, Carol A. and Weissman, Jonathan S.},
  doi          = {10.1038/s41587-019-0387-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {355-364},
  shortjournal = {Nature Biotech.},
  title        = {Titrating gene expression using libraries of systematically attenuated CRISPR guide RNAs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Single-cell analysis of structural variations and complex
rearrangements with tri-channel processing. <em>NBT</em>,
<em>38</em>(3), 343–354. (<a
href="https://doi.org/10.1038/s41587-019-0366-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural variation (SV), involving deletions, duplications, inversions and translocations of DNA segments, is a major source of genetic variability in somatic cells and can dysregulate cancer-related pathways. However, discovering somatic SVs in single cells has been challenging, with copy-number-neutral and complex variants typically escaping detection. Here we describe single-cell tri-channel processing (scTRIP), a computational framework that integrates read depth, template strand and haplotype phase to comprehensively discover SVs in individual cells. We surveyed SV landscapes of 565 single cells, including transformed epithelial cells and patient-derived leukemic samples, to discover abundant SV classes, including inversions, translocations and complex DNA rearrangements. Analysis of the leukemic samples revealed four times more somatic SVs than cytogenetic karyotyping, submicroscopic copy-number alterations, oncogenic copy-neutral rearrangements and a subclonal chromothripsis event. Advancing current methods, single-cell tri-channel processing can directly measure SV mutational processes in individual cells, such as breakage–fusion–bridge cycles, facilitating studies of clonal evolution, genetic mosaicism and SV formation mechanisms, which could improve disease classification for precision medicine. Complex structural variations in single cells are detected by integrating read depth, template strand and haplotype phase information.},
  archive      = {J_NBT},
  author       = {Sanders, Ashley D. and Meiers, Sascha and Ghareghani, Maryam and Porubsky, David and Jeong, Hyobin and van Vliet, M. Alexandra C. C. and Rausch, Tobias and Richter-Pechańska, Paulina and Kunz, Joachim B. and Jenni, Silvia and Bolognini, Davide and Longo, Gabriel M. C. and Raeder, Benjamin and Kinanen, Venla and Zimmermann, Jürgen and Benes, Vladimir and Schrappe, Martin and Mardin, Balca R. and Kulozik, Andreas E. and Bornhauser, Beat and Bourquin, Jean-Pierre and Marschall, Tobias and Korbel, Jan O.},
  doi          = {10.1038/s41587-019-0366-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {343-354},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell analysis of structural variations and complex rearrangements with tri-channel processing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Integrating microarray-based spatial transcriptomics and
single-cell RNA-seq reveals tissue architecture in pancreatic ductal
adenocarcinomas. <em>NBT</em>, <em>38</em>(3), 333–342. (<a
href="https://doi.org/10.1038/s41587-019-0392-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing (scRNA-seq) enables the systematic identification of cell populations in a tissue, but characterizing their spatial organization remains challenging. We combine a microarray-based spatial transcriptomics method that reveals spatial patterns of gene expression using an array of spots, each capturing the transcriptomes of multiple adjacent cells, with scRNA-Seq generated from the same sample. To annotate the precise cellular composition of distinct tissue regions, we introduce a method for multimodal intersection analysis. Applying multimodal intersection analysis to primary pancreatic tumors, we find that subpopulations of ductal cells, macrophages, dendritic cells and cancer cells have spatially restricted enrichments, as well as distinct coenrichments with other cell types. Furthermore, we identify colocalization of inflammatory fibroblasts and cancer cells expressing a stress-response gene module. Our approach for mapping the architecture of scRNA-seq-defined subpopulations can be applied to reveal the interactions inherent to complex tissues. Combining single-cell RNA-seq data and microarray-based spatial transcriptomics maps the location of different cell types and cell states in pancreatic tumors.},
  archive      = {J_NBT},
  author       = {Moncada, Reuben and Barkley, Dalia and Wagner, Florian and Chiodin, Marta and Devlin, Joseph C. and Baron, Maayan and Hajdu, Cristina H. and Simeone, Diane M. and Yanai, Itai},
  doi          = {10.1038/s41587-019-0392-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {333-342},
  shortjournal = {Nature Biotech.},
  title        = {Integrating microarray-based spatial transcriptomics and single-cell RNA-seq reveals tissue architecture in pancreatic ductal adenocarcinomas},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Peptide–TLR-7/8a conjugate vaccines chemically programmed
for nanoparticle self-assembly enhance CD8 t-cell immunity to tumor
antigens. <em>NBT</em>, <em>38</em>(3), 320–332. (<a
href="https://doi.org/10.1038/s41587-019-0390-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Personalized cancer vaccines targeting patient-specific neoantigens are a promising cancer treatment modality; however, neoantigen physicochemical variability can present challenges to manufacturing personalized cancer vaccines in an optimal format for inducing anticancer T cells. Here, we developed a vaccine platform (SNP-7/8a) based on charge-modified peptide–TLR-7/8a conjugates that are chemically programmed to self-assemble into nanoparticles of uniform size (~20 nm) irrespective of the peptide antigen composition. This approach provided precise loading of diverse peptide neoantigens linked to TLR-7/8a (adjuvant) in nanoparticles, which increased uptake by and activation of antigen-presenting cells that promote T-cell immunity. Vaccination of mice with SNP-7/8a using predicted neoantigens (n = 179) from three tumor models induced CD8 T cells against ~50\% of neoantigens with high predicted MHC-I binding affinity and led to enhanced tumor clearance. SNP-7/8a delivering in silico-designed mock neoantigens also induced CD8 T cells in nonhuman primates. Altogether, SNP-7/8a is a generalizable approach for codelivering peptide antigens and adjuvants in nanoparticles for inducing anticancer T-cell immunity. Cancer vaccines that self-assemble into uniform nanoparticles improve tumor clearance.},
  archive      = {J_NBT},
  author       = {Lynn, Geoffrey M. and Sedlik, Christine and Baharom, Faezzah and Zhu, Yaling and Ramirez-Valdez, Ramiro A. and Coble, Vincent L. and Tobin, Kennedy and Nichols, Sarah R. and Itzkowitz, Yaakov and Zaidi, Neeha and Gammon, Joshua M. and Blobel, Nicolas J. and Denizeau, Jordan and de la Rochere, Philippe and Francica, Brian J. and Decker, Brennan and Maciejewski, Mateusz and Cheung, Justin and Yamane, Hidehiro and Smelkinson, Margery G. and Francica, Joseph R. and Laga, Richard and Bernstock, Joshua D. and Seymour, Leonard W. and Drake, Charles G. and Jewell, Christopher M. and Lantz, Olivier and Piaggio, Eliane and Ishizuka, Andrew S. and Seder, Robert A.},
  doi          = {10.1038/s41587-019-0390-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {320-332},
  shortjournal = {Nature Biotech.},
  title        = {Peptide–TLR-7/8a conjugate vaccines chemically programmed for nanoparticle self-assembly enhance CD8 T-cell immunity to tumor antigens},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Accurate detection of mosaic variants in sequencing data
without matched controls. <em>NBT</em>, <em>38</em>(3), 314–319. (<a
href="https://doi.org/10.1038/s41587-019-0368-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Detection of mosaic mutations that arise in normal development is challenging, as such mutations are typically present in only a minute fraction of cells and there is no clear matched control for removing germline variants and systematic artifacts. We present MosaicForecast, a machine-learning method that leverages read-based phasing and read-level features to accurately detect mosaic single-nucleotide variants and indels, achieving a multifold increase in specificity compared with existing algorithms. Using single-cell sequencing and targeted sequencing, we validated 80–90\% of the mosaic single-nucleotide variants and 60–80\% of indels detected in human brain whole-genome sequencing data. Our method should help elucidate the contribution of mosaic somatic mutations to the origin and development of disease. MosaicForecast detects mosaic single-nucleotide variants and indels in human samples.},
  archive      = {J_NBT},
  author       = {Dou, Yanmei and Kwon, Minseok and Rodin, Rachel E. and Cortés-Ciriano, Isidro and Doan, Ryan and Luquette, Lovelace J. and Galor, Alon and Bohrson, Craig and Walsh, Christopher A. and Park, Peter J.},
  doi          = {10.1038/s41587-019-0368-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {314-319},
  shortjournal = {Nature Biotech.},
  title        = {Accurate detection of mosaic variants in sequencing data without matched controls},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An engineered enzyme that targets circulating lactate to
alleviate intracellular NADH: NAD+ imbalance. <em>NBT</em>,
<em>38</em>(3), 309–313. (<a
href="https://doi.org/10.1038/s41587-019-0377-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An elevated intracellular NADH:NAD+ ratio, or ‘reductive stress’, has been associated with multiple diseases, including disorders of the mitochondrial electron transport chain. As the intracellular NADH:NAD+ ratio can be in near equilibrium with the circulating lactate:pyruvate ratio, we hypothesized that reductive stress could be alleviated by oxidizing extracellular lactate to pyruvate. We engineered LOXCAT, a fusion of bacterial lactate oxidase (LOX) and catalase (CAT), which irreversibly converts lactate and oxygen to pyruvate and water. Addition of purified LOXCAT to the medium of cultured human cells with a defective electron transport chain decreased the extracellular lactate:pyruvate ratio, normalized the intracellular NADH:NAD+ ratio, upregulated glycolytic ATP production and restored cellular proliferation. In mice, tail-vein-injected LOXCAT lowered the circulating lactate:pyruvate ratio, blunted a metformin-induced rise in blood lactate:pyruvate ratio and improved NADH:NAD+ balance in the heart and brain. Our study lays the groundwork for a class of injectable therapeutic enzymes that alleviates intracellular redox imbalances by directly targeting circulating redox-coupled metabolites. Intracellular redox defects are treated with an extracellular chimeric enzyme.},
  archive      = {J_NBT},
  author       = {Patgiri, Anupam and Skinner, Owen S. and Miyazaki, Yusuke and Schleifer, Grigorij and Marutani, Eizo and Shah, Hardik and Sharma, Rohit and Goodman, Russell P. and To, Tsz-Leung and Robert Bao, Xiaoyan and Ichinose, Fumito and Zapol, Warren M. and Mootha, Vamsi K.},
  doi          = {10.1038/s41587-019-0377-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {309-313},
  shortjournal = {Nature Biotech.},
  title        = {An engineered enzyme that targets circulating lactate to alleviate intracellular NADH: NAD+ imbalance},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Identifying drug targets in tissues and whole blood with
thermal-shift profiling. <em>NBT</em>, <em>38</em>(3), 303–308. (<a
href="https://doi.org/10.1038/s41587-019-0388-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monitoring drug–target interactions with methods such as the cellular thermal-shift assay (CETSA) is well established for simple cell systems but remains challenging in vivo. Here we introduce tissue thermal proteome profiling (tissue-TPP), which measures binding of small-molecule drugs to proteins in tissue samples from drug-treated animals by detecting changes in protein thermal stability using quantitative mass spectrometry. We report organ-specific, proteome-wide thermal stability maps and derive target profiles of the non-covalent histone deacetylase inhibitor panobinostat in rat liver, lung, kidney and spleen and of the B-Raf inhibitor vemurafenib in mouse testis. In addition, we devised blood-CETSA and blood-TPP and applied it to measure target and off-target engagement of panobinostat and the BET family inhibitor JQ1 directly in whole blood. Blood-TPP analysis of panobinostat confirmed its binding to known targets and also revealed thermal stabilization of the zinc-finger transcription factor ZNF512. These methods will help to elucidate the mechanisms of drug action in vivo. The targets of small-molecule drugs are detected in tissue and blood using thermal proteome assays.},
  archive      = {J_NBT},
  author       = {Perrin, Jessica and Werner, Thilo and Kurzawa, Nils and Rutkowska, Anna and Childs, Dorothee D. and Kalxdorf, Mathias and Poeckel, Daniel and Stonehouse, Eugenia and Strohmer, Katrin and Heller, Bianca and Thomson, Douglas W. and Krause, Jana and Becher, Isabelle and Eberl, H. Christian and Vappiani, Johanna and Sevin, Daniel C. and Rau, Christina E. and Franken, Holger and Huber, Wolfgang and Faelth-Savitski, Maria and Savitski, Mikhail M. and Bantscheff, Marcus and Bergamini, Giovanna},
  doi          = {10.1038/s41587-019-0388-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {303-308},
  shortjournal = {Nature Biotech.},
  title        = {Identifying drug targets in tissues and whole blood with thermal-shift profiling},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Generation of human endothelium in pig embryos deficient in
ETV2. <em>NBT</em>, <em>38</em>(3), 297–302. (<a
href="https://doi.org/10.1038/s41587-019-0373-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scarcity of donor organs may be addressed in the future by using pigs to grow humanized organs with lower potential for immunological rejection after transplantation in humans. Previous studies have demonstrated that interspecies complementation of rodent blastocysts lacking a developmental regulatory gene can generate xenogeneic pancreas and kidney1,2. However, such organs contain host endothelium, a source of immune rejection. We used gene editing and somatic cell nuclear transfer to engineer porcine embryos deficient in ETV2, a master regulator of hematoendothelial lineages3–7. ETV2-null pig embryos lacked hematoendothelial lineages and were embryonic lethal. Blastocyst complementation with wild-type porcine blastomeres generated viable chimeric embryos whose hematoendothelial cells were entirely donor-derived. ETV2-null blastocysts were injected with human induced pluripotent stem cells (hiPSCs) or hiPSCs overexpressing the antiapoptotic factor BCL2, transferred to synchronized gilts and analyzed between embryonic day 17 and embryonic day 18. In these embryos, all endothelial cells were of human origin. Pig embryos with a human endothelium are generated through blastocyst complementation using human induced pluripotent stem cells.},
  archive      = {J_NBT},
  author       = {Das, Satyabrata and Koyano-Nakagawa, Naoko and Gafni, Ohad and Maeng, Geunho and Singh, Bhairab N. and Rasmussen, Tara and Pan, Xiaoyan and Choi, Kyung-Dal and Mickelson, Daniel and Gong, Wuming and Pota, Pruthvi and Weaver, Cyprian V. and Kren, Stefan and Hanna, Jacob H. and Yannopoulos, Demetris and Garry, Mary G. and Garry, Daniel J.},
  doi          = {10.1038/s41587-019-0373-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {297-302},
  shortjournal = {Nature Biotech.},
  title        = {Generation of human endothelium in pig embryos deficient in ETV2},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Label-free metabolic imaging by mid-infrared optoacoustic
microscopy in living cells. <em>NBT</em>, <em>38</em>(3), 293–296. (<a
href="https://doi.org/10.1038/s41587-019-0359-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We develop mid-infrared optoacoustic microscopy (MiROM) for label-free, bond-selective, live-cell metabolic imaging, enabling spatiotemporal monitoring of carbohydrates, lipids and proteins in cells and tissues. Using acoustic detection of optical absorption, MiROM converts mid-infrared sensing into a positive-contrast imaging modality with negligible photodamage and high sensitivity. We use MiROM to observe changes in intrinsic carbohydrate distribution from a diffusive spatial pattern to tight co-localization with lipid droplets during adipogenesis. Mid-infrared optoacoustic microscopy enables label-free, bond-selective imaging in living cells},
  archive      = {J_NBT},
  author       = {Pleitez, Miguel A. and Khan, Asrar Ali and Soldà, Alice and Chmyrov, Andriy and Reber, Josefine and Gasparin, Francesca and Seeger, Markus R. and Schätz, Benedikt and Herzig, Stephan and Scheideler, Marcel and Ntziachristos, Vasilis},
  doi          = {10.1038/s41587-019-0359-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {293-296},
  shortjournal = {Nature Biotech.},
  title        = {Label-free metabolic imaging by mid-infrared optoacoustic microscopy in living cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Butler enables rapid cloud-based analysis of thousands of
human genomes. <em>NBT</em>, <em>38</em>(3), 288–292. (<a
href="https://doi.org/10.1038/s41587-019-0360-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Butler, a computational tool that facilitates large-scale genomic analyses on public and academic clouds. Butler includes innovative anomaly detection and self-healing functions that improve the efficiency of data processing and analysis by 43\% compared with current approaches. Butler enabled processing of a 725-terabyte cancer genome dataset from the Pan-Cancer Analysis of Whole Genomes (PCAWG) project in a time-efficient and uniform manner. Efficient, large-scale genomic analysis is facilitated on the cloud by a computational tool with error-diagnosing and self-healing capabilities.},
  archive      = {J_NBT},
  author       = {Yakneen, Sergei and Waszak, Sebastian M. and Gertz, Michael and Korbel, Jan O.},
  doi          = {10.1038/s41587-019-0360-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {288-292},
  shortjournal = {Nature Biotech.},
  title        = {Butler enables rapid cloud-based analysis of thousands of human genomes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Forum: Molecular mapping of tumor heterogeneity.
<em>NBT</em>, <em>38</em>(3), 286. (<a
href="https://doi.org/10.1038/s41587-020-0445-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0445-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {286},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Molecular mapping of tumor heterogeneity},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Deciphering the human phosphoproteome. <em>NBT</em>,
<em>38</em>(3), 285–286. (<a
href="https://doi.org/10.1038/s41587-020-0441-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning provides a functional score for ~120,000 human phosphorylation sites.},
  archive      = {J_NBT},
  author       = {Franciosa, Giulia and Martinez-Val, Ana and Olsen, Jesper V.},
  doi          = {10.1038/s41587-020-0441-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {285-286},
  shortjournal = {Nature Biotech.},
  title        = {Deciphering the human phosphoproteome},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Neuroengineering. <em>NBT</em>, <em>38</em>(3), 284. (<a
href="https://doi.org/10.1038/s41587-020-0454-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to methods of transcranial magnetic and electrostimulation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0454-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {284},
  shortjournal = {Nature Biotech.},
  title        = {Neuroengineering},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). One year after vanda, are diagnostics patents transforming
into methods of treatment to overcome mayo-based rejections?
<em>NBT</em>, <em>38</em>(3), 279–283. (<a
href="https://doi.org/10.1038/s41587-020-0440-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What impact have Mayo and Vanda had for applicants attempting to obtain patent protection for inventions involving methods of diagnosis and methods of treatment?},
  archive      = {J_NBT},
  author       = {Aboy, Mateo and Crespo, Cristina and Liddell, Kathleen and Davey, Neil and Liddicoat, Johnathon and Minssen, Timo},
  doi          = {10.1038/s41587-020-0440-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {279-283},
  shortjournal = {Nature Biotech.},
  title        = {One year after vanda, are diagnostics patents transforming into methods of treatment to overcome mayo-based rejections?},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The nf-core framework for community-curated bioinformatics
pipelines. <em>NBT</em>, <em>38</em>(3), 276–278. (<a
href="https://doi.org/10.1038/s41587-020-0439-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ewels, Philip A. and Peltzer, Alexander and Fillinger, Sven and Patel, Harshil and Alneberg, Johannes and Wilm, Andreas and Garcia, Maxime Ulysse and Di Tommaso, Paolo and Nahnsen, Sven},
  doi          = {10.1038/s41587-020-0439-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {276-278},
  shortjournal = {Nature Biotech.},
  title        = {The nf-core framework for community-curated bioinformatics pipelines},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). MEMOTE for standardized genome-scale metabolic model
testing. <em>NBT</em>, <em>38</em>(3), 272–276. (<a
href="https://doi.org/10.1038/s41587-020-0446-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Lieven, Christian and Beber, Moritz E. and Olivier, Brett G. and Bergmann, Frank T. and Ataman, Meric and Babaei, Parizad and Bartell, Jennifer A. and Blank, Lars M. and Chauhan, Siddharth and Correia, Kevin and Diener, Christian and Dräger, Andreas and Ebert, Birgitta E. and Edirisinghe, Janaka N. and Faria, José P. and Feist, Adam M. and Fengos, Georgios and Fleming, Ronan M. T. and García-Jiménez, Beatriz and Hatzimanikatis, Vassily and van Helvoirt, Wout and Henry, Christopher S. and Hermjakob, Henning and Herrgård, Markus J. and Kaafarani, Ali and Kim, Hyun Uk and King, Zachary and Klamt, Steffen and Klipp, Edda and Koehorst, Jasper J. and König, Matthias and Lakshmanan, Meiyappan and Lee, Dong-Yup and Lee, Sang Yup and Lee, Sunjae and Lewis, Nathan E. and Liu, Filipe and Ma, Hongwu and Machado, Daniel and Mahadevan, Radhakrishnan and Maia, Paulo and Mardinoglu, Adil and Medlock, Gregory L. and Monk, Jonathan M. and Nielsen, Jens and Nielsen, Lars Keld and Nogales, Juan and Nookaew, Intawat and Palsson, Bernhard O. and Papin, Jason A. and Patil, Kiran R. and Poolman, Mark and Price, Nathan D. and Resendis-Antonio, Osbaldo and Richelle, Anne and Rocha, Isabel and Sánchez, Benjamín J. and Schaap, Peter J. and Malik Sheriff, Rahuman S. and Shoaie, Saeed and Sonnenschein, Nikolaus and Teusink, Bas and Vilaça, Paulo and Vik, Jon Olav and Wodke, Judith A. H. and Xavier, Joana C. and Yuan, Qianqian and Zakhartsev, Maksim and Zhang, Cheng},
  doi          = {10.1038/s41587-020-0446-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {272-276},
  shortjournal = {Nature Biotech.},
  title        = {MEMOTE for standardized genome-scale metabolic model testing},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). To publish or not to publish. <em>NBT</em>, <em>38</em>(3),
271. (<a href="https://doi.org/10.1038/s41587-020-0435-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Baylis, Françoise},
  doi          = {10.1038/s41587-020-0435-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {271},
  shortjournal = {Nature Biotech.},
  title        = {To publish or not to publish},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Understanding the basics of patenting. <em>NBT</em>,
<em>38</em>(3), 263–270. (<a
href="https://doi.org/10.1038/s41587-020-0447-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Some practical advice on how to protect research that you are seeking to commercialize.},
  archive      = {J_NBT},
  author       = {Chitale, Sadhana and Lawler, Colm and Macfarlane, Scott},
  doi          = {10.1038/s41587-020-0447-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {263-270},
  shortjournal = {Nature Biotech.},
  title        = {Understanding the basics of patenting},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). Around the world in a month. <em>NBT</em>, <em>38</em>(3),
257. (<a href="https://doi.org/10.1038/s41587-020-0451-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0451-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {257},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). BIO’s bias report. <em>NBT</em>, <em>38</em>(3), 256. (<a
href="https://doi.org/10.1038/s41587-020-0460-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0460-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {256},
  shortjournal = {Nature Biotech.},
  title        = {BIO’s bias report},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Single-cell RNA-seq analysis software providers scramble to
offer solutions. <em>NBT</em>, <em>38</em>(3), 254–257. (<a
href="https://doi.org/10.1038/s41587-020-0449-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A raft of tools have sprung up to help biologists work through the single-cell transcriptomic bottleneck, but integration remains elusive.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-020-0449-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {254-257},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell RNA-seq analysis software providers scramble to offer solutions},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Go-ahead for first peanut allergy drug. <em>NBT</em>,
<em>38</em>(3), 254. (<a
href="https://doi.org/10.1038/s41587-020-0458-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0458-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {254},
  shortjournal = {Nature Biotech.},
  title        = {Go-ahead for first peanut allergy drug},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). First rounders: Bassil dahiyat. <em>NBT</em>,
<em>38</em>(3), 253. (<a
href="https://doi.org/10.1038/s41587-020-0436-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0436-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {253},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Bassil dahiyat},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). FDA dictates on gene therapy sameness. <em>NBT</em>,
<em>38</em>(3), 253. (<a
href="https://doi.org/10.1038/s41587-020-0459-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0459-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {253},
  shortjournal = {Nature Biotech.},
  title        = {FDA dictates on gene therapy sameness},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020h). Huntington’s antisense drug marches into clinic.
<em>NBT</em>, <em>38</em>(3), 251–253. (<a
href="https://doi.org/10.1038/s41587-020-0452-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prospects for agents aimed at stopping neuronal loss brighten, driven by new mechanistic understanding.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-020-0452-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {251-253},
  shortjournal = {Nature Biotech.},
  title        = {Huntington’s antisense drug marches into clinic},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Making the best of brexit. <em>NBT</em>, <em>38</em>(3),
249. (<a href="https://doi.org/10.1038/s41587-020-0463-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the post-Brexit era, Britain must rapidly harmonize national and European law, secure unfettered access to global talent and address funding shortfalls to ensure continued leadership in the life sciences.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0463-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {249},
  shortjournal = {Nature Biotech.},
  title        = {Making the best of brexit},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Brain’s drain. <em>NBT</em>, <em>38</em>(3), 258–262. (<a
href="https://doi.org/10.1038/s41587-020-0443-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lymphatics in the central nervous system are attracting both controversy and drug hunters in search of therapeutic targets for a host of diseases.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-0443-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {258-262},
  shortjournal = {Nature Biotech.},
  title        = {Brain’s drain},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). People. <em>NBT</em>, <em>38</em>(2), 248. (<a
href="https://doi.org/10.1038/s41587-020-0421-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0421-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {248},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Fourth-quarter biotech job picture. <em>NBT</em>,
<em>38</em>(2), 247. (<a
href="https://doi.org/10.1038/s41587-020-0416-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-020-0416-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {247},
  shortjournal = {Nature Biotech.},
  title        = {Fourth-quarter biotech job picture},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Author correction: Genomic and phenotypic analyses of six
offspring of a genome-edited hornless bull. <em>NBT</em>,
<em>38</em>(2), 245. (<a
href="https://doi.org/10.1038/s41587-020-0423-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Young, Amy E. and Mansour, Tamer A. and McNabb, Bret R. and Owen, Joseph R. and Trott, Josephine F. and Brown, C. Titus and Van Eenennaam, Alison L.},
  doi          = {10.1038/s41587-020-0423-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {245},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Genomic and phenotypic analyses of six offspring of a genome-edited hornless bull},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: In situ readout of DNA barcodes and
single base edits facilitated by in vitro transcription. <em>NBT</em>,
<em>38</em>(2), 245. (<a
href="https://doi.org/10.1038/s41587-020-0432-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Askary, Amjad and Sanchez-Guardado, Luis and Linton, James M. and Chadly, Duncan M. and Budde, Mark W. and Cai, Long and Lois, Carlos and Elowitz, Michael B.},
  doi          = {10.1038/s41587-020-0432-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {245},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: In situ readout of DNA barcodes and single base edits facilitated by in vitro transcription},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The therapeutic landscape for cells engineered with chimeric
antigen receptors. <em>NBT</em>, <em>38</em>(2), 233–244. (<a
href="https://doi.org/10.1038/s41587-019-0329-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the global rapid increase in the number of clinical trials employing chimeric antigen receptors (CARs), no comprehensive survey of their scope, targets and design exists. In this study, we present an interactive CAR clinical trial database, spanning 64 targets deployed in T cells (CAR-T), natural killer cells (CAR-NK) or mixtures (CAR-NK/T) from over 500 clinical trials in 20 countries, encompassing &amp;gt;20,000 patients. By combining these data with transcriptional and proteomic data, we create a ‘targetable landscape’ for CAR cell therapies based on 13,206 proteins and RNAs across 78 tissues, 124 cell types and 20 cancer types. These data suggest a landscape of over 100 single targets and over 100,000 target pairs using logical switches for CAR cell engineering. Our analysis of the CAR cellular therapeutic landscape may aid the design of future therapies, improve target-to-patient matching, and ultimately help inform our understanding of CAR therapy’s safety and efficacy. A survey of the landscape of targets of chimeric antigen receptor cell therapies currently in clinical trials, with a browsable portal for patients and clinicians.},
  archive      = {J_NBT},
  author       = {MacKay, Matthew and Afshinnekoo, Ebrahim and Rub, Jonathan and Hassan, Ciaran and Khunte, Mihir and Baskaran, Nithyashri and Owens, Bryan and Liu, Lauren and Roboz, Gail J. and Guzman, Monica L. and Melnick, Ari M. and Wu, Shixiu and Mason, Christopher E.},
  doi          = {10.1038/s41587-019-0329-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {233-244},
  shortjournal = {Nature Biotech.},
  title        = {The therapeutic landscape for cells engineered with chimeric antigen receptors},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Genomic and phenotypic analyses of six offspring of a
genome-edited hornless bull. <em>NBT</em>, <em>38</em>(2), 225–232. (<a
href="https://doi.org/10.1038/s41587-019-0266-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing followed by reproductive cloning was previously used to produce two hornless dairy bulls. We crossed one genome-edited dairy bull, homozygous for the dominant PC Celtic POLLED allele, with horned cows (pp) and obtained six heterozygous (PCp) polled calves. The calves had no horns and were otherwise healthy and phenotypically unremarkable. We conducted whole-genome sequencing of all animals using an Illumina HiSeq4000 to achieve ~20× coverage. Bioinformatics analyses revealed the bull was a compound heterozygote, carrying one naturally occurring PC Celtic POLLED allele and an allele containing an additional introgression of the homology-directed repair donor plasmid along with the PC Celtic allele. These alleles segregated in the offspring of this bull, and inheritance of either allele produced polled calves. No other unintended genomic alterations were observed. These data can be used to inform conversations in the scientific community, with regulatory authorities and with the public around ‘intentional genomic alterations’ and future regulatory actions regarding genome-edited animals. A genome-edited hornless compound heterozygous dairy bull transmitted this trait to his offspring.},
  archive      = {J_NBT},
  author       = {Young, Amy E. and Mansour, Tamer A. and McNabb, Bret R. and Owen, Joseph R. and Trott, Josephine F. and Brown, C. Titus and Van Eenennaam, Alison L.},
  doi          = {10.1038/s41587-019-0266-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {225-232},
  shortjournal = {Nature Biotech.},
  title        = {Genomic and phenotypic analyses of six offspring of a genome-edited hornless bull},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A laser-engraved wearable sensor for sensitive detection of
uric acid and tyrosine in sweat. <em>NBT</em>, <em>38</em>(2), 217–224.
(<a href="https://doi.org/10.1038/s41587-019-0321-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wearable sweat sensors have the potential to provide continuous measurements of useful biomarkers. However, current sensors cannot accurately detect low analyte concentrations, lack multimodal sensing or are difficult to fabricate at large scale. We report an entirely laser-engraved sensor for simultaneous sweat sampling, chemical sensing and vital-sign monitoring. We demonstrate continuous detection of temperature, respiration rate and low concentrations of uric acid and tyrosine, analytes associated with diseases such as gout and metabolic disorders. We test the performance of the device in both physically trained and untrained subjects under exercise and after a protein-rich diet. We also evaluate its utility for gout monitoring in patients and healthy controls through a purine-rich meal challenge. Levels of uric acid in sweat were higher in patients with gout than in healthy individuals, and a similar trend was observed in serum. Low levels of uric acid and tyrosine in sweat are detected in patients with gout and healthy individuals with an entirely laser-engraved wearable sensor.},
  archive      = {J_NBT},
  author       = {Yang, Yiran and Song, Yu and Bo, Xiangjie and Min, Jihong and Pak, On Shun and Zhu, Lailai and Wang, Minqiang and Tu, Jiaobing and Kogan, Adam and Zhang, Haixia and Hsiai, Tzung K. and Li, Zhaoping and Gao, Wei},
  doi          = {10.1038/s41587-019-0321-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {217-224},
  shortjournal = {Nature Biotech.},
  title        = {A laser-engraved wearable sensor for sensitive detection of uric acid and tyrosine in sweat},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The industrial yeast pichia pastoris is converted from a
heterotroph into an autotroph capable of growth on CO2. <em>NBT</em>,
<em>38</em>(2), 210–216. (<a
href="https://doi.org/10.1038/s41587-019-0363-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The methylotrophic yeast Pichia pastoris is widely used in the manufacture of industrial enzymes and pharmaceuticals. Like most biotechnological production hosts, P. pastoris is heterotrophic and grows on organic feedstocks that have competing uses in the production of food and animal feed. In a step toward more sustainable industrial processes, we describe the conversion of P. pastoris into an autotroph that grows on CO2. By addition of eight heterologous genes and deletion of three native genes, we engineer the peroxisomal methanol-assimilation pathway of P. pastoris into a CO2-fixation pathway resembling the Calvin–Benson–Bassham cycle, the predominant natural CO2-fixation pathway. The resulting strain can grow continuously with CO2 as a sole carbon source at a µmax of 0.008 h−1. The specific growth rate was further improved to 0.018 h−1 by adaptive laboratory evolution. This engineered P. pastoris strain may promote sustainability by sequestering the greenhouse gas CO2, and by avoiding consumption of an organic feedstock with alternative uses in food production. A yeast species used to produce proteins and chemicals is engineered to grow solely on the greenhouse gas CO2.},
  archive      = {J_NBT},
  author       = {Gassler, Thomas and Sauer, Michael and Gasser, Brigitte and Egermeier, Michael and Troyer, Christina and Causon, Tim and Hann, Stephan and Mattanovich, Diethard and Steiger, Matthias G.},
  doi          = {10.1038/s41587-019-0363-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {210-216},
  shortjournal = {Nature Biotech.},
  title        = {The industrial yeast pichia pastoris is converted from a heterotroph into an autotroph capable of growth on CO2},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A large peptidome dataset improves HLA class i epitope
prediction across most of the human population. <em>NBT</em>,
<em>38</em>(2), 199–209. (<a
href="https://doi.org/10.1038/s41587-019-0322-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prediction of HLA epitopes is important for the development of cancer immunotherapies and vaccines. However, current prediction algorithms have limited predictive power, in part because they were not trained on high-quality epitope datasets covering a broad range of HLA alleles. To enable prediction of endogenous HLA class I-associated peptides across a large fraction of the human population, we used mass spectrometry to profile &amp;gt;185,000 peptides eluted from 95 HLA-A, -B, -C and -G mono-allelic cell lines. We identified canonical peptide motifs per HLA allele, unique and shared binding submotifs across alleles and distinct motifs associated with different peptide lengths. By integrating these data with transcript abundance and peptide processing, we developed HLAthena, providing allele-and-length-specific and pan-allele-pan-length prediction models for endogenous peptide presentation. These models predicted endogenous HLA class I-associated ligands with 1.5-fold improvement in positive predictive value compared with existing tools and correctly identified &amp;gt;75\% of HLA-bound peptides that were observed experimentally in 11 patient-derived tumor cell lines. Prediction of HLA class I epitopes is improved in accuracy and breath with peptidomes from 95 mono-allelic cell lines.},
  archive      = {J_NBT},
  author       = {Sarkizova, Siranush and Klaeger, Susan and Le, Phuong M. and Li, Letitia W. and Oliveira, Giacomo and Keshishian, Hasmik and Hartigan, Christina R. and Zhang, Wandi and Braun, David A. and Ligon, Keith L. and Bachireddy, Pavan and Zervantonakis, Ioannis K. and Rosenbluth, Jennifer M. and Ouspenskaia, Tamara and Law, Travis and Justesen, Sune and Stevens, Jonathan and Lane, William J. and Eisenhaure, Thomas and Lan Zhang, Guang and Clauser, Karl R. and Hacohen, Nir and Carr, Steven A. and Wu, Catherine J. and Keskin, Derin B.},
  doi          = {10.1038/s41587-019-0322-9},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {199-209},
  shortjournal = {Nature Biotech.},
  title        = {A large peptidome dataset improves HLA class i epitope prediction across most of the human population},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). An integrated perfusion machine preserves injured human
livers for 1 week. <em>NBT</em>, <em>38</em>(2), 189–198. (<a
href="https://doi.org/10.1038/s41587-019-0374-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to preserve metabolically active livers ex vivo for 1 week or more could allow repair of poor-quality livers that would otherwise be declined for transplantation. Current approaches for normothermic perfusion can preserve human livers for only 24 h. Here we report a liver perfusion machine that integrates multiple core physiological functions, including automated management of glucose levels and oxygenation, waste-product removal and hematocrit control. We developed the machine in a stepwise fashion using pig livers. Study of multiple ex vivo parameters and early phase reperfusion in vivo demonstrated the viability of pig livers perfused for 1 week without the need for additional blood products or perfusate exchange. We tested the approach on ten injured human livers that had been declined for transplantation by all European centers. After a 7-d perfusion, six of the human livers showed preserved function as indicated by bile production, synthesis of coagulation factors, maintained cellular energy (ATP) and intact liver structure. Livers are stored long term in a sophisticated perfusion system.},
  archive      = {J_NBT},
  author       = {Eshmuminov, Dilmurodjon and Becker, Dustin and Bautista Borrego, Lucia and Hefti, Max and Schuler, Martin J. and Hagedorn, Catherine and Muller, Xavier and Mueller, Matteo and Onder, Christopher and Graf, Rolf and Weber, Achim and Dutkowski, Philipp and Rudolf von Rohr, Philipp and Clavien, Pierre-Alain},
  doi          = {10.1038/s41587-019-0374-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {189-198},
  shortjournal = {Nature Biotech.},
  title        = {An integrated perfusion machine preserves injured human livers for 1 week},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Rapid customization of solanaceae fruit crops for urban
agriculture. <em>NBT</em>, <em>38</em>(2), 182–188. (<a
href="https://doi.org/10.1038/s41587-019-0361-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivation of crops in urban environments might reduce the environmental impact of food production1–4. However, lack of available land in cities and a need for rapid crop cycling, to yield quickly and continuously, mean that so far only lettuce and related ‘leafy green’ vegetables are cultivated in urban farms5. New fruit varieties with architectures and yields suitable for urban farming have proven difficult to breed1,5. We identified a regulator of tomato stem length (SlER) and devised a trait-stacking strategy to combine mutations for condensed shoots, rapid flowering (SP5G) and precocious growth termination (SP). Application of our strategy using one-step CRISPR–Cas9 genome editing restructured vine-like tomato plants into compact, early yielding plants suitable for urban agriculture. Field data confirmed that yields were maintained, and we demonstrated cultivation in indoor farming systems. Targeting the same stem length regulator alone in groundcherry, another Solanaceae plant, also enabled engineering to a compact stature. Our approach can expand the repertoire of crops for urban agriculture. Compact early fruiting tomato and groundcherry plants suitable for urban farming are produced using genome editing.},
  archive      = {J_NBT},
  author       = {Kwon, Choon-Tak and Heo, Jung and Lemmon, Zachary H. and Capua, Yossi and Hutton, Samuel F. and Van Eck, Joyce and Park, Soon Ju and Lippman, Zachary B.},
  doi          = {10.1038/s41587-019-0361-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {182-188},
  shortjournal = {Nature Biotech.},
  title        = {Rapid customization of solanaceae fruit crops for urban agriculture},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Electrical recognition of the twenty proteinogenic amino
acids using an aerolysin nanopore. <em>NBT</em>, <em>38</em>(2),
176–181. (<a href="https://doi.org/10.1038/s41587-019-0345-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efforts to sequence single protein molecules in nanopores1–5 have been hampered by the lack of techniques with sufficient sensitivity to discern the subtle molecular differences among all twenty amino acids. Here we report ionic current detection of all twenty proteinogenic amino acids in an aerolysin nanopore with the help of a short polycationic carrier. Application of molecular dynamics simulations revealed that the aerolysin nanopore has a built-in single-molecule trap that fully confines a polycationic carrier-bound amino acid inside the sensing region of the aerolysin. This structural feature means that each amino acid spends sufficient time in the pore for sensitive measurement of the excluded volume of the amino acid. We show that distinct current blockades in wild-type aerolysin can be used to identify 13 of the 20 natural amino acids. Furthermore, we show that chemical modifications, instrumentation advances and nanopore engineering offer a route toward identification of the remaining seven amino acids. These findings may pave the way to nanopore protein sequencing. Individual amino acids fused to a highly charged heptapeptide are discriminated in an aerolysin nanopore.},
  archive      = {J_NBT},
  author       = {Ouldali, Hadjer and Sarthak, Kumar and Ensslen, Tobias and Piguet, Fabien and Manivet, Philippe and Pelta, Juan and Behrends, Jan C. and Aksimentiev, Aleksei and Oukhaled, Abdelghani},
  doi          = {10.1038/s41587-019-0345-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {176-181},
  shortjournal = {Nature Biotech.},
  title        = {Electrical recognition of the twenty proteinogenic amino acids using an aerolysin nanopore},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A reversible RNA on-switch that controls gene expression of
AAV-delivered therapeutics in vivo. <em>NBT</em>, <em>38</em>(2),
169–175. (<a href="https://doi.org/10.1038/s41587-019-0357-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Widespread use of gene therapy technologies is limited in part by the lack of small genetic switches with wide dynamic ranges that control transgene expression without the requirement of additional protein components1–5. In this study, we engineered a class of type III hammerhead ribozymes to develop RNA switches that are highly efficient at cis-cleaving mammalian mRNAs and showed that they can be tightly regulated by a steric-blocking antisense oligonucleotide. Our variant ribozymes enabled in vivo regulation of adeno-associated virus (AAV)-delivered transgenes, allowing dose-dependent and up to 223-fold regulation of protein expression over at least 43 weeks. To test the potential of these reversible on-switches in gene therapy for anemia of chronic kidney disease6, we demonstrated regulated expression of physiological levels of erythropoietin with a well-tolerated dose of the inducer oligonucleotide. These small, modular and efficient RNA switches may improve the safety and efficacy of gene therapies and broaden their use. In vivo control of expression levels of AAV-delivered genes is enabled by an engineered RNA switch.},
  archive      = {J_NBT},
  author       = {Zhong, Guocai and Wang, Haimin and He, Wenhui and Li, Yujun and Mou, Huihui and Tickner, Zachary J. and Tran, Mai H. and Ou, Tianling and Yin, Yiming and Diao, Huitian and Farzan, Michael},
  doi          = {10.1038/s41587-019-0357-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {169-175},
  shortjournal = {Nature Biotech.},
  title        = {A reversible RNA on-switch that controls gene expression of AAV-delivered therapeutics in vivo},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Efficient, continuous mutagenesis in human cells using a
pseudo-random DNA editor. <em>NBT</em>, <em>38</em>(2), 165–168. (<a
href="https://doi.org/10.1038/s41587-019-0331-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we describe TRACE (T7 polymerase-driven continuous editing), a method that enables continuous, targeted mutagenesis in human cells using a cytidine deaminase fused to T7 RNA polymerase. TRACE induces high rates of mutagenesis over multiple cell generations in genes under the control of a T7 promoter integrated in the genome. We used TRACE in a MEK1 inhibitor-resistance screen, and identified functionally correlated mutations. Efficient diversification of DNA sequences is enabled by a T7 RNA polymerase fused to a cytidine deaminase.},
  archive      = {J_NBT},
  author       = {Chen, Haiqi and Liu, Sophia and Padula, Samuel and Lesman, Daniel and Griswold, Kettner and Lin, Allen and Zhao, Tongtong and Marshall, Jamie L. and Chen, Fei},
  doi          = {10.1038/s41587-019-0331-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {165-168},
  shortjournal = {Nature Biotech.},
  title        = {Efficient, continuous mutagenesis in human cells using a pseudo-random DNA editor},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Template plasmid integration in germline genome-edited
cattle. <em>NBT</em>, <em>38</em>(2), 163–164. (<a
href="https://doi.org/10.1038/s41587-019-0394-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Norris, Alexis L. and Lee, Stella S. and Greenlees, Kevin J. and Tadesse, Daniel A. and Miller, Mayumi F. and Lombardi, Heather A.},
  doi          = {10.1038/s41587-019-0394-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {163-164},
  shortjournal = {Nature Biotech.},
  title        = {Template plasmid integration in germline genome-edited cattle},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Vertical farms bear fruit. <em>NBT</em>, <em>38</em>(2),
160–162. (<a href="https://doi.org/10.1038/s41587-019-0400-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineering perishable crops for use in indoor farms promises to expand the adoption of this high-yielding, efficient means of food production.},
  archive      = {J_NBT},
  author       = {O’Sullivan, Cathryn A. and McIntyre, C. Lynne and Dry, Ian B. and Hani, Susan M. and Hochman, Zvi and Bonnett, Graham D.},
  doi          = {10.1038/s41587-019-0400-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {160-162},
  shortjournal = {Nature Biotech.},
  title        = {Vertical farms bear fruit},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reading amino acids in a nanopore. <em>NBT</em>,
<em>38</em>(2), 159–160. (<a
href="https://doi.org/10.1038/s41587-019-0401-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In a step toward nanopore sequencing of proteins, an aerolysin pore discriminates many of the proteinogenic amino acids.},
  archive      = {J_NBT},
  author       = {Howorka, Stefan and Siwy, Zuzanna S.},
  doi          = {10.1038/s41587-019-0401-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {159-160},
  shortjournal = {Nature Biotech.},
  title        = {Reading amino acids in a nanopore},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Lysosomes. <em>NBT</em>, <em>38</em>(2), 158. (<a
href="https://doi.org/10.1038/s41587-020-0420-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to lysosome-targeted therapeutics and methods of screening and visualization.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0420-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {158},
  shortjournal = {Nature Biotech.},
  title        = {Lysosomes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Emerging patent landscape for non-viral vectors used for
gene therapy. <em>NBT</em>, <em>38</em>(2), 151–157. (<a
href="https://doi.org/10.1038/s41587-019-0402-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of the emerging patent landscape of gene therapies under development, focusing on non-viral vectors.},
  archive      = {J_NBT},
  author       = {Picanço-Castro, Virginia and Pereira, Cristiano Gonçalves and Covas, Dimas Tadeu and Porto, Geciane Silveira and Athanassiadou, Aglaia and Figueiredo, Marxa Leão},
  doi          = {10.1038/s41587-019-0402-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {151-157},
  shortjournal = {Nature Biotech.},
  title        = {Emerging patent landscape for non-viral vectors used for gene therapy},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Droplet scRNA-seq is not zero-inflated. <em>NBT</em>,
<em>38</em>(2), 147–150. (<a
href="https://doi.org/10.1038/s41587-019-0379-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Svensson, Valentine},
  doi          = {10.1038/s41587-019-0379-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {147-150},
  shortjournal = {Nature Biotech.},
  title        = {Droplet scRNA-seq is not zero-inflated},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Reply to “assessing the impact of generative AI on medicinal
chemistry.” <em>NBT</em>, <em>38</em>(2), 146. (<a
href="https://doi.org/10.1038/s41587-020-0417-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zhavoronkov, Alex and Aspuru-Guzik, Alán},
  doi          = {10.1038/s41587-020-0417-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {146},
  shortjournal = {Nature Biotech.},
  title        = {Reply to ‘Assessing the impact of generative AI on medicinal chemistry’},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Assessing the impact of generative AI on medicinal
chemistry. <em>NBT</em>, <em>38</em>(2), 143–145. (<a
href="https://doi.org/10.1038/s41587-020-0418-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Walters, W. Patrick and Murcko, Mark},
  doi          = {10.1038/s41587-020-0418-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {143-145},
  shortjournal = {Nature Biotech.},
  title        = {Assessing the impact of generative AI on medicinal chemistry},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Genome editing in animals: Why FDA regulation matters.
<em>NBT</em>, <em>38</em>(2), 142–143. (<a
href="https://doi.org/10.1038/s41587-020-0413-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Solomon, Steven M.},
  doi          = {10.1038/s41587-020-0413-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {142-143},
  shortjournal = {Nature Biotech.},
  title        = {Genome editing in animals: Why FDA regulation matters},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The biotech living and the walking dead. <em>NBT</em>,
<em>38</em>(2), 132–141. (<a
href="https://doi.org/10.1038/s41587-019-0399-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A detailed look at the economic impact of university-licensed life science startups across the United States reveals vast differences in the effectiveness of different regions to create ventures and sustain them as viable entities.},
  archive      = {J_NBT},
  author       = {Godfrey, Paul C. and Allen, Gove N. and Benson, David},
  doi          = {10.1038/s41587-019-0399-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {132-141},
  shortjournal = {Nature Biotech.},
  title        = {The biotech living and the walking dead},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020j). Drug pipeline 4Q19. <em>NBT</em>, <em>38</em>(2), 124–125.
(<a href="https://doi.org/10.1038/s41587-020-0408-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0408-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {124-125},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 4Q19},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). 2019 — steady as she goes. <em>NBT</em>, <em>38</em>(2),
122–123. (<a href="https://doi.org/10.1038/s41587-020-0409-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-020-0409-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {122-123},
  shortjournal = {Nature Biotech.},
  title        = {2019 — steady as she goes},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). First rounders: Greg verdine. <em>NBT</em>, <em>38</em>(2),
121. (<a href="https://doi.org/10.1038/s41587-019-0406-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0406-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {121},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Greg verdine},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020k). Around the world in a month. <em>NBT</em>, <em>38</em>(2),
121. (<a href="https://doi.org/10.1038/s41587-020-0410-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0410-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {121},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Drugmakers turn sights on scarred lungs. <em>NBT</em>,
<em>38</em>(2), 118–120. (<a
href="https://doi.org/10.1038/s41587-020-0415-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A ‘master regulator’ of fibrosis, pentraxin, is in drugmakers’ sights, but drivers and mechanisms of the resulting remorseless and fatal lung disease remain unsolved.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-020-0415-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {118-120},
  shortjournal = {Nature Biotech.},
  title        = {Drugmakers turn sights on scarred lungs},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Statin alternative awaits go-ahead. <em>NBT</em>,
<em>38</em>(2), 118. (<a
href="https://doi.org/10.1038/s41587-020-0424-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0424-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {118},
  shortjournal = {Nature Biotech.},
  title        = {Statin alternative awaits go-ahead},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Digital drug fortunes falter. <em>NBT</em>, <em>38</em>(2),
117. (<a href="https://doi.org/10.1038/s41587-020-0425-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0425-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {117},
  shortjournal = {Nature Biotech.},
  title        = {Digital drug fortunes falter},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Illumina, PacBio deal dead. <em>NBT</em>, <em>38</em>(2),
116. (<a href="https://doi.org/10.1038/s41587-020-0429-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0429-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {116},
  shortjournal = {Nature Biotech.},
  title        = {Illumina, PacBio deal dead},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Cancer–neuronal crosstalk and the startups working to
silence it. <em>NBT</em>, <em>38</em>(2), 115–117. (<a
href="https://doi.org/10.1038/s41587-020-0411-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Investors back ‘cancer neuroscience’ outfits as the field breaks out from academia.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-0411-9},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {115-117},
  shortjournal = {Nature Biotech.},
  title        = {Cancer–neuronal crosstalk and the startups working to silence it},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Course correction. <em>NBT</em>, <em>38</em>(2), 113. (<a
href="https://doi.org/10.1038/s41587-020-0433-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The US Food and Drug Administration is sticking to its plan to carry out mandatory premarket review of all gene-edited livestock, irrespective of trait risk. It should rethink.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-0433-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {113},
  shortjournal = {Nature Biotech.},
  title        = {Course correction},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Fresh from the biotech pipeline—2019. <em>NBT</em>,
<em>38</em>(2), 126–131. (<a
href="https://doi.org/10.1038/s41587-019-0405-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biopharmaceutical industry’s positive regulatory momentum carried on through 2019, despite an early-year government shutdown, a data manipulation scandal around a high-profile approval, and many months without permanent leadership at the FDA.},
  archive      = {J_NBT},
  author       = {Morrison, Chris},
  doi          = {10.1038/s41587-019-0405-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {126-131},
  shortjournal = {Nature Biotech.},
  title        = {Fresh from the biotech pipeline—2019},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). People. <em>NBT</em>, <em>38</em>(1), 112. (<a
href="https://doi.org/10.1038/s41587-019-0380-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0380-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {112},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Championing leadership development in healthcare.
<em>NBT</em>, <em>38</em>(1), 110–111. (<a
href="https://doi.org/10.1038/s41587-019-0370-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effective development of a physician–leadership workforce enables trainees to keep pace with the dynamic and increasingly complex future of healthcare.},
  archive      = {J_NBT},
  author       = {Law, Arjun D. and Rodin, Danielle and Giuliani, Meredith and Moraes, Fabio Y.},
  doi          = {10.1038/s41587-019-0370-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {110-111},
  shortjournal = {Nature Biotech.},
  title        = {Championing leadership development in healthcare},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Publisher correction: Harnessing the intracellular
triacylglycerols for titer improvement of polyketides in streptomyces.
<em>NBT</em>, <em>38</em>(1), 109. (<a
href="https://doi.org/10.1038/s41587-019-0389-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Wang, Weishan and Li, Shanshan and Li, Zilong and Zhang, Jingyu and Fan, Keqiang and Tan, Gaoyi and Ai, Guomin and Lam, Sin Man and Shui, Guanghou and Yang, Zhiheng and Lu, Hongzhong and Jin, Pinjiao and Li, Yihong and Chen, Xiangyin and Xia, Xuekui and Liu, Xueting and Dannelly, H. Kathleen and Yang, Chen and Yang, Yi and Zhang, Siliang and Alterovitz, Gil and Xiang, Wensheng and Zhang, Lixin},
  doi          = {10.1038/s41587-019-0389-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {109},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Harnessing the intracellular triacylglycerols for titer improvement of polyketides in streptomyces},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Efficient proximity labeling in living
cells and organisms with TurboID. <em>NBT</em>, <em>38</em>(1), 108. (<a
href="https://doi.org/10.1038/s41587-019-0355-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Branon, Tess C. and Bosch, Justin A. and Sanchez, Ariana D. and Udeshi, Namrata D. and Svinkina, Tanya and Carr, Steven A. and Feldman, Jessica L. and Perrimon, Norbert and Ting, Alice Y.},
  doi          = {10.1038/s41587-019-0355-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {108},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Efficient proximity labeling in living cells and organisms with TurboID},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Author correction: Visualizing structure and transitions in
high-dimensional biological data. <em>NBT</em>, <em>38</em>(1), 108. (<a
href="https://doi.org/10.1038/s41587-019-0395-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Moon, Kevin R. and van Dijk, David and Wang, Zheng and Gigante, Scott and Burkhardt, Daniel B. and Chen, William S. and Yim, Kristina and van den Elzen, Antonia and Hirn, Matthew J. and Coifman, Ronald R. and Ivanova, Natalia B. and Wolf, Guy and Krishnaswamy, Smita},
  doi          = {10.1038/s41587-019-0395-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {108},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Visualizing structure and transitions in high-dimensional biological data},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A community effort to create standards for evaluating tumor
subclonal reconstruction. <em>NBT</em>, <em>38</em>(1), 97–107. (<a
href="https://doi.org/10.1038/s41587-019-0364-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor DNA sequencing data can be interpreted by computational methods that analyze genomic heterogeneity to infer evolutionary dynamics. A growing number of studies have used these approaches to link cancer evolution with clinical progression and response to therapy. Although the inference of tumor phylogenies is rapidly becoming standard practice in cancer genome analyses, standards for evaluating them are lacking. To address this need, we systematically assess methods for reconstructing tumor subclonality. First, we elucidate the main algorithmic problems in subclonal reconstruction and develop quantitative metrics for evaluating them. Then we simulate realistic tumor genomes that harbor all known clonal and subclonal mutation types and processes. Finally, we benchmark 580 tumor reconstructions, varying tumor read depth, tumor type and somatic variant detection. Our analysis provides a baseline for the establishment of gold-standard methods to analyze tumor heterogeneity. Methods for reconstructing tumor evolution are benchmarked in the DREAM Somatic Mutation Calling Tumour Heterogeneity Challenge.},
  archive      = {J_NBT},
  author       = {Salcedo, Adriana and Tarabichi, Maxime and Espiritu, Shadrielle Melijah G. and Deshwar, Amit G. and David, Matei and Wilson, Nathan M. and Dentro, Stefan and Wintersinger, Jeff A. and Liu, Lydia Y. and Ko, Minjeong and Sivanandan, Srinivasan and Zhang, Hongjiu and Zhu, Kaiyi and Ou Yang, Tai-Hsien and Chilton, John M. and Buchanan, Alex and Lalansingh, Christopher M. and P’ng, Christine and Anghel, Catalina V. and Umar, Imaad and Lo, Bryan and Zou, William and Simpson, Jared T. and Stuart, Joshua M. and Anastassiou, Dimitris and Guan, Yuanfang and Ewing, Adam D. and Ellrott, Kyle and Wedge, David C. and Morris, Quaid and Van Loo, Peter and Boutros, Paul C.},
  doi          = {10.1038/s41587-019-0364-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {97-107},
  shortjournal = {Nature Biotech.},
  title        = {A community effort to create standards for evaluating tumor subclonal reconstruction},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Large-scale analysis of acquired chromosomal alterations in
non-tumor samples from patients with cancer. <em>NBT</em>,
<em>38</em>(1), 90–96. (<a
href="https://doi.org/10.1038/s41587-019-0297-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mosaicism, the presence of subpopulations of cells bearing somatic mutations, is associated with disease and aging and has been detected in diverse tissues, including apparently normal cells adjacent to tumors. To analyze mosaicism on a large scale, we surveyed haplotype-specific somatic copy number alterations (sCNAs) in 1,708 normal-appearing adjacent-to-tumor (NAT) tissue samples from 27 cancer sites and in 7,149 blood samples from The Cancer Genome Atlas. We find substantial variation across tissues in the rate, burden and types of sCNAs, including those spanning entire chromosome arms. We document matching sCNAs in the NAT tissue and the adjacent tumor, suggesting a shared clonal origin, as well as instances in which both NAT tissue and tumor tissue harbor a gain of the same oncogene arising in parallel from distinct parental haplotypes. These results shed light on pan-tissue mutations characteristic of field cancerization, the presence of oncogenic processes adjacent to cancer cells. A survey of copy number alterations in normal-appearing tissue in cancer samples reveals pan-tissue patterns of mosaicism.},
  archive      = {J_NBT},
  author       = {Jakubek, Y. A. and Chang, K. and Sivakumar, S. and Yu, Y. and Giordano, M. R. and Fowler, J. and Huff, C. D. and Kadara, H. and Vilar, E. and Scheet, P.},
  doi          = {10.1038/s41587-019-0297-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {90-96},
  shortjournal = {Nature Biotech.},
  title        = {Large-scale analysis of acquired chromosomal alterations in non-tumor samples from patients with cancer},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Plant gene editing through de novo induction of meristems.
<em>NBT</em>, <em>38</em>(1), 84–89. (<a
href="https://doi.org/10.1038/s41587-019-0337-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant gene editing is typically performed by delivering reagents such as Cas9 and single guide RNAs to explants in culture. Edited cells are then induced to differentiate into whole plants by exposure to various hormones. The creation of edited plants through tissue culture is often inefficient, time-consuming, works for only limited species and genotypes, and causes unintended changes to the genome and epigenome. Here we report two methods to generate gene-edited dicotyledonous plants through de novo meristem induction. Developmental regulators and gene-editing reagents are delivered to somatic cells of whole plants. This induces meristems that produce shoots with targeted DNA modifications, and gene edits are transmitted to the next generation. The de novo induction of gene-edited meristems sidesteps the need for tissue culture and promises to overcome a bottleneck in plant gene editing. Methods to induce edited somatic plant cells to form meristems circumvent tissue culture and enable genome editing of a wider set of plant species.},
  archive      = {J_NBT},
  author       = {Maher, Michael F. and Nasti, Ryan A. and Vollbrecht, Macy and Starker, Colby G. and Clark, Matthew D. and Voytas, Daniel F.},
  doi          = {10.1038/s41587-019-0337-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {84-89},
  shortjournal = {Nature Biotech.},
  title        = {Plant gene editing through de novo induction of meristems},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Harnessing the intracellular triacylglycerols for titer
improvement of polyketides in streptomyces. <em>NBT</em>,
<em>38</em>(1), 76–83. (<a
href="https://doi.org/10.1038/s41587-019-0335-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pharmaceutically important polyketides such as avermectin are mainly produced as secondary metabolites during the stationary phase of growth of Streptomyces species in fermenters. The source of intracellular metabolites that are funneled into polyketide biosynthesis has proven elusive. We applied multi-omics to reveal that intracellular triacylglycerols (TAGs), which accumulates in primary metabolism, are degraded during stationary phase. This process could channel carbon flux from both intracellular TAGs and extracellular substrates into polyketide biosynthesis. We devised a strategy named ‘dynamic degradation of TAG’ (ddTAG) to mobilize the TAG pool and increase polyketide biosynthesis. Using ddTAG we increased the titers of actinorhodin, jadomycin B, oxytetracycline and avermectin B1a in Streptomyces coelicolor, Streptomyces venezuelae, Streptomyces rimosus and Streptomyces avermitilis. Application of ddTAG increased the titer of avermectin B1a by 50\% to 9.31 g l−1 in a 180-m3 industrial-scale fermentation, which is the highest titer ever reported. Our strategy could improve polyketide titers for pharmaceutical production. Polyketide yields in Streptomyces are boosted by routing stored intracellular triacylglycerol into pathways that make industrially relevant products.},
  archive      = {J_NBT},
  author       = {Wang, Weishan and Li, Shanshan and Li, Zilong and Zhang, Jingyu and Fan, Keqiang and Tan, Gaoyi and Ai, Guomin and Lam, Sin Man and Shui, Guanghou and Yang, Zhiheng and Lu, Hongzhong and Jin, Pinjiao and Li, Yihong and Chen, Xiangyin and Xia, Xuekui and Liu, Xueting and Dannelly, H. Kathleen and Yang, Chen and Yang, Yi and Zhang, Siliang and Alterovitz, Gil and Xiang, Wensheng and Zhang, Lixin},
  doi          = {10.1038/s41587-019-0335-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {76-83},
  shortjournal = {Nature Biotech.},
  title        = {Harnessing the intracellular triacylglycerols for titer improvement of polyketides in streptomyces},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). In situ readout of DNA barcodes and single base edits
facilitated by in vitro transcription. <em>NBT</em>, <em>38</em>(1),
66–75. (<a href="https://doi.org/10.1038/s41587-019-0299-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular barcoding technologies that uniquely identify single cells are hampered by limitations in barcode measurement. Readout by sequencing does not preserve the spatial organization of cells in tissues, whereas imaging methods preserve spatial structure but are less sensitive to barcode sequence. Here we introduce a system for image-based readout of short (20-base-pair) DNA barcodes. In this system, called Zombie, phage RNA polymerases transcribe engineered barcodes in fixed cells. The resulting RNA is subsequently detected by fluorescent in situ hybridization. Using competing match and mismatch probes, Zombie can accurately discriminate single-nucleotide differences in the barcodes. This method allows in situ readout of dense combinatorial barcode libraries and single-base mutations produced by CRISPR base editors without requiring barcode expression in live cells. Zombie functions across diverse contexts, including cell culture, chick embryos and adult mouse brain tissue. The ability to sensitively read out compact and diverse DNA barcodes by imaging will facilitate a broad range of barcoding and genomic recording strategies. The spatial location and sequence of DNA barcodes are detected with high sensitivity in fixed tissues.},
  archive      = {J_NBT},
  author       = {Askary, Amjad and Sanchez-Guardado, Luis and Linton, James M. and Chadly, Duncan M. and Budde, Mark W. and Cai, Long and Lois, Carlos and Elowitz, Michael B.},
  doi          = {10.1038/s41587-019-0299-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {66-75},
  shortjournal = {Nature Biotech.},
  title        = {In situ readout of DNA barcodes and single base edits facilitated by in vitro transcription},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Deciphering eukaryotic gene-regulatory logic with 100
million random promoters. <em>NBT</em>, <em>38</em>(1), 56–65. (<a
href="https://doi.org/10.1038/s41587-019-0315-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How transcription factors (TFs) interpret cis-regulatory DNA sequence to control gene expression remains unclear, largely because past studies using native and engineered sequences had insufficient scale. Here, we measure the expression output of &amp;gt;100 million synthetic yeast promoter sequences that are fully random. These sequences yield diverse, reproducible expression levels that can be explained by their chance inclusion of functional TF binding sites. We use machine learning to build interpretable models of transcriptional regulation that predict ~94\% of the expression driven from independent test promoters and ~89\% of the expression driven from native yeast promoter fragments. These models allow us to characterize each TF’s specificity, activity and interactions with chromatin. TF activity depends on binding-site strand, position, DNA helical face and chromatin context. Notably, expression level is influenced by weak regulatory interactions, which confound designed-sequence studies. Our analyses show that massive-throughput assays of fully random DNA can provide the big data necessary to develop complex, predictive models of gene regulation. Gene expression levels in yeast are predicted using a massive dataset on promoters with random sequences.},
  archive      = {J_NBT},
  author       = {de Boer, Carl G. and Vaishnav, Eeshit Dhaval and Sadeh, Ronen and Abeyta, Esteban Luis and Friedman, Nir and Regev, Aviv},
  doi          = {10.1038/s41587-019-0315-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {56-65},
  shortjournal = {Nature Biotech.},
  title        = {Deciphering eukaryotic gene-regulatory logic with 100 million random promoters},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Dose-dependent activation of gene expression is achieved
using CRISPR and small molecules that recruit endogenous chromatin
machinery. <em>NBT</em>, <em>38</em>(1), 50–55. (<a
href="https://doi.org/10.1038/s41587-019-0296-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression can be activated or suppressed using CRISPR­–Cas9 systems. However, tools that enable dose-dependent activation of gene expression without the use of exogenous transcription regulatory proteins are lacking. Here we describe chemical epigenetic modifiers (CEMs) designed to activate the expression of target genes by recruiting components of the endogenous chromatin-activating machinery, eliminating the need for exogenous transcriptional activators. The system has two parts: catalytically inactive Cas9 (dCas9) in complex with FK506-binding protein (FKBP) and a CEM consisting of FK506 linked to a molecule that interacts with cellular epigenetic machinery. We show that CEMs upregulate gene expression at target endogenous loci up to 20-fold or more depending on the gene. We also demonstrate dose-dependent control of transcriptional activation, function across multiple diverse genes, reversibility of CEM activity and specificity of our best-in-class CEM across the genome. Activation of gene expression with chemical epigenetic modifiers},
  archive      = {J_NBT},
  author       = {Chiarella, Anna M. and Butler, Kyle V. and Gryder, Berkley E. and Lu, Dongbo and Wang, Tiffany A. and Yu, Xufen and Pomella, Silvia and Khan, Javed and Jin, Jian and Hathaway, Nathaniel A.},
  doi          = {10.1038/s41587-019-0296-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {50-55},
  shortjournal = {Nature Biotech.},
  title        = {Dose-dependent activation of gene expression is achieved using CRISPR and small molecules that recruit endogenous chromatin machinery},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Polymer-stabilized cas9 nanoparticles and modified repair
templates increase genome editing efficiency. <em>NBT</em>,
<em>38</em>(1), 44–49. (<a
href="https://doi.org/10.1038/s41587-019-0325-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Versatile and precise genome modifications are needed to create a wider range of adoptive cellular therapies1–5. Here we report two improvements that increase the efficiency of CRISPR–Cas9-based genome editing in clinically relevant primary cell types. Truncated Cas9 target sequences (tCTSs) added at the ends of the homology-directed repair (HDR) template interact with Cas9 ribonucleoproteins (RNPs) to shuttle the template to the nucleus, enhancing HDR efficiency approximately two- to fourfold. Furthermore, stabilizing Cas9 RNPs into nanoparticles with polyglutamic acid further improves editing efficiency by approximately twofold, reduces toxicity, and enables lyophilized storage without loss of activity. Combining the two improvements increases gene targeting efficiency even at reduced HDR template doses, yielding approximately two to six times as many viable edited cells across multiple genomic loci in diverse cell types, such as bulk (CD3+) T cells, CD8+ T cells, CD4+ T cells, regulatory T cells (Tregs), γδ T cells, B cells, natural killer cells, and primary and induced pluripotent stem cell-derived6 hematopoietic stem progenitor cells (HSPCs). Precise genome editing is made more efficient by stabilizing Cas9 and enhancing shuttling to the nucleus.},
  archive      = {J_NBT},
  author       = {Nguyen, David N. and Roth, Theodore L. and Li, P. Jonathan and Chen, Peixin Amy and Apathy, Ryan and Mamedov, Murad R. and Vo, Linda T. and Tobin, Victoria R. and Goodman, Daniel and Shifrut, Eric and Bluestone, Jeffrey A. and Puck, Jennifer M. and Szoka, Francis C. and Marson, Alexander},
  doi          = {10.1038/s41587-019-0325-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {44-49},
  shortjournal = {Nature Biotech.},
  title        = {Polymer-stabilized cas9 nanoparticles and modified repair templates increase genome editing efficiency},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A DNA-of-things storage architecture to create materials
with embedded memory. <em>NBT</em>, <em>38</em>(1), 39–43. (<a
href="https://doi.org/10.1038/s41587-019-0356-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA storage offers substantial information density1–7 and exceptional half-life3. We devised a ‘DNA-of-things’ (DoT) storage architecture to produce materials with immutable memory. In a DoT framework, DNA molecules record the data, and these molecules are then encapsulated in nanometer silica beads8, which are fused into various materials that are used to print or cast objects in any shape. First, we applied DoT to three-dimensionally print a Stanford Bunny9 that contained a 45 kB digital DNA blueprint for its synthesis. We synthesized five generations of the bunny, each from the memory of the previous generation without additional DNA synthesis or degradation of information. To test the scalability of DoT, we stored a 1.4 MB video in DNA in plexiglass spectacle lenses and retrieved it by excising a tiny piece of the plexiglass and sequencing the embedded DNA. DoT could be applied to store electronic health records in medical implants, to hide data in everyday objects (steganography) and to manufacture objects containing their own blueprint. It may also facilitate the development of self-replicating machines. A DNA-based method for embedding data in materials enables the conversion of everyday objects into data storage devices.},
  archive      = {J_NBT},
  author       = {Koch, Julian and Gantenbein, Silvan and Masania, Kunal and Stark, Wendelin J. and Erlich, Yaniv and Grass, Robert N.},
  doi          = {10.1038/s41587-019-0356-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {39-43},
  shortjournal = {Nature Biotech.},
  title        = {A DNA-of-things storage architecture to create materials with embedded memory},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Highly multiplexed single-cell RNA-seq by DNA
oligonucleotide tagging of cellular proteins. <em>NBT</em>,
<em>38</em>(1), 35–38. (<a
href="https://doi.org/10.1038/s41587-019-0372-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe a universal sample multiplexing method for single-cell RNA sequencing in which fixed cells are chemically labeled by attaching identifying DNA oligonucleotides to cellular proteins. Analysis of a 96-plex perturbation experiment revealed changes in cell population structure and transcriptional states that cannot be discerned from bulk measurements, establishing an efficient method for surveying cell populations from large experiments or clinical samples with the depth and resolution of single-cell RNA sequencing. Single-cell RNA sequencing is readily multiplexed by labeling cells with ClickTags.},
  archive      = {J_NBT},
  author       = {Gehring, Jase and Hwee Park, Jong and Chen, Sisi and Thomson, Matthew and Pachter, Lior},
  doi          = {10.1038/s41587-019-0372-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {35-38},
  shortjournal = {Nature Biotech.},
  title        = {Highly multiplexed single-cell RNA-seq by DNA oligonucleotide tagging of cellular proteins},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Fast-track for engineered plants. <em>NBT</em>,
<em>38</em>(1), 32–34. (<a
href="https://doi.org/10.1038/s41587-019-0371-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A rapid method to introduce gene edits advances plant science.},
  archive      = {J_NBT},
  author       = {Dong, Oliver Xiaoou and Ronald, Pamela C.},
  doi          = {10.1038/s41587-019-0371-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {32-34},
  shortjournal = {Nature Biotech.},
  title        = {Fast-track for engineered plants},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). DNA storage in everyday objects. <em>NBT</em>,
<em>38</em>(1), 31–32. (<a
href="https://doi.org/10.1038/s41587-019-0376-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Information encoded in DNA can be stored in, and retrieved from, everyday objects.},
  archive      = {J_NBT},
  author       = {Farzadfard, Fahim},
  doi          = {10.1038/s41587-019-0376-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {31-32},
  shortjournal = {Nature Biotech.},
  title        = {DNA storage in everyday objects},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Antibody engineering. <em>NBT</em>, <em>38</em>(1), 30. (<a
href="https://doi.org/10.1038/s41587-019-0383-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents related to methods of identifying protein modulators and treating influenza A virus.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0383-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nature Biotech.},
  title        = {Antibody engineering},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Analysis of the standard for accepting supplementary
experimental data in china’s patent practice. <em>NBT</em>,
<em>38</em>(1), 27–29. (<a
href="https://doi.org/10.1038/s41587-019-0365-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Balancing the interests of the applicants and the public, China has imposed strict restrictions on the examination of supplementary experimental data.},
  archive      = {J_NBT},
  author       = {Li, Wei},
  doi          = {10.1038/s41587-019-0365-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {27-29},
  shortjournal = {Nature Biotech.},
  title        = {Analysis of the standard for accepting supplementary experimental data in china’s patent practice},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Mass spectrometry searches using MASST. <em>NBT</em>,
<em>38</em>(1), 23–26. (<a
href="https://doi.org/10.1038/s41587-019-0375-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Wang, Mingxun and Jarmusch, Alan K. and Vargas, Fernando and Aksenov, Alexander A. and Gauglitz, Julia M. and Weldon, Kelly and Petras, Daniel and da Silva, Ricardo and Quinn, Robert and Melnik, Alexey V. and van der Hooft, Justin J. J. and Caraballo-Rodríguez, Andrés Mauricio and Nothias, Louis Felix and Aceves, Christine M. and Panitchpakdi, Morgan and Brown, Elizabeth and Di Ottavio, Francesca and Sikora, Nicole and Elijah, Emmanuel O. and Labarta-Bajo, Lara and Gentry, Emily C. and Shalapour, Shabnam and Kyle, Kathleen E. and Puckett, Sara P. and Watrous, Jeramie D. and Carpenter, Carolina S. and Bouslimani, Amina and Ernst, Madeleine and Swafford, Austin D. and Zúñiga, Elina I. and Balunas, Marcy J. and Klassen, Jonathan L. and Loomba, Rohit and Knight, Rob and Bandeira, Nuno and Dorrestein, Pieter C.},
  doi          = {10.1038/s41587-019-0375-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {23-26},
  shortjournal = {Nature Biotech.},
  title        = {Mass spectrometry searches using MASST},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Assessing progress in biotech’s gender balance.
<em>NBT</em>, <em>38</em>(1), 19–22. (<a
href="https://doi.org/10.1038/s41587-019-0367-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-019-0367-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {19-22},
  shortjournal = {Nature Biotech.},
  title        = {Assessing progress in biotech’s gender balance},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Integrating scientific considerations into r&amp;d project
valuation. <em>NBT</em>, <em>38</em>(1), 14–18. (<a
href="https://doi.org/10.1038/s41587-019-0358-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Traditional valuation approaches rarely take into account scientific considerations specific to a R&amp;amp;D project. A question-based approach using real options offers a solution.},
  archive      = {J_NBT},
  author       = {de Visser, Saco J. and Cohen, Adam F. and Kenter, Marcel J. H.},
  doi          = {10.1038/s41587-019-0358-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {14-18},
  shortjournal = {Nature Biotech.},
  title        = {Integrating scientific considerations into R&amp;D project valuation},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020l). Around the world in a month. <em>NBT</em>, <em>38</em>(1),
8. (<a href="https://doi.org/10.1038/s41587-019-0378-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0378-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020g). Grail of RAS cancer drugs within reach. <em>NBT</em>,
<em>38</em>(1), 6–8. (<a
href="https://doi.org/10.1038/s41587-019-0382-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first KRAS inhibitors to reach the clinic show preliminary but promising safety and efficacy.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-019-0382-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {6-8},
  shortjournal = {Nature Biotech.},
  title        = {Grail of RAS cancer drugs within reach},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). First ebola vaccine approved. <em>NBT</em>, <em>38</em>(1),
6. (<a href="https://doi.org/10.1038/s41587-019-0385-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0385-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {6},
  shortjournal = {Nature Biotech.},
  title        = {First ebola vaccine approved},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Bayer makes first move into condensates. <em>NBT</em>,
<em>38</em>(1), 5. (<a
href="https://doi.org/10.1038/s41587-019-0386-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0386-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {5},
  shortjournal = {Nature Biotech.},
  title        = {Bayer makes first move into condensates},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). RNAi scores big wins. <em>NBT</em>, <em>38</em>(1), 4. (<a
href="https://doi.org/10.1038/s41587-019-0384-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0384-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nature Biotech.},
  title        = {RNAi scores big wins},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Immunotherapy takes aim at exhausted t cells. <em>NBT</em>,
<em>38</em>(1), 3–5. (<a
href="https://doi.org/10.1038/s41587-019-0381-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Startups attempt to revive exhausted T cells to fight solid tumors, but must strike a balance to avoid triggering adverse events.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-019-0381-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {3-5},
  shortjournal = {Nature Biotech.},
  title        = {Immunotherapy takes aim at exhausted t cells},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). When the world needs to know. <em>NBT</em>, <em>38</em>(1),
1. (<a href="https://doi.org/10.1038/s41587-019-0396-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Was the public re-release of data excerpts from an unpublished manuscript on human germline gene editing ethically justified?},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-019-0396-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nature Biotech.},
  title        = {When the world needs to know},
  volume       = {38},
  year         = {2020},
}
</textarea>
</details></li>
</ul>

</body>
</html>
