<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt---34">NBT - 34</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(3), 452. (<a
href="https://doi.org/10.1038/s41587-025-02588-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02588-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {452},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 2H24 biotech job picture. <em>NBT</em>, <em>43</em>(3), 451.
(<a href="https://doi.org/10.1038/s41587-025-02586-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A semiannual snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-025-02586-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {451},
  shortjournal = {Nature Biotech.},
  title        = {2H24 biotech job picture},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An open letter to graduate students and other
procrastinators: It’s time to write. <em>NBT</em>, <em>43</em>(3),
447–450. (<a href="https://doi.org/10.1038/s41587-025-02584-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hazelett, Dennis J.},
  doi          = {10.1038/s41587-025-02584-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {447-450},
  shortjournal = {Nature Biotech.},
  title        = {An open letter to graduate students and other procrastinators: It’s time to write},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Identification of clinically relevant t
cell receptors for personalized t cell therapy using combinatorial
algorithms. <em>NBT</em>, <em>43</em>(3), 445. (<a
href="https://doi.org/10.1038/s41587-024-02318-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Pétremand, Rémy and Chiffelle, Johanna and Bobisse, Sara and Perez, Marta A. S. and Schmidt, Julien and Arnaud, Marion and Barras, David and Lozano-Rabella, Maria and Genolet, Raphael and Sauvage, Christophe and Saugy, Damien and Michel, Alexandra and Huguenin-Bergenat, Anne-Laure and Capt, Charlotte and Moore, Jonathan S. and De Vito, Claudio and Labidi-Galy, S. Intidhar and Kandalaft, Lana E. and Dangaj Laniti, Denarda and Bassani-Sternberg, Michal and Oliveira, Giacomo and Wu, Catherine J. and Coukos, George and Zoete, Vincent and Harari, Alexandre},
  doi          = {10.1038/s41587-024-02318-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {445},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Computational scoring and experimental
evaluation of enzymes generated by neural networks. <em>NBT</em>,
<em>43</em>(3), 444. (<a
href="https://doi.org/10.1038/s41587-024-02275-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Johnson, Sean R. and Fu, Xiaozhi and Viknander, Sandra and Goldin, Clara and Monaco, Sarah and Zelezniak, Aleksej and Yang, Kevin K.},
  doi          = {10.1038/s41587-024-02275-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {444},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Computational scoring and experimental evaluation of enzymes generated by neural networks},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Bioinstructive implantable scaffolds for
rapid in vivo manufacture and release of CAR-t cells. <em>NBT</em>,
<em>43</em>(3), 443. (<a
href="https://doi.org/10.1038/s41587-025-02572-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Agarwalla, Pritha and Ogunnaike, Edikan A. and Ahn, Sarah and Froehlich, Kristen A. and Jansson, Anton and Ligler, Frances S. and Dotti, Gianpietro and Brudno, Yevgeny},
  doi          = {10.1038/s41587-025-02572-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {443},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Bioinstructive implantable scaffolds for rapid in vivo manufacture and release of CAR-T cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis and benchmarking of small and large genomic
variants across tandem repeats. <em>NBT</em>, <em>43</em>(3), 431–442.
(<a href="https://doi.org/10.1038/s41587-024-02225-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tandem repeats (TRs) are highly polymorphic in the human genome, have thousands of associated molecular traits and are linked to over 60 disease phenotypes. However, they are often excluded from at-scale studies because of challenges with variant calling and representation, as well as a lack of a genome-wide standard. Here, to promote the development of TR methods, we created a catalog of TR regions and explored TR properties across 86 haplotype-resolved long-read human assemblies. We curated variants from the Genome in a Bottle (GIAB) HG002 individual to create a TR dataset to benchmark existing and future TR analysis methods. We also present an improved variant comparison method that handles variants greater than 4 bp in length and varying allelic representation. The 8.1% of the genome covered by the TR catalog holds ~24.9% of variants per individual, including 124,728 small and 17,988 large variants for the GIAB HG002 ‘truth-set’ TR benchmark. We demonstrate the utility of this pipeline across short-read and long-read technologies. A truth-set catalog of human tandem repeats allows benchmarking of variant analysis tools.},
  archive      = {J_NBT},
  author       = {English, Adam C. and Dolzhenko, Egor and Ziaei Jam, Helyaneh and McKenzie, Sean K. and Olson, Nathan D. and De Coster, Wouter and Park, Jonghun and Gu, Bida and Wagner, Justin and Eberle, Michael A. and Gymrek, Melissa and Chaisson, Mark J. P. and Zook, Justin M. and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41587-024-02225-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {431-442},
  shortjournal = {Nature Biotech.},
  title        = {Analysis and benchmarking of small and large genomic variants across tandem repeats},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CRISPRi-based circuits to control gene expression in plants.
<em>NBT</em>, <em>43</em>(3), 416–430. (<a
href="https://doi.org/10.1038/s41587-024-02236-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The construction of synthetic gene circuits in plants has been limited by a lack of orthogonal and modular parts. Here, we implement a CRISPR (clustered regularly interspaced short palindromic repeats) interference (CRISPRi)-based reversible gene circuit platform in plants. We create a toolkit of engineered repressible promoters of different strengths and construct NOT and NOR gates in Arabidopsis thaliana protoplasts. We determine the optimal processing system to express single guide RNAs from RNA Pol II promoters to introduce NOR gate programmability for interfacing with host regulatory sequences. The performance of a NOR gate in stably transformed Arabidopsis plants demonstrates the system’s programmability and reversibility in a complex multicellular organism. Furthermore, cross-species activity of CRISPRi-based logic gates is shown in Physcomitrium patens, Triticum aestivum and Brassica napus protoplasts. Layering multiple NOR gates together creates OR, NIMPLY and AND logic functions, highlighting the modularity of our system. Our CRISPRi circuits are orthogonal, compact, reversible, programmable and modular and provide a platform for sophisticated spatiotemporal control of gene expression in plants. Programmable and reversible CRISPRi-based genetic circuits function in a variety of plants.},
  archive      = {J_NBT},
  author       = {Khan, Muhammad Adil and Herring, Gabrielle and Zhu, Jia Yuan and Oliva, Marina and Fourie, Elliott and Johnston, Benjamin and Zhang, Zhining and Potter, Jarred and Pineda, Luke and Pflueger, Jahnvi and Swain, Tessa and Pflueger, Christian and Lloyd, James P. B. and Secco, David and Small, Ian and Kidd, Brendan N. and Lister, Ryan},
  doi          = {10.1038/s41587-024-02236-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {416-430},
  shortjournal = {Nature Biotech.},
  title        = {CRISPRi-based circuits to control gene expression in plants},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decrypting the molecular basis of cellular drug phenotypes
by dose-resolved expression proteomics. <em>NBT</em>, <em>43</em>(3),
406–415. (<a href="https://doi.org/10.1038/s41587-024-02218-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteomics is making important contributions to drug discovery, from target deconvolution to mechanism of action (MoA) elucidation and the identification of biomarkers of drug response. Here we introduce decryptE, a proteome-wide approach that measures the full dose–response characteristics of drug-induced protein expression changes that informs cellular drug MoA. Assaying 144 clinical drugs and research compounds against 8,000 proteins resulted in more than 1 million dose–response curves that can be interactively explored online in ProteomicsDB and a custom-built Shiny App. Analysis of the collective data provided molecular explanations for known phenotypic drug effects and uncovered new aspects of the MoA of human medicines. We found that histone deacetylase inhibitors potently and strongly down-regulated the T cell receptor complex resulting in impaired human T cell activation in vitro and ex vivo. This offers a rational explanation for the efficacy of histone deacetylase inhibitors in certain lymphomas and autoimmune diseases and explains their poor performance in treating solid tumors. Proteome-wide measurements of dose–response curves for 144 drugs reveal drug mechanisms of action.},
  archive      = {J_NBT},
  author       = {Eckert, Stephan and Berner, Nicola and Kramer, Karl and Schneider, Annika and Müller, Julian and Lechner, Severin and Brajkovic, Sarah and Sakhteman, Amirhossein and Graetz, Christian and Fackler, Jonas and Dudek, Michael and Pfaffl, Michael W. and Knolle, Percy and Wilhelm, Stephanie and Kuster, Bernhard},
  doi          = {10.1038/s41587-024-02218-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {406-415},
  shortjournal = {Nature Biotech.},
  title        = {Decrypting the molecular basis of cellular drug phenotypes by dose-resolved expression proteomics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Computational scoring and experimental evaluation of
enzymes generated by neural networks. <em>NBT</em>, <em>43</em>(3),
396–405. (<a href="https://doi.org/10.1038/s41587-024-02214-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, generative protein sequence models have been developed to sample novel sequences. However, predicting whether generated proteins will fold and function remains challenging. We evaluate a set of 20 diverse computational metrics to assess the quality of enzyme sequences produced by three contrasting generative models: ancestral sequence reconstruction, a generative adversarial network and a protein language model. Focusing on two enzyme families, we expressed and purified over 500 natural and generated sequences with 70–90% identity to the most similar natural sequences to benchmark computational metrics for predicting in vitro enzyme activity. Over three rounds of experiments, we developed a computational filter that improved the rate of experimental success by 50–150%. The proposed metrics and models will drive protein engineering research by serving as a benchmark for generative protein sequence models and helping to select active variants for experimental testing. Metrics that predict the success of folding and the activity of designed protein sequences are developed and experimentally validated.},
  archive      = {J_NBT},
  author       = {Johnson, Sean R. and Fu, Xiaozhi and Viknander, Sandra and Goldin, Clara and Monaco, Sarah and Zelezniak, Aleksej and Yang, Kevin K.},
  doi          = {10.1038/s41587-024-02214-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {396-405},
  shortjournal = {Nature Biotech.},
  title        = {Computational scoring and experimental evaluation of enzymes generated by neural networks},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping variant effects on anti-tumor hallmarks of primary
human t cells with base-editing screens. <em>NBT</em>, <em>43</em>(3),
384–395. (<a href="https://doi.org/10.1038/s41587-024-02235-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nucleotide variants (SNVs) in key T cell genes can drive clinical pathologies and could be repurposed to improve cellular cancer immunotherapies. Here, we perform massively parallel base-editing screens to generate thousands of variants at gene loci annotated with known or potential clinical relevance. We discover a broad landscape of putative gain-of-function (GOF) and loss-of-function (LOF) mutations, including in PIK3CD and the gene encoding its regulatory subunit, PIK3R1, LCK, SOS1, AKT1 and RHOA. Base editing of PIK3CD and PIK3R1 variants in T cells with an engineered T cell receptor specific to a melanoma epitope or in different generations of CD19 chimeric antigen receptor (CAR) T cells demonstrates that discovered GOF variants, but not LOF or silent mutation controls, enhanced signaling, cytokine production and lysis of cognate melanoma and leukemia cell models, respectively. Additionally, we show that generations of CD19 CAR T cells engineered with PIK3CD GOF mutations demonstrate enhanced antigen-specific signaling, cytokine production and leukemia cell killing, including when benchmarked against other recent strategies. Base-editing screens of primary T cells reveal genetic variants beneficial to immunotherapies.},
  archive      = {J_NBT},
  author       = {Walsh, Zachary H. and Shah, Parin and Kothapalli, Neeharika and Shah, Shivem B. and Nikolenyi, Gergo and Brodtman, D. Zack and Leuzzi, Giuseppe and Rogava, Meri and Mu, Michael and Ho, Patricia and Abuzaid, Sinan and Vasan, Neil and AlQuraishi, Mohammed and Milner, Joshua D. and Ciccia, Alberto and Melms, Johannes C. and Izar, Benjamin},
  doi          = {10.1038/s41587-024-02235-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {384-395},
  shortjournal = {Nature Biotech.},
  title        = {Mapping variant effects on anti-tumor hallmarks of primary human t cells with base-editing screens},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered CRISPR-cas12a for higher-order combinatorial
chromatin perturbations. <em>NBT</em>, <em>43</em>(3), 369–383. (<a
href="https://doi.org/10.1038/s41587-024-02224-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to double-stranded DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays. However, current CRISPRi pooled screening approaches are limited to targeting one to three genomic sites per cell. We engineer an Acidaminococcus Cas12a (AsCas12a) variant, multiplexed transcriptional interference AsCas12a (multiAsCas12a), that incorporates R1226A, a mutation that stabilizes the ribonucleoprotein–DNA complex via DNA nicking. The multiAsCas12a-KRAB fusion improves CRISPRi activity over DNase-dead AsCas12a-KRAB fusions, often rescuing the activities of lentivirally delivered CRISPR RNAs (crRNA) that are inactive when used with the latter. multiAsCas12a-KRAB supports CRISPRi using 6-plex crRNA arrays in high-throughput pooled screens. Using multiAsCas12a-KRAB, we discover enhancer elements and dissect the combinatorial function of cis-regulatory elements in human cells. These results instantiate a group testing framework for efficiently surveying numerous combinations of chromatin perturbations for biological discovery and engineering. An engineered Cas12a enables higher-order combinatorial functional genomic screens using CRISPR interference.},
  archive      = {J_NBT},
  author       = {Hsiung, C. C.-S. and Wilson, C. M. and Sambold, N. A. and Dai, R. and Chen, Q. and Teyssier, N. and Misiukiewicz, S. and Arab, A. and O’Loughlin, T. and Cofsky, J. C. and Shi, J. and Gilbert, L. A.},
  doi          = {10.1038/s41587-024-02224-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {369-383},
  shortjournal = {Nature Biotech.},
  title        = {Engineered CRISPR-cas12a for higher-order combinatorial chromatin perturbations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bidirectional epigenetic editing reveals hierarchies in gene
regulation. <em>NBT</em>, <em>43</em>(3), 355–368. (<a
href="https://doi.org/10.1038/s41587-024-02213-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR perturbation methods are limited in their ability to study non-coding elements and genetic interactions. In this study, we developed a system for bidirectional epigenetic editing, called CRISPRai, in which we apply activating (CRISPRa) and repressive (CRISPRi) perturbations to two loci simultaneously in the same cell. We developed CRISPRai Perturb-seq by coupling dual perturbation gRNA detection with single-cell RNA sequencing, enabling study of pooled perturbations in a mixed single-cell population. We applied this platform to study the genetic interaction between two hematopoietic lineage transcription factors, SPI1 and GATA1, and discovered novel characteristics of their co-regulation on downstream target genes, including differences in SPI1 and GATA1 occupancy at genes that are regulated through different modes. We also studied the regulatory landscape of IL2 (interleukin-2) in Jurkat T cells, primary T cells and chimeric antigen receptor (CAR) T cells and elucidated mechanisms of enhancer-mediated IL2 gene regulation. CRISPRai facilitates investigation of context-specific genetic interactions, provides new insights into gene regulation and will enable exploration of non-coding disease-associated variants. CRISPRai simultaneously activates and represses two genes in single cells.},
  archive      = {J_NBT},
  author       = {Pacalin, Naomi M. and Steinhart, Zachary and Shi, Quanming and Belk, Julia A. and Dorovskyi, Dmytro and Kraft, Katerina and Parker, Kevin R. and Shy, Brian R. and Marson, Alexander and Chang, Howard Y.},
  doi          = {10.1038/s41587-024-02213-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {355-368},
  shortjournal = {Nature Biotech.},
  title        = {Bidirectional epigenetic editing reveals hierarchies in gene regulation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Self-pigmenting textiles grown from cellulose-producing
bacteria with engineered tyrosinase expression. <em>NBT</em>,
<em>43</em>(3), 345–354. (<a
href="https://doi.org/10.1038/s41587-024-02194-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Environmental concerns are driving interest in postpetroleum synthetic textiles produced from microbial and fungal sources. Bacterial cellulose (BC) is a promising sustainable leather alternative, on account of its material properties, low infrastructure needs and biodegradability. However, for alternative textiles like BC to be fully sustainable, alternative ways to dye textiles need to be developed alongside alternative production methods. To address this, we genetically engineer Komagataeibacter rhaeticus to create a bacterial strain that grows self-pigmenting BC. Melanin biosynthesis in the bacteria from recombinant tyrosinase expression achieves dark black coloration robust to material use. Melanated BC production can be scaled up for the construction of prototype fashion products, and we illustrate the potential of combining engineered self-pigmentation with tools from synthetic biology, through the optogenetic patterning of gene expression in cellulose-producing bacteria. With this study, we demonstrate that combining genetic engineering with current and future methods of textile biofabrication has the potential to create a new class of textiles. Genetic engineering of material-producing bacteria creates pigmented products.},
  archive      = {J_NBT},
  author       = {Walker, Kenneth T. and Li, Ivy S. and Keane, Jennifer and Goosens, Vivianne J. and Song, Wenzhe and Lee, Koon-Yang and Ellis, Tom},
  doi          = {10.1038/s41587-024-02194-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {345-354},
  shortjournal = {Nature Biotech.},
  title        = {Self-pigmenting textiles grown from cellulose-producing bacteria with engineered tyrosinase expression},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Generation of allogeneic CAR-NKT cells from hematopoietic
stem and progenitor cells using a clinically guided culture method.
<em>NBT</em>, <em>43</em>(3), 329–344. (<a
href="https://doi.org/10.1038/s41587-024-02226-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer immunotherapy with autologous chimeric antigen receptor (CAR) T cells faces challenges in manufacturing and patient selection that could be avoided by using ‘off-the-shelf’ products, such as allogeneic CAR natural killer T (AlloCAR-NKT) cells. Previously, we reported a system for differentiating human hematopoietic stem and progenitor cells into AlloCAR-NKT cells, but the use of three-dimensional culture and xenogeneic feeders precluded its clinical application. Here we describe a clinically guided method to differentiate and expand IL-15-enhanced AlloCAR-NKT cells with high yield and purity. We generated AlloCAR-NKT cells targeting seven cancers and, in a multiple myeloma model, demonstrated their antitumor efficacy, expansion and persistence. The cells also selectively depleted immunosuppressive cells in the tumor microenviroment and antagonized tumor immune evasion via triple targeting of CAR, TCR and NK receptors. They exhibited a stable hypoimmunogenic phenotype associated with epigenetic and signaling regulation and did not induce detectable graft versus host disease or cytokine release syndrome. These properties of AlloCAR-NKT cells support their potential for clinical translation. Off-the-shelf CAR-NKT cells for cancer immunotherapy are advanced toward clinical translation.},
  archive      = {J_NBT},
  author       = {Li, Yan-Ruide and Zhou, Yang and Yu, Jiaji and Kim, Yu Jeong and Li, Miao and Lee, Derek and Zhou, Kuangyi and Chen, Yuning and Zhu, Yichen and Wang, Yu-Chen and Li, Zhe and Yu, Yanqi and Dunn, Zachary Spencer and Guo, Wenbin and Cen, Xinjian and Husman, Tiffany and Bajpai, Aarushi and Kramer, Adam and Wilson, Matthew and Fang, Ying and Huang, Jie and Li, Shuo and Zhou, Yonggang and Zhang, Yuchong and Hahn, Zoe and Zhu, Enbo and Ma, Feiyang and Pan, Calvin and Lusis, Aldons J. and Zhou, Jin J. and Seet, Christopher S. and Kohn, Donald B. and Wang, Pin and Zhou, Xianghong Jasmine and Pellegrini, Matteo and Puliafito, Benjamin R. and Larson, Sarah M. and Yang, Lili},
  doi          = {10.1038/s41587-024-02226-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {329-344},
  shortjournal = {Nature Biotech.},
  title        = {Generation of allogeneic CAR-NKT cells from hematopoietic stem and progenitor cells using a clinically guided culture method},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Identification of clinically relevant t cell receptors for
personalized t cell therapy using combinatorial algorithms.
<em>NBT</em>, <em>43</em>(3), 323–328. (<a
href="https://doi.org/10.1038/s41587-024-02232-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central challenge in developing personalized cancer cell immunotherapy is the identification of tumor-reactive T cell receptors (TCRs). By exploiting the distinct transcriptomic profile of tumor-reactive T cells relative to bystander cells, we build and benchmark TRTpred, an antigen-agnostic in silico predictor of tumor-reactive TCRs. We integrate TRTpred with an avidity predictor to derive a combinatorial algorithm of clinically relevant TCRs for personalized T cell therapy and benchmark it in patient-derived xenografts. Tumor-reactive T cell receptors are predicted based on single-cell RNA sequencing and single-cell T cell receptor sequencing.},
  archive      = {J_NBT},
  author       = {Pétremand, Rémy and Chiffelle, Johanna and Bobisse, Sara and Perez, Marta A. S. and Schmidt, Julien and Arnaud, Marion and Barras, David and Lozano-Rabella, Maria and Genolet, Raphael and Sauvage, Christophe and Saugy, Damien and Michel, Alexandra and Huguenin-Bergenat, Anne-Laure and Capt, Charlotte and Moore, Jonathan S. and De Vito, Claudio and Labidi-Galy, S. Intidhar and Kandalaft, Lana E. and Dangaj Laniti, Denarda and Bassani-Sternberg, Michal and Oliveira, Giacomo and Wu, Catherine J. and Coukos, George and Zoete, Vincent and Harari, Alexandre},
  doi          = {10.1038/s41587-024-02232-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {323-328},
  shortjournal = {Nature Biotech.},
  title        = {Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward single-molecule protein sequencing using nanopores.
<em>NBT</em>, <em>43</em>(3), 312–322. (<a
href="https://doi.org/10.1038/s41587-025-02587-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past three decades, biological nanopore sequencing has grown from a research curiosity to a mature technology to sequence nucleic acids at the single-molecule level. Now, recent achievements suggest that nanopores might be able to sequence proteins soon. In this Perspective, we analyze the different approaches that have been proposed to measure proteins and peptides using nanopores. We predict that, more likely than not, nanopores will be capable of identifying full-length proteins at the single-molecule level and with single-amino acid resolution, paving the way to single-molecule protein sequencing. This would allow several applications in proteomics that are at present challenging, including measuring the heterogeneity of post-translational modifications, quantifying low-abundance proteins and characterizing protein splicing. Maglia and colleagues discuss advances in nanopore technology en route to single-molecule protein sequencing},
  archive      = {J_NBT},
  author       = {Lu, Chunzhe and Bonini, Andrea and Viel, Jakob H. and Maglia, Giovanni},
  doi          = {10.1038/s41587-025-02587-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {312-322},
  shortjournal = {Nature Biotech.},
  title        = {Toward single-molecule protein sequencing using nanopores},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing programmable gene expression in plants using
CRISPRi-based boolean gates. <em>NBT</em>, <em>43</em>(3), 310–311. (<a
href="https://doi.org/10.1038/s41587-024-02273-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To advance the toolset for controlling plant gene expression, we developed a CRISPR interference-based platform for the construction of synthetic Boolean logic gates that is functional in multiple plant species. These genetic circuits are programmable and reversible in nature, which will enable spatiotemporal control of plant responses to dynamic cues.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02273-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {310-311},
  shortjournal = {Nature Biotech.},
  title        = {Advancing programmable gene expression in plants using CRISPRi-based boolean gates},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Assessing the laboratory performance of AI-generated
enzymes. <em>NBT</em>, <em>43</em>(3), 308–309. (<a
href="https://doi.org/10.1038/s41587-024-02239-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A set of 20 computational metrics was evaluated to determine whether they could predict the functionality of synthetic enzyme sequences produced by generative protein models, resulting in the development of a computational filter, COMPSS, that increased experimental success rates by 50–150%, tested in over 500 natural and AI-generated enzymes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02239-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {308-309},
  shortjournal = {Nature Biotech.},
  title        = {Assessing the laboratory performance of AI-generated enzymes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetically encoding colors and images into bioengineered
microbial materials. <em>NBT</em>, <em>43</em>(3), 306–307. (<a
href="https://doi.org/10.1038/s41587-024-02200-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using synthetic biology, we engineered a cellulose-producing bacterium that can produce eumelanin and respond to light, so that it is possible to grow a microbial leather material that is colored black or contains projected black patterns.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02200-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {306-307},
  shortjournal = {Nature Biotech.},
  title        = {Genetically encoding colors and images into bioengineered microbial materials},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decoding gene regulation with CRISPR perturbations.
<em>NBT</em>, <em>43</em>(3), 304–305. (<a
href="https://doi.org/10.1038/s41587-024-02222-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two CRISPR tools for combinatorial genetic perturbations reveal gene regulatory networks.},
  archive      = {J_NBT},
  author       = {Oberlin, Stefan and McManus, Michael T.},
  doi          = {10.1038/s41587-024-02222-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {304-305},
  shortjournal = {Nature Biotech.},
  title        = {Decoding gene regulation with CRISPR perturbations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A milestone method to make natural killer t cells.
<em>NBT</em>, <em>43</em>(3), 302–303. (<a
href="https://doi.org/10.1038/s41587-024-02243-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A differentiation protocol to produce off-the-shelf natural killer T cells may enable clinical application.},
  archive      = {J_NBT},
  author       = {Metelitsa, Leonid S.},
  doi          = {10.1038/s41587-024-02243-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {302-303},
  shortjournal = {Nature Biotech.},
  title        = {A milestone method to make natural killer t cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Intranasal bacteria transport obesity drugs to the brain.
<em>NBT</em>, <em>43</em>(3), 301. (<a
href="https://doi.org/10.1038/s41587-025-02598-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02598-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Intranasal bacteria transport obesity drugs to the brain},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Fluorescence imaging of RNA and protein in deep tissue.
<em>NBT</em>, <em>43</em>(3), 301. (<a
href="https://doi.org/10.1038/s41587-025-02604-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02604-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Fluorescence imaging of RNA and protein in deep tissue},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025c). Microbial metalloenzyme boosts cellulose breakdown.
<em>NBT</em>, <em>43</em>(3), 301. (<a
href="https://doi.org/10.1038/s41587-025-02615-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02615-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Microbial metalloenzyme boosts cellulose breakdown},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Epigenetics. <em>NBT</em>, <em>43</em>(3), 300. (<a
href="https://doi.org/10.1038/s41587-025-02582-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to epigenetic modification and analysis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02582-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {300},
  shortjournal = {Nature Biotech.},
  title        = {Epigenetics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The puzzle of biologics manufacturing platform patents.
<em>NBT</em>, <em>43</em>(3), 295–299. (<a
href="https://doi.org/10.1038/s41587-025-02579-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The predominance of manufacturing process patents asserted in litigation by originator biologics companies against would-be biosimilar entrants has resulted in a number of Congressional and administrative agency proposals that could increase scrutiny and limit enforceability.},
  archive      = {J_NBT},
  author       = {Jefferson, Osmat Azzam and Price, W. Nicholson and Tu, S. Sean and Vishnubhakat, Saurabh and Rai, Arti K.},
  doi          = {10.1038/s41587-025-02579-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {295-299},
  shortjournal = {Nature Biotech.},
  title        = {The puzzle of biologics manufacturing platform patents},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>,
<em>43</em>(3), 294. (<a
href="https://doi.org/10.1038/s41587-025-02599-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02599-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {294},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trapped in a lake, founder spots a bioplastic opportunity.
<em>NBT</em>, <em>43</em>(3), 293. (<a
href="https://doi.org/10.1038/s41587-025-02605-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-025-02605-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {293},
  shortjournal = {Nature Biotech.},
  title        = {Trapped in a lake, founder spots a bioplastic opportunity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tough times for CRISPR startups. <em>NBT</em>,
<em>43</em>(3), 292. (<a
href="https://doi.org/10.1038/s41587-025-02609-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02609-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {292},
  shortjournal = {Nature Biotech.},
  title        = {Tough times for CRISPR startups},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Baby genomics: Newborn sequencing starts to fulfill its
promise. <em>NBT</em>, <em>43</em>(3), 290–292. (<a
href="https://doi.org/10.1038/s41587-025-02595-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome sequencing in newborns is delivering insights into severe genetic diseases not afforded by traditional tests, but integrating genomics into newborn screening remains a logistics and cost challenge.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-025-02595-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {290-292},
  shortjournal = {Nature Biotech.},
  title        = {Baby genomics: Newborn sequencing starts to fulfill its promise},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Philanthropic giants join forces over antimicrobial
resistance. <em>NBT</em>, <em>43</em>(3), 289. (<a
href="https://doi.org/10.1038/s41587-025-02620-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02620-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {289},
  shortjournal = {Nature Biotech.},
  title        = {Philanthropic giants join forces over antimicrobial resistance},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). First inhaled lentiviral gene therapy enters cystic fibrosis
trial. <em>NBT</em>, <em>43</em>(3), 288. (<a
href="https://doi.org/10.1038/s41587-025-02616-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02616-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {288},
  shortjournal = {Nature Biotech.},
  title        = {First inhaled lentiviral gene therapy enters cystic fibrosis trial},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bite-sized solutions. <em>NBT</em>, <em>43</em>(3), 285–286.
(<a href="https://doi.org/10.1038/s41587-025-02603-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study found high amounts of microplastics in the human brain. This could spur funding and technological advances for plastic degradation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02603-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {285-286},
  shortjournal = {Nature Biotech.},
  title        = {Bite-sized solutions},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
