<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>BECB_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="becb---8">BECB - 8</h2>
<ul>
<li><details>
<summary>
(2023). Deep learning based classification of dermatological
disorders. <em>BECB</em>, <em>14</em>. (<a
href="https://doi.org/10.1177/11795972221138470">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Automated medical diagnosis has become crucial and significantly supports medical doctors. Thus, there is a demand for inventing deep learning (DL) and convolutional networks for analyzing medical images. Dermatology, in particular, is one of the domains that was recently targeted by AI specialists to introduce new DL algorithms or enhance convolutional neural network (CNN) architectures. A significantly high proportion of studies in the field are concerned with skin cancer, whereas other dermatological disorders are still limited. In this work, we examined the performance of 6 CNN architectures named VGG16, EfficientNet, InceptionV3, MobileNet, NasNet, and ResNet50 for the top 3 dermatological disorders that frequently appear in the Middle East. An Image filtering and denoising were imposed in this work to enhance image quality and increase architecture performance. Experimental results revealed that MobileNet achieved the highest performance and accuracy among the CNN architectures and can classify disorder with high performance (95.7% accuracy). Future scope will focus more on proposing a new methodology for deep-based classification. In addition, we will expand the dataset for more images that consider new disorders and variations.},
  archive      = {J_BECB},
  author       = {Lulwah AlSuwaidan},
  doi          = {10.1177/11795972221138470},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {Deep learning based classification of dermatological disorders},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Assessing paracetamol overdose in children: Acceptability
and potential market for a non-invasive testing device. <em>BECB</em>,
<em>14</em>. (<a
href="https://doi.org/10.1177/11795972221140108">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BECB},
  author       = {Debora Freitas and Christopher Parry and Gabrielle Seddon and Jana Lemke and James Moss and Neville Freeman and Julie Grice and Daniel B Hawcutt},
  doi          = {10.1177/11795972221140108},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {Assessing paracetamol overdose in children: Acceptability and potential market for a non-invasive testing device},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Hydroxyapatite-bioceramic/expanded perlite hybrid composites
coating on Ti6Al4V by hydrothermal method and in vitro behavior.
<em>BECB</em>, <em>14</em>. (<a
href="https://doi.org/10.1177/11795972231151348">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study was aimed to coat a hybrid bioceramic composite onto Ti 6 Al 4 V by using hydrothermal method. The Hybrid bioceramic composite for coating was prepared by reinforcing different rations of expanded perlite (EP) and 5 wt.% chitosan into synthesized Hydroxyapatite (HA). Coating was performed at 1800°C for 12 hours. The coated specimens were gradually subjected to a sintering at 6000°C for 1 hour. For in vitro analysis, the specimens were kept in Ringer’s solution for 1, 10, and 25 days. All specimens were examined by SEM, EDX, FTIR, and surface roughness analyses for characterizing. It was concluded that as the reinforcement ratio increased, there was an increase in coating thickness and surface roughness. The optimum reinforcement ratio for expanded perlite can be 10 wt.% (A3-B3). With increasing ratio of calcium (Ca) and phosphate (P) (Ca/P), the surface becomes more active in body fluid and then observed the formation of the hydroxycarbonate apatite (HCA) layer. As the waiting time increased, there was an increase in the formation of an apatite structure.},
  archive      = {J_BECB},
  author       = {Mehtap Muratoğlu and Tuğçe Özcan},
  doi          = {10.1177/11795972231151348},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {Hydroxyapatite-Bioceramic/Expanded perlite hybrid composites coating on Ti6Al4V by hydrothermal method and in vitro behavior},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). In-silico structural modeling of human immunodeficiency
virus proteins. <em>BECB</em>, <em>14</em>. (<a
href="https://doi.org/10.1177/11795972231154402">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human immunodeficiency virus (HIV) is an infectious virus that depletes the CD4 + T lymphocytes of the immune system and causes a chronic life-treating disease—acquired immunodeficiency syndrome (AIDS). The HIV genome encodes different structural and accessory proteins involved in viral entry and life cycle. Determining the 3D structure of HIV proteins is essential for new target position finding, structure-based drug designing, and future planning for computational and laboratory experimentations. Hence, the study aims to predict the 3D structures of all the HIV structural and accessory proteins using computational homology modeling to understand better the structural basis of HIV proteins interacting with host cells and viral replication. The sequences of HIV capsid, matrix, nucleocapsid, p6, reverse transcriptase, invertase, protease, gp120, gp41, virus protein r, viral infectivity factor, virus protein unique, RNA splicing regulator, transactivator protein, negative regulating factor, and virus protein x proteins were retrieved from UniProt. The primary and secondary structures of HIV proteins were predicted by Expasy ProtParam and SOPMA web servers. For the homology modeling, the MODELLER predicted the 3D structures of HIV proteins using templates. Then, the modeled structures were validated by the Ramachandran plot, local and global quality estimation scores, QMEAN scores, and Z -scores. Most of the amino acid residues of HIV proteins were present in the most favored and generously allowed regions in the Ramachandran plots. The local and global quality scores and Z -scores of the HIV proteins confirmed the good quality of modeled structures. The 3D modeled structures of HIV proteins might help further investigate the possible treatment.},
  archive      = {J_BECB},
  author       = {Amir Elalouf},
  doi          = {10.1177/11795972231154402},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {In-silico structural modeling of human immunodeficiency virus proteins},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Digital filtering and signal decomposition: A priori and
adaptive approaches in body area sensing. <em>BECB</em>, <em>14</em>.
(<a href="https://doi.org/10.1177/11795972231166236">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elimination of undesired signals from a mixture of captured signals in body area sensing systems is studied in this paper. A series of filtering techniques including a priori and adaptive approaches are explored in detail and applied involving decomposition of signals along a new system’s axis to separate the desired signals from other sources in the original data. Within the context of a case study in body area systems, a motion capture scenario is designed and the introduced signal decomposition techniques are critically evaluated and a new one is proposed. Applying the studied filtering and signal decomposition techniques demonstrates that the functional based approach outperforms the rest in reducing the effect of undesired changes in collected motion data which are due to random changes in sensors positioning. The results showed that the proposed technique reduces variations in the data for average of 94% outperforming the rest of the techniques in the case study although it will add computational complexity. Such technique helps wider adaptation of motion capture systems with less sensitivity to accurate sensor positioning; therefore, more portable body area sensing system.},
  archive      = {J_BECB},
  author       = {Roya Haratian},
  doi          = {10.1177/11795972231166236},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {Digital filtering and signal decomposition: A priori and adaptive approaches in body area sensing},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Finite element model-computed mechanical behavior of femurs
with metastatic disease varies between physiologic and idealized loading
simulations. <em>BECB</em>, <em>14</em>. (<a
href="https://doi.org/10.1177/11795972231166240">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BECB},
  author       = {Joshua E Johnson and Marc J Brouillette and Benjamin J Miller and Jessica E Goetz},
  doi          = {10.1177/11795972231166240},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {Finite element model-computed mechanical behavior of femurs with metastatic disease varies between physiologic and idealized loading simulations},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Prediction of druggable allosteric sites of undruggable
multidrug resistance efflux pump p. Gingivalis proteins. <em>BECB</em>,
<em>14</em>. (<a
href="https://doi.org/10.1177/11795972231202394">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_BECB},
  author       = {Pradeep Kumar Yadalam and Raghavendra Vamsi Anegundi and Artak Heboyan},
  doi          = {10.1177/11795972231202394},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {Prediction of druggable allosteric sites of undruggable multidrug resistance efflux pump p. gingivalis proteins},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Next-generation microfluidics for biomedical research and
healthcare applications. <em>BECB</em>, <em>14</em>. (<a
href="https://doi.org/10.1177/11795972231214387">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microfluidic systems offer versatile biomedical tools and methods to enhance human convenience and health. Advances in these systems enables next-generation microfluidics that integrates automation, manipulation, and smart readout systems, as well as design and three-dimensional (3D) printing for precise production of microchannels and other microstructures rapidly and with great flexibility. These 3D-printed microfluidic platforms not only control the complex fluid behavior for various biomedical applications, but also serve as microconduits for building 3D tissue constructs—an integral component of advanced drug development, toxicity assessment, and accurate disease modeling. Furthermore, the integration of other emerging technologies, such as advanced microscopy and robotics, enables the spatiotemporal manipulation and high-throughput screening of cell physiology within precisely controlled microenvironments. Notably, the portability and high precision automation capabilities in these integrated systems facilitate rapid experimentation and data acquisition to help deepen our understanding of complex biological systems and their behaviors. While certain challenges, including material compatibility, scaling, and standardization still exist, the integration with artificial intelligence, the Internet of Things, smart materials, and miniaturization holds tremendous promise in reshaping traditional microfluidic approaches. This transformative potential, when integrated with advanced technologies, has the potential to revolutionize biomedical research and healthcare applications, ultimately benefiting human health. This review highlights the advances in the field and emphasizes the critical role of the next generation microfluidic systems in advancing biomedical research, point-of-care diagnostics, and healthcare systems.},
  archive      = {J_BECB},
  author       = {Muhammedin Deliorman and Dima Samer Ali and Mohammad A Qasaimeh},
  doi          = {10.1177/11795972231214387},
  journal      = {Biomedical Engineering and Computational Biology},
  month        = {1-12},
  shortjournal = {Biomed. Eng. Comput. Biol.},
  title        = {Next-generation microfluidics for biomedical research and healthcare applications},
  volume       = {14},
  year         = {2023},
}
</textarea>
</details></li>
</ul>

</body>
</html>
